# A pharmacogenetic study of docetaxel and thalidomide in patients with castration-resistant prostate cancer using the DMET genotyping platform

## Metadata
**Authors:** JF Deeken, T Cormier, DK Price, TM Sissung, SM Steinberg, K Tran, DJ Liewehr, WL Dahut, X Miao, WD Figg
**Journal:** The pharmacogenomics journal
**Date:** 2009 Dec 29
**DOI:** [10.1038/tpj.2009.57](https://doi.org/10.1038/tpj.2009.57)
**PMID:** 20038957
**PMCID:** PMC6631360
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6631360/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6631360/pdf/nihms-1040366.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6631360/pdf/nihms-1040366.pdf)

## Abstract

The anticancer agent docetaxel shows significant inter-individual variation in its pharmacokinetic and toxicity profile. Thalidomide is an active anticancer agent and also shows wide pharmacological variation. Past pharmacogenetic research has not explained this variation. Patients with prostate cancer enrolled in a randomized phase II trial using docetaxel and thalidomide versus docetaxel alone were genotyped using the Affymetrix DMET 1.0 platform, which tests for 1256 genetic variations in 170 drug disposition genes. Genetic polymorphisms were analyzed for associations with clinical response and toxicity. In all, 10 single-nucleotide polymorphisms (SNPs) in three genes were potentially associated with response to therapy: peroxisome proliferator-activated receptor-δ (PPAR-δ), sulfotransferase family, cytosolic, 1C, member 2 (SULT1C2) and carbohydrate (chondroitin 6) sulfotransferase 3 (CHST3). In addition, 11 SNPs in eight genes were associated with toxicities to treatment: spastic paraplegia 7 (pure and complicated autosomal recessive) (SPG 7), CHST3, cytochrome P450, family 2, subfamily D, polypeptide 6 (CYP2D6), N-acetyltransferase 2 (arylamine N-acetyltransferase) (NAT2), ATP-binding cassette, sub-family C (CFTR/MRP), member 6 (ABCC6), ATPase, Cu + + transporting, alpha polypeptide (ATP7A), cytochrome P450, family 4, subfamily B, polypeptide 1 (CYP4B1) and solute carrier family 10 (sodium/bile acid cotransporter family), member 2 (SLC10A2). Genotyping results between drug metabolizing enzymes and transporters (DMET) and direct sequencing showed >96% of concordance. These findings highlight the role that non-CYP450 metabolizing enzymes and transporters may have in the pharmacology of docetaxel and thalidomide.

Keywords: pharmacogenomics, docetaxel, thalidomide, prostate cancer

## Introduction

Individualizing therapy for patients who are treated with pharmaceutical agents is an overarching goal of basic and clinical research in this first part of the twenty-first century. In no area of medicine is this goal more critical than in medical oncology. Genetic variation in genes encoding proteins involved in the metabolism and transport of drugs may account for some of the wide variation observed in the response to, and toxicity from, anticancer agents. Pharmacogenetic research holds the promise of reducing the unpredictable inter-individual variability in the efficacy and toxicity from using chemotherapy drugs.

Enzymes and transporters mediate the absorption, distribution, metabolism and excretion of endogenous as well as exogenous substrates, including drugs. However, for only a relatively small number of absorption, distribution, metabolism and excretion genes have single nucleotide polymorphisms (SNPs) and other genetic variations been identified that mediate the efficacy and toxicity of chemotherapy drugs. These include irinotecan and *UGT1A1*, 6-mercaptopurine and *thiopurine methyltransferase*, 5-fluoruracil and *dihydropyrimidine dehydrogenase* and tamoxifen and *cytochrome P450, family 2, subfamily D, polypeptide 6 (CYP2D6)*.^1^

Docetaxel (Taxotere) is a semisynthetic taxane that is currently used as first-line therapy in castration-resistant prostate cancer (CRPC). It is thought that the major route of metabolism is through the CYP3A family of enzymes.^2–4^ The pharmacokinetic profile of docetaxel shows wide inter-individual variability, with measures such as clearance and area under the concentration curve varying as much as sixfold between patients.^5,6^ Even in an ethnically homogenous patient population, variability is as high as 3.5-fold.^7^ Research using probes of CYP3A4 activity have been unsuccessful in explaining this large variation.^7–10^ In fact, the activity of CYP3A4/3A5 accounts for as little as two-thirds of the inter-individual variable pharmacology of the drug.^6,11^ This led researchers to look at other metabolizing enzymes, including glutathione-*S*-transferases, as well as drug transporters such as ABCB1 (P-glycoprotein), to help explain this large variability in drug pharmacology.^11–14^ The pharmacogenetic factors mediating docetaxel pharmacokinetics have not been fully characterized.^14^ We have previously proposed that a more comprehensive analysis into the expression and genetic polymorphisms of multiple drug enzymes and transporters may improve our understanding of the pharmacokinetics of docetaxel.^15^

Thalidomide, a potent teratogen that causes dysmelia in humans, has been approved for the treatment of multiple myeloma and is under active clinical analysis for its use in treating other malignancies. The anticancer mechanism of action of thalidomide is complex and has not yet been fully charecterized. *In vitro* data suggest that it inhibits angiogenesis. Non-enzymatic spontaneous hydrolysis is a major route of metabolism, with at least 12 different products identified by this process.^16^ However, enzymatic activation of these hydrolysis products is necessary for thalidomide to exert its anticancer activity.^17–19^ Two metabolites, 5-hydroxythalidomide and 5’-hydroxythalidomide, are formed by CYP2C19 and to a lesser extent CYP2B6.^20^ CYP1A1 may have a role in parent drug activation.^21^ Other metabolism products, including dihydroxylated and glucuronide conjugates, have been found in animal models.^22^ Past pharmacogenetic studies analyzing whether SNPs in *CYP2C19* accounted for the inter-individual variation in pharmacokinetics and toxicities have been inconclusive.^19,23^

Testing for the presence of genetic variations and SNPs is typically performed by DNA amplification using PCR and sequencing of specific gene fragments. This work is specialized and labor intensive, which in turn limits the institutions and investigators who can pursue genotyping and pharmacogenetic research. One of the main obstacles facing pharmacogenetic researchers is the lack of a proven, scalable genotyping technology that screens for SNPs in multiple genes with sufficient sensitivity and high-throughput capabilities to be useful in translational and clinical research. The new Affymetrix drug-metabolizing enzyme and transporter (DMET) genotyping platform (Affymetrix, Inc., Santa Clara, CA, USA) screens for 1256 genetic variations in 170 absorption, distribution, metabolism and excretion genes, including 50 CYP450 genes, 72 non-CYP genes, 39 transporters and 9 other proteins involved in drug disposition ([Table 1](#T1)).^24^ We tested this new genotyping platform, termed DMET, using patient samples and clinical data from our previously completed phase II trial in patients with CRPC to analyze the pharmacogenetics of thalidomide and docetaxel.

### Table 1.

| P450 Enzymes |  | Non-P450 enzymes |  |  | Transporters |  | Other |
| --- | --- | --- | --- | --- | --- | --- | --- |
| CYP1A1 | CYP4F2 | ADH1A | CROT | NQO1 | ABCB1 | SLC15A2 | AHR |
| CYP1A2 | CYP4F3 | ADH1B | DYPD | SULT1A1 | ABCB4 | SLC16A1 | NR1I2 |
| CYP1B1 | CYP4F8 | ADH4 | EPHX1 | SULT1A2 | ABCB7 | SLC19A1 | NR3C1 |
| CYP2A13 | CYP4Z1 | ADH5 | FMO1 | SULT1A3 | ABCB11 | SLC22A1 | PPARD |
| CYP2A6 | CYP46A1 | ADH6 | FMO2 | SULT1B1 | ABCC1 | SLC22A2 | PPARG |
| CYP2A7 | CYPA1 | ADH7 | FMO3 | SULT1C1 | ABCC2 | SLC22A3 | RALBP1 |
| CYP2B6 | CYP51A1 | ALDH1A1 | FMO4 | SULT1C2 | ABCC3 | SLC22A4 | RPL13 |
| CYP2B7P1 | CYP7A1 | ALDH2 | FMO5 | SULT1E1 | ABCC4 | SLC22A5 | SPG7 |
| CYP2C9 | CYP7B1 | ALDH3A1 | FMO6 | SULT2A1 | ABCC5 | SLC22A6 | XDH |
| CYP2C8 | CYP8A1 | AOX1 | GSTA1 | SULT2B1 | ABCC6 | SLC22A8 |   |
| CYP2C19 | CYP8B1 | COA | GSTA2 | SULT4A1 | ABCG2 | SLC28A1 |   |
| CYP2C18 | CYP11A1 | CHST1 | GSTA3 | TPMT | ATP7A | SLC28A2 |   |
| CYP2D6 | CYP11B1 | CHST2 | GSTA4 | UGT1A1 | ATP7B | SLC28A3 |   |
| CYP2E1 | CYP11B2 | CHST3 | GSTA5 | UGT1A3 | SLC5A6 | SLC29A1 |   |
| CYP2F1 | CYP17A1 | CHST4 | GSTM2 | UGT1A4 | SLC7A5 | SLC29A2 |   |
| CYP2J2 | CYP19A1 | CHST5 | GSTM3 | UGT1A6 | SLC7A7 | SLCO1A2 |   |
| CYP281 | CYP20A1 | CHST6 | GSTM4 | UGT1A7 | SLC10A1 | SLCO1B1 |   |
| CYP3A4 | CYP21A2 | CHST7 | GSTO1 | UGT1A8 | SLC10A2 | SLC1B3 |   |
| CYP3A43 | CYP24A1 | CHST8 | GSTP1 | UGT2A1 | SLC13A1 | SLCO2B1 |   |
| CYP3A5 | CYP26A1 | CHST9 | HNMT | UGT2B11 | SLC15A1 |   |   |
| CYP3A7 | CYP26C1 | CHST10 | MAOB | UGT2B15 |   |   |   |
| CYP4A11 | CYP27A1 | CHST11 | NAT1 | UGT2B28 |   |   |   |
| CYP4B1 | CYP27B1 | CHST13 | NAT2 | UGT2B4 |   |   |   |
| CYP4F11 | CYP39A1 | COMT | NNMT | UGT8 |   |   |   |
| CYP4F12 | POR |   |   |   |   |   |   |

Table 1 Caption: Genes included in the DMET 1.0 panel

## Patients and methods

### Patients

Patients diagnosed with CRPC were enrolled in a randomized phase II clinical trial comparing docetaxel to docetaxel with thalidomide. The results of this trial have been previously reported.^25^ In brief, 74 patients with CRPC were enrolled with a 2:1 randomization to receive either docetaxel and thalidomide, or docetaxel alone. Docetaxel was dosed at 30m gm^−2^ weekly for 3 weeks followed by a 1-week rest. For patients assigned to the combination arm, they received docetaxel on the same schedule, as well as thalidomide at 200 mg orally each day. In all, 49 patients were in the combination arm and 25 patients were in the control arm.^25^ The study was approved by the institutional review board of the National Cancer Institute.

Response to treatment was determined using the prostate-specific antigen Working Group consensus criteria of a drop in serum prostate-specific antigen of ≥50% with no other evidence of disease progression. Toxicities were graded using the Common Toxicity Criteria (version 2.0) of the Cancer Therapy Evaluation Program/National Cancer Institute. For this pharmacogenetic study, only adverse reactions graded as 3 (no grade 4 adverse events were experienced on this trial) and identified as being possibly, probably or definitely related to study medications were included. Events of bone pain, which is common in this patient population, as well as events of hyperglycemia (which were likely caused by the use of dexamethasone as a docetaxel pre-medication) were not included. Finally, events of venous thrombosis were not included because the trial was modified to add low-molecular-weight heparin to patients receiving thalidomide after an interim analysis found a high rate of such events early in the trial. Toxicity was used as a pharmacodynamic measure as well as a surrogate marker for drug pharmacokinetics as, at least in the case of docetaxel, clearance and pharmacokinetics have been previously found to correlate well with drug toxicity.^26–28^

### Patient samples

For 47 of the 74 originally enrolled patients, stored buffy coat samples were archived and available for DNA extraction. This included 33 patients on the combination arm of the trial and 14 patients on the docetaxel-alone arm. Blood was collected from patients, and the buffy coat layers were resuspended and frozen in Trizol (Invitrogen, Carlsbad, CA, USA). Genomic DNA was extracted from serum or white blood cell buffy coat layers of whole blood from patients using either the QiAamp Ultrasens Viral DNA kit (serum) or the QIAamp DNA blood kit (buffy coat) as described by the manufacturer (Qiagen, Valencia, CA, USA).

### Preparation and processing of the drug-metabolizing enzyme and transporter platform (DMET)

DNA processing and genotype identification for each patient sample were performed using the Affymetrix DMET platform as recently described.^24^ Genotypes were determined for each SNP site, reported as homozygous wild-type, heterozygous, homozygous variant or ‘no call’. Four samples were run in duplicate and the repeatability from both experiments was compared.

### Direct genotyping

Genotyping was conducted through direct sequencing for variants in eight genes that were previously thought to correlate with prostate cancer risk or the pharmacology of docetaxel and thalidomide ([Table 5](#T5)). PCR was performed on gene fragments for each variant site using primers and PCR conditions as previously performed for *CYP17*,^29^ *CYP2C19*,^30^ *CYP1B1*,^31^ *CYP3A5*,^32^ *ABCB1*^33^ and *ABCG2*.^34^ A 50-μl reaction was prepared for PCR amplification. The reaction consisted of 1 × PCR buffer, 2 mmoll^−1^ of each of the four deoxynucleotide triphosphates, 1.5 mmol l magnesium chloride, 20 mmoll^−1^ of the forward and reverse primers and 1 unit of Platinum Taq DNA polymerase. Direct nucleotide sequencing was performed using the dRhodamine Terminator Cycle Sequencing Ready Reaction kit or the Big Dye Terminator Cycle Sequencing Ready Reaction kit version 1.1 on an ABI 310 genetic analyzer (Applied Biosystems, Foster City, CA, USA). The CYP17 polymorphism was analyzed using the PCR-restriction fragment length polymorphism assay. For peroxisome proliferator-activated receptor-*δ* (PPAR-*δ*), genomic DNA containing the 798T >C SNP site was amplified using PCR. Primer sequences were as follows: F1, 5’-AGCAAC ACTCACCGCCGTGTG-30 and R1, 5’-ACCTCTGACATCCCCA TCCCTT-3’. After an initial denaturation at 94 °C for 5m in, 40 cycles of amplification with denaturation at 94 °C for 30 s, annealing at 64 °C for 30 s and extension at 72 °C for 30 s was performed, followed by a final extension step of 7 min at 72 °C. Direct nucleotide sequencing PCR was conducted using the Big Dye Terminator Cycle Sequencing Ready Reaction kit V1.1 on an ABI Prism 3130 × l genetic analyzer (Applied BioSystems). The primer sequences for these reactions were as follows: F2 5’-TGGCTTTGCCGGTGAGGA TGC-3’ and R2 5’-GCAGTCAGCAAGGAGCCCAG-30. Genotyping results by direct sequencing were previously reported and not found to correlate with clinical parameters.^25^

### Table 5.

| Gene | SNP | Concordance |
| --- | --- | --- |
| CYP17 | A1A1 | 100 |
| CYP2C19 | *2 | 97 |
| CYP2C19 | *3 | 100 |
| CYP1B1 | *3 | 96 |
| CYP3A5 | *3C | 100 |
| ABCB1 | 2677G>T | 96 |
| ABCG2 | Q141K | 100 |
| PPAR-δ | Rs2076167 | 100 |

Table 5 Caption: Concordance between DMET and direct sequencing genotyping

### Statistical analysis

In an exploratory manner, the association between the SNP parameters and clinical response or toxicity were tested using Mehta’s modification to Fisher’s exact test.^35^ Results of potential interest were limited to those in which the *P*-value was < 0.01. *P*-values < 0.001 would suggest even stronger association. Results are to be interpreted as hypothesis generating and in the context of a large number of exploratory evaluations performed.

## Results

### Patients

Of the 75 patients in the original study, sufficient DNA was available from 47 individuals. Demographic and clinical data on these patients are presented in [Table 2](#T2). The proportion of patients in each arm differed from the original 2:1 randomization, with slightly more patients on the combination arm (70 vs 67% in the parent study). Of the 14 patients who received docetaxel alone and were included in this pharmacogenetic study, 50% had a clinical response (partial or complete response), compared with a response rate of 37% in the overall study. Of the patients who received thalidomide and docetaxel, 73% of the patients included in this study had a clinical response compared with 53% in the parent trial. Whether a sample was archived and was of sufficient volume to enable DNA extraction was in no way related to the patient’s response on the trial, and the higher response rates in this subset of patients cannot be readily explained. Toxicities experienced during the trial of grade 3 are listed in [Table 3](#T3), along with the percentage of study patients experiencing each toxicity.

### Table 2.

| No. of patients | 47 |
| --- | --- |
| Age, yearsa | 66.1 (41–79) |
| Race |   |
| Caucasian | 43 |
| African American | 3 |
| Asian | 1 |
| Gleason at diagnosisa | 7.9 (6–10) |
| Baseline PSAa | 252 (5–1068) |
| Trial armb |   |
| Docetaxel | 14 |
| Docetaxel and thalidomide | 33 |
| Response |   |
| Docetaxel | 7 (50%) |
| Docetaxel and thalidomide | 24 (73%) |

Table 2 Caption: Patient characteristics

### Table 3.

| Adverse reaction | No. of patients | Incidence (%) |
| --- | --- | --- |
| Constitutional |   |   |
| Fatigue | 3 | 6 |
| Myalgias/weakness | 3 | 6 |
| Allergic reaction | 2 | 4 |
| Cardiovascular |   |   |
| Arrhythmia | 2 | 4 |
| Chest pain/ischemia | 2 | 4 |
| Neurological |   |   |
| Dizziness | 1 | 2 |
| Syncope | 4 | 9 |
| Hallucinations | 1 | 2 |
| Seizure | 1 | 2 |
| Hematological |   |   |
| Neutropenia | 2 | 4 |
| Infection | 7 | 15 |
| Anemia | 2 | 4 |
| Thrombocytopenia | 1 | 2 |
| Bleeding | 3 | 6 |
| Metabolism |   |   |
| Electrolyte disturbances | 9 | 19 |
| Hepatic dysfunction | 3 | 6 |
| Gastrointestinal |   |   |
| Constipation | 1 | 2 |
| Diarrhea | 3 | 6 |
| Ileus | 1 | 2 |
| Pulmonary |   |   |
| Dyspnea | 5 | 11 |

Table 3 Caption: Incidence of adverse reactions

### DNA extraction and processing of the drug-metabolizing enzyme and transporter platform (DMET)

Extracted DNA content from archived samples ranged between 3.0 and 930.9 ng. Although the recommended minimal DNA sample content for optimal DMET analysis was 2000 ng, we obtained call rates of between 88 and 98% for all SNP loci for those samples containing at least 92 ng of DNA. No-call rates in those samples with < 92 ng *(n =* 15) were lower, ranging between 21 and 91%. Four samples were run in duplicate and the results were compared between the two genotyping results across the 1256 genetic variation sites. Repeatability of results ranged between 98.1 and 99.9%.

Single-nucleotide polymorphisms in the 170 DMET genes were identified. An SNP is defined as a genetic variant that occurs in at least 1% of a population. Frequency of the variant alleles was in general < 10% (or > 9 0 % in those cases in which the reference ‘wild-type’ allele is the less common one). For 373 of the 1256 polymorphisms, or 30% of all loci, variant allelic frequencies were between 10 and 90% ([Figure 1](#F1)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b532/6631360/2bbfbde39ecb/nihms-1040366-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6631360_nihms-1040366-f0001.jpg)

Frequency of variant alleles. The frequencies of the single-nucleotide polymorphism (SNP) variant alleles in the 170 genes are graphed. The frequency of variant alleles was in general < 10% (or >90% in those cases in which the reference ‘wild-type’ allele is the less common one). For 373 of the 1256 SNPs, or 30% of all loci, variant allelic frequencies were between 10 and 90%.

### Single-nucleotide polymorphisms (SNPs) associated with clinical response

In an initial screening procedure that used the generalized Fisher’s exact test, 28 SNPs were identified in which the *P*-value was < 0.05. Given the exploratory nature of this study, and to limit false-positive results, we further limited reported results to those with a *P*-value of < 0.01. Using this criterion, 10 SNPs in three genes were found to be associated with response to therapy: *PPAR-δ, sulfotransferase family, cytosolic, 1C, member 2 (SULT1C2)* and *carbohydrate (chondroitin 6) sulfotransferase 3 (CHST3)* ([Table 4](#T4)).

### Table 4.

| Gene | SNP | Variant frequency | Clinical response | Toxicity |
| --- | --- | --- | --- | --- |
| PPAR-δ | rs6922548 | 0.15 | 0.0011 | NS |
| rs2016520 | 0.32 | 0.0056a | NS |  |
| rs1883322 | 0.32 | 0.0061a | NS |  |
| rs3734254 | 0.73 | 0.0089a | NS |  |
| rs7769719 | 0.90 | 0.0055a | NS |  |
| CHST3 | rs4148943 | 0.58 | 0.0001 | NS |
| rs4148947 | 0.43 | 0.0023 | NS |  |
| rs12418 | 0.42 | 0.0005 | NS |  |
| rs730720 | 0.40 | 0.0034 | NS |  |
| rs4148950 | 0.42 | 0.024b | 0.006 |  |
| rs1871450 | 0.44 | 0.048b | 0.006 |  |
| rs4148945 | 0.61 | 0.011b | 0.010 |  |
| SULT1C2 | rs1402467 | 0.22 | 0.0083 | NS |
| SPG7 | rs2292954 | 0.73 | NS | 0.0004 |
| rs12960 | 0.82 | NS | 0.004 |  |
| CYP2D6 | CYP2D6*19 (2539–2542del) | 0.48 | NS | 0.002 |
| NAT2 | rs1799931 | 0.92 | NS | 0.003 |
| ABCC6 | rs2238472 | 0.67 | NS | 0.006 |
| ATP7A | rs2227291 | 0.78 | NS | 0.006 |
| CYP4B1 | rs4646487 | 0.08 | NS | 0.008 |
| SLC10A2 | rs2301159 | 0.34 | NS | 0.010 |

Table 4 Caption: Genes and SNPs correlating with clinical response and toxicity

### Single-nucleotide polymorphisms (SN Ps) and toxicity

For toxicities from therapy, 11 SNPs in eight genes were found to be associated at a significance level of *P* < 0.01 ([Table 4](#T4)). These included two SNPs in *spastic paraplegia* 7 *(pure and complicated autosomal recessive) (SPG7)*, three in *CHST3* and single SNPs in *CYP2D6, N-acetyltransferase 2 (arylamine N-acetyltransferase) (NAT2), ATP-binding cassette, sub-family C (CFTR/MRP), member 6 (ABCC6)*, ATPase, Cu + + transporting, alpha polypeptide *(ATP7A), cytochrome P450, family 4, subfamily B, polypeptide 1 (CYP4B1)* and *solute carrier family 10 (sodium/bile acid cotransporter family), member 2 (SLC10A2)*. There was no association between patients with the *CYP3A5*3C* genotype, a poor metabolizing phenotype previously found to be associated with docetaxel pharmacology, and docetaxel toxicities (*P* = 0.51). No other variant in *CYP3A5* or *CYP3A4* was associated with toxicity (data not shown).

### Genotyping with the drug-metabolizing enzyme and transporter platform (DMET) versus direct sequencing genotyping

Genetic variations in eight genes were identified using direct sequencing, and the results were compared with those results using the DMET platform. Concordance rates between these results ranged between 96 and 100% ([Table 5](#T5)). All of the polymorphisms in [Tables 4](#T4) and [5](#T5) were equally distributed across the two trial arms (*P*>0.05;generalized Fisher’s exact test). It is to be noted that in cases in which there were disagreements between the two result sets, and sufficient DNA was available, patient samples were resequenced. In each case tested, the initial results found by direct sequencing were found to be in error, and the results from the DMET platform processing were confirmed (data not shown).

## Discussion

We present the findings from an exploratory pharmacogenomic study that related SNPs with the efficacy and toxicity of docetaxel and thalidomide used in a clinical trial of patients with CRPC. These data revealed that polymorphisms in three genes (that is, *PPARδ, SULT1C2* and *CHST3)* were associated with clinical outcome measures whereas polymorphisms in eight genes (that is, *SPG7, CHST3, CYP2D6, NAT2, ABCC6, ATP7A, CYP4B1* and *SLC10A2)* were associated with toxicity. To our knowledge, these associations represent novel genes that may be related to either thalidomide- or docetaxel-based therapy, although this remains unclear given that no reports have indicated that any of the above genes are involved in the metabolism or disposition toward either agent. Moreover, the allelic variations in genes that are known to be involved in metabolism, transport and activity of docetaxel and thalidomide were not associated with any of the study end points. Thus, further studies are warranted to validate and explore these relationships, and we can only speculate as to why such strong relationships exist. Although all of these genes may be related to drug metabolism directly, and thus could be related to pharmacokinetics, they also participate in pathways that may affect drug action and could therefore be involved in pharmacodynamic interactions as well. Finally, some of these alleles may be related to disease state and not necessarily alter drug metabolism or disposition.

*Peroxisome proliferator-activated receptor-δ*, one of three members of the PPAR family, is a ligand-activated transcription factor that forms heterodimers with the retinoid X receptor, which then binds to peroxisome proliferator response elements. The gene product has been identified for its role in lipid and cholesterol metabolism,^36^ whereas the polymorphisms in the gene have been related to inter-individual variation in cholesterol metabolism,^37^ insulin sensitivity,^38^ psychological disorders^39^ and reduced height.^40^ Although no study has yet evaluated *PPARδ* SNPs in relation to antiangiogenesis therapy, *PPARδ* gene expression is a major hub node in the angiogesis network,^41^ thus opposing thalidomide action. Given that we observed strong relationships with several SNPs in *PPARδ* in only those patients who received both docetaxel and thalidomide, but not docetaxel alone, it seems that allelic variation in *PPARδ* may influence therapeutic efficacy of the antiangiogenesis agent, thalidomide. However, this relationship is unclear as PPARδ is a nuclear receptor that is responsible for the induction of genes involved in metabolic pathways as well.

*Carbohydrate (chondroitin 6) sulfotransferase 3* encodes a member of the carbohydrate sulfotransferase family, termed chondroitin 6-O-sulfotransferase, as it is known to transfer sulfate groups from 3’-phosphoadenosine 5’-phosphosulfate to the six position of *N*-acetylgalactosamine of chondroitin. CHST3 is involved in extracellular matrix remodeling,^42^ and may thus be involved in invasion/metastasis, or in endothelial cell adhesion.^43^ It is also involved in drug metabolism within the liver.^44^ Numerous SNPs that may affect function have been identified.^45^ An intriguing finding in this study was that genetic variations in *CHST3* were associated with both efficacy and toxicity, increasing the likelihood that this finding of association may indicate a role in docetaxel or thalidomide pharmacokinetics and/or pharmacodynamics. Further analyses into this gene and its possible role in drug pharmacology are warranted.

*Sulfotransferase family, cytosolic, 1C, member 2* encodes a sulfotransferase that is thought to be involved in the metabolism of drugs and endogenous substrates, and is expressed both intra- and extra-hepatically as well as in the adult kidney.^46,47^ Genetic variation in this gene and other cytosolic sulfotransferases were recently reported.^48^ SULT1C2 seems to be downregulated during breast cancer progression in which it may be involved with steroid hormone metabolism and detoxification reactions.^49,50^ Sulfation reactions are not known to be involved in the metabolism of docetaxel or thalidomide. Given the strong extrahepatic expression of *SULT1C2*, it is equally likely that SULT1C2 is involved in the bodily disposition toward these drugs through direct metabolism, or indirect metabolism of other endogenous/exogenous agents that influence the activity of docetaxel and/or thalidomide. It is also possible that these SNPs are merely related to disease state, and not drug activity.

Single-nucleotide polymorphisms in eight genes were found to be associated with docetaxel and thalidomide toxicities. *SPG7* encodes a mitochondrial metalloprotease protein known as paraplegin. Genetic variations in *SPG7* have been found in patients with spastic paraplegia, a degenerative neurological condition.^51^ A closely related gene, *SPG6*, which is also associated with spastic paraplegia, was recently found to function as a transporter.^52^ CYP2D6 and CYP4B1 are members of the P450 family of phase I metabolic enzymes. CYP2D6 is known to be involved in the metabolism of a wide range of drugs.^1^ Inhibition of this enzyme was not found to affect the formation of hydroxyldocetaxel, which is thought to be mediated by CYP3A4.^53^ W hether this enzyme is involved in the formation of other docetaxel metabolites is unknown. Human microsome studies did not find a significant role for CYP2D6 and other CYP enzymes in thalidomide metabolism.^54^ It is also possible that certain CYP2D6 alleles may confer a multiple-chemical sensitivity phenotype that increases the likelihood of development of toxicities after drug treatment in general.^55^

*N-acetyltransferase 2* (*arylamine N-acetyltransferase*) encodes a polymorphic drug and xenobiotic-metabolizing enzyme that acetylates substrates. Genetic variants in *NAT2* can cause a ‘slow’, ‘intermediate’ or ‘rapid’ acetylator phenotype. These variants can predict efficacy and toxicity of a wide range of drugs, including sulfasalazine^56^ and isoniazid.^57^ *ABCC6* encodes a xenobiotic transporter from the C subfamily of the ATP-binding cassette (ABC) family of transporters. ABCC6, also termed MRP6, is known to be involved in the transport of glutathione conjugates and may mediate drug resistance to agents, such as etoposide, teniposide, doxorubicin and daunorubicin.^58^ *ATP7A* encodes a transporter that is characterized best for its role in copper disposition, and is also thought to mediate cisplatin and carboplatin transport and resistance.^59^ Finally, *SLC10A2* encodes a sodium-dependent transporter highly expressed in the ileum and liver involved in the enterohepatic circulation of bile acids.^60^ Polymorphisms in these enzymes and transporters might alter the disposition toward xenobiotics in general, and could potentially be involved in pharmacokinetic and pharmacodynamic relationships with docetaxel and thalidomide. However, as the mechanism behind these associations is unclear, it is difficult to ascertain the importance of these findings.

This is the first study to report that these enzymes and transporters may be involved in the disposition of docetaxel and thalidomide, and/or their metabolites. This raises the possibility that these results are false positives and reflect only random associations that are found to be statistically significant. The risk of false-positive findings can be common in pharmacogenetic research as more polymorphisms, combinations of polymorphisms or phenotypes are included in studies with limited number of study subjects.^61^ Given the breath of the DMET platform, with 170 genes and 1256 polymorphisms, this was a potential shortcoming in our study. To reduce the risk of false-positive results, we restricted reported results to those in which the *P-*value was < 0.01, using Mehta’s modification to Fisher’s exact test. As an exploratory study analyzing associations between SNPs and clinical parameters, this was meant to be hypothesis generating and provide targets for future—and larger—pharmacogenetic clinical studies.^61^ Research to confirm these exploratory findings are ongoing.

The promise of the emerging field of pharmacogenetics is that someday we may enter an era of ‘individualized medicine’ in which physicians can choose the right drug at the right dose for each patient. In no area of pharmacotherapy is this more important than in medical oncology. Given that we still only poorly understand the relative importance of enzymes and transporters involved in the pharmacokinetics of chemotherapy drugs,^1^ the challenge facing pharmacogenetic and oncology researchers is the ability to screen for polymorphisms in many genes in order that we may better narrow our search for the key players in each drug’s metabolic pathway.

We believe that a research strategy for understanding the pharmacogenetics of anticancer agents can be pursued using exploratory screening for polymorphisms that may correlate with pharmacokinetics and/or pharmacodynamics. These exploratory studies can then be followed by high-throughput analyses using larger clinical sample sets.^11,61^ New tools, such as the DMET platform, will make pharmacogenetic research available to more researchers, which may lead to answering clinical questions in medical oncology and other fields. This, in turn, may hasten the day when the promise of individualized pharmacogenetic therapy becomes a reality for more of our cancer patients.

## Acknowledgments

This work was supported, in part, by the Intramural Research Program of the NIH, National Cancer Institute, Center for Cancer Research, Bethesda, MD, USA. Partial findings contained in this article were initially presented at the 2007 Annual Convention of the American Society of Clinical Oncology.

## Abbreviations

## Footnotes

## References

1. Deeken J, Figg W, Bates SE, Sparreboom A. Toward individualized treatment: prediction of anticancer drug disposition and toxicity with pharmacogenetics. Anticancer Drugs 2007; 18: 111–126.  [DOI](https://doi.org/10.1097/CAD.0b013e3280109411) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17159598/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20Drugs&title=Toward%20individualized%20treatment:%20prediction%20of%20anticancer%20drug%20disposition%20and%20toxicity%20with%20pharmacogenetics&author=J%20Deeken&author=W%20Figg&author=SE%20Bates&author=A%20Sparreboom&volume=18&publication_year=2007&pages=111-126&pmid=17159598&doi=10.1097/CAD.0b013e3280109411&)

2. Sparreboom A, Van Tellingen O, Scherrenburg EJ, Boesen JJ, Huizing MT, Nooijen WJ et al. Isolation, purification and biological activity of major docetaxel metabolites from human feces. Drug Metab Dispos 1996; 24: 655–668.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8781781/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Isolation,%20purification%20and%20biological%20activity%20of%20major%20docetaxel%20metabolites%20from%20human%20feces&author=A%20Sparreboom&author=O%20Van%20Tellingen&author=EJ%20Scherrenburg&author=JJ%20Boesen&author=MT%20Huizing&volume=24&publication_year=1996&pages=655-668&pmid=8781781&)

3. Marre F, Sanderink GJ, de Sousa G, Gaillard C, Martinet M, Rahmani R. Hepatic biotransformation of docetaxel (Taxotere) in vitro: involvement of the CYP3A subfamily in humans. Cancer Res 1996; 56: 1296–1302.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8640817/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Hepatic%20biotransformation%20of%20docetaxel%20(Taxotere)%20in%20vitro:%20involvement%20of%20the%20CYP3A%20subfamily%20in%20humans&author=F%20Marre&author=GJ%20Sanderink&author=G%20de%20Sousa&author=C%20Gaillard&author=M%20Martinet&volume=56&publication_year=1996&pages=1296-1302&pmid=8640817&)

4. Shou M, Martinet M, Korzekwa KR, Krausz KW, Gonzalez FJ, Gelboin HV. Role of human cytochrome P450 3A4 and 3A5 in the metabolism of taxotere and its derivatives: enzyme specificity, interindividual distribution and metabolic contribution in human liver. Pharmacogenetics 1998; 8: 391–401.  [DOI](https://doi.org/10.1097/00008571-199810000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9825831/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Role%20of%20human%20cytochrome%20P450%203A4%20and%203A5%20in%20the%20metabolism%20of%20taxotere%20and%20its%20derivatives:%20enzyme%20specificity,%20interindividual%20distribution%20and%20metabolic%20contribution%20in%20human%20liver&author=M%20Shou&author=M%20Martinet&author=KR%20Korzekwa&author=KW%20Krausz&author=FJ%20Gonzalez&volume=8&publication_year=1998&pages=391-401&pmid=9825831&doi=10.1097/00008571-199810000-00004&)

5. Clarke SJ, Rivory LP. Clinical pharmacokinetics of docetaxel. Clin Pharmacokinet 1999; 36: 99–114.  [DOI](https://doi.org/10.2165/00003088-199936020-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10092957/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Clinical%20pharmacokinetics%20of%20docetaxel&author=SJ%20Clarke&author=LP%20Rivory&volume=36&publication_year=1999&pages=99-114&pmid=10092957&doi=10.2165/00003088-199936020-00002&)

6. Hirth J, Watkins PB, Strawderman M, Schott A, Bruno R, Baker LH. The effect of an individual’s cytochrome CYP3A4 activity on docetaxel clearance. Clin Cancer Res 2000; 6: 1255–1268.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10778948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=The%20effect%20of%20an%20individual%E2%80%99s%20cytochrome%20CYP3A4%20activity%20on%20docetaxel%20clearance&author=J%20Hirth&author=PB%20Watkins&author=M%20Strawderman&author=A%20Schott&author=R%20Bruno&volume=6&publication_year=2000&pages=1255-1268&pmid=10778948&)

7. Goh BC, Lee SC, Wang LZ, Fan L, Guo JY, Lamba J et al. Explaining interindividual variability of docetaxel pharmacokinetics and pharmacodynamics in Asians through phenotyping and genotyping strategies. J Clin Oncol 2002; 20: 3683–3690.  [DOI](https://doi.org/10.1200/JCO.2002.01.025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12202670/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Explaining%20interindividual%20variability%20of%20docetaxel%20pharmacokinetics%20and%20pharmacodynamics%20in%20Asians%20through%20phenotyping%20and%20genotyping%20strategies&author=BC%20Goh&author=SC%20Lee&author=LZ%20Wang&author=L%20Fan&author=JY%20Guo&volume=20&publication_year=2002&pages=3683-3690&pmid=12202670&doi=10.1200/JCO.2002.01.025&)

8. Engels FK, Ten Tije AJ, Baker SD, Lee CK, Loos WJ, Vulto AG et al. Effect of cytochrome P450 3A4 inhibition on the pharmacokinetics of docetaxel. Clin Pharmacol Ther 2004; 75: 448–454.  [DOI](https://doi.org/10.1016/j.clpt.2004.01.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15116057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Effect%20of%20cytochrome%20P450%203A4%20inhibition%20on%20the%20pharmacokinetics%20of%20docetaxel&author=FK%20Engels&author=AJ%20Ten%20Tije&author=SD%20Baker&author=CK%20Lee&author=WJ%20Loos&volume=75&publication_year=2004&pages=448-454&pmid=15116057&doi=10.1016/j.clpt.2004.01.001&)

9. Puisset F, Chatelut E, Dalenc F, Busi F, Cresteil T, Azema J et al. Dexamethasone as a probe for docetaxel clearance. Cancer Chemother Pharmacol 2004; 54: 265–272.  [DOI](https://doi.org/10.1007/s00280-004-0823-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15133628/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Dexamethasone%20as%20a%20probe%20for%20docetaxel%20clearance&author=F%20Puisset&author=E%20Chatelut&author=F%20Dalenc&author=F%20Busi&author=T%20Cresteil&volume=54&publication_year=2004&pages=265-272&pmid=15133628&doi=10.1007/s00280-004-0823-0&)

10. Engels FK, Mathot RA, Loos WJ, van Schaik RH, Verweij J. Influence of high-dose ketoconazole on the pharmacokinetics of docetaxel. Cancer Biol Ther 2006; 5: 833–839.  [DOI](https://doi.org/10.4161/cbt.5.7.2839) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16775418/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Biol%20Ther&title=Influence%20of%20high-dose%20ketoconazole%20on%20the%20pharmacokinetics%20of%20docetaxel&author=FK%20Engels&author=RA%20Mathot&author=WJ%20Loos&author=RH%20van%20Schaik&author=J%20Verweij&volume=5&publication_year=2006&pages=833-839&pmid=16775418&doi=10.4161/cbt.5.7.2839&)

11. Bosch TM, Huitema AD, Doodeman VD, Jansen R, Witteveen E, Smit WM et al. Pharmacogenetic screening of CYP3A and ABCB1 in relation to population pharmacokinetics of docetaxel. Clin Cancer Res 2006; 12: 5786–5793.  [DOI](https://doi.org/10.1158/1078-0432.CCR-05-2649) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17020985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Pharmacogenetic%20screening%20of%20CYP3A%20and%20ABCB1%20in%20relation%20to%20population%20pharmacokinetics%20of%20docetaxel&author=TM%20Bosch&author=AD%20Huitema&author=VD%20Doodeman&author=R%20Jansen&author=E%20Witteveen&volume=12&publication_year=2006&pages=5786-5793&pmid=17020985&doi=10.1158/1078-0432.CCR-05-2649&)

12. Park JS, Yamamoto W, Sekikawa T, Matsukawa M, Okamoto R, Sasaki M et al. Cellular sensitivity determinants to docetaxel in human gastro-intestinal cancers. Int J Oncol 2002; 20: 333–338.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11788897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Oncol&title=Cellular%20sensitivity%20determinants%20to%20docetaxel%20in%20human%20gastro-intestinal%20cancers&author=JS%20Park&author=W%20Yamamoto&author=T%20Sekikawa&author=M%20Matsukawa&author=R%20Okamoto&volume=20&publication_year=2002&pages=333-338&pmid=11788897&)

13. Iwao-Koizumi K, Matoba R, Ueno N, Kim SJ, Ando A, Miyoshi Y et al. Prediction of docetaxel response in human breast cancer by gene expression profiling. J Clin Oncol 2005; 23: 422–431.  [DOI](https://doi.org/10.1200/JCO.2005.09.078) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15659489/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Prediction%20of%20docetaxel%20response%20in%20human%20breast%20cancer%20by%20gene%20expression%20profiling&author=K%20Iwao-Koizumi&author=R%20Matoba&author=N%20Ueno&author=SJ%20Kim&author=A%20Ando&volume=23&publication_year=2005&pages=422-431&pmid=15659489&doi=10.1200/JCO.2005.09.078&)

14. Tran A, Jullien V, Alexandre J, Rey E, Rabillon F, Girre V et al. Pharmacokinetics and toxicity of docetaxel: role of CYP3A, MDR1, and GST polymorphisms. Clin Pharmacol Ther 2006; 79: 570–580.  [DOI](https://doi.org/10.1016/j.clpt.2006.02.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16765145/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacokinetics%20and%20toxicity%20of%20docetaxel:%20role%20of%20CYP3A,%20MDR1,%20and%20GST%20polymorphisms&author=A%20Tran&author=V%20Jullien&author=J%20Alexandre&author=E%20Rey&author=F%20Rabillon&volume=79&publication_year=2006&pages=570-580&pmid=16765145&doi=10.1016/j.clpt.2006.02.003&)

15. Figg WD, Chau CH. Heterogeneity in drug disposition determines interindividual variability of docetaxel pharmacokinetics. Cancer Biol Ther 2006; 5: 840–851.  [DOI](https://doi.org/10.4161/cbt.5.7.3025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16880730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Biol%20Ther&title=Heterogeneity%20in%20drug%20disposition%20determines%20interindividual%20variability%20of%20docetaxel%20pharmacokinetics&author=WD%20Figg&author=CH%20Chau&volume=5&publication_year=2006&pages=840-851&pmid=16880730&doi=10.4161/cbt.5.7.3025&)

16. Schumacher H, Smith RL, Williams RT. The metabolism of thalidomide: the spontaneous hydrolysis of thalidomide in solution. Br J Pharmacol Chemother 1965; 25: 324–337.  [DOI](https://doi.org/10.1111/j.1476-5381.1965.tb02053.x) | [PMC free article](/articles/PMC1510736/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/5866715/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol%20Chemother&title=The%20metabolism%20of%20thalidomide:%20the%20spontaneous%20hydrolysis%20of%20thalidomide%20in%20solution&author=H%20Schumacher&author=RL%20Smith&author=RT%20Williams&volume=25&publication_year=1965&pages=324-337&pmid=5866715&doi=10.1111/j.1476-5381.1965.tb02053.x&)

17. Braun AG, Weinreb SL. Teratogen metabolism: spontaneous decay products of thalidomide and thalidomide analogues are not bioactivated by liver microsomes. Teratog Carcinog Mutagen 1985; 5: 149158.  [DOI](https://doi.org/10.1002/tcm.1770050304) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2866599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Teratog%20Carcinog%20Mutagen&title=Teratogen%20metabolism:%20spontaneous%20decay%20products%20of%20thalidomide%20and%20thalidomide%20analogues%20are%20not%20bioactivated%20by%20liver%20microsomes&author=AG%20Braun&author=SL%20Weinreb&volume=5&publication_year=1985&pages=149158&pmid=2866599&doi=10.1002/tcm.1770050304&)

18. Price DK, Ando Y, Kruger EA, Weiss M, Figg WD. 5’-OH-thalidomide, a metabolite of thalidomide, inhibits angiogenesis. Ther Druq Monit 2002; 24: 104–110.  [DOI](https://doi.org/10.1097/00007691-200202000-00017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11805730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Druq%20Monit&title=5%E2%80%99-OH-thalidomide,%20a%20metabolite%20of%20thalidomide,%20inhibits%20angiogenesis&author=DK%20Price&author=Y%20Ando&author=EA%20Kruger&author=M%20Weiss&author=WD%20Figg&volume=24&publication_year=2002&pages=104-110&pmid=11805730&doi=10.1097/00007691-200202000-00017&)

19. Lepper ER, Smith NF, Cox MC, Scripture CD, Figg WD. Thalidomide metabolism and hydrolysis: mechanisms and implications. Curr Drug Metab 2006; 7: 677–685.  [DOI](https://doi.org/10.2174/138920006778017777) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16918319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Metab&title=Thalidomide%20metabolism%20and%20hydrolysis:%20mechanisms%20and%20implications&author=ER%20Lepper&author=NF%20Smith&author=MC%20Cox&author=CD%20Scripture&author=WD%20Figg&volume=7&publication_year=2006&pages=677-685&pmid=16918319&doi=10.2174/138920006778017777&)

20. Ando Y, Fuse E, Figg WD. Thalidomide metabolism by the CYP2C subfamily. Clin Cancer Res 2002; 8: 1964–1973.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12060642/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Thalidomide%20metabolism%20by%20the%20CYP2C%20subfamily&author=Y%20Ando&author=E%20Fuse&author=WD%20Figg&volume=8&publication_year=2002&pages=1964-1973&pmid=12060642&)

21. Miyata M, Tamura E, Motoki K, Nagata K, Yamazoe Y. Thalidomide-induced suppression of embryo fibroblast proliferation requires CYP1A1-mediated activation. Drug Metab Dispos 2003; 31: 469–475.  [DOI](https://doi.org/10.1124/dmd.31.4.469) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12642474/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Thalidomide-induced%20suppression%20of%20embryo%20fibroblast%20proliferation%20requires%20CYP1A1-mediated%20activation&author=M%20Miyata&author=E%20Tamura&author=K%20Motoki&author=K%20Nagata&author=Y%20Yamazoe&volume=31&publication_year=2003&pages=469-475&pmid=12642474&doi=10.1124/dmd.31.4.469&)

22. Lu J, Helsby N, Palmer BD, Tingle M, Baguley BC, Kestell P et al. Metabolism of thalidomide in liver microsomes of mice, rabbits, and humans. J Pharmacol Exp Ther 2004; 310: 571–577.  [DOI](https://doi.org/10.1124/jpet.104.067793) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15075384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Metabolism%20of%20thalidomide%20in%20liver%20microsomes%20of%20mice,%20rabbits,%20and%20humans&author=J%20Lu&author=N%20Helsby&author=BD%20Palmer&author=M%20Tingle&author=BC%20Baguley&volume=310&publication_year=2004&pages=571-577&pmid=15075384&doi=10.1124/jpet.104.067793&)

23. Ando Y, Price DK, Dahut WL, Cox MC, Reed E, Figg WD. Pharmacogenetic associations of CYP2C19 genotype with in vivo metabolisms and pharmacological effects of thalidomide. Cancer Biol Ther 2002; 1: 669–673.  [DOI](https://doi.org/10.4161/cbt.318) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12642692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Biol%20Ther&title=Pharmacogenetic%20associations%20of%20CYP2C19%20genotype%20with%20in%20vivo%20metabolisms%20and%20pharmacological%20effects%20of%20thalidomide&author=Y%20Ando&author=DK%20Price&author=WL%20Dahut&author=MC%20Cox&author=E%20Reed&volume=1&publication_year=2002&pages=669-673&pmid=12642692&doi=10.4161/cbt.318&)

24. Dumaual C, Miao X, Daly TM, Bruckner C, Njau R, Fu DJ et al. Comprehensive assessment of metabolic enzyme and transporter genes using the Affymetrix targeted genotyping system. Pharmacogenomics 2007; 8: 293–305.  [DOI](https://doi.org/10.2217/14622416.8.3.293) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17324118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Comprehensive%20assessment%20of%20metabolic%20enzyme%20and%20transporter%20genes%20using%20the%20Affymetrix%20targeted%20genotyping%20system&author=C%20Dumaual&author=X%20Miao&author=TM%20Daly&author=C%20Bruckner&author=R%20Njau&volume=8&publication_year=2007&pages=293-305&pmid=17324118&doi=10.2217/14622416.8.3.293&)

25. Dahut WL, Gulley JL, Arlen PM, Liu Y, Fedenko KM, Steinberg SM et al. Randomized phase II trial of docetaxel plus thalidomide in androgen-independent prostate cancer. J Clin Oncol 2004; 22: 2532–2539.  [DOI](https://doi.org/10.1200/JCO.2004.05.074) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15226321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Randomized%20phase%20II%20trial%20of%20docetaxel%20plus%20thalidomide%20in%20androgen-independent%20prostate%20cancer&author=WL%20Dahut&author=JL%20Gulley&author=PM%20Arlen&author=Y%20Liu&author=KM%20Fedenko&volume=22&publication_year=2004&pages=2532-2539&pmid=15226321&doi=10.1200/JCO.2004.05.074&)

26. Bruno R, Hille D, Riva A, Vivier N, ten Bokkel Huinnink WW, van Oosterom AT et al. Population pharmacokinetics/pharmacodynamics of docetaxel in phase II studies in patients with cancer. J Clin Oncol 1998; 16: 187–196.  [DOI](https://doi.org/10.1200/JCO.1998.16.1.187) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9440742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Population%20pharmacokinetics/pharmacodynamics%20of%20docetaxel%20in%20phase%20II%20studies%20in%20patients%20with%20cancer&author=R%20Bruno&author=D%20Hille&author=A%20Riva&author=N%20Vivier&author=WW%20ten%20Bokkel%20Huinnink&volume=16&publication_year=1998&pages=187-196&pmid=9440742&doi=10.1200/JCO.1998.16.1.187&)

27. Bruno R, Vivier N, Veyrat-Follet C, Montay G, Rhodes GR. Population pharmacokinetics and pharmacokinetic-pharmacodynamic relationships for docetaxel. Invest New Drugs 2001; 19: 163–169.  [DOI](https://doi.org/10.1023/a:1010687017717) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11392450/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Invest%20New%20Drugs&title=Population%20pharmacokinetics%20and%20pharmacokinetic-pharmacodynamic%20relationships%20for%20docetaxel&author=R%20Bruno&author=N%20Vivier&author=C%20Veyrat-Follet&author=G%20Montay&author=GR%20Rhodes&volume=19&publication_year=2001&pages=163-169&pmid=11392450&doi=10.1023/a:1010687017717&)

28. Bruno R, Olivares R, Berille J, Chaikin P, Vivier N, Hammershaimb L et al. Alpha-1-acid glycoprotein as an independent predictor for treatment effects and a prognostic factor of survival in patients with non-small cell lung cancer treated with docetaxel. Clin Cancer Res 2003; 9: 1077–1082.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12631610/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Alpha-1-acid%20glycoprotein%20as%20an%20independent%20predictor%20for%20treatment%20effects%20and%20a%20prognostic%20factor%20of%20survival%20in%20patients%20with%20non-small%20cell%20lung%20cancer%20treated%20with%20docetaxel&author=R%20Bruno&author=R%20Olivares&author=J%20Berille&author=P%20Chaikin&author=N%20Vivier&volume=9&publication_year=2003&pages=1077-1082&pmid=12631610&)

29. Hamada A, Danesi R, Price DK, Sissung T, Chau C, Venzon D et al. Association of a CYP17 polymorphism with overall survival in Caucasian patients with androgen-independent prostate cancer. Urology 2007; 70: 217–220.  [DOI](https://doi.org/10.1016/j.urology.2007.06.1097) | [PMC free article](/articles/PMC1994899/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17826473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Urology&title=Association%20of%20a%20CYP17%20polymorphism%20with%20overall%20survival%20in%20Caucasian%20patients%20with%20androgen-independent%20prostate%20cancer&author=A%20Hamada&author=R%20Danesi&author=DK%20Price&author=T%20Sissung&author=C%20Chau&volume=70&publication_year=2007&pages=217-220&pmid=17826473&doi=10.1016/j.urology.2007.06.1097&)

30. Gardner ER, Burger H, van Schaik RH, van Oosterom AT, de Bruijn EA, Guetens G et al. Association of enzyme and transporter genotypes with the pharmacokinetics of imatinib. Clin Pharmacol Ther 2006; 80: 192–201.  [DOI](https://doi.org/10.1016/j.clpt.2006.05.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16890580/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Association%20of%20enzyme%20and%20transporter%20genotypes%20with%20the%20pharmacokinetics%20of%20imatinib&author=ER%20Gardner&author=H%20Burger&author=RH%20van%20Schaik&author=AT%20van%20Oosterom&author=EA%20de%20Bruijn&volume=80&publication_year=2006&pages=192-201&pmid=16890580&doi=10.1016/j.clpt.2006.05.003&)

31. Sissung TM, Danesi R, Price DK, Steinberg SM, de Wit R, Zahid M et al. Association of the CYP1B1*3 allele with survival in patients with prostate cancer receiving docetaxel. Mol Cancer Ther 2008; 7: 19–26.  [DOI](https://doi.org/10.1158/1535-7163.MCT-07-0557) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18187806/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&title=Association%20of%20the%20CYP1B1*3%20allele%20with%20survival%20in%20patients%20with%20prostate%20cancer%20receiving%20docetaxel&author=TM%20Sissung&author=R%20Danesi&author=DK%20Price&author=SM%20Steinberg&author=R%20de%20Wit&volume=7&publication_year=2008&pages=19-26&pmid=18187806&doi=10.1158/1535-7163.MCT-07-0557&)

32. Lepper ER, Baker SD, Permenter M, Ries N, van Schaik RH, Schenk PW et al. Effect of common CYP3A4 and CYP3A5 variants on the pharmacokinetics of the cytochrome P450 3A phenotyping probe midazolam in cancer patients. Clin Cancer Res 2005; 11: 7398–7404.  [DOI](https://doi.org/10.1158/1078-0432.CCR-05-0520) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16243813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Effect%20of%20common%20CYP3A4%20and%20CYP3A5%20variants%20on%20the%20pharmacokinetics%20of%20the%20cytochrome%20P450%203A%20phenotyping%20probe%20midazolam%20in%20cancer%20patients&author=ER%20Lepper&author=SD%20Baker&author=M%20Permenter&author=N%20Ries&author=RH%20van%20Schaik&volume=11&publication_year=2005&pages=7398-7404&pmid=16243813&doi=10.1158/1078-0432.CCR-05-0520&)

33. Sissung TM, Baum CE, Deeken J, Price DK, Aragon-Ching J, Steinberg SM et al. ABCB1 genetic variation influences the toxicity and clinical outcome of patients with androgen-independent prostate cancer treated with docetaxel. Clin Cancer Res 2008; 14: 4543–4549.  [DOI](https://doi.org/10.1158/1078-0432.CCR-07-4230) | [PMC free article](/articles/PMC2723795/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18628469/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=ABCB1%20genetic%20variation%20influences%20the%20toxicity%20and%20clinical%20outcome%20of%20patients%20with%20androgen-independent%20prostate%20cancer%20treated%20with%20docetaxel&author=TM%20Sissung&author=CE%20Baum&author=J%20Deeken&author=DK%20Price&author=J%20Aragon-Ching&volume=14&publication_year=2008&pages=4543-4549&pmid=18628469&doi=10.1158/1078-0432.CCR-07-4230&)

34. Honjo Y, Morisaki K, Huff LM, Robey RW, Hung J, Dean M et al. Single-nucleotide polymorphism (SNP) analysis in the ABC half-transporter ABCG2 (MXR/BCRP/ABCP1). Cancer Biol Ther 2002; 1: 696–702.  [DOI](https://doi.org/10.4161/cbt.322) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12642696/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Biol%20Ther&title=Single-nucleotide%20polymorphism%20(SNP)%20analysis%20in%20the%20ABC%20half-transporter%20ABCG2%20(MXR/BCRP/ABCP1)&author=Y%20Honjo&author=K%20Morisaki&author=LM%20Huff&author=RW%20Robey&author=J%20Hung&volume=1&publication_year=2002&pages=696-702&pmid=12642696&doi=10.4161/cbt.322&)

35. Mehta CR, Patel NR. A network algorithm for performing Fisher’s exact test in r × c contingency tables. J Am Stat Assoc 1983; 78: 427–434.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Stat%20Assoc&title=A%20network%20algorithm%20for%20performing%20Fisher%E2%80%99s%20exact%20test%20in%20r%20%C3%97%20c%20contingency%20tables&author=CR%20Mehta&author=NR%20Patel&volume=78&publication_year=1983&pages=427-434&)

36. Kramer DK, Al-Khalili L, Guigas B, Leng Y, Garcia-Roves PM, Krook A. Role of AMP kinase and PPARdelta in the regulation of lipid and glucose metabolism in human skeletal muscle. J Biol Chem 2007; 282: 19313–19320.  [DOI](https://doi.org/10.1074/jbc.M702329200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17500064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Role%20of%20AMP%20kinase%20and%20PPARdelta%20in%20the%20regulation%20of%20lipid%20and%20glucose%20metabolism%20in%20human%20skeletal%20muscle&author=DK%20Kramer&author=L%20Al-Khalili&author=B%20Guigas&author=Y%20Leng&author=PM%20Garcia-Roves&volume=282&publication_year=2007&pages=19313-19320&pmid=17500064&doi=10.1074/jbc.M702329200&)

37. Skogsberg J, Kannisto K, Cassel TN, Hamsten A, Eriksson P, Ehrenborg E. Evidence that peroxisome proliferator-activated receptor delta influences cholesterol metabolism in men. Arterioscler Thromb Vasc Biol 2003; 23: 637–643.  [DOI](https://doi.org/10.1161/01.ATV.0000064383.88696.24) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12615676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Evidence%20that%20peroxisome%20proliferator-activated%20receptor%20delta%20influences%20cholesterol%20metabolism%20in%20men&author=J%20Skogsberg&author=K%20Kannisto&author=TN%20Cassel&author=A%20Hamsten&author=P%20Eriksson&volume=23&publication_year=2003&pages=637-643&pmid=12615676&doi=10.1161/01.ATV.0000064383.88696.24&)

38. Vanttinen M, Nuutila P, Kuulasmaa T, Pihlajamaki J, Hallsten K, Virtanen KA et al. Single nucleotide polymorphisms in the peroxisome proliferator-activated receptor delta gene are associated with skeletal muscle glucose uptake. Diabetes 2005; 54: 3587–3591.  [DOI](https://doi.org/10.2337/diabetes.54.12.3587) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16306381/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=Single%20nucleotide%20polymorphisms%20in%20the%20peroxisome%20proliferator-activated%20receptor%20delta%20gene%20are%20associated%20with%20skeletal%20muscle%20glucose%20uptake&author=M%20Vanttinen&author=P%20Nuutila&author=T%20Kuulasmaa&author=J%20Pihlajamaki&author=K%20Hallsten&volume=54&publication_year=2005&pages=3587-3591&pmid=16306381&doi=10.2337/diabetes.54.12.3587&)

39. Zandi PP, Belmonte PL, Willour VL, Goes FS, Badner JA, Simpson SG et al. Association study of Wnt signaling pathway genes in bipolar disorder. Arch Gen Psychiatry 2008; 65: 785–793.  [DOI](https://doi.org/10.1001/archpsyc.65.7.785) | [PMC free article](/articles/PMC3170992/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18606951/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=Association%20study%20of%20Wnt%20signaling%20pathway%20genes%20in%20bipolar%20disorder&author=PP%20Zandi&author=PL%20Belmonte&author=VL%20Willour&author=FS%20Goes&author=JA%20Badner&volume=65&publication_year=2008&pages=785-793&pmid=18606951&doi=10.1001/archpsyc.65.7.785&)

40. Burch LR, Zhou K, Donnelly LA, Doney AS, Brady J, Goddard C et al. A single nucleotide polymorphism on exon-4 of the gene encoding PPAR\{delta\} is associated with reduced height in adults and children. J Clin Endocrinol Metab 2009; 94: 2587–2593.  [DOI](https://doi.org/10.1210/jc.2009-0392) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19383774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Endocrinol%20Metab&title=A%20single%20nucleotide%20polymorphism%20on%20exon-4%20of%20the%20gene%20encoding%20PPAR%5C%7Bdelta%5C%7D%20is%20associated%20with%20reduced%20height%20in%20adults%20and%20children&author=LR%20Burch&author=K%20Zhou&author=LA%20Donnelly&author=AS%20Doney&author=J%20Brady&volume=94&publication_year=2009&pages=2587-2593&pmid=19383774&doi=10.1210/jc.2009-0392&)

41. Abdollahi A, Schwager C, Kleeff J, Esposito I, Domhan S, Peschke P et al. Transcriptional network governing the angiogenic switch in human pancreatic cancer. Proc Natl Acad Sci USA 2007; 104: 12890–12895.  [DOI](https://doi.org/10.1073/pnas.0705505104) | [PMC free article](/articles/PMC1931565/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17652168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Transcriptional%20network%20governing%20the%20angiogenic%20switch%20in%20human%20pancreatic%20cancer&author=A%20Abdollahi&author=C%20Schwager&author=J%20Kleeff&author=I%20Esposito&author=S%20Domhan&volume=104&publication_year=2007&pages=12890-12895&pmid=17652168&doi=10.1073/pnas.0705505104&)

42. Nordgard SH, Johansen FE, Alnaes GI, Bucher E, Syvanen AC, Naume B et al. Genome-wide analysis identifies 16q deletion associated with survival, molecular subtypes, mRNA expression, and germline haplotypes in breast cancer patients. Genes Chromosomes Cancer 2008; 47: 680–696.  [DOI](https://doi.org/10.1002/gcc.20569) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18398821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Chromosomes%20Cancer&title=Genome-wide%20analysis%20identifies%2016q%20deletion%20associated%20with%20survival,%20molecular%20subtypes,%20mRNA%20expression,%20and%20germline%20haplotypes%20in%20breast%20cancer%20patients&author=SH%20Nordgard&author=FE%20Johansen&author=GI%20Alnaes&author=E%20Bucher&author=AC%20Syvanen&volume=47&publication_year=2008&pages=680-696&pmid=18398821&doi=10.1002/gcc.20569&)

43. Li X, Tu L, Murphy PG, Kadono T, Steeber DA, Tedder TF. CHST1 and CHST2 sulfotransferase expression by vascular endothelial cells regulates shear-resistant leukocyte rolling via L-selectin. J Leukoc Biol 2001; 69: 565–574.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11310842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Leukoc%20Biol&title=CHST1%20and%20CHST2%20sulfotransferase%20expression%20by%20vascular%20endothelial%20cells%20regulates%20shear-resistant%20leukocyte%20rolling%20via%20L-selectin&author=X%20Li&author=L%20Tu&author=PG%20Murphy&author=T%20Kadono&author=DA%20Steeber&volume=69&publication_year=2001&pages=565-574&pmid=11310842&)

44. Nishimura M, Imai T, Morioka Y, Kuribayashi S, Kamataki T, Naito S. Effects of NO-1886 (Ibrolipim), a lipoprotein lipase-promoting agent, on gene induction of cytochrome P450s, carboxylesterases, and sulfotransferases in primary cultures of human hepatocytes. Drug Metab Pharmacokinet 2004; 19: 422–429.  [DOI](https://doi.org/10.2133/dmpk.19.422) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15681896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Effects%20of%20NO-1886%20(Ibrolipim),%20a%20lipoprotein%20lipase-promoting%20agent,%20on%20gene%20induction%20of%20cytochrome%20P450s,%20carboxylesterases,%20and%20sulfotransferases%20in%20primary%20cultures%20of%20human%20hepatocytes&author=M%20Nishimura&author=T%20Imai&author=Y%20Morioka&author=S%20Kuribayashi&author=T%20Kamataki&volume=19&publication_year=2004&pages=422-429&pmid=15681896&doi=10.2133/dmpk.19.422&)

45. Iida A, Saito S, Sekine A, Mishima C, Kitamura Y, Kondo K et al. Catalog of 77 single-nucleotide polymorphisms (SNPs) in the carbohydrate sulfotransferase 1 (CHST1) and carbohydrate sulfotransferase 3 (CHST3) genes. J Hum Genet 2002; 47: 14–19.  [DOI](https://doi.org/10.1007/s10038-002-8650-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11829137/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=Catalog%20of%2077%20single-nucleotide%20polymorphisms%20(SNPs)%20in%20the%20carbohydrate%20sulfotransferase%201%20(CHST1)%20and%20carbohydrate%20sulfotransferase%203%20(CHST3)%20genes&author=A%20Iida&author=S%20Saito&author=A%20Sekine&author=C%20Mishima&author=Y%20Kitamura&volume=47&publication_year=2002&pages=14-19&pmid=11829137&doi=10.1007/s10038-002-8650-9&)

46. Xiangrong L, Johnk C, Hartmann D, Schestag F, Kramer W, Gieselmann V. Enzymatic properties, tissue-specific expression, and lysosomal location of two highly homologous rat SULT1C2 sulfotransferases. Biochem Biophys Res Commun 2000; 272: 242–250.  [DOI](https://doi.org/10.1006/bbrc.2000.2744) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10872834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Enzymatic%20properties,%20tissue-specific%20expression,%20and%20lysosomal%20location%20of%20two%20highly%20homologous%20rat%20SULT1C2%20sulfotransferases&author=L%20Xiangrong&author=C%20Johnk&author=D%20Hartmann&author=F%20Schestag&author=W%20Kramer&volume=272&publication_year=2000&pages=242-250&pmid=10872834&doi=10.1006/bbrc.2000.2744&)

47. Tamura HO, Taniguchi K, Hayashi E, Hiyoshi Y, Nagai F. Expression profiling of sulfotransferases in human cell lines derived from extra-hepatic tissues. Biol Pharm Bull 2001; 24: 1258–1262.  [DOI](https://doi.org/10.1248/bpb.24.1258) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11725959/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Pharm%20Bull&title=Expression%20profiling%20of%20sulfotransferases%20in%20human%20cell%20lines%20derived%20from%20extra-hepatic%20tissues&author=HO%20Tamura&author=K%20Taniguchi&author=E%20Hayashi&author=Y%20Hiyoshi&author=F%20Nagai&volume=24&publication_year=2001&pages=1258-1262&pmid=11725959&doi=10.1248/bpb.24.1258&)

48. Hildebrandt MA, Carrington DP, Thomae BA, Eckloff BW, Schaid DJ, Yee VC et al. Genetic diversity and function in the human cytosolic sulfotransferases. Pharmacogenomics J 2007; 7: 133–143.  [DOI](https://doi.org/10.1038/sj.tpj.6500404) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16801938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genetic%20diversity%20and%20function%20in%20the%20human%20cytosolic%20sulfotransferases&author=MA%20Hildebrandt&author=DP%20Carrington&author=BA%20Thomae&author=BW%20Eckloff&author=DJ%20Schaid&volume=7&publication_year=2007&pages=133-143&pmid=16801938&doi=10.1038/sj.tpj.6500404&)

49. Aust S, Obrist P, Klimpfinger M, Tucek G, Jager W, Thalhammer T. Altered expression of the hormone- and xenobiotic-metabolizing sulfotransferase enzymes 1A2 and 1C1 in malignant breast tissue. Int J Oncol 2005; 26: 1079–1085.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15754005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Oncol&title=Altered%20expression%20of%20the%20hormone-%20and%20xenobiotic-metabolizing%20sulfotransferase%20enzymes%201A2%20and%201C1%20in%20malignant%20breast%20tissue&author=S%20Aust&author=P%20Obrist&author=M%20Klimpfinger&author=G%20Tucek&author=W%20Jager&volume=26&publication_year=2005&pages=1079-1085&pmid=15754005&)

50. Stanley EL, Hume R, Coughtrie MW. Expression profiling of human fetal cytosolic sulfotransferases involved in steroid and thyroid hormone metabolism and in detoxification. Mol Cell Endocrinol 2005; 240: 32–42.  [DOI](https://doi.org/10.1016/j.mce.2005.06.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16024168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Endocrinol&title=Expression%20profiling%20of%20human%20fetal%20cytosolic%20sulfotransferases%20involved%20in%20steroid%20and%20thyroid%20hormone%20metabolism%20and%20in%20detoxification&author=EL%20Stanley&author=R%20Hume&author=MW%20Coughtrie&volume=240&publication_year=2005&pages=32-42&pmid=16024168&doi=10.1016/j.mce.2005.06.003&)

51. Casari G, De Fusco M, Ciarmatori S, Zeviani M, Mora M, Fernandez P et al. Spastic paraplegia and OXPHOS impairment caused by mutations in paraplegin, a nuclear-encoded mitochondrial metalloprotease. Cell 1998; 93: 973–983.  [DOI](https://doi.org/10.1016/s0092-8674(00)81203-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9635427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Spastic%20paraplegia%20and%20OXPHOS%20impairment%20caused%20by%20mutations%20in%20paraplegin,%20a%20nuclear-encoded%20mitochondrial%20metalloprotease&author=G%20Casari&author=M%20De%20Fusco&author=S%20Ciarmatori&author=M%20Zeviani&author=M%20Mora&volume=93&publication_year=1998&pages=973-983&pmid=9635427&doi=10.1016/s0092-8674(00)81203-9&)

52. Goytain A, Hines RM, El-Husseini A, Quamme GA. NIPA1(SPG6), the basis for autosomal dominant form of hereditary spastic paraplegia, encodes a functional Mg2+ transporter. J Biol Chem 2007; 282: 8060–8068.  [DOI](https://doi.org/10.1074/jbc.M610314200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17166836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=NIPA1(SPG6),%20the%20basis%20for%20autosomal%20dominant%20form%20of%20hereditary%20spastic%20paraplegia,%20encodes%20a%20functional%20Mg2+%20transporter&author=A%20Goytain&author=RM%20Hines&author=A%20El-Husseini&author=GA%20Quamme&volume=282&publication_year=2007&pages=8060-8068&pmid=17166836&doi=10.1074/jbc.M610314200&)

53. Royer I, Monsarrat B, Sonnier M, Wright M, Cresteil T. Metabolism of docetaxel by human cytochromes P450: interactions with paclitaxel and other antineoplastic drugs. Cancer Res 1996; 56: 58–65.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8548776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Metabolism%20of%20docetaxel%20by%20human%20cytochromes%20P450:%20interactions%20with%20paclitaxel%20and%20other%20antineoplastic%20drugs&author=I%20Royer&author=B%20Monsarrat&author=M%20Sonnier&author=M%20Wright&author=T%20Cresteil&volume=56&publication_year=1996&pages=58-65&pmid=8548776&)

54. Teo SK, Sabourin PJ, O’Brien K, Kook KA, Thomas SD. Metabolism of thalidomide in human microsomes, cloned human cytochrome P-450 isozymes, and Hansen’s disease patients. J Biochem Mol Toxicol 2000; 14: 140–147.  [DOI](https://doi.org/10.1002/(sici)1099-0461(2000)14:3<140::aid-jbt3>3.0.co;2-p) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10711629/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biochem%20Mol%20Toxicol&title=Metabolism%20of%20thalidomide%20in%20human%20microsomes,%20cloned%20human%20cytochrome%20P-450%20isozymes,%20and%20Hansen%E2%80%99s%20disease%20patients&author=SK%20Teo&author=PJ%20Sabourin&author=K%20O%E2%80%99Brien&author=KA%20Kook&author=SD%20Thomas&volume=14&publication_year=2000&pages=140-147&pmid=10711629&doi=10.1002/(sici)1099-0461(2000)14:3<140::aid-jbt3>3.0.co;2-p&)

55. McKeown-Eyssen G, Baines C, Cole DE, Riley N, Tyndale RF, Marshall L et al. Case-control study of genotypes in multiple chemical sensitivity: CYP2D6, NAT1, NAT2, PON1, PON2 and MTHFR. Int J Epidemiol 2004; 33: 971–978.  [DOI](https://doi.org/10.1093/ije/dyh251) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15256524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Epidemiol&title=Case-control%20study%20of%20genotypes%20in%20multiple%20chemical%20sensitivity:%20CYP2D6,%20NAT1,%20NAT2,%20PON1,%20PON2%20and%20MTHFR&author=G%20McKeown-Eyssen&author=C%20Baines&author=DE%20Cole&author=N%20Riley&author=RF%20Tyndale&volume=33&publication_year=2004&pages=971-978&pmid=15256524&doi=10.1093/ije/dyh251&)

56. Sabbagh N, Delaporte E, Marez D, Lo-Guidice JM, Piette F, Broly F. NAT2 genotyping and efficacy of sulfasalazine in patients with chronic discoid lupus erythematosus. Pharmacogenetics 1997; 7: 131–135.  [DOI](https://doi.org/10.1097/00008571-199704000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9170150/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=NAT2%20genotyping%20and%20efficacy%20of%20sulfasalazine%20in%20patients%20with%20chronic%20discoid%20lupus%20erythematosus&author=N%20Sabbagh&author=E%20Delaporte&author=D%20Marez&author=JM%20Lo-Guidice&author=F%20Piette&volume=7&publication_year=1997&pages=131-135&pmid=9170150&doi=10.1097/00008571-199704000-00006&)

57. Parkin DP, Vandenplas S, Botha FJ, Vandenplas ML, Seifart HI, van Helden PD et al. Trimodality of isoniazid elimination: phenotype and genotype in patients with tuberculosis. Am J Respir Crit Care Med 1997; 155: 1717–1722.  [DOI](https://doi.org/10.1164/ajrccm.155.5.9154882) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9154882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Trimodality%20of%20isoniazid%20elimination:%20phenotype%20and%20genotype%20in%20patients%20with%20tuberculosis&author=DP%20Parkin&author=S%20Vandenplas&author=FJ%20Botha&author=ML%20Vandenplas&author=HI%20Seifart&volume=155&publication_year=1997&pages=1717-1722&pmid=9154882&doi=10.1164/ajrccm.155.5.9154882&)

58. Belinsky MG, Chen ZS, Shchaveleva I, Zeng H, Kruh GD. Characterization of the drug resistance and transport properties of multidrug resistance protein 6 (MRP6, ABCC6). Cancer Res 2002; 62: 6172–6177.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12414644/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Characterization%20of%20the%20drug%20resistance%20and%20transport%20properties%20of%20multidrug%20resistance%20protein%206%20(MRP6,%20ABCC6)&author=MG%20Belinsky&author=ZS%20Chen&author=I%20Shchaveleva&author=H%20Zeng&author=GD%20Kruh&volume=62&publication_year=2002&pages=6172-6177&pmid=12414644&)

59. Safaei R, Howell SB. Copper transporters regulate the cellular pharmacology and sensitivity to Pt drugs. Crit Rev Oncol Hematol 2005; 53: 13–23.  [DOI](https://doi.org/10.1016/j.critrevonc.2004.09.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15607932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit%20Rev%20Oncol%20Hematol&title=Copper%20transporters%20regulate%20the%20cellular%20pharmacology%20and%20sensitivity%20to%20Pt%20drugs&author=R%20Safaei&author=SB%20Howell&volume=53&publication_year=2005&pages=13-23&pmid=15607932&doi=10.1016/j.critrevonc.2004.09.007&)

60. Trauner M, Boyer JL. Bile salt transporters: molecular characterization, function, and regulation. Physiol Rev 2003; 83: 633–671.  [DOI](https://doi.org/10.1152/physrev.00027.2002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12663868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiol%20Rev&title=Bile%20salt%20transporters:%20molecular%20characterization,%20function,%20and%20regulation&author=M%20Trauner&author=JL%20Boyer&volume=83&publication_year=2003&pages=633-671&pmid=12663868&doi=10.1152/physrev.00027.2002&)

61. Maitland ML, Ratain MJ, Cox NJ. Interpreting P values in pharmaco-genetic studies: a call for process and perspective. J Clin Oncol 2007; 25: 4513–4515.  [DOI](https://doi.org/10.1200/JCO.2007.12.7803) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17925544/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Interpreting%20P%20values%20in%20pharmaco-genetic%20studies:%20a%20call%20for%20process%20and%20perspective&author=ML%20Maitland&author=MJ%20Ratain&author=NJ%20Cox&volume=25&publication_year=2007&pages=4513-4515&pmid=17925544&doi=10.1200/JCO.2007.12.7803&)
