# Impact of Promoter Polymorphisms on the Transcriptional Regulation of the Organic Cation Transporter OCT1 (SLC22A1)

## Metadata
**Authors:** Kristin Bokelmann, Jürgen Brockmöller, Mladen V Tzvetkov
**Journal:** Journal of Personalized Medicine
**Date:** 2018 Dec 11
**DOI:** [10.3390/jpm8040042](https://doi.org/10.3390/jpm8040042)
**PMID:** 30544975
**PMCID:** PMC6313513
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6313513/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6313513/pdf/jpm-08-00042.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6313513/pdf/jpm-08-00042.pdf)

## Abstract

The organic cation transporter 1 (OCT1, SLC22A1) is strongly expressed in the human liver and facilitates the hepatic uptake of drugs such as morphine, metformin, tropisetron, sumatriptan and fenoterol and of endogenous substances such as thiamine. OCT1 expression is inter-individually highly variable. Here, we analyzed SNPs in the OCT1 promoter concerning their potential contribution to the variability in OCT1 expression. Using electrophoretic mobility shift and luciferase reporter gene assays in HepG2, Hep3B, and Huh7 cell lines, we identified the SNPs −1795G>A (rs6935207) and −201C>G (rs58812592) as having effects on transcription factor binding and/or promoter activity. The A-allele of the −1795G>A SNP showed allele-specific binding of the transcription factor NF-Y leading to 2.5-fold increased enhancer activity of the artificial SV40 promoter. However, the −1795G>A SNP showed no significant effects on the native OCT1 promoter activity. Furthermore, the −1795G>A SNP was not associated with the pharmacokinetics of metformin, fenoterol, sumatriptan and proguanil in healthy individuals or tropisetron efficacy in patients undergoing chemotherapy. Allele-dependent differences in USF1/2 binding and nearly total loss in OCT1 promoter activity were detected for the G-allele of −201C>G, but the SNP is apparently very rare. In conclusion, common OCT1 promoter SNPs have only minor effects on OCT1 expression.

Keywords: OCT1, SNP, promoter, EMSA, luciferase reporter gene assay, expression, NF-Y

## 1. Introduction

The organic cation transporter OCT1 (SLC22A1) is one of the most strongly expressed drug transporters in the human liver [[1](#B1-jpm-08-00042),[2](#B2-jpm-08-00042),[3](#B3-jpm-08-00042)]. OCT1 plays a role in the hepatocellular uptake of organic cationic molecules from the blood into the liver. Drugs such as metformin, tramadol, morphine, sumatriptan, tropisetron, fenoterol and proguanil are substrates of OCT1 [[4](#B4-jpm-08-00042),[5](#B5-jpm-08-00042),[6](#B6-jpm-08-00042),[7](#B7-jpm-08-00042),[8](#B8-jpm-08-00042),[9](#B9-jpm-08-00042),[10](#B10-jpm-08-00042),[11](#B11-jpm-08-00042)].

*OCT1* is genetically highly polymorphic in humans. Resequencing analyses of 2171 unrelated individuals from 67 worldwide populations report 29 variants that cause amino acid substitutions [[12](#B12-jpm-08-00042)]. The variants could be grouped into 30 haplotypes constituting 16 major alleles, which substantially affect OCT1 activity. Due to the presence of these coding polymorphisms and depending on substrate-specific differences of allele *OCT1*2*, between 3% and 9% of Caucasians have zero or strongly reduced OCT1 activity. This results in decreased hepatic metabolism by affecting their uptake into hepatocytes. These effects have been demonstrated for drugs such as tramadol, debrisoquine, morphine, sumatriptan and tropisetron, and decreased efficacy of metformin in these individuals has been suggested [[4](#B4-jpm-08-00042),[5](#B5-jpm-08-00042),[6](#B6-jpm-08-00042),[7](#B7-jpm-08-00042),[8](#B8-jpm-08-00042),[13](#B13-jpm-08-00042),[14](#B14-jpm-08-00042)]. For numerous other drugs such as lamivudine, bendamustine and debrisoquine, the effects of *OCT1* polymorphisms have been shown *in vitro* or ex *vivo* [[13](#B13-jpm-08-00042),[15](#B15-jpm-08-00042),[16](#B16-jpm-08-00042)]. This is now acknowledged as an important cause of variability in the pharmacokinetics and efficacy of drugs [[17](#B17-jpm-08-00042),[18](#B18-jpm-08-00042),[19](#B19-jpm-08-00042)]. On the other hand, the tyrosine kinase inhibitors imatinib and sorafenib have been suggested to be affected by *OCT1* polymorphisms, but the data is controversial with some detailed studies that were unable to confirm these drugs as substrates of OCT1 [[20](#B20-jpm-08-00042),[21](#B21-jpm-08-00042),[22](#B22-jpm-08-00042),[23](#B23-jpm-08-00042)].

Beside the coding variations that may strongly affect OCT1 activity, the expression of OCT1 also varies widely among individuals. Nies et al. measured 113-fold variability in *OCT1* mRNA and a corresponding 83-fold variability in OCT1 protein levels [[1](#B1-jpm-08-00042)]. The variability in mRNA expression has been confirmed in further studies [[24](#B24-jpm-08-00042),[25](#B25-jpm-08-00042)]. Already identified reasons for the variable OCT1 expression, which may cause considerable inter-individual variability of hepatic OCT1 activity, include cholestasis and epigenetic variations [[1](#B1-jpm-08-00042),[26](#B26-jpm-08-00042)].

However, transporter expression, and as a consequence drug pharmacokinetics, may also be affected by single nucleotide polymorphisms (SNPs) in their promoter regions [[27](#B27-jpm-08-00042),[28](#B28-jpm-08-00042),[29](#B29-jpm-08-00042),[30](#B30-jpm-08-00042),[31](#B31-jpm-08-00042),[32](#B32-jpm-08-00042)]. Thus, we hypothesize that also *OCT1* promoter polymorphisms, especially in or next to *cis*-regulatory elements, may affect the binding of regulatory factors and *OCT1* gene expression.

The expression of OCT1 is controlled by three transcription factors: USF1/2, HNF4α, and HNF1. The upstream stimulatory factors USF1/2 are binding to an E-box at −200 to −195 and the hepatocyte nuclear factor HNF4α is binding to the two DR2 elements at −1642 to −1604 bp from the transcriptional start site (TSS) [[33](#B33-jpm-08-00042),[34](#B34-jpm-08-00042)]. Also, the hepatocyte nuclear factor HNF1 has been demonstrated to regulate OCT1 expression by binding to an evolutionary conserved enhancer element in the intron 1 of the *OCT1* gene [[24](#B24-jpm-08-00042)].

In the present study, we characterized the functional effects of polymorphisms in the 5 kb upstream region of the *OCT1* gene in order to identify polymorphisms that may contribute to the high variability of OCT1 expression. Therefore, we used electrophoretic mobility shift assays (EMSA) and luciferase reporter gene assays to evaluate the effects of the polymorphisms on promoter activity. We further analyzed those SNPs that showed effects on promoter activity in the *in vitro* assays for associations with pharmacokinetics of metfomin, fenoterol, sumatriptan and proguanil in healthy volunteers and tropisetron efficacy in patients. To the best of our knowledge, these are the first systemic analyses on the effects of single nucleotide polymorphisms on *OCT1* promoter activity.

## 2. Materials and Methods

### 2.1. Cell Culture and Transfection

HepG2 cells (DSMZ-German Collection of Microorganisms and Cell Cultures, Braunschweig, Germany) were cultured in RPMI 1640 GlutaMAX™-I supplemented with 10% fetal bovine serum, 100 U/mL penicillin, and 100 μg/mL streptomycin. Hep-3B cells (DSMZ, Braunschweig, Germany) and Huh7 cells (JCRB Cell Bank, Tokyo, Japan) were cultured in DMEM supplemented with 10% fetal bovine serum, 100 U/mL penicillin, and 100 μg/mL streptomycin.

Cells were detached with TrypLE Express W/Phenol red (Life Technologies, Darmstadt, Germany). Media and additives were obtained from Gibco (Life Technologies). Cells were cultured under standard conditions at 37 °C in a humidified atmosphere supplemented with 5% CO_2_.

For transfection experiments, 1.5 × 10^5^ Huh7 and Hep-3B cells, respectively, and 2 × 10^5^ HepG2 cells were plated per well of a 12-well plate (Nunc, Langenselbold, Germany) and grown for 24 h to reach approximately 80% confluence. Lipofectamine 2000 (Invitrogen, Karlsruhe, Germany) was used to transfect the cells. Per well, 4 µL Lipofectamine, 1.6 µg plasmid DNA and 3 ng pRL-CMV Renilla luciferase control vector (Promega, Mannheim, Germany) were applied. The cells were harvested and luciferase activity was measured as described previously [[24](#B24-jpm-08-00042)].

### 2.2. Electrophoretic Mobility Shift Assays (EMSA)

Nuclear protein preparation of HepG2, Hep-3B and Huh7 cells was performed according to a protocol for the CelLytic^®^ NuCLEAR^®^ Extraction Kit from Sigma-Aldrich (Deisenhofen, Germany), but with modifications and self-made buffers. Media of 1 × 10^7^–1 × 10^8^ cells was discarded and cells were detached using Tryple Express W/Phenol red. Trypsination was stopped with culture media and cells were transferred into a 50 mL tube and centrifuged at 600× *g* for 3 min at 4 °C. Media was discarded, cells were washed with 10 mL 1× PBS including 1 mM sodium-orthovanadat and centrifuged again. Supernatant was discarded, cells were washed with 1 mL 1× PBS including 1 mM sodium-orthovanadat, transferred into a 2 mL tube and centrifuged again. Then, the volume of the cell pellet was determined and mixed with 5× volume of 1× Lysis buffer (10 mM HEPES, pH 7.9, with 1.5 mM MgCl_2_, 10 mM KCl, 0.5 mM DTT, 1 mM sodium-orthovanadat, 2.5 mM PMSF). Cells were incubated on ice for 30 min and optionally disrupted with a glass tissue homogenizer until the cells were lysed. Then, NP-40 was added to a final concentration of 1% and cells were vortexed vigorously for 10 s. Subsequently, the tubes were centrifuged at 11,000× *g* for 2 min at 4 °C and the supernatant was discarded. The pellet was then resuspended in 2/3 of its volume with the extraction buffer (20 mM HEPES, pH 7.9, with 1.5 mM MgCl_2_, 420 mM NaCl, 0.2 mM EDTA, and 25% (*v*/*v*) Glycerol, 1% NP-40, 0.5 mM DTT, 1 mM sodium-orthovanadat, 2.5 mM PMSF). The tube was mounted on a vortex mixer and agitated at 1800 rpm for 30 min and then centrifuged at 17,000× *g* for 10 min at 4 °C. Optionally, the extraction step could be repeated with 1/3 of the former volume and agitated for 3 h. The supernatant contains the nuclear proteins and was stored at −80 °C. The concentrations of the proteins were assessed by the bicinchoninic acid (Sigma-Aldrich, Deisenhofen, Germany) protein assay with BSA as standard.

Oligonucleotides used for EMSA are shown in [Table 1](#jpm-08-00042-t001). EMSA was performed as described previously [[24](#B24-jpm-08-00042)]. For supershift assays, a 4% polyacrylamide gel was used and was run for 2 h. Competition assays were performed with specific non-labeled double-stranded oligonucleotide probes in 2- to 15-fold excesses of the labeled probe. For supershift assays, nuclear extracts were incubated with 2 µg antibody (NF-Ya (sc-10779) and IgG (sc-2027), all from Santa Cruz Biotechnology, Heidelberg, Germany) for 1 h on ice, prior to the addition of radiolabeled probes.

### Table 1.

| Name | Direction | Sequence |
| --- | --- | --- |
| EMSA Probes * |   |   |
| rs41267793_T_f | Forward | 5′-ACTTGGACAGCAAACTGATTTCAAACCACTCa-3′ |
| rs41267793_T_r | Reverse | 5′-gatctGAGTGGTTTGAAATCAGTTTGCTGTCCAAGTgtac-3′ |
| rs41267793_C_f | Forward | 5′-ACTTGGACAGCAAACCGATTTCAAACCACTCa-3′ |
| rs41267793_C_r | Reverse | 5′-gatctGAGTGGTTTGAAATCGGTTTGCTGTCCAAGTgtac-3′ |
| rs6899549_T_f | Forward | 5′-AGGGTAAAAGATTATTTCTACTTGGTTGCCTa-3′ |
| rs6899549_T_r | Reverse | 5′-gatctAGGCAACCAAGTAGAAATAATCTTTTACCCTgtac-3′ |
| rs6899549_C_f | Forward | 5′-AGGGTAAAAGATTATCTCTACTTGGTTGCCTa-3′ |
| rs6899549_C_r | Reverse | 5′-gatctAGGCAACCAAGTAGAGATAATCTTTTACCCTgtac-3′ |
| rs58812592_C_f | Forward | 5′-TTGATCAGATGGCCACGTGCATTCTTCCTTTa-3′ |
| rs58812592_C_r | Reverse | 5′-gatctAAAGGAAGAATGCACGTGGCCATCTGATCAAgtac-3′ |
| rs58812592_G_f | Forward | 5′-TTGATCAGATGGGCACGTGCATTCTTCCTTTa-3′ |
| rs58812592_G_r | Reverse | 5′-gatctAAAGGAAGAATGCACGTGCCCATCTGATCAAgtac-3′ |
| rs60541448_G_f | Forward | 5′-CACTGACTCGCTCCCGGGCAAAGCAAACGATa-3′ |
| rs60541448_G_r | Reverse | 5′-gatctATCGTTTGCTTTGCCCGGGAGCGAGTCAGTGgtac-3′ |
| rs60541448_A_f | Forward | 5′-CACTGACTCGCTCCCAGGCAAAGCAAACGATa-3′ |
| rs60541448_A_r | Reverse | 5′-gatctATCGTTTGCTTTGCCTGGGAGCGAGTCAGTGgtac-3′ |
| rs73598465_T_f | Forward | 5′-TATCACAGAACTAATTAGCCGAATACAGTATa-3′ |
| rs73598465_T_r | Reverse | 5′-gatctATACTGTATTCGGCTAATTAGTTCTGTGATAgtac-3′ |
| rs73598465_A_f | Forward | 5′-TATCACAGAACTAATAAGCCGAATACAGTATa-3′ |
| rs73598465_A_r | Reverse | 5′-gatctATACTGTATTCGGCTTATTAGTTCTGTGATAgtac-3′ |
| rs9457840_T_f | Forward | 5′-GGAGCCCTATTGACCCTGGAGTCCTGTTGATCTCTTGTCCTTCa-3′ |
| rs9457840_T_r | Reverse | 5′-gatctGAAGGACAAGAGATCAACAGGACTCCAGGGTCAATAGGGCTCCgtac-3′ |
| rs9457840_C_f | Forward | 5′-GGAGCCCTATTGACCCTGGAGTCCCGTTGATCTCTTGTCCTTCa-3′ |
| rs9457840_C_r | Reverse | 5′-gatctGAAGGACAAGAGATCAACGGGACTCCAGGGTCAATAGGGCTCCgtac-3′ |
| rs6935207_G_f | Forward | 5′-CATGCCAAATGGCCAGTTGAATTCATGGAGGa-3′ |
| rs6935207_G_r | Reverse | 5′-gatctCCTCCATGAATTCAACTGGCCATTTGGCATGgtac-3′ |
| rs6935207_A_f | Forward | 5′-CATGCCAAATGGCCAATTGAATTCATGGAGGa-3′ |
| rs6935207_A_r | Reverse | 5′-gatctCCTCCATGAATTCAATTGGCCATTTGGCATGgtac-3′ |
| rs6935207_mut_f | Forward | 5′-CATGCCAAATGGCTGATTGAATTCATGGAGGa-3′ |
| rs6935207_mut_r | Reverse | 5′-gatctCCTCCATGAATTCAATCAGCCATTTGGCATGgtac-3′ |
| rs4709399_T_f | Forward | 5′-AAACCTCTGTGGTCATGGTGCCTTTGCATGAa-3′ |
| rs4709399_T_r | Reverse | 5′-gatctTCATGCAAAGGCACCATGACCACAGAGGTTTgtac-3′ |
| rs4709399_C_f | Forward | 5′-AAACCTCTGTGGTCACGGTGCCTTTGCATGAa-3′ |
| rs4709399_C_r | Reverse | 5′-gatctTCATGCAAAGGCACCGTGACCACAGAGGTTTgtac-3′ |
| rs456598_G_f | Forward | 5′-GTAAGGCACTTTTTGGATGGTAGGACTGGTTa-3′ |
| rs456598_G_r | Reverse | 5′-gatctAACCAGTCCTACCATCCAAAAAGTGCCTTACgtac-3′ |
| rs456598_A_f | Forward | 5′-GTAAGGCACTTTTTGAATGGTAGGACTGGTTa-3′ |
| rs456598_A_r | Reverse | 5′-gatctAACCAGTCCTACCATTCAAAAAGTGCCTTACgtac-3′ |
| rs9457839_G_f | Forward | 5′-CCACATGCACTCCTAGGTCTGAAAATGGGGGa-3′ |
| rs9457839_G_r | Reverse | 5′-gatctCCCCCATTTTCAGACCTAGGAGTGCATGTGGgtac-3′ |
| rs9457839_A_f | Forward | 5′-CCACATGCACTCCTAAGTCTGAAAATGGGGGa-3′ |
| rs9457839_A_r | Reverse | 5′-gatctCCCCCATTTTCAGACTTAGGAGTGCATGTGGgtac-3′ |
| GS_NF-Y_f | Forward | 5′-GATCAGACCGTACGTGATTGGTTAATCTCTT-3′ |
| GS_NF-Y_r | Reverse | 5′-GATCAAGAGATTAACCAATCACGTACGGTCT-3′ |
| CRE2_f | Forward | 5′-actggTCCTTGGCTGACGTCAGAGAGAGAG-3′ |
| CRE_r | Reverse | 5′-taCTCTCTCTCTGACGTCAGCCAAGGAgg-3′ |
| Primers used for site-directed mutagenesis |   |   |
| 588_mut_f | Forward | 5′-GATTTGATCAGATGGGCACGTGCATTCTTCC-3′ |
| 588_mut_r | Reverse | 5′-GGAAGAATGCACGTGCCCATCTGATCAAATC-3′ |
| 840_mut_f | Forward | 5′-TTGACCCTGGAGTCCCGTTGATCTCTTGTCC-3′ |
| 840_mut_r | Reverse | 5′-GGACAAGAGATCAACGGGACTCCAGGGTCAA-3′ |
| Primers used for Sanger sequencing |   |   |
| Seq_840_f | Forward | 5′-TGCAACCAGTTTGCACAGAGAG-3′ |
| Seq_588_f | Forward | 5′-GCCTCATACCATCACATCTAGA-3′ |
| Seq_pGL3prom_f | Forward | 5′-GAATCGATAGTACTAACATA-3′ |
| Seq_SV40_r | Reverse | 5′-AAGCCTCCTCACTACTTCTG-3′ |
| Primers used for generation of the reporter gene construct |   |   |
| OCT1-1853bp_promoter_f | Forward | 5′-ATCGCGGTACCTTTTTAAGAAGTCCTTTTAAGT-3′ |
| OCT1-1853bp_promoter_r | Reverse | 5′-AAGAAGGGAAGGACAAGAGATCAAC-3′ |

Table 1 Caption: Oligonucleotides used in this study.

### 2.3. Generation of the Luciferase Reporter Plasmids

*OCT1* promoter SNPs were cloned into the pGL3-promoter vector. Therefore, 20 pmol of the appropriate *OCT1* promoter SNP oligonucleotides ([Table 1](#jpm-08-00042-t001): “EMSA probes”) was annealed. The resulting dsDNA probe carries a *Kpn*I-overhang at the 5′-end and a *Bgl*II-overhang at the 3′-end. The pGL3-promoter vector was cut with *Kpn*I and *Bgl*II, respectively, and ligated with the different dsDNA *OCT1* promoter probes.

Generation of the pGL3b::OCT1promoter is described in [[24](#B24-jpm-08-00042)]. To also analyze the effect of −1795G>A on the promoter activity, the *OCT1* promoter was extended to −1853 from the translational start site of *OCT1*. Therefore, the adjacent downstream part of the promoter was amplified from gDNA of lymphoblastoid cell lines of homozygote −1795G and −1795A carriers, respectively, with KOD-Polymerase, Q-Solution (Qiagen, Hilden, Germany) and the OCT1-1853bp_promoter primers ([Table 1](#jpm-08-00042-t001)) under the following reaction conditions: 94 °C for 5 min, followed by 35 cycles of 94 °C for 15 s, 57 °C for 30 s, 72 °C for 40 s, and a final elongation of 72 °C for 10 min. The PCR product and pGL3b::OCT1 promoter were cut with *Kpn*I and *Sac*I and afterwards ligated to each other.

Site-directed mutagenesis was used to mutate the wild-types of the SNPs −201C>G and −1620T>C to its variant. The amplification was carried out using the 588mut and 840mut forward and reverse primers, respectively ([Table 1](#jpm-08-00042-t001)) under the following reaction conditions: 95 °C for 3 min, followed by 19 cycles of 95 °C for 30 s, 55 °C for 30 s, and 72 °C for 4 min. Then, 2 µL of the restriction enzyme *Dpn*I was added to the product and incubated for 2 h at 37 °C. Afterwards, it was cloned into *E. coli* TOP10.

The correct sequences of all plasmids were monitored by restriction analyses and sequencing of the inserts and the flanking regions.

## 3. Results

### 3.1. Screening of SNPs in the OCT1 Promoter for Effects on Transcription Factor Binding and Promoter Activity

We analyzed all ten SNPs in the 5 kb upstream region of *OCT1*, which were listed in the NCBI dbSNP database at the time point of the start of this study ([Figure 1](#jpm-08-00042-f001)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/86a1/6313513/1c895860106e/jpm-08-00042-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6313513_jpm-08-00042-g001.jpg)

Position of the analyzed OCT1 polymorphisms. (A) The coordinates are given in base pairs relative to the translational start codon ATG (A = +1) of the OCT1 gene. The transcriptional start site (TSS) is located at −105. The OCT1 promoter contains two transcriptionally functional cis-regulatory elements shown here as gray boxes: a doubled DR2-element at −1642 to −1604 and an E-box at −200 to −195. Binding of the transcription factors HNF4α (blue oval) and USF1/2 (orange oval) to its cis-elements regulate OCT1 expression. (B) Three SNPs are in special focus: −1795 G>A (rs6935207) shows functional effects and is located within a potential CCAAT-box; −1620 C>G (rs9457840) and −201 T>C (rs58812592) are located in close proximity to the DR2-elements and the E-box, respectively. SNPs and their positions are shown in red; cis-regulatory elements are framed.

First, we performed EMSAs with nuclear proteins of HepG2 and Hep3B cells to screen for transcription factor binding in SNPs’ vicinity ([Figure 2](#jpm-08-00042-f002)). We observed nuclear protein binding for the SNPs −201C>G (rs58812592) and −1795G>A (rs6935207). Thereof, the −1795G>A signal was highly allele-specific. A clear and strong retention signal was detected for the −1795A, but not for the −1795G-allele.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/86a1/6313513/2e627ff3afff/jpm-08-00042-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6313513_jpm-08-00042-g002.jpg)

Electrophoretic mobility shift assays with probes carrying OCT1 promoter SNPs indicate nuclear protein binding for the A-allele of the rs6935207 locus and for both alleles of the rs58812592 locus. The 32P-labeled probes, containing ~30 bp around each SNP locus (Table 1), were incubated with nuclear extracts from HepG2 and Hep3B cells and separated on 5% polyacrylamide gels. CRE (=cAMP response element) was used as a positive control for the assay.

Second, we screened all ten SNPs for their effect on the constitutive SV40 promoter activity ([Figure 3](#jpm-08-00042-f003)). Therefore, we used the same dsDNA probes as before for EMSA, cloned them into the pGL3-promoter vector and performed luciferase reporter gene assays in HepG2, Hep3B and Huh7 cells. Allele-specific differences in SV40 promoter activity were measured only for −1795G>A. The −1795A-allele showed a 2.5-fold (*p* < 0.001) increase in promoter activity in Hep3B, 1.6-fold (*p* < 0.001) increase in HepG2 cells and 1.4-fold (not significant) increase in Huh7 cells, respectively, compared with −1795G. Therefore, we focused our further analysis on −1795 G>A, as well as on −201 C>G (rs58812592) and −1620 T>C (rs9457840) due to their proximity to *cis*-regulatory elements in the *OCT1* promoter.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/86a1/6313513/a4ed9588f1df/jpm-08-00042-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6313513_jpm-08-00042-g003.jpg)

Analysis of the enhancer activity of the OCT1 promoter SNPs (WT and variant) on a constitutive Simian virus 40 (SV40) promoter. Annealed oligonucleotides harboring ~30 bp around each SNP locus (Table 1) were cloned in front of a constitutive SV40 promoter in the pGL3-promoter vector. Luciferase reporter gene assays were performed in the hepatocellular carcinoma cell lines HepG2, Hep3B and Huh7. The data represent the means and standard deviations of at least two independent experiments conducted in duplicate.

### 3.2. Specific Binding of NF-Y to the A-Allele of the -1795G>A (rs6935207) Polymorphism

Within the probe for the −1795G>A SNP, we detected a potential CCAAT box sequence that was located at −1798bp to −1794 bp and so the −1795A-allele was essential for the CCAAT box consensus ([Figure 4](#jpm-08-00042-f004)A). Cold competition and supershift EMSAs demonstrate the binding of the nuclear transcription factor NF-Y to the sequence containing the −1795A-allele ([Figure 4](#jpm-08-00042-f004)B,C).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/86a1/6313513/4f37d2e2ebf9/jpm-08-00042-g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6313513_jpm-08-00042-g004.jpg)

In vitro effects of the −1795G>A SNP on the binding of transcription factor NF-Y and OCT1 promoter activity. (A) Model for the allele-specific binding of the transcription factor NF-Y to a CCAAT-box built by the variant A- but not by the wild-type G-allele of −1795 G>A. (B) Sequences of the annealed oligonucleotides used as EMSA probes. The specific sequences are given in upper case, and the unspecific sequences used in the radioactive labeling of the EMSA probes are given in lower case letters. The SNP and mutated bases are shown in red. (C) A 32P-labeled probe containing either a −1795G or −1795A probe was incubated with nuclear extracts from HepG2 (left) and Hep3B cells (right), respectively, in the absence or presence of unlabeled probes (cold competition) or antibodies (supershift). The unlabeled probes were given in 5- and 15-fold molar excess of the 32P-labeled probe. (D) Luciferase reporter gene assay with the native −1736 and −1853 OCT1 promoter cloned in front of the luciferase gene in the pGL3-basic vector. The −1853 constructs include either the −1795G or the −1795 A-allele. The coordinates are given in base pairs related to the distance to the translational start site of OCT1. The data represent means and standard deviations of at least three independent experiments conducted in duplicate.

With strong evidence for NF-Y binding, leading to an allele-specific increase of activity in the pGL3-promoter constructs, we investigated the effect of −1795G>A on the native *OCT1* promoter. To this end, we cloned the *OCT1* promoter region spanning from −1853 to −61 bp from the translational start codon in the pGL3-basic vector ([Figure 4](#jpm-08-00042-f004)D). In luciferase reporter gene assays, we measured a significant increase in the luciferase activity in wild-type *OCT1* promoter compared to the empty vector in all cell lines tested: 2.34-fold in HepG2 (*p* < 0.0001), 2.10-fold in Hep3B (*p* < 0.001) and 3.51-fold in Huh7 cells (*p* < 0.0001). Surprisingly, there was no difference in the activity for the *OCT1* promoter between the two −1795G>A-alleles. Also, completely omitting the potential NF-Y binding region by using a construct spanning only from −1736 to −61 bp of the *OCT1* promoter did not lead to a significant reduction of the observed promoter activity. Moreover, in clinical human samples (the clinical studies have been described in detail previously [[5](#B5-jpm-08-00042),[6](#B6-jpm-08-00042),[8](#B8-jpm-08-00042),[9](#B9-jpm-08-00042),[11](#B11-jpm-08-00042)]) we found no effects of −1795G>A on the pharmacokinetics of metformin, fenoterol, sumatriptan and proguanil in healthy volunteers and on the efficacy of tropisetron in cancer patients ([Figure 5](#jpm-08-00042-f005)).

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/86a1/6313513/93f85923ef6d/jpm-08-00042-g005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6313513_jpm-08-00042-g005.jpg)

Clinical effects of the −1795G>A SNP. Genotyping was performed using a custom TaqMan® SNP genotyping assay (Thermo Fisher Scientific). (A) Renal clearance of metformin in 51 healthy volunteers. (B) Efficacy of tropisetron in 45 cancer patients suffering from vomiting. (C–E) Shown are the area under the time-concentration curves of fenoterol, sumatriptan and the ratio of cycloguanil to proguanil in dependence of the −1795G>A genotype of the volunteers. As comparison, the effects of amino acid mutations resulting in complete loss of OCT1 activity are shown on the right of each chart as red dots. p-values are based on non-parametric comparison using the Mann–Whitney-U Test.

Therefore, despite rather clear results with the artificial SV40 promoter, there is no conclusive evidence that the −1795G>A SNP is relevant for the native *OCT1* promoter activity.

### 3.3. Allele-Dependent Binding of USF1/2 to the −201C>G (rs58812592) Polymorphism

As previously demonstrated by Kajiwara et al. [[33](#B33-jpm-08-00042)], we could confirm binding of USF1/2 to an E-box in the *OCT1* promoter at −200 to −195 from the translational start codon of *OCT1* ([Figure 2](#jpm-08-00042-f002) and [Figure 6](#jpm-08-00042-f006)A). Furthermore, we analyzed the effect of the −201C>G SNP that is located in close proximity to the E-box ([Figure 1](#jpm-08-00042-f001)). Although, apparently, the intensity of USF1/2 binding to both alleles is quite similar ([Figure 2](#jpm-08-00042-f002)), more detailed cold competition analyses suggest that USF1/2 is binding more strongly to the −201C compared to the −201G-allele ([Figure 6](#jpm-08-00042-f006)A). Moreover, we detected significant allele-specific changes in the native *OCT1* promoter activity in dependence of the −201C>G allele ([Figure 6](#jpm-08-00042-f006)B). However, the −201C>G SNP was monomorphic in our genotyped volunteers and patients and could not be analyzed for clinical effects.

### Figure 6.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/86a1/6313513/c6b20507d8ff/jpm-08-00042-g006.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6313513_jpm-08-00042-g006.jpg)

Effects of the −201C>G SNP on binding of the transcription factors USF1/2 and on OCT1 promoter activity. (A) A 32P-labeled probe containing the −201C probe was incubated with nuclear extracts from HepG2 and Hep3B cells, respectively, in the absence or presence of unlabeled probes (cold competition) or antibodies (supershift). The unlabeled probes were given in 2-, 4- and 6-fold molar excess of the 32P-labeled probe. (B) Luciferase reporter gene assay with the native -1853 OCT1 promoter cloned in front of the luciferase gene in the pGL3-basic vector. The constructs include either the −201C- or the −201 G-allele. The coordinates are given in base pairs related to the distance to the translational start site of OCT1. The data represent means and standard deviations of at least three independent experiments conducted in duplicate.

### 3.4. Lack of Effect of -1620T>C on HNF4α Binding and OCT1 Promoter Activity

Despite not detecting the effects of −1620T>C in the functional screens ([Figure 2](#jpm-08-00042-f002) and [Figure 3](#jpm-08-00042-f003)), we further analyzed this SNP because of its localization between the two transcriptionally important DR2 *cis*-elements in the *OCT1* promoter. However, in line with the lack of retention signal in the EMSA analyses, −1620T>C does not affect native *OCT1* promoter activity in all cell lines tested ([Figure 7](#jpm-08-00042-f007)).

### Figure 7.

![Figure 7](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/86a1/6313513/de1764257d4d/jpm-08-00042-g007.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6313513_jpm-08-00042-g007.jpg)

Effect of the −1620 T>C SNP on OCT1 promoter activity. Luciferase reporter gene assay with the native −1853 OCT1 promoter cloned in front of the luciferase gene in the pGL3-basic vector. The constructs include either the −1620 T- or the −1620 C-allele. The coordinates are given in base pairs related to the distance to the translational start site of OCT1. The data represent means and standard deviations of at least three independent experiments conducted in duplicate.

## 4. Discussion

Here, we analyzed ten polymorphisms in the proximal promoter and up to 5 kb upstream of the human *SLC22A1* gene encoding for the organic cation transporter 1 (OCT1) as a possible source of the known high inter-individual variability in OCT1 expression. We performed functional analyses using luciferase reporter gene and electrophoretic mobility shift assays. The SNPs showing some effect on nuclear protein binding and/or luciferase gene activity were further analyzed for association with the pharmacokinetics parameter, which is known to be limited by OCT1 expression and activity. In contrast to other genes encoding for the important cation transporters OCT2, OCTN2, MATE1 and MATE2-K, where promoter polymorphisms were shown to affect expression and function [[29](#B29-jpm-08-00042),[31](#B31-jpm-08-00042),[32](#B32-jpm-08-00042),[35](#B35-jpm-08-00042)], we were not able to observe the significant effects of common polymorphisms on the expression or activity of OCT1.

In this study, we found that the nuclear transcription factor Y (NF-Y) may bind to a region −1798 bp to −1794 bp upstream of the start codon of *OCT1* and that this binding may be strongly affected by the common polymorphism −1795G>A (rs6935207). The consensus sequence for NF-Y binding is RRCCAATCA including the essential CCAAT box, which corresponds to the A-allele of the −1795G>A SNP. Indeed, both EMSA and luciferase reporter gene assays show strong functional binding only to the A-allele of this variant ([Figure 2](#jpm-08-00042-f002), [Figure 3](#jpm-08-00042-f003) and [Figure 4](#jpm-08-00042-f004)). However, in our study, we detected allele-specific enhancer activity for the region around −1795G>A, but only in the artificially shortened SV40 promoter construct and not in the extended native *OCT1* promoter ([Figure 3](#jpm-08-00042-f003) and [Figure 4](#jpm-08-00042-f004)). Both, our data ([Figure 2](#jpm-08-00042-f002), [Figure 3](#jpm-08-00042-f003) and [Figure 4](#jpm-08-00042-f004)C) and other studies [[36](#B36-jpm-08-00042),[37](#B37-jpm-08-00042),[38](#B38-jpm-08-00042),[39](#B39-jpm-08-00042),[40](#B40-jpm-08-00042)], showed clear functional activity on the transcription factor NF-Y in HepG2 and Hep3B cells, which we used as models in this study. Therefore, the lack of NF-Y activity in our model could not be an explanation for this observation. One possible explanation is that because of the proximity between the NF-Y binding site and the transcriptional start site (TSS), NF-Y contributes to polymerase II recruitment in the artificial pGL3-promoter construct ([Figure 8](#jpm-08-00042-f008)A). In the native *OCT1* promoter, however, the distance is too large to affect transcription ([Figure 8](#jpm-08-00042-f008)B). Furthermore, the common −1795G>A polymorphism (MAF = 19.7% in Caucasians) neither shows association with renal clearance of metformin or the pharmacokinetics of fenoterol, sumatriptan and proguanil, nor with the efficacy of tropisetron ([Figure 5](#jpm-08-00042-f005))—parameters known to be dependent on OCT1 activity in humans [[5](#B5-jpm-08-00042),[6](#B6-jpm-08-00042),[8](#B8-jpm-08-00042),[9](#B9-jpm-08-00042),[11](#B11-jpm-08-00042),[41](#B41-jpm-08-00042)]. The lack of effects of −1795G>A on OCT1 expression suggested in our work is in line with the observation of Kim et al., who genotyped 65 Koreans and correlated it with genetic and non-genetic factors, but they found no significant associations with any factor (including −1795G>A) to clarify the variability in OCT1 expression [[42](#B42-jpm-08-00042)]. In contrast, Maffioli et al. found a correlation between the −1795G>A SNP with inadequate response to imatinib [[43](#B43-jpm-08-00042)]. However, whether OCT1 plays any role in imatinib response or not is still widely questioned [[20](#B20-jpm-08-00042),[21](#B21-jpm-08-00042)].

### Figure 8.

![Figure 8](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/86a1/6313513/431f11656aa7/jpm-08-00042-g008.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6313513_jpm-08-00042-g008.jpg)

Model for the potential role of the transcription factor NF-Y in an artificial SV40 promoter construct and in the native OCT1 promoter. (A) A possible explanation for the enhancing effect on the SV40 promoter (Figure 3) is that NF-Y binding in close proximity to the TSS contributes to RNA polymerase II recruitment. (B) The lack of effect in the native OCT1 promoter (Figure 4D) may be caused by the large distance between NF-Y and the TSS.

OCT1 expression is known to be regulated by two transcription factors, HNF4α and USF1/2, binding to the *OCT1* promoter, and the transcription factor HNF1 binding to an evolutionary conserved region (ECR) in intron 1 of the *OCT1* gene [[24](#B24-jpm-08-00042),[33](#B33-jpm-08-00042),[34](#B34-jpm-08-00042)]. Our analysis of the polymorphism −1620T>C (rs9457840) that is located between the two DR2 elements to which HNF4α is binding, showed no effects on HNF4α binding to the DR2 elements and no effects on SV40 and *OCT1* promoter activity ([Figure 3](#jpm-08-00042-f003) and [Figure 7](#jpm-08-00042-f007)). However, in the case of a strong overexpression, this effect may change, but is less representative of the *in vivo* conditions in the human liver. As HNF4α expression levels are quite similar in HepG2 and Hep3B cells compared to human liver [[44](#B44-jpm-08-00042)], we kept native HNF4α expression conditions to simulate the native liver environment. Indeed, the polymorphism −201C>G (rs58812592), which is located directly adjacent to the E-box known to be involved in the binding of the transcription factors USF1/2, showed substantial effects on USF1/2 binding and *OCT1* promoter activity ([Figure 6](#jpm-08-00042-f006)), comparable to the effects of a targeted mutation of the E-Box [[33](#B33-jpm-08-00042)]. As binding of USF1/2 to the E-Box is an essential mechanism in the regulation of basal hepatic OCT1 expression, the −201C>G SNP suppresses the transcription of *OCT1*, which would result most likely in decreased OCT1 expression followed by a decreased uptake of drugs into the liver. However, −201C>G is actually a very rare genetic variant. Its minor allele frequency is 0.0001 (TOPMED) and 0.0002 (GnomAD), respectively, according to the dbSNP-database ([https://www.ncbi.nlm.nih.gov/snp/rs58812592](https://www.ncbi.nlm.nih.gov/snp/rs58812592) accessed on 10th December 2018). This SNP illustrates nicely the potential role of genetic variants affecting the binding of essential transcriptional factors, but cannot explain the commonly observed variation in OCT1 expression. Alternatively, variation in epigenetic [[26](#B26-jpm-08-00042)] or indirect regulation via regulating the expression of key transcriptional factors should be considered [[45](#B45-jpm-08-00042),[46](#B46-jpm-08-00042),[47](#B47-jpm-08-00042)].

This study has some limitations. Since we began planning the present study, more and more SNPs were reported in the analyzed 5 kb upstream region of the *OCT1* gene. However, most of the SNPs are very rare or are only found in diverse populations. In total, there are only 11 common SNPs with an MAF of at least 1% in this region ([Table S1](#app1-jpm-08-00042)). Thereof, seven SNPs (inclusive the six most common SNPs) were included in the present study. Additionally, in the ECR in intron 1, contributing to the transcriptional regulation of *OCT1* via HNF1 binding [[24](#B24-jpm-08-00042)], one SNP (rs145668795) had an MAF of at least 1%.

In conclusion, two out of ten analyzed polymorphisms in the *OCT1* promoter, −1795G>A (rs6935207) and −201C>G (rs58812592), showed functional effects *in vitro*. However, −1795G>A showed only allele-specific effects in EMSA and in an artificial SV40 promoter construct, but not in the native *OCT1* promoter. A possible explanation is that the SNP affects the binding of the transcriptional factor NF-Y, but due to the large distance to the transcriptional start site, this binding may not affect the native *OCT1* promoter activity. Furthermore, we identified the SNP −201C>G to be causing strong functional effects in the native *OCT1* promoter by affecting the binding of the transcription factor USF1/2. Although extremely rare, if present, this SNP may strongly influence OCT1 expression. Finally, none of the analyzed common SNPs in the promoter region of *OCT1* could explain the high inter-individual variability in OCT1 expression.

## Acknowledgments

The authors thank Vivien Hichert for her contribution to the initial parts of the project.

## Supplementary Materials

The following are available online at [http://www.mdpi.com/2075-4426/8/4/42/s1](http://www.mdpi.com/2075-4426/8/4/42/s1), Table S1: Common SNPs (MAF > 1%) in the 5 kb upstream region and in the ECR in intron 1 of the *OCT1* gene.

## Author Contributions

Conceptualization, J.B. and M.V.T.; Formal analysis, K.B.; Funding acquisition, J.B. and M.V.T.; Investigation, K.B.; Methodology, K.B.; Resources, J.B.; Supervision, M.V.T.; Validation, K.B. and M.V.T.; Visualization, K.B. and M.V.T.; Writing—original draft, K.B.; Writing—review and editing, K.B., J.B. and M.V.T.

## Funding

This project has been financially supported by the Deutsche Forschungsgemeinschaft [grant DFG TZ 74/1-1] to M.V.T. and J.B.

## Conflicts of Interest

The authors declare no conflict of interest.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Nies A.T., Koepsell H., Winter S., Burk O., Klein K., Kerb R., Zanger U.M., Keppler D., Schwab M., Schaeffeler E. Expression of organic cation transporters OCT1 (SLC22A1) and OCT3 (SLC22A3) is affected by genetic factors and cholestasis in human liver. Hepatology. 2009;50:1227–1240. doi: 10.1002/hep.23103.  [DOI](https://doi.org/10.1002/hep.23103) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19591196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Expression%20of%20organic%20cation%20transporters%20OCT1%20(SLC22A1)%20and%20OCT3%20(SLC22A3)%20is%20affected%20by%20genetic%20factors%20and%20cholestasis%20in%20human%20liver&author=A.T.%20Nies&author=H.%20Koepsell&author=S.%20Winter&author=O.%20Burk&author=K.%20Klein&volume=50&publication_year=2009&pages=1227-1240&pmid=19591196&doi=10.1002/hep.23103&)

2. Schaefer O., Ohtsuki S., Kawakami H., Inoue T., Liehner S., Saito A., Sakamoto A., Ishiguro N., Matsumaru T., Terasaki T., et al. Absolute quantification and differential expression of drug transporters, cytochrome p450 enzymes, and UDP-glucuronosyltransferases in cultured primary human hepatocytes. Drug Metab. Dispos. 2012;40:93–103. doi: 10.1124/dmd.111.042275.  [DOI](https://doi.org/10.1124/dmd.111.042275) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21979928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Absolute%20quantification%20and%20differential%20expression%20of%20drug%20transporters,%20cytochrome%20p450%20enzymes,%20and%20UDP-glucuronosyltransferases%20in%20cultured%20primary%20human%20hepatocytes&author=O.%20Schaefer&author=S.%20Ohtsuki&author=H.%20Kawakami&author=T.%20Inoue&author=S.%20Liehner&volume=40&publication_year=2012&pages=93-103&pmid=21979928&doi=10.1124/dmd.111.042275&)

3. Wang L., Prasad B., Salphati L., Chu X., Gupta A., Hop C.E., Evers R., Unadkat J.D. Interspecies variability in expression of hepatobiliary transporters across human, dog, monkey, and rat as determined by quantitative proteomics. Drug Metab. Dispos. 2015;43:367–374. doi: 10.1124/dmd.114.061580.  [DOI](https://doi.org/10.1124/dmd.114.061580) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25534768/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Interspecies%20variability%20in%20expression%20of%20hepatobiliary%20transporters%20across%20human,%20dog,%20monkey,%20and%20rat%20as%20determined%20by%20quantitative%20proteomics&author=L.%20Wang&author=B.%20Prasad&author=L.%20Salphati&author=X.%20Chu&author=A.%20Gupta&volume=43&publication_year=2015&pages=367-374&pmid=25534768&doi=10.1124/dmd.114.061580&)

4. Tzvetkov M.V., Saadatmand A.R., Lotsch J., Tegeder I., Stingl J.C., Brockmoller J. Genetically polymorphic oct1: Another piece in the puzzle of the variable pharmacokinetics and pharmacodynamics of the opioidergic drug tramadol. Clin. Pharmacol. Ther. 2011;90:143–150. doi: 10.1038/clpt.2011.56.  [DOI](https://doi.org/10.1038/clpt.2011.56) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21562485/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genetically%20polymorphic%20oct1:%20Another%20piece%20in%20the%20puzzle%20of%20the%20variable%20pharmacokinetics%20and%20pharmacodynamics%20of%20the%20opioidergic%20drug%20tramadol&author=M.V.%20Tzvetkov&author=A.R.%20Saadatmand&author=J.%20Lotsch&author=I.%20Tegeder&author=J.C.%20Stingl&volume=90&publication_year=2011&pages=143-150&pmid=21562485&doi=10.1038/clpt.2011.56&)

5. Tzvetkov M.V., Vormfelde S.V., Balen D., Meineke I., Schmidt T., Sehrt D., Sabolic I., Koepsell H., Brockmoller J. The effects of genetic polymorphisms in the organic cation transporters OCT1, OCT2, and OCT3 on the renal clearance of metformin. Clin. Pharmacol. Ther. 2009;86:299–306. doi: 10.1038/clpt.2009.92.  [DOI](https://doi.org/10.1038/clpt.2009.92) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19536068/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20effects%20of%20genetic%20polymorphisms%20in%20the%20organic%20cation%20transporters%20OCT1,%20OCT2,%20and%20OCT3%20on%20the%20renal%20clearance%20of%20metformin&author=M.V.%20Tzvetkov&author=S.V.%20Vormfelde&author=D.%20Balen&author=I.%20Meineke&author=T.%20Schmidt&volume=86&publication_year=2009&pages=299-306&pmid=19536068&doi=10.1038/clpt.2009.92&)

6. Tzvetkov M.V., Saadatmand A.R., Bokelmann K., Meineke I., Kaiser R., Brockmoller J. Effects of OCT1 polymorphisms on the cellular uptake, plasma concentrations and efficacy of the 5-HT(3) antagonists tropisetron and ondansetron. Pharmacogenomics J. 2012;12:22–29. doi: 10.1038/tpj.2010.75.  [DOI](https://doi.org/10.1038/tpj.2010.75) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20921968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Effects%20of%20OCT1%20polymorphisms%20on%20the%20cellular%20uptake,%20plasma%20concentrations%20and%20efficacy%20of%20the%205-HT(3)%20antagonists%20tropisetron%20and%20ondansetron&author=M.V.%20Tzvetkov&author=A.R.%20Saadatmand&author=K.%20Bokelmann&author=I.%20Meineke&author=R.%20Kaiser&volume=12&publication_year=2012&pages=22-29&pmid=20921968&doi=10.1038/tpj.2010.75&)

7. Tzvetkov M.V., dos Santos Pereira J.N., Meineke I., Saadatmand A.R., Stingl J.C., Brockmoller J. Morphine is a substrate of the organic cation transporter OCT1 and polymorphisms in OCT1 gene affect morphine pharmacokinetics after codeine administration. Biochem. Pharmacol. 2013;86:666–678. doi: 10.1016/j.bcp.2013.06.019.  [DOI](https://doi.org/10.1016/j.bcp.2013.06.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23835420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Pharmacol.&title=Morphine%20is%20a%20substrate%20of%20the%20organic%20cation%20transporter%20OCT1%20and%20polymorphisms%20in%20OCT1%20gene%20affect%20morphine%20pharmacokinetics%20after%20codeine%20administration&author=M.V.%20Tzvetkov&author=J.N.%20dos%20Santos%20Pereira&author=I.%20Meineke&author=A.R.%20Saadatmand&author=J.C.%20Stingl&volume=86&publication_year=2013&pages=666-678&pmid=23835420&doi=10.1016/j.bcp.2013.06.019&)

8. Matthaei J., Kuron D., Faltraco F., Knoch T., Dos Santos Pereira J.N., Abu Abed M., Prukop T., Brockmoller J., Tzvetkov M.V. OCT1 mediates hepatic uptake of sumatriptan and loss-of-function OCT1 polymorphisms affect sumatriptan pharmacokinetics. Clin. Pharmacol. Ther. 2016;99:633–641. doi: 10.1002/cpt.317.  [DOI](https://doi.org/10.1002/cpt.317) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26659468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=OCT1%20mediates%20hepatic%20uptake%20of%20sumatriptan%20and%20loss-of-function%20OCT1%20polymorphisms%20affect%20sumatriptan%20pharmacokinetics&author=J.%20Matthaei&author=D.%20Kuron&author=F.%20Faltraco&author=T.%20Knoch&author=J.N.%20Dos%20Santos%20Pereira&volume=99&publication_year=2016&pages=633-641&pmid=26659468&doi=10.1002/cpt.317&)

9. Tzvetkov M.V., Matthaei J., Pojar S., Faltraco F., Vogler S., Prukop T., Seitz T., Brockmoller J. Increased systemic exposure and stronger cardiovascular and metabolic adverse reactions to fenoterol in individuals with heritable oct1 deficiency. Clin. Pharmacol. Ther. 2018;103:868–878. doi: 10.1002/cpt.812.  [DOI](https://doi.org/10.1002/cpt.812) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28791698/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Increased%20systemic%20exposure%20and%20stronger%20cardiovascular%20and%20metabolic%20adverse%20reactions%20to%20fenoterol%20in%20individuals%20with%20heritable%20oct1%20deficiency&author=M.V.%20Tzvetkov&author=J.%20Matthaei&author=S.%20Pojar&author=F.%20Faltraco&author=S.%20Vogler&volume=103&publication_year=2018&pages=868-878&pmid=28791698&doi=10.1002/cpt.812&)

10. Sundelin E., Gormsen L.C., Jensen J.B., Vendelbo M.H., Jakobsen S., Munk O.L., Christensen M., Brosen K., Frokiaer J., Jessen N. Genetic polymorphisms in organic cation transporter 1 attenuates hepatic metformin exposure in humans. Clin. Pharmacol. Ther. 2017;102:841–848. doi: 10.1002/cpt.701.  [DOI](https://doi.org/10.1002/cpt.701) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28380657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genetic%20polymorphisms%20in%20organic%20cation%20transporter%201%20attenuates%20hepatic%20metformin%20exposure%20in%20humans&author=E.%20Sundelin&author=L.C.%20Gormsen&author=J.B.%20Jensen&author=M.H.%20Vendelbo&author=S.%20Jakobsen&volume=102&publication_year=2017&pages=841-848&pmid=28380657&doi=10.1002/cpt.701&)

11. Matthaei J., Seitz T., Jensen O., Tann A., Prukop T., Tadjerpisheh S., Brockmoller J., Tzvetkov M.V. Oct1 deficiency affects hepatocellular concentrations and pharmacokinetics of cycloguanil, the active metabolite of the antimalarial drug proguanil. Clin. Pharmacol. Ther. 2018 doi: 10.1002/cpt.1128.  [DOI](https://doi.org/10.1002/cpt.1128) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29882324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Oct1%20deficiency%20affects%20hepatocellular%20concentrations%20and%20pharmacokinetics%20of%20cycloguanil,%20the%20active%20metabolite%20of%20the%20antimalarial%20drug%20proguanil&author=J.%20Matthaei&author=T.%20Seitz&author=O.%20Jensen&author=A.%20Tann&author=T.%20Prukop&publication_year=2018&pmid=29882324&doi=10.1002/cpt.1128&)

12. Seitz T., Stalmann R., Dalila N., Chen J., Pojar S., Dos Santos Pereira J.N., Kratzner R., Brockmoller J., Tzvetkov M.V. Global genetic analyses reveal strong inter-ethnic variability in the loss of activity of the organic cation transporter oct1. Genome Med. 2015;7:56. doi: 10.1186/s13073-015-0172-0.  [DOI](https://doi.org/10.1186/s13073-015-0172-0) | [PMC free article](/articles/PMC4495841/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26157489/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med.&title=Global%20genetic%20analyses%20reveal%20strong%20inter-ethnic%20variability%20in%20the%20loss%20of%20activity%20of%20the%20organic%20cation%20transporter%20oct1&author=T.%20Seitz&author=R.%20Stalmann&author=N.%20Dalila&author=J.%20Chen&author=S.%20Pojar&volume=7&publication_year=2015&pages=56&pmid=26157489&doi=10.1186/s13073-015-0172-0&)

13. Saadatmand A.R., Tadjerpisheh S., Brockmoller J., Tzvetkov M.V. The prototypic pharmacogenetic drug debrisoquine is a substrate of the genetically polymorphic organic cation transporter OCT1. Biochem. Pharmacol. 2012;83:1427–1434. doi: 10.1016/j.bcp.2012.01.032.  [DOI](https://doi.org/10.1016/j.bcp.2012.01.032) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22342776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Pharmacol.&title=The%20prototypic%20pharmacogenetic%20drug%20debrisoquine%20is%20a%20substrate%20of%20the%20genetically%20polymorphic%20organic%20cation%20transporter%20OCT1&author=A.R.%20Saadatmand&author=S.%20Tadjerpisheh&author=J.%20Brockmoller&author=M.V.%20Tzvetkov&volume=83&publication_year=2012&pages=1427-1434&pmid=22342776&doi=10.1016/j.bcp.2012.01.032&)

14. Shu Y., Sheardown S.A., Brown C., Owen R.P., Zhang S., Castro R.A., Ianculescu A.G., Yue L., Lo J.C., Burchard E.G., et al. Effect of genetic variation in the organic cation transporter 1 (OCT1) on metformin action. J. Clin. Investig. 2007;117:1422–1431. doi: 10.1172/JCI30558.  [DOI](https://doi.org/10.1172/JCI30558) | [PMC free article](/articles/PMC1857259/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17476361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Investig.&title=Effect%20of%20genetic%20variation%20in%20the%20organic%20cation%20transporter%201%20(OCT1)%20on%20metformin%20action&author=Y.%20Shu&author=S.A.%20Sheardown&author=C.%20Brown&author=R.P.%20Owen&author=S.%20Zhang&volume=117&publication_year=2007&pages=1422-1431&pmid=17476361&doi=10.1172/JCI30558&)

15. Arimany-Nardi C., Minuesa G., Keller T., Erkizia I., Koepsell H., Martinez-Picado J., Pastor-Anglada M. Role of human organic cation transporter 1 (hOCT1) polymorphisms in lamivudine (3TC) uptake and drug-drug interactions. Front. Pharmacol. 2016;7:175. doi: 10.3389/fphar.2016.00175.  [DOI](https://doi.org/10.3389/fphar.2016.00175) | [PMC free article](/articles/PMC4919327/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27445813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Role%20of%20human%20organic%20cation%20transporter%201%20(hOCT1)%20polymorphisms%20in%20lamivudine%20(3TC)%20uptake%20and%20drug-drug%20interactions&author=C.%20Arimany-Nardi&author=G.%20Minuesa&author=T.%20Keller&author=I.%20Erkizia&author=H.%20Koepsell&volume=7&publication_year=2016&pages=175&pmid=27445813&doi=10.3389/fphar.2016.00175&)

16. Arimany-Nardi C., Montraveta A., Lee-Verges E., Puente X.S., Koepsell H., Campo E., Colomer D., Pastor-Anglada M. Human organic cation transporter 1 (hOCT1) as a mediator of bendamustine uptake and cytotoxicity in chronic lymphocytic leukemia (CLL) cells. Pharmacogenomics J. 2015;15:363–371. doi: 10.1038/tpj.2014.77.  [DOI](https://doi.org/10.1038/tpj.2014.77) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25582574/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Human%20organic%20cation%20transporter%201%20(hOCT1)%20as%20a%20mediator%20of%20bendamustine%20uptake%20and%20cytotoxicity%20in%20chronic%20lymphocytic%20leukemia%20(CLL)%20cells&author=C.%20Arimany-Nardi&author=A.%20Montraveta&author=E.%20Lee-Verges&author=X.S.%20Puente&author=H.%20Koepsell&volume=15&publication_year=2015&pages=363-371&pmid=25582574&doi=10.1038/tpj.2014.77&)

17. Zamek-Gliszczynski M.J., Giacomini K.M., Zhang L. Emerging clinical importance of hepatic organic cation transporter 1 (OCT1) in drug pharmacokinetics, dynamics, pharmacogenetic variability, and drug interactions. Clin. Pharmacol. Ther. 2018;103:758–760. doi: 10.1002/cpt.941.  [DOI](https://doi.org/10.1002/cpt.941) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29193038/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Emerging%20clinical%20importance%20of%20hepatic%20organic%20cation%20transporter%201%20(OCT1)%20in%20drug%20pharmacokinetics,%20dynamics,%20pharmacogenetic%20variability,%20and%20drug%20interactions&author=M.J.%20Zamek-Gliszczynski&author=K.M.%20Giacomini&author=L.%20Zhang&volume=103&publication_year=2018&pages=758-760&pmid=29193038&doi=10.1002/cpt.941&)

18. Tzvetkov M.V. OCT1 pharmacogenetics in pain management: Is a clinical application within reach? Pharmacogenomics. 2017;18:1515–1523. doi: 10.2217/pgs-2017-0095.  [DOI](https://doi.org/10.2217/pgs-2017-0095) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29061087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=OCT1%20pharmacogenetics%20in%20pain%20management:%20Is%20a%20clinical%20application%20within%20reach?&author=M.V.%20Tzvetkov&volume=18&publication_year=2017&pages=1515-1523&pmid=29061087&doi=10.2217/pgs-2017-0095&)

19. Yee S.W., Brackman D.J., Ennis E.A., Sugiyama Y., Kamdem L.K., Blanchard R., Galetin A., Zhang L., Giacomini K.M. Influence of transporter polymorphisms on drug disposition and response: A perspective from the international transporter consortium. Clin. Pharmacol. Ther. 2018 doi: 10.1002/cpt.1098.  [DOI](https://doi.org/10.1002/cpt.1098) | [PMC free article](/articles/PMC6348109/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29679469/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Influence%20of%20transporter%20polymorphisms%20on%20drug%20disposition%20and%20response:%20A%20perspective%20from%20the%20international%20transporter%20consortium&author=S.W.%20Yee&author=D.J.%20Brackman&author=E.A.%20Ennis&author=Y.%20Sugiyama&author=L.K.%20Kamdem&publication_year=2018&pmid=29679469&doi=10.1002/cpt.1098&)

20. Nies A.T., Schaeffeler E., van der Kuip H., Cascorbi I., Bruhn O., Kneba M., Pott C., Hofmann U., Volk C., Hu S., et al. Cellular uptake of imatinib into leukemic cells is independent of human organic cation transporter 1 (OCT1) Clin Cancer Res. 2014;20:985–994. doi: 10.1158/1078-0432.CCR-13-1999.  [DOI](https://doi.org/10.1158/1078-0432.CCR-13-1999) | [PMC free article](/articles/PMC3932302/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24352644/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res.&title=Cellular%20uptake%20of%20imatinib%20into%20leukemic%20cells%20is%20independent%20of%20human%20organic%20cation%20transporter%201%20(OCT1)&author=A.T.%20Nies&author=E.%20Schaeffeler&author=H.%20van%20der%20Kuip&author=I.%20Cascorbi&author=O.%20Bruhn&volume=20&publication_year=2014&pages=985-994&pmid=24352644&doi=10.1158/1078-0432.CCR-13-1999&)

21. Tzvetkov M.V., Seitz T., Bokelmann K., Mueller T., Brockmoller J., Koepsell H. Does the haplotype Met408-Del420, which was apparently predictive for imatinib efficacy, really exist and how strongly may it affect oct1 activity? Blood. 2014;123:1427–1429. doi: 10.1182/blood-2013-11-535864.  [DOI](https://doi.org/10.1182/blood-2013-11-535864) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24578499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Does%20the%20haplotype%20Met408-Del420,%20which%20was%20apparently%20predictive%20for%20imatinib%20efficacy,%20really%20exist%20and%20how%20strongly%20may%20it%20affect%20oct1%20activity?&author=M.V.%20Tzvetkov&author=T.%20Seitz&author=K.%20Bokelmann&author=T.%20Mueller&author=J.%20Brockmoller&volume=123&publication_year=2014&pages=1427-1429&pmid=24578499&doi=10.1182/blood-2013-11-535864&)

22. Neul C., Baker S.D., Sparreboom A., Schaeffeler E., Laufer S., Schwab M., Nies A.T. Evaluation of organic cation transporter 1 (OCT1, SLC22A1) as transporter for sorafenib; Proceedings of the AACR 107th Annual Meeting 2016; New Orleans, LA, USA. 16–20 April 2016.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20AACR%20107th%20Annual%20Meeting%202016&title=Evaluation%20of%20organic%20cation%20transporter%201%20(OCT1,%20SLC22A1)%20as%20transporter%20for%20sorafenib&author=C.%20Neul&author=S.D.%20Baker&author=A.%20Sparreboom&author=E.%20Schaeffeler&author=S.%20Laufer&)

23. Neul C., Schaeffeler E., Laufer S., Schwab M., Nies A.T. Cellular uptake of sorafenib is independent of major human organic cation and organic anion uptake transporters of the hepatocyte; Proceedings of the 17th Jahreskongress für Klinische Pharmakologie; Cologne, Germany. 1–2 October 2015.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%2017th%20Jahreskongress%20f%C3%BCr%20Klinische%20Pharmakologie&title=Cellular%20uptake%20of%20sorafenib%20is%20independent%20of%20major%20human%20organic%20cation%20and%20organic%20anion%20uptake%20transporters%20of%20the%20hepatocyte&author=C.%20Neul&author=E.%20Schaeffeler&author=S.%20Laufer&author=M.%20Schwab&author=A.T.%20Nies&)

24. O’Brien V.P., Bokelmann K., Ramirez J., Jobst K., Ratain M.J., Brockmoller J., Tzvetkov M.V. Hepatocyte nuclear factor 1 regulates the expression of the organic cation transporter 1 via binding to an evolutionary conserved region in intron 1 of the oct1 gene. J. Pharmacol. Exp. Ther. 2013;347:181–192. doi: 10.1124/jpet.113.206359.  [DOI](https://doi.org/10.1124/jpet.113.206359) | [PMC free article](/articles/PMC3781413/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23922447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=Hepatocyte%20nuclear%20factor%201%20regulates%20the%20expression%20of%20the%20organic%20cation%20transporter%201%20via%20binding%20to%20an%20evolutionary%20conserved%20region%20in%20intron%201%20of%20the%20oct1%20gene&author=V.P.%20O%E2%80%99Brien&author=K.%20Bokelmann&author=J.%20Ramirez&author=K.%20Jobst&author=M.J.%20Ratain&volume=347&publication_year=2013&pages=181-192&pmid=23922447&doi=10.1124/jpet.113.206359&)

25. Fattah S., Shinde A.B., Matic M., Baes M., van Schaik R.H.N., Allegaert K., Parmentier C., Richert L., Augustijns P., Annaert P. Inter-subject variability in OCT1 activity in 27 batches of cryopreserved human hepatocytes and association with oct1 mrna expression and genotype. Pharm. Res. 2017;34:1309–1319. doi: 10.1007/s11095-017-2148-9.  [DOI](https://doi.org/10.1007/s11095-017-2148-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28364304/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Res.&title=Inter-subject%20variability%20in%20OCT1%20activity%20in%2027%20batches%20of%20cryopreserved%20human%20hepatocytes%20and%20association%20with%20oct1%20mrna%20expression%20and%20genotype&author=S.%20Fattah&author=A.B.%20Shinde&author=M.%20Matic&author=M.%20Baes&author=R.H.N.%20van%20Schaik&volume=34&publication_year=2017&pages=1309-1319&pmid=28364304&doi=10.1007/s11095-017-2148-9&)

26. Schaeffeler E., Hellerbrand C., Nies A.T., Winter S., Kruck S., Hofmann U., van der Kuip H., Zanger U.M., Koepsell H., Schwab M. DNA methylation is associated with downregulation of the organic cation transporter OCT1 (SLC22A1) in human hepatocellular carcinoma. Genome Med. 2011;3:82. doi: 10.1186/gm298.  [DOI](https://doi.org/10.1186/gm298) | [PMC free article](/articles/PMC3334547/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22196450/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med.&title=DNA%20methylation%20is%20associated%20with%20downregulation%20of%20the%20organic%20cation%20transporter%20OCT1%20(SLC22A1)%20in%20human%20hepatocellular%20carcinoma&author=E.%20Schaeffeler&author=C.%20Hellerbrand&author=A.T.%20Nies&author=S.%20Winter&author=S.%20Kruck&volume=3&publication_year=2011&pages=82&pmid=22196450&doi=10.1186/gm298&)

27. Hesselson S.E., Matsson P., Shima J.E., Fukushima H., Yee S.W., Kobayashi Y., Gow J.M., Ha C., Ma B., Poon A., et al. Genetic variation in the proximal promoter of ABC and SLC superfamilies: Liver and kidney specific expression and promoter activity predict variation. PLoS ONE. 2009;4:e6942. doi: 10.1371/journal.pone.0006942.  [DOI](https://doi.org/10.1371/journal.pone.0006942) | [PMC free article](/articles/PMC2735003/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19742321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Genetic%20variation%20in%20the%20proximal%20promoter%20of%20ABC%20and%20SLC%20superfamilies:%20Liver%20and%20kidney%20specific%20expression%20and%20promoter%20activity%20predict%20variation&author=S.E.%20Hesselson&author=P.%20Matsson&author=J.E.%20Shima&author=H.%20Fukushima&author=S.W.%20Yee&volume=4&publication_year=2009&pages=e6942&pmid=19742321&doi=10.1371/journal.pone.0006942&)

28. Chen L., Hong C., Chen E.C., Yee S.W., Xu L., Almof E.U., Wen C., Fujii K., Johns S.J., Stryke D., et al. Genetic and epigenetic regulation of the organic cation transporter 3, SLC22A3. Pharmacogenomics J. 2013;13:110–120. doi: 10.1038/tpj.2011.60.  [DOI](https://doi.org/10.1038/tpj.2011.60) | [PMC free article](/articles/PMC3396779/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22231567/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Genetic%20and%20epigenetic%20regulation%20of%20the%20organic%20cation%20transporter%203,%20SLC22A3&author=L.%20Chen&author=C.%20Hong&author=E.C.%20Chen&author=S.W.%20Yee&author=L.%20Xu&volume=13&publication_year=2013&pages=110-120&pmid=22231567&doi=10.1038/tpj.2011.60&)

29. Ogasawara K., Terada T., Motohashi H., Asaka J., Aoki M., Katsura T., Kamba T., Ogawa O., Inui K. Analysis of regulatory polymorphisms in organic ion transporter genes (SLC22A) in the kidney. J. Hum. Genet. 2008;53:607–614. doi: 10.1007/s10038-008-0288-9.  [DOI](https://doi.org/10.1007/s10038-008-0288-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18414781/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hum.%20Genet.&title=Analysis%20of%20regulatory%20polymorphisms%20in%20organic%20ion%20transporter%20genes%20(SLC22A)%20in%20the%20kidney&author=K.%20Ogasawara&author=T.%20Terada&author=H.%20Motohashi&author=J.%20Asaka&author=M.%20Aoki&volume=53&publication_year=2008&pages=607-614&pmid=18414781&doi=10.1007/s10038-008-0288-9&)

30. Chung J.Y., Cho S.K., Kim T.H., Kim K.H., Jang G.H., Kim C.O., Park E.M., Cho J.Y., Jang I.J., Choi J.H. Functional characterization of MATE2-K genetic variants and their effects on metformin pharmacokinetics. Pharmacogenet. Genom. 2013;23:365–373. doi: 10.1097/FPC.0b013e3283622037.  [DOI](https://doi.org/10.1097/FPC.0b013e3283622037) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23652408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=Functional%20characterization%20of%20MATE2-K%20genetic%20variants%20and%20their%20effects%20on%20metformin%20pharmacokinetics&author=J.Y.%20Chung&author=S.K.%20Cho&author=T.H.%20Kim&author=K.H.%20Kim&author=G.H.%20Jang&volume=23&publication_year=2013&pages=365-373&pmid=23652408&doi=10.1097/FPC.0b013e3283622037&)

31. Choi J.H., Yee S.W., Ramirez A.H., Morrissey K.M., Jang G.H., Joski P.J., Mefford J.A., Hesselson S.E., Schlessinger A., Jenkins G., et al. A common 5′-UTR variant in MATE2-K is associated with poor response to metformin. Clin. Pharmacol. Ther. 2011;90:674–684. doi: 10.1038/clpt.2011.165.  [DOI](https://doi.org/10.1038/clpt.2011.165) | [PMC free article](/articles/PMC3329222/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21956618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=A%20common%205%E2%80%B2-UTR%20variant%20in%20MATE2-K%20is%20associated%20with%20poor%20response%20to%20metformin&author=J.H.%20Choi&author=S.W.%20Yee&author=A.H.%20Ramirez&author=K.M.%20Morrissey&author=G.H.%20Jang&volume=90&publication_year=2011&pages=674-684&pmid=21956618&doi=10.1038/clpt.2011.165&)

32. Ha Choi J., Wah Yee S., Kim M.J., Nguyen L., Ho Lee J., Kang J.O., Hesselson S., Castro R.A., Stryke D., Johns S.J., et al. Identification and characterization of novel polymorphisms in the basal promoter of the human transporter, mate1. Pharmacogenet. Genom. 2009;19:770–780. doi: 10.1097/FPC.0b013e328330eeca.  [DOI](https://doi.org/10.1097/FPC.0b013e328330eeca) | [PMC free article](/articles/PMC2976711/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19745787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=Identification%20and%20characterization%20of%20novel%20polymorphisms%20in%20the%20basal%20promoter%20of%20the%20human%20transporter,%20mate1&author=J.%20Ha%20Choi&author=S.%20Wah%20Yee&author=M.J.%20Kim&author=L.%20Nguyen&author=J.%20Ho%20Lee&volume=19&publication_year=2009&pages=770-780&pmid=19745787&doi=10.1097/FPC.0b013e328330eeca&)

33. Kajiwara M., Terada T., Asaka J., Aoki M., Katsura T., Ikai I., Inui K. Regulation of basal core promoter activity of human organic cation transporter 1 (OCT1/SLC22A1) Am. J. Physiol. Gastrointest. Liver Physiol. 2008;295:G1211–1216. doi: 10.1152/ajpgi.90360.2008.  [DOI](https://doi.org/10.1152/ajpgi.90360.2008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18845576/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Physiol.%20Gastrointest.%20Liver%20Physiol.&title=Regulation%20of%20basal%20core%20promoter%20activity%20of%20human%20organic%20cation%20transporter%201%20(OCT1/SLC22A1)&author=M.%20Kajiwara&author=T.%20Terada&author=J.%20Asaka&author=M.%20Aoki&author=T.%20Katsura&volume=295&publication_year=2008&pages=G1211-1216&pmid=18845576&doi=10.1152/ajpgi.90360.2008&)

34. Saborowski M., Kullak-Ublick G.A., Eloranta J.J. The human organic cation transporter-1 gene is transactivated by hepatocyte nuclear factor-4aα. J. Pharmacol. Exp. Ther. 2006;317:778–785. doi: 10.1124/jpet.105.099929.  [DOI](https://doi.org/10.1124/jpet.105.099929) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16436500/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=The%20human%20organic%20cation%20transporter-1%20gene%20is%20transactivated%20by%20hepatocyte%20nuclear%20factor-4a%CE%B1&author=M.%20Saborowski&author=G.A.%20Kullak-Ublick&author=J.J.%20Eloranta&volume=317&publication_year=2006&pages=778-785&pmid=16436500&doi=10.1124/jpet.105.099929&)

35. Tahara H., Yee S.W., Urban T.J., Hesselson S., Castro R.A., Kawamoto M., Stryke D., Johns S.J., Ferrin T.E., Kwok P.Y., et al. Functional genetic variation in the basal promoter of the organic cation/carnitine transporters OCTN1 (SLC22A4) and OCTN2 (SLC22A5) J. Pharmacol. Exp. Ther. 2009;329:262–271. doi: 10.1124/jpet.108.146449.  [DOI](https://doi.org/10.1124/jpet.108.146449) | [PMC free article](/articles/PMC2670604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19141711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=Functional%20genetic%20variation%20in%20the%20basal%20promoter%20of%20the%20organic%20cation/carnitine%20transporters%20OCTN1%20(SLC22A4)%20and%20OCTN2%20(SLC22A5)&author=H.%20Tahara&author=S.W.%20Yee&author=T.J.%20Urban&author=S.%20Hesselson&author=R.A.%20Castro&volume=329&publication_year=2009&pages=262-271&pmid=19141711&doi=10.1124/jpet.108.146449&)

36. Zhang Y., Chen B., Li Y., Chen J., Lou G., Chen M., Zhou D. Transcriptional regulation of the human PNRC promoter by nfy in HEPG2 cells. J. Biochem. 2008;143:675–683. doi: 10.1093/jb/mvn019.  [DOI](https://doi.org/10.1093/jb/mvn019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18281297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biochem.&title=Transcriptional%20regulation%20of%20the%20human%20PNRC%20promoter%20by%20nfy%20in%20HEPG2%20cells&author=Y.%20Zhang&author=B.%20Chen&author=Y.%20Li&author=J.%20Chen&author=G.%20Lou&volume=143&publication_year=2008&pages=675-683&pmid=18281297&doi=10.1093/jb/mvn019&)

37. Novak E.M., Bydlowski S.P. NFY transcription factor binds to regulatory element AIC and transactivates the human apolipoprotein A-I promoter in HEPG2 cells. Biochem. Biophys. Res. Commun. 1997;231:140–143. doi: 10.1006/bbrc.1997.6056.  [DOI](https://doi.org/10.1006/bbrc.1997.6056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9070236/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&title=NFY%20transcription%20factor%20binds%20to%20regulatory%20element%20AIC%20and%20transactivates%20the%20human%20apolipoprotein%20A-I%20promoter%20in%20HEPG2%20cells&author=E.M.%20Novak&author=S.P.%20Bydlowski&volume=231&publication_year=1997&pages=140-143&pmid=9070236&doi=10.1006/bbrc.1997.6056&)

38. Pallai R., Simpkins H., Chen J., Parekh H.K. The CCAAT box binding transcription factor, nuclear factor-Y (NF-Y) regulates transcription of human aldo-keto reductase 1C1 (AKR1C1) gene. Gene. 2010;459:11–23. doi: 10.1016/j.gene.2010.03.006.  [DOI](https://doi.org/10.1016/j.gene.2010.03.006) | [PMC free article](/articles/PMC2874818/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20338228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=The%20CCAAT%20box%20binding%20transcription%20factor,%20nuclear%20factor-Y%20(NF-Y)%20regulates%20transcription%20of%20human%20aldo-keto%20reductase%201C1%20(AKR1C1)%20gene&author=R.%20Pallai&author=H.%20Simpkins&author=J.%20Chen&author=H.K.%20Parekh&volume=459&publication_year=2010&pages=11-23&pmid=20338228&doi=10.1016/j.gene.2010.03.006&)

39. Yanagawa Y., Chen J.C., Hsu L.C., Yoshida A. The transcriptional regulation of human aldehyde dehydrogenase I gene. The structural and functional analysis of the promoter. J. Biol. Chem. 1995;270:17521–17527. doi: 10.1074/jbc.270.29.17521.  [DOI](https://doi.org/10.1074/jbc.270.29.17521) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7615557/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=The%20transcriptional%20regulation%20of%20human%20aldehyde%20dehydrogenase%20I%20gene.%20The%20structural%20and%20functional%20analysis%20of%20the%20promoter&author=Y.%20Yanagawa&author=J.C.%20Chen&author=L.C.%20Hsu&author=A.%20Yoshida&volume=270&publication_year=1995&pages=17521-17527&pmid=7615557&doi=10.1074/jbc.270.29.17521&)

40. Ueda A., Yoshimura T. Characterization of cis-acting elements of the gene for macrophage-stimulating protein from the human. The involvement of positive and negative regulatory elements. J. Biol. Chem. 1996;271:20265–20272. doi: 10.1074/jbc.271.34.20265.  [DOI](https://doi.org/10.1074/jbc.271.34.20265) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8702759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Characterization%20of%20cis-acting%20elements%20of%20the%20gene%20for%20macrophage-stimulating%20protein%20from%20the%20human.%20The%20involvement%20of%20positive%20and%20negative%20regulatory%20elements&author=A.%20Ueda&author=T.%20Yoshimura&volume=271&publication_year=1996&pages=20265-20272&pmid=8702759&doi=10.1074/jbc.271.34.20265&)

41. Christensen M.M., Brasch-Andersen C., Green H., Nielsen F., Damkier P., Beck-Nielsen H., Brosen K. The pharmacogenetics of metformin and its impact on plasma metformin steady-state levels and glycosylated hemoglobin A1c. Pharmacogenet. Genom. 2011;21:837–850. doi: 10.1097/FPC.0b013e32834c0010.  [DOI](https://doi.org/10.1097/FPC.0b013e32834c0010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21989078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=The%20pharmacogenetics%20of%20metformin%20and%20its%20impact%20on%20plasma%20metformin%20steady-state%20levels%20and%20glycosylated%20hemoglobin%20A1c&author=M.M.%20Christensen&author=C.%20Brasch-Andersen&author=H.%20Green&author=F.%20Nielsen&author=P.%20Damkier&volume=21&publication_year=2011&pages=837-850&pmid=21989078&doi=10.1097/FPC.0b013e32834c0010&)

42. Kim M.H., Shin H.J., Lim S.J., Park J.S., Lee S.S., Song I.S., Shin J.G. Inter-individual variability in OCT1 expression and its relationship with OCT1 genotype in liver samples from a korean population. Drug Metab. Pharmacokinet. 2012;27:530–535. doi: 10.2133/dmpk.DMPK-11-RG-109.  [DOI](https://doi.org/10.2133/dmpk.DMPK-11-RG-109) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22498645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Pharmacokinet.&title=Inter-individual%20variability%20in%20OCT1%20expression%20and%20its%20relationship%20with%20OCT1%20genotype%20in%20liver%20samples%20from%20a%20korean%20population&author=M.H.%20Kim&author=H.J.%20Shin&author=S.J.%20Lim&author=J.S.%20Park&author=S.S.%20Lee&volume=27&publication_year=2012&pages=530-535&pmid=22498645&doi=10.2133/dmpk.DMPK-11-RG-109&)

43. Maffioli M., Camos M., Gaya A., Hernandez-Boluda J.C., Alvarez-Larran A., Domingo A., Granell M., Guillem V., Vallansot R., Costa D., et al. Correlation between genetic polymorphisms of the hOCT1 and mdr1 genes and the response to imatinib in patients newly diagnosed with chronic-phase chronic myeloid leukemia. Leuk. Res. 2011;35:1014–1019. doi: 10.1016/j.leukres.2010.12.004.  [DOI](https://doi.org/10.1016/j.leukres.2010.12.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21185600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk.%20Res.&title=Correlation%20between%20genetic%20polymorphisms%20of%20the%20hOCT1%20and%20mdr1%20genes%20and%20the%20response%20to%20imatinib%20in%20patients%20newly%20diagnosed%20with%20chronic-phase%20chronic%20myeloid%20leukemia&author=M.%20Maffioli&author=M.%20Camos&author=A.%20Gaya&author=J.C.%20Hernandez-Boluda&author=A.%20Alvarez-Larran&volume=35&publication_year=2011&pages=1014-1019&pmid=21185600&doi=10.1016/j.leukres.2010.12.004&)

44. Martinez-Jimenez C.P., Gomez-Lechon M.J., Castell J.V., Jover R. Underexpressed coactivators PGC1α and SRC1 impair hepatocyte nuclear factor 4α function and promote dedifferentiation in human hepatoma cells. J. Biol. Chem. 2006;281:29840–29849. doi: 10.1074/jbc.M604046200.  [DOI](https://doi.org/10.1074/jbc.M604046200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16891307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Underexpressed%20coactivators%20PGC1%CE%B1%20and%20SRC1%20impair%20hepatocyte%20nuclear%20factor%204%CE%B1%20function%20and%20promote%20dedifferentiation%20in%20human%20hepatoma%20cells&author=C.P.%20Martinez-Jimenez&author=M.J.%20Gomez-Lechon&author=J.V.%20Castell&author=R.%20Jover&volume=281&publication_year=2006&pages=29840-29849&pmid=16891307&doi=10.1074/jbc.M604046200&)

45. Hyrsova L., Smutny T., Carazo A., Moravcik S., Mandikova J., Trejtnar F., Gerbal-Chaloin S., Pavek P. The pregnane x receptor down-regulates organic cation transporter 1 (SLC22A1) in human hepatocytes by competing for (“squelching”) SRC-1 coactivator. Br. J. Pharmacol. 2016;173:1703–1715. doi: 10.1111/bph.13472.  [DOI](https://doi.org/10.1111/bph.13472) | [PMC free article](/articles/PMC4842917/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26920453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Pharmacol.&title=The%20pregnane%20x%20receptor%20down-regulates%20organic%20cation%20transporter%201%20(SLC22A1)%20in%20human%20hepatocytes%20by%20competing%20for%20(%E2%80%9Csquelching%E2%80%9D)%20SRC-1%20coactivator&author=L.%20Hyrsova&author=T.%20Smutny&author=A.%20Carazo&author=S.%20Moravcik&author=J.%20Mandikova&volume=173&publication_year=2016&pages=1703-1715&pmid=26920453&doi=10.1111/bph.13472&)

46. Rulcova A., Krausova L., Smutny T., Vrzal R., Dvorak Z., Jover R., Pavek P. Glucocorticoid receptor regulates organic cation transporter 1 (OCT1, SLC22A1) expression via hnf4alpha upregulation in primary human hepatocytes. Pharmacol. Rep. 2013;65:1322–1335. doi: 10.1016/S1734-1140(13)71491-9.  [DOI](https://doi.org/10.1016/S1734-1140(13)71491-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24399729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Rep.&title=Glucocorticoid%20receptor%20regulates%20organic%20cation%20transporter%201%20(OCT1,%20SLC22A1)%20expression%20via%20hnf4alpha%20upregulation%20in%20primary%20human%20hepatocytes&author=A.%20Rulcova&author=L.%20Krausova&author=T.%20Smutny&author=R.%20Vrzal&author=Z.%20Dvorak&volume=65&publication_year=2013&pages=1322-1335&pmid=24399729&doi=10.1016/S1734-1140(13)71491-9&)

47. Hyrsova L., Smutny T., Trejtnar F., Pavek P. Expression of organic cation transporter 1 (OCT1): Unique patterns of indirect regulation by nuclear receptors and hepatospecific gene regulation. Drug Metab. Rev. 2016;48:139–158. doi: 10.1080/03602532.2016.1188936.  [DOI](https://doi.org/10.1080/03602532.2016.1188936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27278216/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Rev.&title=Expression%20of%20organic%20cation%20transporter%201%20(OCT1):%20Unique%20patterns%20of%20indirect%20regulation%20by%20nuclear%20receptors%20and%20hepatospecific%20gene%20regulation&author=L.%20Hyrsova&author=T.%20Smutny&author=F.%20Trejtnar&author=P.%20Pavek&volume=48&publication_year=2016&pages=139-158&pmid=27278216&doi=10.1080/03602532.2016.1188936&)
