# Impact of CYP3A4*22 on Pazopanib Pharmacokinetics in Cancer Patients

## Metadata
**Authors:** Sander Bins, Alwin D R Huitema, Pim Laven, Samira el Bouazzaoui, Huixin Yu, Nielka van Erp, Carla van Herpen, Paul Hamberg, Hans Gelderblom, Neeltje Steeghs, Stefan Sleijfer, Ron H N van Schaik, Ron H J Mathijssen, Stijn L W Koolen
**Journal:** Clinical Pharmacokinetics
**Date:** 2018 Oct 27
**DOI:** [10.1007/s40262-018-0719-5](https://doi.org/10.1007/s40262-018-0719-5)
**PMID:** 30367352
**PMCID:** PMC6451710
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6451710/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6451710/pdf/40262_2018_Article_719.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6451710/pdf/40262_2018_Article_719.pdf)

## Abstract

**Background and Objective:** 
As pazopanib plasma trough concentrations are correlated with treatment outcome, we explored whether single nucleotide polymorphisms in the elimination pathway of pazopanib affect systemic pazopanib concentrations.

**Methods:** 
The decreased function alleles CYP3A4 15389 C > T (*22), ABCB1 3435 C >T, ABCG2 421 C >A, and ABCG2 34G >A were analyzed within a recently developed population-pharmacokinetic model.

**Results:** 
Incorporation of CYP3A4*22 in the model resulted in a 35% lower clearance for variant carriers (0.18 vs. 0.27 L/h; difference in objective function value: − 9.7; p < 0.005). Simulated median trough concentrations of cancer patients with CYP3A4*22 with 600 mg once daily or 800 mg once daily were 31 and 35 mg/L, respectively. The simulated trough concentrations for the population excluding the CYP3A4*22 carriers after 600 mg once daily or 800 mg once daily were 18 and 20 mg/L, respectively.

**Conclusion:** 
This analysis shows that CYP3A4*22 heterozygotes have a substantial lower pazopanib clearance and that dose adjustments based on CYP3A4*22 status could be considered.

**Electronic supplementary material:** 
The online version of this article (10.1007/s40262-018-0719-5) contains supplementary material, which is available to authorized users.

### Background and Objective

As pazopanib plasma trough concentrations are correlated with treatment outcome, we explored whether single nucleotide polymorphisms in the elimination pathway of pazopanib affect systemic pazopanib concentrations.

### Methods

The decreased function alleles *CYP3A4 15389 C *> *T (*22)*, *ABCB1 3435 C *>*T*, *ABCG2 421 C *>*A*, and *ABCG2 34G *>*A* were analyzed within a recently developed population-pharmacokinetic model.

### Results

Incorporation of *CYP3A4*22* in the model resulted in a 35% lower clearance for variant carriers (0.18 vs. 0.27 L/h; difference in objective function value: − 9.7; *p* < 0.005). Simulated median trough concentrations of cancer patients with *CYP3A4*22* with 600 mg once daily or 800 mg once daily were 31 and 35 mg/L, respectively. The simulated trough concentrations for the population excluding the *CYP3A4*22* carriers after 600 mg once daily or 800 mg once daily were 18 and 20 mg/L, respectively.

### Conclusion

This analysis shows that *CYP3A4*22* heterozygotes have a substantial lower pazopanib clearance and that dose adjustments based on *CYP3A4*22* status could be considered.

### Electronic supplementary material

The online version of this article (10.1007/s40262-018-0719-5) contains supplementary material, which is available to authorized users.

## Key Points

| Patients carrying CYP3A4*22 have a lower pazopanib clearance than CYP3A4 wild types. |
| --- |
| In our simulations, a pazopanib dose of 600 mg in these patients leads to equal pazopanib exposure as wild type patients dosed at 800 mg. |
| This knowledge may potentially lead to alternative dosing of patients carrying CYP3A4*22. |
## Introduction

Pazopanib is a tyrosine kinase inhibitor that is currently a first-line treatment option for renal cell carcinoma, and a second-line treatment or beyond for non-adipocytic soft-tissue sarcoma [[3](#CR3), [4](#CR4)]. Despite its proven clinical efficacy in these tumor types, dosing may be a challenge, as toxicity may become a serious limitation: 14–15% of patients discontinue pazopanib because of adverse events [[3](#CR3), [4](#CR4)]. Nausea, diarrhea, headache, hand-foot skin reaction, fatigue, and liver function disorders are among the most common side effects. From the outset, it is unknown which patients will develop side effects and which patients will not.

Likewise, it cannot be predicted which cancer patients will benefit from treatment with pazopanib. There are many factors that determine the outcome of an individual patient to a particular treatment; the pharmacology of an anti-tumor drug being one of these. As trough concentrations below 20.5 µg/mL have been associated with a worse clinical outcome [[5](#CR5), [6](#CR6)], it seems vital to dose patients up to this threshold value. Recently, we showed that this strategy is feasible and does not lead to increased toxicity in most patients [[7](#CR7)]. However, toxicity is dose limiting in a substantial number of patients, and may be the result of high systemic concentrations [[8](#CR8)].

Several factors have been described to influence pazopanib exposure. Its absorption has been shown to increase with concomitant food intake [[9](#CR9), [10](#CR10)] and to decrease when stomach pH is reduced [[11](#CR11)], whereas its elimination is known to be increased as a result of cytochrome P450 (CYP)3A4 induction by ifosfamide or aprepitant [[2](#CR2), [12](#CR12)]. Although these drugs are generally not being used in combination with pazopanib, these drug–drug interactions stress the importance of the function of the enzyme CYP3A4. However, drug transporters such as ABCB1 (P-glycoprotein) and ABCG2 (breast cancer resistant protein), for which pazopanib is a substrate, may also be relevant according to the summary of product characteristics.

Transporter and enzyme function are not solely affected by co-medication, but also by single nucleotide polymorphisms (SNPs) in their encoding genes. Up till now, genetic polymorphisms have been associated only with pazopanib pharmacodynamics rather than with its pharmacokinetics [[13](#CR13), [14](#CR14)]. Therefore, in the current study, it was our goal to associate four potentially relevant SNPs in *CYP3A4*, *ABCB1*, and *ABCG2* with pazopanib pharmacokinetics. These SNPs have earlier been shown to be associated with the exposure of other tyrosine kinase inhibitors such as sunitinib [[15](#CR15)]. We did this by implementing SNP data into a recently developed population-pharmacokinetic model for pazopanib systemic exposure, describing the complex metabolism of pazopanib in cancer patients [[16](#CR16)].

## Methods

### Study Population

Patients were selected from three separate studies in which pazopanib pharmacokinetics had been assessed: two studies assessed the pharmacokinetic and pharmacodynamic drug interactions between pazopanib and either docetaxel (NTR2309) [[1](#CR1)] or ifosfamide (NTR2063) [[2](#CR2)] and the third study investigated the feasibility of therapeutic drug monitoring in pazopanib-treated patients (NTR3293) [[17](#CR17)]/ Respectively 40, 41, and 13 patients were included in these studies. All patients signed informed consent.

### Single Nucleotide Polymorphism Selection

As pazopanib is acknowledged as a substrate for CYP3A4, ABCB1, and ABCG2, four SNPs in the encoding genes for these proteins were selected, based on previous studies investigating other tyrosine kinase inhibitors [[18](#CR18)–[21](#CR21)]. Details of the investigated SNPs are depicted in Table [1](#Tab1).

### Table 1.

| Gene | rs number | Allele | Annotation | WT/WT | WT/var | Var/var | Missing | MAFa (%) | MAFb (%) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| ABCG2 | rs2231142 | 421C>A |   | 75 | 18 | 1 | 0 | 11 | 9 |
| ABCG2 | rs2231137 | 34G>A |   | 83 | 10 | 0 | 1 | 5 | 6 |
| ABCB1 | rs1045642 | 3435C>T |   | 15 | 50 | 29 | 0 | 43 | 48 |
| CYP3A4 | rs35599367 | 15389C>T | *22 | 83 | 11 | 0 | 0 | 6 | 5 |

Table 1 Caption: Investigated single nucleotide polymorphisms

### DNA Isolation

If available, 2 mL of plasma or 3 mL of whole blood, stored at − 20 °C, was extracted on the MagNAPure Compact instrument (Roche Diagnostics GmbH, Mannheim, Germany) using the Nucleic Acid Isolation Kit I (Roche Diagnostics GmbH) and a final elution volume of 200 μL.

### Taqman® Genotyping

Predesigned drug-metabolizing enzymes (Taqman^®^ allelic discrimination assays) were used to perform genotyping on the Life Technologies Taqman^®^ 7500 system (Applied Biosystems, Life Technologies Europe BV, Bleiswijk, The Netherlands). Each assay consisted of two allele-specific minor groove-binding probes, labeled with the fluorescent dyes VIC and FAM. Polymerase chain reactions were performed in a reaction volume of 10 μL, containing assay-specific primers, allele-specific Taqman^®^ minor groove-binding probes, Abgene Absolute QPCR Rox Mix (Thermo Scientific, Life Technologies Europe BV, Bleiswijk, The Netherlands), and genomic DNA (20 ng). The thermal profile consisted of 40 cycles of denaturation at 95 °C for 20 s and annealing at 92 °C for 3 s and extension at 60 °C for 30 s. Allele-specific fluorescence was measured using the 7500 software Version 2.3 for allelic discrimination (Applied Biosystems) to score genotypes.

### Population-Pharmacokinetic Model

A previously developed population-pharmacokinetic model with pharmacokinetic data from the same patients was used as a base model for the current analysis [[16](#CR16)]. This model described the complex pazopanib pharmacokinetics including the saturable absorption process and the drug–drug interaction with ifosfamide [[22](#CR22)]. The remaining unexplained variation in pazopanib pharmacokinetics was predominantly reflected in the between-subject variability of the fast absorption rate constant (coefficient of variation = 140%) and clearance (coefficient of variation = 30.9%).

### Statistical Analysis

All SNPs were in Hardy–Weinberg equilibrium. The genetic data were added to the data used for developing the pharmacokinetic model. NONMEM (Version 7.2; ICON, Ellicott City, MD, USA) with the first-order conditional estimation method with interaction was used for parameter estimation. Piraña was used as the modeling environment [[23](#CR23)] and R (Version 3.3.1) was used for graphical interpretation of the data.

Genotypes with a frequency of more than 5% were tested in the base model and genotypes with a frequency of less than 5% were ignored. Covariates were included as dichotomous covariates using the following equation:

| Ppop=θx×θypg, |
| --- |
where *P*_pop_ is the pharmacokinetic parameter of interest, pg was scored ‘1’ for patients of whom the genotype of interest was present and ‘0’ for patients of whom the genotype was absent, *θ*_x_ is the typical parameter value of the wild-type population, and *θ*_y_ is the covariate effect size estimate. The association of the SNPs with pazopanib clearance and bioavailability was investigated following a stepwise procedure. The potential association of all SNPs was univariately tested for both pharmacokinetic endpoints. Univariate forward inclusion was performed using a *p* value of < 0.05 as statistically significant (likelihood ratio test; 1 degree of freedom, difference in objective function value: 3.84). This was followed by a backward elimination procedure using a *p* value of < 0.005 (difference in objective function value: 7.88) as the significance threshold (likelihood ratio test).

Ultimately, the final population-pharmacokinetic model [see Electronic Supplementary Material (ESM)] was used for simulating median trough concentrations for patients with and without *CYP3A4*22* dosed at 400, 600, and 800 mg once daily. For each combination of dose and genotype, 500 individuals were simulated. The simulated results were analyzed using R to present a concentration–time curve at steady state (6 weeks after start) and to calculate the percentage of simulated individuals reaching the target trough concentration of 20.5 ug/L. Goodness-of-fit plots are depicted in Fig. [1](#Fig1) of the ESM for all patients and in Fig. [2](#Fig2) of the ESM for the *CYP3A4*22* carriers.

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4ed1/6451710/575f5db69b1b/40262_2018_719_Fig1_HTML.jpg)

Schematic overview of the pazopanib (pazo) pharmacokinetic model [16]. ALAG lag time, CL clearance, fr fraction of pazopanib, KaF fast absorption rate constant, KaS slow absorption rate constant, Vc central volume of distribution, Vp peripheral volume of distribution

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4ed1/6451710/9b6ce2e6ca4d/40262_2018_719_Fig2_HTML.jpg)

Median simulated pazopanib trough concentration after a single dose at steady state. The red line and the red dashed line represent the simulated concentration–time curves for CYP3A4*22 patients, treated with 800 mg and 600 mg of pazopanib daily, respectively. The black line and the black dashed line represent the simulated concentration–time curve for wild-type (WT) patients for CYP3A4, treated with 800 mg and 600 mg of pazopanib daily, respectively. 1dd once daily

## Results

### Patients

From the 96 patients in the original population-pharmacokinetic model by Yu et al. [[16](#CR16)], germline DNA was genotyped successfully in 94 (98%) of them. From these 94 patients, 761 samples had been drawn for pharmacokinetic analysis. Because most patients had participated in phase I studies in which pazopanib was combined with either ifosfamide or docetaxel, tumor types varied widely between patients and were therefore not recorded in this study. Pazopanib starting dose ranged from 200 to 1000 mg daily. Most patients were male (71%) and Caucasian (97%; Table [2](#Tab2)).

### Table 2.

| Characteristics | No. of patients (%) |
| --- | --- |
| Original study |  |
| Pazopanib TDM [17] | 13 (14) |
| Phase I with docetaxel [1] | 40 (43) |
| Phase I with ifosfamide [2] | 41 (44) |
| Sex |   |
| Male | 67 (71) |
| Female | 27 (29) |
| Age, y, median (IQR) | 57 (48–64) |
| Pazopanib starting dose (mg QD) |   |
| 200 | 18 (19) |
| 400 | 38 (40) |
| 600 800 | 3 (3)20 (21) |
| 1000 | 15 (16) |
| Ethnicity |  |
| Caucasian | 91 (97) |
| Asian | 2 (2) |
| Black | 1 (1) |

Table 2 Caption: Patient characteristics

### Effect of Single Nucleotide Polymorphisma on Clearance and Bioavailability

Genotyping details are shown in Table [1](#Tab1). A planned backwards elimination procedure was not performed because only a single association was found to be significantly associated with an endpoint in the primary univariate tests; incorporation of *CYP3A4*22* in the model resulted in a 35% lower clearance for variant carriers (0.18 vs. 0.27 L/h; difference in objective function value: – 9.47; *p* = 0.002). Effects of the other four SNPs on clearance and bioavailability are presented in Table [3](#Tab3).

### Table 3.

| Factor | Compared genotypes | PK parameter | Parameter estimate (RSE %) | p value (ΔOFV) |
| --- | --- | --- | --- | --- |
| ABCG2 421C > A | WT/WT vs. other | FCL | Other: F *1.02WT/WT: 0.27 L/h, other: 0.28 L/ha | 0.78 (0.08) |
| 0.67 (0.18)a |  |  |  |  |
| ABCG2 34G >A | WT/WT vs. WT/var | FCL | WT/var: F *1.24WT/WT: 0.27 L/h, WT/var: 0.26 L/h | 0.17 (1.87) |
| 0.86 (0.03) |  |  |  |  |
| ABCB1 3435C >T | WT/WT vs. other | FCL | Other: F *1.17aWT/WT: 0.28 L/h, other: 0.27 L/h | 0.21a (1.58) |
| 0.64 (0.22) |  |  |  |  |
| ABCB1 3435C >T | var/var vs. other | FCL | Var/var: F*1.01Var/var: 0.28 L/h, other: 0.25 L/h | 0.89 (0.02) |
| 0.40 (0.71) |  |  |  |  |
| CYP3A4*22 | WT/WT vs. WT/var | FCL | Var: F*0.81WT/WT: 0.27 L/h (23%), WT/var: 0.18 L/h (17%) | 0.16 (2.00) |
| 0.002 (9.47) |  |  |  |  |

Table 3 Caption: Effects of genotypes on parameter estimates

Incorporation of the *CYP3A4*22* in the model (Fig. [1](#Fig1)) slightly reduced the inter-individual variability in clearance from 31 to 27%. Parameter estimates and between-subject variability of other pharmacokinetic parameters remained largely unaffected (Table [4](#Tab4)).

### Table 4.

| Parameters | Unit | Base model estimate [RSE (%)] [16] | Final model estimates [RSE (%)] |
| --- | --- | --- | --- |
| Fast absorption rate constant | h−1 | 0.38 (32) | 0.39 (23) |
| Slow absorption rate constant | h−1 | 0.12 (28) | 0.12 (28) |
| Fraction of fast absorption | % | 35 (34) | 36 (36) |
| Fraction of slow absorption | % | 65 (34) | 64 (36) |
| Lag time between fast and slow absorption | h | 0.98 (6) | 0.97 (6) |
| Clearance | L/h | 0.27 (23) | 0.28 (23) |
| Clearance CYP3A4*22 | L/h |   | 0.18 (18) |
| Volume of distribution of central compartment | L | 2.39 (35) | 2.4 (34) |
| Inter-compartment clearance | L/h | 0.99 (29) | 0.97 (29) |
| Volume of distribution of peripheral compartment | L | 25.3 (27) | 24.3 (28) |
| Dose level with half of bioavailability at dose 200 mg | mg | 478 (23) | 466 (23) |
| Magnitude of decrease in relative bioavailability in time | % | 49.7 (27) | 51 (27) |
| First-order decay constant | Day−1 | 0.15 (44) | 0.16 (44) |
| Between-subject variability |  |  |  |
| Fast absorption rate constant | CV% | 142 (20) | 142 (20) |
| Clearance | CV% | 31 (20) | 27 (28) |
| Volume of distribution of peripheral compartment | CV% | 98 (17) | 98 (18) |
| Relative bioavailability | CV% | 36 (16) | 38 (15) |
| Within-subject variability |  |  |  |
| Relative bioavailability | CV% | 74 (22) | 76 (22) |
| Residual unexplained variability |  |  |  |
| Proportional residual error | CV% | 8.0 (6) | 8.0 (11) |
| Additive residual error | mg/L | 3.0 (26) | 2.91 (27) |

Table 4 Caption: Parameter estimates of pazopanib

To translate our findings to clinical practice, the final model (including *CYP3A4*22*) was used for the simulation of several dosing regimens for *CYP3A4*22* carriers. Next, we compared these with the 600- and 800-mg once-daily dose for non-carriers. The median trough concentrations for the *CYP3A4* wild-type population after 400, 600, or 800 mg once daily were 16, 18 and 20 mg/L, respectively (Fig. [2](#Fig2); Table [1](#Tab1) of the ESM). The median trough concentrations for the *CYP3A4*22* heterozygotes with 400, 600, or 800 mg once daily were 27, 31, and 35 mg/L, respectively.

## Discussion

In the current analysis, we found that *CYP3A4*22* is associated with a significantly lower pazopanib clearance. Our simulations estimate that trough concentrations at steady state are more than 50% higher for these patients, compared with patients who are wild type for this gene. This percentage is deemed clinically relevant, as this is not an uncommon mutation in the European population (minor allele frequency 5% [[24](#CR24)]) and *CYP3A4*22* carriers could be more prone to develop toxicity compared with wild-type patients.

Five years previously, it was shown that *CYP3A4*22* is associated with a lower expression of the CYP3A4 enzyme, which resulted in decreased CYP3A4-dependent activity in the liver [[25](#CR25)]. This reduced CYP3A4 activity has been shown to be associated with altered pharmacokinetics of several drugs, e.g., tacrolimus, midazolam, and erythromycin [[26](#CR26)], and to be associated with an increased risk of side effects with for example, cyclosporine and paclitaxel [[27](#CR27), [28](#CR28)]. Furthermore, the clearance of sunitinib, another vascular endothelial growth factor receptor tyrosine kinase inhibitor, was found to be approximately 23% lower in *CYP3A4*22* carriers [[15](#CR15)]. In transplant patients, for whom it is important to quickly dose optimally to prevent organ rejection, this SNP has been used to predict tacrolimus exposure and to adjust its starting dose depending on the genotype [[29](#CR29), [30](#CR30)]. In contrast, in drugs that are also metabolized by CYP2D6, such as tamoxifen and several antipsychotics, *CYP3A4*22* appears to be of less importance [[31](#CR31), [32](#CR32)].

The investigated SNPs in *ABCB1* and *ABCG2* were not significantly associated with pazopanib bioavailability or clearance, although both proteins encoded by these genes are located on intestinal enterocytes and on hepatocytes. Possibly, the full contribution of these transporters to pazopanib pharmacokinetics is not covered by the selected polymorphisms or is not reflected by bioavailability or clearance. However, given the fact that little to no literature on these transporters and pazopanib pharmacokinetics is available, apart from the summary of product characteristics, it is likely that their contribution to pazopanib pharmacokinetics is rather small.

Given the limited number of investigated patients in our study, only a few SNPs could be selected to have sufficient statistical power for the analyses. Although the analyses might therefore have not been comprehensive, we deemed that the investigated SNPs were the most likely candidates to cause relevant pharmacokinetic alterations, based on the currently available literature. As reported previously [[16](#CR16)], bootstrapping is not a suitable method for validating this model given its complexity and the different sampling schedules used.

Furthermore, clinical outcome (i.e., toxicity and survival) was not assessed in this analysis. As the pharmacokinetic data were collected from different phase I and pharmacokinetic studies, the study population was largely heterogeneous regarding confounding factors such as pazopanib dose, tumor type, disease stage, and concomitant (anti-cancer) medication. These factors should be taken into account to determine true effects on clinical outcome. The effect of *CYP3A4*22* on the effect of pazopanib therefore remains to be investigated further in prospective clinical trials, but our simulations show that a dose reduction to 600 mg daily in *CYP3A4*22* carriers provided even higher pazopanib trough concentrations than the approved 800-mg dose in wild-type patients, which potentially makes them more susceptible to toxicity.

## Conclusion

*CYP3A4*22* carriers have a significantly reduced pazopanib clearance, which potentially makes them more susceptible to (severe) toxicity. Simulations suggest that dosing these patients at pazopanib 600 mg daily leads to adequate pazopanib trough concentrations, higher than those in wild-type patients dosed at 800 mg daily. Future studies should investigate whether dose reductions also lead to less toxicity, without compromising the anti-cancer effect in this specific group of patients.

## Electronic supplementary material

Below is the link to the electronic supplementary material.

## Acknowledgements

The authors thank Djoeke de Wit for her assistance in collecting clinical data, and Daan Hurkmans for designing Fig. [1](#Fig1). The authors also thank Roxanne C. Jewell (GSK/Novartis) and Gaaled Haj Mohammad (GSK/Novartis) for providing raw pazopanib data from two clinical trials (VEG109601 [[1](#CR1)] and VEG110725 [[2](#CR2)]).

### Funding

### Funding

No sources of funding were received for the preparation of this article.

### Conflict of interest

### Conflict of interest

Huixin Yu is currently employed by Novartis. All her contributions to this article were made before that employment. Nielka van Erp received grants from Astellas, Janssen-Cilag BV, Novartis, GSK, Boehringer-Ingelheim, Ipsen, Roche, Pfizer, Gilead, and Sanofi and payment for lectures by Novartis, Bayer, and Sanofi. Ron H.J. Mathijssen received grants and payment for lectures by Novartis. Stijn L.W. Koolen received payment for lectures by Novartis. Sander Bins, Alwin D.R. Huitema, Pim Laven, Samira el Bouazzaoui, Carla van Herpen, Paul Hamberg, Hans Gelderblom, Neeltje Steeghs, Stefan Sleijfer, and Ron H.N. van Schaik have no conflicts of interest that are directly relevant to the contents of this article.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Hamberg P, Mathijssen RH, de Bruijn P, Leonowens C, van der Biessen D, Eskens FA, et al. Impact of pazopanib on docetaxel exposure: results of a phase I combination study with two different docetaxel schedules. Cancer Chemother Pharmacol. 2015;75(2):365–371. doi: 10.1007/s00280-014-2655-x.  [DOI](https://doi.org/10.1007/s00280-014-2655-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25533184/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Impact%20of%20pazopanib%20on%20docetaxel%20exposure:%20results%20of%20a%20phase%20I%20combination%20study%20with%20two%20different%20docetaxel%20schedules&author=P%20Hamberg&author=RH%20Mathijssen&author=P%20de%20Bruijn&author=C%20Leonowens&author=D%20van%20der%20Biessen&volume=75&issue=2&publication_year=2015&pages=365-371&pmid=25533184&doi=10.1007/s00280-014-2655-x&)

2. Hamberg P, Boers-Sonderen MJ, van der Graaf WT, de Bruijn P, Suttle AB, Eskens FA, et al. Pazopanib exposure decreases as a result of an ifosfamide-dependent drug-drug interaction: results of a phase I study. Br J Cancer. 2014;110(4):888–893. doi: 10.1038/bjc.2013.798.  [DOI](https://doi.org/10.1038/bjc.2013.798) | [PMC free article](/articles/PMC3929878/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24366297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Pazopanib%20exposure%20decreases%20as%20a%20result%20of%20an%20ifosfamide-dependent%20drug-drug%20interaction:%20results%20of%20a%20phase%20I%20study&author=P%20Hamberg&author=MJ%20Boers-Sonderen&author=WT%20van%20der%20Graaf&author=P%20de%20Bruijn&author=AB%20Suttle&volume=110&issue=4&publication_year=2014&pages=888-893&pmid=24366297&doi=10.1038/bjc.2013.798&)

3. Sternberg CN, Davis ID, Mardiak J, Szczylik C, Lee E, Wagstaff J, et al. Pazopanib in locally advanced or metastatic renal cell carcinoma: results of a randomized phase III trial. J Clin Oncol. 2010;28(6):1061–1068. doi: 10.1200/JCO.2009.23.9764.  [DOI](https://doi.org/10.1200/JCO.2009.23.9764) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20100962/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Pazopanib%20in%20locally%20advanced%20or%20metastatic%20renal%20cell%20carcinoma:%20results%20of%20a%20randomized%20phase%20III%20trial&author=CN%20Sternberg&author=ID%20Davis&author=J%20Mardiak&author=C%20Szczylik&author=E%20Lee&volume=28&issue=6&publication_year=2010&pages=1061-1068&pmid=20100962&doi=10.1200/JCO.2009.23.9764&)

4. van der Graaf WT, Blay JY, Chawla SP, Kim DW, Bui-Nguyen B, Casali PG, et al. Pazopanib for metastatic soft-tissue sarcoma (PALETTE): a randomised, double-blind, placebo-controlled phase 3 trial. Lancet. 2012;379(9829):1879–1886. doi: 10.1016/S0140-6736(12)60651-5.  [DOI](https://doi.org/10.1016/S0140-6736(12)60651-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22595799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Pazopanib%20for%20metastatic%20soft-tissue%20sarcoma%20(PALETTE):%20a%20randomised,%20double-blind,%20placebo-controlled%20phase%203%20trial&author=WT%20van%20der%20Graaf&author=JY%20Blay&author=SP%20Chawla&author=DW%20Kim&author=B%20Bui-Nguyen&volume=379&issue=9829&publication_year=2012&pages=1879-1886&pmid=22595799&doi=10.1016/S0140-6736(12)60651-5&)

5. Suttle AB, Ball HA, Molimard M, Hutson TE, Carpenter C, Rajagopalan D, et al. Relationships between pazopanib exposure and clinical safety and efficacy in patients with advanced renal cell carcinoma. Br J Cancer. 2014;111(10):1909–1916. doi: 10.1038/bjc.2014.503.  [DOI](https://doi.org/10.1038/bjc.2014.503) | [PMC free article](/articles/PMC4229638/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25349968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Relationships%20between%20pazopanib%20exposure%20and%20clinical%20safety%20and%20efficacy%20in%20patients%20with%20advanced%20renal%20cell%20carcinoma&author=AB%20Suttle&author=HA%20Ball&author=M%20Molimard&author=TE%20Hutson&author=C%20Carpenter&volume=111&issue=10&publication_year=2014&pages=1909-1916&pmid=25349968&doi=10.1038/bjc.2014.503&)

6. Verheijen RB, Swart LE, Beijnen JH, Schellens JHM, Huitema ADR, Steeghs N. Exposure-survival analyses of pazopanib in renal cell carcinoma and soft tissue sarcoma patients: opportunities for dose optimization. Cancer Chemother Pharmacol. 2017;80(6):1171–1178. doi: 10.1007/s00280-017-3463-x.  [DOI](https://doi.org/10.1007/s00280-017-3463-x) | [PMC free article](/articles/PMC5686255/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29051995/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Exposure-survival%20analyses%20of%20pazopanib%20in%20renal%20cell%20carcinoma%20and%20soft%20tissue%20sarcoma%20patients:%20opportunities%20for%20dose%20optimization&author=RB%20Verheijen&author=LE%20Swart&author=JH%20Beijnen&author=JHM%20Schellens&author=ADR%20Huitema&volume=80&issue=6&publication_year=2017&pages=1171-1178&pmid=29051995&doi=10.1007/s00280-017-3463-x&)

7. Verheijen RB, Bins S, Mathijssen RH, Lolkema MP, van Doorn L, Schellens JH, et al. Individualized pazopanib dosing: a prospective feasibility study in cancer patients. Clin Cancer Res. 2016;22(23):5738–5746. doi: 10.1158/1078-0432.CCR-16-1255.  [DOI](https://doi.org/10.1158/1078-0432.CCR-16-1255) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27470967/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Individualized%20pazopanib%20dosing:%20a%20prospective%20feasibility%20study%20in%20cancer%20patients&author=RB%20Verheijen&author=S%20Bins&author=RH%20Mathijssen&author=MP%20Lolkema&author=L%20van%20Doorn&volume=22&issue=23&publication_year=2016&pages=5738-5746&pmid=27470967&doi=10.1158/1078-0432.CCR-16-1255&)

8. Soomers V, Desar IME, van Erp NP, Verwiel J, Kaal SEJ, van der Graaf WTA. Fatal heart failure in a young adult female sarcoma patient treated with pazopanib. Acta Oncol. 2017;56(9):1233–1234. doi: 10.1080/0284186X.2017.1296582.  [DOI](https://doi.org/10.1080/0284186X.2017.1296582) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28537442/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Oncol&title=Fatal%20heart%20failure%20in%20a%20young%20adult%20female%20sarcoma%20patient%20treated%20with%20pazopanib&author=V%20Soomers&author=IME%20Desar&author=NP%20van%20Erp&author=J%20Verwiel&author=SEJ%20Kaal&volume=56&issue=9&publication_year=2017&pages=1233-1234&pmid=28537442&doi=10.1080/0284186X.2017.1296582&)

9. Heath EI, Chiorean EG, Sweeney CJ, Hodge JP, Lager JJ, Forman K, et al. A phase I study of the pharmacokinetic and safety profiles of oral pazopanib with a high-fat or low-fat meal in patients with advanced solid tumors. Clin Pharmacol Ther. 2010;88(6):818–823. doi: 10.1038/clpt.2010.199.  [DOI](https://doi.org/10.1038/clpt.2010.199) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20980999/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=A%20phase%20I%20study%20of%20the%20pharmacokinetic%20and%20safety%20profiles%20of%20oral%20pazopanib%20with%20a%20high-fat%20or%20low-fat%20meal%20in%20patients%20with%20advanced%20solid%20tumors&author=EI%20Heath&author=EG%20Chiorean&author=CJ%20Sweeney&author=JP%20Hodge&author=JJ%20Lager&volume=88&issue=6&publication_year=2010&pages=818-823&pmid=20980999&doi=10.1038/clpt.2010.199&)

10. Lubberman F, Gelderblom H, Jansman F, Colbers A, Van der Graaf WTA, Mulders P, et al. Food intervention to make therapy with pazopanib more patient-friendly and affordable. J Clin Oncol. 2016;34(15_suppl):11040. doi: 10.1200/JCO.2016.34.15_suppl.11040.  [DOI](https://doi.org/10.1200/JCO.2016.34.15_suppl.11040) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol.&title=Food%20intervention%20to%20make%20therapy%20with%20pazopanib%20more%20patient-friendly%20and%20affordable&author=F%20Lubberman&author=H%20Gelderblom&author=F%20Jansman&author=A%20Colbers&author=WTA%20Van%20der%20Graaf&volume=34&issue=15_suppl&publication_year=2016&pages=11040&doi=10.1200/JCO.2016.34.15_suppl.11040&)

11. Tan AR, Gibbon DG, Stein MN, Lindquist D, Edenfield JW, Martin JC, et al. Effects of ketoconazole and esomeprazole on the pharmacokinetics of pazopanib in patients with solid tumors. Cancer Chemother Pharmacol. 2013;71(6):1635–1643. doi: 10.1007/s00280-013-2164-3.  [DOI](https://doi.org/10.1007/s00280-013-2164-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23636448/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Effects%20of%20ketoconazole%20and%20esomeprazole%20on%20the%20pharmacokinetics%20of%20pazopanib%20in%20patients%20with%20solid%20tumors&author=AR%20Tan&author=DG%20Gibbon&author=MN%20Stein&author=D%20Lindquist&author=JW%20Edenfield&volume=71&issue=6&publication_year=2013&pages=1635-1643&pmid=23636448&doi=10.1007/s00280-013-2164-3&)

12. Imbs DC, Dieras V, Bachelot T, Campone M, Isambert N, Joly F, et al. Pharmacokinetic interaction between pazopanib and cisplatin regimen. Cancer Chemother Pharmacol. 2016;77(2):385–392. doi: 10.1007/s00280-015-2953-y.  [DOI](https://doi.org/10.1007/s00280-015-2953-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26779916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Pharmacokinetic%20interaction%20between%20pazopanib%20and%20cisplatin%20regimen&author=DC%20Imbs&author=V%20Dieras&author=T%20Bachelot&author=M%20Campone&author=N%20Isambert&volume=77&issue=2&publication_year=2016&pages=385-392&pmid=26779916&doi=10.1007/s00280-015-2953-y&)

13. Maitland ML, Xu CF, Cheng YC, Kistner-Griffin E, Ryan KA, Karrison TG, et al. Identification of a variant in KDR associated with serum VEGFR2 and pharmacodynamics of pazopanib. Clin Cancer Res. 2015;21(2):365–372. doi: 10.1158/1078-0432.CCR-14-1683.  [DOI](https://doi.org/10.1158/1078-0432.CCR-14-1683) | [PMC free article](/articles/PMC4323272/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25411163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Identification%20of%20a%20variant%20in%20KDR%20associated%20with%20serum%20VEGFR2%20and%20pharmacodynamics%20of%20pazopanib&author=ML%20Maitland&author=CF%20Xu&author=YC%20Cheng&author=E%20Kistner-Griffin&author=KA%20Ryan&volume=21&issue=2&publication_year=2015&pages=365-372&pmid=25411163&doi=10.1158/1078-0432.CCR-14-1683&)

14. Xu CF, Johnson T, Wang X, Carpenter C, Graves AP, Warren L, et al. HLA-B*57:01 confers susceptibility to pazopanib-associated liver injury in patients with cancer. Clin Cancer Res. 2016;22(6):1371–1377. doi: 10.1158/1078-0432.CCR-15-2044.  [DOI](https://doi.org/10.1158/1078-0432.CCR-15-2044) | [PMC free article](/articles/PMC7444994/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26546620/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=HLA-B*57:01%20confers%20susceptibility%20to%20pazopanib-associated%20liver%20injury%20in%20patients%20with%20cancer&author=CF%20Xu&author=T%20Johnson&author=X%20Wang&author=C%20Carpenter&author=AP%20Graves&volume=22&issue=6&publication_year=2016&pages=1371-1377&pmid=26546620&doi=10.1158/1078-0432.CCR-15-2044&)

15. Diekstra MH, Klumpen HJ, Lolkema MP, Yu H, Kloth JS, Gelderblom H, et al. Association analysis of genetic polymorphisms in genes related to sunitinib pharmacokinetics, specifically clearance of sunitinib and SU12662. Clin Pharmacol Ther. 2014;96(1):81–89. doi: 10.1038/clpt.2014.47.  [DOI](https://doi.org/10.1038/clpt.2014.47) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24566734/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Association%20analysis%20of%20genetic%20polymorphisms%20in%20genes%20related%20to%20sunitinib%20pharmacokinetics,%20specifically%20clearance%20of%20sunitinib%20and%20SU12662&author=MH%20Diekstra&author=HJ%20Klumpen&author=MP%20Lolkema&author=H%20Yu&author=JS%20Kloth&volume=96&issue=1&publication_year=2014&pages=81-89&pmid=24566734&doi=10.1038/clpt.2014.47&)

16. Yu H, van Erp N, Bins S, Mathijssen RH, Schellens JH, Beijnen JH, et al. Development of a pharmacokinetic model to describe the complex pharmacokinetics of pazopanib in cancer patients. Clin Pharmacokinet. 2017;56(3):293–303. doi: 10.1007/s40262-016-0443-y.  [DOI](https://doi.org/10.1007/s40262-016-0443-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27534647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Development%20of%20a%20pharmacokinetic%20model%20to%20describe%20the%20complex%20pharmacokinetics%20of%20pazopanib%20in%20cancer%20patients&author=H%20Yu&author=N%20van%20Erp&author=S%20Bins&author=RH%20Mathijssen&author=JH%20Schellens&volume=56&issue=3&publication_year=2017&pages=293-303&pmid=27534647&doi=10.1007/s40262-016-0443-y&)

17. de Wit D, van Erp NP, den Hartigh J, Wolterbeek R, den Hollander-van Deursen M, Labots M, et al. Therapeutic drug monitoring to individualize the dosing of pazopanib: a pharmacokinetic feasibility study. Ther Drug Monit. 2015;37(3):331–338. doi: 10.1097/FTD.0000000000000141.  [DOI](https://doi.org/10.1097/FTD.0000000000000141) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25271729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Therapeutic%20drug%20monitoring%20to%20individualize%20the%20dosing%20of%20pazopanib:%20a%20pharmacokinetic%20feasibility%20study&author=D%20de%20Wit&author=NP%20van%20Erp&author=J%20den%20Hartigh&author=R%20Wolterbeek&author=Deursen%20M%20den%20Hollander-van&volume=37&issue=3&publication_year=2015&pages=331-338&pmid=25271729&doi=10.1097/FTD.0000000000000141&)

18. Elens L, van Gelder T, Hesselink DA, Haufroid V, van Schaik RH. CYP3A4*22: promising newly identified CYP3A4 variant allele for personalizing pharmacotherapy. Pharmacogenomics. 2013;14(1):47–62. doi: 10.2217/pgs.12.187.  [DOI](https://doi.org/10.2217/pgs.12.187) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23252948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics.&title=CYP3A4*22:%20promising%20newly%20identified%20CYP3A4%20variant%20allele%20for%20personalizing%20pharmacotherapy&author=L%20Elens&author=T%20van%20Gelder&author=DA%20Hesselink&author=V%20Haufroid&author=RH%20van%20Schaik&volume=14&issue=1&publication_year=2013&pages=47-62&pmid=23252948&doi=10.2217/pgs.12.187&)

19. Qin C, Cao Q, Li P, Wang S, Wang J, Wang M, et al. The influence of genetic variants of sorafenib on clinical outcomes and toxic effects in patients with advanced renal cell carcinoma. Sci Rep. 2016;6:20089. doi: 10.1038/srep20089.  [DOI](https://doi.org/10.1038/srep20089) | [PMC free article](/articles/PMC4735712/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26830973/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep.&title=The%20influence%20of%20genetic%20variants%20of%20sorafenib%20on%20clinical%20outcomes%20and%20toxic%20effects%20in%20patients%20with%20advanced%20renal%20cell%20carcinoma&author=C%20Qin&author=Q%20Cao&author=P%20Li&author=S%20Wang&author=J%20Wang&volume=6&publication_year=2016&pages=20089&pmid=26830973&doi=10.1038/srep20089&)

20. Kim DH, Sriharsha L, Xu W, Kamel-Reid S, Liu X, Siminovitch K, et al. Clinical relevance of a pharmacogenetic approach using multiple candidate genes to predict response and resistance to imatinib therapy in chronic myeloid leukemia. Clin Cancer Res. 2009;15(14):4750–4758. doi: 10.1158/1078-0432.CCR-09-0145.  [DOI](https://doi.org/10.1158/1078-0432.CCR-09-0145) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19584153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Clinical%20relevance%20of%20a%20pharmacogenetic%20approach%20using%20multiple%20candidate%20genes%20to%20predict%20response%20and%20resistance%20to%20imatinib%20therapy%20in%20chronic%20myeloid%20leukemia&author=DH%20Kim&author=L%20Sriharsha&author=W%20Xu&author=S%20Kamel-Reid&author=X%20Liu&volume=15&issue=14&publication_year=2009&pages=4750-4758&pmid=19584153&doi=10.1158/1078-0432.CCR-09-0145&)

21. Takahashi N, Miura M, Scott SA, Kagaya H, Kameoka Y, Tagawa H, et al. Influence of CYP3A5 and drug transporter polymorphisms on imatinib trough concentration and clinical response among patients with chronic phase chronic myeloid leukemia. J Hum Genet. 2010;55(11):731–737. doi: 10.1038/jhg.2010.98.  [DOI](https://doi.org/10.1038/jhg.2010.98) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20720558/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=Influence%20of%20CYP3A5%20and%20drug%20transporter%20polymorphisms%20on%20imatinib%20trough%20concentration%20and%20clinical%20response%20among%20patients%20with%20chronic%20phase%20chronic%20myeloid%20leukemia&author=N%20Takahashi&author=M%20Miura&author=SA%20Scott&author=H%20Kagaya&author=Y%20Kameoka&volume=55&issue=11&publication_year=2010&pages=731-737&pmid=20720558&doi=10.1038/jhg.2010.98&)

22. Kerbusch T, Huitema AD, Ouwerkerk J, Keizer HJ, Mathot RA, Schellens JH, et al. Evaluation of the autoinduction of ifosfamide metabolism by a population pharmacokinetic approach using NONMEM. Br J Clin Pharmacol. 2000;49(6):555–561. doi: 10.1046/j.1365-2125.2000.00217.x.  [DOI](https://doi.org/10.1046/j.1365-2125.2000.00217.x) | [PMC free article](/articles/PMC2015043/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10848719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Evaluation%20of%20the%20autoinduction%20of%20ifosfamide%20metabolism%20by%20a%20population%20pharmacokinetic%20approach%20using%20NONMEM&author=T%20Kerbusch&author=AD%20Huitema&author=J%20Ouwerkerk&author=HJ%20Keizer&author=RA%20Mathot&volume=49&issue=6&publication_year=2000&pages=555-561&pmid=10848719&doi=10.1046/j.1365-2125.2000.00217.x&)

23. Keizer RJ, van Benten M, Beijnen JH, Schellens JH, Huitema AD. Pirana and PCluster: a modeling environment and cluster infrastructure for NONMEM. Comput Methods Programs Biomed. 2011;101(1):72–79. doi: 10.1016/j.cmpb.2010.04.018.  [DOI](https://doi.org/10.1016/j.cmpb.2010.04.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20627442/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Comput%20Methods%20Programs%20Biomed&title=Pirana%20and%20PCluster:%20a%20modeling%20environment%20and%20cluster%20infrastructure%20for%20NONMEM&author=RJ%20Keizer&author=M%20van%20Benten&author=JH%20Beijnen&author=JH%20Schellens&author=AD%20Huitema&volume=101&issue=1&publication_year=2011&pages=72-79&pmid=20627442&doi=10.1016/j.cmpb.2010.04.018&)

24. Zhou Y, Ingelman-Sundberg M, Lauschke VM. Worldwide distribution of cytochrome P450 alleles: a meta-analysis of population-scale sequencing projects. Clin Pharmacol Ther. 2017;102(4):688–700. doi: 10.1002/cpt.690.  [DOI](https://doi.org/10.1002/cpt.690) | [PMC free article](/articles/PMC5600063/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28378927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Worldwide%20distribution%20of%20cytochrome%20P450%20alleles:%20a%20meta-analysis%20of%20population-scale%20sequencing%20projects&author=Y%20Zhou&author=M%20Ingelman-Sundberg&author=VM%20Lauschke&volume=102&issue=4&publication_year=2017&pages=688-700&pmid=28378927&doi=10.1002/cpt.690&)

25. Okubo M, Murayama N, Shimizu M, Shimada T, Guengerich FP, Yamazaki H. CYP3A4 intron 6 C > T polymorphism (CYP3A4*22) is associated with reduced CYP3A4 protein level and function in human liver microsomes. J Toxicol Sci. 2013;38(3):349–354. doi: 10.2131/jts.38.349.  [DOI](https://doi.org/10.2131/jts.38.349) | [PMC free article](/articles/PMC4018728/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23665933/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Toxicol%20Sci&title=CYP3A4%20intron%206%20C%E2%80%89>%E2%80%89T%20polymorphism%20(CYP3A4*22)%20is%20associated%20with%20reduced%20CYP3A4%20protein%20level%20and%20function%20in%20human%20liver%20microsomes&author=M%20Okubo&author=N%20Murayama&author=M%20Shimizu&author=T%20Shimada&author=FP%20Guengerich&volume=38&issue=3&publication_year=2013&pages=349-354&pmid=23665933&doi=10.2131/jts.38.349&)

26. Elens L, Nieuweboer A, Clarke SJ, Charles KA, de Graan AJ, Haufroid V, et al. CYP3A4 intron 6 C > T SNP (CYP3A4*22) encodes lower CYP3A4 activity in cancer patients, as measured with probes midazolam and erythromycin. Pharmacogenomics. 2013;14(2):137–149. doi: 10.2217/pgs.12.202.  [DOI](https://doi.org/10.2217/pgs.12.202) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23327575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics.&title=CYP3A4%20intron%206%20C%E2%80%89>%E2%80%89T%20SNP%20(CYP3A4*22)%20encodes%20lower%20CYP3A4%20activity%20in%20cancer%20patients,%20as%20measured%20with%20probes%20midazolam%20and%20erythromycin&author=L%20Elens&author=A%20Nieuweboer&author=SJ%20Clarke&author=KA%20Charles&author=AJ%20de%20Graan&volume=14&issue=2&publication_year=2013&pages=137-149&pmid=23327575&doi=10.2217/pgs.12.202&)

27. Elens L, Bouamar R, Hesselink DA, Haufroid V, van Gelder T, van Schaik RH. The new CYP3A4 intron 6 C > T polymorphism (CYP3A4*22) is associated with an increased risk of delayed graft function and worse renal function in cyclosporine-treated kidney transplant patients. Pharmacogenet Genom. 2012;22(5):373–380. doi: 10.1097/FPC.0b013e328351f3c1.  [DOI](https://doi.org/10.1097/FPC.0b013e328351f3c1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22388796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genom&title=The%20new%20CYP3A4%20intron%206%20C%E2%80%89>%E2%80%89T%20polymorphism%20(CYP3A4*22)%20is%20associated%20with%20an%20increased%20risk%20of%20delayed%20graft%20function%20and%20worse%20renal%20function%20in%20cyclosporine-treated%20kidney%20transplant%20patients&author=L%20Elens&author=R%20Bouamar&author=DA%20Hesselink&author=V%20Haufroid&author=T%20van%20Gelder&volume=22&issue=5&publication_year=2012&pages=373-380&pmid=22388796&doi=10.1097/FPC.0b013e328351f3c1&)

28. de Graan AJ, Elens L, Sprowl JA, Sparreboom A, Friberg LE, van der Holt B, et al. CYP3A4*22 genotype and systemic exposure affect paclitaxel-induced neurotoxicity. Clin Cancer Res. 2013;19(12):3316–3324. doi: 10.1158/1078-0432.CCR-12-3786.  [DOI](https://doi.org/10.1158/1078-0432.CCR-12-3786) | [PMC free article](/articles/PMC3686845/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23640974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=CYP3A4*22%20genotype%20and%20systemic%20exposure%20affect%20paclitaxel-induced%20neurotoxicity&author=AJ%20de%20Graan&author=L%20Elens&author=JA%20Sprowl&author=A%20Sparreboom&author=LE%20Friberg&volume=19&issue=12&publication_year=2013&pages=3316-3324&pmid=23640974&doi=10.1158/1078-0432.CCR-12-3786&)

29. Elens L, Capron A, van Schaik RH, De Meyer M, De Pauw L, Eddour DC, et al. Impact of CYP3A4*22 allele on tacrolimus pharmacokinetics in early period after renal transplantation: toward updated genotype-based dosage guidelines. Ther Drug Monit. 2013;35(5):608–616. doi: 10.1097/FTD.0b013e318296045b.  [DOI](https://doi.org/10.1097/FTD.0b013e318296045b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24052064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Impact%20of%20CYP3A4*22%20allele%20on%20tacrolimus%20pharmacokinetics%20in%20early%20period%20after%20renal%20transplantation:%20toward%20updated%20genotype-based%20dosage%20guidelines&author=L%20Elens&author=A%20Capron&author=RH%20van%20Schaik&author=M%20De%20Meyer&author=L%20De%20Pauw&volume=35&issue=5&publication_year=2013&pages=608-616&pmid=24052064&doi=10.1097/FTD.0b013e318296045b&)

30. Woillard JB, Mourad M, Neely M, Capron A, van Schaik RH, van Gelder T, et al. Tacrolimus updated guidelines through popPK modeling: how to benefit more from CYP3A pre-emptive genotyping prior to kidney transplantation. Front Pharmacol. 2017;8:358. doi: 10.3389/fphar.2017.00358.  [DOI](https://doi.org/10.3389/fphar.2017.00358) | [PMC free article](/articles/PMC5462973/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28642710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Pharmacol.&title=Tacrolimus%20updated%20guidelines%20through%20popPK%20modeling:%20how%20to%20benefit%20more%20from%20CYP3A%20pre-emptive%20genotyping%20prior%20to%20kidney%20transplantation&author=JB%20Woillard&author=M%20Mourad&author=M%20Neely&author=A%20Capron&author=RH%20van%20Schaik&volume=8&publication_year=2017&pages=358&pmid=28642710&doi=10.3389/fphar.2017.00358&)

31. van der Weide K, van der Weide J. The influence of the CYP3A4*22 polymorphism and CYP2D6 polymorphisms on serum concentrations of aripiprazole, haloperidol, pimozide, and risperidone in psychiatric patients. J Clin Psychopharmacol. 2015;35(3):228–236. doi: 10.1097/JCP.0000000000000319.  [DOI](https://doi.org/10.1097/JCP.0000000000000319) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25868121/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychopharmacol&title=The%20influence%20of%20the%20CYP3A4*22%20polymorphism%20and%20CYP2D6%20polymorphisms%20on%20serum%20concentrations%20of%20aripiprazole,%20haloperidol,%20pimozide,%20and%20risperidone%20in%20psychiatric%20patients&author=K%20van%20der%20Weide&author=J%20van%20der%20Weide&volume=35&issue=3&publication_year=2015&pages=228-236&pmid=25868121&doi=10.1097/JCP.0000000000000319&)

32. Sanchez Spitman AB, Moes D, Gelderblom H, Dezentje VO, Swen JJ, Guchelaar HJ. Effect of CYP3A4*22, CYP3A5*3, and CYP3A combined genotypes on tamoxifen metabolism. Eur J Clin Pharmacol. 2017;73(12):1589–1598. doi: 10.1007/s00228-017-2323-2.  [DOI](https://doi.org/10.1007/s00228-017-2323-2) | [PMC free article](/articles/PMC5684327/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28849250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Effect%20of%20CYP3A4*22,%20CYP3A5*3,%20and%20CYP3A%20combined%20genotypes%20on%20tamoxifen%20metabolism&author=AB%20Sanchez%20Spitman&author=D%20Moes&author=H%20Gelderblom&author=VO%20Dezentje&author=JJ%20Swen&volume=73&issue=12&publication_year=2017&pages=1589-1598&pmid=28849250&doi=10.1007/s00228-017-2323-2&)
