# IFNL4 haplotype, linkage disequilibrium and their influence on virological response to hepatitis C virus infection in Indian population

## Metadata
**Authors:** Neelanjana Roy, Chandreswar Prasad, Anand Kumar, Krishnendu Mondol, Kajal Jain, Renu Yadav, Jyotish Kumar Jha, Neeti Nadda, Subrat Kumar Acharya, Shalimar, Baibaswata Nayak
**Journal:** VirusDisease
**Date:** 2019 Jul 24
**DOI:** [10.1007/s13337-019-00535-4](https://doi.org/10.1007/s13337-019-00535-4)
**PMID:** 31803800
**PMCID:** PMC6863998
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6863998/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6863998/pdf/13337_2019_Article_535.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6863998/pdf/13337_2019_Article_535.pdf)

## Abstract

Type III interferon (IFNs) encoded by IFN lambda (IFNL) genes induce antiviral activity. The IFNL clusters include IFNL1/IL29, IFNL2/IL28A, IFNL3/IL28B and IFNL4 genes. The single nucleotide polymorphisms (SNPs, rs12979860 and rs8099917) associated with virological responses against hepatitis C virus (HCV) infections are recently mapped to IFNL4 gene. The IFNL gene polymorphisms also plays role in immune clearance, inflammation and risk of developing hepatocellular carcinoma. There is significant genetic heterogeneity of IFNL4 polymorphisms among ethnic populations that need to be regionally studied for viral infection, treatment response and relapse. The IFNL4 risk allele, genotype and haplotype frequencies across north Indian cohort were determined among chronic hepatitis C (CHC) cases (n = 141) and healthy controls (n = 111) by allele specific real-time PCR. Odds ratio was calculated for HCV exposure and treatment response using dominant and minor allele/genotype as reference. Non-random associations of these two SNP loci were evaluated by linkage disequilibrium plot. The minor allele (T) frequency of rs12979860C/T is 0.241 and 0.229; and minor allele (G) frequency for SNP rs8099917T/G is 0.174 and 0.171 among CHC cases and healthy control respectively. Coefficient of linkage disequilibrium (D′) of these two SNPs is very high (D′ = 0.98, r2 > 0.6) in CHC group than in healthy control (D′ = 0.76, r2 = 0.39) which indicate that both SNPs are strongly linked in CHC population than healthy control. Favorable association of IFNL4 haplotype (C–T), genotype (CC for rs12979860 and TT for rs8099917) with anti HCV therapy were found significant (p = 0.009, 0.021 and 0.001) for SVR. Favorable genotypes are also found to be predominant across the Indian study population.

Keywords: HCV, IFNL4, IFN, Virological response, Chronic hepatitis C

## Introduction

Interferons (IFNs) mediate host immune response against viral infections. The IFNs are classified into three (type I, II and III) distinct types on the basis of their structural features, receptor and biological activities. Type I and III IFN mediate innate immune response and type II IFN mediate adaptive immune response [[15](#CR15)]. Type III IFNs are encoded by Interferon lambda (IFNL) cluster of genes located at 19q13 chromosomal region. The IFNL genes are designated as *IFNL1* or *IL29*, *IFNL2* or *IL28A*, *IFNL3 or IL28B* and *IFNL4* genes. These IFNLs exert antiviral activity through a receptor complex consisting of IFNLR1 and IL10R2 by activation of JAK-STAT and AMPK pathways [[2](#CR2), [26](#CR26)]. The type III IFN mediates antiviral activity by inhibiting virus replication or promoting virus clearance. This has been demonstrated against hepatitis C virus (HCV), hepatitis B virus (HBV) and human immune deficiency virus (HIV) infections [[15](#CR15), [18](#CR18), [19](#CR19)].

Genetic association of single nucleotide polymorphisms (SNPs) with HCV infections are mostly related to the host genes responsible for antiviral response or virus entry which either improve treatment induced clearance or promote/impair virus clearance against acute/chronic infections. In a recent study, HCV entry related *SCARB1* (scavenger receptor type B class I) gene polymorphism was found to be associated with HCV RNA level and therapeutic response [[14](#CR14)]. In earlier studies, *IL28b* polymorphisms (rs12979860G/C and rs8099917C/T) were found to be associated with sustained virological response (SVR) to pegylated IFN and ribavirin (PEG-IFN/RBV) therapy in both genome wide and candidate gene approach studies [[10](#CR10), [29](#CR29)]. Recently, the SNP rs12979860 is mapped to be within the first intron of *IFNL4* [[23](#CR23)] and rs8099917 is mapped 3.6 kb upstream to the *IFNL4* gene [[22](#CR22)] and therefore considered *IFNL4* variants. The gene *IFNL3* and *IFNL4* are located at very close (1.2 kb apart) proximity to each other. The dinucleotide polymorphism (TT/ΔG) at rs368234815 SNP locus causes an open reading frame-shift in the *IFNL4* gene resulting in expression of a functional IFN-λ4, a new type III IFN [[23](#CR23)]. The proposed mechanisms for causal association of these *IFNL4* SNPs with SVR are related to increase expression of type III IFN in favorable SNP genotypes [[5](#CR5), [16](#CR16)]. Higher rate of favorable response to antiviral therapy was observed in CHC patients either with CC genotype of rs12979860 or TT genotype of rs8099917 [[10](#CR10), [29](#CR29)]. The risk of hepatocellular carcinoma (HCC) was also observed more in CHC patients with unfavorable or risk genotypes of SNPs rs12979860 and rs8099917 [[17](#CR17), [24](#CR24)]. The favorable haplotype of both SNPs have shown significant association with spontaneous virus clearance and less liver damage [[11](#CR11)] whereas; the risk haplotype have shown significant association with hepatic inflammation and fibrosis in patients with HCV infection [[9](#CR9)]. However, there is significant heterogeneity in these *IFNL4* genotypes and haplotypes. The distribution of these favorable genotypes varies among different ethnic populations such as higher prevalence of favorable genotype CC (rs12979860) in CHC patients of European ancestry than in African-Americans [[10](#CR10), [29](#CR29)]. Significant genetic heterogeneity of these *IFNL4* haplotypes was also observed in CHC patients of West Mexico population [[11](#CR11)].

Therefore, we aimed to evaluate the *IFNL4* genotypes and haplotypes in the heterogeneously populated country as India to understand its influence on the course viral infections, disease outcome and treatment response. Both of these *IFNL4* SNPs were not studied together in healthy and HCV infected patients of Indian population. Whether these SNPs genotypes or derived haplotypes will have any additive role for the risk of disease, protection or as predictor of response need to be evaluated for healthy or chronically HCV infected Indian population.

## Material and methods

### Study population and design

Two hundred and fifty-two consecutive subjects were enrolled for the study, of which 141 were CHC and 111 were healthy controls. We included CHC patients with detectable HCV RNA and those who were treatment eligible for pegylated interferon and ribavirin during the period from March, 2013 to February, 2015. The exclusion criteria for CHC patients were alcohol consumption of more than 20 g per day, co-infection with HBV or HIV. Controls subjects were unrelated healthy individuals, negative for HBV and HCV or HIV; and had normal serum levels of alanine aminotransferase (ALT) and aspartate aminotransferase (AST). Ethical clearance (IEC/NP-37/RP-04.03.2013) from the institute ethics committee was obtained. The case–control association study was designed to determine the risk allele, genotype and haplotype associated with HCV infections and the cohort study design for CHC patients was to assess the association of *IFNL4* with treatment induced viral clearance.

### Sample collection, host genomic DNA and viral nucleic acid isolation

The venous whole blood samples were collected with and without anticoagulant (EDTA) for host genomic DNA and viral nucleic acid isolation respectively. Host genomic DNA was extracted from whole blood using the QIAamp DNA blood mini kit (Qiagen, Germany). The viral nucleic acid was isolated form the sera of CHC patients using QIAsymphony DSP virus/pathogen kit (Cat No. 937036) in Qiasymphony SP automated nucleic acid isolation system.

### HCV RNA quantitation, genotyping and virological response

HCV RNA was quantitated by one step real time PCR using the HCV specific primers and probe HCV 149 Sense: 5′-TGCGGAACCGGTGAGTACA-3′, HCV-342 Antisense: 5′-CTTAAGGTTTAGGATTCGTGCTCAT-3′ and HCV UTR probe: 5′-FAM-CACCCTATCAGGCAGTACCACAAGGCC-BHQ1-3′. Briefly, 20 µl reactions was carried out using 10 µl of 2× One step RT-qPCR-probe master mix (Veriquest), 2 µl of 10× primer–probe mix and 8 µl of HCV RNA. The reaction was carried out in Quantstudio 7 flex (Applied Biosystems, Life Technologies) using following thermal cycle parameters: 50 °C for 17 min; 95 °C for 5 min for one cycle and 50 cycles of 95 °C for 14 s, 60 °C for 1 min. Acrometrix HCV panels (Applied Biosystem) of standards were used for standard curve. HCV genotyping was carried out by AmpliSens HCV-genotype-FRT PCR kit which can detect 1a, 1b, 2, 3a, 4, 5a, 6 genotypes using genotype specific probe hybridization method. Real-time PCR of HCV cDNA was performed in Quantstudio 7 flex (Applied Biosystems, Life Technologies) using parameters such as: 95 °C for 15 min; 5 cycles at 95 °C for 5 s, 60 °C for 20 s, and 72 °C for 15 s and 40 cycles of 95 °C for 5 s, 60 °C for 20 s, and 72 °C for 15 s. The channel for fluorescence detection was FAM for 1a, 1b, 5a and JOE/HEX for 3a, 2, 4, and 6.

Response to IFN/RBV therapy in CHC patients was monitored by quantifying HCV RNA load at regular intervals and decline in HCV RNA load was considered as a strong predictor of response. An undetectable level of HCV RNA (< 50 IU/ml) after 4 weeks of therapy is termed as rapid virological response (RVR), undetectable at 12 weeks is known as complete early virological response (EVR), undetectable at 24 weeks and 48 weeks for genotype 1 and 3 is known as end-of-treatment virological response (ETVR). HCV RNA undetectable at 6 month after ETVR is known as sustained virological response (SVR).

### Genotyping of IFNL4 polymorphisms

*IFNL4* SNP genotyping was carried out by SYBR green based Real-time PCR using amplification-refractory mutation system (ARMS) PCR primers and confirmed by single/multiplex fragment analysis. The ARMS-PCR based detection employed a set of three primers for allelic discrimination at dbSNP sites (rs12979860C/T: rs12F-GGAAGGAGCAGTTGCGCT, rs12R/C-GCAATTCAACCCTGGTTCG and rs12R/T-GCAATTCAACCCTGGTTCA; rs8099917G/T: rs80F-CCACTTCTGGAACAAATCGTC, rs80R/G-GGTTCCAATTTGGGTGAC and rs80R/T-GGTTCCAATTTGGGTGAA). For each of the two SNPs (rs12979860 C/T and rs8099917 T/G), two sets of real-time PCR reactions were performed at different annealing temperature and cyclic conditions. Both ARMS-PCR and Real-time PCR was carried out using parameters such as: 95 °C for 5 min; 24 cycles at 95 °C for 30 s, 56 °C (rs8099917) and 60 °C (rs12979860) for 30 s, and 72 °C for 60 s. Confirmation of SNP genotype by singleplexing or/multiplexing was carried out by single base primer extension using Genome Lab SNP Start primer extension Kit in Genome Lab™ GeXP Genetic Analysis System (Beckman and coulter, USA). Briefly, the regions spanning both SNPs were amplified using sense and antisense primer (rs12979860: 315F-GCCCCCAGCTCAGCGCCTCTTCCT and 639R-ACCCCCGCAGTCCCCGTCGTCAG; rs8099917: 325F-TTGTGCATATGTTTTCTGACTA CC and 642R-AATTGACGGGCCATCTGTTTCCTG). Single or multiplexing for these SNPs were carried for 6 bp peak differences using interrogation primers (rs12979860 C/T-CTGAACCAGGGAGCTCCCCGAAGGCG and rs8099917 G/T-AAAAAAACTCCTTTTGTTTTCC TGTGAGCAAT).

### Statistical analysis

Genotypic and allelic frequencies were calculated by direct counting method. The categorical variables were expressed as frequency/percentage and estimation of the significance of differences between categorical variables were compared by Pearson’s Chi square test. Means of different variables between any two groups were compared with Student’s *t* test. Further statistical analyses were performed using the SPSS software (version 20.0) (SPSS, Inc, Chicago, IL).The odds ratio (OR) was calculated with 95% confidence interval using GraphPad Prism (version 3.0) (GraphPad Software Inc., San Diego, Ca, USA). A *p* value < 0.05 is considered statistically significant. Hardy–Weinberg equilibrium (HWE) testing was carried out to determine whether individual loci or SNP across the population is in equilibrium or under influence of any evolutionary forces. HWE and haplotype inference as well as Linkage disequilibrium (LD) and r^2^ were analyzed with HAPLOVIEW (version 4.1) (Dr. Mark Daly’s lab, MIT/Harvard Broad Institute).

## Results

### Study enrolment and patient characteristics

The mean age of CHC patients (n = 141) and healthy controls (n = 111) were 39.6 ± 13.4 year and 34.5 ± 10.6 year, respectively. The percentage of male in CHC and healthy controls were 64.6% (n = 91) and 64.9% (n = 72) respectively. In our recruited CHC patients, prevalence of HCV genotype 3 is highest (71.63%, n = 101), followed by genotype 1 (24.82%, n = 35), 4 (2.84%, n = 4) and 2 (0.71% n = 1). The baseline mean viral load of respective genotypes are 1.03 × 10^8^ (genotype 1), 4 × 10^4^ (genotype 2), 5.69 × 10^6^ (genotype 3) and 3.42 × 10^6^ (genotype 4) IU/ml respectively. The mean value of liver enzyme AST and ALT in CHC patients were 103.1 ± 97.77 U/ml and 128.4 ± 222.8 U/ml respectively.

### Frequency distribution and Hardy–Weinberg equilibrium testing of IFNL4 genotypes and alleles among CHC patients and healthy controls

The SNP genotyping was carried out by allele specific ARMS-PCR (Fig. [1](#Fig1)a, b) and SYBR green based real time PCR. The selected genotypes were confirmed by fragment analysis of primer extension product (Fig. [1](#Fig1)c–f). The observed SNP rs12979860C/T genotype and allele frequencies are 0.589 (CC), 0.340 (CT), 0.071 (TT), 0.759 (C allele) and 0.241 (T allele) for CHC cases (n = 141); and for healthy controls, genotype and allele frequencies are 0.604 (CC), 0.333 (CT), 0.063 (TT), 0.771 (C allele) and 0.229 (T allele) respectively (Table [1](#Tab1)). The observed SNP rs8099917 T/G genotype and allele frequencies for CHC case and healthy controls are 0.688 and 0.713 (TT), 0.277 and 0.231 (GT), 0.035 and 0.056 (GG), 0.826 and 0.829 (T allele) and 0.174 and 0.171 (G allele) respectively (Table [1](#Tab1)).

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/04f2/6863998/e2d0761b322c/13337_2019_535_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6863998_13337_2019_535_Fig1_HTML.jpg)

IFNL4 SNP genotyping: a Genotyping of rs12979860 of IL28b gene by ARMS-PCR with rs12F, rs12R/C and rs12R/T; PCR was carried out in two tubes using a common primer (rs12F) and allele discriminating primer (Tube 1-rs12F and rs12R/C; Tube 2-rs12F and rs12R/T). Single band in gel from tube 1 in sample A and B designated as homozygous CC genotype; single band from tube 2 in sample E and F designated as homozygous TT genotype; whereas both band in sample C and D designated as heterozygous CT genotype. b Genotyping of rs8099917 of IL28b gene by ARMS-PCR using a common primer (rs80F) and allele discriminating primer (Tube 1-rs80F and rs80/G; Tube 2-rs80F and rs80/T) and SNP gentyping was interpreted the ways mentioned above; c, d Singleplex for rs12979860 and rs8099917; e Multiplex assay (CC genotype for rs12979860 and TT for rs8099917); f Multiplex assay (CT genotype for rs12979860 and TT for rs8099917)

### Table 1.

| Genotype | Observed frequency (n) |  | Expected frequency (n)a |  | Chi square (χ2) |  |
| --- | --- | --- | --- | --- | --- | --- |
| CHC | Healthy | CHC | Healthy | CHC | Healthy |  |
| Genotype and allele frequency distribution of rs12979860 in CHC patient and healthy control |  |  |  |  |  |  |
| CC | 0.589 (83) | 0.604 (67) | 0.576 (81.19) | 0.593 (65.84) | 0.040 | 0.020 |
| CT | 0.340 (48) | 0.333 (37) | 0.366 (51.60) | 0.354 (39.28) | 0.251 | 0.132 |
| TT | 0.071(10) | 0.063 (7) | 0.058 (8.21) | 0.053 (5.87) | 0.390 | 0.218 |
| C allele | 0.759 (214) | 0.771 (171) |   |   | HWE-p value (1 df) |  |
| T allele | 0.241(68) | 0.229 (51) |   |   | p = 0.409 | p = 0.542 |
| Genotype and allele frequency distribution of rs8099917 in CHC patient and healthy control |  |  |  |  |  |  |
| TT | 0.688 (97) | 0.713 (77) | 0.683 (96.25) | 0.687 (74.16) | 0.006 | 0.109 |
| GT | 0.277 (39) | 0.231 (25) | 0.287 (40.49) | 0.284 (30.66) | 0.055 | 1.045 |
| GG | 0.035 (5) | 0.056 (6) | 0.030 (4.26) | 0.029 (3.18) | 0.129 | 1.325 |
| T allele | 0.826 (233) | 0.829 (179) |   |   | HWE-p value (1 df) |  |
| G allele | 0.174 (49) | 0.171 (37) |   |   | p = 0.663 | p = 0.115 |

Table 1 Caption: Hardy–Weinberg Equilibrium testing and frequency distribution of IFNL4 (rs12979860 and rs8099917) allele, genotype and haplotype

Hardy–Weinberg equilibrium (HWE) testing was carried out using ternary plot of De Finetti diagram which uses genotype and allele frequencies of populations. Hardy–Weinberg parabola in this diagram defines the proportions that are in HWE. In Fig. [2](#Fig2), the dotted black line represents healthy control and blue line CHC cases for each SNP (Fig. [2](#Fig2)A, B). The common joining points of three genotypes of SNPs for health and CHC cases are under the curve indicating that SNPs rs12979860 and rs8099917 are satisfying HWE. This indicates that the allele or genotype frequencies of these SNPs do not change from generation to generation under the pressure of evolutionary influences like mutation, genetic drift, migration or non-random mating. The test of deviations from HWE was carried out by Chi square test by comparing observed and expected genotype frequencies (Table [1](#Tab1)) for each SNP. The HWE *p* value (1 *df*) for each genotype was found to be not significant for SNPs rs12979860 (*p* = 0.409 and 0.542) and rs8099917 (*p* = 0.663 and 0.115) in both CHC cases and healthy controls respectively (Table [1](#Tab1)).

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/04f2/6863998/abb21b90084c/13337_2019_535_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6863998_13337_2019_535_Fig2_HTML.jpg)

Hardy–Weinberg testing and Linkage Disequilibrium of IFNL4 polymorphisms: De Finetti diagram of rs12979860 (A) and rs8099917 (B) with Hardy Weinberg Parabola and C Linkage Disequilibrium Profile between rs12979860 and rs8099917 of IL28b gene. (a) Overall, (b) Cases, (c) Control

Four haplotypes (C–T, T–G, T–T and C–G) were constructed from two SNP (rs12979860C/T and rs8099917 T/G) loci of *IFNL4* gene. The haplotype frequencies among CHC and healthy controls were calculated by Haploview using total number of chromosome (N) as mentioned in Table [1](#Tab1). The C–T haplotype was found to be most common and had high prevalence in both the CHC (75.5%) groups and healthy (73.9%) population whereas haplotype C–G is least common.

### Genetic association of IFNL4 allele, genotype and haplotype with HCV exposure

To test genetic association of SNP with HCV exposure, we evaluated odds ratio (OR) of genotypes and alleles between CHC cases and in healthy control group. Assuming dominant allele (C) and genotype (CC) of the SNP rs12979860 (C/T) as reference or protective, the OR for risk allele (T) and genotype (TT, CT and CT + TT) was calculated (Table [2](#Tab2)). For SNP rs8099917 OR for risk allele G versus T as reference and OR for risk genotypes GT, GG and GT + GG against dominant TT genotype as reference was calculated. The *p* values for all are > 0.05 and not significant (Table [2](#Tab2)).

### Table 2.

| Locus ID | Genotype | CHC/Healthy (n) | Odds Ratio, 95% CI | p value* | Odds Ratio, 95% CI | p value* |
| --- | --- | --- | --- | --- | --- | --- |
| rs12979860 | TT | 10/7 | Reference |   | 1.15, 0.42–3.19 | 0.78 |
| CT | 48/37 | 0.91, 0.31–2.40 | 0.86 | 1.04, 0.61–1.79 | 0.86 |  |
| CC | 83/67 | 0.87, 0.32–2.61 | 0.78 | Reference |   |  |
| CT + CC | 131/104 | 0.88, 0.32–2.40 | 0.81 |   |   |  |
| CT + TT | 58/44 |   |   | 1.06, 0.64–1.76 | 0.81 |  |
| T allele | 68/51 | Reference |   | 1.06, 0.70–1.61 | 0.83 |  |
| C allele | 214/171 | 0.94, 0.62–1.42 | 0.77 | Reference |   |  |
| rs8099917 | GG | 5/6 | Reference |   | 0.66, 0.19–2.25 | 0.51 |
| GT | 39/25 | 1.87, 0.52–6.79 | 0.335 | 1.23, 0.69–2.22 | 0.47 |  |
| TT | 97/77 | 1.51, 0.44–5.14 | 0.51 | Reference |   |  |
| GT + TT | 136/102 | 1.60, 0.48–5.39 | 0.44 |   |   |  |
| GT + GG | 44/31 |   |   | 1.12, 0.65–1.95 | 0.67 |  |
| G allele | 49/37 | Reference |   | 1.01, 0.63–1.62 | 1.00 |  |
| T allele | 233/179 | 0.98, 0.61–1.57 | 0.943 | Reference |   |  |

Table 2 Caption: Odds ratio (OR) for association of IFNL4 allele, genotype and haplotype with HCV infections

The C–T haplotype is comprised of two favorable alleles (C of rs12979860 and T of rs8099917) which at homozygous condition were found to be associated with HCV viral clearance. Therefore, we have considered C–T haplotype as ‘reference and other three (T–G, T–T and C–G) as risk haplotypes. Genetic association of these risk haplotypes with HCV infection was analyzed and not found significant. The protective effect (OR = 0.11, *p* = 0.01) of C–G haplotype may be due to presence of strong favorable allele C of rs12979860 SNP (Table [2](#Tab2)).

The non-random association of alleles at these two SNPs loci (rs12979860 and rs8099917) was analyzed for linkage disequilibrium (LD) using Haploview (Fig. [2](#Fig2)C) and LD plot was obtained for CHC group, healthy individuals and combined groups. The pairwise LD value or coefficient of linkage disequilibrium (D′) between rs12979860 and rs8099917 is very high among CHC group that forms a block with D′ = 0.98 (Fig. [2](#Fig2)C-b) whereas it does not form block in healthy group with D′ = 0.76 (Fig. [2](#Fig2)C-c). There is a strong (r^2^ > 0.6) correlation between these two SNPs in CHC group (r^2^ = 0.62) than healthy group (r^2^ = 0.39). The LOD (logarithm of Odd) score is very high within CHC group (LOD = 25.40) indicating both SNPs are strongly linked but LOD score is low 9.98 in healthy population.

### IFNL4 polymorphisms as predictor of treatment induced clearance in CHC patient

Period of therapy to achieve RVR, EVR, ETVR and SVR was completed in 136,135, 132 and 119 patients respectively, for this cohort of CHC patients [n = 141, HCV genotype 1 (n = 35), genotype 2 (n = 1), genotype 3 (n = 101), genotype 4 (n = 4)]. The percentage of virological response for RVR, EVR, ETVR and SVR were 80.15%, 87.04%, 87.12% and 84.87% respectively (Fig. [3](#Fig3)).

### Fig. 3.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/04f2/6863998/631eb33cd31c/13337_2019_535_Fig3_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6863998_13337_2019_535_Fig3_HTML.jpg)

Consort diagram of CHC patients receiving PEG-IFN/Ribavirin therapy and achieving therapeutic response

Association of *IFNL4* rs12979860 SNP (CC, CT, TT, CT + TT and CT + CC) and rs8099917 (TT, GT, GG, GT + GG, GT + TT) for favorable virological response to antiviral therapy were found to be significant for all HCV genotype infected patient with Chi square test *p* value of < 0.0001(RVR) and 0.01(SVR) for SNP rs12979860 and Chi square test *p* value < 0.001(RVR, SVR) for SNP rs8099917 respectively (Table [3](#Tab3)). Odds ratio was calculated using major CC or minor TT genotype of rs12979860; and major TT or minor GG genotype of rs8099917 as reference versus other genotypes as case; treatment response positive (yes) as no-risk versus negative (no) as risk (Table [3](#Tab3)). Significant OR < 1 for major allele, genotype and haplotype indicate protective or favorable for virological response; whereas OR > 1 while considering dominant allele as reference indicates risk or relapse or infection (Table [3](#Tab3)).

### Table 3.

|   | rs12979860a |  |  |  | rs8099917a |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Genotype | Yes/no | OR (95% CI); p value | OR (95% CI); p value | Genotype | Yes/no | OR (95% CI); p value | OR (95% CI); p value |  |
| RVR | CC | 76/6 | Reference | 0.08 (0.02–0.35); < 0.0001 | TT | 87/8 | Reference | 0.02 (0.0–0.23); < 0.0001 |
| CT | 28/16 | 7.24 (2.58–20.35); < 0.0001 | 0.57 (0.14–2.28); 0.425 | GT | 21/15 | 7.77 (2.91–20.73); < 0.0001 | 0.18 (0.02–1.76); 0.107 |  |
| TT | 5/5 | 12.7 (2.85–56.34); < 0.0001 | Reference | GG | 1/4 | 43.5 (4.33–437.3); < 0.0001 | Reference |  |
| CT + TT | 33/21 | 8.06 (2.98–21.80); < 0.0001 |   | GT + GG | 22/19 | 9.39 (3.63–24.27); < 0.0001 |   |  |
| CT + CC | 104/22 |   | 0.21 (0.06–0.79); 0.013 | GT + TT | 108/23 |   | 0.05 (0.01–0.50); 0.001 |  |
| χ2, df |   | 25.41, 3 | 22.588 |   |   | 36.256, 3 | 31.529 |  |
| p value |   | < 0.0001 | < 0.0001 |   |   | < 0.0001 | < 0.0001 |  |
|   | Haplotype |   |   |   |   |   |   |   |
| RVR | C–T | 104/22 | Reference | 0.23 (0.11–0.51); < 0.0001 |   |   |   |   |
| T–G | 21/19 | 4.28 (1.98–9.26); < 0.0001 | Reference |   |   |   |   |  |
| T–T | 32/17 | 2.51 (1.19–5.30); 0.014 | 0.59 (0.25–1.38); 0.221 |   |   |   |   |  |
| C–G | 19/14 | 3.48 (1.52–7.89); 0.002 | 0.81 (0.32–2.06); 0.665 |   |   |   |   |  |
| χ2, df |   | 18.45, 3 | 18.45, 3 |   |   |   |   |  |
| p value |   | 0.0004 | 0.0004 |   |   |   |   |  |
| SVR | CC | 74/5 | Reference | 0.09 (0.02–0.52); 0.001 | TT | 86/6 | Reference | 0.03 (0.0–0.44); < 0.0001 |
| CT | 32/8 | 3.70 (1.12–12.18); 0.024 | 0.33 (0.06–1.80); 0.188 | GT | 23/8 | 4.99 (1.57–15.81); 0.003 | 0.17 (0.01–2.19); 0.138 |  |
| TT | 4/3 | 11.1 (1.93–63.85); 0.001 | Reference | GG | 1/2 | 28.7 (2.26–363.2); < 0.0001 | Reference |  |
| CT + TT | 36/11 | 4.52 (1.46–13.99); 0.005 |   | GT + GG | 24/10 | 5.97 (1.97–18.1); 0.001 |   |  |
| CT + CC | 106/13 |   | 0.16 (0.03–0.81); 0.014 | GT + TT | 109/14 |   | 0.06 (0.01–0.75); 0.004 |  |
| χ2, df |   | 11.87, 3 | 11.40, 3 |   |   | 18.4, 3 | 16.686,3 |  |
| p value |   | 0.0079 | 0.010 |   |   | <0.0001 | 0.001 |  |
| SVR | C–T | 108/13 | Reference | 0.26 (0.11–0.66); 0.003 |   |   |   |   |
| T–G | 24/11 | 3.81 (1.52–9.52); 0.003 | Reference |   |   |   |   |  |
| T–T | 36/10 | 2.31 (0.93–5.71); 0.065 | 0.61 (0.22–1.65); 0.324 |   |   |   |   |  |
| C–G | 23/8 | 2.89 (1.07–7.77); 0.030 | 0.76 (0.26–2.22); 0.615 |   |   |   |   |  |

Table 3 Caption: Association of IFNL4 genotype and haplotype (rs12979860 and rs8099917) with virological response to antiviral therapy among the CHC patients

#### Association of IFNL4 haplotype with virological response to therapy

As mentioned above, both SNPs rs12979860 and rs8099917 formed a haplotype block by LD among CHC patient group. Association of haplotype which was derived from above mentioned SNPs was carried out for therapeutic response against HCV genotype 1 and 3. Statistical test for significance was carried out by Chi square test at 3 degree of freedom (Table [3](#Tab3)). Association of haplotypes (C–T, T–G, T–T and C–G) for favorable virological response to antiviral therapy were found to be significant for HCV with Chi square test *p* value of 0.0004 (RVR) and 0.016 (SVR). The odds ratio was calculated using haplotype C–T as reference versus other haplotypes. Significant association of haplotype with virological response was found with OR = 4.28 (95% CI 1.98–9.26) for RVR (*p* < 0.0001) and 3.81(95% CI 1.52–9.52) for SVR (*p* = 0.003) (Table [3](#Tab3)).

## Discussion

This study applied population genetic based approaches to evaluate potential differences in allele, genotype frequency and haplotype structure among the groups of CHC patients and healthy individuals of Indian ethnicity. The genetic associations of *IFNL4* polymorphisms were analyzed in perspective of the disease response and outcome of antiviral treatment. In this case–control genetic association study, the alleles, genotypes of individual SNP and haplotypes derived from both SNPs (rs12979860 and rs8099917) were considered risk factor as disease trait for chronic hepatitis in the individuals of cases or control, with or without HCV infection respectively. Test of deviation from HWE for individual causal SNPs were found non-significant (Fig. [2](#Fig2)A, B and Table [1](#Tab1)) which indicates that SNPs are not under influence of any natural or evolutionary forces and can be considered as good genetic traits for Indian populations. In our study, the minor allele frequency was 0.241 and 0.176 in CHC cases (n = 141) and 0.229 and 0.172 in healthy control (n = 111) for SNP rs12979860 (T allele) and rs8099917 (G allele), respectively (Table [1](#Tab1)). Similar result for minor allele frequency was also reported for Indian populations in a study conducted earlier using one SNP only in a healthy control group (0.238 for rs12979860 T allele) from southern region of India, the state of Andhra Pradesh [[27](#CR27)] and pooled samples CHC cases (0.17 for rs8099917 G allele) [[7](#CR7)]. We have derived haplotypes from both SNPs in healthy control and CHC cases by LD plot to determine population divergence to assess whether alleles at flanking loci on the same chromosomes tend to be inherited together or are there any associated ancestral recombination events. Both the studied SNPs are situated 4 kb apart and formed a LD block with strong correlation (r^2^ > 0.6, D′ = 0.98 and LOD = 25.4) for CHC cases (Fig. [2](#Fig2)C) than healthy control (r^2^ = 0.39, D′ = 0.76 and LOD = 9.98). In a cross-sectional study of European CHC cohort, Smith et al. [[28](#CR28)] studied 18 SNPs across the *IL28b* gene by massive parallel sequencing and also found that the SNPs (rs12979860 and rs8099917) associated with drug response, formed single LD (D′ = 0.86) block.

The genetics of *IFNL4* polymorphism is important for host response to HCV infection and treatment [[13](#CR13)]. Association of *IFNL4* polymorphisms with HCV infection can be evaluated in the context of spontaneous clearance of virus in acute infection or impaired virus clearance leading to chronic liver disease [[4](#CR4)]. In our study, CHC cases with impaired virus clearance were compared with healthy individuals of Indian origin. The allele/genotype of individual SNP and haplotype of both SNPs were analyzed for protective or risk phenotype assuming either major or minor allele/genotype as reference (Table [3](#Tab3)). The test of association for each allele, genotype and haplotype of both SNPs with risk of chronic HCV infection were not found significant but found to have protective role (C–G haplotype, OR = 0.11 and *p* = 0.01) (Table [2](#Tab2)). In previous reports, *IFNL4* variants such as the major CC genotype of rs12979860, TT genotype of rs8099917 [[30](#CR30)] and ΔG allele at rs368234815 [[1](#CR1)] were found to be associated with spontaneous clearance of virus in individuals with acute HCV infection. Polymorphisms of other genes such as virus entry related *SCARB1* gene [[14](#CR14)] and *MxA* gene-rs2071430 TT genotype [[32](#CR32)] also have shown association with spontaneous clearance or impaired clearance to HCV infections.

Out of all HCV genotypes, the prevalence of HCV genotype 3 was very high (71.63%) in our CHC cohort which is in concurrence with Indian prevalent HCV genotypes [[20](#CR20)]. The most common cause of global HCV infections are due to genotypes 1 (44%) and 3 (25%) among all seven genotypes [[6](#CR6)]. The treatments of chronic HCV infections are based on either IFN (IFN/RBV) regimen or IFN free direct acting antiviral (DAA) therapy [[3](#CR3), [12](#CR12)]. The IFN free DAA (Sofosbuvir based) therapy are now more potent in controlling HCV infection than PEG-IFN/RBV therapy [[8](#CR8)]. However, there are limitations of DAA therapy such as expected drug resistance, suboptimal activity against diverse HCV genotypes, suitability in advanced cirrhotic or impaired renal function patients and most importantly high cost of DAAs for CHC patients [[25](#CR25)]. As a predictor of response, the CC genotype of rs12979860 and TT genotype of rs8099917 had shown strong favorable association to PEG-IFN/RBV therapy in initial GWAS studies with ethnic variation of these genotypes [[10](#CR10), [29](#CR29)]. In our study, genetic association of *IFNL4* genotype and haplotype with virological response (RVR and SVR) to PEG/IFN therapy was found to be highly significant (Table [3](#Tab3)). The major genotype and haplotype found to be protective/favorable (OR < 1) for therapeutic response are CC genotype of rs12979860 (OR = 0.08, 0.09 for RVR and SVR), TT genotype of rs8099917 (OR = 0.02, 0.03 for RVR and SVR) and C–T haplotype of both SNP (OR = 0.23, 0.26 for RVR and SVR). Other genotypes and haplotypes were also found to be significantly associated with risk (OR > 1) or no response status (Table [3](#Tab3)). The genetic association of *IFNL4* polymorphisms with virological response to IFN sparing regimen or DAA therapy is not clear yet. However, a recent study found association of SNP rs12979860 with relapse of infection at 8 weeks of DAA (Sofosbuvir, Velpatasvir, and Voxilaprevir) therapy. The relapse was observed to be less for CC genotype (1.2%) as compared to higher relapse (10.8%) for risk genotype CT and TT [[21](#CR21)]. Besides viral and host genotype, we have also reported other host and disease factor such as high body mass index, insulin resistance and significant fibrosis as predictors of non-response to IFN/RBV therapy in Indian CHC cohorts [[31](#CR31)].

In conclusion, our study found higher prevalence of favorable allele in both the healthy and CHC population. The risk genotypes and haplotypes of both the SNPs were not found to be associated with chronic HCV infection. Favorable genotype and haplotype were also found to be significantly associated with virological response to IFN based therapy. Therefore, association of *IFNL4* polymorphism with respect to relapse of DAA therapy needs to be studied for Indian populations along with evolution of resistant associated variants.

## Acknowledgements

This study was supported by AIIMS intramural funding A-147.

## Compliance with ethical standards

### Conflict of interest

The authors declare that they have no conflict of interest.

## Footnotes

## References

1. Aka PV, Kuniholm MH, Pfeiffer RM, et al. Association of the IFNL4-DeltaG allele with impaired spontaneous clearance of hepatitis C virus. J Infect Dis. 2013;209(3):350–354. doi: 10.1093/infdis/jit433.  [DOI](https://doi.org/10.1093/infdis/jit433) | [PMC free article](/articles/PMC3883162/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23956438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=Association%20of%20the%20IFNL4-DeltaG%20allele%20with%20impaired%20spontaneous%20clearance%20of%20hepatitis%20C%20virus&author=PV%20Aka&author=MH%20Kuniholm&author=RM%20Pfeiffer&volume=209&issue=3&publication_year=2013&pages=350-354&pmid=23956438&doi=10.1093/infdis/jit433&)

2. Ank N, Iversen MB, Bartholdy C, et al. An important role for type III interferon (IFN-lambda/IL-28) in TLR-induced antiviral activity. J Immunol. 2008;180(4):2474–2485. doi: 10.4049/jimmunol.180.4.2474.  [DOI](https://doi.org/10.4049/jimmunol.180.4.2474) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18250457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol&title=An%20important%20role%20for%20type%20III%20interferon%20(IFN-lambda/IL-28)%20in%20TLR-induced%20antiviral%20activity&author=N%20Ank&author=MB%20Iversen&author=C%20Bartholdy&volume=180&issue=4&publication_year=2008&pages=2474-2485&pmid=18250457&doi=10.4049/jimmunol.180.4.2474&)

3. Asselah T, Benhamou Y, Marcellin P. Protease and polymerase inhibitors for the treatment of hepatitis C. Liver Int. 2009;29(Suppl 1):57–67. doi: 10.1111/j.1478-3231.2008.01928.x.  [DOI](https://doi.org/10.1111/j.1478-3231.2008.01928.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19207967/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Liver%20Int&title=Protease%20and%20polymerase%20inhibitors%20for%20the%20treatment%20of%20hepatitis%20C&author=T%20Asselah&author=Y%20Benhamou&author=P%20Marcellin&volume=29&issue=Suppl%201&publication_year=2009&pages=57-67&pmid=19207967&doi=10.1111/j.1478-3231.2008.01928.x&)

4. Asthana M, Sahu SK, Kumar A, et al. Role of Interleukin 28B polymorphisms in response to Interferon based therapy for hepatitis C virus clearance. Curr Drug Metab. 2018;19:215–223. doi: 10.2174/1389200219666180129115359.  [DOI](https://doi.org/10.2174/1389200219666180129115359) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29380700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Metab&title=Role%20of%20Interleukin%2028B%20polymorphisms%20in%20response%20to%20Interferon%20based%20therapy%20for%20hepatitis%20C%20virus%20clearance&author=M%20Asthana&author=SK%20Sahu&author=A%20Kumar&volume=19&publication_year=2018&pages=215-223&pmid=29380700&doi=10.2174/1389200219666180129115359&)

5. Bhushan A, Chinnaswamy S. Identifying causal variants at the interferon lambda locus in case-control studies: utilizing non-synonymous variant rs117648444 to probe the role of IFN-lambda4. Gene. 2018;664:168–180. doi: 10.1016/j.gene.2018.04.076.  [DOI](https://doi.org/10.1016/j.gene.2018.04.076) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29705128/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Identifying%20causal%20variants%20at%20the%20interferon%20lambda%20locus%20in%20case-control%20studies:%20utilizing%20non-synonymous%20variant%20rs117648444%20to%20probe%20the%20role%20of%20IFN-lambda4&author=A%20Bhushan&author=S%20Chinnaswamy&volume=664&publication_year=2018&pages=168-180&pmid=29705128&doi=10.1016/j.gene.2018.04.076&)

6. Blach ZS, Manns M, Altraif I, Duberg AS, Muljono DH, Polaris Observatory HCV Collaborators et al. Global prevalence and genotype distribution of hepatitis C virus infection in 2015: a modelling study. Lancet Gastroenterol Hepatol. 2017;2(3):161–176. doi: 10.1016/S2468-1253(16)30181-9.  [DOI](https://doi.org/10.1016/S2468-1253(16)30181-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28404132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Gastroenterol%20Hepatol&title=Global%20prevalence%20and%20genotype%20distribution%20of%20hepatitis%20C%20virus%20infection%20in%202015:%20a%20modelling%20study&author=ZS%20Blach&author=M%20Manns&author=I%20Altraif&author=AS%20Duberg&author=DH%20Muljono&volume=2&issue=3&publication_year=2017&pages=161-176&pmid=28404132&doi=10.1016/S2468-1253(16)30181-9&)

7. Chinnaswamy S, Das K, Bairagya BB, et al. Association of IL28B single nucleotide polymorphism rs8099917 with response to treatment in genotype 3 HCV-infected patients from India. Trop Gastroenterol. 2014;35(2):96–102. doi: 10.7869/tg.187.  [DOI](https://doi.org/10.7869/tg.187) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25470871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trop%20Gastroenterol&title=Association%20of%20IL28B%20single%20nucleotide%20polymorphism%20rs8099917%20with%20response%20to%20treatment%20in%20genotype%203%20HCV-infected%20patients%20from%20India&author=S%20Chinnaswamy&author=K%20Das&author=BB%20Bairagya&volume=35&issue=2&publication_year=2014&pages=96-102&pmid=25470871&doi=10.7869/tg.187&)

8. EASL. EAftSotL EASL recommendations on treatment of hepatitis C 2016. J Hepatol. 2017;66(1):153–194. doi: 10.1016/j.jhep.2016.09.001.  [DOI](https://doi.org/10.1016/j.jhep.2016.09.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27667367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=EASL%20recommendations%20on%20treatment%20of%20hepatitis%20C%202016&volume=66&issue=1&publication_year=2017&pages=153-194&pmid=27667367&doi=10.1016/j.jhep.2016.09.001&)

9. Eslam M, McLeod D, Kelaeng KS, et al. IFN-lambda3, not IFN-lambda4, likely mediates IFNL3–IFNL4 haplotype-dependent hepatic inflammation and fibrosis. Nat Genet. 2017;49(5):795–800. doi: 10.1038/ng.3836.  [DOI](https://doi.org/10.1038/ng.3836) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28394349/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=IFN-lambda3,%20not%20IFN-lambda4,%20likely%20mediates%20IFNL3%E2%80%93IFNL4%20haplotype-dependent%20hepatic%20inflammation%20and%20fibrosis&author=M%20Eslam&author=D%20McLeod&author=KS%20Kelaeng&volume=49&issue=5&publication_year=2017&pages=795-800&pmid=28394349&doi=10.1038/ng.3836&)

10. Ge D, Fellay J, Thompson AJ, et al. Genetic variation in IL28B predicts hepatitis C treatment-induced viral clearance. Nature. 2009;461(7262):399–401. doi: 10.1038/nature08309.  [DOI](https://doi.org/10.1038/nature08309) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20predicts%20hepatitis%20C%20treatment-induced%20viral%20clearance&author=D%20Ge&author=J%20Fellay&author=AJ%20Thompson&volume=461&issue=7262&publication_year=2009&pages=399-401&pmid=19684573&doi=10.1038/nature08309&)

11. Gonzalez-Aldaco K, Rebello Pinho JR, Roman S, et al. Association with spontaneous hepatitis C viral clearance and genetic differentiation of IL28B/IFNL4 haplotypes in populations from Mexico. PLoS ONE. 2016;11(1):e0146258. doi: 10.1371/journal.pone.0146258.  [DOI](https://doi.org/10.1371/journal.pone.0146258) | [PMC free article](/articles/PMC4704808/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26741362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Association%20with%20spontaneous%20hepatitis%20C%20viral%20clearance%20and%20genetic%20differentiation%20of%20IL28B/IFNL4%20haplotypes%20in%20populations%20from%20Mexico&author=K%20Gonzalez-Aldaco&author=JR%20Rebello%20Pinho&author=S%20Roman&volume=11&issue=1&publication_year=2016&pages=e0146258&pmid=26741362&doi=10.1371/journal.pone.0146258&)

12. Hadziyannis SJ, Sette H, Jr, Morgan TR, et al. Peginterferon-alpha2a and ribavirin combination therapy in chronic hepatitis C: a randomized study of treatment duration and ribavirin dose. Ann Intern Med. 2004;140(5):346–355. doi: 10.7326/0003-4819-140-5-200403020-00010.  [DOI](https://doi.org/10.7326/0003-4819-140-5-200403020-00010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14996676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Peginterferon-alpha2a%20and%20ribavirin%20combination%20therapy%20in%20chronic%20hepatitis%20C:%20a%20randomized%20study%20of%20treatment%20duration%20and%20ribavirin%20dose&author=SJ%20Hadziyannis&author=H%20Sette&author=TR%20Morgan&volume=140&issue=5&publication_year=2004&pages=346-355&pmid=14996676&doi=10.7326/0003-4819-140-5-200403020-00010&)

13. Hayes CN, Imamura M, Aikata H, et al. Genetics of IL28B and HCV-response to infection and treatment. Nat Rev Gastroenterol Hepatol. 2012;9(7):406–417. doi: 10.1038/nrgastro.2012.101.  [DOI](https://doi.org/10.1038/nrgastro.2012.101) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22641049/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Gastroenterol%20Hepatol&title=Genetics%20of%20IL28B%20and%20HCV-response%20to%20infection%20and%20treatment&author=CN%20Hayes&author=M%20Imamura&author=H%20Aikata&volume=9&issue=7&publication_year=2012&pages=406-417&pmid=22641049&doi=10.1038/nrgastro.2012.101&)

14. Hsu CS, Hsu SJ, Liu WL, et al. Association of SCARB1 gene polymorphisms with virological response in chronic hepatitis C patients receiving pegylated interferon plus ribavirin therapy. Sci Rep. 2016;6:32303. doi: 10.1038/srep32303.  [DOI](https://doi.org/10.1038/srep32303) | [PMC free article](/articles/PMC4999819/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27561198/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Association%20of%20SCARB1%20gene%20polymorphisms%20with%20virological%20response%20in%20chronic%20hepatitis%20C%20patients%20receiving%20pegylated%20interferon%20plus%20ribavirin%20therapy&author=CS%20Hsu&author=SJ%20Hsu&author=WL%20Liu&volume=6&publication_year=2016&pages=32303&pmid=27561198&doi=10.1038/srep32303&)

15. Huang M, Jiang JD, Peng Z. Recent advances in the anti-HCV mechanisms of interferon. Acta Pharm Sin B. 2014;4(4):241–247. doi: 10.1016/j.apsb.2014.06.010.  [DOI](https://doi.org/10.1016/j.apsb.2014.06.010) | [PMC free article](/articles/PMC4629091/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26579391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Pharm%20Sin%20B&title=Recent%20advances%20in%20the%20anti-HCV%20mechanisms%20of%20interferon&author=M%20Huang&author=JD%20Jiang&author=Z%20Peng&volume=4&issue=4&publication_year=2014&pages=241-247&pmid=26579391&doi=10.1016/j.apsb.2014.06.010&)

16. Lang Kuhs KA, Kuniholm MH, Pfeiffer RM, et al. Interferon lambda 4 genotype is not associated with recurrence of oral or genital herpes. PLoS ONE. 2015;10(10):e0138827. doi: 10.1371/journal.pone.0138827.  [DOI](https://doi.org/10.1371/journal.pone.0138827) | [PMC free article](/articles/PMC4592222/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26431156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Interferon%20lambda%204%20genotype%20is%20not%20associated%20with%20recurrence%20of%20oral%20or%20genital%20herpes&author=KA%20Lang%20Kuhs&author=MH%20Kuniholm&author=RM%20Pfeiffer&volume=10&issue=10&publication_year=2015&pages=e0138827&pmid=26431156&doi=10.1371/journal.pone.0138827&)

17. Lee MH, Yang HI, Lu SN, et al. Polymorphisms near the IFNL3 gene associated with HCV RNA spontaneous clearance and hepatocellular carcinoma risk. Sci Rep. 2015;5:17030. doi: 10.1038/srep17030.  [DOI](https://doi.org/10.1038/srep17030) | [PMC free article](/articles/PMC4658500/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26602024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Polymorphisms%20near%20the%20IFNL3%20gene%20associated%20with%20HCV%20RNA%20spontaneous%20clearance%20and%20hepatocellular%20carcinoma%20risk&author=MH%20Lee&author=HI%20Yang&author=SN%20Lu&volume=5&publication_year=2015&pages=17030&pmid=26602024&doi=10.1038/srep17030&)

18. Liao Y, Cai B, Li Y, et al. Association of HLA-DP/DQ, STAT4 and IL-28B variants with HBV viral clearance in Tibetans and Uygurs in China. Liver Int. 2014;35(3):886–896. doi: 10.1111/liv.12643.  [DOI](https://doi.org/10.1111/liv.12643) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25041342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Liver%20Int&title=Association%20of%20HLA-DP/DQ,%20STAT4%20and%20IL-28B%20variants%20with%20HBV%20viral%20clearance%20in%20Tibetans%20and%20Uygurs%20in%20China&author=Y%20Liao&author=B%20Cai&author=Y%20Li&volume=35&issue=3&publication_year=2014&pages=886-896&pmid=25041342&doi=10.1111/liv.12643&)

19. Liu MQ, Zhou DJ, Wang X, et al. IFN-lambda3 inhibits HIV infection of macrophages through the JAK-STAT pathway. PLoS ONE. 2012;7(4):e35902. doi: 10.1371/journal.pone.0035902.  [DOI](https://doi.org/10.1371/journal.pone.0035902) | [PMC free article](/articles/PMC3338759/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22558263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=IFN-lambda3%20inhibits%20HIV%20infection%20of%20macrophages%20through%20the%20JAK-STAT%20pathway&author=MQ%20Liu&author=DJ%20Zhou&author=X%20Wang&volume=7&issue=4&publication_year=2012&pages=e35902&pmid=22558263&doi=10.1371/journal.pone.0035902&)

20. NCDC. Quarterly newsletter from the National Centre for Disease Control (NCDC). NCDC India. 2014;3(1):1–12.

21. O’Brien TR, Kottilil S, Pfeiffer RM. IFNL4 genotype is associated with virologic relapse after 8-week treatment with sofosbuvir, velpatasvir, and voxilaprevir. Gastroenterology. 2017;153(1–2):1694–1695. doi: 10.1053/j.gastro.2017.06.069.  [DOI](https://doi.org/10.1053/j.gastro.2017.06.069) | [PMC free article](/articles/PMC7390973/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29107709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=IFNL4%20genotype%20is%20associated%20with%20virologic%20relapse%20after%208-week%20treatment%20with%20sofosbuvir,%20velpatasvir,%20and%20voxilaprevir&author=TR%20O%E2%80%99Brien&author=S%20Kottilil&author=RM%20Pfeiffer&volume=153&issue=1%E2%80%932&publication_year=2017&pages=1694-1695&pmid=29107709&doi=10.1053/j.gastro.2017.06.069&)

22. O’brien TR, Prokunina-Olsson L, Donnelly RP. IFN-lambda4: the paradoxical new member of the interferon lambda family. J Interferon Cytokine Res. 2014;34(11):829–838. doi: 10.1089/jir.2013.0136.  [DOI](https://doi.org/10.1089/jir.2013.0136) | [PMC free article](/articles/PMC4217005/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24786669/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Interferon%20Cytokine%20Res&title=IFN-lambda4:%20the%20paradoxical%20new%20member%20of%20the%20interferon%20lambda%20family&author=TR%20O%E2%80%99brien&author=L%20Prokunina-Olsson&author=RP%20Donnelly&volume=34&issue=11&publication_year=2014&pages=829-838&pmid=24786669&doi=10.1089/jir.2013.0136&)

23. Prokunina-Olsson L, Muchmore B, Tang W, et al. A variant upstream of IFNL3 (IL28B) creating a new interferon gene IFNL4 is associated with impaired clearance of hepatitis C virus. Nat Genet. 2013;45(2):164–171. doi: 10.1038/ng.2521.  [DOI](https://doi.org/10.1038/ng.2521) | [PMC free article](/articles/PMC3793390/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23291588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20variant%20upstream%20of%20IFNL3%20(IL28B)%20creating%20a%20new%20interferon%20gene%20IFNL4%20is%20associated%20with%20impaired%20clearance%20of%20hepatitis%20C%20virus&author=L%20Prokunina-Olsson&author=B%20Muchmore&author=W%20Tang&volume=45&issue=2&publication_year=2013&pages=164-171&pmid=23291588&doi=10.1038/ng.2521&)

24. Rauch A, Kutalik Z, Descombes P, et al. Genetic variation in IL28B is associated with chronic hepatitis C and treatment failure: a genome-wide association study. Gastroenterology. 2010;138(4):1338–1345. doi: 10.1053/j.gastro.2009.12.056.  [DOI](https://doi.org/10.1053/j.gastro.2009.12.056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20060832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Genetic%20variation%20in%20IL28B%20is%20associated%20with%20chronic%20hepatitis%20C%20and%20treatment%20failure:%20a%20genome-wide%20association%20study&author=A%20Rauch&author=Z%20Kutalik&author=P%20Descombes&volume=138&issue=4&publication_year=2010&pages=1338-1345&pmid=20060832&doi=10.1053/j.gastro.2009.12.056&)

25. Sarrazin C. The importance of resistance to direct antiviral drugs in HCV infection in clinical practice. J Hepatol. 2015;64:486–504. doi: 10.1016/j.jhep.2015.09.011.  [DOI](https://doi.org/10.1016/j.jhep.2015.09.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26409317/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=The%20importance%20of%20resistance%20to%20direct%20antiviral%20drugs%20in%20HCV%20infection%20in%20clinical%20practice&author=C%20Sarrazin&volume=64&publication_year=2015&pages=486-504&pmid=26409317&doi=10.1016/j.jhep.2015.09.011&)

26. Sheppard P, Kindsvogel W, Xu W, et al. IL-28, IL-29 and their class II cytokine receptor IL-28R. Nat Immunol. 2003;4(1):63–68. doi: 10.1038/ni873.  [DOI](https://doi.org/10.1038/ni873) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12469119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Immunol&title=IL-28,%20IL-29%20and%20their%20class%20II%20cytokine%20receptor%20IL-28R&author=P%20Sheppard&author=W%20Kindsvogel&author=W%20Xu&volume=4&issue=1&publication_year=2003&pages=63-68&pmid=12469119&doi=10.1038/ni873&)

27. Sivaprasad S, Rao PN, Gupta R, et al. The distribution of genotype and allelic frequency of IL28B gene polymorphism in Andhra Pradesh, India. J Clin Exp Hepatol. 2012;2(2):112–115. doi: 10.1016/S0973-6883(12)60098-X.  [DOI](https://doi.org/10.1016/S0973-6883(12)60098-X) | [PMC free article](/articles/PMC3940249/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25755419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Exp%20Hepatol&title=The%20distribution%20of%20genotype%20and%20allelic%20frequency%20of%20IL28B%20gene%20polymorphism%20in%20Andhra%20Pradesh,%20India&author=S%20Sivaprasad&author=PN%20Rao&author=R%20Gupta&volume=2&issue=2&publication_year=2012&pages=112-115&pmid=25755419&doi=10.1016/S0973-6883(12)60098-X&)

28. Smith KR, Suppiah V, O’Connor K, et al. Identification of improved IL28B SNPs and haplotypes for prediction of drug response in treatment of hepatitis C using massively parallel sequencing in a cross-sectional European cohort. Genome Med. 2011;3(8):57. doi: 10.1186/gm273.  [DOI](https://doi.org/10.1186/gm273) | [PMC free article](/articles/PMC3238183/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21884576/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med&title=Identification%20of%20improved%20IL28B%20SNPs%20and%20haplotypes%20for%20prediction%20of%20drug%20response%20in%20treatment%20of%20hepatitis%20C%20using%20massively%20parallel%20sequencing%20in%20a%20cross-sectional%20European%20cohort&author=KR%20Smith&author=V%20Suppiah&author=K%20O%E2%80%99Connor&volume=3&issue=8&publication_year=2011&pages=57&pmid=21884576&doi=10.1186/gm273&)

29. Tanaka Y, Nishida N, Sugiyama M, et al. Genome-wide association of IL28B with response to pegylated interferon-alpha and ribavirin therapy for chronic hepatitis C. Nat Genet. 2009;41(10):1105–1109. doi: 10.1038/ng.449.  [DOI](https://doi.org/10.1038/ng.449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20of%20IL28B%20with%20response%20to%20pegylated%20interferon-alpha%20and%20ribavirin%20therapy%20for%20chronic%20hepatitis%20C&author=Y%20Tanaka&author=N%20Nishida&author=M%20Sugiyama&volume=41&issue=10&publication_year=2009&pages=1105-1109&pmid=19749757&doi=10.1038/ng.449&)

30. Thomas DL, Thio CL, Martin MP, et al. Genetic variation in IL28B and spontaneous clearance of hepatitis C virus. Nature. 2009;461(7265):798–801. doi: 10.1038/nature08463.  [DOI](https://doi.org/10.1038/nature08463) | [PMC free article](/articles/PMC3172006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19759533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20and%20spontaneous%20clearance%20of%20hepatitis%20C%20virus&author=DL%20Thomas&author=CL%20Thio&author=MP%20Martin&volume=461&issue=7265&publication_year=2009&pages=798-801&pmid=19759533&doi=10.1038/nature08463&)

31. Vasudevan S, Kavimandan A, Kalra N, et al. Demographic profile, host, disease and viral predictive factors of response in patients with chronic hepatitis C virus infection at a tertiary care hospital in north India. Indian J Med Res. 2016;143(3):331–340. doi: 10.4103/0971-5916.182624.  [DOI](https://doi.org/10.4103/0971-5916.182624) | [PMC free article](/articles/PMC4892080/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27241647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indian%20J%20Med%20Res&title=Demographic%20profile,%20host,%20disease%20and%20viral%20predictive%20factors%20of%20response%20in%20patients%20with%20chronic%20hepatitis%20C%20virus%20infection%20at%20a%20tertiary%20care%20hospital%20in%20north%20India&author=S%20Vasudevan&author=A%20Kavimandan&author=N%20Kalra&volume=143&issue=3&publication_year=2016&pages=331-340&pmid=27241647&doi=10.4103/0971-5916.182624&)

32. Zang F, Yue M, Yao Y, et al. Influence of IL28B and MxA gene polymorphisms on HCV clearance in Han Chinese population. Epidemiol Infect. 2018;146(3):379–385. doi: 10.1017/S0950268817002928.  [DOI](https://doi.org/10.1017/S0950268817002928) | [PMC free article](/articles/PMC9134562/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29271328/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epidemiol%20Infect&title=Influence%20of%20IL28B%20and%20MxA%20gene%20polymorphisms%20on%20HCV%20clearance%20in%20Han%20Chinese%20population&author=F%20Zang&author=M%20Yue&author=Y%20Yao&volume=146&issue=3&publication_year=2018&pages=379-385&pmid=29271328&doi=10.1017/S0950268817002928&)
