# Potential role of PIN1 genotypes in predicting benefit from oxaliplatin- and irinotecan-based treatment in patients with metastatic colorectal cancer

## Metadata
**Authors:** Mitsukuni Suenaga, Marta Schirripa, Shu Cao, Wu Zhang, Dongyun Yang, Chiara Cremolini, Sara Lonardi, Francesca Bergamo, Yang Ning, Noriko Yamamoto, Satoshi Okazaki, Martin D Berger, Yuji Miyamoto, Roel Gopez, Jr, Afsaneh Barzi, Toshiharu Yamaguchi, Sebastian Stintzing, Volker Heinemann, Fotios Loupakis, Alfredo Falcone, Heinz-Josef Lenz
**Journal:** The pharmacogenomics journal
**Date:** 2018 Jun 21
**DOI:** [10.1038/s41397-018-0030-8](https://doi.org/10.1038/s41397-018-0030-8)
**PMID:** 29925895
**PMCID:** PMC6151284
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6151284/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6151284/pdf/nihms959884.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6151284/pdf/nihms959884.pdf)

## Abstract

PIN1-mediated substrate isomerization plays a role in the repair of DNA double-strand breaks. We hypothesized that genetic polymorphisms in PIN1-related pathways may affect tumor sensitivity to oxaliplatin or irinotecan in metastatic colorectal cancer (mCRC) patients. We analyzed genomic DNA from five cohorts of mCRC patients (total 950) treated with different first-line treatments: oxaliplatin cohorts 1 (n=146) and 2 (n=70); irinotecan cohorts 1 (n=228) and 2 (n=276); and combination cohort (n=230). Single nucleotide polymorphisms of candidate genes were analyzed by PCR-based direct sequencing. In the oxaliplatin cohort 1, patients carrying any PIN1 rs2233678 C allele had shorter progression-free survival (PFS) and overall survival (OS) than the G/G variant (PFS, 7.4 vs. 15.0 months, hazard ratio [HR] 3.24, P<0.001; OS, 16.9 vs. 31.5 months, HR: 2.38, P=0.003). In contrast, patients with C allele had longer median PFS than patients with G/G (11.9 vs. 9.4 months, HR: 0.64, 95%CI: 0.45–0.91, P=0.009) in the irinotecan cohort 1. No significant differences were observed in the combination cohort. In comparison between the irinotecan cohort 1 and combination cohort, the patients carrying the G/G variant benefit greatly from the combination compared with irinotecan-based regimen (PFS, 11.6 vs. 9.4 months, HR 0.61, 95%CI: 0.47–0.78, P<0.001; OS, 30.6 vs. 24.0 months, HR 0.79, 95%CI: 0.62–1.02, P=0.060), while no significant difference was shown in any C allele. Germline PIN1 polymorphisms may predict clinical outcomes in mCRC patients receiving oxaliplatin- or irinotecan-based therapy, and identify specific populations favorable to oxaliplatin plus irinotecan combination therapy.

## Introduction

The never-in-mitosis-gene-a (NIMA) kinases are a family of serine/threonine kinases that play essential roles in mitotic entry by modulating the cell cycle in both *Aspergillus nidulans* and human cells.^1–3^ Overexpression of NIMA promotes progression of the cell cycle irrespective of the cell cycle phase, while inactivation of NIMA reversibly induces late G2 arrest.^4^ Serine/threonine (S/T) phosphorylation regulates the function of proteins by inducing conformational changes. Peptidyl-prolyl cis/trans isomerase NIMA-interacting 1 (PIN1) interacts with NIMA in a manner that correlates with cell cycle progression, cellular proliferation, and neoplastic transformation. PIN1 binds to and isomerizes specific phosphorylated S/T-proline motifs, resulting in conformational changes in a subset of target proteins. The PIN1-induced conformational changes affect the function of its substrates through a post-phosphorylation regulatory mechanism, leading to enhanced or suppressed phosphorylation in cell signaling.^5^

PIN1 is also involved in the repair of DNA double-strand breaks (DSBs) as part of the DNA damage response (DDR), which maintains genomic integrity and reduces the risk of tumorigenesis.^6,7^ The DDR activates DNA repair and regulates cell cycle checkpoint pathways, leading to cell cycle arrest or apoptosis.^8^ *TP53* is a tumor suppressor gene that transcriptionally regulates genes involved in cell cycle checkpoints, genomic stability, and apoptosis.^9^ As part of the checkpoints, DNA damage triggers PIN1 isomerization of p53, which directs the transcriptional activity of p53 toward cell cycle arrest or apoptosis depending on the degree of DNA damage.^10,11^ In addition, PIN1 inhibits the ubiquitination of p53 by mouse double minute 2 homolog (Mdm2).^12^ Homologous recombination is a crucial DNA repair pathway triggered by DSBs in the late S and G2 phases of the cell cycle. Isomerization of CtIP by PIN1 results in diminished DSB repair by restricting DNA end-resection ([Figure 1A](#F1)). Furthermore, PIN1 appears to restrict homologous recombination and promote another DNA repair pathway non-homologous end-joining (NHEJ) under DSBs via isomerization of CtIP.^6,13^

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97dc/6151284/d41f49ed1f1e/nihms959884f1.jpg)

Chemotherapeutic agents and PIN1-related pathways: roles of PIN1 in (A) oxaliplatin-based treatment; (B) irinotecan-based treatment; (C) PIN1 SNPs and oxaliplatin sensitivity; and (D) PIN1 SNPs and irinotecan sensitivity.

Casein kinase 2 (CK2) is a tetrameric protein consisting of two catalytic (CK2α/2α′) and two regulatory (CK2β) subunits.^14^ CK2 is a highly conserved serine/threonine protein kinase that regulates cell viability, cell proliferation, and signal transduction by phosphorylating numerous cellular targets.^15–17^ PIN1 interacts with CK2α/2α′, especially phosphorylated CK2α, but not with CK2β. PIN1-mediated isomerization of CK2α regulates the hyperphosphorylation and activity of topoisomerase I (Top1), an enzyme that unwinds the DNA double helix and is the target of irinotecan ^18,19^ ([Figure 1B](#F1)).

As key components of current standard treatments for colorectal cancer (CRC),^20^ oxaliplatin is a platinum compound that crosslinks DNA, whereas irinotecan inhibits Top1.^21,22^ Although both agents induce DNA damage, we speculated that they may inflict different degrees of damage, and oxaliplatin would be predicted to have the greater effect. In this study, we determined whether gene polymorphisms in PIN1-related pathway components could predict the efficacy of oxaliplatin- and irinotecan-based treatments and their combination therapy in patients with metastatic CRC (mCRC).

## Materials and methods

### Study design and patients

We investigated five independent cohorts of 950 patients with mCRC who received different first-line treatments. Oxaliplatin cohort 1 (Ox-cohort 1) consisted of 146 patients who received FOLFOX ± bevacizumab treatment at the Cancer Institute Hospital (Tokyo, Japan) as discovery.^23^ Oxaliplatin cohort 2 (Ox-cohort 2) consisted of 70 patients in a retrospective cohort who received first-line FOLFOX or XELOX + bevacizumab treatment as validation (Tokyo, Japan). Irinotecan cohort 1 (Iri-cohort 1) of 228 patients received FOLFIRI + bevacizumab treatment (TRIBE arm A) in the randomized, open-label, phase III TRIBE trial as discovery.^24^ Irinotecan cohort 2 (Iri-cohort 2) of 276 patients received FOLFIRI + bevacizumab treatment (FIRE-3 arm B) in the randomized, open-label, phase III FIRE-3 trial.^25^ Combination cohort (Ox + Iri-cohort) consisted of 230 patients who received FOLFOXIRI + bevacizumab treatment (TRIBE arm B) in the TRIBE trial.^24^ Eligible patients had a histologically confirmed diagnosis of mCRC, had measurable or evaluable disease according to Response Evaluation Criteria in Solid Tumors (RECIST) v1.1, had no prior treatment for metastatic disease, and provided signed informed consent. FOLFOX ± bevacizumab treatment (oxaliplatin 85 mg/m^2^, 5-fluorouracil [5-FU] bolus 400 mg/m^2^, 5-FU infusion 2400 mg/m^2^, levofolinate calcium 200 mg/m^2^, with or without bevacizumab 5 mg/kg) was administered every 2 weeks. XELOX + bevacizumab treatment (oxaliplatin 130 mg/m^2^, capecitabine 1,000 mg/m^2^ given twice daily from evening of day 1 to morning of day 15, with or without bevacizumab 5 mg/kg) was administered every 3 weeks. FOLFIRI + bevacizumab treatment (irinotecan 180 mg/m^2^, 5-FU bolus 400 mg/m^2^, 5-FU infusion 2400 mg/m^2^, leucovorin 200 mg/m^2^, bevacizumab 5 mg/kg) was administered every 2 weeks. FOLFOXIRI + bevacizumab treatment (oxaliplatin 85 mg/m^2^, irinotecan 165 mg/m^2^, 5-FU bolus 400 mg/m^2^, 5-FU infusion 3200 mg/m^2^, leucovorin 200 mg/m^2^, with or without bevacizumab 5 mg/kg) was administered every 2 weeks. Treatment was continued until any of the following occurred: disease progression, unmanageable toxicity, or patient refusal. We were fully compliant with the Reporting Recommendations for Tumor Marker Prognostic Studies (REMARK) guidelines. Tissue analysis was approved by the Institutional Review Board of each institute and was conducted at the University of Southern California/Norris Comprehensive Cancer Center in accordance with the Declaration of Helsinki and Good Clinical Practice Guidelines.

### Single nucleotide polymorphism (SNP) selection

The eight candidate SNPs examined in this study were present in seven genes: *PIN1; TP53; CHE1/apoptosis antagonizing transcription factor* (*AATF*); *casein kinase 2, alpha 1 polypeptide* (*CSNK2A1*); *casein kinase 2, beta polypeptide* (*CSNK2B*); *TOP1;* and *CTIP/retinoblastoma binding protein 8* (*RBBP8*), and were selected by the following criteria: (i) SNPs revealed to be of biological significance according to a literature review; (ii) tagged SNPs using the HapMap genotype data with r^2^ threshold of 0.8: [http://snpinfo.niehs.nih.gov/snpinfo/snptag.htm](http://snpinfo.niehs.nih.gov/snpinfo/snptag.htm); or (iii) SNPs with a cut-off for minor allele frequency of at least 10% in both Caucasians and East Asians (in the Ensembl Genome Browser: [http://uswest.ensembl.org/index.html](http://uswest.ensembl.org/index.html)). Functional significance was predicted based on the functional SNP (F-SNP) database: [http://compbio.cs.queensu.ca/F-SNP/](http://compbio.cs.queensu.ca/F-SNP/) ([Supplementary Table S1](#SD1)).

### DNA extraction and genotyping

Genomic DNA was isolated from peripheral whole blood samples from the patients in the TRIBE cohorts and from formalin-fixed paraffin-embedded tissues from the patients in the Ox-cohort 1, Ox-cohort 2 and the Iri-cohort 2. DNA was extracted using a QIAmp Kit (Qiagen, Valencia, CA, USA) according to the manufacturer’s protocol. Genotyping of the selected SNPs was performed by PCR-based direct DNA sequence analysis using an ABI 3100A Capillary Genetic Analyzer and Sequencing Scanner v1.0 (Applied Biosystems, Life Technologies, Grand Island, NY, USA). The extracted DNA samples were amplified using forward and reverse primers for each SNP. For quality control purposes, 10% of the samples were randomly selected and analyzed for each SNP by direct DNA sequencing; this yielded genotype concordance rates of at least 99%. SNP analysis was performed using an ABI Sequencing Scanner v1.0 (Applied Biosystems) and carried out by investigators blinded to the patients’ clinical data.

### Immunohistochemistry

To determine whether the expression of p53 correlated with the *PIN1* polymorphisms, we performed immunohistochemical staining of formalin-fixed, paraffin-embedded tumor sections using an anti-p53 protein antibody (D0-7; DAKO, Santa Clara, CA, USA). A pathologist assessed 131 of 146 samples from the Ox-cohort 1 who received oxaliplatin-based treatment at the Cancer Institute Hospital in Japan. Tumors from the remaining 15 patients were not analyzed due to a lack of tissue. Three patterns of p53 staining were identified: strong nuclear staining was defined as ‘overexpression (high expression)’, moderate or mild nuclear staining was defined as ‘wild-type expression’, and no nuclear staining was defined as ‘loss (low expression)’.

### Statistical analysis

The study endpoints were progression-free survival (PFS), overall survival (OS), and tumor response. PFS was defined as the interval between the date of starting treatment (oxaliplatin cohorts) or randomization (TRIBE and FIRE-3 cohorts) and the date of confirmed disease progression or death. For patients who had no disease progression or who died, PFS was censored at the date of last follow-up. OS was calculated from the date of starting treatment (Ox-cohorts 1 and 2) or randomization (TRIBE and FIRE-3 cohorts) until the date of death from any cause. For patients who were lost to follow-up, data were censored on the date when the patient was last confirmed to be alive. The objective response rate was calculated from the number of patients who achieved complete response (CR) and partial response (PR) to treatment, according to RECIST v1.0, among all patients whose tumor responses were evaluated. The patient baseline characteristics were compared across cohorts using Chi-square and Kruskal–Wallis tests, as appropriate. The allelic distribution of SNPs by ethnicity for deviation from the Hardy–Weinberg equilibrium and the allelic frequencies of the SNPs were examined by exact tests. The associations between SNPs and baseline characteristics were examined by Fisher’s exact test. Kaplan–Meier curves were constructed and analyzed by the log-rank test to estimate the associations between the candidate SNPs and PFS or OS. A multivariable Cox proportional hazards model was used to adjust for factors influencing the association between SNPs and PFS or OS. Baseline patient demographic and clinical characteristics that were significantly associated with PFS or OS at *P* < 0.1 were included in the multivariable analyses. Fisher’s exact test was used to examine associations between SNPs and tumor responses. Co-dominant, dominant, or recessive genetic models were used for SNPs, because no true model has yet been established. The study had 80% power to detect a minimum hazard ratio (HR) of 1.77 to 2.03 for the association between a SNP and PFS when the minor allele frequencies ranged from 0.1 to 0.4 in the Ox-cohort 1 (*n* = 146, PFS events: 98) using a dominant model and a two-sided log-rank test at the 0.05 level, and 80% power to detect a minimum HR of 1.53 to 1.72 for the association between a SNP and PFS when the minor allele frequencies ranged from 0.1 to 0.4 in the Iri-cohort 1 (*n* = 228, PFS events: 174) using a dominant model and a two-sided log-rank test at the 0.05 level. The power for detecting a minimum HR of 1.77 to 2.03 was at least 44% for the same SNPs with the same minor allele frequencies in the Ox-cohort 2 (*n* = 70, PFS events: 43), 97% in the irinotecan cohort 2 (*n* = 276, PFS events: 236), and 93% in the combination cohort (*n* = 230, PFS events: 162) using the same tests. All analyses were carried out with SAS 9.4 software (SAS Institute, Cary, NC, USA). All tests were two-sided at a significance level of 0.05.

## Results

### Baseline demographics and clinical characteristics

The baseline characteristics of the five cohorts are summarized in [Table 1](#T1). The allelic frequencies and genotype distribution of each SNP met the reported results for Japanese or Caucasian populations, and all candidate SNPs were genotyped at a success rate of ≥93% in all cohorts, based on the quantity or quality of the extracted genomic DNA. The allelic frequencies for all candidate SNPs analyzed in this study were within the Hardy–Weinberg equilibrium (*P*>0.05) in all cohorts.

### Table 1.

| Characteristics | Oxaliplatin cohort 1(N=146) |  | Oxaliplatin cohort 2(N=70) |  | Irinotecan cohort 1(N=228) |  | Irinotecan cohort 2(N=276) |  | Combination cohort(N=230) |  | P value a |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | N | % | N | % | N | % | N | % | N | % |   |
| Sex |   |   |   |   |   |   |   |   |   |   | < 0.001 |
| Male | 71 | 49 | 30 | 43 | 138 | 61 | 185 | 67 | 137 | 60 |   |
| Female | 75 | 51 | 40 | 57 | 90 | 39 | 91 | 33 | 93 | 40 |   |
| Age (years) |   |   |   |   |   |   |   |   |   |   |   |
| Median (range) | 61 (27 – 75) |  | 60 (32 – 79) |  | 60 (29 – 75) |  | 64 (31 – 76) |  | 60 (29 – 75) |  | < 0.001 |
| ≤ 65 | 105 | 72 | 48 | 69 | 163 | 71 | 146 | 53 | 156 | 68 |   |
| > 65 | 41 | 28 | 22 | 31 | 65 | 29 | 130 | 47 | 74 | 32 |   |
| ECOG Performance status |   |   |   |   |   |   |   |   |   |   | < 0.001 |
| 0 | 128 | 88 | 70 | 100 | 188 | 83 | 151 | 55 | 205 | 89 |   |
| 1-2 | 18 | 12 | 0 | 0 | 39 | 17 | 125 | 45 | 25 | 11 |   |
| Primary tumor site |   |   |   |   |   |   |   |   |   |   | < 0.001 |
| Right | 41 | 28 | 24 | 34 | 57 | 27 | 60 | 26 | 78 | 36 |   |
| Left | 105 | 72 | 46 | 66 | 156 | 73 | 170 | 74 | 141 | 64 |   |
| Number of metastases |   |   |   |   |   |   |   |   |   |   | 0.015 |
| 1 | 48 | 33 | 30 | 43 | 99 | 43 | 143 | 52 | 100 | 43 |   |
| 2 | 61 | 42 | 30 | 43 | 90 | 39 | 82 | 30 | 91 | 40 |   |
| ≥ 3 | 37 | 25 | 10 | 14 | 39 | 17 | 51 | 18 | 39 | 17 |   |
| Liver metastasis |   |   |   |   |   |   |   |   |   |   | < 0.001 |
| Yes | 80 | 55 | 37 | 53 | 72 | 32 | 92 | 33 | 80 | 35 |   |
| No | 66 | 45 | 33 | 47 | 156 | 68 | 184 | 67 | 150 | 65 |   |
| Primary tumor resection |   |   |   |   |   |   |   |   |   |   | < 0.001 |
| Yes | 116 | 79 | 54 | 77 | 144 | 63 | 239 | 87 | 157 | 68 |   |
| No | 30 | 21 | 16 | 23 | 84 | 37 | 37 | 13 | 73 | 32 |   |
| Adjuvant chemotherapy |   |   |   |   |   |   |   |   |   |   | 0.055 |
| Yes | 24 | 16 | 17 | 24 | 28 | 12 | 51 | 18 | 29 | 13 |   |
| No | 122 | 84 | 53 | 76 | 200 | 88 | 225 | 82 | 201 | 87 |   |
| KRAS exon2 status |   |   |   |   |   |   |   |   |   |   | < 0.001 |
| Wild-type | 38 | 68 | 29 | 43 | 96 | 51 | 236 | 86 | 90 | 47 |   |
| Mutant | 18 | 32 | 39 | 57 | 93 | 49 | 40 | 14 | 101 | 53 |   |

Table 1 Caption: Comparisons of baseline demographic and clinical characteristics among the patient cohorts

### Candidate polymorphisms and clinical outcomes in oxaliplatin cohorts

Univariate analysis of the Ox-cohort 1 revealed that patients carrying any C allele in *PIN1* rs2233678 had significantly shorter PFS (7.4 vs. 15.0 months, HR 3.24, 95% CI 1.60–6.54, *P*<0.001) and OS (16.9 vs. 31.5 months, HR 2.38, 95% CI 1.32–4.30, *P*=0.003) than patients with the G/G variant. These effects remained significant in the multivariable analysis at *P*=0.009 and *P* = 0.044 for PFS and OS, respectively. Regarding tumor response, patients with the C/C variant in *TP53* rs1042522 achieved a significantly higher response rate than patients with any G allele (74% vs. 46%, *P*=0.001) ([Figure 2A](#F2); [Table 2](#T2); [Supplementary Table S2](#SD1)). In the Ox-cohort 2, patients carrying the *PIN1* rs2233678 G/C allele had shorter OS than patients with the G/G variant (17.3 vs. 36.0 months, HR 2.43, 95% CI 0.83–7.15, *P*=0.087); in the multivariable model, the effect remained marginally significant (HR 3.01, 95% CI 0.98–9.20, *P*=0.054) ([Table 2](#T2)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97dc/6151284/54f44e680b54/nihms959884f2.jpg)

Progression-free survival (PFS) and overall survival (OS) in three cohorts: oxaliplatin cohort 1 (A); irinotecan cohort 1 (B); and combination cohort (C) according to G/G or any C PIN1 rs2233678 alleles.

### Table 2.

|   |   | Tumor response |  | Progression-free Survival |  |  | Overall Survival |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | N | CR+PR | SD+PD | Median, months (95%CI) | HR (95%CI) † | HR (95%CI) ‡ | Median, months (95%CI) | HR (95%CI) † | HR (95%CI) ‡ |
| Oxaliplatin cohort 1 |  |  |  |  |  |  |  |  |  |
| PIN1 rs2233678 |   |   |   |   |   |   |   |   |   |
| G/G | 129 | 74 (60%) | 50 (40%) | 15.0 (12.6, 18.8) | 1 (Reference) | 1 (Reference) | 31.5 (28.1, 42.1) | 1 (Reference) | 1 (Reference) |
| G/C a | 13 | 6 (46%) | 7 (54%) | 7.4 (2.0, 13.1) | 3.24 (1.60, 6.54) | 2.67 (1.28, 5.57) | 16.9 (6.2, 23.4) | 2.38 (1.32, 4.30) | 1.91 (1.02, 3.59) |
| C/C a | 1 | 0 | 1 (100%) |   |   |   |   |   |   |
| P value* |   |   | 0.30 |   | <0.001 | 0.009 |   | 0.003 | 0.044 |
| TP53 rs1042522 |   |   |   |   |   |   |   |   |   |
| C/C | 56 | 40 (74%) | 14 (26%) | 13.3 (9.8, 18.5) | 1 (Reference) | 1 (Reference) | 29.1 (22.7, 31.5) | 1 (Reference) | 1 (Reference) |
| C/G | 70 | 31 (46%) | 36 (54%) | 14.1 (9.3, 19.4) | 0.90 (0.58, 1.39) | 0.87 (0.54, 1.38) | 35.0 (20.1, 42.0) | 0.96 (0.63, 1.47) | 0.93 (0.60, 1.43) |
| G/G | 16 | 7 (44%) | 9 (56%) | 13.1 (5.4, 30.3) | 1.04 (0.52, 2.10) | 0.96 (0.47, 1.97) | 32.3 (13.3, 58.5) | 0.71 (0.33, 1.51) | 0.62 (0.29, 1.35) |
| P value* |   |   | 0.005 |   | 0.84 | 0.83 |   | 0.66 | 0.48 |
| C/C | 56 | 40 (74%) | 14 (26%) | 13.3 (9.8, 18.5) | 1 (Reference) | 1 (Reference) | 29.1 (22.7, 31.5) | 1 (Reference) | 1 (Reference) |
| Any G | 86 | 38 (46%) | 45 (54%) | 13.8 (10.1, 17.3) | 0.92 (0.60, 1.40) | 0.88 (0.57, 1.37) | 35.0 (20.8, 42.4) | 0.92 (0.61, 1.38) | 0.87 (0.57, 1.32) |
| P value* |   |   | 0.001 |   | 0.68 | 0.58 |   | 0.67 | 0.50 |
| Oxaliplatin cohort 2 |  |  |  |  |  |  |  |  |  |
| PIN1 rs2233678 |   |   |   |   |   |   |   |   |   |
| G/G | 64 | 26 (41%) | 38 (59%) | 13.0 (9.6, 17.7) | 1 (Reference) | 1 (Reference) | 36.0 (26.5, 49.8) | 1 (Reference) | 1 (Reference) |
| G/C | 6 | 2 (33%) | 4 (67%) | 13.1 (5.6, 35.0) | 1.11 (0.43, 2.82) | 1.15 (0.44, 2.98) | 17.3 (11.9, 35.0) | 2.43 (0.83, 7.15) | 3.01 (0.98, 9.20) |
| P value* |   |   | 1.00 |   | 0.83 | 0.78 |   | 0.087 | 0.054 |
| Irinotecan cohort 1 |  |  |  |  |  |  |  |  |  |
| PIN1 rs2233678 |   |   |   |   |   |   |   |   |   |
| G/G | 167 | 86 (53%) | 75 (47%) | 9.4 (8.7, 10.4) | 1 (Reference) | 1 (Reference) | 24.0 (20.5, 26.8) | 1 (Reference) | 1 (Reference) |
| G/C a | 56 | 41 (75%) | 14 (25%) | 11.9 (9.5, 13.6) | 0.64 (0.45, 0.91) | 0.61 (0.38, 0.96) | 31.6 (25.2, 37.8) | 0.79 (0.56, 1.11) | 0.73 (0.47, 1.13) |
| C/C a | 5 | 2 (40%) | 3 (60%) |   |   |   |   |   |   |
| P value* |   |   | 0.011 |   | 0.009 | 0.033 |   | 0.17 | 0.15 |
| Irinotecan cohort 2 |  |  |  |  |  |  |  |  |  |
| PIN1 rs2233678 |   |   |   |   |   |   |   |   |   |
| G/G | 224 | 123 (59%) | 84 (41%) | 9.9 (9.0, 10.8) | 1 (Reference) | 1 (Reference) | 24.8 (21.5, 26.7) | 1 (Reference) | 1 (Reference) |
| G/C a | 46 | 24 (59%) | 17 (41%) | 11.8 (8.9, 13.4) | 0.98 (0.71, 1.36) | 1.00 (0.73, 1.39) | 23.1 (17.4, 31.5) | 1.03 (0.71, 1.50) | 1.11 (0.76, 1.62) |
| C/C a | 6 | 4 (67%) | 2 (33%) |   |   |   |   |   |   |
| P value* |   |   | 1.00 |   | 0.91 | 0.98 |   | 0.86 | 0.59 |
| Combination cohort |  |  |  |  |  |  |  |  |  |
| PIN1 rs2233678 |   |   |   |   |   |   |   |   |   |
| G/G | 171 | 113 (68%) | 53 (32%) | 11.6 (10.3, 13.0) | 1 (Reference) | 1 (Reference) | 30.6 (24.0, 35.4) | 1 (Reference) | 1 (Reference) |
| G/C a | 54 | 32 (63%) | 19 (37%) | 12.0 (9.5, 16.9) | 0.99 (0.70, 1.41) | 1.18 (0.77, 1.82) | 28.6 (23.6, 34.7) | 1.09 (0.76, 1.55) | 1.07 (0.71, 1.61) |
| C/C a | 4 | 4 (100%) | 0 |   |   |   |   |   |   |
| P value* |   |   | 0.36 |   | 0.95 | 0.45 |   | 0.65 | 0.76 |

Table 2 Caption: Associations between PIN1-related genes and clinical outcomes

### Candidate polymorphisms and clinical outcomes in irinotecan cohorts

Univariate analysis of the Iri-cohort 1 revealed that patients carrying any C allele in *PIN1* rs2233678 had significantly longer PFS (11.9 vs. 9.4 months, HR 0.64, 95% CI 0.45–0.91, *P*=0.009) than patients with the G/G variant. Although the patients carrying a C allele also had longer OS, the difference did not reach the level of statistical significance (31.6 vs. 24.0 months, HR 0.79, 95% CI 0.56–1.11, *P* = 0.17) ([Figure 2B](#F2)). In the multivariable analysis, *PIN1* rs2233678 remained significantly associated with PFS (HR 0.61, 95% CI 0.38–0.96, *P*=0.033) ([Table 2](#T2); [Supplementary Table S3](#SD1)). In the analysis of tumor response, any *PIN1* rs2233678 C allele variant was significantly associated with better tumor response compared with the G/G variant (Fisher’s exact *P*=0.011). In the Iri-cohort 2, no significant associations were observed between SNPs and tumor response or outcomes ([Table 2](#T2)).

### Candidate polymorphisms and clinical outcomes in combination (Ox + Iri) cohort

Univariate and multivariate analyses showed no significant differences between SNPs and PFS, OS, or tumor response ([Table 2](#T2); [Figure 2C](#F2)).

### Comparison between FOLFIRI + bevacizumab and FOLFOXIRI + bevacizumab according to PIN1 polymorphisms in TRIBE trial

In comparison of the Iri-cohort 1 and Ox + Iri-cohort (TRIBE arm A vs. B), the patients with the G/G variant in *PIN1* rs2233678 benefitted from FOLFOXIRI + bevacizumab compared with FOLFIRI + bevacizumab for clinical outcomes (PFS, 11.6 vs. 9.4 months, HR 0.61, 95%CI: 0.47–0.78, *P*<0.001; OS, 30.6 vs. 24.0 months, HR 0.79, 95%CI: 0.62–1.02, *P*=0.060); while no significant survival benefit was shown in any C allele ([Figures 3A](#F3) and [3B](#F3); [Supplementary Table S4](#SD1)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97dc/6151284/8723a8bbacde/nihms959884f3.jpg)

Progression-free survival (PFS) and overall survival (OS) in treatment arms in TRIBE study according to G/G or any C PIN1 rs2233678 alleles: arm A receiving FOLFIRI + bevacizumab (irinotecan cohort 1); arm B receiving FOLFOXIRI + bevacizumab (combination cohort).

### Associations between p53 immunohistochemistry and PIN1 polymorphisms

Patients in the Ox-cohort 1 were assigned to three groups based on immunohistochemical staining for p53 in 131 tissue samples: 69 patients showed high expression, 37 showed wild-type expression, and 25 showed low expression. Patients in the wild-type or low expression groups had longer PFS and OS, but the effects were not significant ([Supplementary Table S5](#SD1)). Interaction analyses between p53 staining status and *TP53* rs1042522 or *PIN1* rs2233678 were performed. Interaction terms for p53 and SNPs were included in the multivariable model adjusted for baseline characteristics and estimated using the maximum likelihood test. However, no significant interactions were observed between p53 expression and these SNPs.

## Discussion

We report here the first evidence that *PIN1* gene polymorphisms may influence the antitumor activity of oxaliplatin in first-line treatment for mCRC. The SNPs potentially predicted the outcomes for efficacy of oxaliplatin-based treatment and additional effect of oxaliplatin in combination treatment with irinotecan.

In the TRIBE trial, first-line FOLFOXIRI + bevacizumab significantly improved PFS and OS compared with FOLFIRI + bevacizumab in mCRC patients.^24^ To our knowledge, there have been no reports of useful biological markers that can distinguish clinical outcomes between them. In addition, it remains unknown whether the antitumor effect of the combination regimen is based on a synergistic action. .

Recent data in a review showed that PIN1 acts as a post-phosphorylation regulator of biological function of its substrate proteins and alters their conformation.^5,26^ The conformational changes can modulate biological activity, bioavailability, phosphorylation status, protein-protein interactions, protein stability, and subcellular localization, leading to regulation of various cellular events such as transcriptional regulation, cell proliferation, and mitosis.^27^ Lu et al. analyzed the allele-specific effects of *PIN1* rs2233678 variants on promoter activity using a luciferase reporter assay, and showed that the G allele was more highly associated with *PIN1* gene expression compared with the C allele. The authors suggested that the change from G to C might diminish binding of regulatory proteins to the *PIN1* promoter, although differences in *PIN1* mRNA levels in cells with different *PIN1* rs2233678 genotypes were not assessed. In addition, the C allele was associated with a decreased risk of squamous cell carcinoma of the head and neck, indicating that the G/G variant was a high risk factor.^28^

Our data showed that the dominant *PIN1* rs2233678 G/G variant was associated with the outcome of oxaliplatin-based treatment; that is, G/G was favorable for oxaliplatin treatment. To our knowledge, there is currently no strong evidence of an association between *PIN1* polymorphisms and sensitivity to chemotherapeutic agents. The findings from the functional analysis described above shed light on the role of PIN1 in the DDR to oxaliplatin treatment.^28^ The PIN1-mediated conformational change in CtIP restricts homologous recombination while simultaneously promoting NHEJ. A recent in vitro study demonstrated that depletion of PIN1 significantly decreased NHEJ frequency, whereas PIN1 overexpression markedly reduced homologous recombination by diminishing the formation of single-stranded DNA.^13^ These data support our findings of better outcomes for patients with the *PIN1* rs2233678 G/G variant following oxaliplatin-based treatment, and suggest the possibility that the high expression of PIN1 in these patients inhibits CtIP and thus inactivates homologous recombination ([Figure 1C](#F1)). We found no interaction effects between p53 staining and *TP53* rs1042522 and *PIN1* rs2233678, suggesting that the PIN1-induced change in conformation of p53 does not affect its expression. A recent study also investigated the correlation between *TP53* rs1042522 alleles and p53 expression by immunohistochemical staining, and observed no association.^29^ However, the diverse and varied effects of oncologic signaling resulting from a PIN1-induced change in p53 conformation remain to be clarified. Thus, we assume that dysfunction in the DDR to oxaliplatin-induced DSBs is the main reason for the increased sensitivity to the oxaliplatin-based regimens.

In our study, the *PIN1* rs2233678 G/G variant had potential opposite effects on sensitivity to oxaliplatin and irinotecan between the Ox-cohort 1 and Iri-cohort 1; these findings are sustained by previous reports ^18,19^ that PIN1 isomerization of CK2α inhibits Top1 activity ([Figure 1D](#F1)). Given the effect was not confirmed in the Iri-cohort 2, a further validation study is warranted to explore whether the PIN1-mediated isomerization of CK2α varying Top1 activity is associated with clinical outcome. Intriguingly, the combination cohort and Iri-cohort 1 in the TRIBE trial had comparable effects on PFS and OS in patients with any *PIN1* rs2233678 C allele, whereas patients carrying the G/G variant continued to have a significant advantage in adding oxaliplatin to irinotecan. Consequently, we assume that the preferable effect of oxaliplatin is most likely to appear in patients with the G/G variant when receiving oxaliplatin-based treatment, and which confers improved outcomes when combining oxaliplatin to irinotecan-based treatment.

Our study has some limitations. The sample size was limiting for detecting small percentages of minor alleles. For example, there was no statistical significance in PFS for *PIN1* rs2233678 in the Ox-cohort 2; however, the trend in HR was consistent with that in Ox-cohort 1 as well as tumor response. Nevertheless, we need a further study to confirm the predictive value of *PIN1* rs2233678 for oxaliplatin-based treatment. The study lacked in vitro and in vivo results. The patients in the oxaliplatin cohorts were Asian (Japanese) while those in the irinotecan cohorts were Caucasians (Italian or German); and the population homogeneity in each cohort might have affected the allele frequency and outcomes. However, we observed no significant associations between *PIN1* rs2233678 genotypes and other prognostic variables at baseline in our study. In contrast, our study has a number of strengths, including the presence of cohorts derived from two phase III studies, the presence of validation cohorts with comparable baseline characteristics, immunochemical confirmation of correlations between *PIN1* genotype and protein expression. In addition, the superiority of FOLFOXIRI to FOLFIRI in PFS and OS in the phase III TRIBE trial was reproduced, when the study population was limited to patients carrying the *PIN1* rs2233678 G/G variant. This strongly suggests the clinical importance of oxaliplatin in combination treatment in first-line treatment for mCRC.

In conclusion, germline *PIN1* polymorphisms may predict especially oxaliplatin sensitivity rather than irinotecan in mCRC patients. Further validation studies comparing oxaliplatin- or irinotecan-based treatment to combination treatment is warranted to verify the unique multifunctionality of *PIN1* SNPs for providing an optimal treatment plan for selected patients.

## Supplementary Material

## Acknowledgments

The National Institutes of Health (P30CA014089-27S1), the Gloria Borges Wunderglo Project, the Dhont Family Foundation, the Dave Butler Research Fund, and the Call to Cure Research Fund partially supported this work. Mitsukuni Suenaga received a grant from the Takashi Tsuruo Memorial Fund. Martin D. Berger received a grant from the Swiss Cancer League (BIL KLS-3334-02-2014) and the Werner and Hedy Berger-Janser Foundation for Cancer Research. Yuji Miyamoto received a grant from the Japan Society for the Promotion of Science (S2606).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Govindaraghavan M, Lad AA, Osmani SA. The NIMA kinase is required to execute stage-specific mitotic functions after initiation of mitosis. Eukaryot Cell. 2014;13:99–109. doi: 10.1128/EC.00231-13.  [DOI](https://doi.org/10.1128/EC.00231-13) | [PMC free article](/articles/PMC3910965/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24186954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eukaryot%20Cell&title=The%20NIMA%20kinase%20is%20required%20to%20execute%20stage-specific%20mitotic%20functions%20after%20initiation%20of%20mitosis&author=M%20Govindaraghavan&author=AA%20Lad&author=SA%20Osmani&volume=13&publication_year=2014&pages=99-109&pmid=24186954&doi=10.1128/EC.00231-13&)

2. O’regan L, Blot J, Fry AM. Mitotic regulation by NIMA-related kinases. Cell Div. 2007;2:25. doi: 10.1186/1747-1028-2-25.  [DOI](https://doi.org/10.1186/1747-1028-2-25) | [PMC free article](/articles/PMC2018689/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17727698/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Div&title=Mitotic%20regulation%20by%20NIMA-related%20kinases&author=L%20O%E2%80%99regan&author=J%20Blot&author=AM%20Fry&volume=2&publication_year=2007&pages=25&pmid=17727698&doi=10.1186/1747-1028-2-25&)

3. Lu KP, Hunter T. Evidence for a NIMA-like mitotic pathway in vertebrate cells. Cell. 1995;81:413–424. doi: 10.1016/0092-8674(95)90394-1.  [DOI](https://doi.org/10.1016/0092-8674(95)90394-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7736593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Evidence%20for%20a%20NIMA-like%20mitotic%20pathway%20in%20vertebrate%20cells&author=KP%20Lu&author=T%20Hunter&volume=81&publication_year=1995&pages=413-424&pmid=7736593&doi=10.1016/0092-8674(95)90394-1&)

4. Osmani AH, O’Donnell K, Pu RT, Osmani SA. Activation of the nimA protein kinase plays a unique role during mitosis that cannot be bypassed by absence of the bimE checkpoint. EMBO J. 1991;10:2669–2679. doi: 10.1002/j.1460-2075.1991.tb07810.x.  [DOI](https://doi.org/10.1002/j.1460-2075.1991.tb07810.x) | [PMC free article](/articles/PMC452969/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1868838/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Activation%20of%20the%20nimA%20protein%20kinase%20plays%20a%20unique%20role%20during%20mitosis%20that%20cannot%20be%20bypassed%20by%20absence%20of%20the%20bimE%20checkpoint&author=AH%20Osmani&author=K%20O%E2%80%99Donnell&author=RT%20Pu&author=SA%20Osmani&volume=10&publication_year=1991&pages=2669-2679&pmid=1868838&doi=10.1002/j.1460-2075.1991.tb07810.x&)

5. Ryo A, Liou YC, Lu KP, Wulf G. Prolyl isomerase Pin1: a catalyst for oncogenesis and a potential therapeutic target in cancer. J Cell Sci. 2003;116:773–783. doi: 10.1242/jcs.00276.  [DOI](https://doi.org/10.1242/jcs.00276) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12571275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&title=Prolyl%20isomerase%20Pin1:%20a%20catalyst%20for%20oncogenesis%20and%20a%20potential%20therapeutic%20target%20in%20cancer&author=A%20Ryo&author=YC%20Liou&author=KP%20Lu&author=G%20Wulf&volume=116&publication_year=2003&pages=773-783&pmid=12571275&doi=10.1242/jcs.00276&)

6. Steger M, Murina O, Hühn D, et al. Prolyl isomerase PIN1 regulates DNA double-strand break repair by counteracting DNA end resection. Mol Cell. 2013;50:333–343. doi: 10.1016/j.molcel.2013.03.023.  [DOI](https://doi.org/10.1016/j.molcel.2013.03.023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23623683/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell&title=Prolyl%20isomerase%20PIN1%20regulates%20DNA%20double-strand%20break%20repair%20by%20counteracting%20DNA%20end%20resection&author=M%20Steger&author=O%20Murina&author=D%20H%C3%BChn&volume=50&publication_year=2013&pages=333-343&pmid=23623683&doi=10.1016/j.molcel.2013.03.023&)

7. Jackson SP, Bartek J. The DNA-damage response in human biology and disease. Nature. 2009;461:1071–1078. doi: 10.1038/nature08467.  [DOI](https://doi.org/10.1038/nature08467) | [PMC free article](/articles/PMC2906700/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19847258/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20DNA-damage%20response%20in%20human%20biology%20and%20disease&author=SP%20Jackson&author=J%20Bartek&volume=461&publication_year=2009&pages=1071-1078&pmid=19847258&doi=10.1038/nature08467&)

8. Branzei D, Foiani M. Regulation of DNA repair throughout the cell cycle. Nat Rev Mol Cell Biol. 2008;9:297–308. doi: 10.1038/nrm2351.  [DOI](https://doi.org/10.1038/nrm2351) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18285803/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&title=Regulation%20of%20DNA%20repair%20throughout%20the%20cell%20cycle&author=D%20Branzei&author=M%20Foiani&volume=9&publication_year=2008&pages=297-308&pmid=18285803&doi=10.1038/nrm2351&)

9. Vousden KH, Lu X. Live or let die: the cell’s response to p53. Nat Rev Cancer. 2002;2:594–604. doi: 10.1038/nrc864.  [DOI](https://doi.org/10.1038/nrc864) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12154352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=Live%20or%20let%20die:%20the%20cell%E2%80%99s%20response%20to%20p53&author=KH%20Vousden&author=X%20Lu&volume=2&publication_year=2002&pages=594-604&pmid=12154352&doi=10.1038/nrc864&)

10. Wulf GM, Liou YC, Ryo A, Lee SW, Lu KP. Role of Pin1 in the regulation of p53 stability and p21 transactivation, and cell cycle checkpoints in response to DNA damage. J Biol Chem. 2002;277:47976–47979. doi: 10.1074/jbc.C200538200.  [DOI](https://doi.org/10.1074/jbc.C200538200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12388558/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Role%20of%20Pin1%20in%20the%20regulation%20of%20p53%20stability%20and%20p21%20transactivation,%20and%20cell%20cycle%20checkpoints%20in%20response%20to%20DNA%20damage&author=GM%20Wulf&author=YC%20Liou&author=A%20Ryo&author=SW%20Lee&author=KP%20Lu&volume=277&publication_year=2002&pages=47976-47979&pmid=12388558&doi=10.1074/jbc.C200538200&)

11. Desantis A, Bruno T, Catena V, De Nicola F, Goeman F, Iezzi S, et al. Che-1 modulates the decision between cell cycle arrest and apoptosis by its binding to p53. Cell Death Dis. 2015;6:e1764. doi: 10.1038/cddis.2015.117.  [DOI](https://doi.org/10.1038/cddis.2015.117) | [PMC free article](/articles/PMC4669697/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25996291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Dis&title=Che-1%20modulates%20the%20decision%20between%20cell%20cycle%20arrest%20and%20apoptosis%20by%20its%20binding%20to%20p53&author=A%20Desantis&author=T%20Bruno&author=V%20Catena&author=F%20De%20Nicola&author=F%20Goeman&volume=6&publication_year=2015&pages=e1764&pmid=25996291&doi=10.1038/cddis.2015.117&)

12. Berger M, Stahl N, Del Sal G, Haupt Y. Mutations in proline 82 of p53 impair its activation by Pin1 and Chk2 in response to DNA damage. Mol Cell Biol. 2005;25:5380–5388. doi: 10.1128/MCB.25.13.5380-5388.2005.  [DOI](https://doi.org/10.1128/MCB.25.13.5380-5388.2005) | [PMC free article](/articles/PMC1156984/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15964795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=Mutations%20in%20proline%2082%20of%20p53%20impair%20its%20activation%20by%20Pin1%20and%20Chk2%20in%20response%20to%20DNA%20damage&author=M%20Berger&author=N%20Stahl&author=G%20Del%20Sal&author=Y%20Haupt&volume=25&publication_year=2005&pages=5380-5388&pmid=15964795&doi=10.1128/MCB.25.13.5380-5388.2005&)

13. Sartori AA, Steger M. Prolyl isomerization: a new PIN code for DSB repair. Cell Cycle. 2013;12:2717–2718. doi: 10.4161/cc.26077.  [DOI](https://doi.org/10.4161/cc.26077) | [PMC free article](/articles/PMC3899182/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23966151/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Cycle&title=Prolyl%20isomerization:%20a%20new%20PIN%20code%20for%20DSB%20repair&author=AA%20Sartori&author=M%20Steger&volume=12&publication_year=2013&pages=2717-2718&pmid=23966151&doi=10.4161/cc.26077&)

14. Pinna LA. Protein kinase CK2: a challenge to canons. J Cell Sci. 2002;115:3873–3878. doi: 10.1242/jcs.00074.  [DOI](https://doi.org/10.1242/jcs.00074) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12244125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&title=Protein%20kinase%20CK2:%20a%20challenge%20to%20canons&author=LA%20Pinna&volume=115&publication_year=2002&pages=3873-3878&pmid=12244125&doi=10.1242/jcs.00074&)

15. Ahmad KA, Wang G, Unger G, Slaton J, Ahmed K. Protein kinase CK2–a key suppressor of apoptosis. Adv Enzyme Regul. 2008;48:179–187. doi: 10.1016/j.advenzreg.2008.04.002.  [DOI](https://doi.org/10.1016/j.advenzreg.2008.04.002) | [PMC free article](/articles/PMC2593134/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18492491/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Enzyme%20Regul&title=Protein%20kinase%20CK2%E2%80%93a%20key%20suppressor%20of%20apoptosis&author=KA%20Ahmad&author=G%20Wang&author=G%20Unger&author=J%20Slaton&author=K%20Ahmed&volume=48&publication_year=2008&pages=179-187&pmid=18492491&doi=10.1016/j.advenzreg.2008.04.002&)

16. Duncan JS, Litchfield DW. Too much of a good thing: the role of protein kinase CK2 in tumorigenesis and prospects for therapeutic inhibition of CK2. Biochim Biophys Acta. 2008;1784:33–47. doi: 10.1016/j.bbapap.2007.08.017.  [DOI](https://doi.org/10.1016/j.bbapap.2007.08.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17931986/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Too%20much%20of%20a%20good%20thing:%20the%20role%20of%20protein%20kinase%20CK2%20in%20tumorigenesis%20and%20prospects%20for%20therapeutic%20inhibition%20of%20CK2&author=JS%20Duncan&author=DW%20Litchfield&volume=1784&publication_year=2008&pages=33-47&pmid=17931986&doi=10.1016/j.bbapap.2007.08.017&)

17. Homma MK, Homma Y. Cell cycle and activation of CK2. Mol Cell Biochem. 2008;316:49–55. doi: 10.1007/s11010-008-9823-4.  [DOI](https://doi.org/10.1007/s11010-008-9823-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18649047/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biochem&title=Cell%20cycle%20and%20activation%20of%20CK2&author=MK%20Homma&author=Y%20Homma&volume=316&publication_year=2008&pages=49-55&pmid=18649047&doi=10.1007/s11010-008-9823-4&)

18. Messenger MM, Saulnier RB, Gilchrist AD, Diamond P, Gorbsky GJ, Litchfield DW. Interactions between protein kinase CK2 and Pin1. Evidence for phosphorylation-dependent interactions. J Biol Chem. 2002;277:23054–23064. doi: 10.1074/jbc.M200111200.  [DOI](https://doi.org/10.1074/jbc.M200111200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11940573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Interactions%20between%20protein%20kinase%20CK2%20and%20Pin1.%20Evidence%20for%20phosphorylation-dependent%20interactions&author=MM%20Messenger&author=RB%20Saulnier&author=AD%20Gilchrist&author=P%20Diamond&author=GJ%20Gorbsky&volume=277&publication_year=2002&pages=23054-23064&pmid=11940573&doi=10.1074/jbc.M200111200&)

19. Bandyopadhyay K, Gjerset RA. Protein kinase CK2 is a central regulator of topoisomerase I hyperphosphorylation and camptothecin sensitivity in cancer cell lines. Biochemistry. 2011;50:704–714. doi: 10.1021/bi101110e.  [DOI](https://doi.org/10.1021/bi101110e) | [PMC free article](/articles/PMC3046806/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21182307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Protein%20kinase%20CK2%20is%20a%20central%20regulator%20of%20topoisomerase%20I%20hyperphosphorylation%20and%20camptothecin%20sensitivity%20in%20cancer%20cell%20lines&author=K%20Bandyopadhyay&author=RA%20Gjerset&volume=50&publication_year=2011&pages=704-714&pmid=21182307&doi=10.1021/bi101110e&)

20. Tournigand C, André T, Achille E, Lledo G, Flesh M, Mery-Mignard D, et al. FOLFIRI followed by FOLFOX6 or the reverse sequence in advanced colorectal cancer: a randomized GERCOR study. J Clin Oncol. 2004;22:229–237. doi: 10.1200/JCO.2004.05.113.  [DOI](https://doi.org/10.1200/JCO.2004.05.113) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14657227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=FOLFIRI%20followed%20by%20FOLFOX6%20or%20the%20reverse%20sequence%20in%20advanced%20colorectal%20cancer:%20a%20randomized%20GERCOR%20study&author=C%20Tournigand&author=T%20Andr%C3%A9&author=E%20Achille&author=G%20Lledo&author=M%20Flesh&volume=22&publication_year=2004&pages=229-237&pmid=14657227&doi=10.1200/JCO.2004.05.113&)

21. Cheung-Ong K, Giaever G, Nislow C. DNA-damaging agents in cancer chemotherapy: serendipity and chemical biology. Chem Biol. 2013;20:648–659. doi: 10.1016/j.chembiol.2013.04.007.  [DOI](https://doi.org/10.1016/j.chembiol.2013.04.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23706631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Biol&title=DNA-damaging%20agents%20in%20cancer%20chemotherapy:%20serendipity%20and%20chemical%20biology&author=K%20Cheung-Ong&author=G%20Giaever&author=C%20Nislow&volume=20&publication_year=2013&pages=648-659&pmid=23706631&doi=10.1016/j.chembiol.2013.04.007&)

22. Faivre S, Chan D, Salinas R, Woynarowska B, Woynarowski JM. DNA strand breaks and apoptosis induced by oxaliplatin in cancer cells. Biochem Pharmacol. 2003;66:225–237. doi: 10.1016/s0006-2952(03)00260-0.  [DOI](https://doi.org/10.1016/s0006-2952(03)00260-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12826265/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=DNA%20strand%20breaks%20and%20apoptosis%20induced%20by%20oxaliplatin%20in%20cancer%20cells&author=S%20Faivre&author=D%20Chan&author=R%20Salinas&author=B%20Woynarowska&author=JM%20Woynarowski&volume=66&publication_year=2003&pages=225-237&pmid=12826265&doi=10.1016/s0006-2952(03)00260-0&)

23. Suenaga M, Mizunuma N, Matsusaka S, Shinozaki E, Ueno M, Yamaguchi T. Retrospective analysis on the efficacy of bevacizumab with FOLFOX as a first-line treatment in Japanese patients with metastatic colorectal cancer. Asia Pac J Clin Oncol. 2014;10:322–329. doi: 10.1111/ajco.12094.  [DOI](https://doi.org/10.1111/ajco.12094) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23915091/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asia%20Pac%20J%20Clin%20Oncol&title=Retrospective%20analysis%20on%20the%20efficacy%20of%20bevacizumab%20with%20FOLFOX%20as%20a%20first-line%20treatment%20in%20Japanese%20patients%20with%20metastatic%20colorectal%20cancer&author=M%20Suenaga&author=N%20Mizunuma&author=S%20Matsusaka&author=E%20Shinozaki&author=M%20Ueno&volume=10&publication_year=2014&pages=322-329&pmid=23915091&doi=10.1111/ajco.12094&)

24. Loupakis F, Cremolini C, Masi G, Lledo G, Flesh M, Mery-Mignard D, et al. Initial therapy with FOLFOXIRI and bevacizumab for metastatic colorectal cancer. N Engl J Med. 2014;371:1609–1618. doi: 10.1056/NEJMoa1403108.  [DOI](https://doi.org/10.1056/NEJMoa1403108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25337750/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Initial%20therapy%20with%20FOLFOXIRI%20and%20bevacizumab%20for%20metastatic%20colorectal%20cancer&author=F%20Loupakis&author=C%20Cremolini&author=G%20Masi&author=G%20Lledo&author=M%20Flesh&volume=371&publication_year=2014&pages=1609-1618&pmid=25337750&doi=10.1056/NEJMoa1403108&)

25. Stintzing S, Modest DP, Rossius L, Lerch MM, von Weikersthal LF, Decker T, et al. FOLFIRI plus cetuximab versus FOLFIRI plus bevacizumab for metastatic colorectal cancer (FIRE-3): a post-hoc analysis of tumour dynamics in the final RAS wild-type subgroup of this randomised open-label phase 3 trial. Lancet Oncol. 2016;17:1426–1434. doi: 10.1016/S1470-2045(16)30269-8.  [DOI](https://doi.org/10.1016/S1470-2045(16)30269-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27575024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=FOLFIRI%20plus%20cetuximab%20versus%20FOLFIRI%20plus%20bevacizumab%20for%20metastatic%20colorectal%20cancer%20(FIRE-3):%20a%20post-hoc%20analysis%20of%20tumour%20dynamics%20in%20the%20final%20RAS%20wild-type%20subgroup%20of%20this%20randomised%20open-label%20phase%203%20trial&author=S%20Stintzing&author=DP%20Modest&author=L%20Rossius&author=MM%20Lerch&author=LF%20von%20Weikersthal&volume=17&publication_year=2016&pages=1426-1434&pmid=27575024&doi=10.1016/S1470-2045(16)30269-8&)

26. Polonio-Vallon T, Krüger D, Hofmann TG. ShaPINg Cell Fate Upon DNA Damage: Role of Pin1 Isomerase in DNA Damage-Induced Cell Death and Repair. Front Oncol. 2014;4:148. doi: 10.3389/fonc.2014.00148.  [DOI](https://doi.org/10.3389/fonc.2014.00148) | [PMC free article](/articles/PMC4058901/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24982848/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&title=ShaPINg%20Cell%20Fate%20Upon%20DNA%20Damage:%20Role%20of%20Pin1%20Isomerase%20in%20DNA%20Damage-Induced%20Cell%20Death%20and%20Repair&author=T%20Polonio-Vallon&author=D%20Kr%C3%BCger&author=TG%20Hofmann&volume=4&publication_year=2014&pages=148&pmid=24982848&doi=10.3389/fonc.2014.00148&)

27. Lu KP, Hanes SD, Hunter T. A human peptidyl-prolyl isomerase essential for regulation of mitosis. Nature. 1996;380:544–547. doi: 10.1038/380544a0.  [DOI](https://doi.org/10.1038/380544a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8606777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20human%20peptidyl-prolyl%20isomerase%20essential%20for%20regulation%20of%20mitosis&author=KP%20Lu&author=SD%20Hanes&author=T%20Hunter&volume=380&publication_year=1996&pages=544-547&pmid=8606777&doi=10.1038/380544a0&)

28. Lu J, Hu Z, Wei S, Lerch MM, von Weikersthal LF, Decker T, et al. A novel functional variant (-842G>C) in the PIN1 promoter contributes to decreased risk of squamous cell carcinoma of the head and neck by diminishing the promoter activity. Carcinogenesis. 2009;30:1717–1721. doi: 10.1093/carcin/bgp171.  [DOI](https://doi.org/10.1093/carcin/bgp171) | [PMC free article](/articles/PMC2757547/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19625347/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=A%20novel%20functional%20variant%20(-842G>C)%20in%20the%20PIN1%20promoter%20contributes%20to%20decreased%20risk%20of%20squamous%20cell%20carcinoma%20of%20the%20head%20and%20neck%20by%20diminishing%20the%20promoter%20activity&author=J%20Lu&author=Z%20Hu&author=S%20Wei&author=MM%20Lerch&author=LF%20von%20Weikersthal&volume=30&publication_year=2009&pages=1717-1721&pmid=19625347&doi=10.1093/carcin/bgp171&)

29. De Oliveira LP, López I, Dos Santos EM, Lerch MM, von Weikersthal LF, Decker T, et al. Association of the p53 codon 72 polymorphism with clinicopathological characteristics of colorectal cancer through mRNA analysis. Oncol Rep. 2014;31:1396–1406. doi: 10.3892/or.2013.2940.  [DOI](https://doi.org/10.3892/or.2013.2940) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24366026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncol%20Rep&title=Association%20of%20the%20p53%20codon%2072%20polymorphism%20with%20clinicopathological%20characteristics%20of%20colorectal%20cancer%20through%20mRNA%20analysis&author=LP%20De%20Oliveira&author=I%20L%C3%B3pez&author=EM%20Dos%20Santos&author=MM%20Lerch&author=LF%20von%20Weikersthal&volume=31&publication_year=2014&pages=1396-1406&pmid=24366026&doi=10.3892/or.2013.2940&)
