# Germline single nucleotide polymorphisms in ERBB3 and BARD1 genes result in a worse relapse free survival response for HER2-positive breast cancer patients treated with adjuvant based docetaxel, carboplatin and trastuzumab (TCH)

## Metadata
**Authors:** Damien Coté, Alex Eustace, Sinead Toomey, Mattia Cremona, Malgorzata Milewska, Simon Furney, Aoife Carr, Joanna Fay, Elaine Kay, Susan Kennedy, John Crown, Bryan Hennessy, Stephen Madden
**Journal:** PLoS ONE
**Date:** 2018 Aug 2
**DOI:** [10.1371/journal.pone.0200996](https://doi.org/10.1371/journal.pone.0200996)
**PMID:** 30071039
**PMCID:** PMC6071997
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6071997/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6071997/pdf/pone.0200996.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6071997/pdf/pone.0200996.pdf)

## Abstract

Breast cancer is the leading cause of cancer related deaths in women worldwide and is classified into subtypes based on the cancer’s receptor status. Of these subtypes, those expressing the human epidermal growth factor receptor 2 (HER2) receptor were traditionally associated with poor prognosis. Several advances have been made in the treatment of HER2-positive breast cancer, yet issues of resistance and poor response to therapy remains prevalent. In this study we explored the impact of HER-family and homologous recombination deficiency SNPs on response to patients who received TCH-based (docetaxel (T), carboplatin (C), and trastuzumab (H)) treatment versus those who received other treatment regimens. Using Cox regression analysis, we identified 6 SNPs that correlate with recurrence free survival in our patients and supported our findings using support vector machines. We also used reverse phase protein array analysis to examine the impact ERBB3 SNPs may have on both the PI3K/AKT and MAPK/ERK signaling pathways. Finally, using cell line models, we correlated SNP status with sensitivity to platinum based drugs and docetaxel. We found that patients on a TCH based regimen with the minor allele of the ERBB3 (rs2229046 and rs773123) and BARD1 (rs2070096) SNPs, were significantly more likely to relapse than those women who were not. Additionally, we observed that patients with these ERBB3 SNPs had shown elevated protein expression/phosphorylation of Src kinase, c-MET (Y1234/1235), GSK-3β (S9) and p27, indicating that these SNPs are associated with non-PI3K/AKT signaling. Finally, using cell line models, we demonstrate that the BARD1 SNP (rs2229571) is associated with greater sensitivity to both carboplatin and cisplatin. The BARD1 and ERBB3 SNPs can potentially be used to determine those patients that will have a worse response to TCH based treatment, an effect that may arise from the SNPs impact on altered cellular signaling.

## Introduction

HER-2 positive breast cancer comprises cancers which exhibit the overexpression or amplification of the Erb-B2 Receptor Tyrosine Kinase 2 (*ERBB2*) gene (also known as the HER2 protein) and accounts for approximately 20% of all breast cancers. It is associated with a significantly worse survival [[1](#pone.0200996.ref001),[2](#pone.0200996.ref002)]. Current adjuvant treatment strategies for women with HER2-positive breast cancer after their surgery include the use of trastuzumab (H), a humanized monoclonal antibody in combination with anthracyclines (A), taxanes (T) and platinum salts (C). Recent clinical studies have shown that combinations of ACT or ACTH or TCH have shown improved benefit in the adjuvant treatment of women with HER2-positive breast cancer, and are now the standard of care [[3](#pone.0200996.ref003)–[6](#pone.0200996.ref006)].

Recent data has suggested that germline single nucleotide polymorphisms (SNPs) play a role in both sensitivity and resistance to targeted therapies [[7](#pone.0200996.ref007),[8](#pone.0200996.ref008)]. In fact, we and others have shown that SNPs which occur in the epidermal growth factor receptor (*EGFR*), *ERBB2*, *ERBB3* or *ERBB4* genes (HER-family genes) can be associated with either a worse relapse free survival (RFS) or worse overall survival (OS) rates in women with HER2-positive breast cancer who received adjuvant trastuzumab as part of their treatment regimen [[9](#pone.0200996.ref009),[10](#pone.0200996.ref010)]. Mutations in HER-family genes have been shown to activate the PI3K/AKT signaling pathway and both germline SNPs and somatic mutations may act as biomarkers of sensitivity and resistance in both gastric and breast cancers [[11](#pone.0200996.ref011)].

Cancer cells with impairment in DNA repair mechanisms, such as homologous recombination deficiency (HRD), are sensitive to platinum based drugs, which directly damage a cell’s DNA. Mutations in the tumour suppressor genes *BRCA1/2* are the most common cause of HRD. However, any gene mutations which cause HRD can potentially result in a phenotype like that of *BRCA1/2* mutated cancer. This phenomenon is called ‘BRCAness’ and is also characterized by HRD [[12](#pone.0200996.ref012)].

Our current study aims to explore the effect on survival and functional impact of both HER-family and HRD related SNPs on response to TCH based treatment, versus patients who received a non-TCH based treatment.

## Materials and methods

### Patients

A total of 157 patients with operable primary BC were used in this study, which included 78 patient samples from the [NCT01485926](https://clinicaltrials.gov/ct2/show/NCT01485926) phase II neo-adjuvant study of TCH/TCHL in women with early stage HER2-positive breast cancer. The remaining 79 patients came from the CTI-09-07 translational study, which enabled the collection of FFPE samples and the relevant clinicopathological data and treatment history. We selected an additional 32 samples from the CTI-09-07 translational study on which to perform high depth sequencing, as this discovery cohort had tumour blocks with sufficient tissue to perform NGS analysis. These studies were approved by the research ethics committees of all hospitals, were run by the All-Ireland Clinical Oncology Research Group (ICORG now Cancer Trials Ireland) and included women who are confirmed as clinically HER2-positive by a 3+ HER2 immunohistochemistry score and or/ a FISH ratio of >2. Detailed clinical information is available in [Table 1](#pone.0200996.t001). A Fisher exact test was used to compare the clinical parameters between TCH and non-TCH to show that there was no imbalance between the two cohorts. The p-values have been included in [Table 1](#pone.0200996.t001).

### Table 1. Summary of patient characteristics (n = 157).

| Feature | Sample Number | P-value |
| --- | --- | --- |
| TCH Samples (n = 102) |   |   |
| Mean Age ± SD (years) | 51 ± 11 |   |
| Grade |   |  |
| I | 0 |   |
| II | 10 |   |
| III | 15 |   |
| Unknown | 77 | 0.43 |
| LN Status |   |  |
| Positive | 15 |   |
| Negative | 14 |   |
| Unknown | 73 | 0.86 |
| ER Status |   |  |
| Positive | 62 |   |
| Negative | 39 |   |
| Unknown | 1 | 0.73 |
| PR Status |   |  |
| Positive | 41 |   |
| Negative | 54 |   |
| Unknown | 7 | 0.07 |
| Mean OS ± SD (months) | 66 ± 59 |   |
| Mean PFS ± SD (months) | 52 ± 33 |   |
| Non TCH Samples (n = 55) |   |  |
| Mean Age ± SD (years) | 53 ± 12 |   |
| Grade |   |  |
| I | 2 |   |
| II | 15 |   |
| III | 37 |   |
| Unknown | 1 | 0.43 |
| LN Status |   |  |
| Positive | 30 |   |
| Negative | 24 |   |
| Unknown | 1 | 0.86 |
| ER Status |   |  |
| Positive | 31 |   |
| Negative | 23 |   |
| Unknown | 1 | 0.73 |
| PR Status |   |  |
| Positive | 9 |   |
| Negative | 27 |   |
| Unknown | 19 | 0.07 |
| Mean OS ± SD (months) | 82 ± 35 |   |
| Mean PFS ± SD (months) | 78 ± 37 |   |

Table 1 Caption: OS = Overall survival; PFS = Progression Free Survival; SD = Standard Deviation; LN = Lymph Node; ER = Estrogen Receptor; PR = Progesterone Receptor.

The patient data was analyzed anonymously, the study was approved by the ethics committee of all the individual hospitals in which the patients were treated, and approved the respective studies in accordance with the Declaration of Helsinki. CTI 09–07 is a translational study and enrolment on TCHL and CTI-0907 was possible. Central REC review for TCHL: University College Cork Clinical Research Ethics Committee, and Local REC reviews for 09–07: University College Cork Clinical Research Ethics Committee, Beaumont Hospital Ethics Committee, University Hospital Waterford Research Ethics Committee.

### High depth sequencing

In our initial SNP screen, we performed high depth sequencing on our discovery cohort, which contained 32 tumour formalin-fixed, paraffin-embedded (FFPE) samples, from patients with HER2-positive BC. Haematoxylin and eosin sections cut from the patient’s FFPE surgical blocks were analysed by a pathologist for tumour content and those with >50% tumour cellularity had a further 7*10μM sections cut, from which DNA was extracted using the Qiagen DDNA FFPE kit as outlined in a previous study [[10](#pone.0200996.ref010)].

### Sequencing analysis

The reads from our discovery cohort were trimmed using Trimmomatic [[13](#pone.0200996.ref013)] and aligned with BWA mem (version 0.7.5a-r405: [http://bio-bwa.sourceforge.net/](http://bio-bwa.sourceforge.net/)) under default parameters. Duplicate reads were marked by Picard tools ([http://broadinstitute.github.io/picard/](http://broadinstitute.github.io/picard/)) and local realignment and base recalibration were conducted with GATK [[14](#pone.0200996.ref014)] (version v3.2-2-gec30cee, human genome version 19). Pileup files were generated using Samtools [[15](#pone.0200996.ref015)] (version 0.1.19-44428cd), excluding reads with mapping quality <20, and variants were called with Varscan [[16](#pone.0200996.ref016)] (version v2.3.7) at positions with coverage ≥20. Variants were annotated by Variant Effect Predictor [[17](#pone.0200996.ref017)].

### Protein extraction and reverse phase protein array analysis

We extracted protein from 60 women’s FFPE surgical samples and performed RPPA analysis to identify the impact of HER-family and HRD related SNPs on PI3K/AKT and MAPK/ERK signaling. We only selected FFPE samples from surgical tumour resections which had greater than 50% tumour or were enriched by macro dissection to ensure >50% tumour, and samples which had low protein yield were excluded from our analysis. RPPA on the 60 clinical samples was performed as previously described [[18](#pone.0200996.ref018)]. The data was normalized by protein loading using the entire antibody panel. These 60 samples are a subset of the original patient cohort. A breakdown of the clinical parameters of these samples is shown in [Table 2](#pone.0200996.t002). A Fisher exact test was used to compare the clinical parameters between TCH and non-TCH to show that there was no imbalance between the two cohorts and the p-values are shown in [Table 2](#pone.0200996.t002). The protein levels between TCH receiving patients and patients receiving another regiment were then compared using Tukey Honest Significant Difference and an ANOVA model. The resultant p-values are adjusted for multiple testing using the Benjamini-Hochberg method [[19](#pone.0200996.ref019)].

### Table 2. Summary of patient characteristics used in the RPPA analysis (n = 60).

| Feature | Sample Number | P-value |
| --- | --- | --- |
| TCH Samples (n = 27) |   |   |
| Mean Age ± SD (years) | 50 ± 12 |   |
| Grade |   |  |
| I | 0 |   |
| II | 10 |   |
| III | 14 |   |
| Unknown | 3 | 0.38 |
| LN Status |   |  |
| Positive | 14 |   |
| Negative | 13 |   |
| Unknown | 0 | 0.80 |
| ER Status |   |  |
| Positive | 18 |   |
| Negative | 9 |   |
| Unknown | 0 | 1.0 |
| PR Status |   |  |
| Positive | 11 |   |
| Negative | 11 |   |
| Unknown | 5 |   |
| Mean OS ± SD (months) | 65 ± 59 |   |
| Mean PFS ± SD (months) | 59 ± 57 | 0.76 |
| Non TCH Samples (n = 33) |   |  |
| Mean Age ± SD (years) | 53 ± 13 |   |
| Grade |   |  |
| I | 2 |   |
| II | 9 |   |
| III | 22 |   |
| Unknown | 0 | 0.38 |
| LN Status |   |  |
| Positive | 19 |   |
| Negative | 14 |   |
| Unknown | 0 | 0.80 |
| ER Status |   |  |
| Positive | 21 |   |
| Negative | 12 |   |
| Unknown | 0 | 1.0 |
| PR Status |   |  |
| Positive | 9 |   |
| Negative | 13 |   |
| Unknown | 11 | 0.76 |
| Mean OS ± SD (months) | 72 ± 34 |   |
| Mean PFS ± SD (months) | 70 ± 36 |   |
### Agena MassArray

Mass spectrometry-based genotyping (Agena MassARRAY, Sequenom, San Diego, CA) was applied to confirm the allele calls of the 6 SNPs which are listed in [Table 3](#pone.0200996.t003) from the NGS screen of HER-family and HRD related genes. We also tested the 157 HER2-positive BC patient cohort focusing only on the 6 most significant SNPs from the initial 32 sample screen. Reactions where >15% of the resultant mass ran in the mutant site were scored as positive.

### Table 3. Impact of HER-family and HRD SNPs on relapse free and overall survival of HER2-positive BC patients who have received either TCH based regimen or a non-TCH based regimen as part of their therapeutic regimen (n = 157).

| Gene Information |  |  |  | TCH Treated Patients |  | Patients Not Treated With TCH |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Gene | Accession Number | SNP | R/MA Sub | RFS HR(95% CI) | Adjustedp-value | RFS HR(95% CI) | Adjustedp-value |
| ErbB2 | rs1136201 | I655V | A/G | 2.67(1.05–6.78) | 0.05 | 2.75(0.80–9.52) | 0.29 |
| ErbB3 | rs2229046 | I449I | T/C | 4.95(1.91–2.79) | 1.51x10-3 | 1.22(0.26–5.77) | 0.89 |
| rs773123 | S1119C | T/A | 2.67(1.02–7.03) | 0.05 | 0.65(0.14–3.07) | 0.89 |  |
| RNF8 | rs2284922 | T448T | C/T | 1.11(0.24–5.06) | 0.90 | 12.42(2.00–77.19) | 0.01 |
| BARD1 | rs2070096 | T351T | G/T | 3.27(1.16–9.17) | 0.05 | 0.89(0.25–3.17) | 0.89 |
| rs2229571 | R378S | G/C | 2.02(0.66–6.13) | 0.25 | 1.1(0.28–4.26) | 0.89 |  |

Table 3 Caption: TCH = docetaxel, carboplatin, trastuzumab; R/MA = Reference/Minor Allele; RFS = Recurrence Free Survival; HR = Hazard Ratio; CI = Confidence Interval.

### Survival analysis

Survival analysis was only performed using data from trastuzumab treated samples, with RFS as the survival endpoint. Survival curves are based on Kaplan-Meier estimates and the log-rank p-value is shown for difference in survival. The resultant p-values are adjusted for multiple testing using the Benjamini-Hochberg method [[19](#pone.0200996.ref019)]. Cox regression analysis was used to calculate hazard ratios and perform multivariate analysis. The R package ‘survival’ is used to calculate and plot the Kaplan-Meier survival curves [[20](#pone.0200996.ref020)]. All calculations are carried out in the R statistical environment ([http://cran.r-project.org/](http://cran.r-project.org/)).

### Cell line analysis

Exome sequencing data for 70 breast cancer cell lines treated with 90 therapeutic agents [[21](#pone.0200996.ref021)] were downloaded from the gene expression omnibus ([https://www.ncbi.nlm.nih.gov/geo/](https://webmail.rcsi.ie/owa/redir.aspx?SURL=863XSRrg4RwE-6Y-7eSAfGYNMt0CyFvpGP9Tn7tYXu-AwsJ7kM3UCGgAdAB0AHAAcwA6AC8ALwB3AHcAdwAuAG4AYwBiAGkALgBuAGwAbQAuAG4AaQBoAC4AZwBvAHYALwBnAGUAbwAvAA..&URL=https%3a%2f%2fwww.ncbi.nlm.nih.gov%2fgeo%2f)) in SRA format (accession number [GSE48216](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48216)). Samples were processed as per GATK [[22](#pone.0200996.ref022)] best practices ([https://software.broadinstitute.org/gatk/best-practices/](https://webmail.rcsi.ie/owa/redir.aspx?SURL=jth_YA4t9jfA4Brdu-TcdnBqfeLJfOxqIdEzbE5ScMuAwsJ7kM3UCGgAdAB0AHAAcwA6AC8ALwBzAG8AZgB0AHcAYQByAGUALgBiAHIAbwBhAGQAaQBuAHMAdABpAHQAdQB0AGUALgBvAHIAZwAvAGcAYQB0AGsALwBiAGUAcwB0AC0AcAByAGEAYwB0AGkAYwBlAHMALwA.&URL=https%3a%2f%2fsoftware.broadinstitute.org%2fgatk%2fbest-practices%2f)). Briefly, the SRA files were converted to FASTQ files using the SRA Toolkit ([https://www.ncbi.nlm.nih.gov/sra/docs/toolkitsoft/](https://www.ncbi.nlm.nih.gov/sra/docs/toolkitsoft/)), quality control was conducted using FastQC ([http://www.bioinformatics.babraham.ac.uk/projects/fastqc/](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/)), the adapter sequences were trimmed using BBmap ([http://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbmap-guide/](http://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbmap-guide/)) and the FASTQ files were converted to SAM format using BWA [[23](#pone.0200996.ref023)]. Next, the duplicates were marked using Picard Tools ([http://broadinstitute.github.io/picard/](http://broadinstitute.github.io/picard/)) and reordered before being passed on to GATK for base recalibration, using known sites as provided by the Broad Institute best practices guide. The files were then run through GATK Haplotype caller using discovery mode and recommended settings and the files were run through the GATK Variant Recalibrator, using the recommended settings for SNP discovery and known sites as provided by the Broad Institute best practices guide. Finally, the files were annotated using Oncotator [[24](#pone.0200996.ref024)].

For statistical analysis, the locus status of the mutations in each HER2-positive cell line was used to divide the cell lines into three groups (homogenous reference allele, heterogeneous, and homogenous minor allele). The GD_50_ values of the three groups were then compared using Tukey Honest Significant Difference and an ANOVA model. The GD_50_ values and the HER2 status of the cell line were as per the original authors [[21](#pone.0200996.ref021)]. This test was done for each SNP and each treatment of interest (carboplatin and cisplatin). A p-value of < 0.05 was considered significant. All p-values were adjusted using the Benjamini-Hochberg method [[19](#pone.0200996.ref019)].

### Machine learning methods

To perform pattern recognition analysis, a predictive model was built and tested using a Support Vector Machine (SVM). The features used were our 6 SNPs and the time to last follow-up (in months) to predict RFS with (1) or without (0) event and a radial-kernel SVM was built using the e1071 package in R [[25](#pone.0200996.ref025)]. 5-fold cross validation was used to combat the issues that are associated with training a SVM on a dataset of this size when building on the development set, and leave one out cross validation (LOOCV) was used on the test set. This provided a vector of predictions equal in length to the vector of true values for RFS, where the two were compared and the following metrics were reported; accuracy, AUC and Phi score.

The accuracy was chosen to represent the predictive power of the model, the Phi score was utilized to determine how strong of a relationship existed between features and the classes, and the AUC was chosen to assess how efficient the features are at separating the classes, where a value of 0.5 represents the machine just ‘guessing’ the classes. The Phi score is a metric with a value between -1 and 1, where -1 represents an inverse correlation between the features and the classes, 1 represents a strong positive correlation [[26](#pone.0200996.ref026)].

## Results

We focused on 6 SNPs identified in our high depth sequencing study of the exome of 28 genes ([S1 Table](#pone.0200996.s002)) from DNA extracted from the FFPE tumour samples of 32 women with HER2-positive breast cancer due to their correlation with RFS ([Table 4](#pone.0200996.t004)). Included in our analysis were 3 SNPs from *ERBB2* and *ERBB3* genes; the *ERBB2* rs1136201 SNP, the *ERBB3* rs2229046 and rs773123 SNPs, which were found in approximately 15%, less than 5%, and 6% of the 1000 genome UK population, respectively. Our analysis also included 3 SNPs related to HRD; *RNF8* rs2284922 SNP, found in approximately 44% of the 1000 genome UK population and *BARD1*’s rs2070096 and rs2229571 SNPs, which occur in approximately 19% and 45% of the 1000 genome population, respectively. All 6 SNPs are exonic and occur in protein coding domains, where 3 (rs2229046, rs2284922, and rs2070096) are nonsynonymous variations, with rs2229046 being implicated in alternative splicing for *HER3* [[27](#pone.0200996.ref027)]. The frequency of each genotype within our study population can be found in [S2 Table](#pone.0200996.s003).

### Table 4. List of SNPs including Gene ID, accession number, mutant allele frequency and haplotype details which were detected by high depth next generation sequencing and selected for further analysis.

| Gene | Accession Number | GMAF | R/MASub | Haplotype | AA Change | Protein Domain |
| --- | --- | --- | --- | --- | --- | --- |
| HER2 | rs1136201 | 0.1556 | A/G | 37879588 A>G | I655V | Transmembrane |
| HER3 | rs2229046 | 0.0495 | T/C | 56487201 T>C | I449I | Receptor L domain |
| HER3 | rs773123 | 0.0665 | T/A | 56494998 A>T | S1119C | No defined domain |
| RNF8 | rs2284922 | 0.4397 | C/T | 37349033 G>A | T448T | No defined domain |
| BARD1 | rs2070096 | 0.1903 | G/T | 215645545 C>A | T351T | No defined domain |
| BARD1 | rs2229571 | 0.4593 | G/C | 215645464 C>G | R378S | No defined domain |

Table 4 Caption: GMAF = Global Minor Allele Frequency; R/MA = Reference/Minor Allele; AA = Amino Acid.

### Association between HER-family and HRD related polymorphisms with drug response in HER2-positive patients

In our study, we assessed the impact of HER-family and HRD SNPs on the response of patients who received TCH-based treatment versus those who received other standard treatment regimens which did not include TCH.

We found that women who were heterozygous for the *ERBB2* rs1136201 SNP and the *ERBB3* SNPs rs2229046 and rs773123, were significantly more likely to relapse on a TCH based regimen (rs1136201: p = 0.05, rs2229046: p = 1.51x10^-3^, rs773123: p = 0.05) than those women who received a non-TCH based treatment ([Table 3](#pone.0200996.t003), [Fig 1A, 1B and 1C](#pone.0200996.g001)). After multivariate analysis, the difference in the rate of RFS for rs1136201 and rs2229046 remains significant when adjusted for age, estrogen receptor (ER) and progesterone receptor (PR) status (rs1136201: p = 0.01, rs2229046: p = 0.01), however, rs773123 did not remain significant when adjusted for the same parameters (p = 0.29).

### Fig 1. The prognostic role of ERBB2, ERBB3 and BARD1 SNPs in TCH versus non-TCH treated primary tumours (n = 157).

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8380/6071997/59108bdd7ae3/pone.0200996.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6071997_pone.0200996.g001.jpg)

In each plot, the blue and red lines represent the samples with the wild type (reference) and heterozygous alleles respectively. Kaplan Meier estimates of (A) the ERBB2-rs1136201 SNP where RFS is the survival endpoint (HR = 2.67 (1.05–6.78), p = 0.05) in TCH treated patients versus no-significant impact in non-TCH treated patients, (B) the ERBB3 rs2229046 SNP, where RFS is the survival endpoint (HR = 4.95 (1.91–2.79), p = 1.51x10-3) in TCH treated patients versus no significant impact on non-TCH treated patients, (C) the ERBB3 rs773123 SNP, where RFS is the survival endpoint (HR = 2.67 (1.02–7.03), p = 0.05) in TCH treated patients versus no significant impact on non-TCH treated patients, and (D) the BARD1 rs2070096 SNP, where RFS is the survival endpoint (HR = 3.27 (1.16–9.17), p = 0.05) in TCH treated patients versus no significant impact in non-TCH treated patients. p-values <0.05 indicates a significant p-value after multiple testing correction.

We also found that patients who were heterozygous for the *BARD1* rs2070096 SNP were significantly more likely to relapse on a TCH-based treatment than those patients (p = 0.05) who received a non-TCH based treatment (p = 0.89) ([Fig 1D](#pone.0200996.g001)), which maintained its significance in RFS when adjusted for ER and PR status and age (p = 1.69x10^-4^). Conversely, we found that women who had the SNP for the *RNF8*, rs2284922, were significantly more likely to have a higher chance of relapse on non-TCH based treatment (p = 0.01), than those women who received TCH based treatment (p = 0.90) ([Table 1](#pone.0200996.t001), [Fig 2](#pone.0200996.g002)). This maintains significance in RFS when adjusted for ER and PR status and age (p = 0.02).

### Fig 2. The prognostic role of RNF8-T448T SNP in TCH versus non-TCH treated primary tumours (n = 157).

![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8380/6071997/84667b9f8bc2/pone.0200996.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6071997_pone.0200996.g002.jpg)

The blue, red and green lines represent the samples with the wild type (reference), heterozygous and minor alleles respectively. Kaplan Meier estimates where RFS is the survival endpoint (HR = 12.42 (2.00–77.19) p = 0.01) in non-TCH treated patients versus no-significant impact in TCH treated patients.

### Cross-validation

Using machine learning, we demonstrate an increase in metric scores for support vector machine (SVM) model predicting RFS events in patients who underwent TCH based treatment versus those that received non-TCH based treatment ([Table 5](#pone.0200996.t005)). A rise in these metrics reflects the increased ability of this model to predict RFS events in TCH vs non-TCH treated patients. Specifically, the Phi score shows a far stronger correlation between our SNPs and RFS in the TCH treated patients vs non-TCH treated patients.

### Table 5. Metrics from the Support Vector Machine trained on the patient cohort using cross validation.

|   | Non-TCH | TCH |
| --- | --- | --- |
| Accuracy | 0.7962 | 0.8400 |
| Phi | -0.0700 | 0.3600 |
| AUC | 0.4886 | 0.5789 |

Table 5 Caption: TCH = docetaxel, carboplatin, trastuzumab; SVM = Support Vector Machine; AUC = Area Under the receiver-operator Curve

### ERBB3 rs2229046 and rs773123 alter the signaling properties of human HER2-positive breast cancer cells

Our results indicated that patients who had the *ERBB3* rs2229046 SNP had a trend towards lower P27 expression (p-value = 0.07) but had significantly higher expression of SRC kinase (p-value = 0.04). These patients also had a trend towards higher phosphorylation of EGFR (Y1068) and Shc (Y317) (p-value = both 0.06). These results indicate that the *ERBB3* rs2229046 SNP may be associated with tumours which preferentially signal through the EGFR/MAPK/SRC kinase pathway ([Fig 3](#pone.0200996.g003)).

### Fig 3.

![Fig 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8380/6071997/f2c31e9508f8/pone.0200996.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6071997_pone.0200996.g003.jpg)

Reverse Phase Protein array analysis correlating differential expression and phosphorylation of proteins involved in either the A) AKT or B) MAPK pathway versus the presence or absence of the minor allele of the ERBB3 rs2229046 SNP. p-values <0.05 included on the graph demonstrate significantly differential protein expression between the presence of the reference allele or the presence of the minor allele and are corrected for multiple testing.

We also found that patients who had the *ERBB3* rs773123 SNP were significantly more likely to have lower expression of P27 ([Fig 4](#pone.0200996.g004)) (p-value 8.0x10^-3^), whilst also having significantly lower phosphorylation of c-MET (Y1234/1235) (p-value = 0.02) but significantly higher phosphorylation of GSK-3β (S9) (p-value = 0.03) ([Fig 4](#pone.0200996.g004)). These results indicate that patients with the *ERBB3* rs773123 SNP may preferentially signal through the c-MET/AKT/GSK-3β pathway. However, reflecting the small size of this dataset (n = 60), it would be important to validate these findings in a larger dataset.

### Fig 4.

![Fig 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8380/6071997/6589428127c4/pone.0200996.g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6071997_pone.0200996.g004.jpg)

Reverse Phase Protein array analysis correlating differential expression and phosphorylation of proteins involved in either the A) AKT or B) MAPK pathway versus the presence or absence of the minor allele of the ERBB3 rs773123 SNP. p-values <0.05 included on the graph demonstrate significantly differential protein expression between the presence of the reference allele or the presence of the minor allele and are corrected for multiple testing.

### The minor allele of BARD1 rs2229571 is associated with higher sensitivity to platinum based drugs in in HER2-positive breast cancer cell lines

Using available sequencing data from the Genomics of Drug Sensitivity in Cancer (GDSC) database [[21](#pone.0200996.ref021)] we determined the specific allele for each of the HER-family and HRD related SNPs in a panel of 11 HER2-positive cell lines ([Table 6](#pone.0200996.t006)). We then correlated the sensitivity of the different cell lines based on the expression of the allele for each SNP against sensitivity to the platinum based drugs carboplatin and cisplatin, as well as docetaxel. Platinum and docetaxel based drugs form part of the TCH regimen used to treat women with HER2-positive breast cancer. Our results show that cell lines which had the minor allele of *BARD1* rs2229571 were significantly more likely to be sensitive to a platinum-based drug relative to those cell lines that had either the reference allele or were heterozygous for the SNP (carboplatin: p = 0.04, cisplatin: p = 0.02) ([Fig 5](#pone.0200996.g005)). However, this result did not correlate with a difference in either RFS or OS in either TCH or non TCH treated patients. Furthermore, the same phenomena were not seen when we looked at breast cancer cell lines across multiple subtypes, demonstrating that this is unique to the HER2 enriched group ([S1 File](#pone.0200996.s001)).

### Table 6. The GD50 levels and SNP calls for the 11 HER2 Cell lines used as secondary validation in this study.

| Cell Line | CarboplatinGD50 | CisplatinGD50 | Docetaxel GD50 | rs1136201ERBB2I655V | rs2229046ERBB3I449I | rs773123ERBB3S1119C | rs2284922RNF8T448T | rs2070096BARD1T351T | rs2229571BARD1R378S |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| AU565 | 4.95 | 5.72 | 8.30 | Ref | Ref | Ref | Het | Ref | Het |
| BT474 | 3.97 | 4.42 | 8.20 | Ref | Ref | Ref | Ref | Ref | Minor |
| HCC1419 | 4.02 | 4.91 | 7.80 | Ref | Ref | Ref | Ref | Ref | Ref |
| HCC1569 | 3.96 | 5.11 | 8.12 | Ref | Ref | Ref | Het | Ref | Ref |
| HCC1954 | 4.42 | 5.29 | 8.78 | Ref | Ref | Ref | Minor | Ref | Ref |
| HCC202 | 4.44 | 6.06 | 8.63 | Ref | Het | Het | Ref | Het | Het |
| MDAMB361 | 4.34 | 5.06 | 7.92 | Ref | Ref | Ref | Ref | Ref | Het |
| SKBR3 | 5.00 | 4.77 | 8.00 | Ref | Ref | Ref | Het | Ref | Het |
| UACC812 | 4.76 | 5.48 | 8.39 | Ref | Ref | Ref | Het | Het | Het |
| UACC893 | 3.04 | 3.76 | 7.77 | Ref | Het | Het | Ref | Ref | Minor |
| ZR7530 | 4.55 | 5.58 | 8.40 | Ref | Ref | Ref | Het | Ref | Het |
### Fig 5. Analysis of the differential sensitivity of HER2-positive breast cancer cell lines to the platinum drugs cisplatin and carboplatin.

![Fig 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8380/6071997/18a536dc9257/pone.0200996.g005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6071997_pone.0200996.g005.jpg)

We correlated the presence or absence of the minor allele of HER-family or BARD1 versus the GD50 to either drug. GD50 values were taken from the GDSC cell line analysis database [21]. p-value of <0.05 indicates a significant p-value after multiple testing correction.

## Discussion

Despite the improvement in response rates of women with HER2-positive breast cancer to trastuzumab based treatments, a significant population of women have innate resistance to the trastuzumab based treatments such as the standard of care, TCH based therapy [[2](#pone.0200996.ref002)–[5](#pone.0200996.ref005)]. This innate resistance results in a clinically worse RFS for the patient. Currently there is no curative treatment for metastatic HER2-positive breast cancer; therefore, biomarkers which can identify at diagnosis patients unlikely to benefit from TCH based treatment are critical to allow clinicians to direct these patients to alternate therapeutic options.

Previously we and others demonstrated that the minor allele of the ERBB–I655V SNP was associated with a worse RFS in women with HER2-positive breast cancer [[9](#pone.0200996.ref009),[10](#pone.0200996.ref010)]. A limitation of these studies was that a patient’s specific therapeutic regimens were not considered when the analysis was done. As trastuzumab is standardly prescribed in combination with either docetaxel or platinum based drugs (TCH) [[3](#pone.0200996.ref003)–[6](#pone.0200996.ref006)], we wanted to correlate SNPs as predictive markers of response to TCH based treatment versus other therapeutic regimens. In our study we not only looked at SNPs in the ERBB-family of genes but as the TCH regimen includes a platinum drug we also assessed the importance of SNPs in genes associated with HRD and correlated these differences with a patient’s response.

We have shown that patients who have the minor allele of the *ERBB3* SNPs, (rs2229046 and rs77123) and who received TCH based treatment are significantly more likely to have a worse RFS than patients who received alternate therapies. To support our hypothesis that this is a TCH dependent effect, we previously demonstrated that patients who had the minor allele of the *ERBB3* SNP rs2229046 and who received trastuzumab as part of their therapeutic regimen did not have a significantly worse RFS [[10](#pone.0200996.ref010)].

The *ERBB3* SNPs identified in our study represent both a synonymous (rs2229046) and non-synonymous (rs77123) SNP. SNPs which are synonymous variants were until recently believed to be silent, resulting in little to no impact on the ensuing protein. However, recent studies have shown that synonymous SNPs can play an important role in the functionality of the cancer cell and how a patient responds to targeted therapies [[8](#pone.0200996.ref008),[10](#pone.0200996.ref010),[28](#pone.0200996.ref028),[29](#pone.0200996.ref029)].

Previous studies have shown that despite the *ERBB3* SNPs rs77123 and rs2229046 SNPs not having a described role in cancer susceptibility [[30](#pone.0200996.ref030),[31](#pone.0200996.ref031)], both have been associated with alternative splicing [[27](#pone.0200996.ref027)]. To date though, no-one has analysed the impact of these SNPs on cellular signalling. Our results show that patients who have the minor allele of the *ERBB3* SNP rs2229046 have significantly elevated expression of Src kinase, and a nearly significant increase in EGFR Y1068 phosphorylation, whilst patients with the *ERBB3* SNP (rs77123) have a significantly higher level of GSK-3β (s9) phosphorylation relative to those with the reference allele. HER2-positive breast cancer is generally classified as being associated with activation of the PI3K/AKT signalling pathway [[2](#pone.0200996.ref002),[32](#pone.0200996.ref032),[33](#pone.0200996.ref033)]. Trastuzumab, the monoclonal antibody which targets HER2 is designed to block this PI3K/AKT activation by inhibiting downstream signalling of HER2 activation [[2](#pone.0200996.ref002)]. The elevated expression and phosphorylation of non-classically PI3K/AKT signalling pathways in cells that have minor allele of the *ERBB3* rs2229046 and rs77123 SNPs could potentially indicate why these patients do not respond as well to TCH based regimens relative to the patients who receive non-TCH based treatment.

Our analysis also identified that patients with the minor allele of the *BARD1* SNP rs2070096 were significantly more likely to have a worse RFS than those patients who received a non-TCH based treatment. Our hypothesis that impediment of the HRD pathway would result in altered sensitivity to platinum based drugs given as part of the TCH backbone is supported by our result that patients who were homozygous for the minor allele of *RNF8* rs2284992 had a worse RFS when they were given non-TCH based treatment, compared to those receiving TCH based treatment. The confounding effects of differing HRD related SNPs in relation to response to treatment would require further functional interrogation to elucidate why this effect occurs.

The role of RNF8 in DNA damage repair has been extensively studied, however there are no studies which associate *RNF8* rs2284922 as playing a role in cancer susceptibility or drug response [[34](#pone.0200996.ref034)]. However whilst both *BARD1* SNPs (rs2070096, rs2229571) in this study have been associated with nephroblastoma and neuroblastoma [[35](#pone.0200996.ref035)–[38](#pone.0200996.ref038)], no studies have associated them with response to platinum based drugs as part of a therapeutic regimen.

To further examine the impact of HRD related SNPs on platinum response we analysed the SNP status of a panel of 11 HER2-positive cell lines and correlated this against the cell line GD_50_ to platinum (cisplatin and carboplatin) and docetaxel based drugs. We demonstrated that those cell lines with the minor allele of the *BARD1* SNP rs2229571 had greater sensitivity to both carboplatin and cisplatin. Interestingly, the *BARD1* SNP rs2229571 had no significant effect on a patient’s response to TCH or non-TCH treatment. These results may indicate that whilst SNPs in HRD related genes do impact on a drug’s sensitivity in vitro, that their impact on a patient’s response clinically may be overcome using multiple therapies, which can have confounding effects on how a patient responds.

Our study does have limitations that must be taken into consideration, as these may lead to residual confounding biases. These issues include a small sample size and, in some cases, incomplete clinicopathological information. Future studies building from our work should test these variables to see if there are any residual effects on the results we have observed. One of the ways we addressed the small sample size in our study, was to perform rigorous statistical analysis to ensure that our results remain valid after correcting for multiple testing. We also used machine learning as another statistical tool to support that our patient response results are robust. Finally using the publicly available GDSC database we could determine the SNP status of cell line models which could be used to demonstrate the validity of our *BARD1* SNP results from patients in cell line models. We demonstrated that patients with the minor allele of the *ERBB3* and *BARD1* SNPs were more likely to have a worse RFS than those who received non-TCH based treatment. For the first time we also demonstrated using NGS analyses of cell line data that SNPs which occur in and *BARD1* correlate with carboplatin or cisplatin sensitivity in vitro. Finally, we have shown that patients with the minor allele of the *ERBB3* rs77123 and rs2229046 SNPs have elevated expression and phosphorylation of proteins in pathways not associated with PI3K/AKT signalling, likely indicating why these patients do worse to TCH based treatment.

This study has demonstrated the potential of SNPs, both synonymous and non-synonymous to be important markers in tumour response to therapy. These hereditary changes should be taken into account when clinicians decide on a suitable course of treatment for women who will receive adjuvant treatment of their HER2-positive breast cancer.

## Supporting information

## Data Availability

All relevant data are uploaded at Sequence Read Archive under SRA number SRP150438. ([https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP150438&go=go](https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP150438&go=go)).

## Funding Statement

The TCHL clinical study was sponsored by Cancer Trials Ireland (formerly known as the Irish Clinical Oncology Research Group [ICORG]), which received funding from GlaxoSmithKline. This work was supported by the Irish Cancer Society Collaborative Cancer Research Centre under BREAST-PREDICT grant CCRC13GAL ([www.breastpredict.com](http://www.breastpredict.com)), the Health Research Board (HRA/POR2012/054), the North East Cancer Research and Education Trust, and the Science Foundation Ireland-funded Molecular Therapeutics for Cancer Ireland (08-SRC-B1410). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All relevant data are uploaded at Sequence Read Archive under SRA number SRP150438. ([https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP150438&go=go](https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP150438&go=go)).

### Supplementary Materials

### Data Availability Statement

All relevant data are uploaded at Sequence Read Archive under SRA number SRP150438. ([https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP150438&go=go](https://trace.ncbi.nlm.nih.gov/Traces/study/?acc=SRP150438&go=go)).

## References

1. Slamon DJ, Godolphin W, Jones LA, Holt JA, Wong SG, Keith DE, et al. Studies of the HER-2 / neu Proto-Oncogene in Human Breast and Ovarian Cancer. Science (80-). 1989;244: 707–712. 10.1126/science.2470152  [DOI](https://doi.org/10.1126/science.2470152) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2470152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Studies%20of%20the%20HER-2%20/%20neu%20Proto-Oncogene%20in%20Human%20Breast%20and%20Ovarian%20Cancer&author=DJ%20Slamon&author=W%20Godolphin&author=LA%20Jones&author=JA%20Holt&author=SG%20Wong&volume=244&publication_year=1989&pages=707-712&pmid=2470152&doi=10.1126/science.2470152&)

2. Elster N, Collins DM, Toomey S, Crown J, Eustace AJ, Hennessy BT. HER2-family signalling mechanisms, clinical implications and targeting in breast cancer. Breast Cancer Research and Treatment. 2015. pp. 5–15. 10.1007/s10549-014-3250-x  [DOI](https://doi.org/10.1007/s10549-014-3250-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25542271/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Research%20and%20Treatment&title=HER2-family%20signalling%20mechanisms,%20clinical%20implications%20and%20targeting%20in%20breast%20cancer&author=N%20Elster&author=DM%20Collins&author=S%20Toomey&author=J%20Crown&author=AJ%20Eustace&publication_year=2015&pages=5-15&pmid=25542271&doi=10.1007/s10549-014-3250-x&)

3. Slamon D, Eiermann W, Robert N, Pienkowski T, Martin M, Press M, et al. Adjuvant trastuzumab in HER2-positive breast cancer. N Engl J Med. 2011;365: 1273–1283. 10.1056/NEJMoa0910383  [DOI](https://doi.org/10.1056/NEJMoa0910383) | [PMC free article](/articles/PMC3268553/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21991949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Adjuvant%20trastuzumab%20in%20HER2-positive%20breast%20cancer&author=D%20Slamon&author=W%20Eiermann&author=N%20Robert&author=T%20Pienkowski&author=M%20Martin&volume=365&publication_year=2011&pages=1273-1283&pmid=21991949&doi=10.1056/NEJMoa0910383&)

4. Piccart-Gebhart M, Holmes E, Baselga J, de Azambuja E, Dueck AC, Viale G, et al. Adjuvant Lapatinib and Trastuzumab for Early Human Epidermal Growth Factor Receptor 2-Positive Breast Cancer: Results From the Randomized Phase III Adjuvant Lapatinib and/or Trastuzumab Treatment Optimization Trial. J Clin Oncol. 2016;34: 1034–42. 10.1200/JCO.2015.62.1797  [DOI](https://doi.org/10.1200/JCO.2015.62.1797) | [PMC free article](/articles/PMC4872016/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26598744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Adjuvant%20Lapatinib%20and%20Trastuzumab%20for%20Early%20Human%20Epidermal%20Growth%20Factor%20Receptor%202-Positive%20Breast%20Cancer:%20Results%20From%20the%20Randomized%20Phase%20III%20Adjuvant%20Lapatinib%20and/or%20Trastuzumab%20Treatment%20Optimization%20Trial&author=M%20Piccart-Gebhart&author=E%20Holmes&author=J%20Baselga&author=E%20de%20Azambuja&author=AC%20Dueck&volume=34&publication_year=2016&pages=1034-42&pmid=26598744&doi=10.1200/JCO.2015.62.1797&)

5. Perez EA, Romond EH, Suman VJ, Jeong JH, Sledge G, Geyer CE, et al. Trastuzumab plus adjuvant chemotherapy for human epidermal growth factor receptor 2-positive breast cancer: planned joint analysis of overall survival from NSABP B-31 and NCCTG N9831. J Clin Oncol. 2014;32: 3744–3752. 10.1200/JCO.2014.55.5730  [DOI](https://doi.org/10.1200/JCO.2014.55.5730) | [PMC free article](/articles/PMC4226805/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25332249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Trastuzumab%20plus%20adjuvant%20chemotherapy%20for%20human%20epidermal%20growth%20factor%20receptor%202-positive%20breast%20cancer:%20planned%20joint%20analysis%20of%20overall%20survival%20from%20NSABP%20B-31%20and%20NCCTG%20N9831&author=EA%20Perez&author=EH%20Romond&author=VJ%20Suman&author=JH%20Jeong&author=G%20Sledge&volume=32&publication_year=2014&pages=3744-3752&pmid=25332249&doi=10.1200/JCO.2014.55.5730&)

6. Piccart-Gebhart M, Procter M, Leyland-Jones B, Goldhirsch A, Untch M, Smith I, et al. Trastuzumab after Adjuvant Chemotherapy in HER2-Positive Breast Cancer. N Engl J Med. 2005;353: 1659–1672. 10.1056/NEJMoa052306  [DOI](https://doi.org/10.1056/NEJMoa052306) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16236737/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Trastuzumab%20after%20Adjuvant%20Chemotherapy%20in%20HER2-Positive%20Breast%20Cancer&author=M%20Piccart-Gebhart&author=M%20Procter&author=B%20Leyland-Jones&author=A%20Goldhirsch&author=M%20Untch&volume=353&publication_year=2005&pages=1659-1672&pmid=16236737&doi=10.1056/NEJMoa052306&)

7. Alaoui-Jamali MA, Morand GB, da Silva SD. ErbB polymorphisms: Insights and implications for response to targeted cancer therapeutics. Frontiers in Genetics. 2015. p. 17 10.3389/fgene.2015.00017  [DOI](https://doi.org/10.3389/fgene.2015.00017) | [PMC free article](/articles/PMC4316710/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25699077/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&title=ErbB%20polymorphisms:%20Insights%20and%20implications%20for%20response%20to%20targeted%20cancer%20therapeutics&author=MA%20Alaoui-Jamali&author=GB%20Morand&author=SD%20da%20Silva&publication_year=2015&pages=17&pmid=25699077&doi=10.3389/fgene.2015.00017&)

8. Sauna ZE, Kimchi-Sarfaty C. Understanding the contribution of synonymous mutations to human disease. Nat Rev Genet. 2011;12: 683–91. 10.1038/nrg3051  [DOI](https://doi.org/10.1038/nrg3051) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21878961/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Understanding%20the%20contribution%20of%20synonymous%20mutations%20to%20human%20disease&author=ZE%20Sauna&author=C%20Kimchi-Sarfaty&volume=12&publication_year=2011&pages=683-91&pmid=21878961&doi=10.1038/nrg3051&)

9. Han X, Diao L, Xu Y, Xue W, Ouyang T, Li J, et al. Association between the HER2 Ile655Val polymorphism and response to trastuzumab in women with operable primary breast cancer. Ann Oncol Off J Eur Soc Med Oncol ESMO. 2014;25 10.1093/annonc/mdu111  [DOI](https://doi.org/10.1093/annonc/mdu111) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24608202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol%20Off%20J%20Eur%20Soc%20Med%20Oncol%20ESMO&title=Association%20between%20the%20HER2%20Ile655Val%20polymorphism%20and%20response%20to%20trastuzumab%20in%20women%20with%20operable%20primary%20breast%20cancer&author=X%20Han&author=L%20Diao&author=Y%20Xu&author=W%20Xue&author=T%20Ouyang&volume=25&publication_year=2014&pmid=24608202&doi=10.1093/annonc/mdu111&)

10. Toomey S, Madden SF, Furney SJ, Fan Y, McCormack M, Stapleton C, et al. The impact of ERBB-family germline single nucleotide polymorphisms on survival response to adjuvant trastuzumab treatment in HER2-positive breast cancer. Oncotarget. 2016;7: 75518–75525. 10.18632/oncotarget.12782  [DOI](https://doi.org/10.18632/oncotarget.12782) | [PMC free article](/articles/PMC5342757/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27776352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=The%20impact%20of%20ERBB-family%20germline%20single%20nucleotide%20polymorphisms%20on%20survival%20response%20to%20adjuvant%20trastuzumab%20treatment%20in%20HER2-positive%20breast%20cancer&author=S%20Toomey&author=SF%20Madden&author=SJ%20Furney&author=Y%20Fan&author=M%20McCormack&volume=7&publication_year=2016&pages=75518-75525&pmid=27776352&doi=10.18632/oncotarget.12782&)

11. Jaiswal BS, Kljavin NM, Stawiski EW, Chan E, Parikh C, Durinck S, et al. Oncogenic ERBB3 Mutations in Human Cancers. Cancer Cell. 2013;23: 603–617. 10.1016/j.ccr.2013.04.012  [DOI](https://doi.org/10.1016/j.ccr.2013.04.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23680147/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&title=Oncogenic%20ERBB3%20Mutations%20in%20Human%20Cancers&author=BS%20Jaiswal&author=NM%20Kljavin&author=EW%20Stawiski&author=E%20Chan&author=C%20Parikh&volume=23&publication_year=2013&pages=603-617&pmid=23680147&doi=10.1016/j.ccr.2013.04.012&)

12. Rigakos G, Razis E. BRCAness: finding the Achilles heel in ovarian cancer. Oncologist. AlphaMed Press; 2012;17: 956–62. 10.1634/theoncologist.2012-0028  [DOI](https://doi.org/10.1634/theoncologist.2012-0028) | [PMC free article](/articles/PMC3399652/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22673632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=BRCAness:%20finding%20the%20Achilles%20heel%20in%20ovarian%20cancer&author=G%20Rigakos&author=E%20Razis&volume=17&publication_year=2012&pages=956-62&pmid=22673632&doi=10.1634/theoncologist.2012-0028&)

13. Bolger AM, Lohse M, Usadel B. Trimmomatic: A flexible trimmer for Illumina sequence data. Bioinformatics. 2014;30: 2114–2120. 10.1093/bioinformatics/btu170  [DOI](https://doi.org/10.1093/bioinformatics/btu170) | [PMC free article](/articles/PMC4103590/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24695404/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Trimmomatic:%20A%20flexible%20trimmer%20for%20Illumina%20sequence%20data&author=AM%20Bolger&author=M%20Lohse&author=B%20Usadel&volume=30&publication_year=2014&pages=2114-2120&pmid=24695404&doi=10.1093/bioinformatics/btu170&)

14. DePristo MA, Banks E, Poplin R, Garimella K V, Maguire JR, Hartl C, et al. A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nat Genet. Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.; 2011;43: 491–498. 10.1038/ng.806  [DOI](https://doi.org/10.1038/ng.806) | [PMC free article](/articles/PMC3083463/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21478889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&author=MA%20DePristo&author=E%20Banks&author=R%20Poplin&author=K%20V%20Garimella&author=JR%20Maguire&volume=43&publication_year=2011&pages=491-498&pmid=21478889&doi=10.1038/ng.806&)

15. Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics. 2009;25: 2078–2079. 10.1093/bioinformatics/btp352  [DOI](https://doi.org/10.1093/bioinformatics/btp352) | [PMC free article](/articles/PMC2723002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19505943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=The%20Sequence%20Alignment/Map%20format%20and%20SAMtools&author=H%20Li&author=B%20Handsaker&author=A%20Wysoker&author=T%20Fennell&author=J%20Ruan&volume=25&publication_year=2009&pages=2078-2079&pmid=19505943&doi=10.1093/bioinformatics/btp352&)

16. Koboldt DC, Zhang Q, Larson DE, Shen D, McLellan MD, Lin L, et al. VarScan 2: Somatic mutation and copy number alteration discovery in cancer by exome sequencing. Genome Res. 2012;22: 568–576. 10.1101/gr.129684.111  [DOI](https://doi.org/10.1101/gr.129684.111) | [PMC free article](/articles/PMC3290792/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22300766/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=VarScan%202:%20Somatic%20mutation%20and%20copy%20number%20alteration%20discovery%20in%20cancer%20by%20exome%20sequencing&author=DC%20Koboldt&author=Q%20Zhang&author=DE%20Larson&author=D%20Shen&author=MD%20McLellan&volume=22&publication_year=2012&pages=568-576&pmid=22300766&doi=10.1101/gr.129684.111&)

17. McLaren W, Pritchard B, Rios D, Chen Y, Flicek P, Cunningham F. Deriving the consequences of genomic variants with the Ensembl API and SNP Effect Predictor. Bioinformatics. 2010;26: 2069–2070. 10.1093/bioinformatics/btq330  [DOI](https://doi.org/10.1093/bioinformatics/btq330) | [PMC free article](/articles/PMC2916720/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20562413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Deriving%20the%20consequences%20of%20genomic%20variants%20with%20the%20Ensembl%20API%20and%20SNP%20Effect%20Predictor&author=W%20McLaren&author=B%20Pritchard&author=D%20Rios&author=Y%20Chen&author=P%20Flicek&volume=26&publication_year=2010&pages=2069-2070&pmid=20562413&doi=10.1093/bioinformatics/btq330&)

18. Holmes FA, Espina V, Liotta LA, Nagarwala YM, Danso M, McIntyre KJ, et al. Pathologic complete response after preoperative anti-HER2 therapy correlates with alterations in PTEN, FOXO, phosphorylated Stat5, and autophagy protein signaling. BMC Res Notes. 2013;6: 507 10.1186/1756-0500-6-507  [DOI](https://doi.org/10.1186/1756-0500-6-507) | [PMC free article](/articles/PMC3915616/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24304724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Res%20Notes&title=Pathologic%20complete%20response%20after%20preoperative%20anti-HER2%20therapy%20correlates%20with%20alterations%20in%20PTEN,%20FOXO,%20phosphorylated%20Stat5,%20and%20autophagy%20protein%20signaling&author=FA%20Holmes&author=V%20Espina&author=LA%20Liotta&author=YM%20Nagarwala&author=M%20Danso&volume=6&publication_year=2013&pages=507&pmid=24304724&doi=10.1186/1756-0500-6-507&)

19. Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J R Stat Soc. WileyRoyal Statistical Society; 1995;57: 289–300. 10.2307/2346101  [DOI](https://doi.org/10.2307/2346101) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20R%20Stat%20Soc&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&publication_year=1995&pages=289-300&doi=10.2307/2346101&)

20. Therneau T, Grambsch P. Modeling Survival Data: Extending the Cox Model. Technometrics. Springer; 2002;44: 85–86. 10.1198/tech.2002.s656  [DOI](https://doi.org/10.1198/tech.2002.s656) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Technometrics&title=Modeling%20Survival%20Data:%20Extending%20the%20Cox%20Model&author=T%20Therneau&author=P%20Grambsch&volume=44&publication_year=2002&pages=85-86&doi=10.1198/tech.2002.s656&)

21. Daemen A, Griffith OL, Heiser LM, Wang NJ, Enache OM, Sanborn Z, et al. Modeling precision treatment of breast cancer. Genome Biol. BioMed Central; 2013;14: R110 10.1186/gb-2013-14-10-r110  [DOI](https://doi.org/10.1186/gb-2013-14-10-r110) | [PMC free article](/articles/PMC3937590/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24176112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Modeling%20precision%20treatment%20of%20breast%20cancer&author=A%20Daemen&author=OL%20Griffith&author=LM%20Heiser&author=NJ%20Wang&author=OM%20Enache&volume=14&publication_year=2013&pages=R110&pmid=24176112&doi=10.1186/gb-2013-14-10-r110&)

22. McKenna A, Hanna M, Banks E, Sivachenko A, Cibulskis K, Kernytsky A, et al. The genome analysis toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. 2010;20: 1297–1303. 10.1101/gr.107524.110  [DOI](https://doi.org/10.1101/gr.107524.110) | [PMC free article](/articles/PMC2928508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20644199/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=The%20genome%20analysis%20toolkit:%20A%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&author=A%20McKenna&author=M%20Hanna&author=E%20Banks&author=A%20Sivachenko&author=K%20Cibulskis&volume=20&publication_year=2010&pages=1297-1303&pmid=20644199&doi=10.1101/gr.107524.110&)

23. Li H, Durbin R. Fast and accurate long-read alignment with Burrows-Wheeler transform. Bioinformatics. 2010;26: 589–595. 10.1093/bioinformatics/btp698  [DOI](https://doi.org/10.1093/bioinformatics/btp698) | [PMC free article](/articles/PMC2828108/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20080505/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Fast%20and%20accurate%20long-read%20alignment%20with%20Burrows-Wheeler%20transform&author=H%20Li&author=R%20Durbin&volume=26&publication_year=2010&pages=589-595&pmid=20080505&doi=10.1093/bioinformatics/btp698&)

24. Ramos AH, Lichtenstein L, Gupta M, Lawrence MS, Pugh TJ, Saksena G, et al. Oncotator: Cancer variant annotation tool. Hum Mutat. 2015;36: E2423–E2429. 10.1002/humu.22771  [DOI](https://doi.org/10.1002/humu.22771) | [PMC free article](/articles/PMC7350419/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25703262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Oncotator:%20Cancer%20variant%20annotation%20tool&author=AH%20Ramos&author=L%20Lichtenstein&author=M%20Gupta&author=MS%20Lawrence&author=TJ%20Pugh&volume=36&publication_year=2015&pages=E2423-E2429&pmid=25703262&doi=10.1002/humu.22771&)

25. Meyer D, Dimitriadou E, Hornik K, Weingessel A, Leisch F. Misc functions of the Department of Statistics (e1071), TU Wien. R Packag version 16–2. 2014; http://cran.r-project.org/package=e1071. doi:citeulike-article-id:9958545  [http://cran.r-project.org/package=e1071](http://cran.r-project.org/package=e1071)

26. Revelle W. psych: Procedures for Psychological, Psychometric, and Personality Research. Evanston, Illinois; 2016.

27. Raghav D, Sharma V. An In Silico Evaluation of Deleterious Nonsynonymous Single Nucleotide Polymorphisms in the ErbB3 Oncogene. Biores Open Access. 2013;2: 206–11. 10.1089/biores.2013.0007  [DOI](https://doi.org/10.1089/biores.2013.0007) | [PMC free article](/articles/PMC3666215/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23741632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biores%20Open%20Access&title=An%20In%20Silico%20Evaluation%20of%20Deleterious%20Nonsynonymous%20Single%20Nucleotide%20Polymorphisms%20in%20the%20ErbB3%20Oncogene&author=D%20Raghav&author=V%20Sharma&volume=2&publication_year=2013&pages=206-11&pmid=23741632&doi=10.1089/biores.2013.0007&)

28. Sauna ZE, Kimchi-Sarfaty C, Ambudkar S V, Gottesman MM. Silent polymorphisms speak: How they affect pharmacogenomics and the treatment of cancer. Cancer Research. 2007. pp. 9609–9612. 10.1158/0008-5472.CAN-07-2377  [DOI](https://doi.org/10.1158/0008-5472.CAN-07-2377) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17942888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Research&title=Silent%20polymorphisms%20speak:%20How%20they%20affect%20pharmacogenomics%20and%20the%20treatment%20of%20cancer&author=ZE%20Sauna&author=C%20Kimchi-Sarfaty&author=S%20V%20Ambudkar&author=MM%20Gottesman&publication_year=2007&pages=9609-9612&pmid=17942888&doi=10.1158/0008-5472.CAN-07-2377&)

29. Findlay JM, Middleton MR, Tomlinson I. A systematic review and meta-analysis of somatic and germline DNA sequence biomarkers of esophageal cancer survival, therapy response and stage. Annals of Oncology. 2015. pp. 624–644. 10.1093/annonc/mdu449  [DOI](https://doi.org/10.1093/annonc/mdu449) | [PMC free article](/articles/PMC4374384/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25214541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Oncology&title=A%20systematic%20review%20and%20meta-analysis%20of%20somatic%20and%20germline%20DNA%20sequence%20biomarkers%20of%20esophageal%20cancer%20survival,%20therapy%20response%20and%20stage&author=JM%20Findlay&author=MR%20Middleton&author=I%20Tomlinson&publication_year=2015&pages=624-644&pmid=25214541&doi=10.1093/annonc/mdu449&)

30. Kanazawa T, Glatt SJ, Tsutsumi A, Kikuyama H, Koh J, Yoneda H, et al. Schizophrenia is not associated with the functional candidate gene ERBB3: Results from a case-control study. Am J Med Genet Part B Neuropsychiatr Genet. 2007;144: 113–116. 10.1002/ajmg.b.30367  [DOI](https://doi.org/10.1002/ajmg.b.30367) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16958035/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20Part%20B%20Neuropsychiatr%20Genet&title=Schizophrenia%20is%20not%20associated%20with%20the%20functional%20candidate%20gene%20ERBB3:%20Results%20from%20a%20case-control%20study&author=T%20Kanazawa&author=SJ%20Glatt&author=A%20Tsutsumi&author=H%20Kikuyama&author=J%20Koh&volume=144&publication_year=2007&pages=113-116&pmid=16958035&doi=10.1002/ajmg.b.30367&)

31. Li D, Feng G, He L. Case-control study of association between the functional candidate gene ERBB3 and schizophrenia in Caucasian population. World J Biol Psychiatry. Taylor & Francis; 2009;10: 595–598. 10.1080/15622970903304442  [DOI](https://doi.org/10.1080/15622970903304442) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19995212/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J%20Biol%20Psychiatry&title=Case-control%20study%20of%20association%20between%20the%20functional%20candidate%20gene%20ERBB3%20and%20schizophrenia%20in%20Caucasian%20population.&author=D%20Li&author=G%20Feng&author=L%20He&volume=10&publication_year=2009&pages=595-598&pmid=19995212&doi=10.1080/15622970903304442&)

32. Zhang S, Huang W-C, Li P, Guo H, Poh S-B, Brady SW, et al. Combating trastuzumab resistance by targeting SRC, a common node downstream of multiple resistance pathways. Nat Med. Nature Research; 2011;17: 461–469. 10.1038/nm.2309  [DOI](https://doi.org/10.1038/nm.2309) | [PMC free article](/articles/PMC3877934/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21399647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=Combating%20trastuzumab%20resistance%20by%20targeting%20SRC,%20a%20common%20node%20downstream%20of%20multiple%20resistance%20pathways&author=S%20Zhang&author=W-C%20Huang&author=P%20Li&author=H%20Guo&author=S-B%20Poh&volume=17&publication_year=2011&pages=461-469&pmid=21399647&doi=10.1038/nm.2309&)

33. Dua R, Zhang J, Nhonthachit P, Penuel E, Petropoulos C, Parry G. EGFR over-expression and activation in high HER2, ER negative breast cancer cell line induces trastuzumab resistance. Breast Cancer Res Treat. Springer US; 2010;122: 685–697. 10.1007/s10549-009-0592-x  [DOI](https://doi.org/10.1007/s10549-009-0592-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19859802/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=EGFR%20over-expression%20and%20activation%20in%20high%20HER2,%20ER%20negative%20breast%20cancer%20cell%20line%20induces%20trastuzumab%20resistance&author=R%20Dua&author=J%20Zhang&author=P%20Nhonthachit&author=E%20Penuel&author=C%20Petropoulos&volume=122&publication_year=2010&pages=685-697&pmid=19859802&doi=10.1007/s10549-009-0592-x&)

34. Nakada S. Opposing roles of RNF8/RNF168 and deubiquitinating enzymes in ubiquitination-dependent DNA double-strand break response signaling and DNA-repair pathway choice. Journal of Radiation Research. Oxford University Press; 2016. pp. i33–i40. 10.1093/jrr/rrw027  [DOI](https://doi.org/10.1093/jrr/rrw027) | [PMC free article](/articles/PMC4990112/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26983989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Radiation%20Research&title=Opposing%20roles%20of%20RNF8/RNF168%20and%20deubiquitinating%20enzymes%20in%20ubiquitination-dependent%20DNA%20double-strand%20break%20response%20signaling%20and%20DNA-repair%20pathway%20choice&author=S%20Nakada&publication_year=2016&pages=i33-i40&pmid=26983989&doi=10.1093/jrr/rrw027&)

35. Capasso M, Devoto M, Hou C, Asgharzadeh S, Glessner JT, Attiyeh EF, et al. Common variations in BARD1 influence susceptibility to high-risk neuroblastoma. Nat Genet. Nature Research; 2009;41: 718–723. 10.1038/ng.374  [DOI](https://doi.org/10.1038/ng.374) | [PMC free article](/articles/PMC2753610/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19412175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Common%20variations%20in%20BARD1%20influence%20susceptibility%20to%20high-risk%20neuroblastoma&author=M%20Capasso&author=M%20Devoto&author=C%20Hou&author=S%20Asgharzadeh&author=JT%20Glessner&volume=41&publication_year=2009&pages=718-723&pmid=19412175&doi=10.1038/ng.374&)

36. Capasso M, Diskin SJ, Totaro F, Longo L, de Mariano M, Russo R, et al. Replication of GWAS-identified neuroblastoma risk loci strengthens the role of BARD1 and affirms the cumulative effect of genetic variations on disease susceptibility. Carcinogenesis. Oxford University Press; 2013;34: 605–611. 10.1093/carcin/bgs380  [DOI](https://doi.org/10.1093/carcin/bgs380) | [PMC free article](/articles/PMC3716226/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23222812/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Replication%20of%20GWAS-identified%20neuroblastoma%20risk%20loci%20strengthens%20the%20role%20of%20BARD1%20and%20affirms%20the%20cumulative%20effect%20of%20genetic%20variations%20on%20disease%20susceptibility&author=M%20Capasso&author=SJ%20Diskin&author=F%20Totaro&author=L%20Longo&author=M%20de%20Mariano&volume=34&publication_year=2013&pages=605-611&pmid=23222812&doi=10.1093/carcin/bgs380&)

37. Fu W, Zhu J, Xiong SW, Jia W, Zhao Z, Zhu SB, et al. BARD1 Gene Polymorphisms Confer Nephroblastoma Susceptibility. EBioMedicine. 2017;16: 101–105. 10.1016/j.ebiom.2017.01.038  [DOI](https://doi.org/10.1016/j.ebiom.2017.01.038) | [PMC free article](/articles/PMC5474516/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28161399/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EBioMedicine&title=BARD1%20Gene%20Polymorphisms%20Confer%20Nephroblastoma%20Susceptibility&author=W%20Fu&author=J%20Zhu&author=SW%20Xiong&author=W%20Jia&author=Z%20Zhao&volume=16&publication_year=2017&pages=101-105&pmid=28161399&doi=10.1016/j.ebiom.2017.01.038&)

38. Nguyen LB, Diskin SJ, Capasso M, Wang K, Diamond MA, Glessner J, et al. Phenotype restricted genome-wide association study using a gene-centric approach identifies three low-risk neuroblastoma susceptibility loci. PLoS Genet. Public Library of Science; 2011;7: e1002026 10.1371/journal.pgen.1002026  [DOI](https://doi.org/10.1371/journal.pgen.1002026) | [PMC free article](/articles/PMC3060064/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21436895/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Phenotype%20restricted%20genome-wide%20association%20study%20using%20a%20gene-centric%20approach%20identifies%20three%20low-risk%20neuroblastoma%20susceptibility%20loci&author=LB%20Nguyen&author=SJ%20Diskin&author=M%20Capasso&author=K%20Wang&author=MA%20Diamond&volume=7&publication_year=2011&pages=e1002026&pmid=21436895&doi=10.1371/journal.pgen.1002026&)
