# No association between IFNL3 (IL28B) genotype and response to peginterferon alfa-2a in HBeAg-positive or -negative chronic hepatitis B

## Metadata
**Authors:** Lai Wei, Heiner Wedemeyer, Yun-Fan Liaw, Henry Lik-Yuen Chan, Teerha Piratvisuth, Patrick Marcellin, Jidong Jia, Deming Tan, Wan-Cheng Chow, Maurizia R Brunetto, Moisés Diago, Selim Gurel, Viacheslav Morozov, Hua He, Yonghong Zhu, Cynthia Wat, Bernadette Surujbally, Alexander J Thompson
**Journal:** PLoS ONE
**Date:** 2018 Jul 17
**DOI:** [10.1371/journal.pone.0199198](https://doi.org/10.1371/journal.pone.0199198)
**PMID:** 30016335
**PMCID:** PMC6049926
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6049926/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6049926/pdf/pone.0199198.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6049926/pdf/pone.0199198.pdf)

## Abstract

**Background & aims:** 
It has yet to be firmly established whether host IFNL3 (IL28B) genotype influences interferon responsiveness in patients with chronic hepatitis B. We investigated associations between single-nucleotide polymorphisms (SNPs) in the IFNL3 region and response to peginterferon alfa-2a in 701 patients enrolled in three large, randomized, international studies.

**Methods:** 
Responses were defined as hepatitis B surface antigen (HBsAg) loss and/or hepatitis B e antigen (HBeAg) seroconversion plus hepatitis B virus (HBV) DNA <2000 IU/ml in HBeAg-positive patients, and HBsAg loss and/or HBV DNA <2000 IU/ml in HBeAg-negative patients (24 weeks after end of treatment). Associations between treatment response and the number of copies of the poor-response allele at three SNPs (rs8099917, rs12980275, rs12979860) were explored with logistic regression models in Asian and white patients.

**Results:** 
The HBeAg-positive and -negative populations comprised 465 (92% Asian, 50% HBV genotype C) and 236 (79% Asian, 41% HBV genotype C) patients, respectively, and had respective response rates of 26% and 47%. The IFNL3 genotype was strongly associated with ethnicity. There was no association between IFNL3 genotype and treatment response in HBeAg-positive or -negative patients. Independent predictors of treatment response were: sex, HBV DNA level and alanine aminotransferase level in HBeAg-positive Asian patients; age in HBeAg-negative Asian patients; and HBV DNA in HBeAg-negative white patients.

**Conclusions:** 
This is the largest analysis to date of associations between IFNL3 genotype and peginterferon response in patients with chronic hepatitis B. The data suggest that IFNL3 polymorphism is not a major determinant of the response to peginterferon alfa-2a in either HBeAg-positive or HBeAg-negative patients.

### Background & aims

It has yet to be firmly established whether host *IFNL3* (*IL28B*) genotype influences interferon responsiveness in patients with chronic hepatitis B. We investigated associations between single-nucleotide polymorphisms (SNPs) in the *IFNL3* region and response to peginterferon alfa-2a in 701 patients enrolled in three large, randomized, international studies.

### Methods

Responses were defined as hepatitis B surface antigen (HBsAg) loss and/or hepatitis B e antigen (HBeAg) seroconversion plus hepatitis B virus (HBV) DNA <2000 IU/ml in HBeAg-positive patients, and HBsAg loss and/or HBV DNA <2000 IU/ml in HBeAg-negative patients (24 weeks after end of treatment). Associations between treatment response and the number of copies of the poor-response allele at three SNPs (rs8099917, rs12980275, rs12979860) were explored with logistic regression models in Asian and white patients.

### Results

The HBeAg-positive and -negative populations comprised 465 (92% Asian, 50% HBV genotype C) and 236 (79% Asian, 41% HBV genotype C) patients, respectively, and had respective response rates of 26% and 47%. The *IFNL3* genotype was strongly associated with ethnicity. There was no association between *IFNL3* genotype and treatment response in HBeAg-positive or -negative patients. Independent predictors of treatment response were: sex, HBV DNA level and alanine aminotransferase level in HBeAg-positive Asian patients; age in HBeAg-negative Asian patients; and HBV DNA in HBeAg-negative white patients.

### Conclusions

This is the largest analysis to date of associations between *IFNL3* genotype and peginterferon response in patients with chronic hepatitis B. The data suggest that *IFNL3* polymorphism is not a major determinant of the response to peginterferon alfa-2a in either HBeAg-positive or HBeAg-negative patients.

## Introduction

Single-nucleotide polymorphisms (SNPs) in the region of the gene for interferon lambda 3 (*IFNL3*, also known as *IL28B*; Gene ID: 282617) on chromosome 19 are strongly associated with clinical outcomes in patients infected with hepatitis C virus (HCV). *IFNL3* genotype has been shown to be the strongest baseline predictor of a sustained virologic response to dual therapy with pegylated interferon (PegIFN) alfa plus ribavirin [[1](#pone.0199198.ref001)–[6](#pone.0199198.ref006)]. Differences in *IFNL3* allele frequencies between different population groups largely explain the well-established ethnic differences in response rates to interferon (IFN)-based therapies [[1](#pone.0199198.ref001),[7](#pone.0199198.ref007),[8](#pone.0199198.ref008)]. *IFNL3* genotype is also associated with the rate of spontaneous clearance of acute HCV infection [[9](#pone.0199198.ref009)–[12](#pone.0199198.ref012)]. The biological mechanism responsible for the *IFNL3* genotype associations remains poorly defined, but responsiveness to exogenous/endogenous IFN is believed to be central, reflected by the observation that patterns of intrahepatic expression of IFN-stimulated genes differ according to *IFNL3* genotype in patients with chronic HCV infection [[13](#pone.0199198.ref013),[14](#pone.0199198.ref014)].

In contrast, it has yet to be firmly established whether, and to what extent, host *IFNL3* genotype might influence the response to IFN-based therapy in patients with chronic hepatitis B (CHB). Several retrospective analyses with discordant results have examined the association between the response to IFN-based treatment and *IFNL3* genotype [[15](#pone.0199198.ref015)–[23](#pone.0199198.ref023)]. Some studies have reported significant associations between SNPs in the *IFNL3* region and treatment outcomes in hepatitis B e antigen (HBeAg)-positive [[18](#pone.0199198.ref018)] and HBeAg-negative [[22](#pone.0199198.ref022)] patients, while other studies have reported no association between these genetic polymorphisms and treatment outcomes [[15](#pone.0199198.ref015)–[17](#pone.0199198.ref017),[19](#pone.0199198.ref019),[21](#pone.0199198.ref021),[23](#pone.0199198.ref023)]. However, between-study comparisons have been hampered by important differences in patient populations, such as HBeAg-antigen status, ethnic background, and hepatitis B virus (HBV) genotype, as well as treatment regimens and outcome definitions. HBV genotype is particularly important. Patients infected with HBV genotypes A or B are more likely to experience HBeAg seroconversion and hepatitis B surface antigen (HBsAg) loss after treatment with IFN [[24](#pone.0199198.ref024)]. The prevalence of HBV genotypes varies geographically, with genotypes A and D being more prevalent in the West, and genotypes B and C being more prevalent in Asia. Thus, the present study was designed with these deficiencies in mind. We examined associations between three markers of host *IFNL3* genotype (rs12979860, rs8099917, and rs12980275) and response to 48 weeks of treatment with PegIFN alfa-2a using consistent outcome definitions in a large cohort of patients who were enrolled in three international randomized studies. The dataset included a large number of Asian and white patients with HBeAg-positive and HBeAg-negative CHB [[25](#pone.0199198.ref025)–[27](#pone.0199198.ref027)].

## Methods

### Subjects

Patients with CHB who were included in this analysis had received 48 weeks of treatment with PegIFN alfa-2a 180 μg/week with or without oral lamivudine in three large, randomized, international clinical studies [[25](#pone.0199198.ref025)–[27](#pone.0199198.ref027)]. The rationale for combining data from patients treated with and without lamivudine was that lamivudine therapy did not influence long-term outcomes with PegIFN alfa-2a in two large trials [[25](#pone.0199198.ref025),[26](#pone.0199198.ref026)]. The complete inclusion and exclusion criteria, study designs, and primary results of these studies have been published elsewhere [[25](#pone.0199198.ref025)–[27](#pone.0199198.ref027)]; thus, only a brief overview will be provided. In the first study, HBeAg-negative patients with an HBV DNA level >100,000 copies/ml and elevated alanine aminotransferase (ALT) levels were randomized to 48 weeks of therapy with one of three treatments: (1) PegIFN alfa-2a plus oral lamivudine, (2) PegIFN alfa-2a plus matched placebo, or (3) oral lamivudine alone [[25](#pone.0199198.ref025)]. In the second study, the same three treatment regimens were evaluated in HBeAg-positive patients with an HBV DNA level >500,000 copies/ml and elevated ALT levels [[26](#pone.0199198.ref026)]. In the third study (NEPTUNE), HBeAg-positive patients with an HBV DNA level >100,000 copies/ml and elevated ALT levels were randomized to either 24 or 48 weeks of treatment with PegIFN alfa-2a at a dosage of either 90 or 180 μg/week [[27](#pone.0199198.ref027)]. Patients co-infected with HCV or human immunodeficiency virus and individuals with a history or evidence of decompensated cirrhosis were excluded from each of the three trials.

This pooled retrospective analysis included only patients who received PegIFN alfa-2a at a dose of 180 μg/week and for a duration of 48 weeks, had provided informed consent for *IFNL3* genotyping, and had an adequate sample available for testing.

### Sample collection, processing, and analysis

Three tag SNPs in the *IFNL3* region of chromosome 19 were investigated (rs12980275, rs8099917, and rs12979860). These SNPs were selected because they have been shown to influence the response to IFN-based therapies in patients with chronic HCV infection [[1](#pone.0199198.ref001)–[3](#pone.0199198.ref003),[9](#pone.0199198.ref009),[27](#pone.0199198.ref027)].

DNA was extracted from stored serum samples using the QIAamp DNA Blood Mini QIAcube Kit (Qiagen, Hilden, Germany). The concentration and presence of extracted DNA was measured and confirmed using a PicoGreen assay with a DNA/RNA spectrophotometer. Samples from China were genotyped using the MassARRAY System (Agena Bioscience, San Diego, CA, USA) at CapitalBio Corp., Beijing, China. Samples from regions outside of China were genotyped via TaqMan PCR (Roche Molecular Diagnostics, Pleasanton, CA, USA) at Quintiles-Singapore, Singapore.

### Definition of response

The pre-specified primary outcome in all three studies was treatment response at 24 weeks after the end of treatment. For the purposes of this analysis, treatment response was defined as follows: (1) HBeAg seroconversion plus HBV DNA <2000 IU/ml or HBsAg loss in HBeAg-positive patients; or (2) HBV DNA <2000 IU/ml or HBsAg loss in HBeAg-negative patients.

### Statistics

Separate statistical analyses were conducted for Asian and white patients within the HBeAg-positive and HBeAg-negative populations because of differences in HBV genotype distribution among ethnic groups. Baseline characteristics (age, sex, ethnic origin, HBV genotype, baseline ALT, baseline HBV DNA) were summarized overall and by ethnicity for the HBeAg-positive and HBeAg-negative populations. The Wald chi-square test was used to test for significant associations between treatment response and *IFNL3* genotype for each SNP. For each of the three SNPs, unadjusted Wald odds ratios, 95% confidence intervals, and associated p-values were estimated from univariate logistic regression models to assess the quantitative effect of SNP genotype on outcome, fitting the *IFNL3* SNP genotype as a fixed binary effect and treatment response as the binary outcome variable.

To identify other significant baseline predictors of treatment response, baseline covariates (sex, age, race, HBV genotype, HBV DNA level, and ALT level) were explored separately using univariate logistic regression models in HBeAg-positive Asian and white patients and in HBeAg-negative Asian and white patients. Significant associations were defined as p≤0.05. The final analysis for each SNP was an additive logistic regression model, modelling the effect of genotype, adjusted for significant baseline covariates, on treatment response. All statistical analyses were performed using SAS 9.2 (SAS Institute Inc., Cary, NC, USA).

Deviation from Hardy–Weinberg equilibrium was assessed by Fisher’s exact test, and linkage disequilibrium was assessed by the squared Pearson correlation coefficient (r^2^) for each SNP using Plink (v1.07) ([http://pngu.mgh.harvard.edu/purcell/plink/](http://pngu.mgh.harvard.edu/purcell/plink/)) [[28](#pone.0199198.ref028)]. Markers yielding Fisher’s exact p-values of <0.001 were investigated for genotyping errors but were not excluded from further analysis.

### Ethics

The clinical studies included in this analysis were conducted in accordance with the Declaration of Helsinki and the laws and regulations in force in the countries in which the research was conducted. In the original studies the protocols and all amendments were approved by the independent Institutional Review Boards of relevant institutions listed below. All patients provided informed written consent prior to participating in the studies. Genetic analysis of samples in China was approved by the Administration Office of National Human Genetic Resources of China.

### Ethics committees and institutional review boards

The following ethics committees/institutional review boards, listed by country, considered and approved the study protocol.

**Australia:** Greenslopes Research and Ethics Committee, Greenslopes; St Vincent's Hospital (Melbourne) HREC D, Melbourne; Research Ethics Committee Royal Brisbane Hospital, Herston; CSAHS Human Research Ethics Committee- CRGH Zone, Concord.

**Brazil:** Comitê de Ética e Pesquisa do Hospital das Clínicas e Faculdade de Medicina de Ribeirão Preto da USP–HCFM, São Paulo; Comitê de Ética e Pesquisa da Faculdade de Ciências Médicas—UNICAMP, São Paulo; Comitê de Ética e Pesquisa do Hospital Universitário Prof.Edgard Santos-UFBA, Bahia; Comitê de Ética e Pesquisa da Faculdade de Medicina do ABC\Fundação do ABC—FMABC, São Paulo; Comitê de Ética e Pesquisa do Hospital das Clínicas da Faculdade de Medicina da USP–HCFMUSP, São Paulo; Faculdade de Ciencias Medicas, São Paulo, Bioethiccs Research Committee, Salvador Bahia; HCPA Institutional Review Board.

**Canada:** Biomedical Research Ethics Board, Winnipeg.

**China:** EC of Shanghai Public Health Clinical Center, Shanghai; EC of 3rd affiliated hospital of Sun Yat-sen University, Guangzhou; Ethics committee of Ruijin Hospital, Shanghai; Ethics committee of Xiangya Hospital, Changsha; EC of Beijing Friendship Hospital, Beijing; EC of Beijing You’an Hospital, Beijing; Zhe Jiang University Ethics Committee, Hangzhou; Shanghai Second Medical University Ethics Committee, Shanghai; Jing An Central Hospital Ethics Committee, Shanghai; 1st Hospital of Beijing University Ethics Committee, Beijing; Beijing 302 Hospital Ethics Committee, Beijing; The Ethics Committee of Beijing Ditan Hospital, Beijing; The 1st Hospital of West China University of Medical Science Ethics Committee, Chengdu; Shanghai Hua Shan Hospital Ethics Committee, Shanghai; Shanghai Changhai Hospital Ethics Committee, Shanghai; 2nd Affiliated Hospital of Chongqing Medical University Ethics Committee, Chongqing; Ethics Committee of 1st PLA Medical University, Guangzhou; The EC of 1st affiliated Hospital, College of Medical Science Zhejiang University, Hangzhou; The EC of Shanghai Huashan Hospital, Shanghai; Ethics Committee of People's Hospital, Peking University, Beijing.

**France:** Comité de Protection des Personnes Ile-de-France IV, Paris; Independent Ethics Committee of Paris Saint Louis (CCPPRB Paris Saint Louis), Paris.

**Germany:** Landesamt fuer Gesundheit und Soziales (LaGeSo), Berlin; Ethik-Kommission der Medizinischen Fakultaet der Universitaet zu Koeln, Koeln; Ethik-Kommission der Albert-Ludwigs-Universitaet Freiburg, Freiburg; Ethics Committee of the Campus Virchow Klinikum, Berlin; Ethics Committee of the Medical Faculty Ludwig Maximilians University, Munich; Ethik-Kommission der Medizinische Fakultät, Klinikum der Universität München; UniveHannover Medical University (MHH) Ethics Committee, Hannover; Ethics Committee of the Faculty of Medicine University of Düsseldorf, Düsseldorf; Ethics Committee Ruhr University Bochum, Bochum; Ethik-Kommission der Universitätsklinikum Frankfurt, Frankfurt.

**Greece:** Board of Directors "G. Papanikolaou" Hospital, Thessaloniki; Board of Directors "H. Dynant" Hospital, Athens; Board of Directors University Hospital of Ioannina, Ioannina; Board of Directors "Papageorgiou" Gen. District Hospital of Thessaloniki.

**Hong Kong:** HKU/HA HKW Institutional Review Board, Hong Kong; Joint CUHK-NTEC Clinical Research Ethics Committee, Hong Kong; Clinical & Research EthicsCommittee, KW Cluster, Hong Kong.

**Italy:** Segreteria Comitato Etico, Pisa; Comitato Etico Policlinico Universitario di Cagliari, Cagliari; Comitato Etico Policlinico Universitario di Palermo, Palermo; Comitato Etico Indipendente Locale Azienda Ospedaliera, Bari; Comitato Etico Azienda Ospedaliera, Brescia.

**Israel:** The State of Israel Ministry of Health, Tel Aviv; Rabin Medical Center Affiliated with the Tel Aviv University, Petach-Tiqwa; Rambam Medical Center, Haifa.

**Korea:** Seoul National Univ. Ethics Committee, Seoul; Asan Medical Center Ethics Committee, Seoul; Severance Hospital–Yonsei University Ethics Committee, Seoul; Yonsei University College of Medicine, Seoul; Samsung Medical Centre Institutional Review Board, South Korea; The Catholic University of Korea, Seoul.

**New Zealand:** Northern X Ethics Committee, Auckland; Auckland Ethics Committee, Auckland.

**Poland:** Committee of Bioethics Ludwik Rydygier Memorial Medical University of Bydgoszcz, Bydgoszez; Bioethics Committee of Postgraduate Medical Training Centre, Warsaw; Committee of Bioethics Medical University in Bialystok, Bialystok; Bioethics Committee of Medical University of Lodz, CMKP Bioethics Committee, Warsaw.

**Russia:** Ethics Committee of Stavropol State Medical Academy, Stavropol; Ethics Committee of Hepatologist LLC, Samara.

**Singapore:** Central IRB Domain E, Singapore; Singapore General Hospital Ethics Committee, Singapore; Parkway Independent Ethics Committee, Singapore.

**Spain:** Hospital General Universitario de Valencia Comité Etico de Investigación Clínica, Valencia; Hospital Virgen del Rocio Comité Etico de Investigación, Clínic, Sevilla.

**Switzerland:** Ethics Committee of both Basel EKBB, Basel; Kantonale Ethik-Kommission Bern, Bern;.

**Taiwan:** Ethics Committee National Taiwan University Hospital, Taipei; Joint Institutional Review Board Department of Medical Research Education, Taipei; Institutional Review Board, Taipei Veterans General Hospital, Taipei; The Protection of Human Subjects Institutional Review Board Tri-Service General Hospital, Taipei.

**Thailand:** The Ethics Committee, Prince of Songkla University, Had Yai; Ethical Review Committee Royal Thai Army Medical Department, Bangkok; Ethical Clearance Committee on Human Rights Related to Researches Involving Human Subjects, Bangkok; Research Ethics Committee, Faculty of Medicine, Chiang Mai University, Chiang Mai; Siriraj Institutional Review Board, Bangkok; The Khon Kaen University Ethics Committee For Human Research, Khon Kaen; The Ethics Committee, Faculty of Medicine, Chulalongkorn University, Bangkok.

**Turkey:** Marmara Universitesi Tip Fakultesi Arastirma Etik Kurulu, Istanbul; Ankara Universitesi Tip Fakultesi Etik Kurulu, Ankara; Cukurova Universitesi Tip Fakultesi Etik Kurulu, Adana; Istanbul Universitesi Istanbul Tip Fakultesi Yerel Etik Kurulu, Istanbul; Uludag Universitesi Universitesi Tip Fakultesi Etik Kurulu, Bursa.

**United States:** California Pacific Medical Center Institutional Review Board, San Francisco; McGuire Institutional Review Board, Richmond; Stanford University Research Compliance Office, Stanford; Chesapeake Research Review, Inc., Columbia; Kaiser Permanente Northwest Institutional Review Board, Portland; Office for the Protection of Research Subjects, Los Angeles; Institutional Review Board Ernory University, Atlanta; Mayo Foundation Institutional Review Board, Rochester; Western Institutional Review Board, Olympia; New England Medical Centre, Boston; Committee on the Protection of the Rights of Human Subjects, The School of Medicine, Chapel Hill; California Pacific Medical Center Institutional Review Board, San Francisco; Western Institutional Review Board, Olympia; Human Subjects Review Committee, Seattle; Human Subjects Division, University of Michigan Medical School, Ann Arbor; Cedars-Sinai Medical Center, Office of Research Compliance, Beverly Hills; Minnesota Clinical Research Center, Institutional Human Research Committee, Minneapolis; Institutional Review Board Huntington Memorial Hospital; Pasadena; Institutional Review Board; St Francis Medical Center, Honolulu; Institutional Review Board, University of Puerto Rico Medical Sciences Campus, San Juan, Puerto Rico.

## Results

A total of 701 patients with CHB who received peginterferon alfa-2a 180 μg/week for 48 weeks had *IFNL3* genotype data available and were included in this analysis. Most patients (614, 88%) were Asian. For the majority of analyses patients were divided into four populations: Asian, HBeAg-positive (n = 428), Asian, HBeAg-negative (n = 186), white, HBeAg-positive (n = 31), and white, HBeAg-negative (n = 47). Nine patients who were not classified as either Asian or white were not included in analyses according to race. Among the 465 HBeAg-positive patients, 233 (50%) were infected with HBV genotype C and 125 (27%) with genotype B. Among the 236 HBeAg-negative patients, 96 (41%) were infected with genotype C and 55 (23%) with genotype B. The discontinuation rate in the original three trials was low (5–7%) and few patients were excluded for failure to complete 48 weeks of planned treatment. The number of patients from each of the original three trials is shown in Fig A in [S1 File](#pone.0199198.s001). The baseline characteristics of HBeAg-positive and HBeAg-negative patients are shown in [Table 1](#pone.0199198.t001), stratified by ethnicity. HBV genotypes B and C predominated in Asian patients, whereas genotypes A and D were more common in white patients ([Table 1](#pone.0199198.t001)).

### Table 1. Demographic and clinical characteristics of patients Included in the analyses.

| Characteristic | HBeAg-Positive Patients |  |  | HBeAg-Negative Patients |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| All patients (n = 465)a | Asian patients (n = 428) | White patients (n = 31) | All patients (n = 236)b | Asian patients (n = 186) | White patients (n = 47) |  |
| Male, n (%) | 345 (74.2) | 315 (73.6) | 25 (80.6) | 196 (83.1) | 158 (84.9) | 35 (74.5) |
| Female, n (%) | 120 (25.8) | 113 (26.4) | 6 (19.4) | 40 (16.9) | 28 (15.1) | 12 (25.5) |
| Mean ± SD age, years | 32.0 ± 9.6 | 32.0 ± 9.3 | 33.2 ± 13.4 | 38.8 ± 10.8 | 38.1 ± 10.9 | 41.0 ± 9.7 |
| HBV genotype, n (%) |   |   |   |   |   |   |
| A | 24 (5.2) | 2 (0.5) | 20 (64.5) | 16 (6.8) | 0 | 15 (31.9) |
| B | 125 (26.9) | 125 (29.2) | 0 | 55 (23.3) | 53 (28.5) | 1 (2.1) |
| C | 233 (50.1) | 231 (54.0) | 0 | 96 (40.7) | 96 (51.6) | 0 |
| D | 7 (1.5) | 0 | 7 (22.6) | 24 (10.2) | 0 | 24 (51.1) |
| Other/unknown | 76 (16.3) | 70 (16.4) | 4 (12.9) | 45 (19.1) | 37 (19.9) | 7 (14.9) |
| Mean ± SD HBV DNA level, log10 | 9.5 ± 2.0 | 9.5 ± 2.0 | 9.4 ± 2.1 | 7.2 ± 2.0 | 7.3 ± 2.1 | 7.0 ± 1.35 |
| Mean ± SD ALT level, log10 | 2.0 ± 0.3 | 2.0 ± 0.3 | 1.9 ± 0.3 | 1.8 ± 0.3 | 1.8 ± 0.3 | 1.8 ± 0.3 |
| Response rate, n (%) | 119 (25.6) | 109 (25.5) | 10 (32.3) | 110 (46.6) | 97 (52.2) | 13 (27.7) |
| IFNL3 genotype distribution, n (%) |   |   |   |   |   |   |
| rs12979860 |   |   |   |   |   |   |
| CC | 353/442 (79.9) | 339/405 (83.7)* | 11/31 (35.5) | 164/209 (78.5) | 137/159 (86.2)* | 26/47 (55.3) |
| non-CC | 89/442 (20.1) | 66/405 (16.3) | 20/31 (64.5) | 45/209 (21.5) | 22/159 (13.8) | 21/47 (44.7) |
| rs12980275 |   |   |   |   |   |   |
| AA | 374/457 (81.8) | 359/420 (85.5)* | 12/31 (38.7) | 189/230 (82.2) | 162/180 (90.0)* | 26/47 (55.3) |
| non-AA | 83/457 (18.2) | 61/420 (14.5) | 19/31 (61.3) | 41/230 (17.8) | 18/180 (10.0) | 21/47 (44.7) |
| rs8099917 |   |   |   |   |   |   |
| TT | 398/457 (87.1) | 372/420 (88.6)* | 20/31 (64.5) | 199/230 (86.5) | 164/180 (91.1)* | 32/47 (68.1) |
| non-TT | 59/457 (12.9) | 48/420 (11.4) | 11/31 (35.5) | 31/230 (13.5) | 16/180 (8.9) | 15/47 (31.9) |

Table 1 Caption: ALT, alanine aminotransferase; HBeAg, hepatitis B e antigen; HBV, hepatitis B virus; SD, standard deviation. a Includes four black patients, one non-Asian patient, and one Pacific Islander. b Includes two black patients and one non-Asian patient. * p<0.0001 (Chi-square) for the difference in genotype distribution between the Asian and white populations.

The genotype distribution of *IFNL3* genotypes at rs12979860 in HBeAg-positive and HBeAg-negative patients is shown in [Table 1](#pone.0199198.t001). The overall frequency of CC and non-CC genotypes was similar in HBeAg-positive and HBeAg-negative patients. In the HBeAg-positive population, the CC genotype was more common in Asian (84%) than in white (36%) patients (p<0.0001). Similarly, in the HBeAg-negative population more Asian than white patients (86% versus 55%, p<0.0001) had a CC genotype. Similar patterns in genotype distribution were observed at rs12980275 and rs8099917 ([Table 1](#pone.0199198.t001)). Linkage disequilibrium r^2^ results indicated the strongest linkage disequilibrium was between rs12979860 and rs12980275, with r^2^ values 0.68–1.0 across the subgroups of Asian and white HBeAg-positive and -negative patients. Scores were similar for rs12979860 and rs8099917 (r^2^ 0.43–0.65) and for rs12980275 and rs8099917 (r^2^ 0.47–0.79) (Table A in [S1 File](#pone.0199198.s001)). Fisher’s exact p-values, to test for deviation from Hardy–Weinberg equilibrium for rs12980275, rs12979860, and rs8099917, respectively, were 0.296, 0.011, and 0.188 (HBeAg-positive Asian patients), 1.0, 0.212, and 1.0 (HBeAg-negative Asian patients), 0.420, 0.242, and 1.0 (HBeAg-positive white patients) and 0.705, 0.705, and 0.578 (HBeAg-negative white patients). The deviation from Hardy–Weinberg equilibrium for rs12979860 in HBeAg-positive Asian patients could indicate either violation of one or more assumptions of Hardy–Weinberg equilibrium, could be an indication of genotyping errors [[29](#pone.0199198.ref029)], or could be by chance. The potential consequence of such a deviation is a biased estimate of the association’s magnitude for rs12979860 in HBeAg-positive Asian patients.

The overall response rate was 26% (119/465) among HBeAg-positive patients and 47% (110/236) among HBeAg-negative patients.

Asian and white patients were analyzed separately, given the difference in frequency of HBV genotypes between the two groups ([Table 1](#pone.0199198.t001)). Results of the univariate logistic regression analysis indicated that there were no statistically significant associations between treatment response and genotype at any of the three SNPs in the HBeAg-positive Asian ([Fig 1A](#pone.0199198.g001)) or white ([Fig 1B](#pone.0199198.g001)) populations, or in the HBeAg-negative Asian ([Fig 1C](#pone.0199198.g001)) or white populations ([Fig 1D](#pone.0199198.g001)). The results did not change when HBeAg seroconversion alone was considered as a treatment outcome in HBeAg-positive patients ([Table 2](#pone.0199198.t002)). In both the HBeAg-positive and -negative populations, numerically lower response rates were observed among white carriers of non-CC rs12979860 genotypes compared with carriers of CC rs12979860 genotypes ([Fig 1B and 1D](#pone.0199198.g001)); however, as both analyses included only a very small number of samples, these differences failed to reach statistical significance.

### Fig 1. Response rates according to host IFNL3 genotype.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4775/6049926/ad25a4cca065/pone.0199198.g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6049926_pone.0199198.g001.jpg)

Response rates according to genotype at rs12980275, rs12979860, and rs8099917 in HBeAg-positive Asian (A) and white (B) patients, and in HBeAg-negative Asian (C) and white (D) patients. Some patients did not have data available for all genotypes, and response rates are calculated on the basis of the number of patients with data.

### Table 2. Rates of HBeAg seroconversion in HBeAg-positive patients according to genotype at rs12980275, rs12979860, and rs8099917.

| HBeAg seroconversion rate, n (%) | IFNL3 genotype |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs12979860 |  |  | rs12980275 |  |  | rs8099917 |  |  |  |
| CC | non-CC | p-value | AA | non-AA | p-value | TT | non-TT | p-value |  |
| All patients | 124/353 (35.1) | 33/89 (37.1) | 0.7311 | 134/374 (35.8) | 30/83 (36.1) | 0.9567 | 143/398 (35.9) | 20/59 (33.9) | 0.7612 |
| Asian patients | 117/339 (34.5) | 25/66 (37.9) | 0.6003 | 127/359 (35.4) | 22/61 (36.1) | 0.916 | 133/372 (35.8) | 15/48 (31.3) | 0.5394 |
| White patients | 6/11 (54.5) | 8/20 (40.0) | 0.4383 | 6/12 (50.0) | 8/19 (42.1) | 0.6674 | 9/20 (45.0) | 5/11 (45.5) | 0.9806 |

Table 2 Caption: Some patients did not have data available for all genotypes, and percentages are calculated on the basis of the number of patients with data.

### Exploration of baseline covariates

Associations between baseline covariates and treatment response were explored separately by univariate logistic regression analyses in HBeAg-positive Asian and white patients ([Fig 2A and 2B](#pone.0199198.g002)) and in HBeAg-negative Asian and white patients ([Fig 2C and 2D](#pone.0199198.g002)). For HBeAg-positive Asian patients, significant covariates (p<0.05) were sex, HBV DNA, and ALT. For HBeAg-negative Asian patients, age was the only significant baseline predictor of response. For HBeAg-negative white patients, HBV DNA was found to be significant. No significant associations were found for HBeAg positive white patients.

### Fig 2. Associations between baseline covariates and treatment outcome in univariate logistic regression analyses.

![Fig 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4775/6049926/3461b5be963c/pone.0199198.g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6049926_pone.0199198.g002.jpg)

Associations are shown for HBeAg-positive Asian (A) and white (B) patients, and in HBeAg-negative Asian (C) and white (D) patients. ALT, alanine aminotransferase; HBeAg, hepatitis B e antigen; HBV, hepatitis B virus. After adjusting for significant baseline covariates, there were no statistically significant associations between IFNL3 genotypes and response in the final logistic regression models (Fig 3).

### Fig 3. Associations between baseline covariates and treatment outcome in multivariate logistic regression analyses.

![Fig 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4775/6049926/ee12513e50c1/pone.0199198.g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6049926_pone.0199198.g003.jpg)

Associations are shown for HBeAg-positive (A) and HBeAg-negative (B) patients. ALT, alanine aminotransferase; HBeAg, hepatitis B e antigen; HBV, hepatitis B virus.

### HBsAg loss

A total of 18 of 465 (3.9%) HBeAg-positive patients (including 12/428 Asian and 6/31 white patients) and 10 of 236 (4.2%) HBeAg-negative patients (including 8/186 Asian and 2/47 white patients) included in the analysis experienced HBsAg loss. Given these low event rates, it was not possible to test for an association between *IFNL3* genotype and HBsAg loss.

## Discussion

The identification of *IFNL3* genotype as a biomarker predicting response to dual PegIFN alfa/ribavirin therapy for chronic HCV infection was immediately useful in the clinic for identifying patients most likely to respond to therapy. A similar tool for stratifying IFN responsiveness in CHB patients would be extremely useful to predict, at baseline, which patients are most likely to respond to PegIFN alfa.

The underlying hypothesis for this study was that *IFNL3* genotype would predict the outcome of treatment with PegIFN alfa in patients with CHB. However, our primary finding was that there was no association between *IFNL3* genotype and treatment response. *IFNL3* genotype was not associated with treatment response in HBeAg-positive or -negative patients of either Asian or white background.

Previous analyses that have examined relationships between these SNPs and response to IFN in patients with CHB have provided discordant results [[15](#pone.0199198.ref015)–[23](#pone.0199198.ref023)]. The largest of these analyses (n = 205) considered the relationship between *IFNL3* genotype (rs12980275 and rs12979860) and treatment outcome in HBeAg-positive patients [[18](#pone.0199198.ref018)]. A significant association was reported between *IFNL3* genotype and the specified primary outcome, which was HBeAg seroconversion alone. However, the relationship between response and *IFNL3* genotype was no longer significant when a more stringent composite endpoint (i.e. HBeAg seroconversion plus HBV DNA level <2000 IU/ml at 24 weeks post-treatment) was considered [[18](#pone.0199198.ref018)]. Moreover, Sonneveld *et al*. [[18](#pone.0199198.ref018)] found that *IFNL3* genotype was not a significant predictor of response to IFN when included in a previously calibrated PegIFN treatment index developed by Buster *et al*. [[30](#pone.0199198.ref030)] Similarly, *IFNL3* genotype was not found to be associated with treatment response in a number of smaller cohorts of patients with HBeAg-positive CHB [[15](#pone.0199198.ref015), [17](#pone.0199198.ref017), [21](#pone.0199198.ref021)].

Lampertico *et al*. [[22](#pone.0199198.ref022)] found the rs12979860 CC genotype to be significantly associated with response to IFN in 101 HBeAg-negative patients with HBV genotype D infection who had received IFN-based therapies. *IFNL3* genotype was associated with the primary response (HBsAg clearance) after a median of 11 years of follow-up. *IFNL3* genotype was also associated with response defined as HBV DNA <2000 IU/ml at 6 months after treatment. It is important to note that the median duration of therapy was 23 months (range 10–48 months) [[22](#pone.0199198.ref022)]. Contrary to the results of Lampertico *et al*., a number of other analyses have not found a significant relationship between *IFNL3* genotype and response to PegIFN alfa in patients with HBeAg-negative CHB [[17](#pone.0199198.ref017), [21](#pone.0199198.ref021)], including a number of southern European cohorts, most of whom had HBV genotype D infection [[20](#pone.0199198.ref020), [23](#pone.0199198.ref023), [31](#pone.0199198.ref031)].

The explanation for these discordant results is most likely related to differences in sample size, patient characteristics (including ethnicity and HBV genotype), and the treatment regimen used. One of the strengths of the current study is the large sample size, as well as the very detailed patient phenotypes available from the PegIFN alfa-2a registration studies. The distribution of genotypes in Asian and white patients was consistent with previous reports of minor allele frequency in these populations [[32](#pone.0199198.ref032)–[37](#pone.0199198.ref037)]. The data are clear that, among Asian patients, there is no evidence for clinical utility of *IFNL3* genotyping prior to consideration of PegIFN alfa-based therapy. Similarly, the data do not support a role for *IFNL3* genotyping in white CHB patients prior to PegIFN alfa therapy, although we cannot entirely exclude a weak association with IFN responsiveness in this subgroup, given the smaller number of white patients, particularly with HBeAg-negative disease. In addition, it is also possible that subgroups within the heterogeneous Asian population with different minor allele frequencies may show some association.

The explanation for this difference in the relationship between *IFNL3* genotype and IFN responsiveness between HBV and HCV is not clear. HBV and HCV are both hepatotropic viruses, and both respond to IFN therapy *in vitro* and *in vivo*. However, they have a number of fundamental differences in lifecycle and immune-evasion mechanisms. HBV is a DNA virus that has an episomal reservoir in the nucleus (cccDNA), and reverse transcription largely occurs within cytoplasmic capsids, protected from cytoplasmic RNA-sensing mechanisms. The strong association between intrahepatic IFN-stimulated gene expression and IFN responsiveness that is a feature of HCV infection, and that was recently linked to *IFNL3* genotype, has not been described for HBV. Indeed, this seems to be a specific association between *IFNL3* genotype and HCV, as it is not observed in normal liver [[38](#pone.0199198.ref038)]. Moreover, treatment with PegIFN alfa-2a has been observed to significantly reduce circulating and intrahepatic antigens in the absence of immune cell responses in a murine model of HBV infection [[39](#pone.0199198.ref039)]. Thus, fundamental differences in the host–virus relationship for HBV and HCV are clearly present, and remain an important focus for future research.

Prokunina-Olsson *et al* [[40](#pone.0199198.ref040)] have described a variant upstream of *IFNL3* that creates a novel interferon gene, *IFNL4*, that is linked with HCV clearance in PegIFN-treated patients, possibly more so even than SNPs in *IFNL3*. The role of this variant in HBV response to PegIFN is currently unknown, however, and was not investigated in the current study.

In conclusion, the present study is the largest analysis to date of the association between *IFNL3* genotype and response to IFN in patients with CHB. The data suggest that *IFNL3* polymorphisms are not a major determinant of response to PegIFN alfa-2a in patients with HBeAg-positive or HBeAg-negative CHB. When selecting patients for treatment with PegIFN alfa-2a, clinicians should continue to use established baseline predictors of response.

## Supporting information

## Acknowledgments

The authors thank the patients in this study for consenting to sample analysis.

## Data Availability

All relevant data are within the paper and its supporting information files.

## Funding Statement

This study was funded in part by F. Hoffmann-La Roche Ltd, Basel, Switzerland. Professor Lai Wei had additional funding provided by the China National Science and Technology Major Project for Infectious Diseases Control during the 12th Five-Year Plan Period (2012ZX10002003). The funder provided support in the form of salaries for authors HH, YZ, CW and BS, but did not have any additional role in the study design, data collection and analysis, decision to publish, or preparation of the manuscript. The specific roles of these authors are articulated in the ‘Contributions to the manuscript’ section. Support for third-party writing assistance for this manuscript, furnished by Blair Jarvis Health Interactions, was provided by Roche Products Ltd, Welwyn, UK. LW has received fees for serving as a speaker, consultant, and advisory board member for AbbVie, Bristol-Myers Squibb (BMS), Gilead, Johnson & Johnson (J&J), and GlaxoSmithKline (GSK) within the last 3 years. Additional funding for the study was provided to LW by the China National Science and Technology Major Project for Infectious Diseases Control during the 12th Five-Year Plan Period. HW has received grant/research support from Abbott, BMS, Gilead, Merck, Novartis, Roche, Roche Diagnostics, and Siemens, and has received consulting fees and has been an advisory board and speakers’ bureau member for Abbott, AbbVie, Biolex, BMS, Boehringer Ingelheim, Eiger Pharmaceuticals, Falk Foundation, Gilead, IST, J&J/Janssen-Cilag/Janssen TE, Medgenics, Merck/Schering-Plough, Novartis, Novira, Roche, Roche Diagnostics, Siemens, Transgene, and ViiV. Y-FL has served as an advisory board member for Roche. HL-YC has received fees for serving as a speaker and advisory board member for Roche, BMS, Gilead, Novartis, and MSD, and as a speaker for Echosens and GSK, and has received an unrestricted grant for HBV research from Roche. TP has received grant/research support from BMS, Roche, MSD, Novartis, Fibrogen, and Bayer, and has been a member of advisory boards or speakers’ bureaux for BMS, Roche, MSD, Novartis, and GSK. JJ has received fees for serving as a speaker, consultant, and advisory board member for BMS, MSD, Novartis, and Roche within the last 3 years. MRB has been a member of speakers’ bureaux for BMS, Gilead, Roche, and Janssen, and has been a member of advisory boards for AbbVie, Roche, Gilead, and MSD. MD has been a member of speakers’ bureaux for Roche, MSD, AbbVie, BMS, Gilead, and Janssen. SG has received fees as a member of advisory boards and speakers’ bureaux for Roche, BMS, Gilead, Janssen, and MSD. YZ has been a Genentech/Roche employee and has owned Roche stock and options. HH and CW are employees of Roche and declare stock ownership. BS has received fees for serving as a consultant/contract statistician for Roche, through BStats Solutions Ltd, from March 2013 to the present. AJT has received grant funding from AbbVie, Gilead Sciences, BMS, and Merck, has received consultancy fees from AbbVie, Gilead Sciences, BMS, Roche Diagnostics, Merck, and Spring Bank Pharmaceuticals, and has provided sponsored lectures (national or international) for AbbVie, Roche Diagnostics, Gilead Sciences, BMS, and Merck. The China National Science and Technology Major Project for Infectious Diseases Control during the 12th Five-Year Plan Period provided support in the form of salaries to HH, YZ, CW, and BS.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its supporting information files.

### Supplementary Materials

### Data Availability Statement

All relevant data are within the paper and its supporting information files.

## References

1. Ge D, Fellay J, Thompson AJ, Simon JS, Shianna KV, Urban TJ, et al. Genetic variation in IL28B predicts hepatitis C treatment-induced viral clearance. Nature 2009;461:399–401. 10.1038/nature08309  [DOI](https://doi.org/10.1038/nature08309) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20predicts%20hepatitis%20C%20treatment-induced%20viral%20clearance&author=D%20Ge&author=J%20Fellay&author=AJ%20Thompson&author=JS%20Simon&author=KV%20Shianna&volume=461&publication_year=2009&pages=399-401&pmid=19684573&doi=10.1038/nature08309&)

2. Suppiah V, Moldovan M, Ahlenstiel G, Berg T, Weltman M, Abate ML, et al. IL28B is associated with response to chronic hepatitis C interferon-alpha and ribavirin therapy. Nat Genet 2009;41:1100–1104. 10.1038/ng.447  [DOI](https://doi.org/10.1038/ng.447) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=IL28B%20is%20associated%20with%20response%20to%20chronic%20hepatitis%20C%20interferon-alpha%20and%20ribavirin%20therapy&author=V%20Suppiah&author=M%20Moldovan&author=G%20Ahlenstiel&author=T%20Berg&author=M%20Weltman&volume=41&publication_year=2009&pages=1100-1104&pmid=19749758&doi=10.1038/ng.447&)

3. Tanaka Y, Nishida N, Sugiyama M, Kurosaki M, Matsuura K, Sakamoto N, et al. Genome-wide association of IL28B with response to pegylated interferon-alpha and ribavirin therapy for chronic hepatitis C. Nat Genet 2009;41:1105–1109. 10.1038/ng.449  [DOI](https://doi.org/10.1038/ng.449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20of%20IL28B%20with%20response%20to%20pegylated%20interferon-alpha%20and%20ribavirin%20therapy%20for%20chronic%20hepatitis%20C&author=Y%20Tanaka&author=N%20Nishida&author=M%20Sugiyama&author=M%20Kurosaki&author=K%20Matsuura&volume=41&publication_year=2009&pages=1105-1109&pmid=19749757&doi=10.1038/ng.449&)

4. Thompson AJ, Muir AJ, Sulkowski MS, Ge D, Fellay J, Shianna KV, et al. Interleukin-28B polymorphism improves viral kinetics and is the strongest pretreatment predictor of sustained virologic response in genotype 1 hepatitis C virus. Gastroenterology 2010;139:120–129. 10.1053/j.gastro.2010.04.013  [DOI](https://doi.org/10.1053/j.gastro.2010.04.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20399780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Interleukin-28B%20polymorphism%20improves%20viral%20kinetics%20and%20is%20the%20strongest%20pretreatment%20predictor%20of%20sustained%20virologic%20response%20in%20genotype%201%20hepatitis%20C%20virus&author=AJ%20Thompson&author=AJ%20Muir&author=MS%20Sulkowski&author=D%20Ge&author=J%20Fellay&volume=139&publication_year=2010&pages=120-129&pmid=20399780&doi=10.1053/j.gastro.2010.04.013&)

5. Poordad F, Bronowicki JP, Gordon SC, Zeuzem S, Jacobson IM, Sulkowski MS, et al. Factors that predict response of patients with hepatitis C virus infection to boceprevir. Gastroenterology 2012;143:608–618. 10.1053/j.gastro.2012.05.011  [DOI](https://doi.org/10.1053/j.gastro.2012.05.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22626609/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Factors%20that%20predict%20response%20of%20patients%20with%20hepatitis%20C%20virus%20infection%20to%20boceprevir&author=F%20Poordad&author=JP%20Bronowicki&author=SC%20Gordon&author=S%20Zeuzem&author=IM%20Jacobson&volume=143&publication_year=2012&pages=608-618&pmid=22626609&doi=10.1053/j.gastro.2012.05.011&)

6. Thompson AJ. Genetic factors and hepatitis C virus infection. Gastroenterology 2012;142:1335–1339. 10.1053/j.gastro.2012.01.046  [DOI](https://doi.org/10.1053/j.gastro.2012.01.046) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22537440/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Genetic%20factors%20and%20hepatitis%20C%20virus%20infection&author=AJ%20Thompson&volume=142&publication_year=2012&pages=1335-1339&pmid=22537440&doi=10.1053/j.gastro.2012.01.046&)

7. Muir AJ, Bornstein JD, Killenberg PG. Peginterferon alfa-2b and ribavirin for the treatment of chronic hepatitis C in blacks and non-Hispanic whites. N Engl J Med 2004;350:2265–2271. 10.1056/NEJMoa032502  [DOI](https://doi.org/10.1056/NEJMoa032502) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15163776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Peginterferon%20alfa-2b%20and%20ribavirin%20for%20the%20treatment%20of%20chronic%20hepatitis%20C%20in%20blacks%20and%20non-Hispanic%20whites&author=AJ%20Muir&author=JD%20Bornstein&author=PG%20Killenberg&volume=350&publication_year=2004&pages=2265-2271&pmid=15163776&doi=10.1056/NEJMoa032502&)

8. Rodriguez-Torres M, Jeffers LJ, Sheikh MY, Rossaro L, Ankoma-Sey V, Hamzeh FM, et al. Peginterferon alfa-2a and ribavirin in Latino and non-Latino whites with hepatitis C. N Engl J Med 2009;360:257–267. 10.1056/NEJMoa0805062  [DOI](https://doi.org/10.1056/NEJMoa0805062) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19144941/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Peginterferon%20alfa-2a%20and%20ribavirin%20in%20Latino%20and%20non-Latino%20whites%20with%20hepatitis%20C&author=M%20Rodriguez-Torres&author=LJ%20Jeffers&author=MY%20Sheikh&author=L%20Rossaro&author=V%20Ankoma-Sey&volume=360&publication_year=2009&pages=257-267&pmid=19144941&doi=10.1056/NEJMoa0805062&)

9. Thomas DL, Thio CL, Martin MP, Qi Y, Ge D, O'Huigin C, et al. Genetic variation in IL28B and spontaneous clearance of hepatitis C virus. Nature 2009;461:798–801. 10.1038/nature08463  [DOI](https://doi.org/10.1038/nature08463) | [PMC free article](/articles/PMC3172006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19759533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20and%20spontaneous%20clearance%20of%20hepatitis%20C%20virus&author=DL%20Thomas&author=CL%20Thio&author=MP%20Martin&author=Y%20Qi&author=D%20Ge&volume=461&publication_year=2009&pages=798-801&pmid=19759533&doi=10.1038/nature08463&)

10. Grebely J, Petoumenos K, Hellard M, Matthews GV, Suppiah V, Applegate T, et al. Potential role for interleukin-28B genotype in treatment decision-making in recent hepatitis C virus infection. Hepatology 2010;52:1216–1224. 10.1002/hep.23850  [DOI](https://doi.org/10.1002/hep.23850) | [PMC free article](/articles/PMC2947598/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20803561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Potential%20role%20for%20interleukin-28B%20genotype%20in%20treatment%20decision-making%20in%20recent%20hepatitis%20C%20virus%20infection&author=J%20Grebely&author=K%20Petoumenos&author=M%20Hellard&author=GV%20Matthews&author=V%20Suppiah&volume=52&publication_year=2010&pages=1216-1224&pmid=20803561&doi=10.1002/hep.23850&)

11. Rauch A, Kutalik Z, Descombes P, Cai T, Di Iulio J, Mueller T, et al. Genetic variation in IL28B is associated with chronic hepatitis C and treatment failure: a genome-wide association study. Gastroenterology 2010;138:1338–1345. 10.1053/j.gastro.2009.12.056  [DOI](https://doi.org/10.1053/j.gastro.2009.12.056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20060832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Genetic%20variation%20in%20IL28B%20is%20associated%20with%20chronic%20hepatitis%20C%20and%20treatment%20failure:%20a%20genome-wide%20association%20study&author=A%20Rauch&author=Z%20Kutalik&author=P%20Descombes&author=T%20Cai&author=J%20Di%20Iulio&volume=138&publication_year=2010&pages=1338-1345&pmid=20060832&doi=10.1053/j.gastro.2009.12.056&)

12. Tillmann HL, Thompson AJ, Patel K, Wiese M, Tenckhoff H, Nischalke HD, et al. A polymorphism near IL28B is associated with spontaneous clearance of acute hepatitis C virus and jaundice. Gastroenterology 2010;139:1586–1592. 10.1053/j.gastro.2010.07.005  [DOI](https://doi.org/10.1053/j.gastro.2010.07.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20637200/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=A%20polymorphism%20near%20IL28B%20is%20associated%20with%20spontaneous%20clearance%20of%20acute%20hepatitis%20C%20virus%20and%20jaundice&author=HL%20Tillmann&author=AJ%20Thompson&author=K%20Patel&author=M%20Wiese&author=H%20Tenckhoff&volume=139&publication_year=2010&pages=1586-1592&pmid=20637200&doi=10.1053/j.gastro.2010.07.005&)

13. Honda M, Sakai A, Yamashita T, Nakamoto Y, Mizukoshi E, Sakai Y, et al. Hepatic ISG expression is associated with genetic variation in interleukin 28B and the outcome of IFN therapy for chronic hepatitis C. Gastroenterology 2010;139:499–509. 10.1053/j.gastro.2010.04.049  [DOI](https://doi.org/10.1053/j.gastro.2010.04.049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20434452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Hepatic%20ISG%20expression%20is%20associated%20with%20genetic%20variation%20in%20interleukin%2028B%20and%20the%20outcome%20of%20IFN%20therapy%20for%20chronic%20hepatitis%20C&author=M%20Honda&author=A%20Sakai&author=T%20Yamashita&author=Y%20Nakamoto&author=E%20Mizukoshi&volume=139&publication_year=2010&pages=499-509&pmid=20434452&doi=10.1053/j.gastro.2010.04.049&)

14. Urban TJ, Thompson AJ, Bradrick SS, Fellay J, Schuppan D, Cronin KD, et al. IL28B genotype is associated with differential expression of intrahepatic interferon-stimulated genes in patients with chronic hepatitis C. Hepatology 2010;52:1888–1896. 10.1002/hep.23912  [DOI](https://doi.org/10.1002/hep.23912) | [PMC free article](/articles/PMC3653303/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20931559/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=IL28B%20genotype%20is%20associated%20with%20differential%20expression%20of%20intrahepatic%20interferon-stimulated%20genes%20in%20patients%20with%20chronic%20hepatitis%20C&author=TJ%20Urban&author=AJ%20Thompson&author=SS%20Bradrick&author=J%20Fellay&author=D%20Schuppan&volume=52&publication_year=2010&pages=1888-1896&pmid=20931559&doi=10.1002/hep.23912&)

15. Tseng TC, Yu ML, Liu CJ, Lin CL, Huang YW, Hsu CS, et al. Effect of host and viral factors on hepatitis B e antigen-positive chronic hepatitis B patients receiving pegylated interferon-alpha-2a therapy. Antivir Ther 2011;16:629–637. 10.3851/IMP1841  [DOI](https://doi.org/10.3851/IMP1841) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21817184/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antivir%20Ther&title=Effect%20of%20host%20and%20viral%20factors%20on%20hepatitis%20B%20e%20antigen-positive%20chronic%20hepatitis%20B%20patients%20receiving%20pegylated%20interferon-alpha-2a%20therapy&author=TC%20Tseng&author=ML%20Yu&author=CJ%20Liu&author=CL%20Lin&author=YW%20Huang&volume=16&publication_year=2011&pages=629-637&pmid=21817184&doi=10.3851/IMP1841&)

16. Colombatto P, Cherubini B, Brunetto MR, Coco B, Cavallone D, Stabile A, et al. IL28B polymorphism in CHB infection: the CC allele does not correlate with the phase of HBV infection, but with viral load decline in Peg-IFN treated HBeAg positive patients only (abstract). Hepatology 2011;54(Suppl):258.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=IL28B%20polymorphism%20in%20CHB%20infection:%20the%20CC%20allele%20does%20not%20correlate%20with%20the%20phase%20of%20HBV%20infection,%20but%20with%20viral%20load%20decline%20in%20Peg-IFN%20treated%20HBeAg%20positive%20patients%20only%20(abstract)&author=P%20Colombatto&author=B%20Cherubini&author=MR%20Brunetto&author=B%20Coco&author=D%20Cavallone&volume=54&issue=Suppl&publication_year=2011&pages=258&)

17. de Niet A, Takkenberg RB, Benayed R, Riley-Gillis B, Weegink CJ, Zaaijer HL, et al. Genetic variation in IL28B and treatment outcome in HBeAg-positive and -negative chronic hepatitis B patients treated with Peg interferon alfa-2a and adefovir. Scand J Gastroenterol 2012;47:475–481. 10.3109/00365521.2011.648952  [DOI](https://doi.org/10.3109/00365521.2011.648952) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22263608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scand%20J%20Gastroenterol&title=Genetic%20variation%20in%20IL28B%20and%20treatment%20outcome%20in%20HBeAg-positive%20and%20-negative%20chronic%20hepatitis%20B%20patients%20treated%20with%20Peg%20interferon%20alfa-2a%20and%20adefovir&author=A%20de%20Niet&author=RB%20Takkenberg&author=R%20Benayed&author=B%20Riley-Gillis&author=CJ%20Weegink&volume=47&publication_year=2012&pages=475-481&pmid=22263608&doi=10.3109/00365521.2011.648952&)

18. Sonneveld MJ, Wong VW, Woltman AM, Wong GL, Cakaloglu Y, Zeuzem S, et al. Polymorphisms near IL28B and serologic response to peginterferon in HBeAg-positive patients with chronic hepatitis B. Gastroenterology 2012;142:513–520. 10.1053/j.gastro.2011.11.025  [DOI](https://doi.org/10.1053/j.gastro.2011.11.025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22108195/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Polymorphisms%20near%20IL28B%20and%20serologic%20response%20to%20peginterferon%20in%20HBeAg-positive%20patients%20with%20chronic%20hepatitis%20B&author=MJ%20Sonneveld&author=VW%20Wong&author=AM%20Woltman&author=GL%20Wong&author=Y%20Cakaloglu&volume=142&publication_year=2012&pages=513-520&pmid=22108195&doi=10.1053/j.gastro.2011.11.025&)

19. Brouwer WP, Sonneveld M, Arends P, Rijckborst V, Boonstra A, de Knegt RJ, et al. Single-nucleotide polymorphisms of the HLA-DP gene are associated with virologic response to peginterferon in caucasian patients with chronic hepatitis B (abstract 956). Hepatology 2013;58(Suppl):666A–667A.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Single-nucleotide%20polymorphisms%20of%20the%20HLA-DP%20gene%20are%20associated%20with%20virologic%20response%20to%20peginterferon%20in%20caucasian%20patients%20with%20chronic%20hepatitis%20B%20(abstract%20956)&author=WP%20Brouwer&author=M%20Sonneveld&author=P%20Arends&author=V%20Rijckborst&author=A%20Boonstra&volume=58&issue=Suppl&publication_year=2013&pages=666A-667A&)

20. Hadziyannis E, Laras A, Panopoulou E, Anagnostopoulou S, Hadziyannis S. IL-28B genotypes in spontaneous and treatment induced HBsAg clearance of chronic hepatitis B patients (abstract 897). Hepatology 2013;58(Suppl):633A–634A.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=IL-28B%20genotypes%20in%20spontaneous%20and%20treatment%20induced%20HBsAg%20clearance%20of%20chronic%20hepatitis%20B%20patients%20(abstract%20897)&author=E%20Hadziyannis&author=A%20Laras&author=E%20Panopoulou&author=S%20Anagnostopoulou&author=S%20Hadziyannis&volume=58&issue=Suppl&publication_year=2013&pages=633A-634A&)

21. Holmes JA, Nguyen T, Ratnam D, Heerasing NM, Tehan JV, Bonanzinga S, et al. IL28B genotype is not useful for predicting treatment outcome in Asian chronic hepatitis B patients treated with pegylated interferon-alpha. J Gastroenterol Hepatol 2013;28:861–866. 10.1111/jgh.12110  [DOI](https://doi.org/10.1111/jgh.12110) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23301835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gastroenterol%20Hepatol&title=IL28B%20genotype%20is%20not%20useful%20for%20predicting%20treatment%20outcome%20in%20Asian%20chronic%20hepatitis%20B%20patients%20treated%20with%20pegylated%20interferon-alpha&author=JA%20Holmes&author=T%20Nguyen&author=D%20Ratnam&author=NM%20Heerasing&author=JV%20Tehan&volume=28&publication_year=2013&pages=861-866&pmid=23301835&doi=10.1111/jgh.12110&)

22. Lampertico P, Vigano M, Cheroni C, Facchetti F, Invernizzi F, Valveri V, et al. IL28B polymorphisms predict interferon-related hepatitis B surface antigen seroclearance in genotype D hepatitis B e antigen-negative patients with chronic hepatitis B. Hepatology 2013;57:890–896. 10.1002/hep.25749  [DOI](https://doi.org/10.1002/hep.25749) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22473858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=IL28B%20polymorphisms%20predict%20interferon-related%20hepatitis%20B%20surface%20antigen%20seroclearance%20in%20genotype%20D%20hepatitis%20B%20e%20antigen-negative%20patients%20with%20chronic%20hepatitis%20B&author=P%20Lampertico&author=M%20Vigano&author=C%20Cheroni&author=F%20Facchetti&author=F%20Invernizzi&volume=57&publication_year=2013&pages=890-896&pmid=22473858&doi=10.1002/hep.25749&)

23. Papatheodoridis G, Gatselis N, Goulis I, Karatapanis S, Deutsch M, Mimidis K, et al. IL28B polymorphisms as predictors of response to peginterferon-alfa-2a in HBeAg-negative chronic hepatitis B (abstract 993). Hepatology 2013;58(Suppl):686A.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=IL28B%20polymorphisms%20as%20predictors%20of%20response%20to%20peginterferon-alfa-2a%20in%20HBeAg-negative%20chronic%20hepatitis%20B%20(abstract%20993)&author=G%20Papatheodoridis&author=N%20Gatselis&author=I%20Goulis&author=S%20Karatapanis&author=M%20Deutsch&volume=58&issue=Suppl&publication_year=2013&pages=686A&)

24. European Association for the Study of the Liver. EASL Clinical Practice Guidelines: Management of chronic hepatitis B virus infection. J Hepatol 2012;57:167–185. 10.1016/j.jhep.2012.02.010  [DOI](https://doi.org/10.1016/j.jhep.2012.02.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22436845/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=EASL%20Clinical%20Practice%20Guidelines:%20Management%20of%20chronic%20hepatitis%20B%20virus%20infection&volume=57&publication_year=2012&pages=167-185&pmid=22436845&doi=10.1016/j.jhep.2012.02.010&)

25. Marcellin P, Lau GK, Bonino F, Farci P, Hadziyannis S, Jin R, et al. Peginterferon alfa-2a alone, lamivudine alone, and the two in combination in patients with HBeAg-negative chronic hepatitis B. N Engl J Med 2004;351:1206–1217. 10.1056/NEJMoa040431  [DOI](https://doi.org/10.1056/NEJMoa040431) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15371578/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Peginterferon%20alfa-2a%20alone,%20lamivudine%20alone,%20and%20the%20two%20in%20combination%20in%20patients%20with%20HBeAg-negative%20chronic%20hepatitis%20B&author=P%20Marcellin&author=GK%20Lau&author=F%20Bonino&author=P%20Farci&author=S%20Hadziyannis&volume=351&publication_year=2004&pages=1206-1217&pmid=15371578&doi=10.1056/NEJMoa040431&)

26. Lau GK, Piratvisuth T, Luo KX, Marcellin P, Thongsawat S, Cooksley G, et al. Peginterferon Alfa-2a, lamivudine, and the combination for HBeAg-positive chronic hepatitis B. N Engl J Med 2005;352:2682–2695. 10.1056/NEJMoa043470  [DOI](https://doi.org/10.1056/NEJMoa043470) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15987917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Peginterferon%20Alfa-2a,%20lamivudine,%20and%20the%20combination%20for%20HBeAg-positive%20chronic%20hepatitis%20B&author=GK%20Lau&author=T%20Piratvisuth&author=KX%20Luo&author=P%20Marcellin&author=S%20Thongsawat&volume=352&publication_year=2005&pages=2682-2695&pmid=15987917&doi=10.1056/NEJMoa043470&)

27. Liaw YF, Jia JD, Chan HL, Han KH, Tanwandee T, Chuang WL, et al. Shorter durations and lower doses of peginterferon alfa-2a are associated with inferior hepatitis B e antigen seroconversion rates in hepatitis B virus genotypes B or C. Hepatology 2011;54:1591–1599. 10.1002/hep.24555  [DOI](https://doi.org/10.1002/hep.24555) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22045673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Shorter%20durations%20and%20lower%20doses%20of%20peginterferon%20alfa-2a%20are%20associated%20with%20inferior%20hepatitis%20B%20e%20antigen%20seroconversion%20rates%20in%20hepatitis%20B%20virus%20genotypes%20B%20or%20C&author=YF%20Liaw&author=JD%20Jia&author=HL%20Chan&author=KH%20Han&author=T%20Tanwandee&volume=54&publication_year=2011&pages=1591-1599&pmid=22045673&doi=10.1002/hep.24555&)

28. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet 2007;81:559–575. 10.1086/519795  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

29. Hosking L, Lumsden S, Lewis K, Yeo A, McCarthy L, Bansal A, et al. Detection of genotyping errors by Hardy–Weinberg equilibrium testing. Eur J Hum Genet 2004;12:395–399. 10.1038/sj.ejhg.5201164  [DOI](https://doi.org/10.1038/sj.ejhg.5201164) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14872201/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Detection%20of%20genotyping%20errors%20by%20Hardy%E2%80%93Weinberg%20equilibrium%20testing&author=L%20Hosking&author=S%20Lumsden&author=K%20Lewis&author=A%20Yeo&author=L%20McCarthy&volume=12&publication_year=2004&pages=395-399&pmid=14872201&doi=10.1038/sj.ejhg.5201164&)

30. Buster EH, Hansen BE, Lau GK, Piratvisuth T, Zeuzem S, Steyerberg EW, et al. Factors that predict response of patients with hepatitis B e antigen-positive chronic hepatitis B to peginterferon-alfa. Gastroenterology 2009;137:2002–2009. 10.1053/j.gastro.2009.08.061  [DOI](https://doi.org/10.1053/j.gastro.2009.08.061) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19737568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Factors%20that%20predict%20response%20of%20patients%20with%20hepatitis%20B%20e%20antigen-positive%20chronic%20hepatitis%20B%20to%20peginterferon-alfa&author=EH%20Buster&author=BE%20Hansen&author=GK%20Lau&author=T%20Piratvisuth&author=S%20Zeuzem&volume=137&publication_year=2009&pages=2002-2009&pmid=19737568&doi=10.1053/j.gastro.2009.08.061&)

31. Mangia A, Santoro R, Mottolo L, Fasano M, Minerva N, Bacca D, et al. Lack of association between IL28B variants and HBsAg clearance after interferon treatment (abstract 1331). J Hepatol 2011;54:S525.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Lack%20of%20association%20between%20IL28B%20variants%20and%20HBsAg%20clearance%20after%20interferon%20treatment%20(abstract%201331)&author=A%20Mangia&author=R%20Santoro&author=L%20Mottolo&author=M%20Fasano&author=N%20Minerva&volume=54&publication_year=2011&pages=S525&)

32. Rao H, Wei L, Lopez-Talavera JC, Shang J, Chen H, Li J, et al. Distribution and clinical correlates of viral and host genotypes in Chinese patients with chronic hepatitis C virus infection. J Gastroenteraol Hepatol 2014; 29:545–553.  [DOI](https://doi.org/10.1111/jgh.12398) | [PMC free article](/articles/PMC4272577/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24090188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gastroenteraol%20Hepatol&title=Distribution%20and%20clinical%20correlates%20of%20viral%20and%20host%20genotypes%20in%20Chinese%20patients%20with%20chronic%20hepatitis%20C%20virus%20infection&author=H%20Rao&author=L%20Wei&author=JC%20Lopez-Talavera&author=J%20Shang&author=H%20Chen&volume=29&publication_year=2014&pages=545-553&pmid=24090188&doi=10.1111/jgh.12398&)

33. Chen Y, Xu HX, Wang LJ, Liu XX, Mahato RI, Zhao YR. Meta-analysis: IL28B polymorphisms predict sustained viral response in HCV patients treated with pegylated interferon-α and ribavirin. Aliment Pharmacol Ther 2012; 36:91–103. 10.1111/j.1365-2036.2012.05131.x  [DOI](https://doi.org/10.1111/j.1365-2036.2012.05131.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22591106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aliment%20Pharmacol%20Ther&title=Meta-analysis:%20IL28B%20polymorphisms%20predict%20sustained%20viral%20response%20in%20HCV%20patients%20treated%20with%20pegylated%20interferon-%CE%B1%20and%20ribavirin&author=Y%20Chen&author=HX%20Xu&author=LJ%20Wang&author=XX%20Liu&author=RI%20Mahato&volume=36&publication_year=2012&pages=91-103&pmid=22591106&doi=10.1111/j.1365-2036.2012.05131.x&)

34. Bota S, Sporea I, Şirli R, Neghină AM, Popescu A, Străin M. Role of interleukin-28B polymorphism as a predictor of sustained virological response in patients with chronic hepatitis C treated with triple therapy: a systematic review and meta-analysis. Clin Drug Investig 2013; 33:325–331. 10.1007/s40261-013-0074-0  [DOI](https://doi.org/10.1007/s40261-013-0074-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23532802/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Drug%20Investig&title=Role%20of%20interleukin-28B%20polymorphism%20as%20a%20predictor%20of%20sustained%20virological%20response%20in%20patients%20with%20chronic%20hepatitis%20C%20treated%20with%20triple%20therapy:%20a%20systematic%20review%20and%20meta-analysis&author=S%20Bota&author=I%20Sporea&author=R%20%C5%9Eirli&author=AM%20Neghin%C4%83&author=A%20Popescu&volume=33&publication_year=2013&pages=325-331&pmid=23532802&doi=10.1007/s40261-013-0074-0&)

35. Zheng H, Li M, Chi B, Wu XX, Wang J, Liu DW. IL28B rs12980275 variant as a predictor of sustained virologic response to pegylated-interferon and ribavirin in chronic hepatitis C patients: A systematic review and meta-analysis. Clin Res Gastroenterol Hepatol 2015; 39:576–583.  [DOI](https://doi.org/10.1016/j.clinre.2015.01.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25769643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Res%20Gastroenterol%20Hepatol&title=IL28B%20rs12980275%20variant%20as%20a%20predictor%20of%20sustained%20virologic%20response%20to%20pegylated-interferon%20and%20ribavirin%20in%20chronic%20hepatitis%20C%20patients:%20A%20systematic%20review%20and%20meta-analysis&author=H%20Zheng&author=M%20Li&author=B%20Chi&author=XX%20Wu&author=J%20Wang&volume=39&publication_year=2015&pages=576-583&pmid=25769643&doi=10.1016/j.clinre.2015.01.009&)

36. Mi Y, Gao YT, Jiao LJ, Guo H, Liu T, Jing L, et al. The role of interleukin-28b gene polymorphisms in chinese patients with chronic hepatitis C treated with pegylated interferon and ribavirin. Hepat Mo 2015;14:e18793.  [DOI](https://doi.org/10.5812/hepatmon.18793) | [PMC free article](/articles/PMC4199147/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25337144/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepat%20Mo&title=The%20role%20of%20interleukin-28b%20gene%20polymorphisms%20in%20chinese%20patients%20with%20chronic%20hepatitis%20C%20treated%20with%20pegylated%20interferon%20and%20ribavirin&author=Y%20Mi&author=YT%20Gao&author=LJ%20Jiao&author=H%20Guo&author=T%20Liu&volume=14&publication_year=2015&pages=e18793&pmid=25337144&doi=10.5812/hepatmon.18793&)

37. Dong ZX, Zhou HJ, Xiang XG, Guo SM, Zhuang Y, Zhao GD, et al. IL28B genetic variations are associated with treatment response of patients with chronic hepatitis C in a Chinese Han population. J Dig Dis. 2015;6:90–97.  [DOI](https://doi.org/10.1111/1751-2980.12202) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25312023/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Dig%20Dis&title=IL28B%20genetic%20variations%20are%20associated%20with%20treatment%20response%20of%20patients%20with%20chronic%20hepatitis%20C%20in%20a%20Chinese%20Han%20population&author=ZX%20Dong&author=HJ%20Zhou&author=XG%20Xiang&author=SM%20Guo&author=Y%20Zhuang&volume=6&publication_year=2015&pages=90-97&pmid=25312023&doi=10.1111/1751-2980.12202&)

38. Shebl FM, Maeder D, Shao Y, Prokunina-Olsson L, Schadt EE, O'Brien TR. In the absence of HCV infection, interferon stimulated gene expression in liver is not associated with IL28B genotype. Gastroenterology 2010;139:1422–1424. 10.1053/j.gastro.2010.07.055  [DOI](https://doi.org/10.1053/j.gastro.2010.07.055) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20800110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=In%20the%20absence%20of%20HCV%20infection,%20interferon%20stimulated%20gene%20expression%20in%20liver%20is%20not%20associated%20with%20IL28B%20genotype&author=FM%20Shebl&author=D%20Maeder&author=Y%20Shao&author=L%20Prokunina-Olsson&author=EE%20Schadt&volume=139&publication_year=2010&pages=1422-1424&pmid=20800110&doi=10.1053/j.gastro.2010.07.055&)

39. Allweiss L, Volz T, Lütgehetmann M, Giersch K, Bornscheuer T, Lohse AW, et al. Immune cell responses are not required to induce substantial hepatitis B virus antigen decline during pegylated interferon-alpha administration. J Hepatol 2014;60:500–507. 10.1016/j.jhep.2013.10.021  [DOI](https://doi.org/10.1016/j.jhep.2013.10.021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24398036/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Immune%20cell%20responses%20are%20not%20required%20to%20induce%20substantial%20hepatitis%20B%20virus%20antigen%20decline%20during%20pegylated%20interferon-alpha%20administration&author=L%20Allweiss&author=T%20Volz&author=M%20L%C3%BCtgehetmann&author=K%20Giersch&author=T%20Bornscheuer&volume=60&publication_year=2014&pages=500-507&pmid=24398036&doi=10.1016/j.jhep.2013.10.021&)

40. Prokunina-Olsson L, Muchmore B, Tang W, Pfeiffer RM, Park H, Dickensheets H, et al. A variant upstream of IFNL3 (IL28B) creating a novel interferon gene IFNL4 is associated with impaired clearance of hepatitis C virus. Nature Genetics 2013; 45:163–171.  [DOI](https://doi.org/10.1038/ng.2521) | [PMC free article](/articles/PMC3793390/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23291588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&title=A%20variant%20upstream%20of%20IFNL3%20(IL28B)%20creating%20a%20novel%20interferon%20gene%20IFNL4%20is%20associated%20with%20impaired%20clearance%20of%20hepatitis%20C%20virus&author=L%20Prokunina-Olsson&author=B%20Muchmore&author=W%20Tang&author=RM%20Pfeiffer&author=H%20Park&volume=45&publication_year=2013&pages=163-171&pmid=23291588&doi=10.1038/ng.2521&)
