# Whole-Genome Sequencing of Pharmacogenetic Drug Response in Racially Diverse Children with Asthma

## Metadata
**Authors:** Angel C Y Mak, Marquitta J White, Walter L Eckalbar, Zachary A Szpiech, Sam S Oh, Maria Pino-Yanes, Donglei Hu, Pagé Goddard, Scott Huntsman, Joshua Galanter, Ann Chen Wu, Blanca E Himes, Soren Germer, Julia M Vogel, Karen L Bunting, Celeste Eng, Sandra Salazar, Kevin L Keys, Jennifer Liberto, Thomas J Nuckton, Thomas A Nguyen, Dara G Torgerson, Pui-Yan Kwok, Albert M Levin, Juan C Celedón, Erick Forno, Hakon Hakonarson, Patrick M Sleiman, Amber Dahlin, Kelan G Tantisira, Scott T Weiss, Denise Serebrisky, Emerita Brigino-Buenaventura, Harold J Farber, Kelley Meade, Michael A Lenoir, Pedro C Avila, Saunak Sen, Shannon M Thyne, William Rodriguez-Cintron, Cheryl A Winkler, Andrés Moreno-Estrada, Karla Sandoval, Jose R Rodriguez-Santana, Rajesh Kumar, L Keoki Williams, Nadav Ahituv, Elad Ziv, Max A Seibold, Robert B Darnell, Noah Zaitlen, Ryan D Hernandez
**Journal:** American Journal of Respiratory and Critical Care Medicine
**Date:** 2018 Jun 15
**DOI:** [10.1164/rccm.201712-2529OC](https://doi.org/10.1164/rccm.201712-2529OC)
**PMID:** 29509491
**PMCID:** PMC6006403
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6006403/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6006403/pdf/rccm.201712-2529OC.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6006403/pdf/rccm.201712-2529OC.pdf)

## Abstract

Rationale: Albuterol, a bronchodilator medication, is the first-line therapy for asthma worldwide. There are significant racial/ethnic differences in albuterol drug response.

Objectives: To identify genetic variants important for bronchodilator drug response (BDR) in racially diverse children.

Methods: We performed the first whole-genome sequencing pharmacogenetics study from 1,441 children with asthma from the tails of the BDR distribution to identify genetic association with BDR.

Measurements and Main Results: We identified population-specific and shared genetic variants associated with BDR, including genome-wide significant (P < 3.53 × 10−7) and suggestive (P < 7.06 × 10−6) loci near genes previously associated with lung capacity (DNAH5), immunity (NFKB1 and PLCB1), and β-adrenergic signaling (ADAMTS3 and COX18). Functional analyses of the BDR-associated SNP in NFKB1 revealed potential regulatory function in bronchial smooth muscle cells. The SNP is also an expression quantitative trait locus for a neighboring gene, SLC39A8. The lack of other asthma study populations with BDR and whole-genome sequencing data on minority children makes it impossible to perform replication of our rare variant associations. Minority underrepresentation also poses significant challenges to identify age-matched and population-matched cohorts of sufficient sample size for replication of our common variant findings.

Conclusions: The lack of minority data, despite a collaboration of eight universities and 13 individual laboratories, highlights the urgent need for a dedicated national effort to prioritize diversity in research. Our study expands the understanding of pharmacogenetic analyses in racially/ethnically diverse populations and advances the foundation for precision medicine in at-risk and understudied minority populations.

Keywords: albuterol, asthma, minority, NFKB1, Latinos

## At a Glance Commentary

## At a Glance Commentary

### Scientific Knowledge on the Subject

Asthma is the most common chronic disease among children. Albuterol, a bronchodilator medication, is the first-line therapy for asthma treatment worldwide. In the United States, asthma prevalence is the highest among Puerto Ricans, intermediate among African Americans and white persons, and lowest in Mexicans. Asthma mortality is fourfold to fivefold higher in Puerto Ricans and African Americans compared with Mexicans. Puerto Ricans and African Americans, the populations with the highest asthma prevalence and mortality, also have the lowest albuterol bronchodilator drug response.

### What This Study Adds to the Field

We conducted the largest pharmacogenetic study using whole-genome sequencing data from 1,441 minority children with asthma who had extremely high or low albuterol bronchodilator drug response. We identified population-specific and shared pharmacogenetic variants associated with bronchodilator drug response. We prioritized variants in *NFKB1* with multiple levels of existing biologic evidence and demonstrated their potential regulatory functions using chromatin immunoprecipitation sequencing, RNA sequencing, and luciferase enhancer assays. Our study reveals the challenges and importance of increasing diversity in research. Our findings help inform the direction of future development of asthma medications and advance the foundation of precision medicine for at-risk, yet understudied, racially/ethnically diverse populations.

## Methods

### Study Cohorts and Sample Details

This study examined a subset of subjects with asthma from SAGE II (Study of African Americans, Asthma, Genes and Environments) ([23](#bib23)) and GALA II (Genes-Environments and Admixture in Latino Americans) studies ([18](#bib18)).

A total of 1,441 individuals from three ethnic groups (483 Puerto Ricans, 483 Mexicans, and 475 African Americans) representing the tails of the BDR distribution were selected for WGS ([Figure 1](#fig1)). Sequencing quality control metrics are summarized in Table E1. Subject selection and filtering processes are described in the online supplement. Descriptive data of study participants are summarized in [Table 1](#tbl1). Detailed descriptions of BDR assessment and analysis are described in the online supplement.

### Table 1.

| Descriptive Statistics | Puerto Ricans (n = 483) |  |  | Mexicans (n = 483) |  |  | African Americans (n = 475) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| High BDR | Low BDR | P Value | High BDR | Low BDR | P Value | High BDR | Low BDR | P Value |  |
| Number of subjects | 239 | 244 | — | 243 | 240 | — | 233 | 242 | — |
| Percent male | 53.6 | 53.3 | 1.0 | 60.1 | 52.1 | 0.08 | 55.4 | 47.9 | 0.12 |
| Median (IQR) age, yr | 11.6 (9.7–14.8) | 12.2 (10.1–15.2) | 0.18 | 11.7 (9.6–14.0) | 13.3 (10.6–16.0) | <0.001 | 13.8 (11.0–16.8) | 13.8 (10.9–17.1) | 0.48 |
| Mean global ancestry proportions |   |   |   |   |   |   |   |   |   |
| AFR | 0.24 | 0.22 | 0.44 | 0.05 | 0.05 | 0.37 | 0.79 | 0.79 | 0.80 |
| EUR | 0.63 | 0.64 | 0.27 | 0.37 | 0.36 | 0.84 | 0.19 | 0.20 | 0.70 |
| NAM | 0.13 | 0.13 | 0.93 | 0.58 | 0.59 | 0.90 | 0.02 | 0.02 | 0.90 |
| BMI category, n |   |   |   |   |   |   |   |   |   |
| Obese, nonobese | 76, 163 | 67, 177 | 0.32 | 100, 143 | 96, 144 | 0.85 | 82, 151 | 83, 159 | 0.85 |
| Pre-FEV1% predicted, n |   |   |   |   |   |   |   |   |   |
| <80%, ≥80% | 149, 90 | 56, 188 | <0.001 | 43, 200 | 7, 233 | <0.001 | 47, 186 | 6, 236 | <0.001 |
| Median (IQR) ΔFEV1, % | 21.2 (18.2–25.7) | 5.0 (2.9–6.3) | — | 12.7 (10.3–16.8) | 3.6 (2.0–4.9) | — | 15.5 (13.3–20.3) | 3.3 (2.0–4.4) | — |
| Median (IQR) tIgE, ml | 407.5 (126.8–952.8) | 191.9 (50.5–542.2) | <0.001 | 247.5 (64.2–817.0) | 105.7 (35.4–332.0) | <0.001 | 281.6 (97.3–552.4) | 128.8 (36.6–351.3) | <0.001 |
| Atopy, n | 177 | 118 | <0.001 | 155 | 117 | <0.001 | 139 | 111 | <0.001 |

Table 1 Caption: Study Population Description (N = 1,441)

### WGS Data Generation, Processing, and Quality Control

Details regarding DNA processing, WGS, variant calling, data quality controls, and variant annotation are described in the online supplement.

### Data Availability

TOPMed WGS data are available to download by submitting a data access request through dbGaP. The dbGaP study accession numbers for GALA II and SAGE II are phs000920.v1.p1 and phs000921.v1.p1, respectively.

### BDR Association Testing and Replication

Single and multivariant testing for BDR association were performed. Logistic regression analysis (high vs. low responder status) was controlled for age, sex, body mass index category, and the first 10 principal components. We conducted population-specific analyses and also performed a transethnic meta-analysis using METASOFT ([24](#bib24)). Multivariant analysis was also performed on individuals combined across all three populations, including local genetic ancestries as additional covariates. The contribution of individual common and rare variants to multivariant association significance was evaluated by excluding common or rare variants in reduced and drop-one analyses. Variation in BDR explained by identified associations was calculated using McFadden pseudo *R*^2^ ([25](#bib25)). Detailed descriptions of these analyses are described in the online supplement.

We were unable to identify another age- and population-matched asthma cohort of sufficient size to replicate our drug response findings. Replication was nevertheless attempted in smaller cohorts and is described in detail in the online supplement.

### Variant Prioritization and Functional Validation

To prioritize BDR-associated variants for further evaluation, an H3K27ac chromatin immunoprecipitation sequencing (ChIP-seq) assay was performed to identify variants overlapping with regulatory regions in primary bronchial smooth muscle cells (BSMCs). The Diverse Convergent Evidence approach ([26](#bib26)) was then used to score BDR-associated variants using multiple levels of observational, bioinformatic, and laboratory evidence. Prioritized variants were further validated by luciferase enhancer assays to verify enhancer activity and by RNA sequencing to identify potential expression quantitative trait locus function. Detailed descriptions of these procedures are described in the online supplement.

## Results

### Descriptive Characteristics of Study Subjects

Descriptive characteristics for all study subjects (*n* = 1,441) are summarized in [Table 1](#tbl1). Covariates and demographic variables were assessed for significant differences between high and low drug responders for each racial/ethnic group. Significant differences were found for age (Mexicans, *P* < 0.001), baseline lung function (pre-FEV_1_% predicted, *P* < 0.001), total IgE (*P* < 0.001), and atopy. Pre-FEV_1_% predicted was defined as the percentage of observed FEV_1_ relative to the expected population average FEV_1_ based on the Hankinson lung function prediction equations ([27](#bib27)). Results and descriptions of genetic ancestry and genetic substructure are shown in [Table 1](#tbl1), supplemental text E1, and Figures E2 and E3.

### BDR Association Testing with Common Variants

Detailed descriptions of variant summary statistics are described in supplemental text E2, Figure E4, and Tables E2 and E3. Study design is described in Figure E5. All subsequent analyses in this study were performed only with biallelic SNPs. Throughout this study, low drug responders were assigned as the reference (i.e., control) group, and high drug responders were classified as cases. We performed genome-wide association testing of common variants with BDR (dichotomized as high/low drug responders) for each population, adjusting by age, sex, body mass index category, and the first 10 principal components. We then performed a transethnic meta-analysis across all three populations.

The commonly used GWAS *P* value threshold of 5 × 10^−8^ is derived from a Bonferroni correction under the assumption of 1,000,000 independent tests, based primarily on patterns of linkage disequilibrium (LD) from individuals of European descent. This threshold has been shown to be nongeneralizable for genetic studies among populations of non-European descent ([28](#bib28), [29](#bib29)). We therefore empirically calculated the effective number of independent tests for each population and for our transethnic meta-analysis. The resulting genome-wide significance thresholds were 1.57 × 10^−7^ for Puerto Ricans, 2.42 × 10^−7^ for Mexicans, 9.59 × 10^−8^ for African Americans, and 3.53 × 10^−7^ for the transethnic meta-analysis. These numbers are highly concordant with WGS significance thresholds derived from the 1,000 Genomes sequencing data ([28](#bib28)). Significance thresholds for discovery analyses in GWASs can often produce false-negative results. To minimize type II error, suggestive associations are often included in replication and functional validation studies. We identified suggestive associations based on the following widely used formula: 1/(effective number of tests) ([30](#bib30)).

Although no significant associations were identified from the population-specific analyses (*see* Figure E6), our transethnic meta-analysis identified 10 unique loci (represented by 27 SNPs) significantly (*P* < 3.53 × 10^−7^) or suggestively (*P* < 7.06 × 10^−6^) associated with BDR status ([Figure 2A](#fig2) and [Table 2](#tbl2); *see* Table E4). After LD pruning, these 27 SNPs explain 23%, 16%, and 18% of the variation in BDR status in Puerto Ricans, Mexicans, and African Americans, respectively (*see* Table E5). To demonstrate that the results of our regression models were robust, a *post hoc* analysis was performed on the 27 SNPs by including Native American and African local ancestries as covariates. The association results before and after adjusting for local ancestry remained consistent (*see* Text E3, Table E6). We annotated all 27 SNPs by performing a thorough bioinformatic search in ENCODE, the NHGRI-EBL GWAS Catalog, and PubMed. Their previously reported lung-related phenotype associations and functional annotations are reported in Tables E7 and E8.

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cda4/6006403/3289df2894c0/rccm.201712-2529OC_f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6006403_rccm.201712-2529OC_f2.jpg)

(A) Manhattan plot of the transethnic meta-analysis of single locus bronchodilator drug response association testing. Top 10 bronchodilator drug response–associated loci are circled. The black line indicates the universal genome-wide significance threshold (5.00 × 10−8), the red line indicates the adjusted genome-wide significance threshold (3.53 × 10−7), and the blue line indicates the suggestive significance threshold (7.06 × 10−6). (B) Forest plot of the two most significantly associated SNPs, rs17834628 and rs35661809. The R2 between these two SNPs is 0.93 in Puerto Ricans, 0.96 in Mexicans, and 0.66 in African Americans. (C) The most significantly associated SNP (rs17834628) is plotted with 400-kb flanking regions on either side. Dot color shows each SNP linkage disequilibrium with rs17834628 based on the 1,000 Genomes November 2014 admixed American population. Multiple SNPs in high linkage disequilibrium (R2 > 0.8, red) reached a suggestive significance level.

### Table 2.

| Chr | Start | rsID | Effect Allele | OR (95% CI) | P Value | Effect Allele Frequency |  |  | Nearest Genes |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| PR | MX | AA |  |  |  |  |  |  |  |
| 5 | 12978566 | rs17834628 | A | 1.67 (1.29–2.16) | 1.18 × 10−8* | 0.32 | 0.42 | 0.17 | LINC01194 (−173 kb), MIR4454 (−311 kb), CTNND2 (−1,074kb), DNAH5 (712 kb) |
| 5 | 12968341 | rs35661809 | G | 1.59 (1.20–2.10) | 3.33 × 10−8* | 0.34 | 0.43 | 0.24 | LINC01194 (−163 kb), MIR4454 (−300 kb), CTNND2 (−1,064kb), DNAH5 (722 kb) |
| 5 | 12975934 | rs17237639 | G | 1.61 (1.30–2.00) | 1.22 × 10−7 | 0.31 | 0.43 | 0.16 | LINC01194 (−171 kb), MIR4454 (−308 kb), CTNND2 (−1,072kb), DNAH5 (715 kb) |
| 5 | 12975187 | rs1017452 | G | 1.60 (1.31–1.96) | 2.11 × 10−7 | 0.31 | 0.43 | 0.16 | LINC01194 (−170 kb), MIR4454 (−307 kb), CTNND2 (−1,071kb), DNAH5 (715 kb) |
| 5 | 12975322 | rs1017454 | A | 1.60 (1.31–1.96) | 2.11 × 10−7 | 0.31 | 0.43 | 0.16 | LINC01194 (−170 kb), MIR4454 (−307 kb), CTNND2 (−1,071kb), DNAH5 (715 kb) |
| 5 | 12975265 | rs1017453 | C | 1.56 (1.25–1.95) | 6.40 × 10−7 | 0.31 | 0.42 | 0.16 | LINC01194 (−170 kb), MIR4454 (−307 kb), CTNND2 (−1,071kb), DNAH5 (715 kb) |
| 5 | 12972636 | rs17237443 | C | 1.59 (1.28–1.97) | 9.85 × 10−7 | 0.29 | 0.42 | 0.11 | LINC01194 (−170 kb), MIR4454 (−307 kb), CTNND2 (−1,071kb), DNAH5 (715 kb) |
| 1 | 209324294 | rs10746419 | T | 1.29 (0.75–2.25) | 1.19 × 10−6 | 0.49 | 0.54 | 0.53 | MIR205HG (278 kb), MIR205 (281 kb), CAMK1G (433 kb), LAMB3 (464 kb) |
| 5 | 12961545 | rs17833938 | A | 1.56 (1.28–1.91) | 1.45 × 10−6 | 0.30 | 0.42 | 0.12 | LINC01194 (−156 kb), MIR4454 (−294 kb), CTNND2 (−1,057kb), DNAH5 (729 kb) |
| 6 | 104240500 | rs13437006 | C | 1.56 (1.21–2.02) | 1.61 × 10−6 | 0.22 | 0.24 | 0.32 | HACE1 (935 kb), LINC00577 (1,144kb), LIN28B (1,164kb) |
| 15 | 101230457 | rs1565749 | A | 1.66 (1.18–2.32) | 1.64 × 10−6 | 0.18 | 0.15 | 0.18 | ASB7 (−39 kb), LINS1 (−88 kb), PRKXP1 (−131 kb) |
| 5 | 12948369 | rs34845041 | T | 1.56 (1.26–1.92) | 1.77 × 10−6 | 0.30 | 0.42 | 0.12 | LINC01194 (−143 kb), MIR4454 (−280 kb), CTNND2 (−1,044kb), DNAH5 (742 kb) |
| 5 | 12975108 | rs1017451 | T | 1.55 (1.24–1.93) | 1.96 × 10−6 | 0.30 | 0.42 | 0.13 | LINC01194 (−170 kb), MIR4454 (−307 kb), CTNND2 (−1,071kb), DNAH5 (715 kb) |
| 5 | 12950432 | rs62347395 | G | 1.55 (1.26–1.92) | 2.02 × 10−6 | 0.30 | 0.42 | 0.12 | LINC01194 (−145 kb), MIR4454 (−282 kb), CTNND2 (−1,046kb), DNAH5 (740 kb) |
| 15 | 101231049 | rs57924834 | A | 1.59 (1.25–2.03) | 2.04 × 10−6 | 0.23 | 0.20 | 0.20 | ASB7 (−39 kb), LINS1 (−89 kb), PRKXP1 (−132 kb) |
| 4 | 137382142 | rs17048684 | A | 1.8 (1.06–3.05) | 2.20 × 10−6 | 0.11 | 0.14 | 0.18 | LINC00613 (−547 kb), PCDH18 (1,058kb) |
| 5 | 12959598 | rs1438293 | G | 1.55 (1.24–1.93) | 2.73 × 10−6 | 0.29 | 0.42 | 0.11 | LINC01194 (−154 kb), MIR4454 (−292 kb), CTNND2 (−1,055kb), DNAH5 (731 kb) |
| 20 | 8635168 | rs16995064 | G | 1.96 (1.12–3.43) | 3.30 × 10−6 | 0.12 | 0.13 | 0.05 | PLCB1 (intron 7), PLCB4 (415 kb) |
| 12 | 19821401 | rs66544720 | T | 0.66 (0.55–0.78) | 3.66 × 10−6 | 0.33 | 0.37 | 0.16 | AEBP2 (−146 kb), PLEKHA5 (−292 kb) |
| 6 | 104235591 | rs6926020 | C | 1.57 (1.25–1.97) | 3.68 × 10−6 | 0.19 | 0.23 | 0.27 | HACE1 (940 kb), LINC00577 (1,149kb), LIN28B (1,169kb) |
| 4 | 103453535 | rs28450894 | T | 0.47 (0.34–0.64) | 3.75 × 10−6 | 0.06 | 0.03 | 0.12 | SLC39A8 (−187 kb), NFKB1 (intron 3), MANBA (99 kb) |
| 4 | 103461559 | rs4648006 | T | 0.47 (0.34–0.64) | 3.75 × 10−6 | 0.06 | 0.03 | 0.12 | SLC39A8 (−195 kb), NFKB1 (intron 5), MANBA (91 kb) |
| 22 | 27826429 | rs60163793 | G | 2.01 (1.20–3.38) | 4.30 × 10−6 | 0.04 | 0.14 | 0.15 | MN1 (318 kb), PITPNB (421 kb) |
| 12 | 19824386 | rs7313907 | C | 0.66 (0.55–0.79) | 4.35 × 10−6 | 0.33 | 0.37 | 0.16 | AEBP2 (−149 kb), PLEKHA5 (−295 kb) |
| 12 | 19820677 | rs11044754 | A | 0.66 (0.55–0.79) | 4.54 × 10−6 | 0.33 | 0.37 | 0.16 | AEBP2 (−146 kb), PLEKHA5 (−291 kb) |
| 15 | 101233236 | rs55638658 | A | 1.61 (1.13–2.30) | 5.08 × 10−6 | 0.18 | 0.15 | 0.18 | ASB7 (−41 kb), LINS1 (−91 kb), PRKXP1 (−134 kb) |
| 6 | 54581204 | rs13200833 | A | 0.66 (0.48–0.90) | 5.15 × 10−6 | 0.32 | 0.24 | 0.22 | TINAG (−326 kb), MLIP (−450 kb), FAM83B (130 kb) |

Table 2 Caption: Results from Transethnic BDR Association Tests for Common Variants

Two SNPs located on chromosome 5 (rs17834628 and rs35661809) were significantly associated with BDR (*P* = 1.18 × 10^−8^ and 3.33 × 10^−8^). The direction of effect for these two variants is concordant across all three populations ([Figure 2B](#fig2); *see* Table E9). [Figure 2C](#fig2) displays a LocusZoom plot of rs17834628 with 400-kb flanking regions. Three of the 27 SNPs were located within genes. Specifically, two SNPs are located in the third and fifth introns of *NFKB1* (rs28450894 and rs4648006), and a third SNP, rs16995064, mapped to intron 7 of *PLCB1* ([Table 2](#tbl2)). Based on 1,000 Genomes data, the low BDR-associated T allele of *NFKB1* (rs28450894) is found predominantly among African populations (minor allele frequency [MAF], 8.8–28.7%), followed by European populations (MAF, 3.7–7.6%) and Puerto Ricans (MAF, 6.2%), and rare in Mexicans (MAF, 1.5%) (*see* Figure E7).

We were unable to identify age- and population-matched asthma cohorts with albuterol drug response data of sufficient sample size for replication. Nevertheless, we attempted to replicate all 27 SNPs in five independent populations (GALA I, SAGE I, HPR, SAPPHIRE, and CHOP) separately and by meta-analysis (*see* Tables E10 and E11). Although none of the 27 SNPs were significantly associated with BDR status in our replication analyses (*see* Table E12), it is important to note that age-specific associations with asthma and asthma-related phenotypes have been reported ([31](#bib31), [32](#bib32)), and that children were not included in our largest replication cohort (SAPPHIRE, median age >28) (*see* Table E10). All other replication populations were smaller than our discovery populations (ranging from 108 to 414 individuals per study), and some had an imbalance of cases and control subjects (*see* Table E10), which further diminished power for replication analyses.

Albuterol binds to β_2_-adrenergic receptors in BSMC, causing rapid onset of airway tissue relaxation and bronchodilation. BSMCs are therefore considered one of the most relevant cell types for molecular studies of BDR ([33](#bib33)). We performed H3K27ac ChIP-seq experiments in primary BSMCs to identify potential regulatory regions marked by H3K27ac peaks. We then overlapped the H3K27ac peaks with the 27 BDR-associated SNPs and SNPs that were in high LD with them (*R*^2^ ≥ 0.8 within 1 Mb in any of our study populations). Variants overlapping with H3K27ac peaks may imply regulatory functions in BSMC (*see* Table E8).

We applied the Diverse Convergent Evidence ([26](#bib26)) approach to prioritize the 27 BDR-associated SNPs for inclusion in further functional analyses (*see* Table E12). After integrating information from our WGS analysis, publicly available bioinformatics data, and ChIP-seq experiments in BSMCs, the *NFKB1* locus had the highest Diverse Convergent Evidence score, indicating that *NFKB1* had the strongest evidence of functional relevance to BDR variation (*see* Figure E8). Therefore, all further functional experiments were focused on variants within this locus.

### NFKB1 Functional Assays

Two H3K27ac ChIP-seq regions that overlapped with the BDR-associated *NFKB1* locus were tested for enhancer activity using luciferase enhancer assays in BSMCs (*see* Figure E9A and Table E13). One enhancer, NFKB1 Region 2, showed significantly increased enhancer activity over empty vector (log_2_ 2.24-fold increase, *P* = 1.59 × 10^−5^, unpaired Wilcoxon test) (*see* Figure E9B).

Given the relevance of *NFKB1* in immune pathways and asthma, we also performed RNA sequencing experiments in African American children with asthma to verify whether the identified intronic *NFKB1* SNPs regulate expression of neighboring genes. Among genes within 1 Mb of rs28450894 meeting expression reliability cutoffs, we found that the low BDR-associated T allele of rs28450894 is significantly associated with decreased expression of *SLC39A8* in blood (*see* Figure E10) [*P* = 0.0066, FDR-adjusted *P* = 0.0856, log_2_(β) = −0.327].

### Comparison with Previous BDR Association

We observed that two known BDR candidate genes, *ADCY9* and *CRHR2*, which replicated in a previous BDR GWAS performed in the full GALA II population ([18](#bib18)), did not replicate in the current study (*see* Table E14). In that study, imputed GWAS array data were used to evaluate genetic associations with BDR measured as a continuous trait. To determine whether the discrepancy between findings was caused by data type (imputed array-based vs. WGS-based) or study design (continuous trait vs. extreme phenotype), the common variant analysis in the current analysis was repeated among the subset of samples with array-based and WGS data (*n* = 1,414 out of 1,441). Based on the top 1,000 BDR-associated SNPs from the current common variant analysis, there was perfect correlation between association *P* values generated from imputed array-based and WGS-based genotypes (Spearman correlation, 1.0), suggesting that data type is not the cause of the observed discrepancy (*see* Figure E11A). Nearly all SNPs with high imputation *R*^2^ exhibited high genotype concordance between array-based and WGS-based genotypes, confirming high imputation quality for most common SNPs (≥99.7%; *see* Figures E11B and E11C). We also performed linear regression to analyze BDR as a continuous trait (ΔFEV_1_) using imputed array-based data. The most significantly associated SNP identified in the current extreme phenotype analysis displayed the same direction of effect as analyzing BDR as a continuous trait (odds ratio, 1.67 in extreme phenotype analysis; β = 0.51 in continuous analysis). These observations indicate that the discrepancy between findings may be caused by differences in statistical power afforded by the different study designs (continuous trait vs. extreme phenotype). For common variant analyses, dichotomization of a continuous outcome results in a loss of statistical power ([34](#bib34)). The opposite effect is observed in rare variant analyses. The extreme phenotype study design has been shown to increase power and the probability of identifying functional rare variants ([35](#bib35), [36](#bib36)). Also noteworthy is that the previously published results were discovered in one population (Puerto Ricans), whereas the results from our transethnic meta-analysis describe associations that are conserved across three populations (Puerto Ricans, Mexicans, and African Americans).

### BDR Association Testing Using Common and Rare Variants

We tested the combined effects of common and rare variants on BDR using SKAT-O ([37](#bib37)) to examine variants in 1-kb sliding windows with 500-bp increments. The same covariates used for common variant association testing were applied.

After determining the effective number of tests and adjusting for multiple comparisons on each population separately, we identified three population-specific loci associated with BDR at genome-wide significance levels; two were found in Mexicans on chromosome 1 and chromosome 11, and one in African Americans on chromosome 19 ([Figures 3A–3C](#fig3) and [Table 3](#tbl3); *see* Table E15).

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cda4/6006403/44e1b2a8a6ad/rccm.201712-2529OC_f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6006403_rccm.201712-2529OC_f3.jpg)

Manhattan plot of SKAT-O analysis of biallelic common and rare SNPs grouped by 1-kb windows sliding across chromosome 1–22 in (A) Puerto Ricans, (B) Mexicans, (C) African Americans, and (D) all populations combined. Bonferroni-corrected genome-wide and suggestive significance levels are marked by red and blue lines, respectively.

### Table 3.

| Chr | Start | Stop | P Value | Population | nCommon | nRare | Nearest Genes |
| --- | --- | --- | --- | --- | --- | --- | --- |
| 1 | 114177000 | 114178000 | 4.40 × 10−9 | MX | 2 | 1 | MAGI3 (intron 9), PHTF1 (62 kb), RSBN1 (126 kb) |
| 11 | 27507000 | 27508000 | 6.59 × 10−9 | MX | 2 | 3 | LOC105376671 (−3 kb), LGR4 (−13 kb), LIN7C (8 kb) |
| 19 | 10424000 | 10425000 | 3.12 × 10−11 | AA | 1 | 2 | ZGLP1 (−4 kb), ICAM5 (−17 kb), FDX1L (intron 3), RAVER1 (2 kb) |
| 4 | 73478000 | 73479000 | 6.25 × 10−8 | Combined | 10 | 23 | ADAMTS3 (−43 kb), COX18 (441 kb) |
| 8 | 97926000 | 97927000 | 1.32 × 10−7 | Combined | 3 | 13 | SDC2 (−302 kb), CPQ (intron 4), LOC101927066 (37 kb), TSPYL5 (359 kb) |

Table 3 Caption: Association Testing for Combined Effects of Common and Rare Variants on BDR

We also performed association testing across all three populations in a single analysis to maximize power. To minimize confounding by population substructure, association testing also included local genetic ancestry, defined as the proportions of Native American and African ancestries for the window under testing. Two loci on chromosomes 4 and 8 attained genome-wide significance (*P* < 1.53 × 10^−7^) ([Figure 3D](#fig3) and [Table 3](#tbl3)). Sixty variants were identified from all SKAT-O regions reported in [Table 3](#tbl3). Six of the 60 variants were located within predicted regulatory regions (*see* Table E16). Three variants on chromosome 11 identified in Mexicans overlap with a CTCF (transcriptional repressor) binding site and comprise a chromatin insulator region. The five regions identified in combined and population-specific SKAT-O analyses independently explained 4–8% of BDR variation in their respective populations (*see* Table E5).

To investigate whether common and rare variants both contributed to the BDR association *P* value, we performed reduced and drop-one SKAT analyses by excluding common or rare variants in the associated region one by one. Because excluding either all common or all rare variants would reduce the significance of the BDR association *P* value (*see* Table E17), the reduced (*see* Table E18) and drop-one (*see* Table E19) analyses indicated that both common and rare variants contribute to the significance of the BDR association.

Although we believe the sliding window method is the most appropriate approach for WGS data, we considered alternative grouping strategies for rare variants, including grouping by *1*) genes from transcription start sites to transcription end sites, with or without 50-kb flanking regions; *2*) transcription start sites with 20-kb flanking regions; and *3*) H3K27ac ChIP-seq peaks from airway epithelial cells and airway smooth muscle cells. Association tests with these grouping strategies identified no further significant associations. Because these analyses were performed *post hoc*, multiple testing correction for testing alternative grouping strategies was not applied to the results of the sliding window approach.

## Discussion

We identified population-specific and shared common and rare variants associated with BDR in three racially and ethnically diverse populations of children with asthma. WGS provides comprehensive detection of common and rare variants in coding and noncoding regions. Combined, the 27 variants (after LD pruning) identified from our common variant analyses ([Table 2](#tbl2)) explained 23%, 16%, and 18% of the variation in BDR in Puerto Ricans, Mexicans, and African Americans, respectively (*see* Table E5). The five SKAT-O regions identified in our combined and population-specific analyses independently explained 4–8% of BDR variation ([Tables 3](#tbl3); *see* E5). Our study represents an important investment from the NIH/NHLBI to improve racial and ethnic diversity in clinical and biomedical research.

Our transethnic common variant meta-analysis identified one locus on chromosome 5 that was associated with BDR at a genome-wide significance level (*P* < 5 × 10^−8^). The proximity of this BDR-associated locus to *DNAH5* and *LINC01194* is of particular interest. An SNP in *DNAH5* has been associated with total lung capacity in white subjects with chronic obstructive pulmonary disease ([38](#bib38)). In a separate GWAS, the *DNAH5*/*LINC01194* locus was reported among Europeans to be associated with levels of IgE ([39](#bib39), [40](#bib40)), a biomarker associated with asthma endotypes. Baseline lung function (FEV_1_) and total IgE levels are associated with asthma severity and can predispose an individual to lower BDR ([7](#bib7), [8](#bib8), [41](#bib41)). We found two *NFKB1* intronic variants on chromosome 4 that were suggestively associated with BDR. The nuclear factor-κB protein has a known role in allergic response, and various studies have demonstrated that the nuclear factor-κB pathway is activated in patients with asthma, as reviewed by Edwards and colleagues ([42](#bib42)).

ChIP-seq and functional enhancer assays in BSMCs imply that the regions containing the NFKB1 intronic variants regulate expression of nearby genes, but do not directly suggest that the genetic variants themselves alter the expression of nearby genes. However, the latter was supported by our RNA sequencing data, which showed that individuals with the low BDR-associated T allele displayed reduced expression of the neighboring SLC39A8 gene. Additionally, SLC39A8 has been found to be responsive to cytokine treatment in airway epithelial cells ([42](#bib42)) and had reduced expression in mice with allergic airway inflammation ([43](#bib43)).

Studies have shown that up-regulation of *SLC39A8* is sufficient to protect the lung epithelium against tumor necrosis factor-α–induced cytotoxicity ([44](#bib44)). Additionally, the higher frequency of the low BDR-associated allele (T allele of rs28450894 in *NFKB1*) in African populations suggests that the low BDR-associated allele tracks with African ancestry. This may explain why admixed populations with higher proportions of African ancestry (e.g., African Americans and Puerto Ricans) have lower BDR ([8](#bib8)), and by extension may shed light on the higher asthma morbidity and mortality in these populations.

Another intronic variant (chromosome 20, rs16995064, *PLCB1* intron 7) suggestively associated with BDR is relevant to childhood asthma. *PLCB1* is differentially expressed among children with therapy-resistant asthma versus controlled persistent asthma or age-matched healthy control subjects ([45](#bib45)). Silencing *PLCB1* inhibited the effect of lipopolysaccharide-induced endothelial cell inflammation by inhibiting expression of proinflammatory cytokines ([46](#bib46)). Additional functional studies are necessary to establish the role of *NFKB1* and *PLCB1* on BDR.

We identified various combined effects of rare variants that were population-specific or shared across populations. Although some nearest genes show no known functional relationship to BDR (*MAGI3*, *LOC105376671*, *LIN7C*, and *CPQ*), the locus between *ADAMTS3* and *COX18* may be functionally relevant. The *ADAMTS3*/*COX18* locus was associated with β-adrenergic responses in murine cardiovascular-related traits ([47](#bib47)). This locus was significantly associated with cardiac atrial weight in mice treated with the β-blocker atenolol and replicated under β-agonist isoproterenol treatment. These findings suggest that SNPs in this locus may modify β-adrenergic signaling pathways in BDR. In the present study, we also identified BDR association with rare variants within *CPQ*, which encodes a protein from the carboxypeptidase family. Although no previous BDR association has been identified for *CPQ*, another member of the carboxypeptidase family, carboxypeptidase A3, is expressed at higher levels in the airway epithelium among subjects with Th2-high asthma ([48](#bib48), [49](#bib49)). Further studies are necessary to determine the role of *CPQ* in BDR.

GWAS-based BDR-associated common variants in GALA II have previously been reported ([18](#bib18)). However, those variants were not significantly associated with extreme BDR in the current study, likely because of differences in study design. The previous BDR GWAS used an array-based genotyping panel to examine children with asthma from the entire BDR spectrum. In contrast, our current study sequenced the entire genome to investigate only the extremes of the BDR distribution. By repeating our current extreme phenotype analysis using a subset of individuals who had array and WGS data, we confirmed that the major discrepancy between the two studies was caused by study design instead of differences in data type. The contrast in results between GWAS and WGS caused by differences in study design implies that varied study designs are necessary for a comprehensive understanding of variants associated with asthma-related phenotypes and drug response. The extreme phenotype approach is recognized as one of the success factors in the study design of pharmacogenomic GWASs ([50](#bib50)). Furthermore, the power gain from studying extreme phenotypes is much greater in analyses of rare variants compared with common variant studies ([51](#bib51)). An extreme phenotype study design provides a cost-effective means for studying common and rare variant associations that may otherwise be missed when sampling across the entire phenotypic spectrum.

We did not identify BDR-associated variants from β_2_-adrenergic receptors signaling pathways. Instead, most of the BDR-associated genes identified in this study are related to lung function and allergic response, including total IgE levels and cytokine production in mast cells. This suggests that BDR depends in part on the intrinsic state of airway smooth muscle cells. Genetic variation may determine individuals’ intrinsic expression levels of candidate genes, which in turn determine whether their response to albuterol is beneficial.

Including admixed populations in WGS studies has important scientific implications. First, it facilitates discovery of genetic variation of multiple ancestral populations in a single study. Second, studying multiethnic admixed populations with ancestries that are underrepresented in existing genetic repositories can yield novel pharmacogenomic insights. For example, the widely popular PCSK9 inhibitors used to treat hypercholesterolemia were discovered by studying the genetics of African Americans but the drug development eventually benefited patients of all ethnicities ([52](#bib52)). Finally, studying admixed populations, such as Mexicans, enhances the understanding of genetic variation in Native American ancestry, a population largely absent in major sequencing efforts.

We and others have documented the implications and challenges posed by the lack of non-European study populations in biomedical research ([53](#bib53)–[55](#bib55)). We made extensive efforts to test our top BDR-associated variants in other populations, but the unique characteristics of our discovery cohort (minority children with asthma who have BDR and WGS data) posed significant challenges for finding comparable replication cohorts. Testing ancestry-dependent pharmacogenetic variants without age- and ethnicity-matched replication populations poses an impossible statistical bar for publication. We therefore conducted additional analyses involving bioinformatics and experimental assays ([56](#bib56)). These challenges highlight the need to include more racially/ethnically diverse populations in all clinical and biomedical research.

In an era of precision medicine, addressing questions about the impact of genetic factors on therapeutic drug response in globally diverse populations is essential for making precision medicine socially and scientifically precise ([57](#bib57)). This study advances the understanding of genetic analysis in admixed populations and helps to lay the foundation of precision medicine for understudied and racially and ethnically diverse populations.

## Acknowledgments

### Acknowledgment

The authors thank the New York Genome Center investigators and teams for whole-genome sequencing sample preparation, quality control, data generation, data processing, and initial joint genotyping. They gratefully acknowledge the studies and participants who provided biologic samples and data for TOPMed. They also gratefully acknowledge the contributions of the investigators of the NHLBI TOPMed Consortium ([https://www.nhlbiwgs.org/topmed-banner-authorship](https://www.nhlbiwgs.org/topmed-banner-authorship)).

## Footnotes

## Contributor Information

Collaborators: on behalf of the NHLBI Trans-Omics for Precision Medicine (TOPMed) Consortium

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

TOPMed WGS data are available to download by submitting a data access request through dbGaP. The dbGaP study accession numbers for GALA II and SAGE II are phs000920.v1.p1 and phs000921.v1.p1, respectively.

### Data Availability Statement

TOPMed WGS data are available to download by submitting a data access request through dbGaP. The dbGaP study accession numbers for GALA II and SAGE II are phs000920.v1.p1 and phs000921.v1.p1, respectively.

## References

1. World Health Organization. Asthma. 2017 [accessed 2017 Sep 12]. Available from: http://www.who.int/mediacentre/factsheets/fs307/en/  [http://www.who.int/mediacentre/factsheets/fs307/en/](http://www.who.int/mediacentre/factsheets/fs307/en/)

2. Barr RG, Aviles-Santa L, Davis SM, Aldrich TK, Gonzalez F, 2nd, Henderson AG, et al. Pulmonary disease and age at immigration among Hispanics: results from the Hispanic Community Health Study/Study of Latinos. Am J Respir Crit Care Med. 2016;193:386–395. doi: 10.1164/rccm.201506-1211OC.  [DOI](https://doi.org/10.1164/rccm.201506-1211OC) | [PMC free article](/articles/PMC4803083/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26451874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Pulmonary%20disease%20and%20age%20at%20immigration%20among%20Hispanics:%20results%20from%20the%20Hispanic%20Community%20Health%20Study/Study%20of%20Latinos&author=RG%20Barr&author=L%20Aviles-Santa&author=SM%20Davis&author=TK%20Aldrich&author=F%20Gonzalez&volume=193&publication_year=2016&pages=386-395&pmid=26451874&doi=10.1164/rccm.201506-1211OC&)

3. Akinbami L. Asthma prevalence, health care use and mortality: United States, 2003-05. 2015 [accessed 2017 Sep 12]. Available from: http://www.cdc.gov/nchs/data/hestat/asthma03-05/asthma03-05.htm.  [http://www.cdc.gov/nchs/data/hestat/asthma03-05/asthma03-05.htm](http://www.cdc.gov/nchs/data/hestat/asthma03-05/asthma03-05.htm) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21355352/)

4. Palmer LJ, Silverman ES, Weiss ST, Drazen JM. Pharmacogenetics of asthma. Am J Respir Crit Care Med. 2002;165:861–866. doi: 10.1164/ajrccm.165.7.2109096.  [DOI](https://doi.org/10.1164/ajrccm.165.7.2109096) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11934710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Pharmacogenetics%20of%20asthma&author=LJ%20Palmer&author=ES%20Silverman&author=ST%20Weiss&author=JM%20Drazen&volume=165&publication_year=2002&pages=861-866&pmid=11934710&doi=10.1164/ajrccm.165.7.2109096&)

5. Eggleston PA, Malveaux FJ, Butz AM, Huss K, Thompson L, Kolodner K, et al. Medications used by children with asthma living in the inner city. Pediatrics. 1998;101:349–354. doi: 10.1542/peds.101.3.349.  [DOI](https://doi.org/10.1542/peds.101.3.349) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9480996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatrics&title=Medications%20used%20by%20children%20with%20asthma%20living%20in%20the%20inner%20city&author=PA%20Eggleston&author=FJ%20Malveaux&author=AM%20Butz&author=K%20Huss&author=L%20Thompson&volume=101&publication_year=1998&pages=349-354&pmid=9480996&doi=10.1542/peds.101.3.349&)

6. Finkelstein JA, Lozano P, Farber HJ, Miroshnik I, Lieu TA. Underuse of controller medications among Medicaid-insured children with asthma. Arch Pediatr Adolesc Med. 2002;156:562–567. doi: 10.1001/archpedi.156.6.562.  [DOI](https://doi.org/10.1001/archpedi.156.6.562) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12038888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Pediatr%20Adolesc%20Med&title=Underuse%20of%20controller%20medications%20among%20Medicaid-insured%20children%20with%20asthma&author=JA%20Finkelstein&author=P%20Lozano&author=HJ%20Farber&author=I%20Miroshnik&author=TA%20Lieu&volume=156&publication_year=2002&pages=562-567&pmid=12038888&doi=10.1001/archpedi.156.6.562&)

7. Burchard EG, Avila PC, Nazario S, Casal J, Torres A, Rodriguez-Santana JR, et al. Genetics of Asthma in Latino Americans (GALA) Study. Lower bronchodilator responsiveness in Puerto Rican than in Mexican subjects with asthma. Am J Respir Crit Care Med. 2004;169:386–392. doi: 10.1164/rccm.200309-1293OC.  [DOI](https://doi.org/10.1164/rccm.200309-1293OC) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14617512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Lower%20bronchodilator%20responsiveness%20in%20Puerto%20Rican%20than%20in%20Mexican%20subjects%20with%20asthma&author=EG%20Burchard&author=PC%20Avila&author=S%20Nazario&author=J%20Casal&author=A%20Torres&volume=169&publication_year=2004&pages=386-392&pmid=14617512&doi=10.1164/rccm.200309-1293OC&)

8. Naqvi M, Thyne S, Choudhry S, Tsai HJ, Navarro D, Castro RA, et al. Ethnic-specific differences in bronchodilator responsiveness among African Americans, Puerto Ricans, and Mexicans with asthma. J Asthma. 2007;44:639–648. doi: 10.1080/02770900701554441.  [DOI](https://doi.org/10.1080/02770900701554441) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17943575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Asthma&title=Ethnic-specific%20differences%20in%20bronchodilator%20responsiveness%20among%20African%20Americans,%20Puerto%20Ricans,%20and%20Mexicans%20with%20asthma&author=M%20Naqvi&author=S%20Thyne&author=S%20Choudhry&author=HJ%20Tsai&author=D%20Navarro&volume=44&publication_year=2007&pages=639-648&pmid=17943575&doi=10.1080/02770900701554441&)

9. Wechsler ME, Castro M, Lehman E, Chinchilli VM, Sutherland ER, Denlinger L, et al. NHLBI Asthma Clinical Research Network. Impact of race on asthma treatment failures in the asthma clinical research network. Am J Respir Crit Care Med. 2011;184:1247–1253. doi: 10.1164/rccm.201103-0514OC.  [DOI](https://doi.org/10.1164/rccm.201103-0514OC) | [PMC free article](/articles/PMC3361331/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21885625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Impact%20of%20race%20on%20asthma%20treatment%20failures%20in%20the%20asthma%20clinical%20research%20network&author=ME%20Wechsler&author=M%20Castro&author=E%20Lehman&author=VM%20Chinchilli&author=ER%20Sutherland&volume=184&publication_year=2011&pages=1247-1253&pmid=21885625&doi=10.1164/rccm.201103-0514OC&)

10. Nelson HS, Weiss ST, Bleecker ER, Yancey SW, Dorinsky PM SMART Study Group. The Salmeterol Multicenter Asthma Research Trial: a comparison of usual pharmacotherapy for asthma or usual pharmacotherapy plus salmeterol. Chest. 2006;129:15–26. doi: 10.1378/chest.129.1.15.  [DOI](https://doi.org/10.1378/chest.129.1.15) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16424409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=The%20Salmeterol%20Multicenter%20Asthma%20Research%20Trial:%20a%20comparison%20of%20usual%20pharmacotherapy%20for%20asthma%20or%20usual%20pharmacotherapy%20plus%20salmeterol&author=HS%20Nelson&author=ST%20Weiss&author=ER%20Bleecker&author=SW%20Yancey&author=PM%20Dorinsky&volume=129&publication_year=2006&pages=15-26&pmid=16424409&doi=10.1378/chest.129.1.15&)

11. McGeachie MJ, Stahl EA, Himes BE, Pendergrass SA, Lima JJ, Irvin CG, et al. Polygenic heritability estimates in pharmacogenetics: focus on asthma and related phenotypes. Pharmacogenet Genomics. 2013;23:324–328. doi: 10.1097/FPC.0b013e3283607acf.  [DOI](https://doi.org/10.1097/FPC.0b013e3283607acf) | [PMC free article](/articles/PMC3767309/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23532052/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Polygenic%20heritability%20estimates%20in%20pharmacogenetics:%20focus%20on%20asthma%20and%20related%20phenotypes&author=MJ%20McGeachie&author=EA%20Stahl&author=BE%20Himes&author=SA%20Pendergrass&author=JJ%20Lima&volume=23&publication_year=2013&pages=324-328&pmid=23532052&doi=10.1097/FPC.0b013e3283607acf&)

12. Nieminen MM, Kaprio J, Koskenvuo M. A population-based study of bronchial asthma in adult twin pairs. Chest. 1991;100:70–75. doi: 10.1378/chest.100.1.70.  [DOI](https://doi.org/10.1378/chest.100.1.70) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2060393/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=A%20population-based%20study%20of%20bronchial%20asthma%20in%20adult%20twin%20pairs&author=MM%20Nieminen&author=J%20Kaprio&author=M%20Koskenvuo&volume=100&publication_year=1991&pages=70-75&pmid=2060393&doi=10.1378/chest.100.1.70&)

13. Fagnani C, Annesi-Maesano I, Brescianini S, D’Ippolito C, Medda E, Nisticò L, et al. Heritability and shared genetic effects of asthma and hay fever: an Italian study of young twins. Twin Res Hum Genet. 2008;11:121–131. doi: 10.1375/twin.11.2.121.  [DOI](https://doi.org/10.1375/twin.11.2.121) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18361712/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Twin%20Res%20Hum%20Genet&title=Heritability%20and%20shared%20genetic%20effects%20of%20asthma%20and%20hay%20fever:%20an%20Italian%20study%20of%20young%20twins&author=C%20Fagnani&author=I%20Annesi-Maesano&author=S%20Brescianini&author=C%20D%E2%80%99Ippolito&author=E%20Medda&volume=11&publication_year=2008&pages=121-131&pmid=18361712&doi=10.1375/twin.11.2.121&)

14. Himes BE, Jiang X, Hu R, Wu AC, Lasky-Su JA, Klanderman BJ, et al. Genome-wide association analysis in asthma subjects identifies SPATS2L as a novel bronchodilator response gene. PLoS Genet. 2012;8:e1002824. doi: 10.1371/journal.pgen.1002824.  [DOI](https://doi.org/10.1371/journal.pgen.1002824) | [PMC free article](/articles/PMC3390407/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22792082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Genome-wide%20association%20analysis%20in%20asthma%20subjects%20identifies%20SPATS2L%20as%20a%20novel%20bronchodilator%20response%20gene&author=BE%20Himes&author=X%20Jiang&author=R%20Hu&author=AC%20Wu&author=JA%20Lasky-Su&volume=8&publication_year=2012&pages=e1002824&pmid=22792082&doi=10.1371/journal.pgen.1002824&)

15. Duan QL, Lasky-Su J, Himes BE, Qiu W, Litonjua AA, Damask A, et al. A genome-wide association study of bronchodilator response in asthmatics. Pharmacogenomics J. 2014;14:41–47. doi: 10.1038/tpj.2013.5.  [DOI](https://doi.org/10.1038/tpj.2013.5) | [PMC free article](/articles/PMC3706515/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23508266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=A%20genome-wide%20association%20study%20of%20bronchodilator%20response%20in%20asthmatics&author=QL%20Duan&author=J%20Lasky-Su&author=BE%20Himes&author=W%20Qiu&author=AA%20Litonjua&volume=14&publication_year=2014&pages=41-47&pmid=23508266&doi=10.1038/tpj.2013.5&)

16. Israel E, Lasky-Su J, Markezich A, Damask A, Szefler SJ, Schuemann B, et al. SHARP Investigators. Genome-wide association study of short-acting β2-agonists. A novel genome-wide significant locus on chromosome 2 near ASB3. Am J Respir Crit Care Med. 2015;191:530–537. doi: 10.1164/rccm.201408-1426OC.  [DOI](https://doi.org/10.1164/rccm.201408-1426OC) | [PMC free article](/articles/PMC4384768/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25562107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Genome-wide%20association%20study%20of%20short-acting%20%CE%B22-agonists.%20A%20novel%20genome-wide%20significant%20locus%20on%20chromosome%202%20near%20ASB3&author=E%20Israel&author=J%20Lasky-Su&author=A%20Markezich&author=A%20Damask&author=SJ%20Szefler&volume=191&publication_year=2015&pages=530-537&pmid=25562107&doi=10.1164/rccm.201408-1426OC&)

17. Padhukasahasram B, Yang JJ, Levin AM, Yang M, Burchard EG, Kumar R, et al. Gene-based association identifies SPATA13-AS1 as a pharmacogenomic predictor of inhaled short-acting beta-agonist response in multiple population groups. Pharmacogenomics J. 2014;14:365–371. doi: 10.1038/tpj.2013.49.  [DOI](https://doi.org/10.1038/tpj.2013.49) | [PMC free article](/articles/PMC4098013/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24418963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Gene-based%20association%20identifies%20SPATA13-AS1%20as%20a%20pharmacogenomic%20predictor%20of%20inhaled%20short-acting%20beta-agonist%20response%20in%20multiple%20population%20groups&author=B%20Padhukasahasram&author=JJ%20Yang&author=AM%20Levin&author=M%20Yang&author=EG%20Burchard&volume=14&publication_year=2014&pages=365-371&pmid=24418963&doi=10.1038/tpj.2013.49&)

18. Drake KA, Torgerson DG, Gignoux CR, Galanter JM, Roth LA, Huntsman S, et al. A genome-wide association study of bronchodilator response in Latinos implicates rare variants. J Allergy Clin Immunol. 2014;133:370–378. doi: 10.1016/j.jaci.2013.06.043.  [DOI](https://doi.org/10.1016/j.jaci.2013.06.043) | [PMC free article](/articles/PMC3938989/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23992748/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=A%20genome-wide%20association%20study%20of%20bronchodilator%20response%20in%20Latinos%20implicates%20rare%20variants&author=KA%20Drake&author=DG%20Torgerson&author=CR%20Gignoux&author=JM%20Galanter&author=LA%20Roth&volume=133&publication_year=2014&pages=370-378&pmid=23992748&doi=10.1016/j.jaci.2013.06.043&)

19. Mak AC, White MJ, Eng C, Hu D, Huntsman S, Oh SS, et al. Whole genome sequencing to identify genetic variation associated with bronchodilator response in minority children with asthma [abstract] Am J Respir Crit Care Med. 2016;193:A3709.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Whole%20genome%20sequencing%20to%20identify%20genetic%20variation%20associated%20with%20bronchodilator%20response%20in%20minority%20children%20with%20asthma%20%5Babstract%5D&author=AC%20Mak&author=MJ%20White&author=C%20Eng&author=D%20Hu&author=S%20Huntsman&volume=193&publication_year=2016&pages=A3709&)

20. Mak AC, White MJ, Szpiech ZA, Pino-Yanes M, Oh SS, Hu D, et al. Whole genome sequencing of pharmacogenetic drug response in ethnically diverse children with asthma. Presented at the American Society of Human Genetics Annual Meeting; Oct 18–22, 2016, Vancouver, Canada. Abstract 1660W.  [Google Scholar](https://scholar.google.com/scholar_lookup?Mak%20AC,%20White%20MJ,%20Szpiech%20ZA,%20Pino-Yanes%20M,%20Oh%20SS,%20Hu%20D,%20et%20al.%20Whole%20genome%20sequencing%20of%20pharmacogenetic%20drug%20response%20in%20ethnically%20diverse%20children%20with%20asthma.%20Presented%20at%20the%20American%20Society%20of%20Human%20Genetics%20Annual%20Meeting;%20Oct%2018%E2%80%9322,%202016,%20Vancouver,%20Canada.%20Abstract%201660W.)

21. Mak AC, White MJ, Szpiech ZA, Eckalbar W, Oh SS, Pino-Yanes M, et al. Whole genome sequencing of pharmacogenetic drug response in racially and ethnically diverse children with asthma. Presented at the American Society of Human Genetics Annual Meeting; Oct 17–21, 2017, Orlando, FL. Abstract 1941F.  [Google Scholar](https://scholar.google.com/scholar_lookup?Mak%20AC,%20White%20MJ,%20Szpiech%20ZA,%20Eckalbar%20W,%20Oh%20SS,%20Pino-Yanes%20M,%20et%20al.%20Whole%20genome%20sequencing%20of%20pharmacogenetic%20drug%20response%20in%20racially%20and%20ethnically%20diverse%20children%20with%20asthma.%20Presented%20at%20the%20American%20Society%20of%20Human%20Genetics%20Annual%20Meeting;%20Oct%2017%E2%80%9321,%202017,%20Orlando,%20FL.%20Abstract%201941F.)

22. Mak AC, White MJ, Szpiech ZA, Eckalbar W, Pino-Yanes M, Oh SS, et al. Whole genome sequencing study on bronchodilator drug response in ethnically diverse children with asthma [abstract] Am J Respir Crit Care Med. 2017;195:A4969. doi: 10.1164/rccm.201712-2529OC.  [DOI](https://doi.org/10.1164/rccm.201712-2529OC) | [PMC free article](/articles/PMC6006403/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29509491/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Whole%20genome%20sequencing%20study%20on%20bronchodilator%20drug%20response%20in%20ethnically%20diverse%20children%20with%20asthma%20%5Babstract%5D&author=AC%20Mak&author=MJ%20White&author=ZA%20Szpiech&author=W%20Eckalbar&author=M%20Pino-Yanes&volume=195&publication_year=2017&pages=A4969&pmid=29509491&doi=10.1164/rccm.201712-2529OC&)

23. Borrell LN, Nguyen EA, Roth LA, Oh SS, Tcheurekdjian H, Sen S, et al. Childhood obesity and asthma control in the GALA II and SAGE II studies. Am J Respir Crit Care Med. 2013;187:697–702. doi: 10.1164/rccm.201211-2116OC.  [DOI](https://doi.org/10.1164/rccm.201211-2116OC) | [PMC free article](/articles/PMC3678111/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23392439/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Childhood%20obesity%20and%20asthma%20control%20in%20the%20GALA%20II%20and%20SAGE%20II%20studies&author=LN%20Borrell&author=EA%20Nguyen&author=LA%20Roth&author=SS%20Oh&author=H%20Tcheurekdjian&volume=187&publication_year=2013&pages=697-702&pmid=23392439&doi=10.1164/rccm.201211-2116OC&)

24. Han B, Eskin E. Random-effects model aimed at discovering associations in meta-analysis of genome-wide association studies. Am J Hum Genet. 2011;88:586–598. doi: 10.1016/j.ajhg.2011.04.014.  [DOI](https://doi.org/10.1016/j.ajhg.2011.04.014) | [PMC free article](/articles/PMC3146723/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21565292/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Random-effects%20model%20aimed%20at%20discovering%20associations%20in%20meta-analysis%20of%20genome-wide%20association%20studies&author=B%20Han&author=E%20Eskin&volume=88&publication_year=2011&pages=586-598&pmid=21565292&doi=10.1016/j.ajhg.2011.04.014&)

25. McFadden D. Quantitative methods for analysing travel behavior of individuals: some recent developments. In: Hensher DA, Stopher PR, editors. Behavioural travel modelling. London: Croom Helm; 1979. pp. 279–318.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Behavioural%20travel%20modelling&author=D%20McFadden&publication_year=1979&)

26. Ciesielski TH, Pendergrass SA, White MJ, Kodaman N, Sobota RS, Huang M, et al. Diverse convergent evidence in the genetic analysis of complex disease: coordinating omic, informatic, and experimental evidence to better identify and validate risk factors. BioData Min. 2014;7:10. doi: 10.1186/1756-0381-7-10.  [DOI](https://doi.org/10.1186/1756-0381-7-10) | [PMC free article](/articles/PMC4112852/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25071867/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BioData%20Min&title=Diverse%20convergent%20evidence%20in%20the%20genetic%20analysis%20of%20complex%20disease:%20coordinating%20omic,%20informatic,%20and%20experimental%20evidence%20to%20better%20identify%20and%20validate%20risk%20factors&author=TH%20Ciesielski&author=SA%20Pendergrass&author=MJ%20White&author=N%20Kodaman&author=RS%20Sobota&volume=7&publication_year=2014&pages=10&pmid=25071867&doi=10.1186/1756-0381-7-10&)

27. Hankinson JL, Odencrantz JR, Fedan KB. Spirometric reference values from a sample of the general U.S. population. Am J Respir Crit Care Med. 1999;159:179–187. doi: 10.1164/ajrccm.159.1.9712108.  [DOI](https://doi.org/10.1164/ajrccm.159.1.9712108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9872837/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Spirometric%20reference%20values%20from%20a%20sample%20of%20the%20general%20U.S.%20population&author=JL%20Hankinson&author=JR%20Odencrantz&author=KB%20Fedan&volume=159&publication_year=1999&pages=179-187&pmid=9872837&doi=10.1164/ajrccm.159.1.9712108&)

28. Sobota RS, Shriner D, Kodaman N, Goodloe R, Zheng W, Gao YT, et al. Addressing population-specific multiple testing burdens in genetic association studies. Ann Hum Genet. 2015;79:136–147. doi: 10.1111/ahg.12095.  [DOI](https://doi.org/10.1111/ahg.12095) | [PMC free article](/articles/PMC4334751/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25644736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Hum%20Genet&title=Addressing%20population-specific%20multiple%20testing%20burdens%20in%20genetic%20association%20studies&author=RS%20Sobota&author=D%20Shriner&author=N%20Kodaman&author=R%20Goodloe&author=W%20Zheng&volume=79&publication_year=2015&pages=136-147&pmid=25644736&doi=10.1111/ahg.12095&)

29. Pe’er I, Yelensky R, Altshuler D, Daly MJ. Estimation of the multiple testing burden for genomewide association studies of nearly all common variants. Genet Epidemiol. 2008;32:381–385. doi: 10.1002/gepi.20303.  [DOI](https://doi.org/10.1002/gepi.20303) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18348202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Estimation%20of%20the%20multiple%20testing%20burden%20for%20genomewide%20association%20studies%20of%20nearly%20all%20common%20variants&author=I%20Pe%E2%80%99er&author=R%20Yelensky&author=D%20Altshuler&author=MJ%20Daly&volume=32&publication_year=2008&pages=381-385&pmid=18348202&doi=10.1002/gepi.20303&)

30. Duggal P, Gillanders EM, Holmes TN, Bailey-Wilson JE. Establishing an adjusted p-value threshold to control the family-wide type 1 error in genome wide association studies. BMC Genomics. 2008;9:516. doi: 10.1186/1471-2164-9-516.  [DOI](https://doi.org/10.1186/1471-2164-9-516) | [PMC free article](/articles/PMC2621212/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18976480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=Establishing%20an%20adjusted%20p-value%20threshold%20to%20control%20the%20family-wide%20type%201%20error%20in%20genome%20wide%20association%20studies&author=P%20Duggal&author=EM%20Gillanders&author=TN%20Holmes&author=JE%20Bailey-Wilson&volume=9&publication_year=2008&pages=516&pmid=18976480&doi=10.1186/1471-2164-9-516&)

31. Castro-Giner F, de Cid R, Gonzalez JR, Jarvis D, Heinrich J, Janson C, et al. Positionally cloned genes and age-specific effects in asthma and atopy: an international population-based cohort study (ECRHS) Thorax. 2010;65:124–131. doi: 10.1136/thx.2009.119628.  [DOI](https://doi.org/10.1136/thx.2009.119628) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19996348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thorax&title=Positionally%20cloned%20genes%20and%20age-specific%20effects%20in%20asthma%20and%20atopy:%20an%20international%20population-based%20cohort%20study%20(ECRHS)&author=F%20Castro-Giner&author=R%20de%20Cid&author=JR%20Gonzalez&author=D%20Jarvis&author=J%20Heinrich&volume=65&publication_year=2010&pages=124-131&pmid=19996348&doi=10.1136/thx.2009.119628&)

32. White MJ, Risse-Adams O, Goddard P, Contreras MG, Adams J, Hu D, et al. Novel genetic risk factors for asthma in African American children: precision medicine and the SAGE II Study. Immunogenetics. 2016;68:391–400. doi: 10.1007/s00251-016-0914-1.  [DOI](https://doi.org/10.1007/s00251-016-0914-1) | [PMC free article](/articles/PMC4927336/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27142222/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunogenetics&title=Novel%20genetic%20risk%20factors%20for%20asthma%20in%20African%20American%20children:%20precision%20medicine%20and%20the%20SAGE%20II%20Study&author=MJ%20White&author=O%20Risse-Adams&author=P%20Goddard&author=MG%20Contreras&author=J%20Adams&volume=68&publication_year=2016&pages=391-400&pmid=27142222&doi=10.1007/s00251-016-0914-1&)

33. Johnson M. The beta-adrenoceptor. Am J Respir Crit Care Med. 1998;158:S146–S153. doi: 10.1164/ajrccm.158.supplement_2.13tac110.  [DOI](https://doi.org/10.1164/ajrccm.158.supplement_2.13tac110) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9817738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=The%20beta-adrenoceptor&author=M%20Johnson&volume=158&publication_year=1998&pages=S146-S153&pmid=9817738&doi=10.1164/ajrccm.158.supplement_2.13tac110&)

34. Cumberland PM, Czanner G, Bunce C, Doré CJ, Freemantle N, García-Fiñana M Ophthalmic Statistics Group. Ophthalmic statistics note: the perils of dichotomising continuous variables. Br J Ophthalmol. 2014;98:841–843. doi: 10.1136/bjophthalmol-2014-304930.  [DOI](https://doi.org/10.1136/bjophthalmol-2014-304930) | [PMC free article](/articles/PMC4107463/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24682179/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Ophthalmol&title=Ophthalmic%20statistics%20note:%20the%20perils%20of%20dichotomising%20continuous%20variables&author=PM%20Cumberland&author=G%20Czanner&author=C%20Bunce&author=CJ%20Dor%C3%A9&author=N%20Freemantle&volume=98&publication_year=2014&pages=841-843&pmid=24682179&doi=10.1136/bjophthalmol-2014-304930&)

35. Li D, Lewinger JP, Gauderman WJ, Murcray CE, Conti D. Using extreme phenotype sampling to identify the rare causal variants of quantitative traits in association studies. Genet Epidemiol. 2011;35:790–799. doi: 10.1002/gepi.20628.  [DOI](https://doi.org/10.1002/gepi.20628) | [PMC free article](/articles/PMC4238184/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21922541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Using%20extreme%20phenotype%20sampling%20to%20identify%20the%20rare%20causal%20variants%20of%20quantitative%20traits%20in%20association%20studies&author=D%20Li&author=JP%20Lewinger&author=WJ%20Gauderman&author=CE%20Murcray&author=D%20Conti&volume=35&publication_year=2011&pages=790-799&pmid=21922541&doi=10.1002/gepi.20628&)

36. Guey LT, Kravic J, Melander O, Burtt NP, Laramie JM, Lyssenko V, et al. Power in the phenotypic extremes: a simulation study of power in discovery and replication of rare variants. Genet Epidemiol. 2011;35:236–246. doi: 10.1002/gepi.20572.  [DOI](https://doi.org/10.1002/gepi.20572) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21308769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=Power%20in%20the%20phenotypic%20extremes:%20a%20simulation%20study%20of%20power%20in%20discovery%20and%20replication%20of%20rare%20variants&author=LT%20Guey&author=J%20Kravic&author=O%20Melander&author=NP%20Burtt&author=JM%20Laramie&volume=35&publication_year=2011&pages=236-246&pmid=21308769&doi=10.1002/gepi.20572&)

37. Lee S, Wu MC, Lin X. Optimal tests for rare variant effects in sequencing association studies. Biostatistics. 2012;13:762–775. doi: 10.1093/biostatistics/kxs014.  [DOI](https://doi.org/10.1093/biostatistics/kxs014) | [PMC free article](/articles/PMC3440237/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22699862/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biostatistics&title=Optimal%20tests%20for%20rare%20variant%20effects%20in%20sequencing%20association%20studies&author=S%20Lee&author=MC%20Wu&author=X%20Lin&volume=13&publication_year=2012&pages=762-775&pmid=22699862&doi=10.1093/biostatistics/kxs014&)

38. Lee JH, McDonald ML, Cho MH, Wan ES, Castaldi PJ, Hunninghake GM, et al. COPDGene and ECLIPSE Investigators. DNAH5 is associated with total lung capacity in chronic obstructive pulmonary disease. Respir Res. 2014;15:97. doi: 10.1186/s12931-014-0097-y.  [DOI](https://doi.org/10.1186/s12931-014-0097-y) | [PMC free article](/articles/PMC4169636/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25134640/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir%20Res&title=DNAH5%20is%20associated%20with%20total%20lung%20capacity%20in%20chronic%20obstructive%20pulmonary%20disease&author=JH%20Lee&author=ML%20McDonald&author=MH%20Cho&author=ES%20Wan&author=PJ%20Castaldi&volume=15&publication_year=2014&pages=97&pmid=25134640&doi=10.1186/s12931-014-0097-y&)

39. Ortiz RA, Barnes KC. Genetics of allergic diseases. Immunol Allergy Clin North Am. 2015;35:19–44. doi: 10.1016/j.iac.2014.09.014.  [DOI](https://doi.org/10.1016/j.iac.2014.09.014) | [PMC free article](/articles/PMC4415518/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25459575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunol%20Allergy%20Clin%20North%20Am&title=Genetics%20of%20allergic%20diseases&author=RA%20Ortiz&author=KC%20Barnes&volume=35&publication_year=2015&pages=19-44&pmid=25459575&doi=10.1016/j.iac.2014.09.014&)

40. Ramasamy A, Curjuric I, Coin LJ, Kumar A, McArdle WL, Imboden M, et al. A genome-wide meta-analysis of genetic variants associated with allergic rhinitis and grass sensitization and their interaction with birth order. J Allergy Clin Immunol. 2011;128:996–1005. doi: 10.1016/j.jaci.2011.08.030.  [DOI](https://doi.org/10.1016/j.jaci.2011.08.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22036096/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=A%20genome-wide%20meta-analysis%20of%20genetic%20variants%20associated%20with%20allergic%20rhinitis%20and%20grass%20sensitization%20and%20their%20interaction%20with%20birth%20order&author=A%20Ramasamy&author=I%20Curjuric&author=LJ%20Coin&author=A%20Kumar&author=WL%20McArdle&volume=128&publication_year=2011&pages=996-1005&pmid=22036096&doi=10.1016/j.jaci.2011.08.030&)

41. Naqvi M, Choudhry S, Tsai HJ, Thyne S, Navarro D, Nazario S, et al. Association between IgE levels and asthma severity among African American, Mexican, and Puerto Rican patients with asthma. J Allergy Clin Immunol. 2007;120:137–143. doi: 10.1016/j.jaci.2007.02.045.  [DOI](https://doi.org/10.1016/j.jaci.2007.02.045) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17498790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Association%20between%20IgE%20levels%20and%20asthma%20severity%20among%20African%20American,%20Mexican,%20and%20Puerto%20Rican%20patients%20with%20asthma&author=M%20Naqvi&author=S%20Choudhry&author=HJ%20Tsai&author=S%20Thyne&author=D%20Navarro&volume=120&publication_year=2007&pages=137-143&pmid=17498790&doi=10.1016/j.jaci.2007.02.045&)

42. Edwards MR, Bartlett NW, Clarke D, Birrell M, Belvisi M, Johnston SL. Targeting the NF-kappaB pathway in asthma and chronic obstructive pulmonary disease. Pharmacol Ther. 2009;121:1–13. doi: 10.1016/j.pharmthera.2008.09.003.  [DOI](https://doi.org/10.1016/j.pharmthera.2008.09.003) | [PMC free article](/articles/PMC7172981/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18950657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Ther&title=Targeting%20the%20NF-kappaB%20pathway%20in%20asthma%20and%20chronic%20obstructive%20pulmonary%20disease&author=MR%20Edwards&author=NW%20Bartlett&author=D%20Clarke&author=M%20Birrell&author=M%20Belvisi&volume=121&publication_year=2009&pages=1-13&pmid=18950657&doi=10.1016/j.pharmthera.2008.09.003&)

43. Zhen G, Park SW, Nguyenvu LT, Rodriguez MW, Barbeau R, Paquet AC, et al. IL-13 and epidermal growth factor receptor have critical but distinct roles in epithelial cell mucin production. Am J Respir Cell Mol Biol. 2007;36:244–253. doi: 10.1165/rcmb.2006-0180OC.  [DOI](https://doi.org/10.1165/rcmb.2006-0180OC) | [PMC free article](/articles/PMC1899314/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16980555/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Cell%20Mol%20Biol&title=IL-13%20and%20epidermal%20growth%20factor%20receptor%20have%20critical%20but%20distinct%20roles%20in%20epithelial%20cell%20mucin%20production&author=G%20Zhen&author=SW%20Park&author=LT%20Nguyenvu&author=MW%20Rodriguez&author=R%20Barbeau&volume=36&publication_year=2007&pages=244-253&pmid=16980555&doi=10.1165/rcmb.2006-0180OC&)

44. Besecker B, Bao S, Bohacova B, Papp A, Sadee W, Knoell DL. The human zinc transporter SLC39A8 (Zip8) is critical in zinc-mediated cytoprotection in lung epithelia. Am J Physiol Lung Cell Mol Physiol. 2008;294:L1127–L1136. doi: 10.1152/ajplung.00057.2008.  [DOI](https://doi.org/10.1152/ajplung.00057.2008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18390834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol%20Lung%20Cell%20Mol%20Physiol&title=The%20human%20zinc%20transporter%20SLC39A8%20(Zip8)%20is%20critical%20in%20zinc-mediated%20cytoprotection%20in%20lung%20epithelia&author=B%20Besecker&author=S%20Bao&author=B%20Bohacova&author=A%20Papp&author=W%20Sadee&volume=294&publication_year=2008&pages=L1127-L1136&pmid=18390834&doi=10.1152/ajplung.00057.2008&)

45. Persson H, Kwon AT, Ramilowski JA, Silberberg G, Söderhäll C, Orsmark-Pietras C, et al. Transcriptome analysis of controlled and therapy-resistant childhood asthma reveals distinct gene expression profiles. J Allergy Clin Immunol. 2015;136:638–648. doi: 10.1016/j.jaci.2015.02.026.  [DOI](https://doi.org/10.1016/j.jaci.2015.02.026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25863981/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Transcriptome%20analysis%20of%20controlled%20and%20therapy-resistant%20childhood%20asthma%20reveals%20distinct%20gene%20expression%20profiles&author=H%20Persson&author=AT%20Kwon&author=JA%20Ramilowski&author=G%20Silberberg&author=C%20S%C3%B6derh%C3%A4ll&volume=136&publication_year=2015&pages=638-648&pmid=25863981&doi=10.1016/j.jaci.2015.02.026&)

46. Lin YJ, Chang JS, Liu X, Tsang H, Chien WK, Chen JH, et al. Genetic variants in PLCB4/PLCB1 as susceptibility loci for coronary artery aneurysm formation in Kawasaki disease in Han Chinese in Taiwan. Sci Rep. 2015;5:14762. doi: 10.1038/srep14762.  [DOI](https://doi.org/10.1038/srep14762) | [PMC free article](/articles/PMC4593004/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26434682/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Genetic%20variants%20in%20PLCB4/PLCB1%20as%20susceptibility%20loci%20for%20coronary%20artery%20aneurysm%20formation%20in%20Kawasaki%20disease%20in%20Han%20Chinese%20in%20Taiwan&author=YJ%20Lin&author=JS%20Chang&author=X%20Liu&author=H%20Tsang&author=WK%20Chien&volume=5&publication_year=2015&pages=14762&pmid=26434682&doi=10.1038/srep14762&)

47. Hersch M, Peter B, Kang HM, Schüpfer F, Abriel H, Pedrazzini T, et al. Mapping genetic variants associated with beta-adrenergic responses in inbred mice. PLoS One. 2012;7:e41032. doi: 10.1371/journal.pone.0041032.  [DOI](https://doi.org/10.1371/journal.pone.0041032) | [PMC free article](/articles/PMC3409184/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22859963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Mapping%20genetic%20variants%20associated%20with%20beta-adrenergic%20responses%20in%20inbred%20mice&author=M%20Hersch&author=B%20Peter&author=HM%20Kang&author=F%20Sch%C3%BCpfer&author=H%20Abriel&volume=7&publication_year=2012&pages=e41032&pmid=22859963&doi=10.1371/journal.pone.0041032&)

48. Dougherty RH, Sidhu SS, Raman K, Solon M, Solberg OD, Caughey GH, et al. Accumulation of intraepithelial mast cells with a unique protease phenotype in T(H)2-high asthma. J Allergy Clin Immunol. 2010;125:1046–1053. doi: 10.1016/j.jaci.2010.03.003.  [DOI](https://doi.org/10.1016/j.jaci.2010.03.003) | [PMC free article](/articles/PMC2918406/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20451039/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Accumulation%20of%20intraepithelial%20mast%20cells%20with%20a%20unique%20protease%20phenotype%20in%20T(H)2-high%20asthma&author=RH%20Dougherty&author=SS%20Sidhu&author=K%20Raman&author=M%20Solon&author=OD%20Solberg&volume=125&publication_year=2010&pages=1046-1053&pmid=20451039&doi=10.1016/j.jaci.2010.03.003&)

49. Woodruff PG, Boushey HA, Dolganov GM, Barker CS, Yang YH, Donnelly S, et al. Genome-wide profiling identifies epithelial cell genes associated with asthma and with treatment response to corticosteroids. Proc Natl Acad Sci USA. 2007;104:15858–15863. doi: 10.1073/pnas.0707413104.  [DOI](https://doi.org/10.1073/pnas.0707413104) | [PMC free article](/articles/PMC2000427/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17898169/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Genome-wide%20profiling%20identifies%20epithelial%20cell%20genes%20associated%20with%20asthma%20and%20with%20treatment%20response%20to%20corticosteroids&author=PG%20Woodruff&author=HA%20Boushey&author=GM%20Dolganov&author=CS%20Barker&author=YH%20Yang&volume=104&publication_year=2007&pages=15858-15863&pmid=17898169&doi=10.1073/pnas.0707413104&)

50. Motsinger-Reif AA, Jorgenson E, Relling MV, Kroetz DL, Weinshilboum R, Cox NJ, et al. Genome-wide association studies in pharmacogenomics: successes and lessons. Pharmacogenet Genomics. 2013;23:383–394. doi: 10.1097/FPC.0b013e32833d7b45.  [DOI](https://doi.org/10.1097/FPC.0b013e32833d7b45) | [PMC free article](/articles/PMC3003940/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20639796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Genome-wide%20association%20studies%20in%20pharmacogenomics:%20successes%20and%20lessons&author=AA%20Motsinger-Reif&author=E%20Jorgenson&author=MV%20Relling&author=DL%20Kroetz&author=R%20Weinshilboum&volume=23&publication_year=2013&pages=383-394&pmid=20639796&doi=10.1097/FPC.0b013e32833d7b45&)

51. Peloso GM, Rader DJ, Gabriel S, Kathiresan S, Daly MJ, Neale BM. Phenotypic extremes in rare variant study designs. Eur J Hum Genet. 2016;24:924–930. doi: 10.1038/ejhg.2015.197.  [DOI](https://doi.org/10.1038/ejhg.2015.197) | [PMC free article](/articles/PMC4867440/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26350511/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Phenotypic%20extremes%20in%20rare%20variant%20study%20designs&author=GM%20Peloso&author=DJ%20Rader&author=S%20Gabriel&author=S%20Kathiresan&author=MJ%20Daly&volume=24&publication_year=2016&pages=924-930&pmid=26350511&doi=10.1038/ejhg.2015.197&)

52. Hall SS. Genetics: a gene of rare effect. Nature. 2013;496:152–155. doi: 10.1038/496152a.  [DOI](https://doi.org/10.1038/496152a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23579660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetics:%20a%20gene%20of%20rare%20effect&author=SS%20Hall&volume=496&publication_year=2013&pages=152-155&pmid=23579660&doi=10.1038/496152a&)

53. Oh SS, Galanter J, Thakur N, Pino-Yanes M, Barcelo NE, White MJ, et al. Diversity in clinical and biomedical research: a promise yet to be fulfilled. PLoS Med. 2015;12:e1001918. doi: 10.1371/journal.pmed.1001918.  [DOI](https://doi.org/10.1371/journal.pmed.1001918) | [PMC free article](/articles/PMC4679830/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26671224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Med&title=Diversity%20in%20clinical%20and%20biomedical%20research:%20a%20promise%20yet%20to%20be%20fulfilled&author=SS%20Oh&author=J%20Galanter&author=N%20Thakur&author=M%20Pino-Yanes&author=NE%20Barcelo&volume=12&publication_year=2015&pages=e1001918&pmid=26671224&doi=10.1371/journal.pmed.1001918&)

54. Popejoy AB, Fullerton SM. Genomics is failing on diversity. Nature. 2016;538:161–164. doi: 10.1038/538161a.  [DOI](https://doi.org/10.1038/538161a) | [PMC free article](/articles/PMC5089703/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27734877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genomics%20is%20failing%20on%20diversity&author=AB%20Popejoy&author=SM%20Fullerton&volume=538&publication_year=2016&pages=161-164&pmid=27734877&doi=10.1038/538161a&)

55. Hindorff LA, Bonham VL, Brody LC, Ginoza MEC, Hutter CM, Manolio TA, et al. Prioritizing diversity in human genomics research. Nat Rev Genet. 2018;19:175–185. doi: 10.1038/nrg.2017.89.  [DOI](https://doi.org/10.1038/nrg.2017.89) | [PMC free article](/articles/PMC6532668/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29151588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Prioritizing%20diversity%20in%20human%20genomics%20research&author=LA%20Hindorff&author=VL%20Bonham&author=LC%20Brody&author=MEC%20Ginoza&author=CM%20Hutter&volume=19&publication_year=2018&pages=175-185&pmid=29151588&doi=10.1038/nrg.2017.89&)

56. MacArthur DG, Manolio TA, Dimmock DP, Rehm HL, Shendure J, Abecasis GR, et al. Guidelines for investigating causality of sequence variants in human disease. Nature. 2014;508:469–476. doi: 10.1038/nature13127.  [DOI](https://doi.org/10.1038/nature13127) | [PMC free article](/articles/PMC4180223/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24759409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Guidelines%20for%20investigating%20causality%20of%20sequence%20variants%20in%20human%20disease&author=DG%20MacArthur&author=TA%20Manolio&author=DP%20Dimmock&author=HL%20Rehm&author=J%20Shendure&volume=508&publication_year=2014&pages=469-476&pmid=24759409&doi=10.1038/nature13127&)

57. Oh SS, White MJ, Gignoux CR, Burchard EG. Making precision medicine socially precise: take a deep breath. Am J Respir Crit Care Med. 2016;193:348–350. doi: 10.1164/rccm.201510-2045ED.  [DOI](https://doi.org/10.1164/rccm.201510-2045ED) | [PMC free article](/articles/PMC4803087/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26871667/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Making%20precision%20medicine%20socially%20precise:%20take%20a%20deep%20breath&author=SS%20Oh&author=MJ%20White&author=CR%20Gignoux&author=EG%20Burchard&volume=193&publication_year=2016&pages=348-350&pmid=26871667&doi=10.1164/rccm.201510-2045ED&)
