# Comprehensive Ara-C SNP score predicts leukemic cell intracellular ara-CTP levels in pediatric acute myeloid leukemia patients

## Metadata
**Authors:** Abdelrahman H Elsayed, Xueyuan Cao, Kristine R Crews, Varsha Gandhi, William Plunkett, Jeffrey E Rubnitz, Raul C Ribeiro, Stanley B Pounds, Jatinder K Lamba
**Journal:** Pharmacogenomics
**Date:** 2018 Aug 8
**DOI:** [10.2217/pgs-2018-0086](https://doi.org/10.2217/pgs-2018-0086)
**PMID:** 30088438
**PMCID:** PMC6219441
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6219441/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6219441/pdf/pgs-19-1101.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6219441/pdf/pgs-19-1101.pdf)

## Abstract

**Aim:** 
Cytarabine (Ara-C), a mainstay of acute myeloid leukemia (AML) treatment, is a prodrug requiring activation to ara-CTP for its antileukemic activity. Aim of this study was to evaluate impact of genetic variants in the key genes involved in ara-C metabolism on the leukemic cell intracellular levels of ara-CTP.

**Method:** 
We investigated SNPs in 14 ara-C metabolic-pathway genes, for association with intracellular ara-CTP levels, in leukemic cells obtained post-initiation of cytarabine infusion in pediatric AML patients (n = 68).

**Results:** 
Nine SNPs were significantly associated with leukemic cell intracellular concentration of ara-CTP. A comprehensive ara-CTP-SNP-score (ACSS) was further developed from top four SNPs identified in regression model. Patients were classified into three groups based on ACSS: high-ACSS (score >0), intermediate-ACSS (score = 0) and low-ACSS (score <0). ACSS designation was significant predictor of intracellular ara-CTP levels (p = 0.00012), suggesting a cumulative or synergistic effect of the significant SNPs.

**Conclusion:** 
ACSS score designation holds promise in definfing ara-C dose. Validation of the clinical utility of ACSS score in other independent cohorts will help identification of patients with potentially lower or higher levels of the ara-CTP in leukemic cells, thereby opening up opportunities for dose management to reduce toxicity and enhance efficacy.

Keywords: : acute myeloid leukemia, cytarabine, pharmacogenomics, SNPs

### Aim:

Cytarabine (Ara-C), a mainstay of acute myeloid leukemia (AML) treatment, is a prodrug requiring activation to ara-CTP for its antileukemic activity. Aim of this study was to evaluate impact of genetic variants in the key genes involved in ara-C metabolism on the leukemic cell intracellular levels of ara-CTP.

### Method:

We investigated SNPs in 14 ara-C metabolic-pathway genes, for association with intracellular ara-CTP levels, in leukemic cells obtained post-initiation of cytarabine infusion in pediatric AML patients (n = 68).

### Results:

Nine SNPs were significantly associated with leukemic cell intracellular concentration of ara-CTP. A comprehensive ara-CTP-SNP-score (ACSS) was further developed from top four SNPs identified in regression model. Patients were classified into three groups based on ACSS: high-ACSS (score >0), intermediate-ACSS (score = 0) and low-ACSS (score <0). ACSS designation was significant predictor of intracellular ara-CTP levels (p = 0.00012), suggesting a cumulative or synergistic effect of the significant SNPs.

### Conclusion:

ACSS score designation holds promise in definfing ara-C dose. Validation of the clinical utility of ACSS score in other independent cohorts will help identification of patients with potentially lower or higher levels of the ara-CTP in leukemic cells, thereby opening up opportunities for dose management to reduce toxicity and enhance efficacy.

## Materials & methods

### Patient cohort

Patients included in this study were enrolled in the St Jude AML-97 clinical trial. The details of study design, patient eligibility and clinical outcome are published elsewhere [[17](#B17),[18](#B18)]. Briefly, between March 1997 and June 2002, 96 patients younger than 22 years with previously untreated AML or myelodysplastic syndrome, were randomized to receive cytarabine as a 2 h 500 mg/m^2^ intravenous (iv.) bolus infusion (arm A, n = 50) or as a 500 mg/m^2^/day continuous infusion (arm B, n = 46) for 5 days. The patients in both arms received cladribine 9 mg/m^2^/day as a 30-min iv. infusion for 5 days that was initiated 24 hr after the start of cytarabine infusion, thus patients in both arms received cytarabine alone either as short daily or continuous infusion on day 1 (overall study schema is shown in [Figure 1](#F0001)).

### Figure 1. . Overall schema of the study.

![Figure 1. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0c76/6219441/8c4ac86c35c3/pgs-19-1101-g1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6219441_pgs-19-1101-g1.jpg)

AML: Acute myeloid leukemia; LD: Linkage disequilibrium.

### Intracellular ara-CTP levels

Bone marrow aspirates were obtained on day 1 after initiation of cytarabine, as previously described [[17](#B17),[18](#B18)]. The patients with ≥70% leukemic blast percentage in the bone marrow were assessed for intracellular ara-CTP levels. Briefly, mononuclear cells were isolated using Ficoll-Hypaque density-gradient centrifugation and intracellular ara-CTP was quantified. Intracellular levels were calculated as nmol nucleotide per 2.0 × 10^7^ cells as described previously [[17](#B17),[18](#B18)]. Bone marrow samples were also obtained on day 2 (details have been published elsewhere [[17](#B17),[18](#B18)]).

### Genotyping for SNPs

We genotyped 108 SNPs in 14 genes known to be involved in ara-C metabolism and transport including *DCK, CMPK1, CDA, DCTD, RRM1, RRM2, NT5C2, NT5C3, CTPS1, NME4, NME3*, in addition to drug transporters *SLC29A1, SLC28A1* and *SLC28A3*). SNPs were genotyped using Sequenome (CA, USA) based genotyping that uses MALDI-TOF-based chemistry at University of Minnesota, Biomedical Genomics Center. SNPs with call rate of <95%, minor allele frequency of <0.05 were excluded. Additionally, we tested for SNPs on a gene locus for linkage disequilibrium and for SNPs co-occurring with high LD r^2^ >0.8, we included one SNP from the group for further analysis. Overall, we evaluated 88 SNPs in 14 genes for association with intracellular levels of ara-CTP (details provided in [Supplementary Table 1](http:///doi/suppl/10.2217/pgs-2018-0086/suppl_file/pgs-19-1101-s2.docx)).

### Statistical analysis

Kruskal–Wallis or Wilcoxon rank test was used to compare ara-CTP levels between genotype groups or groups defined by combinatorial risk scores. Ara-CTP levels were log transformed before analysis. We evaluated all 88 SNPs using linear regression model to test impact for possible combinations of two SNPs per model. This method can extend the linear modeling framework to include variables that are not normally distributed. Bayesian Information Criterion (BIC) has been used to compare and order the models by the weight of evidence in favor of each model. Information measures have penalties if the added variable did not significantly improve the model fit. So that, the models with lowest BIC is preferred and they are more likely to have generated the observed intracellular ara-CTP levels. Thereafter, 1000 permutation tests have been done for each model to evaluate if by chance we can get better models to fit the data compared with the observed model. Permutation p-value <0.05 was considered significant. Univariate multiple linear regression models were built by stepwise addition of SNP and other major demographic and diagnostic factors such as race, cytogenetic risk groups and treatment arms. We have been able to obtain percentage of variability in ara-CTP, which can be explained by the independent variables individually.

### Development of comprehensive ara-C SNP score results

Comprehensive ara-C SNP score (ACSS) was constructed using four SNPs representing top three models demonstrating significant association with ara-CTP levels. As a step 1, we designated directional genotype score for each SNP in each patient by taking into account direction of association of variant allele with ara-CTP levels as well as the potential mode of inheritance. The genotype score was then utilized for generation of a composite score by addition of individual genotype scores. Thus, instead of multiple genotypes each patient now had one composite score. Addition of genotype scores for four unique SNPs from top three models resulted in composite ACSS ranging from -2 to 3 with negative score indicating more number of low-ara-CTP associated alleles and positive score indicating more number of high ara-CTP associated alleles.

## Results

### Interpatient variation in ara-CTP levels

Intracellular ara-CTP levels at day 1 and the genotype data were successfully obtained for 68 patients (arm A, n = 35 and arm B, n = 33) making this to date the largest cohort of AML patients with intracellular ara-CTP levels following treatment with ara-C. The patient characteristics are listed in [Table 1](#T1). Wide interpatient variation in intracellular levels of ara-CTP was observed: 40.5-fold in arm A (range: 0.06–2.43 nmol ara-CTP per 2 × 10^7^ leukemic cells) and 100-fold in arm B (range: 0.012–1.222 nmol ara-CTP per 2 × 10^7^ leukemic cells). Intracellular ara-CTP levels did not differ by treatment arm, age, gender, initial cytogenetic risk group or race (all p > 0.05). Given that cytarabine was the only agent patients received before day 1 sample collection we focused primarily on the ara-CTP levels determined at this time point.

### Table 1. . Patient demographics and characteristic features in all patients and by treatment arm.

| Characteristics | Overall (n = 68) | Arm A (n = 35) | Arm B (n = 33) |
| --- | --- | --- | --- |
| Age (years) |  |  |  |
|   |  |  |  |
| Median | 10 | 9 | 12 |
|   |  |  |  |
| Range | 0.5–21 | 0.5–21 | 1–20 |
|   |  |  |  |
| Race |  |  |  |
|   |  |  |  |
| White | 43 | 26 | 17 |
|   |  |  |  |
| Black | 13 | 4 | 9 |
|   |  |  |  |
| Other | 12 | 5 | 7 |
|   |  |  |  |
| Gender |  |  |  |
|   |  |  |  |
| Male | 31 | 18 | 13 |
|   |  |  |  |
| Female | 37 | 17 | 20 |
|   |  |  |  |
| Risk group |  |  |  |
|   |  |  |  |
| Favorable | 26 | 18 | 8 |
|   |  |  |  |
| Intermediate | 38 | 16 | 22 |
|   |  |  |  |
| Adverse | 2 | 0 | 2 |
|   |  |  |  |
| NA | 2 | 1 | 1 |
### Association of ara-C metabolic pathway SNPs with intracellular levels of ara-CTP at day 1 post-treatment

Nine SNPs demonstrated association with day 1 intracellular ara-CTP levels at p < 0.05 ([Table 2](#T2)). Among these, SNPs with variant allele associated with lower intracellular levels included: rs4643786 T >C in 3′UTR of *DCK* (TT vs CT +CC; p = 0.023; consistent with our previous results in smaller subset of patients); *SLC28A1* intronic SNP rs11853372 G >T (TT vs GT + GG; p = 0.018); *DCTD* synonymous coding SNP rs4742 T >C (CC vs CT + TT; p = 0.016), which occurs in strong LD with at least 17 different intronic variants within *DCTD* locus and *CTPS1* intronic SNP rs4364871 C >T which occurs in LD with 15 additional SNPs on *CTPS1* locus (CC vs CT + TT; p = 0.038).

### Table 2. . SNPs demonstrating significant association with day 1 intracellular ara-CTP levels post-infusion of cytarabine in pediatric acute myeloid leukemia patients treated under St Jude AML97 clinical trial (n = 68).

| Gene name | SNP rsID | SNP location | MAF | Ara-CTP levels (nmol/2 × 107 cells) |  |  | Direction association with ara-CTP levels | Genotype score (inheritance mode) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |  |
|   |   |   |   | Genotype | Mean ± SD | p-value |   |   |
| DCK | rs4643786†; T >C | Intron (occurs in LD with several intronic SNPs [11]) | 0.125 | TT (n = 51)CT (n = 11)CC (n = 3) | 0.488 ± 0.3910.267 ± 0.2210.145 ± 0.091 | TT vs CT + CC; p = 0.023 | Low | Additive(TT = 0, CT = -1, CC = -2) |
|   |  |  |  |  |  |  |  |  |
| SLC28A1 | rs11853372; G >T | Intron | 0.328 | GG (n = 32)GT (n = 30)TT (n = 6) | 0.510 ± 0.5080.461 ± 0.3840.146 ± 0.095 | TT vs GT + GG, p = 0.018 | Low | Recessive(GG/GT = 0, TT = -1) |
|   |  |  |  |  |  |  |  |  |
| DCTD | rs4742; T >C | Synonymous (occurs in LD with 17 intronic SNPs in DCTD) | 0.364 | TT (n = 25)CT (n = 32)CC (n = 10) | 0.450 ± 0.3520.533 ± 0.5280.239 ± 0.274 | CC vs CT + TT; p = 0.016 | Low | Recessive(TT/CT = 0, CC = -1) |
|   |  |  |  |  |  |  |  |  |
| CTPS1 | rs4364871; C >T | Intron (occurs in LD with 15 SNPs 2 in CTPS1) | 0.123 | CC (n = 47)CT (n = 19)TT (n = 1) | 0.539 ± 0.4900.271 ± 0.2180.36 | CC vs CT + TT, p = 0.038 | Low | Dominant(CC = 0, CT/TT = -1) |
|   |  |  |  |  |  |  |  |  |
| CTPS1 | rs12067645; G >A | Intergenic (occurs in LD with 35 SNPs in 5′ region of CTPS1) | 0.222 | GG (n = 45)AG (n = 20)AA (n = 2) | 0.379 ± 0.3530.602 ± 0.5570.873 ± 0.704 | GG vs AG + AA, p = 0.047 | High | Additive(GG = 0, AG = 1, AA = 2) |
|   |  |  |  |  |  |  |  |  |
| CTPS1 | rs11577910; G >A | 3′UTR (occurs in LD with 12 SNPs in CTPS1) | 0.093 | GG (n = 61)AG (n = 7) | 0.425 ± 0.4330.732 ± 0.439 | GG vs AG, p = 0.045 | High | (GG = 0, AG = 1) |
|   |  |  |  |  |  |  |  |  |
| CDA | rs12404655; A > G | Occurs in LD with 4 SNPs in 3′ region and 6 in CDA introns | 0.21 | AA (n = 44)AG (n = 21)GG (n = 3) | 0.372 ± 0.3620.642 ± 0.5550.385 ± 0.269 | AA vs AG + GG; p = 0.017 | High | Dominant(AA = 0, AG/GG = 1) |
|   |  |  |  |  |  |  |  |  |
| RRM1 | rs11030918†; T >C | 5′ UTR (occurs in LD with 2-5′UTR one 3′UTR and 19 intronic SNPs in RRM1) | 0.317 | TT (n = 25)CT (n = 32)CC (n = 9) | 0.396 ± 0.326 0.354 ± 0.348 0.785 ± 0.411 | CC vs CT + TT; p = 0.0025 | High | Recessive(TT/CT = 0, CC = 1) |
|   |  |  |  |  |  |  |  |  |
| SLC28A3 | rs17343066; G >A | Occurs in LD with 7 intronic SNPs in SLC28A3 | 0.457 | GG (n = 20)AG (n = 34)AA (n = 12) | 0.558 ± 0.649 0.361 ± 0.311 0.591 ± 0.293 | AA vs AG + GG, p = 0.028 | High | Recessive(GG/AG = 0, AA = 1) |

Table 2 Caption: †SNPs previously reported [10,11]: LD = Linkage disequilibrium: data from Haploreg database (Version4.1) variants with r2 ≥ 0.8 are indicated in third column (http://archive.broadinstitute.org/mammals/haploreg/haploreg.php)

SNPs with significant association of variant allele with higher ara-CTP level included: *SLC28A3* SNP rs17343066 G >A, which occurs in LD with 7 other *SLC28A3* SNPs (AA vs AG + GG; p = 0.028); rs12067645 G >A (GG vs AG + AA; p = 0.047) which occurs in linkage with multiple promoter and intronic SNP in *CTPS1* and another 3′UTR *CTPS1* SNP rs11577910 (GG vs AG p = 0.045); *CDA* SNP rs12404655 A >G (AA vs AG + GG p = 0.017), which occurs in LD with six intronic and four 3′UTR SNPs within *CDA* and previously reported SNP in *RRM1* rs11030918 (occurs in LD with at least 22 other SNPs within *RRM1* ^10^). Box plots of association of these SNPs with intracellular levels of ara-CTP are shown in [Supplementary Figure 1](http:///doi/suppl/10.2217/pgs-2018-0086/suppl_file/pgs-19-1101-s1.pptx).

### Cytarabine score determination using generalized linear models

We evaluated all 88 SNPs using linear models for any combination of two SNPs with the most significant prediction of leukemic cell cytarabine levels at day 1. As shown in [Table 3](#T3), our results identified three models for two SNP combinations (that included four unique SNPs) with significant association with ara-CTP levels. These models included combinations of *DCK*-rs4643786 and *RRM1*-rs11030918 in model 1; *DCK*-rs4643786 and *CTPS1*-rs12067645 in model 2 and *RRM1*-rs11030918 and *SLC28A3*-rs17343066 in model 3. [Figure 2](#F0002) shows distribution of the genotypes for these four SNPs across patient's sorted by low to high levels of ara-CTP. A comprehensive score was created using genotype information for four SNPs that passed the models. Directional genotype score was provided for each SNP in each patient after consideration for direction of association of variant allele with ara-CTP levels as well as the potential mode of inheritance. Addition of directional genotype scores resulted in composite ACSS ranging from -2 to 3 with negative score indicating more number of low-ara-CTP associated alleles and positive score indicating more number of high ara-CTP associated alleles. The score was further compressed to classify patients into three groups low-ACSS (L-ACSS) with scores <0 (ara-CTP levels 0.133 ± 0.150 nmol/2 × 10^7^ cells); intermediate-ACSS (I-ACSS) group with score = 0 (ara-CTP levels 0.342 ± 0.324 nmol/2 × 10^7^ cells) and high-ACSS (H-ACSS) group with scores >0 (ara-CTP levels 0.660 ± 0.508 nmol/2 × 10^7^ cells). ACSS score demonstrated significant association with intracellular levels of ara-CTP (p = 0.0001) ([Table 4](#T4) & [Figure 3](#F0003)). Further, the ACSS could explain approximately 30% of observed variation in ara-CTP levels which was significantly higher than contribution from any other covariates such as race, treatment arm, cytogenetic-risk group and gender, which together explained only approximately 8% of the observed variation ([Table 5](#T5)). Although day 1, ara-CTP levels was the primary focus of this study, we observed similar association between ACSS score and intracellular ara-CTP levels in samples obtained on day 2 ([Supplementary Figure 2](http:///doi/suppl/10.2217/pgs-2018-0086/suppl_file/pgs-19-1101-s1.pptx)).

### Table 3. . Top three models for two SNP combinations that were significantly associated with ara-CTP levels at day 1.

| Model | Coefficient | p-value | BIC-wt | P.perm |
| --- | --- | --- | --- | --- |
| rs4643786 (DCK) + rs11030918 (RRM1) | -0.3390.383 | 0.0060.009 | 0.038097 | 0.004 |
|   |  |  |  |  |
| rs4643786 (DCK) + rs12067645 (CTPS1) | -0.420.228 | 0.00090.016 | 0.030693 | 0.001 |
|   |  |  |  |  |
| rs11030918 (RRM1) + rs17343066 (SLC28A3) | 0.5210.406 | 0.00050.0022 | 0.030467 | 0 |

Table 3 Caption: BIC: Bayesian information criterion.

### Figure 2. . Genotype HeatMap showing distribution of genotypes for four SNPs across patients.

![Figure 2. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0c76/6219441/43975f77ce65/pgs-19-1101-g2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6219441_pgs-19-1101-g2.jpg)

The patients (n = 68) are arranged from low to high levels of intracellular ara-CTP (the levels are depicted as pmol/2 × 107 cells). Genotypes with significant association of variant allele with low levels of ara-CTP are depicted in red and those with significant associated with higher ara-CTP are depicted in blue, grey boxes reflect wild type genotype.

### Table 4. . Association of ara-C SNP score group with intracellular ara-CTP levels.

| ACSS groups | Ara-CTP level by ACSS group at day 1 (nmol/2 × 107 cells) |  |  |
| --- | --- | --- | --- |
|   |  |  |  |
|   | Score | Mean ± SD | p-value |
| L-ACSS | <0 (n = 7) | 0.133 ± 0.150 | 1.20E-04 |
|   |  |  |  |
| I-ACSS | 0 (n = 32) | 0.342 ± 0.324 |   |
|   |  |  |  |
| H-ACSS | >0 (n = 29) | 0.660 ± 0.508 |   |

Table 4 Caption: ACSS: Ara-C SNP score.

### Figure 3. . Association of comprehensive ara-C SNP score with intracellular levels of active ara-CTP (Log transformed ata-CTP pmol/2 × 107 cells).

![Figure 3. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0c76/6219441/c6f5cc2e431d/pgs-19-1101-g3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6219441_pgs-19-1101-g3.jpg)

Box plot showing association of low-ACSS (score <0), intermediate-ACSS (score = 0) and high-ACSS (score > 0) score groups with intracellular ara-CTP levels obtained at day 1 post-initiation of ara-C infusion. ACSS: Ara-C SNP score.

### Table 5. . Ara-C SNP score group classification explains majority of interpatient variation in ara-CTP as compared with other factors.

| Independent variable | Variability in day 1 intracellular ara-CTP levels explained | R-squared |
| --- | --- | --- |
| ACSS score groups | 0.302 | 0.302 |
|   |  |  |
| Race | 0.027 | 0.329 |
|   |  |  |
| Arm | 0.032 | 0.361 |
|   |  |  |
| Cytogenetic-risk group | 0.007 | 0.368 |
|   |  |  |
| Gender | 0.007 | 0.375 |

Table 5 Caption: CSS score groups are given in bold to emphasize its importance in explaining 30 out of 37% of total variablity in ara-CTP levels. ACSS: Ara-C SNP score.

## Discussion

Cytarabine has been used for AML treatment for more than 40 years. It is a prodrug that requires uptake to cells and activation by series of phosphorylation steps to form the active form of the drug, ara-CTP, which competes with dCTP for incorporation into DNA and inhibits DNA synthesis, and triggers apoptosis. Thus, intracellular abundance of ara-CTP is critical for its therapeutic efficacy [[19](#B19)] and at the same time higher levels might be contributing to observed toxicity and adverse events. Genetic variation in genes involved in ara-C uptake, activation of ara-CTP or inactivation as well as genes involved in regulating pools of dCTP can impact its intracellular levels of ara-CTP and thus its therapeutic efficacy. Several studies on association of SNPs within ara-C metabolic pathway genes with clinical end points have been reported [[10–14](#B10),[16](#B16),[20–30](#B20)]; however, there is lack of information on direct association of ara-C pathway SNPs with abundance of ara-CTP within leukemic cells of AML patients. One of biggest challenges associated with designing such a study is obtaining bone marrow aspirates post cytarabine infusion, which is not a standard clinical procedure in the treatment of AML. We have previously reported SNPs in select genes *DCK, NT5C2* and *RRM1/RRM2* to be associated with intracellular ara-CTP levels in patients treated on St Jude AML97 clinical trial [[10–12](#B10)]. In the current study, we performed a comprehensive evaluation of 14 genes and 88 SNPs involved in ara-C pathway.

We observed SNPs in two influx transporters: *SLC28A1* (rs11853372 G >T) with presence of variant allele predictive of low-intracellular ara-CTP levels and *SLC28A3* rs17343066 G >A with variant allele associated with higher ara-CTP levels. The 3′UTR SNP in *DCK* rs4643786 T >C has been previously been associated with lower *DCK* mRNA levels and intracellular ara-CTP levels in a smaller cohort of patients [[11](#B11)]; however in contrast to this, it has been associated with higher remission rate in adult Chinese AML patients [[20](#B20)]. *CDA* and *DCTD* both are involved in deamination and convert ara-C >ara-U and ara-CMP >ara-UMP, respectively. A synonymous SNP rs4742 C >T in DCTD occurs in high LD with at least 17 other SNPs on *DCTD* locus and predicted lower ara-CTP levels where as *CDA* SNP was associated with high ara-CTP levels. Among the genes involved in regulating cellular pools of CTP, one SNP in *RRM1* and three SNPs in *CTPS1* showed significant association with intracellular levels of ara-CTP. *RRM1* codes for ribonucleotide reductase and *CTPS1* encodes for cytidine-triphosphate synthase and both play critical role in pyrimidine synthesis and maintaining cellular pools of dCTP. Intracellular dCTP can impact ara-CTP production and its efficacy by feedback inhibition of *DCK*, by allosteric activation of the inactivating enzyme *CDA* and by competing with the ara-CTP for incorporation into DNA. Although, the functional relevance of these SNPs still needs to be investigated, majority of the significant SNPs occurred in LD with multiple SNPs on the respective gene locus, warranting in-depth functional characterization of these and potential linked SNPs. As an exploratory evaluation for the potential functional impact of the nine SNPs significantly associated with we utilized the Genotype-Tissue Expression (GTEx) project ([www.gtexportal.org](http://www.gtexportal.org)). Of the nine SNPs, seven were represented in the GTEx databse. The single tissue (whole blood) expression quantitative trait loci (eQTL) analysis from GTEx database showed association of *CDA* SNP rs12404655 A >G with lower *CDA* mRNA expression which is consistent with G allele predictive of high ara-CTP concentration. Interestingly for *CTPS1* SNPs, variant alleles for rs12067645 G >A, and rs11577910 C >T with higher and for rs4364871 C >T with lower mRNA levels, which was not consistent with the direction of association of these SNPs with ara-CTP levels. Although this requires further investigation, *CTPS1* gene has 12 alternatively spliced transcripts, which might interfere with accurate mRNA quantification of the wild-type reference transcript. Similarly, *RRM1* SNP rs11030918 T >C demonstrated association with the higher mRNA levels in whole blood, and was associated with higher ara-CTP levels. Although molecular mechanisms underlying this observation require further investigation, previous report has shown that higher *RRM1* expression is associated with higher ara-C sensitivity as well as better DFS [[2](#B2)]. *SLC28A3* and *DCTD* SNPs were not significantly associated with the respective genes mRNA levels.

As multiple SNPs can co-exist in a patient, we evaluated influence of SNP combinations on intracellular ara-CTP levels, three top two SNP models with significant impact on ara-CTP levels consisted of four unique SNPs. Thus, we developed a comprehensive ara-C SNP risk score (ACSS) for each patient by taking into account the directional genotype and inheritance mode for each of the four SNPs. Our results show that the comprehensive ACSS has improved power to significantly predict leukemic cell intracellular levels of ara-CTP and explained approximately 30% of variation observed in intracellular levels of ara-CTP. Given the ACSS score subclassifies patients into three groups: low-ACSS, intermediate-ACSS and high-ACSS independent of patient demographics and risk groups, once validated this score has a potential to identify patients with potentially lower levels ara-CTP (L-ACSS group) in the leukemic cells with high risk of treatment failure. Conversely, patients with H-ACSS might be at higher risk of ara-C related side effects.

The uniqueness of AML97 study is the availability of bone marrow aspirates from day 1 and day 2 post ara-C infusion allowing for intracellular ara-CTP level quantification; however, one of the limitations in extending our analysis to outcome was differences duration between initiation of course 2 in treatment arms, in timing of chemotherapy and two major amendments during the course of the study that influenced clinical outcome (described in detail in Rubnitz *et al*. [[17](#B17)]). Due to these reasons for the purpose of this study, we primarily focused our evaluation on intracellular ara-CTP levels obtained at day 1 post ara-C infusion. Since patients received only ara-C as a single agent on the first day of remission induction therapy, so ara-CTP measurements at this time-point were free of potentially confounding effects of other agents.

## Conclusion & future perspectives

ACSS developed using potentially relevant SNPs in ara-C metabolic pathway can predict intracellular levels of ara-CTP in leukemic cells. Future studies are needed to validate the clinical utility of ACSS in predicting response and toxicity after induction treatment. Although, current AML chemotherapeutic regimens use low- and high-dose of ara-C, these are not guided by pharmacogenomics. Once validated the ACSS holds potential for designing individualized ara-C dosing for maximizing benefit for the patient. Our results open up opportunities to pair L-ACSS score with high-dose ara-C, H-ACSS with low-dose and I-ACSS with intermediate dose of ara-C to treat AML patients.

#### Purpose of the study

#### Purpose of the study

#### Purpose of the study

#### Results

#### Conclusion

## Supplementary Material

## Acknowledgements

The authors would like to acknowledge the support provided by Biomedical Genomics Center (BMGC) and Minnesota Supercomputing Institute at the University of Minnesota.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Khwaja A, Bjorkholm M, Gale RE, et al. Acute myeloid leukaemia. Nat. Rev. Dis. Primers. 2016;2:16010. doi: 10.1038/nrdp.2016.10.  [DOI](https://doi.org/10.1038/nrdp.2016.10) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27159408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Dis.%20Primers&title=Acute%20myeloid%20leukaemia&author=A%20Khwaja&author=M%20Bjorkholm&author=RE%20Gale&volume=2&publication_year=2016&pages=16010&pmid=27159408&doi=10.1038/nrdp.2016.10&)

2. Abraham A, Varatharajan S, Karathedath S, et al. RNA expression of genes involved in cytarabine metabolism and transport predicts cytarabine response in acute myeloid leukemia. Pharmacogenomics. 2015;16:877–890. doi: 10.2217/pgs.15.44.  [DOI](https://doi.org/10.2217/pgs.15.44) | [PMC free article](/articles/PMC7115907/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26083014/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=RNA%20expression%20of%20genes%20involved%20in%20cytarabine%20metabolism%20and%20transport%20predicts%20cytarabine%20response%20in%20acute%20myeloid%20leukemia&author=A%20Abraham&author=S%20Varatharajan&author=S%20Karathedath&volume=16&publication_year=2015&pages=877-890&pmid=26083014&doi=10.2217/pgs.15.44&)

3. Rowe JM. AML in 2016: where we are now? Best Pract. Res. Clin. Haematol. 2016;29:315–319. doi: 10.1016/j.beha.2016.10.001.  [DOI](https://doi.org/10.1016/j.beha.2016.10.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27890252/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Best%20Pract.%20Res.%20Clin.%20Haematol.&title=AML%20in%202016:%20where%20we%20are%20now?&author=JM%20Rowe&volume=29&publication_year=2016&pages=315-319&pmid=27890252&doi=10.1016/j.beha.2016.10.001&)

4. Ravandi F. Relapsed acute myeloid leukemia: why is there no standard of care? Best Pract. Res. Clin. Haematol. 2013;26:253–259. doi: 10.1016/j.beha.2013.10.005.  [DOI](https://doi.org/10.1016/j.beha.2013.10.005) | [PMC free article](/articles/PMC4090680/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24309527/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Best%20Pract.%20Res.%20Clin.%20Haematol.&title=Relapsed%20acute%20myeloid%20leukemia:%20why%20is%20there%20no%20standard%20of%20care?&author=F%20Ravandi&volume=26&publication_year=2013&pages=253-259&pmid=24309527&doi=10.1016/j.beha.2013.10.005&)

5. Braess J. [Acute myeloid Leukemia] Dtsch Med. Wochenschr. 2016;141:1748–1751. doi: 10.1055/s-0042-112505.  [DOI](https://doi.org/10.1055/s-0042-112505) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27835920/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dtsch%20Med.%20Wochenschr.&title=%5BAcute%20myeloid%20Leukemia%5D&author=J%20Braess&volume=141&publication_year=2016&pages=1748-1751&pmid=27835920&doi=10.1055/s-0042-112505&)

6. Short NJ, Ravandi F. Acute myeloid leukemia: past, present, and prospects for the future. Clin. Lymphoma Myeloma Leuk. 2016;16(Suppl.):S25–S29. doi: 10.1016/j.clml.2016.02.007.  [DOI](https://doi.org/10.1016/j.clml.2016.02.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27521321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Lymphoma%20Myeloma%20Leuk.&title=Acute%20myeloid%20leukemia:%20past,%20present,%20and%20prospects%20for%20the%20future&author=NJ%20Short&author=F%20Ravandi&volume=16&issue=Suppl.&publication_year=2016&pages=S25-S29&pmid=27521321&doi=10.1016/j.clml.2016.02.007&)

7. Kufe DW, Munroe D, Herrick D, Egan E, Spriggs D. Effects of 1-beta-D-arabinofuranosylcytosine incorporation on eukaryotic DNA template function. Mol. Pharmacol. 1984;26:128–134.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/6431261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=Effects%20of%201-beta-D-arabinofuranosylcytosine%20incorporation%20on%20eukaryotic%20DNA%20template%20function&author=DW%20Kufe&author=D%20Munroe&author=D%20Herrick&author=E%20Egan&author=D%20Spriggs&volume=26&publication_year=1984&pages=128-134&pmid=6431261&)

8. Chou TC, Arlin Z, Clarkson BD, Phillips FS. Metabolism of 1-beta-D-arabinofuranosylcytosine in human leukemic cells. Cancer Res. 1977;37:3561–3570.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/332337/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=Metabolism%20of%201-beta-D-arabinofuranosylcytosine%20in%20human%20leukemic%20cells&author=TC%20Chou&author=Z%20Arlin&author=BD%20Clarkson&author=FS%20Phillips&volume=37&publication_year=1977&pages=3561-3570&pmid=332337&)

9. Verschuur AC, Brinkman J, Van Gennip AH, et al. Cyclopentenyl cytosine induces apoptosis and increases cytarabine-induced apoptosis in a T-lymphoblastic leukemic cell-line. Leuk. Res. 2001;25:891–900. doi: 10.1016/s0145-2126(01)00047-9.  [DOI](https://doi.org/10.1016/s0145-2126(01)00047-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11532523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk.%20Res.&title=Cyclopentenyl%20cytosine%20induces%20apoptosis%20and%20increases%20cytarabine-induced%20apoptosis%20in%20a%20T-lymphoblastic%20leukemic%20cell-line&author=AC%20Verschuur&author=J%20Brinkman&author=AH%20Van%20Gennip&volume=25&publication_year=2001&pages=891-900&pmid=11532523&doi=10.1016/s0145-2126(01)00047-9&)

10. Cao X, Mitra AK, Pounds S, et al. RRM1 and RRM2 pharmacogenetics: association with phenotypes in HapMap cell lines and acute myeloid leukemia patients. Pharmacogenomics. 2013;14:1449–1466. doi: 10.2217/pgs.13.131.  [DOI](https://doi.org/10.2217/pgs.13.131) | [PMC free article](/articles/PMC3860742/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24024897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=RRM1%20and%20RRM2%20pharmacogenetics:%20association%20with%20phenotypes%20in%20HapMap%20cell%20lines%20and%20acute%20myeloid%20leukemia%20patients&author=X%20Cao&author=AK%20Mitra&author=S%20Pounds&volume=14&publication_year=2013&pages=1449-1466&pmid=24024897&doi=10.2217/pgs.13.131&)

11. Lamba JK, Crews K, Pounds S, et al. Pharmacogenetics of deoxycytidine kinase: identification and characterization of novel genetic variants. J. Pharmacol. Exp. Ther. 2007;323:935–945. doi: 10.1124/jpet.107.128595.  [DOI](https://doi.org/10.1124/jpet.107.128595) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17855478/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=Pharmacogenetics%20of%20deoxycytidine%20kinase:%20identification%20and%20characterization%20of%20novel%20genetic%20variants&author=JK%20Lamba&author=K%20Crews&author=S%20Pounds&volume=323&publication_year=2007&pages=935-945&pmid=17855478&doi=10.1124/jpet.107.128595&)

12. Mitra AK, Crews KR, Pounds S, et al. Genetic variants in cytosolic 5′-nucleotidase II are associated with its expression and cytarabine sensitivity in HapMap cell lines and in patients with acute myeloid leukemia. J. Pharmacol. Exp. Ther. 2011;339:9–23. doi: 10.1124/jpet.111.182873.  [DOI](https://doi.org/10.1124/jpet.111.182873) | [PMC free article](/articles/PMC3186292/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21712425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=Genetic%20variants%20in%20cytosolic%205%E2%80%B2-nucleotidase%20II%20are%20associated%20with%20its%20expression%20and%20cytarabine%20sensitivity%20in%20HapMap%20cell%20lines%20and%20in%20patients%20with%20acute%20myeloid%20leukemia&author=AK%20Mitra&author=KR%20Crews&author=S%20Pounds&volume=339&publication_year=2011&pages=9-23&pmid=21712425&doi=10.1124/jpet.111.182873&)

13. Amaki J, Onizuka M, Ohmachi K, et al. Single nucleotide polymorphisms of cytarabine metabolic genes influence clinical outcome in acute myeloid leukemia patients receiving high-dose cytarabine therapy. Int. J. Hematol. 2015;101:543–553. doi: 10.1007/s12185-015-1766-4.  [DOI](https://doi.org/10.1007/s12185-015-1766-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25735499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Hematol.&title=Single%20nucleotide%20polymorphisms%20of%20cytarabine%20metabolic%20genes%20influence%20clinical%20outcome%20in%20acute%20myeloid%20leukemia%20patients%20receiving%20high-dose%20cytarabine%20therapy&author=J%20Amaki&author=M%20Onizuka&author=K%20Ohmachi&volume=101&publication_year=2015&pages=543-553&pmid=25735499&doi=10.1007/s12185-015-1766-4&)

14. Kim KI, Huh IS, Kim IW, et al. Combined interaction of multi-locus genetic polymorphisms in cytarabine arabinoside metabolic pathway on clinical outcomes in adult acute myeloid leukaemia (AML) patients. Eur. J. Cancer. 2013;49:403–410. doi: 10.1016/j.ejca.2012.07.022.  [DOI](https://doi.org/10.1016/j.ejca.2012.07.022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22964418/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Cancer&title=Combined%20interaction%20of%20multi-locus%20genetic%20polymorphisms%20in%20cytarabine%20arabinoside%20metabolic%20pathway%20on%20clinical%20outcomes%20in%20adult%20acute%20myeloid%20leukaemia%20(AML)%20patients&author=KI%20Kim&author=IS%20Huh&author=IW%20Kim&volume=49&publication_year=2013&pages=403-410&pmid=22964418&doi=10.1016/j.ejca.2012.07.022&)

15. Lamba JK. Pharmacogenomics of cytarabine in childhood leukemia. Pharmacogenomics. 2011;12:1629–1632. doi: 10.2217/pgs.11.148.  [DOI](https://doi.org/10.2217/pgs.11.148) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22118048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenomics%20of%20cytarabine%20in%20childhood%20leukemia&author=JK%20Lamba&volume=12&publication_year=2011&pages=1629-1632&pmid=22118048&doi=10.2217/pgs.11.148&)

16. Lamba JK. Genetic factors influencing cytarabine therapy. Pharmacogenomics. 2009;10:1657–1674. doi: 10.2217/pgs.09.118.  [DOI](https://doi.org/10.2217/pgs.09.118) | [PMC free article](/articles/PMC2828057/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19842938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genetic%20factors%20influencing%20cytarabine%20therapy&author=JK%20Lamba&volume=10&publication_year=2009&pages=1657-1674&pmid=19842938&doi=10.2217/pgs.09.118&)

17. Rubnitz JE, Crews KR, Pounds S, et al. Combination of cladribine and cytarabine is effective for childhood acute myeloid leukemia: results of the St Jude AML97 trial. Leukemia. 2009;23:1410–1416. doi: 10.1038/leu.2009.30.  [DOI](https://doi.org/10.1038/leu.2009.30) | [PMC free article](/articles/PMC2726271/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19242495/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Combination%20of%20cladribine%20and%20cytarabine%20is%20effective%20for%20childhood%20acute%20myeloid%20leukemia:%20results%20of%20the%20St%20Jude%20AML97%20trial&author=JE%20Rubnitz&author=KR%20Crews&author=S%20Pounds&volume=23&publication_year=2009&pages=1410-1416&pmid=19242495&doi=10.1038/leu.2009.30&)

18. Crews KR, Gandhi V, Srivastava DK, et al. Interim comparison of a continuous infusion versus a short daily infusion of cytarabine given in combination with cladribine for pediatric acute myeloid leukemia. J. Clin. Oncol. 2002;20:4217–4224. doi: 10.1200/JCO.2002.10.006.  [DOI](https://doi.org/10.1200/JCO.2002.10.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12377965/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Interim%20comparison%20of%20a%20continuous%20infusion%20versus%20a%20short%20daily%20infusion%20of%20cytarabine%20given%20in%20combination%20with%20cladribine%20for%20pediatric%20acute%20myeloid%20leukemia&author=KR%20Crews&author=V%20Gandhi&author=DK%20Srivastava&volume=20&publication_year=2002&pages=4217-4224&pmid=12377965&doi=10.1200/JCO.2002.10.006&)

19. Kufe D, Spriggs D, Egan EM, Munroe D. Relationships among Ara-CTP pools, formation of (Ara-C)DNA, and cytotoxicity of human leukemic cells. Blood. 1984;64:54–58.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/6587917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Relationships%20among%20Ara-CTP%20pools,%20formation%20of%20(Ara-C)DNA,%20and%20cytotoxicity%20of%20human%20leukemic%20cells&author=D%20Kufe&author=D%20Spriggs&author=EM%20Egan&author=D%20Munroe&volume=64&publication_year=1984&pages=54-58&pmid=6587917&)

20. Zhang DY, Yuan XQ, Yan H, et al. Association between DCK 35708 T >C variation and clinical outcomes of acute myeloid leukemia in South Chinese patients. Pharmacogenomics. 2016;17:1519–1531. doi: 10.2217/pgs-2016-0084.  [DOI](https://doi.org/10.2217/pgs-2016-0084) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27548009/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Association%20between%20DCK%2035708%20T%20>C%20variation%20and%20clinical%20outcomes%20of%20acute%20myeloid%20leukemia%20in%20South%20Chinese%20patients&author=DY%20Zhang&author=XQ%20Yuan&author=H%20Yan&volume=17&publication_year=2016&pages=1519-1531&pmid=27548009&doi=10.2217/pgs-2016-0084&)

21. Xu PP, Chen BA, Feng JF, et al. Association of polymorphisms of cytosine arabinoside-metabolizing enzyme gene with therapeutic efficacy for acute myeloid leukemia. Chin. Med. J. 2012;125:2137–2143.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/22884143/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chin.%20Med.%20J.&title=Association%20of%20polymorphisms%20of%20cytosine%20arabinoside-metabolizing%20enzyme%20gene%20with%20therapeutic%20efficacy%20for%20acute%20myeloid%20leukemia&author=PP%20Xu&author=BA%20Chen&author=JF%20Feng&volume=125&publication_year=2012&pages=2137-2143&pmid=22884143&)

22. Wan H, Zhu J, Chen F, et al. SLC29A1 single nucleotide polymorphisms as independent prognostic predictors for survival of patients with acute myeloid leukemia: an in vitro study. J. Exp. Clin. Cancer Res. 2014;33:90. doi: 10.1186/s13046-014-0090-9.  [DOI](https://doi.org/10.1186/s13046-014-0090-9) | [PMC free article](/articles/PMC4234887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25398670/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Clin.%20Cancer%20Res.&title=SLC29A1%20single%20nucleotide%20polymorphisms%20as%20independent%20prognostic%20predictors%20for%20survival%20of%20patients%20with%20acute%20myeloid%20leukemia:%20an%20in%20vitro%20study&author=H%20Wan&author=J%20Zhu&author=F%20Chen&volume=33&publication_year=2014&pages=90&pmid=25398670&doi=10.1186/s13046-014-0090-9&)

23. Mahlknecht U, Dransfeld CL, Bulut N, et al. SNP analyses in cytarabine metabolizing enzymes in AML patients and their impact on treatment response and patient survival: identification of CDA SNP C-451T as an independent prognostic parameter for survival. Leukemia. 2009;23:1929–1932. doi: 10.1038/leu.2009.113.  [DOI](https://doi.org/10.1038/leu.2009.113) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19458626/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=SNP%20analyses%20in%20cytarabine%20metabolizing%20enzymes%20in%20AML%20patients%20and%20their%20impact%20on%20treatment%20response%20and%20patient%20survival:%20identification%20of%20CDA%20SNP%20C-451T%20as%20an%20independent%20prognostic%20parameter%20for%20survival&author=U%20Mahlknecht&author=CL%20Dransfeld&author=N%20Bulut&volume=23&publication_year=2009&pages=1929-1932&pmid=19458626&doi=10.1038/leu.2009.113&)

24. Kim SR, Saito Y, Maekawa K, et al. Twenty novel genetic variations and haplotype structures of the DCK gene encoding human deoxycytidine kinase (dCK) Drug Metab. Pharmacokinet. 2008;23:379–384. doi: 10.2133/dmpk.23.379.  [DOI](https://doi.org/10.2133/dmpk.23.379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18974616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Pharmacokinet.&title=Twenty%20novel%20genetic%20variations%20and%20haplotype%20structures%20of%20the%20DCK%20gene%20encoding%20human%20deoxycytidine%20kinase%20(dCK)&author=SR%20Kim&author=Y%20Saito&author=K%20Maekawa&volume=23&publication_year=2008&pages=379-384&pmid=18974616&doi=10.2133/dmpk.23.379&)

25. Kim JH, Lee C, Cheong HS, et al. SLC29A1 (ENT1) polymorphisms and outcome of complete remission in acute myeloid leukemia. Cancer Chemother. Pharmacol. 2016;78:533–540. doi: 10.1007/s00280-016-3103-x.  [DOI](https://doi.org/10.1007/s00280-016-3103-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27422302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother.%20Pharmacol.&title=SLC29A1%20(ENT1)%20polymorphisms%20and%20outcome%20of%20complete%20remission%20in%20acute%20myeloid%20leukemia&author=JH%20Kim&author=C%20Lee&author=HS%20Cheong&volume=78&publication_year=2016&pages=533-540&pmid=27422302&doi=10.1007/s00280-016-3103-x&)

26. Hyo Kim L, Sub Cheong H, Koh Y, et al. Cytidine deaminase polymorphisms and worse treatment response in normal karyotype AML. J. Hum. Genet. 2015;60:749–54. doi: 10.1038/jhg.2015.105.  [DOI](https://doi.org/10.1038/jhg.2015.105) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26354033/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hum.%20Genet.&title=Cytidine%20deaminase%20polymorphisms%20and%20worse%20treatment%20response%20in%20normal%20karyotype%20AML&author=L%20Hyo%20Kim&author=H%20Sub%20Cheong&author=Y%20Koh&volume=60&publication_year=2015&pages=749-54&pmid=26354033&doi=10.1038/jhg.2015.105&)

27. Falk IJ, Fyrberg A, Paul E, et al. Decreased survival in normal karyotype AML with single-nucleotide polymorphisms in genes encoding the AraC metabolizing enzymes cytidine deaminase and 5′-nucleotidase. Am. J. Hematol. 2013;88:1001–1006. doi: 10.1002/ajh.23549.  [DOI](https://doi.org/10.1002/ajh.23549) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23873772/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hematol.&title=Decreased%20survival%20in%20normal%20karyotype%20AML%20with%20single-nucleotide%20polymorphisms%20in%20genes%20encoding%20the%20AraC%20metabolizing%20enzymes%20cytidine%20deaminase%20and%205%E2%80%B2-nucleotidase&author=IJ%20Falk&author=A%20Fyrberg&author=E%20Paul&volume=88&publication_year=2013&pages=1001-1006&pmid=23873772&doi=10.1002/ajh.23549&)

28. Cheong HS, Koh Y, Ahn KS, et al. NT5C3 polymorphisms and outcome of first induction chemotherapy in acute myeloid leukemia. Pharmacogenet. Genomics. 2014;24:436–441. doi: 10.1097/FPC.0000000000000072.  [DOI](https://doi.org/10.1097/FPC.0000000000000072) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25000516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=NT5C3%20polymorphisms%20and%20outcome%20of%20first%20induction%20chemotherapy%20in%20acute%20myeloid%20leukemia&author=HS%20Cheong&author=Y%20Koh&author=KS%20Ahn&volume=24&publication_year=2014&pages=436-441&pmid=25000516&doi=10.1097/FPC.0000000000000072&)

29. Banklau C, Jindadamrongwech S, Sawangpanich R, et al. Effect of genetic alterations of cytarabine- metabolizing enzymes in childhood acute lymphoblastic leukemia. Hematol. Oncol. Stem Cell Ther. 2010;3:103–108. doi: 10.1016/s1658-3876(10)50019-0.  [DOI](https://doi.org/10.1016/s1658-3876(10)50019-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20890066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hematol.%20Oncol.%20Stem%20Cell%20Ther.&title=Effect%20of%20genetic%20alterations%20of%20cytarabine-%20metabolizing%20enzymes%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=C%20Banklau&author=S%20Jindadamrongwech&author=R%20Sawangpanich&volume=3&publication_year=2010&pages=103-108&pmid=20890066&doi=10.1016/s1658-3876(10)50019-0&)

30. Abraham A, Varatharajan S, Abbas S, et al. Cytidine deaminase genetic variants influence RNA expression and cytarabine cytotoxicity in acute myeloid leukemia. Pharmacogenomics. 2012;13:269–282. doi: 10.2217/pgs.11.149.  [DOI](https://doi.org/10.2217/pgs.11.149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22304580/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Cytidine%20deaminase%20genetic%20variants%20influence%20RNA%20expression%20and%20cytarabine%20cytotoxicity%20in%20acute%20myeloid%20leukemia&author=A%20Abraham&author=S%20Varatharajan&author=S%20Abbas&volume=13&publication_year=2012&pages=269-282&pmid=22304580&doi=10.2217/pgs.11.149&)
