# UGT1A1 Genotype-Dependent Dose Adjustment of Belinostat in Patients With Advanced Cancers Using Population Pharmacokinetic Modeling and Simulation

## Metadata
**Authors:** Cody J Peer, Andrew K L Goey, Tristan M Sissung, Sheryl Erlich, Min-Jung Lee, Yusuke Tomita, Jane B Trepel, Richard Piekarz, Sanjeeve Balasubramaniam, Susan E Bates, William D Figg
**Journal:** Journal of clinical pharmacology
**Date:** 2015 Dec 4
**DOI:** [10.1002/jcph.627](https://doi.org/10.1002/jcph.627)
**PMID:** 26637161
**PMCID:** PMC6357964
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6357964/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6357964/pdf/nihms-1008576.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6357964/pdf/nihms-1008576.pdf)

## Abstract

Belinostat is a second-generation zinc-binding histone deacetylase inhibitor that is approved for peripheral T-cell lymphoma and is currently being studied in small cell lung cancer and other advanced carcinomas as a 48-hour continuous intravenous infusion. Belinostat is predominantly metabolized by UGT1A1, which is polymorphic. Preliminary analyses revealed a difference in belinostat clearance based on UGT1A1 genotype. A 2-compartment population pharmacokinetic (PK) model was developed and validated that incorporated the UGT1A1 genotype, albumin, and creatinine clearance on the clearance parameter; body weight was a significant covariate on volume. Simulated doses of 600 and 400 mg/m2/24 h given to patients considered extensive or impaired metabolizers, respectively, provided equivalent AUCs. This model and subsequent simulations supported additional PK/toxicity and pharmacogenomics/toxicity analyses to suggest a UGT1A1 genotype-based dose adjustment to normalize belinostat exposure and allow for more tolerable therapy. In addition, global protein lysine acetylation was modeled with PK and demonstrated a reversible belinostat exposure/response relationship, consistent with previous reports.

Keywords: clinical pharmacology, oncology, population, pharmacokinetics, pharmacometrics, pharmacogenetics

## Methods

### Study Design

This trial ([NCT00926640](https://clinicaltrials.gov/ct2/show/NCT00926640)) was conducted at the NCI in Bethesda, Maryland, and was approved for human studies by the NCI Institutional Review Board. Informed consent was obtained from every patient prior to enrollment. Belinostat was administered as a 48-hour CIVI to adult patients with small cell lung carcinoma and other advanced cancers beginning in the evening on day 1, along with 80 mg/m^2^ cisplatin as a 1-hour intravenous infusion 14 hours into the belinostat CIVI (typically around noon on day 2) and 100 mg/m^2^ etoposide as a 1-hour intravenous infusion given immediately after the cisplatin, as well as that same time on days 3 and 4. Therapy was repeated every 3 weeks. Belinostat doses began at 400 mg/m^2^/24 h, and could be deescalated down to 100 mg/m^2^/24 h or escalated up to 800 mg/m^2^/24 h. Cisplatin and etoposide blood/plasma concentrations were not measured. Patient demographics are presented in [Table 1](#T1). The primary objective of this single-center, single-arm, open-label phase 1 dose-escalation trial was to determine the optimal belinostat dose for phase 2 trials in terms of safety and tolerability with regard to the combination with cisplatin and etoposide. This trial was open to adults (≥18 years old) with confirmed cancers for which there is no known standard therapy capable of extending life expectancy who had to be ≥4 weeks removed from cytotoxic chemotherapy, have an Eastern Cooperative Oncology Group performance status of 0–2, and have at least 3 months of life expectancy with acceptable organ function.

### Table 1.

| Parameter | Value |
| --- | --- |
| Demographic | Median (Range) or n (%) |
|   |  |
| Age (y) | 54.5 (39.8–78.3) |
| Body weight (kg) | 82.4 (50.3–121.5) |
| BSA (m2) | 1.9 (1.2–2.49) |
| Sex (M/F) | 15 (60%)/10 (40%) |
| Ethnicity (white/AA/other) | 21 (84%)/2 (8%)/2 (8%) |
|   |  |
| Laboratory | Median (Range) |
|   |  |
| Albumin (g/dL) | 3.7 (2.1–4.2) |
| Alkaline phosphatase (U/L) | 99 (59–239) |
| Total bilirubin (mg/dL) | 0.3 (0.1–0.6) |
| ALT (U/L) | 28 (8–65) |
| CLCr (mL/min) | 117 (47.9–193.1) |
|   |  |
| Genetic | wt/het/var, n (%) |
|   |  |
| UGT1A1*6a,b | 22 (96%)/0 (0%)/1 (4%) |
| UGT1A1*28 | 11 (44%)/11 (44%)/3 (12%) |
| UGT1A1*60c | 5 (22%)/11 (48%)/7 (30%) |
| UGT1A1 SNP carrierd | 5 (22%)/18 (78%) |

Table 1 Caption: Patient Characteristics at Screening (n = 25)

### Pharmacokinetic Sampling and Sample Bioanalysis

Blood for pharmacokinetic analysis was drawn just before starting the belinostat infusion; 0.5, 1, and 2 hours after the start of belinostat infusion on day 1; immediately prior to (approximately 14 hours post–belinostat start) and following the 1-hour cisplatin CIVI (approximately 15 hours post–belinostat start); immediately prior to (approximately 16 hours post–belinostat start) and following day 1 etoposide CIVI (approximately 17 hours post–belinostat start); immediately prior to (approximately 30 hours post–belinostat start) and following day 2 etoposide CIVI (approximately 31 hours post–belinostat start); at the end of 48 hours of belinostat CIVI and 5, 15, and 30 minutes and 1, 2, 4, 6, 8, and 12 hours after the end of belinostat CIVI. These samples were analyzed for measurement of belinostat plasma concentration using a validated ultra-high-performance liquid chromatography with tandem mass spectrometric detection (LC-MS/MS) assay with a lower limit of quantification (LLOQ) of 5 ng/mL, a dynamic range of 5–1500 ng/mL, and dilution capabilities up to 10 000 ng/mL. Belinostat was extracted from 100 uL of plasma using 1 mL of methyl tert-butyl ether, dried down and resuspended before injection into the LC-MS/MS system. Concentrations below the LLOQ were flagged as below quantifiable limits (BQL), and the Phoenix NLME version of the M3 method (as previously described^20–22^) was applied to reduce potential estimation bias and improve parameter estimate precision.

### Genotyping Analysis

All enrolled patients were genotyped, per protocol, for *UGT1A1***6*, *UGT1A1***28*, and *UGT1A1***60* polymorphisms. Deviation from Hardy-Weinberg equilibrium was assessed by chi-square statistics. The polymorphisms *UGT1A1***6*, *UGT1A1***28*, and *UGT1A1***60* were determined from plasma samples using fragment analysis or direct sequencing. Genomic DNA was extracted from blood samples using either the QiAamp Ultrasens Viral DNA or QIAamp DNA Blood kits (Qiagen). Full sequencing methods can be found in the [Supplemental Information](#SD1).

### Pharmacokinetic Model Development

A population pharmacokinetic (PPK) model using nonlinear mixed-effects (NLME) modeling was developed and validated using Pharsight Phoenix NLME 1.3 (Certara, Cary, North Carolina). A first-order conditional estimation method with Laplacian estimation algorithm was used to provide maximum likelihood estimates for fixed and random effects. A Windows 7–based computer, with a 2.2 GHz Intel Core i7 processor and 16 GB RAM was used to run the modeling and supporting software.

Development of a multicompartment structural model was initially based on previous PK studies on belinostat 30-minute CIVI either alone^5,7^ or in combination with other drugs^6,23^ that demonstrated multiphasic disposition. The between-subject variability (BSV) was described by an exponential function, in which the individual parameter estimate (θ_i_) is the product of the population parameter estimate (θ_POP_) and the exponentiated BSV estimate for that individual (η_i_).

| (θi=θpop∗eηi). | (1) |
| --- | --- |
### Model Selection Criteria

The objective function value (OFV), calculated by Phoenix NLME 1.3 as minus 2 times the log likelihood (−2×LL), was primarily used for model selection; The Aikaike information criterion (AIC) was used for non-nested model comparisons. Using the likelihood ratio test, a significant (alpha = 05) improvement between nested models requires a delta OFV > 3.84, based on an approximately chi-square distribution. Visual inspection of the model included goodness-of-fit plots, such as observed concentrations (dependent variable, DV), population predicted (PRED) and individual predicted (IPRED) concentrations versus time, DV versus PRED, DV versus IPRED, and conditional weighted residuals (CWRES) versus time or IPRED. Quantile-quantile (QQ) plots for η were assessed for all parameters to check the assumption of eta normal distribution. The η-shrinkage for each parameter was also assessed, and the BSV estimates for parameters with high η-shrinkage (>40%) were used with caution because of insufficient individual observations that shrink the individual estimate toward the population estimate.^24^ Covariance between parameters was evaluated using a variance-covariance block matrix. Correlation between continuous covariates was assessed using a scatter matrix plot (R v3.1.3), in which a correlation > 0.5 was considered significant. All statistical analyses were performed in Phoenix NLME or R.

### Analysis of Covariate Models

Patient sex, age, race, body weight, height, body surface area (BSA), alkaline phosphatase, creatinine clearance (CLCR), bilirubin, albumin, SGOT score, alanine transaminase (ALT), and *UGT1A1***28*, and **60* genotype status were covariates available for analysis ([Table 1](#T1)). *UGT1A1* genotype was divided into 2 categories, extensive metabolizers (EMs) or impaired metabolizers (IMs), and was tested as a categorical covariate for fractional change (θ_frc_) when patient was IM (IsVar=1; [equation 2](#FD2)).

| θi=θpop∗(θfrc)IsVar=1∗eηi | (2) |
| --- | --- |
Creatinine clearance and BSA were calculated using Cockcroft-Gault and Dubois and Dubois formulas, respectively, with creatinine clearance values adjusted for standard body size (CLCR×[1.73/BSA]). Continuous covariates were scaled to their median values with an estimable parameter for change (θ_chng_):

| θi=θpop∗(Cov∕Median)θchng∗eηi | (3) |
| --- | --- |
Identification of potential covariate effects on parameters was made by visual and statistical relationships between individual post hoc eta (η) estimates for each parameter and covariate values. Inclusion of covariates was initially performed in a stepwise, univariate forward-inclusion approach based on significant reductions in the OFV for that parameter (*P* < .05). Covariates that significantly improved the base model in a univariate fashion were then added to each other in a multivariate forward-inclusion approach. Once a full model was achieved, a stepwise backward elimination (*P* < .01) approach was used to identify the final model.

### PK Model Evaluation

The ability of the final model to predict the DV was tested by visual predictive checks (VPCs). Parameter estimates and variances were used to simulate data for 23 000 virtual patients (23 patients replicated 1000 times each), and the 5th, 50th, and 95th predicted percentiles were calculated in bins defined by explicit centers at 2, 6, 10, 14, 18, 22, 26, 30, 36, 42,48, 54, 60, and 66 hours, then visually compared with observed data and corresponding distributions. Standard errors and 95% confidence intervals were obtained through a nonparametric bootstrap (n = 1000 replicates). No runs were excluded in the calculation of standard errors and confidence intervals.

### PK Model Simulations

Monte Carlo simulations were performed using Pharsight Phoenix NLME 1.3 (Certara Corp, Cary, North Carolina) as follows. Patients from this data set who were wild type for both *UGT1A1***28* and *UGT1A1***60* or carriers of 1 copy of **28* (n = 5), ultimately referred to as EMs, were given doses of 600 mg/m^2^/24 h, whereas carriers from this data set of 2 copies of *UGT1A1***28* or at least 1 copy of **60* (n = 18), referred to as IMs, were given a dose of 400 mg/m^2^/24 h. The simulation was replicated 10 times. The resulting model-predicted plasma concentrations in each replicate for each patient over the course of 72 hours were analyzed with noncompartmental methods to calculate the AUC_inf_ with the goal of normalizing mean AUC in IMs given the lower dose with mean AUC in EMs given the higher dose. To confirm the accuracy of the simulations, the suggested dose reduction from the Beleodaq prescribing information of 1000 mg/m^2^ as a 30-minute CIVI reduced to 750 mg/m^2^ in patients with *UGT1A1***28* was simulated using the aforementioned genotype stratification using this data set.

### Pharmacodynamic Model Development

Global protein lysine acetylation (AcK) was assessed by multiparametric flow cytometry (MACSQuant, Miltenyi Biotec, Bergisch Gladbach, Germany) as described previously,^25^ and flow cytometric data were analyzed using FlowJo (FlowJo LLC, Ashland, Oregon) single-cell analysis software. AcK data were collected for each patient predose, 12 hours after the start of belinostat, 36 hours after the start of belinostat, and 60 hours after the start of belinostat (12 hours after the end of the belinostat infusion). Fold changes in AcK at 12, 36, and 60 hours were normalized per patient based on predose AcK raw values. These fold changes were used to build an indirect response model based on equations below (model II^26^).

| dR∕dt=kin−kout∗1−[(IMAX∗C)∕C+IC50]∗RRMAX=R0∕(1−IMAX) |
| --- |
The fixed effects of the final PPK model were fixed, but random effects were allowed to be estimated, during PD model development and validation because of model complexity. A VPC was conducted with 500 replicates, and 200 nonparametric bootstraps were performed to obtain standard errors for the PD parameters. No covariate model was attempted on the PD data, as no covariate appeared to demonstrate an influence on any of the 4 PD parameters (I_max_, IC_50_, k_in_, and k_out_). Model-predicted PK and PD output, that is, belinostat plasma concentrations and fold change in AcK, were graphed together to depict an exposure/response relationship.

## Results

### Data Set

Belinostat plasma concentration-versus-time data were available from 25 patients with full pharmacokinetic data available, encompassing 546 samples. Three of these 25 patients had their belinostat infusion cut short (end of infusion at 36–38 hours) because of the development of treatment-related toxicities. Full demographic data are available in [Table 1](#T1). Two patients did not have *UGT1A1***60* or *UGT1A1***6* genotype information available because of PCR failure; 22 of 23 patients with *UGT1A1***6* data were wild type. Therefore, *UGT1A1***6* was eliminated as a potential covariate in this analysis because of insufficient numbers of patients carrying variant alleles. The 2 patients without **6* or **60* data were ultimately excluded from analysis, bringing the data set to 23 patients encompassing 475 sampling points. Concentration values up to 2 hours after the end of infusion were largely above the LLOQ (5 ng/mL), with only 1 of 22 available samples (4.5%) BQL, as opposed to the samples 4, 6, 8, and 12 hours after the end of infusion having 64%, 90%, 95%, and 95% BQL, respectively. The Phoenix NLME version of the M3 method, which uses a cumulative distribution function to calculate likelihoods for BQL-censored data, was applied to improve precision of parameter estimates and reduce potential bias. The final data set consisted of 399 uncensored samples and 76 censored samples from 23 patients ([Figure 1](#F1)).

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5977/6357964/0dac7f890266/nihms-1008576-f0001.jpg)

Belinostat plasma concentration versus time profiles. Measured belinostat plasma concentrations (n = 475) from 23 patients included in the final data set. Extensive metabolizers (blue squares) had visibly lower plasma levels during the infusion compared with impaired metabolizers (black circles); top, y-axis cropped at 0.02 mg/L to not count the patients with early infusion termination. Both demonstrated rapid elimination (bottom, log scale). A LOWESS smoothing curve was applied to each group to demonstrate trends in the data.

### Base PK Model Development

A 3-compartment structural model did not significantly improve the model (*P* = .996, *df* = 4) compared with a 2-compartment structural model. The differential equations below describe the base structural PK model, where Q represents intercompartmental clearance between the central compartment and peripheral compartment, C is the predicted concentration, A is the drug amount, and V is the volume of distribution:

| dC1dt=−CL∗C1−Q(C1−C2) | (4) |
| --- | --- |
| dC2dt=−Q(C2−C1) | (5) |
| --- | --- |
| C1=A1∕V1C2=A2∕V2 | (6) |
| --- | --- |
A proportional residual unexplained variability (RUV) model best fit the data, where ε was an estimable parameter for the proportional error with a point estimate and standard error ([Table 2](#T2)).

### Table 2.

| Parameter | Point Estimate (%CV) | Bootstrap Estimatea (%CV) | Bootstrap 95%CI |
| --- | --- | --- | --- |
| CL (L/h) | 104.14 (0.82%) | 106.03 (5.47%) | 97.35–123.5 |
| BSV CL (%) | 32.11b (0.03%) | 30.21 (0.06%) | — |
| VI (L) | 32.24 (1.17%) | 31.82 (10.04%) | 24.07–39.57 |
| BSV VI (%) | 50.45 (0.55%) | 50.45 (0.08%) | — |
| Q (L/h) | 7.09 (1.13%) | 7.84 (23.19%) | 6.61–12.98 |
| BSV (% Q) | 92.53 (0.73%) | 93.59 (0.23%) | — |
| V2 (L) | 8.75 (1.60%) | 8.91 (11.05%) | 7.34–11.67 |
| BSV V2 (%) | 33.35 (0.37%) | 34.14 (0.09%) | — |
|   |  |  |  |
| CLCR on CLc | 0.3791 (1.29%) | 0.3799 (10.29%) | 0.2858–0.4613 |
| UGT genotype on CLd | 0.8615 (1.27%) | 0.8475 (5.34%) | 0.7337–0.9062 |
| Albumin on CLe | −0.3743 (1.15%) | −0.3897 (16.66%) | −0.5793 to −0.3520 |
| Body weight on Vf | 0.3965 (0.12%) | 0.3848 (22.48%) | 0.2314–0.4254 |
|   |  |  |  |
| Proportional RUV (%CV) | 39.92 (0.81%) | 40.39 (7.49%) | 0.3410–0.4730 |
| IMAX (fold change)g | 0.6179 | 0.6042 (11.52%) | 0.473–0.755 |
| BSV Imax | 24.7 | 25.8 (0.09%) | — |
| IC50 (ng/mL) | 165.7 | 164.3 (14.5%) | 0.101–0.212 |
| BSV IC50 | 31.8 | 34.2 (0.29%) | — |
| KIN (L/h) | 1.836 | 1.762 (17.0%) | 0.641–2.02 |
| BSV Kin | 16.3 | 17.2 (0.06%) | — |
| KOUT (L/h) | 1.54 | 1.46 (16.7%) | 0.562–1.64 |
| BSV Kout | 15.6 | 16.0 (0.05%) | — |

Table 2 Caption: Final PPKPD Model Parameter Estimates

| Y=Y^∗(1+ε) | (7) |
| --- | --- |
The proportional RUV model greatly improved the overall base model compared with the additive (dOFV, −329.79; *P* = 1.27 × 10^−73^; using AIC). A mixed RUV model actually worsened the base model compared with proportional RUV alone (dOFV, 24.97) with no improvement in model diagnostics to justify inclusion. Several trends were observed in eta-eta plots; therefore, a variance-covariance block matrix was applied, resulting in significant improvement over a diagonal matrix (dOFV, −15.69; *df*= 6, *P* = .015) and decreasing eta shrinkage in V2 and Q from 40.4% and 40.5%, respectively, to 10.9% and 16.9%, respectively. The most significant covariance was between CL and V2 (rho = 0.879) and CL and V (rho = 0.566). BSV for V, CL, V2, and Q was 51.4%, 34%, 33.4%, and 88.4%, respectively.

### Covariate Model Development

Several covariates were assessed to determine the impact on structural parameters, including sex, race, age, body weight, body surface area, *UGT1A1* genotype, and other laboratory tests (albumin, bilirubin, ALT, AST). [Figure 2](#F2) depicts apparent trends that were explored for covariate inclusion. Numerous *UGT1A1* stratification scenarios involving **28* and **60* were tested as a binary categorical covariate to determine which scenario best improved the model. *UGT1A1***28* and **60* genotypes were in Hardy-Weinberg equilibrium (*P* < .05). The first scenario (scenario 1) considered EMs as patients who carried no variant alleles at either locus (**28* or **60*; n = 4) and IMs as patients carrying any of a possible 4 variant alleles (n = 19). This scenario of categorical genotype covariate on the CL parameter only slightly improved the model (dOFV, −2.301; *P* = .129). A second scenario (scenario 2) considered EMs as patients carrying no variant alleles or 1 copy of a **28* or **60* allele (n = 9); IMs were all others (patients carrying at least 2 variant alleles, n= 14). This scenario hardly improved the model (dOFV, −1.208; *P* =.272). Scenario 3 considered EMs as carriers of 0–2 copies of variant alleles, regardless of whether it was **28* or **60* (n = 18); IMs were carriers of 3–4 copies of variant alleles (n = 5) but did not significantly improve the model at all (dOFV, −0.329; *P* = . 566). The ideal scenario (scenario 4) was one in which patients were categorized as being either wild type for both *UGT1A1***28* and *UGT1A1***60* or having 1 copy of *UGT1A1***28* (henceforth, referred to as EMs; n = 5) or a carrier of at least 1 copy of *UGT1A1***60* and/or 2 copies of *UGT1A1***28* (IMs; n=18). Scenario 4 differed from scenario 1 by a single patient who was wild type for **60* and heterozygous for **28*, whose inclusion in scenario 1 as an IM lessened the difference in η_CL_ distributions between EMs and IMs. When this patient was considered an EM in scenario 4, the difference in η_CL_ distributions was greater, suggesting a single copy of **28* has no appreciable effect on belinostat CL. This was supported by a separate analysis using noncompartmental PK data that demonstrated *UGT1A1***60* has a greater effect on lowering belinostat CL than *UGT1A1***28*.^27^ When scenario 4 genotype was factored into the CL parameter as a fractional-change binary categorical covariate, the model was significantly improved (dOFV, −4.89; *P* = .027).

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5977/6357964/7b518858358e/nihms-1008576-f0002.jpg)

UGT1A1 genotype, albumin, body weight, and creatinine clearance were significant covariates on the CL parameter. (A) Inclusion of the categorical covariate for UGT1A1 genotype status (0 = wild type, I = variant) significantly improved the model and normalized the distribution of individual eta CL (nCL) estimates around zero. (B) Continuous covariate trends in base model (top) and final covariate model (bottom). The y-axis represents the individual eta CL (nCL) estimates.

Several continuous covariates (each scaled to the population medians) were added onto their respective parameter in a univariate manner, and each significantly improved the model: body weight on V (dOFV, −8.74; *P* = .0031); creatinine clearance on CL (dOFV, −7.387; *P* = .00657); albumin on CL (dOFV, −5.09; *P* = .0241). Using a stepwise forward inclusion/backward elimination approach, a full covariate model was arrived at, whereupon *UGT1A1* genotype status, creatinine clearance, and albumin were all factored into the CL parameter and body weight on volume. Removal of genotype, CLCr, albumin, and body weight significantly worsened the model by *P* = .0055, *P* = .0042, *P* = .0026, and *P* = .0089, respectively, justifying their inclusion in the final model. As body weight and CLCr are commonly correlated with each other, this relationship was assessed, but the correlation was deemed nonsignificant (*r*^2^ < 0.002), as was correlations among any other significant covariate. The final model was significantly improved from the base model (dOFV, −17.079; *df* = 4; *P* = .0018), with covariates explaining roughly 1% of volume BSV and 4% of CL BSV.

### Model Evaluation

Goodness-of-fit plots of the final PPK model demonstrated no misspecification or trends ([Figure 3A,B](#F3)). A visual predictive check ([Figure 3C](#F3)) demonstrated that the model can reliably predict the 5th, 50th, and 95th percentiles of observed data. The final PPK model had negligible epsilon shrinkage (0), an acceptable condition number (88867), and η shrinkage values for CL, V1, Q, and V2 of 14.9%, 10.8%, 22.7%, and 20.2%, respectively. Standard errors and 95% confidence intervals of all fixed and random (BSV) parameter estimates were calculated from nonparametric bootstrap estimates (n = 1000 replicates) and indicated precise parameter estimates ([Table 2](#T2)).

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5977/6357964/072e8bf2c5cf/nihms-1008576-f0003.jpg)

Model diagnostics. (A) Plots of the dependent variable (DV; observed plasma concentrations) versus individual predicted concentration (IPRED; mg/L; i), versus population predicted concentrations (PRED; mg/L; ii), CWRES versus time (iii), and CWRES versus PRED (iv) showed no model misspecification. In (iii) and (iv), a 20-point LOWESS regression to characterize trends in the conditional weighted residuals with respect to time and predicted concentrations was used. (B) Quantile-quantile plots of the eta estimates were normally distributed. (C) Visual predictive check was performed on the final PPK model, and the predicted 5th, 50th, and 95th percentiles (shaded with 95% confidence intervals) accurately predicted the observed 5th (dashed), 50th (solid), and 95th (dashed) percentiles.

### Model Simulations

The final PPK model was used to simulate the optimal dosage based on genotype status (EMs vs IMs) to provide equivalent AUCs using the patients from this data set. The desire to normalize belinostat exposures between EMs and IMs is based on a greater incidence of thrombocytopenia in carriers of variant alleles of *UGT1A1* (**28* [*P* = .038] or **60* [*P* = .0041]; Jonckheere-Terpstra trend test), which was likely due to greater belinostat exposure (*P* = .0037; Jonckheere-Terpstra trend test for **60* carriers) from lower CL_T_ (*P* = .020; Jonckheere-Terpstra trend test for **60* carriers).^27^ To ensure accurate Monte Carlo simulations using the final PPK model, the simulation model was first validated based on the suggested 25% dose reduction outlined in the Beleodaq prescribing information^8^: 1000 mg/m^2^ as a 30-minute CIVI for patients wild type for *UGT1A1***28* and *UGT1A1***60* or 1 copy of *UGT1A1***28*; 750 mg/m^2^ (30-minute CIVI) for patients with at least 1 copy of *UGT1A1***60* or 2 copies of *UGT1A1***28*. The simulated CL for each genotype group looked similar to observed data with comparable AUCs (not shown). This provided confidence in the simulation model to accurately predict AUCs through a dose adjustment. When the simulations were formatted for the 48-hour CIVI, the optimal doses found were 600 mg/m^2^/24 h for EMs and 400 mg/m^2^/24 h for IMs within this data set, a 33% dose reduction for IMs ([Figure 4](#F4)).

### Figure 4.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5977/6357964/eb11abda65e0/nihms-1008576-f0004.jpg)

Simulations for optimal belinostat doses to predict equivalent exposures. Simulated AUCinf from EM given 600 mg/m2/24 h versus IM given 400 mg/mg2/24 h from a single 48-hour intravenous infusion.

### PD Model Development

The inhibition of HDAC enzymes, as measured by global protein lysine acetylation, was added onto the final PPK model, where the best fit came from an indirect response model. A proportional RUV model best described the data. No covariates significantly improved the PD model. The exposure–response relationship between predicted belinostat plasma concentrations (PK) and the predicted fold change in AcK over time (PD) is depicted in [Figure 5](#F5). As PK levels rise, PD levels rise with little to no lag time (although, the first PD time was 12 hours, so this is difficult to assess). At the end of infusion (48 hours), there is a sharp drop in both PK and PD levels, suggesting a highly reversible inhibition of HDAC, consistent with preclinical data^28^ and justifying the need for a prolonged infusion and exposure to the drug.

### Figure 5.

![Figure 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5977/6357964/6240b2a45603/nihms-1008576-f0005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6357964_nihms-1008576-f0005.jpg)

Model-predicted exposure–response relationship between belinostat plasma concentrations and fold change in global protein lysine acetylation. Global protein lysine acetylation (AcK) was assessed predose and 12, 36, and 60 hours postdose, and fold changes in AcK at 12, 36, and 60 hours were incorporated into the final PPK model using an indirect response model (model II). The PK parameters were fixed during PK/PD model development and validation because of the complexity of the model. Model-predicted AcK fold changes paralleled that of predicted belinostat plasma concentrations. The reversible inhibition of histone deacetylase enzymes by belinostat is evident in the rapid decrease in fold change pf AcK upon the end of belinostat infusion.

## Discussion

Belinostat is an HDI metabolized by the UGT1A1 pathway that offers the unique opportunity to examine the relationships among pharmacokinetics, pharmacodynamics, and pharmacogenomics, and this model provides insight into these relationships. This study evaluated belinostat PPK while incorporating several covariates, including *UGT1A1* genotype, which is based on previous in vitro data suggesting the *UGT1A1***28* genotype causes reduced belinostat glucuronidation,^9^ thus posing a risk for toxicity in this subpopulation. The data were taken from a phase 1 clinical trial of belinostat, given as a 48-hour CIVI, along with etoposide and cisplatin in patients with small cell lung cancer and other advanced carcinomas. With only 23 evaluable patients used for this modeling exercise, the inherent problems of small sample size are acknowledged and taken into consideration when discussing the clinical applicability of this model and simulation.

The final model used 2 compartments to best describe the kinetic disposition of belinostat when administered as a 48-hour CIVI. Several covariates were available as potential predictor variables of structural clearance or volume parameters, such as sex, race, age, body weight, height, BSA, creatinine clearance, liver function tests, QTc interval, bilirubin, albumin, and *UGT1A1* genotype status (including both **28* and **60* alleles). After a thorough investigation of these continuous and categorical covariates, the CL parameter was significantly influenced by numerous covariates, including albumin levels, creatinine clearance, and *UGT1A1* genotype status. The volume parameter was significantly influenced by body weight. Although less than 2% of the belinostat dose is excreted unchanged in urine,^8^ it was not unexpected that creatinine clearance influenced belinostat clearance significantly, considering glucuronide metabolites that are not enterohepatically recirculated are eliminated through the kidneys.

Several stratification scenarios for *UGT1A1* status were explored based on preliminary PK data from a noncompartmental analysis that demonstrated *UGT1A1***60* had a greater effect on CL_T_ than *UGT1A1***28*.^27^ The final scenario was determined to be patients carrying wild-type alleles or 1 copy of **28*, who were labeled EMs, whereas all others (patients with at least 2 copies of **28* or 1–2 copies of **60* alleles at these loci) were labeled IMs. Most clinical research involving drugs that undergo glucuronidation by UGT1A1 has focused on the importance of *UGT1A1***28* as a genotype that can have phenotypic effects. Yet, in this data set, the allele frequency of **60* was 54%, almost identical to the reported allele frequency in whites (55%),^17^ who made up the majority (83%; 19 of 23) of the final data set ([Table 1](#T1)). This allowed for a more robust clinical analysis of any potential phenotypic effects of *UGT1A1***60*. These data also support previous findings that *UGT1A1***28* alone is not enough to decrease expression of *UGT1A1*, based on bilirubin glucuronidation in Gilbert’s patients and that *UGT1A1***60* was needed in combination with *UGT1A1***28* to decrease expression of *UGT1A1* to 56% of the wild type.^29^ *UGT1A1***6* is a rare polymorphism in whites (<5%),^17^ who made up the majority of the study population, and all but 1 patient was wild type for *UGT1A1***6*, thus precluding this genotype in the analyses.

The point estimate for total systemic clearance (CL) of 104.1 L/h was faster than previously published data (50–74 L/h^7,8^), comparable to other reports (41–74 L/h/m^2^)^6,30^ and yet slower than other studies (59–132 L/h/m^2^)^5^; see [Table 2](#T2). Each of the aforementioned references was based on a 30-minute infusion. There was 1 published clinical study administering belinostat as a 48-hour infusion, and the mean CL from that study was 133 L/h (106 L/h if excluding 1 outlier),^23^ which is almost identical to the point estimate from this study. Hepatic blood flow is approximately 90 L/h, confirming previous reports^8^ of belinostat having high hepatic extraction, with its total systemic clearance largely dependent on hepatic blood flow. The point estimate for volume of distribution in the central compartment (V1) was estimated at 32.2 L (total volume of distribution, ie, V1 + V2, was 40.9 L; [Table 2](#T2)). This volume is above the total plasma volume (5–7 L), but comparable to total body water volume of 42 L (based on a typical 70-kg man), which is consistent with belinostat being highly protein bound (93%–96%), thereby limiting tissue distribution.^8^ Further, a logP_oct/water_ of 1.7, suggests unionized belinostat is not lipophilic enough to extensively distribute into tissues. The model-predicted half-life (0.2 hours, based on HL = ln2/k_el_, where k_el_ = CL/V1) was slightly shorter than the observed mean from NCA (ln2/*λ*_z_, 0.82 hours^27^) and the reported range of 0.3– hours.^5,8^

A pharmacogenomic study of patients treated with belinostat indicated that carriers of allelic variants of *UGT1A1***28* and *UGT1A1***60* had greater exposure and higher toxicity.^27^ Therefore, simulations were performed to aid in accounting for *UGT1A1* genotype in dose optimization, and it was determined that doses of 600 mg/m^2^/24 h to EMs and 400 mg/m^2^/24 h to IMs would provide equivalent exposures. This dose adjustment based on *UGT1A1* genotype will hopefully allow patients with variant alleles that may have impaired metabolism and clearance, allowing for otherwise higher exposures to now have exposures equivalent to those patients with wild-type *UGT1A1* and therefore experience adverse events at a similar rate, as well as increase treatment tolerability. This dose adjustment simulation supported an amendment to this study (recently accepted by the NCI IRB) to modify dosing based on *UGT1A1* genotype. This has applicability not only to the 48-hour continuous infusion schedule but also to the 30-minute daily ×5 schedule. It will be important in developing combination therapies with belinostat to understand and correct for the impact of *UGT1A1* genotype. Not only is there potential for increased toxicity in those carrying variant alleles, but also there is the risk that phase 2 doses selected as safe in the development of combinations will undertreat patients carrying wild-type alleles. Indeed, determination of drug-metabolizing enzyme genotypes has been shown to be important in several agents in the clinic, including irinotecan (*UGT1A1*)^31^ and clopidogrel (*CYP2C19*).^32^

In conclusion, it was the objective of this study to report a PPK model that incorporated several covariates on CL, including the *UGT1A1* genotype. The PPK model was further developed and validated to incorporate PD data of reversible global protein lysine acetylation to visualize a relevant exposure/response relationship. Because a common problem with clinical pharmacology in oncology is the narrow therapeutic window of many drugs, determining more individualized optimal doses can hopefully reduce the incidence of adverse events and allow patients to stay on beneficial therapy longer. With that in mind, the PPK model described here was used to simulate optimal doses of belinostat given as a 48-hour CIVI based on *UGT1A1* genotype.

## Supplementary Material

## Acknowledgments

We thank all the nurses and clinical support staff for their patient care, the Blood Processing Core of the NCI for their sample handling/processing, and Seth Steinberg for his statistical analyses.

Funding

This project has been funded in whole or in part with federal funds from the National Cancer Institute, National Institutes of Health. This work was supported by the Intramural Research Program of the NIH, National Cancer Institute, Center for Cancer Research.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Peart MJ, Smyth GK, van Laar RK, et al. Identification and functional significance of genes regulated by structurally different histone deacetylase inhibitors. Proc Natl Acad Sci U S.A. 2005;102(10):3697–3702.  [DOI](https://doi.org/10.1073/pnas.0500369102) | [PMC free article](/articles/PMC552783/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15738394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S.A&title=Identification%20and%20functional%20significance%20of%20genes%20regulated%20by%20structurally%20different%20histone%20deacetylase%20inhibitors&author=MJ%20Peart&author=GK%20Smyth&author=RK%20van%20Laar&volume=102&issue=10&publication_year=2005&pages=3697-3702&pmid=15738394&doi=10.1073/pnas.0500369102&)

2. Xu WS, Parmigiani RB, Marks PA, Histone deacetylase inhibitors: molecular mechanisms of action. Oncogene. 2007;26(37):5541–5552.  [DOI](https://doi.org/10.1038/sj.onc.1210620) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17694093/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Histone%20deacetylase%20inhibitors:%20molecular%20mechanisms%20of%20action&author=WS%20Xu&author=RB%20Parmigiani&author=PA%20Marks&volume=26&issue=37&publication_year=2007&pages=5541-5552&pmid=17694093&doi=10.1038/sj.onc.1210620&)

3. Piekarz RL, Bates SE. Epigenetic modifiers: basic understanding and clinical development. Clin Cancer Res. 2009;15(12):3918–3926.  [DOI](https://doi.org/10.1158/1078-0432.CCR-08-2788) | [PMC free article](/articles/PMC6946182/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19509169/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Epigenetic%20modifiers:%20basic%20understanding%20and%20clinical%20development&author=RL%20Piekarz&author=SE%20Bates&volume=15&issue=12&publication_year=2009&pages=3918-3926&pmid=19509169&doi=10.1158/1078-0432.CCR-08-2788&)

4. Luchenko VL, Litman T, Chakraborty AR, et al. Histone deacetylase inhibitor-mediated cell death is distinct from its global effect on chromatin. Mol Oncol. 2014;8(8):1379–1392.  [DOI](https://doi.org/10.1016/j.molonc.2014.05.001) | [PMC free article](/articles/PMC4646083/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24954856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Oncol&title=Histone%20deacetylase%20inhibitor-mediated%20cell%20death%20is%20distinct%20from%20its%20global%20effect%20on%20chromatin&author=VL%20Luchenko&author=T%20Litman&author=AR%20Chakraborty&volume=8&issue=8&publication_year=2014&pages=1379-1392&pmid=24954856&doi=10.1016/j.molonc.2014.05.001&)

5. Steele NL, Plumb JA, Vidal L, et al. A phase 1 pharmacokinetic and pharmacodynamic study of the histone deacetylase inhibitor belinostat in patients with advanced solid tumors. Clin Cancer Res. 2008;14(3):804–810.  [DOI](https://doi.org/10.1158/1078-0432.CCR-07-1786) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18245542/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=A%20phase%201%20pharmacokinetic%20and%20pharmacodynamic%20study%20of%20the%20histone%20deacetylase%20inhibitor%20belinostat%20in%20patients%20with%20advanced%20solid%20tumors&author=NL%20Steele&author=JA%20Plumb&author=L%20Vidal&volume=14&issue=3&publication_year=2008&pages=804-810&pmid=18245542&doi=10.1158/1078-0432.CCR-07-1786&)

6. Lassen U, Molife LR, Sorensen M, et al. A phase I study of the safety and pharmacokinetics of the histone deacetylase inhibitor belinostat administered in combination with carboplatin and/or paclitaxel in patients with solid tumours. Br J Cancer. 2010;103(1):12–17.  [DOI](https://doi.org/10.1038/sj.bjc.6605726) | [PMC free article](/articles/PMC2905291/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20588278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=A%20phase%20I%20study%20of%20the%20safety%20and%20pharmacokinetics%20of%20the%20histone%20deacetylase%20inhibitor%20belinostat%20administered%20in%20combination%20with%20carboplatin%20and/or%20paclitaxel%20in%20patients%20with%20solid%20tumours&author=U%20Lassen&author=LR%20Molife&author=M%20Sorensen&volume=103&issue=1&publication_year=2010&pages=12-17&pmid=20588278&doi=10.1038/sj.bjc.6605726&)

7. Yeo W, Chung HC, Chan SL, et al. Epigenetic therapy using belinostat for patients with unresectable hepatocellular carcinoma: a multicenter phase I/II study with biomarker and pharmacokinetic analysis of tumors from patients in the Mayo Phase II Consortium and the Cancer Therapeutics Research Group. J Clin Oncol. 2012;30(27):3361–67.  [DOI](https://doi.org/10.1200/JCO.2011.41.2395) | [PMC free article](/articles/PMC3438233/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22915658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Epigenetic%20therapy%20using%20belinostat%20for%20patients%20with%20unresectable%20hepatocellular%20carcinoma:%20a%20multicenter%20phase%20I/II%20study%20with%20biomarker%20and%20pharmacokinetic%20analysis%20of%20tumors%20from%20patients%20in%20the%20Mayo%20Phase%20II%20Consortium%20and%20the%20Cancer%20Therapeutics%20Research%20Group&author=W%20Yeo&author=HC%20Chung&author=SL%20Chan&volume=30&issue=27&publication_year=2012&pages=3361-67&pmid=22915658&doi=10.1200/JCO.2011.41.2395&)

8. Spectrum Pharmaceuticals. Highlights of BELEODAQ prescribing information;2014. http://www.accessdata.fda.gov/drugsatfda_docs/label/2014/206256lbl.pdf  [http://www.accessdata.fda.gov/drugsatfda_docs/label/2014/206256lbl.pdf](http://www.accessdata.fda.gov/drugsatfda_docs/label/2014/206256lbl.pdf)

9. Wang LZ, Ramirez J, Yeo W, et al. Glucuronidation by UGT1A1 is the dominant pathway of the metabolic disposition of belinostat in liver cancer patients. PLoS One. 2013;8(1):e54522.  [DOI](https://doi.org/10.1371/journal.pone.0054522) | [PMC free article](/articles/PMC3559838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23382909/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Glucuronidation%20by%20UGT1A1%20is%20the%20dominant%20pathway%20of%20the%20metabolic%20disposition%20of%20belinostat%20in%20liver%20cancer%20patients&author=LZ%20Wang&author=J%20Ramirez&author=W%20Yeo&volume=8&issue=1&publication_year=2013&pages=e54522&pmid=23382909&doi=10.1371/journal.pone.0054522&)

10. Kiesel BF, Parise RA, Tjornelund J, et al. LC-MS/MS assay for the quantitation of the HDAC inhibitor belinostat and five major metabolites in human plasma. J Pharm Biomed Anal. 2013;81–82:89–98.  [DOI](https://doi.org/10.1016/j.jpba.2013.03.025) | [PMC free article](/articles/PMC3663884/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23644904/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharm%20Biomed%20Anal&title=LC-MS/MS%20assay%20for%20the%20quantitation%20of%20the%20HDAC%20inhibitor%20belinostat%20and%20five%20major%20metabolites%20in%20human%20plasma&author=BF%20Kiesel&author=RA%20Parise&author=J%20Tjornelund&volume=81%E2%80%9382&publication_year=2013&pages=89-98&pmid=23644904&doi=10.1016/j.jpba.2013.03.025&)

11. Ando Y, Saka H, Asai G, et al. UGT1A1 genotypes and glucuronidation of SN-38, the active metabolite of irinotecan. Ann Oncol. 1998;9(8):845–847.  [DOI](https://doi.org/10.1023/a:1008438109725) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9789606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=UGT1A1%20genotypes%20and%20glucuronidation%20of%20SN-38,%20the%20active%20metabolite%20of%20irinotecan&author=Y%20Ando&author=H%20Saka&author=G%20Asai&volume=9&issue=8&publication_year=1998&pages=845-847&pmid=9789606&doi=10.1023/a:1008438109725&)

12. Lee JS, Wang J, Martin M, et al. Genetic variation in UGT1A1 typical of Gilbert syndrome is associated with unconjugated hyperbilirubinemia in patients receiving tocilizumab. Pharmacogenet Genomics. 2011;21(7):365–374.  [DOI](https://doi.org/10.1097/FPC.0b013e32834592fe) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21412181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Genetic%20variation%20in%20UGT1A1%20typical%20of%20Gilbert%20syndrome%20is%20associated%20with%20unconjugated%20hyperbilirubinemia%20in%20patients%20receiving%20tocilizumab&author=JS%20Lee&author=J%20Wang&author=M%20Martin&volume=21&issue=7&publication_year=2011&pages=365-374&pmid=21412181&doi=10.1097/FPC.0b013e32834592fe&)

13. Toffoli G, Cecchin E, Corona G, et al. The role of UGT1A1*28 polymorphism in the pharmacodynamics and pharmacokinetics of irinotecan in patients with metastatic colorectal cancer. J Clin Oncol. 2006;24(19):3061–3068.  [DOI](https://doi.org/10.1200/JCO.2005.05.5400) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16809730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=The%20role%20of%20UGT1A1*28%20polymorphism%20in%20the%20pharmacodynamics%20and%20pharmacokinetics%20of%20irinotecan%20in%20patients%20with%20metastatic%20colorectal%20cancer&author=G%20Toffoli&author=E%20Cecchin&author=G%20Corona&volume=24&issue=19&publication_year=2006&pages=3061-3068&pmid=16809730&doi=10.1200/JCO.2005.05.5400&)

14. Zhou CF, Ma T, Su Y, et al. UGT1A1 gene polymorphisms and the toxicities of FOLFIRI in Chinese Han patients with gastrointestinal cancer. Anticancer Agents Med Chem. 2013;13(2):235–241.  [DOI](https://doi.org/10.2174/1871520611313020008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22934695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20Agents%20Med%20Chem&title=UGT1A1%20gene%20polymorphisms%20and%20the%20toxicities%20of%20FOLFIRI%20in%20Chinese%20Han%20patients%20with%20gastrointestinal%20cancer&author=CF%20Zhou&author=T%20Ma&author=Y%20Su&volume=13&issue=2&publication_year=2013&pages=235-241&pmid=22934695&doi=10.2174/1871520611313020008&)

15. Innocenti F, Grimsley C, Das S, et al. Haplotype structure of the UDP-glucuronosyltransferase 1A1 promoter in different ethnic groups. Pharmacogenetics. 2002;12(9):725–733.  [DOI](https://doi.org/10.1097/00008571-200212000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12464801/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Haplotype%20structure%20of%20the%20UDP-glucuronosyltransferase%201A1%20promoter%20in%20different%20ethnic%20groups&author=F%20Innocenti&author=C%20Grimsley&author=S%20Das&volume=12&issue=9&publication_year=2002&pages=725-733&pmid=12464801&doi=10.1097/00008571-200212000-00006&)

16. Sugatani J, Yamakawa K, Yoshinari K, et al. Identification of a defect in the UGT1A1 gene promoter and its association with hyperbilirubinemia. Biochem Biophys Res Commun. 2002;292(2):492–497.  [DOI](https://doi.org/10.1006/bbrc.2002.6683) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11906189/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Identification%20of%20a%20defect%20in%20the%20UGT1A1%20gene%20promoter%20and%20its%20association%20with%20hyperbilirubinemia&author=J%20Sugatani&author=K%20Yamakawa&author=K%20Yoshinari&volume=292&issue=2&publication_year=2002&pages=492-497&pmid=11906189&doi=10.1006/bbrc.2002.6683&)

17. Kaniwa N, Kurose K, Jinno H, et al. Racial variability in haplotype frequencies of UGT1A1 and glucuronidation activity of a novel single nucleotide polymorphism 686C> T (P229L) found in an African-American. Drug Metab Dispos. 2005;33(3):458–465.  [DOI](https://doi.org/10.1124/dmd.104.001800) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15572581/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Racial%20variability%20in%20haplotype%20frequencies%20of%20UGT1A1%20and%20glucuronidation%20activity%20of%20a%20novel%20single%20nucleotide%20polymorphism%20686C>%20T%20(P229L)%20found%20in%20an%20African-American&author=N%20Kaniwa&author=K%20Kurose&author=H%20Jinno&volume=33&issue=3&publication_year=2005&pages=458-465&pmid=15572581&doi=10.1124/dmd.104.001800&)

18. Bosma PJ, Chowdhury JR, Bakker C, et al. The genetic basis of the reduced expression of bilirubin UDP-glucuronosyltransferase 1 in Gilbert’s syndrome. N Engl J Med. 1995;333(18):1171–1175.  [DOI](https://doi.org/10.1056/NEJM199511023331802) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7565971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=The%20genetic%20basis%20of%20the%20reduced%20expression%20of%20bilirubin%20UDP-glucuronosyltransferase%201%20in%20Gilbert%E2%80%99s%20syndrome&author=PJ%20Bosma&author=JR%20Chowdhury&author=C%20Bakker&volume=333&issue=18&publication_year=1995&pages=1171-1175&pmid=7565971&doi=10.1056/NEJM199511023331802&)

19. Sato H, Adachi Y, Koiwai O. The genetic basis of Gilbert’s syndrome. Lancet. 1996;347(9001):557–558.  [DOI](https://doi.org/10.1016/s0140-6736(96)91266-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8596313/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=The%20genetic%20basis%20of%20Gilbert%E2%80%99s%20syndrome&author=H%20Sato&author=Y%20Adachi&author=O%20Koiwai&volume=347&issue=9001&publication_year=1996&pages=557-558&pmid=8596313&doi=10.1016/s0140-6736(96)91266-0&)

20. Beal SL. Ways to fit a PK model with some data below the quantification limit. J Pharmacokinet Pharmacodyn. 2001;28(5):481–504.  [DOI](https://doi.org/10.1023/a:1012299115260) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11768292/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacokinet%20Pharmacodyn&title=Ways%20to%20fit%20a%20PK%20model%20with%20some%20data%20below%20the%20quantification%20limit&author=SL%20Beal&volume=28&issue=5&publication_year=2001&pages=481-504&pmid=11768292&doi=10.1023/a:1012299115260&)

21. Ahn JE, Karlsson MO, Dunne A, Ludden TM. Likelihood based approaches to handling data below the quantification limit using NONMEM VI. J Pharmacokinet Pharmacodyn. 2008;35(4):401–421.  [DOI](https://doi.org/10.1007/s10928-008-9094-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18686017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacokinet%20Pharmacodyn&title=Likelihood%20based%20approaches%20to%20handling%20data%20below%20the%20quantification%20limit%20using%20NONMEM%20VI&author=JE%20Ahn&author=MO%20Karlsson&author=A%20Dunne&author=TM%20Ludden&volume=35&issue=4&publication_year=2008&pages=401-421&pmid=18686017&doi=10.1007/s10928-008-9094-4&)

22. Bergstrand M, Karlsson MO. Handling data below the limit of quantification in mixed effect models. AAPS J. 2009;11(2):371–380.  [DOI](https://doi.org/10.1208/s12248-009-9112-5) | [PMC free article](/articles/PMC2691472/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19452283/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AAPS%20J&title=Handling%20data%20below%20the%20limit%20of%20quantification%20in%20mixed%20effect%20models&author=M%20Bergstrand&author=MO%20Karlsson&volume=11&issue=2&publication_year=2009&pages=371-380&pmid=19452283&doi=10.1208/s12248-009-9112-5&)

23. Thomas A, Rajan A, Szabo E, et al. A phase I/II trial of belinostat in combination with cisplatin, doxorubicin, and cyclophosphamide in thymic epithelial tumors: a clinical and translational study. Clin Cancer Res. 2014;20(21):5392–5402.  [DOI](https://doi.org/10.1158/1078-0432.CCR-14-0968) | [PMC free article](/articles/PMC4216756/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25189481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=A%20phase%20I/II%20trial%20of%20belinostat%20in%20combination%20with%20cisplatin,%20doxorubicin,%20and%20cyclophosphamide%20in%20thymic%20epithelial%20tumors:%20a%20clinical%20and%20translational%20study&author=A%20Thomas&author=A%20Rajan&author=E%20Szabo&volume=20&issue=21&publication_year=2014&pages=5392-5402&pmid=25189481&doi=10.1158/1078-0432.CCR-14-0968&)

24. Savic RM, Karlsson MO. Importance of shrinkage in empirical bayes estimates for diagnostics: problems and solutions. AAPS J. 2009;11(3):558–569.  [DOI](https://doi.org/10.1208/s12248-009-9133-0) | [PMC free article](/articles/PMC2758126/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19649712/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AAPS%20J&title=Importance%20of%20shrinkage%20in%20empirical%20bayes%20estimates%20for%20diagnostics:%20problems%20and%20solutions&author=RM%20Savic&author=MO%20Karlsson&volume=11&issue=3&publication_year=2009&pages=558-569&pmid=19649712&doi=10.1208/s12248-009-9133-0&)

25. Giaccone G, Rajan A, Berman A, et al. Phase II study of belinostat in patients with recurrent or refractory advanced thymic epithelial tumors. J Clin Oncol. 2011;29(15):2052–2059.  [DOI](https://doi.org/10.1200/JCO.2010.32.4467) | [PMC free article](/articles/PMC3107761/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21502553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Phase%20II%20study%20of%20belinostat%20in%20patients%20with%20recurrent%20or%20refractory%20advanced%20thymic%20epithelial%20tumors&author=G%20Giaccone&author=A%20Rajan&author=A%20Berman&volume=29&issue=15&publication_year=2011&pages=2052-2059&pmid=21502553&doi=10.1200/JCO.2010.32.4467&)

26. Sharma A, Jusko WJ. Characteristics of indirect pharmacodynamic models and applications to clinical drug responses. Br J Clin Pharmacol. 1998;45(3):229–239.  [DOI](https://doi.org/10.1046/j.1365-2125.1998.00676.x) | [PMC free article](/articles/PMC1873365/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9517366/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Characteristics%20of%20indirect%20pharmacodynamic%20models%20and%20applications%20to%20clinical%20drug%20responses&author=A%20Sharma&author=WJ%20Jusko&volume=45&issue=3&publication_year=1998&pages=229-239&pmid=9517366&doi=10.1046/j.1365-2125.1998.00676.x&)

27. Goey AK, Sissung TM, Peer CJ, et al. Effects of UGT1A1 genotype on the pharmacokinetics, pharmacodynamics and toxicities of belinostat administered by 48 h continuous infusion in patients with cancer. J Clin Pharmacol. In press.  [DOI](https://doi.org/10.1002/jcph.625) | [PMC free article](/articles/PMC6361127/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26313268/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Effects%20of%20UGT1A1%20genotype%20on%20the%20pharmacokinetics,%20pharmacodynamics%20and%20toxicities%20of%20belinostat%20administered%20by%2048%20h%20continuous%20infusion%20in%20patients%20with%20cancer&author=AK%20Goey&author=TM%20Sissung&author=CJ%20Peer&pmid=26313268&doi=10.1002/jcph.625&)

28. Marquard L, Petersen KD, Persson M, et al. Monitoring the effect of belinostat in solid tumors by H4 acetylation. APMIS. 2008;116(5):382–392.  [DOI](https://doi.org/10.1111/j.1600-0463.2008.00957.x) | [PMC free article](/articles/PMC2774150/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18452428/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=APMIS&title=Monitoring%20the%20effect%20of%20belinostat%20in%20solid%20tumors%20by%20H4%20acetylation&author=L%20Marquard&author=KD%20Petersen&author=M%20Persson&volume=116&issue=5&publication_year=2008&pages=382-392&pmid=18452428&doi=10.1111/j.1600-0463.2008.00957.x&)

29. Matsui K, Maruo Y, Sato H, Takeuchi Y. Combined effect of regulatory polymorphisms on transcription of UGT1A1 as a cause of Gilbert syndrome. BMC Gastroenterol. 2010;10:57.  [DOI](https://doi.org/10.1186/1471-230X-10-57) | [PMC free article](/articles/PMC2894006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20529348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Gastroenterol&title=Combined%20effect%20of%20regulatory%20polymorphisms%20on%20transcription%20of%20UGT1A1%20as%20a%20cause%20of%20Gilbert%20syndrome&author=K%20Matsui&author=Y%20Maruo&author=H%20Sato&author=Y%20Takeuchi&volume=10&publication_year=2010&pages=57&pmid=20529348&doi=10.1186/1471-230X-10-57&)

30. CuraGen Corporation. Investigator’s brochure for belinostat (PXD101), version 5.0;2007.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Investigator%E2%80%99s%20brochure%20for%20belinostat%20(PXD101),%20version%205.0&publication_year=2007&)

31. Ramchandani RP, Wang Y, Booth BP, et al. The role of SN-38 exposure, UGT1A1*28 polymorphism, and baseline bilirubin level in predicting severe irinotecan toxicity. J Clin Pharmacol. 2007;47(1):78–86.  [DOI](https://doi.org/10.1177/0091270006295060) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17192505/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=The%20role%20of%20SN-38%20exposure,%20UGT1A1*28%20polymorphism,%20and%20baseline%20bilirubin%20level%20in%20predicting%20severe%20irinotecan%20toxicity&author=RP%20Ramchandani&author=Y%20Wang&author=BP%20Booth&volume=47&issue=1&publication_year=2007&pages=78-86&pmid=17192505&doi=10.1177/0091270006295060&)

32. Horenstein RB, Madabushi R, Zineh I, et al. Effectiveness of clopidogrel dose escalation to normalize active metabolite exposure and antiplatelet effects in CYP2C19 poor metabolizers. J Clin Pharmacol. 2014;54(8):865–873.  [DOI](https://doi.org/10.1002/jcph.293) | [PMC free article](/articles/PMC4113831/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24710841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Effectiveness%20of%20clopidogrel%20dose%20escalation%20to%20normalize%20active%20metabolite%20exposure%20and%20antiplatelet%20effects%20in%20CYP2C19%20poor%20metabolizers&author=RB%20Horenstein&author=R%20Madabushi&author=I%20Zineh&volume=54&issue=8&publication_year=2014&pages=865-873&pmid=24710841&doi=10.1002/jcph.293&)
