# Genomic Association Analysis Reveals Variants Associated With Blood Pressure Response to Beta‐Blockers in European Americans

## Metadata
**Authors:** Sonal Singh, Nihal El Rouby, Caitrin W McDonough, Yan Gong, Kent R Bailey, Eric Boerwinkle, Arlene B Chapman, John G Gums, Stephen T Turner, Rhonda M Cooper‐DeHoff, Julie A Johnson
**Journal:** Clinical and Translational Science
**Date:** 2019 Jun 4
**DOI:** [10.1111/cts.12643](https://doi.org/10.1111/cts.12643)
**PMID:** 31033190
**PMCID:** PMC6742943
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6742943/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6742943/pdf/CTS-12-497.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6742943/pdf/CTS-12-497.pdf)

## Abstract

European Americans (EA) have a better antihypertensive response to β‐blockers when compared with African Americans, albeit with some variability. We undertook a genomewide association study to elucidate the underlying genetic determinants in EA contributing to this variability in blood pressure (BP) response. A discovery genomewide association study of change in BP post–metoprolol treatment was performed in EA participants (n = 201) from the Pharmacogenomic Evaluation of Antihypertensive Responses‐2 (PEAR‐2) study and tested for replication in the atenolol‐treated EA from the PEAR study (n = 233). Rs294610 in the FGD5, which encodes for FYVE, RhoGEF and PH Domain Containing 5, (expression quantitative trait loci for FGD5 in the small intestine) was significantly associated with increased diastolic BP response to β‐blockers in the PEAR‐2 study (P = 3.41 × 10−6, β = −2.70) and replicated (P = 0.01, β = −1.17) in the PEAR study. Post–meta‐analysis of these studies, an additional single nucleotide polymorphism rs45545233 in the SLC4A1, encoding for Solute Carrier Family 4 Member 1, (expression quantitative trait loci for dual specificity phosphatase 3 in the artery tibial) was identified that was significantly associated with a poor response to β‐blockers (P = 3.43 × 10−6, β = 4.57) and was replicated in the atenolol add‐on cohort (P = 0.007, β = 4.97). We identified variants in FGD5 and SLC4A1, which have been previously cited as candidate genes for hypertension, to be associated with a β‐blocker BP response in EA. Further elucidation is warranted of the underlying mechanisms of these variants and genes by which they influence the BP response to β‐blockers.

☑ β‐blockers are one of the commonly used antihypertensive medications. European Americans respond better to β‐blockers, albeit with a huge interpatient variability for which genetics appears to be one of the contributing factors.

☑ We performed a pharmacogenomic genomewide association analysis with the goal of elucidating the underlying genetic determinants of β‐blocker blood pressure response.

☑ Using a multistage genomewide association study approach, we identified and validated variants in FYVE, RhoGEF and PH Domain Containing 5 (*FGD5*) and Solute Carrier Family 4 Member 1 (*SLC4A1*). These genes have been associated with hypertension, but ours is the first study to identify an association between these genes and β‐blocker blood pressure response.

☑ Understanding the exact mechanisms of these identified variants and genes in the β‐blocker blood pressure response pathway can aid in their development as genetic markers that can aid in the *a priori* identification of patients better suited for β‐blocker therapy.

Hypertension (HTN) is prevalent in developed and developing countries[1](#cts12643-bib-0001), [2](#cts12643-bib-0002) and poses considerable risk for stroke, coronary heart disease, and renal and heart failure. It has one of the biggest impacts on the burden of the cardiovascular diseases globally. In the United States, HTN remains one of the most common chronic conditions, affecting 46% of the adult population, and the prevalence increases with age.[3](#cts12643-bib-0003) Blood pressure (BP) control because of antihypertensive medications is associated with a decrease in cardiovascular morbidity and mortality.[4](#cts12643-bib-0004) However, despite antihypertensive medications being among the most commonly prescribed classes of medications in the United States, only about 50% of the hypertensive individuals achieve BP control.[5](#cts12643-bib-0005) It has been well recognized that there exists wide interpatient variability in BP response to all classes of antihypertensives.

Studies have documented multiple pathways underlying HTN pathophysiology, which vary based on race/ethnicity. In addition, there is a strong correlation between race/ethnicity and BP response to antihypertensive drugs.[6](#cts12643-bib-0006) We have previously published our findings regarding the genetic determinants of the variability in BP response to β‐blockers (atenolol and metoprolol) in African Americans.[7](#cts12643-bib-0007) European Americans generally respond better to β‐blocker treatment when compared with African Americans[8](#cts12643-bib-0008) however, there is wide interpatient variability in BP response among European Americans, for which genetic makeup is likely one of the contributing factors.[9](#cts12643-bib-0009), [10](#cts12643-bib-0010), [11](#cts12643-bib-0011), [12](#cts12643-bib-0012) Understanding the genetic determinants of BP response to β‐blockers can help in *a priori* identification of hypertensive patients better suited for β‐blocker therapy for better BP control and outcomes. In this study, our focus was to elucidate the genetic underpinnings of the variability in BP response to β‐blockers in European Americans for the following two β‐blockers: metoprolol and atenolol. To this end, we performed a discovery genomewide association study (GWAS) for BP response to metoprolol monotherapy followed by testing the prioritized variants for replication in participants treated with atenolol monotherapy. We also performed a meta‐analysis between the two studies and further performed a secondary replication to test the top associations of meta‐analysis using an independent cohort treated with atenolol as add‐on therapy.

## Methods

### Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR and PEAR‐2) study design

Details of the Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) and PEAR‐2 clinical studies have been described previously.[13](#cts12643-bib-0013), [14](#cts12643-bib-0014) Both studies were prospective randomized clinical trials and were conducted with the goal of evaluating the influence of genetics on the BP variability and adverse metabolic side effects following treatment with antihypertensive medication(s). Details of both PEAR and PEAR‐2 have been previously published. In brief, for both studies, participants with mild to moderate uncomplicated HTN of either gender and of any race and ethnicity were recruited. Exclusions included participants with secondary HTN and known history of cardiovascular disease or diabetes. For a period of 3–4 weeks, eligible participants underwent an antihypertensive medication washout. Upon this, hypertensive BP measurements were confirmed, following which in PEAR, the patients were randomized to a monotherapy of either hydrochlorothiazide (HCTZ) or atenolol followed by a combination therapy of the other drug added to initial therapy (ClinicalTrials.gov identifier [NCT00246519](https://clinicaltrials.gov/ct2/show/NCT00246519)), and in PEAR‐2, the participants were sequentially treated with metoprolol monotherapy followed by chlorthalidone monotherapy (ClinicalTrials.gov identifier [NCT01203852](https://clinicaltrials.gov/ct2/show/NCT01203852)).

Specifically, the β‐blocker regimens were as follows: For PEAR, the participants were treated with atenolol 50 mg daily for 3 weeks followed by dose titration to atenolol 100 mg for 6 weeks. If BP was not controlled in patients who received HCTZ monotherapy, atenolol was added to HCTZ in the same dosage described previously. In PEAR‐2, the participants were treated with metoprolol tartrate 50 mg twice daily for 2 weeks followed by dose up‐titration to 100 mg twice daily for an additional 6 weeks. In both PEAR and PEAR‐2, a BP > 120/70 mmHg was the cutoff for initiating a dose up‐titration unless the patient's heart rate was below 55 beats per minute, in which case up‐titration did not occur. In PEAR, more than 85% of the patients treated with atenolol monotherapy or add‐on therapy had their dose up‐titrated to 100 mg twice a day, and in PEAR‐2, more than 95% of the participants treated with metoprolol monotherapy had their dose up‐titrated to 100 mg twice a day. In addition to the BP measurements, blood samples (for DNA, RNA, plasma, and serum) were also collected before (end of washout) and after the 8–9 week treatment with the study drugs. This analysis specifically focuses on the BP response to atenolol mono and add‐on therapy in PEAR and metoprolol monotherapy in PEAR‐2 in a population of European Americans ancestry.

All of the genotype and phenotype data supporting the conclusion of this article for both studies, PEAR and PEAR‐2, are available at the database of Genotypes and Phenotypes (dbGAP) repository.[15](#cts12643-bib-0015) The PEAR study data are available under dbGaP accession phs000649.v1.p1, and currently the PEAR‐2 study data are in the process of being uploaded and will soon be available under dbGaP accession phs000649.v2.p1.

The institutional review boards at each of the participating sites, which include the University of Florida, Emory University, and Mayo Clinic, reviewed and approved the PEAR and PEAR‐2 studies. All study participants were required to provide written informed consent prior to participation in each trial. The studies were conducted in accordance with the Declaration of Helsinki.

### BP response phenotype

For each study, the most precise BP measurement available was selected. In the PEAR study, the following three different methods were used for BP measurement: home, office, and ambulatory. The composite weighted average of the office, home, ambulatory daytime and nighttime BP responses was used for the analysis herein because it had higher signal‐to‐noise ratio and thus provided greater power.[16](#cts12643-bib-0016) In PEAR‐2, only home and office BP measurements were collected, and we have previously demonstrated that a single home BP measurement is more informative than an office BP measurement.[16](#cts12643-bib-0016) Therefore, home BP in PEAR2 was used for the analysis. In both of the studies, home BP was self‐measured by the patients using the home BP monitors by Microlife, model 3AC1‐PC (Minneapolis, MN) in PEAR and models BP3AC1‐PC and BP3MC1‐PC (Dunedin, FL) for PEAR‐2.

Delta diastolic and systolic BP (∆DBP and ∆SBP) were defined as the change in BP from the start to the end of the treatment (Posttreatment BP − Pretreatment BP).

### Genotyping and imputation

The details of the genotyping and imputation performed on the PEAR‐2 and PEAR samples have been previously published.[17](#cts12643-bib-0017) PEAR‐2 participant DNA samples were genotyped using the Illumina Human Omni 2.5S Beadchip (Illumina, San Diego, CA) for 2.5 million single nucleotide polymorphisms (SNPs), whereas DNA from the PEAR participants was genotyped for 1 million SNPs using the Illumina Human Omni1M Quad Beadchip (Illumina). Genotyping for both PEAR‐2 and PEAR was carried out at the University of Texas at Houston Health Science Center, Human Genetics Center. Quality control procedures were applied to both genetic data sets. A principal component analysis was performed using the EIGENSTRAT method to determine the genetic ancestry of the PEAR‐2 and PEAR participants. The high‐quality SNPs obtained post these quality control steps were imputed to the 1000 genomes phase3 version 5 reference panel using Minimac3 (version 1.0.16).[18](#cts12643-bib-0018) For postimputation quality control, the SNPs with imputation quality (R‐squared‐Rsq) < 0.3 and minor allele frequency < 3% were excluded.

### Statistical analysis

The participant characteristics are presented as mean ± SD for continuous variables and as numbers and percentages for categorical variables. The analysis in this article is focused on determining the genetic underpinnings of BP response in the European American population treated with β‐blockers. The discovery cohort composed of metoprolol‐treated participants from PEAR‐2 and the atenolol‐treated participants from PEAR served as the replication cohort for the associated signals from PEAR‐2. Participants receiving atenolol as an add‐on drug to the HCTZ treatment served as a secondary replication cohort for the associated signals from the meta‐analysis between PEAR‐2 and PEAR.

#### GWAS analysis

A multistaged GWAS analysis plan was used to identify genetic variants associated with BP response to β‐blockers, the details of which are represented in **Figure ** [1](#cts12643-fig-0001) **.** We have previously published pharmacogenomic findings related to β‐blocker BP response using only PEAR as the discovery cohort because this was before the PEAR‐2 trial was completed.[19](#cts12643-bib-0019) Hence, in this study, PEAR‐2 was used as the discovery and PEAR as the replication cohort.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8b11/6742943/54f8b170b63e/CTS-12-497-g001.jpg)

Overall approaches and analysis framework used in this study. eQTL, expression quantitative trait loci; GWAS, genomewide association study; LD, linkage disequilibrium; PEAR, Pharmacogenomic Evaluation of Antihypertensive Responses; PEAR‐2, Pharmacogenomic Evaluation of Antihypertensive Responses‐2; SNP, single nucleotide polymorphism.

In the PEAR‐2 European American participants, the association between the SNPs and the BP response to metoprolol (∆DBP and ∆SBP) was tested in 201 participants using linear regression assuming an additive model of inheritance. The analysis was adjusted for age, gender, baseline BP, and principal components 1 and 2. Genomewide significance and suggestive association were set at 5 × 10^−8^ and 1 × 10^−5^, respectively. Studies have demonstrated the successful use of expression quantitative trait loci (eQTL) as an SNP prioritization method for GWAS to identify true functional and regulatory variants that are taken forward for successful replication.[20](#cts12643-bib-0020) Thus in our study, SNPs meeting the suggestive level of significance (*P* < 1 × 10^−5^) were further screened based on their eQTL annotation using the GTEx database that has archived a large number of tissue‐specific and regulatory eQTLs.[21](#cts12643-bib-0021) The variants were queried using the GTEx portal and selected if they were associated with the expression of one or more genes in one or more tissues. These eQTL‐prioritized variants were then pruned based on their linkage disequilibrium (LD; *r* ^2^ > 0.8) using LDlink3.0[22](#cts12643-bib-0022) to ensure that only a single independent lead eQTL SNP with the lowest *P* value was selected from the multiple eQTL SNPs that were present in each gene locus.

The independent associated SNPs from PEAR‐2 that were prioritized through eQTL and LD pruned were moved forward for replication (one SNP for SBP and four SNPs for DBP). Another cohort of 233 atenolol‐treated European American participants was used for replication. Considering a Bonferroni‐corrected, one‐sided hypothesis (0.05/No. of tests), SNPs that met the significance level of 0.05 for an association with SBP response and a *P* < 0.012 for an association with DBP with the same direction of association as that of discovery were considered replicated. We used the one‐sided *P* value threshold because the direction of association for successful replication was already determined by the discovery cohort, and studies have shown the appropriateness of using one‐sided *P* value for directional hypothesis.[23](#cts12643-bib-0023)

To increase power with the aim of identifying additional associations, the summary statistics of the PEAR‐2 and PEAR results were combined in meta‐analyses using Meta‐Analysis Helper[24](#cts12643-bib-0024) and using a fixed effect model with inverse variance weighing.

SNPs with *P* < 1 × 10^−5^ were subjected to an eQTL‐prioritization strategy as described previously. The eQTL‐prioritized SNPs were LD pruned to obtain independent signals, which were then tested for replication in an independent European American cohort from PEAR treated with add‐on atenolol.

## Results

The demographic and clinical characteristics of the PEAR‐2 participants treated with metoprolol monotherapy and the PEAR participants treated with atenolol monotherapy and add‐on therapy are outlined in **Table ** [1](#cts12643-tbl-0001). The average age of the participants was similar in all three studies (51 years in PEAR‐2 vs. 49 years in PEAR monotherapy vs. 52 years in PEAR add‐on therapy). After 8 weeks of metoprolol treatment, the PEAR‐2 participants had a mean ∆SBP of −10.19 mmHg and a mean ∆DBP of −9.05 mmHg. After 8 weeks of atenolol treatment in the PEAR participants, the mean ∆SBP and ∆DBP were −9.35 mmHg and −7.17 mmHg, respectively. Among the PEAR patients treated with atenolol add‐on therapy, the mean ∆SBP was −10.10 mmHg and the mean ∆DBP was −8.87 mmHg.

### Table 1.

| Baseline characteristics | PEAR‐2 (metoprolol) | PEAR (atenolol) | PEAR (atenolol add‐on) |
| --- | --- | --- | --- |
| N | 201 | 233 | 207 |
| Age, years | 51 ± 8.99 | 49 ± 9.52 | 50 ± 9.52 |
| Female, N (%) | 65 (48.14) | 109 (46.78) | 82 (39.13) |
| Baseline SBP (mmHg) | 147.49 ± 10.83 | 145.46 ± 9.68 | 138.65 ± 9.95 |
| Baseline DBP (mmHg) | 93.94 ± 5.63 | 93.20 ± 5.54 | 89.34 ± 6.53 |
| Posttreatment SBP (mmHg) | 137.32 ± 12.82 | 136.11 ± 11.46 | 128.16 ± 9.90 |
| Posttreatment DBP (mmHg) | 84.90 ± 7.45 | 86.05 ± 7.75 | 79.92 ± 6.63 |
| Delta SBP (mmHg) | −10.19 ± 9.20 | −12.67 ± 8.61 | −10.10 ± 6.04 |
| Delta DBP (mmHg) | −9.05 ± 6.07 | −10.50 ± 5.77 | −8.87 ± 4.39 |

Table 1 Caption: Characteristics of PEAR‐2, PEAR, and PEAR add‐on cohort participants

### Genomewide analysis of BP response to metoprolol

Genomewide analysis was performed to test the association between variants and the change in ∆DBP or ∆SBP after treatment with metoprolol monotherapy in the PEAR‐2 cohort. Manhattan and qq plots for the association analysis results for both ∆SBP and ∆DBP are presented in **Figures ** [S1](#cts12643-sup-0001) **and** [S2](#cts12643-sup-0002) **,** respectively. None of the SNPs reached genomewide significance for an association with either ∆SBP or ∆DBP after treatment with metoprolol. However, a total of 38 and 116 SNPs met the suggestive threshold of significance (1 × 10^−5^) for association with ∆SBP and ∆DBP, respectively. Of the 38 SNPs associated with ∆SBP that met the suggestive level of significance, two SNPs were eQTLs. Similarly, of the 116 SNPs associated with ∆DBP that met the suggestive level of significance, 52 SNPs were eQTLs. Post‐LD pruning, one independent signal was associated with ∆SBP, and four independent signals were associated with ∆DBP. The independent signals for the associations with the ∆SBP and ∆DBP responses are presented in **Table ** [S1](#cts12643-sup-0006).

### Replication in PEAR atenolol monotherapy

These independent signals were moved forward for replication to be tested in an independent cohort of 233 atenolol monotherapy–treated European American participants from PEAR. Of the SNPs that were tested for association with ∆DBP, an SNP (rs294610) that is 13 kilobase 5′ of FYVE, RhoGEF And PH Domain Containing 5 (*FGD5*) was successfully replicated (one‐sided *P* value of 0.010, β = −1.17; **Table ** [2)](#cts12643-tbl-0002). rs294610 is an intronic SNP and results in a C‐>A change. It is also an eQTL for *FGD5*. The A allele carriers of this SNP had significantly higher BP responses to β‐blockers when compared with the noncarriers in both the discovery and replication cohorts (**Figure ** [2](#cts12643-fig-0002)). The regional plot for this region is presented in **Figure ** [S3](#cts12643-sup-0003).

### Table 2.

| SNP | CHR | Base‐pair position (hg19 position) | Nearest gene | Minor allele | MAF | PEAR‐2 metoprolol DBP response |  |  | PEAR atenolol DBP response |  |  | Meta‐analysis |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| β | SE | P value | β | SE | One‐sided P value | β | SE | P value |  |  |  |  |  |  |
| rs294610 | 3 | 14843159 | FGD5 | A | 0.38 | −2.70 | 0.56 | 3.41E‐06 | −1.17 | 0.50 | 0.01 | −1.86 | 0.37 | 9.20E‐07 |

Table 2 Caption: Replication of PEAR‐2 prioritized and LD pruned association in PEAR

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8b11/6742943/21982dc0af93/CTS-12-497-g002.jpg)

Diastolic blood pressure (DBP) change post–metoprolol monotherapy treatment among Pharmacogenomic Evaluation of Antihypertensive Responses‐2 (PEAR‐2) study European Americans and Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) study European Americans post–atenolol monotherapy by the FYVE, RhoGEF And PH Domain Containing 5 (FGD5) rs296410 genotype.

The one independent signal for ∆SBP that was taken forward for replication in PEAR did not replicate in the PEAR monotherapy cohort.

### Meta‐analysis of BP response to β‐blockers in PEAR‐2 and PEAR monotherapy

The results of the PEAR‐2 and PEAR association analyses (for ∆SBP and ∆DBP) were further combined in meta‐analyses. For ∆SBP, a post–meta‐analysis resulted in 80 SNPs that reached the suggestive level of significance. For ∆DBP, one SNP, rs367649416, in the death domain containing 1 (*DTHD1*) gene reached genomewide significance, and 107 SNPs met the suggestive threshold of significance (*P* < 1 × 10^−5^). Upon meta‐analysis, the association of the previously replicated SNP (rs294610) in *FGD5* became stronger and reached a *P* value of 8.58 × 10^−7^. For both ∆SBP and ∆DBP, SNPs that met the suggestive level of significance were prioritized using eQTL and LD pruning as before. Post prioritization, there were two independent signals that were associated with ∆SBP and seven SNPs that were associated with ∆DBP as listed in **Table ** [S2](#cts12643-sup-0007).

### Validation of meta‐analysis in PEAR atenolol add‐on therapy

In this secondary replication step, the SNPs prioritized in the PEAR2 and PEAR monotherapy meta‐analysis (seven SNPs for ∆DBP and two SNPs for ∆SBP) were tested for an association with a change in BP response in atenolol add‐on treated participants from PEAR. For the association with ∆SBP, none of the SNPs replicated in the PEAR add‐on atenolol therapy. However, for the association with ∆DBP, one SNP, rs45545233, was successfully replicated in the PEAR add‐on cohort and met the Bonferroni correction. This SNP was present in solute carrier family 4 (*SLC4A1*) and was significantly associated with ∆DBP in the same direction as the discovery signal (*P* = 0.007, β = 4.97). This SNP is also an eQTL for dual specificity phosphatase 3. The SNP was significantly associated with a decreased DBP response (**Table ** [3](#cts12643-tbl-0003)). Because of the comparatively lower allele frequency of this SNP (minor allele frequency = 0.09), we also tested for the association of this SNP using a dominant model, and it was significantly associated with a decreased DBP response in a dominant model as well (**Figure ** [3](#cts12643-fig-0003)). The regional plot for this region is presented in **Figure ** [S4](#cts12643-sup-0004) **.**

### Table 3.

| SNP | CHR | Base‐pair position (hg19 position) | Nearest gene | Minor allele | MAF | PEAR‐2/PEAR meta‐analysis DBP response |  |  | PEAR add‐on DBP response (replication) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| β | SE | P value | β | SE | One‐sided P value |  |  |  |  |  |  |
| rs45545233 | 17 | 42338352 | SLC4A1 | C | 0.0829 | 4.57 | 0.98 | 3.43E‐06 | 4.97 | 2.03 | 0.007 |
| rs11177995 | 12 | 70350702 | 134 kb 3′ of RAB3IP | T | 0.315 | 1.71 | 0.38 | 8.79E‐06 | 0.890 | 0.43 | 0.02 |

Table 3 Caption: Replication of SNPs from PEAR‐2 and PEAR meta‐analysis in the PEAR atenolol add‐on cohort

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8b11/6742943/33ed388d6bcf/CTS-12-497-g003.jpg)

Diastolic blood pressure (DBP) change post–metoprolol monotherapy treatment among European Americans post–meta‐analysis of the Pharmacogenomic Evaluation of Antihypertensive Responses‐2 (PEAR‐2) and Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) studies and in the PEAR study atenolol add‐on therapy cohort by the Solute Carrier Family 4 (SLC4A1) rs45545233 genotype.

Another SNP in RAB3A Interacting Protein (*RAB3IP*), rs11177995, met the nominal *P* value for replication and was associated with the ∆DBP (*P* = 0.02, β = 0.89) and was associated with a significantly poor BP response to β‐blocker treatment. The regional plot for this region is presented in **Figure ** [S5](#cts12643-sup-0005) **.**

## Discussion

We sought to identify the genetic determinants of β‐blocker BP response in a cohort of hypertensive European Americans. Using a multistage genomewide association approach, we were able to identify multiple variants that encode for proteins that have a biologically plausible involvement in HTN and BP regulation. Through our initial discovery and replication efforts, we identified rs294610 in *FGD5* that was significantly associated with better BP response following metoprolol treatment and was replicated in an independent cohort of European Americans treated with atenolol. This SNP was significantly associated with BP response in which the variant carriers had significantly better BP response when compared with noncarriers. *FGD5* belongs to the family of FGD5‐guanine nucleotide exchange factor (*FGD5‐GEF*) and encodes for FYVE, RhoGEF, and PH domain containing 5. Several GWAS studies have identified the association of *FGD5* with several BP‐related phenotypes. A GWAS study published by Ehret *et al*.[25](#cts12643-bib-0025) that aimed at identifying the genetics of BP regulation using 342,415 European American hypertensive patients discovered *FGD5* as one of the novel associated loci with DBP and SBP. The Welcome Trust Case Control Consortium used an intermediate approach between the genomewide and candidate gene approach by testing the association with HTN for genes expressed in endothelial. They identified *FGD5* as one of the associated loci, which was then replicated in the Nordic Diltiazem Study (NORDIL) study.[26](#cts12643-bib-0026) This gene may be implicated in HTN through its role in vascular remodeling, which is an important factor in the development and progression of HTN pathology. In support of the involvement of *FGD5* in HTN pathophysiology, studies in mice have reported the regulatory role of *FGD5* in endothelial cell‐specific apoptosis, with a resultant effect on vascular pruning.[27](#cts12643-bib-0027) Moreover, *in vitro* studies using human cell lines have documented the involvement of *FGD5 as* an important factor in the proangiogenic action of endothelial growth factor,[28](#cts12643-bib-0028) further highlighting the potential role of *FGD5* in the development/progression of vasculature‐related diseases including HTN. Studies have shown the positive effect of β‐blockers on the endothelial dysfunction[29](#cts12643-bib-0029) and several mechanistic studies indicate toward the involvement of the endothelial and vasculature‐related mechanism of the BP‐lowering effects of the β‐blockers.[30](#cts12643-bib-0030) Further studies are needed to understand the functional underpinning of this SNP in the *FGD5* gene in improved BP response to β‐blockers.

We also used a secondary replication approach to validate additional signals identified in the meta‐analysis between the two studies (PEAR‐2 and PEAR) with an independent atenolol add‐on therapy cohort from PEAR. Using this approach, we were able to identify and replicate a signal rs45545233 in *SLC4A1* that was significantly associated with a decreased BP response to β‐blockers. The *SLC4A1* signal was also one of the top associations in the discovery analysis of the PEAR‐2 metoprolol‐treated cohort. *SLC4A1* encodes for the glycoprotein in the plasma membrane–band 3 anion transporter and is part of the anion exchanger family. It is primarily expressed in the erythrocyte membrane and the collecting ducts of the kidney, wherein it facilitates the electro‐neutral exchange of chloride‐bicarbonate exchange and the transport of glucose and water. Studies have shown that mutations in *SLC4A1* results in distal renal tubular acidosis.[31](#cts12643-bib-0031) A study by Kokubo *et al*.[32](#cts12643-bib-0032) identified polymorphisms in *SLC4A1* to be significantly associated with HTN as well as BP variation in a Japanese population. However, the exact mechanism of the gene and the underlying functional underpinning of this association are needed to better understand the influence on the β‐blocker BP response phenotype.

The other SNP that was validated in the PEAR‐atenolol add‐on cohort was rs11177995 in the *RAB3IP* gene, which encodes for the RAB3A‐interacting protein. This protein belongs to the family of RAB proteins that are involved in the Rab guanyl‐nucleotide exchange factor activity that has been implicated as one of the pathways in HTN. Furthermore, a recent study by our group investigated a locus on chromosome 12 that has been previously associated with BP response to thiazides via targeted deep‐sequencing results. Through these deep‐sequencing efforts, we identified a potentially functional SNP in bestrophin 3 (*BEST3*), which was significantly associated with BP response to HCTZ. This SNP rs61747221 is a missense mutation and an eQTL for the *RAB3IP* gene, which further highlights the potential involvement of *RAB3IP* in BP regulation.[33](#cts12643-bib-0033)

A recent report by Evangelou *et al*.[34](#cts12643-bib-0034) which used data from about 1 million patients and performed the largest GWAS to date for BP, identified genetic variants in *FGD5* to be associated with baseline BP. They also reported *RAB3IP* to be one of the distal genes associated with the genetic variants they identified using chromatin interaction Hi‐C data. Although Evangelou *et al*.[34](#cts12643-bib-0034) reported *FGD5* and *RAB3IP* to be associated with BP, they did not report the exact SNPs identified in our study, possibly because of the differences in the LD‐pruning and loci‐defining strategies between the two studies. Given the overlap between pathways related to BP regulation and BP response to β‐blockers, these reports further strengthen our genetic findings in support of these loci and indicate the potential of these genes and related pathways as potential targets for BP response. Other strengths of our study include a well‐characterized BP response phenotype and the use of multiple replication cohorts in addition to the stringent criteria used for prioritization and Bonferroni‐corrected replication that led to the identification of two associations in *FGD5* and *SLC4A1,* both of which have been previously cited as candidate genes for HTN. Also, the use of the participants with the same ancestry in the discovery and replication cohorts further strengthens these findings.

However, we do acknowledge the limitations of our study as well. The relatively small sample size of our study limits the detection of signals with only large effect sizes, as is the case in most pharmacogenomic GWAS studies. However, it should be noted that pharmacogenomic variants tend to have large to moderate effect sizes compared with variants associated with complex diseases even for similar sample sizes.[35](#cts12643-bib-0035) Furthermore, even though we expect our identified signals to be true for all β‐blockers, especially all selective β1 blockers, we do acknowledge that our discovery and replication cohorts did not use the same β‐blocker (discovery = metoprolol, replication = atenolol), which is an additional limitation of our study. Our original discovery finding of *FGD5* that was replicated in the atenolol‐monotherapy cohort was not replicated in the PEAR‐atenolol add‐on therapy cohort. We recognize that add‐on therapy BP response may not be a pure β‐blocker response and thus may not be an optimal validation cohort. The importance of *FGD5* association should not be discounted because of a lack of association in atenolol add‐on therapy, especially given that it successfully replicated in the monotherapy arm. Also, the strong evidence in the literature of the involvement of *FGD5* further indicates toward the plausible role in the antihypertensive response to β‐blockers.

## Conclusion

In conclusion, in using a multistaged GWAS approach, we were able to identify variants in the *FGD5, SLC4A1, and RAB3IP* genes that were significantly associated with a BP response to β‐blockers. Although all of these genes are involved in pathways related to HTN and BP regulation, further studies understanding the exact mechanism of these genes and variants are needed. Elucidation of the mechanistic underpinnings of these genes can help shed further light on the pathways that influence the β‐blocker antihypertensive response.

## Funding

Both the PEAR‐2 and PEAR studies were supported by the National Institutes of Health Pharmacogenomics Research Network Grant U01‐GM074492 and the National Center for Advancing Translational Sciences under award numbers UL1 TR000064 (University of Florida), UL1 TR000454 (Emory University), and UL1 TR000135 (Mayo Clinic). PEAR was also supported by funds from the Mayo Foundation. Additional support for this project comes from National Institutes of Health Grant KL2 TR001429 (C.W.M.).

## Conflict of Interest

The authors declared no competing interests for this work.

## Author Contributions

S.S. and J.A.J. wrote the manuscript. S.S., C.W.M., Y.G., K.R.B., E.B., A.B.C., J.G.G., S.T.T., R.M.C.D., and J.A.J. designed the research. S.S. performed the research. S.S. and N.E. analyzed the data.

## Supporting information

## Acknowledgments

The authors thank the participants, staff, and study physicians of the PEAR and PEAR‐2 studies for their contributions.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Redon, J. et al Impact of hypertension on mortality and cardiovascular disease burden in patients with cardiovascular risk factors from a general practice setting: the ESCARVAL‐risk study. J. Hypertens. 34, 1075–1083 (2016).  [DOI](https://doi.org/10.1097/HJH.0000000000000930) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27074896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hypertens.&title=Impact%20of%20hypertension%20on%20mortality%20and%20cardiovascular%20disease%20burden%20in%20patients%20with%20cardiovascular%20risk%20factors%20from%20a%20general%20practice%20setting:%20the%20ESCARVAL%E2%80%90risk%20study&author=J.%20Redon&volume=34&publication_year=2016&pages=1075-1083&pmid=27074896&doi=10.1097/HJH.0000000000000930&)

2. Pereira, M. , Lunet, N. , Azevedo, A. & Barros, H. Differences in prevalence, awareness, treatment and control of hypertension between developing and developed countries. J. Hypertens. 27, 963–975 (2009).  [DOI](https://doi.org/10.1097/hjh.0b013e3283282f65) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19402221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hypertens.&title=Differences%20in%20prevalence,%20awareness,%20treatment%20and%20control%20of%20hypertension%20between%20developing%20and%20developed%20countries&author=M.%20Pereira&author=N.%20Lunet&author=A.%20Azevedo&author=H.%20Barros&volume=27&publication_year=2009&pages=963-975&pmid=19402221&doi=10.1097/hjh.0b013e3283282f65&)

3. Whelton, P.K. et al 2017 ACC/AHA/AAPA/ABC/ACPM/AGS/APhA/ASH/ASPC/NMA/PCNA Guideline for the prevention, detection, evaluation, and management of high blood pressure in adults: executive summary: a report of the American College of Cardiology/American Heart Association Task Force on Clinical Practice Guidelines. J. Am. Coll. Cardiol. 71, 2199–2269 (2018).  [DOI](https://doi.org/10.1016/j.jacc.2017.11.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29146535/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=2017%20ACC/AHA/AAPA/ABC/ACPM/AGS/APhA/ASH/ASPC/NMA/PCNA%20Guideline%20for%20the%20prevention,%20detection,%20evaluation,%20and%20management%20of%20high%20blood%20pressure%20in%20adults:%20executive%20summary:%20a%20report%20of%20the%20American%20College%20of%20Cardiology/American%20Heart%20Association%20Task%20Force%20on%20Clinical%20Practice%20Guidelines&author=P.K.%20Whelton&volume=71&publication_year=2018&pages=2199-2269&pmid=29146535&doi=10.1016/j.jacc.2017.11.006&)

4. Black, H.R. et al Principal results of the Controlled Onset Verapamil Investigation of Cardiovascular End Points (CONVINCE) trial. JAMA 289, 2073–2082 (2003).  [DOI](https://doi.org/10.1001/jama.289.16.2073) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12709465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Principal%20results%20of%20the%20Controlled%20Onset%20Verapamil%20Investigation%20of%20Cardiovascular%20End%20Points%20(CONVINCE)%20trial&author=H.R.%20Black&volume=289&publication_year=2003&pages=2073-2082&pmid=12709465&doi=10.1001/jama.289.16.2073&)

5. Thoenes, M. et al Antihypertensive drug therapy and blood pressure control in men and women: an international perspective. J. Hum. Hypertens. 24, 336–344 (2010).  [DOI](https://doi.org/10.1038/jhh.2009.76) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19798089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hum.%20Hypertens.&title=Antihypertensive%20drug%20therapy%20and%20blood%20pressure%20control%20in%20men%20and%20women:%20an%20international%20perspective&author=M.%20Thoenes&volume=24&publication_year=2010&pages=336-344&pmid=19798089&doi=10.1038/jhh.2009.76&)

6. Padmanabhan, S. , Paul, L. & Dominczak, A.F. The pharmacogenomics of anti‐hypertensive therapy. Pharmaceuticals (Basel) 3, 1779–1791 (2010).  [DOI](https://doi.org/10.3390/ph3061779) | [PMC free article](/articles/PMC4033952/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27713329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmaceuticals%20(Basel)&title=The%20pharmacogenomics%20of%20anti%E2%80%90hypertensive%20therapy&author=S.%20Padmanabhan&author=L.%20Paul&author=A.F.%20Dominczak&volume=3&publication_year=2010&pages=1779-1791&pmid=27713329&doi=10.3390/ph3061779&)

7. Gong, Y. et al Pharmacogenomic genome‐wide meta‐analysis of blood pressure response to β‐blockers in hypertensive African Americans. Hypertension 67, 556–563 (2016).  [DOI](https://doi.org/10.1161/HYPERTENSIONAHA.115.06345) | [PMC free article](/articles/PMC4752391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26729753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Pharmacogenomic%20genome%E2%80%90wide%20meta%E2%80%90analysis%20of%20blood%20pressure%20response%20to%20%CE%B2%E2%80%90blockers%20in%20hypertensive%20African%20Americans&author=Y.%20Gong&volume=67&publication_year=2016&pages=556-563&pmid=26729753&doi=10.1161/HYPERTENSIONAHA.115.06345&)

8. Brewster, L.M. , van Montfrans, G.A. & Kleijnen, J. Systematic review: antihypertensive drug therapy in black patients. Ann. Intern. Med. 141, 614–627 (2004).  [DOI](https://doi.org/10.7326/0003-4819-141-8-200410190-00009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15492341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Intern.%20Med.&title=Systematic%20review:%20antihypertensive%20drug%20therapy%20in%20black%20patients&author=L.M.%20Brewster&author=G.A.%20van%20Montfrans&author=J.%20Kleijnen&volume=141&publication_year=2004&pages=614-627&pmid=15492341&doi=10.7326/0003-4819-141-8-200410190-00009&)

9. Munroe, P.B. , Barnes, M.R. & Caulfield, M.J. Advances in blood pressure genomics. Circ. Res. 112, 1365–1379 (2013).  [DOI](https://doi.org/10.1161/CIRCRESAHA.112.300387) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23661711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Res.&title=Advances%20in%20blood%20pressure%20genomics&author=P.B.%20Munroe&author=M.R.%20Barnes&author=M.J.%20Caulfield&volume=112&publication_year=2013&pages=1365-1379&pmid=23661711&doi=10.1161/CIRCRESAHA.112.300387&)

10. Ehret, G.B. et al Genetic variants in novel pathways influence blood pressure and cardiovascular disease risk. Nature 478, 103–109 (2011).  [DOI](https://doi.org/10.1038/nature10405) | [PMC free article](/articles/PMC3340926/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21909115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variants%20in%20novel%20pathways%20influence%20blood%20pressure%20and%20cardiovascular%20disease%20risk&author=G.B.%20Ehret&volume=478&publication_year=2011&pages=103-109&pmid=21909115&doi=10.1038/nature10405&)

11. Fontana, V. , Luizon, M.R. & Sandrim, V.C. An update on the pharmacogenetics of treating hypertension. J. Hum. Hypertens. 29, 283–291 (2015).  [DOI](https://doi.org/10.1038/jhh.2014.76) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25355012/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hum.%20Hypertens.&title=An%20update%20on%20the%20pharmacogenetics%20of%20treating%20hypertension&author=V.%20Fontana&author=M.R.%20Luizon&author=V.C.%20Sandrim&volume=29&publication_year=2015&pages=283-291&pmid=25355012&doi=10.1038/jhh.2014.76&)

12. Levy, D. et al Genome‐wide association study of blood pressure and hypertension. Nat. Genet. 41, 677–687 (2009).  [DOI](https://doi.org/10.1038/ng.384) | [PMC free article](/articles/PMC2998712/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19430479/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Genome%E2%80%90wide%20association%20study%20of%20blood%20pressure%20and%20hypertension&author=D.%20Levy&volume=41&publication_year=2009&pages=677-687&pmid=19430479&doi=10.1038/ng.384&)

13. Johnson, J.A. et al Pharmacogenomics of antihypertensive drugs: rationale and design of the Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) study. Am. Heart J. 157, 442–449 (2009).  [DOI](https://doi.org/10.1016/j.ahj.2008.11.018) | [PMC free article](/articles/PMC2671287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19249413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20Heart%20J.&title=Pharmacogenomics%20of%20antihypertensive%20drugs:%20rationale%20and%20design%20of%20the%20Pharmacogenomic%20Evaluation%20of%20Antihypertensive%20Responses%20(PEAR)%20study&author=J.A.%20Johnson&volume=157&publication_year=2009&pages=442-449&pmid=19249413&doi=10.1016/j.ahj.2008.11.018&)

14. Hamadeh, I.S. et al Impact of CYP2D6 polymorphisms on clinical efficacy and tolerability of metoprolol tartrate. Clin. Pharmacol. Ther. 96, 175–181 (2014).  [DOI](https://doi.org/10.1038/clpt.2014.62) | [PMC free article](/articles/PMC4111800/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24637943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Impact%20of%20CYP2D6%20polymorphisms%20on%20clinical%20efficacy%20and%20tolerability%20of%20metoprolol%20tartrate&author=I.S.%20Hamadeh&volume=96&publication_year=2014&pages=175-181&pmid=24637943&doi=10.1038/clpt.2014.62&)

15. National Center for Biotechnology Information (NCBI) ‐ The database of Genotypes and Phenotypes (dbGap) <https://www.ncbi.nlm.nih.gov/gap/">>. Accessed April 18, 2019.  [https://www.ncbi.nlm.nih.gov/gap/">](https://www.ncbi.nlm.nih.gov/gap/)

16. Turner, S.T. et al Power to identify a genetic predictor of antihypertensive drug response using different methods to measure blood pressure response. J. Transl. Med. 10, 47 (2012).  [DOI](https://doi.org/10.1186/1479-5876-10-47) | [PMC free article](/articles/PMC3342146/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22413836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Transl.%20Med.&title=Power%20to%20identify%20a%20genetic%20predictor%20of%20antihypertensive%20drug%20response%20using%20different%20methods%20to%20measure%20blood%20pressure%20response&author=S.T.%20Turner&volume=10&publication_year=2012&pages=47&pmid=22413836&doi=10.1186/1479-5876-10-47&)

17. Singh, S. et al Genome wide association study identifies the HMGCS2 locus to be associated with chlorthalidone induced glucose increase in hypertensive patients. J. Am. Heart Assoc. 7, e007339 (2018).  [DOI](https://doi.org/10.1161/JAHA.117.007339) | [PMC free article](/articles/PMC5907544/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29523524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Heart%20Assoc.&title=Genome%20wide%20association%20study%20identifies%20the%20HMGCS2%20locus%20to%20be%20associated%20with%20chlorthalidone%20induced%20glucose%20increase%20in%20hypertensive%20patients&author=S.%20Singh&volume=7&publication_year=2018&pages=e007339&pmid=29523524&doi=10.1161/JAHA.117.007339&)

18. Das, S. et al Next‐generation genotype imputation service and methods. Nat. Genet. 48, 1284–1287 (2016).  [DOI](https://doi.org/10.1038/ng.3656) | [PMC free article](/articles/PMC5157836/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27571263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Next%E2%80%90generation%20genotype%20imputation%20service%20and%20methods&author=S.%20Das&volume=48&publication_year=2016&pages=1284-1287&pmid=27571263&doi=10.1038/ng.3656&)

19. Gong, Y. et al PTPRD gene associated with blood pressure response to atenolol and resistant hypertension. J. Hypertens. 33, 2278–2285 (2015).  [DOI](https://doi.org/10.1097/HJH.0000000000000714) | [PMC free article](/articles/PMC4788379/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26425837/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hypertens.&title=PTPRD%20gene%20associated%20with%20blood%20pressure%20response%20to%20atenolol%20and%20resistant%20hypertension&author=Y.%20Gong&volume=33&publication_year=2015&pages=2278-2285&pmid=26425837&doi=10.1097/HJH.0000000000000714&)

20. Nicolae, D.L. et al Trait‐associated SNPs are more likely to be eQTLs: annotation to enhance discovery from GWAS. PLoS Genet. 6, e1000888 (2010).  [DOI](https://doi.org/10.1371/journal.pgen.1000888) | [PMC free article](/articles/PMC2848547/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20369019/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Trait%E2%80%90associated%20SNPs%20are%20more%20likely%20to%20be%20eQTLs:%20annotation%20to%20enhance%20discovery%20from%20GWAS&author=D.L.%20Nicolae&volume=6&publication_year=2010&pages=e1000888&pmid=20369019&doi=10.1371/journal.pgen.1000888&)

21. GTEx Consortium . The genotype‐tissue expression (GTEx) project. Nat. Genet. 45, 580–585 (2013).  [DOI](https://doi.org/10.1038/ng.2653) | [PMC free article](/articles/PMC4010069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23715323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=The%20genotype%E2%80%90tissue%20expression%20(GTEx)%20project&volume=45&publication_year=2013&pages=580-585&pmid=23715323&doi=10.1038/ng.2653&)

22. Machiela, M.J. & Chanock, S.J. LDlink: a web‐based application for exploring population‐specific haplotype structure and linking correlated alleles of possible functional variants. Bioinformatics 31, 3555–3557 (2015).  [DOI](https://doi.org/10.1093/bioinformatics/btv402) | [PMC free article](/articles/PMC4626747/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26139635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LDlink:%20a%20web%E2%80%90based%20application%20for%20exploring%20population%E2%80%90specific%20haplotype%20structure%20and%20linking%20correlated%20alleles%20of%20possible%20functional%20variants&author=M.J.%20Machiela&author=S.J.%20Chanock&volume=31&publication_year=2015&pages=3555-3557&pmid=26139635&doi=10.1093/bioinformatics/btv402&)

23. Ruxton, G.D. & Neuhäuser, M. When should we use one‐tailed hypothesis testing? Methods Ecol. Evol. 1, 114–117 (2010).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Ecol.%20Evol.&title=When%20should%20we%20use%20one%E2%80%90tailed%20hypothesis%20testing?&author=G.D.%20Ruxton&author=M.%20Neuh%C3%A4user&volume=1&publication_year=2010&pages=114-117&)

24. Willer, C.J. , Li, Y. & Abecasis, G.R. METAL: fast and efficient meta‐analysis of genomewide association scans. Bioinformatics 26, 2190–2191 (2010).  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta%E2%80%90analysis%20of%20genomewide%20association%20scans&author=C.J.%20Willer&author=Y.%20Li&author=G.R.%20Abecasis&volume=26&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

25. Ehret, G.B. et al The genetics of blood pressure regulation and its target organs from association studies in 342,415 individuals. Nat. Genet. 48, 1171–1184 (2016).  [DOI](https://doi.org/10.1038/ng.3667) | [PMC free article](/articles/PMC5042863/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27618452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=The%20genetics%20of%20blood%20pressure%20regulation%20and%20its%20target%20organs%20from%20association%20studies%20in%20342,415%20individuals&author=G.B.%20Ehret&volume=48&publication_year=2016&pages=1171-1184&pmid=27618452&doi=10.1038/ng.3667&)

26. Larsson, E. et al Hypertension and genetic variation in endothelial‐specific genes. PLoS ONE 8, e62035 (2013).  [DOI](https://doi.org/10.1371/journal.pone.0062035) | [PMC free article](/articles/PMC3639261/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23637959/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Hypertension%20and%20genetic%20variation%20in%20endothelial%E2%80%90specific%20genes&author=E.%20Larsson&volume=8&publication_year=2013&pages=e62035&pmid=23637959&doi=10.1371/journal.pone.0062035&)

27. Cheng, C. et al Endothelial cell‐specific FGD5 involvement in vascular pruning defines neovessel fate in mice. Circulation 125, 3142–3158 (2012).  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.111.064030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22661514/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Endothelial%20cell%E2%80%90specific%20FGD5%20involvement%20in%20vascular%20pruning%20defines%20neovessel%20fate%20in%20mice&author=C.%20Cheng&volume=125&publication_year=2012&pages=3142-3158&pmid=22661514&doi=10.1161/CIRCULATIONAHA.111.064030&)

28. Kurogane, Y. et al FGD5 mediates proangiogenic action of vascular endothelial growth factor in human vascular endothelial cells. Arterioscler. Thromb. Vasc. Biol. 32, 988–996 (2012).  [DOI](https://doi.org/10.1161/ATVBAHA.111.244004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22328776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler.%20Thromb.%20Vasc.%20Biol.&title=FGD5%20mediates%20proangiogenic%20action%20of%20vascular%20endothelial%20growth%20factor%20in%20human%20vascular%20endothelial%20cells&author=Y.%20Kurogane&volume=32&publication_year=2012&pages=988-996&pmid=22328776&doi=10.1161/ATVBAHA.111.244004&)

29. Peller, M. et al Influence of beta‐blockers on endothelial function: a meta‐analysis of randomized controlled trials. Cardiol. J. 22, 708–716 (2015).  [DOI](https://doi.org/10.5603/CJ.a2015.0042) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26202651/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiol.%20J.&title=Influence%20of%20beta%E2%80%90blockers%20on%20endothelial%20function:%20a%20meta%E2%80%90analysis%20of%20randomized%20controlled%20trials&author=M.%20Peller&volume=22&publication_year=2015&pages=708-716&pmid=26202651&doi=10.5603/CJ.a2015.0042&)

30. Poirier, L. & Tobe, S.W. Contemporary use of β‐blockers: clinical relevance of subclassification. Can. J. Cardiol. 30, S9–S15 (2014).  [DOI](https://doi.org/10.1016/j.cjca.2013.12.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24684855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Can.%20J.%20Cardiol.&title=Contemporary%20use%20of%20%CE%B2%E2%80%90blockers:%20clinical%20relevance%20of%20subclassification&author=L.%20Poirier&author=S.W.%20Tobe&volume=30&publication_year=2014&pages=S9-S15&pmid=24684855&doi=10.1016/j.cjca.2013.12.001&)

31. Jarolim, P. et al Autosomal dominant distal renal tubular acidosis is associated in three families with heterozygosity for the R589H mutation in the AE1 (band 3) Cl‐/HCO3‐ exchanger. J. Biol. Chem. 273, 6380–6388 (1998).  [DOI](https://doi.org/10.1074/jbc.273.11.6380) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9497368/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Autosomal%20dominant%20distal%20renal%20tubular%20acidosis%20is%20associated%20in%20three%20families%20with%20heterozygosity%20for%20the%20R589H%20mutation%20in%20the%20AE1%20(band%203)%20Cl%E2%80%90/HCO3%E2%80%90%20exchanger&author=P.%20Jarolim&volume=273&publication_year=1998&pages=6380-6388&pmid=9497368&doi=10.1074/jbc.273.11.6380&)

32. Kokubo, Y. et al Association of sixty‐one non‐synonymous polymorphisms in forty‐one hypertension candidate genes with blood pressure variation and hypertension. Hypertens. Res. 29, 611–619 (2006).  [DOI](https://doi.org/10.1291/hypres.29.611) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17137217/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertens.%20Res.&title=Association%20of%20sixty%E2%80%90one%20non%E2%80%90synonymous%20polymorphisms%20in%20forty%E2%80%90one%20hypertension%20candidate%20genes%20with%20blood%20pressure%20variation%20and%20hypertension&author=Y.%20Kokubo&volume=29&publication_year=2006&pages=611-619&pmid=17137217&doi=10.1291/hypres.29.611&)

33. Singh, S. et al Targeted sequencing identifies a missense variant in the BEST3 gene associated with antihypertensive response to hydrochlorothiazide. Pharmacogenet. Genomics 28, 251–255 (2018).  [DOI](https://doi.org/10.1097/FPC.0000000000000353) | [PMC free article](/articles/PMC6262886/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30289819/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Targeted%20sequencing%20identifies%20a%20missense%20variant%20in%20the%20BEST3%20gene%20associated%20with%20antihypertensive%20response%20to%20hydrochlorothiazide&author=S.%20Singh&volume=28&publication_year=2018&pages=251-255&pmid=30289819&doi=10.1097/FPC.0000000000000353&)

34. Evangelou, E. et al Genetic analysis of over 1 million people identifies 535 new loci associated with blood pressure traits. Nat. Genet. 50, 1412–1425 (2018).  [DOI](https://doi.org/10.1038/s41588-018-0205-x) | [PMC free article](/articles/PMC6284793/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30224653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Genetic%20analysis%20of%20over%201%20million%20people%20identifies%20535%20new%20loci%20associated%20with%20blood%20pressure%20traits&author=E.%20Evangelou&volume=50&publication_year=2018&pages=1412-1425&pmid=30224653&doi=10.1038/s41588-018-0205-x&)

35. Maranville, J.C. & Cox, N.J. Pharmacogenomic variants have larger effect sizes than genetic variants associated with other dichotomous complex traits. Pharmacogenomics J. 16, 388–392 (2016).  [DOI](https://doi.org/10.1038/tpj.2015.47) | [PMC free article](/articles/PMC4704992/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26149738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Pharmacogenomic%20variants%20have%20larger%20effect%20sizes%20than%20genetic%20variants%20associated%20with%20other%20dichotomous%20complex%20traits&author=J.C.%20Maranville&author=N.J.%20Cox&volume=16&publication_year=2016&pages=388-392&pmid=26149738&doi=10.1038/tpj.2015.47&)
