# Single Nucleotide Variant in Nucleoporin 107 may be Predictive of Sensitivity to Chemotherapy in Patient with Ovarian Cancer

## Metadata
**Authors:** Shaheen Alanee, Kristin Delfino, Andrew Wilber, Kathy Robinson, Laurent Brard, Assaad Semaan
**Journal:** Pharmacogenetics and genomics
**Date:** 2017 Jul
**DOI:** [10.1097/FPC.0000000000000288](https://doi.org/10.1097/FPC.0000000000000288)
**PMID:** 28562428
**PMCID:** PMC6298606
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6298606/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6298606/pdf/nihms875390.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6298606/pdf/nihms875390.pdf)

## Abstract

**Background:** 
Alterations in nuclear pore complex (NPC) genes have been previously associated with response to chemotherapy. By using agnostic exome sequencing, we envisioned that new alleles in NPC genes, predictive of sensitivity to platinum treatment, could be discovered.

**Methods:** 
Twenty-two Platinum-Sensitive (Pt-S) and 6 Platinum-Resistant (Pt-R) ovarian cancer patients were tested. Platinum sensitivity was defined as disease free survival greater than 6 months. Next-generation sequencing of exomes was used to compare Pt-S and Pt-R patients. Single nucleotide variants (SNVs) associated with platinum sensitivity in NPC genes (n=30 genes) were identified.

**Results:** 
SNVs in three NPC genes were associated with response to platinum on univariate analysis. SNV rs79419059 (10T>C) in Nucleoporin 107 (Nup107) was associated with platinum resistance (p=0.0061), while rs2302811 (3662-4A>G) in Nucleoporin 188 (Nup188) and rs77246077 (3420-67T>A) in Nucleoporin 214 (Nup214) were associated with platinum sensitivity (p= 0.0483 and 0.0091 respectively). Controlling for other confounders, multivariate age adjusted Cox proportional hazard analysis showed rs79419059 to be significantly associated with platinum resistance (Odds ratio 4.519, 95% confidence interval 1.317–15.501, p=0.0457).

**Conclusion:** 
We identified variant in the 3UTR region Nup107 unique to sensitivity to platinum in ovarian cancer. With validation of this variant, it is possible that a new marker predictive of patient response may be identified.

Keywords: Chemotherapy, exome sequencing, drug-sensitivity, nuclear pore genes, single nucleotide polymorphism, mutation

### Background

Alterations in nuclear pore complex (NPC) genes have been previously associated with response to chemotherapy. By using agnostic exome sequencing, we envisioned that new alleles in NPC genes, predictive of sensitivity to platinum treatment, could be discovered.

### Methods

Twenty-two Platinum-Sensitive (Pt-S) and 6 Platinum-Resistant (Pt-R) ovarian cancer patients were tested. Platinum sensitivity was defined as disease free survival greater than 6 months. Next-generation sequencing of exomes was used to compare Pt-S and Pt-R patients. Single nucleotide variants (SNVs) associated with platinum sensitivity in NPC genes (n=30 genes) were identified.

### Results

SNVs in three NPC genes were associated with response to platinum on univariate analysis. SNV rs79419059 (10T>C) in Nucleoporin 107 (*Nup107*) was associated with platinum resistance (p=0.0061), while rs2302811 (3662-4A>G) in Nucleoporin 188 (*Nup188*) and rs77246077 (3420-67T>A) in Nucleoporin 214 (*Nup214*) were associated with platinum sensitivity (p= 0.0483 and 0.0091 respectively). Controlling for other confounders, multivariate age adjusted Cox proportional hazard analysis showed rs79419059 to be significantly associated with platinum resistance (Odds ratio 4.519, 95% confidence interval 1.317–15.501, p=0.0457).

### Conclusion

We identified variant in the 3UTR region *Nup107* unique to sensitivity to platinum in ovarian cancer. With validation of this variant, it is possible that a new marker predictive of patient response may be identified.

## Introduction

The incidence of ovarian cancer, in the United States, ranks eighth among all cancers (excluding skin cancer), but fifth in terms of age-adjusted mortality.[[1](#R1)] The introduction of Paclitaxel to platinum-based treatment in the 1990s significantly improved survival in ovarian cancer patients.[[2](#R2)] Today, the combination of Carboplatin and Paclitaxel, in addition to optimal surgical cytoreduction, represents the standard of care in epithelial ovarian carcinoma (EOC) treatment. Although chemotherapy achieves a clinical response in most cases, recurrence remains a major problem due to the appearance of drug-resistance from residual tumor cells.[[3](#R3)] The absence of effective screening methods to detect early-stage disease, and the propensity of ovarian cancer cells to develop into chemotherapy-resistant clones substantially decreases the five-year overall survival of ovarian cancer patients, underscoring the need for more effective therapies.[[4](#R4)]

The nuclear pore complex (NPC) functions to regulate nucleocytoplasmic transport and chromatin organization and may exert control over cancer cells behavior and response to therapy. The NPC assembles in a modular form from multiples of eight nucleoporin (NUP) subcomplexes with a central framework consisting of a symmetric pair of central inner rings (containing NUP107-160 subcomplexes). Asymmetric annular rings line the cytoplasmic (containing the NUP88-NUP214 subcomplex)[[5](#R5)] and nuclear (containing NUP153)[[6](#R6)] faces of the pore, whereas asymmetric filamentous structures project from the annular rings into the nucleus (containing NUP153-Tpr)[[6](#R6)] or cytoplasm (containing NUP358)[[7](#R7)]. Nuclear-cytoplasmic trafficking of RNA, ribosomes, important regulators of transcription, cell cycle inhibitors, and specific drugs targets is tightly regulated by the nuclear pore complex and alterations in the NPC could potentially alter the availability of some drugs in the nucleus and thus increase/decrease cancer cells response to major chemotherapeutic agents[[8](#R8)–[10](#R10)].

In this study, we sought to use agnostic whole exome sequencing of germline DNA to identify variants in NPC genes that may predict response to platinum-based chemotherapy in ovarian cancer patients. We hypothesized that germline genetic polymorphisms in NPC genes are potentially important predictors of treatment response defined as progression free survival 6 months after chemotherapy. To investigate our hypothesis, we examined germline DNA extracted from saliva of ovarian cancer patients receiving adjuvant platinum-based chemotherapy. Using this approach, we successfully identified variants that may predict treatment sensitivity in patients receiving platinum drugs for ovarian cancer.

## Methods

### Ovarian Cancer Patients and Sample Collection

Saliva samples were collected from ovarian cancer patients seen at the Gynecologic Oncology clinics at Southern Illinois University School of Medicine in accordance with a protocol approved by the Springfield Committee for Research Involving Human Subjects (SCRIHS). Patients who already received or were scheduled to receive platinum-based adjuvant therapy were identified, and included in this study. Pre-operative pathology, post chemotherapy imaging results, surgical details, and post-operative pathology reports were obtained. Levels of response to platinum-based chemotherapy were then ascertained from the clinical data using definitions of response agreed on by the medical communities for ovarian cancer.

### Genomic DNA Isolation from Saliva

Saliva samples (at least 2 mL) were self-collected by enrolled patients using the Oragene DISCOVER all-in-one system (DNA Genotek Inc., Ottawa, Ontario, Canada). Samples from patients with a complete set of study data were isolated of high molecular weight genomic DNA according to supplied instructions. Collected saliva samples were briefly incubated in a 50°C water bath for at least 1 hour. The entire sample was then transferred to a 15 mL conical tube followed by an addition of PT-L2P solution (40 μL per mL). Samples were vortexed, incubated on ice for 10 minutes, and cleared of precipitate by centrifugation at room temperature for 15 minutes at 3,500 x*g*. The cleared solution was transferred to a new 15 mL conical tube and DNA precipitated by adding 1.2× volume of absolute ethanol. DNA was pelleted by centrifugation at room temperature for 10 minutes at 3,500 x*g* and washed twice with 70% ethanol. DNA was air-dried and rehydrated in Tris-EDTA (TE) solution. DNA concentration and purity was determined using a Nanodrop 2000 UV-Vis Spectrophotometer (Thermo Scientific). Samples with OD260/280 ratio greater than 1.8 were normalized to 50 ng/μL and a total of 250 ng separated through 1% agarose gel in Tris-Acetic acid-EDTA for inspection. Only DNA samples with OD260/260 ≥1.8 and no evidence of shearing or degradation were considered of sufficient quality for submission to Washington University Genomics Institute for additional quality control measurements and whole exome sequence analysis.

### Whole Exome Sequencing and Analysis

Genomic DNA was evaluated using a Bioanalyzer and sequenced using next generation sequencing (exome sequencing at 30x) provided by Genomics and Pathology Services at Washington University. Patients were divided into two groups based on platinum sensitivity. Disease free interval (DFI) was defined as the time from the end of adjuvant chemotherapy to the date of cancer recurrence. In the absence of recurrence, DFI was calculated from the time of completion of adjuvant chemotherapy to the last follow up date. Patients with DFI greater than 6 months, or with no recurrence, were classified as having platinum-sensitive disease. Patients with DFI less than 6 months were classified as having platinum-resistant disease. In this analysis, only variants with high call quality (at least 20 calls in any case or at least 20 in any control) were included.

## Statistical Analysis

The Kaplan-Meier method with the log-rank test was used to compare survival curves between the two groups (those with variant vs those without variant). Cox proportional hazards regression was used to analyze the age adjusted significant associations between the variants and progression-free survival controlling for stage and grade. Significance was determined at p-value<0.05. Power analysis was not possible because genetic variants associated with the outcome of interest are unknown; therefore we had no estimate of the population prevalence of the exposure (Variant) to perform power calculations. SNVs with significant association were also examined in genomic data on ovarian cancer patients available from the National Cancer Institute data portal. Information on time until progression was absent, hence all we could compare was the overall survival time between those who had or did not have the SNV that we identified in our exome sequencing experiment.

## Results

In total, there were 28 patients. Six patients were classified as platinum-resistant with a mean age of 58.2 years (49–68) and a mean DFI of 1.51 months. All 6 patients in this group developed recurrence of the disease. Twenty-five patients received carboplatin based first line therapy and the rest cisplatin. Twenty-two patients were classified as platinum-sensitive with a mean age of 58.4 years (48–73) and mean DFI of 23.7 months. Only 6 patients in this group developed recurrence (27.3%) as of the date of analysis. All patients were white except for one African American patient in the sensitive group ([Table 1](#T1)).

### Table 1.

| Classification |   | Resistant | Sensitive | Resistant % | Sensitive % |
| --- | --- | --- | --- | --- | --- |
| EOC Subtypes | Clear Cell | 0 | 2 | 0.0 | 9.1 |
| Endometrioid | 0 | 4 | 0.0 | 18.2 |  |
| Mucinous | 0 | 3 | 0.0 | 13.6 |  |
| Papillary Serous | 4 | 8 | 66.6 | 36.4 |  |
| Mixed Serous and Endometrioid | 0 | 2 | 0.0 | 9.1 |  |
| Mixed Serous and Papillary Serous | 2 | 2 | 33.3 | 9.1 |  |
| Mixed Serous, Endometrioid, and Clear Cell | 0 | 1 | 0.0 | 4.5 |  |
| TOTAL | 6 | 22 | 100.0 | 100.0 |  |
| Stage | IA | 0 | 2 | 0.0 | 9.1 |
| IB | 0 | 1 | 0.0 | 4.5 |  |
| IC | 0 | 7 | 0.0 | 31.8 |  |
| IIB | 1 | 5 | 16.7 | 22.7 |  |
| IIIB | 2 | 0 | 33.3 | 0.0 |  |
| IIIC | 0 | 6 | 0.0 | 27.3 |  |
| IVB | 2 | 1 | 33.3 | 4.5 |  |
| TOTAL | 5 | 22 | 83.3 | 100.0 |  |
| Grade | High | 6 | 21 | 100.0 | 95.5 |
| Low | 0 | 1 | 0.0 | 4.5 |  |
| TOTAL | 6 | 22 | 100.0 | 100.0 |  |
| Race | White | 6 | 21 | 100.0 | 95.5 |
| African American | 0 | 1 | 0.0 | 4.5 |  |
| TOTAL | 6 | 22 | 100.0 | 100.0 |  |

Table 1 Caption: Characteristics of patients included in the study stratified by histology, stage, grade, and race

Whole exome sequencing identified 519 variants associated with sensitivity to platinum-based chemotherapy. Six genes (IGF1R, LRP2, NCAPD2, AFF1, BRCA2, and PRKDC) included high mutational load that was associated with response to chemotherapy, but there was no individual SNV in these genes with significant association with the outcome of interest in our study. Of the 519 variants, 121 variants in 30 NPC genes of interest to this analysis ([Table 2](#T2)). This number was reduced to 3 SNVs in 3 genes that were associated with 6 months progression free survival on univariate analysis ([Table 3](#T3) and Kaplan Meier curves 1a-c). These localized to Nucleoporin 107 (*Nup107*), *Nup188*, and *Nup214*. SNV dbSNP ID rs79419059 (10T>C) in *Nup107* was detected in 2/22=9.1% of platinum-sensitive and 4/6=66.7% of platinum-resistant patients (p=0.0061), rs2302811 in *Nup188* was detected in 16/22=72.7% of platinum-sensitive and 3/6=50% of platinum-resistant patients (p=0.0483), and rs77246077 in *Nup214* was detected in 8/22=36.4% of platinum-sensitive and 0/6=0% of platinum-resistant patients (p=0.0091). Controlling for other confounders, multivariate age adjusted Cox proportional hazard analysis showed rs79419059 to be significantly associated with platinum resistance (Odds ratio 4.519, 95% confidence interval 1.317–15.501, p=0.0457). Because most patient were high grade, the variables of high grade and stage were not significant predictors of outcome in our analysis. Analysis of data available from the National Cancer Institute portal showed no significant association between these SNVs and the outcome of overall survival.

### Table 2.

| Gene Symbol | dbSNP ID (MAF) | Gene Symbol | dbSNP ID (MAF) | Gene Symbol | dbSNP ID (MAF) | Gene Symbol | dbSNP ID (MAF) | Gene Symbol | dbSNP ID (MAF) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| AHCTF1 | 12410563(0.2) | NUP155 | 3835198(0.3) | NUP214 | 2296710(0.4) | NUP88 | 2285748(0.3) | TPR | 76057663(0.1) |
| AHCTF1 | 2642990(0.4) | NUP155 | 150594(0.4) | NUP214 | 77246077(0.02) | NUP88 | 11209(0.4) | TPR | 3817586(0.1) |
| DDX19A | 1134074(0.3) | NUP155 | 13172306(0.2) | NUP214 | 11244329(0.2) | NUP88 | 2301739(0.3) | TPR | 3753565(0.1) |
| GLE1 | 2275260(0.3) | NUP160 | 7114813(0.4) | NUP35 | 1980160(0.2) | NUP93 | 1803870(0.09) | TPR | 61744267(0.09) |
| GLE1 | 869455(0.2) | NUP160 | 11039402(0.4) | NUP37 | 77619182(0.04) | NUP93 | 1561140(0.3) | TPR | 3841803(0.1) |
| GLE1 | 10760563(0.3) | NUP160 | 77201654(0.1) | NUP43 | 10872646(0.5) | NUP93 | 1347591(0.5) |   |   |
| NUP62 | 11547267(0.08) | NUP160 | 6485786(0.3) | NUP50 | 132874(0.3) | NUP93 | 145442551(0.2) |   |   |
| NUP62 | 1062798(0.3) | NUP160 | 2305982(0.2) | NUP50 | 132875(0.3) | NUP93 | 3764266(0.2) |   |   |
| NDC1 | 2273139(0.1) | NUP160 | 3816605(0.4) | NUP50 | 132881(0.3) | NUP93 | 1138295(0.2) |   |   |
| NDC1 | 12089209(0.2) | NUP188 | 17452428(0.3) | NUP50 | 226524(0.3) | NUP93 | 3214652(0.1) |   |   |
| NDC1 | 72662336(0.1) | NUP188 | 719703(0.3) | NUP50 | 6006950(0.5) | NUP98 | 1875(0.3) |   |   |
| NUP107 | 2259702(0.3) | NUP188 | 2302811(0.3) | NUP54 | 7657846(0.2) | NUP98 | 35404087(0.04) |   |   |
| NUP107 | 2259588(0.2) | NUP188 | 2287363(0.4) | NUP54 | 6532201(0.2) | NUP98 | 276901(0.3) |   |   |
| NUP107 | 11608404(0.3) | NUP205 | 73725182(0.2) | NUP54 | 6813732(0.2) | POM121 | 71554687(0.3) |   |   |
| NUP107 | 2701088(0.2) | NUP205 | 7810260(0.4) | NUP54 | 6840007(0.2) | POM121 | 782012234(0.006) |   |   |
| NUP107 | 17851475(0.2) | NUP205 | 10260691(0.3) | NUP54 | 13115827(0.2) | POM121 | 199633193(NA) |   |   |
| NUP107 | 7976687(0.2) | NUP205 | 12540688(0.2) | NUP54 | 10567711(0.2) | POM121 | 9955(0.1) |   |   |
| NUP107 | 79419059(0.02) | NUP210 | 2276748(0.4) | NUP54 | 61750814(0.1) | POM121 | 6465926(0.4) |   |   |
| NUP133 | 1.46E+08(0.2) | NUP210 | 354478(0.4) | NUP54 | 11097244(0.2) | POM121 | 117499978(0.06) |   |   |
| NUP133 | 1065675(0.3) | NUP210 | 514084(0.5) | NUP54 | 11558468(0.2) | POM121 | 398003(0.1) |   |   |
| NUP133 | 1065674(0.3) | NUP210 | 665064(0.5) | NUP54 | 36024409(0.2) | POM121 | 398016(0.3) |   |   |
| NUP133 | 11805194(0.2) | NUP210 | 3773301(0.2) | NUP54 | 11946345(0.4) | POM121 | 55838432(0.5) |   |   |
| NUP133 | 10916495(0.2) | NUP210 | 2280085(0.2) | NUP54 | 11558469(0.2) | NUPL1 | 11556093(0.4) |   |   |
| NUP153 | 61744976(0.1) | NUP210 | 6795271(0.2) | NUP54 | 735696(0.4) | RAE1 | 4811837(0.4) |   |   |
| NUP153 | 6906499(0.3) | NUP210 | 6807718(0.2) | NUP85 | 34126097(0.05) | RANBP2 | 700876(0.3) |   |   |
| NUP153 | 2274136(0.2) | NUP210 | 7628923(0.2) | NUP88 | 1071705(0.5) | SEC13 | 35670(0.2) |   |   |
| NUP153 | 2228375(0.1) | NUP214 | 77164468(0.07) | NUP88 | 739768(0.5) | SEC13 | 35671(0.2) |   |   |
| NUP153 | 1065057(0.3) | NUP214 | 2281902(0.3) | NUP88 | 748486(0.5) | SEC13 | 1642977(0.1) |   |   |
| NUP155 | 1045908(0.3) | NUP214 | 2281901(0.1) | NUP88 | 14231(0.5) | SEH1L | 1474043(0.2) |   |   |

Table 2 Caption: Single nucleotide variants in nuclear pore complex genes identified through next generation whole exome sequencing

### Table 3.

| Gene | Variant | Percent chemotherapy responders affected | Percent chemotherapy resistant affected | P value |
| --- | --- | --- | --- | --- |
| Nup107 | rs79419059 | 9.1% | 66.7% | 0.0061 |
| Nup188 | rs2302811 | 72.7% | 50.0% | 0.0483 |
| Nup214 | rs77246077 | 36.4% | 00.0% | 0.0091 |

Table 3 Caption: Single nucleotide variants in nuclear pore complex genes significantly associated with response to platinum based chemotherapy on univariate analysis

## Discussion

Identification of genetic alterations associated with response to chemotherapy is vital to maximize benefit from treatment while sparing individuals with chemotherapy-resistant and refractory cancers the side effects of such treatments. For individuals with ovarian cancer, somatic genetic changes have not been shown to contribute to improved selection of chemotherapy-sensitive patients. In this study, we analyzed germline DNA extracted from saliva samples from 28 ovarian cancer patients treated with platinum-based chemotherapy attempting to identify variants in NPC genes predictive of response to treatment. Our analysis identified a potentially significant association between the mutational status of *Nup107*, *Nup188*, and *Nup214* and sensitivity to platinum chemotherapy. On age adjusted multivariate analysis, a SNV (rs79419059) in *Nup107* continued to be a significant predictor of shorter progression free survival.

The NPC is a protein assembly made of several subcomplexes. Nup107 is part of a hetero-oligomeric complex that also contains Nup160, Nup133, Nup96, and the mammalian homolog of yeast Sec13p. In previous research, Boehmer et al. showed that Nup107 depletion caused codepletion of a subset of other Nups on their protein level including Nup133. Nup 107 depletion also caused Nup358, Nup214 on the cytoplasmic, and Nup153 on the nucleoplasmic side of the NPC, to fail to assemble into Nup107/Nup133-depleted NPCs. Finally, the filamentous, NPC-associated protein Tpr also failed to assemble into the NPCs of Nup107-depleted cells. The authors therefore concluded that their data indicate that Nup107 functions as a keystone Nup that is required for the assembly of a subset of Nups into the NPC.[[11](#R11)] Thus, an alteration in the function of Nup107 could affect the entry of platinum into the cell and confer chemotherapy resistance to ovarian cancer tumors

SNV rs79419059 is a 10T>C alteration in the 3UTR region of *Nup107* on chromosome 12. The effects of this SNV on the function of Nup107 protein is uncertain as predicted by the function filter we used for this analysis. However, the literature has many examples of 3UTR variants that significantly influenced response to therapy in cancer patients. A famous example is the alteration in the function of the ‘wild-type’ KRAS by a functional single-nucleotide polymorphism in the *KRAS* 3UTR (rs61764370) that disrupts let-7 microRNA binding, thus decreasing *KRAS*-negative regulation and initiating gain of function and leading to poor outcomes of therapy in many cancer types.[[12](#R12)] In another example, Arcaroli et al. showed that a novel 3UTR mutation in *PIK3CA* was associated with increased sensitivity to saracatinib, a Src inhibitor, in colorectal cancer cell lines through possibly reducing the affinity for miR-520a and miR-525a.[[13](#R13)] Finally, within ovarian cancer patients, Wynendaele et al. identified a SNV in the 3UTR of *MDM4* that created a putative target site for hsa-miR-191, a microRNA that is highly expressed in normal and tumor tissues. In that study, biochemical evidence supported specific miR-191–dependent regulation of the *MDM4*-C, but not *MDM4*-A, variant. Consistently, the A-allele was associated with statistically significant increased expression of MDM4 mRNA and protein levels in ovarian carcinomas. Importantly, A/A patients who do not express the estrogen receptor had a 4.2-fold [95% CI 1.2–13.5; p=0.02] increased risk of recurrence and 5.5-fold (95% CI 1.5–20.5; p=0.01) increased risk of tumor-related death.[[14](#R14)] A commonality to all of these examples is that 3UTR variants influenced gene function through a microRNA mediated mechanism. The next logical step in our study is to examine the hypothesis that a similar microRNA mediated mechanism is governing the interaction between rs79419059 and *Nup107*, however, expertise needed to test this hypothesis is not currently available to us.

To try to validate our results, we examined genomic data from the National Cancer Institute portal databases (TCGA-OV MuTect2 (443 samples). Information on time until progression was absent for disease free interval, hence all we could compare was the overall survival time between those who had or did not have the mutation detected by this study. In this analysis, no significant association was found between our SNVs and overall survival. This finding may not be surprising considering that not all studies showing improved DFI translated to better overall all survival in patients with ovarian cancer. Most patients with ovarian cancer will initially respond well to treatment, but about more than three quarters of these patients will developed recurrent disease that cannot be cured.[[15](#R15)] Therefore, DFI continues to be an important end point studies designed to improve DFI can positively affect patients’ quality of life, help understand toxicity profiles of different drugs, and to evaluate the cost of healthcare of such patients.

It is interesting that the frequency of the SNVs we examined was higher in our population than what was reported in publically available genomic data. We therefore examined multiple genomic data bases and found that the prevalence of these SNVs was variable depending on the database by about 10% (rs2302811 was observed in 35.67% of patients included ExAC and in only 25.9% in TOPMED). Such variability between databases still falls short of completely explaining the high frequency of SNVs of interest in our population. The only explanation that we can offer, after we made sure the analysis was done accurately, is that we have recruited patients from a very homogeneous population in rural Illinois with shared heritage that may have a higher frequency of these SNVs.

This study has multiple limitations; first, the small number of subjects included may affect the validity of our findings. Power calculations for pilot studies using agnostic investigation of the genome are not usually possible. In addition, any clinical application of our findings will have to go through multiple validation studies using larger cohorts of patients. Second, our results would have been greatly substantiated by providing biochemical explanation of how rs79419059 is affecting the function of *Nup107*. As we have discussed above, the expertise needed to elucidate the mechanism of action of rs79419059 is not available to us at the time of writing of this paper. Finally, applying Bonferroni correction to our results would make our p values insignificant. A larger sample size is thus needed in future validation studies to prove the value of rs79419059 in predicting response to platinum based chemotherapy in ovarian cancer patients.

## Conclusion

We identified variant in NUP 107 unique to sensitivity to platinum in ovarian cancer. With validation of this variant, it is possible that a new marker predictive of patient response may be identified. At our institution, we continue to collect saliva samples on ovarian cancer patients in an attempt to replicate these results in a larger cohort of patients for validation purposes with the goal of devising a genetic score that could serve as a tool to help select patients that would benefit most from treatment and spare chemotherapy resistant patients the toxicities of a futile treatment.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cc1c/6298606/7878d8cc29f9/nihms875390f1a.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6298606_nihms875390f1a.jpg)

Figure 1a. Kaplan-Meier curve for the association of rs79419059 in Nup107 with response to platinum-based chemotherapy in patients with ovarian cancer (variant predicted worse progression free survival) Figure 1b. Kaplan-Meier curve for the association of rs2302811 in Nup188 with response to platinum-based chemotherapy in patients with ovarian cancer (variant predicted better progression free survival) Figure 1c. Kaplan-Meier curve for the association of rs77246077 in Nup214 with response to platinum-based chemotherapy in patients with ovarian cancer (variant predicted better progression free survival)

## Acknowledgments

**Funding**

This research was funded by a Memorial Medical Center Foundation grant, Springfield, Illinois, USA

## Footnotes

## References

1. Jemal A, et al. Cancer statistics, 2009. CA: a Cancer Journal for Clinicians. 2009;59(4):225. doi: 10.3322/caac.20006.  [DOI](https://doi.org/10.3322/caac.20006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19474385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA:%20a%20Cancer%20Journal%20for%20Clinicians&title=Cancer%20statistics,%202009&author=A%20Jemal&volume=59&issue=4&publication_year=2009&pages=225&pmid=19474385&doi=10.3322/caac.20006&)

2. McGuire W, et al. Cyclophosphamide and cisplatin compared with paclitaxel and cisplatin in patients with stage III and stage IV ovarian cancer. The New England journal of medicine. 1996;334(1):1. doi: 10.1056/NEJM199601043340101.  [DOI](https://doi.org/10.1056/NEJM199601043340101) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7494563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20New%20England%20journal%20of%20medicine&title=Cyclophosphamide%20and%20cisplatin%20compared%20with%20paclitaxel%20and%20cisplatin%20in%20patients%20with%20stage%20III%20and%20stage%20IV%20ovarian%20cancer&author=W%20McGuire&volume=334&issue=1&publication_year=1996&pages=1&pmid=7494563&doi=10.1056/NEJM199601043340101&)

3. Aghajanian C. Clinical update: novel targets in gynecologic malignancies. Elsevier; 2004.  [DOI](https://doi.org/10.1053/j.seminoncol.2004.10.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15799241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Clinical%20update:%20novel%20targets%20in%20gynecologic%20malignancies&author=C%20Aghajanian&publication_year=2004&)

4. Trimbos J, Timmers P. Chemotherapy for early ovarian cancer. Current Opinion in Obstetrics and Gynecology. 2004;16(1):43. doi: 10.1097/00001703-200402000-00009.  [DOI](https://doi.org/10.1097/00001703-200402000-00009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15128007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Obstetrics%20and%20Gynecology&title=Chemotherapy%20for%20early%20ovarian%20cancer&author=J%20Trimbos&author=P%20Timmers&volume=16&issue=1&publication_year=2004&pages=43&pmid=15128007&doi=10.1097/00001703-200402000-00009&)

5. Boer JM, et al. The nucleoporin CAN/Nup214 binds to both the cytoplasmic and the nucleoplasmic sides of the nuclear pore complex in overexpressing cells. Exp Cell Res. 1997;232(1):182–5. doi: 10.1006/excr.1997.3502.  [DOI](https://doi.org/10.1006/excr.1997.3502) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9141635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp%20Cell%20Res&title=The%20nucleoporin%20CAN/Nup214%20binds%20to%20both%20the%20cytoplasmic%20and%20the%20nucleoplasmic%20sides%20of%20the%20nuclear%20pore%20complex%20in%20overexpressing%20cells&author=JM%20Boer&volume=232&issue=1&publication_year=1997&pages=182-5&pmid=9141635&doi=10.1006/excr.1997.3502&)

6. Fahrenkrog B, et al. Domain-specific antibodies reveal multiple-site topology of Nup153 within the nuclear pore complex. J Struct Biol. 2002;140(1–3):254–67. doi: 10.1016/s1047-8477(02)00524-5.  [DOI](https://doi.org/10.1016/s1047-8477(02)00524-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12490173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Struct%20Biol&title=Domain-specific%20antibodies%20reveal%20multiple-site%20topology%20of%20Nup153%20within%20the%20nuclear%20pore%20complex&author=B%20Fahrenkrog&volume=140&issue=1%E2%80%933&publication_year=2002&pages=254-67&pmid=12490173&doi=10.1016/s1047-8477(02)00524-5&)

7. Hutten S, et al. The nuclear pore component Nup358 promotes transportin-dependent nuclear import. J Cell Sci. 2009;122(Pt 8):1100–10. doi: 10.1242/jcs.040154.  [DOI](https://doi.org/10.1242/jcs.040154) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19299463/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&title=The%20nuclear%20pore%20component%20Nup358%20promotes%20transportin-dependent%20nuclear%20import&author=S%20Hutten&volume=122&issue=Pt%208&publication_year=2009&pages=1100-10&pmid=19299463&doi=10.1242/jcs.040154&)

8. Ben Abdelali R, et al. SET-NUP214 is a recurrent gammadelta lineage-specific fusion transcript associated with corticosteroid/chemotherapy resistance in adult T-ALL. Blood. 2014;123(12):1860–3. doi: 10.1182/blood-2013-08-521518.  [DOI](https://doi.org/10.1182/blood-2013-08-521518) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24449214/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=SET-NUP214%20is%20a%20recurrent%20gammadelta%20lineage-specific%20fusion%20transcript%20associated%20with%20corticosteroid/chemotherapy%20resistance%20in%20adult%20T-ALL&author=R%20Ben%20Abdelali&volume=123&issue=12&publication_year=2014&pages=1860-3&pmid=24449214&doi=10.1182/blood-2013-08-521518&)

9. Turner JG, Dawson J, Sullivan DM. Nuclear export of proteins and drug resistance in cancer. Biochem Pharmacol. 2012;83(8):1021–32. doi: 10.1016/j.bcp.2011.12.016.  [DOI](https://doi.org/10.1016/j.bcp.2011.12.016) | [PMC free article](/articles/PMC4521586/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22209898/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Nuclear%20export%20of%20proteins%20and%20drug%20resistance%20in%20cancer&author=JG%20Turner&author=J%20Dawson&author=DM%20Sullivan&volume=83&issue=8&publication_year=2012&pages=1021-32&pmid=22209898&doi=10.1016/j.bcp.2011.12.016&)

10. Kinoshita Y, et al. Alterations in nuclear pore architecture allow cancer cell entry into or exit from drug-resistant dormancy. Am J Pathol. 2012;180(1):375–89. doi: 10.1016/j.ajpath.2011.09.024.  [DOI](https://doi.org/10.1016/j.ajpath.2011.09.024) | [PMC free article](/articles/PMC3338353/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22074739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Pathol&title=Alterations%20in%20nuclear%20pore%20architecture%20allow%20cancer%20cell%20entry%20into%20or%20exit%20from%20drug-resistant%20dormancy&author=Y%20Kinoshita&volume=180&issue=1&publication_year=2012&pages=375-89&pmid=22074739&doi=10.1016/j.ajpath.2011.09.024&)

11. Boehmer T, et al. Depletion of a single nucleoporin, Nup107, prevents the assembly of a subset of nucleoporins into the nuclear pore complex. Proc Natl Acad Sci U S A. 2003;100(3):981–5. doi: 10.1073/pnas.252749899.  [DOI](https://doi.org/10.1073/pnas.252749899) | [PMC free article](/articles/PMC298712/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12552102/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Depletion%20of%20a%20single%20nucleoporin,%20Nup107,%20prevents%20the%20assembly%20of%20a%20subset%20of%20nucleoporins%20into%20the%20nuclear%20pore%20complex&author=T%20Boehmer&volume=100&issue=3&publication_year=2003&pages=981-5&pmid=12552102&doi=10.1073/pnas.252749899&)

12. Chung CH, et al. A 3′-UTR KRAS-variant is associated with cisplatin resistance in patients with recurrent and/or metastatic head and neck squamous cell carcinoma. Ann Oncol. 2014;25(11):2230–6. doi: 10.1093/annonc/mdu367.  [DOI](https://doi.org/10.1093/annonc/mdu367) | [PMC free article](/articles/PMC4207729/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25081901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=A%203%E2%80%B2-UTR%20KRAS-variant%20is%20associated%20with%20cisplatin%20resistance%20in%20patients%20with%20recurrent%20and/or%20metastatic%20head%20and%20neck%20squamous%20cell%20carcinoma&author=CH%20Chung&volume=25&issue=11&publication_year=2014&pages=2230-6&pmid=25081901&doi=10.1093/annonc/mdu367&)

13. Arcaroli JJ, et al. Common PIK3CA mutants and a novel 3′ UTR mutation are associated with increased sensitivity to saracatinib. Clin Cancer Res. 2012;18(9):2704–14. doi: 10.1158/1078-0432.CCR-11-3167.  [DOI](https://doi.org/10.1158/1078-0432.CCR-11-3167) | [PMC free article](/articles/PMC3836589/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22553375/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Common%20PIK3CA%20mutants%20and%20a%20novel%203%E2%80%B2%20UTR%20mutation%20are%20associated%20with%20increased%20sensitivity%20to%20saracatinib&author=JJ%20Arcaroli&volume=18&issue=9&publication_year=2012&pages=2704-14&pmid=22553375&doi=10.1158/1078-0432.CCR-11-3167&)

14. Wynendaele J, et al. An illegitimate microRNA target site within the 3′ UTR of MDM4 affects ovarian cancer progression and chemosensitivity. Cancer Res. 2010;70(23):9641–9. doi: 10.1158/0008-5472.CAN-10-0527.  [DOI](https://doi.org/10.1158/0008-5472.CAN-10-0527) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21084273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=An%20illegitimate%20microRNA%20target%20site%20within%20the%203%E2%80%B2%20UTR%20of%20MDM4%20affects%20ovarian%20cancer%20progression%20and%20chemosensitivity&author=J%20Wynendaele&volume=70&issue=23&publication_year=2010&pages=9641-9&pmid=21084273&doi=10.1158/0008-5472.CAN-10-0527&)

15. Hess LM, et al. Continued chemotherapy after complete response to primary therapy among women with advanced ovarian cancer: a meta-analysis. Cancer. 2010;116(22):5251–60. doi: 10.1002/cncr.25487.  [DOI](https://doi.org/10.1002/cncr.25487) | [PMC free article](/articles/PMC3740230/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20665885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Continued%20chemotherapy%20after%20complete%20response%20to%20primary%20therapy%20among%20women%20with%20advanced%20ovarian%20cancer:%20a%20meta-analysis&author=LM%20Hess&volume=116&issue=22&publication_year=2010&pages=5251-60&pmid=20665885&doi=10.1002/cncr.25487&)
