# An expanded pharmacogenomics warfarin dosing table with utility in generalised dosing guidance

## Metadata
**Authors:** Payman Shahabi, Laura B Scheinfeldt, Daniel E Lynch, Tara J Schmidlen, Sylvie Perreault, Margaret A Keller, Rachel Kasper, Lisa Wawak, Joseph P Jarvis, Norman P Gerry, Erynn S Gordon, Michael F Christman, Marie-Pierre Dubé, Neda Gharani
**Journal:** Thrombosis and Haemostasis
**Date:** 2018 Mar 9
**DOI:** [10.1160/TH15-12-0955](https://doi.org/10.1160/TH15-12-0955)
**PMID:** 27121899
**PMCID:** PMC6375065
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6375065/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6375065/pdf/10-1160-TH15-12-0955.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6375065/pdf/10-1160-TH15-12-0955.pdf)

## Abstract

Pharmacogenomics (PGx) guided warfarin dosing, using a comprehensive dosing algorithm, is expected to improve dose optimisation and lower the risk of adverse drug reactions. As a complementary tool, a simple genotype-dosing table, such as in the US Food and Drug Administration (FDA) Coumadin drug label, may be utilised for general risk assessment of likely over- or under-anticoagulation on a standard dose of warfarin. This tool may be used as part of the clinical decision support for the interpretation of genetic data, serving as a first step in the anticoagulation therapy decision making process. Here we used a publicly available warfarin dosing calculator ( www.warfarindosing.org ) to create an expanded gene-based warfarin dosing table, the CPMC-WD table that includes nine genetic variants in CYP2C9 , VKORC1 , and CYP4F2 . Using two datasets, a European American cohort (EUA, n=73) and the Quebec Warfarin Cohort (QWC, n=769), we show that the CPMC-WD table more accurately predicts therapeutic dose than the FDA table (51 % vs 33 %, respectively, in the EUA, McNemar’s two-sided p=0.02; 52 % vs 37 % in the QWC, p<1×10 −6 ). It also outperforms both the standard of care 5 mg/day dosing (51 % vs 34 % in the EUA, p=0.04; 52 % vs 31 % in the QWC, p<1×10 −6 ) as well as a clinical-only algorithm (51 % vs 38 % in the EUA, trend p=0.11; 52 % vs 45 % in the QWC, p=0.003). This table offers a valuable update to the PGx dosing guideline in the drug label.

Supplementary Material to this article is available at www.thrombosis-online.com .

Keywords: Pharmacogenomics, warfarin, CYP2C9, VKORC1, CYP4F2

## Introduction

Warfarin is a widely prescribed and highly effective oral anticoagulant used for the treatment and prevention of thrombotic events. Despite its common use, warfarin-related adverse drug reactions (ADRs) are among the most common reasons for emergency room visits and hospitalisations in the USA ( [1](#R0955-1) – [4](#R0955-4) ). The high rate of ADRs is due in a large part to its narrow therapeutic window and wide inter-individual variability in response, making dosing problematic and requiring extensive patient monitoring during the dose-initiation and dose-titration period of warfarin use. There is a >10 fold variability in inter-patient therapeutic dose (<1.5 mg/day to >15 mg/day) ( [5](#R0955-5) , [6](#R0955-6) ). Dosing is determined empirically, typically by starting with a standard dose (commonly 5mg/ day) and adjusting until a target International Normalised Ratio (INR) is reached ( [7](#R0955-7) – [9](#R0955-9) ). A supra-therapeutic INR can result in dangerous bleeding episodes while a sub-therapeutic INR is associated with an increased risk of thrombosis ( [9](#R0955-9) ).

Several genetic and non-genetic factors explain up to 60 % of variance in warfarin dose in populations of European descent ( [10](#R0955-10) – [12](#R0955-12) ), with genetic factors accounting for two-thirds of this variability. Given the effect of genetic factors, the FDA added pharmacogenomic (PGx) information about variation in two genes, *CYP2C9* (coding for cytochrome P-450 2C9) and *VKORC1* (coding for vitamin K epoxide reductase), to the warfarin drug label in 2007. This information was updated in 2010 with specific recommended warfarin therapeutic dose ranges based on the composite effect of genetic variations at *CYP2C9* and *VKORC1* ( [13](#R0955-13) ).

A number of warfarin dosing algorithms that include genetic variants, demographic and clinical factors have been developed, primarily using retrospective patient series on stable warfarin maintenance dose ( [10](#R0955-10) , [11](#R0955-11) , [14](#R0955-14) – [16](#R0955-16) ). Furthermore, several prospective randomised clinical trials have attempted to address the question of clinical utility of PGx-guided warfarin dosing ( [17](#R0955-17) – [22](#R0955-22) ). These, along with several subsequent meta-analyses ( [23](#R0955-23) – [27](#R0955-27) ) have provided conflicting and therefore inconclusive results. What is clear is that genetic factors have shown the largest, clinically validated, influence on dose variability ( [6](#R0955-6) , [10](#R0955-10) – [12](#R0955-12) , [14](#R0955-14) , [28](#R0955-28) ). The influence of genetic factors along with the fact that warfarin induced bleeding complications are among the leading causes of severe ADRs ( [1](#R0955-1) – [4](#R0955-4) ), makes warfarin an ideal drug for PGx-guided dosing. Support for this comes from the largest and most insightful trial that prospectively followed over 5,000 genotyped patients receiving warfarin ( [29](#R0955-29) ) and showed that the approximately 40 % of individuals who are genetically sensitive or highly sensitive to warfarin, are at a significant increased risk of early bleeding with standard dosing practices. The findings of this study support the utility of PGx-testing to guide anticoagulation therapy, whereby patients identified as sensitive to warfarin can either be treated with an alternative drug or a PGx algorithm-predicted initial dose of warfarin with more frequent INR monitoring to reduce the time to therapeutic dose and lower the risk of ADRs ( [29](#R0955-29) ). Furthermore, this study underscores the importance of having genetic data available at the time of warfarin prescribing. Ideally genetic data and the interpretation of the results (including any necessary clinical decision support (CDS) alerts), would be available preemptively, that is, in the medical records as a pre-prescription patient characteristic ( [30](#R0955-30) ) ready to guide anticoagulation therapy when indicated.

Recently, we systematically reviewed and critically appraised published and public PGx data from a variety of sources for seven commonly prescribed drugs ( [28](#R0955-28) ). As part of that review, our analysis of existing data found that nine variants across three genes (CYP2C9, VKORC1 and CYP4F2) had a significant impact on warfarin dose response ( [28](#R0955-28) ). Given that the FDA-approved warfarin prescribing information includes only three variants in two genes we set out to expand on this by developing a genotype-based warfarin dosing table (the CPMC-WD table) that includes all nine variants.

## Materials and methods

### Study design

The warfarin clinical and PGx algorithm of Gage et al. [as implemented in the warfarin dosing calculator at [www.warfarindosing](http://www.warfarindosing) . org version 2.40 ( [11](#R0955-11) , [31](#R0955-31) ) and referred to as the “WD algorithm” in this study] was used to develop the CPMC warfarin dosing table (the CPMC-WD table) as described in detail in the Supplementary Material (Suppl. Table 1, available at [www.thrombosis-online.com](http://www.thrombosis-online.com) ). Briefly, this table incorporates the effect of the following variants: CYP2C9*1, *2, *3, *5, *6, *8, *11 and *14; VKORC1–1639G>A and CYP4F2 V433M. Also, given the impact of age and gender on variability in warfarin dose ( [5](#R0955-5) ), warfarin therapeutic daily dose was estimated using the warfarin dosing calculator for the composite-genotype of variants in *CYP2C9* , *VKORC1* and *CYP4F2* based on six hypothetical patients (for rationale see Suppl. Material, available at [www.thrombosis-online.com](http://www.thrombosis-online.com) ): a female (weight 63.5 kg, height 1.63 m) and a male (weight 77 kg, height 1.78 m) both non-hispanic, white, non-smokers, no medications, starting INR 1, target INR 2.5, a primary indication of atrial fibrillation; each at ages 50 years, 60 years and 75 years (results are recorded in Suppl. Table 1, available at [www.thrombosis-online.com](http://www.thrombosis-online.com) ). For each possible genotype combination the predicted therapeutic dose was averaged across the six dose predictions to give a composite-genotype mean dose. Given that a dose deviation of >1 mg/day from the therapeutic dose is considered to be clinically significant ( [14](#R0955-14) , [32](#R0955-32) ), genotype mean doses were also converted to a warfarin dosing category of standard dose (StD) 4.1–5.9 mg/day (29–41 mg/week); high dose (HD) ≥6 mg/day (≥42 mg/week); low dose (LD) 2.1–4.0 mg/day (15–8 mg/week); and very low dose (VLD) ≤2 mg/day (≤14 mg/week) ( [Figure 1](#FI0955-1) and Suppl. Table 1, available at [www.thrombosis-online.com](http://www.thrombosis-online.com) ).

### Figure 1: CPMC-WD therapeutic warfarin dosing (mg/week) and categories based on CYP2C9 , VKORC1 and CYP4F2.

![
Figure 1: CPMC-WD therapeutic warfarin dosing (mg/week) and categories based on
CYP2C9
,
VKORC1
and
CYP4F2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ac2/6375065/4bf95b1f691b/im_10-1160-th15-12-0955-i1.jpg)

. Predicted mean weekly warfarin dose for each CYP2C9-VKORC1-CYP4F2 genotype combination estimated using the algorithm implemented in the web-based calculator www.warfarindosing.org (version 2.40) (see Methods and Suppl. Table 1, available online at www.thrombosis-online.com ). Weekly warfarin dose categories are also provided: STD – “standard dose” 4.1–5.9 mg/day (29–41 mg/week) which includes doses within 1 mg/day of the standard 5 mg/day warfarin dose; HD – “high dose” ≥6 mg/day (≥42 mg/week); LD – “low dose” 2.1–4.0 mg/day (15–28 mg/week); and VLD – “very low dose” ≤2 mg/day (≤14 mg/ week). Additional rare CYP2C9 genotypes include: a (*1/*8, *1/*11); b (*1/*5, *1/*6, *1/*14); c (*2/*8, *2/*11, *8/*8, *8/*11, *11/*11,); d (*2/*5, *2/*6, *2/*14,*8/*3, *8/*5, *8/*6, *8/*14, *11/*3, *11/*5, *11/*6, *11/*14); e (*3/*5, *3/*6, *3/*14, *5/*5, *5/*6, *5/*14, *6/*6, *6/*14, *14/*14). Dose categories that differ from the FDA table (see Suppl. Tables 1 and 2, available online at www.thrombosis-online.com ): f FDA table dose range 3–4 mg/day (21–28 mg/week); g FDA table dose range 0.5–2 mg/day (3.5–14 mg/week).

To compare the performance of the CPMC-WD table with the current FDA-approved warfarin prescribing information (Suppl. Table 2, available at [www.thrombosis-online.com](http://www.thrombosis-online.com) ) (referred to as the “FDA table” in this study), the standard of care 5 mg/day dosing (referred to as the “fixed dose” method), and the clinical variable only algorithm ( [11](#R0955-11) , [21](#R0955-21) ) (referred to as the “clinical-only” algorithm), each patient was assigned a CPMC-WD table-predicted dose, a FDA table-predicted dose (Suppl. Table 2, available at [www.thrombosis-online.com](http://www.thrombosis-online.com) ), a 5 mg/day fixed dose and a clinical-only algorithm predicted dose. The mean absolute error (MAE), i. e. the mean of the absolute values for the difference between the actual therapeutic and the predicted doses, was estimated for each method. The percentage of patients with a predicted dose within 1 mg/day of the stable therapeutic dose and the proportion of patients with predicted doses ≥1 mg/day above or below the stable therapeutic dose was estimated for each dosing method. For the FDA table-predicted doses, a value equal to the midpoint of the daily warfarin dose range (1.25 mg/day for the low dose range; 3.5 mg/day for intermediate dose range; and 6 mg/day for the high dose range), was used to estimate the MAE and the proportions predicted to within 1 mg/day of therapeutic dose or ≥1 mg/day over or under the therapeutic dose. The clinical algorithm of Gage et al. 2008 ( [11](#R0955-11) , [21](#R0955-21) ) was used for the clinical-only dose predictions (Suppl. Material, available at [www.thrombosis-online.com](http://www.thrombosis-online.com) ).

### Data collection and study cohorts

The performance of the CPMC-WD table compared to the FDA table, the fixed dose method, and the clinical-only algorithm was evaluated first in the European American cohort (EUA cohort) and subsequently replicated in an independent population, the Quebec Warfarin Cohort (QWC).

#### European American Cohort

The European American cohort (EUA cohort) is made up of CPMC research study participants enrolled through the institutional review board-approved study. A description of the CPMC study and early results from the Coriell Community Cohort are published elsewhere ( [33](#R0955-33) – [38](#R0955-38) ). All participants were over 18 years of age and were unselected for disease state and medication usage. Participants provided informed consent and saliva samples for genotyping and completed CPMC web-based medical, medication, family history, and lifestyle questionnaires (MFLQ). For the present study genotypes for *CYP2C9* (*2, *3, *5, *6, *11 and *14), *VKORC1* –1639G>A and *CYP4F2* (V433M) were obtained using the Affymetrix DMET ^™^ Plus Array.

The warfarin specific subset of the CPMC cohort consists of 73 primarily European Americans with self-reported warfarin dose data. Criteria for inclusion of study participants in the present study are described in the Suppl. Material (available at [www.thrombosis-online.com](http://www.thrombosis-online.com) ). Whether a stable warfarin dose had been reached was self-reported and based on response to two questions in the targeted survey. Participants were queried whether they had reached a stable dose of warfarin, those responding as not having reached a stable dose and those reporting that they do not remember were excluded from further analysis. In addition, participants were asked how long it had been since their warfarin dose last changed and all those reporting a dose change within the previous two weeks was also excluded from the study.

#### Quebec Warfarin Cohort

The Quebec Warfarin Cohort (QWC) is an observational, prospective inception cohort of warfarin users enrolled consecutively between October 2009 and July 2013 at 18 anticoagulation clinics in the Quebec province of Canada, among which, the Montreal Heart Institute was the leading and coordinating centre. Patients older than 18 years of age were eligible if warfarin therapy was expected for greater than 12 months and for an indication other than deep venous thrombosis, pulmonary embolism or isolated left ventricular thrombosis. Also, patients with the following conditions were excluded: patients with at least one major bleeding episode, including gastro-intestinal bleeding and haemorrhagic stroke, within the past three months; and patients with cirrhosis, chronic hepatitis, icterus, end-stage renal failure and mental illness. Following a face-to-face recruitment interview in which the patients’ baseline and demographic characteristics were collected, patients were followed-up for a 12 month period with five structured telephone questionnaires. We assumed that all patients have reached the stable dose by three months after warfarin initiation. This, however, was confirmed by INR values collected from the anticoagulation clinics or the pharmacists at the three-month time point. Data about warfarin doses were obtained from the patients and validated by those provided by the care providers.

Of 1072 patients who were originally recruited in the cohort, 21 patients withdrew from the study, 11 patients died, and 16 patients were excluded from the analysis due to missing genotype data. Moreover, 51 patients stopped taking warfarin within the first three months of treatment ( [Figure 2](#FI0955-2) ). Accordingly, 973 patients, with full genotype data (referred to as the “full cohort”), were available for the genotype-grouping distribution analysis. Within the full cohort, amiodarone usage data was missing for 204 patients, leaving 769 patients with complete clinical data (referred to as the “clinical subset”), that were included in the analyses comparing the performance of the four dosing methods (the CPMC-WD table, the FDA table, the fixed dose method and the clinical-only algorithm) ( [Figure 2](#FI0955-2) ).

![Figure 2: Flow chart showing patient selection in the Quebec Warfarin Cohort](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ac2/6375065/d79324fd2dd3/im_10-1160-th15-12-0955-i2.jpg)

.

QWC patients provided blood samples for genotyping and were genotyped for *CYP2C9* *2, *3, *5, *6, *8, *11, *14 alleles and also for *VKORC1* –1639 G>A allele, using iPLEX ^®^ ADME PGx Panel (Sequenom Inc., San Diego, CA, USA). Also, data on *CYP4F2* V433M (C>T), were retrieved from the cohort GWAS-dataset obtained by Illumina Infinium HumanOmni2.5 Exome-8v1_A BeadChip (Illumina, San Diego, CA, USA).

Both studies were performed under the terms of the Declaration of Helsinki. The study protocols were approved by their respective local review boards or ethics committees and all patients gave written informed consent.

### Statistical analysis

Descriptive statistics such as frequency distributions, and where applicable, means and standard deviations (given evidence for normal data distribution) were calculated for demographic and clinical characteristics, warfarin dose and warfarin dose category (StD; HD; LD; VLD) separately for each cohort (EUA and QWC).

Comparisons between the dosing methods were made using McNemar’s exact 2×2 test to determine whether the marginal frequencies were equal. The statistical tests were two-sided, and the type I error was set at 0.05 with no correction for multiple comparisons.

## Results

The patient clinical and demographic characteristics are provided in [Table 1](#TB0955-1) and [Table 2](#TB0955-2) for the EUA cohort and the QWC, respectively.

### Table 1.

| Variable |  | EUA cohort (n=73) |
| --- | --- | --- |
| Reported daily therapeutic warfarin dose (mg/day) | Mean(SD) | 5.2 (2.0) |
| range | 1.0–12.1 |  |
| STD (4.1–5.9) | 25 (34) |  |
| Daily Dose Category (mg/day) | HD (≥6.0) | 25 (34) |
| n (%) | LD (2.1–4.0) | 20 (27.5) |
| VLD (≤2.0) | 3 (4) |  |
| Age, years | Mean(SD) | 66.2 (13.4) |
| range | 24–89 |  |
| Height, cm | Mean (SD) | 172.9 (10.9) |
| range | 149.9–198.1 |  |
| Weight, kg | Mean (SD) | 85.8 (19.5) |
| range | 52.2–145.1 |  |
| BMI | Mean (SD) | 28.6 (5.7) |
| range | 20.1–47.4 |  |
| Race, n (%) | White (Caucasian) | 68 (93) |
| African American | 3 (4) |  |
| Other (mixed) | 2 (3) |  |
| Gender, n(%) | Male | 49 (67) |
| Female | 24 (33) |  |
| Smoking Status, n(%) | Smoker | 1 (1.4) |
| Co-morbidity, n(%) | Liver disease | 1 (1.4) |
| Co-medications, n(%) | Use of statin | 47(64) |
| Use of amiodarone | 11(15) |  |
| Use of azole use | 1 (1.4) |  |
| Primary Indication, n(%) | Atrial fibrillation | 44 (60) |
| DVT/PE | 12 (16) |  |
| Heart valve replacement | 8 (11) |  |
| Stroke | 1 (1.4) |  |
| Other | 8 (11) |  |

Table 1 Caption: Demographic and clinical characteristics of the European American cohort .

### Table 2.

| Variable |  | QWC full cohort (n = 973) | QWC clinical subset (n = 769) |
| --- | --- | --- | --- |
| Reported daily therapeutic warfarin dose, (mg/day) | Mean (SD) | 4.7 (2.2) | 4.5 (2.04) |
| Range | 0.3–18 | (0.6–15.7) |  |
| Daily dose category (mg/day), n (%) | STD (4.1–5.9) | 508 (52.2) | 406 (52.8) |
| HD (≥6.0) | 114 (11.7) | 85 (11.0) |  |
| LD (2.1–4.0) | 348 (35.8) | 276 (35.9 |  |
| VLD (≤2.0) | 3 (0.3) | 2 (0.3) |  |
| Age, years | Mean (SD) | 70.0 (11.87) | 72.6 (10.38) |
| Range | 19–96 | 19–96 |  |
| Height, cm | Mean (SD) | 168.7 (10.2) | 166.8 (9.9) |
| Range | 122–196 | 122–196 |  |
| Weight, kg | Mean (SD) | 81.0 (19.3) | 80.0 (18.9) |
| Range | 36.5–215–9 | 36.5–173 |  |
| BMI, (kg/m 2 ) | Mean (SD) | 28.7 (6.1) | 28.6 (6.2) |
| Range | 13.7–58.0 | 13.7–57.5 |  |
| Race, n (%) | White (Caucasian) | 927 (95.3) | 743 (96.6) |
| Hispanic | 4 (0.4) | 4 (0.5) |  |
| Black | 11 (1.1) | 9 (1.2) |  |
| Asian | 8 (8.2) | 6 (0.8) |  |
| Indian-American | 2 (0.2) | 2 (0.3) |  |
| Other (mixed) | 21 (2.1) | 18 (2.3) |  |
| Gender, n (%) | Male | 596 (61) | 450 (58.5) |
| Female | 377 (39) | 319 (41.5 |  |
| Smoking status, n (%) | Smoker | 75 (7.7) | 56 (7.3) |
| Co-morbidity, n (%) | Hypertension | 667/967 (68.9) | 551 (71.6) |
| Diabetes | 266/971 (27.4) | 219 (28.5) |  |
| Hyperlipidemia | 597/963 (62.0) | 482 (62.7) |  |
| Myocardial infarction | 226/947 (23.9) | 181 (23.5) |  |
| Stroke | 67/962 (7.0) | 52 (6.8) |  |
| Primary indication, n (%) | Atrial fibrillation | 725 (74.5) | 609 (79.2) |
| Flutter | 105 (10.8) | 88 (11.4) |  |
| Heart valve replacement | 154 (15.8) | 83 (11.0) |  |
| Mitral stenosis | 12 (1.2) | 9 (1.2) |  |
| Other | 8 (0.8) | 3 (0.4) |  |
| Co-medication, n (%) | Use of amiodarone | NA | 112 (14.6) |

Table 2 Caption: Demographic and clinical characteristics of the Quebec Warfarin Cohort (QWC) .

[Table 3](#TB0955-3) provides the observed CPMC-WD table-based genotype groupings and their distribution for the two cohorts. More-over, the CPMC-WD table predicted daily dose, the actual daily stable dose and the difference between the doses for each genotype group, are presented in [Table 3](#TB0955-3) . For the QWC, the full cohort of 973 patients was included in this analysis. Given the larger sample size of the QWC, a greater diversity of genotype combinations for *CYP2C9* , *VKORC1* – *1639G* >A and *CYP4F2 V433M* were observed (48 genotype groupings) than in the EUA cohort (21 genotype groupings). The two most common genotype combinations in both cohorts were *1/*1-GG-CC and *1/*1-AG-CC and both were associated with a standard dose category (i. e. are within 1 mg/day of the standard 5 mg/day dose). Comparison of the predicted and actual therapeutic dose for individual genotype groupings showed that of the 48 genotype groups observed in the QWC, 27 have four or more individuals and all 27 genotype groups (100 %) have a mean error in predicted dose less than ± 1 mg/day, i. e. are within therapeutic range ( [Table 3](#TB0955-3) ). Of the remaining 21 genotype groupings with three or less individuals, 12 (57 %) are also within therapeutic range (mean error in predicted dose less than ± 1 mg/ day); leaving nine groups with a mean error of ≥ ± 1 mg/day. Similarly, 15 of the 21 EUA cohort genotype groups have a mean error in predicted dose of less than ± 1 mg/day, and the six genotype groupings with a mean error of ≥ ± 1 mg/day all consisted of three or fewer individuals ( [Table 3](#TB0955-3) ). Thus, in the larger QWC cohort the predicted dose for over 81 % (39 of 48) of genotype groups are within 1 mg/day of the actual dose. In contrast, the mean error in predicted dose for the FDA table mid-range value shows that only 30 of the 48 QWC genotype groupings (62.5 %) and 12 of the 21 EUA cohort genotype groups (57 %) are within ± 1 mg/day of the genotype group mean actual dose (Suppl. [Table 3](#TB0955-3) , available online at [www.thrombosis-online.com](http://www.thrombosis-online.com) ). Overall, these data support the greater accuracy of the CPMC-WD table composite-genotype mean predicted doses compared to those of the FDA table.

### Table 3.

| Genotype grouping a | CPMC-WD table predicted mean daily warfarin dose (mg) b | cCPMC-WD table predicted daily warfarin dose category | EUA Cohort (N=73) |  |  | QWC full cohort (N=973) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| No. (freq.) | Mean actual daily warfarin dose (mg)d | Mean error in predicted dose (mg)e | No. (freq.) | Mean actual daily warfarin dose (mg)d | Mean error in predicted dose (mg)e |  |  |  |
| *1/*1-GG-CC | 5.9 | STD | 16 (22 %) | 6.5 | −0.6 | 118 (12.1 %) | 6.5 | − 0.6 |
| *1/*1-AG-CT | 4.6 | STD | 9 (12 %) | 5.4 | −0.8 | 106 (10.9 %) | 4.7 | − 0.1 |
| *1/*1-GG-CT | 6.3 | HD | 4 (5.5 %) | 6.3 | 0 | 88 (9.0 %) | 6.0 | 0.3 |
| *1/*1-GG-TT | 6.8 | HD | 3 (4 %) | 6.1 | 0.7 | 26 (2.7 %) | 7.7 | − 0.9 |
| *1/*2-GG-CC | 4.8 | STD | 3 (4 %) | 5.9 | −1.1 | 50 (5.1 %) | 5.3 | − 0.5 |
| *1/*2-GG-CT | 5.2 | STD | 2 (3 %) | 8.6 | −3.4 | 30 (3.1 %) | 5.5 | − 0.3 |
| *1/*2-GG-TT | 5.6 | STD | 1 (1.4 %) | 5.7 | −0.1 | 5 (0.5 %) | 5.9 | − 0.3 |
| *1/*1-AG-TT | 5.0 | STD | 1 (1.4 %) | 5.0 | 0 | 27 (2.8 %) | 4.6 | 0.4 |
| *1/*11-GG-CT | 4.8 | STD | 1 (1.4 %) | 5.0 | −0.2 | 0 | -- | -- |
| *1/*1-AG-CC | 4.2 | STD | 12 (16 %) | 4.5 | − 0.3 | 137 (14.1 %) | 4.8 | − 0.6 |
| *1/*1-AA-CC | 3.1 | LD | 5 (7 %) | 4.0 | − 0.9 | 49 (5.0 %) | 3.3 | − 0.2 |
| *1/*2-AG-CC | 3.5 | LD | 4 (5.5 %) | 4.0 | − 0.5 | 52 (5.3 %) | 3.9 | − 0.4 |
| *1/*3-GG-CT f | 4.3 | STD | 3 (4 %) | 5.1 | −0.8 | 15 (1.5 %) | 4.7 | − 0.4 |
| *1/*3-AG-CC | 2.9 | LD | 2 (3 %) | 3.0 | −0.1 | 21 (2.2 %) | 3.5 | − 0.6 |
| *1/*1-AA-CT | 3.3 | LD | 1 (1.4 %) | 4.2 | −0.9 | 41 (4.2 %) | 3.2 | 0.1 |
| *1/*2-AA-CC | 2.5 | LD | 1 (1.4 %) | 3.6 | −1.1 | 12 (1.2 %) | 2.5 | 0.0 |
| *1/*2-AA-CT | 2.7 | LD | 1 (1.4 %) | 1.0 | 1.7 | 14 (1.4 %) | 2.9 | − 0.2 |
| *1/*2-AG-CT | 3.8 | LD | 1 (1.4 %) | 7.0 | −3.2 | 41 (4.2 %) | 4.2 | − 0.4 |
| *1/*3-GG-CC | 4.0 | LD | 0 | -- | -- | 16 (1.6 %) | 4.2 | − 0.2 |
| *1/*3-AG-TT | 3.3 | LD | 1 (1.4 %) | 2.5 | 0.8 | 3 (0.3 %) | 5.1 | − 1.8 |
| *2/*2-AG-CT | 3.1 | LD | 1 (1.4 %) | 4.0 | −0.9 | 4 (0.4 %) | 3.6 | − 0.5 |
| *2/*3-GG-CC | 3.2 | LD | 1 (1.4 %) | 1.9 | 1.3 | 5 (0.5 %) | 3.3 | − 0.1 |
| *1/*11-AG-CC | 3.5 | LD | 0 | -- | -- | 2 (0.2 %) | 3.0 | 0.5 |
| *1/*11-AG-CT | 3.8 | LD | 0 | -- | -- | 2 (0.2 %) | 4.0 | −0.2 |
| *1/*11-AG-TT | 4.1 | STD | 0 | -- | -- | 1 (0.1 %) | 5.0 | −0.9 |
| *1/*1-AA-TT | 3.6 | LD | 0 | -- | -- | 10 (1.0 %) | 3.6 | 0.0 |
| *1/*2-AA-TT | 3 | LD | 0 | -- | -- | 3 (0.3 %) | 4.3 | − 1.3 |
| *1/*2-AG-TT | 4.1 | STD | 0 | -- | -- | 11 (1.1 %) | 4.8 | − 0.7 |
| *1/*3-AA-CC g | 2.1 | LD | 0 | -- | -- | 11 (1.1 %) | 1.9 | 0.2 |
| *1/*3-AA-CT g | 2.3 | LD | 0 | -- | -- | 9 (0.9 %) | 2.6 | − 0.3 |
| *1/*3-AA-TT g | 2.4 | LD | 0 | -- | -- | 1 (0.1 %) | 2.5 | −0.1 |
| *1/*3-AG-CT | 3.1 | LD | 0 | -- | -- | 26 (2.7 %) | 3.7 | − 0.6 |
| *1/*3-GG-TT f | 4.6 | STD | 0 | -- | -- | 3 (0.3 %) | 4.6 | 0.0 |
| *1/*8-AG-CT | 3.8 | LD | 0 | -- | -- | 1 (0.1 %) | 5.4 | − 1.6 |
| *1/*8-GG-CT | 5.2 | STD | 0 | -- | -- | 1 (0.1 %) | 2.4 | 2.8 |
| *2/*2-AA-CT g | 2.2 | LD | 0 | -- | -- | 3 (0.3 %) | 2.5 | −0.3 |
| *2/*2-AA-TT g | 2.4 | LD | 0 | -- | -- | 1 (0.1 %) | 1.0 | 1.4 |
| *2/*2-AG-CC | 2.9 | LD | 0 | -- | -- | 5 (0.5 %) | 3.1 | − 0.2 |
| *2/*2-GG-CC | 3.9 | LD | 0 | -- | -- | 2 (0.2 %) | 3.3 | 0.6 |
| *2/*2-GG-CT f | 4.3 | STD | 0 | -- | -- | 2 (0.2 %) | 2.6 | 1.7 |
| *2/*2-GG-TT f | 4.6 | STD | 0 | -- | -- | 2 (0.2 %) | 3.8 | 0.8 |
| *2/*3-AG-CC g | 2.4 | LD | 0 | -- | -- | 7 (0.7 %) | 2.1 | 0.3 |
| *2/*3-AG-CT g | 2.5 | LD | 0 | -- | -- | 1 (0.1 %) | 2.3 | 0.2 |
| *2/*3-AG-TT g | 2.7 | LD | 0 | -- | -- | 1 (0.1 %) | 2.8 | −0.1 |
| *2/*3-GG-CT | 3.5 | LD | 0 | -- | -- | 3 (0.3 %) | 2.6 | 0.9 |
| *3/*11-AG-CC | 2.4 | LD | 0 | -- | -- | 1 (0.1 %) | 1.9 | 0.5 |
| *3/*3-AA-CC | 1.4 | VLD | 0 | -- | -- | 1 (0.1 %) | 0.3 | 1.1 |
| *3/*3-AG-CC | 1.9 | VLD | 0 | -- | -- | 2 (0.2 %) | 0.7 | 1.2 |
| *3/*3-GG-CC g | 2.7 | LD | 0 | -- | -- | 1 (0.1 %) | 1.1 | 1.6 |

Table 3 Caption: Patient genotype distribution data: comparison of CPMC-WD table predicted mean dose and mean actual warfarin dose .

Comparison of the overall performances between the dosing methods in each cohort, showed the MAE was 1.3 mg/day for the CPMC-WD table, 1.4 mg/day for both the FDA table and the fixed dose regimen and 1.5 mg/day for the clinical-only in the EUA cohort. In the QWC clinical subset (N=769), the corresponding MAEs were 1.2 mg/day (CPMC-WD table) 1.5 mg/day (FDA table), 1.7 mg/day (fixed dose), and 1.4 mg/day (clinical-only) ( [Figure 3](#FI0955-3) ).

### Figure 3: Comparison of proportions within range of therapeutic dose in the European American cohort and at three month intervals in the Quebec Warfarin Cohort.

![Figure 3: Comparison of proportions within range of therapeutic dose in the European American cohort and at three month intervals in the Quebec Warfarin Cohort](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ac2/6375065/75c68be2249b/im_10-1160-th15-12-0955-i3.jpg)

. EUA – European American; MAE – mean absolute error; QWC – Quebec Warfarin Cohort. The proportion of patients with predicted doses within 1mg/day of the stable therapeutic dose and the proportion of patients with predicted doses ≥1 mg/day above or below the stable therapeutic dose was compared between each method using the McNemar’s exact 2×2 test (two-sided p-value reported) to determine whether the marginal frequencies were equal. A ≥1 mg/day difference was considered a clinically significant difference ( 14 , 32 ). For statistically significant differences in the pairwise comparisons, p-value font color indicates which method performed best (CPMC-WD table – green; FDA table – pink; fixed dose – brown and clinical-only – turquoise).

The percentage of EUA patients whose predicted dose was within the therapeutic range (WTR) was 51 % for the CPMC-WD table, 33 % for the FDA table, 34 % for the fixed dose and 38 % for the clinical-only algorithm. Thus the CPMC-WD table significantly outperformed the FDA table (p=0.02) and the fixed dose method (p=0.036), and showed a non-significant trend (p=0.11) towards improved performance compared with the clinical-only ( [Figure 3](#FI0955-3) ). Furthermore, the proportion ≥1 mg/day above the therapeutic dose (i. e. those potentially at risk of overdosing) was 12 % (CPMC-WD table), 37 % (FDA table), 32 % (fixed dose) and 19 % (clinical-only), with the CPMC-WD table significantly outperforming the FDA table (p=8×10 ^–6^ ) and the fixed dose method (p=1×10 ^–4^ ) and again showing a trend towards improved performance compared to the clinical-only algorithm. The proportion of patients predicted to be ≥1 mg/day below the therapeutic dose (i. e. those potentially at risk of underdosing) was not significantly different between the CPMC-WD table and any of the other dosing methods in the EUA cohort ( [Figure 3](#FI0955-3) ).

The improved performance of the CPMC-WD table compared to the other dosing methods was more clearly demonstrated in the larger QWC ( [Figure 3](#FI0955-3) ). The CPMC-WD table outperformed the FDA table (p<1×10 ^–6^ ), the fixed dose method (p<1×10 ^–6^ ) and the clinical-only algorithm (p=0.003), predicting 52 %, 37 %, 31 % and 45 % WTR, respectively. Moreover, the QWC showed a similar result as the EUA cohort in the proportions predicted to ≥1 mg/ day above the therapeutic dose, with the CPMC-WD table again significantly outperforming the FDA table (p<1×10 ^–6^ ) and the fixed dose method (p<1×10 ^–6^ ), but not the clinical-only algorithm (p=0.11). On the other hand, both the FDA table and the fixed dose method predicted marginally fewer cases under the therapeutic dose compared to the CPMC-WD table (p<0.05), and all three dosing methods outperformed the clinical-only algorithm in this dosing category (p<1×10 ^–6^ ) ( [Figure 3](#FI0955-3) ).

Interestingly, the FDA table only outperformed the fixed dose method in the QWC, with respect to the proportion of patients predicted to be WTR (37 % vs 31 %, respectively, p=0.007) ( [Figure 3](#FI0955-3) ). In contrast, the clinical-only algorithm predicted significantly higher proportions WTR than both the FDA table (45 % vs 37 %, respectively, p=0.002) and the fixed dose method (45 % vs 31 %, respectively, p<1×10–6).

## Discussion

In this study, we used a publicly-available warfarin dosing algorithm ( [11](#R0955-11) , [31](#R0955-31) ) to develop a genotype-based dose prediction table and showed, in two independent datasets, that it more accurately predicts therapeutic dose than the PGx table in the FDA-approved drug label ( [13](#R0955-13) ), the standard of care 5 mg/day dosing and a clinical variable-only dosing algorithm ( [11](#R0955-11) , [21](#R0955-21) ). This table is similar to the FDA table in that it provides a composite-genotype predicted dose. The key difference between the CPMC-WD table and the FDA table is that the FDA table uses a combination of only three variants in two genes that are common in populations of European descent, whereas the CPMC-WD table includes six additional genetic variants in that are observed in both European and non-European populations. Thus, the comparison with the FDA table demonstrates the increased accuracy of the CPMC-WD table, resulting from an expansion of the genetic markers of warfarin sensitivity. The purpose of comparing the CPMC-WD table with the fixed-dose method was to demonstrate the added value of including genetic information in dose prediction. We also made a comparison with a clinical-only algorithm. However, it is important to highlight the differences between our analysis and that of some recent genotype guided randomised-controlled trials ( [21](#R0955-21) , [22](#R0955-22) ) that compared a PGx algorithm (which included both genetic and clinical variables) with a clinical characteristics-only algorithm. In the latter, the PGx algorithm included the same clinical characteristics as the clinical-only algorithm. The only difference between the two dosing methods was the presence or absence of the genetic information. Therefore, any difference in outcome could conceivably be assigned to the genetic difference between the two models.

In contrast, the CPMC-WD table does not take into account the patient-specific clinical characteristics, and therefore this comparison does not address the added value of the genetic component. Here the comparison is more akin to genetics versus clinical, and since both are important in guiding dosing, especially on an individual-patient level, the comparison is less meaningful. Furthermore, since the CPMC-WD table is intended as a tool to identify individuals who are genetically sensitive to warfarin and not for direct patient dosing, again the comparison becomes less relevant in this situation. However, what the data show in addition to the CPMC-WD table outperforming the clinical-only algorithm, is that both methods are superior to fixed dosing ( [Figure 3](#FI0955-3) ), demonstrating the added value of both genetic and clinical factors in warfarin dose prediction.

Ultimately, the adoption of PGx-guided warfarin dosing will depend on whether the association of genotype with therapeutic dose translates to a clinically-significant improved outcome. Traditionally, the gold standard for establishing clinical utility is a prospective randomised-controlled trial powered to detect the impact of PGx-guided dosing on patient outcomes (e. g. lowered risk of bleeding and hospitalisation) compared with the current standard of care. Although a number of randomised trials have been conducted, the results have been conflicting, with some providing support for PGx-guided warfarin therapy ( [17](#R0955-17) – [20](#R0955-20) ), while others have not ( [21](#R0955-21) , [22](#R0955-22) ). Several recent meta-analyses have also provided inconsistent results ( [23](#R0955-23) – [26](#R0955-26) ), highlighting the significant heterogeneity, high risk of bias, low power and low quality of evidence across the trials ( [26](#R0955-26) ). These conclusions ( [26](#R0955-26) ), together with thoughtful reflection by others ( [39](#R0955-39) – [43](#R0955-43) ), underscore the need for an appropriate study design (i. e. the treatment setting, characteristics of the populations tested including sufficient representation of the genetically at risk patients, inclusion of pan-ethnic risk variants, the analytic approach, choice of control groups, and end-point definitions) as critical to clarifying the public health relevance of PGx-guided warfarin dosing. A key consideration raised ( [42](#R0955-42) ), is whether the genotype-guided dosing algorithms used in the clinical trials accurately identify all individuals who are genetically sensitive to warfarin. This will depend on the composition of the genetic variants included in the dosing algorithm as well as the population diversity and admixture. For example, if a patient has inherited a variant affecting *CYP2C9* enzyme activity that is not included in the dosing algorithm they will be assumed to have normal enzyme function and the algorithm-predicted starting dose will be inaccurate. This will add ‘noise’ to the results for the genotype-guided arm of the clinical trial ( [42](#R0955-42) ), as is likely the case for the African American subset in the Clarification of Optimal Anticoagulation through Genetics (COAG) trial ( [21](#R0955-21) , [43](#R0955-43) ). To date, the vast majority of published trials have included a maximum of three genetic variants (typically *CYP2C9* *2, *3 and/or *VKORC1* –1639 G>A) ( [17](#R0955-17) – [22](#R0955-22) ) common in populations of European descent. The expected impact of the missing *CYP2C9* variants (e. g. *CYP2C9* *5, *6, *8, *11 and *14) on the performance of the genotype-guided dosing algorithms will therefore depend on the ancestral composition of the study cohorts ( [42](#R0955-42) – [44](#R0955-44) ), which may not be accurately captured by self-reported race/ethnicity ( [45](#R0955-45) ). Similarly, inclusion of other validated variants such as *CYP4F2* (V433M) is likely to increase the accuracy of the predicted dose, as suggested by the performance of the CPMC-WD table compared to the FDA table in this study.

We anticipate that with the appropriate design, the clinical utility of PGx-guided warfarin dosing will be demonstrated particularly for those who are genetically sensitive to warfarin and at increased risk of bleeding ( [29](#R0955-29) ).

We agree with others ( [6](#R0955-6) ) that a starting dose of warfarin should be estimated at the time of therapy, using a comprehensive dosing algorithm. Ideally the algorithm should include known pan-ethnic genetic variants. We further agree that genetic variants, along with sex, are inherent and invariable characteristics of a patient ( [30](#R0955-30) ), which together with age ( [5](#R0955-5) ) have a significant impact on predicted dose. Given that genetic factors have the largest influence on dose variability ( [10](#R0955-10) – [12](#R0955-12) , [14](#R0955-14) , [29](#R0955-29) ), we further believe that a simple gene-based dosing table, such as the CPMC-WD table, has utility by allowing general risk assessment of likely over- or under-anti-coagulation when given a standard dose of warfarin. Such a table, presented in the drug label, can be utilised as a first step for anticoagulation therapy clinical decision making ( [Figure 4](#FI0955-4) ). For example, whether to opt for another drug ( [29](#R0955-29) ) (in a patient that is expected to be sensitive to warfarin) or to go onto using a dosing algorithm to guide the appropriate warfarin dosage.

### Figure 4: PGx-based anticoagulation therapy options and paths.

![Figure 4: PGx-based anticoagulation therapy options and paths](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ac2/6375065/809ed64e2723/im_10-1160-th15-12-0955-i4.jpg)

. EMR – electronic medical record; STD – “standard dose” 4.1–5.9 mg/day which includes doses within 1 mg/day of the standard 5 mg/day warfarin dose; HD – “high dose” ≥6 mg/day; LD – “low dose” 2.1–4.0 mg/day; and VLD – “very low dose” ≤2 mg/day ( Table 1 ). a – Since clinical, demographic and medication use may change if and when warfarin use is warranted in the future. b – 5mg/day of warfarin. c – For example, the warfarin dosing calculator at www.warfarindosing.org ( 31 ). d – The laboratory conducting genotyping analysis is unlikely to have access to the patient’s full clinical, demographic and medication data, thus a baseline gene-based interpretation of likely sensitivity to warfarin should be provided along with individual genetic results. colour: blue depicts preemptive path prior to indication for warfarin therapy; red depicts path once anticoagulation therapy is indicated. Schematic illustration of the utility of a gene-only based warfarin dosing table such as the CPMC-WD table. This table provides a generalised interpretation of likely under- or over-anticoagulation on a standard daily dose of warfarin. Such an interpretation can be stored preemptively in a patient’s medical records and together with an appropriate clinical decision support serves as a flag should the need for anticoagulation therapy arise in the future. If anticoagulation therapy is warranted a baseline genetic interpretation might be incorporated in the clinical decision support used in the anticoagulation therapy decision making process as illustrated here.

A gene-based table can also be used for anticipatory interpretation of genetic results for future reference, for example of preemptive genetic data stored in a patient’s electronic medical record (EMR). There is growing consensus that genetic testing as a preemptive clinical tool is key to the successful implementation of PGx ( [30](#R0955-30) ). Accordingly, programs have been established in five US medical centres to develop the processes for integration of genetic data into genome-enabled EMRs that include interpretive CDS tools and alerts to guide patient pharmacotherapy for specific, clinically validated high risk gene/drug pairs, including warfarin ( [30](#R0955-30) ). In a preemptive model, the interpretation of genetic data would be available to clinicians through the EMR in the form of passive CDS; i. e. for guidance at any time prior to any prescribing decision. Given that a patient’s clinical and medication history may change from the time preemptive genetic data are entered into the EMR and the time at which anticoagulation therapy might be indicated, it makes sense to use a gene-based table rather than a dosing algorithm to develop interpretive passive CDS for warfarin ( [Figure 4](#FI0955-4) ). Furthermore, taking the combined (composite) genetic results into account, as presented in the CPMC-WD table, is important for interpretation of risks. For example, the predicted dose for an individual with a CYP2C9*1/*2 genotype could range, depending on their genotype at VKORC1 and CYP4F2, from >5.5 mg/day (for someone with VKORC1-GG and CYP4F2-TT) to about 2.5 mg/day for a VKORC1-AA, CYP4F2-CC carrier ( [Table 1](#TB0955-1) ). Therefore, communicating risk for each gene individually is neither accurate nor advisable in this situation. Finally, a gene-based warfarin dosing table may be used in other anticipatory situations such as the return of genetic results and interpretations by clinical diagnostic laboratories and in direct-to-patient warfarin PGx reports.

A further advantage of the CPMC-WD table over the FDA table is that for each combination of genetic results, it provides both a mean dose as well as a dose category relative to the standard fixed 5 mg/day starting dose. Genotype results are categorized into those that are within 1 mg/day of the standard dose (STD), are greater than 1 mg/day above the standard dose (HD) or are associated with a low (LD) or very low dose (VLD) relative to the standard dose ( [Figure 1](#FI0955-1) ). The FDA table provides three dose range categories of 0.5–2 mg/day, 3–4 mg/day and 5–7 mg/day. In our opinion, the CPMC-WD dose categories are easier to interpret, differentiating those that are expected to respond safely to a standard dose from those with a genetic susceptibility to either over- or under-anti-coagulation.

There are several limitations to our study. First, most patients in both cohorts had a primary indication of atrial fibrillation and replication in other patient series is needed to know if the results can be generalised to other indications. Second, the two independent cohorts are both primarily (>93 %) of European descent. Therefore, it remains to be determined how well the CPMC-WD table captures the genetic contributors to dose variability in non-Europeans compared to the FDA table. The key point here is whether the variants in the three genes include those that are common in other ethnic/racial groups. For example, the FDA table only includes *CYP2C9*2* and *3 variants which have a combined allele frequency of 3 % in African Americans ( [28](#R0955-28) ). The combined allele frequency of *CYP2C9*5* , *6, *8 and *11 variants in African Americans is 10 % ( [28](#R0955-28) ), and inclusion of these additional variants is expected to improve dose predictions in this population, as clearly demonstrated by Drozda et al. ( [43](#R0955-43) ). Interestingly, in their approach, Drozda et al. made a 20 % dose adjustments to the Gage algorithm ( [31](#R0955-31) ) predicted doses (WD-AA algorithm) to account for the presence of a *CYP2C9* *8 or *CYP2C9* *11 allele ( [43](#R0955-43) ). This is in line with our approach of pooling *CYP2C9* *8 and *CYP2C9* *11 alleles with the *CYP2C9* *2, which is adjusted for in the WD algorithm by a 19 % dose reduction ( [11](#R0955-11) , [31](#R0955-31) ).

Another limitation of the current study is that the FDA table only provides a composite-genotype dose range and not a category-mean warfarin dose. In order to make a direct comparison between the four dosing methods we converted the FDA dose ranges to mid-range dose values. This approach has been used previously ( [46](#R0955-46) ) and although not ideal the mid-range value of 1.25 mg/day for the 0.5–2 mg/day range, 3.5 mg/day for the 3–4 mg/ day range and the 6 mg/day for the 5–7 mg/day range are all within 1 mg/day of the extremes of the ranges and therefore fairly conservative representations of the dose categories. A limitation of the EUA cohort is the relatively small sample size, reducing the power to detect statistically significant differences in proportions as observed for the comparison of the CPMC-WD table and the clinical-only algorithm. Although the p<0.05 threshold of significance has not been reached in this cohort, the proportions WTR (51 % vs 38 %, respectively) mirror the findings in the larger QWC (52 % vs 45 %, respectively) where statistical significance was demonstrated. Another limitation in the EUA cohort is that demographic and clinical data, including information on therapeutic dose and whether stable dose had been reached is self-reported and lacks direct INR measures for confirmation, increasing the possibility of data entry or recall errors. This might be expected to add noise to the analyses making it harder to demonstrate the accuracy of the CPMC-WD dosing table. The effect of self-reported data may be less significant than expected, as previously suggested by us ( [47](#R0955-47) ) and as reflected in the performance of the CPMC-WD table relative to the other dosing methods in the EUA cohort. Furthermore, the data from the QWC is less likely to suffer from recall and self-reported errors. Finally, the present study demonstrated that the CPMC-WD table more accurately predicts the reported therapeutic dose, than the other evaluated dosing methods. Although therapeutic dose prediction is a surrogate outcome, it is assumed that the greater the accuracy of the starting dose, the shorter the time to attain stable INR, the longer the time within the therapeutic INR range, and consequently the lower the risk of an adverse event ( [6](#R0955-6) ). Thus, another limitation of the present study is a lack of primary outcomes data such as time in therapeutic INR range and cerebral and non-cerebral embolism and hemorrhage. Such data should be included in future studies.

The findings described here would be strengthened by replication in other patient groups. If such future studies confirm that the CPMC-WD table-based dosing is superior to the FDA table, this table should be considered for inclusion in a future update of the warfarin FDA drug label.

**What is known about this topic?**

**What does this paper add?**

**What is known about this topic?**

**What does this paper add?**

## Acknowledgements

We are grateful to the participants of the Coriell Personalized Medicine Collaborative and to the patients from the Quebec Warfarin Cohort, for their ongoing commitment to the respective research studies that are included in this manuscript. We acknowledge the staff of the Coriell Genotyping and Microarray Center and of the Coriell Information Systems Department for their significant contribution to the CPMC. We thank Felipe García-España for advice and contribution to the analysis of early EUA cohort data and Andrew Brangan for spot checking the warfarindosing.org dose predictions for the CPMC-WD table and the EUA cohort. This work was supported in part by the United States Air Force Medical Support Agency, Innovations Division (SG9Z) (MFC, ESG, NG, DEL, TJS, JPJ, LW, RK, LS, and NPG), the RNR Foundation (NG, MAK), the FRQ-S (SP, MPD), and the Canadian Gene Cure Foundation (PS). The QWC was funded by the Canadian Institutes of Health Research (CIHR) and by the Center of Excellence in Personalized Medicine (Cepmed).

## 

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Budnitz DS, Lovegrove MC, Shehab N et al. Emergency hospitalisations for adverse drug events in older Americans. N Engl J Med. 2011;365:2002–2012. doi: 10.1056/NEJMsa1103053.  [DOI](https://doi.org/10.1056/NEJMsa1103053) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22111719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Emergency%20hospitalisations%20for%20adverse%20drug%20events%20in%20older%20Americans&author=DS%20Budnitz&author=MC%20Lovegrove&author=N%20Shehab&volume=365&publication_year=2011&pages=2002-2012&pmid=22111719&doi=10.1056/NEJMsa1103053&)

2. McWilliam A, Nardinelli C.Health care savings from personalizing medicine using genetic testing: The case for warfarinIn:AEI Brookings Joint Center for Regulatory Studies 20061–17.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AEI%20Brookings%20Joint%20Center%20for%20Regulatory%20Studies&title=Health%20care%20savings%20from%20personalizing%20medicine%20using%20genetic%20testing:%20The%20case%20for%20warfarin&author=A%20McWilliam&author=C%20Nardinelli&publication_year=2006&pages=1-17&)

3. Shehab N, Sperling LS, Kegler SR et al. National estimates of emergency department visits for hemorrhage-related adverse events from clopidogrel plus aspirin and from warfarin. Arch Intern Med. 2010;170:1926–1933. doi: 10.1001/archinternmed.2010.407.  [DOI](https://doi.org/10.1001/archinternmed.2010.407) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21098354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=National%20estimates%20of%20emergency%20department%20visits%20for%20hemorrhage-related%20adverse%20events%20from%20clopidogrel%20plus%20aspirin%20and%20from%20warfarin&author=N%20Shehab&author=LS%20Sperling&author=SR%20Kegler&volume=170&publication_year=2010&pages=1926-1933&pmid=21098354&doi=10.1001/archinternmed.2010.407&)

4. Wysowski DK, Nourjah P, Swartz L. Bleeding complications with warfarin use: a prevalent adverse effect resulting in regulatory action. Arch Intern Med. 2007;167:1414–1419. doi: 10.1001/archinte.167.13.1414.  [DOI](https://doi.org/10.1001/archinte.167.13.1414) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17620536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Intern%20Med&title=Bleeding%20complications%20with%20warfarin%20use:%20a%20prevalent%20adverse%20effect%20resulting%20in%20regulatory%20action&author=DK%20Wysowski&author=P%20Nourjah&author=L%20Swartz&volume=167&publication_year=2007&pages=1414-1419&pmid=17620536&doi=10.1001/archinte.167.13.1414&)

5. Garcia D, Regan S, Crowther M et al. Warfarin maintenance dosing patterns in clinical practice: implications for safer anticoagulation in the elderly population. Chest. 2005;127:2049–2056. doi: 10.1378/chest.127.6.2049.  [DOI](https://doi.org/10.1378/chest.127.6.2049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15947319/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=Warfarin%20maintenance%20dosing%20patterns%20in%20clinical%20practice:%20implications%20for%20safer%20anticoagulation%20in%20the%20elderly%20population&author=D%20Garcia&author=S%20Regan&author=M%20Crowther&volume=127&publication_year=2005&pages=2049-2056&pmid=15947319&doi=10.1378/chest.127.6.2049&)

6. Johnson JA, Gong L, Whirl-Carrillo M et al. Clinical Pharmacogenetics Implementation Consortium Guidelines for CYP2C9 and VKORC1 genotypes and warfarin dosing . Clin Pharmacol Therap. 2011;90:625–629. doi: 10.1038/clpt.2011.185.  [DOI](https://doi.org/10.1038/clpt.2011.185) | [PMC free article](/articles/PMC3187550/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21900891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Therap&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20Guidelines%20for%20CYP2C9%20and%20VKORC1%20genotypes%20and%20warfarin%20dosing&author=JA%20Johnson&author=L%20Gong&author=M%20Whirl-Carrillo&volume=90&publication_year=2011&pages=625-629&pmid=21900891&doi=10.1038/clpt.2011.185&)

7. Hirsh J, Dalen JE, Anderson DR et al. Oral anticoagulants: mechanism of action, clinical effectiveness, and optimal therapeutic range. Chest. 1998;114 05:445S–469S.. doi: 10.1378/chest.114.5_supplement.445s.  [DOI](https://doi.org/10.1378/chest.114.5_supplement.445s) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9822057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chest&title=Oral%20anticoagulants:%20mechanism%20of%20action,%20clinical%20effectiveness,%20and%20optimal%20therapeutic%20range&author=J%20Hirsh&author=JE%20Dalen&author=DR%20Anderson&volume=114&publication_year=1998&pages=445S-469S.&pmid=9822057&doi=10.1378/chest.114.5_supplement.445s&)

8. Group TEAFTS . Optimal oral anticoagulant therapy in patients with nonrheumatic atrial fibrillation and recent cerebral ischemia. N Engl J Med. 1995;333:5–10. doi: 10.1056/NEJM199507063330102.  [DOI](https://doi.org/10.1056/NEJM199507063330102) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7776995/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Optimal%20oral%20anticoagulant%20therapy%20in%20patients%20with%20nonrheumatic%20atrial%20fibrillation%20and%20recent%20cerebral%20ischemia.&volume=333&publication_year=1995&pages=5-10&pmid=7776995&doi=10.1056/NEJM199507063330102&)

9. Fuster V, Ryden LE, Asinger RW et al. ACC/AHA/ESC guidelines for the management of patients with atrial fibrillation: executive summary. A Report of the American College of Cardiology/ American Heart Association Task Force on Practice Guidelines and the European Society of Cardiology Committee for Practice Guidelines and Policy Conferences (Committee to Develop Guidelines for the Management of Patients With Atrial Fibrillation): developed in Collaboration With the North American Society of Pacing and Electrophysiology. J Am Coll Cardiol. 2001;38:1231–1266. doi: 10.1016/s0735-1097(01)01587-x.  [DOI](https://doi.org/10.1016/s0735-1097(01)01587-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11583910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol&title=ACC/AHA/ESC%20guidelines%20for%20the%20management%20of%20patients%20with%20atrial%20fibrillation:%20executive%20summary.%20A%20Report%20of%20the%20American%20College%20of%20Cardiology/%20American%20Heart%20Association%20Task%20Force%20on%20Practice%20Guidelines%20and%20the%20European%20Society%20of%20Cardiology%20Committee%20for%20Practice%20Guidelines%20and%20Policy%20Conferences%20(Committee%20to%20Develop%20Guidelines%20for%20the%20Management%20of%20Patients%20With%20Atrial%20Fibrillation):%20developed%20in%20Collaboration%20With%20the%20North%20American%20Society%20of%20Pacing%20and%20Electrophysiology&author=V%20Fuster&author=LE%20Ryden&author=RW%20Asinger&volume=38&publication_year=2001&pages=1231-1266&pmid=11583910&doi=10.1016/s0735-1097(01)01587-x&)

10. Wadelius M, Chen LY, Lindh JD et al. The largest prospective warfarin-treated cohort supports genetic forecasting. Blood. 2009;113:784–792. doi: 10.1182/blood-2008-04-149070.  [DOI](https://doi.org/10.1182/blood-2008-04-149070) | [PMC free article](/articles/PMC2630264/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18574025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20largest%20prospective%20warfarin-treated%20cohort%20supports%20genetic%20forecasting&author=M%20Wadelius&author=LY%20Chen&author=JD%20Lindh&volume=113&publication_year=2009&pages=784-792&pmid=18574025&doi=10.1182/blood-2008-04-149070&)

11. Gage BF, Eby C, Johnson JA et al. Use of pharmacogenetic and clinical factors to predict the therapeutic dose of warfarin. Clin Pharmacol Therap. 2008;84:326–331. doi: 10.1038/clpt.2008.10.  [DOI](https://doi.org/10.1038/clpt.2008.10) | [PMC free article](/articles/PMC2683977/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18305455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Therap&title=Use%20of%20pharmacogenetic%20and%20clinical%20factors%20to%20predict%20the%20therapeutic%20dose%20of%20warfarin&author=BF%20Gage&author=C%20Eby&author=JA%20Johnson&volume=84&publication_year=2008&pages=326-331&pmid=18305455&doi=10.1038/clpt.2008.10&)

12. Caldwell MD, Awad T, Johnson JA et al. CYP4F2 genetic variant alters required warfarin dose. Blood. 2008;111:4106–4112. doi: 10.1182/blood-2007-11-122010.  [DOI](https://doi.org/10.1182/blood-2007-11-122010) | [PMC free article](/articles/PMC2288721/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18250228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=CYP4F2%20genetic%20variant%20alters%20required%20warfarin%20dose&author=MD%20Caldwell&author=T%20Awad&author=JA%20Johnson&volume=111&publication_year=2008&pages=4106-4112&pmid=18250228&doi=10.1182/blood-2007-11-122010&)

13. Coumadin Drug Label October 2011. Available at:http://www.accessdata.fda.gov/drugsatfda_docs/label/2011/009218s107lbl.pdf  [http://www.accessdata.fda.gov/drugsatfda_docs/label/2011/009218s107lbl.pdf](http://www.accessdata.fda.gov/drugsatfda_docs/label/2011/009218s107lbl.pdf)

14. International Warfarin Pharmacogenetics Consortium . Klein TE, Altman RB et al. Estimation of the warfarin dose with clinical and pharmacogenetic data. N Engl J Med. 2009;360:753–764. doi: 10.1056/NEJMoa0809329.  [DOI](https://doi.org/10.1056/NEJMoa0809329) | [PMC free article](/articles/PMC2722908/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19228618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Estimation%20of%20the%20warfarin%20dose%20with%20clinical%20and%20pharmacogenetic%20data&author=TE%20Klein&author=RB%20Altman&volume=360&publication_year=2009&pages=753-764&pmid=19228618&doi=10.1056/NEJMoa0809329&)

15. Hernandez W, Gamazon ER, Aquino-Michaels K et al. Ethnicity-specific pharmacogenetics: the case of warfarin in African Americans. Pharmacogenom J. 2014;14:223–228. doi: 10.1038/tpj.2013.34.  [DOI](https://doi.org/10.1038/tpj.2013.34) | [PMC free article](/articles/PMC4016191/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24018621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom%20J&title=Ethnicity-specific%20pharmacogenetics:%20the%20case%20of%20warfarin%20in%20African%20Americans&author=W%20Hernandez&author=ER%20Gamazon&author=K%20Aquino-Michaels&volume=14&publication_year=2014&pages=223-228&pmid=24018621&doi=10.1038/tpj.2013.34&)

16. Ramirez AH, Shi Y, Schildcrout JS et al. Predicting warfarin dosage in European-Americans and African-Americans using DNA samples linked to an electronic health record. Pharmacogenom. 2012;13:407–418. doi: 10.2217/pgs.11.164.  [DOI](https://doi.org/10.2217/pgs.11.164) | [PMC free article](/articles/PMC3361510/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22329724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom&title=Predicting%20warfarin%20dosage%20in%20European-Americans%20and%20African-Americans%20using%20DNA%20samples%20linked%20to%20an%20electronic%20health%20record&author=AH%20Ramirez&author=Y%20Shi&author=JS%20Schildcrout&volume=13&publication_year=2012&pages=407-418&pmid=22329724&doi=10.2217/pgs.11.164&)

17. Anderson JL, Horne BD, Stevens SM et al. Randomized trial of genotype-guided versus standard warfarin dosing in patients initiating oral anticoagulation. Circulation. 2007;116:2563–2570. doi: 10.1161/CIRCULATIONAHA.107.737312.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.107.737312) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17989110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Randomized%20trial%20of%20genotype-guided%20versus%20standard%20warfarin%20dosing%20in%20patients%20initiating%20oral%20anticoagulation&author=JL%20Anderson&author=BD%20Horne&author=SM%20Stevens&volume=116&publication_year=2007&pages=2563-2570&pmid=17989110&doi=10.1161/CIRCULATIONAHA.107.737312&)

18. Anderson JL, Horne BD, Stevens SM et al. A randomized and clinical effectiveness trial comparing two pharmacogenetic algorithms and standard care for individualizing warfarin dosing (CoumaGen-II) Circulation. 2012;125:1997–2005. doi: 10.1161/CIRCULATIONAHA.111.070920.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.111.070920) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22431865/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=A%20randomized%20and%20clinical%20effectiveness%20trial%20comparing%20two%20pharmacogenetic%20algorithms%20and%20standard%20care%20for%20individualizing%20warfarin%20dosing%20(CoumaGen-II)&author=JL%20Anderson&author=BD%20Horne&author=SM%20Stevens&volume=125&publication_year=2012&pages=1997-2005&pmid=22431865&doi=10.1161/CIRCULATIONAHA.111.070920&)

19. Caraco Y, Blotnick S, Muszkat M. CYP2C9 genotype-guided warfarin prescribing enhances the efficacy and safety of anticoagulation: a prospective randomized controlled study. Clin Pharmacol Therap. 2008;83:460–470. doi: 10.1038/sj.clpt.6100316.  [DOI](https://doi.org/10.1038/sj.clpt.6100316) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17851566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Therap&title=CYP2C9%20genotype-guided%20warfarin%20prescribing%20enhances%20the%20efficacy%20and%20safety%20of%20anticoagulation:%20a%20prospective%20randomized%20controlled%20study&author=Y%20Caraco&author=S%20Blotnick&author=M%20Muszkat&volume=83&publication_year=2008&pages=460-470&pmid=17851566&doi=10.1038/sj.clpt.6100316&)

20. Pirmohamed M, Burnside G, Eriksson N et al. A randomized trial of genotype-guided dosing of warfarin. N Engl J Med. 2013;369:2294–2303. doi: 10.1056/NEJMoa1311386.  [DOI](https://doi.org/10.1056/NEJMoa1311386) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24251363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=A%20randomized%20trial%20of%20genotype-guided%20dosing%20of%20warfarin&author=M%20Pirmohamed&author=G%20Burnside&author=N%20Eriksson&volume=369&publication_year=2013&pages=2294-2303&pmid=24251363&doi=10.1056/NEJMoa1311386&)

21. Kimmel SE, French B, Kasner SE et al. A pharmacogenetic versus a clinical algorithm for warfarin dosing. N Engl J Med. 2013;369:2283–2293. doi: 10.1056/NEJMoa1310669.  [DOI](https://doi.org/10.1056/NEJMoa1310669) | [PMC free article](/articles/PMC3942158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24251361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=A%20pharmacogenetic%20versus%20a%20clinical%20algorithm%20for%20warfarin%20dosing&author=SE%20Kimmel&author=B%20French&author=SE%20Kasner&volume=369&publication_year=2013&pages=2283-2293&pmid=24251361&doi=10.1056/NEJMoa1310669&)

22. Jonas DE, Evans JP, McLeod HL et al. Impact of genotype-guided dosing on anticoagulation visits for adults starting warfarin: a randomized controlled trial. Pharmacogenomics. 2013;14:1593–1603. doi: 10.2217/pgs.13.145.  [DOI](https://doi.org/10.2217/pgs.13.145) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24088130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Impact%20of%20genotype-guided%20dosing%20on%20anticoagulation%20visits%20for%20adults%20starting%20warfarin:%20a%20randomized%20controlled%20trial&author=DE%20Jonas&author=JP%20Evans&author=HL%20McLeod&volume=14&publication_year=2013&pages=1593-1603&pmid=24088130&doi=10.2217/pgs.13.145&)

23. Stergiopoulos K, Brown DL. Genotype-guided vs clinical dosing of warfarin and its analogues: meta-analysis of randomized clinical trials. J Am Med Assoc Intern Med. 2014;174:1330–1338. doi: 10.1001/jamainternmed.2014.2368.  [DOI](https://doi.org/10.1001/jamainternmed.2014.2368) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24935087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Med%20Assoc%20Intern%20Med&title=Genotype-guided%20vs%20clinical%20dosing%20of%20warfarin%20and%20its%20analogues:%20meta-analysis%20of%20randomized%20clinical%20trials&author=K%20Stergiopoulos&author=DL%20Brown&volume=174&publication_year=2014&pages=1330-1338&pmid=24935087&doi=10.1001/jamainternmed.2014.2368&)

24. Li X, Yang J, Wang X et al. Clinical benefits of pharmacogenetic algorithm-based warfarin dosing: meta-analysis of randomized controlled trials. Thromb Res. 2015;135:621–629. doi: 10.1016/j.thromres.2015.01.018.  [DOI](https://doi.org/10.1016/j.thromres.2015.01.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25628141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Res&title=Clinical%20benefits%20of%20pharmacogenetic%20algorithm-based%20warfarin%20dosing:%20meta-analysis%20of%20randomized%20controlled%20trials&author=X%20Li&author=J%20Yang&author=X%20Wang&volume=135&publication_year=2015&pages=621-629&pmid=25628141&doi=10.1016/j.thromres.2015.01.018&)

25. Franchini M, Mengoli C, Cruciani M et al. Effects on bleeding complications of pharmacogenetic testing for initial dosing of vitamin K antagonists: a systematic review and meta-analysis. J Thromb Haemost. 2014;12:1480–1487. doi: 10.1111/jth.12647.  [DOI](https://doi.org/10.1111/jth.12647) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25040440/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Haemost&title=Effects%20on%20bleeding%20complications%20of%20pharmacogenetic%20testing%20for%20initial%20dosing%20of%20vitamin%20K%20antagonists:%20a%20systematic%20review%20and%20meta-analysis&author=M%20Franchini&author=C%20Mengoli&author=M%20Cruciani&volume=12&publication_year=2014&pages=1480-1487&pmid=25040440&doi=10.1111/jth.12647&)

26. Belley-Cote EP, Hanif H, D’Aragon F et al. Genotype-guided versus standard vitamin K antagonist dosing algorithms in patients initiating anticoagulation. A systematic review and meta-analysis. Thromb Haemost. 2015;114:768–777. doi: 10.1160/TH15-01-0071.  [DOI](https://doi.org/10.1160/TH15-01-0071) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26158747/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Genotype-guided%20versus%20standard%20vitamin%20K%20antagonist%20dosing%20algorithms%20in%20patients%20initiating%20anticoagulation.%20A%20systematic%20review%20and%20meta-analysis&author=EP%20Belley-Cote&author=H%20Hanif&author=F%20D%E2%80%99Aragon&volume=114&publication_year=2015&pages=768-777&pmid=26158747&doi=10.1160/TH15-01-0071&)

27. Wang ZQ, Zhang R, Zhang PP et al. Pharmacogenetics-based warfarin dosing algorithm decreases time to stable anticoagulation and the risk of major hemorrhage: an updated meta-analysis of randomized controlled trials. J Cardiovasc Pharmacol. 2015;65:364–370. doi: 10.1097/FJC.0000000000000204.  [DOI](https://doi.org/10.1097/FJC.0000000000000204) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25551322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cardiovasc%20Pharmacol&title=Pharmacogenetics-based%20warfarin%20dosing%20algorithm%20decreases%20time%20to%20stable%20anticoagulation%20and%20the%20risk%20of%20major%20hemorrhage:%20an%20updated%20meta-analysis%20of%20randomized%20controlled%20trials&author=ZQ%20Wang&author=R%20Zhang&author=PP%20Zhang&volume=65&publication_year=2015&pages=364-370&pmid=25551322&doi=10.1097/FJC.0000000000000204&)

28. Gharani N, Keller MA, Stack CB et al. The Coriell personalized medicine collaborative pharmacogenomics appraisal, evidence scoring and interpretation system. Genome Med. 2013;5:93. doi: 10.1186/gm499.  [DOI](https://doi.org/10.1186/gm499) | [PMC free article](/articles/PMC3978656/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24134832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med&title=The%20Coriell%20personalized%20medicine%20collaborative%20pharmacogenomics%20appraisal,%20evidence%20scoring%20and%20interpretation%20system&author=N%20Gharani&author=MA%20Keller&author=CB%20Stack&volume=5&publication_year=2013&pages=93&pmid=24134832&doi=10.1186/gm499&)

29. Mega JL, Walker JR, Ruff CTet al. Genetics and the clinical response to warfarin and edoxaban: findings from the randomised, double-blind ENGAGE AF-TIMI 48 trial. Lancet 2015. Epub ahead of print.  [DOI](https://doi.org/10.1016/S0140-6736(14)61994-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25769357/)

30. Dunnenberger HM, Crews KR, Hoffman JM et al. Preemptive clinical pharmacogenetics implementation: current programs in five US medical centers. Ann Rev Pharmacol Toxicol. 2015;55:89–106. doi: 10.1146/annurev-pharmtox-010814-124835.  [DOI](https://doi.org/10.1146/annurev-pharmtox-010814-124835) | [PMC free article](/articles/PMC4607278/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25292429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Rev%20Pharmacol%20Toxicol&title=Preemptive%20clinical%20pharmacogenetics%20implementation:%20current%20programs%20in%20five%20US%20medical%20centers&author=HM%20Dunnenberger&author=KR%20Crews&author=JM%20Hoffman&volume=55&publication_year=2015&pages=89-106&pmid=25292429&doi=10.1146/annurev-pharmtox-010814-124835&)

31. Gage BF.warfarindosingAvailable at:http://www.warfarindosing.org  [http://www.warfarindosing.org](http://www.warfarindosing.org)

32. Sagreiya H, Berube C, Wen A et al. Extending and evaluating a warfarin dosing algorithm that includes CYP4F2 and pooled rare variants of CYP2C9. Pharmacogenet Genom. 2010;20:407–413. doi: 10.1097/FPC.0b013e328338bac2.  [DOI](https://doi.org/10.1097/FPC.0b013e328338bac2) | [PMC free article](/articles/PMC3098751/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20442691/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genom&title=Extending%20and%20evaluating%20a%20warfarin%20dosing%20algorithm%20that%20includes%20CYP4F2%20and%20pooled%20rare%20variants%20of%20CYP2C9&author=H%20Sagreiya&author=C%20Berube&author=A%20Wen&volume=20&publication_year=2010&pages=407-413&pmid=20442691&doi=10.1097/FPC.0b013e328338bac2&)

33. Keller M, Gordon ES, Stack CB et al. The Coriell Personalized Medicine Collaborative: A prospective study of the utility of personalized medicine. Personal Med. 2010;7:301–317. doi: 10.2217/pme.10.13.  [DOI](https://doi.org/10.2217/pme.10.13) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29776223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Personal%20Med&title=The%20Coriell%20Personalized%20Medicine%20Collaborative:%20A%20prospective%20study%20of%20the%20utility%20of%20personalized%20medicine&author=M%20Keller&author=ES%20Gordon&author=CB%20Stack&volume=7&publication_year=2010&pages=301-317&pmid=29776223&doi=10.2217/pme.10.13&)

34. Stack CB, Gharani N, Gordon ES et al. Genetic risk estimation in the Coriell Personalized Medicine Collaborative. Genet Med. 2011;13:131–139. doi: 10.1097/GIM.0b013e318201164c.  [DOI](https://doi.org/10.1097/GIM.0b013e318201164c) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21233721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=Genetic%20risk%20estimation%20in%20the%20Coriell%20Personalized%20Medicine%20Collaborative&author=CB%20Stack&author=N%20Gharani&author=ES%20Gordon&volume=13&publication_year=2011&pages=131-139&pmid=21233721&doi=10.1097/GIM.0b013e318201164c&)

35. Diseati L, Scheinfeldt LB, Kasper RS et al. Common genetic risk for melanoma encourages preventive behavior change. J Personal Med. 2015;5:36–49. doi: 10.3390/jpm5010036.  [DOI](https://doi.org/10.3390/jpm5010036) | [PMC free article](/articles/PMC4384058/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25695399/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Personal%20Med&title=Common%20genetic%20risk%20for%20melanoma%20encourages%20preventive%20behavior%20change&author=L%20Diseati&author=LB%20Scheinfeldt&author=RS%20Kasper&volume=5&publication_year=2015&pages=36-49&pmid=25695399&doi=10.3390/jpm5010036&)

36. Schmidlen TJ, Wawak L, Kasper R et al. Personalized genomic results: analysis of informational needs. J Genetic Counsel. 2014;23:578–587. doi: 10.1007/s10897-014-9693-8.  [DOI](https://doi.org/10.1007/s10897-014-9693-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24488620/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Genetic%20Counsel&title=Personalized%20genomic%20results:%20analysis%20of%20informational%20needs&author=TJ%20Schmidlen&author=L%20Wawak&author=R%20Kasper&volume=23&publication_year=2014&pages=578-587&pmid=24488620&doi=10.1007/s10897-014-9693-8&)

37. Schmidlen TJ, Scheinfeldt L, Zhaoyang Ret al. Genetic Knowledge Among Participants in the Coriell Personalized Medicine Collaborative. J Genetic Counsel 2015. Epub ahead of print.  [DOI](https://doi.org/10.1007/s10897-015-9883-z) | [PMC free article](/articles/PMC4769688/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26306685/)

38. Scheinfeldt LB, Gharani N, Kasper RS et al. Using the Coriell Personalized Medicine Collaborative Data to conduct a genome-wide association study of sleep duration. Am J Med Genetics B. 2015;168:697–705. doi: 10.1002/ajmg.b.32362.  [DOI](https://doi.org/10.1002/ajmg.b.32362) | [PMC free article](/articles/PMC5049662/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26333835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genetics%20B&title=Using%20the%20Coriell%20Personalized%20Medicine%20Collaborative%20Data%20to%20conduct%20a%20genome-wide%20association%20study%20of%20sleep%20duration&author=LB%20Scheinfeldt&author=N%20Gharani&author=RS%20Kasper&volume=168&publication_year=2015&pages=697-705&pmid=26333835&doi=10.1002/ajmg.b.32362&)

39. Zineh I, Pacanowski M, Woodcock J. Pharmacogenetics and coumarin dosing--recalibrating expectations. N Engl J Med. 2013;369:2273–2275. doi: 10.1056/NEJMp1314529.  [DOI](https://doi.org/10.1056/NEJMp1314529) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24328463/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Pharmacogenetics%20and%20coumarin%20dosing--recalibrating%20expectations&author=I%20Zineh&author=M%20Pacanowski&author=J%20Woodcock&volume=369&publication_year=2013&pages=2273-2275&pmid=24328463&doi=10.1056/NEJMp1314529&)

40. Rothman KJ. Six Persistent Research Misconceptions. J Gen Intern Med. 2014;29:1060–1064. doi: 10.1007/s11606-013-2755-z.  [DOI](https://doi.org/10.1007/s11606-013-2755-z) | [PMC free article](/articles/PMC4061362/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24452418/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gen%20Intern%20Med&title=Six%20Persistent%20Research%20Misconceptions&author=KJ%20Rothman&volume=29&publication_year=2014&pages=1060-1064&pmid=24452418&doi=10.1007/s11606-013-2755-z&)

41. Ray T.Two Conflicting Prospective, RCTs on Warfarin PGx Provide No Definitive Guidance to PhysiciansAvailable at:http://www.genomeweb.com/clinical-genomics/two-conflicting-prospective-rcts-warfarin-pgx-provide-no-definitive-guidance-phy  [http://www.genomeweb.com/clinical-genomics/two-conflicting-prospective-rcts-warfarin-pgx-provide-no-definitive-guidance-phy](http://www.genomeweb.com/clinical-genomics/two-conflicting-prospective-rcts-warfarin-pgx-provide-no-definitive-guidance-phy)

42. Cavallari LH, Kittles RA, Perera MA. Genotype-guided dosing of vitamin K antagonists. N Engl J Med. 2014;370:1763. doi: 10.1056/NEJMc1402521.  [DOI](https://doi.org/10.1056/NEJMc1402521) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24785218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Genotype-guided%20dosing%20of%20vitamin%20K%20antagonists&author=LH%20Cavallari&author=RA%20Kittles&author=MA%20Perera&volume=370&publication_year=2014&pages=1763&pmid=24785218&doi=10.1056/NEJMc1402521&)

43. Drozda K, Wong S, Patel SR et al. Poor warfarin dose prediction with pharmacogenetic algorithms that exclude genotypes important for African Americans. Pharmacogen Genom. 2015;25:73–81. doi: 10.1097/FPC.0000000000000108.  [DOI](https://doi.org/10.1097/FPC.0000000000000108) | [PMC free article](/articles/PMC4280295/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25461246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogen%20Genom&title=Poor%20warfarin%20dose%20prediction%20with%20pharmacogenetic%20algorithms%20that%20exclude%20genotypes%20important%20for%20African%20Americans&author=K%20Drozda&author=S%20Wong&author=SR%20Patel&volume=25&publication_year=2015&pages=73-81&pmid=25461246&doi=10.1097/FPC.0000000000000108&)

44. Duconge J, Cadilla CL, Seip RL et al. Why admixture matters in genetically-guided therapy: missed targets in the COAG and EU-PACT trials. Puerto Rico Health Sci J. 2015;34:175–177.  [PMC free article](/articles/PMC4770896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26454897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Puerto%20Rico%20Health%20Sci%20J&title=Why%20admixture%20matters%20in%20genetically-guided%20therapy:%20missed%20targets%20in%20the%20COAG%20and%20EU-PACT%20trials&author=J%20Duconge&author=CL%20Cadilla&author=RL%20Seip&volume=34&publication_year=2015&pages=175-177&pmid=26454897&)

45. Mersha TB, Abebe T. Self-reported race/ethnicity in the age of genomic research: its potential impact on understanding health disparities. Human Genom. 2015;9:1. doi: 10.1186/s40246-014-0023-x.  [DOI](https://doi.org/10.1186/s40246-014-0023-x) | [PMC free article](/articles/PMC4307746/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25563503/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Human%20Genom&title=Self-reported%20race/ethnicity%20in%20the%20age%20of%20genomic%20research:%20its%20potential%20impact%20on%20understanding%20health%20disparities&author=TB%20Mersha&author=T%20Abebe&volume=9&publication_year=2015&pages=1&pmid=25563503&doi=10.1186/s40246-014-0023-x&)

46. Finkelman BS, Gage BF, Johnson JA et al. Genetic warfarin dosing: tables versus algorithms. J Am Coll Cardiol. 2011;57:612–618. doi: 10.1016/j.jacc.2010.08.643.  [DOI](https://doi.org/10.1016/j.jacc.2010.08.643) | [PMC free article](/articles/PMC3107000/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21272753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol&title=Genetic%20warfarin%20dosing:%20tables%20versus%20algorithms&author=BS%20Finkelman&author=BF%20Gage&author=JA%20Johnson&volume=57&publication_year=2011&pages=612-618&pmid=21272753&doi=10.1016/j.jacc.2010.08.643&)

47. Dumas S, Rouleau-Mailloux E, Barhdadi A et al. Validation of patient-reported warfarin dose in a prospective incident cohort study. Pharmacoepidemiol Drug Safety. 2014;23(03):285–289. doi: 10.1002/pds.3571.  [DOI](https://doi.org/10.1002/pds.3571) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24458454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacoepidemiol%20Drug%20Safety&title=Validation%20of%20patient-reported%20warfarin%20dose%20in%20a%20prospective%20incident%20cohort%20study&author=S%20Dumas&author=E%20Rouleau-Mailloux&author=A%20Barhdadi&volume=23&issue=03&publication_year=2014&pages=285-289&pmid=24458454&doi=10.1002/pds.3571&)
