# Genomewide Gene-by-Sex Interaction Scans Identify ADGRV1 for Sex Differences in Opioid Dependent African Americans

## Metadata
**Authors:** Bao-Zhu Yang, Hang Zhou, Zhongshan Cheng, Henry R Kranzler, Joel Gelernter
**Journal:** Scientific Reports
**Date:** 2019 Dec 2
**DOI:** [10.1038/s41598-019-53560-0](https://doi.org/10.1038/s41598-019-53560-0)
**PMID:** 31792237
**PMCID:** PMC6889277
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6889277/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6889277/pdf/41598_2019_Article_53560.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6889277/pdf/41598_2019_Article_53560.pdf)

## Abstract

Sex differences in opioid dependence (OD) are genetically influenced. We conducted genomewide gene-by-sex interaction scans for the DSM-IV diagnosis of OD in 8,387 African-American (AA) or European-American subjects (43.6% women; 4,715 OD subjects). Among AAs, 9 SNPs were genome-wide significant at ADGRV1 (adhesion G-protein-coupled receptor V1, lead-SNP rs2366929*(C/T), p = 1.5 × 10−9) for sex-different risk of OD, with the rs2366929*C-allele increasing OD risk only for men. The top co-expressions in brain were between ADGRV1 and GRIK2 in substantia nigra and medullary inferior olivary nucleus, and between ADGRV1 and EFHC2 in frontal cortex and putamen. Significant sex-differential ADGRV1 expression from GTEx was detected in breast (Bonferroni-corrected-p < 0.002) and in heart (p < 0.0125), with nominal significance identified in brain, thyroid, lung, and stomach (p < 0.05). ADGRV1 co-expression and disease-enrichment analysis identifying the top 10 diseases showed strikingly sexually dimorphic risks. The enrichment and transcriptome analyses provided convergent support that ADGRV1 exerts a sex-different effect on OD risk. This is the first study to identify genetic variants contributing to sex differences in OD. It shows that ADGRV1 contributes to OD risk only in AA men, a finding that warrants further study.

Subject terms: Gene regulatory networks, Microarrays, Behavioural genetics

## Introduction

Sex differences in opioid dependence (OD) have a strong biological basis^1^ with a genetic component^2,3^. Heritability estimates of heroin use are around 0.5^2^. Although there is no report for sex-specific heritability estimates of OD specifically, heritability estimates of drug dependence are higher in men than women^4^. In animal models, genetic effects related to OD have differed by sex^5–7^. For example, a mouse model of the human *OPRM1* (A118G) polymorphism found genotype-by-sex-specific reductions in the rewarding properties of morphine^5^. This variant does not, however, appear to affect OD risk in humans^8^. To our knowledge, the only systematic genome-wide search for the genes responsible for sex differences in human OD was a linkage study^3^.

Systematic search for the OD genetic risk variants (regardless of sex differences) using the genome-wide association study (GWAS) design has been reported. We published the first GWAS with genomewide significant findings^9^. The most compelling results in that study were the identification of genes involved in potassium signaling pathways (i.e., *KCNC1* (*Potassium Voltage-Gated Channel Subfamily C Member 1*) and *KCNG2* (*Potassium Voltage-Gated Channel Modifier Subfamily G Member 2*)) in the African American (AA) population, and genes involved in calcium signaling and long-term potentiation. Another GWAS was conducted in an Australian cohort, in which genetic data from opioid-dependent daily injectors were compared with that from opioid misusers who never progressed to daily injection, and identified several genomewide significant variants in *CNIH3* (*Cornichon Family AMPA Receptor Auxiliary Protein 3*)^10^. In a recent study, we found that a variant on chromosome 15, rs12442183, near *RGMA* (*Repulsive Guidance Molecule A*), was genome-wide significantly associated with OD in the European American (EA) population^11^. *RGMA* encodes a central nervous system axon guidance protein called repulsive guidance molecule A. Risk allele rs12442183*T was related with higher expression of a specific *RGMA* transcript variant in frontal cortex. After chronic morphine injection, the homologous mouse gene, *Rgma*, was upregulated in the striatum of C57BL/6 J mice^11^.

We aimed to identify genetic variants exerting a sex difference in susceptibility to OD in a GWAS framework using the cohort of substance use disorder we have collected; characterize and annotate the identified genetic variants using publicly available databases of co-expressed genes and enrichment analysis; and use transcriptome analysis to identify biological mechanisms consistent with sex-specific effects on OD risk for the variants identified.

## Materials and Methods

### Subjects and the semi-structured assessment

This study analyzed data obtained from African-American (AA) and European-American (EA) participants in the Yale-Penn genetics of substance dependence cohort (total N = 8,387). The 4,944 AA and 3,443 EA samples were recruited from 2000 to 2013, as previously described^9,12^. These subjects represent two phases of collection: Yale-Penn-1 (N = 4,970) and Yale-Penn-2 (N = 3,417), which differed in their period of recruitment and the genotyping platforms used. The demographic characteristics of the study cohort are presented in Table [1](#Tab1). We assessed all subjects using the Semi-Structured Assessment for Drug Dependence and Alcoholism (SSADDA)^13^ and obtained certificates of confidentiality for all subjects from the National Institute on Drug Abuse (NIDA) and the National Institute on Alcohol Abuse and Alcoholism (NIAAA). The study protocol was approved by the Yale Institutional Review Board and the study was performed in accordance with the relevant guidelines and regulations. All subjects provided written informed consent

### Table 1.

| Characteristic | African American |  |  | European American |  |  | Total |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Yale-Penn 1 | Yale-Penn 2 | Subtotal | Yale-Penn 1 | Yale-Penn 2 | Subtotal |  |  |
| N Male, N Female, N Female % | 3,227 1,707 1,520 47.10% | 1,717 1,001 716 41.70% | 4,944 2,709 2,235 45.20% | 1,743 1,011 732 42% | 1,700 1,013 687 40.40% | 3,443 2,024 1,419 41.20% | 8,387 4,730 3,657 43.60% |
| Age (SD) | 41.1 (9.0) | 40.8 (10.9) | 41.0 (9.7) | 38.0 (10.9) | 39.4 (13.0) | 38.7 (12.0) | 40.1 (10.7) |
| OD affected, N Male, N Female, N Female % | 1,684 994 690 41% | 919 643 276 30% | 2,603 1,637 966 37.10% | 1,086 682 404 37.20% | 1,026 683 343 33.40% | 2,112 1,364 748 35.40% | 4,715 2,999 1,716 36.40% |
| OD unaffected, N Male, N Female, N Female % | 1,543 713 830 53.80% | 798 358 440 55.10% | 2,341 1,070 1,271 54.30% | 657 329 328 49.90% | 674 330 344 51% | 1,331 659 672 50.50% | 3,672 1,730 1,942 52.90% |

Table 1 Caption: Demographic characteristics of the study samples.

### Genotype quality control, population stratification, and imputation

We genotyped the Yale-Penn-1 sample with the Illumina HumanOmni1-Quad array of approximately 988,000 single nucleotide polymorphisms (SNPs), and the Yale-Penn-2 sample with the Illumina HumanCore Exome array of approximately 266,000 exonic SNPs and 240,000 tagging SNPs. We excluded SNPs with a genotype call rate <98% or minor allele frequency (MAF) <1%.

We used the following measures to differentiate AA or EA subjects and control for population stratification. First, we conducted principal component (PC) analysis^14^ for the SNPs common among the genetic data for the Yale-Penn-1, Yale-Penn-2, and the 1000 Genomes phase 3 reference panel, which contains African, admixed American, European, East Asian and South Asian populations^15^. We then trimmed SNPs in LD (*r*^2^ > 0.2) using PLINK^16^. Using the remaining SNPs, we clustered the Yale-Penn subjects into different groups compared to the reference populations by the first three PCs in Euclidean space. Subjects were removed from the subsequent analyses if they were not clustered with African or European populations. Finally, we conducted a second PC analysis within each group to remove outliers greater than three standard deviations from the mean Euclidean distances. The resultant first 10 PCs were covariates in all subsequent association analyses to adjust for residual population stratification.

We imputed GWAS data to the 1000 Genomes phase 3 reference panel^15^, using Minimac3 implemented in the Michigan Imputation Server^17^. Post-imputation SNP exclusion metrics included: Hardy-Weinberg equilibrium *p < *10^−6^, imputation accuracy < 0.8, or MAF < 5%. The final SNP counts for the subsequent association analyses were: Yale-Penn-1 sample, 8,775,706 SNPs in AAs and 6,417,418 in EAs; Yale-Penn-2 sample, 6,702,161 SNPs in AAs and 5,205,763 in EAs.

### Genomewide gene-by-sex interaction scan for sex differences in OD

We performed genetic association tests for the DSM-IV diagnosis OD using a linear mixed model implemented in the program GEMMA^18^. A standard linear mixed-effect model was chosen to control for the relatedness among participants as our cohort contained a mixture of individuals ascertained using both an unrelated case-control design and a family design (11.9% of AAs and 7.9% of EAs came from the family design). We chose the standard linear mixed effect model implemented in GEMMA, where the effect estimates asymptotically converge to estimates of logistic regression for a binary outcome (such as OD affected versus OD unaffected here) when the sample size is large. We tested the gene-by-sex interaction effect and included the main effects of sex and SNP in addition to controlling for age, four other substance dependence diagnoses (alcohol, cocaine, nicotine, cannabis) and the first 10 PCs, to examine sex differences in genetic risk for OD. This model was examined separately for each of four datasets, i.e., both AA and EA subjects in the Yale-Penn-1 and Yale-Penn-2 samples. We then meta-analyzed the interactive effects of SNP-by-sex from Yale-Penn-1 and Yale-Penn-2 within each population using the inverse variance method that converted all the effects into a signed Z-score and tested for association using the Z-test, which was implemented in the program METAL^19^.

### Co-expression, sex differences in gene expression, disease enrichment, and functional annotation

For the enrichment analysis, we first queried the target gene, i.e., the gene harboring the genomewide significant variants detected, to identify co-expressed genes using COXPRESdb^20^, a coexpression database of DNA-microarray and RNAseq-based expression data. After obtaining the top 100 co-expressed genes of the targeted gene (or genes) identified from the association analyses, we assessed functional enrichment using the WEB-based GEne SeT AnaLysis Toolkit (WebGestalt)^21^, a functional enrichment analysis web tool (see supplementary material for the setting of parameters). WebGestalt 2017 supports 12 organisms and 150,937 functional categories from public databases and computational analyses^21^. The enrichment analysis method ‘Disease Association Analysis’ of WebGestalt was used to test for enrichment of disease-associated genes among the top co-expressed genes with the targeted gene.

We studied sex differences in the expression of the targeted gene, i.e., *ADGRV1*, to investigate the functional roles of the top variants and the disease-enriched genes to provide insight into the molecular mechanism of sex differences in OD. To do this, we used the Genotype-Tissue Expression (GTEx) data and the tissue-specific gene expression database, the Brain eQTL Almanac, Braineac^22^. The RNA-Seq data of GTEx Analysis V7 (phs000424.v7.p2), included in the file, ‘GTEx_Analysis_2016-01-15_v7_RNASeQCv1.1.8_gene_tpm.gct.gz’, were downloaded from the GTEx Portal datasets^23^, and information about tissues and sex was retrieved from the files, ‘GTEx_v7_Annotations_SampleAttributesDS.txt’ and ‘GTEx_v7_Annotations_SubjectPhenotypesDS.txt’, respectively. We downloaded the gene expression (RNA-seq) data and the related phenotypes and analyzed the data using the general linear model, implemented in the SAS procedure “ proc glm,” where sex was modeled as the main effect in association with the levels of gene expression.

## Results

### Genomewide gene-by-sex interaction scan

We summarized the result for each dataset and the meta-analysis within each population in the Manhattan plots (Supplementary Figure [S1](#MOESM1)). The QQ plots show no systematic bias (Figure [S1](#MOESM1)). A peak with the lead SNP rs2366929 and another eight SNPs in strong LD (*r*^2^ > 0.9) with the lead SNP (*r*^2^ > 0.8) reached genomewide significance (all *p’s* < 5 × 10^−8^) in the gene-by-sex interaction scans for OD in the meta-analyzed AA samples (Fig. [1a](#Fig1)). For lead SNP rs2366929, each subsample also showed dose-response effects of nominal significance with *p* = 2.19 × 10^−5^ and *p* = 1.23 × 10^−5^ for Yale-Penn 1 and Yale-Penn 2, respectively. This peak of genomewide significance is located at *ADGRV1* (*Adhesion G protein-coupled receptor V1*) on chromosome 5q14.3. The gene-by-sex interaction effects exhibit a gene dose-response relationship for the OD risk in AA men but not women (Fig. [1b](#Fig1)). That is, the proportions of OD-affected to the total subjects (OD affected plus controls), within each of the three genotypes consistently show a gene dose-response trend relationship in men but not women (for lead SNP rs2366929*(C/T), chi-square test for trend in proportions, *p* = 1.1 × 10^−14^ in men and *p* = 0.15 in women). Figure [1b](#Fig1) displays the increasing trend of the OD-affected proportions for men (CC > CT > TT) compared with women for the lead SNP rs2366929. This increasing trend in men versus women was observed for all nine SNPs identified in the association analyses (Supplementary Table [S1](#MOESM1) shows the trend tests for all nine SNPs). The MAF for the lead SNP increased 10.5% for the OD-affected versus the OD-unaffected men (0.353 vs. 0.248; comparison of two proportions, *p < *0.0001); however, this increase was not present in women (0.241 vs. 0.265; *p* = 0.15). Table [2](#Tab2) displays the characteristics of the nine genome-wide significant SNPs.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3509/6889277/44057331c661/41598_2019_53560_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6889277_41598_2019_53560_Fig1_HTML.jpg)

Genomewide gene-by-sex interaction scans on opioid dependence (OD). (a) Regional association plot at the ADGRV1 (aka. GPR98) locus for the meta-analysis of the two phases of the African American (AA) sample. (b) Proportions of the OD affected for the AA sample for each genotype for the lead SNP rs2366929.

### Table 2.

| SNP | SNP rsID | Position on Chr5/GRCh37 | LD (r2) | Distance from the lead SNP (bp) | Alleles | Male |  | Female |  | Beta* | Standard error* | Meta p-value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| MAF OD Affected N = 1637 | MAF OD Unaffected N = 1070 | MAF OD Affected N = 966 | MAF OD Unaffected N = 1271 |  |  |  |  |  |  |  |  |  |
| 1 | rs2030272 | 90025842 | 0.75 | −792 | C/G | 0.31 | 0.213 | 0.209 | 0.222 | 0.10 | 0.019 | 4.55e−08 |
| 2 | rs2366929 | 90026634 |   | 0 | C/T | 0.353 | 0.248 | 0.241 | 0.265 | 0.11 | 0.018 | 1.51e−09 |
| 3 | rs2222243 | 90033697 | 0.75 | 7063 | A/C | 0.31 | 0.213 | 0.209 | 0.223 | 0.10 | 0.019 | 4.96e−08 |
| 4 | rs2443067 | 90034302 | 0.75 | 7668 | A/G | 0.31 | 0.213 | 0.209 | 0.223 | 0.10 | 0.019 | 4.96e−08 |
| 5 | rs2443066 | 90034784 | 0.73 | 8150 | A/G | 0.31 | 0.213 | 0.209 | 0.223 | 0.10 | 0.019 | 4.49e−08 |
| 6 | rs2460186 | 90036980 | 0.75 | 10346 | A/G | 0.311 | 0.213 | 0.209 | 0.224 | 0.10 | 0.019 | 4.21e−08 |
| 7 | rs2460187 | 90038450 | 0.72 | 11816 | G/T | 0.311 | 0.213 | 0.209 | 0.224 | 0.10 | 0.019 | 4.21e−08 |
| 8 | rs2443065 | 90053056 | 0.70 | 26422 | A/G | 0.313 | 0.216 | 0.211 | 0.227 | 0.10 | 0.019 | 4.04e−08 |
| 9 | rs2443064 | 90053280 | 0.70 | 26646 | T/C | 0.313 | 0.216 | 0.211 | 0.227 | 0.10 | 0.019 | 4.04e−08 |

Table 2 Caption: Genomewide significant SNPs located at the ADGRV1 locus with differential effects on susceptibility to opioid dependence between male and female for the African American sample.

We identified no genomewide significant signals in the EA sample (Supplementary Figure [S2](#MOESM1)) and the trans-ethnic meta-analysis of the AA and EA samples (Supplementary Figure [S3](#MOESM1)).

### Sex difference in ADGRV1 gene expression

We used GTEx data to identify tissues with differential *ADGRV1* gene expression by sex for further annotating *ADGRV1*. We detected significant sex-differential *ADGRV1* gene expression in breast (Bonferroni correction *p* < 0.002) and in heart (*p* < 0.0125) based on GTEx transcriptome data (dbGaP Accession phs000424.v7.p2) (Table [3](#Tab3)). The *ADGRV1* expression in heart supports a previous study of sexually dimorphic *ADGRV1* expression in patients with non-ischemic human heart failure (Supplementary Table [S2](#MOESM1)), in which *ADGRV1* expression was lower in men than women, the same effect direction to what we found in the GTEx transcriptome. We identified four additional tissues with sex-differential *ADGRV1* gene expression that were nominally significant, including brain (*p* = 0.018), thyroid (*p* = 0.021), lung (*p* = 0.038), and stomach (*p* = 0.045) (Table [3](#Tab3)).

### Table 3.

| Tissue | Average Gene Expressiona (N) |  | df | F Value | Pr > F |
| --- | --- | --- | --- | --- | --- |
| Male | Female |  |  |  |  |
| Adipose Tissue | 0.031 (526) | 0.025 (271) | (1, 795) | 1.46 | 0.227 |
| Adrenal Gland | 62.92 (108) | 53.50 (82) | (1, 188) | 2.67 | 0.104 |
| Bladder | 1.31 (6) | 0.76 (5) | (1, 9) | 0.66 | 0.439 |
| Blood | 0.089 (349) | 0.070 (188) | (1, 535) | 0.36 | 0.546 |
| Blood Vessel | 0.042 (593) | 0.036 (320) | (1, 911) | 0.13 | 0.722 |
| Brain | 6.06 (1162) | 6.81 (509) | (1, 1669) | 5.6 | 0.0181* |
| Breast | 0.65 (175) | 1.33 (115) | (1, 288) | 35.27 | <0.0001** |
| Colon | 0.10 (306) | 0.11 (201) | (1, 505) | 1.87 | 0.173 |
| Esophagus | 0.12 (647) | 0.12 (374) | (1, 1019) | 0.21 | 0.644 |
| Heart | 0.018 (402) | 0.025 (198) | (1, 598) | 6.28 | 0.0125* |
| Kidney | 0.50 (36) | 0.37 (9) | (1, 43) | 0.38 | 0.542 |
| Liver | 0.59 (120) | 0.58 (55) | (1, 173) | 0 | 0.962 |
| Lung | 1.42 (286) | 1.61 (141) | (1, 425) | 4.33 | 0.038* |
| Muscle | 0.043 (371) | 0.039 (193) | (1, 562) | 0.17 | 0.684 |
| Nerve | 0.10 (274) | 0.10 (140) | (1, 412) | 0.11 | 0.738 |
| Pancreas | 0.43 (150) | 0.45 (98) | (1, 246) | 0.75 | 0.386 |
| Pituitary | 9.96 (129) | 9.95 (54) | (1, 181) | 0 | 0.995 |
| Salivary Gland | 0.62 (71) | 0.53 (26) | (1, 95) | 1.64 | 0.203 |
| Skin | 0.17 (790) | 0.17 (413) | (1, 1201) | 0.01 | 0.909 |
| Small Intestine | 0.070 (86) | 0.059 (51) | (1, 135) | 1.24 | 0.268 |
| Spleen | 0.03 (98) | 0.02 (64) | (1, 160) | 0.61 | 0.436 |
| Stomach | 0.16 (160) | 0.20 (102) | (1, 260) | 4.08 | 0.0445* |
| Thyroid | 6.91 (293) | 8.42 (153) | (1, 444) | 5.36 | 0.0211* |

Table 3 Caption: ADGRV1 gene expression: Tests of sexually dimorphic expression using Genotype-Tissue Expression (GTEx) project data.

### ADGRV1 co-expression and disease enrichment analysis

The top 100 genes co-expressed with *ADGRV1* (Supplementary Table [S3](#MOESM1)) identified by COEXPRESdb were subject to disease enrichment analysis. We found that *ADGRV1* and 13 *ADGRV1* co-expressed genes were significantly enriched within 10 diseases (all raw *p’s* < 0.01 and adjusted *p’s* < 0.05, Supplementary Table [S4](#MOESM1)). The 13 genes are *ABCC12*, *CFTR*, *CTAGE1*, *DAOA-AS1*, *EFHC2*, *FHL5*, *GRIK2*, *NAV3*, *SPO11*, *SYCP2*, *TAAR9*, *TPH2*, and *ZNF157*. Among the 10 diseases, two are male-specific (male infertility and non-syndromic oligospermia), four affect males more than females (X-linked mental retardation, generalized epilepsy, Sezary syndrome, cutaneous T-cell lymphoma), three affect females more than males (chronic fatigue syndrome, panic disorder, and pseudoxanthoma elasticum), and one displays distinct disease characteristics between males and females (personality disorders).

We also correlated the expression of *ADGRV1* with the expression of the 12 co-expressed disease risk genes (*DAOA-AS1* expression was not available) across the 10 human brain tissues in the Braineac database. The correlation pattern of gene expression across the 10 brain tissues is displayed in Fig. [2](#Fig2). Robust correlations of gene expression between *ADGRV1* and two genes, *GRIK2* (Glutamate ionotropic receptor kainate type subunit 2) and *EFHC2* (EF-hand domain containing 2), are ubiquitous across various human brain tissues. The highest correlations between the gene expressions of *ADGRV1* and *GRIK2* are in the substantia nigra and medulla inferior olivary nucleus and between *ADGRV1* and *EFHC2* in the frontal cortex and putamen (Fig. [2](#Fig2)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3509/6889277/c68592d35ed6/41598_2019_53560_Fig2_HTML.jpg)

The mRNA expression of ADGRV1 was associated with 12 co-expressed genes across 10 human brain tissues in the Braineac database (Ramasamy et al.22). Pairwise Pearson correlation coefficients (r) for ADGRV1 and the 12 genes are labeled in each square of the heatmap. The color bar indicates the extent of significance, −log10(p−values), for the correlation coefficients.

## Discussion

To our knowledge, this is the first systematic search for genetic variants contributing to sex differences in OD risk using GWAS. We identified a male-specific effect of *ADGRV1* on risk of OD in the AA sample using genome-wide gene-by-sex interaction scans. Co-expressed genes, the enrichment analysis, and the transcriptome analysis provided mechanistic support for the finding that *ADGRV1* exerts a sex-specific effect on OD risk.

*ADGRV1* spans approximately 605.5 kb on chromosome 5q14.3 and encodes a member of the G-protein coupled receptor superfamily. In addition to the N- and C-terminal domains of ADGRV1, the encoded protein contains 7 putative Na( + )/Ca(2 + ) exchangers (defining the cation binding domain) and 21 N-linked glycosylation sites^24^. Its extracellular repeat domains bind Ca(2 + ) and are involved in signal transduction^24^. Members of the ADGR family play key roles in regulating cortical patterning, dendrite and synapse formation, and myelination^25^. Using GTEx transcriptome data and the Human Protein Atlas^26^, we found that *ADGRV1* is also highly expressed in three endocrine glands (adrenal gland, thyroid, pituitary), lung, and multiple brain regions (Supplementary Figure [S4](#MOESM1)). Opioid use disrupts hypothalamic-pituitary-adrenal (HPA) dynamics at the level of the pituitary or adrenal^27^. Chronic opioid use can cause pervasive endocrine dysfunction, which leads to hypogonadism, infertility, fatigue, anxiety, depression, menstrual irregularities, and so on^28,29^. Many of these disorders are among what we have identified for this study in the top ten diseases significantly enriched by the *ADGRV1* co-expressed genes, such as fatigue, infertility, anxiety or panic disorder (Supplemental Table [S4](#MOESM1)).

In the co-expressed genes and enrichment analyses, all of the top 10 diseases identified demonstrate sexually dimorphic risks or manifestations. We identified *ADGRV1* as a male-specific risk factor in OD. The top 10 diseases also include two that are male-specific (i.e., male infertility and non-syndromic oligospermia) or four that predominantly affect males more than females (i.e., X-linked mental retardation, generalized epilepsy, Sezary syndrome, or cutaneous T-cell lymphoma). On the other hand, three of the 10 diseases affect more females than males (chronic fatigue syndrome, panic disorder, and pseudoxanthoma elasticum)^30,31^. Collectively, the top 10 diseases for *ADGRV1* co-expressed genes are strikingly sexually dimorphic.

In disease-association studies, *ADGRV1* has been well examined. Mutations in *ADGRV1* are associated with familial febrile seizures^32^, and also contribute to focal and generalized epilepsy^33,34^, and epilepsy with myoclonic seizures^35^. Besides, *ADGRV1* is associated with cardiac conduction disorder^33^ and *ADGRV1* variants segregated in families with epilepsy co-occurring sudden death (due to cardiac conduction disorder), showing shared *ADGRV1* risk variants between epilepsy and cardiac conduction disorder^33^. *ADGRV1* expression in the heart has been reported for patients with non-ischemic heart failure (Supplementary Table [S2](#MOESM1)), where expression was lower in men than women^36^. This sex difference in *ADGRV1* expression in the heart was replicated in the GTEx transcriptome data (Table [3](#Tab3)), although disease status was not controlled in this analysis because of the disease heterogeneity of GTEx donors. *ADGRV1* has also been implicated in adverse metabolic effects of antipsychotic drugs^37^. Taken together, our novel discovery of *ADGRV1* for contributing to OD risk in men add into this scientific literature on the disease-gene association of the strong candidate gene *ADGRV1*.

In the GTEx data, individual-level data do not include the diagnosis of OD. Thus our investigation into the association between sex differences in *ADGRV1* expression and those in OD is not possible in that dataset. However, *ADGRV1* is highly expressed in endocrine tissues (Supplementary Figure [S4](#MOESM1)); opioid use could act on the tissues in which *ADGRV1* is highly expressed (i.e., endocrine and brain), disrupting the normal G-protein coupled receptor signaling and hormone production and causing pathogenic cellular processes. The differences in *ADGRV1* expression could affect stress responses or produce hormonal or behavioral effects that differ by sex in opioid users.

Regarding the co-expression patterns for the enriched genes expressed in brain, the identified brain regions show supportive evidence for neuropathology. The robust correlations in gene expression between *ADGRV1* and *GRIK2* and *EFHC2* are ubiquitous in human brain. The top correlations in gene expressions between *ADGRV1* and *GRIK2* are in the substantia nigra and medulla inferior olivary nucleus (Fig. [2](#Fig2)). The substantia nigra is enriched in dopaminergic neurons and plays an important role in reward^38^, while the medulla inferior olivary nucleus is implicated in motor learning and control^39^. Another set of top correlations in the expression are between *ADGRV1* and *EFHC2* in frontal cortex and putamen (Fig. [2](#Fig2)). The frontal cortex contains most of the dopamine-sensitive neurons in the cerebrum and is associated with reward, attention, short-term memory, planning, and motivation^40^. The putamen is involved in reinforcement learning and various movements^41^. These results support a role for *ADGRV1* in networks of co-expressed genes that regulate the neural activities involved in addiction-related functions, including reward, memory, and learning.

As for the other co-expressed genes involved in the top correlated brain regions, *TPH2* encodes tryptophan hydroxylase 2, which catalyzes the rate-limiting step in the synthesis of serotonin. Chronic morphine and cocaine exert common effects on tyrosine hydroxylase in dopaminergic brain reward regions^42^. Mutations in *TPH2* were associated with quality of life of patients in methadone maintenance treatment for heroin use disorder^43^, responses to cocaine treatment^44^, and a spectrum of psychiatric disorders (cf. a review and meta-analysis^45^). *GRIK2* encodes a member of the kainate family of glutamate receptor subunits. In a mouse model, *Grik2* mRNA levels were decreased after prolonged morphine treatment^46^. These results again support a role for *ADGRV1* in networks of co-expressed genes that regulate the neural activities involved in opioid or drug addiction.

The variants at *ADGRV1* reaching genomewide significance for a sex-difference in OD risk were identified in AAs, but not EAs. The minor allele “C” of lead SNP rs2366929 in the International Genome Sample Resource (IGSR)^15^ has MAF = 0.47 for Europeans, and MAF = 0.22 for Africans, consistent with the MAFs (0.248 male; 0.265 female) we observed in our admixed AA population (Table [2](#Tab2)). Local or population-specific variation is important in mapping disease risk^47,48^. Although the current information is not enough to determine why the genetic effects in *ADGRV1* were not seen in EAs, there are several plausible explanations. For example, effect sizes might be smaller in EA ancestries such that larger sample size may be necessary for discovery. Further studies with larger sample sizes and more power are warranted to investigate sex-differences in OD risk.

The GEMMA model we used for the current study includes 17 main effects (i.e., age, sex, SNP, and four other substance dependence on alcohol, cocaine, marijuana, and nicotine, and ten PCs) and one two-way interaction (i.e., SNP-by-sex). In theory, a saturated model that incorporates all of the two-way interaction effects would be ideal but would involve a total of 153 terms in the model, comprised of the 17 main effects and 136 pairwise interaction effects. We chose not to apply the saturated model because in previous simulations (unpublished data) we obtained effect estimates for the targeted interactions that were generally close to those estimated by the saturated model. However, the power was greatly reduced by the inclusion of a large number of interaction terms. We opted to use the more limited model.

The strengths of the present study include a moderately large cohort of substance dependence subjects who were carefully phenotyped and whose data were rigorously analyzed. However, the study findings are limited by the absence of a suitable cohort of AAs with OD in which to replicate the findings after we made extensive efforts to locate an existing dataset sufficiently powered for replication. Despite this limitation, we believe that the findings from this first study of sex- differences in genetic risk for OD can aid in understanding the underlying differences and facilitate better sex-specific prevention and treatment efforts for OD.

In summary, we identified *ADGRV1* as a risk locus contributing to increased risk of OD in AA men by examining genetic variants systematically on a genome-wide scale. Functional annotation of this finding corroborated a substantial role for *ADGRV1* in increasing OD risk, especially the potential pathogenic effects of variation in *AGDRV1* on cardio-cerebral mechanisms, which could contribute to the risk of fatal opioid overdose or respiratory depression that has been observed following high-dose opioid exposure. Further study of this finding is warranted.

## Supplementary information

## Acknowledgements

The Genotype-Tissue Expression Project was supported by the Common Fund of the Office of the Director of the National Institutes of Health, and by NCI, NHGRI, NHLBI, NIDA, NIMH, and NINDS. The data used for the analyses described in this manuscript were obtained from: ‘GTEx_Analysis_2016-01-15_v7_RNASeQCv1.1.8_gene_tpm.gct.gz’ in the GTEx Portal and the dbGaP accession number phs000424.v7.p2 on 07/31/2018. This work was supported by the National Institutes of Health Grant Nos R03DA047562 (to B.Z.Y.), and RC2 DA028909, R01 DA12690, R01 DA12690S1, R01 DA12849, R01 DA18432, R01 AA11330, and R01 AA017535 (to J.G.) and VA Connecticut Healthcare Center and Philadelphia VA Medical Center Mental Illness Research, Education and Clinical Centers. Genotyping services for a part of our genome-wide association study were provided by the Center for Inherited Disease Research and the Yale University Center for Genome Analysis, which is fully funded by Federal Contract No. N01-HG-65403 from the National Institutes of Health to the Johns Hopkins University. We thank James Poling, Ph.D., Yale University School of Medicine and APT Foundation, Roger Weiss, M.D., McLean Hospital, Kathleen Brady, M.D., Ph.D., and Raymond Anton, M.D., Medical University of South Carolina, and David Oslin, M.D., University of Pennsylvania, for their work in recruitment and assessment. We thank Ann Marie Lacobelle, M.S., and Christa Robinson, B.S., who provided technical assistance; Yaira Nunez and Michelle Slivinsky, who led the quality control effort for the Semi-Structured Assessment for Drug Dependence and Alcoholism interviews and phenotyping for the study sample; and John Farrell, Ph.D., Section of Biomedical Genetics, Boston University School of Medicine, who provided database management assistance.

## Author contributions

All authors contributed to the study design, data acquisition, or data analysis and interpretation. B.Z.Y. conceived the study, contributed to analytical modeling, data analysis, and interpretation, and drafted the manuscript. Both H.Z. and Z.C. performed data analysis and interpretation. H.R.K. contributed to data acquisition and interpretation. J.G. contributed to the study design as well as data acquisition and interpretation. All authors participated in revising the manuscript and approved the final version.

## Data availability

The datasets analyzed for the current study are not all publicly available. The Yale-Penn-1 subsample can be requested from the NCBI dbGaP repository: A Genome-Wide Association Study of Heroin Dependence (dbGaP Study Accession: phs000277.v1.p1). The Yale-Penn-2 subsample has not deposited to the dbGaP, but will be available from the corresponding author on reasonable request.

## Competing interests

H.R.K. has been a consultant or advisory board member for Alkermes, Indivior, and Lundbeck, and he is a member of the American Society of Clinical Psychopharmacology Alcohol Clinical Trials Initiative, which was supported in the past 3 years by AbbVie, Alkermes, Ethypharm, Indivior, Eli Lilly, Lundbeck, Otsuka, Pfizer, Arbor Pharmaceuticals, and Amygdala Neurosciences. Drs Kranzler and Gelernter are named as inventors on PCT patent application #15/878,640 entitled: “Genotype-guided dosing of opioid agonists”, filed January 24, 2018.

## Footnotes

## Supplementary information

is available for this paper at 10.1038/s41598-019-53560-0.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The datasets analyzed for the current study are not all publicly available. The Yale-Penn-1 subsample can be requested from the NCBI dbGaP repository: A Genome-Wide Association Study of Heroin Dependence (dbGaP Study Accession: phs000277.v1.p1). The Yale-Penn-2 subsample has not deposited to the dbGaP, but will be available from the corresponding author on reasonable request.

### Supplementary Materials

### Data Availability Statement

The datasets analyzed for the current study are not all publicly available. The Yale-Penn-1 subsample can be requested from the NCBI dbGaP repository: A Genome-Wide Association Study of Heroin Dependence (dbGaP Study Accession: phs000277.v1.p1). The Yale-Penn-2 subsample has not deposited to the dbGaP, but will be available from the corresponding author on reasonable request.

## References

1. Craft RM. Sex differences in analgesic, reinforcing, discriminative, and motoric effects of opioids. Experimental and clinical psychopharmacology. 2008;16:376–385. doi: 10.1037/a0012931.  [DOI](https://doi.org/10.1037/a0012931) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18837634/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Experimental%20and%20clinical%20psychopharmacology&title=Sex%20differences%20in%20analgesic,%20reinforcing,%20discriminative,%20and%20motoric%20effects%20of%20opioids&author=RM%20Craft&volume=16&publication_year=2008&pages=376-385&pmid=18837634&doi=10.1037/a0012931&)

2. Mistry CJ, Bawor M, Desai D, Marsh DC, Samaan Z. Genetics of Opioid Dependence: A Review of the Genetic Contribution to Opioid Dependence. Current psychiatry reviews. 2014;10:156–167. doi: 10.2174/1573400510666140320000928.  [DOI](https://doi.org/10.2174/1573400510666140320000928) | [PMC free article](/articles/PMC4155832/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25242908/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Current%20psychiatry%20reviews&title=Genetics%20of%20Opioid%20Dependence:%20A%20Review%20of%20the%20Genetic%20Contribution%20to%20Opioid%20Dependence&author=CJ%20Mistry&author=M%20Bawor&author=D%20Desai&author=DC%20Marsh&author=Z%20Samaan&volume=10&publication_year=2014&pages=156-167&pmid=25242908&doi=10.2174/1573400510666140320000928&)

3. Yang BZ, Han S, Kranzler HR, Palmer AA, Gelernter J. Sex-specific linkage scans in opioid dependence. American journal of medical genetics. Part B, Neuropsychiatric genetics: the official publication of the International Society of Psychiatric Genetics. 2017;174:261–268. doi: 10.1002/ajmg.b.32507.  [DOI](https://doi.org/10.1002/ajmg.b.32507) | [PMC free article](/articles/PMC5695218/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27762075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20medical%20genetics.%20Part%20B,%20Neuropsychiatric%20genetics:%20the%20official%20publication%20of%20the%20International%20Society%20of%20Psychiatric%20Genetics&title=Sex-specific%20linkage%20scans%20in%20opioid%20dependence&author=BZ%20Yang&author=S%20Han&author=HR%20Kranzler&author=AA%20Palmer&author=J%20Gelernter&volume=174&publication_year=2017&pages=261-268&pmid=27762075&doi=10.1002/ajmg.b.32507&)

4. Kendler KS, Maes HH, Sundquist K, Ohlsson H, Sundquist J. Genetic and family and community environmental effects on drug abuse in adolescence: a Swedish national twin and sibling study. The American journal of psychiatry. 2014;171:209–217. doi: 10.1176/appi.ajp.2013.12101300.  [DOI](https://doi.org/10.1176/appi.ajp.2013.12101300) | [PMC free article](/articles/PMC3927993/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24077613/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20American%20journal%20of%20psychiatry&title=Genetic%20and%20family%20and%20community%20environmental%20effects%20on%20drug%20abuse%20in%20adolescence:%20a%20Swedish%20national%20twin%20and%20sibling%20study&author=KS%20Kendler&author=HH%20Maes&author=K%20Sundquist&author=H%20Ohlsson&author=J%20Sundquist&volume=171&publication_year=2014&pages=209-217&pmid=24077613&doi=10.1176/appi.ajp.2013.12101300&)

5. Mague SD, et al. Mouse model of OPRM1 (A118G) polymorphism has sex-specific effects on drug-mediated behavior. Proceedings of the National Academy of Sciences of the United States of America. 2009;106:10847–10852. doi: 10.1073/pnas.0901800106.  [DOI](https://doi.org/10.1073/pnas.0901800106) | [PMC free article](/articles/PMC2705603/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19528658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&title=Mouse%20model%20of%20OPRM1%20(A118G)%20polymorphism%20has%20sex-specific%20effects%20on%20drug-mediated%20behavior&author=SD%20Mague&volume=106&publication_year=2009&pages=10847-10852&pmid=19528658&doi=10.1073/pnas.0901800106&)

6. Mogil JS, Chesler EJ, Wilson SG, Juraska JM, Sternberg WF. Sex differences in thermal nociception and morphine antinociception in rodents depend on genotype. Neuroscience and biobehavioral reviews. 2000;24:375–389. doi: 10.1016/S0149-7634(00)00015-4.  [DOI](https://doi.org/10.1016/S0149-7634(00)00015-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10781697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscience%20and%20biobehavioral%20reviews&title=Sex%20differences%20in%20thermal%20nociception%20and%20morphine%20antinociception%20in%20rodents%20depend%20on%20genotype&author=JS%20Mogil&author=EJ%20Chesler&author=SG%20Wilson&author=JM%20Juraska&author=WF%20Sternberg&volume=24&publication_year=2000&pages=375-389&pmid=10781697&doi=10.1016/S0149-7634(00)00015-4&)

7. Verzillo V, Madia PA, Liu NJ, Chakrabarti S, Gintzler AR. Mu-opioid receptor splice variants: sex-dependent regulation by chronic morphine. Journal of neurochemistry. 2014;130:790–796. doi: 10.1111/jnc.12768.  [DOI](https://doi.org/10.1111/jnc.12768) | [PMC free article](/articles/PMC4156545/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24848866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20neurochemistry&title=Mu-opioid%20receptor%20splice%20variants:%20sex-dependent%20regulation%20by%20chronic%20morphine&author=V%20Verzillo&author=PA%20Madia&author=NJ%20Liu&author=S%20Chakrabarti&author=AR%20Gintzler&volume=130&publication_year=2014&pages=790-796&pmid=24848866&doi=10.1111/jnc.12768&)

8. Schwantes-An TH, et al. Association of the OPRM1 Variant rs1799971 (A118G) with Non-Specific Liability to Substance Dependence in a Collaborative de novo Meta-Analysis of European-Ancestry Cohorts. Behavior genetics. 2016;46:151–169. doi: 10.1007/s10519-015-9737-3.  [DOI](https://doi.org/10.1007/s10519-015-9737-3) | [PMC free article](/articles/PMC4752855/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26392368/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behavior%20genetics&title=Association%20of%20the%20OPRM1%20Variant%20rs1799971%20(A118G)%20with%20Non-Specific%20Liability%20to%20Substance%20Dependence%20in%20a%20Collaborative%20de%20novo%20Meta-Analysis%20of%20European-Ancestry%20Cohorts&author=TH%20Schwantes-An&volume=46&publication_year=2016&pages=151-169&pmid=26392368&doi=10.1007/s10519-015-9737-3&)

9. Gelernter J, et al. Genome-wide association study of opioid dependence: multiple associations mapped to calcium and potassium pathways. Biological psychiatry. 2014;76:66–74. doi: 10.1016/j.biopsych.2013.08.034.  [DOI](https://doi.org/10.1016/j.biopsych.2013.08.034) | [PMC free article](/articles/PMC3992201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24143882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biological%20psychiatry&title=Genome-wide%20association%20study%20of%20opioid%20dependence:%20multiple%20associations%20mapped%20to%20calcium%20and%20potassium%20pathways&author=J%20Gelernter&volume=76&publication_year=2014&pages=66-74&pmid=24143882&doi=10.1016/j.biopsych.2013.08.034&)

10. Nelson EC, et al. Evidence of CNIH3 involvement in opioid dependence. Molecular psychiatry. 2016;21:608–614. doi: 10.1038/mp.2015.102.  [DOI](https://doi.org/10.1038/mp.2015.102) | [PMC free article](/articles/PMC4740268/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26239289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20psychiatry&title=Evidence%20of%20CNIH3%20involvement%20in%20opioid%20dependence&author=EC%20Nelson&volume=21&publication_year=2016&pages=608-614&pmid=26239289&doi=10.1038/mp.2015.102&)

11. Cheng Z, et al. Genome-wide Association Study Identifies a Regulatory Variant of RGMA Associated With Opioid Dependence in European Americans. Biological psychiatry. 2018 doi: 10.1016/j.biopsych.2017.12.016.  [DOI](https://doi.org/10.1016/j.biopsych.2017.12.016) | [PMC free article](/articles/PMC6041180/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29478698/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biological%20psychiatry&title=Genome-wide%20Association%20Study%20Identifies%20a%20Regulatory%20Variant%20of%20RGMA%20Associated%20With%20Opioid%20Dependence%20in%20European%20Americans&author=Z%20Cheng&publication_year=2018&pmid=29478698&doi=10.1016/j.biopsych.2017.12.016&)

12. Sherva R, et al. Genome-wide Association Study of Cannabis Dependence Severity, Novel Risk Variants, and Shared Genetic Risks. JAMA psychiatry. 2016;73:472–480. doi: 10.1001/jamapsychiatry.2016.0036.  [DOI](https://doi.org/10.1001/jamapsychiatry.2016.0036) | [PMC free article](/articles/PMC4974817/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27028160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20psychiatry&title=Genome-wide%20Association%20Study%20of%20Cannabis%20Dependence%20Severity,%20Novel%20Risk%20Variants,%20and%20Shared%20Genetic%20Risks&author=R%20Sherva&volume=73&publication_year=2016&pages=472-480&pmid=27028160&doi=10.1001/jamapsychiatry.2016.0036&)

13. Pierucci-Lagha A, et al. Reliability of DSM-IV diagnostic criteria using the semi-structured assessment for drug dependence and alcoholism (SSADDA) Drug and alcohol dependence. 2007;91:85–90. doi: 10.1016/j.drugalcdep.2007.04.014.  [DOI](https://doi.org/10.1016/j.drugalcdep.2007.04.014) | [PMC free article](/articles/PMC2039919/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17590536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20and%20alcohol%20dependence&title=Reliability%20of%20DSM-IV%20diagnostic%20criteria%20using%20the%20semi-structured%20assessment%20for%20drug%20dependence%20and%20alcoholism%20(SSADDA)&author=A%20Pierucci-Lagha&volume=91&publication_year=2007&pages=85-90&pmid=17590536&doi=10.1016/j.drugalcdep.2007.04.014&)

14. Price AL, et al. Principal components analysis corrects for stratification in genome-wide association studies. Nature genetics. 2006;38:904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

15. Auton A, et al. A global reference for human genetic variation. Nature. 2015;526:68–74. doi: 10.1038/nature15393.  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation&author=A%20Auton&volume=526&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

16. Purcell S, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. American journal of human genetics. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20human%20genetics&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

17. Das S, et al. Next-generation genotype imputation service and methods. Nature genetics. 2016;48:1284–1287. doi: 10.1038/ng.3656.  [DOI](https://doi.org/10.1038/ng.3656) | [PMC free article](/articles/PMC5157836/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27571263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Next-generation%20genotype%20imputation%20service%20and%20methods&author=S%20Das&volume=48&publication_year=2016&pages=1284-1287&pmid=27571263&doi=10.1038/ng.3656&)

18. Zhou X, Stephens M. Efficient multivariate linear mixed model algorithms for genome-wide association studies. Nature methods. 2014;11:407–409. doi: 10.1038/nmeth.2848.  [DOI](https://doi.org/10.1038/nmeth.2848) | [PMC free article](/articles/PMC4211878/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24531419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20methods&title=Efficient%20multivariate%20linear%20mixed%20model%20algorithms%20for%20genome-wide%20association%20studies&author=X%20Zhou&author=M%20Stephens&volume=11&publication_year=2014&pages=407-409&pmid=24531419&doi=10.1038/nmeth.2848&)

19. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics (Oxford, England) 2010;26:2190–2191. doi: 10.1093/bioinformatics/btq340.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics%20(Oxford,%20England)&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&author=CJ%20Willer&author=Y%20Li&author=GR%20Abecasis&volume=26&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

20. Okamura Y, et al. COXPRESdb in 2015: coexpression database for animal species by DNA-microarray and RNAseq-based expression data with multiple quality assessment systems. Nucleic Acids Res. 2015;43:D82–86. doi: 10.1093/nar/gku1163.  [DOI](https://doi.org/10.1093/nar/gku1163) | [PMC free article](/articles/PMC4383961/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25392420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=COXPRESdb%20in%202015:%20coexpression%20database%20for%20animal%20species%20by%20DNA-microarray%20and%20RNAseq-based%20expression%20data%20with%20multiple%20quality%20assessment%20systems&author=Y%20Okamura&volume=43&publication_year=2015&pages=D82-86&pmid=25392420&doi=10.1093/nar/gku1163&)

21. Wang J, Vasaikar S, Shi Z, Greer M, Zhang B. WebGestalt 2017: a more comprehensive, powerful, flexible and interactive gene set enrichment analysis toolkit. Nucleic Acids Res. 2017;45:W130–w137. doi: 10.1093/nar/gkx356.  [DOI](https://doi.org/10.1093/nar/gkx356) | [PMC free article](/articles/PMC5570149/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28472511/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=WebGestalt%202017:%20a%20more%20comprehensive,%20powerful,%20flexible%20and%20interactive%20gene%20set%20enrichment%20analysis%20toolkit&author=J%20Wang&author=S%20Vasaikar&author=Z%20Shi&author=M%20Greer&author=B%20Zhang&volume=45&publication_year=2017&pages=W130-w137&pmid=28472511&doi=10.1093/nar/gkx356&)

22. Ramasamy A, et al. Genetic variability in the regulation of gene expression in ten regions of the human brain. Nature neuroscience. 2014;17:1418–1428. doi: 10.1038/nn.3801.  [DOI](https://doi.org/10.1038/nn.3801) | [PMC free article](/articles/PMC4208299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25174004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20neuroscience&title=Genetic%20variability%20in%20the%20regulation%20of%20gene%20expression%20in%20ten%20regions%20of%20the%20human%20brain&author=A%20Ramasamy&volume=17&publication_year=2014&pages=1418-1428&pmid=25174004&doi=10.1038/nn.3801&)

23. GTEx-Consortium. Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Science (New York, N.Y.)348, 648–660, 10.1126/science.1262110 (2015).  [DOI](https://doi.org/10.1126/science.1262110) | [PMC free article](/articles/PMC4547484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25954001/)

24. Nikkila H, et al. Sequence similarities between a novel putative G protein-coupled receptor and Na+/Ca2+ exchangers define a cation binding domain. Molecular endocrinology (Baltimore, Md.) 2000;14:1351–1364. doi: 10.1210/mend.14.9.0511.  [DOI](https://doi.org/10.1210/mend.14.9.0511) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10976914/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20endocrinology%20(Baltimore,%20Md.)&title=Sequence%20similarities%20between%20a%20novel%20putative%20G%20protein-coupled%20receptor%20and%20Na+/Ca2+%20exchangers%20define%20a%20cation%20binding%20domain&author=H%20Nikkila&volume=14&publication_year=2000&pages=1351-1364&pmid=10976914&doi=10.1210/mend.14.9.0511&)

25. Langenhan T, Piao X, Monk KR. Adhesion G protein-coupled receptors in nervous system development and disease. Nature reviews. Neuroscience. 2016;17:550–561. doi: 10.1038/nrn.2016.86.  [DOI](https://doi.org/10.1038/nrn.2016.86) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27466150/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20reviews.%20Neuroscience&title=Adhesion%20G%20protein-coupled%20receptors%20in%20nervous%20system%20development%20and%20disease&author=T%20Langenhan&author=X%20Piao&author=KR%20Monk&volume=17&publication_year=2016&pages=550-561&pmid=27466150&doi=10.1038/nrn.2016.86&)

26. Thul Peter J., Åkesson Lovisa, Wiking Mikaela, Mahdessian Diana, Geladaki Aikaterini, Ait Blal Hammou, Alm Tove, Asplund Anna, Björk Lars, Breckels Lisa M., Bäckström Anna, Danielsson Frida, Fagerberg Linn, Fall Jenny, Gatto Laurent, Gnann Christian, Hober Sophia, Hjelmare Martin, Johansson Fredric, Lee Sunjae, Lindskog Cecilia, Mulder Jan, Mulvey Claire M., Nilsson Peter, Oksvold Per, Rockberg Johan, Schutten Rutger, Schwenk Jochen M., Sivertsson Åsa, Sjöstedt Evelina, Skogs Marie, Stadler Charlotte, Sullivan Devin P., Tegel Hanna, Winsnes Casper, Zhang Cheng, Zwahlen Martin, Mardinoglu Adil, Pontén Fredrik, von Feilitzen Kalle, Lilley Kathryn S., Uhlén Mathias, Lundberg Emma. A subcellular map of the human proteome. Science. 2017;356(6340):eaal3321. doi: 10.1126/science.aal3321.  [DOI](https://doi.org/10.1126/science.aal3321) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28495876/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20subcellular%20map%20of%20the%20human%20proteome&author=Peter%20J.%20Thul&author=Lovisa%20%C3%85kesson&author=Mikaela%20Wiking&author=Diana%20Mahdessian&author=Aikaterini%20Geladaki&volume=356&issue=6340&publication_year=2017&pages=eaal3321&pmid=28495876&doi=10.1126/science.aal3321&)

27. Brown TT, Wisniewski AB, Dobs AS. Gonadal and Adrenal Abnormalities in Drug Users: Cause or Consequence of Drug Use Behavior and Poor Health Outcomes. American journal of infectious diseases. 2006;2:130–135. doi: 10.3844/ajidsp.2006.130.135.  [DOI](https://doi.org/10.3844/ajidsp.2006.130.135) | [PMC free article](/articles/PMC1821355/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17364020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20infectious%20diseases&title=Gonadal%20and%20Adrenal%20Abnormalities%20in%20Drug%20Users:%20Cause%20or%20Consequence%20of%20Drug%20Use%20Behavior%20and%20Poor%20Health%20Outcomes&author=TT%20Brown&author=AB%20Wisniewski&author=AS%20Dobs&volume=2&publication_year=2006&pages=130-135&pmid=17364020&doi=10.3844/ajidsp.2006.130.135&)

28. Das G. Chronic heroin dependence leading to adrenal insufficiency. Case reports in endocrinology. 2014;2014:461816. doi: 10.1155/2014/461816.  [DOI](https://doi.org/10.1155/2014/461816) | [PMC free article](/articles/PMC4158160/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25221675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Case%20reports%20in%20endocrinology&title=Chronic%20heroin%20dependence%20leading%20to%20adrenal%20insufficiency&author=G%20Das&volume=2014&publication_year=2014&pages=461816&pmid=25221675&doi=10.1155/2014/461816&)

29. Rhodin A, Stridsberg M, Gordh T. Opioid endocrinopathy: a clinical problem in patients with chronic pain and long-term oral opioid treatment. The Clinical journal of pain. 2010;26:374–380. doi: 10.1097/AJP.0b013e3181d1059d.  [DOI](https://doi.org/10.1097/AJP.0b013e3181d1059d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20473043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Clinical%20journal%20of%20pain&title=Opioid%20endocrinopathy:%20a%20clinical%20problem%20in%20patients%20with%20chronic%20pain%20and%20long-term%20oral%20opioid%20treatment&author=A%20Rhodin&author=M%20Stridsberg&author=T%20Gordh&volume=26&publication_year=2010&pages=374-380&pmid=20473043&doi=10.1097/AJP.0b013e3181d1059d&)

30. Faro M, et al. Gender differences in chronic fatigue syndrome. Reumatologia clinica. 2016;12:72–77. doi: 10.1016/j.reuma.2015.05.007.  [DOI](https://doi.org/10.1016/j.reuma.2015.05.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26190206/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Reumatologia%20clinica&title=Gender%20differences%20in%20chronic%20fatigue%20syndrome&author=M%20Faro&volume=12&publication_year=2016&pages=72-77&pmid=26190206&doi=10.1016/j.reuma.2015.05.007&)

31. Germain DP. Pseudoxanthoma elasticum. Orphanet journal of rare diseases. 2017;12:85. doi: 10.1186/s13023-017-0639-8.  [DOI](https://doi.org/10.1186/s13023-017-0639-8) | [PMC free article](/articles/PMC5424392/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28486967/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Orphanet%20journal%20of%20rare%20diseases&title=Pseudoxanthoma%20elasticum&author=DP%20Germain&volume=12&publication_year=2017&pages=85&pmid=28486967&doi=10.1186/s13023-017-0639-8&)

32. Nakayama J, et al. A nonsense mutation of the MASS1 gene in a family with febrile and afebrile seizures. Annals of neurology. 2002;52:654–657. doi: 10.1002/ana.10347.  [DOI](https://doi.org/10.1002/ana.10347) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12402266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annals%20of%20neurology&title=A%20nonsense%20mutation%20of%20the%20MASS1%20gene%20in%20a%20family%20with%20febrile%20and%20afebrile%20seizures&author=J%20Nakayama&volume=52&publication_year=2002&pages=654-657&pmid=12402266&doi=10.1002/ana.10347&)

33. Coll M, et al. Targeted next-generation sequencing provides novel clues for associated epilepsy and cardiac conduction disorder/SUDEP. PloS one. 2017;12:e0189618. doi: 10.1371/journal.pone.0189618.  [DOI](https://doi.org/10.1371/journal.pone.0189618) | [PMC free article](/articles/PMC5736193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29261713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20one&title=Targeted%20next-generation%20sequencing%20provides%20novel%20clues%20for%20associated%20epilepsy%20and%20cardiac%20conduction%20disorder/SUDEP&author=M%20Coll&volume=12&publication_year=2017&pages=e0189618&pmid=29261713&doi=10.1371/journal.pone.0189618&)

34. Scheel H, Tomiuk S, Hofmann K. A common protein interaction domain links two recently identified epilepsy genes. Human molecular genetics. 2002;11:1757–1762. doi: 10.1093/hmg/11.15.1757.  [DOI](https://doi.org/10.1093/hmg/11.15.1757) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12095917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Human%20molecular%20genetics&title=A%20common%20protein%20interaction%20domain%20links%20two%20recently%20identified%20epilepsy%20genes&author=H%20Scheel&author=S%20Tomiuk&author=K%20Hofmann&volume=11&publication_year=2002&pages=1757-1762&pmid=12095917&doi=10.1093/hmg/11.15.1757&)

35. Myers Kenneth A., Nasioulas Steven, Boys Amber, McMahon Jacinta M., Slater Howard, Lockhart Paul, Sart Desirée du, Scheffer Ingrid E. ADGRV1 is implicated in myoclonic epilepsy. Epilepsia. 2017;59(2):381–388. doi: 10.1111/epi.13980.  [DOI](https://doi.org/10.1111/epi.13980) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29266188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=ADGRV1%20is%20implicated%20in%20myoclonic%20epilepsy&author=Kenneth%20A.%20Myers&author=Steven%20Nasioulas&author=Amber%20Boys&author=Jacinta%20M.%20McMahon&author=Howard%20Slater&volume=59&issue=2&publication_year=2017&pages=381-388&pmid=29266188&doi=10.1111/epi.13980&)

36. Fermin DR, et al. Sex and age dimorphism of myocardial gene expression in nonischemic human heart failure. Circulation. Cardiovascular genetics. 2008;1:117–125. doi: 10.1161/circgenetics.108.802652.  [DOI](https://doi.org/10.1161/circgenetics.108.802652) | [PMC free article](/articles/PMC3090727/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20031553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation.%20Cardiovascular%20genetics&title=Sex%20and%20age%20dimorphism%20of%20myocardial%20gene%20expression%20in%20nonischemic%20human%20heart%20failure&author=DR%20Fermin&volume=1&publication_year=2008&pages=117-125&pmid=20031553&doi=10.1161/circgenetics.108.802652&)

37. Adkins DE, et al. Genomewide pharmacogenomic study of metabolic side effects to antipsychotic drugs. Molecular psychiatry. 2011;16:321–332. doi: 10.1038/mp.2010.14.  [DOI](https://doi.org/10.1038/mp.2010.14) | [PMC free article](/articles/PMC2891163/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20195266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20psychiatry&title=Genomewide%20pharmacogenomic%20study%20of%20metabolic%20side%20effects%20to%20antipsychotic%20drugs&author=DE%20Adkins&volume=16&publication_year=2011&pages=321-332&pmid=20195266&doi=10.1038/mp.2010.14&)

38. Ilango A, et al. Similar roles of substantia nigra and ventral tegmental dopamine neurons in reward and aversion. The Journal of neuroscience: the official journal of the Society for Neuroscience. 2014;34:817–822. doi: 10.1523/jneurosci.1703-13.2014.  [DOI](https://doi.org/10.1523/jneurosci.1703-13.2014) | [PMC free article](/articles/PMC3891961/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24431440/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20neuroscience:%20the%20official%20journal%20of%20the%20Society%20for%20Neuroscience&title=Similar%20roles%20of%20substantia%20nigra%20and%20ventral%20tegmental%20dopamine%20neurons%20in%20reward%20and%20aversion&author=A%20Ilango&volume=34&publication_year=2014&pages=817-822&pmid=24431440&doi=10.1523/jneurosci.1703-13.2014&)

39. Schweighofer N, Lang EJ, Kawato M. Role of the olivo-cerebellar complex in motor learning and control. Frontiers in neural circuits. 2013;7:94. doi: 10.3389/fncir.2013.00094.  [DOI](https://doi.org/10.3389/fncir.2013.00094) | [PMC free article](/articles/PMC3664774/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23754983/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20neural%20circuits&title=Role%20of%20the%20olivo-cerebellar%20complex%20in%20motor%20learning%20and%20control&author=N%20Schweighofer&author=EJ%20Lang&author=M%20Kawato&volume=7&publication_year=2013&pages=94&pmid=23754983&doi=10.3389/fncir.2013.00094&)

40. Kandel, E. R., Schwartz, J. H. & Jessell, T. M. Principles of Neural Science (Fourth ed.). 324 (McGraw-Hill, 2000).

41. Packard MG, Knowlton BJ. Learning and memory functions of the Basal Ganglia. Annual review of neuroscience. 2002;25:563–593. doi: 10.1146/annurev.neuro.25.112701.142937.  [DOI](https://doi.org/10.1146/annurev.neuro.25.112701.142937) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12052921/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annual%20review%20of%20neuroscience&title=Learning%20and%20memory%20functions%20of%20the%20Basal%20Ganglia&author=MG%20Packard&author=BJ%20Knowlton&volume=25&publication_year=2002&pages=563-593&pmid=12052921&doi=10.1146/annurev.neuro.25.112701.142937&)

42. Beitner-Johnson D, Nestler EJ. Morphine and cocaine exert common chronic actions on tyrosine hydroxylase in dopaminergic brain reward regions. Journal of neurochemistry. 1991;57:344–347. doi: 10.1111/j.1471-4159.1991.tb02133.x.  [DOI](https://doi.org/10.1111/j.1471-4159.1991.tb02133.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1675665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20neurochemistry&title=Morphine%20and%20cocaine%20exert%20common%20chronic%20actions%20on%20tyrosine%20hydroxylase%20in%20dopaminergic%20brain%20reward%20regions&author=D%20Beitner-Johnson&author=EJ%20Nestler&volume=57&publication_year=1991&pages=344-347&pmid=1675665&doi=10.1111/j.1471-4159.1991.tb02133.x&)

43. Wang RY, et al. Impacts of GRIN3A, GRM6 and TPH2 genetic polymorphisms on quality of life in methadone maintenance therapy population. PloS one. 2018;13:e0201408. doi: 10.1371/journal.pone.0201408.  [DOI](https://doi.org/10.1371/journal.pone.0201408) | [PMC free article](/articles/PMC6066242/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30059533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20one&title=Impacts%20of%20GRIN3A,%20GRM6%20and%20TPH2%20genetic%20polymorphisms%20on%20quality%20of%20life%20in%20methadone%20maintenance%20therapy%20population&author=RY%20Wang&volume=13&publication_year=2018&pages=e0201408&pmid=30059533&doi=10.1371/journal.pone.0201408&)

44. Nielsen DA, Harding MJ, Hamon SC, Huang W, Kosten TR. Modifying the role of serotonergic 5-HTTLPR and TPH2 variants on disulfiram treatment of cocaine addiction: a preliminary study. Genes, brain, and behavior. 2012;11:1001–1008. doi: 10.1111/j.1601-183X.2012.00839.x.  [DOI](https://doi.org/10.1111/j.1601-183X.2012.00839.x) | [PMC free article](/articles/PMC3521860/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22925276/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes,%20brain,%20and%20behavior&title=Modifying%20the%20role%20of%20serotonergic%205-HTTLPR%20and%20TPH2%20variants%20on%20disulfiram%20treatment%20of%20cocaine%20addiction:%20a%20preliminary%20study&author=DA%20Nielsen&author=MJ%20Harding&author=SC%20Hamon&author=W%20Huang&author=TR%20Kosten&volume=11&publication_year=2012&pages=1001-1008&pmid=22925276&doi=10.1111/j.1601-183X.2012.00839.x&)

45. Ottenhof KW, Sild M, Levesque ML, Ruhe HG, Booij L. TPH2 polymorphisms across the spectrum of psychiatric morbidity: A systematic review and meta-analysis. Neuroscience and biobehavioral reviews. 2018;92:29–42. doi: 10.1016/j.neubiorev.2018.05.018.  [DOI](https://doi.org/10.1016/j.neubiorev.2018.05.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29775696/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroscience%20and%20biobehavioral%20reviews&title=TPH2%20polymorphisms%20across%20the%20spectrum%20of%20psychiatric%20morbidity:%20A%20systematic%20review%20and%20meta-analysis&author=KW%20Ottenhof&author=M%20Sild&author=ML%20Levesque&author=HG%20Ruhe&author=L%20Booij&volume=92&publication_year=2018&pages=29-42&pmid=29775696&doi=10.1016/j.neubiorev.2018.05.018&)

46. Korostynski M, Piechota M, Kaminska D, Solecki W, Przewlocki R. Morphine effects on striatal transcriptome in mice. Genome biology. 2007;8:R128. doi: 10.1186/gb-2007-8-6-r128.  [DOI](https://doi.org/10.1186/gb-2007-8-6-r128) | [PMC free article](/articles/PMC2394777/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17598886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20biology&title=Morphine%20effects%20on%20striatal%20transcriptome%20in%20mice&author=M%20Korostynski&author=M%20Piechota&author=D%20Kaminska&author=W%20Solecki&author=R%20Przewlocki&volume=8&publication_year=2007&pages=R128&pmid=17598886&doi=10.1186/gb-2007-8-6-r128&)

47. Bustamante CD, Burchard EG, De la Vega FM. Genomics for the world. Nature. 2011;475:163–165. doi: 10.1038/475163a.  [DOI](https://doi.org/10.1038/475163a) | [PMC free article](/articles/PMC3708540/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21753830/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genomics%20for%20the%20world&author=CD%20Bustamante&author=EG%20Burchard&author=FM%20De%20la%20Vega&volume=475&publication_year=2011&pages=163-165&pmid=21753830&doi=10.1038/475163a&)

48. Kim MS, Patel KP, Teng AK, Berens AJ, Lachance J. Genetic disease risks can be misestimated across global populations. Genome biology. 2018;19:179. doi: 10.1186/s13059-018-1561-7.  [DOI](https://doi.org/10.1186/s13059-018-1561-7) | [PMC free article](/articles/PMC6234640/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30424772/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20biology&title=Genetic%20disease%20risks%20can%20be%20misestimated%20across%20global%20populations&author=MS%20Kim&author=KP%20Patel&author=AK%20Teng&author=AJ%20Berens&author=J%20Lachance&volume=19&publication_year=2018&pages=179&pmid=30424772&doi=10.1186/s13059-018-1561-7&)
