# Impact of Interferon Lambda 4 Genotype on Interferon‐Stimulated Gene Expression During Direct‐Acting Antiviral Therapy for Hepatitis C

## Metadata
**Authors:** Narayan Ramamurthy, Emanuele Marchi, M Azim Ansari, Vincent Pedergnana, Angela Mclean, Emma Hudson, STOP HCV consortium, Rory Bowden, Chris CA Spencer, Eleanor Barnes, Paul Klenerman
**Journal:** Hepatology (Baltimore, Md.)
**Date:** 2018 Sep 15
**DOI:** [10.1002/hep.29877](https://doi.org/10.1002/hep.29877)
**PMID:** 29534310
**PMCID:** PMC6207923
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6207923/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6207923/pdf/HEP-68-859.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6207923/pdf/HEP-68-859.pdf)

## Abstract

New directly acting antivirals (DAAs) provide very high cure rates in most patients infected by hepatitis C virus (HCV). However, some patient groups have been relatively harder to treat, including those with cirrhosis or infected with HCV genotype 3. In the recent BOSON trial, genotype 3, patients with cirrhosis receiving a 16‐week course of sofosbuvir and ribavirin had a sustained virological response (SVR) rate of around 50%. In patients with cirrhosis, interferon lambda 4 (IFNL4) CC genotype was significantly associated with SVR. This genotype was also associated with a lower interferon‐stimulated gene (ISG) signature in peripheral blood and in liver at baseline. Unexpectedly, patients with the CC genotype showed a dynamic increase in ISG expression between weeks 4 and 16 of DAA therapy, whereas the reverse was true for non‐CC patients. Conclusion: These data provide an important dynamic link between host genotype and phenotype in HCV therapy also potentially relevant to naturally acquired infection. (Hepatology 2018; 00:000‐000).

## Materials and Methods

### STUDY DESIGN, PATIENTS, AND ETHICS

This study was part of the study described earlier.[10](#hep29877-bib-0010), [22](#hep29877-bib-0022) PAXgene and liver biopsy samples came from patients enrolled in the BOSON study. The BOSON study is a phase 3 randomized open‐label trial to determine the efficacy and safety of sofosbuvir with and without Peg‐IFN‐alfa, in treatment‐experienced patients with cirrhosis and HCV genotype 2 infection and treatment‐naïve or ‐experienced patients with HCV genotype 3 infection.[10](#hep29877-bib-0010) We focused analyses here on patients given DAA therapy without IFN (arms 1 and 2). All patients provided written informed consent before undertaking any study‐related procedures. The study protocol was approved by each institution's review board or ethics committee before study initiation. The study was conducted in accord with the International Conference on Harmonization Good Clinical Practice Guidelines and the Declaration of Helsinki. The study reported here is not a clinical trial, but is based on the analysis of patients from a clinical trial (registration number: 510 [NCT01962441](https://clinicaltrials.gov/ct2/show/NCT01962441)).

### MICROARRAY ANALYSIS

Globin‐depleted total RNA (500ng) was analyzed using Illumina Human HT12v4.0 Expression Beadchips according to the manufacturer's instructions (v1.9.0; Illumina, San Diego, CA). Gene expression data were checked and exported using Illumina's GenomeStudio (V2011.1) software.

### HCV VIREMIA, IFNL4 GENOTYPING, RNA EXTRACTION, AND PCR

See the Supporting Information for further details.

### BIOINFORMATIC ANALYSIS

Illumina bead chip output files were processed and analyzed using the R package ([https://www.r-project.org](https://www.r-project.org)),[25](#hep29877-bib-0025) wherein samples were normalized and gene expression levels calculated to determine statistical significance. Differential gene expression between different groups was assessed by generating relevant contrasts corresponding to the possible comparisons. Statistical testing was performed using the Linear Models for Microarray Analysis (LIMMA) package ([https://bioconductor.org/packages/release/bioc/html/limma.html](https://bioconductor.org/packages/release/bioc/html/limma.html)).[26](#hep29877-bib-0026) Raw *P* values were corrected for multiple testing using the false discovery rate controlling procedure of Benjamini and Hochberg.[27](#hep29877-bib-0027) Following this correction, adjusted *P* values <0.01 were considered significant. Pathway over‐representation analysis and plotting was conducted using Reactome PA.[28](#hep29877-bib-0028) We also used the CAMERA package[29](#hep29877-bib-0029) to assess the enrichment of Reactome pathways. Gene set enrichment analysis (GSEA)[30](#hep29877-bib-0030) and relative enrichment plots were performed using *HTSanalyzeR* package[31](#hep29877-bib-0031) from within the R environment.

## Results

### IFNL4 CC GENOTYPE IS ASSOCIATED WITH SVR IN GENOTYPE 3 PATIENTS WITH CIRRHOSIS UNDERGOING TREATMENT WITH SOFOSBUVIR AND RIBAVIRIN

In the BOSON study, patients were randomised to receive sofosbuvir with ribavirin for 16 or 24 weeks or for 12 weeks with accompanying Peg‐IFN.[10](#hep29877-bib-0010) To understand the impact of *IFNL4* genotype on attaining SVR, we estimated the effect size (odds ratio; OR) of *IFNL4* CC versus non‐CC on achieving SVR stratified by the cirrhosis status in the two IFN‐free treatment arms. In patients with cirrhosis, those with IFNL4 CC genotype were significantly more likely to achieve SVR (16‐week arm, *P* value = 0.0015; OR = 5.50; CC SVR rate, 76% [26 of 34]; non‐CC SVR rate, 37% [13 of 35]; 24‐week arm, *P* value = 0.009; OR = 11.38; CC SVR rate, 96% [30 of 31]; non‐CC SVR rate, 73% [29 of 40]). However, this was not the case in patients without cirrhosis (16‐week arm: *P* value = 1; OR = 1.1; CC SVR rate, 82% [32 of 39]; non‐CC SVR rate, 81% [62 of 77]; 24‐week arm, *P* value = 0.42; OR = 1.75; CC SVR rate, 90% [36 of 40; non‐CC SVR rate, 84% [67 of 80]; Fig. [1](#hep29877-fig-0001)A).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f119/6207923/583df4c2686c/HEP-68-859-g001.jpg)

IFNL4 genotypes and clinical outcomes in the BOSON study. (A) OR graph showing the impact of IFNL4 genotype on SVR during therapy in chronic HCV patients from the BOSON study (DAA only arms). (B) Summary data from gene expression studies of peripheral blood from patients with cirrhosis taken at baseline. Venn diagram shows differentially regulated genes in IFNL4 CC and non‐CC patients (adjusted P value <0.05) when compared to common referenced healthy control group (n = 8).

### PATIENTS WITH CIRRHOSIS WITH NON‐CC GENOTYPE HAVE GREATER EXPRESSION OF ISGs IN WHOLE BLOOD

We next addressed whether the host genetic polymorphisms associated with *IFNL4* genotypes (CC vs. non‐CC) were associated with host responses in whole blood using microarray analysis of transcription. We used a healthy control group of 8 individuals as a common reference point and measured up‐ and down‐regulated gene expression in the *IFNL4* CC and non‐CC groups (Fig. [1](#hep29877-fig-0001)B). We detected genes, which were exclusively up‐ and down‐regulated in each group (CC or non‐CC) as well as a set of genes expressed in common (threshold for inclusion, *P* < 0.05). Supporting Table [S1A,B](#hep29877-sup-0001) gives a full list of genes that were up‐regulated in each of the groups of patients.

To test (in an unsupervised manner) for over‐representation of certain categories of genes among differentially expressed genes, we checked for statistical enrichment of genes in curated reactome pathways. Among the genes up‐regulated in the non‐CC patients versus healthy controls, we observe a clear and significant enrichment of genes in the IFN α/β and γ signaling pathways (Fig. [2](#hep29877-fig-0002)A‐D; Supporting Table [S1C,D](#hep29877-sup-0001)). The volcano plots (Fig. [2](#hep29877-fig-0002)A,B) highlight the general differences in significance and variation in gene expression in non‐CC samples when compared to healthy controls or to CC samples.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f119/6207923/9d8445d5080b/HEP-68-859-g002.jpg)

IFNL4 genotypes and gene expression in peripheral blood during chronic HCV infection. (A,B) Volcano plot showing change in overall gene expression in peripheral blood between non‐CC patients and healthy controls (A) and direct comparison between non‐CC versus CC patients (B) Log2 fold change is plotted along the X‐axis and the P value along the y‐axis. The dotted red line shows the threshold of significance (P < 0.05). Genes labeled in the figure are those that were validated by real‐time PCR. (C,D) GSEA analysis showing enrichment of IFNα/β gene set in non‐CC versus control (C) and a similar comparison of genes comparing non‐CC versus CC patients. (E) IFNL4 genotypes and pathway analyses during chronic HCV infection: over‐representation analysis of reactome pathways on the list of up‐regulated genes between non‐CC versus CC patient groups. The Y‐axis shows the top six pathways (significance is color coded) X‐axis shows the proportion of gene in each pathway compared to the total list of genes. The size of the circles represent the gene counts in the pathway.

The labeled genes in both volcano plots are a selection of transcripts validated by quantitative real‐time PCR (qPCR) known to be representative of an IFN response. Up‐regulation of IFN genes is observable and significant even in CC samples versus healthy controls (data not shown), but is evidently higher in non‐CC samples (Fig. [2](#hep29877-fig-0002)E).

Over‐representation of cytokine signaling, IFN signaling, and, specifically, IFN α/β signaling pathways in non‐CC patients was further confirmed by testing enrichments of the reactome gene sets in a direct comparison between non‐CC and CC samples (Fig. [2](#hep29877-fig-0002)B); in this “tug‐of‐war” between the two genotypes, expression levels were ranked between the two groups of patients and the analysis clearly showed an over‐representation of cytokine signaling, general IFN signaling and, specifically, IFN α/β signaling. Figure [2](#hep29877-fig-0002)E shows the most significantly represented reactome pathways analyzed using Reactome PA,[28](#hep29877-bib-0028) which takes into account the proportion of genes in each pathway that contribute to the total list of the top ranking genes in the list (*P* < 0.05; lfc > 0). Using an alternate approach to assess the enrichment of reactome pathways (Camera, package),[29](#hep29877-bib-0029) we performed a gene set enrichment analysis (GSEA). IFN α/β signalling ranks first (*P* value = 1.47e‐09) among a list of categories populated with immunologically relevant pathways (Fig. [2](#hep29877-fig-0002)B,D). These data are consistent with previous studies showing enhanced ISG expression in the nonfavorable genotype, although past data have had variable results with respect to genotype 3 and also measurements in peripheral blood.[12](#hep29877-bib-0012), [32](#hep29877-bib-0032)

### PATIENTS WITH CIRRHOSIS PATIENTS WITH NON‐CC GENOTYPE HAVE GREATER EXPRESSION OF IFN‐STIMULATED GENES IN THE LIVER

We also profiled gene expression in a subset of liver biopsy samples (13 biopsies) taken before treatment (baseline) identifying an enrichment of IFN α/β signaling pathways in non‐CC versus CC patients (Fig. [3](#hep29877-fig-0003)A,B). RNA sequencing (RNAseq) confirmed enhanced expression of ISGs in the non‐CC population (Fig. [4](#hep29877-fig-0004)A).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f119/6207923/1ddd64641e60/HEP-68-859-g003.jpg)

IFNL4 genotypes and gene expression in liver of HCV patients. (A). Volcano plot showing differentially modulated genes in liver by microarray analysis comparing non‐CC versus CC patients. X‐axis shows log2 fold change and Y‐axis the –log 10 P value. The dotted red line shows the threshold of significance (P < 0.05). Genes illustrated in the figure are those that were found in the blood in Fig. 1C/D when comparing the non‐CC patients versus healthy control and the non‐CC versus CC groups. (B) Reactome pathways enrichment analysis on differentially up‐regulated list of genes obtained by microarray analysis of liver tissue between non‐CC versus CC patients. The X‐axis shows the gene counts in each pathway and the Y‐axis shows the eight most significant pathways sorted by significance. Abbreviations: MDA5, melanoma differentiation‐associated gene 5; RIG‐I, retinoic acid‐inducible gene I.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f119/6207923/8448112a5091/HEP-68-859-g004.jpg)

IFNL4 genotypes and RNAseq analysis in liver of HCV patients. (A) Reactome pathways enrichment analysis on differentially up‐regulated list of genes obtained by RNAseq analysis between non‐CC versus CC patients. Criteria for inclusion were a minimum of 1 lfc (log 2 fold change) in expression and P value <0.05. The size of circles reflects the gene count in the pathway, the X‐axis represent the gene ratio (N genes in pathway/n genes in the entire list of genes), and the color of the circles reflects the adjusted P value. (B) IFNL4 genotypes and cytoband analysis in liver of HCV patients: expression graph of chromosomally sorted genes in cytoband 19q13.2. Lines across the graph represent average gene expression in each IFNL4 genotype (TT in red, black dotted lines CT, and black solid line represent expression in CC groups). IFNL3/4 gene is not shown because the genes included in the graph are those that have an average gene expression of log2 cpm >2. (C,D) GSEA showing enrichment of IFNα and IFNL Huh‐7 signature gene (uniquely up‐regulated genes when Huh7 cells are stimulated by IFNα or IFNL3) on non‐CC versus CC gene list in liver. Abbreviations: MDA5, melanoma differentiation‐associated gene 5; RIG‐I, retinoic acid‐inducible gene I.

When genes between non‐CC and CC were compared, pathway analysis (Fig. [3](#hep29877-fig-0003)B) showed IFN signalling among the most significantly enriched pathways. To understand whether the enrichment more closely resembles IFNλ3‐driven or IFNα‐driven signaling pathways, we performed a GSEA on these genes and compared them with a preranked list obtained from our previous study on hepatocytes (Huh7) stimulated with IFNα (100 ng/mL) or IFNλ3 (100 ng/mL) for 72 hours (GEO accession: [GSE89610](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE89610)).[33](#hep29877-bib-0033) GSEA analysis showed that genes from both IFNα and IFNλ3 stimulated lists were up‐regulated (Fig. [4](#hep29877-fig-0004)C,D and Supporting Fig. [S1](#hep29877-sup-0001)).

In case the effect of *IFNL4* genotype is mediated by a nearby highly expressed gene, we investigated gene expression in the *IFNL4* region (cytoband 19q13.2) in liver biopsies of patients of CC and non‐CC *IFNL4* genotypes. Expression of *SPINT2* (serine peptidase inhibitor, Kunitz type 2) was most significantly associated: TT carriers had the highest expression followed by CT and then CC genotype patients (Fig. [4](#hep29877-fig-0004)B), but the association was observed only in the liver and not in whole blood. No significant association between expression of *IFNL3* or *IFNL4* genes was observed.

### qPCR VALIDATION SHOWS SIGNIFICANT INCREASE IN ISG LEVELS IN NON‐CC PATIENTS

We undertook qPCR to validate the enhanced expression of several classical ISGs in non‐CC patients with cirrhosis at baseline (Supporting Table [S1B](#hep29877-sup-0001); Fig. [5](#hep29877-fig-0005)), expanding the cohort by 17 patients to 45 patient samples in total. In six of eight genes tested, we confirmed a strong association between *IFNL4* status and gene expression levels in peripheral blood. The effect was observed on classical ISGs—*IFIT1*, *MX1*, *ISG15*, *IFIT1*, *IFI44L*, and *HERC5*—where the non‐CC genotype was associated with elevated gene expression, that is, significantly higher than the favorable CC genotype, as well as compared to healthy controls. Of note, these differences in peripheral blood ISG responses were not observed in patients without cirrhosis (Supporting Fig. [S2A](#hep29877-sup-0001)). Previous groups have shown enhanced expression of ISG in the non‐CC group in the liver,[12](#hep29877-bib-0012), [34](#hep29877-bib-0034), [35](#hep29877-bib-0035) but not consistently in blood, and the contrast with our result seems potentially to be attributed to a dilution of the effect when both patients with cirrhosis and patients without cirrhosis are studied.

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f119/6207923/a08a97f78f38/HEP-68-859-g005.jpg)

qPCR analysis of peripheral blood gene expression at baseline according to IFNL4 genotype. A set of genes was quantified by PCR from RNA derived from peripheral blood using the relative quantification method with genes normalized against a housekeeping gene (GAPDH). Results from eight ISG gene expressions are shown in a total of 45 patients from the same cohort, of which 27 were CC and 18 patients were non‐CC genotype. Results from qPCR analysis were fed into GraphPad Prism (7.0; GraphPad Software Inc., La Jolla, CA) to generate the graphs. P values were generated using the statistical software within Prism using log values of fold changes and SEM is shown. Abbreviation: GAPDH, glyceraldehyde 3‐phosphate dehydrogenase.

### DYNAMICS OF ISG EXPRESSION DURING TREATMENT DIFFER BETWEEN IFNL4 CC AND NON‐CC GENOTYPE PATIENTS

We next addressed to what extent gene expression changed over time during therapy, in relation to *IFNL4* genotypes. In both groups (CC and non‐CC), virus was rapidly suppressed and viral RNA PCRs became negative until end of treatment in most cases (Supporting Fig. [S2B](#hep29877-sup-0001)). We first analyzed a subset of genes (genes analyzed in Fig. [5](#hep29877-fig-0005).) pretherapy, at 4 weeks, 16 weeks, and at the end of study (24 weeks after the end of treatment). Here, we observed an unexpected phenomenon—and a difference between the groups. In the CC group, gene expression was observed to increase toward end of treatment (rising between 4 and 16 weeks). However, the same genes in the non‐CC group do not show the similar increase in expression levels, rather falling in expression over time during therapy (Fig. [6](#hep29877-fig-0006)A‐C and Supporting Fig. [S3A‐C](#hep29877-sup-0001)).

### Figure 6.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f119/6207923/41bcd3185ef2/HEP-68-859-g006.jpg)

Dynamic changes in peripheral blood gene expression during therapy. (A) Expression of an average of a subset of genes (RSAD2, IFIT1, IFIT3, IFI44L, MX1, CXCL10, ISG15, and HERC5) over time, that is, BL, 4 weeks, 16 weeks, and at the end of study in CC versus non‐CC patient group. Similar results were obtained when all genes of the IFNα/β pathway were analyzed. (B) Gene expression data on the average of 27 genes shown in Table 1 over time as described above. (C) Representative example of gene expression in gene IFIT3 showing the increased expression in the CC patient group at week 16 of treatment compared to week 4. Significance was determined using a paired t test. Abbreviations: BL, baseline; EOS, end of study.

To understand this increase in gene expression between 4 and 16 weeks in CC patients, and to analyze what sets of genes were up‐regulated, we analyzed the up‐regulation of all genes over this time period in the CC group. Supporting Fig. [S4](#hep29877-sup-0001) shows the Venn diagram of differentially regulated genesin IFNL4 CC genotype patients at 4 and 16 weeks of treatment in whole blood when compared to healthy controls, revealing a new set of up‐regulated genes at the later time point. The reactome pathways enrichment analysis on up‐regulated genes between 4 and 16 weeks showed that the genes involved in the IFN α/β gene pathway in the CC group at 16 weeks were among the most represented categories (Fig. [7](#hep29877-fig-0007)A; Supporting Table [S2A,B](#hep29877-sup-0001)).

### Figure 7.

![Figure 7](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f119/6207923/b41a60ac7aa0/HEP-68-859-g007.jpg)

(A) Dynamic changes in reactome pathways during therapy. Over‐representation analysis of reactome pathway genes in CC patients at 16 versus 4 weeks. The up‐regulated list of genes was generated using paired samples at 16 and 4 weeks for each patient and analyzed using paired t test. The Y‐axis shows the eight most significant over‐represented reactome pathways (significance color coded) and X‐axis shows gene counts in each pathway. (B,C) Up‐regulation of IFN‐regulated genes according to IFNL4 genotypes: GSEA analysis showing enrichment of the IFNα/β gene set between 16 versus 4 weeks in CC and non‐CC patient blood samples. The dotted vertical line shows the enrichment “core.” Genes were enriched for the IFNα/β pathway in the CC patient group at 16 weeks. However, in the non‐CC patient group, such an enrichment was not observed.

GSEA analysis of genes of paired samples of the IFN α/β pathway show that patients with the IFNL4 CC genotype show up‐regulation of genes of this host response pathway at 16 weeks compared to 4 weeks (Fig. [7](#hep29877-fig-0007)A,B). Importantly, equivalent analysis in the non‐CC group show significant up‐regulation of these genes at 4 weeks rather than at 16 weeks, that is, a down‐regulation over time (Fig. [6](#hep29877-fig-0006)B). A contextualized differential expression analysis exclusively on only IFN genes was performed to highlight what genes are the most significant in the context of IFN signaling pathways (Table [1](#hep29877-tbl-0001)). In this analysis, we also note that there is significant up‐regulation of genes in the intrinsic apoptosis pathway.

### Table 1.

| ID | logFC | AveExpr | t | P.Value | adj.P.Val | B |
| --- | --- | --- | --- | --- | --- | --- |
| IFITM3 | 0.54369485 | 12.0023006 | 4.32021235 | 0.00011602 | 0.00498869 | 1.0622434 |
| OASL | 0.39797067 | 8.55867409 | 3.74314074 | 0.000629 | 0.00956154 | −0.526625 |
| IFIT1 | 0.54431309 | 9.49978316 | 3.72259827 | 0.00066708 | 0.00956154 | −0.5815558 |
| STAT1 | 0.31810156 | 12.2367207 | 3.27827781 | 0.00230924 | 0.02482437 | −1.734864 |
| IFIT2 | 0.30959571 | 10.6864439 | 2.93766556 | 0.00571849 | 0.04841688 | −2.5658952 |
| MX1 | 0.30156683 | 11.3422034 | 2.87312169 | 0.00675584 | 0.04841688 | −2.717316 |
| IRF8 | −0.2415412 | 9.74662371 | −2.785396 | 0.00844986 | 0.05190631 | −2.9197644 |
| HLA‐C | 0.35011366 | 10.5071659 | 2.66580015 | 0.01140094 | 0.06128004 | −3.1892545 |
| OAS3 | 0.32389362 | 8.4497729 | 2.5733493 | 0.01430667 | 0.06835411 | −3.3921917 |
| IRF3 | −0.2630716 | 8.94593252 | −2.4067598 | 0.02131318 | 0.09164667 | −3.7453383 |
| IFIT3 | 0.26489315 | 9.57162048 | 2.23364813 | 0.03176682 | 0.12417939 | −4.0941835 |
| USP18 | 0.20892888 | 5.24041669 | 1.97128898 | 0.05636246 | 0.20196548 | −4.5848292 |
| ADAR | 0.15686838 | 11.8777284 | 1.64870926 | 0.10784976 | 0.35673383 | −5.1195189 |
| ISG15 | 0.21228863 | 9.50434984 | 1.60359579 | 0.1174832 | 0.36084126 | −5.1878628 |
| XAF1 | 0.15720552 | 9.46221489 | 1.54965909 | 0.12991208 | 0.37241463 | −5.2674177 |
| PTPN6 | −0.1880362 | 10.8742509 | −1.4995268 | 0.14239785 | 0.38062752 | −5.3392283 |
| IRF5 | −0.1719974 | 6.95892964 | −1.4689217 | 0.15048065 | 0.38062752 | −5.3820457 |
| SOCS3 | 0.14342655 | 5.70325015 | 1.41310533 | 0.16615553 | 0.39025422 | −5.458117 |
| IRF7 | 0.14788072 | 9.83850934 | 1.39189293 | 0.17243791 | 0.39025422 | −5.4863376 |
| OAS1 | 0.13757945 | 8.02691589 | 1.32924865 | 0.1920728 | 0.39463516 | −5.5674377 |
| GBP2 | 0.13532043 | 11.5723117 | 1.32724256 | 0.1927288 | 0.39463516 | −5.5699791 |
| IFNAR2 | 0.14983542 | 9.8565919 | 1.27620484 | 0.21000064 | 0.4104558 | −5.6334638 |
| STAT2 | 0.10353448 | 10.6768576 | 1.22034482 | 0.23021475 | 0.43040149 | −5.7003404 |
| IFI6 | 0.13818007 | 10.8491464 | 1.06586612 | 0.29353312 | 0.52591351 | −5.8708442 |
| JAK1 | 0.11415208 | 8.9418609 | 1.01099816 | 0.31871926 | 0.54819712 | −5.9262115 |
| IFITM1 | 0.10006014 | 11.9218544 | 0.96719248 | 0.33985766 | 0.56207229 | −5.9684321 |
| EGR1 | 0.09783 | 7.13441316 | 0.79499262 | 0.43179315 | 0.67667142 | −6.1170782 |
| IP6K2 | −0.0912588 | 6.29982658 | −0.7797106 | 0.44062325 | 0.67667142 | −6.1289201 |
| ISG20 | −0.0653383 | 11.5325772 | −0.7494629 | 0.45841531 | 0.67971925 | −6.1517034 |
| HLA‐A | −0.0626801 | 12.3825849 | −0.6908859 | 0.49403856 | 0.68575951 | −6.1933378 |
| PTPN1 | −0.0654316 | 10.2460777 | −0.6761645 | 0.50322832 | 0.68575951 | −6.2032832 |
| IRF4 | 0.07786415 | 6.50909709 | 0.64195778 | 0.52494027 | 0.68575951 | −6.2255858 |
| HLA‐G | 0.06172342 | 10.2813706 | 0.63987139 | 0.52628055 | 0.68575951 | −6.2269095 |
| PSMB8 | 0.04403661 | 10.5311175 | 0.44826974 | 0.65662689 | 0.83043989 | −6.3304112 |
| IFI27 | −0.0774203 | 10.1136656 | −0.4171174 | 0.67905287 | 0.83426495 | −6.3438388 |
| OAS2 | −0.0359996 | 9.79019653 | −0.3883369 | 0.70003847 | 0.83615706 | −6.3553932 |
| IFI35 | 0.02987224 | 9.50135665 | 0.31427078 | 0.75511728 | 0.87756873 | −6.3813588 |
| MX2 | −0.0350638 | 10.1767308 | −0.2755001 | 0.78449668 | 0.88771992 | −6.39278 |
| SOCS1 | 0.02139765 | 6.50760493 | 0.20060399 | 0.84212968 | 0.92850196 | −6.4106056 |
| IRF6 | −0.0125211 | 4.88767786 | −0.1426073 | 0.88739013 | 0.93684233 | −6.420564 |
| IRF2 | −0.0163081 | 6.95712133 | −0.1351153 | 0.89326826 | 0.93684233 | −6.4216053 |
| HLA‐F | −0.0030076 | 10.4839197 | −0.0284221 | 0.97748143 | 0.98520595 | −6.4303397 |
| IRF1 | −0.0030119 | 11.5593311 | −0.018671 | 0.98520595 | 0.98520595 | −6.4305696 |

Table 1 Caption: Differential Expression Analysis Performed Exclusively on Genes in the IFNα/β Signaling Pathway (43 Features) in CC Patients Up‐Regulated at 16 Weeks When Compared to Those at 4 Weeks

### Discussion

In this study, we analyzed the host response in blood and liver of patients suffering from chronic HCV genotype 3 infection, recruited in the BOSON trial. In these analyses, we selected the group that had the lowest response rate to the DAA therapy (sofosbuvir and ribavirin), with an SVR of around 50%. It is in this group where a host response might have the largest demonstrable impact.

Previous analyses of this study have revealed that both *IFNL4* and cirrhosis have an impact on outcome in a multivariate model, together with male sex and treatment arm, baseline viral load, and baseline alanine aminotransferase.[10](#hep29877-bib-0010) In the current study, the interaction between cirrhosis and *IFNL4* status is clearly demonstrated. Interestingly, in the absence of cirrhosis, we could not detect a significant impact of IFNL4 genotypes in the analyses performed (DAA therapy arms only). This feature suggests that a further host factor modifies the impact of a genetic risk. One simple explanation is that the effect of the *IFNL4* locus gene variant is only detectable in settings where the treatment efficacy is low. However, the SVR at 12 weeks (SVR12) rate in the 24‐week DAA only arm among patients with cirrhosis (where a clear *IFNL4* effect was observed) was 79% in this study, comparable to the 80% response rate in the 16‐week arm among patients without cirrhosis (where no *IFNL4* gene effect was observed). Furthermore, previous multivariate analyses did not reveal an independent effect of cirrhosis in the 24 week treatment group.[10](#hep29877-bib-0010) An alternative explanation therefore is that there is a true interaction between these biological variables, and the protective mechanisms induced by a favorable genotype have additional relevance in viral control in cirrhosis.

Although we did not address the mechanism behind this interaction, we speculate that an additional critical role for host responses is observed in cirrhosis because of the disrupted microanatomy and immunological milieu of the liver. For example, functionality of the dominant MAIT cell population in the liver is impaired in cirrhosis.[36](#hep29877-bib-0036) This cell type can secrete potentially potent IFNγ following stimulation by IFNα.[37](#hep29877-bib-0037) Rescue of responses in these resident populations through a boost of local IFNα may lead to amplified intrahepatic effectors crucial in clearance of residual infection. Alternatively, attributed to disruption of tissue continuity, higher local concentrations of IFNs are required for clearance of all infected hepatocyte niches.

We were able to clearly demonstrate differences in whole‐blood ISG expression between *IFNL4* CC and non‐CC patients. The increase in baseline expression of genes in the *IFNL4* non‐CC genotype in chronic HCV patients observed (Supporting Fig. [S1A,B](#hep29877-sup-0001)) has been previously shown[20](#hep29877-bib-0020), [38](#hep29877-bib-0038) with increase in ISG at baseline in non‐SVR, but this difference in ISG expression was significantly detected in liver and not in whole‐blood RNA expression.[20](#hep29877-bib-0020), [38](#hep29877-bib-0038), [39](#hep29877-bib-0039) In a study of HCV genotype 1 patients with similar DAA for 24 weeks,[40](#hep29877-bib-0040) liver biopsies collected pretreatment and at end of treatment showed down‐regulation of ISGs, although (consistent with our data) those with an SVR showed higher end‐of‐treatment intrahepatic ISGs than those without. There are a number of differences in our study—first of which is the genotype (focused on genotype 3). Most interestingly, however, we focused additionally on patients with cirrhosis. When we analyzed specific genes by qPCR in patients without cirrhosis at baseline, we observed no significant difference from controls or between *IFNL4* CC and non‐CC groups. These data suggest that the presence of cirrhosis in CHC patients exposes a systemic ISG signal, which is effectively contained within the liver in patients with milder fibrosis. This may reflect the magnitude of the response or potentially a difference in the physiology of the cirrhotic liver, which primes or amplifies systemic responses. It should be noted that the patients recruited to the BOSON study had stable, well‐compensated cirrhosis.

The findings above represent an extrapolation of data that is already available regarding the impact of host genotype on response to HCV.[41](#hep29877-bib-0041) However, the most unexpected and challenging findings came in the analyses of the dynamic changes in responses on therapy. In non‐CC patients, during DAA treatment, levels of ISGs stabilized and did not show any dynamic oscillation. In contrast, numerous gene sets were activated during DAA therapy in the CC group. Comparison of RNA of whole blood in paired samples from patients of the *IFNL4* CC groups at 4 and 16 weeks, using reactome analysis of the genes up‐regulated over this period, shows up‐regulation of IFN α/β and IFNγ pathways, among others, significantly represented in matching samples (Supporting Table [S2A](#hep29877-sup-0001)).

How may such transcriptional changes impact on the outcome of therapy in a DAA‐only regimen? Because of the very large impact of *IFNL4* genotype on outcome, in this relatively small study, the same changes we have described as associated with genotype are associated with SVR12. Interestingly, on examining the groups closely, 2 patients who had non‐CC genotype and attained SVR show a gene expression at weeks 4 and 16 similar to that of patients from the CC group (CC SVR vs. non‐CC SVR, data not shown). If this transcriptional response is responsible for clearance, as opposed to being correlated with a further biological effector, then there are a number of possibilities, based on the effects of IFNα, IFNλ, and also IFNγ. IFNα itself could drive the elimination of virus through up‐regulation of a number of effector molecules[42](#hep29877-bib-0042)—especially viruses which have not adapted *in vivo*. There is considerable overlap between the ISGs induced in hepatocytes by IFNα and IFNL3[43](#hep29877-bib-0043) whereas the expression of IFNL4 itself appears to be highly suppressed by the host.[44](#hep29877-bib-0044) As we have recently identified,[22](#hep29877-bib-0022) mutations in the *IFNL4* locus are strongly associated with viral adaptations. These mutations are also associated with viral load, and the likely explanation is that they could provide resistance to the impact of endogeneous ISGs, although the exact mechanism requires further exploration. Previous studies have also indicated that elimination of HCV may be orchestrated by *IFNγ*‐driven expression.[45](#hep29877-bib-0045)

Modeling of the dynamics of HCV under therapy indicates that destruction of persistently infected cells could play a role in long‐term clearance.[46](#hep29877-bib-0046) Our findings may be relevant to this idea given that activation of ISGs can be linked to induction of cell death mechanisms.[47](#hep29877-bib-0047) This is evident from our own data, which indicate cell death pathway up‐regulation both using *in vitro* IFNα and in the DAA trial (Supporting Table [S2A](#hep29877-sup-0001)). The up‐regulation of relevant gene pathways and cell death toward the end of therapy would provide a plausible explanation for the impact of *IFNL4* polymorphisms in this context. A similar effect has been demonstrated with host IL28B genetic differences associated with differential effects on beta‐phase decay in viral load; the more rapid decay is thought to reflect enhanced loss of infected hepatocytes in the CC genotype.[48](#hep29877-bib-0048)

The most important aspect in our view of this finding is the paradoxical increase in ISGs observed during viral suppression. One parsimonious explanation for this finding is that of reduction in an inhibitory pathway suppressing responses to viral RNA in the CC group. A simple model is as follows: With the favorable *IFNL4* genotype, *IFNα* and *IFNL3* are induced early (without *IFNL4*). This is either associated with viral clearance or, if not, rapidly damped such that in chronic infection, low levels of IFN gene expression are observed, with low levels of ISGs and higher viral loads. In the non‐CC genotype, IFNs are induced, and if the virus is not cleared, their expression is not damped, leading to high levels of ISGs and a modest reduction in viral loads. Upon DAA‐induced virus control in the favorable genotype, IFNL3 and/or IFNα induction are no longer damped and can be reinduced in response to residual levels of viral RNA. In the non‐CC genotype, IFN levels simply decline in response to viral suppression. Thus, the critical difference between genotypes under this model is not the specific activity of the molecules, but the sensitivity of the regulatory networks to damping. This model, that genetic polymorphism drives dynamic changes in regulatory networks following an intervention, could provide a paradigm relevant to other settings where attempts to link genotype and phenotype are complex. A model for the hypothesis is shown in Supporting Fig. [S5](#hep29877-sup-0001).

Finally, because *IFNL3* and *IFNL4* transcripts were barely detected in both liver and whole blood in these patients, we investigated differential transcriptional activity in nearby genes in the same cyto‐genetic band (19q13.2 comprising 86 genes in 6 Mbp region). We did not detect *IFNL3/4* gene expression in liver (microarray and RNAseq data) or blood samples, nor immediately surrounding genes. However, other genes in the cytogenetic band showed high level of expression, and above all a significant difference was observed between the genotypes. Of note, *SPINT2* was more expressed in the non‐CC patients. Further analysis revealed highest expression in the TT genotype, dropping in the CT genotype and then the CC genotype (Fig. [4](#hep29877-fig-0004)E). It is also of note that SPINT2 is found to be associated with hepatocarcinoma,[49](#hep29877-bib-0049) and exon sequencing analysis in HCV genotype 1b patients showed an SNP (rs3745948) in SPINT2 to be one of the 39 SNPs significantly associated with spontaneous clearance.[50](#hep29877-bib-0050)

Overall, our data indicate a clear impact of *IFNL4* genotype on clinical outcome in patients with cirrhosis treated with a suboptimal DAA regimen, together with a clear association between IFNL4 genotypes and gene expression. These data suggest that a significant role remains for host immunity in such patients and also suggest that the detected changes in gene expression linked to the IFNL4 region drive favorable outcomes. In a clinical setting where liver biopsy is rarely performed, peripheral blood gene expression studies can provide significant signals to link genotype, gene expression, and clinical response. However, further work is required to understand the physiological basis for the powerful impact of cirrhosis on outcome in relation to genotype.

Author names in bold designate shared co‐first authorship.

## Supporting information

## Acknowledgments

We thank Gilead Sciences for providing samples and data from the BOSON clinical study for use in this article. We also thank HCV Research UK (funded by the Medical Research Foundation) for their assistance in handling and coordinating the release of samples for these analyses. We thank the High Throughput Genomics Group at the Wellcome Trust Centre for Human Genetics (funded by Wellcome Trust grant reference 090532/Z/09/Z) for generating the Sequencing data.

## Contributor Information

Narayan Ramamurthy, Email: narayan.ramamurthy@ndm.ox.ac.uk.

Paul Klenerman, Email: paul.klenerman@ndm.ox.ac.uk.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Thimme R, Bukh J, Spangenberg HC, Wieland S, Pemberton J, Steiger C, et al. Viral and immunological determinants of hepatitis C virus clearance, persistence, and disease. Proc Natl Acad Sci 2002;99:235‐268.  [DOI](https://doi.org/10.1073/pnas.202608299) | [PMC free article](/articles/PMC137773/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12441397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci&title=Viral%20and%20immunological%20determinants%20of%20hepatitis%20C%20virus%20clearance,%20persistence,%20and%20disease&author=R%20Thimme&author=J%20Bukh&author=HC%20Spangenberg&author=S%20Wieland&author=J%20Pemberton&volume=99&publication_year=2002&pages=235-268&pmid=12441397&doi=10.1073/pnas.202608299&)

2. Heim MH, Bochud P‐Y, George J. Host‐hepatitis C viral interactions: the role of genetics. J Hepatol 2016;65:S22‐S32.  [DOI](https://doi.org/10.1016/j.jhep.2016.07.037) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27641986/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Host%E2%80%90hepatitis%20C%20viral%20interactions:%20the%20role%20of%20genetics&author=MH%20Heim&author=P%E2%80%90Y%20Bochud&author=J%20George&volume=65&publication_year=2016&pages=S22-S32&pmid=27641986&doi=10.1016/j.jhep.2016.07.037&)

3. Lauer G, Walker B. Hepatitis C virus infection. N Engl J Med 2001;345:41‐52.  [DOI](https://doi.org/10.1056/NEJM200107053450107) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11439948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Hepatitis%20C%20virus%20infection&author=G%20Lauer&author=B%20Walker&volume=345&publication_year=2001&pages=41-52&pmid=11439948&doi=10.1056/NEJM200107053450107&)

4. Fried MW, Shiffman ML, Reddy KR, Smith C, Marinos G, Goncales L, Jr. , et al. PEG‐interferon alfa‐2A plus ribarivin for chronic hepatitis C virus infection. N Engl J Med 2002;347:975‐982.  [DOI](https://doi.org/10.1056/NEJMoa020047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12324553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=PEG%E2%80%90interferon%20alfa%E2%80%902A%20plus%20ribarivin%20for%20chronic%20hepatitis%20C%20virus%20infection&author=MW%20Fried&author=ML%20Shiffman&author=KR%20Reddy&author=C%20Smith&author=G%20Marinos&volume=347&publication_year=2002&pages=975-982&pmid=12324553&doi=10.1056/NEJMoa020047&)

5. Zeuzem S, Andreone P, Pol S, Lawitz E, Diago M, Roberts S, et al. Telaprevir for retreatment of HCV infection. N Engl J Med 2011;364:2417‐2428.  [DOI](https://doi.org/10.1056/NEJMoa1013086) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21696308/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Telaprevir%20for%20retreatment%20of%20HCV%20infection&author=S%20Zeuzem&author=P%20Andreone&author=S%20Pol&author=E%20Lawitz&author=M%20Diago&volume=364&publication_year=2011&pages=2417-2428&pmid=21696308&doi=10.1056/NEJMoa1013086&)

6. Webster DP, Klenerman P, Dusheiko GM. Hepatitis C. Lancet 2015;385:1124‐1135.  [DOI](https://doi.org/10.1016/S0140-6736(14)62401-6) | [PMC free article](/articles/PMC4878852/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25687730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Hepatitis%20C&author=DP%20Webster&author=P%20Klenerman&author=GM%20Dusheiko&volume=385&publication_year=2015&pages=1124-1135&pmid=25687730&doi=10.1016/S0140-6736(14)62401-6&)

7. Messina JP, Humphreys I, Flaxman A, Brown A, Cooke GS, Pybus OG, et al. Global distribution and prevalence of hepatitis C virus genotypes. Hepatology 2015;61:77‐87.  [DOI](https://doi.org/10.1002/hep.27259) | [PMC free article](/articles/PMC4303918/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25069599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Global%20distribution%20and%20prevalence%20of%20hepatitis%20C%20virus%20genotypes&author=JP%20Messina&author=I%20Humphreys&author=A%20Flaxman&author=A%20Brown&author=GS%20Cooke&volume=61&publication_year=2015&pages=77-87&pmid=25069599&doi=10.1002/hep.27259&)

8. Lawitz E, Mangia A, Wyles D, Rodriguez‐Torres M, Hassanein T, Gordon SC, et al. Sofosbuvir for previously untreated chronic hepatitis C infection. N Engl J Med 2013;368:1878‐1887.  [DOI](https://doi.org/10.1056/NEJMoa1214853) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23607594/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Sofosbuvir%20for%20previously%20untreated%20chronic%20hepatitis%20C%20infection&author=E%20Lawitz&author=A%20Mangia&author=D%20Wyles&author=M%20Rodriguez%E2%80%90Torres&author=T%20Hassanein&volume=368&publication_year=2013&pages=1878-1887&pmid=23607594&doi=10.1056/NEJMoa1214853&)

9. Jacobson IM, Gordon SC, Kowdley KV, Yoshida EM, Rodriguez‐Torres M, Sulkowski MS, et al. Sofosbuvir for hepatitis C genotype 2 or 3 in patients without treatment options. N Engl J Med 2013;368:1867‐1877.  [DOI](https://doi.org/10.1056/NEJMoa1214854) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23607593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Sofosbuvir%20for%20hepatitis%20C%20genotype%202%20or%203%20in%20patients%20without%20treatment%20options&author=IM%20Jacobson&author=SC%20Gordon&author=KV%20Kowdley&author=EM%20Yoshida&author=M%20Rodriguez%E2%80%90Torres&volume=368&publication_year=2013&pages=1867-1877&pmid=23607593&doi=10.1056/NEJMoa1214854&)

10. Foster GR, Pianko S, Brown A, Forton D, Nahass RG, George J, et al. Efficacy of sofosbuvir plus ribavirin with or without peginterferon‐alfa in patients with hepatitis C virus genotype 3 infection and treatment‐experienced patients with cirrhosis and hepatitis C virus genotype 2 infection. Gastroenterology 2015;149:1462‐1470.  [DOI](https://doi.org/10.1053/j.gastro.2015.07.043) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26248087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Efficacy%20of%20sofosbuvir%20plus%20ribavirin%20with%20or%20without%20peginterferon%E2%80%90alfa%20in%20patients%20with%20hepatitis%20C%20virus%20genotype%203%20infection%20and%20treatment%E2%80%90experienced%20patients%20with%20cirrhosis%20and%20hepatitis%20C%20virus%20genotype%202%20infection&author=GR%20Foster&author=S%20Pianko&author=A%20Brown&author=D%20Forton&author=RG%20Nahass&volume=149&publication_year=2015&pages=1462-1470&pmid=26248087&doi=10.1053/j.gastro.2015.07.043&)

11. Thomas DL, Thio CL, Martin MP, Qi Y, Ge D, O'hUigin C, et al. Genetic variation in IL28B and spontaneous clearance of hepatitis C virus. Nature 2009;461:798‐801.  [DOI](https://doi.org/10.1038/nature08463) | [PMC free article](/articles/PMC3172006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19759533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20and%20spontaneous%20clearance%20of%20hepatitis%20C%20virus&author=DL%20Thomas&author=CL%20Thio&author=MP%20Martin&author=Y%20Qi&author=D%20Ge&volume=461&publication_year=2009&pages=798-801&pmid=19759533&doi=10.1038/nature08463&)

12. Ge D, Fellay J, Thompson AJ, Simon JS, Shianna KV, Urban TJ, et al. Genetic variation in IL28B predicts hepatitis C treatment‐induced viral clearance. Nature 2009;461:399‐401.  [DOI](https://doi.org/10.1038/nature08309) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19684573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20variation%20in%20IL28B%20predicts%20hepatitis%20C%20treatment%E2%80%90induced%20viral%20clearance&author=D%20Ge&author=J%20Fellay&author=AJ%20Thompson&author=JS%20Simon&author=KV%20Shianna&volume=461&publication_year=2009&pages=399-401&pmid=19684573&doi=10.1038/nature08309&)

13. Rauch A, Kutalik Z, Descombes P, Cai T, Di Iulio J, Mueller T, et al. Genetic variation in IL28B is associated with chronic hepatitis C and treatment failure: a genome‐wide association study. Gastroenterology 2010;138:1338‐1345, 1345‐1347.  [DOI](https://doi.org/10.1053/j.gastro.2009.12.056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20060832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Genetic%20variation%20in%20IL28B%20is%20associated%20with%20chronic%20hepatitis%20C%20and%20treatment%20failure:%20a%20genome%E2%80%90wide%20association%20study&author=A%20Rauch&author=Z%20Kutalik&author=P%20Descombes&author=T%20Cai&author=J%20Di%20Iulio&volume=138&publication_year=2010&pages=1338-1345&pmid=20060832&doi=10.1053/j.gastro.2009.12.056&)

14. Suppiah V, Moldovan M, Ahlenstiel G, Berg T, Weltman M, Abate ML, et al. IL28B is associated with response to chronic hepatitis C interferon‐α and ribavirin therapy. Nat Genet 2009;41:1100‐1104.  [DOI](https://doi.org/10.1038/ng.447) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=IL28B%20is%20associated%20with%20response%20to%20chronic%20hepatitis%20C%20interferon%E2%80%90%CE%B1%20and%20ribavirin%20therapy&author=V%20Suppiah&author=M%20Moldovan&author=G%20Ahlenstiel&author=T%20Berg&author=M%20Weltman&volume=41&publication_year=2009&pages=1100-1104&pmid=19749758&doi=10.1038/ng.447&)

15. Tanaka Y, Nishida N, Sugiyama M, Kurosaki M, Matsuura K, Sakamoto N, et al. Genome‐wide association of IL28B with response to pegylated interferon‐α and ribavirin therapy for chronic hepatitis C. Nat Genet 2009;41:1105‐1109.  [DOI](https://doi.org/10.1038/ng.449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19749757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome%E2%80%90wide%20association%20of%20IL28B%20with%20response%20to%20pegylated%20interferon%E2%80%90%CE%B1%20and%20ribavirin%20therapy%20for%20chronic%20hepatitis%20C&author=Y%20Tanaka&author=N%20Nishida&author=M%20Sugiyama&author=M%20Kurosaki&author=K%20Matsuura&volume=41&publication_year=2009&pages=1105-1109&pmid=19749757&doi=10.1038/ng.449&)

16. Tillmann HL, Thompson AJ, Patel K, Wiese M, Tenckhoff H, Nischalke HD, et al. A polymorphism near IL28B is associated with spontaneous clearance of acute hepatitis C virus and jaundice. Gastroenterology 2010;139:1586‐1592.e1.  [DOI](https://doi.org/10.1053/j.gastro.2010.07.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20637200/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=A%20polymorphism%20near%20IL28B%20is%20associated%20with%20spontaneous%20clearance%20of%20acute%20hepatitis%20C%20virus%20and%20jaundice&author=HL%20Tillmann&author=AJ%20Thompson&author=K%20Patel&author=M%20Wiese&author=H%20Tenckhoff&volume=139&publication_year=2010&pages=1586-1592&pmid=20637200&doi=10.1053/j.gastro.2010.07.005&)

17. Bochud PY, Bibert S, Negro F, Haagmans B, Soulier A, Ferrari C, et al. IL28B polymorphisms predict reduction of HCV RNA from the first day of therapy in chronic hepatitis C. J Hepatol 2011;55:980‐988.  [DOI](https://doi.org/10.1016/j.jhep.2011.01.050) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21354446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=IL28B%20polymorphisms%20predict%20reduction%20of%20HCV%20RNA%20from%20the%20first%20day%20of%20therapy%20in%20chronic%20hepatitis%20C&author=PY%20Bochud&author=S%20Bibert&author=F%20Negro&author=B%20Haagmans&author=A%20Soulier&volume=55&publication_year=2011&pages=980-988&pmid=21354446&doi=10.1016/j.jhep.2011.01.050&)

18. Stenkvist J, Sönnerborg A, Weiland O. HCV RNA decline in chronic HCV genotype 2 and 3 during standard of care treatment according to IL28B polymorphism. J Viral Hepat 2013;20:193‐199.  [DOI](https://doi.org/10.1111/j.1365-2893.2012.01645.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23383658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Viral%20Hepat&title=HCV%20RNA%20decline%20in%20chronic%20HCV%20genotype%202%20and%203%20during%20standard%20of%20care%20treatment%20according%20to%20IL28B%20polymorphism&author=J%20Stenkvist&author=A%20S%C3%B6nnerborg&author=O%20Weiland&volume=20&publication_year=2013&pages=193-199&pmid=23383658&doi=10.1111/j.1365-2893.2012.01645.x&)

19. Scott J, Holte S, Urban T, Burgess C, Coppel E, Wang C, et al. IL28B genotype effects during early treatment with peginterferon and ribavirin in difficult‐to‐treat hepatitis C virus infection. J Infect Dis 2011;204:419‐425.  [DOI](https://doi.org/10.1093/infdis/jir264) | [PMC free article](/articles/PMC3132139/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21742841/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=IL28B%20genotype%20effects%20during%20early%20treatment%20with%20peginterferon%20and%20ribavirin%20in%20difficult%E2%80%90to%E2%80%90treat%20hepatitis%20C%20virus%20infection&author=J%20Scott&author=S%20Holte&author=T%20Urban&author=C%20Burgess&author=E%20Coppel&volume=204&publication_year=2011&pages=419-425&pmid=21742841&doi=10.1093/infdis/jir264&)

20. Honda M, Shirasaki T, Shimakami T, Sakai A, Horii R, Arai K, et al. Hepatic interferon‐stimulated genes are differentially regulated in the liver of chronic hepatitis C patients with different interleukin‐28B genotypes. Hepatology 2014;59:828‐838.  [DOI](https://doi.org/10.1002/hep.26788) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24311440/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Hepatic%20interferon%E2%80%90stimulated%20genes%20are%20differentially%20regulated%20in%20the%20liver%20of%20chronic%20hepatitis%20C%20patients%20with%20different%20interleukin%E2%80%9028B%20genotypes&author=M%20Honda&author=T%20Shirasaki&author=T%20Shimakami&author=A%20Sakai&author=R%20Horii&volume=59&publication_year=2014&pages=828-838&pmid=24311440&doi=10.1002/hep.26788&)

21. Egli A, Santer DM, O'shea D, Tyrrell DL, Houghton M. The impact of the interferon‐lambda family on the innate and adaptive immune response to viral infections. Emerg Microbes Infect 2014;3:e51.  [DOI](https://doi.org/10.1038/emi.2014.51) | [PMC free article](/articles/PMC4126180/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26038748/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Emerg%20Microbes%20Infect&title=The%20impact%20of%20the%20interferon%E2%80%90lambda%20family%20on%20the%20innate%20and%20adaptive%20immune%20response%20to%20viral%20infections&author=A%20Egli&author=DM%20Santer&author=D%20O'shea&author=DL%20Tyrrell&author=M%20Houghton&volume=3&publication_year=2014&pages=e51&pmid=26038748&doi=10.1038/emi.2014.51&)

22. Ansari MA, Pedergnana V, L C Ip C, Magri A, Von Delft A, Bonsall D, et al. Genome‐to‐genome analysis highlights the effect of the human innate and adaptive immune systems on the hepatitis C virus. Nat Genet 2017;49:666‐673.  [DOI](https://doi.org/10.1038/ng.3835) | [PMC free article](/articles/PMC5873514/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28394351/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome%E2%80%90to%E2%80%90genome%20analysis%20highlights%20the%20effect%20of%20the%20human%20innate%20and%20adaptive%20immune%20systems%20on%20the%20hepatitis%20C%20virus&author=MA%20Ansari&author=V%20Pedergnana&author=C%20L%20C%20Ip&author=A%20Magri&author=A%20Von%20Delft&volume=49&publication_year=2017&pages=666-673&pmid=28394351&doi=10.1038/ng.3835&)

23. Terczyńska‐Dyla E, Bibert S, Duong FH, Krol I, Jørgensen S, Collinet E, et al. Reduced IFNλ4 activity is associated with improved HCV clearance and reduced expression of interferon‐stimulated genes. Nat Commun 2014;5:5699.  [DOI](https://doi.org/10.1038/ncomms6699) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25534433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Reduced%20IFN%CE%BB4%20activity%20is%20associated%20with%20improved%20HCV%20clearance%20and%20reduced%20expression%20of%20interferon%E2%80%90stimulated%20genes&author=E%20Terczy%C5%84ska%E2%80%90Dyla&author=S%20Bibert&author=FH%20Duong&author=I%20Krol&author=S%20J%C3%B8rgensen&volume=5&publication_year=2014&pages=5699&pmid=25534433&doi=10.1038/ncomms6699&)

24. Pedergnana V, Smith D, STOP‐HCV Consortium , Klenerman P, Barnes E, Spencer CC, et al. Interferon lambda 4 variant rs12979860 is not associated with RAV NS5A Y93H in hepatitis C virus genotype 3a. Hepatology 2016;64:1377‐1378.  [DOI](https://doi.org/10.1002/hep.28533) | [PMC free article](/articles/PMC5042047/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26945900/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Interferon%20lambda%204%20variant%20rs12979860%20is%20not%20associated%20with%20RAV%20NS5A%20Y93H%20in%20hepatitis%20C%20virus%20genotype%203a&author=V%20Pedergnana&author=D%20Smith&author=P%20Klenerman&author=E%20Barnes&author=CC%20Spencer&volume=64&publication_year=2016&pages=1377-1378&pmid=26945900&doi=10.1002/hep.28533&)

25. Team RDC. R: a language and environment for statistical computing | GBIF.ORG. https://www.gbif.org/resource/81287. Accessed December 14, 2016.  [GBIF.ORG](https://GBIF.ORG) | [https://www.gbif.org/resource/81287](https://www.gbif.org/resource/81287)

26. Smyth G. limma: Linear Models for Microarray Data. Bioinforma Comput Biol Solut Using R Bioconductor 2005;397‐420. 10.1007/0-387-29362-0_23. Accessed December 14, 2016.  [DOI](https://doi.org/10.1007/0-387-29362-0_23) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinforma%20Comput%20Biol%20Solut%20Using%20R%20Bioconductor&title=limma:%20Linear%20Models%20for%20Microarray%20Data&author=G%20Smyth&publication_year=2005&pages=397-420&doi=10.1007/0-387-29362-0_23&)

27. Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing when researchers tend to select pursuing multiple the (statistically) and support of conclusions. An unguarded use in a greatly results of single‐inference inc. J R Stat Soc B 1995;57:289‐300.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20R%20Stat%20Soc%20B&title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing%20when%20researchers%20tend%20to%20select%20pursuing%20multiple%20the%20(statistically)%20and%20support%20of%20conclusions.%20An%20unguarded%20use%20in%20a%20greatly%20results%20of%20single%E2%80%90inference%20inc&author=Y%20Benjamini&author=Y%20Hochberg&volume=57&publication_year=1995&pages=289-300&)

28. Yu G, He QY. ReactomePA: an R/Bioconductor package for reactome pathway analysis and visualization. Mol BioSyst 2016;12:477‐479.  [DOI](https://doi.org/10.1039/c5mb00663e) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26661513/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20BioSyst&title=ReactomePA:%20an%20R/Bioconductor%20package%20for%20reactome%20pathway%20analysis%20and%20visualization&author=G%20Yu&author=QY%20He&volume=12&publication_year=2016&pages=477-479&pmid=26661513&doi=10.1039/c5mb00663e&)

29. Wu D, Smyth GK. Camera: a competitive gene set test accounting for inter‐gene correlation. Nucleic Acids Res 2012;40:e133.  [DOI](https://doi.org/10.1093/nar/gks461) | [PMC free article](/articles/PMC3458527/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22638577/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Camera:%20a%20competitive%20gene%20set%20test%20accounting%20for%20inter%E2%80%90gene%20correlation&author=D%20Wu&author=GK%20Smyth&volume=40&publication_year=2012&pages=e133&pmid=22638577&doi=10.1093/nar/gks461&)

30. Subramanian A, Subramanian A, Tamayo P, Tamayo P, Mootha VK, Mootha VK, et al. Gene set enrichment analysis: a knowledge‐based approach for interpreting genome‐wide expression profiles. Proc Natl Acad Sci U S A 2005;102:15545‐15550.  [DOI](https://doi.org/10.1073/pnas.0506580102) | [PMC free article](/articles/PMC1239896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16199517/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Gene%20set%20enrichment%20analysis:%20a%20knowledge%E2%80%90based%20approach%20for%20interpreting%20genome%E2%80%90wide%20expression%20profiles&author=A%20Subramanian&author=A%20Subramanian&author=P%20Tamayo&author=P%20Tamayo&author=VK%20Mootha&volume=102&publication_year=2005&pages=15545-15550&pmid=16199517&doi=10.1073/pnas.0506580102&)

31. Wang X, Terfve C, Rose JC, Markowetz F. HTSanalyzeR: an R/Bioconductor package for integrated network analysis of high‐throughput screens. Bioinformatics 2011;27:879‐880.  [DOI](https://doi.org/10.1093/bioinformatics/btr028) | [PMC free article](/articles/PMC3051329/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21258062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=HTSanalyzeR:%20an%20R/Bioconductor%20package%20for%20integrated%20network%20analysis%20of%20high%E2%80%90throughput%20screens&author=X%20Wang&author=C%20Terfve&author=JC%20Rose&author=F%20Markowetz&volume=27&publication_year=2011&pages=879-880&pmid=21258062&doi=10.1093/bioinformatics/btr028&)

32. Rauch A, Kutalik Z, Descombes P, Cai T, Di Iulio J, Mueller T, et al. Genetic variation in IL28B is associated with chronic hepatitis C and treatment failure: a genome‐wide association study. Gastroenterology 2010;138:1338‐1345.e7.  [DOI](https://doi.org/10.1053/j.gastro.2009.12.056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20060832/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Genetic%20variation%20in%20IL28B%20is%20associated%20with%20chronic%20hepatitis%20C%20and%20treatment%20failure:%20a%20genome%E2%80%90wide%20association%20study&author=A%20Rauch&author=Z%20Kutalik&author=P%20Descombes&author=T%20Cai&author=J%20Di%20Iulio&volume=138&publication_year=2010&pages=1338-1345&pmid=20060832&doi=10.1053/j.gastro.2009.12.056&)

33. Ramamurthy N, Boninsegna S, Adams R, Sahgal N, Lockstone H, Baban D, et al. Impact of IL‐27 on hepatocyte antiviral gene expression and function. [version1;referees:1 approved,2approved with reservations] Wellcome Open Res 2016;1:17.  [DOI](https://doi.org/10.12688/wellcomeopenres.9917.1) | [PMC free article](/articles/PMC5207303/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28058287/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Wellcome%20Open%20Res&title=Impact%20of%20IL%E2%80%9027%20on%20hepatocyte%20antiviral%20gene%20expression%20and%20function.%20%5Bversion1;referees:1%20approved,2approved%20with%20reservations%5D&author=N%20Ramamurthy&author=S%20Boninsegna&author=R%20Adams&author=N%20Sahgal&author=H%20Lockstone&volume=1&publication_year=2016&pages=17&pmid=28058287&doi=10.12688/wellcomeopenres.9917.1&)

34. Hayes CN, Imamura M, Aikata H, Chayama K. Genetics of IL28B and HCV—response to infection and treatment. Nat Rev Gastroenterol Hepatol 2012;9:406‐417.  [DOI](https://doi.org/10.1038/nrgastro.2012.101) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22641049/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Gastroenterol%20Hepatol&title=Genetics%20of%20IL28B%20and%20HCV%E2%80%94response%20to%20infection%20and%20treatment&author=CN%20Hayes&author=M%20Imamura&author=H%20Aikata&author=K%20Chayama&volume=9&publication_year=2012&pages=406-417&pmid=22641049&doi=10.1038/nrgastro.2012.101&)

35. Honda M, Sakai A, Yamashita T, Nakamoto Y, Mizukoshi E, Sakai Y, et al. Hepatic ISG expression is associated with genetic variation in interleukin 28B and the outcome of IFN therapy for chronic hepatitis C. Gastroenterology 2010;139:499‐509.  [DOI](https://doi.org/10.1053/j.gastro.2010.04.049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20434452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Hepatic%20ISG%20expression%20is%20associated%20with%20genetic%20variation%20in%20interleukin%2028B%20and%20the%20outcome%20of%20IFN%20therapy%20for%20chronic%20hepatitis%20C&author=M%20Honda&author=A%20Sakai&author=T%20Yamashita&author=Y%20Nakamoto&author=E%20Mizukoshi&volume=139&publication_year=2010&pages=499-509&pmid=20434452&doi=10.1053/j.gastro.2010.04.049&)

36. Billerbeck E, Kang YH, Walker L, Lockstone H, Grafmueller S, Fleming V, et al. Analysis of CD161 expression on human CD8+ T cells defines a distinct functional subset with tissue‐homing properties. Proc Natl Acad Sci U S A 2010;107:3006‐3011.  [DOI](https://doi.org/10.1073/pnas.0914839107) | [PMC free article](/articles/PMC2840308/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20133607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Analysis%20of%20CD161%20expression%20on%20human%20CD8+%20T%20cells%20defines%20a%20distinct%20functional%20subset%20with%20tissue%E2%80%90homing%20properties&author=E%20Billerbeck&author=YH%20Kang&author=L%20Walker&author=H%20Lockstone&author=S%20Grafmueller&volume=107&publication_year=2010&pages=3006-3011&pmid=20133607&doi=10.1073/pnas.0914839107&)

37. Van Wilgenburg B, Scherwitzl I, Hutchinson EC, Leng T, Kurioka A, Kulicke C, et al. MAIT cells are activated during human viral infections. Nat Commun 2016;7:11653.  [DOI](https://doi.org/10.1038/ncomms11653) | [PMC free article](/articles/PMC4931007/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27337592/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=MAIT%20cells%20are%20activated%20during%20human%20viral%20infections&author=B%20Van%20Wilgenburg&author=I%20Scherwitzl&author=EC%20Hutchinson&author=T%20Leng&author=A%20Kurioka&volume=7&publication_year=2016&pages=11653&pmid=27337592&doi=10.1038/ncomms11653&)

38. Sarasin‐Filipowicz M, Oakeley EJ, Duong FH, Christen V, Terracciano L, Filipowicz W, et al. Interferon signaling and treatment outcome in chronic hepatitis C. Proc Natl Acad Sci U S A 2008;105:7034‐7039.  [DOI](https://doi.org/10.1073/pnas.0707882105) | [PMC free article](/articles/PMC2383932/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18467494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Interferon%20signaling%20and%20treatment%20outcome%20in%20chronic%20hepatitis%20C&author=M%20Sarasin%E2%80%90Filipowicz&author=EJ%20Oakeley&author=FH%20Duong&author=V%20Christen&author=L%20Terracciano&volume=105&publication_year=2008&pages=7034-7039&pmid=18467494&doi=10.1073/pnas.0707882105&)

39. Taylor MW, Tsukahara T, Brodsky L, Schaley J, Sanda C, Stephens MJ, et al. Changes in gene expression during pegylated interferon and ribavirin therapy of chronic hepatitis C virus distinguish responders from nonresponders to antiviral therapy. J Virol 2007;81:3391‐3401.  [DOI](https://doi.org/10.1128/JVI.02640-06) | [PMC free article](/articles/PMC1866036/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17267482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Virol&title=Changes%20in%20gene%20expression%20during%20pegylated%20interferon%20and%20ribavirin%20therapy%20of%20chronic%20hepatitis%20C%20virus%20distinguish%20responders%20from%20nonresponders%20to%20antiviral%20therapy&author=MW%20Taylor&author=T%20Tsukahara&author=L%20Brodsky&author=J%20Schaley&author=C%20Sanda&volume=81&publication_year=2007&pages=3391-3401&pmid=17267482&doi=10.1128/JVI.02640-06&)

40. Meissner EG, Wu D, Osinusi A, Bon D, Virtaneva K, Sturdevant D, et al. Endogenous intrahepatic IFNs and association with IFN‐free HCV treatment outcome. J Clin Invest 2014;124:3352‐3363.  [DOI](https://doi.org/10.1172/JCI75938) | [PMC free article](/articles/PMC4109554/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24983321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Endogenous%20intrahepatic%20IFNs%20and%20association%20with%20IFN%E2%80%90free%20HCV%20treatment%20outcome&author=EG%20Meissner&author=D%20Wu&author=A%20Osinusi&author=D%20Bon&author=K%20Virtaneva&volume=124&publication_year=2014&pages=3352-3363&pmid=24983321&doi=10.1172/JCI75938&)

41. Holmes JA, Congiu M, Bonanzinga S, Sandhu MK, Kia YH, Bell SJ, et al. The relationships between IFNL4 genotype, intrahepatic interferon‐stimulated gene expression and interferon treatment response differs in HCV‐1 compared with HCV‐3. Aliment Pharmacol Ther 2015;42:296‐306.  [DOI](https://doi.org/10.1111/apt.13263) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26032235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aliment%20Pharmacol%20Ther&title=The%20relationships%20between%20IFNL4%20genotype,%20intrahepatic%20interferon%E2%80%90stimulated%20gene%20expression%20and%20interferon%20treatment%20response%20differs%20in%20HCV%E2%80%901%20compared%20with%20HCV%E2%80%903&author=JA%20Holmes&author=M%20Congiu&author=S%20Bonanzinga&author=MK%20Sandhu&author=YH%20Kia&volume=42&publication_year=2015&pages=296-306&pmid=26032235&doi=10.1111/apt.13263&)

42. Schoggins JW, MacDuff DA, Imanaka N, Gainey MD, Shrestha B, Eitson JL, et al. Pan‐viral specificity of IFN‐induced genes reveals new roles for cGAS in innate immunity. Nature 2014;505:691‐695.  [DOI](https://doi.org/10.1038/nature12862) | [PMC free article](/articles/PMC4077721/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24284630/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Pan%E2%80%90viral%20specificity%20of%20IFN%E2%80%90induced%20genes%20reveals%20new%20roles%20for%20cGAS%20in%20innate%20immunity&author=JW%20Schoggins&author=DA%20MacDuff&author=N%20Imanaka&author=MD%20Gainey&author=B%20Shrestha&volume=505&publication_year=2014&pages=691-695&pmid=24284630&doi=10.1038/nature12862&)

43. Sheahan T, Imanaka N, Marukian S, Dorner M, Liu P, Ploss A, et al. Interferon lambda alleles predict innate antiviral immune responses and hepatitis C virus permissiveness. Cell Host Microbe 2014;15:190‐202.  [DOI](https://doi.org/10.1016/j.chom.2014.01.007) | [PMC free article](/articles/PMC4104123/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24528865/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20Microbe&title=Interferon%20lambda%20alleles%20predict%20innate%20antiviral%20immune%20responses%20and%20hepatitis%20C%20virus%20permissiveness&author=T%20Sheahan&author=N%20Imanaka&author=S%20Marukian&author=M%20Dorner&author=P%20Liu&volume=15&publication_year=2014&pages=190-202&pmid=24528865&doi=10.1016/j.chom.2014.01.007&)

44. Hong M, Schwerk J, Lim C, Kell A, Jarret A, Pangallo J, et al. Interferon lambda 4 expression is suppressed by the host during viral infection. J Exp Med 2016;213:2539‐2552.  [DOI](https://doi.org/10.1084/jem.20160437) | [PMC free article](/articles/PMC5110018/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27799623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Med&title=Interferon%20lambda%204%20expression%20is%20suppressed%20by%20the%20host%20during%20viral%20infection&author=M%20Hong&author=J%20Schwerk&author=C%20Lim&author=A%20Kell&author=A%20Jarret&volume=213&publication_year=2016&pages=2539-2552&pmid=27799623&doi=10.1084/jem.20160437&)

45. Dill MT, Makowska Z, Duong FH, Merkofer F, Filipowicz M, Baumert TF, et al. Interferon‐γ‐stimulated genes, but not USP18, are expressed in livers of patients with acute hepatitis C. Gastroenterology 2012;143:777‐786.e6.  [DOI](https://doi.org/10.1053/j.gastro.2012.05.044) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22677194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Interferon%E2%80%90%CE%B3%E2%80%90stimulated%20genes,%20but%20not%20USP18,%20are%20expressed%20in%20livers%20of%20patients%20with%20acute%20hepatitis%20C&author=MT%20Dill&author=Z%20Makowska&author=FH%20Duong&author=F%20Merkofer&author=M%20Filipowicz&volume=143&publication_year=2012&pages=777-786&pmid=22677194&doi=10.1053/j.gastro.2012.05.044&)

46. Guedj J, Rong L, Dahari H, Perelson AS. A perspective on modelling hepatitis C virus infection. J Viral Hepat 2010;17:825‐833.  [DOI](https://doi.org/10.1111/j.1365-2893.2010.01348.x) | [PMC free article](/articles/PMC3070361/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20723038/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Viral%20Hepat&title=A%20perspective%20on%20modelling%20hepatitis%20C%20virus%20infection&author=J%20Guedj&author=L%20Rong&author=H%20Dahari&author=AS%20Perelson&volume=17&publication_year=2010&pages=825-833&pmid=20723038&doi=10.1111/j.1365-2893.2010.01348.x&)

47. Ravi LI, Li L, Sutejo R, Chen H, Wong PS, Tan BH, et al. A systems‐based approach to analyse the host response in murine lung macrophages challenged with respiratory syncytial virus. BMC Genomics 2013;14:190.  [DOI](https://doi.org/10.1186/1471-2164-14-190) | [PMC free article](/articles/PMC3618260/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23506210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=A%20systems%E2%80%90based%20approach%20to%20analyse%20the%20host%20response%20in%20murine%20lung%20macrophages%20challenged%20with%20respiratory%20syncytial%20virus&author=LI%20Ravi&author=L%20Li&author=R%20Sutejo&author=H%20Chen&author=PS%20Wong&volume=14&publication_year=2013&pages=190&pmid=23506210&doi=10.1186/1471-2164-14-190&)

48. Chu TW, Kulkarni R, Gane EJ, Roberts SK, Stedman C, Angus PW, et al. Effect of IL28B genotype on early viral kinetics during interferon‐free treatment of patients with chronic hepatitis C. Gastroenterology 2012;142:790‐795.  [DOI](https://doi.org/10.1053/j.gastro.2011.12.057) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22248659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Effect%20of%20IL28B%20genotype%20on%20early%20viral%20kinetics%20during%20interferon%E2%80%90free%20treatment%20of%20patients%20with%20chronic%20hepatitis%20C&author=TW%20Chu&author=R%20Kulkarni&author=EJ%20Gane&author=SK%20Roberts&author=C%20Stedman&volume=142&publication_year=2012&pages=790-795&pmid=22248659&doi=10.1053/j.gastro.2011.12.057&)

49. Fukai K, Yokosuka O, Chiba T, Hirasawa Y, Tada M, Imazeki F, et al. Hepatocyte growth factor activator inhibitor 2/placental bikunin (HAI‐2/PB) gene is frequently hypermethylated in human hepatocellular carcinoma. Cancer Res 2003;63:8674‐8679.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/14695180/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Hepatocyte%20growth%20factor%20activator%20inhibitor%202/placental%20bikunin%20(HAI%E2%80%902/PB)%20gene%20is%20frequently%20hypermethylated%20in%20human%20hepatocellular%20carcinoma&author=K%20Fukai&author=O%20Yokosuka&author=T%20Chiba&author=Y%20Hirasawa&author=M%20Tada&volume=63&publication_year=2003&pages=8674-8679&pmid=14695180&)

50. You H, Liu S, Xie Y, Cong R, Sun Y, Ren J, et al. Novel host genetic variations associated with spontaneous clearance of a single‐source outbreak of HCV1b infections. BMJ Open Gastroenterol 2014;1:e000010.  [DOI](https://doi.org/10.1136/bmjgast-2014-000010) | [PMC free article](/articles/PMC4533326/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26462265/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ%20Open%20Gastroenterol&title=Novel%20host%20genetic%20variations%20associated%20with%20spontaneous%20clearance%20of%20a%20single%E2%80%90source%20outbreak%20of%20HCV1b%20infections&author=H%20You&author=S%20Liu&author=Y%20Xie&author=R%20Cong&author=Y%20Sun&volume=1&publication_year=2014&pages=e000010&pmid=26462265&doi=10.1136/bmjgast-2014-000010&)
