# Genomewide Association Study of Statin‐Induced Myopathy in Patients Recruited Using the UK Clinical Practice Research Datalink

## Metadata
**Authors:** Daniel F Carr, Ben Francis, Andrea L Jorgensen, Eunice Zhang, Hector Chinoy, Susan R Heckbert, Joshua C Bis, Jennifer A Brody, James S Floyd, Bruce M Psaty, Mariam Molokhia, Maryse Lapeyre‐Mestre, Anita Conforti, Ana Alfirevic, Tjeerd van Staa, Munir Pirmohamed
**Journal:** Clinical Pharmacology and Therapeutics
**Date:** 2019 Jul 31
**DOI:** [10.1002/cpt.1557](https://doi.org/10.1002/cpt.1557)
**PMID:** 31220337
**PMCID:** PMC6896237
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6896237/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6896237/pdf/CPT-106-1353.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6896237/pdf/CPT-106-1353.pdf)

## Abstract

Statins can be associated with myopathy. We have undertaken a genomewide association study (GWAS) to discover and validate genetic risk factors for statin‐induced myopathy in a “real‐world” setting. One hundred thirty‐five patients with statin myopathy recruited via the UK Clinical Practice Research Datalink were genotyped using the Illumina OmniExpress Exome version 1.0 Bead Chip and compared with the Wellcome Trust Case‐Control Consortium (n = 2,501). Nominally statistically significant single nucleotide polymorphism (SNP) signals in the GWAS (P < 5 × 10−5) were further evaluated in several independent cohorts (comprising 332 cases and 449 drug‐tolerant controls). Only one (rs4149056/c.521C>T in the SLCO1B1 gene) SNP was genomewide significant in the severe myopathy (creatine kinase > 10 × upper limit of normal or rhabdomyolysis) group (P = 2.55 × 10−9; odds ratio 5.15; 95% confidence interval 3.13–8.45). The association with SLCO1B1 was present for several statins and replicated in the independent validation cohorts. The data highlight the role of SLCO1B1 c.521C>T SNP as a replicable genetic risk factor for statin myopathy. No other novel genetic risk factors with a similar effect size were identified.

☑ Risk of statin‐induced myopathy is associated with variation of the *SLCO1B1* gene, which encodes the OATP1B1 hepatic uptake transporter, of which statins are substrates. To date, no other validated genetic risk factors have been identified.

☑ Undertaking a genomewide association study in a “real‐world” patient cohort recruited via the Clinical Practice Research Datalink, this study aimed to determine whether any other novel genetic risk loci for statin myopathy could be identified.

☑ The study suggests that, aside from SLCO1B1, no other risk loci for statin myopathy are apparent. The unexplained statin myopathy risk is likely due to nongenetic risk factors or the influence of rare genetic variants analyzed in this study.

☑ Common genetic variants do not seem to explain statin myopathy risk. The data presented seem to suggest that future translational work in this field should focus on rare variant analysis and on identifying nongenetic risk factors.

The 3‐hydroxy‐3‐methyl‐glutaryl‐coenzyme A inhibitors, or statins, are a widely prescribed class of drugs for the treatment of hyperlipidemia. Although generally well tolerated, a small proportion of patients can develop muscle‐related adverse effects.[1](#cpt1557-bib-0001) These can range from mild muscle pain without creatine kinase (CK) elevation, where causality can be difficult to assess, to myopathy where the CK becomes elevated (>4 × upper limit of normal (ULN)), with the most extreme reactions being rhabdomyolysis with renal impairment.[2](#cpt1557-bib-0002) A systematic review suggested that the incidence of statin‐induced mild muscle pain is 190 cases/100,000 patient years with myopathy and rhabdomyolysis at 5 and 1.6 cases/100,000 patient years, respectively.[3](#cpt1557-bib-0003)

A number of genetic studies[4](#cpt1557-bib-0004), [5](#cpt1557-bib-0005), [6](#cpt1557-bib-0006), [7](#cpt1557-bib-0007), [8](#cpt1557-bib-0008) have identified a nonsynonymous polymorphism (p.V147L/c.521C>T) in the *SLCO1B1* gene (rs4149056), encoding a hepatic uptake transporter protein as a predisposing factor for statin myopathy. Our pilot proof‐of‐principle candidate gene study, which analyzed a subset of the cohort in this study (77 cases and 372 statin‐tolerant controls) replicated the association between the *SLC01B1* gene polymorphism and statin myopathy[9](#cpt1557-bib-0009) showing the validity of our recruitment strategy via the UK Clinical Practice Research Datalink (CPRD), an electronic health record database. The association with the *SLCO1B1* gene has biological plausibility in that it leads to impaired hepatic uptake of statins by the transporter,[10](#cpt1557-bib-0010), [11](#cpt1557-bib-0011) causing increased circulating drug concentrations.[12](#cpt1557-bib-0012), [13](#cpt1557-bib-0013) However, to date, no other clinically relevant, reproducible genetic variants have been identified.

Other genetic markers have been associated with statin‐myopathy, including polymorphisms in the coenzyme Q2 4‐hydroxybenzoate polyprenyltransferase (*COQ2*)[14](#cpt1557-bib-0014) and human eyes shut ortholog[15](#cpt1557-bib-0015) genes, but have not been independently replicated. The genetic association of statin myopathy with the *GATM* gene[16](#cpt1557-bib-0016) has also not been replicated.[17](#cpt1557-bib-0017), [18](#cpt1557-bib-0018)

Utilizing patients with statin myopathy recruited via CPRD,[19](#cpt1557-bib-0019) the aims of our study were twofold: to undertake a genomewide association study (GWAS) to identify novel genetic risk factors predisposing individuals to statin‐induced myopathy, and to validate any association in independent patient groups and perform meta‐analysis of any association signals. Taking all cohorts together, this represents the largest GWAS of the pharmacogenetics of statin myopathy undertaken to date.

## RESULTS

### Case‐control discovery GWAS

A total of 128 of 135 myopathy case samples and 654,642 single nucleotide polymorphisms (SNPs) passed the predefined genotyping quality control (QC) criteria. Of the seven individuals excluded, three failed sample call‐rate criteria, three failed the gender identity check (due to sample mislabeling), and one individual was excluded as a population outlier after principal component analysis ([Figure S1](#cpt1557-sup-0001)). The individual statins responsible for the muscle toxicity are shown in [Table S1](#cpt1557-sup-0001).

From the case‐control discovery GWAS, a total of 21 SNPs were initially identified as notionally significant (*P* < 5 × 10^−5^; 12 from the all myopathy analysis and 9 from the severe myopathy analysis; **Figure** [1](#cpt1557-fig-0001) **a,b**, respectively). However, only one signal reached genomewide significance: rs4149056 in the *SLCO1B1* locus (*P* = 2.5 × 10^−9^; **Table** [1](#cpt1557-tbl-0001)). Sensitivity analysis of discovery cases (all myopathy) for simvastatin cases only ([Figure S3](#cpt1557-sup-0001) and [Table S4](#cpt1557-sup-0001)) showed no genomewide significant association signals (*P* > 5 × 10^−8^), although *SLCO1B1* was among the top associated loci. Similarly, sensitivity analysis of atorvastatin cases only ([Figure S4](#cpt1557-sup-0001) and [Table S5](#cpt1557-sup-0001)) also showed no genomewide significant associations. *SLCO1B1* was not identified in the top associated loci.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/58af/6896237/f4fece404829/CPT-106-1353-g001.jpg)

Manhattan plot of genomewide association analysis of statin‐induced myopathy. The data represents logistic regression derived log P values (y‐axis) of single nucleotide polymorphisms (SNPs) for the discovery case‐control analysis of (a) the “all myopathy” phenotype (n = 128) and (b) the “severe myopathy” sub‐phenotype (n = 32) with the Wellcome Trust Case‐Control Consortium 2 (WTCCC2) population controls (n = 2,501). X‐axis is the position of the SNP with the chromosome indicated. [Colour figure can be viewed at http://www.wileyonlinelibrary.com]

### Table 1.

| All myopathy |  |  | Discovery case‐control study CPRD cases (n = 128) vs. WTCCC (n = 2,501) |  | Replication study EUDRAGENE cases (n = 19) vs. CPRD statin‐tolerant (n = 585) |  | Simvastatin validation case‐control study definite/incipient myopathy cases (n = 141) vs. controls (n = 4,046) |  | Cerivastatin validation case‐control study cases (n = 172) vs. controls (n = 361) |  | Combined meta‐analysis cases (n = 460) vs. controls (n = 7,493) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs# | Chr | Gene | Per allele OR (95% CI) | P value | Per allele OR (95% CI) | P value | Per allele OR (95% CI) | P value | Per allele OR (95% CI) | P value | Per allele OR (95% CI) | P value | I 2 |
| rs36121096 | 5 | PDE4D | 3.82 (2.20–6.62) | 2.0 × 10−5 | 0 (0–∞) | 1.00 | 1.37 (0.56–6.07) | 0.61 | 1.51 (0.60–3.82) | 0.38 | 2.01 (1.22–3.31) | 0.006 | 0.00 |
| rs55902659 | 5 | SLC12A2 | 0.44 (0.31–0.66) | 4.9 × 10−6 | 0.42 (0.15–1.18) | 0.10 | 0.81 (0.76–1.90) | 0.24 | 1.00 (0.72–1.37) | 0.99 | 0.74 (0.59–0.92) | 0.008 | 0.84 |
| rs17359612 | 9 | TLE1 | 2.49 (1.71–3.64) | 1.1 × 10−5 | 1.59 (0.58–4.35) | 0.36 | 1.25 (0.51–4.17) | 0.54 | 1.21 (0.63–2.33) | 0.56 | 1.67 (1.19–2.34) | 0.003 | 0.00 |
| rs79860430 | 14 | ATG14 | 2.59 (1.76–3.82) | 8.4 × 10−6 | 0 (0–∞) | 1.00 | 1.13 (0.34–8.16) | 0.81 | 1.27 (0.62–2.58) | 0.51 | 2.17 (1.48–3.17) | 7.61 × 10‐5 | 0.53 |
| rs77855582 | 16 | GALNS | 3.88 (2.25–6.69) | 1.9 × 10−5 | 3.60 (0.99–13.0) | 0.05 | 1.61 (0.78–4.55) | 0.45 | NA | NA | NA | NA | NA |

Table 1 Caption: SNPs suggested to be associated with all statin‐induced myopathy and severe myopathy from the discovery case‐control analysis, replication analysis, independent simvastatin and cerivastatin study analyses, and the combined meta‐analysis

Univariate analysis of nongenetic variables for myopathy (*n* = 128) and severe myopathy (*n* = 32) cases vs. statin‐tolerant controls (*n* = 585) was undertaken ([Table S1](#cpt1557-sup-0001)). Age, gender, body mass index, antihypertensive comedication, occurrence of cramps, and history of hypertension showed an association with *P* < 0.10 for all myopathy and mean daily dose, age, and occurrence of cramps showed an association with *P* < 0.10 for “severe myopathy.” These variables were incorporated into the replication cohort logistic regression model for case control analysis.

### Replication cohort analysis

Twenty‐one SNPs below a threshold of *P* < 5 × 10^−5^ in the discovery GWAS analysis were carried forward for genotyping in the “Replication Cohort” (consisting of the CPRD statin‐exposed controls (*n* = 585) and the EUDRAGENE statin‐myopathy cases (*n* = 19)). A total of 9 SNPs (two from the severe myopathy analysis and seven from the all myopathy analysis), however, were subsequently excluded because of inability to design TaqMan or MassArray assays due to proximal sequence constraints (*n* = 4), low genotyping call rate (*n* = 4), and Hardy−Weinberg deviation (*n* = 1). Thus, a total of 12 SNPs were genotyped (5 for all myopathy and 7 for severe myopathy; **Table** [1](#cpt1557-tbl-0001)).

Data from simvastatin and atorvastatin cases only sensitivity analyses for the 12 SNPs initially identified in the overall discovery cohort is reported in [Table S3](#cpt1557-sup-0001). None of the 12 signals showed a statistically significant association with atorvastatin myopathy. Eight loci were notionally associated with simvastatin “all myopathy” albeit not to genomewide significance.

#### Replication cohort

Candidate gene analysis of the 19 EUDRAGENE myopathy cases (13 severe) with the 585 statin‐tolerant controls identified 2 of 12 associations of nominal significance (*P* < 0.05), both of which were for the severe myopathy phenotype (**Table** [1](#cpt1557-tbl-0001)). These were *SLCO1B1* rs4149056 (*P* = 0.001; odds ratio (OR) 3.98; 95% confidence interval (CI) 1.75–9.03) and *SLCO1A2* rs4149000 (*P* = 0.05; OR 2.53; 95% CI 1.00–6.39). No other statistically significant associations were observed in the other nine SNPs (*P* < 0.05). Minor allele frequencies for the 12 SNPs were comparable for both simvastatin and atorvastatin controls.

### Association signal validation

#### “Simvastatin” cohort

Summary statistics from this cohort (**Table** [1](#cpt1557-tbl-0001)) showed a genomewide significant association for 2 of 12 SNPs, both of which were associated with the severe myopathy phenotype in the initial analysis. Both rs4149056 in *SLCO1B1* and rs4149000 in *SLCO1A2* were significantly associated with both definite myopathy (*P* = 7.30 × 10^−14^ and *P* = 7.62 × 10^−11^, respectively) and the incipient or definite myopathy phenotype (*P* = 1.33 × 10^−11^ and 6.49 × 10^−12^). None of the other SNPs were significantly associated with either myopathy phenotype (*P* > 0.05).

#### “Cerivastatin” cohort

Summary statistics from the cerivastatin rhabdomyolysis validation cohort (**Table** [1](#cpt1557-tbl-0001)) showed that the same two SNPs (rs4149056 and rs4149000) showed a significant (albeit not to a genomewide threshold) association (3.90 × 10^−4^ and 0.007, respectively). None of the other SNPs showed an association (*P* > 0.05).

### Further analysis of the SLCO1B1 and SLCO1A2 signals

The two SNPs that seem to be strongly associated with statin myopathy in the discovery and replication cohorts are within two gene loci (*SLCO1B1* and *SLCO1A2*) that are in a strong block of linkage disequilibrium in the discovery cohort (data not shown). Conditional analysis correcting for the *SLCO1B1* rs4149056 genotype was undertaken. The analysis of the 32 discovery severe myopathy cases vs. 585 statin‐tolerant controls abolished the rs4149000 genotype association (*P* = 0.934), as well as for the all myopathy phenotype (*P* = 0.368) indicating that the two risk alleles are not acting in *cis* on the same haplotype and that the *SLCO1A2* association is not acting independently of *SLCO1B1*.

### Meta‐analysis of SLCO1B1 and SLCO1A2 association signals

Meta‐analysis combining the discovery and all replication cohorts (limited to severe cases only; 271 cases vs. 7,493 controls) yielded a meta‐analytic genomewide significant *P* value for SLCO1B1, rs4149056 (*P* = 2.63 × 10^−18^; OR 2.99; 95% CI 2.34–3.82; **Table** [1](#cpt1557-tbl-0001)), highlighting the predominant role of *SLCO1B1* in predisposing to myopathy caused by a variety of statins. An increased statistical significance of the association signal within the *SLCO1A2* was also observed, but no other meta‐analysis demonstrated an increased statistical significance of the signal initially identified in the discovery case‐control study (**Table** [1](#cpt1557-tbl-0001)). Meta‐analysis of the *SLCO1B1* signal for rs4149056 limited to the simvastatin‐exposed cases and controls only (**Figure** [2](#cpt1557-fig-0002)) led to a *P* value of 1.46 × 10^−21^ (OR 5.91; 95% CI 4.10–8.51; *I* ^2^ = 1.00) for the severe myopathy phenotype and a *P* value of 2.01 × 10^−14^ (OR 2.75; 95% CI 2.12–3.56; *I* ^2^ = 0.78) for the all myopathy phenotype.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/58af/6896237/20be255da38f/CPT-106-1353-g002.jpg)

Forest plot depicting meta‐analysis for the SLCO1B1 c.521T>C (rs4149056) polymorphism for both “all myopathy” (creatine kinase (CK) > 4 × upper limit of normal (ULN)) and “severe myopathy” (CK > 10 × ULN/rhabdomyolysis) phenotypes caused by all statins (upper panels) and simvastatin only (lower panels). CI, confidence interval; OR, odds ratio; WTCCC, Wellcome Trust Case‐Control Consortium.

## DISCUSSION

Our discovery GWAS identified 12 SNPs, which were nominally associated with either all myopathy (CK > 4 × ULN ± muscle symptoms) or severe myopathy (CK > 10 × ULN or rhabdomyolysis). Replication was undertaken in three separate patient cohorts, which showed that only the previously identified[6](#cpt1557-bib-0006), [9](#cpt1557-bib-0009) and widely replicated c.521C>T variant (rs4149056) in *SLCO1B1* and an intronic SNP in the *SLCO1A2* gene were risk factors for statin myopathy. The latter, however, was not significant after adjustment for *SLCO1B1* genotype. Our data concur with a previous statin myopathy GWAS,[9](#cpt1557-bib-0009) as well as our own pilot data[9](#cpt1557-bib-0009) and other candidate gene studies,[4](#cpt1557-bib-0004), [5](#cpt1557-bib-0005), [8](#cpt1557-bib-0008), [20](#cpt1557-bib-0020) that the *SLCO1B1* c.521C>T polymorphism (rs4149056) is the predominant genetic risk factor for statin‐induced myopathy. Our finding is also consistent with a recent meta‐analysis of 14 studies comprising 3,265 myopathy patients and 7,743 controls.[21](#cpt1557-bib-0021) Additionally, previously reported associations in the *GATM*,[16](#cpt1557-bib-0016) *COQ2*,[14](#cpt1557-bib-0014) and human eyes shut ortholog[15](#cpt1557-bib-0015) gene loci were not replicated.

Our discovery cohort was heterogeneous in terms of the severity of myopathy and statin implicated. The association with the rs4149056 variant in *SLCO1B1* was stronger in patients with the severe form of myopathy (CK > 10 × ULN or rhabdomyolysis) irrespective of the statin involved reaching genomewide significance (**Figure** [2](#cpt1557-fig-0002)). The lower effect size observed in patients with the less severe form of statin myopathy (defined in our discovery cohort as CK > 4 × ULN) may reflect multiple causes in the mildly affected cases and the difficulty in attributing causality to statins in all cases.

Of the different statins implicated, simvastatin was the most common, accounting for 66% of our cases and 69% of the severe cases. Meta‐analysis of our discovery severe myopathy cohort with the simvastatin definite myopathy cohort did strengthen the association (*P* = 7.17 × 10^−19^) with little evidence of heterogeneity between the two (*I* ^2^ = 1). Further incorporation of the cerivastatin cohort marginally weakened the association in keeping with the different effect sizes of the *SLCO1B1* locus for different statins. These data are consistent with the fact that the pharmacokinetic effect of the *SLCO1B1* variant is greatest for simvastatin. The area under the curve for simvastatin acid is increased by 221% in CC homozygotes compared with those individuals who are TT homozygotes for the *SLCO1B1* c.521C>T polymorphism.[13](#cpt1557-bib-0013) Corresponding values for the other statins are as follows: atorvastatin (145%),[22](#cpt1557-bib-0022) fluvastatin (19%),[23](#cpt1557-bib-0023) lovastatin acid (186%),[24](#cpt1557-bib-0024) pitavastatin (208%),[25](#cpt1557-bib-0025) pravastatin (91%),[23](#cpt1557-bib-0023) and rosuvastatin (65%).[22](#cpt1557-bib-0022) No similar data are available for cerivastatin. Based on these pharmacokinetic data and the results of our data, together with the recent meta‐analysis,[21](#cpt1557-bib-0021) it might be suggested that *SLCO1B1* locus is important for all statins, but the effect size will likely vary, being greatest for simvastatin and lowest for fluvastatin.

The aim of our GWAS was to identify other loci associated with statin myopathy. Apart from the association with *SLCO1B1*, we also identified an association with an SNP located in the 5′UTR of the *SLCO1A2* locus ([Figure S2](#cpt1557-sup-0001)). This signal, however, was not independent of the *SLCO1B1* signal. *SLCO1A2* encodes SLCO1A2, a hepatic‐expressed efflux transporter,[26](#cpt1557-bib-0026) which is responsible for the sodium‐independent transport of organic anions, such as bromosulfophthalein, taurocholate, and unconjugated cholate bile acids.[27](#cpt1557-bib-0027), [28](#cpt1557-bib-0028) SLCO1A2 has also been shown to have substrate specificity for pitavastatin[29](#cpt1557-bib-0029) and rosuvastatin,[30](#cpt1557-bib-0030) but, to date, there is no evidence of its ability to transport simvastatin or atorvastatin. Determination of whether the *SLCO1A2* locus can act as an independent risk factor for statin myopathy will require a much larger sample size. None of the other loci identified in the discovery cohort were replicated in any of the cohorts, and meta‐analysis did not provide any indication that these loci acted as predisposing factors for statin myopathy.

For a number of years, the c.521T>C *SLCO1B*1 variant has been recognized as a clinically important risk factor for statin‐induced myopathy, particularly with regard to simvastatin, and to a lesser extent atorvastatin. Indeed, summary of product characteristics labeling for both drugs[31](#cpt1557-bib-0031), [32](#cpt1557-bib-0032) highlights the increased risk of myopathy in individuals who are carriers of the low‐activity C allele. It has been suggested that the maximum dose of simvastatin, pitavastatin, and atorvastatin should be reduced by fourfold in individuals who are CC homozygotes based on pharmacokinetic calculations.[33](#cpt1557-bib-0033) Interestingly, a recent small randomized trial (*n* = 159) of patients not on statins because of prior myalgia attributed to a statin showed that providing information on the *SLCO1B1* genotype improved statin reinitiation and low‐density lipoprotein‐cholesterol lowering, but not adherence, when compared with the usual care arm.[34](#cpt1557-bib-0034)

It is clear that although S*LCO1B1* is a key risk factor for statin myopathy, it does not explain a significant proportion of the interindividual variability in statin toxicity. Genetic studies to date have been limited by recruitment of significant numbers of cases of what is a rare adverse drug reaction. As such, many studies lack statistical power to detect small effect sizes and have, in fact, only identified the “low hanging fruit.” It is possible that much of the heritability of statin myopathy risk may lie either with rare variants of large effect sizes or with other common genetic loci with small to modest effect sizes, which will require much larger patient numbers. A major issue is that we do not fully understand the mechanism of statin myopathy and muscle damage, apart from the fact that high doses or high statin concentrations increase risk. Further functional studies to uncover the mechanisms of muscle damage induced by statins will be important in elucidating further predisposing factors.

In conclusion, our data further confirm the predominant role of the *SLCO1B1* c.521 C>T polymorphism in predisposing to statin‐induced myopathy in a “real‐world” patient population, in particular with simvastatin. Moreover, the data failed to identify other statin myopathy‐associated genetic risk factors. However, this meta‐analysis is relatively small and may lack statistical power. The additional signal identified in the *SLCO1A2* locus was not independent of *SLCO1B1*, but may require further investigation from a functional perspective to determine the role of this transporter in statin transport.

## METHODS

The study design is summarized in **Figure** [3](#cpt1557-fig-0003). Briefly, a discovery case‐control GWAS was undertaken, followed by candidate variant replication in a second case‐control cohort. The same association signals were validated in existing data from two independent validation case‐control studies.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/58af/6896237/18569b3471f0/CPT-106-1353-g003.jpg)

Schematic representation of the discovery, replication, and validation cohort case‐control analyses and subsequent meta‐analysis performed. Patient numbers represent those included in analyses post‐sample quality control. CPRD, Clinical Practice Research Datalink; GWAS, genomewide association study; SNP, single nucleotide polymorphism; WTCCC, Wellcome Trust Case‐Control Consortium 2.

### CPRD case‐control recruitment

From a cohort of ~ 600,000 patients receiving statins identified in the CPRD ([http://www.cprd.com](http://www.cprd.com)), a case‐control design was used to identify suitable patients for the study, as previously described.[19](#cpt1557-bib-0019), [35](#cpt1557-bib-0035) Participation was restricted to white people ≥18 years of age and with the first ever statin prescription at least 1 year after the start of CPRD data collection.

All cases conformed to internationally agreed standards for statin‐induced myopathy and rhabdomyolysis.[2](#cpt1557-bib-0002) Cases were categorized into two groups: (i) myopathy: patients who discontinued their implicated statin with a rise in CK > 4 × ULN; and (ii) severe myopathy: individuals with a history of rhabdomyolysis or CK > 10 × ULN after statin exposure. Controls were defined as individuals receiving statins for at least 3 months with no history of abnormal serum CK measurements.

General practitioners (GPs) were contacted with a list of potential cases and/or controls identified from their practice. They were first asked to review the medical records of listed individuals and remove any patients they considered to not fulfill the case or control criteria. They were then asked to contact suitable patients by letter requesting participation. Individuals who gave written informed consent were invited to provide either a saliva sample (by post) or a blood sample (by visiting the practice). All samples were then forwarded on to the University of Liverpool for processing. To preserve anonymity, patient and practice identifier codes were used throughout the recruitment process and all patient contact was via the GP only. A total of 149 myopathy cases and 585 controls were recruited between April 2010 and June 2013, although only 135 cases were available at the time of genotyping.[19](#cpt1557-bib-0019) Relevant clinical and demographic data (summarized in **Table** [1](#cpt1557-tbl-0001)) was retrospectively obtained from the CPRD.

In addition to the above, five supplemental cases of statin‐induced myopathy conforming to our phenotype criteria were identified in the tertiary adult muscle clinic run through Salford Royal NHS Foundation Trust, UK, and recruited into the UKMYONET genetic study.[36](#cpt1557-bib-0036)

### Additional cohorts and studies

Three replication cohorts were utilized to validate associations identified in the case‐control discovery GWAS (summarized in **Figure** [3](#cpt1557-fig-0003)).

#### EUDRAGENE cohort

A total of 19 adults (>18 years of age) with statin‐exposed myopathy (5 simvastatin, 5 atorvastatin, 2 rosuvastatin, and 1 fluvastatin) matching the case phenotype (defined above) were recruited using spontaneous adverse drug reaction reports and laboratory CK results from the European pharmacovigilance center and UK primary care practices from 2006−2012 via the EUDRAGENE collaborative network.[37](#cpt1557-bib-0037) All cases were adjudicated using internationally accepted criteria for myopathy and severe myopathy[2](#cpt1557-bib-0002) by an independent panel, consisting of clinicians and pharmacovigilance experts. Of the 19 cases, 13 were categorized as having severe myopathy. EUDRAGENE cases were combined with the 585 CPRD statin‐tolerant controls to form the “Replication cohort.”

#### The “simvastatin” validation case‐control study

A total of 141 patients with simvastatin myopathy were recruited via the SEARCH collaborative group.[6](#cpt1557-bib-0006) These consisted of 54 “definite” statin myopathies (defined as otherwise unexplained muscle symptoms with CK > 10 × ULN) and 87 “incipient” statin myopathies (defined as alanine aminotransferase > 1.7 × ULN and CK both > 5 × baseline and > 3 × ULN). For the purpose of meta‐analysis, the “definite” myopathy phenotype aligned to these studies was severe myopathy phenotype with both “incipient and definite” aligning to the all myopathy phenotype. A total of 4,046 statin‐tolerant controls were included from the SEARCH and Heart Protection Study groups.[6](#cpt1557-bib-0006), [38](#cpt1557-bib-0038)

#### The “cerivastatin” validation case‐control study

The study sample consisted of data from 172 cases of cerivastatin rhabdomyolysis and 361 statin‐using controls from the Heart and Vascular Health study, as previously reported.[7](#cpt1557-bib-0007) All cases and controls were of European ancestry. Controls with creatine kinase levels > 10 × ULN were excluded.

Ethical approval for recruitment via CPRD was obtained from the National Research Ethics Committee North West 2 – Liverpool Central. Furthermore, approval to use the CPRD data was obtained from the Independent Scientific Advisory Committee at the Medicines and Healthcare products Regulatory Agency. In addition, site‐specific approval to contact the GP practices was obtained for each of 132 primary care trusts across the United Kingdom, as described previously.[39](#cpt1557-bib-0039) Written informed consent was obtained from all study subjects or their guardians. The UKMYONET study was approved by the North West Research Multi‐Centre Research Ethics Committee (98/8/86), and all participants gave written informed consent.

Multicenter ethics approval was obtained from the South East Research Ethics Committee for the SEARCH study and from the local ethics committees covering each of the 69 UK hospitals involved in the Heart Protection Study.

The recruitment of cerivastatin case subjects was approved by the University of Washington Institutional Review Board, and the use of the Heart and Vascular Health study subjects was approved by the Group Health Subjects Review Committee.

### DNA extraction and genotyping

#### Case‐control “discovery” cohort

For the CPRD recruits (cases and controls), genomic DNA was extracted from 5 mL whole blood or 2 mL saliva (collected using the Oragene DNA Sampling kit; DNAGenotek, Ontario, Canada) using the Chemagic Magnetic Module (MSM) 1 system, as per the manufacturer's protocol (Chemagen Biopolymer‐Technologie AG, Baesweiler, Germany).

At the time of analysis, DNA samples from a total of 135 myopathy cases from the discovery cohort were available. At least 1.5 μg DNA from myopathy cases was genotyped for a total of 982,958 SNPs by ARK‐Genomics, University of Edinburgh (Edinburgh, UK) using the Illumina OmniExpress Exome version 1.0 Bead Chip array according to the manufacturers protocol (Illumina, San Diego, CA).

#### Discovery case‐control study‐population controls

Population control genotype data for the initial discovery case‐control GWAS was obtained from the Wellcome Trust Case‐Control Consortium 2 (WTCCC2) cohort of 2,501 individuals from the UK Blood Service.

#### Replication study

All 585 CPRD statin‐tolerant controls and 19 patients with myopathy from the EUDRAGENE cohort were genotyped for statistically significant association signals identified in the case‐control discovery GWAS (*P* < 5 × 10^−5^) using either the Agena MassArray iPLEX platform (Agena Biosciences, San Diego, CA) or TaqMan real‐time polymerase chain reaction SNP genotyping assay (Life Technologies, Paisley, UK), according to the manufacturer's protocols.

### Genotyping QC and imputation

#### Case‐control “discovery” cohort

Cases identified via CPRD recruitment were excluded if they failed to meet the following criteria: (i) gender as determined by the “Sex Check” function within PLINK[40](#cpt1557-bib-0040) differed from that reported in the clinical data; (ii) genotype call‐rate < 90%; and (iii) principle component analysis (using SNPRelate[41](#cpt1557-bib-0041) in R version 3.01) demonstrated that the individual did not cluster with the HapMap CEU (Utah residents with European ancestry) population ([Figure S1](#cpt1557-sup-0001)).

SNPs, in both the discovery and replication cohorts, were excluded if (i) minor allele frequency was < 0.01, (ii) Hardy–Weinberg Equilibrium *P* value was < 0.0001, and (iii) the genotype success rate was < 95%. All QC analysis was undertaken using PLINK version 1.07[40](#cpt1557-bib-0040) unless otherwise stated.

For the purpose of the discovery case‐control study, the CPRD case genotype dataset was merged with the WTCCC dataset prior to SNP phasing using SHAPEIT[42](#cpt1557-bib-0042) and imputation using IMPUTE2[43](#cpt1557-bib-0043), [44](#cpt1557-bib-0044) was undertaken using the 1000 Genomes Project phase III reference panel.

#### The “simvastatin” validation case‐control study

Genotype imputation was undertaken using minimac[45](#cpt1557-bib-0045) with the 1000 Genomes European reference panel. Data for the association signal SNPs were provided for the validation and meta‐analysis.

#### The “cerivastatin” validation case‐control study

Samples were excluded from analysis for gender mismatch or call rate < 95%. The following variant exclusions were applied to obtain a cleaned set of variants for imputation: call rate < 97%, Hardy–Weinberg Equilibrium *P* < 10^−5^, >2 duplicate errors or Mendelian inconsistencies (for reference Centre d'Etude du Polymophisme Humain (CEPH) trios), heterozygote frequency = 0. MaCH[45](#cpt1557-bib-0045) was used to pre‐phase the genotypes. The phased genotypes were imputed into a reference panel of 1,092 individuals of multiple ethnicities from the phase I (version 3) haplotypes of 1000 Genomes Project using minimac.[45](#cpt1557-bib-0045) Genotyping and SNP calling were performed using the Illumina 370CNV Bead Chip, as previously described.[7](#cpt1557-bib-0007) Data for the association signal SNPs were provided for the validation and meta‐analysis.

### Statistical analysis

The study design and statistical analysis are summarized in **Figure** [3](#cpt1557-fig-0003). In the discovery phase, cases passing genotype QC (*n* = 128) recruited via CPRD were compared with WTCCC2 controls (*n* = 2,501) using a logistic regression analysis undertaken in SNPTest[46](#cpt1557-bib-0046) and adjusting for the first two principle components as covariates. All SNPs giving a *P* value for association of < 5 × 10^−5^ were genotyped in the statin‐tolerant cohort (*n* = 585) and EUDRAGENE cohort (*n* = 19), which together formed the replication cohort.

A univariate analysis of nongenetic covariates (χ^2^ for categorical outcomes and Student's *t*‐test for continuous variables) was undertaken (**Table** [1](#cpt1557-tbl-0001)) using SPSS version 17.0. Variables demonstrating a *P* value < 0.10 between the discovery cohort cases and tolerant controls were carried forward and adjusted for in the SNP association analyses ([Table S1](#cpt1557-sup-0001)). Logistic regression analysis of the candidate SNPs in the cases (discovery and replication) and statin‐tolerant controls was undertaken using SNPTest. Meta‐analysis of the combined discovery and replication cohorts along with the two validation studies was undertaken using a fixed‐effects model with inverse‐variant effect size weighting in GWAMA.[47](#cpt1557-bib-0047) Forest plots were prepared using the “forestplot” function in R.

## Funding

This work was funded by a grant from the e‐Health Initiative funded jointly by the Medical Research Council (reference: MC_qA137929), Wellcome Trust, Engineering and Physical Sciences Research Council, and Economic and Social Research Council, and HL078888. M.P. is an National Institute for Health Research (NIHR) Senior Investigator. M.P. also thanks the Medical Research Council Centre for Drug Safety Science and NIHR Collaborations for Leadership in Applied Health Research and Care North West Coast for infrastructure support. Collaborators of UKMYONET are acknowledged in ref. [36](#cpt1557-bib-0036). This report includes independent research supported by the NIHR Biomedical Research Centre Funding Scheme. The views expressed in this publication are those of the authors and not necessarily those of the National Health Service (NHS), the NIHR, or the Department of Health.

This research was supported by the NIHR Biomedical Research Centre at Guy's and St Thomas’ NHS Foundation Trust and King's College London. The views expressed are those of the author(s) and not necessarily those of the NHS, the NIHR, or the Department of Health.

The EUDRAGENE project was supported with a concerted action grant from the European Commission 5th Framework QLRI‐CT‐2002‐02757, by the Serious Adverse Events Consortium (SAEC), and the NIHR Biomedical Research Centre at Guy's and St Thomas’ NHS Foundation Trust and King's College London.

## Conflict of Interest

B.M.P. serves on the Steering Committee of the Yale Open Data Access Project funded by Johnson & Johnson. No other author declares any conflict of interest.

## Author Contributions

D.F.C., B.F., A.L.J., B.M.P., and M.P. wrote the manuscript. M.P., T.vS., and B.M.P. designed research. D.F.C., E.Z., H.C., S.R.H., J.C.B., J.A.B., J.F., B.M.P., M.M., M.L.‐M., A.C., A.A., T.vS., and M.P. performed research. D.F.C, B.F., A.L.J., A.A., T.vS., and M.P. analyzed data.

## Supporting information

## Acknowledgments

The authors thank the following people: Sir Rory Collins, Jane Armitage, Jemma Hopewell, and Sarah Parish from UK Clinical Trial Service Unit and Epidemiological Studies Unit, Nuffield Department of Population Health, University of Oxford, Oxford, for provision of summary statistical data from their case‐control analysis of 141 myopathy cases and 4,054 controls (The “simvastatin” validation case‐control study). Jennifer Campbell, Mark Hobbs, Gerard McCann, and the Medicines and Healthcare products Regulatory Agency for their help in facilitating the recruitment of patients to the STAGE study via the CPRD. All contributors to the CPRD, the recruiting GP physicians, and patients who participated in the study. All EUDRAGENE and SEARCH collaborators, patients, and staff who supported this study.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Collins, R. et al Interpretation of the evidence for the efficacy and safety of statin therapy. Lancet 388, 2532–2561 (2016).  [DOI](https://doi.org/10.1016/S0140-6736(16)31357-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27616593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Interpretation%20of%20the%20evidence%20for%20the%20efficacy%20and%20safety%20of%20statin%20therapy&author=R.%20Collins&volume=388&publication_year=2016&pages=2532-2561&pmid=27616593&doi=10.1016/S0140-6736(16)31357-5&)

2. Alfirevic, A. et al Phenotype standardization for statin‐induced myotoxicity. Clin. Pharmacol. Ther. 96, 470–476 (2014).  [DOI](https://doi.org/10.1038/clpt.2014.121) | [PMC free article](/articles/PMC4172546/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24897241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Phenotype%20standardization%20for%20statin%E2%80%90induced%20myotoxicity&author=A.%20Alfirevic&volume=96&publication_year=2014&pages=470-476&pmid=24897241&doi=10.1038/clpt.2014.121&)

3. Law, M. & Rudnicka, A.R. Statin safety: a systematic review. Am. J. Cardiol. 97, 52C–60C (2006).  [DOI](https://doi.org/10.1016/j.amjcard.2005.12.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16581329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Cardiol.&title=Statin%20safety:%20a%20systematic%20review&author=M.%20Law&author=A.R.%20Rudnicka&volume=97&publication_year=2006&pages=52C-60C&pmid=16581329&doi=10.1016/j.amjcard.2005.12.010&)

4. Brunham, L.R. et al Differential effect of the rs4149056 variant in SLCO1B1 on myopathy associated with simvastatin and atorvastatin. Pharmacogenom. J. 12, 233–237 (2012).  [DOI](https://doi.org/10.1038/tpj.2010.92) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21243006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom.%20J.&title=Differential%20effect%20of%20the%20rs4149056%20variant%20in%20SLCO1B1%20on%20myopathy%20associated%20with%20simvastatin%20and%20atorvastatin&author=L.R.%20Brunham&volume=12&publication_year=2012&pages=233-237&pmid=21243006&doi=10.1038/tpj.2010.92&)

5. Donnelly, L.A. et al Common nonsynonymous substitutions in SLCO1B1 predispose to statin intolerance in routinely treated individuals with type 2 diabetes: a go‐DARTS study. Clin. Pharmacol. Ther. 89, 210–216 (2011).  [DOI](https://doi.org/10.1038/clpt.2010.255) | [PMC free article](/articles/PMC3353487/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21178985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Common%20nonsynonymous%20substitutions%20in%20SLCO1B1%20predispose%20to%20statin%20intolerance%20in%20routinely%20treated%20individuals%20with%20type%202%20diabetes:%20a%20go%E2%80%90DARTS%20study&author=L.A.%20Donnelly&volume=89&publication_year=2011&pages=210-216&pmid=21178985&doi=10.1038/clpt.2010.255&)

6. Group, S.C. et al SLCO1B1 variants and statin‐induced myopathy–a genomewide study. N. Engl. J. Med. 359, 789–799 (2008).  [DOI](https://doi.org/10.1056/NEJMoa0801936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18650507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=SLCO1B1%20variants%20and%20statin%E2%80%90induced%20myopathy%E2%80%93a%20genomewide%20study&author=S.C.%20Group&volume=359&publication_year=2008&pages=789-799&pmid=18650507&doi=10.1056/NEJMoa0801936&)

7. Marciante, K.D. et al Cerivastatin, genetic variants, and the risk of rhabdomyolysis. Pharmacogenet. Genom. 21, 280–288 (2011).  [DOI](https://doi.org/10.1097/FPC.0b013e328343dd7d) | [PMC free article](/articles/PMC3076530/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21386754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=Cerivastatin,%20genetic%20variants,%20and%20the%20risk%20of%20rhabdomyolysis&author=K.D.%20Marciante&volume=21&publication_year=2011&pages=280-288&pmid=21386754&doi=10.1097/FPC.0b013e328343dd7d&)

8. Voora, D. et al The SLCO1B1*5 genetic variant is associated with statin‐induced side effects. J. Am. Coll. Cardiol. 54, 1609–1616 (2009).  [DOI](https://doi.org/10.1016/j.jacc.2009.04.053) | [PMC free article](/articles/PMC3417133/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19833260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=The%20SLCO1B1*5%20genetic%20variant%20is%20associated%20with%20statin%E2%80%90induced%20side%20effects&author=D.%20Voora&volume=54&publication_year=2009&pages=1609-1616&pmid=19833260&doi=10.1016/j.jacc.2009.04.053&)

9. Carr, D.F. et al SLCO1B1 genetic variant associated with statin‐induced myopathy: a proof of concept study using the Clinical Practice Research Datalink (CPRD). Clin. Pharmacol. Ther. 96, 695–791 (2013).  [DOI](https://doi.org/10.1038/clpt.2013.161) | [PMC free article](/articles/PMC3831180/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23942138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=SLCO1B1%20genetic%20variant%20associated%20with%20statin%E2%80%90induced%20myopathy:%20a%20proof%20of%20concept%20study%20using%20the%20Clinical%20Practice%20Research%20Datalink%20(CPRD)&author=D.F.%20Carr&volume=96&publication_year=2013&pages=695-791&pmid=23942138&doi=10.1038/clpt.2013.161&)

10. Lau, Y.Y. , Huang, Y. , Frassetto, L. & Benet, L.Z. Effect of OATP1B transporter inhibition on the pharmacokinetics of atorvastatin in healthy volunteers. Clin. Pharmacol. Ther. 81, 194–204 (2007).  [DOI](https://doi.org/10.1038/sj.clpt.6100038) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17192770/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Effect%20of%20OATP1B%20transporter%20inhibition%20on%20the%20pharmacokinetics%20of%20atorvastatin%20in%20healthy%20volunteers&author=Y.Y.%20Lau&author=Y.%20Huang&author=L.%20Frassetto&author=L.Z.%20Benet&volume=81&publication_year=2007&pages=194-204&pmid=17192770&doi=10.1038/sj.clpt.6100038&)

11. Noe, J. , Portmann, R. , Brun, M.E. & Funk, C. Substrate‐dependent drug‐drug interactions between gemfibrozil, fluvastatin and other organic anion‐transporting peptide (OATP) substrates on OATP1B1, OATP2B1, and OATP1B3. Drug Metab. Dispos. 35, 1308–1314 (2007).  [DOI](https://doi.org/10.1124/dmd.106.012930) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17470528/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Substrate%E2%80%90dependent%20drug%E2%80%90drug%20interactions%20between%20gemfibrozil,%20fluvastatin%20and%20other%20organic%20anion%E2%80%90transporting%20peptide%20(OATP)%20substrates%20on%20OATP1B1,%20OATP2B1,%20and%20OATP1B3&author=J.%20Noe&author=R.%20Portmann&author=M.E.%20Brun&author=C.%20Funk&volume=35&publication_year=2007&pages=1308-1314&pmid=17470528&doi=10.1124/dmd.106.012930&)

12. Niemi, M. et al High plasma pravastatin concentrations are associated with single nucleotide polymorphisms and haplotypes of organic anion transporting polypeptide‐C (OATP‐C, SLCO1B1). Pharmacogenetics 14, 429–440 (2004).  [DOI](https://doi.org/10.1097/01.fpc.0000114750.08559.32) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15226675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=High%20plasma%20pravastatin%20concentrations%20are%20associated%20with%20single%20nucleotide%20polymorphisms%20and%20haplotypes%20of%20organic%20anion%20transporting%20polypeptide%E2%80%90C%20(OATP%E2%80%90C,%20SLCO1B1)&author=M.%20Niemi&volume=14&publication_year=2004&pages=429-440&pmid=15226675&doi=10.1097/01.fpc.0000114750.08559.32&)

13. Pasanen, M.K. , Neuvonen, M. , Neuvonen, P.J. & Niemi, M. SLCO1B1 polymorphism markedly affects the pharmacokinetics of simvastatin acid. Pharmacogenet. Genom. 16, 873–879 (2006).  [DOI](https://doi.org/10.1097/01.fpc.0000230416.82349.90) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17108811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=SLCO1B1%20polymorphism%20markedly%20affects%20the%20pharmacokinetics%20of%20simvastatin%20acid&author=M.K.%20Pasanen&author=M.%20Neuvonen&author=P.J.%20Neuvonen&author=M.%20Niemi&volume=16&publication_year=2006&pages=873-879&pmid=17108811&doi=10.1097/01.fpc.0000230416.82349.90&)

14. Puccetti, L. , Scarpini, F. , Cappellone, R. & Auteri, A. Genetic influence in statin intolerance. Clin. Pharmacol. Ther. 90, 365 (2011).  [DOI](https://doi.org/10.1038/clpt.2011.114) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21753748/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genetic%20influence%20in%20statin%20intolerance&author=L.%20Puccetti&author=F.%20Scarpini&author=R.%20Cappellone&author=A.%20Auteri&volume=90&publication_year=2011&pages=365&pmid=21753748&doi=10.1038/clpt.2011.114&)

15. Isackson, P.J. et al Association of common variants in the human eyes shut ortholog (EYS) with statin‐induced myopathy: evidence for additional functions of EYS. Muscle Nerve 44, 531–538 (2011).  [DOI](https://doi.org/10.1002/mus.22115) | [PMC free article](/articles/PMC3175321/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21826682/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Muscle%20Nerve&title=Association%20of%20common%20variants%20in%20the%20human%20eyes%20shut%20ortholog%20(EYS)%20with%20statin%E2%80%90induced%20myopathy:%20evidence%20for%20additional%20functions%20of%20EYS&author=P.J.%20Isackson&volume=44&publication_year=2011&pages=531-538&pmid=21826682&doi=10.1002/mus.22115&)

16. Mangravite, L.M. et al A statin‐dependent QTL for GATM expression is associated with statin‐induced myopathy. Nature 502, 337–380 (2013).  [DOI](https://doi.org/10.1038/nature12508) | [PMC free article](/articles/PMC3933266/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23995691/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20statin%E2%80%90dependent%20QTL%20for%20GATM%20expression%20is%20associated%20with%20statin%E2%80%90induced%20myopathy&author=L.M.%20Mangravite&volume=502&publication_year=2013&pages=337-380&pmid=23995691&doi=10.1038/nature12508&)

17. Carr, D.F. , Alfirevic, A. , Johnson, R. , Chinoy, H. , van Staa, T. & Pirmohamed, M. GATM gene variants and statin myopathy risk. Nature 513, E1 (2014).  [DOI](https://doi.org/10.1038/nature13628) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25230669/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=GATM%20gene%20variants%20and%20statin%20myopathy%20risk&author=D.F.%20Carr&author=A.%20Alfirevic&author=R.%20Johnson&author=H.%20Chinoy&author=T.%20van%20Staa&volume=513&publication_year=2014&pages=E1&pmid=25230669&doi=10.1038/nature13628&)

18. Floyd, J.S. , Bis, J.C. , Brody, J.A. , Heckbert, S.R. , Rice, K. & Psaty, B.M. GATM locus does not replicate in rhabdomyolysis study. Nature 513, E1–E3 (2014).  [DOI](https://doi.org/10.1038/nature13629) | [PMC free article](/articles/PMC4230441/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25230668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=GATM%20locus%20does%20not%20replicate%20in%20rhabdomyolysis%20study&author=J.S.%20Floyd&author=J.C.%20Bis&author=J.A.%20Brody&author=S.R.%20Heckbert&author=K.%20Rice&volume=513&publication_year=2014&pages=E1-E3&pmid=25230668&doi=10.1038/nature13629&)

19. O'Meara, H. et al Electronic health records for biological sample collection: feasibility study of statin‐induced myopathy using the clinical practice research datalink. Br. J. Clin. Pharmacol. 77, 831–838 (2013).  [DOI](https://doi.org/10.1111/bcp.12269) | [PMC free article](/articles/PMC4004403/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24308359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Electronic%20health%20records%20for%20biological%20sample%20collection:%20feasibility%20study%20of%20statin%E2%80%90induced%20myopathy%20using%20the%20clinical%20practice%20research%20datalink&author=H.%20O'Meara&volume=77&publication_year=2013&pages=831-838&pmid=24308359&doi=10.1111/bcp.12269&)

20. Bakar, N.S. , Neely, D. , Avery, P. , Brown, C. , Daly, A.K. & Kamali, F. Genetic and clinical factors are associated with statin‐related myotoxicity of moderate severity: a case‐control study. Clin. Pharmacol. Ther. 104, 178–187 (2018).  [DOI](https://doi.org/10.1002/cpt.887) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28940218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genetic%20and%20clinical%20factors%20are%20associated%20with%20statin%E2%80%90related%20myotoxicity%20of%20moderate%20severity:%20a%20case%E2%80%90control%20study&author=N.S.%20Bakar&author=D.%20Neely&author=P.%20Avery&author=C.%20Brown&author=A.K.%20Daly&volume=104&publication_year=2018&pages=178-187&pmid=28940218&doi=10.1002/cpt.887&)

21. Xiang, Q. et al Association between SLCO1B1 T521C polymorphism and risk of statin‐induced myopathy: a meta‐analysis. Pharmacogenom. J. 18, 721–729 (2018).  [DOI](https://doi.org/10.1038/s41397-018-0054-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30250148/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom.%20J.&title=Association%20between%20SLCO1B1%20T521C%20polymorphism%20and%20risk%20of%20statin%E2%80%90induced%20myopathy:%20a%20meta%E2%80%90analysis&author=Q.%20Xiang&volume=18&publication_year=2018&pages=721-729&pmid=30250148&doi=10.1038/s41397-018-0054-0&)

22. Pasanen, M.K. , Fredrikson, H. , Neuvonen, P.J. & Niemi, M. Different effects of SLCO1B1 polymorphism on the pharmacokinetics of atorvastatin and rosuvastatin. Clin. Pharmacol. Ther. 82, 726–733 (2007).  [DOI](https://doi.org/10.1038/sj.clpt.6100220) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17473846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Different%20effects%20of%20SLCO1B1%20polymorphism%20on%20the%20pharmacokinetics%20of%20atorvastatin%20and%20rosuvastatin&author=M.K.%20Pasanen&author=H.%20Fredrikson&author=P.J.%20Neuvonen&author=M.%20Niemi&volume=82&publication_year=2007&pages=726-733&pmid=17473846&doi=10.1038/sj.clpt.6100220&)

23. Niemi, M. , Pasanen, M.K. & Neuvonen, P.J. SLCO1B1 polymorphism and sex affect the pharmacokinetics of pravastatin but not fluvastatin. Clin. Pharmacol. Ther. 80, 356–366 (2006).  [DOI](https://doi.org/10.1016/j.clpt.2006.06.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17015053/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=SLCO1B1%20polymorphism%20and%20sex%20affect%20the%20pharmacokinetics%20of%20pravastatin%20but%20not%20fluvastatin&author=M.%20Niemi&author=M.K.%20Pasanen&author=P.J.%20Neuvonen&volume=80&publication_year=2006&pages=356-366&pmid=17015053&doi=10.1016/j.clpt.2006.06.010&)

24. Tornio, A. , Vakkilainen, J. , Neuvonen, M. , Backman, J.T. , Neuvonen, P.J. & Niemi, M. SLCO1B1 polymorphism markedly affects the pharmacokinetics of lovastatin acid. Pharmacogenet. Genom. 25, 382–387 (2015).  [DOI](https://doi.org/10.1097/FPC.0000000000000148) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26020121/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=SLCO1B1%20polymorphism%20markedly%20affects%20the%20pharmacokinetics%20of%20lovastatin%20acid&author=A.%20Tornio&author=J.%20Vakkilainen&author=M.%20Neuvonen&author=J.T.%20Backman&author=P.J.%20Neuvonen&volume=25&publication_year=2015&pages=382-387&pmid=26020121&doi=10.1097/FPC.0000000000000148&)

25. Deng, J.W. et al The effect of SLCO1B1*15 on the disposition of pravastatin and pitavastatin is substrate dependent: the contribution of transporting activity changes by SLCO1B1*15. Pharmacogenet. Genom. 18, 424–433 (2008).  [DOI](https://doi.org/10.1097/FPC.0b013e3282fb02a3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18408565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=The%20effect%20of%20SLCO1B1*15%20on%20the%20disposition%20of%20pravastatin%20and%20pitavastatin%20is%20substrate%20dependent:%20the%20contribution%20of%20transporting%20activity%20changes%20by%20SLCO1B1*15&author=J.W.%20Deng&volume=18&publication_year=2008&pages=424-433&pmid=18408565&doi=10.1097/FPC.0b013e3282fb02a3&)

26. Kalliokoski, A. & Niemi, M. Impact of OATP transporters on pharmacokinetics. Br. J. Pharmacol. 158, 693–705 (2009).  [DOI](https://doi.org/10.1111/j.1476-5381.2009.00430.x) | [PMC free article](/articles/PMC2765590/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19785645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Pharmacol.&title=Impact%20of%20OATP%20transporters%20on%20pharmacokinetics&author=A.%20Kalliokoski&author=M.%20Niemi&volume=158&publication_year=2009&pages=693-705&pmid=19785645&doi=10.1111/j.1476-5381.2009.00430.x&)

27. Franke, R.M. , Scherkenbach, L.A. & Sparreboom, A. Pharmacogenetics of the organic anion transporting polypeptide 1A2. Pharmacogenomics 10, 339–344 (2009).  [DOI](https://doi.org/10.2217/14622416.10.3.339) | [PMC free article](/articles/PMC4865796/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19290786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetics%20of%20the%20organic%20anion%20transporting%20polypeptide%201A2&author=R.M.%20Franke&author=L.A.%20Scherkenbach&author=A.%20Sparreboom&volume=10&publication_year=2009&pages=339-344&pmid=19290786&doi=10.2217/14622416.10.3.339&)

28. Kullak‐Ublick, G.A. et al Molecular and functional characterization of an organic anion transporting polypeptide cloned from human liver. Gastroenterology 109, 1274–1282 (1995).  [DOI](https://doi.org/10.1016/0016-5085(95)90588-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7557095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Molecular%20and%20functional%20characterization%20of%20an%20organic%20anion%20transporting%20polypeptide%20cloned%20from%20human%20liver&author=G.A.%20Kullak%E2%80%90Ublick&volume=109&publication_year=1995&pages=1274-1282&pmid=7557095&doi=10.1016/0016-5085(95)90588-x&)

29. Shirasaka, Y. , Suzuki, K. , Shichiri, M. , Nakanishi, T. & Tamai, I. Intestinal absorption of HMG‐CoA reductase inhibitor pitavastatin mediated by organic anion transporting polypeptide and P‐glycoprotein/multidrug resistance 1. Drug Metab. Pharmacokinet. 26, 171–179 (2011).  [DOI](https://doi.org/10.2133/dmpk.dmpk-10-rg-073) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21206133/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Pharmacokinet.&title=Intestinal%20absorption%20of%20HMG%E2%80%90CoA%20reductase%20inhibitor%20pitavastatin%20mediated%20by%20organic%20anion%20transporting%20polypeptide%20and%20P%E2%80%90glycoprotein/multidrug%20resistance%201&author=Y.%20Shirasaka&author=K.%20Suzuki&author=M.%20Shichiri&author=T.%20Nakanishi&author=I.%20Tamai&volume=26&publication_year=2011&pages=171-179&pmid=21206133&doi=10.2133/dmpk.dmpk-10-rg-073&)

30. Liu, H. et al Solute carrier family of the organic anion‐transporting polypeptides 1A2‐Madin‐darby canine kidney II: a promising in vitro system to understand the role of organic anion‐transporting polypeptide 1A2 in blood‐brain barrier drug penetration. Drug Metab. Dispos. 43, 1008–1018 (2015).  [DOI](https://doi.org/10.1124/dmd.115.064170) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25908246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Solute%20carrier%20family%20of%20the%20organic%20anion%E2%80%90transporting%20polypeptides%201A2%E2%80%90Madin%E2%80%90darby%20canine%20kidney%20II:%20a%20promising%20in%20vitro%20system%20to%20understand%20the%20role%20of%20organic%20anion%E2%80%90transporting%20polypeptide%201A2%20in%20blood%E2%80%90brain%20barrier%20drug%20penetration&author=H.%20Liu&volume=43&publication_year=2015&pages=1008-1018&pmid=25908246&doi=10.1124/dmd.115.064170&)

31. Medicines.org.uk . Atorvastatin 10 mg film‐coated tablets ‐ Summary of Product Characteristics (SPC). <http://www.medicines.org.uk/emc/product/2959> (2016). Accessed August 21, 2018.  [http://www.medicines.org.uk/emc/product/2959](http://www.medicines.org.uk/emc/product/2959)

32. Medicines.org.uk . Simvastatin 40 mg – Summary of Product Characteristics (SPC). <http://www.medicines.org.uk/emc/product/5688/> (2018). Accessed August 21, 2018.  [http://www.medicines.org.uk/emc/product/5688/](http://www.medicines.org.uk/emc/product/5688/)

33. Niemi, M. Transporter pharmacogenetics and statin toxicity. Clin. Pharmacol. Ther. 87, 130–133 (2010).  [DOI](https://doi.org/10.1038/clpt.2009.197) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19890253/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Transporter%20pharmacogenetics%20and%20statin%20toxicity&author=M.%20Niemi&volume=87&publication_year=2010&pages=130-133&pmid=19890253&doi=10.1038/clpt.2009.197&)

34. Peyser, B. et al Effects of delivering SLCO1B1 pharmacogenetic information in randomized trial and observational settings. Circ. Genom. Precis. Med. 11, e002228 (2018).  [DOI](https://doi.org/10.1161/CIRCGEN.118.002228) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30354330/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Genom.%20Precis.%20Med.&title=Effects%20of%20delivering%20SLCO1B1%20pharmacogenetic%20information%20in%20randomized%20trial%20and%20observational%20settings&author=B.%20Peyser&volume=11&publication_year=2018&pages=e002228&pmid=30354330&doi=10.1161/CIRCGEN.118.002228&)

35. van Staa, T.P. , Carr, D.F. , O'Meara, H. , McCann, G. & Pirmohamed, M. Predictors and outcomes of increases in creatine kinase levels or rhabdomyolysis risk during statin treatment. Br. J. Clin. Pharmacol. 78, 649–659 (2014).  [DOI](https://doi.org/10.1111/bcp.12367) | [PMC free article](/articles/PMC4243915/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24602118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Predictors%20and%20outcomes%20of%20increases%20in%20creatine%20kinase%20levels%20or%20rhabdomyolysis%20risk%20during%20statin%20treatment&author=T.P.%20van%20Staa&author=D.F.%20Carr&author=H.%20O'Meara&author=G.%20McCann&author=M.%20Pirmohamed&volume=78&publication_year=2014&pages=649-659&pmid=24602118&doi=10.1111/bcp.12367&)

36. Rothwell, S. et al Dense genotyping of immune‐related loci in idiopathic inflammatory myopathies confirms HLA alleles as the strongest genetic risk factor and suggests different genetic background for major clinical subgroups. Ann. Rheum. Dis. 75, 1558–1566 (2016).  [DOI](https://doi.org/10.1136/annrheumdis-2015-208119) | [PMC free article](/articles/PMC5300750/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26362759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Rheum.%20Dis.&title=Dense%20genotyping%20of%20immune%E2%80%90related%20loci%20in%20idiopathic%20inflammatory%20myopathies%20confirms%20HLA%20alleles%20as%20the%20strongest%20genetic%20risk%20factor%20and%20suggests%20different%20genetic%20background%20for%20major%20clinical%20subgroups&author=S.%20Rothwell&volume=75&publication_year=2016&pages=1558-1566&pmid=26362759&doi=10.1136/annrheumdis-2015-208119&)

37. Molokhia, M. & McKeigue, P. EUDRAGENE: European collaboration to establish a case‐control DNA collection for studying the genetic basis of adverse drug reactions. Pharmacogenomics 7, 633–638 (2006).  [DOI](https://doi.org/10.2217/14622416.7.4.633) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16753010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=EUDRAGENE:%20European%20collaboration%20to%20establish%20a%20case%E2%80%90control%20DNA%20collection%20for%20studying%20the%20genetic%20basis%20of%20adverse%20drug%20reactions&author=M.%20Molokhia&author=P.%20McKeigue&volume=7&publication_year=2006&pages=633-638&pmid=16753010&doi=10.2217/14622416.7.4.633&)

38. Hopewell, J.C. et al Impact of common genetic variation on response to simvastatin therapy among 18 705 participants in the Heart Protection Study. Eur. Heart J. 34, 982–992 (2013).  [DOI](https://doi.org/10.1093/eurheartj/ehs344) | [PMC free article](/articles/PMC3612775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23100282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Heart%20J.&title=Impact%20of%20common%20genetic%20variation%20on%20response%20to%20simvastatin%20therapy%20among%2018%20705%20participants%20in%20the%20Heart%20Protection%20Study&author=J.C.%20Hopewell&volume=34&publication_year=2013&pages=982-992&pmid=23100282&doi=10.1093/eurheartj/ehs344&)

39. O'Meara, H. et al Electronic health records for biological sample collection: feasibility study of statin‐induced myopathy using the Clinical Practice Research Datalink. Br. J. Clin. Pharmacol. 77, 831–838 (2014).  [DOI](https://doi.org/10.1111/bcp.12269) | [PMC free article](/articles/PMC4004403/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24308359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Electronic%20health%20records%20for%20biological%20sample%20collection:%20feasibility%20study%20of%20statin%E2%80%90induced%20myopathy%20using%20the%20Clinical%20Practice%20Research%20Datalink&author=H.%20O'Meara&volume=77&publication_year=2014&pages=831-838&pmid=24308359&doi=10.1111/bcp.12269&)

40. Purcell, S. et al PLINK: a tool set for whole‐genome association and population‐based linkage analyses. Am. J. Hum. Genet. 81, 559–575 (2007).  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=PLINK:%20a%20tool%20set%20for%20whole%E2%80%90genome%20association%20and%20population%E2%80%90based%20linkage%20analyses&author=S.%20Purcell&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

41. Zheng, X. , Levine, D. , Shen, J. , Gogarten, S.M. , Laurie, C. & Weir, B.S. A high‐performance computing toolset for relatedness and principal component analysis of SNP data. Bioinformatics 28, 3326–3328 (2012).  [DOI](https://doi.org/10.1093/bioinformatics/bts606) | [PMC free article](/articles/PMC3519454/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23060615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=A%20high%E2%80%90performance%20computing%20toolset%20for%20relatedness%20and%20principal%20component%20analysis%20of%20SNP%20data&author=X.%20Zheng&author=D.%20Levine&author=J.%20Shen&author=S.M.%20Gogarten&author=C.%20Laurie&volume=28&publication_year=2012&pages=3326-3328&pmid=23060615&doi=10.1093/bioinformatics/bts606&)

42. Delaneau, O. , Marchini, J. & Zagury, J.F. A linear complexity phasing method for thousands of genomes. Nat. Methods 9, 179–181 (2012).  [DOI](https://doi.org/10.1038/nmeth.1785) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22138821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&title=A%20linear%20complexity%20phasing%20method%20for%20thousands%20of%20genomes&author=O.%20Delaneau&author=J.%20Marchini&author=J.F.%20Zagury&volume=9&publication_year=2012&pages=179-181&pmid=22138821&doi=10.1038/nmeth.1785&)

43. Howie, B. , Marchini, J. & Stephens, M. Genotype imputation with thousands of genomes. G3 (Bethesda) 1, 457–470 (2011).  [DOI](https://doi.org/10.1534/g3.111.001198) | [PMC free article](/articles/PMC3276165/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22384356/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=G3%20(Bethesda)&title=Genotype%20imputation%20with%20thousands%20of%20genomes&author=B.%20Howie&author=J.%20Marchini&author=M.%20Stephens&volume=1&publication_year=2011&pages=457-470&pmid=22384356&doi=10.1534/g3.111.001198&)

44. Howie, B.N. , Donnelly, P. & Marchini, J. A flexible and accurate genotype imputation method for the next generation of genome‐wide association studies. PLoS Genet. 5, e1000529 (2009).  [DOI](https://doi.org/10.1371/journal.pgen.1000529) | [PMC free article](/articles/PMC2689936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19543373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=A%20flexible%20and%20accurate%20genotype%20imputation%20method%20for%20the%20next%20generation%20of%20genome%E2%80%90wide%20association%20studies&author=B.N.%20Howie&author=P.%20Donnelly&author=J.%20Marchini&volume=5&publication_year=2009&pages=e1000529&pmid=19543373&doi=10.1371/journal.pgen.1000529&)

45. Howie, B. , Fuchsberger, C. , Stephens, M. , Marchini, J. & Abecasis, G.R. Fast and accurate genotype imputation in genome‐wide association studies through pre‐phasing. Nat. Genet. 44, 955–959 (2012).  [DOI](https://doi.org/10.1038/ng.2354) | [PMC free article](/articles/PMC3696580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22820512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Fast%20and%20accurate%20genotype%20imputation%20in%20genome%E2%80%90wide%20association%20studies%20through%20pre%E2%80%90phasing&author=B.%20Howie&author=C.%20Fuchsberger&author=M.%20Stephens&author=J.%20Marchini&author=G.R.%20Abecasis&volume=44&publication_year=2012&pages=955-959&pmid=22820512&doi=10.1038/ng.2354&)

46. Marchini, J. , Howie, B. , Myers, S. , McVean, G. & Donnelly, P. A new multipoint method for genome‐wide association studies by imputation of genotypes. Nat. Genet. 39, 906–913 (2007).  [DOI](https://doi.org/10.1038/ng2088) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17572673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=A%20new%20multipoint%20method%20for%20genome%E2%80%90wide%20association%20studies%20by%20imputation%20of%20genotypes&author=J.%20Marchini&author=B.%20Howie&author=S.%20Myers&author=G.%20McVean&author=P.%20Donnelly&volume=39&publication_year=2007&pages=906-913&pmid=17572673&doi=10.1038/ng2088&)

47. Magi, R. & Morris, A.P. GWAMA: software for genome‐wide association meta‐analysis. BMC Bioinform. 11, 288 (2010).  [DOI](https://doi.org/10.1186/1471-2105-11-288) | [PMC free article](/articles/PMC2893603/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20509871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinform.&title=GWAMA:%20software%20for%20genome%E2%80%90wide%20association%20meta%E2%80%90analysis&author=R.%20Magi&author=A.P.%20Morris&volume=11&publication_year=2010&pages=288&pmid=20509871&doi=10.1186/1471-2105-11-288&)
