# Evaluation of the Effect of CYP2D6 Genotypes on Tramadol and O-Desmethyltramadol Pharmacokinetic Profiles in a Korean Population Using Physiologically-Based Pharmacokinetic Modeling

## Metadata
**Authors:** Hyeon-Cheol Jeong, Soo Hyeon Bae, Jung-Woo Bae, Sooyeun Lee, Anhye Kim, Yoojeong Jang, Kwang-Hee Shin
**Journal:** Pharmaceutics
**Date:** 2019 Nov 17
**DOI:** [10.3390/pharmaceutics11110618](https://doi.org/10.3390/pharmaceutics11110618)
**PMID:** 31744222
**PMCID:** PMC6920759
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6920759/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6920759/pdf/pharmaceutics-11-00618.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6920759/pdf/pharmaceutics-11-00618.pdf)

## Abstract

Tramadol is a μ-opioid receptor agonist and a monoamine reuptake inhibitor. O-desmethyltramadol (M1), the major active metabolite of tramadol, is produced by CYP2D6. A physiologically-based pharmacokinetic model was developed to predict changes in time-concentration profiles for tramadol and M1 according to dosage and CYP2D6 genotypes in the Korean population. Parallel artificial membrane permeation assay was performed to determine tramadol permeability, and the metabolic clearance of M1 was determined using human liver microsomes. Clinical study data were used to develop the model. Other physicochemical and pharmacokinetic parameters were obtained from the literature. Simulations for plasma concentrations of tramadol and M1 (after 100 mg tramadol was administered five times at 12-h intervals) were based on a total of 1000 virtual healthy Koreans using SimCYP® simulator. Geometric mean ratios (90% confidence intervals) (predicted/observed) for maximum plasma concentration at steady-state (Cmax,ss) and area under the curve at steady-state (AUClast,ss) were 0.79 (0.69–0.91) and 1.04 (0.85–1.28) for tramadol, and 0.63 (0.51–0.79) and 0.67 (0.54–0.84) for M1, respectively. The predicted time–concentration profiles of tramadol fitted well to observed profiles and those of M1 showed under-prediction. The developed model could be applied to predict concentration-dependent toxicities according to CYP2D6 genotypes and also, CYP2D6-related drug interactions.

Keywords: CYP2D6, O-desmethyltramadol, pharmacokinetics, physiologically-based pharmacokinetics, tramadol

## 1. Introduction

Tramadol is an orally available, centrally acting, weak-opioid analgesic drug [[1](#B1-pharmaceutics-11-00618)]. The anti-nociceptive effect of tramadol is due to two mechanisms: an opioid mechanism, and a non-opioid mechanism [[2](#B2-pharmaceutics-11-00618)]. Tramadol acts as a μ-opioid receptor agonist, like traditional opioids. It also has analgesic effects by inhibiting reuptake of monoamine neurotransmitters, such as 5-hydroxytryptamine (5-HT) and noradrenaline [[3](#B3-pharmaceutics-11-00618),[4](#B4-pharmaceutics-11-00618)]. These mechanisms lead to reduced pain signal conduction and analgesic effects. Tramadol is a racemate, and analgesic mechanisms differ depending on the isomer: (−)-tramadol exhibited ≈ 10-fold higher noradrenaline reuptake inhibitory activity than (+)-tramadol, and (+)-tramadol showed ≈ 4-fold higher 5-HT reuptake inhibitory activity than (−)-tramadol [[5](#B5-pharmaceutics-11-00618)].

Tramadol is predominantly metabolized in the liver. Approximately 10%–30% of administered tramadol is excreted, unmetabolized, in the urine. The well-known metabolic pathway of tramadol is divided into three major pathways: *O*-desmethyltramadol (M1) is produced by cytochrome P450 (CYP) 2D6, and *N*-desmethyltramadol (M2) is produced by CYP2B6 and CYP3A4. These metabolites are converted to either *N*,*O*-didesmethyltramadol (M5) and other inactive metabolites by CYP2D6, CYP2B6, and CYP3A4, or converted to glucuronides by UDP-glucuronosyltransferase (UGT) 1A8 and UGT2B7 [[6](#B6-pharmaceutics-11-00618)]. M1, a major active metabolite of tramadol, has about 700-fold higher affinity for μ-opioid receptors than tramadol [[7](#B7-pharmaceutics-11-00618)]. M5 also has 24-fold higher affinity for μ-opioid receptors than tramadol [[3](#B3-pharmaceutics-11-00618)]. Tramadol is mainly considered to inhibit monoamine reuptake, while M1 and M5 bind to μ-opioid receptors and exhibit analgesic effects. Thus, genetic polymorphism of CYP2D6 could have an effect on the risk of adverse events during tramadol administration [[4](#B4-pharmaceutics-11-00618)].

A physiologically-based pharmacokinetic (PBPK) approach is a bottom-up approach that requires data about the physicochemical properties and pharmacokinetic (PK) parameters (i.e., absorption, distribution, metabolism, and excretion; ADME) of the target drug [[8](#B8-pharmaceutics-11-00618),[9](#B9-pharmaceutics-11-00618)]. In addition, the PBPK model considers bodyweight, height, organ volume, blood flow, and inter-individual variation for metabolizing enzymes and transporters [[10](#B10-pharmaceutics-11-00618)]. Therefore, the PBPK model can be used to predict plasma concentration–time profiles more closely than traditional compartmental PK models [[11](#B11-pharmaceutics-11-00618)]. In this regard, PBPK modeling can predict human PK profiles using in vitro or preclinical study data from drug development. Further, such modeling is also used to investigate the interaction potential with other drugs or food, and to predict PK profiles in special populations, such as pregnant women, geriatric patients, or children [[9](#B9-pharmaceutics-11-00618),[12](#B12-pharmaceutics-11-00618)].

Studies predicting the PK profile of tramadol using PK modeling have been reported previously. Many articles used a population PK approach with nonlinear mixed-effects modeling (NONMEM), however a PBPK approach was rarely used to predict the PK profile of tramadol [[13](#B13-pharmaceutics-11-00618),[14](#B14-pharmaceutics-11-00618),[15](#B15-pharmaceutics-11-00618)]. When tramadol was administered, M1 also had impact on efficacy and toxicity [[16](#B16-pharmaceutics-11-00618)]. Therefore, M1 should be integrated for PBPK model of tramadol for better interpretation.

The aims of this study were to develop a PBPK model that could predict the concentration–time profiles of tramadol and M1 in Koreans, and to investigate effects of the CYP2D6 genotype on PK profiles at routinely administered doses. The developed PBPK model was applied to predict the effects of CYP2D6 genotype and tramadol dosage on the plasma concentration profiles of tramadol and M1 in a healthy Korean population.

## 2. Materials and Methods

### 2.1. Clinical Study Design

The clinical study was approved by the Institutional Review Board of Keimyung University (Deagu, Republic of Korea, approval number: 40525-201509-BR-70-02, 23 February 2016) and Kyungpook National University Hospital (Daegu, Republic of Korea, approval number of clinical trial: 2016-08-005, 24 August 2016) and carried out at the Kyungpook National University Hospital Clinical Trial Center (Daegu, Republic of Korea). A total of 23 subjects participated who voluntarily agreed to take part in the clinical study and signed their informed consent. Subject characteristics are presented in [Table 1](#pharmaceutics-11-00618-t001). All subjects received a 100-mg tramadol hydrochloride tablet (Tridol^®^ extended-release (ER); Yuhan Pharmaceutical, Seoul, Korea) five times at 12-h intervals. Whole blood was collected in an anticoagulant tube at pre-dose, and at 0.5, 1, 1.5, 2, 2.5, 4, 6, 8, 10, 12, 24, 48, and 72 h after administration. The obtained whole blood was used for CYP2D6 genotyping, and the plasma was separated for determination of tramadol and M1 [[17](#B17-pharmaceutics-11-00618)].

### Table 1.

| Characteristic | Mean (SD) |
| --- | --- |
| Age (years) | 24.78 (4.80) |
| Height (cm) | 176.51 (5.64) |
| Weight (kg) | 71.61 (8.87) |
| CYP2D6 genotypes (no. of subjects) |   |
| Wild-type | 14 |
| *5/*5 | 1 |
| *10/*10 | 8 |

Table 1 Caption: Demographic characteristics (n = 23).

### 2.2. CYP2D6 Genotyping

The determination of CYP2D6 genotype was performed for *CYP2D6*2* (normal function), *CYP2D6*5* (no function), and *CYP2D6*10* (decreased function). Genotyping for *CYP2D6*2* and *CYP2D6*10* was performed using pyrosequencing. *CYP2D6*5* was sequenced by long polymerase chain reaction (PCR) because of deletion of a specific sequence. Pyrosequencing was performed using Pyromark Q96 ID and Pyromark Gold Q96 reagents (Qiagen, Hilden, Germany). The conditions of PCR (total 35 cycles) were: denaturation (94 °C for 30 s), annealing (56 °C for 30 s), and polymerization (72 °C for 30 s). The processes were finished by extension at 72 °C for 5 min. *CYP2D6*5* and duplication were determined by long-PCR, as previously described [[18](#B18-pharmaceutics-11-00618),[19](#B19-pharmaceutics-11-00618)]. The *CYP2D6* phenotype was determined based on genotype and activity score [[20](#B20-pharmaceutics-11-00618),[21](#B21-pharmaceutics-11-00618),[22](#B22-pharmaceutics-11-00618),[23](#B23-pharmaceutics-11-00618)]. *CYP2D6**5 was a non-functional allele and homozygous *CYP2D6**5 was classified as a poor metabolizer (PM). Homozygous *CYP2D6**10 was classified as an intermediate metabolizer (IM).

### 2.3. Determination of Tramadol and O-Desmethyltramadol (M1) Using LC-MS/MS

The obtained whole blood samples were immediately centrifuged at 4 °C, 3000 rpm for 10 min. The isolated plasma samples were transferred to a new microcentrifuge tube and kept at −70 °C until analysis. The plasma samples were completely thawed, then 10 μL of internal standard (tramadol ^13^C-d_3_ for tramadol, and M1-d_6_ for M1) were added to 100 μL samples and mixed briefly. A total of 300 μL of acetonitrile was added, and then samples were mixed thoroughly for 30 s prior to centrifugation at 2500 rpm for 10 min. The organic solvent layer was transferred to a new polypropylene tube and evaporated under nitrogen gas at 40 °C. Methanol 200 μL was added into tubes containing pellets for reconstitution, and 5 μL of reconstituted sample were analyzed. Analyses were carried out on API3200 tandem mass spectrometry system (AB SCIEX, Framingham, MA, USA) equipped with an Agilent 1260 series HPLC system (Agilent Technologies, Santa Clara, CA, USA). Separation of tramadol and M1 was conducted using a Luna C18 column (5.0 μm, 2.0 × 50 mm; Phenomenex, Torrance, CA, USA). Five millimoles ammonium formate and 0.1% formic acid in methanol (A), and 5 mM ammonium formate solution (B), were used for the mobile phase. The used gradient method was as follows: 0–2 min (97%–5% B), 2–4 min (5% B), 4–5 min (5%–97% B), and 5–8 min (97% B). Electrospray ionization-positive ion mode was used for mass detection. The mass transitions (m/z) used were 264.2→58.1 for tramadol, 268.2→58.1 for tramadol internal standard (IS), 250.2→58.2 for M1, and 256.2→64.1 for M1 IS. To obtain pharmacokinetic parameters, non-compartmental analysis (NCA) was performed using Phoenix (Certara Inc., Princeton, NJ, USA).

### 2.4. Parallel Artificial Membrane Permeability Assay (PAMPA)

To determine the permeability of tramadol, a parallel artificial membrane permeability assay (PAMPA) was performed [[24](#B24-pharmaceutics-11-00618)]. Gentest™ Pre-coated PAMPA Plate System (Corning, Tewksbury, MA, USA) was used for the permeability assay. All the processes were followed according to the manufacturer’s protocol. Tramadol hydrochloride, dimethyl sulfoxide (DMSO), phosphate-buffered saline (PBS), and acetonitrile were purchased from Sigma-Aldrich (St. Louis, MO, USA). Tramadol stock solution (1 mM) was prepared using 100% DMSO and diluted to 15 μM using PBS (pH 7.4). The PAMPA plate was equilibrated for 30 min at room temperature before performing the permeability assay. PBS 200 μL was dispensed on the acceptor side and 300 μL of working solution was dispensed on the donor side. Incubation was carried out at room temperature for 5 h, and the acceptor and donor side buffers were analyzed using liquid chromatography-tandem mass spectrometry (LC-MS/MS). A total of 12 replicated samples were assayed and mean permeability was calculated and applied to the model.

### 2.5. Assessment of Intrinsic Clearance of M1

For the experiments, *O*-desmethyltramadol HCl (M1), glucose 6-phosphate, glucose 6-phosphate dehydrogenase, MgCl_2_, β-nicotinamide adenine dinucleotide phosphate (NADP), chlorpropamide, Trizma^®^ base, Trizma^®^ hydrochloride, DMSO and formic acid were obtained from Sigma‒Aldrich (St. Louis, MO, USA). To evaluate the intrinsic clearance of M1 by CYPs, metabolic stability studies under NADPH system were conducted in human liver microsoms (HLM) [[25](#B25-pharmaceutics-11-00618),[26](#B26-pharmaceutics-11-00618)]. For details, NADPH-generating system (1.3 mM NADP^+^, 3.3 mM glucose 6-phosphate, 3.3 mM MgCl_2_, and 0.4 unit/mL glucose-6-phosphate dehydrogenase) and HLM 0.25 mg/mL were added and preincubated at 37 °C for 5 min. Then, 20 μM M1 was added and reacted at 37 °C for 0, 1, 5, 10, 20, 30, and 40 min, respectively. The total volume of the reaction mixture was 100 μL. After each reaction, the reaction was terminated by addition of 150 μL of acetonitrile containing an internal standard (100 ng/mL chlorpropamide). All experiments were repeated duplicated and the samples were vortexed for 5 min and centrifuged (13,000 rpm, 4 °C) for 15 min. Then, supernatants were injected into LC-MS/MS and M1 were analyzed. The concentration of M1 (20 μM) at 0 min was used to evaluate the metabolism of CYP through the change of drug concentration over time.

### 2.6. Qunatitaion Methods of M1 in In Vitro Experiments Using LC-QTOF

High-performance liquid chromatography (HPLC)-grade acetonitrile and deionized water were obtained from Berdick and Jackson (Muskegon, MI, USA). HLM (50 donors pooled) were purchased from Corning. Analyses were carried out on Sciex QTOF 5600 plus (AB SCIEX, Framingham, MA, USA) equipped with an Agilent 1260 series HPLC system (Agilent Technologies, Santa Clara, CA, USA). For quantitation of M1, the compounds were separated on a Poroshell 120 (4.6 × 50 mm, 2.7 μm; Agilent Technologies) with an isocratic mobile phase consisting of acetonitrile and 0.1% aqueous formic acid (70:30 *v*/*v*) at a flow rate of 0.5 mL/min. The overall run time was 5 min per sample. The column and autosampler temperatures were maintained at 40 °C and 4 °C, respectively. Time-of-flight mass spectrometry analysis was selected in positive ion mode for the sample analysis. The quantitative analytical data were processed using PeakView^®^ (Version 2.2.0; AB SCIEX, Framingham, MA, USA) and MultiQuant^®^ (Version 3.0.2; Framingham, MA, USA), and the formulas C_15_H_23_NO_2_ (M1), and C_10_H_13_ClN_2_O_3_S (chlorpropamide) were used for quantitation.

### 2.7. Development of PBPK Model for Tramadol and M1

PBPK model development was performed using SimCYP^®^ simulator version 18 (Certara, Sheffield, UK). Most of the parameters for tramadol and M1 were entered with reference to the literature. According to previous reports, tramadol was not substrate for P-glycoprotein (P-gp) (ABCB1) and the role of proton-based pumps such as OATP for tramadol permeability was unclear [[27](#B27-pharmaceutics-11-00618),[28](#B28-pharmaceutics-11-00618)]. Therefore, in this study, the permeability of tramadol was determined by PAMPA assay. The advanced drug absorption and metabolism model was used to consider the ER formulation, and the dissolution profiles of Tridol^®^ ER 100 mg (Yuhan Pharmaceutical, Seoul, Korea) were applied. The elimination profile for M1 applied the intrinsic clearance by HLM. Kp scalar of tramadol was set to match the observed Vss and the predicted V_ss_ value in the model, and Kp scalar of M1 was obtained from parameter estimation. In clinical study, the V_ss_ of tramadol was calculated to 2.6 L/kg by non-compartmental analysis. That of M1 was not calculated because the exact dose of M1 is unknown. Intrinsic clearance involved in tramadol metabolism was estimated using retrograde model option, and human liver microsomal intrinsic clearance was applied to M1 library. The renal clearance of tramadol and M1 were applied for the predicted value which is the closest to the observed blood concentration–time profile by parameter estimation. The PBPK model was evaluated so that it could effectively predict PK profiles for tramadol and M1 when observed mean plasma concentrations fitted to the predicted plasma concentration–time profile and its 90% confidence interval (CI). The other evaluation criteria were geometric mean ratio for peak plasma concentration at steady-state (C_max,ss_), and area under the plasma concentration–time curve at last observation at steady-state (AUC_last,ss_), and the 90% CI for these values. If the geometric mean ratio and its 90% CI were within the range 0.7–1.43, the model was considered to fit well.

### 2.8. Prediction of Changes in Concentration–Time Profiles for Tramadol and M1 According to CYP2D6 Genotype and Dosing Regimen

The therapeutic range and toxic range of tramadol and M1 were determined by reference to the literature. Because the manufacturer’s recommended acceptable maximum dose of Tridol^®^ ER was 400 mg per day, the tramadol ER tablet was administered to a virtual healthy Korean population at 100 and 200 mg (5 times at 12-h intervals) to simulate the change of concentration–time profiles for tramadol and M1. This simulation assumes linear PK properties for multiple doses of tramadol 100 mg and 200 mg [[29](#B29-pharmaceutics-11-00618)]. The effect of CYP2D6 genotype was also simulated for tramadol 100 mg and 200 mg for populations consisting of CYP2D6 poor metabolizers (PM), intermediate metabolizers (IM), extensive metabolizers (EM), and ultra-rapid metabolizers (UM).

## 3. Results

### 3.1. Clinical Study

A total of 23 subjects, aged 20–34 years, were enrolled in the clinical study. The average plasma concentration–time profiles for tramadol and M1 are shown in [Figure 1](#pharmaceutics-11-00618-f001). The allele (phenotype) frequencies for *CYP2D6* were: Wild-type (EM, 14 subjects), *10/*10 (IM, 8 subjects), and *5/*5 (PM, 1 subject).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c629/6920759/a7df044e57c6/pharmaceutics-11-00618-g001a.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6920759_pharmaceutics-11-00618-g001a.jpg)

The average plasma concentration–time profiles after five times oral administration (τ = 12 h) of 100 mg tramadol for (a) tramadol and (b) O-desmethyltramadol (M1). Solid blue line, average for all subjects (n = 23); solid black line, CYP2D6 wild-type subjects (n = 14); short dashed black line, CYP2D6 *5/*5 subject (n = 1); long dashed black line, CYP2D6 *10/*10 (n = 8).

### 3.2. Metabolism Assay for O-Desmethyltramadol (M1)

We conducted metabolic stability study of M1 and confirmed that M1 was metabolized mainly by CYPs and partially by UGTs. In control sample without NADPH or HLM, more than 96% of tramadol and M1 remained during the incubation time, indicating that the disappearance of tramadol and M1 were mainly caused by CYP enzymes. Metabolic stability of M1 by CYP was assessed using results of the disappearance test for M1. The slope of linear regression was calculated and intrinsic clearance (CL_int_) of the drug in the in vitro microsome system was calculated:

| CLint,mic (μL/min/mg protein) = k × Vincubation/Cincubation | (1) |
| --- | --- |
where V_incubation_ is incubation volume, C_incubation_ is concentration of microsomal protein.

To apply the elimination profile to *O*-desmethyltramadol (M1), an HLM assay was performed. The M1 disappearance test showed an HLM intrinsic clearance (CL_int,HLM_) of 52.95 µL/min/mg protein ([Figure 2](#pharmaceutics-11-00618-f002)). CL_int,HLM_ for M1 was applied to the M1 PBPK model.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c629/6920759/aee17bcdfd26/pharmaceutics-11-00618-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6920759_pharmaceutics-11-00618-g002.jpg)

The plot of remaining rate of O-desmethyltramadol (M1) after incubation with human liver microsoms (HLM). Each point (obtained by duplicate measurements) represents the mean value. The intrinsic clearance by HLM (CLint,mic) was calculated as 52.92 μL/min/mg protein.

### 3.3. PAMPA Results

Results of the PAMPA assay for 12 tramadol samples diluted to 15 μM showed permeability ranges from 9.14 × 10^−6^ cm/s to 11.5 × 10^−6^ cm/s. The mean ± standard deviation permeability was calculated as 10.4 × 10^−6^ ± 0.056 cm/s. The calculated mean PAMPA permeability was applied to the tramadol PBPK model.

### 3.4. Development of the PBPK Model for Tramadol and M1

The input parameters for tramadol and M1 and demographic characteristics for virtual population in the PBPK model are presented in [Table 2](#pharmaceutics-11-00618-t002) and [Table 3](#pharmaceutics-11-00618-t003). Data for the healthy Korean population were obtained from the Certara repository. Ten virtual trials, including 100 virtual subjects in each virtual trial (total 1000 subjects), were performed for tramadol and M1. In the tramadol model, most of the observation profiles were within the 5th and 95th percentile range, and the predicted mean tramadol concentration in plasma was similar to the observed profile ([Figure 3](#pharmaceutics-11-00618-f003)a). The geometric mean ratios of C_max,ss_ and AUC_last,ss_ for tramadol were 0.79 and 1.04, respectively ([Table 4](#pharmaceutics-11-00618-t004)). Most observed concentration–time profiles were included in the 5th and 95th percentiles of the predicted concentration–time profiles. In addition, the mean predicted plasma tramadol concentration was well fitted to the observed tramadol concentration. In the concentration–time profiles for M1, most of the observations were also within the 5th and 95th percentile range of the predicted profile ([Figure 3](#pharmaceutics-11-00618-f003)b). The range of 90% CI for C_max,ss_ and AUC_last,ss_ were included in the range of 0.7–1.43 (30% range of geometric mean ratio); however, the geometric mean ratio was predicted to be relatively low ([Table 4](#pharmaceutics-11-00618-t004)). Both tramadol and M1 were predominantly distributed in the liver. The second most common distribution sites were the spleen (tramadol) and heart (M1; [Table 5](#pharmaceutics-11-00618-t005)).

### Table 2.

| Parameters | Tramadol |  | M1 * |  |
| --- | --- | --- | --- | --- |
| Value | Source | Value | Source |  |
| Physicochemical properties and blood binding |  |  |  |  |
| Molecular weight (g/mol) | 263.4 | [30] | 249.354 | [31] |
| Log P | 1.35 | [30] | 2.26 | [32] |
| pKa | 9.41 (Monoprotic base) | [30] | 9.62 (Monoprotic base) | [32] |
| fup | 0.8 | [33] | 0.525 | Predicted in SimCYP |
| Absorption |  |  |  |  |
| Absorption type | PAMPA | - | n/a | - |
| Papp (×10−6 cm/s) | 10.2 | Experimental data | n/a | - |
| Distribution |  |  |  |  |
| Kp scalar | 0.946 | Adjusted using Vss | 0.107 | Estimated |
| Vss (L/kg) | 2.6 | Observed data | 0.628 | Estimated |
| Elimination |  |  |  |  |
| CLint (μL/min/pmol or mg protein) | CYP2D6: 0.447;CYP2B6: 0.028;CYP3A4: 0.020 | Retrograde model | 52.92 (WOMC–HLM) | Experimental data |
| CLR | 1.850 | Estimated | 0.481 | Estimated |

Table 2 Caption: Input parameters for tramadol and O-desmethyltramadol (M1) in the physiologically-based pharmacokinetic (PBPK) model.

### Table 3.

| Parameters | Mean (Range) |
| --- | --- |
| Age (years) | 28.9 (20.2–40.0) |
| Height (cm) | 166.2 (147.6–188.0) |
| Weight (kg) | 62.3 (42.9–93.9) |
| The percentage of female | 50% |

Table 3 Caption: The demographic characteristics of the participated subjects for virtual Korean population (n = 1000).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c629/6920759/69ff91dab7a5/pharmaceutics-11-00618-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6920759_pharmaceutics-11-00618-g003.jpg)

The observed (each symbol, n = 23) and simulated mean (solid dark green line) plasma concentration–time profiles after administration of 100 mg tramadol extended-release (ER) tablet twice daily (total five times) for (a) tramadol, and (b) O-desmethyltramadol (M1); blue dashed line represents 5th and 95th percentiles.

### Table 4.

| Parameters | Observed (Range) | Simulated (Range) | Ratio (90% CI) |
| --- | --- | --- | --- |
| Tramadol |  |  |  |
| Geometric mean Cmax,ss (ng/mL) | 643.8; (294.0–942.1) | 508.4; (122.1–1226) | 0.79; (0.69–0.91) |
| Geometric mean AUClast,ss (ng/mL·h) | 8965; (4127–16,038) | 9346; (1217–42,462) | 1.04; (0.85–1.28) |
| M1 |  |  |  |
| Geometric mean Cmax,ss (ng/mL) | 103.8; (29.8–176.7) | 65.68; (1.07–368.1) | 0.63; (0.51–0.79) |
| Geometric mean AUClast,ss (ng/mL·h) | 1775; (445.3–2875) | 1.187; (6.236–7522) | 0.67; (0.54–0.84) |

Table 4 Caption: Observed and simulated pharmacokinetic (PK) parameters for tramadol and O-desmethyltramadol (M1) after oral administration of 100 mg tramadol ER tablet twice daily (five times in total).

### Table 5.

| Organ | Maximum Concentration at Steady-State in Each Organ (ng/mL) |  |
| --- | --- | --- |
| Tramadol | M1 |  |
| Adipose tissue | 400.6 | 12.89 |
| Bone | 899.8 | 23.88 |
| Brain | 1034 | 19.74 |
| Gut | 2741 | 88.38 |
| Heart | 763.0 | 94.97 |
| Kidney | 1452 | 78.46 |
| Liver | 3034 | 157.5 |
| Lung | 991.0 | 21.09 |
| Muscle | 2411 | 79.13 |
| Pancreas | 2000 | 56.25 |
| Skin | 1355 | 42.03 |
| Spleen | 2821 | 88.53 |

Table 5 Caption: Maximum simulated concentrations at steady-state for tramadol and O-desmethyltramadol (M1) in each organ (Cmax,ss) after oral administration of 100 mg tramadol ER tablet twice daily (five times in total).

### 3.5. Prediction of Changes in Concentration–Time Profiles for Tramadol and M1 According to CYP2D6 Genotype and Dosage

To investigate the effect of CYP2D6 genotype and dosage on PK profiles, simulations were performed for the administration of 100 and 200 mg of tramadol every 12 h (total 5 times). The tramadol/M1 concentration–time profiles were captured from the pre-dose (0 h) to 120 h. The differences on PK profiles according to CYP2D6 genotypes were assessed in the general Korean population in CYP2D6 groups: PM, IM, EM, and UM. As a result, plasma concentration–time profiles for tramadol were within the therapeutic range in all groups after administration of 100 mg tramadol ER. Predicted plasma M1 concentrations were very low in the PM group (mean C_max,ss_ 0.643 ng/mL) compared to the CYP2D6 IM, EM, and UM groups (mean values 40.93, 83.80, and 126.8 ng/mL, respectively).

The plasma concentration–time profiles for tramadol and M1, and changes in PK parameters, in the various CYP2D6 genotype groups following oral administration of 100 and 200 mg tramadol ER tablet twice daily (total five times) are shown in [Figure 4](#pharmaceutics-11-00618-f004) and [Table 6](#pharmaceutics-11-00618-t006) (the plasma concentration–time profiles for each CYP2D6 phenotype after administration of 100 and 200 mg of tramadol were presented in [Supplementary Materials Figures S1 and S2](#app1-pharmaceutics-11-00618)). Following tramadol 100 mg administrations, the C_max,ss_ of tramadol in CYP2D6 PMs reached to toxic range. For CYP2D6 UMs, the C_max,ss_ of M1 exceeded the therapeutic margin ([Supplementary Materials Figure S1](#app1-pharmaceutics-11-00618)). Following tramadol 200 mg administrations, the C_max,ss_ of tramadol were reached to the toxic range in all CYP2D6 metabolizer groups. For M1, the C_max,ss_ exceeded the therapeutic margin in the CYP2D6 IMs, EMs, and UMs ([Figure S2](#app1-pharmaceutics-11-00618)). In [Table 7](#pharmaceutics-11-00618-t007), observed and predicted C_max,ss_ and AUC_last,ss_ values, and predicted/observed geometric mean ratios are presented. The CYP2D6 UM group was excluded from this table because UM subjects did not exist in the clinical study. In the CYP2D6 EM and IM groups, the predicted/observed geometric mean ratios for C_max,ss_ and AUC_last,ss_ for tramadol satisfied the acceptance criteria (0.7–1.43); however, the tramadol AUC_last,ss_ ratio for the CYP2D6 PM group was overestimated at 1.95. The prediction results for M1 showed that AUC_last,ss_ satisfied the acceptance criteria in the CYP2D6 EM group; however, C_max,ss_ and AUC_last,ss_ values were underestimated in both the CYP2D6 IM and PM groups, where the predicted values were much lower than observed values.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c629/6920759/a9fd0fbb2212/pharmaceutics-11-00618-g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6920759_pharmaceutics-11-00618-g004.jpg)

The predicted mean concentration–time profiles after administration of 100 mg and 200 mg tramadol ER tablet twice daily (total five times) for tramadol (a and c), and O-desmethyltramadol (b and d), respectively. Gray areas in (a) and (c) represent the therapeutic concentration range (100–800 ng/mL); checked gray area in (c) represents the toxic range (above 1000 ng/mL) for tramadol; and the gray area in (b) and (d) represents the maximum therapeutic range for M1 (up to 200 ng/mL).

### Table 6.

| Parameters | UM |  | EM |  | IM |  | PM |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Tramadol | M1 | Tramadol | M1 | Tramadol | M1 | Tramadol | M1 |  |
| Tramadol 100 mg |  |  |  |  |  |  |  |  |
| Cmax,ss (ng/mL)(range) | 357.2(72.43–927.6) | 126.8(21.39–449.0) | 469.6(122.2–1117) | 83.80(12.62–368.1) | 593.8(165.8–1379) | 40.93(5.511–240.9) | 721.3(209.5–1675) | 0.6433(0.0975–5.312) |
| AUClast,ss (ng/mL·h)(range) | 5353(648.7–25,267) | 1881(278.6–8560) | 8206(1217–34,213) | 1445(175.2–7522) | 12,049(1932–42,462) | 813.5(79.54–5234) | 16,795(2682–61,319) | 2.919(0.3351–34.42) |
| Tramadol 200 mg |  |  |  |  |  |  |  |  |
| Cmax,ss (ng/mL)(range) | 714.3(144.9–1855) | 253.5 #(42.77–898.0) | 939.1 *(244.3–2235) | 167.6(25.24–736.3) | 1188 **(331.7–2758) | 81.86(11.02–481.7) | 1443 **(418.9–3349) | 1.287(0.1950–10.62) |
| AUClast,ss (ng/mL·h)(range) | 10,706(1297–50,533) | 3761(557.1–17,119) | 16,411(2434–68,426) | 2890(350.3–15,044) | 24,097(3864–84,923) | 1627(159.1–10,467) | 33,591(5365–122,637) | 5.839(0.67–68.83) |

Table 6 Caption: Predicted geometric mean Cmax,ss and AUClast,ss values for tramadol and O-desmethyltramadol (M1) following oral administration of 100 and 200 mg tramadol ER tablet twice daily (total five times) in various CYP2D6 metabolizer groups.

### Table 7.

| Tramadol | EM |  |  | IM |  |  | PM |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Observed (n = 13) | Predicted (n = 1000) | Ratio (90% CI) | Observed (n = 8) | Predicted (n = 1000) | Ratio (90% CI) | Observed (n = 1) | Predicted (n = 1000) | Ratio (90% CI) |  |
| Cmax,ss (ng/mL)(range) | 551.2(294.0–904.4) | 469.6(122.2–1117) | 0.85(0.72–1.01) | 828.5(676.6–942.1) | 593.8(165.8–1379) | 0.72(0.59–0.87) | 751.10 | 721.3(209.5–1675) | 0.96 |
| AUClast,ss (ng/mL·h)(range) | 7116(4127–9345) | 8206(1217–34,213) | 1.15(0.90–1.48) | 13,501(10,527–16,038) | 12,049(1932–42,462) | 0.89(0.66–1.20) | 8591.72 | 16,795(2682–61,319) | 1.95 |
| M1 | EM |  |  | IM |  |  | PM |  |  |
| Observed (n = 13) | Predicted (n = 1000) | Ratio (90% CI) | Observed (n = 8) | Predicted (n = 1000) | Ratio (90% CI) | Observed (n = 1) | Predicted (n = 1000) | Ratio (90% CI) |  |
| Cmax,ss (ng/mL)(range) | 125.0(81.8–176.7) | 83.80(12.62–368.1) | 0.67(0.52–0.86) | 87.79(66.0–114.1) | 40.93(5.511–240.9) | 0.47(0.32–0.69) | 29.8 | 0.6433(0.0975–5.312) | 0.02 |
| AUClast,ss (ng/mL·h)(range) | 1996(1373–2875) | 1445(175.2–7522) | 0.72(0.56–0.94) | 1718(1223–2199) | 813.5(79.54–5234) | 0.47(0.31–0.72) | 445.3 | 2.919(0.3351–34.42) | 0.01 |

Table 7 Caption: Predicted and observed geometric mean PK parameters for tramadol and M1 according to CYP2D6 genotype following oral administration of 100 mg tramadol ER tablet twice daily (total five times).

## 4. Discussion

PBPK models for tramadol and M1 were developed. Tramadol plasma concentration–time profiles were well predicted from the proposed model. Prediction results for M1 included values in the 5th to 95th percentiles of most observed plasma concentration–time values, and the predicted mean plasma concentration was also similar to the observed concentration–time profile. However, geometric mean C_max,ss_ and AUC_last,ss_ ratios were under-predicted (0.63 and 0.67 for C_max,ss_ and AUC_last,ss_, respectively). To predict concentration-dependent toxicities, the therapeutic range (100–800 ng/mL for tramadol, and up to 200 ng/mL for M1) and the tramadol toxic range and lethal concentration (>1000 ng/mL, and >2000 ng/mL, respectively) were obtained from the literatures [[34](#B34-pharmaceutics-11-00618),[35](#B35-pharmaceutics-11-00618)]. In general, the recommended dose of tramadol is up to 400 mg per day for immediate-release formulations and 300 mg per day for ER formulations [[2](#B2-pharmaceutics-11-00618)]. Simulations were performed for 100 and 200 mg with 12-h intervals (5 times) according to CYP2D6 genotypes. After administration of 100 mg of tramadol, the predicted C_max,ss_ of tramadol reached to toxic range in CYP2D6 PMs and exceeded therapeutic range in some IMs, and the predicted C_max,ss_ of M1 exceeded therapeutic margin in CYP2D6 UMs. After tramadol 200 mg administrations, the predicted tramadol C_max,ss_ reached to toxic ranges in all CYP2D6 metabolizer groups, even in some EMs and the predicted M1 C_max,ss_ exceeded the therapeutic margin in CYP2D6 IMs, EMs, and UMs. The concentrations exceeded the therapeutic margins or reached to the toxic range might be related to potential toxicities after tramadol administrations, even though recommended doses of tramadol were administered.

PBPK modeling is useful for predicting PK profiles for rare genotypes in the population. The frequency of CYP2D6 UM in the Korean population has been reported as approximately 1.25% [[36](#B36-pharmaceutics-11-00618)]. In the clinical study used for our PBPK model development, there was only one PM subject, and no UM subject was found. The model developed in this study could predict the plasma concentration–time profiles of tramadol and M1 for these two groups. Using the developed model, plasma tramadol/M1 concentration–time profiles for CYP2D6 UM, a very rare genotype in Koreans, were also predicted.

Tramadol inhibits reuptake of 5-HT and norepinephrine. M1 binds to μ-opioid receptors and exhibits analgesic effects. Due to these actions, the side effects of tramadol differ depending on CYP2D6 genotype. In the PM group, a high risk of side effects due to tramadol, such as serotonin syndrome, can be expected; and in the UM group, a high risk of μ-opioid receptor-related side effects, such as respiratory depression, can be expected relative to other CYP2D6 genotypes [[16](#B16-pharmaceutics-11-00618)]. In our simulation, the plasma concentrations of tramadol and M1 exceeded the therapeutic concentration range, even after administration of recommended doses. These results suggest that the frequency of concentration-related adverse drug reactions may be reduced by optimizing the dosing regimen according to CYP2D6 genotype of the patient or population.

Tramadol and M1 distribution in each tissue were estimated using the PBPK model, and tramadol and M1 were distributed most to the liver. In cases of fatal intoxication due to tramadol, the highest concentration of tramadol was evident in the liver, after the blood and urine. These distribution characteristics are considered due to the hepatic metabolism of tramadol and its metabolites [[37](#B37-pharmaceutics-11-00618)]. The distribution of tramadol to adipose tissue differed from that for M1. Indeed, tramadol is considered to distribute widely to lipid-rich tissues because of its higher affinity for lipids than M1 (logD for tramadol and M1: 1.13 and 0.4, respectively) [[38](#B38-pharmaceutics-11-00618)]. Further research is needed about the distribution characteristics of tramadol and M1 to each tissue.

The predicted plasma M1 concentration–time profiles were under-predicted due to a lack of information about distribution and elimination properties. Since M1 is produced by tramadol metabolism, elimination profiles (intrinsic clearance by CYP, renal clearance and additional clearance) of tramadol were adjusted to improve the M1 model; however, there were no significant changes in M1 concentration–time profiles. This might be due to poor distribution of M1 from liver to plasma, or to exaggeration of elimination. For improvement, the M1 model was built using parameter estimation by observed plasma concentration–time profiles as distribution and elimination profiles (tissue–plasma partition coefficient, additional clearance, renal clearance, bile clearance). When estimating several parameters, the predicted plasma M1 concentration–time profiles changed significantly when values of the unbound fraction in incubated microsomes (fu_mic_) and active hepatic scalar were changed. Thus, the plasma M1 concentration–time profile might be greatly influenced by metabolism. More detailed information and parameters for M1 metabolism are needed for more accurate predictions of plasma M1 concentration–time profiles.

Regarding limitations of our study, tramadol is metabolized not only to M1, but also to *N*-desmethyltramadol (M2) by CYP2D6, CYP2B6, and CYP3A4. In accordance with the literature, the toxicity of tramadol and M1 can be determined using M1/M2 ratio [[34](#B34-pharmaceutics-11-00618)]. Therefore, an M2 model could improve the predictability of concentration-related adverse drug reactions after tramadol administration. Moreover, organic cation transporter 1 (OCT1) and multidrug resistance protein 1 (MDR1) influence the disposition of tramadol and M1. Significant differences in drug disposition according to OCT1 and MDR1 genotypes have been shown, even in same CYP2D6 phenotype [[39](#B39-pharmaceutics-11-00618),[40](#B40-pharmaceutics-11-00618),[41](#B41-pharmaceutics-11-00618)]. Due to lack of information of transporter kinetic parameter for each organ, the transport kinetic parameters for M1 were excluded for the model. For elaborate model prediction, OCT1 and MDR1 genotypes (*OCT**1, *2, *3, *4, *5, and *MDR1* C3435T) could be incorporated.

## 5. Conclusions

In summary, our PBPK model for tramadol and M1 was developed and predicted concentration–time profiles after multiple administrations of a tramadol ER formulation in the Korean population. Differences in PK profiles and concentration-dependent toxicities were predicted according to CYP2D6 phenotype and dosage. Most modeling studies of tramadol used a population PK approach, and the literature using PBPK modeling focused on the PK profile of tramadol itself. However, this study developed a model with predictive power for tramadol and M1, the major active metabolite. This model could be applied to predict concentration-dependent toxicity profiles in cases of tramadol overdose or abuse and also, CYP2D6-related drug interactions.

## Supplementary Materials

The following are available online at [https://www.mdpi.com/1999-4923/11/11/618/s1](https://www.mdpi.com/1999-4923/11/11/618/s1), Figure S1: The predicted mean tramadol and *O*-desmethyltramadol concentration-time profiles after administration of 100 mg tramadol ER tablet twice daily (five times in total) for CYP2D6 poor metabolizer, intermediate metabolizer, extensive metabolizer and ultra-rapid metabolizer, respectively; Figure S2: The predicted mean tramadol and *O*-desmethyltramadol concentration-time profiles after administration of 200 mg tramadol ER tablet twice daily (five times in total) for CYP2D6 poor metabolizer, intermediate metabolizer, extensive metabolizer and ultra-rapid metabolizer, respectively.

## Author Contributions

J.-W.B., S.L. and K.-H.S. conceived and designed the experiments; H.-C.J., S.H.B. and K.-H.S. performed the experiments, simulations, and analyzed the data; H.-C.J., Y.J., S.H.B., A.K. and K.-H.S. reviewed the data and wrote the paper. All authors approved the final manuscript.

## Funding

This research was supported by the Bio and Medical Technology Development Program of the National Research Foundation (NRF) and was funded by the Korean government (MSIP and MOHW; No. NRF-2015M3A9E1028327) and the Korea Health Technology R&D Project through the Korea Health Industry Development Institute (KHIDI), funded by the Ministry of Health and Welfare, Republic of Korea (grant number: HI17C0927).

## Conflicts of Interest

The authors declare no conflict of interest.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Shipton E. Tramadol—Present and future. Anaesth. Intensive Care. 2000;28:363–374. doi: 10.1177/0310057X0002800403.  [DOI](https://doi.org/10.1177/0310057X0002800403) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10969362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anaesth.%20Intensive%20Care&title=Tramadol%E2%80%94Present%20and%20future&author=E.%20Shipton&volume=28&publication_year=2000&pages=363-374&pmid=10969362&doi=10.1177/0310057X0002800403&)

2. Miotto K., Cho A.K., Khalil M.A., Blanco K., Sasaki J.D., Rawson R. Trends in Tramadol: Pharmacology, Metabolism, and Misuse. Anesth. Analg. 2017;124:44–51. doi: 10.1213/ANE.0000000000001683.  [DOI](https://doi.org/10.1213/ANE.0000000000001683) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27861439/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth.%20Analg.&title=Trends%20in%20Tramadol:%20Pharmacology,%20Metabolism,%20and%20Misuse&author=K.%20Miotto&author=A.K.%20Cho&author=M.A.%20Khalil&author=K.%20Blanco&author=J.D.%20Sasaki&volume=124&publication_year=2017&pages=44-51&pmid=27861439&doi=10.1213/ANE.0000000000001683&)

3. Lassen D., Damkier P., Brøsen K. The Pharmacogenetics of Tramadol. Clin. Pharmacokinet. 2015;54:825–836. doi: 10.1007/s40262-015-0268-0.  [DOI](https://doi.org/10.1007/s40262-015-0268-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25910878/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet.&title=The%20Pharmacogenetics%20of%20Tramadol&author=D.%20Lassen&author=P.%20Damkier&author=K.%20Br%C3%B8sen&volume=54&publication_year=2015&pages=825-836&pmid=25910878&doi=10.1007/s40262-015-0268-0&)

4. Kaye A.D. Tramadol, pharmacology, side effects, and serotonin syndrome: A review. Pain Physician. 2015;18:395–400.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/26218943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain%20Physician&title=Tramadol,%20pharmacology,%20side%20effects,%20and%20serotonin%20syndrome:%20A%20review&author=A.D.%20Kaye&volume=18&publication_year=2015&pages=395-400&pmid=26218943&)

5. Leppert W. Tramadol as an analgesic for mild to moderate cancer pain. Pharmacol. Rep. 2009;61:978–992. doi: 10.1016/S1734-1140(09)70159-8.  [DOI](https://doi.org/10.1016/S1734-1140(09)70159-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20081232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Rep.&title=Tramadol%20as%20an%20analgesic%20for%20mild%20to%20moderate%20cancer%20pain&author=W.%20Leppert&volume=61&publication_year=2009&pages=978-992&pmid=20081232&doi=10.1016/S1734-1140(09)70159-8&)

6. Lehtonen P., Sten T., Aitio O., Kurkela M., Vuorensola K., Finel M., Kostiainen R. Glucuronidation of racemic O-desmethyltramadol, the active metabolite of tramadol. Eur. J. Pharm. Sci. 2010;41:523–530. doi: 10.1016/j.ejps.2010.08.005.  [DOI](https://doi.org/10.1016/j.ejps.2010.08.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20797434/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pharm.%20Sci.&title=Glucuronidation%20of%20racemic%20O-desmethyltramadol,%20the%20active%20metabolite%20of%20tramadol&author=P.%20Lehtonen&author=T.%20Sten&author=O.%20Aitio&author=M.%20Kurkela&author=K.%20Vuorensola&volume=41&publication_year=2010&pages=523-530&pmid=20797434&doi=10.1016/j.ejps.2010.08.005&)

7. Grond S., Sablotzki A. Clinical pharmacology of tramadol. Clin. Pharmacokinet. 2004;43:879–923. doi: 10.2165/00003088-200443130-00004.  [DOI](https://doi.org/10.2165/00003088-200443130-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15509185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet.&title=Clinical%20pharmacology%20of%20tramadol&author=S.%20Grond&author=A.%20Sablotzki&volume=43&publication_year=2004&pages=879-923&pmid=15509185&doi=10.2165/00003088-200443130-00004&)

8. Kostewicz E.S., Aarons L., Bergstrand M., Bolger M.B., Galetin A., Hatley O., Jamei M., Lloyd R., Pepin X., Rostami-Hodjegan A. PBPK models for the prediction of in vivo performance of oral dosage forms. Eur. J. Pharm. Sci. 2014;57:300–321. doi: 10.1016/j.ejps.2013.09.008.  [DOI](https://doi.org/10.1016/j.ejps.2013.09.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24060672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pharm.%20Sci.&title=PBPK%20models%20for%20the%20prediction%20of%20in%20vivo%20performance%20of%20oral%20dosage%20forms&author=E.S.%20Kostewicz&author=L.%20Aarons&author=M.%20Bergstrand&author=M.B.%20Bolger&author=A.%20Galetin&volume=57&publication_year=2014&pages=300-321&pmid=24060672&doi=10.1016/j.ejps.2013.09.008&)

9. Zhao P., Zhang L., Grillo J., Liu Q., Bullock J., Moon Y., Song P., Brar S., Madabushi R., Wu T. Applications of physiologically based pharmacokinetic (PBPK) modeling and simulation during regulatory review. Clin. Pharmacol. Ther. 2011;89:259–267. doi: 10.1038/clpt.2010.298.  [DOI](https://doi.org/10.1038/clpt.2010.298) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21191381/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Applications%20of%20physiologically%20based%20pharmacokinetic%20(PBPK)%20modeling%20and%20simulation%20during%20regulatory%20review&author=P.%20Zhao&author=L.%20Zhang&author=J.%20Grillo&author=Q.%20Liu&author=J.%20Bullock&volume=89&publication_year=2011&pages=259-267&pmid=21191381&doi=10.1038/clpt.2010.298&)

10. Abbiati R.A., Manca D. A modeling tool for the personalization of pharmacokinetic predictions. Comput. Chem. Eng. 2016;91:28–37. doi: 10.1016/j.compchemeng.2016.03.008.  [DOI](https://doi.org/10.1016/j.compchemeng.2016.03.008) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Comput.%20Chem.%20Eng.&title=A%20modeling%20tool%20for%20the%20personalization%20of%20pharmacokinetic%20predictions&author=R.A.%20Abbiati&author=D.%20Manca&volume=91&publication_year=2016&pages=28-37&doi=10.1016/j.compchemeng.2016.03.008&)

11. Price P.S., Conolly R.B., Chaisson C.F., Gross E.A., Young J.S., Mathis E.T., Tedder D.R. Modeling interindividual variation in physiological factors used in PBPK models of humans. Crit. Rev. Toxicol. 2003;33:469–503. doi: 10.1080/10408440390242324.  [DOI](https://doi.org/10.1080/10408440390242324) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14594104/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit.%20Rev.%20Toxicol.&title=Modeling%20interindividual%20variation%20in%20physiological%20factors%20used%20in%20PBPK%20models%20of%20humans&author=P.S.%20Price&author=R.B.%20Conolly&author=C.F.%20Chaisson&author=E.A.%20Gross&author=J.S.%20Young&volume=33&publication_year=2003&pages=469-503&pmid=14594104&doi=10.1080/10408440390242324&)

12. Marsousi N., Desmeules J.A., Rudaz S., Daali Y. Usefulness of PBPK Modeling in Incorporation of Clinical Conditions in Personalized Medicine. J. Pharm. Sci. 2017;106:2380–2391. doi: 10.1016/j.xphs.2017.04.035.  [DOI](https://doi.org/10.1016/j.xphs.2017.04.035) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28456730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharm.%20Sci.&title=Usefulness%20of%20PBPK%20Modeling%20in%20Incorporation%20of%20Clinical%20Conditions%20in%20Personalized%20Medicine&author=N.%20Marsousi&author=J.A.%20Desmeules&author=S.%20Rudaz&author=Y.%20Daali&volume=106&publication_year=2017&pages=2380-2391&pmid=28456730&doi=10.1016/j.xphs.2017.04.035&)

13. T’Jollyn H., Snoeys J., Colin P., Van Bocxlaer J., Annaert P., Cuyckens F., Vermeulen A., Van Peer A., Allegaert K., Mannens G., et al. Physiology-Based IVIVE Predictions of Tramadol from in Vitro Metabolism Data. Pharm. Res. 2015;32:260–274. doi: 10.1007/s11095-014-1460-x.  [DOI](https://doi.org/10.1007/s11095-014-1460-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25048637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Res.&title=Physiology-Based%20IVIVE%20Predictions%20of%20Tramadol%20from%20in%20Vitro%20Metabolism%20Data&author=H.%20T%E2%80%99Jollyn&author=J.%20Snoeys&author=P.%20Colin&author=J.%20Van%20Bocxlaer&author=P.%20Annaert&volume=32&publication_year=2015&pages=260-274&pmid=25048637&doi=10.1007/s11095-014-1460-x&)

14. Salman S., Sy S.K., Ilett K.F., Page-Sharp M., Paech M.J. Population pharmacokinetic modeling of tramadol and its O-desmethyl metabolite in plasma and breast milk. Eur. J. Clin. Pharmacol. 2011;67:899–908. doi: 10.1007/s00228-011-1023-6.  [DOI](https://doi.org/10.1007/s00228-011-1023-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21394525/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=Population%20pharmacokinetic%20modeling%20of%20tramadol%20and%20its%20O-desmethyl%20metabolite%20in%20plasma%20and%20breast%20milk&author=S.%20Salman&author=S.K.%20Sy&author=K.F.%20Ilett&author=M.%20Page-Sharp&author=M.J.%20Paech&volume=67&publication_year=2011&pages=899-908&pmid=21394525&doi=10.1007/s00228-011-1023-6&)

15. Garrido M.J., Habre W., Rombout F., Trocóniz I.F. Population Pharmacokinetic/Pharmacodynamic Modelling of the Analgesic Effects of Tramadol in Pediatrics. Pharm. Res. 2006;23:2014–2023. doi: 10.1007/s11095-006-9049-7.  [DOI](https://doi.org/10.1007/s11095-006-9049-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16951997/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Res.&title=Population%20Pharmacokinetic/Pharmacodynamic%20Modelling%20of%20the%20Analgesic%20Effects%20of%20Tramadol%20in%20Pediatrics&author=M.J.%20Garrido&author=W.%20Habre&author=F.%20Rombout&author=I.F.%20Troc%C3%B3niz&volume=23&publication_year=2006&pages=2014-2023&pmid=16951997&doi=10.1007/s11095-006-9049-7&)

16. Faria J., Barbosa J., Moreira R., Queirós O., Carvalho F., Dinis-Oliveira R. Comparative pharmacology and toxicology of tramadol and tapentadol. Eur. J. Pain. 2018;22:827–844. doi: 10.1002/ejp.1196.  [DOI](https://doi.org/10.1002/ejp.1196) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29369473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pain&title=Comparative%20pharmacology%20and%20toxicology%20of%20tramadol%20and%20tapentadol&author=J.%20Faria&author=J.%20Barbosa&author=R.%20Moreira&author=O.%20Queir%C3%B3s&author=F.%20Carvalho&volume=22&publication_year=2018&pages=827-844&pmid=29369473&doi=10.1002/ejp.1196&)

17. Lee J., Yoo H.D., Bae J.W., Lee S., Shin K.H. Population pharmacokinetic analysis of tramadol and O-desmethyltramadol with genetic polymorphism of CYP2D6. Drug Des. Dev. Ther. 2019;13:1751. doi: 10.2147/DDDT.S199574.  [DOI](https://doi.org/10.2147/DDDT.S199574) | [PMC free article](/articles/PMC6537040/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31213765/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Des.%20Dev.%20Ther.&title=Population%20pharmacokinetic%20analysis%20of%20tramadol%20and%20O-desmethyltramadol%20with%20genetic%20polymorphism%20of%20CYP2D6&author=J.%20Lee&author=H.D.%20Yoo&author=J.W.%20Bae&author=S.%20Lee&author=K.H.%20Shin&volume=13&publication_year=2019&pages=1751&pmid=31213765&doi=10.2147/DDDT.S199574&)

18. Yu H., Hong S., Jeong C.H., Bae J.W., Lee S. Development of a linear dual column HPLC–MS/MS method and clinical genetic evaluation for tramadol and its phase I and II metabolites in oral fluid. Arch. Pharmacal Res. 2018;41:288–298. doi: 10.1007/s12272-017-0993-z.  [DOI](https://doi.org/10.1007/s12272-017-0993-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29196917/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Pharmacal%20Res.&title=Development%20of%20a%20linear%20dual%20column%20HPLC%E2%80%93MS/MS%20method%20and%20clinical%20genetic%20evaluation%20for%20tramadol%20and%20its%20phase%20I%20and%20II%20metabolites%20in%20oral%20fluid&author=H.%20Yu&author=S.%20Hong&author=C.H.%20Jeong&author=J.W.%20Bae&author=S.%20Lee&volume=41&publication_year=2018&pages=288-298&pmid=29196917&doi=10.1007/s12272-017-0993-z&)

19. Byeon J.Y., Kim Y.H., Lee C.M., Kim S.H., Chae W.K., Jung E.H., Choi C.I., Jang C.G., Lee S.Y., Bae J.W. CYP2D6 allele frequencies in Korean population, comparison with East Asian, Caucasian and African populations, and the comparison of metabolic activity of CYP2D6 genotypes. Arch. Pharmacal Res. 2018;41:921–930. doi: 10.1007/s12272-018-1075-6.  [DOI](https://doi.org/10.1007/s12272-018-1075-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30191460/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Pharmacal%20Res.&title=CYP2D6%20allele%20frequencies%20in%20Korean%20population,%20comparison%20with%20East%20Asian,%20Caucasian%20and%20African%20populations,%20and%20the%20comparison%20of%20metabolic%20activity%20of%20CYP2D6%20genotypes&author=J.Y.%20Byeon&author=Y.H.%20Kim&author=C.M.%20Lee&author=S.H.%20Kim&author=W.K.%20Chae&volume=41&publication_year=2018&pages=921-930&pmid=30191460&doi=10.1007/s12272-018-1075-6&)

20. Byeon J.Y., Kim Y.H., Na H.S., Jang J.H., Kim S.H., Lee Y.J., Bae J.W., Kim I.S., Jang C.G., Chung M.W., et al. Effects of the CYP2D6* 10 allele on the pharmacokinetics of atomoxetine and its metabolites. Arch. Pharmacal Res. 2015;38:2083–2091. doi: 10.1007/s12272-015-0646-z.  [DOI](https://doi.org/10.1007/s12272-015-0646-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26254792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Pharmacal%20Res.&title=Effects%20of%20the%20CYP2D6*%2010%20allele%20on%20the%20pharmacokinetics%20of%20atomoxetine%20and%20its%20metabolites&author=J.Y.%20Byeon&author=Y.H.%20Kim&author=H.S.%20Na&author=J.H.%20Jang&author=S.H.%20Kim&volume=38&publication_year=2015&pages=2083-2091&pmid=26254792&doi=10.1007/s12272-015-0646-z&)

21. Doki K., Homma M., Kuga K., Kusano K., Watanabe S., Yamaguchi I., Kohda Y. Effect of CYP2D6 genotype on flecainide pharmacokinetics in Japanese patients with supraventricular tachyarrhythmia. Eur. J. Clin. Pharmacol. 2006;62:919–926. doi: 10.1007/s00228-006-0188-x.  [DOI](https://doi.org/10.1007/s00228-006-0188-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16944116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=Effect%20of%20CYP2D6%20genotype%20on%20flecainide%20pharmacokinetics%20in%20Japanese%20patients%20with%20supraventricular%20tachyarrhythmia&author=K.%20Doki&author=M.%20Homma&author=K.%20Kuga&author=K.%20Kusano&author=S.%20Watanabe&volume=62&publication_year=2006&pages=919-926&pmid=16944116&doi=10.1007/s00228-006-0188-x&)

22. Findling R.L., Nucci G., Piergies A.A., Gomeni R., Bartolic E.I., Fong R., Carpenter D.J., Leeder J.S., Gaedigk A., Danoff T.M. Multiple dose pharmacokinetics of paroxetine in children and adolescents with major depressive disorder or obsessive–compulsive disorder. Neuropsychopharmacology. 2006;31:1274. doi: 10.1038/sj.npp.1300960.  [DOI](https://doi.org/10.1038/sj.npp.1300960) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16319918/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Multiple%20dose%20pharmacokinetics%20of%20paroxetine%20in%20children%20and%20adolescents%20with%20major%20depressive%20disorder%20or%20obsessive%E2%80%93compulsive%20disorder&author=R.L.%20Findling&author=G.%20Nucci&author=A.A.%20Piergies&author=R.%20Gomeni&author=E.I.%20Bartolic&volume=31&publication_year=2006&pages=1274&pmid=16319918&doi=10.1038/sj.npp.1300960&)

23. Yoo H.D., Lee S.N., Kang H.A., Cho H.Y., Lee I.K., Lee Y.B. Influence of ABCB1 genetic polymorphisms on the pharmacokinetics of risperidone in healthy subjects with CYP2D6* 10/* 10. Br. J. Pharmacol. 2011;164:433–443. doi: 10.1111/j.1476-5381.2011.01385.x.  [DOI](https://doi.org/10.1111/j.1476-5381.2011.01385.x) | [PMC free article](/articles/PMC3188890/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21449914/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Pharmacol.&title=Influence%20of%20ABCB1%20genetic%20polymorphisms%20on%20the%20pharmacokinetics%20of%20risperidone%20in%20healthy%20subjects%20with%20CYP2D6*%2010/*%2010&author=H.D.%20Yoo&author=S.N.%20Lee&author=H.A.%20Kang&author=H.Y.%20Cho&author=I.K.%20Lee&volume=164&publication_year=2011&pages=433-443&pmid=21449914&doi=10.1111/j.1476-5381.2011.01385.x&)

24. Chen X., Murawski A., Patel K., Crespi C.L., Balimane P.V. A Novel Design of Artificial Membrane for Improving the PAMPA Model. Pharm. Res. 2008;25:1511–1520. doi: 10.1007/s11095-007-9517-8.  [DOI](https://doi.org/10.1007/s11095-007-9517-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18185985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Res.&title=A%20Novel%20Design%20of%20Artificial%20Membrane%20for%20Improving%20the%20PAMPA%20Model&author=X.%20Chen&author=A.%20Murawski&author=K.%20Patel&author=C.L.%20Crespi&author=P.V.%20Balimane&volume=25&publication_year=2008&pages=1511-1520&pmid=18185985&doi=10.1007/s11095-007-9517-8&)

25. Bae S.H., Kwon M.J., Park J.B., Kim D., Kim D.H., Kang J.S., Kim C.G., Oh E., Bae S.K. Metabolic drug-drug interaction potential of macrolactin A and 7-O-succinyl macrolactin A assessed by evaluating cytochrome P450 inhibition and induction and UDP-glucuronosyltransferase inhibition in vitro. Antimicrob. Agents Chemother. 2014;58:5036–5046. doi: 10.1128/AAC.00018-14.  [DOI](https://doi.org/10.1128/AAC.00018-14) | [PMC free article](/articles/PMC4135859/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24890600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Metabolic%20drug-drug%20interaction%20potential%20of%20macrolactin%20A%20and%207-O-succinyl%20macrolactin%20A%20assessed%20by%20evaluating%20cytochrome%20P450%20inhibition%20and%20induction%20and%20UDP-glucuronosyltransferase%20inhibition%20in%20vitro&author=S.H.%20Bae&author=M.J.%20Kwon&author=J.B.%20Park&author=D.%20Kim&author=D.H.%20Kim&volume=58&publication_year=2014&pages=5036-5046&pmid=24890600&doi=10.1128/AAC.00018-14&)

26. Li A.P. In vitro approaches to evaluate ADMET drug properties. Curr. Top. Med. Chem. 2004;4:701–706. doi: 10.2174/1568026043451050.  [DOI](https://doi.org/10.2174/1568026043451050) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15032683/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Top.%20Med.%20Chem.&title=In%20vitro%20approaches%20to%20evaluate%20ADMET%20drug%20properties&author=A.P.%20Li&volume=4&publication_year=2004&pages=701-706&pmid=15032683&doi=10.2174/1568026043451050&)

27. Kanaan M., Daali Y., Dayer P., Desmeules J. Uptake/Efflux Transport of Tramadol Enantiomers and O-Desmethyl-Tramadol: Focus on P-Glycoprotein. Basic Clin. Pharmacol. Toxicol. 2009;105:199–206. doi: 10.1111/j.1742-7843.2009.00428.x.  [DOI](https://doi.org/10.1111/j.1742-7843.2009.00428.x) | [PMC free article](/articles/PMC2774482/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19496778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin.%20Pharmacol.%20Toxicol.&title=Uptake/Efflux%20Transport%20of%20Tramadol%20Enantiomers%20and%20O-Desmethyl-Tramadol:%20Focus%20on%20P-Glycoprotein&author=M.%20Kanaan&author=Y.%20Daali&author=P.%20Dayer&author=J.%20Desmeules&volume=105&publication_year=2009&pages=199-206&pmid=19496778&doi=10.1111/j.1742-7843.2009.00428.x&)

28. Saarikoski T., Saari T.I., Hagelberg N.M., Backman J.T., Neuvonen P.J., Scheinin M., Olkkola K.T., Laine K. Effects of terbinafine and itraconazole on the pharmacokinetics of orally administered tramadol. Eur. J. Clin. Pharmacol. 2015;71:321–327. doi: 10.1007/s00228-014-1799-2.  [DOI](https://doi.org/10.1007/s00228-014-1799-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25560051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=Effects%20of%20terbinafine%20and%20itraconazole%20on%20the%20pharmacokinetics%20of%20orally%20administered%20tramadol&author=T.%20Saarikoski&author=T.I.%20Saari&author=N.M.%20Hagelberg&author=J.T.%20Backman&author=P.J.%20Neuvonen&volume=71&publication_year=2015&pages=321-327&pmid=25560051&doi=10.1007/s00228-014-1799-2&)

29. Mattia C., Coluzzi F. Once-daily tramadol in rheumatological pain. Expert Opin. Pharmacother. 2006;7:1811–1823. doi: 10.1517/14656566.7.13.1811.  [DOI](https://doi.org/10.1517/14656566.7.13.1811) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16925507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin.%20Pharmacother.&title=Once-daily%20tramadol%20in%20rheumatological%20pain&author=C.%20Mattia&author=F.%20Coluzzi&volume=7&publication_year=2006&pages=1811-1823&pmid=16925507&doi=10.1517/14656566.7.13.1811&)

30. T’jollyn H., Vermeulen A., Van Bocxlaer J. PBPK and its virtual populations: The impact of physiology on pediatric pharmacokinetic predictions of tramadol. AAPS J. 2019;21:8. doi: 10.1208/s12248-018-0277-7.  [DOI](https://doi.org/10.1208/s12248-018-0277-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30498862/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AAPS%20J.&title=PBPK%20and%20its%20virtual%20populations:%20The%20impact%20of%20physiology%20on%20pediatric%20pharmacokinetic%20predictions%20of%20tramadol&author=H.%20T%E2%80%99jollyn&author=A.%20Vermeulen&author=J.%20Van%20Bocxlaer&volume=21&publication_year=2019&pages=8&pmid=30498862&doi=10.1208/s12248-018-0277-7&)

31. Pubchem O-Desmethyltramadol. [(accessed on 31 January 2019)]; Available online: https://pubchem.ncbi.nlm.nih.gov/compound/9838803.  [https://pubchem.ncbi.nlm.nih.gov/compound/9838803](https://pubchem.ncbi.nlm.nih.gov/compound/9838803)

32. Wojsławski J., Białk-Bielińska A., Stepnowski P., Dołżonek J. Leaching behavior of pharmaceuticals and their metabolites in the soil environment. Chemosphere. 2019;231:269–275. doi: 10.1016/j.chemosphere.2019.05.031.  [DOI](https://doi.org/10.1016/j.chemosphere.2019.05.031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31129408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chemosphere&title=Leaching%20behavior%20of%20pharmaceuticals%20and%20their%20metabolites%20in%20the%20soil%20environment&author=J.%20Wojs%C5%82awski&author=A.%20Bia%C5%82k-Bieli%C5%84ska&author=P.%20Stepnowski&author=J.%20Do%C5%82%C5%BConek&volume=231&publication_year=2019&pages=269-275&pmid=31129408&doi=10.1016/j.chemosphere.2019.05.031&)

33. T’jollyn H., Snoeys J., Vermeulen A., Michelet R., Cuyckens F., Mannens G., Van Peer A., Annaert P., Allegaert K., Van Bocxlaer J., et al. Physiologically Based Pharmacokinetic Predictions of Tramadol Exposure Throughout Pediatric Life: An Analysis of the Different Clearance Contributors with Emphasis on CYP2D6 Maturation. AAPS J. 2015;17:1376–1387. doi: 10.1208/s12248-015-9803-z.  [DOI](https://doi.org/10.1208/s12248-015-9803-z) | [PMC free article](/articles/PMC4627449/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26209290/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AAPS%20J.&title=Physiologically%20Based%20Pharmacokinetic%20Predictions%20of%20Tramadol%20Exposure%20Throughout%20Pediatric%20Life:%20An%20Analysis%20of%20the%20Different%20Clearance%20Contributors%20with%20Emphasis%20on%20CYP2D6%20Maturation&author=H.%20T%E2%80%99jollyn&author=J.%20Snoeys&author=A.%20Vermeulen&author=R.%20Michelet&author=F.%20Cuyckens&volume=17&publication_year=2015&pages=1376-1387&pmid=26209290&doi=10.1208/s12248-015-9803-z&)

34. Barbera N., Fisichella M., Bosco A., Indorato F., Spadaro G., Romano G. A suicidal poisoning due to tramadol. A metabolic approach to death investigation. J. Forensic Leg. Med. 2013;20:555–558. doi: 10.1016/j.jflm.2013.03.006.  [DOI](https://doi.org/10.1016/j.jflm.2013.03.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23756535/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Forensic%20Leg.%20Med.&title=A%20suicidal%20poisoning%20due%20to%20tramadol.%20A%20metabolic%20approach%20to%20death%20investigation&author=N.%20Barbera&author=M.%20Fisichella&author=A.%20Bosco&author=F.%20Indorato&author=G.%20Spadaro&volume=20&publication_year=2013&pages=555-558&pmid=23756535&doi=10.1016/j.jflm.2013.03.006&)

35. Perdreau E., Iriart X., Mouton J.B., Jalal Z., Thambo J.B. Cardiogenic shock due to acute tramadol intoxication. Cardiovasc. Toxicol. 2015;15:100–103. doi: 10.1007/s12012-014-9262-2.  [DOI](https://doi.org/10.1007/s12012-014-9262-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24811952/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc.%20Toxicol.&title=Cardiogenic%20shock%20due%20to%20acute%20tramadol%20intoxication&author=E.%20Perdreau&author=X.%20Iriart&author=J.B.%20Mouton&author=Z.%20Jalal&author=J.B.%20Thambo&volume=15&publication_year=2015&pages=100-103&pmid=24811952&doi=10.1007/s12012-014-9262-2&)

36. Lee S.Y., Sohn K.M., Ryu J.Y., Yoon Y.R., Shin J.G., Kim J.W. Sequence-based CYP2D6 genotyping in the Korean population. Ther. Drug Monit. 2006;28:382–387. doi: 10.1097/01.ftd.0000211823.80854.db.  [DOI](https://doi.org/10.1097/01.ftd.0000211823.80854.db) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16778723/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther.%20Drug%20Monit.&title=Sequence-based%20CYP2D6%20genotyping%20in%20the%20Korean%20population&author=S.Y.%20Lee&author=K.M.%20Sohn&author=J.Y.%20Ryu&author=Y.R.%20Yoon&author=J.G.%20Shin&volume=28&publication_year=2006&pages=382-387&pmid=16778723&doi=10.1097/01.ftd.0000211823.80854.db&)

37. Vazzana M., Andreani T., Fangueiro J., Faggio C., Silva C., Santini A., Garcia M., Silva A., Souto E. Tramadol hydrochloride: Pharmacokinetics, pharmacodynamics, adverse side effects, co-administration of drugs and new drug delivery systems. Biomed. Pharmacother. 2015;70:234–238. doi: 10.1016/j.biopha.2015.01.022.  [DOI](https://doi.org/10.1016/j.biopha.2015.01.022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25776506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed.%20Pharmacother.&title=Tramadol%20hydrochloride:%20Pharmacokinetics,%20pharmacodynamics,%20adverse%20side%20effects,%20co-administration%20of%20drugs%20and%20new%20drug%20delivery%20systems&author=M.%20Vazzana&author=T.%20Andreani&author=J.%20Fangueiro&author=C.%20Faggio&author=C.%20Silva&volume=70&publication_year=2015&pages=234-238&pmid=25776506&doi=10.1016/j.biopha.2015.01.022&)

38. Costa I., Oliveira A., Guedes de Pinho P., Teixeira H.M., Moreira R., Carvalho F., Jorge Dinis-Oliveira R. Postmortem Redistribution of Tramadol and O-Desmethyltramadol. J. Anal. Toxicol. 2013;37:670–675. doi: 10.1093/jat/bkt084.  [DOI](https://doi.org/10.1093/jat/bkt084) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24084873/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Anal.%20Toxicol.&title=Postmortem%20Redistribution%20of%20Tramadol%20and%20O-Desmethyltramadol&author=I.%20Costa&author=A.%20Oliveira&author=P.%20Guedes%20de%20Pinho&author=H.M.%20Teixeira&author=R.%20Moreira&volume=37&publication_year=2013&pages=670-675&pmid=24084873&doi=10.1093/jat/bkt084&)

39. Tzvetkov M.V., Saadatmand A.R., Lötsch J., Tegeder I., Stingl J.C., Brockmöller J. Genetically polymorphic OCT1: Another piece in the puzzle of the variable pharmacokinetics and pharmacodynamics of the opioidergic drug tramadol. Clin. Pharmacol. Ther. 2011;90:143–150. doi: 10.1038/clpt.2011.56.  [DOI](https://doi.org/10.1038/clpt.2011.56) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21562485/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genetically%20polymorphic%20OCT1:%20Another%20piece%20in%20the%20puzzle%20of%20the%20variable%20pharmacokinetics%20and%20pharmacodynamics%20of%20the%20opioidergic%20drug%20tramadol&author=M.V.%20Tzvetkov&author=A.R.%20Saadatmand&author=J.%20L%C3%B6tsch&author=I.%20Tegeder&author=J.C.%20Stingl&volume=90&publication_year=2011&pages=143-150&pmid=21562485&doi=10.1038/clpt.2011.56&)

40. Stamer U.M., Frank M., Stuber F., Brockmoller J., Steffens M., Tzvetkov M.V. Loss-of-function polymorphisms in the organic cation transporter OCT1 are associated with reduced postoperative tramadol consumption. Pain. 2016;157:2467–2475. doi: 10.1097/j.pain.0000000000000662.  [DOI](https://doi.org/10.1097/j.pain.0000000000000662) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27541716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain&title=Loss-of-function%20polymorphisms%20in%20the%20organic%20cation%20transporter%20OCT1%20are%20associated%20with%20reduced%20postoperative%20tramadol%20consumption&author=U.M.%20Stamer&author=M.%20Frank&author=F.%20Stuber&author=J.%20Brockmoller&author=M.%20Steffens&volume=157&publication_year=2016&pages=2467-2475&pmid=27541716&doi=10.1097/j.pain.0000000000000662&)

41. Slanar O., Nobilis M., Kvétina J., Matousková O., Idle J.R., Perlík F. Pharmacokinetics of tramadol is affected by MDR1 polymorphism C3435T. Eur. J. Clin. Pharmacol. 2007;63:419–421. doi: 10.1007/s00228-006-0255-3.  [DOI](https://doi.org/10.1007/s00228-006-0255-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17265061/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=Pharmacokinetics%20of%20tramadol%20is%20affected%20by%20MDR1%20polymorphism%20C3435T&author=O.%20Slanar&author=M.%20Nobilis&author=J.%20Kv%C3%A9tina&author=O.%20Matouskov%C3%A1&author=J.R.%20Idle&volume=63&publication_year=2007&pages=419-421&pmid=17265061&doi=10.1007/s00228-006-0255-3&)
