# Comprehensive Pharmacogenomic Study Reveals an Important Role of UGT1A3 in Montelukast Pharmacokinetics

## Metadata
**Authors:** Päivi Hirvensalo, Aleksi Tornio, Mikko Neuvonen, Tuija Tapaninen, Maria Paile‐Hyvärinen, Vesa Kärjä, Ville T Männistö, Jussi Pihlajamäki, Janne T Backman, Mikko Niemi
**Journal:** Clinical Pharmacology and Therapeutics
**Date:** 2017 Nov 6
**DOI:** [10.1002/cpt.891](https://doi.org/10.1002/cpt.891)
**PMID:** 28940478
**PMCID:** PMC6033076
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6033076/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6033076/pdf/CPT-104-158.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6033076/pdf/CPT-104-158.pdf)

## Abstract

To identify the genetic basis of interindividual variability in montelukast exposure, we determined its pharmacokinetics and sequenced 379 pharmacokinetic genes in 191 healthy volunteers. An intronic single nucleotide variation (SNV), strongly linked with UGT1A3*2, associated with reduced area under the plasma concentration–time curve (AUC0‐∞) of montelukast (by 18% per copy of the minor allele; P = 1.83 × 10−10). UGT1A3*2 was associated with increased AUC0‐∞ of montelukast acyl‐glucuronide M1 and decreased AUC0‐∞ of hydroxymetabolites M5R, M5S, and M6 (P < 10−9). Furthermore, SNVs in SLCO1B1 and ABCC9 were associated with the AUC0‐∞ of M1 and M5R, respectively. In addition, a candidate gene analysis suggested that CYP2C8 and ABCC9 SNVs also affect the AUC0‐∞ of montelukast. The found UGT1A3 and ABCC9 variants associated with increased expression of the respective genes in human liver samples. Montelukast and its hydroxymetabolites were glucuronidated by UGT1A3 in vitro. These results indicate that UGT1A3 plays an important role in montelukast pharmacokinetics, especially in UGT1A3*2 carriers.

**WHAT IS THE CURRENT KNOWLEDGE ON THE TOPIC?**

☑ Interindividual variability in the pharmacokinetics of montelukast is high. It is not known how genetic variants in genes encoding drug‐metabolizing enzymes, membrane transporters, and regulatory proteins contribute to this variability.

**WHAT QUESTION DID THIS STUDY ADDRESS?**

☑ This study investigated whether genetic variants in pharmacokinetic genes affect the pharmacokinetics of montelukast and its metabolites.

**WHAT THIS STUDY ADDS TO OUR KNOWLEDGE**

☑ The results indicate that UGT1A3 plays an important role in montelukast pharmacokinetics, especially in carriers of the *UGT1A3*2* allele associated with increased UGT1A3 expression. Furthermore, the results suggest that also *CYP2C8* and *ABCC9* variants affect the exposure to montelukast.

**HOW THIS MIGHT CHANGE CLINICAL PHARMACOLOGY OR TRANSLATIONAL SCIENCE**

☑ Genetic variants explain a significant proportion of interindividual variability in montelukast pharmacokinetics. This knowledge may aid in individualizing treatment with leukotriene receptor antagonists.

## RESULTS

### Montelukast pharmacogenomics

In the present study, substantial interindividual variability was observed in the pharmacokinetic variables of montelukast and its acyl‐glucuronide (M1) and hydroxymetabolites (M5R, M5S, and M6) (**Supplementary Table S1**). The areas under the plasma concentration–time curve from 0 h to infinity (AUC_0‐∞_) of montelukast, M1, M5R, M5S, and M6 varied 8.7‐fold, 13‐fold, 30‐fold, 22‐fold, and 23‐fold between individual subjects, respectively.

A total of 105,145 single nucleotide variations (SNVs) were found in the 379 analyzed pharmacokinetic genes (**Supplementary Table S2**), of which 46,064 had a minor allele frequency (MAF) of at least 0.05. In a stepwise linear regression analysis fixed for demographic covariates, nine common variants (MAF ≥0.05) in three genes were independently associated with montelukast or its metabolite pharmacokinetics at a Bonferroni‐corrected significance level of 1.09 × 10^−6^ (**Table** [1](#cpt891-tbl-0001)). The AUC_0‐∞_ of montelukast and its metabolites showed the strongest associations with variants in the *UGT1A* gene (**Figure** [1](#cpt891-fig-0001), **Table** [1](#cpt891-tbl-0001)). For montelukast AUC_0‐∞_, the strongest association was observed with rs7604115, located in the first intron of *UGT1A3*. The AUC_0‐∞_ of montelukast was 18% smaller per copy of the variant allele (*P* = 1.83 × 10^−10^). After adjusting for this variant, no other variant remained statistically significantly associated with montelukast AUC_0‐∞_. The investigated genetic variants had no significant effect on the peak plasma concentration (C_max_) or the elimination half‐life (t_½_) of montelukast.

### Table 1.

| Pharmacokinetic variable | dbSNP ID | Gene | Location | Nucleotide change | MAF | Effecta |  | P value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Average | 90%CI |  |  |  |  |  |  |  |
| Montelukast |   |   |   |   |   |   |   |   |
| AUC0‐∞ | rs7604115 | UGT1A3 | intron 1/4 | c.868‐17564C>T | 0.40 | ‐17.7% | ‐21.6%, ‐13.7% | 1.83 × 10−10 |
| Cmax | — |   |   |   |   |   |   |   |
| t1/2 | — |   |   |   |   |   |   |   |
| M1 |   |   |   |   |   |   |   |   |
| AUC0‐∞ | 1. rs73063122 | SLCO1B1 | intron 11/14 | c.1497 + 2246A>C | 0.32 | 26.4% | 18.5%, 34.8% | 9.20 × 10−9 |
|   | 2. rs3806592 | UGT1A4 | upstream | c.‐1531C>T | 0.40 | 24.7% | 17.4%, 32.3% | 6.02 × 10−9 |
| Cmax | 1. rs3806592 | UGT1A4 | upstream | c.‐1531C>T | 0.40 | 30.4% | 20.8%, 40.9% | 4.45 × 10−8 |
|   | 2. rs4149056 | SLCO1B1 | exon 6/15 | c.521T>C | 0.22 | 34.7% | 22.2%, 48.4% | 9.90 × 10−7 |
| t1/2 | — |   |   |   |   |   |   |   |
| M1/montelukast AUC0‐∞ ratio | rs3806592 | UGT1A4 | upstream | c.‐1531C>T | 0.40 | 52.9% | 43.5%, 63.0% | 4.14 × 10−22 |
| M5R |   |   |   |   |   |   |   |   |
| AUC0‐∞ | 1. rs7604115 | UGT1A3 | intron 1/4 | c.868‐17564C>T | 0.40 | −45.7% | −49.5%, −41.7% | 1.26 × 10−30 |
|   | 2. rs704212 | ABCC9 | intron 12/37 | c.1802 + 2622G>A | 0.14 | −27.6% | −34.6%, −19.8% | 4.83 × 10−7 |
| Cmax | rs7604115 | UGT1A3 | intron 1/4 | c.868‐17564C>T | 0.40 | −33.9% | −38.6%, −28.8% | 7.13 × 10−17 |
| t1/2 | rs7556676 | UGT1A3 | intron 1/4 | c.868‐17430A>G | 0.46 | −24.8% | −28.2%, −21.2% | 2.81 × 10−19 |
| M5R/montelukast AUC0‐∞ ratio | rs4663969 | UGT1A3 | intron 1/4 | c.867 + 16674C>A | 0.46 | −33.9% | −37.3%, −30.2% | 7.06 × 10−27 |
| M5S |   |   |   |   |   |   |   |   |
| AUC0‐∞ | rs7604115 | UGT1A3 | intron 1/4 | c.868‐17564C>T | 0.40 | −33.2% | −38.5%, −27.5% | 8.14 × 10−14 |
| Cmax | rs7604115 | UGT1A3 | intron 1/4 | c.868‐17564C>T | 0.40 | −26.2% | −31.2%, −20.8% | 1.82 × 10−11 |
| t1/2 | rs1875263 | UGT1A4 | upstream | c.‐1845C>T | 0.40 | −22.1% | −27.4%, −16.4% | 2.17 × 10−8 |
| M5S/montelukast AUC0‐∞ ratio | rs2361501 | UGT1A3 | intron 1/4 | c.867 + 51A>T | 0.47 | −20.1% | −24.7%, −15.3% | 1.71 × 10−9 |
| M6 |   |   |   |   |   |   |   |   |
| AUC0‐∞ | rs7604115 | UGT1A3 | intron 1/4 | c.868‐17564C>T | 0.40 | −39.4% | −43.2%, −35.4% | 2.89 × 10−27 |
| Cmax | rs7604115 | UGT1A3 | intron 1/4 | c.868‐17564C>T | 0.40 | −34.0% | −37.8%, −29.9% | 2.22 × 10−23 |
| t1/2 | — |   |   |   |   |   |   |   |
| M6/montelukast AUC0‐∞ ratio | rs4663969 | UGT1A3 | intron 1/4 | c.867 + 16674C>A | 0.46 | −25.4% | −28.1%, −22.7% | 2.77 × 10−29 |

Table 1 Caption: Results of the stepwise forward linear regression analysis of the effects of 46,064 SNVs in 379 genes on montelukast and its metabolite pharmacokinetics

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4b25/6033076/986e780eed3e/CPT-104-158-g001.jpg)

The associations of SNVs in 379 pharmacokinetic genes (a) and in the UGT1A gene (b) with montelukast AUC0‐∞, and LD plot of the UGT1A gene (c). The Y‐axes in (a,b) describe the negative logarithm of the P value for each SNV and the horizontal lines indicate the Bonferroni‐corrected significance level of 1.09 × 10−6. The X‐axis in (a) shows individual SNVs grouped by protein function. The locations of UGT1A exons and linkage disequilibrium blocks are depicted on the top of (b,c).

In agreement with the effects of *UGT1A* variants on parent montelukast, the AUC_0‐∞_ of montelukast acyl‐glucuronide (M1) was 25% larger per copy of the *UGT1A* rs3806592 variant allele (*P* = 6.02 × 10^−9^) (**Table** [1](#cpt891-tbl-0001)). Rs3806592 is in a strong linkage disequilibrium with the *UGT1A3* rs7604115 SNV associated with the AUC_0‐∞_ of parent montelukast (*r* ^2^ = 0.95, *P* = 3.65 × 10^−41^). *UGT1A* variants were also significantly associated with the C_max_ of M1 and the M1/montelukast AUC_0‐∞_ ratio. Furthermore, the solute carrier organic anion transporter gene 1B1 (*SLCO1B1*) SNVs rs73063122 and rs4149056 were significantly associated with the AUC_0‐∞_ and C_max_ of M1, respectively. These two SNVs are in a strong linkage disequilibrium with each other (*r* ^2^ = 0.60, *P* = 5.73 × 10^−27^), suggesting that both of these associations are due to the rs4149056 missense SNV known to markedly impair the activity of OATP1B1.[11](#cpt891-bib-0011)

The effects of *UGT1A* variants on the AUC_0‐∞_ of the hydroxylated M5R, M5S, and M6 metabolites of montelukast were larger than what was observed for montelukast (**Table** [1](#cpt891-tbl-0001)). The AUC_0‐∞_ of M5R, M5S, and M6 were 46% (*P* = 1.26 × 10^−30^), 33% (*P* = 8.14 × 10^−14^), and 39% (*P* = 2.89 × 10^−27^) smaller per copy of the rs7604115 variant allele, respectively. *UGT1A* variants were also significantly associated with the C_max_ and the metabolite/montelukast AUC_0‐∞_ ratios of M5R, M5S, and M6 and the t_½_ of M5R and M5S. The AUC_0‐∞_ of M5R was also associated with the rs704212 SNV in the ATP binding cassette subfamily C member 9 (*ABCC9*) transporter gene.

### UGT1A linkage disequilibrium and haplotype analyses

The *UGT1A* gene region (±20 kb) was found to consist of nine linkage disequilibrium (LD) blocks (**Figure** [1](#cpt891-fig-0001)). The SNVs associated with montelukast or its metabolite pharmacokinetics were located in blocks 3 and 4. Within these blocks, 28 haplotypes were inferred (**Supplementary Figure S1**). The SNVs showing the strongest associations with montelukast pharmacokinetics were strongly linked to the missense variants rs3821242 and rs6431625 (*r* ^2^ ≥ 0.69), which together define the *UGT1A3*2* haplotype. Based on the missense variants rs3821242, rs6431625, and rs45449995, the inferred haplotypes were grouped to subtypes of *UGT1A3*1* (wildtype), **2* (rs3821242 and rs6431625; MAF 0.39), **3* (rs3821242; MAF 0.060), and **6* (rs3821242, rs6431625, and rs45449995; MAF 0.018). The effects of the *UGT1A3*2* haplotype on montelukast and its metabolite pharmacokinetics were similar to the effects of the individual, intronic *UGT1A* SNVs (**Tables** [1](#cpt891-tbl-0001) **and** [2](#cpt891-tbl-0002), **Figure** [2](#cpt891-fig-0002)).

### Table 2.

| Pharmacokinetic variable | Haplotype/dpSNP ID | Effecta |  |  |
| --- | --- | --- | --- | --- |
| Average | 90% CI | P value |  |  |
| Montelukast |   |   |   |   |
| AUC0‐∞ | UGT1A3*2 | −17.5% | −21.4%, −13.5% | 2.75 × 10−10 |
| Cmax | — |   |   |   |
| t1/2 | — |   |   |   |
| M1 |   |   |   |   |
| AUC0‐∞ | UGT1A3*2 | 26.4% | 19.2%, 34.1% | 5.13 × 10−10 |
|   | SLCO1B1 rs4149056 | 34.4% | 24.8%, 44.8% | 5.20 × 10−10 |
| Cmax | UGT1A3*2 | 30.0% | 20.4%, 40.5% | 6.71 × 10−8 |
|   | SLCO1B1 rs4149056 | 35.5% | 22.9%, 49.4% | 6.54 × 10−7 |
| t1/2 | — |   |   |   |
| M1/montelukast AUC0‐∞ ratio | UGT1A3*2 | 52.8% | 43.3%, 62.8% | 5.82 × 10−22 |
| M5R |   |   |   |   |
| AUC0‐∞ | UGT1A3*2 | −45.4% | −49.2%, −41.2% | 1.20 × 10−29 |
|   | ABCC9 rs704212 | −27.6% | −34.8%, −19.7% | 6.90 × 10−7 |
| Cmax | UGT1A3*2 | −33.8% | −38.5%, −28.7% | 7.49 × 10−17 |
| t1/2 | UGT1A3*2 | −25.3% | −28.9%, −21.6% | 1.26 × 10−18 |
| M5R/montelukast AUC0‐∞ ratio | UGT1A3*2 | −35.0% | −38.6%, −31.2% | 1.01 × 10−26 |
| M5S |   |   |   |   |
| AUC0‐∞ | UGT1A3*2 | −33.1% | −38.4%, −27.4% | 9.01 × 10−14 |
| Cmax | UGT1A3*2 | −26.0% | −31.0%, −20.6% | 2.65 × 10−11 |
| t1/2 | UGT1A3*2 | −21.8% | −27.1%, −16.1% | 2.79 × 10−8 |
| M5S/montelukast AUC0‐∞ ratio | UGT1A3*2 | −18.9% | −23.8%, −13.8% | 5.87 × 10−8 |
| M6 |   |   |   |   |
| AUC0‐∞ | UGT1A3*2 | −39.3% | −43.1%, −35.2% | 3.41 × 10−27 |
| Cmax | UGT1A3*2 | −33.9% | −37.7%, −29.8% | 2.65 × 10−23 |
| t1/2 | — |   |   |   |
| M6/montelukast AUC0‐∞ ratio | UGT1A3*2 | −26.1% | −28.9%, −23.1% | 3.56 × 10−27 |

Table 2 Caption: Results of the stepwise forward linear regression analysis of the effects of UGT1A3*2, SLCO1B1 rs4149056, and ABCC9 rs704212 on montelukast and its metabolite pharmacokinetics

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4b25/6033076/e2c5e0cc756a/CPT-104-158-g002.jpg)

Geometric mean (90% CI) plasma concentrations of montelukast, its acyl‐glucuronide (M1), and hydroxymetabolites (M5R, M5S, M6) after a single 10‐mg oral dose of montelukast in 191 healthy volunteers with different UGT1A3 genotypes. Open circles indicate noncarriers of UGT1A3*2 (n = 72), solid circles subjects heterozygous for the UGT1A3*2 (n = 90), and solid triangles subjects homozygous for the UGT1A3*2 haplotype (n = 29). The insets depict the same data on a semilogarithmic scale. Plasma concentrations of montelukast were adjusted for BSA and those of M1 and M5S for lean body weight, as according to the linear regression models of their AUC0‐∞ values.

### Functional validation

Next, we investigated whether montelukast or its hydroxymetabolites are substrates of UGT1A enzymes *in vitro*. As the most strongly associated variants are localized around the first exon of *UGT1A3*, we focused on this enzyme, together with UGT1A1 and UGT1A9, which are also known to catalyze the glucuronidation of carboxylic acids.[12](#cpt891-bib-0012) Montelukast, M5R, M5S, and M6 were all metabolized by UGT1A3, but not significantly by UGT1A1 or UGT1A9 (**Supplementary Figure S2**).

To further elucidate the mechanisms of associations between *UGT1A, ABCC9*, and *SLCO1B1* SNVs and montelukast or its metabolite pharmacokinetics, we then investigated the effects of the SNVs on the respective gene expression in human liver samples (**Table** [3](#cpt891-tbl-0003)). *UGT1A3* expression showed a strong association with the SNVs associated with montelukast pharmacokinetics. The strongest association was observed with rs4663969, present in both *UGT1A3*2* and **3* haplotypes. *UGT1A3* gene expression was 24% higher per copy of the variant allele (*P* = 2.27 × 10^−4^). Of the *UGT1A3*2, *3*, and **6* haplotypes, only *UGT1A3*2* was significantly associated with increased *UGT1A3* gene expression. The *UGT1A3* expression was 24% higher per copy of the *UGT1A3*2* haplotype (*P* = 2.08 × 10^−4^) (**Figure** [3](#cpt891-fig-0003)). *ABCC9* expression was 25% higher (*P* = 3.41 × 10^−4^) per copy of the rs704212 variant allele. *SLCO1B1* expression was 13% lower (*P* = 0.0146) per copy of the rs4149056 variant allele. Of *SLCO1B1* haplotypes, **15* (rs2306283 and rs4149056) was associated with a 15% lower *SLCO1B1* expression per copy of the haplotype (*P* = 0.00733).

### Table 3.

| Gene | n | dpSNP ID/Haplotype | MAF | Effecta |  | P value |
| --- | --- | --- | --- | --- | --- | --- |
| (n = 185‐197) | Average | 90% CI |  |  |  |  |
| UGT1A3 | 136 | rs7604115 | 0.44 | 22.3% | 11.3%, 34.5% | 5.65 × 10−4 |
|   | 136 | rs3806592 | 0.43 | 22.9% | 11.7%, 35.2% | 4.78 × 10−4 |
|   | 136 | rs7556676 | 0.47 | 23.2% | 12.4%, 35.0% | 2.57 × 10−4 |
|   | 136 | rs4663969 | 0.47 | 23.6% | 12.7%, 35.6% | 2.27 × 10−4 |
|   | 136 | rs1875263 | 0.44 | 22.9% | 11.7%, 35.2% | 4.78 × 10−4 |
|   | 136 | UGT1A3*2 | 0.41 | 23.8% | 12.8%, 35.8% | 2.08 × 10−4 |
|   | 136 | UGT1A3*3 | 0.058 | −1.2% | −18.7%, 20.0% | 0.918 |
|   | 136 | UGT1A3*6 | 0.010 | 29.9% | −13.3%, 94.7% | 0.286 |
| ABCC9 | 187 | rs704212 | 0.11 | 25.2% | 13.1%, 38.6% | 3.41 × 10−4 |
| SLCO1B1 | 124 | rs4149056 | 0.24 | −13.5% | −21.4%, −4.7% | 0.0146 |
|   | 124 | rs2306283 | 0.47 | −7.5% | −14.7%, 0.4% | 0.117 |
|   | 123 | SLCO1B1*1B | 0.18 | 10.4% | 0.8%, 20.9% | 0.0740 |
|   | 123 | SLCO1B1*5 | 0.019 | 11.2% | −18.3%, 51.3% | 0.570 |
|   | 123 | SLCO1B1*15 | 0.22 | −15.2% | −23.2%, −6.3% | 0.00733 |

Table 3 Caption: Results of the linear regression analysis of the effects of UGT1A SNVs and haplotypes on UGT1A3 mRNA expression, ABCC9 rs704212 SNV on ABCC9 mRNA expression, and SLCO1B1 SNVs and haplotypes on SLCO1B1 mRNA expression in liver samples from patients undergoing laparoscopic gastric bypass operation

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4b25/6033076/e068dedb4860/CPT-104-158-g003.jpg)

Boxplots of the effects of UGT1A3*2, *3, and *6 haplotypes on UGT1A3 mRNA expression in human liver samples. The horizontal lines inside the boxes represent the median, the box edges show the lower and upper quartiles, and the whiskers show the 10th and 90th percentiles. Individual data points are given as circles for men and as triangles for women.

### Candidate gene analysis

We next carried out a candidate gene analysis for montelukast AUC_0‐∞_, focusing on common (MAF ≥0.05) missense variants in genes suggested to be involved in montelukast pharmacokinetics (*CYP2C8, CYP2C9, CYP3A4, SLCO1B1, SLCO1B3, SLCO2B1, UGT1A3*), as well as the *ABCC9* rs704212 SNV (**Supplementary Table S3**). In a stepwise linear regression analysis, *UGT1A3*2* was associated with a 17% (*P* = 2.99 × 10^−10^), *ABCC9* rs704212 with a 14% (*P* = 2.19 × 10^−4^), and *CYP2C8*3* (rs10509681 and rs11572080) with an 11% (*P* = 0.00659) reduced, and *CYP2C8*4* (rs1058930) with a 13% (*P* = 0.0184) increased AUC_0‐∞_ of montelukast per copy of each minor allele (**Table** [4](#cpt891-tbl-0004); adjusted *R* ^2^ = 0.41).

### Table 4.

| Variable | MAF | Effecta |  | P value | Adjusted R2 for each step |
| --- | --- | --- | --- | --- | --- |
| Average | 90% CI |  |  |  |  |
| BSA | — | −11.5% | −14.0%, −9.0% | 2.10 × 10−11 | 0.17 |
| UGT1A3*2 | 0.39 | −16.8% | −20.5%, −12.9% | 2.99 × 10−10 | 0.32 |
| ABCC9 rs704212 | 0.14 | −13.7% | −19.1%, −7.9% | 2.19 × 10−4 | 0.37 |
| CYP2C8*3 | 0.11 | −11.3% | −17.5%, −4.7% | 0.00659 | 0.40 |
| CYP2C8*4 | 0.07 | 12.7% | 3.7%, 22.5% | 0.0184 | 0.41 |

Table 4 Caption: Results of the candidate gene analysis on montelukast AUC0‐∞

## DISCUSSION

In this study we used targeted massively parallel sequencing of 379 pharmacokinetic genes to characterize the genetic basis of interindividual variability in montelukast pharmacokinetics. The *UGT1A3*2* haplotype and variants located around the first exon of *UGT1A3* were strongly associated with the systemic exposure to parent montelukast and its metabolites. We further demonstrated that montelukast and its hydroxymetabolites are substrates of UGT1A3 *in vitro* and that the variants associated with the pharmacokinetic variables also significantly affect *UGT1A3* gene expression in human liver. In addition, our results indicate involvement of *ABCC9* and *SLCO1B1* in the pharmacokinetics of montelukast metabolites. Moreover, a candidate gene approach suggested that, in addition to *UGT1A3* variants, also *CYP2C8* and *ABCC9* variants affect parent montelukast exposure.

The *UGT1A* gene encodes the UGT1A family enzymes, which catalyze the formation of hydrophilic glucuronide metabolites.[13](#cpt891-bib-0013) Individual *UGT1A* genes have unique first exons but share exons 2–5. Strong linkage disequilibrium exists throughout the whole *UGT1A* gene (**Figure** [1](#cpt891-fig-0001) **c**). The intronic *UGT1A* SNVs showing the strongest associations with montelukast or its metabolite pharmacokinetics are strongly linked to the *UGT1A3* missense variants rs3821242 (c.31T>C, p.Trp11Arg) and rs6431625 (c.140T>C, p.Val47Ala) that together define the haplotype *UGT1A3*2*. The *UGT1A3*2* haplotype and the individual intronic SNVs similarly reduced the exposure to montelukast and its hydroxymetabolites, and increased the exposure to montelukast acyl‐glucuronide, indicating enhanced glucuronidation. Consistently, *UGT1A3*2* has previously been shown to increase the metabolism of the UGT1A3 substrates atorvastatin, telmisartan, and febuxostat in humans.[14](#cpt891-bib-0014), [15](#cpt891-bib-0015), [16](#cpt891-bib-0016)

The *UGT1A3*2* haplotype and the *UGT1A* SNVs associated with montelukast pharmacokinetics significantly increased *UGT1A3* mRNA expression in human liver samples. Similarly, a previous study also showed that *UGT1A3* mRNA and protein expression are significantly increased in *UGT1A3*2* carriers.[14](#cpt891-bib-0014) In addition to *UGT1A3*2*, the *UGT1A3*6* haplotype has been associated with increased *UGT1A3* expression.[14](#cpt891-bib-0014) In our study, the number of *UGT1A3*6* carriers was relatively small, and only a tendency towards increased *UGT1A3* expression could be observed (**Figure** [3](#cpt891-fig-0003)).

The causal variant of *UGT1A3*2* affecting *UGT1A3* expression has remained unknown.[14](#cpt891-bib-0014) We identified several intronic SNVs that are strongly linked to *UGT1A3*2* (**Supplementary Figure S1**). Of these, rs3806597 (c.‐204A>G) is located on a proposed farnesoid X receptor (FXR) binding site upstream of *UGT1A3*, but the variant allele has not affected FXR‐mediated induction of UGT1A3 by the bile acid chenodeoxycholic acid.[17](#cpt891-bib-0017) None of the other strongly linked variants appear to be located in a transcription factor binding site upstream of *UGT1A3*.[18](#cpt891-bib-0018) In addition to being associated with expression, the *UGT1A3*2* missense variants might also alter the enzymatic activity of UGT1A3. However, *in vitro* studies with these variants have shown conflicting results, with reduced, increased, and unchanged activity.[19](#cpt891-bib-0019), [20](#cpt891-bib-0020), [21](#cpt891-bib-0021) Altogether, although the causal variant cannot be identified, the increased *UGT1A3* expression in association with *UGT1A3*2* provides a mechanistic explanation for our pharmacokinetic results.

In accordance with a recently published *in vitro* study,[7](#cpt891-bib-0007) our results demonstrate that montelukast is efficiently glucuronidated by UGT1A3. We also showed that montelukast hydroxymetabolites M5R, M5S, and M6 are efficiently glucuronidated by UGT1A3 *in vitro*, which could explain why the concentrations of the hydroxymetabolites were affected more by the *UGT1A3* variants than those of parent montelukast. Taken together, our results indicate that glucuronidation plays an important role in the metabolism of both parent montelukast and its hydroxymetabolites. However, due to low plasma concentrations, the hydroxymetabolites are unlikely to contribute to the pharmacological effects of montelukast.

In addition to *UGT1A3* variants, SNVs in the *SLCO1B1* gene were associated with significantly higher AUC and C_max_ of montelukast acyl‐glucuronide, M1. *SLCO1B1* encodes OATP1B1, an influx transporter mediating the hepatic uptake of its substrates from sinusoidal blood.[11](#cpt891-bib-0011) Of the associated SNVs, the rs4149056 missense variant (c.521T>C, p.Val174Ala, *SLCO1B1*5* or **15*) markedly impairs the function of OATP1B1, as demonstrated both *in vitro* and *in vivo* in humans.[11](#cpt891-bib-0011), [22](#cpt891-bib-0022), [23](#cpt891-bib-0023), [24](#cpt891-bib-0024) Many glucuronide conjugates are OATP1B1 substrates and our results suggest that also M1 is a substrate of OATP1B1. In a previous study, the rs2306283 (c.388A>G, p.Asn130Asp) missense variant was associated with increased *SLCO1B1* expression in human liver samples (*n* = 143), whereas rs4149056 had no effect.[25](#cpt891-bib-0025) In our study, however, rs4149056 was associated with decreased *SLCO1B1* expression, but rs2306283 appeared to have no effect. Nevertheless, the *SLCO1B1*1B* haplotype, which contains rs2306283 without rs4149056, showed a tendency for a 10% increased expression of *SLCO1B1* per copy of the haplotype.

A recent study suggested that montelukast is a substrate of OATP1B1, 1B3, and 2B1, and that OATP‐CYP2C8 interplay is an important determinant of parent montelukast pharmacokinetics.[10](#cpt891-bib-0010) Our finding that the OATP1B1 function‐impairing rs4149056 SNV did not significantly affect the pharmacokinetics of parent montelukast, even in the candidate gene analysis with no correction for multiple testing, indicates that OATP1B1 is not of major importance for the hepatic elimination of montelukast *in vivo* in humans. Moreover, we fully sequenced the *SLCO1B3* and *SLCO2B1* genes and found no association between montelukast AUC and variants in these genes, indicating that these genes are also not important determinants of interindividual variability in montelukast pharmacokinetics.

Interestingly, the *ABCC9* rs704212 SNV was associated with reduced plasma concentrations of montelukast hydroxymetabolite M5R, and with increased mRNA expression of *ABCC9* in human liver samples. Based on these results, the SNV was selected for the candidate gene analysis, where it was found to be associated also with reduced AUC of montelukast. The *ABCC9* gene encodes the sulfonylurea receptor 2 (SUR2), which is a subunit of an adenosine triphosphate‐sensitive K^+^ channel and has no identified transporter function.[26](#cpt891-bib-0026) Notably, another *ABCC9* SNV, rs1283807, which is in a strong linkage disequilibrium with rs704212 (*r* ^2^ = 0.82, *P* = 1.97 × 10^−37^), has previously been associated with impaired efficacy of angiotensin II receptor antagonists in a genome‐wide association study.[27](#cpt891-bib-0027) The same SNV has also been associated with the risk of virologic failure during efavirenz‐containing antiretroviral therapy via an epistatic interaction with an SNV in transporter 1, ATP‐binding cassette subfamily B member (*TAP1*).[28](#cpt891-bib-0028) Our study is the first to show an association between *ABCC9* and drug pharmacokinetics, suggesting a pharmacokinetic explanation also for the findings concerning angiotensin II receptor antagonists and efavirenz.

It has been estimated that 80% of montelukast metabolism is mediated by CYP2C8.[6](#cpt891-bib-0006) In the analysis of all the 379 genes, *CYP2C8* variants were not significantly associated with montelukast pharmacokinetics. However, in the candidate gene analysis, the AUC of montelukast was reduced by the *CYP2C8*3* allele and increased by the *CYP2C8*4* allele. These findings are consistent with previous studies indicating increased CYP2C8 activity with *CYP2C8*3* and reduced activity with *CYP2C8*4 in vivo* in humans.[29](#cpt891-bib-0029), [30](#cpt891-bib-0030), [31](#cpt891-bib-0031), [32](#cpt891-bib-0032), [33](#cpt891-bib-0033), [34](#cpt891-bib-0034), [35](#cpt891-bib-0035) In the present study, *UGT1A3*2* reduced the AUC of montelukast by 18% per allele copy, equivalent to a 22% increase in oral clearance. The simultaneous increase in the M1/montelukast AUC ratio suggests an about 50% increase in UGT1A3‐mediated clearance per *UGT1A3*2* allele. Thus, it seems plausible that the contribution of UGT1A3 to the metabolism of montelukast varies between 35–60%, depending on the *UGT1A3* genotype. In previous studies, the CYP2C8 inhibitor gemfibrozil has markedly increased the AUC of montelukast.[3](#cpt891-bib-0003), [6](#cpt891-bib-0006) Interestingly, gemfibrozil inhibits UGT1A3 *in vitro*.[7](#cpt891-bib-0007) Therefore, it is possible that UGT1A3 also contributes to the gemfibrozil‐montelukast interaction. In any case, the *UGT1A3* genotype might affect the magnitude of this interaction.

Considerable interindividual variability exists in the efficacy of montelukast, but no firm clinical or genetic predictors of montelukast response have yet been identified.[36](#cpt891-bib-0036), [37](#cpt891-bib-0037) In clinical practice, montelukast is administered on a regular basis. The steady‐state plasma concentrations for montelukast are predictable from its pharmacokinetic parameters measured after a single dose.[38](#cpt891-bib-0038) Therefore, the effects of genetic variants on the steady‐state plasma concentrations of montelukast should be similar to the effects on the AUC_0‐∞_ of montelukast observed in our study after a single dose. Because montelukast dose‐dependently improves chronic asthma,[39](#cpt891-bib-0039) the reduced plasma concentrations of montelukast due to the *UGT1A3*2* haplotype might impair its efficacy. Even though the effect of the *UGT1A3*2* haplotype on montelukast pharmacokinetics is modest, the haplotype is common (**Supplementary Figure S3**). Therefore, it may be an important factor explaining the variability in montelukast response at the population level. Body surface area and the *UGT1A3*2* allele together explained 32% of interindividual variability in montelukast exposure. When the *ABCC9* and *CYP2C8* alleles were added to the model, this percentage increased to 41%. Together with pharmacodynamic markers,[37](#cpt891-bib-0037) this knowledge might aid in individualizing treatment with leukotriene receptor antagonists.

In conclusion, genetic variability in *UGT1A3* significantly affects montelukast pharmacokinetics. This indicates that glucuronidation via UGT1A3 has a larger role in the metabolism of montelukast than previously thought, especially in subjects carrying the *UGT1A3*2* haplotype. These results also further confirm that the *UGT1A3*2* haplotype enhances the glucuronidation of UGT1A3 substrates. Moreover, the candidate gene analysis suggested that also *CYP2C8*3, CYP2C8*4*, and *ABCC9* rs704212 affect the pharmacokinetics of montelukast.

## METHODS

### Study participants

In all, 201 healthy unrelated Finnish Caucasian volunteers participated in the pharmacokinetic study after giving written informed consent. Their health was confirmed by medical history, clinical examination, and laboratory tests. Participants were not on any continuous medication nor were tobacco smokers. The study was approved by the Coordinating Ethics Committee of the Hospital District of Helsinki and Uusimaa and the Finnish Medicines Agency Fimea. Ten participants discontinued the study before montelukast administration and thus the pharmacokinetic data were obtained from 191 participants. Of these, 92 were women and 99 men. Their mean ± SD age was 24 ± 4 years, height 174 ± 9 cm, weight 70 ± 12 kg, and body mass index (BMI) 22.8 ± 2.5 kg/m^2^.

A whole‐blood DNA sample and a liver biopsy were obtained from 201 patients undergoing laparoscopic gastric bypass operation at the Kuopio University Hospital, as part of the Kuopio Obesity Surgery Study.[40](#cpt891-bib-0040), [41](#cpt891-bib-0041), [42](#cpt891-bib-0042) Good quality RNA expression data and genotypes were obtained from 188 patients. These patients consisted of 129 women and 59 men (mean ± SD: age 48 ± 9 years and BMI 44 ± 6 kg/m^2^). Seventy‐six patients had type 2 diabetes, 34 had nonalcoholic fatty liver, 32 had nonalcoholic steatohepatitis, and 58 used lipid‐lowering medication. The degree of liver steatosis was graded from 0 to 3 and that of lobular inflammation from 0 to 2. Written informed consent was obtained from all patients and the study protocol was approved by the Ethics Committee of the Northern Savo Hospital District.

### Montelukast pharmacokinetics

After fasting overnight, the healthy volunteers ingested a single 10‐mg dose of montelukast (Singulair tablet, Merck Sharp & Dohme, Haarlem, The Netherlands) with 150 ml of water at 8 am. Standardized meals were served at 4, 7, and 10 h after montelukast ingestion. Timed blood samples (4–9 ml each) were collected into light‐protected ethylenediaminetetraacetic acid (EDTA) tubes prior to and up to 24 h after montelukast administration. Tubes were immediately placed on ice and plasma was separated within 30 min. Samples were stored at –70°C until analysis.

The concentrations of plasma montelukast, montelukast acyl‐glucuronide (M1), montelukast 1,2‐diol (M6), 21R‐hydroxy montelukast (M5R), and 21S‐hydroxy montelukast (M5S) were measured using a Nexera X2 liquid chromatography instrument (Shimadzu, Kyoto, Japan) interfaced with a 5500 Qtrap tandem mass spectrometer (AB Sciex, Toronto, ON). Prior to quantification, the plasma sample was purified from proteins and phospholipids using a Phree phospholipid removal plate (Phenomenex, Torrance, CA) according to the manufacturer's instructions. In short, the plasma sample was mixed with acetonitrile containing 1% formic acid and internal standards (1:4 v/v) and drawn through the Phree cartridges. The chromatographic separation was achieved on a reversed‐phase Kinetex C18 analytical column (100 × 2.1 mm internal diameter, 2.6 μm particle size; Phenomenex) using 2 mM ammonium acetate (A) (pH 4.0) adjusted with 98% formic acid and acetonitrile (B) as a mobile phase. The injection volume was 3 μl and the column temperature was held at 30°C. The gradient profile was set as follows: a linear increase from 35% B to 62% B over 2.2 min, held at 62% B for 2 min, a gradient from 62% B to 95% B over 0.6 min, and maintained at 95% B for 1.4 min followed by a reversion to the initial conditions. The mass spectrometer was operated in a positive electrospray ionization mode (ESI+) employing scheduled multiple reaction monitoring (MRM) for the optimal dwell time for each analyte. A corresponding deuterated reference compound served as internal standard for each analyte, except for M6, which utilized montelukast‐d6. The target mass‐to‐charge ratios (m/z) for montelukast, M1, M5R/M5S, and M6 were 586→422, 762→422, 602→147, and 602→438, and the limits of quantification (ng/ml) were 1.0, 0.2, 0.1, and 0.1, respectively. The day‐to‐day coefficient of variation (CV) was below 10% at relevant concentrations for all analytes. The AUC_0‐∞_, C_max_, and t_½_ values were calculated for montelukast, M1, M5R, M5S, and M6 with standard noncompartmental methods using Phoenix WinNonlin, v. 6.3 (Certara, Princeton, NJ).

### DNA sequencing and genotyping

Genomic DNA was extracted using the Maxwell 16 LEV Blood DNA Kit on a Maxwell 16 Research automated nucleic acid extraction system (Promega, Madison, WI; pharmacokinetic study) or DNeasy Blood & Tissue Kit (Qiagen, Hilden, Germany; liver samples). DNA concentration and absorbance 260/280 ratio (A260/A280) were determined with the NanoDrop 2000 UV‐Vis Spectrophotometer (Thermo Fisher Scientific, Waltham, MA).

Targeted massively parallel sequencing of the 379 pharmacokinetic genes ±20 kb (**Supplementary Table S2**; genome build GRCh37) was performed in all pharmacokinetic study participants (*n* = 201). For library preparation, 3 μg of genomic DNA was processed according to the NEBNext DNA Sample Prep protocol (New England BioLabs, Ipswich, MA). Target enrichment capture was performed using the NimbleGen SeqCap EZ Choice capture protocol (Roche Sequencing, Pleasanton, CA). Sequencing was done on the Illumina HiSeq2000 platform with 100 bp paired‐end reads (Illumina, San Diego, CA). Quality control, short read alignment, and variant calling and annotation were carried out using an in‐house‐developed pipeline, as described previously.[43](#cpt891-bib-0043) The sequencing and bioinformatics pipelines were carried out at the Technology Centre at the Institute for Molecular Medicine Finland (Helsinki, Finland). Mean coverage depth was 37.2×. Coverage depth ≥10×, Hardy–Weinberg equilibrium *P* < 3.15 × 10^−7^ (Bonferroni‐correction), and proportion missing ≤0.05 were employed as quality thresholds for including genotype data in statistical analysis.

The pharmacokinetic study participants and liver samples were genotyped for the *UGT1A* rs7604115, rs3806592, rs7556676, rs4663969, rs1875263, rs3821242, rs6431625, and rs45449995, *ABCC9* rs704212, and *SLCO1B1* rs4149056 and rs2306283 SNVs with TaqMan genotyping assays on a QuantStudio 12K Flex Real‐Time PCR System according to the manufacturer's protocol (Thermo Fisher Scientific). Call identity with sequencing data was 99–100%. In case of discordant results, genotypes obtained by sequencing were used in the statistical analysis.

### Reverse transcription quantitative real‐time PCR

RNA from liver samples was extracted using the miRNeasy Mini Kit (Qiagen, Chatsworth, CA) and stored at –80°C. RNA was reverse‐transcribed using the SuperScript VILO cDNA Synthesis Kit, according to the manufacturer's instructions (Thermo Fisher Scientific). The cDNA samples were preamplified (14 cycles) with a custom TaqMan pre amp pool containing the assays for *UGT1A, ABCC9, SLCO1B1* and reference genes before quantitative real‐time PCR (qPCR), according to the manufacturer's instructions (Thermo Fisher Scientific).

The qPCR was carried out using OpenArray technology on the QuantStudio 12K Flex Real‐Time PCR System. The custom OpenArray plate contained *UGT1A3* (Hs04194492_g1), *ABCC9* (Hs00245832_m1), *SLCO1B1* (Hs00272374_m1), and reference gene assays, allowing the within‐sample normalization with multiple reference genes. The reference genes were comprised of actin beta (*ACTB*; Hs01060665_g1), ribosomal protein lateral stalk subunit P0 (*RPLP0*; Hs99999902_m1), glyceraldehyde‐3‐phosphate dehydrogenase (*GAPDH*; Hs02758991_g1), and beta‐2‐microglobulin (*B2M*; Hs00984230_m1). Prior to thermal cycling, a volume of 2.5 μL of TaqMan OpenArray Real‐Time PCR Master Mix (Thermo Fisher Scientific) was combined with 1.3 μL nuclease‐free water followed by 1.2 μL of cDNA, and the sample was loaded to OpenArray plate. The samples were processed on the Freedom EVO 150 automated liquid handling system (Tecan Group, Männedorf, Switzerland) and the OpenArray AccuFill instrument (Thermo Fisher Scientific). Data were analyzed with the ExpressionSuite software (v. 1.0.3, Thermo Fisher Scientific) using the comparative ΔCq method.

### In vitro studies with UGT1A1, 1A3, and 1A9 recombinant enzymes

The incubation mixtures (triplicate samples) contained 100 mM phosphate buffer, pH 7.4, 5 mM MgCl_2_, 0.3 mg/ml of supersomal protein (Corning, Woburn, MA), 2 mM UDP‐glucuronic acid (UDPGA, triammonium salt), and 0.1 μM montelukast, M5R, M5S, or M6, in a total volume of 500 μl. Control incubations (duplicate samples) were performed with control supersomes, which do not contain active UGT enzyme. The reactions were initiated by the addition of the substrate and incubated in a shaking water bath at 37°C. Reactions were stopped by moving 50 μl samples to 50 μl acetonitrile containing 1% formic acid and internal standards at timepoints 0, 6, 12, 20, 40, and 60 min. After centrifugation, the supernatants were analyzed using SCIEX API 2000 tandem mass spectrometer (AB Sciex), as described previously.[4](#cpt891-bib-0004) Depletion (% remaining) of the substrates at each timepoint was calculated for UGT1A1, 1A3, 1A9, and control samples.

### Statistical analysis

The data were analyzed with the statistical programs JMP Genomics 7.0 (SAS Institute, Cary, NC) and IBM SPSS 22.0 for Windows (Armonk, NY). The pharmacokinetic variables were logarithmically transformed before analysis.[44](#cpt891-bib-0044) Possible effects of demographic covariates on pharmacokinetic variables and gene expression were investigated using stepwise linear regression analysis, with *P*‐value thresholds of 0.05 for entry and 0.10 for removal. Sex, body weight, lean body weight,[45](#cpt891-bib-0045) and body surface area, (BSA)[46](#cpt891-bib-0046) were tested as demographic covariates for pharmacokinetic data, and age, sex, BMI, type 2 diabetes, degree of lobular inflammation and hepatic steatosis, and use of lipid‐lowering medication for gene expression data (**Supplementary Table S4**). Possible effects of genetic variants on pharmacokinetic variables were investigated using stepwise linear regression analysis fixed for significant demographic covariates. A Bonferroni‐corrected *P*‐value threshold of 1.09 × 10^−6^ was employed for the 379 gene and *UGT1A3* haplotype‐based analysis and thresholds of 0.05 for entry and 0.10 for removal for the candidate gene analysis. The effects of genetic variants on gene expression were investigated with linear regression analysis fixed for significant demographic covariates, with *P* < 0.05 considered statistically significant. Additive coding was employed for genetic variants and multiallelic variants were expanded. The LD blocks for *UGT1A* (±20 kb) SNVs (MAF >0.05) were created using a previously described algorithm,[47](#cpt891-bib-0047), [48](#cpt891-bib-0048) as implemented in JMP Genomics. Haplotype computations were performed with PHASE v. 2.1.1.[49](#cpt891-bib-0049), [50](#cpt891-bib-0050)

## CONFLICT OF INTEREST

The authors have no conflicts of interest to declare.

## AUTHOR CONTRIBUTIONS

P.H. and M.Ni. wrote the article; P.H., A.T., J.T.B., and M.Ni. designed the research; P.H., A.T., M.Ne., T.T., M.P‐H., V.K., V.T.M., J.P., J.T.B., and M.Ni. performed the research; P.H. and M.Ni. analyzed the data.

## Supporting information

## ACKNOWLEDGMENTS

This study was supported by grants from the European Research Council (Grant agreement 282106), State funding for university‐level health research, and the Sigrid Jusélius Foundation (Helsinki, Finland). We thank Ms. Katja Halme, Ms. Hanna Hyvärinen, Ms. Satu Karjalainen, Mr. Jouko Laitila, Ms. Eija Mäkinen‐Pulli, Ms. Raija Nevala, and Ms. Lisbet Partanen for skillful assistance, and Mr. Pekka Ellonen and Ms. Maija Lepistö, M.Sc., for the massively parallel sequencing.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Fanta, C.H. Asthma. N. Engl. J. Med. 360, 1002–1014 (2009).  [DOI](https://doi.org/10.1056/NEJMra0804579) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19264689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Asthma&author=C.H.%20Fanta&volume=360&publication_year=2009&pages=1002-1014&pmid=19264689&doi=10.1056/NEJMra0804579&)

2. Balani, S.K. et al Metabolic profiles of montelukast sodium (Singulair), a potent cysteinyl leukotriene1 receptor antagonist, in human plasma and bile. Drug Metab. Dispos. 25, 1282–1287 (1997).  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9351905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Metabolic%20profiles%20of%20montelukast%20sodium%20(Singulair),%20a%20potent%20cysteinyl%20leukotriene1%20receptor%20antagonist,%20in%20human%20plasma%20and%20bile&author=S.K.%20Balani&volume=25&publication_year=1997&pages=1282-1287&pmid=9351905&)

3. Karonen, T. , Filppula, A. , Laitila, J. , Niemi, M. , Neuvonen, P.J. & Backman, J.T. Gemfibrozil markedly increases the plasma concentrations of montelukast: a previously unrecognized role for CYP2C8 in the metabolism of montelukast. Clin. Pharmacol. Ther. 88, 223–230 (2010).  [DOI](https://doi.org/10.1038/clpt.2010.73) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20592724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Gemfibrozil%20markedly%20increases%20the%20plasma%20concentrations%20of%20montelukast:%20a%20previously%20unrecognized%20role%20for%20CYP2C8%20in%20the%20metabolism%20of%20montelukast&author=T.%20Karonen&author=A.%20Filppula&author=J.%20Laitila&author=M.%20Niemi&author=P.J.%20Neuvonen&volume=88&publication_year=2010&pages=223-230&pmid=20592724&doi=10.1038/clpt.2010.73&)

4. Filppula, A.M. , Laitila, J. , Neuvonen, P.J. & Backman, J.T. Reevaluation of the microsomal metabolism of montelukast: major contribution by CYP2C8 at clinically relevant concentrations. Drug Metab. Dispos. 5, 904–911 (2011).  [DOI](https://doi.org/10.1124/dmd.110.037689) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21289076/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Reevaluation%20of%20the%20microsomal%20metabolism%20of%20montelukast:%20major%20contribution%20by%20CYP2C8%20at%20clinically%20relevant%20concentrations&author=A.M.%20Filppula&author=J.%20Laitila&author=P.J.%20Neuvonen&author=J.T.%20Backman&volume=5&publication_year=2011&pages=904-911&pmid=21289076&doi=10.1124/dmd.110.037689&)

5. Chiba, M. , Xu, X. , Nishime, J.A. , Balani, S.K. & Lin, J.H. Hepatic microsomal metabolism of montelukast, a potent leukotriene D4 receptor antagonist, in humans. Drug Metab. Dispos. 25, 1022–1031 (1997).  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9311616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Hepatic%20microsomal%20metabolism%20of%20montelukast,%20a%20potent%20leukotriene%20D4%20receptor%20antagonist,%20in%20humans&author=M.%20Chiba&author=X.%20Xu&author=J.A.%20Nishime&author=S.K.%20Balani&author=J.H.%20Lin&volume=25&publication_year=1997&pages=1022-1031&pmid=9311616&)

6. Karonen, T. , Neuvonen, P.J. & Backman, J.T. CYP2C8 but not CYP3A4 is important in the pharmacokinetics of montelukast. Br. J. Clin. Pharmacol. 73, 257–267 (2012).  [DOI](https://doi.org/10.1111/j.1365-2125.2011.04086.x) | [PMC free article](/articles/PMC3269585/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21838784/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=CYP2C8%20but%20not%20CYP3A4%20is%20important%20in%20the%20pharmacokinetics%20of%20montelukast&author=T.%20Karonen&author=P.J.%20Neuvonen&author=J.T.%20Backman&volume=73&publication_year=2012&pages=257-267&pmid=21838784&doi=10.1111/j.1365-2125.2011.04086.x&)

7. de Oliveira Cardoso, J. , Oliveira, R.V. , Lu, J.B. & Desta, Z. In vitro metabolism of montelukast by cytochrome P450s and UDP‐glucuronosyltransferases. Drug Metab. Dispos. 43, 1905–1916 (2015).  [DOI](https://doi.org/10.1124/dmd.115.065763) | [PMC free article](/articles/PMC4658491/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26374173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=In%20vitro%20metabolism%20of%20montelukast%20by%20cytochrome%20P450s%20and%20UDP%E2%80%90glucuronosyltransferases&author=J.%20de%20Oliveira%20Cardoso&author=R.V.%20Oliveira&author=J.B.%20Lu&author=Z.%20Desta&volume=43&publication_year=2015&pages=1905-1916&pmid=26374173&doi=10.1124/dmd.115.065763&)

8. Mougey, E.B. , Feng, H. , Castro, M. , Irvin, C.G. & Lima, J.J. Absorption of montelukast is transporter mediated: a common variant of OATP2B1 is associated with reduced plasma concentrations and poor response. Pharmacogenet. Genomics 19, 129–138 (2009).  [DOI](https://doi.org/10.1097/FPC.0b013e32831bd98c) | [PMC free article](/articles/PMC2641037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19151602/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Absorption%20of%20montelukast%20is%20transporter%20mediated:%20a%20common%20variant%20of%20OATP2B1%20is%20associated%20with%20reduced%20plasma%20concentrations%20and%20poor%20response&author=E.B.%20Mougey&author=H.%20Feng&author=M.%20Castro&author=C.G.%20Irvin&author=J.J.%20Lima&volume=19&publication_year=2009&pages=129-138&pmid=19151602&doi=10.1097/FPC.0b013e32831bd98c&)

9. Chu, X. , Philip, G. & Evers, R. Comments on Mougey et al. (2009): Absorption of montelukast is transporter mediated: a common variant of OATP2B1 is associated with reduced plasma concentrations and poor response. Pharmacogenet. Genomics 19: 129–138. Pharmacogenet. Genomics. 22, 319–322 (2012).  [DOI](https://doi.org/10.1097/FPC.0b013e32834e9e35) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22173088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics.&title=Comments%20on%20Mougey%20et%20al.%20(2009):%20Absorption%20of%20montelukast%20is%20transporter%20mediated:%20a%20common%20variant%20of%20OATP2B1%20is%20associated%20with%20reduced%20plasma%20concentrations%20and%20poor%20response.%20Pharmacogenet.%20Genomics%2019:%20129%E2%80%93138&author=X.%20Chu&author=G.%20Philip&author=R.%20Evers&volume=22&publication_year=2012&pages=319-322&pmid=22173088&doi=10.1097/FPC.0b013e32834e9e35&)

10. Varma, M.V. et al. Transporter‐mediated hepatic uptake plays an important role in the pharmacokinetics and drug‐drug interactions of montelukast. Clin. Pharmacol. Ther. 101, 406–415 (2017).  [DOI](https://doi.org/10.1002/cpt.520) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27648490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Transporter%E2%80%90mediated%20hepatic%20uptake%20plays%20an%20important%20role%20in%20the%20pharmacokinetics%20and%20drug%E2%80%90drug%20interactions%20of%20montelukast&author=M.V.%20Varma&volume=101&publication_year=2017&pages=406-415&pmid=27648490&doi=10.1002/cpt.520&)

11. Niemi, M. , Pasanen, M.K. & Neuvonen, P.J. Organic anion transporting polypeptide 1B1: a genetically polymorphic transporter of major importance for hepatic drug uptake. Pharmacol. Rev. 63, 157–181 (2011).  [DOI](https://doi.org/10.1124/pr.110.002857) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21245207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Rev.&title=Organic%20anion%20transporting%20polypeptide%201B1:%20a%20genetically%20polymorphic%20transporter%20of%20major%20importance%20for%20hepatic%20drug%20uptake&author=M.%20Niemi&author=M.K.%20Pasanen&author=P.J.%20Neuvonen&volume=63&publication_year=2011&pages=157-181&pmid=21245207&doi=10.1124/pr.110.002857&)

12. Kuehl, G.E. , Lampe, J.W. , Potter, J.D. & Bigler, J. Glucuronidation of nonsteroidal anti‐inflammatory drugs: identifying the enzymes responsible in human liver microsomes. Drug Metab. Dispos. 33, 1027–1035 (2005).  [DOI](https://doi.org/10.1124/dmd.104.002527) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15843492/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Glucuronidation%20of%20nonsteroidal%20anti%E2%80%90inflammatory%20drugs:%20identifying%20the%20enzymes%20responsible%20in%20human%20liver%20microsomes&author=G.E.%20Kuehl&author=J.W.%20Lampe&author=J.D.%20Potter&author=J.%20Bigler&volume=33&publication_year=2005&pages=1027-1035&pmid=15843492&doi=10.1124/dmd.104.002527&)

13. Guillemette, C. , Lévesque, É. & Rouleau, M. Pharmacogenomics of human uridine diphospho‐glucuronosyltransferases and clinical implications. Clin. Pharmacol. Ther. 96, 324–339 (2014).  [DOI](https://doi.org/10.1038/clpt.2014.126) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24922307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther&title=Pharmacogenomics%20of%20human%20uridine%20diphospho%E2%80%90glucuronosyltransferases%20and%20clinical%20implications&author=C.%20Guillemette&author=%C3%89.%20L%C3%A9vesque&author=M.%20Rouleau&volume=96&publication_year=2014&pages=324-339&pmid=24922307&doi=10.1038/clpt.2014.126&)

14. Riedmaier, S. et al UDP‐glucuronosyltransferase (UGT) polymorphisms affect atorvastatin lactonization in vitro and in vivo. Clin. Pharmacol. Ther. 87, 65–73 (2010).  [DOI](https://doi.org/10.1038/clpt.2009.181) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19794410/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=UDP%E2%80%90glucuronosyltransferase%20(UGT)%20polymorphisms%20affect%20atorvastatin%20lactonization%20in%20vitro%20and%20in%20vivo&author=S.%20Riedmaier&volume=87&publication_year=2010&pages=65-73&pmid=19794410&doi=10.1038/clpt.2009.181&)

15. Ieiri, I. et al. Pharmacokinetic and pharmacogenomic profiles of telmisartan after the oral microdose and therapeutic dose. Pharmacogenet. Genomics 21, 495–505 (2011).  [DOI](https://doi.org/10.1097/FPC.0b013e3283489ce2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21691256/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Pharmacokinetic%20and%20pharmacogenomic%20profiles%20of%20telmisartan%20after%20the%20oral%20microdose%20and%20therapeutic%20dose&author=I.%20Ieiri&volume=21&publication_year=2011&pages=495-505&pmid=21691256&doi=10.1097/FPC.0b013e3283489ce2&)

16. Lin, M. et al. Effects of UDP‐glucuronosyltransferase (UGT) polymorphisms on the pharmacokinetics of febuxostat in healthy Chinese volunteers. Drug Metab. Pharmacokinet. 32, 77–84 (2017).  [DOI](https://doi.org/10.1016/j.dmpk.2016.08.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28131654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Pharmacokinet.&title=Effects%20of%20UDP%E2%80%90glucuronosyltransferase%20(UGT)%20polymorphisms%20on%20the%20pharmacokinetics%20of%20febuxostat%20in%20healthy%20Chinese%20volunteers&author=M.%20Lin&volume=32&publication_year=2017&pages=77-84&pmid=28131654&doi=10.1016/j.dmpk.2016.08.003&)

17. Erichsen, T.J. , Aehlen, A. , Ehmer, U. , Kalthoff, S. , Manns, M.P. & Strassburg, C.P. Regulation of the human bile acid UDP‐glucuronosyltransferase 1A3 by the farnesoid X receptor and bile acids. J. Hepatol. 52, 570–578 (2010).  [DOI](https://doi.org/10.1016/j.jhep.2010.01.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20189675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hepatol.&title=Regulation%20of%20the%20human%20bile%20acid%20UDP%E2%80%90glucuronosyltransferase%201A3%20by%20the%20farnesoid%20X%20receptor%20and%20bile%20acids&author=T.J.%20Erichsen&author=A.%20Aehlen&author=U.%20Ehmer&author=S.%20Kalthoff&author=M.P.%20Manns&volume=52&publication_year=2010&pages=570-578&pmid=20189675&doi=10.1016/j.jhep.2010.01.010&)

18. Hu, D.G. , Meech, R. , McKinnon, R.A. & Mackenzie, P.I. Transcriptional regulation of human UDP‐glucuronosyltransferase genes. Drug Metab. Rev. 46, 421–458 (2014).  [DOI](https://doi.org/10.3109/03602532.2014.973037) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25336387/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Rev.&title=Transcriptional%20regulation%20of%20human%20UDP%E2%80%90glucuronosyltransferase%20genes&author=D.G.%20Hu&author=R.%20Meech&author=R.A.%20McKinnon&author=P.I.%20Mackenzie&volume=46&publication_year=2014&pages=421-458&pmid=25336387&doi=10.3109/03602532.2014.973037&)

19. Iwai, M. , Maruo, Y. , Ito, M. , Yamamoto, K. , Sato, H. & Takeuchi, Y. Six novel UDP‐glucuronosyltransferase (UGT1A3) polymorphisms with varying activity. J. Hum. Genet. 49, 123–128 (2004).  [DOI](https://doi.org/10.1007/s10038-003-0119-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14986168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hum.%20Genet.&title=Six%20novel%20UDP%E2%80%90glucuronosyltransferase%20(UGT1A3)%20polymorphisms%20with%20varying%20activity&author=M.%20Iwai&author=Y.%20Maruo&author=M.%20Ito&author=K.%20Yamamoto&author=H.%20Sato&volume=49&publication_year=2004&pages=123-128&pmid=14986168&doi=10.1007/s10038-003-0119-y&)

20. Chen, Y. , Chen, S. , Li, X. , Wang, X. & Zeng, S. Genetic variants of human UGT1A3: functional characterization and frequency distribution in a Chinese Han population. Drug Metab. Dispos. 34, 1462–1467 (2006).  [DOI](https://doi.org/10.1124/dmd.106.009761) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16738032/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Genetic%20variants%20of%20human%20UGT1A3:%20functional%20characterization%20and%20frequency%20distribution%20in%20a%20Chinese%20Han%20population&author=Y.%20Chen&author=S.%20Chen&author=X.%20Li&author=X.%20Wang&author=S.%20Zeng&volume=34&publication_year=2006&pages=1462-1467&pmid=16738032&doi=10.1124/dmd.106.009761&)

21. Caillier, B. et al. A pharmacogenomics study of the human estrogen glucuronosyltransferase UGT1A3. Pharmacogenet. Genomics 17, 481–495 (2007).  [DOI](https://doi.org/10.1097/FPC.0b013e32806d87a4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17558304/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=A%20pharmacogenomics%20study%20of%20the%20human%20estrogen%20glucuronosyltransferase%20UGT1A3&author=B.%20Caillier&volume=17&publication_year=2007&pages=481-495&pmid=17558304&doi=10.1097/FPC.0b013e32806d87a4&)

22. Niemi, M. et al. High plasma pravastatin concentrations are associated with single nucleotide polymorphisms and haplotypes of organic anion transporting polypeptide‐C (OATP‐C, SLCO1B1). Pharmacogenetics 7, 429–440 (2004).  [DOI](https://doi.org/10.1097/01.fpc.0000114750.08559.32) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15226675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=High%20plasma%20pravastatin%20concentrations%20are%20associated%20with%20single%20nucleotide%20polymorphisms%20and%20haplotypes%20of%20organic%20anion%20transporting%20polypeptide%E2%80%90C%20(OATP%E2%80%90C,%20SLCO1B1)&author=M.%20Niemi&volume=7&publication_year=2004&pages=429-440&pmid=15226675&doi=10.1097/01.fpc.0000114750.08559.32&)

23. Pasanen, M.K. , Neuvonen, M. , Neuvonen, P.J. & Niemi, M. SLCO1B1 polymorphism markedly affects the pharmacokinetics of simvastatin acid. Pharmacogenet. Genomics 16, 873–879 (2006).  [DOI](https://doi.org/10.1097/01.fpc.0000230416.82349.90) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17108811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=SLCO1B1%20polymorphism%20markedly%20affects%20the%20pharmacokinetics%20of%20simvastatin%20acid&author=M.K.%20Pasanen&author=M.%20Neuvonen&author=P.J.%20Neuvonen&author=M.%20Niemi&volume=16&publication_year=2006&pages=873-879&pmid=17108811&doi=10.1097/01.fpc.0000230416.82349.90&)

24. Pasanen, M.K. , Fredrikson, H. , Neuvonen, P.J. & Niemi, M. Different effects of SLCO1B1 polymorphism on the pharmacokinetics of atorvastatin and rosuvastatin. Clin. Pharmacol. Ther. 6, 726–733 (2007).  [DOI](https://doi.org/10.1038/sj.clpt.6100220) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17473846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Different%20effects%20of%20SLCO1B1%20polymorphism%20on%20the%20pharmacokinetics%20of%20atorvastatin%20and%20rosuvastatin&author=M.K.%20Pasanen&author=H.%20Fredrikson&author=P.J.%20Neuvonen&author=M.%20Niemi&volume=6&publication_year=2007&pages=726-733&pmid=17473846&doi=10.1038/sj.clpt.6100220&)

25. Nies, A.T. et al. Genetics is a major determinant of expression of the human hepatic uptake transporter OATP1B1, but not of OATP1B3 and OATP2B1. Genome Med. 5, 1–11 (2013).  [DOI](https://doi.org/10.1186/gm405) | [PMC free article](/articles/PMC3706890/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23311897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med.&title=Genetics%20is%20a%20major%20determinant%20of%20expression%20of%20the%20human%20hepatic%20uptake%20transporter%20OATP1B1,%20but%20not%20of%20OATP1B3%20and%20OATP2B1&author=A.T.%20Nies&volume=5&publication_year=2013&pages=1-11&pmid=23311897&doi=10.1186/gm405&)

26. Bryan, J. et al. ABCC8 and ABCC9: ABC transporters that regulate K+ channels. Pflugers Arch. 5, 703–718 (2007).  [DOI](https://doi.org/10.1007/s00424-006-0116-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16897043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pflugers%20Arch.&title=ABCC8%20and%20ABCC9:%20ABC%20transporters%20that%20regulate%20K+%20channels&author=J.%20Bryan&volume=5&publication_year=2007&pages=703-718&pmid=16897043&doi=10.1007/s00424-006-0116-z&)

27. Kamide, K. et al. Genome‐wide response to antihypertensive medication using home blood pressure measurements: a pilot study nested within the HOMED‐BP study. Pharmacogenomics 14, 1709–1721 (2013).  [DOI](https://doi.org/10.2217/pgs.13.161) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24192120/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genome%E2%80%90wide%20response%20to%20antihypertensive%20medication%20using%20home%20blood%20pressure%20measurements:%20a%20pilot%20study%20nested%20within%20the%20HOMED%E2%80%90BP%20study&author=K.%20Kamide&volume=14&publication_year=2013&pages=1709-1721&pmid=24192120&doi=10.2217/pgs.13.161&)

28. Grady, B.J. et al. Use of biological knowledge to inform the analysis of gene‐gene interactions involved in modulating virologic failure with efavirenz‐containing treatment regimens in ART‐naïve ACTG clinical trials participants. Pac. Symp. Biocomput. 253–264 (2011).  [DOI](https://doi.org/10.1142/9789814335058_0027) | [PMC free article](/articles/PMC3094912/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21121053/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pac.%20Symp.%20Biocomput.&title=Use%20of%20biological%20knowledge%20to%20inform%20the%20analysis%20of%20gene%E2%80%90gene%20interactions%20involved%20in%20modulating%20virologic%20failure%20with%20efavirenz%E2%80%90containing%20treatment%20regimens%20in%20ART%E2%80%90na%C3%AFve%20ACTG%20clinical%20trials%20participants&author=B.J.%20Grady&publication_year=2011&pages=253-264&pmid=21121053&doi=10.1142/9789814335058_0027&)

29. Niemi, M. , Leathart, J.B. , Neuvonen, M. , Backman, J.T. , Daly, A.K. & Neuvonen, P.J. Polymorphism in CYP2C8 is associated with reduced plasma concentrations of repaglinide. Clin. Pharmacol. Ther. 74, 380–387 (2003).  [DOI](https://doi.org/10.1016/S0009-9236(03)00228-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14534525/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Polymorphism%20in%20CYP2C8%20is%20associated%20with%20reduced%20plasma%20concentrations%20of%20repaglinide&author=M.%20Niemi&author=J.B.%20Leathart&author=M.%20Neuvonen&author=J.T.%20Backman&author=A.K.%20Daly&volume=74&publication_year=2003&pages=380-387&pmid=14534525&doi=10.1016/S0009-9236(03)00228-5&)

30. Kirchheiner, J. et al. Pharmacokinetics and pharmacodynamics of rosiglitazone in relation to CYP2C8 genotype. Clin. Pharmacol. Ther. 80, 657–667 (2006).  [DOI](https://doi.org/10.1016/j.clpt.2006.09.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17178266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Pharmacokinetics%20and%20pharmacodynamics%20of%20rosiglitazone%20in%20relation%20to%20CYP2C8%20genotype&author=J.%20Kirchheiner&volume=80&publication_year=2006&pages=657-667&pmid=17178266&doi=10.1016/j.clpt.2006.09.008&)

31. Tornio, A. , Niemi, M. , Neuvonen, P.J. & Backman, J.T. Trimethoprim and the CYP2C8*3 allele have opposite effects on the pharmacokinetics of pioglitazone. Drug Metab. Dispos. 36, 73–80 (2008).  [DOI](https://doi.org/10.1124/dmd.107.018010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17913794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Trimethoprim%20and%20the%20CYP2C8*3%20allele%20have%20opposite%20effects%20on%20the%20pharmacokinetics%20of%20pioglitazone&author=A.%20Tornio&author=M.%20Niemi&author=P.J.%20Neuvonen&author=J.T.%20Backman&volume=36&publication_year=2008&pages=73-80&pmid=17913794&doi=10.1124/dmd.107.018010&)

32. Aquilante, C.L. , Niemi, M. , Gong, L. , Altman, R.B. & Klein, T.E. PharmGKB summary: very important pharmacogene information for cytochrome P450, family 2, subfamily C, polypeptide 8. Pharmacogenet. Genomics 12, 721–728 (2013).  [DOI](https://doi.org/10.1097/FPC.0b013e3283653b27) | [PMC free article](/articles/PMC4038626/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23962911/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=PharmGKB%20summary:%20very%20important%20pharmacogene%20information%20for%20cytochrome%20P450,%20family%202,%20subfamily%20C,%20polypeptide%208&author=C.L.%20Aquilante&author=M.%20Niemi&author=L.%20Gong&author=R.B.%20Altman&author=T.E.%20Klein&volume=12&publication_year=2013&pages=721-728&pmid=23962911&doi=10.1097/FPC.0b013e3283653b27&)

33. Tornio, A. et al. Glucuronidation converts clopidogrel to a strong time‐dependent inhibitor of CYP2C8: a phase II metabolite as a perpetrator of drug‐drug interactions. Clin. Pharmacol. Ther. 96, 498–507 (2014).  [DOI](https://doi.org/10.1038/clpt.2014.141) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24971633/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Glucuronidation%20converts%20clopidogrel%20to%20a%20strong%20time%E2%80%90dependent%20inhibitor%20of%20CYP2C8:%20a%20phase%20II%20metabolite%20as%20a%20perpetrator%20of%20drug%E2%80%90drug%20interactions&author=A.%20Tornio&volume=96&publication_year=2014&pages=498-507&pmid=24971633&doi=10.1038/clpt.2014.141&)

34. Barratt, D.T. et al. CYP2C8 genotype significantly alters imatinib metabolism in chronic myeloid leukaemia patients. Clin. Pharmacokinet. 56, 977–985 (2017).  [DOI](https://doi.org/10.1007/s40262-016-0494-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27995529/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet.&title=CYP2C8%20genotype%20significantly%20alters%20imatinib%20metabolism%20in%20chronic%20myeloid%20leukaemia%20patients&author=D.T.%20Barratt&volume=56&publication_year=2017&pages=977-985&pmid=27995529&doi=10.1007/s40262-016-0494-0&)

35. Backman, J.T. , Filppula, A.M. , Niemi, M. & Neuvonen, P.J. Role of cytochrome P450 2C8 in drug metabolism and interactions. Pharmacol. Rev. 68:168–241 (2016).  [DOI](https://doi.org/10.1124/pr.115.011411) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26721703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Rev.&title=Role%20of%20cytochrome%20P450%202C8%20in%20drug%20metabolism%20and%20interactions&author=J.T.%20Backman&author=A.M.%20Filppula&author=M.%20Niemi&author=P.J.%20Neuvonen&volume=68&publication_year=2016&pages=168-241&pmid=26721703&doi=10.1124/pr.115.011411&)

36. Hon, K.L. , Leung, T.F. & Leung, A.K. Clinical effectiveness and safety of montelukast in asthma. What are the conclusions from clinical trials and meta‐analyses? Drug Des. Dev. Ther. 26, 839–850 (2014).  [DOI](https://doi.org/10.2147/DDDT.S39100) | [PMC free article](/articles/PMC4079631/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25061277/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Des.%20Dev.%20Ther.&title=Clinical%20effectiveness%20and%20safety%20of%20montelukast%20in%20asthma.%20What%20are%20the%20conclusions%20from%20clinical%20trials%20and%20meta%E2%80%90analyses?&author=K.L.%20Hon&author=T.F.%20Leung&author=A.K.%20Leung&volume=26&publication_year=2014&pages=839-850&pmid=25061277&doi=10.2147/DDDT.S39100&)

37. Thompson, M.D. et al. Cysteinyl leukotrienes pathway genes, atopic asthma and drug response: from population isolates to large genome‐wide association studies. Front. Pharmacol. 7:299 eCollection (2016).  [DOI](https://doi.org/10.3389/fphar.2016.00299) | [PMC free article](/articles/PMC5131607/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27990118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Cysteinyl%20leukotrienes%20pathway%20genes,%20atopic%20asthma%20and%20drug%20response:%20from%20population%20isolates%20to%20large%20genome%E2%80%90wide%20association%20studies&author=M.D.%20Thompson&volume=7&publication_year=2016&pages=299&pmid=27990118&doi=10.3389/fphar.2016.00299&)

38. Zhao, J.J. et al. Pharmacokinetics and bioavailability of montelukast sodium (MK‐0476) in healthy young and elderly volunteers. Biopharm. Drug Dispos. 18, 769–777 (1997).  [DOI](https://doi.org/10.1002/(sici)1099-081x(199712)18:9<769::aid-bdd60>3.0.co;2-k) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9429741/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biopharm.%20Drug%20Dispos.&title=Pharmacokinetics%20and%20bioavailability%20of%20montelukast%20sodium%20(MK%E2%80%900476)%20in%20healthy%20young%20and%20elderly%20volunteers&author=J.J.%20Zhao&volume=18&publication_year=1997&pages=769-777&pmid=9429741&doi=10.1002/(sici)1099-081x(199712)18:9<769::aid-bdd60>3.0.co;2-k&)

39. Noonan, M.J. et al. Montelukast, a potent leukotriene receptor antagonist, causes dose‐related improvements in chronic asthma. Montelukast Asthma Study Group. Eur. Respir. J. 11, 1232–1239 (1998).  [DOI](https://doi.org/10.1183/09031936.98.11061232) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9657560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Respir.%20J.&title=Montelukast,%20a%20potent%20leukotriene%20receptor%20antagonist,%20causes%20dose%E2%80%90related%20improvements%20in%20chronic%20asthma.%20Montelukast%20Asthma%20Study%20Group&author=M.J.%20Noonan&volume=11&publication_year=1998&pages=1232-1239&pmid=9657560&doi=10.1183/09031936.98.11061232&)

40. Männistö, V.T. et al. Lipoprotein subclass metabolism in nonalcoholic steatohepatitis. J. Lipid Res. 55, 2676–2684 (2014).  [DOI](https://doi.org/10.1194/jlr.P054387) | [PMC free article](/articles/PMC4242459/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25344588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Lipid%20Res.&title=Lipoprotein%20subclass%20metabolism%20in%20nonalcoholic%20steatohepatitis&author=V.T.%20M%C3%A4nnist%C3%B6&volume=55&publication_year=2014&pages=2676-2684&pmid=25344588&doi=10.1194/jlr.P054387&)

41. Nilsson, E. et al. Epigenetic alterations in human liver from subjects with type 2 diabetes in parallel with reduced folate levels. J. Clin. Endocrinol. Metab. 100, E1491–E1501 (2015).  [DOI](https://doi.org/10.1210/jc.2015-3204) | [PMC free article](/articles/PMC4702449/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26418287/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Endocrinol.%20Metab&title=Epigenetic%20alterations%20in%20human%20liver%20from%20subjects%20with%20type%202%20diabetes%20in%20parallel%20with%20reduced%20folate%20levels&author=E.%20Nilsson&volume=100&publication_year=2015&pages=E1491-E1501&pmid=26418287&doi=10.1210/jc.2015-3204&)

42. Pihlajamäki, J. et al. Serum interleukin 1 receptor antagonist as an independent marker of non‐alcoholic steatohepatitis in humans. J. Hepatol. 56, 663–670 (2012).  [DOI](https://doi.org/10.1016/j.jhep.2011.10.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22027586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hepatol.&title=Serum%20interleukin%201%20receptor%20antagonist%20as%20an%20independent%20marker%20of%20non%E2%80%90alcoholic%20steatohepatitis%20in%20humans&author=J.%20Pihlajam%C3%A4ki&volume=56&publication_year=2012&pages=663-670&pmid=22027586&doi=10.1016/j.jhep.2011.10.005&)

43. Sulonen, A.M. et al Comparison of solution‐based exome capture methods for next generation sequencing. Genome Biol. 12, R94 (2011).  [DOI](https://doi.org/10.1186/gb-2011-12-9-r94) | [PMC free article](/articles/PMC3308057/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21955854/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&title=Comparison%20of%20solution%E2%80%90based%20exome%20capture%20methods%20for%20next%20generation%20sequencing&author=A.M.%20Sulonen&volume=12&publication_year=2011&pages=R94&pmid=21955854&doi=10.1186/gb-2011-12-9-r94&)

44. Clinical Pharmacology & Therapeutics Editorial Team . Statistical guide for Clinical Pharmacology & Therapeutics. Clin. Pharmacol. Ther. 88, 150–152 (2010).  [DOI](https://doi.org/10.1038/clpt.2010.113) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20592719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Statistical%20guide%20for%20Clinical%20Pharmacology%20&%20Therapeutics&volume=88&publication_year=2010&pages=150-152&pmid=20592719&doi=10.1038/clpt.2010.113&)

45. Hallynck, T.H. , Soep, H.H. , Thomis, J.A. , Boelaert, J. , Daneels, R. & Dettli, L. Should clearance be normalised to body surface or to lean body mass? Br. J. Clin. Pharmacol. 11, 523–526 (1981).  [DOI](https://doi.org/10.1111/j.1365-2125.1981.tb01163.x) | [PMC free article](/articles/PMC1401592/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7272167/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Should%20clearance%20be%20normalised%20to%20body%20surface%20or%20to%20lean%20body%20mass?&author=T.H.%20Hallynck&author=H.H.%20Soep&author=J.A.%20Thomis&author=J.%20Boelaert&author=R.%20Daneels&volume=11&publication_year=1981&pages=523-526&pmid=7272167&doi=10.1111/j.1365-2125.1981.tb01163.x&)

46. Haycock, G.B. , Schwartz, G.J. & Wisotsky, D.H. Geometric method for measuring body surface area: a height‐weight formula validated in infants, children, and adults. J. Pediatr. 93, 62–66 (1978).  [DOI](https://doi.org/10.1016/s0022-3476(78)80601-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/650346/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pediatr.&title=Geometric%20method%20for%20measuring%20body%20surface%20area:%20a%20height%E2%80%90weight%20formula%20validated%20in%20infants,%20children,%20and%20adults&author=G.B.%20Haycock&author=G.J.%20Schwartz&author=D.H.%20Wisotsky&volume=93&publication_year=1978&pages=62-66&pmid=650346&doi=10.1016/s0022-3476(78)80601-5&)

47. Gabriel, S.B. et al. The structure of haplotype blocks in the human genome. Science 296, 2225–2229 (2002).  [DOI](https://doi.org/10.1126/science.1069424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12029063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome&author=S.B.%20Gabriel&volume=296&publication_year=2002&pages=2225-2229&pmid=12029063&doi=10.1126/science.1069424&)

48. Wall, J.D. & Pritchard, J.K. Assessing the performance of the haplotype block model of linkage disequilibrium. Am. J. Hum. Genet. 73, 502–515 (2003).  [DOI](https://doi.org/10.1086/378099) | [PMC free article](/articles/PMC1180676/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12916017/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Assessing%20the%20performance%20of%20the%20haplotype%20block%20model%20of%20linkage%20disequilibrium&author=J.D.%20Wall&author=J.K.%20Pritchard&volume=73&publication_year=2003&pages=502-515&pmid=12916017&doi=10.1086/378099&)

49. Stephens, M. , Smith, N.J. & Donnelly, P. A new statistical method for haplotype reconstruction from population data. Am. J. Hum. Genet. 68, 978–989 (2001).  [DOI](https://doi.org/10.1086/319501) | [PMC free article](/articles/PMC1275651/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11254454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=A%20new%20statistical%20method%20for%20haplotype%20reconstruction%20from%20population%20data&author=M.%20Stephens&author=N.J.%20Smith&author=P.%20Donnelly&volume=68&publication_year=2001&pages=978-989&pmid=11254454&doi=10.1086/319501&)

50. Stephens, M. & Donnelly, P. A comparison of bayesian methods for haplotype reconstruction from population genotype data. Am. J. Hum. Genet. 73, 1162–1169 (2003).  [DOI](https://doi.org/10.1086/379378) | [PMC free article](/articles/PMC1180495/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14574645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=A%20comparison%20of%20bayesian%20methods%20for%20haplotype%20reconstruction%20from%20population%20genotype%20data&author=M.%20Stephens&author=P.%20Donnelly&volume=73&publication_year=2003&pages=1162-1169&pmid=14574645&doi=10.1086/379378&)
