# Gene‐Wide Tagging Study of the Association Between KCNT1 Polymorphisms and the Susceptibility and Efficacy of Genetic Generalized Epilepsy in Chinese Population

## Metadata
**Authors:** Jian Qu, Ying Zhang, Zhi‐Quan Yang, Xiao‐Yuan Mao, Bo‐Ting Zhou, Ji‐Ye Yin, Hui He, Xiang‐Ping Li, Hong‐Yu Long, Nan Lv, Xiao‐Jing Xu, Bo Xiao, Yu Zhang, Qiang Tang, Dong‐Li Hu, Hong‐Hao Zhou, Zhao‐Qian Liu
**Journal:** CNS Neuroscience & Therapeutics
**Date:** 2013 Nov 27
**DOI:** [10.1111/cns.12169](https://doi.org/10.1111/cns.12169)
**PMID:** 24279416
**PMCID:** PMC6493076
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6493076/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6493076/pdf/CNS-20-140.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6493076/pdf/CNS-20-140.pdf)

## Abstract

**Aims:** 
The causes of genetic generalized epilepsies (GGEs) are still uncertain now. Some studies found that the human potassium channel, subfamily T, member 1 (KCNT1) is the candidate gene causing malignant migrating partial seizures of infancy and autosomal dominant nocturnal frontal lobe epilepsy which are all rare genetic generalized epilepsies. The aims of this study were going to evaluate the association between KCNT1 common variations and the susceptibility and drug resistance of genetic generalized epilepsies in Chinese population.

**Methods:** 
The allele‐specific MALDI–TOF mass spectrometry method was used to assess 17 tagSNPs (tagged single‐nucleotide polymorphisms) of KCNT1 in 284 healthy Chinese controls and 483 Chinese GGEs patients including 279 anti‐epileptic drug‐responsive patients and 204 drug‐resistant patients.

**Results:** 
Genotype distributions of all the selected tagSNPs were consistent with Hardy–Weinberg equilibrium in GGEs and healthy controls. None of the all 17 tagSNPs alleles were found to be related with the susceptibility and drug resistance of genetic generalized epilepsies. The frequencies of haplotype 5 and haplotype 1 were significantly lower in GGEs than that in healthy controls (2% vs. 4%, OR = 0.47 [0.27–0.94], P = 0.03) and obviously higher in drug‐resistant patients than that in drug‐response patients (6% vs. 3%, OR = 2.56 [1.23–5.35], P = 0.01). However, after the correction of multiple comparisons with Bonferroni's method, we found that the above two haplotypes were not associated with the susceptibility and drug resistance in GGEs and healthy controls.

**Conclusion:** 
This gene‐wide tagging study revealed no association between KCNT1 17 common variations and susceptibility of GGEs or AEDs (anti‐epileptic drugs) efficacy of genetic generalized epilepsies in Chinese population.

Keywords: Drug resistance, Efficacy, Genetic generalized epilepsies, Genetic polymorphism, KCNT1, Susceptibility

### Aims

The causes of genetic generalized epilepsies (GGEs) are still uncertain now. Some studies found that the human potassium channel, subfamily T, member 1 (*KCNT1*) is the candidate gene causing malignant migrating partial seizures of infancy and autosomal dominant nocturnal frontal lobe epilepsy which are all rare genetic generalized epilepsies. The aims of this study were going to evaluate the association between *KCNT1* common variations and the susceptibility and drug resistance of genetic generalized epilepsies in Chinese population.

### Methods

The allele‐specific MALDI–TOF mass spectrometry method was used to assess 17 tagSNPs (tagged single‐nucleotide polymorphisms) of *KCNT1* in 284 healthy Chinese controls and 483 Chinese GGEs patients including 279 anti‐epileptic drug‐responsive patients and 204 drug‐resistant patients.

### Results

Genotype distributions of all the selected tagSNPs were consistent with Hardy–Weinberg equilibrium in GGEs and healthy controls. None of the all 17 tagSNPs alleles were found to be related with the susceptibility and drug resistance of genetic generalized epilepsies. The frequencies of haplotype 5 and haplotype 1 were significantly lower in GGEs than that in healthy controls (2% vs. 4%, OR = 0.47 [0.27–0.94], *P *=* *0.03) and obviously higher in drug‐resistant patients than that in drug‐response patients (6% vs. 3%, OR = 2.56 [1.23–5.35], *P *=* *0.01). However, after the correction of multiple comparisons with Bonferroni's method, we found that the above two haplotypes were not associated with the susceptibility and drug resistance in GGEs and healthy controls.

### Conclusion

This gene‐wide tagging study revealed no association between *KCNT1* 17 common variations and susceptibility of GGEs or AEDs (anti‐epileptic drugs) efficacy of genetic generalized epilepsies in Chinese population.

## Introduction

Epilepsy is one of the most common neurological disorders and it presents with a series of clinical features which are recognized to over 50 distinct epilepsy syndromes [1](#cns12169-bib-0001). Seizure disorders always be divided into idiopathic or symptomatic epilepsies. Symptomatic epilepsies have definite cause such as brain‐related tumor or metabolic disorders etc. Genetic generalized epilepsies (GGEs, formerly called the idiopathic generalized epilepsies) are characterized by non‐focal mechanism of onset and no external cause. Genetic generalized epilepsies are thought to be polygenic, including childhood absence epilepsy (CAE), juvenile absence epilepsy (JAE), juvenile myoclonic epilepsy (JME) and epilepsy with generalized tonic‐clonic seizures alone (EGTCS) [2](#cns12169-bib-0002). About 0.3% of whole populations were affected with GGEs, and it accounts for 20% to 30% of all the patients with epilepsy [3](#cns12169-bib-0003). Although scientists pay close attention to the genetic etiology of the GGEs for a long time, it is still only a small fraction of cases can be determined.

Recently, two studies have identified a sodium‐gated potassium channel gene *KCNT1* to be a novel driver of pathogenesis of some epileptic syndromes. Sam Berkovic and his colleagues found a more severe form of autosomal dominant nocturnal frontal lobe epilepsy (ANDFLE) to be associated with mutations in *KCNT1* [4](#cns12169-bib-0004). Another study showed that mutations in *KCNT1* might be linked not only to epileptic activity in these severe epilepsies (malignant migrating partial seizures of infancy (MMPSI)—a rare epileptic encephalopathy for features of drug‐resistant seizures and developmental delay), but also to the brain development protein cascade [5](#cns12169-bib-0005). So, we hypothesize that *KCNT1* variations may affect the occurrence of genetic generalized epilepsies epilepsy.

As a gene encoded sodium‐activated potassium (K_Na_) channel, *KCNT1* may not only related with the susceptibility of GGEs, but also related with the efficacy of anti‐epileptic drugs. Approximately, one third of epileptic patients do not respond to antiepileptic drugs (AEDs) and it has been hypothesized due to polymorphisms in brain targets of AEDs, drug efflux transporters, drug metabolism, and elimination related genes [6](#cns12169-bib-0006). *KCNT1* maps to chromosome 9q34.3 and it is widely expressed in the nervous system, especially in the brain, which is consistent with its role in the pathogenesis of some rare epileptic syndromes. It is stimulated by depolarization and produce a slow current followed by repetitive neuronal firing [7](#cns12169-bib-0007). We hypothesized that *KCNT1* variations may affect the efficacy of anti‐epileptic drugs.

The functional study about *KCNT1*, especially the association with pathogenesis of epileptic syndromes, however, remains unclear. And the previous studies focused on the exon mutations of *KCNT1*, whether the *KCNT1* common variations could affect the susceptibility and drug resistance of genetic generalized epilepsies in Chinese population is unclear. And whether *KCNT1* variations could affect the efficacy of anti‐epileptic drugs is also unknown yet.

In this study, we aimed to find out the association between *KCNT1* common variations and GGEs in Chinese population and further to evaluate whether these loci are related with the susceptibility of GGEs and the resistance of anti‐epileptic drugs (AEDs) in Chinese genetic generalized epilepsies patients.

## Materials and Methods

### Subjects

A total of 483 Chinese GGEs patients treated with AEDs (297 men, 186 women, mean age: 18.3 ± 12.1 years) and 284 healthy control (184 men, 100 women, mean age:18.6 ± 12.2 years) from Xiangya hospital, the Second Xiangya Hospital of Central South University, Hunan Provincial People's Hospital, and the Third Hospital of Huaihua City of Hunan province, were recruited in this study. The patients were diagnosed and classified according to guidelines of the International League Against Epilepsy and standardized protocols ([http://portal.ccg.uni-koeln.de/ccg/research/epilepsy-genetics/sampling-procedure](http://portal.ccg.uni-koeln.de/ccg/research/epilepsy-genetics/sampling-procedure)) [8](#cns12169-bib-0008), [9](#cns12169-bib-0009). Exclusion criteria included severe adverse drug reactions, unreliable record of seizure frequency, poor compliance with AEDs, history of alcohol or drug abuse, presence of progressive or degenerative neurological or systemic disorders, and hepatic or renal failure. A standardized questionnaire was administered to collect demographic details and clinical data such as seizure types and frequency, past medical history, AED history, concomitant drug history, and relevant family history etc. All patients or children parents gave their written consent to participate in the study. The study protocol was approved by the Ethics Committee of Xiangya School of Medicine and Ethics Committee of Institute of Clinical Pharmacology of Central South University. Clinical study admission (the registration number: ChiCTR‐TCH‐0000813) was approved by Chinese Clinical Trail Register. The patients were considered to be drug‐responsive if they had not experienced any type of seizures for a minimum of 1 year after receiving AEDs. Drug resistance was defined as having at least four seizures during the previous year while trying at least three antiepileptic medications at maximal tolerated doses [10](#cns12169-bib-0010), [11](#cns12169-bib-0011).

### Genotyping of KCNT1 polymorphism

Blood samples (3 mL) for genotyping were obtained with EDTA through the venipuncture and frozen at −80°C for 24 h. DNA was isolated using phenol‐chloroform extraction method. All 17 tagSNPs were identified across the gene regions of *KCNT1* (90.96 kilobase pairs [kbp], HapMap Data Rel 24/phase II Nov08, on NCBI B36 asembly dbSNP b126) by Haploview ([http://www.broad.mit.edu/mpg/haploview](http://www.broad.mit.edu/mpg/haploview)). Linkage disequilibrium (LD) data for SNPs with minor allele frequency (MAF) ≧0.1 in Han Chinese in Beijing, China, from the International HapMap Project ([http://www.hapmap.org](http://www.hapmap.org)) were used with tagger to identify SNPs tagging clusters with LD of *r* ^2^ > 0.8. MassArray (Sequenom, SanDiego, CA, USA) was used for genotyping all tagSNPs using allele‐specific MALDI–TOF mass spectrometry. Primers and multiplex reactions were designed using the [RealSNP.com](http://RealSNP.com) Website.

### Statistical analysis

The SPSS software package (Version 13.0 for Windows; SPSS, Chicago, IL, USA) was used for statistical analysis. Hardy–Weinberg equilibrium was analyzed with chi square test or Fisher's Exact test as applicable in the studied samples. Age was compared between responsive, non‐responsive patients or healthy controls with the Student's *t*‐test. Sex and other characteristics about patients are compared with chi square test. Allele and genotype frequencies were compared between cases and controls by chi square test. The association between various genotypes and anti‐epileptic drug efficacy was examined by binary logistic regression, after adjustment for age, sex, and seizure types. Linkage disequilibrium and haplotypes were analyzed using SHEsis ([http://analysis.bio-x.cn/myAnalysis.php](http://analysis.bio-x.cn/myAnalysis.php)) [12](#cns12169-bib-0012), [13](#cns12169-bib-0013). Bonferroni's method was used for correction of multiple comparisons. Statistical significance was accepted when *P *<* *0.05.

## Results

The study population consisted of 284 healthy controls (184 men, 100 women, mean age: 18.6 ± 12.2 years) and 483 Chinese GGEs patients treated with AEDs (297 men, 186 women, mean age: 18.3 ± 12.1 years), including 279 drug‐responsive patients and 204 drug‐resistant patients. In all 483 Chinese GGEs patients, the subtypes of GGEs contained CAE (38.3%), JAE (19.6%), JME (25.6%), and EGTCS (16.5%). No difference was found in sex and age between controls and GGEs. Moreover, there were also no difference in sex, age, and age at onset between drug‐responsive patients and drug‐resistant patients. Patients received a wide variety of drugs, with carbamazepine, valproic acid, and Oxcarbazepine being the most commonly used (Table** ** [1](#cns12169-tbl-0001)). SNP locations of all tagSNPs across the entire *KCNT1* gene region using HapMap data and the Haploview software, and 17 tagSNPs of *KCNT1* were chosen to detect in patients and controls (Figure [1](#cns12169-fig-0001) **,** Table** ** [2](#cns12169-tbl-0002)). The linkage disequilibrium test of 17 tagSNPs in *KCNT1* was shown in Figure [2](#cns12169-fig-0002).

### Table 1.

| Parameters | Drug‐ responsive (279) | Drug‐ resistant (204) | Total (483) | Healthy controls (284) |
| --- | --- | --- | --- | --- |
| Male/Female | 174 (62.4%)/105 (37.6%) | 123 (60.3%)/81 (39.7%) | 297 (61.5%)/186 (38.5%) | 184 (64.8%)/100 (35.2%) |
| Age (years) | 19.3 ± 12.2 | 17.7 ± 11.9 | 18.3 ± 12.1 | 18.6 ± 12.2 |
| Age at onset(years) | 10.6 ± 8.9 | 9.6 ± 10.2 | 10.3 ± 10.1 | – |
| Seizure type (%) |  |  |  |  |
| CAE | 120 (40.4) | 65 (31.9) | 185 (38.3) |  |
| JAE | 55 (19.7) | 39 (19.1) | 94 (19.4) |  |
| JME | 67 (22.6) | 57 (27.9) | 124 (25.7) |  |
| EGTCS | 37 (13.3) | 43 (21.1) | 80 (16.6) |  |
| AEDs used in patients (%) |  |  |  |  |
| Carbamazepine | 173 (62) | 137 (67.2) | 310 (62.3) |  |
| Valproic acid | 182 (65.2) | 168 (82.4) | 350 (72.5) |  |
| Phenytoinum | 23 (8.2) | 18 (8.8) | 41 (8.5) |  |
| Phenobarbital | 12 (4.3) | 8 (3.9) | 20 (4.1) |  |
| lamotrigine | 39 (14.0) | 28 (13.7) | 67 (13.9) |  |
| Levetiracetam | 29 (10.4) | 19 (9.3) | 48 (9.9) |  |
| Topiramate | 44 (15.8) | 23 (11.3) | 67 (13.9) |  |
| Oxcarbazepine | 57 (20.4) | 25 (12.3) | 82 (17.0) |  |

Table 1 Caption: Demographic and clinical characteristics of drug‐responsive and drug‐ resistant patients and healthy controls in GGEs

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b85f/6493076/764be725c4ee/CNS-20-140-g001.jpg)

TagSNPs position in KCNT1 gene. After using Haploview software, we got 17 tagSNPs of KCNT1. Int, intron.

### Table 2.

| Rs Number | SNP | Position | Location |
| --- | --- | --- | --- |
| rs546120 | C/T | 137734769 | Intron1 1503755 |
| rs496503 | A/G | 137736795 | Intron1 1505781 |
| rs487750 | A/G | 137743561 | Intron1 1512547 |
| rs1165016 | A/G | 137745745 | Intron1 1514731 |
| rs498974 | A/T | 137747372 | Intron2 1516358 |
| rs498618 | C/T | 137755787 | Intron2 1524773 |
| rs1537416 | C/T | 137757124 | Intron2 1526110 |
| rs7855716 | C/G | 137757322 | Intron2 1526308 |
| rs755722 | C/T | 137771475 | Intron2 1540461 |
| rs10735239 | C/T | 137784024 | Intron4 1553010 |
| rs11103167 | C/T | 137787985 | Intron7 1556971 |
| rs10118746 | A/G | 137793473 | Intron12 1562459 |
| rs11103182 | G/T | 137803617 | Intron19 1572603 |
| rs1318383 | C/T | 137807894 | Intron21 1576880 |
| rs10776845 | C/A | 137809797 | Intron22 1578783 |
| rs10776850 | T / A | 137817359 | Intron29 1586345 |
| rs10858173 | T / C | 137820722 | Intron31 1589708 |

Table 2 Caption: Position of all tagSNPs and their relation to exon–intron boundaries of KCNT1

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b85f/6493076/006cddc61f14/CNS-20-140-g002.jpg)

The linkage disequilibrium of 17 tagSNPs in KCNT1 in all patients. Linkage disequilibrium between pairs of polymorphisms is shown in diamonds (D′), with darker shading indicating greater D′.

We determined these 17 SNPs in 483 Chinese GGEs patients and 284 healthy controls. The investigated SNPs were all in Hardy–Weinberg equilibrium in 483 Chinese GGEs patients, 284 healthy controls, and also in subtypes groups of Chinese GGEs patients There was no significant difference in allele or genotypes frequencies of *KCNT1* 17 tagSNPs between GGEs and controls (Table** ** [3](#cns12169-tbl-0003)). Moreover, we also analyzed whether there were some associations between subtypes of GGEs and *KCNT1* variations and found no association between them (data not shown). After analyzed haplotypes of 17 tagSNPs in *KCNT1*, we found the frequency of a haplotype (Haplotype 5) was significant lower in GGEs than in healthy controls (2% vs. 4%, OR = 0.47 [0.27–0.94], *P *=* *0.03) (Figure [3](#cns12169-fig-0003)). After correction of multiple comparisons with Bonferroni's method, we did not find any significant association between *KCNT1* haplotypes and the susceptibility of GGEs (*P *<* *0.05/7).

### Table 3.

| SNPs Alleles A/B |   | Genotypes |  |  |  |  |  |  | Alleles |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| GGEs patients (n = 483) |  |  | Controls (n = 284) |  |  | P‐value | GGEs A:B | OR (95% CI) | P‐value |  |  |
| AA | AB | BB | AA | AB | BB | Control A:B |  |  |  |  |  |
| rs546120 | C/T | 226 | 215 | 41 | 125 | 129 | 28 | 0.70 | 667/297; 379/185 | 1.1 (0.88–1.37) | 0.42 |
| rs496503 | A/G | 316 | 117 | 9 | 202 | 74 | 5 | 0.97 | 749/135; 478/84 | 0.97 (0.73–1.31) | 0.87 |
| rs487750 | A/G | 251 | 193 | 38 | 144 | 119 | 21 | 0.87 | 695/269; 407/161 | 1.02 (0.81–1.29) | 0.85 |
| rs1165016 | A/G | 191 | 228 | 63 | 128 | 119 | 35 | 0.29 | 610/354; 375/189 | 0.87 (0.70–1.08) | 0.21 |
| rs498974 | A/T | 131 | 221 | 124 | 72 | 134 | 77 | 0.82 | 483/469; 278/288 | 1.06 (0.87–1.31) | 0.54 |
| rs498618 | C/T | 283 | 177 | 23 | 157 | 110 | 17 | 0.59 | 743/223; 424/144 | 1.13 (0.89–1.44) | 0.31 |
| rs1537416 | C/T | 68 | 216 | 196 | 40 | 124 | 120 | 0.92 | 352/608; 204/364 | 1.03 (0.83–1.28) | 0.77 |
| rs7855716 | C/G | 8 | 101 | 372 | 2 | 75 | 206 | 0.13 | 117/845; 79/487 | 0.85 (0.63–1.16) | 0.31 |
| rs755722 | C/T | 222 | 212 | 48 | 129 | 125 | 30 | 0.96 | 656/308; 383/185 | 1.02 (0.82–1.28) | 0.80 |
| rs10735239 | C/T | 151 | 242 | 88 | 83 | 145 | 54 | 0.85 | 544/418; 311/253 | 1.05 (0.86–1.31) | 0.59 |
| rs11103167 | C/T | 60 | 212 | 210 | 32 | 141 | 111 | 0.32 | 332/632; 205/363 | 0.93 (0.75–1.16) | 0.51 |
| rs10118746 | A/G | 258 | 190 | 32 | 154 | 108 | 21 | 0.88 | 706/254; 416/150 | 1.00 (0.79–1.27) | 0.99 |
| rs11103182 | G/T | 117 | 187 | 175 | 72 | 90 | 118 | 0.15 | 421/537; 234/326 | 1.09 (0.88–1.35) | 0.41 |
| rs1318383 | C/T | 321 | 141 | 18 | 176 | 94 | 13 | 0.41 | 783/177; 446/120 | 1.19 (0.92–1.54) | 0.19 |
| rs10776845 | A/C | 255 | 160 | 27 | 152 | 160 | 18 | 0.79 | 670/214; 410/142 | 0.92 (0.72–1.18) | 0.52 |
| rs10776850 | A/T | 451 | 4 | 1 | 263 | 1 | 0 | 0.55 | 906/6; 524/1 | 3.49 (0.42–29.1) | 0.22 |
| rs10858173 | T/C | 241 | 204 | 34 | 125 | 132 | 26 | 0.22 | 686/272;382/184 | 0.82 (0.66–1.03) | 0.09 |

Table 3 Caption: Allele and genotype frequencies of 17 SNPs of KCNT1 in GGEs patients (n = 483) and case controls (n = 284)

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b85f/6493076/d163d63d205f/CNS-20-140-g003.jpg)

The frequencies of haplotypes (>3%) in all tagSNPs of KCNT1 in GGEs patients (n = 483) and healthy control (n = 284). All those frequencies<3% will be ignored in analysis.

We also analyzed the association between *KCNT1* polymorphisms and anti‐epileptic drug efficacy of GGEs in Chinese population and found no significant difference in allele or genotype frequencies of *KCNT1* 17 tagSNPs between GGEs drug‐responsive patients and drug‐resistant patients (Table** ** [4](#cns12169-tbl-0004)). In haplotypes calculation, we found the frequency of a haplotype (Haplotype 1) was significant higher in drug‐resistant patients than in response patients (6% vs. 3%, OR = 2.56 [1.23–5.35], *P *=* *0.01) (Figure [4](#cns12169-fig-0004)). After correction of multiple comparisons with Bonferroni's method, there were no significant association between *KCNT1* haplotypes and the efficacy of anti‐epileptic drug in GGEs (*P *<* *0.05/8).

### Table 4.

| SNPs Alleles A/B |   | Genotypes |  |  |  |  |  |  | Alleles |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Drug responsive (n = 279) |  |  | Drug resistant (n = 204) |  |  | P‐value | Drug responsive A:B | OR (95% CI) | P‐value |  |  |
| AA | AB | BB | AA | AB | BB | Drug resistant A:B |  |  |  |  |  |
| rs546120 | C/T | 126 | 126 | 26 | 100 | 89 | 14 | 0.52 | 378/178; 289/117 | 1.16 (0.88–1.54) | 0.29 |
| rs496503 | A/G | 182 | 59 | 6 | 133 | 58 | 3 | 0.32 | 423/71;324/64 | 0.85 (0.56–1.23) | 0.38 |
| rs487750 | A/G | 144 | 110 | 24 | 107 | 83 | 13 | 0.66 | 398/158;297/109 | 1.08 (0.81–1.44) | 0.59 |
| rs1165016 | A/G | 114 | 131 | 33 | 76 | 97 | 30 | 0.56 | 359/197;249/157 | 0.87 (0.67–1.13) | 0.31 |
| rs498974 | A/T | 76 | 128 | 72 | 55 | 93 | 51 | 0.99 | 280/272;203/195 | 1.01 (0.78–1.31) | 0.93 |
| rs498618 | C/T | 163 | 102 | 14 | 120 | 74 | 9 | 0.95 | 428/130;314/92 | 1.04 (0.76–1.41) | 0.82 |
| rs1537416 | C/T | 68 | 216 | 196 | 40 | 124 | 120 | 0.92 | 193/354;153/253 | 1.03 (0.83–1.28) | 0.77 |
| rs7855716 | C/G | 4 | 67 | 206 | 4 | 33 | 116 | 0.10 | 75/479;41/365 | 0.72 (0.48–1.07) | 0.11 |
| rs755722 | C/T | 127 | 121 | 30 | 95 | 90 | 18 | 0.78 | 375/181;280/126 | 1.07 (0.81–1.41) | 0.62 |
| rs10735239 | C/T | 92 | 133 | 53 | 59 | 109 | 34 | 0.42 | 317/239;227/177 | 0.97 (0.75–1.25) | 0.80 |
| rs11103167 | C/T | 33 | 129 | 117 | 27 | 83 | 92 | 0.53 | 195/363;137/267 | 0.96 (0.73–1.25) | 0.74 |
| rs10118746 | A/G | 146 | 116 | 14 | 117 | 74 | 18 | 0.17 | 408/144;296/110 | 0.95 (0.71–1.27) | 0.73 |
| rs11103182 | G/T | 62 | 113 | 101 | 54 | 74 | 74 | 0.49 | 237/315;182/222 | 1.09 (0.84–1.41) | 0.52 |
| rs1318383 | C/T | 139 | 55 | 8 | 182 | 85 | 10 | 0.71 | 449/105;333/71 | 1.10 (0.79–1.53) | 0.59 |
| rs10776845 | C/A | 146 | 96 | 17 | 108 | 64 | 10 | 0.79 | 388/130; 280/84 | 0.90 (0.65–1.23) | 0.49 |
| rs10776850 | T/A | 258 | 3 | 0 | 192 | 1 | 0 | 0.48 | 519/3; 385/1 | 0.45 (0.05–4.34) | 0.48 |
| rs10858173 | T/C | 139 | 115 | 22 | 101 | 89 | 12 | 0.66 | 393/159; 291/113 | 0.96 (0.66–1.03) | 0.78 |

Table 4 Caption: Allele and genotype frequencies of 17 SNPs of KCNT1 in drug‐responsive GGEs patients (n = 279) and drug‐resistant GGEs patients (n = 204)

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b85f/6493076/c2c7566b7d21/CNS-20-140-g004.jpg)

The frequencies of haplotypes (>3%) in all s tagSNPs of KCNT1 in GGEs drug‐resistant patients (n = 204) and GGEs drug‐responsive patients (n = 279). All those frequencies<0.03 will be ignored in analysis.

## Discussion

Despite more and more research focused on the pathogenesis of epilepsy, especially on the GGEs, we just know the tip of the iceberg. Most of the currently known genes for rare monogenic forms of genetic generalized epilepsies encode voltage‐gated or ligand‐gated ion channels (e.g., SCN1A, SCN2B, GABRA1, KCNQ2, KCNQ3, CHRNA4) [14](#cns12169-bib-0014), [15](#cns12169-bib-0015). But these candidate genes could not play a substantial role in the exact functional effect on GGEs. Some Genome‐wide linkage scans or Genome‐wide copy number variation researches found some susceptibility loci regions, but no susceptibility gene was found [1](#cns12169-bib-0001), [8](#cns12169-bib-0008), [16](#cns12169-bib-0016), [17](#cns12169-bib-0017).

In contrast to the positional gene mapping strategies, lots of small‐scale linkage and candidate gene association studies failed to identify replicable susceptibility genes for common GGE syndromes [10](#cns12169-bib-0010), [16](#cns12169-bib-0016), [17](#cns12169-bib-0017), [18](#cns12169-bib-0018), [19](#cns12169-bib-0019), [20](#cns12169-bib-0020). Recent studies using exome sequencing identified that *KCNT1* (also known as SLACK, SLO2.2 and KCa4.1) channel mutations cause malignant migrating partial seizures of infancy and severe autosomal dominant nocturnal frontal lobe epilepsy [4](#cns12169-bib-0004), [5](#cns12169-bib-0005). *KCNT1* encodes a sodium‐activated potassium (KNa) channel and its protein represents the largest known potassium channel subunit [11](#cns12169-bib-0011). To investigate the function of *KCNT1* mutations, Barcia transfected *Xenopus laevis* oocytes with rat wild‐type and mutant *KCNT1* constructs, then found the mutations constitutively activate the KCNT1 channel in a way that mimicked the effects of the phosphorylation of the C‐terminal domain by protein kinase C [5](#cns12169-bib-0005), [21](#cns12169-bib-0021).

Now we first performed this research to find out the association between *KCNT1* common variations and the susceptibility of genetic generalized epilepsies in Chinese population. In our study, we selected 17 tagSNPs of *KCNT1*, detected these loci in 483 Chinese GGEs patients and 284 controls, analyzed the association and then found none of these tagSNPs and their haplotypes were associated with GGEs, even the subtypes of GGEs (Table** ** [3](#cns12169-tbl-0003),Figure [3](#cns12169-fig-0003)). It is the first study on novel candidate gene *KCNT1* common variations whether related with the pathogenesis of GGEs. Our data imply that the common variations probability not the cause of GGEs in our study.

Because of the complexity and multiple factors influence, it is hard to find the real susceptibility genes directly inducing the occurrence of seizure. Moreover, the complex phenotypes of epilepsy induced itself more mystery and complex. And we also tried to find more accuracy method to define the phenotypes of epilepsy, and wish to find the inner connection of some subtypes of GGEs and genetic factors. But after we analyzed the subtypes of GGEs, we find no association with KCNT1 common variations.

Despite using lots of anti‐epileptic drugs to cure, even more and more AEDs were used over the past 20 years, there are still 30% of patients with epilepsy continue to have seizures after using AEDs [11](#cns12169-bib-0011). It is becoming increasingly clear that genetic polymorphisms play an integral role in variability of AEDs pharmacodynamics. Anti‐epileptic drugs should through the course of drug absorption in the gastrointestinal tract to drug distribution to the brain, drug actions at brain targets, and finally hepatic metabolism and renal excretion. Each course can change the efficacy of AEDs. Although presently available AEDs appear to be directed against a relatively small number of targets (mainly ion channels or other components of the synaptic machinery), it is complicated to find the real drug resistance by the fact that many AEDs seem to work through multiple mechanisms, some of which are still unresolved. Most of previous studies on AEDs resistance focused on the ABC transporters or voltage‐gated sodium channels and none focused on the novel new candidate susceptibility gene *KCNT1* [11](#cns12169-bib-0011), [22](#cns12169-bib-0022), [23](#cns12169-bib-0023), [24](#cns12169-bib-0024), [25](#cns12169-bib-0025). In those previous studies, the samples have different epileptic pathogens and were not divided to idiopathic or symptomatic. The different pathogens of epilepsy may influence the efficacy of anti‐epileptic drugs.

In our study, we used the genetic generalized epileptic patients as study objects to exclude the interference of different epileptic causes to results. *KCNT1* encoded the sodium‐activated potassium (K_Na_) channel, whose activity contributes to the slow hyperpolarization that follows repetitive firing. The channel could regulate the rate of bursting, enhance the accuracy with which action potentials lock to incoming stimuli to make sure the completion of the action potential [26](#cns12169-bib-0026). Many AEDs could directly bind or indirectly influence the sodium channels to play drug effects, and the function change could directly affect the drug efficacy. Moreover, the channel also could influence the baseline of neuron cell electrophysiological and indirectly affected the drug efficacy. We performed our studies on the association between *KCNT1* common variations and AEDs efficacy in 279 drug‐responsive Chinese GGEs patients and 204 drug‐resistant Chinese GGEs patients, but found no statistically significant association between each other. We also analyzed the haplotypes of *KCNT1* 17 tagSNPs and found that the frequencies of all eight haplotypes were no statistically significant difference (Table** ** [4](#cns12169-tbl-0004) **,** Figure [4](#cns12169-fig-0004)).

Epilepsy is a complex neurological disease and was thought to be the outcome of ploygenic‐environment interactions. And still lots of epileptic patients cannot find the real causes. Moreover, there were no precise evaluation criteria of AEDs efficacy. The relative small sample size also is the shortcomings of our study. All these causes limited the results of association research. In our studies, we performed a gene‐wide tagging study of the association between *KCNT1* polymorphisms and the susceptibility and efficacy of genetic generalized epilepsy in Chinese population. And found that *KCNT1* 17 tagSNPs were not associated with the susceptibility and efficacy of genetic generalized epilepsy in Chinese population in our research.

## Conflict of Interest

The authors declare no conflict of interest.

## Acknowledgments

We thank the study participants and supported grants of the National High‐tech R&D Program of China (863 Program) (2012AA02A517), National Natural Science Foundation of China (81173129, 81202595), Program for the Special Scientific Research Foundation of Doctor Disciplines in University of Ministry of Education of China (20110162110034), Hunan Provincial Natural Science Foundation of China (12JJ7006), Doctoral research and innovation projects in Hunan Province and Open Foundation of Innovative Platform in University of Hunan Province of China (10K078).

## References

1. Mefford HC, Muhle H, Ostertag P, et al. Genome‐wide copy number variation in epilepsy: Novel susceptibility loci in idiopathic generalized and focal epilepsies. PLoS Genet 2010;6:e1000962.  [DOI](https://doi.org/10.1371/journal.pgen.1000962) | [PMC free article](/articles/PMC2873910/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20502679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Genome%E2%80%90wide%20copy%20number%20variation%20in%20epilepsy:%20Novel%20susceptibility%20loci%20in%20idiopathic%20generalized%20and%20focal%20epilepsies&author=HC%20Mefford&author=H%20Muhle&author=P%20Ostertag&volume=6&publication_year=2010&pages=e1000962&pmid=20502679&doi=10.1371/journal.pgen.1000962&)

2. Weber YG, Lerche H. Genetic mechanisms in idiopathic epilepsies. Dev Med Child Neurol 2008;50:648–654.  [DOI](https://doi.org/10.1111/j.1469-8749.2008.03058.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18754913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dev%20Med%20Child%20Neurol&title=Genetic%20mechanisms%20in%20idiopathic%20epilepsies&author=YG%20Weber&author=H%20Lerche&volume=50&publication_year=2008&pages=648-654&pmid=18754913&doi=10.1111/j.1469-8749.2008.03058.x&)

3. Jallon P, Latour P. Epidemiology of idiopathic generalized epilepsies. Epilepsia 2005;46(Suppl. 9):10–14.  [DOI](https://doi.org/10.1111/j.1528-1167.2005.00309.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16302871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Epidemiology%20of%20idiopathic%20generalized%20epilepsies&author=P%20Jallon&author=P%20Latour&volume=46&issue=Suppl.%209&publication_year=2005&pages=10-14&pmid=16302871&doi=10.1111/j.1528-1167.2005.00309.x&)

4. Heron SE, Smith KR, Bahlo M, et al. Missense mutations in the sodium‐gated potassium channel gene KCNT1 cause severe autosomal dominant nocturnal frontal lobe epilepsy. Nat Genet 2012;44:1188–1190.  [DOI](https://doi.org/10.1038/ng.2440) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23086396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Missense%20mutations%20in%20the%20sodium%E2%80%90gated%20potassium%20channel%20gene%20KCNT1%20cause%20severe%20autosomal%20dominant%20nocturnal%20frontal%20lobe%20epilepsy&author=SE%20Heron&author=KR%20Smith&author=M%20Bahlo&volume=44&publication_year=2012&pages=1188-1190&pmid=23086396&doi=10.1038/ng.2440&)

5. Barcia G, Fleming MR, Deligniere A, et al. De novo gain‐of‐function KCNT1 channel mutations cause malignant migrating partial seizures of infancy. Nat Genet 2012;44:1255–1259.  [DOI](https://doi.org/10.1038/ng.2441) | [PMC free article](/articles/PMC3687547/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23086397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=De%20novo%20gain%E2%80%90of%E2%80%90function%20KCNT1%20channel%20mutations%20cause%20malignant%20migrating%20partial%20seizures%20of%20infancy&author=G%20Barcia&author=MR%20Fleming&author=A%20Deligniere&volume=44&publication_year=2012&pages=1255-1259&pmid=23086397&doi=10.1038/ng.2441&)

6. Loscher W, Klotz U, Zimprich F, Schmidt D. The clinical impact of pharmacogenetics on the treatment of epilepsy. Epilepsia 2009;50:1–23.  [DOI](https://doi.org/10.1111/j.1528-1167.2008.01716.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18627414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=The%20clinical%20impact%20of%20pharmacogenetics%20on%20the%20treatment%20of%20epilepsy&author=W%20Loscher&author=U%20Klotz&author=F%20Zimprich&author=D%20Schmidt&volume=50&publication_year=2009&pages=1-23&pmid=18627414&doi=10.1111/j.1528-1167.2008.01716.x&)

7. Bhattacharjee A, Kaczmarek LK. For K+ channels, Na+ is the new Ca2 + . Trends Neurosci 2005;28:422–428.  [DOI](https://doi.org/10.1016/j.tins.2005.06.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15979166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Neurosci&title=For%20K+%20channels,%20Na+%20is%20the%20new%20Ca2%C2%A0+%C2%A0&author=A%20Bhattacharjee&author=LK%20Kaczmarek&volume=28&publication_year=2005&pages=422-428&pmid=15979166&doi=10.1016/j.tins.2005.06.003&)

8. Steffens M, Leu C, Ruppert AK, et al. Genome‐wide association analysis of genetic generalized epilepsies implicates susceptibility loci at 1q43, 2p16.1, 2q22.3 and 17q21.32. Hum Mol Genet 2012;21:5359–5372.  [DOI](https://doi.org/10.1093/hmg/dds373) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22949513/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Genome%E2%80%90wide%20association%20analysis%20of%20genetic%20generalized%20epilepsies%20implicates%20susceptibility%20loci%20at%201q43,%202p16.1,%202q22.3%20and%2017q21.32&author=M%20Steffens&author=C%20Leu&author=AK%20Ruppert&volume=21&publication_year=2012&pages=5359-5372&pmid=22949513&doi=10.1093/hmg/dds373&)

9. Proposal for revised classification of epilepsies and epileptic syndromes. Commission on Classification and Terminology of the International League Against Epilepsy. Epilepsia 1989;30:389–399.  [DOI](https://doi.org/10.1111/j.1528-1157.1989.tb05316.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2502382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Proposal%20for%20revised%20classification%20of%20epilepsies%20and%20epileptic%20syndromes.%20Commission%20on%20Classification%20and%20Terminology%20of%20the%20International%20League%20Against%20Epilepsy&volume=30&publication_year=1989&pages=389-399&pmid=2502382&doi=10.1111/j.1528-1157.1989.tb05316.x&)

10. Greenberg DA, Subaran R. Blinders, phenotype, and fashionable genetic analysis: A critical examination of the current state of epilepsy genetic studies. Epilepsia 2011;52:1–9.  [DOI](https://doi.org/10.1111/j.1528-1167.2010.02904.x) | [PMC free article](/articles/PMC3021750/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21219301/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Blinders,%20phenotype,%20and%20fashionable%20genetic%20analysis:%20A%20critical%20examination%20of%20the%20current%20state%20of%20epilepsy%20genetic%20studies&author=DA%20Greenberg&author=R%20Subaran&volume=52&publication_year=2011&pages=1-9&pmid=21219301&doi=10.1111/j.1528-1167.2010.02904.x&)

11. Kwan P, Wong V, Ng PW, et al. Gene‐wide tagging study of the association between ABCC2, ABCC5 and ABCG2 genetic polymorphisms and multidrug resistance in epilepsy. Pharmacogenomics 2011;12:319–325.  [DOI](https://doi.org/10.2217/pgs.10.183) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21449672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Gene%E2%80%90wide%20tagging%20study%20of%20the%20association%20between%20ABCC2,%20ABCC5%20and%20ABCG2%20genetic%20polymorphisms%20and%20multidrug%20resistance%20in%20epilepsy&author=P%20Kwan&author=V%20Wong&author=PW%20Ng&volume=12&publication_year=2011&pages=319-325&pmid=21449672&doi=10.2217/pgs.10.183&)

12. Shi YY, He L. SHEsis, a powerful software platform for analyses of linkage disequilibrium, haplotype construction, and genetic association at polymorphism loci. Cell Res 2005;15:97–98.  [DOI](https://doi.org/10.1038/sj.cr.7290272) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15740637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=SHEsis,%20a%20powerful%20software%20platform%20for%20analyses%20of%20linkage%20disequilibrium,%20haplotype%20construction,%20and%20genetic%20association%20at%20polymorphism%20loci&author=YY%20Shi&author=L%20He&volume=15&publication_year=2005&pages=97-98&pmid=15740637&doi=10.1038/sj.cr.7290272&)

13. Li Z, Zhang Z, He Z, et al. A partition‐ligation‐combination‐subdivision EM algorithm for haplotype inference with multiallelic markers: Update of the SHEsis (http://analysis.bio-x.cn). Cell Res 2009;19:519–523.  [http://analysis.bio-x.cn](http://analysis.bio-x.cn) | [DOI](https://doi.org/10.1038/cr.2009.33) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19290020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=A%20partition%E2%80%90ligation%E2%80%90combination%E2%80%90subdivision%20EM%20algorithm%20for%20haplotype%20inference%20with%20multiallelic%20markers:%20Update%20of%20the%20SHEsis&author=Z%20Li&author=Z%20Zhang&author=Z%20He&volume=19&publication_year=2009&pages=519-523&pmid=19290020&doi=10.1038/cr.2009.33&)

14. Reid CA, Berkovic SF, Petrou S. Mechanisms of human inherited epilepsies. Prog Neurobiol 2009;87:41–57.  [DOI](https://doi.org/10.1016/j.pneurobio.2008.09.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18952142/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Neurobiol&title=Mechanisms%20of%20human%20inherited%20epilepsies&author=CA%20Reid&author=SF%20Berkovic&author=S%20Petrou&volume=87&publication_year=2009&pages=41-57&pmid=18952142&doi=10.1016/j.pneurobio.2008.09.016&)

15. Gardiner M. Genetics of idiopathic generalized epilepsies. Epilepsia 2005;46(Suppl. 9):15–20.  [DOI](https://doi.org/10.1111/j.1528-1167.2005.00310.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16302872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Genetics%20of%20idiopathic%20generalized%20epilepsies&author=M%20Gardiner&volume=46&issue=Suppl.%209&publication_year=2005&pages=15-20&pmid=16302872&doi=10.1111/j.1528-1167.2005.00310.x&)

16. Hempelmann A, Taylor KP, Heils A, et al. Exploration of the genetic architecture of idiopathic generalized epilepsies. Epilepsia 2006;47:1682–1690.  [DOI](https://doi.org/10.1111/j.1528-1167.2006.00677.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17054691/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Exploration%20of%20the%20genetic%20architecture%20of%20idiopathic%20generalized%20epilepsies&author=A%20Hempelmann&author=KP%20Taylor&author=A%20Heils&volume=47&publication_year=2006&pages=1682-1690&pmid=17054691&doi=10.1111/j.1528-1167.2006.00677.x&)

17. Kasperaviciute D, Catarino CB, Heinzen EL, et al. Common genetic variation and susceptibility to partial epilepsies: A genome‐wide association study. Brain 2010;133:2136–2147.  [DOI](https://doi.org/10.1093/brain/awq130) | [PMC free article](/articles/PMC2892941/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20522523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain&title=Common%20genetic%20variation%20and%20susceptibility%20to%20partial%20epilepsies:%20A%20genome%E2%80%90wide%20association%20study&author=D%20Kasperaviciute&author=CB%20Catarino&author=EL%20Heinzen&volume=133&publication_year=2010&pages=2136-2147&pmid=20522523&doi=10.1093/brain/awq130&)

18. Cavalleri GL, Weale ME, Shianna KV, et al. Multicentre search for genetic susceptibility loci in sporadic epilepsy syndrome and seizure types: A case‐control study. Lancet Neurol 2007;6:970–980.  [DOI](https://doi.org/10.1016/S1474-4422(07)70247-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17913586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Neurol&title=Multicentre%20search%20for%20genetic%20susceptibility%20loci%20in%20sporadic%20epilepsy%20syndrome%20and%20seizure%20types:%20A%20case%E2%80%90control%20study&author=GL%20Cavalleri&author=ME%20Weale&author=KV%20Shianna&volume=6&publication_year=2007&pages=970-980&pmid=17913586&doi=10.1016/S1474-4422(07)70247-8&)

19. Tan NC, Berkovic SF. The Epilepsy Genetic Association Database (epiGAD): Analysis of 165 genetic association studies, 1996–2008. Epilepsia 2010;51:686–689.  [DOI](https://doi.org/10.1111/j.1528-1167.2009.02423.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20074235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=The%20Epilepsy%20Genetic%20Association%20Database%20(epiGAD):%20Analysis%20of%20165%20genetic%20association%20studies,%201996%E2%80%932008&author=NC%20Tan&author=SF%20Berkovic&volume=51&publication_year=2010&pages=686-689&pmid=20074235&doi=10.1111/j.1528-1167.2009.02423.x&)

20. Poduri A, Lowenstein D. Epilepsy genetics–past, present, and future. Curr Opin Genet Dev 2011;21:325–332.  [DOI](https://doi.org/10.1016/j.gde.2011.01.005) | [PMC free article](/articles/PMC4074083/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21277190/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Genet%20Dev&title=Epilepsy%20genetics%E2%80%93past,%20present,%20and%20future&author=A%20Poduri&author=D%20Lowenstein&volume=21&publication_year=2011&pages=325-332&pmid=21277190&doi=10.1016/j.gde.2011.01.005&)

21. Aminkeng F. KCNT1 mutations in ADNFLE and MMPSI: A new driver in the etiology and pathophysiology of early‐onset epileptic syndromes. Clin Genet 2013;83:319–320.  [DOI](https://doi.org/10.1111/cge.12082) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23278465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Genet&title=KCNT1%20mutations%20in%20ADNFLE%20and%20MMPSI:%20A%20new%20driver%20in%20the%20etiology%20and%20pathophysiology%20of%20early%E2%80%90onset%20epileptic%20syndromes&author=F%20Aminkeng&volume=83&publication_year=2013&pages=319-320&pmid=23278465&doi=10.1111/cge.12082&)

22. Qu J, Zhou BT, Yin JY, et al. ABCC2 polymorphisms and haplotype are associated with drug resistance in Chinese epileptic patients. CNS Neurosci Ther 2012;18:647–651.  [DOI](https://doi.org/10.1111/j.1755-5949.2012.00336.x) | [PMC free article](/articles/PMC6493375/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22630058/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CNS%20Neurosci%20Ther&title=ABCC2%20polymorphisms%20and%20haplotype%20are%20associated%20with%20drug%20resistance%20in%20Chinese%20epileptic%20patients&author=J%20Qu&author=BT%20Zhou&author=JY%20Yin&volume=18&publication_year=2012&pages=647-651&pmid=22630058&doi=10.1111/j.1755-5949.2012.00336.x&)

23. Haerian BS, Lim KS, Tan CT, Raymond AA, Mohamed Z. Association of ABCB1 gene polymorphisms and their haplotypes with response to antiepileptic drugs: A systematic review and meta‐analysis. Pharmacogenomics 2011;12:713–725.  [DOI](https://doi.org/10.2217/pgs.10.212) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21391884/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Association%20of%20ABCB1%20gene%20polymorphisms%20and%20their%20haplotypes%20with%20response%20to%20antiepileptic%20drugs:%20A%20systematic%20review%20and%20meta%E2%80%90analysis&author=BS%20Haerian&author=KS%20Lim&author=CT%20Tan&author=AA%20Raymond&author=Z%20Mohamed&volume=12&publication_year=2011&pages=713-725&pmid=21391884&doi=10.2217/pgs.10.212&)

24. Abe T, Seo T, Ishitsu T, Nakagawa T, Hori M, Nakagawa K. Association between SCN1A polymorphism and carbamazepine‐resistant epilepsy. Br J Clin Pharmacol 2008;66:304–307.  [DOI](https://doi.org/10.1111/j.1365-2125.2008.03203.x) | [PMC free article](/articles/PMC2492927/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18489610/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Association%20between%20SCN1A%20polymorphism%20and%20carbamazepine%E2%80%90resistant%20epilepsy&author=T%20Abe&author=T%20Seo&author=T%20Ishitsu&author=T%20Nakagawa&author=M%20Hori&volume=66&publication_year=2008&pages=304-307&pmid=18489610&doi=10.1111/j.1365-2125.2008.03203.x&)

25. Ufer M, Mosyagin I, Muhle H, et al. Non‐response to antiepileptic pharmacotherapy is associated with the ABCC2 ‐24C>T polymorphism in young and adult patients with epilepsy. Pharmacogenet Genomics 2009;19:353–362.  [DOI](https://doi.org/10.1097/fpc.0b013e328329940b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19415824/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Non%E2%80%90response%20to%20antiepileptic%20pharmacotherapy%20is%20associated%20with%20the%20ABCC2%20%E2%80%9024C>T%20polymorphism%20in%20young%20and%20adult%20patients%20with%20epilepsy&author=M%20Ufer&author=I%20Mosyagin&author=H%20Muhle&volume=19&publication_year=2009&pages=353-362&pmid=19415824&doi=10.1097/fpc.0b013e328329940b&)

26. Brown MR, Kronengold J, Gazula VR, et al. Amino‐termini isoforms of the Slack K+ channel, regulated by alternative promoters, differentially modulate rhythmic firing and adaptation. J Physiol 2008;586:5161–5179.  [DOI](https://doi.org/10.1113/jphysiol.2008.160861) | [PMC free article](/articles/PMC2652154/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18787033/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Physiol&title=Amino%E2%80%90termini%20isoforms%20of%20the%20Slack%20K+%20channel,%20regulated%20by%20alternative%20promoters,%20differentially%20modulate%20rhythmic%20firing%20and%20adaptation&author=MR%20Brown&author=J%20Kronengold&author=VR%20Gazula&volume=586&publication_year=2008&pages=5161-5179&pmid=18787033&doi=10.1113/jphysiol.2008.160861&)
