# RYR1 and CACNA1S genetic variants identified with statin-associated muscle symptoms

## Metadata
**Authors:** Paul J Isackson, Jianxin Wang, Mohammad Zia, Paul Spurgeon, Adrian Levesque, Jonathan Bard, Smitha James, Norma Nowak, Tae Keun Lee, Georgirene D Vladutiu
**Journal:** Pharmacogenomics
**Date:** 2018 Oct 16
**DOI:** [10.2217/pgs-2018-0106](https://doi.org/10.2217/pgs-2018-0106)
**PMID:** 30325262
**PMCID:** PMC6563124
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6563124/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6563124/pdf/pgs-19-1235.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6563124/pdf/pgs-19-1235.pdf)

## Abstract

**Aim:** 
To examine the genetic differences between subjects with statin-associated muscle symptoms and statin-tolerant controls.

**Materials & methods:** 
Next-generation sequencing was used to characterize the exomes of 76 subjects with severe statin-associated muscle symptoms and 50 statin-tolerant controls.

**Results:** 
12 probably pathogenic variants were found within the RYR1 and CACNA1S genes in 16% of cases with severe statin-induced myopathy representing a fourfold increase over variants found in statin-tolerant controls. Subjects with probably pathogenic RYR1 or CACNA1S variants had plasma CK 5X to more than 400X the upper limit of normal in addition to having muscle symptoms.

**Conclusions:** 
Genetic variants within the RYR1 and CACNA1S genes are likely to be a major contributor to the susceptibility to statin-associated muscle symptoms.

Keywords: : exome sequencing, malignant hyperthermia, myopathy, RYR1, statin

### Aim:

To examine the genetic differences between subjects with statin-associated muscle symptoms and statin-tolerant controls.

### Materials & methods:

Next-generation sequencing was used to characterize the exomes of 76 subjects with severe statin-associated muscle symptoms and 50 statin-tolerant controls.

### Results:

12 probably pathogenic variants were found within the *RYR1* and *CACNA1S* genes in 16% of cases with severe statin-induced myopathy representing a fourfold increase over variants found in statin-tolerant controls. Subjects with probably pathogenic *RYR1* or *CACNA1S* variants had plasma CK 5X to more than 400X the upper limit of normal in addition to having muscle symptoms.

### Conclusions:

Genetic variants within the *RYR1* and *CACNA1S* genes are likely to be a major contributor to the susceptibility to statin-associated muscle symptoms.

## Methods

### Study subjects

A total of 748 subjects (392 males) with a history of statin therapy were enrolled in a retrospective case–control study from five medical centers across the USA and Canada between 2004 and 2013 and representing 20 states and provinces in the USA and Canada. Collaborating institutions included the Johns Hopkins Myositis Center, MD, USA; Cedars-Sinai Medical Center, CA, USA; the Medical College of Wisconsin, WI, USA; McMaster University Medical Center; and the University of Oklahoma College of Medicine, Tulsa, OK, USA. A standardized classification of SAMS integrates all muscle-related symptoms (e.g., pain, weakness or cramps) as ‘muscle symptoms’, which are then placed in subgroups depending on the presence or absence of elevated plasma CK. Pain and weakness in typical SAMS are usually symmetrical and proximal, and generally affect large muscle groups. The myalgias and weakness typically occur within 4–6 weeks after starting statin therapy, however, they may occur after many years of treatment [[32](#B32)]. The SAMS group was comprised of 634 individuals (340 males) and a statin-tolerant control group of 114 individuals (52 males) who continued to take statins for at least 12 months without muscle symptoms. Individuals were classified as having SAMS based on their responses to a questionnaire outlining the statin type and dosage that first led to myopathic symptoms, the date of onset of symptoms, the extent of muscle symptoms during therapy and the duration of symptoms post-cessation of statin therapy. Additional information was collected pertaining to personal or family history of heart disease, muscle disease and other co-existing medical conditions. Statin-tolerant controls completed a follow-up questionnaire 6 and 12 months post initial enrollment to confirm that statin tolerance was maintained. 26 subjects of Caucasian, Euro-American descent (19 males, mean age 57 years) of 634 individuals in the SAMS group were selected for whole exome sequencing (WES) using specific selection criteria. All had severe muscle pain and/or weakness attributed solely to statin therapy; plasma creatine kinase (CK) levels >5-times the upper limit of normal (ULN); and all were <65 years of age when muscle symptoms began. More than 70% had prolonged symptoms post therapy for at least 6 months. An additional 50 subjects (30 males); 81% Caucasian; 4% Asian; 15% African–American (mean age: 62 years) with severe SAMS were chosen for whole genome sequencing (WGS). These were selected primarily based on severity of muscle symptoms and a plasma CK level >5X ULN. One subject did not have muscle symptoms but had a CK level of 12X ULN and was included. DNA from 50 subjects (20 males, mean age: 63 years, 100% Caucasian, Euro-American) from the statin-tolerant control group were also submitted to WES.

### DNA sequencing

Genomic DNA was purified from blood or saliva specimens with Puregene or OraGene protocols, respectively. Whole exome sequencing was performed on 26 of the samples with the Illumina HiSeq instrument and Nimblegen V2 exome enrichment kits in Dr Deborah Nickerson's laboratory at the Northwest Genome Center in Seattle, WA, USA. Sequence data was processed to generate the VCF file as described by Kim *et al*. [[33](#B33)]. WGS was carried out on 50 of the SAMS cases by the New York Genome Center (NYGC; NY, USA) with the Illumina HiSeq. WES was run on 50 statin-tolerant controls with the Trueseq exome capture kit (Illumina). Sequencing was performed using the Illumina HiSeq. Each sample was run in eight lanes.

### Data processing

This includes sequence alignment (for WGS and WES control sequence data only, WES case data were obtained as BAM files from the Northwest Genome Center and used directly for variant calling, see below), sorting alignment by genomic coordinates and marked PCR duplicates, followed by variant calling. All of these were performed using the high performance computing cluster hosted at the Center for Computational Research, University at Buffalo. To align the raw fastq sequences to the reference assembly, we aligned the fastq files generated for each lane from the Illumina platform against the human_g1k_v37_decoy.fasta (download from [https://software.broadinstitute.org/gatk/download/bundle](https://software.broadinstitute.org/gatk/download/bundle)) using BWA-MEM and provided read group (ID, Library, Sample, etc.) information that was extracted from sequence file names. The resulting bam files are sorted by genomic coordinates and subsequently merged to obtain one bam file per sample using Picard (v.1.131), with PCR duplicates marked during this step. For variant calling, we used an in-house developed variant calling pipeline utilizing modules from the Genome Analysis Tool Kit (GATK, v3.6, Broad Institute) with parameters according to their best practice documentation. Briefly, the bam files are subjected to base quality recalibration, joint genotyping using ‘HaplotypeCaller’ and ‘GenotypeGVCFs.’ Raw variants are filtered using the ‘Variant Quality Score Recalibration’ tool and output as a multisample VCF file for each of the sequencing cohort (WGS, WES case or WES control).

Coverage analysis of the sequencing runs showed in all cases that exon 91 had relatively low coverage ([Supplementary Figure 1](http:///doi/suppl/10.2217/pgs-2018-0106/suppl_file/pgs-19-1235-s2.docx)), which is consistent with no data being presented for this region in the ExAC database. The greatest overall coverage was obtained with whole exome sequencing of the statin-tolerant control group ensuring that there was no bias toward identifying more genetic variants in the case group than in the control group.

### Variant annotation

VCF files from above are used as input for variant annotation using Annovar (1 February, 2016 version) [[34](#B34)]. We annotated variants for population frequencies (ExAC, 1000 Genome Project Phase III, ESP6500), pathogenic predictions (SIFT, PolyPhen2, CADD, REVEL), functional consequences, disease associations (ClinVar, GWAS catalog) – among others.

### Candidate variants selection

Annotated VCF files were used as input to populate the Genomic Data Warehouse (GDW) developed at the Center for Computational Research, University at Buffalo. It is a database tool aimed at facilitating genomic variant data storage as well as fast and efficient retrieval for variants satisfying user defined filtering criteria. To search for candidate variants, three steps are needed, in other words, select dataset, set/select filters and select attributes/field names to return. Specifically, we used the following filtering criteria to obtain our list of candidate disease associated variants: RefGene IDs: [NM_000540](https://www.ncbi.nlm.nih.gov/nuccore/NM_000540), [NM_000069](https://www.ncbi.nlm.nih.gov/nuccore/NM_000069) and the cases and controls were also screened for a list of 114 genes associated with myopathy compiled by Abath Neto [[16](#B16)]. The list was also limited to minor allele frequency (MAF) in ExAC All populations: <0.02. For attributes/field names to return, we included chromosome, chromosomal position, reference allele, alternate allele, variant type, GATK VQSR score, dbSNP ID, gene, exon, cDNA change, amino acid change, REVEL score, dbscSNV_ADA score, dbscSNV_RF score, MAF in the ExAC All populations database, ClinVar clinical significance and sample ID.

Variants were categorized into five classes of pathogenicity based on the type of variant, frequency in the general population from the ExAC database and reports in the literature and the HGMD, ClinVar and Leiden Muscular Dystrophy databases. As previously reported [[35](#B35)], variant effect prediction algorithms do not correctly predict the effect of all *RYR1* and *CACNA1S* variants, both pathogenic and benign. This is shown in [Supplementary Table 1](http:///doi/suppl/10.2217/pgs-2018-0106/suppl_file/pgs-19-1235-s5.xlsx) with the 44 established *RYR1* and *CACNA1S* pathogenic variants that have been functionally demonstrated to cause MHS ([emhg.org](http://emhg.org)). More than half of the 44 known MHS causing mutations are predicted to be tolerated by the SIFT algorithm. Consensus prediction algorithms integrating several different individual algorithms have been found to be more accurate [[36](#B36),[37](#B37)]. We have used REVEL, which integrates prediction scores from 13 different individual algorithm tools [[38](#B38)] and accurately predicts all of the EMHG mutations to be pathogenic. The REVEL score does, however, incorrectly predict one of the benign common *CACNA1S* variants, G258D, examined by Schiemann and Stoell [[35](#B35)] as pathogenic ([Supplementary Table 2](http:///doi/suppl/10.2217/pgs-2018-0106/suppl_file/pgs-19-1235-s6.xlsx)) and also incorrectly predicts the pathogenicity of a number of the variants identified in this study that are too frequent in the population to be pathogenic, such as *RYR1* K1393R, R1679H, R3539H, H3647Q and D4505H ([Table 1](#T1)) and *CACNA1S* G258D, S606N and R683C ([Table 2](#T2)). While the REVEL score cannot be relied upon to be completely accurate, it is the most accurate prediction approach currently available.

### Table 1. . RYR1 variants <0.02 minor allele frequency.

| Chr | Position (hg19) | cDNA change | aa change | Exon | Class | REVEL | Conserved | dbscSNV_ADA | dbscSNV_RF | ExAC ALL | Sample ID |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |  |  |  |  |
| RYR1 variants in statin myopathy cases |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38937141 | c.C661T | p.L221F | exon8 | 4 | 0.726 | Yes | None | None | 0 | 2155 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38948227 | c.C1882T | p.R628C | exon17 | 4 | 0.745 | Yes | None | None | 2.47E-05 | 2048 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38948268 | c.C1923G | p.T641T | exon17 | 1 | None | Yes | 0.5625 | 0.336 | 0.0009 | 2063 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38949859 | c.G2241A | p.L747L | exon19 | 1 | None | Yes | None | None | 0.0014 | 1443, 2157 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38954162 | c.G2677A | p.G893S | exon21 | 2 | 0.837 | Yes | None | None | 0.0006 | 2157 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38956779 | c.C2919T | p.H973H | exon24 | 1 | None | Yes | None | None | 0.0021 | 2168 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38958397 | c.G3326A | p.R1109K | exon25 | 1 | 0.088 | Yes | None | None | 0.0017 | 1443, 2157 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38960067 | c.A3679G | p.I1227V | exon27 | 2 | 0.364 | Yes | None | None | 8.26E-06 | 2170 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38964109 | c.T3858C | p.L1286L | exon28 | 1 | None | No | None | None | 0.0149 | 2048, 2074, 2157 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38964317 | c.G4066A | p.A1356T | exon28 | 2 | 0.157 | No | None | None | 0 | 2063 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38964322 | c.C4071T | p.P1357P | exon28 | 1 | None | Yes | None | None | 0.0006 | 2074 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38966001 | c.C4204T | p.P1402S | exon29 | 4 | 0.563 | Yes | None | None | 0 | 2198 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38968484 | c.G4428C | p.G1476G | exon30 | 1 | None | Yes | None | None | 0 | 2257 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38973933 | c.A4711G | p.I1571V | exon33 | 1 | 0.56 | Yes | None | None | 0.0013 | 2168 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38973941 | c.G4719A | p.P1573P | exon33 | 1 | None | Yes | None | None | 0.003 | 1443, 2157 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38974154 | c.C4932T | p.N1644N | exon33 | 4 | None | No | 0.9923 | 0.898 | 0 | 2196 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38976770 | c.C5475A | p.H1825Q | exon34 | 2 | 0.285 | No | None | None | 0 | 2162 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38979903 | c.G5634C | p.E1878D | exon35 | 1 | 0.061 | No | None | None | 0.0014 | 2074 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38985101 | c.C6384T | p.Y2128Y | exon39 | 1 | None | Yes | None | None | 0.0032 | 2074 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38985215 | c.C6498T | p.L2166L | exon39 | 1 | None | Yes | None | None | 0.0021 | 2157 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38987117 | c.G6732A | p.R2244R | exon41 | 1 | None | Yes | None | None | 0.001 | 2063 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38990456 | c.C7209T | p.R2403R | exon44 | 1 | None | No | None | None | 0.0154 | 2162 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38990457 | c.G7210A | p.E2404K | exon44 | 3 | 0.676 | No | None | None | 2.85E-05 | 2168 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38990632 | c.C7299T | p.L2433L | exon45 | 1 | None | Yes | None | None | 6.61E-05 | 2077 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38991600 | c.C7584T | p.P2528P | exon47 | 1 | None | Yes | None | None | 0.0142 | 2078 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38993269 | c.G7737A | p.V2579V | exon48 | 1 | None | No | None | None | 0.0029 | 2257 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38993279 | c.C7747T | p.L2583L | exon48 | 1 | None | Yes | None | None | 3.35E-05 | 2059 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38994925 | c.C7992T | p.F2664F | exon50 | 1 | None | No | None | None | 4.13E-05 | 2189 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38994959 | c.C8026T | p.R2676W | exon50 | 5 | 0.601 | Yes | None | None | 8.25E-06 | 2236 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38996014 | c.G8376A | p.R2792R | exon53 | 1 | None | Yes | None | None | 0.0048 | 2152 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38998362 | c.G8827A | p.D2943N | exon58 | 2 | 0.725 | Yes | None | None | 0.0009 | 2074 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39001154 | c.T8949C | p.S2983S | exon59 | 1 | None | Yes | None | None | 0.0012 | 2074 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39001380 | c.C9081T | p.S3027S | exon60 | 1 | None | Yes | None | None | 4.94E-05 | 2214 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39003103 | c.A9452G | p.Q3151R | exon63 | 4 | 0.684 | Yes | None | None | 0 | 2189 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39009954 | c.G10119A | p.V3373V | exon67 | 1 | None | Yes | None | None | 0.0007 | 1406 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39016132 | c.G10616A | p.R3539H | exon71 | 1 | 0.877 | Yes | None | None | 0.0018 | 2264 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39018329 | c.C10729T | p.R3577W | exon73 | 2 | 0.344 | No | None | None | 0.0001 | 2261 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39018347 | c.G10747C | p.E3583Q | exon73 | 1 | 0.32 | No | None | None | 0.0149 | 2216, 2242 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39019242 | c.C10941G | p.H3647Q | exon75 | 2 | 0.509 | Yes | None | None | 0.001 | 2074 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39026667 | c.G11547A | p.Q3849Q | exon82 | 1 | None | Yes | None | None | 0.0168 | 2145, 2221, 2223 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39057597 | c.C13484T | p.P4495L | exon92 | 2 | 0.244 | No | None | None | 5.18E-05 | 2024 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39057598 | c.G13485A | p.P4495P | exon92 | 1 | None | No | None | None | 0.0001 | 2157 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39061267 | c.T13680C | p.F4560F | exon94 | 1 | None | Yes | None | None | 0.0007 | 2261 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39070762 | c.G14505A | p.G4835G | exon100 | 1 | None | Yes | None | None | 0.0034 | 1443 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| RYR1 variants in statin-tolerant controls |  |  |  |  |  |  |  |  |  |  |  |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38943526 | c.G1312A | p.E438K | exon13 | 3 | 0.73 | No | None | None | 1.15E-05 | 1017 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38948268 | c.C1923G | p.T641T | exon17 | 1 | None | Yes | 0.5625 | 0.336 | 0.0009 | 1167 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38948886 | c.C2121A | p.G707G | exon18 | 1 | None | Yes | None | None | 0.002 | 1410 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38949938 | c.G2320A | p.G774R | exon19 | 2 | 0.623 | Yes | None | None | 0.0003 | 1230 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38956963 | c.C3103T | p.R1035W | exon24 | 4 | 0.791 | Yes | None | None | 1.35E-05 | 1007 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38964179 | c.G3928T | p.A1310S | exon28 | 2 | 0.415 | No | None | None | 0 | 1088 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38965975 | c.A4178G | p.K1393R | exon29 | 1 | 0.555 | No | None | None | 0.0051 | 1410 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38976331 | c.G5036A | p.R1679H | exon34 | 1 | 0.918 | Yes | None | None | 0.0015 | 1147 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38976655 | c.C5360T | p.P1787L | exon34 | 1 | 0.055 | No | None | None | 0.0196 | 1107, 1109, 1126 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 38987117 | c.G6732A | p.R2244R | exon41 | 1 | None | Yes | None | None | 0.001 | 1167 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39009883 | c.C10048A | p.R3350R | exon67 | 1 | None | No | None | None | 5.81E-05 | 1126 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39009954 | c.G10119A | p.V3373V | exon67 | 1 | None | Yes | None | None | 0.0007 | 1217 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39018347 | c.G10747C | p.E3583Q | exon73 | 1 | 0.32 | No | None | None | 0.0149 | 1017 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39026667 | c.G11547A | p.Q3849Q | exon82 | 1 | None | Yes | None | None | 0.0168 | 1004, 1153 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39057626 | c.G13513C | p.D4505H | exon92 | 1 | 0.663 | Yes | None | None | 0.0061 | 1142, 1147 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 19 | 39070762 | c.G14505A | p.G4835G | exon100 | 1 | None | Yes | None | None | 0.0034 | 1072 |
### Table 2. . CACNA1S variants <0.02 minor allele frequency.

| Chr | Position (hg19) | cDNA change | aa change | Exon | Class | REVEL | Conserved | dbscSNV_ADA | dbscSNV_RF | ExAC ALL | Sample ID |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |  |  |  |  |
| CACNA1S variants in statin myopathy cases |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201079385 | c.G165A | p.T55T | exon2 | 1 | None | Yes | None | None | 0.0003 | 2157 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201079298 | c.C252T | p.L84L | exon2 | 1 | None | No | None | None | 0.0017 | 2157 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201058428 | c.C858T | p.Y286Y | exon6 | 1 | None | Yes | None | None | 0.0061 | 2078, 2184, 2257 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201054624 | c.C1090T | p.R364W | exon8 | 3 | 0.554 | No | None | None | 1.65E-05 | 2214 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201047078 | c.G1548A | p.S516S | exon11 | 1 | None | No | None | None | 0.0094 | 2155, 2179, 2239 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201044689 | c.G1882A | p.G628S | exon13 | 4 | 0.742 | Yes | None | None | 8.24E-06 | 2142 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201043702 | c.C1995T | p.A665A | exon14 | 1 | None | Yes | None | None | 0.0072 | 2155, 2179, 2239 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201043650 | c.C2047T | p.R683C | exon14 | 1 | 0.6 | Yes | None | None | 0.0031 | 2185 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201042742 | c.2089_2091del | p.E697del | exon15 | 4 | None | Yes | None | None | 0 | 2157 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201038705 | c.C2385T | p.I795I | exon18 | 1 | None | No | None | None | 0 | 2339 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201038636 | c.G2454A | p.A818A | exon18 | 1 | None | No | None | None | 0.0018 | 2194 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201038610 | c.T2480C | p.M827T | exon18 | 1 | 0.316 | No | None | None | 0.0042 | 2239 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201035429 | c.C2673T | p.S891S | exon21 | 1 | None | No | None | None | 0.0121 | 2257 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201035035 | c.C2784T | p.I928I | exon22 | 1 | None | No | None | None | 0.0017 | 2179 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201029939 | c.A3261G | p.Q1087Q | exon26 | 1 | None | Yes | None | None | 0.0088 | 2066, 2143, 2181 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201028425 | c.C3417G | p.H1139Q | exon27 | 4 | 0.804 | Yes | None | None | 0 | 1394 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201020172 | c.A4053G | p.T1351T | exon33 | 1 | None | Yes | None | None | 0.0014 | 2179 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201017811 | c.G4340A | p.R1447Q | exon36 | 4 | 0.502 | Yes | 0.9847 | 0.868 | 8.28E-05 | 2024 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201012542 | c.G4915C | p.E1639Q | exon40 | 2 | 0.429 | No | None | None | 0 | 2155 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201012503 | c.C4954T | p.R1652C | exon40 | 2 | 0.366 | No | None | None | 0.0003 | 2078 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201012449 | c.T5008A | p.Y1670N | exon40 | 2 | 0.234 | No | None | None | 0.0002 | 2189 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201009404 | c.T5325C | p.N1775N | exon43 | 1 | None | No | None | None | 4.18E-05 | 2024 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| CACNA1S variants in statin-tolerant controls |  |  |  |  |  |  |  |  |  |  |  |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201058513 | c.G773A | p.G258D | exon6 | 1 | 0.784 | Yes | None | None | 0.0074 | 1010, 1088, 1121 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201046058 | c.G1817A | p.S606N | exon12 | 1 | 0.521 | Yes | None | None | 0.0088 | 1167 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201043637 | c.C2060A | p.S687Y | exon14 | 2 | 0.435 | No | None | None | 0 | 1072 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201029939 | c.A3261G | p.Q1087Q | exon26 | 1 | None | Yes | None | None | 0.0088 | 1113 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201013506 | c.G4747A | p.E1583K | exon39 | 2 | 0.681 | Yes | None | None | 0.0002 | 1237 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| 1 | 201009011 | c.G5570A | p.S1857N | exon44 | 1 | 0.339 | No | None | None | 0.0019 | 1113, 1147 |
Outside of *in silico* predictions, two criteria hold for all of the EMHG established mutations; they are absolutely conserved in sequence comparisons across species ([Supplementary Figures 2](http:///doi/suppl/10.2217/pgs-2018-0106/suppl_file/pgs-19-1235-s3.docx) & [3](http:///doi/suppl/10.2217/pgs-2018-0106/suppl_file/pgs-19-1235-s4.docx)) and they have a very low frequency in the general population. The most frequent of the mutations, T2206M and R2355W, were only present in the ExAC database with a MAF = 0.00003. Based on this consideration, we have set <0.0001 MAF in the ExAC database as the lowest frequency to be considered as a potentially pathogenic mutation. Class 5, pathogenic variants (mutations), have <0.0001 MAF, have been reported more than once in disease cases and have not been identified in unaffected individuals, are nonsense or splicing variants or nonsynonymous variants with REVEL scores >0.5 and absolutely conserved in interspecies amino acid comparisons. Class 4, probably pathogenic variants, have <0.0001 MAF, are nonsense or splicing variants or nonsynonymous variants with REVEL scores >0.5, affect amino acid residues that are absolutely conserved in interspecies amino acid comparisons and have not been previously reported. Class 3, uncertain pathogenicity, have REVEL scores >0.5, <0.0001 MAF, introduce nonconservative amino acid substitutions into residues that are not absolutely conserved in interspecies comparisons, but have only conservative, structurally similar substitutions in other species. Class 2, likely benign variants, have <0.001 MAF and are nonsynonymous variants that may have failed to meet all of the criteria of strong algorithmic predictions, sequence conservation or complete segregation with disease cases. Class 1, benign variants (polymorphisms), include synonymous variants and nonsynonymous variants with >0.001 MAF.

All class 3, 4 or 5 genetic variants identified by exome or genome sequencing were verified by bidirectional Sanger sequencing. Sequencing was performed with the BigDye Terminator v3.1 cycle sequencing kit (ABI) and an ABI 3500 Genetic Analyzer. Oligonucleotide primer sequences used for PCR amplification and sequencing are available upon request.

## Results

In the examination of the *RYR1* and *CACNA1S* genes in 76 genomic DNA samples from subjects with severe SAMS, 66 rare variants (<0.02 MAF in the ExAC database) were found compared with 22 in the control group of 50 statin-tolerant subjects ([Tables 1](#T1) & [2](#T2)). Categorization of these variants into five classes of pathogenicity resulted in one class 5, pathogenic variant, nine class 4, probably pathogenic variants and two class 3, variants of uncertain pathogenicity in the SAMS cases. In the statin-tolerant controls, there was one class 4 and one class 3 variant. Three of the variants in *RYR1* have been previously reported as causative of congenital myopathy (G893S [[39](#B39),[40](#B40)], R2676W [[41](#B41)]) or MHS (E2404K, R2676W [[42](#B42)]). By our classification scheme, R2676W, was ranked class 5, E2404K was ranked class 3 and G893S was categorized as class 2, probably benign.

Case 2236: a 56-year-old Caucasian male with severe SAMS presented with muscle pain, weakness and rhabdomyolysis, plasma CK of 8000 IU/dl and was diagnosed with inflammatory myositis ([Table 3](#T3)). The subject had coronary artery disease and a history of a previous heart attack. He had been taking simvastatin (80 mgs) at the time muscle symptoms began. After stopping statin therapy, his symptoms persisted for 25 months. The subject was found to have a probably pathogenic variant, R2676W, in the *RYR1* gene which has been previously reported in a family study as a novel variant associated with MHS and the presence of multiminicores in muscle biopsies from MHS family members [[41](#B41)]. In that study, the R2676W variant as well as an additional variant, T2787S, on the same allele segregated completely with disease in 19 family members studied by genetic testing. R2676W has also been reported in an unrelated MHS patient along with three additional *RYR1* variants, A1352G, T2787S and P4501L [[42](#B42)]. Arg2676 is absolutely conserved in interspecies sequence comparisons ([Supplementary Figure 2](http:///doi/suppl/10.2217/pgs-2018-0106/suppl_file/pgs-19-1235-s3.docx)) and the R2676W variant is extremely rare, 0.000008 MAF in the ExAC database. Thr2787 is also absolutely conserved but the variant T2787S is too frequent in the general population (0.003 MAF) to be considered pathogenic. The co-occurrence of this variant along with the R2676W in unrelated MHS patients by both Guis *et al*. [[41](#B41)] and Levano *et al*. [[42](#B42)] is suggestive of a possible role of the T2787S influencing the effect of the R2676W variant; however, Thr2787 is located 25 Å from Arg2676 and would not have an obvious effect. The T2787S variant was not detected in the sample containing R2676W in this study.

### Table 3. . Clinical features of individuals with probably pathogenic variants in RYR1 and CACNA1S.

| ID | Sex/Race | Gene | Variant | Sx Onset (y) | Skeletal muscle symptoms |  |  | CK (IU/l) | Statin (mg) | Pers Sx (mos) | Cardiac symptoms |  |  | Other features |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
|   |   |   |   |   | Pain | Weakness | Inflam |   |   |   | CAD | Ht Attack | Fam Hx Dis | Hypothy | Hypertens | Diabetes | Obesity | Other |
| Test Subjects |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 2236 | M/Cauc | RYR1 | R2676W | 56 | Y | Y | Y | 8000 | ATOR (80) | 25 | Y | Y | N | N | N | N | N | Rhabdo |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 2155 | M/Cauc | RYR1 | L221F | 34 | Y | N | N | 7500 | SIM (20) | 0.75 | N | N | N | N | N | N | N |   |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 2048 | M/Cauc | RYR1 | R628C | 77 | N | Y | N | 3000 | ATOR (10) | >3 | N | N | N | N | N | Y | N | Fam Hx MD |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 2189 | M/Cauc | RYR1 | Q3151R | 37 | Y | N | N | 35,000 | ATOR (NA) | 24 | N | N | N | N | N | N | N | Elev liv enz |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 2198 | F/Cauc | RYR1 | P1402S | 60 | Y | N | Y | 1074 | SIM (10) | 0 | N | N | Y | Y | N | N | N |   |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 2196 | M/Trindad | RYR1 | N1644N | 56 | Y | Y | N | 1600 | ROSUV (25) | NA | Y | Y | Y | Y | Y | Y | N |   |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 1394 | M/Cauc | CACNA1S | H1139Q | 60 | Y | N | Y | 1000 | ATOR (20) | 0 | N | N | N | N | Y | Y | Y | Smoker |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 2142 | M/Cauc | CACNA1S | G628S | 63 | Y | Y | N | 78,000 | ROSUV (20) | NA | N | N | N | N | N | N | N | Rhabdo/Myogl |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 2024 | M/Cauc | CACNA1S | R1447Q | 80 | N | N | N | 2200 | SIM (40) | Ongoing | N | N | N | N | Y | N | N | Liv dis |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 2157 | F/AA | CACNA1S | E697del | 61 | Y | Y | N | 23,000 | SIM (40) | 3 | N | N | N | N | N | N | N | Rhabdo |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 2214 | F/First Nation | CACNA1S | R364W | 60 | N | Y | N | 5590 | ATOR (10) | >12 | N | N | Y | N | Y | Y | Y | Smoker |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 2168 | M/Mauritius | RYR1 | E2404K | 46 | Y | Y | N | 1300 | ATOR (NA) | 4+ | Y | Y | N | N | Y | N | N | Liv dis |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| Control subjects |  |  |  | Age entering study |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 1007 | M/Cauc | RYR1 | R1035W | 49 | N | N | N | 67 | LOVA (40) | N | N | N | N | N | N | N | N |   |
|   |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| 1017 | M/Cauc | RYR1 | E438K | 49 | N | N | N | 145 | ATOR (10) | N | N | N | N | N | N | N | N |   |

Table 3 Caption: Table 3 describes 12 subjects with severe statin myopathy and class 3–5 variants in the RYR1 or CACNA1S genes; 2 of 50 statin-tolerant controls also had class 3 or 4 variants. The age of onset of symptoms in the myopathic group ranged from 34 to 80 years. All myopathic subjects had plasma CK >5X ULN. AA: African–American; ATOR: Atorvastatin; CAD: Coronary artery disease; Cauc: Caucasian; CK: Creatine kinase; Dis: Disease; Elev: Elevated; Enz: Enzymes; Exerc Intol: Exercise intolerance; F: Female; Fam: Family; Ht: Heart; Hx: History; Hypertens: Hypertension; Hypothy: Hypothyroidism; Inflam: Inflammation; Liv: Liver; M: Male; MD: Myotonic dystrophy; mos: Months; Myogl: Myoglobinuria; N: No; NA: Not available; Pers: Persistent; Rhabdo: Rhabdomyolysis; ROSUV: Rosuvastatin; SIM: Simvastatin; Sx: Symptoms; y: Years; Y: Yes.

This subject also has a nonsynonymous variant, G178R (REVEL = 0.741, MAF = 0.0002), in the *ISPD* gene which is predicted to affect splicing. *ISPD* variants have been associated with congenital muscular dystrophy-dystroglycanopathy with brain and eye anomalies type A7 [[43](#B43),[44](#B44)].

Case 2155: a 34-year-old Caucasian male with severe statin myopathy presented with muscle pain and a plasma CK of 7500 IU/dl. A history of heart disease was absent in this subject. The subject was taking simvastatin (20 mg) when symptoms began and persisted for 3 weeks post therapy. The subject had a probably pathogenic variant, L221F, in the *RYR1* gene. The L221F variant is located within the N-terminal disease hotspot of *RYR1*, has been previously unreported and is not present in the ExAC database. There are several disease causative nonsynonymous variants located close to Leu221 including T214M [[31](#B31),[45](#B45)], G215E [[46](#B46),[47](#B47)], V218I [[23](#B23),[47](#B47)], R220C [[48](#B48)], M226K [[23](#B23)] and D227V [[49](#B49)]. The Leu221 residue is absolutely conserved in inter-species comparisons and the L221F substitution is predicted to be damaging by REVEL. This is within a region of RyR1 with high-resolution crystal structure information [[47](#B47)]. Modeling based on this structure indicates that Leu221 is located at the interface between N-terminal domains A and B and the L221F substitution is predicted to destabilize this interface ([Supplementary Data](http:///doi/suppl/10.2217/pgs-2018-0106/suppl_file/pgs-19-1235-s1.docx)).

This individual also had rare (MAF <0.001), REVEL-predicted deleterious nonsynonymous variants in three additional myopathy associated genes, *CAPN3* (M666T), *PLEC* (R102L) and *MYH2* (A111V). Variants in *CAPN3* cause limb-girdle muscular dystrophy type 2A [[50](#B50)]. *PLEC* variants cause limb-girdle muscular dystrophy type 2Q [[51](#B51)]. Variants in *MYH2* cause congenital myopathy [[52](#B52),[53](#B53)].

Case 2048: a 77-year-old Caucasian male with statin myopathy presented with muscle weakness only and a plasma CK of 3000 IU/dl. He was taking atorvastatin (10 mgs) at the time symptoms began and reported unexpected weight loss 3 months before stopping statin therapy. Muscle weakness was ongoing post-therapy for at least 3 months. The subject had diabetes and a family history of myotonic dystrophy; however, myotonic dystrophy was ruled out genetically in this individual. He was found to have a probably pathogenic variant, R628C, in the *RYR1* gene. R628C is a previously unreported very rare (MAF = 0.00002) *RYR1* variant, absolutely conserved in inter-species sequence comparisons and strongly predicted to be deleterious by *in-silico* analysis (REVEL = 0.745). Modeling of this region of RyR1 based on structures of Yuchi *et al*. [[54](#B54)] suggests that the R628H variant would lead to the destabilization of FKBP12 binding ([Supplementary Data](http:///doi/suppl/10.2217/pgs-2018-0106/suppl_file/pgs-19-1235-s1.docx)).

This individual also had a nonsynonymous variant in the *CRYAB* gene, G154S, REVEL = 0.507, MAF = 0.0008. Variants in *CRYAB* have been associated with myofibrillar myopathy [[55](#B55),[56](#B56)].

Case 2189: a 37 year-old Caucasian male presented with statin-induced muscle pain and a plasma CK of 35,000 IU/dl at its highest post-exercise. He was taking atorvastatin (dose unknown) at the time of symptom onset and symptoms persisted 24 months post-therapy. He had exercise intolerance and abnormally elevated liver enzymes. For 6 years post therapy he continued to have aches in his muscles primarily in the quadriceps. The subject had a family history of heart disease without a personal history. The subject was found to have a probably pathogenic variant, Q3151R, in the *RYR1* gene and a co-existing Class 2 variant, Y1670N, in the *CACNA1S* gene. Q3151R has not been reported and is not in the ExAC database. Q3151R is located in helical domain 2 [[57](#B57)] and could play a role in destabilizing the helical region. Gln3151 is absolutely conserved in inter-species comparison and is predicted to be deleterious (REVEL = 0.684).

Case 2198: a 60-year-old Caucasian female presented with muscle pain and a plasma CK of 1024 IU/dl 4 days after initiating simvastatin (10 mgs) therapy; symptoms did not persist post-therapy, although the subject self-reported having inflammatory myositis. The subject had a family history of heart disease and a personal history of hypothyroidism. The subject had a probably pathogenic variant, P1402S, in the *RYR1* gene that has not been reported and is not found in the ExAC database. Pro1402 is absolutely conserved in interspecies comparisons and P1402S is predicted to be deleterious (REVEL = 0.563). Pro1402 is located in the SPRY3 domain and is not well resolved in available structural models.

This subject also had a nonsynonymous variant in *MYH7*, R1475C (REVEL = 0.659, MAF = 0.00008). Genetic variants in *MYH7* are primarily causative of hypertrophic cardiomyopathy [[58](#B58)], but have also been associated with Laing distal myopathy and myosin storage myopathy [[59](#B59)].

Case 2196: a 56-year-old male from Trinidad presented with muscle pain, cramps stiffness, muscle swelling and weakness with a plasma CK of 1600 IU/dl while taking rosuvastatin (25 mgs). He had originally taken atorvastatin 15 years earlier that made pre-existing muscle aches worse. His symptoms persisted for 10 years at which time he began rosuvastatin therapy. The subject had many pre-existing conditions including coronary artery disease, a previous heart attack, diabetes, hypertension, hypothyroidism and a family history of heart disease and muscle disease. The subject had a probably pathogenic variant, N1644N, in the *RYR1* gene. This synonymous variant was not present in the ExAC database and is predicted to cause aberrant splicing (dbscSNV_ADA score = 0.9923, dbscSNV_RF score = 0.898). It is categorized as Class 4 since it is a predicted splicing defect but has not been demonstrated experimentally.

This individual also had a nonsynonymous variant in *DYSF*, E1471K (REVEL = 0.651, MAF = 0.0003), that is predicted to affect splicing. *DYSF* variants cause two main types of muscle disease, Myoshi myopathy and limb-girdle muscular dystrophy type 2B [[60](#B60),[61](#B61)].

Case 1394: a 60-year-old Caucasian male presented with muscle pain primarily in his arms and a plasma CK of 1000 IU/dl while taking atorvastatin (20 mgs). The subject self-reported dark colored urine that may be indicative of unconfirmed myoglobinuria. He reported having diabetes, hypertension and a family history of inflammatory muscle disease in his father and brother. A probably pathogenic variant, H1139Q, was found in the *CACNA1S* gene. H1139Q is very rare, not present in the ExAC database and strongly predicted to be deleterious (REVEL = 0.804). This subject also had a nonsynonymous variant, R1064W (REVEL = 0.716, MAF = 0.0002) in *COL6A3*. Collagen VI-related disorders are caused by variants in *COL6A1, COL6A2* and *COL6A3* and include Bethlem myopathy and congenital Ullrich muscular dystrophy [[62](#B62),[63](#B63)].

Case 2142: a 63-year-old Caucasian male presented with pain and weakness predominantly in his arms and calves with rhabdomyolysis and a plasma CK of 78,000 IU/dl occurring within 3 months of taking rosuvastatin (20 mgs). The subject reported no personal or family history of heart disease or muscle disease. A probably pathogenic variant, G628S, was found in the *CACNA1S* gene. G628S is rare (MAF = 8 × 10^-6^), Gly628 is absolutely conserved and G628S is strongly predicted to be deleterious *in silico*.

Case 2024: an 80-year-old Caucasian male presented with an elevated plasma CK of 2200 IU/dl without apparent muscle pain or weakness while initially taking simvastatin (40 mgs) which was changed to pravastatin (40 mg). Since the subject did not have a baseline CK measured before starting statins, it is unclear whether he had pre-existing elevations of CK, however, CK decreased after stopping statins altogether. A probably pathogenic variant, R1447Q, was found in the in the *CACNA1S* gene. R1447Q is rare (MAF = 0.00008), Arg 1447 is absolutely conserved and R1447Q is strongly predicted to be deleterious *in silico*. This subject also had a nonsynonymous variant in *ENO3*, K193N (REVEL = 0.591, MAF = 8 × 10^-6^). Muscle ENO3 deficiency has been associated with exercise intolerance and rhabdomyolysis [[64](#B64)].

Case 2157: a 61-year-old African–American female presented with muscle aches and burning pain as well as muscle weakness and rhabdomyolysis with dark-colored urine and plasma CKs exceeding 23,000 IU/dl while taking simvastatin (40 mgs). In addition, she reportedly had hypertension, liver disease and renal failure. A probably pathogenic variant, E697del, was found in the *CACNA1S* gene with a co-existing Class 2 variant, G893S, in the *RYR1* gene. E697del is extremely rare, not present in the ExAC database and Glu697 is absolutely conserved in interspecies amino acid sequence comparisons. Glu697 is located in the region of CACNA1S linking repeats II and III. These loop regions, particularly residues within the II–III loop, have been shown to be important for excitation–contraction coupling [[65](#B65),[66](#B66)]. While the *RYR1* variant, G893S, has been reported in patients with congenital myopathy [[39](#B39),[40](#B40)], it has also been identified in general population samples [[67](#B67)]. The identification of variants in the general population does not necessarily rule out potential pathogenicity since nonmanifesting patients with known established pathogenic *RYR1* variants have been previously reported. For example, in one study [[68](#B68)], a patient with the known MHS causative *RYR1* variant, R614C, did not have a family history of disease and did not have adverse reactions to previous surgical anesthesia. Gly893 is absolutely conserved across species and the G893S variant is strongly predicted to be damaging by all variant effect prediction algorithms; however, we have categorized G893S as class 2, likely benign, because of the relatively high frequency (0.0006 MAF) of this variant in the ExAC database. Furthermore, although the location of Gly893 is clearly resolved in structural studies [[54](#B54)], the alteration to a serine residue is not obviously predicted to influence other residues. Subject 10 also had a nonsynonymous variant in *AGRN*, G719S (REVEL = 0.56, MAF = 0.0005). *AGRN* variants have been associated with myasthenia syndrome and muscle weakness [[69](#B69)].

Case 2214: a 60-year-old First Nation female presented with muscle weakness in the absence of muscle pain and a plasma CK of 5590 IU/dl while taking atorvastatin (10 mgs). Her symptoms persisted for more than 12 months post-therapy with CKs remaining in the range of 3500–5000 until she was given prednisone with subsequent normalization of CK. She was eventually tested for HMG-CoA reductase autoantibodies and found to be positive with unknown significance. She had hypertension, diabetes, obesity and there was a family history of heart disease. A class 3 variant was found in the *CACNA1S* gene, R364W, that is rare (MAF = 2 × 10^-6^) and predicted to be deleterious (REVEL = 0.554). In interspecies amino acid comparisons only the conservative change to Lys is observed in distant species. Arg364 is in the linker region between CACNA1S repeats I and II.

Case 2168: a 46-year-old Mauritian male developed muscle pain and weakness with a plasma CK of 1300 IU/dl while taking atorvastatin (unknown dose). He had persistent symptoms post therapy for more than 4 months. The subject had a history of hypertension, a previous heart attack, coronary artery disease and liver disease of unspecified etiology; there was no family history of heart disease. A variant (E2404K) was found in the *RYR1* gene that has been previously reported once in a MHS patient [[42](#B42)]. E2404K is located within the central hotspot region of *RYR1*, is rare (<0.00003 MAF) and strongly predicted to be deleterious (REVEL = 0.676). Characterization of lymphoblastoid cells containing DNA from patients with the E2404K variant show increased resting calcium levels and an enhanced response to caffeine [[42](#B42)]. We have categorized E2404K as class 3, of uncertain pathogenicity, since Glu2404 is not completely conserved across species. Levano *et al*. [[42](#B42)] mentioned that it was interesting that this residue was also Lys in the homologous *RYR2* gene, however, in our alignment of human *RYR1* and *RYR2* amino acid sequences, Glu2404 is altered to Thr in *RYR2*. Subject 12 also contained a probably pathogenic nonsynonymous variant in *DYSF*, D467E (REVEL = 0.549), that is not present in the ExAC database.

Control 1007. A 49-year-old Caucasian male had no muscle symptoms while taking lovastatin (40 mgs) and had a plasma CK of 67 IU/dl. The subject was taking lovastatin for 4 months before entering the study. He reported no personal or family history of heart disease or other related risk factors. A probably pathogenic variant, R1035W, was found in the *RYR1* gene. R1035W is rare (MAF = 0.00001), absolutely conserved and strongly predicted to be deleterious (REVEL = 0.791). At 6 weeks and 6 months follow-up, the subject continued to be asymptomatic with no change in his statin medication.

Control 1017. A 49-year-old Caucasian male had no muscle symptoms while taking atorvastatin (10 mgs). There was no personal or family history of heart disease or other cardiovascular risk factors. The subject had a plasma CK of 145 IU/dl. A probably pathogenic variant, E438K, was identified in the *RYR1* gene. This variant is located within the N-terminal disease hotspot region and is both rare and predicted to be deleterious *in silico*. We have classified E438K of uncertain pathogenicity, class 3, because Glu438 is not completely conserved across species, although only the conservative aspartate substitution is found in other species at this residue.

## Discussion

Of 76 subjects with clinically severe SAMS and whose genomic DNA was analyzed by WES or WGS, 12 (16%) had probably pathogenic nonsynonymous variants in either the *RYR1* gene or the *CACNA1S* gene using stringent *in silico* criteria. Since 44 variants in these genes have been functionally and genetically proven to be causative for malignant hyperthermia as determined by the European Malignant Hyperthermia Group ([EMHG.org](http://www.EMHG.org)), two criteria met by all 44 EMHG variants were used as a basis in the assignment of pathogenicity to variants in this study. These included absolute sequence conservation across species and a very low frequency in the general population (MAF <0.0001). In addition, REVEL scoring, which integrates prediction scores from 13 different algorithm tools was used with a cutoff of >0.50 for pathogenicity [[38](#B38)]. It is possible that we have left out variants that are actually pathogenic using these strict criteria as many had REVEL scores >0.50 and/or were very rare but, considering all factors, were predicted as being Class 2 variants (e.g., D943N in the *RYR1* gene with a REVEL score of 0.725 and MAF 0.0009) ([Table 1](#T1)). The statin-tolerant controls carrying probably pathogenic variants should still be monitored for muscle symptoms during statin therapy over time since, by definition, ‘awake state’ phenotypes caused by pathogenic variants in MH-causing genes must be initiated by environmental triggers including but not limited to anesthesia, exertion, heat stroke and statin therapy [[28](#B28),[70](#B70)]. The two positive controls may be at risk for MHS, but may also have other genetic variants or lifestyle habits that make them less susceptible to statin side effects or the dose of statin taken did not exceed a threshold for triggering muscle symptoms in these cases.

Clusters of pathogenic variants or hotspots have been identified in three regions of the *RYR1* gene: in the N-terminal region, the central region and the C-terminal region [[23](#B23),[71](#B71)]. Additional *RYR1* pathogenic variants are increasingly being identified outside of the predominant hotspots [[39](#B39)]. *RYR1* pathogenic variants causative of MHS only or of both MHS and CCD are generally gain of function mutations and result in calcium leakage [[72](#B72),[73](#B73)]. *RYR1* mutations causative of CCD without MHS are loss of function variants and primarily located in the C-terminal region of RyR1 [[74](#B74),[75](#B75)]. *RYR1* mutations causative of MHS with mild pathology or in asymptomatic individuals are mostly located in the N-terminal and central region hotspots. *RYR1* nonsynonymous variants identified here in this group of SAMS cases were primarily located within the N-terminal region of RyR1 consistent with a disease mechanism similar to MHS.

Structural data is available for the overall RyR1 tetramer from cryo-EM experiments [[57](#B57),[76–78](#B76)]. Some regions of RyR1 have been expressed and used to obtain higher resolution crystal structures such as the N-terminal hotspot [[47](#B47),[54](#B54),[79](#B79),[80](#B80)]. Many of the nonsynonymous *RYR1* variants identified in this study were not in regions of RyR1 that have sufficient resolution to look for possible structural or functional effects. However, substitutions L221F, R628C, R2676W and Q3151R were in regions that were highly resolved.

L221F is located within the N-terminal hotspot region. The N-terminal region contains domains A, B and C [[79](#B79)], also referred to as β-trefoil domains 1 and 2 and part of the α-solenoid 1 [[81](#B81)]. Docking of high-resolution crystal structures of the N-terminal region, residues 1-559 of RyR1, with lower resolution cryo-EM structures has shown that the N-terminal region is located at the central rim of RyR1 and form a cytoplasmic vestibule [[79](#B79)]. Interactions between domains within the N-terminal region in the closed state appear to be disrupted by the transition to an open state [[47](#B47),[79](#B79)]. A disease mechanism has been proposed for pathogenic variants in the N-terminal region in which MHS pathogenic variants disrupt interactions between N-terminal domains and act as gain of function mutations that favor the transition to an open state and increased calcium release [[47](#B47),[79](#B79)].

R628C in RyR1 falls within a region predicted by docking studies to contact FKBP12 ([Supplementary Data](http:///doi/suppl/10.2217/pgs-2018-0106/suppl_file/pgs-19-1235-s1.docx)). FKBP12 binding has been shown to promote the closed state of RyRs and reduce the occurrence of subconductance states [[82](#B82)]. Because of this, variants that interfere with FKBP binding have been proposed to cause RyR Ca^++^ leakage and associated disease states [[54](#B54)]. The precise binding site for FKBP12 is not completely understood, perhaps due to the large allosteric changes that occur within the RyR N-terminal regions [[54](#B54)].

Limitations of this study include aspects of its retrospective design such as the acquisition of limited or incomplete clinical details for review and summary for individual subjects. Another limitation is that because of our stringent selection of the probably pathogenic variants, we likely have left behind other important variants among the remaining 54 found with MAF <0.02 in the ExAC database. These should not be discounted in future studies.

All of the subjects with probably pathogenic *RYR1* or *CACNA1S* variants also had co-existing variants of unknown significance within these genes that are too common in the general population to be individually pathogenic. The association of disease with *RYR1* variants of predicted pathogenicity, but with frequencies too high to be considered as pathogenic have not been well studied, particularly when occurring along with other known deleterious variants. Likewise, the predicted pathogenic rare variants in other muscle disease genes (*DYSF, CAPN3, MYH2, MYH7* and others) found in subjects with variants in *RYR1* or *CACNA1S* may be partially, synergistically or completely responsible for the manifesting symptoms, however, we have no way of testing this possibility.

## Conclusion

Probably pathogenic variants in the *RYR1* and *CACNA1S* genes were found to be significant potential contributors to risk for severe SAMS. Previously unreported, predicted pathogenic variants were found in the *RYR1* and *CACNA1S* genes in 16% of the severe SAMS cases examined and may explain their susceptibility to statin therapy.

All of the subjects with probably pathogenic *RYR1* or *CACNA1S* variants also have less rare co-existing variants of unknown significance within these genes as well as probably pathogenic variants within other genes associated with myopathic phenotypes. Clinicians should be mindful of the severity and longevity of muscle symptoms in patients with SAMS as well as plasma CK levels ≥4X ULN and consider in severe cases molecular testing for triggerable myopathies known to cause rhabdomyolysis.

## Future Perspectives

Continued study of variants in *RYR1* and *CACNA1S* at both the clinical and experimental level will contribute to a better understanding of their contributions to disease. Further, the identification and characterization of additional genes and pathogenic variants contributing to SAMS will lead to better therapeutic strategies for patients requiring cholesterol-lowering treatment.

## Supplementary Material

## Acknowledgements

The authors acknowledge the generous contributions of many subjects, their specimens and medical history information from the following: M Tarnopolsky (McMaster University, Hamilton, ON, Canada); M Weisman (Cedars-Sinai Medical Center, Los Angeles, CA, USA); W Peltier (Medical College of Wisconsin, Milwuakee, WI, USA); L Christopher-Stine (Hopkins School of Medicine, Johns Hopkins University, Baltimore, MD, USA); and RL Wortmann (Dartmouth–Hitchcock Medical Center, Lebanon, NH, USA). The authors also acknowledge The People's Pharmacy (Joe & Terry Graeden) for assisting in the recruitment of subjects via an announcement in their syndicated newspaper column.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Rosenbaum D, Dallongeville J, Sabouret P, Bruckert E. Discontinuation of statin therapy due to muscular side effects: a survey in real life. Nutr. Metab. Cardiovasc. Dis. 2013;23:871–875. doi: 10.1016/j.numecd.2012.04.012.  [DOI](https://doi.org/10.1016/j.numecd.2012.04.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22748604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nutr.%20Metab.%20Cardiovasc.%20Dis.&title=Discontinuation%20of%20statin%20therapy%20due%20to%20muscular%20side%20effects:%20a%20survey%20in%20real%20life&author=D%20Rosenbaum&author=J%20Dallongeville&author=P%20Sabouret&author=E%20Bruckert&volume=23&publication_year=2013&pages=871-875&pmid=22748604&doi=10.1016/j.numecd.2012.04.012&)

2. Cohen JD, Brinton EA, Ito MK, Jacobson TA. Understanding statin use in America and gaps in patient education (USAGE): an internet-based survey of 10,138 current and former statin users. J. Clin. Lipidol. 2012;6:208–215. doi: 10.1016/j.jacl.2012.03.003.  [DOI](https://doi.org/10.1016/j.jacl.2012.03.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22658145/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Lipidol.&title=Understanding%20statin%20use%20in%20America%20and%20gaps%20in%20patient%20education%20(USAGE):%20an%20internet-based%20survey%20of%2010,138%20current%20and%20former%20statin%20users&author=JD%20Cohen&author=EA%20Brinton&author=MK%20Ito&author=TA%20Jacobson&volume=6&publication_year=2012&pages=208-215&pmid=22658145&doi=10.1016/j.jacl.2012.03.003&)

3. Shek A, Ferrill MJ. Statin–fibrate combination therapy. Ann. Pharmacother. 2001;35:908–917. doi: 10.1345/aph.10315.  [DOI](https://doi.org/10.1345/aph.10315) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11485144/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Pharmacother.&title=Statin%E2%80%93fibrate%20combination%20therapy&author=A%20Shek&author=MJ%20Ferrill&volume=35&publication_year=2001&pages=908-917&pmid=11485144&doi=10.1345/aph.10315&)

4. SEARCH Collaborative Group. Link E, Parish S, et al. SLCO1B1 variants and statin-induced myopathy – a genomewide study. N. Engl. J. Med. 2008;359:789–799. doi: 10.1056/NEJMoa0801936.  [DOI](https://doi.org/10.1056/NEJMoa0801936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18650507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=SLCO1B1%20variants%20and%20statin-induced%20myopathy%20%E2%80%93%20a%20genomewide%20study&author=E%20Link&author=S%20Parish&volume=359&publication_year=2008&pages=789-799&pmid=18650507&doi=10.1056/NEJMoa0801936&)

5. Oh J, Ban MR, Miskie BA, Pollex RL, Hegele RA. Genetic determinants of statin intolerance. Lipids Health Dis. 2007;6:7. doi: 10.1186/1476-511X-6-7.  [DOI](https://doi.org/10.1186/1476-511X-6-7) | [PMC free article](/articles/PMC1832194/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17376224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lipids%20Health%20Dis.&title=Genetic%20determinants%20of%20statin%20intolerance&author=J%20Oh&author=MR%20Ban&author=BA%20Miskie&author=RL%20Pollex&author=RA%20Hegele&volume=6&publication_year=2007&pages=7&pmid=17376224&doi=10.1186/1476-511X-6-7&)

6. Isackson PJ, Ochs-Balcom HM, Ma C, et al. Association of common variants in the human eyes shut ortholog (EYS) with statin-induced myopathy: evidence for additional functions of EYS. Muscle Nerve. 2011;44:531–538. doi: 10.1002/mus.22115.  [DOI](https://doi.org/10.1002/mus.22115) | [PMC free article](/articles/PMC3175321/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21826682/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Muscle%20Nerve&title=Association%20of%20common%20variants%20in%20the%20human%20eyes%20shut%20ortholog%20(EYS)%20with%20statin-induced%20myopathy:%20evidence%20for%20additional%20functions%20of%20EYS&author=PJ%20Isackson&author=HM%20Ochs-Balcom&author=C%20Ma&volume=44&publication_year=2011&pages=531-538&pmid=21826682&doi=10.1002/mus.22115&)

7. Mangravite LM, Engelhardt BE, Medina MW, et al. A statin-dependent QTL for GATM expression is associated with statin-induced myopathy. Nature. 2013;502:377–380. doi: 10.1038/nature12508.  [DOI](https://doi.org/10.1038/nature12508) | [PMC free article](/articles/PMC3933266/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23995691/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20statin-dependent%20QTL%20for%20GATM%20expression%20is%20associated%20with%20statin-induced%20myopathy&author=LM%20Mangravite&author=BE%20Engelhardt&author=MW%20Medina&volume=502&publication_year=2013&pages=377-380&pmid=23995691&doi=10.1038/nature12508&)

8. Luzum JA, Kitzmiller JP, Isackson PJ, et al. GATM polymorphism associated with the risk for statin-induced myopathy does not replicate in case–control analysis of 715 dyslipidemic individuals. Cell Metab. 2015;21:622–627. doi: 10.1016/j.cmet.2015.03.003.  [DOI](https://doi.org/10.1016/j.cmet.2015.03.003) | [PMC free article](/articles/PMC4394188/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25863251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Metab.&title=GATM%20polymorphism%20associated%20with%20the%20risk%20for%20statin-induced%20myopathy%20does%20not%20replicate%20in%20case%E2%80%93control%20analysis%20of%20715%20dyslipidemic%20individuals&author=JA%20Luzum&author=JP%20Kitzmiller&author=PJ%20Isackson&volume=21&publication_year=2015&pages=622-627&pmid=25863251&doi=10.1016/j.cmet.2015.03.003&)

9. Floyd JS, Bis JC, Brody JA, Heckbert SR, Rice K, Psaty BM. GATM locus does not replicate in rhabdomyolysis study. Nature. 2014;513:E1–E3. doi: 10.1038/nature13629.  [DOI](https://doi.org/10.1038/nature13629) | [PMC free article](/articles/PMC4230441/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25230668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=GATM%20locus%20does%20not%20replicate%20in%20rhabdomyolysis%20study&author=JS%20Floyd&author=JC%20Bis&author=JA%20Brody&author=SR%20Heckbert&author=K%20Rice&volume=513&publication_year=2014&pages=E1-E3&pmid=25230668&doi=10.1038/nature13629&)

10. Santos PC, Gagliardi AC, Miname MH, et al. SLCO1B1 haplotypes are not associated with atorvastatin-induced myalgia in Brazilian patients with familial hypercholesterolemia. Eur. J. Clin. Pharmacol. 2012;68:273–279. doi: 10.1007/s00228-011-1125-1.  [DOI](https://doi.org/10.1007/s00228-011-1125-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21928084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=SLCO1B1%20haplotypes%20are%20not%20associated%20with%20atorvastatin-induced%20myalgia%20in%20Brazilian%20patients%20with%20familial%20hypercholesterolemia&author=PC%20Santos&author=AC%20Gagliardi&author=MH%20Miname&volume=68&publication_year=2012&pages=273-279&pmid=21928084&doi=10.1007/s00228-011-1125-1&)

11. Brunham LR, Lansberg PJ, Zhang L, et al. Differential effect of the rs4149056 variant in SLCO1B1 on myopathy associated with simvastatin and atorvastatin. Pharmacogenomics. 2012;12:233–237. doi: 10.1038/tpj.2010.92.  [DOI](https://doi.org/10.1038/tpj.2010.92) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21243006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Differential%20effect%20of%20the%20rs4149056%20variant%20in%20SLCO1B1%20on%20myopathy%20associated%20with%20simvastatin%20and%20atorvastatin&author=LR%20Brunham&author=PJ%20Lansberg&author=L%20Zhang&volume=12&publication_year=2012&pages=233-237&pmid=21243006&doi=10.1038/tpj.2010.92&)

12. Vladutiu GD, Simmons Z, Isackson PJ, et al. Genetic risk factors associated with lipid-lowering drug-induced myopathies. Muscle Nerve. 2006;34:153–162. doi: 10.1002/mus.20567.  [DOI](https://doi.org/10.1002/mus.20567) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16671104/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Muscle%20Nerve&title=Genetic%20risk%20factors%20associated%20with%20lipid-lowering%20drug-induced%20myopathies&author=GD%20Vladutiu&author=Z%20Simmons&author=PJ%20Isackson&volume=34&publication_year=2006&pages=153-162&pmid=16671104&doi=10.1002/mus.20567&)

13. Hamel Y, Mamoune A, Mauvais FX, et al. Acute rhabdomyolysis and inflammation. J. Inherit. Metab. Dis. 2015;38:621–628. doi: 10.1007/s10545-015-9827-7.  [DOI](https://doi.org/10.1007/s10545-015-9827-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25778939/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Inherit.%20Metab.%20Dis.&title=Acute%20rhabdomyolysis%20and%20inflammation&author=Y%20Hamel&author=A%20Mamoune&author=FX%20Mauvais&volume=38&publication_year=2015&pages=621-628&pmid=25778939&doi=10.1007/s10545-015-9827-7&)

14. Hunter JM, Ahearn ME, Balak CD, et al. Novel pathognic variants and genes for myopathies identified by whole exome sequencing. Mol. Genet. Genomic Med. 2015;3:283–301. doi: 10.1002/mgg3.142.  [DOI](https://doi.org/10.1002/mgg3.142) | [PMC free article](/articles/PMC4521965/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26247046/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Genet.%20Genomic%20Med.&title=Novel%20pathognic%20variants%20and%20genes%20for%20myopathies%20identified%20by%20whole%20exome%20sequencing&author=JM%20Hunter&author=ME%20Ahearn&author=CD%20Balak&volume=3&publication_year=2015&pages=283-301&pmid=26247046&doi=10.1002/mgg3.142&)

15. Efthymiou S, Manole A, Houlden H. Next-generation sequencing in neuromuscular diseases. Curr. Opin. Neurol. 2016;29:527–536. doi: 10.1097/WCO.0000000000000374.  [DOI](https://doi.org/10.1097/WCO.0000000000000374) | [PMC free article](/articles/PMC5082606/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27588584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Neurol.&title=Next-generation%20sequencing%20in%20neuromuscular%20diseases&author=S%20Efthymiou&author=A%20Manole&author=H%20Houlden&volume=29&publication_year=2016&pages=527-536&pmid=27588584&doi=10.1097/WCO.0000000000000374&)

16. Abath Neto O, Tassy O, Biancalana V, Zanoteli E, Pourquié O, Laporte J. Integrative data mining highlights candidate genes for monogenic myopathies. PLoS ONE. 2014;9:e110888. doi: 10.1371/journal.pone.0110888.  [DOI](https://doi.org/10.1371/journal.pone.0110888) | [PMC free article](/articles/PMC4213015/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25353622/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Integrative%20data%20mining%20highlights%20candidate%20genes%20for%20monogenic%20myopathies&author=O%20Abath%20Neto&author=O%20Tassy&author=V%20Biancalana&author=E%20Zanoteli&author=O%20Pourqui%C3%A9&volume=9&publication_year=2014&pages=e110888&pmid=25353622&doi=10.1371/journal.pone.0110888&)

17. Knoblauch M, Dagnino-Acosta A, Hamilton SL. Mice with RyR1 mutation (Y524S) undergo hypermetabolic response to simvastatin. Skelet. Muscle. 2013;3(1):22. doi: 10.1186/2044-5040-3-22.  [DOI](https://doi.org/10.1186/2044-5040-3-22) | [PMC free article](/articles/PMC3846650/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24004537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Skelet.%20Muscle&title=Mice%20with%20RyR1%20mutation%20(Y524S)%20undergo%20hypermetabolic%20response%20to%20simvastatin&author=M%20Knoblauch&author=A%20Dagnino-Acosta&author=SL%20Hamilton&volume=3&issue=1&publication_year=2013&pages=22&pmid=24004537&doi=10.1186/2044-5040-3-22&)

18. Carpenter D, Ringrose C, Leo V, et al. The role of CACNA1S in predisposition to malignant hyperthermia. BMC Med. Genet. 2009;10:104. doi: 10.1186/1471-2350-10-104.  [DOI](https://doi.org/10.1186/1471-2350-10-104) | [PMC free article](/articles/PMC2770053/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19825159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med.%20Genet.&title=The%20role%20of%20CACNA1S%20in%20predisposition%20to%20malignant%20hyperthermia&author=D%20Carpenter&author=C%20Ringrose&author=V%20Leo&volume=10&publication_year=2009&pages=104&pmid=19825159&doi=10.1186/1471-2350-10-104&)

19. Horstick EJ, Linsley JW, Dowling JJ, et al. Stac3 is a component of the excitation–contraction coupling machinery and mutated in Native American myopathy. Nat. Commun. 2013;4:1952. doi: 10.1038/ncomms2952.  [DOI](https://doi.org/10.1038/ncomms2952) | [PMC free article](/articles/PMC4056023/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23736855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Stac3%20is%20a%20component%20of%20the%20excitation%E2%80%93contraction%20coupling%20machinery%20and%20mutated%20in%20Native%20American%20myopathy&author=EJ%20Horstick&author=JW%20Linsley&author=JJ%20Dowling&volume=4&publication_year=2013&pages=1952&pmid=23736855&doi=10.1038/ncomms2952&)

20. Voermans NC, Snoeck M, Jungbluth H. RYR-1 related rhabdomyolysis: a common but probably underdiagnosed manifestation of skeletal muscle ryanodine receptor dysfunction. Rev. Neurol. 2016;172:546–558. doi: 10.1016/j.neurol.2016.07.018.  [DOI](https://doi.org/10.1016/j.neurol.2016.07.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27663056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rev.%20Neurol.&title=RYR-1%20related%20rhabdomyolysis:%20a%20common%20but%20probably%20underdiagnosed%20manifestation%20of%20skeletal%20muscle%20ryanodine%20receptor%20dysfunction&author=NC%20Voermans&author=M%20Snoeck&author=H%20Jungbluth&volume=172&publication_year=2016&pages=546-558&pmid=27663056&doi=10.1016/j.neurol.2016.07.018&)

21. Rosenberg H, Davis M, James D, Pollock N, Stowell K. Malignant hyperthermia. Orphanet. J. Rare Dis. 2007;2:21. doi: 10.1186/1750-1172-2-21.  [DOI](https://doi.org/10.1186/1750-1172-2-21) | [PMC free article](/articles/PMC1867813/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17456235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Orphanet.%20J.%20Rare%20Dis.&title=Malignant%20hyperthermia&author=H%20Rosenberg&author=M%20Davis&author=D%20James&author=N%20Pollock&author=K%20Stowell&volume=2&publication_year=2007&pages=21&pmid=17456235&doi=10.1186/1750-1172-2-21&)

22. Monnier N, Krivosic-Horber R, Payen JF, et al. Presence of two different genetic traits in malignant hyperthermia families: implication for genetic analysis, diagnosis, and incidence of malignant hyperthermia susceptibility. Anesthesiology. 2002;97:1067–1074. doi: 10.1097/00000542-200211000-00007.  [DOI](https://doi.org/10.1097/00000542-200211000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12411788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Presence%20of%20two%20different%20genetic%20traits%20in%20malignant%20hyperthermia%20families:%20implication%20for%20genetic%20analysis,%20diagnosis,%20and%20incidence%20of%20malignant%20hyperthermia%20susceptibility&author=N%20Monnier&author=R%20Krivosic-Horber&author=JF%20Payen&volume=97&publication_year=2002&pages=1067-1074&pmid=12411788&doi=10.1097/00000542-200211000-00007&)

23. Robinson R, Carpenter D, Shaw MA, Halsall J, Hopkins P. Mutations in RYR1 in malignant hyperthermia and central core disease. Human Mutat. 2006;27:977–989. doi: 10.1002/humu.20356.  [DOI](https://doi.org/10.1002/humu.20356) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16917943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Human%20Mutat.&title=Mutations%20in%20RYR1%20in%20malignant%20hyperthermia%20and%20central%20core%20disease&author=R%20Robinson&author=D%20Carpenter&author=MA%20Shaw&author=J%20Halsall&author=P%20Hopkins&volume=27&publication_year=2006&pages=977-989&pmid=16917943&doi=10.1002/humu.20356&)

24. Amburgey K, Bailey A, Hwang JH, et al. Genotype–phenotype correlations in recessive RYR1-related myopathies. Orphanet. J. Rare Dis. 2013;8:117. doi: 10.1186/1750-1172-8-117.  [DOI](https://doi.org/10.1186/1750-1172-8-117) | [PMC free article](/articles/PMC3751094/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23919265/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Orphanet.%20J.%20Rare%20Dis.&title=Genotype%E2%80%93phenotype%20correlations%20in%20recessive%20RYR1-related%20myopathies&author=K%20Amburgey&author=A%20Bailey&author=JH%20Hwang&volume=8&publication_year=2013&pages=117&pmid=23919265&doi=10.1186/1750-1172-8-117&)

25. Attali R, Aharoni S, Treves S, et al. Variable myopathic presentation in a single family with novel skeletal RYR1 mutation. PLoS ONE. 2013;8:e69296. doi: 10.1371/journal.pone.0069296.  [DOI](https://doi.org/10.1371/journal.pone.0069296) | [PMC free article](/articles/PMC3722152/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23894444/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Variable%20myopathic%20presentation%20in%20a%20single%20family%20with%20novel%20skeletal%20RYR1%20mutation&author=R%20Attali&author=S%20Aharoni&author=S%20Treves&volume=8&publication_year=2013&pages=e69296&pmid=23894444&doi=10.1371/journal.pone.0069296&)

26. Duarte ST, Oliveira J, Santos R, et al. Dominant and recessive RYR1 mutations in adults with core lesions and mild muscle symptoms. Muscle Nerve. 2011;44:102–108. doi: 10.1002/mus.22009.  [DOI](https://doi.org/10.1002/mus.22009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21674524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Muscle%20Nerve&title=Dominant%20and%20recessive%20RYR1%20mutations%20in%20adults%20with%20core%20lesions%20and%20mild%20muscle%20symptoms&author=ST%20Duarte&author=J%20Oliveira&author=R%20Santos&volume=44&publication_year=2011&pages=102-108&pmid=21674524&doi=10.1002/mus.22009&)

27. Dlamini N, Voermans NC, Lillis S, et al. Mutations in RYR1 are a common cause of exertional myalgia and rhabdomyolysis. Neuromuscul. Disord. 2013;23:540–548. doi: 10.1016/j.nmd.2013.03.008.  [DOI](https://doi.org/10.1016/j.nmd.2013.03.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23628358/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuromuscul.%20Disord.&title=Mutations%20in%20RYR1%20are%20a%20common%20cause%20of%20exertional%20myalgia%20and%20rhabdomyolysis&author=N%20Dlamini&author=NC%20Voermans&author=S%20Lillis&volume=23&publication_year=2013&pages=540-548&pmid=23628358&doi=10.1016/j.nmd.2013.03.008&)

28. Vladutiu GD, Isackson PJ, Kaufman K. Genetic risk for malignant hyperthermia in non-anesthesia-induced myopathies. Mol. Genet. Metab. 2011;104:167–173. doi: 10.1016/j.ymgme.2011.07.001.  [DOI](https://doi.org/10.1016/j.ymgme.2011.07.001) | [PMC free article](/articles/PMC3171598/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21795085/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Genet.%20Metab.&title=Genetic%20risk%20for%20malignant%20hyperthermia%20in%20non-anesthesia-induced%20myopathies&author=GD%20Vladutiu&author=PJ%20Isackson&author=K%20Kaufman&volume=104&publication_year=2011&pages=167-173&pmid=21795085&doi=10.1016/j.ymgme.2011.07.001&)

29. Tammaro A, Di Martino A, Bracco A, et al. Novel missense mutations and unexpected multiple changes of RYR1 gene in 75 malignant hyperthermia families. Clin. Genet. 2011;79:438–447. doi: 10.1111/j.1399-0004.2010.01493.x.  [DOI](https://doi.org/10.1111/j.1399-0004.2010.01493.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20681998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Genet.&title=Novel%20missense%20mutations%20and%20unexpected%20multiple%20changes%20of%20RYR1%20gene%20in%2075%20malignant%20hyperthermia%20families&author=A%20Tammaro&author=A%20Di%20Martino&author=A%20Bracco&volume=79&publication_year=2011&pages=438-447&pmid=20681998&doi=10.1111/j.1399-0004.2010.01493.x&)

30. Roux-Buisson N, Monnier N, Sagui E, et al. Identification of variants of the ryanodine receptor type 1 in patients with exertional heat stroke and positive response to the malignant hyperthermia in vitro contracture test. Br. J. Anaesth. 2016;116:566–568. doi: 10.1093/bja/aew047.  [DOI](https://doi.org/10.1093/bja/aew047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26994242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Anaesth.&title=Identification%20of%20variants%20of%20the%20ryanodine%20receptor%20type%201%20in%20patients%20with%20exertional%20heat%20stroke%20and%20positive%20response%20to%20the%20malignant%20hyperthermia%20in%20vitro%20contracture%20test&author=N%20Roux-Buisson&author=N%20Monnier&author=E%20Sagui&volume=116&publication_year=2016&pages=566-568&pmid=26994242&doi=10.1093/bja/aew047&)

31. Fiszer D, Shaw MA, Fisher NA, et al. Next-generation sequencing of RYR1 and CACNA1S in malignant hyperthermia and exertional heat illness. Anesthesiology. 2015;122:1033–1046. doi: 10.1097/ALN.0000000000000610.  [DOI](https://doi.org/10.1097/ALN.0000000000000610) | [PMC free article](/articles/PMC4472733/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25658027/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Next-generation%20sequencing%20of%20RYR1%20and%20CACNA1S%20in%20malignant%20hyperthermia%20and%20exertional%20heat%20illness&author=D%20Fiszer&author=MA%20Shaw&author=NA%20Fisher&volume=122&publication_year=2015&pages=1033-1046&pmid=25658027&doi=10.1097/ALN.0000000000000610&)

32. Stroes ES, Thompson P, Corsini A, et al. Statin-associated muscle symptoms: impact on statin therapy. Eur. Heart J. 2015;36(17):1012–1022. doi: 10.1093/eurheartj/ehv043.  [DOI](https://doi.org/10.1093/eurheartj/ehv043) | [PMC free article](/articles/PMC4416140/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25694464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Heart%20J.&title=Statin-associated%20muscle%20symptoms:%20impact%20on%20statin%20therapy&author=ES%20Stroes&author=P%20Thompson&author=A%20Corsini&volume=36&issue=17&publication_year=2015&pages=1012-1022&pmid=25694464&doi=10.1093/eurheartj/ehv043&)

33. Kim JH, Jarvik GP, Browning BL, et al. Exome sequencing reveals novel rare variants in the ryanodine receptor and calcium channel genes in malignant hyperthermia families. Anesthesiology. 2013;119:1054–1065. doi: 10.1097/ALN.0b013e3182a8a998.  [DOI](https://doi.org/10.1097/ALN.0b013e3182a8a998) | [PMC free article](/articles/PMC4115638/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24013571/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Exome%20sequencing%20reveals%20novel%20rare%20variants%20in%20the%20ryanodine%20receptor%20and%20calcium%20channel%20genes%20in%20malignant%20hyperthermia%20families&author=JH%20Kim&author=GP%20Jarvik&author=BL%20Browning&volume=119&publication_year=2013&pages=1054-1065&pmid=24013571&doi=10.1097/ALN.0b013e3182a8a998&)

34. Wang K, Li M, Hakonarson H. ANNOVAR: functional annotation of genetic variants from next-generation sequencing data. Nucleic Acids Res. 2010;38:e164. doi: 10.1093/nar/gkq603.  [DOI](https://doi.org/10.1093/nar/gkq603) | [PMC free article](/articles/PMC2938201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20601685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=ANNOVAR:%20functional%20annotation%20of%20genetic%20variants%20from%20next-generation%20sequencing%20data&author=K%20Wang&author=M%20Li&author=H%20Hakonarson&volume=38&publication_year=2010&pages=e164&pmid=20601685&doi=10.1093/nar/gkq603&)

35. Schiemann AH, Stowell KM. Comparison of pathogenicity prediction tools on missense variants in RYR1 and CACNA1S associated with malignant hyperthermia. Br. J. Anaesth. 2016;117:124–128. doi: 10.1093/bja/aew065.  [DOI](https://doi.org/10.1093/bja/aew065) | [PMC free article](/articles/PMC4913391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27147545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Anaesth.&title=Comparison%20of%20pathogenicity%20prediction%20tools%20on%20missense%20variants%20in%20RYR1%20and%20CACNA1S%20associated%20with%20malignant%20hyperthermia&author=AH%20Schiemann&author=KM%20Stowell&volume=117&publication_year=2016&pages=124-128&pmid=27147545&doi=10.1093/bja/aew065&)

36. Lopes MC, Joyce C, Ritchie GR, et al. A combined functional annotation score for non-synonymous variants. Hum. Hered. 2012;73(1):47–51. doi: 10.1159/000334984.  [DOI](https://doi.org/10.1159/000334984) | [PMC free article](/articles/PMC3390741/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22261837/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Hered.&title=A%20combined%20functional%20annotation%20score%20for%20non-synonymous%20variants&author=MC%20Lopes&author=C%20Joyce&author=GR%20Ritchie&volume=73&issue=1&publication_year=2012&pages=47-51&pmid=22261837&doi=10.1159/000334984&)

37. Gonzalez-Perez A, Lopez-Bigas N. Improving the assessment of the outcome of nonsynonymous SNVs with a consensus deleteriousness score, Condel. Am. J. Hum. Genet. 2011;88(4):440–449. doi: 10.1016/j.ajhg.2011.03.004.  [DOI](https://doi.org/10.1016/j.ajhg.2011.03.004) | [PMC free article](/articles/PMC3071923/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21457909/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Improving%20the%20assessment%20of%20the%20outcome%20of%20nonsynonymous%20SNVs%20with%20a%20consensus%20deleteriousness%20score,%20Condel&author=A%20Gonzalez-Perez&author=N%20Lopez-Bigas&volume=88&issue=4&publication_year=2011&pages=440-449&pmid=21457909&doi=10.1016/j.ajhg.2011.03.004&)

38. Ioannidis NM, Rothstein JH, Pejaver V, et al. REVEL: an ensemble method for predicting the pathogenicity of rare missense variants. Am. J. Hum. Genet. 2016;99:877–885. doi: 10.1016/j.ajhg.2016.08.016.  [DOI](https://doi.org/10.1016/j.ajhg.2016.08.016) | [PMC free article](/articles/PMC5065685/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27666373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=REVEL:%20an%20ensemble%20method%20for%20predicting%20the%20pathogenicity%20of%20rare%20missense%20variants&author=NM%20Ioannidis&author=JH%20Rothstein&author=V%20Pejaver&volume=99&publication_year=2016&pages=877-885&pmid=27666373&doi=10.1016/j.ajhg.2016.08.016&)

39. Klein A, Lillis S, Munteanu I, Scoto M, et al. Clinical and genetic findings in a large cohort of patients with ryanodine receptor 1 gene-associated myopathies. Hum. Mutat. 2012;33:981–988. doi: 10.1002/humu.22056.  [DOI](https://doi.org/10.1002/humu.22056) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22473935/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mutat.&title=Clinical%20and%20genetic%20findings%20in%20a%20large%20cohort%20of%20patients%20with%20ryanodine%20receptor%201%20gene-associated%20myopathies&author=A%20Klein&author=S%20Lillis&author=I%20Munteanu&author=M%20Scoto&volume=33&publication_year=2012&pages=981-988&pmid=22473935&doi=10.1002/humu.22056&)

40. Maggi L, Scoto M, Cirak S, et al. Congenital myopathies – clinical features and frequency of individual subtypes diagnosed over a 5-year period in the United Kingdom. Neuromuscul. Disord. 2013;23:195–205. doi: 10.1016/j.nmd.2013.01.004.  [DOI](https://doi.org/10.1016/j.nmd.2013.01.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23394784/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuromuscul.%20Disord.&title=Congenital%20myopathies%20%E2%80%93%20clinical%20features%20and%20frequency%20of%20individual%20subtypes%20diagnosed%20over%20a%205-year%20period%20in%20the%20United%20Kingdom&author=L%20Maggi&author=M%20Scoto&author=S%20Cirak&volume=23&publication_year=2013&pages=195-205&pmid=23394784&doi=10.1016/j.nmd.2013.01.004&)

41. Guis S, Figarella-Branger D, Monnier N, et al. Multiminicore disease in a family susceptible to malignant hyperthermia: histology, in vitro contracture tests, and genetic characterization. Arch. Neurol. 2004;61:106–113. doi: 10.1001/archneur.61.1.106.  [DOI](https://doi.org/10.1001/archneur.61.1.106) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14732627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Neurol.&title=Multiminicore%20disease%20in%20a%20family%20susceptible%20to%20malignant%20hyperthermia:%20histology,%20in%20vitro%20contracture%20tests,%20and%20genetic%20characterization&author=S%20Guis&author=D%20Figarella-Branger&author=N%20Monnier&volume=61&publication_year=2004&pages=106-113&pmid=14732627&doi=10.1001/archneur.61.1.106&)

42. Levano S, Vukcevic M, Singer M. Increasing the number of diagnostic mutations in malignant hyperthermia. Hum. Mutat. 2009;30:590–598. doi: 10.1002/humu.20878.  [DOI](https://doi.org/10.1002/humu.20878) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19191329/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mutat.&title=Increasing%20the%20number%20of%20diagnostic%20mutations%20in%20malignant%20hyperthermia&author=S%20Levano&author=M%20Vukcevic&author=M%20Singer&volume=30&publication_year=2009&pages=590-598&pmid=19191329&doi=10.1002/humu.20878&)

43. Roscioli T, Kamsteeg E-J, Buysse K, et al. Mutations in ISPD cause Walker–Warburg syndrome and defective glycosylation of alpha-dystroglycan. Nature Genet. 2012;44(581–585) doi: 10.1038/ng.2253.  [DOI](https://doi.org/10.1038/ng.2253) | [PMC free article](/articles/PMC3378661/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22522421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genet.&title=Mutations%20in%20ISPD%20cause%20Walker%E2%80%93Warburg%20syndrome%20and%20defective%20glycosylation%20of%20alpha-dystroglycan&author=T%20Roscioli&author=E-J%20Kamsteeg&author=K%20Buysse&volume=44&issue=581%E2%80%93585&publication_year=2012&pmid=22522421&doi=10.1038/ng.2253&)

44. Willer T, Lee H, Lommel M, et al. ISPD loss-of-function mutations disrupt dystroglycan O-mannosylation and cause Walker–Warburg syndrome. Nat. Genet. 2012;44:575–580. doi: 10.1038/ng.2252.  [DOI](https://doi.org/10.1038/ng.2252) | [PMC free article](/articles/PMC3371168/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22522420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=ISPD%20loss-of-function%20mutations%20disrupt%20dystroglycan%20O-mannosylation%20and%20cause%20Walker%E2%80%93Warburg%20syndrome&author=T%20Willer&author=H%20Lee&author=M%20Lommel&volume=44&publication_year=2012&pages=575-580&pmid=22522420&doi=10.1038/ng.2252&)

45. Stephens J, Schiemann AH, Roesl C, et al. Functional analysis of RYR1 variants linked to malignant hyperthermia. Temperature (Austin) 2016;3(2):328–339. doi: 10.1080/23328940.2016.1153360.  [DOI](https://doi.org/10.1080/23328940.2016.1153360) | [PMC free article](/articles/PMC4964997/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27857962/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Temperature%20(Austin)&title=Functional%20analysis%20of%20RYR1%20variants%20linked%20to%20malignant%20hyperthermia&author=J%20Stephens&author=AH%20Schiemann&author=C%20Roesl&volume=3&issue=2&publication_year=2016&pages=328-339&pmid=27857962&doi=10.1080/23328940.2016.1153360&)

46. Romero NB, Monnier N, Viollet L, et al. Dominant and recessive central core disease associated RYR1 mutations and fetal akinesia. Brain. 2003;126:2341–2349. doi: 10.1093/brain/awg244.  [DOI](https://doi.org/10.1093/brain/awg244) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12937085/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain&title=Dominant%20and%20recessive%20central%20core%20disease%20associated%20RYR1%20mutations%20and%20fetal%20akinesia&author=NB%20Romero&author=N%20Monnier&author=L%20Viollet&volume=126&publication_year=2003&pages=2341-2349&pmid=12937085&doi=10.1093/brain/awg244&)

47. Kimlicka L, Lau K, Tung CC, Van Petegem F. Disease mutations in the ryanodine receptor N-terminal region couple to a mobile intersubumit interface. Nature Commun. 2013;4:1506. doi: 10.1038/ncomms2501.  [DOI](https://doi.org/10.1038/ncomms2501) | [PMC free article](/articles/PMC3586727/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23422674/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Commun.&title=Disease%20mutations%20in%20the%20ryanodine%20receptor%20N-terminal%20region%20couple%20to%20a%20mobile%20intersubumit%20interface&author=L%20Kimlicka&author=K%20Lau&author=CC%20Tung&author=F%20Van%20Petegem&volume=4&publication_year=2013&pages=1506&pmid=23422674&doi=10.1038/ncomms2501&)

48. Wang Y, Peng W, Guo HY, et al. Next-generation sequencing-based molecular diagnosis of neonatal hypotonia in Chinese population. Sci. Rep. 2016;6:29088. doi: 10.1038/srep29088.  [DOI](https://doi.org/10.1038/srep29088) | [PMC free article](/articles/PMC4926250/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27353517/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Next-generation%20sequencing-based%20molecular%20diagnosis%20of%20neonatal%20hypotonia%20in%20Chinese%20population&author=Y%20Wang&author=W%20Peng&author=HY%20Guo&volume=6&publication_year=2016&pages=29088&pmid=27353517&doi=10.1038/srep29088&)

49. Monnier N, Kozak-Ribbens G, Krivosic-Horber R, et al. Correlations between genotype and pharmacological, histological, functional, and clinical phenotypes in malignant hyperthermia susceptibility. Hum. Mutat. 2005;26(5):413–425. doi: 10.1002/humu.20231.  [DOI](https://doi.org/10.1002/humu.20231) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16163667/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mutat.&title=Correlations%20between%20genotype%20and%20pharmacological,%20histological,%20functional,%20and%20clinical%20phenotypes%20in%20malignant%20hyperthermia%20susceptibility&author=N%20Monnier&author=G%20Kozak-Ribbens&author=R%20Krivosic-Horber&volume=26&issue=5&publication_year=2005&pages=413-425&pmid=16163667&doi=10.1002/humu.20231&)

50. Beckmann JS, Spencer M. Calpain 3, the “gatekeeper” of proper sarcomere assembly, turnover and maintenance. Neuromuscul. Disord. 2008;18(12):913–921. doi: 10.1016/j.nmd.2008.08.005.  [DOI](https://doi.org/10.1016/j.nmd.2008.08.005) | [PMC free article](/articles/PMC2614824/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18974005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuromuscul.%20Disord.&title=Calpain%203,%20the%20%E2%80%9Cgatekeeper%E2%80%9D%20of%20proper%20sarcomere%20assembly,%20turnover%20and%20maintenance&author=JS%20Beckmann&author=M%20Spencer&volume=18&issue=12&publication_year=2008&pages=913-921&pmid=18974005&doi=10.1016/j.nmd.2008.08.005&)

51. Gundesli H, Talim B, Korkusuz P, et al. Mutation in exon 1f of PLEC, leading to disruption of plectin isoform 1f, causes autosomal-recessive limb-girdle muscular dystrophy. Am. J. Hum. Genet. 2010;87:834–841. doi: 10.1016/j.ajhg.2010.10.017.  [DOI](https://doi.org/10.1016/j.ajhg.2010.10.017) | [PMC free article](/articles/PMC2997373/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21109228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Mutation%20in%20exon%201f%20of%20PLEC,%20leading%20to%20disruption%20of%20plectin%20isoform%201f,%20causes%20autosomal-recessive%20limb-girdle%20muscular%20dystrophy&author=H%20Gundesli&author=B%20Talim&author=P%20Korkusuz&volume=87&publication_year=2010&pages=834-841&pmid=21109228&doi=10.1016/j.ajhg.2010.10.017&)

52. Hernández-Laín A, Esteban-Pérez J, Montenegro DC, Domínguez-González C. Myosin myopathy with external ophthalmoplegia associated with a novel homozygous mutation in MYH2. Muscle Nerve. 2017;55(2):E8–E10. doi: 10.1002/mus.25365.  [DOI](https://doi.org/10.1002/mus.25365) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27490141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Muscle%20Nerve&title=Myosin%20myopathy%20with%20external%20ophthalmoplegia%20associated%20with%20a%20novel%20homozygous%20mutation%20in%20MYH2&author=A%20Hern%C3%A1ndez-La%C3%ADn&author=J%20Esteban-P%C3%A9rez&author=DC%20Montenegro&author=C%20Dom%C3%ADnguez-Gonz%C3%A1lez&volume=55&issue=2&publication_year=2017&pages=E8-E10&pmid=27490141&doi=10.1002/mus.25365&)

53. Tajsharghi HHS, Lindberg C, Lossos A, et al. Recessive myosin myopathy with external ophthalmoplegia associated with MYH2 mutations. Eur. J. Hum. Genet. 2014;22(6):801–808. doi: 10.1038/ejhg.2013.250.  [DOI](https://doi.org/10.1038/ejhg.2013.250) | [PMC free article](/articles/PMC4023224/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24193343/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Hum.%20Genet.&title=Recessive%20myosin%20myopathy%20with%20external%20ophthalmoplegia%20associated%20with%20MYH2%20mutations&author=HHS%20Tajsharghi&author=C%20Lindberg&author=A%20Lossos&volume=22&issue=6&publication_year=2014&pages=801-808&pmid=24193343&doi=10.1038/ejhg.2013.250&)

54. Yuchi Z, Yuen SM, Lau K, et al. Crystal structures of ryanodine receptor SPRY1 and tandom-repeat domains reveal a critical FKBP12 binding determinant. Nat. Commun. 2015;6:7947. doi: 10.1038/ncomms8947.  [DOI](https://doi.org/10.1038/ncomms8947) | [PMC free article](/articles/PMC4530471/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26245150/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Crystal%20structures%20of%20ryanodine%20receptor%20SPRY1%20and%20tandom-repeat%20domains%20reveal%20a%20critical%20FKBP12%20binding%20determinant&author=Z%20Yuchi&author=SM%20Yuen&author=K%20Lau&volume=6&publication_year=2015&pages=7947&pmid=26245150&doi=10.1038/ncomms8947&)

55. Fichna JP, Potulska-Chromik A, Miszta P, et al. A novel dominant D109A CRYAB muation in a family with myofibrillar myopathy affects αB-crystallin structure. BBA Cln. 2017;7:1–7. doi: 10.1016/j.bbacli.2016.11.004.  [DOI](https://doi.org/10.1016/j.bbacli.2016.11.004) | [PMC free article](/articles/PMC5124346/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27904835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BBA%20Cln.&title=A%20novel%20dominant%20D109A%20CRYAB%20muation%20in%20a%20family%20with%20myofibrillar%20myopathy%20affects%20%CE%B1B-crystallin%20structure&author=JP%20Fichna&author=A%20Potulska-Chromik&author=P%20Miszta&volume=7&publication_year=2017&pages=1-7&pmid=27904835&doi=10.1016/j.bbacli.2016.11.004&)

56. Vincent AE, Grady JP, Rocha MC, et al. Mitochondrial dysfunction in myofibrillar myopathy. Neuromuscul. Disord. 2016;26(10):691–701. doi: 10.1016/j.nmd.2016.08.004.  [DOI](https://doi.org/10.1016/j.nmd.2016.08.004) | [PMC free article](/articles/PMC5066370/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27618136/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuromuscul.%20Disord.&title=Mitochondrial%20dysfunction%20in%20myofibrillar%20myopathy&author=AE%20Vincent&author=JP%20Grady&author=MC%20Rocha&volume=26&issue=10&publication_year=2016&pages=691-701&pmid=27618136&doi=10.1016/j.nmd.2016.08.004&)

57. Yan Z, Bai XC, Yan C, et al. Structure of the rabbit ryanodine receptor RyR1 at near-atomic resolution. Nature. 2015;517:50–55. doi: 10.1038/nature14063.  [DOI](https://doi.org/10.1038/nature14063) | [PMC free article](/articles/PMC4338550/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25517095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Structure%20of%20the%20rabbit%20ryanodine%20receptor%20RyR1%20at%20near-atomic%20resolution&author=Z%20Yan&author=XC%20Bai&author=C%20Yan&volume=517&publication_year=2015&pages=50-55&pmid=25517095&doi=10.1038/nature14063&)

58. Walsh R, Rutland C, Thomas R, Loughna S. Cardiomyopathy: a systematic review of disease-causing mutations in myosin heavy chain 7 and their phenotypic manifestations. Cardiology. 2010;115(1):49–60. doi: 10.1159/000252808.  [DOI](https://doi.org/10.1159/000252808) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19864899/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiology&title=Cardiomyopathy:%20a%20systematic%20review%20of%20disease-causing%20mutations%20in%20myosin%20heavy%20chain%207%20and%20their%20phenotypic%20manifestations&author=R%20Walsh&author=C%20Rutland&author=R%20Thomas&author=S%20Loughna&volume=115&issue=1&publication_year=2010&pages=49-60&pmid=19864899&doi=10.1159/000252808&)

59. Oldfors A. Hereditary myosin myopathies. Neuromuscul. Disord. 2007;17(5):355–367. doi: 10.1016/j.nmd.2007.02.008.  [DOI](https://doi.org/10.1016/j.nmd.2007.02.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17434305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuromuscul.%20Disord.&title=Hereditary%20myosin%20myopathies&author=A%20Oldfors&volume=17&issue=5&publication_year=2007&pages=355-367&pmid=17434305&doi=10.1016/j.nmd.2007.02.008&)

60. Patel NJ, Van Dyke KW, Espinoza LR. Limb-girdle muscular dystrophy 2B and Myoshi presentations in dysferlinopathy. Am. J. Med. Sci. 2017;353(5):484–491. doi: 10.1016/j.amjms.2016.05.024.  [DOI](https://doi.org/10.1016/j.amjms.2016.05.024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28502335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Med.%20Sci.&title=Limb-girdle%20muscular%20dystrophy%202B%20and%20Myoshi%20presentations%20in%20dysferlinopathy&author=NJ%20Patel&author=KW%20Van%20Dyke&author=LR%20Espinoza&volume=353&issue=5&publication_year=2017&pages=484-491&pmid=28502335&doi=10.1016/j.amjms.2016.05.024&)

61. Amato AA, Brown RH., Jr Dysferlinopathies. Handb. Clin. Neurol. 2011;101:111–118. doi: 10.1016/B978-0-08-045031-5.00007-4.  [DOI](https://doi.org/10.1016/B978-0-08-045031-5.00007-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21496627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Handb.%20Clin.%20Neurol.&title=Dysferlinopathies&author=AA%20Amato&author=RH%20Brown&volume=101&publication_year=2011&pages=111-118&pmid=21496627&doi=10.1016/B978-0-08-045031-5.00007-4&)

62. Lampe LK, Bushby KM. Collagen VI-related muscle disorders. J. Med. Genet. 2005;42(9):673–685. doi: 10.1136/jmg.2002.002311.  [DOI](https://doi.org/10.1136/jmg.2002.002311) | [PMC free article](/articles/PMC1736127/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16141002/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Genet.&title=Collagen%20VI-related%20muscle%20disorders&author=LK%20Lampe&author=KM%20Bushby&volume=42&issue=9&publication_year=2005&pages=673-685&pmid=16141002&doi=10.1136/jmg.2002.002311&)

63. Bushby KM, Collins J, Hicks D. Collagen type VI myopathies. Adv. Exp. Med. Biol. 2014;802:185–199. doi: 10.1007/978-94-007-7893-1_12.  [DOI](https://doi.org/10.1007/978-94-007-7893-1_12) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24443028/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv.%20Exp.%20Med.%20Biol.&title=Collagen%20type%20VI%20myopathies&author=KM%20Bushby&author=J%20Collins&author=D%20Hicks&volume=802&publication_year=2014&pages=185-199&pmid=24443028&doi=10.1007/978-94-007-7893-1_12&)

64. Musumeci O, Brady S, Rodolico C, et al. Recurrent rhabdomyolysis due to muscle β-enolase deficiency: very rare or underestimated? J. Neurol. 2014;261(12):2424–2428. doi: 10.1007/s00415-014-7512-7.  [DOI](https://doi.org/10.1007/s00415-014-7512-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25267339/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Neurol.&title=Recurrent%20rhabdomyolysis%20due%20to%20muscle%20%CE%B2-enolase%20deficiency:%20very%20rare%20or%20underestimated?&author=O%20Musumeci&author=S%20Brady&author=C%20Rodolico&volume=261&issue=12&publication_year=2014&pages=2424-2428&pmid=25267339&doi=10.1007/s00415-014-7512-7&)

65. Tanabe T, Beam KG, Adams BA, Niidome T, Numa S. Regions of the skeletal muscle dihydropyridine receptor critical for excitation–contraction coupling. Nature. 1990;346:567–569. doi: 10.1038/346567a0.  [DOI](https://doi.org/10.1038/346567a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2165570/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Regions%20of%20the%20skeletal%20muscle%20dihydropyridine%20receptor%20critical%20for%20excitation%E2%80%93contraction%20coupling&author=T%20Tanabe&author=KG%20Beam&author=BA%20Adams&author=T%20Niidome&author=S%20Numa&volume=346&publication_year=1990&pages=567-569&pmid=2165570&doi=10.1038/346567a0&)

66. Lu X, Xu L, Meissner G. Activation of the skeletal muscle calcium release channel by a cytoplasmic loop of the dihydropyridine receptor. J. Biol. Chem. 1994;269:6511–6516.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8120002/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Activation%20of%20the%20skeletal%20muscle%20calcium%20release%20channel%20by%20a%20cytoplasmic%20loop%20of%20the%20dihydropyridine%20receptor&author=X%20Lu&author=L%20Xu&author=G%20Meissner&volume=269&publication_year=1994&pages=6511-6516&pmid=8120002&)

67. Dorschner MO, Amendola LM, Turner EH, et al. Actionable, pathogenic incidental findings in 1,000 participants’ exomes. Am. J. Hum. Genet. 2013;93:631–640. doi: 10.1016/j.ajhg.2013.08.006.  [DOI](https://doi.org/10.1016/j.ajhg.2013.08.006) | [PMC free article](/articles/PMC3791261/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24055113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Actionable,%20pathogenic%20incidental%20findings%20in%201,000%20participants%E2%80%99%20exomes&author=MO%20Dorschner&author=LM%20Amendola&author=EH%20Turner&volume=93&publication_year=2013&pages=631-640&pmid=24055113&doi=10.1016/j.ajhg.2013.08.006&)

68. Gonsalves SG, Ng D, Johnston JJ, et al. Using exome data to identify malignant hyperthermia susceptibility mutations. Anesthesiology. 2013;119:1043–1053. doi: 10.1097/ALN.0b013e3182a8a8e7.  [DOI](https://doi.org/10.1097/ALN.0b013e3182a8a8e7) | [PMC free article](/articles/PMC4077354/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24195946/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Using%20exome%20data%20to%20identify%20malignant%20hyperthermia%20susceptibility%20mutations&author=SG%20Gonsalves&author=D%20Ng&author=JJ%20Johnston&volume=119&publication_year=2013&pages=1043-1053&pmid=24195946&doi=10.1097/ALN.0b013e3182a8a8e7&)

69. Nicole S, Chaouch A, Torbergsen T, et al. Agrin mutations lead to a congenital myasthenic syndrome with distal muscle weakness and atrophy. Brain. 2014;137:2429–2443. doi: 10.1093/brain/awu160.  [DOI](https://doi.org/10.1093/brain/awu160) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24951643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain&title=Agrin%20mutations%20lead%20to%20a%20congenital%20myasthenic%20syndrome%20with%20distal%20muscle%20weakness%20and%20atrophy&author=S%20Nicole&author=A%20Chaouch&author=T%20Torbergsen&volume=137&publication_year=2014&pages=2429-2443&pmid=24951643&doi=10.1093/brain/awu160&)

70. Kraeva N, Sapa A, Dowling JJ, Riazi S. Malignant hyperthermia susceptibility in patients with exertional rhabdomyolysis: a retrospective cohort study and updated systematic review. Can. J. Anaesth. 2017;64(7):736–743. doi: 10.1007/s12630-017-0865-5.  [DOI](https://doi.org/10.1007/s12630-017-0865-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28326467/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Can.%20J.%20Anaesth.&title=Malignant%20hyperthermia%20susceptibility%20in%20patients%20with%20exertional%20rhabdomyolysis:%20a%20retrospective%20cohort%20study%20and%20updated%20systematic%20review&author=N%20Kraeva&author=A%20Sapa&author=JJ%20Dowling&author=S%20Riazi&volume=64&issue=7&publication_year=2017&pages=736-743&pmid=28326467&doi=10.1007/s12630-017-0865-5&)

71. McCarthy TV, Quane K, Lynch PJ. Ryanodine receptor mutations in malignant hyperthermia and central core disease. Hum. Mutat. 2000;15:410–417. doi: 10.1002/(SICI)1098-1004(200005)15:5<410::AID-HUMU2>3.0.CO;2-D.  [DOI](https://doi.org/10.1002/(SICI)1098-1004(200005)15:5<410::AID-HUMU2>3.0.CO;2-D) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10790202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mutat.&title=Ryanodine%20receptor%20mutations%20in%20malignant%20hyperthermia%20and%20central%20core%20disease&author=TV%20McCarthy&author=K%20Quane&author=PJ%20Lynch&volume=15&publication_year=2000&pages=410-417&pmid=10790202&doi=10.1002/(SICI)1098-1004(200005)15:5<410::AID-HUMU2>3.0.CO;2-D&)

72. Avila G, Dirksen RT. Functional effects of central core disease mutations in the cytoplasmic region of the skeletal muscle ryanodine receptor. J. Gen. Physiol. 2001;118:277–290. doi: 10.1085/jgp.118.3.277.  [DOI](https://doi.org/10.1085/jgp.118.3.277) | [PMC free article](/articles/PMC2229502/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11524458/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Gen.%20Physiol.&title=Functional%20effects%20of%20central%20core%20disease%20mutations%20in%20the%20cytoplasmic%20region%20of%20the%20skeletal%20muscle%20ryanodine%20receptor&author=G%20Avila&author=RT%20Dirksen&volume=118&publication_year=2001&pages=277-290&pmid=11524458&doi=10.1085/jgp.118.3.277&)

73. Dirksen RT, Avila G. Distinct effects on Ca2+ handling caused by malignant hyperthermia and central core disease mutations in RYR1. Biophys. J. 2004;87:3193–3204. doi: 10.1529/biophysj.104.048447.  [DOI](https://doi.org/10.1529/biophysj.104.048447) | [PMC free article](/articles/PMC1304789/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15347586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biophys.%20J.&title=Distinct%20effects%20on%20Ca2+%20handling%20caused%20by%20malignant%20hyperthermia%20and%20central%20core%20disease%20mutations%20in%20RYR1&author=RT%20Dirksen&author=G%20Avila&volume=87&publication_year=2004&pages=3193-3204&pmid=15347586&doi=10.1529/biophysj.104.048447&)

74. Monnier N, Romero NB, Lerale J, et al. Familial and sporadic forms of central core disease are associated with mutations in the C-terminal domain of the skeletal muscle ryanodine receptor. Human Mol. Genet. 2001;10:2581–2592. doi: 10.1093/hmg/10.22.2581.  [DOI](https://doi.org/10.1093/hmg/10.22.2581) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11709545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Human%20Mol.%20Genet.&title=Familial%20and%20sporadic%20forms%20of%20central%20core%20disease%20are%20associated%20with%20mutations%20in%20the%20C-terminal%20domain%20of%20the%20skeletal%20muscle%20ryanodine%20receptor&author=N%20Monnier&author=NB%20Romero&author=J%20Lerale&volume=10&publication_year=2001&pages=2581-2592&pmid=11709545&doi=10.1093/hmg/10.22.2581&)

75. Dirksen RT, Avila G. Altered ryanodine receptor function in central core disease: leaky or uncoupled Ca(2+) release channels? Trends Cardiovasc. Med. 2002;12:189–197. doi: 10.1016/s1050-1738(02)00163-9.  [DOI](https://doi.org/10.1016/s1050-1738(02)00163-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12161072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Cardiovasc.%20Med.&title=Altered%20ryanodine%20receptor%20function%20in%20central%20core%20disease:%20leaky%20or%20uncoupled%20Ca(2+)%20release%20channels?&author=RT%20Dirksen&author=G%20Avila&volume=12&publication_year=2002&pages=189-197&pmid=12161072&doi=10.1016/s1050-1738(02)00163-9&)

76. Efremov RG, Leitner A, Aebersold R, Raunser S. Architecture and conformational switch mechanism of the ryanodine receptor. Nature. 2015;517:39–43. doi: 10.1038/nature13916.  [DOI](https://doi.org/10.1038/nature13916) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25470059/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Architecture%20and%20conformational%20switch%20mechanism%20of%20the%20ryanodine%20receptor&author=RG%20Efremov&author=A%20Leitner&author=R%20Aebersold&author=S%20Raunser&volume=517&publication_year=2015&pages=39-43&pmid=25470059&doi=10.1038/nature13916&)

77. Zalk R, Clarke OB, des Georges A, et al. Structure of a mammalian ryanodine receptor. Nature. 2015;517:44–49. doi: 10.1038/nature13950.  [DOI](https://doi.org/10.1038/nature13950) | [PMC free article](/articles/PMC4300236/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25470061/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Structure%20of%20a%20mammalian%20ryanodine%20receptor&author=R%20Zalk&author=OB%20Clarke&author=A%20des%20Georges&volume=517&publication_year=2015&pages=44-49&pmid=25470061&doi=10.1038/nature13950&)

78. des Georges A, Clarke OB, Zalk R, et al. Structural basis for gating and activation of RYR1. Cell. 2016;167:145–157. doi: 10.1016/j.cell.2016.08.075.  [DOI](https://doi.org/10.1016/j.cell.2016.08.075) | [PMC free article](/articles/PMC5142848/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27662087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Structural%20basis%20for%20gating%20and%20activation%20of%20RYR1&author=A%20des%20Georges&author=OB%20Clarke&author=R%20Zalk&volume=167&publication_year=2016&pages=145-157&pmid=27662087&doi=10.1016/j.cell.2016.08.075&)

79. Tung CC, Lobo PA, Kimlicka L, Van Petegem F. The amino-terminal disease hotspot of ryanodine receptors forms a cytoplasmic vestibule. Nature. 2010;468:585–588. doi: 10.1038/nature09471.  [DOI](https://doi.org/10.1038/nature09471) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21048710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20amino-terminal%20disease%20hotspot%20of%20ryanodine%20receptors%20forms%20a%20cytoplasmic%20vestibule&author=CC%20Tung&author=PA%20Lobo&author=L%20Kimlicka&author=F%20Van%20Petegem&volume=468&publication_year=2010&pages=585-588&pmid=21048710&doi=10.1038/nature09471&)

80. Lobo PA, Van Petegem F. Crystal structures of the N-terminal domains of cardiac and skeletal muscle ryanodine receptors: insights into disease mutations. Structure. 2009;17:1505–1514. doi: 10.1016/j.str.2009.08.016.  [DOI](https://doi.org/10.1016/j.str.2009.08.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19913485/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Structure&title=Crystal%20structures%20of%20the%20N-terminal%20domains%20of%20cardiac%20and%20skeletal%20muscle%20ryanodine%20receptors:%20insights%20into%20disease%20mutations&author=PA%20Lobo&author=F%20Van%20Petegem&volume=17&publication_year=2009&pages=1505-1514&pmid=19913485&doi=10.1016/j.str.2009.08.016&)

81. Yuchi Z, Van Petegem F. Ryanodine receptors under the magnifying lens: Insights and limitations of cryo-electron microscopy and x-ray crystallography studies. Cell Calcium. 2016;59:209–227. doi: 10.1016/j.ceca.2016.04.003.  [DOI](https://doi.org/10.1016/j.ceca.2016.04.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27103405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Calcium&title=Ryanodine%20receptors%20under%20the%20magnifying%20lens:%20Insights%20and%20limitations%20of%20cryo-electron%20microscopy%20and%20x-ray%20crystallography%20studies&author=Z%20Yuchi&author=F%20Van%20Petegem&volume=59&publication_year=2016&pages=209-227&pmid=27103405&doi=10.1016/j.ceca.2016.04.003&)

82. Brillantes A-MB, Ondrias K, Scott A, et al. Stabilization of calcium release channel (ryanodine receptor) function by FK506-binding protein. Cell. 1994;77:513–523. doi: 10.1016/0092-8674(94)90214-3.  [DOI](https://doi.org/10.1016/0092-8674(94)90214-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7514503/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Stabilization%20of%20calcium%20release%20channel%20(ryanodine%20receptor)%20function%20by%20FK506-binding%20protein&author=A-MB%20Brillantes&author=K%20Ondrias&author=A%20Scott&volume=77&publication_year=1994&pages=513-523&pmid=7514503&doi=10.1016/0092-8674(94)90214-3&)
