# Pharmacogenomics of Vincristine‐Induced Peripheral Neuropathy Implicates Pharmacokinetic and Inherited Neuropathy Genes

## Metadata
**Authors:** Galen EB Wright, Ursula Amstutz, Britt I Drögemöller, Joanne Shih, Shahrad R Rassekh, Michael R Hayden, Bruce C Carleton, Colin JD Ross, The Canadian Pharmacogenomics Network for Drug Safety Consortium
**Journal:** Clinical Pharmacology and Therapeutics
**Date:** 2018 Aug 17
**DOI:** [10.1002/cpt.1179](https://doi.org/10.1002/cpt.1179)
**PMID:** 29999516
**PMCID:** PMC6519044
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6519044/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6519044/pdf/CPT-105-402.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6519044/pdf/CPT-105-402.pdf)

## Abstract

Vincristine is an effective chemotherapeutic drug for various cancers, including acute lymphoblastic leukemia (ALL). Unfortunately, clinical utility is restricted by dose‐limiting vincristine‐induced peripheral neuropathies (VIPN). We sought to determine the association of VIPN with a recently identified risk variant, CEP72 rs924607, and drug absorption, distribution, metabolism, and excretion (ADME) gene variants in pediatric ALL. This was followed by a meta‐analysis of pharmacogenomic data from over 500 patients. CEP72 rs924607 was significantly associated with VIPN (P = 0.02; odds ratio (OR) = 3.4). ADME analyses identified associations between VIPN and ABCC1 rs3784867 (P = 5.34 × 10−5; OR = 4.9), and SLC5A7 rs1013940 (P = 9.00 × 10−4; OR= 8.6); genes involved in vincristine transport and inherited neuropathies, respectively. Meta‐analysis identified an association with a variant related to TTPA (rs10504361: P = 6.85 × 10−4; OR = 2.0), a heritable neuropathy‐related gene. This study provides essential corroboratory evidence for CEP72 rs924607 and highlights the importance of drug transporter and inherited neuropathy genes in VIPN.

#### WHAT IS THE CURRENT KNOWLEDGE ON THE TOPIC?

#### WHAT IS THE CURRENT KNOWLEDGE ON THE TOPIC?

#### WHAT IS THE CURRENT KNOWLEDGE ON THE TOPIC?

☑Dose‐limiting VIPNs have been shown to have a genetic component. Pharmacogenomic studies have begun to identify genetic factors that can help explain these adverse reactions, but more research is required in this regard.

#### WHAT QUESTION DID THIS STUDY ADDRESS?

☑To assess whether the *CEP72* biomarker and drug ADME pharmacogenomic variants are associated with VIPN in pediatric patients with ALL.

#### WHAT DOES THIS STUDY ADD TO OUR KNOWLEDGE?

☑We confirmed the association of *CEP72* rs924607 with VIPN, providing additional evidence for the potential clinical utility of this biomarker. Further, we showed that genetic variation within the vincristine drug transporter (*ABCC1*) was also associated with VIPN. Last, we uncovered the first evidence that genes that cause inherited neuropathies (*SLC5A7* and *TTPA*) may contribute to VIPN susceptibility.

#### HOW MIGHT THIS CHANGE CLINICAL PHARMACOLOGY OR TRANSLATIONAL SCIENCE?

☑These findings will help guide the development of predictive markers and neuroprotective strategies for VIPN in order facilitate precision medicine and optimal care on an individualized basis for oncology patients.s

Vincristine is an antineoplastic agent that is used to treat various cancers. This includes the most common pediatric form, acute lymphoblastic leukemia (ALL), a cancer that has survival rates that have increased to over 90% in certain world regions in recent decades.[1](#cpt1179-bib-0001), [2](#cpt1179-bib-0002) Vincristine‐induced neurotoxicity is a common and severe adverse drug reaction (ADR) that can occur in over 50% of vincristine‐treated patients.[3](#cpt1179-bib-0003), [4](#cpt1179-bib-0004), [5](#cpt1179-bib-0005) This potentially life‐threatening and debilitating toxicity includes vincristine‐induced peripheral neuropathy (VIPN), which can cause loss of motor function, abnormal sensation, and pain.[6](#cpt1179-bib-0006) This reaction severely affects the quality of life of patients, as well as influences the dosing and clinical utility of the drug.[7](#cpt1179-bib-0007) Genetic causes are thought to play an important role in this ADR because VIPN‐susceptibility varies across ancestries[8](#cpt1179-bib-0008) and the trait has been shown to be heritable in model organisms.[9](#cpt1179-bib-0009)

Pharmacogenomics provide an attractive approach to mitigate VIPN – predictive testing would allow patients to be stratified according to risk before treatment, allowing for either increased monitoring or dose changes in at‐risk patients. Furthermore, the identification of genetic risk factors may provide novel insights into the mechanisms underlying variability in VIPN‐susceptibility. Recently, a genome‐wide association study (GWAS) in two cohorts reported that variation linked to the microtubule‐organizing centrosomal protein, *CEP72*, significantly contributes to the risk of VIPN in pediatric ALL.[10](#cpt1179-bib-0010) This is in line with the mechanism of action of vincristine, which perturbs mitosis by interfering with microtubule formation.[7](#cpt1179-bib-0007), [11](#cpt1179-bib-0011) An initial replication study failed to detect a significant association between this *CEP72* risk genotype and VIPN,[12](#cpt1179-bib-0012) which may have been driven by differences in the therapy stage during which VIPN was assessed (i.e., continuation vs. induction).[13](#cpt1179-bib-0013) Subsequently, members of the research group involved in the discovery of the *CEP72*‐VIPN association replicated this finding in a clinical trial of adult patients with ALL,[14](#cpt1179-bib-0014) but confirmation in an independent pediatric cohort is still warranted.

We, therefore, assessed the relationship between VIPN and *CEP72* genotype in a retrospective cohort of pediatric patients with ALL. This was followed by an exploratory analysis of pharmacogenomic variants from absorption, distribution, metabolism, and excretion (ADME) genes and a meta‐analysis of ADME genetic variants that overlapped variants from the previous GWAS.[10](#cpt1179-bib-0010) Studies such as these provide essential corroboratory clinical evidence for previously identified drug safety biomarkers, while also identifying additional variants for future study.

## Results

A total of 167 cases (i.e., VIPN grade ≥ 2) and 57 controls (i.e., VIPN grade 0) met clinical and genomic quality control and were included in the discovery pharmacogenomic association studies, after excluding 29 patients with grade 1 VIPN and 12 patients with only cranial or autonomic neurotoxicities (**Figure** [S1](#cpt1179-sup-0001), **Table** [S1](#cpt1179-sup-0001)). This study design is classified as a nonmatched case‐control association study, in which significantly different clinical and demographic factors between cases and controls were corrected for during pharmacogenomic association analyses. The majority of VIPN cases were classified as grade 2 (*n *=* *100), followed by grade 3 (*n *=* *66), and grade 4 (*n *=* *1). The median days from first vincristine dose to VIPN were 57 (interquartile range: 18–127 days). Patients were predominantly of European genetic ancestry (78%, **Figure** [S2](#cpt1179-sup-0001)) and treatment protocol information is listed in **Table** [S2](#cpt1179-sup-0001). **Table** [1](#cpt1179-tbl-0001) depicts differences in clinical and demographic factors between VIPN cases and controls. Cases were more likely to be male and received vincristine for a longer duration compared to controls (median of 812 days vs. median of 757 days). All controls, however, received vincristine for at least 16 months, a period of time in which 94% of cases had developed VIPN (**Figure** [S3](#cpt1179-sup-0001)), indicating sufficient follow‐up time. The cumulative vincristine dose was not significantly associated with case status, whereas duration of treatment was. This observation may have resulted from the fact that many VIPN cases had either vincristine doses held or reduced during treatment as a result of the experienced toxicities, which may explain the difference in therapy duration and the lack of association with cumulative dose. Cases were also more likely to have received nifedipine compared to controls. To help reduce the influence of potential confounders on the genetic association analyses, all significantly associated demographic/clinical variables (i.e., sex, vincristine duration, and nifedipine use) were included as covariates in regression models along with genetic ancestry.

### Table 1.

| Patient characteristics | Cases (n = 167) VIPN grade ≥ 2 | Controls (n = 57) VIPN grade = 0 | P value |
| --- | --- | --- | --- |
| Age, years (median (IQR)) | 4.8 (3.3–9.0) | 5.4 (3.3–9.0) | 0.63 |
| Sex (male n, (%)) | 101 (60.4%) | 23 (40.4%) | 0.009 |
| , days (median (IQR)) | 812 (759–1137) | 757 (745–825) | 0.0004 |
| dose, mg/m2 (median (IQR)) | 61.4 (48.0–72.0) | 66.0 (51.0–74.8) | 0.17 |
| Vincristine doses, number (median (IQR)) | 39 (32–45) | 38 (37–39) | 0.47 |
| Concomitant medication (n, (%)) |  |  |  |
| Strong cytochrome P450 3A inhibitorsa | 59 (35.3%) | 17 (29.8%) | 0.52 |
| Antifungal azoles | 54 (32.3%) | 14 (24.6%) | 0.32 |
| Nifedipine | 22 (13.2%) | 1 (1.8%) | 0.01 |
| Genetic ancestry (median (IQR)) |  |  |  |
| PC 1 (African from non‐African) | 0.011 [0.010–0.012] | 0.010 [0.008–0.012] | 0.03 |
| PC 2 (European from Asian) | 0.020 [0.017–0.022] | 0.019 [0.011–0.021] | 0.17 |
| PC 3 (South Asian from non‐South Asian) | 0.008 [0.005–0.012] | 0.006 [0.002–0.010] | 0.02 |
| PC 4 (Amerindian from non‐Amerindian) | 0.010 [0.006–0.013] | 0.010 [0.005–0.012] | 0.37 |

Table 1 Caption: Clinical and demographic characteristics of the CPNDS pediatric patients with ALL

The *CEP72* rs924607 TT genotype was significantly associated with VIPN in the Canadian Pharmacogenomics Network for Drug Safety (CPNDS) cohort (one‐sided *P *=* *0.02; odds ratio (OR) = 3.43; 90% confidence interval (CI) = 1.15–10.3). The TT risk genotype frequencies for *CEP72* rs924607 were 16.0% in VIPN cases and 5.4% in controls (**Figure** [1](#cpt1179-fig-0001) **a**, **Table** [S3](#cpt1179-sup-0001)), displaying a specificity of 0.95 (95% CI = 0.85–0.99) and sensitivity of 0.16 (95% CI = 0.11–0.22). The association between the *CEP72* rs924607‐TT genotype and VIPN was further confirmed in a meta‐analysis of the CPNDS results and published‐*CEP72‐*VIPN studies (**Figure** [1](#cpt1179-fig-0001) **b**;* n* studies = 4; *P *=* *8.02 × 10^−11^; OR = 2.77; 95% CI = 2.04–3.76; no heterogeneity: *I* ^2^ = 0.0%).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8822/6519044/34a44e254b1f/CPT-105-402-g001.jpg)

CEP72 is significantly associated with vincristine‐induced peripheral neuropathies (VIPN) in pediatric patients with acute lymphoblastic leukemia. (a) Recessive genotype frequencies for CEP72 rs924607 differed significantly between VIPN cases and controls (P = 0.02). The TT risk genotype occurred in 16.0% of cases (VIPN grade ≥2) and 5.4% of controls (VIPN grade 0). (b) Meta‐analysis of the association of CEP72 rs924607 and VIPN confirms the role of this variant with vincristine‐related adverse effects. The gray summary diamond indicates meta‐analysis results including all studies, whereas the blue summary diamond shows the results for studies where VIPN was assessed throughout continuation therapy (i.e., excluding the Gutierrez‐Camino et al.12 study). Genotype 95% confidence intervals (CIs) are represented for consistency across all studies, as opposed to the 90% CI that was used to determine CEP72 rs924607 replication evidence for the Canadian Pharmacogenomics Network for Drug Safety cohort in the current study.10,13,14

Subsequent analysis of 4,522 ADME variants meeting the inclusion criteria for association testing revealed that no other previously identified variant was significantly associated with VIPN (i.e., *P *<* *0.05; **Table** [S4](#cpt1179-sup-0001)). Further, no variants assessed within the vincristine metabolism‐related gene cluster (i.e., *CYP3A5*‐*CYP3A7*‐*CYP3A4*) displayed *P *<* *0.05. An intronic *ABCC1* polymorphism (**Table** [S5](#cpt1179-sup-0001)) was the top ADME variant with the strongest evidence of association (rs3784867 *P *=* *5.34 × 10^−5^; OR = 4.91; 95% CI = 1.99–12.10; specificity = 0.88; sensitivity = 0.36; **Table** [S6](#cpt1179-sup-0001)). The frequency of rs3784867 genotypes correlated with VIPN grade upon the inclusion of grade 1 patients (**Figure** [2](#cpt1179-fig-0002) **a**;* P* = 7.12 × 10^−4^). We assessed *ABCC1* rs3784867 for replication in the previously published VIPN‐GWAS summary statistics,[10](#cpt1179-bib-0010) and, although the risk allele was more frequent in cases in both the St Jude Total XIIIB and COG AALL0433, only suggestive evidence for replication in the combined XIIIB‐COG cohorts was observed (one‐sided *P *=* *0.08). Interestingly, the third most significant finding from the ADME analyses was a missense variant in *SLC5A7* (rs1013940 *P *=* *9.00 × 10^−4^; OR = 8.60; 95% CI = 1.68–44.15; specificity = 0.96; sensitivity = 0.18; **Table** [S7](#cpt1179-sup-0001)), a gene that encodes a choline transporter, where mutations are known to cause a form of hereditary motor neuropathy.[15](#cpt1179-bib-0015) Of note, all homozygous risk variant carriers presented with the most severe VIPN grade (**Figure** [2](#cpt1179-fig-0002) **b**). This variant is also an *SLC5A7* expression quantitative trait locus (eQTL and Genotype‐Tissue Expression (GTEx) multitissue *P *=* *3.62 × 10^−7^).[16](#cpt1179-bib-0016) Unfortunately, no data were available for this variant or those in linkage disequilibrium in the GWAS summary statistics, so we were unable to assess the *SLC5A7* variant for replication or in the meta‐analysis.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8822/6519044/276e0ab164e0/CPT-105-402-g002.jpg)

Top absorption, distribution, metabolism, and excretion (ADME)‐vincristine‐induced peripheral neuropathies (VIPN) variant genotypes stratified by severity of the adverse event. (a) ABCC1 rs3784867 was the most significantly associated pharmacogenomic variant in the VIPN‐ADME analyses (P = 5.34 × 10−5; odds ratio (OR) = 4.9). Genotype frequency was also correlated with severity of the adverse event (P = 7.12 × 10−4). (b) SLC5A7 rs1013940 is a missense variant in a peripheral neuropathy gene that was among the most significantly associated ADME variants (P = 9.00 × 10−4; OR = 8.6). The variant was also moderately associated with VIPN grade severity (P = 0.01), with all homozygous carriers of the risk allele presenting with severe VIPN (i.e., grade 3 and 4). Because only one case of grade 4 VIPN was observed in this cohort, this patient was combined with the grade 3 group.

We performed annotation of the *ABCC1* rs3784867 variant to delineate its potential functional role. No variants in strong LD (i.e., *r* ^2^ > 0.8 and no missense variants with *D*′ > 0.8) with the marker were identified in the 1,000 Genomes Project. Annotation of rs3784867 with DNAse and histone ChIP‐Seq data indicated the presence of enhancer and histone marks in five tissues/cell types (i.e., embryonic stem cells, induced pluripotent stem cells, hematopoietic stem cells, smooth muscle, and the pancreas). Additionally, rs3784867 was predicted to alter homeobox transcription factor motifs (**Figure** [S4](#cpt1179-sup-0001)).

The most significant association detected in the VIPN‐ADME‐GWAS meta‐analysis (**Table** [S8](#cpt1179-sup-0001)) of 379 variants was rs10504361 (*P *=* *6.85 × 10^−4^; OR = 1.98; 95% CI = 1.34–2.94; **Figure** [3](#cpt1179-fig-0003)). This variant is a GTEx‐eQTL[16](#cpt1179-bib-0016) for the *TTPA* and *GGH* genes. Because *TTPA* mutations are known to cause ataxia with isolated vitamin E deficiency, a neurologic condition associated with neuropathies,[17](#cpt1179-bib-0017) we assessed differences in predicted gene expression of *TTPA* in the tibial nerve with S‐PrediXcan, which collapses eQTL data by gene, in the GWAS‐array genotyped cohorts. These analyses revealed positive gene‐level *Z*‐scores in both cohorts, indicating that VIPN cases were predicted to have increased expression compared to controls (*P *=* *6.1 × 10^−5^).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8822/6519044/ee91d3a9c24f/CPT-105-402-g003.jpg)

Association of TTPA‐related variation and gene expression with vincristine‐induced peripheral neuropathy (VIPN). (a) Meta‐analysis of overlapping absorption, distribution, metabolism, and excretion (ADME)‐genomewide association study (GWAS) variants in the Canadian Pharmacogenomics Network for Drug Safety (CPNDS)/St Jude Total XIIIB/COG AALL0433 cohorts identified rs10504361 as the most significantly associated variant. (b) This variant has been shown to be associated with differential expression of the TTPA gene in the tibial nerve in GTEx. (c) Imputing expression of TTPA using genome wide array data revealed that VIPN cases were predicted to have higher levels of TTPA expression compared to controls as reflected by gene‐level S‐PrediXcan Z‐scores (i.e., positive scores indicate increased expression is associated with increased risk for VIPN). CI, confidence interval; OR, odds ratio.

## Discussion

Pediatric oncology has seen a dramatic improvement in the prognosis for patients over recent decades, yet ADRs from these drugs have become an increasing concern. In this study, we independently confirmed a role of *CEP72* rs924607 in VIPN susceptibility for the first time in a retrospective cohort of pediatric patients. This finding validates the importance of this pharmacogenomic variant for the prediction of a serious ADR caused by an essential chemotherapeutic agent. Our exploratory pharmacogenomic analyses also brought to light the potential VIPN‐related importance of the transporter, *ABCC1*, and highlighted genes associated with inherited neuropathy conditions, such as *SLC5A7* and *TTPA*.

### Confirmation of CEP72 as a pharmacogenomic risk factor for VIPN

*CEP72* rs924607 is a promising VIPN biomarker given its potential role in the mechanism of action of this antimitotic agent. However, prior to the current study, all evidence supporting this finding was obtained from a single research group, although for the most recent replication study,[14](#cpt1179-bib-0014) these investigators collaborated on the study by performing genotyping of *CEP72* rs924607 while blinded to VIPN phenotype. Of note, this the only other external study refuting the role of this variant in VIPN. Our positive results, supported by a meta‐analysis of published VIPN‐*CEP72* studies,[10](#cpt1179-bib-0010), [12](#cpt1179-bib-0012), [14](#cpt1179-bib-0014) provide the first external validation of this finding in pediatric patients. In particular, the replication of *CEP72* in a diverse pediatric ALL cohort confirms the generalizability of this pharmacogenomic marker. Our data also improve the precision and accuracy of estimates regarding the effect of this variant (meta‐analysis: *P *=* *8.02 × 10^−11^; OR = 2.77; 95% CI = 2.04–3.76) in studies assessing VIPN from vincristine continuation therapies. Concurrently, these results corroborate the validity of our grading scheme that retrospectively assessed VIPN based on medical records of patients recruited via active surveillance.

The failure of the first rs924607‐VIPN replication attempt[12](#cpt1179-bib-0012) brings to light important phenotypic considerations for pharmacogenomic studies. Unlike the discovery study, which included monitoring during induction and continuation phases of vincristine therapy, Gutierrez‐Camino *et al*.[12](#cpt1179-bib-0012) assessed VIPN solely during the 4‐week induction phase of vincristine therapy. Of note, the majority (63%) of the VIPN cases in our cohort developed toxicity after this 4‐week time frame (**Figure** [S3](#cpt1179-sup-0001)). Our meta‐analysis was highly significant when excluding this study[12](#cpt1179-bib-0012) assessing VIPN only during induction therapy from analysis with no evidence for heterogeneity between studies. It has further been suggested that *CEP72* rs924607 may be primarily relevant to patients receiving vincristine doses of <2 mg/m^2^ because higher doses of vincristine may overcome the protective influence of the ancestral genotypes.[13](#cpt1179-bib-0013) Indeed, when excluding the CPNDS patients that received vincristine doses ≥ 2 mg/m^2^ (i.e., 34.3% of the cohort), the risk genotype was carried exclusively in cases (i.e., 16.1% of 112 cases; Fisher's exact *P *=* *0.007). These findings emphasize the importance of carefully matching drug‐induced phenotypes in pharmacogenomic replication studies.[18](#cpt1179-bib-0018)

### Exploratory analyses of ADME genetic variation

Because numerous pharmacogenes belong to ADME‐related pathways,[19](#cpt1179-bib-0019) we conducted an exploratory association analysis of ADME variants with regards to VIPN using a custom genotyping panel that extensively captures variation within these regions. Initial pharmacogenomic studies investigating VIPN hypothesized that *CYP3A* genetic variation may be relevant for this ADR because vincristine is chiefly metabolized by these enzymes.[8](#cpt1179-bib-0008) Our results, however, indicate that pharmacogenomic variation in *CYP3A* genes did not contribute significantly toward this phenotype in our patient cohort.

Our analyses revealed that the ADME variant with the strongest evidence for an association with VIPN, rs3784867 (*P *=* *5.34 × 10^−5^; OR = 4.91; 95% CI = 1.99–12.10), occurs in the intron of *ABCC1*. This gene encodes multidrug resistance protein 1, a protein, which is involved in both transport of, and resistance to, vincristine.[20](#cpt1179-bib-0020), [21](#cpt1179-bib-0021) *ABCC1* has also been implicated in anthracycline‐related and methotrexate‐related pharmacogenomic phenotypes.[22](#cpt1179-bib-0022) Rs3784867 is predicted to alter binding sites for homeobox transcription factors (**Figure** [S4](#cpt1179-sup-0001)), which are involved in hematopoiesis and are dysregulated in leukemia,[23](#cpt1179-bib-0023) indicating that rs3784867 could affect the expression of *ABCC1*. Notably, previous pharmacogenomic studies have detected associations with *ABCC1* and the paralogous transporter *ABCC2* and VIPN‐risk,[24](#cpt1179-bib-0024), [25](#cpt1179-bib-0025) although we did not replicate associations of the variants that were implicated in these studies. As discussed previously, heterogeneity in VIPN assessment may contribute toward the lack of transferability of pharmacogenomic markers between studies and because our study used an intervention‐based VIPN grading scheme, this may have contributed toward these discrepant results. The correlation between rs3784867 and ADR severity adds support for this variant's role in VIPN risk. However, because only suggestive evidence for replication was observed in the GWAS summary statistics for rs3784867, it is, important to further investigate this variant in additional VIPN cohorts.

### Overlap between the genomics of VIPN and inherited neuropathy conditions

It has been suggested that a continuum exists between the genetic etiology of Mendelian disorders and drug‐induced traits that present with similar clinical phenotypes.[26](#cpt1179-bib-0026) This is in line with what has been observed for GWAS of chemotherapy‐induced peripheral neuropathies from paclitaxel (*FGD4* gene)[27](#cpt1179-bib-0027) and docetaxel (*VAC14* gene),[28](#cpt1179-bib-0028) whereas both genes have also been implicated in inherited neuropathy conditions. Phenotype overlaps are further substantiated by the fact that vincristine is contraindicated in patients with hereditary neuropathies (e.g., Charcot‐Marie‐Tooth disease), because VIPN can be particularly severe in these patients.[29](#cpt1179-bib-0029) Our VIPN‐ADME‐GWAS meta‐analysis identified a variant (rs10504361) that is associated with the expression of *TTPA* in the peripheral nervous system. This protein influences bioavailability of alpha‐tocopherol, a form of vitamin E. Deleterious *TTPA* mutations cause the recessive neurodegenerative condition ataxia with isolated vitamin E deficiency, which is associated with peripheral neuropathy.[30](#cpt1179-bib-0030) *In silico* gene expression analysis indicated that cases were predicted to have increased levels of *TTPA* in the tibial nerve when compared to controls. This finding warrants further research to delineate the potential mechanism of action of the variant in the context of VIPN. Sensory conduction in the tibial nerve has been shown to be impaired in vincristine treated patients with ALL[31](#cpt1179-bib-0031) and it has motor functions through innervating the muscles of the leg, including the tibialis posterior muscle, which is involved in plantar flexion.[32](#cpt1179-bib-0032) This indicates that the tibial nerve may be an appropriate model to study gene expression in VIPN. Finally, our third most highly associated VIPN‐ADME gene was *SLC5A7*, a choline transporter that is implicated in distal hereditary motor neuropathy type 7A [15](#cpt1179-bib-0015) and congenital myasthenic syndromes, which are associated with muscle weakness.[33](#cpt1179-bib-0033) Given that the association with VIPN was observed for a missense *SLC5A7* variant (rs1013940) with a large effect size (OR > 8), and the similarity of the ADR and inherited phenotypes related to this gene, this candidate variant also warrants further investigation.

### Context of current VIPN pharmacogenomic findings

With the availability of high throughput genotyping platforms, pharmacogenomic studies are beginning to uncover novel biology that contributes toward ADRs in a clinically relevant manner.[19](#cpt1179-bib-0019) Although the current investigation provides valuable findings, there are caveats. Despite the fact that the sample size of the current study is relatively large for a pharmacogenomic study, including more patients in future cohorts would be beneficial. This is reflected by the fact that our exploratory analysis of ADME variants yielded no associations reaching statistical significance after stringent Bonferroni multiple testing correction. Additional confirmatory evidence should also be pursued in the future through functional experiments for the novel candidate genes identified in the ADME/GWAS analyses. Nonetheless, given the known involvement of the highlighted genes in vincristine pharmacokinetics and inherited neuropathy conditions, this study provides interesting avenues for future research. The retrospective nature of this study is also a potential limitation, because milder forms of VIPN may be poorly captured in standard medical records. We took great care in developing a slightly augmented Common Terminology Criteria for Adverse Events (CTCAE) grading scheme that can be applied in this context. Indeed, our results support the appropriateness of this grading scheme through the: (i) replication of the VIPN *CEP72* GWAS variant with a similar effect size as previously reported; (ii) identification of candidate genes that have a clear biological mechanism pertaining to the ADR; and (iii) the correlation of the genotype frequencies of top candidate variants with VIPN severity.

### Conclusions and future directions

VIPNs are important clinical concerns, especially given the increased survival rates of pediatric patients with cancer and potential long‐term effects of this ADR. The confirmatory results obtained in the current study highlight the *CEP72* rs924607 risk marker for further evaluation of its clinical utility in the context of potential pharmacogenomic risk predictions and preventive interventions. Specifically, predicting in whom debilitating VIPN is likely to occur would allow clinicians to discuss other therapeutic options, additional physiotherapy interventions, and, importantly, inform patients and parents in advance what toxicity may occur, further enhancing the family's role in cancer management.

Given the current level of evidence for the *CEP72* risk genotype, increased clinical monitoring and earlier initiation of physiotherapy interventions for VIPN in rs924607 TT patients could potentially be beneficial. Oncologists may also consider ALL treatment protocols with lower doses of vincristine and/or shorter vincristine treatment durations in high‐risk genotypes. It is important to note that alternative protocols should fall within current standards of care and display similar efficacy and survival outcomes. The results of an ongoing clinical trial investigating the influence of vincristine dose and duration reductions in *CEP72* rs924607 TT patients (ClinicalTrials.gov: [NCT03117751](https://clinicaltrials.gov/ct2/show/NCT03117751)) will also be highly informative in determining whether pharmacogenomic genotyping can minimize VIPN risk while maintaining treatment efficacy.

Additional VIPN pharmacogenomic markers, if confirmed through future replication, may further improve VIPN risk stratification. Finally, it will also be beneficial for future studies to use high‐throughput sequencing approaches to delineate extreme cases of VIPN in future studies, given the importance of rare variation for pharmacogenomic traits.[19](#cpt1179-bib-0019) In particular, resequencing studies would enable the investigation of rare variants in candidate genes for VIPN, such as the ones identified in this study. Comprehensive risk assessments that take different clinical, demographic, and pharmacogenomic factors into account may, therefore, ultimately lead to safer and more effective use of vincristine‐based chemotherapy in children.

## Methods

### Patients and clinical assessment

Pediatric patients with ALL (age at diagnosis ≤18 years) receiving treatment protocols, including vincristine, were recruited from seven pediatric hospitals across Canada (Calgary, London, Ottawa, Montréal, Toronto, Vancouver, and Winnipeg). Written informed consent and/or assent was obtained from all participants or their parents/legal guardians. The study was approved by ethics committees at participating universities and hospitals. Relevant clinical and demographic data were collected using the CPNDS active surveillance methodology.[34](#cpt1179-bib-0034) This included capturing information for relevant concomitant medications (e.g., CYP3A inhibitors, because these enzymes represent the major metabolic path of vincristine).[7](#cpt1179-bib-0007), [35](#cpt1179-bib-0035)

Patients were assessed for vincristine‐related neurotoxicities using a refined National Cancer Institute (NCI) CTCAE version 4.0 Grading System (**Table** [S1](#cpt1179-sup-0001)). Specifically, to facilitate grading of the retrospective clinical data and standardize grading across multiple sites, this system included definitions of neurotoxicity‐related clinically based interventions, such as pharmacotherapy, rehabilitative therapy, and vincristine dose reductions, to complement the relatively broad definitions (e.g., “noninvasive intervention” or “limiting activities of daily living”) used in the CTCAE. To be considered for grading, any intervention had to be used specifically for the treatment for vincristine‐related neurotoxicities, and any symptom had to be attributed to vincristine‐related neurotoxicities. We excluded patients with grade one (minor) VIPN from our initial pharmacogenomic nonmatched case‐control association analyses (**Figure** [S1](#cpt1179-sup-0001), cases were defined as grade ≥2, whereas controls were defined as patients not developing neuropathy, referred to as grade 0), because the need for a predictive biomarker is greater in those patients with neurotoxicity of sufficient severity to warrant a modification in clinical care (increased monitoring and drug therapy changes). In addition, patients presenting only with symptoms of cranial or central nervous system neurotoxicities without motor or sensory neurotoxicity were not included in the analysis.

### Pharmacogenomic genotyping and quality control

Genomic DNA from participants was genotyped for *CEP72* rs924607 using a TaqMan assay (C___8292459_20; ThermoFisher, Waltham, MA) and for 7,907 ADME variants with a custom Infinium Panel (Illumina, San Diego, CA).[36](#cpt1179-bib-0036) ADME genotype clusters were extracted from GenomeStudio Software (Illumina) and underwent standard quality control procedures. Specifically, array data were assessed for variant (≥95%) and sample call rates (≥90%), and deviations from Hardy Weinberg equilibrium (Fisher's exact *P *<* *0.001 in controls).

### Bioinformatic and statistical analyses

Statistical and bioinformatic analyses were performed using either R or SNP & Variation Suite (SVS) version 8.4 (Golden Helix, Bozeman, MT), unless otherwise stated. Genetic ancestry was determined via principal component analyses using pruned ADME genotypes (50‐variant window, shifted by 5‐variants after each assessment, pruning *r* ^2^ > 0.5) and minor allele frequency <0.01 with EIGENSOFT version 5.0, incorporating the 1,000 Genomes Project[37](#cpt1179-bib-0037) samples as a reference. The first four principal components, along with clinical factors that differed significantly (*P *<* *0.05) between cases and controls were included in subsequent case‐control association analyses to correct for possible population stratification and clinical confounders. Sensitivity and specificity of prioritized variants were calculated with epiR ([https://cran.r-project.org/web/packages/epiR](https://cran.r-project.org/web/packages/epiR)).

To investigate the *CEP72* rs924607 VIPN risk genotype, we decided *a priori* to first test for an association between this marker and VIPN using a recessive model, as reported in the initial study. For this analysis, a one‐sided *P* value < 0.05 was considered statistically significant. A random‐effects meta‐analysis was performed on *CEP72* rs924607, incorporating the results from this study along with published data, using the GWAMA software[38](#cpt1179-bib-0038); related plots were generated in the R package, forestplot ([https://CRAN.R-project.org/package=forestplot](https://CRAN.R-project.org/package=forestplot)).

These analyses were followed by exploratory association analyses of ADME variants passing quality control, and displaying minor allele frequency >0.05, using logistic regression and an additive genetic model. Due to the exploratory nature of this analysis, multiple testing corrections using a conservative Bonferroni threshold was not applied. Instead, variants with *P *<* *0.001 were prioritized. To provide corroboratory evidence for top VIPN‐associated variants, ordinal regression of VIPN grade was also performed using the R package ordinal.

An Embase literature search (further details are provided in **Table** [S4](#cpt1179-sup-0001)) was completed to identify previously identified VIPN‐related variants and assess their association with the ADR in the CPNDS cohort. Further, VIPN‐GWAS summary statistics were obtained for the two pediatric ALL cohorts from Diouf *et al*.[10](#cpt1179-bib-0010) and a meta‐analysis of nonambiguous overlapping ADME variants with evidence for association (*P *<* *0.05) in at least one cohort was performed as specified above.

To examine predicted functional effects of variants prioritized from the association analyses, Ensembl Variant Effect Predictor,[39](#cpt1179-bib-0039) HaploReg version 4.1,[40](#cpt1179-bib-0040) GTEx,[16](#cpt1179-bib-0016) and the JASPAR Core Database[41](#cpt1179-bib-0041) were used to annotate variants. Differences in predicted gene expression profiles in the tibial nerve[16](#cpt1179-bib-0016) between GWAS‐genotyped VIPN cases and controls were calculated using S‐PrediXcan,[42](#cpt1179-bib-0042) where specified, with *P* values being combined using Fisher's method in metap ([https://cran.r-project.org/web/packages/metap](https://cran.r-project.org/web/packages/metap)).

## Funding

This work was supported by the Canadian Foundation for Innovation/Canadian Institutes of Health Research (CIHR, grant No. CIHR CRI‐88362) and the CIHR‐Drug Safety and Effectiveness Network (grants CIHR TD1 137714 and CIHR TD2 117588), as well as the BC Children's Hospital Foundation.

## Conflict of Interest

The authors declared no competing interests for this work.

## Author Contributions

G.E.B.W. wrote the manuscript. G.E.B.W., U.A., S.R.R., B.C.C., and C.J.D. designed the research. G.E.B.W., U.A., B.I.D., J.S., S.R.R., M.R.H., B.C.C., and C.J.D.R. performed the research. G.E.B.W., U.A., B.I.D, B.C.C., and C.J.D.R. analyzed the data.

## Supporting information

## Acknowledgments

The authors thank the patients and their families for their participation in this study. We would also like to acknowledge the authors of Diouf *et al*.^10^ *JAMA* 2015 study for providing access to the vincristine‐induced neuropathy GWAS summary statistics.

The members of the Canadian Pharmacogenomics Network for Drug Safety Consortium are: Vancouver, BC Children's Hospital: Bruce Carleton, Colin Ross, Shahrad Rassekh, Henk Visscher, Folefac Aminkeng, Galen Wright, Britt Drögemöller, Michelle Higginson, Nasim Massah, Fudan Miao, Ursula Amstutz, Amit Bhavsar, Jong Lee, Tessa Bendyshe‐Walton, and Reo Tanoshima. Calgary, Alberta Children's Hospital: David Johnson and Carnation Zhuwaki. Winnipeg Health Sciences Centre: Nick Honcharik, Geert‘t Jong, Sara Israels, and Michelle Staub. London Health Sciences Centre: Michael Rieder and Lauren Faught; Toronto, Hospital for Sick Children: Shinya Ito, Paul Nathan, and Smita Karande. Ottawa, Children's Hospital of Eastern Ontario: Régis Vaillancourt, Donna Johnston, and Kimmy Nguyen.

Montreal, CHU Sainte‐Justine: Jean‐François Bussières, Denis Lebel, and Jennifer Jean‐Louis.

## Contributor Information

Colin J.D. Ross, Email: colin.ross@ubc.ca

The Canadian Pharmacogenomics Network for Drug Safety Consortium:

[Henk Visscher](https://pubmed.ncbi.nlm.nih.gov/?term=%22Visscher%20H%22%5BAuthor%5D), [Folefac Aminkeng](https://pubmed.ncbi.nlm.nih.gov/?term=%22Aminkeng%20F%22%5BAuthor%5D), [Michelle Higginson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Higginson%20M%22%5BAuthor%5D), [Nasim Massah](https://pubmed.ncbi.nlm.nih.gov/?term=%22Massah%20N%22%5BAuthor%5D), [Fudan Miao](https://pubmed.ncbi.nlm.nih.gov/?term=%22Miao%20F%22%5BAuthor%5D), [Amit Bhavsar](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhavsar%20A%22%5BAuthor%5D), [Jong Lee](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20J%22%5BAuthor%5D), [Tessa Bendyshe‐Walton](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bendyshe%E2%80%90Walton%20T%22%5BAuthor%5D), [Reo Tanoshima](https://pubmed.ncbi.nlm.nih.gov/?term=%22Tanoshima%20R%22%5BAuthor%5D), [David Johnson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Johnson%20D%22%5BAuthor%5D), [Carnation Zhuwaki](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhuwaki%20C%22%5BAuthor%5D), [Nick Honcharik](https://pubmed.ncbi.nlm.nih.gov/?term=%22Honcharik%20N%22%5BAuthor%5D), [Geert‘t Jong](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jong%20G%22%5BAuthor%5D), [Sara Israels](https://pubmed.ncbi.nlm.nih.gov/?term=%22Israels%20S%22%5BAuthor%5D), [Michelle Staub](https://pubmed.ncbi.nlm.nih.gov/?term=%22Staub%20M%22%5BAuthor%5D), [Michael Rieder](https://pubmed.ncbi.nlm.nih.gov/?term=%22Rieder%20M%22%5BAuthor%5D), [Lauren Faught](https://pubmed.ncbi.nlm.nih.gov/?term=%22Faught%20L%22%5BAuthor%5D), [Shinya Ito](https://pubmed.ncbi.nlm.nih.gov/?term=%22Ito%20S%22%5BAuthor%5D), [Paul Nathan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Nathan%20P%22%5BAuthor%5D), [Smita Karande](https://pubmed.ncbi.nlm.nih.gov/?term=%22Karande%20S%22%5BAuthor%5D), [Régis Vaillancourt](https://pubmed.ncbi.nlm.nih.gov/?term=%22Vaillancourt%20R%22%5BAuthor%5D), [Donna Johnston](https://pubmed.ncbi.nlm.nih.gov/?term=%22Johnston%20D%22%5BAuthor%5D), [Kimmy Nguyen](https://pubmed.ncbi.nlm.nih.gov/?term=%22Nguyen%20K%22%5BAuthor%5D), [Jean‐François Bussières](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bussi%C3%A8res%20JF%22%5BAuthor%5D), [Denis Lebel](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lebel%20D%22%5BAuthor%5D), and [Jennifer Jean‐Louis](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jean%E2%80%90Louis%20J%22%5BAuthor%5D)

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Global Burden of Disease Cancer, C. et al Global, regional, and national cancer incidence, mortality, years of life lost, years lived with disability, and disability‐adjusted life‐years for 32 cancer groups, 1990 to 2015: a systematic analysis for the Global Burden of Disease Study. JAMA Oncol. 3, 524–548 (2017).  [DOI](https://doi.org/10.1001/jamaoncol.2016.5688) | [PMC free article](/articles/PMC6103527/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27918777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Oncol.&title=Global,%20regional,%20and%20national%20cancer%20incidence,%20mortality,%20years%20of%20life%20lost,%20years%20lived%20with%20disability,%20and%20disability%E2%80%90adjusted%20life%E2%80%90years%20for%2032%20cancer%20groups,%201990%20to%202015:%20a%20systematic%20analysis%20for%20the%20Global%20Burden%20of%20Disease%20Study&volume=3&publication_year=2017&pages=524-548&pmid=27918777&doi=10.1001/jamaoncol.2016.5688&)

2. Gidding, C.E. , Kellie, S.J. , Kamps, W.A. & de Graaf, S.S. Vincristine revisited. Crit. Rev. Oncol. Hematol. 29, 267–287 (1999).  [DOI](https://doi.org/10.1016/s1040-8428(98)00023-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10226730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit.%20Rev.%20Oncol.%20Hematol.&title=Vincristine%20revisited&author=C.E.%20Gidding&author=S.J.%20Kellie&author=W.A.%20Kamps&author=S.S.%20de%20Graaf&volume=29&publication_year=1999&pages=267-287&pmid=10226730&doi=10.1016/s1040-8428(98)00023-7&)

3. Gomber, S. , Dewan, P. & Chhonker, D. Vincristine induced neurotoxicity in cancer patients. Indian J. Pediatr. 77, 97–100 (2010).  [DOI](https://doi.org/10.1007/s12098-009-0254-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19936661/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indian%20J.%20Pediatr.&title=Vincristine%20induced%20neurotoxicity%20in%20cancer%20patients&author=S.%20Gomber&author=P.%20Dewan&author=D.%20Chhonker&volume=77&publication_year=2010&pages=97-100&pmid=19936661&doi=10.1007/s12098-009-0254-3&)

4. Lavoie Smith, E.M. et al Patterns and severity of vincristine‐induced peripheral neuropathy in children with acute lymphoblastic leukemia. J. Peripheral Nervous Syst. 20, 37–46 (2015).  [DOI](https://doi.org/10.1111/jns.12114) | [PMC free article](/articles/PMC4610712/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25977177/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Peripheral%20Nervous%20Syst.&title=Patterns%20and%20severity%20of%20vincristine%E2%80%90induced%20peripheral%20neuropathy%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=E.M.%20Lavoie%20Smith&volume=20&publication_year=2015&pages=37-46&pmid=25977177&doi=10.1111/jns.12114&)

5. Mora, E. , Smith, E.M. , Donohoe, C. & Hertz, D.L. Vincristine‐induced peripheral neuropathy in pediatric cancer patients. Am. J. Cancer Res. 6, 2416–2430 (2016).  [PMC free article](/articles/PMC5126263/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27904761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Cancer%20Res.&title=Vincristine%E2%80%90induced%20peripheral%20neuropathy%20in%20pediatric%20cancer%20patients&author=E.%20Mora&author=E.M.%20Smith&author=C.%20Donohoe&author=D.L.%20Hertz&volume=6&publication_year=2016&pages=2416-2430&pmid=27904761&)

6. van de Velde, M.E. , Kaspers, G.L. , Abbink, F.C.H. , Wilhelm, A.J. , Ket, J.C.F. & van den Berg, M.H. Vincristine‐induced peripheral neuropathy in children with cancer: a systematic review. Crit. Rev. Oncol. Hematol. 114, 114–130 (2017).  [DOI](https://doi.org/10.1016/j.critrevonc.2017.04.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28477739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit.%20Rev.%20Oncol.%20Hematol.&title=Vincristine%E2%80%90induced%20peripheral%20neuropathy%20in%20children%20with%20cancer:%20a%20systematic%20review&author=M.E.%20van%20de%20Velde&author=G.L.%20Kaspers&author=F.C.H.%20Abbink&author=A.J.%20Wilhelm&author=J.C.F.%20Ket&volume=114&publication_year=2017&pages=114-130&pmid=28477739&doi=10.1016/j.critrevonc.2017.04.004&)

7. Park, S.B. et al Chemotherapy‐induced peripheral neurotoxicity: a critical analysis. CA Cancer J. Clin. 63, 419–437 (2013).  [DOI](https://doi.org/10.3322/caac.21204) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24590861/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J.%20Clin.&title=Chemotherapy%E2%80%90induced%20peripheral%20neurotoxicity:%20a%20critical%20analysis&author=S.B.%20Park&volume=63&publication_year=2013&pages=419-437&pmid=24590861&doi=10.3322/caac.21204&)

8. Renbarger, J.L. , McCammack, K.C. , Rouse, C.E. & Hall, S.D. Effect of race on vincristine‐associated neurotoxicity in pediatric acute lymphoblastic leukemia patients. Pediatr. Blood Cancer 50, 769–771 (2008).  [DOI](https://doi.org/10.1002/pbc.21435) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18085684/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Blood%20Cancer&title=Effect%20of%20race%20on%20vincristine%E2%80%90associated%20neurotoxicity%20in%20pediatric%20acute%20lymphoblastic%20leukemia%20patients&author=J.L.%20Renbarger&author=K.C.%20McCammack&author=C.E.%20Rouse&author=S.D.%20Hall&volume=50&publication_year=2008&pages=769-771&pmid=18085684&doi=10.1002/pbc.21435&)

9. Kislukhin, G. , Murphy, M.L. , Jafari, M. & Long, A.D. Chemotherapy‐induced toxicity is highly heritable in Drosophila melanogaster. Pharmacogenet. Genomics 22, 285–289 (2012).  [DOI](https://doi.org/10.1097/FPC.0b013e3283514395) | [PMC free article](/articles/PMC3303944/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22336958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Chemotherapy%E2%80%90induced%20toxicity%20is%20highly%20heritable%20in%20Drosophila%20melanogaster&author=G.%20Kislukhin&author=M.L.%20Murphy&author=M.%20Jafari&author=A.D.%20Long&volume=22&publication_year=2012&pages=285-289&pmid=22336958&doi=10.1097/FPC.0b013e3283514395&)

10. Diouf, B. et al Association of an inherited genetic variant with vincristine‐related peripheral neuropathy in children with acute lymphoblastic leukemia. JAMA 313, 815–823 (2015).  [DOI](https://doi.org/10.1001/jama.2015.0894) | [PMC free article](/articles/PMC4377066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25710658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Association%20of%20an%20inherited%20genetic%20variant%20with%20vincristine%E2%80%90related%20peripheral%20neuropathy%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=B.%20Diouf&volume=313&publication_year=2015&pages=815-823&pmid=25710658&doi=10.1001/jama.2015.0894&)

11. Kavallaris, M. Microtubules and resistance to tubulin‐binding agents. Nat. Rev. Cancer 10, 194–204 (2010).  [DOI](https://doi.org/10.1038/nrc2803) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20147901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&title=Microtubules%20and%20resistance%20to%20tubulin%E2%80%90binding%20agents&author=M.%20Kavallaris&volume=10&publication_year=2010&pages=194-204&pmid=20147901&doi=10.1038/nrc2803&)

12. Gutierrez‐Camino, A. et al Lack of association of the CEP72 rs924607 TT genotype with vincristine‐related peripheral neuropathy during the early phase of pediatric acute lymphoblastic leukemia treatment in a Spanish population. Pharmacogenet. Genomics 26, 100–102 (2016).  [DOI](https://doi.org/10.1097/FPC.0000000000000191) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26618658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Lack%20of%20association%20of%20the%20CEP72%20rs924607%20TT%20genotype%20with%20vincristine%E2%80%90related%20peripheral%20neuropathy%20during%20the%20early%20phase%20of%20pediatric%20acute%20lymphoblastic%20leukemia%20treatment%20in%20a%20Spanish%20population&author=A.%20Gutierrez%E2%80%90Camino&volume=26&publication_year=2016&pages=100-102&pmid=26618658&doi=10.1097/FPC.0000000000000191&)

13. Diouf, B. , Crews, K.R. & Evans, W.E. Vincristine pharmacogenomics: ‘winner's curse’ or a different phenotype? Pharmacogenet. Genomics 26, 51–52 (2016).  [DOI](https://doi.org/10.1097/FPC.0000000000000192) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26704670/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Vincristine%20pharmacogenomics:%20%E2%80%98winner's%20curse%E2%80%99%20or%20a%20different%20phenotype?&author=B.%20Diouf&author=K.R.%20Crews&author=W.E.%20Evans&volume=26&publication_year=2016&pages=51-52&pmid=26704670&doi=10.1097/FPC.0000000000000192&)

14. Stock, W. et al An inherited genetic variant in CEP72 promoter predisposes to vincristine‐induced peripheral neuropathy in adults with acute lymphoblastic leukemia. Clin. Pharmacol. Ther. 101, 391–395 (2017).  [DOI](https://doi.org/10.1002/cpt.506) | [PMC free article](/articles/PMC5320866/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27618250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=An%20inherited%20genetic%20variant%20in%20CEP72%20promoter%20predisposes%20to%20vincristine%E2%80%90induced%20peripheral%20neuropathy%20in%20adults%20with%20acute%20lymphoblastic%20leukemia&author=W.%20Stock&volume=101&publication_year=2017&pages=391-395&pmid=27618250&doi=10.1002/cpt.506&)

15. Barwick, K.E. et al Defective presynaptic choline transport underlies hereditary motor neuropathy. Am. J. Hum. Genet. 91, 1103–1107 (2012).  [DOI](https://doi.org/10.1016/j.ajhg.2012.09.019) | [PMC free article](/articles/PMC3516609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23141292/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Defective%20presynaptic%20choline%20transport%20underlies%20hereditary%20motor%20neuropathy&author=K.E.%20Barwick&volume=91&publication_year=2012&pages=1103-1107&pmid=23141292&doi=10.1016/j.ajhg.2012.09.019&)

16. Consortium, G.T. Human genomics The Genotype‐Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Science 348, 648–660 (2015).  [DOI](https://doi.org/10.1126/science.1262110) | [PMC free article](/articles/PMC4547484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25954001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20Genotype%E2%80%90Tissue%20Expression%20(GTEx)%20pilot%20analysis:%20multitissue%20gene%20regulation%20in%20humans&volume=348&publication_year=2015&pages=648-660&pmid=25954001&doi=10.1126/science.1262110&)

17. El Euch‐Fayache, G. , Bouhlal, Y. , Amouri, R. , Feki, M. & Hentati, F. Molecular, clinical and peripheral neuropathy study of Tunisian patients with ataxia with vitamin E deficiency. Brain 137, 402–410 (2014).  [DOI](https://doi.org/10.1093/brain/awt339) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24369383/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain&title=Molecular,%20clinical%20and%20peripheral%20neuropathy%20study%20of%20Tunisian%20patients%20with%20ataxia%20with%20vitamin%20E%20deficiency&author=G.%20El%20Euch%E2%80%90Fayache&author=Y.%20Bouhlal&author=R.%20Amouri&author=M.%20Feki&author=F.%20Hentati&volume=137&publication_year=2014&pages=402-410&pmid=24369383&doi=10.1093/brain/awt339&)

18. Maagdenberg, H. et al Pharmacogenomics in pediatric patients: towards personalized medicine. Paediatr. Drugs 18, 251–260 (2016).  [DOI](https://doi.org/10.1007/s40272-016-0176-2) | [PMC free article](/articles/PMC4920853/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27142473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Paediatr.%20Drugs&title=Pharmacogenomics%20in%20pediatric%20patients:%20towards%20personalized%20medicine&author=H.%20Maagdenberg&volume=18&publication_year=2016&pages=251-260&pmid=27142473&doi=10.1007/s40272-016-0176-2&)

19. Wright, G.E.B. , Carleton, B. , Hayden, M.R. & Ross, C.J.D. The global spectrum of protein‐coding pharmacogenomic diversity. Pharmacogenom. J. 18, 187–195 (2018).  [DOI](https://doi.org/10.1038/tpj.2016.77) | [PMC free article](/articles/PMC5817389/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27779249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom.%20J.&title=The%20global%20spectrum%20of%20protein%E2%80%90coding%20pharmacogenomic%20diversity&author=G.E.B.%20Wright&author=B.%20Carleton&author=M.R.%20Hayden&author=C.J.D.%20Ross&volume=18&publication_year=2018&pages=187-195&pmid=27779249&doi=10.1038/tpj.2016.77&)

20. Conseil, G. & Cole, S.P. Two polymorphic variants of ABCC1 selectively alter drug resistance and inhibitor sensitivity of the multidrug and organic anion transporter multidrug resistance protein 1. Drug Metab. Dispos. 41, 2187–2196 (2013).  [DOI](https://doi.org/10.1124/dmd.113.054213) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24080162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Two%20polymorphic%20variants%20of%20ABCC1%20selectively%20alter%20drug%20resistance%20and%20inhibitor%20sensitivity%20of%20the%20multidrug%20and%20organic%20anion%20transporter%20multidrug%20resistance%20protein%201&author=G.%20Conseil&author=S.P.%20Cole&volume=41&publication_year=2013&pages=2187-2196&pmid=24080162&doi=10.1124/dmd.113.054213&)

21. Winter, S.S. et al ATP Binding Cassette C1 (ABCC1/MRP1)‐mediated drug efflux contributes to disease progression in T‐lineage acute lymphoblastic leukemia. Health (Irvine Calif) 5, 41–50 (2013).  [DOI](https://doi.org/10.4236/health.2013.55A005) | [PMC free article](/articles/PMC3819435/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24224072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Health%20(Irvine%20Calif)&title=ATP%20Binding%20Cassette%20C1%20(ABCC1/MRP1)%E2%80%90mediated%20drug%20efflux%20contributes%20to%20disease%20progression%20in%20T%E2%80%90lineage%20acute%20lymphoblastic%20leukemia&author=S.S.%20Winter&volume=5&publication_year=2013&pages=41-50&pmid=24224072&doi=10.4236/health.2013.55A005&)

22. Kunicka, T. & Soucek, P. Importance of ABCC1 for cancer therapy and prognosis. Drug Metab. Rev. 46, 325–342 (2014).  [DOI](https://doi.org/10.3109/03602532.2014.901348) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24670052/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Rev.&title=Importance%20of%20ABCC1%20for%20cancer%20therapy%20and%20prognosis&author=T.%20Kunicka&author=P.%20Soucek&volume=46&publication_year=2014&pages=325-342&pmid=24670052&doi=10.3109/03602532.2014.901348&)

23. Alharbi, R.A. , Pettengell, R. , Pandha, H.S. & Morgan, R. The role of HOX genes in normal hematopoiesis and acute leukemia. Leukemia 27, 1000–1008 (2013).  [DOI](https://doi.org/10.1038/leu.2012.356) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23212154/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=The%20role%20of%20HOX%20genes%20in%20normal%20hematopoiesis%20and%20acute%20leukemia&author=R.A.%20Alharbi&author=R.%20Pettengell&author=H.S.%20Pandha&author=R.%20Morgan&volume=27&publication_year=2013&pages=1000-1008&pmid=23212154&doi=10.1038/leu.2012.356&)

24. Broyl, A. et al Mechanisms of peripheral neuropathy associated with bortezomib and vincristine in patients with newly diagnosed multiple myeloma: a prospective analysis of data from the HOVON‐65/GMMG‐HD4 trial. Lancet Oncol. 11, 1057–1065 (2010).  [DOI](https://doi.org/10.1016/S1470-2045(10)70206-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20864405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol.&title=Mechanisms%20of%20peripheral%20neuropathy%20associated%20with%20bortezomib%20and%20vincristine%20in%20patients%20with%20newly%20diagnosed%20multiple%20myeloma:%20a%20prospective%20analysis%20of%20data%20from%20the%20HOVON%E2%80%9065/GMMG%E2%80%90HD4%20trial&author=A.%20Broyl&volume=11&publication_year=2010&pages=1057-1065&pmid=20864405&doi=10.1016/S1470-2045(10)70206-0&)

25. Lopez‐Lopez, E. et al Vincristine pharmacokinetics pathway and neurotoxicity during early phases of treatment in pediatric acute lymphoblastic leukemia. Pharmacogenomics 17, 731–741 (2016).  [DOI](https://doi.org/10.2217/pgs-2016-0001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27180762/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Vincristine%20pharmacokinetics%20pathway%20and%20neurotoxicity%20during%20early%20phases%20of%20treatment%20in%20pediatric%20acute%20lymphoblastic%20leukemia&author=E.%20Lopez%E2%80%90Lopez&volume=17&publication_year=2016&pages=731-741&pmid=27180762&doi=10.2217/pgs-2016-0001&)

26. Wheeler, H.E. et al Variants in WFS1 and other Mendelian deafness genes are associated with cisplatin‐associated ototoxicity. Clin. Cancer Res. 23, 3325–3333 (2017).  [DOI](https://doi.org/10.1158/1078-0432.CCR-16-2809) | [PMC free article](/articles/PMC5493516/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28039263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Variants%20in%20WFS1%20and%20other%20Mendelian%20deafness%20genes%20are%20associated%20with%20cisplatin%E2%80%90associated%20ototoxicity&author=H.E.%20Wheeler&volume=23&publication_year=2017&pages=3325-3333&pmid=28039263&doi=10.1158/1078-0432.CCR-16-2809&)

27. Baldwin, R.M. et al A genome‐wide association study identifies novel loci for paclitaxel‐induced sensory peripheral neuropathy in CALGB 40101. Clin. Cancer Res. 18, 5099–5109 (2012).  [DOI](https://doi.org/10.1158/1078-0432.CCR-12-1590) | [PMC free article](/articles/PMC3445665/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22843789/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=A%20genome%E2%80%90wide%20association%20study%20identifies%20novel%20loci%20for%20paclitaxel%E2%80%90induced%20sensory%20peripheral%20neuropathy%20in%20CALGB%2040101&author=R.M.%20Baldwin&volume=18&publication_year=2012&pages=5099-5109&pmid=22843789&doi=10.1158/1078-0432.CCR-12-1590&)

28. Hertz, D.L. et al Pharmacogenetic discovery in CALGB (Alliance) 90401 and mechanistic validation of a VAC14 polymorphism that increases risk of docetaxel‐induced neuropathy. Clin. Cancer Res. 22, 4890–4900 (2016).  [DOI](https://doi.org/10.1158/1078-0432.CCR-15-2823) | [PMC free article](/articles/PMC5050068/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27143689/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Pharmacogenetic%20discovery%20in%20CALGB%20(Alliance)%2090401%20and%20mechanistic%20validation%20of%20a%20VAC14%20polymorphism%20that%20increases%20risk%20of%20docetaxel%E2%80%90induced%20neuropathy&author=D.L.%20Hertz&volume=22&publication_year=2016&pages=4890-4900&pmid=27143689&doi=10.1158/1078-0432.CCR-15-2823&)

29. Ichikawa, M. et al Successful alternative treatment containing vindesine for acute lymphoblastic leukemia with Charcot‐Marie‐Tooth disease. J. Pediatr. Hematol. Oncol. 34, 239–241 (2012).  [DOI](https://doi.org/10.1097/MPH.0b013e3182352cf5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22246157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pediatr.%20Hematol.%20Oncol.&title=Successful%20alternative%20treatment%20containing%20vindesine%20for%20acute%20lymphoblastic%20leukemia%20with%20Charcot%E2%80%90Marie%E2%80%90Tooth%20disease&author=M.%20Ichikawa&volume=34&publication_year=2012&pages=239-241&pmid=22246157&doi=10.1097/MPH.0b013e3182352cf5&)

30. Schmolz, L. , Birringer, M. , Lorkowski, S. & Wallert, M. Complexity of vitamin E metabolism. World J. Biol. Chem. 7, 14–43 (2016).  [DOI](https://doi.org/10.4331/wjbc.v7.i1.14) | [PMC free article](/articles/PMC4768118/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26981194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J.%20Biol.%20Chem.&title=Complexity%20of%20vitamin%20E%20metabolism&author=L.%20Schmolz&author=M.%20Birringer&author=S.%20Lorkowski&author=M.%20Wallert&volume=7&publication_year=2016&pages=14-43&pmid=26981194&doi=10.4331/wjbc.v7.i1.14&)

31. Vainionpaa, L. , Kovala, T. , Tolonen, U. & Lanning, M. Vincristine therapy for children with acute lymphoblastic leukemia impairs conduction in the entire peripheral nerve. Pediatr. Neurol. 13, 314–318 (1995).  [DOI](https://doi.org/10.1016/0887-8994(95)00191-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8771166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Neurol.&title=Vincristine%20therapy%20for%20children%20with%20acute%20lymphoblastic%20leukemia%20impairs%20conduction%20in%20the%20entire%20peripheral%20nerve&author=L.%20Vainionpaa&author=T.%20Kovala&author=U.%20Tolonen&author=M.%20Lanning&volume=13&publication_year=1995&pages=314-318&pmid=8771166&doi=10.1016/0887-8994(95)00191-3&)

32. Rosson, G.D. , Larson, A.R. , Williams, E.H. & Dellon, A.L. Tibial nerve decompression in patients with tarsal tunnel syndrome: pressures in the tarsal, medial plantar, and lateral plantar tunnels. Plast. Reconstr. Surg. 124, 1202–1210 (2009).  [DOI](https://doi.org/10.1097/PRS.0b013e3181b5a3c3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19935304/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plast.%20Reconstr.%20Surg.&title=Tibial%20nerve%20decompression%20in%20patients%20with%20tarsal%20tunnel%20syndrome:%20pressures%20in%20the%20tarsal,%20medial%20plantar,%20and%20lateral%20plantar%20tunnels&author=G.D.%20Rosson&author=A.R.%20Larson&author=E.H.%20Williams&author=A.L.%20Dellon&volume=124&publication_year=2009&pages=1202-1210&pmid=19935304&doi=10.1097/PRS.0b013e3181b5a3c3&)

33. Bauche, S. et al Impaired presynaptic high‐affinity choline transporter causes a congenital myasthenic syndrome with episodic apnea. Am. J. Hum. Genet. 99, 753–761 (2016).  [DOI](https://doi.org/10.1016/j.ajhg.2016.06.033) | [PMC free article](/articles/PMC5011057/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27569547/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Impaired%20presynaptic%20high%E2%80%90affinity%20choline%20transporter%20causes%20a%20congenital%20myasthenic%20syndrome%20with%20episodic%20apnea&author=S.%20Bauche&volume=99&publication_year=2016&pages=753-761&pmid=27569547&doi=10.1016/j.ajhg.2016.06.033&)

34. Carleton, B. et al Adverse drug reaction active surveillance: developing a national network in Canada's children's hospitals. Pharmacoepidemiol. Drug Saf. 18, 713–721 (2009).  [DOI](https://doi.org/10.1002/pds.1772) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19507171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacoepidemiol.%20Drug%20Saf.&title=Adverse%20drug%20reaction%20active%20surveillance:%20developing%20a%20national%20network%20in%20Canada's%20children's%20hospitals&author=B.%20Carleton&volume=18&publication_year=2009&pages=713-721&pmid=19507171&doi=10.1002/pds.1772&)

35. Flockhart, D.A . Flockhart Table™ P450 Drug Interaction Table. <http://medicine.iupui.edu/clinpharm/ddis/main-table/> (2016). Accessed 5 July 2018.  [http://medicine.iupui.edu/clinpharm/ddis/main-table/](http://medicine.iupui.edu/clinpharm/ddis/main-table/)

36. Drogemoller, B.I. et al Association between SLC16A5 genetic variation and cisplatin‐induced ototoxic effects in adult patients with testicular cancer. JAMA Oncol. 3, 1558–1562 (2017).  [DOI](https://doi.org/10.1001/jamaoncol.2017.0502) | [PMC free article](/articles/PMC5824214/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28448657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Oncol.&title=Association%20between%20SLC16A5%20genetic%20variation%20and%20cisplatin%E2%80%90induced%20ototoxic%20effects%20in%20adult%20patients%20with%20testicular%20cancer&author=B.I.%20Drogemoller&volume=3&publication_year=2017&pages=1558-1562&pmid=28448657&doi=10.1001/jamaoncol.2017.0502&)

37. Genomes Project, C. et al A global reference for human genetic variation. Nature 526, 68–74 (2015).  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation&volume=526&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

38. Magi, R. & Morris, A.P. GWAMA: software for genome‐wide association meta‐analysis. BMC Bioinformat. 11, 288 (2010).  [DOI](https://doi.org/10.1186/1471-2105-11-288) | [PMC free article](/articles/PMC2893603/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20509871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformat.&title=GWAMA:%20software%20for%20genome%E2%80%90wide%20association%20meta%E2%80%90analysis&author=R.%20Magi&author=A.P.%20Morris&volume=11&publication_year=2010&pages=288&pmid=20509871&doi=10.1186/1471-2105-11-288&)

39. McLaren, W. et al The Ensembl variant effect predictor. Genome Biol. 17, 122 (2016).  [DOI](https://doi.org/10.1186/s13059-016-0974-4) | [PMC free article](/articles/PMC4893825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27268795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&title=The%20Ensembl%20variant%20effect%20predictor&author=W.%20McLaren&volume=17&publication_year=2016&pages=122&pmid=27268795&doi=10.1186/s13059-016-0974-4&)

40. Ward, L.D. & Kellis, M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Nucleic Acids Res. 40, D930–D934 (2012).  [DOI](https://doi.org/10.1093/nar/gkr917) | [PMC free article](/articles/PMC3245002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22064851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=HaploReg:%20a%20resource%20for%20exploring%20chromatin%20states,%20conservation,%20and%20regulatory%20motif%20alterations%20within%20sets%20of%20genetically%20linked%20variants&author=L.D.%20Ward&author=M.%20Kellis&volume=40&publication_year=2012&pages=D930-D934&pmid=22064851&doi=10.1093/nar/gkr917&)

41. Mathelier, A. et al JASPAR 2016: a major expansion and update of the open‐access database of transcription factor binding profiles. Nucleic Acids Res. 44, D110–D115 (2016).  [DOI](https://doi.org/10.1093/nar/gkv1176) | [PMC free article](/articles/PMC4702842/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26531826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=JASPAR%202016:%20a%20major%20expansion%20and%20update%20of%20the%20open%E2%80%90access%20database%20of%20transcription%20factor%20binding%20profiles&author=A.%20Mathelier&volume=44&publication_year=2016&pages=D110-D115&pmid=26531826&doi=10.1093/nar/gkv1176&)

42. Barbeira, A.N. et al Exploring the phenotypic consequences of tissue specific gene expression variation inferred from GWAS summary statistics. Nat. Commun. 9, 1825 (2018).  [DOI](https://doi.org/10.1038/s41467-018-03621-1) | [PMC free article](/articles/PMC5940825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29739930/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Exploring%20the%20phenotypic%20consequences%20of%20tissue%20specific%20gene%20expression%20variation%20inferred%20from%20GWAS%20summary%20statistics&author=A.N.%20Barbeira&volume=9&publication_year=2018&pages=1825&pmid=29739930&doi=10.1038/s41467-018-03621-1&)
