# Effects of the Human SULT1A1 Polymorphisms on the Sulfation of Acetaminophen, O-Desmethylnaproxen, and Tapentadol

## Metadata
**Authors:** Mohammed I Rasool, Ahsan F Bairam, Saud A Gohal, Amal A El Daibani, Fatemah A Alherz, Maryam S Abunnaja, Eid S Alatwi, Katsuhisa Kurogi, Ming-Cheh Liu
**Journal:** Pharmacological reports : PR
**Date:** 2018 Dec 10
**DOI:** [10.1016/j.pharep.2018.12.001](https://doi.org/10.1016/j.pharep.2018.12.001)
**PMID:** 30822619
**PMCID:** PMC6498441
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6498441/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6498441/pdf/nihms-1522689.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6498441/pdf/nihms-1522689.pdf)

## Abstract

**Background:** 
Non-opioid and opioid analgesics, as over-the-counter or prescribed medications, are widely used for the management of a diverse array of pathophysiological conditions. Previous studies have demonstrated the involvement of human cytosolic sulfotransferase (SULT) SULT1A1 in the sulfation of acetaminophen, O-desmethylnaproxen (O-DMN), and tapentadol. The current study was designed to investigate the impact of single nucleotide polymorphisms (SNPs) of the human SULT1A1 gene on the sulfation of these analgesic compounds by SULT1A1 allozymes.

**Methods:** 
Human SULT1A1 genotypes were identified by database search. cDNAs corresponding to nine SULT1A1 nonsynonymous missense coding SNPs (cSNPs) were generated by site-directed mutagenesis. Recombinant wild-type and SULT1A1 allozymes were bacterially expressed and affinity-purified. Purified SULT1A1 allozymes were analyzed for sulfation activity using an established assay procedure.

**Results:** 
Compared with the wild-type enzyme, SULT1A1 allozymes were shown to display differential sulfating activities toward three analgesic compounds, acetaminophen, O-desmethylnaproxen (O-DMN), and tapentadol, as well as the prototype substrate 4NP.

**Conclusion:** 
Results obtained indicated clearly the impact of genetic polymorphisms on the drug-sulfation activity of SULT1A1 allozymes. Such information may contribute to a better understanding about the differential metabolism of acetaminophen, O-DMN, and tapentadol in individuals with different SULT1A1 genotypes.

### Background

Non-opioid and opioid analgesics, as over-the-counter or prescribed medications, are widely used for the management of a diverse array of pathophysiological conditions. Previous studies have demonstrated the involvement of human cytosolic sulfotransferase (SULT) SULT1A1 in the sulfation of acetaminophen, *O*-desmethylnaproxen (*O*-DMN), and tapentadol. The current study was designed to investigate the impact of single nucleotide polymorphisms (SNPs) of the human *SULT1A1* gene on the sulfation of these analgesic compounds by SULT1A1 allozymes.

### Methods

Human *SULT1A1* genotypes were identified by database search. cDNAs corresponding to nine *SULT1A1* nonsynonymous missense coding SNPs (cSNPs) were generated by site-directed mutagenesis. Recombinant wild-type and SULT1A1 allozymes were bacterially expressed and affinity-purified. Purified SULT1A1 allozymes were analyzed for sulfation activity using an established assay procedure.

### Results

Compared with the wild-type enzyme, SULT1A1 allozymes were shown to display differential sulfating activities toward three analgesic compounds, acetaminophen, *O*-desmethylnaproxen (*O*-DMN), and tapentadol, as well as the prototype substrate 4NP.

### Conclusion

Results obtained indicated clearly the impact of genetic polymorphisms on the drug-sulfation activity of SULT1A1 allozymes. Such information may contribute to a better understanding about the differential metabolism of acetaminophen, *O*-DMN, and tapentadol in individuals with different *SULT1A1* genotypes.

## Introduction

The non-opiate analgesics, including acetaminophen and nonsteroidal anti-inflammatory drugs (NSAIDs) such as naproxen, are among the most commonly used analgesics [[1](#R1)]. These analgesics along with opioids such as tapentadol are widely used in the management of different levels of pain with a stepwise progression based on pain intensity [[2](#R2)–[4](#R4)]. Combinations of analgesic drugs are commonly used for the clinical management of acute and chronic pain either as over the counter or prescribed medications [[5](#R5)]. Acetaminophen occupies a unique position among analgesic drugs, both for the type of pain relieved and the associated adverse effects [[1](#R1)]. Among the NSAIDs, naproxen is one of the oldest and most widely used drug due to its antipyretic, anti-inflammatory, and analgesic properties [[2](#R2)]. Tapentadol, on the other hand, is a novel and efficient centrally-acting analgesic, prescribed mostly for the management of moderate to severe chronic pain [[3](#R3)].

In adults, the primary metabolic pathways of acetaminophen are the phase II glucuronidation and sulfoconjugation reactions, producing inactive and harmless metabolites. In premature infants, newborns, and young infants, the primary metabolic pathway of acetaminophen has been shown to be the sulfoconjugation due to the low levels of UDP-glucuronosyltransferases enzymes (UGTs) [[1](#R1), [6](#R6)]. Sulfoconjugation has also been reported to play a quantitatively more important role in tapentadol metabolism during the prenatal and neonatal stages than in adults [[7](#R7)]. Both glucuronide and sulfate conjugates of tapentadol have been shown to lack the analgesic activity [[8](#R8), [9](#R9)]. For naproxen, *O*-desmethylnaproxen (*O*-DMN) has been shown to be a major phase I metabolite, which is further biotransformed by phase II glucuronidation and sulfoconjugation reactions [[10](#R10)]. These latter conjugates are primarily excreted in the urine as inactive metabolites [[11](#R11)].

Sulfation, as mediated by the cytosolic sulfotransferase (SULT), is increasingly being recognized as a metabolic reaction which is of fundamental importance in the modulation of endogenous compounds as well as the bioinactivation of a variety of xenobiotics [[12](#R12)–[15](#R15)]. Among the thirteen known human SULTs, SULT1A1, originally referred to as the thermostable phenol sulfotransferase, has been shown to be the primary enzyme responsible for the sulfation of acetaminophen [[16](#R16), [17](#R17)] and *O*-DMN [[18](#R18), [19](#R19)]. SULT1A1 has also been shown to be a major enzyme responsible for the sulfation of tapentadol [[20](#R20)]. The gene coding for *SULT1A1* has been shown to be located in the short arm of chromosome 16 [[21](#R21), [22](#R22),[23](#R23)–[25](#R25)]. Genomic studies have revealed single nucleotide polymorphisms (SNPs) of *SULT1A1* [[22](#R22)–[24](#R24), [26](#R26)]. Such SNPs might result in amino acid substitutions that have the potential to dramatically influence protein function [[26](#R26)]. It is an intriguing question whether the missense coding SNPs (cSNPs) of *SULT1A1* may lead to SULT1A1 allozymes with differential sulfation activity toward acetaminophen, *O*-DMN, and tapentadol, thereby impacting their pharmacokinetics and thus their efficacy as well as adverse effects in individuals with distinct *SULT1A1* genotypes.

In this communication, we report a systematic survey of *SULT1A1* SNPs deposited in several major SNP databases. Nine missense cSNPs were selected, and the corresponding SULT1A1 allozymes were generated, expressed and purified. Purified SULT1A1 allozymes were enzymatically characterized using acetaminophen, *O*-DMN, and tapentadol as substrates.

## Materials and Methods

### Materials

Acetaminophen (≥ 99% in purity), 4-nitrophenol (4NP) (≥ 99% in purity), dimethyl sulfoxide (DMSO), isopropyl β-D-1-thiogalactopyranoside (IPTG), adenosine 5′-triphosphate (ATP), N-2-hydroxylpiperazine-N′-2-ethanesulfonic acid (HEPES), and dithiothreitol (DTT) were products of Sigma-Aldrich (St. Louis, MO, USA). Tapentadol (≥ 98% in purity) was purchased from Cayman Chemical Company (Ann Arbor, MI, USA). *O*-DMN (≥ 98% in purity) was a product of Toronto Research Chemicals (Toronto, ON, Canada). Carrier-free sodium [^35^S] sulfate was from American Radiolabeled Chemicals (St. Louis, MO, USA). Recombinant human bifunctional PAPS synthase was prepared as described previously [[27](#R27)]. Polygram® Cellulose 300 thin-layer chromatography (TLC) plates were from Macherey-Nagel GmbH and Co. KG (Düren, Germany). PrimeSTAR® Max DNA polymerase was a product of Takara Bio (Mountain View, CA, USA). Protein molecular weight markers were from Bioland Scientific LLC (Paramount, CA, USA). Ecolume scintillation cocktail was purchased from MP Biomedicals, LLC (Irvine, CA, USA). X-Ray films were products of FUJIFILM Corporation (Tokyo, Japan). Oligonucleotide primers were custom synthesized by Eurofins Genomics (Louisville, KY, USA). QIAprep® Spin Miniprep Kit was a product of QIAGEN (Düren, Germany). All other chemicals were of the highest standards commercially available.

### Identification and analysis of human SULT1A1 SNPs

A comprehensive search was performed for *SULT1A1* SNPs deposited in three online databases located at the websites of, respectively, UniProt Knowledgebase (UniProtKB), the U.S. National Center for Biotechnology Information (NCBI), and the Ensembl Variation database. Additionally, the NCBI PubMed was searched for previous studies describing *SULT1A1* SNPs.

### Generation of cDNAs encoding selected human SULT1A1 allozymes

Site-directed mutagenesis was performed to generate specific “mutations” in the nucleotide sequence of SULT1A1 cDNA based on a previously described procedure [[28](#R28)]. Briefly, wild-type SULT1A1 cDNA packaged in pGEX-2TK prokaryotic expression vector was utilized as the template in conjunction with specific mutagenic primers (cf. [Table 1](#T1)) to amplify “mutated” SULT1A1 cDNAs coding for different SULT1A1 allozymes. The PCR amplification conditions were 12 cycles of 30 s at 94°C, 1 min at 55°C, and 7 min at 72°C. Subsequently, the plasmids that carry the mutated SULT1A1 cDNAs were individually transformed into NEB 5-α *E. coli* competent cells. Colony PCR followed by agarose gel electrophoresis was carried out to identify positive colonies. Positive colonies were grown for plasmid extraction utilizing a Miniprep Kit. Nucleotide sequencing was performed to verify the sequences of the “mutated” SULT1A1 cDNAs [[29](#R29)] before using them for protein expression.

### Table 1: List of selected single nucleotide polymorphisms with their minor allele frequencies and designed mutagenic primers of the human SULT1A1 genes.

| hSULT1A 1 allozyme | Mutagenic primer | MAF* |
| --- | --- | --- |
| hSULT1A1 - R37Q | 5′-TGCAGAGCTTCCAGGCCCAGCCTGATGACCTGCTCAT-3′5′-AT GAGCAGGTCATCAGGCTGGGCCTGGAAGCTCTGCA-3′ | 0.0004 |
| hSULT1A1-P47S | 5′-CTGCTCATCAGCACCTACTCCAAGTCCGGCACTACCT-3′5′-AGGT AGT GCCGGACTTGGAGT AGGT GCTGATGAGCAG-3′ | 0.0002 |
| hSULT1A1-M77I | 5′-CCGAGCTCCCATCTTCATACGGGTGCCCTTCCTTGAG-3′5′-CTCAAGGAAGGGCACCCGTATGAAGATGGGAGCTCGG-3′ | 0.000008 |
| hSULT1A1-H149Y | 5′-TACCACATGGCCAAGGTGTACCCTGAGCCTGGGACCT-3′5′-AGGTCCCAGGCTCAGGGTACACCTTGGCCATGTGGTA-3′ | 0.000008 |
| hSULT1A1-Y169D | 5′-ATGGTCGGAGAAGTGTCCGACGGATCCTGGTACCAGC-3′5′-GCTGGTACCAGGATCCGTCGGACACTTCTCCGACCAT-3′ | 0.000008 |
| hSULT1A1 - R213H | 5′-TCCTGGAGTTTGTGGGGCACTCCCTGCCAGAGGAGAC-3′5′-GTCTCCTCTGGCAGGGAGTGCCCCACAAACTCCAGGA-3′ | 0.2401 |
| hSULT1A1-T227P | 5′-GACTTCGTGGTTCAGCACCCGTCGTTCAAGGAGATGA-3′5′-TCATCTCCTTGAACGACGGGTGCTGAACCACGAAGTC-3′ | 0.0006 |
| hSULT1A1-V243D | 5′-TGACCAACTACACCACCGACCCCCAGGAGTTCATGGA-3′5′-TCCATGAACTCCTGGGGGTCGGTGGTGTAGTTGGTCA-3′ | 0.00002 |
| hSULT1A1 - F247L | 5′-ACCACCGTCCCCCAGGAGCTCATGGACCACAGCATCT-3′5′-AGATGCTGTGGTCCATGAGCTCCTGGGGGACGGTGGT-3′ | 0.0252 |

Table 1 Caption: The bold underlined letters indicate altered/mutated nucleotides (SNPs).

### Expression and purification of recombinant SULT1A1 allozymes

pGEX-2TK vector harboring individual mutated SULT1A1 cDNA was transformed into BL21 *E. coli* competent cells for protein expression. Transformed BL21 cells were induced with IPTG for recombinant protein expression. Induced cells were collected and homogenized using an Aminco French press. Affinity chromatography using glutathione-Sepharose was performed, based on a previously established procedure [[30](#R30)], followed by thrombin digestion to release the recombinant SULT1A1 allozyme. The purity of the SULT1A1 allozyme thus prepared was examined using SDS-polyacrylamide gel electrophoresis (SDS-PAGE) [[31](#R31), [32](#R32)]. The protein concentration of the purified SULT1A1 allozymes was determined based on Bradford protein assay [[33](#R33)].

### Sulfotransferase assay

The sulfation activity of purified recombinant human SULT1A1 allozymes toward 4NP, acetaminophen, *O*-DMN, and tapentadol was assayed based on a previously established protocol, with radioactive PAP[^35^S] as the sulfate donor [[34](#R34)]. Briefly, the enzymatic assay was conducted at pH 7.4 and incubated at 37°C. After a 10 min reaction, the assay mixture was terminated by heating for 3 min at 100°C, followed by TLC using a solvent system containing n-butanol: isopropanol: formic acid: water in a ratio of 3:1:1:1 (by volume) for sulfated 4NP, acetaminophen, and *O*-DMN, or 2:1:1:2 (by volume) for sulfated tapentadol. Upon completion of TLC, autoradiography was performed using an X-ray film to locate the radioactive spots corresponding to [^35^S] sulfated products. The radioactive spots located were cut out and eluted for the quantitative measurement of [^35^S]-radioactivity using a liquid scintillation counter. The specific activity of the SULT1A1 allozymes was calculated in the unit of nmol of sulfated product produced/min/mg of enzyme. For the kinetic studies, varying substrate concentrations, ranging from 0-100 μM for 4NP, 0-4000 μM for acetaminophen, 0-400 μM for *O*-DMN, and 0-500 μM for tapentadol, were tested. All assays were performed in triplicates and the data obtained were processed as described below.

### Data analysis

The data obtained from the kinetic experiments were analyzed based on the non-linear regression of the Michaelis-Menten equation using the GraphPad Prism® 7 software. One-way ANOVA was performed in conjunction with Dunnett’s multiple comparison test to find data means that are significantly different from SULT1A1 wild-type enzyme.

## Results

### Identification and categorization of human SULT1A1 SNPs

Three online databases including the NCBI, the Ensembl Variation database, and the UniProt database, as well as relevant research articles, were searched for *SULT1A1* SNPs. In total, 5765 distinct *SULT1A1* SNPs were identified. The identified *SULT1A1* SNPs were analyzed and categorized into coding including (synonymous, non-synonymous (missense), and nonsense SNPs) and non-coding SNPs including (introns, 3’-untranslated regions (3’UTR), and 5′-untranslated regions (5′UTR) SNPs) (cf. [Table 2](#T2)). The missense coding SNPs (cSNPs) identified were analyzed for locations of the amino acid changes being in the phosphosulfate binding (PSB) loop or PAPS-binding sites, substrate and/or catalysis binding sites [[25](#R25), [35](#R35)], and dimerization motif [[36](#R36), [37](#R37)]. Additionally, the impact in terms of the chemical properties of the altered amino acid residues, such as non-turn-inducing to (or from) turn-inducing residues, charged to (or from) uncharged residues, basic to (or from) acidic residues), was scrutinized. Based on these criteria, nine missense *SULT1A1* cSNPs were chosen for further studies. The designated names and SNP ID numbers of these nine cSNPs are: SULT1A1-R37Q (Reference SNP (rs) rs72547527), SULT1A1-P47S (rs544820732), SULT1A1-M77I (rs765399160), SULT1A1-H149Y (rs1042008), SULT1A1-Y169D (rs767487725), SULT1A1-R213H (rs9282861), SULT1A1-T227P (rs552524124), SULT1A1-V243D (rs758145522), and SULT1A1-F247L (rs28374453). [Table 1](#T1) shows these nine *SULT1A1* cSNPs, their amino acid changes and locations, the documented allele frequency of individual *SULT1A1* genotypes, as well as sense and antisense mutagenic primers designed for use in the PCR-amplification of the corresponding SULT1A1 cDNAs. The previously reported amino acids sequences/residues that are involved in the PAPS-binding, substrate-binding, catalysis, and dimerization, together with the locations of amino acid variations associated with the selected *SULT1A1* cSNPs are depicted in [Figure 1](#F1). Moreover, the locations of amino acid variations associated with above-mentioned cSNPs are illustrated in the crystal structures of SULT1A3 (Protein Data Bank code: 1LS6 [[35](#R35)]) ([Figure S1](#SD1)). The cDNAs corresponding to the nine selected *SULT1A1* missense cSNPs were generated and the recombinant SULT1A1 enzymes were expressed and purified as described in the [Materials and Methods](#S6) section. As shown in [Figure 2](#F2), the SDS-PAGE electrophoretic pattern of the purified SULT1A1 allozymes revealed that the apparent molecular weights of all nine SULT1A1 allozymes were comparable to the predicted molecular weight (34.165 kDa) of the wild-type SULT1A1.

### Table 2. List of single nucleotide polymorphisms in human SULT1A1 genes.

| SNP1 location | SULT1A1 |
| --- | --- |
| 3′-untranslated regions (3′-UTR) | 85 |
| 5′-untranslated regions (5′-UTR) | 527 |
| Intron regions | 4510 |
| Synonymous coding SNPs | 198 |
| Nonsense coding SNP | 22 |
| Missense coding SNPs | 423 |
| Total | 5765 |

Table 2 Caption: 1SNP refers to single nucleotide polymorphism.

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f8dc/6498441/6f6db2c3d922/nihms-1522689-f0001.jpg)

Amino acid sequence of human SULT1A1 marked with residues/regions reported to be critical to PAPS-binding, substrate-binding, dimerization, and/or catalysis. Residues circled with a white background are those involved in PAPS-binding. Residues enclosed in diamond symbols are those involved in catalysis. Residues circled with light gray background refer to the dimerization-motif. Residues enclosed in square symbols are those involved in substrate-binding. Residues Met77, His149, Tyr169, Val243, and Phe247 are part of the L-shaped substrate-binding pocket. Residues circled with black background refer to the locations of amino acid substitutions associated with the SULT1A1 allozymes analyzed. Residues circled with dark gray background represent the substituting amino acid residues. The figure was generated using Protter, a web tool for interactive protein feature visualization [54].

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f8dc/6498441/fb0e6c4a6d8f/nihms-1522689-f0002.jpg)

SDS gel electrophoretic pattern of the purified human SULT1A1 allozymes. SDS-PAGE was performed on a 12% gel, followed by Coomassie Blue staining. Samples analyzed in lanes 1 through 10 correspond to SULT1A1-wild-type, SULT1A1-P47S, SULT1A1-M77I, SULT1A1-H149Y, SULT1A1-Y169D, SULT1A1-T227P, SULT1A1-V243D, SULT1A1-R37Q, SULT1A1-R213H, and SULT1A1-F247L. Positions of protein molecular weight markers co-electrophoresed are indicated on the right.

### Sulfation of 4NP, acetaminophen, O-DMN, and tapentadol by the wild-type SULT1A1

Sulfating properties of the wild-type SULT1A1 were first analyzed using 4NP, acetaminophen, *O*-DMN, and tapentadol as substrates. [Figure 3](#F3) shows the concentration-dependent sulfation of each of the four substrates by the wild-type SULT1A1. Data obtained were processed using GraphPad Prism® 7 software for the determination of kinetic constants, *Km, Vmax*, and *Vmax/Km*. As shown in [Table 3](#T3), the *Km* value (2.67 ± 0.12 μM) determined for the wild-type SULT1A1 enzyme with the prototypic substrate 4NP was lower than those (394.20 ± 17.82, 8.25 ± 0.67 and 92.54 ± 4.13 μM) determined with acetaminophen, *O*-DMN, and tapentadol. The *Km* with *O*-DMN (8.25 ± 0.67 μM) was more than three times higher than that with 4NP. Much higher *Km* values (394.20 ± 17.82 and 92.54 ± 4.13 μM) were found with acetaminophen and tapentadol, being approximately 147 and 34 times that of 4NP. Regarding the *Vmax*, the value determined with acetaminophen (21.58 ± 0.29) was found to be comparable with that with 4NP (21.76±0.83 nmol/min/mg). A slightly higher *Vmax* value (27.73 ± 0.51 nmol/min/mg) was found with *O*-DMN, whereas a more than 90% lower *Vmax* value (1.14 ± 0.01 nmol/min/mg) was found with tapentadol. Based on these data, the catalytic efficiency as reflected by *Vmax/Km* appeared to be lower with acetaminophen, *O*-DMN, and tapentadol than with 4NP.

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f8dc/6498441/0ea4e34bea55/nihms-1522689-f0003.jpg)

Kinetic analysis of the sulfation of the tested substrates by wild-type human SULT1A1.(A), (B), (C), and (D) show the nonlinear Michaelis–Menten curves for the sulfation of, respectively, 4NP, acetaminophen, O-DMN, and tapentadol. The nonlinear curves were generated using GraphPad Prism® 7 software. Arrow signs indicate the concentrations at which substrate inhibition started taking place. Data shown represent calculated mean ± standard deviation derived from three experiments.

### Table 3.

| Substrate | Km (μM) | Vmax (nmol/min/mg) | Vmax/Km (ml/min/mg) |
| --- | --- | --- | --- |
| 4NP | 2.67 ± 0.12 | 21.76 ± 0.83 | 8.15 |
| Acetaminophen | 394.20 ± 17.82 | 21.58 ± 0.29 | 0.05 |
| O-DMN | 8.25 ± 0.67 | 27.73 ± 0.51 | 3.36 |
| Tapentadol | 92.54 ± 4.13 | 1.14 ± 0.01 | 0.01 |

Table 3 Caption: Kinetic parameters of the human SULT1A1 wild type with 4NP, acetaminophen, O-DMN, and tapentadol as substrates.

### Enzymatic characterization of the SULT1A1 allozymes

Purified SULT1A1 allozymes were analyzed, in comparison with the wild-type enzyme, for their sulfation activity with 4NP, acetaminophen, *O*-DMN, and tapentadol as substrates. Based on the results from the above-mentioned studies with the wild-type SULT1A1, three different concentrations (one much below *Km*, one close to *Km*, and one well above *Km*) of each of the four substrates were tested. [Figures 4](#F4)–[7](#F7) show the results obtained from these comparative studies.

### Figure 4.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f8dc/6498441/63dd85332eb4/nihms-1522689-f0004.jpg)

Specific activities of the sulfation of 4NP by human SULT1A1 allozymes. The concentrations of 4NP used in the enzymatic assays were 0.2 μM (A), 2 μM (B), and 20 μM (C). Data shown represent mean ± standard deviation derived from three determinations. One-way ANOVA was performed in combination with Dunnett’s multiple comparison test (statistically significant from SULTlAl-wild-type: *p<0.05, **p<0.01, ***p<0.001, and ****p<0.0001).

### Figure 7.

![Figure 7.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f8dc/6498441/9c265521c732/nihms-1522689-f0007.jpg)

Specific activities of the sulfation of tapentadol by human SULT1A1 allozymes. The concentrations of tapentadol used in the enzymatic assays were 5 μM (A), 100 μM (B), and 500 μM (C). Data shown represent mean ± standard deviation derived from three determinations. One-way ANOVA was performed in combination with Dunnett’s multiple comparison test (statistically significant from SULTlAl-wild-type: *p<0.05, **p<0.01, ***p<0.001, and ****p<0.0001).

### 4NP as the substrate

As shown in [Figure 4A](#F4) and [4B](#F4), at low substrate concentrations (0.2 and 2 μM), all SULT1A1 allozymes displayed sulfating activities comparable to that of the wild-type enzyme, except SULT1A1-T227P which exhibited a dramatically lower activity (less than 5% of that of wild-type SULT1A1). At high substrate concentration (20 μM), SULT1A1-Y169D showed a sulfation activity comparable to that of the wild-type enzyme, while SULT1A1-M77I and SULT1A1-F247L displayed higher sulfating activities than that of the wild-type. In contrast, the other six SULT1A1 allozymes (SULT-R37Q, SULT-P47S, SULT-H149Y, SULT-R213H, SULT-T227P, and SULT-V243D) all showed significantly lower sulfating activities, with SULT1A1-T227P again displaying the lowest specific activity that was only about 2.8% of that of the wild-type enzyme ([Figure 4C](#F4)).

### Acetaminophen as the substrate

At all three substrate concentrations (40, 400, and 1000 μM), similar patterns of differential acetaminophen-sulfating activities were found for the SULT1A1 allozymes tested ([Figure 5](#F5)). Of the nine human SULT1A1 allozymes, seven (SULT1A1-R37Q, SULT1A1-P47S, SULT1A1-H149Y, SULT1A1-Y169D, SULT1A1-R213H, SULT1A1-T227P, and SULT1A1-V243D) consistently exhibited lower sulfating activities than the wild-type enzyme. Among them, SULT1A1-T227P allozyme displayed barely detectable activities with all the three substrates concentrations tested. Of the remaining six allozymes, two (SULT1A1-R37Q and SULT1A1-R213H) showed sulfating activities that were more than 13% lower and four (SULT1A1-P47S, SULT1A1-H149Y, SULT1A1-Y169D, and SULT1A1-V243D) showed sulfating activities that were greater than 77% lower than the wild-type enzyme. In contrast, SULT1A1-M77I and SULT1A1-F247L displayed sulfating activities significantly higher that of the wild-type enzyme.

### Figure 5.

![Figure 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f8dc/6498441/20e23d995f63/nihms-1522689-f0005.jpg)

Specific activities of the sulfation of acetaminohen by human SULT1A1 allozymes. The concentrations of acetaminophen used in the enzymatic assays were 40 μM (A), 400 μM (A), and 1000 μM (C). Data shown represent mean ± standard deviation derived from three determinations. One-way ANOVA was performed in combination with Dunnett’s multiple comparison test (statistically significant from SULT1A1-wild-type: *p<0.05, **p<0.01, ***p<0.001, and ****p<0.0001).

### O-DMN as the substrate

Like with acetaminophen, similar patterns of differential *O*-DMN-sulfating activities were found for the SULT1A1 allozymes tested at all three substrate concentrations ([Figure 6](#F6)). Among the nine allozymes, SULT1A1-F247L displayed higher sulfation activity than the wild-type enzyme by about 33%. SULT1A1-M77I, on the other hand, exhibited sulfating activities that were moderately higher (with 4%, 8%, and 0.12% increase at, respectively, 1, 10, and 100 μM) than the wild-type. The other seven SULT1A1 allozymes (SULT1A1-R37Q, SULT1A1-P47S, SULT1A1-H149Y, SULT1A1-Y169D, SULT1A1-R213H, SULT1A1-T227P, and SULT1A1-V243D) all showed lower sulfation activities than with the wild-type enzyme. Of these seven allozymes, SULT1A1-R37Q and SULT1A1-R213H showed approximately a 20% decrease in *O*-DMN-sulfation activity, while SULT1A1-H149Y and SULT1A1-V243D showed approximately a 60% decrease in *O*-DMN-sulfation activity, compared with the wild-type SULT1A1. SULT1A1-P47S and SULT1A1-Y169D, on the other hand, displayed approximately an 80% decrease in sulfation activity than the wild-type enzyme. The lowest activity was detected for SULT1A1-T227P, which showed a greater than 97% decrease in *O*-DMN-sulfation activity.

### Figure 6.

![Figure 6.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f8dc/6498441/712e8972ab27/nihms-1522689-f0006.jpg)

Specific activities of the sulfation of O-DMN by human SULT1A1 allozymes. The concentrations of O-DMN used in the enzymatic assays were 1 μM (A), 10 μM (B), and 100 μM (C). Data shown represent mean ± standard deviation derived from three determinations. One-way ANOVA was performed in combination with Dunnett’s multiple comparison test (statistically significant from SULTlAl-wild-type: *p<0,05, **p<0.01, ***p<0.001, and ****p<0.0001).

### Tapentadol as the substrate

With 5 μM tapentadol as a substrate ([Figure 7A](#F7)), two allozymes, SULT1A1-M77I and SULT1A1-F247L, displayed sulfating activities comparable to that of the wild-type enzyme, while three allozymes, SULT1A1-R37Q, SULT1A1-H149Y, and SULT1A1-R213H, exhibited slightly lower (75%, 84%, and 90%, respectively, that of the wild-type) sulfating activities. The remaining four SULT1A1 allozymes exhibited much lower sulfating activities. Of these four allozymes, SULT1A1-T227D showed the lowest sulfation activity, being only 31% of that of the wild-type enzyme. At mid and high substrate concentrations (100 and 500 μM, respectively), similar patterns of tapentadol-sulfating activities were found for the SULT1A1 allozymes analyzed ([Figure 7B](#F7) and [C](#F7)). Seven of them, SULT1A1-R37Q, SULT1A1-P47S, SULT1A1-H149Y, SULT1A1-Y169D, SULT1A1-R213H, SULT1A1-T227P, and SULT1A1-V243D, displayed significantly lower sulfation activities than the wild-type enzyme. Of the seven, SULT1A1-T227P displayed the lowest tapentadol-sulfation activity, being only 13% and 5% (at, respectively, 100 μM and 500 μM substrate concentration) of that of the wild-type enzyme. Of the other six allozymes, the sulfation activities of three (SULT1A1-R37Q, SULT1A1-H149Y, and SULT1A1-R213H) were approximately 12% lower than that of the wild-type, while the sulfation activities of the other three (SULT1A1-P47S, SULT1A1-Y169D, and SULT1A1-V243D) were approximately 24%, 32%, and 28% at 100 μM, respectively, and 27%, 28%, and 36% at 500 μM, respectively, of that for the wild-type enzyme. In contrast, SULT1A1-M77I allozyme showed slightly higher sulfation activities, being 7% and 11% higher at, respectively, 100 μM and 500 μM than that of the wild-type enzyme, while SULT1A1-F247L displayed higher tapentadol-sulfation activities, being 39% and 27%, respectively, higher at 100 μM and 500 μM than that of the wild-type enzyme.

#### In silico analyses of the effect of the substitution on the SULT1A1 structure

Rotamer analyses of the substituted-residues were carried out using the Dunbrack backbone-dependent rotamer library [[38](#R38)]. Top-ranked rotamers of the substituted residues, with 0.30, 0.50, 0.82, 0.70, 0.42, 0.16, 0.99, 0.41, 0.57 probabilities for Q37, S47, T77, Y149, D169, H213, P227, D243, and L247, respectively, are shown in [Figure S2](#SD1). Additional *in silico* analyses showed the effect of substitutions on the structural interactions between the substituted residues and the surrounding residues. Among the substitutions, H149Y, V243D, and F247L appeared to change the interaction with the substrate and T227P appeared to change the interaction with PAP (PAPS). While P47S and Y169D appeared to affect the substrate-binding pockets by altering the interaction between the substrate and enzyme, M77T seemed to have smaller effect on the substrate-binding pocket. R37Q and R213H, on the other hand, appeared to affect the surface structures, which may change the overall structure of SULT1A1.

## Discussion

The nonopioid and opioid analgesic drugs, such as acetaminophen, *O*-DMN, and tapentadol, are widely used in the clinical management of a variety of diseases and symptoms in various age groups [[1](#R1)–[3](#R3)]. Therefore, it is important to understand better the possible impact of genetic polymorphism on the metabolism hence the efficacy and toxicity of these drug compounds in different individuals. Members of the SULT enzyme family catalyze the sulfation of a multitude of endogenous and exogenous compounds [[14](#R14)]. Among the known human SULT enzymes, SULT1A1 is known to be a major SULT in xenobiotic metabolism [[25](#R25)], due in part to its broad substrate range and its wide distribution throughout the body including the brain, gastrointestinal tract, liver, lung, skin, platelets, and kidney [[14](#R14), [25](#R25)]. The enzyme is also important from a developmental biology perspective, since SULT1A1 has been reported to be extensively expressed at fetal stages [[39](#R39)–[41](#R41)]. As such, SULT1A1 plays an essential role in the chemical defense during fetal development, through neonatl/child development, onto adulthood [[41](#R41)].

Recent studies have demonstrated the involvement of SULT1A1 in the sulfation of acetaminophen [[17](#R17)], *O*-DMN [[19](#R19)], and tapentadol [[20](#R20)]. Studies have shown that genetic variability in the genes encoding enzymes, including SULTs, responsible for the metabolism of acetaminophen may be associated with risk of certain individuals for acetaminophen-induced toxicity [[16](#R16), [42](#R42), [43](#R43)]. It is noted that single nucleotide polymorphisms (SNPs) of the *SULT1A1* gene have been reported [[22](#R22), [23](#R23)]. Earlier studies using platelet samples collected from human subjects with different *SULT1A1* genotypes showed greater than 50-fold variations in their sulfation activity toward 4-nitrophenol, a prototype substrate for SULT1A1 [[22](#R22)]. In another study [[44](#R44)], two SULT1A1 allozymes expressed in cultured cells were found to display significantly different activity with 3,3′,5-triiodothyronine as the substrate. In the current study, a systematic database search was performed to identify the missense cSNPs of human *SULT1A1* genes. Nine *SULT1A1* missense cSNPs were selected, and corresponding SULT1A1 allozymes were expressed, purified, and analyzed in regard to their sulfation activity toward acetaminophen, *O*-DMN, and tapentadol, as well as 4NP as a prototype substrate.

Our initial study on the sulfation of 4NP, a prototype substrate, and the three tested analgesic compounds showed that the wild-type SULT1A1 displayed kinetic constants similar to those previously reported [[17](#R17), [19](#R19), [20](#R20), [45](#R45)–[47](#R47); cf [Table 2](#T2)]. Based on the *Km* values determined with each of the four compounds, three concentrations each of these four substrates, one well below *Km*, one around *Km*, and one well above *Km*, were used to evaluate the effects of amino acid variations on the sulfation activity of different SULT1A1 allozymes. Activity data shown in [Figures 4](#F4)–[7](#F7) indicated clearly the differential sulfation activity of different SULT1A1 allozymes. Several crystal structures of SULT1A1 have been reported [[35](#R35), [48](#R48), [49](#R49)]. Some of the structural elements involved in the functioning of the enzyme include the phosphosulfate binding loop (PSB) and other PAPS-binding regions/residues (residues 45TYPKSGTT51, Arg130, Ser138, and 257RKG259), as well as the substrate-binding residues (Ile-21, Phe24, Met77, Pro90, Ala146, His149, Tyr169, Phe247, and Val243) [[25](#R25), [35](#R35), [37](#R37)]. It is interesting to note that most of the *SULT1A1* cSNPs examined involve amino acid changes either within or close to these important structural elements. For example, in the case of SULT1A1-P47S which showed a decrease in sulfation activity, Pro47 has been reported to be part of the PSB loop that is important for orienting the cofactor, PAPS, for in-line sulfuryl transfer to the acceptor substrate [[25](#R25), [50](#R50)]. The replacement of Pro (a non-polar, turning inducing amino acid residue) with Ser (a polar, non-turning inducing residue) might result in conformational changes that affect the role of the PSB loop in the sulfation reaction, and ultimately render the decrease in catalytic activity of this allozyme. In addition, *in silico* analyses showed that the replacement of Pro with Ser may affect the structure of substrate-binding pocket ([Figure S2](#SD1)). Similarly, for SULT1A1-T227P, the replacement of Thr with Pro resulted in the near abolishment of sulfation activity. In SULT1A1-T227P, Pro as a turn-inducing amino acid may compel the local structure to adopt a more restricted conformation. Furthermore, the PAPS adenine ring has been shown to interact with the conserved residues Trp53 and Phe229. These interactions are stabilized by hydrogen bond interactions which are determined by the side chains of residues Thr227 and Tyr193 [[25](#R25), [35](#R35)]. *In silico* analysis demonstrated indeed that the replacement of Pro with Ser may affect the interaction between Thr227 and Trp53 and PAP, which may disrupt the stability of PAPS binding ([Figure S2](#SD1)). In SULT1A1-T227P, these interactions are likely weakened/disrupted, thereby destabilizing the PAPS binding and leading to the dramatic decrease in sulfation activity toward all four tested substrates.

It was noted that compared to the wild-type enzyme, only slight changes in sulfation activity were detected for all SULT1A1 allozymes tested with low and mid 4NP concentrations, except SULT1A1-T227P which showed a dramatic decrease ([Figure 4](#F4)). In contrast, at high substrate concentration, a wide variation in 4NP-sulfation activity was detected among the nine SULT1A1 allozymes. It has been reported that the active site of SULT1A1 is capable of binding two molecules of 4NP simultaneously (cf. [Figure S1](#SD1)) and that decreased enzyme activity at high substrate concentrations is caused by impeded catalysis when both binding sites are occupied [[35](#R35)]. Therefore, the differential sulfation activities of SULT1A1 allozymes at high substrate concentration might reflect their differential binding affinities for 4NP. It is noted that substrate inhibition has been reported previously for SULT1A1 [[46](#R46), [51](#R51)].

SULT1A1-H149Y allozyme exhibited lower sulfating activities than the wild-type enzyme toward the four substrates tested. The positively charged/basic amino acid His149 residue has been shown to be part of the substrate binding pocket in the structure of SULT1A1 [[49](#R49); cf [Figure S2](#SD1)], and is critical to the determination of the enzyme substrate specificity [[35](#R35), [49](#R49)]. The side chain of His149 may play a role in the stabilization of the substrate in the substrate binding pocket through the formation of electrostatic interactions with the negatively charged nitro group of the substrate, 4NP [[49](#R49)]. It is therefore not surprising that the replacement of His149 with Tyr resulted in dramatically lower sulfating activities, compared with the wild-type enzyme. SULT1A1-Y169D and SULT1A1-V243D also displayed decrease sulfating activities toward the tested substrates compared with the wild-type enzyme. These two allozymes involve amino acid changes to a negatively charged/acidic residue, Asp and may dramatically alter the structure of substrate-binding pocket as indicated by *in silico* analyses ([Figure S2](#SD1)). Tyr169 residue has been shown to be a component of the substrate binding pocket of SULT1A1 [[35](#R35), [51](#R51); cf. [Figure S2](#SD1)]. *In silico* analyses suggested that replacement of Tyr169 with Asp may eliminate the hydrogen bonding between Tyr169 and Glu83 in the loop-1, leading to the conformational change of loop-2. Replacement of Tyr169 with the negatively charged Asp thereby disrupting the substrate binding pocket may underscore the 26-fold decrease in sulfation activity with acetaminophen as a substrate, compared with the wild-type enzyme ([Figure 5](#F5)). Val243, as part of the eight aliphatic residues in the L-shaped substrate binding pocket of the SULT1A1 structure, has been shown to be involved in the catalytic activity of SULT1A1 [[35](#R35), [37](#R37), [51](#R51)]. Moreover, as shown in [Figure S2](#SD1), replacement with Asp may cause the alteration of the conformation of loop-3. Substitution of this amino acid with the negatively charged Asp also exhibited significant lower sulfating activities with all four tested substrates ([Figure 4](#F4)–[7](#F7)).

SULT1A1-R37Q and SULT1A1-R213H, with the positively charged Arg replaced by either a Gin or His, also showed lower sulfation activities than the wild-type enzyme with all four tested substrates ([Figure 4](#F4)–[7](#F7)). Previous studies have demonstrated the importance of the Arg residues for the SULT1A1 catalytic activity [[26](#R26), [35](#R35), [51](#R51)]. These two SULT1A1 SNPs had been identified in earlier polymorphic studies, which showed also that the platelet samples corresponding to the *SULT1A1*-R213H genotype displayed much lower sulfation activity than the platelet samples corresponding to wild-type *SULT1A1* genotype [[22](#R22), [23](#R23), [44](#R44)]. Substitution of a basic Arg residue with an uncharged Gin residue in SULT1A1-R37Q may alter the loop structure containing R37 by eliminating the hydrogen bonding with Asp39 (cf. [Figure S2](#SD1)), which could in part explain the impairment in the sulfation activity of SULT1A1-R37Q. In SULT1A1-R213H, the substitution of Arg with His may cause altered interaction with neighboring residues, leading to changes in protein stability as well as PAPS-binding [[35](#R35), [52](#R52)]. While His213 in SULT1A1-R213H is not thought to contribute directly to the substrate-binding pocket, it may interact with other residues, leading to certain structural alterations in SULT1A1 protein and indirectly the substrate binding pocket [[52](#R52)]. Indeed, *in silico* analyses indicated that the replacement of Arg213 with His may dramatically alter the interaction between Arg213 and Tyr62 located in a-helix3, which may affect the interaction of PAPS and PSB-loop near the α-helix3. It is important to note that segregation analysis of data derived from family studies revealed *SULT*1A1-R213H genotype as a rather common polymorphism found among Caucasians, Asians, and Africans [[22](#R22), [53](#R53)]. This *SULT1A1* genotype was found to have an allele frequency of about 30% in the Caucasians, 16.8% in the Japanese population [[25](#R25), [53](#R53)], and 27% in African American population [[26](#R26)].

To conclude, the current study aimed to systematically evaluate the effects of genetic polymorphisms on the sulfation activity of SULT1A1 allozymes towards 4NP, acetaminophen, *O*-DMN, and tapentadol. Enzymatic characterization of these allozymes revealed significant and sometimes dramatic differences in their catalytic activity with the tested substrates. These data obtained help provide insights into the functional consequences of polymorphic changes that may in the future aid in the interpretation of the pharmacokinetic profiles of those drugs found for individuals with different *SULT1A1* genotypes.

## Supplementary Material

## Acknowledgments:

This work was supported in part by a grant from National Institutes of Health (Grant # R03HD071146).

## Abbreviations:

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Bertolini A, Ferrari A, Ottani A, Guerzoni S, Tacchi R, Leone S. Paracetamol: new vistas of an old drug. CNS Drug. 2006;12(3-4):250–75.  [DOI](https://doi.org/10.1111/j.1527-3458.2006.00250.x) | [PMC free article](/articles/PMC6506194/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17227290/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CNS%20Drug&title=Paracetamol:%20new%20vistas%20of%20an%20old%20drug&author=A%20Bertolini&author=A%20Ferrari&author=A%20Ottani&author=S%20Guerzoni&author=R%20Tacchi&volume=12&issue=3-4&publication_year=2006&pages=250-75&pmid=17227290&doi=10.1111/j.1527-3458.2006.00250.x&)

2. Duggan KC, Walters MJ, Musee J, Harp JM, Kiefer JR, Oates JA, Marnett LJ. Molecular basis for cyclooxygenase inhibition by the non-steroidal anti-inflammatory drug naproxen. J Biol Chem. 2010; 5;285(45):34950–9.  [DOI](https://doi.org/10.1074/jbc.M110.162982) | [PMC free article](/articles/PMC2966109/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20810665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Molecular%20basis%20for%20cyclooxygenase%20inhibition%20by%20the%20non-steroidal%20anti-inflammatory%20drug%20naproxen&author=KC%20Duggan&author=MJ%20Walters&author=J%20Musee&author=JM%20Harp&author=JR%20Kiefer&volume=285&issue=45&publication_year=2010&pages=34950-9&pmid=20810665&doi=10.1074/jbc.M110.162982&)

3. Kress HG. Tapentadol and its two mechanisms of action: is there a new pharmacological class of centrally-acting analgesics on the horizon? Eur J Pain. 2010;14(8):781–3.  [DOI](https://doi.org/10.1016/j.ejpain.2010.06.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20659810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pain&title=Tapentadol%20and%20its%20two%20mechanisms%20of%20action:%20is%20there%20a%20new%20pharmacological%20class%20of%20centrally-acting%20analgesics%20on%20the%20horizon?&author=HG%20Kress&volume=14&issue=8&publication_year=2010&pages=781-3&pmid=20659810&doi=10.1016/j.ejpain.2010.06.017&)

4. Constance JE, Campbell SC, Somani AA, Yellepeddi V, Owens KH, Sherwin CMT. Pharmacokinetics, pharmacodynamics, and pharmacogenetics associated with nonsteroidal anti-inflammatory drugs and opioids in pediatric cancer patients. Expert Opinion on Drug Metabolism & Toxicology. 2017;13(7):715–724.  [DOI](https://doi.org/10.1080/17425255.2017.1329415) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28490206/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opinion%20on%20Drug%20Metabolism%20&%20Toxicology&title=Pharmacokinetics,%20pharmacodynamics,%20and%20pharmacogenetics%20associated%20with%20nonsteroidal%20anti-inflammatory%20drugs%20and%20opioids%20in%20pediatric%20cancer%20patients&author=JE%20Constance&author=SC%20Campbell&author=AA%20Somani&author=V%20Yellepeddi&author=KH%20Owens&volume=13&issue=7&publication_year=2017&pages=715-724&pmid=28490206&doi=10.1080/17425255.2017.1329415&)

5. Li Wan Po A, Zhang WY. Systematic overview of co-proxamol to assess analgesic effects of addition of dextropropoxyphene to paracetamol. BMJ. 1997;13;315(7122):1565–71.  [DOI](https://doi.org/10.1136/bmj.315.7122.1565) | [PMC free article](/articles/PMC2127964/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9437273/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Systematic%20overview%20of%20co-proxamol%20to%20assess%20analgesic%20effects%20of%20addition%20of%20dextropropoxyphene%20to%20paracetamol&author=A%20Li%20Wan%20Po&author=WY%20Zhang&volume=13&publication_year=1997&pages=1565-71&pmid=9437273&doi=10.1136/bmj.315.7122.1565&)

6. Adjei AA, Gaedigk A, Simon SD, Weinshilboum RM, Leeder JS. Interindividual variability in acetaminophen sulfation by human fetal liver: implications for pharmacogenetic investigations of drug-induced birth defects. Birth Defects Res A Clin Mol Teratol. 2008;82(3):155–65.  [DOI](https://doi.org/10.1002/bdra.20535) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18232020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Birth%20Defects%20Res%20A%20Clin%20Mol%20Teratol&title=Interindividual%20variability%20in%20acetaminophen%20sulfation%20by%20human%20fetal%20liver:%20implications%20for%20pharmacogenetic%20investigations%20of%20drug-induced%20birth%20defects&author=AA%20Adjei&author=A%20Gaedigk&author=SD%20Simon&author=RM%20Weinshilboum&author=JS%20Leeder&volume=82&issue=3&publication_year=2008&pages=155-65&pmid=18232020&doi=10.1002/bdra.20535&)

7. Terlinden R, Ossig J, Fliegert F, Lange C, Göhler K. Absorption, metabolism, and excretion of 14C-labeled tapentadol HCl in healthy male subjects. Eur J Drug Metab Pharmacokinet. 2007;32(3):163–9.  [DOI](https://doi.org/10.1007/BF03190478) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18062408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Drug%20Metab%20Pharmacokinet&title=Absorption,%20metabolism,%20and%20excretion%20of%2014C-labeled%20tapentadol%20HCl%20in%20healthy%20male%20subjects&author=R%20Terlinden&author=J%20Ossig&author=F%20Fliegert&author=C%20Lange&author=K%20G%C3%B6hler&volume=32&issue=3&publication_year=2007&pages=163-9&pmid=18062408&doi=10.1007/BF03190478&)

8. Coulter C, Taruc M, Tuyay J, Moore C. Determination of tapentadol and its metabolite N-desmethyltapentadol in urine and oral fluid using liquid chromatography with tandem mass spectral detection. J Anal Toxicol. 2010;34(8):458–63.  [DOI](https://doi.org/10.1093/jat/34.8.458) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21819790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Anal%20Toxicol&title=Determination%20of%20tapentadol%20and%20its%20metabolite%20N-desmethyltapentadol%20in%20urine%20and%20oral%20fluid%20using%20liquid%20chromatography%20with%20tandem%20mass%20spectral%20detection&author=C%20Coulter&author=M%20Taruc&author=J%20Tuyay&author=C%20Moore&volume=34&issue=8&publication_year=2010&pages=458-63&pmid=21819790&doi=10.1093/jat/34.8.458&)

9. Singh DR, Nag K, Shetti AN, Krishnaveni N. Tapentadol hydrochloride: A novel analgesic. Saudi J Anaesth. 2013;7(3):322–6.  [DOI](https://doi.org/10.4103/1658-354X.115319) | [PMC free article](/articles/PMC3757808/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24015138/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Saudi%20J%20Anaesth&title=Tapentadol%20hydrochloride:%20A%20novel%20analgesic&author=DR%20Singh&author=K%20Nag&author=AN%20Shetti&author=N%20Krishnaveni&volume=7&issue=3&publication_year=2013&pages=322-6&pmid=24015138&doi=10.4103/1658-354X.115319&)

10. Segre EJ. Naproxen metabolism in man. J Clin Pharmacol. 1975;15(4 Pt. 2):316–23.  [DOI](https://doi.org/10.1002/j.1552-4604.1975.tb01458.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1127135/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Naproxen%20metabolism%20in%20man&author=EJ%20Segre&volume=15&issue=4%20Pt.%202&publication_year=1975&pages=316-23&pmid=1127135&doi=10.1002/j.1552-4604.1975.tb01458.x&)

11. Davies NM and Anderson KE. Clinical pharmacokinetics of naproxen. Clin Pharmacokinet. 1997;32(4):268–93.  [DOI](https://doi.org/10.2165/00003088-199732040-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9113437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Clinical%20pharmacokinetics%20of%20naproxen&author=NM%20Davies&author=KE%20Anderson&volume=32&issue=4&publication_year=1997&pages=268-93&pmid=9113437&doi=10.2165/00003088-199732040-00002&)

12. Mulder GJ, Jakoby WB, editors. Drug Metabolism, London: Taylor and Francis; 1990.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Drug%20Metabolism&author=GJ%20Mulder&author=WB%20Jakoby&publication_year=1990&)

13. Weinshilboum RM, Otterness DM, Aksoy IA, Wood TC, Her C, Raftogianis RB. Sulfation and sulfotransferases 1: Sulfotransferase molecular biology: cDNAs and genes. FASEB J. 1997;11(1):3–14.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9034160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FASEB%20J&title=Sulfation%20and%20sulfotransferases%201:%20Sulfotransferase%20molecular%20biology:%20cDNAs%20and%20genes&author=RM%20Weinshilboum&author=DM%20Otterness&author=IA%20Aksoy&author=TC%20Wood&author=C%20Her&volume=11&issue=1&publication_year=1997&pages=3-14&pmid=9034160&)

14. Coughtrie MW. Sulfation through the looking glass--recent advances in sulfotransferase research for the curious. Pharmacogenomics J. 2002;2(5):297–308.  [DOI](https://doi.org/10.1038/sj.tpj.6500117) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12439736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Sulfation%20through%20the%20looking%20glass--recent%20advances%20in%20sulfotransferase%20research%20for%20the%20curious&author=MW%20Coughtrie&volume=2&issue=5&publication_year=2002&pages=297-308&pmid=12439736&doi=10.1038/sj.tpj.6500117&)

15. Strott CA. Sulfonation and molecular action. Endocr Rev. 2002;23(5):703–32.  [DOI](https://doi.org/10.1210/er.2001-0040) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12372849/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocr%20Rev&title=Sulfonation%20and%20molecular%20action&author=CA%20Strott&volume=23&issue=5&publication_year=2002&pages=703-32&pmid=12372849&doi=10.1210/er.2001-0040&)

16. Zhao L, Pickering G. Paracetamol metabolism and related genetic differences. Drug Metab Rev. 2011;43(1):41–52.  [DOI](https://doi.org/10.3109/03602532.2010.527984) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21108564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Paracetamol%20metabolism%20and%20related%20genetic%20differences&author=L%20Zhao&author=G%20Pickering&volume=43&issue=1&publication_year=2011&pages=41-52&pmid=21108564&doi=10.3109/03602532.2010.527984&)

17. Yamamoto A, Liu MY, Kurogi K, Sakakibara Y, Saeki Y, Suiko M, et al. Sulphation of acetaminophen by the human cytosolic sulfotransferases: a systematic analysis. J Biochem. 2015;158(6):497–504.  [DOI](https://doi.org/10.1093/jb/mvv062) | [PMC free article](/articles/PMC4819960/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26067475/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biochem&title=Sulphation%20of%20acetaminophen%20by%20the%20human%20cytosolic%20sulfotransferases:%20a%20systematic%20analysis&author=A%20Yamamoto&author=MY%20Liu&author=K%20Kurogi&author=Y%20Sakakibara&author=Y%20Saeki&volume=158&issue=6&publication_year=2015&pages=497-504&pmid=26067475&doi=10.1093/jb/mvv062&)

18. Falany CN, Ström P, Swedmark S. Sulphation of O-desmethylnaproxen and related compounds by human cytosolic sulfotransferases. Br J Clin Pharmacol 2005;60(6):632–40.  [DOI](https://doi.org/10.1111/j.1365-2125.2005.02506.x) | [PMC free article](/articles/PMC1884888/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16305588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Sulphation%20of%20O-desmethylnaproxen%20and%20related%20compounds%20by%20human%20cytosolic%20sulfotransferases&author=CN%20Falany&author=P%20Str%C3%B6m&author=S%20Swedmark&volume=60&issue=6&publication_year=2005&pages=632-40&pmid=16305588&doi=10.1111/j.1365-2125.2005.02506.x&)

19. Alherz FA, Almarghalani DA, Hussein NA, Kurogi K, Liu MC. A reappraisal of the 6-O-desmethylnaproxen-sulfating activity of the human cytosolic sulfotransferases. Can J Physiol Pharmacol. 2017;95(6):647–651.  [DOI](https://doi.org/10.1139/cjpp-2016-0403) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28177672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Can%20J%20Physiol%20Pharmacol&title=A%20reappraisal%20of%20the%206-O-desmethylnaproxen-sulfating%20activity%20of%20the%20human%20cytosolic%20sulfotransferases&author=FA%20Alherz&author=DA%20Almarghalani&author=NA%20Hussein&author=K%20Kurogi&author=MC%20Liu&volume=95&issue=6&publication_year=2017&pages=647-651&pmid=28177672&doi=10.1139/cjpp-2016-0403&)

20. Bairam AF, Rasool MI, Kurogi K, Liu MC. On the Molecular Basis Underlying the Metabolism of Tapentadol Through Sulfation. Eur J Drug Metab Pharmacokinet. 2017;42(5):793–800.  [DOI](https://doi.org/10.1007/s13318-016-0392-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28070880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Drug%20Metab%20Pharmacokinet&title=On%20the%20Molecular%20Basis%20Underlying%20the%20Metabolism%20of%20Tapentadol%20Through%20Sulfation&author=AF%20Bairam&author=MI%20Rasool&author=K%20Kurogi&author=MC%20Liu&volume=42&issue=5&publication_year=2017&pages=793-800&pmid=28070880&doi=10.1007/s13318-016-0392-8&)

21. Raftogianis RB, Her C, Weinshilboum RM. Human phenol sulfotransferase pharmacogenetics: STP1 gene cloning and structural characterization. Pharmacogenetics. 1996;6(6):473–87.  [DOI](https://doi.org/10.1097/00008571-199612000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9014197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Human%20phenol%20sulfotransferase%20pharmacogenetics:%20STP1%20gene%20cloning%20and%20structural%20characterization&author=RB%20Raftogianis&author=C%20Her&author=RM%20Weinshilboum&volume=6&issue=6&publication_year=1996&pages=473-87&pmid=9014197&doi=10.1097/00008571-199612000-00001&)

22. Raftogianis RB, Wood TC, Otterness DM, Van Loon JA, Weinshilboum RM. Phenol sulfotransferase pharmacogenetics in humans: association of common SULT1A1 alleles with TS PST phenotype. Biochem Biophys Res Commun. 1997;239(1):298–304.  [DOI](https://doi.org/10.1006/bbrc.1997.7466) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9345314/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Phenol%20sulfotransferase%20pharmacogenetics%20in%20humans:%20association%20of%20common%20SULT1A1%20alleles%20with%20TS%20PST%20phenotype&author=RB%20Raftogianis&author=TC%20Wood&author=DM%20Otterness&author=JA%20Van%20Loon&author=RM%20Weinshilboum&volume=239&issue=1&publication_year=1997&pages=298-304&pmid=9345314&doi=10.1006/bbrc.1997.7466&)

23. Raftogianis RB, Wood TC, Weinshilboum RM. Human phenol sulfotransferases SULT1A2 and SULT1A1: genetic polymorphisms, allozyme properties, and human liver genotype-phenotype correlations. Biochem Pharmacol. 1999;58(4):605–16.  [DOI](https://doi.org/10.1016/s0006-2952(99)00145-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10413297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Human%20phenol%20sulfotransferases%20SULT1A2%20and%20SULT1A1:%20genetic%20polymorphisms,%20allozyme%20properties,%20and%20human%20liver%20genotype-phenotype%20correlations&author=RB%20Raftogianis&author=TC%20Wood&author=RM%20Weinshilboum&volume=58&issue=4&publication_year=1999&pages=605-16&pmid=10413297&doi=10.1016/s0006-2952(99)00145-8&)

24. Hebbring SJ, Adjei AA, Baer JL, Jenkins GD, Zhang J, Cunningham JM, et al. Human SULT1A1 gene: copy number differences and functional implications. Hum Mol Genet. 2007;16(5):463–70.  [DOI](https://doi.org/10.1093/hmg/ddl468) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17189289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Human%20SULT1A1%20gene:%20copy%20number%20differences%20and%20functional%20implications&author=SJ%20Hebbring&author=AA%20Adjei&author=JL%20Baer&author=GD%20Jenkins&author=J%20Zhang&volume=16&issue=5&publication_year=2007&pages=463-70&pmid=17189289&doi=10.1093/hmg/ddl468&)

25. Hempel N, Gamage N, Martin JL, McManus ME. Human cytosolic sulfotransferase SULT1A1. Int J Biochem Cell Biol. 2007;39(4):685–9.  [DOI](https://doi.org/10.1016/j.biocel.2006.10.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17110154/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Biochem%20Cell%20Biol&title=Human%20cytosolic%20sulfotransferase%20SULT1A1&author=N%20Hempel&author=N%20Gamage&author=JL%20Martin&author=ME%20McManus&volume=39&issue=4&publication_year=2007&pages=685-9&pmid=17110154&doi=10.1016/j.biocel.2006.10.002&)

26. Hildebrandt MA, Carrington DP, Thomae BA, Eckloff BW, Schaid DJ, Yee VC, et al. Genetic diversity and function in the human cytosolic sulfotransferases. Pharmacogenomics J. 2007;7(2):133–43.  [DOI](https://doi.org/10.1038/sj.tpj.6500404) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16801938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genetic%20diversity%20and%20function%20in%20the%20human%20cytosolic%20sulfotransferases&author=MA%20Hildebrandt&author=DP%20Carrington&author=BA%20Thomae&author=BW%20Eckloff&author=DJ%20Schaid&volume=7&issue=2&publication_year=2007&pages=133-43&pmid=16801938&doi=10.1038/sj.tpj.6500404&)

27. Yanagisawa K, Sakakibara Y, Suiko M, Takami Y, Nakayama T, Nakajima H, et al. Cdna cloning, expression, and characterization of the human bifunctional ATP sulfurylase/adenosine 5′-phosphosulfate kinase enzyme. Biosci Biotechnol Biochem. 1998;62(5):1037–40.  [DOI](https://doi.org/10.1271/bbb.62.1037) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9648242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biosci%20Biotechnol%20Biochem&title=Cdna%20cloning,%20expression,%20and%20characterization%20of%20the%20human%20bifunctional%20ATP%20sulfurylase/adenosine%205%E2%80%B2-phosphosulfate%20kinase%20enzyme&author=K%20Yanagisawa&author=Y%20Sakakibara&author=M%20Suiko&author=Y%20Takami&author=T%20Nakayama&volume=62&issue=5&publication_year=1998&pages=1037-40&pmid=9648242&doi=10.1271/bbb.62.1037&)

28. Hui Y, Liu MC. Sulfation of ritodrine by the human cytosolic sulfotransferases (SULTs): Effects of SULT1A3 genetic polymorphism. Eur J Pharmacol. 2015; 761:125–9.  [DOI](https://doi.org/10.1016/j.ejphar.2015.04.039) | [PMC free article](/articles/PMC4532560/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25941087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&title=Sulfation%20of%20ritodrine%20by%20the%20human%20cytosolic%20sulfotransferases%20(SULTs):%20Effects%20of%20SULT1A3%20genetic%20polymorphism&author=Y%20Hui&author=MC%20Liu&volume=761&publication_year=2015&pages=125-9&pmid=25941087&doi=10.1016/j.ejphar.2015.04.039&)

29. Sanger F, Nicklen S, Coulson AR. DNA sequencing with chain-terminating inhibitors. Proc Natl Acad Sci U S A. 1977;74(12):5463–7.  [DOI](https://doi.org/10.1073/pnas.74.12.5463) | [PMC free article](/articles/PMC431765/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/271968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=DNA%20sequencing%20with%20chain-terminating%20inhibitors&author=F%20Sanger&author=S%20Nicklen&author=AR%20Coulson&volume=74&issue=12&publication_year=1977&pages=5463-7&pmid=271968&doi=10.1073/pnas.74.12.5463&)

30. Harper S, Speicher DW. Purification of proteins fused to glutathione S-transferase. Methods Mol Biol. 2011; 681:259–80.  [DOI](https://doi.org/10.1007/978-1-60761-913-0_14) | [PMC free article](/articles/PMC3584333/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20978970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&title=Purification%20of%20proteins%20fused%20to%20glutathione%20S-transferase&author=S%20Harper&author=DW%20Speicher&volume=681&publication_year=2011&pages=259-80&pmid=20978970&doi=10.1007/978-1-60761-913-0_14&)

31. Shapiro AL, Viñuela E, Maizel JV Jr. Molecular weight estimation of polypeptide chains by electrophoresis in SDS-polyacrylamide gels. Biochem Biophys Res Commun. 1967;28(5):815–20.  [DOI](https://doi.org/10.1016/0006-291x(67)90391-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4861258/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Molecular%20weight%20estimation%20of%20polypeptide%20chains%20by%20electrophoresis%20in%20SDS-polyacrylamide%20gels&author=AL%20Shapiro&author=E%20Vi%C3%B1uela&author=JV%20Maizel&volume=28&issue=5&publication_year=1967&pages=815-20&pmid=4861258&doi=10.1016/0006-291x(67)90391-9&)

32. Laemmli UK. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature. 1970;227(5259):680–5.  [DOI](https://doi.org/10.1038/227680a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/5432063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Cleavage%20of%20structural%20proteins%20during%20the%20assembly%20of%20the%20head%20of%20bacteriophage%20T4&author=UK%20Laemmli&volume=227&issue=5259&publication_year=1970&pages=680-5&pmid=5432063&doi=10.1038/227680a0&)

33. Bradford MM. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal Biochem. 1976; 72:248–54.  [DOI](https://doi.org/10.1016/0003-2697(76)90527-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/942051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Biochem&title=A%20rapid%20and%20sensitive%20method%20for%20the%20quantitation%20of%20microgram%20quantities%20of%20protein%20utilizing%20the%20principle%20of%20protein-dye%20binding&author=MM%20Bradford&volume=72&publication_year=1976&pages=248-54&pmid=942051&doi=10.1016/0003-2697(76)90527-3&)

34. Kurogi K, Chen M, Lee Y, Shi B, Yan T, Liu MY, et al. Sulfation of buprenorphine, pentazocine, and naloxone by human cytosolic sulfotransferases. Drug Metab Lett. 2012;6(2):109–15.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/22946908/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Lett&title=Sulfation%20of%20buprenorphine,%20pentazocine,%20and%20naloxone%20by%20human%20cytosolic%20sulfotransferases&author=K%20Kurogi&author=M%20Chen&author=Y%20Lee&author=B%20Shi&author=T%20Yan&volume=6&issue=2&publication_year=2012&pages=109-15&pmid=22946908&)

35. Gamage NU, Duggleby RG, Barnett AC, Tresillian M, Latham CF, Liyou NE, et al. Structure of a human carcinogen-converting enzyme, SULT1A1. Structural and kinetic implications of substrate inhibition. J Biol Chem. 2003;278(9):7655–62.  [DOI](https://doi.org/10.1074/jbc.M207246200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12471039/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Structure%20of%20a%20human%20carcinogen-converting%20enzyme,%20SULT1A1.%20Structural%20and%20kinetic%20implications%20of%20substrate%20inhibition&author=NU%20Gamage&author=RG%20Duggleby&author=AC%20Barnett&author=M%20Tresillian&author=CF%20Latham&volume=278&issue=9&publication_year=2003&pages=7655-62&pmid=12471039&doi=10.1074/jbc.M207246200&)

36. Petrotchenko EV, Pedersen LC, Borchers CH, Tomer KB, Negishi M. The dimerization motif of cytosolic sulfotransferases. FEBS Lett. 2001;490(1–2):39–43.  [DOI](https://doi.org/10.1016/s0014-5793(01)02129-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11172807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&title=The%20dimerization%20motif%20of%20cytosolic%20sulfotransferases&author=EV%20Petrotchenko&author=LC%20Pedersen&author=CH%20Borchers&author=KB%20Tomer&author=M%20Negishi&volume=490&issue=1%E2%80%932&publication_year=2001&pages=39-43&pmid=11172807&doi=10.1016/s0014-5793(01)02129-9&)

37. Tibbs ZE, Rohn-Glowacki KJ, Crittenden F, Guidry AL, Falany CN. Structural plasticity in the human cytosolic sulfotransferase dimer and its role in substrate selectivity and catalysis. Drug Metab Pharmacokinet. 2015;30(1):3–20.  [DOI](https://doi.org/10.1016/j.dmpk.2014.10.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25760527/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Structural%20plasticity%20in%20the%20human%20cytosolic%20sulfotransferase%20dimer%20and%20its%20role%20in%20substrate%20selectivity%20and%20catalysis&author=ZE%20Tibbs&author=KJ%20Rohn-Glowacki&author=F%20Crittenden&author=AL%20Guidry&author=CN%20Falany&volume=30&issue=1&publication_year=2015&pages=3-20&pmid=25760527&doi=10.1016/j.dmpk.2014.10.004&)

38. Dunbrack RL Jr. Rotamer libraries in the 21st century. Curr. Opin. Struct. Biol 2002: 2(4):431–40.  [DOI](https://doi.org/10.1016/s0959-440x(02)00344-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12163064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Struct.%20Biol&title=Rotamer%20libraries%20in%20the%2021st%20century&author=RL%20Dunbrack&volume=2&issue=4&publication_year=2002&pages=431-40&pmid=12163064&doi=10.1016/s0959-440x(02)00344-5&)

39. Richard K, Hume R, Kaptein E, Stanley EL, Visser TJ, Coughtrie MW. Sulfation of thyroid hormone and dopamine during human development: ontogeny of phenol sulfotransferases and arylsulfatase in liver, lung, and brain. J Clin Endocrinol Metab. 2001;86(6):2734–42.  [DOI](https://doi.org/10.1210/jcem.86.6.7569) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11397879/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Endocrinol%20Metab&title=Sulfation%20of%20thyroid%20hormone%20and%20dopamine%20during%20human%20development:%20ontogeny%20of%20phenol%20sulfotransferases%20and%20arylsulfatase%20in%20liver,%20lung,%20and%20brain&author=K%20Richard&author=R%20Hume&author=E%20Kaptein&author=EL%20Stanley&author=TJ%20Visser&volume=86&issue=6&publication_year=2001&pages=2734-42&pmid=11397879&doi=10.1210/jcem.86.6.7569&)

40. Vietri M, Pietrabissa A, Mosca F, Rane A, Pacific GM. Human adult and fetal liver sulphotransferases: inhibition by mefenamic acid and salicylic acid. Xenobiotica. 2001;31(3):153–61.  [DOI](https://doi.org/10.1080/00498250110043481) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11465392/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Human%20adult%20and%20fetal%20liver%20sulphotransferases:%20inhibition%20by%20mefenamic%20acid%20and%20salicylic%20acid&author=M%20Vietri&author=A%20Pietrabissa&author=F%20Mosca&author=A%20Rane&author=GM%20Pacific&volume=31&issue=3&publication_year=2001&pages=153-61&pmid=11465392&doi=10.1080/00498250110043481&)

41. Stanley EL, Hume R, Coughtrie MW. Expression profiling of human fetal cytosolic sulfotransferases involved in steroid and thyroid hormone metabolism and in detoxification. Mol Cell Endocrinol. 2005;240(1–2):32–42.  [DOI](https://doi.org/10.1016/j.mce.2005.06.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16024168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Endocrinol&title=Expression%20profiling%20of%20human%20fetal%20cytosolic%20sulfotransferases%20involved%20in%20steroid%20and%20thyroid%20hormone%20metabolism%20and%20in%20detoxification&author=EL%20Stanley&author=R%20Hume&author=MW%20Coughtrie&volume=240&issue=1%E2%80%932&publication_year=2005&pages=32-42&pmid=16024168&doi=10.1016/j.mce.2005.06.003&)

42. Patel M, Tang BK, Kalow W. Variability of acetaminophen metabolism in Caucasians and Orientals. Pharmacogenetics. 1992;2(1):38–45.  [DOI](https://doi.org/10.1097/00008571-199202000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1302042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Variability%20of%20acetaminophen%20metabolism%20in%20Caucasians%20and%20Orientals&author=M%20Patel&author=BK%20Tang&author=W%20Kalow&volume=2&issue=1&publication_year=1992&pages=38-45&pmid=1302042&doi=10.1097/00008571-199202000-00007&)

43. Court MH, Peter I, Hazarika S, Vasiadi M, Greenblatt DJ, Lee WM. Candidate gene polymorphisms in patients with acetaminophen-induced acute liver failure. Drug Metab Dispos. 2014;42(1):28–32.  [DOI](https://doi.org/10.1124/dmd.113.053546) | [PMC free article](/articles/PMC3876784/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24104197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Candidate%20gene%20polymorphisms%20in%20patients%20with%20acetaminophen-induced%20acute%20liver%20failure&author=MH%20Court&author=I%20Peter&author=S%20Hazarika&author=M%20Vasiadi&author=DJ%20Greenblatt&volume=42&issue=1&publication_year=2014&pages=28-32&pmid=24104197&doi=10.1124/dmd.113.053546&)

44. Li X, Clemens DL, Cole JR, Anderson RJ. Characterization of human liver thermostable phenol sulfotransferase (SULT1A1) allozymes with 3,3’,5-triiodothyronine as the substrate. J Endocrinol. 2001;171(3):525–32.  [DOI](https://doi.org/10.1677/joe.0.1710525) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11739018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Endocrinol&title=Characterization%20of%20human%20liver%20thermostable%20phenol%20sulfotransferase%20(SULT1A1)%20allozymes%20with%203,3%E2%80%99,5-triiodothyronine%20as%20the%20substrate&author=X%20Li&author=DL%20Clemens&author=JR%20Cole&author=RJ%20Anderson&volume=171&issue=3&publication_year=2001&pages=525-32&pmid=11739018&doi=10.1677/joe.0.1710525&)

45. Tabrett CA. and Coughtrie MW. Phenol sulfotransferase 1A1 activity in human liver: kinetic properties, interindividual variation and re-evaluation of the suitability of 4-nitrophenol as a probe substrate. Biochem Pharmacol. 2003;66(11):2089–97.  [DOI](https://doi.org/10.1016/s0006-2952(03)00582-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14609733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Phenol%20sulfotransferase%201A1%20activity%20in%20human%20liver:%20kinetic%20properties,%20interindividual%20variation%20and%20re-evaluation%20of%20the%20suitability%20of%204-nitrophenol%20as%20a%20probe%20substrate&author=CA%20Tabrett&author=MW%20Coughtrie&volume=66&issue=11&publication_year=2003&pages=2089-97&pmid=14609733&doi=10.1016/s0006-2952(03)00582-3&)

46. Wang LQ, Lehmler HJ, Robertson LW, James MO. Polychlorobiphenylols are selective inhibitors of human phenol sulfotransferase 1A1 with 4-nitrophenol as a substrate. Chem Biol Interact. 2006;159(3):235–46.  [DOI](https://doi.org/10.1016/j.cbi.2005.12.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16413005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Biol%20Interact&title=Polychlorobiphenylols%20are%20selective%20inhibitors%20of%20human%20phenol%20sulfotransferase%201A1%20with%204-nitrophenol%20as%20a%20substrate&author=LQ%20Wang&author=HJ%20Lehmler&author=LW%20Robertson&author=MO%20James&volume=159&issue=3&publication_year=2006&pages=235-46&pmid=16413005&doi=10.1016/j.cbi.2005.12.004&)

47. Riches Z, Bloomer JC, Coughtrie MW. Comparison of 2-aminophenol and 4-nitrophenol as in vitro probe substrates for the major human hepatic sulfotransferase, SULT1A1, demonstrates improved selectivity with 2-aminophenol. Biochem Pharmacol. 2007;74(2):352–8.  [DOI](https://doi.org/10.1016/j.bcp.2007.04.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17506995/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Comparison%20of%202-aminophenol%20and%204-nitrophenol%20as%20in%20vitro%20probe%20substrates%20for%20the%20major%20human%20hepatic%20sulfotransferase,%20SULT1A1,%20demonstrates%20improved%20selectivity%20with%202-aminophenol&author=Z%20Riches&author=JC%20Bloomer&author=MW%20Coughtrie&volume=74&issue=2&publication_year=2007&pages=352-8&pmid=17506995&doi=10.1016/j.bcp.2007.04.006&)

48. Gamage NU, Tsvetanov S, Duggleby RG, McManus ME, Martin JL. The structure of human SULT1A1 crystallized with estradiol. An insight into active site plasticity and substrate inhibition with multi-ring substrates. J Biol Chem. 2005;280(50):41482–6.  [DOI](https://doi.org/10.1074/jbc.M508289200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16221673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=The%20structure%20of%20human%20SULT1A1%20crystallized%20with%20estradiol.%20An%20insight%20into%20active%20site%20plasticity%20and%20substrate%20inhibition%20with%20multi-ring%20substrates&author=NU%20Gamage&author=S%20Tsvetanov&author=RG%20Duggleby&author=ME%20McManus&author=JL%20Martin&volume=280&issue=50&publication_year=2005&pages=41482-6&pmid=16221673&doi=10.1074/jbc.M508289200&)

49. Lu J, Li H, Zhang J, Li M, Liu MY, An X, et al. Crystal structures of SULT1A2 and SULT1A1 *3: insights into the substrate inhibition and the role of Tyr149 in SULT1A2. Biochem Biophys Res Commun. 2010;396(2):429–34.  [DOI](https://doi.org/10.1016/j.bbrc.2010.04.109) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20417180/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Crystal%20structures%20of%20SULT1A2%20and%20SULT1A1%20*3:%20insights%20into%20the%20substrate%20inhibition%20and%20the%20role%20of%20Tyr149%20in%20SULT1A2&author=J%20Lu&author=H%20Li&author=J%20Zhang&author=M%20Li&author=MY%20Liu&volume=396&issue=2&publication_year=2010&pages=429-34&pmid=20417180&doi=10.1016/j.bbrc.2010.04.109&)

50. Kakuta Y, Petrotchenko EV, Pedersen LC, Negishi M. The sulfuryl transfer mechanism. Crystal structure of a vanadate complex of estrogen sulfotransferase and mutational analysis. J Biol Chem. 1998;273(42):27325–30.  [DOI](https://doi.org/10.1074/jbc.273.42.27325) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9765259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=The%20sulfuryl%20transfer%20mechanism.%20Crystal%20structure%20of%20a%20vanadate%20complex%20of%20estrogen%20sulfotransferase%20and%20mutational%20analysis&author=Y%20Kakuta&author=EV%20Petrotchenko&author=LC%20Pedersen&author=M%20Negishi&volume=273&issue=42&publication_year=1998&pages=27325-30&pmid=9765259&doi=10.1074/jbc.273.42.27325&)

51. Barnett AC, Tsvetanov S, Gamage N, Martin JL, Duggleby RG, McManus ME. Active site mutations and substrate inhibition in human sulfotransferase 1A1 and 1A3. J Biol Chem. 2004;279(18):18799–805.  [DOI](https://doi.org/10.1074/jbc.M312253200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14871892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Active%20site%20mutations%20and%20substrate%20inhibition%20in%20human%20sulfotransferase%201A1%20and%201A3&author=AC%20Barnett&author=S%20Tsvetanov&author=N%20Gamage&author=JL%20Martin&author=RG%20Duggleby&volume=279&issue=18&publication_year=2004&pages=18799-805&pmid=14871892&doi=10.1074/jbc.M312253200&)

52. Nagar S, Walther S, Blanchard RL. Sulfotransferase (SULT) 1A1 polymorphic variants *1, *2, and *3 are associated with altered enzymatic activity, cellular phenotype, and protein degradation. Mol Pharmacol. 2006;69(6):2084–92.  [DOI](https://doi.org/10.1124/mol.105.019240) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16517757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Sulfotransferase%20(SULT)%201A1%20polymorphic%20variants%20*1,%20*2,%20and%20*3%20are%20associated%20with%20altered%20enzymatic%20activity,%20cellular%20phenotype,%20and%20protein%20degradation&author=S%20Nagar&author=S%20Walther&author=RL%20Blanchard&volume=69&issue=6&publication_year=2006&pages=2084-92&pmid=16517757&doi=10.1124/mol.105.019240&)

53. Ozawa S, Shimizu M, Katoh T, Miyajima A, Ohno Y, Matsumoto Y, et al. Sulfating-activity and stability of cDNA-expressed allozymes of human phenol sulfotransferase, ST1A3*1 ((213) Arg) and ST1A3*2 ((213) His), both of which exist in Japanese as well as Caucasians. J Biochem. 1999;126(2):271–7.  [DOI](https://doi.org/10.1093/oxfordjournals.jbchem.a022445) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10423517/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biochem&title=Sulfating-activity%20and%20stability%20of%20cDNA-expressed%20allozymes%20of%20human%20phenol%20sulfotransferase,%20ST1A3*1%20((213)%20Arg)%20and%20ST1A3*2%20((213)%20His),%20both%20of%20which%20exist%20in%20Japanese%20as%20well%20as%20Caucasians&author=S%20Ozawa&author=M%20Shimizu&author=T%20Katoh&author=A%20Miyajima&author=Y%20Ohno&volume=126&issue=2&publication_year=1999&pages=271-7&pmid=10423517&doi=10.1093/oxfordjournals.jbchem.a022445&)

54. Omasits U, Ahrens CH, Müller S, Wollscheid B. Protter: interactive protein feature visualization and integration with experimental proteomic data. Bioinformatics. 2014;30(6):884–6.  [DOI](https://doi.org/10.1093/bioinformatics/btt607) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24162465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Protter:%20interactive%20protein%20feature%20visualization%20and%20integration%20with%20experimental%20proteomic%20data&author=U%20Omasits&author=CH%20Ahrens&author=S%20M%C3%BCller&author=B%20Wollscheid&volume=30&issue=6&publication_year=2014&pages=884-6&pmid=24162465&doi=10.1093/bioinformatics/btt607&)
