# Glucocorticoid receptor single nucleotide polymorphisms are associated with acute crisis pain in sickle cell disease

## Metadata
**Authors:** Ellie H Jhun, Nilanjana Sadhu, Yingwei Yao, Ying He, Robert E Molokie, Diana J Wilkie, Zaijie Jim Wang
**Journal:** Pharmacogenomics
**Date:** 2018 Aug 6
**DOI:** [10.2217/pgs-2018-0064](https://doi.org/10.2217/pgs-2018-0064)
**PMID:** 30079801
**PMCID:** PMC6275559
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6275559/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6275559/pdf/pgs-19-1003.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6275559/pdf/pgs-19-1003.pdf)

## Abstract

**Aim:** 
Pain in sickle cell disease patients is heterogeneous and genetic polymorphisms may predispose an individual to varied vulnerability to painful events. We studied the association of SNPs in the glucocorticoid receptor gene (NR3C1) with pain in sickle cell disease.

**Method:** 
Acute pain was scored as the number of utilizations due to crisis pain in a 12-month period. Chronic pain was calculated as the Composite Pain Index score.

**Results & conclusion:** 
rs33389 T allele (IRR = 1.53, p = 0.014 additive; IRR = 1.64, p = 0.011 recessive), rs2963155 G allele (IRR = 1.80, p < 0.001 additive; IRR = 2.25, p = 0.021 dominant; IRR = 2.07, p < 0.001 recessive) and rs9324918 C allele (IRR = 1.43, p = 0.021 additive) were associated with higher utilization rates, indicating the potential contribution of NR3C1 polymorphisms to acute pain heterogeneity in sickle cell disease.

Keywords: : glucocorticoid receptor, NR3C1, pain, polymorphism, sickle cell crisis, sickle cell disease, SNP, utilizations

### Aim:

Pain in sickle cell disease patients is heterogeneous and genetic polymorphisms may predispose an individual to varied vulnerability to painful events. We studied the association of SNPs in the glucocorticoid receptor gene (*NR3C1*) with pain in sickle cell disease.

### Method:

Acute pain was scored as the number of utilizations due to crisis pain in a 12-month period. Chronic pain was calculated as the Composite Pain Index score.

### Results & conclusion:

rs33389 T allele (IRR = 1.53, p = 0.014 additive; IRR = 1.64, p = 0.011 recessive), rs2963155 G allele (IRR = 1.80, p < 0.001 additive; IRR = 2.25, p = 0.021 dominant; IRR = 2.07, p < 0.001 recessive) and rs9324918 C allele (IRR = 1.43, p = 0.021 additive) were associated with higher utilization rates, indicating the potential contribution of *NR3C1* polymorphisms to acute pain heterogeneity in sickle cell disease.

## Methods

### Subjects

This study was approved by the Institutional Review Board in the University of Illinois at Chicago (IL, USA) and participants gave written informed consent. A total of 136 patients with both clinical data and genetic samples were included in study analyses. Subjects were recruited during routine outpatient clinic visits and eligibility requirements are explained in detail in another article [[19](#B19)].

### PAINReportIt®

Baseline pain from SCD subjects was collected by an electronic touch screen format of the 1970 McGill Pain Questionnaire, PAINReportIt^®^ [[20](#B20)]. PAINReportIt^®^ collects baseline pain location, intensity, quality and pattern scores which are calculated into a single Composite Pain Index (CPI) value, representing the multidimensional pain score [[3](#B3),[19](#B19),[21](#B21)]. The number of pain sites; average of current pain intensity, least and worst pain intensity in the past 24 h; pain rating index total and a pain pattern score that ranges from 0 to 6 were used to calculate this CPI value, which has been validated in SCD patients [[20](#B20),[22](#B22),[23](#B23)].

### Pain crisis

Utilization is the number of admissions to the emergency department and acute care center, due to sickle cell pain crisis for 1 year after completing the baseline pain assessment. These data were collected by reviewing medical records along with biweekly telephone calls to subjects. Utilization is the surrogate marker for acute pain in our study.

### Genotyping

Blood and/or buccal swab samples were collected at the University of Illinois Hospital and Health Sciences System (IL, USA). DNA was extracted using a modified salting out procedure [[24](#B24)] for blood samples, a modified phenol/chloroform procedure [[25](#B25)] for buccal swab samples and the QuickGene-mini80 isolation device and QuickGene DNA whole blood extraction method (AutoGen, MA, USA). Genotyping was performed on extracted DNA with the MassARRAY iPLEX Platform (Seuqenom, CA, USA) [[26](#B26)]. Genotyping success rate was >90%.

### Statistics

Hardy–Weinberg equilibrium for the three *NR3C1* SNPs was calculated by a χ^2^ goodness-of-fit test. SNP effects on utilization data were analyzed by an additive, dominant and recessive negative binomial regression model [[27](#B27),[28](#B28)] adjusted for age, sex, ethnicity and sickle cell type. The effect of SNPs on CPI value was analyzed by an additive, dominant and recessive multiple linear regression model [[29](#B29),[30](#B30)] adjusted for the same covariates. Analysis was performed with SPSS software (version 20; IBM, NY, USA). The linkage disequilibrium plot was created in Haploview version 4.2 (Broad Institute, MA, USA) [[31](#B31)].

## Results

The majority of the SCD subjects in our cohort had hemoglobin SS (77.2%) as shown in [Table 1](#T1). Other sickle cell types included hemoglobin SC (11.0%), Sβ^+^ thalassemia (5.9%), hemoglobin Sβ° thalassemia (5.2%) and hemoglobin SS-sickle-α thalassemia (0.7%). African–Americans and females predominated our cohort (97.1 and 65.4%, respectively). There was no known preference given to females or any other subject demographic during recruitment. The average age of our cohort was 34 years and ranged from 15 to 70 years.

### Table 1. . Subject demographics.

| Category | Values |
| --- | --- |
| Sickle cell type, n (%) |  |
|   |  |
| – SCD-SS | 105 (77.2) |
|   |  |
| – SCD-SC | 15 (11.0) |
|   |  |
| – SCD-Sβ+ | 8 (5.9) |
|   |  |
| – SCD-Sβ° | 7 (5.1) |
|   |  |
| – SCD-Sα | 1 (0.7) |
|   |  |
| Ethnicity, n (%) |  |
|   |  |
| – African–Americans | 132 (97.1) |
|   |  |
| – Hispanics | 3 (2.2) |
|   |  |
| – Caucasians | 1 (0.7) |
|   |  |
| Sex, n (%) |  |
|   |  |
| – Females | 89 (65.4) |
|   |  |
| – Males | 47 (34.6) |
|   |  |
| Age (years) |  |
|   |  |
| – Mean ± SD | 34.0 ± 11.7 |
|   |  |
| – Minimum | 15 |
|   |  |
| – Maximum | 70 |

Table 1 Caption: SCD-Sα: Sickle cell disease-sickle α thalassemia; SCD-Sβ+: Sickle cell disease-sickle β+ thalassemia; SCD-Sβ°: Sickle cell disease-sickle β° thalassemia; SCD-SC: Sickle cell disease-sickle hemoglobin C; SCD-SS: Sickle cell disease-homozygous hemoglobin S, sickle cell anemia.

A phenotype summary of CPI and utilization is given in [Table 2](#T2). The mean CPI in our cohort was 40.4, with a wide range from 14.8 to 86.5. Utilization had an average of 4.4 utilizations/year with a range from 0 to 38 utilizations. [Figure 1](#F0001) shows the distribution of subjects and utilizations. There were 19 subjects with no utilization, of whom 14 were hemoglobin SS type. Also, one subject with hemoglobin Sβ° genotype had 38 utilizations (the highest number of utilizations) within 1 year of completing the baseline pain assessment. [Figure 2](#F0002) depicts the distribution of sickle cell genotypes of subjects by utilization groups. Hemoglobin SS and hemoglobin Sβ° represent the most severe form of SCD and account for a majority of subjects in each utilization group.

### Table 2. . Phenotype summary.

| Category | CPI | Utilization |
| --- | --- | --- |
| Mean ± standard deviation | 40.4 ± 13.5 | 4.4 ± 5.2 |
|   |  |  |
| Minimum | 14.8 | 0 |
|   |  |  |
| Maximum | 86.5 | 38 |

Table 2 Caption: Phenotype summary is provided for all 136 subjects. CPI: Composite pain index

### Figure 1. . Distribution of subjects based on the number of utilizations.

![Figure 1. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1444/6275559/edb7a23a9f6d/pgs-19-1003-g1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6275559_pgs-19-1003-g1.jpg)

### Figure 2. . Distribution of sickle cell genotype of subjects in different utilization groups.

![Figure 2. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1444/6275559/c2754bd07810/pgs-19-1003-g2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6275559_pgs-19-1003-g2.jpg)

Others include sickle hemoglobin C, sickle β+ thalassemia and sickle α thalassemia. HbSβ°: Hemoglobin Sβ°; HbSS: Hemoglobin SS.

*NR3C1* SNP genotype and allele frequencies are shown in [Table 3](#T3). The SNPs are listed in order of chromosome position according to the National Center for Biotechnology Information GRCh38 assembly. *NR3C1* gene is located on chromosome 5. The major allele is listed first as all analyses are performed with the major allele/genotype as reference. No significant deviations were detected from Hardy–Weinberg equilibrium for all three SNPs (p > 0.05).

### Table 3. . Genotype and allele frequencies, n (%).

| dbSNPID | Chromosome location† | Dominant allele | Recessive allele | Dominant homozygote | Heterozygote | Recessive homozygote |
| --- | --- | --- | --- | --- | --- | --- |
| 33389 | 143320934 | C, 230 (85.2) | T, 40 (14.8) | 98 (72.6) | 34 (25.2) | 3 (2.2) |
|   |  |  |  |  |  |  |
| 2963155 | 143376439 | A, 182 (74.0) | G, 64 (26.0) | 68 (55.3) | 46 (37.4) | 9 (7.3) |
|   |  |  |  |  |  |  |
| 9324918 | 143387595 | T, 228 (83.8) | C, 44 (16.2) | 97 (71.3) | 34 (25.0) | 5 (3.7) |

Table 3 Caption: †Chromosome position is from National Center for Biotechnology Information GRCh38 assembly. dbSNP IDs are from the National Center for Biotechnology Information database.

In the regression analysis shown in [Table 4](#T4), the rs33389 T allele was associated with higher utilization (IRR = 1.53; 95% CI: 1.09–2.15; p = 0.014). Each T allele is additive in this model. The dominant model was not significant; however, the recessive model showed that the combined TT and CT genotypes had 1.64-times the utilization of the CC genotype (IRR = 1.64; [1.12–2.40]; p = 0.011). For rs2963155, all three models were statistically significant: G allele was associated with higher utilization rate (IRR = 1.80; [1.37–2.38]; p = 0.0000277), GG genotype had 2.25-times the utilization rate of combined GA and AA genotypes (IRR = 2.25; [1.13–4.49]; p = 0.021) and combined GG and GA genotypes had 2.07-times the utilization rate of AA genotype (IRR = 2.07; [1.45–2.93]; p < 0.001). The additive model was the only one that showed statistical significance for rs9324918 where the C allele was associated with higher utilization rate (IRR = 1.43; [1.06–1.93]; p = 0.021). None of the *NR3C1* SNPs was significantly associated with CPI in our cohort. Regression analyses adjusted for covariates are shown in [Table 5](#T5).

### Table 4. . Evaluating the effect of NR3C1 SNPs on utilizations.

| dbSNP ID | Model | IRR (95% CI)† | p-value |
| --- | --- | --- | --- |
| 33389 | Additive | 1.53 (1.09, 2.15) | 0.014 |
|   |  |  |  |
|   | Dominant | 1.61 (0.54, 4.84) | 0.394 |
|   |  |  |  |
|   | Recessive | 1.64 (1.12, 2.40) | 0.011 |
|   |  |  |  |
| 2963155 | Additive | 1.80 (1.37, 2.38) | 0.000‡ |
|   |  |  |  |
|   | Dominant | 2.25 (1.13, 4.49) | 0.021 |
|   |  |  |  |
|   | Recessive | 2.07 (1.45, 2.93) | 0.000§ |
|   |  |  |  |
| 9324918 | Additive | 1.43 (1.06, 1.93) | 0.021 |
|   |  |  |  |
|   | Dominant | 2.22 (0.96, 5.12) | 0.061 |
|   |  |  |  |
|   | Recessive | 1.46 (1.00, 2.12) | 0.050 |

Table 4 Caption: †Incident rate ratio and 95% CI. ‡p = 0.00003. §p = 0.00005. Significant p-values (p < 0.05) are in bold. The major alleles are references in the analyses. Negative binomial regression models are adjusted for age, sex, ethnicity and sickle cell type.

### Table 5. . Evaluating the effects of NR3C1 SNPs on composite pain index.

| dbSNP ID | Model | B (95% CI)† | p-value |
| --- | --- | --- | --- |
| 33389 | Additive | -0.38 (-5.03–4.27) | 0.872 |
|   |  |  |  |
|   | Dominant | 8.03 (-7.94–24.01) | 0.321 |
|   |  |  |  |
|   | Recessive | -1.35 (-6.62–3.91) | 0.612 |
|   |  |  |  |
| 2963155 | Additive | 2.81 (-1.11–6.72) | 0.158 |
|   |  |  |  |
|   | Dominant | 5.22 (-4.47–14.90) | 0.288 |
|   |  |  |  |
|   | Recessive | 3.11 (-1.85–8.07) | 0.216 |
|   |  |  |  |
| 9324918 | Additive | -1.91 (-6.21–2.38) | 0.380 |
|   |  |  |  |
|   | Dominant | -0.20 (-12.57–12.17) | 0.975 |
|   |  |  |  |
|   | Recessive | -2.70 (-7.83–2.43) | 0.299 |

Table 5 Caption: †Unstandardized regression coefficient and 95% CI. The major alleles are references in the analyses. Multiple linear regression model was adjusted for age, sex, ethnicity and sickle cell type.

The linkage-disequilibrium plot generated from Haploview did not result in any haplotype blocks ([Figure 3](#F0003)); therefore, haplotype analysis was not performed. We note, however, strong linkage-disequilibrium between rs33389 and rs2963155, and also rs33389 and rs9324918 as defined by confidence bounds on D′ where the one-sided upper 95% confidence bound is >0.98 and lower bound is >0.7 [[32](#B32)]. Only self-reported African–Americans are included for generating the linkage-disequilibrium plot.

### Figure 3. . Linkage disequilibrium plot of NR3C1 SNPs.

![Figure 3. ](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1444/6275559/b714512f6a55/pgs-19-1003-g3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6275559_pgs-19-1003-g3.jpg)

Haplotype block organization of NR3C1 SNPs generated from Haploview 4.2 using the standard D′/LOD color scheme. D′ values show the linkage disequilibrium coefficient. LOD is the logarithm of odds of two loci having linkage disequilibrium. Red = high D′ and high LOD, white = low D′ and low LOD; blue = high D′ and low LOD; shades of pink/red = low D′ and high LOD. Only self-reported African–American subjects are included (n = 132). Upper bound for D′ between rs33389 and rs2963155 is 0.99 while lower bound is 0.8. For rs33389 and rs9324918, upper bound for D′ is 1.0 and lower bound is 0.88.

## Discussion

This is the first study reporting the association of three intronic SNPs (rs33389, rs2963155 and rs9324918) in the *NR3C1* gene with painful crisis in SCD. The minor alleles of these three SNPs were associated with higher utilization rate, used as a surrogate marker for painful crisis that is commonly referred to as vaso-occlusive crisis [[35](#B35)]. The patients with the combined TT and CT genotypes of rs33389 or rs2963155 GG genotype had greater utilization rates. Our study did not find any association of these *NR3C1* SNPs with CPI, which was used as a measurement for chronic pain in patients with SCD [[19](#B19),[21](#B21)].

SNPs rs33389 and rs2963155 lie within ten base-pairs of putative splicing junctions (as predicted by SplicePort [[http://spliceport.cbcb.umd.edu/](http://spliceport.cbcb.umd.edu/)] and NetGene2 [[www.cbs.dtu.dk/services/NetGene2/](http://www.cbs.dtu.dk/services/NetGene2/)]) [[33](#B33),[34](#B34)]. Therefore, these SNPs could potentially affect alternative splicing that may affect GR function and/or expression, although experiments will be needed to determine alternative splicing.

The* NR3C1* encodes for the GR, which is expressed in the cytoplasm and is transported into the nucleus upon binding to glucocorticoid hormones, which can then act as a transcription factor by binding to the glucocorticoid response elements on genes, thus regulating other transcription factors and genes [[36–39](#B36)]. Several lines of evidence support show the involvement of GR in pain. Inflammation triggered by tissue injuries can be tightly regulated by glucocorticoid hormones and GR [[40](#B40),[41](#B41)]. In addition, GR and glucocorticoid hormones may affect pain through their role in regulating stress [[42–44](#B42)]. Prolonged incision-induced hypersensitivity to mechanical, thermal and cold stimuli was observed in rats that underwent short-term stress by forced swimming or immobilization postoperatively, and blocking with a GR antagonist attenuated this prolongation [[17](#B17)]. Preterm infants that experience stress and pain stimuli in the neonatal intensive care unit show blunted cortisol reactivity (a glucocorticoid that binds and activates GRs) later in life compared with controls [[45](#B45)]. There is blunted cortisol release both basally and due to acute stress as well. In addition, both human and animal studies show that early life pain results in general thermal or mechanical hypoalgesia and a decrease in the release of stress hormones due to acute stressors later in life; however, chronic or severe stressors result in significant hyperalgesia in this same group. The interactions appeared to be both way. For examples, chronic water avoidance stress in rats led to increased methylation of the *NR3C1* promoter region, decreased mRNA and decreased GR protein in the dorsal root ganglion sensory neurons [[46](#B46)]. Patients with SCD may have dysfunction hypothalamic–pituitary–adrenal axis and abnormal regulation of GR. Cortisol was found to be significantly lower in patients with SCD when compared with healthy subjects with HbAA genotype; however, levels were significantly higher during a painful crisis compared with subjects at steady state [[47](#B47)].

Stress reactivity is reduced with opioid administration and cortisol levels increase with opioid receptor antagonists [[45](#B45)]. The hypothalamic–pituitary–adrenal axis and the endogenous opioid system interact as corticotropin-releasing factor and its receptor are colocalized and co-expressed with endogenous opioids, and are both transcriptionally regulated by the GR [[45](#B45)]. It was observed that pain intensity and frequency along with sickle cell-related stress were found to be higher among subjects who used opioids compared with those who did not [[48](#B48)]. Sickle cell subjects with hypothalamic–pituitary–adrenal axis dysfunction may show altered stress reactivity with opioid administration [[49](#B49)]. It would be interesting to observe the relationship between opioid use [[3](#B3),[48](#B48)], stress, pain and *NR3C1* SNPs in SCD. A limitation of our study is that we do not have information on fetal hemoglobin levels and hydroxyurea use at the time of the sample acquisition for genomic analysis. Our center contributed a large number of subjects to the HU study, and patients continued on it if they tolerated the drug. Besides drug use and treatment regime, other factors such as age, sex, ethnicity and sickle cell genotype can also influence phenotypic outcomes. We have adjusted for these potentially confounding variables in our analyses. The current study is not a case–controlled study. Instead, sickle cell patients were grouped into different utilization groups for statistical analysis. We are in the process of conducting a case–control study and plan to replicate the study with the new cohorts.

The minor alleles of these SNPs were associated with a greater risk for utilization in SCD. These results were different from the reported protective effect against TMD exhibited by the minor alleles of these three SNPs [[18](#B18)]. Although TMD is characterized by orofacial pain, it represents a musculoskeletal disorder involving the muscles in the jaw, face and nearby regions. Pain is only one component of TMD, whereas sickle cell pain has very different clinical presentations in the cause, location, severity, on-set time and duration of pain [[1–3](#B1)].

In conclusion, by investigating *NR3C1* polymorphisms on acute pain crisis, we found that *NR3C1* may influence the frequency of the crisis pain in SCD. Such information will not only help to advance our understanding of genetic factors influencing pain heterogeneity but may ultimately provide a new approach to achieve precise medicine for acute pain in SCD.

## Footnotes

## References

1. Platt OS, Thorington BD, Brambilla DJ, et al. Pain in sickle cell disease. Rates and risk factors. N. Engl. J. Med. 1991;325(1):11–16. doi: 10.1056/NEJM199107043250103.  [DOI](https://doi.org/10.1056/NEJM199107043250103) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1710777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Pain%20in%20sickle%20cell%20disease.%20Rates%20and%20risk%20factors&author=OS%20Platt&author=BD%20Thorington&author=DJ%20Brambilla&volume=325&issue=1&publication_year=1991&pages=11-16&pmid=1710777&doi=10.1056/NEJM199107043250103&)

2. Ballas SK, Gupta K, Adams-Graves P. Sickle cell pain: a critical reappraisal. Blood. 2012;120(18):3647–3656. doi: 10.1182/blood-2012-04-383430.  [DOI](https://doi.org/10.1182/blood-2012-04-383430) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22923496/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Sickle%20cell%20pain:%20a%20critical%20reappraisal&author=SK%20Ballas&author=K%20Gupta&author=P%20Adams-Graves&volume=120&issue=18&publication_year=2012&pages=3647-3656&pmid=22923496&doi=10.1182/blood-2012-04-383430&)

3. Wilkie DJ, Molokie R, Boyd-Seal D, et al. Patient-reported outcomes: descriptors of nociceptive and neuropathic pain and barriers to effective pain management in adult outpatients with sickle cell disease. J. Natl Med. Assoc. 2010;102(1):18–27. doi: 10.1016/s0027-9684(15)30471-5.  [DOI](https://doi.org/10.1016/s0027-9684(15)30471-5) | [PMC free article](/articles/PMC3641147/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20158132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl%20Med.%20Assoc.&title=Patient-reported%20outcomes:%20descriptors%20of%20nociceptive%20and%20neuropathic%20pain%20and%20barriers%20to%20effective%20pain%20management%20in%20adult%20outpatients%20with%20sickle%20cell%20disease&author=DJ%20Wilkie&author=R%20Molokie&author=D%20Boyd-Seal&volume=102&issue=1&publication_year=2010&pages=18-27&pmid=20158132&doi=10.1016/s0027-9684(15)30471-5&)

4. Rees DC, Williams TN, Gladwin MT. Sickle-cell disease. Lancet. 2010;376(9757):2018–2031. doi: 10.1016/S0140-6736(10)61029-X.  [DOI](https://doi.org/10.1016/S0140-6736(10)61029-X) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21131035/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Sickle-cell%20disease&author=DC%20Rees&author=TN%20Williams&author=MT%20Gladwin&volume=376&issue=9757&publication_year=2010&pages=2018-2031&pmid=21131035&doi=10.1016/S0140-6736(10)61029-X&)

5. Molokie RE, Wang ZJ, Wilkie DJ. Presence of neuropathic pain as an underlying mechanism for pain associated with cold weather in patients with sickle cell disease. Med. Hypotheses. 2011;77(4):491–493. doi: 10.1016/j.mehy.2011.06.018.  [DOI](https://doi.org/10.1016/j.mehy.2011.06.018) | [PMC free article](/articles/PMC3701405/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21763079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med.%20Hypotheses&title=Presence%20of%20neuropathic%20pain%20as%20an%20underlying%20mechanism%20for%20pain%20associated%20with%20cold%20weather%20in%20patients%20with%20sickle%20cell%20disease&author=RE%20Molokie&author=ZJ%20Wang&author=DJ%20Wilkie&volume=77&issue=4&publication_year=2011&pages=491-493&pmid=21763079&doi=10.1016/j.mehy.2011.06.018&)

6. Wang ZJ, Wilkie DJ, Molokie R. Neurobiological mechanisms of pain in sickle cell disease. Hematology Am. Soc. Hematol. Educ. Program. 2010;2010:403–408. doi: 10.1182/asheducation-2010.1.403.  [DOI](https://doi.org/10.1182/asheducation-2010.1.403) | [PMC free article](/articles/PMC3650026/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21239826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hematology%20Am.%20Soc.%20Hematol.%20Educ.%20Program&title=Neurobiological%20mechanisms%20of%20pain%20in%20sickle%20cell%20disease&author=ZJ%20Wang&author=DJ%20Wilkie&author=R%20Molokie&volume=2010&publication_year=2010&pages=403-408&pmid=21239826&doi=10.1182/asheducation-2010.1.403&)

7. Smith WR, Penberthy LT, Bovbjerg VE, et al. Daily assessment of pain in adults with sickle cell disease. Ann. Intern. Med. 2008;148(2):94–101. doi: 10.7326/0003-4819-148-2-200801150-00004.  [DOI](https://doi.org/10.7326/0003-4819-148-2-200801150-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18195334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Intern.%20Med.&title=Daily%20assessment%20of%20pain%20in%20adults%20with%20sickle%20cell%20disease&author=WR%20Smith&author=LT%20Penberthy&author=VE%20Bovbjerg&volume=148&issue=2&publication_year=2008&pages=94-101&pmid=18195334&doi=10.7326/0003-4819-148-2-200801150-00004&)

8. Jhun E, He Y, Yao Y, Molokie RE, Wilkie DJ, Wang ZJ. Dopamine D3 receptor Ser9Gly and catechol-o-methyltransferase Val158Met polymorphisms and acute pain in sickle cell disease. Anesth. Analg. 2014;119(5):1201–1207. doi: 10.1213/ANE.0000000000000382.  [DOI](https://doi.org/10.1213/ANE.0000000000000382) | [PMC free article](/articles/PMC4205211/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25102390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth.%20Analg.&title=Dopamine%20D3%20receptor%20Ser9Gly%20and%20catechol-o-methyltransferase%20Val158Met%20polymorphisms%20and%20acute%20pain%20in%20sickle%20cell%20disease&author=E%20Jhun&author=Y%20He&author=Y%20Yao&author=RE%20Molokie&author=DJ%20Wilkie&volume=119&issue=5&publication_year=2014&pages=1201-1207&pmid=25102390&doi=10.1213/ANE.0000000000000382&)

9. Jhun EH, Hu X, Sadhu N, et al. Transient receptor potential polymorphism and haplotype associate with crisis pain in sickle cell disease. Pharmacogenomics. 2018;19(5):401–411. doi: 10.2217/pgs-2017-0198.  [DOI](https://doi.org/10.2217/pgs-2017-0198) | [PMC free article](/articles/PMC6275563/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29620434/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Transient%20receptor%20potential%20polymorphism%20and%20haplotype%20associate%20with%20crisis%20pain%20in%20sickle%20cell%20disease&author=EH%20Jhun&author=X%20Hu&author=N%20Sadhu&volume=19&issue=5&publication_year=2018&pages=401-411&pmid=29620434&doi=10.2217/pgs-2017-0198&)

10. Hu X, Jhun EH, Yao Y, et al. IL1A rs1800587 associates with chronic noncrisis pain in sickle cell disease. Pharmacogenomics. 2016;17(18):1999–2006. doi: 10.2217/pgs-2016-0085.  [DOI](https://doi.org/10.2217/pgs-2016-0085) | [PMC free article](/articles/PMC5220521/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27883292/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=IL1A%20rs1800587%20associates%20with%20chronic%20noncrisis%20pain%20in%20sickle%20cell%20disease&author=X%20Hu&author=EH%20Jhun&author=Y%20Yao&volume=17&issue=18&publication_year=2016&pages=1999-2006&pmid=27883292&doi=10.2217/pgs-2016-0085&)

11. Belfer I, Youngblood V, Darbari DS, et al. A GCH1 haplotype confers sex-specific susceptibility to pain crises and altered endothelial function in adults with sickle cell anemia. Am. J. Hematol. 2014;89(2):187–193. doi: 10.1002/ajh.23613.  [DOI](https://doi.org/10.1002/ajh.23613) | [PMC free article](/articles/PMC4281092/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24136375/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hematol.&title=A%20GCH1%20haplotype%20confers%20sex-specific%20susceptibility%20to%20pain%20crises%20and%20altered%20endothelial%20function%20in%20adults%20with%20sickle%20cell%20anemia&author=I%20Belfer&author=V%20Youngblood&author=DS%20Darbari&volume=89&issue=2&publication_year=2014&pages=187-193&pmid=24136375&doi=10.1002/ajh.23613&)

12. Lettre G, Sankaran VG, Bezerra MA, et al. DNA polymorphisms at the BCL11A, HBS1L-MYB and beta-globin loci associate with fetal hemoglobin levels and pain crises in sickle cell disease. Proc. Natl Acad. Sci. USA. 2008;105(33):11869–11874. doi: 10.1073/pnas.0804799105.  [DOI](https://doi.org/10.1073/pnas.0804799105) | [PMC free article](/articles/PMC2491485/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18667698/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=DNA%20polymorphisms%20at%20the%20BCL11A,%20HBS1L-MYB%20and%20beta-globin%20loci%20associate%20with%20fetal%20hemoglobin%20levels%20and%20pain%20crises%20in%20sickle%20cell%20disease&author=G%20Lettre&author=VG%20Sankaran&author=MA%20Bezerra&volume=105&issue=33&publication_year=2008&pages=11869-11874&pmid=18667698&doi=10.1073/pnas.0804799105&)

13. Mendonça TF, Oliveira MC, Vasconcelos LR, et al. Association of variant alleles of MBL2 gene with vasoocclusive crisis in children with sickle cell anemia. Blood Cells Mol. Dis. 2010;44(4):224–228. doi: 10.1016/j.bcmd.2010.02.004.  [DOI](https://doi.org/10.1016/j.bcmd.2010.02.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20172753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Cells%20Mol.%20Dis.&title=Association%20of%20variant%20alleles%20of%20MBL2%20gene%20with%20vasoocclusive%20crisis%20in%20children%20with%20sickle%20cell%20anemia&author=TF%20Mendon%C3%A7a&author=MC%20Oliveira&author=LR%20Vasconcelos&volume=44&issue=4&publication_year=2010&pages=224-228&pmid=20172753&doi=10.1016/j.bcmd.2010.02.004&)

14. Encio IJ, Detera-Wadleigh SD. The genomic structure of the human glucocorticoid receptor. J. Biol. Chem. 1991;266(11):7182–7188.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1707881/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=The%20genomic%20structure%20of%20the%20human%20glucocorticoid%20receptor&author=IJ%20Encio&author=SD%20Detera-Wadleigh&volume=266&issue=11&publication_year=1991&pages=7182-7188&pmid=1707881&)

15. Krupoves A, Mack D, Deslandres C, Seidman E, Amre DK. Variation in the glucocorticoid receptor gene (NR3C1) may be associated with corticosteroid dependency and resistance in children with Crohn's disease. Pharmacogenet. Genomics. 2011;21(8):454–460. doi: 10.1097/FPC.0b013e3283476a01.  [DOI](https://doi.org/10.1097/FPC.0b013e3283476a01) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21633323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Variation%20in%20the%20glucocorticoid%20receptor%20gene%20(NR3C1)%20may%20be%20associated%20with%20corticosteroid%20dependency%20and%20resistance%20in%20children%20with%20Crohn's%20disease&author=A%20Krupoves&author=D%20Mack&author=C%20Deslandres&author=E%20Seidman&author=DK%20Amre&volume=21&issue=8&publication_year=2011&pages=454-460&pmid=21633323&doi=10.1097/FPC.0b013e3283476a01&)

16. Chatzikyriakidou A, Georgiou I, Voulgari PV, Georgiadis AN, Argyriou ES, Drosos AA. Glucocorticoid receptor variants may predispose to rheumatoid arthritis susceptibility. Scand. J. Rheumatol. 2009;38(1):1–5. doi: 10.1080/03009740802366068.  [DOI](https://doi.org/10.1080/03009740802366068) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18830906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Scand.%20J.%20Rheumatol.&title=Glucocorticoid%20receptor%20variants%20may%20predispose%20to%20rheumatoid%20arthritis%20susceptibility&author=A%20Chatzikyriakidou&author=I%20Georgiou&author=PV%20Voulgari&author=AN%20Georgiadis&author=ES%20Argyriou&volume=38&issue=1&publication_year=2009&pages=1-5&pmid=18830906&doi=10.1080/03009740802366068&)

17. Cao J, Wang PK, Tiwari V, et al. Short-term pre- and post-operative stress prolongs incision-induced pain hypersensitivity without changing basal pain perception. Mol. Pain. 2015;11(1):73. doi: 10.1186/s12990-015-0077-3.  [DOI](https://doi.org/10.1186/s12990-015-0077-3) | [PMC free article](/articles/PMC4667457/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26626404/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pain&title=Short-term%20pre-%20and%20post-operative%20stress%20prolongs%20incision-induced%20pain%20hypersensitivity%20without%20changing%20basal%20pain%20perception&author=J%20Cao&author=PK%20Wang&author=V%20Tiwari&volume=11&issue=1&publication_year=2015&pages=73&pmid=26626404&doi=10.1186/s12990-015-0077-3&)

18. Smith SB, Maixner DW, Greenspan JD, et al. Potential genetic risk factors for chronic TMD: genetic associations from the OPPERA case control study. J. Pain. 2011;12(11 Suppl.):T92–T101. doi: 10.1016/j.jpain.2011.08.005.  [DOI](https://doi.org/10.1016/j.jpain.2011.08.005) | [PMC free article](/articles/PMC3268684/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22074755/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pain&title=Potential%20genetic%20risk%20factors%20for%20chronic%20TMD:%20genetic%20associations%20from%20the%20OPPERA%20case%20control%20study&author=SB%20Smith&author=DW%20Maixner&author=JD%20Greenspan&volume=12&issue=11%20Suppl.&publication_year=2011&pages=T92-T101&pmid=22074755&doi=10.1016/j.jpain.2011.08.005&)

19. Ezenwa MO, Molokie RE, Wang ZJ, et al. Outpatient pain predicts subsequent one-year acute health care utilization among adults with sickle cell disease. J. Pain Symptom Manage. 2014;48(1):65–74. doi: 10.1016/j.jpainsymman.2013.08.020.  [DOI](https://doi.org/10.1016/j.jpainsymman.2013.08.020) | [PMC free article](/articles/PMC4082743/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24636960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pain%20Symptom%20Manage.&title=Outpatient%20pain%20predicts%20subsequent%20one-year%20acute%20health%20care%20utilization%20among%20adults%20with%20sickle%20cell%20disease&author=MO%20Ezenwa&author=RE%20Molokie&author=ZJ%20Wang&volume=48&issue=1&publication_year=2014&pages=65-74&pmid=24636960&doi=10.1016/j.jpainsymman.2013.08.020&)

20. Melzack R. The McGill pain questionnaire: major properties and scoring methods. Pain. 1975;1:277–299. doi: 10.1016/0304-3959(75)90044-5.  [DOI](https://doi.org/10.1016/0304-3959(75)90044-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1235985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain&title=The%20McGill%20pain%20questionnaire:%20major%20properties%20and%20scoring%20methods&author=R%20Melzack&volume=1&publication_year=1975&pages=277-299&pmid=1235985&doi=10.1016/0304-3959(75)90044-5&)

21. Wilkie DJ, Molokie RE, Suarez ML, Ezenwa MO, Wang ZJ. Composite Pain Index: reliability, validity and sensitivity of a patient-reported outcome for research. Pain Med. 2015;16(7):1341–1348. doi: 10.1111/pme.12703.  [DOI](https://doi.org/10.1111/pme.12703) | [PMC free article](/articles/PMC4504760/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25712169/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain%20Med.&title=Composite%20Pain%20Index:%20reliability,%20validity%20and%20sensitivity%20of%20a%20patient-reported%20outcome%20for%20research&author=DJ%20Wilkie&author=RE%20Molokie&author=ML%20Suarez&author=MO%20Ezenwa&author=ZJ%20Wang&volume=16&issue=7&publication_year=2015&pages=1341-1348&pmid=25712169&doi=10.1111/pme.12703&)

22. Ngamkham S, Holden JE, Wilkie DJ. Differences in pain location, intensity and quality by pain pattern in outpatients with cancer. Cancer Nurs. 2011;34(3):228–237. doi: 10.1097/NCC.0b013e3181faab63.  [DOI](https://doi.org/10.1097/NCC.0b013e3181faab63) | [PMC free article](/articles/PMC3703616/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21512345/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Nurs.&title=Differences%20in%20pain%20location,%20intensity%20and%20quality%20by%20pain%20pattern%20in%20outpatients%20with%20cancer&author=S%20Ngamkham&author=JE%20Holden&author=DJ%20Wilkie&volume=34&issue=3&publication_year=2011&pages=228-237&pmid=21512345&doi=10.1097/NCC.0b013e3181faab63&)

23. Wilkie DJ, Judge MK, Berry DL, Dell J, Zong S, Gilespie R. Usability of a computerized PAINReportIt in the general public with pain and people with cancer pain. J. Pain Symptom Manage. 2003;25(3):213–224. doi: 10.1016/s0885-3924(02)00638-3.  [DOI](https://doi.org/10.1016/s0885-3924(02)00638-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12614956/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pain%20Symptom%20Manage.&title=Usability%20of%20a%20computerized%20PAINReportIt%20in%20the%20general%20public%20with%20pain%20and%20people%20with%20cancer%20pain&author=DJ%20Wilkie&author=MK%20Judge&author=DL%20Berry&author=J%20Dell&author=S%20Zong&volume=25&issue=3&publication_year=2003&pages=213-224&pmid=12614956&doi=10.1016/s0885-3924(02)00638-3&)

24. Vandenbergh DJ, Anthony K, Whitfield KE. Optimizing DNA yield from buccal swabs in the elderly: attempts to promote buccal cell growth in culture. Am. J. Hum. Biol. 2003;15(5):637–642. doi: 10.1002/ajhb.10177.  [DOI](https://doi.org/10.1002/ajhb.10177) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12953175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Biol.&title=Optimizing%20DNA%20yield%20from%20buccal%20swabs%20in%20the%20elderly:%20attempts%20to%20promote%20buccal%20cell%20growth%20in%20culture&author=DJ%20Vandenbergh&author=K%20Anthony&author=KE%20Whitfield&volume=15&issue=5&publication_year=2003&pages=637-642&pmid=12953175&doi=10.1002/ajhb.10177&)

25. Miller SA, Dykes DD, Polesky HF. A simple salting out procedure for extracting DNA from human nucleated cells. Nucleic Acids Res. 1988;16(3):1215. doi: 10.1093/nar/16.3.1215.  [DOI](https://doi.org/10.1093/nar/16.3.1215) | [PMC free article](/articles/PMC334765/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3344216/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=A%20simple%20salting%20out%20procedure%20for%20extracting%20DNA%20from%20human%20nucleated%20cells&author=SA%20Miller&author=DD%20Dykes&author=HF%20Polesky&volume=16&issue=3&publication_year=1988&pages=1215&pmid=3344216&doi=10.1093/nar/16.3.1215&)

26. Gabriel S, Ziaugra L, Tabbaa D. SNP genotyping using the Sequenom MassARRAY iPLEX platform. Curr. Protoc. Hum. Genet. 2009;Chapter 2(Unit 2):12. doi: 10.1002/0471142905.hg0212s60.  [DOI](https://doi.org/10.1002/0471142905.hg0212s60) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19170031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.%20Hum.%20Genet.&title=SNP%20genotyping%20using%20the%20Sequenom%20MassARRAY%20iPLEX%20platform&author=S%20Gabriel&author=L%20Ziaugra&author=D%20Tabbaa&volume=Chapter%202&issue=Unit%202&publication_year=2009&pages=12&pmid=19170031&doi=10.1002/0471142905.hg0212s60&)

27. SPSS Data Analysis Examples. UCLA: Statistical Consulting Group; Negative Binomial Regression.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=SPSS%20Data%20Analysis%20Examples&)

28. Cameron A, Trivedi PK. Regression Analysis of Count Data. Cambridge University Press; Cambridge, UK: 1998.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Regression%20Analysis%20of%20Count%20Data&author=A%20Cameron&author=PK%20Trivedi&publication_year=1998&)

29. Lettre G, Lange C, Hirschhorn JN. Genetic model testing and statistical power in population-based association studies of quantitative traits. Genet. Epidemiol. 2007;31(4):358–362. doi: 10.1002/gepi.20217.  [DOI](https://doi.org/10.1002/gepi.20217) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17352422/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Epidemiol.&title=Genetic%20model%20testing%20and%20statistical%20power%20in%20population-based%20association%20studies%20of%20quantitative%20traits&author=G%20Lettre&author=C%20Lange&author=JN%20Hirschhorn&volume=31&issue=4&publication_year=2007&pages=358-362&pmid=17352422&doi=10.1002/gepi.20217&)

30. Clarke GM, Anderson CA, Pettersson FH, Cardon LR, Morris AP, Zondervan KT. Basic statistical analysis in genetic case–control studies. Nat. Protoc. 2011;6(2):121–133. doi: 10.1038/nprot.2010.182.  [DOI](https://doi.org/10.1038/nprot.2010.182) | [PMC free article](/articles/PMC3154648/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21293453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&title=Basic%20statistical%20analysis%20in%20genetic%20case%E2%80%93control%20studies&author=GM%20Clarke&author=CA%20Anderson&author=FH%20Pettersson&author=LR%20Cardon&author=AP%20Morris&volume=6&issue=2&publication_year=2011&pages=121-133&pmid=21293453&doi=10.1038/nprot.2010.182&)

31. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21(2):263–265. doi: 10.1093/bioinformatics/bth457.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&issue=2&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

32. Gabriel SB, Schaffner SF, Nguyen H, et al. The structure of haplotype blocks in the human genome. Science. 2002;296(5576):2225–2229. doi: 10.1126/science.1069424.  [DOI](https://doi.org/10.1126/science.1069424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12029063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20structure%20of%20haplotype%20blocks%20in%20the%20human%20genome&author=SB%20Gabriel&author=SF%20Schaffner&author=H%20Nguyen&volume=296&issue=5576&publication_year=2002&pages=2225-2229&pmid=12029063&doi=10.1126/science.1069424&)

33. Dogan RI, Getoor L, Wilbur WJ, Mount SM. SplicePort–an interactive splice-site analysis tool. Nucleic Acids Res. 2007;35(Web Server issue):W285–W291. doi: 10.1093/nar/gkm407.  [DOI](https://doi.org/10.1093/nar/gkm407) | [PMC free article](/articles/PMC1933122/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17576680/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=SplicePort%E2%80%93an%20interactive%20splice-site%20analysis%20tool&author=RI%20Dogan&author=L%20Getoor&author=WJ%20Wilbur&author=SM%20Mount&volume=35&issue=Web%20Server%20issue&publication_year=2007&pages=W285-W291&pmid=17576680&doi=10.1093/nar/gkm407&)

34. Brunak S, Engelbrecht J, Knudsen S. Prediction of human mRNA donor and acceptor sites from the DNA sequence. J. Mol. Biol. 1991;220(1):49–65. doi: 10.1016/0022-2836(91)90380-o.  [DOI](https://doi.org/10.1016/0022-2836(91)90380-o) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2067018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&title=Prediction%20of%20human%20mRNA%20donor%20and%20acceptor%20sites%20from%20the%20DNA%20sequence&author=S%20Brunak&author=J%20Engelbrecht&author=S%20Knudsen&volume=220&issue=1&publication_year=1991&pages=49-65&pmid=2067018&doi=10.1016/0022-2836(91)90380-o&)

35. Molokie RE, Montminy C, Dionisio C, et al. Opioid doses and acute care utilization outcomes for adults with sickle cell disease: ED versus acute care unit. Am. J. Emerg. Med. 2018;36(1):88–92. doi: 10.1016/j.ajem.2017.07.037.  [DOI](https://doi.org/10.1016/j.ajem.2017.07.037) | [PMC free article](/articles/PMC7083150/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28802541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Emerg.%20Med.&title=Opioid%20doses%20and%20acute%20care%20utilization%20outcomes%20for%20adults%20with%20sickle%20cell%20disease:%20ED%20versus%20acute%20care%20unit&author=RE%20Molokie&author=C%20Montminy&author=C%20Dionisio&volume=36&issue=1&publication_year=2018&pages=88-92&pmid=28802541&doi=10.1016/j.ajem.2017.07.037&)

36. Weikum ER, Knuesel MT, Ortlund EA, Yamamoto KR. Glucocorticoid receptor control of transcription: precision and plasticity via allostery. Nat. Rev. Mol. Cell. Biol. 2017;18(3):159–174. doi: 10.1038/nrm.2016.152.  [DOI](https://doi.org/10.1038/nrm.2016.152) | [PMC free article](/articles/PMC6257982/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28053348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Mol.%20Cell.%20Biol.&title=Glucocorticoid%20receptor%20control%20of%20transcription:%20precision%20and%20plasticity%20via%20allostery&author=ER%20Weikum&author=MT%20Knuesel&author=EA%20Ortlund&author=KR%20Yamamoto&volume=18&issue=3&publication_year=2017&pages=159-174&pmid=28053348&doi=10.1038/nrm.2016.152&)

37. Hayashi R, Wada H, Ito K, Adcock IM. Effects of glucocorticoids on gene transcription. Eur. J. Pharmacol. 2004;500(1–3):51–62. doi: 10.1016/j.ejphar.2004.07.011.  [DOI](https://doi.org/10.1016/j.ejphar.2004.07.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15464020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pharmacol.&title=Effects%20of%20glucocorticoids%20on%20gene%20transcription&author=R%20Hayashi&author=H%20Wada&author=K%20Ito&author=IM%20Adcock&volume=500&issue=1%E2%80%933&publication_year=2004&pages=51-62&pmid=15464020&doi=10.1016/j.ejphar.2004.07.011&)

38. Nezi M, Mastorakos G, Mouslech Z. Corticotropin releasing hormone and the immune/inflammatory response. In: De Groot LJ, Chrousos G, Dungan K, et al., editors. Endotext. South Dartmouth, MA, USA: 2000.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Endotext&author=M%20Nezi&author=G%20Mastorakos&author=Z%20Mouslech&publication_year=2000&)

39. Ratman D, Vanden Berghe W, Dejager L, et al. How glucocorticoid receptors modulate the activity of other transcription factors: a scope beyond tethering. Mol. Cell. Endocrinol. 2013;380(1–2):41–54. doi: 10.1016/j.mce.2012.12.014.  [DOI](https://doi.org/10.1016/j.mce.2012.12.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23267834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Endocrinol.&title=How%20glucocorticoid%20receptors%20modulate%20the%20activity%20of%20other%20transcription%20factors:%20a%20scope%20beyond%20tethering&author=D%20Ratman&author=W%20Vanden%20Berghe&author=L%20Dejager&volume=380&issue=1%E2%80%932&publication_year=2013&pages=41-54&pmid=23267834&doi=10.1016/j.mce.2012.12.014&)

40. Vandevyver S, Dejager L, Tuckermann J, Libert C. New insights into the anti-inflammatory mechanisms of glucocorticoids: an emerging role for glucocorticoid-receptor-mediated transactivation. Endocrinology. 2013;154(3):993–1007. doi: 10.1210/en.2012-2045.  [DOI](https://doi.org/10.1210/en.2012-2045) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23384835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocrinology&title=New%20insights%20into%20the%20anti-inflammatory%20mechanisms%20of%20glucocorticoids:%20an%20emerging%20role%20for%20glucocorticoid-receptor-mediated%20transactivation&author=S%20Vandevyver&author=L%20Dejager&author=J%20Tuckermann&author=C%20Libert&volume=154&issue=3&publication_year=2013&pages=993-1007&pmid=23384835&doi=10.1210/en.2012-2045&)

41. Dostert A, Heinzel T. Negative glucocorticoid receptor response elements and their role in glucocorticoid action. Curr. Pharm. Des. 2004;10(23):2807–2816. doi: 10.2174/1381612043383601.  [DOI](https://doi.org/10.2174/1381612043383601) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15379669/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Pharm.%20Des.&title=Negative%20glucocorticoid%20receptor%20response%20elements%20and%20their%20role%20in%20glucocorticoid%20action&author=A%20Dostert&author=T%20Heinzel&volume=10&issue=23&publication_year=2004&pages=2807-2816&pmid=15379669&doi=10.2174/1381612043383601&)

42. Oakley RH, Cidlowski JA. The biology of the glucocorticoid receptor: new signaling mechanisms in health and disease. J. Allergy Clin. Immunol. 2013;132(5):1033–1044. doi: 10.1016/j.jaci.2013.09.007.  [DOI](https://doi.org/10.1016/j.jaci.2013.09.007) | [PMC free article](/articles/PMC4084612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24084075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.&title=The%20biology%20of%20the%20glucocorticoid%20receptor:%20new%20signaling%20mechanisms%20in%20health%20and%20disease&author=RH%20Oakley&author=JA%20Cidlowski&volume=132&issue=5&publication_year=2013&pages=1033-1044&pmid=24084075&doi=10.1016/j.jaci.2013.09.007&)

43. Labeur M, Holsboer F. Molecular mechanisms of glucocorticoid receptor signaling. Medicina (B Aires) 2010;70(5):457–462.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/20920967/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Medicina%20(B%20Aires)&title=Molecular%20mechanisms%20of%20glucocorticoid%20receptor%20signaling&author=M%20Labeur&author=F%20Holsboer&volume=70&issue=5&publication_year=2010&pages=457-462&pmid=20920967&)

44. Bao AM, Meynen G, Swaab DF. The stress system in depression and neurodegeneration: focus on the human hypothalamus. Brain Res. Rev. 2008;57(2):531–553. doi: 10.1016/j.brainresrev.2007.04.005.  [DOI](https://doi.org/10.1016/j.brainresrev.2007.04.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17524488/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res.%20Rev.&title=The%20stress%20system%20in%20depression%20and%20neurodegeneration:%20focus%20on%20the%20human%20hypothalamus&author=AM%20Bao&author=G%20Meynen&author=DF%20Swaab&volume=57&issue=2&publication_year=2008&pages=531-553&pmid=17524488&doi=10.1016/j.brainresrev.2007.04.005&)

45. Victoria NC, Murphy AZ. The long-term impact of early life pain on adult responses to anxiety and stress: historical perspectives and empirical evidence. Exp. Neurol. 2016;275(Pt 2):261–273. doi: 10.1016/j.expneurol.2015.07.017.  [DOI](https://doi.org/10.1016/j.expneurol.2015.07.017) | [PMC free article](/articles/PMC4977201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26210872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp.%20Neurol.&title=The%20long-term%20impact%20of%20early%20life%20pain%20on%20adult%20responses%20to%20anxiety%20and%20stress:%20historical%20perspectives%20and%20empirical%20evidence&author=NC%20Victoria&author=AZ%20Murphy&volume=275&issue=Pt%202&publication_year=2016&pages=261-273&pmid=26210872&doi=10.1016/j.expneurol.2015.07.017&)

46. Hong S, Zheng G, Wiley JW. Epigenetic regulation of genes that modulate chronic stress-induced visceral pain in the peripheral nervous system. Gastroenterology. 2015;148(1):148–157. doi: 10.1053/j.gastro.2014.09.032.  [DOI](https://doi.org/10.1053/j.gastro.2014.09.032) | [PMC free article](/articles/PMC4274248/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25263804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Epigenetic%20regulation%20of%20genes%20that%20modulate%20chronic%20stress-induced%20visceral%20pain%20in%20the%20peripheral%20nervous%20system&author=S%20Hong&author=G%20Zheng&author=JW%20Wiley&volume=148&issue=1&publication_year=2015&pages=148-157&pmid=25263804&doi=10.1053/j.gastro.2014.09.032&)

47. Akinlade KS, Atere AD, Olaniyi JA, Rahamon SK, Adewale CO. Serum copeptin and cortisol do not accurately predict sickle cell anaemia vaso-occlusive crisis as C-reactive protein. PLoS ONE. 2013;8(11):e77913. doi: 10.1371/journal.pone.0077913.  [DOI](https://doi.org/10.1371/journal.pone.0077913) | [PMC free article](/articles/PMC3817164/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24223742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Serum%20copeptin%20and%20cortisol%20do%20not%20accurately%20predict%20sickle%20cell%20anaemia%20vaso-occlusive%20crisis%20as%20C-reactive%20protein&author=KS%20Akinlade&author=AD%20Atere&author=JA%20Olaniyi&author=SK%20Rahamon&author=CO%20Adewale&volume=8&issue=11&publication_year=2013&pages=e77913&pmid=24223742&doi=10.1371/journal.pone.0077913&)

48. Smith WR, McClish DK, Dahman BA, et al. Daily home opioid use in adults with sickle cell disease: the PiSCES project. J. Opioid. Manag. 2015;11(3):243–253. doi: 10.5055/jom.2015.0273.  [DOI](https://doi.org/10.5055/jom.2015.0273) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25985809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Opioid.%20Manag.&title=Daily%20home%20opioid%20use%20in%20adults%20with%20sickle%20cell%20disease:%20the%20PiSCES%20project&author=WR%20Smith&author=DK%20McClish&author=BA%20Dahman&volume=11&issue=3&publication_year=2015&pages=243-253&pmid=25985809&doi=10.5055/jom.2015.0273&)

49. Bodnar RJ. Endogenous opiates and behavior: 2007. Peptides. 2008;29(12):2292–2375. doi: 10.1016/j.peptides.2008.09.007.  [DOI](https://doi.org/10.1016/j.peptides.2008.09.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18851999/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Peptides&title=Endogenous%20opiates%20and%20behavior:%202007&author=RJ%20Bodnar&volume=29&issue=12&publication_year=2008&pages=2292-2375&pmid=18851999&doi=10.1016/j.peptides.2008.09.007&)
