# Enantiospecific Pharmacogenomics of Fluvastatin

## Metadata
**Authors:** Päivi Hirvensalo, Aleksi Tornio, Mikko Neuvonen, Wilma Kiander, Heidi Kidron, Maria Paile‐Hyvärinen, Tuija Tapaninen, Janne T Backman, Mikko Niemi
**Journal:** Clinical Pharmacology and Therapeutics
**Date:** 2019 May 14
**DOI:** [10.1002/cpt.1463](https://doi.org/10.1002/cpt.1463)
**PMID:** 30989645
**PMCID:** PMC6767327
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6767327/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6767327/pdf/CPT-106-668.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6767327/pdf/CPT-106-668.pdf)

## Abstract

The aim of this study was to investigate how variability in multiple genes related to pharmacokinetics affects fluvastatin exposure. We determined fluvastatin enantiomer pharmacokinetics and sequenced 379 pharmacokinetic genes in 200 healthy volunteers. CYP2C9*3 associated with significantly increased area under the plasma concentration‐time curve (AUC) of both 3R,5S‐fluvastatin and 3S,5R‐fluvastatin (by 67% and 94% per variant allele copy, P = 3.77 × 10−9 and P = 3.19 × 10−12). In contrast, SLCO1B1 c.521T>C associated with increased AUC of active 3R,5S‐fluvastatin only (by 34% per variant allele copy; P = 8.15 × 10−8). A candidate gene analysis suggested that CYP2C9*2 also affects the AUC of both fluvastatin enantiomers and that SLCO2B1 single‐nucleotide variations may affect the AUC of 3S,5R‐fluvastatin. Thus, SLCO transporters have enantiospecific effects on fluvastatin pharmacokinetics in humans. Genotyping of both CYP2C9 and SLCO1B1 may be useful in predicting fluvastatin efficacy and myotoxicity.

☑High interindividual variability exists in the pharmacokinetics of fluvastatin. *CYP2C9*3* has previously shown to increase the area under the plasma concentration‐time curve (AUC) of fluvastatin enantiomers, but comprehensive studies evaluating the effects of variability in multiple pharmacokinetic genes on fluvastatin exposure have not been conducted.

☑This study investigated how genetic variants in pharmacokinetic genes affect the exposure to fluvastatin enantiomers.

☑This study shows that *CYP2C9*3* and *CYP2C9*2* affect the pharmacokinetics of both fluvastatin enantiomers. *SLCO1B1* c.521T>C has an enantiospecific effect on active 3R,5S‐fluvastatin AUC. Furthermore, the results suggest that *SLCO2B1* missense variants may affect 3S,5R‐fluvastatin exposure. Based on the results, genotype scores were generated to predict how combinations of *CYP2C9* and transporter variants affect fluvastatin enantiomer plasma exposures.

☑This knowledge might aid in predicting the risk of fluvastatin adverse reactions and thus in individualizing treatment with statins.

Fluvastatin is a 3‐hydroxy‐3‐methylglutaryl‐coenzyme A reductase inhibitor, which is used in the treatment of hypercholesterolemia. It is a racemic mixture of two enantiomers, of which 3R,5S‐fluvastatin is 30 times more active than 3S,5R‐fluvastatin.[1](#cpt1463-bib-0001) Fluvastatin is extensively metabolized via cytochrome P450 (CYP) 2C9.[2](#cpt1463-bib-0002), [3](#cpt1463-bib-0003) *In vitro* also CYP3A4, CYP2C8, and possibly CYP2D6 and CYP1A1 contribute to fluvastatin metabolism.[2](#cpt1463-bib-0002) In addition, fluvastatin is a substrate of several drug transporters, including organic anion transporting polypeptides (OATP) 1B1, 1B3, 2B1, breast cancer resistance protein, multidrug resistance‐associated protein 2, and sodium‐dependent taurocholate cotransporting polypeptide.[4](#cpt1463-bib-0004), [5](#cpt1463-bib-0005), [6](#cpt1463-bib-0006), [7](#cpt1463-bib-0007), [8](#cpt1463-bib-0008)

High interindividual variability exists in the pharmacokinetics of fluvastatin. The decreased‐function *CYP2C9*3* (c.1075A>C, p.Ile359Leu, rs1057910) allele[9](#cpt1463-bib-0009), [10](#cpt1463-bib-0010) has been associated with markedly increased plasma concentrations of both fluvastatin enantiomers.[11](#cpt1463-bib-0011) Furthermore, the decreased‐function *ABCG2* c.421C>A (p.Gln141Lys, rs2231142) variant[12](#cpt1463-bib-0012), [13](#cpt1463-bib-0013), [14](#cpt1463-bib-0014), [15](#cpt1463-bib-0015) has been associated with markedly increased plasma concentrations of racemic fluvastatin.[14](#cpt1463-bib-0014) On the other hand, the decreased‐function *SLCO1B1* c.521T>C (p.Val174Ala, rs4149056) variant[16](#cpt1463-bib-0016), [17](#cpt1463-bib-0017), [18](#cpt1463-bib-0018), [19](#cpt1463-bib-0019), [20](#cpt1463-bib-0020), [21](#cpt1463-bib-0021), [22](#cpt1463-bib-0022), [23](#cpt1463-bib-0023) and the *ABCB1* c.1236T‐c.2677T‐c.3435T haplotype have not affected fluvastatin pharmacokinetics.[21](#cpt1463-bib-0021), [24](#cpt1463-bib-0024) Comprehensive studies evaluating the effects of variants in multiple genes on fluvastatin pharmacokinetics have not been conducted previously. Therefore, the aim of this study was to investigate how variability in pharmacokinetic genes affects fluvastatin exposure. We determined the pharmacokinetics of 3R,5S‐fluvastatin and 3S,5R‐fluvastatin after a 40 mg oral dose of racemic fluvastatin in 200 healthy volunteers and fully sequenced 379 pharmacokinetic genes using massive parallel sequencing.

## Results

### Fluvastatin pharmacogenomics

Among the 200 healthy volunteers, the areas under the plasma concentration‐time curve from 0 hour to infinity (AUC_0–∞_) of 3R,5S‐fluvastatin and 3S,5R‐fluvastatin varied 16‐fold and 19‐fold, respectively (Table [S1](#cpt1463-sup-0001)). A single‐nucleotide variation (SNV), rs77760615, located upstream of *CYP2C9* showed the strongest association with the pharmacokinetics of both fluvastatin enantiomers (**Table ** [1](#cpt1463-tbl-0001), **Figure ** [1](#cpt1463-fig-0001)). The AUC_0–∞_ of 3R,5S‐fluvastatin was 70% (*P *=* *2.16 × 10^−10^) and that of 3S,5R‐fluvastatin was 89% (*P *=* *2.93 × 10^−11^) larger per copy of the variant allele. After adjusting for this variant, an intronic *SLCO1B1* variant, rs58310495, associated with a 34% larger AUC_0–∞_ of 3R,5S‐fluvastatin per copy of the variant allele (*P *=* *3.07 × 10^−10^). No other variants remained statistically significantly associated with the AUC_0–∞_ of 3S,5R‐fluvastatin. The investigated genetic variants had no significant effect on the peak plasma concentrations (C_max_) or the elimination half‐lives (*t* _½_) of fluvastatin enantiomers.

### Table 1.

| Pharmacokinetic variable | dbSNP ID | Gene | Location | Nucleotide change | MAF | Effecta |  | P value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Average (%) | 90% CI |  |  |  |  |  |  |  |
| 3R,5S‐fluvastatin |  |  |  |  |  |  |  |  |
| AUC0–∞ | 1. rs77760615 | CYP2C9 | Upstream | c.‐5813A>G | 0.072 | 69.7 | 49.0, 93.2 | 2.16 × 10−10 |
| 2. rs58310495 | SLCO1B1 | Intron 10/14 | c.1332‐1091C>T | 0.29 | 34.4 | 24.9, 44.6 | 3.07 × 10−10 |  |
| Cmax | — | — | — | — | — | — | — | — |
| t 1/2 | — | — | — | — | — | — | — | — |
| 3S,5R‐fluvastatin |  |  |  |  |  |  |  |  |
| AUC0–∞ | rs77760615 | CYP2C9 | Upstream | c.‐5813A>G | 0.072 | 88.9 | 62.8, 119.1 | 2.93 × 10−11 |
| Cmax | — | — | — | — | — | — | — | — |
| t 1/2 | — | — | — | — | — | — | — | — |
| Total fluvastatin AUC0–∞ | rs77760615 | CYP2C9 | Upstream | c.‐5813A>G | 0.072 | 81.6 | 57.4, 109.6 | 8.50 × 10−11 |
| 3R,5S/3S,5R AUC0–∞ ratio | 1. rs4149056 | SLCO1B1 | Exon 6/15 | c.521T>C | 0.22 | 23.3 | 19.6, 27.2 | 1.30 × 10−22 |
| 2. rs12367888 | SLCO1B1 | Intron 7/14 | c.728‐2859G>A | 0.16 | −11.6 | −14.7, −8.3 | 5.24 × 10−8 |  |

Table 1 Caption: Results of the stepwise forward linear regression analysis of the effects of 46,064 SNVs in 379 genes on fluvastatin pharmacokinetics

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/79a6/6767327/af0c4a69f5cb/CPT-106-668-g001.jpg)

The associations of 46,064 SNVs in 379 pharmacokinetic genes with 3R,5S‐fluvastatin (a) and 3S,5R‐fluvastatin (c) AUC 0–∞, adjusting for BSA (left panel), and BSA and the CYP2C9 rs77760615 SNV (right panel). The y‐axes in (a) and (c) describe the negative logarithm of the P value for each SNV, and the horizontal lines indicate the Bonferroni‐corrected significance level of 1.09 × 10−6. The x‐axes show individual SNVs grouped by protein function. The geometric mean ± geometric standard deviation BSA‐adjusted AUC 0–∞ values grouped by the top associated SNVs are illustrated in (b) and (d). AUC0–∞, area under the plasma concentration‐time curve from 0 hour to infinity; BSA, body surface area; SNV, single‐nucleotide variation. [Colour figure can be viewed at wileyonlinelibrary.com]

In agreement with these results, also the total 3R,5S+3S,5R‐fluvastatin AUC_0–∞_ associated significantly with the *CYP2C9* upstream variant rs77760615 (**Table ** [1](#cpt1463-tbl-0001)). The AUC_0–∞_ was 82% larger per copy of the variant allele (*P *=* *8.50 × 10^−11^). After adjusting for this variant, no other variants remained statistically significantly associated with the total fluvastatin AUC_0–∞_. The investigated variants had no significant effect on the C_max_ or *t* _½_ of total fluvastatin.

To further identify the genetic factors that differently affect the two enantiomers, we investigated the associations of the genetic variants with 3R,5S/3S,5R‐fluvastatin AUC_0–∞_ ratio. The AUC_0–∞_ ratio was significantly associated with the *SLCO1B1* missense SNV c.521T>C (p.V174A, rs4149056). The AUC_0–∞_ ratio was increased by 23% per copy of the variant allele (*P *=* *1.30 × 10^−22^). After adjusting for this variant, an intronic triallelic *SLCO1B1* SNV rs12367888 remained significantly associated with the AUC_0–∞_ ratio. The ratio was 12% lower per copy of the rs12367888 A‐allele (*P *=* *5.24 × 10^−8^).

### Linkage disequilibrium and haplotype analysis

In order to identify the causative SNVs underlying the associations of *CYP2C9* rs77760615 and *SLCO1B1* rs58310495 and rs12367888, we next investigated the linkage disequilibrium of these variants with missense SNVs in the respective genes. The rs77760615 SNV was in a complete linkage disequilibrium (*r* ^2^ = 1, *P *=* *7.04 × 10^−44^) with the c.1075A>C (p.Ile359Leu, rs1057910) missense SNV, defining the *CYP2C9*3* allele (**Figure ** [2](#cpt1463-fig-0002) **a**). The *SLCO1B1* rs58310495 SNV was strongly linked with the *SLCO1B1* missense variants c.521T>C (*r* ^2^ = 0.69, *P *=* *5.78 × 10^−32^) and c.388A>G (p.Asn130Asp, rs2306283; *r* ^2^ = 0.41, *P *=* *1.58 × 10^−19^) (**Figure ** [2](#cpt1463-fig-0002) **b**). Moreover, the A allele of the triallelic rs12367888 SNV was relatively strongly linked with the missense variants c.463C>A (Pro155Thr, rs11045819; *r* ^2^ = 0.48, *P *=* *2.95 × 10^−22^), c.1929A>C (p.Leu643Phe, rs34671512; *r* ^2^ = 0.28, *P *=* *1.19 × 10^−13^), and c.388A>G (*r* ^2^ = 0.25, *P *=* *3.42 × 10^−12^).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/79a6/6767327/2b15cc453b04/CPT-106-668-g002.jpg)

Linkage disequilibrium of (a) CYP2C9 and (b) SLCO1B1 missense and top noncoding SNVs. (c) SLCO1B1 haplotypes (MAF ≥0.01) inferred with missense and top noncoding SLCO1B1 SNVs. Intronic nucleotide changes are depicted in yellow and blue, and missense variations in red. MAF, minor allele frequency; SNV, single‐nucleotide variation. [Colour figure can be viewed at wileyonlinelibrary.com]

Previous studies have suggested that the functional effects of *SLCO1B1* SNVs may depend on the combinations of SNVs in the same haplotype.[17](#cpt1463-bib-0017), [18](#cpt1463-bib-0018), [19](#cpt1463-bib-0019), [20](#cpt1463-bib-0020), [21](#cpt1463-bib-0021), [22](#cpt1463-bib-0022), [23](#cpt1463-bib-0023) We therefore computed *SLCO1B1* haplotypes using missense variations and the noncoding rs58310495 and rs12367888 SNVs (**Figure ** [2](#cpt1463-fig-0002) **c**). The rs58310495 variant was found to be present in all *SLCO1B1*5* and **15* haplotypes and in 63% of the **1B* haplotypes, but not in **14* or **35*. The rs12367888 A allele was present in all *SLCO1B1*14* and **35* haplotypes, and in 24% of the **1B* haplotypes.

### Candidate gene analysis

To mitigate the risk of false negatives due to the conservative Bonferroni correction used in the primary analysis, we next carried out a candidate gene analysis without correction for multiple testing, including missense and functional variants with minor allele frequencies of ≥ 0.01 in genes involved in fluvastatin pharmacokinetics (Table [S2](#cpt1463-sup-0002)). The candidate gene analysis focused on AUC_0–∞_, because it describes total exposure and is likely the best surrogate for drug response. In this analysis, *CYP2C9*3* was associated with a 67% (*P *=* *3.77 × 10^−9^), *SLCO1B1* c.521T>C with a 34% (*P *=* *8.15 × 10^−8^), and *CYP2C9*2* (rs1799853, c.430C>T, p.Arg144Cys) with a 23% (*P *=* *0.00213) increased AUC_0–∞_ of 3R,5S‐fluvastatin per copy of each variant allele (**Table ** [2](#cpt1463-tbl-0002)). The AUC_0–∞_ of 3S,5R‐fluvastatin was 94% (*P *=* *3.19 × 10^−12^), 29% (*P *=* *3.94 × 10^−4^), and 45% (*P *=* *0.0246) larger per copy of *CYP2C9*3*,* CYP2C9*2*, and *SLCO2B1* c.601G>A (p.Val201Met, rs35199625) variant allele, respectively, and 22% (*P *=* *0.0306) smaller per copy of the *SLCO2B1* c.1457C>T (p.Ser486Phe, rs2306168) variant allele. When the candidate gene analysis was performed with *SLCO1B1* haplotypes instead of SNVs, the *SLCO1B1*15* haplotype was associated with 3R,5S‐fluvastatin AUC_0–∞_ (Table [S3](#cpt1463-sup-0003)). None of the *SLCO1B1* haplotypes associated with 3S,5R‐fluvastatin AUC_0–∞_.

### Table 2.

| Pharmacokinetic variable | Covariate/SNV | Effecta |  | P value | Bonferroni‐adjusted P value | Adjusted R 2 for each step |
| --- | --- | --- | --- | --- | --- | --- |
| Average (%) | 90% CI |  |  |  |  |  |
| 3R,5S‐fluvastatin AUC0–∞ | BSA | −19.9 | −23.4, −16.2 | 3.10 × 10−14 | — | 0.24 |
| CYP2C9*3 (rs1057910) | 66.7 | 45.4, 91.0 | 3.77 × 10−9 | 1.01 × 10−7 | 0.36 |  |
| SLCO1B1 c.521T>C (rs4149056) | 34.2 | 23.0, 46.4 | 8.15 × 10−8 | 2.20 × 10−6 | 0.44 |  |
| CYP2C9*2 (rs1799853) | 22.7 | 10.1, 36.7 | 0.00213 | 0.0575 | 0.46 |  |
| 3S,5R‐fluvastatin AUC0–∞ | BSA | −20.7 | −24.4, −16.8 | 1.06 × 10−13 | — | 0.20 |
| CYP2C9*3 (rs1057910) | 93.6 | 67.2, 124.2 | 3.19 × 10−12 | 8.61 × 10−11 | 0.35 |  |
| CYP2C9*2 (rs1799853) | 29.1 | 14.9, 45.1 | 3.94 × 10−4 | 0.0106 | 0.38 |  |
| SLCO2B1 c.601G>A (rs35199625) | 45.2 | 10.6, 90.7 | 0.0246 | 0.664 | 0.39 |  |
| SLCO2B1 c.1457C>T (rs2306168) | −21.7 | −35.0, −5.7 | 0.0306 | 0.825 | 0.40 |  |
| Total fluvastatin AUC0–∞ | BSA | −19.7 | −23.3, −16.0 | 1.75 × 10−13 | — | 0.23 |
| CYP2C9*3 (rs1057910) | 75.4 | 52.4, 101.8 | 3.17 × 10−10 | 1.00 × 10−8 | 0.37 |  |
| SLCO1B1 c.521T>C (rs4149056) | 20.4 | 10.1, 31.7 | 7.54 × 10−4 | 0.0203 | 0.40 |  |
| CYP2C9*2 (rs1799853) | 25.4 | 12.2, 40.2 | 9.63 × 10−4 | 0.0260 | 0.43 |  |
| 3R,5S/3S,5R AUC0–∞ ratio | SLCO1B1 c.521T>C (rs4149056) | 28.2 | 23.7, 32.8 | 4.37 × 10−24 | 1.18 × 10−22 | 0.41 |
| SLCO1B1 c.1929A>C (rs34671512) | −15.6 | −20.0, −11.0 | 4.52 × 10−7 | 1.22 × 10−5 | 0.48 |  |
| SLCO1B1 c.463C>A (rs11045819) | −10.4 | −14.1, −6.6 | 2.62 × 10−5 | 7.08 × 10−4 | 0.53 |  |
| CYP2C9*3 (rs1057910) | −10.7 | −14.7, −6.6 | 4.41 × 10−5 | 0.00119 | 0.55 |  |
| SLCO2B1 c.1457C>T (rs2306168) | 8.8 | 2.9, 15.1 | 0.0134 | 0.361 | 0.56 |  |
| SLCO2B1 c.935G>A (rs12422149) | −5.1 | −8.5, −1.5 | 0.0221 | 0.597 | 0.57 |  |
| SLCO1B3 c.699G>A (rs7311358)b | −4.0 | −7.2, −0.8 | 0.0434 | 1.173 | 0.58 |  |

Table 2 Caption: Results of the candidate gene analysis on 3R,5S‐fluvastatin and 3S,5R‐fluvastatin AUC0–∞, total fluvastatin AUC0–∞, and 3R,5S/3S,5R‐fluvastatin AUC0–∞ ratio

The total fluvastatin AUC_0–∞_ was 75% (*P *=* *3.17 × 10^−10^), 20% (*P *=* *7.54 × 10^−4^), and 25% (*P *=* *9.63 × 10^−4^) larger per copy of the *CYP2C9*3*,* SLCO1B1* c.521T>C, and *CYP2C9*2* variant allele, respectively (**Table ** [2](#cpt1463-tbl-0002)). The 3R,5S/3S,5R‐fluvastatin AUC_0–∞_ ratio showed significant association with variants in the *SLCO1B1*,* CYP2C9*,* SLCO2B1*, and *SLCO1B3* genes. Of the *SLCO1B1* haplotypes, *SLCO1B1*15* was associated with increased total fluvastatin AUC_0–∞_; *SLCO1B1*5* and **15* were associated with increased, and the **14* and **35* haplotypes with decreased, 3R,5S/3S,5R‐fluvastatin AUC_0–∞_ ratio (Table [S3](#cpt1463-sup-0003)).

To predict the relative effects of combinations of *CYP2C9* and *SLCO1B1* or *SLCO2B1* genotypes on fluvastatin and fluvastatin enantiomer exposures, we calculated genotype scores (GS) using the following equations based on the candidate gene linear regression models:

| GS3R,5S−fluvastatin=1.67n(CYP2C9∗3)×1.34n(SLCO1B1c.521C)×1.23n(CYP2C9∗2) |
| --- |
| GS3S,5R−fluvastatin=1.94n(CYP2C9∗3)×1.29n(CYP2C9∗2)×1.45n(SLCO2B1c.601A)×0.78n(SLCO2B1c.1457T) |
| --- |
| GStotalfluvastatin=1.75n(CYP2C9∗3)×1.20n(SLCO1B1c.521C)×1.25n(CYP2C9∗2) |
| --- |
where *n* is the number of variant alleles (0, 1, or 2).

The genotype scores predict the fold differences in AUCs between carriers of different genotype combinations and non‐carriers (**Table ** [3](#cpt1463-tbl-0003), **Figure ** [3](#cpt1463-fig-0003)). A total of 12%, 2.0%, and 9.5% of the study population had GS predicting more than twofold increased AUCs of 3R,5S‐fluvastatin, 3S,5R‐fluvastatin, and total fluvastatin, respectively.

### Table 3.

|   | Cmax (ng/mL) | T max (hour) | AUC0–∞ (ng/hour/mL) | t 1/2 (hour) |
| --- | --- | --- | --- | --- |
| 3R,5S‐fluvastatin |  |  |  |  |
| Genotype score ≥0.80, ≤1.25b |  |  |  |  |
| Geometric mean (CV%) | 124 (72%) | 1 (0.5–5) | 172 (46%) | 1.9 (33%) |
| Genotype score > 1.25, < 2.00 |  |  |  |  |
| Geometric mean (CV%) | 183 (65%) | 0.75 (0.5–2) | 249 (37%) | 2.0 (23%) |
| Ratio to control (90%CI) | 1.48 (1.26, 1.73) | — | 1.45 (1.30, 1.61) | 1.05 (0.98, 1.13) |
| P value | 5.37 × 10−5 | 0.547 | 3.02 × 10−8 | 0.241 |
| Genotype score ≥ 2.00 |  |  |  |  |
| Geometric mean (CV%) | 235 (59%) | 1 (0.5–2) | 386 (49%) | 2.1 (18%) |
| Ratio to control (90% CI) | 1.90 (1.50, 2.40) | — | 2.24 (1.91, 2.63) | 1.10 (0.99, 1.22) |
| P value | 1.03 × 10−5 | 0.400 | 7.53 × 10−15 | 0.141 |
| 3S,5R‐fluvastatin |  |  |  |  |
| Genotype score ≥0.80, ≤1.25b |  |  |  |  |
| Geometric mean (CV%) | 145 (78%) | 0.5 (0.5–5) | 175 (45%) | 2.5 (29%) |
| Genotype score < 0.80 |  |  |  |  |
| Geometric mean (CV%) | 94 (93%) | 1 (0.5–2) | 125 (48%) | 2.3 (18%) |
| Ratio to control (90% CI) | 0.65 (0.44, 0.97) | — | 0.72 (0.55, 0.93) | 0.90 (0.78, 1.04) |
| P value | 0.0740 | 0.088 | 0.0358 | 0.246 |
| Genotype score > 1.25, < 2.00 |  |  |  |  |
| Geometric mean (CV%) | 187 (78%) | 1 (0.5–2) | 250 (54%) | 2.5 (29%) |
| Ratio to control (90% CI) | 1.30 (1.09, 1.54) | — | 1.43 (1.27, 1.60) | 1.01 (0.95, 1.08) |
| P value | 0.0149 | 0.009 | 9.72 × 10−7 | 0.807 |
| Genotype score ≥ 2.00 |  |  |  |  |
| Geometric mean (CV%) | 299 (23%) | 1 (0.5–1) | 573 (28%) | 2.4 (20%) |
| Ratio to control (90% CI) | 2.06 (1.16, 3.68) | — | 3.27 (2.23, 4.81) | 0.96 (0.78, 1.19) |
| P value | 0.0396 | 0.349 | 8.65 × 10−7 | 0.762 |
| Total fluvastatin |  |  |  |  |
| Genotype score ≥0.80, ≤1.25b |  |  |  |  |
| Geometric mean (CV%) | 286 (75%) | 1 (0.5–5) | 369 (46%) | 2.1 (32%) |
| Genotype score > 1.25, < 2.00 |  |  |  |  |
| Geometric mean (CV%) | 378 (79%) | 1 (0.5–2) | 543 (47%) | 2.1 (20%) |
| Ratio to control (90% CI) | 1.32 (1.05, 1.67) | — | 1.47 (1.26, 1.71) | 1.01 (0.91, 1.12) |
| P value | 0.0442 | 0.319 | 4.88 × 10−5 | 0.832 |
| Genotype score ≥ 2.00 |  |  |  |  |
| Geometric mean (CV%) | 481 (50%) | 1 (0.5–1.5) | 763 (47%) | 2.2 (21%) |
| Ratio to control (90% CI) | 1.68 (1.29, 2.19) | — | 2.07 (1.74, 2.47) | 1.08 (0.96, 1.22) |
| P value | 1.26 × 10−3 | 0.368 | 1.05 × 10−10 | 0.263 |

Table 3 Caption: Pharmacokinetic variables of 3R,5S‐fluvastatin, 3S,5R‐fluvastatin, and total fluvastatin grouped by genotype scoresa

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/79a6/6767327/9276517cabe6/CPT-106-668-g003.jpg)

Geometric mean (90% CI) BSA‐adjusted plasma concentrations of (a) 3R,5S‐fluvastatin and (c) 3S,5R‐ fluvastatin after a single 40 mg oral dose of racemic fluvastatin in 200 healthy volunteers with different combinations of CYP2C9 and SLCO1B1 or SLCO2B1 genotypes. The insets depict the same data on a semilogarithmic scale. The volunteers were grouped by genotype scores predicting the fold differences in AUC 0–∞ values between carriers of different genotype combinations and non‐carriers. The right panels in (a) and (c) show the genotype scores for individuals with different genotypes. Reference genotypes are depicted with white, heterozygous with gray, and homozygous variant genotypes with black rectangles. The geometric mean ± geometric standard deviation BSA‐adjusted AUC 0–∞ values grouped by combinations of CYP2C9 and SLCO1B1 or SLCO2B1 genotypes, as well as the genotype scores are illustrated in (b) and (d). AUC0–∞, area under the plasma concentration‐time curve from 0 hour to infinity; BSA, body surface area; CI, confidence interval.

### Fluvastatin transport by OATP1B1 and metabolism in vitro

To mechanistically validate the findings, we investigated the transport of fluvastatin enantiomers in OATP1B1‐transfected human embryonic kidney 293 cells, expressing either the reference or the c.521T>C variant *SLCO1B1*. Both 3R,5S‐fluvastatin and 3S,5R‐fluvastatin were substrates of reference OATP1B1, as indicated by 7.3‐fold and 4.1‐fold higher cellular uptake at 0.5 μM, and 2.5‐fold and 2.5‐fold higher at 4 μM in OATP1B1‐transfected cells than in mock‐transfected cells (*P *<* *0.001). The c.521T>C SNV reduced the uptake of both enantiomers to 7% of the control (*P *<* *10^−5^) (**Figure ** [4](#cpt1463-fig-0004)).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/79a6/6767327/bc2d63ce212e/CPT-106-668-g004.jpg)

Fluvastatin enantiomer transport by reference OATP1B1 (a), metabolism in HLM (b), and metabolism by reference CYP2C9.1 (c); effects of SLCO1B1 c.521T>C on their transport (d), and sulfaphenazole (e) and CYP2C9.2 and CYP2C9.3 (f) on their metabolism in vitro. Fluvastatin enantiomer concentrations were 0.1–16 μM (a), 0.1 μM (b, c, e, f), and 0.5 μM (d). Sulfaphenazole concentration was 10 μM (e). CLint, intrinsic clearance; HLM, human liver microsomes; Km, Michaelis–Menten constant; Vmax, maximum transport rate.

We also investigated the metabolism of fluvastatin enantiomers by human liver microsomes (HLM), and recombinant CYP2C9.1, CYP2C9.2, and CYP2C9.3 enzymes. The microsomal intrinsic clearance (CL_int,HLM_) of 3R,5S‐fluvastatin was 39 μL/minute/mg protein and that of 3S,5R‐fluvastatin was 33 μL/minute/mg protein (*P *=* *1.32 × 10^−4^, **Figure ** [4](#cpt1463-fig-0004)). The CYP2C9 inhibitor sulfaphenazole[25](#cpt1463-bib-0025) inhibited the CL_int,HLM_ of 3R,5S‐fluvastatin by 51% (*P *=* *2.14 × 10^−9^) and that of 3S,5R‐fluvastatin by 69% (*P *=* *1.60 × 10^−7^). Fluvastatin enantiomers were similarly metabolized by CYP2C9.1 (*P *=* *0.455). The CL_int_ of 3R,5S‐fluvastatin was reduced by 38% (*P *=* *0.0132) and 65% (*P *=* *0.00273), and that of 3S,5R‐fluvastatin by 25% (*P *=* *0.00620) and 84% (*P *=* *4.95 × 10^−4^) by recombinant CYP2C9.2 and CYP2C9.3, respectively, as compared with CYP2C9.1.

## Discussion

These results show that genetic variation in *CYP2C9* strongly affects the pharmacokinetics of both fluvastatin enantiomers. *SLCO1B1* variants, on the other hand, affect the pharmacokinetics of 3R,5S‐fluvastatin only. Moreover, the candidate gene approach suggests that *SLCO2B1* variants may affect the plasma concentrations of 3S,5R‐fluvastatin. Taken together, although the activity of CYP2C9 strongly affects the exposures to both fluvastatin enantiomers, the effects of *SLCO* transporters are enantiospecific.

The *CYP2C9*3* and *CYP2C9*2* alleles are known to impair the function of CYP2C9.[9](#cpt1463-bib-0009), [10](#cpt1463-bib-0010), [26](#cpt1463-bib-0026) The finding that *CYP2C9*3* had a markedly larger effect than *CYP2C9*2 in vivo* is in line with our *in vitro* results showing a larger effect of *CYP2C9*3* than *CYP2C9*2* on fluvastatin metabolism. In a previous study, *CYP2C9*3* was associated with increased AUC of both fluvastatin enantiomers, whereas the effect of *CYP2C9*2* was not significant.[11](#cpt1463-bib-0011) However, the number of participants in that study was relatively small. Nevertheless, our results indicate that both *CYP2C9*3* and **2* affect the pharmacokinetics of both fluvastatin enantiomers.

The finding that *CYP2C9*3* had a larger effect on the AUC of 3S,5R‐fluvastatin than 3R,5S‐fluvastatin could be explained by different contributions of CYP2C9 to the elimination of the enantiomers, enantiomer‐dependent effect of *CYP2C9*3*, or both. Our *in vitro* results support both of these hypotheses, as the CYP2C9 inhibitor sulfaphenazole and recombinant CYP2C9.3 reduced the metabolism of 3S,5R‐fluvastatin more than that of 3R,5S‐fluvastatin.

In addition to *CYP2C9* variants, an intronic *SLCO1B1* rs58310495 SNV was associated with significantly increased AUC of 3R,5S‐fluvastatin. This SNV is strongly linked with the decreased‐function *SLCO1B1* c.521T>C variant,[16](#cpt1463-bib-0016), [17](#cpt1463-bib-0017), [18](#cpt1463-bib-0018), [19](#cpt1463-bib-0019), [20](#cpt1463-bib-0020), [21](#cpt1463-bib-0021), [22](#cpt1463-bib-0022), [23](#cpt1463-bib-0023) which suggests that the association is due to c.521T>C. Interestingly, no associations with *SLCO1B1* variants were present for 3S,5R‐fluvastatin even in the candidate gene analysis with no correction for multiple testing (effect size 6.5% per copy of the c.521C allele, *P *=* *0.259). Accordingly, the c.521T>C variant was associated with significantly increased 3R,5S‐/3S,5R‐fluvastatin AUC ratio. Thus, our results demonstrate that the effect of the *SLCO1B1* genotype on fluvastatin pharmacokinetics is enantiospecific.

In addition to *SLCO1B1* c.521T>C, the A allele of a triallelic intronic *SLCO1B1* SNV (rs12367888) was associated with significantly decreased 3R,5S/3S,5R‐fluvastatin AUC ratio. Similarly, in the candidate gene analysis, the missense c.463C>A and c.1929A>C SNVs, which are strongly linked with the rs12367888 A allele, significantly decreased the AUC ratio. As the *SLCO1B1* genotype has an enantiospecific effect on 3R,5S‐fluvastatin, the decreased AUC ratio is likely due to increased OATP1B1 activity. In line with this, the *SLCO1B1*14* (containing c.463C>A) and **35* (containing c.1929A>C) haplotypes have previously been associated with increased hepatic expression of OATP1B1[27](#cpt1463-bib-0027) and increased clearance of the OATP1B1 substrate methotrexate.[28](#cpt1463-bib-0028) *SLCO1B1*14* has also been associated with a decreased AUC of atorvastatin[27](#cpt1463-bib-0027) and c.463C>A with decreased plasma concentrations of rifampin.[29](#cpt1463-bib-0029)

Although *SLCO1B1* c.521T>C markedly raises the plasma concentrations of many statins[17](#cpt1463-bib-0017), [18](#cpt1463-bib-0018), [19](#cpt1463-bib-0019), [20](#cpt1463-bib-0020), [21](#cpt1463-bib-0021), [22](#cpt1463-bib-0022), [23](#cpt1463-bib-0023) the AUC of total fluvastatin was only nonsignificantly (by 19%) increased in c.521CC homozygotes in a previous study.[21](#cpt1463-bib-0021) In our candidate gene analysis, the effect of c.521T>C on total fluvastatin was significant and larger than in the previous study. This difference may be explained by the substantially larger sample size in the present study and that the present study also included carriers of *SLCO1B1* haplotypes that enhance OATP1B1 activity, e.g., *SLCO1B1*14* and **35*. In another previous study, individuals homozygous for the *ABCG2* missense variant c.421C>A (rs2231142, p.Gln141Lys) had significantly increased plasma concentrations of racemic fluvastatin.[14](#cpt1463-bib-0014) In the present study, the variant showed no significant association with fluvastatin pharmacokinetics. However, only one participant was homozygous for c.421C>A.

To elucidate the mechanism underlying the enantiospecific effects of *SLCO1B1* genotype, we investigated the transport of fluvastatin enantiomers by OATP1B1 *in vitro*. The enantiomers had less than twofold difference in the *in vitro* uptake clearance and the c.521T>C variant similarly impaired their uptake. These preliminary *in vitro* results cannot explain the enantiospecific effects of *SLCO1B1* genotype on fluvastatin pharmacokinetics in humans. Therefore, it seems that different contributions of other transporters to the hepatic disposition of the enantiomers is the most likely explanation for the enantiospecific effects of the *SLCO1B1* genotype. In addition to OATP1B1, racemic fluvastatin is a substrate of OATP1B3, OATP2B1, and sodium‐dependent taurocholate cotransporting polypeptide, with the highest affinity reported for OATP2B1.[4](#cpt1463-bib-0004) Our candidate gene analysis suggested that *SLCO2B1* missense variants affect 3S,5R‐fluvastatin but not 3R,5S‐fluvastatin AUC. The c.1457C>T SNV, which associated with decreased 3S,5R‐fluvastatin AUC, has previously been associated with decreased fexofenadine and celiprolol AUC.[30](#cpt1463-bib-0030), [31](#cpt1463-bib-0031) In addition to *SLCO2B1* SNVs, a missense SNV in *SLCO1B3* was associated with a minor effect on the 3R,5S/3S,5R‐fluvastatin AUC ratio. It is noteworthy that the *SLCO2B1* and *SLCO1B3* associations were significant only when no correction for multiple testing was applied, and therefore the results should be interpreted with caution.

Although statins are generally well tolerated, they can cause muscle toxicity. The clinical spectrum of statin‐induced myotoxicity ranges from mild myopathy to potentially life‐threatening rhabdomyolysis.[32](#cpt1463-bib-0032) There seem to be no published studies on how the risk of myotoxicity is related to fluvastatin exposure. Because statin‐induced myotoxicity is, however, generally dose‐dependent and concentration‐dependent,[32](#cpt1463-bib-0032), [33](#cpt1463-bib-0033) it is likely that increased systemic exposure to the active 3R,5S‐fluvastatin predisposes to this adverse reaction. Both *CYP2C9*3* and **2*, as well as *SLCO1B1* c.521T>C increase 3R,5S‐fluvastatin AUC. Of these, the *CYP2C9*3* allele has previously been associated with fluvastatin‐induced adverse reactions.[34](#cpt1463-bib-0034) Moreover, the *SLCO1B1* c.521T>C SNV increases simvastatin acid AUC more than threefold and associates with a markedly increased risk of simvastatin‐induced myotoxicity.[20](#cpt1463-bib-0020), [35](#cpt1463-bib-0035) As the effect of the c.521T>C SNV on 3R,5S‐fluvastatin AUC is smaller than on simvastatin acid AUC, the increase in myotoxicity risk is also likely smaller. Markedly larger increases in 3R,5S‐fluvastatin AUC are seen in individuals carrying the c.521T>C SNV together with *CYP2C9* variants, however. Based on *CYP2C9* and *SLCO* variants, we calculated GS to predict the exposure to fluvastatin. The GS might be useful in predicting the risk of fluvastatin‐induced myotoxicity. For patients with 3R,5S‐fluvastatin GS ≥2.0, it could be advisable to prescribe a lower dose of fluvastatin or consider an alternative statin.

The site of action of statins is within the hepatocytes, and fluvastatin efficacy is therefore related to the hepatocyte exposure to 3R,5S‐fluvastatin.[1](#cpt1463-bib-0001), [3](#cpt1463-bib-0003) The maximum fluvastatin dose (80 mg/day) reduces Low‐density lipoprotein cholesterol on average by 33%,[36](#cpt1463-bib-0036) which is less than that seen with the maximum doses of rosuvastatin and atorvastatin. As the *CYP2C9*3* and **2* variants impair the hepatic metabolism of fluvastatin, they should increase the hepatocyte concentration of 3R,5S‐fluvastatin to at least the same extent as its plasma concentration and therefore enhance its cholesterol‐lowering efficacy. Low‐density lipoprotein cholesterol decreases by 6% for every twofold increase in fluvastatin dose.[36](#cpt1463-bib-0036) Therefore, individuals with, e.g., the *CYP2C9*2/*3* genotype, associated with twofold increased 3R,5S‐fluvastatin AUC (**Figure ** [3](#cpt1463-fig-0003)), should show at least 6% greater low‐density lipoprotein cholesterol‐lowering efficacy of fluvastatin than individuals with the **1/*1* genotype. In one small study, there was a tendency for greater cholesterol‐lowering effect of fluvastatin in patients with the *CYP2C9*1/*3* genotype than in those with the **1/*1* genotype.[37](#cpt1463-bib-0037) On the other hand, the *SLCO1B1* c.521T>C variant reduces the hepatic uptake of 3R,5S‐fluvastatin and might thus reduce fluvastatin efficacy. In a previous study, a trend of impaired fluvastatin efficacy has been reported in c.521C allele carriers.[38](#cpt1463-bib-0038) Furthermore, the c.463C>A missense SNV, which associates with increased OATP1B1 activity, has been associated with increased cholesterol‐lowering efficacy of fluvastatin.[39](#cpt1463-bib-0039) Taken together, the *CYP2C9*3* and **2* alleles are expected to increase both the efficacy and the risk of myotoxicity of fluvastatin, whereas the *SLCO1B1* c.521T>C allele likely increases myotoxicity risk only.

The present study was conducted in Finnish white volunteers. The frequencies of *CYP2C9* and *SLCO1B1* variants, however, differ between populations. The allele frequencies of *CYP2C9*2*,* *3*, and *SLCO1B1* c.521T>C are 0.12, 0.056, 0.18 in Europeans, 0.024, 0.013, and 0.019 in Sub‐Saharan Africans, and < 0.001, 0.034, and 0.12 in East Asians, respectively.[40](#cpt1463-bib-0040), [41](#cpt1463-bib-0041) Therefore, the extent to which these variants can explain population variability in fluvastatin pharmacokinetics differs between ethnic groups. Nevertheless, *CYP2C9*2*,* *3*, and *SLCO1B1* c.521T>C are decreased‐function alleles, as demonstrated both *in vitro* and *in vivo*, and individuals carrying these should have similarly increased fluvastatin exposures, irrespective of the ethnic background.

In conclusion, genetic variability in *CYP2C9* significantly affects fluvastatin pharmacokinetics. In addition, *SLCO1B1* genotype has an enantiospecific effect on active 3R,5S‐fluvastatin pharmacokinetics. However, no obvious difference between the enantiomers was observed in the uptake by OATP1B1 *in vitro*. The genotype scores may predict how combinations of *CYP2C9* and *SLCO1B1* variants affect 3R,5S‐fluvastatin exposure and aid in predicting the risk of myotoxicity.

## Methods

A total of 200 healthy unrelated Finnish white volunteers participated in the study after giving written informed consent. Their health was confirmed by medical history, clinical examination, and laboratory tests. Participants were not on any continuous medication nor were tobacco smokers. The study was approved by the Coordinating Ethics Committee of the Hospital District of Helsinki and Uusimaa and the Finnish Medicines Agency Fimea. Of the participants, 99 were women and 101 were men. Their mean ± SD age was 23 ± 4 years, height 174 ± 9 cm, weight 70 ± 12 kg, and body mass index 22.8 ± 2.6 kg/m^2^.

### Fluvastatin pharmacokinetics

After fasting overnight, the participants ingested a 40 mg dose of racemic fluvastatin (Lescol capsule, Novartis Finland Oy, Espoo, Finland) with 150 mL of water at 8 am. Standardized meals were served at 4, 7, and 10 hours after fluvastatin ingestion. Timed blood samples (4–9 mL each) were collected to light‐protected ethylenediaminetetraacetic acid tubes prior to and up to 12 hours after fluvastatin administration. Tubes were immediately placed on ice. Plasma was separated within 30 minutes and stored at −70°C until analysis.

The determination of 3R,5S‐fluvastatin and 3S,5R‐fluvastatin in plasma was based on chiral liquid chromatographic‐tandem mass spectrometric (LC‐MS/MS) method employing commercial pure reference compounds (Toronto Research Chemicals, Inc., North York, ON). The analytical system consisted of a Nexera X2 liquid chromatography instrument (Shimadzu, Kyoto, Japan) interfaced with a 5500 Qtrap mass spectrometer (AB Sciex, Toronto, ON, Canada). Chromatographic separation of the enantiomers was achieved on a Lux Cellulose‐1 chiral column (150 × 2.0‐mm internal diameter, 3‐μm particle size; Phenomenex, Torrance, CA). The mobile phase consisted of a mixture of acetonitrile, methanol and water (12:18:20 v/v/v) containing 0.01% formic acid, and the flow rate was 220 μL/minute. The mass spectrometer was operated in a negative electrospray ionization mode, and the targeted mass‐to‐charge ion transition for both fluvastatin enantiomers was 410–348. Deuterium‐labeled fluvastatin was monitored as an internal standard. The lower limits of quantification were 0.25 ng/mL, and the inter day coefficients of variation of the quality control samples were below 7% for both enantiomers. The AUC_0–∞_, C_max_, and *t* _½_ values were calculated for 3R,5S‐fluvastatin, 3S,5R‐fluvastatin, and total fluvastatin with standard noncompartmental methods using Phoenix WinNonlin, version 6.3 (Certara, Princeton, NJ).

### DNA sequencing and genotyping

Genomic DNA was extracted from ethylenediaminetetraacetic acid blood samples using the Maxwell 16 LEV Blood DNA Kit on a Maxwell 16 Research automated nucleic acid extraction system (Promega, Madison, WI). A total of 379 pharmacokinetic genes ± 20 kb were completely sequenced in the study participants using targeted massive parallel sequencing at the Technology Centre at Institute for Molecular Medicine Finland (Helsinki, Finland).[42](#cpt1463-bib-0042), [43](#cpt1463-bib-0043) A total of 46,064 SNVs with minor allele frequency ≥ 0.05 were included in the statistical analysis.

In order to verify genotype calls and to supplement missing data, all study participants were genotyped for the *CYP2C9*3*,* CYP2C9*2*, and *SLCO1B1* c.521T>C, c.388A>G, and c.463C>A SNVs with TaqMan genotyping assays on a QuantStudio 12K Flex Real‐Time PCR System (Thermo Fisher Scientific, Waltham, MA). Call identity with sequencing data was 100%. TaqMan genotyping was also used to supplement missing data of *SLCO1B1* c.1929A>C, *SLCO1B3* c.334T>G (p.Ser112Ala, rs4149117), *SLCO2B1* c.1457C>T, and *SLCO2B1* c.601G>A SNVs. Triallelic rs12367888 genotype calling was verified and missing data supplemented by Sanger sequencing. All participants were genotyped for relevant *CYP2D6* alleles with TaqMan genotyping assays.[44](#cpt1463-bib-0044) The *CYP2D6* copy number was determined with a TaqMan copy number assay targeting exon 9. *CYP2D6* metabolizer status was inferred from the genotypes using the activity score method.[45](#cpt1463-bib-0045)

### In vitro studies with transfected OATP1B1 cells

Recombinant baculoviruses containing reference or c.521T>C variant *SLCO1B1* gene or the gene for enhanced yellow fluorescent protein (eYFP) were prepared using a modified Bac‐to‐Bac method.[46](#cpt1463-bib-0046) Human embryonic kidney 293 cells were seeded on Corning BioCoat Poly‐D‐Lysine 24 well plates (Corning, Woburn, MA) at a density of 250,000 cells per well and transfected with recombinant baculoviruses carrying either reference or variant *SLCO1B1* gene or the eYFP gene for the control cells. Sodium butyrate (5 mM) was used to stimulate the expression of the recombinant proteins.

The cellular uptake assays were performed on a heated orbital shaker plate 48 hours post‐transfection. Cell culture medium was first removed and 1 mL of transport buffer (Hank's Balanced Salt Solution; Gibco, Thermo Fisher Scientific; 4.17 mM NaHCO3 and 25 mM HEPES adjusted to pH 7.4 with HCl) was added to wells for 3‐minute preincubation. The buffer was then replaced with 250 μL of test solution, which contained 0.1, 0.5, 1, 2, 4, 8, or 16 μM (*n *=* *2–3) of 3R,5S‐fluvastatin or 3S,5R‐fluvastatin for determining the transport kinetics, or 0.5 μM (*n *=* *6) for testing the effect of c.521T>C. After 1 minute, the test solution was removed, and the wells were washed with ice‐cold transport buffer three times and left to dry. Thereafter, 250 μL of methanol water mixture (3:1) containing deuterium‐labeled fluvastatin was added to lyse the cells. After 30 minutes, the samples were centrifuged and the supernatant was separated for LC‐MS/MS analysis.

Fluvastatin concentrations were measured with the Nexera X2 liquid chromatography‐AB Sciex 5500 Qtrap mass spectrometer. The chromatographic separation was achieved on a reversed phase Kinetex C8 column (75 × 2.1‐mm internal diameter, 2.6‐μm particle size; Phenomenex) with a mobile phase of 0.1% formic acid (A) and acetonitrile (B). The mobile phase flow rate was 300 μL/minute, and a gradient profile was applied as follows: a linear increase from 25% B to 65% B over 2.5 minute, then 0.5 minute at 90% B on hold before a re‐equilibration step to the starting composition. The mass spectrometric detection was performed as described for plasma samples. Dichlorofluorescein (Santa Cruz Biotechnology, Dallas, TX) was used for assay quality control in three wells expressing reference OATP1B1 or eYFP. These wells were lysed with 0.1 M NaOH. Protein concentration was determined by adding 300 μL of Coomassie Plus reagent (Thermo Fisher Scientific) to 10 μL of the quality control cell lysate and measuring absorbance at 595 nm with Varioskan LUX (Thermo Fischer Scientific). The cellular uptake was normalized to the protein content.

The OATP1B1‐mediated uptake was obtained by subtracting the uptake in eYFP‐transfected cells from that in OATP1B1‐transfected cells. Transport kinetics were calculated with GraphPad Prism 7 (GraphPad Software, San Diego, CA) to obtain Michaelis‐Menten constant (*K* _m_) and maximum transport rate (*V* _max_) values for both fluvastatin enantiomers. The CL_int,uptake_ was calculated as *V* _max_/*K* _m_.

### In vitro studies with HLM and recombinant enzymes

The incubations were performed with pooled HLM (Corning) and recombinant CYP2C9.1, CYP2C9.2, and CYP2C9.3 (EasyCYP, Cypex Ltd., Dundee, UK) enzymes. The incubation mixtures (*n *=* *3 or 6) contained 100 mM phosphate buffer, pH 7.4, 0.3 mg/mL of protein, 1 mM β‐nicotinamide adenine dinucleotide phosphate (β‐NADPH, tetrasodium salt), and 0.1 μM 3R,5S‐fluvastatin or 3S,5R‐fluvastatin in a total volume of 500 μL. Sulfaphenazole concentration was 10 μM in the inhibition experiments. Control incubations were performed in duplicate without the addition of NADPH. The reactions were initiated by the addition of the substrate or the mixture of substrate and inhibitor, and incubated in a shaking ThermoMixer (Eppendorf AG, Hamburg, Germany) at 37 °C. Reactions were stopped by transferring 50 μL samples to 50‐μL acetonitrile‐containing deuterium‐labeled fluvastatin at time points 0, 20, 40, and 60 minutes. After centrifugation, the supernatants were analyzed by using LC‐MS/MS as described for the OATP1B1 uptake experiments. The depletion rate constants (*k* _dep_) of the substrates were calculated with the first‐order elimination kinetics equation. CL_int_ in each matrix was obtained by dividing the *k* _dep_ with protein concentration.

### Statistical analysis

The data were analyzed with the statistical programs JMP Genomics 7.0 (SAS Institute, Inc., Cary, NC) and IBM SPSS 22.0 for Windows (Armonk, NY). The pharmacokinetic variables were logarithmically transformed before analysis. Sex, body weight, lean body weight,[47](#cpt1463-bib-0047) and body surface area[48](#cpt1463-bib-0048) were tested as demographic covariates for pharmacokinetic data using stepwise linear regression analysis, with *P* value thresholds of 0.05 for entry and 0.10 for removal. Possible effects of genetic variants on pharmacokinetic variables were investigated using linear regression analysis fixed for significant demographic covariates with a stepwise approach. A Bonferroni‐corrected *P* value threshold of 1.09 × 10^−6^ was employed for the 379 genes and thresholds of 0.05 for entry and 0.10 for removal for the candidate gene analysis. Additive coding was employed for genetic variants, and multiallelic variants were expanded. *CYP2D6* data were included as the activity scores in the candidate gene analysis. Haplotype computations for *SLCO1B1* were performed with PHASE v2.1.1.[49](#cpt1463-bib-0049), [50](#cpt1463-bib-0050) Statistical comparisons of *in vitro* data were done using independent samples Student's *t*‐test, with logarithmic transformation as appropriate.

## Funding

This study was supported by grants from the European Research Council (Grant agreement 282106), State funding for university‐level health research, and the Sigrid Jusélius Foundation (Helsinki, Finland).

## Conflicts of Interest

The authors declared no competing interests for this work.

## Author Contributions

P.H. and M.Ni. wrote the manuscript; P.H., A.T., J.T.B., and M.Ni. designed the research; P.H., A.T., M.Ne., W.K., T.T., M.P‐H., H.K., J.T.B., and M.Ni. performed the research; P.H. and M.Ni. analyzed the data.

## Supporting information

## Acknowledgments

The authors thank Katja Halme, Hanna Hyvärinen, Satu Karjalainen, Jouko Laitila, Eija Mäkinen‐Pulli, Raija Nevala, and Lisbet Partanen for skillful assistance in conducting the clinical pharmacokinetic study, and Pekka Ellonen and Maija Lepistö, for the massive parallel sequencing.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Kathawala, F.G. HMG‐CoA reductase inhibitors: an exciting development in the treatment of hyperlipoproteinemia. Med. Res. Rev. 11, 121–146 (1991).  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1673489/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med.%20Res.%20Rev.&title=HMG%E2%80%90CoA%20reductase%20inhibitors:%20an%20exciting%20development%20in%20the%20treatment%20of%20hyperlipoproteinemia&author=F.G.%20Kathawala&volume=11&publication_year=1991&pages=121-146&pmid=1673489&)

2. Fischer, V. et al The 3‐hydroxy‐3‐methylglutaryl coenzyme A reductase inhibitor fluvastatin: effect on human cytochrome P‐450 and implications for metabolic drug interactions. Drug Metab. Dispos. 27, 410–416 (1999).  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10064574/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=The%203%E2%80%90hydroxy%E2%80%903%E2%80%90methylglutaryl%20coenzyme%20A%20reductase%20inhibitor%20fluvastatin:%20effect%20on%20human%20cytochrome%20P%E2%80%90450%20and%20implications%20for%20metabolic%20drug%20interactions&author=V.%20Fischer&volume=27&publication_year=1999&pages=410-416&pmid=10064574&)

3. Neuvonen, P.J. , Niemi, M. & Backman, J.T. Drug interactions with lipid‐lowering drugs: mechanisms and clinical relevance. Clin. Pharmacol. Ther. 80, 565–581 (2006).  [DOI](https://doi.org/10.1016/j.clpt.2006.09.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17178259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Drug%20interactions%20with%20lipid%E2%80%90lowering%20drugs:%20mechanisms%20and%20clinical%20relevance&author=P.J.%20Neuvonen&author=M.%20Niemi&author=J.T.%20Backman&volume=80&publication_year=2006&pages=565-581&pmid=17178259&doi=10.1016/j.clpt.2006.09.003&)

4. Kopplow, K. , Letschert, K. , König, J. , Walter, B. & Keppler, D. Human hepatobiliary transport of organic anions analyzed by quadruple‐transfected cells. Mol. Pharmacol. 68, 1031–1038 (2005).  [DOI](https://doi.org/10.1124/mol.105.014605) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16046661/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=Human%C2%A0hepatobiliary%20transport%20of%20organic%20anions%20analyzed%20by%20quadruple%E2%80%90transfected%20cells&author=K.%20Kopplow&author=K.%20Letschert&author=J.%20K%C3%B6nig&author=B.%20Walter&author=D.%20Keppler&volume=68&publication_year=2005&pages=1031-1038&pmid=16046661&doi=10.1124/mol.105.014605&)

5. Hirano, M. , Maeda, K. , Matsushima, S. , Nozaki, Y. , Kusuhara, H. & Sugiyama, Y. Involvement of BCRP (ABCG2) in the biliary excretion of pitavastatin. Mol. Pharmacol. 68, 800–807 (2005).  [DOI](https://doi.org/10.1124/mol.105.014019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15955871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=Involvement%20of%20BCRP%20(ABCG2)%20in%20the%20biliary%20excretion%20of%20pitavastatin&author=M.%20Hirano&author=K.%20Maeda&author=S.%20Matsushima&author=Y.%20Nozaki&author=H.%20Kusuhara&volume=68&publication_year=2005&pages=800-807&pmid=15955871&doi=10.1124/mol.105.014019&)

6. Kalliokoski, A. & Niemi, M. Impact of OATP transporters on pharmacokinetics. Br. J. Pharmacol. 158, 693–705 (2009).  [DOI](https://doi.org/10.1111/j.1476-5381.2009.00430.x) | [PMC free article](/articles/PMC2765590/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19785645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Pharmacol.&title=Impact%20of%20OATP%20transporters%20on%20pharmacokinetics&author=A.%20Kalliokoski&author=M.%20Niemi&volume=158&publication_year=2009&pages=693-705&pmid=19785645&doi=10.1111/j.1476-5381.2009.00430.x&)

7. Greupink, R. , Dillen, L. , Monshouwer, M. , Huisman, M.T. & Russel, F.G. Interaction of fluvastatin with the liver‐specific Na + ‐dependent taurocholate cotransporting polypeptide (NTCP). Eur. J. Pharm. Sci. 44, 487–496 (2011).  [DOI](https://doi.org/10.1016/j.ejps.2011.09.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21945488/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pharm.%20Sci.&title=Interaction%20of%20fluvastatin%20with%20the%20liver%E2%80%90specific%20Na%C2%A0+%C2%A0%E2%80%90dependent%20taurocholate%20cotransporting%20polypeptide%20(NTCP)&author=R.%20Greupink&author=L.%20Dillen&author=M.%20Monshouwer&author=M.T.%20Huisman&author=F.G.%20Russel&volume=44&publication_year=2011&pages=487-496&pmid=21945488&doi=10.1016/j.ejps.2011.09.009&)

8. Ellis, L.C. , Hawksworth, G.M. & Weaver, R.J. ATP‐dependent transport of statins by human and rat MRP2/Mrp2. Toxicol. Appl. Pharmacol. 269, 187–194 (2013).  [DOI](https://doi.org/10.1016/j.taap.2013.03.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23562342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol.%20Appl.%20Pharmacol.&title=ATP%E2%80%90dependent%20transport%20of%20statins%20by%20human%20and%20rat%20MRP2/Mrp2&author=L.C.%20Ellis&author=G.M.%20Hawksworth&author=R.J.%20Weaver&volume=269&publication_year=2013&pages=187-194&pmid=23562342&doi=10.1016/j.taap.2013.03.019&)

9. Haining, R.L. , Hunter, A.P. , Veronese, M.E. , Trager, W.F. & Rettie, A.E. Allelic variants of human cytochrome P450 2C9: baculovirus‐mediated expression, purification, structural characterization, substrate stereoselectivity, and prochiral selectivity of the wild‐type and I359L mutant forms. Arch. Biochem. Biophys. 333, 447–458 (1996).  [DOI](https://doi.org/10.1006/abbi.1996.0414) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8809086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Biochem.%20Biophys.&title=Allelic%20variants%20of%20human%20cytochrome%20P450%202C9:%20baculovirus%E2%80%90mediated%20expression,%20purification,%20structural%20characterization,%20substrate%20stereoselectivity,%20and%20prochiral%20selectivity%20of%20the%20wild%E2%80%90type%20and%20I359L%20mutant%20forms&author=R.L.%20Haining&author=A.P.%20Hunter&author=M.E.%20Veronese&author=W.F.%20Trager&author=A.E.%20Rettie&volume=333&publication_year=1996&pages=447-458&pmid=8809086&doi=10.1006/abbi.1996.0414&)

10. Daly, A.K. , Rettie, A.E. , Fowler, D.M. & Miners, J.O. Pharmacogenomics of CYP2C9: functional and clinical considerations. J. Pers. Med. 8 (2017) 10.3390/jpm8010001.  [DOI](https://doi.org/10.3390/jpm8010001) | [PMC free article](/articles/PMC5872075/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29283396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pers.%20Med.&title=Pharmacogenomics%20of%20CYP2C9:%20functional%20and%20clinical%20considerations&author=A.K.%20Daly&author=A.E.%20Rettie&author=D.M.%20Fowler&author=J.O.%20Miners&volume=8&publication_year=2017&pmid=29283396&doi=10.3390/jpm8010001&)

11. Kirchheiner, J. et al Influence of CYP2C9 polymorphisms on the pharmacokinetics and cholesterol‐lowering activity of (‐)‐3S,5R‐fluvastatin and (+)‐3R,5S‐fluvastatin in healthy volunteers. Clin. Pharmacol. Ther. 74, 186–194 (2003).  [DOI](https://doi.org/10.1016/S0009-9236(03)00121-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12891229/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Influence%20of%20CYP2C9%20polymorphisms%20on%20the%20pharmacokinetics%20and%20cholesterol%E2%80%90lowering%20activity%20of%20(%E2%80%90)%E2%80%903S,5R%E2%80%90fluvastatin%20and%20(+)%E2%80%903R,5S%E2%80%90fluvastatin%20in%20healthy%20volunteers&author=J.%20Kirchheiner&volume=74&publication_year=2003&pages=186-194&pmid=12891229&doi=10.1016/S0009-9236(03)00121-8&)

12. Kondo, C. et al Functional analysis of SNPs variants of BCRP/ABCG2. Pharm. Res. 21, 1895–1903 (2004).  [DOI](https://doi.org/10.1023/b:pham.0000045245.21637.d4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15553238/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Res.&title=Functional%20analysis%20of%20SNPs%20variants%20of%20BCRP/ABCG2&author=C.%20Kondo&volume=21&publication_year=2004&pages=1895-1903&pmid=15553238&doi=10.1023/b:pham.0000045245.21637.d4&)

13. Keskitalo, J.E. , Zolk, O. , Fromm, M.F. , Kurkinen, K.J. , Neuvonen, P.J. & Niemi, M. ABCG2 polymorphism markedly affects the pharmacokinetics of atorvastatin and rosuvastatin. Clin. Pharmacol. Ther. 86, 197–203 (2009).  [DOI](https://doi.org/10.1038/clpt.2009.79) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19474787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=ABCG2%20polymorphism%20markedly%20affects%20the%20pharmacokinetics%20of%20atorvastatin%20and%20rosuvastatin&author=J.E.%20Keskitalo&author=O.%20Zolk&author=M.F.%20Fromm&author=K.J.%20Kurkinen&author=P.J.%20Neuvonen&volume=86&publication_year=2009&pages=197-203&pmid=19474787&doi=10.1038/clpt.2009.79&)

14. Keskitalo, J.E. , Pasanen, M.K. , Neuvonen, P.J. & Niemi, M. Different effects of the ABCG2 c.421C>A SNP on the pharmacokinetics of fluvastatin, pravastatin and simvastatin. Pharmacogenomics 10, 1617–1624 (2009).  [DOI](https://doi.org/10.2217/pgs.09.85) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19842935/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Different%20effects%20of%20the%20ABCG2%20c.421C>A%20SNP%20on%20the%20pharmacokinetics%20of%20fluvastatin,%20pravastatin%20and%20simvastatin&author=J.E.%20Keskitalo&author=M.K.%20Pasanen&author=P.J.%20Neuvonen&author=M.%20Niemi&volume=10&publication_year=2009&pages=1617-1624&pmid=19842935&doi=10.2217/pgs.09.85&)

15. Giacomini, K.M. et al International Transporter Consortium commentary on clinically important transporter polymorphisms. Clin. Pharmacol. Ther. 94, 23–26 (2013).  [DOI](https://doi.org/10.1038/clpt.2013.12) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23778707/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=International%20Transporter%20Consortium%20commentary%20on%20clinically%20important%20transporter%20polymorphisms&author=K.M.%20Giacomini&volume=94&publication_year=2013&pages=23-26&pmid=23778707&doi=10.1038/clpt.2013.12&)

16. Tirona, R.G. , Leake, B.F. , Merino, G. & Kim, R.B. Polymorphisms in OATP‐C: identification of multiple allelic variants associated with altered transport activity among European‐ and African‐Americans. J. Biol. Chem. 276, 35669–35675 (2001).  [DOI](https://doi.org/10.1074/jbc.M103792200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11477075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Polymorphisms%C2%A0in%C2%A0OATP%E2%80%90C:%20identification%20of%20multiple%20allelic%20variants%C2%A0associated%20with%20altered%20transport%20activity%20among%20European%E2%80%90%20and%20African%E2%80%90Americans&author=R.G.%20Tirona&author=B.F.%20Leake&author=G.%20Merino&author=R.B.%20Kim&volume=276&publication_year=2001&pages=35669-35675&pmid=11477075&doi=10.1074/jbc.M103792200&)

17. Nishizato, Y. et al Polymorphisms of OATP‐C (SLC21A6) and OAT3 (SLC22A8) genes: consequences for pravastatin pharmacokinetics. Clin. Pharmacol. Ther. 73, 554–565 (2003).  [DOI](https://doi.org/10.1016/S0009-9236(03)00060-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12811365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Polymorphisms%20of%20OATP%E2%80%90C%20(SLC21A6)%20and%20OAT3%20(SLC22A8)%20genes:%20consequences%20for%20pravastatin%20pharmacokinetics&author=Y.%20Nishizato&volume=73&publication_year=2003&pages=554-565&pmid=12811365&doi=10.1016/S0009-9236(03)00060-2&)

18. Niemi, M. et al High plasma pravastatin concentrations are associated with single nucleotide polymorphisms and haplotypes of organic anion transporting polypeptide‐C (OATP‐C, SLCO1B1). Pharmacogenetics 14, 429–440 (2004).  [DOI](https://doi.org/10.1097/01.fpc.0000114750.08559.32) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15226675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=High%20plasma%20pravastatin%20concentrations%20are%20associated%20with%20single%20nucleotide%20polymorphisms%20and%20haplotypes%20of%20organic%20anion%20transporting%20polypeptide%E2%80%90C%20(OATP%E2%80%90C,%20SLCO1B1)&author=M.%20Niemi&volume=14&publication_year=2004&pages=429-440&pmid=15226675&doi=10.1097/01.fpc.0000114750.08559.32&)

19. Mwinyi, J. , Johne, A. , Bauer, S. , Roots, I. & Gerloff, T. Evidence for inverse effects of OATP‐C (SLC21A6) 5 and 1b haplotypes on pravastatin kinetics. Clin. Pharmacol. Ther. 75, 415–421 (2004).  [DOI](https://doi.org/10.1016/j.clpt.2003.12.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15116054/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Evidence%20for%20inverse%20effects%20of%20OATP%E2%80%90C%20(SLC21A6)%205%20and%201b%20haplotypes%20on%20pravastatin%20kinetics&author=J.%20Mwinyi&author=A.%20Johne&author=S.%20Bauer&author=I.%20Roots&author=T.%20Gerloff&volume=75&publication_year=2004&pages=415-421&pmid=15116054&doi=10.1016/j.clpt.2003.12.016&)

20. Pasanen, M.K. , Neuvonen, M. , Neuvonen, P.J. & Niemi, M. SLCO1B1 polymorphism markedly affects the pharmacokinetics of simvastatin acid. Pharmacogenet. Genomics 16, 873–879 (2006).  [DOI](https://doi.org/10.1097/01.fpc.0000230416.82349.90) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17108811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=SLCO1B1%20polymorphism%20markedly%20affects%20the%20pharmacokinetics%20of%20simvastatin%20acid&author=M.K.%20Pasanen&author=M.%20Neuvonen&author=P.J.%20Neuvonen&author=M.%20Niemi&volume=16&publication_year=2006&pages=873-879&pmid=17108811&doi=10.1097/01.fpc.0000230416.82349.90&)

21. Niemi, M. , Pasanen, M.K. & Neuvonen, P.J. SLCO1B1 polymorphism and sex affect the pharmacokinetics of pravastatin but not fluvastatin. Clin. Pharmacol. Ther. 80, 356–366 (2006).  [DOI](https://doi.org/10.1016/j.clpt.2006.06.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17015053/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=SLCO1B1%20polymorphism%20and%20sex%20affect%20the%20pharmacokinetics%20of%20pravastatin%20but%20not%20fluvastatin&author=M.%20Niemi&author=M.K.%20Pasanen&author=P.J.%20Neuvonen&volume=80&publication_year=2006&pages=356-366&pmid=17015053&doi=10.1016/j.clpt.2006.06.010&)

22. Niemi, M. , Pasanen, M.K. & Neuvonen, P.J. Organic anion transporting polypeptide 1B1: a genetically polymorphic transporter of major importance for hepatic drug uptake. Pharmacol. Rev. 63, 157–181 (2011).  [DOI](https://doi.org/10.1124/pr.110.002857) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21245207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Rev.&title=Organic%20anion%20transporting%20polypeptide%201B1:%20a%20genetically%20polymorphic%20transporter%20of%20major%20importance%20for%20hepatic%20drug%20uptake&author=M.%20Niemi&author=M.K.%20Pasanen&author=P.J.%20Neuvonen&volume=63&publication_year=2011&pages=157-181&pmid=21245207&doi=10.1124/pr.110.002857&)

23. Tornio, A. , Vakkilainen, J. , Neuvonen, M. , Backman, J.T. , Neuvonen, P.J. & Niemi, M. SLCO1B1 polymorphism markedly affects the pharmacokinetics of lovastatin acid. Pharmacogenet. Genomics 25, 382–387 (2015).  [DOI](https://doi.org/10.1097/FPC.0000000000000148) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26020121/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=SLCO1B1%20polymorphism%20markedly%20affects%20the%20pharmacokinetics%20of%20lovastatin%20acid&author=A.%20Tornio&author=J.%20Vakkilainen&author=M.%20Neuvonen&author=J.T.%20Backman&author=P.J.%20Neuvonen&volume=25&publication_year=2015&pages=382-387&pmid=26020121&doi=10.1097/FPC.0000000000000148&)

24. Keskitalo, J.E. , Kurkinen, K.J. , Neuvonen, M. , Backman, J.T. , Neuvonen, P.J. & Niemi, M. No significant effect of ABCB1 haplotypes on the pharmacokinetics of fluvastatin, pravastatin, lovastatin, and rosuvastatin. Br. J. Clin. Pharmacol. 68, 207–213 (2009).  [DOI](https://doi.org/10.1111/j.1365-2125.2009.03440.x) | [PMC free article](/articles/PMC2767284/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19694740/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=No%20significant%20effect%20of%20ABCB1%20haplotypes%20on%20the%20pharmacokinetics%20of%20fluvastatin,%20pravastatin,%20lovastatin,%20and%20rosuvastatin&author=J.E.%20Keskitalo&author=K.J.%20Kurkinen&author=M.%20Neuvonen&author=J.T.%20Backman&author=P.J.%20Neuvonen&volume=68&publication_year=2009&pages=207-213&pmid=19694740&doi=10.1111/j.1365-2125.2009.03440.x&)

25. Baldwin, S.J. , Bloomer, J.C. , Smith, G.J. , Ayrton, A.D. , Clarke, S.E. & Chenery, R.J. Ketoconazole and sulphaphenazole as the respective selective inhibitors of P4503A and 2C9. Xenobiotica 25, 261–270 (1995).  [DOI](https://doi.org/10.3109/00498259509061850) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7618352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Ketoconazole%20and%20sulphaphenazole%20as%20the%20respective%20selective%20inhibitors%20of%20P4503A%20and%202C9&author=S.J.%20Baldwin&author=J.C.%20Bloomer&author=G.J.%20Smith&author=A.D.%20Ayrton&author=S.E.%20Clarke&volume=25&publication_year=1995&pages=261-270&pmid=7618352&doi=10.3109/00498259509061850&)

26. Rettie, A.E. , Wienkers, L.C. , Gonzalez, F.J. , Trager, W.F. & Korzekwa, K.R. Impaired (S)‐warfarin metabolism catalysed by the R144C allelic variant of CYP2C9. Pharmacogenetics 4, 39–42 (1994).  [DOI](https://doi.org/10.1097/00008571-199402000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8004131/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Impaired%20(S)%E2%80%90warfarin%20metabolism%20catalysed%20by%20the%20R144C%20allelic%20variant%20of%20CYP2C9&author=A.E.%20Rettie&author=L.C.%20Wienkers&author=F.J.%20Gonzalez&author=W.F.%20Trager&author=K.R.%20Korzekwa&volume=4&publication_year=1994&pages=39-42&pmid=8004131&doi=10.1097/00008571-199402000-00005&)

27. Nies, A.T. et al Genetics is a major determinant of expression of the human hepatic uptake transporter OATP1B1, but not of OATP1B3 and OATP2B1. Genome Med. 5 (2013) 10.1186/gm405.  [DOI](https://doi.org/10.1186/gm405) | [PMC free article](/articles/PMC3706890/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23311897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med.&title=Genetics%20is%20a%20major%20determinant%20of%20expression%20of%20the%20human%20hepatic%20uptake%20transporter%20OATP1B1,%20but%20not%20of%20OATP1B3%20and%20OATP2B1&author=A.T.%20Nies&volume=5&publication_year=2013&pmid=23311897&doi=10.1186/gm405&)

28. Ramsey, L.B. et al Rare versus common variants in pharmacogenetics: SLCO1B1 variation and methotrexate disposition. Genome Res. 22, 1–8 (2012).  [DOI](https://doi.org/10.1101/gr.129668.111) | [PMC free article](/articles/PMC3246196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22147369/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Rare%20versus%20common%20variants%20in%20pharmacogenetics:%20SLCO1B1%20variation%20and%20methotrexate%20disposition&author=L.B.%20Ramsey&volume=22&publication_year=2012&pages=1-8&pmid=22147369&doi=10.1101/gr.129668.111&)

29. Weiner, M. et al Effects of tuberculosis, race, and human gene SLCO1B1 polymorphisms on rifampin concentrations. Antimicrob. Agents Chemother. 54, 4192–4200 (2010).  [DOI](https://doi.org/10.1128/AAC.00353-10) | [PMC free article](/articles/PMC2944564/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20660695/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Effects%20of%20tuberculosis,%20race,%20and%20human%20gene%20SLCO1B1%20polymorphisms%20on%20rifampin%20concentrations&author=M.%20Weiner&volume=54&publication_year=2010&pages=4192-4200&pmid=20660695&doi=10.1128/AAC.00353-10&)

30. Imanaga, J. et al The effects of the SLCO2B1 c.1457C > T polymorphism and apple juice on the pharmacokinetics of fexofenadine and midazolam in humans. Pharmacogenet. Genomics 21, 84–93 (2011).  [DOI](https://doi.org/10.1097/fpc.0b013e32834300cc) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21280267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=The%20effects%20of%20the%20SLCO2B1%20c.1457C%20>%20T%20polymorphism%20and%20apple%20juice%20on%20the%20pharmacokinetics%20of%20fexofenadine%20and%20midazolam%20in%20humans&author=J.%20Imanaga&volume=21&publication_year=2011&pages=84-93&pmid=21280267&doi=10.1097/fpc.0b013e32834300cc&)

31. Ieiri, I. et al Microdosing clinical study: pharmacokinetic, pharmacogenomic (SLCO2B1), and interaction (grapefruit juice) profiles of celiprolol following the oral microdose and therapeutic dose. J. Clin. Pharmacol. 52, 1078–1089 (2012).  [DOI](https://doi.org/10.1177/0091270011408612) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21593283/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharmacol.&title=Microdosing%20clinical%20study:%20pharmacokinetic,%20pharmacogenomic%20(SLCO2B1),%20and%20interaction%20(grapefruit%20juice)%20profiles%20of%20celiprolol%20following%20the%20oral%20microdose%20and%20therapeutic%20dose&author=I.%20Ieiri&volume=52&publication_year=2012&pages=1078-1089&pmid=21593283&doi=10.1177/0091270011408612&)

32. Alfirevic, A. et al Phenotype standardization for statin‐induced myotoxicity. Clin. Pharmacol. Ther. 96, 470–476 (2014).  [DOI](https://doi.org/10.1038/clpt.2014.121) | [PMC free article](/articles/PMC4172546/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24897241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Phenotype%20standardization%20for%20statin%E2%80%90induced%20myotoxicity&author=A.%20Alfirevic&volume=96&publication_year=2014&pages=470-476&pmid=24897241&doi=10.1038/clpt.2014.121&)

33. Thompson, P.D. , Clarkson, P. & Karas, R.H. Statin‐associated myopathy. JAMA 289, 1681–1690 (2003).  [DOI](https://doi.org/10.1001/jama.289.13.1681) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12672737/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Statin%E2%80%90associated%20myopathy&author=P.D.%20Thompson&author=P.%20Clarkson&author=R.H.%20Karas&volume=289&publication_year=2003&pages=1681-1690&pmid=12672737&doi=10.1001/jama.289.13.1681&)

34. Miroševic Skvrce, N. , Božina, N. , Zibar, L. , Barišic, I. , Pejnovic, L. & Macolic Šarinic, V. CYP2C9 and ABCG2 polymorphisms as risk factors for developing adverse drug reactions in renal transplant patients taking fluvastatin: a case‐control study. Pharmacogenomics 14, 1419–1431 (2013).  [DOI](https://doi.org/10.2217/pgs.13.135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24024895/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CYP2C9%20and%20ABCG2%20polymorphisms%20as%20risk%20factors%20for%20developing%20adverse%20drug%20reactions%20in%20renal%20transplant%20patients%20taking%20fluvastatin:%20a%20case%E2%80%90control%20study&author=N.%20Miro%C5%A1evic%20Skvrce&author=N.%20Bo%C5%BEina&author=L.%20Zibar&author=I.%20Bari%C5%A1ic&author=L.%20Pejnovic&volume=14&publication_year=2013&pages=1419-1431&pmid=24024895&doi=10.2217/pgs.13.135&)

35. SEARCH Collaborative Group SLCO1B1 variants and statin‐induced myopathy–a genomewide study. N. Engl. J. Med. 359, 789–799 (2008).  [DOI](https://doi.org/10.1056/NEJMoa0801936) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18650507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=SLCO1B1%20variants%20and%20statin%E2%80%90induced%20myopathy%E2%80%93a%20genomewide%20study&volume=359&publication_year=2008&pages=789-799&pmid=18650507&doi=10.1056/NEJMoa0801936&)

36. Adams, S.P. , Sekhon, S.S. , Tsang, M. & Wright, J.M. Fluvastatin for lowering lipids. Cochrane Database Syst. Rev. 3, CD012282 (2018).  [DOI](https://doi.org/10.1002/14651858.CD012282.pub2) | [PMC free article](/articles/PMC6494196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29508377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Database%20Syst.%20Rev.&title=Fluvastatin%20for%20lowering%20lipids&author=S.P.%20Adams&author=S.S.%20Sekhon&author=M.%20Tsang&author=J.M.%20Wright&volume=3&publication_year=2018&pmid=29508377&doi=10.1002/14651858.CD012282.pub2&)

37. Buzková, H. et al Lipid‐lowering effect of fluvastatin in relation to cytochrome P450 2C9 variant alleles frequently distributed in the Czech population. Med. Sci. Monit. 18, 512–517 (2012).  [DOI](https://doi.org/10.12659/MSM.883272) | [PMC free article](/articles/PMC3560696/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22847201/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med.%20Sci.%20Monit.&title=Lipid%E2%80%90lowering%20effect%20of%20fluvastatin%20in%20relation%20to%20cytochrome%20P450%202C9%20variant%20alleles%20frequently%20distributed%20in%20the%20Czech%20population&author=H.%20Buzkov%C3%A1&volume=18&publication_year=2012&pages=512-517&pmid=22847201&doi=10.12659/MSM.883272&)

38. Meyer zu Schwabedissen, H.E. et al Function‐impairing polymorphisms of the hepatic uptake transporter SLCO1B1 modify the therapeutic efficacy of statins in a population‐based cohort. Pharmacogenet. Genomics 25, 8–18 (2015).  [DOI](https://doi.org/10.1097/FPC.0000000000000098) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25379722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Function%E2%80%90impairing%20polymorphisms%20of%20the%20hepatic%20uptake%20transporter%20SLCO1B1%20modify%20the%20therapeutic%20efficacy%20of%20statins%20in%20a%20population%E2%80%90based%20cohort&author=H.E.%20Meyer%20zu%20Schwabedissen&volume=25&publication_year=2015&pages=8-18&pmid=25379722&doi=10.1097/FPC.0000000000000098&)

39. Couvert, P. et al Association between a frequent allele of the gene encoding OATP1B1 and enhanced LDL‐lowering response to fluvastatin therapy. Pharmacogenomics 9, 1217–1227 (2008).  [DOI](https://doi.org/10.2217/14622416.9.9.1217) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18781850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Association%20between%20a%20frequent%20allele%20of%20the%20gene%20encoding%20OATP1B1%20and%20enhanced%20LDL%E2%80%90lowering%20response%20to%20fluvastatin%20therapy&author=P.%20Couvert&volume=9&publication_year=2008&pages=1217-1227&pmid=18781850&doi=10.2217/14622416.9.9.1217&)

40. Pasanen, M.K. , Neuvonen, P.J. & Niemi, M. Global analysis of genetic variation in SLCO1B1 . Pharmacogenomics 9, 19–33 (2008).  [DOI](https://doi.org/10.2217/14622416.9.1.19) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18154446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Global%20analysis%20of%20genetic%20variation%20in%20SLCO1B1&author=M.K.%20Pasanen&author=P.J.%20Neuvonen&author=M.%20Niemi&volume=9&publication_year=2008&pages=19-33&pmid=18154446&doi=10.2217/14622416.9.1.19&)

41. Zhou, Y. , Ingelman‐Sundberg, M. & Lauschke, V.M. Worldwide distribution of cytochrome P450 alleles: a meta‐analysis of population‐scale sequencing projects. Clin. Pharmacol. Ther. 102, 688–700 (2017).  [DOI](https://doi.org/10.1002/cpt.690) | [PMC free article](/articles/PMC5600063/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28378927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Worldwide%20distribution%20of%20cytochrome%20P450%20alleles:%20a%20meta%E2%80%90analysis%20of%20population%E2%80%90scale%20sequencing%20projects&author=Y.%20Zhou&author=M.%20Ingelman%E2%80%90Sundberg&author=V.M.%20Lauschke&volume=102&publication_year=2017&pages=688-700&pmid=28378927&doi=10.1002/cpt.690&)

42. Hirvensalo, P. et al Comprehensive pharmacogenomic study reveals an important role of UGT1A3 in montelukast pharmacokinetics. Clin. Pharmacol. Ther. 104, 158–168 (2018).  [DOI](https://doi.org/10.1002/cpt.891) | [PMC free article](/articles/PMC6033076/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28940478/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Comprehensive%20pharmacogenomic%20study%20reveals%20an%20important%20role%20of%20UGT1A3%20in%20montelukast%20pharmacokinetics&author=P.%20Hirvensalo&volume=104&publication_year=2018&pages=158-168&pmid=28940478&doi=10.1002/cpt.891&)

43. Sulonen, A.M. et al Comparison of solution‐based exome capture methods for next generation sequencing. Genome Biol. 12, R94 (2011).  [DOI](https://doi.org/10.1186/gb-2011-12-9-r94) | [PMC free article](/articles/PMC3308057/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21955854/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&title=Comparison%20of%20solution%E2%80%90based%20exome%20capture%20methods%20for%20next%20generation%20sequencing&author=A.M.%20Sulonen&volume=12&publication_year=2011&pages=R94&pmid=21955854&doi=10.1186/gb-2011-12-9-r94&)

44. Gonçalves, B.P. et al Age, weight, and CYP2D6 genotype are major determinants of primaquine pharmacokinetics in African children. Antimicrob. Agents Chemother. 61, e02590‐16 (2017).  [DOI](https://doi.org/10.1128/AAC.02590-16) | [PMC free article](/articles/PMC5404566/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28289025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&title=Age,%20weight,%20and%20CYP2D6%20genotype%20are%20major%20determinants%20of%20primaquine%20pharmacokinetics%20in%20African%20children&author=B.P.%20Gon%C3%A7alves&volume=61&publication_year=2017&pages=e02590%E2%80%9016&pmid=28289025&doi=10.1128/AAC.02590-16&)

45. Gaedigk, A. , Simon, S.D. , Pearce, R.E. , Bradford, L.D. , Kennedy, M.J. & Leeder, J.S. The CYP2D6 activity score: translating genotype information into a qualitative measure of phenotype. Clin. Pharmacol. Ther. 83, 234–242 (2008).  [DOI](https://doi.org/10.1038/sj.clpt.6100406) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20CYP2D6%20activity%20score:%20translating%20genotype%20information%20into%20a%20qualitative%20measure%20of%20phenotype&author=A.%20Gaedigk&author=S.D.%20Simon&author=R.E.%20Pearce&author=L.D.%20Bradford&author=M.J.%20Kennedy&volume=83&publication_year=2008&pages=234-242&pmid=17971818&doi=10.1038/sj.clpt.6100406&)

46. Sjöstedt, N. , van den Heuvel, J.J.M.W. , Koenderink, J.B. & Kidron, H. Transmembrane domain single‐nucleotide polymorphisms impair expression and transport activity of ABC transporter ABCG2. Pharm. Res. 34, 1626–1636 (2017).  [DOI](https://doi.org/10.1007/s11095-017-2127-1) | [PMC free article](/articles/PMC5498656/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28281205/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Res.&title=Transmembrane%20domain%20single%E2%80%90nucleotide%20polymorphisms%20impair%20expression%20and%20transport%20activity%20of%20ABC%20transporter%20ABCG2&author=N.%20Sj%C3%B6stedt&author=J.J.M.W.%20van%20den%20Heuvel&author=J.B.%20Koenderink&author=H.%20Kidron&volume=34&publication_year=2017&pages=1626-1636&pmid=28281205&doi=10.1007/s11095-017-2127-1&)

47. Hallynck, T.H. , Soep, H.H. , Thomis, J.A. , Boelaert, J. , Daneels, R. & Dettli, L. Should clearance be normalised to body surface or to lean body mass? Br. J. Clin. Pharmacol. 11, 523–526 (1981).  [DOI](https://doi.org/10.1111/j.1365-2125.1981.tb01163.x) | [PMC free article](/articles/PMC1401592/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7272167/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Should%20clearance%20be%20normalised%20to%20body%20surface%20or%C2%A0to%C2%A0lean%20body%20mass?&author=T.H.%20Hallynck&author=H.H.%20Soep&author=J.A.%20Thomis&author=J.%20Boelaert&author=R.%20Daneels&volume=11&publication_year=1981&pages=523-526&pmid=7272167&doi=10.1111/j.1365-2125.1981.tb01163.x&)

48. Haycock, G.B. , Schwartz, G.J. & Wisotsky, D.H. Geometric method for measuring body surface area: a height‐weight formula validated in infants, children, and adults. J. Pediatr. 93, 62–66 (1978).  [DOI](https://doi.org/10.1016/s0022-3476(78)80601-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/650346/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pediatr.&title=Geometric%20method%20for%20measuring%20body%20surface%20area:%20a%20height%E2%80%90weight%20formula%20validated%20in%20infants,%20children,%20and%20adults&author=G.B.%20Haycock&author=G.J.%20Schwartz&author=D.H.%20Wisotsky&volume=93&publication_year=1978&pages=62-66&pmid=650346&doi=10.1016/s0022-3476(78)80601-5&)

49. Stephens, M. , Smith, N.J. & Donnelly, P. A new statistical method for haplotype reconstruction from population data. Am. J. Hum. Genet. 68, 978–989 (2001).  [DOI](https://doi.org/10.1086/319501) | [PMC free article](/articles/PMC1275651/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11254454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=A%20new%20statistical%20method%20for%20haplotype%20reconstruction%20from%20population%20data&author=M.%20Stephens&author=N.J.%20Smith&author=P.%20Donnelly&volume=68&publication_year=2001&pages=978-989&pmid=11254454&doi=10.1086/319501&)

50. Stephens, M. & Donnelly, P. A comparison of bayesian methods for haplotype reconstruction from population genotype data. Am. J. Hum. Genet. 73, 1162–1169 (2003).  [DOI](https://doi.org/10.1086/379378) | [PMC free article](/articles/PMC1180495/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14574645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=A%20comparison%20of%20bayesian%20methods%20for%20haplotype%20reconstruction%20from%20population%20genotype%20data&author=M.%20Stephens&author=P.%20Donnelly&volume=73&publication_year=2003&pages=1162-1169&pmid=14574645&doi=10.1086/379378&)
