# Genome‐Wide Meta‐Analysis of Blood Pressure Response to β1‐Blockers: Results From ICAPS (International Consortium of Antihypertensive Pharmacogenomics Studies)

## Metadata
**Authors:** Sonal Singh, Helen R Warren, Timo P Hiltunen, Caitrin W McDonough, Nihal El Rouby, Erika Salvi, Zhiying Wang, Tatiana Garofalidou, Frej Fyhrquist, Kimmo K Kontula, Valeria Glorioso, Roberta Zaninello, Nicola Glorioso, Carl J Pepine, Patricia B Munroe, Stephan T Turner, Arlene B Chapman, Eric Boerwinkle, Julie A Johnson, Yan Gong, Rhonda M Cooper‐DeHoff
**Journal:** Journal of the American Heart Association: Cardiovascular and Cerebrovascular Disease
**Date:** 2019 Aug 19
**DOI:** [10.1161/JAHA.119.013115](https://doi.org/10.1161/JAHA.119.013115)
**PMID:** 31423876
**PMCID:** PMC6759913
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6759913/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6759913/pdf/JAH3-8-e013115.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6759913/pdf/JAH3-8-e013115.pdf)

## Abstract

**Background:** 
There exists a wide interindividual variability in blood pressure (BP) response to β1‐blockers. To identify the genetic determinants of this variability, we performed a pharmacogenomic genome‐wide meta‐analysis of genetic variants influencing β1‐blocker BP response.

**Methods and Results:** 
Genome‐wide association analysis for systolic BP and diastolic BP response to β1‐blockers from 5 randomized clinical trials consisting of 1254 patients with hypertension of European ancestry were combined in meta‐analysis and single nucleotide polymorphisms (SNPs) with P<10−4 were tested for replication in 2 independent randomized clinical trials of β1‐blocker–treated patients of European ancestry (n=1552). Regions harboring the replicated SNPs were validated in a β1‐blocker–treated black cohort from 2 randomized clinical trials (n=315). A missense SNP rs28404156 in BST1 was associated with systolic BP response to β1‐blockers in the discovery meta‐analysis (P=9.33×10−5, β=−3.21 mm Hg) and replicated at Bonferroni significance (P=1.85×10−4, β=−4.86 mm Hg) in the replication meta‐analysis with combined meta‐analysis approaching genome‐wide significance (P=2.18×10−7). This SNP in BST1 is in linkage disequilibrium with several SNPs with putative regulatory functions in nearby genes, including CD38,FBXL5, and FGFBP1, all of which have been implicated in BP regulation. SNPs in this genetic region were also associated with BP response in the black cohort.

**Conclusions:** 
Data from randomized clinical trials of 8 European ancestry and 2 black cohorts support the assumption that BST1 containing locus on chromosome 4 is associated with β1‐blocker BP response. Given the previous associations of this region with BP, this is a strong candidate region for future functional studies and potential use in precision medicine approaches for BP management and risk prediction.

Keywords: blood pressure, hypertension, meta‐analysis, pharmacogenomics, β1‐blocker, β‐blocker

Subject Categories: Genetic, Association Studies; Hypertension; High Blood Pressure; Clinical Studies

### Background

There exists a wide interindividual variability in blood pressure (BP) response to β_1_‐blockers. To identify the genetic determinants of this variability, we performed a pharmacogenomic genome‐wide meta‐analysis of genetic variants influencing β_1_‐blocker BP response.

### Methods and Results

Genome‐wide association analysis for systolic BP and diastolic BP response to β_1_‐blockers from 5 randomized clinical trials consisting of 1254 patients with hypertension of European ancestry were combined in meta‐analysis and single nucleotide polymorphisms (SNPs) with *P*<10^−4^ were tested for replication in 2 independent randomized clinical trials of β_1_‐blocker–treated patients of European ancestry (n=1552). Regions harboring the replicated SNPs were validated in a β_1_‐blocker–treated black cohort from 2 randomized clinical trials (n=315). A missense SNP rs28404156 in *BST1* was associated with systolic BP response to β_1_‐blockers in the discovery meta‐analysis (*P*=9.33×10^−5^, β=−3.21 mm Hg) and replicated at Bonferroni significance (*P*=1.85×10^−4^, β=−4.86 mm Hg) in the replication meta‐analysis with combined meta‐analysis approaching genome‐wide significance (*P*=2.18×10^−7^). This SNP in *BST1* is in linkage disequilibrium with several SNPs with putative regulatory functions in nearby genes, including *CD38*,*FBXL5*, and *FGFBP1*, all of which have been implicated in BP regulation. SNPs in this genetic region were also associated with BP response in the black cohort.

### Conclusions

Data from randomized clinical trials of 8 European ancestry and 2 black cohorts support the assumption that *BST1* containing locus on chromosome 4 is associated with β_1_‐blocker BP response. Given the previous associations of this region with BP, this is a strong candidate region for future functional studies and potential use in precision medicine approaches for BP management and risk prediction.

## Clinical Perspective

## Clinical Perspective

### What Is New?

### What Are the Clinical Implications?

Hypertension is a preventable risk factor for stroke, chronic heart failure, coronary artery disease, and kidney disease resulting in premature death and disability.[1](#jah34354-bib-0001) Hypertension remains a global burden with an estimated direct cost of ≈$200 billion by 2030.[2](#jah34354-bib-0002) More than 1.3 billion people worldwide are estimated to have hypertension and this estimate is expected to increase to 1.56 billion people globally by 2025.[1](#jah34354-bib-0001), [3](#jah34354-bib-0003) Moreover, ≈4 in 10 adults older than 25 years have hypertension, and 9 of 10 adults living up to 80 years will develop hypertension.[4](#jah34354-bib-0004)

β‐Blockers, one of the many available pharmacotherapeutic interventions, have been used for the past 45 years and previously were recommended as first‐line agents for treating uncomplicated hypertension. However, over recent years, β‐blockers have fallen out of favor as first‐line therapy because of results of large meta‐analyses documenting poorer outcomes when compared with other drug classes such as diuretics, angiotensin‐converting enzyme inhibitors, angiotensin receptor blockers, or calcium channel blockers, for treating hypertension.[5](#jah34354-bib-0005) Based on findings from these large studies, recent guidelines including American College of Cardiology/American Heart Association and National Institute for Health and Care Excellence no longer recommend β‐blockers as a first line of treatment in all or in a subset of patients.[6](#jah34354-bib-0006), [7](#jah34354-bib-0007) While β‐blockers are not recommended as first‐line agents, they are still a preferred choice for treating patients with hypertension who have comorbid cardiovascular conditions such as angina, chronic heart failure, and arrhythmias, and as part of combination therapy for resistant hypertension.[8](#jah34354-bib-0008), [9](#jah34354-bib-0009) As a result of this continued use of β‐blockers, there remains a need for individualizing β‐blocker–based antihypertensive therapy for better hypertension management, blood pressure (BP) control, and improved outcomes of patients who require β‐blocker therapy.

Studies have documented that patients of white ancestry usually respond favorably to β‐blocker therapy compared with other antihypertensive medications such as diuretics that result in better BP control in patients of black ancestry.[10](#jah34354-bib-0010) However, there exists a wide interindividual variability in BP response to β‐blockers among patients of white ancestry, suggesting a need to understand related factors (genetic and nongenetic) that contribute to variability in BP response to β‐blockers, with an ultimate goal of optimizing BP management.[11](#jah34354-bib-0011) Furthermore, β_1_‐selective blockers are the β‐blocker of choice for patients with hypertension who require β‐blocker therapy.[16](#jah34354-bib-0016), [17](#jah34354-bib-0017) Identification of genetic factors of β_1_‐selective blocker therapy response through pharmacogenomics analyses could have the potential to achieve some of the goals for individualizing therapy among individuals with hypertension. To this end, several genome‐wide analyses conducted through individual studies have identified a few replicated signals of β_1_‐selective blocker therapy response.[12](#jah34354-bib-0012), [13](#jah34354-bib-0013), [14](#jah34354-bib-0014), [15](#jah34354-bib-0015) To the best of our knowledge, there are no genome‐wide analyses for nonselective β‐blockers, which are not currently recommended as first line for treatment of hypertension. In this article, we aimed to perform the largest 2‐stage genome‐wide meta‐analysis using data from randomized clinical trials with genome‐wide data, as part of the ICAPS (International Consortium for Antihypertensive Pharmacogenomics Studies). However, as mentioned before, ICAPS only had pharmacogenomics trials with whole genome data for β_1_‐selective blocker–centered studies. Based on the phenotype of interest, our meta‐analysis study reported in this article focused on BP response to β_1_‐selective blockers, which comprised a total of 8 cohorts of hypertensive patients of white ancestry treated with β_1_‐blockers to identify pharmacogenetic markers associated with BP response.

## Methods

### Study Participants and Inclusion Criteria

The discovery genome‐wide association study (GWAS) meta‐analysis consisted of 5 study cohorts: PEAR (Pharmacogenomic Evaluation of Antihypertensive Responses),[18](#jah34354-bib-0018) PEAR‐2 (Pharmacogenomic Evaluation of Antihypertensive Responses 2),[19](#jah34354-bib-0019) LIFE‐Fin (Finnish arm of the Losartan Intervention For Endpoint Reduction in Hypertension Study),[20](#jah34354-bib-0020) GENRES (Genetics of Drug Responsiveness in Essential Hypertension)[13](#jah34354-bib-0013) and BB‐SS (Pharmacogenomics of Beta‐Blockers Sardinian Study). Detailed information of each of these cohorts is described in Data [S1](#jah34354-sup-0001). For all participating cohorts, patients with uncomplicated hypertension and untreated systolic BP (SBP) >140 mm Hg or diastolic BP (DBP) >90 mm Hg were included in the study. Patients from all participating studies in the discovery cohort underwent an antihypertensive medication washout before starting the study, after which they were treated with β_1_‐blocker monotherapy for varying lengths of time according to each study‐specific protocol.

For replication, a meta‐analysis consisting of 3 study cohorts (INVEST [International Verapamil SR‐Trandolapril Study],[21](#jah34354-bib-0021) ASCOT‐UK [Anglo‐Scandinavian Cardiac Outcomes Trial‐United Kingdom],[22](#jah34354-bib-0022) and ASCOT‐SC [Anglo‐Scandinavian Cardiac Outcomes Trial‐Scandinavian][22](#jah34354-bib-0022)) was used. ASCOT‐UK, ASCOT‐SC, and INVEST did not have a hypertensive medication washout period before starting the β_1_‐blocker therapy. Black participants from PEAR and PEAR‐2 were used for ethnic validation.

All of the studies were approved by institutional review committee, all of the participants signed informed consent, and studies were conducted in accordance with the regulations set forth by the Declaration of Helsinki. The data for PEAR have been made publicly available in the database of Genotypes and Phenotypes (dbGaP)[23](#jah34354-bib-0023) (Accession: phs000649.v1.p1). The PEAR‐2 and INVEST data are currently in the process of being uploaded and will soon be available to other researchers in dbGaP. Clinical Trials included in this meta‐analysis can be found at: URL: [https://www.clinicaltrials.gov](https://www.clinicaltrials.gov). Unique identifiers: PEAR (Pharmacogenomic Evaluation of Antihypertensive Responses): NCT00246519; PEAR‐2 (Pharmacogenomic Evaluation of Antihypertensive Responses 2): NCT01203852; INVEST (International Verapamil SR Trandolapril Study): NCT00133692; GENRES (A Study on Molecular Genetics of Drug Responsiveness in Essential Hypertension): NCT03276598; and LIFE (Losartan Intervention For Endpoint Reduction in Hypertension): NCT00338260.

### BP Response Phenotype

Multiple measures of BP response were available for each study and the most accurate BP measurement available for each study was used to determine the phenotype. PEAR‐2[19](#jah34354-bib-0019) used home BP measurements, whereas GENRES,[24](#jah34354-bib-0024) BB‐SS, LIFE‐Fin,[20](#jah34354-bib-0020) INVEST,[21](#jah34354-bib-0021) ASCOT‐UK, and ASCOT‐SC used office BP measurements to define response to β_1_‐blockers. PEAR used a composite weighted average of BP response from office, home, and ambulatory day and nighttime values.[25](#jah34354-bib-0025)

For all studies, using the above‐mentioned BP measurements, β_1_‐blocker BP responses (diastolic and systolic BP response) were calculated as the difference between the start to the end of the β_1_‐blocker treatment.

### Genotyping and Imputation

For each study, participants were genotyped on either an Illumina or an Affymetrix panel. Each study underwent standard quality control (QC) protocols for their genotype data to obtain high‐quality single nucleotide polymorphisms (SNPs) for their study, which were then imputed to the 1000G phase 3 reference panel for a total number of ≈47 million SNPs.

### Statistical Analysis

Continuous variables are presented as mean±SD and categorical variables are presented as numbers and percentages.

We performed a 2‐stage genome‐wide meta‐analysis. The first stage comprised of a discovery GWAS meta‐analysis, which was followed by testing the top signals in a meta‐analysis replication cohort. The second stage was a combined meta‐analysis of the findings from both discovery and replication meta‐analysis. The detailed flowchart outlining the steps in the study is presented in Figure [1](#jah34354-fig-0001).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/36a5/6759913/4af11e6c1b04/JAH3-8-e013115-g001.jpg)

Overall design framework for the discovery and replication meta‐analysis. ASCOT‐SC indicates Anglo‐Scandinavian Cardiac Outcomes Trial‐Scandinavian; ASCOT‐UK, Anglo‐Scandinavian Cardiac Outcomes Trial‐United Kingdom; BB‐SS, Pharmacogenomics of Beta‐Blockers Sardinian Study; BP, blood pressure; EAF, effect allele frequency; GENRES, Genetics of Drug Responsiveness in Essential Hypertension; GWAS, genome‐wide association study; INVEST, International Verapamil SR‐Trandolapril Study; LIFE‐Fin, Finnish arm of the Losartan Intervention For Endpoint Reduction in Hypertension Study; METAL, Meta‐Analysis Helper; PEAR, Pharmacogenomic Evaluation of Antihypertensive Responses; QC, quality control; Rsq, imputation quality; SNPs, single nucleotide polymorphisms.

#### GWAS Per Study

A total of 1254 individuals from 5 cohorts (PEAR, PEAR‐2, LIFE‐Fin, GENRES, and BB‐SS) were included as part of the discovery GWAS meta‐analysis for BP response to β_1_‐blocker treatment. First, a GWAS for imputed SNPs was performed separately in each study for association with SBP and DBP response using linear regression adjusting for age, sex, pretreatment BP, and principal components for ancestry. Regression analysis was conducted using either EPACTS or SNPTEST. Each study filtered the tested SNPs at a minor allele frequency cutoff of >1% and imputation QC (Rsq) of >0.30.

#### QC Per Study

Results from each study further underwent stringent standard QC procedures using the EasyQC package as per protocol described by Winkler et al,[26](#jah34354-bib-0026) to account for heterozygosity, missingness, and allele mismatches. Quantile‐quantile plots, P‐Z test, and allele frequency mismatch plots were generated using EasyQC.

#### Discovery GWAS Meta‐Analysis

After QC, the GWAS results from each cohort were combined in a meta‐analysis using the inverse variance method with fixed effect model performed using Meta‐Analysis Helper (METAL).[27](#jah34354-bib-0027) A genomic inflation (λ) correction was applied if genomic inflation was deemed to exist (λ >1) for any of the studies. As mentioned previously, each of the studies in the discovery meta‐analysis GWAS were imputed to the same reference panel. However, each of the GWAS results from the study was subjected to stringent QC and a different number of SNPs were removed from each study based on the QC criteria. However, for the purpose of discovery meta‐analysis we focused only on the 6596214 SNPs that were present in all 5 studies. After meta‐analysis, we applied a stringent prioritization criterion to select SNPs from the discovery meta‐analysis and focused only on SNPs with the same direction of association across all 5 studies in the discovery meta‐analysis and deemed SNPs that met the conventionally acceptable *P*<5×10^−8^ as genome‐wide significant. The suggestive level of association was set at a threshold of *P*=1×10^−4^. SNPs that met the suggestive level of association with a concordant direction of association across all 5 studies were further linkage disequilibrium (LD) pruned at *r* ^2^>0.2 using LD Link,[28](#jah34354-bib-0028) to obtain single independent SNPs representative of each associated locus that were tested for replication. The number of independent signals were used to calculate a Bonferroni‐corrected α level (0.05/# of independent signals), to determine whether SNPs were replicated for SBP and DBP responses in the replication cohort.

#### Replication GWAS Meta‐Analysis

A total of 1592 individuals from 3 study cohorts (INVEST, ASCOT‐UK, and ASCOT‐SC) were included in the replication meta‐analysis. Since replication cohorts did not have a hypertensive medication washout before starting the β_1_‐Blocker therapy, the analysis was adjusted for a dummy variable indicative of patient treatment status (yes=1, no=0), if patients were or were not treated, respectively, with any antihypertensive medication at the start of their β_1_‐blocker therapy. Summary statistics for SNPs meeting the suggestive level of significance from the discovery GWAS meta‐analysis were looked up in each study of the defined replication cohorts, which were then combined in meta‐analysis to form the replication meta‐analysis cohort. SNPs were deemed replicated if they met the Bonferroni‐corrected *P* value for association with SBP (3.4×10^−4^) and DBP (4.3×10^−4^) response, with concordant direction of association as in the discovery meta‐analysis.

#### Combined Discovery and Replication Meta‐Analysis

Summary statistics of discovery and replication GWAS and meta‐analyses were further combined in a meta‐analysis. Genome‐wide significance for this combined meta‐analysis was set at *P=*5×10^−8.^

#### Secondary Validation

The SBP and DBP association results for black participants with hypertension from PEAR‐2 and PEAR were combined in a meta‐analysis and used for ethnic validation of SNPs that were successfully replicated in an INVEST/ASCOT‐UK/ASCOT‐SC replication cohort. Given the differences in LD between the genome of people with white and African ancestry, we did not expect the same SNPs to be necessarily validated. Hence, we scanned the genetic region surrounding the index SNP (±500 kb) for any SNPs that were associated with the SBP or DBP response in the same direction as that of the index SNP at a Bonferroni‐corrected *P* value.

### In Silico Analysis

In an attempt to identify the putative functional SNP, all SNPs in high LD (*r* ^2^>0.8) with the replicated SNP (rs28404156 of *BST1* locus) based on the 1000G phase 3 reference panel were extracted using the rAggar.[29](#jah34354-bib-0029) RegulomeDB[30](#jah34354-bib-0030) was used to annotate and score these SNPs to identify potentially regulatory SNPs in LD with the index SNP using a stringent score of ≤4.[30](#jah34354-bib-0030) The expression Quantitative Trait Locus (eQTL) of all of these SNPs in LD with the replicated SNPs was evaluated using HaploReg v4.1 databases.[31](#jah34354-bib-0031)

## Results

Clinical characteristics of all 8 participating cohorts in the β_1_‐blocker discovery and replication meta‐analysis are summarized in Table [1](#jah34354-tbl-0001). All participants were of European ancestry. Men and women were similarly represented in all of the studies, with the exceptions of GENRES, which was composed of men only, and ASCOT, which was composed of predominantly men. The clinical characteristics of the black patients from PEAR‐2 and PEAR treated with a β_1_‐blocker, and included in the secondary validation, are summarized in Table [S1](#jah34354-sup-0001).

### Table 1.

|   | Discovery Meta‐Analysis |  |  |  |  | Replication Meta‐Analysis |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| PEAR | PEAR‐2 | GENRES | LIFE‐Fin | BB‐SS | ASCOT‐UK | ASCOT‐SC | INVEST |  |
| No. | 233 | 201 | 216 | 202 | 403 | 745 | 773 | 74 |
| β‐Blocker | Atenolol | Metoprolol | Bisoprolol | Atenolol | Atenolol | Atenolol | Atenolol | Atenolol |
| Treatment duration | 8 wk | 8 wk | 4 wk | 61 d (average) | 4 wk | 6 wk | 6 wk | 6 wk |
| Age, y | 49±9.52 | 51±8.99 | 50.5±6.3 | 64.0±6.5 | 51.7±11.4 | 63.1±8.3 | 60.8±8.9 | 68.35±9.21 |
| Women, No. (%) | 109 (46.78) | 65 (48.14) | 0 | 97 (48.0) | 214 (53.1) | 114 (15.2) | 177 (22.9) | 41 (54.66) |
| Body mass index, kg/m2 | 30.30±4.90 | 30.66±4.95 | 26.7±2.7 | 27.1±3.6 | 26.9±4.08 | 28.4±4.6 | 28.4±4.2 | 29 ± 6 |
| Baseline SBP, mm Hg | 145.46±9.68 | 147.49±10.83 | 151.9±13.1 | 166.4±13.2 | 159.3±15.3 | 159.7±14.9 | 164.8±16.5 | 146.41±15.43 |
| Baseline DBP, mm Hg | 93.20±5.54 | 93.94±5.63 | 100.0±7.2 | 97.6±6.4 | 102.5±10.7 | 92.6±9.1 | 96.3±9.8 | 84.01±10.21 |
| Post‐treatment SBP, mm Hg | 136.11±11.46 | 137.32±12.82 | 138.9±15.2 | 145.1±13.1 | 136.3±16.5 | 148.8±20.7 | 156.4±22.1 | 136.82±15.48 |
| Post‐treatment DBP, mm Hg | 86.05±7.75 | 84.90±7.45 | 90.2±8.7 | 84.8±6.4 | 84.9±10.9 | 83.9±10.1 | 89.6±10.5 | 77.53±8.63 |
| ∆SBP, mm Hg | −12.67±8.61 | −10.19±9.20 | −13.0±10.5 | −21.3±12.8 | −23.0±17.7 | −10.8±19.7 | −8.4±20.7 | −9.59±16.96 |
| ∆DBP, mm Hg | −10.50±5.77 | −9.05±6.07 | −9.8±6.9 | −12.8±6.6 | −17.7±11.2 | −8.5±10.6 | −6.7±10.4 | −6.47±10.84 |

Table 1 Caption: Clinical Characteristics of all White Participants for the 8 Studies in the Discovery and Replication Meta‐Analysis

### Discovery GWAS Meta‐Analysis

GWAS discovery meta‐analysis for SBP and DBP responses to β_1_‐blockers was performed using 5 cohorts (PEAR, PEAR‐2, LIFE‐Fin, GENRES, and BB‐SS). Manhattan and QQ plots for association are shown in Figure [2](#jah34354-fig-0002). The QQ plots for both SBP and DBP do not exhibit any inflation indicative of population substructure beyond that expected by chance alone. None of the SNPs reached genome‐wide significance for either SBP or DBP response. However, 1207 SNPs for SBP and 842 SNPs for DBP met the suggestive level of significance (*P*<1×10^−4^). With LD pruning (*r* ^2^>0.2), a total of 280 and 218 independent signals associated with SBP and DBP, respectively, were identified and tested for replication.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/36a5/6759913/7e4c95ccb8fc/JAH3-8-e013115-g002.jpg)

Manhattan (A and B) and quantile‐quantile (C and D) plots from meta‐analysis of genome‐wide association for systolic (A and C) and diastolic (B and D) blood pressure responses to β1‐blocker treatment. The blue line refers to the suggestive level of significance (P<1×10−4). The red line refers to the genome‐wide significance (P=5×10−8).

### Replication GWAS Meta‐Analysis

For association with SBP response, the SNP rs28404156 was successfully replicated with a Bonferroni‐corrected *P* value and was associated in the same direction as in the discovery meta‐analysis. (Discovery: *P*=9.33×10^−5^ and β=−3.21±0.82; replication: *P*=1.85×10^−4^ and β=−4.86±1.3) (Table [2](#jah34354-tbl-0002)). rs28404156 (G‐>A) is a missense variant that causes an arginine to histidine (Arg153His) amino acid change in BST1 and is an eQTL[31](#jah34354-bib-0031) for *FAM200B*(Artery_Tibial). The regional plot for rs28404516 for the white population is presented in Figure [3](#jah34354-fig-0003)A.

### Table 2.

| SNP | CHR | BP | EA | OA | Gene/Region | Discovery Meta‐Analysis |  |  |  | Replication Meta‐Analysis |  |  |  | Combined Meta‐Analysis |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| EAF | Effect | SE | P Value | EAF | Effect | SE | P Value | Effect | SE | P Value |  |  |  |  |  |  |
| rs28404156 | 4 | 1573732 | A | G | BST1 | 0.080 | −3.21 | 0.82 | 9.33×10−05 | 0.072 | −4.86 | 1.30 | 1.8×10−4 | −3.60 | 0.69 | 2.18×10−07 |

Table 2 Caption: Discovery Meta‐Analysis Association Result and Replication Meta‐Analysis and Combined Meta‐Analysis of Discovery and Replication Meta‐Analysis

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/36a5/6759913/7cc58f88bbee/JAH3-8-e013115-g003.jpg)

Regional plot of the BST1 locus in white (A) and blacks (B).

The least square adjusted means for SBP response by genotype for rs28404156 was plotted for the studies that were part of the discovery meta‐analysis (PEAR, PEAR‐2, LIFE‐Fin, GENRES, and BB‐SS) as these were the only studies for which this analysis was available. Given the low minor allele frequency of rs28404156 (0.08), only heterozygotes but no homozygotes for the minor allele were available in all of the discovery studies except BB‐SS. Among the 5 studies, the A allele carriers had better SBP response compared with the noncarriers (Figure [S1](#jah34354-sup-0001)).

In contrast, for association with DBP responses, none of the SNPs that were tested met the Bonferroni‐corrected *P* value for replication.

### Combined Discovery and Replication Meta‐Analysis

None of the SNPs met genome‐wide significance in the combined discovery‐replication meta‐analysis and the association of the replicated SNP rs28404156 with SBP response reached a meta‐analysis *P* value of 1.15×10^−7^.

### Ethnic Validation

The black cohort from PEAR and PEAR‐2 were used for ethnic validation of the replicated SNP rs28404156. rs28404156 was not significantly associated in the black cohort. However, considering the region (±500 kb) surrounding the SNP rs28404156, another SNP, rs193131744, was found to be significantly associated with reduction in SBP in response to β_1_‐blocker treatment with a concordant direction as that of discovery meta‐analysis (discovery: *P*=2.64×10^−6^ and β=−16.87) (Figure [3](#jah34354-fig-0003)B).

### In Silico Analysis

A total of 12 SNPs in high LD (*r* ^2^>0.8) with the replicated SNP (rs28404156) were extracted using rAggar. Of these, the functional prediction performed using regulome DB identified 3 variants that met the stringent cutoff threshold of <4. We also investigated the eQTL traits of these 12 SNPs using HaploReg v4.1, and 10 of these variants were also eQTLs for either *BST1* or other nearby genes of this locus such as *CD38*,* FBXL5*, and *FAM200B*. The details of the LD (*r* ^2^ and D’), eQTLs and the correlated genes and tissues, the RegulomeDB scores, and the potential regulatory functions have been outlined in Table [S2](#jah34354-sup-0001). There were 3 variants that were positively predicted by both tools used (RegulomeDB and Haplroreg v4.1), further strengthening their putative roles as functional/regulatory variants (Table [S2](#jah34354-sup-0001)).

## Discussion

To our knowledge, this is the largest genome‐wide meta‐analysis for antihypertensive response to β_1_‐blocker treatment, consisting of 5 randomized controlled trials for discovery and 3 randomized clinical trials for replication, followed by ethnic validation of the replicated SNPs among a cohort of black patients from 2 randomized clinical trials. We successfully identified and replicated a missense variant, rs28404156 in the *BST1* locus, which was significantly associated with SBP response to β_1_‐blockers. The carriers of the A allele of this SNP had a significantly better SBP response in both discovery and the replication meta‐analysis. Given LD differences across race groups, even though the exact SNP did not replicate in the black cohort, we were able to validate the genomic region by identifying other SNPs that were significantly associated with BP response to β_1_‐blockers. This further points towards the potential importance of this locus in β_1_‐blocker BP responsiveness.

The SNP rs28404156 is present in the *BST1* gene and represents a missense SNP along with being an eQTL for *FAM200B*. This SNP results in an arginine to histidine substitution. Arginines are usually involved in formation of salt bridges needed for protein stability and, accordingly, its replacement could have implication on the BST1 protein function. *BST1*, also known as CD157, encodes for ADP‐ribosyl cyclase 2, which is a cell surface receptor expressed in bone marrow stromal cells and is primarily involved in regulation of pre–B‐cell growth. It is now known that *BST1* is also expressed in multiple other cell types including vascular endothelial stem cells, monocytes, and mast cells.[32](#jah34354-bib-0032) Although no direct association between *BST1* and BP regulation has been reported, there are various pathways and signaling components that *BST1* may interact with to influence the pathophysiology of hypertension. For example, *BST1/CD157* influences the phosphorylation of focal adhesion kinase[33](#jah34354-bib-0033) and calcium homeostasis[34](#jah34354-bib-0034) which are known to be involved in the hypertension‐vascular signaling cascade. *BST1/CD157* is also involved in various immune and inflammation responses, including regulation of motility and transendothelial migration of monocytes and their adhesion to extracellular matrix proteins.[32](#jah34354-bib-0032) The fact that *BST1* is expressed in a larger number of tissue subtypes was only recently discovered, implying that the interaction of *BST1* with these signaling cascades with respect to hypertension may warrant further study.

This replicated SNP is located on chromosome 4p15.32 on the distal part of the small segment of chromosome 4. Multiple previous studies have identified and reported the distal region of chromosome 4p to be involved with BP regulation, which further highlights the importance of this locus in BP response.[35](#jah34354-bib-0035), [36](#jah34354-bib-0036) There are several genes present in this locus, with variants in high LD with our successfully replicated index SNP (±500 kb) such as *CD38, FAM200B, FBXL5, FGFBP1*, and *FGFBP2*. Many of these genes have been implicated in various physiological and pathophysiological components of hypertension signaling. To further investigate the plausible involvement of this locus in BP response, we used in silico approaches and identified several SNPs in high LD with the replicated SNP in *BST1* as well as in other genes in this genomic locus (±500 kb). Further investigation using in silico analysis revealed that several of these SNPs were predicted to have regulatory function and are eQTLs for the genes in the same locus.

Some of these SNPs with potential regulatory function were found to be in high LD with our index replicated SNP and were present in *CD38*. Interestingly, *CD38* is a transmembrane protein that shares 33% homology with *BST1* and has both ADP‐ribosyl cyclase and cyclic adenosine 5′‐diphosphate ribose hydrolase activities.[37](#jah34354-bib-0037) *CD38* has been shown to be involved in multiple key processes such as vascular contraction, apoptosis, and neural signaling via its involvement in calcium regulation, which plays a key role in hypertension as well as cardiovascular diseases, renal regulation, and energy metabolism.[38](#jah34354-bib-0038) Studies have also shown that elimination of *CD38* prevents mice from gaining weight and becoming obese while on a high‐fat diet via a SIRT‐dependent mechanism.[39](#jah34354-bib-0039) Some recent human studies have found a similar association between *CD38* and metabolic syndrome in which high BP is an important factor.[40](#jah34354-bib-0040) Of note, one study has shown that *CD38*‐deficient mice have increased plasma renin activity.[41](#jah34354-bib-0041) This could have implications for β_1_‐blocker BP response given that higher plasma renin activity has been reported to be associated with better BP responses to β_1_‐blocker monotherapy as well as to various add‐on therapies.[42](#jah34354-bib-0042) Several studies have reported the direct activation of *CD38* by angiotensin II (Ang II), endothelin‐1, and norepinephrine, as well as indirect activation via the generation of reactive oxygen species.[43](#jah34354-bib-0043), [44](#jah34354-bib-0044) Upon activation, *CD38* generates the endogenous calcium secondary messengers, ie, cyclic adenosine 5′‐diphosphate ribose and nicotinic acid adenosine dinucleotide phosphate, which modulate the intracellular calcium transients resulting in vasoconstriction.[38](#jah34354-bib-0038), [45](#jah34354-bib-0045) These studies indicate that *CD38* plays an important role in the modulation of agonist‐induced vascular smooth muscle cell responses. These responses might have an important implication in the regulation of BP by β_1_‐blockers and need further elucidation.

*FBXL5* is another gene harboring SNPs in LD with the index SNPs. *FBXL5* belongs to the F‐box family of proteins and encodes for F‐box and leucine‐rich repeat protein 5. It acts as an iron sensor and has been reported to be a tumor suppressor.[46](#jah34354-bib-0046) One of the meta‐analyses of data from 6 global whole blood gene expression studies for BP and hypertension found *FBXL5* to be one of the differentially expressed genes and to be significantly associated with DBP.[47](#jah34354-bib-0047) The index SNP as well as several other SNPs in LD with the index SNP were also eQTLs for *FAM200B*. However, not much information is available in the literature regarding *FAM200B* in relation to the BP phenotype or related pathways.

*FGFBP1* is among other genes in this associated locus. Even though we could not find SNPs in *FGFBP1* that were in LD with the rs28404156 SNP, the importance of this gene in hypertension and BP response cannot be overlooked. *FGFBP1* encodes for fibroblast growth factor (FGF)–binding protein 1 (FGFBP1) and is involved in the modulation of the FGF signaling. FGFs are a family of proteins, many of which have been implicated in pathways related to hypertension.[48](#jah34354-bib-0048), [49](#jah34354-bib-0049) Specifically, *FGFBP1* is a precursor protein that releases FGF1 from the extracellular matrix and has been investigated for its role in modulating BP. Cuevas et al[50](#jah34354-bib-0050), [51](#jah34354-bib-0051) reported a decrease in the endothelial FGF in the blood vessels of spontaneously hypertensive rats and further showed that systemic administration of FGF improves the impaired endothelial cells and nitric oxide synthase resulting in improvement in hypertension. Similarly, studies in hypertensive rats have deemed the *FGFBP1* locus to be important in the development of glomerular damage and hypertension.[52](#jah34354-bib-0052) Further, a recent family‐based study from Poland reported a novel association between variants in the *FGFBP1* and familial hypertension. This report also showed an increased expression of the FGFBP1 in the kidneys of these patients with hypertension.[51](#jah34354-bib-0051), [53](#jah34354-bib-0053) More recently, studies in transgenic mice have shown that increased expression of FGFBP1 resulted in an increase of >30 mm Hg in arterial pressure by increasing the sensitivity of the blood vessels to Ang II.[54](#jah34354-bib-0054) The investigators of this study postulated that targeted inhibition of the FGF signaling using FGF inhibitors, which are conventionally used for cancer treatment, might help in controlling hypertension as well. They tested this theory by using an FGF inhibitor in transgenic mice overexpressing FGFBP1 and found that Ang II sensitivity of the vasculature was reversed, resulting in decreased BP.[54](#jah34354-bib-0054) Given the fact that β_1_‐blockers are known to suppress the renin system and Ang II levels in the plasma,[55](#jah34354-bib-0055) it is important to consider the cross‐talk between β_1_‐blockers, Ang II, and *FGFBP1*. Additional investigations to better understand this cross‐talk is needed to further elucidate the possible contribution of FGFBP1 to β_1_‐blocker BP response.

## Study Strengths and Limitations

Overall, this study has certain noteworthy strengths. To our knowledge, this is the largest pharmacogenomic GWAS assessing the underlying genetic variants contributing to variable BP response. Through our discovery, replication, and ethnic validation efforts we were able to identify and thoroughly validate the association of the *BST1*‐containing locus with β_1_‐blocker BP response. Although various genes in this locus have been previously implicated in BP‐related signaling, this is the first time a direct association with response to β_1_‐blocker treatment has been identified. Furthermore, our criteria for selecting markers from the discovery meta‐analysis for testing in replication were stringent, which ensured the consistency of direction across the data sets for the identified association.

We acknowledge the limitations of our study as well. While this is considered the largest pharmacogenomics analysis of β_1_‐blockers response including 8 cohorts (5 discovery and 3 replication) and consisted of >2000 participants, our results did not identify genome‐wide significance, which may be partly attributed to reduced power driven by variants with low‐modest effect size. Additionally, subtle heterogeneity in BP response phenotype across the different studies may have led to failure to identify genome‐wide significant signals. Nevertheless, the association of SNPs in the replication cohort at a Bonferroni‐corrected *P* value, as well as ethnic validation of the region, suggest the importance of the genomic locus. Another limitation is the use of different β_1_‐blockers across studies included in this meta‐analysis. Although the majority of the studies used atenolol in their design, GENRES and PEAR‐2 used bisoprolol and metoprolol, respectively. Bisoprolol was used at 5 mg/d, which is equivalent to 50 mg/d as that of atenolol[56](#jah34354-bib-0056) and is the dose that was used in the majority of the atenolol studies in this article, except for PEAR, in which the target atenolol dose was 100 mg/d upon titration. Likewise, for metoprolol, the patients were started on 50 mg/d with uptitration to 100 mg twice per day in PEAR‐2; in this case, the equivalent atenolol doses are ≈100 mg once per day.[57](#jah34354-bib-0057) Hormones are known to play a role in modulating BP, and hormone replacement therapy can interfere with BP levels. Another limitation of this study is that exclusion of women taking any kind of hormone replacement therapy was not consistently across all of the studies that are part of this meta‐analysis. While some studies excluded women taking oral contraceptives or hormone replacement therapy, others did not. This precludes the ability to include hormone replacement therapy, which may influence BP, as a covariate in our association analysis. However, using our stringent thresholds and multistage GWAS efforts, we were able to identify a strong association with multiple levels of evidence. This article leverages the power of collaborative efforts within ICAPS to investigate genetic determinants of BP response, and showcases potential opportunities to investigate other important related phenotypes such as heart rate reduction response to β‐blockers in future

## Conclusions

We performed the largest genome‐wide meta‐analysis of BP response to β_1_‐blockers and successfully identified and replicated an SNP in the BST1 locus that was significantly associated with BP response to β_1_‐blockers. We were further able to gather additional evidence in support of the region containing this SNP using in silico approaches, and further identified SNPs in high LD with the index SNP with putative regulatory and functional effects. These data collectively suggest the importance of this region in mediating BP response to β_1_‐blockers, warranting further studies to understand the exact underlying mechanism(s). Deeper insights about such pharmacogenomic markers and loci can ultimately pave the way for personalized β_1_‐blocker use with optimum BP control and outcomes.

## Sources of Funding

Both PEAR‐2 and PEAR studies were supported by the National Institutes of Health (NIH) Pharmacogenomics Research Network grant U01‐GM074492 and the National Center for Advancing Translational Sciences. Award numbers UL1 TR000064 (University of Florida); UL1 TR000454 (Emory University), and UL1 TR000135 (Mayo Clinic). PEAR was also supported by funds from the Mayo Foundation. LIFE‐Fin and GENRES were supported by the Sigrid Juselius Foundation, The Finnish Foundation for Cardiovascular Research, and the Helsinki University Central Hospital. BB‐SS studies were supported by the HYPERGENES project (FP7‐HEALTHF4‐2007‐201550), InterOmics (PB05 MIUR‐CNR Italian Flagship Project), and the ‘Associazione per lo sviluppo della ricerca sull'ipertensione arteriosa e sulle malattie cardiovascolari—ONLUS’. ASCOT was funded by the National Institutes for Health Research (NIHR) as part of the portfolio of translational research of the NIHR Barts Biomedical Research Unit and the NIHR Biomedical Research Centre at Imperial College, the International Centre for Circulatory Health Charity, and the Medical Research Council through G952010. INVEST was funded by NIH grants HL074730, HL69758, and RR017568; a grant from Abbott Pharmaceuticals; and an American Heart Association Postdoctoral Fellowship 0625619B.

## Disclosures

None.

## Supporting information

## Acknowledgments

We would like to thank the participants, staff, and study physicians of the PEAR, PEAR‐2, LIFE‐Fin, GENRES, BB‐SS INVEST, and ASCOT studies for their contributions.

(J Am Heart Assoc. 2019;8:e013115 DOI: 10.1161/JAHA.119.013115.)

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Mills KT, Bundy JD, Kelly TN, Reed JE, Kearney PM, Reynolds K, Chen J, He J. Global disparities of hypertension prevalence and control: a systematic analysis of population‐based studies from 90 countries. Circulation. 2016;134:441–450.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.115.018912) | [PMC free article](/articles/PMC4979614/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27502908/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Global%20disparities%20of%20hypertension%20prevalence%20and%20control:%20a%20systematic%20analysis%20of%20population%E2%80%90based%20studies%20from%2090%20countries&author=KT%20Mills&author=JD%20Bundy&author=TN%20Kelly&author=JE%20Reed&author=PM%20Kearney&volume=134&publication_year=2016&pages=441-450&pmid=27502908&doi=10.1161/CIRCULATIONAHA.115.018912&)

2. DePalma SM, Himmelfarb CD, MacLaughlin EJ, Taler SJ. Hypertension guideline update: a new guideline for a new era. JAAPA. 2018;31:16–22.  [DOI](https://doi.org/10.1097/01.JAA.0000533656.93911.38) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29762202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAAPA&title=Hypertension%20guideline%20update:%20a%20new%20guideline%20for%20a%20new%20era&author=SM%20DePalma&author=CD%20Himmelfarb&author=EJ%20MacLaughlin&author=SJ%20Taler&volume=31&publication_year=2018&pages=16-22&pmid=29762202&doi=10.1097/01.JAA.0000533656.93911.38&)

3. Kearney PM, Whelton M, Reynolds K, Muntner P, Whelton PK, He J. Global burden of hypertension: analysis of worldwide data. Lancet. 2005;365:217–223.  [DOI](https://doi.org/10.1016/S0140-6736(05)17741-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15652604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Global%20burden%20of%20hypertension:%20analysis%20of%20worldwide%20data&author=PM%20Kearney&author=M%20Whelton&author=K%20Reynolds&author=P%20Muntner&author=PK%20Whelton&volume=365&publication_year=2005&pages=217-223&pmid=15652604&doi=10.1016/S0140-6736(05)17741-1&)

4. Campbell NR, Khalsa T, Lackland DT, Niebylski ML, Nilsson PM, Redburn KA, Orias M, Zhang XH, Burrell L, Horiuchi M, Poulter NR, Prabhakaran D, Ramirez AJ, Schiffrin EL, Touyz RM, Wang JG, Weber MA; World Stroke Organization; International Diabetes Federation; International Council of Cardiovascular Prevention and Rehabilitation; International Society of Nephrology . High Blood Pressure 2016: Why Prevention and Control Are Urgent and Important. The World Hypertension League, International Society of Hypertension, World Stroke Organization, International Diabetes Foundation, International Council of Cardiovascular Prevention and Rehabilitation, International Society of Nephrology. J Clin Hypertens (Greenwich). 2016;18:714–717.  [DOI](https://doi.org/10.1111/jch.12840) | [PMC free article](/articles/PMC8031792/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27316336/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Hypertens%20(Greenwich)&title=High%20Blood%20Pressure%202016:%20Why%20Prevention%20and%20Control%20Are%20Urgent%20and%20Important.%20The%20World%20Hypertension%20League,%20International%20Society%20of%20Hypertension,%20World%20Stroke%20Organization,%20International%20Diabetes%20Foundation,%20International%20Council%20of%20Cardiovascular%20Prevention%20and%20Rehabilitation,%20International%20Society%20of%20Nephrology&author=NR%20Campbell&author=T%20Khalsa&author=DT%20Lackland&author=ML%20Niebylski&author=PM%20Nilsson&volume=18&publication_year=2016&pages=714-717&pmid=27316336&doi=10.1111/jch.12840&)

5. Wiysonge CS, Bradley HA, Volmink J, Mayosi BM. Cochrane corner: beta‐blockers for hypertension. Heart. 2018;104:282–283.  [DOI](https://doi.org/10.1136/heartjnl-2017-311585) | [PMC free article](/articles/PMC5861391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28756403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heart&title=Cochrane%20corner:%20beta%E2%80%90blockers%20for%20hypertension&author=CS%20Wiysonge&author=HA%20Bradley&author=J%20Volmink&author=BM%20Mayosi&volume=104&publication_year=2018&pages=282-283&pmid=28756403&doi=10.1136/heartjnl-2017-311585&)

6. James PA, Oparil S, Carter BL, Cushman WC, Dennison‐Himmelfarb C, Handler J, Lackland DT, LeFevre ML, MacKenzie TD, Ogedegbe O, Smith SC, Svetkey LP, Taler SJ, Townsend RR, Wright JT, Narva AS, Ortiz E. 2014 evidence‐based guideline for the management of high blood pressure in adults: report from the panel members appointed to the Eighth Joint National Committee (JNC 8). JAMA. 2014;311:507–520.  [DOI](https://doi.org/10.1001/jama.2013.284427) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24352797/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=2014%20evidence%E2%80%90based%20guideline%20for%20the%20management%20of%20high%20blood%20pressure%20in%20adults:%20report%20from%20the%20panel%20members%20appointed%20to%20the%20Eighth%20Joint%20National%20Committee%20(JNC%208)&author=PA%20James&author=S%20Oparil&author=BL%20Carter&author=WC%20Cushman&author=C%20Dennison%E2%80%90Himmelfarb&volume=311&publication_year=2014&pages=507-520&pmid=24352797&doi=10.1001/jama.2013.284427&)

7. Krause T, Lovibond K, Caulfield M, McCormack T, Williams B; Guideline Development Group . Management of hypertension: summary of NICE guidance. BMJ. 2011;343:d4891.  [DOI](https://doi.org/10.1136/bmj.d4891) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21868454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Management%20of%20hypertension:%20summary%20of%20NICE%20guidance&author=T%20Krause&author=K%20Lovibond&author=M%20Caulfield&author=T%20McCormack&author=B%20Williams&volume=343&publication_year=2011&pages=d4891&pmid=21868454&doi=10.1136/bmj.d4891&)

8. Jarari N, Rao N, Peela JR, Ellafi KA, Shakila S, Said AR, Nelapalli NK, Min Y, Tun KD, Jamallulail SI, Rawal AK, Ramanujam R, Yedla RN, Kandregula DK, Argi A, Peela LT. A review on prescribing patterns of antihypertensive drugs. Clin Hypertens. 2015;22:7.  [DOI](https://doi.org/10.1186/s40885-016-0042-0) | [PMC free article](/articles/PMC4808570/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27019747/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Hypertens&title=A%20review%20on%20prescribing%20patterns%20of%20antihypertensive%20drugs&author=N%20Jarari&author=N%20Rao&author=JR%20Peela&author=KA%20Ellafi&author=S%20Shakila&volume=22&publication_year=2015&pages=7&pmid=27019747&doi=10.1186/s40885-016-0042-0&)

9. Shah SJ, Stafford RS. Current trends of hypertension treatment in the United States. Am J Hypertens. 2017;30:1008–1014.  [DOI](https://doi.org/10.1093/ajh/hpx085) | [PMC free article](/articles/PMC6887988/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28531239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hypertens&title=Current%20trends%20of%20hypertension%20treatment%20in%20the%20United%20States&author=SJ%20Shah&author=RS%20Stafford&volume=30&publication_year=2017&pages=1008-1014&pmid=28531239&doi=10.1093/ajh/hpx085&)

10. Jamerson K, DeQuattro V. The impact of ethnicity on response to antihypertensive therapy. Am J Med. 1996;101:22S–32S.  [DOI](https://doi.org/10.1016/s0002-9343(96)00265-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8876472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med&title=The%20impact%20of%20ethnicity%20on%20response%20to%20antihypertensive%20therapy&author=K%20Jamerson&author=V%20DeQuattro&volume=101&publication_year=1996&pages=22S-32S&pmid=8876472&doi=10.1016/s0002-9343(96)00265-3&)

11. Materson BJ. Variability in response to antihypertensive drugs. Am J Med. 2007;120:S10–S20.  [DOI](https://doi.org/10.1016/j.amjmed.2007.02.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17403377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med&title=Variability%20in%20response%20to%20antihypertensive%20drugs&author=BJ%20Materson&volume=120&publication_year=2007&pages=S10-S20&pmid=17403377&doi=10.1016/j.amjmed.2007.02.003&)

12. Gong Y, McDonough CW, Beitelshees AL, El Rouby N, Hiltunen TP, O'Connell JR, Padmanabhan S, Langaee TY, Hall K, Schmidt SO, Curry RW, Gums JG, Donner KM, Kontula KK, Bailey KR, Boerwinkle E, Takahashi A, Tanaka T, Kubo M, Chapman AB, Turner ST, Pepine CJ, Cooper‐DeHoff RM, Johnson JA. PTPRD gene associated with blood pressure response to atenolol and resistant hypertension. J Hypertens. 2015;33:2278–2285.  [DOI](https://doi.org/10.1097/HJH.0000000000000714) | [PMC free article](/articles/PMC4788379/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26425837/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hypertens&title=PTPRD%20gene%20associated%20with%20blood%20pressure%20response%20to%20atenolol%20and%20resistant%20hypertension&author=Y%20Gong&author=CW%20McDonough&author=AL%20Beitelshees&author=N%20El%20Rouby&author=TP%20Hiltunen&volume=33&publication_year=2015&pages=2278-2285&pmid=26425837&doi=10.1097/HJH.0000000000000714&)

13. Hiltunen TP, Donner KM, Sarin AP, Saarela J, Ripatti S, Chapman AB, Gums JG, Gong Y, Cooper‐DeHoff RM, Frau F, Glorioso V, Zaninello R, Salvi E, Glorioso N, Boerwinkle E, Turner ST, Johnson JA, Kontula KK. Pharmacogenomics of hypertension: a genome‐wide, placebo‐controlled cross‐over study, using four classes of antihypertensive drugs. J Am Heart Assoc. 2015;4:e001521 DOI: 10.1161/JAHA.115.001778  [DOI](https://doi.org/10.1161/JAHA.115.001778) | [PMC free article](/articles/PMC4330076/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25622599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Heart%20Assoc&title=Pharmacogenomics%20of%20hypertension:%20a%20genome%E2%80%90wide,%20placebo%E2%80%90controlled%20cross%E2%80%90over%20study,%20using%20four%20classes%20of%20antihypertensive%20drugs&author=TP%20Hiltunen&author=KM%20Donner&author=AP%20Sarin&author=J%20Saarela&author=S%20Ripatti&volume=4&publication_year=2015&pages=e001521&pmid=25622599&doi=10.1161/JAHA.115.001778&)

14. Gong Y, Wang Z, Beitelshees AL, McDonough CW, Langaee TY, Hall K, Schmidt SO, Curry RW, Gums JG, Bailey KR, Boerwinkle E, Chapman AB, Turner ST, Cooper‐DeHoff RM, Johnson JA. Pharmacogenomic genome‐wide meta‐analysis of blood pressure response to β‐blockers in hypertensive African Americans. Hypertension. 2016;67:556–563.  [DOI](https://doi.org/10.1161/HYPERTENSIONAHA.115.06345) | [PMC free article](/articles/PMC4752391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26729753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Pharmacogenomic%20genome%E2%80%90wide%20meta%E2%80%90analysis%20of%20blood%20pressure%20response%20to%20%CE%B2%E2%80%90blockers%20in%20hypertensive%20African%20Americans&author=Y%20Gong&author=Z%20Wang&author=AL%20Beitelshees&author=CW%20McDonough&author=TY%20Langaee&volume=67&publication_year=2016&pages=556-563&pmid=26729753&doi=10.1161/HYPERTENSIONAHA.115.06345&)

15. Singh S, El Rouby N, McDonough CW, Gong Y, Bailey KR, Boerwinkle E, Chapman AB, Gums JG, Turner ST, Cooper‐DeHoff RM, Johnson JA. Genomic association analysis reveals variants associated with blood pressure response to beta‐blockers in European Americans. Clin Transl Sci. 2019; DOI: 10.1111/cts.12643.  [DOI](https://doi.org/10.1111/cts.12643) | [PMC free article](/articles/PMC6742943/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31033190/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Genomic%20association%20analysis%20reveals%20variants%20associated%20with%20blood%20pressure%20response%20to%20beta%E2%80%90blockers%20in%20European%20Americans&author=S%20Singh&author=N%20El%20Rouby&author=CW%20McDonough&author=Y%20Gong&author=KR%20Bailey&publication_year=2019&pmid=31033190&doi=10.1111/cts.12643&)

16. Taylor J. 2013 ESH/ESC guidelines for the management of arterial hypertension. Eur Heart J. 2013;34:2108–2109.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/24040671/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Heart%20J&title=2013%20ESH/ESC%20guidelines%20for%20the%20management%20of%20arterial%20hypertension&author=J%20Taylor&volume=34&publication_year=2013&pages=2108-2109&pmid=24040671&)

17. Prichard BN, Cruickshank JM, Graham BR. Beta‐adrenergic blocking drugs in the treatment of hypertension. Blood Press. 2001;10:366–386.  [DOI](https://doi.org/10.1080/080370501753400665) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11822540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Press&title=Beta%E2%80%90adrenergic%20blocking%20drugs%20in%20the%20treatment%20of%20hypertension&author=BN%20Prichard&author=JM%20Cruickshank&author=BR%20Graham&volume=10&publication_year=2001&pages=366-386&pmid=11822540&doi=10.1080/080370501753400665&)

18. Johnson JA, Boerwinkle E, Zineh I, Chapman AB, Bailey K, Cooper‐DeHoff RM, Gums J, Curry RW, Gong Y, Beitelshees AL, Schwartz G, Turner ST. Pharmacogenomics of antihypertensive drugs: rationale and design of the Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR) study. Am Heart J. 2009;157:442–449.  [DOI](https://doi.org/10.1016/j.ahj.2008.11.018) | [PMC free article](/articles/PMC2671287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19249413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20Heart%20J&title=Pharmacogenomics%20of%20antihypertensive%20drugs:%20rationale%20and%20design%20of%20the%20Pharmacogenomic%20Evaluation%20of%20Antihypertensive%20Responses%20(PEAR)%20study&author=JA%20Johnson&author=E%20Boerwinkle&author=I%20Zineh&author=AB%20Chapman&author=K%20Bailey&volume=157&publication_year=2009&pages=442-449&pmid=19249413&doi=10.1016/j.ahj.2008.11.018&)

19. Hamadeh IS, Langaee TY, Dwivedi R, Garcia S, Burkley BM, Skaar TC, Chapman AB, Gums JG, Turner ST, Gong Y, Cooper‐DeHoff RM, Johnson JA. Impact of CYP2D6 polymorphisms on clinical efficacy and tolerability of metoprolol tartrate. Clin Pharmacol Ther. 2014;96:175–181.  [DOI](https://doi.org/10.1038/clpt.2014.62) | [PMC free article](/articles/PMC4111800/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24637943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Impact%20of%20CYP2D6%20polymorphisms%20on%20clinical%20efficacy%20and%20tolerability%20of%20metoprolol%20tartrate&author=IS%20Hamadeh&author=TY%20Langaee&author=R%20Dwivedi&author=S%20Garcia&author=BM%20Burkley&volume=96&publication_year=2014&pages=175-181&pmid=24637943&doi=10.1038/clpt.2014.62&)

20. Rimpelä JM, Kontula KK, Fyhrquist F, Donner KM, Tuiskula AM, Sarin AP, Mohney RP, Stirdivant SM, Hiltunen TP. Replicated evidence for aminoacylase 3 and nephrin gene variations to predict antihypertensive drug responses. Pharmacogenomics. 2017;18:445–458.  [DOI](https://doi.org/10.2217/pgs-2016-0204) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28353407/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Replicated%20evidence%20for%20aminoacylase%203%20and%20nephrin%20gene%20variations%20to%20predict%20antihypertensive%20drug%20responses&author=JM%20Rimpel%C3%A4&author=KK%20Kontula&author=F%20Fyhrquist&author=KM%20Donner&author=AM%20Tuiskula&volume=18&publication_year=2017&pages=445-458&pmid=28353407&doi=10.2217/pgs-2016-0204&)

21. Pepine CJ, Handberg‐Thurmond E, Marks RG, Conlon M, Cooper‐DeHoff R, Volkers P, Zellig P. Rationale and design of the International Verapamil SR/Trandolapril Study (INVEST): an Internet‐based randomized trial in coronary artery disease patients with hypertension. J Am Coll Cardiol. 1998;32:1228–1237.  [DOI](https://doi.org/10.1016/s0735-1097(98)00423-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9809930/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol&title=Rationale%20and%20design%20of%20the%20International%20Verapamil%20SR/Trandolapril%20Study%20(INVEST):%20an%20Internet%E2%80%90based%20randomized%20trial%20in%20coronary%20artery%20disease%20patients%20with%20hypertension&author=CJ%20Pepine&author=E%20Handberg%E2%80%90Thurmond&author=RG%20Marks&author=M%20Conlon&author=R%20Cooper%E2%80%90DeHoff&volume=32&publication_year=1998&pages=1228-1237&pmid=9809930&doi=10.1016/s0735-1097(98)00423-9&)

22. Sever PS, Dahlöf B, Poulter NR, Wedel H, Beevers G, Caulfield M, Collins R, Kjeldsen SE, McInnes GT, Mehlsen J, Nieminen M, O'Brien E, Ostergren J; ASCOT Steering Committee, Anglo‐Scandinavian Cardiac Outcomes Trial: a brief history, rationale and outline protocol. J Hum Hypertens. 2001;15(suppl 1):S11–S12.  [DOI](https://doi.org/10.1038/sj.jhh.1001212) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11685901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Hypertens&title=Anglo%E2%80%90Scandinavian%20Cardiac%20Outcomes%20Trial:%20a%20brief%20history,%20rationale%20and%20outline%20protocol&author=PS%20Sever&author=B%20Dahl%C3%B6f&author=NR%20Poulter&author=H%20Wedel&author=G%20Beevers&volume=15&issue=suppl%201&publication_year=2001&pages=S11-S12&pmid=11685901&doi=10.1038/sj.jhh.1001212&)

23. Johnson JA. Pharmacogenomic Evaluation of Antihypertensive Responses (PEAR). NCBI's Database of Genotypes and Phenotypes: dbGaP. Accession phs000649.v1.p1. 2016.

24. Hiltunen TP, Suonsyrjä T, Hannila‐Handelberg T, Paavonen KJ, Miettinen HE, Strandberg T, Tikkanen I, Tilvis R, Pentikäinen PJ, Virolainen J, Kontula K. Predictors of antihypertensive drug responses: initial data from a placebo‐controlled, randomized, cross‐over study with four antihypertensive drugs (The GENRES Study). Am J Hypertens. 2007;20:311–318.  [DOI](https://doi.org/10.1016/j.amjhyper.2006.09.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17324745/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hypertens&title=Predictors%20of%20antihypertensive%20drug%20responses:%20initial%20data%20from%20a%20placebo%E2%80%90controlled,%20randomized,%20cross%E2%80%90over%20study%20with%20four%20antihypertensive%20drugs%20(The%20GENRES%20Study)&author=TP%20Hiltunen&author=T%20Suonsyrj%C3%A4&author=T%20Hannila%E2%80%90Handelberg&author=KJ%20Paavonen&author=HE%20Miettinen&volume=20&publication_year=2007&pages=311-318&pmid=17324745&doi=10.1016/j.amjhyper.2006.09.006&)

25. Turner ST, Schwartz GL, Chapman AB, Beitelshees AL, Gums JG, Cooper‐Dehoff RM, Boerwinkle E, Johnson JA, Bailey KR. Power to identify a genetic predictor of antihypertensive drug response using different methods to measure blood pressure response. J Transl Med. 2012;10:47.  [DOI](https://doi.org/10.1186/1479-5876-10-47) | [PMC free article](/articles/PMC3342146/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22413836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Transl%20Med&title=Power%20to%20identify%20a%20genetic%20predictor%20of%20antihypertensive%20drug%20response%20using%20different%20methods%20to%20measure%20blood%20pressure%20response&author=ST%20Turner&author=GL%20Schwartz&author=AB%20Chapman&author=AL%20Beitelshees&author=JG%20Gums&volume=10&publication_year=2012&pages=47&pmid=22413836&doi=10.1186/1479-5876-10-47&)

26. Winkler TW, Day FR, Croteau‐Chonka DC, Wood AR, Locke AE, Mägi R, Ferreira T, Fall T, Graff M, Justice AE, Luan J, Gustafsson S, Randall JC, Vedantam S, Workalemahu T, Kilpeläinen TO, Scherag A, Esko T, Kutalik Z, Heid IM, Loos RJ; Genetic Investigation of Anthropometric Traits (GIANT) Consortium . Quality control and conduct of genome‐wide association meta‐analyses. Nat Protoc. 2014;9:1192–1212.  [DOI](https://doi.org/10.1038/nprot.2014.071) | [PMC free article](/articles/PMC4083217/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24762786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Quality%20control%20and%20conduct%20of%20genome%E2%80%90wide%20association%20meta%E2%80%90analyses&author=TW%20Winkler&author=FR%20Day&author=DC%20Croteau%E2%80%90Chonka&author=AR%20Wood&author=AE%20Locke&volume=9&publication_year=2014&pages=1192-1212&pmid=24762786&doi=10.1038/nprot.2014.071&)

27. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta‐analysis of genomewide association scans. Bioinformatics. 2010;26:2190–2191.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta%E2%80%90analysis%20of%20genomewide%20association%20scans&author=CJ%20Willer&author=Y%20Li&author=GR%20Abecasis&volume=26&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

28. Machiela MJ, Chanock SJ. LDlink: a web‐based application for exploring population‐specific haplotype structure and linking correlated alleles of possible functional variants. Bioinformatics. 2015;31:3555–3557.  [DOI](https://doi.org/10.1093/bioinformatics/btv402) | [PMC free article](/articles/PMC4626747/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26139635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LDlink:%20a%20web%E2%80%90based%20application%20for%20exploring%20population%E2%80%90specific%20haplotype%20structure%20and%20linking%20correlated%20alleles%20of%20possible%20functional%20variants&author=MJ%20Machiela&author=SJ%20Chanock&volume=31&publication_year=2015&pages=3555-3557&pmid=26139635&doi=10.1093/bioinformatics/btv402&)

29. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics. 2005;21:263–265.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

30. Boyle AP, Hong EL, Hariharan M, Cheng Y, Schaub MA, Kasowski M, Karczewski KJ, Park J, Hitz BC, Weng S, Cherry JM, Snyder M. Annotation of functional variation in personal genomes using RegulomeDB. Genome Res. 2012;22:1790–1797.  [DOI](https://doi.org/10.1101/gr.137323.112) | [PMC free article](/articles/PMC3431494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Annotation%20of%20functional%20variation%20in%20personal%20genomes%20using%20RegulomeDB&author=AP%20Boyle&author=EL%20Hong&author=M%20Hariharan&author=Y%20Cheng&author=MA%20Schaub&volume=22&publication_year=2012&pages=1790-1797&pmid=22955989&doi=10.1101/gr.137323.112&)

31. Ward LD, Kellis M. HaploReg v4: systematic mining of putative causal variants, cell types, regulators and target genes for human complex traits and disease. Nucleic Acids Res. 2016;44:D877–D881.  [DOI](https://doi.org/10.1093/nar/gkv1340) | [PMC free article](/articles/PMC4702929/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26657631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=HaploReg%20v4:%20systematic%20mining%20of%20putative%20causal%20variants,%20cell%20types,%20regulators%20and%20target%20genes%20for%20human%20complex%20traits%20and%20disease&author=LD%20Ward&author=M%20Kellis&volume=44&publication_year=2016&pages=D877-D881&pmid=26657631&doi=10.1093/nar/gkv1340&)

32. Ortolan E, Augeri S, Fissolo G, Musso I, Funaro A. CD157: from immunoregulatory protein to potential therapeutic target. Immunol Lett. 2019;205:59–64.  [DOI](https://doi.org/10.1016/j.imlet.2018.06.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29936181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunol%20Lett&title=CD157:%20from%20immunoregulatory%20protein%20to%20potential%20therapeutic%20target&author=E%20Ortolan&author=S%20Augeri&author=G%20Fissolo&author=I%20Musso&author=A%20Funaro&volume=205&publication_year=2019&pages=59-64&pmid=29936181&doi=10.1016/j.imlet.2018.06.007&)

33. Okuyama Y, Ishihara K, Kimura N, Hirata Y, Sato K, Itoh M, Ok LB, Hirano T. Human BST‐1 expressed on myeloid cells functions as a receptor molecule. Biochem Biophys Res Commun. 1996;228:838–845.  [DOI](https://doi.org/10.1006/bbrc.1996.1741) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8941363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Human%20BST%E2%80%901%20expressed%20on%20myeloid%20cells%20functions%20as%20a%20receptor%20molecule&author=Y%20Okuyama&author=K%20Ishihara&author=N%20Kimura&author=Y%20Hirata&author=K%20Sato&volume=228&publication_year=1996&pages=838-845&pmid=8941363&doi=10.1006/bbrc.1996.1741&)

34. Funaro A, Ortolan E, Ferranti B, Gargiulo L, Notaro R, Luzzatto L, Malavasi F. CD157 is an important mediator of neutrophil adhesion and migration. Blood. 2004;104:4269–4278.  [DOI](https://doi.org/10.1182/blood-2004-06-2129) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15328157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=CD157%20is%20an%20important%20mediator%20of%20neutrophil%20adhesion%20and%20migration&author=A%20Funaro&author=E%20Ortolan&author=B%20Ferranti&author=L%20Gargiulo&author=R%20Notaro&volume=104&publication_year=2004&pages=4269-4278&pmid=15328157&doi=10.1182/blood-2004-06-2129&)

35. Chen W, Li S, Srinivasan SR, Boerwinkle E, Berenson GS. Autosomal genome scan for loci linked to blood pressure levels and trends since childhood: the Bogalusa Heart Study. Hypertension. 2005;45:954–959.  [DOI](https://doi.org/10.1161/01.HYP.0000161881.02361.11) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15809362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Autosomal%20genome%20scan%20for%20loci%20linked%20to%20blood%20pressure%20levels%20and%20trends%20since%20childhood:%20the%20Bogalusa%20Heart%20Study&author=W%20Chen&author=S%20Li&author=SR%20Srinivasan&author=E%20Boerwinkle&author=GS%20Berenson&volume=45&publication_year=2005&pages=954-959&pmid=15809362&doi=10.1161/01.HYP.0000161881.02361.11&)

36. Allayee H, de Bruin TW, Michelle Dominguez K, Cheng LS, Ipp E, Cantor RM, Krass KL, Keulen ET, Aouizerat BE, Lusis AJ, Rotter JI. Genome scan for blood pressure in Dutch dyslipidemic families reveals linkage to a locus on chromosome 4p. Hypertension. 2001;38:773–778.  [DOI](https://doi.org/10.1161/hy1001.092617) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11641285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Genome%20scan%20for%20blood%20pressure%20in%20Dutch%20dyslipidemic%20families%20reveals%20linkage%20to%20a%20locus%20on%20chromosome%204p&author=H%20Allayee&author=TW%20de%20Bruin&author=K%20Michelle%20Dominguez&author=LS%20Cheng&author=E%20Ipp&volume=38&publication_year=2001&pages=773-778&pmid=11641285&doi=10.1161/hy1001.092617&)

37. Deaglio S, Malavasi F. The CD38/CD157 mammalian gene family: an evolutionary paradigm for other leukocyte surface enzymes. Purinergic Signal. 2006;2:431–441.  [DOI](https://doi.org/10.1007/s11302-006-9002-6) | [PMC free article](/articles/PMC2096639/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18404481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Purinergic%20Signal&title=The%20CD38/CD157%20mammalian%20gene%20family:%20an%20evolutionary%20paradigm%20for%20other%20leukocyte%20surface%20enzymes&author=S%20Deaglio&author=F%20Malavasi&volume=2&publication_year=2006&pages=431-441&pmid=18404481&doi=10.1007/s11302-006-9002-6&)

38. Chini EN. CD38 as a regulator of cellular NAD: a novel potential pharmacological target for metabolic conditions. Curr Pharm Des. 2009;15:57–63.  [DOI](https://doi.org/10.2174/138161209787185788) | [PMC free article](/articles/PMC2883294/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19149603/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Pharm%20Des&title=CD38%20as%20a%20regulator%20of%20cellular%20NAD:%20a%20novel%20potential%20pharmacological%20target%20for%20metabolic%20conditions&author=EN%20Chini&volume=15&publication_year=2009&pages=57-63&pmid=19149603&doi=10.2174/138161209787185788&)

39. Wang LF, Miao LJ, Wang XN, Huang CC, Qian YS, Huang X, Wang XL, Jin WZ, Ji GJ, Fu M, Deng KY, Xin HB. CD38 deficiency suppresses adipogenesis and lipogenesis in adipose tissues through activating Sirt1/PPARγ signaling pathway. J Cell Mol Med. 2018;22:101–110.  [DOI](https://doi.org/10.1111/jcmm.13297) | [PMC free article](/articles/PMC5742727/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28816006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Mol%20Med&title=CD38%20deficiency%20suppresses%20adipogenesis%20and%20lipogenesis%20in%20adipose%20tissues%20through%20activating%20Sirt1/PPAR%CE%B3%20signaling%20pathway&author=LF%20Wang&author=LJ%20Miao&author=XN%20Wang&author=CC%20Huang&author=YS%20Qian&volume=22&publication_year=2018&pages=101-110&pmid=28816006&doi=10.1111/jcmm.13297&)

40. Chiang SH, Harrington WW, Luo G, Milliken NO, Ulrich JC, Chen J, Rajpal DK, Qian Y, Carpenter T, Murray R, Geske RS, Stimpson SA, Kramer HF, Haffner CD, Becherer JD, Preugschat F, Billin AN. Genetic ablation of CD38 protects against western diet‐induced exercise intolerance and metabolic inflexibility. PLoS One. 2015;10:e0134927.  [DOI](https://doi.org/10.1371/journal.pone.0134927) | [PMC free article](/articles/PMC4546114/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26287487/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Genetic%20ablation%20of%20CD38%20protects%20against%20western%20diet%E2%80%90induced%20exercise%20intolerance%20and%20metabolic%20inflexibility&author=SH%20Chiang&author=WW%20Harrington&author=G%20Luo&author=NO%20Milliken&author=JC%20Ulrich&volume=10&publication_year=2015&pages=e0134927&pmid=26287487&doi=10.1371/journal.pone.0134927&)

41. Xiong J, Xia M, Yi F, Abais JM, Li N, Boini KM, Li PL. Regulation of renin release via cyclic ADP‐ribose‐mediated signaling: evidence from mice lacking CD38 gene. Cell Physiol Biochem. 2013;31:44–55.  [DOI](https://doi.org/10.1159/000343348) | [PMC free article](/articles/PMC3753399/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23343681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Physiol%20Biochem&title=Regulation%20of%20renin%20release%20via%20cyclic%20ADP%E2%80%90ribose%E2%80%90mediated%20signaling:%20evidence%20from%20mice%20lacking%20CD38%20gene&author=J%20Xiong&author=M%20Xia&author=F%20Yi&author=JM%20Abais&author=N%20Li&volume=31&publication_year=2013&pages=44-55&pmid=23343681&doi=10.1159/000343348&)

42. Turner ST, Schwartz GL, Chapman AB, Beitelshees AL, Gums JG, Cooper‐DeHoff RM, Boerwinkle E, Johnson JA, Bailey KR. Plasma renin activity predicts blood pressure responses to beta‐blocker and thiazide diuretic as monotherapy and add‐on therapy for hypertension. Am J Hypertens. 2010;23:1014–1022.  [DOI](https://doi.org/10.1038/ajh.2010.98) | [PMC free article](/articles/PMC2941699/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20725057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hypertens&title=Plasma%20renin%20activity%20predicts%20blood%20pressure%20responses%20to%20beta%E2%80%90blocker%20and%20thiazide%20diuretic%20as%20monotherapy%20and%20add%E2%80%90on%20therapy%20for%20hypertension&author=ST%20Turner&author=GL%20Schwartz&author=AB%20Chapman&author=AL%20Beitelshees&author=JG%20Gums&volume=23&publication_year=2010&pages=1014-1022&pmid=20725057&doi=10.1038/ajh.2010.98&)

43. Thai TL, Churchill GC, Arendshorst WJ. NAADP receptors mediate calcium signaling stimulated by endothelin‐1 and norepinephrine in renal afferent arterioles. Am J Physiol Renal Physiol. 2009;297:F510–F516.  [DOI](https://doi.org/10.1152/ajprenal.00116.2009) | [PMC free article](/articles/PMC2724244/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19439521/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol%20Renal%20Physiol&title=NAADP%20receptors%20mediate%20calcium%20signaling%20stimulated%20by%20endothelin%E2%80%901%20and%20norepinephrine%20in%20renal%20afferent%20arterioles&author=TL%20Thai&author=GC%20Churchill&author=WJ%20Arendshorst&volume=297&publication_year=2009&pages=F510-F516&pmid=19439521&doi=10.1152/ajprenal.00116.2009&)

44. Kim SY, Gul R, Rah SY, Kim SH, Park SK, Im MJ, Kwon HJ, Kim UH. Molecular mechanism of ADP‐ribosyl cyclase activation in angiotensin II signaling in murine mesangial cells. Am J Physiol Renal Physiol. 2008;294:F982–F989.  [DOI](https://doi.org/10.1152/ajprenal.00483.2007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18272599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol%20Renal%20Physiol&title=Molecular%20mechanism%20of%20ADP%E2%80%90ribosyl%20cyclase%20activation%20in%20angiotensin%20II%20signaling%20in%20murine%20mesangial%20cells&author=SY%20Kim&author=R%20Gul&author=SY%20Rah&author=SH%20Kim&author=SK%20Park&volume=294&publication_year=2008&pages=F982-F989&pmid=18272599&doi=10.1152/ajprenal.00483.2007&)

45. Lee S, Paudel O, Jiang Y, Yang XR, Sham JS. CD38 mediates angiotensin II‐induced intracellular Ca(2+) release in rat pulmonary arterial smooth muscle cells. Am J Respir Cell Mol Biol. 2015;52:332–341.  [DOI](https://doi.org/10.1165/rcmb.2014-0141OC) | [PMC free article](/articles/PMC4370261/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25078456/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Cell%20Mol%20Biol&title=CD38%20mediates%20angiotensin%20II%E2%80%90induced%20intracellular%20Ca(2+)%20release%20in%20rat%20pulmonary%20arterial%20smooth%20muscle%20cells&author=S%20Lee&author=O%20Paudel&author=Y%20Jiang&author=XR%20Yang&author=JS%20Sham&volume=52&publication_year=2015&pages=332-341&pmid=25078456&doi=10.1165/rcmb.2014-0141OC&)

46. Yao H, Su S, Xia D, Wang M, Li Z, Chen W, Ren L, Xu L. F‐box and leucine‐rich repeat protein 5 promotes colon cancer progression by modulating PTEN/PI3K/AKT signaling pathway. Biomed Pharmacother. 2018;107:1712–1719.  [DOI](https://doi.org/10.1016/j.biopha.2018.08.119) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30257389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Pharmacother&title=F%E2%80%90box%20and%20leucine%E2%80%90rich%20repeat%20protein%205%20promotes%20colon%20cancer%20progression%20by%20modulating%20PTEN/PI3K/AKT%20signaling%20pathway&author=H%20Yao&author=S%20Su&author=D%20Xia&author=M%20Wang&author=Z%20Li&volume=107&publication_year=2018&pages=1712-1719&pmid=30257389&doi=10.1016/j.biopha.2018.08.119&)

47. Huan T, Esko T, Peters MJ, Pilling LC, Schramm K, Schurmann C, Chen BH, Liu C, Joehanes R, Johnson AD, Yao C, Ying SX, Courchesne P, Milani L, Raghavachari N, Wang R, Liu P, Reinmaa E, Dehghan A, Hofman A, Uitterlinden AG, Hernandez DG, Bandinelli S, Singleton A, Melzer D, Metspalu A, Carstensen M, Grallert H, Herder C, Meitinger T, Peters A, Roden M, Waldenberger M, Dörr M, Felix SB, Zeller T, Vasan R, O'Donnell CJ, Munson PJ, Yang X, Prokisch H, Völker U, van Meurs JB, Ferrucci L, Levy D; International Consortium for Blood Pressure GWAS (ICBP) . A meta‐analysis of gene expression signatures of blood pressure, hypertension. PLoS Genet. 2015;11:e1005035.  [DOI](https://doi.org/10.1371/journal.pgen.1005035) | [PMC free article](/articles/PMC4365001/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25785607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20meta%E2%80%90analysis%20of%20gene%20expression%20signatures%20of%20blood%20pressure,%20hypertension&author=T%20Huan&author=T%20Esko&author=MJ%20Peters&author=LC%20Pilling&author=K%20Schramm&volume=11&publication_year=2015&pages=e1005035&pmid=25785607&doi=10.1371/journal.pgen.1005035&)

48. Ornitz DM, Itoh N. The Fibroblast Growth Factor signaling pathway. Wiley Interdiscip Rev Dev Biol. 2015;4:215–266.  [DOI](https://doi.org/10.1002/wdev.176) | [PMC free article](/articles/PMC4393358/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25772309/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Wiley%20Interdiscip%20Rev%20Dev%20Biol&title=The%20Fibroblast%20Growth%20Factor%20signaling%20pathway&author=DM%20Ornitz&author=N%20Itoh&volume=4&publication_year=2015&pages=215-266&pmid=25772309&doi=10.1002/wdev.176&)

49. Ren Y, Jiao X, Zhang L. Expression level of fibroblast growth factor 5 (FGF5) in the peripheral blood of primary hypertension and its clinical significance. Saudi J Biol Sci. 2018;25:469–473.  [DOI](https://doi.org/10.1016/j.sjbs.2017.11.043) | [PMC free article](/articles/PMC5911643/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29692649/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Saudi%20J%20Biol%20Sci&title=Expression%20level%20of%20fibroblast%20growth%20factor%205%20(FGF5)%20in%20the%20peripheral%20blood%20of%20primary%20hypertension%20and%20its%20clinical%20significance&author=Y%20Ren&author=X%20Jiao&author=L%20Zhang&volume=25&publication_year=2018&pages=469-473&pmid=29692649&doi=10.1016/j.sjbs.2017.11.043&)

50. Cuevas P, Carceller F, Ortega S, Zazo M, Nieto I, Giménez‐Gallego G. Hypotensive activity of fibroblast growth factor. Science. 1991;254:1208–1210.  [DOI](https://doi.org/10.1126/science.1957172) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1957172/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Hypotensive%20activity%20of%20fibroblast%20growth%20factor&author=P%20Cuevas&author=F%20Carceller&author=S%20Ortega&author=M%20Zazo&author=I%20Nieto&volume=254&publication_year=1991&pages=1208-1210&pmid=1957172&doi=10.1126/science.1957172&)

51. Cuevas P, García‐Calvo M, Carceller F, Reimers D, Zazo M, Cuevas B, Muñoz‐Willery I, Martínez‐Coso V, Lamas S, Giménez‐Gallego G. Correction of hypertension by normalization of endothelial levels of fibroblast growth factor and nitric oxide synthase in spontaneously hypertensive rats. Proc Natl Acad Sci USA. 1996;93:11996–12001.  [DOI](https://doi.org/10.1073/pnas.93.21.11996) | [PMC free article](/articles/PMC38172/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8876251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Correction%20of%20hypertension%20by%20normalization%20of%20endothelial%20levels%20of%20fibroblast%20growth%20factor%20and%20nitric%20oxide%20synthase%20in%20spontaneously%20hypertensive%20rats&author=P%20Cuevas&author=M%20Garc%C3%ADa%E2%80%90Calvo&author=F%20Carceller&author=D%20Reimers&author=M%20Zazo&volume=93&publication_year=1996&pages=11996-12001&pmid=8876251&doi=10.1073/pnas.93.21.11996&)

52. Braun MC, Herring SM, Gokul N, Monita M, Bell R, Hicks MJ, Wenderfer SE, Doris PA. Hypertensive renal disease: susceptibility and resistance in inbred hypertensive rat lines. J Hypertens. 2013;31:2050–2059.  [DOI](https://doi.org/10.1097/HJH.0b013e328362f9a5) | [PMC free article](/articles/PMC4130350/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24107734/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hypertens&title=Hypertensive%20renal%20disease:%20susceptibility%20and%20resistance%20in%20inbred%20hypertensive%20rat%20lines&author=MC%20Braun&author=SM%20Herring&author=N%20Gokul&author=M%20Monita&author=R%20Bell&volume=31&publication_year=2013&pages=2050-2059&pmid=24107734&doi=10.1097/HJH.0b013e328362f9a5&)

53. Tomaszewski M, Charchar FJ, Nelson CP, Barnes T, Denniff M, Kaiser M, Debiec R, Christofidou P, Rafelt S, van der Harst P, Wang WY, Maric C, Zukowska‐Szczechowska E, Samani NJ. Pathway analysis shows association between FGFBP1 and hypertension. J Am Soc Nephrol. 2011;22:947–955.  [DOI](https://doi.org/10.1681/ASN.2010080829) | [PMC free article](/articles/PMC3083316/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21436287/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Soc%20Nephrol&title=Pathway%20analysis%20shows%20association%20between%20FGFBP1%20and%20hypertension&author=M%20Tomaszewski&author=FJ%20Charchar&author=CP%20Nelson&author=T%20Barnes&author=M%20Denniff&volume=22&publication_year=2011&pages=947-955&pmid=21436287&doi=10.1681/ASN.2010080829&)

54. Tassi E, Lai EY, Li L, Solis G, Chen Y, Kietzman WE, Ray PE, Riegel AT, Welch WJ, Wilcox CS, Wellstein A. Blood pressure control by a secreted FGFBP1 (Fibroblast Growth Factor‐Binding Protein). Hypertension. 2018;71:160–167.  [DOI](https://doi.org/10.1161/HYPERTENSIONAHA.117.10268) | [PMC free article](/articles/PMC5730494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29158353/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Blood%20pressure%20control%20by%20a%20secreted%20FGFBP1%20(Fibroblast%20Growth%20Factor%E2%80%90Binding%20Protein)&author=E%20Tassi&author=EY%20Lai&author=L%20Li&author=G%20Solis&author=Y%20Chen&volume=71&publication_year=2018&pages=160-167&pmid=29158353&doi=10.1161/HYPERTENSIONAHA.117.10268&)

55. Blumenfeld JD, Sealey JE, Mann SJ, Bragat A, Marion R, Pecker MS, Sotelo J, August P, Pickering TG, Laragh JH. Beta‐adrenergic receptor blockade as a therapeutic approach for suppressing the renin‐angiotensin‐aldosterone system in normotensive and hypertensive subjects. Am J Hypertens. 1999;12:451–459.  [DOI](https://doi.org/10.1016/s0895-7061(99)00005-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10342782/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hypertens&title=Beta%E2%80%90adrenergic%20receptor%20blockade%20as%20a%20therapeutic%20approach%20for%20suppressing%20the%20renin%E2%80%90angiotensin%E2%80%90aldosterone%20system%20in%20normotensive%20and%20hypertensive%20subjects&author=JD%20Blumenfeld&author=JE%20Sealey&author=SJ%20Mann&author=A%20Bragat&author=R%20Marion&volume=12&publication_year=1999&pages=451-459&pmid=10342782&doi=10.1016/s0895-7061(99)00005-9&)

56. Park S, Rhee MY, Lee SY, Park SW, Jeon D, Kim BW, Kwan J, Choi D. A prospective, randomized, open‐label, active‐controlled, clinical trial to assess central haemodynamic effects of bisoprolol and atenolol in hypertensive patients. J Hypertens. 2013;31:813–819.  [DOI](https://doi.org/10.1097/HJH.0b013e32835e8f5b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23385648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hypertens&title=A%20prospective,%20randomized,%20open%E2%80%90label,%20active%E2%80%90controlled,%20clinical%20trial%20to%20assess%20central%20haemodynamic%20effects%20of%20bisoprolol%20and%20atenolol%20in%20hypertensive%20patients&author=S%20Park&author=MY%20Rhee&author=SY%20Lee&author=SW%20Park&author=D%20Jeon&volume=31&publication_year=2013&pages=813-819&pmid=23385648&doi=10.1097/HJH.0b013e32835e8f5b&)

57. Harry JD, Cruickshank JM, Young J, Barker N. Relative activities of atenolol and metoprolol on the cardiovascular system of man. Eur J Clin Pharmacol. 1981;20:9–15.  [DOI](https://doi.org/10.1007/BF00554660) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7308277/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Relative%20activities%20of%20atenolol%20and%20metoprolol%20on%20the%20cardiovascular%20system%20of%20man&author=JD%20Harry&author=JM%20Cruickshank&author=J%20Young&author=N%20Barker&volume=20&publication_year=1981&pages=9-15&pmid=7308277&doi=10.1007/BF00554660&)
