# Pharmacogenetic analysis of belimumab fails to identify robust genetic predictors of efficacy in lupus

## Metadata
**Authors:** Pamela L St Jean, David A Roth, Linda C McCarthy, Arlene R Hughes
**Journal:** Pharmacogenetics and Genomics
**Date:** 2019 Jul 4
**DOI:** [10.1097/FPC.0000000000000378](https://doi.org/10.1097/FPC.0000000000000378)
**PMID:** 31058715
**PMCID:** PMC6613715
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6613715/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6613715/pdf/fpc-29-132.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6613715/pdf/fpc-29-132.pdf)

## Abstract

Supplemental Digital Content is available in the text.

Keywords: belimumab, BENLYSTA, efficacy, lupus, pharmacogenetics, systemic lupus erythematosus

## Introduction

Systemic lupus erythematosus (SLE or lupus) is a chronic autoimmune disorder characterized by autoantibody production and abnormal B-lymphocyte function [1](#R1). Genetics, sex hormones, and environmental conditions are believed to contribute to lupus [1](#R1)–[3](#R3). Lupus is more prevalent in women (∼90% of patients) than men [4](#R4) and the incidence and prevalence vary across racial populations. The lowest prevalence is observed in Europeans and the highest prevalence in African populations. African and South Asian populations, respectively, have a 5–9- and 1.2–6-fold increased incidence and 2–3- and 2–2.4-fold increased prevalence compared with Europeans [5](#R5). Lupus manifestations vary among patients and the American College of Rheumatology criteria for lupus diagnosis requires that four of 11 criteria be met.

BENLYSTA (belimumab) is a B-lymphocyte stimulator (BLyS)/B-cell activating factor (BAFF)-specific inhibitor indicated for the treatment of adult patients with active, auto-antibody-positive SLE who receive standard therapy. Belimumab does not directly bind B cells but, by binding BLyS, inhibits the survival of B cells, including autoreactive B cells, and reduces their differentiation into immunoglobulin-producing plasma cells. An intravenous formulation of belimumab is approved in over 65 countries; as of January 2018, a subcutaneous formulation is approved in the USA, EU, Japan, and Canada and has been submitted in other countries.

Precision medicine, which includes the integration of clinical, genomic, and molecular profiling into medical practice, aims to increase the cost-effectiveness of healthcare and improve patient outcomes. The presence of antinuclear autoantibodies and low complement C3 and C4 levels are established clinical criteria that differentiate lupus from other autoimmune diseases. We conducted pharmacogenetic (PGx) analyses to determine whether genetic variants identify lupus patients who preferentially respond to belimumab. This paper describes an exploratory belimumab PGx efficacy analysis, identification of two significantly associated and completely correlated genetic variants in the *ANO3* (anoctamin 3) gene region, and an unsuccessful attempt to confirm the association in an independent study.

## Patients and methods

Patients in the GSK phase 3 Benlysta (belimumab; GlaxoSmithKline, Brentford, UK) clinical program met American College of Rheumatology criteria and entered the studies on stable lupus therapy. The primary efficacy end point, the SLE Response Index (SRI4) integrates three validated lupus instruments measuring disease activity (SELENA SLEDAI), organ worsening (British Isles Lupus Assessment Group or BILAG) and overall patient condition (Physician’s Global Assessment) [6](#R6). SRI4 defines efficacy as at least a four-point reduction in the SELENA SLEDAI disease activity rating, no new BILAG A organ domain or two new BILAG B organ domain scores, and no worsening (<0.30 increase) in the Physician’s Global Assessment compared with baseline. In accordance with the Declaration of Helsinki [7](#R7) and following ethics committee approval, randomized patients were invited to participate in genetic research. Participation was optional and required written informed consent and collection of a 6 ml blood (DNA) sample. Patients were permitted to withdraw consent at any time. The composition of the analysis populations is summarized in Table [1](#T1). Genotyping and imputation details for the exploratory analyses, including Manhattan, Q–Q, and principal component analysis plots are summarized in the Supplementary Figs 1–3, respectively (Supplemental digital content 1, [http://links.lww.com/FPC/B352](http://links.lww.com/FPC/B352)).

### Table 1.

Table 1 Caption: Composition of the PGx analysis populations and SRI4 response rates

[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6613715_fpc-29-132-g001.jpg)

### Analysis

#### Exploratory analysis

The genetic analysis of belimumab efficacy was conducted using the 10 mg/kg intravenous [BEL110751 ([https://clinicaltrials.gov](https://clinicaltrials.gov), [NCT00410384](https://clinicaltrials.gov/ct2/show/NCT00410384)) and BEL110752 ([https://clinicaltrials.gov](https://clinicaltrials.gov), [NCT00424476](https://clinicaltrials.gov/ct2/show/NCT00424476))] belimumab treatment groups and 200 mg subcutaneous [BEL112341 ([https://clinicaltrials.gov](https://clinicaltrials.gov), [NCT01484496](https://clinicaltrials.gov/ct2/show/NCT01484496))] belimumab treatment group. The 1 mg/kg intravenous arm was not included as this dose did not meet the primary efficacy objective in one phase 3 intravenous study. Because two genotyping platforms were used, analyses were conducted separately in the combined BEL110751 and BEL110752 studies and the BEL112341 study to control for technical batch effects. A meta-analysis was then conducted using effect estimates from the two aforementioned analyses. Five efficacy end points were evaluated as described on the GSK Clinical Study Register ([https://www.gsk-clinicalstudyregister.com/study/204901](https://www.gsk-clinicalstudyregister.com/study/204901)). This paper focuses on the primary SRI4 end point that had the strongest statistical association. The association of genotype with the binary SRI4 efficacy outcome was assessed in the combined BEL110751 and BEL110752 analysis and BEL112341 by logistic regression, assuming an additive genetic model and adjusting for 10 genetic ancestry principal components, baseline SELENA SLEDAI scores and baseline C3 and C4 complement levels. Genomic control adjustments corrected for test statistic inflation. Fifteen candidate variants in nine genes were included on the basis of functional significance or suggestive trends from previous exploratory PGx investigations. Additional variants in these genes were included in the GWAS. Genome-wide variants with minor allele frequencies of at least 1% were analyzed. A fixed effect variance weighted meta-analysis was then conducted using the effect estimates from the by-study analyses. Prespecified statistical significance thresholds for the meta-analysis results were as follows: *P* values of less than or equal to 0.0017 for candidate variants and *P* values of less than or equal to 2.50×10^−8^ for genome-wide variants to control for the family-wise error rate of 0.05 per end point. Per-allele odds ratios (ORs) with 95% confidence intervals (CIs) were generated. To aid in results interpretation, the placebo groups were analyzed in the same manner as the belimumab treatment groups (single-study analyses followed by a meta-analysis of the effect estimates).

#### Confirmatory analysis (https://www.gsk-clinicalstudyregister.com/study/207521)

The rs293983 variant was modeled as T-allele dose (0, 1, or 2 T alleles corresponding to genotypes CC, CT, or TT, respectively) or T-allele carriage (CT/TT vs. CC). Association of rs293983 with SRI4 response in belimumab-treated patients in BEL113750 was tested using logistic regression adjusting for the country, baseline SELENA SLEDAI scores, and baseline complement levels. A one-tailed test, with a prespecified significance threshold of *P* less than or equal to 0.05 was used. The placebo group was analyzed to aid interpretation.

## Results and discussion

In the exploratory meta-analysis, 10.9 million genetic variants were analyzed and two completely correlated (*r*^2^=1.0) genetic variants in the *ANO3* gene region, rs293983 and rs364370, were associated with belimumab efficacy as measured by SRI4 with the T-allele of both variants being associated with improved response (Table [2](#T2)). These variants were not associated with placebo response, suggesting the variants might be predictive of drug response rather than prognostic of lupus progression [8](#R8). No candidate variants were associated with belimumab efficacy (Supplementary Table 1, Supplemental digital content 1, [http://links.lww.com/FPC/B352](http://links.lww.com/FPC/B352)), including four variants in the *TNFSF13B/BAFF/BLYS* gene. Recently, genetic variants in the *BAFF* gene, rs12874404 and *BAFF-var* [a combination of rs374039502 and an insertion-deletion variant GCTGT→A (rs200748895)], were associated with increased levels of soluble BAFF, B lymphocytes, and immunoglobulins in a predominantly Sardinian SLE population [9](#R9). The OR for the carriage of *BAFF-var* and SLE risk was ∼1.4. The authors hypothesized that these variants may predict response to B-cell depleting therapies, including belimumab. The *BAFF* genetic variants are more frequent (33%) among Sardinians than other populations (Europeans ∼3–5%). While we did not directly genotype these variants in our 2015 GWAS analysis, rs200748895 was imputed (imputation *r*^2^=0.73). This variant was not associated with belimumab efficacy as measured by the SRI4 end point (*P*=0.84), although we had less than 10% power to detect an association for an additive per-allele OR of 2 for variants with a minor allele frequency of 5% or less.

### Table 2.

Table 2 Caption: Analysis of rs293983 ANO3 gene region variant and belimumab efficacy as measured by change in the SRI4 response index from baseline to the week 52 end point

Post-hoc analyses of BEL110751 and BEL110752 identified baseline disease activity characteristics that predict moderate to severe SLE flare, including baseline BLyS levels of at least 2 ng/ml [10](#R10). Treatment differences for SRI responses at week 52 between belimumab 10 mg/kg and placebo were greater in the group with BLyS of at least 2 ng/ml (24.1%, *P*<0.0001) compared with the group with BLyS of less than 2 ng/ml (8.2%, *P*=0.0158) and the risk of severe flare was reduced with belimumab versus placebo in the group with BLyS of at least 2 ng/ml (*P*=0.0002). Patients with baseline BLyS levels of at least 2 ng/ml had a poorer response to standard background therapy and benefited from belimumab treatment, presumably through belimumab’s inhibition of BLyS. Taken together, these data are consistent with belimumab efficacy in patients with elevated BLyS/BAFF levels.

Because the GWAS meta-analysis is an exploratory analysis of millions of genetic variants, subject to many potential sources of bias and technical artifacts, it is important to confirm any novel associations in an independent dataset. We determined that Study BEL113750 ([http://www.clinicaltrials.gov identifier NCT01345253](http://www.clinicaltrials.govidentifierNCT01345253)), a 52-week study of belimumab versus placebo added to standard of care in Northeast Asian lupus patients, provided the best opportunity to evaluate the preliminary association as this study included the SRI4 composite efficacy end point. In the meta-analysis, the SRI4 belimumab response rate was 54.1% in Asians (10 mg/kg intravenous or 200 mg subcutaneous) and 57.0% in non-Asians. The preliminary association between rs293983 and belimumab efficacy was not confirmed in BEL113750 (Table [2](#T2)). These negative results are unlikely to be due to a lack of generalizability of the preliminary association to an Asian population; a post-hoc analysis of the exploratory meta-analysis revealed the preliminary genetic association was observed in the subset of Asian patients treated with belimumab (*n*=157, ∼19% of the meta-analysis study population) (per-T-allele OR=2.85, 95% CI: 1.41–5.74, *P*=0.0021) was slightly stronger than in the overall exploratory meta-analysis. Furthermore, while the preliminary association did not meet genome-wide significance criteria in the underlying studies (BEL110751/BEL110752 that included belimumab intravenous and BEL112341 that included belimumab subcutaneous), carriage of the rs293983 T allele was enriched among the SRI4 responders (Table [3](#T3)). The sample size of the confirmatory study, BEL113750, had 85% power to confirm the original OR of 2.15. The BEL113750 PGx analysis group was very similar to the GWAS discovery population with respect to age, sex, SLE duration, and baseline SELENA SLEDAI scores. The most likely explanation is that the preliminary PGx associations were false positives. This was not entirely surprising as investigation of the *ANO3* gene region did not identify plausible biological mechanisms that might explain improved belimumab efficacy in carriers of *ANO3* gene variants. Our failure to replicate GWAS findings emphasizes the importance of replication cohorts to confirm or refute preliminary associations. The failure to identify statistically significant associations does not imply that there are no genetic influences on belimumab efficacy, but that they are unlikely to be common enough or large enough to be identified in a study of this size. Future studies with larger sample sizes may support further explorations of genetic contributions to belimumab response.

### Table 3.

Table 3 Caption: SRI4 response rates by clinical study

[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6613715_fpc-29-132-g002.jpg)

[](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6613715_fpc-29-132-g003.jpg)

## Supplementary Material

## Acknowledgements

### Conflicts of interest

D.A.R. and L.A.M. are employees of GlaxoSmithKline (GSK), the manufacturer of BENLYSTA (belimumab). P.L.S. and A.R.H. are former GSK employees. D.A.R., L.C.M., and A.R.H. own GSK stock. GSK funded the exploratory (GSK identifier 204901) and confirmatory (GSK identifier 207521) pharmacogenetic analyses described in this paper.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Pisetsky DS. Immune recognition of DNA in SLE. Clin Immunol Immunopathol 1998; 86:1–2.  [DOI](https://doi.org/10.1006/clin.1997.4464) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9434790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Immunol%20Immunopathol&title=Immune%20recognition%20of%20DNA%20in%20SLE&author=DS%20Pisetsky&volume=86&publication_year=1998&pages=1-2&pmid=9434790&doi=10.1006/clin.1997.4464&)

2. Kotzin BL. Systemic lupus erythematosus. Cell 1996; 85:303–306.  [DOI](https://doi.org/10.1016/s0092-8674(00)81108-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8616885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Systemic%20lupus%20erythematosus&author=BL%20Kotzin&volume=85&publication_year=1996&pages=303-306&pmid=8616885&doi=10.1016/s0092-8674(00)81108-3&)

3. Sobel ES, Mohan C, Morel L, Schiffenbauer J, Wakeland EK. Genetic dissection of SLE pathogenesis: adoptive transfer of SLE1 mediates the loss of tolerance by bone marrow-derived B cells. J Immunol 1999; 162:2415–2421.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9973523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol&title=Genetic%20dissection%20of%20SLE%20pathogenesis:%20adoptive%20transfer%20of%20SLE1%20mediates%20the%20loss%20of%20tolerance%20by%20bone%20marrow-derived%20B%20cells&author=ES%20Sobel&author=C%20Mohan&author=L%20Morel&author=J%20Schiffenbauer&author=EK%20Wakeland&volume=162&publication_year=1999&pages=2415-2421&pmid=9973523&)

4. A fact sheet from the office on Women’s Health: Lupus and Women; 2017. Available at: https://www.womenshealth.gov/files/documents/fact-sheet-lupus.pdf. [Accessed 1 May 2018].  [https://www.womenshealth.gov/files/documents/fact-sheet-lupus.pdf](https://www.womenshealth.gov/files/documents/fact-sheet-lupus.pdf)

5. Lewis MJ, Jawad AS. The effect of ethnicity and genetic ancestry on the epidemiology, clinical features and outcome of systemic lupus erythematous. Rheumatology 2017; 56:i67–i77.  [DOI](https://doi.org/10.1093/rheumatology/kew399) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27940583/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatology&title=The%20effect%20of%20ethnicity%20and%20genetic%20ancestry%20on%20the%20epidemiology,%20clinical%20features%20and%20outcome%20of%20systemic%20lupus%20erythematous&author=MJ%20Lewis&author=AS%20Jawad&volume=56&publication_year=2017&pages=i67-i77&pmid=27940583&doi=10.1093/rheumatology/kew399&)

6. Furie RA, Petri MA, Wallace DJ, Ginzler EM, Merrill JT, Stohl W, et al. Novel evidence‐based systemic lupus erythematosus responder index. Arthritis Rheum 2009; 61:1143–1151.  [DOI](https://doi.org/10.1002/art.24698) | [PMC free article](/articles/PMC2748175/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19714615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheum&title=Novel%20evidence%E2%80%90based%20systemic%20lupus%20erythematosus%20responder%20index&author=RA%20Furie&author=MA%20Petri&author=DJ%20Wallace&author=EM%20Ginzler&author=JT%20Merrill&volume=61&publication_year=2009&pages=1143-1151&pmid=19714615&doi=10.1002/art.24698&)

7. World Medical Association. WMA declaration of Helsinki: ethical principles for medical research involving human subjects. Available at: https://www.wma.net/policies-post/wma-declaration-of-helsinki-ethical-principles-for-medical-research-involving-human-subjects/. [Accessed 1 May 2018].  [https://www.wma.net/policies-post/wma-declaration-of-helsinki-ethical-principles-for-medical-research-involving-human-subjects/](https://www.wma.net/policies-post/wma-declaration-of-helsinki-ethical-principles-for-medical-research-involving-human-subjects/) | [Google Scholar](https://scholar.google.com/scholar_lookup?World%20Medical%20Association.%20WMA%20declaration%20of%20Helsinki:%20ethical%20principles%20for%20medical%20research%20involving%20human%20subjects.%20Available%20at:%20https://www.wma.net/policies-post/wma-declaration-of-helsinki-ethical-principles-for-medical-research-involving-human-subjects/.%20%5BAccessed%201%20May%202018%5D.)

8. FDA-NIH Biomarker Working Group. BEST (Biomarkers, EndpointS, and other Tools) Resource. Silver Spring, MD: Food and Drug Administration (US); 2016.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/27010052/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=BEST%20(Biomarkers,%20EndpointS,%20and%20other%20Tools)%20Resource&publication_year=2016&)

9. Steri M, Orru V, Idda ML, Pitzalis M, Pala M, Zara I, et al. Overexpression of cytokine BAFF and autoimmunity risk. N Engl J Med 2017; 376:1615–1626.  [DOI](https://doi.org/10.1056/NEJMoa1610528) | [PMC free article](/articles/PMC5605835/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28445677/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Overexpression%20of%20cytokine%20BAFF%20and%20autoimmunity%20risk&author=M%20Steri&author=V%20Orru&author=ML%20Idda&author=M%20Pitzalis&author=M%20Pala&volume=376&publication_year=2017&pages=1615-1626&pmid=28445677&doi=10.1056/NEJMoa1610528&)

10. Roth DA, Thompson A, Tang Y, Hammer AE, Molta CT, Gordon D. Elevated BLyS levels in patients with systemic lupus erythematosus: associated factors and responses to belimumab. Lupus 2016; 25:346–354.  [DOI](https://doi.org/10.1177/0961203315604909) | [PMC free article](/articles/PMC4785993/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26385220/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lupus&title=Elevated%20BLyS%20levels%20in%20patients%20with%20systemic%20lupus%20erythematosus:%20associated%20factors%20and%20responses%20to%20belimumab&author=DA%20Roth&author=A%20Thompson&author=Y%20Tang&author=AE%20Hammer&author=CT%20Molta&volume=25&publication_year=2016&pages=346-354&pmid=26385220&doi=10.1177/0961203315604909&)
