# Genome-wide association analysis of common genetic variants of resistant hypertension

## Metadata
**Authors:** Nihal El Rouby, Caitrin W McDonough, Yan Gong, Leslie A McClure, Braxton D Mitchell, Richard B Horenstein, Robert L Talbert, Dana C Crawford, Matthew A Gitzendanner, Atsushi Takahashi, Toshihiro Tanaka, Michiaki Kubo, Carl J Pepine, Rhonda M Cooper-DeHoff, Oscar R Benavente, Alan R Shuldiner, Julie A Johnson
**Journal:** The pharmacogenomics journal
**Date:** 2018 Sep 20
**DOI:** [10.1038/s41397-018-0049-x](https://doi.org/10.1038/s41397-018-0049-x)
**PMID:** 30237584
**PMCID:** PMC6426691
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6426691/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6426691/pdf/nihms-990476.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6426691/pdf/nihms-990476.pdf)

## Abstract

Resistant hypertension (RHTN), defined as uncontrolled blood pressure (BP) ≥ 140/90 using three or more drugs or controlled BP (<140/90) using four or more drugs, is associated with adverse outcomes, including decline in kidney function. We conducted a genome-wide association analysis in 1194 White and Hispanic participants with hypertension and coronary artery disease from the INternational VErapamil-SR Trandolapril STudy—GENEtic Substudy (INVEST-GENES). Top variants associated with RHTN at p < 10−4 were tested for replication in 585 White and Hispanic participants with hypertension and subcortical strokes from the Secondary Prevention of Subcortical Strokes GENEtic Substudy (SPS3-GENES). A genetic risk score for RHTN was created by summing the risk alleles of replicated RHTN signals. rs11749255 in MSX2 was associated with RHTN in INVEST (odds ratio (OR) (95% CI) = 1.50 (1.2–1.8), p = 7.3 × 10−5) and replicated in SPS3 (OR = 2.0 (1.4–2.8), p = 4.3 × 10−5), with genome-wide significance in meta-analysis (OR = 1.60 (1.3–1.9), p = 3.8 × 10−8). Other replicated signals were in IFLTD1 and PTPRD. IFLTD1 rs6487504 was associated with RHTN in INVEST (OR = 1.90 (1.4–2.5), p = 1.1 × 10−5) and SPS3 (OR = 1.70 (1.2–2.5), p = 4 × 10−3). PTPRD rs324498, a previously reported RHTN signal, was among the top signals in INVEST (OR = 1.60 (1.3–2.0), p = 3.4 × 10−5) and replicated in SPS3 (OR = 1.60 (1.1–2.4), one-sided p = 0.005). Participants with the highest number of risk alleles were at increased risk of RHTN compared to participants with a lower number (p-trend = 1.8 × 10−15). Overall, we identified and replicated associations with RHTN in the MSX2, IFLTD1, and PTPRD regions, and combined these associations to create a genetic risk score.

## Introduction

Despite the availability of numerous effective antihypertensive drug classes and medications within each class, nearly half of the patients with hypertension (HTN) continue to have uncontrolled BP and a subset of these of patients suffer from resistant hypertension (RHTN) [[1](#R1)].

According to the American Heart Association position statement in 2008, RHTN is defined as uncontrolled BP despite the use of maximum tolerated doses of three or more antihypertensive medications or controlled BP with the use of four or more medications, ideally with a diuretic included [[2](#R2)]. The prevalence of RHTN is estimated at 8–12% of adult population on the basis of 140/90 mmHg BP control cut-off [[3](#R3), [4](#R4)], these prevalence rates are estimated to increase by 4% with the release of the new American College of Cardiology/American Heart Association clinical guidelines, proposing both lower thresholds for hypertension detection and treatment goals [[5](#R5)]. The 2017 guidelines describe the diagnosis, risk factors and treatment of RHTN, driving recognition to this clinically important, high-risk phenotype. Hypertensive patients with RHTN are especially at a higher risk of adverse outcomes, including stroke, congestive heart failure, end stage renal disease compared to patients with easily controlled BP [[6](#R6), [7](#R7)].

RHTN is a complex phenotype driven by genetic and non-genetic factors. Lifestyle risk factors, lack of adherence, physician inertia and inaccurate measurements of blood pressure (BP), are considered non-genetic factors of RHTN [[2](#R2)]. While the clinical and lifestyle risk factors of RHTN have been extensively studied, the genetic risk factors of RHTN are less well studied and most of the published data come from small or candidate gene studies [[8](#R8)].

We hypothesize that RHTN is a pharmacogenomics phenotype for which some of the genetic variants will have large effect sizes, like other pharmacogenomics phenotypes, that may lead to inadequate response to different classes of BP lowering medications. In the current study, we sought to identify genetic variants of RHTN through the use of a genome-wide association analysis (GWAS) and create a genetic risk score using validated RHTN signals from this analysis. Discovery GWAS was performed in a cohort of hypertensive patients with documented coronary artery disease from the INternational VErapamil-SR Trandolapril Study (INVEST)—GENEtic Substudy (INVEST-GENES) and replication was performed in an independent cohort of hypertensive patients with stroke from the Secondary Prevention of Small Subcortical Strokes (SPS3) Genetic Substudy (SPS3-GENES). As a secondary validation, we performed a look-up of associated SNPs in a third RHTN dataset from the electronic Medical and Genomics network (eMERGE).

## Materials and methods

### Study design and participants

#### INVEST (discovery)

The INternational VErapamil and Trandolapril STudy (INVEST) was an international, multi-center clinical trial investigating cardiovascular (CV) outcomes of hypertensive patients with coronary artery disease after randomization to a β-blocker strategy (βB, atenolol) or calcium antagonist strategy (CA, verapamil); ([https://clinicaltrials.gov/identifier](https://clinicaltrials.gov/identifier), [NCT00133692](https://clinicaltrials.gov/ct2/show/NCT00133692)) [[9](#R9)]. INVEST-GENES, the genetic substudy of INVEST included 5979 participants with DNA samples, 1529 of whom had genome-wide genotypic information.

#### SPS3 (replication)

The Secondary Prevention of Small Subcortical Strokes study was an international, multi-center clinical trial evaluating the optimal antiplatelet regimen and BP target goal for patients with a history of subcortical stroke [[10](#R10)] ([http://www.clinicaltrials.gov](http://www.clinicaltrials.gov), [NCT00059306](https://clinicaltrials.gov/ct2/show/NCT00059306)). SPS3-GENES included 1139 participants with available DNA samples, 1049 of whom had genome-wide genotypic information.

The main studies and genetic substudies of INVEST and SPS3 were approved by the institutional review boards, and were conducted in accordance with the Declaration of Helsinki. Participants provided separate, voluntary, written informed consent for participation in main studies and genetic substudies.

#### eMERGE (secondary validation)

The electronic MEdical Records & GEnomics (eMERGE) network consists of electronic health record (EHR) linked bio-repositories from 10 sites in the US [[11](#R11)].

Detailed information about the studies is included in the Data Supplement.

### Resistant hypertension phenotype

#### INVEST

RHTN was defined using medication and BP measurements at the visit prior to experiencing study outcomes or censoring [[7](#R7), [12](#R12)]. Participants were classified as RHTN if their SBP was ≤140 or DBP ≤ 90 using three or more medications, or if they were using four or greater antihypertensive medications regardless of BP. Participants with SBP < 140 and DBP < 90 mmHg using three or fewer medications were included in the controlled BP group. Participants with SBP ≥ 140 or DBP ≥ 90 mmHg while on two or fewer medications were excluded from this analysis.

For the analyses described herein, we included a total of 1194 participants with GWAS data who met the criteria for RHTN dataset as either having RHTN or controlled BP. This included 657 Whites (226 RHTN; 431 controlled) and 537 Hispanics (143 RHTN; 394 controlled). Patients with uncontrolled BP on two or fewer antihypertensive drugs were excluded from the analysis.

#### SPS3

To construct the RHTN phenotype in SPS3, we excluded non-hypertensive participants. RHTN status was defined at the 12-month follow-up visit, which allowed enough time for BP medication titration to be complete and ensure that RHTN status was not driven by addition of more BP lowering medications in the low BP target goal. We observed a high concordance rate (>90%) in RHTN status between any two consecutive visits within a 6 months window from the 12-month visit (i.e., 12 months ± 6 months). RHTN phenotype was defined similarly to INVEST, with the exclusion of participants with SBP ≥ 140 or DBP ≥ 90 mmHg on two or fewer medications.

For our analyses in SPS3-GENES, we classified 585 hypertensive participants with available GWAS data who met the criteria for RHTN dataset as either having RHTN or controlled BP. This included 263 Whites (71 RHTN; 192 controlled) and 321 Hispanics (83 RHTN; 239 controlled BP). Patients with uncontrolled BP on two or fewer anti-hypertensive drugs were excluded from the analysis.

#### eMERGE

The RHTN dataset was constructed using EHR-linked data of hypertensive patients from seven sites in eMERGE (excluding the pediatric sites) [[11](#R11)]. RHTN was defined according to two algorithms; the first algorithm defined patients as RHTN if they have an outpatient SBP > 140 mmHg or DBP > 90 despite the use of ≥3 anti-hypertensive medication classes for at least one month after meeting medication criteria, and the second algorithm defined patients as RHTN if they used at least four concomitant antihypertensive medication classes. Patients with controlled BP were defined as hypertensive patients with SBP < 135 mmHg and DBP < 90 Hg, and used one antihypertensive medication. Patients were excluded if they had systolic heart failure or chronic kidney disease. The RHTN dataset within eMERGE included predominantly White patients and a very few Hispanics. Therefore, we looked up five top associations within INVEST–SPS3 patients using a cohort of RHTN and non-RHTN White patients within eMERGE (1946 cases and 471 controls).

### Genotyping, quality control, and imputation

The details on genotyping, quality control, and imputation performed on INVEST, SPS3, and eMERGE participants are presented in the online-only Data Supplement.

### Statistical analysis

Clinical characteristics of INVEST and SPS3 participants are presented as frequency and percentage for categorical variables, and means ± standard deviations for continuous variables. Univariate logistic regression was used to evaluate the differences in clinical characteristics between participants with and without RHTN. Analysis of clinical characteristics was performed using SAS version 9.3 (SAS Institute Inc, Cary, NC).

#### INVEST-SPS3

First, we assessed the associations between RHTN and 696,317 genotyped SNPs in INVEST-GENES (*n* = 1194) using logistic regression analysis and adjusting for clinical predictors of RHTN in INVEST as reported previously by Smith et al [[7](#R7)]. GWAS analysis was performed separately in Whites and Hispanics, based on PCA-defined genetic race, using PLINK v1.07 [[13](#R13)] and adjusting for clinical predictors of RHTN and ancestry specific PCA: PC1 in Whites and PC1 and PC2 in Hispanics. Our main analysis focused on genotyped SNPs, however, we performed a confirmatory analysis using imputed, 1000 Genomes, phase 3v5 data. For imputed data, we used EPACTS v3.2.6 software ([http://genome.sph.umich.edu/wiki/EPACTS](http://genome.sph.umich.edu/wiki/EPACTS)) to perform GWAS analysis on dosage files.

Second, we performed study-wide, fixed effect, inverse variance weighted meta-analysis in METAL [[14](#R14)] using association summary statistics of INVEST Whites and INVEST Hispanics, with the assumptions that functional SNPs should have consistent associations across racial/ethnic groups [[15](#R15)]. Genome-wide significance was set at 5 × 10^−8^ and suggestive SNPs were arbitrarily set at 1 × 10^−4^ in order to not dismiss biologically important SNPs if they do not meet a more stringent cut-off *p*-value. Studies have shown that carefully selected SNPs based on functional evidence and biological plausibility are more likely to be replicated [[16](#R16), [17](#R17)]. Therefore, we adopted a screening strategy to prioritize loci for validation, which is detailed in the online-only Data Supplement. Based on available support from literature to validate SNPs that did not meet the stringent genome-wide significance [[18](#R18), [19](#R19)], we set out to validate both genome-wide significant and suggestive SNPs and selected SNPs from INVEST meta-analysis to test their association in independent hypertensive participants (*n* = 585) from SPS3, the primary validation cohort for this GWAS study. A total of ten SNPs in ten independent genomic loci were tested for validation, ([Supplementary Table S2](#SD1)) and SNPs were considered validated in SPS3 by meeting a one-sided (same direction as discovery cohort) Bonferroni-corrected *p*-value of 0.005 (0.05/10 signals). Codes for GWAS association analysis can be made available upon request.

#### INVEST-SPS3-eMERGE

Next, to increase the power of GWAS to detect associations for signals with small to modest effect size, we conducted a meta-analysis of summary statistics of White and Hispanic participants of INVEST and SPS3 using fixed effect, inverse variance weighted meta-analysis in METAL [[14](#R14)]. eMERGE was used as a secondary validation cohort for genome-wide (*p* < 5 × 10^−8^), and suggestive SNPs (*p* < 5 × 10^−5^) from INVEST-SPS3 meta-analysis that met the SNP prioritization criteria and had the same direction of association in INVEST and SPS3. SNPs were considered validated if they had similar association as in INVEST and SPS3 at a Bonferroni-corrected one-sided *p*-value of 0.01 (0.05/5 SNPs) since a one-sided hypothesis was being tested.

#### Risk score analysis in INVEST and SPS3

We set out to construct a genetic score of RHTN SNPs to evaluate the effect of having multiple risk alleles on the phenotype, and advance potential translations of the findings. A risk score was generated using three independent SNPs that were replicated in SPS3 and included: rs11749255 in *MSX2*, rs6487504 in *IFLTD1*, and rs324498 in *PTPRD*. Genetic risk scores were constructed using an unweighted, allele counting method [[20](#R20)]. One participant was exluded from this analysis due to missing genotype. A single point was given to the risk allele associated with increased odds of RHTN [[20](#R20)]. The risk score ranged from 0 to 6 (2 points if participant was homozygous for risk allele, 1 point if heterozygous for risk allele, 0 points if homozygous for the protective allele). We evaluated the prevalence of RHTN across the risk score groups using a Cochran-Armitage Trend test separately within the four ancestry/ethnic groups, and the combined dataset of INVEST-SPS3.

## Results

The baseline clinical characteristics of patients in INVEST and SPS3 are summarized in [Table 1](#T1). On average, INVEST participants were older (mean age is 68 years) than SPS3 participants (mean age is 63 years). In INVEST, participants with RHTN had a higher prevalence of other cardiovascular co-morbidities such as congestive heart failure, myocardial infarction, and peripheral vascular disease. Participants with RHTN were more likely to be diabetic and have higher BMI compared to non-RHTN participants ([Table 1](#T1)). In eMERGE, approximately, half of patients were males, with a median BMI in the overweight category (~30–31 kg/m^2^), a median birth decade of 1940 (25%; 75% quartiles = 1930;1940, respectively) in both cases and controls.

### Table 1.

| Clinical characteristics | INVEST |  |  |  | SPS3 |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Whites |  | Hispanics |  | Whites |  | Hispanics |  |  |
| Controls N = 431 | RHTN cases N = 226 | Controls N = 394 | RHTN cases N = 143 | Controls N = 192 | RHTN cases N = 71 | Controls N = 239 | RHTN cases N = 83 |  |
| Age | 70 ± 10 | 70 ± 9 | 66 ± 10 | 66 ± 10 | 64 ± 10 | 63 ± 9 | 63 ± 11 | 63 ± 11 |
| Female | 184 (43%) | 114 (50%) | 225 (57%) | 87 (61%) | 70 (36%) | 18 (25%) | 98 (41%) | 37 (45%) |
| BMI, mean ± SD | 29 ± 6 | 29 ± 6 | 28.5 ± 5 | 30 ± 5** | 29 ± 6 | 31 ± 10 | 28 ± 4 | 30 ± 6## |
| SBP at RHTN classification | 126 ± 9 | 141 ± 17* | 124 ± 9 | 143 ± 18** | 126 ± 9 | 138 ± 14# | 122 ± 11 | 133 ± 14## |
| DBP at RHTN classification | 73 ± 8 | 76 ± 11* | 76 ± 7 | 84 ± 10** | 71 ± 8 | 73 ± 9 | 67 ± 10 | 70 ± 10 |
| Diabetes | 64 (15%) | 66 (29%)* | 51 (13%) | 25 (18%) | 44 (23%) | 25 (35%)# | 71 (30%) | 33 (40%) |
| Heart failure | 27 (6%) | 17 (8%) | 6 (2%) | 8 (6%)** | 1 (1%) | 2 (1.4%) | 2 (0.9%) | 1 (1.2%) |
| Myocardial infarction | 170 (39%) | 92 (41%) | 35 (9%) | 23 (16%) | 6 (3%) | 8 (11%)# | 8 (3%) | 2 (2%) |
| Peripheral vascular disease | 40 (9%) | 35 (16%)* | 34 (9%) | 23 (16%)** | 4 (2%) | 2 (3%) | 0 (0%) | 2 (2%) |
| Smoking | 204 (47%) | 113 (50%) | 137 (35%) | 44 (31%) | 41 (21%) | 21 (30%) | 18 (8%) | 3 (4%) |

Table 1 Caption: Clinical characteristics of INVEST and SPS3

Patients with RHTN generally had a significantly higher use of major antihypertensive medication drug classes compared to patients without RHTN, as shown in [Table 2](#T2). A significantly higher percentage of patients with RHTN used the recommended combination of medications for RHTN management such as thiazide diuretics, calcium channel blockers (CCBs), and angiotensin converting enzyme inhibitors (ACEIs), compared to patients without RHTN, suggesting that these patients in both trials were optimally managed with medications to reach their BP goals. In INVEST, approximately >80%, 50%, and 80% of patients with RHTN were on thiazide diuretics, CCBs, and ACEIs, respectively. Similarly, in SPS3, approximately >80% and, 70% of patients with RHTN were on thiazide diuretics and CCBs, respectively, and almost 60% were on ACEIs.

### Table 2.

| Drug class | INVEST |  |  |  | SPS3 |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Whites |  | Hispanics |  | Whites |  | Hispanics |  |  |
| Controls N = 431 | RHTN cases N = 226 | Controls N = 394 | RHTN cases N = 143 | Controls N = 192 | RHTN cases N = 71 | Controls N = 239 | RHTN cases N = 83 |  |
| SBP at RHTN classification | 126 ± 9 | 141 ± 17* | 124 ± 9 | 143 ± 18** | 126 ± 9 | 138 ± 14# | 122 ± 11 | 133 ± 14## |
| DBP at RHTN classification | 73 ± 8 | 76 ± 11* | 76 ± 7 | 84 ± 10** | 71 ± 8 | 73 ± 9 | 67 ± 10 | 70 ± 10 |
| Thiazide diuretics | 261 (61%) | 190 (84%)* | 240 (61%) | 122 (85%)** | 114 (59%) | 57 (80%)# | 123 (51%) | 70 (84%)## |
| Calcium channel blockers | 202 (47%) | 113 (50%) | 201 (51%) | 62 (43%) | 61 (32%) | 55 (77%)# | 80 (33%) | 74 (89%)## |
| Beta blockers | 212 (49%) | 107 (47%) | 178 (45%) | 79 (55%)** | 39 (20%) | 49 (69%)# | 30 (13%) | 61 (73%)## |
| ACE inhibitors | 292 (68%) | 196 (87%)* | 251 (64%) | 129 (90%)** | 99 (52%) | 43 (61%) | 110 (46%) | 49 (59%)## |

Table 2 Caption: Blood pressure and drug use at the visit of RHTN classification in INVEST and SPS3

The majority of patients were defined as RHTN based on having controlled BP on ≥4 drugs. This was specifically the case in 61% and 73% of INVEST Whites and Hispanics, respectively, and 45% and 55% of SPS3 Whites and Hispanics, respectively. This reflects the effective BP titration protocols in both INVEST and SPS3 in which patients’ BP was closely monitored, and medications were added and optimized, and explains the reason that the average BP in RHTN cases ([Table 2](#T2)) was <140 and <90 mmHg.

GWAS analysis in INVEST did not identify SNPs that reached genome-wide significance. However, 43 independent SNPs ([Supplementary Table S1](#SD1)) from INVEST (White-Hispanic meta-analysis) met the suggestive evidence of association and had consistent association among INVEST Whites and Hispanics participants; 10 of which ([Supplementary Table S2](#SD1)) were selected for replication in SPS3 (White-Hispanic meta-analysis) since they had the strongest evidence for a functional role according to Haploreg v.4 [[21](#R21)] and RegulomeDB v1.1 [[22](#R22)], and/or a biological role.

Among the 10 evaluated SNPs, 3 SNPs in the *MSX2*, *IFLTD1*, and *PTPRD* gene regions were replicated in SPS3 ([Table 3](#T3)). Minor allele frequencies and Hardy–Weinberg equilibrium *p*-values for the 3 SNPs are shown in [Supplementary Table S3](#SD1). The first replicated gene region included a SNP (rs11749255) located 82 kb upstream of *MSX2*. The A allele of rs11749255 was associated with a 50% increase in odds of RHTN in INVEST (OR (95% CI) 1.50 (1.2–1.8), *p* = 7.3 × 10^−5^) and twofold increased odds in RHTN in SPS3 (OR (95% CI) 2.00 (1.4–2.8), *p* = 4.4 × 10^−5^). This SNP reached genome-wide significance when INVEST and SPS3 were combined (OR (95% CI) 1.60 (1.3–1.9), *p* = 3.8 × 10^−8^) ([Fig. 1](#F1), [Table 2](#T2)). The *MSX2* gene region has several signals in linkage disequilibrium (LD) with rs11749255 as shown in the regional plot ([Supplementary Figure S2 and S4](#SD1)).

### Table 3.

| SNP | Ch | Position | Nearest gene | A1 | Study | White-Hispanic OR (95% CI) | White-Hispanic meta-analysis p | INVEST–SPS3 meta-analysis OR (95% CI) | INVEST–SPS3 meta-analysis p | Heterogeneity p |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs11749255 | 5 | 174642665 | MSX2 | A | INVEST | 1.5 (1.2, 1.8) | 7.3 × 10−5 | 1.60 (1.3, 1.9) | 3.8 × 10−8 | 0.14 |
|   |   |   |   |   | SPS3 | 2.0 (1.4, 2.8) | 4.4 × 10−5 |   |   |   |
| rs6487504 | 12 | 25654374 | IFLTD1 | A | INVEST | 1.9 (1.4, 2.5) | 1.1 × 10−5 | 1.81 (1.4, 2.3) | 1.6 × 10−7 | 0.92 |
|   |   |   |   |   | SPS3 | 1.7 (1.2, 2.5) | 4.0 × 10−3 |   |   |   |
| rs324498 | 9 | 9059545 | PTPRD | G | INVEST | 1.62 (1.3, 2.0) | 3.4 × 10−5 | 1.62 (1.3, 2.0) | 1.3 × 10−6 | 1 |
|   |   |   |   |   | SPS3 | 1.63 (1.1, 2.4) | 0.01 |   |   |   |

Table 3 Caption: RHTN SNPs: discovery in INVEST with replication in SPS3

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/16de/6426691/5bd081e12a0a/nihms-990476-f0001.jpg)

Adjusted odds ratios and 95% CIs for resistant hypertension risk for MSX2 rs11749255 in INternational VErapamil-SR Trandolapril STudy (INVEST) Whites, INVEST Hispanics, Secondary Prevention of Small Subcortical Strokes (SPS3) Whites, SPS3 Hispanics, and meta-analysis

The second region was found near the *IFLTD1* region, where rs6487504 was consistently associated with RHTN in both INVEST (OR (95% CI) 1.9 (1.4–2.5), *p* = 1.1 × 10^−5^) and SPS3 (OR (95% CI) 1.7(1.2–2.5), *p* = 4.0 × 10^−3^). Each additional copy of the variant allele (A) was associated with 81% higher odds for RHTN in the INVEST and SPS3 metaanalysis (OR (95% CI) = 1.81 (1.5–2.3)), *p* = 1.6 × 10^−7^ ([Fig. 2](#F2)).

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/16de/6426691/fc119ff2d1f0/nihms-990476-f0002.jpg)

Adjusted odds ratios and 95% CIs for resistant hypertension risk for IFLTD1 rs6487504 in INternational VErapamil-SR Trandolapril STudy (INVEST) Whites, INVEST Hispanics, Secondary Prevention of Small Subcortical Strokes (SPS3) Whites, SPS3 Hispanics, and meta-analysis

The third association of interest was an intronic SNP rs324498 in the *PTPRD*, a previously reported association with RHTN from INVEST that was identified using a large gene-centric chip analysis [[12](#R12)]. The SNP was associated with RHTN in INVEST (OR (95% CI) 1.62 (1.30–2.0), *p* = 3.4 × 10^−5^) and replicated in SPS3 (OR (95% CI) 1.63 (1.10–2.4), one-sided *p* = 0.005). Each additional copy of the variant allele (G) was associated with 62% increase in RHTN risk in the INVEST and SPS3 meta-analysis (OR (95% CI) = 1.62 (1.30–2.0)), *p* = 1.3 × 10^−6^ ([Fig. 3](#F3)).

### Fig. 3.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/16de/6426691/ce93fa05b734/nihms-990476-f0003.jpg)

Adjusted odds ratios and 95% CIs for resistant hypertension risk for PTPRD rs324498 in INternational VErapamil-SR Trandolapril STudy (INVEST) Whites, INVEST Hispanics, Secondary Prevention of Small Subcortical Strokes (SPS3) Whites, SPS3 Hispanics, and meta-analysis

The Manhattan and Q–Q plot of the INVEST–SPS3 meta-analysis are shown in [Supplementary Figures S2 and S3](#SD1). We selected five SNPs to validate in eMERGE including the rs11749255 *MSX2* and rs324498 *PTPRD* that replicated in SPS3 (rs6487504 *IFLTD1* SNP was not available in eMERGE). We were not able to validate signals rs11749255 *MSX2* and rs324498 *PTPRD* associations in eMERGE. However, we found a SNP rs16934621 in the *BNC2* gene region ([Supplementary Figure S5](#SD1)) that was associated with RHTN in the INVEST–SPS3 meta-analysis and had a directionally similar association in eMERGE ([Supplementary Table S4](#SD1)). New genetic loci were not identified when data was re-analyzed using imputed data.

We constructed a genetic score based on the three replicated SNPs (*MSX2* rs11749255, *PTPRD* rs324498, and *IFLTD1* rs6487504).The Cochran-Armitage Trend test revealed that participants with increased number of risk alleles (higher risk score) had a higher prevalence of RHTN compared to lower score participants (*p* = 1.8 × 10^−15^, [Fig. 4](#F4)). The association was consistent across the four ancestry/ethnic groups of INVEST and SPS3 ([Supplementary Figure S6A–D](#SD1)).

### Fig. 4.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/16de/6426691/f0a724e90964/nihms-990476-f0004.jpg)

Genetic risk score association with resistant hypertension (RHTN) in INternational VErapamil-SR Trandolapril STudy (INVEST) and Secondary Prevention of Small Subcortical Strokes (SPS3) datasets. Risk score was calculated using three SNPs: rs11749255 MSX2, rs324498 PTPRD and rs6487504 IFLTD1. One point was given to each allele conferring risk for RHTN. Participants with a higher risk score had a higher prevalence of RHTN compared to participants with a lower risk score

## Discussion

We sought to identify and replicate common genetic variants associated with RHTN across two cohorts of hypertensive patients treated with antihypertensive medications for BP control. Through a GWAS analysis approach, we identified three regions associated with RHTN in INVEST that validated in SPS3: *MSX2*, *IFLTD1*, and *PTPRD*. The lead SNPs of these associated loci were the same, and were tested for association in both whites and Hispanics. We also found another region of interest near the *BNC2* region, which was first identified in a meta-analysis of INVEST and SPS3 and validated in a cohort of hypertensive patients from EHR data in eMERGE.

The first identified region is in *MSX2* and included multiple associated variants. *MSX2* (msh homeobox 2) encodes for MSX2, a transcriptional factor that promotes the expression of osteogenic factors including alkaline phosphatases and plays a role in bone development [[23](#R23)]. Additionally, MSX2 acts as transcriptional modulator in vascular calcification [[24](#R24)]. In a model of vascular calcification, transgenic overexpression of Msx2 in mice was shown to activate Wnt-dependent signaling and promote vascular calcification [[25](#R25)]. The *MSX2* rs11739255 was associated with increased risk of RHTN and reached genome-wide significance when INVEST and SPS3 were combined, with a consistent association across INVEST whites, INVEST Hispanics; SPS3 whites, and SPS3 Hispanics. Additionally, rs11749255 is associated with histone modification mark (H3k4me1) in fetal heart tissue and placenta and altered binding of several regulatory motifs according to Haploreg v4 [[21](#R21)]. This SNP is an eQTL for *MSX2* in brain cortex according to the GTEx portal ([Supplementary Figure S7](#SD1)), which suggests that rs11749255 may modulate gene expression of *MSX2*.

The second identified region of interest is in the intermediate filament tail domain containing 1 (*IFLTD1*), which has a role in structural activity and cell proliferation. *IFLTD1* rs6487504 was associated with RHTN in INVEST and SPS3. Although it is unclear how the association in the *IFLTD1* gene region influences HTN and RHTN, several associations in *IFLTD1* with cardiovascular phenotypes including body mass index, carotid femoral pulse wave velocity, and left ventricular ejection time have been reported [[26](#R26), [27](#R27)].

The third associated and replicated region is in the *PTPRD* locus. The PTPRD protein belongs to the protein tyrosine phosphatase (PTP), a family of signaling molecules involved in a variety of cellular processes including mitotic cycle and cellular differentiation. The *PTPRD* rs324498 association was first identified in a large-centric gene analysis as a RHTN association [[12](#R12)] and was among the top associated SNPs in this analysis. We confirmed this association in hypertensive patients with a history of stroke from SPS3. Recently, two SNPs, rs12346562 and rs10739150 near the *PTPRD* were associated with BP response to atenolol in hypertensive participants from the Pharmacogenetics Evaluation of Antihypertensive Responses (PEAR) study [[28](#R28)].

*BNC2* encodes basonuclin 2, a zinc finger transcriptional factor [[29](#R29)]. SNPs in *BNC2* have been associated with glycemic control in type I diabetes and glycemic complications including diabetic nephropathy and retinal complications [[30](#R30)]. A recent analysis from the GenSalt study reported an association of *BNC2*—potassium interaction with diastolic blood pressure [[31](#R31), [32](#R32)]. *BNC2* is characterized by extreme conservation among vertebrates, suggesting its important regulatory function. While the exact mechanism of the associated *BNC2* SNP in the context of RHTN is unknown, data from ENCODE [[33](#R33)] illustrate that rs16934621 is associated with chromatin states in cell lines and affects protein binding ([Supplementary Figure S8](#SD1)).

Finally, participants with increased number of risk alleles were at a higher risk of developing RHTN compared to participants with lower number of risk alleles. This is in line with the polygenic nature of complex phenotypes in which multiple genetic variants are likely to act in concert to derive the phenotype. The genetic risk score has yet to be validated in independent RHTN cohorts. To date, there are no available RHTN cohorts with genome-wide data in which the risk score can be replicated. However, the International Consortium for Antihypertensive Pharmacogenomics Studies ([https://icaps-htn.org/](https://icaps-htn.org/)) include GWAS data available on antihypertensive drug response from 29 hypertensive cohorts. Datasets with ascertained BP response and potential to infer the RHTN phenotype, similar to INVEST and SPS3 are available in ICAPS, and present potential validation cohorts for the identified RHTN signals and genetic risk score. This is likely to promote the utility of prediction risk scores to identify high-risk patients, with whom nephrologists/clinicians need to be strict with risk factor modifications, for example, dietary sodium restriction. Such patients should have their antihypertensive regimen optimized with the recommended agents that include diuretic, long acting non-dihydropyridine calcium channel blocker, and a renin–angiotensin system blocker (ACEI or ARB). If BP is still uncontrolled, spironolactone, a highly effective mineralocorticoid receptor antagonist should be added as a fourth agent [[34](#R34)]. These patients may also benefit from referral to hypertension specialists and focused interviews with clinicians and pharmacists to educate about their disease risk and enhance compliance with pharmacological and non-pharmacological interventions.

The RHTN prevalence rates in our studies were higher than that in the most recent BP clinical trial called SPRINT [[35](#R35)]. Compared to INVEST and SPS3, the SPRINT trial randomized patients who were generally at lower risk, and particularly excluded certain patients in whom RHTN is more prevalent, e.g., patients with stroke and diabetes, both of which are well-documented risk factors for RHTN. In contrast, both INVEST and SPS3 allowed those patients and INVEST required patients to have coronary disease for enrollment and SPS3 required patients to have had a previous small subcortical stroke [[9](#R9), [36](#R36)]. Thus, the SPRINT inclusion criteria likely led to a cohort with lower prevalence of RHTN.

To our knowledge, this is the first GWAS analysis to identify RHTN using data from two randomized, outcomes-driven clinical trials. Strengths of this study are the consistency of findings across two clinical trials with well-documented drug use and dose optimization to a BP-driven protocol, overcoming physician inertia seen in clinical practice. Specifically in INVEST, a centralized and electronic data reporting system was used, which allowed for accurate monitoring of drug use. INVEST involved mechanisms to eliminate the reliance on patients to obtain study medications, and therefore allowing for consistent filling of medications. Specifically, a mail ordering pharmacy was used for processing and delivery of medications to the patient’s home, and the receipt of medications was confirmed via patient’s postcards [[37](#R37)]. Moreover, patients in INVEST experienced BP and heart rate lowering effects of atenolol and verapamil, an expected pharmacodynamics effect, further confirming ingestion of the drugs [[9](#R9)]. In SPS3, patients were followed monthly until BP is in goal, and then quarterly. Compliance with the medications was assessed in the follow-up visits and adherence was reported to be good or excellent in >75% of the visits [[10](#R10)]. Additionally, medications were offered at no cost whenever appropriate [[10](#R10)]. Finally, the consistency of associations and replicating/validating the signals in other datasets suggest that RHTN observed in these studies reflect a difficult to treat BP phenotype.

We acknowledge some limitations in our study. First, we were powered to detect signals of large effect sizes, therefore, the power in our discovery cohort (INVEST) was limited to detect associations with variants of small to moderate genetic effect; this was overcome to some extent by combining association results of two hypertensive cohorts. Second, we sought to utilize data derived from the EHR as part of eMERGE, as a secondary validation for the identified signals; however, we believe that heterogeneity in RHTN phenotype between INVEST–SPS3 and eMERGE, and the general quality of data in clinical trials versus within the electronic health records may have precluded replication of some the signals. In general, the data in EHR data were not necessarily collected to answer a specific type of research question, rather, were intended for clinical care. Additionally, some phenotypes may be more prone to error than others, resistant hypertension is one example. The challenges of creating RHTN from EHR was highlighted in a manuscript by Newton et al. [[38](#R38)]. Some of these challenges were related to the involvement of many variables that needed to be extracted from EHR to create the RHTN phenotype such as systolic, and diastolic blood pressures, free texts, ICD9 codes, medications, and laboratory tests. The need to accurately define the most meaningful time for blood pressure measurements using repeated measures data in EHR was also among the major challenges encountered in creating the RHTN phenotype [[38](#R38)]. Finally, the movement of patients in and out of the systems known as transience could have resulted in fragmented data, slightly decreasing the number of RHTN cases and controls within eMERGE and negatively influencing the power [[11](#R11)]. Despite the general limitations of using EHR in GWAS associations, the fact that one of INVEST–SPS3 signals in *BNC2* locus, a recently reported BP gene [[32](#R32)], was consistently associated in eMERGE at a nominal *p*-value suggests that the association with RHTN found in our analyses are likely real, and demonstrate the usefulness of collaborative approaches in discovering RHTN signals.

Third, systematic measures were not taken to completely rule out pseudo-resistance, for example, urine pharmacological screens were not performed to rule out nonadherence, and thus, we cannot ascertain that RHTN phenotype in our studies is strictly a true RHTN. The fact that we replicated/validated signals in three independent datasets suggest that the phenotype studied is driven by resistance to pharmacological treatments.

In conclusion, we identified and validated multiple variants for RHTN in different gene loci. Further validating the association of these variants and risk score in emerging RHTN cohorts may help in the precision medicine era, where patients with genetic predisposition to RHTN can be identified and treated accordingly to prevent adverse CV sequelae.

## Supplementary Material

## Acknowledgments

**Funding** INVEST was supported by grants from the University of Florida Opportunity Fund and Abbott Pharmaceuticals. INVEST-GENES was supported by NIH grants U01-GM074492, NIH R01 HL074730. The SPS3 trial was funded by the National Institute of Health and Neurological Disorders and Stroke Grant No. U01NS38529-04A1. The SPS3-GENES was funded by R01 NS073346 and U01-GM074492-05S109. Dr. El Rouby is supported by NIH grant T32HL083810 and Dr. McDonough is supported by NIH Grant 1 KL2 TR001429. The eMERGE Network is funded by NHGRI, with additional funding from NIGMS through the following grants: U01HG04599 and U01HG006379 to Mayo Clinic; U01HG004610 and U01HG006375 to Group Health Cooperative and University of Washington, Seattle; U01HG004608 to Marshfield Clinic; U01HG006389 to Essentia Institute of Rural Health; U01HG004609 and U01HG006388 to Northwestern University; U01HG04603 and U01HG006378 to Vanderbilt University; U01HG006385 to the Coordinating Center; U01HG006382 to Geisinger Clinic; U01HG006380 to Mount Sinai School of Medicine. A portion of the dataset(s) used for the analyses described were obtained from Vanderbilt University Medical Center’s BioVU, supported by institutional funding and by the Vanderbilt CTSA grant UL1 TR000445 from NCATS/NIH, and the Mayo Clinic Biobank supported by the Mayo Clinic Center for Individualized Medicine.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Egan BM, Zhao Y, Axon RN. US trends in prevalence, awareness, treatment, and control of hypertension, 1988-2008. JAMA. 2010;303:2043–50.  [DOI](https://doi.org/10.1001/jama.2010.650) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20501926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=US%20trends%20in%20prevalence,%20awareness,%20treatment,%20and%20control%20of%20hypertension,%201988-2008&author=BM%20Egan&author=Y%20Zhao&author=RN%20Axon&volume=303&publication_year=2010&pages=2043-50&pmid=20501926&doi=10.1001/jama.2010.650&)

2. Calhoun DA, Jones D, Textor S, Goff DC, Murphy TP, Toto RD, et al. Resistant hypertension: diagnosis, evaluation, and treatment: a scientific statement from the American Heart Association Professional Education Committee of the Council for High Blood Pressure Research. Circulation. 2008;117:e510–26.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.108.189141) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18574054/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Resistant%20hypertension:%20diagnosis,%20evaluation,%20and%20treatment:%20a%20scientific%20statement%20from%20the%20American%20Heart%20Association%20Professional%20Education%20Committee%20of%20the%20Council%20for%20High%20Blood%20Pressure%20Research&author=DA%20Calhoun&author=D%20Jones&author=S%20Textor&author=DC%20Goff&author=TP%20Murphy&volume=117&publication_year=2008&pages=e510-26&pmid=18574054&doi=10.1161/CIRCULATIONAHA.108.189141&)

3. Persell SD. Prevalence of resistant hypertension in the United States, 2003-2008. Hypertension. 2011;57:1076–80.  [DOI](https://doi.org/10.1161/HYPERTENSIONAHA.111.170308) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21502568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Prevalence%20of%20resistant%20hypertension%20in%20the%20United%20States,%202003-2008&author=SD%20Persell&volume=57&publication_year=2011&pages=1076-80&pmid=21502568&doi=10.1161/HYPERTENSIONAHA.111.170308&)

4. Sarafidis PA, Georgianos P, Bakris GL. Resistant hypertension—its identification and epidemiology. Nat Rev Nephrol. 2013;9:51–8.  [DOI](https://doi.org/10.1038/nrneph.2012.260) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23165303/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Nephrol&title=Resistant%20hypertension%E2%80%94its%20identification%20and%20epidemiology&author=PA%20Sarafidis&author=P%20Georgianos&author=GL%20Bakris&volume=9&publication_year=2013&pages=51-8&pmid=23165303&doi=10.1038/nrneph.2012.260&)

5. Whelton PK, Carey RM, Aronow WS, Casey DE, Collins KJ, Dennison Himmelfarb C, et al. 2017 ACC/AHA/AAPA/ABC/ ACPM/AGS/APhA/ASH/ASPC/NMA/PCNA guideline for the prevention, detection, evaluation, and management of high blood pressure in adults: a report of the American College of Cardiology/American Heart Association Task Force on Clinical Practice Guidelines. Hypertension. 2018;71:1269–324.  [DOI](https://doi.org/10.1161/HYP.0000000000000066) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29133354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=2017%20ACC/AHA/AAPA/ABC/%20ACPM/AGS/APhA/ASH/ASPC/NMA/PCNA%20guideline%20for%20the%20prevention,%20detection,%20evaluation,%20and%20management%20of%20high%20blood%20pressure%20in%20adults:%20a%20report%20of%20the%20American%20College%20of%20Cardiology/American%20Heart%20Association%20Task%20Force%20on%20Clinical%20Practice%20Guidelines&author=PK%20Whelton&author=RM%20Carey&author=WS%20Aronow&author=DE%20Casey&author=KJ%20Collins&volume=71&publication_year=2018&pages=1269-324&pmid=29133354&doi=10.1161/HYP.0000000000000066&)

6. Daugherty SL, Powers JD, Magid DJ, Tavel HM, Masoudi FA, Margolis KL, et al. Incidence and prognosis of resistant hypertension in hypertensive patients. Circulation. 2012;125:1635–42.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.111.068064) | [PMC free article](/articles/PMC3343635/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22379110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Incidence%20and%20prognosis%20of%20resistant%20hypertension%20in%20hypertensive%20patients&author=SL%20Daugherty&author=JD%20Powers&author=DJ%20Magid&author=HM%20Tavel&author=FA%20Masoudi&volume=125&publication_year=2012&pages=1635-42&pmid=22379110&doi=10.1161/CIRCULATIONAHA.111.068064&)

7. Smith SM, Gong Y, Handberg E, Messerli FH, Bakris GL, Ahmed A, et al. Predictors and outcomes of resistant hypertension among patients with coronary artery disease and hypertension. J Hypertens. 2014;32:635–43.  [DOI](https://doi.org/10.1097/HJH.0000000000000051) | [PMC free article](/articles/PMC4118668/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24299915/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hypertens&title=Predictors%20and%20outcomes%20of%20resistant%20hypertension%20among%20patients%20with%20coronary%20artery%20disease%20and%20hypertension&author=SM%20Smith&author=Y%20Gong&author=E%20Handberg&author=FH%20Messerli&author=GL%20Bakris&volume=32&publication_year=2014&pages=635-43&pmid=24299915&doi=10.1097/HJH.0000000000000051&)

8. El Rouby N, Cooper-DeHoff RM. Genetics of resistant hypertension: a novel pharmacogenomics phenotype. Curr Hypertens Rep. 2015;17:583.  [DOI](https://doi.org/10.1007/s11906-015-0583-8) | [PMC free article](/articles/PMC4717470/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26198781/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Hypertens%20Rep&title=Genetics%20of%20resistant%20hypertension:%20a%20novel%20pharmacogenomics%20phenotype&author=N%20El%20Rouby&author=RM%20Cooper-DeHoff&volume=17&publication_year=2015&pages=583&pmid=26198781&doi=10.1007/s11906-015-0583-8&)

9. Pepine CJ, Handberg EM, Cooper-DeHoff RM, Marks RG, Kowey P, Messerli FH, et al. A calcium antagonist vs a noncalcium antagonist hypertension treatment strategy for patients with coronary artery disease. The International Verapamil-Trandolapril Study (INVEST): a randomized controlled trial. JAMA. 2003;290:2805–16.  [DOI](https://doi.org/10.1001/jama.290.21.2805) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14657064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=A%20calcium%20antagonist%20vs%20a%20noncalcium%20antagonist%20hypertension%20treatment%20strategy%20for%20patients%20with%20coronary%20artery%20disease.%20The%20International%20Verapamil-Trandolapril%20Study%20(INVEST):%20a%20randomized%20controlled%20trial&author=CJ%20Pepine&author=EM%20Handberg&author=RM%20Cooper-DeHoff&author=RG%20Marks&author=P%20Kowey&volume=290&publication_year=2003&pages=2805-16&pmid=14657064&doi=10.1001/jama.290.21.2805&)

10. Benavente OR, White CL, Pearce L, Pergola P, Roldan A, Benavente MF, et al. The secondary prevention of small subcortical strokes (SPS3) study. Int J Stroke. 2011;6:164–75.  [DOI](https://doi.org/10.1111/j.1747-4949.2010.00573.x) | [PMC free article](/articles/PMC4214141/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21371282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Stroke&title=The%20secondary%20prevention%20of%20small%20subcortical%20strokes%20(SPS3)%20study&author=OR%20Benavente&author=CL%20White&author=L%20Pearce&author=P%20Pergola&author=A%20Roldan&volume=6&publication_year=2011&pages=164-75&pmid=21371282&doi=10.1111/j.1747-4949.2010.00573.x&)

11. Dumitrescu L, Ritchie MD, Denny JC, El Rouby NM, McDonough CW, Bradford Y, et al. Genome-wide study of resistant hypertension identified from electronic health records. PLoS ONE. 2017;12:e0171745.  [DOI](https://doi.org/10.1371/journal.pone.0171745) | [PMC free article](/articles/PMC5319785/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28222112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Genome-wide%20study%20of%20resistant%20hypertension%20identified%20from%20electronic%20health%20records&author=L%20Dumitrescu&author=MD%20Ritchie&author=JC%20Denny&author=NM%20El%20Rouby&author=CW%20McDonough&volume=12&publication_year=2017&pages=e0171745&pmid=28222112&doi=10.1371/journal.pone.0171745&)

12. Fontana V, McDonough CW, Gong Y, El Rouby NM, Sa AC, Taylor KD, et al. Large-scale gene-centric analysis identifies polymorphisms for resistant hypertension. J Am Heart Assoc. 2014;3:e001398.  [DOI](https://doi.org/10.1161/JAHA.114.001398) | [PMC free article](/articles/PMC4338734/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25385345/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Heart%20Assoc&title=Large-scale%20gene-centric%20analysis%20identifies%20polymorphisms%20for%20resistant%20hypertension&author=V%20Fontana&author=CW%20McDonough&author=Y%20Gong&author=NM%20El%20Rouby&author=AC%20Sa&volume=3&publication_year=2014&pages=e001398&pmid=25385345&doi=10.1161/JAHA.114.001398&)

13. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–75.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

14. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics. 2010;26:2190–1.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&author=CJ%20Willer&author=Y%20Li&author=GR%20Abecasis&volume=26&publication_year=2010&pages=2190-1&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

15. DIAbetes Genetics Replication And Meta-analysis (DIAGRAM) Consortium, Asian Genetic Epidemiology Network Type 2 Diabetes (AGEN-T2D) Consortium; South Asian Type 2 Diabetes (SAT2D) Consortium, Mexican American Type 2 Diabetes (MAT2D) Consortium, Type 2 Diabetes Genetic Exploration by Nex-generation sequencing in muylti-Ethnic Samples (T2D-GENES) Consortium, et al. Genome-wide trans-ancestry meta-analysis provides insight into the genetic architecture of type 2 diabetes susceptibility. Nat Genet. 2014;46:234–44.  [DOI](https://doi.org/10.1038/ng.2897) | [PMC free article](/articles/PMC3969612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24509480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20trans-ancestry%20meta-analysis%20provides%20insight%20into%20the%20genetic%20architecture%20of%20type%202%20diabetes%20susceptibility&volume=46&publication_year=2014&pages=234-44&pmid=24509480&doi=10.1038/ng.2897&)

16. Gorlov IP, Moore JH, Peng B, Jin JL, Gorlova OY, Amos CI. SNP characteristics predict replication success in association studies. Hum Genet. 2014;133:1477–86.  [DOI](https://doi.org/10.1007/s00439-014-1493-6) | [PMC free article](/articles/PMC4384517/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25273843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=SNP%20characteristics%20predict%20replication%20success%20in%20association%20studies&author=IP%20Gorlov&author=JH%20Moore&author=B%20Peng&author=JL%20Jin&author=OY%20Gorlova&volume=133&publication_year=2014&pages=1477-86&pmid=25273843&doi=10.1007/s00439-014-1493-6&)

17. Hou L, Zhao H. A review of post-GWAS prioritization approaches. Front Genet. 2013;4:280.  [DOI](https://doi.org/10.3389/fgene.2013.00280) | [PMC free article](/articles/PMC3856625/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24367376/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet&title=A%20review%20of%20post-GWAS%20prioritization%20approaches&author=L%20Hou&author=H%20Zhao&volume=4&publication_year=2013&pages=280&pmid=24367376&doi=10.3389/fgene.2013.00280&)

18. Barrett JC, Hansoul S, Nicolae DL, Cho JH, Duerr RH, Rioux JD, et al. Genome-wide association defines more than 30 distinct susceptibility loci for Crohn’s disease. Nat Genet. 2008;40:955–62.  [DOI](https://doi.org/10.1038/NG.175) | [PMC free article](/articles/PMC2574810/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18587394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20defines%20more%20than%2030%20distinct%20susceptibility%20loci%20for%20Crohn%E2%80%99s%20disease&author=JC%20Barrett&author=S%20Hansoul&author=DL%20Nicolae&author=JH%20Cho&author=RH%20Duerr&volume=40&publication_year=2008&pages=955-62&pmid=18587394&doi=10.1038/NG.175&)

19. Lee TH, Ko TM, Chen CH, Chang YJ, Lu LS, Chang CH, et al. A genome-wide association study links small-vessel ischemic stroke to autophagy. Sci Rep. 2017;7:15229.  [DOI](https://doi.org/10.1038/s41598-017-14355-3) | [PMC free article](/articles/PMC5680343/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29123153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=A%20genome-wide%20association%20study%20links%20small-vessel%20ischemic%20stroke%20to%20autophagy&author=TH%20Lee&author=TM%20Ko&author=CH%20Chen&author=YJ%20Chang&author=LS%20Lu&volume=7&publication_year=2017&pages=15229&pmid=29123153&doi=10.1038/s41598-017-14355-3&)

20. McDonough CW, Gong Y, Padmanabhan S, Burkley B, Langaee TY, Melander O, et al. Pharmacogenomic association of non-synonymous SNPs in SIGLEC12, A1BG, and the selectin region and cardiovascular outcomes. Hypertension. 2013. 62:48–54.  [DOI](https://doi.org/10.1161/HYPERTENSIONAHA.111.00823) | [PMC free article](/articles/PMC3686553/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23690342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Pharmacogenomic%20association%20of%20non-synonymous%20SNPs%20in%20SIGLEC12,%20A1BG,%20and%20the%20selectin%20region%20and%20cardiovascular%20outcomes&author=CW%20McDonough&author=Y%20Gong&author=S%20Padmanabhan&author=B%20Burkley&author=TY%20Langaee&volume=62&publication_year=2013&pages=48-54&pmid=23690342&doi=10.1161/HYPERTENSIONAHA.111.00823&)

21. Ward LD, Kellis M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Nucleic Acids Res. 2012;40 (Database issue):D930–4.  [DOI](https://doi.org/10.1093/nar/gkr917) | [PMC free article](/articles/PMC3245002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22064851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=HaploReg:%20a%20resource%20for%20exploring%20chromatin%20states,%20conservation,%20and%20regulatory%20motif%20alterations%20within%20sets%20of%20genetically%20linked%20variants&author=LD%20Ward&author=M%20Kellis&volume=40&issue=Database%20issue&publication_year=2012&pages=D930-4&pmid=22064851&doi=10.1093/nar/gkr917&)

22. Boyle AP, Hong EL, Hariharan M, Cheng Y, Schaub MA, Kasowski M, et al. Annotation of functional variation in personal genomes using RegulomeDB. Genome Res. 2012;22:1790–7.  [DOI](https://doi.org/10.1101/gr.137323.112) | [PMC free article](/articles/PMC3431494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Annotation%20of%20functional%20variation%20in%20personal%20genomes%20using%20RegulomeDB&author=AP%20Boyle&author=EL%20Hong&author=M%20Hariharan&author=Y%20Cheng&author=MA%20Schaub&volume=22&publication_year=2012&pages=1790-7&pmid=22955989&doi=10.1101/gr.137323.112&)

23. Cohen MM Jr. Craniofacial disorders caused by mutations in homeobox genes MSX1 and MSX2. J Craniofac Genet Dev Biol. 2000;20:19–25.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10879654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Craniofac%20Genet%20Dev%20Biol&title=Craniofacial%20disorders%20caused%20by%20mutations%20in%20homeobox%20genes%20MSX1%20and%20MSX2&author=MM%20Cohen&volume=20&publication_year=2000&pages=19-25&pmid=10879654&)

24. Towler DA, Bidder M, Latifi T, Coleman T, Semenkovich CF. Diet-induced diabetes activates an osteogenic gene regulatory program in the aortas of low density lipoprotein receptor-deficient mice. J Biol Chem. 1998;273:30427–34.  [DOI](https://doi.org/10.1074/jbc.273.46.30427) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9804809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Diet-induced%20diabetes%20activates%20an%20osteogenic%20gene%20regulatory%20program%20in%20the%20aortas%20of%20low%20density%20lipoprotein%20receptor-deficient%20mice&author=DA%20Towler&author=M%20Bidder&author=T%20Latifi&author=T%20Coleman&author=CF%20Semenkovich&volume=273&publication_year=1998&pages=30427-34&pmid=9804809&doi=10.1074/jbc.273.46.30427&)

25. Shao JS, Cheng SL, Pingsterhaus JM, Charlton-Kachigian N, Loewy AP, Towler DA. Msx2 promotes cardiovascular calcification by activating paracrine Wnt signals. J Clin Invest. 2005;115:1210–20.  [DOI](https://doi.org/10.1172/JCI24140) | [PMC free article](/articles/PMC1077175/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15841209/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Msx2%20promotes%20cardiovascular%20calcification%20by%20activating%20paracrine%20Wnt%20signals&author=JS%20Shao&author=SL%20Cheng&author=JM%20Pingsterhaus&author=N%20Charlton-Kachigian&author=AP%20Loewy&volume=115&publication_year=2005&pages=1210-20&pmid=15841209&doi=10.1172/JCI24140&)

26. Fox CS, Heard-Costa N, Cupples LA, Dupuis J, Vasan RS, Atwood LD. Genome-wide association to body mass index and waist circumference: the Framingham Heart Study 100K project. BMC Med Genet. 2007;8:S18.  [DOI](https://doi.org/10.1186/1471-2350-8-S1-S18) | [PMC free article](/articles/PMC1995618/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17903300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genet&title=Genome-wide%20association%20to%20body%20mass%20index%20and%20waist%20circumference:%20the%20Framingham%20Heart%20Study%20100K%20project&author=CS%20Fox&author=N%20Heard-Costa&author=LA%20Cupples&author=J%20Dupuis&author=RS%20Vasan&volume=8&publication_year=2007&pages=S18&pmid=17903300&doi=10.1186/1471-2350-8-S1-S18&)

27. Levy D, Larson MG, Benjamin EJ, Newton-Cheh C, Wang TJ, Hwang SJ, et al. Framingham Heart Study 100K Project: genomewide associations for blood pressure and arterial stiffness. BMC Med Genet. 2007;8:S3.  [DOI](https://doi.org/10.1186/1471-2350-8-S1-S3) | [PMC free article](/articles/PMC1995621/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17903302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genet&title=Framingham%20Heart%20Study%20100K%20Project:%20genomewide%20associations%20for%20blood%20pressure%20and%20arterial%20stiffness&author=D%20Levy&author=MG%20Larson&author=EJ%20Benjamin&author=C%20Newton-Cheh&author=TJ%20Wang&volume=8&publication_year=2007&pages=S3&pmid=17903302&doi=10.1186/1471-2350-8-S1-S3&)

28. Gong Y, McDonough CW, Beitelshees AL, El Rouby N, Hiltunen TP, O’Connell JR, et al. PTPRD gene associated with blood pressure response to atenolol and resistant hypertension. J Hypertens. 2015;33:2278–85.  [DOI](https://doi.org/10.1097/HJH.0000000000000714) | [PMC free article](/articles/PMC4788379/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26425837/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hypertens&title=PTPRD%20gene%20associated%20with%20blood%20pressure%20response%20to%20atenolol%20and%20resistant%20hypertension&author=Y%20Gong&author=CW%20McDonough&author=AL%20Beitelshees&author=N%20El%20Rouby&author=TP%20Hiltunen&volume=33&publication_year=2015&pages=2278-85&pmid=26425837&doi=10.1097/HJH.0000000000000714&)

29. Visser M, Palstra RJ, Kayser M. Human skin color is influenced by an intergenic DNA polymorphism regulating transcription of the nearby BNC2 pigmentation gene. Hum Mol Genet. 2014;23:5750–62.  [DOI](https://doi.org/10.1093/hmg/ddu289) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24916375/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Human%20skin%20color%20is%20influenced%20by%20an%20intergenic%20DNA%20polymorphism%20regulating%20transcription%20of%20the%20nearby%20BNC2%20pigmentation%20gene&author=M%20Visser&author=RJ%20Palstra&author=M%20Kayser&volume=23&publication_year=2014&pages=5750-62&pmid=24916375&doi=10.1093/hmg/ddu289&)

30. Paterson AD, Waggott D, Boright AP, Hosseini SM, Shen E, Sylvestre MP, et al. A genome-wide association study identifies a novel major locus for glycemic control in type 1 diabetes, as measured by both A1C and glucose. Diabetes. 2010; 59:539–49.  [DOI](https://doi.org/10.2337/db09-0653) | [PMC free article](/articles/PMC2809960/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19875614/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=A%20genome-wide%20association%20study%20identifies%20a%20novel%20major%20locus%20for%20glycemic%20control%20in%20type%201%20diabetes,%20as%20measured%20by%20both%20A1C%20and%20glucose&author=AD%20Paterson&author=D%20Waggott&author=AP%20Boright&author=SM%20Hosseini&author=E%20Shen&volume=59&publication_year=2010&pages=539-49&pmid=19875614&doi=10.2337/db09-0653&)

31. Li C, He J, Hixson JE, Gu D, Rao DC, Shimmin LC, et al. Abstract P253: genome-wide gene-potassium interaction analyses on blood pressure: The GenSalt Study. Circulation. 2016;133:AP253.  [DOI](https://doi.org/10.1161/CIRCGENETICS.117.001811) | [PMC free article](/articles/PMC5728702/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29212900/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Abstract%20P253:%20genome-wide%20gene-potassium%20interaction%20analyses%20on%20blood%20pressure:%20The%20GenSalt%20Study&author=C%20Li&author=J%20He&author=JE%20Hixson&author=D%20Gu&author=DC%20Rao&volume=133&publication_year=2016&pages=AP253&pmid=29212900&doi=10.1161/CIRCGENETICS.117.001811&)

32. Li C, He J, Chen J, Zhao J, Gu D, Hixson JE, et al. Genome-wide gene-potassium interaction analyses on blood pressure: the gensalt study (genetic epidemiology network of salt sensitivity). Circ Cardiovasc Genet. 2017;10:6.  [DOI](https://doi.org/10.1161/CIRCGENETICS.117.001811) | [PMC free article](/articles/PMC5728702/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29212900/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Cardiovasc%20Genet&title=Genome-wide%20gene-potassium%20interaction%20analyses%20on%20blood%20pressure:%20the%20gensalt%20study%20(genetic%20epidemiology%20network%20of%20salt%20sensitivity)&author=C%20Li&author=J%20He&author=J%20Chen&author=J%20Zhao&author=D%20Gu&volume=10&publication_year=2017&pages=6&pmid=29212900&doi=10.1161/CIRCGENETICS.117.001811&)

33. Rosenbloom KR, Sloan CA, Malladi VS, Dreszer TR, Learned K, Kirkup VM, et al. ENCODE data in the UCSC Genome Browser: year 5 update. Nucleic Acids Res. 2013;41:D56–63.  [DOI](https://doi.org/10.1093/nar/gks1172) | [PMC free article](/articles/PMC3531152/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23193274/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=ENCODE%20data%20in%20the%20UCSC%20Genome%20Browser:%20year%205%20update&author=KR%20Rosenbloom&author=CA%20Sloan&author=VS%20Malladi&author=TR%20Dreszer&author=K%20Learned&volume=41&publication_year=2013&pages=D56-63&pmid=23193274&doi=10.1093/nar/gks1172&)

34. Williams B, MacDonald TM, Morant S, Webb DJ, Sever P, McInnes G, et al. Spironolactone versus placebo, bisoprolol, and doxazosin to determine the optimal treatment for drug-resistant hypertension (PATHWAY-2): a randomised, double-blind, crossover trial. Lancet. 2015;386:2059–68.  [DOI](https://doi.org/10.1016/S0140-6736(15)00257-3) | [PMC free article](/articles/PMC4655321/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26414968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Spironolactone%20versus%20placebo,%20bisoprolol,%20and%20doxazosin%20to%20determine%20the%20optimal%20treatment%20for%20drug-resistant%20hypertension%20(PATHWAY-2):%20a%20randomised,%20double-blind,%20crossover%20trial&author=B%20Williams&author=TM%20MacDonald&author=S%20Morant&author=DJ%20Webb&author=P%20Sever&volume=386&publication_year=2015&pages=2059-68&pmid=26414968&doi=10.1016/S0140-6736(15)00257-3&)

35. Wright JT, Williamson JD, Whelton PK, Snyder JK, Sink KM, Rocco MV, et al. A randomized trial of intensive versus standard blood-pressure control. N Engl J Med. 2015;373:2103–16.  [DOI](https://doi.org/10.1056/NEJMoa1511939) | [PMC free article](/articles/PMC4689591/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26551272/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=A%20randomized%20trial%20of%20intensive%20versus%20standard%20blood-pressure%20control&author=JT%20Wright&author=JD%20Williamson&author=PK%20Whelton&author=JK%20Snyder&author=KM%20Sink&volume=373&publication_year=2015&pages=2103-16&pmid=26551272&doi=10.1056/NEJMoa1511939&)

36. Group SPSS Benavente, Coffey OR, et al. Blood-pressure targets in patients with recent lacunar stroke: the SPS3 randomised trial. Lancet. 2013;382:507–15.  [DOI](https://doi.org/10.1016/S0140-6736(13)60852-1) | [PMC free article](/articles/PMC3979302/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23726159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Blood-pressure%20targets%20in%20patients%20with%20recent%20lacunar%20stroke:%20the%20SPS3%20randomised%20trial&author=Benavente%20Group%20SPSS&author=OR%20Coffey&volume=382&publication_year=2013&pages=507-15&pmid=23726159&doi=10.1016/S0140-6736(13)60852-1&)

37. Cooper-DeHoff R, Handberg E, Heissenberg C, Johnson K. Electronic prescribing via the internet for a coronary artery disease and hypertension megatrial. Clin Cardiol. 2001;24:V14–6.  [DOI](https://doi.org/10.1002/clc.4960241706) | [PMC free article](/articles/PMC6654999/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11712771/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cardiol&title=Electronic%20prescribing%20via%20the%20internet%20for%20a%20coronary%20artery%20disease%20and%20hypertension%20megatrial&author=R%20Cooper-DeHoff&author=E%20Handberg&author=C%20Heissenberg&author=K%20Johnson&volume=24&publication_year=2001&pages=V14-6&pmid=11712771&doi=10.1002/clc.4960241706&)

38. Newton KM, Peissig PL, Kho AN, Bielinski SJ, Berg RL, Choudhary V, et al. Validation of electronic medical record-based phenotyping algorithms: results and lessons learned from the eMERGE network. J Am Med Inform Assoc. 2013;20:e147–54.  [DOI](https://doi.org/10.1136/amiajnl-2012-000896) | [PMC free article](/articles/PMC3715338/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23531748/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Med%20Inform%20Assoc&title=Validation%20of%20electronic%20medical%20record-based%20phenotyping%20algorithms:%20results%20and%20lessons%20learned%20from%20the%20eMERGE%20network&author=KM%20Newton&author=PL%20Peissig&author=AN%20Kho&author=SJ%20Bielinski&author=RL%20Berg&volume=20&publication_year=2013&pages=e147-54&pmid=23531748&doi=10.1136/amiajnl-2012-000896&)
