# Influence of Genetic Variants on Steady-State Etonogestrel Concentrations Among Contraceptive Implant Users

## Metadata
**Authors:** Aaron Lazorwitz, Christina L Aquilante, Kris Oreschak, Jeanelle Sheeder, Maryam Guiahi, Stephanie Teal
**Journal:** Obstetrics and gynecology
**Date:** 2019 Apr
**DOI:** [10.1097/AOG.0000000000003189](https://doi.org/10.1097/AOG.0000000000003189)
**PMID:** 30870275
**PMCID:** PMC6448146
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6448146/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6448146/pdf/nihms-1519638.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6448146/pdf/nihms-1519638.pdf)

## Abstract

**Objectives:** 
To identify genetic variants that influence steady-state etonogestrel concentrations among contraceptive implant users.

**Methods:** 
We enrolled healthy, reproductive-aged, women in our pharmacogenomic study using etonogestrel implants for 12–36 months without concomitant use of hepatic enzyme inducers or inhibitors. We collected participant characteristics, measured serum etonogestrel concentrations, and genotyped each participant for 120 single nucleotide variants in 14 genes encoding proteins involved in steroid hormone (i.e. estrogens, progestins) metabolism, regulation, or function. We performed generalized linear modeling to identify genetic variants associated with steady-state etonogestrel concentrations.

**Results:** 
We enrolled 350 women, who had a median serum etonogestrel concentration of 137.4pg/mL (range 55.8–695.1). Our final generalized linear model contained three genetic variants associated with serum ENG concentrations: NR1I2 (PXR) rs2461817 (β= 13.36, p=0.005), PGR rs537681 (β= −29.77, p=0.007), and CYP3A7*1C (β= −35.06, p=0.025). Variant allele frequencies were 69.4%, 84.9%, and 5.1%, respectively. Our linear model also contained two non-genetic factors associated with etonogestrel concentrations: body-mass index (β= −3.08, p=7.0×10−7) and duration of implant use (β= −1.60, p=5.8×10−5); R2 for the model = 0.17.

**Conclusion:** 
Only body-mass index and duration of implant use remained significantly associated with steady-state etonogestrel concentrations. Of the three novel genetic associations found, one variant associated with increased etonogestrel metabolism (CYP3A7*1C) causes adult expression of fetal CYP3A7 proteins and can consequently alter steroid hormone metabolism. Women with this variant may potentially have increased metabolism of all steroid hormones, as 27.8% (5/18) of CYP3A7*1C carriers had serum etonogestrel concentrations that fell below the threshold for consistent ovulatory suppression (<90pg/mL). More pharmacogenomic investigations are needed to advance our understanding of how genetic variation can influence the effectiveness and safety of hormonal contraception, and lay the groundwork for personalized medicine approaches in women’s health.

**Clinical Trial Registration:** 
ClinicalTrials.gov, NCT03092037.

### Objectives:

To identify genetic variants that influence steady-state etonogestrel concentrations among contraceptive implant users.

### Methods:

We enrolled healthy, reproductive-aged, women in our pharmacogenomic study using etonogestrel implants for 12–36 months without concomitant use of hepatic enzyme inducers or inhibitors. We collected participant characteristics, measured serum etonogestrel concentrations, and genotyped each participant for 120 single nucleotide variants in 14 genes encoding proteins involved in steroid hormone (i.e. estrogens, progestins) metabolism, regulation, or function. We performed generalized linear modeling to identify genetic variants associated with steady-state etonogestrel concentrations.

### Results:

We enrolled 350 women, who had a median serum etonogestrel concentration of 137.4pg/mL (range 55.8–695.1). Our final generalized linear model contained three genetic variants associated with serum ENG concentrations: *NR1I2 (PXR)* rs2461817 (β= 13.36, p=0.005), *PGR* rs537681 (β= −29.77, p=0.007), and *CYP3A7*1C* (β= −35.06, p=0.025). Variant allele frequencies were 69.4%, 84.9%, and 5.1%, respectively. Our linear model also contained two non-genetic factors associated with etonogestrel concentrations: body-mass index (β= −3.08, p=7.0×10^−7^) and duration of implant use (β= −1.60, p=5.8×10^−5^); R^2^ for the model = 0.17.

### Conclusion:

Only body-mass index and duration of implant use remained significantly associated with steady-state etonogestrel concentrations. Of the three novel genetic associations found, one variant associated with increased etonogestrel metabolism (*CYP3A7*1C*) causes adult expression of fetal CYP3A7 proteins and can consequently alter steroid hormone metabolism. Women with this variant may potentially have increased metabolism of all steroid hormones, as 27.8% (5/18) of *CYP3A7*1C* carriers had serum etonogestrel concentrations that fell below the threshold for consistent ovulatory suppression (<90pg/mL). More pharmacogenomic investigations are needed to advance our understanding of how genetic variation can influence the effectiveness and safety of hormonal contraception, and lay the groundwork for personalized medicine approaches in women’s health.

### Clinical Trial Registration:

[ClinicalTrials.gov](http://ClinicalTrials.gov), [NCT03092037](https://clinicaltrials.gov/ct2/show/NCT03092037).

## Introduction

In this era of precision and personalized medicine, as the field of pharmacogenomics and our understanding of the human genome advance, the potential to customize medical treatments based on an individual’s genetic profile is becoming a reality ([1](#R1), [2](#R2)). Pharmacogenomics is the study of the relationship between genetic variations and interindividual variability in drug disposition, response, and toxicity. Some of these genetic variations significantly affect the function of drug metabolizing enzymes and their regulatory proteins, leading to altered metabolism of medications including clopidogrel, warfarin, codeine, tacrolimus, and many others ([3](#R3)–[5](#R5)). The Clinical Pharmacogenetics Implementation Consortium currently provides clinical guidelines for over 35 medications ([1](#R1)). Drug-gene research often consists of both candidate gene approaches, selecting specific genetic variants to study based on physiologic plausibility, and genome wide association studies, searching for novel associations across the whole genome. If well-designed, both of these approaches can provide high quality evidence for the development of clinical guidelines ([6](#R6)). However, there is a dearth of information about genetic determinants of hormonal contraceptive disposition, response, and toxicity, which is surprising given that these are some of the most commonly prescribed medications in the United States ([7](#R7)).

Pharmacogenomics is particularly useful for medications with large interindividual differences in drug pharmacokinetics or pharmacodynamics. Pharmacokinetic data already demonstrate that there is a wide range (>12 fold difference) in drug concentrations for women using the exact same hormonal contraceptive method ([8](#R8)–[10](#R10)). This wide inter-individual variability can be beyond the accepted criteria for bioequivalence (95% confidence interval of the mean parameter values within 80–125% of the accepted standard), which may put some women outside the therapeutic range of their hormonal contraceptive method ([11](#R11), [12](#R12)). Estrogens and progestins are primarily metabolized by cytochrome P-450 (CYP) 3A enzymes, predominantly through CYP3A4 ([13](#R13)). Previous studies with other CYP3A substrates have shown that *CYP3A* genetic variants significantly affect drug metabolism and concentrations ([14](#R14), [15](#R15)). However, the influence of variation in *CYP3A* and related genes on steroid hormone medication pharmacokinetics is not known. According to the Guttmacher Institute, 27.6% of women have used a hormonal contraceptive method in the past month and four out of five sexually experienced women report current or past usage of an oral hormonal contraceptive method ([7](#R7), [16](#R16)). Given the personal and public health implications of unintended pregnancies from contraceptive failures, there is an urgent need to better understand the role of genetic variations on hormonal contraceptive serum concentrations, efficacy, and toxicity.

The etonogestrel contraceptive implant (Nexplanon®, formerly Implanon®, Merck & Co., Whitehouse Station NJ) has a well-described pharmacokinetic profile and represents an ideal model for pharmacogenomic studies of exogenous steroid hormones given its steady-state drug release and independence from issues of protocol adherence ([17](#R17)). To address current gaps in knowledge, we used a candidate gene approach to identify genetic variants that influence steady-state etonogestrel concentrations among a large, racially and ethnically diverse group of contraceptive implant users. We hypothesized that variants in genes encoding proteins involved in steroid hormone (i.e. estrogens, progestins) metabolism, regulation, and function would be associated with serum etonogestrel concentrations in contraceptive implant users and account for some of the known interindividual pharmacokinetic variability of this contraceptive method.

## Materials and Methods

In this pharmacogenomic study, we recruited English or Spanish speaking reproductive aged women (18–45 years old) with an etonogestrel contraceptive implant in place for at least 12 and no more than 36 months. We chose this duration of implant use because the etonogestrel implant has a pharmacokinetic burst early in the first year of use that then resolves to a relative steady-state for the remaining two years of use ([17](#R17)). We determined the length of implant use by participant report, corroborated with medical records if available, and confirmed presence of the implant by physical exam. We recruited participants through community advertising and contraceptive clinics at the University of Colorado Anschutz Medical Campus.

We excluded women over the age of 45 years due to potential altered drug metabolism from aging effects ([18](#R18)). We excluded women using medications or supplements that could alter serum etonogestrel levels through inhibition or induction of CYP enzymes (specifically CYP3A4). We reviewed concomitant medications through both participant report and medical record review and screened for any medications included in the U.S. Food and Drug Administration list of known CYP3A4 inducers or inhibitors ([19](#R19)). We also excluded women who reported medical conditions that affect baseline liver function (e.g. hepatitis, cirrhosis). As low body mass index (BMI) has been associated with abnormal etonogestrel metabolism, we measured a height and weight for each participant and excluded women with a measured BMI less than 18.5 kg/m^2^. We did not exclude women with high BMIs, as the present literature is inconclusive about the existence of an association between high BMI and serum etonogestrel concentrations ([9](#R9), [20](#R20)). Our protocol was approved by the Colorado Multiple Institutional Review Board and all participants gave written informed consent before study initiation.

After enrollment, we collected two blood samples from each participant at a single time point: one for serum etonogestrel concentration analysis and one for DNA extraction. Participants then completed a questionnaire that assessed baseline medical, contraceptive, and gynecological history in addition to self-reported demographics.

For etonogestrel analysis, we allowed the blood to clot for at least 10 minutes at room temperature, centrifuged the samples, and then stored the serum in aliquots at −80°C. We shipped all de-identified frozen serum samples to the Biomarkers Core Laboratory of the Irving Institute of Clinical and Translational Research at Columbia University Medical Center (New York City, NY). We used a previously validated liquid chromatography-mass-spectrometry assay protocol for sample analysis and performed serum etonogestrel concentration quantification in batches (after enrollment of 150, 250, and 350 participants) ([21](#R21)).

For our candidate gene approach, we selected 120 single nucleotide polymorphisms in 14 genes reported to be involved in the metabolism, regulation, and function of estrogens and progestins. We selected five metabolizing enzyme genes (*CYP3A4*, *CYP3A5*, *CYP3A7*, *CYP2C9*, and *CYP2C19*), seven genes known to regulate the activity of these metabolizing enzymes (cytochrome P450 oxidoreductase *[POR]*, nuclear receptor subfamily 1, group I, member 2 *[NR1I2(PXR)]*, aldo-keto reductase family 1, member C3 and member C4 *[AKR1C3* and *AKR1C4]*, hydroxyl-delta-5-steroid dehydrogenase, 3 beta- and steroid delta-isomerase 1 *[HSD3B1]*, low density lipoprotein receptor-related protein 2 *[LRP2]*, and UDP glucuronosyltransferase 1 family, polypeptide 1A *[UGT1A]*), and two genes encoding receptors involved in the action of estrogens and progestins (estrogen receptor alpha *[ESR1]* and progesterone receptor *[PGR]* ([2](#R2)).

Using PharmGKB and PubMed, we searched for known genetic variants in our 14 target genes with reported functional effects or published associations with medication pharmacokinetics, pharmacodynamics, or adverse effects ([2](#R2)). We then used dbSNP (The Database for Short Genetic Variations managed by the National Center for Biotechnology Information, catalog available online at [http://www.ncbi.nlm.nih.gov/projects/SNP/](http://www.ncbi.nlm.nih.gov/projects/SNP/)) to identify the frequency of each variant in the general population ([22](#R22)). We excluded any genetic variants with a frequency of less than 3% in either Caucasian or African populations, the two predominant racial groups in our local population, due to inadequate power to find associations with these rare variants. We did, however, include rare genetic variants (<3% frequency) that are known to alter drug metabolism. Based on these criteria, we selected 120 single nucleotide polymorphisms in drug metabolism and related genes, as outlined above. See [Table 1](#T1) for a complete list of all variants selected and their reference single nucleotide polymorphism (rs) ID numbers.

### Table 1.

| Gene | Variant (rs #) | Designation (if available) |
| --- | --- | --- |
| CYP3A4 | rs2740574 | CYP3A4*1B |
|   | rs2242480 | CYP3A4*1G |
|   | rs55785340 | CYP3A4*2 |
|   | rs55951658 | CYP3A4*4 |
|   | rs56324128 | CYP3A4*7 |
|   | rs72552799 | CYP3A4*8 |
|   | rs72552798 | CYP3A4*9 |
|   | rs67784355 | CYP3A4*11 |
|   | rs12721629 | CYP3A4*12 |
|   | rs4987161 | CYP3A4*17 |
|   | rs67666821 | CYP3A4*20 |
|   | rs35599367 | CYP3A4*22 |
|   | rs2246709 |   |
|   | rs28371759 |   |
|   | rs3735451 |   |
|   | rs4646437 |   |
|   | rs4646438 |   |
|   | rs4646440 |   |
|   | rs4986909 |   |
|   | rs4986910 |   |
| CYP3A5 | rs28365083 | CYP3A5*2 |
|   | rs776746 | CYP3A5*3A/*10/*11 |
|   | rs15524 | CYP3A5*3F/*1D/*10 |
|   | rs10264272 | CYP3A5*6 |
|   | rs41303343 | CYP3A5*7 |
|   | rs17161788 |   |
|   | rs41279854 |   |
|   | rs55965422 |   |
| CYP3A7 | rs11568824 | CYP3A7*1C |
|   | rs45446698 | CYP3A7*1C |
|   | rs11568826 | CYP3A7*1C |
|   | rs28451617 | CYP3A7*1E |
|   | rs2257401 | CYP3A7*2 |
|   | rs10211 |   |
|   | rs45465393 |   |
| CYP2C9 | rs1799853 | CYP2C9*2 |
|   | rs1057910 | CYP2C9*3 |
|   | rs56165452 | CYP2C9*4 |
|   | rs28371686 | CYP2C9*5 |
|   | rs7900194 | CYP2C9*8 |
|   | rs28371685 | CYP2C9*11 |
|   | rs10509680 |   |
|   | rs12782374 |   |
|   | rs1934969 |   |
|   | rs4086116 |   |
|   | rs4918758 |   |
|   | rs7089580 |   |
|   | rs9332096 |   |
|   | rs9332098 |   |
|   | rs9332238 |   |
| CYP2C19 | rs17885098 | CYP2C19*1B/2C/5B |
|   | rs4244285 | CYP2C19*2 |
|   | rs3758580 | CYP2C19*2B/2C |
|   | rs17878459 | CYP2C19*2B/2C |
|   | rs12571421 | CYP2C19*2C |
|   | rs28399513 | CYP2C19*2C |
|   | rs7916649 | CYP2C19*2C/3B |
|   | rs12769205 | CYP2C19*2C/2D |
|   | rs4417205 | CYP2C19*2C/2D |
|   | rs4986894 | CYP2C19*2C/2D |
|   | rs4986893 | CYP2C19*3 |
|   | rs11568732 | CYP2C19*3B |
|   | rs17884832 | CYP2C19*3B |
|   | rs7088784 | CYP2C19*3B |
|   | rs28399504 | CYP2C19*4 |
|   | rs56337013 | CYP2C19*5 |
|   | rs72552267 | CYP2C19*6 |
|   | rs11188072 | CYP2C19*17 |
|   | rs12248560 | CYP2C19*17 |
|   | rs4917623 | CYP2C19*19 |
|   | rs7902257 | CYP2C19*27 |
| POR | rs1057868 | POR*28 |
|   | rs2868177 |   |
| NR1I2 (PXR) | rs10934498 |   |
|   | rs1464602 |   |
|   | rs1464603 |   |
|   | rs1523127 |   |
|   | rs1523130 |   |
|   | rs2276706 |   |
|   | rs2276707 |   |
|   | rs2461817 |   |
|   | rs2472677 |   |
|   | rs3732359 |   |
|   | rs3732360 |   |
|   | rs3814055 |   |
|   | rs3814058 |   |
|   | rs4688040 |   |
|   | rs6785049 |   |
|   | rs72551372 |   |
|   | rs72551374 |   |
|   | rs7643645 |   |
| AKR1C3 | rs12529 |   |
|   | rs1937840 |   |
|   | rs34186955 |   |
|   | rs35575889 |   |
|   | rs4987102 |   |
| AKR1C4 | rs11253043 |   |
|   | rs17134592 |   |
| HSD3B1 | rs7553527 |   |
| LRP2 | rs2075252 |   |
|   | rs2228171 |   |
| UGT1A1 | rs887829 | UGT1A1*80 |
| ESR1 | rs2077647 |   |
|   | rs2207396 |   |
|   | rs2228480 |   |
|   | rs2234693 |   |
|   | rs2813543 |   |
|   | rs3020314 |   |
|   | rs3798577 |   |
|   | rs4870061 |   |
|   | rs9322335 |   |
|   | rs9322336 |   |
|   | rs9340799 |   |
| PGR | rs11224556 |   |
|   | rs3740751 |   |
|   | rs471767 |   |
|   | rs537681 |   |
|   | rs555653 |   |
|   | rs561610 |   |
|   | rs572943 |   |

Table 1 Caption: Complete list of all 120 genetic variants selected and their reference single nucleotide polymorphism (rs) ID numbers

We extracted genomic DNA from whole blood samples using commercially available Qiagen® kits. In collaboration with LifeTechnologies®, we designed customized Taqman® OpenArray® microarray genotyping panels based on the 120 selected single nucleotide polypmorphisms. The prepared DNA samples were de-identified and shipped to the Genomics Core Facility at the University of Utah for analysis using standard TaqMan® protocols and QuantStudio® 12K Flex Real-Time PCR System ([23](#R23)). The genotyping results were plotted using the TaqMan® Genotyper Software (version 1.3) to create an allelic discrimination plot for each single nucleotide polymorphisms. We assigned a genotype to each participant if the quality score was ≥0.90 for the sample.

We used IBM SPSS® version 25 statistical software for our analyses. We performed descriptive frequencies and conducted simple linear regression to identify genetic variants associated with serum etonogestrel concentrations with a p-value cut-off of 0.05 for inclusion in our multivariable analysis. We grouped genotypes into three groups (homozygous wild-type versus heterozygous versus homozygous variant) for analysis. We calculated allele and genotype frequencies and determined whether each single nucleotide polymorphism was in Hardy-Weinberg equilibrium, which is based on the principle that genetic frequencies will remain constant across generations when solely inherited by random selection and barring outside influences. We excluded any variants with frequencies <0.5% (<2/350) in our participants from the multivariable analysis.

Given that serum etonogestrel concentrations do not follow a normal distribution ([9](#R9)), we used a generalized linear model, which allows for multivariable linear regression analysis regardless of the distribution of the dependent variable ([24](#R24)). In this generalized linear model, we included all genetic variants found to be significantly associated with serum etonogestrel concentrations in simple linear modeling, and also included the variables of months of implant use and BMI based on prior published associations ([9](#R9), [17](#R17), [20](#R20)). We utilized a backward-stepwise approach to create our final linear model where all variables were entered into the model initially. We then sequentially removed variables without significant associations (p<0.05) until we obtained a model with the minimal Akaike’s Information Criterion value ([25](#R25)). Given the multiple hypothesis testing performed using simple linear regression, we determined a *Bonferroni* corrected p-value based on the number of simple linear regression analyses performed. We used this corrected p-value to determine overall significance for the variables included in our final generalized linear model.

We powered this study to detect a 50% difference in median etonogestrel concentrations between variant carriers and women with the respective homozygous wild-type genotype assuming a variant prevalence of 5% in the general population. We selected a 50% difference as this would cause the mean serum etonogestrel concentration to fall below 90pg/mL, which is the manufacturer’s reported threshold for maintain ovulatory suppression with the implant ([13](#R13)). We converted the median and range etonogestrel levels from the data of McNicholas et al (2017) to a mean and standard deviation using the methods described by Hozo et al (2005) as an estimate of the mean etonogestrel levels in contraceptive implant users with a wild-type genotype ([9](#R9), [26](#R26)). To detect our selected difference with a 5% risk of type I error and 20% risk of type II error, we needed 311 participants. We decided a priori to enroll 350 participants to increase our chances of identifying less common genetic variants.

## Results

We enrolled 350 participants ([Figure 1](#F1)) over 15 months (March 2017 to May 2018). [Table 2](#T2) demonstrates pertinent participant characteristics. The median age was 22.5 years (range 18.0 – 39.1), median BMI was 26.0 kg/m^2^ (range 18.5 – 52.0), and participants had a median duration of etonogestrel implant use of 25.7 months (range 12 – 36). The most frequent self-reported race was White (Caucasian) (46.6% [163/350]) and 51.4% (180/350) of participants reported Hispanic or Latina ethnicity. Amongst the 77 participants who responded with “No response or Unknown” in relation to self-identified race, all identified their ethnicity as “Hispanic or Latina.”

### Figure 1:

![Figure 1:](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5369/6448146/feb07e3d9482/nihms-1519638-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6448146_nihms-1519638-f0001.jpg)

Flow diagram for recruitment, screening, and enrollment of all 350 participants. CYP3A4, cytochrome P-450 3A4.

### Table 2:

|   | Median (Range) |
| --- | --- |
| Age (years) | 22.5 (18.0 – 39.1) |
| Months of implant use | 26.0 (12.0 – 36.0) |
| BMI (kg/m2) | 25.7 (18.5 – 52.0) |

Table 2 Caption: Characteristics and demographics of all 350 participants

The median etonogestrel serum concentration was 137.4pg/mL (range 55.8–695.1) with an interquartile range of 63.5pg/mL. [Figure 2](#F2) shows the distribution of serum etonogestrel concentrations. On average, we were able to confidently assign a genotype to 95.1% (range 82.6% to 98.9%) of participants for each single nucleotide polymorphism tested using our selected quality score cut-off of ≥0.90. Thus, for less than 5% of participants on average, we could not determine if a specific single nucleotide polymorphism was present and so these participants were not included in the simple linear regression analysis for that specific single nucleotide polymorphism. Out of the 120 genetic variants interrogated, 19 (15.8%) were not present in this cohort and were excluded from the analysis. The remaining genetic variants had frequencies ranging from 0.3% to 49.9%. To assess if these frequencies were consistent with the minor allele frequencies (MAFs) in the general population, we tested for Hardy-Weinberg equilibrium ([27](#R27)). For all but eight genetic variants, MAFs were consistent with Hardy-Weinberg equilibrium using Chi-squared tests with a p-value cut-off of 0.05 ([28](#R28)). Two specific variants had such significant deviation from Hardy-Weinberg equilibrium that these variants were excluded from analysis: *CYP3A7* rs10211 and *CYP3A4* rs72552799. The remaining six variants with deviations from Hardy-Weinberg equilibrium were deemed due to chance or population stratification and so were included in the analysis: *AKR1C4* rs17134592, *CYP2C19* rs17885098, *CYP3A4* rs4646437, *PGR* rs471767, *CYP2C19* rs7916649, and *CYP2C9* rs9332238.

### Figure 2:

![Figure 2:](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5369/6448146/b62ee7ebb4a0/nihms-1519638-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6448146_nihms-1519638-f0002.jpg)

Box plot of serum etonogestrel concentrations for all 350 participants. The box represents the first and third quartiles (IQR=63.5 pg/mL) with the band inside the box representing the median (137.4 pg/mL). Whiskers represent the data within 1.5 interquartile range of the upper and lower quartile. Circles indicate outliers with values between 1.5 and 3 times the IQR and asterisks indicate outliers with values greater than 3 times the IQR.

Three genetic variants chosen for *CYP3A7* (rs11568824, rs11568826, and rs45446698) were part of seven single nucleotide polymorphisms known to be in linkage disequilibrium that designate the *CYP3A7*1C* genotype. Therefore, we grouped participants with these variants as *CYP3A7*1C* carriers for our analysis. Additionally, rs776746 (*CYP3A5*3*) was used to assign CYP3A5 expressor (*1 carriers) or non-expressor (*3/*3 diplotype) status for analysis.

We performed univariate linear regression for each genetic variant that remained in the analysis to test for associations with serum etonogestrel concentrations. Five variants had significant associations: *NR1I2(PXR)* rs2461817, *PGR* rs537681, *CYP3A4* rs55785340 (designated *CYP3A4*2*), *NR1I2(PXR)* rs7643645, and *CYP3A7*1C*. We excluded *CYP3A4* rs55785340 from the multivariable analysis as it had a prevalence of 0.3% (1/350). Using a generalized linear model and backward step-wise approach as outlined in the methods ([Table 3](#T3)), three genetic variants (*NR1I2(PXR)* rs2461817, *PGR* rs537681, and *CYP3A7*1C*) remained in the final model in addition to duration of implant use and BMI, resulting in a final regression model with an R^2^ of 0.17. Only one variable was associated with increased serum etonogestrel concentrations (*NR1I2[PXR]* rs2461817), whereas *CYP3A7*1C* carrier status, *PGR* rs537681 carrier status, longer duration of implant use, and higher BMI were all associated with decreased serum etonogestrel concentrations. All genetic variants had minimal change in their β-coefficients when self-reported participant race and ethnicity were factored into the linear model ([Table 3](#T3)). To determine overall statistical significance, we used a *Bonferroni* correction to account for the 100 statistical tests performed: corrected p-value cut-off of 5.0×10^−4^. Only duration of implant use (p=5.8×10^−5^) and BMI (p=7.0×10^−7^) remained statistically significant based on this conservative threshold.

### Table 3:

| Variable | β-coefficient | 95% confidence interval | p-value |
| --- | --- | --- | --- |
| Months of implant use | −1.60 | −2.38, −0.82 | 5.8 × 10−5 |
| BMI (kg/m2) | −3.08 | −4.30, −1.86 | 7.0 ×10−7 |
| NR1I2 (PXR) rs2461817 | 13.36 | 3.93, 22.78 | 0.005 |
| PGR rs537681 | −29.77 | −51.47, −8.07 | 0.007 |
| CYP3A7*1C | −35.06 | −65.82, −4.30 | 0.025 |
| Expanded Model including Race and Ethnicity |  |   |  |
| NR1I2 (PXR) rs2461819 | 16.71 |  |  |
| PGR rs537681 | −32.06 |  |  |
| CYP3A7*1C | −31.85 |  |  |

Table 3 Caption: Beta-coefficients and 95% confidence intervals for variables in the final generalized linear model.

[Table 4](#T4) shows the frequencies and median serum etonogestrel concentrations for each genetic variant in the final linear model and the respective wild-type participants. *PGR* rs537681 was the most common variant allele included in the final model (found in 84.9% of participants), followed by *NR1I2(PXR)* rs2461817 in 69.4% of participants and *CYP3A7**1C in only 5.1% of participants. [Figures 3](#F3), [4](#F4), and [5](#F5) show the box-plot distributions of serum etonogestrel concentrations in participants with and without the *CYP3A7*1C* variant, all three genotypes of *NR1I2(PXR)* rs2461817, and with and without the *PGR* rs537681 variant, respectively. We evaluated for linkage disequilibrium between these three genetic variants and other variants in *CYP3A4* or *CYP3A5* using Ensembl^©^ release 94 ([29](#R29)). No genetic variants included in this analysis had linkage disequilibrium with these three variants in the model.

### Table 4:

| Genetic variantGenotype | Number of Participants(N = 350) | Serum etonogestrel concentration (pg/mL)Median (range) |
| --- | --- | --- |
| CYP3A7*1C |   |   |
| Wild-type | 332 (94.9%) | 139.8 (55.8 – 695.1) |
| Variant carriers | 18 (5.1%) | 113.0 (65.5 – 180.6) |
| NR1I2 (PXR) rs2461817 |   |   |
| No variant alleles (homozygous wild-type A/A) | 107 (30.6%) | 127.8 (61.0 – 345.7) |
| One variant alleles (heterozygous A/C) | 151 (43.1%) | 135.2 (61.2 – 695.1) |
| Two variant alleles (homozygous variant C/C) | 92 (26.3%) | 153.4 (55.8 – 434.1) |
| PGR rs537681 |   |   |
| Wild-type (homozygous C/C) | 53 (15.1%) | 171.7 (86.2 – 484.7) |
| Variant carriers (heterozygous C/T & homozygous T/T) | 297 (84.9%) | 134.8 (55.8 – 695.1) |

Table 4 Caption: Frequencies and median serum etonogestrel concentrations for genetic variants in the final generalized linear model.

### Figure 3:

![Figure 3:](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5369/6448146/ffdc950e3e03/nihms-1519638-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6448146_nihms-1519638-f0003.jpg)

Box plot of serum etonogestrel concentrations for participants with the CYP3A7 wild-type genotype versus CYP3A7*1C variant carriers. The box represents the first and third quartiles with the band inside the box representing the median for each respective group of participants. The whiskers represent the data within 1.5 interquartile range of the upper and lower quartile. The red dotted line represents a serum etonogestrel concentration of 90 pg/mL. Circles indicate outliers with values between 1.5 and 3 times the IQR and asterisks indicate outliers with values greater than 3 times the IQR. The distribution of serum etonogestrel concentrations significantly differs between the groups (Mann-Whitney U Test, P=.003).

### Figure 4:

![Figure 4:](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5369/6448146/53d50d5f4924/nihms-1519638-f0004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6448146_nihms-1519638-f0004.jpg)

Box plot of serum etonogestrel concentrations by rs2461817 genotype in the NR1I2 (PXR) gene. Participants are split by the number of rs2461817 alleles with homozygous WT having no variant alleles, heterozygous A/C having one variant allele, and homozygous C/C having two variant alleles. The boxes represent the first and third quartiles with the band inside the box representing the median for each respective group of participants. The whiskers represent the data within 1.5 IQR of the upper and lower quartile. Circles indicate outliers with values between 1.5 and 3 times the IQR and asterisks indicate outliers with values greater than 3 times the IQR.

### Figure 5:

![Figure 5:](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5369/6448146/87880447ee32/nihms-1519638-f0005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=6448146_nihms-1519638-f0005.jpg)

Box plot of serum etonogestrel concentrations for participants with the progesterone receptor rs537681 wild-type genotype compared with progesterone receptor rs537681 variant carriers. The box represents the first and third quartiles with the band inside the box representing the median for each respective group of participants. The whiskers represent the data within 1.5 IQR of the upper and lower quartile. The red dotted line represents a serum etonogestrel concentration of 90 pg/mL. Circles indicate outliers with values between 1.5 and 3 times the IQR and asterisks indicate outliers with values greater than 3 times the IQR. The distribution of serum etonogestrel concentrations significantly differs between the groups (Mann-Whitney U Test, P=.002).

## Discussion

In this large and diverse group of subdermal etonogestrel implant users, we found two patient characteristics and three genetic variants associated with serum etonogestrel concentrations. BMI and duration of implant use remained significantly associated with decreased serum etonogestrel concentrations using our conservative threshold. However, for the majority of women, the effects of increasing BMI and longer duration of implant use will not cause their serum etonogestrel concentrations to fall below the level needed for consistent ovulatory suppression (90pg/mL). For every 1kg/m^2^ increase in BMI, the serum etonogestrel concentration will decrease on average 3pg/mL. Similarly, for every month of implant use past 12 months, the serum etonogestrel concentration will decrease on average by 1.6pg/mL. Though these decreases are not clinically significant on their own, they may contribute to clinically significant effects in combination with other factors, such as genetics.

Although ultimately not statistically significant in our final model, two genetic variants (*CYP3A7*1C* and *PGR* rs537681) were associated with decreased serum etonogestrel concentrations. Carriers of *CYP3A7*1C* had, on average, 23% (35 pg/mL) lower etonogestrel concentrations than participants with the wild-type genotype based upon our generalized linear model findings. CYP3A7 is a fetal cytochrome P450 enzyme that is not typically expressed in adults. Individuals with the *CYP3A7*1C* allele, however, demonstrate expression of this fetal enzyme, which results in metabolically active enzymes that increase metabolism of steroid hormones such as estrogen and progestin ([30](#R30)).

Prior studies have demonstrated that *CYP3A7*1C* carriers have 45.3% lower urinary estrone glucuronide levels and decreased 2- or 4- hydroxylation estrogen metabolites due to bias towards the 16α-hydroxylation pathway catalyzed by CYP3A7 ([31](#R31), [32](#R32)). This allele has also been associated with increased breast cancer mortality, all-cause mortality in lung cancer patients, and chronic lymphocytic leukemia progression due to suspected increased drug metabolism of chemotherapeutic agents that are CYP3A substrates ([31](#R31)).

Our findings reaffirm that the expression of fetal CYP3A7 enzymes in *CYP3A7*1C* carriers is associated with increased steroid hormone metabolism, specifically etonogestrel in contraceptive implant users. In further support of this metabolic association, 27.8% (5/18) of *CYP3A7*1C* carriers had etonogestrel concentrations that fell below 90pg/mL, which is the manufacturer’s reported threshold for consistent ovulatory suppression with the implant ([13](#R13)). Among participants with the respective wild-type genotype, only 9% (30/332) had serum etonogestrel concentrations that similarly fell below 90pg/mL. Furthermore, based on the definition of bioequivalence, the mean serum etonogestrel concentration of *CYP3A7*1C* carriers falls outside the acceptable range when compared to participants with the wild-type genotype (77% versus acceptable range of 80–125%) ([12](#R12)).

Many theorize that other secondary contraceptive methods of action (e.g. thickened cervical mucus) remain protective under this threshold, which may explain the rarity of unintended pregnancies in contraceptive implant users. However, our finding is particularly concerning for *CYP3A7*1C* carriers who use low-dose hormonal contraceptive methods (e.g low-dose combined hormonal pills and progestin-only pills), which have poorer contraceptive efficacies and are potentially more susceptible to pharmacologic failures.

The other genetic variant we found that was associated with decreased serum etonogestrel concentrations, rs537681, does not currently have a known affect on gene expression or function ([2](#R2)). This variant is located on chromosome 11 in the intronic region of the progesterone receptor (*PGR)* gene, which is related to steroid hormone function, but not metabolism ([28](#R28)). However, this variant may be in linkage disequilibrium with a metabolically related single nucleotide polymorphism or we may have only found this association by chance. In support of the latter, only 11.7% (31/266) of carriers for rs537681 had serum etonogestrel concentrations that fell below 90pg/mL (7.3% in participants with the respective wild-type genotype) and the difference in mean serum etonogestrel concentrations (19.9%) falls within the acceptable range for bioequivalence ([12](#R12)).

Unlike the other two variants discussed, the final genetic variant in our model (rs2461817) was associated with increased serum etonogestrel concentrations. This genetic variant is located on chromosome 3 in the *NR1I2 (PXR)* gene, which is involved in CYP enzyme transcription and regulation. One prior pharmacogenomic study that examined patients with epilepsy taking carbamazepine found that patients with the homozygous variant genotype for rs2461817 had increased carbamazepine clearance compared to patients with the heterozygous or homozygous wild-type genotype ([4](#R4)). Our findings, on the other hand, demonstrate an increase in serum etonogestrel concentrations in participants having one or two rs2461817 variant alleles (9% increase per variant allele) compared with participants with the homozygous wild-type genotype. Although this association was ultimately not statistically significant, our findings may be indicative of potential decreased CYP enzyme activity with an additive effect of each rs2461817 variant allele. Though carbamazepine is a known CYP3A4 substrate, it is also a CYP3A4 inducer and has autoinduction properties that complicate its pharmacokinetics, which could potentially contribute to this disparate effect of rs2461718 with carbamazepine ([19](#R19)). The rs2461817 variant is an intronic single nucleotide polymorphism and may be linked to another variant that alters the function of the NR1I2 (PXR) protein in such a way that decreases transcription of CYP enzymes involved in steroid hormone metabolism.

The *CYP3A4* rs55785340 variant was the only *CYP3A4* or *CYP3A5* variant associated with serum etonogestrel concentrations, but was found in only one participant (heterozygote) and previous pharmacogenomic studies have found no association between this variant and metabolism of other CYP3A4 substrates (e.g. axitinib) ([33](#R33)). Furthermore, despite good prevalence in our participants of other *CYP3A4* and *CYP3A5* variants with well published effects on drug metabolism (e.g. rs776746 [CYP3A5*3] and rs35599367 [CYP3A4*22]), these *CYP3A4or CYP3A5* variants did not have significant associations with serum etonogestrel concentrations even using a less conservative significance threshold ([3](#R3)). A recent review of the oral contraceptive drug-drug interaction literature by Zhang et al. found similarly that CYP3A4 appears to have a limited role in the metabolism of the steroid hormone components of oral contraceptives ([19](#R19)). These findings challenge the current dogma that CYP3A4 is the primary metabolizing pathway for steroid hormones and this may explain the lack of effect these variants had on etonogestrel metabolism in our cohort. Despite identifying three novel genetic variants associated with altered steroid hormone metabolism, our best explanatory model accounts for less than 20% of the variability seen in the metabolism of the etonogestrel steroid hormone. Given the lack of association between *CYP3A4 or CYP3A5* variants and etonogestrel metabolism, novel genetic loci potentially involved in different phases of steroid hormone metabolism may account for some of this remaining variability.

The major strength of this study was using etonogestrel contraceptive implant users for our pharmacokinetic outcome. By analyzing a medication with a controlled-release rate and steady-state pharmacokinetics, we were able to efficiently capture each individual’s pharmacokinetic profile with only a single etonogestrel measurement during the relative steady-state period between 12 and 36 months ([13](#R13), [17](#R17)). Our study population was also relatively young and this age group represents a time of relatively normal physiological metabolism as compared to the developing metabolism in childhood/adolescence and the altered metabolism associated with advanced age ([18](#R18)). Thus, the findings of our study are less likely to be confounded by age-related changes in drug metabolism. Additionally, we chose genetic variants based on enzymes and proteins involved in the known steroid hormone metabolism pathway, thus increasing our power to find associations with etonogestrel metabolism as compared to a genome-wide exploratory approach.

Our study’s primary limitation, however, is our use of a candidate gene approach. By limiting our investigation to variants in pre-selected genes, we could not identify novel genetic loci with potential effects on steroid hormone metabolism. This study was also not powered for rare variants (prevalence <5%) which may have significant associations with serum etonogestrel concentrations but were not prevalent enough in our population. Our population was predominantly White (Caucasian) with a large Hispanic or Latina ethnicity, which further limited our ability to capture genetic variants more commonly found in differentially admixed populations. Some genetic variants with known effects on the metabolism of CYP3A4 and CYP3A5 substrates have only been found in small, non-Caucasian populations, and thus we would likely not find these variant alleles in our population ([22](#R22)).

Pharmacogenomics has the potential to dramatically alter the field of women’s health, especially in light of the social, financial, and emotional consequences of a contraceptive failure and the breadth of indications for steroid hormones throughout a woman’s life-span. Our data demonstrates that women who are *CYP3A7*1C* carriers may be at increased risk of contraceptive failure, especially with lower dose hormonal methods. Women also take steroid hormone medications for various medical indications throughout their lives (e.g. abnormal bleeding, preterm labor prevention, hormonal replacement therapy) and future investigations should evaluate the influence of the *CYP3A7*1C* genotype on the efficacy of steroid hormones for these other indications. As more genetic data become available, clinicians may need to consider adding genetic predisposition to increased steroid hormone metabolism in their differential diagnosis for unintended pregnancies in women reporting perfect adherence to hormonal contraceptive methods. The assumption that these unintended pregnancies are solely due to medication adherence issues may adversely affect clinician-patient relationships for women with genetic factors that result in lower steroid hormone levels. Our findings can contribute to the development of precision medicine tools and inform individually tailored provision of women’s healthcare that improve both patient satisfaction and clinical outcomes.

## Supplementary Material

## Acknowledgements:

The authors thank Dr. Serge Cremers at the Biomarkers Core Laboratory at Columbia University for assisting with the etonogestrel analysis and Dr. Derek Warner at the Genomics Core Facility at the University of Utah for assisting with the genotyping.

**Disclosure of Funding:** This work was primarily supported by the Society of Family Planning Research Fund [grant number SFPRF17–3]. This work was also supported by NIH/NCATS Colorado CTSA Grant Number UL1 TR001082. Contents are the authors’ sole responsibility and do not necessarily represent official NIH views.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Caudle KE, Dunnenberger HM, Freimuth RR, Peterson JF, Burlison JD, Whirl-Carrillo M, et al. Standardizing terms for clinical pharmacogenetic test results: consensus terms from the Clinical Pharmacogenetics Implementation Consortium (CPIC). Genet Med 2017. February;19(2):215–23.  [DOI](https://doi.org/10.1038/gim.2016.87) | [PMC free article](/articles/PMC5253119/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27441996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=Standardizing%20terms%20for%20clinical%20pharmacogenetic%20test%20results:%20consensus%20terms%20from%20the%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)&author=KE%20Caudle&author=HM%20Dunnenberger&author=RR%20Freimuth&author=JF%20Peterson&author=JD%20Burlison&volume=19&issue=2&publication_year=2017&pages=215-23&pmid=27441996&doi=10.1038/gim.2016.87&)

2. Whirl-Carrillo M, McDonagh EM, Hebert JM, Gong L, Sangkuhl K, Thorn CF, et al. Pharmacogenomics knowledge for personalized medicine. Clin Pharmacol Ther 2012. October;92(4):414–7.  [DOI](https://doi.org/10.1038/clpt.2012.96) | [PMC free article](/articles/PMC3660037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22992668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacogenomics%20knowledge%20for%20personalized%20medicine&author=M%20Whirl-Carrillo&author=EM%20McDonagh&author=JM%20Hebert&author=L%20Gong&author=K%20Sangkuhl&volume=92&issue=4&publication_year=2012&pages=414-7&pmid=22992668&doi=10.1038/clpt.2012.96&)

3. Deininger KM, Vu A, Page RL 2nd, Ambardekar AV, Lindenfeld J, Aquilante CL. CYP3A pharmacogenetics and tacrolimus disposition in adult heart transplant recipients. Clin Transplant 2016. September;30(9):1074–81.  [DOI](https://doi.org/10.1111/ctr.12790) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27314545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transplant&title=CYP3A%20pharmacogenetics%20and%20tacrolimus%20disposition%20in%20adult%20heart%20transplant%20recipients&author=KM%20Deininger&author=A%20Vu&author=RL%20Page&author=AV%20Ambardekar&author=J%20Lindenfeld&volume=30&issue=9&publication_year=2016&pages=1074-81&pmid=27314545&doi=10.1111/ctr.12790&)

4. Puranik YG, Birnbaum AK, Marino SE, Ahmed G, Cloyd JC, Remmel RP, et al. Association of carbamazepine major metabolism and transport pathway gene polymorphisms and pharmacokinetics in patients with epilepsy. Pharmacogenomics 2013. January;14(1):35–45.  [DOI](https://doi.org/10.2217/pgs.12.180) | [PMC free article](/articles/PMC3570048/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23252947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Association%20of%20carbamazepine%20major%20metabolism%20and%20transport%20pathway%20gene%20polymorphisms%20and%20pharmacokinetics%20in%20patients%20with%20epilepsy&author=YG%20Puranik&author=AK%20Birnbaum&author=SE%20Marino&author=G%20Ahmed&author=JC%20Cloyd&volume=14&issue=1&publication_year=2013&pages=35-45&pmid=23252947&doi=10.2217/pgs.12.180&)

5. Sukasem C, Chamnanphon M, Koomdee N, Santon S, Jantararoungtong T, Prommas S, et al. Pharmacogenetics and clinical biomarkers for subtherapeutic plasma efavirenz concentration in HIV-1 infected Thai adults. Drug Metab Pharmacokinet 2014;29(4):289–95.  [DOI](https://doi.org/10.2133/dmpk.dmpk-13-rg-077) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24477223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Pharmacogenetics%20and%20clinical%20biomarkers%20for%20subtherapeutic%20plasma%20efavirenz%20concentration%20in%20HIV-1%20infected%20Thai%20adults&author=C%20Sukasem&author=M%20Chamnanphon&author=N%20Koomdee&author=S%20Santon&author=T%20Jantararoungtong&volume=29&issue=4&publication_year=2014&pages=289-95&pmid=24477223&doi=10.2133/dmpk.dmpk-13-rg-077&)

6. Caudle KE, Klein TE, Hoffman JM, Muller DJ, Whirl-Carrillo M, Gong L, et al. Incorporation of pharmacogenomics into routine clinical practice: the Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline development process. Current drug metabolism 2014;15(2):209–17.  [DOI](https://doi.org/10.2174/1389200215666140130124910) | [PMC free article](/articles/PMC3977533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24479687/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Current%20drug%20metabolism&title=Incorporation%20of%20pharmacogenomics%20into%20routine%20clinical%20practice:%20the%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guideline%20development%20process&author=KE%20Caudle&author=TE%20Klein&author=JM%20Hoffman&author=DJ%20Muller&author=M%20Whirl-Carrillo&volume=15&issue=2&publication_year=2014&pages=209-17&pmid=24479687&doi=10.2174/1389200215666140130124910&)

7. Daniels K, Daugherty J, Jones J. Current contraceptive status among women aged 15–44: United States, 2011–2013. NCHS data brief 2014. December(173):1–8.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/25500343/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=NCHS%20data%20brief&title=Current%20contraceptive%20status%20among%20women%20aged%2015%E2%80%9344:%20United%20States,%202011%E2%80%932013&author=K%20Daniels&author=J%20Daugherty&author=J%20Jones&issue=173&publication_year=2014&pages=1-8&pmid=25500343&)

8. Fotherby K Variability of pharmacokinetic parameters for contraceptive steroids. Journal of steroid biochemistry 1983. July;19(1c):817–20.  [DOI](https://doi.org/10.1016/0022-4731(83)90017-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6887900/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20steroid%20biochemistry&title=Variability%20of%20pharmacokinetic%20parameters%20for%20contraceptive%20steroids&author=K%20Fotherby&volume=19&issue=1c&publication_year=1983&pages=817-20&pmid=6887900&doi=10.1016/0022-4731(83)90017-1&)

9. McNicholas C, Swor E, Wan L, Peipert JF. Prolonged use of the etonogestrel implant and levonorgestrel intrauterine device: 2 years beyond Food and Drug Administration-approved duration. American journal of obstetrics and gynecology 2017. June;216(6):586.e1–.e6.  [DOI](https://doi.org/10.1016/j.ajog.2017.01.036) | [PMC free article](/articles/PMC5449254/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28147241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20obstetrics%20and%20gynecology&title=Prolonged%20use%20of%20the%20etonogestrel%20implant%20and%20levonorgestrel%20intrauterine%20device:%202%20years%20beyond%20Food%20and%20Drug%20Administration-approved%20duration&author=C%20McNicholas&author=E%20Swor&author=L%20Wan&author=JF%20Peipert&volume=216&issue=6&publication_year=2017&pages=586.e1-.e6&pmid=28147241&doi=10.1016/j.ajog.2017.01.036&)

10. Westhoff CL, Torgal AH, Mayeda ER, Pike MC, Stanczyk FZ. Pharmacokinetics of a combined oral contraceptive in obese and normal-weight women. Contraception 2010. June;81(6):474–80.  [DOI](https://doi.org/10.1016/j.contraception.2010.01.016) | [PMC free article](/articles/PMC3522459/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20472113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Contraception&title=Pharmacokinetics%20of%20a%20combined%20oral%20contraceptive%20in%20obese%20and%20normal-weight%20women&author=CL%20Westhoff&author=AH%20Torgal&author=ER%20Mayeda&author=MC%20Pike&author=FZ%20Stanczyk&volume=81&issue=6&publication_year=2010&pages=474-80&pmid=20472113&doi=10.1016/j.contraception.2010.01.016&)

11. Guidance for Industry: Bioavailability and Bioequivalence Studies for Orally Administered Drug Products - General Considerations. 2002. [cited; Available from: https://www.fda.gov/downloads/Drugs/DevelopmentApprovalProcess/HowDrugsareDevelopedandApproved/ApprovalApplications/AbbreviatedNewDrugApplicationANDAGenerics/UCM154838.pdf  [https://www.fda.gov/downloads/Drugs/DevelopmentApprovalProcess/HowDrugsareDevelopedandApproved/ApprovalApplications/AbbreviatedNewDrugApplicationANDAGenerics/UCM154838.pdf](https://www.fda.gov/downloads/Drugs/DevelopmentApprovalProcess/HowDrugsareDevelopedandApproved/ApprovalApplications/AbbreviatedNewDrugApplicationANDAGenerics/UCM154838.pdf)

12. ACOG Committee Opinion No. 375: Brand versus generic oral contraceptives. Obstet Gynecol 2007. August;110(2 Pt 1):447–8.  [DOI](https://doi.org/10.1097/01.AOG.0000263922.08190.3b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17666623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Obstet%20Gynecol&title=ACOG%20Committee%20Opinion%20No.%20375:%20Brand%20versus%20generic%20oral%20contraceptives&volume=110&issue=2%20Pt%201&publication_year=2007&pages=447-8&pmid=17666623&doi=10.1097/01.AOG.0000263922.08190.3b&)

13. Hatcher RA, Trussell J, Stewart F, Nelson A, Cates W, Guest F, et al. Contraceptive technology. New York: Ardent Media: Inc; 2011.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Contraceptive%20technology&author=RA%20Hatcher&author=J%20Trussell&author=F%20Stewart&author=A%20Nelson&author=W%20Cates&publication_year=2011&)

14. Anderson PL, Lamba J, Aquilante CL, Schuetz E, Fletcher CV. Pharmacogenetic characteristics of indinavir, zidovudine, and lamivudine therapy in HIV-infected adults: a pilot study. J Acquir Immune Defic Syndr 2006. August 1;42(4):441–9.  [DOI](https://doi.org/10.1097/01.qai.0000225013.53568.69) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16791115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Pharmacogenetic%20characteristics%20of%20indinavir,%20zidovudine,%20and%20lamivudine%20therapy%20in%20HIV-infected%20adults:%20a%20pilot%20study&author=PL%20Anderson&author=J%20Lamba&author=CL%20Aquilante&author=E%20Schuetz&author=CV%20Fletcher&volume=42&issue=4&publication_year=2006&pages=441-9&pmid=16791115&doi=10.1097/01.qai.0000225013.53568.69&)

15. Tavira B, Coto E, Diaz-Corte C, Alvarez V, Lopez-Larrea C, Ortega F. A search for new CYP3A4 variants as determinants of tacrolimus dose requirements in renal-transplanted patients. Pharmacogenet Genomics 2013. August;23(8):445–8.  [DOI](https://doi.org/10.1097/FPC.0b013e3283636856) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23778326/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=A%20search%20for%20new%20CYP3A4%20variants%20as%20determinants%20of%20tacrolimus%20dose%20requirements%20in%20renal-transplanted%20patients&author=B%20Tavira&author=E%20Coto&author=C%20Diaz-Corte&author=V%20Alvarez&author=C%20Lopez-Larrea&volume=23&issue=8&publication_year=2013&pages=445-8&pmid=23778326&doi=10.1097/FPC.0b013e3283636856&)

16. Daniels K, Mosher WD. Contraceptive methods women have ever used: United States, 1982–2010. National health statistics reports 2013. February 14(62):1–15.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/24988816/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=National%20health%20statistics%20reports&title=Contraceptive%20methods%20women%20have%20ever%20used:%20United%20States,%201982%E2%80%932010&author=K%20Daniels&author=WD%20Mosher&volume=14&issue=62&publication_year=2013&pages=1-15&pmid=24988816&)

17. Le J, Tsourounis C. Implanon: a critical review. The Annals of pharmacotherapy 2001. March;35(3):329–36.  [DOI](https://doi.org/10.1345/aph.10149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11261531/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Annals%20of%20pharmacotherapy&title=Implanon:%20a%20critical%20review&author=J%20Le&author=C%20Tsourounis&volume=35&issue=3&publication_year=2001&pages=329-36&pmid=11261531&doi=10.1345/aph.10149&)

18. Wilson K, Hanson J. The effects of extremes of age on drug action. Methods Find Exp Clin Pharmacol 1980. December;2(6):303–12.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/6152996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Find%20Exp%20Clin%20Pharmacol&title=The%20effects%20of%20extremes%20of%20age%20on%20drug%20action&author=K%20Wilson&author=J%20Hanson&volume=2&issue=6&publication_year=1980&pages=303-12&pmid=6152996&)

19. Zhang N, Shon J, Kim MJ, Yu C, Zhang L, Huang SM, et al. Role of CYP3A in Oral Contraceptives Clearance. Clin Transl Sci 2018. May;11(3):251–60.  [DOI](https://doi.org/10.1111/cts.12499) | [PMC free article](/articles/PMC5944580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28986954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Role%20of%20CYP3A%20in%20Oral%20Contraceptives%20Clearance&author=N%20Zhang&author=J%20Shon&author=MJ%20Kim&author=C%20Yu&author=L%20Zhang&volume=11&issue=3&publication_year=2018&pages=251-60&pmid=28986954&doi=10.1111/cts.12499&)

20. Morrell KM, Cremers S, Westhoff CL, Davis AR. Relationship between etonogestrel level and BMI in women using the contraceptive implant for more than 1 year. Contraception 2016. March;93(3):263–5.  [DOI](https://doi.org/10.1016/j.contraception.2015.11.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26577754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Contraception&title=Relationship%20between%20etonogestrel%20level%20and%20BMI%20in%20women%20using%20the%20contraceptive%20implant%20for%20more%20than%201%20year&author=KM%20Morrell&author=S%20Cremers&author=CL%20Westhoff&author=AR%20Davis&volume=93&issue=3&publication_year=2016&pages=263-5&pmid=26577754&doi=10.1016/j.contraception.2015.11.005&)

21. Thomas T, Petrie K, Shim J, Abildskov KM, Westhoff CL, Cremers S. A UPLC-MS/MS method for therapeutic drug monitoring of etonogestrel. Ther Drug Monit 2013. December;35(6):844–8.  [DOI](https://doi.org/10.1097/FTD.0b013e31829a10fa) | [PMC free article](/articles/PMC3838448/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24081205/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=A%20UPLC-MS/MS%20method%20for%20therapeutic%20drug%20monitoring%20of%20etonogestrel&author=T%20Thomas&author=K%20Petrie&author=J%20Shim&author=KM%20Abildskov&author=CL%20Westhoff&volume=35&issue=6&publication_year=2013&pages=844-8&pmid=24081205&doi=10.1097/FTD.0b013e31829a10fa&)

22. Kitts A, Phan L, Ward M, Holmes JB. The Database of Short Genetic Variation (dbSNP). 2013. June 30 [Updated 2014 Apr 3] [cited; 2nd edition:[In: The NCBI Handbook [Internet]]. Available from: https://www.ncbi.nlm.nih.gov/books/NBK174586/  [https://www.ncbi.nlm.nih.gov/books/NBK174586/](https://www.ncbi.nlm.nih.gov/books/NBK174586/)

23. TaqMan® SNP Genotyping Assays: User Guide. ThermoFisher Scientific: Life Technologies Coporation. 2014. Updated: 2017 Sept 29 [cited; Revision B.0:[Available from: https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0009593_TaqManSNP_UG.pdf  [https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0009593_TaqManSNP_UG.pdf](https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0009593_TaqManSNP_UG.pdf)

24. Feng C, Wang H, Lu N, Chen T, He H, Lu Y, et al. Log-transformation and its implications for data analysis. Shanghai archives of psychiatry 2014. April;26(2):105–9.  [DOI](https://doi.org/10.3969/j.issn.1002-0829.2014.02.009) | [PMC free article](/articles/PMC4120293/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25092958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Shanghai%20archives%20of%20psychiatry&title=Log-transformation%20and%20its%20implications%20for%20data%20analysis&author=C%20Feng&author=H%20Wang&author=N%20Lu&author=T%20Chen&author=H%20He&volume=26&issue=2&publication_year=2014&pages=105-9&pmid=25092958&doi=10.3969/j.issn.1002-0829.2014.02.009&)

25. Burnham KP, Anderson DR. Multimodel Inference:Understanding AIC and BIC in Model Selection. Sociological Methods & Research 2004;33(2):261–304.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sociological%20Methods%20&%20Research&title=Multimodel%20Inference:Understanding%20AIC%20and%20BIC%20in%20Model%20Selection&author=KP%20Burnham&author=DR%20Anderson&volume=33&issue=2&publication_year=2004&pages=261-304&)

26. Hozo SP, Djulbegovic B, Hozo I. Estimating the mean and variance from the median, range, and the size of a sample. BMC Med Res Methodol 2005. April 20;5:13.  [DOI](https://doi.org/10.1186/1471-2288-5-13) | [PMC free article](/articles/PMC1097734/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15840177/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Res%20Methodol&title=Estimating%20the%20mean%20and%20variance%20from%20the%20median,%20range,%20and%20the%20size%20of%20a%20sample&author=SP%20Hozo&author=B%20Djulbegovic&author=I%20Hozo&volume=5&publication_year=2005&pages=13&pmid=15840177&doi=10.1186/1471-2288-5-13&)

27. Cox DG, Kraft P. Quantification of the power of Hardy-Weinberg equilibrium testing to detect genotyping error. Hum Hered 2006;61(1):10–4.  [DOI](https://doi.org/10.1159/000091787) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16514241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Hered&title=Quantification%20of%20the%20power%20of%20Hardy-Weinberg%20equilibrium%20testing%20to%20detect%20genotyping%20error&author=DG%20Cox&author=P%20Kraft&volume=61&issue=1&publication_year=2006&pages=10-4&pmid=16514241&doi=10.1159/000091787&)

28. Huusko JM, Karjalainen MK, Graham BE, Zhang G, Farrow EG, Miller NA, et al. Whole exome sequencing reveals HSPA1L as a genetic risk factor for spontaneous preterm birth. PLoS Genet 2018. July;14(7):e1007394.  [DOI](https://doi.org/10.1371/journal.pgen.1007394) | [PMC free article](/articles/PMC6042692/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30001343/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Whole%20exome%20sequencing%20reveals%20HSPA1L%20as%20a%20genetic%20risk%20factor%20for%20spontaneous%20preterm%20birth&author=JM%20Huusko&author=MK%20Karjalainen&author=BE%20Graham&author=G%20Zhang&author=EG%20Farrow&volume=14&issue=7&publication_year=2018&pages=e1007394&pmid=30001343&doi=10.1371/journal.pgen.1007394&)

29. Zerbino DR, Achuthan P, Akanni W, Amode MR, Barrell D, Bhai J, et al. Ensembl 2018. Nucleic Acids Res 2018;46(D1):D754–D61.  [DOI](https://doi.org/10.1093/nar/gkx1098) | [PMC free article](/articles/PMC5753206/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29155950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Ensembl%202018&author=DR%20Zerbino&author=P%20Achuthan&author=W%20Akanni&author=MR%20Amode&author=D%20Barrell&volume=46&issue=D1&publication_year=2018&pages=D754-D61&pmid=29155950&doi=10.1093/nar/gkx1098&)

30. Sim SC, Edwards RJ, Boobis AR, Ingelman-Sundberg M. CYP3A7 protein expression is high in a fraction of adult human livers and partially associated with the CYP3A7*1C allele. Pharmacogenet Genomics 2005. September;15(9):625–31.  [DOI](https://doi.org/10.1097/01.fpc.0000171516.84139.89) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16041241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=CYP3A7%20protein%20expression%20is%20high%20in%20a%20fraction%20of%20adult%20human%20livers%20and%20partially%20associated%20with%20the%20CYP3A7*1C%20allele&author=SC%20Sim&author=RJ%20Edwards&author=AR%20Boobis&author=M%20Ingelman-Sundberg&volume=15&issue=9&publication_year=2005&pages=625-31&pmid=16041241&doi=10.1097/01.fpc.0000171516.84139.89&)

31. Johnson N, De Ieso P, Migliorini G, Orr N, Broderick P, Catovsky D, et al. Cytochrome P450 Allele CYP3A7*1C Associates with Adverse Outcomes in Chronic Lymphocytic Leukemia, Breast, and Lung Cancer. Cancer Res 2016. March 15;76(6):1485–93.  [DOI](https://doi.org/10.1158/0008-5472.CAN-15-1410) | [PMC free article](/articles/PMC4795533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26964624/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Cytochrome%20P450%20Allele%20CYP3A7*1C%20Associates%20with%20Adverse%20Outcomes%20in%20Chronic%20Lymphocytic%20Leukemia,%20Breast,%20and%20Lung%20Cancer&author=N%20Johnson&author=P%20De%20Ieso&author=G%20Migliorini&author=N%20Orr&author=P%20Broderick&volume=76&issue=6&publication_year=2016&pages=1485-93&pmid=26964624&doi=10.1158/0008-5472.CAN-15-1410&)

32. Sood D, Johnson N, Jain P, Siskos AP, Bennett M, Gilham C, et al. CYP3A7*1C allele is associated with reduced levels of 2-hydroxylation pathway oestrogen metabolites. Br J Cancer 2017. January;116(3):382–8.  [DOI](https://doi.org/10.1038/bjc.2016.432) | [PMC free article](/articles/PMC5294487/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28072767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=CYP3A7*1C%20allele%20is%20associated%20with%20reduced%20levels%20of%202-hydroxylation%20pathway%20oestrogen%20metabolites&author=D%20Sood&author=N%20Johnson&author=P%20Jain&author=AP%20Siskos&author=M%20Bennett&volume=116&issue=3&publication_year=2017&pages=382-8&pmid=28072767&doi=10.1038/bjc.2016.432&)

33. Brennan M, Williams JA, Chen Y, Tortorici M, Pithavala Y, Liu YC. Meta-analysis of contribution of genetic polymorphisms in drug-metabolizing enzymes or transporters to axitinib pharmacokinetics. Eur J Clin Pharmacol 2012. May;68(5):645–55.  [DOI](https://doi.org/10.1007/s00228-011-1171-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22170007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Meta-analysis%20of%20contribution%20of%20genetic%20polymorphisms%20in%20drug-metabolizing%20enzymes%20or%20transporters%20to%20axitinib%20pharmacokinetics&author=M%20Brennan&author=JA%20Williams&author=Y%20Chen&author=M%20Tortorici&author=Y%20Pithavala&volume=68&issue=5&publication_year=2012&pages=645-55&pmid=22170007&doi=10.1007/s00228-011-1171-8&)
