# Genome-wide association study identifies a regulatory variant of RGMA associated with opioid dependence in European Americans

## Metadata
**Authors:** Zhongshan Cheng, Hang Zhou, Richard Sherva, Lindsay Farrer, Henry R Kranzler, Joel Gelernter
**Journal:** Biological psychiatry
**Date:** 2018 Jan 11
**DOI:** [10.1016/j.biopsych.2017.12.016](https://doi.org/10.1016/j.biopsych.2017.12.016)
**PMID:** 29478698
**PMCID:** PMC6041180
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6041180/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC6041180/pdf/nihms933995.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC6041180/pdf/nihms933995.pdf)

## Abstract

**Background:** 
Opioid dependence (OD) is at epidemic levels in the United States. Genetic studies can provide insight into its biology.

**Methods:** 
We completed an OD genome-wide association study (GWAS) in 3058 opioid-exposed European-Americans, 1290 of whom met criteria for a DSM-IV diagnosis of OD. Analysis used DSM-IV criterion count.

**Results:** 
By meta-analysis of four cohorts, Yale-Penn 1 (n=1388), Yale-Penn 2 (n=996), Yale-Penn 3 (n=98) and SAGE (n=576), we identified a variant on chromosome 15, rs12442183, near RGMA (repulsive guidance molecule A (RGMa)), associated with OD (P=1.3×10-8). The association was also genome-wide significant (GWS) in Yale-Penn 1 taken individually, and nominally significant in two of the other three samples. The finding was further supported in a meta-analysis of all available opioid-exposed African-Americans (n=2014, 1106 meeting DSM-IV OD criteria; P=3.0×10-3) from three cohorts; there was nominal significance in two of these samples. Thus, of the seven subsamples examined in two populations, one was GWS and four of six were nominally significant. RGMa is a CNS axon guidance protein. Risk allele rs12442183*T was correlated with higher expression of a specific RGMA transcript variant in frontal cortex (P=2×10-3). After chronic morphine injection, the homologous mouse gene (Rgma) was upregulated in C57BL/6J striatum. Co-expression analysis of 1301 brain samples revealed that RGMA mRNA expression was associated with that of four genes implicated in other psychiatric disorders, including GRIN1.

**Conclusion:** 
This is the first study to demonstrate an association of RGMA with OD. It provides a new lead into our understanding of OD pathophysiology.

Keywords: Genome-wide association, opioid dependence, RGMA, regulatory variant, European-Americans, psychiatric genetics

### Background

Opioid dependence (OD) is at epidemic levels in the United States. Genetic studies can provide insight into its biology.

### Methods

We completed an OD genome-wide association study (GWAS) in 3058 opioid-exposed European-Americans, 1290 of whom met criteria for a DSM-IV diagnosis of OD. Analysis used DSM-IV criterion count.

### Results

By meta-analysis of four cohorts, Yale-Penn 1 (n=1388), Yale-Penn 2 (n=996), Yale-Penn 3 (n=98) and SAGE (n=576), we identified a variant on chromosome 15, rs12442183, near *RGMA* (repulsive guidance molecule A (RGMa)), associated with OD (P=1.3×10^-8^). The association was also genome-wide significant (GWS) in Yale-Penn 1 taken individually, and nominally significant in two of the other three samples. The finding was further supported in a meta-analysis of all available opioid-exposed African-Americans (n=2014, 1106 meeting DSM-IV OD criteria; P=3.0×10^-3^) from three cohorts; there was nominal significance in two of these samples. Thus, of the seven subsamples examined in two populations, one was GWS and four of six were nominally significant. RGMa is a CNS axon guidance protein. Risk allele rs12442183*T was correlated with higher expression of a specific *RGMA* transcript variant in frontal cortex (P=2×10^-3^). After chronic morphine injection, the homologous mouse gene (*Rgma*) was upregulated in C57BL/6J striatum. Co-expression analysis of 1301 brain samples revealed that *RGMA* mRNA expression was associated with that of four genes implicated in other psychiatric disorders, including *GRIN1*.

### Conclusion

This is the first study to demonstrate an association of *RGMA* with OD. It provides a new lead into our understanding of OD pathophysiology.

**Keywords:** Genome-wide association, opioid dependence, *RGMA*, regulatory variant, European-Americans, psychiatric genetics

## Introduction

Opioid dependence (OD) is a major cause of medical, legal, and social problems, resulting in increased mortality and decreased productivity. It is presently at epidemic levels in the United States. OD is a genetically complex disorder that is influenced by complex environmental factors ([1](#R1)), such as drug availability, making it difficult to identify genetic factors that underlie the disorder. Genome-wide association study (GWAS) is an unbiased way to search for novel genetic factors contributing to OD. We completed the first GWAS of OD (with DSM-IV criterion count as the phenotype definition, using all available subjects) and found two genome-wide significant single nucleotide polymorphisms (SNPs) in African-Americans (AAs) ([2](#R2)). These two SNPs map to *KCNC1* (potassium voltage-gated channel subfamily C member 1) and *KCNG2* (potassium voltage-gated channel modifier subfamily G member 2), respectively, which are involved in potassium signaling pathways. Pathway-based GWAS analysis also demonstrated an enrichment of genes involved in calcium signaling and long-term potentiation ([2](#R2)). There have been fewer findings in European-Americans (EAs), where the only GWAS to date to identify variants associated to OD in EAs showed an association with *CNIH3* ([3](#R3)). Additionally, we used GWAS to identify a variant close to the *OPRM1* locus that affected opioid dose requirements in AAs ([4](#R4)). These studies demonstrated that the use of an ordinal trait can provide more statistical power than a categorical diagnosis, as it can take account of affection severity.

In the present study, we used an ordinal trait analysis that was limited to OD-exposed individuals in our GWAS. We identified a variant of *RGMA*, which encodes the RGM domain family member A (RGMa, which was associated with OD in EAs at a genome-wide significant (GWS) level. The opioid-exposed EA subjects we studied previously ([2](#R2)) are included in the present study, augmented by additional subjects recruited more recently and we used a new analytic strategy that included only OD-exposed subjects. The finding was then replicated in both EAs and AAs.

## Materials and Methods

### Subjects, Genotyping and Imputation

Subjects in our three cohorts, Yale-Penn 1-3, were recruited from five eastern US sites to participate in studies of the genetics of drug (opioid or cocaine) or alcohol dependence (see [Supplementary Table 1](#SD1) for the specific relationship between the samples used in the present study and our previous study ([2](#R2))). These samples contain unrelated individuals and a small proportion of small nuclear families (SNFs) originally collected for linkage studies. All participants provided written informed consent as approved by the institutional review board at each site. Certificates of confidentiality were issued by the National Institute on Drug Abuse and the National Institute on Alcohol Abuse and Alcoholism.

Subjects are designated as Yale-Penn 1, 2, and 3 by the recruitment epoch and microarray platform used for genotyping. Samples for Yale-Penn 1 (n=5540) were genotyped on the Illumina (San Diego, CA, USA) HumanOmni1-Quad v1.0 microarray at the Center for Inherited Disease Research and the Yale Center for Genome Analysis. A total of ∼1M SNPs were targeted by microarray in Yale-Penn 1. Samples for Yale-Penn 2 (n=3675) were genotyped with the Illumina HumanCore Exome array, which includes a total of ∼0.5M SNPs, including exomic SNPs and tagging SNPs. Genotyping in Yale-Penn 3 (n=592) was performed with the Illumina Multi-Ethnic Genotyping Array, an array targeting ∼1.7M genome-wide markers and optimized for GWAS in populations of diverse ancestries. Yale-Penn 2 and Yale-Penn 3 genotyping data were obtained at the Gelernter Lab at Yale. Quality control (QC) for microarrays in each cohort were carried out using PLINK1.9 ([5](#R5)) based on the following criteria: ([1](#R1)) an individual genotype missing rate <2%, ([2](#R2)) a SNP genotype missing rate <2%, ([3](#R3)) a Hardy-Weinberg P>1×10^-6^, and ([4](#R4)) a minor allele frequency (MAF) >1%. After QC, there were ∼0.70M, ∼0.27M, and ∼1.1M SNPs remaining in Yale-Penn 1-3 respectively, which were subjected to imputation and downstream analysis.

GWAS genotypes from all subjects taken separately in each sample group of Yale-Penn 1-3 were compared with the 1000 Genomes Project phase 1 reference panel ([6](#R6)). Eigensoft ([7](#R7)) was used for principal components analysis. The first 10 PC scores served to differentiate European-Americans (EAs) and African-Americans (AAs) through K-means clustering ([8](#R8)). Here, we present GWAS data only for EAs (our previous study presented numerous positive GWAS results in AAs). For each Yale-Penn cohort, genotypes from EAs and AAs were imputed together with Minimac3 implemented in Michigan Imputation Server ([https://imputationserver.sph.umich.edu/index.html](https://imputationserver.sph.umich.edu/index.html)) with genotyped SNPs and the 1000 Genomes reference phase 3 release that contains haplotypes on 2504 samples (n = 5008 haplotypes for a total of ∼81.2M polymorphic markers). The imputation data for Yale-Penn 1-3 were generated in dosage format separately. Under imputation R^2^>0.3, there was an average of ∼47.1M variants included in dosage data from Yale-Penn 1-3. The dosage genotypes for each cohort were transformed into hard call genotypes with PLINK1.9 ([5](#R5)). Using hard-call genotypes to analyze imputed GWAS genotypes has been reported previously ([9](#R9)). In our study, the dosage genotypes for each cohort were transformed into hard call genotypes with PLINK1.9, the rationale for which was to keep only high-quality dosage data by filtering with stricter QC parameters of genotype probability (GP >=0.9) using Plink1.9. The high-quality hard-call genotyping data are easier to QC with Plink1.9 than dosage data, and the genotype data can be directly used in association tests with GEMMA (Genome-wide Efficient Mixed Model Association) ([10](#R10)) (improving the efficiency of computation), which incorporates and account for the relatedness among our family samples and model the relatedness to increase statistical power. Only genotypes with genotype imputation probability (GP) ≥0.9, an individual genotyping missing rate <5%, MAF >1% and missing call frequency <5% were kept for the association analysis. Following these QC efforts, there were ∼13 M, ∼9.8 M, and ∼5.9 M variants left for Yale-Penn 1-3, respectively.

We also included GWAS data from SAGE (n=4077) (dbGaP study id phs000092.v1.p1) ([11](#R11), [12](#R12)), which were genotyped by the Illumina Human 1M array covering ∼1 M total SNPs. SAGE genotypes were compared with the HapMap3 reference datasets, comprised of CEU, YRI, and CHB populations, and separated into EA and AA populations by using the first PC scores in principal components analysis with Eigensoft. The raw genotypes were imputed with IMPUTE2 ([13](#R13)). The 1000 Genomes phase 1 reference panel ([6](#R6)) containing phased haplotypes for 1092 individuals was used for imputation. Dosage genotypes were transformed into hard call genotypes with a custom perl script for heterozygous and homozygous genotypes, i.e., dosage probability for heterozygous genotypes were between 0.8 and 1.4 and homozygous genotypes of reference and alternative alleles ranged from 0 to 0.4 and 1.6 to 2, respectively, with others assigned as missing. The QC criteria for the individual genotyping missing rate <5%, MAF >1% and missing call frequency <5% were also utilized to filter GWAS data. After QC, ∼14 M variants were retained for meta-analysis with Yale-Penn 1-3.

### Analytic Methods

The sum of DSM-IV criterion counts for all EA OD-exposed individuals from the Yale-Penn 1-3 samples were used as an ordinal (quantitative) trait in association analysis ([Supplementary figure 1](#SD1)). GEMMA ([10](#R10)) was performed for each cohort separately, with an adjustment for sex, age, the first three PCs of ancestry, and the degree of relatedness among subjects. In our previous study, we also adjusted for the number of other substance dependence criteria, but did not do so here. This was intended to capture more OD information, at the cost of possible confounding by association driven in part by substance use comorbidities. Summary statistics from GEMMA were reformatted for PLINK1.9 for meta-analysis, and all variants were evaluated by matching their chromosomal positions and two alleles among the four GWAS datasets. Variants with conflicting allele information that could not be resolved by being flipped were excluded from the meta-analysis that used PLINK1.9. The inverse variance method implemented in PLINK1.9 was used to generate fixed-effects meta-analysis P-values (meta-P). In the above process, ∼8 M distinct variants present in at least in one cohort were reported in the meta-analysis; among these, ∼4.7 M variants were present among all four cohorts.

### Correlation between rs12442183 Genotype with RGMA Expression

The potential correlation between rs12442183 genotype and *RGMA* expression was tested with publicly available microarray data. First, R package ESLiMc (Exon Splicing by Linear Modeling Analysis core) ([14](#R14)) was used to analyze alternative expression of *RGMA* transcripts among 11 different healthy human tissues, including breast, cerebellum, heart, kidney, liver, muscle, pancreas, prostate, spleen, testes, and thyroid, data from which were obtained from [www.affymetrix.com](http://www.affymetrix.com) and included in ESLiMc analysis by its author ([14](#R14)). The rationale for using ESLiMc was that if there are no alternative splicing events, an increase in the global expression of a gene should correlate with a higher expression of all of its exons. Linear regression was used by ESLiMc to establish the above relationship between each exon and its corresponding gene. The difference (referred to as gene-exon residual regression score) generated by ESLiMc between the observed exon expression and the predicted exon expression was interpreted as being due to an alternative splicing ([14](#R14)). Next, we tested the correlation between rs12442183 genotype and the expression of *RGMA* transcript variants. We downloaded rs12442183 genotype information from BRAINEAC ([15](#R15)) and reprocessed its corresponding microarray data from GEO (Gene Expression Omnibus) ([16](#R16)) (accession No. [GSE60863](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60863)) ([17](#R17)) with R package ESLiMc. [GSE60863](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60863) is comprised of 1231 samples originating from 10 brain tissues (occipital cortex, frontal cortex, temporal cortex, hippocampus, intralobular white matter, cerebellar cortex, thalamus, putamen, substantia nigra, and medulla (inferior olivary nucleus)) derived from 134 healthy human brains from UK Biobank ([17](#R17)). These brain samples were confirmed to be free of neurodegenerative disorders and assayed by Affymetrix Human Exon 1.0 ST array. We also used PExFIns ([18](#R18)), an expression quantitative locus (eQTL) analysis software package, to test the association of rs12442183 genotype with *RGMA* mRNA expression in 423 lymphoblastoid cell lines (LCLs) across 6 global populations, including 73 CEU (Utah residents with northern and western European ancestry), 77 CHB (Han Chinese in Beijing, China), 72 JPT (Japanese in Tokyo, Japan), 80 LWK (Luhya in Webuye, Kenya), 42 MEX (Mexican ancestry in Los Angeles) and 79 YRI (Yoruba in Ibadan, Nigeria).

Besides *RGMA*, we also tested the association between rs12442183 genotype and the expression of other genes close to the SNP at both mRNA and exon level (see Integrative cis-eQTL Analysis in [Supplementary Information](#SD1)).

### Effect of Morphine Injection on Mouse Rgma Expression

The publicly available microarray experiment dataset [GSE7762](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE7762) was designed to determine the impact of genetic background on the transcriptional effects of morphine in mouse striatum ([19](#R19)). It used different morphine treatments, including a single morphine injection of 20 mg/kg, repeated (3 times daily for 5 days) morphine injections of 10-40 mg/kg, and saline, in four inbred mouse strains (129P3/J, DBA/2J, C57BL/6J and SWR/J). We used the R package GEOquery ([20](#R20)) to download the normalized [GSE7762](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE7762) microarray data from GEO and analyze the expression of *Rgma* (the mouse gene homologous to *RGMA*). One-way ANOVA implemented in MATLAB (Statistics Toolbox Release 2015b, MathWorks, Inc., Natick, Massachusetts, United States) was used to test the effect of morphine on *Rgma* expression in different mouse strains. Statistical significance was set at P<0.05.

### Co-expression Analysis of RGMA

A novel bioinformatics pipeline was developed to search and prioritize genes co-expressed with *RGMA* in an unbiased way genome-wide. The correlations identified between *RGMA* and the identified co-expressed genes were subject to validation in brain tissue samples (n=1301) downloaded from GEO (see *RGMA* Co-Expression Analysis in [Supplementary Information](#SD1)).

## Results

One GWS SNP was identified: rs12442183 (MAF 0.41 in EAs), which is ∼110 kb upstream of *RGMA*, in Yale-Penn 1 (P=4.7×10^-8^, n=1388). Meta-analysis of this sample with the other three samples (Yale-Penn 2 and 3, and SAGE) improved the GWS association signal (meta-P=1×10^-8^, n=3058) ([Figure 1](#F1) and [Table 1](#T1)). The robust association signals around the top SNP ([Figure 1](#F1)) and the observed negligible inflation factor in the QQ plot ([Supplementary figure 2](#SD1)) support the validity of the GWAS result. As noted above, the association was GWS in Yale-Penn 1 taken individually. We also observed P <0.05 in both Yale-Penn 2 and Yale-Penn 3: the risk allele T of rs12442183 was significantly correlated with increased OD criterion counts in all cohorts, except for SAGE ([Table 1](#T1) and [Figure 2](#F2)). SAGE has little variation in OD criterion count and >50% of individuals have a criterion count of 0, yielding low power in that sample.

### Figure 1. Manhattan plots showing genome-wide association signals of opioid dependence in opioid-exposed European-Americans by meta-analysis.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d143/6041180/d2558a479d18/nihms933995f1.jpg)

(A) Manhattan plot with one significant variant, rs12442183, on chromosome 15, in meta-analysis of four OD cohorts. The line in the plot represents the genome-wide significance cutoff (5×10-8). (B) Regional Manhattan plot demonstrates rs12442183 is close to gene RGMA (and regulates RGMA expression; see Figure 3 and text). The light blue line and right Y-axis show the observed recombination rate in the 1000 Genomes Project European samples (EUR, hg19). The SNPs are colored in accordance to R2 with rs12442183 (in purple), except for rs113200177, which is specifically highlighted in dark. rs113200177 (T/TG, meta-P=2×10-3) is located in the promoter of RGMA and is predicted to be able to destroy the E-box motif (5′-CANNTG-3′) of transcription factor 3 or 4 (TCF3 or TCF4) by Haploreg4 (45). By adjusting the effect of the indel in conditional association analysis for rs12442183, the association signal of rs12442183 became stronger in Yale-Penn 1 taken individually (P=1.9×10-8).

### Table 1. Meta-analysis of rs12442183 Risk Allele T in OD Exposed Individuals.

| Population | Cohort1 | Total Subjects No. | Exposed Subject No. | Criterion Counts2 | MAF3 | Effect Beta (95% CI) 4 | Imputation R2 | P |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | Yale-Penn 1 | 1768 | 1388 | 5.0 ± 2.8 | 0.41 | 0.60 (0.39 - 0.82) | 0.98 | 4.7 × 10-8 |
| European-Americans (EA) | Yale-Penn 2 | 1725 | 996 | 4.7 ± 2.9 | 0.38 | 0.30 (0.02 - 0.58) | 0.82 | 3.8 × 10-2 |
| Yale-Penn 3 | 403 | 98 | 3.6 ± 3.2 | 0.41 | 1.6 (0.52 - 2.8) | 0.81 | 5.3 × 10-3 |  |
| SAGE | 2752 | 576 | 2.0 ± 2.8 | 0.38 | -0.14 (-0.18 - 0.50) | 0.98 | 3.9 × 10-1* |  |
|   | Meta-analysis |   |   |   |   | 0.44 |   | 1.3 × 10-8 |
|   | Yale-Penn 1 | 3289 | 1209 | 3.8 ± 3.1 | 0.33 | 0.14(0.04 - 0.6) | 0.98 | 2.3 × 10-2 |
| African-Americans (AA) | Yale-Penn 2 | 1744 | 245 | 3.4± 3.1 | 0.3 | 0.50 (-0.08 - 1.1) | 0.82 | 8.0 × 10-2 |
| Yale-Penn 3 | 139 | 12 | - | - | - | - | - |  |
| SAGE | 1312 | 252 | 2.5 ± 3.0 | 0.34 | 0.25 (-0.31 - 0.81) | 0.98 | 3.7 × 10-1 |  |
|   | Meta-analysis |   |   |   |   | 0.34 |   | 3.0 × 10-3 |

Table 1 Caption: 1Illumina microarray HumanOmni1-Quad v1.0, HumanCore Exome array, and Human 1M were used to genotype samples for Yale-Penn 1-3, respectively. 2Mean ± standard deviation for criterion counts of ordinal traits. 3Minor allele frequency. 495% confidence interval of continuous effect beta based on likelihood ratio test by using the software Genome-wide Efficient Mixed Model Association (GEMMA). *indicates the opposite effect direction.

### Figure 2. Forest plot for rs12442183 risk allele T in opioid dependence GWAS.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d143/6041180/bdd34b0a939a/nihms933995f2.jpg)

Effect Beta and its 95% confidence interval are plotted with squares (proportional to sample size in each cohort) and horizontal lines with whiskers, respectively. The vertical line (beta=0) indicates no effect. Names of cohorts are shown on the left. Meta-analysis P-values and their corresponding effect betas (red diamonds) for European-American (EA) and African-American (AA) populations are also provided. The forest plot was generated by DistillerSR Forest Plot Generator from Evidence Partners (https://www.evidencepartners.com/resources/forest-plot-generator).

The association of the lead SNP rs12442183 with OD was tested further in opioid-exposed AAs from three cohorts ([Table 1](#T1)), including Yale-Penn 1 (n=1506), Yale-Penn 2 (n=245) and SAGE (n=263) (MAF 0.33 in AAs). Yale-Penn 3 was not included because of the small number of AAs (n=12). Rs12442183 was significantly (P=1.2×10^-2^) or marginally significantly (P=8×10^-2^) associated with OD in Yale-Penn 1 and Yale-Penn 2, respectively. Possibly due to the same low variation of OD criterion counts in SAGE AAs, the association of rs12442183 with OD was not significant. However, by meta-analysis of the 2014 AA samples, the association of rs12442183 with OD was replicated (meta-P=3×10^-3^) ([Table 1](#T1) and [Figure 2](#F2)).

The validity of the association of rs12442183 with OD in EAs and potentially also in AAs was demonstrated by the observation that 5 out of 7 cohorts ([Table 1](#T1) and [Figure 2](#F2)) show at least nominal significance at this marker. Rs12442183 and rs1872052 (hg19 position chr15: 93752878, ∼4.9 kb downstream of rs12442183) are the only two of the top 10 signals in the EA GWAS that were also nominally associated in AAs ([Supplementary Table 2](#SD1)). Many of the variants that were associated in EAs were not present or were uncommon in AAs, so there was no relevant information for them. The meta-analysis P-value for rs12442183 was also significant (3.0×10^-8^) across all cohorts (EA and AA), but the combined P-value is higher (i.e., less significant) than the European-specific P-value, which may be due to the different haplotype structures for EAs and AAs.

We next considered the effects of other substance use disorder criteria, including those for cocaine dependence (CD), alcohol dependence (AD), and nicotine dependence (ND), on the rs12442183 association signal. When we included CD, AD and ND criterion counts as covariates along with age, sex and first three PCs, there was decreased evidence of association of rs12442183 with OD in Yale-Penn 1-3 EAs, where nominal P-values were 3.7×10^-7^, 0.17, and 0.08, respectively (meta-analysis P=3.91×10^-7^). Thus, the effect of CD, AD and ND on the observed association of rs12442183 with OD was observable, but limited.

The association of a previously implicated OD locus, *CNIH3* ([3](#R3)), was evaluated in the current GWAS meta-analysis of EAs. The lead SNP for *CNIH3*, rs10799590, was identified in a GWAS that compared opioid-dependent daily drug injectors with opioid misusers who never progressed to daily injection ([5](#R5)), whereas in our OD GWAS, the phenotype was OD criterion count. With that in mind, we note that the meta-analysis P-value for rs10799590 with OD criterion count was 0.038 in EAs.

The regulatory relationship between rs12442183 and *RGMA* expression was studied using publicly available microarray data. *RGMA* encodes a glycosylphosphatidylinosital-anchored glycoprotein (RGMa). There are three RGMa isoforms ([Supplementary figure 3](#SD1)) that are generated by six *RGMA* transcript variants ([Figure 3A](#F3)), including [NM_001166283](https://www.ncbi.nlm.nih.gov/nuccore/NM_001166283) (variant 1), [NM_001166286](https://www.ncbi.nlm.nih.gov/nuccore/NM_001166286) (variant 2), [NM_001166287](https://www.ncbi.nlm.nih.gov/nuccore/NM_001166287) (variant 3), [NM_020211](https://www.ncbi.nlm.nih.gov/nuccore/NM_020211) (variant 4), [NM_001166288](https://www.ncbi.nlm.nih.gov/nuccore/NM_001166288) (variant 5) and [NM_001166289](https://www.ncbi.nlm.nih.gov/nuccore/NM_001166289) (variant 6). Transcript variant 1 and 4 encode RGMa isoforms 1 and 3, respectively, and all other transcripts produce the same RGMa isoform 2. To identify alternative expression of *RGMA* among 11 healthy human tissues, four *RGMA* exons, including ENSE00001325239, ENSE00001319132, ENSE00001304160 and ENSE00001532613, were analyzed via R package ESLiMc ([Figure 3A](#F3)). ENSE00001325239 and ENSE00001319132 reside in the overlapping exonic regions in all 6 transcripts of *RGMA*, i.e., the expression signals of the two exons represent the total mRNA of *RGMA*. ENSE00001532613 is a specific exon representing transcript variant 4 ([NM_020211](https://www.ncbi.nlm.nih.gov/nuccore/NM_020211), encoding RGMa isoform 3). Similarly, exon ENSE00001304160, specifically represents transcript variant 2 ([NM_001166286](https://www.ncbi.nlm.nih.gov/nuccore/NM_001166286), encoding RGMa isoform 2). Only ENSE00001532613, representing transcript variant 4, was variably expressed among the 11 tissues studied, and cerebellum was the only brain tissue with both the lowest expression and with the lowest gene-exon residual regression score for *RGMA* ([Figure 3A and 3B](#F3)). Further correlation analysis between rs12442183 genotype and the gene-exon expression residual scores of the four exons revealed that only ENSE00001532613 (representative of transcript variant 4) was differentially expressed (P=2×10^-3^) among rs12442183*T risk allele carriers in frontal cortex ([Figure 3C](#F3)). Among the other nine brain tissues, there were no significant correlations between rs12442183 genotype and gene-exon residual scores (data not shown). We also tested whether rs12442183 was an eQTL in LCLs across 6 global populations, including CEU, YRI, LWK, MEX, CHB and JPT, but found no significant associations (all linear model P-values >0.05), indicating that rs12442183 may be a specific eQTL for *RGMA* transcript variant 4 (encoding RGMa isoform 3) in frontal cortex.

### Figure 3. rs12442183 is a regulatory variant of RGMA and Rgma is an acute opioid response gene in mouse.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d143/6041180/57e8a3e8780d/nihms933995f3.jpg)

(A) Six transcripts of RGMA and four exons used for alternative splicing analysis are mapped together. The relationship between the expression of these four exons and RGMA gene expression was determined by using a linear regression model implemented in R package ESLiMc. The difference between the observed exon expression and the predicted exon expression is named as residual regression score. The alternative splicing events of RGMA across 11 healthy human tissues are displayed in (B), with dot and error-bar represent mean and standard deviation of 3 replicates, respectively. Boxplots (C) are used to illustrate the association between rs12442183 genotype and the residual regression scores of four exons in brain tissue frontal cortex in European samples. Dot and line within each boxplot represent mean and median, respectively, and box region indicates the range from first quantile to third quantile. Up and down whiskers, as well as dots outside box region represent maximum value, minimum value, and outliers, respectively. (D) The expression of Rgma (homologous gene in mouse) among different mouse strains acutely after morphine injection (a single injection, 20 mg/kg) in comparison with chronic injections (repeated morphine administration, 10-40 mg/kg, 3 times daily for 5 days). X-axis shows the four inbred mouse strains (129P3/J, DBA/2J, C57BL/6J and SWR/J). Y-axis demonstrates the mRNA expression of mouse gene Rgma. The expression data were derived from NCBI Gene Expression Omnibus (GEO) dataset GSE7762. The standard deviation is at the top of each bar.

We also tested whether the observed association between rs12442183 and OD was driven by the SNP's regulatory effect on other genes in the region (see Integrative cis-eQTL Analysis in [Supplementary Information](#SD1)). With the independent data from the R package ESLiM, only *RGMA* and *MCTP2* showed alternative splicing in healthy brain tissue. We further correlated the genotype of rs12442183 with available exon expression of *RGMA*, *CHD2*, *FAM174B* and *MCTP2* from BRAINEAC. There was a significant correlation of rs12442183 genotype with specific exons of *RGMA* and *MCTP2*. However, the expression of *MCTP2* is very low among 27 human tissues.

We re-analyzed publicly available data from a microarray experiment (GEO accession No. [GSE7762](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE7762)) designed to test the effects of different morphine treatments on gene expression in mouse striatum tissue. By comparison with control and chronic morphine treatment, we found that the expression of mouse *Rgma* was significantly decreased (all ANOVA P-values<0.01) after acute morphine challenge in all 4 mouse strains, except for C57BL/6J. Compared to acute morphine challenge, significant upregulation of *Rgma* was found in C57BL/6J mouse striatum after chronic morphine injection (ANOVA P-value<0.01). These results demonstrate that *Rgma* was differentially regulated in mouse striatum when comparing C57BL/6J with other mouse strains.

The expression of six genes previously associated with psychiatric disorders and two potential regulators for *RGMA* were correlated with that of *RGMA* in brain tissue samples (n=1301). These genes are *GRIN1* (glutamate receptor, ionotropic, N-methyl D-aspartate 1), *KCNIP3* (Kv channel interacting protein 3, calsenilin), *PTCHD1* (patched domain containing 1), *FXYD6* (FXYD domain containing ion transport regulator 6), *DLG4* (discs, large homolog 4) and *NRG1* (neuregulin 1), as well as two potential regulators (*TCF3* and *TCF4*). *GRIN1*, *FXYD6*, *NRG1* and *DLG4* have been associated with schizophrenia ([21](#R21)-[24](#R24)), *KCNIP3* with Alzheimer's disease ([25](#R25)), *PTCHD1* with autism ([26](#R26)), and *NRG1* with psychosis ([27](#R27)). The E-box (5′-CANNTG-3′) motif of the two potential regulators, *TCF3* and *TCF4*, was found in the promoter region of *RGMA*, *GRIN1* and *PTCHD1*. We combined normalized expression data (z-scores) of 1301 brain tissue samples to test the pairwise correlation among the six genes. As shown in [Supplementary figure 4A](#SD1), the expressions of *GRIN1*, *KCNIP3*, *FXYD6*, *PTCHD1*, *RGMA*, *TCF3* and *TCF4* were highly inter-correlated. For *RGMA*, there was a strong positive correlation with *GRIN1* (Pearson correlation coefficient r=0.40 and P=7.0×10^-50^) and *KCNIP3* (r=0.38 and P=1.35×10^-45^) ([Supplementary figure 4B](#SD1)), with the latter two also significantly inter-correlated ([Supplementary figure 4A](#SD1): r=0.73 and P=1×10^-200^).

## Discussion

We report here results from a GWAS of opioid criterion count in EA subjects. There was a GWS association of an *RGMA* variant, rs12442183, in Yale-Penn 1. There was nominally significant evidence for association of the same variant in both Yale-Penn 2 and Yale-Penn 3, and when all EA samples (including SAGE, where no evidence of the association was obtained) were meta-analyzed together, the significance of the association finding increased. Because the minor allele frequency of rs12442183 was similar for the EA and AA populations, we further evaluated the association of rs12442183 with OD in the Yale-Penn AA sample. The nominally significant association in the AA meta-analysis (n=2014, meta-P=3×10^-3^) further supports the association. (AA replication of results from EA samples is not universally applicable, but in this case, we found evidence that we believe is of value.) Cis-eQTL analysis of *RGMA* in brain showed the rs12442183*T risk allele to be associated with higher expression of *RGMA* transcript variant 4 (encoding RGMa isoform 3) in frontal cortex. This observation is consistent with the upregulation of *Rgma* in C57BL/J6 mice treated with morphine chronically, which is not the case in three other mouse strains: DBA/2J, 129P3/J and SWR/J. C57BL/6J mice have a greater preference for oral self-administration of morphine than three other strains ([19](#R19)). Further co-expression analysis in large brain tissue expression datasets demonstrated significant correlation between the expression of *RGMA* and that of four genes that have been shown to affect risk for other psychiatric disorders, *GRIN1* (schizophrenia) ([23](#R23)), *KCNIP3* (Alzheimer's disease) ([25](#R25)), *FXYD6* (schizophrenia) ([21](#R21)) and *PTCHD1* (autism) ([26](#R26)), as well as two transcription factors *TCF3* and *TCF4*. The latter gene TCF4 is a schizophrenia-risk gene ([28](#R28)). These *RGMA* co-expressed genes are critical to normal brain function, supporting the potential involvement of *RGMA* in related biological processes.

The protein product of *RGMA*, RGMa, is a glycosylphosphatidylinosital-anchored glycoprotein that exists in membrane-bound and soluble forms ([29](#R29)), both of which are potent regulators of cell death and inhibitors of nerve growth. RGMa was first reported to be a developmental neurite growth inhibitor ([30](#R30)). It has since emerged as an important factor inhibiting neuronal regeneration and functional recovery ([31](#R31), [32](#R32)). RGMa inhibits neurite growth by binding to its neuronal receptor, neogenin, a member of the immunoglobulin (Ig) superfamily. Neogenin overexpression and RGMa down expression in the developing embryonic neural tube induce apoptosis ([33](#R33)). RGMa may be involved in neuroplasticity consistent with increased secretion of RGMa in stroke- or injury-damaged human brain tissue ([34](#R34)), in the plaque regions of Alzheimer's disease patients ([35](#R35)), in the substantia nigra of Parkinson's disease patients ([36](#R36)), and in multiple sclerosis (MS) patients ([37](#R37)). Considering the example of progressive MS ([38](#R38)), RGMa was found in active and chronic lesions, and in normal-appearing gray and white matter, and was expressed by cellular meningeal infiltrates in autopsy material. Levels of soluble RGMa in the cerebrospinal fluid were decreased in progressive MS patients for whom treatment resulted in functional improvement. Antibody-mediated neutralization of RGMa or downregulation of RGMa can result in neuroprotection and enhanced functional recovery ([38](#R38)). Whether RGMa plays similar roles in OD patients and thereby modulates addiction risk warrants further study.

The mouse experiment that investigated the effects of morphine on the striatal transcriptome ([19](#R19)) identified 661 morphine-responsive genes. *Rgma* was one of the top 98 genes that were down-regulated with *acute* morphine treatment. Consistent downregulation of *Rgma* was observed among 3 mouse strains, including DBA/2J, 129P3/J and SWR/J, but not C57BL/6J. However, *Rgma* was specifically upregulated in C57BL/6J mice (in contrast to the other 3 mouse strains) following *chronic* morphine treatment. The different responses of these mouse strains after chronic or acute morphine treatment may reflect genetic differences, as C57BL/6J mice have been reported to have the greatest preference for orally self-administered morphine of the strains studied ([19](#R19)). Further, in the current study, risk rs12442183*T-allele carriers demonstrated a higher expression of one *RGMA* transcript in frontal cortex. As downregulation of RGMa can result in neuroprotection and enhanced functional recovery ([38](#R38)) and increased RGMa in brain is associated with risk of multiple neurological disorders, including Alzheimer's disease, Parkinson's disease, and multiple sclerosis, downregulation of RGMa may have a protective effect on neurons after acute morphine treatment, while upregulation of RGMa may induce brain vulnerability with chronic morphine treatment.

In the analysis of several brain tissue samples, *RGMA* expression was associated with the expression of four other genes related to psychiatric traits, *GRIN1*, *KCNIP3*, *FXYD6* and *PTCHD1*. *GRIN1* was the gene that was most highly co-expressed (r=0.40) with *RGMA* in brain. It plays a key role in synaptic plasticity, synaptogenesis, excitotoxicity, memory acquisition and learning ([39](#R39)). *GRIN1* can tether the μ-opioid receptor (the main biological target of opioid drugs) and subsequently immobilize the μ-opioid receptor within lipid rafts ([40](#R40)). The overexpression of *GRIN1* increases the amount of μ-opioid receptor in lipid rafts and the magnitude of receptor signaling. The significant correlation between the expression of other genes (*PTCHD1*, *FXYD6*, and *KCNIP3*) and that of *RGMA*, may reflect their critical function in the brain ([21](#R21), [26](#R26), [41](#R41)). These findings raise the question of whether, in response to opioid stimulation in brain, there is crosstalk between *RGMA* and *GRIN1*, and possibly *KCNIP3*, *FXYD6* and *PTCHD1* as well.

Different proportions of prescribed vs. illegal users of opioids in the Yale-Penn and the SAGE samples could have affected our results and may help to understand the different results from these samples. Neither the SSADDA (Yale-Penn) nor the SSAGA (SAGE) differentiates prescribed from illegal-recreational use of opioids, and therefore some subjects who endorsed DSM-IV OD criteria probably did so despite using opioids therapeutically. However, a majority (59.5%) of individuals in our samples who met criteria for OD endorsed at least five of the seven diagnostic criteria (mean criterion count = 6.6, [Supplementary figure 1](#SD1)), suggesting that the GWS finding is attributable to moderate-to-severe OD, rather than being contributed to substantially by individuals who used opioids as prescribed, but required an increased dosage for pain control or who experienced withdrawal symptoms when opioid treatment was discontinued. The much greater proportion of participants in the Yale-Penn samples than SAGE who met the maximum number of (i.e., seven) OD criteria supports that interpretation. Also, compared to our (Yale-Penn) samples, the SAGE samples were recruited for alcoholism ([42](#R42)), cocaine dependence ([43](#R43)), or nicotine dependence ([44](#R44)), rather than OD. Therefore, a much higher percentage of opioid-exposed individuals in SAGE are “controls”: viz. individuals who used opioids fewer than 11 times lifetime and would have skipped out of the remainder of the SSAGA opioid section and had a null criterion count ([Supplementary figure 1](#SD1)). Additionally, the relative percentage of opioid-exposed individuals varies considerably across our three Yale-Penn EA samples because of the different recruiting epochs, although the assessment protocol did not change: the earlier Yale-Penn samples were more focused on OD recruitment than the later ones.

In summary, we report a GWS association of rs12442183, at the *RGMA* locus, with OD. In contrast to our earlier study, the present study includes only opioid-exposed subjects selected from an expanded sample; and our analyses for this study did not co-vary for the criterion count for other substance use disorder traits. We believe that the former two differences were key in our identification of the present very robust association finding. The finding was GWS in our largest sample taken separately, with nominally significant associations of the same variant in two of three other available samples of EAs. Further support was obtained in AA samples. Our current data support a model in which the regulatory variant rs12442183 affects expression of *RGMA* transcript variant 4 in frontal cortex, and the consequent variation in RGMa isoform 3 may alter the response to opioids, a potential underlying mechanism for its association with OD. This mechanism and locus merit further study, which could increase our understanding of the genetics and pathophysiology of OD.

## Supplementary Material

## Acknowledgments

This study was supported by National Institutes of Health grants RC2 DA028909, R01 DA12690, R01 DA12849, R01 DA18432, R01 AA11330, R01 AA017535, and the VA Connecticut Healthcare Center and Philadelphia VA MIRECCs. We appreciate the work in recruitment and assessment by James Poling, PhD at Yale University School of Medicine and the APT Foundation; Roger Weiss, MD at McLean Hospital; by Kathleen Brady, MD/PhD and Raymond Anton, MD at the Medical University of South Carolina; and David Oslin, MD at the University of Pennsylvania. Genotyping services for a part of our GWAS were provided by the Center for Inherited Disease Research and Yale University (Center for Genome Analysis), which is fully funded by Federal contract N01-HG-65403 from the NIH to The Johns Hopkins University. We thank Ann Marie Lacobelle, MS and Christa Robinson, BS, who provided technical assistance. We thank Yaira Nunez and Michelle Slivinsky who led the quality control effort for the Semi-Structured Assessment for Drug Dependence and Alcoholism interviews and phenotyping for the study sample. We thank John Farrell, PhD, Section of Biomedical Genetics, Boston University School of Medicine, who provided database management assistance. We also thank three anonymous reviewers for their constructive suggestions on the manuscript.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Mistry CJ, Bawor M, Desai D, Marsh DC, Samaan Z. Genetics of Opioid Dependence: A Review of the Genetic Contribution to Opioid Dependence. Curr Psychiatry Rev. 2014;10:156–167. doi: 10.2174/1573400510666140320000928.  [DOI](https://doi.org/10.2174/1573400510666140320000928) | [PMC free article](/articles/PMC4155832/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25242908/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Psychiatry%20Rev&title=Genetics%20of%20Opioid%20Dependence:%20A%20Review%20of%20the%20Genetic%20Contribution%20to%20Opioid%20Dependence&author=CJ%20Mistry&author=M%20Bawor&author=D%20Desai&author=DC%20Marsh&author=Z%20Samaan&volume=10&publication_year=2014&pages=156-167&pmid=25242908&doi=10.2174/1573400510666140320000928&)

2. Gelernter J, Kranzler HR, Sherva R, Koesterer R, Almasy L, Zhao H, et al. Genome-wide association study of opioid dependence: multiple associations mapped to calcium and potassium pathways. Biol Psychiatry. 2014;76:66–74. doi: 10.1016/j.biopsych.2013.08.034.  [DOI](https://doi.org/10.1016/j.biopsych.2013.08.034) | [PMC free article](/articles/PMC3992201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24143882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Genome-wide%20association%20study%20of%20opioid%20dependence:%20multiple%20associations%20mapped%20to%20calcium%20and%20potassium%20pathways&author=J%20Gelernter&author=HR%20Kranzler&author=R%20Sherva&author=R%20Koesterer&author=L%20Almasy&volume=76&publication_year=2014&pages=66-74&pmid=24143882&doi=10.1016/j.biopsych.2013.08.034&)

3. Nelson EC, Agrawal A, Heath AC, Bogdan R, Sherva R, Zhang B, et al. Evidence of CNIH3 involvement in opioid dependence. Mol Psychiatry. 2016;21:608–614. doi: 10.1038/mp.2015.102.  [DOI](https://doi.org/10.1038/mp.2015.102) | [PMC free article](/articles/PMC4740268/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26239289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Evidence%20of%20CNIH3%20involvement%20in%20opioid%20dependence&author=EC%20Nelson&author=A%20Agrawal&author=AC%20Heath&author=R%20Bogdan&author=R%20Sherva&volume=21&publication_year=2016&pages=608-614&pmid=26239289&doi=10.1038/mp.2015.102&)

4. Smith AH, Jensen KP, Li J, Nunez Y, Farrer LA, Hakonarson H, et al. Genome-wide association study of therapeutic opioid dosing identifies a novel locus upstream of OPRM1. Mol Psychiatry. 2017;22:346–352. doi: 10.1038/mp.2016.257.  [DOI](https://doi.org/10.1038/mp.2016.257) | [PMC free article](/articles/PMC5407902/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28115739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Genome-wide%20association%20study%20of%20therapeutic%20opioid%20dosing%20identifies%20a%20novel%20locus%20upstream%20of%20OPRM1&author=AH%20Smith&author=KP%20Jensen&author=J%20Li&author=Y%20Nunez&author=LA%20Farrer&volume=22&publication_year=2017&pages=346-352&pmid=28115739&doi=10.1038/mp.2016.257&)

5. Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience. 2015;4:7. doi: 10.1186/s13742-015-0047-8.  [DOI](https://doi.org/10.1186/s13742-015-0047-8) | [PMC free article](/articles/PMC4342193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25722852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gigascience&title=Second-generation%20PLINK:%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&author=CC%20Chang&author=CC%20Chow&author=LC%20Tellier&author=S%20Vattikuti&author=SM%20Purcell&volume=4&publication_year=2015&pages=7&pmid=25722852&doi=10.1186/s13742-015-0047-8&)

6. 1000-Genomes-Project-Consortium. Abecasis GR, Altshuler D, Auton A, Brooks LD, Durbin RM, et al. A map of human genome variation from population-scale sequencing. Nature. 2010;467:1061–1073. doi: 10.1038/nature09534.  [DOI](https://doi.org/10.1038/nature09534) | [PMC free article](/articles/PMC3042601/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20981092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20map%20of%20human%20genome%20variation%20from%20population-scale%20sequencing&author=GR%20Abecasis&author=D%20Altshuler&author=A%20Auton&author=LD%20Brooks&author=RM%20Durbin&volume=467&publication_year=2010&pages=1061-1073&pmid=20981092&doi=10.1038/nature09534&)

7. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

8. Sherva R, Wang Q, Kranzler H, Zhao H, Koesterer R, Herman A, et al. Genome-wide association study of cannabis dependence severity, novel risk variants, and shared genetic risks. JAMA Psychiatry. 2016;73:472–480. doi: 10.1001/jamapsychiatry.2016.0036.  [DOI](https://doi.org/10.1001/jamapsychiatry.2016.0036) | [PMC free article](/articles/PMC4974817/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27028160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Psychiatry&title=Genome-wide%20association%20study%20of%20cannabis%20dependence%20severity,%20novel%20risk%20variants,%20and%20shared%20genetic%20risks&author=R%20Sherva&author=Q%20Wang&author=H%20Kranzler&author=H%20Zhao&author=R%20Koesterer&volume=73&publication_year=2016&pages=472-480&pmid=27028160&doi=10.1001/jamapsychiatry.2016.0036&)

9. Radmanesh F, Devan WJ, Anderson CD, Rosand J, Falcone GJ Alzheimer's Disease Neuroimaging I. Accuracy of imputation to infer unobserved APOE epsilon alleles in genome-wide genotyping data. Eur J Hum Genet. 2014;22:1239–1242. doi: 10.1038/ejhg.2013.308.  [DOI](https://doi.org/10.1038/ejhg.2013.308) | [PMC free article](/articles/PMC4169533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24448547/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Accuracy%20of%20imputation%20to%20infer%20unobserved%20APOE%20epsilon%20alleles%20in%20genome-wide%20genotyping%20data&author=F%20Radmanesh&author=WJ%20Devan&author=CD%20Anderson&author=J%20Rosand&author=GJ%20Falcone&volume=22&publication_year=2014&pages=1239-1242&pmid=24448547&doi=10.1038/ejhg.2013.308&)

10. Zhou X, Stephens M. Genome-wide efficient mixed-model analysis for association studies. Nat Genet. 2012;44:821–824. doi: 10.1038/ng.2310.  [DOI](https://doi.org/10.1038/ng.2310) | [PMC free article](/articles/PMC3386377/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22706312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20efficient%20mixed-model%20analysis%20for%20association%20studies&author=X%20Zhou&author=M%20Stephens&volume=44&publication_year=2012&pages=821-824&pmid=22706312&doi=10.1038/ng.2310&)

11. Bierut LJ, Strickland JR, Thompson JR, Afful SE, Cottler LB. Drug use and dependence in cocaine dependent subjects, community-based individuals, and their siblings. Drug Alcohol Depen. 2008;95:14–22. doi: 10.1016/j.drugalcdep.2007.11.023.  [DOI](https://doi.org/10.1016/j.drugalcdep.2007.11.023) | [PMC free article](/articles/PMC2384165/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18243582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depen&title=Drug%20use%20and%20dependence%20in%20cocaine%20dependent%20subjects,%20community-based%20individuals,%20and%20their%20siblings&author=LJ%20Bierut&author=JR%20Strickland&author=JR%20Thompson&author=SE%20Afful&author=LB%20Cottler&volume=95&publication_year=2008&pages=14-22&pmid=18243582&doi=10.1016/j.drugalcdep.2007.11.023&)

12. Edenberg HJ. The collaborative study on the genetics of alcoholism: an update. Alcohol Res Health. 2002;26:214–218.  [PMC free article](/articles/PMC6683843/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12875050/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Res%20Health&title=The%20collaborative%20study%20on%20the%20genetics%20of%20alcoholism:%20an%20update&author=HJ%20Edenberg&volume=26&publication_year=2002&pages=214-218&pmid=12875050&)

13. Howie BN, Donnelly P, Marchini J. A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. PLoS Genet. 2009;5:e1000529. doi: 10.1371/journal.pgen.1000529.  [DOI](https://doi.org/10.1371/journal.pgen.1000529) | [PMC free article](/articles/PMC2689936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19543373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=A%20flexible%20and%20accurate%20genotype%20imputation%20method%20for%20the%20next%20generation%20of%20genome-wide%20association%20studies&author=BN%20Howie&author=P%20Donnelly&author=J%20Marchini&volume=5&publication_year=2009&pages=e1000529&pmid=19543373&doi=10.1371/journal.pgen.1000529&)

14. Risueno A, Roson-Burgo B, Dolnik A, Hernandez-Rivas JM, Bullinger L, De Las Rivas J. A robust estimation of exon expression to identify alternative spliced genes applied to human tissues and cancer samples. BMC Genomics. 2014;15:879. doi: 10.1186/1471-2164-15-879.  [DOI](https://doi.org/10.1186/1471-2164-15-879) | [PMC free article](/articles/PMC4298068/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25297679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&title=A%20robust%20estimation%20of%20exon%20expression%20to%20identify%20alternative%20spliced%20genes%20applied%20to%20human%20tissues%20and%20cancer%20samples&author=A%20Risueno&author=B%20Roson-Burgo&author=A%20Dolnik&author=JM%20Hernandez-Rivas&author=L%20Bullinger&volume=15&publication_year=2014&pages=879&pmid=25297679&doi=10.1186/1471-2164-15-879&)

15. Hernandez DG, Nalls MA, Moore M, Chong S, Dillman A, Trabzuni D, et al. Integration of GWAS SNPs and tissue specific expression profiling reveal discrete eQTLs for human traits in blood and brain. Neurobiol Dis. 2012;47:20–28. doi: 10.1016/j.nbd.2012.03.020.  [DOI](https://doi.org/10.1016/j.nbd.2012.03.020) | [PMC free article](/articles/PMC3358430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22433082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurobiol%20Dis&title=Integration%20of%20GWAS%20SNPs%20and%20tissue%20specific%20expression%20profiling%20reveal%20discrete%20eQTLs%20for%20human%20traits%20in%20blood%20and%20brain&author=DG%20Hernandez&author=MA%20Nalls&author=M%20Moore&author=S%20Chong&author=A%20Dillman&volume=47&publication_year=2012&pages=20-28&pmid=22433082&doi=10.1016/j.nbd.2012.03.020&)

16. Barrett T, Wilhite SE, Ledoux P, Evangelista C, Kim IF, Tomashevsky M, et al. NCBI GEO: archive for functional genomics data sets-update. Nucleic Acids Res. 2013;41:D991–D995. doi: 10.1093/nar/gks1193.  [DOI](https://doi.org/10.1093/nar/gks1193) | [PMC free article](/articles/PMC3531084/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23193258/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=NCBI%20GEO:%20archive%20for%20functional%20genomics%20data%20sets-update&author=T%20Barrett&author=SE%20Wilhite&author=P%20Ledoux&author=C%20Evangelista&author=IF%20Kim&volume=41&publication_year=2013&pages=D991-D995&pmid=23193258&doi=10.1093/nar/gks1193&)

17. Ramasamy A, Trabzuni D, Guelfi S, Varghese V, Smith C, Walker R, et al. Genetic variability in the regulation of gene expression in ten regions of the human brain. Nat Neurosci. 2014;17:1418–1428. doi: 10.1038/nn.3801.  [DOI](https://doi.org/10.1038/nn.3801) | [PMC free article](/articles/PMC4208299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25174004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Neurosci&title=Genetic%20variability%20in%20the%20regulation%20of%20gene%20expression%20in%20ten%20regions%20of%20the%20human%20brain&author=A%20Ramasamy&author=D%20Trabzuni&author=S%20Guelfi&author=V%20Varghese&author=C%20Smith&volume=17&publication_year=2014&pages=1418-1428&pmid=25174004&doi=10.1038/nn.3801&)

18. Cheng Z, Chu H, Fan Y, Li C, Song YQ, Zhou J, et al. PExFInS: an integrative post-GWAS explorer for functional indels and SNPs. Sci Rep. 2015;5:17302. doi: 10.1038/srep17302.  [DOI](https://doi.org/10.1038/srep17302) | [PMC free article](/articles/PMC4661514/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26612672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=PExFInS:%20an%20integrative%20post-GWAS%20explorer%20for%20functional%20indels%20and%20SNPs&author=Z%20Cheng&author=H%20Chu&author=Y%20Fan&author=C%20Li&author=YQ%20Song&volume=5&publication_year=2015&pages=17302&pmid=26612672&doi=10.1038/srep17302&)

19. Korostynski M, Piechota M, Kaminska D, Solecki W, Przewlocki R. Morphine effects on striatal transcriptome in mice. Genome Biol. 2007;8 doi: 10.1186/gb-2007-8-6-r128.  [DOI](https://doi.org/10.1186/gb-2007-8-6-r128) | [PMC free article](/articles/PMC2394777/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17598886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Morphine%20effects%20on%20striatal%20transcriptome%20in%20mice&author=M%20Korostynski&author=M%20Piechota&author=D%20Kaminska&author=W%20Solecki&author=R%20Przewlocki&volume=8&publication_year=2007&pmid=17598886&doi=10.1186/gb-2007-8-6-r128&)

20. Sean D, Meltzer PS. GEOquery: a bridge between the gene expression omnibus (GEO) and BioConductor. Bioinformatics. 2007;23:1846–1847. doi: 10.1093/bioinformatics/btm254.  [DOI](https://doi.org/10.1093/bioinformatics/btm254) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17496320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=GEOquery:%20a%20bridge%20between%20the%20gene%20expression%20omnibus%20(GEO)%20and%20BioConductor&author=D%20Sean&author=PS%20Meltzer&volume=23&publication_year=2007&pages=1846-1847&pmid=17496320&doi=10.1093/bioinformatics/btm254&)

21. Choudhury K, McQuillin A, Puri V, Pimm J, Datta S, Thirumalai S, et al. A genetic association study of chromosome 11q22-24 in two different samples implicates the FXYD6 gene, encoding phosphohippolin, in susceptibility to schizophrenia. Am J Hum Genet. 2007;80:664–672. doi: 10.1086/513475.  [DOI](https://doi.org/10.1086/513475) | [PMC free article](/articles/PMC1852702/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17357072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20genetic%20association%20study%20of%20chromosome%2011q22-24%20in%20two%20different%20samples%20implicates%20the%20FXYD6%20gene,%20encoding%20phosphohippolin,%20in%20susceptibility%20to%20schizophrenia&author=K%20Choudhury&author=A%20McQuillin&author=V%20Puri&author=J%20Pimm&author=S%20Datta&volume=80&publication_year=2007&pages=664-672&pmid=17357072&doi=10.1086/513475&)

22. Jonsson EG, Saetre P, Vares M, Andreou D, Larsson K, Timm S, et al. DTNBP1, NRG1, DAOA, DAO and GRM3 polymorphisms and schizophrenia: an association study. Neuropsychobiology. 2009;59:142–150. doi: 10.1159/000218076.  [DOI](https://doi.org/10.1159/000218076) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19439994/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychobiology&title=DTNBP1,%20NRG1,%20DAOA,%20DAO%20and%20GRM3%20polymorphisms%20and%20schizophrenia:%20an%20association%20study&author=EG%20Jonsson&author=P%20Saetre&author=M%20Vares&author=D%20Andreou&author=K%20Larsson&volume=59&publication_year=2009&pages=142-150&pmid=19439994&doi=10.1159/000218076&)

23. Demontis D, Nyegaard M, Buttenschon HN, Hedemand A, Pedersen CB, Grove J, et al. Association of GRIN1 and GRIN2A-D with schizophrenia and genetic interaction with maternal herpes simplex virus-2 infection affecting disease risk. Am J Med Genet B Neuropsychiatr Genet. 2011;156B:913–922. doi: 10.1002/ajmg.b.31234.  [DOI](https://doi.org/10.1002/ajmg.b.31234) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21919190/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med%20Genet%20B%20Neuropsychiatr%20Genet&title=Association%20of%20GRIN1%20and%20GRIN2A-D%20with%20schizophrenia%20and%20genetic%20interaction%20with%20maternal%20herpes%20simplex%20virus-2%20infection%20affecting%20disease%20risk&author=D%20Demontis&author=M%20Nyegaard&author=HN%20Buttenschon&author=A%20Hedemand&author=CB%20Pedersen&volume=156B&publication_year=2011&pages=913-922&pmid=21919190&doi=10.1002/ajmg.b.31234&)

24. Tsai SJ, Hong CJ, Cheng CY, Liao DL, Liou YJ. Association study of polymorphisms in post-synaptic density protein 95 (PSD-95) with schizophrenia. J Neural Transm (Vienna) 2007;114:423–426. doi: 10.1007/s00702-006-0587-2.  [DOI](https://doi.org/10.1007/s00702-006-0587-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17093888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neural%20Transm%20(Vienna)&title=Association%20study%20of%20polymorphisms%20in%20post-synaptic%20density%20protein%2095%20(PSD-95)%20with%20schizophrenia&author=SJ%20Tsai&author=CJ%20Hong&author=CY%20Cheng&author=DL%20Liao&author=YJ%20Liou&volume=114&publication_year=2007&pages=423-426&pmid=17093888&doi=10.1007/s00702-006-0587-2&)

25. Pereira AC, Gray JD, Kogan JF, Davidson RL, Rubin TG, Okamoto M, et al. Age and Alzheimer/'s disease gene expression profiles reversed by the glutamate modulator riluzole. Mol Psychiatry. 2017;22:296–305. doi: 10.1038/mp.2016.33.  [DOI](https://doi.org/10.1038/mp.2016.33) | [PMC free article](/articles/PMC5042881/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27021815/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Age%20and%20Alzheimer/'s%20disease%20gene%20expression%20profiles%20reversed%20by%20the%20glutamate%20modulator%20riluzole&author=AC%20Pereira&author=JD%20Gray&author=JF%20Kogan&author=RL%20Davidson&author=TG%20Rubin&volume=22&publication_year=2017&pages=296-305&pmid=27021815&doi=10.1038/mp.2016.33&)

26. Torrico B, Fernandez-Castillo N, Hervas A, Mila M, Salgado M, Rueda I, et al. Contribution of common and rare variants of the PTCHD1 gene to autism spectrum disorders and intellectual disability. Eur J Hum Genet. 2015;23:1694–1701. doi: 10.1038/ejhg.2015.37.  [DOI](https://doi.org/10.1038/ejhg.2015.37) | [PMC free article](/articles/PMC4795195/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25782667/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Contribution%20of%20common%20and%20rare%20variants%20of%20the%20PTCHD1%20gene%20to%20autism%20spectrum%20disorders%20and%20intellectual%20disability&author=B%20Torrico&author=N%20Fernandez-Castillo&author=A%20Hervas&author=M%20Mila&author=M%20Salgado&volume=23&publication_year=2015&pages=1694-1701&pmid=25782667&doi=10.1038/ejhg.2015.37&)

27. Okochi T, Kishi T, Ikeda M, Kitajima T, Kinoshita Y, Kawashima K, et al. Genetic association analysis of NRG1 with methamphetamine-induced psychosis in a Japanese population. Prog Neuropsychopharmacol Biol Psychiatry. 2009;33:903–905. doi: 10.1016/j.pnpbp.2009.04.016.  [DOI](https://doi.org/10.1016/j.pnpbp.2009.04.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19394386/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Neuropsychopharmacol%20Biol%20Psychiatry&title=Genetic%20association%20analysis%20of%20NRG1%20with%20methamphetamine-induced%20psychosis%20in%20a%20Japanese%20population&author=T%20Okochi&author=T%20Kishi&author=M%20Ikeda&author=T%20Kitajima&author=Y%20Kinoshita&volume=33&publication_year=2009&pages=903-905&pmid=19394386&doi=10.1016/j.pnpbp.2009.04.016&)

28. Steinberg S, de Jong S, Andreassen OA, Werge T, Borglum AD, Mors O, et al. Common variants at VRK2 and TCF4 conferring risk of schizophrenia. Hum Mol Genet. 2011;20:4076–4081. doi: 10.1093/hmg/ddr325.  [DOI](https://doi.org/10.1093/hmg/ddr325) | [PMC free article](/articles/PMC3298077/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21791550/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Common%20variants%20at%20VRK2%20and%20TCF4%20conferring%20risk%20of%20schizophrenia&author=S%20Steinberg&author=S%20de%20Jong&author=OA%20Andreassen&author=T%20Werge&author=AD%20Borglum&volume=20&publication_year=2011&pages=4076-4081&pmid=21791550&doi=10.1093/hmg/ddr325&)

29. Tassew NG, Charish J, Seidah NG, Monnier PP. SKI-1 and Furin generate multiple RGMa fragments that regulate axonal growth. Dev Cell. 2012;22:391–402. doi: 10.1016/j.devcel.2011.11.022.  [DOI](https://doi.org/10.1016/j.devcel.2011.11.022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22340500/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dev%20Cell&title=SKI-1%20and%20Furin%20generate%20multiple%20RGMa%20fragments%20that%20regulate%20axonal%20growth&author=NG%20Tassew&author=J%20Charish&author=NG%20Seidah&author=PP%20Monnier&volume=22&publication_year=2012&pages=391-402&pmid=22340500&doi=10.1016/j.devcel.2011.11.022&)

30. Monnier PP, Sierra A, Macchi P, Deitinghoff L, Andersen JS, Mann M, et al. RGM is a repulsive guidance molecule for retinal axons. Nature. 2002;419:392–395. doi: 10.1038/nature01041.  [DOI](https://doi.org/10.1038/nature01041) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12353034/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=RGM%20is%20a%20repulsive%20guidance%20molecule%20for%20retinal%20axons&author=PP%20Monnier&author=A%20Sierra&author=P%20Macchi&author=L%20Deitinghoff&author=JS%20Andersen&volume=419&publication_year=2002&pages=392-395&pmid=12353034&doi=10.1038/nature01041&)

31. Hata K, Fujitani M, Yasuda Y, Doya H, Saito T, Yamagishi S, et al. RGMa inhibition promotes axonal growth and recovery after spinal cord injury. JCB. 2006;173:47–58. doi: 10.1083/jcb.200508143.  [DOI](https://doi.org/10.1083/jcb.200508143) | [PMC free article](/articles/PMC2063787/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16585268/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JCB&title=RGMa%20inhibition%20promotes%20axonal%20growth%20and%20recovery%20after%20spinal%20cord%20injury&author=K%20Hata&author=M%20Fujitani&author=Y%20Yasuda&author=H%20Doya&author=T%20Saito&volume=173&publication_year=2006&pages=47-58&pmid=16585268&doi=10.1083/jcb.200508143&)

32. Kyoto A, Hata K, Yamashita T. Synapse formation of the cortico-spinal axons is enhanced by RGMa inhibition after spinal cord injury. Brain Res. 2007;1186:74–86. doi: 10.1016/j.brainres.2007.10.038.  [DOI](https://doi.org/10.1016/j.brainres.2007.10.038) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17996222/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res&title=Synapse%20formation%20of%20the%20cortico-spinal%20axons%20is%20enhanced%20by%20RGMa%20inhibition%20after%20spinal%20cord%20injury&author=A%20Kyoto&author=K%20Hata&author=T%20Yamashita&volume=1186&publication_year=2007&pages=74-86&pmid=17996222&doi=10.1016/j.brainres.2007.10.038&)

33. Matsunaga E, Tauszig-Delamasure S, Monnier PP, Mueller BK, Strittmatter SM, Mehlen P, et al. RGM and its receptor neogenin regulate neuronal survival. Nat Cell Biol. 2004;6:749–755. doi: 10.1038/ncb1157.  [DOI](https://doi.org/10.1038/ncb1157) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15258591/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Cell%20Biol&title=RGM%20and%20its%20receptor%20neogenin%20regulate%20neuronal%20survival&author=E%20Matsunaga&author=S%20Tauszig-Delamasure&author=PP%20Monnier&author=BK%20Mueller&author=SM%20Strittmatter&volume=6&publication_year=2004&pages=749-755&pmid=15258591&doi=10.1038/ncb1157&)

34. Schwab JM, Monnier PP, Schluesener HJ, Conrad S, Beschorner R, Chen L, et al. Central nervous system injury-induced repulsive guidance molecule expression in the adult human brain. Arch Neurol. 2005;62:1561–1568. doi: 10.1001/archneur.62.10.1561.  [DOI](https://doi.org/10.1001/archneur.62.10.1561) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16216939/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Neurol&title=Central%20nervous%20system%20injury-induced%20repulsive%20guidance%20molecule%20expression%20in%20the%20adult%20human%20brain&author=JM%20Schwab&author=PP%20Monnier&author=HJ%20Schluesener&author=S%20Conrad&author=R%20Beschorner&volume=62&publication_year=2005&pages=1561-1568&pmid=16216939&doi=10.1001/archneur.62.10.1561&)

35. Satoh J, Tabunoki H, Ishida T, Saito Y, Arima K. Accumulation of a repulsive axonal guidance molecule RGMa in amyloid plaques: a possible hallmark of regenerative failure in Alzheimer's disease brains. Neuropathol Appl Neurobiol. 2013;39:109–120. doi: 10.1111/j.1365-2990.2012.01281.x.  [DOI](https://doi.org/10.1111/j.1365-2990.2012.01281.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22582881/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropathol%20Appl%20Neurobiol&title=Accumulation%20of%20a%20repulsive%20axonal%20guidance%20molecule%20RGMa%20in%20amyloid%20plaques:%20a%20possible%20hallmark%20of%20regenerative%20failure%20in%20Alzheimer's%20disease%20brains&author=J%20Satoh&author=H%20Tabunoki&author=T%20Ishida&author=Y%20Saito&author=K%20Arima&volume=39&publication_year=2013&pages=109-120&pmid=22582881&doi=10.1111/j.1365-2990.2012.01281.x&)

36. Bossers K, Meerhoff G, Balesar R, van Dongen JW, Kruse CG, Swaab DF, et al. Analysis of gene expression in Parkinson's disease: possible involvement of neurotrophic support and axon guidance in dopaminergic cell death. Brain Pathol. 2009;19:91–107. doi: 10.1111/j.1750-3639.2008.00171.x.  [DOI](https://doi.org/10.1111/j.1750-3639.2008.00171.x) | [PMC free article](/articles/PMC8094761/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18462474/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Pathol&title=Analysis%20of%20gene%20expression%20in%20Parkinson's%20disease:%20possible%20involvement%20of%20neurotrophic%20support%20and%20axon%20guidance%20in%20dopaminergic%20cell%20death&author=K%20Bossers&author=G%20Meerhoff&author=R%20Balesar&author=JW%20van%20Dongen&author=CG%20Kruse&volume=19&publication_year=2009&pages=91-107&pmid=18462474&doi=10.1111/j.1750-3639.2008.00171.x&)

37. Nohra R, Beyeen AD, Guo JP, Khademi M, Sundqvist E, Hedreul M, et al. RGMA and IL21R show association with experimental inflammation and multiple sclerosis. Genes Immun. 2010;11:279–293. doi: 10.1038/gene.2009.111.  [DOI](https://doi.org/10.1038/gene.2009.111) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20072140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Immun&title=RGMA%20and%20IL21R%20show%20association%20with%20experimental%20inflammation%20and%20multiple%20sclerosis&author=R%20Nohra&author=AD%20Beyeen&author=JP%20Guo&author=M%20Khademi&author=E%20Sundqvist&volume=11&publication_year=2010&pages=279-293&pmid=20072140&doi=10.1038/gene.2009.111&)

38. Demicheva E, Cui YF, Bardwell P, Barghorn S, Kron M, Meyer AH, et al. Targeting repulsive guidance molecule a to promote regeneration and neuroprotection in multiple sclerosis. Cell Rep. 2015;10:1887–1898. doi: 10.1016/j.celrep.2015.02.048.  [DOI](https://doi.org/10.1016/j.celrep.2015.02.048) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25801027/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&title=Targeting%20repulsive%20guidance%20molecule%20a%20to%20promote%20regeneration%20and%20neuroprotection%20in%20multiple%20sclerosis&author=E%20Demicheva&author=YF%20Cui&author=P%20Bardwell&author=S%20Barghorn&author=M%20Kron&volume=10&publication_year=2015&pages=1887-1898&pmid=25801027&doi=10.1016/j.celrep.2015.02.048&)

39. Mundo E, Tharmalingham S, Neves-Pereira M, Dalton EJ, Macciardi F, Parikh SV, et al. Evidence that the N-methyl-D-aspartate subunit 1 receptor gene (GRIN1) confers susceptibility to bipolar disorder. Mol Psychiatry. 2003;8:241–245. doi: 10.1038/sj.mp.4001218.  [DOI](https://doi.org/10.1038/sj.mp.4001218) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12610658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Evidence%20that%20the%20N-methyl-D-aspartate%20subunit%201%20receptor%20gene%20(GRIN1)%20confers%20susceptibility%20to%20bipolar%20disorder&author=E%20Mundo&author=S%20Tharmalingham&author=M%20Neves-Pereira&author=EJ%20Dalton&author=F%20Macciardi&volume=8&publication_year=2003&pages=241-245&pmid=12610658&doi=10.1038/sj.mp.4001218&)

40. Ge X, Qiu Y, Loh HH, Law PY. GRIN1 regulates micro-opioid receptor activities by tethering the receptor and G protein in the lipid raft. J Biol Chem. 2009;284:36521–36534. doi: 10.1074/jbc.M109.024109.  [DOI](https://doi.org/10.1074/jbc.M109.024109) | [PMC free article](/articles/PMC2794768/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19861419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=GRIN1%20regulates%20micro-opioid%20receptor%20activities%20by%20tethering%20the%20receptor%20and%20G%20protein%20in%20the%20lipid%20raft&author=X%20Ge&author=Y%20Qiu&author=HH%20Loh&author=PY%20Law&volume=284&publication_year=2009&pages=36521-36534&pmid=19861419&doi=10.1074/jbc.M109.024109&)

41. Jo DG, Lee JY, Hong YM, Song S, Mook-Jung I, Koh JY, et al. Induction of pro-apoptotic calsenilin/DREAM/KChIP3 in Alzheimer's disease and cultured neurons after amyloid-beta exposure. J Neurochem. 2004;88:604–611. doi: 10.1111/j.1471-4159.2004.02159.x.  [DOI](https://doi.org/10.1111/j.1471-4159.2004.02159.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14720210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurochem&title=Induction%20of%20pro-apoptotic%20calsenilin/DREAM/KChIP3%20in%20Alzheimer's%20disease%20and%20cultured%20neurons%20after%20amyloid-beta%20exposure&author=DG%20Jo&author=JY%20Lee&author=YM%20Hong&author=S%20Song&author=I%20Mook-Jung&volume=88&publication_year=2004&pages=604-611&pmid=14720210&doi=10.1111/j.1471-4159.2004.02159.x&)

42. Wang JC, Foroud T, Hinrichs AL, Le NX, Bertelsen S, Budde JP, et al. A genome-wide association study of alcohol-dependence symptom counts in extended pedigrees identifies C15orf53. Mol Psychiatry. 2013;18:1218–1224. doi: 10.1038/mp.2012.143.  [DOI](https://doi.org/10.1038/mp.2012.143) | [PMC free article](/articles/PMC3752321/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23089632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=A%20genome-wide%20association%20study%20of%20alcohol-dependence%20symptom%20counts%20in%20extended%20pedigrees%20identifies%20C15orf53&author=JC%20Wang&author=T%20Foroud&author=AL%20Hinrichs&author=NX%20Le&author=S%20Bertelsen&volume=18&publication_year=2013&pages=1218-1224&pmid=23089632&doi=10.1038/mp.2012.143&)

43. Bierut LJ, Strickland JR, Thompson JR, Afful SE, Cottler LB. Drug use and dependence in cocaine dependent subjects, community-based individuals, and their siblings. Drug Alcohol Depen. 2008;95:14–22. doi: 10.1016/j.drugalcdep.2007.11.023.  [DOI](https://doi.org/10.1016/j.drugalcdep.2007.11.023) | [PMC free article](/articles/PMC2384165/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18243582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depen&title=Drug%20use%20and%20dependence%20in%20cocaine%20dependent%20subjects,%20community-based%20individuals,%20and%20their%20siblings&author=LJ%20Bierut&author=JR%20Strickland&author=JR%20Thompson&author=SE%20Afful&author=LB%20Cottler&volume=95&publication_year=2008&pages=14-22&pmid=18243582&doi=10.1016/j.drugalcdep.2007.11.023&)

44. Luo ZH, Alvarado GF, Hatsukami DK, Johnson EO, Bierut LJ, Breslau N. Race differences in nicotine dependence in the Collaborative Genetic study of Nicotine Dependence (COGEND) Nicotine Tob Res. 2008;10:1223–1230. doi: 10.1080/14622200802163266.  [DOI](https://doi.org/10.1080/14622200802163266) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18629733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nicotine%20Tob%20Res&title=Race%20differences%20in%20nicotine%20dependence%20in%20the%20Collaborative%20Genetic%20study%20of%20Nicotine%20Dependence%20(COGEND)&author=ZH%20Luo&author=GF%20Alvarado&author=DK%20Hatsukami&author=EO%20Johnson&author=LJ%20Bierut&volume=10&publication_year=2008&pages=1223-1230&pmid=18629733&doi=10.1080/14622200802163266&)

45. Ward LD, Kellis M. HaploReg v4: systematic mining of putative causal variants, cell types, regulators and target genes for human complex traits and disease. Nucleic Acids Res. 2016;44:D877–881. doi: 10.1093/nar/gkv1340.  [DOI](https://doi.org/10.1093/nar/gkv1340) | [PMC free article](/articles/PMC4702929/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26657631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=HaploReg%20v4:%20systematic%20mining%20of%20putative%20causal%20variants,%20cell%20types,%20regulators%20and%20target%20genes%20for%20human%20complex%20traits%20and%20disease&author=LD%20Ward&author=M%20Kellis&volume=44&publication_year=2016&pages=D877-881&pmid=26657631&doi=10.1093/nar/gkv1340&)
