# UGT1A1*6,1A7*3, and 1A9*22 genotypes predict severe neutropenia in FOLFIRI‐treated metastatic colorectal cancer in two prospective studies in Japan

## Metadata
**Authors:** Shoichi Hazama, Hideyuki Mishima, Ryouichi Tsunedomi, Yusuke Okuyama, Takeshi Kato, Ken‐ichi Takahashi, Hiroshi Nozawa, Hideaki Ando, Michiya Kobayashi, Hiroyoshi Takemoto, Naoki Nagata, Shinsuke Kanekiyo, Yuka Inoue, Yoshihiko Hamamoto, Yusuke Fujita, Yuji Hinoda, Naoko Okayama, Koji Oba, Jun‐ichi Sakamoto, Masaaki Oka
**Journal:** Cancer Science
**Date:** 2013 Oct 27
**DOI:** [10.1111/cas.12283](https://doi.org/10.1111/cas.12283)
**PMID:** 24033692
**PMCID:** PMC7653527
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7653527/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7653527/pdf/CAS-104-1662.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7653527/pdf/CAS-104-1662.pdf)

## Abstract

Retrospective studies have suggested that UDP‐glucuronosyltransferase (UGT)1A1,UGT1A7, and UGT1A9 predict severe toxicity and efficacy of irinotecan‐containing regimens. We prospectively evaluated the impact of UGT1A genotypes and haplotypes on severe toxicity and efficacy in patients treated with fluorouracil, leucovorin, and irinotecan combination chemotherapy (FOLFIRI) for metastatic colorectal cancer (mCRC) from the two prospective multicenter phase II studies in Japan. The FLIGHT1 study was a first‐line FOLFIRI trial, and FLIGHT2 was a FOLFOX‐refractory, second‐line FOLFIRI trial. A total of 73 patients agreed to additional analysis, and were genotyped for UGT1A polymorphisms, UGT1A1*28 (TA6>TA7), UGT1A1*6 (211G>A), UGT1A1*27 (686C>A), UGT1A1*60 (−3279T>G), UGT1A1*93 (−3156G>A), UGT1A7 (−57T>G), UGT1A7*3 (387T>G, 622T>C), and UGT1A9*22 (T9>T10). Of 73 patients, 34 developed G3/4 severe hematological toxicities. The toxicities were significantly more frequent in patients with UGT1A1*6 (211A), UGT1A7 (387G), and UGT1A9*22 reference alleles (T9). Haplotype I, which consists of all favorable alleles, was associated with a significant reduction in hematologic toxicity (P = 0.031). In contrast, haplotype II, which contains four high‐risk alleles, showed significantly higher hematologic toxicity than the other haplotypes (P = 0.010). Six out of seven patients who were homozygous for UGT1A1*28 or *6 experienced severe hematological toxicity despite the fact that their response rate was not impaired (42.9%). We concluded that UGT1A polymorphisms, especially UGT1A1*6, are important for the prediction of severe toxicity of FOLFIRI in northeast Asian populations. In this regard, haplotype analyses should substantially impact the prediction of severe hematological toxicities of FOLFIRI. (Clinical Trial Registration: UMIN000002388 and UMIN000002476).

## Patients and Methods

### Study design and patient eligibility

The objective of this study was to assess the relationship between *UGT1A* polymorphisms and toxicity as well as tumor response. The study was carried out as an ancillary investigation of two prospective studies that involved 20 treatment centers in Japan. The study participants have been described in detail elsewhere.[17](#cas12283-bib-0017), [18](#cas12283-bib-0018) This study was carried out according to the Declaration of Helsinki. The study protocol was approved by the Ethics and Scientific Committee and the institutional review board of each participating institution, and all patients gave written informed consent before entering the study. Eligibility criteria included: histologically proven mCRC; no prior chemotherapy for metastatic disease in the case of FLIGHT1[17](#cas12283-bib-0017) (UMIN000002388), or resistance to FOLFOX as the first‐line chemotherapy in the case of FLIGHT2[18](#cas12283-bib-0018) (UMIN000002476); aged 20–80 years; Eastern Cooperative Oncology Group performance status of 0–1; at least one measurable lesion according to the Response Evaluation Criteria in Solid Tumors (RECIST) version 1.0[19](#cas12283-bib-0019); and adequate organ function.

Thirty‐nine of 53 patients from the FLIGHT1 study and 36 of 50 patients from the FLIGHT2 study agreed to the additional investigation. Two patients who were homozygous for *UGT1A1*28* and received a reduced dosage of irinotecan were excluded from the whole analysis of toxicities and efficacies and analyzed individually.

### Treatment

All patients were treated with the modified FOLFIRI regimen as described by Tournigand *et al*.[3](#cas12283-bib-0003) Briefly, the regimen consisted of irinotecan 150 mg/m^2^ (instead of 180 mg/m^2^ commonly used in North America and Europe) given i.v. for 90 min on day 1 + 400 mg/m^2^ fluorouracil bolus followed by 2400 mg/m^2^ fluorouracil continuous infusion during 46 h + 200 mg/m^2^ leucovorin on day 1 every 2 weeks. Blood tests and clinical evaluation were carried out every 2 weeks before treatment. Chemotherapy could be given if patients had a leukocyte count >3000 mm^3^, a neutrophil count >1500 mm^3^, a platelet count >75 000 mm^3^, and clinical toxicity that was resolved or grade 1. In patients who were homozygous for *UGT1A1*28*, the starting dose of irinotecan was reduced to 100 mg/m^2^, based on the recommendation of an advisory meeting by the subcommittee of the Food and Drug Administration Center for Drug Evaluation and Research held November 2004[20](#cas12283-bib-0020) and our previous phase I study.[21](#cas12283-bib-0021) Two patients were homozygous for **28* and therefore excluded from the whole analysis of toxicities and efficacies and analyzed individually.

### Efficacy and toxicity assessment

Objective clinical evaluation, blood counts, and hepatic and renal function tests were carried out within 48 h before each cycle. Computed tomography scans of measurable lesions were assessed at baseline and then repeated at least every four cycles. Objective tumor response and duration of response were assessed by RECIST version 1.0. Toxicity was evaluated according to Common Toxicity Criteria for Adverse Events version 3.0.

### Uridine diphosphate–glucuronosyltransferase genetic analyses

Genomic DNA was extracted from peripheral blood anticoagulated with EDTA‐2Na using a conventional NaI method.[22](#cas12283-bib-0022) *UGT1A1*28* (TA6>TA7), UGT1A7*3 (387T>G, 622T>C)/*UGT1A9*22* (**1b* [T9>T10]), and *UGT1A1*93* (−3156G>A)/*UGT1A1*6* (211G>A)/*UGT1A1*27* (686C>A)/*UGT1A1*60* (−3279T>G)/*UGT1A7* (−57T>G) were genotyped as described previously by fragment size analysis, direct sequencing, and TaqMan assay, respectively.[21](#cas12283-bib-0021), [23](#cas12283-bib-0023) *UGT1A7**1, *UGT1A7*2*,* UGT1A7*3*, and *UGT1A7*4* can be tagged by sequencing of 387T>G and 622T>C.[24](#cas12283-bib-0024) For fragment size analysis, PCR reactions were carried out in a total volume of 10 μL containing template DNA (80 ng/μL), according to the manufacturer's instructions (Ex Taq; Takara, Tokyo, Japan). The amplification was carried out with a GeneAmp PCR System PC808 (ASTEC, Tokyo, Japan) with an initial denaturation at 95°C for 2 min followed by 27 cycles of denaturation at 94°C for 30 s, annealing at 60°C for 20 s, and extension at 72°C for 30 s. The PCR products of TA6 and TA7, whose sizes were 94 and 96 bp, respectively, were mixed with Hi‐Di formamide, including the internal size standard (GeneScan 500; Applied Biosystems, Foster City, CA, USA) at a 1:10 ratio. Then, samples were run in the ABI Prism 3100 Genetic Analyzer (Applied Biosystems). Fragment sizes were determined by comparison with the internal size standard (GeneScan LIZ‐500) by using the local Southern algorithm, and the data were analyzed by GeneMapper software version 3.5 (Applied Biosystems). For direct sequencing, PCR amplifications were carried out by using the Gene Amp PCR System PC808 (ASTEC) with Ex Taq polymerase. Amplification conditions were 30 cycles of 95°C for 30 s, each annealing temperature for 20 s, and 72°C for 30 s. The PCR products were purified using ExoSAP‐IT (Amersham Bioscience, Tokyo, Japan) for 20 min at 37°C and then for 20 min at 80°C. Sequencing reactions were carried out by using a BigDye Terminator Cycle Sequencing Kit (Applied Biosystems, Tokyo, Japan). After purification with ethanol, the reaction products were analyzed by an ABI 3100‐Avant Genetic Analyzer (Applied Biosystems). TaqMan assays of PCR products were carried out according to the manufacturer's protocol. Specific forward/reverse PCR primers and TaqMan probes for *UGT1A1*93* were custom synthesized by Applied Biosystems. Primers and probes for *UGT1A1*6*,* UGT1A1*27*,* UGT1A1*60*, and *UGT1A7* (−57) were purchased from Applied Biosystems (TaqMan SNP Genotyping Assays). Reaction mixtures were loaded into 384‐well plates and placed in the ABI Prism 7900HT Sequence Detection System (Applied Biosystems). The PCR amplifications were carried out as follows: initial denaturation at 95°C for 10 min, followed by 40 cycles of PCR with a denaturation at 95°C for 15 s, and one‐step annealing/extension for 1 min at 60°C.

### Statistical analysis

These analyses are an additional investigation of the FLIGHT1[17](#cas12283-bib-0017) and FLIGHT2[18](#cas12283-bib-0018) studies, which were prospectively designed to evaluate the associations between *UGT1A* genotypes/haplotypes and outcomes of toxicity (severe toxicity at first cycle and in all cycles) and efficacy (response rate [complete response, partial response] and progression‐free survival [PFS]). In the present study, we evaluated associations between genotypes/haplotypes as well as clinical variables (sex, age, and performance status), toxicity, and response rate by Fisher's exact test and the Cochran–Armitage trend test (C.A.‐test). Logistic regression models were investigated for toxicity. Genetic variants were included in a stepwise logistic regression analysis in the multivariate model. Calculations were carried out by using R version 2.13.0 software.[25](#cas12283-bib-0025) Each nucleotide polymorphism was evaluated for Hardy–Weinberg equilibrium and LD, and case–control haplotype analyses were carried out with Haploview 4.2 software.[26](#cas12283-bib-0026) Lewontin's coefficient D' and correlation coefficient *r* ^2^ were calculated as measures of LD. *P* < 0.05 was considered statistically significant.

## Results

### Clinical features, genotype/haplotype frequency, and LD

In the 75 Japanese colorectal cancer patients, genotypes of *UGT1A1*28* and *UGT1A1*93* as well as *UGT1A7* (622T>C) and *UGT1A7* (−57T>G) matched perfectly, so *UGT1A1*93* and *UGT1A7* (−57T>G) were excluded from further examination (Table [1](#cas12283-tbl-0001), Fig. [1](#cas12283-fig-0001)). Minor allele frequencies of seven remaining *UGT1A* polymorphisms are shown in Table S1. The *P*‐values of all seven *UGT1A* polymorphisms were >0.05, under the Hardy–Weinberg equilibrium. High LD was also observed between *UGT1A1*6* and *UGT1A7* (622T>C), *UGT1A1*6* and *UGT1A7* (−57T>G), and *UGT1A7* (387T>G) and *UGT1A9*22(*1b)* (Fig. [1](#cas12283-fig-0001)).

### Table 1.

| Clinical feature or genotype | Detail | Frequency (%) |
| --- | --- | --- |
| Sex | Male | 50 (66.7) |
| Female | 25 (33.3) |  |
| Age, years | ≤60 | 28 (37.3) |
| >60 | 47 (62.7) |  |
| Performance statusa | 0 | 58 (77.3) |
| 1 | 17 (22.7) |  |
| Hematologic toxicity (entire course) | No | 40 (53.3) |
| Yes | 35 (46.7) |  |
| Hematologic toxicity(first cycle) | No | 63 (84.0) |
| Yes | 12 (16.0) |  |
| Objective response | CR + PR | 24 (33.3) |
| SD + PD | 48 (66.7) |  |
| UGT1A1*6 | −/− | 50 (66.7) |
| −/*6 | 23 (30.7) |  |
| *6/*6 | 2 (2.7) |  |
| UGT1A1*27 | −/− | 73 (97.3) |
| −/*27 | 2 (2.7) |  |
| UGT1A1*28 | −/− | 59 (78.7) |
| −/*28 | 14 (18.7) |  |
| *28/*28 | 2 (2.7) |  |
| UGT1A1*60 | −/− | 40 (53.3) |
| −/*60 | 29 (38.7) |  |
| *60/*60 | 6 (8.0) |  |
| UGT1A1*93 | −/− | 59 (78.7) |
| −/*93 | 14 (18.7) |  |
| *93/*93 | 2 (2.7) |  |
| UGT1A7 (−57T>G) | −57T/T | 40 (53.3) |
| −57T/G | 29 (38.7) |  |
| −57G/G | 6 (8.0) |  |
| UGT1A7*3 (387T>G) | 387T/T | 25 (33.3) |
| 387T/G | 37 (49.3) |  |
| 387G/G | 13 (17.3) |  |
| UGT1A7*3 (622T>C) | 622T/T | 40 (53.3) |
| 622T/C | 29 (38.7) |  |
| 622C/C | 6 (8.0) |  |
| UGT1A9*22 (UGT1A9*1b) | *22/*22 | 26 (34.7) |
| −/*22 | 36 (48.0) |  |
| −/− | 13 (17.3) |  |

Table 1 Caption: Clinical features and genotype frequencies in Japanese patients with metastatic colorectal cancer

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/312a/7653527/e859aecec881/CAS-104-1662-g001.jpg)

Pairwise linkage disequilibrium relationships between UGT1A polymorphisms in Japanese patients with metastatic colorectal cancer. Lewontin's coefficient D' is represented by the color scheme: log of the odds (LOD) ≥ 2 shown in pink/red; LOD < 2 and D' = 1 shown in blue; and LOD < 2 and D' < 1 shown in white. The correlation coefficient r 2 is shown in each box.

The frequency of *UGT1A1*28* was lower (Table [1](#cas12283-tbl-0001)) in the Japanese patients than Caucasians as reported previously.[9](#cas12283-bib-0009), [27](#cas12283-bib-0027) *UGT1A1*6* frequency was discriminatively high, but it was rare among Caucasians.[28](#cas12283-bib-0028) The frequency of *UGT1A1*60* and *UGT1A7*3* was low in Japanese patients, *UGT1A9*22* had the same frequency in Japanese and Caucasian patients, and *UGT1A1*27* had very low frequency and no homozygosity.

Haplotype frequencies are listed in Table [2](#cas12283-tbl-0002). The major haplotypes in the Japanese population were I, II, and III, and our analyses were performed on these.

### Table 2.

| Hp | UGT1A1 |  |  |  | UGT1A7 |  | UGT1A9 |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| *60 | *28 | *6 | *27 | 387T>G | 622T>C | *22 | Frequency |  |
| I | T | TA6 | G | C | T | T | T10 | 0.520 |
| II | T | TA6 | Aa | C | Ga | Ca | T9 a | 0.173 |
| III | Ga | TA6 | G | C | Ga | T | T9 a | 0.127 |
| IV | Ga | TA7 a | G | C | Ga | Ca | T9 a | 0.053 |
| V | Ga | TA7 a | G | C | T | T | T10 | 0.053 |
| VI | T | TA6 | G | C | Ga | Ca | T9 a | 0.020 |
| VII | Ga | TA7 a | G | Aa | Ga | T | T9 a | 0.013 |
| VIII | Ga | TA6 | G | C | Ga | Ca | T9 a | 0.013 |
| IX | Ga | TA6 | G | C | T | T | T10 | 0.013 |

Table 2 Caption: UGT1A haplotype frequencies in Japanese patients with metastatic colorectal cancer

### UGT1A genotypes/haplotypes and severe toxicity

Correlations between *UGT1A* genotypes/haplotypes and severe (grade 3–4) toxicity were analyzed. Of 73 patients, 34 had G3/4 hematological toxicity. Severe hematological toxicity during the entire course of therapy was more frequent in patients with *UGT1A1*6* (211A, *P* = 0.018 by C.A.‐test) and *UGT1A7* (387G, *P* = 0.039 by C.A.‐test) alleles than in patients without these variant alleles. In the *UGT1A9*22* polymorphism, the variant T10 allele was a marker for reduced toxicity (*P* = 0.028 by C.A.‐test), as previously reported.[8](#cas12283-bib-0008), [9](#cas12283-bib-0009) Severe hematologic toxicity was trend toward higher among the *UGT1A7* (622C) allele (*P* = 0.058 by Fisher's exact test). For severe hematological toxicity during the entire course of therapy, multivariate analysis indicated that *UGT1A1*6* was the only significant predictor (*P* = 0.022, odds ratio 3.00, 95% confidence interval 1.17–7.69; Table S5). Older age (60 years or more) was also a risk factor for severe toxicity (Table [3](#cas12283-tbl-0003)).

### Table 3.

| Genotype or clinical feature | Detail | Severe hematologic toxicity |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| After first cycle |  |  |  | During entire course of therapy |  |  |  |  |  |
| Toxicity |  | P‐value |  | Toxicity |  | P‐value |  |  |  |
| Yes | No | Fisher's exact test | C.A.‐trend test | Yes | No | Fisher's exact test | C.A.‐trend test |  |  |
| UGT1A1*6 | −/− | 6 | 42 | 0.175 | 0.132 | 18 | 30 | 0.049 | 0.018 |
| −/*6 | 5 | 18 | 14 | 9 |  |  |  |  |  |
| *6/*6 | 1 | 1 | 2 | 0 |  |  |  |  |  |
| UGT1A1*27 | −/− | 11 | 60 | 0.304 | ND | 33 | 38 | 1.000 | ND |
| −/*27 | 1 | 1 | 1 | 1 |  |  |  |  |  |
| UGT1A1*28 | −/− | 8 | 51 | 0.227 | ND | 27 | 32 | 1.000 | ND |
| −/*28 | 4 | 10 | 7 | 7 |  |  |  |  |  |
| UGT1A1*60 | −/− | 4 | 36 | 0.114 | 0.313 | 16 | 24 | 0.402 | 0.279 |
| −/*60 | 8 | 21 | 16 | 13 |  |  |  |  |  |
| *60/*60 | 0 | 4 | 2 | 2 |  |  |  |  |  |
| UGT1A7 (387T>G) | 387T/T | 2 | 23 | 0.360 | 0.328 | 7 | 18 | 0.079 | 0.039 |
| 387T/G | 8 | 28 | 20 | 16 |  |  |  |  |  |
| 387G/G | 2 | 10 | 7 | 5 |  |  |  |  |  |
| UGT1A7 (622T>C) | 622T/T | 4 | 36 | 0.191 | 0.162 | 14 | 26 | 0.058 | 0.105 |
| 622T/C | 7 | 21 | 18 | 10 |  |  |  |  |  |
| 622C/C | 1 | 4 | 2 | 3 |  |  |  |  |  |
| UGT1A9*22 (UGT1A9*1b) | *22/*22 | 2 | 24 | 0.356 | 0.296 | 7 | 19 | 0.044 | 0.028 |
| −/*22 | 8 | 27 | 20 | 15 |  |  |  |  |  |
| −/− | 2 | 10 | 7 | 5 |  |  |  |  |  |
| Haplotype I | 0 | 3 | 12 | 0.041 | 0.080 | 9 | 6 | 0.072 | 0.031 |
| 1 | 9 | 29 | 20 | 18 |  |  |  |  |  |
| 2 | 0 | 20 | 5 | 15 |  |  |  |  |  |
| Haplotype II | 0 | 6 | 43 | 0.170 | 0.106 | 18 | 31 | 0.025 | 0.010 |
| 1 | 5 | 17 | 14 | 8 |  |  |  |  |  |
| 2 | 1 | 1 | 2 | 0 |  |  |  |  |  |
| Haplotype III | 0 | 10 | 45 | 0.765 | 0.449 | 24 | 31 | 0.488 | 0.282 |
| 1 | 2 | 15 | 9 | 8 |  |  |  |  |  |
| 2 | 0 | 1 | 1 | 0 |  |  |  |  |  |
| Sex | Male | 8 | 40 | 1.000 | ND | 22 | 26 | 1.000 | ND |
| Female | 4 | 21 | 12 | 13 |  |  |  |  |  |
| Age, years | ≤60 | 2 | 26 | 0.114 | ND | 8 | 20 | 0.018 | ND |
| >60 | 10 | 35 | 26 | 19 |  |  |  |  |  |
| PS | 0 | 9 | 48 | 0.718 | ND | 27 | 30 | 1.000 | ND |
| 1 | 3 | 13 | 7 | 9 |  |  |  |  |  |

Table 3 Caption: UGT1A genotypes/clinical features and severe hematologic toxicity in Japanese patients with metastatic colorectal cancer treated with FOLFIRI

Haplotype I, which consisted of all favorable alleles, was associated with low hematologic toxicity both after the first cycle (*P* = 0.080 by C.A.‐test) and during the entire course of therapy (*P* = 0.031 by C.A.‐test). In contrast, haplotype II, which contains four high‐risk alleles, was significantly associated with high hematologic toxicity during the entire course of therapy (*P* = 0.010 by C.A.‐test) (Table [3](#cas12283-tbl-0003)).

In the trial of first‐line therapy (FLIGHT1, *n* = 38, Table [4](#cas12283-tbl-0004)), *UGT1A1*28* (*P* = 0.063 by Fisher's exact test) genotypes were risk factors for severe hematological toxicity after the first cycle, and the allele number of *UGT1A7* (387G, *P* = 0.026 by C.A.‐test) and *UGT1A9*22* (T10, *P* = 0.016 by C.A.‐test) were risk factors for severe hematological toxicity during the entire course of therapy.

### Table 4.

| Genotype or clinical feature | Detail | Severe hematologic toxicity |  |  |  |  |  |  |  | Objective responses |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| After first cycle |  |  |  | During entire course of therapy |  |  |  | During entire course of therapy |  |  |  |  |  |
| Toxicity |  | P‐value |  | Toxicity |  | P‐value |  | Responses |  | P‐value |  |  |  |
| Yes | No | Fisher's exact test | C.A.‐ trend test | Yes | No | Fisher's exact test | C.A.‐ trend test | CR + PR | SD + PD | Fisher's exact test | C.A.‐ trend test |  |  |
| UGT1A1*6 | –/– | 4 | 21 | 0.243 | 0.266 | 10 | 15 | 0.384 | 0.150 | 11 | 12 | 0.725 | 1.000 |
| –/*6 | 2 | 10 | 7 | 5 | 7 | 5 |  |  |  |  |  |  |  |
| *6/*6 | 1 | 0 | 1 | 0 | 0 | 1 |  |  |  |  |  |  |  |
| UGT1A1*27 | –/– | 6 | 31 | 0.184 | ND | 17 | 20 | 0.474 | ND | 17 | 18 | 1.000 | ND |
| –/*27 | 1 | 0 | 1 | 0 | 1 | 0 |  |  |  |  |  |  |  |
| UGT1A1*28 | –/– | 4 | 28 | 0.063 | ND | 14 | 18 | 0.395 | ND | 16 | 15 | 1.000 | ND |
| –/*28 | 3 | 3 | 4 | 2 | 2 | 3 |  |  |  |  |  |  |  |
| UGT1A1*60 | –/– | 1 | 18 | 0.079 | 0.133 | 7 | 12 | 0.206 | 0.095 | 8 | 11 | 0.109 | 0.781 |
| –/*60 | 6 | 11 | 9 | 8 | 10 | 5 |  |  |  |  |  |  |  |
| *60/*60 | 0 | 2 | 2 | 0 | 0 | 2 |  |  |  |  |  |  |  |
| UGT1A7 (387T>G) | 387T/T | 1 | 12 | 0.490 | 0.213 | 4 | 9 | 0.065 | 0.026 | 7 | 5 | 0.678 | 0.815 |
| 387T/G | 4 | 14 | 8 | 10 | 7 | 10 |  |  |  |  |  |  |  |
| 387G/G | 2 | 5 | 6 | 1 | 4 | 3 |  |  |  |  |  |  |  |
| UGT1A7 (622T>C) | 622T/T | 2 | 19 | 0.113 | 0.079 | 7 | 14 | 0.132 | 0.102 | 10 | 10 | 1.000 | 1.000 |
| 622T/C | 4 | 11 | 10 | 5 | 7 | 7 |  |  |  |  |  |  |  |
| 622C/C | 1 | 1 | 1 | 1 | 1 | 1 |  |  |  |  |  |  |  |
| UGT1A9*22 (UGT1A9*1b) | *22/*22 | 1 | 13 | 0.413 | 0.183 | 4 | 10 | 0.057 | 0.016 | 8 | 5 | 0.474 | 0.646 |
| –/*22 | 4 | 13 | 8 | 9 | 6 | 10 |  |  |  |  |  |  |  |
| –/– | 2 | 5 | 6 | 1 | 4 | 3 |  |  |  |  |  |  |  |
| Haplotype I | 0 | 3 | 5 | 0.100 | 0.040 | 7 | 1 | 0.028 | 0.019 | 4 | 4 | 1.000 | 1.000 |
| 1 | 4 | 16 | 8 | 12 | 9 | 9 |  |  |  |  |  |  |  |
| 2 | 0 | 10 | 3 | 7 | 5 | 5 |  |  |  |  |  |  |  |
| Haplotype II | 0 | 4 | 22 | 0.249 | 0.202 | 10 | 16 | 0.212 | 0.080 | 12 | 12 | 1.000 | 0.755 |
| 1 | 2 | 9 | 7 | 4 | 6 | 5 |  |  |  |  |  |  |  |
| 2 | 1 | 0 | 1 | 0 | 0 | 1 |  |  |  |  |  |  |  |
| Haplotype III | 0 | 6 | 22 | 0.720 | 0.398 | 12 | 16 | 0.568 | 0.253 | 13 | 13 | 1.000 | 0.747 |
| 1 | 1 | 8 | 5 | 4 | 5 | 4 |  |  |  |  |  |  |  |
| 2 | 0 | 1 | 1 | 0 | 0 | 1 |  |  |  |  |  |  |  |
| Sex | Male | 5 | 19 | 1.000 | ND | 12 | 12 | 0.745 | ND | 13 | 10 | 0.489 | ND |
| Female | 2 | 12 | 6 | 8 | 5 | 8 |  |  |  |  |  |  |  |
| Age, years | ≤60 | 2 | 12 | 1.000 | ND | 5 | 9 | 0.328 | ND | 6 | 7 | 1.000 | ND |
| >60 | 5 | 19 | 13 | 11 | 12 | 11 |  |  |  |  |  |  |  |
| PS | 0 | 5 | 28 | 0.223 | ND | 15 | 18 | 0.653 | ND | 17 | 15 | 0.603 | ND |
| 1 | 2 | 3 | 3 | 2 | 1 | 3 |  |  |  |  |  |  |  |

Table 4 Caption: Associations between UGT1A genotypes and irinotecan toxicity and objective responses in Japanese patients with metastatic colorectal cancer treated with FOLFIRI as first‐line therapy (FLIGHT1)

In the trial of second‐line therapy (FLIGHT2, *n* = 35, Table [5](#cas12283-tbl-0005)), older age (60 years and older, *P* = 0.069 by Fisher's exact test) trends to a risk factor for severe hematological toxicity after the first cycle. The severe hematological toxicity during the entire course of therapy was significantly different among the genotypes of *UGT1A7* (387G, *P* = 0.047 by Fisher's exact test), *UGT1A9*22* (T9, *P* = 0.047 by Fisher's exact test), and patient age (*P* = 0.036 by Fisher's exact test).

### Table 5.

| Genotype or clinical feature | Detail | Severe hematologic toxicity |  |  |  |  |  |  |  | Objective responses |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| After first cycle |  |  |  | During entire course of therapy |  |  |  | During entire course of therapy |  |  |  |  |  |
| Toxicity |  | P‐value |  | Toxicity |  | P‐value |  | Responses |  | P‐value |  |  |  |
| Yes | No | Fisher's exact test | C.A.‐trend test | Yes | No | Fisher's exact test | C.A.‐trend test | CR + PR | SD + PD | Fisher's exact test | C.A.‐trend test |  |  |
| UGT1A1*6 | −/− | 2 | 21 | 0.400 | 0.306 | 8 | 15 | 0.105 | 0.054 | 2 | 21 | 0.058 | 0.021 |
| −/*6 | 3 | 8 | 7 | 4 | 3 | 8 |  |  |  |  |  |  |  |
| *6/*6 | 0 | 1 | 1 | 0 | 1 | 0 |  |  |  |  |  |  |  |
| UGT1A1*27 | −/− | 5 | 29 | 1.000 | ND | 16 | 18 | 1.000 | ND | 6 | 28 | 1.000 | ND |
| −/*27 | 0 | 1 | 0 | 1 | 0 | 1 |  |  |  |  |  |  |  |
| UGT1A1*28 | −/− | 4 | 23 | 1.000 | ND | 13 | 14 | 0.700 | ND | 6 | 21 | 0.299 | ND |
| −/*28 | 1 | 7 | 3 | 5 | 0 | 8 |  |  |  |  |  |  |  |
| UGT1A1*60 | −/− | 3 | 18 | 1.000 | 0.819 | 9 | 12 | 0.414 | 0.859 | 6 | 15 | 0.106 | 0.041 |
| −/*60 | 2 | 10 | 7 | 5 | 0 | 12 |  |  |  |  |  |  |  |
| *60/*60 | 0 | 2 | 0 | 2 | 0 | 2 |  |  |  |  |  |  |  |
| UGT1A7 (387T>G) | 387T/T | 1 | 11 | 0.528 | 1.000 | 3 | 9 | 0.047 | 0.542 | 2 | 10 | 1.000 | 0.893 |
| 387T/G | 4 | 14 | 12 | 6 | 3 | 15 |  |  |  |  |  |  |  |
| 387G/G | 0 | 5 | 1 | 4 | 1 | 4 |  |  |  |  |  |  |  |
| UGT1A7 (622T>C) | 622T/T | 2 | 17 | 0.612 | 0.831 | 7 | 12 | 0.403 | 0.491 | 2 | 17 | 0.317 | 0.228 |
| 622T/C | 3 | 10 | 8 | 5 | 3 | 10 |  |  |  |  |  |  |  |
| 622C/C | 0 | 3 | 1 | 2 | 1 | 2 |  |  |  |  |  |  |  |
| UGT1A9*22 (UGT1A9*1b) | *22/*22 | 1 | 11 | 0.528 | 1.000 | 3 | 9 | 0.047 | 0.542 | 2 | 10 | 1.000 | 0.893 |
| −/*22 | 4 | 14 | 12 | 6 | 3 | 15 |  |  |  |  |  |  |  |
| −/− | 0 | 5 | 1 | 4 | 1 | 4 |  |  |  |  |  |  |  |
| Haplotype I | 0 | 0 | 7 | 0.095 | 0.765 | 2 | 5 | 0.049 | 0.501 | 1 | 6 | 1.000 | 0.753 |
| 1 | 5 | 13 | 12 | 6 | 3 | 15 |  |  |  |  |  |  |  |
| 2 | 0 | 10 | 2 | 8 | 2 | 8 |  |  |  |  |  |  |  |
| Haplotype II | 0 | 2 | 21 | 0.400 | 0.306 | 8 | 15 | 0.105 | 0.054 | 2 | 21 | 0.058 | 0.021 |
| 1 | 3 | 8 | 7 | 4 |   | 3 | 8 |  |  |  |  |  |  |
| 2 | 0 | 1 | 1 | 0 |   | 1 | 0 |  |  |  |  |  |  |
| Haplotype III | 0 | 4 | 23 | 1.000 | ND | 12 | 15 | 1.000 | ND | 6 | 21 | 0.299 | ND |
| 1 | 1 | 7 | 4 | 4 | 0 | 8 |  |  |  |  |  |  |  |
| 2 | 0 | 0 | 0 | 0 | 0 | 0 |  |  |  |  |  |  |  |
| Sex | Male | 3 | 21 | 0.640 | ND | 10 | 14 | 0.716 | ND | 4 | 20 | 1.000 | ND |
| Female | 2 | 9 | 6 | 5 | 2 | 9 |  |  |  |  |  |  |  |
| Age, years | ≤60 | 0 | 14 | 0.069 | ND | 3 | 11 | 0.036 | ND | 2 | 12 | 1.000 | ND |
| >60 | 5 | 16 | 13 | 8 |   | 4 | 17 |  |  |  |  |  |  |
| PS | 0 | 4 | 20 | 1.000 | ND | 12 | 12 | 0.493 | ND | 5 | 19 | 0.640 | ND |
| 1 | 1 | 10 | 4 | 7 | 1 | 10 |  |  |  |  |  |  |  |

Table 5 Caption: Associations between UGT1A genotypes and irinotecan toxicity and objective responses in Japanese patients with metastatic colorectal cancer treated with FOLFIRI as second‐line therapy (FLIGHT2)

Only two patients, one in FLIGHT1 and one in FLIGHT2, developed grade 3 diarrhea. Both of them had UGT1A diplotype I/II.

### UGT1A genotypes/haplotypes and response/PFS

Objective tumor responses were analyzed in 71 patients. Of 36 patients, 18 had complete or partial tumor responses in FLIGHT 1, and 6 out of 35 patients had objective responses in FLIGHT 2. No genotype/haplotype was associated with tumor response in FLIGHT 1. In FLIGHT2, the numbers of *UGT1A1*6* and haplotype II were positive factors (*P* = 0.021 by C.A.‐test) and *UGT1A1*60* was a negative factor (*P* = 0.041 by C.A.‐test) for objective responses (Table [5](#cas12283-tbl-0005)).

Effects of UGT1A polymorphisms on PFS in FLIGHT1 and FLIGHT2 were analyzed. There was no association between UGT1A polymorphisms and PFS. Only female sex was the negative factor for PFS (*P* = 0.016) (Table S6).

### UGT1A1*28 and UGT1A1*6 homozygous

Homozygosity of *UGT1A1*28* or **6* and compound heterozygosity of **28* and **6* were reported as high‐risk factors for hematologic toxicity, especially in Asia,[29](#cas12283-bib-0029) therefore, we examined these genotypes (Table S7). Although the initial dose of irinotecan was reduced to 100 mg/m^2^ for patients with homozygous *UGT1A1*28*, one of two patients suffered from severe toxicity. In all cases, six of seven patients suffered from severe hematological toxicity. Although in these patients the toxicity was very severe, the response rate was not impaired (3/7, 42.9%).

## Discussion

The present study evaluated whether *UGT1A* polymorphisms influence the toxicity and efficacy of the FOLFIRI regimen in Japanese mCRC patients. This study was carried out as an integrated investigation of two prospective studies that involved 20 treatment centers in Japan.[17](#cas12283-bib-0017), [18](#cas12283-bib-0018) To the best of our knowledge, this is the first prospective study to assess the role of *UGT1A* polymorphisms on the efficacy and toxicity of FOLFIRI.

The *UGT1A1*,* UGT1A7*, and *UGT1A9* genetic variants and their haplotypes play a predictive role in the outcome of FOLFIRI treatments in Caucasian mCRC patients.[9](#cas12283-bib-0009) There are ethnic differences in *UGT1A* genotypes. Among the Japanese population, *UGT1A1*28* has a low frequency and *UGT1A1*6* has a discriminatively high frequency, unlike the frequencies among Caucasians.[11](#cas12283-bib-0011), [12](#cas12283-bib-0012), [16](#cas12283-bib-0016), [17](#cas12283-bib-0017) Therefore, the relationship between *UGT1A* genetics and the outcome of FOLFIRI should be evaluated in each ethnic group.

In our analysis of Japanese colorectal cancer patients treated with FOLFIRI, severe hematological toxicity during the entire course of therapy was more frequent in patients with *UGT1A1*6* (211A) and *UGT1A7*3* (387G) genotypes. In contrast, the variant T10 allele of the *UGT1A9*22* (*1b) polymorphism was a marker for reduced toxicity (Table [3](#cas12283-tbl-0003)).[8](#cas12283-bib-0008), [9](#cas12283-bib-0009) In addition to certain genotypes, older age was also a risk factor for severe hematological toxicity. Unlike findings from Caucasian patients, the *UGT1A1*28* allele was not necessarily related to hematological toxicity in Japanese patients. On the contrary, *UGT1A1*6* was strongly correlated with severe hematological toxicity, which is consistent with other clinical studies of Asian populations.[11](#cas12283-bib-0011), [12](#cas12283-bib-0012), [16](#cas12283-bib-0016) We can assume that both *UGT1A9*22* (reference T9 genotype) and *UGT1A7*3* (387G) are risk alleles that have strong LD (correlation coefficient *r* ^2^ = 0.97) for each other (Fig. [1](#cas12283-fig-0001)), as previously reported.[9](#cas12283-bib-0009) Interestingly, there was no significant relationship between hematological toxicity and female sex, unlike the previous findings of Cecchin *et al*.[9](#cas12283-bib-0009) for Caucasians.

The close LD between *UGT1A1*6* and *UGT1A7*3* (387T>G, 0.32; 622T>C, 0.64), as well as between *UGT1A9*22* and *UGT1A7*3* (387T>G, 0.97; 622T>C, 0.46) was thought to be associated with toxicity profiles of each genotype. Therefore, the influence of not only genotypes but also haplotypes must be examined.[9](#cas12283-bib-0009), [14](#cas12283-bib-0014), [30](#cas12283-bib-0030) For haplotype I, which consists of all favorable alleles, there was a significant reduction in hematologic toxicity during the entire course of therapy (*P* = 0.031) as well as after the first cycle (*P* = 0.080). This protective haplotype was the most frequent haplotype (0.520) in Japanese patients.[9](#cas12283-bib-0009) In contrast, haplotype II, which contains four high‐risk alleles (UGT1A1*6, *UGT1A7*3* [387G and 622C] and UGT1A9*1 [T9]), showed significantly higher hematologic toxicity during the entire course of therapy (haplotype frequency, 0.173; *P* = 0.010). The odds ratio for that toxicity was extremely high, and this haplotype was the second most common haplotype in Japanese patients, which differs considerably from that of Caucasians. This haplotype may be the leading cause of ethnic differences in irinotecan toxicity. Consequently, haplotype analysis seemed to have a stronger impact than each genotype analysis. Although toxicity was very severe in these patients, the response rate was not impaired.

There were some differences in profiles between first‐line and second‐line FOLFIRI treatments. In the first‐line group, *UGT1A1*28* was a risk factor for severe hematological toxicity after the first cycle of FOLFIRI, and *UGT1A7* (387G) and *UGT1A9*22* (T9) genotypes were risk factors during the entire course of therapy. In the second‐line group, older age was a risk factor after the first cycle, and *UGT1A7* (387G) and *UGT1A9* (T9) alleles and older age were risk factors during the entire course of therapy for severe hematological toxicity. In the second‐line therapy group, older patients could not tolerate the sequential treatments. Objective tumor responses were predicted by *UGT1A1*6* and *UGT1A1*60*, but these results might be controversial.

Homozygosity of *UGT1A1*28* or **6* and compound heterozygosity of **28* and **6* have been reported as high‐risk factors for hematologic toxicity, especially in Asia.[29](#cas12283-bib-0029) In our study, all **28/*6* and **6/*6* patients developed severe hematological toxicity. Moreover, one of two **28* homozygous patients suffered from severe hematological toxicity despite the reduced dosage of irinotecan (100 mg/m^2^) used for these patients (Table S7). Although toxicity was very severe in these patients, the response rate was not impaired (3/7: 42.9%). For these patients, although the FOLFIRI regimen should be given in a careful manner, it is effective, and its use for advanced colorectal cancer patients should not be precluded from the treatment options.

In conclusion, assessment of *UGT1A1*28* and **6* and also *UGT1A7*3* and *UGT1A9*22* is very important to predict the toxicity of irinotecan in Japanese (or Asian) patients. Although each *UGT1A1*6* (211A), *UGT1A7*3* (387G), and *UGT1A9*22* (T9) allele might predict hematological toxicity, haplotype II (containing four risk alleles, *UGT1A1*6*,* UGT1A7*3* [387G and 622C], *UGT1A9*1* [T9]) and homozygosity of *UGT1A1*28* and **6* were better predictors of toxicity. For these patients, the FOLFIRI regimen should be given carefully. We will attempt further haplotype analysis by carrying out a new prospective study and using bioinformatics.

## Disclosure Statement

The authors have no conflict of interest.

## Supporting information

## Acknowledgments

This study was supported in part by a non‐profit organization Epidemiological and Clinical Research Information Network (ECRIN), as well as a Grant‐in‐Aid for Scientific Research of the Ministry of Education, Science, Sports and Culture of Japan (project No. 21591725 and 19591545). We thank Ms. Mai Hatta for her excellent clinical research coordination.

(Cancer Sci 2013; 104: 1662–1669)

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Douillard JY, Cunningham D, Roth AD et al Irinotecan combined with fluorouracil compared with fluorouracil alone as first‐line treatment for metastatic colorectal cancer: a multicentre randomised trial. Lancet 2000; 355: 1041–7.  [DOI](https://doi.org/10.1016/s0140-6736(00)02034-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10744089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Irinotecan%20combined%20with%20fluorouracil%20compared%20with%20fluorouracil%20alone%20as%20first%E2%80%90line%20treatment%20for%20metastatic%20colorectal%20cancer:%20a%20multicentre%20randomised%20trial&author=JY%20Douillard&author=D%20Cunningham&author=AD%20Roth&volume=355&publication_year=2000&pages=1041-7&pmid=10744089&doi=10.1016/s0140-6736(00)02034-1&)

2. Saltz LB, Cox JV, Blanke C et al Irinotecan plus fluorouracil and leucovorin for metastatic colorectal cancer. Irinotecan Study Group. N Engl J Med 2000; 343: 905–14.  [DOI](https://doi.org/10.1056/NEJM200009283431302) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11006366/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Irinotecan%20plus%20fluorouracil%20and%20leucovorin%20for%20metastatic%20colorectal%20cancer.%20Irinotecan%20Study%20Group&author=LB%20Saltz&author=JV%20Cox&author=C%20Blanke&volume=343&publication_year=2000&pages=905-14&pmid=11006366&doi=10.1056/NEJM200009283431302&)

3. Tournigand C, Andre T, Achille E et al FOLFIRI followed by FOLFOX6 or the reverse sequence in advanced colorectal cancer: a randomized GERCOR study. J Clin Oncol 2004; 22: 229–37.  [DOI](https://doi.org/10.1200/JCO.2004.05.113) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14657227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=FOLFIRI%20followed%20by%20FOLFOX6%20or%20the%20reverse%20sequence%20in%20advanced%20colorectal%20cancer:%20a%20randomized%20GERCOR%20study&author=C%20Tournigand&author=T%20Andre&author=E%20Achille&volume=22&publication_year=2004&pages=229-37&pmid=14657227&doi=10.1200/JCO.2004.05.113&)

4. Kawato Y, Aonuma M, Hirota Y, Kuga H, Sato K. Intracellular roles of SN‐38, a metabolite of the camptothecin derivative CPT‐11, in the antitumor effect of CPT‐11. Cancer Res 1991; 51: 4187–91.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1651156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Intracellular%20roles%20of%20SN%E2%80%9038,%20a%20metabolite%20of%20the%20camptothecin%20derivative%20CPT%E2%80%9011,%20in%20the%20antitumor%20effect%20of%20CPT%E2%80%9011&author=Y%20Kawato&author=M%20Aonuma&author=Y%20Hirota&author=H%20Kuga&author=K%20Sato&volume=51&publication_year=1991&pages=4187-91&pmid=1651156&)

5. Iyer L, King CD, Whitington PF et al Genetic predisposition to the metabolism of irinotecan (CPT‐11). Role of uridine diphosphate glucuronosyltransferase isoform 1A1 in the glucuronidation of its active metabolite (SN‐38) in human liver microsomes. J Clin Invest 1998; 101: 847–54.  [DOI](https://doi.org/10.1172/JCI915) | [PMC free article](/articles/PMC508633/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9466980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Genetic%20predisposition%20to%20the%20metabolism%20of%20irinotecan%20(CPT%E2%80%9011).%20Role%20of%20uridine%20diphosphate%20glucuronosyltransferase%20isoform%201A1%20in%20the%20glucuronidation%20of%20its%20active%20metabolite%20(SN%E2%80%9038)%20in%20human%20liver%20microsomes&author=L%20Iyer&author=CD%20King&author=PF%20Whitington&volume=101&publication_year=1998&pages=847-54&pmid=9466980&doi=10.1172/JCI915&)

6. Beutler E, Gelbart T, Demina A. Racial variability in the UDP‐glucuronosyltransferase 1 (UGT1A1) promoter: a balanced polymorphism for regulation of bilirubin metabolism? Proc Natl Acad Sci U S A 1998; 95: 8170–4.  [DOI](https://doi.org/10.1073/pnas.95.14.8170) | [PMC free article](/articles/PMC20948/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9653159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Racial%20variability%20in%20the%20UDP%E2%80%90glucuronosyltransferase%201%20(UGT1A1)%20promoter:%20a%20balanced%20polymorphism%20for%20regulation%20of%20bilirubin%20metabolism?&author=E%20Beutler&author=T%20Gelbart&author=A%20Demina&volume=95&publication_year=1998&pages=8170-4&pmid=9653159&doi=10.1073/pnas.95.14.8170&)

7. Gagne JF, Montminy V, Belanger P, Journault K, Gaucher G, Guillemette C. Common human UGT1A polymorphisms and the altered metabolism of irinotecan active metabolite 7‐ethyl‐10‐hydroxycamptothecin (SN‐38). Mol Pharmacol 2002; 62: 608–17.  [DOI](https://doi.org/10.1124/mol.62.3.608) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12181437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Common%20human%20UGT1A%20polymorphisms%20and%20the%20altered%20metabolism%20of%20irinotecan%20active%20metabolite%207%E2%80%90ethyl%E2%80%9010%E2%80%90hydroxycamptothecin%20(SN%E2%80%9038)&author=JF%20Gagne&author=V%20Montminy&author=P%20Belanger&author=K%20Journault&author=G%20Gaucher&volume=62&publication_year=2002&pages=608-17&pmid=12181437&doi=10.1124/mol.62.3.608&)

8. Yamanaka H, Nakajima M, Katoh M et al A novel polymorphism in the promoter region of human UGT1A9 gene (UGT1A9*22) and its effects on the transcriptional activity. Pharmacogenetics 2004; 14: 329–32.  [DOI](https://doi.org/10.1097/00008571-200405000-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15115919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=A%20novel%20polymorphism%20in%20the%20promoter%20region%20of%20human%20UGT1A9%20gene%20(UGT1A9*22)%20and%20its%20effects%20on%20the%20transcriptional%20activity&author=H%20Yamanaka&author=M%20Nakajima&author=M%20Katoh&volume=14&publication_year=2004&pages=329-32&pmid=15115919&doi=10.1097/00008571-200405000-00008&)

9. Cecchin E, Innocenti F, D'Andrea M et al Predictive role of the UGT1A1, UGT1A7, and UGT1A9 genetic variants and their haplotypes on the outcome of metastatic colorectal cancer patients treated with fluorouracil, leucovorin, and irinotecan. J Clin Oncol 2009; 27: 2457–65.  [DOI](https://doi.org/10.1200/JCO.2008.19.0314) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19364970/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Predictive%20role%20of%20the%20UGT1A1,%20UGT1A7,%20and%20UGT1A9%20genetic%20variants%20and%20their%20haplotypes%20on%20the%20outcome%20of%20metastatic%20colorectal%20cancer%20patients%20treated%20with%20fluorouracil,%20leucovorin,%20and%20irinotecan&author=E%20Cecchin&author=F%20Innocenti&author=M%20D'Andrea&volume=27&publication_year=2009&pages=2457-65&pmid=19364970&doi=10.1200/JCO.2008.19.0314&)

10. Ando M, Ando Y, Sekido Y, Ando M, Shimokata K, Hasegawa Y. Genetic polymorphisms of the UDP‐glucuronosyltransferase 1A7 gene and irinotecan toxicity in Japanese cancer patients. Jpn J Cancer Res 2002; 93: 591–7.  [DOI](https://doi.org/10.1111/j.1349-7006.2002.tb01295.x) | [PMC free article](/articles/PMC5927033/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12036456/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Jpn%20J%20Cancer%20Res&title=Genetic%20polymorphisms%20of%20the%20UDP%E2%80%90glucuronosyltransferase%201A7%20gene%20and%20irinotecan%20toxicity%20in%20Japanese%20cancer%20patients&author=M%20Ando&author=Y%20Ando&author=Y%20Sekido&author=M%20Ando&author=K%20Shimokata&volume=93&publication_year=2002&pages=591-7&pmid=12036456&doi=10.1111/j.1349-7006.2002.tb01295.x&)

11. Fujita K, Ando Y, Nagashima F et al Genetic linkage of UGT1A7 and UGT1A9 polymorphisms to UGT1A1*6 is associated with reduced activity for SN‐38 in Japanese patients with cancer. Cancer Chemother Pharmacol 2007; 60: 515–22.  [DOI](https://doi.org/10.1007/s00280-006-0396-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17406868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Genetic%20linkage%20of%20UGT1A7%20and%20UGT1A9%20polymorphisms%20to%20UGT1A1*6%20is%20associated%20with%20reduced%20activity%20for%20SN%E2%80%9038%20in%20Japanese%20patients%20with%20cancer&author=K%20Fujita&author=Y%20Ando&author=F%20Nagashima&volume=60&publication_year=2007&pages=515-22&pmid=17406868&doi=10.1007/s00280-006-0396-1&)

12. Han JY, Lim HS, Shin ES et al Comprehensive analysis of UGT1A polymorphisms predictive for pharmacokinetics and treatment outcome in patients with non‐small‐cell lung cancer treated with irinotecan and cisplatin. J Clin Oncol 2006; 24: 2237–44.  [DOI](https://doi.org/10.1200/JCO.2005.03.0239) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16636344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Comprehensive%20analysis%20of%20UGT1A%20polymorphisms%20predictive%20for%20pharmacokinetics%20and%20treatment%20outcome%20in%20patients%20with%20non%E2%80%90small%E2%80%90cell%20lung%20cancer%20treated%20with%20irinotecan%20and%20cisplatin&author=JY%20Han&author=HS%20Lim&author=ES%20Shin&volume=24&publication_year=2006&pages=2237-44&pmid=16636344&doi=10.1200/JCO.2005.03.0239&)

13. Lankisch TO, Schulz C, Zwingers T et al Gilbert's Syndrome and irinotecan toxicity: combination with UDP‐glucuronosyltransferase 1A7 variants increases risk. Cancer Epidemiol Biomarkers Prev 2008; 17: 695–701.  [DOI](https://doi.org/10.1158/1055-9965.EPI-07-2517) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18349289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&title=Gilbert's%20Syndrome%20and%20irinotecan%20toxicity:%20combination%20with%20UDP%E2%80%90glucuronosyltransferase%201A7%20variants%20increases%20risk&author=TO%20Lankisch&author=C%20Schulz&author=T%20Zwingers&volume=17&publication_year=2008&pages=695-701&pmid=18349289&doi=10.1158/1055-9965.EPI-07-2517&)

14. Levesque E, Belanger AS, Harvey M et al Refining the UGT1A haplotype associated with irinotecan‐induced hematological toxicity in metastatic colorectal cancer patients treated with 5‐fluorouracil/irinotecan‐based regimens. J Pharmacol Exp Ther 2013; 345: 95–101.  [DOI](https://doi.org/10.1124/jpet.112.202242) | [PMC free article](/articles/PMC3920089/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23386248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Refining%20the%20UGT1A%20haplotype%20associated%20with%20irinotecan%E2%80%90induced%20hematological%20toxicity%20in%20metastatic%20colorectal%20cancer%20patients%20treated%20with%205%E2%80%90fluorouracil/irinotecan%E2%80%90based%20regimens&author=E%20Levesque&author=AS%20Belanger&author=M%20Harvey&volume=345&publication_year=2013&pages=95-101&pmid=23386248&doi=10.1124/jpet.112.202242&)

15. Ando Y, Saka H, Ando M et al Polymorphisms of UDP‐glucuronosyltransferase gene and irinotecan toxicity: a pharmacogenetic analysis. Cancer Res 2000; 60: 6921–6.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11156391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Polymorphisms%20of%20UDP%E2%80%90glucuronosyltransferase%20gene%20and%20irinotecan%20toxicity:%20a%20pharmacogenetic%20analysis&author=Y%20Ando&author=H%20Saka&author=M%20Ando&volume=60&publication_year=2000&pages=6921-6&pmid=11156391&)

16. Innocenti F, Vokes EE, Ratain MJ. Irinogenetics: what is the right star? J Clin Oncol 2006; 24: 2221–4.  [DOI](https://doi.org/10.1200/JCO.2005.05.2464) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16636339/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Irinogenetics:%20what%20is%20the%20right%20star?&author=F%20Innocenti&author=EE%20Vokes&author=MJ%20Ratain&volume=24&publication_year=2006&pages=2221-4&pmid=16636339&doi=10.1200/JCO.2005.05.2464&)

17. Okuyama Y, Hazama S, Nozawa H et al Prospective phase II study of FOLFIRI for mCRC in Japan, including the analysis of UGT1A1 28/6 polymorphisms. Jpn J Clin Oncol 2011; 41: 477–82.  [DOI](https://doi.org/10.1093/jjco/hyr001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21303789/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Jpn%20J%20Clin%20Oncol&title=Prospective%20phase%20II%20study%20of%20FOLFIRI%20for%20mCRC%20in%20Japan,%20including%20the%20analysis%20of%20UGT1A1%2028/6%20polymorphisms&author=Y%20Okuyama&author=S%20Hazama&author=H%20Nozawa&volume=41&publication_year=2011&pages=477-82&pmid=21303789&doi=10.1093/jjco/hyr001&)

18. Kato T, Okuyama Y, Nagata N et al Multicenter phase II studies of FOLFIRI with reduced starting dose of irinotecan in metastatic CRC patients with homozygous for UGT1A1*28/*6. Presented at the ASCO Gastrointestinal Cancers Symposium; 14–17 Jan 2009, San Francisco, CA.  [Google Scholar](https://scholar.google.com/scholar_lookup?Kato%20T,%20Okuyama%20Y,%20Nagata%20N%20et%C2%A0al%20Multicenter%20phase%20II%20studies%20of%20FOLFIRI%20with%20reduced%20starting%20dose%20of%20irinotecan%20in%20metastatic%20CRC%20patients%20with%20homozygous%20for%20UGT1A1*28/*6.%20Presented%20at%20the%20ASCO%20Gastrointestinal%20Cancers%20Symposium;%2014%E2%80%9317%20Jan%202009,%20San%20Francisco,%20CA.)

19. Therasse P, Arbuck SG, Eisenhauer EA et al New guidelines to evaluate the response to treatment in solid tumors. European Organization for Research and Treatment of Cancer, National Cancer Institute of the United States, National Cancer Institute of Canada. J Natl Cancer Inst 2000; 92: 205–16.  [DOI](https://doi.org/10.1093/jnci/92.3.205) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10655437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=New%20guidelines%20to%20evaluate%20the%20response%20to%20treatment%20in%20solid%20tumors.%20European%20Organization%20for%20Research%20and%20Treatment%20of%20Cancer,%20National%20Cancer%20Institute%20of%20the%20United%20States,%20National%20Cancer%20Institute%20of%20Canada&author=P%20Therasse&author=SG%20Arbuck&author=EA%20Eisenhauer&volume=92&publication_year=2000&pages=205-16&pmid=10655437&doi=10.1093/jnci/92.3.205&)

20. The Food and Drug Administration Center for Drug Evaluation and Research held in November 2004. [Cited 21 Jul 2005.] Available from URL: http://www.fda.gov/2004.  [http://www.fda.gov/](http://www.fda.gov/)

21. Hazama S, Nagashima A, Kondo H et al Phase I study of irinotecan and doxifluridine for metastatic colorectal cancer focusing on the UGT1A1*28 polymorphism. Cancer Sci 2010; 101: 722–7.  [DOI](https://doi.org/10.1111/j.1349-7006.2009.01428.x) | [PMC free article](/articles/PMC11159193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20028383/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&title=Phase%20I%20study%20of%20irinotecan%20and%20doxifluridine%20for%20metastatic%20colorectal%20cancer%20focusing%20on%20the%20UGT1A1*28%20polymorphism&author=S%20Hazama&author=A%20Nagashima&author=H%20Kondo&volume=101&publication_year=2010&pages=722-7&pmid=20028383&doi=10.1111/j.1349-7006.2009.01428.x&)

22. Wang L, Hirayasu K, Ishizawa M, Kobayashi Y. Purification of genomic DNA from human whole blood by isopropanol‐fractionation with concentrated Nal and SDS. Nucleic Acids Res 1994; 22: 1774–5.  [DOI](https://doi.org/10.1093/nar/22.9.1774) | [PMC free article](/articles/PMC308067/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8202389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Purification%20of%20genomic%20DNA%20from%20human%20whole%20blood%20by%20isopropanol%E2%80%90fractionation%20with%20concentrated%20Nal%20and%20SDS&author=L%20Wang&author=K%20Hirayasu&author=M%20Ishizawa&author=Y%20Kobayashi&volume=22&publication_year=1994&pages=1774-5&pmid=8202389&doi=10.1093/nar/22.9.1774&)

23. Kobayashi M, Hazama S, Takahashi K et al Is there diversity among UGT1A1 polymorphism in Japan? World J Gastrointest Oncol 2011; 4: 170–5.  [DOI](https://doi.org/10.4251/wjgo.v4.i7.170) | [PMC free article](/articles/PMC3406281/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22848786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J%20Gastrointest%20Oncol&title=Is%20there%20diversity%20among%20UGT1A1%20polymorphism%20in%20Japan?&author=M%20Kobayashi&author=S%20Hazama&author=K%20Takahashi&volume=4&publication_year=2011&pages=170-5&pmid=22848786&doi=10.4251/wjgo.v4.i7.170&)

24. Guillemette C, Ritter JK, Auyeung DJ, Kessler FK, Housman DE. Structural heterogeneity at the UDP‐glucuronosyltransferase 1 locus: functional consequences of three novel missense mutations in the human UGT1A7 gene. Pharmacogenetics 2000; 10: 629–44.  [DOI](https://doi.org/10.1097/00008571-200010000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11037804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Structural%20heterogeneity%20at%20the%20UDP%E2%80%90glucuronosyltransferase%201%20locus:%20functional%20consequences%20of%20three%20novel%20missense%20mutations%20in%20the%20human%20UGT1A7%20gene&author=C%20Guillemette&author=JK%20Ritter&author=DJ%20Auyeung&author=FK%20Kessler&author=DE%20Housman&volume=10&publication_year=2000&pages=629-44&pmid=11037804&doi=10.1097/00008571-200010000-00006&)

25. The R Project for Statistical Computing. [Cited 20 May 2013.] Available from URL: http://www.r-project.org/.  [http://www.r-project.org/](http://www.r-project.org/)

26. Barrett JC, Fry B, Maller J, Daly MJ. Haploview: analysis and visualization of LD and haplotype maps. Bioinformatics 2005; 21: 263–5.  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=JC%20Barrett&author=B%20Fry&author=J%20Maller&author=MJ%20Daly&volume=21&publication_year=2005&pages=263-5&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

27. Akiyama Y, Fujita K, Nagashima F et al Genetic testing for UGT1A1*28 and *6 in Japanese patients who receive irinotecan chemotherapy. Ann Oncol 2008; 19: 2089–90.  [DOI](https://doi.org/10.1093/annonc/mdn645) | [PMC free article](/articles/PMC2733123/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18953066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Genetic%20testing%20for%20UGT1A1*28%20and%20*6%20in%20Japanese%20patients%20who%20receive%20irinotecan%20chemotherapy&author=Y%20Akiyama&author=K%20Fujita&author=F%20Nagashima&volume=19&publication_year=2008&pages=2089-90&pmid=18953066&doi=10.1093/annonc/mdn645&)

28. Innocenti F, Undevia SD, Iyer L et al Genetic variants in the UDP‐glucuronosyltransferase 1A1 gene predict the risk of severe neutropenia of irinotecan. J Clin Oncol 2004; 22: 1382–8.  [DOI](https://doi.org/10.1200/JCO.2004.07.173) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15007088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Genetic%20variants%20in%20the%20UDP%E2%80%90glucuronosyltransferase%201A1%20gene%20predict%20the%20risk%20of%20severe%20neutropenia%20of%20irinotecan&author=F%20Innocenti&author=SD%20Undevia&author=L%20Iyer&volume=22&publication_year=2004&pages=1382-8&pmid=15007088&doi=10.1200/JCO.2004.07.173&)

29. Satoh T, Ura T, Yamada Y et al Genotype‐directed, dose‐finding study of irinotecan in cancer patients with UGT1A1*28 and/or UGT1A1*6 polymorphisms. Cancer Sci 2011; 102: 1868–73.  [DOI](https://doi.org/10.1111/j.1349-7006.2011.02030.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21740478/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&title=Genotype%E2%80%90directed,%20dose%E2%80%90finding%20study%20of%20irinotecan%20in%20cancer%20patients%20with%20UGT1A1*28%20and/or%20UGT1A1*6%20polymorphisms&author=T%20Satoh&author=T%20Ura&author=Y%20Yamada&volume=102&publication_year=2011&pages=1868-73&pmid=21740478&doi=10.1111/j.1349-7006.2011.02030.x&)

30. Martinez‐Balibrea E, Abad A, Martinez‐Cardus A et al UGT1A and TYMS genetic variants predict toxicity and response of colorectal cancer patients treated with first‐line irinotecan and fluorouracil combination therapy. Br J Cancer 2010; 103: 581–9.  [DOI](https://doi.org/10.1038/sj.bjc.6605776) | [PMC free article](/articles/PMC2939780/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20628391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=UGT1A%20and%20TYMS%20genetic%20variants%20predict%20toxicity%20and%20response%20of%20colorectal%20cancer%20patients%20treated%20with%20first%E2%80%90line%20irinotecan%20and%20fluorouracil%20combination%20therapy&author=E%20Martinez%E2%80%90Balibrea&author=A%20Abad&author=A%20Martinez%E2%80%90Cardus&volume=103&publication_year=2010&pages=581-9&pmid=20628391&doi=10.1038/sj.bjc.6605776&)
