# Genome-wide Scan Identifies Opioid Overdose Risk Locus Close to MCOLN1

## Metadata
**Authors:** Zhongshan Cheng, Bao-Zhu Yang, Hang Zhou, Yaira Nunez, Henry R Kranzler, Joel Gelernter
**Journal:** Addiction biology
**Date:** 2019 Jul 30
**DOI:** [10.1111/adb.12811](https://doi.org/10.1111/adb.12811)
**PMID:** 31362332
**PMCID:** PMC7485539
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7485539/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7485539/pdf/nihms-1602844.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7485539/pdf/nihms-1602844.pdf)

## Abstract

The US is experiencing the worst opioid overdose (OpOD) crisis in its history. We carried out a genome-wide association study on OpOD severity among 3,477 opioid-exposed individuals, 1,019 of whom experienced opioid overdoses, including 2,032 European Americans (EAs) (653 overdose cases) and 1,445 African Americans (AAs) (366 overdose cases). Participants were scored 1–4 based on their reported overdose status and the number of times that medical treatment was required. Genome-wide association study (GWAS) of EAs and AAs separately resulted in two genome-wide significant (GWS) signals in AAs but none in EAs. The first signal was represented by three closely-mapped SNPs (rs115208233, rs116181528, and rs114077267) located near MCOLN1 (Mucolipin-1) and PNPLA6 (Patatin like phospholipase domain containing 6), and the other signal was represented by rs369098800 near DDX18; There were no additional GWS signals in the trans-population meta-analysis, so that post-GWAS analysis focused on these loci. In network analysis, MCOLN1 was co-expressed with PNPLA6, but only MCOLN1-associated genes were enriched in functional categories relevant to OpOD, including calcium and cation channel activities; no enrichment was observed for PNPLA6-associated genes. Drug repositioning analysis was carried out in the Connectivity Map (CMap) database for MCOLN1 (PNPLA6 was not available in CMap) and showed that the opioid agonist drug-induced expression profile is similar to that of MCOLN1 over-expression, and yielded the highest-ranked expression profile of 83 drug classes. Thus, MCOLN1 may be a risk gene for OpOD but replication is needed. This knowledge could be helpful in the identification of drug targets for preventing OpOD.

Keywords: Genome-wide association, Opioid overdose, MCOLN1, European Americans, African Americans

## INTRODUCTION

The United States is experiencing an opioid use and overdose crisis. In 2017, the death toll from drug overdoses hit a record high with more than 72,000 deaths ([Centers for Disease Control and Prevention, 2018](#R5)), and was driven by opioid overdose (OpOD) ([Buchanich et al., 2018](#R4); [Jalal et al., 2018](#R15)). Clinical symptoms of OpOD include stupor, miosis, and respiratory depression ([Boyer, 2012](#R3)). The best clinical strategy to reverse OpOD requires immediate administration of the opioid antagonist naloxone ([Chou et al., 2017](#R8)).

Genetic factors contribute to opioid dependence (OD) and opioid sensitivity. Several genome-wide association studies (GWAS) have identified SNPs contributing to the risk of OD, including rs60349741, rs62103177, rs12442183 and rs10799590 mapped to *KCNC1*, *KCNG2* ([Gelernter et al., 2014](#R13)), *RGMA* ([Cheng et al., 2018](#R7)) and *CNIH3* ([Nelson et al., 2016](#R21)), respectively. The first two of these risk genes were identified in African Americans (AAs), while the latter two were found in European Americans (EAs). Additionally, rs73568641, ~300kb upstream of *OPRM1*, was associated with therapeutic methadone and morphine dosage in AAs ([Smith et al., 2017](#R27)).

While progress has been made in understanding the genetics of OD risk and opioid dosing, there are no publications that address genetic factors predisposing to OpOD using a genome-wide approach. The diagnosis of a substance use disorder in the previous six months was reported to be the strongest risk factor for OpOD ([Webster, 2017](#R30)), but additional risk factors include the presence of other psychiatric disorders, impaired liver function, impaired vascular and pulmonary function, and a high daily opioid dosage ([Nadpara et al., 2018](#R20); [Zedler et al., 2015](#R32)). Many of these risk factors are themselves genetically influenced. Variation in opioid metabolism, also genetically influenced, accounts for differential sensitivity to OpOD ([Kharasch et al., 2015](#R18); [White and Irvine, 1999](#R31)). Thus, knowledge of the specific genetic risk factors for OpOD may guide the clinical care of patients receiving opioids therapeutically and those using the drugs illegally.

We conducted a GWAS of OpOD severity among subjects from the Yale-Penn study of the genetics of substance dependence, in a sample of 5,540 EAs and 3,675 AAs. Of these 9,215 subjects, we analyzed data from 3,477 subjects who were exposed to opioids (used at least once) and responded to questions regarding their overdose histories, resulting in an analysis sample of 2,032 EAs (653 overdose cases) and 1,445 AAs (366 overdose cases).

## METHODS AND MATERIALS

### Study Participants and the OpOD Trait

We collected samples of EA and AA subjects for genetic studies of substance dependence at five eastern US sites to participate in studies of the genetics of drug (opioid or cocaine) or alcohol dependence ([Gelernter et al., 2014](#R13)). Each participant was carefully phenotyped using the Semi-Structured Assessment for Drug Dependence and Alcoholism (SSADDA) ([Gelernter et al., 2014](#R13); [Pierucci-Lagha et al., 2007](#R23)). Subjects comprised two cohorts: Yale-Penn 1 (n=5,540) and Yale-Penn 2 (n=3,675). Each cohort includes some related subjects and they were assessed identically for phenotype but differ by the genotyping array employed. Written informed consent that was approved by the institutional review board at each site was obtained from each participant, and certificates of confidentiality were obtained from the National Institute on Drug Abuse and the National Institute on Alcohol Abuse and Alcoholism.

The OpOD trait was defined for subjects who were exposed to opioids at least once in their lives (n=4,989) and provided information on their OpOD experience (n=3,477); subjects with missing answers for OpOD were excluded from the analysis. The definition of opioid-exposed status was based on the following question in the G1 section of the SSADDA: *“Have you ever used any of the following opiate drugs: Heroin, Morphine, Methadone, Fentanyl or P-dope, Codeine, Percodan, Darvon, Dilaudid, Demerol, Percocet, Opium, and Other Opiate?”*. All subjects – opioid dependent, exposed, etc. – were ascertained with questions in the G1 section of the full SSADDA interview, in which information regarding illegal and prescription opioid abuse was collected. Severity of OpOD was scored on a 4-point scale as follows: *1*: used opioids but no overdose history; *2*: had a history of overdose but no consequent medical treatment; *3*: overdosed and received medical treatment <3 times; *4*: overdosed and received medical treatment ≥3 times. A total of 1,019 participants (653 EAs, 366 AAs) had an OpOD severity score ≥2, meaning that they all endorsed a history of overdose history ([Table 1](#T1)). Thus, the average overdose rate in opioid-exposed subjects was 29.3% (25.3% and 32.1% in AAs and EAs, respectively).

### Table 1.

| Opioid Overdose Coding | African American |  |  |  |  | European American |  |  |  |  |   |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| YP1 N | YP2 N | N | Mean Age | YP1+YP2 Male (%) | YP1 N | YP2 N | N | Mean Age | YP1+YP2 Male (%) | Total N |  |
| 1 | 741 | 338 | 1079 | 41.9 | 0.71 | 777 | 602 | 1379 | 36.2 | 0.67 | 2458 |
| 2 | 116 | 35 | 151 | 46.1 | 0.83 | 125 | 75 | 200 | 37.1 | 0.69 | 351 |
| 3 | 97 | 29 | 126 | 43.7 | 0.93 | 191 | 114 | 305 | 37.2 | 0.68 | 431 |
| 4 | 66 | 23 | 89 | 45.2 | 0.83 | 88 | 60 | 148 | 37.2 | 0.72 | 237 |
| Overdose | 279 | 87 | 366 | 45 | 0.86 | 404 | 249 | 653 | 37.2 | 0.69 | 1019 |
| Subtotal | 1020 | 425 | 1445 | 42.7 | 0.74 | 1181 | 851 | 2032 | 36.5 | 0.68 | 3477 |

Table 1 Caption: Sample counts and characteristics for the severity of the opioid overdose trait.

### Genome-wide Association Study (GWAS) Analyses

Yale-Penn 1 samples were genotyped on the Illumina (San Diego, CA, USA) HumanOmni1-Quad v1.0 microarray (total SNPs = 1,140,419) and Yale-Penn 2 samples were genotyped on the Illumina HumanCore Exome array (total SNPs = 550,601). We used PLINK1.9 ([Chang et al., 2015](#R6)) for quality control (QC) in each cohort, applying the following criteria: (1) individual genotype missing rate <2%, (2) SNP genotype missing rate <2%, (3) Hardy-Weinberg P>1×10^−6^, and (4) minor allele frequency (MAF) >3%. Samples from Yale-Penn 1 and 2 that passed the initial QC were subjected to ancestry analysis by comparison with the 1000 Genomes Project phase 1 reference panel ([1000 Genomes Project Consortium et al., 2010](#R1)). Principal component analysis was performed with Eigensoft ([Price et al., 2006](#R24)) and the resulting first 10 principal components (PCs) served to differentiate AAs and EAs ([Sherva et al., 2016](#R26)). Subsequently, AAs and EAs from each Yale-Penn cohort were subjected to SNP imputation by using Minimac3 implemented in the Michigan Imputation Server (Das et al, 2016) with the 1000 Genomes Project phase 3 reference panel. Dosage data were transformed into best-guess genotypes using PLINK1.9, retaining high-quality data by filtering imputed data with genotype imputation probability (GP) ≥ 0.8, individual genotyping missing rate <5%, SNP MAF >3%, SNP missing call rate <5%, and Hardy-Weinberg *P*>1×10^−6^.

We conducted GWAS on the severity score of the OpOD trait in AAs and EAs separately, with adjustment for the relatedness among participants and included age, sex, and the first 10 PCs (to account for population structure using Genome-wide Efficient Mixed Model Association (GEMMA) software ([Zhou and Stephens, 2012](#R34))) as covariates. Our initial data QC included a procedure to address population stratification, as for our previous studies ([Sherva et al., 2016](#R26); [Zhou et al., 2017](#R33)). GEMMA can also account for relatedness among samples and can control for population stratification and other confounding factors ([Kang et al., 2010](#R16); [Zhou and Stephens, 2012](#R34)). We chose a univariate linear mixed model to associate genotype with overdose severity score, the rationale for which is that the overdose severity score represents the continuous nature of opioid overdose severity. Subsequent meta-analysis was carried out using the inverse variance method in PLINK 1.9. The number of SNPs by cohort that passed the QC and were included in the analyses were 10,295,719 in Yale-Penn 1 AA (n=1,020), 6,981,522 in Yale-Penn 1 EA (n=1,181), 7,501,713 in Yale-Penn 2 AA (n=425), and 7,501,713 in Yale-Penn 2 EA (n=851). The significance threshold for GWAS analyses was set at *P* ≤5×10^−8^.

### Post-GWAS Prioritization of Possible OpOD Risk Genes with Network Analysis and Drug Repositioning Analysis

Variants close to *MCOLN1* and *PNPLA6* were genome-wide significant (GWS) in AAs; see below for detailed GWAS results. The three GWS SNPs map closely within a region [−1kb, 5kb] relative to the transcription start site (TSS) of *MCOLN1*. They are also ~10 kb distant from the TSS of *PNPLA6*. To evaluate whether *MCOLN1* or *PNPLA6* was more likely to be the risk gene for OpOD (beyond that of the somewhat greater proximity of *MCOLN1* to the GWS markers), we conducted network analysis by querying *MCOLN1* and *PNPLA6* individually and jointly with GeneMANIA ([Franz et al., 2018](#R12)), which provides functional association data taking account of similarities in protein and genetic interactions, pathways, co-expression, co-localization, and protein domains.

Drug repositioning analysis was then carried out for *MCOLN1* with the Connectivity Map (CMap) Database ([Subramanian et al., 2017](#R28)); data for *PNPLA6* were not available in CMap. CMap contains >1 million transcriptional profiles derived from perturbations of multiple cancer- related cell lines that arise from a genetic mutation affecting protein function, genetic perturbation (knockdown or overexpression of a gene), or treatment with a small molecule or a drug. CMap used 8 human cancer-related cell lines: A375 (malignant melanoma), A549 (non-small cell lung carcinoma), HCC515 (non-small cell lung adenocarcinoma), HEPG2 (hepatocellular carcinoma), HT29 (colorectal adenocarcinoma), MCF7 (breast adenocarcinoma), PC3 (prostate adenocarcinoma), and VCAP (metastatic prostate cancer), as well as HA1E (kidney epithelial immortalized non-cancer cell line). Transcriptional profiles with high similarity between two proteins operating in the same pathway, between a drug and its protein target, or between two drugs of similar function but with structural dissimilarity, have the potential to reveal useful and previously unrecognized connections. CMap has provided expression similarity scores for a specific expression profile (here, we use the *MCOLN1* overexpression-induced expression profile) with other drug-induced transcriptional profiles, including consensus transcriptional signatures of 83 drug classes, i.e., transcriptional profiles induced by 2,837 drugs grouped into 83 drug classes. Expression similarity is evaluated by means of scores that vary from −100 to 100, with −100 the most extreme opposite expression profile and 100 the most extreme similar expression profile. To accomplish this, we downloaded expression similarity scores for *MCOLN1* by selecting the option ‘See all connections’ after querying ‘*MCOLN1*’ directly in the CMap cloud server ([https://clue.io/command?q=/conn](https://clue.io/command?q=/conn)). CMap includes 32 drugs that target mu-, delta-, and kappa-opioid receptors. Only one opioid drug class comprising three opioid receptor agonists is included in the summary statistics of 83 drug classes provided by CMap for *MCOLN1* (described below, “Results”). Accordingly, we also downloaded the individual expression similarity scores for all 2,837 drugs to *MCOLN1* over-expression, and then evaluated expression similarity scores between *MCOLN1* and 29 other opioid-related drugs that were not included in the 83 drug classes. Other drugs with transcriptional signatures opposite that of *MCOLN1* over-expression (negative similarity scores to *MCOLN1*) were also identified.

## RESULTS

For the two Yale-Penn cohorts, we analyzed EAs and AAs separately, then meta-analyzed the two cohorts by population (Yale-Penn 1 and 2 EAs, and Yale-Penn 1 and 2 AAs). Then, a trans-ancestry meta-analysis was performed that included all of the EA and AA samples. We identified two GWS signals in AAs, which mapped to the genes *MCOLN1* (also close to *PNPLA6*) and *DDX18* ([Figure 1](#F1)–[2](#F2) and [Figure S1](#SD1)–[S2](#SD1)). There were no GWS signals identified in EAs. In the meta-analysis of all subjects, only the signal that mapped to *MCOLN1*, close to *PNPLA6* on chromosome 19, represented by three SNPs (rs115208233, rs116181528, and rs114077267), were GWS ([Figure 1](#F1)). The other SNP that was GWS in AAs, rs369098800, which is ~470 kb from *DDX18*, was only nominally significant in the trans-ancestral meta-analysis ([Figures 1](#F1) and S1). We found no evidence of inflation (λ~1.0, [Figure 1](#F1)) both in the meta-analyses of AAs and EAs separately and together.

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4de5/7485539/83ac864b5d0a/nihms-1602844-f0001.jpg)

Genome-wide association studies of opioid-overdose severity among four cohorts, including Yale-Penn1 and Yale-Penn2, each of which included both African Americans (AAs) and European Americans (EAs). The three panels display the Manhattan plots (left) and qq plots (right) for the meta-analysis of both population groups, and the AA and EA cohorts separately. Four SNPs reached genome-wide significance; three SNPs (rs115208233, rs116181528 and rs114077267) on chromosome 19 are specific to the AA population and map to MCOLN1, close to PNPLA6. Another SNP (rs369098800) is on chromosome 2, ~471kb away from DDX18. On chromosome 4, a suggestive significant association peak close to gene C4orf22 was seen in the meta-analysis of EA and AA.

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4de5/7485539/3ae35a03daef/nihms-1602844-f0002.jpg)

Regional Manhattan plots displaying one independent signal associated with opioid overdose (OpOD) severity on the locus MCOLN1. Three panels illustrate the association signals that emerged from the meta-analysis of all European Americans (EAs) and African Americans (AAs) and in the meta-analysis of EAs and AAs separately. Three genome-wide significant SNPs specific to the AA population map to MCOLN1: rs115208233, rs116181528 and rs114077267. Among these, rs115208233 is the lead SNP and rs114077267 is in the promoter region of MCOLN1. Two other AA-specific SNPs, rs116362433 (an intronic SNP of MCOLN1) and rs115515588 (a promoter SNP of PNPLA6), were nominally associated with OpOD severity (all P values ~5×10−6). One SNP, rs2081195, located in the intronic region of PNPLA6, was nominally significantly associated with OpOD severity in EAs (P=1.4×10−4). Note: the SNPs are colored to reflect the linkage disequilibrium (R2) with the lead SNP (rs115208233 for EAs and EAs+AAs; rs2081195 for EAs) based on the African (for AAs and EAs+AAs) or European (for EAs) population data from the 1000 Genomes Project. The light blue line and right Y-axis indicate the observed recombination rate estimated from HapMap samples. Only association signals existing in ≥2 cohorts are plotted.

Of the three AA-specific GWS SNPs, the first two are located in different intronic regions of *MCOLN1* and the last resides in the *MCOLN1* promoter region (477bp upstream of the *MCOLN1*; [Figure 2](#F2) and [3](#F3)). The top SNP, rs115208233, is in high linkage disequilibrium (LD) with the promoter SNP rs114077267 and in moderate LD with rs116181528 (rs115208233:rs114077267 R^2^=0.9; rs115208233:rs116181528 R^2^=0.74; rs116181528:rs114077267 R^2^=0.67), which are polymorphic only in AAs, with the minor G allele postively associated with the OpOD severity score (see [Figure S3](#SD1)). The MAFs for these SNPs range from 0.04 to 0.06 ([Table 2](#T2)) in that population. The association signals were nominally significant in both Yale-Penn 1 and Yale-Penn 2 AA samples separately (all *P* values <5.0×10^−4^, [Table 2](#T2)). The minor alleles are all associated with higher overdose severity scores in AAs.

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4de5/7485539/a54aeb1cb675/nihms-1602844-f0003.jpg)

Genomic and functional annotation for rs114077267, rs115208233, and rs116181528. The upper panel denotes the chromosomal region of the three genome-wide significant (GWS) SNPs. In addition, three regulatory features, including H3K27Ac marks, DNase clusters, and transcription factor ChIP-seq binding sites, are seen in the region. A subset of the region covered by the red dashed line is magnified in the lower panel, with the GWS SNPs in bold. The signals for the H3K27Ac mark, DNAse I hypersensitivity cluster, and transcriptional factor binding site are annotated according to experimental data from the ENCODE Consortium (Encode_Project_Consortium, 2012). The height and color in the H3K27Ac marks represent signal intensity and data source from different cell lines. The gray box indicates the extent of the hypersensitive region or cluster of transcriptional factor occupancy. The shade is proportional to the maximum signal strength observed in any cell line. The green line indicates the highest scoring site of an identified canonical motif for the corresponding factor.

### Table 2.

| Chr | Position | SNP | A1 | A2 | MAF | Beta | SE | p-value | Cohort |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 19 | 7587017 | rs114077267 | T | C | 0.055 | 3.41E-01 | 8.88E-02 | 1.16E-04 | YP1 AA |
| 0.041 | 6.85E-01 | 1.51E-01 | 5.09E-06 | YP2 AA |  |  |  |  |  |
|   | 4.30E-01 |   | 1.97E-08 | Meta AA YP1+YP2 |  |  |  |  |  |
| 19 | 7589061 | rs115208233 | G | C | 0.053 | 3.61E-01 | 8.98E-02 | 5.62E-05 | YP1 AA |
| 0.039 | 7.67E-01 | 1.55E-01 | 6.84E-07 | YP2 AA |  |  |  |  |  |
|   | 4.63E-01 |   | 2.51E-09 | Meta AA YP1+YP2 |  |  |  |  |  |
| 19 | 7594247 | rs116181528 | A | C | 0.061 | 3.23E-01 | 8.40E-02 | 1.17E-04 | YP1 AA |
| 0.048 | 7.01E-01 | 1.43E-01 | 8.89E-07 | YP2 AA |  |  |  |  |  |
|   | 4.20E-01 |   | 6.85E-09 | Meta AA YP1+YP2 |  |  |  |  |  |

Table 2 Caption: Characteristics of the three genome-wide significant SNPs located on MCOLN1.

There were no GWS SNPs in both populations, though one SNP, rs4146322, located in the intronic region of *C4orf22*, showed suggestive evidence of association with OpOD severity in the meta-analysis of AAs and EAs ([Figure S2](#SD1)). According to the brain expression quantitative trait locus (eQTL) database of European samples, BRAINEAC ([Ramasamy et al., 2014](#R25)), rs4146322 is an eQTL for both *C4orf22* (*P*=7.1×10^−3^) and *ANTXR2* (*P*=8.9×10^−3^) in the putamen and frontal cortex, respectively. In another eQTL database, GTEx ([GTEx-Consortium, 2015](#R14)), rs4146322 is an eQTL for *C4orf22* in testis (*P*=7.0×10^−11^). Another SNP, rs369098800, (which maps near *DDX18*), was GWS in AAs but only nominally significant in the trans-ancestry GWAS of EAs+AAs, but it is not available in BRAINEAC or GTEx databases.

SNP rs73568641, ~300kb upstream of *OPRM1*, associated with therapeutic opioid dose in AAs ([Smith et al., 2017](#R27)), was not significant in the OpOD analysis in the AA part of the sample (*P*=0.54). Two other SNPs associated with opioid dependence in AAs ([Gelernter et al., 2014](#R13)), rs62103177 (mapped to *KCNG2*) and rs60349741 (mapped to *KCNC1*), were also not significantly associated with OpOD (*P* = 0.80 and 0.28, respectively). Similarly, rs12442183 close to *RGMA* and associated with OD in EAs ([Cheng et al., 2018](#R7)), was not associated with OpOD in EAs (*P*=0.56).

Based on functional prediction from Haploreg4 ([Ward and Kellis, 2016](#R29)) and as described above, the three OpOD GWS SNPs reside in the predicted enhancer regions, and rs114077267 is also in the promoter region of *MCOLN1*. Also, according to annotation data from Haploreg4 ([Ward and Kellis, 2016](#R29)), this promoter region is active in 23 tissues, including brain, blood, muscle, and heart. The risk allele, rs114077267*T, is predicted to generate motifs with high binding affinity for the transcription factors GFI1 (transcription repressor), GATA (transcription activator), and HOXA7 (transcription repressor). Also, as noted above, these three GWS SNPs are about 10 kb from another gene, *PNPLA6*. Per the UCSC Genome Browser ([Kent et al., 2002](#R17)), the 3’ region of *MCOLN1* is only 144 bp upstream of the TSS of *PNPLA6* ([Figure 3](#F3)), suggesting that common genetic elements could regulate the expression of both loci.

The potential involvement of *MCOLN1*, *PNPLA6*, or both of these loci in OpOD was evaluated further by performing network analysis for biologically relevant genes by querying *MCOLN1* and *PNPLA6* jointly ([Figure 4a](#F4)) or separately ([Figure 4b](#F4) and [4c](#F4)) in the GeneMANIA database ([Franz et al., 2018](#R12)). This showed that the expression of *MCOLN1* is correlated with that of *PNPLA6*. Further network analysis with GeneMANIA revealed that MCOLN1, the protein encoded by *MCOLN1*, displays protein-protein interaction with MCOLN3 (encoded by *MCOLN3*, a paralog of *MCOLN1*) and SLC35E1 (a putative transporter). PNPLA6, the protein product of *PNPLA6*, demonstrates a very different protein-protein interaction pattern from that of MCOLN1 ([Figures 4a](#F4)–[4c](#F4)); proteins that interact with PNPLA6 do not have strong associations with MCOLN1. In addition, in functional enrichment analysis, most genes associated with MCOLN1 in network analysis have similar protein domains and are involved in calcium and other metal ion and cation channel activities ([Figure 4d](#F4)). In contrast, no functional categories were enriched for *PNPLA6*-associated genes.

### Figure 4.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4de5/7485539/94626370efe7/nihms-1602844-f0004.jpg)

Network analysis prioritizes MCOLN1 but not PNPLA6 as a potential opioid overdose risk gene. GeneMANIA was applied to obtain biologically related genes that are associated with MCOLN1 and PNPLA6 when queried jointly (a) or individually (b and c), the enriched functional categories of which were also illustrated (d). The above networks display different types of association between MCOLN1 or PNPLA6 and their biologically related genes, in which the color and size of each line linking two genes indicates the type and strength of the association. These associations comprise protein and genetic interactions, pathways, co-expression, co-localization, and protein domain similarity. The enrichment P values (-log10(FDR)) provided by GeneMANIA are demonstrated for different functional categories. Most enriched functional categories are observed among MCOLN1-associated genes, and no functional categories are enriched in PNPLA6-associated genes. (FDR: false discovery rate)

The possible involvement of *MCOLN1* in OpOD was further evaluated by performing a drug repositioning analysis with CMap (see [Methods](#S2)). The consensus expression profile of the drug class “opioid receptor agonists” provided the transcriptional signature most similar to that of an *MCOLN1* over-expression profile ([Figure 5a](#F5), expression similarity score= 99). The opioid receptor agonist class is comprised of FIT (fentanyl isothiocyanate, an irreversible opioid receptor agonist targeting the delta opioid receptor), Ioperamide (a peripheral opioid receptor agonist affecting multiple intestinal opioid receptors), and ICI-199441 (a kappa opioid receptor agonist), and their expression similarity scores relative to *MCOLN1* over-expression are 99, 95 and 88, respectively ([Figure 5b](#F5)). Other than these three opioidergic drugs allocated to drug classes in CMap, only 3 of the 32 opioidergic drugs included are allocated to a group. The other four top drug classes ranked by their expression similarity scores (>96) to an *MCOLN1* over-expression induced profile include tricyclic antidepressants, serotonin receptor antagonists, dopamine receptor antagonists, and calcium receptor inhibitors. In addition, as the opioid receptor antagonist naloxone can reverse OpOD, we specifically evaluated a possible relationship between *MCOLN1* over-expression and the naloxone-stimulated transcriptome signature, including a set of overexpressed and downregulated genes. Transcriptional profiles with high similarity between what is observed in the case of *MCOLN1* over-expression and what is observed with opioidergic drugs might represent useful and previously unrecognized connections; the *MCOLN1* induced expression profile yielded an expression similarity score of 87 to that of naloxone, and naloxone was ranked sixth among 32 opioid drugs based on their similarity to transcriptional signatures of *MCOLN1* over-expression. Overall, the average similarity score to that of *MCOLN1* over-expression for all 32 available opioid drugs was 62±27. Other opioid drugs that target mu-, kappa-, and delta-opioid receptors also demonstrated high similarity for perturbated transcriptional signatures with that of *MCOLN1* over-expression (all expression similarity scores >90), including the three opioid receptor agonists, and naltrindole and naltriben, which are opioid receptor antagonists. That is, a total of 5 opioid-related drugs shared similarities in the drug-induced expression pattern to *MCOLN1*, each scoring ≥90. Furthermore, we identified BRD-A80383043, a glutamate receptor agonist, as the only drug that can induce a transcriptional profile opposite that of an *MCOLN1* over-expression stimulated profile (expression similarity score = −94).

### Figure 5.

![Figure 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4de5/7485539/9638fb81d414/nihms-1602844-f0005.jpg)

Drug repositioning analysis supports the involvement of MCOLN1 in the opioid receptor system. (a) Among 83 drug classes, opioid receptor agonist-induced expression profiles display the highest expression similarity score with the MCOLN1 over-expression induced profile in CMap. Expression similarity scores are generated by comparing the MCOLN1 over-expression induced expression profile to the signatures of 83 classes of drug-induced transcription. (b) Single drugs from these top 5 drug classes (highlighted in part a) with expression similarity score >=80 are shown.

## DISCUSSION

We conducted a GWAS for OpOD among AAs and EAs, and found a GWS signal specific to AAs, which spans a total of only 7,231 basepairs, mapped to *MCOLN1* and ~10 kb from *PNPLA6* ([Figure 3](#F3)). Among the three GWS SNPs, rs114077267 is potentially functional, residing in the promoter region of *MCOLN1*, suggesting that *MCOLN1* is a more likely risk gene for OpOD than *PNPLA6*. This conclusion is further supported by a network analysis and a drug repositioning analysis, in which (1) genes correlated with *MCOLN1* but not *PNPLA6* were enriched in OpOD-related functional categories, including regulation by Ca++, and (2) transcriptional signatures induced by opioid receptor agonists were very similar to the *MCOLN1* over-expression stimulated profile. Since the pattern of *MCOLN1* overexpression is similar to that of opioid drug induced expression, we infer that they have related biology.

The association of *MCOLN1* with OpOD could be related to its connection to opioid receptor signaling. Previous research showed that MCOLN1 (Mucolipin-1) is a receptor protein with potential cationic channel activities ([Bassi et al., 2000](#R2); [Montell, 2001](#R19)). Loss-of-function mutations in *MCOLN1* cause mucolipidosis type IV, a severe lysosomal storage disorder ([Bassi et al., 2000](#R2)). MCOLN1 has also been reported to be an endolysosomal iron release channel ([Dong et al., 2008](#R10)), and knockdown of *MCOLN1* in human retinal pigmented epithelial 1 cells promotes the production of reactive oxygen species by Fe2^+^ trapped in lysosomes ([Coblentz et al., 2014](#R9)). In our study, network analysis demonstrated that *MCOLN1*-associated genes were enriched in cation and metal ion channel activities ([Figure 4d](#F4)). Furthermore, in a drug repositioning analysis, the *MCOLN1* overexpression-induced transcription signature was highly similar to that of three opioid receptor agonists’ induced transcriptional profiles ([Figure 5](#F5)). These three opioid receptor agonists – the only ones included in the defined CMap drug classes – are FIT (fentanyl isothiocyanate, irreversible delta opioid receptor agonist), Ioperamide (peripheral opioid receptor agonist), and ICI-199441 (kappa opioid receptor agonist). We also found that three other opioid receptor antagonists not included in CMap classes – naloxone, naltrindole, and naltriben – induced transcriptional profiles similar to an *MCOLN1* over-expression induced transcriptional signature, which suggests that both opioid receptor antagonists and agonists are functionally related to *MCOLN1*. Further, a glutamate receptor agonist, BRD-A80383043, displayed a transcriptional signature opposite to that of the *MCOLN1* over-expression induced profile. Glutamate is the major excitatory neurotransmitter in the brain and plays an integral role in opioid addiction ([Peters and De Vries, 2012](#R22)) and OpOD ([White and Irvine, 1999](#R31)). As previously shown, the glutamate receptor antagonists JNJ16259685 and [LY341495](https://www.ncbi.nlm.nih.gov/nuccore/LY341495) increase the antinociceptive efficacy of opioids (Fischer et al., 2008). We suspect that *MCOLN1* overexpression may be associated with OpOD severity, the underlying mechanism being, potentially, its ability to enhance the antinociceptive efficacy of opioids. This is supported by drug repositioning analyses that revealed similarities between the drug-induced expression pattern of 5 opioid-related drugs and *MCOLN1* overexpression, while a glutamate receptor agonist drug demonstrated an expression profile opposite to *MCOLN1* overexpression.

Additionally, in considering the context of our OpOD findings, we looked up in the OpOD GWAS data SNPs that were previously associated with opioid dosing in AAs ([Smith et al., 2017](#R27)) and with opioid dependence in AAs ([Gelernter et al., 2014](#R13)) and EAs ([Cheng et al., 2018](#R7)), and did not find even nominal association in either case. The OpOD trait is different biologically from both opioid dosing and opioid dependence *per se*; these are not “failures to replicate.”

Our finding requires independent validation. Because one of our top SNPs, rs114077267, from the AA OpOD GWAS, maps to the promoter region of *MCOLN1* and is AA specific, we checked the extent of promoter SNPs (MAF>=0.03) at *MCOLN1* and compared their allele frequencies among different populations, including African (AFR), Admixed American (AMR), Asian (ASN) and European (EUR) populations ([Figure S4](#SD1)). Compared to the other three populations, the AFR population has more SNPs (MAF>0.03) in the promoter region of *MCOLN1* and, thus there is a greater potential to detect true associations. Among EUR and ASN, there are few SNPs in the *MCOLN1* promoter. Thus, another AA population would be required for replication, although it may be possible to identify consistent evidence (with other SNPs mapped to the same region) in other populations.

Our study has potential limitations. *MCOLN1* may be associated with quantity or frequency of exposure to opioids. However, we do not have accurate measures of quantity and frequency of opioid exposure, so we cannot test this directly. Potential polygenic risk score (PRS) analyses between OpOD and other independent psychiatric and SUD traits should be carried out in the future to identify pleiotropy, but at this point our sample is not adequately powered for this step. This is a critical problem for the research field: namely, that recruitment of more subjects to increase power is absolutely essential.

Our OpOD GWAS provides statistically significant support for *MCOLN1* as a risk gene for OpOD, with the hypothesis that *MCOLN1* is the effect locus supported by both network analysis and drug repositioning analysis. An *MCOLN1* overexpression-induced profile was highly similar to opioid receptor agonist-induced expression profiles. The apparent involvement of *MCOLN1* in the activity of the opioidergic system warrants evaluation in cellular and animal models. In conclusion, our GWAS supports *MCOLN1* as a risk gene for OpOD. Considering all of the data, the effect of *MCOLN1* over-expression on the opioid receptor system could be a mechanism for its association with OpOD. In any case, the findings suggest a novel biology underlying OpOD.

## Supplementary Material

## Acknowledgements

This study was supported by National Institutes of Health grants R01 DA12690, R01 AA11330, R01 AA017535, and the VA Connecticut Healthcare Center and Philadelphia VA MIRECCs. Genotyping services for a part of our GWAS were provided by the Center for Inherited Disease Research and Yale University (Center for Genome Analysis), which is fully funded by Federal contract N01-HG-65403 from the NIH to The Johns Hopkins University. Ann Marie Lacobelle, MS and Christa Robinson provided technical support.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. 1000 Genomes Project Consortium, Abecasis GR, Altshuler, Auton A, Brooks LD, Durbin RM, Gibbs RA, Hurles ME, McVean GA (2010) A map of human genome variation from population-scale sequencing. Nature 467:1061–1073.  [DOI](https://doi.org/10.1038/nature09534) | [PMC free article](/articles/PMC3042601/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20981092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20map%20of%20human%20genome%20variation%20from%20population-scale%20sequencing&author=GR%20Abecasis&author=%20Altshuler&author=A%20Auton&author=LD%20Brooks&author=RM%20Durbin&volume=467&publication_year=2010&pages=1061-1073&pmid=20981092&doi=10.1038/nature09534&)

2. Bassi MT, Manzoni M, Monti E, Pizzo MT, Ballabio A, Borsani G (2000) Cloning of the gene encoding a novel integral membrane protein, mucolipidin-and identification of the two major founder mutations causing mucolipidosis type IV. Am J Hum Genet 67:1110–1120.  [DOI](https://doi.org/10.1016/s0002-9297(07)62941-3) | [PMC free article](/articles/PMC1288553/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11013137/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Cloning%20of%20the%20gene%20encoding%20a%20novel%20integral%20membrane%20protein,%20mucolipidin-and%20identification%20of%20the%20two%20major%20founder%20mutations%20causing%20mucolipidosis%20type%20IV&author=MT%20Bassi&author=M%20Manzoni&author=E%20Monti&author=MT%20Pizzo&author=A%20Ballabio&volume=67&publication_year=2000&pages=1110-1120&pmid=11013137&doi=10.1016/s0002-9297(07)62941-3&)

3. Boyer EW (2012) Management of opioid analgesic overdose. N Engl J Med 367:146–155.  [DOI](https://doi.org/10.1056/NEJMra1202561) | [PMC free article](/articles/PMC3739053/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22784117/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Management%20of%20opioid%20analgesic%20overdose&author=EW%20Boyer&volume=367&publication_year=2012&pages=146-155&pmid=22784117&doi=10.1056/NEJMra1202561&)

4. Buchanich JM, Balmert LC, Williams KE, Burke DS (2018) The effect of incomplete death certificates on estimates of unintentional opioid-related overdose deaths in the United States, 1999–2015. Public Health Rep 133:423–431.  [DOI](https://doi.org/10.1177/0033354918774330) | [PMC free article](/articles/PMC6055296/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29945473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Public%20Health%20Rep&title=The%20effect%20of%20incomplete%20death%20certificates%20on%20estimates%20of%20unintentional%20opioid-related%20overdose%20deaths%20in%20the%20United%20States,%201999%E2%80%932015&author=JM%20Buchanich&author=LC%20Balmert&author=KE%20Williams&author=DS%20Burke&volume=133&publication_year=2018&pages=423-431&pmid=29945473&doi=10.1177/0033354918774330&)

5. Centers for Disease Control and Prevention (2018) Provisional Counts of Drug Overdose Death. National-Center-for-Health-Statistics.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Provisional%20Counts%20of%20Drug%20Overdose%20Death&publication_year=2018&)

6. Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ (2015) Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience 4:7.  [DOI](https://doi.org/10.1186/s13742-015-0047-8) | [PMC free article](/articles/PMC4342193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25722852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gigascience&title=Second-generation%20PLINK:%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&author=CC%20Chang&author=CC%20Chow&author=LC%20Tellier&author=S%20Vattikuti&author=SM%20Purcell&volume=4&publication_year=2015&pages=7&pmid=25722852&doi=10.1186/s13742-015-0047-8&)

7. Cheng Z, Zhou H, Sherva R, Farrer LA, Kranzler HR, Gelernter J (2018) Genome-wide association study identifies a regulatory variant of RGMA associated with opioid dependence in European Americans. Biological psychiatry 84:762–770.  [DOI](https://doi.org/10.1016/j.biopsych.2017.12.016) | [PMC free article](/articles/PMC6041180/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29478698/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biological%20psychiatry&title=Genome-wide%20association%20study%20identifies%20a%20regulatory%20variant%20of%20RGMA%20associated%20with%20opioid%20dependence%20in%20European%20Americans&author=Z%20Cheng&author=H%20Zhou&author=R%20Sherva&author=LA%20Farrer&author=HR%20Kranzler&volume=84&publication_year=2018&pages=762-770&pmid=29478698&doi=10.1016/j.biopsych.2017.12.016&)

8. Chou R, Korthuis PT, McCarty D, Coffin PO, Griffin JC, Davis-O’Reilly C, Grusing S, Daya M (2017) Management of suspected opioid overdose with naloxone in out-of-hospital settings: a systematic review. Ann Intern Med 167:867–875.  [DOI](https://doi.org/10.7326/M17-2224) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29181532/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Management%20of%20suspected%20opioid%20overdose%20with%20naloxone%20in%20out-of-hospital%20settings:%20a%20systematic%20review&author=R%20Chou&author=PT%20Korthuis&author=D%20McCarty&author=PO%20Coffin&author=JC%20Griffin&volume=167&publication_year=2017&pages=867-875&pmid=29181532&doi=10.7326/M17-2224&)

9. Coblentz J, St Croix C, Kiselyov K (2014) Loss of TRPML1 promotes production of reactive oxygen species: is oxidative damage a factor in mucolipidosis type IV? Biochem J 457:361–368.  [DOI](https://doi.org/10.1042/BJ20130647) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24192042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20J&title=Loss%20of%20TRPML1%20promotes%20production%20of%20reactive%20oxygen%20species:%20is%20oxidative%20damage%20a%20factor%20in%20mucolipidosis%20type%20IV?&author=J%20Coblentz&author=C%20St%20Croix&author=K%20Kiselyov&volume=457&publication_year=2014&pages=361-368&pmid=24192042&doi=10.1042/BJ20130647&)

10. Dong XP, Cheng X, Mills E, Delling M, Wang F, Kurz T, Xu H (2008) The type IV mucolipidosis-associated protein TRPML1 is an endolysosomal iron release channel. Nature 455:992–996.  [DOI](https://doi.org/10.1038/nature07311) | [PMC free article](/articles/PMC4301259/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18794901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20type%20IV%20mucolipidosis-associated%20protein%20TRPML1%20is%20an%20endolysosomal%20iron%20release%20channel&author=XP%20Dong&author=X%20Cheng&author=E%20Mills&author=M%20Delling&author=F%20Wang&volume=455&publication_year=2008&pages=992-996&pmid=18794901&doi=10.1038/nature07311&)

11. Encode_Project_Consortium (2012) An integrated encyclopedia of DNA elements in the human genome. Nature 489:57–74.  [DOI](https://doi.org/10.1038/nature11247) | [PMC free article](/articles/PMC3439153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20encyclopedia%20of%20DNA%20elements%20in%20the%20human%20genome&volume=489&publication_year=2012&pages=57-74&pmid=22955616&doi=10.1038/nature11247&)

12. Franz M, Rodriguez H, Lopes C, Zuberi K, Montojo J, Bader GD, Morris Q (2018) GeneMANIA update 2018. Nucleic Acids Res 46:W60–W64.  [DOI](https://doi.org/10.1093/nar/gky311) | [PMC free article](/articles/PMC6030815/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29912392/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=GeneMANIA%20update%202018&author=M%20Franz&author=H%20Rodriguez&author=C%20Lopes&author=K%20Zuberi&author=J%20Montojo&volume=46&publication_year=2018&pages=W60-W64&pmid=29912392&doi=10.1093/nar/gky311&)

13. Gelernter J, Kranzler HR, Sherva R, Koesterer R, Almasy L, Zhao H, Farrer LA (2014) Genome-wide association study of opioid dependence: multiple associations mapped to calcium and potassium pathways. Biological psychiatry 76:66–74.  [DOI](https://doi.org/10.1016/j.biopsych.2013.08.034) | [PMC free article](/articles/PMC3992201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24143882/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biological%20psychiatry&title=Genome-wide%20association%20study%20of%20opioid%20dependence:%20multiple%20associations%20mapped%20to%20calcium%20and%20potassium%20pathways&author=J%20Gelernter&author=HR%20Kranzler&author=R%20Sherva&author=R%20Koesterer&author=L%20Almasy&volume=76&publication_year=2014&pages=66-74&pmid=24143882&doi=10.1016/j.biopsych.2013.08.034&)

14. GTEx-Consortium (2015) Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Science 348:648–660.  [DOI](https://doi.org/10.1126/science.1262110) | [PMC free article](/articles/PMC4547484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25954001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Human%20genomics.%20The%20Genotype-Tissue%20Expression%20(GTEx)%20pilot%20analysis:%20multitissue%20gene%20regulation%20in%20humans&volume=348&publication_year=2015&pages=648-660&pmid=25954001&doi=10.1126/science.1262110&)

15. Jalal H, Buchanich JM, Roberts MS, Balmert LC, Zhang K, Burke DS (2018) Changing dynamics of the drug overdose epidemic in the United States from 1979 through 2016. Science 361.  [DOI](https://doi.org/10.1126/science.aau1184) | [PMC free article](/articles/PMC8025225/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30237320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Changing%20dynamics%20of%20the%20drug%20overdose%20epidemic%20in%20the%20United%20States%20from%201979%20through%202016&author=H%20Jalal&author=JM%20Buchanich&author=MS%20Roberts&author=LC%20Balmert&author=K%20Zhang&volume=361&publication_year=2018&pmid=30237320&doi=10.1126/science.aau1184&)

16. Kang HM, Sul JH, Service SK, Zaitlen NA, Kong SY, Freimer NB, Sabatti C, Eskin E (2010) Variance component model to account for sample structure in genome-wide association studies. Nat Genet 42:348–354.  [DOI](https://doi.org/10.1038/ng.548) | [PMC free article](/articles/PMC3092069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20208533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Variance%20component%20model%20to%20account%20for%20sample%20structure%20in%20genome-wide%20association%20studies&author=HM%20Kang&author=JH%20Sul&author=SK%20Service&author=NA%20Zaitlen&author=SY%20Kong&volume=42&publication_year=2010&pages=348-354&pmid=20208533&doi=10.1038/ng.548&)

17. Kent WJ, Sugnet CW, Furey TS, Roskin KM, Pringle TH, Zahler AM, Haussler D (2002) The human genome browser at UCSC. Genome Research 12:996–1006.  [DOI](https://doi.org/10.1101/gr.229102) | [PMC free article](/articles/PMC186604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12045153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Research&title=The%20human%20genome%20browser%20at%20UCSC&author=WJ%20Kent&author=CW%20Sugnet&author=TS%20Furey&author=KM%20Roskin&author=TH%20Pringle&volume=12&publication_year=2002&pages=996-1006&pmid=12045153&doi=10.1101/gr.229102&)

18. Kharasch ED, Regina KJ, Blood J, Friedel C (2015) Methadone pharmacogenetics: CYP2B6 polymorphisms determine plasma concentrations, clearance, and metabolism. Anesthesiology 123:1142–1153.  [DOI](https://doi.org/10.1097/ALN.0000000000000867) | [PMC free article](/articles/PMC4667947/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26389554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Methadone%20pharmacogenetics:%20CYP2B6%20polymorphisms%20determine%20plasma%20concentrations,%20clearance,%20and%20metabolism&author=ED%20Kharasch&author=KJ%20Regina&author=J%20Blood&author=C%20Friedel&volume=123&publication_year=2015&pages=1142-1153&pmid=26389554&doi=10.1097/ALN.0000000000000867&)

19. Montell C (2001) Physiology, phylogeny, and functions of the TRP superfamily of cation channels. Science’s STKE : signal transduction knowledge environment 2001:re1.  [DOI](https://doi.org/10.1126/stke.2001.90.re1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11752662/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science%E2%80%99s%20STKE%20:%20signal%20transduction%20knowledge%20environment&title=Physiology,%20phylogeny,%20and%20functions%20of%20the%20TRP%20superfamily%20of%20cation%20channels&author=C%20Montell&volume=2001&publication_year=2001&pages=re1&pmid=11752662&doi=10.1126/stke.2001.90.re1&)

20. Nadpara PA, Joyce AR, Murrelle EL, Carroll NW, Carroll NV, Barnard M, Zedler BK (2018) Risk factors for serious prescription opioid-induced respiratory depression or overdose: comparison of commercially insured and veterans health affairs populations. Pain medicine (Malden, Mass) 19:79–96.  [DOI](https://doi.org/10.1093/pm/pnx038) | [PMC free article](/articles/PMC5939871/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28419384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain%20medicine%20(Malden,%20Mass)&title=Risk%20factors%20for%20serious%20prescription%20opioid-induced%20respiratory%20depression%20or%20overdose:%20comparison%20of%20commercially%20insured%20and%20veterans%20health%20affairs%20populations&author=PA%20Nadpara&author=AR%20Joyce&author=EL%20Murrelle&author=NW%20Carroll&author=NV%20Carroll&volume=19&publication_year=2018&pages=79-96&pmid=28419384&doi=10.1093/pm/pnx038&)

21. Nelson EC, Agrawal A, Heath AC, Bogdan R, Sherva R, Zhang B, Al-Hasani R, Bruchas MR, Chou YL, Demers CH, Carey CE, Conley ED, Fakira AK, Farrer LA, Goate A, Gordon S, Henders AK, Hesselbrock V, Kapoor M, Lynskey MT, Madden PA, Moron JA, Rice JP, Saccone NL, Schwab SG, Shand FL, Todorov AA, Wallace L, Wang T, Wray NR, Zhou X, Degenhardt L, Martin NG, Hariri AR, Kranzler HR, Gelernter J, Bierut LJ, Clark DJ, Montgomery GW (2016) Evidence of CNIH3 involvement in opioid dependence. Mol Psychiatry 21:608–614.  [DOI](https://doi.org/10.1038/mp.2015.102) | [PMC free article](/articles/PMC4740268/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26239289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Evidence%20of%20CNIH3%20involvement%20in%20opioid%20dependence&author=EC%20Nelson&author=A%20Agrawal&author=AC%20Heath&author=R%20Bogdan&author=R%20Sherva&volume=21&publication_year=2016&pages=608-614&pmid=26239289&doi=10.1038/mp.2015.102&)

22. Peters J, De Vries TJ (2012) Glutamate mechanisms underlying opiate memories. Cold Spring Harb Perspect Med 2:a012088.  [DOI](https://doi.org/10.1101/cshperspect.a012088) | [PMC free article](/articles/PMC3426813/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22951449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb%20Perspect%20Med&title=Glutamate%20mechanisms%20underlying%20opiate%20memories&author=J%20Peters&author=TJ%20De%20Vries&volume=2&publication_year=2012&pages=a012088&pmid=22951449&doi=10.1101/cshperspect.a012088&)

23. Pierucci-Lagha A, Gelernter J, Chan G, Arias A, Cubells JF, Farrer L, Kranzler HR (2007) Reliability of DSM-IV diagnostic criteria using the semi-structured assessment for drug dependence and alcoholism (SSADDA). Drug Alcohol Depen 91:85–90.  [DOI](https://doi.org/10.1016/j.drugalcdep.2007.04.014) | [PMC free article](/articles/PMC2039919/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17590536/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depen&title=Reliability%20of%20DSM-IV%20diagnostic%20criteria%20using%20the%20semi-structured%20assessment%20for%20drug%20dependence%20and%20alcoholism%20(SSADDA)&author=A%20Pierucci-Lagha&author=J%20Gelernter&author=G%20Chan&author=A%20Arias&author=JF%20Cubells&volume=91&publication_year=2007&pages=85-90&pmid=17590536&doi=10.1016/j.drugalcdep.2007.04.014&)

24. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D (2006) Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet 38:904–909.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

25. Ramasamy A, Trabzuni D, Guelfi S, Varghese V, Smith C, Walker R, De T, Coin L, de Silva R, Cookson MR, Singleton AB, Hardy J, Ryten M, Weale ME, Consortium UBE, Consor NABE (2014) Genetic variability in the regulation of gene expression in ten regions of the human brain. Nat Neurosci 17:1418–1428.  [DOI](https://doi.org/10.1038/nn.3801) | [PMC free article](/articles/PMC4208299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25174004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Neurosci&title=Genetic%20variability%20in%20the%20regulation%20of%20gene%20expression%20in%20ten%20regions%20of%20the%20human%20brain&author=A%20Ramasamy&author=D%20Trabzuni&author=S%20Guelfi&author=V%20Varghese&author=C%20Smith&volume=17&publication_year=2014&pages=1418-1428&pmid=25174004&doi=10.1038/nn.3801&)

26. Sherva R, Wang Q, Kranzler H, Zhao H, Koesterer R, Herman A, Farrer LA, Gelernter J (2016) Genome-wide association study of cannabis dependence severity, novel risk variants, and shared genetic risks. JAMA Psychiatry 73:472–480.  [DOI](https://doi.org/10.1001/jamapsychiatry.2016.0036) | [PMC free article](/articles/PMC4974817/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27028160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Psychiatry&title=Genome-wide%20association%20study%20of%20cannabis%20dependence%20severity,%20novel%20risk%20variants,%20and%20shared%20genetic%20risks&author=R%20Sherva&author=Q%20Wang&author=H%20Kranzler&author=H%20Zhao&author=R%20Koesterer&volume=73&publication_year=2016&pages=472-480&pmid=27028160&doi=10.1001/jamapsychiatry.2016.0036&)

27. Smith AH, Jensen KP, Li J, Nunez Y, Farrer LA, Hakonarson H, Cook-Sather SD, Kranzler HR, Gelernter J (2017) Genome-wide association study of therapeutic opioid dosing identifies a novel locus upstream of OPRM1. Mol Psychiatry 22:346–352.  [DOI](https://doi.org/10.1038/mp.2016.257) | [PMC free article](/articles/PMC5407902/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28115739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Genome-wide%20association%20study%20of%20therapeutic%20opioid%20dosing%20identifies%20a%20novel%20locus%20upstream%20of%20OPRM1&author=AH%20Smith&author=KP%20Jensen&author=J%20Li&author=Y%20Nunez&author=LA%20Farrer&volume=22&publication_year=2017&pages=346-352&pmid=28115739&doi=10.1038/mp.2016.257&)

28. Subramanian A, Narayan R, Corsello SM, Peck DD, Natoli TE, Lu X, Gould J, Davis JF, Tubelli AA, Asiedu JK, Lahr DL, Hirschman JE, Liu Z, Donahue M, Julian B, Khan M, Wadden D, Smith IC, Lam D, Liberzon A, Toder C, Bagul M, Orzechowski M, Enache OM, Piccioni F, Johnson SA, Lyons NJ, Berger AH, Shamji AF, Brooks AN, Vrcic A, Flynn C, Rosains J, Takeda DY, Hu R, Davison D, Lamb J, Ardlie K, Hogstrom L, Greenside P, Gray NS, Clemons PA, Silver S, Wu X, Zhao WN, Read-Button W, Wu X, Haggarty SJ, Ronco LV, Boehm JS, Schreiber SL, Doench JG, Bittker JA, Root DE, Wong B, Golub TR (2017) A next generation connectivity map: L1000 platform and the first 1,000,000 frofiles. Cell 171:1437–1452 e1417.  [DOI](https://doi.org/10.1016/j.cell.2017.10.049) | [PMC free article](/articles/PMC5990023/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29195078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=A%20next%20generation%20connectivity%20map:%20L1000%20platform%20and%20the%20first%201,000,000%20frofiles&author=A%20Subramanian&author=R%20Narayan&author=SM%20Corsello&author=DD%20Peck&author=TE%20Natoli&volume=171&publication_year=2017&pages=1437-1452%20e1417&pmid=29195078&doi=10.1016/j.cell.2017.10.049&)

29. Ward LD, Kellis M (2016) HaploReg v4: systematic mining of putative causal variants, cell types, regulators and target genes for human complex traits and disease. Nucleic Acids Res 44:D877–881.  [DOI](https://doi.org/10.1093/nar/gkv1340) | [PMC free article](/articles/PMC4702929/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26657631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=HaploReg%20v4:%20systematic%20mining%20of%20putative%20causal%20variants,%20cell%20types,%20regulators%20and%20target%20genes%20for%20human%20complex%20traits%20and%20disease&author=LD%20Ward&author=M%20Kellis&volume=44&publication_year=2016&pages=D877-881&pmid=26657631&doi=10.1093/nar/gkv1340&)

30. Webster LR (2017) Risk factors for opioid-use disorder and overdose. Anesth Analg 125:1741–1748.  [DOI](https://doi.org/10.1213/ANE.0000000000002496) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29049118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesth%20Analg&title=Risk%20factors%20for%20opioid-use%20disorder%20and%20overdose&author=LR%20Webster&volume=125&publication_year=2017&pages=1741-1748&pmid=29049118&doi=10.1213/ANE.0000000000002496&)

31. White JM, Irvine RJ (1999) Mechanisms of fatal opioid overdose. Addiction 94:961–972.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10707430/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=Mechanisms%20of%20fatal%20opioid%20overdose&author=JM%20White&author=RJ%20Irvine&volume=94&publication_year=1999&pages=961-972&pmid=10707430&)

32. Zedler B, Xie L, Wang L, Joyce A, Vick C, Brigham J, Kariburyo F, Baser O, Murrelle L (2015) Development of a risk index for serious prescription opioid-induced respiratory depression or overdose in veterans’ health administration patients. Pain medicine (Malden, Mass) 16:1566–1579.  [DOI](https://doi.org/10.1111/pme.12777) | [PMC free article](/articles/PMC4744747/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26077738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain%20medicine%20(Malden,%20Mass)&title=Development%20of%20a%20risk%20index%20for%20serious%20prescription%20opioid-induced%20respiratory%20depression%20or%20overdose%20in%20veterans%E2%80%99%20health%20administration%20patients&author=B%20Zedler&author=L%20Xie&author=L%20Wang&author=A%20Joyce&author=C%20Vick&volume=16&publication_year=2015&pages=1566-1579&pmid=26077738&doi=10.1111/pme.12777&)

33. Zhou H, Polimanti R, Yang BZ, Wang Q, Han S, Sherva R, Nunez YZ, Zhao H, Farrer LA, Kranzler HR, Gelernter J (2017) Genetic risk variants associated with comorbid alcohol dependence and major depression. JAMA Psychiatry 74:1234–1241.  [DOI](https://doi.org/10.1001/jamapsychiatry.2017.3275) | [PMC free article](/articles/PMC6331050/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29071344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Psychiatry&title=Genetic%20risk%20variants%20associated%20with%20comorbid%20alcohol%20dependence%20and%20major%20depression&author=H%20Zhou&author=R%20Polimanti&author=BZ%20Yang&author=Q%20Wang&author=S%20Han&volume=74&publication_year=2017&pages=1234-1241&pmid=29071344&doi=10.1001/jamapsychiatry.2017.3275&)

34. Zhou X, Stephens M (2012) Genome-wide efficient mixed-model analysis for association studies. Nat Genet 44:821–824.  [DOI](https://doi.org/10.1038/ng.2310) | [PMC free article](/articles/PMC3386377/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22706312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20efficient%20mixed-model%20analysis%20for%20association%20studies&author=X%20Zhou&author=M%20Stephens&volume=44&publication_year=2012&pages=821-824&pmid=22706312&doi=10.1038/ng.2310&)
