# Predicting steady‐state endoxifen plasma concentrations in breast cancer patients by CYP2D6 genotyping or phenotyping. Which approach is more reliable?

## Metadata
**Authors:** Milena Gusella, Felice Pasini, Barbara Corso, Laura Bertolaso, Giovanni De Rosa, Cristina Falci, Yasmina Modena, Carmen Barile, Donatella Da Corte Z, AnnaPaola Fraccon, Silvia Toso, Elisabetta Cretella, Antonella Brunello, Caterina Modonesi, Romana Segati, Cristina Oliani, Nadia Minicuci, Roberto Padrini, for the Italian TAM Group
**Journal:** Pharmacology Research & Perspectives
**Date:** 2020 Aug 19
**DOI:** [10.1002/prp2.646](https://doi.org/10.1002/prp2.646)
**PMID:** 32813313
**PMCID:** PMC7437348
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7437348/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7437348/pdf/PRP2-8-e00646.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7437348/pdf/PRP2-8-e00646.pdf)

## Abstract

In previous studies, steady‐state Z‐endoxifen plasma concentrations (ENDOss) correlated with relapse‐free survival in women on tamoxifen (TAM) treatment for breast cancer. ENDOss also correlated significantly with CYP2D6 genotype (activity score) and CYP2D6 phenotype (dextromethorphan test). Our aim was to ascertain which method for assessing CYP2D6 activity is more reliable in predicting ENDOss. The study concerned 203 Caucasian women on tamoxifen‐adjuvant therapy (20 mg q.d.). Before starting treatment, CYP2D6 was genotyped (and activity scores computed), and the urinary log(dextromethorphan/dextrorphan) ratio [log(DM/DX)] was calculated after 15 mg of oral dextromethorphan. Plasma concentrations of TAM, N‐desmethyl‐tamoxifen (ND‐TAM), Z‐4OH‐tamoxifen (4OH‐TAM) and ENDO were assayed 1, 4, and 8 months after first administering TAM. Multivariable regression analysis was used to identify the clinical and laboratory variables predicting log‐transformed ENDOss (log‐ENDOss). Genotype‐derived CYP2D6 phenotypes (PM, IM, NM, EM) and log(DM/DX) correlated independently with log‐ENDOss. Genotype‐phenotype concordance was almost complete only for poor metabolizers, whereas it emerged that 34% of intermediate, normal, and ultrarapid metabolizers were classified differently based on log(DM/DX). Multivariable regression analysis selected log(DM/DX) as the best predictor, with patients’ age, weak inhibitor use, and CYP2D6 phenotype decreasingly important: log‐ENDOss = 0.162 ‐ log(DM/DX) × 0.170 + age × 0.0063 ‐ weak inhibitor use × 0.250 + IM × 0.105 + (NM + UM) × 0.210; (R 2 = 0.51). In conclusion, log(DM/DX) seems superior to genotype‐derived CYP2D6 phenotype in predicting ENDOss.

Keywords: breast cancer, CYP2D6, dextromethorphan, endoxifen

## What is already known about this subject

## What is already known about this subject

## What this study adds

## 1. INTRODUCTION

Tamoxifen (TAM) is a selective estrogen receptor antagonist used as adjuvant therapy to prevent estrogen‐receptor‐positive breast cancer recurrence. TAM is de facto a prodrug because its anti‐estrogen activity is 30‐100 times less than that of its metabolites Z‐4OH‐tamoxifen (4OH‐TAM) and Z‐endoxifen (ENDO).[1](#prp2646-bib-0001), [2](#prp2646-bib-0002) ENDO is considered the most effective metabolite in vivo, with plasma concentrations 5‐10 times higher than 4OH‐TAM.[3](#prp2646-bib-0003) Two well‐powered trials found ENDO plasma concentrations correlated with recurrence risk.[4](#prp2646-bib-0004), [5](#prp2646-bib-0005) Madlensky et al [4](#prp2646-bib-0004) reported that women with ENDO concentrations >5.97 ng ml‐1 (>16 nM) had a 30% lower relative risk of breast cancer recurrence. Saladores et al [5](#prp2646-bib-0005) found ENDO levels <5.2ng ml‐1 (<14 nM) associated with a shorter relapse‐free survival (RFS) compared with >13.0 ng ml‐1 (>35 nM). Hence the suggestion that monitoring ENDO concentrations can be used to individualize adjuvant TAM therapy.[6](#prp2646-bib-0006), [7](#prp2646-bib-0007)

An alternative strategy involves measuring predictors of steady‐state ENDO levels (ENDOss) before starting TAM therapy. While several enzymes contribute to ENDO formation (CYP3A4/5, CYP2C9, CYP2C19, CYP1A2) and elimination (UGTs, SULTs), the main metabolic pathway is ENDO formation from N‐desmethyl‐tamoxifen (ND‐TAM) by the cytochrome *CYP2D6* [8](#prp2646-bib-0008) (Figure [1](#prp2646-fig-0001)). *CYP2D6* activity has been estimated indirectly by combining the several *CYP2D6* allelic variants with a different gene expression,[9](#prp2646-bib-0009) or calculated directly from the dextromethorphan (DM)/dextrorphan (DX) urinary metabolic ratio [log(DM/DX)].[10](#prp2646-bib-0010), [11](#prp2646-bib-0011) Both methods can predict ENDOss. *CYP2D6* phenotyping is considered superior to genotyping because non‐genetic factors like age, drug‐drug interactions, or co‐morbidities can affect phenotype (phenoconversion phenomenon),[12](#prp2646-bib-0012) but the two methods’ performance had yet to be compared directly.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/016c/7437348/488b2a805781/PRP2-8-e00646-g001.jpg)

Main metabolic pathways of tamoxifen. TAM: tamoxifen; DM‐TAM: desmethyl‐tamoxifen; 4OH‐TAM: Z‐4OH‐tamoxifen; ENDO: Z‐endoxifen; UGTs: UDP‐glucuronosyl transferases; SULTs: sulfotransferases

Our primary aim was to ascertain which method ‐ *CYP2D6* genotyping or phenotyping ‐ can predict ENDOss more accurately. The findings presented here are part of an ongoing prospective trial (TAM study) to correlate ENDOss with breast cancer recurrence.

## 2. METHODS

### 2.1. Patients and study design

This study concerned 203 Caucasian women with estrogen‐receptor‐positive breast cancer (stage IA 67.2%, IIA 23.2%, IIB 6.8%, and IIIA 2.8%) on TAM adjuvant therapy (20 mg q.d.) involved in a trial enrolling patients from 20 oncology units in Northern Italy.

Before starting TAM, blood samples were drawn for *CYP2D6* genotyping. *CYP2D6* phenotyping was done as follows: 15 mg of oral dextromethorphan were administered at 10 PM, then urine was collected overnight until 8 AM, when a sample was frozen at −20°C until analysis of DM and DX concentrations (see below).

One, 4, and 8 months after starting TAM, blood was sampled before a drug dose to assay plasma concentrations of TAM, ND‐TAM, 4OH‐TAM, and ENDO. All other routine procedures were completed according to local clinical practice.

The study protocol was approved by the Ethics Committee of Rovigo Hospital (Italy) and all participants gave their written informed consent.

### 2.2. Plasma assay of ENDO, 4OH‐TAM, ND‐TAM, and TAM

Tamoxifen and its metabolites were analyzed in patients’ plasma using a validated high‐performance liquid chromatography (HPLC) method,[13](#prp2646-bib-0013) with partial adaptations. Briefly, blood was centrifuged within an hour of sampling and plasma was stored at −20°C until analysis. One mL of plasma was alkalinized with 1 ml‐glycine/NaOH buffer (1M, pH: 11.3) and extracted with 7 ml of hexane/2‐propanol (95:5, v:v). After centrifugation, the supernatant was collected, dried under nitrogen stream and re‐suspended in 200 µl of mobile phase, then 30 µl were injected in a HPLC system (mod. 1515; Waters Corp, Milford, MA) for separation in a C18 column (Kromasil 100‐3.5C18, 150x4.6 mm). All compounds were then converted to more fluorescent derivatives with an UV photochemical reactor (PHRED, Aura Industries, NY, USA) using a 254 nm wavelength, then detected with a fluorescence detector (mod. 2487; Waters Corp, Milford, MA) with excitation and emission wavelengths set at 256 and 380 nm, respectively. The mobile phase consisted of 40% acetonitrile in phosphate buffer (20 mM, pH 3.0), with a flow rate of 1ml min‐1. Calibration curves were obtained with plasma from healthy volunteers by adding known concentrations of ENDO (range 1.25‐20 ng ml‐1), 4OH‐TAM (0.625‐10 ng ml‐1), ND‐TAM (25‐400 ng ml‐1), and TAM (25‐ 400 ng ml‐1). Two internal standards were used: propranolol for TAM; and ND‐TAM (0.5 µg ml‐1) and verapamil for ENDO and 4OH‐TAM (0.25 µg ml‐1). Calibration curves were considered acceptable if *R* ^2^ ≥ 0.99. Precision, accuracy, and quantification limits are shown in Appendix [1](#prp2646-sup-0001).

### 2.3. CYP2D6 genotyping procedure

Germline DNA was isolated from blood using Wizard Genomic DNA Purification Kit (Promega) according to the manufacturer's recommendations.

Samples were analyzed for six polymorphisms and a full gene deletion, accounting for most of the clinically significant variants of *CYP2D6* in Caucasian populations.[14](#prp2646-bib-0014)

Genotyping was conducted using PCR/RLFP‐based methods for *CYP2D6*3 (*2549 A del, rs35742686*), CYP2D6*4* (1846G>A, rs3892097)*, CYP2D6*6* (1707T del, rs5030655*), CYP2D6*9* (2615_2617del AAG, rs5030656), *CYP2D6*10* (100C>T, rs1065852), with digestion by MspI, BstN1, BtsI, MboII, and HphI, respectively, as in other studies.[15](#prp2646-bib-0015), [16](#prp2646-bib-0016), [17](#prp2646-bib-0017), [18](#prp2646-bib-0018)

Allele *41 (G2988A, rs28371725) was detected using denaturing HPLC. Specific primers were designed with Primer3 software [19](#prp2646-bib-0019) and confirmed with Human Genome Browser in silico tools as follows: fw 5’‐GAGCCCATCTGGGAAACAGT‐3’ and rv 5’‐CCTCCTATGTTGGAGGAGGTC‐3’. PCR was performed with 1U of Hot Start DNA polymerase AmpliTaq Gold (Applied Biosystems) in a final volume of 50 µL; the annealing temperature was 58°C, for 38 cycles. The optimal melting temperatures for SNP detection was experimentally determined as 62.8°C. Each sample was run alone and with a plasmid positive control (containing the *CYP2D6* 2988A variant, obtained with the QuikChange Site‐Directed Mutagenesis kit by Agilent Technologies) for 8 min with a gradient mobile phase consisting of Buffers A (triethyl ammonium acetate) and B (triethyl ammonium acetate and acetonitrile) at a flow rate of 0.9 ml min‐1. Retention times of 4.5 and 5 min were associated with heteroduplex (2988G/A) and homoduplex (2988A) profiles, detecting wild‐type and mutant alleles, respectively. Variant genotypes were verified by direct Sanger sequencing (CEQ2000XL, Beckman Coulter). Full *CYP2D6* deletion (*CYP2D6**5) analysis was conducted with a long‐range PCR using the DyNAzyme II DNA Polymerase kit (Thermo Fisher Scientific) according to the manufacturer's instructions and a 1% agarose gel run, as described by Sistonen et al.[20](#prp2646-bib-0020)

The *CYP2D6* activity score was calculated according to the Clinical Pharmacogenetic Implementation Consortium and Dutch Pharmacogenetics Working Group criteria,[21](#prp2646-bib-0021) which assigned scores of 0, 0.25, 0.5, 1, or 2 to each allele based on their relative activity compared with the wild type (=1), as follows: *3, *4, *5, *6 = 0; *10 = 0.25; *9, *41 = 0.5; no variant alleles = 1; and *1 × 2N = 2. The sum of the activity scores for each allele (AS) was translated into the following *CYP2D6* phenotypes: ultrarapid metabolizers (UM), AS > 2.25; normal metabolizers (NM), 1.25 ≤ AS ≤2.25; intermediate metabolizers (IM), 0 < AS <1.25; poor metabolizers (PM), AS = 0.

### 2.4. Urinary DM and DX assay

Urinary DM and DX were tested using HPLC according to Flores‐Péres et al,[22](#prp2646-bib-0022) with slight modifications. Before the extraction procedure, 0.5 ml of urine was hydrolyzed overnight at 37°C by adding 0.5 ml of a solution of β‐glucuronidase (2000 U ml‐1) in acetate buffer (pH 5). This step was necessary because most DX in urine is in the form of glucuronide. Then 500 mL of hydrolysate were spiked with 25 µl of a 0.1 mg ml‐1 levallophan solution (as internal standard) and 500 µL of carbonate buffer (pH 9.2) were added. Extraction was done with 3.5 mL of a hexane‐butanol mixture (95:5, v/v) in a shaker rotated for 10 minutes. After centrifugation at 855 g for 5 minutes, the organic phase was separated and evaporated to dryness at 55°C under gentle nitrogen stream. The residue was reconstituted with 1 ml of mobile phase and 10 µl were injected in the HPLC column (Kromasil® 100‐5 phenyl, 250 × 4.6mm), thermostated at 30°C. The mobile phase, a mixture of acetonitrile and acetic acid 1% + triethylamine 0.1% (35:65), was fluxed at 1 ml min‐1 with an isocratic pump (mod. 1515; Waters Corp, Milford, MA). The effluent was analyzed with a fluorescence detector (mod. 2487; Waters Corp, Milford, MA) set at excitation and emission wavelengths of 275 nm 310 nm, connected with Empower 3 software (Waters Corp Milford, MA).

Calibration curves were prepared by adding increasing volumes of the working solutions of dextromethorphan hydrobromide (0.1 mg ml‐1 = 270mM) and dextrorphan tartrate (0.1 mg ml‐1 = 245mM) to distilled water to obtain concentrations in the range of 0.25‐10 μg ml‐1. Within this range, the curves were linear with a coefficient of determination (*R* ^2^) always > 0.99. Precision, accuracy, and quantification limits are shown in Appendix [1](#prp2646-sup-0001).

### 2.5. CYP2D6 phenotyping procedure

The logarithm of the ratio of urinary DM to DX molar concentrations [log(DM/DX)] was taken as a measure of *CYP2D6* activity. Patients were classified as poor metabolizers (PM), intermediate metabolizers (IM), extensive metabolizers (NM), or ultra‐rapid metabolizers (UM) according to their log(DM/DX) ratio, as follows: PM ≥ −0.52; IM <−0.52 and ≥ −1.52; NM, <−1.52 and ≥‐2.52; UM <−2.52.[10](#prp2646-bib-0010)

Since ND‐TAM is metabolized to ENDO by the cytochrome *CYP2D6*, the logarithm of the ratio of ND‐TAM to ENDO plasma concentrations [log(ND‐TAM/ENDO)] was considered another independent measure of *CYP2D6* activity.

### 2.6. Statistical analysis

In the tables, continuous variables are presented as means ± standard deviations (unless otherwise stated), and categorical variables as absolute numbers and percentages.

Continuous variables with a normal distribution were compared with Student's *t* test. One‐way ANOVA was used for comparing more than two independent groups, followed by Bonferroni post‐hoc tests for pairwise comparisons, and the test for linear trend, as needed. Two‐way ANOVA was used to compare repeated measures from the same patient. The homoscedasticity assumption was verified with the Bartlett and Levene test. Equivalent non‐parametric tests (the Mann‐Whitney U, Kruskal‐Wallis and Friedman tests) were used whenever applicability conditions were not met. Categorical data frequencies were examined using Pearson's chi‐square and Fisher's exact test, as appropriate.

ENDOss was calculated as the mean of ENDO concentrations at 4 and 8 months, if both measurements were available, or at 4 months otherwise. A log‐transformation was applied (log‐ENDOss) to achieve a normal distribution of ENDOss.

The following independent variables were used in the regression analyses: age (years), body weight (kg), body surface area (BSA; calculated with the Haycock formulae, m^2^), body mass index (BMI; kg m^2^‐1), log(DM/DX), concomitant use of *CYP2D6* weak inhibitors, and *CYP2D6* activity score. *CYP2D6* activity scores were translated into four phenotypes (UM, NM, IM, PM), according to the updated CPIC guidelines.[21](#prp2646-bib-0021) Since only one patient was classified as UM (genotype 1 × 2N*1), she was included in the NM group.

First, univariable linear regression analyses were run, taking one independent variable at a time. Then, a stepwise multivariable forward regression was conducted (*P* < .05 for variable inclusion, and *P* < .15 for variable removal) to select the best log‐ENDOss prediction model. Multicollinearity was checked using the tolerance and the variance inflation factor (VIF); variables with a tolerance <0.4 (VIF > 2.5) were discarded from the analysis.

Possible confounding effects were investigated with all variables excluded by the stepwise selection. Residuals analysis was performed to examine the models’ goodness of fit and adherence to the regression assumptions. The validity of the final model was assessed by measuring the *R* ^2^ coefficient and the mean absolute and percentage errors (MAE and MAPE) of the ENDOss predicted.

Pearson's correlation coefficients (r) were used to test the relationship between log(DM/DX) and log‐ENDOss, and between log(DM/DX) and log(NDT/ENDO).

A receiver operating characteristics (ROC) analysis was used to identify the threshold for the log(DM/DX) ratio associated with ENDOss <5.97 ng ml‐1, and the area under the ROC curve (AUC) was estimated.

All statistical analyses were performed using STATA SE, version 12.1 (Stata Statistical Software, College Station, TX: StataCorp LP), setting the level of significance at 0.05.

## 3. RESULTS

### 3.1. Patients’ characteristics

Of the population of 203 women, only 164 were suitable for multivariable regression analyses as all the independent variables were available. Table [1](#prp2646-tbl-0001) summarizes patients’ characteristics for the whole group and for the regression group. No patients were taking strong *CYP2D6* inhibitors, while 12 in the whole group and 9 in the regression group were taking weak inhibitors (citalopram, sertraline, duloxetine, venlafaxine). There were no statistically significant differences between the variables considered in the two groups, except for age (*P* = .0083), and menopausal status (*P* = .023).

### Table 1.

| Variable | Whole sample (N = 203) | Sample for regression analysis (N = 164) |
| --- | --- | --- |
| Age (years), mean ± SD [range] | 56.2 ± 11.7 [29‐89] | 57.2 ± 11.2 [33‐89] |
| Body weight (kg), mean ± SD [range] | 67.4 ± 13.5 [42 ‐ 115] | 67.8 ± 13.9 [43‐115] |
| Body surface area (m2), mean ± SD [range] | 1.75 ± 0.20 [1.33‐2.44] | 1.75 ± 0.20 [1.36‐2.44] |
| BMI (kg m2‐1), mean ± SD [range] | 25.7 ± 5.0 [15.8‐42.9] | 25.9 ± 5.1 [15.8‐42.9] |
| In menopause, n (%) |   |   |
| Yes | 147 (73%) | 126 (77%) |
| No | 53 (27%) | 38 (23%) |
| Weak inhibitor use, n (%) |   |   |
| Yes | 12 (6%) | 9 (5%) |
| No | 191 (94%) | 155 (95%) |
| ENDO concentration (ng ml‐1) after 1 month, mean ± SD [range] | 8.05 ± 4.85 [1.20‐27.00] | 8.13 ± 5.05 [1.20‐27.00] |
| ENDO concentration (ng ml‐1) in steady state, mean ± SD [range] | 10.57 ± 6.83 [1.60‐40.41] | 10.69 ± 6.88 [1.60‐40.41] |
| Log(DM/DX), mean ± SD [range] | −1.59 ± 0.89 [−3.08‐1.39] | −1.61 ± 0.87 [−3.08‐1.39] |
| CYP2D6 phenotype, n (%) |   |   |
| PM | 15 (8.1%) | 14 (8.6%) |
| IM | 64 (34.2%) | 53 (32.3%) |
| NM | 107 (57.2%) | 96 (58.5%) |
| UM | 1 (0.5%) | 1 (0.6%) |

Table 1 Caption: Patients’ characteristics in the whole sample and in the group for regression analysis

### 3.2. ENDO, 4OH‐TAM, ND‐TAM and TAM plasma concentrations

Figure [2](#prp2646-fig-0002) shows the median plasma concentrations (box and whisker plots) of all compounds during the follow‐up. All measures showed a wide inter‐subject variability.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/016c/7437348/8f35961a2a22/PRP2-8-e00646-g002.jpg)

Box and whisker plots (circles are outliers) of plasma concentrations of endoxifen (ENDO), 4OH‐tamoxifen (4OH‐TAM), N‐desmethyl‐tamoxifen (ND‐TAM), and tamoxifen (TAM) after 1, 4, and 8 months. Asterisks indicate significant differences from values at 1 month

Four separate two‐way repeated‐measures ANOVAs were run to identify any differences in the concentrations of the four compounds at the different times (1, 4 and 8 months). The results showed that mean ENDO, ND‐TAM and TAM concentrations rose significantly from the first to the fourth month, then remained stable (for all three compounds, comparisons were significant [*P* < .0001] for month 1 vs month 4, and for month 1 vs month 8), while 4OH‐TAM concentrations remained stable throughout the observation period. The mean absolute difference in ENDO concentrations between month 4 and month 8 was 2.5 ± 2.6 ng mL‐1 (mean change: +8%, n.s.).

### 3.3. Log(DM/DX) and CYP2D6 phenotype

One‐way ANOVA followed by testing for linear trends showed a significant difference in the mean log(DM/DX) values across groups identified by *CYP2D6* phenotype (*P* < .0001) (Figure [3](#prp2646-fig-0003)). These log(DM/DX) values varied considerably within each group, however, indicating that CYP2D6 activity inferred from *CYP2D6* genotype cannot accurately predict the phenotype. In fact, 34% of patients classified according to the *CYP2D6* genotype [21](#prp2646-bib-0021) did not match the phenotype assessed with the log (DM/DX) classification system.[10](#prp2646-bib-0010)

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/016c/7437348/84620f5fc020/PRP2-8-e00646-g003.jpg)

Distribution of log(DM/DX) across the four CYP2D6 phenotypes. Filled symbols refer to the concentrations in users of weak inhibitors. Horizontal lines represent means and vertical bars 95% confidence intervals. Dashed arrows indicate the log(DM/DX) cut‐offs that separate poor (PM), intermediate (IM), extensive (EM), and ultra‐rapid metabolizers (UM)

### 3.4. ENDOss, Log(DM/DX), and CYP2D6 phenotype

Log‐ENDOss correlated inversely with log(DM/DX) (r = 0.63; *P* < .0001) (Figure [4](#prp2646-fig-0004), panel a) and one‐way ANOVA showed a rising trend of log‐ENDOss in parallel with *CYP2D6* phenotype (significant comparisons: PM vs IM, NM + UM; and IM vs NM + UM; *P* < .0001) (Figure [4](#prp2646-fig-0004), panel b). Similar correlations emerged for steady‐state 4OH‐TAM concentrations (4OH‐TAMss), whereas steady‐state DM‐TAM levels (DM‐TAMss) correlated directly with log(DM/DX), and inversely with *CYP2D6* phenotype (data not shown). TAM concentrations did not correlate with *CYP2D6* activity markers.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/016c/7437348/84babdce174e/PRP2-8-e00646-g004.jpg)

Panel (a): correlation between log(DM/DX) and log‐transformed steady‐state endoxifen concentrations (log‐ENDOss). The dashed arrow indicates the best log(DM/DX) cut‐off associated with ENDOss < 5.97 ng ml‐1 Panel (b): distribution of log‐ENDOss across the four CYP2D6 phenotypes. The dashed line indicates the log‐ENDOss cut‐off of 0.779, corresponding to ENDOss of 5.97 ng ml‐1.

Of note, urinary log(DM/DX) correlated significantly with plasma log(ND‐TAM/ENDO) at 1 month (*r* = 0.70; *P* < .0001), indicating that both ratios reflect *CYP2D6* metabolic activity (Figure [5](#prp2646-fig-0005)).

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/016c/7437348/d3b6a6c7e68d/PRP2-8-e00646-g005.jpg)

Correlation between urinary log(DM/DX) ratio and plasma log(ND‐TAM/ENDO) ratio measured after 1 month of therapy

Considering the ENDOss concentration of 5.97 ng ml‐1 indicated by Madlensky et al [4](#prp2646-bib-0004) as the threshold for a favorable clinical outcome, our data show that all patients with an activity score of 0 had sub‐therapeutic ENDO levels (Figure [4](#prp2646-fig-0004), panel b). ROC analysis identified a cut‐off for log(DM/DX) of −1.445 beyond which most patients had ENDOss ≤5.97 ng ml‐1 (0.776 in log‐scale), with 89% sensitivity and 67% specificity (AUC = 0.82, 95% confidence interval: 0.74‐0.91, *P* < .001) (Figure [4](#prp2646-fig-0004), panel a).

### 3.5. Univariable e multivariable analyses

On univariable linear regression, the following variables significantly predicted log‐ENDOss: log(DM/DX) (*R* ^2^ = 0.39); *CYP2D6* phenotype (*R* ^2^ = 0.37); weak inhibitor use (*R* ^2^ = 0.055); and body surface area (*R* ^2^ = 0.032) (Table [2](#prp2646-tbl-0002)). After multicollinearity checking, body surface area was discarded from subsequent analyses due to its collinearity with BMI.

### Table 2.

| Variables | Intercept (95% CI) | β Coefficients (95% CI) | P‐value | R 2 |
| --- | --- | --- | --- | --- |
| Log(DM/DX) | 0.61 (0.54 to 0.68) | −0.21 (−0.25 to −0.17) | <.0001 | 39.25 |
| CYP2D6 phenotype | 0.44 (0.32 to 0.56) |   |   |   |
| IM |   | 0.46 (0.32 to 0.60) | <.0001 | 34.73 |
| NM + UM |   | 0.60 (0.47 to 0.73) | <.0001 |   |
| Weak inhibitor use | 0.96 (0.91 to 1.00) | −0.29 (−0.48 to −0.11) | .002 | 5.53 |
| Body surface area (m2) | 1.39 (1.00 to 1.77) | −0.25 (−0.47 to −0.037) | .022 | 3.19 |
| BMI (kg m2‐1) | 1.05 (0.83 to 1.28) | −0.0043 (−0.013 to 0.004) | ns (.33) | 0.59 |
| Age, (years) | 0.89 (0.66 to 1.12) | 0.0009 (−0.003 to 0.005) | ns (.65) | 0.12 |

Table 2 Caption: Intercepts, β coefficients and significance levels obtained by univariable regression analyses

The stepwise multivariable regression analysis identified log(DM/DX), patient's age, weak inhibitor use, and *CYP2D6* phenotype as significant independent predictors of log‐ENDOss, ruling out BMI:

| log‐ENDOss=0.162‐logDM/DX×0.170+age×0.0063‐weak inhibitor use×0.250+IM×0.105+NM+UM×0.210 | (1) |
| --- | --- |
*R* ^2^ = 0.510; MAE = 0.16 ng ml‐1; MAPE = 19.9%

Log(DM/DX) and *CYP2D6* phenotype were collinear so they were alternately removed from the regression to see which model performed better:

| log‐ENDOss=0.225‐logDM/DX×0.223+age×0.0065‐weak inhibitor use×0.235 | (2) |
| --- | --- |
*R* ^2^ = 0.478; MAE = 0.16 ng ml‐1; MAPE = 21.2%

| log‐ENDOss=0.218+IM×0.444+NM+UM×0.608+age×0.0041‐weak inhibitor use×0.265 | (3) |
| --- | --- |
*R* ^2^ = 0.410; MAE = 0.17 ng ml‐1; MAPE = 21.8%

Equation [2](#prp2646-disp-0002), which included log(DM/DX), yielded a higher *R* ^2^ than Equation [3](#prp2646-disp-0003), with small changes in MAE and MAPE.

To translate these models into clinically relevant information, linear ENDOss were predicted for each patient by transforming the results of Equations [(1)](#prp2646-disp-0001), [(2)](#prp2646-disp-0002), [(3)](#prp2646-disp-0003) into the corresponding anti‐logarithms. Tables [3](#prp2646-tbl-0003), [4](#prp2646-tbl-0004), [5](#prp2646-tbl-0005) show the partial variances (*R* ^2^) explained by each variable in Equation [1](#prp2646-disp-0001), 2, and 3. Table [6](#prp2646-tbl-0006) shows the MAEs and MAPE (± SD, range) of the ENDOss obtained with each equation.

### Table 3.

| Variable | β Coefficient (95% CI) | P‐value | Partial R 2 |
| --- | --- | --- | --- |
| Intercept | 0.162 (−0.047 to 0.371) | .127 | — |
| Log(DM/DX) | −0.170 (−0.228 to −0.111) | <.0001 | 39.25 |
| Age (years) | 0.0063 (0.003 to 0.009) | <.0001 | 5.13 |
| Weak inhibitor use | −0.250 (−0.389 to −0.110) | .001 | 3.46 |
| CYP2D6 phenotype |   |   |   |
| IM | 0.105 (−0.064 to 0.275) | .221 | 3.17 |
| NM + UM | 0.210 (0.030 to 0.391) | .023 |   |

Table 3 Caption: β Coefficients and significance levels of variables significantly associated with log‐transformed ENDOss, by multivariable regression analysis (Equation 1)

### Table 4.

| Variable | β Coefficient (95% CI) | P‐value | Partial R 2 |
| --- | --- | --- | --- |
| Intercept | 0.225 (0.023 to 0.427) | .030 | — |
| Log(DM/DX) | −0.223 (−0.262 to −0.184) | <.0001 | 39.25 |
| Age (years) | 0.0065 (0.003 to 0.009) | <.0001 | 5.13 |
| Weak inhibitor use | −0.235 (−0.377 to −0.092) | .001 | 3.46 |

Table 4 Caption: β Coefficients and significance levels of variables significantly associated with log‐transformed ENDOss, after substituting Log(DM/DX) for CYP2D6 phenotype in multivariable regression analysis (Equation 2)

### Table 5.

| Variable | β Coefficient (95% CI) | P‐value | Partial R 2 |
| --- | --- | --- | --- |
| Intercept | 0.218 (−0.009 to 0.445) | .060 | — |
| CYP2D6 phenotype |   |   |   |
| IM | 0.444 (0.310 to 0.577) | <.0001 | 34.73 |
| NM + UM | 0.608 (0.480 to 0.735) | <.0001 |   |
| Weak inhibitor use | −0.265 (−0.418 to −0.112) | .001 | 3.47 |
| Age (years) | 0.0041 (0.001 to 0.007) | .010 | 1.86 |

Table 5 Caption: β Coefficients and significance levels of variables significantly associated with log‐transformed ENDOss, after removing log(DM/DX) from multivariable regression analysis (Equation 3)

### Table 6.

|   | Equations (n°) |  |  |
| --- | --- | --- | --- |
| 1 | 2 | 3 |  |
| MAE (ng ml‐1) |   |   |   |
| mean | 3.76 | 3.89 | 4.07 |
| SD | 4.09 | 4.09 | 4.49 |
| range | 0.002‐24.19 | 0.018‐24.49 | 0.016‐27.69 |
| MAPE (%) |   |   |   |
| mean | 39.74 | 41.84 | 43.53 |
| SD | 42.52 | 42.95 | 46.24 |
| range | 0.02‐252.59 | 0.18 −227.81 | 0.59‐239.69 |

Table 6 Caption: Mean absolute error (MAE) and mean absolute percentage error (MAPE) of ENDOss predictions obtained with the three models developed

## 4. DISCUSSION

This study showed that the co‐variables log(DM/DX), age and weak inhibitor use, and *CYP2D6* phenotype correlated significantly with log‐ENDOss on multiple regression analysis, explaining 51.0% of log‐ENDOss variability. The best predictor was log(DM/DX) (partial *R* ^2^ = 0.39), with the contributions of age (partial *R* ^2^ = 0.051), weak inhibitor use (partial *R* ^2^ = 0.035), and *CYP2D6* phenotype (partial *R* ^2^ = 0.032) decreasingly important (Table [3](#prp2646-tbl-0003)). When *CYP2D6* phenotype was removed from the regression, the overall *R* ^2^ was marginally lower (0.48), whereas removing log(DM/DX) resulted in a greater decrease in *R* ^2^ (0.41).

These results were not unexpected because the expression of *CYP2D6* activity is controlled by several nongenetic factors,[12](#prp2646-bib-0012) which matter especially in patients with intermediate‐to‐fast genotypes. In fact, 34% of IMs, NMs and UMs did not match the phenotype derived from log(DM/DX), whereas only 1 of 14 PMs was classified as IM by log(DM/DX) (Figure [3](#prp2646-fig-0003)).

Other potential limitations of the activity score system are that not all *CYP2D6* variant alleles are routinely genotyped and that the scoring criteria may change as new information becomes available. Indeed, the score has been challenged by Schroth et al,[23](#prp2646-bib-0023) who showed that downgrading *CYP2D6**10 activity score from 0.5 to 0.25 improved ENDOss prediction, so new guidelines have recently been updated.[21](#prp2646-bib-0021) In short, phenoconversion and activity score misclassification can both weaken the predictive power of *CYP2D6* genotype. On the other hand, genotype is stable for life, whereas DM levels may change due to intervening, non‐genetic factors (Table [7](#prp2646-tbl-0007)).

### Table 7.

|   | PROs | CONs |
| --- | --- | --- |
| Log(DM/DX) metabolic ratio | The influence of non‐genetic factors (drug‐drug interactions, co‐morbidities, pregnancy, etc) is included | Results may change over time Renal function and urine pH may affect the log(DM/DX) metabolic ratio Time‐consuming (urine collection, drug/metabolite assay) |
| CYP2D6 phenotype (CPIC) | Single blood sample required Genotype does not change over time | Not all CYP2D6 variants are routinely assessed The activity score attributed to each variant allele may be challenged Phenoconversion can bias the results |

Table 7 Caption: Pros and cons of the two methods used for phenotyping CYP2D6 activity

Several clinical studies investigated the correlation between *CYP2D6* genotype‐derived activity and ENDOss, using various methods and phenotyping criteria, and with mixed results. In general, individuals labelled as PMs have significantly lower ENDOss than EMs.[24](#prp2646-bib-0024) Only four studies phenotyped *CYP2D6* activity with the dextromethorphan test, using different experimental approaches. De Graan et al [25](#prp2646-bib-0025) calculated the area under the concentration‐time curve of DM during a 6‐hour interval in 40 women, finding it correlated inversely with trough ENDOss (*r* = −0.72). Opdam et al,[26](#prp2646-bib-0026) and Safgren et al [27](#prp2646-bib-0027) used the ^13^C‐DM breath test, measuring the expired ^13^CO_2_ as an index of DM demethylation: they reported significant correlations between the changes in ^13^CO_2_ and ENDOss, with r values of 0.56 (n = 77) and 0.69 (n = 65), respectively. Antunes et al [28](#prp2646-bib-0028) simultaneously phenotyped *CYP2D6* and *CYP3A4* activities in 116 patients by calculating the [DM]/[DX] and [omeprazole]/[omeprazole sulfone] metabolic ratios in a single plasma sample obtained 3 hours after oral administration of DM and omeprazole. They found that the [DM]/[DX] ratio was associated with ENDOss (r = −0.52), but the [omeprazole]/[omeprazole sulfone] ratio was not.

Incidentally, our phenotyping method based on the urinary log(DM/DX) ratio was validated by comparison with the reference debrisoquine test [10](#prp2646-bib-0010) and found to correlate with the partial metabolic clearance of DM to DX.[11](#prp2646-bib-0011) A good correlation was also demonstrated in our population between urinary log(DM/DX) ratio and plasma log(ND‐TAM/ENDO) ratio, which is another measure of *CYP2D6* activity (Figure [5](#prp2646-fig-0005)). Notably, Saladores et al [5](#prp2646-bib-0005) reported that the ND‐TAM/ENDO metabolic ratio correlated significantly with the RFS hazard ratio (HR) on multivariable Cox's regression.

Efforts to predict ENDOss may be justified to the extent that they can forecast treatment outcomes. Conflicting data are available for now. Madlensky et al,[4](#prp2646-bib-0004) and Saladores et al [5](#prp2646-bib-0005) documented better outcomes when ENDOss plasma concentrations exceeded 14‐16 nM (5.2‐5.97 ng ml‐1). It should be noted that both studies included women with early‐stage breast cancer and Saladores's study only considered premenopausal patients. Another study on a small sample (n = 86) with a long median follow‐up (13.8 years) found long‐term overall survival worse for patients with ENDOss concentrations <9 nM or 4OH‐TAM <3.26 nM.[29](#prp2646-bib-0029) The CYPTAM study showed that neither *CYP2D6* genotypes nor ENDOss levels were associated with RFS in 667 women taking TAM (20 mg q.d.) [30](#prp2646-bib-0030) for a median of 2.5 years (median follow‐up 6.4 years), then shifted to an aromatase inhibitor in 66% of cases. Given the long time to recurrence of breast cancer, the short duration of therapy and follow‐up may explain the negative results of this study. In addition, the HR used to estimate the sample size was probably too high as well (HR = 2), compared with the HR of 1.4 found significant in Madlensky's study. Two other studies found no association between ENDOss and clinical endpoints.[31](#prp2646-bib-0031), [32](#prp2646-bib-0032) All patients had advanced breast cancer, however, and most of them were post‐menopausal. The conflicting results may therefore be due to differences in patient selection (cancer stage, menopausal status) and study design (duration of therapy and follow‐up, sample size calculation). In agreement with this hypothesis, Margolin et al [33](#prp2646-bib-0033) reported that *CYP2D6* genotypes with low activity scores (presumably associated with low ENDO levels) had negative outcomes mainly in pre‐menopausal women, and suggested that higher estrogen levels require a more efficient TAM bio‐activation. In the same vein, advanced cancer stages may be less responsive to anti‐estrogen therapy, thus yielding a flat correlation between ENDOss and clinical outcomes.

While waiting for more conclusive results, it has been suggested that TAM dose be adjusted according to ENDOss rather than *CYP2D6* geno‐phenotype.[29](#prp2646-bib-0029), [34](#prp2646-bib-0034), [35](#prp2646-bib-0035) In line with this view, Fox et al [36](#prp2646-bib-0036) increased the TAM doses in 68 of 122 patients based on their individual ENDOss levels. Following this dose escalation, the percentage of patients with ENDOss >15 nM rose from 76% to 96% and the percentage of those with ENDOss >30 nM from 34% to 76%. Two other studies showed that TAM dose escalation from 20 mg to 40 mg q.d. did not increase the frequency or severity of side effects.[37](#prp2646-bib-0037), [38](#prp2646-bib-0038)

This approach seems appealing, but means that TAM dosage can only be adjusted after 2‐3 months of therapy, when a steady state has been reached.

An alternative strategy—suggested by Hertz and Rae,[39](#prp2646-bib-0039) and supported by our results—could reduce the time it takes to optimize the TAM dosage. Before starting treatment, we can compute patients’ log(DM/DX) or CYP2D6 phenotype (depending on the method available at the point of care), and predict their log‐ENDOss using Equation [2](#prp2646-disp-0001) or Equation [3](#prp2646-disp-0003), then obtain their ENDOss by calculating the anti‐logarithm.

Whatever the method used, patients whose predicted ENDOss concentration is lower than the threshold of 5.97 ng mL‐1 should start therapy with doses >20 mg. Assuming a linear dose‐concentration relationship, the dose should be increased by the threshold‐to‐predicted‐concentration ratio and rounded up to 30 or 40 mg. Should higher (off‐label) doses be required, aromatase inhibitors may be an alternative option, since little is known about the long‐term safety of higher doses of TAM.[36](#prp2646-bib-0036)

Adequately‐powered prospective trials are obviously needed to test this strategy.

Our study has some limitations. First, we assumed that all patients adhered to their TAM treatment. Though we could not prove as much, it is reasonable to assume a good compliance at the start of the therapy. The long half‐lives of TAM and its metabolites should also guarantee stable ENDO concentrations even if a TAM dose is missed occasionally. Second, we did not genotype all the known CYP2D6 variants, but only those most common in Caucasians, so our results cannot be extended to other ethnicities. Third, ENDOss are reportedly 20% lower in winter than the mean year‐round levels.[40](#prp2646-bib-0040) Our study covered a period of 8 months, so ENDOss may have been influenced by seasonal changes. That said, a post‐hoc analysis of our data (not shown) found no differences in ENDOss measured in January‐March versus July‐September. Fourth, the results of urinary DM testing may be affected by changes in urinary pH and renal function, thus leading to misphenotyping *CYP2D6*. Although we cannot exclude this possibility, such a bias can be minimized by collecting urine over a long period (10 hours), as we did. The log(DM/DX) ratio also correlated strongly with the log(ND‐TAM/ENDO), which more closely reflects ENDO production by *CYP2D6*.

## 5. CONCLUSIONS

Our study found that phenotyping *CYP2D6* activity by means of a urinary DM test is the single best predictor of ENDOss. A model including log(DM/DX), patient's age, and use of *CYP2D6* inhibitors has an acceptable predictive performance, and could be used as an alternative to genotyping tests. Despite some uncertainty regarding the optimal ENDOss, a therapeutic approach that aims at personalizing TAM dosage early on is worth testing in a prospective trial.

## CONFLICT OF INTERESTS

None of the authors have any competing interests to disclose.

## AUTHORS’ CONTRIBUTIONS

MG, FP, NM, and RP contributed to study conception and design. LB, GDR, CF, YM, CB, DDC, APF, ST, EC, AB, CM, and RS contributed to data acquisition. MG, BC, NM, and RP contributed to data analysis and interpretation. MG, BC, and RP contributed to manuscript drafting. MG, FP, BC, CO, NM, and RP contributed to manuscript revision.

## Supporting information

## ACKNOWLEDGEMENTS

This research was funded by grants from the Regione Veneto (Ricerca Finalizzata 2009), LILT ‐ Rovigo (grants 2012‐2020), and the Department of Medicine at the University of Padova (DOR 2017).

The Italian TAM Study Group

The following people contributed to this study:

Silvia Angelini^5^, Susanne Baier^10^, Carmen Barile^4^, Franco Bassan^20^, Emanuela Beda^13^, Laura Bertolaso^4^, Andrea Bonetti^16^, Lucia Borgato^1^, Antonella Brunello^1^, Maria Paola Cecchini^16^, Barbara Corso^21^, Giorgio Crepaldi^4^, Elisabetta Cretella^10^, Donatella Da Corte Z.^9^, Cristina Dealis^10^, Giovanni DeRosa^22^, Emilia Durante^16^, Cristina Falci^2^, Adolfo Favaretto^5^, Elena Fiorio^7^, Anna Paola Fraccon^6^, Lara Furini^12^, Alice Giacobino^3^, Tommaso Giarratano^2^, Carlo Alberto Giorgi^2^, Filippo Greco^16^, Milena Gusella^4^, Alessandro Inno^4^, Micaela Lenotti^7^, Maria Rita Lusso^10^, Marta Mandarà^19^, Daniela Menon^4^, Federica Merlin^19^, Marta Mion^13^, Nadia Minicuci^21^, Yasmina Modena^4^, Caterina Modonesi^14^, Linda Nicolardi^20^, Cristina Oliani^4^, Roberto Padrini^22^, Dario Palleschi^5^, Felice Pasini^6^, Maria Cristina Pegoraro^17^, Elena Pellegrinelli^4^, Elisa Perfetti^3^, Elisa Pezzolo^4^, Paolo Piacentini^16^, Valentina Polo^5^, Concetta Raiti^18^, Elvira Rampello^19^, Ilaria Rocco^21^, Romana Segati^15^, Elena Seles^3^, Mariella Sorarù^13^, Laura Stievano^4^, Ottaviano Tomassi^18^, Silvia Toso^8^, Francesca Vastola^14^, Emanuela Vattemi^10^, Alberto Zaniboni^11^, Marta Zaninelli^12^

^1^Oncology Unit 1, Istituto Oncologico Veneto (IOV), IRCCS Padova, Italy; ^2^Oncology Unit 2, Istituto Oncologico Veneto (IOV), IRCCS Padova, Italy; ^3^Oncology Unit, Ospedale degli Infermi, Biella, Italy; ^4^Oncology Unit, Ospedale di Rovigo, AULSS5 Polesana, Italy; ^5^Oncology Unit, Ospedale di Treviso, AULSS2 Marca Trevigiana, Italy; ^6^Oncology Unit, Casa di Cura Pederzoli, Peschiera del Garda, Italy; ^7^Oncology Unit, AOUI Verona, Italy; ^8^Oncology Unit, Ospedale di Adria, AULSS5 Polesana, Italy; ^9^Oncology Unit, Ospedale di Belluno, AULSS1 Dolomiti, Italy; ^10^Oncology Unit, Ospedale di Bolzano, Az. Sanitaria dell'Alto Adige, Italy; ^11^Oncology Unit, Fondazione Poliambulanza, Brescia, Italy; ^12^Oncology Unit Ospedale di Villafranca, AULSS9 Scaligera, Italy; ^13^Oncology Unit, Ospedale di Camposampiero e Cittadella, AULSS6 Euganea, Italy; ^14^Oncology Unit, Ospedali Riuniti Padova Sud, AULSS6 Euganea, Italy; ^15^Oncology Unit, Ospedale di Feltre, AULSS1 Dolomiti, Italy; ^16^Oncology Unit, Ospedale di Legnago, AULSS 9 Scaligera, Italy; ^17^Oncology Unit, Ospedale di Montecchio Maggiore, AULSS8 Berica, Italy; ^18^Oncology Unit, Ospedale di S Dona', AULSS4 Veneto Orientale, Italy; ^19^Oncology Unit, Ospedale di San Bonifacio, AULSS9 Scaligera, Italy; ^20^Oncology Unit, Ospedale Altovicentino, AULSS 7 Pedemontana, Italy; ^21^National Research Council, Neuroscience Institute, Padova, Italy; ^22^Clinical Pharmacology Unit of the Department of Medicine (DIMED), University of Padova, Padova, Italy

Gusella M, Pasini F, Corso B, et al; Italian TAM Group . Predicting steady-state endoxifen plasma concentrations in breast cancer patients by *CYP2D6* genotyping or phenotyping. Which approach is more reliable? Pharmacol Res Perspect. 2020;8:e00646 10.1002/prp2.646

## DATA AVAILABILITY STATEMENT

The data analyzed in this study are available from the corresponding author on reasonable request.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data analyzed in this study are available from the corresponding author on reasonable request.

### Supplementary Materials

### Data Availability Statement

The data analyzed in this study are available from the corresponding author on reasonable request.

## References

1. Johnson MD, Zuo H, Lee K‐H, et al. Pharmacological characterization of 4‐hydroxy‐N‐desmethyl tamoxifen, a novel active metabolite of tamoxifen. Breast Cancer Res Treat. 2004;85:151‐159.  [DOI](https://doi.org/10.1023/B:BREA.0000025406.31193.e8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15111773/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=Pharmacological%20characterization%20of%204%E2%80%90hydroxy%E2%80%90N%E2%80%90desmethyl%20tamoxifen,%20a%20novel%20active%20metabolite%20of%20tamoxifen&author=MD%20Johnson&author=H%20Zuo&author=K%E2%80%90H%20Lee&volume=85&publication_year=2004&pages=151-159&pmid=15111773&doi=10.1023/B:BREA.0000025406.31193.e8&)

2. Lim YC, Desta Z, Flockhart DA, Skaar TC. Endoxifen (4‐hydroxy‐N‐desmethyl‐tamoxifen) has anti‐estrogenic effects in breast cancer cells with potency similar to 4‐hydroxy‐tamoxifen. Cancer Chemother Pharmacol. 2005;55:471‐478.  [DOI](https://doi.org/10.1007/s00280-004-0926-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15685451/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Endoxifen%20(4%E2%80%90hydroxy%E2%80%90N%E2%80%90desmethyl%E2%80%90tamoxifen)%20has%20anti%E2%80%90estrogenic%20effects%20in%20breast%20cancer%20cells%20with%20potency%20similar%20to%204%E2%80%90hydroxy%E2%80%90tamoxifen&author=YC%20Lim&author=Z%20Desta&author=DA%20Flockhart&author=TC%20Skaar&volume=55&publication_year=2005&pages=471-478&pmid=15685451&doi=10.1007/s00280-004-0926-7&)

3. de Vries Schultink AHM, Alexi X, van Werkhoven E, et al. An Antiestrogenic Activity Score for tamoxifen and its metabolites is associated with breast cancer outcome. Breast Cancer Res Treat. 2017;161:567‐574.  [DOI](https://doi.org/10.1007/s10549-016-4083-6) | [PMC free article](/articles/PMC5812686/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28005246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=An%20Antiestrogenic%20Activity%20Score%20for%20tamoxifen%20and%20its%20metabolites%20is%20associated%20with%20breast%20cancer%20outcome&author=AHM%20de%20Vries%20Schultink&author=X%20Alexi&author=E%20van%20Werkhoven&volume=161&publication_year=2017&pages=567-574&pmid=28005246&doi=10.1007/s10549-016-4083-6&)

4. Madlensky L, Natarajan L, Tchu S, et al. Tamoxifen metabolite concentrations, CYP2D6 genotype, and breast cancer outcomes. Clin Pharmacol Ther. 2011;89:718‐725.  [DOI](https://doi.org/10.1038/clpt.2011.32) | [PMC free article](/articles/PMC3081375/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21430657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Tamoxifen%20metabolite%20concentrations,%20CYP2D6%20genotype,%20and%20breast%20cancer%20outcomes&author=L%20Madlensky&author=L%20Natarajan&author=S%20Tchu&volume=89&publication_year=2011&pages=718-725&pmid=21430657&doi=10.1038/clpt.2011.32&)

5. Saladores P, Mürdter T, Eccles D, et al. Tamoxifen metabolism predicts drug concentrations and outcome in premenopausal patients with early breast cancer. Pharmacogenomics J. 2015;15:84‐94.  [DOI](https://doi.org/10.1038/tpj.2014.34) | [PMC free article](/articles/PMC4308646/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25091503/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Tamoxifen%20metabolism%20predicts%20drug%20concentrations%20and%20outcome%20in%20premenopausal%20patients%20with%20early%20breast%20cancer&author=P%20Saladores&author=T%20M%C3%BCrdter&author=D%20Eccles&volume=15&publication_year=2015&pages=84-94&pmid=25091503&doi=10.1038/tpj.2014.34&)

6. de Vries Schultink AHM, Huitema ADR, Beijnen JH. Therapeutic Drug Monitoring of endoxifen as an alternative for CYP2D6 genotyping in individualizing tamoxifen therapy. Breast. 2018;42:38‐40.  [DOI](https://doi.org/10.1016/j.breast.2018.08.100) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30153552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast&title=Therapeutic%20Drug%20Monitoring%20of%20endoxifen%20as%20an%20alternative%20for%20CYP2D6%20genotyping%20in%20individualizing%20tamoxifen%20therapy&author=AHM%20de%20Vries%20Schultink&author=ADR%20Huitema&author=JH%20Beijnen&volume=42&publication_year=2018&pages=38-40&pmid=30153552&doi=10.1016/j.breast.2018.08.100&)

7. Binkhorst L, Mathijssen RH, Jager A, van Gelder T. Individualization of tamoxifen therapy: much more than just CYP2D6 genotyping. Cancer Treat Rev. 2015;41:289‐299.  [DOI](https://doi.org/10.1016/j.ctrv.2015.01.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25618289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Treat%20Rev&title=Individualization%20of%20tamoxifen%20therapy:%20much%20more%20than%20just%20CYP2D6%20genotyping&author=L%20Binkhorst&author=RH%20Mathijssen&author=A%20Jager&author=T%20van%20Gelder&volume=41&publication_year=2015&pages=289-299&pmid=25618289&doi=10.1016/j.ctrv.2015.01.002&)

8. Kiyotani K, Mushiroda T, Nakamura Y, Zembutsu H. Pharmacogenomics of tamoxifen: roles of drug metabolizing enzymes and transporters. Drug Metab Pharmacokinet. 2012;27:122‐131.  [DOI](https://doi.org/10.2133/dmpk.dmpk-11-rv-084) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22041137/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Pharmacogenomics%20of%20tamoxifen:%20roles%20of%20drug%20metabolizing%20enzymes%20and%20transporters&author=K%20Kiyotani&author=T%20Mushiroda&author=Y%20Nakamura&author=H%20Zembutsu&volume=27&publication_year=2012&pages=122-131&pmid=22041137&doi=10.2133/dmpk.dmpk-11-rv-084&)

9. Goetz MP, Sangkuhl K, Guchelaar HJ, et al. Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for CYP2D6 and tamoxifen therapy. Clin Pharmacol Ther. 2018;103:770‐777.  [DOI](https://doi.org/10.1002/cpt.1007) | [PMC free article](/articles/PMC5931215/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29385237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guideline%20for%20CYP2D6%20and%20tamoxifen%20therapy&author=MP%20Goetz&author=K%20Sangkuhl&author=HJ%20Guchelaar&volume=103&publication_year=2018&pages=770-777&pmid=29385237&doi=10.1002/cpt.1007&)

10. Schmid B, Bircher J, Preisig R, Küpfer A. Polymorphic dextromethorphan metabolism: co‐segregation of oxidative O‐demethylation with debrisoquin hydroxylation. Clin Pharmacol Ther. 1985;38:618‐624.  [DOI](https://doi.org/10.1038/clpt.1985.235) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4064464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Polymorphic%20dextromethorphan%20metabolism:%20co%E2%80%90segregation%20of%20oxidative%20O%E2%80%90demethylation%20with%20debrisoquin%20hydroxylation&author=B%20Schmid&author=J%20Bircher&author=R%20Preisig&author=A%20K%C3%BCpfer&volume=38&publication_year=1985&pages=618-624&pmid=4064464&doi=10.1038/clpt.1985.235&)

11. Capon DA, Bochner F, Kerry N, Mikus G, Danz C, Somogyi AA. The influence of CYP2D6 polymorphism and quinidine on the disposition and antitussive effect of dextromethorphan in humans. Clin Pharmacol Ther. 1996;60:295‐307.  [DOI](https://doi.org/10.1016/S0009-9236(96)90056-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8841152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20influence%20of%20CYP2D6%20polymorphism%20and%20quinidine%20on%20the%20disposition%20and%20antitussive%20effect%20of%20dextromethorphan%20in%20humans&author=DA%20Capon&author=F%20Bochner&author=N%20Kerry&author=G%20Mikus&author=C%20Danz&volume=60&publication_year=1996&pages=295-307&pmid=8841152&doi=10.1016/S0009-9236(96)90056-9&)

12. Shah RR, Smith RL. Br J Clin Pharmacol. Addressing phenoconversion: the Achilles' heel of personalized medicine. 2015;79:222‐240.  [DOI](https://doi.org/10.1111/bcp.12441) | [PMC free article](/articles/PMC4309629/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24913012/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addressing%20phenoconversion:%20the%20Achilles'%20heel%20of%20personalized%20medicine&title=Br%20J%20Clin%20Pharmacol&author=RR%20Shah&author=RL%20Smith&volume=79&publication_year=2015&pages=222-240&pmid=24913012&doi=10.1111/bcp.12441&)

13. Lee KH, Ward BA, Desta Z, Flockhart DA, Jones DR. Quantification of tamoxifen and three metabolites in plasma by high‐performance liquid chromatography with fluorescence detection: application to a clinical trial. J Chromatogr B Analyt Technol Biomed Life Sci. 2003;791:245‐253.  [DOI](https://doi.org/10.1016/s1570-0232(03)00218-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12798184/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Chromatogr%20B%20Analyt%20Technol%20Biomed%20Life%20Sci&title=Quantification%20of%20tamoxifen%20and%20three%20metabolites%20in%20plasma%20by%20high%E2%80%90performance%20liquid%20chromatography%20with%20fluorescence%20detection:%20application%20to%20a%20clinical%20trial&author=KH%20Lee&author=BA%20Ward&author=Z%20Desta&author=DA%20Flockhart&author=DR%20Jones&volume=791&publication_year=2003&pages=245-253&pmid=12798184&doi=10.1016/s1570-0232(03)00218-6&)

14. Brooks JD, Comen EA, Reiner AS, et al. CYP2D6 phenotype, tamoxifen, and risk of contralateral breast cancer in the WECARE Study. Breast Cancer Res. 2018;20:149.  [DOI](https://doi.org/10.1186/s13058-018-1083-y) | [PMC free article](/articles/PMC6288916/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30526633/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=CYP2D6%20phenotype,%20tamoxifen,%20and%20risk%20of%20contralateral%20breast%20cancer%20in%20the%20WECARE%20Study&author=JD%20Brooks&author=EA%20Comen&author=AS%20Reiner&volume=20&publication_year=2018&pages=149&pmid=30526633&doi=10.1186/s13058-018-1083-y&)

15. Saghafi F, Salehifar E, Janbabai G, et al. CYP2D6*3 (A2549del), *4 (G1846A), *10 (C100T) and *17 (C1023T) genetic polymorphisms in Iranian breast cancer patients treated with adjuvant tamoxifen. Biomed Rep. 2018;9:446‐452.  [DOI](https://doi.org/10.3892/br.2018.1145) | [PMC free article](/articles/PMC6176100/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30345040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Rep&title=CYP2D6*3%20(A2549del),%20*4%20(G1846A),%20*10%20(C100T)%20and%20*17%20(C1023T)%20genetic%20polymorphisms%20in%20Iranian%20breast%20cancer%20patients%20treated%20with%20adjuvant%20tamoxifen&author=F%20Saghafi&author=E%20Salehifar&author=G%20Janbabai&volume=9&publication_year=2018&pages=446-452&pmid=30345040&doi=10.3892/br.2018.1145&)

16. Levo A, Koski A, Ojanperä I, Vuori E, Sajantila A. Post‐mortem SNP analysis of CYP2D6 gene reveals correlation between genotype and opioid drug (tramadol) metabolite ratios in blood. Forensic Sci Int. 2003;135:9‐15.  [DOI](https://doi.org/10.1016/s0379-0738(03)00159-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12893130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Forensic%20Sci%20Int&title=Post%E2%80%90mortem%20SNP%20analysis%20of%20CYP2D6%20gene%20reveals%20correlation%20between%20genotype%20and%20opioid%20drug%20(tramadol)%20metabolite%20ratios%20in%20blood&author=A%20Levo&author=A%20Koski&author=I%20Ojanper%C3%A4&author=E%20Vuori&author=A%20Sajantila&volume=135&publication_year=2003&pages=9-15&pmid=12893130&doi=10.1016/s0379-0738(03)00159-2&)

17. Kobylecki CJ, Jakobsen KD, Hansen T, Jakobsen IV, Rasmussen HB, Werge T. CYP2D6 genotype predicts antipsychotic side effects in schizophrenia inpatients: a retrospective matched case‐control study. Neuropsychobiology. 2009;59:222‐226.  [DOI](https://doi.org/10.1159/000223734) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19521114/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychobiology&title=CYP2D6%20genotype%20predicts%20antipsychotic%20side%20effects%20in%20schizophrenia%20inpatients:%20a%20retrospective%20matched%20case%E2%80%90control%20study&author=CJ%20Kobylecki&author=KD%20Jakobsen&author=T%20Hansen&author=IV%20Jakobsen&author=HB%20Rasmussen&volume=59&publication_year=2009&pages=222-226&pmid=19521114&doi=10.1159/000223734&)

18. Al‐Dosari MS, Al‐Jenoobi FI, Alkharfy KM, et al. High prevalence of CYP2D6*41 (G2988A) allele in Saudi Arabians. Environ Toxicol Pharmacol. 2013;36:1063‐1067.  [DOI](https://doi.org/10.1016/j.etap.2013.09.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24121619/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ%20Toxicol%20Pharmacol&title=High%20prevalence%20of%20CYP2D6*41%20(G2988A)%20allele%20in%20Saudi%20Arabians&author=MS%20Al%E2%80%90Dosari&author=FI%20Al%E2%80%90Jenoobi&author=KM%20Alkharfy&volume=36&publication_year=2013&pages=1063-1067&pmid=24121619&doi=10.1016/j.etap.2013.09.008&)

19. http://primer3.ut.ee. Accessed October 10, 2019  [http://primer3.ut.ee](http://primer3.ut.ee)

20. Sistonen J, Fuselli S, Levo A, Sajantila A.CYP2D6 genotyping by a multiplex primer extension reaction.  [DOI](https://doi.org/10.1373/clinchem.2004.046466) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15905314/)

21. Caudle KE, Sangkuhl K, Whirl‐Carrillo M, et al. Standardizing CYP2D6 genotype to phenotype translation: consensus recommendations from the Clinical Pharmacogenetics Implementation Consortium and Dutch Pharmacogenetics Working Group. Clin Transl Sci. 2020;13:116‐124.  [DOI](https://doi.org/10.1111/cts.12692) | [PMC free article](/articles/PMC6951851/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31647186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Standardizing%20CYP2D6%20genotype%20to%20phenotype%20translation:%20consensus%20recommendations%20from%20the%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20and%20Dutch%20Pharmacogenetics%20Working%20Group&author=KE%20Caudle&author=K%20Sangkuhl&author=M%20Whirl%E2%80%90Carrillo&volume=13&publication_year=2020&pages=116-124&pmid=31647186&doi=10.1111/cts.12692&)

22. Flores‐Perez J, Flores‐Perez C, Juarez‐Olguin H, Lares‐Asseff I, Sosa‐Marcias M. Determination of dextromethorphan and dextrorphan in human urine by high performance liquid chromatography for pharmacogenetic investigations. Chromatographia. 2004;59:481‐485.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chromatographia&title=Determination%20of%20dextromethorphan%20and%20dextrorphan%20in%20human%20urine%20by%20high%20performance%20liquid%20chromatography%20for%20pharmacogenetic%20investigations&author=J%20Flores%E2%80%90Perez&author=C%20Flores%E2%80%90Perez&author=H%20Juarez%E2%80%90Olguin&author=I%20Lares%E2%80%90Asseff&author=M%20Sosa%E2%80%90Marcias&volume=59&publication_year=2004&pages=481-485&)

23. Schroth W, Winter S, Mürdter T, et al. Prediction of endoxifen metabolism by CYP2D6 genotype in breast cancer patients treated with tamoxifen. Front Pharmacol. 2017;8:article 582.  [DOI](https://doi.org/10.3389/fphar.2017.00582) | [PMC free article](/articles/PMC5609540/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28955222/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Pharmacol&title=Prediction%20of%20endoxifen%20metabolism%20by%20CYP2D6%20genotype%20in%20breast%20cancer%20patients%20treated%20with%20tamoxifen&author=W%20Schroth&author=S%20Winter&author=T%20M%C3%BCrdter&volume=8&publication_year=2017&pmid=28955222&doi=10.3389/fphar.2017.00582&)

24. Hwang GS, Bhat R, Crutchley RD, Trivedi MV. Impact of CYP2D6 polymorphisms on endoxifen concentrations and breast cancer outcomes. Pharmacogenomics J. 2018;18:201‐208.  [DOI](https://doi.org/10.1038/tpj.2017.36) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28762370/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Impact%20of%20CYP2D6%20polymorphisms%20on%20endoxifen%20concentrations%20and%20breast%20cancer%20outcomes&author=GS%20Hwang&author=R%20Bhat&author=RD%20Crutchley&author=MV%20Trivedi&volume=18&publication_year=2018&pages=201-208&pmid=28762370&doi=10.1038/tpj.2017.36&)

25. de Graan A‐J, Teunissen SF, de Vos FYFL, et al. Dextromethorphan as a phenotyping test to predict endoxifen exposure in patients on tamoxifen treatment. J Clin Oncol. 2011;29:3240‐3246.  [DOI](https://doi.org/10.1200/JCO.2010.32.9839) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21768449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Dextromethorphan%20as%20a%20phenotyping%20test%20to%20predict%20endoxifen%20exposure%20in%20patients%20on%20tamoxifen%20treatment&author=A%E2%80%90J%20de%20Graan&author=SF%20Teunissen&author=FYFL%20de%20Vos&volume=29&publication_year=2011&pages=3240-3246&pmid=21768449&doi=10.1200/JCO.2010.32.9839&)

26. Opdam FL, Dezentje VO, den Hartigh J, et al. The use of the 13C‐dextromethorphan breath test for phenotyping CYP2D6 in breast cancer patients using tamoxifen: association with CYP2D6 genotype and serum endoxifen levels. Cancer Chemother Pharmacol. 2013;71:593‐601.  [DOI](https://doi.org/10.1007/s00280-012-2034-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23228987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=The%20use%20of%20the%2013C%E2%80%90dextromethorphan%20breath%20test%20for%20phenotyping%20CYP2D6%20in%20breast%20cancer%20patients%20using%20tamoxifen:%20association%20with%20CYP2D6%20genotype%20and%20serum%20endoxifen%20levels&author=FL%20Opdam&author=VO%20Dezentje&author=J%20den%20Hartigh&volume=71&publication_year=2013&pages=593-601&pmid=23228987&doi=10.1007/s00280-012-2034-4&)

27. Safgren SL, Suman VJ, Kosel ML, et al. Evaluation of CYP2D6 enzyme activity using a 13C‐dextromethorphan breath test in women receiving adjuvant tamoxifen. Pharmacogenet Genomics. 2015;25:157‐163.  [DOI](https://doi.org/10.1097/FPC.0000000000000121) | [PMC free article](/articles/PMC4356640/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25714002/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Evaluation%20of%20CYP2D6%20enzyme%20activity%20using%20a%2013C%E2%80%90dextromethorphan%20breath%20test%20in%20women%20receiving%20adjuvant%20tamoxifen&author=SL%20Safgren&author=VJ%20Suman&author=ML%20Kosel&volume=25&publication_year=2015&pages=157-163&pmid=25714002&doi=10.1097/FPC.0000000000000121&)

28. Antunes MV, da Fontoura Timm TA, de Oliveira V, et al. Influence of CYP2D6 and CYP3A4 phenotypes, drug Interactions, and vitamin D status on tamoxifen biotransformation. Ther Drug Monit. 2015;37:733‐744.  [DOI](https://doi.org/10.1097/FTD.0000000000000212) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25853922/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Influence%20of%20CYP2D6%20and%20CYP3A4%20phenotypes,%20drug%20Interactions,%20and%20vitamin%20D%20status%20on%20tamoxifen%20biotransformation&author=MV%20Antunes&author=TA%20da%20Fontoura%20Timm&author=V%20de%20Oliveira&volume=37&publication_year=2015&pages=733-744&pmid=25853922&doi=10.1097/FTD.0000000000000212&)

29. Helland T, Henne N, Bifulco E, et al. Serum concentrations of active tamoxifen metabolites predict long‐term survival in adjuvantly treated breast cancer patients. Breast Cancer Res. 2017;19:125.  [DOI](https://doi.org/10.1186/s13058-017-0916-4) | [PMC free article](/articles/PMC5706168/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29183390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=Serum%20concentrations%20of%20active%20tamoxifen%20metabolites%20predict%20long%E2%80%90term%20survival%20in%20adjuvantly%20treated%20breast%20cancer%20patients&author=T%20Helland&author=N%20Henne&author=E%20Bifulco&volume=19&publication_year=2017&pages=125&pmid=29183390&doi=10.1186/s13058-017-0916-4&)

30. Sanchez‐Spitman A, Dezentjé V, Swen J, et al. Tamoxifen pharmacogenetics and metabolism: results from the prospective CYPTAM study. J Clin Oncol. 2019;37:636‐646.  [DOI](https://doi.org/10.1200/JCO.18.00307) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30676859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Tamoxifen%20pharmacogenetics%20and%20metabolism:%20results%20from%20the%20prospective%20CYPTAM%20study&author=A%20Sanchez%E2%80%90Spitman&author=V%20Dezentj%C3%A9&author=J%20Swen&volume=37&publication_year=2019&pages=636-646&pmid=30676859&doi=10.1200/JCO.18.00307&)

31. Neven P, Jongen L, Lintermans A, et al. Tamoxifen metabolism and efficacy in breast cancer: a prospective multicenter trial. Clin Cancer Res. 2018;24:2312‐2318.  [DOI](https://doi.org/10.1158/1078-0432.CCR-17-3028) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29459457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Tamoxifen%20metabolism%20and%20efficacy%20in%20breast%20cancer:%20a%20prospective%20multicenter%20trial&author=P%20Neven&author=L%20Jongen&author=A%20Lintermans&volume=24&publication_year=2018&pages=2312-2318&pmid=29459457&doi=10.1158/1078-0432.CCR-17-3028&)

32. Tamura K, Imamura CK, Takano T, et al. CYP2D6 genotype‐guided tamoxifen dosing in hormone receptor‐positive metastatic breast cancer (TARGET‐1): a randomized, open‐label, phase ii study. J Clin Oncol. 2020;38:558‐566.  [DOI](https://doi.org/10.1200/JCO.19.01412) | [PMC free article](/articles/PMC7030889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31821071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=CYP2D6%20genotype%E2%80%90guided%20tamoxifen%20dosing%20in%20hormone%20receptor%E2%80%90positive%20metastatic%20breast%20cancer%20(TARGET%E2%80%901):%20a%20randomized,%20open%E2%80%90label,%20phase%20ii%20study&author=K%20Tamura&author=CK%20Imamura&author=T%20Takano&volume=38&publication_year=2020&pages=558-566&pmid=31821071&doi=10.1200/JCO.19.01412&)

33. Margolin S, Lindh JD, Thorén L, et al. CYP2D6 and adjuvant tamoxifen: possible differences of outcome in pre‐ and post‐menopausal patients. Pharmacogenomics. 2013;14:613‐622.  [DOI](https://doi.org/10.2217/pgs.13.47) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23570465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CYP2D6%20and%20adjuvant%20tamoxifen:%20possible%20differences%20of%20outcome%20in%20pre%E2%80%90%20and%20post%E2%80%90menopausal%20patients&author=S%20Margolin&author=JD%20Lindh&author=L%20Thor%C3%A9n&volume=14&publication_year=2013&pages=613-622&pmid=23570465&doi=10.2217/pgs.13.47&)

34. Groenland SL, van Nuland M, Verheijen RB, et al. Therapeutic drug monitoring of oral anti‐hormonal drugs in oncology. Clin Pharmacokinet. 2019;58:299‐308.  [DOI](https://doi.org/10.1007/s40262-018-0683-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29862467/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Therapeutic%20drug%20monitoring%20of%20oral%20anti%E2%80%90hormonal%20drugs%20in%20oncology&author=SL%20Groenland&author=M%20van%20Nuland&author=RB%20Verheijen&volume=58&publication_year=2019&pages=299-308&pmid=29862467&doi=10.1007/s40262-018-0683-0&)

35. Hennig EE, Piatkowska M, Karczmarski J, et al. Limited predictive value of achieving beneficial plasma (Z)‐endoxifen threshold level by CYP2D6 genotyping in tamoxifen‐treated Polish women with breast cancer. BMC Cancer. 2015;15:570.  [DOI](https://doi.org/10.1186/s12885-015-1575-4) | [PMC free article](/articles/PMC4522133/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26232141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Limited%20predictive%20value%20of%20achieving%20beneficial%20plasma%20(Z)%E2%80%90endoxifen%20threshold%20level%20by%20CYP2D6%20genotyping%20in%20tamoxifen%E2%80%90treated%20Polish%20women%20with%20breast%20cancer&author=EE%20Hennig&author=M%20Piatkowska&author=J%20Karczmarski&volume=15&publication_year=2015&pages=570&pmid=26232141&doi=10.1186/s12885-015-1575-4&)

36. Fox P, Balleine RL, Lee C, et al. Dose escalation of tamoxifen in patients with low endoxifen level: evidence for therapeutic drug monitoring‐the TADE Study. Clin Cancer Res. 2016;22:3164‐3171.  [DOI](https://doi.org/10.1158/1078-0432.CCR-15-1470) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26847054/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Dose%20escalation%20of%20tamoxifen%20in%20patients%20with%20low%20endoxifen%20level:%20evidence%20for%20therapeutic%20drug%20monitoring%E2%80%90the%20TADE%20Study&author=P%20Fox&author=RL%20Balleine&author=C%20Lee&volume=22&publication_year=2016&pages=3164-3171&pmid=26847054&doi=10.1158/1078-0432.CCR-15-1470&)

37. Dezentjé VO, Opdam FL, Gelderblom H, et al. CYP2D6 genotype‐ and endoxifen‐guided tamoxifen dose escalation increases endoxifen serum concentrations without increasing side effects. Breast Cancer Res Treat. 2015;153:583‐590.  [DOI](https://doi.org/10.1007/s10549-015-3562-5) | [PMC free article](/articles/PMC4589558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26369533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=CYP2D6%20genotype%E2%80%90%20and%20endoxifen%E2%80%90guided%20tamoxifen%20dose%20escalation%20increases%20endoxifen%20serum%20concentrations%20without%20increasing%20side%20effects&author=VO%20Dezentj%C3%A9&author=FL%20Opdam&author=H%20Gelderblom&volume=153&publication_year=2015&pages=583-590&pmid=26369533&doi=10.1007/s10549-015-3562-5&)

38. Hertz DL, Deal A, Ibrahim JG, et al. Tamoxifen dose escalation in patients with diminished CYP2D6 activity normalizes endoxifen concentrations without increasing toxicity. Oncologist. 2016;21:795‐803.  [DOI](https://doi.org/10.1634/theoncologist.2015-0480) | [PMC free article](/articles/PMC4943390/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27226358/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Tamoxifen%20dose%20escalation%20in%20patients%20with%20diminished%20CYP2D6%20activity%20normalizes%20endoxifen%20concentrations%20without%20increasing%20toxicity&author=DL%20Hertz&author=A%20Deal&author=JG%20Ibrahim&volume=21&publication_year=2016&pages=795-803&pmid=27226358&doi=10.1634/theoncologist.2015-0480&)

39. Hertz DL, Rae JM. Individualized tamoxifen dose escalation: confirmation of feasibility, question of utility. Clin Cancer Res. 2016;22:3121‐3123.  [DOI](https://doi.org/10.1158/1078-0432.CCR-16-0370) | [PMC free article](/articles/PMC4930675/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27012810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Individualized%20tamoxifen%20dose%20escalation:%20confirmation%20of%20feasibility,%20question%20of%20utility&author=DL%20Hertz&author=JM%20Rae&volume=22&publication_year=2016&pages=3121-3123&pmid=27012810&doi=10.1158/1078-0432.CCR-16-0370&)

40. Teft WA, Gong IY, Dingle B, et al. CYP3A4 and seasonal variation in vitamin D status in addition to CYP2D6 contribute to therapeutic endoxifen level during tamoxifen therapy. Breast Cancer Res Treat. 2013;139:95‐105.  [DOI](https://doi.org/10.1007/s10549-013-2511-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23580071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=CYP3A4%20and%20seasonal%20variation%20in%20vitamin%20D%20status%20in%20addition%20to%20CYP2D6%20contribute%20to%20therapeutic%20endoxifen%20level%20during%20tamoxifen%20therapy&author=WA%20Teft&author=IY%20Gong&author=B%20Dingle&volume=139&publication_year=2013&pages=95-105&pmid=23580071&doi=10.1007/s10549-013-2511-4&)
