# A Polymorphism within the Vitamin D Transporter Gene Predicts Outcome in Metastatic Colorectal Cancer Patients Treated with FOLFIRI/Bevacizumab or FOLFIRI/Cetuximab

## Metadata
**Authors:** Martin D Berger, Sebastian Stintzing, Volker Heinemann, Shu Cao, Dongyun Yang, Yu Sunakawa, Satoshi Matsusaka, Yan Ning, Satoshi Okazaki, Yuji Miyamoto, Mitsukuni Suenaga, Marta Schirripa, Diana L Hanna, Shivani Soni, Alberto Puccini, Wu Zhang, Chiara Cremolini, Alfredo Falcone, Fotios Loupakis, Heinz-Josef Lenz
**Journal:** Clinical cancer research : an official journal of the American Association for Cancer Research
**Date:** 2017 Dec 5
**DOI:** [10.1158/1078-0432.CCR-17-1663](https://doi.org/10.1158/1078-0432.CCR-17-1663)
**PMID:** 29208668
**PMCID:** PMC7505162
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7505162/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7505162/pdf/nihms-1623472.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7505162/pdf/nihms-1623472.pdf)

## Abstract

**Purpose:** 
Vitamin D exerts its inhibitory influence on colon cancer growth by inhibiting Wnt signaling and angiogenesis. We hypothesized that SNPs in genes involved in vitamin D transport, metabolism, and signaling are associated with outcome in metastatic colorectal cancer (mCRC) patients treated with first-line FOLFIRI and bevacizumab.

**Experimental Design:** 
522 mCRC patients enrolled in the FIRE-3 (discovery cohort) and TRIBE (validation set) trials treated with FOLFIRI/bevacizumab were included in this study. 278 patients receiving FOLFIRI and cetuximab (FIRE-3) served as a control cohort. Six SNPs in 6 genes (GC, CYP24A1, CYP27B1, VDR, DKK1, CST5) were analyzed.

**Results:** 
In the discovery cohort, AA carriers of the GC rs4588 SNP encoding for the vitamin D–binding protein, and treated with FOLFIRI/bevacizumab had a shorter overall survival (OS) than those harboring any C allele (15.9 vs. 25.1 months) in both univariable (P = 0.001) and multivariable analyses (P = 0.047). This association was confirmed in the validation cohort in multivariable analysis (OS 18.1 vs. 26.2 months, HR, 1.83; P = 0.037). Interestingly, AA carriers in the control set exhibited a longer OS (48.0 vs. 25.2 months, HR, 0.50; P = 0.021). This association was further confirmed in a second validation cohort comprising refractory mCRC patients treated with cetuximab ± irinotecan (PFS 8.7 vs. 3.7 months) in univariable (P = 0.033) and multivariable analyses (P = 0.046).

**Conclusions:** 
GC rs4588 SNP might serve as a predictive marker in mCRC patients treated with FOLFIRI/bevacizumab or FOLFIRI/cetuximab. Whereas AA carriers derive a survival benefit with FOLFIRI/cetuximab, treatment with FOLFIRI/bevacizumab is associated with a worse outcome.

### Purpose

Vitamin D exerts its inhibitory influence on colon cancer growth by inhibiting Wnt signaling and angiogenesis. We hypothesized that SNPs in genes involved in vitamin D transport, metabolism, and signaling are associated with outcome in metastatic colorectal cancer (mCRC) patients treated with first-line FOLFIRI and bevacizumab.

### Experimental Design

522 mCRC patients enrolled in the FIRE-3 (discovery cohort) and TRIBE (validation set) trials treated with FOLFIRI/bevacizumab were included in this study. 278 patients receiving FOLFIRI and cetuximab (FIRE-3) served as a control cohort. Six SNPs in 6 genes (*GC, CYP24A1, CYP27B1, VDR, DKK1, CST5*) were analyzed.

### Results

In the discovery cohort, AA carriers of the *GC* rs4588 SNP encoding for the vitamin D–binding protein, and treated with FOLFIRI/bevacizumab had a shorter overall survival (OS) than those harboring any C allele (15.9 vs. 25.1 months) in both univariable (*P* = 0.001) and multivariable analyses (*P* = 0.047). This association was confirmed in the validation cohort in multivariable analysis (OS 18.1 vs. 26.2 months, HR, 1.83; *P* = 0.037). Interestingly, AA carriers in the control set exhibited a longer OS (48.0 vs. 25.2 months, HR, 0.50; *P* = 0.021). This association was further confirmed in a second validation cohort comprising refractory mCRC patients treated with cetuximab ± irinotecan (PFS 8.7 vs. 3.7 months) in univariable (*P* = 0.033) and multivariable analyses (*P* = 0.046).

### Conclusions

*GC* rs4588 SNP might serve as a predictive marker in mCRC patients treated with FOLFIRI/bevacizumab or FOLFIRI/cetuximab. Whereas AA carriers derive a survival benefit with FOLFIRI/cetuximab, treatment with FOLFIRI/bevacizumab is associated with a worse outcome.

## Introduction

Colorectal cancer is the fourth leading cause of cancer-related death worldwide. With the addition of biologicals to 5-fluorouracil (5-FU)–based treatment regimens, the outcome of metastatic colorectal cancer (mCRC) patients has markedly improved during the last decade ([1](#R1)). However, to further optimize treatment, new effective drugs are highly warranted.

Accumulating evidence points to a critical role of vitamin D not only in maintenance of calcium and phosphate homeostasis, but also in several other biological processes such as cell proliferation, differentiation, and the immune response ([2](#R2)). Interestingly, several studies indicate that vitamin D deficiency might facilitate the development of diabetes mellitus type 2, cardiovascular diseases, autoimmune disorders and cancer ([3](#R3)). A meta-analysis comprising nine studies showed that higher vitamin D intake and blood 25-hydroxyvitamin D (25-OHD) levels are associated with lower colorectal cancer risk ([4](#R4)). Another study demonstrated that increased plasma 25-OHD levels were associated with prolonged overall survival (OS) in colorectal cancer patients with a trend toward a decreased cancer-specific mortality ([5](#R5)). Previous studies have also shown a tight link between higher vitamin D levels and longer cancer-specific survival in patients with prostate, breast, lung cancer, and lymphoma ([6](#R6)–[8](#R8)).

However, the exact underlying mechanism by which vitamin D exerts its antitumor effect remains largely elusive. Alvarez-Diaz and colleagues proposed that the antitumor effect of vitamin D in colon cancer cells is mediated by cystatin D, a cysteine proteinase inhibitor encoded by the *CST5* gene, which inhibits proliferation, migration and antagonizes Wnt signaling ([9](#R9)). Another group reported that in colorectal cancer, vitamin D exerts its antineoplastic activity by increasing the expression of DKK1, a potent inhibitor of the Wnt pathway ([10](#R10)).

Cholecalciferol derived from sunlight exposure or dietary intake binds to vitamin D binding protein (DBP; 80%) or albumin (20%) and is metabolized in the liver to 25-hydroxyvitamin D, which is further converted in the kidney by 25-hydroxyvitamin D1α-hydroxylase (CYP27B1) to its biologically active form, 1,25-dihydroxyvitamin D. The biological functions of 1,25-dihydroxyvitamin D are mediated by the vitamin D receptor (VDR), which translocates as vitamin D-bound VDR to the nucleus and regulates gene expression ([2](#R2)).

The degradation of 25-hydroxyvitamin D and 1,25-dihydroxyvitamin D is initiated by the enzyme, 25-hydroxyvitamin D 24-hydroxylase (CYP24A1; ref. [2](#R2)).

The literature regarding variations within genes encoding for critical proteins involved in vitamin D transport, metabolism, and signaling as well as their associations with outcome in colorectal cancer patients is scarce. A few studies have demonstrated a correlation between SNPs within the *VDR* gene and increased colon cancer risk ([11](#R11)), yet no association with outcome has been observed ([12](#R12)). Interestingly, preclinical studies suggest that vitamin D possesses antiangiogenic properties ([2](#R2), [13](#R13)), by decreasing both VEGF-dependent and -independent angiogenesis ([14](#R14)). With bevacizumab treatment, VEGF-independent escape mechanisms might occur to circumvent the angiogenesis blockade ([15](#R15)). Given the association between the AA genotype of the *GC* rs4588 SNP and lower circulating 25-OHD in several studies ([16](#R16)–[21](#R21)), we hypothesized that AA carriers treated with FOLFIRI and bevacizumab may have a worse outcome compared with C allele carriers, as the inhibitory effect on VEGF- independent angiogenesis might be smaller among AA carriers. Conversely, we hypothesized that AA carriers treated with FOLFIRI and cetuximab might have a better outcome, as low 25-OHD levels have been associated with a more activated adaptive immune response, and cetuximab exerts a more pronounced anticancer effect when adaptive immunity is activated ([22](#R22)–[24](#R24)). Furthermore, given the inhibitory influence of 25-OHD on the adaptive immune response ([22](#R22)), we hypothesized that in AA carriers with expectedly decreased 25-OHD levels, the inhibitory effect on the adaptive immune response might be less compared with C allele carriers, who have constitutively higher 25-OHD levels. In addition to our main SNP of interest, *GC* rs4588, we investigated five SNPs within genes, involved in 25-OHD metabolism, transport or signaling which may have an impact on circulating 25-OHD levels and therefore, outcome.

## Patients and Methods

### Study design and patient population

This study was comprised of 893 mCRC patients. 800 mCRC patients were enrolled in the randomized phase III FIRE-3 and TRIBE trials, and received either first-line FOLFIRI/bevacizumab (discovery and validation cohorts) or FOLFIRI and cetuximab (FIRE-3, control cohort). 93 patients with refractory mCRC enrolled in a prospective cohort (Department of Medical Oncology, Pisa, Italy) and a phase II trial (IMCL-0144), treated with either irinotecan plus cetuximab (Pisa, Italy, 2008–2011) or cetuximab monotherapy (University of Southern California, Norris Comprehensive Cancer Center, Los Angeles, CA, 2002–2003), served as a second validation cohort ([25](#R25), [26](#R26)). Because we expected the opposite effect on outcome between AA carriers of the *GC* rs4588 SNP treated with FOLFIRI/bevacizumab (worse outcome) versus FOLFIRI/cetuximab (better outcome), we arbitrarily chose the FIRE-3 FOLFIRI/bevacizumab group as the discovery and the FIRE-3 FOLFIRI/cetuximab group as the control cohort. In the FIRE-3 FOLFIRI/bevacizumab cohort, bevacizumab was given at a dose of 5 mg/kg biweekly. In the FIRE-3 FOLFIRI/cetuximab group, the first cetuximab infusion was administered at a dose of 400 mg/m^2^, thereafter 250 mg/m^2^ once a week. The FOLFIRI backbone therapy consisted of 180 mg/m^2^ irinotecan, 400 mg/m^2^ leucovorin, 400 mg/m^2^ fluorouracil (5-FU) bolus infusion and 2,400 mg/m^2^ infusion over 46 hours. Each cycle was repeated every 2 weeks until disease progression or intolerable toxic effects occurred ([27](#R27)). Validation cohort 1 included mCRC patients in the randomized phase III TRIBE trial treated with the same regimen (FOLFIRI/bevacizumab) as described above, with the exception of leucovorin, which was given at a dose of 200 mg/m^2^. After 12 cycles, patients in TRIBE underwent a maintenance therapy with 5-FU/bevacizumab until disease progression ([28](#R28)). Validation cohort 2 was comprised of treatment refractory mCRC patients who received 500 mg/m^2^ cetuximab and 180 mg/m^2^ irinotecan on day 1 every 2 weeks, or cetuximab monotherapy (first dose 400 mg/m^2^ followed by 250 mg/m^2^ weekly; refs. [25](#R25), [26](#R26)). This study was approved by the local review boards for each participating site. All patients provided informed consent for the analysis of molecular correlates, which were conducted at the USC / Norris Comprehensive Cancer Center in Los Angeles, CA. Our study was performed according to the reporting recommendations for tumor marker prognostic studies (REMARK; ref. [29](#R29)).

### Candidate polymorphisms

Potentially functional variations within genes involved in vitamin D transport, metabolism and signaling were chosen according to the following criteria: minor allele frequency >10% in Caucasians; potential to alter the function of a gene according to public databases (National Institute of Environmental Health Science SNP Function Prediction, [snpinfo.niehs.nih.gov](http://snpinfo.niehs.nih.gov), Queen’s University FSNP, compbio.cs.queensu.ca, NCBI-Pubmed and dbSNP, [www.ncbi.nlm.nih.gov](http://www.ncbi.nlm.nih.gov/) as well as Genecards, [genecards.org](http://genecards.org)).

### Genotyping

Genomic DNA was extracted from formalin-fixed paraffin-embedded (FFPE) tissue in the discovery and control cohorts, and from blood in both validation cohorts, using the QIAmp DNA Easy Kit (Qiagen). In the FIRE-3 study (discovery and control cohorts), only FFPE tissue was available for analyses, whereas in TRIBE (validation cohort 1) and validation cohort 2 (USA and Italy cetuximab cohort), only blood was available. The assays to extract DNA from both blood and FFPE tissue were performed with the same extraction kit (QIAamp DNA Mini Kit, Qiagen), using the identical standardized workflow according to the manufacturer’s protocol. Briefly, for DNA extraction from FFPE tissue, we transferred the tissue from the glass slide into a tube, added 180 μL of tissue lysis buffer, followed by 40 μL proteinase and incubated the tubes at 56ºC overnight in a heated water bath to dissolve the paraffin from the tissue. For DNA extraction from blood, we pipetted 200 μL blood into a tube and added 40 μL proteinase. All subsequent steps were identical for both DNA extraction from FFPE tissue and from blood according to the manufacturer’s protocol (QIAamp DNA Mini Kit, sample and assay technologies). Because of lack of blood in the discovery and control cohorts (FIRE-3) and FFPE tissue in the validation cohorts (TRIBE and USA/Italy cohorts), we did not perform the analyses in both tissue and blood on the same patient. Six potentially functional SNPs in 6 genes (*GC, CYP24A1, CYP27B1, VDR, DKK1*, and *CST5*) were analyzed by PCR-based direct sequencing. Forward and reverse primers ([Supplementary Table S1](#SD2)) were used for PCR amplification. PCR fragments were then sequenced on an ABI 3100A Capillary Genetic Analyzer (Applied Biosystem) to identify the SNP. The investigator (M.D. Berger) reading the sequence was blinded to the clinical outcome data.

### Statistical analysis

The aim of this study was to identify associations between SNPs within genes involved in vitamin D transport, metabolism, and signaling and clinical outcome in mCRC patients enrolled in two phase III randomized trials, FIRE-3, and TRIBE. The discovery cohort consisted of patients receiving first-line FOLFIRI/bevacizumab within the FIRE-3 trial, whereas patients treated with the same regimen in TRIBE served as a validation set. The control cohort consisted of patients receiving first-line FOLFIRI/cetuximab in FIRE-3, and the second validation cohort was comprised of patients treated with cetuximab ± irinotecan.

The primary endpoint was progression-free survival (PFS), and secondary endpoints were OS and overall tumor response rate (ORR). PFS was defined as the time from randomization until disease progression, death or until last follow-up in patients who were alive and remained free of disease progression. OS was defined as the time from randomization until death. Patients still alive were censored at the last date of follow-up. ORR represented the percentage of patients who achieved either a complete (CR) or a partial (PR) response according to the Response Evaluation Criteria in Solid Tumors (RECIST). Allelic distribution of genetic variants was tested for deviation from Hardy–Weinberg equilibrium (HWE) using the χ^2^ test. Differences between baseline characteristics among the cohorts were compared by using the χ^2^ test. To evaluate the effects of different SNPs on PFS and OS, the log-rank test was used in the univariable analysis and Wald test in the multivariable Cox proportional regression model. The associations between SNPs and tumor response were evaluated using the Fisher’s exact test. Moreover, we used the Fisher’s exact test to assess, whether there is an association of potentially significant and validated SNPs and genomic subtypes such as *BRAF* and *(K)RAS* mutational status and other clinical variables such as location, age and sex. SNPs significantly associated with outcome in the discovery cohort (FIRE-3) were tested in the validation set (TRIBE) and a control cohort (FIRE-3). With 295 patients (251 PFS events) in the FOLFIRI/bevacizumab arm of FIRE-3 (discovery cohort) successfully sequenced, we would have 80% power to detect a minimum hazard ratio (HR) of 1.43 to 1.59 for PFS for a SNP with a minor allele frequency from 0.1 to 0.5, using a two-sided 0.05 level log-rank test. A HR of 1.44 to 1.60 would be detected in the FOLFIRI/cetuximab arm of FIRE-3 (control set, *N* = 278, 241 PFS events), 1.54 to 1.73 in the FOLFIRI/bevacizumab arm of TRIBE (validation set 1, *N* = 227, 173 events), and 1.88 to 2.32 in the second validation cohort treated with cetuximab ± irinotecan (*N* = 93, 83 events) for the same SNP with similar allele frequencies using the same test and power. The adjusting factors for multivariate analyses were sex, age, ECOG performance status, primary tumor site, liver limited metastases, primary tumor resection, previous adjuvant chemotherapy, *RAS* and *BRAF* status in the discovery, control, and combined cohorts and sex, age, ECOG performance status, primary tumor site, primary tumor resection, prior adjuvant chemotherapy and *BRAF* status in the validation cohort. Because of the limited data available for validation cohort 2 (studies were conducted in 2002–2003 (Los Angeles) and 2008–2011 (Italy), when information regarding *BRAF/RAS* mutational status and sidedness was not routinely collected in clinical trials) and the lack of specimen available to conduct additional molecular analyses, we could only include age and sex as adjusting factors for multivariate analysis. All *P* values were from two-sided Wald tests at a 0.05 significance level. All tests were conducted by using the SAS statistical package version 9.4 (SAS Institute).

## Results

Patient characteristics of the four study cohorts (one discovery, one control, and two validation sets) are outlined in [Table 1](#T1). Briefly, 893 patients with mCRC receiving either FOLFIRI/bevacizumab (discovery cohort FIRE-3, *N* = 295 and validation cohort 1 TRIBE, *N* = 227), FOLFIRI/cetuximab (control cohort FIRE-3, *N* = 278) or cetuximab ± irinotecan (validation set 2, *N* = 93) were included in this study. Genotyping was successful in >94% of the cases for each polymorphism. Causes of failure included limited quality or quantity of extracted DNA. All tested SNPs were within HWE. A genotyping quality control showed 99% concordance. The frequency of genotypes of the *GC* rs4588 SNP in our study corresponded well with those observed in the general population according to the Ensembl database ([https://www.ensembl.org/](https://www.ensembl.org/)). In the discovery cohort (FIRE-3 FOLFIRI/bevacizumab), the *GC* rs4588 SNP was significantly associated with OS. Patients carrying the AA genotype had a shorter OS (15.9 vs. 25.1 months) compared to those harboring any C allele in both univariable [HR, 2.19; 95% confidence interval (CI), 1.32–3.64; *P* = 0.001] and multivariable analyses (HR, 1.72; 95% CI, 1.01–2.92; *P* = 0.047; [Table 2](#T2), [Fig. 1A](#F1)). In the validation cohort (TRIBE FOLFIRI/bevacizumab), no statistically significant difference in OS was observed between AA carriers and patients harboring any C allele of the *GC* rs4588 SNP in univariable analysis, as shown in the Kaplan–Meier curve in [Fig. 1B](#F1) and outlined in [Table 2](#T2) (18.1 vs. 26.2 months, HR, 1.32; 95% CI, 0.76–2.29; *P* = 0.31). However, after adjusting for sex, age, ECOG performance status, primary tumor site, primary tumor resection, *BRAF* status, and previous adjuvant chemotherapy, the association between the *GC* rs4588 SNP and OS was statistically significant in multivariate analysis (HR, 1.83; 95% CI, 1.04–3.23; *P* = 0.037; [Table 2](#T2)). Because of the small number of patients harboring the AA genotype, we combined both FOLFIRI/bevacizumab cohorts. As expected, the association between the *GC* rs4588 SNP and OS was more evident. AA genotype carriers still exhibited a shorter OS in comparison with those having any C allele (16.4 vs. 25.8 months) in univariable (HR, 1.63; 95% CI, 1.12–2.35; *P* = 0.009) and multivariable analyses (HR, 1.59; 95% CI, 1.08–2.33; *P* = 0.019; [Table 2](#T2), [Fig. 2](#F2)). Conversely, mCRC patients with the AA genotype receiving FOLFIRI/cetuximab (control cohort, FIRE-3) had a longer OS (48.0 vs. 25.2 months) in comparison with those with any C allele in univariable analysis (HR, 0.50; 95%CI, 0.27–0.91; *P* = 0.021; [Table 2](#T2), [Fig. 1C](#F1)). This favorable outcome observed among AA carriers treated with first-line FOLFIRI and cetuximab was also confirmed in an independent second validation cohort comprising refractory mCRC patients who were treated with cetuximab ± irinotecan (PFS 8.7 vs. 3.7 months), in both univariable (HR, 0.40; 95% CI, 0.16–0.99; *P* = 0.033) and multivariable analyses (HR, 0.39; 95% CI, 0.16–0.98; *P* = 0.046; [Table 2](#T2), [Fig. 1D](#F1)). However, in mCRC patients treated with FOLFIRI and bevacizumab, no difference in PFS could be observed between AA carriers and those harboring any C allele in the discovery, validation and combined cohorts (10.5 vs. 10.1 months, HR, 1.14; 95% CI, 0.71–1.82; *P* = 0.59; 7.5 vs. 9.7 months, HR, 1.27; 95% CI, 0.75–2.16; *P* = 0.37; and 9.3 vs. 10.1 months, HR, 1.19; 95% CI, 0.84–1.70; *P* = 0.32, respectively). Likewise, the PFS of mCRC patients receiving FOLFIRI and cetuximab was similar in AA genotype versus any C allele carriers in the control cohort (10.4 vs. 9.5 months, HR, 0.96; 95% CI, 0.63–1.47; *P* = 0.86; [Table 2](#T2)). Furthermore, the *VDR* rs731236 SNP was associated with OS in mCRC patients of the discovery cohort treated with FOLFIRI/bevacizumab ([Supplementary Table S2](#SD3)). Here, GG genotype carriers had a significantly better OS (28.8 vs. 23.7 months) in multivariable analysis (HR, 0.62; 95% CI, 0.39–0.99; *P* = 0.046). However, this association was not observed in validation cohort 1 ([Supplementary Table S3](#SD1)). No associations between the *GC* rs4588 SNP and *BRAF* and *(K)RAS* status or other clinical variables such as location, age and sex were observed in the discovery, validation or control cohorts ([Supplementary Table S4](#SD4)). There was an insufficient number of patients with right-sided, *RAS* mutant/wild-type or *BRAF* mutant cancers harboring the AA genotype of the *GC* rs4588 SNP in the discovery and validation cohorts. Therefore, we restricted our analysis to the overall population and performed an exploratory subgroup analysis in patients with left-sided cancers, in whom sufficient numbers of AA carriers were present. Here, patients in the discovery group harboring the AA genotype (*N* = 13) had a shorter OS than C allele carriers (*N* = 193), but only in univariate analysis (17.4 vs. 27.4 months, HR, 2.40; 95% CI, 1.28–4.50; *P* = 0.004). PFS was similar in AA genotype and C allele carriers (10.8 vs. 10.7 months, HR, 1.41; 95% CI, 0.76–2.61; *P* = 0.26). Neither PFS nor OS differed between AA and C allele carriers in the validation cohort (data not shown).

### Table 1.

| Treatment | N | FIRE-3FOLFIRI+Bevn = 295 | TRIBEFOLFIRI+Bevn = 227 | FIRE-3FOLFIRI+Cetn = 278 | Italy + USCet ± irinotecann = 93 | Pa |
| --- | --- | --- | --- | --- | --- | --- |
| Sex |   |   |   |   |   | 0.44 |
| Male | 578 | 193 (65%) | 138 (61%) | 188 (68%) | 59 (63%) |   |
| Female | 315 | 102 (35%) | 89 (39%) | 90 (32%) | 34 (37%) |   |
| Age |   |   |   |   |   | <0.001 |
| Median (range) |   | 65 (31–76) | 60 (29–75) | 65 (38–79) | 63 (35–82) |   |
| ≤65 | 516 | 155 (53%) | 162 (71%) | 146 (53%) | 53 (57%) |   |
| >65 | 377 | 140 (47%) | 65 (29%) | 132 (47%) | 40 (43%) |   |
| ECOG performance status |   |   |   |   |   | <0.001 |
| 0 | 491 | 162 (55%) | 188 (83%) | 141 (51%) |   |   |
| 1–2 | 308 | 133 (45%) | 38 (17%) | 137 (49%) |   |   |
| Unspecified | 1 | – | 1 (1%) | – |   |   |
| Tumor site |   |   |   |   |   | 0.073 |
| Right colon | 182 | 75 (25%) | 56 (25%) | 51 (18%) |   |   |
| Left colon or rectum | 588 | 213 (72%) | 156 (69%) | 219 (79%) |   |   |
| Unspecified | 30 | 7 (3%) | 15 (7%) | 8 (3%) |   |   |
| Liver limited metastases |   |   |   |   |   | 0.88 |
| Yes | 259 | 95 (32%) | 71 (31%) | 93 (33%) |   |   |
| No | 541 | 200 (68%) | 156 (69%) | 185 (67%) |   |   |
| Primary tumor resected |   |   |   |   |   | <0.001 |
| Yes | 631 | 255 (86%) | 143 (63%) | 233 (84%) |   |   |
| No | 167 | 40 (14%) | 84 (37%) | 43 (15%) |   |   |
| Unspecified | 2 | – | – | 2 (1%) |   |   |
| Adjuvant therapy |   |   |   |   |   | 0.009 |
| Yes | 145 | 54 (18%) | 28 (12%) | 63 (23%) |   |   |
| No | 653 | 241 (82%) | 199 (88%) | 213 (77%) |   |   |
| Unspecified | 2 | – | – | 2 (1%) |   |   |
| KRAS status |   |   |   |   |   | <0.001 |
| Wildtype | 570 | 248 (84%) | 96 (42%) | 226 (81%) |   |   |
| Mutant | 192 | 47 (16%) | 93 (41%) | 52 (19%) |   |   |
| Unspecified | 38 | – | 38 (17%) | – |   |   |
| RAS status |   |   |   |   |   | <0.001 |
| Wild-type | 432 | 199 (67%) | 55 (24%) | 178 (64%) |   |   |
| Mutant | 288 | 83 (28%) | 116 (51%) | 89 (32%) |   |   |
| Unspecified | 80 | 13 (4%) | 56 (25%) | 11 (4%) |   |   |
| BRAF status |   |   |   |   |   | 0.61 |
| Wild-type | 690 | 261 (88%) | 177 (78%) | 252 (91%) |   |   |
| Mutant | 56 | 25 (8%) | 12 (5%) | 19 (7%) |   |   |
| Unspecified | 54 | 9 (3%) | 38 (17%) | 7 (2%) |   |   |

Table 1 Caption: Baseline characteristics

### Table 2.

|   |   | Tumor response |  |  | Progression-free survival |  |  |  |  | OS |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | N | PR+CR | SDb PD | Pa | Median, months (95% CI) | HR (95% CI)a(i) | Pa | HR (95% CI)a(ii) | Pa | Median, months (95% CI) | HR (95% Cl)a(i) | Pa | HR (95% CI)a(ii) | P valuea |
| Discovery cohort |   |   |   | 0.42 |   |   | 0.48 |   | 0.65 |   |   | 0.004 |   | 0.13 |
| C/C | 149 | 89 (65%) | 48 (35%) |   | 10.3 (9.2–12.2) | 1 (Reference) |   | 1 (Reference) |   | 26.4 (23.0–29.1) | 1 (Reference) |   | 1 (Reference) |   |
| C/A | 117 | 61 (57%) | 46 (43%) |   | 9.8 (8.9–11.9) | 1.16 (0.89–1.50) |   | 1.13 (0.87–1.48) |   | 23.6 (20.6–26.5) | 1.15 (0.86–1.55) |   | 1.05 (0.77–1.43) |   |
| A/A | 21 | 12 (57%) | 9 (43%) |   | 10.5 (6.9–12.0) | 1.21 (0.75–1.96) |   | 1.10 (0.66–1.83) |   | 15.9 (11.5–18.9) | 2.32 (1.37–3.93) |   | 1.76 (1.01–3.05) |   |
|   |   |   |   | 0.70 |   |   | 0.59 |   | 0.91 |   |   | 0.001 |   | 0.047 |
| Any C | 266 | 150 (61%) | 94 (39%) |   | 10.1 (9.2–11.5) | 1 (Reference) |   | 1 (Reference) |   | 25.1 (22.7–27.5) | 1 (Reference) |   | 1 (Reference) |   |
| A/A | 21 | 12 (57%) | 9 (43%) |   | 10.5 (6.9–12.0) | 1.14 (0.71–1.82) |   | 1.03 (0.63–1.69) |   | 15.9 (11.5–18.9) | 2.19 (1.32–3.64) |   | 1.72 (1.01–2.92) |   |
| Validation cohort 1 |   |   |   | 0.20 |   |   | 0.62 |   | 0.16 |   |   | 0.46 |   | 0.052 |
| C/C | 124 | 73 (61%) | 47 (39%) |   | 10.3 (8.8–11.1) | 1 (Reference) |   | 1 (Reference) |   | 26.2 (22.4–32.9) | 1 (Reference) |   | 1 (Reference) |   |
| C/A | 87 | 50 (58%) | 36 (42%) |   | 9.6 (8.8–11.1) | 0.94 (0.68–1.29) |   | 0.90 (0.65–1.25) |   | 26.1 (20.5–30.8) | 1.12 (0.82–1.54) |   | 1.24 (0.89–1.71) |   |
| A/A | 16 | 5 (36%) | 9 (64%) |   | 7.5 (3.6–14.1) | 1.24 (0.71–2.14) |   | 1.66 (0.90–3.05) |   | 18.1 (5.5–36.7) | 1.39 (0.79–2.43) |   | 2.01 (1.12–3.62) |   |
|   |   |   |   | 0.078 |   |   | 0.37 |   | 0.070 |   |   | 0.31 |   | 0.037 |
| Any C | 211 | 123 (60%) | 83 (40%) |   | 9.7 (9.3–10.8) | 1 (Reference) |   | 1 (Reference) |   | 26.2 (22.7–30.8) | 1 (Reference) |   | 1 (Reference) |   |
| A/A | 16 | 5 (36%) | 9 (64%) |   | 7.5 (3.6–14.1) | 1.27 (0.75–2.16) |   | 1.73 (0.96–3.14) |   | 18.1 (5.5–36.7) | 1.32 (0.76–2.29) |   | 1.83 (1.04–3.23) |   |
| Combined cohort |   |   |   | 0.19 |   |   | 0.52 |   | 0.54 |   |   | 0.015 |   | 0.044 |
| C/C | 273 | 162 (63%) | 95 (37%) |   | 10.3 (9.4–11.1) | 1 (Reference) |   | 1 (Reference) |   | 26.3 (23.8–30.2) | 1 (Reference) |   | 1 (Reference) |   |
| C/A | 204 | 111 (58%) | 82 (42%) |   | 9.7 (9.2–10.9) | 1.06 (0.87–1.30) |   | 1.04 (0.84–1.28) |   | 24.8 (21.3–27.3) | 1.14 (0.92–1.42) |   | 1.10 (0.88–1.37) |   |
| A/A | 37 | 17 (49%) | 18 (51%) |   | 9.3 (6.9–11.2) | 1.23 (0.85–1.76) |   | 1.24 (0.85–1.81) |   | 16.4 (14.1–20.1) | 1.72 (1.18–2.52) |   | 1.66 (1.12–2.47) |   |
|   |   |   |   | 0.16 |   |   | 0.32 |   | 0.29 |   |   | 0.009 |   | 0.019 |
| Any C | 477 | 273 (61%) | 177 (39%) |   | 10.1 (9.5–10.8) | 1 (Reference) |   | 1 (Reference) |   | 25.8 (23.6–27.4) | 1 (Reference) |   | 1 (Reference) |   |
| A/A | 37 | 17 (49%) | 18 (51%) |   | 9.3 (6.9–11.2) | 1.19 (0.84–1.70) |   | 1.22 (0.85–1.76) |   | 16.4 (14.1–20.1) | 1.63 (1.12–2.35) |   | 1.59 (1.08–2.33) |   |
| Control cohort |   |   |   | 0.65 |   |   | 0.72 |   | 0.64 |   |   | 0.061 |   | 0.090 |
| C/C | 128 | 74 (66%) | 38 (34%) |   | 9.7 (8.0–10.8) | 1 (Reference) |   | 1 (Reference) |   | 22.6 (19.3–28.7) | 1 (Reference) |   | 1 (Reference) |   |
| C/A | 124 | 66 (63%) | 38 (37%) |   | 9.3 (7.3–10.6) | 1.11 (0.85–1.45) |   | 1.06 (0.80–1.40) |   | 28.8 (23.0–37.1) | 0.92 (0.68–1.26) |   | 0.79 (0.57–1.09) |   |
| A/A | 26 | 19 (73%) | 7 (27%) |   | 10.4 (7.9–13.3) | 1.02 (0.65–1.58) |   | 1.26 (0.79–2.01) |   | 48.0 (23.8–61.6) | 0.48 (0.25–0.90) |   | 0.52 (0.27–1.00) |   |
|   |   |   |   | 0.40 |   |   | 0.86 |   | 0.38 |   |   | 0.021 |   | 0.10 |
| Any C | 252 | 140 (65%) | 76 (35%) |   | 9.5 (8.2–10.3) | 1 (Reference) |   | 1 (Reference) |   | 25.2 (21.7–30.4) | 1 (Reference) |   | 1 (Reference) |   |
| A/A | 26 | 19 (73%) | 7 (27%) |   | 10.4 (7.9–13.3) | 0.96 (0.63–1.47) |   | 1.22 (0.78–1.90) |   | 48.0 (23.8–61.6) | 0.50 (0.27–0.91) |   | 0.60 (0.32–1.11) |   |
| Validation cohort 2 |   |   |   | 0.51 |   |   | 0.082 |   | 0.089 |   |   | 0.49 |   | 0.45 |
| C/C | 48 | 10 (22%) | 36 (78%) |   | 3.4 (2.4–5.3) | 1 (Reference) |   | 1 (Reference) |   | 9.1 (7.2–13.3) | 1 (Reference) |   | 1 (Reference) |   |
| C/A | 38 | 8 (22%) | 29 (78%) |   | 4.1 (2.5–5.6) | 0.86 (0.55–1.36) |   | 0.80 (0.50–1.29) |   | 12.1 (5.5–15.7) | 1.01 (0.60–1.72) |   | 0.96 (0.56–1.63) |   |
| A/A | 7 | 3 (43%) | 4 (57%) |   | 8.7 (3.4–15.6b) | 0.37 (0.15–0.94) |   | 0.36 (0.14–0.92) |   | 18.4b (4.7–18.4b) | 0.51 (0.15–1.66) |   | 0.46 (0.14–1.53) |   |
|   |   |   |   | 0.35 |   |   | 0.033 |   | 0.046 |   |   | 0.23 |   | 0.21 |
| Any C | 86 | 18 (22%) | 65 (78%) |   | 3.7 (2.6–5.2) | 1 (Reference) |   | 1 (Reference) |   | 10.5 (7.4–13.3) | 1 (Reference) |   | 1 (Reference) |   |
| A/A | 7 | 3 (43%) | 4 (57%) |   | 8.7 (3.4–15.6b) | 0.40 (0.16–0.99) |   | 0.39 (0.16–0.98) |   | 18.4b (4.7–18.4b) | 0.50 (0.16–1.61) |   | 0.47 (0.15–1.52) |   |

Table 2 Caption: Association between GC rs4588 and clinical outcomes in mCRC patients

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/eb45/7505162/798b817a9bd9/nihms-1623472-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7505162_nihms-1623472-f0001.jpg)

A, Discovery cohort: GC rs4588 and OS (FIRE-3 FOLFIRI/bevacizumab arm). B, Validation cohort 1: GC rs4588 and OS (TRIBE FOLFIRI/bevacizumab arm). C, Control cohort: GC rs4588 and OS (FIRE-3 FOLFIRI/cetuximab arm). D, Validation cohort 2: GC rs4588 and PFS in refractory mCRC patients treated with cetuximab ± irinotecan.

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/eb45/7505162/a34a19deed9b/nihms-1623472-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7505162_nihms-1623472-f0002.jpg)

Combined cohort: GC rs4588 and OS in FIRE-3 and TRIBE FOLFIRI/bevacizumab arms.

## Discussion

Previous studies have demonstrated that 25-OHD inhibits proliferation, induces differentiation and stimulates apoptosis in colon cancer cells ([30](#R30)). Furthermore, 25-OHD decreases tumor progression by inhibiting angiogenesis through downregulation of angiogenic factors such as VEGF, IL8, and prostaglandin E2 (PGE2; ref. [14](#R14)). Although vitamin D3 exerts its inhibitory influence on *RAS* signaling through downregulation of EGFR, it simultaneously inhibits proliferation of endothelial cells *in vitro* and decreases angiogenesis *in vivo* ([2](#R2), [13](#R13), [31](#R31)). A previous study demonstrated that the addition of 25-OHD resulted in upregulation of both the antiangiogenic factor, thrombospondin 1, and the proangiogenic factor, VEGF, in the SW480 human colon cancer cell line, leading to a change in the angiogenetic profile ([32](#R32)). Iseki and colleagues ([33](#R33)) observed a significant decrease in vessel counts and VEGF staining in colon tumors after prolonged administration of 25-OHD. Similarly, another study demonstrated that 25-OHD decreases mRNA and protein expression of HIF1alpha and VEGF in colon cancer cells and inhibits cell proliferation under hypoxia ([34](#R34)). A previous study also observed a critical role of 25-OHD in suppressing angiogenesis through downregulation of IL8 in prostate cancer ([35](#R35)). Currently, several clinical trials are evaluating the impact of vitamin D supplementation in patients receiving both adjuvant chemotherapy for stage III colon cancer as well as palliative therapy for mCRC ([NCT02603757](https://clinicaltrials.gov/ct2/show/NCT02603757) and [NCT01516216](https://clinicaltrials.gov/ct2/show/NCT01516216)). Just recently, Ng and colleagues ([36](#R36)) presented preliminary results from the SUNSHINE trial ([NCT01516216](https://clinicaltrials.gov/ct2/show/NCT01516216)) at the 2017 ASCO Annual Meeting, demonstrating that mCRC patients treated with first-line FOLFOX/bevacizumab plus high-dose vitamin D achieved a longer PFS than those receiving FOLFOX/bevacizumab plus low-dose vitamin D.

According to our findings, genotyping of the *GC* gene might help identify which subset of patients benefits from adding bevacizumab to first-line chemotherapy.

Powe and colleagues ([37](#R37)) demonstrated a positive correlation between 25-OHD and DBP levels. Beside its role as a vitamin D transporter, DBP serves as a precursor for the macrophage-activating factor which exerts its antineoplastic effects by inhibiting angiogenesis ([38](#R38)).

The *GC* gene, located on chromosome 4, encodes for DBP and consists of 13 exons. The missense SNP rs4588, located on exon 11, results in an amino acid change of threonine to lysine at position 436 that affects the binding affinity of DBP.

Previous studies have shown that rs4588 AA carriers have the lowest affinity of DBP for vitamin D and lowest plasma 25-hydroxyvitamin D levels ([16](#R16), [17](#R17)). This functional association was further confirmed by Janssens and colleagues ([18](#R18)) who demonstrated that COPD patients harboring the AA genotype had significantly lower baseline 25-OHD levels than those carrying a C allele. In addition, the same trend was observed in a healthy control group, albeit without statistical significance. Similarly, lower 25-OHD levels among AA carriers of the rs4588 SNP was shown in a large cohort of 741 premenopausal Caucasian women ([19](#R19)). In a meta-analysis comprising nine studies, the ancestral C allele was found to be significantly associated with higher 25-OHD concentrations ([20](#R20)). Chinese children carrying an AA genotype exhibited significantly lower 25-OHD levels than C allele carriers. This association persisted after adjusting for age, sex, BMI, and vitamin D intake ([21](#R21)). Notably, these functional correlations between the *GC* rs4588 SNP and 25-OHD levels were consistent across all studies ([20](#R20)). On the basis of these mechanistic data and our validated clinical associations, we hypothesize that mCRC patients carrying the AA genotype of the *GC* rs4588 SNP might have constitutively lower 25-OHD levels, which translates into an inferior OS with FOLFIRI/bevacizumab, but an improved OS with FOLFIRI/cetuximab.

Importantly, we should note that previous findings of associations between the *GC* rs4588 AA genotype and lower concentrations of circulating vitamin D levels are derived from populations of healthy individuals and patients with diseases other than cancer ([16](#R16)–[21](#R21)). Therefore, we cannot state with absolute certainty, whether these findings can be completely translated to our study population of mCRC patients.

To our knowledge, this is the first study demonstrating that the rs4588 SNP within the *GC* gene, encoding for the DBP, might serve as a predictive biomarker in mCRC patients treated with both FOLFIRI/bevacizumab and FOLFIRI/cetuximab. Carriers of the AA genotype had a shorter OS compared with those harboring any C allele when receiving FOLFIRI/bevacizumab in both the discovery (FIRE-3) and validation (TRIBE) cohorts. On combining these cohorts, this association became even more significant. Conversely, AA carriers treated with FOLFIRI/cetuximab (FIRE-3 control cohort) had a better OS compared with those patients harboring any C allele. This favorable outcome was also observed in a second validation set comprising refractory mCRC patients treated with either cetuximab monotherapy or cetuximab and irinotecan. That the second validation cohort consisted of patients who received cetuximab ± irinotecan as later line therapy (vs. the other cohorts who underwent first-line therapy) might explain the significant association between *GC* rs4588 SNP and PFS but not OS. Nevertheless, the favorable effect on outcome among AA carriers treated with cetuximab ± irinotecan or FOLFIRI/cetuximab was consistent, regardless of the number of prior treatment lines.

The inverse associations with outcome among AA carriers treated with FOLFIRI/bevacizumab and FOLFIRI/cetuximab or cetuximab ± irinotecan suggest opposing effects of the *GC* gene with either bevacizumab or cetuximab treatment. With blockade of the VEGF angiogenic pathway by bevacizumab, VEGF-independent angiogenic escape mechanisms will be activated, mainly through the IL8 and Wnt signaling pathways ([14](#R14), [15](#R15)). Moreover, mounting evidence has shown that 25-OHD exerts its anticancer effect not only by decreasing VEGF-dependent and -independent angiogenesis, but also through downregulating Wnt signaling, which in turn further impairs tumor angiogenesis ([14](#R14), [15](#R15)). Given the association between the AA genotype of the *GC* rs4588 SNP with lower circulating 25-OHD ([16](#R16)–[21](#R21)), we hypothesized that in AA carriers the inhibitory effect on VEGF-independent angiogenesis, mainly driven by Wnt signaling but also IL8 and other angiogenic factors ([14](#R14), [15](#R15)), may be smaller compared with those with a C allele (higher circulating 25-OHD levels). This may serve as an explanation as to why mCRC patients harboring the AA genotype treated with FOLFIRI and bevacizumab have a worse outcome compared with C allele carriers. Given the inhibitory effect of 25-OHD on the adaptive immune system ([22](#R22)), we speculate that in AA carriers with constitutively lower 25-OHD levels, the inhibitory influence on the adaptive immune response might be less than in C allele carriers, who exhibit higher 25-OHD levels. Thus, in AA carriers, the equilibrium might slightly shift toward an activated adaptive immune state, where proinflammatory Th1 and Th17 cells are more predominant than immunosuppressive Tregs and Th2 cells ([22](#R22), [23](#R23)). Moreover, accumulating data indicate that the EGFR inhibitor, cetuximab, exhibits an improved anticancer effect when the adaptive immune system is activated ([24](#R24)). This phenomenon has not been described with bevacizumab. This distinction may explain why mCRC patients carrying the AA genotype treated with FOLFIRI and cetuximab have a better outcome compared to those harboring any C allele. Our results support our hypothesis of opposing effects on outcome between AA carriers treated with FOLFIRI/bevacizumab (worse outcome) or FOLFIRI/cetuximab (better outcome). Importantly, our study demonstrates that the selection of appropriate first-line treatment among patients with the same genotype of the *GC* rs4588 SNP might have a meaningful impact on survival. Although AA carriers with mCRC benefit most from adding cetuximab to FOLFIRI backbone chemotherapy in the first-line setting, bevacizumab may even cause harm. As we report about a new potential prognostic and predictive biomarker, it is important to keep in mind that in recent years a number of both prognostic and predictive markers have been identified in mCRC. For example, patients harboring a *BRAF* mutation have a shorter PFS and OS than those with *BRAF* wild-type tumors ([39](#R39)). Even though there are conflicting results from two meta-analyses investigating the clinical benefit of anti-EGFR treatment in mCRC patients with *RAS* wild-type and *BRAF* mutant tumors, the vast majority of clinicians do not treat patients with *BRAF* mutant tumors with cetuximab or panitumumab ([40](#R40)–[43](#R43)). Furthermore, it is well established that mCRC patients with *RAS*-mutant tumors do not derive benefit from anti-EGFR therapy ([44](#R44)). In addition, mounting evidence has shown that mCRC patients with right-sided primary tumors have a shorter OS compared with those with left-sided primary tumors and that patients with right-sided cancers might not derive the same benefit from cetuximab as those with left-sided primary tumors ([45](#R45), [46](#R46)). Interestingly, as shown in [Supplementary Table S4](#SD4), *GC* rs4588 genotypes were not associated with *BRAF* or *RAS* mutational status, nor primary tumor location, suggesting the *GC* rs4588 SNP to be an independent predictive and prognostic biomarker. Limitations of our study are that we neither assessed circulating 25-OHD levels, nor evaluated the inhibitory effect of vitamin D on angiogenesis. Moreover, we did not determine the ability of vitamin D to downregulate EGFR expression directly on the human tissue in this study. The lack of adequate specimen, including fresh-frozen tissue and the paucity of blood in validation cohorts 1 and 2, precluded us from conducting these assays. In addition, although in the discovery and control cohorts (FIRE-3) only FFPE tissue was available, we tested blood in the validation cohorts (TRIBE and USA/Italy). Despite the different origin of tissue (FFPE samples versus blood) used in our study, there is increasing evidence from several comparison studies, that DNA from FFPE tissue can serve as a valid proxy for germline DNA in pharmacogenetic association studies, not only in colorectal cancer ([47](#R47), [48](#R48)) but also in several other malignancies ([49](#R49), [50](#R50)). Furthermore, validation cohort 2 was comprised of total only 93 patients—86 with a C allele and 7 with the AA genotype. The number of AA genotype carriers (*N*=7) in validation cohort 2 is relatively small due to the low minor allele frequency (MAF = 0.28) of the *GC* rs4588 SNP. Therefore, the results obtained in the second validation cohort must be interpreted within the limitations of these small numbers. However, the findings in the second validation cohort resonate with the ones showed in the FIRE-3 cetuximab cohort, as we observed the same statistically significant difference in outcome in these two independent cohorts, whose patients were uniformly treated with cetuximab ± irinotecan-based chemotherapy. Strengths of our study include the total sample size of 893 mCRC patients and the inclusion of a discovery, control and two validation cohorts from two phase III randomized trials, one phase II study and one cohort with prospectively enrolled patients. Second, the results we obtained in both the discovery and control cohorts were confirmed in two independent validation sets. Furthermore, the rationale to test the impact of the rs4588 SNP within the *GC* gene, encoding for the DBP, on outcome is based on previously published mechanistic data. Sample size limitations and the low frequency of *GC* rs4588 AA genotype carriers (7%–9.4%) precluded us from conducting subgroup analyses in *RAS* wild-type/mutant or *BRAF*-mutant patients, and in those with right-sided primary tumors, as it would not allow us to draw any firm conclusions. Associations between *GC* SNP rs4588 and outcome with either bevacizumab or cetuximab should be further evaluated before clinical use, and should take into account known prognostic factors, particularly *RAS*/*BRAF* mutation status and sidedness. Retrospective, pooled biomarker studies using larger datasets, preferentially from randomized trials, are needed to define which subgroups of patients will derive the most benefit from either bevacizumab- or cetuximab-based chemotherapy, according to *GC* rs4588 SNP (AA versus any C allele). For example, there is increasing evidence from recent studies that mCRC patients with right-sided tumors do not derive benefit from cetuximab ([45](#R45), [46](#R46)). Thus, our findings might provide an impetus to further explore whether AA carriers of the *GC* rs4588 SNP with right-sided, *RAS* wild-type mCRC would benefit from treatment with cetuximab. As our study design did not allow for evaluation of circulating 25-OHD, it would be interesting to conduct a study exploring whether vitamin D supplementation might result in increased vitamin D levels among AA carriers of the *GC* rs4588 SNP, as the AA genotype has been shown to be associated with constitutively decreased 25-OHD levels ([16](#R16)–[21](#R21)). Looking forward, to use the *GC* rs4588 SNP in the context of already established prognostic and predictive biomarkers, we will need future algorithms combining this SNP with information about *RAS/BRAF* mutational status and sidedness, which will help us to further optimize personalized treatment for mCRC patients. In conclusion, the *GC* rs4588 SNP might serve as both a prognostic and predictive biomarker in mCRC patients. Our results suggest that genotyping of the rs4588 polymorphism within the *GC* gene may help identify patients who will derive the most benefit from adding either bevacizumab or cetuximab to irinotecan-based chemotherapy.

## Supplementary Material

Epidemiologic studies have shown that higher blood 25-hydroxyvitamin D (25-OHD) levels are associated with lower colorectal cancer risk. In addition, preclinical studies have demonstrated that 25-OHD inhibits colon cancer growth by stimulating apoptosis and decreasing cancer cell proliferation and tumor angiogenesis. A variation of the vitamin D–binding protein gene, *GC rs4588*, might serve as a predictive marker in mCRC patients treated with FOLFIRI and bevacizumab or FOLFIRI and cetuximab. Whereas AA genotype carriers derive a survival benefit from FOLFIRI/cetuximab, treatment with FOLFIRI/bevacizumab is associated with worse outcome. Genotyping of the rs4588 polymorphism within the *GC* gene may help identify patients who will benefit most from adding either bevacizumab or cetuximab to irinotecan-based chemotherapy.

Epidemiologic studies have shown that higher blood 25-hydroxyvitamin D (25-OHD) levels are associated with lower colorectal cancer risk. In addition, preclinical studies have demonstrated that 25-OHD inhibits colon cancer growth by stimulating apoptosis and decreasing cancer cell proliferation and tumor angiogenesis. A variation of the vitamin D–binding protein gene, *GC rs4588*, might serve as a predictive marker in mCRC patients treated with FOLFIRI and bevacizumab or FOLFIRI and cetuximab. Whereas AA genotype carriers derive a survival benefit from FOLFIRI/cetuximab, treatment with FOLFIRI/bevacizumab is associated with worse outcome. Genotyping of the rs4588 polymorphism within the *GC* gene may help identify patients who will benefit most from adding either bevacizumab or cetuximab to irinotecan-based chemotherapy.

## Acknowledgments

M.D. Berger received a grant from the Werner and Hedy Berger-Janser Foundation for cancer research and the Swiss Cancer League (BIL KLS-3334-02-2014). This work was partly supported by the National Cancer Institute (grant number P30CA014089), the Gloria Borges Wunderglo Project, Daniel Butler Research Fund, Dhont Family Foundation and Call to Cure. Y. Miyamoto received a grant from the Japan Society for the Promotion of Science (S2606).

Disclosure of Potential Conflicts of Interest

S. Stintzing reports receiving speakers bureau honoraria from AMGEN, Bayer, Lilly, Merck KgaA, Nordic Pharma, Roche, Sanofi, and Takeda, and is a consultant/advisory board member for AMGEN, Bayer, Lilly, Merck KgaA, Nordic Pharma, Roche, and Sanofi. V. Heinemann reports receiving commercial research grants from AMGEN, Boehringer Ingelheim, Celgene, MERCK KgaA, Pfizer, Roche, and Shire, speakers bureau honoraria from Amgen, Baxalta, Merck KgaA, Roche AG, Sanofi, Servier, and SIRTEX, and is a consultant/advisory board member for Amgen, Baxalta, Boehringer Ingelheim, Lilly, Merck KgaA, Merrimack, Roche AG, Sanofi, Servier, SIRTEX, and Taiho. A. Falcone is a consultant/advisory board member for Amgen, Bayer, Bristol, Lilly, Merck, Roche, and Servier. H. Lenz is a consultant/advisory board member for Merck Serono and Roche. No potential conflicts of interest were disclosed by the other authors.

The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked *advertisement* in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Berger MD, Lenz HJ. The safety of monoclonal antibodies for treatment of colorectal cancer. Expert Opin Drug Saf 2016;15:799–808.  [DOI](https://doi.org/10.1517/14740338.2016.1167186) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26982510/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Saf&title=The%20safety%20of%20monoclonal%20antibodies%20for%20treatment%20of%20colorectal%20cancer&author=MD%20Berger&author=HJ%20Lenz&volume=15&publication_year=2016&pages=799-808&pmid=26982510&doi=10.1517/14740338.2016.1167186&)

2. Christakos S, Dhawan P, Verstuyf A, Verlinden L, Carmeliet G. Vitamin D: metabolism, molecular mechanism of action, and pleiotropic effects. Physiol Rev 2016;96:365–408.  [DOI](https://doi.org/10.1152/physrev.00014.2015) | [PMC free article](/articles/PMC4839493/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26681795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiol%20Rev&title=Vitamin%20D:%20metabolism,%20molecular%20mechanism%20of%20action,%20and%20pleiotropic%20effects&author=S%20Christakos&author=P%20Dhawan&author=A%20Verstuyf&author=L%20Verlinden&author=G%20Carmeliet&volume=96&publication_year=2016&pages=365-408&pmid=26681795&doi=10.1152/physrev.00014.2015&)

3. Holick MF. Vitamin D deficiency. N Engl J Med 2007;357:266–81.  [DOI](https://doi.org/10.1056/NEJMra070553) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17634462/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Vitamin%20D%20deficiency&author=MF%20Holick&volume=357&publication_year=2007&pages=266-81&pmid=17634462&doi=10.1056/NEJMra070553&)

4. Ma Y, Zhang P, Wang F, Yang J, Liu Z, Qin H. Association between vitamin D and risk of colorectal cancer: a systematic review of prospective studies. J Clin Oncol 2011;29:3775–82.  [DOI](https://doi.org/10.1200/JCO.2011.35.7566) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21876081/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Association%20between%20vitamin%20D%20and%20risk%20of%20colorectal%20cancer:%20a%20systematic%20review%20of%20prospective%20studies&author=Y%20Ma&author=P%20Zhang&author=F%20Wang&author=J%20Yang&author=Z%20Liu&volume=29&publication_year=2011&pages=3775-82&pmid=21876081&doi=10.1200/JCO.2011.35.7566&)

5. Ng K, Meyerhardt JA, Wu K, Feskanich D, Hollis BW, Giovannucci EL, et al. Circulating 25-hydroxyvitamin d levels and survival in patients with colorectal cancer. J Clin Oncol 2008;26:2984–91.  [DOI](https://doi.org/10.1200/JCO.2007.15.1027) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18565885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Circulating%2025-hydroxyvitamin%20d%20levels%20and%20survival%20in%20patients%20with%20colorectal%20cancer&author=K%20Ng&author=JA%20Meyerhardt&author=K%20Wu&author=D%20Feskanich&author=BW%20Hollis&volume=26&publication_year=2008&pages=2984-91&pmid=18565885&doi=10.1200/JCO.2007.15.1027&)

6. Tretli S, Hernes E, Berg JP, Hestvik UE, Robsahm TE. Association between serum 25(OH)D and death from prostate cancer. Br J Cancer 2009; 100:450–4.  [DOI](https://doi.org/10.1038/sj.bjc.6604865) | [PMC free article](/articles/PMC2658536/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19156140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Association%20between%20serum%2025(OH)D%20and%20death%20from%20prostate%20cancer&author=S%20Tretli&author=E%20Hernes&author=JP%20Berg&author=UE%20Hestvik&author=TE%20Robsahm&volume=100&publication_year=2009&pages=450-4&pmid=19156140&doi=10.1038/sj.bjc.6604865&)

7. Kim Y, Je Y. VitaminDintake, blood 25(OH)D levels, and breast cancer risk or mortality: a meta-analysis. Br J Cancer 2014;110:2772–84.  [DOI](https://doi.org/10.1038/bjc.2014.175) | [PMC free article](/articles/PMC4037823/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24714744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=VitaminDintake,%20blood%2025(OH)D%20levels,%20and%20breast%20cancer%20risk%20or%20mortality:%20a%20meta-analysis&author=Y%20Kim&author=Y%20Je&volume=110&publication_year=2014&pages=2772-84&pmid=24714744&doi=10.1038/bjc.2014.175&)

8. Tretli S, Schwartz GG, Torjesen PA, Robsahm TE. Serum levels of 25-hydroxyvitamin D and survival in Norwegian patients with cancer of breast, colon, lung, and lymphoma: a population-based study. Cancer Causes Control 2012;23:363–70.  [DOI](https://doi.org/10.1007/s10552-011-9885-6) | [PMC free article](/articles/PMC3261400/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22193397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Causes%20Control&title=Serum%20levels%20of%2025-hydroxyvitamin%20D%20and%20survival%20in%20Norwegian%20patients%20with%20cancer%20of%20breast,%20colon,%20lung,%20and%20lymphoma:%20a%20population-based%20study&author=S%20Tretli&author=GG%20Schwartz&author=PA%20Torjesen&author=TE%20Robsahm&volume=23&publication_year=2012&pages=363-70&pmid=22193397&doi=10.1007/s10552-011-9885-6&)

9. Alvarez-Díaz S, Valle N, García JM, Peña C, Freije JM, Quesada V, et al. Cystatin D is a candidate tumor suppressor gene induced by vitamin D in human colon cancer cells. J Clin Invest 2009;119:2343–58.  [DOI](https://doi.org/10.1172/JCI37205) | [PMC free article](/articles/PMC2719930/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19662683/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Cystatin%20D%20is%20a%20candidate%20tumor%20suppressor%20gene%20induced%20by%20vitamin%20D%20in%20human%20colon%20cancer%20cells&author=S%20Alvarez-D%C3%ADaz&author=N%20Valle&author=JM%20Garc%C3%ADa&author=C%20Pe%C3%B1a&author=JM%20Freije&volume=119&publication_year=2009&pages=2343-58&pmid=19662683&doi=10.1172/JCI37205&)

10. Aguilera O, Peña C, García JM, Larriba MJ, Ordóñez-Morán P, Navarro D, et al. The Wnt antagonist DICKKOPF-1 gene is induced by 1alpha,25-dihydroxyvitamin D3 associated to the differentiation of human colon cancer cells. Carcinogenesis 2007;28:1877–84.  [DOI](https://doi.org/10.1093/carcin/bgm094) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17449905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=The%20Wnt%20antagonist%20DICKKOPF-1%20gene%20is%20induced%20by%201alpha,25-dihydroxyvitamin%20D3%20associated%20to%20the%20differentiation%20of%20human%20colon%20cancer%20cells&author=O%20Aguilera&author=C%20Pe%C3%B1a&author=JM%20Garc%C3%ADa&author=MJ%20Larriba&author=P%20Ord%C3%B3%C3%B1ez-Mor%C3%A1n&volume=28&publication_year=2007&pages=1877-84&pmid=17449905&doi=10.1093/carcin/bgm094&)

11. Bai YH, Lu H, Hong D, Lin CC, Yu Z, Chen BC. Vitamin D receptor gene polymorphisms and colorectal cancer risk: a systematic meta-analysis. World J Gastroenterol 2012;18:1672–9.  [DOI](https://doi.org/10.3748/wjg.v18.i14.1672) | [PMC free article](/articles/PMC3325535/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22529698/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J%20Gastroenterol&title=Vitamin%20D%20receptor%20gene%20polymorphisms%20and%20colorectal%20cancer%20risk:%20a%20systematic%20meta-analysis&author=YH%20Bai&author=H%20Lu&author=D%20Hong&author=CC%20Lin&author=Z%20Yu&volume=18&publication_year=2012&pages=1672-9&pmid=22529698&doi=10.3748/wjg.v18.i14.1672&)

12. Perna L, Hoffmeister M, Schöttker B, Arndt V, Haug U, Holleczek B, et al. Vitamin D receptor polymorphism and colorectal cancer-specific and all-cause mortality. Cancer Epidemiol 2013;37:905–7.  [DOI](https://doi.org/10.1016/j.canep.2013.09.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24075799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol&title=Vitamin%20D%20receptor%20polymorphism%20and%20colorectal%20cancer-specific%20and%20all-cause%20mortality&author=L%20Perna&author=M%20Hoffmeister&author=B%20Sch%C3%B6ttker&author=V%20Arndt&author=U%20Haug&volume=37&publication_year=2013&pages=905-7&pmid=24075799&doi=10.1016/j.canep.2013.09.007&)

13. Mantell DJ, Owens PE, Bundred NJ, Mawer EB, Canfield AE. 1 alpha,25-dihydroxyvitamin D(3) inhibits angiogenesis in vitro and in vivo. Circ Res 2000;87:214–20.  [DOI](https://doi.org/10.1161/01.res.87.3.214) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10926872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Res&title=1%20alpha,25-dihydroxyvitamin%20D(3)%20inhibits%20angiogenesis%20in%20vitro%20and%20in%20vivo&author=DJ%20Mantell&author=PE%20Owens&author=NJ%20Bundred&author=EB%20Mawer&author=AE%20Canfield&volume=87&publication_year=2000&pages=214-20&pmid=10926872&doi=10.1161/01.res.87.3.214&)

14. Feldman D, Krishnan AV, Swami S, Giovannucci E, Feldman BJ. The role of vitamin D in reducing cancer risk and progression. Nat Rev Cancer 2014;14:342–57.  [DOI](https://doi.org/10.1038/nrc3691) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24705652/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=The%20role%20of%20vitamin%20D%20in%20reducing%20cancer%20risk%20and%20progression&author=D%20Feldman&author=AV%20Krishnan&author=S%20Swami&author=E%20Giovannucci&author=BJ%20Feldman&volume=14&publication_year=2014&pages=342-57&pmid=24705652&doi=10.1038/nrc3691&)

15. Gacche RN. Compensatory angiogenesis and tumor refractoriness. Oncogenesis 2015;4:e153.  [DOI](https://doi.org/10.1038/oncsis.2015.14) | [PMC free article](/articles/PMC4753522/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26029827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogenesis&title=Compensatory%20angiogenesis%20and%20tumor%20refractoriness&author=RN%20Gacche&volume=4&publication_year=2015&pages=e153&pmid=26029827&doi=10.1038/oncsis.2015.14&)

16. Nissen J, Rasmussen LB, Ravn-Haren G, Andersen EW, Hansen B, Andersen R, et al. Common variants in CYP2R1 and GC genes predict vitamin D concentrations in healthy Danish children and adults. PLoS One 2014;9: e89907.  [DOI](https://doi.org/10.1371/journal.pone.0089907) | [PMC free article](/articles/PMC3937412/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24587115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Common%20variants%20in%20CYP2R1%20and%20GC%20genes%20predict%20vitamin%20D%20concentrations%20in%20healthy%20Danish%20children%20and%20adults&author=J%20Nissen&author=LB%20Rasmussen&author=G%20Ravn-Haren&author=EW%20Andersen&author=B%20Hansen&volume=9&publication_year=2014&pages=e89907&pmid=24587115&doi=10.1371/journal.pone.0089907&)

17. Arnaud J, Constans J. Affinity differences for vitamin D metabolites associated with the genetic isoforms of the human serum carrier protein (DBP). Hum Genet 1993;92:183–8.  [DOI](https://doi.org/10.1007/BF00219689) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8370586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Affinity%20differences%20for%20vitamin%20D%20metabolites%20associated%20with%20the%20genetic%20isoforms%20of%20the%20human%20serum%20carrier%20protein%20(DBP)&author=J%20Arnaud&author=J%20Constans&volume=92&publication_year=1993&pages=183-8&pmid=8370586&doi=10.1007/BF00219689&)

18. Janssens W, Bouillon R, Claes B, Carremans C, Lehouck A, Buysschaert I, et al. Vitamin D deficiency is highly prevalent in COPD and correlates with variants in the vitamin D-binding gene. Thorax 2010; 65:215–20.  [DOI](https://doi.org/10.1136/thx.2009.120659) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19996341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thorax&title=Vitamin%20D%20deficiency%20is%20highly%20prevalent%20in%20COPD%20and%20correlates%20with%20variants%20in%20the%20vitamin%20D-binding%20gene&author=W%20Janssens&author=R%20Bouillon&author=B%20Claes&author=C%20Carremans&author=A%20Lehouck&volume=65&publication_year=2010&pages=215-20&pmid=19996341&doi=10.1136/thx.2009.120659&)

19. Sinotte M, Diorio C, Bérubé S, Pollak M, Brisson J. Genetic polymorphisms of the vitamin D binding protein and plasma concentrations of 25-hydroxyvitamin D in premenopausal women. Am J Clin Nutr 2009; 89:634–40.  [DOI](https://doi.org/10.3945/ajcn.2008.26445) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19116321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Clin%20Nutr&title=Genetic%20polymorphisms%20of%20the%20vitamin%20D%20binding%20protein%20and%20plasma%20concentrations%20of%2025-hydroxyvitamin%20D%20in%20premenopausal%20women&author=M%20Sinotte&author=C%20Diorio&author=S%20B%C3%A9rub%C3%A9&author=M%20Pollak&author=J%20Brisson&volume=89&publication_year=2009&pages=634-40&pmid=19116321&doi=10.3945/ajcn.2008.26445&)

20. McGrath JJ, Saha S, Burne TH, Eyles DW. A systematic review of the association between common single nucleotide polymorphisms and 25-hydroxyvitamin D concentrations. J Steroid Biochem Mol Biol 2010; 121:471–7.  [DOI](https://doi.org/10.1016/j.jsbmb.2010.03.073) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20363324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Steroid%20Biochem%20Mol%20Biol&title=A%20systematic%20review%20of%20the%20association%20between%20common%20single%20nucleotide%20polymorphisms%20and%2025-hydroxyvitamin%20D%20concentrations&author=JJ%20McGrath&author=S%20Saha&author=TH%20Burne&author=DW%20Eyles&volume=121&publication_year=2010&pages=471-7&pmid=20363324&doi=10.1016/j.jsbmb.2010.03.073&)

21. Zhang Y, Wang X, Liu Y, Qu H, Qu S, Wang W, et al. The GC, CYP2R1 and DHCR7 genes are associated with vitamin D levels in northeastern Han Chinese children. Swiss Med Wkly 2012;142:w13636.  [DOI](https://doi.org/10.4414/smw.2012.13636) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22801813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Swiss%20Med%20Wkly&title=The%20GC,%20CYP2R1%20and%20DHCR7%20genes%20are%20associated%20with%20vitamin%20D%20levels%20in%20northeastern%20Han%20Chinese%20children&author=Y%20Zhang&author=X%20Wang&author=Y%20Liu&author=H%20Qu&author=S%20Qu&volume=142&publication_year=2012&pages=w13636&pmid=22801813&doi=10.4414/smw.2012.13636&)

22. Bikle DD. Vitamin D metabolism, mechanism of action, and clinical applications. Chem Biol 2014;21:319–29.  [DOI](https://doi.org/10.1016/j.chembiol.2013.12.016) | [PMC free article](/articles/PMC3968073/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24529992/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Biol&title=Vitamin%20D%20metabolism,%20mechanism%20of%20action,%20and%20clinical%20applications&author=DD%20Bikle&volume=21&publication_year=2014&pages=319-29&pmid=24529992&doi=10.1016/j.chembiol.2013.12.016&)

23. Hewison M Vitamin D and immune function: an overview. Proc Nutr Soc 2012;71:50–61.  [DOI](https://doi.org/10.1017/S0029665111001650) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21849106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Nutr%20Soc&title=Vitamin%20D%20and%20immune%20function:%20an%20overview&author=M%20Hewison&volume=71&publication_year=2012&pages=50-61&pmid=21849106&doi=10.1017/S0029665111001650&)

24. Yang X, Zhang X, Mortenson ED, Radkevich-Brown O, Wang Y, Fu YX. Cetuximab-mediated tumor regression depends on innate and adaptive immune responses. Mol Ther 2013;21:91–100.  [DOI](https://doi.org/10.1038/mt.2012.184) | [PMC free article](/articles/PMC3538305/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22990672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Ther&title=Cetuximab-mediated%20tumor%20regression%20depends%20on%20innate%20and%20adaptive%20immune%20responses&author=X%20Yang&author=X%20Zhang&author=ED%20Mortenson&author=O%20Radkevich-Brown&author=Y%20Wang&volume=21&publication_year=2013&pages=91-100&pmid=22990672&doi=10.1038/mt.2012.184&)

25. Loupakis F, Antoniotti C, Cremolini C, Zhang W, Yang D, Wakatsuki T, et al. Prospective study of EGFR intron 1 (CA)n repeats variants as predictors of benefit from cetuximab and irinotecan in chemo-refractory metastatic colorectal cancer (mCRC) patients. Pharmacogenomics J 2014;14:322–7.  [DOI](https://doi.org/10.1038/tpj.2014.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24513691/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Prospective%20study%20of%20EGFR%20intron%201%20(CA)n%20repeats%20variants%20as%20predictors%20of%20benefit%20from%20cetuximab%20and%20irinotecan%20in%20chemo-refractory%20metastatic%20colorectal%20cancer%20(mCRC)%20patients&author=F%20Loupakis&author=C%20Antoniotti&author=C%20Cremolini&author=W%20Zhang&author=D%20Yang&volume=14&publication_year=2014&pages=322-7&pmid=24513691&doi=10.1038/tpj.2014.1&)

26. Zhang W, Gordon M, Schultheis AM, Yang DY, Nagashima F, Azuma M, et al. FCGR2A and FCGR3A polymorphisms associated with clinical outcome of epidermal growth factor receptor expressing metastatic colorectal cancer patients treated with single-agent cetuximab. J Clin Oncol 2007;25:3712–8.  [DOI](https://doi.org/10.1200/JCO.2006.08.8021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17704420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=FCGR2A%20and%20FCGR3A%20polymorphisms%20associated%20with%20clinical%20outcome%20of%20epidermal%20growth%20factor%20receptor%20expressing%20metastatic%20colorectal%20cancer%20patients%20treated%20with%20single-agent%20cetuximab&author=W%20Zhang&author=M%20Gordon&author=AM%20Schultheis&author=DY%20Yang&author=F%20Nagashima&volume=25&publication_year=2007&pages=3712-8&pmid=17704420&doi=10.1200/JCO.2006.08.8021&)

27. Stintzing S, Modest DP, Rossius L, Lerch MM, von Weikersthal LF, Decker T, et al. FOLFIRI plus cetuximab versus FOLFIRI plus bevacizumab for metastatic colorectal cancer (FIRE-3): a post-hoc analysis of tumour dynamics in the final RAS wild-type subgroup of this randomised open-label phase 3 trial. Lancet Oncol 2016;17:1426–1434.  [DOI](https://doi.org/10.1016/S1470-2045(16)30269-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27575024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=FOLFIRI%20plus%20cetuximab%20versus%20FOLFIRI%20plus%20bevacizumab%20for%20metastatic%20colorectal%20cancer%20(FIRE-3):%20a%20post-hoc%20analysis%20of%20tumour%20dynamics%20in%20the%20final%20RAS%20wild-type%20subgroup%20of%20this%20randomised%20open-label%20phase%203%20trial&author=S%20Stintzing&author=DP%20Modest&author=L%20Rossius&author=MM%20Lerch&author=LF%20von%20Weikersthal&volume=17&publication_year=2016&pages=1426-1434&pmid=27575024&doi=10.1016/S1470-2045(16)30269-8&)

28. Loupakis F, Cremolini C, Masi G, Lonardi S, Zagonel V, Salvatore L, et al. Initial therapy with FOLFOXIRI and bevacizumab for metastatic colorectal cancer. N Engl J Med 2014;371:1609–18.  [DOI](https://doi.org/10.1056/NEJMoa1403108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25337750/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Initial%20therapy%20with%20FOLFOXIRI%20and%20bevacizumab%20for%20metastatic%20colorectal%20cancer&author=F%20Loupakis&author=C%20Cremolini&author=G%20Masi&author=S%20Lonardi&author=V%20Zagonel&volume=371&publication_year=2014&pages=1609-18&pmid=25337750&doi=10.1056/NEJMoa1403108&)

29. Altman DG, McShane LM, Sauerbrei W, Taube SE. Reporting recommendations for tumor marker prognostic studies (REMARK): explanation and elaboration. BMC Med 2012;10:51.  [DOI](https://doi.org/10.1186/1741-7015-10-51) | [PMC free article](/articles/PMC3362748/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22642691/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med&title=Reporting%20recommendations%20for%20tumor%20marker%20prognostic%20studies%20(REMARK):%20explanation%20and%20elaboration&author=DG%20Altman&author=LM%20McShane&author=W%20Sauerbrei&author=SE%20Taube&volume=10&publication_year=2012&pages=51&pmid=22642691&doi=10.1186/1741-7015-10-51&)

30. Padi SK, Zhang Q, Rustum YM, Morrison C, Guo B. MicroRNA-627 mediates the epigenetic mechanisms of vitamin D to suppress proliferation of human colorectal cancer cells and growth of xenograft tumors in mice. Gastroenterology 2013;145:437–46.  [DOI](https://doi.org/10.1053/j.gastro.2013.04.012) | [PMC free article](/articles/PMC3722307/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23619147/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=MicroRNA-627%20mediates%20the%20epigenetic%20mechanisms%20of%20vitamin%20D%20to%20suppress%20proliferation%20of%20human%20colorectal%20cancer%20cells%20and%20growth%20of%20xenograft%20tumors%20in%20mice&author=SK%20Padi&author=Q%20Zhang&author=YM%20Rustum&author=C%20Morrison&author=B%20Guo&volume=145&publication_year=2013&pages=437-46&pmid=23619147&doi=10.1053/j.gastro.2013.04.012&)

31. Deeb KK, Trump DL, Johnson CS. Vitamin D signalling pathways in cancer: potential for anticancer therapeutics. Nat Rev Cancer 2007;7:684–700.  [DOI](https://doi.org/10.1038/nrc2196) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17721433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=Vitamin%20D%20signalling%20pathways%20in%20cancer:%20potential%20for%20anticancer%20therapeutics&author=KK%20Deeb&author=DL%20Trump&author=CS%20Johnson&volume=7&publication_year=2007&pages=684-700&pmid=17721433&doi=10.1038/nrc2196&)

32. Fernandez-Garcia NI, Palmer HG, Garcia M, Gonzalez-Martin A, del Rio M, Barettino D, et al. 1alpha,25-Dihydroxyvitamin D3 regulates the expression of Id1 and Id2 genes and the angiogenic phenotype of human colon carcinoma cells. Oncogene 2005;24:6533–44.  [DOI](https://doi.org/10.1038/sj.onc.1208801) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16007183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=1alpha,25-Dihydroxyvitamin%20D3%20regulates%20the%20expression%20of%20Id1%20and%20Id2%20genes%20and%20the%20angiogenic%20phenotype%20of%20human%20colon%20carcinoma%20cells&author=NI%20Fernandez-Garcia&author=HG%20Palmer&author=M%20Garcia&author=A%20Gonzalez-Martin&author=M%20del%20Rio&volume=24&publication_year=2005&pages=6533-44&pmid=16007183&doi=10.1038/sj.onc.1208801&)

33. Iseki K, Tatsuta M, Uehara H, Iishi H, Yano H, Sakai N, et al. Inhibition of angiogenesis as a mechanism for inhibition by 1alpha-hydroxyvitamin D3 and 1,25-dihydroxyvitamin D3 of colon carcinogenesis induced by azoxymethane in Wistar rats. Int J Cancer 1999;81:730–3.  [DOI](https://doi.org/10.1002/(sici)1097-0215(19990531)81:5<730::aid-ijc11>3.0.co;2-q) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10328225/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Inhibition%20of%20angiogenesis%20as%20a%20mechanism%20for%20inhibition%20by%201alpha-hydroxyvitamin%20D3%20and%201,25-dihydroxyvitamin%20D3%20of%20colon%20carcinogenesis%20induced%20by%20azoxymethane%20in%20Wistar%20rats&author=K%20Iseki&author=M%20Tatsuta&author=H%20Uehara&author=H%20Iishi&author=H%20Yano&volume=81&publication_year=1999&pages=730-3&pmid=10328225&doi=10.1002/(sici)1097-0215(19990531)81:5<730::aid-ijc11>3.0.co;2-q&)

34. Ben-Shoshan M, Amir S, Dang DT, Dang LH, Weisman Y, Mabjeesh NJ. 1alpha,25-dihydroxyvitamin D3 (Calcitriol) inhibits hypoxia-inducible factor-1/vascular endothelial growth factor pathway in human cancer cells. Mol Cancer Ther 2007;6:1433–9.  [DOI](https://doi.org/10.1158/1535-7163.MCT-06-0677) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17431122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&title=1alpha,25-dihydroxyvitamin%20D3%20(Calcitriol)%20inhibits%20hypoxia-inducible%20factor-1/vascular%20endothelial%20growth%20factor%20pathway%20in%20human%20cancer%20cells&author=M%20Ben-Shoshan&author=S%20Amir&author=DT%20Dang&author=LH%20Dang&author=Y%20Weisman&volume=6&publication_year=2007&pages=1433-9&pmid=17431122&doi=10.1158/1535-7163.MCT-06-0677&)

35. Bao BY, Yao J, Lee YF. 1alpha, 25-dihydroxyvitamin D3 suppresses interleukin-8-mediated prostate cancer cell angiogenesis. Carcinogenesis 2006; 27:1883–93.  [DOI](https://doi.org/10.1093/carcin/bgl041) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16624828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=1alpha,%2025-dihydroxyvitamin%20D3%20suppresses%20interleukin-8-mediated%20prostate%20cancer%20cell%20angiogenesis&author=BY%20Bao&author=J%20Yao&author=YF%20Lee&volume=27&publication_year=2006&pages=1883-93&pmid=16624828&doi=10.1093/carcin/bgl041&)

36. Ng K, Nimeiri HS, McCleary NJ, Abrams TA, Yurgelun MB, Cleary JM, et al. SUNSHINE: Randomized double-blind phase II trial of vitamin D supplementation in patients with previously untreated metastatic colorectal cancer. J Clin Oncol 35:15s, 2017(suppl; abstr 3506).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=SUNSHINE:%20Randomized%20double-blind%20phase%20II%20trial%20of%20vitamin%20D%20supplementation%20in%20patients%20with%20previously%20untreated%20metastatic%20colorectal%20cancer&author=K%20Ng&author=HS%20Nimeiri&author=NJ%20McCleary&author=TA%20Abrams&author=MB%20Yurgelun&volume=35&publication_year=2017&pages=15s&)

37. Powe CE, Ricciardi C, Berg AH, Erdenesanaa D, Collerone G, Ankers E, et al. Vitamin D-binding protein modifies the vitamin D-bone mineral density relationship. J Bone Miner Res 2011;26:1609–16.  [DOI](https://doi.org/10.1002/jbmr.387) | [PMC free article](/articles/PMC3351032/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21416506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Bone%20Miner%20Res&title=Vitamin%20D-binding%20protein%20modifies%20the%20vitamin%20D-bone%20mineral%20density%20relationship&author=CE%20Powe&author=C%20Ricciardi&author=AH%20Berg&author=D%20Erdenesanaa&author=G%20Collerone&volume=26&publication_year=2011&pages=1609-16&pmid=21416506&doi=10.1002/jbmr.387&)

38. Kisker O, Onizuka S, Becker CM, Fannon M, Flynn E, D’Amato R, et al. Vitamin D binding protein-macrophage activating factor (DBP-maf) inhibits angiogenesis and tumor growth in mice. Neoplasia 2003;5: 32–40.  [DOI](https://doi.org/10.1016/s1476-5586(03)80015-5) | [PMC free article](/articles/PMC1502120/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12659668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neoplasia&title=Vitamin%20D%20binding%20protein-macrophage%20activating%20factor%20(DBP-maf)%20inhibits%20angiogenesis%20and%20tumor%20growth%20in%20mice&author=O%20Kisker&author=S%20Onizuka&author=CM%20Becker&author=M%20Fannon&author=E%20Flynn&volume=5&publication_year=2003&pages=32-40&pmid=12659668&doi=10.1016/s1476-5586(03)80015-5&)

39. Modest DP, Ricard I, Heinemann V, Hegewisch-Becker S, Schmiegel W, Porschen R, et al. Outcome according to KRAS-, NRAS- and BRAF-mutation as well as KRAS mutation variants: pooled analysis of five randomized trials in metastatic colorectal cancer by the AIO colorectal cancer study group. Ann Oncol 2016;27:1746–53.  [DOI](https://doi.org/10.1093/annonc/mdw261) | [PMC free article](/articles/PMC4999563/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27358379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Outcome%20according%20to%20KRAS-,%20NRAS-%20and%20BRAF-mutation%20as%20well%20as%20KRAS%20mutation%20variants:%20pooled%20analysis%20of%20five%20randomized%20trials%20in%20metastatic%20colorectal%20cancer%20by%20the%20AIO%20colorectal%20cancer%20study%20group&author=DP%20Modest&author=I%20Ricard&author=V%20Heinemann&author=S%20Hegewisch-Becker&author=W%20Schmiegel&volume=27&publication_year=2016&pages=1746-53&pmid=27358379&doi=10.1093/annonc/mdw261&)

40. Rowland A, Dias MM, Wiese MD, Kichenadasse G, McKinnon RA, Karapetis CS, et al. Meta-analysis of BRAF mutation as a predictive biomarker of benefit from anti-EGFR monoclonal antibody therapy for RAS wild-type metastatic colorectal cancer. Br J Cancer 2015;112:1888–94.  [DOI](https://doi.org/10.1038/bjc.2015.173) | [PMC free article](/articles/PMC4580381/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25989278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Meta-analysis%20of%20BRAF%20mutation%20as%20a%20predictive%20biomarker%20of%20benefit%20from%20anti-EGFR%20monoclonal%20antibody%20therapy%20for%20RAS%20wild-type%20metastatic%20colorectal%20cancer&author=A%20Rowland&author=MM%20Dias&author=MD%20Wiese&author=G%20Kichenadasse&author=RA%20McKinnon&volume=112&publication_year=2015&pages=1888-94&pmid=25989278&doi=10.1038/bjc.2015.173&)

41. Pietrantonio F, Petrelli F, Coinu A, Di Bartolomeo M, Borgonovo K, Maggi C, et al. Predictive role of BRAF mutations in patients with advanced colorectal cancer receiving cetuximab and panitumumab: a meta-analysis. Eur J Cancer 2015;51:587–94.  [DOI](https://doi.org/10.1016/j.ejca.2015.01.054) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25673558/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=Predictive%20role%20of%20BRAF%20mutations%20in%20patients%20with%20advanced%20colorectal%20cancer%20receiving%20cetuximab%20and%20panitumumab:%20a%20meta-analysis&author=F%20Pietrantonio&author=F%20Petrelli&author=A%20Coinu&author=M%20Di%20Bartolomeo&author=K%20Borgonovo&volume=51&publication_year=2015&pages=587-94&pmid=25673558&doi=10.1016/j.ejca.2015.01.054&)

42. van Brummelen EMJ, de Boer A, Beijnen JH, Schellens JHM. BRAF mutations as predictive biomarker for response to anti-EGFR monoclonal antibodies. Oncologist 2017;22:864–872.  [DOI](https://doi.org/10.1634/theoncologist.2017-0031) | [PMC free article](/articles/PMC5507642/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28576857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=BRAF%20mutations%20as%20predictive%20biomarker%20for%20response%20to%20anti-EGFR%20monoclonal%20antibodies&author=EMJ%20van%20Brummelen&author=A%20de%20Boer&author=JH%20Beijnen&author=JHM%20Schellens&volume=22&publication_year=2017&pages=864-872&pmid=28576857&doi=10.1634/theoncologist.2017-0031&)

43. Cremolini C, Antoniotti C, Moretto R, Masi G, Falcone A. First-line therapy for mCRC - the influence of primary tumour location on the therapeutic algorithm. Nat Rev Clin Oncol 2017;14:113.  [DOI](https://doi.org/10.1038/nrclinonc.2016.219) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28094264/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Clin%20Oncol&title=First-line%20therapy%20for%20mCRC%20-%20the%20influence%20of%20primary%20tumour%20location%20on%20the%20therapeutic%20algorithm&author=C%20Cremolini&author=C%20Antoniotti&author=R%20Moretto&author=G%20Masi&author=A%20Falcone&volume=14&publication_year=2017&pages=113&pmid=28094264&doi=10.1038/nrclinonc.2016.219&)

44. Allegra CJ, Rumble RB, Hamilton SR, Mangu PB, Roach N, Hantel A, et al. Extended RAS gene mutation testing in metastatic colorectal carcinoma to predict response to anti-epidermal growth factor receptor monoclonal antibody therapy: American Society of Clinical Oncology Provisional Clinical Opinion Update 2015. J Clin Oncol 2016;34:179–85.  [DOI](https://doi.org/10.1200/JCO.2015.63.9674) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26438111/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Extended%20RAS%20gene%20mutation%20testing%20in%20metastatic%20colorectal%20carcinoma%20to%20predict%20response%20to%20anti-epidermal%20growth%20factor%20receptor%20monoclonal%20antibody%20therapy:%20American%20Society%20of%20Clinical%20Oncology%20Provisional%20Clinical%20Opinion%20Update%202015&author=CJ%20Allegra&author=RB%20Rumble&author=SR%20Hamilton&author=PB%20Mangu&author=N%20Roach&volume=34&publication_year=2016&pages=179-85&pmid=26438111&doi=10.1200/JCO.2015.63.9674&)

45. Venook AP, Niedzwicki D, Innocenti F, Fruth B, Green C, O’Neil BH, et al. Impact of primary (1º) tumor location on overall survival (OS) and progression-free survival (PFS) in patients (pts) with metastatic colorectal cancer (mCRC): analysis of CALGB/SWOG 80405 (Alliance). J Clin Oncol 34:15s, 2016(suppl; abstr 3504).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=and%20progression-free%20survival%20(PFS)%20in%20patients%20(pts)%20with%20metastatic%20colorectal%20cancer%20(mCRC):%20analysis%20of%20CALGB/SWOG%2080405%20(Alliance)&author=AP%20Venook&author=D%20Niedzwicki&author=F%20Innocenti&author=B%20Fruth&author=C%20Green&volume=34&publication_year=2016&pages=15s&)

46. Tejpar S, Stintzing S, Ciardiello F, Tabernero J, Van CutsemE, Beier F, et al. Prognostic and predictive relevance of primary tumor location in patients with RAS wild-type metastatic colorectal cancer: retrospective analyses of the CRYSTAL and FIRE-3 trials. JAMA Oncol. 2016. October 10 [Epub ahead of print].  [DOI](https://doi.org/10.1001/jamaoncol.2016.3797) | [PMC free article](/articles/PMC7505121/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27722750/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Oncol&title=Van%20CutsemE,%20Beier%20F,%20et%20al.%20Prognostic%20and%20predictive%20relevance%20of%20primary%20tumor%20location%20in%20patients%20with%20RAS%20wild-type%20metastatic%20colorectal%20cancer:%20retrospective%20analyses%20of%20the%20CRYSTAL%20and%20FIRE-3%20trials&author=S%20Tejpar&author=S%20Stintzing&author=F%20Ciardiello&author=J%20Tabernero&publication_year=2016&pmid=27722750&doi=10.1001/jamaoncol.2016.3797&)

47. Van Huis-Tanja L, Kweekel D, Gelderblom H, Koopman M, Punt K, Guchelaar HJ, et al. Concordance of genotype for polymorphisms in DNA isolated from peripheral blood and colorectal cancer tumor samples. Pharmacogenomics 2013;14:2005–12.  [DOI](https://doi.org/10.2217/pgs.13.169) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24279855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Concordance%20of%20genotype%20for%20polymorphisms%20in%20DNA%20isolated%20from%20peripheral%20blood%20and%20colorectal%20cancer%20tumor%20samples&author=L%20Van%20Huis-Tanja&author=D%20Kweekel&author=H%20Gelderblom&author=M%20Koopman&author=K%20Punt&volume=14&publication_year=2013&pages=2005-12&pmid=24279855&doi=10.2217/pgs.13.169&)

48. Marsh S, Mallon MA, Goodfellow P, McLeod HL. Concordance of pharmacogenetic markers in germline and colorectal tumor DNA. Pharmacogenomics 2005;6:873–7.  [DOI](https://doi.org/10.2217/14622416.6.8.873) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16296950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Concordance%20of%20pharmacogenetic%20markers%20in%20germline%20and%20colorectal%20tumor%20DNA&author=S%20Marsh&author=MA%20Mallon&author=P%20Goodfellow&author=HL%20McLeod&volume=6&publication_year=2005&pages=873-7&pmid=16296950&doi=10.2217/14622416.6.8.873&)

49. Goetz MP, Rae JM, Suman VJ, Safgren SL, Ames MM, Visscher DW, et al. Pharmacogenetics of tamoxifen biotransformation is associated with clinical outcomes of efficacy and hot flashes. J Clin Oncol 2005;23:9312–8.  [DOI](https://doi.org/10.1200/JCO.2005.03.3266) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16361630/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Pharmacogenetics%20of%20tamoxifen%20biotransformation%20is%20associated%20with%20clinical%20outcomes%20of%20efficacy%20and%20hot%20flashes&author=MP%20Goetz&author=JM%20Rae&author=VJ%20Suman&author=SL%20Safgren&author=MM%20Ames&volume=23&publication_year=2005&pages=9312-8&pmid=16361630&doi=10.1200/JCO.2005.03.3266&)

50. McWhinney SR, McLeod HL. Using germline genotype in cancer pharmacogenetic studies. Pharmacogenomics 2009;10:489–93.  [DOI](https://doi.org/10.2217/14622416.10.3.489) | [PMC free article](/articles/PMC2722037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19650256/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Using%20germline%20genotype%20in%20cancer%20pharmacogenetic%20studies&author=SR%20McWhinney&author=HL%20McLeod&volume=10&publication_year=2009&pages=489-93&pmid=19650256&doi=10.2217/14622416.10.3.489&)
