# HLA-B*57:01 Confers Susceptibility to Pazopanib-Associated Liver Injury in Patients with Cancer

## Metadata
**Authors:** Chun-Fang Xu, Toby Johnson, Xiaojing Wang, Chris Carpenter, Alan P Graves, Liling Warren, Zhengyu Xue, Karen S King, Dana J Fraser, Sandy Stinnett, Linda P Briley, Ionel Mitrica, Colin F Spraggs, Matthew R Nelson, Hiroomi Tada, Andreas du Bois, Thomas Powles, Neil Kaplowitz, Lini N Pandite
**Journal:** Clinical cancer research : an official journal of the American Association for Cancer Research
**Date:** 2015 Nov 6
**DOI:** [10.1158/1078-0432.CCR-15-2044](https://doi.org/10.1158/1078-0432.CCR-15-2044)
**PMID:** 26546620
**PMCID:** PMC7444994
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7444994/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7444994/pdf/nihms-1618212.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7444994/pdf/nihms-1618212.pdf)

## Abstract

**Purpose:** 
Pazopanib is an effective treatment for advanced renal cell carcinoma and soft-tissue sarcoma. Transaminase elevations have been commonly observed in pazopanib-treated patients. We conducted pharmacogenetic analyses to explore mechanistic insight into pazopanib-induced liver injury.

**Experimental Design:** 
The discovery analysis tested association between four-digit HLA alleles and alanine aminotransferase (ALT) elevation in pazopanib-treated patients with cancer from eight clinical trials (N = 1,188). We conducted confirmatory analysis using an independent dataset of pazopanib-treated patients from 23 additional trials (N = 1,002). Genome-wide association study (GWAS) for transaminase elevations was also conducted.

**Results:** 
The discovery study identified an association between HLA-B*57:01 carriage and ALT elevation [P = 5.0 × 10−5 for maximum on-treatment ALT (MaxALT); P = 4.8 × 10−4 for time to ALT > 3× upper limit of normal (ULN) event; P = 4.1 × 10−5 for time to ALT > 5× ULN event] that is significant after adjustment for number of HLA alleles tested. We confirmed these associations with time to ALT elevation event (P = 8.1 × 10−4 for ALT > 3× ULN, P = 9.8 × 10−3 for ALT > 5× ULN) in an independent dataset. In the combined data, HLA-B*57:01 carriage was associated with ALT elevation (P = 4.3 × 10−5 for MaxALT, P = 5.1 × 10−6 for time to ALT > 3× ULN event, P = 5.8 × 10−6 for time to ALT > 5× ULN event). In HLA-B*57:01 carriers and noncarriers, frequency of ALT > 3× ULN was 31% and 19%, respectively, and frequency of ALT > 5× ULN was 18% and 10%, respectively. GWAS revealed a possible borderline association, which requires further evaluation.

**Conclusions:** 
These data indicate that HLA-B*57:01 carriage confers higher risk of ALT elevation in patients receiving pazopanib and provide novel insight implicating an immune-mediated mechanism for pazopanib-associated hepatotoxicity in some patients.

### Purpose:

Pazopanib is an effective treatment for advanced renal cell carcinoma and soft-tissue sarcoma. Transaminase elevations have been commonly observed in pazopanib-treated patients. We conducted pharmacogenetic analyses to explore mechanistic insight into pazopanib-induced liver injury.

### Experimental Design:

The discovery analysis tested association between four-digit *HLA* alleles and alanine aminotransferase (ALT) elevation in pazopanib-treated patients with cancer from eight clinical trials (*N* = 1,188). We conducted confirmatory analysis using an independent dataset of pazopanib-treated patients from 23 additional trials (*N* = 1,002). Genome-wide association study (GWAS) for transaminase elevations was also conducted.

### Results:

The discovery study identified an association between *HLA-B**57:01 carriage and ALT elevation [*P* = 5.0 × 10^−5^ for maximum on-treatment ALT (MaxALT); *P* = 4.8 × 10^−4^ for time to ALT > 3× upper limit of normal (ULN) event; *P* = 4.1 × 10^−5^ for time to ALT > 5× ULN event] that is significant after adjustment for number of *HLA* alleles tested. We confirmed these associations with time to ALT elevation event (*P* = 8.1 × 10^−4^ for ALT > 3× ULN, *P* = 9.8 × 10^−3^ for ALT > 5× ULN) in an independent dataset. In the combined data, *HLA-B**57:01 carriage was associated with ALT elevation (*P* = 4.3 × 10^−5^ for MaxALT, *P* = 5.1 × 10^−6^ for time to ALT > 3× ULN event, *P* = 5.8 × 10^−6^ for time to ALT > 5× ULN event). In *HLA-B**57:01 carriers and noncarriers, frequency of ALT > 3× ULN was 31% and 19%, respectively, and frequency of ALT > 5× ULN was 18% and 10%, respectively. GWAS revealed a possible borderline association, which requires further evaluation.

### Conclusions:

These data indicate that *HLA-B**57:01 carriage confers higher risk of ALT elevation in patients receiving pazopanib and provide novel insight implicating an immune-mediated mechanism for pazopanib-associated hepatotoxicity in some patients.

## Introduction

Pazopanib, a multitargeted tyrosine kinase inhibitor, is an effective treatment for advanced renal cell carcinoma (RCC) and soft-tissue sarcoma (STS; refs. [1](#R1)^–^[3](#R3)). Hepatotoxicity is an established adverse event associated with pazopanib treatment, commonly presenting as isolated serum transaminase or total bilirubin elevations ([4](#R4)). Alanine aminotransferase (ALT) elevation >3× upper limit of normal (ULN) occurs in approximately 20% of patients receiving pazopanib in clinical trials, with 91% of such events occurring within 18 weeks of commencing treatment ([5](#R5)). Concurrent elevations of ALT (>3× ULN) and total bilirubin (>2× ULN) were observed in approximately 1.8% of patients ([5](#R5)). After clinical adjudication, Hy’s Law cases, a hallmark for significant risk of developing severe drug-induced liver injury ([6](#R6)), were seen in 0.4% of patients ([5](#R5)). Previous pharmacogenetic analyses in pazopanib-treated patients identified that Gilbert syndrome *UGT1A1* variants were associated with bilirubin elevation ([7](#R7), [8](#R8)). A suggestive association between *HFE* polymorphisms and ALT was also reported ([9](#R9)).

Adverse drug reactions from off-target effects are generally unpredictable. Recent studies identified significant association between adverse drug reactions and specific human leukocyte antigen (*HLA*) alleles ([10](#R10)), suggesting that many such reactions involve immune mechanisms. In particular, specific *HLA* alleles are strongly associated with hepatotoxicity for amoxicillin clavulanate (*HLA-DRB1**15:01, *HLA-A**02:01; ref. [11](#R11)), ticlopidine (*HLA-A**33:03; ref. [12](#R12)), ximelagatran (*HLA-DRB1**07:01; ref. [13](#R13)), flucloxacillin (*HLA-B**57:01; ref. [14](#R14)), lumiracoxib (*HLA-DRB1**15:01; ref. [15](#R15)), and lapatinib (*HLA-DRB1**07:01; ref. [16](#R16), [17](#R17)), with ORs ranging from 2 to approximately 80. Here, we sought to characterize the molecular mechanisms of pazopanib-induced ALT elevations and identify genetic markers that might predict the development of liver injury, using patient data and samples collected during pazopanib clinical trials.

## Patients and Methods

### Patients

This pharmacogenetic analysis included patients who provided written informed consent for the clinical study and genetic research and who received at least one dose of pazopanib in 31 GlaxoSmithKline (GSK)-sponsored clinical trials. The discovery analysis included patients from eight phase II to III clinical trials that evaluated efficacy and safety of pazopanib monotherapy in patients with advanced RCC, advanced STS, or ovarian cancer ([Supplementary Table S1](#SD1), online only). The confirmatory analysis included patients from 23 additional phase I–III clinical trials that evaluated safety, tolerability, pharmacokinetics, and efficacy of pazopanib as monotherapy or in combination with other agents in patients with solid tumors ([Supplementary Table S2](#SD1), online only). All trials were conducted in accordance with the Declaration of Helsinki; protocols and informed consent forms were reviewed and approved by Institutional Review Boards/Independent Ethics Committees according to local guidelines.

### Genotyping

Germline DNA was extracted from peripheral blood using the Qiagen Autopure LS or QiAmp DNA Blood Kit by Quest Diagnostics and MDS Pharma Services. *HLA* genotyping using sequencing was conducted by Histogenetics and Beijing Anapure Bioscientific. Genome-wide SNP genotyping was performed using Illumina arrays (Human1M BeadChip, HumanOmni5-Quad BeadChip, or HumanOmniExpressExome BeadChip) by Expression Analysis, Illumina, and ShanghaiBio Corporation for patients in the discovery analysis, and using the Affymetrix Axiom Biobank Plus GSK custom array by BioStorage Technologies for patients in the confirmatory analysis. *UGT1A1*28* genotyping was performed using Sanger sequencing by ShanghaiBio Corporation, Gen-Probe, and GSK, or using the Invader UGT1A1 Molecular Assay by LabCorp and Cogenics. All genotype calling and quality control were performed in accordance with the manufacturers’ protocols.

### Liver chemistry measurement and monitoring

Serum liver chemistry assessments were performed by local institutional laboratories. Measured values (in IU/L) were converted to multiples of ULN by dividing by the institutional laboratory–specific ULN values. The details of liver chemistry monitoring were reported previously ([5](#R5)). Briefly, most studies had entry criteria of ALT/AST ≤ 2.5× ULN and total bilirubin ≤ 1.5× ULN. For isolated ALT > 3–8× ULN, study treatment could continue with liver chemistry monitored until normalization or stabilization; for isolated ALT > 8× ULN, dose would be interrupted and liver chemistry properly monitored. For concurrent ALT > 3× ULN and total bilirubin > 2× ULN with >35% direct bilirubin or with hypersensitivity (i.e., potential Hy’s Law cases), dosing was permanently discontinued.

### Statistical analysis

In this pharmacogenetic analysis, we used ALT for characterization of liver events as it is considered hepatic specific ([18](#R18)). To maximize the statistical power to detect a genetic association, we analyzed maximum ALT (MaxALT) within the pazopanib on-therapy window (defined as first day of therapy until 28 days after last day of therapy) and time-to-event endpoints. In the time-to-event analysis, events were defined as first ALT > 3× ULN or >5× ULN within the on-therapy window; patients who never had on-therapy ALT > ULN (strict controls) were censored at the end of pazopanib treatment, and patients who neither had an event nor were strict controls were excluded. To characterize predictive values and cumulative incidence of events, we analyzed binary endpoints including all patients in the analysis (ever had on-therapy ALT > 3× ULN or >5× ULN, vs. broad controls who never had on-therapy ALT > 3× ULN or >5× ULN, respectively). We used linear regression to test genetic association with MaxALT, Cox regression to test genetic association with time-to-event endpoints, and logistic regression to test genetic association with binary (ever vs. never) endpoints. All regression models were adjusted for clinical study and arm, sex, age at baseline, baseline ALT, and ancestry principal components to control for population stratification ([19](#R19)). Two-tailed *P* values are reported unless otherwise specified.

In the discovery analysis, we tested 92 *HLA* markers [minor allele frequency (MAF) ≥ 1%], initially assuming an additive genetic model for each allele, and used a Bonferroni adjusted significance threshold *P* ≤ 5.4 × 10^−4^ (0.05/92). MaxALT and time to ALT> 3× ULN were co-primary endpoints, with time to 5× ULN as an exploratory endpoint. On the basis of results from the discovery analysis and the low frequency of *HLA-B**57:01 homozygotes, all results for *HLA-B**57:01 are reported assuming a dominant genetic model (carriers vs. noncarriers). Confirmatory analyses specified one-tailed tests with MaxALT as the primary endpoint and time to events (ALT > 3× ULN and >5× ULN) as secondary endpoints. A *post hoc* analysis with the combined datasets further evaluated the *HLA-B**57:01 association with ALT elevation and characterized predictive values and cumulative incidence of events.

Genome-wide analyses (GWAS) used a cosmopolitan reference panel of 2,048 haplotypes ([20](#R20)) to impute a common set of variants across the different genotyping platforms ([21](#R21), [22](#R22)). To control for technical batch effects, we analyzed the data in five subgroups according to genotyping platform and batch. Within each subgroup, we used Cox regression to test association for each genetic variant with time to ALT > 3× ULN, adjusting for clinical study and arm, sex, age at baseline, baseline ALT, and ancestry principal components, and assuming an additive genetic model for each variant genome-wide. Analyses were combined across subgroups using a sample size weighted genome-wide meta-analysis ([23](#R23)). In total, 6,736,730 common variants (MAF ≥5%) genome-wide were tested, and the conventional GWAS significance threshold (*P* ≤ 5.0 × 10^−8^) was applied. Variants identified by the GWAS meta-analysis were further evaluated for association with all ALT elevation endpoints (MaxALT, time to ALT > 3× ULN and >5× ULN, ever vs. never ALT > 3× ULN and >5× ULN) in a pooled analysis using the combined dataset as described above.

### Computational modeling

The binding of pazopanib to HLA-B*57:01 was modeled using Molecular Operating Environment (MOE) software version 2012.10 (Chemical Computing Group Inc.). Illing and colleagues demonstrated that the interaction of abacavir with HLA-B*57:01 altered antigen recognition and triggered immune self-reactivity by T-cell activation ([24](#R24)). The shared features between pazopanib and abacavir supported the use of the reported crystal structure of abacavir bound to HLA-B*57:01 (pdb ID 3VRJ) as a guide for modeling pazopanib binding. The structure was prepared using the Structure Preparation Tool in MOE. Atomic charges were assigned using the PFROSST force-field parameters. Hydrogens were added and their positions optimized using the Protonate 3-dimensional (3D) tool. The observed binding mode of abacavir guided the manual placement of pazopanib in the antigen-binding cleft of HLA-B*57:01. Once a reasonable pose for pazopanib was obtained, LigX was used to minimize the ligand and nearby protein residues with the PFROSST molecular mechanics force field, Born solvation model, and default settings to optimize favorable interactions and relieve remaining steric clashes between pazopanib and the HLA-B*57:01 receptor site residues.

## Results

In total, we evaluated clinical and genetic data from 2,190 pazopanib-treated patients in 31 clinical trials ([Table 1](#T1); [Supplementary Table S3](#SD1), online only). In the discovery analysis, using data from eight clinical trials of pazopanib 800 mg monotherapy (*N* = 1,188), *HLA-B**57:01, *HLA-C**06:02, and *HLA-C**04:01 were significantly associated with ALT elevations (multiple-test corrected threshold *P* ≤ 5.4 × 10^−4^). The strongest association was between *HLA-B**57:01 and MaxALT (*P* = 5.0 × 10^−5^; [Table 2](#T2)), and a similar strength of association (*P* = 4.1 × 10^−5^) was seen for time to ALT > 5× ULN event (NCI CTCAEv4 grade 3 ALT elevation). Multivariate analyses showed that the associations of *HLA-C**06:02 and *HLA-C**04:01 with ALT were not statistically significant after conditioning on *HLA-B**57:01, indicating a single independent genetic effect best explained by *HLA-B**57:01.

### Table 1.

| Parameters | Discovery (n = 1,188) | Confirmatory (n = 1,002) | Total (n = 2,190) |
| --- | --- | --- | --- |
| Age, median (range), y | 59 (18-86) | 56 (18-86) | 58 (18-86) |
| Sex, n (%) |   |   |   |
| Male | 557 (47) | 359 (36) | 916 (42) |
| Female | 631 (53) | 643 (64) | 1,274 (58) |
| Race, n (%) |   |   |   |
| White | 844 (71) | 811 (81) | 1,655 (76) |
| Asian | 325 (27) | 114 (11) | 439 (20) |
| Othera | 19 (2) | 77 (8) | 96 (4) |
| Baseline liver metastasis,b n (%) |  |  |  |
| Yes | 165 (14) | 208 (21) | 373 (17) |
| Baseline ALT, ×ULN |   |   |   |
| Median (range) | 0.46 (0.07-2.82) | 0.48 (0.06-6.91) | 0.47 (0.06-6.91) |

Table 1 Caption: Baseline characteristics for patients in the pharmacogenetic analyses

### Table 2.

|   | Discovery study |  |  |  |  | Confirmatory study |  |  |  |  | Combined analysis |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| ALT measure | N | Noncarriers, n | Carriers,a n | Effect (95% CI) | P | N | Noncarriers, n | Carriers,a n | Effect (95% CI) | P | N | Noncarriers, n | Carriers, n | Effect (95% CI) | P |
| MaxALTb | 1,188 | 1,124 | 64 | 1.6 (1.3–2.0) | 5.0 × 10−5 | 1,002 | 935 | 67 | 1.2 (0.97–1.51) | 0.085 | 2,190 | 2,059 | 131 | 1.4 (1.2–1.6) | 4.3 × 10−5 |
| Time to >3× ULNc | 242 | 219 | 23 | HR = 2.4 (1.5–3.8) | 4.8 × 10−4 | 187 | 169 | 18 | HR = 3.3 (1.7–6.2) | 8.1 × 10−4 | 429 | 388 | 41 | HR = 2.5 (1.7–3.6) | 5.1 × 10−6 |
| Censored | 498 | 481 | 17 |   |   | 452 | 430 | 22 |   |   | 950 | 911 | 39 |   |   |
| Time to >5× ULNc | 138 | 121 | 17 | HR = 3.6 (2.1–6.2) | 4.1 × 10−5 | 93 | 86 | 7 | HR = 4.6 (1.7–12.6) | 9.8 × 10−3 | 231 | 207 | 24 | HR = 3.4 (2.1–5.5) | 5.8 × 10−6 |
| Censored | 498 | 481 | 17 |   |   | 452 | 430 | 22 |   |   | 950 | 911 | 39 |   |   |
| >3× ULN | 242 | 219 | 23 | OR = 2.3 (1.3–4.1) | 0.0039 | 187 | 169 | 18 | OR = 1.8 (0.93–3.5) | 0.086 | 429 | 388 | 41 | OR = 2.0 (1.3–3.1) | 0.0014 |
| ≤3× ULN | 946 | 905 | 41 |   |   | 815 | 766 | 49 |   |   | 1,761 | 1,671 | 90 |   |   |
| >5× ULN | 138 | 121 | 17 | OR = 3.1 (1.7–5.8) | 7.2 × 10−4 | 93 | 86 | 7 | OR = 1.2 (0.45–3.1) | 0.75 | 231 | 207 | 24 | OR = 2.1 (1.3–3.6) | 0.0058 |
| ≤5× ULN | 1,050 | 1,003 | 47 |   |   | 909 | 849 | 60 |   |   | 1,959 | 1,852 | 107 |   |   |

Table 2 Caption: Association between HLA-B*57:01 carriage and ALT elevation in pazopanib-treated patients with cancer

We therefore attempted to confirm the *HLA-B**57:01 association with ALT in an independent dataset of 1,002 patients who received pazopanib (at various doses, either as monotherapy or in combination with other agents) for solid tumors. *HLA-B**57:01 was borderline significantly associated with MaxALT in this dataset (*P* = 0.042 for prespecified one-tailed test; [Table 2](#T2) reported the two-tailed *P* value as *P* = 0.085) and demonstrated a statistically significant replication of the association with time to ALT > 5× ULN event [HR, 4.6; 95% confidence interval (CI), 1.7–12.6; *P* = 9.8 × 10^−3^] and time to ALT > 3× ULN event (HR, 3.3; 95% CI, 1.7–6.2; *P* = 8.1 × 10^−4^; [Table 2](#T2)).

In the combined dataset of discovery and confirmatory studies, *HLA-B**57:01 was significantly associated (*P* ≤ 5.4 × 10^−4^) with ALT elevation in pazopanib-treated patients for all three ALT endpoints (*P* = 4.3 × 10^−5^ for MaxALT, *P* = 5.1 × 10^−6^ for time to ALT > 3× ULN, and *P* = 5.8 × 10^−6^ for time to ALT > 5× ULN). In *HLA-B**57:01 carriers (*n* = 131) and non-carriers (*n* = 2,059), the median values (25th–75th percentiles) of MaxALT were 1.7 (0.9–3.5)× ULN and 1.2 (0.7–2.2)× ULN, respectively ([Fig. 1](#F1)), consistent with the 1.4-fold increase in carriers estimated by regression ([Table 2](#T2)). The frequency of ALT > 3× ULN was 31% in *HLA-B**57:01 carriers and 19% in non-carriers, and the frequency of ALT > 5× ULN was 18% in *HLA-B**57:01 carriers and 10% in non-carriers ([Fig. 2](#F2)).

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e630/7444994/127c4f922ca5/nihms-1618212-f0001.jpg)

Association between HLA-B*57:01 carriage and maximum on-treatment ALT in pazopanib-treated patients from the combined data (N = 2,190).

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e630/7444994/f58b1e560963/nihms-1618212-f0002.jpg)

Cumulative incidence of ALT > 3× ULN (A) and ALT > 5× ULN events (B) by HLA-B*57:01 carriage status in pazopanib-treated patients from the combined data (N = 2,190). The x-axis was truncated at 100 days.

As reported previously ([5](#R5)), baseline ALT was associated with ALT elevation in pazopanib-treated patients in the combined data analyzed here (Spearman rank correlation, *ρ* = 0.40, *P*~ 10^−85^). However, *HLA-B**57:01 was not associated with baseline ALT (*P* = 0.52), indicating that the association with ALT elevation is specific for response to treatment. In *post hoc* sensitivity analyses, we excluded 192 patients with baseline ALT > ULN and observed slightly stronger associations between *HLA-B**57:01 and ALT elevation ([Supplementary Table S4](#SD1)) compared with the main analyses that modeled the effect of baseline ALT as a covariate ([Table 2](#T2)).

Within the combined data, ALT > 3× ULN and >5× ULN events occurred in 20% and 11% of pazopanib-treated patients, respectively, with *HLA-B**57:01 carriers having a higher risk of experiencing ALT elevation than non-carriers (OR, 2.0; 95% CI, 1.3–3.1 for ALT > 3× ULN and OR, 2.1; 95% CI, 1.3–3.6 for ALT > 5× ULN; [Table 2](#T2)). Consistent with the low carriage frequency for *HLA-B**57:01 (6% in the combined data) and modest effect size, the fraction of patients with ALT elevation potentially attributable to *HLA-B**57:01 is modest: 10% for ALT > 3× ULN and 10% for ALT > 5× ULN. As a predictor of ALT elevation in pazopanib-treated patients (ALT > 3× ULN vs. ALT ≤ 3× ULN), *HLA-B**57:01 carriage improves the positive predictive value (PPV), 31% (95% CI, 23%–40%) compared with 20% in all treated patients, but has little impact on the negative predictive value (NPV), 81% (95% CI, 79%–83%) compared with background 80%. The predictive performance was similar for ALT > 5× ULN versus ALT ≤ 5× ULN, with PPV 18% (95% CI, 12%–26%) improved over background 11%; and NPV 90% (95% CI, 89%–91%), similar to background 89%.

A key liver safety signal is concurrent elevation of ALT (>3× ULN) and total bilirubin (>2× ULN) with no evidence of biliary obstruction, which may reflect extensive hepatocyte damage and loss of hepatic function. We characterized 26 patients with laboratory ALT > 3× ULN, total bilirubin >2× ULN by *HLA-B**57:01, and the Gilbert syndrome (*UGT1A1*) genotypes. Eight patients (33%) had the Gilbert *UGT1A1* genotypes and three patients (12%) were *HLA-B**57:01 carriers. No patients carried both the Gilbert *UGT1A1* and *HLA-B**57:01 risk genotypes. We also made detailed causality assessments according to DILIN criteria (definite, highly likely, probable, possible, unlikely; ref. [25](#R25)) for all 26 cases based on available clinical, laboratory, and *UGT1A1* genotyping data ([Supplementary Table S5](#SD1), online only). This assessment adjudicated four cases that met criteria for Hy’s Law; of these, two carried *HLA-B**57:01 alleles. Although the numbers are very small, the point estimate 50% (2 of 4) is notably greater than approximately 10% of isolated ALT elevations potentially attributable to *HLA-B**57:01. Evaluation of other *HLA* genotypes (*HLA-A, -B, -C, DRB1, DQA1, DQB1*, and *DPB1*) in patients with combined ALT and bilirubin elevation did not reveal additional robust significant association.

The molecular mechanism for the observed *HLA* association with pazopanib ALT elevation is unknown. The reported crystal structure of abacavir bound to HLA-B*57:01 guided our computational model of pazopanib binding with HLA-B*57:01 ([Fig. 3](#F3)). The resulting model suggests that the pyrimidine ring of pazopanib can make a pi-methyl stacking interaction with Val97 and a pi-edge stacking interaction with Trp147, whereas the N1 pyrimidine nitrogen is predicted to form a hydrogen bond with the side chain of Asp114, similar to the observed interactions of the purinyl group of abacavir in the E pocket of HLA-B*57:01. In this binding mode, the *N*-methyl of pazopanib occupies a similar region as the cyclopropyl of abacavir, whereas the indazole of pazopanib fills more of the F pocket relative to abacavir by making additional hydrophobic contacts with Asn77, Ile80, Tyr84, Thr143, and Trp147. Finally, the phenyl group of pazopanib is predicted to bind in a similar region as the cyclopentene group of abacavir in the D pocket of HLA-B*57:01 and make hydrophobic contacts with Tyr9, Tyr99, Leu156, and Tyr159. The sulfonamide group of pazopanib is directed toward solvent and is predicted to form a water-mediated interaction with the side-chain hydroxyls of Tyr9 and Ser70. Our computational modeling therefore suggests that pazopanib is able to make similar interactions with HLA-B*57:01 compared with abacavir.

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e630/7444994/d3b9bf842533/nihms-1618212-f0003.jpg)

A model of pazopanib (light pink carbons) bound to HLA-B*57:01. Abacavir (orange carbons) from the crystal structure of abacavir bound to HLA-B*57:01 is overlaid for reference. Crystallographic water molecules retained in the model are represented as red spheres. Protein-ligand interactions are represented by dashed lines. The binding site surface is colored green for hydrophobic regions, blue for mild polar regions, and magenta for hydrogen-bonding regions. Image produced using Molecular Operating Environment software version 2012.10; Chemical Computing Group Inc.

An exploratory GWAS meta-analysis for time to ALT > 3× ULN event using data from all patients in the combined dataset (*N* = 1,379) did not reveal any common variant associations at genome-wide significance (i.e., no common variant with *P* ≤ 5.0 × 10^−8^, [Supplementary Fig. S1](#SD1)). The most significant GWAS signal was at SNP rs1800625 (GWAS meta-analysis *P* = 1.5 × 10^−7^), which was further evaluated using the combined dataset in a pooled analysis (*P* = 7.7 × 10^−9^, [Supplementary Table S6](#SD1)). This SNP maps to the MHC class III region, ~800 kb away from *HLA-B*, and is weakly correlated with *HLA-B**57:01 carriage (*r*^2^ = 0.07). A joint analysis of rs1800625 and *HLA-B**57:01 indicates that the two associations are mostly independent ([Supplementary Table S7](#SD1)).

## Discussion

In this large pharmacogenetic study using data from all completed GSK-sponsored pazopanib clinical trials available at the time, *HLA-B**57:01 is associated with pazopanib-induced ALT elevation. In the discovery analysis, MaxALT and time to event were considered co-primary endpoints. With hindsight, specifying MaxALT as the sole primary endpoint in the confirmatory analysis may have been a suboptimal choice. Because the discovery *P* values for MaxALT (5.0 × 10^−5^) and time to 5× ULN (4.1 × 10^−5^) were similar, either or both could have been chosen as (co-;)primary endpoints in the confirmatory analysis. A more objective approach is to evaluate all endpoints in the full (combined) sample size, and this approach demonstrates compelling evidence for the association.

*HLA-B**57:01 carriers have approximately 1.5- to 2.0-fold greater risk of experiencing ALT increases (>3× ULN, >5× ULN) compared with non-carriers. As expected for a common adverse event (20% for ALT > 3× ULN, 11% for ALT > 5× ULN) and a less common risk marker (6% *HLA-B**57:01 carriage), the NPV of 81% for ALT > 3× ULN and 90% for ALT > 5× ULN is very close to the overall frequency of non-cases in the population studied. *HLA-B**57:01 provides some modest improvements on PPV (ALT > 3× ULN, 31%; ALT > 5× ULN, 18%) over the overall frequency of cases in the population studied, and thus the potential clinical utility of this association very much depends on the availability of other treatment options and on the available strategies for managing the adverse event. Consideration of the therapeutic benefit of pazopanib for advanced RCC or STS versus risk of ALT elevation and the safety profile of alternative treatment likely precludes prospective *HLA-B* genotyping to exclude *B**57:01 allele carriers from pazopanib treatment. Nevertheless, our findings suggest that testing for *HLA-B**57:01 could be used to augment current liver safety management in patients who develop ALT elevations during pazopanib treatment. For example, the widespread availability of *HLA-B* genotyping could be used to support the determination of pazopanib causality for ALT elevations observed during therapy. Furthermore, as the benefit:risk profile in the adjuvant setting will differ from that in metastatic disease, clinical utility of *HLA-B* genotyping should be further evaluated in pazopanib adjuvant clinical trials.

In addition to pazopanib, *HLA-B**57:01 has been shown to be associated with abacavir hypersensitivity syndrome ([26](#R26), [27](#R27)) and flucloxacillin-induced liver injury ([14](#R14)). Screening for *HLA-B**57:01 is advocated in patients who are to receive abacavir but not for flucloxacillin. Illing and colleagues demonstrated that direct interaction of abacavir with the antigen-binding cleft of HLA altered the repertoire of peptides bound to HLA-B* 57:01 ([24](#R24)). This alteration can result in new “self” antigen presentation or new presentation of constitutive self-peptides leading to the drug-induced hypersensitivity. In contrast, mechanistic studies showed that flucloxacillin binds covalently to selective lysine residues on albumin, and the degree of protein binding determines the intensity of the T-cell proliferative response ([28](#R28)). These data indicate that different chemistries associated with these two drugs result in presentation of unique HLA-B*57:01–restricted epitopes to T cells and activation of immune inflammatory damage responses. On the basis of the crystal structure of abacavir bound to HLA-B* 57:01 ([24](#R24)), we used computational modeling to evaluate the possibility of a direct interaction between pazopanib and the antigen-binding cleft of HLA-B* 57:01. Our binding-mode hypothesis indicates that pazopanib is capable of making interactions that are similar to those made by abacavir, which is suggestive, albeit not proof, that pazopanib binds to HLA-B*57:01. Although the effects of such a binding event on T-cell activation are not yet known, we speculate that the proposed interaction between pazopanib and HLA may alter antigen recognition and trigger immune self-reactivity in HLA-B*57:01 carriers. Differences may exist among HLA-B*57:01 carriers in the degree of antigen repertoire shift and polyclonal T-cell receptor engagement, which could explain why only a proportion of specified *HLA* allele carriers progress to drug-induced liver injury. Alternatively, similar to flucloxacillin, the binding of pazopanib to serum albumin may form pazopanib hapten and stimulate T-cell responses in some *HLA-B**57:01 carriers. Inflammatory infiltrate was observed in liver biopsies in two cases of severe pazopanib-induced hepatotoxicity ([29](#R29)). Although the specific molecular mechanism for HLA-B*57:01-mediated liver injury in pazopanib-treated patients is undefined, our finding provides novel insight for a possible immune-mediated mechanism for pazopanib-induced hepatotoxicity in some patients. The fact that only a subset of *HLA-B**57:01 carriers experienced ALT elevation when receiving pazopanib suggests the presence of other modifiers, warranting further investigation.

Within this dataset, *HLA-B**57:01 accounted for approximately 10% of patients who experienced ALT > 3× ULN or ALT > 5× ULN when receiving pazopanib, suggesting there are additional factors contributing to the observed ALT elevations. Previous meta-analyses of pazopanib clinical trials showed older age and concomitant use of simvastatin were associated with increased risk of transaminase elevations ([5](#R5), [30](#R30)). In a previous study using data from *n* = 243 patients (a subset of ~10% of the patients analyzed here), we reported that two SNPs in the *HFE* gene were associated with ALT elevation, with an estimated 12% probability that this observation occurred by chance ([9](#R9)). We tested the association between these two *HFE* SNPs and ALT elevation in the full dataset now available (*n* = 2,190) and were unable to validate this previous finding.

Our GWAS meta-analysis and subsequent analysis of the most strongly associated variant (rs1800625) suggests the possibility of a second independent association within the MHC region. Rs1800625 is located in the promoter region of the *AGER* (*RAGE*) gene, encoding the advanced glycosylation end product receptor, and has been associated with promoter activity and expression ([31](#R31)). This finding awaits confirmation in an independent dataset.

Strengths of the present pharmacogenetic study include the large datasets and frequent laboratory assessment of ALT levels for patients receiving pazopanib. Prospective collection of germline DNA samples during clinical trials enabled this genetic evaluation of pazopanib-induced liver toxicity, and all available data from completed GSK-sponsored pazopanib clinical trials (as of August 1, 2014) were included. The discovery study included patients who received pazopanib as monotherapy for cancers, whereas the confirmatory study included patients who received pazopanib as monotherapy as well as in combination with other anticancer agents. The confirmation study had similar sample size compared with the discovery study but had greater heterogeneity of patients analyzed (including multiple tumor types, different pazopanib dosages and treatment durations, and presence of combination therapies, all of which could dilute the pazopanib-specific genetic signal). Confirmation of the association despite the heterogeneity of the confirmatory dataset may, therefore, demonstrate the robustness of the signal.

In summary, these data indicate that *HLA-B**57:01 carriers have a higher risk of ALT elevation than non-carriers when receiving pazopanib treatment. Our results provide new insights that implicate an immune-mediated mechanism for at least some cases of pazopanib-induced hepatotoxicity. This finding, and the established widespread availability of *HLA* testing, could potentially support the determination of pazopanib causality of hepatotoxicity during pazopanib treatment. Additional studies to further understand the immune-mediated mechanism and further evaluate the rs1800625 association are warranted.

## Supplementary Material

Hepatotoxicity is frequently observed in pazopanib-treated patients; however, the underlying mechanisms have not been elucidated. Because drug-induced liver injury is a potentially serious consequence of therapy, there is a growing need to identify biomarkers that could characterize causality and predict the patient population most likely to experience hepatotoxicity. Recent data linking adverse drug reactions to *HLA* alleles led us to test the hypothesis that specific *HLA* alleles may be related to transaminase elevations in pazopanib-treated patients. Our analysis, which was conducted in two large independent datasets (discovery and confirmatory), demonstrated that *HLA-B**57:01 allele carriage is associated with increased alanine aminotransferase. Together with abacavir hypersensitivity syndrome and flucloxacillin-induced liver injury, pazopanib-induced hepatotoxicity is the third immune-mediated adverse effect involving *HLA-B**57:01. Our results provide additional scientific insight into the mechanisms of pazopanib-related liver toxicity, support the determination of pazopanib causality of hepatotoxicity, and potentially could support safety management for patients receiving pazopanib.

Hepatotoxicity is frequently observed in pazopanib-treated patients; however, the underlying mechanisms have not been elucidated. Because drug-induced liver injury is a potentially serious consequence of therapy, there is a growing need to identify biomarkers that could characterize causality and predict the patient population most likely to experience hepatotoxicity. Recent data linking adverse drug reactions to *HLA* alleles led us to test the hypothesis that specific *HLA* alleles may be related to transaminase elevations in pazopanib-treated patients. Our analysis, which was conducted in two large independent datasets (discovery and confirmatory), demonstrated that *HLA-B**57:01 allele carriage is associated with increased alanine aminotransferase. Together with abacavir hypersensitivity syndrome and flucloxacillin-induced liver injury, pazopanib-induced hepatotoxicity is the third immune-mediated adverse effect involving *HLA-B**57:01. Our results provide additional scientific insight into the mechanisms of pazopanib-related liver toxicity, support the determination of pazopanib causality of hepatotoxicity, and potentially could support safety management for patients receiving pazopanib.

## Acknowledgments

The authors thank the investigators and their patients who made this study possible. Medical editorial assistance for this article was provided by William Sinkins, PhD, of ProEd Communications, Inc., Beachwood, OH, and was funded by GlaxoSmithKline, Philadelphia, PA.

Grant Support

This study was sponsored by GlaxoSmithKline.

Disclosure of Potential Conflicts of Interest

T. Johnson, C. Carpenter, A. Graves, Z. Xue, D.J. Fraser, L.N. Pandite, and M.R. Nelson have ownership interest (including patents) in GlaxoSmithKline. L. Warren is an employee of OmicSoft and PAREXEL. L.P. Briley is an employee of PAREXEL and has ownership interest (including patents) in GlaxoSmithKline. A. du Bois is a consultant/advisory board member for Astra Zeneca, MSD, Pharmamar, and Roche. T. Powles reports receiving commercial research grants from Novartis, Pfizer, and Roche, and speakers bureau honoraria from Novartis and Pfizer. N. Kaplowitz is a consultant/advisory board member for Daiichi Sankyo, GlaxoSmithKline, Johnson & Johnson, Pfizer, Sanofi, and Takeda. No potential conflicts of interest were disclosed by the other authors.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Sternberg CN, Davis ID, Mardiak J, Szczylik C, Lee E, Wagstaff J, et al. Pazopanib in locally advanced or metastatic renal cell carcinoma: results of a randomized phase III trial. J Clin Oncol 2010;28:1061–8.  [DOI](https://doi.org/10.1200/JCO.2009.23.9764) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20100962/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Pazopanib%20in%20locally%20advanced%20or%20metastatic%20renal%20cell%20carcinoma:%20results%20of%20a%20randomized%20phase%20III%20trial&author=CN%20Sternberg&author=ID%20Davis&author=J%20Mardiak&author=C%20Szczylik&author=E%20Lee&volume=28&publication_year=2010&pages=1061-8&pmid=20100962&doi=10.1200/JCO.2009.23.9764&)

2. Motzer RJ, Hutson TE, Cella D, Reeves J, Hawkins R, Guo J, et al. Pazopanib versus sunitinib in metastatic renal-cell carcinoma. N Engl J Med 2013; 369:722–31.  [DOI](https://doi.org/10.1056/NEJMoa1303989) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23964934/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Pazopanib%20versus%20sunitinib%20in%20metastatic%20renal-cell%20carcinoma&author=RJ%20Motzer&author=TE%20Hutson&author=D%20Cella&author=J%20Reeves&author=R%20Hawkins&volume=369&publication_year=2013&pages=722-31&pmid=23964934&doi=10.1056/NEJMoa1303989&)

3. van der Graaf WT, Blay JY, Chawla SP, Kim DW, Bui-Nguyen B, Casali PG, et al. Pazopanib for metastatic soft-tissue sarcoma (PALETTE): a randomised, double-blind, placebo-controlled phase 3 trial. Lancet 2012;379:1879–86.  [DOI](https://doi.org/10.1016/S0140-6736(12)60651-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22595799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Pazopanib%20for%20metastatic%20soft-tissue%20sarcoma%20(PALETTE):%20a%20randomised,%20double-blind,%20placebo-controlled%20phase%203%20trial&author=WT%20van%20der%20Graaf&author=JY%20Blay&author=SP%20Chawla&author=DW%20Kim&author=B%20Bui-Nguyen&volume=379&publication_year=2012&pages=1879-86&pmid=22595799&doi=10.1016/S0140-6736(12)60651-5&)

4. US Food and Drug Administration. Guidance for Industry. Drug-induced liver injury: premarketing clinical evaluation. 2009. [accessed 2015 Mar 18]. Available from: http://www.fda.gov/downloads/Drugs/GuidanceComplianceRegulatoryInformation/Guidances/UCM174090.pdf.  [http://www.fda.gov/downloads/Drugs/GuidanceComplianceRegulatoryInformation/Guidances/UCM174090.pdf](http://www.fda.gov/downloads/Drugs/GuidanceComplianceRegulatoryInformation/Guidances/UCM174090.pdf)

5. Powles T, Bracarda S, Chen M, Norry E, Compton N, Heise M, et al. Characterisation of liver chemistry abnormalities associated with pazopanib monotherapy: a systematic review and meta-analysis of clinical trials in advanced cancer patients. Eur J Cancer 2015;51:1293–302.  [DOI](https://doi.org/10.1016/j.ejca.2015.03.019) | [PMC free article](/articles/PMC7451810/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25899987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=Characterisation%20of%20liver%20chemistry%20abnormalities%20associated%20with%20pazopanib%20monotherapy:%20a%20systematic%20review%20and%20meta-analysis%20of%20clinical%20trials%20in%20advanced%20cancer%20patients&author=T%20Powles&author=S%20Bracarda&author=M%20Chen&author=E%20Norry&author=N%20Compton&volume=51&publication_year=2015&pages=1293-302&pmid=25899987&doi=10.1016/j.ejca.2015.03.019&)

6. Abboud G, Kaplowitz N. Drug-induced liver injury. Drug Saf 2007;30: 277–94.  [DOI](https://doi.org/10.2165/00002018-200730040-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17408305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Saf&title=Drug-induced%20liver%20injury&author=G%20Abboud&author=N%20Kaplowitz&volume=30&publication_year=2007&pages=277-94&pmid=17408305&doi=10.2165/00002018-200730040-00001&)

7. Xu CF, Reck BH, Xue Z, Huang L, Baker KL, Chen M, et al. Pazopanib-induced hyperbilirubinemia is associated with Gilber’s syndrome UGT1A1 polymorphism. Br J Cancer 2010;102:1371–7.  [DOI](https://doi.org/10.1038/sj.bjc.6605653) | [PMC free article](/articles/PMC2865761/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20389299/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Pazopanib-induced%20hyperbilirubinemia%20is%20associated%20with%20Gilber%E2%80%99s%20syndrome%20UGT1A1%20polymorphism&author=CF%20Xu&author=BH%20Reck&author=Z%20Xue&author=L%20Huang&author=KL%20Baker&volume=102&publication_year=2010&pages=1371-7&pmid=20389299&doi=10.1038/sj.bjc.6605653&)

8. Motzer RJ, Johnson T, Choueiri TK, Deen KC, Xue Z, Pandite LN, et al. Hyperbilirubinemia in pazopanib- or sunitinib-treated patients in COMPARZ is associated with UGT1A1 polymorphisms. Ann Oncol 2013;24:2927–8.  [DOI](https://doi.org/10.1093/annonc/mdt394) | [PMC free article](/articles/PMC6394110/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24107802/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Hyperbilirubinemia%20in%20pazopanib-%20or%20sunitinib-treated%20patients%20in%20COMPARZ%20is%20associated%20with%20UGT1A1%20polymorphisms&author=RJ%20Motzer&author=T%20Johnson&author=TK%20Choueiri&author=KC%20Deen&author=Z%20Xue&volume=24&publication_year=2013&pages=2927-8&pmid=24107802&doi=10.1093/annonc/mdt394&)

9. Xu CF, Reck BH, Goodman VL, Xue Z, Huang L, Barnes MR, et al. Association of the hemochromatosis gene with pazopanib-induced transaminase elevation in renal cell carcinoma. J Hepatol 2011;54:1237–43.  [DOI](https://doi.org/10.1016/j.jhep.2010.09.028) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21145803/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Association%20of%20the%20hemochromatosis%20gene%20with%20pazopanib-induced%20transaminase%20elevation%20in%20renal%20cell%20carcinoma&author=CF%20Xu&author=BH%20Reck&author=VL%20Goodman&author=Z%20Xue&author=L%20Huang&volume=54&publication_year=2011&pages=1237-43&pmid=21145803&doi=10.1016/j.jhep.2010.09.028&)

10. Daly AK. Pharmacogenomics of adverse drug reactions. Genome Med 2013;5:5.  [DOI](https://doi.org/10.1186/gm409) | [PMC free article](/articles/PMC3707028/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23360680/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med&title=Pharmacogenomics%20of%20adverse%20drug%20reactions&author=AK%20Daly&volume=5&publication_year=2013&pages=5&pmid=23360680&doi=10.1186/gm409&)

11. Lucena MI, Molokhia M, Shen Y, Urban TJ, Aithal GP, Andrade RJ, et al. Susceptibility to amoxicillin-clavulanate-induced liver injury is influenced by multiple HLA class I and II alleles. Gastroenterology 2011;141:338–47.  [DOI](https://doi.org/10.1053/j.gastro.2011.04.001) | [PMC free article](/articles/PMC3129430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21570397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Susceptibility%20to%20amoxicillin-clavulanate-induced%20liver%20injury%20is%20influenced%20by%20multiple%20HLA%20class%20I%20and%20II%20alleles&author=MI%20Lucena&author=M%20Molokhia&author=Y%20Shen&author=TJ%20Urban&author=GP%20Aithal&volume=141&publication_year=2011&pages=338-47&pmid=21570397&doi=10.1053/j.gastro.2011.04.001&)

12. Hirata K, Takagi H, Yamamoto M, Matsumoto T, Nishiya T, Mori K, et al. Ticlopidine-induced hepatotoxicity is associated with specific human leukocyte antigen genomic subtypes in Japanese patients: a preliminary case-control study. Pharmacogenomics J 2008;8:29–33.  [DOI](https://doi.org/10.1038/sj.tpj.6500442) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17339877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Ticlopidine-induced%20hepatotoxicity%20is%20associated%20with%20specific%20human%20leukocyte%20antigen%20genomic%20subtypes%20in%20Japanese%20patients:%20a%20preliminary%20case-control%20study&author=K%20Hirata&author=H%20Takagi&author=M%20Yamamoto&author=T%20Matsumoto&author=T%20Nishiya&volume=8&publication_year=2008&pages=29-33&pmid=17339877&doi=10.1038/sj.tpj.6500442&)

13. Kindmark A, Jawaid A, Harbron CG, Barratt BJ, Bengtsson OF, Andersson TB, et al. Genome-wide pharmacogenetic investigation of a hepatic adverse event without clinical signs of immunopathology suggests an underlying immune pathogenesis. Pharmacogenomics J 2008;8:186–95.  [DOI](https://doi.org/10.1038/sj.tpj.6500458) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17505501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genome-wide%20pharmacogenetic%20investigation%20of%20a%20hepatic%20adverse%20event%20without%20clinical%20signs%20of%20immunopathology%20suggests%20an%20underlying%20immune%20pathogenesis&author=A%20Kindmark&author=A%20Jawaid&author=CG%20Harbron&author=BJ%20Barratt&author=OF%20Bengtsson&volume=8&publication_year=2008&pages=186-95&pmid=17505501&doi=10.1038/sj.tpj.6500458&)

14. Daly AK, Donaldson PT, Bhatnagar P, Shen Y, Pe’er I, Floratos A, et al. HLA-B*5701 genotype is a major determinant of drug-induced liver injury due to flucloxacillin. Nat Genet 2009;41:816–9.  [DOI](https://doi.org/10.1038/ng.379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19483685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=HLA-B*5701%20genotype%20is%20a%20major%20determinant%20of%20drug-induced%20liver%20injury%20due%20to%20flucloxacillin&author=AK%20Daly&author=PT%20Donaldson&author=P%20Bhatnagar&author=Y%20Shen&author=I%20Pe%E2%80%99er&volume=41&publication_year=2009&pages=816-9&pmid=19483685&doi=10.1038/ng.379&)

15. Singer JB, Lewitzky S, Leroy E, Yang F, Zhao X, Klickstein L, et al. A genome-wide study identifies HLA alleles associated with lumiracoxib-related liver injury. Nat Genet 2010;42:711–4.  [DOI](https://doi.org/10.1038/ng.632) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20639878/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20genome-wide%20study%20identifies%20HLA%20alleles%20associated%20with%20lumiracoxib-related%20liver%20injury&author=JB%20Singer&author=S%20Lewitzky&author=E%20Leroy&author=F%20Yang&author=X%20Zhao&volume=42&publication_year=2010&pages=711-4&pmid=20639878&doi=10.1038/ng.632&)

16. Spraggs CF, Budde LR, Briley LP, Bing N, Cox CJ, King KS, et al. HLA-DQA1*02:01 is a major risk factor for lapatinib-induced hepatotoxicity in women with advanced breast cancer. J Clin Oncol 2011;29:667–73.  [DOI](https://doi.org/10.1200/JCO.2010.31.3197) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21245432/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=HLA-DQA1*02:01%20is%20a%20major%20risk%20factor%20for%20lapatinib-induced%20hepatotoxicity%20in%20women%20with%20advanced%20breast%20cancer&author=CF%20Spraggs&author=LR%20Budde&author=LP%20Briley&author=N%20Bing&author=CJ%20Cox&volume=29&publication_year=2011&pages=667-73&pmid=21245432&doi=10.1200/JCO.2010.31.3197&)

17. Schaid DJ, Spraggs CF, McDonnell SK, Parham LR, Cox CJ, Ejlertsen B, et al. Prospective validation of HLA-DRB1*07:01 allele carriage as a predictive risk factor for lapatinib-induced liver injury. J Clin Oncol 2014;32:2296–303.  [DOI](https://doi.org/10.1200/JCO.2013.52.9867) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24687830/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Prospective%20validation%20of%20HLA-DRB1*07:01%20allele%20carriage%20as%20a%20predictive%20risk%20factor%20for%20lapatinib-induced%20liver%20injury&author=DJ%20Schaid&author=CF%20Spraggs&author=SK%20McDonnell&author=LR%20Parham&author=CJ%20Cox&volume=32&publication_year=2014&pages=2296-303&pmid=24687830&doi=10.1200/JCO.2013.52.9867&)

18. Green RM, Flamm S. AGA technical review on the evaluation of liver chemistry tests. Gastroenterology 2002;123:1367–84.  [DOI](https://doi.org/10.1053/gast.2002.36061) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12360498/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=AGA%20technical%20review%20on%20the%20evaluation%20of%20liver%20chemistry%20tests&author=RM%20Green&author=S%20Flamm&volume=123&publication_year=2002&pages=1367-84&pmid=12360498&doi=10.1053/gast.2002.36061&)

19. Price AL, Patterson NJ, Plenge RM, Weinblatt ME, Shadick NA, Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet 2006;38:904–9.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=AL%20Price&author=NJ%20Patterson&author=RM%20Plenge&author=ME%20Weinblatt&author=NA%20Shadick&volume=38&publication_year=2006&pages=904-9&pmid=16862161&doi=10.1038/ng1847&)

20. Genomes Project C Abecasis GR, Auton A, Brooks LD, DePristo MA, Durbin RM, et al. An integrated map of genetic variation from 1,092 human genomes. Nature 2012;491:56–65.  [DOI](https://doi.org/10.1038/nature11632) | [PMC free article](/articles/PMC3498066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23128226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20map%20of%20genetic%20variation%20from%201,092%20human%20genomes&author=A%20Auton&author=LD%20Brooks&author=MA%20DePristo&author=RM%20Durbin&volume=491&publication_year=2012&pages=56-65&pmid=23128226&doi=10.1038/nature11632&)

21. Li Y, Willer C, Sanna S, Abecasis G. Genotype imputation. Annu Rev Genomics Hum Genet 2009;10:387–406.  [DOI](https://doi.org/10.1146/annurev.genom.9.081307.164242) | [PMC free article](/articles/PMC2925172/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19715440/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Genomics%20Hum%20Genet&title=Genotype%20imputation&author=Y%20Li&author=C%20Willer&author=S%20Sanna&author=G%20Abecasis&volume=10&publication_year=2009&pages=387-406&pmid=19715440&doi=10.1146/annurev.genom.9.081307.164242&)

22. Howie B, Fuchsberger C, Stephens M, Marchini J, Abecasis GR. Fast and accurate genotype imputation in genome-wide association studies through pre-phasing. Nat Genet 2012;44:955–9.  [DOI](https://doi.org/10.1038/ng.2354) | [PMC free article](/articles/PMC3696580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22820512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Fast%20and%20accurate%20genotype%20imputation%20in%20genome-wide%20association%20studies%20through%20pre-phasing&author=B%20Howie&author=C%20Fuchsberger&author=M%20Stephens&author=J%20Marchini&author=GR%20Abecasis&volume=44&publication_year=2012&pages=955-9&pmid=22820512&doi=10.1038/ng.2354&)

23. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genome-wide association scans. Bioinformatics 2010;26:2190–1.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genome-wide%20association%20scans&author=CJ%20Willer&author=Y%20Li&author=GR%20Abecasis&volume=26&publication_year=2010&pages=2190-1&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

24. Illing PT, Vivian JP, Dudek NL, Kostenko L, Chen Z, Bharadwaj M, et al. Immune self-reactivity triggered by drug-modified HLA-peptide repertoire. Nature 2012;486:554–8.  [DOI](https://doi.org/10.1038/nature11147) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22722860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Immune%20self-reactivity%20triggered%20by%20drug-modified%20HLA-peptide%20repertoire&author=PT%20Illing&author=JP%20Vivian&author=NL%20Dudek&author=L%20Kostenko&author=Z%20Chen&volume=486&publication_year=2012&pages=554-8&pmid=22722860&doi=10.1038/nature11147&)

25. Fontana RJ, Watkins PB, Bonkovsky HL, Chalasani N, Davern T, Serrano J, et al. Drug-Induced Liver Injury Network (DILIN) prospective study: rationale, design and conduct. Drug Saf 2009;32:55–68.  [DOI](https://doi.org/10.2165/00002018-200932010-00005) | [PMC free article](/articles/PMC3637941/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19132805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Saf&title=Drug-Induced%20Liver%20Injury%20Network%20(DILIN)%20prospective%20study:%20rationale,%20design%20and%20conduct&author=RJ%20Fontana&author=PB%20Watkins&author=HL%20Bonkovsky&author=N%20Chalasani&author=T%20Davern&volume=32&publication_year=2009&pages=55-68&pmid=19132805&doi=10.2165/00002018-200932010-00005&)

26. Mallal S, Nolan D, Witt C, Masel G, Martin AM, Moore C, et al. Association between presence of HLA-B*5701, HLA-DR7, and HLA-DQ3 and hypersensitivity to HIV-1 reverse-transcriptase inhibitor abacavir. Lancet 2002; 359:727–32.  [DOI](https://doi.org/10.1016/s0140-6736(02)07873-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11888582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Association%20between%20presence%20of%20HLA-B*5701,%20HLA-DR7,%20and%20HLA-DQ3%20and%20hypersensitivity%20to%20HIV-1%20reverse-transcriptase%20inhibitor%20abacavir&author=S%20Mallal&author=D%20Nolan&author=C%20Witt&author=G%20Masel&author=AM%20Martin&volume=359&publication_year=2002&pages=727-32&pmid=11888582&doi=10.1016/s0140-6736(02)07873-x&)

27. Hetherington S, Hughes AR, Mosteller M, Shortino D, Baker KL, Spreen W, et al. Genetic variations in HLA-B region and hypersensitivity reactions to abacavir. Lancet 2002;359:1121–2.  [DOI](https://doi.org/10.1016/S0140-6736(02)08158-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11943262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genetic%20variations%20in%20HLA-B%20region%20and%20hypersensitivity%20reactions%20to%20abacavir&author=S%20Hetherington&author=AR%20Hughes&author=M%20Mosteller&author=D%20Shortino&author=KL%20Baker&volume=359&publication_year=2002&pages=1121-2&pmid=11943262&doi=10.1016/S0140-6736(02)08158-8&)

28. Monshi MM, Faulkner L, Gibson A, Jenkins RE, Farrell J, Earnshaw CJ, et al. Human leukocyte antigen (HLA)-B*57:01-restricted activation of drug-specific T cells provides the immunological basis for flucloxacillin-induced liver injury. Hepatology 2013;57:727–39.  [DOI](https://doi.org/10.1002/hep.26077) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22987284/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Human%20leukocyte%20antigen%20(HLA)-B*57:01-restricted%20activation%20of%20drug-specific%20T%20cells%20provides%20the%20immunological%20basis%20for%20flucloxacillin-induced%20liver%20injury&author=MM%20Monshi&author=L%20Faulkner&author=A%20Gibson&author=RE%20Jenkins&author=J%20Farrell&volume=57&publication_year=2013&pages=727-39&pmid=22987284&doi=10.1002/hep.26077&)

29. Klempner SJ, Choueiri TK,Yee E, Doyle LA, Schuppan D, Atkins MB. Severe pazopanib-induced hepatotoxicity: clinical and histologic course in two patients. J Clin Oncol 2012;30:e264–8.  [DOI](https://doi.org/10.1200/JCO.2011.41.0332) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22802316/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Severe%20pazopanib-induced%20hepatotoxicity:%20clinical%20and%20histologic%20course%20in%20two%20patients&author=SJ%20Klempner&author=TK%20Choueiri&author=E%20Yee&author=LA%20Doyle&author=D%20Schuppan&volume=30&publication_year=2012&pages=e264-8&pmid=22802316&doi=10.1200/JCO.2011.41.0332&)

30. Xu CF, Xue Z, Bing N, King KS, McCann LA, de Souza PL, et al. Concomitant use of pazopanib and simvastatin increases the risk of transaminase elevations in patients with cancer. Ann Oncol 2012;23:2470–1.  [DOI](https://doi.org/10.1093/annonc/mds249) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22918879/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Concomitant%20use%20of%20pazopanib%20and%20simvastatin%20increases%20the%20risk%20of%20transaminase%20elevations%20in%20patients%20with%20cancer&author=CF%20Xu&author=Z%20Xue&author=N%20Bing&author=KS%20King&author=LA%20McCann&volume=23&publication_year=2012&pages=2470-1&pmid=22918879&doi=10.1093/annonc/mds249&)

31. Hudson BI, Stickland MH, Futers TS, Grant PJ. Effects of novel polymorphisms in the RAGE gene on transcriptional regulation and their association with diabetic retinopathy. Diabetes 2001;50:1505–11.  [DOI](https://doi.org/10.2337/diabetes.50.6.1505) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11375354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=Effects%20of%20novel%20polymorphisms%20in%20the%20RAGE%20gene%20on%20transcriptional%20regulation%20and%20their%20association%20with%20diabetic%20retinopathy&author=BI%20Hudson&author=MH%20Stickland&author=TS%20Futers&author=PJ%20Grant&volume=50&publication_year=2001&pages=1505-11&pmid=11375354&doi=10.2337/diabetes.50.6.1505&)
