# Genetic factors influencing warfarin dose in Black-African patients: a systematic review and meta-analysis

## Metadata
**Authors:** Innocent G Asiimwe, Eunice J Zhang, Rostam Osanlou, Amanda Krause, Chrisly Dillon, Guilherme Suarez-Kurtz, Honghong Zhang, Jamila A Perini, Jessicca Y Renta, Jorge Duconge, Larisa H Cavallari, Leiliane R Marcatto, Mark T Beasly, Minoli A Perera, Nita A Limdi, Paulo C J L Santos, Stephen E Kimmel, Steven A Lubitz, Stuart A Scott, Vivian K Kawai, Andrea L Jorgensen, Munir Pirmohamed
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2020 Jan 28
**DOI:** [10.1002/cpt.1755](https://doi.org/10.1002/cpt.1755)
**PMID:** 31869433
**PMCID:** PMC7217737
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7217737/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7217737/pdf/nihms-1065643.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7217737/pdf/nihms-1065643.pdf)

## Abstract

Warfarin is the most commonly used oral anticoagulant in sub-Saharan Africa. Dosing is challenging due to a narrow therapeutic index and high inter-individual variability in dose requirements. To evaluate the genetic factors affecting warfarin dosing in Black-Africans, we performed a meta-analysis of 48 studies (2,336 patients). Significant predictors for CYP2C9 and stable dose included rs1799853 (CYP2C9*2), rs1057910 (CYP2C9*3), rs28371686 (CYP2C9*5), rs9332131 (CYP2C9*6), and rs28371685 (CYP2C9*11) reducing dose by 6.8, 12.5, 13.4, 8.1, and 5.3 mg/week respectively. VKORC1 variants rs9923231 (−1639G>A), rs9934438 (1173C>T), rs2359612 (2255C>T), rs8050894 (1542G>C), and rs2884737 (497T>G) decreased dose by 18.1, 21.6, 17.3, 11.7, and 19.6 mg/week, respectively while rs7294 (3730G>A) increased dose by 6.9 mg/week. Finally, rs12777823 (CYP2C gene cluster) was associated with a dose reduction of 12.7 mg/week. Few studies were conducted in Africa, and patient numbers were small, highlighting the need for further work in Black Africans to evaluate genetic factors determining warfarin response.

## INTRODUCTION

Cardiovascular disease (CVD) is a major public health burden worldwide, sub-Saharan Africa (SSA) included. In 2016, approximately 1.2 million deaths (12.9% of all deaths) in SSA were attributed to CVD ([1](#R1)). Warfarin, an oral anticoagulant, is important for management of venous thromboembolism, valvular heart disease and prevention of stroke in patients with atrial fibrillation. Despite the advent of new oral anticoagulants, warfarin remains the anticoagulant of choice in SSA, and other low-income and emerging countries, mainly because of its significantly lower cost. Treatment with warfarin is difficult due to its narrow therapeutic window, large inter-patient variability in dose requirements and INR monitoring requirements. In SSA, the problems are compounded by high HIV and TB prevalence, lack of clinical expertise and infrastructure, and lack of validated dosing algorithms. Poor anticoagulation can lead to thrombotic or bleeding events: warfarin is among the top four drugs leading to hospitalization from adverse reactions in South Africa ([2](#R2)).

To improve accuracy of warfarin dosing, several dose-prediction algorithms based on both clinical and genetic factors have been developed ([3](#R3)). Studies in Caucasians have revealed that genetic polymorphisms in *CYP2C9* (encodes a warfarin-metabolizing enzyme) and *VKORC1* (encodes warfarin’s molecular target) together with age, height, weight and interacting drugs account for approximately 50% of the required individual daily dose variability ([3](#R3)). However, these algorithms have largely been developed in White patients, and may not be applicable to other populations, including Black-Africans ([4](#R4)). This was demonstrated by the Clarification of Optimal Anticoagulation through Genetics (COAG) trial, in which a genotype-guided dosing algorithm performed worse for African Americans when compared to a clinical algorithm ([5](#R5)). This has been partly explained by the different allele frequencies in *CYP2C9* and *VKORC1* across the ethnicities. For instance, whereas the *VKORC1* rs9923231 allele alone explains approximately 20–25% of the variance in warfarin maintenance dose in Caucasian and Asian populations (respective allele frequencies 0.39 and 0.89), it only accounts for approximately 6% of dose variability in African populations (allele frequency 0.05) ([6](#R6), [7](#R7)). Similarly, the *CYP2C9* alleles *CYP2C9*2* (rs1799853) and *CYP2C9*3* (rs1057910) are more prevalent in Caucasians (respective allele frequencies of 0.12 and 0.07) when compared to Asians (<0.01 and 0.03) and Africans (both <0.01) ([7](#R7)). In Black-Africans, additional *CYP2C9* alleles (*CYP2C9*5*, **6*, **8*, and **11*) may be more important than *CYP2C9*2* and *CYP2C9*3* ([4](#R4), [8](#R8)).

It is important that all relevant ethnicity-specific variants affecting warfarin dose requirements are identified, characterized and accounted for to improve effectiveness of algorithms and to ensure that health inequities are not worsened. Previous reviews evaluating genetic factors modulating warfarin response in Black-African patients have had several limitations including a lack of structured search strategy and focus on a limited number of genetic factors. This systematic review and meta-analysis has therefore critically evaluated the current evidence on Black-African specific genetic factors affecting warfarin dose requirements, and other outcomes representing warfarin response.

## METHODS

### Search strategy and selection criteria

A predefined protocol (PROSPERO: CRD42018110485), based on the principles of the Cochrane Handbook for Systematic Reviews of Interventions ([9](#R9)) and the Human Genome Epidemiology Network (HuGENet) HuGE Review Handbook ([10](#R10)) was followed. This report adheres to the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA) statement ([Table S1](#SD1)). On 30 October 2018, the University of Liverpool’s DISCOVER platform was used to search >480 online databases. Studies were identified using medical subject headings (MeSH terms) and text words related to “African” AND “warfarin” AND “genetic factors” ([Table S2](#SD1)). To determine the completeness of the DISCOVER search, a separate search was conducted in the MEDLINE database ([Table S2](#SD1)). 100% consistency was observed. Next, lists of references from the identified studies were examined to identify additional eligible articles. To identify unpublished trials, trial registries including [ClinicalTrials.gov](http://ClinicalTrials.gov) and the International Clinical Trials Registry Platform were searched. Experts in the field were also contacted to identify further eligible studies. Unless a translated text was available, studies reported in a non-English language were excluded. There was neither restriction by year of publication nor by publication status.

Observational and interventional studies where at least 5% of recruited warfarin-treated patients were Black-Africans, and which investigated the effect of at least one genetic factor on warfarin dose requirements and/or treatment response were included. For randomized controlled trials, only data from patients in the genotyped arm(s) were considered. The primary outcome was stable maintenance dose, and co-primary outcomes were time to stable dose and bleeding events. Secondary outcomes were: International Normalized Ratio (INR) above range in week 1 of treatment, time to achieving therapeutic INR, proportion of time spent within therapeutic INR range, warfarin sensitivity (≤1.5mg/day on 3 successive clinic visits), and warfarin resistance (>10mg/day on 3 successive clinic visits). However, there was no restriction of inclusion criteria to studies that only investigated one or more of these outcomes; rather, studies investigating any other outcomes of warfarin response were also included.

### Data Extraction and quality assessment

Two reviewers (IGA and RO) independently screened titles and abstracts of the retrieved bibliographic records for eligibility and assessed full texts of potentially eligible studies for inclusion. A data extraction tool was developed to extract relevant information related to study and patient characteristics, study quality, outcomes and results. If key information could not be extracted from the published report, the study’s authors were contacted, and the data requested. Studies using the same or overlapping datasets (identified with reference to geographic regions, authors and their affiliations and recruitment sites) were flagged as such by identifying them as being part of a cluster of studies, to ensure that effect estimates from the same dataset were not included in the same meta-analysis more than once. To assess the methodological quality of each included study, two reviewers (IGA and RO) used criteria previously developed to assess the methodological quality of pharmacogenetic studies ([11](#R11))). Disagreements were resolved by consensus.

### Data synthesis and analysis

#### Data synthesis

If ≥2 studies were present, a pooled estimate of effect for each gene variant and outcome combination was obtained by undertaking a meta-analysis. Which genotype groups to compare was dependent on what comparisons had been made in the primary papers. For *CYP2C9* and where only summary genotype data was provided, the three genotype groups (wild-type homozygote, heterozygote and mutant-type homozygote) for each variant were obtained using a strategy provided in Text S1. A genetic model-free approach ([12](#R12)) was used to calculate the pooled effect estimates, such that a particular genetic model did not have to be assumed. The genetic model-free approach was implemented in Stata (version 14) ([13](#R13)) using code provided in Text S1. Where there were no variant-type homozygotes, standard meta-analyses using R (version 3.5.1) ([14](#R14)) (R meta package ([15](#R15))) were performed. Pooled mean differences with 95% confidence intervals (CIs) were generated and forest plots prepared for each genetic variant-outcome combination analyzed, using R (version 3.5.1) ([14](#R14)).

#### Heterogeneity measures

The magnitude of inconsistency in the study results was assessed by visually examining forest plots and considering the *I*^2^ statistic ([9](#R9)). Arbitrarily chosen categories of heterogeneity were defined as follows: *I*^2^ <30 %, low; *I*^2^ 30–70 %, moderate; and *I*^2^ >70 %, high.

#### Selective reporting

Selective reporting was investigated as part of the methodological quality assessment.

#### Publication bias

Where >10 studies were available, publication bias was assessed using the linear regression test of funnel plot asymmetry (implemented using the metabias function in the R meta package ([15](#R15))). A p-value <0.1 was considered to show publication bias. Where asymmetry was suggested by a visual assessment, we performed exploratory analyses to investigate and adjust it (trim and fill analysis) using the trimfill function (R metafor package ([16](#R16))).

#### Sensitivity and subgroup analyses

Sensitivity analyses were performed to assess the impact of the analysis approach used (the genetic-model free approach versus the commonly-used bivariate pairwise approach ([12](#R12))) and the strategy used to infer summary data for each *CYP2C9* genotype group (Text S1) on the pooled effect estimates while subgroup analyses were performed stratified by subpopulation (country used as proxy for ethnicity), to try and address moderate and high heterogeneity.

#### Secondary meta-analyses

One of the largest warfarin-related studies to date is the International Warfarin Pharmacogenetics Consortium (IWPC) study in which 21 research groups from 9 countries contributed individual patient data for a total of 5700 warfarin-treated patients ([17](#R17)). As IWPC was a secondary study, it did not fit our eligibility criteria for inclusion, but we felt it important to include data from IWPC where it had not been reported in any of the included papers. However, as it was not possible to identify which of the 21 datasets in IWPC corresponded to which included study report, the IWPC data was ignored from the primary meta-analyses but secondary meta-analyses in which eligible IWPC sites were included (while excluding all studies whose population came from a site that was part of IWPC to avoid duplication) were conducted. It was not possible to assess the methodological quality of the IWPC datasets.

#### Confidence in cumulative evidence

The strength of the body of evidence and the quality and strength of recommendations was assessed according to the Venice interim criteria ([18](#R18)).

## RESULTS

### Study selection and characteristics

[Figure 1](#F1) depicts the literature search and selection process. Over 150 SNPs across >18 genes and >25 outcomes were investigated by 77 studies. [Table S3](#SD1) provides details of the studies; studies including similar populations are clustered together. Most studies (*n* = 42, 55%) had a retrospective cohort design while others were prospective (*n* = 25, 33%), both retrospective and prospective (*n* = 4, 5%), randomized controlled trials (*n* = 4, 5%) and case-control studies (*n =* 2, 3%). The median number of Black-African patients in the included studies was 115 (IQR: 31–269). Variant-specific details and the associations investigated for each of the primary, secondary and other outcomes in the different studies are provided in [Tables S4](#SD1) (*CYP2C9*), S5 (*VKORC1*), S6 (*CYP4F2*), S7 (other genes) and S8 (other outcomes).

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a386/7217737/9adcdb5d7d9d/nihms-1065643-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7217737_nihms-1065643-f0001.jpg)

Evidence search and selection

### Methodological quality and risk of bias

Qualitative methods were used to assess the methodological rigor of included studies ([11](#R11)) ([Table S9](#SD1) and [Spreadsheet S1](#SD2)). Most did not report using genotype quality control procedures (*n* = 39, 51%) and had not reported whether genotyping personnel were blinded to outcome status (*n* = 56, 73%). The reporting of missing genotype data was low across studies, with none of the studies reporting missing data (n = 38, 49%) conducting checks for missingness at random. Only 15 (20%) studies undertook tests for cryptic population stratification, with 27 (35%) studies not reporting testing for Hardy-Weinberg equilibrium. Only 2 (2.6%) studies provided details and justification of the modes of inheritance utilised. There was also a large variability in outcome definitions. For instance, 40 different ‘stable dose’ definitions were observed in 56 studies ([Table S10](#SD1)). The definitions of time to stable dose (5 different definitions in 5 studies), bleeding events (16 in 17 studies), time to therapeutic INR (6 in 6 studies), warfarin sensitivity and resistance (1 in 1 study) in the included papers are shown in [Table S11](#SD1). Lastly, only 14 (18%) of the trials reported measuring adherence to treatment. Whilst many issues of concern were raised in terms of the methodological quality, no studies stood out in terms of being of particularly low quality overall, and therefore sensitivity analyses based on methodological rigor were not performed.

### Meta-analyses

Forty-eight studies representing 2,336 patients were included in the primary meta-analyses. For the remaining studies, even after contacting authors, data was insufficient to allow their inclusion. Summary results for all included studies are provided in [Table 1](#T1) and [Figures 2](#F2)–[4](#F4) (stable dose), [Figure S1](#SD1) (time to stable dose) and [Figure S2](#SD1) (proportion of time in therapeutic range). [Tables S4](#SD1)–[S8](#SD1) show which studies were excluded and why. Results, if available, for studies that could not be included in the meta-analyses are also summarized ([Tables S4](#SD1)–[S7](#SD1) and [S12](#SD1)–[S14](#SD1)).

### Table 1.

| Variants |  |  |  | Included studies | Total N | Genotype counts |  |  | Pooled estimates (mean differences (95% CI), heterogeneity) |  | Strength of evidence§ |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Gene | rs ID† | Common name | Function† | WT / WT | WT / V | V / V | WT / V vs WT / WT | V / V vs WT / WT |  |  |  |
| CYP2C9 | rs1799853 | *2 | Missense | 13 (5, 21, 37–47) | 2193 | 2054 | 139 | 0 | −6.75 (−8.91; –4.59), I2 = 15% | NA | Moderate |
| rs1057910 | *3 | Missense | 9 (5, 21, 38, 39, 42–44, 46, 47) | 1754 | 1706 | 48 | 0 | −12.51 (−18.18; −6.83), I2 = 62% | NA | Moderate |  |
| rs28371686 | *5 | Missense | 6 (39–41, 43, 46, 47) | 1297 | 1277 | 20 | 0 | −13.38 (−16.68; −10.07), I2 = 0%. | NA | Moderate |  |
| rs9332131 | *6 | Frame-shift | 3 (40, 43, 47) | 1044 | 1027 | 17 | 0 | −8.10 (−15.36; −0.83), I2 = 0% | NA | Moderate |  |
| rs7900194 | *8 | Missense | 7 (39–41, 43, 45, 46, 48) | 929 | 821 | 97 | 11 | −4.54 (−126.77; 117.70), I2 = 0% | −0.01 (−0.05; 0.04), I2 = 0% | Moderate |  |
| rs2256871 | *9 | Missense | 2 (40, 48) | 299 | 242 | 54 | 3 | 0.06 (−4.29; 4.41), I2 = 0% | 17.15 (9.14; 25.16), I2 = 0% | Weak |  |
| rs28371685 | *11 | Missense | 7 (38, 40, 43, 45–48) | 1509 | 1474 | 35 | 0 | −5.31 (−10.18; −0.43), I2 = 0% | NA | Moderate |  |
| VKORC1 | rs9923231 | −1639G>A | Promoter region | 12 (5, 21, 38, 39, 41–47, 49) | 2019 | 1601 | 389 | 29 | −10.28 (−13.25; −7.31), I2 = 23% | −18.13 (−22.33; −13.92), I2 = 64% | Moderate |
| rs9934438 | 1173C>T | Intronic | 5 (39, 40, 50–52) | 713 | 506 | 177 | 30 | −11.14 (−14.76; −7.53), I2 = 0% | −21.56 (−25.92; −17.20), I2 = 0% | Moderate |  |
| rs7294 | 3730G>A | 3’ UTR | 4 (40, 48, 51, 53) | 666 | 222 | 319 | 125 | 4.83 (1.11; 8.55), I2 = 0% | 6.93 (3.48; 10.38), I2 = 2% | Moderate |  |
| rs2359612 | 2255C>T | Intronic | 3 (40, 51, 53) | 535 | 298 | 182 | 55 | −6.40 (−10.05; −2.76), I2 = 0% | −17.30 (−21.86; −12.74), I2 = 47% | Moderate |  |
| rs8050894 | 1542G>C | Intronic | 4 (40, 50, 51, 53) | 663 | 313 | 284 | 66 | −3.77 (−7.49; −0.05), I2 = 30% | −11.66 (−18.91; −4.42), I2 = 60% | Moderate |  |
| rs2884737 | 497T>G | Intronic | 2 (40, 53) | 282 | 185 | 77 | 20 | −8.16 (−12.87; −3.46), I2 = 0% | −19.61 (−24.90; −14.32), I2 = 0% | Moderate¶ |  |
| CYP4F2 | rs2108622 | *3 | Missense | 5 (21, 39, 47, 54, 55) | 990 | 824 | 160 | 6 | 2.64 (−292.45; 297.72), I2 = 0% | 2.46 (−32.97; 37.89), I2 = 0% | Moderate |
| CYP2C cluster | rs12777823 | − | Intergenic | 4 (43, 45, 47, 56) | 970 | 556 | 344 | 70 | −4.40 (−7.38; −1.42), I2 = 23% | −12.74 (−17.58; −7.91), I2 = 34%; | Moderate |
| CALU1 | rs2290228 | 11G>A | Missense | 2 (39, 47) | 293 | 254 | 34 | 5 | −1.53 (−22.47; 25.53), I2 = 52% | NA\|\| | Weak |
| NQO1 | rs1800566 | *2 | Missense | 2 (39, 55) | 303 | 208 | 85 | 10 | −0.18 (−7.22; 6.87), I2 = 0% | 7.63 (−7.33; 22.60), I2 = 0% | Weak |

Table 1 Caption: Summary results for associations between genetic variants and stable warfarin dose (mg/week)

### Figure 2. Forest plots for associations between CYP2C9 and stable warfarin dose.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a386/7217737/13402075ed68/nihms-1065643-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7217737_nihms-1065643-f0002.jpg)

*CYP2C9 star allele, †standard meta-analysis (fixed effects assumed with low heterogeneity (I2 <30%), else random effects), ‡article as data source (otherwise author-provided). CI = confidence intervals; CYP2C9 = cytochrome P450 family 2 subfamily C member 9; MD = mean difference; SD = standard deviation.

### Figure 4. Forest plots for associations between other genes and stable warfarin dose.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a386/7217737/d03710a5d2d5/nihms-1065643-f0006.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7217737_nihms-1065643-f0006.jpg)

*CYP4F2 and NQO1 star alleles, †standard meta-analysis (fixed effects assumed with low heterogeneity (I2 <30%), else random effects), ‡article as data source (otherwise author-provided). CI = confidence intervals; CYP2C = cytochrome P450 family 2 subfamily C; CYP4F2 = cytochrome P450 family 4 subfamily F member 2; MD = mean difference; NQO1 = NAD(P)H quinone dehydrogenase 1; SD = standard deviation.

### Stable dose

Regarding *CYP2C9* and stable dose ([Table 1](#T1), [Figure 2](#F2)), significant predictors included rs1799853 (**2*), rs1057910 (**3*), rs28371686 (**5*), rs9332131 (**6*), and rs28371685 (**11*) with heterozygotes respectively requiring 6.75 (95% CI 4.59; 8.91), 12.51 (6.83; 18.18), 13.38 (10.07; 16.68), 8.10 (0.83; 15.36) and 5.31 (0.43; 10.18) mg/week less warfarin compared to wild-type homozygotes. rs2256871 (*CYP2C9*9*) mutant-type homozygotes on the other hand required 17.15 (9.14; 15.16) mg/week more compared to wild-type homozygotes, although the strength of evidence for this association was considered weak (only 3 mutant-type homozygotes). Only the association between rs1799853 (*CYP2C9*2*) and stable dose met our pre-defined criteria for assessing publication bias (>10 included studies), and for this we did not find any evidence to support it (linear regression test of funnel plot asymmetry p-value = 0.85).

The *VKORC1* variants rs9923231 (*−1639G>A*), rs9934438 (*1173C>T*), rs2359612 (*2255C>T*), rs8050894 (*1542G>C*), and rs2884737 (*497T>G*) also led to reductions in weekly dose requirements: homozygotes for the variant alleles required 18.13 (13.92; 22.33), 21.56 (17.20; 25.92), 17.30 (12.74; 21.86), 11.66 (4.42; 18.91), and 19.61 (14.32; 24.90) mg/week less warfarin, respectively, when compared with wild-type homozygotes. By contrast, heterozygotes required 10.28 (7.31; 13.25), 11.14 (7.53; 14.76), 6.40 (2.76; 10.05), 3.77 (0.05; 7.49), and 8.16 (3.46; 12.87) mg/week less, respectively, compared with wild-type homozygotes ([Table 1](#T1), [Figure 3](#F3)). Conversely, mutant-type homozygotes and heterozygotes for the rs7294 (*3730G>A*) variant required a modest warfarin weekly dose increment of 6.93 (3.48; 10.38) and 4.83 (1.11; 8.55) mg, respectively. Regarding publication bias, only the first genotype-contrast (wild-type homozygotes versus heterozygotes) for the rs9923231 (*−1639G>A*) allele met our pre-defined criteria (four studies had zero weight in the second genotype contrast) and for this, we found some evidence of publication bias (linear regression test of funnel plot asymmetry p-value = 0.05). The trim and fill random effects analysis method estimated that the number of missing studies was two and that these missing trials did not affect the statistical significance of the pooled effect estimate ([Figure S3](#SD1)).

### Figure 3. Forest plots for associations between VKORC1 and stable warfarin dose.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a386/7217737/d1876a1b6fc6/nihms-1065643-f0004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7217737_nihms-1065643-f0004.jpg)

‡article as data source (otherwise author-provided), §Shrif study estimates flipped, first genotype contrast with high heterogeneity so requires cautious interpretation. CI = confidence intervals; MD = mean difference; SD = standard deviation; VKORC1 = vitamin K epoxide reductase complex subunit 1.

For other gene regions ([Table 1](#T1), [Figure 4](#F4)), only the mutant-type homozygotes and heterozygotes for the rs12777823 (*CYP2C* gene cluster) variant required weekly doses that were significantly different from those of the corresponding wild-type homozygotes (respectively 12.74 (7.91; 17.58) and 4.40 (1.42; 7.38) mg less).

The results of the secondary analyses including IWPC sites (coded site_1, site_2, site_5, site_11, site_14, site_16, site_17, site_19, site_20, site_21 and site_22 in the IWPC data and ethnicity datasets ([19](#R19))) ([Figure S4](#SD1)) were similar to those obtained during the primary analyses except for *CYP2C9*6*, *CYP2C9*11*, *VKORC1 2255C>T* and *VKORC1 1542G>C* which were no longer statistically-significant (estimates for heterozygotes versus wild-type homozygotes respectively being −0.45 (−11.87; 10.96), −4.05 (−10.44; 2.35), −3.25 (−6.75; 0.25) and −3.11 (−6.31; 0.10) mg/week). On the other hand, *CYP2C9*8* produced statistically significant estimates for heterozygotes versus wild-type homozygotes (−6.42 (−9.44; −3.31) mg/week) in the pairwise meta-analysis ([Figure S5](#SD1)). Interestingly, the non-significant estimate for variant-type homozygotes versus wild-type homozygotes was in the opposite direction (6.41 (−2.22; 15.05) mg/week). Regarding the strategy used to infer summary data for each *CYP2C9* genotype group and except for **11* which was no longer statistically-significant (heterozygotes versus wild-type homozygotes estimate −3.36 (−9.24; 2.53) mg/week), the results mirrored those of the primary meta-analyses ([Figure S6](#SD1) and [Tables S15](#SD1)–[S16](#SD1)).

Finally, where it was possible to conduct country-specific analyses (≥2 studies included from same country), we carried out sub-group analyses based on country from which participants were recruited. The countries where studies were conducted included the USA (*n* = 40 studies), Brazil (*n* = 5), South Africa (*n* = 2), and Sudan (*n* = 1) ([Table S17](#SD1)). Population-specific analyses produced non-significant estimates for only rs1057910 (*CYP2C9*3*) (Brazil-only studies, wild-type homozygotes versus heterozygotes estimate −6.27 (−14.26; 1.72) mg/week, *I*^2^ = 0%) which differed from the overall pooled estimates which were statistically significant.

### Bleeding events and other outcomes

We could not conduct meta-analyses for this outcome because follow-up time differed across the three studies (28 days ([5](#R5)) versus 2 years ([20](#R20)) versus 5 years ([21](#R21))). In the individual studies, the comparisons between genetic variants and bleeding events were not statistically significant ([Tables S4](#SD1)–[S7](#SD1)). Other outcomes are shown in [Figures S1](#SD1)–[S2](#SD1).

## DISCUSSION

We have comprehensively evaluated the effect of genetic factors that determine warfarin stable dose requirements and other end-points in Black-African patients. We have largely focused on genes involved in warfarin’s pharmacokinetics (*CYP2C9*), and pharmacodynamics (*VKORC1*), all of which have been implicated in determining warfarin response in Whites.

*CYP2C9* is the main metabolizing enzyme for the more potent S-enantiomer. The most commonly studied polymorphisms, rs1799853 (*CYP2C9*2*) and rs1057910 (*CYP2C9*3*), produce protein isoforms with only about 12% and <5% of wild-type enzyme activity ([22](#R22), [23](#R23)). Interestingly, although these polymorphisms are less common in Black-Africans ([7](#R7)), the effect on the reduction in weekly warfarin dose requirement (6.8 and 12.5mg, respectively), was similar to that observed in White patients (3.9 and 12.5 mg/week less) ([24](#R24)), indicating that these polymorphisms should not be excluded from dosing algorithms.

Other *CYP2C9* polymorphisms (rs28371686 (*CYP2C9*5*), rs9332131 (*CYP2C9*6*), and rs28371685 (*CYP2C9**11)) which are more prevalent in Black-Africans ([7](#R7)) led to reductions in warfarin weekly dose by 13.4, 8.1, and 5.3 mg respectively. These polymorphisms also lead to reduced ([25](#R25)), null ([26](#R26)), and reduced ([27](#R27)) catalytic function, respectively. *CYP2C9*8* (rs7900194) heterozygotes also required decreased weekly warfarin dose (4.5mg) as predicted by the functional effects of the allele ([28](#R28)), but this did not reach statistical significance. This could be attributed to the mutant-type homozygotes (*n* = 7) who required higher warfarin doses as shown in the bivariate sensitivity analysis. *CYP2C9*9* (rs2256871), despite the fact that it results in a change from histidine to arginine at position 251, has minimal effect on protein function ([8](#R8)). Thus, the higher warfarin dose requirements (17.2 mg/week) for the 3 mutant-type heterozygotes should be interpreted with caution, given the small sample size.

The *VKORC1* variants rs9923231 (−*1639G>A*), rs9934438 (*1173C>T*), rs2359612 (*2255C>T*), rs8050894 (*1542G>C*), and rs2884737 (*497T>G*) also led to reductions in weekly dose requirements by up to 18.1, 21.6, 17.3, 11.7, and 19.6 mg, respectively. Some of these results are similar to those previously observed in White patients ([24](#R24)). For instance, homozygotes for the rs9923231 and rs9934438 variant alleles required 20.0 and 22.0 mg/week less warfarin respectively compared to wild-type homozygotes (comparable to 18.1 and 21.6 mg/week in Black-African patients). This is biologically plausible for rs9923231 (*−1639G>A*), which is part of an enhancer box (E-box) consensus sequence CANNTG that may function as a repressor binding site ([29](#R29)). The G>A polymorphism leads to reduced transcription and decreased warfarin requirements ([29](#R29), [30](#R30)). However, this mechanism is yet to be confirmed, a process complicated by the fact that this variant is in near perfect linkage disequilibrium (LD) with several other variants including the intronic rs9934438 (*1173C>T*) variant also investigated in this study. The functions of the intronic rs2359612 (*2255C>T*), rs8050894 (*1542G>C*), and rs2884737 (*497T>G*) variants are also unknown. The 1000 genomes population frequencies of these *VKORC1* variants (rs9923231, rs9934438, rs2359612, rs8050894 and rs2884737) are respectively 0.05, 0.05, 0.18, 0.26 and 0.01 in Black-Africans compared with 0.39, 0.39, 0.39, 0.40 and 0.26 in individuals of European ancestry ([7](#R7)). Two (rs9923231, rs9934438) are in LD (r^2^ > 0.9) in the Black-Africans whereas four (rs9923231, rs9934438, rs8050894, rs2359612) are in LD in Europeans.

The rs7294 (*VKORC1 3730G>A*) variant (population frequency 0.45 in Black-Africans, 0.37 in Europeans ([7](#R7))) increased weekly warfarin requirements by up to 6.9 mg. It is located in the 3’-untranslated region (3’ UTR), which can be targeted by microRNAs resulting in gene silencing either by translational repression or by mRNA degradation. For instance, *miR-133a*, which targets this region, has been previously implicated in *VKORC1* regulation ([31](#R31)). Specifically, the G>A mutation decreases the binding capacity of *miR-133a* leading to decreased translational repression and more copies of *VKORC1* mRNA which would lead to higher warfarin dose requirements as we observed.

In addition, rs12777823 (*CYP2C* cluster) was also associated with stable dose. Although, its role is currently unknown, it is associated with warfarin clearance in Blacks ([32](#R32)). Despite being common in other populations (respective allelic frequencies of 0.15 and 0.31 in European and East Asian 1000 genomes ([7](#R7)) populations), this effect is observed in Africans only (allelic frequency of 0.25), suggesting that it may be in linkage disequilibrium with an unknown causal variant ([32](#R32)).

Even though *CALU1* and *NQO1* are thought to be involved in warfarin’s mechanism of action (respectively binding to the vitamin K epoxide reductase complex and potentially reducing the quinone form of vitamin K) ([33](#R33)), the missense variants that we included in the meta-analyses (respectively rs2290228 and rs1800566) were not significantly associated with stable dose requirements. However, the strength of evidence for these associations was considered weak given the small sample sizes (only two studies included for each meta-analysis).

We did not conduct meta-analyses for bleeding events. Although the individual studies did not show an effect of genetic factors on the risk of bleeding, this may merely reflect a lack of power. Indeed, Limdi et al. ([20](#R20)) found that rs1057910 (*CYP2C9*3*) increased the risk of bleeding (hazard ratio 1.85) when African American data was combined with European American data.

Several limitations of this study should be acknowledged. First, despite our comprehensive search strategy, we could not undertake some meta-analyses because of inadequate reporting. This was confounded by a low response rate (30.3%, 10 of 33 authors) when we requested additional information. However, the information obtained from the 10 individual authors was extensive, representing 39 studies and 10 abstracts. Consequently, only 33 (35.1%) of 94 studies could not be included in the quantitative syntheses. Although no studies stood out in terms of being of particularly low quality overall, there were many methodological issues of concern including heterogeneity in study populations and outcomes. Most studies included African Americans and Brazilians who are of West African ancestry, and so generalization to other sub-Saharan African populations should be done with caution. Another issue that needs to be taken into account is the degree of admixture with European and Ameridian populations ([34](#R34)) in the Black populations studied so far, which was not evaluated in most of the individual studies. Despite these concerns, the sub-group analyses showed that most pooled estimates were not significantly affected by the sub-populations involved. Finally, we excluded studies that did not report ‘Black’ participants and we could have therefore missed important data. For instance, we excluded 9 Egyptian studies during full-text screening. Our decision to exclude Egyptians could be justified by a genome-wide association study that revealed that North African populations share more genomic ancestry with some Asian populations compared to those from sub-Saharan Africa ([35](#R35)).

In conclusion, our systematic review provides a quantitative estimate of the effect of different genetic variants on warfarin weekly dose requirements in Black-African patients. By showing that some variants that are more prevalent in Black-Africans may be important determinants of warfarin weekly dose requirements, this review has further demonstrated the importance of population-specific dosing algorithms. Moreover, the total number of Black patients studied (*n* = 2,336) is much lower compared to Whites (*n* > 5400 as of December 19, 2007 ([36](#R36))) and many of the studies were conducted in the US and Brazil, where there is a significant degree of admixture. This further emphasizes the fact that the number of studies conducted in Africa is small, which is worrisome given that warfarin is the most commonly used anticoagulant on this continent. In response to the poor quality of anticoagulation in sub-Saharan Africa, we have recently embarked on a collaborative project in Uganda and South Africa (War-PATH: WARfarin anticoagulation in PATients in Sub-SaHaran Africa; [http://warpath.info/](http://warpath.info/)) with the main aim of identifying clinical and genetic factors determining warfarin dose variability and ultimately develop better clinical and genetic dosing algorithms to improve anticoagulation quality.

## Supplementary Material

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

Warfarin dosing requirements vary due to clinical and genetic factors.

### What question did this study address?

What are the genetic factors affecting warfarin dosing in Black-African patients?

### What does this study add to our knowledge?

This paper provides a quantitative estimate of the effect of different genetic variants on weekly warfarin dose requirements in Black-African patients.

### How might this change clinical pharmacology or translational science?

Most of the work in genomics, including pharmacogenomics, has been undertaken in White patients. This paper therefore provides valuable insights into what has been done in Black-African patients, and where further work needs to be undertaken. Understanding important ethnicity-specific genetic factors and incorporating them in warfarin dosing algorithms should ultimately improve anticoagulation quality for an underrepresented patient group.

## ACKNOWLEDGEMENTS

I.G.A. thanks the University of Liverpool for studentship funding support.

FUNDING

This research was commissioned by the National Institute for Health Research (NIHR) Global Health Research Group on Warfarin anticoagulation in patients with cardiovascular disease in sub-Saharan Africa [ref: 16/137/101] using UK aid from the UK Government. The views expressed in this publication are those of the author(s) and not necessarily those of the NIHR or the Department of Health and Social Care. S.A.L. is supported by NIH grant 1R01HL139731 and American Heart Association 18SFRN34250007. V.K.K. is funded by NIH/NIAMS K23GM117395.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. World Health Organization. (2018). Global Health Estimates 2016, Deaths by Cause, Age, Sex, by Country and by Region, 2000–2016 (2018).

2. Mouton JP et al. Adverse Drug Reactions Causing Admission to Medical Wards: A Cross-Sectional Survey at 4 Hospitals in South Africa. Medicine (Baltimore) 95, e3437 (2016).  [DOI](https://doi.org/10.1097/MD.0000000000003437) | [PMC free article](/articles/PMC4902486/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27175644/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Medicine%20(Baltimore)&title=Adverse%20Drug%20Reactions%20Causing%20Admission%20to%20Medical%20Wards:%20A%20Cross-Sectional%20Survey%20at%204%20Hospitals%20in%20South%20Africa&author=JP%20Mouton&volume=95&publication_year=2016&pages=e3437&pmid=27175644&doi=10.1097/MD.0000000000003437&)

3. Lee MT & Klein TE Pharmacogenetics of warfarin: challenges and opportunities. J Hum Genet 58, 334–8 (2013).  [DOI](https://doi.org/10.1038/jhg.2013.40) | [PMC free article](/articles/PMC4074513/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23657428/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=Pharmacogenetics%20of%20warfarin:%20challenges%20and%20opportunities&author=MT%20Lee&author=TE%20Klein&volume=58&publication_year=2013&pages=334-8&pmid=23657428&doi=10.1038/jhg.2013.40&)

4. Cavallari LH & Perera MA The future of warfarin pharmacogenetics in under-represented minority groups. Future Cardiol 8, 563–76 (2012).  [DOI](https://doi.org/10.2217/fca.12.31) | [PMC free article](/articles/PMC3463230/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22871196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Future%20Cardiol&title=The%20future%20of%20warfarin%20pharmacogenetics%20in%20under-represented%20minority%20groups&author=LH%20Cavallari&author=MA%20Perera&volume=8&publication_year=2012&pages=563-76&pmid=22871196&doi=10.2217/fca.12.31&)

5. Kimmel SE et al. A pharmacogenetic versus a clinical algorithm for warfarin dosing. N Engl J Med 369, 2283–93 (2013).  [DOI](https://doi.org/10.1056/NEJMoa1310669) | [PMC free article](/articles/PMC3942158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24251361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=A%20pharmacogenetic%20versus%20a%20clinical%20algorithm%20for%20warfarin%20dosing&author=SE%20Kimmel&volume=369&publication_year=2013&pages=2283-93&pmid=24251361&doi=10.1056/NEJMoa1310669&)

6. Johnson JA et al. Clinical Pharmacogenetics Implementation Consortium Guidelines for CYP2C9 and VKORC1 genotypes and warfarin dosing. Clin Pharmacol Ther 90, 625–9 (2011).  [DOI](https://doi.org/10.1038/clpt.2011.185) | [PMC free article](/articles/PMC3187550/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21900891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20Guidelines%20for%20CYP2C9%20and%20VKORC1%20genotypes%20and%20warfarin%20dosing&author=JA%20Johnson&volume=90&publication_year=2011&pages=625-9&pmid=21900891&doi=10.1038/clpt.2011.185&)

7. Genomes Project C et al. A global reference for human genetic variation. Nature 526, 68–74 (2015).  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation&author=C%20Genomes%20Project&volume=526&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

8. Pratt VM et al. Recommendations for Clinical CYP2C9 Genotyping Allele Selection: A Joint Recommendation of the Association for Molecular Pathology and College of American Pathologists. J Mol Diagn, (2019).  [DOI](https://doi.org/10.1016/j.jmoldx.2019.04.003) | [PMC free article](/articles/PMC7057225/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31075510/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Diagn&title=Recommendations%20for%20Clinical%20CYP2C9%20Genotyping%20Allele%20Selection:%20A%20Joint%20Recommendation%20of%20the%20Association%20for%20Molecular%20Pathology%20and%20College%20of%20American%20Pathologists&author=VM%20Pratt&publication_year=2019&pmid=31075510&doi=10.1016/j.jmoldx.2019.04.003&)

9. JPT H & Green S, e. Cochrane Handbook for Systematic Reviews of Interventions Version 5.1.0 [updated March 2011]. (ed. The Cochrane Collaboration; ) (Online, 2011).  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Cochrane%20Handbook%20for%20Systematic%20Reviews%20of%20Interventions%20Version%205.1.0%20%5Bupdated%20March%202011%5D&author=H%20JPT&author=S,%20e.%20Green&publication_year=2011&)

10. Little J & (editors), H. J The HuGENet™ HuGE Review Handbook, version 1.0 (Online, 2006).  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20HuGENet%E2%84%A2%20HuGE%20Review%20Handbook,%20version%201.0&author=J%20Little&author=J%20H.&publication_year=2006&)

11. Jorgensen AL & Williamson PR Methodological quality of pharmacogenetic studies: issues of concern. Stat Med 27, 6547–69 (2008).  [DOI](https://doi.org/10.1002/sim.3420) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18837075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stat%20Med&title=Methodological%20quality%20of%20pharmacogenetic%20studies:%20issues%20of%20concern&author=AL%20Jorgensen&author=PR%20Williamson&volume=27&publication_year=2008&pages=6547-69&pmid=18837075&doi=10.1002/sim.3420&)

12. Minelli C, Thompson JR, Abrams KR, Thakkinstian A & Attia J The choice of a genetic model in the meta-analysis of molecular association studies. Int J Epidemiol 34, 1319–28 (2005).  [DOI](https://doi.org/10.1093/ije/dyi169) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16115824/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Epidemiol&title=The%20choice%20of%20a%20genetic%20model%20in%20the%20meta-analysis%20of%20molecular%20association%20studies&author=C%20Minelli&author=JR%20Thompson&author=KR%20Abrams&author=A%20Thakkinstian&author=J%20Attia&volume=34&publication_year=2005&pages=1319-28&pmid=16115824&doi=10.1093/ije/dyi169&)

13. StataCorp. Stata Statistical Software: Release 14. . (StataCorp LP., College Station, TX, 2015).  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Stata%20Statistical%20Software:%20Release%2014&publication_year=2015&)

14. R Core Team. R: A Language and Environment for Statistical Computing. (R Foundation for Statistical Computing, Online, 2018).  [Google Scholar](https://scholar.google.com/scholar_lookup?title=R:%20A%20Language%20and%20Environment%20for%20Statistical%20Computing&publication_year=2018&)

15. Schwarzer G meta: An R package for meta-analysis. R News 7, 40–5 (2007).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=R%20News&title=meta:%20An%20R%20package%20for%20meta-analysis&author=G%20Schwarzer&volume=7&publication_year=2007&pages=40-5&)

16. Viechtbauer W Conducting meta-analyses in R with the metafor package. Journal of Statistical Software 36, 1–48 (2010).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&title=Conducting%20meta-analyses%20in%20R%20with%20the%20metafor%20package&author=W%20Viechtbauer&volume=36&publication_year=2010&pages=1-48&)

17. International Warfarin Pharmacogenetics Consortium et al. Estimation of the warfarin dose with clinical and pharmacogenetic data. N Engl J Med 360, 753–64 (2009).  [DOI](https://doi.org/10.1056/NEJMoa0809329) | [PMC free article](/articles/PMC2722908/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19228618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Estimation%20of%20the%20warfarin%20dose%20with%20clinical%20and%20pharmacogenetic%20data&volume=360&publication_year=2009&pages=753-64&pmid=19228618&doi=10.1056/NEJMoa0809329&)

18. Ioannidis JP et al. Assessment of cumulative evidence on genetic associations: interim guidelines. Int J Epidemiol 37, 120–32 (2008).  [DOI](https://doi.org/10.1093/ije/dym159) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17898028/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Epidemiol&title=Assessment%20of%20cumulative%20evidence%20on%20genetic%20associations:%20interim%20guidelines&author=JP%20Ioannidis&volume=37&publication_year=2008&pages=120-32&pmid=17898028&doi=10.1093/ije/dym159&)

19. International Warfarin Pharmacogenetics Consortium. Data from Estimation of the warfarin dose with clinical and pharmacogenetic data. (ed. Consortium I.W.P.) (PharmGKB, Online, 2009).  [DOI](https://doi.org/10.1016/j.jvn.2009.09.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19914573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Data%20from%20Estimation%20of%20the%20warfarin%20dose%20with%20clinical%20and%20pharmacogenetic%20data&publication_year=2009&)

20. Limdi NA, Brown TM, Shendre A, Liu N, Hill CE & Beasley TM Quality of anticoagulation control and hemorrhage risk among African American and European American warfarin users. Pharmacogenet Genomics 27, 347–55 (2017).  [DOI](https://doi.org/10.1097/FPC.0000000000000298) | [PMC free article](/articles/PMC5657532/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28806200/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Quality%20of%20anticoagulation%20control%20and%20hemorrhage%20risk%20among%20African%20American%20and%20European%20American%20warfarin%20users&author=NA%20Limdi&author=TM%20Brown&author=A%20Shendre&author=N%20Liu&author=CE%20Hill&volume=27&publication_year=2017&pages=347-55&pmid=28806200&doi=10.1097/FPC.0000000000000298&)

21. Kawai VK et al. Genotype and risk of major bleeding during warfarin treatment. Pharmacogenomics 15, 1973–83 (2014).  [DOI](https://doi.org/10.2217/pgs.14.153) | [PMC free article](/articles/PMC4304738/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25521356/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genotype%20and%20risk%20of%20major%20bleeding%20during%20warfarin%20treatment&author=VK%20Kawai&volume=15&publication_year=2014&pages=1973-83&pmid=25521356&doi=10.2217/pgs.14.153&)

22. Rettie AE, Wienkers LC, Gonzalez FJ, Trager WF & Korzekwa KR Impaired (S)-warfarin metabolism catalysed by the R144C allelic variant of CYP2C9. Pharmacogenetics 4, 39–42 (1994).  [DOI](https://doi.org/10.1097/00008571-199402000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8004131/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Impaired%20(S)-warfarin%20metabolism%20catalysed%20by%20the%20R144C%20allelic%20variant%20of%20CYP2C9&author=AE%20Rettie&author=LC%20Wienkers&author=FJ%20Gonzalez&author=WF%20Trager&author=KR%20Korzekwa&volume=4&publication_year=1994&pages=39-42&pmid=8004131&doi=10.1097/00008571-199402000-00005&)

23. Haining RL, Hunter AP, Veronese ME, Trager WF & Rettie AE Allelic variants of human cytochrome P450 2C9: baculovirus-mediated expression, purification, structural characterization, substrate stereoselectivity, and prochiral selectivity of the wild-type and I359L mutant forms. Arch Biochem Biophys 333, 447–58 (1996).  [DOI](https://doi.org/10.1006/abbi.1996.0414) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8809086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Biochem%20Biophys&title=Allelic%20variants%20of%20human%20cytochrome%20P450%202C9:%20baculovirus-mediated%20expression,%20purification,%20structural%20characterization,%20substrate%20stereoselectivity,%20and%20prochiral%20selectivity%20of%20the%20wild-type%20and%20I359L%20mutant%20forms&author=RL%20Haining&author=AP%20Hunter&author=ME%20Veronese&author=WF%20Trager&author=AE%20Rettie&volume=333&publication_year=1996&pages=447-58&pmid=8809086&doi=10.1006/abbi.1996.0414&)

24. Jorgensen AL, FitzGerald RJ, Oyee J, Pirmohamed M & Williamson PR Influence of CYP2C9 and VKORC1 on patient response to warfarin: a systematic review and meta-analysis. PLoS One 7, e44064 (2012).  [DOI](https://doi.org/10.1371/journal.pone.0044064) | [PMC free article](/articles/PMC3430615/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22952875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Influence%20of%20CYP2C9%20and%20VKORC1%20on%20patient%20response%20to%20warfarin:%20a%20systematic%20review%20and%20meta-analysis&author=AL%20Jorgensen&author=RJ%20FitzGerald&author=J%20Oyee&author=M%20Pirmohamed&author=PR%20Williamson&volume=7&publication_year=2012&pages=e44064&pmid=22952875&doi=10.1371/journal.pone.0044064&)

25. Niinuma Y et al. Functional characterization of 32 CYP2C9 allelic variants. Pharmacogenomics J 14, 107–14 (2014).  [DOI](https://doi.org/10.1038/tpj.2013.22) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23752738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Functional%20characterization%20of%2032%20CYP2C9%20allelic%20variants&author=Y%20Niinuma&volume=14&publication_year=2014&pages=107-14&pmid=23752738&doi=10.1038/tpj.2013.22&)

26. Kidd RS, Curry TB, Gallagher S, Edeki T, Blaisdell J & Goldstein JA Identification of a null allele of CYP2C9 in an African-American exhibiting toxicity to phenytoin. Pharmacogenetics 11, 803–8 (2001).  [DOI](https://doi.org/10.1097/00008571-200112000-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11740344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Identification%20of%20a%20null%20allele%20of%20CYP2C9%20in%20an%20African-American%20exhibiting%20toxicity%20to%20phenytoin&author=RS%20Kidd&author=TB%20Curry&author=S%20Gallagher&author=T%20Edeki&author=J%20Blaisdell&volume=11&publication_year=2001&pages=803-8&pmid=11740344&doi=10.1097/00008571-200112000-00008&)

27. Tai G et al. In-vitro and in-vivo effects of the CYP2C9*11 polymorphism on warfarin metabolism and dose. Pharmacogenet Genomics 15, 475–81 (2005).  [DOI](https://doi.org/10.1097/01.fpc.0000162005.80857.98) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15970795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=In-vitro%20and%20in-vivo%20effects%20of%20the%20CYP2C9*11%20polymorphism%20on%20warfarin%20metabolism%20and%20dose&author=G%20Tai&volume=15&publication_year=2005&pages=475-81&pmid=15970795&doi=10.1097/01.fpc.0000162005.80857.98&)

28. Liu Y et al. Decreased warfarin clearance associated with the CYP2C9 R150H (*8) polymorphism. Clin Pharmacol Ther 91, 660–5 (2012).  [DOI](https://doi.org/10.1038/clpt.2011.269) | [PMC free article](/articles/PMC3879795/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22378156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Decreased%20warfarin%20clearance%20associated%20with%20the%20CYP2C9%20R150H%20(*8)%20polymorphism&author=Y%20Liu&volume=91&publication_year=2012&pages=660-5&pmid=22378156&doi=10.1038/clpt.2011.269&)

29. Yuan HY et al. A novel functional VKORC1 promoter polymorphism is associated with inter-individual and inter-ethnic differences in warfarin sensitivity. Hum Mol Genet 14, 1745–51 (2005).  [DOI](https://doi.org/10.1093/hmg/ddi180) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15888487/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=A%20novel%20functional%20VKORC1%20promoter%20polymorphism%20is%20associated%20with%20inter-individual%20and%20inter-ethnic%20differences%20in%20warfarin%20sensitivity&author=HY%20Yuan&volume=14&publication_year=2005&pages=1745-51&pmid=15888487&doi=10.1093/hmg/ddi180&)

30. Rieder MJ et al. Effect of VKORC1 haplotypes on transcriptional regulation and warfarin dose. N Engl J Med 352, 2285–93 (2005).  [DOI](https://doi.org/10.1056/NEJMoa044503) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15930419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Effect%20of%20VKORC1%20haplotypes%20on%20transcriptional%20regulation%20and%20warfarin%20dose&author=MJ%20Rieder&volume=352&publication_year=2005&pages=2285-93&pmid=15930419&doi=10.1056/NEJMoa044503&)

31. Perez-Andreu V et al. miR-133a regulates vitamin K 2,3-epoxide reductase complex subunit 1 (VKORC1), a key protein in the vitamin K cycle. Mol Med 18, 1466–72 (2013).  [DOI](https://doi.org/10.2119/molmed.2012.00062) | [PMC free article](/articles/PMC3563709/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23154637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Med&title=miR-133a%20regulates%20vitamin%20K%202,3-epoxide%20reductase%20complex%20subunit%201%20(VKORC1),%20a%20key%20protein%20in%20the%20vitamin%20K%20cycle&author=V%20Perez-Andreu&volume=18&publication_year=2013&pages=1466-72&pmid=23154637&doi=10.2119/molmed.2012.00062&)

32. Perera MA et al. Genetic variants associated with warfarin dose in African-American individuals: a genome-wide association study. Lancet 382, 790–6 (2013).  [DOI](https://doi.org/10.1016/S0140-6736(13)60681-9) | [PMC free article](/articles/PMC3759580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23755828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genetic%20variants%20associated%20with%20warfarin%20dose%20in%20African-American%20individuals:%20a%20genome-wide%20association%20study&author=MA%20Perera&volume=382&publication_year=2013&pages=790-6&pmid=23755828&doi=10.1016/S0140-6736(13)60681-9&)

33. Wadelius M et al. Association of warfarin dose with genes involved in its action and metabolism. Hum Genet 121, 23–34 (2007).  [DOI](https://doi.org/10.1007/s00439-006-0260-8) | [PMC free article](/articles/PMC1797064/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17048007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Association%20of%20warfarin%20dose%20with%20genes%20involved%20in%20its%20action%20and%20metabolism&author=M%20Wadelius&volume=121&publication_year=2007&pages=23-34&pmid=17048007&doi=10.1007/s00439-006-0260-8&)

34. Suarez-Kurtz G & Botton MR Pharmacogenomics of warfarin in populations of African descent. Br J Clin Pharmacol 75, 334–46 (2013).  [DOI](https://doi.org/10.1111/j.1365-2125.2012.04354.x) | [PMC free article](/articles/PMC3579249/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22676711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Pharmacogenomics%20of%20warfarin%20in%20populations%20of%20African%20descent&author=G%20Suarez-Kurtz&author=MR%20Botton&volume=75&publication_year=2013&pages=334-46&pmid=22676711&doi=10.1111/j.1365-2125.2012.04354.x&)

35. Henn BM et al. Genomic ancestry of North Africans supports back-to-Africa migrations. PLoS Genet 8, e1002397 (2012).  [DOI](https://doi.org/10.1371/journal.pgen.1002397) | [PMC free article](/articles/PMC3257290/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22253600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Genomic%20ancestry%20of%20North%20Africans%20supports%20back-to-Africa%20migrations&author=BM%20Henn&volume=8&publication_year=2012&pages=e1002397&pmid=22253600&doi=10.1371/journal.pgen.1002397&)

36. Lindh JD, Holm L, Andersson ML & Rane A Influence of CYP2C9 genotype on warfarin dose requirements--a systematic review and meta-analysis. Eur J Clin Pharmacol 65, 365–75 (2009).  [DOI](https://doi.org/10.1007/s00228-008-0584-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19031075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Influence%20of%20CYP2C9%20genotype%20on%20warfarin%20dose%20requirements--a%20systematic%20review%20and%20meta-analysis&author=JD%20Lindh&author=L%20Holm&author=ML%20Andersson&author=A%20Rane&volume=65&publication_year=2009&pages=365-75&pmid=19031075&doi=10.1007/s00228-008-0584-5&)

37. Kimmel SE et al. Apolipoprotein E genotype and warfarin dosing among Caucasians and African Americans. Pharmacogenomics J 8, 53–60 (2008).  [DOI](https://doi.org/10.1038/sj.tpj.6500445) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17325732/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Apolipoprotein%20E%20genotype%20and%20warfarin%20dosing%20among%20Caucasians%20and%20African%20Americans&author=SE%20Kimmel&volume=8&publication_year=2008&pages=53-60&pmid=17325732&doi=10.1038/sj.tpj.6500445&)

38. Perini JA et al. Pharmacogenetics of warfarin: development of a dosing algorithm for brazilian patients. Clin Pharmacol Ther 84, 722–8 (2008).  [DOI](https://doi.org/10.1038/clpt.2008.166) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18754001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacogenetics%20of%20warfarin:%20development%20of%20a%20dosing%20algorithm%20for%20brazilian%20patients&author=JA%20Perini&volume=84&publication_year=2008&pages=722-8&pmid=18754001&doi=10.1038/clpt.2008.166&)

39. Lubitz SA et al. Comparative performance of gene-based warfarin dosing algorithms in a multiethnic population. J Thromb Haemost 8, 1018–26 (2010).  [DOI](https://doi.org/10.1111/j.1538-7836.2010.03792.x) | [PMC free article](/articles/PMC4441275/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20128861/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Haemost&title=Comparative%20performance%20of%20gene-based%20warfarin%20dosing%20algorithms%20in%20a%20multiethnic%20population&author=SA%20Lubitz&volume=8&publication_year=2010&pages=1018-26&pmid=20128861&doi=10.1111/j.1538-7836.2010.03792.x&)

40. Shrif NE et al. Evaluation of the effects of VKORC1 polymorphisms and haplotypes, CYP2C9 genotypes, and clinical factors on warfarin response in Sudanese patients. Eur J Clin Pharmacol 67, 1119–30 (2011).  [DOI](https://doi.org/10.1007/s00228-011-1060-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21590310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Evaluation%20of%20the%20effects%20of%20VKORC1%20polymorphisms%20and%20haplotypes,%20CYP2C9%20genotypes,%20and%20clinical%20factors%20on%20warfarin%20response%20in%20Sudanese%20patients&author=NE%20Shrif&volume=67&publication_year=2011&pages=1119-30&pmid=21590310&doi=10.1007/s00228-011-1060-1&)

41. Ramos AS et al. Development of a pharmacogenetic-guided warfarin dosing algorithm for Puerto Rican patients. Pharmacogenomics 13, 1937–50 (2012).  [DOI](https://doi.org/10.2217/pgs.12.171) | [PMC free article](/articles/PMC3538136/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23215886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Development%20of%20a%20pharmacogenetic-guided%20warfarin%20dosing%20algorithm%20for%20Puerto%20Rican%20patients&author=AS%20Ramos&volume=13&publication_year=2012&pages=1937-50&pmid=23215886&doi=10.2217/pgs.12.171&)

42. Santos PC et al. Development of a pharmacogenetic-based warfarin dosing algorithm and its performance in Brazilian patients: highlighting the importance of population-specific calibration. Pharmacogenomics 16, 865–76 (2015).  [DOI](https://doi.org/10.2217/pgs.15.48) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26050796/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Development%20of%20a%20pharmacogenetic-based%20warfarin%20dosing%20algorithm%20and%20its%20performance%20in%20Brazilian%20patients:%20highlighting%20the%20importance%20of%20population-specific%20calibration&author=PC%20Santos&volume=16&publication_year=2015&pages=865-76&pmid=26050796&doi=10.2217/pgs.15.48&)

43. Drozda K et al. Poor warfarin dose prediction with pharmacogenetic algorithms that exclude genotypes important for African Americans. Pharmacogenet Genomics 25, 73–81 (2015).  [DOI](https://doi.org/10.1097/FPC.0000000000000108) | [PMC free article](/articles/PMC4280295/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25461246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Poor%20warfarin%20dose%20prediction%20with%20pharmacogenetic%20algorithms%20that%20exclude%20genotypes%20important%20for%20African%20Americans&author=K%20Drozda&volume=25&publication_year=2015&pages=73-81&pmid=25461246&doi=10.1097/FPC.0000000000000108&)

44. Liu N et al. Influence of common and rare genetic variation on warfarin dose among African-Americans and European-Americans using the exome array. Pharmacogenomics 18, 1059–73 (2017).  [DOI](https://doi.org/10.2217/pgs-2017-0046) | [PMC free article](/articles/PMC5619051/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28686080/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Influence%20of%20common%20and%20rare%20genetic%20variation%20on%20warfarin%20dose%20among%20African-Americans%20and%20European-Americans%20using%20the%20exome%20array&author=N%20Liu&volume=18&publication_year=2017&pages=1059-73&pmid=28686080&doi=10.2217/pgs-2017-0046&)

45. Ndadza A et al. Warfarin Dose and CYP2C Gene Cluster: An African Ancestral-Specific Variant Is a Strong Predictor of Dose in Black South African Patients. OMICS A Journal of Integrative Biology 23, 36–44 (2019).  [DOI](https://doi.org/10.1089/omi.2018.0174) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30566377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=OMICS%20A%20Journal%20of%20Integrative%20Biology&title=Warfarin%20Dose%20and%20CYP2C%20Gene%20Cluster:%20An%20African%20Ancestral-Specific%20Variant%20Is%20a%20Strong%20Predictor%20of%20Dose%20in%20Black%20South%20African%20Patients&author=A%20Ndadza&volume=23&publication_year=2019&pages=36-44&pmid=30566377&doi=10.1089/omi.2018.0174&)

46. Hernandez W et al. Integrated analysis of genetic variation and gene expression reveals novel variant for increased warfarin dose requirement in African Americans. J Thromb Haemost 15, 735–43 (2017).  [DOI](https://doi.org/10.1111/jth.13639) | [PMC free article](/articles/PMC5862636/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28135054/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Haemost&title=Integrated%20analysis%20of%20genetic%20variation%20and%20gene%20expression%20reveals%20novel%20variant%20for%20increased%20warfarin%20dose%20requirement%20in%20African%20Americans&author=W%20Hernandez&volume=15&publication_year=2017&pages=735-43&pmid=28135054&doi=10.1111/jth.13639&)

47. Limdi NA et al. Race influences warfarin dose changes associated with genetic factors. Blood 126, 539–45 (2015).  [DOI](https://doi.org/10.1182/blood-2015-02-627042) | [PMC free article](/articles/PMC4513254/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26024874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Race%20influences%20warfarin%20dose%20changes%20associated%20with%20genetic%20factors&author=NA%20Limdi&volume=126&publication_year=2015&pages=539-45&pmid=26024874&doi=10.1182/blood-2015-02-627042&)

48. Mitchell C, Gregersen N & Krause A Novel CYP2C9 and VKORC1 gene variants associated with warfarin dosage variability in the South African black population. Pharmacogenomics 12, 953–63 (2011).  [DOI](https://doi.org/10.2217/pgs.11.36) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21635147/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Novel%20CYP2C9%20and%20VKORC1%20gene%20variants%20associated%20with%20warfarin%20dosage%20variability%20in%20the%20South%20African%20black%20population&author=C%20Mitchell&author=N%20Gregersen&author=A%20Krause&volume=12&publication_year=2011&pages=953-63&pmid=21635147&doi=10.2217/pgs.11.36&)

49. Schelleman H et al. Warfarin response and vitamin K epoxide reductase complex 1 in African Americans and Caucasians. Clin Pharmacol Ther 81, 742–7 (2007).  [DOI](https://doi.org/10.1038/sj.clpt.6100144) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17329985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Warfarin%20response%20and%20vitamin%20K%20epoxide%20reductase%20complex%201%20in%20African%20Americans%20and%20Caucasians&author=H%20Schelleman&volume=81&publication_year=2007&pages=742-7&pmid=17329985&doi=10.1038/sj.clpt.6100144&)

50. Wang D, Chen H, Momary KM, Cavallari LH, Johnson JA & Sadee W Regulatory polymorphism in vitamin K epoxide reductase complex subunit 1 (VKORC1) affects gene expression and warfarin dose requirement. Blood 112, 1013–21 (2008).  [DOI](https://doi.org/10.1182/blood-2008-03-144899) | [PMC free article](/articles/PMC2515137/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18523153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Regulatory%20polymorphism%20in%20vitamin%20K%20epoxide%20reductase%20complex%20subunit%201%20(VKORC1)%20affects%20gene%20expression%20and%20warfarin%20dose%20requirement&author=D%20Wang&author=H%20Chen&author=KM%20Momary&author=LH%20Cavallari&author=JA%20Johnson&volume=112&publication_year=2008&pages=1013-21&pmid=18523153&doi=10.1182/blood-2008-03-144899&)

51. Limdi NA et al. Influence of CYP2C9 and VKORC1 on warfarin dose, anticoagulation attainment and maintenance among European-Americans and African-Americans. Pharmacogenomics 9, 511–26 (2008).  [DOI](https://doi.org/10.2217/14622416.9.5.511) | [PMC free article](/articles/PMC2757655/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18466099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Influence%20of%20CYP2C9%20and%20VKORC1%20on%20warfarin%20dose,%20anticoagulation%20attainment%20and%20maintenance%20among%20European-Americans%20and%20African-Americans&author=NA%20Limdi&volume=9&publication_year=2008&pages=511-26&pmid=18466099&doi=10.2217/14622416.9.5.511&)

52. Shendre A, Beasley TM, Brown TM, Hill CE, Arnett DK & Limdi NA Influence of regular physical activity on warfarin dose and risk of hemorrhagic complications. Pharmacotherapy 34, 545–54 (2014).  [DOI](https://doi.org/10.1002/phar.1401) | [PMC free article](/articles/PMC4109410/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25032265/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Influence%20of%20regular%20physical%20activity%20on%20warfarin%20dose%20and%20risk%20of%20hemorrhagic%20complications&author=A%20Shendre&author=TM%20Beasley&author=TM%20Brown&author=CE%20Hill&author=DK%20Arnett&volume=34&publication_year=2014&pages=545-54&pmid=25032265&doi=10.1002/phar.1401&)

53. Perera MA et al. The missing association: sequencing-based discovery of novel SNPs in VKORC1 and CYP2C9 that affect warfarin dose in African Americans. Clin Pharmacol Ther 89, 408–15 (2011).  [DOI](https://doi.org/10.1038/clpt.2010.322) | [PMC free article](/articles/PMC3625373/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21270790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20missing%20association:%20sequencing-based%20discovery%20of%20novel%20SNPs%20in%20VKORC1%20and%20CYP2C9%20that%20affect%20warfarin%20dose%20in%20African%20Americans&author=MA%20Perera&volume=89&publication_year=2011&pages=408-15&pmid=21270790&doi=10.1038/clpt.2010.322&)

54. Perini JA, Struchiner CJ, Silva-Assuncao E & Suarez-Kurtz G Impact of CYP4F2 rs2108622 on the stable warfarin dose in an admixed patient cohort. Clin Pharmacol Ther 87, 417–20 (2010).  [DOI](https://doi.org/10.1038/clpt.2009.307) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20182420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Impact%20of%20CYP4F2%20rs2108622%20on%20the%20stable%20warfarin%20dose%20in%20an%20admixed%20patient%20cohort&author=JA%20Perini&author=CJ%20Struchiner&author=E%20Silva-Assuncao&author=G%20Suarez-Kurtz&volume=87&publication_year=2010&pages=417-20&pmid=20182420&doi=10.1038/clpt.2009.307&)

55. Bress A, Patel SR, Perera MA, Campbell RT, Kittles RA & Cavallari LH Effect of NQO1 and CYP4F2 genotypes on warfarin dose requirements in Hispanic-Americans and African-Americans. Pharmacogenomics 13, 1925–35 (2012).  [DOI](https://doi.org/10.2217/pgs.12.164) | [PMC free article](/articles/PMC3586586/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23215885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Effect%20of%20NQO1%20and%20CYP4F2%20genotypes%20on%20warfarin%20dose%20requirements%20in%20Hispanic-Americans%20and%20African-Americans&author=A%20Bress&author=SR%20Patel&author=MA%20Perera&author=RT%20Campbell&author=RA%20Kittles&volume=13&publication_year=2012&pages=1925-35&pmid=23215885&doi=10.2217/pgs.12.164&)

56. Hernandez W et al. Ethnicity-specific pharmacogenetics: the case of warfarin in African Americans. Pharmacogenomics J 14, 223–8 (2014).  [DOI](https://doi.org/10.1038/tpj.2013.34) | [PMC free article](/articles/PMC4016191/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24018621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Ethnicity-specific%20pharmacogenetics:%20the%20case%20of%20warfarin%20in%20African%20Americans&author=W%20Hernandez&volume=14&publication_year=2014&pages=223-8&pmid=24018621&doi=10.1038/tpj.2013.34&)
