# Shared Genetic Risk Factors Across Carbamazepine‐Induced Hypersensitivity Reactions

## Metadata
**Authors:** Paola Nicoletti, Sarah Barrett, Laurence McEvoy, Ann K Daly, Guruprasad Aithal, M Isabel Lucena, Raul J Andrade, Mia Wadelius, Pär Hallberg, Camilla Stephens, Einar S Bjornsson, Peter Friedmann, Kati Kainu, Tarja Laitinen, Anthony Marson, Mariam Molokhia, Elizabeth Phillips, Werner Pichler, Antonino Romano, Neil Shear, Graeme Sills, Luciana K Tanno, Ashley Swale, Aris Floratos, Yufeng Shen, Matthew R Nelson, Paul B Watkins, Mark J Daly, Andrew P Morris, Ana Alfirevic, Munir Pirmohamed
**Journal:** Clinical Pharmacology and Therapeutics
**Date:** 2019 Jul 3
**DOI:** [10.1002/cpt.1493](https://doi.org/10.1002/cpt.1493)
**PMID:** 31066027
**PMCID:** PMC7156285
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7156285/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7156285/pdf/CPT-106-1028.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7156285/pdf/CPT-106-1028.pdf)

## Abstract

Carbamazepine (CBZ) causes life‐threating T‐cell‐mediated hypersensitivity reactions, including serious cutaneous adverse reactions (SCARs) and drug‐induced liver injury (CBZ‐DILI). In order to evaluate shared or phenotype‐specific genetic predisposing factors for CBZ hypersensitivity reactions, we performed a meta‐analysis of two genomewide association studies (GWAS) on a total of 43 well‐phenotyped Northern and Southern European CBZ‐SCAR cases and 10,701 population controls and a GWAS on 12 CBZ‐DILI cases and 8,438 ethnically matched population controls. HLA‐A*31:01 was identified as the strongest genetic predisposing factor for both CBZ‐SCAR (odds ratio (OR) = 8.0; 95% CI 4.10–15.80; P = 1.2 × 10−9) and CBZ‐DILI (OR = 7.3; 95% CI 2.47–23.67; P = 0.0004) in European populations. The association with HLA‐A*31:01 in patients with SCAR was mainly driven by hypersensitivity syndrome (OR = 12.9; P = 2.1 × 10−9) rather than by Stevens‐Johnson syndrome/toxic epidermal necrolysis cases, which showed an association with HLA‐B*57:01. We also identified a novel risk locus mapping to ALK only for CBZ‐SCAR cases, which needs replication in additional cohorts and functional evaluation.

☑ Carbamazepine (CBZ) is associated with serious, and sometimes fatal, cutaneous and liver adverse reactions. Genomewide profiling has shown that these predisposing factors largely reside in the HLA region (*HLA‐B*15:02* and *HLA‐A*31:01*) consistent with the immune pathogenesis.

☑ What are the genetic predisposing factors in Northern and Southern European populations for CBZ‐induced hypersensitivity reactions affecting both the skin and liver?

☑ Genetic profiling confirmed that *HLA‐A*31:01* predisposes to serious cutaneous adverse reaction (SCAR) in both Northern and Southern European populations. *HLA‐A*31:01* also seems to predispose to CBZ‐induced liver injury. In addition, an uncommon variant in the *ALK* gene was associated with an increased risk of CBZ‐SCAR.

☑ Our study adds to the overwhelming evidence of the role of *HLA‐A*31:01* in predisposing to a variety of CBZ hypersensitivity phenotypes and highlights the need to implement its pre‐prescription and preemptive use in clinical settings.

Carbamazepine (CBZ) is prescribed for epilepsy, trigeminal neuralgia, and bipolar disorder.[1](#cpt1493-bib-0001) In 3–10% of patients, CBZ causes a variety of hypersensitivity reactions,[2](#cpt1493-bib-0002) ranging from mild maculopapular exanthemas to hypersensitivity syndrome, drug reaction with eosinophilia and systemic symptoms (DRESS), acute generalized exanthematous pustulosis (AGEP), Stevens‐Johnson syndrome (SJS), and toxic epidermal necrolysis (TEN); the latter four phenotypes are referred to as serious cutaneous adverse reactions (SCARs). CBZ can also lead to liver injury, which can occur either as part of DRESS or in isolation (the latter is referred to as CBZ‐drug‐induced liver injury (DILI) in this paper).[3](#cpt1493-bib-0003), [4](#cpt1493-bib-0004) These unpredictable clinical phenotypes are T‐cell‐mediated, in which CBZ and/or its metabolites bind specific HLA molecules triggering a T‐cell response.[5](#cpt1493-bib-0005)

CBZ‐induced SJS/TEN is strongly associated with *HLA‐B***15:02* in Han Chinese, Thais, and Malays,[6](#cpt1493-bib-0006) whereas *HLA‐A***24:02* has also recently been identified as a risk factor in Han Chinese.[7](#cpt1493-bib-0007) *HLA‐A*31:01* is associated with a variety of CBZ‐SCAR phenotypes and maculopapular exanthema in Japanese,[8](#cpt1493-bib-0008) Korean,[9](#cpt1493-bib-0009) and European‐descent populations.[10](#cpt1493-bib-0010) Several other HLA loci have been suggested as susceptibility loci, including *HLA‐A*02:06*,[8](#cpt1493-bib-0008) *HLA‐B*15:11*,[11](#cpt1493-bib-0011) and *HLA‐B*51:01*,[12](#cpt1493-bib-0012), [13](#cpt1493-bib-0013), [14](#cpt1493-bib-0014) but the findings have not always been replicated.[15](#cpt1493-bib-0015) Currently, we have limited knowledge of susceptibility loci for CBZ‐DILI.

The purpose of our study was to perform a meta‐analysis between our previously published CBZ hypersensitivity cohort[10](#cpt1493-bib-0010) and a newly recruited cohort in order to increase study power and further investigate (i) novel risk loci within and outside the HLA region, and (ii) the role of *HLA‐A*31:01* stratifying by European subpopulations and clinical phenotypes, including SCAR and DILI.

## Results

### Case collection and population structure

Our meta‐analysis included a total of 43 CBZ‐SCAR cases and 10,701 population controls from two genomewide association studies (GWAS) studies: British GWAS (14 cases and 2,263 controls) and broadly European GWAS (29 SCAR cases and 8,438 controls). The British GWAS included Northern European patients from the United Kingdom, which we have reported previously.[10](#cpt1493-bib-0010) The broadly European GWAS included a newly recruited set of CBZ‐SCAR cases with different clinical phenotypes (**Table** [1](#cpt1493-tbl-0001)) and European ancestries ([Figure S1](#cpt1493-sup-0001)). The inferred population structure of the combined cohorts is shown in **Figure** [1](#cpt1493-fig-0001). A summary of the clinical characteristics of the SCAR cases is provided in **Table** [1](#cpt1493-tbl-0001). We analyzed 25 patients with DRESS, 16 patients with SJS/TEN, and 2 patients with AGEP. The average age of patients at the time of adverse reaction was 34 years in both study phases. Nearly two‐thirds (63%) of cases were women.

### Table 1.

| Phenotype | Category | Phase I cases | Phase II cases |
| --- | --- | --- | --- |
| SCAR | Total number of cases | 14 | 29 |
| Age at the onset in years (mean) | 34 | 34 |  |
| Gender (% female) | 43% | 66% |  |
| Allergy (% yes) | 36% | 24% |  |
| Clinical subphenotypes |  |  |  |
| AGEP | 1 | 1 |  |
| Hypersensitivity syndrome (DRESS) | 13 | 12 |  |
| SJS | 0 | 5 |  |
| SJS/TEN | 0 | 10 |  |
| TEN | 0 | 1 |  |
| Systemic symptoms |  |  |  |
| Eosinophilia | 6 | 6 |  |
| Liver involvement | 0 | 10 |  |
| Fever | 13 | 22 |  |
| Pneumonitis | 2 | 0 |  |
| Multi‐organ failure, death | 1 | 0 |  |
| Additional evidences for the diagnosis |  |  |  |
| Skin biopsy | 1 | 8 |  |
| Skin patch testing performed | 1 | 5 |  |
| Multiple drug‐induced skin reactions | 3 | 1 |  |
| DILI | Total number of cases | 0 | 12 |
| Age at the onset in years (mean) | – | 37.6 |  |
| Gender (% female) | – | 67% |  |
| Pattern of liver injury |   |   |  |
| Cholestatic | – | 3 |  |
| Mixed | – | 2 |  |
| Hepatocellular | – | 5 |  |
| Unknown | – | 2 |  |
| Systemic symptoms |  |  |  |
| Eosinophilia | – | 0 |  |
| Cutaneous rashes | – | 0 |  |
| RUCAM scoreb |   |   |  |
| 3–5 (possible) | – | 2 |  |
| 6–8 (probable) | – | 6 |  |
| > 8 (highly probable) | – | 2 |  |

Table 1 Caption: Clinical information of the CBZ‐SCAR and CBZ‐DILI cases included in the studya

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f2f7/8045549/244733203376/CPT-106-1028-g001.jpg)

Scatterplot representing the first two principal components (PCs) of the current study cohort. Colored dots are the cases divided by clinical phenotypes and the gray dots are the controls. The controls cluster in four groups representing the Italian, Spanish, United Kingdom, and Swedish major control populations. AGEP, acute generalized exanthematous pustulosis; DILI, drug‐induced liver injury; DRESS, drug reaction with eosinophilia and systemic symptoms; SJS/TEN, Stevens‐Johnson syndrome/toxic epidermal necrolysis.

Separately, we recruited 12 patients with CBZ‐DILI of European descent (**Figure** [1](#cpt1493-fig-0001)). The type of liver injury in these patients was 50% hepatocellular and 50% cholestatic/mixed. None of them had cutaneous involvement or eosinophilia. Their clinical characteristics are also provided in **Table** [1](#cpt1493-tbl-0001). CBZ‐DILI cases were compared with the 8,438 controls from the broadly European cohort.

### Meta‐analysis of CBZ‐SCAR

After quality control of the imputed data, we retained 5,271,349 single nucleotide polymorphisms (SNPs) for association analyses. Our meta‐analysis identified two loci attaining genomewide significance (*P *<* *5 × 10^−8^):

### Table 2.

| Locus | Lead SNP | Chr | Position | Alleles |  | Phase I |  | Phase II |  | Combined meta‐analysis |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Risk | Other | OR (95% CI) | P value | OR (95% CI) | P value | OR (95% CI) | P value | Cochran's Q P value | Frequency in controls | Gnomad |  |  |  |  |
| MHC | rs192543598 | 6 | 29,931,345 | G | A | 127 (28.9–560) | 1.5 × 10−10 | 7.8 (2.95–20.7) | 3.5 × 10−5 | 18.1 (8.03–40.88) | 1.7 × 10−12 | 0.002 | 0.01 | 0.02 |
| ALK | rs187926838 | 2 | 29,818,291 | G | A | 15.3 (2.95–79.7) | 0.001 | 11.0 (3.77–32.1) | 1.1 × 10−5 | 12.1 (4.94–29.80) | 4.9 × 10−8 | 0.74 | 0.007 | 0.008 |

Table 2 Caption: Loci attaining genomewide significant evidence of association (P < 5 × 10−8) with CBZ hypersensitivity in a combined meta‐analysis of 43 patients and 10,701 controls of European ancestry

### Dissection of the MHC CBZ‐SCAR association signal

The lead SNP representing the MHC CBZ‐SCAR association signal, rs192543598, was in strong linkage disequilibrium (LD) with *HLA‐A*31:01* (*r* ^2^ = 0.75). The HLA class I allele also reached genomewide significance in the meta‐analysis (OR = 8.0; 95% CI 4.10–15.80; *P *=* *2.2 × 10^−9^; P_perm_ * *<* *5 × 10^−8^; **Table** [3](#cpt1493-tbl-0003) **and** [Table S2](#cpt1493-sup-0001)). The phase II GWAS showed an MHC‐wide *HLA‐A*31:01* association, replicating the previously reported signal.

### Table 3.

| HLA allele | Phase I |  | Phase II |  | Combined meta‐analysis |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| OR (95% CI) | P value | OR (95% CI) | P value | OR (95% CI) | P value | Cochran's Q P value | Frequency in controls |  |
| HLA‐A*31:01 | 16.9 (5.40–52.40) | 1.0 × 10−6 | 5.3 (2.29–12.40) | 9.9 × 10−5 | 8.0 (4.10–15.80) | 2.2 × 10−9 | 0.11 | 0.02 |
| HLA‐B*51:01 | 2.8 (0.80–9.60) | 0.10 | 3.7 (1.87–7.28) | 0.0002 | 3.5 (1.91–6.27) | 5.6 × 10−5 | 0.70 | 0.06 |
| HLA‐C*15:02 | 2.1 (0.23–16.5) | 0.47 | 4.1 (1.74–9.92) | 0.001 | 3.7 (1.68–8.35) | 0.002 | 0.56 | 0.02 |
| HLA‐DPB1*09:01 | 4.9 (0.70–35.60) | 0.12 | 4.5 (1.17–17.60) | 0.03 | 4.6 (1.51–14.20) | 0.007 | 0.96 | 0.009 |

Table 3 Caption: Imputed HLA alleles attaining nominal evidence of association (P < 0.01) with CBZ hypersensitivity in combined meta‐analysis of 43 patients and 10,701 controls of European ancestry

The *HLA‐A*31:01* association was less significant in the broadly European cases than in the British cases, probably because of greater heterogeneity in their geographic origin (**Figure** [1](#cpt1493-fig-0001)). A lower allele frequency was observed in the Spanish cases (6%) compared with other ethnic groups (15% in Northern European and 13% Italian cases) despite an equivalent frequency in the three control groups. However, the effect of *HLA‐A*31:01* was conserved in all European populations (OR_Italian_ = 8.7; OR_Spanish_ = 3.66; and OR_North Europeans_ = 5.51). There were no differences in allele frequencies of the GWAS variants in control groups despite the different genotyping arrays used.

Reciprocal conditional analyses demonstrated that rs192543598 and *HLA‐A*31:01* represent the same underlying association signal ([Table S2](#cpt1493-sup-0001)). After conditioning on the lead SNP, rs192543598, we observed some evidence for residual association (at locus‐wide significance, *P *<* *10^−5^) with CBZ‐SCAR, mapping to *MUC22* (lead SNP rs116071718, OR = 4.0; 95% CI 2.25–7.05; *P = *1.5 × 10^−6^; [Table S2](#cpt1493-sup-0001) **and** [Figure S4](#cpt1493-sup-0001)). After conditioning on both rs192543598 and rs116071718, the association signal with CBZ‐SCAR in the MHC was fully accounted for ([Figure S4](#cpt1493-sup-0001)).

Marginal associations were also identified with *HLA‐B*51:01* and *HLA‐C*15:02* (**Table** [3](#cpt1493-tbl-0003)). These HLA alleles along with *HLA‐A*31:01* constitute an uncommon haplotype present in European descent individuals ([http://www.allelefrequencies.net](http://www.allelefrequencies.net)). Joint carriage of the three alleles had a stronger signal compared with *HLA‐A*31:01* alone (OR = 22.1, *P *=* *2.4 × 10^−6^ vs. OR = 5.7, *P *=* *6.6 × 10^−6^; [Table S3](#cpt1493-sup-0001)).

### Assessment of polymorphic amino acid residues in HLA molecules

Predisposition to CBZ‐SCAR may be related to the same essential amino acid (AA) residues in different HLA alleles. Therefore, we performed an aggregated investigation by analyzing the polymorphic AA residues at these proteins. The strongest association was observed for isoleucine^73^ on the *HLA‐A* locus (OR = 5.5; 95% CI 2.94–10.40; *P *=* *1.4 × 10^−7^), but could not better explain the CBZ‐SCAR association than *HLA‐A*31:01*. The AA is shared by the A*33 and A*31 alleles.[16](#cpt1493-bib-0016) Conditional analysis to isoleucine^73^ or to *HLA‐A*31:01* revealed that isoleucine^80^ in the B locus is a new independent factor, and it is shared with *HLA‐B*57:01* among other HLA B alleles.[16](#cpt1493-bib-0016) After conditioning on both the sites, the residual independent association signal was still accounted for by rs116071718 in the *MUC22* gene ([Table S4](#cpt1493-sup-0001)). These AA associations have not previously been found to be implicated in genetic predisposition to other adverse drug reactions.

### GWAS of CBZ‐SCAR clinical subtypes

We stratified the CBZ‐SCAR cases by clinical phenotypes (AGEP, DRESS, and SJS/TEN; see [Methods](#cpt1493-sec-0011)). The AGEP analysis was not performed because we had only two cases, although it is worth noting that one of the two cases was positive for *HLA‐A*31:01*.

In the CBZ‐DRESS subgroup, *HLA‐A*31:01* and its proxy SNP, rs192543598, reached genomewide significance (*P* _HLA‐A*31:01_ = 2.1 × 10^−9^ and *P* _rs192543598_ = 2.4 × 10^−13^). Consistent with the literature,[17](#cpt1493-bib-0017) *HLA‐A*31:01* showed a stronger effect with CBZ‐DRESS than with SJS/TEN (OR = 12 vs. OR = 2.5, respectively; **Table** [4](#cpt1493-tbl-0004)) with no significant association in the latter group. There was no other phenotype‐specific genomewide significant association for both DRESS and SJS‐TEN ([Figure S5](#cpt1493-sup-0001)).

### Table 4.

| Clinical groups | N | OR (95% CI) | P value | Allele frequency in cases | Allele frequency in controls |
| --- | --- | --- | --- | --- | --- |
| SJS/TEN | 16 | 2.4 (0.55–10.59) | 0.2 | 0.06 | 0.02 |
| DRESS | 25 | 12.9 (5.58–29.78) | 2.1 × 10−9 | 0.14 | 0.02 |
| DILI | 12 | 7.3 (2.47–23.67) | 0.0004 | 0.17 | 0.02 |
| All clinical phenotypes (SCAR and DILI) | 55 | 8.2 (4.56–14.66) | 1.8 × 10−12 | 0.14 | 0.02 |

Table 4 Caption: Association effect size of HLA‐A*31:01 across different clinical phenotypes

The effect of rs187926838 in the *ALK* gene was conserved among the CBZ‐SCAR clinical subtypes ([Table S5](#cpt1493-sup-0001)). We also found that the SJS/TEN phenotype showed an MHC‐region‐wide significant association with *HLA‐B*57:01* (OR = 6.2; 95% CI 2.47–15.37; *P *=* *9.9 × 10^−5^). *HLA‐B*57:01* was in strong LD (*r* ^2^ = 0.8) with the most highly associated SNP in the MHC region (rs116347890; OR = 11.0; 95% CI 4.62–26.42; *P *=* *6.6 × 10^−8^) but was independent of *HLA‐A*31:01* (OR = 6.4; 95% CI 2.55–16.11; *P *=* *7.6 × 10^−5^) after conditioning on *HLA‐A*31:01*. The polymorphic AA position 97 showed an association when valine was present (OR = 6.0; 95% CI 2.45–15.18; *P *=* *0.0001). Valine^97^ is specific for *HLA‐B*57:01*,* HLA‐B*57:02*, and *HLA‐B*57:03* and other very rare B57 alleles.[16](#cpt1493-bib-0016)

### GWAS of CBZ‐DILI

GWAS analysis of 12 European CBZ‐DILI cases against 8,438 European controls did not identify a genomewide association because of limited power. However, 33% of cases were *HLA‐A*31:01* carriers (OR = 7.3; 95% CI 2.47–23.67; *P *=* *0.0004), although the OR was less than that seen with DRESS (OR = 12.9). Our prediction of the association with *HLA‐A*31:01* was fully confirmed by HLA sequencing in 11 cases (DNA was no longer available in one patient). CBZ‐DILI cases did not show an association with rs187926838 (in the *ALK* gene), distinct from that seen with CBZ‐SCAR cases ([Table S5](#cpt1493-sup-0001)). The most significant AA residues associated with CBZ‐DILI were isoleucine^73^ (OR = 7.29; 95% CI 2.578–20.6; *P *=* *0.0002) and threonine^9^ (OR = 5.5; 95% CI 2.31–13.14; *P *=* *0.0001) at the *HLA‐A* locus. The two residues were in LD (*P* _73I conditional to 9T_ = 0.2). Moreover, there was enrichment of *HLA‐A*31:01* carriers in the 10 patients with CBZ‐SCAR with liver involvement compared with the remaining cases (25% vs. 12%, respectively).

Taking all the cases of CBZ‐induced hypersensitivity reactions (SCAR and DILI) together, the risk of developing either DILI or SCAR when given CBZ was eightfold higher in cases carrying *HLA‐A*31:01* (OR = 8.2; 95% CI 4.56–14.66; *P = *1.8 × 10^−12^; **Table** [4](#cpt1493-tbl-0004)).

## Discussion

In the current study, we show that *HLA‐A*31:01* is the strongest predictor of CBZ‐SCAR in a European‐ancestry population, extending our previous study in Northern Europeans[10](#cpt1493-bib-0010) to include Southern Europeans. In addition, we also show that (i) *HLA‐A*31:01* seems to predispose to CBZ‐induced liver injury, and (ii) a variant in the *ALK* gene was associated with an increased risk of CBZ‐SCAR but not CBZ‐DILI.

Our finding of the association between CBZ‐SCAR and *HLA‐A*31:01*, initially reported in 2011,[10](#cpt1493-bib-0010) has also been reported by Amstutz *et al*.[18](#cpt1493-bib-0018) in a multiethnic North American pediatric cohort and by Genin *et al*.[17](#cpt1493-bib-0017) in a European SCAR cohort. In our study, *HLA‐A*31:01* was a stronger risk factor for DRESS than SJS/TEN, in line with Genin *et al*.[17](#cpt1493-bib-0017) (OR = 57.6 vs. OR = 4.4, respectively) and Amstutz *et al*.[18](#cpt1493-bib-0018) (OR = 31.5 vs. OR = 2.8, respectively), but not with our original study (OR = 12 vs. OR = 25, respectively)[10](#cpt1493-bib-0010) where numbers were much smaller. It is likely that *HLA‐A*31:01* is the most important shared risk factor across CBZ hypersensitivity phenotypes, with the strongest predisposition being to DRESS. Indeed, our meta‐analysis shows that *HLA‐A*31:01* carriers have eightfold higher risk of developing CBZ‐SCAR than noncarriers. Interestingly, a recent prospective study in Japan was able to show that pre‐prescription genotyping for *HLA‐A*31:01* reduces the incidence of CBZ hypersensitivity reactions.[19](#cpt1493-bib-0019) Taken together, the overwhelming evidence of the role of *HLA‐A*31:01* in predisposing to CBZ‐SCAR shows that there is a need to implement its use in clinical settings, as outlined in the recent Clinical Pharmacogenetics Implementation Consortium guideline.[20](#cpt1493-bib-0020) Independent of the *HLA‐A*31:01* association, we identified a residual effect in the MHC region mapping to *MUC22* gene (lead SNP rs116071718). *MUC22* codes for panbronchiolitis‐related mucin‐like protein 1 and has been associated with SJS/TEN caused by a variety of drugs, including CBZ in European patients.[21](#cpt1493-bib-0021) Its role in SCAR needs further investigation. We also found that *HLA‐B*51:01* was the second most significant HLA allele. *HLA‐B*51:01* has already been reported to be marginally associated with CBZ‐SCAR[12](#cpt1493-bib-0012), [13](#cpt1493-bib-0013), [14](#cpt1493-bib-0014) and more recently with SCAR due to other drugs.[22](#cpt1493-bib-0022) *HLA‐B*51:01* together with *HLA‐A*31:01* and *HLA‐C*15:02* constitute a haplotype that has a larger OR than each single allele ([Table S3](#cpt1493-sup-0001)). This is an interesting finding, which suggests that *in vivo*, the T‐cell response to CBZ‐derived antigens requires cooperativity between different HLA alleles, as we have previously demonstrated in an *HLA‐A*31:01*‐positive patient.[23](#cpt1493-bib-0023)

We also identified an association between CBZ‐induced SJS/TEN and *HLA‐B*57:01*. This allele is a well‐known risk factor for other CD8+ T‐cell‐mediated reactions, including abacavir hypersensitivity syndrome[24](#cpt1493-bib-0024), [25](#cpt1493-bib-0025) and flucloxacillin‐induced DILI,[26](#cpt1493-bib-0026) and more recently, DILI induced by two other drug combinations, pazopanib and a combination of antituberculosis and anti‐HIV drugs.[27](#cpt1493-bib-0027), [28](#cpt1493-bib-0028) Given that the association with CBZ‐induced SJS/TEN was not genomewide significant, it needs replication in other cohorts. Interestingly, our finding of the association of valine^97^ with CBZ‐SJS/TEN is in line with the association of valine^97^ with flucloxacillin‐DILI,[29](#cpt1493-bib-0029) suggesting that the binding site of *HLA‐B*57:01* may be promiscuous for a number of drugs, which would be in keeping with the increasing number of reports of immune‐mediated reactions associated with this allele.

We have, we believe, for the first time evaluated whether *HLA‐A*31:01* is a risk factor for DILI. Taking all cases of DILI into account, the carriage rate of *HLA‐A*31:01* was 33% with an OR of 7, higher than that found in association of HLA class II alleles with lumiracoxib‐induced DILI.[30](#cpt1493-bib-0030) Interestingly, CBZ‐SCAR and CBZ‐DILI also shared the most significant AA, isoleucine.^73^ Isoleucine^73^ is a cryptic epitope specific for A31 and A33 antigens.[31](#cpt1493-bib-0031) Position 73 is not normally exposed. When the antigen changes its conformation and β2m and peptide dissociate from the heavy chain, isoleucine^73^ is externalized with the potential to lead to an autoimmune reaction.[32](#cpt1493-bib-0032) Interestingly, *HLA‐A*33:01* and *HLA‐A*33:03* have recently been associated with DILI due to several unrelated drugs.[33](#cpt1493-bib-0033) The association with *HLA‐A*31:01*, however, was not genomewide significant, which may reflect the small numbers studied. It may also reflect that the mechanism of antigen presentation (be it the parent drug or metabolite) differs between the skin and liver given the major role of the liver in drug metabolism and its ability to form chemically reactive intermediates.[34](#cpt1493-bib-0034) Further work in larger numbers of patients with CBZ‐DILI, together with mechanistic studies, will be needed to understand the role of drug/metabolites as antigens in the context of different drug metabolizing and antigen presentation capabilities, and, indeed, whether these differences are responsible for the remarkable organ‐specificity of the reactions and their severity seen in different patients.

A tantalizing association that we have identified in our meta‐analysis (that did not pass genomewide significance after permutation) that was observed in the SCAR but not in the DILI cases was with uncommon variants in the *ALK* gene, which codes for the anaplastic lymphoma kinase gene. Somatic mutations in the *ALK* gene have been identified in different cancers,[35](#cpt1493-bib-0035) including lung cancer, which has led to the development of ALK‐inhibitors for therapy. The product of the *ALK* gene, a receptor tyrosine kinase belonging to the insulin receptor family, seems to be important for the balance between proliferation and apoptosis.[36](#cpt1493-bib-0036) Furthermore, the associated region falls within a keratinocyte‐specific predicted insulator ([Figure S6](#cpt1493-sup-0001)). Given that ALK is important in cellular proliferation and cell death and shows ubiquitous tissue distribution ([http://www.proteinatlas.org/ENSG00000171094-ALK/tissue](http://www.proteinatlas.org/ENSG00000171094-ALK/tissue)), it may have an important role in T‐cell proliferation and keratinocyte death, both important in the pathogenesis of SCAR.

In conclusion, we have provided further data regarding genetic factors predisposing to different CBZ adverse reaction phenotypes, which are thought to have an immune pathogenesis, namely SCAR and DILI. We have extended our studies to include Southern Europeans in addition to our previous study in Northern Europeans[10](#cpt1493-bib-0010) and included an analysis of CBZ‐DILI. We have also identified novel associations with the *ALK* and *MUC22* genes, which require further validation and experimental investigation to determine the mechanisms. It is possible that there are also other genetic factors outside the MHC region, but we may not have had the statistical power to detect them in this study. It is, therefore, important that further work also involves a trans‐ethnic meta‐analysis of the GWAS in diverse populations, which have been undertaken to date, to fully assess differences in the genetic architecture of CBZ adverse reactions between ancestries and, more precisely, localize the underlying causal alleles.

## Methods

### Study design overview

Our study combines results from two GWAS conducted in subjects with ancestry from Northern Europe, Italy, and Spain. All participants provided written informed consent and each study was approved by the appropriate national or institutional ethical review boards. Because the reactions have a very low prevalence, we used general population samples as study controls.

### British CBZ‐SCAR GWAS

The study included 14 patients with CBZ‐hypersensitivity of Northern European descent of the 21 recruited at the University of Liverpool, as described by McCormack *et al*.[10](#cpt1493-bib-0010) A total of 2,263 population controls of Northern European descent from the Wellcome Trust Case Control Consortium were utilized. The other Liverpool samples (*n* = 7) originally described were excluded either because of population stratification issues or because two patients had DILI rather than skin involvement (and so have been included in the liver analysis).

### Broadly European CBZ‐SCAR GWAS

The study included a total of 29 CBZ‐SCAR samples obtained from two available sources: 3 cases were from the [PGX40001](https://www.ncbi.nlm.nih.gov/protein/PGX40001) study[37](#cpt1493-bib-0037) and 26 were newly recruited between 2009 and 2013 as part of the International Consortium of Drug Hypersensitivity (ITCH) study. The ITCH study was run under the auspices of the International Serious Adverse Event Consortium, involving 12 recruitment centers in Europe, Australia, and South America. Clinical inclusion criteria for all ITCH‐related SCAR cases were as described by our previous phenotype standardization protocol for drug‐induced skin injury.[38](#cpt1493-bib-0038) All patient phenotypes were independently adjudicated by the co‐authors N.S. and P.F. Further clinical details are provided in **Table** [1](#cpt1493-tbl-0001).

For this group, a total of 8,438 population controls of European descent were selected from the Wellcome Trust Case Control Consortium (not overlapping with samples from the British cohort), the population reference sample (POPRES),[39](#cpt1493-bib-0039) the [PGX40001](https://www.ncbi.nlm.nih.gov/protein/PGX40001) study,[37](#cpt1493-bib-0037) the Spanish Bladder Cohort (dbGaP phs000346.v1),[40](#cpt1493-bib-0040) the Hypergenes cohort ([http://www.hypergenes.eu](http://www.hypergenes.eu)), the National Spanish DNA Bank cohort ([http://www.bancoadn.org/](http://www.bancoadn.org/)), the Swedish Twin Registry cohort ([http://ki.se/en/research/the-swedish-twin-registry](http://ki.se/en/research/the-swedish-twin-registry)), LAM30004 study,[37](#cpt1493-bib-0037) TSI (Hapmap data), and the iSAEC Italian Penicillin Tolerant Cohort (IPTC).

### Broadly European CBZ‐DILI GWAS

As part of the phase II study, we also collected 12 patients of European descent with CBZ‐DILI, in which the liver was involved by itself. The cases were recruited from two large international DILI consortia: iDILIC and DILIN,[33](#cpt1493-bib-0033) whereas two other cases were as described previously.[10](#cpt1493-bib-0010) The iDILIC cases were evaluated by application of the Council for International Organizations of Medical Science (CIOMS) scale, also called the Roussel Uclaf Causality Assessment Method (RUCAM), and by expert review by a panel of three hepatologists. Only cases having at least possible causality (score ≥ 3) were included in the study.[33](#cpt1493-bib-0033) DILIN causality assessment is determined by a panel of three hepatologists who independently assign a causality score ranging from 1 (definite) to 5 (unlikely) as well as a severity score ranging from 1 (mild) to 5 (fatal), as previously described.[33](#cpt1493-bib-0033) Besides the DILI phenotype, DILIN and iDILIC also collected additional clinical information, including the occurrence of eosinophilia and cutaneous involvement. The cases were compared with the 8,438 phase II population controls of European descent.

### Genotyping of cases and controls

Out of the cumulative 43 CBZ‐SCAR cases, three [PGX40001](https://www.ncbi.nlm.nih.gov/protein/PGX40001) cases were previously genotyped with Illumina 1M Duo chip.[41](#cpt1493-bib-0041) DNA from the rest of British and broadly European cases was extracted from whole blood and stored in the Wolfson Centre for Personalized Medicine in Liverpool. Genomewide genotyping was profiled by the Illumina Infinium HumanCoreExome Bead Chip for 16 cases and by Illumina HumanOmniExpress Bead Chip for 24 cases at the Broad Institute, Boston. Among the CBZ‐DILI cases, three cases were previously genotyped with Illumina 1M Duo Chip, whereas two cases were newly profiled by HumanCoreExome Bead Chip and seven cases by HumanOmniExpress Bead Chip at the Broad Institute, Boston.[33](#cpt1493-bib-0033) A total of 10,701 previously genotyped population controls were cumulatively used in British and broadly European cohorts. Information about the genotyping platform used by each of the control cohorts is reported in [Table S6](#cpt1493-sup-0001). For each of the sample batches (defined as a set of subjects—either cases or controls—genotyped together by the same array), quality control was conducted at both SNP and subject levels before performing the imputation as previously described.[41](#cpt1493-bib-0041) Analysis of population structure was performed by the EIGENSTRAT package version 3.0.[42](#cpt1493-bib-0042) Pre‐phasing and imputation were performed in batches by dividing the cases and controls according to genotyping platform, using SHAPEIT (version v2.r727)[43](#cpt1493-bib-0043) and IMPUTE2 (version 3)[44](#cpt1493-bib-0044) with 1000 Genomes Project (release version 3) as reference.[43](#cpt1493-bib-0043) For downstream analysis, we used best‐guess genotypes retaining imputed genotypes with posterior probability > 0.9. Detailed methods are outlined in the **Supplementary Materials**.

### Association analysis and meta‐analysis

We tested for association of each SNP with CBZ‐SCAR, separately in British and broadly European GWAS, in a logistic regression framework, under an additive genetic model, with adjustment for the principal components from smartPCA to account for population structure using PLINK version 1.07.[45](#cpt1493-bib-0045) No other additional covariates were included in the model because we did not have clinical information for controls. Association summary statistics from the two phases were combined using effective sample size weighted *z*‐score fixed‐effects meta‐analysis, implemented in METAL.[46](#cpt1493-bib-0046) Allelic ORs across the two phases were obtained through inverse‐variance weighting of effect sizes, with heterogeneity assessed with Cochran's *Q* statistic,[47](#cpt1493-bib-0047) implemented in METAL. We reported only those SNPs that attained, in addition to genomewide significance, nominal evidence of association (*P *<* *0.05) with the same direction of effect on CBZ‐SCAR in both GWAS phases (internal validation). Furthermore, we tested for association of each SNP with CBZ‐SCAR clinical subtypes: SJS/TEN in phase II GWAS and DRESS across both phases in the same meta‐analysis framework. We also tested for association of each SNP with CBZ‐DILI in phase II GWAS, in a logistic regression framework, under an additive genetic model, with adjustment for the principal components to account for population structure. Genomewide significance was defined using a common threshold of *P *<* *5 × 10^−8^. To account for the small sample size and the disproportionate case/control ratio, we applied a permutation approach for genomewide significant signals. In particular, we randomly permuted genotypes among individuals within the same phase of the design, tested for association, and then meta‐analyzed. We applied 5 × 10^8^ permutations to demonstrate genomewide significance. All detailed analyses and Manhattan plots were performed with R version 3.0.2.[14](#cpt1493-bib-0014)

### HLA imputation, genotyping, and analysis

For each batch, HLA alleles were inferred using HLA genotype imputation with attribute bagging[48](#cpt1493-bib-0048) using the reference predictor panels specific for the genotyping chip. AA changes were inferred by SNP2HLA using reference data collected by the Type 1 Diabetes Genetics Consortium.[49](#cpt1493-bib-0049) We tested for association of carriage of each allele/AA/specific HLA haplotypes with CBZ‐SCAR, CBZ‐SCAR subtypes, and CBZ‐DILI using the same methods described above. MHC significance was defined using the Bonferroni correction threshold of *P* < 0.00025 accounting for 200 observed HLA alleles across loci (0.05/200). Conditional analyses in the MHC region were undertaken and the genotypes at the conditioning SNP(s) were included as covariates under an additive model. Fixed‐effects meta‐analyses across the two phases of GWAS were performed using the methods described above for unconditional analyses. High‐resolution genotyping of HLA loci was performed on all DILI cases by Histogenetics (Ossining, NY), as previously described.[33](#cpt1493-bib-0033)

## Funding

This work was supported by the International Serious Adverse Events Consortium (iSAEC). The iSAEC is a nonprofit organization dedicated to identifying and validating DNA‐variants useful in predicting the risk of drug‐related serious adverse events. The Consortium brings together the pharmaceutical industry, regulatory authorities, and academic centers to address clinical and scientific issues associated with the genetics of drug‐related serious adverse events. The iSAEC's current funding members include: Abbott, Amgen, AstraZeneca, Daiichi Sankyo, GlaxoSmithKline, Merck, Novartis, Pfizer, Takeda, and the Wellcome Trust. M.P. is an NIHR Senior Investigator. M.P. and A.A. thank the MRC Centre for Drug Safety Science for support (MR/L006758/1). P.N. was supported by iSAEC. A.P.M. is a Wellcome Trust Senior Fellow in Basic Biomedical Science (under award WT098017). This is a summary of independent research partly (the DILIGEN and iDILIC sample collection) funded by the National Institute for Health Research (NIHR) Nottingham Digestive Diseases Biomedical Research Unit at the Nottingham University Hospitals NHS Trust and University of Nottingham. The DILIN ([https://dilin.org/](https://dilin.org/)) is supported by the National Institute of Diabetes and Digestive and Kidney Diseases of the National Institutes of Health (NIH) as a Cooperative Agreement (U01s). The Spanish DILI Registry is partly funded by the Spanish Medicine Agency, Fondo Europeo de Desarrollo Regional (FEDER; FIS 12/00378, FIS 16/01748). CIBERehd is funded by Instituto de Salud Carlos III. The Swedish case collection SWEDEGENE ([www.swedegene.se](http://www.swedegene.se)) has received support from the Swedish Medical Products Agency, the Swedish Society of Medicine (2008‐21619), Swedish Research Council (Medicine 521‐2011‐2440 and 521‐2014‐3370), and Swedish Heart and Lung Foundation (20120557 and 20140291). The EUDRAGENE collaboration received support from the EC 5th Framework program (QLRI‐CT‐2002‐02757). M.M. is supported by the National Institute for Health Research (NIHR) Biomedical Research Centre at Guy's and St. Thomas’ NHS Foundation Trust and King's College London. The views expressed are those of the author(s) and not necessarily those of the NHS, the NIHR, or the Department of Health.

## Conflict of Interest

P.N. is an employee of Sema4 Mount Sinai venture, Stamford, CT, USA. M.R.N. is an employee of GlaxoSmithKline. All other authors declared no competing interests for this work.

## Author Contributions

P.N., A.K.D., M.R.N., G.A., A.P.M., A.A., and M.P. wrote the manuscript. P.N., A.P.M., M.R.N., M.J.D., A.K.D., G.A., Y.S., A.A., and M.P. designed the research. P.N., A.S., and A.P.M. analyzed the data. M.P., A.K.D., P.B.W., S.B., L.M., G.A., M.I.L., R.J.A., M.W., P.H., C.S., E.S.B., P.F., K.K., T.L., A.M., M.M., E.P., W.P., A.R., N.S., G.S., L.K.T., and A.F. contributed new reagents/analytical tools.

## Supporting information

## Acknowledgments

Special thanks to Arthur Holden for his help and effort in guiding this collaborative work and to the broad genotyping facility for their contribution to the genomewide association studies genotyping. We are grateful to Daniele Cusi (Hypergenes), Patrik K. Magnusson (Swedish Twin Registry), and Javier Martin (Spanish DNA bank) for provision of control data. The data presented in the current publication are also based on the use of study data downloaded from the dbGaP website, under phs000346.v1.p1. We also acknowledge the contribution of all our clinical collaborators and the study participants.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Yip, V.L. , Marson, A.G. , Jorgensen, A.L. , Pirmohamed, M. & Alfirevic, A. HLA genotype and carbamazepine‐induced cutaneous adverse drug reactions: a systematic review. Clin. Pharmacol. Ther. 92, 757–765 (2012).  [DOI](https://doi.org/10.1038/clpt.2012.189) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23132554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=HLA%20genotype%20and%20carbamazepine%E2%80%90induced%20cutaneous%20adverse%20drug%20reactions:%20a%20systematic%20review&author=V.L.%20Yip&author=A.G.%20Marson&author=A.L.%20Jorgensen&author=M.%20Pirmohamed&author=A.%20Alfirevic&volume=92&publication_year=2012&pages=757-765&pmid=23132554&doi=10.1038/clpt.2012.189&)

2. Marson, A.G. et al. The SANAD study of effectiveness of carbamazepine, gabapentin, lamotrigine, oxcarbazepine, or topiramate for treatment of partial epilepsy: an unblinded randomised controlled trial. Lancet 369, 1000–1015 (2007).  [DOI](https://doi.org/10.1016/S0140-6736(07)60460-7) | [PMC free article](/articles/PMC2080688/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17382827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=The%20SANAD%20study%20of%20effectiveness%20of%20carbamazepine,%20gabapentin,%20lamotrigine,%20oxcarbazepine,%20or%20topiramate%20for%20treatment%20of%20partial%20epilepsy:%20an%20unblinded%20randomised%20controlled%20trial&author=A.G.%20Marson&volume=369&publication_year=2007&pages=1000-1015&pmid=17382827&doi=10.1016/S0140-6736(07)60460-7&)

3. Uetrecht, J. & Naisbitt, D.J. Idiosyncratic adverse drug reactions: current concepts. Pharmacol. Rev. 65, 779–808 (2013).  [DOI](https://doi.org/10.1124/pr.113.007450) | [PMC free article](/articles/PMC3639727/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23476052/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Rev.&title=Idiosyncratic%20adverse%20drug%20reactions:%20current%20concepts&author=J.%20Uetrecht&author=D.J.%20Naisbitt&volume=65&publication_year=2013&pages=779-808&pmid=23476052&doi=10.1124/pr.113.007450&)

4. Bjornsson, E. , Kalaitzakis, E. & Olsson, R. The impact of eosinophilia and hepatic necrosis on prognosis in patients with drug‐induced liver injury. Aliment. Pharmacol. Ther. 25, 1411–1421 (2007).  [DOI](https://doi.org/10.1111/j.1365-2036.2007.03330.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17539980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aliment.%20Pharmacol.%20Ther.&title=The%20impact%20of%20eosinophilia%20and%20hepatic%20necrosis%20on%20prognosis%20in%20patients%20with%20drug%E2%80%90induced%20liver%20injury&author=E.%20Bjornsson&author=E.%20Kalaitzakis&author=R.%20Olsson&volume=25&publication_year=2007&pages=1411-1421&pmid=17539980&doi=10.1111/j.1365-2036.2007.03330.x&)

5. Cheng, C.Y. , Su, S.C. , Chen, C.H. , Chen, W.L. , Deng, S.T. & Chung, W.H. HLA associations and clinical implications in T‐cell mediated drug hypersensitivity reactions: an updated review. J. Immunol. Res. 2014, 565320 (2014).  [DOI](https://doi.org/10.1155/2014/565320) | [PMC free article](/articles/PMC4034438/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24901010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.%20Res.&title=HLA%20associations%20and%20clinical%20implications%20in%20T%E2%80%90cell%20mediated%20drug%20hypersensitivity%20reactions:%20an%20updated%20review&author=C.Y.%20Cheng&author=S.C.%20Su&author=C.H.%20Chen&author=W.L.%20Chen&author=S.T.%20Deng&volume=2014&publication_year=2014&pages=565320&pmid=24901010&doi=10.1155/2014/565320&)

6. Chung, W.H. et al. Medical genetics: a marker for Stevens‐Johnson syndrome. Nature 428, 486 (2004).  [DOI](https://doi.org/10.1038/428486a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15057820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Medical%20genetics:%20a%20marker%20for%20Stevens%E2%80%90Johnson%20syndrome&author=W.H.%20Chung&volume=428&publication_year=2004&pages=486&pmid=15057820&doi=10.1038/428486a&)

7. Shi, Y.W. et al. HLA‐A*24:02 as a common risk factor for antiepileptic drug‐induced cutaneous adverse reactions. Neurology 88, 2183–2191 (2017).  [DOI](https://doi.org/10.1212/WNL.0000000000004008) | [PMC free article](/articles/PMC5467955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28476759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurology&title=HLA%E2%80%90A*24:02%20as%20a%20common%20risk%20factor%20for%20antiepileptic%20drug%E2%80%90induced%20cutaneous%20adverse%20reactions&author=Y.W.%20Shi&volume=88&publication_year=2017&pages=2183-2191&pmid=28476759&doi=10.1212/WNL.0000000000004008&)

8. Ozeki, T. et al. Genome‐wide association study identifies HLA‐A*3101 allele as a genetic risk factor for carbamazepine‐induced cutaneous adverse drug reactions in Japanese population. Hum. Mol. Genet. 20, 1034–1041 (2011).  [DOI](https://doi.org/10.1093/hmg/ddq537) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21149285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Genome%E2%80%90wide%20association%20study%20identifies%20HLA%E2%80%90A*3101%20allele%20as%20a%20genetic%20risk%20factor%20for%20carbamazepine%E2%80%90induced%20cutaneous%20adverse%20drug%20reactions%20in%20Japanese%20population&author=T.%20Ozeki&volume=20&publication_year=2011&pages=1034-1041&pmid=21149285&doi=10.1093/hmg/ddq537&)

9. Kim, S.H. et al. Carbamazepine‐induced severe cutaneous adverse reactions and HLA genotypes in Koreans. Epilepsy Res. 97, 190–197 (2011).  [DOI](https://doi.org/10.1016/j.eplepsyres.2011.08.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21917426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Res.&title=Carbamazepine%E2%80%90induced%20severe%20cutaneous%20adverse%20reactions%20and%20HLA%20genotypes%20in%20Koreans&author=S.H.%20Kim&volume=97&publication_year=2011&pages=190-197&pmid=21917426&doi=10.1016/j.eplepsyres.2011.08.010&)

10. McCormack, M. et al. HLA‐A*3101 and carbamazepine‐induced hypersensitivity reactions in Europeans. N. Engl. J. Med. 364, 1134–1143 (2011).  [DOI](https://doi.org/10.1056/NEJMoa1013297) | [PMC free article](/articles/PMC3113609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=HLA%E2%80%90A*3101%20and%20carbamazepine%E2%80%90induced%20hypersensitivity%20reactions%20in%20Europeans&author=M.%20McCormack&volume=364&publication_year=2011&pages=1134-1143&pmid=21428769&doi=10.1056/NEJMoa1013297&)

11. Shi, Y.W. et al. Association between HLA and Stevens‐Johnson syndrome induced by carbamazepine in Southern Han Chinese: genetic markers besides B*1502? Basic Clin. Pharmacol. Toxicol. 111, 58–64 (2012).  [DOI](https://doi.org/10.1111/j.1742-7843.2012.00868.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22348435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin.%20Pharmacol.%20Toxicol.&title=Association%20between%20HLA%20and%20Stevens%E2%80%90Johnson%20syndrome%20induced%20by%20carbamazepine%20in%20Southern%20Han%20Chinese:%20genetic%20markers%20besides%20B*1502?&author=Y.W.%20Shi&volume=111&publication_year=2012&pages=58-64&pmid=22348435&doi=10.1111/j.1742-7843.2012.00868.x&)

12. Tassaneeyakul, W. et al. Association between HLA‐B*1502 and carbamazepine‐induced severe cutaneous adverse drug reactions in a Thai population. Epilepsia 51, 926–930 (2010).  [DOI](https://doi.org/10.1111/j.1528-1167.2010.02533.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20345939/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Association%20between%20HLA%E2%80%90B*1502%20and%20carbamazepine%E2%80%90induced%20severe%20cutaneous%20adverse%20drug%20reactions%20in%20a%20Thai%20population&author=W.%20Tassaneeyakul&volume=51&publication_year=2010&pages=926-930&pmid=20345939&doi=10.1111/j.1528-1167.2010.02533.x&)

13. Niihara, H. , Kakamu, T. , Fujita, Y. , Kaneko, S. & Morita, E. HLA‐A31 strongly associates with carbamazepine‐induced adverse drug reactions but not with carbamazepine‐induced lymphocyte proliferation in a Japanese population. J. Dermatol. 39, 594–601 (2012).  [DOI](https://doi.org/10.1111/j.1346-8138.2011.01457.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22211527/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Dermatol.&title=HLA%E2%80%90A31%20strongly%20associates%20with%20carbamazepine%E2%80%90induced%20adverse%20drug%20reactions%20but%20not%20with%20carbamazepine%E2%80%90induced%20lymphocyte%20proliferation%20in%20a%20Japanese%20population&author=H.%20Niihara&author=T.%20Kakamu&author=Y.%20Fujita&author=S.%20Kaneko&author=E.%20Morita&volume=39&publication_year=2012&pages=594-601&pmid=22211527&doi=10.1111/j.1346-8138.2011.01457.x&)

14. Hsiao, Y.H. et al. Genotype‐phenotype association between HLA and carbamazepine‐induced hypersensitivity reactions: strength and clinical correlations. J. Dermatol. Sci. 73, 101–109 (2014).  [DOI](https://doi.org/10.1016/j.jdermsci.2013.10.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24268988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Dermatol.%20Sci.&title=Genotype%E2%80%90phenotype%20association%20between%20HLA%20and%20carbamazepine%E2%80%90induced%20hypersensitivity%20reactions:%20strength%20and%20clinical%20correlations&author=Y.H.%20Hsiao&volume=73&publication_year=2014&pages=101-109&pmid=24268988&doi=10.1016/j.jdermsci.2013.10.003&)

15. Gonzalez‐Galarza, F.F. et al. Allele frequency net 2015 update: new features for HLA epitopes, KIR and disease and HLA adverse drug reaction associations. Nucleic Acids Res. 43, D784–D788 (2015).  [DOI](https://doi.org/10.1093/nar/gku1166) | [PMC free article](/articles/PMC4383964/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25414323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Allele%20frequency%20net%202015%20update:%20new%20features%20for%20HLA%20epitopes,%20KIR%20and%20disease%20and%20HLA%20adverse%20drug%20reaction%20associations&author=F.F.%20Gonzalez%E2%80%90Galarza&volume=43&publication_year=2015&pages=D784-D788&pmid=25414323&doi=10.1093/nar/gku1166&)

16. Duquesnoy, R.J. et al. 16th IHIW: a website for antibody‐defined HLA epitope Registry. Int. J. Immunogenet. 40, 54–59 (2013).  [DOI](https://doi.org/10.1111/iji.12017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23136897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Immunogenet.&title=16th%20IHIW:%20a%20website%20for%20antibody%E2%80%90defined%20HLA%20epitope%20Registry&author=R.J.%20Duquesnoy&volume=40&publication_year=2013&pages=54-59&pmid=23136897&doi=10.1111/iji.12017&)

17. Genin, E. et al. HLA‐A*31:01 and different types of carbamazepine‐induced severe cutaneous adverse reactions: an international study and meta‐analysis. Pharmacogenomics J. 14, 281–288 (2014).  [DOI](https://doi.org/10.1038/tpj.2013.40) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24322785/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=HLA%E2%80%90A*31:01%20and%20different%20types%20of%20carbamazepine%E2%80%90induced%20severe%20cutaneous%20adverse%20reactions:%20an%20international%20study%20and%20meta%E2%80%90analysis&author=E.%20Genin&volume=14&publication_year=2014&pages=281-288&pmid=24322785&doi=10.1038/tpj.2013.40&)

18. Amstutz, U. et al. HLA‐A 31:01 and HLA‐B 15:02 as genetic markers for carbamazepine hypersensitivity in children. Clin. Pharmacol. Ther. 94, 142–149 (2013).  [DOI](https://doi.org/10.1038/clpt.2013.55) | [PMC free article](/articles/PMC3839910/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23588310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=HLA%E2%80%90A%2031:01%20and%20HLA%E2%80%90B%2015:02%20as%20genetic%20markers%20for%20carbamazepine%20hypersensitivity%20in%20children&author=U.%20Amstutz&volume=94&publication_year=2013&pages=142-149&pmid=23588310&doi=10.1038/clpt.2013.55&)

19. Mushiroda, T. et al. Association of HLA‐A*31:01 screening with the incidence of carbamazepine‐induced cutaneous adverse reactions in a Japanese population. JAMA Neurol. 75, 842–849 (2018).  [DOI](https://doi.org/10.1001/jamaneurol.2018.0278) | [PMC free article](/articles/PMC6145764/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29610831/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Neurol.&title=Association%20of%20HLA%E2%80%90A*31:01%20screening%20with%20the%20incidence%20of%20carbamazepine%E2%80%90induced%20cutaneous%20adverse%20reactions%20in%20a%20Japanese%20population&author=T.%20Mushiroda&volume=75&publication_year=2018&pages=842-849&pmid=29610831&doi=10.1001/jamaneurol.2018.0278&)

20. Phillips, E.J. et al. Clinical Pharmacogenetics Implementation Consortium Guideline for HLA genotype and use of carbamazepine and oxcarbazepine: 2017 update. Clin. Pharmacol. Ther. 103, 574–581 (2018).  [DOI](https://doi.org/10.1002/cpt.1004) | [PMC free article](/articles/PMC5847474/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29392710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20Guideline%20for%20HLA%20genotype%20and%20use%20of%20carbamazepine%20and%20oxcarbazepine:%202017%20update&author=E.J.%20Phillips&volume=103&publication_year=2018&pages=574-581&pmid=29392710&doi=10.1002/cpt.1004&)

21. Genin, E. et al. Genome‐wide association study of Stevens‐Johnson syndrome and toxic epidermal necrolysis in Europe. Orphanet. J. Rare Dis. 6, 52 (2011).  [DOI](https://doi.org/10.1186/1750-1172-6-52) | [PMC free article](/articles/PMC3173287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21801394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Orphanet.%20J.%20Rare%20Dis.&title=Genome%E2%80%90wide%20association%20study%20of%20Stevens%E2%80%90Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis%20in%20Europe&author=E.%20Genin&volume=6&publication_year=2011&pages=52&pmid=21801394&doi=10.1186/1750-1172-6-52&)

22. Yang, Y. et al. HLA‐B*51:01 is strongly associated with clindamycin‐related cutaneous adverse drug reactions. Pharmacogenomics J. 17, 501–505 (2017).  [DOI](https://doi.org/10.1038/tpj.2016.61) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27527109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=HLA%E2%80%90B*51:01%20is%20strongly%20associated%20with%20clindamycin%E2%80%90related%20cutaneous%20adverse%20drug%20reactions&author=Y.%20Yang&volume=17&publication_year=2017&pages=501-505&pmid=27527109&doi=10.1038/tpj.2016.61&)

23. Lichtenfels, M. et al. HLA restriction of carbamazepine‐specific T‐cell clones from an HLA‐A*31:01‐positive hypersensitive patient. Chem. Res. Toxicol. 27, 175–177 (2014).  [DOI](https://doi.org/10.1021/tx400460w) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24476427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Res.%20Toxicol.&title=HLA%20restriction%20of%20carbamazepine%E2%80%90specific%20T%E2%80%90cell%20clones%20from%20an%20HLA%E2%80%90A*31:01%E2%80%90positive%20hypersensitive%20patient&author=M.%20Lichtenfels&volume=27&publication_year=2014&pages=175-177&pmid=24476427&doi=10.1021/tx400460w&)

24. Mallal, S. et al. Association between presence of HLA‐B*5701, HLA‐DR7, and HLA‐DQ3 and hypersensitivity to HIV‐1 reverse‐transcriptase inhibitor abacavir. Lancet 359, 727–732 (2002).  [DOI](https://doi.org/10.1016/s0140-6736(02)07873-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11888582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Association%20between%20presence%20of%20HLA%E2%80%90B*5701,%20HLA%E2%80%90DR7,%20and%20HLA%E2%80%90DQ3%20and%20hypersensitivity%20to%20HIV%E2%80%901%20reverse%E2%80%90transcriptase%20inhibitor%20abacavir&author=S.%20Mallal&volume=359&publication_year=2002&pages=727-732&pmid=11888582&doi=10.1016/s0140-6736(02)07873-x&)

25. Hetherington, S. et al. Genetic variations in HLA‐B region and hypersensitivity reactions to abacavir. Lancet 359, 1121–1122 (2002).  [DOI](https://doi.org/10.1016/S0140-6736(02)08158-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11943262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genetic%20variations%20in%20HLA%E2%80%90B%20region%20and%20hypersensitivity%20reactions%20to%20abacavir&author=S.%20Hetherington&volume=359&publication_year=2002&pages=1121-1122&pmid=11943262&doi=10.1016/S0140-6736(02)08158-8&)

26. Daly, A.K. et al. HLA‐B*5701 genotype is a major determinant of drug‐induced liver injury due to flucloxacillin. Nat. Genet. 41, 816–819 (2009).  [DOI](https://doi.org/10.1038/ng.379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19483685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=HLA%E2%80%90B*5701%20genotype%20is%20a%20major%20determinant%20of%20drug%E2%80%90induced%20liver%20injury%20due%20to%20flucloxacillin&author=A.K.%20Daly&volume=41&publication_year=2009&pages=816-819&pmid=19483685&doi=10.1038/ng.379&)

27. Petros, Z. , Kishikawa, J. , Makonnen, E. , Yimer, G. , Habtewold, A. & Aklillu, E. HLA‐B*57 allele is associated with concomitant anti‐tuberculosis and antiretroviral drugs induced liver toxicity in Ethiopians. Front. Pharmacol. 8, 90 (2017).  [DOI](https://doi.org/10.3389/fphar.2017.00090) | [PMC free article](/articles/PMC5326775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28289388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=HLA%E2%80%90B*57%20allele%20is%20associated%20with%20concomitant%20anti%E2%80%90tuberculosis%20and%20antiretroviral%20drugs%20induced%20liver%20toxicity%20in%20Ethiopians&author=Z.%20Petros&author=J.%20Kishikawa&author=E.%20Makonnen&author=G.%20Yimer&author=A.%20Habtewold&volume=8&publication_year=2017&pages=90&pmid=28289388&doi=10.3389/fphar.2017.00090&)

28. Xu, C.F. et al. HLA‐B*57:01 confers susceptibility to pazopanib‐associated liver injury in patients with cancer. Clin. Cancer Res. 22, 1371–1377 (2016).  [DOI](https://doi.org/10.1158/1078-0432.CCR-15-2044) | [PMC free article](/articles/PMC7444994/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26546620/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=HLA%E2%80%90B*57:01%20confers%20susceptibility%20to%20pazopanib%E2%80%90associated%20liver%20injury%20in%20patients%20with%20cancer&author=C.F.%20Xu&volume=22&publication_year=2016&pages=1371-1377&pmid=26546620&doi=10.1158/1078-0432.CCR-15-2044&)

29. Nicoletti, P. et al. Drug‐induced liver injury due to flucloxacillin: relevance of multiple human leukocyte antigen alleles. Clin. Pharmacol. Ther. 106, 245–253 (2019).  [DOI](https://doi.org/10.1002/cpt.1375) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30661239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Drug%E2%80%90induced%20liver%20injury%20due%20to%20flucloxacillin:%20relevance%20of%20multiple%20human%20leukocyte%20antigen%20alleles&author=P.%20Nicoletti&volume=106&publication_year=2019&pages=245-253&pmid=30661239&doi=10.1002/cpt.1375&)

30. Singer, J.B. et al. A genome‐wide study identifies HLA alleles associated with lumiracoxib‐related liver injury. Nat. Genet. 42, 711–714 (2010).  [DOI](https://doi.org/10.1038/ng.632) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20639878/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=A%20genome%E2%80%90wide%20study%20identifies%20HLA%20alleles%20associated%20with%20lumiracoxib%E2%80%90related%20liver%20injury&author=J.B.%20Singer&volume=42&publication_year=2010&pages=711-714&pmid=20639878&doi=10.1038/ng.632&)

31. El‐Awar, N. , Jucaud, V. & Nguyen, A. HLA epitopes: the targets of monoclonal and alloantibodies defined. J. Immunol. Res. 2017, 3406230 (2017).  [DOI](https://doi.org/10.1155/2017/3406230) | [PMC free article](/articles/PMC5463109/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28626773/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.%20Res.&title=HLA%20epitopes:%20the%20targets%20of%20monoclonal%20and%20alloantibodies%20defined&author=N.%20El%E2%80%90Awar&author=V.%20Jucaud&author=A.%20Nguyen&volume=2017&publication_year=2017&pages=3406230&pmid=28626773&doi=10.1155/2017/3406230&)

32. Lanzavecchia, A. How can cryptic epitopes trigger autoimmunity? J. Exp. Med. 181, 1945–1948 (1995).  [DOI](https://doi.org/10.1084/jem.181.6.1945) | [PMC free article](/articles/PMC2192044/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7539032/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Med.&title=How%20can%20cryptic%20epitopes%20trigger%20autoimmunity?&author=A.%20Lanzavecchia&volume=181&publication_year=1995&pages=1945-1948&pmid=7539032&doi=10.1084/jem.181.6.1945&)

33. Nicoletti, P. et al. Association of liver injury from specific drugs, or groups of drugs, with polymorphisms in HLA and other genes in a genome‐wide association study. Gastroenterology 152, 1078–1089 (2017).  [DOI](https://doi.org/10.1053/j.gastro.2016.12.016) | [PMC free article](/articles/PMC5367948/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28043905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Association%20of%20liver%20injury%20from%20specific%20drugs,%20or%20groups%20of%20drugs,%20with%20polymorphisms%20in%20HLA%20and%20other%20genes%20in%20a%20genome%E2%80%90wide%20association%20study&author=P.%20Nicoletti&volume=152&publication_year=2017&pages=1078-1089&pmid=28043905&doi=10.1053/j.gastro.2016.12.016&)

34. Pirmohamed, M. , Ostrov, D.A. & Park, B.K. New genetic findings lead the way to a better understanding of fundamental mechanisms of drug hypersensitivity. J. Allergy Clin. Immunol. 136, 236–244 (2015).  [DOI](https://doi.org/10.1016/j.jaci.2015.06.022) | [PMC free article](/articles/PMC4534769/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26254050/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Allergy%20Clin.%20Immunol.&title=New%20genetic%20findings%20lead%20the%20way%20to%20a%20better%20understanding%20of%20fundamental%20mechanisms%20of%20drug%20hypersensitivity&author=M.%20Pirmohamed&author=D.A.%20Ostrov&author=B.K.%20Park&volume=136&publication_year=2015&pages=236-244&pmid=26254050&doi=10.1016/j.jaci.2015.06.022&)

35. Holla, V.R. et al. ALK: a tyrosine kinase target for cancer therapy. Cold Spring Harb. Mol. Case Stud. 3, a001115 (2017).  [DOI](https://doi.org/10.1101/mcs.a001115) | [PMC free article](/articles/PMC5171696/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28050598/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb.%20Mol.%20Case%20Stud.&title=ALK:%20a%20tyrosine%20kinase%20target%20for%20cancer%20therapy&author=V.R.%20Holla&volume=3&publication_year=2017&pages=a001115&pmid=28050598&doi=10.1101/mcs.a001115&)

36. Souttou, B. , Carvalho, N.B. , Raulais, D. & Vigny, M. Activation of anaplastic lymphoma kinase receptor tyrosine kinase induces neuronal differentiation through the mitogen‐activated protein kinase pathway. J. Biol. Chem. 276, 9526–9531 (2001).  [DOI](https://doi.org/10.1074/jbc.M007333200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11121404/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Activation%20of%20anaplastic%20lymphoma%20kinase%20receptor%20tyrosine%20kinase%20induces%20neuronal%20differentiation%20through%20the%20mitogen%E2%80%90activated%20protein%20kinase%20pathway&author=B.%20Souttou&author=N.B.%20Carvalho&author=D.%20Raulais&author=M.%20Vigny&volume=276&publication_year=2001&pages=9526-9531&pmid=11121404&doi=10.1074/jbc.M007333200&)

37. Pirmohamed, M. et al. Investigation into the multidimensional genetic basis of drug‐induced Stevens‐Johnson syndrome and toxic epidermal necrolysis. Pharmacogenomics 8, 1661–1691 (2007).  [DOI](https://doi.org/10.2217/14622416.8.12.1661) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18085998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Investigation%20into%20the%20multidimensional%20genetic%20basis%20of%20drug%E2%80%90induced%20Stevens%E2%80%90Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis&author=M.%20Pirmohamed&volume=8&publication_year=2007&pages=1661-1691&pmid=18085998&doi=10.2217/14622416.8.12.1661&)

38. Pirmohamed, M. et al. Phenotype standardization for immune‐mediated drug‐induced skin injury. Clin. Pharmacol. Ther. 89, 896–901 (2011).  [DOI](https://doi.org/10.1038/clpt.2011.79) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21562486/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Phenotype%20standardization%20for%20immune%E2%80%90mediated%20drug%E2%80%90induced%20skin%20injury&author=M.%20Pirmohamed&volume=89&publication_year=2011&pages=896-901&pmid=21562486&doi=10.1038/clpt.2011.79&)

39. Nelson, M.R. et al. The Population Reference Sample, POPRES: a resource for population, disease, and pharmacological genetics research. Am. J. Hum. Genet. 83, 347–358 (2008).  [DOI](https://doi.org/10.1016/j.ajhg.2008.08.005) | [PMC free article](/articles/PMC2556436/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18760391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=The%20Population%20Reference%20Sample,%20POPRES:%20a%20resource%20for%20population,%20disease,%20and%20pharmacological%20genetics%20research&author=M.R.%20Nelson&volume=83&publication_year=2008&pages=347-358&pmid=18760391&doi=10.1016/j.ajhg.2008.08.005&)

40. Tryka, K.A. et al. NCBI's database of genotypes and phenotypes: dbGaP. Nucleic Acids Res. 42, D975–D979 (2014).  [DOI](https://doi.org/10.1093/nar/gkt1211) | [PMC free article](/articles/PMC3965052/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24297256/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=NCBI's%20database%20of%20genotypes%20and%20phenotypes:%20dbGaP&author=K.A.%20Tryka&volume=42&publication_year=2014&pages=D975-D979&pmid=24297256&doi=10.1093/nar/gkt1211&)

41. Shen, Y. et al. Genome‐wide association study of serious blistering skin rash caused by drugs. Pharmacogenomics J. 12, 96–104 (2012).  [DOI](https://doi.org/10.1038/tpj.2010.84) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21221126/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Genome%E2%80%90wide%20association%20study%20of%20serious%20blistering%20skin%20rash%20caused%20by%20drugs&author=Y.%20Shen&volume=12&publication_year=2012&pages=96-104&pmid=21221126&doi=10.1038/tpj.2010.84&)

42. Price, A.L. , Patterson, N.J. , Plenge, R.M. , Weinblatt, M.E. , Shadick, N.A. & Reich, D. Principal components analysis corrects for stratification in genome‐wide association studies. Nat. Genet. 38, 904–909 (2006).  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome%E2%80%90wide%20association%20studies&author=A.L.%20Price&author=N.J.%20Patterson&author=R.M.%20Plenge&author=M.E.%20Weinblatt&author=N.A.%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

43. Delaneau, O. , Zagury, J.F. & Marchini, J. Improved whole‐chromosome phasing for disease and population genetic studies. Nat. Methods 10, 5–6 (2013).  [DOI](https://doi.org/10.1038/nmeth.2307) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23269371/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&title=Improved%20whole%E2%80%90chromosome%20phasing%20for%20disease%20and%20population%20genetic%20studies&author=O.%20Delaneau&author=J.F.%20Zagury&author=J.%20Marchini&volume=10&publication_year=2013&pages=5-6&pmid=23269371&doi=10.1038/nmeth.2307&)

44. Marchini, J. & Howie, B. Genotype imputation for genome‐wide association studies. Nat. Rev. Genet. 11, 499–511 (2010).  [DOI](https://doi.org/10.1038/nrg2796) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20517342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&title=Genotype%20imputation%20for%20genome%E2%80%90wide%20association%20studies&author=J.%20Marchini&author=B.%20Howie&volume=11&publication_year=2010&pages=499-511&pmid=20517342&doi=10.1038/nrg2796&)

45. Purcell, S. et al. PLINK: a tool set for whole‐genome association and population‐based linkage analyses. Am. J. Hum. Genet. 81, 559–575 (2007).  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=PLINK:%20a%20tool%20set%20for%20whole%E2%80%90genome%20association%20and%20population%E2%80%90based%20linkage%20analyses&author=S.%20Purcell&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

46. Willer, C.J. , Li, Y. & Abecasis, G.R. METAL: fast and efficient meta‐analysis of genomewide association scans. Bioinformatics 26, 2190–2191 (2010).  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta%E2%80%90analysis%20of%20genomewide%20association%20scans&author=C.J.%20Willer&author=Y.%20Li&author=G.R.%20Abecasis&volume=26&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

47. Ioannidis, J.P. , Patsopoulos, N.A. & Evangelou, E. Heterogeneity in meta‐analyses of genome‐wide association investigations. PLoS One 2, e841 (2007).  [DOI](https://doi.org/10.1371/journal.pone.0000841) | [PMC free article](/articles/PMC1950790/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17786212/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Heterogeneity%20in%20meta%E2%80%90analyses%20of%20genome%E2%80%90wide%20association%20investigations&author=J.P.%20Ioannidis&author=N.A.%20Patsopoulos&author=E.%20Evangelou&volume=2&publication_year=2007&pages=e841&pmid=17786212&doi=10.1371/journal.pone.0000841&)

48. Zheng, X. et al. HIBAG–HLA genotype imputation with attribute bagging. Pharmacogenomics J. 14, 192–200 (2014).  [DOI](https://doi.org/10.1038/tpj.2013.18) | [PMC free article](/articles/PMC3772955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23712092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=HIBAG%E2%80%93HLA%20genotype%20imputation%20with%20attribute%20bagging&author=X.%20Zheng&volume=14&publication_year=2014&pages=192-200&pmid=23712092&doi=10.1038/tpj.2013.18&)

49. Jia, X. et al. Imputing amino acid polymorphisms in human leukocyte antigens. PLoS One 8, e64683 (2013).  [DOI](https://doi.org/10.1371/journal.pone.0064683) | [PMC free article](/articles/PMC3675122/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23762245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Imputing%20amino%20acid%20polymorphisms%20in%20human%20leukocyte%20antigens&author=X.%20Jia&volume=8&publication_year=2013&pages=e64683&pmid=23762245&doi=10.1371/journal.pone.0064683&)
