# SLCO1B1 Phenotype and CYP3A5 Polymorphism Significantly Affect Atorvastatin Bioavailability

## Metadata
**Authors:** Pablo Zubiaur, Maria Dolores Benedicto, Gonzalo Villapalos-García, Marcos Navares-Gómez, Gina Mejía-Abril, Manuel Román, Samuel Martín-Vílchez, Dolores Ochoa, Francisco Abad-Santos
**Journal:** Journal of Personalized Medicine
**Date:** 2021 Mar 13
**DOI:** [10.3390/jpm11030204](https://doi.org/10.3390/jpm11030204)
**PMID:** 33805706
**PMCID:** PMC7999651
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7999651/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7999651/pdf/jpm-11-00204.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7999651/pdf/jpm-11-00204.pdf)

## Abstract

Atorvastatin, prescribed for the treatment of hypercholesterolemia, demonstrated overwhelming benefits in reducing cardiovascular morbidity and mortality. However, many patients discontinue therapy due to adverse reactions, especially myopathy. The Dutch Pharmacogenetics Working Group (DPWG) recommends an alternative agent to atorvastatin and simvastatin or a dose adjustment depending on other risk factors for statin-induced myopathy in SLCO1B1 rs4149056 CC or TC carriers. In contrast, the Clinical Pharmacogenetics Implementation Consortium (CPIC) published their guideline on simvastatin, but not on atorvastatin. In this work, we aimed to demonstrate the effect of SLCO1B1 phenotype and other variants (e.g., in CYP3A4/5, UGT enzymes or SLC transporters) on atorvastatin pharmacokinetics. For this purpose, a candidate-gene pharmacogenetic study was proposed. The study population comprised 156 healthy volunteers enrolled in atorvastatin bioequivalence clinical trials. The genotyping strategy comprised a total of 60 variants in 15 genes. Women showed higher exposure to atorvastatin compared to men (p = 0.001), however this difference disappeared after dose/weight (DW) correction. The most relevant pharmacogenetic differences were the following: AUC/DW and Cmax /DW based on (a) SLCO1B1 phenotype (p < 0.001 for both) and (b) CYP3A5*3 (p = 0.004 and 0.018, respectively). As secondary findings: SLC22A1 *2/*2 genotype was related to higher Cmax/DW (ANOVA p = 0.030) and SLC22A1 *1/*5 genotype was associated with higher Vd/F (ANOVA p = 0.032) compared to SLC22A1 *1/*1, respectively. Finally, UGT2B7 rs7439366 *1/*1 genotype was associated with higher tmax as compared with the *1/*3 genotype (ANOVA p = 0.024). Based on our results, we suggest that SLCO1B1 is the best predictor for atorvastatin pharmacokinetic variability and that prescription should be adjusted based on it. We suggest that the CPIC should include atorvastatin in their statin-SLCO1B1 guidelines. Interesting and novel results were observed based on CYP3A5 genotype, which should be confirmed with further studies.

Atorvastatin, prescribed for the treatment of hypercholesterolemia, demonstrated overwhelming benefits in reducing cardiovascular morbidity and mortality. However, many patients discontinue therapy due to adverse reactions, especially myopathy. The Dutch Pharmacogenetics Working Group (DPWG) recommends an alternative agent to atorvastatin and simvastatin or a dose adjustment depending on other risk factors for statin-induced myopathy in SLCO1B1 rs4149056 CC or TC carriers. In contrast, the Clinical Pharmacogenetics Implementation Consortium (CPIC) published their guideline on simvastatin, but not on atorvastatin. In this work, we aimed to demonstrate the effect of SLCO1B1 phenotype and other variants (e.g., in CYP3A4/5, UGT enzymes or SLC transporters) on atorvastatin pharmacokinetics. For this purpose, a candidate-gene pharmacogenetic study was proposed. The study population comprised 156 healthy volunteers enrolled in atorvastatin bioequivalence clinical trials. The genotyping strategy comprised a total of 60 variants in 15 genes. Women showed higher exposure to atorvastatin compared to men (p = 0.001), however this difference disappeared after dose/weight (DW) correction. The most relevant pharmacogenetic differences were the following: AUC/DW and Cmax /DW based on (a) SLCO1B1 phenotype (p < 0.001 for both) and (b) CYP3A5*3 (p = 0.004 and 0.018, respectively). As secondary findings: SLC22A1 *2/*2 genotype was related to higher Cmax/DW (ANOVA p = 0.030) and SLC22A1 *1/*5 genotype was associated with higher Vd/F (ANOVA p = 0.032) compared to SLC22A1 *1/*1, respectively. Finally, UGT2B7 rs7439366 *1/*1 genotype was associated with higher tmax as compared with the *1/*3 genotype (ANOVA p = 0.024). Based on our results, we suggest that SLCO1B1 is the best predictor for atorvastatin pharmacokinetic variability and that prescription should be adjusted based on it. We suggest that the CPIC should include atorvastatin in their statin-SLCO1B1 guidelines. Interesting and novel results were observed based on CYP3A5 genotype, which should be confirmed with further studies.

Keywords: atorvastatin, pharmacogenetics, SLCO1B1, precision medicine

## 1. Introduction

Statins are the most frequently prescribed drugs for the management of hypercholesterolemia, due to their effectiveness and safety profile [[1](#B1-jpm-11-00204)]. They all inhibit the hydroxymethylglutaryl coenzyme A (HMG-CoA) reductase, which reduces cholesterol biosynthesis and modulates lipid metabolism. Statins have an antiatherosclerotic effect correlated with the decrease in LDL cholesterol [[2](#B2-jpm-11-00204)]. Atorvastatin is a member of the statin family with greater effectiveness in cholesterol control compared to other statins, namely lovastatin, pravastatin, simvastatin, and fluvastatin, with a similar tolerability profile [[3](#B3-jpm-11-00204)].

Atorvastatin daily dose ranges between 10 to 80 mg, depending on initial LDL blood levels. After oral administration, atorvastatin is quickly and almost entirely absorbed (95–99%), with maximum concentrations (C_max_) reached at 1 to 2 h (t_max_) [[4](#B4-jpm-11-00204),[5](#B5-jpm-11-00204)]. It suffers pronounced pre-systemic clearance at the gastrointestinal tract and first-pass hepatic clearance, which explains its low systemic bioavailability (around 12%) [[5](#B5-jpm-11-00204),[6](#B6-jpm-11-00204)]. Atorvastatin binds to plasma proteins (>98%), and its volume of distribution is approximately 38 L. It undergoes cytochrome P450 (CYP) 3A4 (CYP3A4) mediated metabolism to active metabolites [[5](#B5-jpm-11-00204),[7](#B7-jpm-11-00204)]. Elimination is principally biliary with apparently no significant enterohepatic recirculation. Half-life (t_1/2_) is approximately 14 h for atorvastatin and 20–30 h for its metabolites [[5](#B5-jpm-11-00204),[7](#B7-jpm-11-00204)]. Atorvastatin is a substrate of the organic anion transporter polypeptides 1B1 (OATP1B1) and 1B3 (OATP1B3), encoded by *SLCO1B1* and *SLCO1B3* genes, respectively [[5](#B5-jpm-11-00204),[8](#B8-jpm-11-00204)].

Genetic polymorphism is related to variability in atorvastatin pharmacokinetics, pharmacodynamics, drug exposure [[9](#B9-jpm-11-00204)], and effectiveness [[10](#B10-jpm-11-00204)]. However, to date, the only clinical guideline for atorvastatin dose adjustment based on a pharmacogenetic biomarker is the one published by the Dutch Pharmacogenetics Working Group (DPWG) [[11](#B11-jpm-11-00204)]. The use of an alternative statin (e.g., fluvastatin) is recommended for patients with *SLCO1B1* rs4149056 T>C, C/C (*5/*5) or T/C (*1/*5) genotypes and additional significant risk factors for statin-induced myopathy. In other words, therapy must be adjusted in those without the *SLCO1B1* normal function (NF) phenotype (*1/*1), i.e., decreased function (DF) or poor function (PF) phenotypes (*1/*5 and *5/*5 genotypes, respectively). These recommendations are the same for simvastatin (DPWG). In addition, the Clinical Pharmacogenetics Implementation Consortium (CPIC) published their pharmacogenetic guideline for *SLCO1B1* and simvastatin [[12](#B12-jpm-11-00204)], with similar recommendations as those from DPWG.

To confirm the influence of the SLCO1B1 phenotype on atorvastatin exposure, and the need for dose adjustments based on it, we aimed to perform a candidate gene pharmacogenetic study in healthy volunteers enrolled in bioequivalence clinical trials. In addition, we proposed to investigate the influence of single nucleotide polymorphisms (SNPs) in other genes in relation to the disposition of atorvastatin, namely *CYP3A*, other *CYP* enzymes or transporters (e.g., *ABCB1* or *SLC22A1*), as well as on atorvastatin tolerability.

## 2. Materials and Methods

### 2.1. Study Population

The study population comprised healthy volunteers enrolled in five different atorvastatin or atorvastatin/ezetimibe bioequivalence clinical trials performed at the Clinical Trial Unit of Hospital Universitario de La Princesa (UECHUP), Madrid, Spain. Study protocols were revised and approved by the Hospital’s Research Ethics Committee and by the Spanish Drugs Agency (AEMPS). Complying with Spanish and European legislation on research in humans, all of them were accomplished under the Good Clinical Practice guidelines and endorsing the Declaration of Helsinki. EUDRA-CT numbers were as follows: 2018-000082-37, 2019-002222-67, 2019-000891-41, 2019-001670-29, and 2019-000656-34. All the subjects (n = 178) provided their informed consent for their enrolment in the bioequivalence clinical trial. For the pharmacogenetic study, 156 volunteers signed a specific informed consent.

All the volunteers satisfied the inclusion criteria: being healthy males or females, aged 18 to 55. Exclusion criteria comprised the following: any organic or physical pathology, the use of any pharmacological treatment in the previous 48 h, body mass index (BMI) out of the 18 to 30 range, history of any kind of drug hypersensitivity, positive abuse drug screening, smokers, alcohol addicts or ethylic intoxication in the previous week, having donated blood within the previous month, pregnancy or breastfeeding, having participated in a similar study within the previous 3 months, grapefruit intake in the previous 48 h, swallowing difficulty, and lactose or galactose intolerance.

### 2.2. Study Design and Procedures

The current observational pharmacogenetic study was based on five independent bioequivalence clinical trials (A, B, C, D, E). In four of them (A, B, C, E), atorvastatin 80 mg film-coated formulations were used. All of them were phase I, single oral dose, open-label, crossover and randomized clinical trials; the reference formulations were Cardyl or Zarator (Pfizer, Spain). Of the latter, three were replicated (B, C, E) (i.e., with four sequences and four periods) and one was not replicated (A) (i.e., with two sequences and two periods). The fifth clinical trial (D) assessed the bioequivalence for atorvastatin/ezetimibe 80/10 mg coated bilayer tablets versus ezetimibe 10 mg tablets (Ezetrol, MSD, Madrid Spain) and atorvastatin 80 mg (Zarator, Pfizer, Spain). It was replicated, with four periods and four sequences.

In all of them, the determination of plasma concentrations was blinded. Volunteers were hospitalized from 10 h before drug intake to 12 or 24 h after dosing. Formulations were administered by oral route under fasting conditions with 240 mL of water. Blood samples were extracted in EDTA K2 tubes (a) at twenty time-points between pre-dose and 48 h after drug intake (A, B, C, E) or (b) at thirty time-points between pre-dose and 72 h after drug intake (D). Plasma was extracted by centrifugation and frozen until its shipment to an external analytical laboratory. The analytical method involved a liquid–liquid extraction procedure with tert-butyl methyl ether after which atorvastatin and an internal standard were determined by reversed phase ultra-high-performance reversed phase liquid chromatography coupled to tandem mass spectrometry (UPLC/MS/MS). Method validation satisfied the European Medicines Agency (EMA) requirements for bioequivalence demonstration.

### 2.3. Pharmacokinetic Analysis

A non-compartmental approach was used to calculate pharmacokinetic parameters. Following the trapezoidal rule, the area under the curve between pre-dose and the last time-point (t) (AUC_t_) was calculated. The terminal rate constant (k_e_) was calculated by linear regression of the log-linear part of the concentration–time curve. The AUC between t and infinite was estimated as C_t_/k_e_ (AUC_t-∞_). The AUC between 0 and ∞ was calculated as AUC_t_ + AUC_t-∞_ (AUC_∞_). Drug clearance was calculated adjusted for bioavailability (Cl/F) as dose (D) divided by AUC_∞_ and weight (W) (i.e., D/AUC*W). Similarly, the volume of distribution was calculated adjusted for bioavailability (Vd/F) as Cl/F divided by k_e_. Half-life (t_1/2_) was estimated as –ln 2/k_e_. The remaining pharmacokinetic parameters were directly obtained from the concentration–time curves: the maximum concentration (C_max_) and the time to reach the C_max_ (t_max_). The CERTARA Phoenix WinNonlin software, version 6.0 (Certara USA, Princeton, NJ, USA) was used.

### 2.4. Safety

The tolerability assessment consisted of the evaluation of abnormalities in analytical values, blood parameters, physical examination or any other clinically relevant event. Furthermore, to monitor vascular and heart function, a 12-lead electrocardiogram (ECG) was carried out at predose and 1.5 h after drug intake; in all but one study, another ECG was carried out 3–4 h after dosing. Vital signs (VS), i.e., systolic and diastolic blood pressure, heart rate and tympanic temperature, were monitored simultaneously to ECG. For the notation of adverse events (AEs), volunteers were asked for abnormalities in their health status; those reported spontaneously were additionally considered. The Spanish Pharmacovigilance System algorithm was used for causality determination [[13](#B13-jpm-11-00204)]. Only those AEs with a definite or possible causality were considered adverse drug reactions (ADRs).

### 2.5. Genotyping, Haplotyping and Phenotyping

DNA was extracted from peripheral blood in a MagNA Pure automatic DNA extractor (Roche Applied Science, Pleasanton, CA, USA). The genotyping strategy comprised the genotyping of 60 variants in 15 genes. However, not all variants could be genotyped for all samples. Firstly, a customized genotyping array was used in an Applied Biosystems QuantStudio 12K flex qPCR instrument with an OpenArray thermal block (ThermoFisher, USA). [Table 1](#jpm-11-00204-t001) depicts the variants genotyped in four of the five clinical trials (n = 120). The *CYP3A4**20 (rs67666821) polymorphism was genotyped by KASPar SNP Genotyping System (LGC Genomics, Herts, UK) in an ABI PRISM 7900HT Sequence Detection System (Applied Biosystems, Darmstadt, Germany. A *CYP2D6* copy number variation assay (CNV) was performed in a QuantStudio 12k flex thermal cycler with a 96-well thermal block, following the methodology previously reported [[14](#B14-jpm-11-00204)]. The remaining samples (n = 36) could not be genotyped with the OpenArray technology. Their genotyping was outsourced to CEGEN-PRB3-ISCIII (Santiago de Compostela, Galicia, Spain), supported by grant PT17/0019, of the PE I+D+i 2013–2016, funded by ISCIII and ERDF, for the following 24 matching variants: *ABCB1* C1236T (rs1128503), C3435T (rs1045642), G2677TA (rs2032582), *ABCC2* (rs717620), *CYP1A2**1B (rs2470890), *1F (rs762551), *CYP2A6**9 (rs28399433), *CYP2B6**5 (rs3211371), *CYP2C19**17 (rs12248560) *2 (rs4244285), *3 (rs4986893), *CYP2C8**2 (rs11572103), *3 (rs10509681 and rs11572080), *4 (rs1058930), *CYP2C9**2 (rs1799853), *3 (rs1057910), *CYP3A4**20 (rs67666821), *CYP3A4**22 (rs35599367), *CYP3A5**3 (rs776746), *CYP3A5**6 (rs10264272), *CYP4F2* (rs2108622) and *UGT1A1**28 (rs887829). Another five variants not included in the OpenArray plate were genotyped: *UGT1A1* rs35350960, rs4124874, rs4148323, *UGT2B4* rs4557343 and *UGT2B7* rs7439366.

### Table 1.

| Gene | Allele/SNP | Gene | Allele/SNP |
| --- | --- | --- | --- |
| CYP1A2 | *1C (rs2069514) | CYP3A4 | *22 (rs35599367) |
| *1F (rs762551) | rs55785340 |  |  |
| *1B (rs2470890) | rs4646438 |  |  |
| CYP2A6 | *9 (rs28399433) | CYP3A5 | *3 (rs776746) |
| CYP2B6 | *9 (rs3745274) | *6 (rs10264272) |  |
| *5 (rs3211371) | ABCB1 | C3435T (rs1045642) |  |
| *4 (rs2279343) | G2677T/A (rs2032582) |  |  |
| rs2279345 | C1236T (rs1128503) |  |  |
| rs4803419 | 1000-44G>T (rs10276036) |  |  |
| CYP2C8 | *2 (rs11572103) | 2895+3559C>T (rs7787082) |  |
| *3 (rs10509681) | 330-3208C>T (rs4728709) |  |  |
| *4 (rs1058930) | 2481+788T>C (rs10248420) |  |  |
| CYP2C9 | *2 (rs1799853) | 2686-3393T>G (rs10280101) |  |
| *3 (rs1057910) | 2320-695G>A (rs12720067) |  |  |
| CYP2C19 | *2 (rs4244285) | 2482-707A>G (rs11983225) |  |
| *3 (rs4986893) | 2212-372A>G (rs4148737) |  |  |
| *4 (rs28399504) | rs3842 |  |  |
| *17 (rs12248560) | ABCC2 | c.1247G>A (rs2273697) |  |
| CYP2D6 | *3 (rs35742686) | rs717620 |  |
| *4 (rs3892097) | SLCO1B1 | *1B (rs2306283) |  |
| *6 (rs5030655) | *5 (rs4149056) |  |  |
| *7 (rs5030867) | c.-910G>A (rs4149015) |  |  |
| *8 (rs5030865) | rs11045879 |  |  |
| *9 (rs5030656) | SLC22A1 | *2 (rs72552763) |  |
| *10 (rs1065852) | *3 (rs12208357) |  |  |
| *14 (rs5030865) | *5 (rs34059508) |  |  |
| *17 (rs28371706) | UGT1A1 | *28 (rs887829) |  |
| *41 (rs28371725) |   |   |  |

Table 1 Caption: Variants/alleles* genotyped with the Open Array/QuantStudio 12k flex platform.

*CYP3A5 (**3, *6), *CYP2D6* (*3, *4, *5, *6, *7, *8, *9, *10, *14, *17, *41 and the gene copy number)*, CYP2C19* (*2, *3, *4, *17), *SLCO1B1* (*1B, *5), *CYP2B6* (*5 and *9) and *CYP2C9* (*2, *3) variants were used to infer the enzymatic phenotype according to CPIC guidelines [[12](#B12-jpm-11-00204),[15](#B15-jpm-11-00204),[16](#B16-jpm-11-00204),[17](#B17-jpm-11-00204),[18](#B18-jpm-11-00204),[19](#B19-jpm-11-00204)]. Since not all samples were genotyped for the same variants, the absence of genotyping data was assumed to be “not mutated”. The same strategy was implemented for genotyping errors (e.g., absence of amplification). *CYP1A2* (*1C, *1F and *1B) variants were used to infer the activity score and phenotype as described in previous publications [[20](#B20-jpm-11-00204),[21](#B21-jpm-11-00204)]. *SLC22A1* and *ABCB1* variants were merged into haplotypes: the absence of any variant was assigned the wild-type haplotype, the presence of one variant was assigned the heterozygous haplotype and the presence of two or more variants was assigned the mutant haplotype. Another *ABCB1* haplotype was inferred by exclusively considering C3435T, G2677T/A and C1236T variants, as these were elsewhere reported to have a greater impact on the transporter’s activity or expression levels [[22](#B22-jpm-11-00204)].

### 2.6. Statistical Analysis

All pharmacokinetic parameters were logarithmically transformed to normalize distributions. Prior to logarithmic transformations, AUC_∞_ and C_max_ were divided by the dose/weight ratio (AUC/DW, C_max_ /DW) to correct the differences in weight between sexes or races which can produce pharmacokinetic variability. To avoid random associations, the following statistical analysis strategy was followed: initially, a univariate analysis was performed, where the mean of pharmacokinetic parameters or the incidence of adverse drug reactions (ADRs) were compared according to categorical variables (e.g., sex, race, haplotypes, phenotypes). For the comparison of means, a *t* test (variables with two categories) or an ANOVA test followed by a Bonferroni post-hoc (variables with three or more categories) were used. For the comparison of the incidence of ADRs according to categorical variables, a Chi-squared test was used. Afterwards, each pharmacokinetic parameter or ADR were individually analyzed with a multivariate analysis. Either by linear or logistic regression, pharmacokinetic parameters or ADRs were explored, respectively. As independent variables, only variables with *p <* 0.05 in the univariate analysis were explored; in addition, pharmacokinetic parameters were introduced as independent variables in the logistic regression. A Bonferroni correction for multiple comparisons was carried out; the value of *p* < 0.05 for statistical significance was divided by the number of variables introduced in the multivariate analysis. The Hardy–Weinberg equilibrium (HWE) was calculated for the genotyped variants using the *HardyWeinberg* package [[23](#B23-jpm-11-00204)] and the R-studio v.4.0.3. software. Deviations from the equilibrium were considered Pearson’s goodness-of-fit chi-square *p* values below 0.05; other statistics (e.g., Fisher exact test) were calculated with an online software (Institute of Human Genetics, University of Munich, available at [https://ihg.gsf.de/cgi-bin/hw/hwa1.pl](https://ihg.gsf.de/cgi-bin/hw/hwa1.pl), accessed on 5 March 2021). The remaining statistical analysis were computed in SPSS v.23.0.

## 3. Results

### 3.1. Demographic Characteristics

Study population was composed by 85 women (54%) and 71 men (46%). Men’s height, weight and body mass index (BMI) were significantly superior to that of women (*p* < 0.0001, *p* < 0.0001 and *p* = 0.005, respectively) ([Table 1](#jpm-11-00204-t001)). Caucasian was the most prevalent race (52%) compared to Latin-Americans (45%), Black (3%) and Arabic (one male). Demographics also differed significantly according to races ([Table 2](#jpm-11-00204-t002)). The Black or Arabic group was related to higher weight compared to Caucasians and Latin-Americans (*p* < 0.0001 and *p* = 0.005, respectively) and to higher height compared to Latin-Americans (*p* = 0.027). Moreover, Caucasians were younger than Latin-Americans and Black or Arabic volunteers (*p* < 0.0001 and *p* = 0.049, respectively) and showed lower BMI (*p* < 0.001 and *p* = 0.001). Of note, the Black or Arabic group was composed by four men and one woman.

### Table 2.

| Sex | n | Weight (kg) | CV% | BMI (kg/m2) | CV% | Height (m) | CV% | Age (years) | CV% |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Women | 85 | 61.5 | 13.8 | 23.1 | 13.4 | 1.63 | 3.7 | 30.1 | 28.2 |
| Men | 71 | 75.3 * | 12.7 | 24.4 * | 10.2 | 1.75 * | 4.0 | 27.8 | 27.0 |
| Race |   |   |   |   |   |   |   |   |   |
| Caucasian | 81 | 65.5 | 16.8 | 22.6 *2 | 10.6 | 1.7 | 5.9 | 25.5 *2 | 22.4 |
| Latin-American | 70 | 69.2 | 15.0 | 24.8 | 11.7 | 1.67 | 6.0 | 32.7 | 26.9 |
| Black or Arabic | 5 | 85.3 *1 | 15.0 | 27.0 | 7.4 | 1.77 *3 | 5.6 | 33.7 | 18.7 |
| Total | 156 | 67.7 | 16.8 | 23.7 | 12.2 | 1.69 | 5.9 | 29.0 | 27.9 |

Table 2 Caption: Demographic characteristics of the study population.

### 3.2. Pharmacokinetics

Atorvastatin mean AUC_∞_ was 166.6 ± 89.1 ng*h/mL (183.6 ± 90.7 ng*h/mL for females and 146.3 ± 83.4 ng*h/mL for males, *p* = 0.001) and mean C_max_ was 39.0 ± 25.3 ng/mL (44.8 ± 25.5 ng/mL for females and 32.0 ± 23.3 ng/mL for males, *p <* 0.001). After DW correction, the differences disappeared ([Table 3](#jpm-11-00204-t003)).

### Table 3.

|   |   | N | AUC/DW (kg*h*ng/mL*mg) | CV% | Cmax/DW (kg*ng/mL*mg) | CV% | tmax (h) | CV% | t1/2 (h) | CV% | Vd/F (l/kg) | CV% | Cl/F (L/h*kg) | CV% |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Sex | Female | 85 | 142.6 | 56.7 | 34.3 | 61.2 | 1.4 | 57.1 | 9.3 | 31.2 | 124.0 | 66.3 | 9023.4 | 49.6 |
| Male | 71 | 136.6 | 58.4 | 29.9 | 74.9 | 1.4 | 57.1 | 8.7 | 25.3 | 117.0 | 52.3 | 9248.5 | 42.7 |  |
| Clinical trial | A | 14 | 129.2 | 35.9 | 22.7 | 36.1 | 1.7 | 88.2 | 8.1 | 23.5 | 103.5 | 45.1 | 8745 | 35.5 |
| B | 30 | 174.9 | 63.0 | 36.1 | 58.2 | 1.5 | 53.3 | 9.1 | 23.1 | 107.3 | 70.7 | 7836.7 | 61.5 |  |
| C | 39 | 116.3 *1 | 59.3 | 26.6 *2 | 60.2 | 1.3 | 61.5 | 9.4 | 28.7 | 147.1 *1*2 | 53.0 | 11,036.7 *1 | 45.7 |  |
| D | 37 | 149.4 | 46.3 | 37.4 | 61.0 | 1.4 | 42.9 | 8.7 | 33.3 | 103.9 | 71.9 | 8247.0 | 45.9 |  |
| E | 36 | 130.5 | 57.5 | 33.7 | 82.2 | 1.4 | 42.9 | 9.5 | 30.5 | 127.7 | 52.2 | 9181.5 | 30.4 |  |
| Ezetimibe | No | 119 | 136.9 | 60.9 | 30.7 | 69.1 | 1.4 | 64.3 | 9.2 | 27.2 | 126.1 | 57.3 | 9399.1 | 46.2 |
| Yes | 37 | 149.4 ! | 46.3 | 37.4 *! | 61.0 | 1.4 | 42.9 | 8.7 | 33.3 | 103.9 | 71.9 | 8247.0 | 45.9 |  |
| Race | Caucasian | 81 | 132.5 | 61.1 | 33.5 | 71.6 | 1.4 | 50.0 | 9.5 | 29.5 | 138.1 *3! | 63.8 | 9938.9 | 49.4 |
| Latin-American | 70 | 147.8 | 54.2 | 31.0 | 62.9 | 1.5 | 60.0 | 8.7 | 27.6 | 103.4 | 45.9 | 8279.3 | 38.7 |  |
| Black or Arabic | 5 | 148.7 | 48.4 | 31.8 | 34.3 | 1.6 | 43.8 | 7.6 | 9.2 | 83.8 | 33.4 | 7808.3 | 36.8 |  |

Table 3 Caption: Atorvastatin pharmacokinetic parameters based on sex, study design, use of ezetimibe, and race.

Eight variables were significantly related to pharmacokinetic variability in the univariate analysis; for multiple-testing correction, the level of significance in the multivariate analysis was set at *p* = 0.00625 (*p* < 0.05 divided by 8, the number of variables introduced in the multivariate analysis). Healthy volunteers enrolled in the “C” clinical trial exhibited lower AUC/DW (*p* = 0.010) and higher Cl/F (*p* = 0.007) than those enrolled in the “B” clinical trial and lower C_max_ /DW (*p* = 0.039) compared to that of volunteers enrolled in the “D” clinical trial. Moreover, Vd/F in “C” was higher than that of “B” and “C” (*p* = 0.028 and *p* = 0.037, respectively), which was confirmed in the multivariate analysis (unstandardized beta coefficient = 0.184, *p* = 0.013, model R^2^ = 0.272). The use of ezetimibe (i.e., the “D” clinical trial) was associated with higher AUC/DW compared to the other clinical trials, where ezetimibe was not administered (unstandardized beta coefficient = 0.177, *p* = 0.048, model R^2^ = 0.222), to higher C_max_/DW (ANOVA *p* = 0.029, unstandardized beta coefficient = 0.288, *p* = 0.002, model R^2^ = 0.225) and to lower Vd/F (unstandardized beta coefficient = −0.517, *p* = 0.001, model R^2^ = 0.272). Finally, Caucasians showed higher Vd/F compared to Latin-Americans (ANOVA *p* = 0.028, unstandardized beta coefficient = 0.184, *p* = 0.013, R^2^ = 0.225) ([Table 3](#jpm-11-00204-t003)).

All variants were in Hardy–Weinberg equilibrium, except for *CYP3A4**2 rs55785340, *CYP3A4**6 rs4646438, *CYP2A6* rs28399433, *CYP2C19**3 rs4986893, *UGT1A1* rs4124874, *UGT1A1* rs4148323, *CYP2D6**4 rs3892097, *CYP2D6**7 rs5030867, *CYP2D6**8 rs5030865, *CYP1A2**1F rs762551, CYP2B6*9 rs3745274, and *CYP2B6**5 rs2279343. Eight of these SNPs showed no allelic variability ([Supplementary Table S1](#app1-jpm-11-00204)). Genetic polymorphism was associated with atorvastatin pharmacokinetic variability. Carriers of the SLCO1B1 decreased function (DF) and poor function (PF) phenotypes were related to higher AUC/DW, C_max_/DW and to lower Vd/F and Cl/F compared to carriers of the normal function (NF) phenotype (ANOVA *p <* 0.001, *p <* 0.001, *p* = 0.002, *p <* 0.001, respectively, *p <* 0.05 after Bonferroini post-hoc) which was confirmed by multivariate analysis (unstandardized beta coefficients = 0.365, 0.332, −0.341, −0.357, *p <* 0.001, <0.001, <0.001, <0.001, and R^2^ = 0.222, 0.225, 0.272 and 0.200, respectively). Moreover, *CYP3A5* *1/*3 and *3/*3 genotypes were related to lower AUC/DW, C_max_/DW and t_max_ and to higher Cl/F compared to the *1/*1 genotype (ANOVA *p* = 0.004, 0.018, <0.001, 0.005, respectively, *p <* 0.05 after Bonferroini post-hoc); the associations for AUC/DW, C_max_/DW and Cl/F were confirmed by multivariate analysis (unstandardized beta coefficients = −0.208, −0.202, 0.189, *p=* 0.007, 0.009, 0.013 and R^2^ = 0.222, 0.225 and 0.200, respectively). Moreover, the *SLC22A1* *2/*2 genotype was related to higher C_max_/DW (ANOVA *p=* 0.030, *p* > 0.05 in the multivariate analysis) and *SLC22A1* *1/*5 genotype was associated with higher Vd/F (ANOVA *p=* 0.032, unstandardized beta coefficient= 0.535, *p* = 0.011 and R^2^ = 0.272) compared to *SLC22A1* *1/*1, respectively. Finally, *UGT2B7* rs7439366 TT genotype was associated with higher t_max_ as compared with the TC genotype (ANOVA *p* = 0.024); this variable could not be analyzed in the multivariate analysis. ([Table 4](#jpm-11-00204-t004)).

### Table 4.

|   |   | N | AUC/DW (kg*h*ng/mL*mg) | CV% | Cmax (kg*ng/mL*mg) | CV% | tmax (h) | CV% | t1/2 (h) | CV% | Vd/F (l/kg) | CV% | Cl/F (L/h*kg) | CV% |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| SLCO1B1 | NF | 86 | 122.6 *1! | 44.9 | 28.5 *1! | 59.3 | 1.4 | 64.3 | 9 | 30.0 | 127.8 *1! | 57.8 | 9827.4 *1! | 43.1 |
| DF | 30 | 181.5 | 59.3 | 37.5 | 60.0 | 1.4 | 57.1 | 8.7 | 25.3 | 99.7 | 78.0 | 7680.4 | 67.0 |  |
| PF | 4 | 283.7 | 41.4 | 62.4 | 35.9 | 1.3 | 38.5 | 10.5 | 9.5 | 66.4 | 58.9 | 4382.7 | 60.2 |  |
| CYP3A5*3 | *1/*1 | 5 | 244.2 *1 | 21.5 | 57.1 *1 | 55.2 | 1.9 *1 | 42.1 | 9.8 | 26.5 | 65.2 | 41.9 | 4405.1 *1 | 23.7 |
| *1/*3 | 32 | 153.7 | 70.3 | 35.1 | 75.5 | 1.4 | 71.4 | 8.8 | 21.6 | 114.3 | 60.6 | 9181.8 | 55.0 |  |
| *3/*3 | 119 | 131.8 | 52.0 | 30.5 | 63.0 | 1.4 | 50.0 | 9.1 | 30.8 | 124.9 | 59.9 | 9309.2 | 42.7 |  |
| SLC22A1*2 | *1/*1 | 71 | 131.8 | 48.7 | 28.9 | 53.6 | 1.4 | 57.1 | 8.9 | 29.2 | 121.4 | 57.5 | 9393.2 | 47.7 |
| *1/*2 | 41 | 151.2 | 66.2 | 33.5 | 64.5 | 1.4 | 64.3 | 9 | 28.9 | 121.1 | 72.0 | 9136.4 | 54.0 |  |
| *2/*2 | 8 | 195.7 | 51.8 | 49.8 *2 | 64.7 | 1.6 | 68.8 | 8.9 | 19.1 | 83 | 59.5 | 6449.1 | 47.9 |  |
| SLC22A1*5 | *1/*1 | 114 | 144.6 | 57.0 | 32.3 | 61.9 | 1.4 | 64.3 | 8.9 | 28.1 | 114.8 | 59.7 | 8933.1 | 49.3 |
| *1/*5 | 6 | 105.6 | 58.4 | 24.2 | 34.3 | 1.6 | 43.8 | 10.8 | 30.6 | 193.9 * | 75.6 | 12,455.2 | 56.3 |  |
| UGT2B7 rs7439366 | *1/*1 | 9 | 159.6 | 80.6 | 44.1 | 103.9 | 1.8 *3 | 44.4 | 8.2 | 24.4 | 106.9 | 50.4 | 8821 | 43.8 |
| *1/*2 | 12 | 108.9 | 29.5 | 26.6 | 35.3 | 1.1 | 36.4 | 9.5 | 22.1 | 137.8 | 35.6 | 9990.2 | 22.7 |  |
| *2/*2 | 15 | 130.4 | 41.1 | 33.2 | 69.3 | 1.4 | 28.6 | 10.1 | 36.6 | 132 | 64.2 | 8751 | 28.3 |  |
| Total |  | 156 | 139.9 | 57.3 | 32.3 | 67.2 | 1.4 | 57.1 | 9.1 | 28.6 | 120.8 | 60.6 | 9125.9 | 46.4 |

Table 4 Caption: Atorvastatin pharmacokinetic parameters based on genotypes or phenotypes with significant variability.

### 3.3. Safety

No serious ADR was reported during any of the five clinical trials. No clinically relevant alteration of VS or ECG was observed. Twenty-one volunteers suffered a total of 27 ADRs. Three types of ADR were reported: first, gastrointestinal symptoms (flatulence, loose stools, or diarrhea) were reported at least once in 13 volunteers; second, headache was reported at least once in 11 volunteers; third, three cases of myalgia or arthralgia were reported at least once in three volunteers. Ten out of 13 cases (76.9%) of gastrointestinal symptoms occurred in the “E” clinical trial compared to two cases in the D clinical trial (15.4%) and one in the B clinical trial (9.8%) (*p <* 0.001). Males were related to a lower risk for developing headache (logOR = −19.054, *p <* 0.001, R^2^ (Cox and Snell) = 0.068). Pharmacokinetics or genetic polymorphism were unrelated to occurrence of ADRs.

## 4. Discussion

Statins are widely prescribed for the treatment of hypercholesterolemia, having demonstrated overwhelming benefits in reducing cardiovascular morbidity and mortality. However, a considerable percentage of patients discontinue therapy due to the occurrence of adverse reactions, mainly myopathies [[24](#B24-jpm-11-00204)]. Therefore, the personalized prescription of these drugs is recommended to avoid excessive exposure, which may lead to ADRs. In line with the above, the DPGW published its pharmacogenetic guidelines on atorvastatin and simvastatin, where drug dose adjustment is recommended in relation to the SLCO1B1 phenotype. In contrast, the CPIC published the clinical guideline on SLCO1B1 and simvastatin but not for atorvastatin. Our interest was to demonstrate the effect of SLCO1B1 phenotype on atorvastatin pharmacokinetics, which was certainly observed and reported. Furthermore, we aimed to describe the impact of variants in other genes, demographics and the study design in atorvastatin exposure and safety.

Similar to previous works, men exhibited higher weight, height, and BMI than women [[25](#B25-jpm-11-00204)]. Black (n = 4) and Arabic volunteers (n = 1) had to be merged in a combined group for statistical analysis. The differences related to this group in weight and height may be explained by four of the Black or Arabic volunteers being men and only one being a Black woman.

The observed mean atorvastatin pharmacokinetic parameters, e.g., AUC_∞_ = 167 ng*h/mL and C_max_ = 39 ng/mL were consistent with previous works: e.g., after a 40 mg atorvastatin dose, a mean AUC_∞_ of 96 ng*h/mL and a C_max_ of 28 ng/mL was previously reported [[26](#B26-jpm-11-00204)]. Because 80 mg fixed-dose formulations were administered in these clinical trials, women received atorvastatin to a higher dose–weight ratio than men, which was evidenced in a significantly higher AUC_∞_ (25%) and C_max_ (40%) compared to men. These results contrast with a previous study where an 11% AUC reduction was observed in women compared to men [[27](#B27-jpm-11-00204)]. Nevertheless, after DW correction, these differences disappeared. It could therefore be concluded that dosage strength, and not sex, is related to atorvastatin pharmacokinetic variability. Moreover, the differences observed in Vd/F between Caucasian and Black or Arabic volunteers are likely explained by the different sex distribution in both groups. Hence, again, it would be the dosage strength the responsible for the differences.

The differences observed in drug exposure according to the clinical trial design were expected due to the relatively small sample size of each clinical trial (from n = 14 to n = 39) and the different characteristics (e.g., different number of periods, sequences, reference formulations). The use of ezetimibe was related to an increased atorvastatin exposure. A possible drug–drug interaction between both drugs was interrogated previously [[26](#B26-jpm-11-00204)]. Our results contrast with the previous consensus, in which no such interaction was demonstrated. Of note, the methodology for AUC_∞_ extrapolation in the ezetimibe clinical trial was based on AUC_0-72 h_ compared to the other clinical trials which used AUC_0-48 h_. Mean atorvastatin t_1/2_ in this study was 9.1 h. Considering five half-lives, the 95% of AUC would be covered 45.5 h after drug intake. Therefore, the sampling time (0 to 48 h vs. 0 to 72 h) will certainly not be a confounding factor. As mentioned before, the relatively small sample size of each clinical trial likely explains the observed differences. Notwithstanding, should ezetimibe increase atorvastatin exposure to the extent observed in this work (i.e., less than 10% of AUC/DW), the effects may not be relevant in the clinical setting.

As expected, SLCO1B1 phenotype was the main pharmacogenetic predictor of atorvastatin pharmacokinetic variability, which justifies a dose reduction or a drug switch in DF and PF phenotype carriers. Consequently, we suggest that the CPIC should extend their pharmacogenetic guideline on simvastatin and SLCO1B1 [[12](#B12-jpm-11-00204)] to atorvastatin, which is congruent with DPWG recommendations [[11](#B11-jpm-11-00204)] and with previous scientific consensus [[28](#B28-jpm-11-00204),[29](#B29-jpm-11-00204),[30](#B30-jpm-11-00204)], and probably to other statins [[28](#B28-jpm-11-00204)] (e.g., fluvastatin, pravastatin).

Moreover, we identified *CYP3A5**3 to be significantly related to atorvastatin pharmacokinetic variability. Our findings are controversial and require an in-depth discussion. As shown in [Figure 1](#jpm-11-00204-f001), CYP3A4 and CYP3A5 can metabolize atorvastatin in the intestinal and hepatic cells. Based on Ensembl data (available at: [https://www.ensembl.org/index.html](https://www.ensembl.org/index.html), accessed on 12 January 2021), the *3 allele (rs776746) has a prevalence of 80–94% in Americans and Europeans, respectively, which is consistent with our findings: approximately 87% of prevalence in a mixed population, with Caucasians, Latin-Americans mainly.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8579/7999651/3b5a626769b2/jpm-11-00204-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7999651_jpm-11-00204-g001.jpg)

The pharmacokinetic pathway of atorvastatin focused on CYP3A and SLCO1B1.

In theory, the higher metabolic capacity of *CYP3A5* expressers would be expected to result in a lower bioavailability of atorvastatin. Previous works did not observe a clinically relevant interaction [[31](#B31-jpm-11-00204),[32](#B32-jpm-11-00204)]. However, these were in vitro approaches [[31](#B31-jpm-11-00204)] or clinical observational studies with a low sample size (n = 23) [[32](#B32-jpm-11-00204)]. In contrast, several pharmacodynamic interactions were published. *CYP3A5* *3/*3 subjects were related to higher risk of myalgia and muscle damage compared to *1/*3 subjects [[33](#B33-jpm-11-00204)] and to increased response to atorvastatin compared to *1/*1 + *1/*3 subjects [[34](#B34-jpm-11-00204)]. Moreover, the *3 allele was associated with increased response to statins, including atorvastatin, compared to the *1 allele [[35](#B35-jpm-11-00204),[36](#B36-jpm-11-00204)]. A higher metabolism of the parent drug, assuming that its metabolites are more active, does not necessarily correlate to a lower risk for toxicity or for drug effectiveness. In contrast, CYP3A4 inhibition relates to toxicity, actually supporting that a higher metabolism indeed relates to lower effectiveness and lower risk for toxicity. Therefore, from these studies, we can conclude that the sum of atorvastatin and atorvastatin active metabolites could be higher in *CYP3A5* non-expressers, and this event relates to a higher risk for toxicity and to a better response to the drug. In our study, *CYP3A5**1 (defined as the absence of *3 and *6 alleles) allele was associated with atorvastatin accumulation (e.g., carriers of *1/*1 exhibited an AUC 1.58 or 1.85 times higher than that of carriers of *1/*3 or *3/*3, respectively, *p* = 0.007). Our study is, to our knowledge, the one with the largest sample size (n = 156) published to date suggesting such an interaction. A possible explanation for this is as follows: atorvastatin, administered in the form of acid, suffers a strong first-pass metabolic effect, involving both intestinal and hepatic CYP3A4 and CYP3A5, with an oral bioavailability of 12–14% [[5](#B5-jpm-11-00204),[6](#B6-jpm-11-00204),[37](#B37-jpm-11-00204)]. CYP3A4 is the main enzyme responsible for atorvastatin metabolism, with an intrinsic clearance 2.4 to 5 times higher than that of CYP3A5 [[31](#B31-jpm-11-00204)]. In *CYP3A5* expressers, atorvastatin is metabolized to a wider extent in the gut, leading to active metabolites with CYP3A4 inhibitory effect. These metabolites, together with atorvastatin, reach the liver and inhibit CYP3A4, which is consistent with the previously demonstrated substrate inhibition [[31](#B31-jpm-11-00204)]. Since in our study population a high dose of atorvastatin was administered to healthy subjects without any atorvastatin in the organism, this effect is evident. That is, the inhibitory effect is greater than the enhanced metabolic capacity. Given the high inducibility of CYP3A4, these effects are likely to be normalized over time in patients in steady state. In our study design, however, there was insufficient time for the induction of CYP3A4 expression to compensate for the inhibition associated with the CYP3A5*1 allele. Clearly, considering the controversy with these results, we encourage other researchers to proceed cautiously with them. Further studies are necessary to replicate this effect.

The *SLC22A1* gene encodes for the organic cationic transporter 1 (OCT), a transporter responsible for the hepatic uptake of xenobiotics and for the capture of organic cations from blood to epithelial cells [[38](#B38-jpm-11-00204)]. There is very little information available regarding atorvastatin and *SLC22A1*. Atorvastatin is known to alter the in vitro expression of *SLC22A1* and in rats co-administered with nicotine [[39](#B39-jpm-11-00204),[40](#B40-jpm-11-00204)]. To date, no study with a robust design or study in humans evaluated if atorvastatin is an OCT1 substrate. This is the first study to date to suggest a similar conclusion. The *SLC22A1**2 allele (rs72552763) or Met420 deletion, was related to a reduced metformin uptake in vitro [[41](#B41-jpm-11-00204)]. Here, *SLC22A1**2 allele was related to a higher C_max_ compared to the *1 allele, which would be consistent with a reduced hepatic uptake and, therefore, to a reduced metabolism; this association disappeared after multivariate analysis. On the other hand, the *SLC22A1**5 allele (rs34059508) was demonstrated to be another reduced-function allele and was related to a reduced metformin uptake in vitro [[41](#B41-jpm-11-00204)]. Here, *1/*5 carriers were related to higher Vd/F. However, this association did not reach the level of significance after Bonferroni correction; these differences were probably explained by the very high standard deviation observed in the *1/*5 group, caused by the presence of outliers.

*UGT2B7*, among other UDP-glucuroniltransferases, was demonstrated to metabolize statins, including atorvastatin [[42](#B42-jpm-11-00204)]. The *2 allele, defined by the rs7439366 variant, was previously associated with reduced activity in diclofenac and efavirenz acyl glucuronidation [[43](#B43-jpm-11-00204),[44](#B44-jpm-11-00204)]. However, to the best of our knowledge, this is the first study to report an effect of this variant on atorvastatin pharmacokinetics. Here, the *1/*1 subject exhibited higher t_max_ compared to *1/*2 but not to *2/*2. Unfortunately, the number of samples analyzed for this variant (n = 36) was small and therefore these findings could be spurious.

The scarcity in ADR data is congruent with the study design, i.e., single-dose administrations. Gastrointestinal symptoms, headache and musculoskeletal ADRs are common based on atorvastatin drug label 5 which is consistent with our findings. The lower risk for headache development observed in men compared to women is likely explained by the lower exposure to atorvastatin observed in men, who were dosed to a lower dose–weight ratio.

It would be particularly interesting to validate these results in a cohort of patients chronically treated with atorvastatin for the management of hypercholesterolemia. For instance, it would be of interest to find out whether patients with a poor or reduced-function SLCO1B1 phenotype required lower doses of atorvastatin or had a higher incidence of myalgias.

### Limitations

The main limitation of this study is that the administration of a single atorvastatin dose to healthy subjects did not permit drawing any conclusion on long-term effectiveness or safety. Further studies are needed to confirm our hypotheses in a stationary state and in patients treated with atorvastatin. In contrast, bioequivalence clinical trials offer a controlled setting for evaluating pharmacokinetic variability based on genetic polymorphism or demographics as confounding factors are avoided.

## 5. Conclusions

The best predictor of atorvastatin exposure is SLCO1B1 phenotype. Accordingly, a dose adjustment could be beneficial to avoid toxicities, especially statin-related myalgias, which could lead to interruption of treatment. This conclusion is consistent with DPWG guideline on atorvastatin and SLCO1B1. We suggest that the CPIC should extend their guideline on simvastatin and SLCO1B1 to atorvastatin as the effect of the transporter phenotype on pharmacokinetics is well demonstrated. Moreover, this candidate-gene study is, to the best of our opinion, the most robust one published to date, with the highest sample size (n = 156) and the widest genotyping screening strategy. In this line, a very novel association was observed, between *CYP3A5**1 and a significant increase in atorvastatin exposure. Further studies are warranted to confirm or reject our findings and hypotheses.

## Supplementary Materials

The following are available online at [https://www.mdpi.com/2075-4426/11/3/204/s1](https://www.mdpi.com/2075-4426/11/3/204/s1), Table S1.

## Author Contributions

Conceptualization, P.Z. and F.A.-S.; methodology, P.Z.; software, P.Z.; validation, F.A.-S.; formal analysis, P.Z.; investigation, P.Z., M.D.B., G.V.-G., S.M.-V., D.O., F.A.-S.; resources, F.A.-S.; data curation, P.Z.; writing—original draft preparation, P.Z.; writing—review and editing, P.Z., M.D.B., G.V.-G., S.M.-V., D.O., F.A.-S.; visualization, P.Z.; supervision, P.Z., F.A.-S.; project administration, P.Z. and F.A.-S.; funding acquisition, F.A.-S. All authors have read and agreed to the published version of the manuscript.

## Funding

G. Villapalos-García is co-financed by Instituto de Salud Carlos III (ISCIII) and the European Social Fund (PFIS predoctoral grant, number FI20/00090). M. Navares-Gómez is financed by the ICI20/00131 grant, Acción Estratégica en Salud 2017-2020, ISCIII.

## Institutional Review Board Statement

Study protocols were revised and approved by the Hospital’s Research Ethics Committee and by the Spanish Drugs Agency (AEMPS). Complying with Spanish and European legislation on research in humans, all of them were accomplished under the Good Clinical Practice guidelines and endorsing the Declaration of Helsinki. EUDRA-CT numbers were as follows: 2018-000082-37, 2019-002222-67, 2019-000891-41, 2019-001670-29 and 2019-000656-34.

## Informed Consent Statement

All the subjects (n = 178) provided their informed consent for their enrolment in the bioequivalence clinical trial. For the pharmacogenetic study, 156 volunteers signed a specific informed consent.

## Data Availability Statement

The data presented in this study are available on request from the corresponding author. The data are not publicly available as they belong to the sponsors of the clinical trials.

## Conflicts of Interest

F. Abad-Santos and D. Ochoa have been consultants or investigators in clinical trials sponsored by the following pharmaceutical companies: Abbott, Alter, Chemo, Cinfa, FAES, Farmalíder, Ferrer, GlaxoSmithKline, Galenicum, Gilead, Italfarmaco Janssen-Cilag, Kern, Normon, Novartis, Servier, Silverpharma, Teva, and Zambon. The remaining authors declare no conflicts of interest.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data presented in this study are available on request from the corresponding author. The data are not publicly available as they belong to the sponsors of the clinical trials.

### Supplementary Materials

### Data Availability Statement

The data presented in this study are available on request from the corresponding author. The data are not publicly available as they belong to the sponsors of the clinical trials.

## References

1. Weng T.-C., Yang Y.-H.K., Lin S.-J., Tai S.-H. A Systematic Review and Meta-Analysis on the Therapeutic Equivalence of Statins. J. Clin. Pharm. Ther. 2010;35:139–151. doi: 10.1111/j.1365-2710.2009.01085.x.  [DOI](https://doi.org/10.1111/j.1365-2710.2009.01085.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20456733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharm.%20Ther.&title=A%20Systematic%20Review%20and%20Meta-Analysis%20on%20the%20Therapeutic%20Equivalence%20of%20Statins&author=T.-C.%20Weng&author=Y.-H.K.%20Yang&author=S.-J.%20Lin&author=S.-H.%20Tai&volume=35&publication_year=2010&pages=139-151&pmid=20456733&doi=10.1111/j.1365-2710.2009.01085.x&)

2. Stancu C., Sima A. Statins: Mechanism of Action and Effects. J. Cell. Mol. Med. 2001;5:378–387. doi: 10.1111/j.1582-4934.2001.tb00172.x.  [DOI](https://doi.org/10.1111/j.1582-4934.2001.tb00172.x) | [PMC free article](/articles/PMC6740083/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12067471/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cell.%20Mol.%20Med.&title=Statins:%20Mechanism%20of%20Action%20and%20Effects&author=C.%20Stancu&author=A.%20Sima&volume=5&publication_year=2001&pages=378-387&pmid=12067471&doi=10.1111/j.1582-4934.2001.tb00172.x&)

3. Malhotra H.S., Goa K.L. Atorvastatin: An Updated Review of Its Pharmacological Properties and Use in Dyslipidaemia. Drugs. 2001;61:1835–1881. doi: 10.2165/00003495-200161120-00012.  [DOI](https://doi.org/10.2165/00003495-200161120-00012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11693468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drugs&title=Atorvastatin:%20An%20Updated%20Review%20of%20Its%20Pharmacological%20Properties%20and%20Use%20in%20Dyslipidaemia&author=H.S.%20Malhotra&author=K.L.%20Goa&volume=61&publication_year=2001&pages=1835-1881&pmid=11693468&doi=10.2165/00003495-200161120-00012&)

4. Liu Y.-M., Pu H.-H., Liu G.-Y., Jia J.-Y., Weng L.-P., Xu R.-J., Li G.-X., Wang W., Zhang M.-Q., Lu C., et al. Pharmacokinetics and Bioequivalence Evaluation of Two Different Atorvastatin Calcium 10-Mg Tablets: A Single-Dose, Randomized-Sequence, Open-Label, Two-Period Crossover Study in Healthy Fasted Chinese Adult Males. Clin. Ther. 2010;32:1396–1407. doi: 10.1016/j.clinthera.2010.07.004.  [DOI](https://doi.org/10.1016/j.clinthera.2010.07.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20678686/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Ther.&title=Pharmacokinetics%20and%20Bioequivalence%20Evaluation%20of%20Two%20Different%20Atorvastatin%20Calcium%2010-Mg%20Tablets:%20A%20Single-Dose,%20Randomized-Sequence,%20Open-Label,%20Two-Period%20Crossover%20Study%20in%20Healthy%20Fasted%20Chinese%20Adult%20Males&author=Y.-M.%20Liu&author=H.-H.%20Pu&author=G.-Y.%20Liu&author=J.-Y.%20Jia&author=L.-P.%20Weng&volume=32&publication_year=2010&pages=1396-1407&pmid=20678686&doi=10.1016/j.clinthera.2010.07.004&)

5. Agencia Española del Medicamento y Productos Sanitarios (AEMPS) Cardyl Comprimidos Recubiertos Con Película (Drug Label), Avda. de Europa, 20B, Parque Empresarial La Moraleja, 28108, Alcobendas, Madrid, Spain

6. García M.J., Reinoso R.F., Sánchez Navarro A., Prous J.R. Clinical Pharmacokinetics of Statins. Methods Find. Exp. Clin. Pharmacol. 2003;25:457–481. doi: 10.1358/mf.2003.25.6.769652.  [DOI](https://doi.org/10.1358/mf.2003.25.6.769652) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12949632/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Find.%20Exp.%20Clin.%20Pharmacol.&title=Clinical%20Pharmacokinetics%20of%20Statins&author=M.J.%20Garc%C3%ADa&author=R.F.%20Reinoso&author=A.%20S%C3%A1nchez%20Navarro&author=J.R.%20Prous&volume=25&publication_year=2003&pages=457-481&pmid=12949632&doi=10.1358/mf.2003.25.6.769652&)

7. Schachter M. Chemical, Pharmacokinetic and Pharmacodynamic Properties of Statins: An Update. Fundam. Clin. Pharmacol. 2005;19:117–125. doi: 10.1111/j.1472-8206.2004.00299.x.  [DOI](https://doi.org/10.1111/j.1472-8206.2004.00299.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15660968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fundam.%20Clin.%20Pharmacol.&title=Chemical,%20Pharmacokinetic%20and%20Pharmacodynamic%20Properties%20of%20Statins:%20An%20Update&author=M.%20Schachter&volume=19&publication_year=2005&pages=117-125&pmid=15660968&doi=10.1111/j.1472-8206.2004.00299.x&)

8. Kalliokoski A., Niemi M. Impact of OATP Transporters on Pharmacokinetics: OATP Transporters and Pharmacokinetics. Br. J. Pharmacol. 2009;158:693–705. doi: 10.1111/j.1476-5381.2009.00430.x.  [DOI](https://doi.org/10.1111/j.1476-5381.2009.00430.x) | [PMC free article](/articles/PMC2765590/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19785645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Pharmacol.&title=Impact%20of%20OATP%20Transporters%20on%20Pharmacokinetics:%20OATP%20Transporters%20and%20Pharmacokinetics&author=A.%20Kalliokoski&author=M.%20Niemi&volume=158&publication_year=2009&pages=693-705&pmid=19785645&doi=10.1111/j.1476-5381.2009.00430.x&)

9. DeGorter M.K., Tirona R.G., Schwarz U.I., Choi Y.-H., Dresser G.K., Suskin N., Myers K., Zou G., Iwuchukwu O., Wei W.-Q., et al. Clinical and Pharmacogenetic Predictors of Circulating Atorvastatin and Rosuvastatin Concentrations in Routine Clinical Care. Circ. Cardiovasc. Genet. 2013;6:400–408. doi: 10.1161/CIRCGENETICS.113.000099.  [DOI](https://doi.org/10.1161/CIRCGENETICS.113.000099) | [PMC free article](/articles/PMC3922121/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23876492/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc.%20Genet.&title=Clinical%20and%20Pharmacogenetic%20Predictors%20of%20Circulating%20Atorvastatin%20and%20Rosuvastatin%20Concentrations%20in%20Routine%20Clinical%20Care&author=M.K.%20DeGorter&author=R.G.%20Tirona&author=U.I.%20Schwarz&author=Y.-H.%20Choi&author=G.K.%20Dresser&volume=6&publication_year=2013&pages=400-408&pmid=23876492&doi=10.1161/CIRCGENETICS.113.000099&)

10. Yue Y.-H., Bai X., Zhang H., Li Y., Hu L., Liu L., Mao J., Yang X., Dila N. Gene Polymorphisms Affect the Effectiveness of Atorvastatin in Treating Ischemic Stroke Patients. Cell. Physiol. Biochem. 2016;39:630–638. doi: 10.1159/000445654.  [DOI](https://doi.org/10.1159/000445654) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27415775/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell.%20Physiol.%20Biochem.&title=Gene%20Polymorphisms%20Affect%20the%20Effectiveness%20of%20Atorvastatin%20in%20Treating%20Ischemic%20Stroke%20Patients&author=Y.-H.%20Yue&author=X.%20Bai&author=H.%20Zhang&author=Y.%20Li&author=L.%20Hu&volume=39&publication_year=2016&pages=630-638&pmid=27415775&doi=10.1159/000445654&)

11. Dutch Pharmacogenetics Working Group Pharmacogenetic Recommendations. 2005.

12. Ramsey L.B., Johnson S.G., Caudle K.E., Haidar C.E., Voora D., Wilke R.A., Maxwell W.D., McLeod H.L., Krauss R.M., Roden D.M., et al. The Clinical Pharmacogenetics Implementation Consortium Guideline for SLCO1B1 and Simvastatin-Induced Myopathy: 2014 Update. Clin. Pharmacol. Ther. 2014;96:423–428. doi: 10.1038/clpt.2014.125.  [DOI](https://doi.org/10.1038/clpt.2014.125) | [PMC free article](/articles/PMC4169720/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24918167/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20Guideline%20for%20SLCO1B1%20and%20Simvastatin-Induced%20Myopathy:%202014%20Update&author=L.B.%20Ramsey&author=S.G.%20Johnson&author=K.E.%20Caudle&author=C.E.%20Haidar&author=D.%20Voora&volume=96&publication_year=2014&pages=423-428&pmid=24918167&doi=10.1038/clpt.2014.125&)

13. Aguirre C., García M. Causality assessment in reports on adverse drug reactions. Algorithm of Spanish pharmacovigilance system. Med. Clin. (Barc) 2016;147:461–464. doi: 10.1016/j.medcli.2016.06.012.  [DOI](https://doi.org/10.1016/j.medcli.2016.06.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27450163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med.%20Clin.%20(Barc)&title=Causality%20assessment%20in%20reports%20on%20adverse%20drug%20reactions.%20Algorithm%20of%20Spanish%20pharmacovigilance%20system&author=C.%20Aguirre&author=M.%20Garc%C3%ADa&volume=147&publication_year=2016&pages=461-464&pmid=27450163&doi=10.1016/j.medcli.2016.06.012&)

14. Belmonte C., Ochoa D., Román M., Saiz-Rodríguez M., Wojnicz A., Gómez-Sánchez C.I., Martín-Vílchez S., Abad-Santos F. Influence of CYP2D6, CYP3A4, CYP3A5 and ABCB1 Polymorphisms on Pharmacokinetics and Safety of Aripiprazole in Healthy Volunteers. Basic Clin. Pharmacol. Toxicol. 2018;122:596–605. doi: 10.1111/bcpt.12960.  [DOI](https://doi.org/10.1111/bcpt.12960) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29325225/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin.%20Pharmacol.%20Toxicol.&title=Influence%20of%20CYP2D6,%20CYP3A4,%20CYP3A5%20and%20ABCB1%20Polymorphisms%20on%20Pharmacokinetics%20and%20Safety%20of%20Aripiprazole%20in%20Healthy%20Volunteers&author=C.%20Belmonte&author=D.%20Ochoa&author=M.%20Rom%C3%A1n&author=M.%20Saiz-Rodr%C3%ADguez&author=A.%20Wojnicz&volume=122&publication_year=2018&pages=596-605&pmid=29325225&doi=10.1111/bcpt.12960&)

15. Caudle K.E., Sangkuhl K., Whirl-Carrillo M., Swen J.J., Haidar C.E., Klein T.E., Gammal R.S., Relling M.V., Scott S.A., Hertz D.L., et al. Standardizing CYP 2D6 Genotype to Phenotype Translation: Consensus Recommendations from the Clinical Pharmacogenetics Implementation Consortium and Dutch Pharmacogenetics Working Group. Clin. Transl. Sci. 2019 doi: 10.1111/cts.12692.  [DOI](https://doi.org/10.1111/cts.12692) | [PMC free article](/articles/PMC6951851/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31647186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Transl.%20Sci.&title=Standardizing%20CYP%202D6%20Genotype%20to%20Phenotype%20Translation:%20Consensus%20Recommendations%20from%20the%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20and%20Dutch%20Pharmacogenetics%20Working%20Group&author=K.E.%20Caudle&author=K.%20Sangkuhl&author=M.%20Whirl-Carrillo&author=J.J.%20Swen&author=C.E.%20Haidar&publication_year=2019&pmid=31647186&doi=10.1111/cts.12692&)

16. Scott S.A., Sangkuhl K., Stein C.M., Hulot J.-S., Mega J.L., Roden D.M., Klein T.E., Sabatine M.S., Johnson J.A., Shuldiner A.R. Clinical Pharmacogenetics Implementation Consortium Guidelines for CYP2C19 Genotype and Clopidogrel Therapy: 2013 Update. Clin. Pharmacol. Ther. 2013;94:317–323. doi: 10.1038/clpt.2013.105.  [DOI](https://doi.org/10.1038/clpt.2013.105) | [PMC free article](/articles/PMC3748366/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23698643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20Guidelines%20for%20CYP2C19%20Genotype%20and%20Clopidogrel%20Therapy:%202013%20Update&author=S.A.%20Scott&author=K.%20Sangkuhl&author=C.M.%20Stein&author=J.-S.%20Hulot&author=J.L.%20Mega&volume=94&publication_year=2013&pages=317-323&pmid=23698643&doi=10.1038/clpt.2013.105&)

17. Caudle K.E., Rettie A.E., Whirl-Carrillo M., Smith L.H., Mintzer S., Lee M.T.M., Klein T.E., Callaghan J.T. Clinical Pharmacogenetics Implementation Consortium Guidelines for CYP2C9 and HLA-B Genotypes and Phenytoin Dosing. Clin. Pharmacol. Ther. 2014;96:542–548. doi: 10.1038/clpt.2014.159.  [DOI](https://doi.org/10.1038/clpt.2014.159) | [PMC free article](/articles/PMC4206662/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25099164/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20Guidelines%20for%20CYP2C9%20and%20HLA-B%20Genotypes%20and%20Phenytoin%20Dosing&author=K.E.%20Caudle&author=A.E.%20Rettie&author=M.%20Whirl-Carrillo&author=L.H.%20Smith&author=S.%20Mintzer&volume=96&publication_year=2014&pages=542-548&pmid=25099164&doi=10.1038/clpt.2014.159&)

18. Desta Z., Gammal R.S., Gong L., Whirl-Carrillo M., Gaur A.H., Sukasem C., Hockings J., Myers A., Swart M., Tyndale R.F., et al. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for CYP2B6 and Efavirenz-Containing Antiretroviral Therapy. Clin. Pharmacol. Ther. 2019;106:726–733. doi: 10.1002/cpt.1477.  [DOI](https://doi.org/10.1002/cpt.1477) | [PMC free article](/articles/PMC6739160/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31006110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20Guideline%20for%20CYP2B6%20and%20Efavirenz-Containing%20Antiretroviral%20Therapy&author=Z.%20Desta&author=R.S.%20Gammal&author=L.%20Gong&author=M.%20Whirl-Carrillo&author=A.H.%20Gaur&volume=106&publication_year=2019&pages=726-733&pmid=31006110&doi=10.1002/cpt.1477&)

19. Birdwell K., Decker B., Barbarino J., Peterson J., Stein C., Sadee W., Wang D., Vinks A., He Y., Swen J., et al. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guidelines for CYP3A5 Genotype and Tacrolimus Dosing. Clin. Pharmacol. Ther. 2015;98:19–24. doi: 10.1002/cpt.113.  [DOI](https://doi.org/10.1002/cpt.113) | [PMC free article](/articles/PMC4481158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25801146/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20Guidelines%20for%20CYP3A5%20Genotype%20and%20Tacrolimus%20Dosing&author=K.%20Birdwell&author=B.%20Decker&author=J.%20Barbarino&author=J.%20Peterson&author=C.%20Stein&volume=98&publication_year=2015&pages=19-24&pmid=25801146&doi=10.1002/cpt.113&)

20. Zubiaur P., Saiz-Rodríguez M., Ochoa D., Belmonte C., Román M., Mejía G., Martín-Vilchez S., Abad-Santos F. Influence of CYP2B6 Activity Score on the Pharmacokinetics and Safety of Single Dose Efavirenz in Healthy Volunteers. Pharmacogenom. J. 2019 doi: 10.1038/s41397-019-0103-3.  [DOI](https://doi.org/10.1038/s41397-019-0103-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31628422/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom.%20J.&title=Influence%20of%20CYP2B6%20Activity%20Score%20on%20the%20Pharmacokinetics%20and%20Safety%20of%20Single%20Dose%20Efavirenz%20in%20Healthy%20Volunteers&author=P.%20Zubiaur&author=M.%20Saiz-Rodr%C3%ADguez&author=D.%20Ochoa&author=C.%20Belmonte&author=M.%20Rom%C3%A1n&publication_year=2019&pmid=31628422&doi=10.1038/s41397-019-0103-3&)

21. Saiz-Rodríguez M., Ochoa D., Belmonte C., Román M., Vieira de Lara D., Zubiaur P., Koller D., Mejía G., Abad-Santos F. Polymorphisms in CYP1A2, CYP2C9 and ABCB1 Affect Agomelatine Pharmacokinetics. J. Psychopharmacol. 2019;33:522–531. doi: 10.1177/0269881119827959.  [DOI](https://doi.org/10.1177/0269881119827959) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30789308/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Psychopharmacol.&title=Polymorphisms%20in%20CYP1A2,%20CYP2C9%20and%20ABCB1%20Affect%20Agomelatine%20Pharmacokinetics&author=M.%20Saiz-Rodr%C3%ADguez&author=D.%20Ochoa&author=C.%20Belmonte&author=M.%20Rom%C3%A1n&author=D.%20Vieira%20de%20Lara&volume=33&publication_year=2019&pages=522-531&pmid=30789308&doi=10.1177/0269881119827959&)

22. Mittal B., Tulsyan S., Mittal R. The Effect of ABCB1 Polymorphisms on the Outcome of Breast Cancer Treatment. Pharmacogenom. Pers. Med. 2016;9:47–58. doi: 10.2147/PGPM.S86672.  [DOI](https://doi.org/10.2147/PGPM.S86672) | [PMC free article](/articles/PMC4854269/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27175090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom.%20Pers.%20Med.&title=The%20Effect%20of%20ABCB1%20Polymorphisms%20on%20the%20Outcome%20of%20Breast%20Cancer%20Treatment&author=B.%20Mittal&author=S.%20Tulsyan&author=R.%20Mittal&volume=9&publication_year=2016&pages=47-58&pmid=27175090&doi=10.2147/PGPM.S86672&)

23. Graffelman J., Weir B.S. Testing for Hardy–Weinberg Equilibrium at Biallelic Genetic Markers on the X Chromosome. Heredity. 2016;116:558–568. doi: 10.1038/hdy.2016.20.  [DOI](https://doi.org/10.1038/hdy.2016.20) | [PMC free article](/articles/PMC4868269/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27071844/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heredity&title=Testing%20for%20Hardy%E2%80%93Weinberg%20Equilibrium%20at%20Biallelic%20Genetic%20Markers%20on%20the%20X%20Chromosome&author=J.%20Graffelman&author=B.S.%20Weir&volume=116&publication_year=2016&pages=558-568&pmid=27071844&doi=10.1038/hdy.2016.20&)

24. Ward N.C., Watts G.F., Eckel R.H. Statin Toxicity: Mechanistic Insights and Clinical Implications. Circ. Res. 2019;124:328–350. doi: 10.1161/CIRCRESAHA.118.312782.  [DOI](https://doi.org/10.1161/CIRCRESAHA.118.312782) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31170055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Res.&title=Statin%20Toxicity:%20Mechanistic%20Insights%20and%20Clinical%20Implications&author=N.C.%20Ward&author=G.F.%20Watts&author=R.H.%20Eckel&volume=124&publication_year=2019&pages=328-350&pmid=31170055&doi=10.1161/CIRCRESAHA.118.312782&)

25. Zubiaur P., Soria-Chacartegui P., Koller D., Navares-Gómez M., Ochoa D., Almenara S., Saiz-Rodriguez M., Mejía-Abril G., Villapalos-García G., Román M., et al. Impact of Polymorphisms in Transporter and Metabolizing Enzyme Genes on Olanzapine Pharmacokinetics and Safety in Healthy Volunteer. Biomed. Pharmacother. 2021;133:111087. doi: 10.1016/j.biopha.2020.111087.  [DOI](https://doi.org/10.1016/j.biopha.2020.111087) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33378980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed.%20Pharmacother.&title=Impact%20of%20Polymorphisms%20in%20Transporter%20and%20Metabolizing%20Enzyme%20Genes%20on%20Olanzapine%20Pharmacokinetics%20and%20Safety%20in%20Healthy%20Volunteer&author=P.%20Zubiaur&author=P.%20Soria-Chacartegui&author=D.%20Koller&author=M.%20Navares-G%C3%B3mez&author=D.%20Ochoa&volume=133&publication_year=2021&pages=111087&pmid=33378980&doi=10.1016/j.biopha.2020.111087&)

26. Park J., Kim C.O., Jin B.H., Yang S., Park M.S., Hong T. Pharmacokinetic Drug Interaction between Atorvastatin and Ezetimibe in Healthy Korean Volunteers. Transl. Clin. Pharmacol. 2017;25:202. doi: 10.12793/tcp.2017.25.4.202.  [DOI](https://doi.org/10.12793/tcp.2017.25.4.202) | [PMC free article](/articles/PMC7033406/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32095476/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transl.%20Clin.%20Pharmacol.&title=Pharmacokinetic%20Drug%20Interaction%20between%20Atorvastatin%20and%20Ezetimibe%20in%20Healthy%20Korean%20Volunteers&author=J.%20Park&author=C.O.%20Kim&author=B.H.%20Jin&author=S.%20Yang&author=M.S.%20Park&volume=25&publication_year=2017&pages=202&pmid=32095476&doi=10.12793/tcp.2017.25.4.202&)

27. Gibson D.M., Bron N.J., Richens M.A., Hounslow N.J., Sedman A.J., Whitfield L.R. Effect of Age and Gender on Pharmacokinetics of Atorvastatin in Humans. J. Clin. Pharmacol. 1996;36:242–246. doi: 10.1002/j.1552-4604.1996.tb04194.x.  [DOI](https://doi.org/10.1002/j.1552-4604.1996.tb04194.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8690818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharmacol.&title=Effect%20of%20Age%20and%20Gender%20on%20Pharmacokinetics%20of%20Atorvastatin%20in%20Humans&author=D.M.%20Gibson&author=N.J.%20Bron&author=M.A.%20Richens&author=N.J.%20Hounslow&author=A.J.%20Sedman&volume=36&publication_year=1996&pages=242-246&pmid=8690818&doi=10.1002/j.1552-4604.1996.tb04194.x&)

28. Romaine S.P.R., Bailey K.M., Hall A.S., Balmforth A.J. The Influence of SLCO1B1 (OATP1B1) Gene Polymorphisms on Response to Statin Therapy. Pharmacogenom. J. 2010;10:1–11. doi: 10.1038/tpj.2009.54.  [DOI](https://doi.org/10.1038/tpj.2009.54) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19884908/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom.%20J.&title=The%20Influence%20of%20SLCO1B1%20(OATP1B1)%20Gene%20Polymorphisms%20on%20Response%20to%20Statin%20Therapy&author=S.P.R.%20Romaine&author=K.M.%20Bailey&author=A.S.%20Hall&author=A.J.%20Balmforth&volume=10&publication_year=2010&pages=1-11&pmid=19884908&doi=10.1038/tpj.2009.54&)

29. He Y.-J., Zhang W., Chen Y., Guo D., Tu J.-H., Xu L.-Y., Tan Z.-R., Chen B.-L., Li Z., Zhou G., et al. Rifampicin Alters Atorvastatin Plasma Concentration on the Basis of SLCO1B1 521T>C Polymorphism. Clin. Chim. Acta. 2009;405:49–52. doi: 10.1016/j.cca.2009.04.003.  [DOI](https://doi.org/10.1016/j.cca.2009.04.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19374892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Chim.%20Acta&title=Rifampicin%20Alters%20Atorvastatin%20Plasma%20Concentration%20on%20the%20Basis%20of%20SLCO1B1%20521T>C%20Polymorphism&author=Y.-J.%20He&author=W.%20Zhang&author=Y.%20Chen&author=D.%20Guo&author=J.-H.%20Tu&volume=405&publication_year=2009&pages=49-52&pmid=19374892&doi=10.1016/j.cca.2009.04.003&)

30. Rodrigues A.C., Perin P.M.S., Purim S.G., Silbiger V.N., Genvigir F.D.V., Willrich M.A.V., Arazi S.S., Luchessi A.D., Hirata M.H., Bernik M.M.S., et al. Pharmacogenetics of OATP Transporters Reveals That SLCO1B1 c.388A>G Variant Is Determinant of Increased Atorvastatin Response. Int. J. Mol. Sci. 2011;12:5815–5827. doi: 10.3390/ijms12095815.  [DOI](https://doi.org/10.3390/ijms12095815) | [PMC free article](/articles/PMC3189752/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22016628/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Pharmacogenetics%20of%20OATP%20Transporters%20Reveals%20That%20SLCO1B1%20c.388A>G%20Variant%20Is%20Determinant%20of%20Increased%20Atorvastatin%20Response&author=A.C.%20Rodrigues&author=P.M.S.%20Perin&author=S.G.%20Purim&author=V.N.%20Silbiger&author=F.D.V.%20Genvigir&volume=12&publication_year=2011&pages=5815-5827&pmid=22016628&doi=10.3390/ijms12095815&)

31. Park J.-E., Kim K.-B., Bae S.K., Moon B.-S., Liu K.-H., Shin J.-G. Contribution of Cytochrome P450 3A4 and 3A5 to the Metabolism of Atorvastatin. Xenobiotica. 2008;38:1240–1251. doi: 10.1080/00498250802334391.  [DOI](https://doi.org/10.1080/00498250802334391) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18720283/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Contribution%20of%20Cytochrome%20P450%203A4%20and%203A5%20to%20the%20Metabolism%20of%20Atorvastatin&author=J.-E.%20Park&author=K.-B.%20Kim&author=S.K.%20Bae&author=B.-S.%20Moon&author=K.-H.%20Liu&volume=38&publication_year=2008&pages=1240-1251&pmid=18720283&doi=10.1080/00498250802334391&)

32. Shin J., Pauly D.F., Pacanowski M.A., Langaee T., Frye R.F., Johnson J.A. Effect of Cytochrome P450 3A5 Genotype on Atorvastatin Pharmacokinetics and Its Interaction with Clarithromycin. Pharmacotherapy. 2011;31:942–950. doi: 10.1592/phco.31.10.942.  [DOI](https://doi.org/10.1592/phco.31.10.942) | [PMC free article](/articles/PMC3712822/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21950641/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&title=Effect%20of%20Cytochrome%20P450%203A5%20Genotype%20on%20Atorvastatin%20Pharmacokinetics%20and%20Its%20Interaction%20with%20Clarithromycin&author=J.%20Shin&author=D.F.%20Pauly&author=M.A.%20Pacanowski&author=T.%20Langaee&author=R.F.%20Frye&volume=31&publication_year=2011&pages=942-950&pmid=21950641&doi=10.1592/phco.31.10.942&)

33. Wilke R.A., Moore J.H., Burmester J.K. Relative Impact of CYP3A Genotype and Concomitant Medication on the Severity of Atorvastatin-Induced Muscle Damage. Pharmacogenet. Genom. 2005;15:415–421. doi: 10.1097/01213011-200506000-00007.  [DOI](https://doi.org/10.1097/01213011-200506000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15900215/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=Relative%20Impact%20of%20CYP3A%20Genotype%20and%20Concomitant%20Medication%20on%20the%20Severity%20of%20Atorvastatin-Induced%20Muscle%20Damage&author=R.A.%20Wilke&author=J.H.%20Moore&author=J.K.%20Burmester&volume=15&publication_year=2005&pages=415-421&pmid=15900215&doi=10.1097/01213011-200506000-00007&)

34. Willrich M.A.V., Hirata M.H., Genvigir F.D.V., Arazi S.S., Rebecchi I.M.M., Rodrigues A.C., Bernik M.M.S., Dorea E.L., Bertolami M.C., Faludi A.A., et al. CYP3A53A Allele Is Associated with Reduced Lowering-Lipid Response to Atorvastatin in Individuals with Hypercholesterolemia. Clin. Chim. Acta. 2008;398:15–20. doi: 10.1016/j.cca.2008.07.032.  [DOI](https://doi.org/10.1016/j.cca.2008.07.032) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18727922/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Chim.%20Acta&title=CYP3A53A%20Allele%20Is%20Associated%20with%20Reduced%20Lowering-Lipid%20Response%20to%20Atorvastatin%20in%20Individuals%20with%20Hypercholesterolemia&author=M.A.V.%20Willrich&author=M.H.%20Hirata&author=F.D.V.%20Genvigir&author=S.S.%20Arazi&author=I.M.M.%20Rebecchi&volume=398&publication_year=2008&pages=15-20&pmid=18727922&doi=10.1016/j.cca.2008.07.032&)

35. Kivistö K.T., Niemi M., Schaeffeler E., Pitkälä K., Tilvis R., Fromm M.F., Schwab M., Eichelbaum M., Strandberg T. Lipid-Lowering Response to Statins Is Affected by CYP3A5 Polymorphism. Pharmacogenetics. 2004;14:523–525. doi: 10.1097/01.fpc.0000114762.78957.a5.  [DOI](https://doi.org/10.1097/01.fpc.0000114762.78957.a5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15284534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Lipid-Lowering%20Response%20to%20Statins%20Is%20Affected%20by%20CYP3A5%20Polymorphism&author=K.T.%20Kivist%C3%B6&author=M.%20Niemi&author=E.%20Schaeffeler&author=K.%20Pitk%C3%A4l%C3%A4&author=R.%20Tilvis&volume=14&publication_year=2004&pages=523-525&pmid=15284534&doi=10.1097/01.fpc.0000114762.78957.a5&)

36. Thompson J.F., Man M., Johnson K.J., Wood L.S., Lira M.E., Lloyd D.B., Banerjee P., Milos P.M., Myrand S.P., Paulauskis J., et al. An Association Study of 43 SNPs in 16 Candidate Genes with Atorvastatin Response. Pharmacogenom. J. 2005;5:352–358. doi: 10.1038/sj.tpj.6500328.  [DOI](https://doi.org/10.1038/sj.tpj.6500328) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16103896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom.%20J.&title=An%20Association%20Study%20of%2043%20SNPs%20in%2016%20Candidate%20Genes%20with%20Atorvastatin%20Response&author=J.F.%20Thompson&author=M.%20Man&author=K.J.%20Johnson&author=L.S.%20Wood&author=M.E.%20Lira&volume=5&publication_year=2005&pages=352-358&pmid=16103896&doi=10.1038/sj.tpj.6500328&)

37. Lennernäs H. Clinical Pharmacokinetics of Atorvastatin. Clin. Pharmacokinet. 2003;42:1141–1160. doi: 10.2165/00003088-200342130-00005.  [DOI](https://doi.org/10.2165/00003088-200342130-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14531725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet.&title=Clinical%20Pharmacokinetics%20of%20Atorvastatin&author=H.%20Lennern%C3%A4s&volume=42&publication_year=2003&pages=1141-1160&pmid=14531725&doi=10.2165/00003088-200342130-00005&)

38. Goswami S., Gong L., Giacomini K., Altman R.B., Klein T.E. PharmGKB Summary: Very Important Pharmacogene Information for SLC22A1. Pharmacogenet. Genom. 2014;24:324–328. doi: 10.1097/FPC.0000000000000048.  [DOI](https://doi.org/10.1097/FPC.0000000000000048) | [PMC free article](/articles/PMC4035531/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24681965/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=PharmGKB%20Summary:%20Very%20Important%20Pharmacogene%20Information%20for%20SLC22A1&author=S.%20Goswami&author=L.%20Gong&author=K.%20Giacomini&author=R.B.%20Altman&author=T.E.%20Klein&volume=24&publication_year=2014&pages=324-328&pmid=24681965&doi=10.1097/FPC.0000000000000048&)

39. Rodrigues A.C., Curi R., Genvigir F.D.V., Hirata M.H., Hirata R.D.C. The Expression of Efflux and Uptake Transporters Are Regulated by Statins in Caco-2 and HepG2 Cells. Acta Pharmacol. Sin. 2009;30:956–964. doi: 10.1038/aps.2009.85.  [DOI](https://doi.org/10.1038/aps.2009.85) | [PMC free article](/articles/PMC4006650/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19543298/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Pharmacol.%20Sin.&title=The%20Expression%20of%20Efflux%20and%20Uptake%20Transporters%20Are%20Regulated%20by%20Statins%20in%20Caco-2%20and%20HepG2%20Cells&author=A.C.%20Rodrigues&author=R.%20Curi&author=F.D.V.%20Genvigir&author=M.H.%20Hirata&author=R.D.C.%20Hirata&volume=30&publication_year=2009&pages=956-964&pmid=19543298&doi=10.1038/aps.2009.85&)

40. Syam Das S., Nair S.S., Indira M. Atorvastatin Modulates Drug Transporters and Ameliorates Nicotine-Induced Testicular Toxicity. Andrologia. 2018;50:e13029. doi: 10.1111/and.13029.  [DOI](https://doi.org/10.1111/and.13029) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29740849/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Andrologia&title=Atorvastatin%20Modulates%20Drug%20Transporters%20and%20Ameliorates%20Nicotine-Induced%20Testicular%20Toxicity&author=S.%20Syam%20Das&author=S.S.%20Nair&author=M.%20Indira&volume=50&publication_year=2018&pages=e13029&pmid=29740849&doi=10.1111/and.13029&)

41. Shu Y., Sheardown S.A., Brown C., Owen R.P., Zhang S., Castro R.A., Ianculescu A.G., Yue L., Lo J.C., Burchard E.G., et al. Effect of Genetic Variation in the Organic Cation Transporter 1 (OCT1) on Metformin Action. J. Clin. Investig. 2007;117:1422–1431. doi: 10.1172/JCI30558.  [DOI](https://doi.org/10.1172/JCI30558) | [PMC free article](/articles/PMC1857259/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17476361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Investig.&title=Effect%20of%20Genetic%20Variation%20in%20the%20Organic%20Cation%20Transporter%201%20(OCT1)%20on%20Metformin%20Action&author=Y.%20Shu&author=S.A.%20Sheardown&author=C.%20Brown&author=R.P.%20Owen&author=S.%20Zhang&volume=117&publication_year=2007&pages=1422-1431&pmid=17476361&doi=10.1172/JCI30558&)

42. Sakaeda T., Fujino H., Komoto C., Kakumoto M., Jin J., Iwaki K., Nishiguchi K., Nakamura T., Okamura N., Okumura K. Effects of Acid and Lactone Forms of Eight HMG-CoA Reductase Inhibitors on CYP-Mediated Metabolism and MDR1-Mediated Transport. Pharm. Res. 2006;23:506–512. doi: 10.1007/s11095-005-9371-5.  [DOI](https://doi.org/10.1007/s11095-005-9371-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16388406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Res.&title=Effects%20of%20Acid%20and%20Lactone%20Forms%20of%20Eight%20HMG-CoA%20Reductase%20Inhibitors%20on%20CYP-Mediated%20Metabolism%20and%20MDR1-Mediated%20Transport&author=T.%20Sakaeda&author=H.%20Fujino&author=C.%20Komoto&author=M.%20Kakumoto&author=J.%20Jin&volume=23&publication_year=2006&pages=506-512&pmid=16388406&doi=10.1007/s11095-005-9371-5&)

43. Haas D.W., Kwara A., Richardson D.M., Baker P., Papageorgiou I., Acosta E.P., Morse G.D., Court M.H. Secondary Metabolism Pathway Polymorphisms and Plasma Efavirenz Concentrations in HIV-Infected Adults with CYP2B6 Slow Metabolizer Genotypes. J. Antimicrob. Chemother. 2014;69:2175–2182. doi: 10.1093/jac/dku110.  [DOI](https://doi.org/10.1093/jac/dku110) | [PMC free article](/articles/PMC4100708/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24729586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Antimicrob.%20Chemother.&title=Secondary%20Metabolism%20Pathway%20Polymorphisms%20and%20Plasma%20Efavirenz%20Concentrations%20in%20HIV-Infected%20Adults%20with%20CYP2B6%20Slow%20Metabolizer%20Genotypes&author=D.W.%20Haas&author=A.%20Kwara&author=D.M.%20Richardson&author=P.%20Baker&author=I.%20Papageorgiou&volume=69&publication_year=2014&pages=2175-2182&pmid=24729586&doi=10.1093/jac/dku110&)

44. Lazarska K.E., Dekker S.J., Vermeulen N.P.E., Commandeur J.N.M. Effect of UGT2B7*2 and CYP2C8*4 Polymorphisms on Diclofenac Metabolism. Toxicol. Lett. 2018;284:70–78. doi: 10.1016/j.toxlet.2017.11.038.  [DOI](https://doi.org/10.1016/j.toxlet.2017.11.038) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29203276/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol.%20Lett.&title=Effect%20of%20UGT2B7*2%20and%20CYP2C8*4%20Polymorphisms%20on%20Diclofenac%20Metabolism&author=K.E.%20Lazarska&author=S.J.%20Dekker&author=N.P.E.%20Vermeulen&author=J.N.M.%20Commandeur&volume=284&publication_year=2018&pages=70-78&pmid=29203276&doi=10.1016/j.toxlet.2017.11.038&)
