# Role of enterocyte-specific gene polymorphisms in response to adjuvant treatment for stage III colorectal cancer

## Metadata
**Authors:** Mitsukuni Suenaga, Shu Cao, Wu Zhang, Satoshi Matsusaka, Satoshi Okazaki, Martin D Berger, Yuji Miyamoto, Marta Schirripa, Afsaneh Barzi, Noriko Yamamoto, Toshiharu Yamaguchi, Heinz-Josef Lenz
**Journal:** Pharmacogenetics and genomics
**Date:** 2021 Jan
**DOI:** [10.1097/FPC.0000000000000416](https://doi.org/10.1097/FPC.0000000000000416)
**PMID:** 32732498
**PMCID:** PMC7655616
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7655616/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7655616/pdf/nihms-1605709.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7655616/pdf/nihms-1605709.pdf)

## Abstract

**Objectives:** 
The enterocyte subtype of colorectal cancer (CRC) responds favorably to oxaliplatin-based adjuvant treatment for stage III CRC. We examined the clinical significance of single nucleotide polymorphisms (SNPs) in enterocyte-related genes MS4A12 and CDX2 in response to adjuvant treatment for stage III CRC.

**Patients and Methods:** 
A total of 350 patients with stage III CRC were included: 274 received adjuvant treatment with surgical resection (discovery cohort) and 76 received surgery alone (control cohort). In the discovery cohort, 68 patients received FOLFOX and 206 received oral fluoropyrimidine. SNPs were analyzed by PCR-based direct sequencing. Disease-free and overall survival were analyzed using Kaplan–Meier curves, log-rank test, and Cox proportional hazards regression.

**Results:** 
In the discovery cohort, the MS4A12 rs4939378 G/G variant was associated with lower 5-year survival than any A allele (70% vs. 90%, univariate: HR 2.29, 95%CI: 1.03–5.06, P=0.035; multivariate: HR 2.58, 95%CI: 1.15–5.76, P=0.021). Patients with the CDX2 rs3812863 G/G variant had better overall survival than those with any A allele, although this was not significant in multivariate analysis (5 year-survival: 95% vs. 82%, univariate: HR 0.34, 95%CI: 0.12–0.97, P=0.034; multivariate: HR 0.39, 95%CI: 0.13–1.11, P=0.078). The SNPs did not show significant association with overall survival in the control cohort, and significant interaction was observed between MS4A12 genotypes and groups (P=0.007).

**Conclusion:** 
Our findings suggest that MS4A12 and CDX2 gene polymorphisms may predict outcome in stage III CRC. However, the clinical significance of SNPs for response to oxaliplatin may differ by tumor stage.

Keywords: enterocyte, MS4A12, CDX2, oxaliplatin, early colorectal cancer

### Objectives:

The enterocyte subtype of colorectal cancer (CRC) responds favorably to oxaliplatin-based adjuvant treatment for stage III CRC. We examined the clinical significance of single nucleotide polymorphisms (SNPs) in enterocyte-related genes *MS4A12* and *CDX2* in response to adjuvant treatment for stage III CRC.

### Patients and Methods:

A total of 350 patients with stage III CRC were included: 274 received adjuvant treatment with surgical resection (discovery cohort) and 76 received surgery alone (control cohort). In the discovery cohort, 68 patients received FOLFOX and 206 received oral fluoropyrimidine. SNPs were analyzed by PCR-based direct sequencing. Disease-free and overall survival were analyzed using Kaplan–Meier curves, log-rank test, and Cox proportional hazards regression.

### Results:

In the discovery cohort, the *MS4A12* rs4939378 G/G variant was associated with lower 5-year survival than any A allele (70% vs. 90%, univariate: HR 2.29, 95%CI: 1.03–5.06, *P*=0.035; multivariate: HR 2.58, 95%CI: 1.15–5.76, *P*=0.021). Patients with the *CDX2* rs3812863 G/G variant had better overall survival than those with any A allele, although this was not significant in multivariate analysis (5 year-survival: 95% vs. 82%, univariate: HR 0.34, 95%CI: 0.12–0.97, *P*=0.034; multivariate: HR 0.39, 95%CI: 0.13–1.11, *P*=0.078). The SNPs did not show significant association with overall survival in the control cohort, and significant interaction was observed between *MS4A12* genotypes and groups (*P*=0.007).

### Conclusion:

Our findings suggest that *MS4A12* and *CDX2* gene polymorphisms may predict outcome in stage III CRC. However, the clinical significance of SNPs for response to oxaliplatin may differ by tumor stage.

## Introduction

Colorectal cancer (CRC) is the third leading cause of cancer death in the world [[1](#R1)]. Two different classifiers, the CRC Assigner (CRCA) classifier [[2](#R2)] and the Colon Cancer Subtypes (CCS) classifier [[3](#R3)], have been reported and divide CRC into different groups. The CRCA assigner defines six CRC subtypes: enterocyte, goblet-like, inflammatory, stem-like, transit-amplifying (TA) composed of cetuximab-resistant TA and cetuximab-sensitive TA. Among them, CA1, CA2, KRT20, SLC26A3, AQP8, and MS4A12 are enterocyte-specific markers.

Enterocytes are hyperpolarized epithelial cells, joined together by tight junctions, that participate in digestive processes and are often directly involved in immune processes [[4](#R4)]. MS4A12 is an enterocyte subtype-specific marker of the CRC Assigner classifier [[2](#R2)]. In healthy human tissue, MS4A12 expression is confined to normal colon mucosa, whereas MS4A12 expression is detected in colon cancer tissue, but not in other cancer types [[5](#R5)]. A bioinformatics analysis of samples from colon cancer patients identified MS4A12 expression as a prognostic marker in early stage CRC but not in advanced stage [[6](#R6)]. MS4A12 transcript and protein expressions are selectively activated by CDX2 [[7](#R7)], which regulates intestinal development and also functions in oncogenesis. CDX2-negative CRC is associated with advanced stage and poor prognostic factors such as BRAF mutation as well as microsatellite stable tumors. Lack of CDX2 expression identifies a subgroup of patients with high-risk stage II CRC favorable to adjuvant chemotherapy [[8](#R8)].

A previous study reported that the enterocyte subtype according to the Colorectal Cancer (CRC) Assigner classifier is associated with sensitivity to oxaliplatin in adjuvant treatment for stage III CRC [[9](#R9)]. We therefore conducted a retrospective study. We investigated the clinical significance of single nucleotide polymorphisms (SNPs) in the enterocyte-related gene MS4A12 and its activator caudal-type homeobox transcription factor 2 (CDX2) in predicting oxaliplatin efficacy in first-line treatment for metastatic CRC (mCRC) using one cohort receiving FOLFOX or FOLFOXIRI and a control cohort receiving FOLFIRI [[10](#R10)]. The results showed that genetic variants in *MS4A12* and *CDX2* gene polymorphisms were potential predictive markers of oxaliplatin-based treatment in mCRC patients.

In this study, we examined clinical significance of *MS4A12* and *CDX2* SNPs in the response to adjuvant treatment with or without oxaliplatin for stage III CRC.

## Materials and methods

### Study design and patients

A total of 350 patients with stage III CRC were included in this study: 274 received adjuvant treatment with surgical resection (discovery cohort) and 76 received surgery alone (control cohort). All patients were Japanese patients who were treated at the Cancer Institute Hospital (Tokyo, Japan) and all patients provided signed informed consent.

Eligible patients had a histologically confirmed diagnosis of stage III colon cancer and received surgical resection with D2 or greater lymphadenectomy confirmed as curative resection (curative level A) with no pathologically residual tumors. Among the 274 patients in the discovery cohort, 68 patients received FOLFOX and 206 received oral fluoropyrimidine. There was no clear criteria for treatment selection. FOLFOX treatment (oxaliplatin 85 mg/m^2^, 5-fluorouracil [5-FU] bolus 400 mg/m^2^, 5-FU infusion 2400 mg/m^2^, levofolinate calcium 200 mg/m^2^, with or without BEV 5 mg/kg) was administered every 2 weeks. Capecitabine was administered every 3 weeks for a total of eight cycles; capecitabine 1,000 mg/m^2^ was given twice daily from day 1 to day 14. Uracil and tegafur plus leucovorin (UFT/LV) was administered every five weeks for total of five cycles; UFT 300 mg/m^2^ and LV 25 mg three times daily 8 h apart on days 1–28.

The study was approved by the Institutional Review Board of each participating site, and the molecular analyses were conducted at the University of Southern California/Norris Comprehensive Cancer Center in accordance with the Declaration of Helsinki and Good Clinical Practice Guidelines. We were fully compliant with the Reporting Recommendations for Tumor Marker Prognostic Studies (REMARK) guidelines.

### Selection of candidate single-nucleotide polymorphisms (SNPs)

The candidate SNPs of genes were selected using any of the following criteria: i) SNP with biological significance according to a published literature review, ii) tagging SNPs selected by the HapMap genotype data with r^2^threshold = 0.8: [http://snpinfo.niehs.nih.gov/snpinfo/snptag.html](http://snpinfo.niehs.nih.gov/snpinfo/snptag.html), or iii) minor allele frequency ≥10% in Caucasians (in the Ensembl Genome Browser: [http://uswest.ensembl.org/index.html](http://uswest.ensembl.org/index.html)). Functional significance was predicted based on the functional SNP (F-SNP) database [[11](#R11)]. To identify the potential gene function, we referred to other public databases ([https://www.ncbi.nlm.nih.gov](https://www.ncbi.nlm.nih.gov/), [https://www.genecards.org/](https://www.genecards.org/)). According to our previous report [[10](#R10)], two SNPs (*MS4A12* rs4939378 gene; *CDX2* rs3812863 gene) were finally selected for this investigation ([Table 1](#T1)).

### Table 1.

| Genesrs number | Allele location | Base exchange | MAF†(CEU/JPN) | Function of polymorphism | Forward / Reverse primer (5’−3’) |
| --- | --- | --- | --- | --- | --- |
| MS4A12rs4939378 | IntronChromosome 11:60499325 | G>A | G=0.48/0.42 | Unknown | F: GGCCAAAAATGCAGACAAATR: CCTTGATGAGGGGCAGTTAG |
| CDX2rs3812863 | 5’ prime UTRChromosome 13:27971131 | G>A | G=0.44/0.57 | Transcriptional regulation | F: GACCGAGCTGGTCCTCCTR: CTCCCCTACTGCCCTTCTCT |

Table 1 Caption: Candidate SNPs in enterocyte-related genes

### DNA extraction and genotyping

Genomic DNA was extracted from formalin-fixed paraffin-embedded tissues from patients in the discovery and control cohorts using the QIAmp Kit (Qiagen, Valencia, CA, USA) according to the manufacturer’s protocol. The SNPs were tested using PCR-based direct DNA sequence analysis by ABI 3100A Capillary Genetic Analyzer and Sequencing Scanner v1.0 (Applied Biosystems, Foster City, CA). The forward and reverse primers used for amplification of extracted DNA are listed in [Table 1](#T1). For quality control purposes, a randomly selected 10% of the samples were analyzed by direct DNA sequencing for each SNP, and the genotype concordance rate was of 99% or more. The investigators analyzing SNPs were blinded to the clinical data.

### Statistical analysis

The primary endpoint of the current study was disease-free survival (DFS) and the secondary endpoint was overall survival (OS). DFS was defined as the interval between the date of surgery and the date of confirmed relapse, secondary cancer or death from any cause. OS was calculated from the date of the date of surgery until the date of death from any cause. Data of patients without relapse or death were censored at the date of last follow-up. For patients who were lost to follow-up, data were censored at the date when the patient was last confirmed to be alive. Chi-square tests were used to examine the difference in baseline patient characteristics between the two cohorts. Association between SNPs and DFS and OS were estimated by the Kaplan–Meier method and compared using the log-rank test, with predictive or prognostic clinical factors and candidate SNPs that were identified by univariate analysis using codominant, dominant, or recessive genetic models if appropriate. Multivariate analysis using the Cox proportional hazards model was then conducted to re-evaluate factors influencing DFS and OS. The baseline demographic and clinical characteristics that remained statistically significantly associated with DFS and OS in multivariate analyses were included in the final models. All analyses were carried out with SAS 9.4 (SAS Institute, Cary, NC, USA). All tests were two-sided at a significance level of 0.05.

## Results

### Baseline patients and tumor characteristics

The baseline characteristics of the discovery cohort (n=274) and control cohort (n=76) are summarized in [Table 2](#T2). The median patient age was 62 years old and median follow-up was 59.9 months; the median DFS and OS were not reached for the discovery cohort. The median patient age was 75 years old and median follow-up was 58.0 months; the median DFS and OS were not reached for the control cohort. Patients ≤65 years of age (*P*<0.001), with left-sided tumor (*P*<0.001) and with tumor stage IIIC (*P*=0.024) were more frequent in the discovery cohort compared with the control cohort. Frequencies of the genetic variants of the SNPs satisfied the Hardy–Weinberg equilibrium (*P*>0.01) using the exact test in the Haploview software version 4.2.

### Table 2.

| Cohort | Total (N=350) | Discovery cohort (N=274) |  | Control cohort (N=76) |  | P value * |
| --- | --- | --- | --- | --- | --- | --- |
|   |   | N | % | N | % |   |
| Sex |   |   |   |   |   | 0.12 |
| Male | 175 | 143 | 52 | 32 | 42 |   |
| Female | 175 | 131 | 48 | 44 | 58 |   |
| Age (year) |   |   |   |   |   | <0.001 |
| Median (range) |   | 62 (28–85) |  | 75 (36–86) |  |   |
| ≤ 65 | 183 | 167 | 61 | 16 | 21 |   |
| > 65 | 167 | 107 | 39 | 60 | 79 |   |
| Primary tumor site |   |   |   |   |   | <0.001 |
| Right | 110 | 77 | 28 | 33 | 43 |   |
| Left | 238 | 197 | 72 | 41 | 54 |   |
| Others** | 2 | 0 | 0 | 2 | 3 |   |
| N of resected LN |   |   |   |   |   | 0.78 |
| <12 | 12 | 9 | 3 | 3 | 4 |   |
| ≥12 | 337 | 264 | 96 | 73 | 96 |   |
| pT stage |   |   |   |   |   |   |
| T1 | 0 | 0 | 0 | 0 |   | 0.59 |
| T2 | 5 | 3 | 1 | 2 | 3 |   |
| T3 | 232 | 183 | 67 | 49 | 64 |   |
| T4 | 113 | 88 | 32 | 25 | 33 |   |
| pN stage |   |   |   |   |   | 0.068 |
| N1 | 223 | 166 | 61 | 57 | 75 |   |
| N2 | 105 | 89 | 32 | 16 | 21 |   |
| N3 | 22 | 19 | 7 | 3 | 4 |   |
| UICC Stage |   |   |   |   |   | 0.024 |
| IIIB | 229 | 171 | 62 | 58 | 76 |   |
| IIIC | 121 | 103 | 38 | 18 | 24 |   |
| Histology |   |   |   |   |   | 0.083 |
| Well | 75 | 56 | 20 | 19 | 25 |   |
| Moderate | 236 | 192 | 70 | 44 | 58 |   |
| Poor/others | 39 | 26 | 9 | 13 | 17 |   |
| Adjuvant treatment |   |   |   |   |   | NA |
| FOLFOX | 68 | 68 | 25 | - | - |   |
| UFT/LV | 126 | 126 | 46 | - | - |   |
| Capecitabine | 80 | 80 | 29 | - | - |   |

Table 2 Caption: Baseline patients and tumor characteristics

### Association of clinical outcome and SNPs in the discovery cohort

In the discovery cohort, the *MS4A12* rs4939378 G/G variant was associated with a worse 5-year survival rate compared with A alleles (70% vs. 90%, univariate: HR 2.29, 95%CI: 1.03–5.06, *P*=0.035; multivariate: HR 2.58, 95%CI: 1.15–5.76, *P*=0.021) ([Fig. 1A](#F1)). Patients with the G/G variant in *CDX2* rs3812863 had better OS than those with any A allele, although the difference was not significant in multivariate analysis (5 year survival rate: 95% vs. 82%, univariate: HR 0.34, 95%CI: 0.12–0.97, *P*=0.034; multivariate: HR 0.39, 95%CI: 0.13–1.11, *P*=0.078) ([Fig. 1B](#F1), [Table 3](#T3)). When analyzing the subgroups of patients receiving FOLFOX (n=68) or oral fluoropyrimidine (n=206), no significance difference in DFS and OS was found in each subgroup, although the trend in *MS4A12* rs4939378 was more evident in the oral fluoropyrimidine subgroup (G/G, 73% vs. any A, 91%, univariate: HR 2.39, 95%CI: 0.95–5.99, *P*=0.056; multivariate: HR2.51, 95%CI: 0.98–6.46, *P*=0.021) ([Table S1](#SD2)).

### Fig. 1.

![Fig. 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d994/7655616/ce8997836086/nihms-1605709-f0001.jpg)

Overall survival (OS) in the discovery cohort according to (A) MS4A12 rs4939378 variants and (B) CDX2 rs3812863 variants.

### Table 3.

|   |   | Disease-free Survival |  |  |  |  |  | Overall survival |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   | Median, months (95%CI) | 3-year of recurrence ± SE | HR (95%CI) † | P value* | HR (95%CI) ‡ | P value* | Median, months (95%CI) | 5-year of survival ± SE | HR (95%CI) † | P value* | HR (95%CI) ‡ | P value* |
| Discovery cohort |  |  |  |  |  |  |  |  |  |  |  |  |  |
| CDX2 rs3812863 |   |   |   |   | 0.85 |   | 0.95 |   |   |   | 0.10 |   | 0.20 |
| A/A | 61 | 7.0+ | 0.33±0.06 | 1 (Reference) |   | 1 (Reference) |   | 8.6+ | 0.83±0.07 | 1 (Reference) |   | 1 (Reference) |   |
| A/G | 131 | 8.0+ | 0.29±0.04 | 0.89(0.52,1.53) |   | 0.97(0.56,1.66) |   | 8.0+ | 0.82±0.06 | 0.97(0.44,2.16) |   | 1.12(0.50,2.51) |   |
| G/G | 81 | 7.6+(6.3,7.6+) | 0.25±0.05 | 0.84(0.46,1.53) |   | 0.91(0.50,1.66) |   | 8.0+ | 0.95±0.04 | 0.33(0.10,1.08) |   | 0.42(0.13,1.36) |   |
|   |   |   |   |   | 0.60 |   | 0.83 |   |   |   | 0.40 |   | 0.702 |
| A/A | 61 | 7.0+ | 0.33±0.06 | 1 (Reference) |   | 1 (Reference) |   | 8.6+ | 0.83±0.07 | 1 (Reference) |   | 1 (Reference) |   |
| Any G | 212 | 8.0+ | 0.27±0.03 | 0.87(0.53,1.44) |   | 0.95(0.57,1.57) |   | 8.0+ | 0.87±0.04 | 0.72(0.33,1.56) |   | 0.86(0.39,1.87) |   |
|   |   |   |   |   | 0.70 |   | 0.77 |   |   |   | 0.034 |   | 0.078 |
| Any A | 192 | 8.0+ | 0.30±0.03 | 1 (Reference) |   | 1 (Reference) |   | 8.6+ | 0.82±0.04 | 1 (Reference) |   | 1 (Reference) |   |
| G/G | 81 | 7.6+(6.3,7.6+) | 0.25±0.05 | 0.91(0.56,1.47) |   | 0.93(0.57,1.51) |   | 8.0+ | 0.95±0.04 | 0.34(0.12,0.97) |   | 0.39(0.13,1.11) |   |
| MS4A12 rs4939378 |   |   |   |   | 0.30 |   | 0.22 |   |   |   | 0.081 |   | 0.035 |
| A/A | 92 | 7.4+ | 0.29±0.05 | 1 (Reference) |   | 1 (Reference) |   | 7.4+ | 0.88±0.05 | 1 (Reference) |   | 1 (Reference) |   |
| A/G | 125 | 8.0+(6.3,8.0+) | 0.23±0.04 | 0.75(0.45,1.24) |   | 0.72(0.43,1.20) |   | 8.0+ | 0.92±0.03 | 0.68(0.28,1.67) |   | 0.53(0.21,1.31) |   |
| G/G | 49 | 8.0+(3.2,8.0+) | 0.34±0.07 | 1.15(0.63,2.10) |   | 1.17(0.64,2.14) |   | 8.6+ (5.7,8.6+) | 0.70±0.13 | 1.87(0.76,4.59) |   | 1.83(0.74,4.52) |   |
|   |   |   |   |   | 0.50 |   | 0.45 |   |   |   | 0.99 |   | 0.65 |
| A/A | 92 | 7.4+ | 0.29±0.05 | 1 (Reference) |   | 1 (Reference) |   | 7.4+ | 0.88±0.05 | 1 (Reference) |   | 1 (Reference) |   |
| Any G | 174 | 8.0+ | 0.26±0.03 | 0.85(0.54,1.36) |   | 0.84(0.53,1.33) |   | 8.6+ | 0.86±0.05 | 0.99(0.46,2.15) |   | 0.83(0.38,1.82) |   |
|   |   |   |   |   | 0.27 |   | 0.22 |   |   |   | 0.035 |   | 0.021 |
| Any A | 217 | 8.0+(6.3,8.0+) | 0.26±0.03 | 1 (Reference) |   | 1 (Reference) |   | 8.0+ | 0.90±0.03 | 1 (Reference) |   | 1 (Reference) |   |
| G/G | 49 | 8.0+(3.2,8.0+) | 0.34±0.07 | 1.36(0.79,2.32) |   | 1.40(0.82,2.41) |   | 8.6+ (5.7,8.6+) | 0.70±0.13 | 2.29(1.03,5.06) |   | 2.58(1.15,5.76) |   |
| Control cohort |  |  |  |  |  |  |  |  |  |  |  |  |  |
|   |   | Disease-free Survival |   | Overall survival |   |   |   |   |   |   |   |   |   |
|   |   | Median, months (95%CI) | 3-year of recurrence ± SE | HR (95%CI) † | P value* | HR (95%CI) ‡ | P value* | Median, months (95%CI) | 5-year of survival ± SE | HR (95%CI) † | P value* | HR (95%CI) ‡ | P value* |
| CDX2 rs3812863 |   |   |   |   | 0.61 |   | 0.93 |   |   |   | 0.58 |   | 0.81 |
| A/A | 19 | 6.2+(2.1,6.2+) | 0.40±0.12 | 1 (Reference) |   | 1 (Reference) |   | 6.2+(3.9,6.2+) | 0.78±0.17 | 1 (Reference) |   | 1 (Reference) |   |
| A/G | 36 | 8.2+(2.7,8.2+) | 0.33±0.08 | 0.76(0.31,1.84) |   | 0.85(0.32,2.22) |   | 8.2+ | 0.73±0.10 | 1.40(0.38,5.19) |   | 1.49(0.38,5.78) |   |
| G/G | 21 | 6.9+(0.9,6.9+) | 0.43±0.11 | 1.13(0.45,2.88) |   | 0.98(0.38,2.50) |   | 6.9+(3.8,6.9+) | 0.65±0.13 | 1.97(0.51,7.65) |   | 1.53(0.39,6.06) |   |
|   |   |   |   |   | 0.77 |   | 0.83 |   |   |   | 0.45 |   | 0.52 |
| A/A | 19 | 6.2+(2.1,6.2+) | 0.40±0.12 | 1 (Reference) |   | 1 (Reference) |   | 6.2+(3.9,6.2+) | 0.78±0.17 | 1 (Reference) |   | 1 (Reference) |   |
| Any G | 57 | 8.2+(3.0,8.2+) | 0.37±0.06 | 0.89(0.39,1.99) |   | 0.91(0.40,2.10) |   | 8.2+ | 0.70±0.07 | 1.61(0.47,5.52) |   | 1.51(0.43,5.25) |   |
|   |   |   |   |   | 0.43 |   | 0.87 |   |   |   | 0.35 |   | 0.74 |
| Any A | 55 | 8.2+(3.7,8.2+) | 0.35±0.07 | 1 (Reference) |   | 1 (Reference) |   | 8.2+ | 0.75±0.08 | 1 (Reference) |   | 1 (Reference) |   |
| G/G | 21 | 6.9+(0.9,6.9+) | 0.43±0.11 | 1.36(0.64,2.88) |   | 1.07(0.48,2.38) |   | 6.9+(3.8,6.9+) | 0.65±0.13 | 1.55(0.61,3.94) |   | 1.19(0.43,3.26) |   |
| MS4A12 rs4939378 |   |   |   |   | 1.00 |   | 0.80 |   |   |   | 0.46 |   | 0.15 |
| A/A | 25 | 7.5+(1.5,7.5+) | 0.36±0.10 | 1 (Reference) |   | 1 (Reference) |   | 7.9+(3.8,7.9+) | 0.67±0.11 | 1 (Reference) |   | 1 (Reference) |   |
| A/G | 40 | 7.4+(2.4,7.4+) | 0.38±0.08 | 1.03(0.47,2.28) |   | 0.96(0.43,2.13) |   | 7.4+ | 0.71±0.09 | 0.80(0.32,2.03) |   | 0.61(0.23,1.58) |   |
| G/G | 10 | 8.2+(0.3,8.2+) | 0.30±0.14 | 1.02(0.32,3.26) |   | 0.68(0.20,2.27) |   | 8.2+(4.2,8.2+) | 0.83±0.29 | 0.29(0.04,2.30) |   | 0.12(0.01,1.08) |   |
|   |   |   |   |   | 0.93 |   | 0.78 |   |   |   | 0.42 |   | 0.14 |
| A/A | 25 | 7.5+(1.5,7.5+) | 0.36±0.10 | 1 (Reference) |   | 1 (Reference) |   | 7.9+(3.8,7.9+) | 0.67±0.11 | 1 (Reference) |   | 1 (Reference) |   |
| Any G | 50 | 8.2+(2.7,8.2+) | 0.37±0.07 | 1.03(0.48,2.21) |   | 0.90(0.41,1.94) |   | 8.2+ | 0.74±0.09 | 0.69(0.28,1.71) |   | 0.49(0.19,1.27) |   |
|   |   |   |   |   | 1.00 |   | 0.51 |   |   |   | 0.25 |   | 0.099 |
| Any A | 65 | 7.5+(3.0,7.5+) | 0.38±0.06 | 1 (Reference) |   | 1 (Reference) |   | 7.9+ | 0.70±0.07 | 1 (Reference) |   | 1 (Reference) |   |
| G/G | 10 | 8.2+(0.3,8.2+) | 0.30±0.14 | 1.00(0.35,2.87) |   | 0.69(0.23,2.08) |   | 8.2+(4.2,8.2+) | 0.83±0.29 | 0.33(0.04,2.45) |   | 0.18(0.02,1.39) |   |

Table 3 Caption: SNPs and clinical outcome

### Association of clinical outcome and SNPs in the control cohort

There was no statistical significance between SNPs and clinical outcome in the control cohort ([Fig. 2A](#F2), [B](#F2); [Table 3](#T3)). However, a significant interaction for OS was observed between *MS4A12* genotypes and cohorts (interaction *P*=0.007). In contrast, there was no significant interaction between *CDX2* genotypes and cohorts (interaction *P*=0.19). There was no significant difference in DFS in both cohorts ([Fig. S1](#SD1); [Table 3](#T3)).

### Fig. 2.

![Fig. 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d994/7655616/a136d5bff213/nihms-1605709-f0002.jpg)

Overall survival (OS) in the control cohort according to (A) MS4A12 rs4939378 variants, and (B) CDX2 rs3812863 variants.

## Discussion

Our previous report suggested the possibility of *MS4A12* and *CDX2* gene polymorphisms as potential predictive markers of the response of mCRC patients to oxaliplatin-based treatment [[10](#R10)]. In patients treated with FOLFOX ± bevacizumab (BEV), the G/G variant in *CDX2* rs3812863 was associated with a higher response rate compared with any A allele and the *MS4A12* rs4939378 G/G variant was associated with a trend toward longer OS than any A allele. In patients treated with FOLFOXIRI + BEV, a similar trend was observed: any G allele of *MS4A12* rs4939378 was associated with longer PFS than the A/A variant, especially in *KRAS* mutant tumors, and the *CDX2* rs3812863 G/G variant was associated with a longer PFS than any A allele, especially in *KRAS* wild-type tumors. Together these results suggest that *CDX2* rs3812863 is likely to play a similar role regarding prognosis among stage III CRC and mCRC, whereas clinical significance of *MS4A12* rs4939378 seems to vary depending on tumor stage. In addition, association of sensitivity to oxaliplatin and *MS4A12* rs4939378 may potentially depend on tumor mutation burden or mutation status of genes such as *KRAS*. In this study, we did not find any significance between the outcomes of subgroups receiving either oral fluoropyrimidine or FOLFOX; the small number of patients with stage IIIC receiving FOLFOX may account for the lack of significance. High frequency of stage III in the FOLFOX subgroup compared with the oral fluoriopyrimidine subgroup might be another reason. A further large-scale study is warranted to explore whether *MS4A12* rs4939378 affects response to oxaliplatin-based adjuvant therapy depending on the difference in molecular profiles between early and metastatic CRC.

Two phase III studies, MOSAIC and NSABP C-07, demonstrated significant improvement of 3-year DFS by adding oxaliplatin to 5-FU/LV (FOLFOX regimen) or weekly bolus 5-FU/LV (FLOX regimen) for patients with early stage colon cancer [[12](#R12), [13](#R13)]. OS was also improved by oxaliplatin-based treatment after long follow-up [[14](#R14)]. Among the CRCA classifier [[2](#R2)], the enterocyte subtype is associated with intermediate prognosis in untreated patients, while no relationship has been demonstrated between subtypes and DFS in treated patients. According to the NSABP C-07 trial, the enterocyte subtype has been shown to benefit from adjuvant oxaliplatin plus 5-fluorouracil/leucovorin (FL) compared with FL alone for stage III colon cancer [[9](#R9)]. In our study, we therefore focused on the enterocyte-related gene *MS4A12* and found some clinical benefit in patients receiving adjuvant chemotherapy who had specific genotypes of *MS4A12*; however, we did not observe significant benefit from oxaliplatin. We also examined CDX2, which regulates MS4A12 expression [[7](#R7), [15](#R15)], and an association of *CDX2* genotypes and survival was found. A significant interaction between genotypes and cohorts (discovery vs. control) was observed for *MS4A12* but not for *CDX2*; however, there was no significant difference between genotypes and treatment regimen (FOLFOX vs. oral fluoropyrimidine). Since only 68 patients received FOLFOX while 206 received oral fluoriopyrimidine, a further large-scale study is needed to confirm this observation. To our knowledge, little evidence is available for the association of the SNPs tested in our study and clinical outcome in CRC patients, except one report in benign disease [[16](#R16),[17](#R17)]. Furthermore, few studies have examined the crosstalk of CDX2 and *MS4A12* gene expression, and the functions of the SNPs remain unclear. Further investigation on the associations between CDX2, MS4A12 and other prognostic factors such as MSI or CIMP should be investigated [[8](#R8), [18](#R18)–[20](#R20)]. Meanwhile, prognostic impact of tumor location in stage III colon cancer has previously been reported [[21](#R21)]; however, there was no significant difference between genotypes and tumor location in our study (data not shown).

This study has several limitations including its retrospective study design with a limited sample size; absence of randomized treatment selection, oral fluoropyrimidine or FOLFOX; absence of validation of data; and lack of molecular information of tumors.

In conclusion, our findings suggest that *MS4A12* and *CDX2* gene polymorphisms may predict outcome in patients with stage III CRC receiving adjuvant therapy. However, the clinical significance of the SNPs for the response to oxaliplatin seems to differ depending on the tumor stage. Further research and validation study are warranted to explore the association of the SNPs with carcinogenesis or cancer progression.

## Supplementary Material

## Acknowledgments

Mitsukuni Suenaga is the recipient of Takashi Tsuruo Memorial Fund. Martin D. Berger received a grant from the Swiss Cancer League (BIL KLS-3334-02-2014) and the Werner and Hedy Berger-Janser Foundation for cancer research. Yuji Miyamoto received a grant from Japan Society for the Promotion of Science (S2606).

Funding Support

This work was partly supported by the National Cancer Institute (grant number P30CA014089), The Gloria Borges WunderGlo Foundation-The Wunder Project, the Dhont Family Foundation, the San Pedro Peninsula Cancer Guild, the Daniel Butler Research Fund and the Call to Cure Fund.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Bray F, Ferlay J, Soerjomataram I, Siegel RL, Torre LA, Jemal A. Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. CA Cancer J Clin 2018; 68:394–424.  [DOI](https://doi.org/10.3322/caac.21492) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30207593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&title=Global%20cancer%20statistics%202018:%20GLOBOCAN%20estimates%20of%20incidence%20and%20mortality%20worldwide%20for%2036%20cancers%20in%20185%20countries&author=F%20Bray&author=J%20Ferlay&author=I%20Soerjomataram&author=RL%20Siegel&author=LA%20Torre&volume=68&publication_year=2018&pages=394-424&pmid=30207593&doi=10.3322/caac.21492&)

2. Sadanandam A, Lyssiotis CA, Homicsko K, Collisson EA, Gibb WJ, Wullschleger S, Ostos LC, Lannon WA, Grotzinger C, Del Rio M, Lhermitte B, Olshen AB, Wiedenmann B, Cantley LC, Gray JW, Hanahan D. A colorectal cancer classification system that associates cellular phenotype and responses to therapy. Nat Med 2013; 19:619–25.  [DOI](https://doi.org/10.1038/nm.3175) | [PMC free article](/articles/PMC3774607/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23584089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=A%20colorectal%20cancer%20classification%20system%20that%20associates%20cellular%20phenotype%20and%20responses%20to%20therapy&author=A%20Sadanandam&author=CA%20Lyssiotis&author=K%20Homicsko&author=EA%20Collisson&author=WJ%20Gibb&volume=19&publication_year=2013&pages=619-25&pmid=23584089&doi=10.1038/nm.3175&)

3. De Sousa E Melo F, Wang X, Jansen M, Fessler E, Trinh A, de Rooij LP, de Jong JH, de Boer OJ, van Leersum R, Bijlsma MF, Rodermond H, van der Heijden M, van Noesel CJ, Tuynman JB, Dekker E, Markowetz F, Medema JP, Vermeulen L. Poor-prognosis colon cancer is defined by a molecularly distinct subtype and develops from serrated precursor lesions. Nat Med 2013; 19:614–8.  [DOI](https://doi.org/10.1038/nm.3174) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23584090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=Poor-prognosis%20colon%20cancer%20is%20defined%20by%20a%20molecularly%20distinct%20subtype%20and%20develops%20from%20serrated%20precursor%20lesions&author=F%20De%20Sousa%20E%20Melo&author=X%20Wang&author=M%20Jansen&author=E%20Fessler&author=A%20Trinh&volume=19&publication_year=2013&pages=614-8&pmid=23584090&doi=10.1038/nm.3174&)

4. Snoeck V, Goddeeris B, Cox E. The role of enterocytes in the intestinal barrier function and antigen uptake. Microbes Infect 2005; 7:997–1004.  [DOI](https://doi.org/10.1016/j.micinf.2005.04.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15925533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbes%20Infect&title=The%20role%20of%20enterocytes%20in%20the%20intestinal%20barrier%20function%20and%20antigen%20uptake&author=V%20Snoeck&author=B%20Goddeeris&author=E%20Cox&volume=7&publication_year=2005&pages=997-1004&pmid=15925533&doi=10.1016/j.micinf.2005.04.003&)

5. Koslowski M, Sahin U, Dhaene K, Huber C, Türeci O. MS4A12 is a colon-selective store-operated calcium channel promoting malignant cell processes. Cancer Res 2008; 68:3458–66.  [DOI](https://doi.org/10.1158/0008-5472.CAN-07-5768) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18451174/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=MS4A12%20is%20a%20colon-selective%20store-operated%20calcium%20channel%20promoting%20malignant%20cell%20processes&author=M%20Koslowski&author=U%20Sahin&author=K%20Dhaene&author=C%20Huber&author=O%20T%C3%BCreci&volume=68&publication_year=2008&pages=3458-66&pmid=18451174&doi=10.1158/0008-5472.CAN-07-5768&)

6. He L, Deng HY, C Wang X. Decreased expression of MS4A12 inhibits differentiation and predicts early stage survival in colon cancer. Neoplasma 2017; 64:65–73.  [DOI](https://doi.org/10.4149/neo_2017_108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27881006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neoplasma&title=Decreased%20expression%20of%20MS4A12%20inhibits%20differentiation%20and%20predicts%20early%20stage%20survival%20in%20colon%20cancer&author=L%20He&author=HY%20Deng&author=X%20C%20Wang&volume=64&publication_year=2017&pages=65-73&pmid=27881006&doi=10.4149/neo_2017_108&)

7. Koslowski M, Türeci O, Huber C, Sahin U. Selective activation of tumor growth-promoting Ca2+ channel MS4A12 in colon cancer by caudal type homeobox transcription factor CDX2. Mol Cancer 2009; 8:77.  [DOI](https://doi.org/10.1186/1476-4598-8-77) | [PMC free article](/articles/PMC2759907/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19781065/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer&title=Selective%20activation%20of%20tumor%20growth-promoting%20Ca2+%20channel%20MS4A12%20in%20colon%20cancer%20by%20caudal%20type%20homeobox%20transcription%20factor%20CDX2&author=M%20Koslowski&author=O%20T%C3%BCreci&author=C%20Huber&author=U%20Sahin&volume=8&publication_year=2009&pages=77&pmid=19781065&doi=10.1186/1476-4598-8-77&)

8. Dalerba P, Sahoo D, Paik S, Guo X, Yothers G, Song N, Wilcox-Fogel N, Forgó E, Rajendran PS, Miranda SP, Hisamori S, Hutchison J, Kalisky T, Qian D, Wolmark N, Fisher GA, van de Rijn M, Clarke MF. CDX2 as a Prognostic Biomarker in Stage II and Stage III Colon Cancer. N Engl J Med 2016; 374:211–22.  [DOI](https://doi.org/10.1056/NEJMoa1506597) | [PMC free article](/articles/PMC4784450/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26789870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=CDX2%20as%20a%20Prognostic%20Biomarker%20in%20Stage%20II%20and%20Stage%20III%20Colon%20Cancer&author=P%20Dalerba&author=D%20Sahoo&author=S%20Paik&author=X%20Guo&author=G%20Yothers&volume=374&publication_year=2016&pages=211-22&pmid=26789870&doi=10.1056/NEJMoa1506597&)

9. Song N, Pogue-Geile KL, Gavin PG, Yothers G, Kim SR, Johnson NL, Lipchik C, Allegra CJ, Petrelli NJ, O’Connell MJ, Wolmark N, Paik S. Clinical Outcome From Oxaliplatin Treatment in Stage II/III Colon Cancer According to Intrinsic Subtypes: Secondary Analysis of NSABP C-07/NRG Oncology Randomized Clinical Trial. JAMA Oncol 2016; 2:1162–9.  [DOI](https://doi.org/10.1001/jamaoncol.2016.2314) | [PMC free article](/articles/PMC5065181/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27270348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Oncol&title=Clinical%20Outcome%20From%20Oxaliplatin%20Treatment%20in%20Stage%20II/III%20Colon%20Cancer%20According%20to%20Intrinsic%20Subtypes:%20Secondary%20Analysis%20of%20NSABP%20C-07/NRG%20Oncology%20Randomized%20Clinical%20Trial&author=N%20Song&author=KL%20Pogue-Geile&author=PG%20Gavin&author=G%20Yothers&author=SR%20Kim&volume=2&publication_year=2016&pages=1162-9&pmid=27270348&doi=10.1001/jamaoncol.2016.2314&)

10. Suenaga M, Schirripa M, Loupakis F, Cao S, Zhang W, Cremolini C, Lonardi S, Okazaki S, Berger MD, Miyamoto Y, Soni S, Barzi A, Yamaguchi T, Falcone A, Lenz HJ. J Clin Oncol 36, 2018(suppl; abstr 12066).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&author=M%20Suenaga&author=M%20Schirripa&author=F%20Loupakis&author=S%20Cao&author=W%20Zhang&volume=36&pages=2018&)

11. Lee PH, Shatkay H. F-SNP: computationally predicted functional SNPs for disease association studies. Nucleic Acids Res 2008; 36(Database issue):D820–4.  [DOI](https://doi.org/10.1093/nar/gkm904) | [PMC free article](/articles/PMC2238878/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17986460/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=F-SNP:%20computationally%20predicted%20functional%20SNPs%20for%20disease%20association%20studies&author=PH%20Lee&author=H%20Shatkay&volume=36&publication_year=2008&pages=D820-4&pmid=17986460&doi=10.1093/nar/gkm904&)

12. André T, Boni C, Mounedji-Boudiaf L, Navarro M, Tabernero J, Hickish T, Topham C, Zaninelli M, Clingan P, Bridgewater J, Tabah-Fisch I and de Gramont A: Oxaliplatin, fluorouracil, and leucovorin as adjuvant treatment for colon cancer. N Engl J Med 2004; 350:2343–2351.  [DOI](https://doi.org/10.1056/NEJMoa032709) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15175436/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Oxaliplatin,%20fluorouracil,%20and%20leucovorin%20as%20adjuvant%20treatment%20for%20colon%20cancer&author=T%20Andr%C3%A9&author=C%20Boni&author=L%20Mounedji-Boudiaf&author=M%20Navarro&author=J%20Tabernero&volume=350&publication_year=2004&pages=2343-2351&pmid=15175436&doi=10.1056/NEJMoa032709&)

13. Kuebler JP, Wieand HS, O’Connell MJ, Smith RE, Colangelo LH, Yothers G, Petrelli NJ, Findlay MP, Seay TE, Atkins JN, Zapas JL, Goodwin JW, Fehrenbacher L, Ramanathan RK, Conley BA, Flynn PJ, Soori G, Colman LK, Levine EA, Lanier KS and Wolmark N: Oxaliplatin combined with weekly bolus fluorouracil and leucovorin as surgical adjuvant chemotherapy for stage II and III colon cancer: results from NSABP C-07. J Clin Oncol 2007; 25:2198–2204.  [DOI](https://doi.org/10.1200/JCO.2006.08.2974) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17470851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Oxaliplatin%20combined%20with%20weekly%20bolus%20fluorouracil%20and%20leucovorin%20as%20surgical%20adjuvant%20chemotherapy%20for%20stage%20II%20and%20III%20colon%20cancer:%20results%20from%20NSABP%20C-07&author=JP%20Kuebler&author=HS%20Wieand&author=MJ%20O%E2%80%99Connell&author=RE%20Smith&author=LH%20Colangelo&volume=25&publication_year=2007&pages=2198-2204&pmid=17470851&doi=10.1200/JCO.2006.08.2974&)

14. André T, Boni C, Navarro M, Tabernero J, Hickish T, Topham C, Bonetti A, Clingan P, Bridgewater J, Rivera F and de Gramont A: Improved overall survival with oxaliplatin, fluorouracil, and leucovorin as adjuvant treatment in stage II or III colon cancer in the MOSAIC trial. J Clin Oncol 2009; 27:3109–3116.  [DOI](https://doi.org/10.1200/JCO.2008.20.6771) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19451431/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Improved%20overall%20survival%20with%20oxaliplatin,%20fluorouracil,%20and%20leucovorin%20as%20adjuvant%20treatment%20in%20stage%20II%20or%20III%20colon%20cancer%20in%20the%20MOSAIC%20trial&author=T%20Andr%C3%A9&author=C%20Boni&author=M%20Navarro&author=J%20Tabernero&author=T%20Hickish&volume=27&publication_year=2009&pages=3109-3116&pmid=19451431&doi=10.1200/JCO.2008.20.6771&)

15. Drew JE, Farquharson AJ, Mayer CD, Vase HF, Coates PJ, Steele RJ, Carey FA. Predictive gene signatures: molecular markers distinguishing colon adenomatous polyp and carcinoma. PLoS One 2014; 9:e113071.  [DOI](https://doi.org/10.1371/journal.pone.0113071) | [PMC free article](/articles/PMC4244109/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25423035/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Predictive%20gene%20signatures:%20molecular%20markers%20distinguishing%20colon%20adenomatous%20polyp%20and%20carcinoma&author=JE%20Drew&author=AJ%20Farquharson&author=CD%20Mayer&author=HF%20Vase&author=PJ%20Coates&volume=9&publication_year=2014&pages=e113071&pmid=25423035&doi=10.1371/journal.pone.0113071&)

16. Ren D, Zheng G, Bream S, Tevebaugh W, Shaheen NJ, Chen X. Single nucleotide polymorphisms of caudal type homeobox 1 and 2 are associated with Barrett’s esophagus. Dig Dis Sci 2014; 59:57–63.  [DOI](https://doi.org/10.1007/s10620-013-2804-9) | [PMC free article](/articles/PMC3947210/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23918153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dig%20Dis%20Sci&title=Single%20nucleotide%20polymorphisms%20of%20caudal%20type%20homeobox%201%20and%202%20are%20associated%20with%20Barrett%E2%80%99s%20esophagus&author=D%20Ren&author=G%20Zheng&author=S%20Bream&author=W%20Tevebaugh&author=NJ%20Shaheen&volume=59&publication_year=2014&pages=57-63&pmid=23918153&doi=10.1007/s10620-013-2804-9&)

17. Findlay JM, Middleton MR, Tomlinson I. Genetic Biomarkers of Barrett’s Esophagus Susceptibility and Progression to Dysplasia and Cancer: A Systematic Review and Meta-Analysis. Dig Dis Sci 2016; 61:25–38.  [DOI](https://doi.org/10.1007/s10620-015-3884-5) | [PMC free article](/articles/PMC4700058/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26445852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dig%20Dis%20Sci&title=Genetic%20Biomarkers%20of%20Barrett%E2%80%99s%20Esophagus%20Susceptibility%20and%20Progression%20to%20Dysplasia%20and%20Cancer:%20A%20Systematic%20Review%20and%20Meta-Analysis&author=JM%20Findlay&author=MR%20Middleton&author=I%20Tomlinson&volume=61&publication_year=2016&pages=25-38&pmid=26445852&doi=10.1007/s10620-015-3884-5&)

18. Bae JM, Lee TH, Cho NY, Kim TY, Kang GH. Loss of CDX2 expression is associated with poor prognosis in colorectal cancer patients. World J Gastroenterol 2015; 21:1457–67.  [DOI](https://doi.org/10.3748/wjg.v21.i5.1457) | [PMC free article](/articles/PMC4316088/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25663765/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J%20Gastroenterol&title=Loss%20of%20CDX2%20expression%20is%20associated%20with%20poor%20prognosis%20in%20colorectal%20cancer%20patients&author=JM%20Bae&author=TH%20Lee&author=NY%20Cho&author=TY%20Kim&author=GH%20Kang&volume=21&publication_year=2015&pages=1457-67&pmid=25663765&doi=10.3748/wjg.v21.i5.1457&)

19. Van Rijnsoever M, Elsaleh H, Joseph D, McCaul K, Iacopetta B. CpG island methylator phenotype is an independent predictor of survival benefit from 5-fluorouracil in stage III colorectal cancer. Clin Cancer Res 2003; 9:2898–903  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12912934/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=CpG%20island%20methylator%20phenotype%20is%20an%20independent%20predictor%20of%20survival%20benefit%20from%205-fluorouracil%20in%20stage%20III%20colorectal%20cancer&author=M%20Van%20Rijnsoever&author=H%20Elsaleh&author=D%20Joseph&author=K%20McCaul&author=B%20Iacopetta&volume=9&publication_year=2003&pages=2898-903&pmid=12912934&)

20. Shiovitz S, Bertagnolli MM, Renfro LA, Nam E, Foster NR, Dzieciatkowski S, Luo Y, Lao VV, Monnat RJ Jr, Emond MJ, Maizels N, Niedzwiecki D, Goldberg RM, Saltz LB, Venook A, Warren RS, Grady WM; Alliance for Clinical Trials in Oncology. CpG island methylator phenotype is associated with response to adjuvant irinotecan-based therapy for stage III colon cancer. Gastroenterology 2014; 147:637–45.  [DOI](https://doi.org/10.1053/j.gastro.2014.05.009) | [PMC free article](/articles/PMC4143495/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24859205/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Alliance%20for%20Clinical%20Trials%20in%20Oncology.%20CpG%20island%20methylator%20phenotype%20is%20associated%20with%20response%20to%20adjuvant%20irinotecan-based%20therapy%20for%20stage%20III%20colon%20cancer&author=S%20Shiovitz&author=MM%20Bertagnolli&author=LA%20Renfro&author=E%20Nam&author=NR%20Foster&volume=147&publication_year=2014&pages=637-45&pmid=24859205&doi=10.1053/j.gastro.2014.05.009&)

21. Weiss JM, Pfau PR, O’Connor ES, King J, LoConte N, Kennedy G and Smith MA: Mortality by stage for right- versus left-sided colon cancer: analysis of surveillance, epidemiology, and end results – Medicare data. J Clin Oncol 2011; 29:4401–9.  [DOI](https://doi.org/10.1200/JCO.2011.36.4414) | [PMC free article](/articles/PMC3221523/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21969498/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Mortality%20by%20stage%20for%20right-%20versus%20left-sided%20colon%20cancer:%20analysis%20of%20surveillance,%20epidemiology,%20and%20end%20results%20%E2%80%93%20Medicare%20data&author=JM%20Weiss&author=PR%20Pfau&author=ES%20O%E2%80%99Connor&author=J%20King&author=N%20LoConte&volume=29&publication_year=2011&pages=4401-9&pmid=21969498&doi=10.1200/JCO.2011.36.4414&)
