# Pharmacogenomics of aromatase inhibitors in postmenopausal breast cancer and additional mechanisms of anastrozole action

## Metadata
**Authors:** Junmei Cairns, James N Ingle, Tanda M Dudenkov, Krishna R Kalari, Erin E Carlson, Jie Na, Aman U Buzdar, Mark E Robson, Matthew J Ellis, Paul E Goss, Lois E Shepherd, Barbara Goodnature, Matthew P Goetz, Richard M Weinshilboum, Hu Li, Mehrab Ghanat Bari, Liewei Wang
**Journal:** JCI Insight
**Date:** 2020 Aug 20
**DOI:** [10.1172/jci.insight.137571](https://doi.org/10.1172/jci.insight.137571)
**PMID:** 32701512
**PMCID:** PMC7455128
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7455128/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7455128/pdf/jciinsight-5-137571.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7455128/pdf/jciinsight-5-137571.pdf)

## Abstract

Aromatase inhibitors (AIs) reduce breast cancer recurrence and prolong survival, but up to 30% of patients exhibit recurrence. Using a genome-wide association study of patients entered on MA.27, a phase III randomized trial of anastrozole versus exemestane, we identified a single nucleotide polymorphism (SNP) in CUB And Sushi multiple domains 1 (CSMD1) associated with breast cancer–free interval, with the variant allele associated with fewer distant recurrences. Mechanistically, CSMD1 regulates CYP19 expression in an SNP- and drug-dependent fashion, and this regulation is different among 3 AIs: anastrozole, exemestane, and letrozole. Overexpression of CSMD1 sensitized AI-resistant cells to anastrozole but not to the other 2 AIs. The SNP in CSMD1 that was associated with increased CSMD1 and CYP19 expression levels increased anastrozole sensitivity, but not letrozole or exemestane sensitivity. Anastrozole degrades estrogen receptor α (ERα), especially in the presence of estradiol (E2). ER+ breast cancer organoids and AI- or fulvestrant-resistant breast cancer cells were more sensitive to anastrozole plus E2 than to AI alone. Our findings suggest that the CSMD1 SNP might help to predict AI response, and anastrozole plus E2 serves as a potential new therapeutic strategy for patients with AI- or fulvestrant-resistant breast cancers.

Keywords: Oncology, Therapeutics

Keywords: Breast cancer, Pharmacogenetics

## Introduction

About 70% of primary breast cancers express estrogen receptor α (ERα). Adjuvant endocrine therapy, including aromatase inhibitors (AIs), is a standard treatment for these patients, regardless of tumor size or nodal status. For postmenopausal women with primary ER^+^ breast cancer, AIs are the standard-of-care to prevent relapse and prolong survival ([1](#B1), [2](#B2)). In advanced disease, AI-based therapy is a standard initial treatment. Despite their efficacy, about 19% of patients with early-stage disease suffer a recurrence by 10 years ([1](#B1)), and resistance to AIs in advanced or metastatic tumors invariably occurs ([3](#B3)). Therefore, there is a great need to understand the underlying mechanisms associated with AI response and resistance.

There are well-accepted biomarkers associated with both de novo and acquired resistance to endocrine therapy ([4](#B4), [5](#B5)), and both host (germline) and somatic (tumor) alterations are known to contribute to resistance ([6](#B6), [7](#B7)). For example, the tumor genome has revealed several mechanisms of acquired AI resistance ([8](#B8)–[10](#B10)) such as amplifications or mutations that can activate ESR1 ([10](#B10)) and ligand-independent ERα activation of downstream pathways such as the phosphatidylinositol 3 kinase ([11](#B11)). At the same time, germline single nucleotide polymorphisms (SNPs) in *CYP19A1* have been associated with variation in circulating estrogen concentrations and with breast cancer risk ([12](#B12)).

In postmenopausal women, estrogen is mainly synthesized in peripheral tissues through the action of aromatase (CYP19A1). The main substrates, testosterone and androstenedione, are transformed by aromatase into 17β-estradiol (E2) and estrone (E1), respectively. AIs potently inhibit aromatase and decrease estrogen levels. Two classes of AIs are currently in clinical use: steroidal — e.g., exemestane, which binds aromatase irreversibly — and nonsteroidal — e.g., anastrozole and letrozole, which block the enzyme reversibly ([13](#B13)). While high BMI has been associated with AI resistance relative to tamoxifen ([14](#B14), [15](#B15)) — suggesting that variability in estrogen suppression might be associated with clinical outcome — and there are no biomarkers unique to an individual AI, large phase III adjuvant clinical trials do not indicate any difference in efficacy between the 3 AIs ([16](#B16), [17](#B17)).

In the present work, following our previous observation that suppression of estrogens is associated with anastrozole treatment outcomes ([18](#B18)), we performed genome-wide association studies (GWAS) using changes in estrogen levels (before and after anastrozole) as the phenotype of interest using patients prospectively enrolled in the Mayo/M.D. Anderson/Memorial Sloan Kettering (designated M3) pharmacogenomics study ([19](#B19)). Here, we identified a series of SNPs that met genome-wide significance. To further determine whether these SNPs in genes associated with estrogen suppression were also associated with long-term clinical outcome, we identified all SNPs within these gene regions based on the 1000 Genome Project and then examined their associations with breast cancer outcomes using GWAS data ([7](#B7)) previously obtained from the Canadian Cancer Trials Group MA.27 trial ([16](#B16)). MA.27 was a phase III trial comparing adjuvant anastrozole and exemestane treatment of postmenopausal women with ER^+^ breast cancer. The GWAS from the M3 clinical study identified SNPs within the human CUB and Sushi multiple domains 1 (*CSMD1*) gene associated with changes in estrogen levels during anastrozole therapy. An additional SNP in *CSMD1* was also found to be associated with breast cancer–free interval (BCFI) in MA.27. The variant SNP genotype was associated with longer BCFI. Functional studies indicated that the SNP altered the CYP19A1 expression in an anastrozole-dependent fashion through transcription regulation. These findings further confirmed our previous observation ([18](#B18)) that anastrozole is different from exemestane and letrozole. Our previous study has already shown that anastrozole, but not letrozole or exemestane, acts as an ERα ligand and degrader, and in the current study, we further show that the addition of E2 could sensitize cells to anastrozole, even in anastrozole- or fulvestrant-resistant models.

## Results

### GWAS of changes in estrogen levels and BCFI GWAS.

To identify SNPs associated with AI estrogen suppression, we performed GWAS for changes in estrogen levels before and after anastrozole in 624 postmenopausal women with resected early-stage ER^+^ breast cancer accrued through the M3 study ([19](#B19)). The flow diagram ([Supplemental Figure 1A](#sd); supplemental material available online with this article; [https://doi.org/10.1172/jci.insight.137571DS1](https://doi.org/10.1172/jci.insight.137571DS1)) shows the number of patients included in the analysis. We performed 3 GWA studies for changes in E1, E2, and the sum of E1 and E2 ([Figure 1A](#F1) and [Supplemental Figure 1B](#sd)). Common SNPs reaching genome wide significance (*P* < 5 × 10^–8^) among the 3 GWAS included rs2449598 within *DLG2* on chromosome 11 (*P* = 2.23 × 10^–10^ to 1.24 × 10^–8^), rs1437153 near *CDH11* on chromosome 16 (*P* = 2.82 × 10^–9^ to 2.7 × 10^–8^), and rs6981827 in the intron of the *CSMD1* gene on chromosome 8 (*P* = 2.02 × 10^–8^ to 2.12 × 10^–8^) ([Figure 1A](#F1); [Supplemental Figure 1B](#sd); [Supplemental Table 1](#sd)). The variant alleles for these SNPs were associated with less estrogen suppression during AI treatment.

### Figure 1. Discovery of CSMD1 SNPs.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5089/7455128/fe84a19a9105/jciinsight-5-137571-g207.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7455128_jciinsight-5-137571-g207.jpg)

(A) Manhattan plot of M3 GWAS results for change in estradiol upon anastrozole treatment. (B) Genomic position of CSMD1 SNP associated with BCFI in MA.27. (C) Kaplan-Meier plots of time to breast events in MA.27 for different copies of CSMD1 variant allele rs6990851. P value represents stratified Cox-proportional hazards analysis.

Because M3 was a pharmacokinetic and pharmacodynamic study, outcome data were not available, and we examined the SNPs identified in M3 in the MA.27 trial where we had long-term follow-up ([16](#B16)). We identified all SNPs in *DLG2*, *CDH11*, and *CSMD1* gene regions (100 kb up- and downstream of the gene) using data from the 1000 Genome Project and determined their association with BCFI in MA.27 for which GWAS results were available ([7](#B7)). The most significant association with the MA.27 BCFI phenotype was the rs6990851 SNP in the *CSMD1* gene (*P* = 4.83 × 10^–6^, HR = 0.56) ([Figure 1B](#F1)). The variant allele (minor allele frequency [MAF] = 0.21) for rs6990851 was associated with longer BCFI ([Figure 1C](#F1)). No difference was observed between the 2 drugs with regard to SNP effect on BCFI. The rs6990851 SNP from MA.27 and the rs6981827 from M3 were only weakly linked in the M3 breast cancer patients (*r2* = 0.014) and the MA.27 population (*r2* = 0.02). Therefore, neither of the SNPs was significantly associated with the other phenotype. The *CSMD1* variant SNP, rs6981827 (MAF = 0.05), associated with less estrogen change in the M3 study (effect size = –0.60), was not an expression quantitative trait loci (eQTL) SNP for *CSMD1* gene expression based on genotype-tissue expression (GTEx), while the *CSMD1* SNP rs6990851, which was associated with longer BCFI, was associated with higher *CSMD1* expression level in adipose and brain tissues, and with a trend in breast tissue ([Supplemental Figure 1C](#sd)).

### Rs6990851 showed SNP- and anastrozole-dependent transcriptional regulation of CSMD1 and CYP19A1.

We first determined whether the expression of *CSMD1* might be estrogen dependent. We treated ER-expressing ZR-75-1 and T47D cells with 0.1 nM E2 and observed that *CSMD1* mRNA was significantly induced (*P* < 0.01; [Supplemental Figure 2](#sd)). Because the known mechanism of action of AIs is to block aromatase activity, we then tested a possible relationship between the expression of CSMD1 and CYP19A1. Overexpression of CSMD1 increased CYP19A1 expression levels in breast cancer cells and human adipocytes ([Figure 2A](#F2)). We then used ENCODE data to search for putative estrogen response elements (EREs) and identified 2 EREs within 500 bp of the *CSMD1* SNP, rs6990851. Rs6990851 mapped to intron 1 of *CSMD1*, and 2 EREs were detected at 291 bp upstream and 296 bp downstream, respectively, while rs6981827 did not have any EREs within 500 bp, in addition to its lack of eQTL relationship with *CSMD1*. Therefore, we focused on rs6990851 for further functional studies. Based on our prior experience of studying SNP effects on gene expression and hormonal therapy, we took advantage of cell lines selected on the basis of *CSMD1*genotype from a genomic data–rich panel of lymphoblastoid cell lines (LCLs) that has already proven to be a powerful tool for generating and testing pharmacogenomic hypotheses ([20](#B20), [21](#B21)). We first exposed LCLs selected based on the rs6990851 SNP genotype of hormones or AIs to determine the effect of the SNP on *CSMD1* and *CYP19A1* expression. This is a strategy that we commonly used to determine the functional impact of GWAS signals ([7](#B7), [20](#B20)). LCLs with homozygous WT or variant genotypes for the *CSMD1* rs6990851 SNP were treated with increasing concentrations of androstenedione, a precursor of estrogen, and *CSMD1* and *CYP19A1* mRNA levels were determined. Cells with homozygous WT SNP genotype showed increased expression of *CSMD1* and, in parallel, *CYP19A1* expression ([Figure 2, B–D](#F2)). In contrast, in LCLs with the variant allele, *CSMD1* and *CYP19A1* expression was virtually unchanged with androstenedione. Importantly, addition of anastrozole reversed the expression patterns of *CSMD1* and *CYP19A1* ([Figure 2B](#F2)). Specifically, addition of anastrozole increased *CSMD1* and *CYP19A1* expression in cells with the variant SNP genotype but decreased *CSMD1* and *CYP19A1* levels to or below baseline in cells with the WT allele ([Figure 2, B and E](#F2)). Interestingly, neither letrozole nor exemestane ([Figure 2, C and D](#F2)) significantly changed the expression patterns of *CSMD1* and *CYP19A1* compared between WT and variant LCLs.

### Figure 2. CSMD1 SNP androstenedione– and SNP anastrozole–dependent regulation of CSMD1 and CYP19A1 expression.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5089/7455128/3ac3415d6dc9/jciinsight-5-137571-g208.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7455128_jciinsight-5-137571-g208.jpg)

(A) CYP19A1 mRNA expression levels in CSMD1-overexpressed breast cancer cells and human adipocytes. (B–E) SNP-dependent gene regulation of CSMD1 and CYP19A1 expression in the presence of increasing concentrations of androstenedione (A) or 20 nM androstenedione plus increasing concentrations of AIs (Ana, anastrozole; Let, letrozole; Exe, exemestane) in lymphoblastoid cell lines (LCLs) selected based on the rs6990851 SNP genotype. CSMD1 WT, homozygous WT LCLs (n = 5); CSMD1 V, homozygous variant LCLs (n = 5). (F) Schematic figure shows the EREs surrounding the rs6990851SNP. Primers 1 and 2 were used in the ChIP assays to determine the regions around the 2 EREs. (G) ERα ChIP assay shows SNP-dependent ERα binding to the ERE region that is 296 bp upstream from the rs6990851 SNP in LCLs with different genotypes treated with the indicated drugs. Error bars (A, B–D, and G) represent ± SEM of 3 independent experiments. *P < 0.05; **P < 0.01. Two-way ANOVA.

To determine whether the SNP impact on gene expression reflects differential binding of ERα to EREs that are located in close proximity of rs6990851, we performed ChIP assays using an ERα antibody. We found that the rs6990851 SNP effect occurred through the ERE located 296 bp upstream from the SNP ([Figure 2F](#F2)) and resulted in an increased binding of ERα in WT cells treated with androstenedione. A similar result was observed in variant cells when anastrozole was added ([Figure 2G](#F2), left panel). However, no differential binding to the second ERE located 291 bp downstream from the SNP was detected ([Supplemental Figure 3](#sd)). Furthermore, we did not observe any SNP-dependent binding with letrozole or exemestane ([Figure 2G](#F2), right panel).

### Rs6990851 alters anastrozole response through CSMD1 and CYP19A1.

We next sought to determine the functional consequences of the *CSMD1* SNP on response to AIs. LCLs homozygous for the variant SNP, which resulted in high CSMD1 expression, were more sensitive to anastrozole than homozygous WT or heterozygous LCLs ([Figure 3A](#F3), left panel), whereas these different alleles had little effect on letrozole or exemestane sensitivity ([Figure 3A](#F3), middle and right panels). To assess the role of *CSMD1* in AI response in breast cancer cells, we used the MCF7/AC1 cell line because of its high expression of the AI target CYP19A1. Additionally, letrozole resistant (LetR) ([22](#B22)), a letrozole-resistant cell line, was used to determine the role of *CSMD1* in AI response in an AI-resistant setting. Finally, we also used human adipocytes because adipose tissue is the predominant source of estrogens in postmenopause women ([23](#B23)). In all 3 lines, overexpression of CSMD1 significantly increased anastrozole sensitivity compared with empty vector, whereas overexpression of CSMD1 showed little effect on letrozole and exemestane sensitivity ([Figure 3B](#F3)). These findings were also observed in 2 other ER^+^ breast cancer cell lines (T47D and ZR-75-1) ([Supplemental Figure 4](#sd)). To further define the relationship between CSMD1 and CYP19A1, we attempted to reverse the AI sensitization effect resulting from CSMD1 overexpression by knocking out endogenous *CYP19A1* using CRISPR/Cas9 methodology in T47D cells ([Figure 3C](#F3)). As shown in [Figure 3D](#F3), *CYP19*-KO abrogated the effect of CSMD1 overexpression on survival, indicating that *CYP19* is a major target that mediates the effect of *CSMD1* on anastrozole response. CSMD1 was recently shown to interact with SMAD3 in melanoma cells ([24](#B24)). SMAD3, activated by TGF-β3, was required for steroidogenic factor-1 (SF-1) binding to the *CYP19A1* type II promoter ([25](#B25)). We therefore investigated whether CSMD1 might be a component of the SMAD3–TGF-β3 receptor (TGF-βR) complex involved in *CYP19A1* transcription regulation. CSMD1 coprecipitated SMAD3 in breast cancer cells and human adipocytes, and a reciprocal immunoprecipitation further confirmed the interaction ([Figure 3E](#F3)). Overexpression of CSMD1 enhanced the interaction between SMAD3 and TGF-βR and resulted in SMAD3 activation, as evidenced by increased pSMAD3 ([Figure 3F](#F3)). This regulation of *CSMD1* on *CYP19A1* was impaired after the knockdown of *SF-1* or *SMAD3* ([Supplemental Figure 5](#sd)). These results suggest that CSMD1 is a scaffolding protein that brings SMAD3 and TGF-βR together and promotes phosphorylation of SMAD3. To further confirm that the regulation of the *CSMD1* SNP on AI response is through the CSMD1/SMAD3/CYP19A1 axis, we treated LCLs carrying either WT or variant SNP genotypes with androstenedione alone or in combination with an individual AI. As shown in [Figure 3G](#F3), cells with different rs6990851 SNP genotypes showed striking differences in pSMAD3 and CYP19A1 levels. In particular, WT cells had increased pSMAD3 and CYP19A1 levels after androstenedione treatment, and significantly reduced pSMAD3 and CYP19A1 levels when anastrozole was added. Importantly, addition of letrozole or exemestane did not alter the protein levels in WT cells. In contrast, in the variant cells, androstenedione treatment did not change pSMAD3 and CYP19A1 levels compared with vehicle treatment, while addition of anastrozole, but not letrozole or exemestane, significantly upregulated pSMAD3 and CYP19A1 levels. Therefore, rs6990851 upregulated CSMD1 expression in the presence of anastrozole, but not the other 2 AIs ([Figure 2](#F2)), resulting in SMAD3 activation and increased *CYP19A1* gene expression ([Figure 3G](#F3)), leading to increased sensitivity to anastrozole ([Figure 3A](#F3)).

### Figure 3. Effects of CSMD1 SNP on anastrozole response and mechanisms involved in CSMD1 regulation of CYP19.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5089/7455128/efac21bc943a/jciinsight-5-137571-g209.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7455128_jciinsight-5-137571-g209.jpg)

(A) CSMD1 SNP-dependent effect on AI responses. w/w, homozygous WT (n = 5); w/v, heterozygous (n = 5); and v/v, homozygous variant LCLs (n = 5). (w/w versus w/v, P < 0.0001; w/w versus v/v, P < 0.0001). (B) Increased anastrozole sensitivity in cells overexpressing CSMD1 (empty vector:Ana [EV:Ana] versus OE CSMD1:Ana, P < 0.0001). Side panel shows mRNA expression in cell lines tested. MCF7AC1 (AC1), AC1-LetR (LetR), and adipocyte. (C) Western blot showing CYP19A1 expression in 3 single clones of CYP19A1-KO T47D cells using CRISPR/Cas9. CYP19A1-KO clone 3 (KO-3) was selected for further experiments. (D) Anastrozole sensitivity after overexpressing CSMD1 in CYP19A1-KO T47D cells (KO) and control cells (Con) shows the reversal of anastrozole sensitivity caused by overexpression of CSMD1. (E) Detection of CSMD1 and SMAD3 interaction using immunoprecipitation, followed by Western blot analysis with indicated antibodies. (F) Increased SMAD3–TGF-βR binding in CSMD1-overexpressed cells. (G) CSMD1 SNP effect on pSMAD3 and CYP19A1 protein levels. A, androstenedione; Ana, anastrozole; Let, letrozole; and Exe, exemestane. Data are represented by ± SEM of 3 independent experiments. **P < 0.01. Two-way ANOVA plus Tukey.

### Anastrozole potentiates estrogen’s effect on ERα transcription activity and ERα degradation.

The observation of the *CSMD1* SNP and AI-dependent regulation of gene expression and differential response to individual AI further confirmed our previous finding. Our previous studies indicated that anastrozole is different from the other 2 AIs by possessing a second mechanism of action ([18](#B18)), being an ERα agonist in a similar fashion to E2. It can activate ERα-dependent transcription, but the effect decreased with increasing concentrations of anastrozole due to induction of ERα degradation ([18](#B18)). To further explore the impact of anastrozole effect on ERα activation and degradation, we tested various concentrations of E2 and anastrozole, as well the 2 combined. We showed that anastrozole plus E2 at 0.1 or 1 nM showed approximately 2.4- and 3.01-fold increases, respectively, in luciferase activity compared with anastrozole or E2 alone at that concentration in *CYP19*-KO T47D cells ([Figure 4A, P](#F4)** < 0.001). However, when E2 concentration increased to 10 nM, combination with anastrozole led to a significant inhibition in luciferase activity compared with anastrozole or E2 alone ([Figure 4A, P](#F4)** < 0.001). Letrozole and exemestane had no impact on E2-induced luciferase activity ([Figure 4A](#F4)). These observations suggest that anastrozole has a cumulative effect with low-dose E2 to activate ERα-mediated transcriptional activation. However, at higher concentration of E2, it showed a cumulative effect with anastrozole on ERα degradation, leading to reduced ERE-dependent transcription activity. We further tested the combination effect on ERα protein level. Anastrozole (10 nM) combined with E2 at 10 or 100 nM decreased ERα protein level, while E2 or anastrozole at 10 nM alone was not able to degrade ERα ([Figure 4B](#F4)). Consistent with previous findings ([26](#B26)) showing that high concentrations of E2 led to a proteasome-mediated degradation of ERα protein, we found that the proteasome inhibitor MG132, but not the autophagy inhibitor 3MA, could inhibit the protein degradation induced by anastrozole ([Figure 4C](#F4)).

### Figure 4. Anastrozole potentiates estrogen effect on ERα transcription activity and ERα degradation.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5089/7455128/333d96b66b82/jciinsight-5-137571-g210.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7455128_jciinsight-5-137571-g210.jpg)

(A) ERE-dependent luciferase assay in CYP19A1 CRISPR-KO T47D cells treated with indicated concentrations of E2, anastrozole (Ana), letrozole (Let), or exemestane (Exe). (B) Cells treated with E2 and anastrozole either alone or combined at the indicated concentrations. Western blot was performed to determine ERα protein level. (C) Anastrozole induced ERα degradation is proteasome dependent. Cells treated with vehicle, 100 nM anastrozole alone or anastrozole plus 20 μM 3MA, or 10 μM MG132 following Western blot analysis of ERα protein levels. (D) Venn diagram showing the overlap between genes differentially expressed in response to anastrozole with or without E2 (FDR < 0.05). Data are represented by ± SEM of 3 independent experiments. **P < 0.01. One-tailed t test, Bonferroni correction for multiple testing.

### Anastrozole regulates a transcriptome distinct from E2 in breast cancer cells.

Since anastrozole can function as an ERα ligand, to investigate whether anastrozole regulates similar or different sets of genes compared with E2, we performed RNA sequencing (RNA-seq) using T47D breast cancer cells treated with anastrozole (10 nM) alone, E2 (0.1 nM) alone or anastrozole (10 nM) plus E2 (0.1 nM). These concentrations were chosen based on the ability of anastrozole to increase luciferase activity without degrading ERα. Anastrozole alone significantly altered levels of 476 transcripts compared with the vehicle control (FDR < 0.05), among which, 398 genes overlapped with E2 single treatment. As shown in the Venn diagram, anastrozole plus E2 treatment resulted in altered expression of 513 transcripts, 204 of which were common with anastrozole alone, and 384 of the 513 were common with E2-alone treatment ([Figure 4D](#F4), [Supplemental Table 2](#sd), and [Supplemental Figure 6A](#sd)). Selected genes identified by RNA-seq were further validated by quantitative PCR (qPCR) following anastrozole, E2, or anastrozole plus E2 treatment ([Supplemental Figure 6B](#sd)). Gene set enrichment analysis (GSEA) for anastrozole treatment showed that, compared with E2, anastrozole regulated genes involved in steroid biosynthesis, DNA replication, toxoplasmosis, VEGF signaling pathway, and actin cytoskeleton ([Supplemental Table 3](#sd)). Collectively, these findings suggest that, although at large, anastrozole behaves very similar to E2 with regard to its effect on downstream gene regulation (384 genes), it also preferentially regulates pathways (204 genes) that are different from E2.

### Therapeutic effect of the combination of anastrozole with E2.

To test whether the combined effects of anastrozole and E2 on ERα protein degradation might lead to augmented antitumor activity compared with anastrozole alone, we evaluated cytotoxicity in *CYP19*-KO, letrozole-resistant AC1-LetR, and anastrozole-resistant MCF7/anastrozole resistant (Ana^R^) cells ([27](#B27)) with anastrozole with or without E2 treatment. As shown in [Figure 5, A–C](#F5), 10 and 100 nM of E2 sensitized these cells to anastrozole, but not letrozole or exemestane, and anastrozole plus 10 nM E2 showed better therapeutic effect than any of the 3 AI alone in anastrozole-resistant cells ([Supplemental Figure 6C](#sd)). Treatment with 10 or 100 nM E2 plus 100 nM anastrozole also increased the degradation of ERα protein compared with cells exposed to 100 nM anastrozole alone ([Figure 5D](#F5)). It is important to emphasize that E2 alone, at the concentrations of 10 or 100 nM, did not induce ERα degradation ([Figure 5D](#F5)) — neither did it exert cytotoxic effect ([Supplemental Figure 6D](#sd)). To further confirm the combination effect, we used organoids grown from 2 primary ER^+^ breast cancer patient-derived xenografts (PDXs) ([28](#B28)) and treated with anastrozole plus E2. In all 3 organoids derived from 2 unique breast cancer patients, we confirmed ERα positivity ([Figure 5D](#F5), Western blot). The dose response after 72-hour exposure to AI alone or in combination with 100 nM E2 was obtained. The number of survival organoids was significantly reduced 72 hours after exposure to anastrozole plus E2, but not to letrozole or exemestane plus E2 ([Figure 6A](#F6)). To further assess the drug combination effects over time (9 days), we used a luminescence survival assay. Here, we selected a 200 nM anastrozole concentration based on the previous organoid response curves ([Figure 6A](#F6)). As shown in [Figure 6B](#F6), anastrozole and E2 significantly inhibited organoid growth compared with anastrozole alone (*P* < 0.01).

### Figure 5. Sensitization effect of E2 on anastrozole response.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5089/7455128/760a5b59e626/jciinsight-5-137571-g211.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7455128_jciinsight-5-137571-g211.jpg)

(A–C) Dose response of anastrozole in the presence of E2 at indicated concentrations in CYP19A1-KO cells (A) (Ana versus Ana + 10E2, P < 0.0001; Ana versus Ana + 100E2, P < 0.0001); AC1-LetR cells, the letrozole-resistant cells (B) (Ana versus Ana + 10E2, P < 0.0001; Ana versus Ana + 100E2, P < 0.0001); and MCF7/AnaR, the anastrozole-resistant cells (C) (Ana versus Ana + 10E2, P < 0.0001; Ana versus Ana + 100E2, P < 0.0001). (D) Western blot and quantification of ERα protein levels after anastrozole and E2 treatment. Data are represented by ± SEM of 3 independent experiments. **P < 0.01. Two-way ANOVA plus Tukey.

### Figure 6. Sensitization effect of E2 on anastrozole response in breast cancer PDX–derived organoids.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5089/7455128/a4ef5d80846e/jciinsight-5-137571-g212.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7455128_jciinsight-5-137571-g212.jpg)

(A) AI dose response in the presence of E2 in 3 organoids derived from breast cancer patients. Western blot showed ER positivity in the organoids (ER-positive control, MCF7 and BT474; ER-negative control, MDA-231). (B) Organoid growth curves in the presence of 200 nM anastrozole and/or 100 nM E2. Data are represented by ± SEM of 3 independent experiments. **P < 0.01. Two-way ANOVA plus Tukey.

Compared with the third-generation AIs, fulvestrant has superior efficacy and is a preferred treatment option for patients with hormone receptor–positive (HR^+^) locally advanced or metastatic breast cancer (MBC) based on the FALCON trial ([29](#B29)) and in the neoadjuvant setting based on the CARMINA 02 trial ([30](#B30), [31](#B31)). However, up to half of ER^+^ MBC patients show intrinsic resistance, and ultimately all of them develop acquired resistance to fulvestrant ([29](#B29)). To test the efficacy of anastrozole plus E2 in the fulvestrant-resistant setting, we treated fulvestrant-resistant MCF7/164R-7 cells ([27](#B27)) with individual AI alone or combined with increasing concentrations of E2. We found that, in fulvestrant-resistant cells, individual AI alone had a slightly better cytotoxic effect than fulvestrant. However, only anastrozole showed an E2 dose-dependent sensitization, and this phenomenon was not observed for letrozole or exemestane ([Figure 7A](#F7) and [Supplemental Figure 6E](#sd)). Since fulvestrant is used in the AI-resistant setting, we then compared the response of anastrozole plus E2 to fulvestrant in MCF7/Ana^R^-2 cells. Our data reveal that the combination of anastrozole with 100 nM E2 showed an effect similar to that of fulvestrant ([Figure 7B](#F7), top panel). MCF7/Ana^R^-2 cell growth was dramatically reduced in the presence of 100 nM anastrozole plus E2 after 3 days ([Figure 7B](#F7), bottom panel). Even in endocrine-naive breast cancer cells (MCF7AC1, ZR-75-1, and T47D), anastrozole plus E2 improved the therapeutic effects compared with anastrozole or fulvestrant alone ([Figure 7B](#F7), top and panels). Finally, anastrozole plus E2 improved the therapeutic response in breast cancer PDX organoids compared with fulvestrant ([Figure 7C](#F7)). Our findings suggest that anastrozole plus E2 could improve outcome in both endocrine naive and -resistant breast cancers.

### Figure 7. The comparison of efficacy between anastrozole plus E2 versus fulvestrant.

![Figure 7](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5089/7455128/54deb4e077f8/jciinsight-5-137571-g213.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7455128_jciinsight-5-137571-g213.jpg)

(A) Effects of anastrozole + E2 in fulvestrant-resistant cells. Fulvestrant (F) versus Ana, P = 0.0009; F versus Ana + 10E2, P < 0.0001; F versus Ana + 100E2, P < 0.0001; F versus Ana + 500E2, P < 0.0001; Ana versus Ana + 10E2, P = 0.0288; Ana versus Ana + 100E2, P < 0.0001; Ana versus Ana + 500E2, P < 0.0001. Quantitative analysis of survival cells corrected back to vehicle treatment was performed after 3 days of treatment with indicated drugs, as shown in the bar graph. **P < 0.01. Two-way ANOVA plus Tukey. (B and C) Comparison of anastrozole + E2 with fulvestrant in anastrozole-resistant MCF7/AnaR and 3 AI-naive cell lines (B), as well as in PDX-derived organoids (C). Quantitative analysis of survival cells corrected back to vehicle treatment was performed, as shown in the bar graph 3 days after treatment. Data are represented by ± SEM of 3 independent experiments. MCF7/AnaR: F versus Ana, P < 0.0001; F versus Ana + 100E2, P = 0.2093; Ana versus Ana + 100E2, P < 0.0001. MCF7AC1: F versus Ana + 100E2, P < 0.0001; Ana versus Ana + 100E2, P = 0.0013. ZR-75-1: F versus Ana + 100E2, P < 0.0001; Ana versus Ana + 100E2, P < 0.0001. T47D: F versus Ana + 100E2, P < 0.0001; Ana versus Ana + 100E2, P < 0.0001. Organoid-1-1: F versus Ana + 100E2, P < 0.0001; Ana versus Ana + 100E2, P = 0.0002; Ana versus F, P = 0.02. Organoid-1-2: F versus Ana + 100E2, P = 0.0013; Ana versus Ana + 100E2, P < 0.0001; Ana versus F, P = 0.013. Organoid-2: F versus Ana + 100E2, P < 0.0001; Ana versus Ana + 100E2, P = 0.0002; Ana versus F, P = 0.0026. *P < 0.05; **P < 0.01. Two-way ANOVA plus Tukey.

## Discussion

This study has identified potentially novel common genetic variants associated with estrogen suppression and breast cancer events in women treated with adjuvant AI therapy. None of these SNPs had been previously reported in association studies ([7](#B7), [12](#B12), [32](#B32)). Our results not only help identify potential new biomarkers for the selection of patients who might benefit from AIs therapy, but also enhance our understanding of mechanisms involved in anastrozole action, all of which have significant clinical implications. For our GWAS, we used 2 independent studies with 2 different but related AI response phenotypes, estrogen suppression and BCFI, and we identified 2 SNPs in the *CSMD1* gene that were associated with 1 of these 2 phenotypes ([Figure 1, B and C](#F1)). However, only rs6990851, and not rs6981827, was eQTL for *CSMD1*. These 2 SNPs have very low linkage disequilibrium. Previous studies have provided evidence that biochemical and genetic manipulations of motif-adjacent sequences can influence transcription factor activity ([20](#B20), [33](#B33)). Here, we show that *CSMD1* regulation is estrogen dependent and only the rs6990851 is SNP dependent, while rs6981827 is not. The rs6990851 SNP is in a region that has EREs within 500 bp of the SNP. We demonstrated that the rs6990851 SNP modifies ERα binding to an ERE in the *CSMD1* gene, leading to altered *CSMD1* and *CYP19* expression in an anastrozole-dependent fashion ([Figure 2G](#F2)). Mechanistically, CSMD1 regulated *CYP19A1* gene transcription by acting as a scaffolding protein for SMAD3 and TGF-βR ([Figure 3, E and G](#F3), and [Supplemental Figure 5](#sd)). The variant rs6990851 SNP allele that increased *CSMD1* expression also increased sensitivity to anastrozole, but not letrozole or exemestane ([Figure 3](#F3)). This result strongly suggested potential differences in mechanisms of action between anastrozole and the other 2 AIs, and this was also confirmed in our previous finding ([18](#B18)). We did not observe an SNP-drug interaction in the MA.27 trial, and the SNP showed a protective effect regardless of the treatment arm, exemestane versus anastrozole. Based on the SNP-dependent drug effects on *CSMD1* and *CYP19* in LCLs, we concluded that the variant rs6990851 SNP results in lower expression of *CSMD1* and, in turn, *CYP19* in the presence of androstenedione, leading to slow cell proliferation and better prognosis. The addition of exemestane or letrozole did not change the *CSMD1* and *CYP19* expression patterns compared with androstenedione treatment ([Figure 2, C and D](#F2)), suggesting that the protective effect of the SNP in the exemestane-treated arm might be mainly due to slow proliferation caused by low *CYP19* expression in subjects carrying the variant SNP. However, treatment with anastrozole in subjects carrying the variant SNP genotype upregulated *CSMD1* and *CYP19*, resulting in increased drug target for AIs. Therefore, we observed a more sensitive phenotype of anastrozole in LCLs containing the variant *CSMD1* rs6990851 SNP ([Figure 3A](#F3)). Therefore, the SNP was a protective SNP with outcomes in MA.27, regardless of the treatment arm. However, based on our results, the mechanisms underlying these association may be different between anastrozole and the other AIs.

The identification of anastrozole and SNP genotype–dependent regulation of gene expression ([Figure 2, B–G](#F2), and [Figure 3](#F3)) could be explained by a potentially novel mechanism of anastrozole action: anastrozole binds to ERα resulting in ERα degradation ([18](#B18)). At the transcriptional level, anastrozole-dependent transcripts significantly overlapped with E2, but also displayed marked differences from E2, indicating a unique function for anastrozole-ERα gene transcription regulation ([Figure 4D](#F4)). The differences between anastrozole and E2-driven transcriptional programs may be due to differential expression of coregulators that dictate specific ERα-DNA binding and transcriptional activity. Like tamoxifen ([34](#B34)), it is also possible that anastrozole might exert different functions through ERα, either as an agonist or antagonist in a tissue context–dependent manner, a process that is also dependent on the receptor-DNA complex interacting with different coregulators ([35](#B35)). Additional work to decipher these hypotheses is ongoing.

Finally, the effect of anastrozole on ERα protein degradation is potentiated by E2 ([Figure 4B](#F4)). In a short-term randomized study, anastrozole treatment significantly reduced mean ER expression from baseline in breast cancer patients ([36](#B36)). The downregulation of cellular ERα protein occurred without a reduction in ESR1 mRNA, consistent with our findings. Taking advantage of this mechanism, we tested the combination of anastrozole plus E2 as a potential therapeutic strategy, especially in AI- or fulvestrant-resistant breast cancer. We showed that anastrozole plus E2 inhibits proliferation of AI- and fulvestrant-resistant breast cancer cells in addition to its effect in hormone therapy–naive breast cancer cells ([Figure 5](#F5) and [Figure 7](#F7)). Furthermore, the same results were also observed in breast cancer PDX–derived organoids ([Figure 6, A and B](#F6)). Hence, the combination might offer a potential alternative therapeutic strategy against refractory ER^+^ breast cancer. E2 is being used in the clinic to treat breast cancer with doses as high as 30 mg/d ([37](#B37)). The E2 effect on tumor inhibition has also been observed in tamoxifen-resistant breast cancer in mice, where estrogen can reverse tamoxifen resistance ([38](#B38)). Therefore, the combination of anastrozole plus E2 might be a feasible therapeutic option that could be tested in the future. In our study, we also showed that the combination of anastrozole and E2 was superior to E2 alone ([Supplemental Figure 6E](#sd)), further suggesting that E2 potentiates the effect of anastrozole on ERα degradation. The development of CDK4/6 inhibitors has changed the therapeutic management of HR^+^ MBC. Palbociclib, ribociclib, and abemaciclib are approved in combination with an AI or fulvestrant for HR^+^ MBC ([39](#B39), [40](#B40)). Abemaciclib is also approved as a monotherapy for pretreated patients ([41](#B41)). Key questions in the field include whether all patients with HR^+^ MBC should receive a CDK4/6 inhibitor up front and what the mechanism may be of clinical resistance. Patients who progressed on CDK4/6 inhibitor plus AI are currently offered additional therapies such as fulvestrant as a single agent, but the PFS with fulvestrant in the post-CDK4/6 setting is unclear. Therefore, there is a great need to identify additional therapies to improve outcome. Our observations here, combination of anastrozole and E2, may offer additional therapeutic strategies to further improve outcomes of HR^+^ MBC.

Taking advantage of the results of 2 GWAS with related but different phenotypes associated with AI action, we have identified a *CSMD1* SNP as a predicative marker for AI response. Furthermore, the cumulative effect of anastrozole plus E2 on ERα degradation might provide an alternative therapy for ER-positive patients, especially those who are already resistant to hormonal therapy. The differences in E2- and anastrozole-dependent transcription regulation should help us better understand mechanisms of anastrozole resistance and could help identify additional drug targets or treatment options to overcome AI resistance.

## Methods

### GWAS of estrogen response in M3 cohort.

For the GWAS of estrogen suppression to anastrozole therapy, we used samples from the M3 AI Pharmacogenomics Study ([19](#B19)). To ensure that all included patients were compliant with therapy, we included only patients who had detectable levels of either anastrozole or one of its metabolites. Therefore, we excluded 12 patients who had a discordant change in direction between their estrone and E2 concentrations after AI treatment. We also excluded 8 additional patients whose preanastrozole and on-anastrozole E2 concentrations were either undetectable or zero. Our final analysis sample size was 624 patients. To assess SNP effects on estrogen response, 3 phenotypes were analyzed: (a) absolute change in E1; (b) absolute change in E2; and (c) absolute change in the sum of E1 and E2. A van der Waerden transformation was applied to address the skewed Gaussian distributions. Linear regression models were used with additive SNP effects and the following covariates: age, BMI, the first 6 eigenvectors from a principal components analysis, and either baseline E1, E2, or the sum of E1 and E2. Q-Q plots were used to assess adherence of the resultant *P* values to the null distribution. Common genes having SNPs reaching GWAS significance were retained for further analyses. All analyses were performed using R statistical computing software (v3.0.2) and PLINK (v1.07) ([7](#B7)).

### Survival analysis in the MA.27 cohort.

To determine if the SNP was associated with BCFI, the nonparametric log-rank test was initially performed, with each SNP coded as a categorical variable as 0, 1, or 2 copies of the minor allele. SNPs found to be associated with BCFI were then forwarded to a Cox proportional hazards model where SNPs were modeled as a continuous dosage variable. In both cases, the null hypothesis of no association between SNP alleles and BCFI were being tested. Satisfaction of the assumption of proportional hazards was tested after fitting the Cox model in R (version 3.0.2) followed by the Grambsch and Therneau test. To select clinical variables that might confound the relationship between SNP and BCFI, univariate Cox regression models were fitted with drug usage, chemotherapy usage, tumor stage, age, BMI, and PR status separately. For all continuous variables, the linearity assumption was tested, and if violated, that variable was modeled using restricted cubic splines. Variables found to have a significant association (*P* < 0.05) with BCFI were included in a forward stepwise manner to build a multivariate Cox model with SNP. These analyses were performed in the full cohort and also separately within each treatment arm. To determine if the SNP modified the effect of the treatment on BCFI, Cox models were fitted with interaction terms between SNPs and treatment arm. Similarly, interactions between the SNPs and chemotherapy use, progesterone receptor status, and tumor stage status were tested. All analyses were done in R (version 3.0.2).

We performed Fisher’s exact tests to examine the imbalances between North American patients who were included in this GWAS and those who were not. Stepwise selection method was used to evaluate additional clinical variables associated with a breast cancer recurrence event. We employed a stratified genome-wide Cox-proportional hazards model using significant stratification factors with control for additional covariates including treatment arm, cohort, race, ER/PR status, tumor stage, Eastern Cooperative Oncology Group (ECOG) performance score, and bisphosphonate use. All the analyses were run using the R statistical computing package, PLINK, and SAS (SAS Institute) ([7](#B7)).

### RNA-seq analysis and normalization.

T47D cells were treated with vehicle, E2 (0.1 nM), anastrozole (10 nM), or a combination of E2 and anastrozole for 24 hours. RNA-seq was performed and analyzed as described in the [Supplemental Methods](#sd). The RNA-seq data are publicly available from NCBI Gene Expression Omnibus ([http://www.ncbi.nlm.nih.gov/geo](http://www.ncbi.nlm.nih.gov/geo)) under SuperSeries accession no. [GSE150683](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE150683).

### Organoid derivation, 3D cell culture, and viability assay.

ER+ PDXs from the Breast Cancer Genome Guided Therapy Study (BEAUTY) were generated according to a previously described protocol ([28](#B28)). Tumors were injected s.c. into 6- to 8-week-old female nonobese diabetic/severe combined immunodeficient NOD/SCID/IL-2γ receptor–null (NSG) mice (The Jackson Laboratory). Tumor cells from 2 breast cancer PDXs were isolated using the human Tumor Dissociation Kit (Miltenyi Biotec). Briefly, tumors were minced and then transferred into the gentle MACS C Tube and run through the 7C_h_TDK3 program according to manufacturer’s protocol. The tubes were then centrifuged (2000 *g* for 5 minutes at room temperature) to collect the sample material. Samples were resuspended and applied to a MACS SmartStrainer (70 μm). A Mouse Cell Depletion Kit (Miltenyi Biotec) was used to enrich human cells. Specifically, the cell pellet was suspended in buffer, 20 μL of the mouse cell depletion cocktail was added, and the mixtures were incubated for 15 minutes at 4°C. Then, magnetic separation with LS Columns was performed to collect human cells. Organoids were cultured in 96-well low-binding NanoCulture plate (Organogenix) in DMEM supplemented with 10% FCS, 1% glutamax, 1% sodium pyruvate, nonessential amino acids, and 1% penicillin-streptomycin (Invitrogen) at 37°C, 5% CO_2_. Details of the organoid viability assay are described in the [Supplemental Methods](#sd).

### Statistics.

For cell survival, cell proliferation, gene expression, and quantifications, data are represented as the mean ± SEM of 3 independent experiments. Unless otherwise described, 2-way ANOVA was performed to test group difference. Then post hoc analysis was carried out to check if specific groups are significantly different or similar. Tukey’s honest significant differences (HSD; R function: TukeyHSD()), which is essentially a modified *t* test corrected for multiple comparisons, was applied in this analysis. Statistical significance level is *P* < 0.05.

### Study approval.

M3 study was reviewed and approved by local IRBs at all participating institutions (Mayo Clinic IRB, M.D. Anderson Cancer Center IRB, and Memorial Sloan Kettering Cancer Center IRB). Written informed consent was obtained from each patient. MA.27 study was approved by local IRBs in accordance with assurances filed with, and approved by, the Department of Health and Human Services. All animal studies were reviewed and approved by the Mayo Clinic IACUC.

## Author contributions

Conception and design were contributed by JC, JNI, and LW. Development of methodology were contributed by JC, TD, LW, JNI, and RMW. Acquisition of data was contributed by JC, JNI, AUB, MER, MJE, PEG, LES, MPG, and BG. Analysis and interpretation of data (e.g., statistical analysis, biostatistics, computational analysis) were contributed by JC, TD, KRK, EEC, JN, HL, and MGB. Writing, review, and/or revision of the manuscript were contributed by JC, JNI, LW, and TD. Administrative, technical, or material support was contributed by JC, TD, EEC, and MGB. Study supervision was contributed by JC, JNI, LW, and RMW.

## Supplementary Material

## Acknowledgments

The authors acknowledge the women who participated in the M3 and MA.27 clinical trials and who provided DNA and consent for its use in genetic studies. This research was supported by the Breast Cancer Research Foundation (BCRF), UG1CA18967, P50CA116201 (Mayo Clinic Breast Cancer Specialized Program of Research Excellence), U1961388 (the Pharmacogenomics Research Network), the George M. Eisenberg Foundation for Charities, the Nan Sawyer Breast Cancer Fund, and NIH/NCI Cancer Center Support Grant P30 CA008748.

### Version 1. 07/23/2020

### Version 1. 07/23/2020

In-Press Preview

### Version 2. 08/20/2020

Electronic publication

## Footnotes

## Contributor Information

Junmei Cairns, Email: cairns.junmei@mayo.edu.

James N. Ingle, Email: ingle.james@mayo.edu.

Krishna R. Kalari, Email: Kalari.Krishna@mayo.edu.

Erin E. Carlson, Email: Carlson.Erin@mayo.edu.

Jie Na, Email: Na.Jie@mayo.edu.

Aman U. Buzdar, Email: abuzdar@mdanderson.org.

Mark E. Robson, Email: robsonm@mskcc.org.

Matthew J. Ellis, Email: matthew.ellis@bcm.edu.

Paul E. Goss, Email: PGOSS@mgh.harvard.edu.

Lois E. Shepherd, Email: Lshepherd@ctg.queensu.ca.

Barbara Goodnature, Email: goodnatureb@hotmail.com.

Matthew P. Goetz, Email: goetz.matthew@mayo.edu.

Richard M. Weinshilboum, Email: weinshilboum.richard@mayo.edu.

Hu Li, Email: li.hu@mayo.edu.

Mehrab Ghanat Bari, Email: m.ghanatbari@gmail.com.

Liewei Wang, Email: wang.liewei@mayo.edu.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Early Breast Cancer Trialists’ Collaborative Group (EBCTCG) et al. Relevance of breast cancer hormone receptors and other factors to the efficacy of adjuvant tamoxifen: patient-level meta-analysis of randomised trials. Lancet. 2011;378(9793):771–784. doi: 10.1016/S0140-6736(11)60993-8.  [DOI](https://doi.org/10.1016/S0140-6736(11)60993-8) | [PMC free article](/articles/PMC3163848/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21802721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Relevance%20of%20breast%20cancer%20hormone%20receptors%20and%20other%20factors%20to%20the%20efficacy%20of%20adjuvant%20tamoxifen:%20patient-level%20meta-analysis%20of%20randomised%20trials&volume=378&issue=9793&publication_year=2011&pages=771-784&pmid=21802721&doi=10.1016/S0140-6736(11)60993-8&)

2. Early Breast Cancer Trialists’ Collaborative Group (EBCTCG) Aromatase inhibitors versus tamoxifen in early breast cancer: patient-level meta-analysis of the randomised trials. Lancet. 2015;386(10001):1341–1352. doi: 10.1016/S0140-6736(15)61074-1.  [DOI](https://doi.org/10.1016/S0140-6736(15)61074-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26211827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Aromatase%20inhibitors%20versus%20tamoxifen%20in%20early%20breast%20cancer:%20patient-level%20meta-analysis%20of%20the%20randomised%20trials&volume=386&issue=10001&publication_year=2015&pages=1341-1352&pmid=26211827&doi=10.1016/S0140-6736(15)61074-1&)

3. Mauri D, Pavlidis N, Polyzos NP, Ioannidis JP. Survival with aromatase inhibitors and inactivators versus standard hormonal therapy in advanced breast cancer: meta-analysis. J Natl Cancer Inst. 2006;98(18):1285–1291. doi: 10.1093/jnci/djj357.  [DOI](https://doi.org/10.1093/jnci/djj357) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16985247/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Survival%20with%20aromatase%20inhibitors%20and%20inactivators%20versus%20standard%20hormonal%20therapy%20in%20advanced%20breast%20cancer:%20meta-analysis&author=D%20Mauri&author=N%20Pavlidis&author=NP%20Polyzos&author=JP%20Ioannidis&volume=98&issue=18&publication_year=2006&pages=1285-1291&pmid=16985247&doi=10.1093/jnci/djj357&)

4. Benz CC, et al. Estrogen-dependent, tamoxifen-resistant tumorigenic growth of MCF-7 cells transfected with HER2/neu. Breast Cancer Res Treat. 1992;24(2):85–95. doi: 10.1007/BF01961241.  [DOI](https://doi.org/10.1007/BF01961241) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8095168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res%20Treat&title=Estrogen-dependent,%20tamoxifen-resistant%20tumorigenic%20growth%20of%20MCF-7%20cells%20transfected%20with%20HER2/neu&author=CC%20Benz&volume=24&issue=2&publication_year=1992&pages=85-95&pmid=8095168&doi=10.1007/BF01961241&)

5. Massarweh S, et al. Mechanisms of tumor regression and resistance to estrogen deprivation and fulvestrant in a model of estrogen receptor-positive, HER-2/neu-positive breast cancer. Cancer Res. 2006;66(16):8266–8273. doi: 10.1158/0008-5472.CAN-05-4045.  [DOI](https://doi.org/10.1158/0008-5472.CAN-05-4045) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16912207/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Mechanisms%20of%20tumor%20regression%20and%20resistance%20to%20estrogen%20deprivation%20and%20fulvestrant%20in%20a%20model%20of%20estrogen%20receptor-positive,%20HER-2/neu-positive%20breast%20cancer&author=S%20Massarweh&volume=66&issue=16&publication_year=2006&pages=8266-8273&pmid=16912207&doi=10.1158/0008-5472.CAN-05-4045&)

6. Middlebrooks CD, et al. Association of germline variants in the APOBEC3 region with cancer risk and enrichment with APOBEC-signature mutations in tumors. Nat Genet. 2016;48(11):1330–1338. doi: 10.1038/ng.3670.  [DOI](https://doi.org/10.1038/ng.3670) | [PMC free article](/articles/PMC6583788/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27643540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Association%20of%20germline%20variants%20in%20the%20APOBEC3%20region%20with%20cancer%20risk%20and%20enrichment%20with%20APOBEC-signature%20mutations%20in%20tumors&author=CD%20Middlebrooks&volume=48&issue=11&publication_year=2016&pages=1330-1338&pmid=27643540&doi=10.1038/ng.3670&)

7. Ingle JN, et al. Genetic Polymorphisms in the Long Noncoding RNA MIR2052HG Offer a Pharmacogenomic Basis for the Response of Breast Cancer Patients to Aromatase Inhibitor Therapy. Cancer Res. 2016;76(23):7012–7023. doi: 10.1158/0008-5472.CAN-16-1371.  [DOI](https://doi.org/10.1158/0008-5472.CAN-16-1371) | [PMC free article](/articles/PMC5135610/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27758888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Genetic%20Polymorphisms%20in%20the%20Long%20Noncoding%20RNA%20MIR2052HG%20Offer%20a%20Pharmacogenomic%20Basis%20for%20the%20Response%20of%20Breast%20Cancer%20Patients%20to%20Aromatase%20Inhibitor%20Therapy&author=JN%20Ingle&volume=76&issue=23&publication_year=2016&pages=7012-7023&pmid=27758888&doi=10.1158/0008-5472.CAN-16-1371&)

8. Ma CX, Reinert T, Chmielewska I, Ellis MJ. Mechanisms of aromatase inhibitor resistance. Nat Rev Cancer. 2015;15(5):261–275. doi: 10.1038/nrc3920.  [DOI](https://doi.org/10.1038/nrc3920) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25907219/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=Mechanisms%20of%20aromatase%20inhibitor%20resistance&author=CX%20Ma&author=T%20Reinert&author=I%20Chmielewska&author=MJ%20Ellis&volume=15&issue=5&publication_year=2015&pages=261-275&pmid=25907219&doi=10.1038/nrc3920&)

9. Li S, et al. Endocrine-therapy-resistant ESR1 variants revealed by genomic characterization of breast-cancer-derived xenografts. Cell Rep. 2013;4(6):1116–1130. doi: 10.1016/j.celrep.2013.08.022.  [DOI](https://doi.org/10.1016/j.celrep.2013.08.022) | [PMC free article](/articles/PMC3881975/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24055055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&title=Endocrine-therapy-resistant%20ESR1%20variants%20revealed%20by%20genomic%20characterization%20of%20breast-cancer-derived%20xenografts&author=S%20Li&volume=4&issue=6&publication_year=2013&pages=1116-1130&pmid=24055055&doi=10.1016/j.celrep.2013.08.022&)

10. Aguilar H, et al. Biological reprogramming in acquired resistance to endocrine therapy of breast cancer. Oncogene. 2010;29(45):6071–6083. doi: 10.1038/onc.2010.333.  [DOI](https://doi.org/10.1038/onc.2010.333) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20711236/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Biological%20reprogramming%20in%20acquired%20resistance%20to%20endocrine%20therapy%20of%20breast%20cancer&author=H%20Aguilar&volume=29&issue=45&publication_year=2010&pages=6071-6083&pmid=20711236&doi=10.1038/onc.2010.333&)

11. Zilli M, et al. Molecular mechanisms of endocrine resistance and their implication in the therapy of breast cancer. Biochim Biophys Acta. 2009;1795(1):62–81. doi: 10.1016/j.bbcan.2008.08.003.  [DOI](https://doi.org/10.1016/j.bbcan.2008.08.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18804516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Molecular%20mechanisms%20of%20endocrine%20resistance%20and%20their%20implication%20in%20the%20therapy%20of%20breast%20cancer&author=M%20Zilli&volume=1795&issue=1&publication_year=2009&pages=62-81&pmid=18804516&doi=10.1016/j.bbcan.2008.08.003&)

12. Wang L, et al. Functional genetic polymorphisms in the aromatase gene CYP19 vary the response of breast cancer patients to neoadjuvant therapy with aromatase inhibitors. Cancer Res. 2010;70(1):319–328. doi: 10.1158/0008-5472.CAN-09-3224.  [DOI](https://doi.org/10.1158/0008-5472.CAN-09-3224) | [PMC free article](/articles/PMC2804935/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20048079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Functional%20genetic%20polymorphisms%20in%20the%20aromatase%20gene%20CYP19%20vary%20the%20response%20of%20breast%20cancer%20patients%20to%20neoadjuvant%20therapy%20with%20aromatase%20inhibitors&author=L%20Wang&volume=70&issue=1&publication_year=2010&pages=319-328&pmid=20048079&doi=10.1158/0008-5472.CAN-09-3224&)

13. Smith IE, Dowsett M. Aromatase inhibitors in breast cancer. N Engl J Med. 2003;348(24):2431–2442. doi: 10.1056/NEJMra023246.  [DOI](https://doi.org/10.1056/NEJMra023246) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12802030/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Aromatase%20inhibitors%20in%20breast%20cancer&author=IE%20Smith&author=M%20Dowsett&volume=348&issue=24&publication_year=2003&pages=2431-2442&pmid=12802030&doi=10.1056/NEJMra023246&)

14. Sendur MA, Aksoy S, Zengin N, Altundag K. Efficacy of adjuvant aromatase inhibitor in hormone receptor-positive postmenopausal breast cancer patients according to the body mass index. Br J Cancer. 2012;107(11):1815–1819. doi: 10.1038/bjc.2012.473.  [DOI](https://doi.org/10.1038/bjc.2012.473) | [PMC free article](/articles/PMC3504952/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23099804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Efficacy%20of%20adjuvant%20aromatase%20inhibitor%20in%20hormone%20receptor-positive%20postmenopausal%20breast%20cancer%20patients%20according%20to%20the%20body%20mass%20index&author=MA%20Sendur&author=S%20Aksoy&author=N%20Zengin&author=K%20Altundag&volume=107&issue=11&publication_year=2012&pages=1815-1819&pmid=23099804&doi=10.1038/bjc.2012.473&)

15. Pfeiler G, et al. Impact of body mass index on the efficacy of endocrine therapy in premenopausal patients with breast cancer: an analysis of the prospective ABCSG-12 trial. J Clin Oncol. 2011;29(19):2653–2659. doi: 10.1200/JCO.2010.33.2585.  [DOI](https://doi.org/10.1200/JCO.2010.33.2585) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21555684/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Impact%20of%20body%20mass%20index%20on%20the%20efficacy%20of%20endocrine%20therapy%20in%20premenopausal%20patients%20with%20breast%20cancer:%20an%20analysis%20of%20the%20prospective%20ABCSG-12%20trial&author=G%20Pfeiler&volume=29&issue=19&publication_year=2011&pages=2653-2659&pmid=21555684&doi=10.1200/JCO.2010.33.2585&)

16. Goss PE, et al. Exemestane versus anastrozole in postmenopausal women with early breast cancer: NCIC CTG MA.27--a randomized controlled phase III trial. J Clin Oncol. 2013;31(11):1398–1404. doi: 10.1200/JCO.2012.44.7805.  [DOI](https://doi.org/10.1200/JCO.2012.44.7805) | [PMC free article](/articles/PMC3612593/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23358971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Exemestane%20versus%20anastrozole%20in%20postmenopausal%20women%20with%20early%20breast%20cancer:%20NCIC%20CTG%20MA.27--a%20randomized%20controlled%20phase%20III%20trial&author=PE%20Goss&volume=31&issue=11&publication_year=2013&pages=1398-1404&pmid=23358971&doi=10.1200/JCO.2012.44.7805&)

17. Smith I, et al. Comparative Efficacy and Safety of Adjuvant Letrozole Versus Anastrozole in Postmenopausal Patients With Hormone Receptor-Positive, Node-Positive Early Breast Cancer: Final Results of the Randomized Phase III Femara Versus Anastrozole Clinical Evaluation (FACE) Trial. J Clin Oncol. 2017;35(10):1041–1048. doi: 10.1200/JCO.2016.69.2871.  [DOI](https://doi.org/10.1200/JCO.2016.69.2871) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28113032/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Comparative%20Efficacy%20and%20Safety%20of%20Adjuvant%20Letrozole%20Versus%20Anastrozole%20in%20Postmenopausal%20Patients%20With%20Hormone%20Receptor-Positive,%20Node-Positive%20Early%20Breast%20Cancer:%20Final%20Results%20of%20the%20Randomized%20Phase%20III%20Femara%20Versus%20Anastrozole%20Clinical%20Evaluation%20(FACE)%20Trial&author=I%20Smith&volume=35&issue=10&publication_year=2017&pages=1041-1048&pmid=28113032&doi=10.1200/JCO.2016.69.2871&)

18. Ingle JN, et al. Anastrozole has an Association between Degree of Estrogen Suppression and Outcomes in Early Breast Cancer and is a Ligand for Estrogen Receptor α. Clin Cancer Res. 2020;26(12):2986–2996. doi: 10.1158/1078-0432.CCR-19-3091.  [DOI](https://doi.org/10.1158/1078-0432.CCR-19-3091) | [PMC free article](/articles/PMC7299827/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32098767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Anastrozole%20has%20an%20Association%20between%20Degree%20of%20Estrogen%20Suppression%20and%20Outcomes%20in%20Early%20Breast%20Cancer%20and%20is%20a%20Ligand%20for%20Estrogen%20Receptor%20%CE%B1&author=JN%20Ingle&volume=26&issue=12&publication_year=2020&pages=2986-2996&pmid=32098767&doi=10.1158/1078-0432.CCR-19-3091&)

19. Ingle JN, et al. Estrogens and their precursors in postmenopausal women with early breast cancer receiving anastrozole. Steroids. 2015;99(Pt A):32–38. doi: 10.1016/j.steroids.2014.08.007.  [DOI](https://doi.org/10.1016/j.steroids.2014.08.007) | [PMC free article](/articles/PMC4339673/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25163006/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Steroids&title=Estrogens%20and%20their%20precursors%20in%20postmenopausal%20women%20with%20early%20breast%20cancer%20receiving%20anastrozole&author=JN%20Ingle&volume=99&issue=Pt%20A&publication_year=2015&pages=32-38&pmid=25163006&doi=10.1016/j.steroids.2014.08.007&)

20. Ingle JN, et al. Selective estrogen receptor modulators and pharmacogenomic variation in ZNF423 regulation of BRCA1 expression: individualized breast cancer prevention. Cancer Discov. 2013;3(7):812–825. doi: 10.1158/2159-8290.CD-13-0038.  [DOI](https://doi.org/10.1158/2159-8290.CD-13-0038) | [PMC free article](/articles/PMC3710533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23764426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&title=Selective%20estrogen%20receptor%20modulators%20and%20pharmacogenomic%20variation%20in%20ZNF423%20regulation%20of%20BRCA1%20expression:%20individualized%20breast%20cancer%20prevention&author=JN%20Ingle&volume=3&issue=7&publication_year=2013&pages=812-825&pmid=23764426&doi=10.1158/2159-8290.CD-13-0038&)

21. Cairns J, et al. The lncRNA MIR2052HG regulates ERα levels and aromatase inhibitor resistance through LMTK3 by recruiting EGR1. Breast Cancer Res. 2019;21(1):47. doi: 10.1186/s13058-019-1130-3.  [DOI](https://doi.org/10.1186/s13058-019-1130-3) | [PMC free article](/articles/PMC6448248/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30944027/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=The%20lncRNA%20MIR2052HG%20regulates%20ER%CE%B1%20levels%20and%20aromatase%20inhibitor%20resistance%20through%20LMTK3%20by%20recruiting%20EGR1&author=J%20Cairns&volume=21&issue=1&publication_year=2019&pages=47&pmid=30944027&doi=10.1186/s13058-019-1130-3&)

22. Macedo LF, Sabnis G, Brodie A. Aromatase inhibitors and breast cancer. Ann N Y Acad Sci. 2009;1155(1):162–173. doi: 10.1111/j.1749-6632.2008.03689.x.  [DOI](https://doi.org/10.1111/j.1749-6632.2008.03689.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19250202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20N%20Y%20Acad%20Sci&title=Aromatase%20inhibitors%20and%20breast%20cancer&author=LF%20Macedo&author=G%20Sabnis&author=A%20Brodie&volume=1155&issue=1&publication_year=2009&pages=162-173&pmid=19250202&doi=10.1111/j.1749-6632.2008.03689.x&)

23. Neven P. The origin of postmenopausal oestrogens. Eur J Cancer. 2002;38 Suppl 6:S29–S30. doi: 10.1016/s0959-8049(02)00275-7.  [DOI](https://doi.org/10.1016/s0959-8049(02)00275-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12409064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=The%20origin%20of%20postmenopausal%20oestrogens&author=P%20Neven&volume=38%20Suppl%206&publication_year=2002&pages=S29-S30&pmid=12409064&doi=10.1016/s0959-8049(02)00275-7&)

24. Tang MR, Wang YX, Guo S, Han SY, Wang D. CSMD1 exhibits antitumor activity in A375 melanoma cells through activation of the Smad pathway. Apoptosis. 2012;17(9):927–937. doi: 10.1007/s10495-012-0727-0.  [DOI](https://doi.org/10.1007/s10495-012-0727-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22538441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Apoptosis&title=CSMD1%20exhibits%20antitumor%20activity%20in%20A375%20melanoma%20cells%20through%20activation%20of%20the%20Smad%20pathway&author=MR%20Tang&author=YX%20Wang&author=S%20Guo&author=SY%20Han&author=D%20Wang&volume=17&issue=9&publication_year=2012&pages=927-937&pmid=22538441&doi=10.1007/s10495-012-0727-0&)

25. Liang N, et al. Steroidogenic factor-1 is required for TGF-beta3-mediated 17beta-estradiol synthesis in mouse ovarian granulosa cells. Endocrinology. 2011;152(8):3213–3225. doi: 10.1210/en.2011-0102.  [DOI](https://doi.org/10.1210/en.2011-0102) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21586554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocrinology&title=Steroidogenic%20factor-1%20is%20required%20for%20TGF-beta3-mediated%2017beta-estradiol%20synthesis%20in%20mouse%20ovarian%20granulosa%20cells&author=N%20Liang&volume=152&issue=8&publication_year=2011&pages=3213-3225&pmid=21586554&doi=10.1210/en.2011-0102&)

26. Powers GL, Ellison-Zelski SJ, Casa AJ, Lee AV, Alarid ET. Proteasome inhibition represses ERalpha gene expression in ER+ cells: a new link between proteasome activity and estrogen signaling in breast cancer. Oncogene. 2010;29(10):1509–1518. doi: 10.1038/onc.2009.434.  [DOI](https://doi.org/10.1038/onc.2009.434) | [PMC free article](/articles/PMC2837136/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19946334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Proteasome%20inhibition%20represses%20ERalpha%20gene%20expression%20in%20ER+%20cells:%20a%20new%20link%20between%20proteasome%20activity%20and%20estrogen%20signaling%20in%20breast%20cancer&author=GL%20Powers&author=SJ%20Ellison-Zelski&author=AJ%20Casa&author=AV%20Lee&author=ET%20Alarid&volume=29&issue=10&publication_year=2010&pages=1509-1518&pmid=19946334&doi=10.1038/onc.2009.434&)

27. Thewes V, et al. The branched-chain amino acid transaminase 1 sustains growth of antiestrogen-resistant and ERα-negative breast cancer. Oncogene. 2017;36(29):4124–4134. doi: 10.1038/onc.2017.32.  [DOI](https://doi.org/10.1038/onc.2017.32) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28319069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=The%20branched-chain%20amino%20acid%20transaminase%201%20sustains%20growth%20of%20antiestrogen-resistant%20and%20ER%CE%B1-negative%20breast%20cancer&author=V%20Thewes&volume=36&issue=29&publication_year=2017&pages=4124-4134&pmid=28319069&doi=10.1038/onc.2017.32&)

28. Goetz MP, et al. Tumor Sequencing and Patient-Derived Xenografts in the Neoadjuvant Treatment of Breast Cancer. J Natl Cancer Inst. 2017;109(7):djw306. doi: 10.1093/jnci/djw306.  [DOI](https://doi.org/10.1093/jnci/djw306) | [PMC free article](/articles/PMC5408989/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28376176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Tumor%20Sequencing%20and%20Patient-Derived%20Xenografts%20in%20the%20Neoadjuvant%20Treatment%20of%20Breast%20Cancer&author=MP%20Goetz&volume=109&issue=7&publication_year=2017&pages=djw306&pmid=28376176&doi=10.1093/jnci/djw306&)

29. Nardone A, De Angelis C, Trivedi MV, Osborne CK, Schiff R. The changing role of ER in endocrine resistance. Breast. 2015;24 Suppl 2:S60–S66. doi: 10.1016/j.breast.2015.07.015.  [DOI](https://doi.org/10.1016/j.breast.2015.07.015) | [PMC free article](/articles/PMC4666002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26271713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast&title=The%20changing%20role%20of%20ER%20in%20endocrine%20resistance&author=A%20Nardone&author=C%20De%20Angelis&author=MV%20Trivedi&author=CK%20Osborne&author=R%20Schiff&volume=24%20Suppl%202&publication_year=2015&pages=S60-S66&pmid=26271713&doi=10.1016/j.breast.2015.07.015&)

30. Robertson JFR, et al. Fulvestrant 500 mg versus anastrozole 1 mg for hormone receptor-positive advanced breast cancer (FALCON): an international, randomised, double-blind, phase 3 trial. Lancet. 2016;388(10063):2997–3005. doi: 10.1016/S0140-6736(16)32389-3.  [DOI](https://doi.org/10.1016/S0140-6736(16)32389-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27908454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Fulvestrant%20500%20mg%20versus%20anastrozole%201%20mg%20for%20hormone%20receptor-positive%20advanced%20breast%20cancer%20(FALCON):%20an%20international,%20randomised,%20double-blind,%20phase%203%20trial&author=JFR%20Robertson&volume=388&issue=10063&publication_year=2016&pages=2997-3005&pmid=27908454&doi=10.1016/S0140-6736(16)32389-3&)

31. Lerebours F, et al. Randomized phase 2 neoadjuvant trial evaluating anastrozole and fulvestrant efficacy for postmenopausal, estrogen receptor-positive, human epidermal growth factor receptor 2-negative breast cancer patients: Results of the UNICANCER CARMINA 02 French trial (UCBG 0609) Cancer. 2016;122(19):3032–3040. doi: 10.1002/cncr.30143.  [DOI](https://doi.org/10.1002/cncr.30143) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27315583/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Randomized%20phase%202%20neoadjuvant%20trial%20evaluating%20anastrozole%20and%20fulvestrant%20efficacy%20for%20postmenopausal,%20estrogen%20receptor-positive,%20human%20epidermal%20growth%20factor%20receptor%202-negative%20breast%20cancer%20patients:%20Results%20of%20the%20UNICANCER%20CARMINA%2002%20French%20trial%20(UCBG%200609)&author=F%20Lerebours&volume=122&issue=19&publication_year=2016&pages=3032-3040&pmid=27315583&doi=10.1002/cncr.30143&)

32. Edavana VK, et al. Potential role of UGT1A4 promoter SNPs in anastrozole pharmacogenomics. Drug Metab Dispos. 2013;41(4):870–877. doi: 10.1124/dmd.112.048157.  [DOI](https://doi.org/10.1124/dmd.112.048157) | [PMC free article](/articles/PMC3608453/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23371966/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Potential%20role%20of%20UGT1A4%20promoter%20SNPs%20in%20anastrozole%20pharmacogenomics&author=VK%20Edavana&volume=41&issue=4&publication_year=2013&pages=870-877&pmid=23371966&doi=10.1124/dmd.112.048157&)

33. Gordân R, et al. Genomic regions flanking E-box binding sites influence DNA binding specificity of bHLH transcription factors through DNA shape. Cell Rep. 2013;3(4):1093–1104. doi: 10.1016/j.celrep.2013.03.014.  [DOI](https://doi.org/10.1016/j.celrep.2013.03.014) | [PMC free article](/articles/PMC3640701/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23562153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&title=Genomic%20regions%20flanking%20E-box%20binding%20sites%20influence%20DNA%20binding%20specificity%20of%20bHLH%20transcription%20factors%20through%20DNA%20shape&author=R%20Gord%C3%A2n&volume=3&issue=4&publication_year=2013&pages=1093-1104&pmid=23562153&doi=10.1016/j.celrep.2013.03.014&)

34. Zhang H, McElrath T, Tong W, Pollard JW. The molecular basis of tamoxifen induction of mouse uterine epithelial cell proliferation. J Endocrinol. 2005;184(1):129–140. doi: 10.1677/joe.1.05987.  [DOI](https://doi.org/10.1677/joe.1.05987) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15642790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Endocrinol&title=The%20molecular%20basis%20of%20tamoxifen%20induction%20of%20mouse%20uterine%20epithelial%20cell%20proliferation&author=H%20Zhang&author=T%20McElrath&author=W%20Tong&author=JW%20Pollard&volume=184&issue=1&publication_year=2005&pages=129-140&pmid=15642790&doi=10.1677/joe.1.05987&)

35. Shanle EK, Xu W. Endocrine disrupting chemicals targeting estrogen receptor signaling: identification and mechanisms of action. Chem Res Toxicol. 2011;24(1):6–19. doi: 10.1021/tx100231n.  [DOI](https://doi.org/10.1021/tx100231n) | [PMC free article](/articles/PMC3119362/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21053929/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Res%20Toxicol&title=Endocrine%20disrupting%20chemicals%20targeting%20estrogen%20receptor%20signaling:%20identification%20and%20mechanisms%20of%20action&author=EK%20Shanle&author=W%20Xu&volume=24&issue=1&publication_year=2011&pages=6-19&pmid=21053929&doi=10.1021/tx100231n&)

36. Robertson JF, et al. A randomized trial to assess the biological activity of short-term (pre-surgical) fulvestrant 500 mg plus anastrozole versus fulvestrant 500 mg alone or anastrozole alone on primary breast cancer. Breast Cancer Res. 2013;15(2):R18. doi: 10.1186/bcr3393.  [DOI](https://doi.org/10.1186/bcr3393) | [PMC free article](/articles/PMC3672778/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23497452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res&title=A%20randomized%20trial%20to%20assess%20the%20biological%20activity%20of%20short-term%20(pre-surgical)%20fulvestrant%20500%20mg%20plus%20anastrozole%20versus%20fulvestrant%20500%20mg%20alone%20or%20anastrozole%20alone%20on%20primary%20breast%20cancer&author=JF%20Robertson&volume=15&issue=2&publication_year=2013&pages=R18&pmid=23497452&doi=10.1186/bcr3393&)

37. Ellis MJ, et al. Lower-dose vs high-dose oral estradiol therapy of hormone receptor-positive, aromatase inhibitor-resistant advanced breast cancer: a phase 2 randomized study. JAMA. 2009;302(7):774–780. doi: 10.1001/jama.2009.1204.  [DOI](https://doi.org/10.1001/jama.2009.1204) | [PMC free article](/articles/PMC3460383/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19690310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Lower-dose%20vs%20high-dose%20oral%20estradiol%20therapy%20of%20hormone%20receptor-positive,%20aromatase%20inhibitor-resistant%20advanced%20breast%20cancer:%20a%20phase%202%20randomized%20study&author=MJ%20Ellis&volume=302&issue=7&publication_year=2009&pages=774-780&pmid=19690310&doi=10.1001/jama.2009.1204&)

38. Osipo C, Gajdos C, Cheng D, Jordan VC. Reversal of tamoxifen resistant breast cancer by low dose estrogen therapy. J Steroid Biochem Mol Biol. 2005;93(2-5):249–256. doi: 10.1016/j.jsbmb.2004.12.005.  [DOI](https://doi.org/10.1016/j.jsbmb.2004.12.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15860267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Steroid%20Biochem%20Mol%20Biol&title=Reversal%20of%20tamoxifen%20resistant%20breast%20cancer%20by%20low%20dose%20estrogen%20therapy&author=C%20Osipo&author=C%20Gajdos&author=D%20Cheng&author=VC%20Jordan&volume=93&issue=2-5&publication_year=2005&pages=249-256&pmid=15860267&doi=10.1016/j.jsbmb.2004.12.005&)

39. Litwin MS, First RA. Maculoerythematous rash after lumbar sympathectomy. N Engl J Med. 1961;265:484–486. doi: 10.1056/NEJM196109072651008.  [DOI](https://doi.org/10.1056/NEJM196109072651008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/13762777/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Maculoerythematous%20rash%20after%20lumbar%20sympathectomy&author=MS%20Litwin&author=RA%20First&volume=265&publication_year=1961&pages=484-486&pmid=13762777&doi=10.1056/NEJM196109072651008&)

40. Goetz MP, et al. MONARCH 3: Abemaciclib As Initial Therapy for Advanced Breast Cancer. J Clin Oncol. 2017;35(32):3638–3646. doi: 10.1200/JCO.2017.75.6155.  [DOI](https://doi.org/10.1200/JCO.2017.75.6155) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28968163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=MONARCH%203:%20Abemaciclib%20As%20Initial%20Therapy%20for%20Advanced%20Breast%20Cancer&author=MP%20Goetz&volume=35&issue=32&publication_year=2017&pages=3638-3646&pmid=28968163&doi=10.1200/JCO.2017.75.6155&)

41. Dickler MN, et al. MONARCH 1, A Phase II Study of Abemaciclib, a CDK4 and CDK6 Inhibitor, as a Single Agent, in Patients with Refractory HR+/HER2- Metastatic Breast Cancer. Clin Cancer Res. 2017;23(17):5218–5224. doi: 10.1158/1078-0432.CCR-17-0754.  [DOI](https://doi.org/10.1158/1078-0432.CCR-17-0754) | [PMC free article](/articles/PMC5581697/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28533223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=MONARCH%201,%20A%20Phase%20II%20Study%20of%20Abemaciclib,%20a%20CDK4%20and%20CDK6%20Inhibitor,%20as%20a%20Single%20Agent,%20in%20Patients%20with%20Refractory%20HR+/HER2-%20Metastatic%20Breast%20Cancer&author=MN%20Dickler&volume=23&issue=17&publication_year=2017&pages=5218-5224&pmid=28533223&doi=10.1158/1078-0432.CCR-17-0754&)
