# CXCR4 polymorphism predicts progression-free survival in metastatic colorectal cancer patients treated with first-line bevacizumab-based chemotherapy

## Metadata
**Authors:** S Matsusaka, S Cao, DL Hanna, Y Sunakawa, M Ueno, N Mizunuma, W Zhang, D Yang, Y Ning, S Stintzing, A Sebio, S Stremitzer, S Yamauchi, A Parekh, S Okazaki, MD Berger, R El-Khoueiry, A Mendez, W Ichikawa, F Loupakis, H-J Lenz
**Journal:** The pharmacogenomics journal
**Date:** 2016 Aug 9
**DOI:** [10.1038/tpj.2016.59](https://doi.org/10.1038/tpj.2016.59)
**PMID:** 27503580
**PMCID:** PMC7496215
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7496215/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7496215/pdf/nihms-1623803.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7496215/pdf/nihms-1623803.pdf)

## Abstract

We analyzed associations between CXCR4/CXCL12 single-nucleotide polymorphisms and outcomes in metastatic colorectal cancer (mCRC) patients who underwent first-line bevacizumab-based chemotherapy. A total of 874 patients were included in this study: 144 treated with bevacizumab and FOLFOX or XELOX (training cohort), 653 treated with bevacizumab and FOLFIRI or FOLFOXIRI (validation cohort A or B) and 77 treated with cetuximab- and oxaliplatin-based regimens (control cohort). One CXCR4 polymorphism (rs2228014) and two CXCL12 polymorphisms (rs1801157 and rs3740085) were analyzed by PCR-based direct sequencing. Patients with a C/C genotype had a prolonged progression-free survival (PFS) compared with those with any T allele (P = 0.030) in the training cohort. Similarly, patients with the C/C genotype had a superior PFS in the validation cohorts, but not in the control cohort. Our findings suggest that a common genetic variant, CXCR4 rs2228014, could predict PFS and may guide therapeutic decisions in mCRC patients receiving first-line bevacizumab-based chemotherapy.

## INTRODUCTION

Bevacizumab-based chemotherapy has improved the survival of patients with metastatic colorectal cancer (mCRC) in first- and subsequent-line settings. However, not all patients benefit from therapy and acquired resistance remains a problem in those who initially respond. Although resistance mechanisms are beginning to be understood, predictive and prognostic markers are needed to guide clinical decision making.

The chemokine receptor, CXCR4, is a 352-amino acid rhodopsinlike GPCR (G-protein couple receptor) that selectively binds SDF-1 (stromal cell-derived factor 1), also known as CXCL12.^1^ Prompted by hypoxic stress and vascular endothelial growth factor (VEGF) stimulation, the CXCL12/CXCR4 axis promotes angiogenesis and the migration of tumor and endothelial cells into the metastatic niches of many cancers.^2,3^ In many tumor models, these molecules permit cancer cell survival in the tumor microenvironment and the recruitment of hematopoietic and endothelial progenitors for neovascularization.^4,5^ In addition to VEGF-mediated angiogenesis, several studies suggest that the CXCR4/CXCL12 axis may contribute to functional vascular establishment and that the antiangiogenic effects of CXCR4 blockade are related to a reduction in tumor endothelium, independent of VEGF inhibition.^5,6^

CXCR4 expression has been associated with tumor recurrence, metastasis and survival in colorectal cancer.^7,8^ *CXCR4* rs2228014 single-nucleotide polymorphisms (SNPs) have also been linked to inflammatory bowel disease, as patients with a T allele have a predilection for developing ulcerative colitis.^9^ However, to date, no study has examined the effect of *CXCR4* genetic variants on clinical outcomes in colorectal cancer. The aim of this study was to investigate the potential of *CXCR4/CXCL12* SNPs as surrogate markers of clinical outcomes in mCRC patients receiving first-line bevacizumab-based chemotherapy.

## MATERIALS AND METHODS

### Patients and treatment protocols

The training set consisted of 144 mCRC patients treated with first-line FOLFOX or XELOX plus bevacizumab at the Cancer Institute Hospital in Japan. Within this cohort, 64% patients received FOLFOX, with bevacizumab 5 mg kg^−1^ administered intravenously (day 1), followed by oxaliplatin 85 mg m^−2^ administered with folinic acid (leucovorin) 200 mg m^−2^ intravenously, 5-fluorouracil 400 mg m^−2^ bolus infusion and 5-fluorouracil 2400 mg m^−2^ administered as a 48 h continuous infusion. This regimen was repeated at 2-week intervals until the time of disease progression, intolerable toxicities or early withdrawal. The remaining 36% patients in this cohort received XELOX, with bevacizumab 10 mg kg^−1^ administered intravenously (day 1), followed by oxaliplatin 150 mg m^−2^ intravenously and oral capecitabine (1000 mg m^−2^) given on days 1–14. This regimen was repeated at 3-week intervals until the time of disease progression, intolerable toxicities or early withdrawal. Responses were measured by intravenous contrast-enhanced computed tomography scans every 8 weeks according to Response Evaluation Criteria in Solid Tumors (RECIST) 1.1.

Validation cohort A consisted of 424 mCRC patients treated with first-line FOLFIRI plus bevacizumab in two studies: 204 enrolled in arm A of the phase III TRIBE trial^10^ and 220 from the prospective pharmacogenomic study, PROVETTA ([NCT01363739](https://clinicaltrials.gov/ct2/show/NCT01363739)). Validation cohort B consisted of 229 mCRC patients treated with first-line FOLFOXIRI plus bevacizumab in TRIBE arm B.^10^ These two validation cohorts comprised the validation set. The standard inclusion and exclusion criteria of these trials have previously been described.^10^ Prior adjuvant oxaliplatin treatment was allowed in both trials if more than 12 months had elapsed between the end of treatment and date of study enrollment. All patients in validation cohort A received bevacizumab 5 mg kg^−1^ intravenously (day 1), followed by irinotecan (180 mg m^−2^ in TRIBE arm A and 165 mg m^−2^ in PROVETTA) administered with folinic acid (leucovorin) 200 mg m^−2^ intravenously, followed by 5-fluorouracil 400 mg m^−2^ bolus infusion and 5-fluorouracil 2400 mg m^−2^ as a 48 h continuous infusion. All patients in validation cohort B received bevacizumab 5 mg kg^−1^ intravenously (day 1), followed by irinotecan (165 mg m^−2^ in TRIBE arm B) and 85 mg m^−2^ oxaliplatin administered with folinic acid (leucovorin) 200 mg m^−2^ intravenously, followed by 5-fluorouracil 3200 mg m^−2^ as a 48 h continuous infusion. The regimen was repeated at 2-week intervals. Responses were measured by intravenous contrast-enhanced computed tomography scans every 8 weeks according to RECIST v1.0.

The control set consisted of 77 mCRC patients treated with first-line FOLFOX or SOX plus cetuximab in two clinical trials (JACCRO-CC05 and JACCRO-CC06).^11,12^ Standard inclusion criteria were applied and have previously been described.^11,12^ All patients received cetuximab 400 mg kg^−1^ intravenously (day 1), followed by oxaliplatin 85 mg m^−2^ administered with folinic acid (leucovorin) 200 mg m^−2^ intravenously. Within this cohort, 36% of patients received FOLFOX, with 5-fluorouracil 400 mg m^−2^ given as a bolus application, followed by 5-fluorouracil 2400 mg m^−2^ as a 48 h continuous infusion. The remaining patients received SOX, with S-1 given orally on days 1–14. FOLFOX/SOX plus cetuximab regimen was administered unless there was evidence of progression, intolerable toxicities or early withdrawal. Responses were measured by intravenous contrast-enhanced computed tomography scans every 8 weeks according to RECIST v1.1.

This study was approved by the institutional review boards of each institute, and all patients signed an informed consent for the analysis of molecular correlates.

### Candidate polymorphisms

Polymorphisms were only chosen for investigation if the minor allele frequency exceeded 5% in Asians and Caucasians according to the ENSEMBL database ([http://www.ensembl.org/index.html](http://www.ensembl.org/index.html)) and if the functional or predicted functional relevance of gene transcription or protein expression was previously described. Functional significance was predicted based on information provided by the National Institute of Environmental Health Science SNP Function Prediction, Queen’s University F-SNP and the location of the SNP in the protein-coding region of the gene ([http://snpinfo.niehs.nih.gov/snpinfo/snptag.htm](http://snpinfo.niehs.nih.gov/snpinfo/snptag.htm)).

### Genotyping

Genomic DNA was extracted from formalin-fixed, paraffin-embedded tissues of 221 patients from the training and control cohorts and from the peripheral whole blood of 653 patients from validation cohorts A and B using the QIAmp Kit (Qiagen, Valencia, CA, USA) according to the manufacturer's protocol ([www.qiagen.com](http://www.qiagen.com/)). PCR-based direct DNA sequence analysis using ABI 3100A Capillary Genetic Analyzer and Sequencing Scanner v1.0 (Applied Biosystems, Yokohama, Kanagawa, Japan) was performed for genotyping the SNPs. Extracted DNA was amplified using the following primer set: forward: 5'-ACCACCTTTTCAGCCAACAGF-3' and reverse: 5'-CTTCCTATGCAAGGCAGTCC-3' for rs2228014 in *CXCR4*; forward: 5'-ATGAGGGCTGGGTCTCACT-3' and reverse: 5'-AGGCTTCTCTCTGTGGGATG-3' for rs1801157 in *CXCL12*; forward: 5'-GACATGTGAGGGAGCACTGA-3' and reverse: 5'-TGGTCCTTTGGGGTGAACTA-3' for rs3740085 in *CXCL12*.

For quality control purposes, a random selection of 10% of the samples were reexamined for each polymorphism and the genotype concordance rate was 100%. The investigator analyzed the sequencing data using the ABI Sequencing Scanner v1.0 (Applied Biosystems, Life Technologies, Grand Island, NY, USA) and analyzed the SNPs blindly to the clinical data set.

### Statistical analysis

The primary end point of this retrospective analysis was progression-free survival (PFS), and the secondary end points were tumor response and overall survival (OS). PFS was defined as the interval from the first day of starting first-line chemotherapy to the first day of documented disease progression or death. If progression or death was not observed, PFS was censored on the day of the last computed tomography scan. OS was defined as the period from starting therapy to the date of death or censored on the date of last contact if alive. Tumor responses based on RECIST were grouped into responders, including complete or partial response, and nonresponders, including stable or progressive disease. The differences in baseline patient characteristics between the four cohorts were examined using X^2^ test or Kruskal–Wallis test when appropriate. Allelic distribution of polymorphisms by ethnicity was tested for deviation from Hardy–Weinberg equilibrium using the exact test.

The training cohort consisted of 144 patients (109 events); therefore, there was 80% power to detect an association between a SNP and PFS with minimum hazard ratio (HR) from 1.72 to 2.21 using a two-sided 0.05level log-rank test. We assumed that the minor allele frequency varied from 0.07 to 0.29, and the dominant model was used. The minimum detectable HR using the same test was 1.56, 1.88 and 2.96 for validation cohorts A and B(*N* = 424; *N* = 229), respectively, when assuming a minor allele frequency of 0.07.

The associations between SNPs and PFS, OS and tumor response were examined using Kaplan–Meier curves, log-rank test and Fisher’s exact test. Cox proportional hazards regression model with stratification factors was fitted to reevaluate the association between SNPs and PFS and OS considering imbalances in the distributions of baseline characteristics among cohorts. The baseline demographic and clinical characteristics that remained significantly associated with end points in the multivariable analysis (*P* < 0.1) were included in the final model.

SAS 9.4 (SAS Institute, Cary, NC, USA) was used to perform all analyses. All tests were two sided at a significance level of 0.05.

## RESULTS

### Patient and tumor characteristics

Clinicopathologic characteristics for the training, validation and control cohorts are summarized in [Table 1](#T1). The median follow-up periods were 5.2 years in the training cohort, 3.7 years in validation cohort A, 3.9 years in validation cohort B and 2.1 years in the control cohort. The median PFS and OS in each cohort were as follows: 1.3 years and 3.2 years (training cohort), 10.4 months and 2.3 years (validation cohort A), 1.0 year and 2.5 years (validation cohort B) and 10.0 months and 2.8 years (control cohort). Associations between baseline characteristics and clinical outcomes are presented for each cohort ([Supplementary Tables 1](#SD1)). All genotype frequencies for candidate polymorphisms analyzed were within Hardy–Weinberg equilibrium (*P*>0.05) for each ethnic/race group in each cohort.

### Table 1.

|   | Training cohort (n = 144) |  | Validation cohort A (n = 424) |  | Validation cohort B (n = 229) |  | Control cohort (n = 77) |  | P-valuea |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | n | % | n | % | n | % | n | % |   |
| Gender |   |   |   |   |   |   |   |   |   |
| Male | 81 | 56.3 | 252 | 59.4 | 136 | 59.4 | 44 | 57.1 | 0.90 |
| Female | 63 | 43.8 | 172 | 40.6 | 93 | 40.6 | 33 | 42.9 |   |
| Age (year) |   |   |   |   |   |   |   |   |   |
| Median (range) | 61 (27–75) |  | 62 (25–81) |  | 61 (29–75) |  | 63 (39–79) |  | 0.048 |
| < 55 | 37 | 25.7 | 112 | 26.4 | 70 | 30.6 | 13 | 16.9 |   |
| 55–65 | 59 | 41.0 | 150 | 35.4 | 79 | 34.5 | 31 | 40.3 | 0.29 |
| ⩾ 65 | 48 | 33.3 | 162 | 38.2 | 80 | 34.9 | 33 | 42.9 |   |
| Tumor site |   |   |   |   |   |   |   |   |   |
| Right-sided colon | 48 | 33.3 | 107 | 25.2 | 78 | 34.1 | 11 | 14.3 | 0.002 |
| Left-sided colon | 96 | 66.7 | 303 | 71.5 | 140 | 61.1 | 64 | 83.1 |   |
| Unknownb | 0 |   | 14 | 3.3 | 11 | 4.8 | 2 | 2.6 |   |
| Liver metastasis |   |   |   |   |   |   |   |   |   |
| Yes | 81 | 56.3 | 314 | 74.1 | 80 | 34.9 | 48 | 62.3 | < 0.001 |
| No | 63 | 43.8 | 110 | 25.9 | 149 | 65.1 | 29 | 37.7 |   |
| Lung metastasis |   |   |   |   |   |   |   |   |   |
| Yes | 62 | 43.1 | 135 | 31.8 | NA |   | 27 | 35.1 | 0.051 |
| No | 82 | 56.9 | 289 | 68.2 | NA |   | 50 | 64.9 |   |
| Lymph nodes |   |   |   |   |   |   |   |   |   |
| Yes | 76 | 52.8 | 145 | 34.2 | NA |   | 40 | 51.9 | < 0.001 |
| No | 68 | 47.2 | 279 | 64.8 | NA |   | 37 | 48.1 |   |
| Number of metastatic sites |   |   |   |   |   |   |   |   |   |
| < 2 | 56 | 38.9 | 194 | 45.8 | 100 | 43.7 | 33 | 42.9 | < 0.001 |
| ⩾ 2 | 88 | 61.1 | 230 | 54.2 | 129 | 56.3 | 44 | 57.1 |   |
| Performance Status |   |   |   |   |   |   |   |   |   |
| ECOG 0 | 138 | 95.8 | 357 | 84.2 | 204 | 89.1 | 69 | 89.6 | 0.002 |
| ECOG 1–2 | 6 | 4.2 | 67 | 15.8 | 25 | 10.9 | 8 | 10.4 |   |
| Time to metastasis |   |   |   |   |   |   |   |   |   |
| Synchronous | 59 | 41.0 | 311 | 73.3 | 180 | 78.6 | 59 | 76.6 | < 0.001 |
| Metachronous | 85 | 59.0 | 113 | 26.7 | 49 | 21.4 | 18 | 23.4 |   |

Table 1 Caption: Baseline clinical characteristics of patients in the training, validation and control cohorts

### Associations between CXCR4, CXCL12 polymorphisms and clinical outcomes in the training cohort

In the training cohort, genotyping for *CXCR4* rs2228014 was successful in 131 (91%) patients. In 13 (9%) patients, genotyping was not successful because of limited quantity or quality of extracted genomic DNA. In univariate analysis, *CXCR4* rs2228014 genotype was significantly associated with PFS and OS. Patients harboring a T allele had a shorter median PFS (13.9 vs 16.3 months, HR 1.67, *P* = 0.030) and median OS (29.5 vs 41.1 months, HR 1.65, *P* = 0.038) compared with those with the C/C genotype ([Table 2](#T2) and [Figure 1a](#F1)). In multivariable analysis, this *CXCR4* rs2228014 polymorphism remained significantly associated with PFS, but not with OS. The two polymorphisms in the *CXCL12* gene, rs1801157 and rs3740085, were not associated with clinical outcomes. There was no significant association between any candidate polymorphisms and tumor response.

### Table 2.

|   |   | Tumor response |  |  | Progression-free survival |  |  |  |  | Overall survival |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | N | CR+PR | SD+PD | P-value* | Median, months (95% CI) | HR (95% CI)† | P-value* | HR (95% CI)‡ | P-value* | Median, months (95%CI) | HR (95% CI)† | P-value* | HR (95% CI)‡ | P-value* |
| CXCL12 rs1801157 |   |   |   | 0.49 |   |   | 0.11 |   | 0.16 |   |   | 0.56 |   | 0.27 |
| C/C | 66 | 51 (80%) | 13 (20%) |   | 16.8 (13.3–20.6) | 1 (Reference) |   | 1 (Reference) |   | 41.1 (32.2, 46.5) | 1 (Reference) |   | 1 (Reference) |   |
| C/Ta | 57 | 41 (73%) | 15 (27%) |   | 14.9 (12.2–17.4) | 1.36 (0.92–2.01) |   | 1.34 (0.89–2.03) |   | 36.1 (26.6–47.8) | 1.12 (0.75–1.68) |   | 1.27 (0.83–1.93) |   |
| T/Ta | 10 | 9 (90%) | 1 (10%) |   |   |   |   |   |   |   |   |   |   |   |
| CXCL12 rs3740085 |   |   |   | 0.86 |   |   | 0.37 |   | 0.41 |   |   | 0.86 |   | 0.90 |
| C/C | 97 | 72 (77%) | 22 (23%) |   | 16.9 (14.6–19.0) | 1 (Reference) |   | 1 (Reference) |   | 39.9 (30.8–44.7) | 1 (Reference) |   | 1 (Reference) |   |
| C/Gb | 34 | 24 (73%) | 9 (27%) |   | 13.3 (9.5–17.4) | 1.21 (0.79–1.86) |   | 1.20 (0.77–1.87) |   | 39.8 (23.1–47.8) | 0.96 (0.61–1.51) |   | 0.97 (0.61–1.54) |   |
| G/Gb | 5 | 4 (80%) | 1 (20%) |   |   |   |   |   |   |   |   |   |   |   |
| CXCR4 rs2228014 |   |   |   | 0.69 |   |   | 0.030 |   | 0.029 |   |   | 0.038 |   | 0.14 |
| C/C | 109 | 78 (74%) | 27 (26%) |   | 16.3 (13.4–18.9) | 1 (Reference) |   | 1 (Reference) |   | 41.1 (32.2–45.9) | 1 (Reference) |   | 1 (Reference) |   |
| T/Cc | 21 | 17 (81%) | 4 (19%) |   | 13.9 (9.1–18.1) | 1.67 (1.03–2.72) |   | 1.77 (1.06–2.94) |   | 29.5 (19.9–43.8) | 1.65 (1.02–2.69) |   | 1.48 (0.88–2.47) |   |
| T/Tc | 1 | 1 (100%) | 0 |   |   |   |   |   |   |   |   |   |   |   |

Table 2 Caption: Associations between CXCR4/CXCR12 SNPs and clinical outcomes in the training cohort

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/94ec/7496215/b2be84d229c8/nihms-1623803-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7496215_nihms-1623803-f0001.jpg)

Comparison of clinical outcomes by CXCR4 rs2228014 genotype in the training and validation cohorts. Progression-free survival probability in (a) training cohort, (b) validation cohort A and (c) validation cohort B.

### Associations between CXCR4 polymorphisms and clinical outcomes in the validation cohorts

In validation cohort A, genotyping for *CXCR4* rs2228014 was successful in 397 patients (94%). In 27 (6%) patients, genotyping was not successful because of a limited quantity or quality of extracted genomic DNA. The *CXCR4* rs2228014 T allele was associated with a significantly shorter PFS of 9.6 months as compared with 10.5 months for the C/C genotype (HR 1.40, *P* = 0.035; [Figure 1b](#F1)). This association did not maintain significance in multivariable analysis.

In validation cohort B, genotyping for the *CXCR4* rs2228014 was successful in all 229 patients. The T allele of *CXCR4* rs2228014 was associated with a significantly shorter PFS. Patients with a T allele had a median PFS of 9.0 months compared with those with the C/C genotype who had a median PFS of 12.4 months (HR 2.08, *P* < 0.001; [Figure 1c](#F1)). This association remained significant in multivariable analysis (HR 1.87, *P* = 0.009; [Table 2](#T2)). There were no other significant associations between the candidate polymorphisms and OS or tumor response in this cohort.

### Associations between gene polymorphisms and clinical outcomes in the control cohort

In the control cohort, genotyping for the *CXCR4* rs2228014 was successful in 76 patients (99%). Genotyping was not successful in 1 patient (1%) because of a limited quantity and quality of extracted genomic DNA. There were no significant associations between the *CXCR4* rs2228014 and clinical outcomes ([Table 3](#T3)).

### Table 3.

|   |   | Tumor response |  |  |  | Progression-free survival |  |  |  | Overall survival |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | N | CR+PR | PD+SD | P-value* | Median, months (95%CI) | HR (95%CI)† | P-value* | HR (95%CI)‡ | P-value* | Median, months (95%CI) | HR (95%CI)† | P-value* | HR (95%CI)‡ | P-value* |
| Validation cohort A (FOLFIRI+Bev) |  |  |   |   |   |   |   |   |   |   |   |   |   |   |
| CXCR4 rs2228014 |   |   |   | 1.00 |   |   | 0.035 |   | 0.19 |   |   | 0.077 |   | 0.55 |
| C/C | 349 | 199 (58%) | 145 (42%) |   | 10.5 (9.8–11.0) | 1 (Reference) |   | 1 (Reference) |   | 28.8 (25.5, 32.6) | 1 (Reference) |   | 1 (Reference) |   |
| T/Ca | 44 | 25 (58%) | 18 (42%) |   | 9.6 (7.5–11.4) | 1.40 (1.02–1.93) |   | 1.25 (0.90–1.73) |   | 23.2 (16.7–30.8) | 1.37 (0.96–1.95) |   | 1.12 (0.77–1.64) |   |
| T/Ta | 4 | 2 (50%) | 2 (50%) |   |   |   |   |   |   |   |   |   |   |   |
| Validation cohort B (FOLFOXIRI+Bev) |  |  |   |   |   |   |   |   |   |   |   |   |   |   |
| CXCR4 rs2228014 |   |   |   | 0.14 |   |   | < 0.001 |   | 0.009 |   |   | 0.28 |   | 0.97 |
| C/C | 198 | 133 (69%) | 59 (31%) |   | 12.4 (11.0–13.5) | 1 (Reference) |   | 1 (Reference) |   | 30.9 (26.7–35.0) | 1 (Reference) |   | 1 (Reference) |   |
| T/Cb | 30 | 16 (55%) | 13 (45%) |   | 9.0 (5.6–11.6) | 2.08 (1.32–3.27) |   | 1.87 (1.17–2.99) |   | 18.2 (13.1–37.1) | 1.28 (0.81–2.01) |   | 1.01 (0.63–1.62) |   |
| T/Tb | 1 | 0 | 0 |   |   |   |   |   |   |   |   |   |   |   |

Table 3 Caption: Associations between CXCR4 SNPs and clinical outcomes in the validation cohorts

Effect of primary tumor location on associations between *CXCR4* rs2228014 and clinical outcomes in the training and validation cohorts

We have previously demonstrated that benefit from bevacizumab is independent of primary tumor location.^13^ We explored whether there were unique associations between *CXCR4* rs2228014 and outcomes when stratified by primary tumor location.

In the left-sided tumor subgroup, *CXCR4* rs2228014 maintained significant associations with PFS and OS in the training cohort, and with PFS in validation cohort B. In the training cohort, patients with a T allele had a trend toward inferior PFS and significantly shorter OS compared with those with the C/C genotype in univariate analysis ([Table 4](#T4)). In multivariable analysis, *CXCR4* rs2228014 genotype was significantly associated with PFS but not with OS. In validation cohort B, patients with a T allele had a significantly shorter PFS compared with those with the C/C genotype in univariate analysis ([Table 4](#T4)). In multivariable analysis, *CXCR4* rs2228014 remained significantly associated with PFS. However, there was no significant association between *CXCR4* rs2228014 and outcomes in the left-sided tumor subgroup of validation cohort A.

### Table 4.

|   |   | Tumor response |  |  | Progression-free survival |  |  |  |  | Overall survival |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | N | CR+PR | SD+PD | P-value* | Median, months (95%CI) | HR (95% CI)† | P-value* | HR (95% CI)‡ | P-value* | Median, months (95%CI) | HR (95% CI)† | P-value* | HR (95% CI)‡ | P-value* |
| Left-sided colon |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| Training cohort (FOLFOX+Bev) |  |  |   |   |   |   |   |   |   |   |   |   |   |   |
| CXCR4 rs2228014 |   |   |   | 0.17 |   |   | 0.054 |   | 0.049 |   |   | 0.017 |   | 0.13 |
| C/C | 70 | 48 (73%) | 18 (27%) |   | 17.0 (13.3–20.1) | 1 (Reference) |   | 1 (Reference) |   | 44.1 (36.1, 50.8) | 1 (Reference) |   | 1 (Reference) |   |
| T/C | 15 | 14 (93%) | 1 (7%) |   | 12.6 (8.9–18.1) | 1.74 (0.96–3.18) |   | 1.88 (1.00–3.54) |   | 36.1 (16.8–43.8) | 1.98 (1.10–3.57) |   | 1.63 (0.87–3.06) |   |
| Validation cohort A (FOLFIRI+Bev) |  |  |   |   |   |   |   |   |   |   |   |   |   |   |
| CXCR4 rs2228014 |   |   |   | 0.60 |   |   | 0.25 |   | 0.36 |   |   | 0.83 |   | 0.54 |
| C/C | 252 | 152 (60%) | 100 (40%) |   | 10.9 (10.1–12.1) | 1 (Reference) |   | 1 (Reference) |   | 32.6 (29.1–37.8) | 1 (Reference) |   | 1 (Reference) |   |
| T/Ca | 26 | 18 (72%) | 7 (28%) |   | 11.1 (8.4–12.5) | 1.27 (0.84–1.93) |   | 1.22 (0.80–1.86) |   | 33.1 (18.4–54.2) | 1.05 (0.65–1.72) |   | 1.17 (0.71–1.91) |   |
| T/Ta | 3 | 2 (67%) | 1 (33%) |   |   |   |   |   |   |   |   |   |   |   |
| Validation cohort B (FOLFOXIRI+Bev) |  |  |   |   |   |   |   |   |   |   |   |   |   |   |
| CXCR4 rs2228014 |   |   |   | 0.77 |   |   | 0.004 |   | 0.008 |   |   | 0.79 |   | 0.57 |
| C/C | 125 | 85 (70%) | 36 (30%) |   | 12.7 (11.0–15.0) | 1 (Reference) |   | 1 (Reference) |   | 34.4 (28.5–38.9) | 1 (Reference) |   | 1 (Reference) |   |
| T/C | 15 | 10 (67%) | 5 (33%) |   | 9.4 (4.1–14.1+) | 2.58 (1.24–5.36) |   | 2.85 (1.32–6.16) |   | 20.6 (12.7–63.5+) | 0.90 (0.44–1.87) |   | 0.80 (0.36–1.74) |   |
| Right-sided colon |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| Training cohort (FOLFOX+Bev) |  |  |   |   |   |   |   |   |   |   |   |   |   |   |
| CXCR4 rs2228014 |   |   |   | 0.50 |   |   | 0.36 |   | 0.72 |   |   | 0.84 |   | 0.99 |
| C/C | 39 | 30 (77%) | 9 (23%) |   | 15.3 (12.5–18.9) | 1 (Reference) |   | 1 (Reference) |   | 30.5 (24.9–45.6) | 1 (Reference) |   | 1 (Reference) |   |
| T/Cb | 6 | 3 (50%) | 3 (50%) |   | 14.9 (6.1–19.4) | 1.46 (0.63–3.36) |   | 1.22 (0.40–3.68) |   | 29.5 (18.6–65.8+) | 1.09 (0.45–2.65) |   | 0.99 (0.31–3.22) |   |
| T/Tb | 1 | 1 (100%) | 0 |   |   |   |   |   |   |   |   |   |   |   |
| Validation cohort A (FOLFIRI+Bev) |  |  |   |   |   |   |   |   |   |   |   |   |   |   |
| CXCR4 rs2228014 |   |   |   | 0.67 |   |   | 0.30 |   | 0.55 |   |   | 0.049 |   | 0.31 |
| C/C | 87 | 45 (54%) | 38 (46%) |   | 9.8 (8.7–10.3) | 1 (Reference) |   | 1 (Reference) |   | 21.6 (18.8–26.3) | 1 (Reference) |   | 1 (Reference) |   |
| T/Cc | 14 | 7 (50%) | 7 (50%) |   | 7.8 (2.5–10.4) | 1.34 (0.77–2.34) |   | 1.20 (0.66–2.20) |   | 16.7 (8.0–19.8) | 1.76 (0.98–3.17) |   | 1.42 (0.72–2.80) |   |
| T/Tc | 1 | 0 | 1 (100%) |   |   |   |   |   |   |   |   |   |   |   |
| Validation cohort B (FOLFOXIRI+Bev) |  |  |   |   |   |   |   |   |   |   |   |   |   |   |
| CXCR4 rs2228014 |   |   |   | 0.12 |   |   | 0.14 |   | 0.54 |   |   | 0.37 |   | 0.79 |
| C/C | 64 | 40 (65%) | 22 (35%) |   | 11.3 (9.2–13.7) | 1 (Reference) |   | 1 (Reference) |   | 25.8 (20.4–31.3) | 1 (Reference) |   | 1 (Reference) |   |
| T/Cd | 14 | 5 (38%) | 8 (62%) |   | 7.9 (4.1–14.8) | 1.59 (0.83–3.05) |   | 1.25 (0.62–2.53) |   | 18.1 (8.1–37.1) | 1.34 (0.71–2.52) |   | 1.09 (0.57–2.09) |   |
| T/Td | 1 |   |   |   |   |   |   |   |   |   |   |   |   |   |

Table 4 Caption: Association between CXCR4 SNPs and clinical outcomes stratified by primary tumor location

In the right-sided CRC subgroup, patients with a T allele had a significantly shorter OS compared with those with the C/C genotype in univariate analysis in validation cohort A. However, there was no association with OS in multivariable analysis. There were no other significant associations between the *CXCR4* rs2228014 polymorphism and outcomes within the right-sided tumor subgroups in any of the cohorts ([Table 4](#T4)).

## DISCUSSION

Bevacizumab has become an integral component in the management of mCRC patients, but not all patients benefit from treatment and all patients inevitably develop resistance. Though numerous markers have been proposed and evaluated, none have been validated to predict those most likely to benefit from bevacizumab. The CXCR4/CXCL12 chemokine axis plays an important role in intestinal homeostasis, inflammation as well as physiologic and malignant vasculogenesis^14^ through both VEGF-dependent and -independent mechanisms. Furthermore, CXCR4 polymorphisms have demonstrated prognostic value in several cancer types.^15–17^ However, the role of CXCR4 genetic variants in mCRC has not been explored. To our knowledge, our study is the first to demonstrate a significant association between CXCR4 rs2228014 polymorphisms and PFS in mCRC patients receiving first-line chemotherapy with bevacizumab.

Hypoxia provides a crucial selective pressure for tumor cells destined for metastatic seeding, and CXCR4 is a key regulator of hypoxia-associated pathways.^18^ Under physiological conditions, the tumor suppressor, von Hippel–Lindau, blunts CXCR4 expression by degrading its transcription factor, hypoxia-inducible factor-1α (HIF1α). Conversely, in hypoxic states, tumor cells at the invasive front activate HIF1α transcription and translation that upregulates CXCR4, thereby promoting angiogenesis and metastasis ([Figure 2](#F2)).^19,20^ It follows that tumors under greater hypoxic stress may exhibit increased sensitivity to bevacizumab. Indeed, the minor allele of *HIF1α* rs11549465 C> T has been associated with higher HIF1α protein expression and more aggressive disease in colorectal cancer,^21^ but also increased PFS in mCRC patients treated with bevacizumab-based therapy.^22^ Based upon these observations, the C/C variant of *CXCR4* rs2228014 might conceivably predict prolonged PFS in mCRC patients treated with bevacizumab-based chemotherapy by serving as a surrogate for HIF pathway activation.

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/94ec/7496215/c319dab5b205/nihms-1623803-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7496215_nihms-1623803-f0002.jpg)

Hypoxia-inducible factor-1α (HIF-1α) is activated as a result of hypoxia induced by bevacizumab therapy. HIF-1α then upregulates CXCR4 expression that, along with interactions between CXCR4 and its ligand CXCL12, plays an important role in angiogenesis and cancer cell migration.

The associations between *CXCR4* rs2228014, CXCR4 expression and clinical outcomes warrant further exploration. With regard to CXCR4 expression, Xu *et al*.^23^ have shown that bevacizumab monotherapy induces *CXCR4* intratumoral gene expression in rectal cancer patients. We have previously demonstrated that an increased number of baseline CXCR4-positive circulating endothelial cells are associated with shorter PFS in mCRC patients receiving FOLFOX plus bevacizumab.^24^ The coding region for *CXCR4* is on chromosome 2q2,^25^ and rs2228014 is a silent point mutation resulting in the substitution of cytosine to thymine at codon 138.^26^ The function of this polymorphism remains unclear; however, the *CXCR4* rs2228014 is a tag SNP that may exert functional effects through linked polymorphisms at other loci. Okuyama Kishima *et al*.^27^ reported that the rs2228014 genetic variant does not alter *CXCR4* mRNA or protein expression. However, CXCR4 mRNA and protein levels were found to be significantly higher in tumor relative to normal tissue, further supporting its role in mediating carcinogenesis.^27^ This genetic variant may affect the establishment of tumor endothelium, independent of VEGF inhibition and *CXCR4* expression levels in endothelial cells. Further preclinical studies are needed to delineate the precise mechanisms mediating its effects on hypoxia, tumor progression and response to antiangiogenic therapy.

The influence of primary tumor site on bevacizumab efficacy is complex and mediated by multiple factors. In a recent study, Loupakis *et al*.^13^ reported that bevacizumab efficacy was independent of primary tumor location, with no significant site-specific differences detected in the intratumoral mRNA expression of VEGF ligands or receptors. However, right-sided cancers were found to have higher *ERCC1* mRNA levels,^13^ and are also known to have a higher frequency of *BRAF* mutations, as well as a hypermethylated and hypermutated phenotype.^28^ Studies in other cancers have shown that *BRAF*-mutated tumors have significantly higher *CXCR4* mRNA and protein expression levels compared with their nonmutated counterparts.^29,30^ Taken together, this suggests that *CXCR4* mRNA and protein expression in right-sided colorectal cancers may be driven mainly by *BRAF* mutation or hypermethylation status, whereas CXCR4 induction by bevacizumab therapy plays a more prominent role in left-sided tumors. Given the relative rarity of *BRAF*-mutated CRC, it may be difficult to detect a significant influence of *CXCR4* polymorphisms on bevacizumab efficacy in patients with right-sided tumors, especially in those receiving an oxaliplatin-based regimen. Notably, our results showed that the effect of *CXCR4* rs2228014 on outcomes was dependent on the cytotoxic chemotherapy regimen administered and primary tumor location. Specifically, the C/C genotype of *CXCR4* rs2228014 was associated with improved PFS in patients with left-sided tumors who received an oxaliplatin-containing regimen (training cohort, validation cohort B), but not in those who received non-oxaliplatin-based therapy (validation cohort A) or those with right-sided tumors. With regard to treatment rendered, our findings are consistent with a prior investigation by Thomaidis *et al.,*^31^ in which esophagogastric cancer patients with CXCR4-negative tumor tissue benefited specifically from oxaliplatin-based chemotherapy.

Our study has its limitations. As this is a retrospective analysis with limited sample size, larger prospective studies are needed to validate our findings. The inclusion of a control patient cohort who received a cetuximab but not bevacizumab-containing regimen supports our hypothesis that the effects of the candidate polymorphisms are specific to antiangiogenic therapy. However, examining patients receiving bevacizumab monotherapy, oxaliplatin- or irinotecan-based regimens without a targeted agent may help further determine the specific predictive and prognostic associations of *CXCR4* rs2228014. In addition, we did not have sufficient power to test the effect of *BRAF* mutation status on the observed associations in all cohorts. Future studies should address these factors, as well as the effect of gender and *ERCC1* expression, and potential associations between genetic variants with early tumor shrinkage and depth of response.

In conclusion, our study is the first to identify and validate genetic variants within the CXCR4/CXCL12 pathway that are associated with clinical outcomes in mCRC patients treated with first-line bevacizumab-based chemotherapy. Furthermore, our findings suggest that the effect of *CXCR4* rs2228014 genetic variants on outcomes may be more clinically relevant in patients with left-sided tumors and in those receiving oxaliplatin-based rather than irinotecan-based chemotherapy regimens. If confirmed in future prospective studies, *CXCR4* genotype may serve as a promising predictive biomarker and guide clinical decision making in the management of mCRC patients.

## Supplementary Material

## ACKNOWLEDGMENTS

S Matsusaka is a recipient of Takashi Tsuruo Memorial Fund. MD Berger received a grant from the Swiss Cancer League (BIL KLS-3334-02-2014). S Stremitzer is a recipient of an Erwin Schrödinger fellowship of the Austrian Science Fund (J3501-B13). This study was partly funded by NIH Grant P30CA14089-27S1 and Yvonne Bogdanovich. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Cancer Institute or the National Institutes of Health.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Zlotnik A. Chemokines and cancer. Int J Cancer 2006; 119: 2026–2029.  [DOI](https://doi.org/10.1002/ijc.22024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16671092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Chemokines%20and%20cancer&author=A%20Zlotnik&volume=119&publication_year=2006&pages=2026-2029&pmid=16671092&doi=10.1002/ijc.22024&)

2. Burger JA, Kipps TJ. CXCR4: a key receptor in the crosstalk between tumor cells and their microenvironment. Blood 2006; 107: 1761–1767.  [DOI](https://doi.org/10.1182/blood-2005-08-3182) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16269611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=CXCR4:%20a%20key%20receptor%20in%20the%20crosstalk%20between%20tumor%20cells%20and%20their%20microenvironment&author=JA%20Burger&author=TJ%20Kipps&volume=107&publication_year=2006&pages=1761-1767&pmid=16269611&doi=10.1182/blood-2005-08-3182&)

3. Kucia M, Reca R, Miekus K, Wanzeck J, Wojakowski W, Janowska-Wieczorek A et al. Trafficking of normal stem cells and metastasis of cancer stem cells involve similar mechanisms: pivotal role of the SDF-1-CXCR4 axis. Stem Cells 2005; 23: 879–894.  [DOI](https://doi.org/10.1634/stemcells.2004-0342) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15888687/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stem%20Cells&title=Trafficking%20of%20normal%20stem%20cells%20and%20metastasis%20of%20cancer%20stem%20cells%20involve%20similar%20mechanisms:%20pivotal%20role%20of%20the%20SDF-1-CXCR4%20axis&author=M%20Kucia&author=R%20Reca&author=K%20Miekus&author=J%20Wanzeck&author=W%20Wojakowski&volume=23&publication_year=2005&pages=879-894&pmid=15888687&doi=10.1634/stemcells.2004-0342&)

4. Zou YR, Kottmann AH, Kuroda M, Taniuchi I, Littman DR. Function of the chemokine receptor CXCR4 in haematopoiesis and in cerebellar development. Nature 1998; 393: 595–599.  [DOI](https://doi.org/10.1038/31269) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9634238/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Function%20of%20the%20chemokine%20receptor%20CXCR4%20in%20haematopoiesis%20and%20in%20cerebellar%20development&author=YR%20Zou&author=AH%20Kottmann&author=M%20Kuroda&author=I%20Taniuchi&author=DR%20Littman&volume=393&publication_year=1998&pages=595-599&pmid=9634238&doi=10.1038/31269&)

5. Salvucci O, Yao L, Villalba S, Sajewicz A, Pittaluga S, Tosato G. Regulation of endothelial cell branching morphogenesis by endogenous chemokine stromal-derived factor-1. Blood 2002; 99: 2703–2711.  [DOI](https://doi.org/10.1182/blood.v99.8.2703) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11929756/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Regulation%20of%20endothelial%20cell%20branching%20morphogenesis%20by%20endogenous%20chemokine%20stromal-derived%20factor-1&author=O%20Salvucci&author=L%20Yao&author=S%20Villalba&author=A%20Sajewicz&author=S%20Pittaluga&volume=99&publication_year=2002&pages=2703-2711&pmid=11929756&doi=10.1182/blood.v99.8.2703&)

6. Guleng B, Tateishi K, Ohta M, Kanai F, Jazag A, Ijichi H et al. Blockade of the stromal cell-derived factor-1/CXCR4 axis attenuates in vivo tumor growth by inhibiting angiogenesis in a vascular endothelial growth factor-independent manner. Cancer Res 2005; 65: 5864–5871.  [DOI](https://doi.org/10.1158/0008-5472.CAN-04-3833) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15994964/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Blockade%20of%20the%20stromal%20cell-derived%20factor-1/CXCR4%20axis%20attenuates%20in%20vivo%20tumor%20growth%20by%20inhibiting%20angiogenesis%20in%20a%20vascular%20endothelial%20growth%20factor-independent%20manner&author=B%20Guleng&author=K%20Tateishi&author=M%20Ohta&author=F%20Kanai&author=A%20Jazag&volume=65&publication_year=2005&pages=5864-5871&pmid=15994964&doi=10.1158/0008-5472.CAN-04-3833&)

7. Kim J, Mori T, Chen SL, Amersi FF, Martinez SR, Kuo C et al. Chemokine receptor CXCR4 expression in patients with melanoma and colorectal cancer liver metastases and the association with disease outcome. Ann Surg 2006; 244: 113–120.  [DOI](https://doi.org/10.1097/01.sla.0000217690.65909.9c) | [PMC free article](/articles/PMC1570598/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16794396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Surg&title=Chemokine%20receptor%20CXCR4%20expression%20in%20patients%20with%20melanoma%20and%20colorectal%20cancer%20liver%20metastases%20and%20the%20association%20with%20disease%20outcome&author=J%20Kim&author=T%20Mori&author=SL%20Chen&author=FF%20Amersi&author=SR%20Martinez&volume=244&publication_year=2006&pages=113-120&pmid=16794396&doi=10.1097/01.sla.0000217690.65909.9c&)

8. Schimanski CC, Schwald S, Simiantonaki N, Jayasinghe C, Gonner U, Wilsberg V et al. Effect of chemokine receptors CXCR4 and CCR7 on the metastatic behavior of human colorectal cancer. Clin Cancer Res 2005; 11: 1743–1750.  [DOI](https://doi.org/10.1158/1078-0432.CCR-04-1195) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15755995/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Effect%20of%20chemokine%20receptors%20CXCR4%20and%20CCR7%20on%20the%20metastatic%20behavior%20of%20human%20colorectal%20cancer&author=CC%20Schimanski&author=S%20Schwald&author=N%20Simiantonaki&author=C%20Jayasinghe&author=U%20Gonner&volume=11&publication_year=2005&pages=1743-1750&pmid=15755995&doi=10.1158/1078-0432.CCR-04-1195&)

9. Mrowicki J, Przybylowska-Sygut K, Dziki L, Sygut A, Chojnacki J, Dziki A et al. The role of polymorphisms of genes CXCL12/CXCR4 and MIF in the risk development IBD the Polish population. Mol Biol Rep 2014; 41: 4639–4652.  [DOI](https://doi.org/10.1007/s11033-014-3335-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24687413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Rep&title=The%20role%20of%20polymorphisms%20of%20genes%20CXCL12/CXCR4%20and%20MIF%20in%20the%20risk%20development%20IBD%20the%20Polish%20population&author=J%20Mrowicki&author=K%20Przybylowska-Sygut&author=L%20Dziki&author=A%20Sygut&author=J%20Chojnacki&volume=41&publication_year=2014&pages=4639-4652&pmid=24687413&doi=10.1007/s11033-014-3335-y&)

10. Loupakis F, Cremolini C, Masi G, Lonardi S, Zagonel V, Salvatore L et al. Initial therapy with FOLFOXIRI and bevacizumab for metastatic colorectal cancer. N Engl J Med 2014; 371: 1609–1618.  [DOI](https://doi.org/10.1056/NEJMoa1403108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25337750/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Initial%20therapy%20with%20FOLFOXIRI%20and%20bevacizumab%20for%20metastatic%20colorectal%20cancer&author=F%20Loupakis&author=C%20Cremolini&author=G%20Masi&author=S%20Lonardi&author=V%20Zagonel&volume=371&publication_year=2014&pages=1609-1618&pmid=25337750&doi=10.1056/NEJMoa1403108&)

11. Tsuji A, Nakamura M, Sunakawa Y, Kochi M, Denda T, Yamaguchi T et al. A phase II study of cetuximab and mFOLFOX6 in mCRC including prospective early tumor shrinkage analysis (JACCRO-CC05). 2013. Ann Oncol 24: iv38–iv121.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=A%20phase%20II%20study%20of%20cetuximab%20and%20mFOLFOX6%20in%20mCRC%20including%20prospective%20early%20tumor%20shrinkage%20analysis%20(JACCRO-CC05)&author=A%20Tsuji&author=M%20Nakamura&author=Y%20Sunakawa&author=M%20Kochi&author=T%20Denda&volume=24&publication_year=2013&pages=iv38-iv121&)

12. Tsuji A, Sunakawa Y, Denda T, Takinishi Y, Kotaka M, Tanioka H et al. A phase I/II study of cetuximab (cet) in combination with S–1 and oxaliplatin (SOX) in first-line treatment for metastatic colorectal cancer (mCRC) (JACCRO CC-06). J Clin Oncol 2014; 32 (Suppl 3): abstract 571.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=A%20phase%20I/II%20study%20of%20cetuximab%20(cet)%20in%20combination%20with%20S%E2%80%931%20and%20oxaliplatin%20(SOX)%20in%20first-line%20treatment%20for%20metastatic%20colorectal%20cancer%20(mCRC)%20(JACCRO%20CC-06)&author=A%20Tsuji&author=Y%20Sunakawa&author=T%20Denda&author=Y%20Takinishi&author=M%20Kotaka&volume=32&issue=Suppl%203&publication_year=2014&)

13. Loupakis F, Yang D, Yau L, Feng S, Cremolini C, Zhang W et al. Primary tumor location as a prognostic factor in metastatic colorectal cancer. J Natl Cancer Inst 2015; 107: dju427.  [DOI](https://doi.org/10.1093/jnci/dju427) | [PMC free article](/articles/PMC4565528/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25713148/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Primary%20tumor%20location%20as%20a%20prognostic%20factor%20in%20metastatic%20colorectal%20cancer&author=F%20Loupakis&author=D%20Yang&author=L%20Yau&author=S%20Feng&author=C%20Cremolini&volume=107&publication_year=2015&pages=dju427&pmid=25713148&doi=10.1093/jnci/dju427&)

14. Dwinell MB, Ogawa H, Barrett KE, Kagnoff MF. SDF-1/CXCL12 regulates cAMP production and ion transport in intestinal epithelial cells via CXCR4. Am J Physiol Gastrointest Liver Physiol 2004; 286: G844–G850.  [DOI](https://doi.org/10.1152/ajpgi.00112.2003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14684377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol%20Gastrointest%20Liver%20Physiol&title=SDF-1/CXCL12%20regulates%20cAMP%20production%20and%20ion%20transport%20in%20intestinal%20epithelial%20cells%20via%20CXCR4&author=MB%20Dwinell&author=H%20Ogawa&author=KE%20Barrett&author=MF%20Kagnoff&volume=286&publication_year=2004&pages=G844-G850&pmid=14684377&doi=10.1152/ajpgi.00112.2003&)

15. Cai C, Wang LH, Dong Q, Wu ZJ, Li MY, Sun YH. Association of CXCL12 and CXCR4 gene polymorphisms with the susceptibility and prognosis of renal cell carcinoma. Tissue Antigens 2013; 82: 165–170.  [DOI](https://doi.org/10.1111/tan.12170) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24032722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tissue%20Antigens&title=Association%20of%20CXCL12%20and%20CXCR4%20gene%20polymorphisms%20with%20the%20susceptibility%20and%20prognosis%20of%20renal%20cell%20carcinoma&author=C%20Cai&author=LH%20Wang&author=Q%20Dong&author=ZJ%20Wu&author=MY%20Li&volume=82&publication_year=2013&pages=165-170&pmid=24032722&doi=10.1111/tan.12170&)

16. Kucukgergin C, Isman FK, Dasdemir S, Cakmakoglu B, Sanli O, Gokkusu C et al. The role of chemokine and chemokine receptor gene variants on the susceptibility and clinicopathological characteristics of bladder cancer. Gene 2012; 511: 7–11.  [DOI](https://doi.org/10.1016/j.gene.2012.09.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22982413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=The%20role%20of%20chemokine%20and%20chemokine%20receptor%20gene%20variants%20on%20the%20susceptibility%20and%20clinicopathological%20characteristics%20of%20bladder%20cancer&author=C%20Kucukgergin&author=FK%20Isman&author=S%20Dasdemir&author=B%20Cakmakoglu&author=O%20Sanli&volume=511&publication_year=2012&pages=7-11&pmid=22982413&doi=10.1016/j.gene.2012.09.011&)

17. Teng YH, Liu TH, Tseng HC, Chung TT, Yeh CM, Li YC et al. Contribution of genetic polymorphisms of stromal cell-derived factor-1 and its receptor, CXCR4, to the susceptibility and clinicopathologic development of oral cancer. Head Neck 2009; 31: 1282–1288.  [DOI](https://doi.org/10.1002/hed.21094) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19373784/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Head%20Neck&title=Contribution%20of%20genetic%20polymorphisms%20of%20stromal%20cell-derived%20factor-1%20and%20its%20receptor,%20CXCR4,%20to%20the%20susceptibility%20and%20clinicopathologic%20development%20of%20oral%20cancer&author=YH%20Teng&author=TH%20Liu&author=HC%20Tseng&author=TT%20Chung&author=CM%20Yeh&volume=31&publication_year=2009&pages=1282-1288&pmid=19373784&doi=10.1002/hed.21094&)

18. Oskarsson T, Batlle E, Massague J. Metastatic stem cells: sources, niches, and vital pathways. Cell Stem Cell 2014; 14: 306–321.  [DOI](https://doi.org/10.1016/j.stem.2014.02.002) | [PMC free article](/articles/PMC3998185/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24607405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Stem%20Cell&title=Metastatic%20stem%20cells:%20sources,%20niches,%20and%20vital%20pathways&author=T%20Oskarsson&author=E%20Batlle&author=J%20Massague&volume=14&publication_year=2014&pages=306-321&pmid=24607405&doi=10.1016/j.stem.2014.02.002&)

19. Schioppa T, Uranchimeg B, Saccani A, Biswas SK, Doni A, Rapisarda A et al. Regulation of the chemokine receptor CXCR4 by hypoxia. J Exp Med 2003; 198: 1391–1402.  [DOI](https://doi.org/10.1084/jem.20030267) | [PMC free article](/articles/PMC2194248/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14597738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Med&title=Regulation%20of%20the%20chemokine%20receptor%20CXCR4%20by%20hypoxia&author=T%20Schioppa&author=B%20Uranchimeg&author=A%20Saccani&author=SK%20Biswas&author=A%20Doni&volume=198&publication_year=2003&pages=1391-1402&pmid=14597738&doi=10.1084/jem.20030267&)

20. Staller P, Sulitkova J, Lisztwan J, Moch H, Oakeley EJ, Krek W. Chemokine receptor CXCR4 downregulated by von Hippel-Lindau tumour suppressor pVHL. Nature 2003; 425: 307–311.  [DOI](https://doi.org/10.1038/nature01874) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/13679920/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Chemokine%20receptor%20CXCR4%20downregulated%20by%20von%20Hippel-Lindau%20tumour%20suppressor%20pVHL&author=P%20Staller&author=J%20Sulitkova&author=J%20Lisztwan&author=H%20Moch&author=EJ%20Oakeley&volume=425&publication_year=2003&pages=307-311&pmid=13679920&doi=10.1038/nature01874&)

21. Fransen K, Fenech M, Fredrikson M, Dabrosin C, Soderkvist P. Association between ulcerative growth and hypoxia inducible factor-1alpha polymorphisms in colorectal cancer patients. Mol Carcinog 2006; 45: 833–840.  [DOI](https://doi.org/10.1002/mc.20209) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16865676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Carcinog&title=Association%20between%20ulcerative%20growth%20and%20hypoxia%20inducible%20factor-1alpha%20polymorphisms%20in%20colorectal%20cancer%20patients&author=K%20Fransen&author=M%20Fenech&author=M%20Fredrikson&author=C%20Dabrosin&author=P%20Soderkvist&volume=45&publication_year=2006&pages=833-840&pmid=16865676&doi=10.1002/mc.20209&)

22. Gerger A, El-Khoueiry A, Zhang W, Yang D, Singh H, Bohanes P et al. Pharmacogenetic angiogenesis profiling for first-line Bevacizumab plus oxaliplatin-based chemotherapy in patients with metastatic colorectal cancer. Clin Cancer Res 2011; 17: 5783–5792.  [DOI](https://doi.org/10.1158/1078-0432.CCR-11-1115) | [PMC free article](/articles/PMC3174848/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21791631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Pharmacogenetic%20angiogenesis%20profiling%20for%20first-line%20Bevacizumab%20plus%20oxaliplatin-based%20chemotherapy%20in%20patients%20with%20metastatic%20colorectal%20cancer&author=A%20Gerger&author=A%20El-Khoueiry&author=W%20Zhang&author=D%20Yang&author=H%20Singh&volume=17&publication_year=2011&pages=5783-5792&pmid=21791631&doi=10.1158/1078-0432.CCR-11-1115&)

23. Xu L, Duda DG, di Tomaso E, Ancukiewicz M, Chung DC, Lauwers GY et al. Direct evidence that bevacizumab, an anti-VEGF antibody, up-regulates SDF1alpha, CXCR4, CXCL6, and neuropilin 1 in tumors from patients with rectal cancer. Cancer Res 2009; 69: 7905–7910.  [DOI](https://doi.org/10.1158/0008-5472.CAN-09-2099) | [PMC free article](/articles/PMC2859041/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19826039/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Direct%20evidence%20that%20bevacizumab,%20an%20anti-VEGF%20antibody,%20up-regulates%20SDF1alpha,%20CXCR4,%20CXCL6,%20and%20neuropilin%201%20in%20tumors%20from%20patients%20with%20rectal%20cancer&author=L%20Xu&author=DG%20Duda&author=E%20di%20Tomaso&author=M%20Ancukiewicz&author=DC%20Chung&volume=69&publication_year=2009&pages=7905-7910&pmid=19826039&doi=10.1158/0008-5472.CAN-09-2099&)

24. Matsusaka S, Mishima Y, Suenaga M, Terui Y, Kuniyoshi R, Mizunuma N et al. Circulating endothelial progenitors and CXCR4-positive circulating endothelial cells are predictive markers for bevacizumab. Cancer 2011; 117: 4026–4032.  [DOI](https://doi.org/10.1002/cncr.25977) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21858803/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Circulating%20endothelial%20progenitors%20and%20CXCR4-positive%20circulating%20endothelial%20cells%20are%20predictive%20markers%20for%20bevacizumab&author=S%20Matsusaka&author=Y%20Mishima&author=M%20Suenaga&author=Y%20Terui&author=R%20Kuniyoshi&volume=117&publication_year=2011&pages=4026-4032&pmid=21858803&doi=10.1002/cncr.25977&)

25. Federsppiel B, Melhado IG, Duncan AM, Delaney A, Schappert K, Clark-Lewis I et al. Molecular cloning of the cDNA and chromosomal localization of the gene for a putative seven-transmembrane segment (7-TMS) receptor isolated from human spleen. Genomics 1993; 16: 707–712.  [DOI](https://doi.org/10.1006/geno.1993.1251) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8325644/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics&title=Molecular%20cloning%20of%20the%20cDNA%20and%20chromosomal%20localization%20of%20the%20gene%20for%20a%20putative%20seven-transmembrane%20segment%20(7-TMS)%20receptor%20isolated%20from%20human%20spleen&author=B%20Federsppiel&author=IG%20Melhado&author=AM%20Duncan&author=A%20Delaney&author=K%20Schappert&volume=16&publication_year=1993&pages=707-712&pmid=8325644&doi=10.1006/geno.1993.1251&)

26. Petersen DC, Glashoff RH, Shrestha S, Bergeron J, Laten A, Gold B et al. Risk for HIV-1 infection associated with a common CXCL12 (SDF1) polymorphism and CXCR4 variation in an African population. J Acquir Immune Defic Syndr 2005; 40: 521–526.  [DOI](https://doi.org/10.1097/01.qai.0000186360.42834.28) | [PMC free article](/articles/PMC1369993/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16284526/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Risk%20for%20HIV-1%20infection%20associated%20with%20a%20common%20CXCL12%20(SDF1)%20polymorphism%20and%20CXCR4%20variation%20in%20an%20African%20population&author=DC%20Petersen&author=RH%20Glashoff&author=S%20Shrestha&author=J%20Bergeron&author=A%20Laten&volume=40&publication_year=2005&pages=521-526&pmid=16284526&doi=10.1097/01.qai.0000186360.42834.28&)

27. Kishima Okuyama M, de Oliveira Brajão K, Ariza CB, de Oliveira CE, Guembarovski Losi R, Hirata Banin B K et al. Genetic polymorphism and expression of CXCR4 in breast cancer. Anal Cell Pathol (Amst) 20152015: 289510.  [DOI](https://doi.org/10.1155/2015/289510) | [PMC free article](/articles/PMC4630341/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26576337/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Cell%20Pathol%20(Amst)&title=Genetic%20polymorphism%20and%20expression%20of%20CXCR4%20in%20breast%20cancer&author=Okuyama%20M%20Kishima&author=Braj%C3%A3o%20K%20de%20Oliveira&author=CB%20Ariza&author=CE%20de%20Oliveira&author=Losi%20R%20Guembarovski&volume=2015&publication_year=2015&pages=289510&pmid=26576337&doi=10.1155/2015/289510&)

28. Cancer Genome Atlas Network. Comprehensive molecular characterization of human colon and rectal cancer. Nature 2012; 487: 330–337.  [DOI](https://doi.org/10.1038/nature11252) | [PMC free article](/articles/PMC3401966/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22810696/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Comprehensive%20molecular%20characterization%20of%20human%20colon%20and%20rectal%20cancer&volume=487&publication_year=2012&pages=330-337&pmid=22810696&doi=10.1038/nature11252&)

29. Mitchell B, Leone D, Feller JK, Bondzie P, Yang S, Park HY et al. Correlation of chemokine receptor CXCR4 mRNA in primary cutaneous melanoma with established histopathologic prognosticators and the BRAF status. Melanoma Res 2014; 24: 621–625.  [DOI](https://doi.org/10.1097/CMR.0000000000000120) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25211166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Melanoma%20Res&title=Correlation%20of%20chemokine%20receptor%20CXCR4%20mRNA%20in%20primary%20cutaneous%20melanoma%20with%20established%20histopathologic%20prognosticators%20and%20the%20BRAF%20status&author=B%20Mitchell&author=D%20Leone&author=JK%20Feller&author=P%20Bondzie&author=S%20Yang&volume=24&publication_year=2014&pages=621-625&pmid=25211166&doi=10.1097/CMR.0000000000000120&)

30. Torregrossa L, Giannini R, Borrelli N, Sensi E, Melillo RM, Leocata P et al. CXCR4 expression correlates with the degree of tumor infiltration and BRAF status in papillary thyroid carcinomas. Mod Pathol 2012; 25: 46–55.  [DOI](https://doi.org/10.1038/modpathol.2011.140) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21909080/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mod%20Pathol&title=CXCR4%20expression%20correlates%20with%20the%20degree%20of%20tumor%20infiltration%20and%20BRAF%20status%20in%20papillary%20thyroid%20carcinomas&author=L%20Torregrossa&author=R%20Giannini&author=N%20Borrelli&author=E%20Sensi&author=RM%20Melillo&volume=25&publication_year=2012&pages=46-55&pmid=21909080&doi=10.1038/modpathol.2011.140&)

31. Thomaidis T, Maderer A, Al-Batran SE, Kany J, Pauligk C, Steinmetz K et al. VEGFR-3 and CXCR4 as predictive markers for treatment with fluorouracil, leucovorin plus either oxaliplatin or cisplatin in patients with advanced esophagogastric cancer: a comparative study of the Arbeitsgemeinschaft Internistische Onkologie (AIO). BMC Cancer 2014; 14: 476.  [DOI](https://doi.org/10.1186/1471-2407-14-476) | [PMC free article](/articles/PMC4094395/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24981311/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=VEGFR-3%20and%20CXCR4%20as%20predictive%20markers%20for%20treatment%20with%20fluorouracil,%20leucovorin%20plus%20either%20oxaliplatin%20or%20cisplatin%20in%20patients%20with%20advanced%20esophagogastric%20cancer:%20a%20comparative%20study%20of%20the%20Arbeitsgemeinschaft%20Internistische%20Onkologie%20(AIO)&author=T%20Thomaidis&author=A%20Maderer&author=SE%20Al-Batran&author=J%20Kany&author=C%20Pauligk&volume=14&publication_year=2014&pages=476&pmid=24981311&doi=10.1186/1471-2407-14-476&)
