# New Genetic Predictors for Abacavir Tolerance in HLA-B*57:01 Positive Individuals

## Metadata
**Authors:** Rebecca PAVLOS, Pooja DESHPANDE, Abha CHOPRA, Shay LEARY, Kaija STRAUTINS, David NOLAN, Daren THORBORN, Mark SHAEFER, Andri RAUCH, David DUNN, Julio MONTANER, Anita RACHLIS, Coral-Ann ALMEIDA, Linda CHOO, Ian JAMES, Alec J REDWOOD, Yueran LI, Silvana GAUDIERI, Simon A MALLAL, Elizabeth J PHILLIPS
**Journal:** Human immunology
**Date:** 2020 Mar 12
**DOI:** [10.1016/j.humimm.2020.02.011](https://doi.org/10.1016/j.humimm.2020.02.011)
**PMID:** 32173028
**PMCID:** PMC7266722
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7266722/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7266722/pdf/nihms-1590559.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7266722/pdf/nihms-1590559.pdf)

## Abstract

Abacavir hypersensitivity syndrome (ABC HSS) is strongly associated with carriage of human leukocyte antigen (HLA)-B*57:01, which has a 100% negative predictive value for the development of ABC HSS. However, 45% of individuals who carry HLA-B*57:01 can tolerate ABC. We investigated immune and non-immune related genes in ABC HSS (n=95) and ABC tolerant (n=43) HLA-B*57:01+ patients to determine other factors required for the development of ABC HSS. Assignment of phenotype showed that ABC HSS subjects were significantly less likely than tolerants to carry only ERAP1 hypoactive trimming allotypes (p=0.02). An altered self-peptide repertoire model by which abacavir activates T cells is in keeping with observation that endoplasmic reticulum aminopeptidase 1 (ERAP1) allotypes that favour efficient peptide trimming are more common in ABC HSS patients compared to patients who tolerate ABC. Independently, non-specific immune activation via soluble cluster of differentiation antigen 14 (sCD14) may also influence susceptibility to ABC HSS.

Keywords: Abacavir hypersensitivity syndrome, Human leukocyte antigen, Endoplasmic reticulum aminopeptidase 1

## 1. Introduction

Abacavir (ABC) is an antiretroviral drug used to treat HIV infection and abacavir hypersensitivity syndrome (ABC HSS) is a model example of a severe, human leukocyte antigen (HLA)-associated, immune-mediated adverse drug reaction (IM-ADR)[[1](#R1)–[3](#R3)]. From studies examining patch test (PT) confirmed cases[[3](#R3)], ABC HSS develops within 36 hours-3 weeks of therapy initiation and is characterized by initial fever, malaise, and systemic features with the development of mild to moderate rash in 70% of cases. Although patients recover within 48–72 hours upon withdrawal of the drug, a severe shock-like syndrome occurs on re-exposure. True immune-mediated ABC HSS only occurs in individuals who carry the HLA-B*57:01 allele, and not the related HLA-B17 serotype alleles (HLA-B*58:01 and HLA-B*57:03). Clinical practice guidelines recommending screening for HLA-B*57:01 before the initiation of ABC therapy has now been widely implemented in the developed world.

The PREDICT-1 study, which randomized patients to real-time HLA-B*57:01 screening versus ABC treatment without screening, supported a 100% negative predictive value (NPV), however it also showed that only 55% of those carrying HLA-B*57:01 developed ABC HSS[[3](#R3)]. These results demonstrate that HLA-B*57:01 is necessary for the development of ABC HSS but it is still unclear why 45% of those carrying HLA-B*57:01 appear ABC tolerant[[3](#R3)] and what specific factors drive the incomplete positive predictive value (PPV). Based on peptide elution data and the crystal structure of ABC bound to HLA-B*57:01, we[[4](#R4)] and others[[5](#R5)] have provided evidence for the altered peptide repertoire model of ABC HSS where non-covalent binding of ABC within the HLA-B*57:01 antigen blinding cleft alters the normal presentation of self-peptides. Accordingly, variations in molecules that are involved in antigen presentation, immune-related functions directly involving, or genetically linked, to HLA-B*57:01 or metabolism of ABC are likely candidates that contribute to ABC HSS susceptibility. Understanding the additional factors that contribute to ABC HSS may provide further clues into the immune-mechanisms of other IM-ADRs[[6](#R6)].

We conducted a retrospective genetic analysis of (PT+) ABC HSS and ABC tolerant HLA-B*57:01 patients to investigate other immune- and drug-related factors in addition to HLA-B*57:01 that contribute to ABC HSS susceptibility. The genes examined were: immune-related major histocompatibility complex (MHC) genes in linkage disequilibrium (LD) with HLA-B; alcohol dehydrogenase (ADH) that metabolises ABC; innate immune system genes cluster of differentiation antigen 14 (CD14); natural killer (NK) cell killer-cell immunoglobulin-like receptors (KIR) KIR3DS1 and KIR3DL1 that are known ligands for HLA-B*57:01; and endoplasmic reticulum aminopeptidase 1 (ERAP1) and ERAP2 that trim the N-terminal of peptides before presentation on HLA class I molecules. All candidate gene SNPs were examined in PT+ ABC HSS and ABC tolerant HLA-B*57:01 patients from multi-centre studies.

## 2. Methods

### 2.1. Samples

HLA-B*57:01+ patients who were either PT+ ABC HSS (using described techniques with 1% and 10% abacavir; [[7](#R7)]) or ABC tolerant for >6 weeks with no symptoms and who had previously consented to genetic testing were identified from multicentre studies: The Prospective Randomized Evaluation of DNA Screening in a Clinical Trial (PREDICT-1; CNA106030), a randomized controlled double blinded study (n=42); the Study of Hypersensitivity to Abacavir and Pharmacogenetic Evaluation (SHAPE; ABC107442), a case control study (n=57); a multinational collaborative retrospective Canadian-Swiss-Australian study; and a single HLA-B*57:01 ABC tolerant patient from the ACTG NWCS 365 (n=39). In total there were n=95 PT+ ABC HSS and n=43 ABC tolerant patients.

For ERAP genotyping a subset of patients (n=56, PT+ ABC HSS; n=28, ABC tolerant; restricted to European ancestry) were available from the SHAPE and PREDICT studies. Genetic mapping for alleles of the HLA-B*57:01 extended MHC haplotype (also termed 57.1 ancestral haplotype) was carried-out on patients from all three studies with high-resolution HLA typing results available from the PREDICT-1 and multinational collaborative studies. Genotyping for ADH isoforms, KIR and CD14 was carried-out on patients with samples available from all three studies.

Ethics approval for this work was provided by the Royal Perth Hospital (2010/135), Murdoch University (2014/020) and Vanderbilt University Medical Center (IRB#131836) Human Research Ethics Committees and from the original PREDICT-1, SHAPE and multinational studies.

### 2.2. Genotyping

Four-digit HLA genotyping was conducted as previously described[[3](#R3)]. Sequence-based typing was used for the TNF single nucleotide polymorphisms (SNPs) −238 (rs361525) and −376 (rs1800750) (forward primer 5’-ACACAGGCCTCAGGACTCAA-3’, reverse primer 5’- AACCAGCGGAAAACTTCCTT-3’). PCR sequence specific primer (PCR-SSP) assays were used to detect non-synonymous SNPs for loci on the HLA-B*57.1 extended MHC haplotype: HCP5 gene (rs2395029), BAT1–223 (rs2239527), HSP70Hom 493T[[8](#R8)], C4A*6 and MICA*017. PCR-SSP was also used for the presence/absence of KIR3DS1/KIR3DL1 and type 1 ADH isoforms 1B (rs1229984) and 1C (rs698) (see [Table S1](#SD2) for PCR-SSP primer details). A restriction fragment length polymorphism approach was used to detect a functional polymorphism in the promoter region of the CD14 gene [−159C/T]. Sequence-based typing was used to determine the genotype of SNPs that characterise ERAP1 allotypes and rs2248374, a tag SNP for functional ERAP2 haplotypes[[9](#R9)].

### 2.3. ERAP trimming assignments

The amino acid (aa) positions relevant to impute ERAP1 allotype based on Reeves et al 2013[[10](#R10)] (spanning the five residues M349V, K528R, D575N, R725Q and Q730E; as indicated in [Figure 1B](#F1)) and Ombrello et al 2015 (12 aa positions overall) were used to assign putative ERAP1 trimming efficiency as efficient, hyperactive or hypoactive. The novel combination (M349/K528/D575/R725/E730) was assigned an efficient trimming phenotype based on the wildtype amino acid at position 528.

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2388/7266722/8d069ff341f0/nihms-1590559-f0001.jpg)

ERAP1 allotypes. A. Schematic representation of ERAP1 with domains and active site indicated (bar). Amino acid positions relevant for the ERAP1 allotype nomenclature based on Reeves et al 2013 (red text)[10] and Ombrello et al 2015 (all sites)[13] indicated. B. Allotype frequency for the PT+ ABC HSS and ABC tolerant groups. The allotype defined by the MKDRE combination for the five defining SNPs has unknown phenotype, while the allotype defined as M349V/K528R also has D575N, R725Q and Q730E SNPs. p=0.02, Fisher’s exact test for carriage of efficient ERAP1 allotype between PT+ ABC HSS and ABC tolerant. Frequency of homozygous hypoactive ERAP in PT+ ABC HSS was 0.18 compared to 0.43 in ABC tolerant group. Note ERAP1 combinations not described elsewhere were assigned the ‘efficient’ allotype based on wildtype amino acid at 349 and 528. Note not all subjects had full typing across all SNPs; in three cases the allotype V349/R528 was imputed given no typing at position 575.

### 2.4. Statistics

Pooled statistical analyses comparing genetic markers between the PT+ ABC HSS and ABC tolerant groups were carried-out using Fisher’s exact test. A p value <0.05 was set as a significance threshold. Multiple case-control/logistic regressions were performed to ascertain independent associations. Due to the small numbers in this study correction for multiple comparisons was not performed. Furthermore, the introduction of HLA-B*57:01 screening before prescribing ABC now prevents the ability to obtain more samples for further analysis. Minor Allele Frequencies (MAF) were taken from the 1000 genomes project database ([http://www.internationalgenome.org/data/](http://www.internationalgenome.org/data/)). Assessment of LD and allocation of ERAP1 haplotypes was performed using the program Arlequin v3.5.2.2[[11](#R11)].

## 3. Results

No trend was observed between PT+ ABC HSS and ABC tolerant HLA-B*57:01 carriers for the haplospecific/typic markers MICA*017, HCP5, Bat-1, TNF, and Hsp70Hom present on the HLA-B*57.1 extended MHC haplotype. Although the presence of HLA-B*57:01 is necessary for ABC HSS, 45% patients carry HLA-B*57:01 but do not develop ABC HSS. This implies that factors in addition to HLA-B*57:01 but outside of the extended 57.1 haplotype may contribute to the development of ABC HSS ([Figure S1](#SD1)). There was also no difference in the presence of other genes known to interact with HLA-B*57:01 such as KIR3DS1/KIR3DL1, or polymorphisms in enzymes that metabolize ABC such as ADH 1B or ADH 1C. In multivariate models incorporating all three studies only the presence of C4A*6 and a functional polymorphism in the promoter of the CD14 gene [−159C/T] were significantly associated with the presence of PT+ ABC HSS ([Table 1](#T1)). We did not observe a trend between African and European ethnicity for any of the markers shown or between the ABC tolerant and PT+ ABC HSS patients ([Table S2](#SD3)).

### Table 1.

| Genetic Marker | PT+ ABC HSSN=95 (%) | Abacavir TolerantN=43 (%) | P-value |
| --- | --- | --- | --- |
| 57.1 ancestral haplotype markers |   |   |   |
| C4A*6 | 79 (83) | 26 (65) | 0.03 |
| Hsp70Hom 493T |   |   |   |
| CC | 11 (11.6) | 5 (12.5) | 0.23 |
| CT | 73 (76.8) | 26 (65) |   |
| TT | 11 (11.6) | 9 (22.5) |   |
| TNF-238 |   |   |   |
| AA | 3 (3.2) | 1 (2.5) | 0.76 |
| GA | 85 (89.5) | 38 (95) |   |
| GG | 7 (7.37) | 1 (2.5) |   |
| TNF-376 |   |   |   |
| GA | 1 (1.1) | 1 (2.5) | 0.51 |
| GG | 94 (98.9) | 39 (97.5) |  |
| BAT1–223 |   |   |   |
| AA | 2 (2.1) | 0 | 0.41 |
| AC | 85 (89.5) | 39 (97.5) |   |
| CC | 8 (8.4) | 1 (2.5) |   |
| HCP5(rs2395029) |   |   |   |
| GG | 2 (2.1) | 0 | 1 |
| GT | 92 (96.8) | 40 (100) |   |
| TT | 1 (1.1) | 0 |   |
| MICA*017 | 89 (98.9) | 38 (100) | 1 |
| Abacavir Metabolism |   |   |   |
| ADH 1C (rs 698) |   |   |   |
| AA | 30 (37) | 15 (46.9) | 0.59 |
| GA | 33 (40.7) | 12 (37.5) |   |
| GG | 18 (22.2) | 5 (15.6) |   |
| ADH 1B (rs 1229984) |   |   |   |
| AA | 1 (1.25) | 0 | 1 |
| GA | 10 (12.5) | 4 (12.5) |   |
| GG | 69 (86.3) | 28 (87.5) |   |
| Innate immune Genes |   |   |   |
| CD14 [159 C/T] |   |   |   |
| CC | 18 (20.2) | 16 (42.1) | 0.025 |
| CT | 51 (57.3) | 13 (34.2) |   |
| TT | 20 (22.5) | 9 (28.1) |   |
| KIR3DS1+ | 44 (49.4) | 17 (44.7) | 0.70 |
| KIR3DL1+ | 85 (95.5) | 37 (97.4) | 1 |

Table 1 Caption: Genotype data for select genes in PT+ ABC HSS and ABC tolerant patients.

The presence of markers for the HLA-B*57:01 extended MHC haplotype was mapped for each of the subjects. Ancestral recombination events broke-down the haplotype in many individuals (e.g. 14/43 ABC tolerant subjects did not carry the full haplotype from C4A*6 to HLA-B*57:01) and a PT+ ABC HSS subject only carried HLA-B*57:01 confirming that HLA-B*57:01 is necessary. However, we have previously shown that only 55% of those carrying HLA-B*57:01 developed ABC HSS making it necessary but not sufficient for the development of ABC HSS. These results suggest these haplotype markers, although in LD with HLA-B*57:01, cannot be used as a surrogate for HLA-B*57:01 genetic screening or predictors of ABC HSS ([Figure S1](#SD1)) and that genes outside of the MHC haplotype such as ERAP might contribute to the remaining positive predictive value.

ERAP1 and ERAP2 variations were examined in an available subset of the PT+ ABC HSS (n=56) and ABC tolerant patients (n=28). A panel of SNPs were selected based on previously published ERAP1 allotypes[[12](#R12)] and analysis was restricted to European samples given the MAF differences observed between different ethnic groups for ERAP[[13](#R13)]. The SNPs rs2287987 (M349V), rs10050860 (D575N) and rs17482078 (R725Q) that encode amino acid changes in functional regions of ERAP showed significant LD (p<0.05). The differences in the carriage of the minor alleles for ERAP1 SNPs between PT+ ABC HSS and ABC tolerant patients were not significant. No significance differences were observed for ERAP2 rs2248374 comparing HLA-B*57:01 PT+ ABC HSS and ABC tolerant patients. A trend was observed for the SNP rs30187 (R528K) in the MAF for PT+ ABC HSS samples compared to ABC tolerant subjects and showed a significant difference compared to the reported MAFs in Europeans (based on the 1000 Genomes consortium; [Table 2](#T2)).

### Table 2.

| SNP | Nucleotide (Major to minor) | Amino Acid (Major to minor) | European MAF | PT + ABC HSS MAF (European) | ABC tolerant MAF (European) | P value (carriage minor allele PT+ ABC HSS v ABC tolerant) |
| --- | --- | --- | --- | --- | --- | --- |
| rs26653 | C > G | P127R | G=0.29 | 0.36 | 0.21 | 0.25 |
| rs26618 | T > G | I276M | G=0.24 | 0.19 | 0.25 | 0.61 |
| rs27895 | G > A | G346D | A=0.06 | 0.04 | 0.05 | 1 |
| rs2287987 | A > G | M349V | G=0.21 | 0.17 | 0.29 | 0.09 |
| rs30187 | G > A | R528K | A=0.35 | 0.48* | 0.32 | 0.08 |
| rs10050860 | G > A | D575N | A=0.21 | 0.15 | 0.25 | 0.12 |
| rs17482078 | G > A | R725Q | A=0.20 | 0.15 | 0.25 | 0.09 |
| rs27044 | C > G | Q730E | G=0.29 | 0.34 | 0.27 | 0.26 |
| rs2248374 | G > A | ERAP2 | A=0.48 | 0.42 | 0.53 | 0.39 |

Table 2 Caption: ERAP1 and ERAP2 SNP frequencies.^

Although individual ERAP SNPs did not show significance when comparing the frequency in PT+ ABC HSS and ABC tolerant patients, it is known that ERAP function is dependent on haplotypes, rather than discrete variants[[10](#R10)]. Amino acid positions relevant for imputing ERAP1 allotype were used to assign putative ERAP1 trimming efficiency as efficient, hyperactive or hypoactive[[10](#R10)]. Given the assignments, ABC tolerant subjects were more likely to carry only hypoactive allotypes than PT+ ABC HSS subjects (p=0.02, Fisher’s exact test; [Figure 1](#F1)).

## 4. Discussion

Previous studies support that HLA-B*57:01 has a 100% negative predictive value for ABC HSS and therefore is necessary for its development. Our data reinforce this and suggest that in reference to the 45% of those carrying HLA-B*57:01 who tolerate ABC there is no one haplospecific marker in LD with HLA-B*57:01 that is associated with ABC tolerance([Figure S1](#SD1)). There was a signficant association between PT+ ABC HSS and CD14 [−159C/T], located within the promoter region of CD14. CD14 is a co-receptor expressed by macrophages, neutorphils and dendritic cells that recognizes pathogen-associated molecular patterns and is the receptor for lipopolysaccharide. The CD14 −159 T variant is associated with increased levels of soluable CD14 (sCD14)[[14](#R14)]. sCD14 levels are predictive of increased mortality rates in treated HIV-infected individuals, independent of T cell count and most striking in individuals with advanced immunodeficiency[[15](#R15), [16](#R16)]. The link between sCD14 and HIV patient mortality is thought to reflect a compromised gut epithelial barrier. sCD14 is elevated in other diseases characterized or exacerbated by endotoxemia, such as hepatitis, rheumatoid arthritis, and systemic lupus erythematosus[[15](#R15)]. It is intriguing that CD14 variants, which reflect increased sCD14, are also significant in ABC HSS PT+ patients given that gastrointestinal symptoms are characteristic of the syndrome. This suggests that additional patient characteristics relating to HIV progression and gut endotoxemia in particular, could confer increased risk to ABC HSS in HLA-B57:01+ patients.

In addition to CD14 variation, ERAP1 functional haplotypes also displayed significant differences when comparing PT+ ABC HSS and ABC Tolerant HLA-B*57:01 patients. Epistatic interactions with ERAP1 have been described in autoimmune conditions such as ankylosing spondylitis[[12](#R12)], birdshot retinopathy[[17](#R17)] and Behcet’s disease[[18](#R18)] with strong HLA associations with HLA-B*27:05, HLA-A*29:02 and HLA-B*51:01, respectively. In these diseases ERAP1 allotypes are known to affect the peptidomes for the risk HLA alleles[[19](#R19)–[21](#R21)]. The C-terminal of the bound peptide influences ERAP1 activity at the N-terminal with a strong preference for peptides with hydrophobic or aromatic C-terminal residues (W, Y, F, M, A, V, L, or I, in order of the rate of N terminal trimming)[[22](#R22)]. Intriguingly, treatment of cells with ABC alters the peptide preference of HLA-B*57:01 at the C-terminal, favouring V and L over W and F [[4](#R4)] and therefore favouring peptides with C-terminal amino acids that promote the fastest rates of ERAP1-mediated peptide trimming. Consistent with this, our data predicts that ABC tolerant subjects tend to have hypoactive trimming ERAP1 allotypes. ABC is shown to bind within the F pocket of the peptide-binding groove of HLA-B*57:01, and thus altering the repertoire of self-peptides presented to T cells [[4](#R4)]. The potential for ERAP1 to influence the repertoire of peptides available to be presented to T cells in the presence of ABC is in keeping with the primary hypothesis of the altered peptide model by which ABC causes HSS.

This work highlights the complexity of ABC HSS and ABC tolerance in HLA-B*57:01 carriers. ABC HSS as a model of HLA-B*57:01 restricted HLA-mediated drug hypersentivity may universally depend on the ability to generate an altered repertoire of peptides presented to T cells by HLA-B*57:01 and may also be influenced independently by other innate factors such as sCD14 levels and the state of the gut epithelial barrier in HIV patients on therapy.

## Supplementary Material

## Acknowledgements

We thank our colleagues at the Institute for Immunology and Infectious Diseases, Murdoch University.

EP and SM are supported through National Institutes of Health grants P50 GM115305 (EP,SM), R34 AI136815(EP), R21 AI139021 (EP,SM), 1 R01 HG010863-01 (EP), P30 Al110527(SM). EP, SM and SG receive funding from the National Health and Medical Research Council (Australia). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.

Funding

Research reported in this publication was supported by the National Institutes of Health grants P50 GM115305, R34 AI136815, R21 AI139021, P30 Al110527, F30 AI131780, UM1AI068634, UM1AI068636 and UM1A106701, the National Health and Medical Research Council (Australia), the Australian Centre for HIV and Hepatitis Research (ACH^2^) and GlaxoSmithKline, Inc.

## Abbreviations

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Phillips EJ, Wong GA, Kaul R, Shahabi K, Nolan DA, Knowles SR et al. : Clinical and immunogenetic correlates of abacavir hypersensitivity. Aids 2005;19:979.  [DOI](https://doi.org/10.1097/01.aids.0000171414.99409.fb) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15905681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aids&title=Clinical%20and%20immunogenetic%20correlates%20of%20abacavir%20hypersensitivity&author=EJ%20Phillips&author=GA%20Wong&author=R%20Kaul&author=K%20Shahabi&author=DA%20Nolan&volume=19&publication_year=2005&pages=979&pmid=15905681&doi=10.1097/01.aids.0000171414.99409.fb&)

2. Mallal S, Nolan D, Witt C, Masel G, Martin AM, Moore C et al. : Association between presence of HLA-B*5701, HLA-DR7, and HLA-DQ3 and hypersensitivity to HIV-1 reverse-transcriptase inhibitor abacavir. Lancet 2002;359:727.  [DOI](https://doi.org/10.1016/s0140-6736(02)07873-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11888582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Association%20between%20presence%20of%20HLA-B*5701,%20HLA-DR7,%20and%20HLA-DQ3%20and%20hypersensitivity%20to%20HIV-1%20reverse-transcriptase%20inhibitor%20abacavir&author=S%20Mallal&author=D%20Nolan&author=C%20Witt&author=G%20Masel&author=AM%20Martin&volume=359&publication_year=2002&pages=727&pmid=11888582&doi=10.1016/s0140-6736(02)07873-x&)

3. Mallal S, Phillips E, Carosi G, Molina J-M, Workman C, Tomažič J et al. : HLA-B*5701 Screening for Hypersensitivity to Abacavir. New England Journal of Medicine 2008;358:568.  [DOI](https://doi.org/10.1056/NEJMoa0706135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18256392/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&title=HLA-B*5701%20Screening%20for%20Hypersensitivity%20to%20Abacavir&author=S%20Mallal&author=E%20Phillips&author=G%20Carosi&author=J-M%20Molina&author=C%20Workman&volume=358&publication_year=2008&pages=568&pmid=18256392&doi=10.1056/NEJMoa0706135&)

4. Ostrov DA, Grant BJ, Pompeu YA, Sidney J, Harndahl M, Southwood S et al. : Drug hypersensitivity caused by alteration of the MHC-presented self-peptide repertoire. Proc Natl Acad Sci U S A 2012;109:9959.  [DOI](https://doi.org/10.1073/pnas.1207934109) | [PMC free article](/articles/PMC3382472/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22645359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Drug%20hypersensitivity%20caused%20by%20alteration%20of%20the%20MHC-presented%20self-peptide%20repertoire&author=DA%20Ostrov&author=BJ%20Grant&author=YA%20Pompeu&author=J%20Sidney&author=M%20Harndahl&volume=109&publication_year=2012&pages=9959&pmid=22645359&doi=10.1073/pnas.1207934109&)

5. Illing PT, Vivian JP, Dudek NL, Kostenko L, Chen Z, Bharadwaj M et al. : Immune self-reactivity triggered by drug-modified HLA-peptide repertoire. Nature 2012;486:554.  [DOI](https://doi.org/10.1038/nature11147) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22722860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Immune%20self-reactivity%20triggered%20by%20drug-modified%20HLA-peptide%20repertoire&author=PT%20Illing&author=JP%20Vivian&author=NL%20Dudek&author=L%20Kostenko&author=Z%20Chen&volume=486&publication_year=2012&pages=554&pmid=22722860&doi=10.1038/nature11147&)

6. Pavlos R, Mallal S, Ostrov D, Buus S, Metushi I, Peters B et al. : T cell-mediated hypersensitivity reactions to drugs. Annu Rev Med 2015;66:439.  [DOI](https://doi.org/10.1146/annurev-med-050913-022745) | [PMC free article](/articles/PMC4295772/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25386935/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Med&title=T%20cell-mediated%20hypersensitivity%20reactions%20to%20drugs&author=R%20Pavlos&author=S%20Mallal&author=D%20Ostrov&author=S%20Buus&author=I%20Metushi&volume=66&publication_year=2015&pages=439&pmid=25386935&doi=10.1146/annurev-med-050913-022745&)

7. Phillips EJ, Sullivan JR, Knowles SR, Shear NH: Utility of patch testing in patients with hypersensitivity syndromes associated with abacavir. AIDS 2002;16:2223.  [DOI](https://doi.org/10.1097/00002030-200211080-00017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12409746/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=Utility%20of%20patch%20testing%20in%20patients%20with%20hypersensitivity%20syndromes%20associated%20with%20abacavir&author=EJ%20Phillips&author=JR%20Sullivan&author=SR%20Knowles&author=NH%20Shear&volume=16&publication_year=2002&pages=2223&pmid=12409746&doi=10.1097/00002030-200211080-00017&)

8. Allcock RJ, Windsor L, Gut IG, Kucharzak R, Sobre L, Lechner D et al. : High-Density SNP genotyping defines 17 distinct haplotypes of the TNF block in the Caucasian population: implications for haplotype tagging. Hum Mutat 2004;24:517.  [DOI](https://doi.org/10.1002/humu.20100) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15523649/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=High-Density%20SNP%20genotyping%20defines%2017%20distinct%20haplotypes%20of%20the%20TNF%20block%20in%20the%20Caucasian%20population:%20implications%20for%20haplotype%20tagging&author=RJ%20Allcock&author=L%20Windsor&author=IG%20Gut&author=R%20Kucharzak&author=L%20Sobre&volume=24&publication_year=2004&pages=517&pmid=15523649&doi=10.1002/humu.20100&)

9. Andres AM, Dennis MY, Kretzschmar WW, Cannons JL, Lee-Lin SQ, Hurle B et al. : Balancing selection maintains a form of ERAP2 that undergoes nonsense-mediated decay and affects antigen presentation. PLoS Genet 2010;6:e1001157.  [DOI](https://doi.org/10.1371/journal.pgen.1001157) | [PMC free article](/articles/PMC2954825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20976248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Balancing%20selection%20maintains%20a%20form%20of%20ERAP2%20that%20undergoes%20nonsense-mediated%20decay%20and%20affects%20antigen%20presentation&author=AM%20Andres&author=MY%20Dennis&author=WW%20Kretzschmar&author=JL%20Cannons&author=SQ%20Lee-Lin&volume=6&publication_year=2010&pages=e1001157&pmid=20976248&doi=10.1371/journal.pgen.1001157&)

10. Reeves E, Edwards CJ, Elliott T, James E: Naturally occurring ERAP1 haplotypes encode functionally distinct alleles with fine substrate specificity. J Immunol 2013;191:35.  [DOI](https://doi.org/10.4049/jimmunol.1300598) | [PMC free article](/articles/PMC3785127/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23733883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol&title=Naturally%20occurring%20ERAP1%20haplotypes%20encode%20functionally%20distinct%20alleles%20with%20fine%20substrate%20specificity&author=E%20Reeves&author=CJ%20Edwards&author=T%20Elliott&author=E%20James&volume=191&publication_year=2013&pages=35&pmid=23733883&doi=10.4049/jimmunol.1300598&)

11. Excoffier L, Lischer HE: Arlequin suite ver 3.5: a new series of programs to perform population genetics analyses under Linux and Windows. Mol Ecol Resour 2010;10:564.  [DOI](https://doi.org/10.1111/j.1755-0998.2010.02847.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21565059/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Ecol%20Resour&title=Arlequin%20suite%20ver%203.5:%20a%20new%20series%20of%20programs%20to%20perform%20population%20genetics%20analyses%20under%20Linux%20and%20Windows&author=L%20Excoffier&author=HE%20Lischer&volume=10&publication_year=2010&pages=564&pmid=21565059&doi=10.1111/j.1755-0998.2010.02847.x&)

12. Reeves E, Colebatch-Bourn A, Elliott T, Edwards CJ, James E: Functionally distinct ERAP1 allotype combinations distinguish individuals with Ankylosing Spondylitis. Proc Natl Acad Sci U S A 2014;111:17594.  [DOI](https://doi.org/10.1073/pnas.1408882111) | [PMC free article](/articles/PMC4267330/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25422414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Functionally%20distinct%20ERAP1%20allotype%20combinations%20distinguish%20individuals%20with%20Ankylosing%20Spondylitis&author=E%20Reeves&author=A%20Colebatch-Bourn&author=T%20Elliott&author=CJ%20Edwards&author=E%20James&volume=111&publication_year=2014&pages=17594&pmid=25422414&doi=10.1073/pnas.1408882111&)

13. Ombrello MJ, Kastner DL, Remmers EF: Endoplasmic reticulum-associated amino-peptidase 1 and rheumatic disease: genetics. Curr Opin Rheumatol 2015;27:349.  [DOI](https://doi.org/10.1097/BOR.0000000000000189) | [PMC free article](/articles/PMC4565054/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26002026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Rheumatol&title=Endoplasmic%20reticulum-associated%20amino-peptidase%201%20and%20rheumatic%20disease:%20genetics&author=MJ%20Ombrello&author=DL%20Kastner&author=EF%20Remmers&volume=27&publication_year=2015&pages=349&pmid=26002026&doi=10.1097/BOR.0000000000000189&)

14. Baldini M, Lohman IC, Halonen M, Erickson RP, Holt PG, Martinez FD: A Polymorphism* in the 5’ flanking region of the CD14 gene is associated with circulating soluble CD14 levels and with total serum immunoglobulin E. Am J Respir Cell Mol Biol 1999;20:976.  [DOI](https://doi.org/10.1165/ajrcmb.20.5.3494) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10226067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Cell%20Mol%20Biol&title=A%20Polymorphism*%20in%20the%205%E2%80%99%20flanking%20region%20of%20the%20CD14%20gene%20is%20associated%20with%20circulating%20soluble%20CD14%20levels%20and%20with%20total%20serum%20immunoglobulin%20E&author=M%20Baldini&author=IC%20Lohman&author=M%20Halonen&author=RP%20Erickson&author=PG%20Holt&volume=20&publication_year=1999&pages=976&pmid=10226067&doi=10.1165/ajrcmb.20.5.3494&)

15. Hunt PW, Sinclair E, Rodriguez B, Shive C, Clagett B, Funderburg N et al. : Gut epithelial barrier dysfunction and innate immune activation predict mortality in treated HIV infection. J Infect Dis 2014;210:1228.  [DOI](https://doi.org/10.1093/infdis/jiu238) | [PMC free article](/articles/PMC4192038/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24755434/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=Gut%20epithelial%20barrier%20dysfunction%20and%20innate%20immune%20activation%20predict%20mortality%20in%20treated%20HIV%20infection&author=PW%20Hunt&author=E%20Sinclair&author=B%20Rodriguez&author=C%20Shive&author=B%20Clagett&volume=210&publication_year=2014&pages=1228&pmid=24755434&doi=10.1093/infdis/jiu238&)

16. Sandler NG, Wand H, Roque A, Law M, Nason MC, Nixon DE et al. : Plasma levels of soluble CD14 independently predict mortality in HIV infection. J Infect Dis 2011;203:780.  [DOI](https://doi.org/10.1093/infdis/jiq118) | [PMC free article](/articles/PMC3071127/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21252259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect%20Dis&title=Plasma%20levels%20of%20soluble%20CD14%20independently%20predict%20mortality%20in%20HIV%20infection&author=NG%20Sandler&author=H%20Wand&author=A%20Roque&author=M%20Law&author=MC%20Nason&volume=203&publication_year=2011&pages=780&pmid=21252259&doi=10.1093/infdis/jiq118&)

17. Kuiper JJW, Setten JV, Devall M, Cretu-Stancu M, Hiddingh S, Ophoff RA et al. : Functionally distinct ERAP1 and ERAP2 are a hallmark of HLA-A29-(Birdshot) Uveitis. Hum Mol Genet 2018;27:4333.  [DOI](https://doi.org/10.1093/hmg/ddy319) | [PMC free article](/articles/PMC6276832/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30215709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Functionally%20distinct%20ERAP1%20and%20ERAP2%20are%20a%20hallmark%20of%20HLA-A29-(Birdshot)%20Uveitis&author=JJW%20Kuiper&author=JV%20Setten&author=M%20Devall&author=M%20Cretu-Stancu&author=S%20Hiddingh&volume=27&publication_year=2018&pages=4333&pmid=30215709&doi=10.1093/hmg/ddy319&)

18. Kirino Y, Bertsias G, Ishigatsubo Y, Mizuki N, Tugal-Tutkun I, Seyahi E et al. : Genome-wide association analysis identifies new susceptibility loci for Behcet’s disease and epistasis between HLA-B*51 and ERAP1. Nat Genet 2013;45:202.  [DOI](https://doi.org/10.1038/ng.2520) | [PMC free article](/articles/PMC3810947/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23291587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Genome-wide%20association%20analysis%20identifies%20new%20susceptibility%20loci%20for%20Behcet%E2%80%99s%20disease%20and%20epistasis%20between%20HLA-B*51%20and%20ERAP1&author=Y%20Kirino&author=G%20Bertsias&author=Y%20Ishigatsubo&author=N%20Mizuki&author=I%20Tugal-Tutkun&volume=45&publication_year=2013&pages=202&pmid=23291587&doi=10.1038/ng.2520&)

19. Alvarez-Navarro C, Martin-Esteban A, Barnea E, Admon A, Lopez de Castro JA: Endoplasmic Reticulum Aminopeptidase 1 (ERAP1) Polymorphism Relevant to Inflammatory Disease Shapes the Peptidome of the Birdshot Chorioretinopathy-Associated HLA-A*29:02 Antigen. Mol Cell Proteomics 2015;14:1770.  [DOI](https://doi.org/10.1074/mcp.M115.048959) | [PMC free article](/articles/PMC4587329/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25892735/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Proteomics&title=Endoplasmic%20Reticulum%20Aminopeptidase%201%20(ERAP1)%20Polymorphism%20Relevant%20to%20Inflammatory%20Disease%20Shapes%20the%20Peptidome%20of%20the%20Birdshot%20Chorioretinopathy-Associated%20HLA-A*29:02%20Antigen&author=C%20Alvarez-Navarro&author=A%20Martin-Esteban&author=E%20Barnea&author=A%20Admon&author=JA%20Lopez%20de%20Castro&volume=14&publication_year=2015&pages=1770&pmid=25892735&doi=10.1074/mcp.M115.048959&)

20. Lopez de Castro JA, Alvarez-Navarro C, Brito A, Guasp P, Martin-Esteban A, Sanz-Bravo A: Molecular and pathogenic effects of endoplasmic reticulum aminopeptidases ERAP1 and ERAP2 in MHC-I-associated inflammatory disorders: Towards a unifying view. Mol Immunol 2016;77:193.  [DOI](https://doi.org/10.1016/j.molimm.2016.08.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27522479/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Immunol&title=Molecular%20and%20pathogenic%20effects%20of%20endoplasmic%20reticulum%20aminopeptidases%20ERAP1%20and%20ERAP2%20in%20MHC-I-associated%20inflammatory%20disorders:%20Towards%20a%20unifying%20view&author=JA%20Lopez%20de%20Castro&author=C%20Alvarez-Navarro&author=A%20Brito&author=P%20Guasp&author=A%20Martin-Esteban&volume=77&publication_year=2016&pages=193&pmid=27522479&doi=10.1016/j.molimm.2016.08.005&)

21. Martin-Esteban A, Guasp P, Barnea E, Admon A, Lopez de Castro JA: Functional Interaction of the Ankylosing Spondylitis-Associated Endoplasmic Reticulum Aminopeptidase 2 With the HLA-B*27 Peptidome in Human Cells. Arthritis Rheumatol 2016;68:2466.  [DOI](https://doi.org/10.1002/art.39734) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27110896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheumatol&title=Functional%20Interaction%20of%20the%20Ankylosing%20Spondylitis-Associated%20Endoplasmic%20Reticulum%20Aminopeptidase%202%20With%20the%20HLA-B*27%20Peptidome%20in%20Human%20Cells&author=A%20Martin-Esteban&author=P%20Guasp&author=E%20Barnea&author=A%20Admon&author=JA%20Lopez%20de%20Castro&volume=68&publication_year=2016&pages=2466&pmid=27110896&doi=10.1002/art.39734&)

22. Chang SC, Momburg F, Bhutani N, Goldberg AL: The ER aminopeptidase, ERAP1, trims precursors to lengths of MHC class I peptides by a “molecular ruler” mechanism. Proc Natl Acad Sci U S A 2005;102:17107.  [DOI](https://doi.org/10.1073/pnas.0500721102) | [PMC free article](/articles/PMC1287962/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16286653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=The%20ER%20aminopeptidase,%20ERAP1,%20trims%20precursors%20to%20lengths%20of%20MHC%20class%20I%20peptides%20by%20a%20%E2%80%9Cmolecular%20ruler%E2%80%9D%20mechanism&author=SC%20Chang&author=F%20Momburg&author=N%20Bhutani&author=AL%20Goldberg&volume=102&publication_year=2005&pages=17107&pmid=16286653&doi=10.1073/pnas.0500721102&)
