# Heterologous Expression and Functional Characterization of Novel CYP2C9 Variants Identified in the Alaska Native People

## Metadata
**Authors:** Matthew G McDonald, Lindsay M Henderson, Sutapa Ray, Catherine K Yeung, Amanda L Johnson, John P Kowalski, Helmut Hanenberg, Constanze Wiek, Kenneth E Thummel, Allan E Rettie
**Journal:** The Journal of Pharmacology and Experimental Therapeutics
**Date:** 2020 Aug
**DOI:** [10.1124/jpet.120.265850](https://doi.org/10.1124/jpet.120.265850)
**PMID:** 32423989
**PMCID:** PMC7366286
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7366286/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7366286/pdf/jpet.120.265850.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7366286/pdf/jpet.120.265850.pdf)

## Abstract

**SIGNIFICANCE STATEMENT:** 
Novel gene variants of CYP2C9—M1L, and N218I, along with P279T (CYP2C9*29)—are expressed in Alaska Native people at relatively high frequencies. In vitro characterization of their functional effects revealed that each variant confers reduced catalytic efficiency toward several substrates, including the low–therapeutic index drugs (S)-warfarin and phenytoin. These data provide the first functional information for new, common CYP2C9 variants in this understudied population. The data may help guide dose adjustments in allele carriers, thus mitigating potential healthcare disparities.

### SIGNIFICANCE STATEMENT

Novel gene variants of *CYP2C9—M1L*, and *N218I*, along with *P279T* (*CYP2C9*29*)—are expressed in Alaska Native people at relatively high frequencies. In vitro characterization of their functional effects revealed that each variant confers reduced catalytic efficiency toward several substrates, including the low–therapeutic index drugs (*S*)-warfarin and phenytoin. These data provide the first functional information for new, common CYP2C9 variants in this understudied population. The data may help guide dose adjustments in allele carriers, thus mitigating potential healthcare disparities.

## Introduction

CYP2C9 is a major form of hepatic cytochrome P450 in terms of both liver content ([Zhang et al., 2016](#B32)) and the proportion of therapeutic agents metabolized by this isoform ([Isvoran et al., 2017](#B14)). CYP2C9 is also a contributor to first-pass metabolism in the gut ([Paine et al., 2006](#B23)). As such, genetic variability in *CYP2C9* has important ramifications for patients treated with drugs that are cleared by this enzyme. Although gene duplications/deletion for *CYP2C9* have not been reported, an array of single nucleotide–coding polymorphisms that confer substantial functional deficits to the enzyme that alter drug elimination and response are known ([Daly et al., 2017](#B5)). The most widely studied of these are *CYP2C9*2* and **3*, which are present at high allele frequencies in people of European ancestry but are much less prevalent in other ethnic populations ([Zhou et al., 2017](#B33)). *CYP2C9*2* is essentially absent in East Asians, and functionally defective *CYP2C9*5*, **6*, **8*, and **11* alleles are the predominant reduced or loss-of-function alleles in people of African ancestry. The clinical impact of the latter alleles is often overlooked. For example, they were not included in the analysis for the recent United States–wide Clarification of Optimal Anticoagulation through Genetics (COAG) randomized clinical trial ([Kimmel et al., 2013](#B15)), which was intended to test whether pharmacogenetic information could improve clinical outcomes for patients receiving warfarin anticoagulation therapy. A later study that did account for these alleles demonstrated improved outcomes for African Americans being treated with warfarin ([Drozda et al., 2015](#B6)). These findings demonstrate the need to consider population-selective alleles in pharmacogenetic studies to avoid healthcare disparities in under-represented populations ([Henderson et al., 2018](#B10)).

We recently described the presence of novel and rare CYP2C9 variants in Alaska Native (AN) people at allele frequencies in excess of 1% ([Fohner et al., 2015](#B8)). The altered initiation start site allele, *Met1Leu* (*M1L*, rs114071557), was identified in multiple AN cultural groups (e.g., Inupiaq, Yup’ik, Athabascan, and Aleut), with the highest minor allele frequency (MAF) of 6.3% in Yup’ik people of the Yukon-Kuskokwim delta. In the same AN subpopulation, *Asn218Ile* (*N218I*, rs780801862) and *Pro279Thr* (*P279T*, rs182132442) alleles were present with frequencies of 3.8% and 2.1%, respectively, and *Asn218Ile* was found at an MAF of 5.4% in Athabascan people from the interior of Alaska receiving healthcare at Southcentral Foundation in Anchorage ([Fohner et al., 2015](#B8)). The *Pro279Thr* allele (*CYP2C9*29*) was first detected in Japanese subjects with a very low MAF of 0.2% ([Maekawa et al., 2006](#B19)). When expressed in COS or insect cells, CYP2C9.29 exhibited catalytic efficiencies for (*S*)-warfarin, flurbiprofen, and phenytoin metabolism that were maintained within a factor of two relative to wild-type CYP2C9 ([Niinuma et al., 2014](#B22); [Wang et al., 2014](#B28), [2015](#B27); [Chen et al., 2016](#B3)). However, to our knowledge, no functional data exist for the Asn218Ile and Met1Leu protein variants.

This manuscript reports on the heterologous expression of CYP2C9.29 and the Asn218Ile and Met1Leu protein variants in lentivirus-infected HepG2 cells and *Escherichia coli* for functional characterization. The infected HepG2 cells were used to assess RNA and protein expression in mammalian cells, whereas *E. coli* was used for high-level expression suitable for purification and detailed catalytic studies with several CYP2C9 substrates after reconstitution with required coenzymes and lipids.

## Materials and Methods

### 

#### General Reagents.

Racemic 7-hydroxywarfarin-d_5_ ([Bush and Trager, 1983](#B1)), unlabeled phenytoin, and 5-(p-hydroxyphenyl)-5-phenylhydantoin (p-HPPH) were obtained as described previously ([Mosher et al., 2009](#B21)). 5-(p-Hydroxyphenyl)-5-phenylhydantoin-d_5_ (p-HPPH-d_5_), unlabeled flurbiprofen, 4′-OH-flurbiprofen, (*S*)-naproxen, and racemic 6-*O*-desmethyl naproxen-d_3_ were purchased from Toronto Research Chemicals (Ontario, Canada). 1,2-Dilauroyl-sn-glycero-3-phosphocholine (DLPC) was procured from Avanti Polar Lipids, Inc. (Alabaster, AL), and unilamellar vesicles (i.e., liposomes) were freshly prepared immediately prior to their use in incubations with purified CYP2C9 protein by repeatedly passing a 1 mM DLPC solution (in 100 mM potassium phosphate buffer, pH 7.4) through a 0.1-μm nucleopore membrane using a miniextruder apparatus according to the manufacturer’s protocol (Avanti). Recombinant CYP2C9 Supersomes, coexpressed in insect cells with cytochrome P450 oxidoreductase (CPR) and cytochrome b5, were purchased from Corning Life Sciences, Inc. (Corning, NY). The CPR and cytochrome b5 used in purified CYP2C9 variant protein incubations were expressed and purified according to established protocols ([Chen et al., 1998](#B4)). Organic solvents and buffer salts (potassium phosphates, NaCl, etc.) were obtained from Fisher Scientific (Fair Lawn, NJ), and all other chemicals (warfarin, phenytoin, 6-*O*-desmethyl-(*S*)-naproxen, NADPH, etc.) were purchased from Sigma-Aldrich (St. Louis, MO).

#### HepG2 Cells Stably Expressing CYP2C9 Variants.

HepG2 cells were transduced with lentiviral constructs for *CYP2C9* wild-type (WT), M1L, N218I, P279T, I359L, and EGFP, a mock-transduced negative control. The *CYP2C9* WT open reading frame was PCR-amplified from our original WT pFB2C9 vector ([Haining et al., 1996](#B9)) and cloned into the lentiviral transfer vector puc2CL12wo. The mutated *CYP2C9* variants were generated by overlap extension PCR using the mutagenesis primers listed below. Constructs were verified by Sanger sequencing. Lentivirus production and cell transduction were performed as previously described ([Wiek et al., 2015](#B30); [Roellecke et al., 2016](#B24)).

M1L forward: 5′-GCG​TCG​ACG​CCA​CCt​tgG​ATT​CTC​TTG

M1L reverse: 5′-CAA​GAG​AAT​Cca​aGG​TGG​CGT​CGA​CGC

N218I forward: 5′-CCC​CTG​GAT​CCA​GAT​CTG​CAA​Tat​tTT​TTC​TCC​TAT​CAT

N218I reverse: 5′-ATG​ATA​GGA​GAA​AAa​atA​TTG​CAG​ATC​TGG​ATC​CAG​GGG

P279T forward: 5′-GGA​GAA​GGA​AAA​GCA​CAA​CCA​Aac​aTC​TGA​ATT​TAC​TAT​TG

P279T reverse: 5′-CAA​TAG​TAA​ATT​CAG​Atg​tTT​GGT​TGT​GCT​TTT​CCT​TCT​CC

I359L forward: 5′-GGT​CCA​GAG​ATA​Cct​tGA​CCT​TCT​CCC​C

I359L reverse: 5′-GGG​GAG​AAG​GTC​aag​GTA​TCT​CTG​GAC​C

#### RNA and Protein Quantitation in HepG2 Cells.

HepG2 cell lysates were prepared for each cell line, RNA was isolated, and gene expression was determined by reverse-transcription PCR. Each sample was normalized to its own 18S ribosomal RNA expression, and relative fold change was determined by comparing expression levels to WT. Thirty-microgram protein aliquots of HepG2 cell lysates were separated on a 4%–12% 2-[bis(2-hydroxyethyl)amino]-2-(hydroxymethyl)propane-1,3-diol (bis-Tris) SDS polyacrylamide gel (Invitrogen) and transferred electrophoretically to nitrocellulose. After blocking for 2.5 hours with 5% w/v bovine serum albumin/5% w/v nonfat dry milk in PBS in 0.1% Triton X-100, the nitrocellulose was reacted overnight with a 1:1000 dilution of a rabbit anti-human CYP2C9 primary antibody generated previously in our laboratory ([Koukouritaki et al., 2004](#B16)), followed by incubation with a secondary anti-rabbit IgG and horseradish peroxidase–linked antibody (Cell Signaling Technology). The nitrocellulose was then stripped and reprobed with an actin antibody for visualization of the positive loading control.

#### E. coli Expression Vector Construction.

*CYP2C9* and the *M1L*, *N218I*, and *P279T* gene variants were cloned into pCWori+ for expression and purification from *E. coli.* The *M1L* and *N218I* variants were constructed using in vitro site-directed mutagenesis according to the manufacturer’s protocol (Stratagene, San Diego, CA) and the primers listed below. The *P279T* gene variant was constructed using overlap extension PCR ([Simionatto et al., 2009](#B25)). The C-terminal flanking primer introduced a His tag and SalI restriction site. The N-terminal primer possessed a Barnes modification and Nde1 restriction site.

*M1L*_forward: 5′-TCC​ATC​GAT​GCT​TAG​GAG​GTC​ATt​tgG​CTC​TGT​TAT

*N219I_*forward: 5′-CCC​CTG​GAT​CCA​GAT​CTG​CAA​Tat​tTT​TTC​TCC

*P279T_*forward: 5′-GGA​GAA​GGA​AAA​GCA​CAA​CCA​Aac​gTC​TGA​ATT​TAC​TAT​TG

N-terminal flanking primer: 5′-CCA​GCC​CAT​ATG​GCT​CTG​TTA​TTA​GCA​G

*M1L_*reverse: 5′-ATA​ACA​GAG​Cca​aAT​GAC​CTC​CTA​AGC​ATC​GAT​GGA

*N219I_*reverse: 5′-GGA​GAA​AAa​atA​TTG​CAG​ATC​TGG​ATC​CAG​GGG

*P279T_*reverse: 5′-CAA​TAG​TAA​ATT​CAG​Acg​tTT​GGT​TGT​GCT​TTT​CCT​TCT​CC

C-terminal flanking primer: 5′-ACA​CCA​GGT​CGA​CAC​AGG​AAT​GAA​GCA​CAG​C

PCR products were cloned and transformed into DH5*α* cells as described previously ([Mosher et al., 2008](#B20), [2009](#B21)). All base changes were verified by DNA sequencing.

#### Bacterial Expression and Purification of CYP2C9 Variants.

The expression and purification of WT, N218I, and P279T protein variants on nickel-nitrilotriacetic acid Superflow resin (Qiagen, Valencia, CA) were performed as described previously ([Cheesman et al., 2003](#B2)). Purified proteins were dialyzed (3 × 1 l) with a buffer containing 20% glycerol and 1 mM EDTA in 100 mM potassium phosphate (pH 7.4) and were then concentrated in a centrifugal concentrator (Amicon Ultra 30K NMWL; Millipore Sigma, Burlington, MA).

#### Quantitative CO Binding Assay.

Carbon monoxide binding spectra by reduced CYP2C9 protein variants were recorded on an Olis modernized Aminco DW-2 spectrophotometer (Olis, Bogart, GA). Purified protein was diluted 10-fold in 100 mM potassium phosphate buffer, pH 7.4. Sodium dithionite (excess, powder) was added, and the reduced protein mix was then split between sample and reference quartz cuvettes before taking a baseline scan from 400 to 500 nm at 25°C. Carbon monoxide gas was bubbled to saturation through the sample cuvette, and difference scans were taken of the CO-bound enzyme. Holo-CYP2C9 variant concentrations were calculated according to Beer’s law, taking the absorbance difference between the peak maximum (at ∼450 nm) and the baseline (at 490 nm) and using an extinction coefficient of 91 mM^−1^cm^−1^.

#### Tryptic Digest Analysis of Recombinant Protein.

Purified CYP2C9 proteins (500 pmol) were diluted to a 1.25 μM final concentration in Tris buffer (50 mM Tris-HCl, pH 7.6, with 1 mM CaCl_2_). Sequencing-grade modified trypsin (Promega, Madison, WI) was added (2.3 μg for an approximate protein:trypsin ratio of 5:1), and the samples were incubated at 37°C/60 rpm for 3 hours in a water bath. Reactions were then quenched with the addition of trichloroacetic acid (to 10% of final volume), and precipitate was removed by centrifugation. Supernatants were analyzed by high-resolution LC-MS.

LC-MS analysis was carried out on a Thermo Linear Trap Quadrupole Orbitrap (LTQ-OT) Xcalibur 2.0 DS mass spectrometer (Thermo Electron Corp, San Jose, CA) coupled to an ACQUITY Ultra Performance Liquid Chromatography (UPLC) System with integral autoinjector (Waters, Milford, MA). The MS was run in scan mode, utilizing positive electrospray ionization with a resolution power of at least 30,000. Peptides from the tryptic enzyme digests were separated on an ACQUITY BEH C_18_, 1.7 μm, 2.1 × 50 mm UPLC column (Waters, Corp) using a solvent gradient of 0.1% aqueous formic acid (solvent A) and acetonitrile (solvent B). Solvent was maintained at 3% B for 1 minute and was then increased linearly to 95% B over 5 minutes. Data analysis was performed on Excalibur 2.0 SR1 software.

#### Enzyme Reconstitution and Assay Incubation Conditions.

Master protein mixes were prepared from concentrated stock solutions of purified CYP2C9 variant enzyme (one molar equivalent), CPR (two molar equivalents), and DLPC (160 molar equivalents—added from a freshly extruded 1 mM aqueous stock solution). Mixtures were incubated on ice for 30 minutes and then diluted with reaction buffer (100 mM potassium phosphate, pH 7.4). One molar equivalent of cytochrome b5 was added, and the mixtures were incubated on ice for an additional 5 minutes. The master mixes were then aliquoted for individual incubation reactions, and substrate was added. A standard incubation reaction contained 10 pmol of CYP2C9 variant, 20 pmol CPR, 10 pmol cytochrome b5, 1 mM NADPH, and substrate at a variable concentration (added from 100× concentrated stock solutions made up in 1:1 methanol:water) in a 250-μl final incubation volume. For kinetic experiments, a substrate concentration range of 100 nM to 100 μM was used for (*S*)-warfarin, 1–250 μM for phenytoin, 1–300 μM for flurbiprofen, and 5–1800 μM for (*S*)-naproxen. Enzyme and substrate were preincubated at 37°C/70 rpm in a shaking water bath for 3 minutes prior to initiation of the reaction with addition of NADPH (1 mM final concentration). Twenty minutes later, warfarin and phenytoin incubations were quenched with addition of 10 μl of ice-cold 70% HClO_4_, and internal standards were added (10 pmol of racemic 7-hydroxywarfarin-d_5_ was added for warfarin metabolic experiments, whereas 10 pmol of p-HPPH-d_5_ was added to incubations in which phenytoin was used as the substrate). Flurbiprofen and (*S*)-naproxen incubations were quenched with the addition of 1 ml of ice-cold methanol containing 2% formic acid after 20 minutes of incubation, and then internal standards were added [200 ng of 6-*O*-desmethyl-(*S*)-naproxen was added for flurbiprofen metabolic experiments, whereas 200 ng of racemic 6-*O*-desmethyl naproxen-d_3_ was added to incubations in which (*S*)-naproxen was used as substrate]. The reaction mixtures were centrifuged to remove protein, and supernatants were analyzed either by liquid chromatography-tandem mass spectrometry for (*S*)-warfarin and phenytoin or by LC-MS for flurbiprofen and (*S*)*-*naproxen metabolic reactions.

Reactions with CYP2C9 Supersomes contained 10 pmol P450 enzyme, substrate (at variable concentration), and 1 mM NADPH in 250 μl of 100 mM potassium phosphate buffer, pH 7.4. Incubation and workup conditions were identical to those described above for the purified CYP2C9 variants.

Calibration curves for the quantitation of (*S*)-warfarin and phenytoin metabolites were prepared by spiking variable amounts of either unlabeled racemic 7-hydroxywarfarin or unlabeled p-HPPH, respectively, into 250-μl volumes of potassium phosphate buffer to generate standard mixtures with final metabolite concentrations between 1 and 1000 nM. Calibration curves for flurbiprofen and (*S*)-naproxen metabolites were prepared by spiking variable amounts of, respectively, either the unlabeled 4-hydroxy flurbiprofen or the unlabeled 6-*O*-desmethyl-(*S*)-naproxen into 250-μl volumes of potassium phosphate buffer to generate standard mixtures with final concentrations between 0.1 and 30 μM. These standard solutions, prepared in duplicate for each metabolite concentration, were immediately worked up and analyzed in an identical fashion to that described for the enzyme incubation samples above.

#### Analysis of (S)-Warfarin, Phenytoin, (S)-Naproxen, and Flurbiprofen Metabolites by LC-MS.

Liquid chromatography-tandem mass spectrometry analyses were conducted on a Waters Xevo TQ-S Tandem Quadrupole Mass Spectrometer (Waters Co.) coupled to an ACQUITY UPLC System with integral autoinjector (Waters). The Xevo was operated in ESI^+^-MS/MS (selected reaction monitoring) mode at a source temperature of 150°C and a desolvation temperature of 350°C. The following mass transitions were monitored in separate ion channels for the warfarin metabolite/standard: *m/z* 325 > 179 (7-hydroxywarfarin-d_0_) and *m/z* 330 > 179 (7-hydroxywarfarin-d_5_); and phenytoin metabolite/standard: *m/z* 269 > 198 (p-HPPH-d_0_) and *m/z* 274 > 203 (p-HPPH-d_5_). Optimized cone voltages and collision energies were set to 25 V and 15 eV for 7-hydroxywarfarin (both d_0_- and d_5_-labeled) and 35 V and 15 eV for p-HPPH (d_0_- and d_5_-labeled), respectively. Metabolic products from the (*S*)-warfarin incubations were separated on an Acquity BEH Phenyl, 1.7 μm, 2.1 × 150 mm UPLC column (Waters, Corp) using an isocratic gradient of 45% solvent A (0.1% aqueous formic acid) and 55% solvent B (methanol), with a constant flow rate of 0.35 ml/min. Phenytoin metabolites were separated using this same BEH Phenyl UPLC column with a solvent gradient of water (solvent A) and acetonitrile (solvent B), both of which contained 0.1% formic acid, running at a flow rate of 0.3 ml/min. Initially, solvent B was set to 28%, where it was maintained for 4.5 minutes, and then increased linearly to 95% over 0.5 minutes, where it was left for an additional 1.5 minutes. Metabolites were quantified through comparison of their peak area ratios (relative to either the racemic 7-hydroxywarfarin-d_5_ or p-HPPH-d_5_ internal standard peak area) to calibration curves using linear regression analysis. The limit of detection for both metabolites was below 5 fmol injected on column.

Flurbiprofen and (*S*)*-*naproxen metabolite concentrations were assessed by LC-MS on an Agilent 1956B single quadrupole mass spectrometer coupled with an Agilent 1200 series (Santa Clara, CA) liquid chromatography system. Chromatographic separation was achieved on a Luna C_18_ (2 mm × 50 mm × 5 µm) column (Torrence, CA) with a mobile phase flow rate of 0.3 ml/min. The mobile phase consisted of 10 mM ammonium formate (solvent A, pH 3.5) and methanol (solvent B), and linear gradients were applied with %B increasing from 40% to 80% between 3 and 8 minutes, then decreasing to 40% at 9 minutes. Column temperature was maintained at 35°C. The electrospray ionization source of the mass spectrometer was operated in positive ion mode. Quantitation was achieved by selected ion monitoring of the following ion channels: *m/z* = 278.1 for 4-hydroxy flurbiprofen, *m/z* = 234.1 for 6-*O*-desmethyl-(*S*)-naproxen, and *m/z* = 237.1 for racemic 6-O-desmethyl-(*S*)-naproxen-d_3_. Data acquisition and analyses were performed using the Agilent MassHunter software. Calibration curves were constructed by plotting the peak area ratio of each compound to the respective internal standard against a range of concentrations.

#### Data Analysis.

Kinetic experiments for the assessment of CYP2C9 probe substrate metabolism were performed in triplicate, with duplicate data points for each substrate concentration. GraphPad Prism v.6 was used to estimate K_m_ and *V*_max_ parameters. Data presented are means ± S.D. of the three separate determinations unless otherwise stated.

## Results

### 

#### Transcript Level and Protein Expression of CYP2C9 Variants in HepG2 Cells.

Robust mRNA expression of *CYP2C9* wild-type and the *M1L*, *N218I*, *P279T*, and *I359L* gene variants in lentivirus-infected HepG2 cells was obtained ([Fig. 1](#F1)), generally with corresponding protein levels, as assessed by Western blot ([Fig. 2](#F2)). However, as expected because of disruption of the *CYP2C9* start codon, the M1L variant protein was undetectable. Comparison of transcript levels with Western blot intensities relative to an actin loading control revealed that the WT and N218I variants were expressed at similar levels. The P279T variant was expressed at a lower transcript level, but with a correspondingly lower protein level. Only the I359L variant showed any evidence of low protein expression relative to its transcript level.

![Fig. 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/543e/7366286/6d3ead94e5b6/jpet.120.265850f1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7366286_jpet.120.265850f1.jpg)

Transcript analysis of CYP2C9 variants expressed in HepG2 cells. Gene expression was determined by reverse-transcription polymerase chain reaction. Each sample was normalized to its own 18S ribosomal RNA expression, and relative fold change was determined by comparing expression levels to WT, which was set to 1.0. Values are the means of two independent experiments.

![Fig. 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/543e/7366286/83b4b933eff1/jpet.120.265850f2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7366286_jpet.120.265850f2.jpg)

Western blot of CYP2C9 variants expressed in HepG2 cells. Thirty micrograms of each variant preparation was boiled in SDS/β-mercaptoethanol, loaded into 4% acrylamide, and run on a 10% resolving gel. EGFP and I359L preparations were included as negative and positive controls, respectively. After transfer to nitrocellulose, CYP2C9 and β-actin were visualized using a polyclonal and monoclonal antibody, respectively. CYP2C9 protein expression level was calculated by measuring the intensities of the CYP2C9 protein bands at ∼55 kDa and dividing by the intensities of the actin bands at ∼42 kDa (used as a loading control). Densitometry measurements were performed using ImageJ. The CYP2C9 WT sample was set to 1.00. Relative expression levels were 0 (EGFP), 0 (M1L), 1.15 (N218I), 0.28 (P279T), and 0.38 (I359L). The experiment was repeated once, with similar results.

#### Expression and Purification of Wild-type CYP2C9 and Variants from E. coli.

Wild-type CYP2C9 and the N218I and P279T variants were each readily expressed in *E. coli*. After affinity purification, CYP2C9 variant concentrations were determined by UV-visible spectroscopy using a quantitative CO binding assay. Upon binding carbon monoxide, all three enzymes showed an identical *λ*_max_ at 449 nm. The final protein preparations were highly pure as assessed by SDS-PAGE ([Fig. 3](#F3)).

![Fig. 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/543e/7366286/d9c6cde335e5/jpet.120.265850f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7366286_jpet.120.265850f3.jpg)

SDS-PAGE of purified CYP2C9 variants. CYP2C9 variants expressed in E. coli and purified by nickel affinity chromatography. Ten picomoles of each variant was boiled in SDS/β-mercaptoethanol, loaded into 4% acrylamide, and run on a 10% resolving gel. CYP2C9 monomer migrates around 50 kDa. The high–molecular mass band around 100 kDa is likely a dimer.

#### Tryptic Digest Analysis of Purified Proteins.

As a quality control step, the purified CYP2C9 proteins were subjected to digestion with trypsin, and the peptide products from these reactions were analyzed by high-resolution ESI^+^ LC-MS ([Fig. 4](#F4)). Substitution of a proline for a threonine at residue 279 will result in an increase of 4 Da in the tryptic peptide fragment containing amino acid residues 276–307. By ESI^+^ MS, it is apparent that the most abundant configuration of the 276–307 peptide fragment carries a charge of z = +3 (i.e., M + 3H); thus, the experimental increase in this peptide’s mass-to-charge ratio for the P279T variant (monoisotopic *m/z* = 1157.5568) in comparison with wild type (*m/z* = 1156.2251) matches the expected increase of 1.33 Da for this mutation. Similarly, replacing an asparagine with an isoleucine at residue 218 will result in a decrease of just under 1 Da in the mass of the tryptic peptide fragment containing amino acid residues 207–232. Thus, for the M + 3H configuration of the peptide (z = +3), the observed change in mass-to-charge ratio for the N218I variant (monoisotopic *m/z* = 1001.8541) in comparison with wild type (*m/z* = 1002.1725) again matches the expected decrease of 0.32 Da for this mutation.

![Fig. 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/543e/7366286/e54c0ccbcb7b/jpet.120.265850f4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7366286_jpet.120.265850f4.jpg)

High-resolution electrospray mass spectrometric analysis of variant CYP2C9 tryptic peptides. High-resolution ESI+ mass isotope distribution envelopes for peptides (with sequences identified at the top of the figures) derived from the tryptic digest of purified CYP2C9 WT (A and B), N218I (C), and P279T (D) protein variants. Variant amino acids within the peptides are identified in bold text, with the amino acid mutations shown in parentheses. Each of the four figures (A–D) shows the mass envelope for the triply charged version of the respective peptide (i.e., M + 3H, z = +3), and the calculated monoisotopic mass-to-charge ratio for each peptide is shown in brackets directly above the experimental m/z value.

#### Comparison of Probe Substrate Metabolism by CYP2C9 Variant Enzymes.

Substrate saturation kinetic experiments were performed in triplicate for the three enzyme variants using a reconstituted system that consisted of purified CYP2C9 protein, CPR, cytochrome b5, and DLPC lipid ([Fig. 5](#F5)). K_m_ and *V*_max_ values were determined for each variant using a selection of CYP2C9-specific probe substrates with both high [(*S*)-naproxen, flurbiprofen] and low [(*S*)-warfarin, phenytoin] turnover rates ([Table 1](#T1)). The same ordering of specific activities for the three enzymes was maintained across all four probe substrates, with the P279T variant showing a low to moderate decrease in catalytic efficiency (59%–87% of WT activity, depending on the substrate), and the N218I variant showing a much more significant decrease (14%–38% of WT activity). Interestingly, although the differences in overall catalytic efficiencies for the variants were similar across all four substrate probes, the individual components of K_m_ and *V*_max_ varied. P279T metabolized (*S*)-warfarin to 7-hydroxy-(*S*)-warfarin, with a K_m_ value that was essentially identical to the WT protein, but showed both an increase in K_m_ for phenytoin and flurbiprofen hydroxylation and a decrease in K_m_ for (*S*)-naproxen *O*-desmethylation in comparison with the WT protein. Likewise, although the *V*_max_ values for (*S*)-warfarin, phenytoin, and flurbiprofen metabolism by P279T were all very close, although slightly lower than the WT rate, the turnover rate for (*S*)-naproxen dealkylation by P279T was approximately half that of WT enzyme. In contrast, the N218I variant showed a similar apparent binding effect across all four substrates, with K_m_ increasing by 1.6- to 2.3-fold compared with WT. However, although the *V*_max_ values for (*S*)-warfarin, phenytoin, and (*S*)-naproxen metabolism by N218I were all similarly low (i.e., less than 50%), there was only a 15% reduction in the flurbiprofen 4′-hydroxylation rate compared with 2C9 WT.

![Fig. 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/543e/7366286/d71593e50c56/jpet.120.265850f5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7366286_jpet.120.265850f5.jpg)

Kinetic analysis of (S)-warfarin (A), phenytoin (B), flurbiprofen (C), and (S)-naproxen (D) metabolism by CYP2C9 WT, N218I, and P279T. Reaction velocities for metabolite formation were plotted against substrate concentration and the kinetic parameters Vmax and Km extracted using GraphPad Prism. The values are presented in Table 1. For (B), p-HPPH is 5-(p-hydroxyphenyl)-5-phenylhydantoin.

| Metabolite | CYP2C9 Variant | Km (μM) | Vmax (pmol/min per picomole) | Cat. Eff. (μl/min per nanomole) | Cat. Eff. (%) |
| --- | --- | --- | --- | --- | --- |
|   | WT | 10.7 ± 0.4 | 0.153 ± 0.021 | 14.3 ± 1.8 | 100 |
| 7-OH-Warf | P279T | 10.7 ± 1.2 | 0.133 ± 0.015 | 12.4 ± 0.2 | 87 |
|   | N218I | 17.1 ± 0.9 | 0.060 ± 0.002 | 3.5 ± 0.3 | 24 |
|   | WT | 27.4 ± 4.7 | 0.204 ± 0.008 | 7.6 ± 1.2 | 100 |
| p-HPPH | P279T | 34.4 ± 1.5 | 0.171 ± 0.013 | 5.0 ± 0.5 | 66 |
|   | N218I | 47.9 ± 3.2 | 0.050 ± 0.006 | 1.1 ± 0.2 | 14 |
|   | WT | 26.0 ± 0.40 | 17.7 ± 0.53 | 680 ± 10 | 100 |
| 4′-OH-Flurb | P279T | 39.0 ± 2.4 | 16.8 ± 2.4 | 430 ± 39 | 63 |
|   | N218I | 59.8 ± 0.25 | 15.3 ± 0.28 | 260 ± 3.7 | 38 |
|   | WT | 419 ± 59 | 15.3 ± 0.72 | 37 ± 3.5 | 100 |
| DM-Nap | P279T | 310 ± 80 | 6.3 ± 0.25 | 22 ± 4.2 | 59 |
|   | N218I | 725 ± 106 | 6.6 ± 1.1 | 9.1 ± 0.2 | 25 |

Table Caption: Kinetics of probe substrate metabolism by purified CYP2C9 variant enzymes. Cytochrome P450 enzymes reconstituted with cytochrome P450 reductase, cytochrome b5, and lipid were incubated with multiple concentrations of (S)-warfarin, phenytoin, flurbiprofen, and (S)-naproxen. The resulting CYP2C9-dependent metabolites were analyzed by LC-MS as described in Materials and Methods to obtain measures of relative catalytic efficiency (Vmax/Km).

## Discussion

Coding-region variants of *CYP2C9* can exhibit metabolic activities that significantly alter the pharmacokinetics and pharmacodynamic response of CYP2C9 narrow therapeutic substrates, necessitating dose changes to avoid adverse drug responses when drug clearance is reduced. We recently completed the resequencing of *CYP2C9* (and several other vitamin K cycle–associated genes) as part of a multi-investigator project designed to identify variation in pharmacogenes relevant to warfarin treatment ([Fohner et al., 2015](#B8)). Single-nucleotide polymorphism discovery efforts were conducted with a diverse cohort of the AN people to establish minor allele frequencies (MAFs) for different cultural subgroups, such as the Yup’ik and Athabascan people. In *CYP2C9*, we discovered three novel coding variants, *M1L*, *N218I,* and *P279T*, that were present at an MAF of 0%–7%, depending on the allele and AN subgroup. The most prevalent of these novel single-nucleotide variants (SNVs), *M1L* (1A>T), occurs at the translation start site and was predicted to disrupt translation and confer a CYP2C9 poor metabolizer phenotype. The *N218I* variant was also predicted to decrease CYP2C9 enzyme activity ([Fohner et al., 2015](#B8)), with the neighboring variant, *Q214L* (*CYP2C9*28*), displaying a 90% decline in (*S*)-warfarin 7-hydroxylation activity in CYP2C9.28-expressing COS-7 cells compared with WT ([Niinuma et al., 2014](#B22)). The *P279T* allele, originally identified in a Japanese population ([Maekawa et al., 2006](#B19)), is now described as *CYP2C9*29* in the star nomenclature. The CYP2C9.29 variant protein was predicted to have reduced enzyme activity based on several in vitro studies that have investigated the catalytic capabilities of the variant protein toward diverse substrates [reviewed in [Hiratsuka (2016)](#B12)]. The P279T enzyme expressed well in COS-7 cells and exhibited 54% of the catalytic efficiency of the wild-type enzyme for formation of 7-hydroxy-(*S*)-warfarin due entirely to a decrease in *V*_max_ ([Niinuma et al., 2014](#B22)). The same investigators reported that CYP2C9.29 exhibited 72% of the activity of wild-type enzyme toward tolbutamide when reactions were conducted at the high substrate concentration of 1 mM.

To our knowledge, functional characterization of the *M1L* (1A>T) and *N218I* variants has never been reported. Expression of *M1L* in lentivirus-infected HepG2 cells confirmed the predicted lack of translation of mRNA that was abundantly present. Consequently, *M1L* should be categorized as a no-function allele, similar to the previously reported base frameshift and premature stop codon variants: *CYP2C9*6*, *CYP2C9*15*, and *CYP2C9*25* ([https://www.pharmvar.org/gene/CYP2C9](https://www.pharmvar.org/gene/CYP2C9)). The absence of mRNA translation from a different altered start codon allele, *CYP2C9 M1V* (*CYP2C9*36*, rs114071557), is reported in the ExAc database with a low MAF of 0.0004% across all studied populations ([Lek et al., 2016](#B17)). Notably, PharmVar describes the start site allele *CYP2C9*36* as a variant of unknown function, and although no experimental data have been provided as yet, the foregoing considerations suggest that *CYP2C9*36* is also null. Interestingly, *CYP2C19*4*, (ATG→TTG, M1L), a mutation analogous to what we discovered for *CYP2C9*, was one of the first null variants to be discovered that conferred poor metabolizer status for metabolism of CYP2C19 substrates because of loss of the translation start site ([Ferguson et al., 1998](#B7)).

In contrast to the *M1L* variant, His-tagged wild-type protein and the N218I and P279T variant proteins expressed well in *E. coli* and were highly purified after affinity chromatography. Upon reconstitution with cytochrome P450 reductase and cytochrome b5, the N218I and P279T variants metabolized (*S*)-warfarin, phenytoin, flurbiprofen, and (*S*)-naproxen to the expected monohydroxylated or *O*-desmethylated metabolites. Kinetic analysis revealed that the relative catalytic efficiency toward probe substrate metabolism was lower compared with WT enzyme for N218I (14%–38%) than for P279T (59%–87%). Since the discovery of the *P279T* allele in a Japanese subject in 2006 ([Maekawa et al., 2006](#B19)), several in vitro studies have been conducted with the P279T variant expressed in COS cells or insect cells. Membrane preparations from P279T-expressing cell lines metabolized (*S*)-warfarin, phenytoin, and flurbiprofen at 54%–140% of the catalytic efficiency of the WT enzyme ([Niinuma et al., 2014](#B22); [Wang et al., 2015](#B27); [Chen et al., 2016](#B3)). The broader range than what was found in the present study may be attributed, in part, to the use by multiple investigators of different expression systems and analytical procedures. The study reported here benefits from a uniformity of experimental procedures and robust quantitation of the purified cytochrome P450 variants used in the kinetic experiments. It should be noted, however, that we observed significantly lower mRNA expression for P279T in comparison with the WT or N218I variants, which led to correspondingly low levels of protein production. Should similarly low mRNA expression be seen in vivo, it could result in an additional decrement in activity for the P279T variant, especially in homozygous subjects, although it is possible that this effect was merely variability inherent in the lentiviral expression system used in these studies.

The functional deficits we observed for the N218I and P279T variants were the result of a combination of changes in *V*_max_ and K_m_, indicating alterations in both substrate binding and k_cat,_ depending on the substrate. The latter change could be the result of altered function of the CYP2C9 catalytic heme intermediate or in electron transfer from P450 reductase or cytochrome b_5_ to CYP2C9. The N218I variant position is located between the F and G helices, which encapsulate the active site ([Williams et al., 2003](#B31)), and, thus, a decrease in the k_cat_ and an increase in the K_m_ due to amino acid substitution in a region of the enzyme known to be important for catalytic activity ([Wester et al., 2004](#B29)) was expected. Moreover, the reduction in catalytic activity of the N218I protein was of similar magnitude as the CYP2C9**3* (I359L) enzyme and, therefore, is also expected to alter the metabolic clearance of (*S*)-warfarin and other CYP2C9 substrates in vivo*.* Interestingly, although the N218I variant showed a similarly low binding affinity toward each of the four substrates tested, increasing K_m_ by roughly 2-fold compared with WT, the change in k_cat_ effected by the mutation showed evidence of substrate dependence, as the decrease in the turnover rate for flurbiprofen, in comparison with WT, was significantly less than what was observed with the other substrates. P279T is located between the H and I helices of the enzyme and resulted in a decrease in catalytic efficiency to approximately 60% of that of the WT protein for (*S*)-naproxen *O*-demethylation. This decrease in catalytic efficiency was mainly attributed to a decrease in *V*_max_ by 2.4-fold compared with WT enzyme. Helix I borders part of the CYP2C9 active site cavity, and perturbances in helix I due to amino acid substitutions have been hypothesized to disturb the active site topography and reduce enzyme activity ([Maekawa et al., 2017](#B18)). This may offer an explanation for the reduction in *O*-demethylation of (*S*)-naproxen observed with the P279T variant, although it should be noted that the decrease in *V*_max_ for phenytoin, (S)-warfarin, and flurbiprofen metabolism by the variant enzyme is much less severe. These observed substrate-dependent effects on CYP2C9 activity, resulting from both the N218I and P279T mutations, are perhaps not surprising when considered in context with the numerous other examples of substrate-dependent polymorphism available in the CYP2C9 literature ([Tracy et al., 2002](#B26); [Hummel et al., 2005](#B13); [Niinuma et al., 2014](#B22)).

Collectively, these studies define, for the first time, the effect of the novel N218I variant that is expressed at relatively high MAF in AN people on the metabolism of four widely used CYP2C9 substrates. New variants require detailed catalytic characterization because variants with altered function can have substantial clinical repercussions. For example, considered collectively, ∼15% of Yup’ik people living in the Yukon-Kuskokwim delta of Alaska are carriers of the *M1L*, *N218I*, *P279T*, and *I359L* alleles. With the emergence of precision medicine and preemptive pharmacogenetic testing, clinical implications for the AN people include phenotypic misclassification from currently available tests that typically screen only for the common *CYP2C9*2* and **3* variants.

The detailed in vitro catalytic data provided here for the P279T, N2I8I, and M1L protein variants enable prediction of dose adjustments that could be made during treatment with low–therapeutic index substrates of CYP2C9 to avoid adverse drug responses. The potential of *CYP2C9 M1L*, *N218I*, and *P279T* alleles to alter the pharmacokinetics of drugs metabolized by CYP2C9 may put carriers at risk of exacerbated therapeutic effects from drugs that rely predominately on CYP2C9 for their metabolic clearance. Indeed, we found recently that heterozygotes for the N218I variant required 1.1 mg warfarin/day less than reference individuals to achieve a comparable therapeutic international normalized ration (INR) response ([Henderson et al., 2019](#B11)). Despite evidence for substrate-dependent polymorphism, the in vitro rank order metabolic capacity of the heterozygous genotypes is clearly *P279T/*1* > *N218I/*1 > M1L/*1*, with *M1L/M1L* homozygotes tentatively assigned as having no CYP2C9 activity. Studies to further characterize the impact of these new variant alleles on drug disposition and response in AN people in vivo are needed to test these predictions and preclude potential healthcare disparities.

## Acknowledgments

The authors would like to thank Denise Dillard and Vanessa Hiratsuka and Julie Beans with the Research Department at Southcentral Foundation, Alaska, and Bert Boyer and Scarlett Hopkins at Oregon Health Sciences University for their valuable editorial comments in the preparation of this manuscript.

## Abbreviations

## Authorship Contributions

*Participated in research design:* McDonald, Henderson, Thummel, Rettie.

*Conducted experiments:* McDonald, Henderson, Ray, Yeung, Johnson, Kowalski.

*Contributed new reagents or analytic tools:* Hanenberg, Wiek.

*Performed data analysis:* McDonald, Henderson, Rettie.

*Wrote or contributed to the writing of the manuscript:* McDonald, Henderson, Ray, Yeung, Johnson, Kowalski, Hanenberg, Wiek, Thummel, Rettie.

## Footnotes

## References

1. Bush E, Trager WF. (1983) High-yield synthesis of warfarin and its phenolic metabolites: new compounds. J Pharm Sci 72:830–831.  [DOI](https://doi.org/10.1002/jps.2600720732) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6886996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharm%20Sci&title=High-yield%20synthesis%20of%20warfarin%20and%20its%20phenolic%20metabolites:%20new%20compounds&author=E%20Bush&author=WF%20Trager&volume=72&publication_year=1983&pages=830-831&pmid=6886996&doi=10.1002/jps.2600720732&)

2. Cheesman MJ, Baer BR, Zheng YM, Gillam EM, Rettie AE. (2003) Rabbit CYP4B1 engineered for high-level expression in Escherichia coli: ligand stabilization and processing of the N-terminus and heme prosthetic group. Arch Biochem Biophys 416:17–24.  [DOI](https://doi.org/10.1016/s0003-9861(03)00278-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12859977/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Biochem%20Biophys&title=Rabbit%20CYP4B1%20engineered%20for%20high-level%20expression%20in%20Escherichia%20coli:%20ligand%20stabilization%20and%20processing%20of%20the%20N-terminus%20and%20heme%20prosthetic%20group&author=MJ%20Cheesman&author=BR%20Baer&author=YM%20Zheng&author=EM%20Gillam&author=AE%20Rettie&volume=416&publication_year=2003&pages=17-24&pmid=12859977&doi=10.1016/s0003-9861(03)00278-9&)

3. Chen LG, Wang Z, Zhu Y, Xiong JH, Sun LR, Dai DP, Cai JP, Hu GX. (2016) In vitro metabolism of phenytoin in 36 CYP2C9 variants found in the Chinese population. Chem Biol Interact 253:93–99.  [DOI](https://doi.org/10.1016/j.cbi.2016.04.040) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27163851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Biol%20Interact&title=In%20vitro%20metabolism%20of%20phenytoin%20in%2036%20CYP2C9%20variants%20found%20in%20the%20Chinese%20population&author=LG%20Chen&author=Z%20Wang&author=Y%20Zhu&author=JH%20Xiong&author=LR%20Sun&volume=253&publication_year=2016&pages=93-99&pmid=27163851&doi=10.1016/j.cbi.2016.04.040&)

4. Chen W, Koenigs LL, Thompson SJ, Peter RM, Rettie AE, Trager WF, Nelson SD. (1998) Oxidation of acetaminophen to its toxic quinone imine and nontoxic catechol metabolites by baculovirus-expressed and purified human cytochromes P450 2E1 and 2A6. Chem Res Toxicol 11:295–301.  [DOI](https://doi.org/10.1021/tx9701687) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9548799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Res%20Toxicol&title=Oxidation%20of%20acetaminophen%20to%20its%20toxic%20quinone%20imine%20and%20nontoxic%20catechol%20metabolites%20by%20baculovirus-expressed%20and%20purified%20human%20cytochromes%20P450%202E1%20and%202A6&author=W%20Chen&author=LL%20Koenigs&author=SJ%20Thompson&author=RM%20Peter&author=AE%20Rettie&volume=11&publication_year=1998&pages=295-301&pmid=9548799&doi=10.1021/tx9701687&)

5. Daly AK, Rettie AE, Fowler DM, Miners JO. (2017) Pharmacogenomics of CYP2C9: functional and clinical considerations. J Pers Med 8:1.  [DOI](https://doi.org/10.3390/jpm8010001) | [PMC free article](/articles/PMC5872075/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29283396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pers%20Med&title=Pharmacogenomics%20of%20CYP2C9:%20functional%20and%20clinical%20considerations&author=AK%20Daly&author=AE%20Rettie&author=DM%20Fowler&author=JO%20Miners&volume=8&publication_year=2017&pages=1&pmid=29283396&doi=10.3390/jpm8010001&)

6. Drozda K, Wong S, Patel SR, Bress AP, Nutescu EA, Kittles RA, Cavallari LH. (2015) Poor warfarin dose prediction with pharmacogenetic algorithms that exclude genotypes important for African Americans. Pharmacogenet Genomics 25:73–81.  [DOI](https://doi.org/10.1097/FPC.0000000000000108) | [PMC free article](/articles/PMC4280295/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25461246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Poor%20warfarin%20dose%20prediction%20with%20pharmacogenetic%20algorithms%20that%20exclude%20genotypes%20important%20for%20African%20Americans&author=K%20Drozda&author=S%20Wong&author=SR%20Patel&author=AP%20Bress&author=EA%20Nutescu&volume=25&publication_year=2015&pages=73-81&pmid=25461246&doi=10.1097/FPC.0000000000000108&)

7. Ferguson RJ, De Morais SM, Benhamou S, Bouchardy C, Blaisdell J, Ibeanu G, Wilkinson GR, Sarich TC, Wright JM, Dayer P, et al. (1998) A new genetic defect in human CYP2C19: mutation of the initiation codon is responsible for poor metabolism of S-mephenytoin. J Pharmacol Exp Ther 284:356–361.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9435198/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=A%20new%20genetic%20defect%20in%20human%20CYP2C19:%20mutation%20of%20the%20initiation%20codon%20is%20responsible%20for%20poor%20metabolism%20of%20S-mephenytoin&author=RJ%20Ferguson&author=SM%20De%20Morais&author=S%20Benhamou&author=C%20Bouchardy&author=J%20Blaisdell&volume=284&publication_year=1998&pages=356-361&pmid=9435198&)

8. Fohner AE, Robinson R, Yracheta J, Dillard DA, Schilling B, Khan B, Hopkins S, Boyer B, Black J, Wiener H, et al. (2015) Variation in genes controlling warfarin disposition and response in American Indian and Alaska Native people: CYP2C9, VKORC1, CYP4F2, CYP4F11, GGCX. Pharmacogenet Genomics 25:343–353.  [DOI](https://doi.org/10.1097/FPC.0000000000000143) | [PMC free article](/articles/PMC4461509/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25946405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Variation%20in%20genes%20controlling%20warfarin%20disposition%20and%20response%20in%20American%20Indian%20and%20Alaska%20Native%20people:%20CYP2C9,%20VKORC1,%20CYP4F2,%20CYP4F11,%20GGCX&author=AE%20Fohner&author=R%20Robinson&author=J%20Yracheta&author=DA%20Dillard&author=B%20Schilling&volume=25&publication_year=2015&pages=343-353&pmid=25946405&doi=10.1097/FPC.0000000000000143&)

9. Haining RL, Hunter AP, Veronese ME, Trager WF, Rettie AE. (1996) Allelic variants of human cytochrome P450 2C9: baculovirus-mediated expression, purification, structural characterization, substrate stereoselectivity, and prochiral selectivity of the wild-type and I359L mutant forms. Arch Biochem Biophys 333:447–458.  [DOI](https://doi.org/10.1006/abbi.1996.0414) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8809086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Biochem%20Biophys&title=Allelic%20variants%20of%20human%20cytochrome%20P450%202C9:%20baculovirus-mediated%20expression,%20purification,%20structural%20characterization,%20substrate%20stereoselectivity,%20and%20prochiral%20selectivity%20of%20the%20wild-type%20and%20I359L%20mutant%20forms&author=RL%20Haining&author=AP%20Hunter&author=ME%20Veronese&author=WF%20Trager&author=AE%20Rettie&volume=333&publication_year=1996&pages=447-458&pmid=8809086&doi=10.1006/abbi.1996.0414&)

10. Henderson LM, Claw KG, Woodahl EL, Robinson RF, Boyer BB, Burke W, Thummel KE. (2018) P450 pharmacogenetics in indigenous North American populations. J Pers Med 8:9.  [DOI](https://doi.org/10.3390/jpm8010009) | [PMC free article](/articles/PMC5872083/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29389890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pers%20Med&title=P450%20pharmacogenetics%20in%20indigenous%20North%20American%20populations&author=LM%20Henderson&author=KG%20Claw&author=EL%20Woodahl&author=RF%20Robinson&author=BB%20Boyer&volume=8&publication_year=2018&pages=9&pmid=29389890&doi=10.3390/jpm8010009&)

11. Henderson LM, Robinson RF, Ray L, Khan BA, Li T, Dillard DA, Schilling BD, Mosley M, Janssen PL, Fohner AE, et al. (2019) VKORC1 and novel CYP2C9 variation predict warfarin response in Alaska Native and American Indian people. Clin Transl Sci 12:312–320.  [DOI](https://doi.org/10.1111/cts.12611) | [PMC free article](/articles/PMC6510382/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30821933/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=VKORC1%20and%20novel%20CYP2C9%20variation%20predict%20warfarin%20response%20in%20Alaska%20Native%20and%20American%20Indian%20people&author=LM%20Henderson&author=RF%20Robinson&author=L%20Ray&author=BA%20Khan&author=T%20Li&volume=12&publication_year=2019&pages=312-320&pmid=30821933&doi=10.1111/cts.12611&)

12. Hiratsuka M. (2016) Genetic polymorphisms and in vitro functional characterization of CYP2C8, CYP2C9, and CYP2C19 allelic variants. Biol Pharm Bull 39:1748–1759.  [DOI](https://doi.org/10.1248/bpb.b16-00605) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27803446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Pharm%20Bull&title=Genetic%20polymorphisms%20and%20in%20vitro%20functional%20characterization%20of%20CYP2C8,%20CYP2C9,%20and%20CYP2C19%20allelic%20variants&author=M%20Hiratsuka&volume=39&publication_year=2016&pages=1748-1759&pmid=27803446&doi=10.1248/bpb.b16-00605&)

13. Hummel MA, Locuson CW, Gannett PM, Rock DA, Mosher CM, Rettie AE, Tracy TS. (2005) CYP2C9 genotype-dependent effects on in vitro drug-drug interactions: switching of benzbromarone effect from inhibition to activation in the CYP2C9.3 variant. Mol Pharmacol 68:644–651.  [DOI](https://doi.org/10.1124/mol.105.013763) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15955872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=CYP2C9%20genotype-dependent%20effects%20on%20in%20vitro%20drug-drug%20interactions:%20switching%20of%20benzbromarone%20effect%20from%20inhibition%20to%20activation%20in%20the%20CYP2C9.3%20variant&author=MA%20Hummel&author=CW%20Locuson&author=PM%20Gannett&author=DA%20Rock&author=CM%20Mosher&volume=68&publication_year=2005&pages=644-651&pmid=15955872&doi=10.1124/mol.105.013763&)

14. Isvoran A, Louet M, Vladoiu DL, Craciun D, Loriot MA, Villoutreix BO, Miteva MA. (2017) Pharmacogenomics of the cytochrome P450 2C family: impacts of amino acid variations on drug metabolism. Drug Discov Today 22:366–376.  [DOI](https://doi.org/10.1016/j.drudis.2016.09.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27693711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Discov%20Today&title=Pharmacogenomics%20of%20the%20cytochrome%20P450%202C%20family:%20impacts%20of%20amino%20acid%20variations%20on%20drug%20metabolism&author=A%20Isvoran&author=M%20Louet&author=DL%20Vladoiu&author=D%20Craciun&author=MA%20Loriot&volume=22&publication_year=2017&pages=366-376&pmid=27693711&doi=10.1016/j.drudis.2016.09.015&)

15. Kimmel SE, French B, Kasner SE, Johnson JA, Anderson JL, Gage BF, Rosenberg YD, Eby CS, Madigan RA, McBane RB, et al. COAG Investigators (2013) A pharmacogenetic versus a clinical algorithm for warfarin dosing. N Engl J Med 369:2283–2293.  [DOI](https://doi.org/10.1056/NEJMoa1310669) | [PMC free article](/articles/PMC3942158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24251361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=A%20pharmacogenetic%20versus%20a%20clinical%20algorithm%20for%20warfarin%20dosing&author=SE%20Kimmel&author=B%20French&author=SE%20Kasner&author=JA%20Johnson&author=JL%20Anderson&volume=369&publication_year=2013&pages=2283-2293&pmid=24251361&doi=10.1056/NEJMoa1310669&)

16. Koukouritaki SB, Manro JR, Marsh SA, Stevens JC, Rettie AE, McCarver DG, Hines RN. (2004) Developmental expression of human hepatic CYP2C9 and CYP2C19. J Pharmacol Exp Ther 308:965–974.  [DOI](https://doi.org/10.1124/jpet.103.060137) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14634042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Developmental%20expression%20of%20human%20hepatic%20CYP2C9%20and%20CYP2C19&author=SB%20Koukouritaki&author=JR%20Manro&author=SA%20Marsh&author=JC%20Stevens&author=AE%20Rettie&volume=308&publication_year=2004&pages=965-974&pmid=14634042&doi=10.1124/jpet.103.060137&)

17. Lek M, Karczewski KJ, Minikel EV, Samocha KE, Banks E, Fennell T, O’Donnell-Luria AH, Ware JS, Hill AJ, Cummings BB, et al. Exome Aggregation Consortium (2016) Analysis of protein-coding genetic variation in 60,706 humans. Nature 536:285–291.  [DOI](https://doi.org/10.1038/nature19057) | [PMC free article](/articles/PMC5018207/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27535533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Analysis%20of%20protein-coding%20genetic%20variation%20in%2060,706%20humans&author=M%20Lek&author=KJ%20Karczewski&author=EV%20Minikel&author=KE%20Samocha&author=E%20Banks&volume=536&publication_year=2016&pages=285-291&pmid=27535533&doi=10.1038/nature19057&)

18. Maekawa K, Adachi M, Matsuzawa Y, Zhang Q, Kuroki R, Saito Y, Shah MB. (2017) Structural basis of single-nucleotide polymorphisms in cytochrome P450 2C9. Biochemistry 56:5476–5480.  [DOI](https://doi.org/10.1021/acs.biochem.7b00795) | [PMC free article](/articles/PMC5761073/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28972767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Structural%20basis%20of%20single-nucleotide%20polymorphisms%20in%20cytochrome%20P450%202C9&author=K%20Maekawa&author=M%20Adachi&author=Y%20Matsuzawa&author=Q%20Zhang&author=R%20Kuroki&volume=56&publication_year=2017&pages=5476-5480&pmid=28972767&doi=10.1021/acs.biochem.7b00795&)

19. Maekawa K, Fukushima-Uesaka H, Tohkin M, Hasegawa R, Kajio H, Kuzuya N, Yasuda K, Kawamoto M, Kamatani N, Suzuki K, et al. (2006) Four novel defective alleles and comprehensive haplotype analysis of CYP2C9 in Japanese. Pharmacogenet Genomics 16:497–514.  [DOI](https://doi.org/10.1097/01.fpc.0000215069.14095.c6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16788382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Four%20novel%20defective%20alleles%20and%20comprehensive%20haplotype%20analysis%20of%20CYP2C9%20in%20Japanese&author=K%20Maekawa&author=H%20Fukushima-Uesaka&author=M%20Tohkin&author=R%20Hasegawa&author=H%20Kajio&volume=16&publication_year=2006&pages=497-514&pmid=16788382&doi=10.1097/01.fpc.0000215069.14095.c6&)

20. Mosher CM, Hummel MA, Tracy TS, Rettie AE. (2008) Functional analysis of phenylalanine residues in the active site of cytochrome P450 2C9. Biochemistry 47:11725–11734.  [DOI](https://doi.org/10.1021/bi801231m) | [PMC free article](/articles/PMC2713172/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18922023/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Functional%20analysis%20of%20phenylalanine%20residues%20in%20the%20active%20site%20of%20cytochrome%20P450%202C9&author=CM%20Mosher&author=MA%20Hummel&author=TS%20Tracy&author=AE%20Rettie&volume=47&publication_year=2008&pages=11725-11734&pmid=18922023&doi=10.1021/bi801231m&)

21. Mosher CM, Tai G, Rettie AE. (2009) CYP2C9 amino acid residues influencing phenytoin turnover and metabolite regio- and stereochemistry. J Pharmacol Exp Ther 329:938–944.  [DOI](https://doi.org/10.1124/jpet.109.150706) | [PMC free article](/articles/PMC2683772/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19258521/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=CYP2C9%20amino%20acid%20residues%20influencing%20phenytoin%20turnover%20and%20metabolite%20regio-%20and%20stereochemistry&author=CM%20Mosher&author=G%20Tai&author=AE%20Rettie&volume=329&publication_year=2009&pages=938-944&pmid=19258521&doi=10.1124/jpet.109.150706&)

22. Niinuma Y, Saito T, Takahashi M, Tsukada C, Ito M, Hirasawa N, Hiratsuka M. (2014) Functional characterization of 32 CYP2C9 allelic variants. Pharmacogenomics J 14:107–114.  [DOI](https://doi.org/10.1038/tpj.2013.22) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23752738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Functional%20characterization%20of%2032%20CYP2C9%20allelic%20variants&author=Y%20Niinuma&author=T%20Saito&author=M%20Takahashi&author=C%20Tsukada&author=M%20Ito&volume=14&publication_year=2014&pages=107-114&pmid=23752738&doi=10.1038/tpj.2013.22&)

23. Paine MF, Hart HL, Ludington SS, Haining RL, Rettie AE, Zeldin DC. (2006) The human intestinal cytochrome P450 “pie”. Drug Metab Dispos 34:880–886.  [DOI](https://doi.org/10.1124/dmd.105.008672) | [PMC free article](/articles/PMC2222892/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16467132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=The%20human%20intestinal%20cytochrome%20P450%20%E2%80%9Cpie%E2%80%9D&author=MF%20Paine&author=HL%20Hart&author=SS%20Ludington&author=RL%20Haining&author=AE%20Rettie&volume=34&publication_year=2006&pages=880-886&pmid=16467132&doi=10.1124/dmd.105.008672&)

24. Roellecke K, Virts EL, Einholz R, Edson KZ, Altvater B, Rossig C, von Laer D, Scheckenbach K, Wagenmann M, Reinhardt D, et al. (2016) Optimized human CYP4B1 in combination with the alkylator prodrug 4-ipomeanol serves as a novel suicide gene system for adoptive T-cell therapies. Gene Ther 23:615–626.  [DOI](https://doi.org/10.1038/gt.2016.38) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27092941/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene%20Ther&title=Optimized%20human%20CYP4B1%20in%20combination%20with%20the%20alkylator%20prodrug%204-ipomeanol%20serves%20as%20a%20novel%20suicide%20gene%20system%20for%20adoptive%20T-cell%20therapies&author=K%20Roellecke&author=EL%20Virts&author=R%20Einholz&author=KZ%20Edson&author=B%20Altvater&volume=23&publication_year=2016&pages=615-626&pmid=27092941&doi=10.1038/gt.2016.38&)

25. Simionatto S, Marchioro SB, Galli V, Luerce TD, Hartwig DD, Moreira AN, Dellagostin OA. (2009) Efficient site-directed mutagenesis using an overlap extension-PCR method for expressing Mycoplasma hyopneumoniae genes in Escherichia coli. J Microbiol Methods 79:101–105.  [DOI](https://doi.org/10.1016/j.mimet.2009.08.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19733599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Microbiol%20Methods&title=Efficient%20site-directed%20mutagenesis%20using%20an%20overlap%20extension-PCR%20method%20for%20expressing%20Mycoplasma%20hyopneumoniae%20genes%20in%20Escherichia%20coli&author=S%20Simionatto&author=SB%20Marchioro&author=V%20Galli&author=TD%20Luerce&author=DD%20Hartwig&volume=79&publication_year=2009&pages=101-105&pmid=19733599&doi=10.1016/j.mimet.2009.08.016&)

26. Tracy TS, Hutzler JM, Haining RL, Rettie AE, Hummel MA, Dickmann LJ. (2002) Polymorphic variants (CYP2C9*3 and CYP2C9*5) and the F114L active site mutation of CYP2C9: effect on atypical kinetic metabolism profiles. Drug Metab Dispos 30:385–390.  [DOI](https://doi.org/10.1124/dmd.30.4.385) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11901091/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Polymorphic%20variants%20(CYP2C9*3%20and%20CYP2C9*5)%20and%20the%20F114L%20active%20site%20mutation%20of%20CYP2C9:%20effect%20on%20atypical%20kinetic%20metabolism%20profiles&author=TS%20Tracy&author=JM%20Hutzler&author=RL%20Haining&author=AE%20Rettie&author=MA%20Hummel&volume=30&publication_year=2002&pages=385-390&pmid=11901091&doi=10.1124/dmd.30.4.385&)

27. Wang L, Bao SH, Pan PP, Xia MM, Chen MC, Liang BQ, Dai DP, Cai JP, Hu GX. (2015) Effect of CYP2C9 genetic polymorphism on the metabolism of flurbiprofen in vitro. Drug Dev Ind Pharm 41:1363–1367.  [DOI](https://doi.org/10.3109/03639045.2014.950274) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25144335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Dev%20Ind%20Pharm&title=Effect%20of%20CYP2C9%20genetic%20polymorphism%20on%20the%20metabolism%20of%20flurbiprofen%20in%20vitro&author=L%20Wang&author=SH%20Bao&author=PP%20Pan&author=MM%20Xia&author=MC%20Chen&volume=41&publication_year=2015&pages=1363-1367&pmid=25144335&doi=10.3109/03639045.2014.950274&)

28. Wang YH, Pan PP, Dai DP, Wang SH, Geng PW, Cai JP, Hu GX. (2014) Effect of 36 CYP2C9 variants found in the Chinese population on losartan metabolism in vitro. Xenobiotica 44:270–275.  [DOI](https://doi.org/10.3109/00498254.2013.820007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23844998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Effect%20of%2036%20CYP2C9%20variants%20found%20in%20the%20Chinese%20population%20on%20losartan%20metabolism%20in%20vitro&author=YH%20Wang&author=PP%20Pan&author=DP%20Dai&author=SH%20Wang&author=PW%20Geng&volume=44&publication_year=2014&pages=270-275&pmid=23844998&doi=10.3109/00498254.2013.820007&)

29. Wester MR, Yano JK, Schoch GA, Yang C, Griffin KJ, Stout CD, Johnson EF. (2004) The structure of human cytochrome P450 2C9 complexed with flurbiprofen at 2.0-A resolution. J Biol Chem 279:35630–35637.  [DOI](https://doi.org/10.1074/jbc.M405427200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15181000/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=The%20structure%20of%20human%20cytochrome%20P450%202C9%20complexed%20with%20flurbiprofen%20at%202.0-A%20resolution&author=MR%20Wester&author=JK%20Yano&author=GA%20Schoch&author=C%20Yang&author=KJ%20Griffin&volume=279&publication_year=2004&pages=35630-35637&pmid=15181000&doi=10.1074/jbc.M405427200&)

30. Wiek C, Schmidt EM, Roellecke K, Freund M, Nakano M, Kelly EJ, Kaisers W, Yarov-Yarovoy V, Kramm CM, Rettie AE, et al. (2015) Identification of amino acid determinants in CYP4B1 for optimal catalytic processing of 4-ipomeanol. Biochem J 465:103–114.  [DOI](https://doi.org/10.1042/BJ20140813) | [PMC free article](/articles/PMC4312185/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25247810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20J&title=Identification%20of%20amino%20acid%20determinants%20in%20CYP4B1%20for%20optimal%20catalytic%20processing%20of%204-ipomeanol&author=C%20Wiek&author=EM%20Schmidt&author=K%20Roellecke&author=M%20Freund&author=M%20Nakano&volume=465&publication_year=2015&pages=103-114&pmid=25247810&doi=10.1042/BJ20140813&)

31. Williams PA, Cosme J, Ward A, Angove HC, Matak Vinković D, Jhoti H. (2003) Crystal structure of human cytochrome P450 2C9 with bound warfarin. Nature 424:464–468.  [DOI](https://doi.org/10.1038/nature01862) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12861225/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Crystal%20structure%20of%20human%20cytochrome%20P450%202C9%20with%20bound%20warfarin&author=PA%20Williams&author=J%20Cosme&author=A%20Ward&author=HC%20Angove&author=D%20Matak%20Vinkovi%C4%87&volume=424&publication_year=2003&pages=464-468&pmid=12861225&doi=10.1038/nature01862&)

32. Zhang HF, Wang HH, Gao N, Wei JY, Tian X, Zhao Y, Fang Y, Zhou J, Wen Q, Gao J, et al. (2016) Physiological content and intrinsic activities of 10 cytochrome P450 isoforms in human normal liver microsomes. J Pharmacol Exp Ther 358:83–93.  [DOI](https://doi.org/10.1124/jpet.116.233635) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27189963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Physiological%20content%20and%20intrinsic%20activities%20of%2010%20cytochrome%20P450%20isoforms%20in%20human%20normal%20liver%20microsomes&author=HF%20Zhang&author=HH%20Wang&author=N%20Gao&author=JY%20Wei&author=X%20Tian&volume=358&publication_year=2016&pages=83-93&pmid=27189963&doi=10.1124/jpet.116.233635&)

33. Zhou Y, Ingelman-Sundberg M, Lauschke VM. (2017) Worldwide distribution of cytochrome P450 alleles: a meta-analysis of population-scale sequencing projects. Clin Pharmacol Ther 102:688–700.  [DOI](https://doi.org/10.1002/cpt.690) | [PMC free article](/articles/PMC5600063/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28378927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Worldwide%20distribution%20of%20cytochrome%20P450%20alleles:%20a%20meta-analysis%20of%20population-scale%20sequencing%20projects&author=Y%20Zhou&author=M%20Ingelman-Sundberg&author=VM%20Lauschke&volume=102&publication_year=2017&pages=688-700&pmid=28378927&doi=10.1002/cpt.690&)
