# Severe Capecitabine Toxicity Associated With a Rare DPYD Variant Identified Through Whole-Genome Sequencing

## Metadata
**Authors:** Reynold C Ly, Remington E Schmidt, Patrick J Kiel, Victoria M Pratt, Bryan P Schneider, Milan Radovich, Steven M Offer, Robert B Diasio, Todd C Skaar
**Journal:** JCO Precision Oncology
**Date:** 2020 Jun 12
**DOI:** [10.1200/PO.20.00067](https://doi.org/10.1200/PO.20.00067)
**PMID:** 32923881
**PMCID:** PMC7446391
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7446391/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7446391/pdf/PO.20.00067.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7446391/pdf/PO.20.00067.pdf)

## INTRODUCTION

Fluoropyrimidine drugs, both fluorouracil (FU) and its prodrug capecitabine, are widely used in the treatment of solid tumors such as breast, colorectal, and gastric cancers.^1^ Over 2 million patients newly diagnosed with cancer are treated each year with fluoropyrimidines.^2^ Between 10% and 40% of these patients develop severe, sometimes life-threatening toxicities, which may include mucositis, neutropenia, nausea, severe diarrhea, vomiting, stomatitis, and hand-foot syndrome.^2^ These toxicities can be caused by genetic variants in *DPYD*, the gene that encodes for dihydropyrimidine dehydrogenase (DPD), the rate-limiting enzyme responsible for FU catabolism.^1,2^

Clinical *DPYD* genotypic testing typically includes the following known toxicity-associated *DPYD* variants: c.1905+1G>A (*2A, rs3918290), c.1679T>G (*13, rs55886062), c.2846A>T (rs67376798), and c.1129-5923T>G (rs75017182). Guidelines for using these results to guide fluoropyrimidine therapy have been published by the Clinical Pharmacogenetics Implementation Consortium (CPIC) and the Dutch Pharmacogenetics Working Group.^2,3^ However, targeted genotyping is limited to testing known toxicity-associated *DPYD* variants and may not include novel or rare variants, which can also be deleterious to DPD function and contribute to severe toxicity.^4-9^

Here, we present a case of a patient with a rare variant of unknown significance in *DPYD* who had a severe, life-threatening capecitabine toxicity. The variant was predicted to be deleterious using a recently reported in silico tool (DPYD-Varifier),^10^ and impaired function was confirmed in vitro.

## METHODS

This study was approved by the Indiana University Institutional Review Board. Informed consent was obtained from the patient to have patient-derived specimens and medical records used for research and reporting. Patient germline DNA was obtained from whole blood and used for whole-genome sequencing (WGS), targeted *DPYD* genotyping, and Sanger sequencing. Detailed methods can be found in the Appendix for WGS and Sanger sequencing. Targeted genotyping was performed in the following Clinical Laboratory Improvement Amendments–certified laboratories: ARUP Laboratories (Salt Lake City, UT) and the Indiana University Pharmacogenomics Laboratory (Indianapolis, IN).

Integrated Genomics Viewer Version 2.4.10 (Broad Institute, Cambridge, MA)^11^ was used to visualize WGS data, and Ingenuity Variant Analysis (Qiagen, Germantown, MD) was used for variant identification and annotation. DPYD-Varifier^10^ was used to evaluate the functional impact of p.R235Q on DPD function. The effect of p.R235Q on DPD enzyme activity was determined in vitro as previously described.^10^

## CASE REPORT

A 59-year-old Indian woman was diagnosed with metastatic colon cancer and started on a neoadjuvant treatment regimen of capecitabine (3 500-mg tablets orally twice a day), oxaliplatin, and bevacizumab (a treatment time line is presented in [Fig 1](#fig1)). After 9 days of treatment, the patient developed grade 4 mucositis at the ileum and was hospitalized, at which time all chemotherapy was stopped. Hospital records indicate the presence of febrile neutropenia at admission; however, WBC counts were not available to report. While hospitalized, the patient was found to have an ileal obstruction. In all, the patient was hospitalized for 3 weeks. Because of the severity of toxicity, the patient was believed to be DPD deficient, and commercial testing for 3 toxicity-associated *DPYD* variants (c.1905+1G>A, c.1679T>G, and c.2846A>T) was performed. The patient was found to be wild-type for all 3 variants.

### FIG 1.

![FIG 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1bd4/7446391/e6938c6df218/PO.20.00067f1.jpg)

Time line of patient’s case report. This diagram shows the time line of key events in the diagnosis, treatment, and laboratory testing of DPYD variants. The horizontal line represents the number of days from initial diagnosis to the patient’s last encounter after modified infusional fluorouracil, leucovorin, and oxaliplatin (mFOLFOX) treatment. (*) Not drawn to scale. CT, computed tomography; DPD, dihydropyrimidine dehydrogenase; FU, fluorouracil; IU, Indiana University; NGS, next-generation sequencing; PGx, pharmacogenomics; PO, oral; WGS, whole-genome sequencing.

After surgical debulking, the patient enrolled into the Indiana University Health Precision Genomics Clinic for genome-guided cancer therapy ([Fig 1](#fig1)). WGS was performed on germline and tumor DNA. As part of care, noncarrier status for c.1905+1G>A was confirmed. The patient remained off chemotherapy until computed tomography scan revealed disease progression. As a result of the previously observed toxicity, the patient was started on a modified leucovorin, FU, and oxaliplatin (mFOLFOX) regimen with a specific 75% FU dose reduction. The patient has tolerated 4 cycles of mFOLFOX therapy with no reported toxicity to date.

On the basis of the extreme observed toxicity, results of targeted *DPYD* genotyping, and tolerance to the reduced mFOLFOX regimen, we hypothesized that the patient might carry a deleterious *DPYD* variant that was not covered by the testing panel. Review of the germline WGS data confirmed noncarrier status for c.1905+1G>A, c.1679T>G, c.2846A>T, and c.1129-5923T>G. However, the patient carried a heterozygous rare missense variant in *DPYD*, rs755416212 ([NM_000110.3](https://www.ncbi.nlm.nih.gov/nuccore/NM_000110.3):c.704G>A; [NP_000101.2](https://www.ncbi.nlm.nih.gov/protein/NP_000101.2):p.Arg235Gln; referred to as p.R235Q; [Fig 2A](#fig2)). Genotype was confirmed using Sanger sequencing ([Fig 2B](#fig2)). Data in the Genome Aggregation Database and Exome Aggregation Consortium database indicate that this variant has a minor allele frequency of 0.00001.

### FIG 2.

![FIG 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1bd4/7446391/75617900545d/PO.20.00067f2.jpg)

Presence of the heterozygous DPYD p.R235Q (c.704G>A) variant in the patient’s germline genome. (A) Whole-genome sequencing reads identified the heterozygous single-nucleotide variant (C>T on the + DNA strand depicted) in DPYD. Sequencing reads matching the reference sequence are shown in gray. Data were displayed using Integrated Genomics Viewer. (B) Sanger sequencing confirmed the presence of the heterozygous single-nucleotide variant (G>A on the − coding strand) in DPYD as identified by the arrow.

DPYD-Varifier, a gene-specific in silico variant classifier,^6,10^ predicted that p.R235Q was deleterious to DPD function. This prediction was validated using a previously reported in vitro assay for measuring variant function,^10^ which showed that the p.R235Q variant significantly reduced DPD activity by 88% compared with wild-type DPD (*P* = 6.26 × 10^−7^; [Fig 3](#fig3)). We conclude that p.R235Q likely contributed to the patient’s severe, life-threatening capecitabine toxicity.

### FIG 3.

![FIG 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1bd4/7446391/6a684a0a11ab/PO.20.00067f3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7446391_PO.20.00067f3.jpg)

Functional assessment of p.R235Q DPYD variant. The in vitro enzyme activity of recombinantly expressed *2A (c.1905+1G>A), *13 (c.1679T>G, p.I560S), p.D949V (c.2846A>T), p.R235Q (c.704G>A), and p.R235W (c.703C>T) variants was compared with reference (wild-type [WT]) dihydropyrimidine dehydrogenase (DPD). For each variant, the mean of 4 independent biologic replicate experiments is presented as a horizontal bar ± standard deviation (SD). Each biologic replicate was measured in triplicate (ie, 3 technical replicates each).

## DISCUSSION

This case study illustrates the limitation of using targeted *DPYD* genotyping and the benefit of using sequence-based approaches to identify individuals at risk for severe fluoropyrimidine toxicity. The 4 known toxicity-associated *DPYD* variants (c.1905+1G>A, c.1679T>G, c.2846A>T, and c.1129-5923T>G) have been well characterized in the White population and have recommendations for genotype-guided dosing.^2,3^ However, several studies have identified novel rare *DPYD* variants associated with fluoropyrimidine toxicity using sequencing.^4-9^ Our study illustrates that sequencing may be beneficial for detecting unique, common, and rare variants, especially in individuals of non-White ancestry for whom there is limited pharmacogenetic knowledge.

The p.R235Q missense variant was previously observed in a patient presenting with hematuria who was diagnosed with DPD deficiency.^12^ The patient was compound heterozygous for the p.R235Q missense and p.C79X nonsense variants. *DPYD* analysis of the parents revealed that the father carried p.C79X and the mother carried p.R235Q. DPD activity in the mother was approximately 50% below the population average, as measured ex vivo using peripheral-blood mononuclear cells. This observation corroborates our in vitro finding of impaired DPD function of the p.R235Q variant. To our knowledge, this is the first report of an association between *DPYD* p.R235Q and capecitabine toxicity. Collectively, these data support the clinical actionability of p.R235Q.

Interestingly, p.R235Q’s effect was similar to the previously reported p.R235W variant ([Fig 3](#fig3)), which had a significant 86% decrease in DPD activity compared with wild-type DPD (*P* = 6.44 × 10^−7^).^13^ This variant was found in a patient with DPD deficiency like the p.R235Q variant.^14^ Furthermore, one study examining the role of amino acid 235 on DPD function reported that additional missense mutations (p.R235A and p.R235K) deplete DPD activity in vitro.^15^ Amino acid 235 is well conserved and important for flavin adenine dinucleotide (FAD) binding.^15,16^ Deleterious variants in DPD have been found enriched at conserved residues and in close proximity to important domains such as the FAD domain.^10^ Altogether, our findings and these studies indicate this amino acid position does not tolerate missense changes and should be considered clinically actionable.

The p.R235W variant has been incorporated into CPIC’s guidelines for *DPYD* genotype-guided dosing with an assigned haploid activity score of 0 as a nonfunctional allele.^2^ Given the similar function of p.R235Q and p.R235W ([Fig 3](#fig3)), we feel it may be appropriate to consider p.R235Q similarly with a haploid activity score of 0. As such, the patient in this study would be considered an intermediate metabolizer with a calculated activity score of 1 according to CPIC guidelines. The recommended FU dose reduction with an activity score of 1 is 50%. The fact that the patient tolerated FU with a 75% specific FU dose reduction with no reported toxicity suggests that CPIC’s recommendation of a 50% FU dose reduction might be sufficient to avoid toxicity while preserving drug efficacy for treatment. However, dose titration with therapeutic drug monitoring to prevent underdosing is recommended when available.

The patient was also tested for *TYMS* germline polymorphisms (rs11280056, rs45445694, and rs34743033) as part of the targeted genotyping panel. The patient was homozygous for the 6 base pair (bp) deletion (rs11280056) in the 3′ untranslated region (UTR; DEL/DEL), which is predictive for low *TYMS* expression.^17,18^ The patient also had 3 copies (3R) of the 28-bp tandem repeat in the 5′ UTR promoter enhancer region (rs45445694) with the absence of a variant (rs34743033, G>C) within the second tandem repeat of the 3R allele. Altogether, the 3RG/3RG genotype is associated with increased *TYMS* expression.^19,20^ The clinical association and utility of these *TYMS* polymorphisms with fluoropyrimidine toxicity are unclear.^21-25^ A recent meta-analysis found no significant association with severe GI or hematologic toxicities for *TYMS* (rs45445694 and rs11280056) or *ENOSF1* (rs2612091) variants.^26^ On the basis of the literature, the *TYMS* DEL/DEL genotype was unlikely to contribute to the patient’s severe toxicity.

In conclusion, this case demonstrates the potential benefits of sequence-based *DPYD* genotyping to fluoropyrimidine dose individualization. One challenge to using sequence-level data is the classification of novel and understudied variants. We overcame this challenge using DPYD-Varifier, an in silico classifier, to predict whether the variant was deleterious to DPD function. The prediction was then validated by assessing the variant’s impact on DPD activity in vitro. Collectively, this approach of using genomic sequencing along with DPYD-Varifier and functional testing can help to identify rare toxicity-linked *DPYD* variants and improve genotype-guided therapy in patients treated with fluoropyrimidines.

## ACKNOWLEDGMENT

We thank Surendra Buddi for bioinformatic support.

## APPENDIX

### Whole-Genome Sequencing

Germline and somatic whole-genome sequencing (WGS) was performed by NantOmics (Culver City, CA), a Clinical Laboratory Improvement Amendments–certified laboratory, as previously described (Rabizadeh S, et al: Oncotarget 9:19223-19232, 2018). Sequencing depth for all *DPYD* exons was ≥ 30× by WGS.

### Sanger Sequencing

The patient’s DNA served as the template to amplify the region containing the p.R235Q variant by polymerase chain reaction (PCR). PCR was performed using Platinum SuperFi PCR Mastermix (Thermofisher Scientific, Waltham, MA) using the following primers: 5′-GCA​TCT​TTC​TGC​TTC​TGC​CTG​AT-3′ (forward 1), 5′-GTA​TTG​AAA​TTG​CTT​TTG​GCC​AGT​T-3′ (reverse 1), 5′-TGT​CCT​CAT​GCA​TAT​CTT​GTG​TG-3′ (forward 2), and 5′-TCC​TTT​CTT​TTT​GAG​CAG​TAC​ACA-3′ (reverse 2). Primers were obtained from Integrated DNA Technologies (Coralville, IA). Amplification conditions were 98°C for 30 seconds, 35 cycles at 98°C for 10 seconds, 62.1°C for 10 seconds, and 72°C for 30 seconds. Final extension was at 72°C for 5 minutes. Individual PCR products from both PCR reactions were purified using the MinElute PCR purification kit (Qiagen, Germantown, MD) according to the user manual. DNA concentration was measured using the Qubit dsDNA BR assay kit according to the user manual (Invitrogen, Waltham, MA). Samples were prepared and Sanger sequenced by ACGT (Germantown, MD) using each PCR product as the template with 1 of the 4 primers individually described earlier.

## AUTHOR CONTRIBUTIONS

**Conception and design:** Reynold C. Ly, Victoria M. Pratt, Bryan P. Schneider, Todd C. Skaar

**Financial support:** Todd C. Skaar

**Provision of study materials or patients:** Milan Radovich

**Collection and assembly of data:** Reynold C. Ly, Remington E. Schmidt, Milan Radovich

**Data analysis and interpretation:** Reynold C. Ly, Remington E. Schmidt, Patrick J. Kiel, Victoria M. Pratt, Milan Radovich, Steven M. Offer, Robert B. Diasio, Todd C. Skaar

**Manuscript writing:** All authors

**Final approval of manuscript:** All authors

**Accountable for all aspects of the work:** All authors

## AUTHORS' DISCLOSURES OF POTENTIAL CONFLICTS OF INTEREST

The following represents disclosure information provided by authors of this manuscript. All relationships are considered compensated unless otherwise noted. Relationships are self-held unless noted. I = Immediate Family Member, Inst = My Institution. Relationships may not relate to the subject matter of this manuscript. For more information about ASCO's conflict of interest policy, please refer to [www.asco.org/rwc](http://www.asco.org/rwc) or [ascopubs.org/po/author-center](http://ascopubs.org/po/author-center).

Open Payments is a public database containing information reported by companies about payments made to US-licensed physicians ([Open Payments](https://openpaymentsdata.cms.gov/)).

### Patrick J. Kiel

**Employment:** Amgen

**Speakers' Bureau:** Takeda, Celgene, Genentech

### Victoria M. Pratt

**Stock and Other Ownership Interests:** Quest Diagnostics, LabCorp

**Other Relationship:** Avalon Healthcare

**Uncompensated Relationships:** Veritas Genetics

### Milan Radovich

**Stock and Other Ownership Interests:** LifeOmic, Macrogenics, Immunomedics, ArQule, Tyme Technologies

**Travel, Accommodations, Expenses:** LifeOmic

### Todd C. Skaar

**Honoraria:** Tabula Rasa Healthcare

**Travel, Accommodations, Expenses:** Tabula Rasa Healthcare

**Other Relationship:** National Institutes of Health

No other potential conflicts of interest were reported.

## References

1. Henricks LM, Lunenburg CATC, de Man FM, et al. DPYD genotype-guided dose individualisation of fluoropyrimidine therapy in patients with cancer: A prospective safety analysis. Lancet Oncol. 2018;19:1459–1467. doi: 10.1016/S1470-2045(18)30686-7.  [DOI](https://doi.org/10.1016/S1470-2045(18)30686-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30348537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=DPYD%20genotype-guided%20dose%20individualisation%20of%20fluoropyrimidine%20therapy%20in%20patients%20with%20cancer:%20A%20prospective%20safety%20analysis&author=LM%20Henricks&author=CATC%20Lunenburg&author=FM%20de%20Man&volume=19&publication_year=2018&pages=1459-1467&pmid=30348537&doi=10.1016/S1470-2045(18)30686-7&)

2. Amstutz U, Henricks LM, Offer SM, et al. Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for dihydropyrimidine dehydrogenase genotype and fluoropyrimidine dosing: 2017 update. Clin Pharmacol Ther. 2018;103:210–216. doi: 10.1002/cpt.911.  [DOI](https://doi.org/10.1002/cpt.911) | [PMC free article](/articles/PMC5760397/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29152729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guideline%20for%20dihydropyrimidine%20dehydrogenase%20genotype%20and%20fluoropyrimidine%20dosing:%202017%20update&author=U%20Amstutz&author=LM%20Henricks&author=SM%20Offer&volume=103&publication_year=2018&pages=210-216&pmid=29152729&doi=10.1002/cpt.911&)

3. Lunenburg CATC, van der Wouden CH, Nijenhuis M, et al. Dutch Pharmacogenetics Working Group (DPWG) guideline for the gene-drug interaction of DPYD and fluoropyrimidines. Eur J Hum Genet. 2020;28:508–517. doi: 10.1038/s41431-019-0540-0.  [DOI](https://doi.org/10.1038/s41431-019-0540-0) | [PMC free article](/articles/PMC7080718/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31745289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Dutch%20Pharmacogenetics%20Working%20Group%20(DPWG)%20guideline%20for%20the%20gene-drug%20interaction%20of%20DPYD%20and%20fluoropyrimidines&author=CATC%20Lunenburg&author=CH%20van%20der%20Wouden&author=M%20Nijenhuis&volume=28&publication_year=2020&pages=508-517&pmid=31745289&doi=10.1038/s41431-019-0540-0&)

4. Del Re M, Quaquarini E, Sottotetti F, et al. Uncommon dihydropyrimidine dehydrogenase mutations and toxicity by fluoropyrimidines: A lethal case with a new variant. Pharmacogenomics. 2016;17:5–9. doi: 10.2217/pgs.15.146.  [DOI](https://doi.org/10.2217/pgs.15.146) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26651493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Uncommon%20dihydropyrimidine%20dehydrogenase%20mutations%20and%20toxicity%20by%20fluoropyrimidines:%20A%20lethal%20case%20with%20a%20new%20variant&author=M%20Del%20Re&author=E%20Quaquarini&author=F%20Sottotetti&volume=17&publication_year=2016&pages=5-9&pmid=26651493&doi=10.2217/pgs.15.146&)

5. van Kuilenburg AB, Meijer J, Maurer D, et al. Severe fluoropyrimidine toxicity due to novel and rare DPYD missense mutations, deletion and genomic amplification affecting DPD activity and mRNA splicing. Biochim Biophys Acta Mol Basis Dis. 2017;1863:721–730. doi: 10.1016/j.bbadis.2016.12.010.  [DOI](https://doi.org/10.1016/j.bbadis.2016.12.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28024938/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta%20Mol%20Basis%20Dis&title=Severe%20fluoropyrimidine%20toxicity%20due%20to%20novel%20and%20rare%20DPYD%20missense%20mutations,%20deletion%20and%20genomic%20amplification%20affecting%20DPD%20activity%20and%20mRNA%20splicing&author=AB%20van%20Kuilenburg&author=J%20Meijer&author=D%20Maurer&volume=1863&publication_year=2017&pages=721-730&pmid=28024938&doi=10.1016/j.bbadis.2016.12.010&)

6. Shrestha S, Tapper EE, Trogstad-Isaacson CS, et al. Dose modification for safe treatment of a compound complex heterozygous DPYD variant carrier with 5-fluorouracil. JCO Precis Oncol. doi: 10.1200/PO.18.00179. 10.1200/PO.18.00179  [10.1200/PO.18.00179](http://ascopubs.org/doi/full/10.1200/PO.18.00179) | [DOI](https://doi.org/10.1200/PO.18.00179) | [PMC free article](/articles/PMC7006951/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32039342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JCO%20Precis%20Oncol&title=Dose%20modification%20for%20safe%20treatment%20of%20a%20compound%20complex%20heterozygous%20DPYD%20variant%20carrier%20with%205-fluorouracil&author=S%20Shrestha&author=EE%20Tapper&author=CS%20Trogstad-Isaacson&pmid=32039342&doi=10.1200/PO.18.00179&)

7. Tong CC, Lam CW, Lam KO, et al. A novel DPYD variant associated with severe toxicity of fluoropyrimidines: Role of pre-emptive DPYD genotype screening. Front Oncol. 2018;8:279. doi: 10.3389/fonc.2018.00279.  [DOI](https://doi.org/10.3389/fonc.2018.00279) | [PMC free article](/articles/PMC6066555/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30087856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&title=A%20novel%20DPYD%20variant%20associated%20with%20severe%20toxicity%20of%20fluoropyrimidines:%20Role%20of%20pre-emptive%20DPYD%20genotype%20screening&author=CC%20Tong&author=CW%20Lam&author=KO%20Lam&volume=8&publication_year=2018&pages=279&pmid=30087856&doi=10.3389/fonc.2018.00279&)

8. Bukhari N, Azam F, Alfawaz M, et al. Identifying a novel DPYD polymorphism associated with severe toxicity to 5-FU chemotherapy in a Saudi patient. Case Rep Genet. 2019;2019:5150725. doi: 10.1155/2019/5150725.  [DOI](https://doi.org/10.1155/2019/5150725) | [PMC free article](/articles/PMC6720358/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31531249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Case%20Rep%20Genet&title=Identifying%20a%20novel%20DPYD%20polymorphism%20associated%20with%20severe%20toxicity%20to%205-FU%20chemotherapy%20in%20a%20Saudi%20patient&author=N%20Bukhari&author=F%20Azam&author=M%20Alfawaz&volume=2019&publication_year=2019&pages=5150725&pmid=31531249&doi=10.1155/2019/5150725&)

9. Palmirotta R, Lovero D, Delacour H, et al. Rare dihydropyrimidine dehydrogenase variants and toxicity by fluoropyrimidines: A case report. Front Oncol. 2019;9:139. doi: 10.3389/fonc.2019.00139.  [DOI](https://doi.org/10.3389/fonc.2019.00139) | [PMC free article](/articles/PMC6421267/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30915274/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&title=Rare%20dihydropyrimidine%20dehydrogenase%20variants%20and%20toxicity%20by%20fluoropyrimidines:%20A%20case%20report&author=R%20Palmirotta&author=D%20Lovero&author=H%20Delacour&volume=9&publication_year=2019&pages=139&pmid=30915274&doi=10.3389/fonc.2019.00139&)

10. Shrestha S, Zhang C, Jerde CR, et al. Gene-specific variant classifier (DPYD-varifier) to identify deleterious alleles of dihydropyrimidine dehydrogenase. Clin Pharmacol Ther. 2018;104:709–718. doi: 10.1002/cpt.1020.  [DOI](https://doi.org/10.1002/cpt.1020) | [PMC free article](/articles/PMC6043412/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29327356/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Gene-specific%20variant%20classifier%20(DPYD-varifier)%20to%20identify%20deleterious%20alleles%20of%20dihydropyrimidine%20dehydrogenase&author=S%20Shrestha&author=C%20Zhang&author=CR%20Jerde&volume=104&publication_year=2018&pages=709-718&pmid=29327356&doi=10.1002/cpt.1020&)

11. Robinson JT, Thorvaldsdóttir H, Winckler W, et al. Integrative genomics viewer. Nat Biotechnol. 2011;29:24–26. doi: 10.1038/nbt.1754.  [DOI](https://doi.org/10.1038/nbt.1754) | [PMC free article](/articles/PMC3346182/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21221095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Integrative%20genomics%20viewer&author=JT%20Robinson&author=H%20Thorvaldsd%C3%B3ttir&author=W%20Winckler&volume=29&publication_year=2011&pages=24-26&pmid=21221095&doi=10.1038/nbt.1754&)

12. van Kuilenburg AB, Meijer J, Dobritzsch D, et al. Identification of two novel mutations C79X and R235Q in the dihydropyrimidine dehydrogenase gene in a patient presenting with hematuria. Nucleosides Nucleotides Nucleic Acids. 2008;27:809–815. doi: 10.1080/15257770802146247.  [DOI](https://doi.org/10.1080/15257770802146247) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18600544/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleosides%20Nucleotides%20Nucleic%20Acids&title=Identification%20of%20two%20novel%20mutations%20C79X%20and%20R235Q%20in%20the%20dihydropyrimidine%20dehydrogenase%20gene%20in%20a%20patient%20presenting%20with%20hematuria&author=AB%20van%20Kuilenburg&author=J%20Meijer&author=D%20Dobritzsch&volume=27&publication_year=2008&pages=809-815&pmid=18600544&doi=10.1080/15257770802146247&)

13. Offer SM, Fossum CC, Wegner NJ, et al. Comparative functional analysis of DPYD variants of potential clinical relevance to dihydropyrimidine dehydrogenase activity. Cancer Res. 2014;74:2545–2554. doi: 10.1158/0008-5472.CAN-13-2482.  [DOI](https://doi.org/10.1158/0008-5472.CAN-13-2482) | [PMC free article](/articles/PMC4012613/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24648345/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Comparative%20functional%20analysis%20of%20DPYD%20variants%20of%20potential%20clinical%20relevance%20to%20dihydropyrimidine%20dehydrogenase%20activity&author=SM%20Offer&author=CC%20Fossum&author=NJ%20Wegner&volume=74&publication_year=2014&pages=2545-2554&pmid=24648345&doi=10.1158/0008-5472.CAN-13-2482&)

14. Vreken P, Van Kuilenburg AB, Meinsma R, et al. Dihydropyrimidine dehydrogenase (DPD) deficiency: Identification and expression of missense mutations C29R, R886H and R235W. Hum Genet. 1997;101:333–338. doi: 10.1007/s004390050637.  [DOI](https://doi.org/10.1007/s004390050637) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9439663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genet&title=Dihydropyrimidine%20dehydrogenase%20(DPD)%20deficiency:%20Identification%20and%20expression%20of%20missense%20mutations%20C29R,%20R886H%20and%20R235W&author=P%20Vreken&author=AB%20Van%20Kuilenburg&author=R%20Meinsma&volume=101&publication_year=1997&pages=333-338&pmid=9439663&doi=10.1007/s004390050637&)

15. Lohkamp B, Voevodskaya N, Lindqvist Y, et al. Insights into the mechanism of dihydropyrimidine dehydrogenase from site-directed mutagenesis targeting the active site loop and redox cofactor coordination. Biochim Biophys Acta. 2010;1804:2198–2206. doi: 10.1016/j.bbapap.2010.08.014.  [DOI](https://doi.org/10.1016/j.bbapap.2010.08.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20831907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Insights%20into%20the%20mechanism%20of%20dihydropyrimidine%20dehydrogenase%20from%20site-directed%20mutagenesis%20targeting%20the%20active%20site%20loop%20and%20redox%20cofactor%20coordination&author=B%20Lohkamp&author=N%20Voevodskaya&author=Y%20Lindqvist&volume=1804&publication_year=2010&pages=2198-2206&pmid=20831907&doi=10.1016/j.bbapap.2010.08.014&)

16. Weidensee S, Goettig P, Bertone M, et al. A mild phenotype of dihydropyrimidine dehydrogenase deficiency and developmental retardation associated with a missense mutation affecting cofactor binding. Clin Biochem. 2011;44:722–724. doi: 10.1016/j.clinbiochem.2011.03.033.  [DOI](https://doi.org/10.1016/j.clinbiochem.2011.03.033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21420945/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Biochem&title=A%20mild%20phenotype%20of%20dihydropyrimidine%20dehydrogenase%20deficiency%20and%20developmental%20retardation%20associated%20with%20a%20missense%20mutation%20affecting%20cofactor%20binding&author=S%20Weidensee&author=P%20Goettig&author=M%20Bertone&volume=44&publication_year=2011&pages=722-724&pmid=21420945&doi=10.1016/j.clinbiochem.2011.03.033&)

17. Mandola MV, Stoehlmacher J, Zhang W, et al. A 6 bp polymorphism in the thymidylate synthase gene causes message instability and is associated with decreased intratumoral TS mRNA levels. Pharmacogenetics. 2004;14:319–327. doi: 10.1097/00008571-200405000-00007.  [DOI](https://doi.org/10.1097/00008571-200405000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15115918/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=A%206%20bp%20polymorphism%20in%20the%20thymidylate%20synthase%20gene%20causes%20message%20instability%20and%20is%20associated%20with%20decreased%20intratumoral%20TS%20mRNA%20levels&author=MV%20Mandola&author=J%20Stoehlmacher&author=W%20Zhang&volume=14&publication_year=2004&pages=319-327&pmid=15115918&doi=10.1097/00008571-200405000-00007&)

18. Pullmann R, Jr, Abdelmohsen K, Lal A, et al. Differential stability of thymidylate synthase 3′-untranslated region polymorphic variants regulated by AUF1. J Biol Chem. 2006;281:23456–23463. doi: 10.1074/jbc.M600282200.  [DOI](https://doi.org/10.1074/jbc.M600282200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16787927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Differential%20stability%20of%20thymidylate%20synthase%203%E2%80%B2-untranslated%20region%20polymorphic%20variants%20regulated%20by%20AUF1&author=R%20Pullmann&author=K%20Abdelmohsen&author=A%20Lal&volume=281&publication_year=2006&pages=23456-23463&pmid=16787927&doi=10.1074/jbc.M600282200&)

19. Mandola MV, Stoehlmacher J, Muller-Weeks S, et al. A novel single nucleotide polymorphism within the 5′ tandem repeat polymorphism of the thymidylate synthase gene abolishes USF-1 binding and alters transcriptional activity. Cancer Res. 2003;63:2898–2904.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12782596/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=A%20novel%20single%20nucleotide%20polymorphism%20within%20the%205%E2%80%B2%20tandem%20repeat%20polymorphism%20of%20the%20thymidylate%20synthase%20gene%20abolishes%20USF-1%20binding%20and%20alters%20transcriptional%20activity&author=MV%20Mandola&author=J%20Stoehlmacher&author=S%20Muller-Weeks&volume=63&publication_year=2003&pages=2898-2904&pmid=12782596&)

20. Yawata A, Kim SR, Miyajima A, et al. Polymorphic tandem repeat sequences of the thymidylate synthase gene correlates with cellular-based sensitivity to fluoropyrimidine antitumor agents. Cancer Chemother Pharmacol. 2005;56:465–472. doi: 10.1007/s00280-005-1018-z.  [DOI](https://doi.org/10.1007/s00280-005-1018-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15918040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Polymorphic%20tandem%20repeat%20sequences%20of%20the%20thymidylate%20synthase%20gene%20correlates%20with%20cellular-based%20sensitivity%20to%20fluoropyrimidine%20antitumor%20agents&author=A%20Yawata&author=SR%20Kim&author=A%20Miyajima&volume=56&publication_year=2005&pages=465-472&pmid=15918040&doi=10.1007/s00280-005-1018-z&)

21. Lecomte T, Ferraz JM, Zinzindohoué F, et al. Thymidylate synthase gene polymorphism predicts toxicity in colorectal cancer patients receiving 5-fluorouracil-based chemotherapy. Clin Cancer Res. 2004;10:5880–5888. doi: 10.1158/1078-0432.CCR-04-0169.  [DOI](https://doi.org/10.1158/1078-0432.CCR-04-0169) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15355920/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Thymidylate%20synthase%20gene%20polymorphism%20predicts%20toxicity%20in%20colorectal%20cancer%20patients%20receiving%205-fluorouracil-based%20chemotherapy&author=T%20Lecomte&author=JM%20Ferraz&author=F%20Zinzindohou%C3%A9&volume=10&publication_year=2004&pages=5880-5888&pmid=15355920&doi=10.1158/1078-0432.CCR-04-0169&)

22. Jennings BA, Kwok CS, Willis G, et al. Functional polymorphisms of folate metabolism and response to chemotherapy for colorectal cancer, a systematic review and meta-analysis. Pharmacogenet Genomics. 2012;22:290–304. doi: 10.1097/FPC.0b013e328351875d.  [DOI](https://doi.org/10.1097/FPC.0b013e328351875d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22388795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Functional%20polymorphisms%20of%20folate%20metabolism%20and%20response%20to%20chemotherapy%20for%20colorectal%20cancer,%20a%20systematic%20review%20and%20meta-analysis&author=BA%20Jennings&author=CS%20Kwok&author=G%20Willis&volume=22&publication_year=2012&pages=290-304&pmid=22388795&doi=10.1097/FPC.0b013e328351875d&)

23. Rosmarin D, Palles C, Pagnamenta A, et al. A candidate gene study of capecitabine-related toxicity in colorectal cancer identifies new toxicity variants at DPYD and a putative role for ENOSF1 rather than TYMS. Gut. 2015;64:111–120. doi: 10.1136/gutjnl-2013-306571.  [DOI](https://doi.org/10.1136/gutjnl-2013-306571) | [PMC free article](/articles/PMC4283622/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24647007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut&title=A%20candidate%20gene%20study%20of%20capecitabine-related%20toxicity%20in%20colorectal%20cancer%20identifies%20new%20toxicity%20variants%20at%20DPYD%20and%20a%20putative%20role%20for%20ENOSF1%20rather%20than%20TYMS&author=D%20Rosmarin&author=C%20Palles&author=A%20Pagnamenta&volume=64&publication_year=2015&pages=111-120&pmid=24647007&doi=10.1136/gutjnl-2013-306571&)

24. Meulendijks D, Jacobs BA, Aliev A, et al. Increased risk of severe fluoropyrimidine-associated toxicity in patients carrying a G to C substitution in the first 28-bp tandem repeat of the thymidylate synthase 2R allele. Int J Cancer. 2016;138:245–253. doi: 10.1002/ijc.29694.  [DOI](https://doi.org/10.1002/ijc.29694) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26189437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Increased%20risk%20of%20severe%20fluoropyrimidine-associated%20toxicity%20in%20patients%20carrying%20a%20G%20to%20C%20substitution%20in%20the%20first%2028-bp%20tandem%20repeat%20of%20the%20thymidylate%20synthase%202R%20allele&author=D%20Meulendijks&author=BA%20Jacobs&author=A%20Aliev&volume=138&publication_year=2016&pages=245-253&pmid=26189437&doi=10.1002/ijc.29694&)

25. Rosmarin D, Palles C, Church D, et al. Genetic markers of toxicity from capecitabine and other fluorouracil-based regimens: Investigation in the QUASAR2 study, systematic review, and meta-analysis. J Clin Oncol. 2014;32:1031–1039. doi: 10.1200/JCO.2013.51.1857.  [DOI](https://doi.org/10.1200/JCO.2013.51.1857) | [PMC free article](/articles/PMC4879695/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24590654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Genetic%20markers%20of%20toxicity%20from%20capecitabine%20and%20other%20fluorouracil-based%20regimens:%20Investigation%20in%20the%20QUASAR2%20study,%20systematic%20review,%20and%20meta-analysis&author=D%20Rosmarin&author=C%20Palles&author=D%20Church&volume=32&publication_year=2014&pages=1031-1039&pmid=24590654&doi=10.1200/JCO.2013.51.1857&)

26. Hamzic S, Kummer D, Froehlich TK, et al. Evaluating the role of ENOSF1 and TYMS variants as predictors in fluoropyrimidine-related toxicities: An IPD meta-analysis. Pharmacol Res. 2020;152:104594. doi: 10.1016/j.phrs.2019.104594.  [DOI](https://doi.org/10.1016/j.phrs.2019.104594) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31838077/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Res&title=Evaluating%20the%20role%20of%20ENOSF1%20and%20TYMS%20variants%20as%20predictors%20in%20fluoropyrimidine-related%20toxicities:%20An%20IPD%20meta-analysis&author=S%20Hamzic&author=D%20Kummer&author=TK%20Froehlich&volume=152&publication_year=2020&pages=104594&pmid=31838077&doi=10.1016/j.phrs.2019.104594&)
