# Variation in RARG increases susceptibility to doxorubicin-induced cardiotoxicity in patient specific induced pluripotent stem cell-derived cardiomyocytes

## Metadata
**Authors:** Effimia Christidi, Haojun Huang, Sanam Shafaattalab, Agnes Maillet, Eric Lin, Kate Huang, Zachary Laksman, Margot K Davis, Glen F Tibbits, Liam R Brunham
**Journal:** Scientific Reports
**Date:** 2020 Jun 25
**DOI:** [10.1038/s41598-020-65979-x](https://doi.org/10.1038/s41598-020-65979-x)
**PMID:** 32587261
**PMCID:** PMC7316788
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7316788/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7316788/pdf/41598_2020_Article_65979.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7316788/pdf/41598_2020_Article_65979.pdf)

## Abstract

Doxorubicin is a potent anticancer drug used to treat a variety of cancer types. However, its use is limited by doxorubicin-induced cardiotoxicity (DIC). A missense variant in the RARG gene (S427L; rs2229774) has been implicated in susceptibility to DIC in a genome wide association study. The goal of this study was to investigate the functional role of this RARG variant in DIC. We used induced pluripotent stem cell derived cardiomyocytes (iPSC-CMs) from patients treated with doxorubicin. iPSC-CMs from individuals who experienced DIC (cases) showed significantly greater sensitivity to doxorubicin compared to iPSC-CMs from doxorubicin-treated individuals who did not develop DIC (controls) in cell viability and optical mapping experiments. Using CRISPR/Cas9, we generated isogenic cell lines that differed only at the RARG locus. Genetic correction of RARG-S427L to wild type resulted in reduced doxorubicin-induced double stranded DNA breaks, reactive oxygen species production, and cell death. Conversely, introduction of RARG-S427L increased susceptibility to doxorubicin. Finally, genetic disruption of the RARG gene resulted in protection from cell death due to doxorubicin treatment. Our findings suggest that the presence of RARG-S427L increases sensitivity to DIC, establishing a direct, causal role for this variant in DIC.

Subject terms: Induced pluripotent stem cells, Cardiovascular genetics

## Introduction

Doxorubicin is an anthracycline chemotherapeutic drug used in the treatment of solid organ and hematological malignancies for both adult and pediatric patients^1^. Despite its effectiveness as an anti-cancer agent, the use of doxorubicin is limited by cardiotoxicity which affects up to one quarter of patients^2^. Doxorubicin-induced cardiotoxicity (DIC) leads to cardiomyopathy with systolic dysfunction, arrhythmia, congestive heart failure, and in some cases death^1,3^. Despite extensive investigation, the mechanisms of DIC remain incompletely understood, and we lack the ability to predict this adverse drug reaction in individual patients.

Genetic association studies have identified several genetic variants that are associated with DIC^4^. However, lack of replication of these findings and the absence of functional validation has precluded their implementation as clinical tests. Among the variants with the strongest genetic evidence is a non-synonymous coding variant (rs2229774, S427L) in the retinoic acid receptor gamma gene, *RARG*, which was identified in a genome-wide association study (GWAS) of DIC (odds ratio = 4.7, p = 5.9 10^−8^)^5^. However, the directionality of this effect is controversial^6^, and genetic association studies cannot prove a causal relationship between a genetic locus and the phenotype of interest. As a consequence, direct functional assessment of the role of this variant in DIC is essential.

Human induced pluripotent stem cell-derived cardiomyocytes (iPSC-CMs) have emerged as a physiologically-relevant model system for studying DIC^7,8^, but to date the impact of specific genetic variants on DIC in this model has not been assessed. The availability of precise genome editing tools, such as CRISPR/Cas9, provides a powerful platform to assess causal relationships between genetic variants and disease processes. iPSC-CMs possess many advantages for studying drug toxicity including their potential for high throughput applications and a higher degree of homology with human cardiomyocytes, compared to animal models^9,10^. In addition, patient derived iPSC-CMs are genetically identical to the patient from whom they were derived, which allows for a direct comparison between the patient’s clinical response and *in vitro* phenotype. The goal of this study was to investigate the functional impact of *RARG-*S427L on DIC in isogenic, patient-derived iPSC-CMs.

## Results

### Increased sensitivity to doxorubicin in patient specific induced pluripotent stem cell-derived cardiomyocytes from individuals who experienced doxorubicin-induced cardiotoxicity

We recruited patients who had been treated with an anthracycline based chemotherapy regimen and either developed DIC (cases) or did not (controls) (see Supplementary Table [S1](#MOESM1) for enrolment criteria). The characteristics of the DIC cases and controls, including genotype for *RARG-*S427L, are shown in Table [1](#Tab1). The mean equivalent doxorubicin dose for cases was 342.5 mg/m^2^ vs. 306.7 mg/m^2^ for controls, (p = 0.6). We isolated peripheral blood mononuclear cells from each patient and performed reprogramming to iPSCs (Fig. [1a](#Fig1)). iPSCs displayed robust expression of pluripotency markers (Fig. [1b](#Fig1) and Supplementary Fig. [S1a](#MOESM1)). Karyotype analysis showed normal karyotypes in all cell lines with the exception of two passages of one cell line (DIC-023, Supplementary Fig. [S2a,b](#MOESM1)),which showed a possible amplification of the minimal critical chromosomal region 12p, an abnormality often seen in iPSCs^11,12^. Other passages of this clone displayed normal karyotype, as did genome edited cell lines derived from this clone. Trilineage differentiation to the three germ layers confirmed pluripotency of these cells (Supplementary Fig. [S1b–d](#MOESM1)). We performed directed differentiation of the iPSCs to CMs, which expressed typical cardiomyocyte markers such as cardiac troponin T (cTnT, 75% cTnT positive cells), Sarcomeric α-actinin (SAA) and Myosin heavy chain 7 (MYH7) (Fig. [1c–e](#Fig1) and Supplementary Fig. S3, S4).

### Table 1.

| Cases | Type of Cancer | Sex | Age at enrolment (years) | Age at treatment (years) | Cumulative Anthracycline Dose (mg/m2) | Post treatment Left Ventricular Ejection Fraction (LVEF) | RARG genotype |
| --- | --- | --- | --- | --- | --- | --- | --- |
| DIC-001 | Breast | F | 70 | 56 | 300 (DOX) | 20% | WT/WT |
| DIC-002 | Sarcoma | F | 52 | 47 | 450 (DOX) | 36% | WT/WT |
| DIC-003* | Breast | F | 87 | 68 | 320 (DOX) | 36% | WT/S427L |
| DIC-008 | Breast | F | 67 | 48 | 300 (DOX) | 20% | WT/WT |
| Controls |  |  |  |  |  |  |  |
| DIC-014** | Sarcoma | F | 32 | 15 | 420 (DOX) | 60% | WT/WT |
| DIC-021 | Lymphoma | F | 27 | 11 | 180 (DAUN) | 60% | WT/S427L |
| DIC-023* | Breast | F | 65 | 51 | 320 (DOX) | 55% | WT/WT |

Table 1 Caption: Characteristics of cases and controls.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97d2/7316788/123c4d4f1e47/41598_2020_65979_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7316788_41598_2020_65979_Fig1_HTML.jpg)

Generation of patient specific induced pluripotent stem cell-derived cardiomyocytes. (a) Representative photomicrographs of peripheral blood mononuclear cells and induced pluripotent stem cell (iPSC) colonies. (b) Confocal microscopy images showing presence of Tra1–60, NANOG and DAPI, scale bar = 50μm. (c) Cardiac Troponin T and Myosin heavy chain 7 (cTnT, MYH7) mRNA expression in iPSC-derived cardiomyocytes (CM), n = 10. Data are shown as mean ± s.e.m. (d) Confocal microscopy images showing the presence of cTnT and Sarcomeric α-actinin (SAA), scale bar = 20μm. (e) Flow cytometry analysis of cTnT expression in iPSC-derived cardiomyocytes.

We compared the *in vitro* sensitivity to doxorubicin of iPSC-CMs from DIC cases and controls. iPSC-CMs from DIC cases displayed significantly greater susceptibility to doxorubicin-induced cell death compared to controls (Fig. [2a,b](#Fig2) cases: IC_50_ = 0.91 ± 0.35μΜ, pooled data from n = 4 patients, controls: IC_50_ = 2.77 ± 0.57μΜ, n = 3 patients, p = 0.003). This finding confirms that the clinical susceptibility to DIC is mirrored by increased sensitivity to the cardiotoxic effects of the drug *in vitro*.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97d2/7316788/e0dab5756d79/41598_2020_65979_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7316788_41598_2020_65979_Fig2_HTML.jpg)

Modelling sensitivity to doxorubicin in patient specific induced pluripotent stem cell-derived cardiomyocytes from cases and controls. (a) Cell viability and (b) IC50 of induced pluripotent stem cell-derived cardiomyocytes (iPSC-CMs) from cases and controls using Cell Titer-Glo after 24 h of doxorubicin treatment. Graphs show pooled data from n = 4 cases and n = 3 controls, each patient cell line was assessed three independent times, in technical triplicates. (c,d) Representative optical mapping experiments showing the action potential of one case (c) and one control (d) cell line in response to increasing doses of doxorubicin, upon 20 mins of doxorubicin incubation. (e) Change of action potential duration at 50% of repolarization 1 μΜ and 10 μΜ doxorubicin n = 3 cases, n = 3 controls, relative to 0μΜ doxorubicin. Data points represent independent, biological replicates of pooled data of cases and controls measured in multiple differentiation batches of one cell line per patient. Data are shown as mean ± s.e.m. *p < 0.05, **p < 0.01, ****p < 0.0001, t-test.

Doxorubicin causes a variety of electrophysiological changes acutely during its administration including nonspecific ST and T-wave changes, decreased QRS voltage, prolongation of the QT interval and ventricular premature beats^3,18,19^. Previous research in experimental models has shown that doxorubicin can cause shortening or prolongation of the action potential duration (APD)^20–22^. We investigated whether iPSC-CMs from cases are more sensitive to the electrophysiological effects of doxorubicin than controls, by assessing the impact of doxorubicin on voltage transients using optical mapping^23,24^. We observed that doxorubicin caused shortening of the APD in a dose-dependent manner, and that this effect was significantly more pronounced in cases than controls (Fig. [2c,d,e](#Fig2), %change of APD_50_ with 1 μM doxorubicin relative to 0 μM: cases = 75.1 ± 10.1%, controls = 98.1 ± 2.3%, n = 3 patients each, p = 0.04).

### Effect of RARG-S427L on doxorubicin induced cardiotoxicity

We hypothesized that part of the increased susceptibility to DIC among cases may be mediated by the presence of the *RARG*-S427L variant. To test this hypothesis, we took a two-step approach. First, we performed genome editing to correct this variant to the wild type sequence in a case that was heterozygous for S427L. Secondly, we used a control patient cell line that was homozygous wild type at *RARG* (WT/WT) and introduced the variant (S427L/WT) (Fig. [3a,b](#Fig3), Table [1](#Tab1)). Sequencing confirmed successful editing of the two cell lines (Fig. [3c,d](#Fig3)) while sequencing of predicted off-target sites did not detect evidence of off-target genome editing (Supplementary Fig. [S5a–c](#MOESM1)). In order to assess that pluripotency and differentiation efficiency of iPSCs was not affected by the genome editing process or the presence of the variant, we performed staining for NANOG and Tra 1-60 (Supplementary Fig. [S1a](#MOESM1)). Both markers were robustly expressed in both genome-edited and unedited iPSCs. In addition, genome-edited and unedited iPSC-CMs displayed similar cTnT expression assessed by flow cytometry (Fig. [3e,f](#Fig3) 76% cTnT positive cells in *RARG*-WT/WT and 74% cTnT positive cells in *RARG*-S427L/WT) and by qPCR (Supplementary Fig. [S6a,b](#MOESM1)), suggesting that differentiation efficiency was not altered. *RARG* mRNA expression was significantly higher in cell lines with the *RARG*-S427L variant (Fig. [3g,](#Fig3) p = 0.03 and [3h](#Fig3) p = 0.03).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97d2/7316788/5cd67dd7ce7a/41598_2020_65979_Fig3_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7316788_41598_2020_65979_Fig3_HTML.jpg)

Genome editing experimental approach and comparison of edited and unedited isogenic cell lines. (a) Experimental approach of CRISPR/Cas9 experiments, correction of RARG-S427L/WT in a case (DIC-003) patient cell line and insertion of RARG-S427L in a control (DIC-023) patient cell line. Images have been acquired from Servier Medical Art by Servier licensed under a Creative Commons Attribution 3.0 Unported License. (b) Schematic overview of RARG locus and CRISPR/Cas9 genome editing approach. (c) Sanger sequencing results showing correction of RARG-S427L/WT to WT/WT (DIC-003). (d) Sanger sequencing results showing introduction of RARG-S427L/WT to a control WT/WT patient cell line (DIC-023). Flow cytometry results of cTnT expression in (e) RARG-WT/WT and in (f) its isogenic genome edited S427L/WT cell line. (g) RT-qPCR results of RARG expression, in the heterozygous case (DIC-003) and the isogenic corrected WT/WT cell line (h) RT-qPCR results of RARG expression in the homozygous control (DIC-023) cell line (WT/WT) and the isogenic edited S427L/WT cell line. Data are shown as mean ± s.e.m., n = 3, representing biological, independent replicates *p < 0.05, t-test.

To assess the functional effect of *RARG*-S427L, we performed assays of cell viability in genome-edited iPSC-CMs and their isogenic controls. Genetic correction of S427L to wild type in iPSC-CMs from a DIC case (DIC-003, Table [1](#Tab1)) resulted in significantly reduced doxorubicin-induced cell death (Fig. [4a,b](#Fig4), IC_50_ 1.2μΜ ± 0.34 μΜ vs. 4.4Μμ ± 1.9 μΜ, p = 0.04, n = 3). We further investigated the impact of *RARG*-S427L on doxorubicin-induced DNA double strand breaks (DSBs) and reactive oxygen species (ROS) production, both validated assays of DIC in iPSC-CMs^7,8^. Correction of S427L to wild type resulted in reduced staining for γ-Η2Α.X, a marker of DSB, indicating less doxorubicin-induced DSBs, relative to iPSC-CMs heterozygous for S427L (Fig. [4c,d](#Fig4), 2.71 ± 0.56 vs. 1.27 ± 0.33 relative fluorescent units, p = 0.02, n = 3). Similarly, we found a statistically significant reduction in doxorubicin-induced ROS production in the genome-corrected compared to the heterozygous iPSC-CMs (Fig. [4e,f](#Fig4), 2.14 ± 0.32 vs. 1.18 ± 0.26 relative fluorescent units, p = 0.02, n = 3). These findings establish that the presence of *RARG*-S427L increases susceptibility to these important cellular phenotypes of DIC.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97d2/7316788/380f3a9d277f/41598_2020_65979_Fig4_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7316788_41598_2020_65979_Fig4_HTML.jpg)

Genetic correction of RARG-S427L in iPSC-CMs decreases sensitivity to doxorubicin-induced cardiotoxicity. (a) Cell viability curve and (b) IC50 of RARG-S427L/WT and its corrected isogenic control (WT/WT) assessed using Cell Titer-Glo after 24 h of doxorubicin incubation (n = 3). (c) Representative images showing staining for γH2A.X. Scale bar = 20 μm. (d) Relative to vehicle control fluorescent intensity of γH2A.X staining (n = 3). (e) Representative images showing reactive oxygen species (ROS) production, scale bar = 50 μm. (f) Relative to vehicle control fluorescent intensity of ROS production (n = 3). All experiments represent biological, independent replicates, each experiment was conducted in triplicates and are shown as mean ± s.e.m., *p < 0.05, ****p < 0.0001, t-test.

Next, to investigate if the S427L variant is sufficient to increase susceptibility to DIC, we introduced this variant into the iPSCs of a control individual (DIC-023, Table [1](#Tab1)) who was wild type for *RARG* (Fig. [3a,d](#Fig3)). Introduction of S427L resulted in a significant increase in sensitivity to doxorubicin-induced cell death (Fig. [5a,b](#Fig5), IC_50_. 2.14 ± 1.2 μΜ vs. 0.53 μΜ ± 0.08 μΜ, p = 0.02, n = 5), indicating that the presence of this variant is sufficient to increase susceptibility to DIC *in vitro*. Introduction of *RARG*-S427L also led to increased doxorubicin-induced DSBs (Fig. [5c,d](#Fig5), 1.74 ± 0.51 vs 3.67 ± 1.75 relative fluorescent units, p = 0.04 n = 5) and ROS production (Fig. [5e,f](#Fig5), 0.91 ± 0.40 vs 1.8 ± 0.59 relative fluorescent units, p = 0.046, n = 4).

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97d2/7316788/0d30f134f4f5/41598_2020_65979_Fig5_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7316788_41598_2020_65979_Fig5_HTML.jpg)

Introduction of RARG-S427L in iPSC-CMs increases sensitivity to doxorubicin-induced cardiotoxicity. (a) Cell viability curve and (b) IC50 of RARG-WT/WT and its isogenic cell line with genome edited S427L/WT pair (n = 5). (c) Representative images showing staining for γH2A.x. Scale bar = 20μm. (d) Relative to vehicle control fluorescent intensity of γH2A.X (n = 5). (e) Representative images showing ROS production. Scale bar = 50 μm. (f) Relative to vehicle control ROS production (n = 4). All experiments represent biological, independent replicates, each experiment was conducted in triplicates and are shown as mean ± s.e.m., *p < 0.05, ****p < 0.0001, t-test.

A potential mechanism by which *RARG*-S427L could make cardiomyocytes more sensitive to doxorubicin is via differential regulation of topoisomerase IIβ (*TOP2B)* compared to wild type^5^. *TOP2B* is thought to be a critical mediator of DIC and RARG has been shown to bind to the promoter of *TOP2B* affecting its transcription^8,25,26^. We investigated the regulation of *TOP2B* expression by doxorubicin in *RARG*-S427L/WT iPSC-CMs and the isogenic corrected WT/WT cell line. In *RARG*-S427L/WT, *TOP2B* and *RARG* expression increased in response to doxorubicin treatment (Fig. [6a,b](#Fig6), Supplementary Fig. [S7](#MOESM1)). In contrast, *TOP2B* expression decreased in response to doxorubicin in genetically corrected *RARG*-WT/WT iPSC-CMs (Fig. [6c](#Fig6)) while *RARG* expression did not (Fig. [6d](#Fig6)). In both the case with the S427L variant and in the control which the S427L variant was inserted via CRISPR/Cas9 we observed increased upregulation of *RARG* and *TOP2B* compared to their isogenic WT/WT iPSC-CMs (Supplementary Fig. [S7a–d](#MOESM1)). This suggests that *RARG*-S427L increases susceptibility to DIC, in part, by leading to de-repression of *TOP2B*.

### Figure 6.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97d2/7316788/3377c57ef9d6/41598_2020_65979_Fig6_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7316788_41598_2020_65979_Fig6_HTML.jpg)

Effect of doxorubicin treatment on TOP2B and RARG in RARG-S427L/WT and corrected RARG-WT/WT. RT-qPCR gene expression of (a) TOP2B and (b) RARG relative to vehicle control in RARG-S427L/WT, (c) TOP2B and (d) RARG expression relative to vehicle control in corrected RARG-WT/WT. Data are biological, independent replicates and are shown as mean ± s.e.m., n = 3, *p < 0.05, **p < 0.01, n.s. = not significant, t-test.

Our finding of increased mRNA expression of *RARG* in iPSC-CMs that carry the S427L allele suggests that the increased expression of this gene may play a role in the increased susceptibility to DIC. As such, we hypothesized that genetic deletion of *RARG* may be protective against DIC. We used CRISPR/Cas9 to disrupt *RARG* in two different cell lines, an embryonic stem cell line (ESC) (Fig. [7a–d](#Fig7)) and in a control patient iPSC line (DIC-014, Table [1](#Tab1), Fig. [7e–h](#Fig7)). In both cell lines, genetic disruption of *RARG* led to a significant reduction in *RARG* expression (Supplementary Fig. [S8a,b](#MOESM1)). Upon doxorubicin treatment, *RARG* knock out lines showed protection from DIC (Fig. [7c,d,g and h](#Fig7) IC50: WT/WT = 69.74μΜ and KO = 91.80μΜ, p = 0.04, n = 6 in ESC-CM, IC50: WT/WT = 2.2 ± 1.54μΜ and KO = 6.7 ± 2.64μΜ p = 0.02, n = 4 in iPSC-CMs), consistent with the concept that increased *RARG* expression is associated with increased susceptibility to DIC.

### Figure 7.

![Figure 7](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97d2/7316788/ab29692189b9/41598_2020_65979_Fig7_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7316788_41598_2020_65979_Fig7_HTML.jpg)

Disruption of RARG protects from doxorubicin induced cardiotoxicity. (a) Schematic overview of RARG locus and CRISPR/Cas9 knock out in ESC-CMs. (b) RARG-KO Sanger sequencing results in ESCs, showing a 35 base pair (bp) deletion in one allele and a 4 bp insertion in the other allele. (c) Cell viability curve and (d) IC50 of RARG-WT/WT and its isogenic cell line with disruption of RARG gene in ESC-CMs (n = 6). (e) Schematic overview of RARG locus and CRISPR/Cas9 knock out in iPSC-CMs from a control patient. (f) RARG-KO Sanger sequencing results in iPSCs, showing a 476 bp deletion. (g) Cell viability curve and (h) IC50 of RARG-WT/WT cell line and KO in iPSC-CMs (n = 4). Data are biological, independent replicates shown as mean ± s.e.m, *p < 0.05, ****p < 0.0001, t-test.

## Discussion

Here we show that a genetic variant associated with DIC in GWAS^5^ has a direct and causal relationship with DIC in patient-specific iPSC-CMs. The major findings of our study are: that iPSC-CMs from individuals who have suffered from DIC retain their sensitivity to doxorubicin *in vitro*; genetic correction of the S427L variant reduces sensitivity to DIC; introduction of this variant increases the sensitivity to DIC; and, that disruption of *RARG* protects from DIC.

Genetic association studies can identify genomic loci associated with specific traits, but cannot prove causality of a specific genetic variant, because of the possibility of linkage disequilibrium with other observed or unobserved genetic variants. While *RARG*-S427L has been implicated in DIC through GWAS, the results have been inconsistent, and the directionality of effect has been unclear^5,6^. Our results provide evidence that S427L directly increases sensitivity to DIC, independent of other genetic variants. An important strength of our study is that we were able to assess the functional impact of S427L in its endogenous genomic context, and without other changes to the genome, proving that this variant itself plays a functional role in increasing susceptibility to DIC. This result has important implications for the use of pharmacogenetic testing for this variant to identify individuals who may be at high risk for DIC.

Our finding that iPSC-CMs from DIC cases are more sensitive to doxorubicin than iPSC-CMs from doxorubicin-treated controls provides important independent validation of a previous finding^7^ and further supports the notion that iPSC-CMs represent a useful model system for dissecting the molecular mechanisms of DIC^8^ or other forms of drug-induced cardiotoxicity. An important implication of this finding is that patient-specific iPSC-CMs may be able to predict an individual’s response to a medication prior to their treatment with it. While this concept would require further validation in a prospective manner, this raises the intriguing possibility that iPSC-CMs could serve as a useful model system, in general, for assessing individual response to potentially cardiotoxic medications.

Our data have implications for understanding the mechanism of how *RARG* influences risk of DIC. Our finding that the presence of the S427L allele leads to increased expression of *RARG* and increased susceptibility to DIC suggests that *RARG* is associated with increased cell death. Similarly, we found that disruption of *RARG* decreased susceptibility to DIC. This is consistent with a recent study which showed that knocking out *RARG* in mouse embryonic fibroblasts protects cells from chemotherapeutic DNA damaging agents including doxorubicin^27^. We also found that *RARG*-S427L leads to doxorubicin-induced upregulation of *TOP2B*, which is a key player in the pathogenesis of DIC as doxorubicin can stabilize the interaction of *TOP2B* with the DNA and prevent its re-ligation, generating double strand breaks^25^. *RARG* is a transcription factor that can both repress and activate gene expression and can bind to the promoter of *TOP2B*^26^, directly affecting its transcription. As a result, this finding could be due to impaired repression (or increased activation) of *TOP2B* by *RARG*-S427L as has been previously suggested^5^. Interestingly this is further supported by RARG protein structure as S427L is located in the F-domain of RARG, nine amino acids downstream of the ligand binding domain (LBD)^28^. The role of the F-domain is to stabilize the LBD^29^. Conformational changes of the LBD expose binding sites for co-activators and co-repressors^30,31^. In RARB, which shares 73% sequence identity with RARG, a frame shift mutation at amino acid position 426 in the F terminal domain lesds to increased co-repressor binding, while deletion of the F-domain results in increased co-activator binding in RARG^29–31^. As a result, it is possible that RARG- S427L could result in a favored activation versus repressive state leading to greater expression of *RARG* and *TOP2B*. Our finding of differential regulation of TOP2B by doxorubicin in iPSC-CMs with or without S427L suggests that RARG influences susceptibility to DIC via a mechanism that involves TOP2B.

Our study has limitations that merit consideration. First, the patients who experienced DIC are older in age than the controls and therefore possess more comorbidities. Secondly, to characterize our iPSCs we used commercially available kits for trilineage differentiation potential and karyotypic abnormalities instead of the gold-standards teratoma and karyotyping respectively. However, for the purpose of this study this approach is reasonable and in line with common practice in the field of disease modelling. Furthermore, iPSC-CMs are not a pure population of cells. Recent single cell RNA-seq data have revealed transcriptional heterogeneity in iPSC-CMs arising at different stages of the differentiation process^32–34^. In addition, iPSC-CMs are a mixture of cardiac cells, including nodal-, atrial- and ventricular-like cells, with ventricular-like cells being the most abundant. While ventricular cells appear to be the most relevant for the clinical phenotype of DIC, whether doxorubicin has cell-type specific effects is unknown and requires further study. Moreover, the *in vitro* cellular assays shown here assess primarily the acute effects of doxorubicin. While the acute toxic effects of doxorubicin are thought to contribute to the longer term cardiotoxicity of this drug^35,36^, further studies will be needed to explore the more chronic effects of this drug using the iPSC-CM model. An additional issue is that the iPSC-CMs model is largely limited to investigation of cell-autonomous effects of doxorubicin. More complex effects that may result from crosstalk between different cell types or organ systems may not be accurately recapitulated using this approach. Moreover, while our results establish a functional role of *RARG*-S427L in DIC, this variant is only carried by ~30% of individuals who experience DIC^5^, and therefore cannot account for, in entirety, the increased susceptibility to DIC observed in iPSC-CMs from cases. This implies that while *RARG*-S427L represents one important pathogenic mechanism that leads to DIC, other mechanisms are likely to exist as well. Additional characterization of these cells may lead to the identification of other genetic or epigenetic factors that modulate sensitivity to doxorubicin. Finally, this study involved one case who was a carrier of the S427L variant, in which the genotype was corrected to wild type, and one cell line from a drug-tolerant control, in which the S427L variant was introduced. The advantage of this approach is that each cell line serves as its own isogenic control, and differs only at the S427L locus, which allows us to draw conclusions about the effect of this variant in isolation of other variables. However, more subjects harboring this mutation and who experienced DIC are needed to further support these findings and strengthen our finding that *RARG*-S427L is a causal variant for DIC.

In summary, we show that iPSC-CMs can identify individuals at risk for DIC, and that *RARG*-S427L has a direct and causal role in increasing susceptibility to doxorubicin. These results have implications for personalized risk prediction to prevent this devastating adverse drug reaction.

## Methods

### Participation recruitment and subject details

Participants were recruited from the Cardio-oncology Clinic at Vancouver General Hospital and provided written informed consent. Exclusion and inclusion criteria are shown in Supplementary Table [S1](#MOESM1). In total, we generated iPSCs from 7 female subjects (4 cases and 3 controls). Details of these individuals are provided in Table [1](#Tab1). This study was approved by the University of British Columbia-Providence Health Care (UBC-PHC) Research Ethics Board. All experiments were performed in accordance with relevant guidelines and regulations.

### TaqMan genotyping

Saliva samples were used to extract DNA (DNAgenotek). SNP Taqman genotyping assay (Invitrogen) with the rs2229774 probe was used to genotype *RARG*-S427L.

### iPSC generation and characterization

Peripheral blood mononuclear cells (PBMCs) were isolated from patients’ blood using SepMate PBMC isolation protocol (Stemcell Technologies). PBMCs in medium containing IL-2, Fetal Bovine Serum (FBS) and RPMI 1640 were incubated in a CD-3 coated plate for 5 days, as previously described^37^. CytoTune Sendai iPSC reprogramming Kit (Life Technologies) was used to deliver the Yamanaka factors in 500,000 T-activated cells. After 24 hours of infection, cells were collected and seeded in plates coated with mouse embryonic fibroblasts in Primate ES Cell media (Reprocell) supplemented with basic fibroblast growth factor (b-fgf). Cells were allowed to grow for 2–3 weeks (changing media every 2 days) and emerging colonies were picked manually and seeded in Matrigel coated plates and were cultured in mTeSR1 (Stemcell Technologies). Subsequent passaging was performed when confluency was reached (5–7 days). Versene 0.02% EDTA (Life Technologies) was used as a dissociating reagent, and mTeSR1 with Rock inhibitor (Y27632, Tocris Bioscience) at 10 μM concentration was used for day 1 of passaging and then mTeSR1 was changed daily. Cultures were kept at 37 °C and 5% CO_2_ incubators. Pluripotency characterization was performed using RT-qPCR for OCT4 and NANOG and protein expression using confocal microscopy with Tra 1–60 and NANOG antibodies. hPSC Analysis Kit (Stemcell Technologies) was used to detect karyotypic abnormalities following the manufacturer’s protocol. Trilineage differentiation was performed using the StemDiff Trilineage Differentiation Kit (Stemcell Technologies) following the manufacturer’s protocol, and differentiation was characterized using RT-qPCR with markers for endoderm (SOX17, FOXA2), mesoderm (Brachyury (T), MIXL1, CDX2) and ectoderm (OTX2, PAX 6).

### iPSCs characterization with confocal microscopy

iPSCs were harvested using Versene 0.02% EDTA (Life Technologies) and were seeded on Matrigel coated coverslips and were let to grow for 5 days in mTeSR1. Cells were washed with 3 X PBS, fixed using 4% parafomaldehyde (15 mins), washed and then permabilized with 0.2% Triton-X (15 mins), washed and blocked for 3 hours in 3% goat serum and incubated with Tra 1–60 (Abcam) and NANOG (Abcam) overnight in 4 °C. Cells were washed with 3 X PBS and incubated for 1 hour with secondary antibodies Alexa-Fluor 488 (Abcam) and Alexa-Fluor 563 (Abcam). Cells were washed with 3 X PBS and mounted on a microscope slide with antifade medium containing DAPI (VECTASHIELD, VECTOR laboratories).

### Quantitative real time PCR

After RNA extraction (RNeasy mini kit, Qiagen) and cDNA synthesis (SuperscriptR II Reverse Transcription Kit, Life Technologies), quantitative real-time PCR was performed on Viia 7 Real Time PCR System (Thermo Fisher Scientific) using SYBR Select Master Mix (Life Technologies) with a volume of 10 μL. The concentration of mRNA was normalized to the mRNA of β-actin, and fold change was calculated using the ΔΔ_Ct_. Data on graphs represent biological, independent replicates conducted at least three independent times, with each experiment conducted in technical triplicates.

### CRISPR/Cas9 genome editing in iPSCs

CRISPR/Cas9 techniques were adapted from published protocols^38–44^. To knock out *RARG*, we designed a pair of 20 bp guide RNAs targeting exon 4 and the intron between exon 5 and 6 of the *RARG* gene using GPP sgRNA Designer ([https://portals.broadinstitute.org/gpp/public/analysis-tools/sgrna-design](https://portals.broadinstitute.org/gpp/public/analysis-tools/sgrna-design))^43,44^. This created a 476 bp deletion in the genome, a 76 amino acid deletion on the DNA binding domain of RARG protein and a premature stop codon was introduced in exon 6. To edit the *RARG* gene in the variant site (to correct or introduce the variant) we designed a 20 bp gRNA (gRNA-HDR) targeting exon 10 (using GPP sgRNA Designer) and two 127 nt single-stranded oligodeoxynucleotides (ssODNs) homology templates with or without the S427L variant that both had the same silencing mutation at the PAM site^38,39,44^. Electroporation (Amaxa Nucleofector 2b with program A-023) was used to deliver 5 µL of ribonucleoprotein (RNP) complex, 120 pmol *in vitro* synthesized gRNA duplex (crRNA:tracrRNA), 104 pmol Cas9 nuclease (Integrated DNA Technologies, following manufacturer’s instructions) and 2 pmol ssODNs in 100 µL of 8 × 10^5^ iPSCs using Human Stem Cell Nucleofector Kit 1(Lonza, following manufacturer’s instructions)^40,42^. Subsequently, cells were seeded on Matrigel-coated 6-well plates (for 48 hours) and then seeded in 10 cm dishes at a low density (for 9 days)^41^. Colonies were then picked manually, transferred to a 96 well plate and when confluency was reached genomic DNA was extracted by QuickExtract DNA Extraction Solution (Epicentre)^38,45^. To identify colonies of interest Taq 5X Master Mix (New England Biolabs) was used to amplify the target region and restriction enzyme BseRI (New England Biolabs) which covers the SNP site was used to perform restriction fragment length polymorphism (RFLP) assay^38^. Colonies with targeted 476 bp deletion for the *RARG*-KO were identified based on different DNA fragment size of PCR products that separated on agarose gels. The PCR products of colonies of interest were purified by QIAquick PCR Purification Kit (QIAGEN) and sequenced by Sanger sequencing. (More details on this method are in Supplemental Information).

### CRISPR/Cas9 genome editing in ESCs

CRISPR/Cas9 techniques were adapted from published protocols^8,38,46^. A pair of single guide RNAs targeting exon 5 of the *RARG* gene were cloned into the BbsI site of pSpCas9n(BB)-2A-Puro (PX462) (Addgene). Neon Transfection System (Life Technologies) was used to insert 2.5 μg of plasmid DNA into 1 × 10^6^ NKX2-5^eGFP/w^ cells. After electroporation the cells were seeded on a Matrigel-coated 6 well plates and twenty four hours later they were treated with 1 μg/mL puromycin for 24 h. Two weeks later colonies were picked manually and genomic DNA was extracted in order to PCR amplify the *RARG* gene. T7E1 was used to identify mutations which was digested and assessed by gel electrophoresis. The purified PCR products of positive clones were cloned in pGem-T easy vector (Promega) and transformed in One Shot TOP10 E.coli competent cells (Life Technologies). After ampicillin selection and amplification, DNA from minimum 10 bacterial clones per sample was extracted and sent for Sanger sequencing. The resulting clone had a 35pb deletion in one allele and a 4pb insertion in the other, creating a premature stop codon on exon 6.

### iPSC-CM differentiation and characterization

iPSCs were differentiated to CMs using Wnt activation with CHIR99021 (day 0) and Wnt inhibition with IWP2 (day 3) cultured in RPMI and B27 medium without insulin. At day 5 medium was changed to RPMI/B27 without insulin and at day 7 onwards RPMI/B27 with insulin was used^47^. Spontaneous beating was observed on day 8. iPSC-CMs were characterized using qPCR and flow cytometry for cTnT and MYH7. For each experiment, we performed biological replicates using different differentiation batches of the same iPSC clone. All experiments were performed between days 30–45 of differentiation.

### Flow cytometry

iPSC-CMs were dissociated using 0.5U/ml Liberase (Roche), >50 U/ml DnaseI (Stemcell Technologies) in RPMI medium and then trypsinized. Cells were washed with DPBS, stained with live-dead stain (Fixable Viability Stain 520, BD Horizon) and anti-cTnT-BV421 antibodies (BD Horizon). Dead cells were excluded during flow cytometry analysis and gating was determined using an isotype antibody. Flow cytometry data were acquired on Gallios Flow Cytometer and analyzed on Kaluza.

### iPSC-CMs characterization with confocal microscopy

Cells were seeded on Matrigel-coated coverslips and were left to attach for 2 days. Cells were washed with 3 X PBS, fixed with 4% parafomaldehyde (15 mins), washed and permabilized with 0.2% Triton-X (15 mins), washed and blocked for 3 hours in 3% goat serum and incubated with cTnT (Abcam) and sarcomeric α-actinin (Abcam) overnight. Cells were washed with 3 X PBS and incubated for one hour with secondary antibody Alexa Fluor-563 (Abcam) and Alexa Fluor-488 (Abcam) and were then washed and mounted on a slide using Antifate mounting medium containing DAPI (VECTASHIELD, VECTOR laboratories). Slides were viewed on Zeiss LSM880 confocal microscope.

### Cell viability assay

Cells were dissociated using TrypLE, seeded on 96-well Matrigel coated plates and incubated with increasing doses of doxorubicin for 24 hours. Cells were then washed with 1xPBS and incubated with Cell Titer-Glo reagents (Promega) for 10 minutes at room temperature. Luminescence was measure in Spectramax plate reader (Molecular Diagnostics). Data on graphs represent biological, independent replicates conducted at least three independent times, with each experiment conducted in technical triplicates. All experiments shown were conducted in multiple differentiation batches, from one cell line from each patient.

### Optical mapping

iPSCs-CMs were superfused with IMDM supplemented with NaCl (final concentrations in mmol/L: 140 NaCl, 3.6 KCL, 1.2 CaCl2, 1 MgCl2. 10 HEPES and 5.5 D-glucose) and loaded with 15 μM of the potentiometric voltage sensitive dye RH-237 (Molecular Probes, Eugene, OR). Blebbistatin, a myosin ATPase inhibitor (Sigma-Aldrich), was employed to avoid motion artifact. The iPSC-CMs were kept in a 37 °C plate and were excited by 532 nm LEDs. Voltage sensitive dye, RH-237 emission was monitored using >710 nm long-pass and signal was captured with a single Hamamatsu ORCA Flash 4 digital CMOS camera. Doxorubicin was diluted in DMSO and added to the culture dish in sequentially increasing concentrations (1.0, and 10.0 μM) at 20-minute intervals. Electrical field stimulation was applied using stainless steel electrodes. The electrodes were ~1 cm apart and were placed in the imaging chamber. Signal processing of the optically mapped data was performed using a custom IDL software program^23,24^. APD_50_ data represent independent, biological replicates with n = 3 cases and n = 3 controls.

### γ-Η2Α.X staining

Cells were seeded on Gelatin-coated coverslips and treated with doxorubicin for 20 hours. After treatment, cells were washed with 3 X PBS, fixed with 4% parafomaldehyde (15 mins), washed permabilized with 0.2% Triton-X (15 mins), washed and blocked for 3 hours in 3% goat serum and incubated with γ-Η2Α.X (Abcam) overnight. Cells were washed with 3 X PBS and incubated for one hour in secondary antibody Alexa Fluor-488 (Abcam) and were then washed and mounted on a slide using Antifate mounting medium containing DAPI (VECTASHIELD, VECTOR laboratories). Coverslips were viewed on Zeiss LSM880 confocal microscope. A minimum of three images were captured from each sample and were quantified using ImageJ based on fluorescence intensity of γ-Η2Α.X staining per nucleus. Graphs show intensity relative to vehicle-only treated cells with a minimum of three independent biological replicates.

### ROS production

Cells were seeded on gelatin-coated glass bottom dishes and treated with doxorubicin for 3 hours two days after seeding. After treatment, cells were washed twice with PBS and incubated 15 min at 37 °C in plain RPMI with 5 μM H2DCFDA (Thermo Fisher Scientific). Cells were washed and analyzed on Zeiss LSM880 confocal microscope. A minimum of three images were captured from each sample and were quantified based on fluorescence intensity that was measured using the ZEN blue Analysis software of the microscope. Graphs show intensity relative to vehicle-only treated cells with a minimum of three independent biological replicates.

### Statistical analysis

All statistical analyses were conducted with GraphPad Prism 5.0, 7.03 and 8.3.0 versions. In graphs, values are expressed as mean ± standard error of mean (s.e.m.) and represent independent, biological repeats (at least three) unless otherwise stated. Statistical analyses were performed using Student’s t-test when comparing two groups. Cell viability curves were also described using F-test. Signal processing of the optically mapped data was performed using a custom IDL software program^23,24^.

## Supplementary information

## Acknowledgements

The authors would like to thank Dr. Francis Lynn and Yi Elizabeth Lin from the British Columbia Children’s Hospital Tissue and Disease Modeling Core for assistance with characterizing iPSCs. This study was funded by the Canadian Institute of Health Research (grant PJT153118 to LRB). LRB is supported by a Michael Smith Foundation for Health Research Scholar Award and is a Canada Research Chair in Precision Cardiovascular Disease Prevention. Images have been acquired from Servier Medical Art by Servier licensed under a Creative Commons Attribution 3.0 Unported License. Confocal microscopy images were generated at the Centre for Heart Lung Innovation Cellular Imaging and Biophysics Core. Funding support for the confocal microscope was provided by the Canadian Foundation for Innovation Grant #31080.

## Author contributions

E.C. designed and performed experiments, analyzed data and wrote the manuscript. H.H. designed and performed CRISPR/Cas9 experiments in iPSCs. S.S. designed and performed optical mapping experiments and provided iPSC training. A.M. designed and performed ESC-KO experiments. E.L. designed optical map apparatus and software. K.H. performed Flow Cytometry experiments. Z.L. provided expert assistant in electrophysiology. M.K.D provided intellectual input and patient recruitment was conducted in M.K.D. clinic. G.F.T. provided intellectual input, helped develop the approaches. L.R.B. conceived of and designed and supervised the study and wrote the manuscript. All authors contributed to the review and approval of the manuscript.

## Competing interests

The authors declare no competing interests.

## Footnotes

## Supplementary information

is available for this paper at 10.1038/s41598-020-65979-x.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Volkova M, Russell R. Anthracycline Cardiotoxicity: Prevalence, Pathogenesis and Treatment. Curr. Cardiol. Rev. 2012;7:214–220. doi: 10.2174/157340311799960645.  [DOI](https://doi.org/10.2174/157340311799960645) | [PMC free article](/articles/PMC3322439/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22758622/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Cardiol.%20Rev.&title=Anthracycline%20Cardiotoxicity:%20Prevalence,%20Pathogenesis%20and%20Treatment&author=M%20Volkova&author=R%20Russell&volume=7&publication_year=2012&pages=214-220&pmid=22758622&doi=10.2174/157340311799960645&)

2. Swain SM, Whaley FS, Ewer MS. Congestive heart failure in patients treated with doxorubicin. Cancer. 2003;97:2869–2879. doi: 10.1002/cncr.11407.  [DOI](https://doi.org/10.1002/cncr.11407) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12767102/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Congestive%20heart%20failure%20in%20patients%20treated%20with%20doxorubicin&author=SM%20Swain&author=FS%20Whaley&author=MS%20Ewer&volume=97&publication_year=2003&pages=2869-2879&pmid=12767102&doi=10.1002/cncr.11407&)

3. Singal PK, Iliskovic N, Kumar D. Adriamycin cardiomyopathy: pathophysiology and prevention. FASEB J. 1997;11:931–936. doi: 10.1096/fasebj.11.12.9337145.  [DOI](https://doi.org/10.1096/fasebj.11.12.9337145) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9337145/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FASEB%20J.&title=Adriamycin%20cardiomyopathy:%20pathophysiology%20and%20prevention&author=PK%20Singal&author=N%20Iliskovic&author=D%20Kumar&volume=11&publication_year=1997&pages=931-936&pmid=9337145&doi=10.1096/fasebj.11.12.9337145&)

4. Linschoten M, Teske AJ, Cramer MJ, van der Wall E, Asselbergs FW. Chemotherapy-Related Cardiac Dysfunction. Circ. Genomic Precis. Med. 2018;11:e001753. doi: 10.1161/CIRCGEN.117.001753.  [DOI](https://doi.org/10.1161/CIRCGEN.117.001753) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29557343/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Genomic%20Precis.%20Med.&title=Chemotherapy-Related%20Cardiac%20Dysfunction&author=M%20Linschoten&author=AJ%20Teske&author=MJ%20Cramer&author=E%20van%20der%20Wall&author=FW%20Asselbergs&volume=11&publication_year=2018&pages=e001753&pmid=29557343&doi=10.1161/CIRCGEN.117.001753&)

5. Aminkeng F, et al. A coding variant in RARG confers susceptibility to anthracycline-induced cardiotoxicity in childhood cancer. Nat. Genet. 2015;47:1079–1084. doi: 10.1038/ng.3374.  [DOI](https://doi.org/10.1038/ng.3374) | [PMC free article](/articles/PMC4552570/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26237429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=A%20coding%20variant%20in%20RARG%20confers%20susceptibility%20to%20anthracycline-induced%20cardiotoxicity%20in%20childhood%20cancer&author=F%20Aminkeng&volume=47&publication_year=2015&pages=1079-1084&pmid=26237429&doi=10.1038/ng.3374&)

6. Schneider BP, et al. Genome-Wide Association Study for Anthracycline-Induced Congestive Heart Failure. Clin. Cancer Res. 2017;23:43–51. doi: 10.1158/1078-0432.CCR-16-0908.  [DOI](https://doi.org/10.1158/1078-0432.CCR-16-0908) | [PMC free article](/articles/PMC5215621/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27993963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Genome-Wide%20Association%20Study%20for%20Anthracycline-Induced%20Congestive%20Heart%20Failure&author=BP%20Schneider&volume=23&publication_year=2017&pages=43-51&pmid=27993963&doi=10.1158/1078-0432.CCR-16-0908&)

7. Burridge PW, et al. Human induced pluripotent stem cell–derived cardiomyocytes recapitulate the predilection of breast cancer patients to doxorubicin-induced cardiotoxicity. Nat. Med. 2016;22:547–556. doi: 10.1038/nm.4087.  [DOI](https://doi.org/10.1038/nm.4087) | [PMC free article](/articles/PMC5086256/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27089514/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&title=Human%20induced%20pluripotent%20stem%20cell%E2%80%93derived%20cardiomyocytes%20recapitulate%20the%20predilection%20of%20breast%20cancer%20patients%20to%20doxorubicin-induced%20cardiotoxicity&author=PW%20Burridge&volume=22&publication_year=2016&pages=547-556&pmid=27089514&doi=10.1038/nm.4087&)

8. Maillet A, et al. Modeling Doxorubicin-Induced Cardiotoxicity in Human Pluripotent Stem Cell Derived-Cardiomyocytes. Sci. Rep. 2016;6:25333. doi: 10.1038/srep25333.  [DOI](https://doi.org/10.1038/srep25333) | [PMC free article](/articles/PMC4855185/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27142468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Modeling%20Doxorubicin-Induced%20Cardiotoxicity%20in%20Human%20Pluripotent%20Stem%20Cell%20Derived-Cardiomyocytes&author=A%20Maillet&volume=6&publication_year=2016&pages=25333&pmid=27142468&doi=10.1038/srep25333&)

9. Liang P, et al. Drug screening using a library of human induced pluripotent stem cell-derived cardiomyocytes reveals disease-specific patterns of cardiotoxicity. Circulation. 2013;127:1677–91. doi: 10.1161/CIRCULATIONAHA.113.001883.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.113.001883) | [PMC free article](/articles/PMC3870148/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23519760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Drug%20screening%20using%20a%20library%20of%20human%20induced%20pluripotent%20stem%20cell-derived%20cardiomyocytes%20reveals%20disease-specific%20patterns%20of%20cardiotoxicity&author=P%20Liang&volume=127&publication_year=2013&pages=1677-91&pmid=23519760&doi=10.1161/CIRCULATIONAHA.113.001883&)

10. Yang X, Pabon L, Murry CE. Engineering Adolescence: Maturation of Human Pluripotent Stem Cell-Derived Cardiomyocytes. Circ. Res. 2014;114:511–523. doi: 10.1161/CIRCRESAHA.114.300558.  [DOI](https://doi.org/10.1161/CIRCRESAHA.114.300558) | [PMC free article](/articles/PMC3955370/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24481842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Res.&title=Engineering%20Adolescence:%20Maturation%20of%20Human%20Pluripotent%20Stem%20Cell-Derived%20Cardiomyocytes&author=X%20Yang&author=L%20Pabon&author=CE%20Murry&volume=114&publication_year=2014&pages=511-523&pmid=24481842&doi=10.1161/CIRCRESAHA.114.300558&)

11. Liang G, Zhang Y. Genetic and epigenetic variations in iPSCs: potential causes and implications for application. Cell Stem Cell. 2013;13:149–59. doi: 10.1016/j.stem.2013.07.001.  [DOI](https://doi.org/10.1016/j.stem.2013.07.001) | [PMC free article](/articles/PMC3760008/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23910082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Stem%20Cell&title=Genetic%20and%20epigenetic%20variations%20in%20iPSCs:%20potential%20causes%20and%20implications%20for%20application&author=G%20Liang&author=Y%20Zhang&volume=13&publication_year=2013&pages=149-59&pmid=23910082&doi=10.1016/j.stem.2013.07.001&)

12. Taapken SM, et al. Karyotypic abnormalities in human induced pluripotent stem cells and embryonic stem cells. Nat. Biotechnol. 2011;29:313–314. doi: 10.1038/nbt.1835.  [DOI](https://doi.org/10.1038/nbt.1835) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21478842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&title=Karyotypic%20abnormalities%20in%20human%20induced%20pluripotent%20stem%20cells%20and%20embryonic%20stem%20cells&author=SM%20Taapken&volume=29&publication_year=2011&pages=313-314&pmid=21478842&doi=10.1038/nbt.1835&)

13. Creutzig U, Diekamp S, Zimmermann M, Reinhardt D. Longitudinal evaluation of early and late anthracycline cardiotoxicity in children with AML. Pediatr. Blood Cancer. 2007;48:651–662. doi: 10.1002/pbc.21105.  [DOI](https://doi.org/10.1002/pbc.21105) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17183582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Blood%20Cancer&title=Longitudinal%20evaluation%20of%20early%20and%20late%20anthracycline%20cardiotoxicity%20in%20children%20with%20AML&author=U%20Creutzig&author=S%20Diekamp&author=M%20Zimmermann&author=D%20Reinhardt&volume=48&publication_year=2007&pages=651-662&pmid=17183582&doi=10.1002/pbc.21105&)

14. Bu’Lock FA, Mott MG, Oakhill A, Martin RP. Left Ventricular Diastolic Filling Patterns Associated with Progressive Anthracycline-Induced Myocardial Damage: A Prospective Study. Pediatr. Cardiol. 1999;20:252–263. doi: 10.1007/s002469900459.  [DOI](https://doi.org/10.1007/s002469900459) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10368449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Cardiol.&title=Left%20Ventricular%20Diastolic%20Filling%20Patterns%20Associated%20with%20Progressive%20Anthracycline-Induced%20Myocardial%20Damage:%20A%20Prospective%20Study&author=FA%20Bu%E2%80%99Lock&author=MG%20Mott&author=A%20Oakhill&author=RP%20Martin&volume=20&publication_year=1999&pages=252-263&pmid=10368449&doi=10.1007/s002469900459&)

15. Mulrooney DA, et al. Cardiac outcomes in a cohort of adult survivors of childhood and adolescent cancer: retrospective analysis of the Childhood Cancer Survivor Study cohort. BMJ. 2009;339:b4606. doi: 10.1136/bmj.b4606.  [DOI](https://doi.org/10.1136/bmj.b4606) | [PMC free article](/articles/PMC3266843/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19996459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Cardiac%20outcomes%20in%20a%20cohort%20of%20adult%20survivors%20of%20childhood%20and%20adolescent%20cancer:%20retrospective%20analysis%20of%20the%20Childhood%20Cancer%20Survivor%20Study%20cohort&author=DA%20Mulrooney&volume=339&publication_year=2009&pages=b4606&pmid=19996459&doi=10.1136/bmj.b4606&)

16. van der Pal HJ, et al. High Risk of Symptomatic Cardiac Events in Childhood Cancer Survivors. J. Clin. Oncol. 2012;30:1429–1437. doi: 10.1200/JCO.2010.33.4730.  [DOI](https://doi.org/10.1200/JCO.2010.33.4730) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22473161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=High%20Risk%20of%20Symptomatic%20Cardiac%20Events%20in%20Childhood%20Cancer%20Survivors&author=HJ%20van%20der%20Pal&volume=30&publication_year=2012&pages=1429-1437&pmid=22473161&doi=10.1200/JCO.2010.33.4730&)

17. Feijen EAM, et al. Derivation of Anthracycline and Anthraquinone Equivalence Ratios to Doxorubicin for Late-Onset Cardiotoxicity. JAMA Oncol. 2019;5:864. doi: 10.1001/jamaoncol.2018.6634.  [DOI](https://doi.org/10.1001/jamaoncol.2018.6634) | [PMC free article](/articles/PMC6490232/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30703192/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Oncol.&title=Derivation%20of%20Anthracycline%20and%20Anthraquinone%20Equivalence%20Ratios%20to%20Doxorubicin%20for%20Late-Onset%20Cardiotoxicity&author=EAM%20Feijen&volume=5&publication_year=2019&pages=864&pmid=30703192&doi=10.1001/jamaoncol.2018.6634&)

18. Steinberg JS, Cohen AJ, Wasserman AG, Cohen P, Ross AM. Acute arrhythmogenicity of doxorubicin administration. Cancer. 1987;60:1213–8. doi: 10.1002/1097-0142(19870915)60:6<1213::aid-cncr2820600609>3.0.co;2-v.  [DOI](https://doi.org/10.1002/1097-0142(19870915)60:6<1213::aid-cncr2820600609>3.0.co;2-v) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3621107/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Acute%20arrhythmogenicity%20of%20doxorubicin%20administration&author=JS%20Steinberg&author=AJ%20Cohen&author=AG%20Wasserman&author=P%20Cohen&author=AM%20Ross&volume=60&publication_year=1987&pages=1213-8&pmid=3621107&doi=10.1002/1097-0142(19870915)60:6<1213::aid-cncr2820600609>3.0.co;2-v&)

19. Octavia Y, et al. Doxorubicin-induced cardiomyopathy: From molecular mechanisms to therapeutic strategies. J. Mol. Cell. Cardiol. 2012;52:1213–1225. doi: 10.1016/j.yjmcc.2012.03.006.  [DOI](https://doi.org/10.1016/j.yjmcc.2012.03.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22465037/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Cell.%20Cardiol.&title=Doxorubicin-induced%20cardiomyopathy:%20From%20molecular%20mechanisms%20to%20therapeutic%20strategies&author=Y%20Octavia&volume=52&publication_year=2012&pages=1213-1225&pmid=22465037&doi=10.1016/j.yjmcc.2012.03.006&)

20. Fernandez-Chas M, Curtis MJ, Niederer SA. Mechanism of doxorubicin cardiotoxicity evaluated by integrating multiple molecular effects into a biophysical model. Br. J. Pharmacol. 2018;175:763–781. doi: 10.1111/bph.14104.  [DOI](https://doi.org/10.1111/bph.14104) | [PMC free article](/articles/PMC5811623/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29161764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Pharmacol.&title=Mechanism%20of%20doxorubicin%20cardiotoxicity%20evaluated%20by%20integrating%20multiple%20molecular%20effects%20into%20a%20biophysical%20model&author=M%20Fernandez-Chas&author=MJ%20Curtis&author=SA%20Niederer&volume=175&publication_year=2018&pages=763-781&pmid=29161764&doi=10.1111/bph.14104&)

21. Wang GX, Wang YX, Zhou XB, Korth M. Effects of doxorubicinol on excitation–contraction coupling in guinea pig ventricular myocytes. Eur. J. Pharmacol. 2001;423:99–107. doi: 10.1016/s0014-2999(01)01096-2.  [DOI](https://doi.org/10.1016/s0014-2999(01)01096-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11448472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pharmacol.&title=Effects%20of%20doxorubicinol%20on%20excitation%E2%80%93contraction%20coupling%20in%20guinea%20pig%20ventricular%20myocytes&author=GX%20Wang&author=YX%20Wang&author=XB%20Zhou&author=M%20Korth&volume=423&publication_year=2001&pages=99-107&pmid=11448472&doi=10.1016/s0014-2999(01)01096-2&)

22. Mitrius JC, Vogel SM. Doxorubicin-induced automaticity in cultured chick heart cell aggregates. Cancer Res. 1990;50:4209–15.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2364378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=Doxorubicin-induced%20automaticity%20in%20cultured%20chick%20heart%20cell%20aggregates&author=JC%20Mitrius&author=SM%20Vogel&volume=50&publication_year=1990&pages=4209-15&pmid=2364378&)

23. Lin E, et al. Construction and use of a zebrafish heart voltage and calcium optical mapping system, with integrated electrocardiogram and programmable electrical stimulation. Am. J. Physiol. Integr. Comp. Physiol. 2015;308:R755–R768. doi: 10.1152/ajpregu.00001.2015.  [DOI](https://doi.org/10.1152/ajpregu.00001.2015) | [PMC free article](/articles/PMC4421745/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25740339/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Physiol.%20Integr.%20Comp.%20Physiol.&title=Construction%20and%20use%20of%20a%20zebrafish%20heart%20voltage%20and%20calcium%20optical%20mapping%20system,%20with%20integrated%20electrocardiogram%20and%20programmable%20electrical%20stimulation&author=E%20Lin&volume=308&publication_year=2015&pages=R755-R768&pmid=25740339&doi=10.1152/ajpregu.00001.2015&)

24. Shafaattalab S, et al. In vitro analyses of suspected arrhythmogenic thin filament variants as a cause of sudden cardiac death in infants. Proc. Natl. Acad. Sci. USA. 2019;116:6969–6974. doi: 10.1073/pnas.1819023116.  [DOI](https://doi.org/10.1073/pnas.1819023116) | [PMC free article](/articles/PMC6452669/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30886088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=In%20vitro%20analyses%20of%20suspected%20arrhythmogenic%20thin%20filament%20variants%20as%20a%20cause%20of%20sudden%20cardiac%20death%20in%20infants&author=S%20Shafaattalab&volume=116&publication_year=2019&pages=6969-6974&pmid=30886088&doi=10.1073/pnas.1819023116&)

25. Zhang S, et al. Identification of the molecular basis of doxorubicin-induced cardiotoxicity. Nat. Med. 2012;18:1639–1642. doi: 10.1038/nm.2919.  [DOI](https://doi.org/10.1038/nm.2919) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23104132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&title=Identification%20of%20the%20molecular%20basis%20of%20doxorubicin-induced%20cardiotoxicity&author=S%20Zhang&volume=18&publication_year=2012&pages=1639-1642&pmid=23104132&doi=10.1038/nm.2919&)

26. Delacroix L, et al. Cell-specific interaction of retinoic acid receptors with target genes in mouse embryonic fibroblasts and embryonic stem cells. Mol. Cell. Biol. 2010;30:231–44. doi: 10.1128/MCB.00756-09.  [DOI](https://doi.org/10.1128/MCB.00756-09) | [PMC free article](/articles/PMC2798310/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19884340/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Biol.&title=Cell-specific%20interaction%20of%20retinoic%20acid%20receptors%20with%20target%20genes%20in%20mouse%20embryonic%20fibroblasts%20and%20embryonic%20stem%20cells&author=L%20Delacroix&volume=30&publication_year=2010&pages=231-44&pmid=19884340&doi=10.1128/MCB.00756-09&)

27. Kadigamuwa C, et al. Role of Retinoic Acid Receptor-γ in DNA Damage-Induced Necroptosis. iScience. 2019;17:74–86. doi: 10.1016/j.isci.2019.06.019.  [DOI](https://doi.org/10.1016/j.isci.2019.06.019) | [PMC free article](/articles/PMC6606929/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31255985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=iScience&title=Role%20of%20Retinoic%20Acid%20Receptor-%CE%B3%20in%20DNA%20Damage-Induced%20Necroptosis&author=C%20Kadigamuwa&volume=17&publication_year=2019&pages=74-86&pmid=31255985&doi=10.1016/j.isci.2019.06.019&)

28. Renaud J-P, et al. Crystal structure of the RAR-γ ligand-binding domain bound to all-trans retinoic acid. Nature. 1995;378:681–689. doi: 10.1038/378681a0.  [DOI](https://doi.org/10.1038/378681a0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7501014/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Crystal%20structure%20of%20the%20RAR-%CE%B3%20ligand-binding%20domain%20bound%20to%20all-trans%20retinoic%20acid&author=J-P%20Renaud&volume=378&publication_year=1995&pages=681-689&pmid=7501014&doi=10.1038/378681a0&)

29. di Masi A, et al. Retinoic acid receptors: From molecular mechanisms to cancer therapy. Mol. Aspects Med. 2015;41:1–115. doi: 10.1016/j.mam.2014.12.003.  [DOI](https://doi.org/10.1016/j.mam.2014.12.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25543955/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Aspects%20Med.&title=Retinoic%20acid%20receptors:%20From%20molecular%20mechanisms%20to%20cancer%20therapy&author=A%20di%20Masi&volume=41&publication_year=2015&pages=1-115&pmid=25543955&doi=10.1016/j.mam.2014.12.003&)

30. Farboud B, Privalsky ML. Retinoic Acid Receptor-α Is Stabilized in a Repressive State by Its C-Terminal, Isotype-Specific F Domain. Mol. Endocrinol. 2004;18:2839–2853. doi: 10.1210/me.2004-0236.  [DOI](https://doi.org/10.1210/me.2004-0236) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15331758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Endocrinol.&title=Retinoic%20Acid%20Receptor-%CE%B1%20Is%20Stabilized%20in%20a%20Repressive%20State%20by%20Its%20C-Terminal,%20Isotype-Specific%20F%20Domain&author=B%20Farboud&author=ML%20Privalsky&volume=18&publication_year=2004&pages=2839-2853&pmid=15331758&doi=10.1210/me.2004-0236&)

31. Farboud B, Hauksdottir H, Wu Y, Privalsky ML. Isotype-restricted corepressor recruitment: a constitutively closed helix 12 conformation in retinoic acid receptors beta and gamma interferes with corepressor recruitment and prevents transcriptional repression. Mol. Cell. Biol. 2003;23:2844–58. doi: 10.1128/MCB.23.8.2844-2858.2003.  [DOI](https://doi.org/10.1128/MCB.23.8.2844-2858.2003) | [PMC free article](/articles/PMC152560/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12665583/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Biol.&title=Isotype-restricted%20corepressor%20recruitment:%20a%20constitutively%20closed%20helix%2012%20conformation%20in%20retinoic%20acid%20receptors%20beta%20and%20gamma%20interferes%20with%20corepressor%20recruitment%20and%20prevents%20transcriptional%20repression&author=B%20Farboud&author=H%20Hauksdottir&author=Y%20Wu&author=ML%20Privalsky&volume=23&publication_year=2003&pages=2844-58&pmid=12665583&doi=10.1128/MCB.23.8.2844-2858.2003&)

32. D’Antonio-Chronowska A, et al. Association of Human iPSC Gene Signatures and X Chromosome Dosage with Two Distinct Cardiac Differentiation Trajectories. Stem Cell Reports. 2019;13:924–938. doi: 10.1016/j.stemcr.2019.09.011.  [DOI](https://doi.org/10.1016/j.stemcr.2019.09.011) | [PMC free article](/articles/PMC6895695/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31668852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stem%20Cell%20Reports&title=Association%20of%20Human%20iPSC%20Gene%20Signatures%20and%20X%20Chromosome%20Dosage%20with%20Two%20Distinct%20Cardiac%20Differentiation%20Trajectories&author=A%20D%E2%80%99Antonio-Chronowska&volume=13&publication_year=2019&pages=924-938&pmid=31668852&doi=10.1016/j.stemcr.2019.09.011&)

33. Friedman, C. E. et al. Single-Cell Transcriptomic Analysis of Cardiac Differentiation from Human PSCs Reveals HOPX-Dependent Cardiomyocyte Maturation. Cell Stem Cell, 10.1016/j.stem.2018.09.009 (2018).  [DOI](https://doi.org/10.1016/j.stem.2018.09.009) | [PMC free article](/articles/PMC6220122/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30290179/)

34. Churko JM, et al. Defining human cardiac transcription factor hierarchies using integrated single-cell heterogeneity analysis. Nat. Commun. 2018;9:4906. doi: 10.1038/s41467-018-07333-4.  [DOI](https://doi.org/10.1038/s41467-018-07333-4) | [PMC free article](/articles/PMC6249224/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30464173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Defining%20human%20cardiac%20transcription%20factor%20hierarchies%20using%20integrated%20single-cell%20heterogeneity%20analysis&author=JM%20Churko&volume=9&publication_year=2018&pages=4906&pmid=30464173&doi=10.1038/s41467-018-07333-4&)

35. Cardinale D, et al. Early Detection of Anthracycline Cardiotoxicity and Improvement With Heart Failure Therapy. Circulation. 2015;131:1981–1988. doi: 10.1161/CIRCULATIONAHA.114.013777.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.114.013777) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25948538/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Early%20Detection%20of%20Anthracycline%20Cardiotoxicity%20and%20Improvement%20With%20Heart%20Failure%20Therapy&author=D%20Cardinale&volume=131&publication_year=2015&pages=1981-1988&pmid=25948538&doi=10.1161/CIRCULATIONAHA.114.013777&)

36. Nousiainen T, Jantunen E, Vanninen E, Hartikainen J. Early decline in left ventricular ejection fraction predicts doxorubicin cardiotoxicity in lymphoma patients. Br. J. Cancer. 2002;86:1697–700. doi: 10.1038/sj.bjc.6600346.  [DOI](https://doi.org/10.1038/sj.bjc.6600346) | [PMC free article](/articles/PMC2375393/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12087452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Cancer&title=Early%20decline%20in%20left%20ventricular%20ejection%20fraction%20predicts%20doxorubicin%20cardiotoxicity%20in%20lymphoma%20patients&author=T%20Nousiainen&author=E%20Jantunen&author=E%20Vanninen&author=J%20Hartikainen&volume=86&publication_year=2002&pages=1697-700&pmid=12087452&doi=10.1038/sj.bjc.6600346&)

37. Seki T, Yuasa S, Fukuda K. Generation of induced pluripotent stem cells from a small amount of human peripheral blood using a combination of activated T cells and Sendai virus. Nat. Protoc. 2012;7:718–28. doi: 10.1038/nprot.2012.015.  [DOI](https://doi.org/10.1038/nprot.2012.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22422317/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&title=Generation%20of%20induced%20pluripotent%20stem%20cells%20from%20a%20small%20amount%20of%20human%20peripheral%20blood%20using%20a%20combination%20of%20activated%20T%20cells%20and%20Sendai%20virus&author=T%20Seki&author=S%20Yuasa&author=K%20Fukuda&volume=7&publication_year=2012&pages=718-28&pmid=22422317&doi=10.1038/nprot.2012.015&)

38. Ran FA, et al. Genome engineering using the CRISPR-Cas9 system. Nat. Protoc. 2013;8:2281–2308. doi: 10.1038/nprot.2013.143.  [DOI](https://doi.org/10.1038/nprot.2013.143) | [PMC free article](/articles/PMC3969860/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24157548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&title=Genome%20engineering%20using%20the%20CRISPR-Cas9%20system&author=FA%20Ran&volume=8&publication_year=2013&pages=2281-2308&pmid=24157548&doi=10.1038/nprot.2013.143&)

39. Richardson CD, Ray GJ, DeWitt MA, Curie GL, Corn JE. Enhancing homology-directed genome editing by catalytically active and inactive CRISPR-Cas9 using asymmetric donor DNA. Nat. Biotechnol. 2016;34:339–344. doi: 10.1038/nbt.3481.  [DOI](https://doi.org/10.1038/nbt.3481) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26789497/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&title=Enhancing%20homology-directed%20genome%20editing%20by%20catalytically%20active%20and%20inactive%20CRISPR-Cas9%20using%20asymmetric%20donor%20DNA&author=CD%20Richardson&author=GJ%20Ray&author=MA%20DeWitt&author=GL%20Curie&author=JE%20Corn&volume=34&publication_year=2016&pages=339-344&pmid=26789497&doi=10.1038/nbt.3481&)

40. Byrne SM, Church GM. CRISPR‐Mediated Gene Targeting of Human Induced Pluripotent. Stem Cells. Curr. Protoc. Stem Cell Biol. 2015;35:5A.8.1–5A.8.22. doi: 10.1002/9780470151808.sc05a08s35.  [DOI](https://doi.org/10.1002/9780470151808.sc05a08s35) | [PMC free article](/articles/PMC4772967/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26949444/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stem%20Cells.%20Curr.%20Protoc.%20Stem%20Cell%20Biol.&title=CRISPR%E2%80%90Mediated%20Gene%20Targeting%20of%20Human%20Induced%20Pluripotent&author=SM%20Byrne&author=GM%20Church&volume=35&publication_year=2015&pages=5A.8.1-5A.8.22&pmid=26949444&doi=10.1002/9780470151808.sc05a08s35&)

41. Santos DP, Kiskinis E, Eggan K, Merkle FT. Comprehensive Protocols for CRISPR/Cas9-based Gene Editing in Human Pluripotent Stem Cells. Curr. Protoc. Stem Cell Biol. 2016;38:5B.6.1–5B.6.60. doi: 10.1002/cpsc.15.  [DOI](https://doi.org/10.1002/cpsc.15) | [PMC free article](/articles/PMC4988528/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27532820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.%20Stem%20Cell%20Biol.&title=Comprehensive%20Protocols%20for%20CRISPR/Cas9-based%20Gene%20Editing%20in%20Human%20Pluripotent%20Stem%20Cells&author=DP%20Santos&author=E%20Kiskinis&author=K%20Eggan&author=FT%20Merkle&volume=38&publication_year=2016&pages=5B.6.1-5B.6.60&pmid=27532820&doi=10.1002/cpsc.15&)

42. Jacobi AM, et al. Simplified CRISPR tools for efficient genome editing and streamlined protocols for their delivery into mammalian cells and mouse zygotes. Methods. 2017;121–122:16–28. doi: 10.1016/j.ymeth.2017.03.021.  [DOI](https://doi.org/10.1016/j.ymeth.2017.03.021) | [PMC free article](/articles/PMC5761324/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28351759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods&title=Simplified%20CRISPR%20tools%20for%20efficient%20genome%20editing%20and%20streamlined%20protocols%20for%20their%20delivery%20into%20mammalian%20cells%20and%20mouse%20zygotes&author=AM%20Jacobi&volume=121%E2%80%93122&publication_year=2017&pages=16-28&pmid=28351759&doi=10.1016/j.ymeth.2017.03.021&)

43. Sanson KR, et al. Optimized libraries for CRISPR-Cas9 genetic screens with multiple modalities. Nat. Commun. 2018;9:5416. doi: 10.1038/s41467-018-07901-8.  [DOI](https://doi.org/10.1038/s41467-018-07901-8) | [PMC free article](/articles/PMC6303322/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30575746/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Optimized%20libraries%20for%20CRISPR-Cas9%20genetic%20screens%20with%20multiple%20modalities&author=KR%20Sanson&volume=9&publication_year=2018&pages=5416&pmid=30575746&doi=10.1038/s41467-018-07901-8&)

44. Doench JG, et al. Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9. Nat. Biotechnol. 2016;34:184–191. doi: 10.1038/nbt.3437.  [DOI](https://doi.org/10.1038/nbt.3437) | [PMC free article](/articles/PMC4744125/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26780180/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&title=Optimized%20sgRNA%20design%20to%20maximize%20activity%20and%20minimize%20off-target%20effects%20of%20CRISPR-Cas9&author=JG%20Doench&volume=34&publication_year=2016&pages=184-191&pmid=26780180&doi=10.1038/nbt.3437&)

45. Kime C, et al. Efficient CRISPR/Cas9-Based Genome Engineering in Human Pluripotent Stem Cells. Curr. Protoc. Hum. Genet. 2016;88:21.4.1–21.4.23. doi: 10.1002/0471142905.hg2104s88.  [DOI](https://doi.org/10.1002/0471142905.hg2104s88) | [PMC free article](/articles/PMC4726454/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26724721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.%20Hum.%20Genet.&title=Efficient%20CRISPR/Cas9-Based%20Genome%20Engineering%20in%20Human%20Pluripotent%20Stem%20Cells&author=C%20Kime&volume=88&publication_year=2016&pages=21.4.1-21.4.23&pmid=26724721&doi=10.1002/0471142905.hg2104s88&)

46. Maillet A, Tan KP, Brunham LR. Use of Human Pluripotent Stem Cell Derived-Cardiomyocytes to Study Drug-Induced Cardiotoxicity. Curr. Protoc. Toxicol. 2017;73:22.5.1–22.5.22. doi: 10.1002/cptx.30.  [DOI](https://doi.org/10.1002/cptx.30) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28777443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.%20Toxicol.&title=Use%20of%20Human%20Pluripotent%20Stem%20Cell%20Derived-Cardiomyocytes%20to%20Study%20Drug-Induced%20Cardiotoxicity&author=A%20Maillet&author=KP%20Tan&author=LR%20Brunham&volume=73&publication_year=2017&pages=22.5.1-22.5.22&pmid=28777443&doi=10.1002/cptx.30&)

47. Lian X, et al. Directed cardiomyocyte differentiation from human pluripotent stem cells by modulating Wnt/β-catenin signaling under fully defined conditions. Nat. Protoc. 2012;8:162–175. doi: 10.1038/nprot.2012.150.  [DOI](https://doi.org/10.1038/nprot.2012.150) | [PMC free article](/articles/PMC3612968/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23257984/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&title=Directed%20cardiomyocyte%20differentiation%20from%20human%20pluripotent%20stem%20cells%20by%20modulating%20Wnt/%CE%B2-catenin%20signaling%20under%20fully%20defined%20conditions&author=X%20Lian&volume=8&publication_year=2012&pages=162-175&pmid=23257984&doi=10.1038/nprot.2012.150&)
