# Effect of the Most Relevant CYP3A4 and CYP3A5 Polymorphisms on the Pharmacokinetic Parameters of 10 CYP3A Substrates

## Metadata
**Authors:** Miriam Saiz-Rodríguez, Susana Almenara, Marcos Navares-Gómez, Dolores Ochoa, Manuel Román, Pablo Zubiaur, Dora Koller, María Santos, Gina Mejía, Alberto M Borobia, Cristina Rodríguez-Antona, Francisco Abad-Santos
**Journal:** Biomedicines
**Date:** 2020 Apr 22
**DOI:** [10.3390/biomedicines8040094](https://doi.org/10.3390/biomedicines8040094)
**PMID:** 32331352
**PMCID:** PMC7235792
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7235792/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7235792/pdf/biomedicines-08-00094.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7235792/pdf/biomedicines-08-00094.pdf)

## Abstract

Several cytochrome P450 (CYP) CYP3A polymorphisms were associated with reduced enzyme function. We aimed to evaluate the influence of these alleles on the pharmacokinetic parameters (PK) of several CYP3A substrates. We included 251 healthy volunteers who received a single dose of ambrisentan, atorvastatin, imatinib, aripiprazole, fentanyl, amlodipine, donepezil, olanzapine, fesoterodine, or quetiapine. The volunteers were genotyped for CYP3A4 and CYP3A5 polymorphisms by qPCR. To compare the PK across studies, measurements were corrected by the mean of each parameter for every drug and were logarithmically transformed. Neither CYP3A phenotype nor individual CYP3A4 or CYP3A5 polymorphisms were significantly associated with differences in PK. However, regarding the substrates that are exclusively metabolized by CYP3A, we observed a higher normalized AUC (p = 0.099) and a tendency of lower normalized Cl (p = 0.069) in CYP3A4 mutated allele carriers what was associated with diminished drug metabolism capacity. CYP3A4 polymorphisms did not show a pronounced influence on PK of the analysed drugs. If so, their impact could be detectable in a very small percentage of subjects. Although there are few subjects carrying CYP3A4 double mutations, the effect in those might be relevant, especially due to the majority of subjects lacking the CYP3A5 enzyme. In heterozygous subjects, the consequence might be less noticeable due to the high inducible potential of the CYP3A4 enzyme.

Keywords: CYP3A4, CYP3A5, pharmacokinetics

## 1. Introduction

Precision medicine uses the tools provided by molecular biology to improve the diagnosis and expand the treatment options of patients based on their biological and molecular profile. Pharmacogenetics, the study of genetic markers that influence the inter-individual variation in drug response, is one of the cornerstones of personalized medicine [[1](#B1-biomedicines-08-00094),[2](#B2-biomedicines-08-00094)]. The role of heritable genetic variation in drug response has been studied since the 1950s [[3](#B3-biomedicines-08-00094)]. In 1959, Vogel coined the term pharmacogenetics to define a new field that aimed to study the influence of inherited factors on drug response variability through genetic and pharmacological knowledge and methods [[4](#B4-biomedicines-08-00094)].

Several genes are related to drug response. The U.S. Food and Drug Administration (FDA) describes four gene categories in drug development that can affect the benefit-risk profile of a certain drug: (1) genes relevant to the pharmacokinetics of the drug, (2) genes related to the pharmacological effect of the drug, (3) genes not directly related to the pharmacological effect of the drug, but may predispose to toxicities, and (4) genes that influence the susceptibility or progression of the disease [[5](#B5-biomedicines-08-00094)]. To date, the most important genes that are related to the pharmacokinetics (PK) of drugs encode cytochrome P450 superfamily (CYP) enzymes.

At least 57 active CYP genes are present in the human genome, along with approximately the same number of pseudogenes. CYP enzymes are divided into 18 families and 44 subfamilies. CYP 1–3 families are particularly active in the detoxification of exogenous chemicals, such as drugs [[6](#B6-biomedicines-08-00094)]. Every CYP protein sequence is designated as CYP (superfamily root symbol), followed by a number (gene family), then a capital letter (subfamily), and another number (protein). Family members should share >40%, while subfamily members should share >55% amino acid sequence identity [[7](#B7-biomedicines-08-00094)].

CYP enzymes are responsible for about 75–80% of phase I drug metabolism [[8](#B8-biomedicines-08-00094)]. Genetic polymorphisms in CYP enzymes which are associated with a characteristic effect in drug metabolism are translated into different metabolizer phenotypes: ultra-rapid metabolizers (UMs), normal metabolizers (NMs), intermediate metabolizers (IMs) and poor metabolizers (PMs). However, these categories give only a static idea about the functionality of an enzyme in an individual. In addition to genetics, variability in CYP activity may be explained by other factors, such as age, sex, morbidity, co-medication, food, or smoking [[9](#B9-biomedicines-08-00094)].

### 1.1. CYP3A Subfamily

The human CYP3A subfamily consists of 4 isoforms, CYP3A4, CYP3A5, CYP3A7, and CYP3A43 [[10](#B10-biomedicines-08-00094)]. These isoforms are present in variable proportion in the human liver, being about 30% of the total CYP enzyme content [[11](#B11-biomedicines-08-00094)]. Genes encoding for *CYP3A* enzymes are located on the 7q21–q22.1 chromosome band [[12](#B12-biomedicines-08-00094)] along with *CYP3AP1* and *CYP3AP2* pseudogenes, in a cluster of about 200kb [[13](#B13-biomedicines-08-00094)]. All four CYP3A enzymes comprise 13 exons [[14](#B14-biomedicines-08-00094)] and show high structural similarity (between 71.5% and 84.1%) [[15](#B15-biomedicines-08-00094)].

The 5′ untranslated region (UTR) of each human CYP3A gene consists of an average of 101 nucleotides, which is below the mean human 5’-UTR length of 150 nucleotides. On the contrary, the size of the 3′-UTR is highly variable, being 111 nucleotides for CYP3A5, 463 for CYP3A7, 549 for CYP3A43 and 1152 for CYP3A4 [[16](#B16-biomedicines-08-00094)]. The role of UTRs lies in the regulation of gene expression. Therefore, as CYP3A4 has the longest 3′UTR, it may be greatly regulated [[17](#B17-biomedicines-08-00094)].

CYP3A43 is generally expressed at low levels in prostate, liver, kidney, and pancreas [[10](#B10-biomedicines-08-00094)]. It is mainly involved in the metabolism of endogenous compounds, but not drugs [[18](#B18-biomedicines-08-00094)].

CYP3A7 represents approximately 30–50% of the total CYP450 enzyme amount expressed in fetal liver, where CYP3A4 is not expressed [[19](#B19-biomedicines-08-00094)]. CYP3A7 was detected also in placenta and endometrium. CYP3A7 is rarely present in adult livers at significant levels as it is gradually substituted by CYP3A4 after birth [[18](#B18-biomedicines-08-00094)].

Enzyme activity and functional protein interactions are key to understanding the cellular machinery. In order to fully understand every biological phenomena, it is of importance to study the connectivity of the whole network [[20](#B20-biomedicines-08-00094)]. CYP3A enzymes are involved in steroid hormone biosynthesis and in lipid and fatty acid metabolism pathways, while interacting with the main molecules described in [Figure 1](#biomedicines-08-00094-f001).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd5f/7235792/618f9cf1cb57/biomedicines-08-00094-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7235792_biomedicines-08-00094-g001.jpg)

Top CYP3A4 (A) and CYP3A5 (B) protein interactants. Data obtained from: STRING Interaction Network. Szklarczyk et al. Nucleic acids research 47.D1 (2018): D607-D613.2.

### 1.2. CYP3A4 and CYP3A5

Among the human CYP3A enzymes, CYP3A4 and CYP3A5 are considered the most important in drug metabolism [[21](#B21-biomedicines-08-00094),[22](#B22-biomedicines-08-00094)]. Both are highly abundant in the adult liver and intestine [[11](#B11-biomedicines-08-00094),[23](#B23-biomedicines-08-00094)].

In 1984, Kleinbloesem et al. reported a bimodal frequency distribution of the area under the concentration-time curve (AUC) of nifedipine and suggested that a polymorphism was related to its disposition kinetics [[24](#B24-biomedicines-08-00094)]. Guengerich in 1985 identified and purified the enzyme responsible for the oxidation of nifedipine from rat and human livers, which he named P450 Nifedipine Oxidase. This enzyme is known as CYP3A4 today [[25](#B25-biomedicines-08-00094)]. Since then, several studies aimed to characterize this enzyme more precisely.

*CYP3A4* gene is located on the 7q22.1 band of the human karyotype [[26](#B26-biomedicines-08-00094)] with a length of nucleotides [[27](#B27-biomedicines-08-00094)]. Previously another CYP3A gene, *CYP3A3*, was thought to exist; however, it seems that it is a transcript variant of *CYP3A4*. To date, 8 splice variants encoding different isoforms have been identified. Six of these contain an Open Reading Frame (ORF) and code proteins with the size range between 153 and 534 amino acids [[28](#B28-biomedicines-08-00094)].

CYP3A4 is expressed predominantly in the liver, where it is the most expressed CYP enzyme [[29](#B29-biomedicines-08-00094)], ranging from 14.5% to 37% of the hepatic P450 pool [[6](#B6-biomedicines-08-00094)]. Overall, CYP3A5 is expressed at lower levels than CYP3A4 (~10% of CYP3A4) [[30](#B30-biomedicines-08-00094)]. Moreover, CYP3A4 is the most abundant CYP enzyme in the human intestinal epithelial cells [[31](#B31-biomedicines-08-00094)]. Intestinal CYP3A4 plays a role in the bioavailability of some drugs, such as midazolam and cyclosporine via first-pass metabolism during liver transplantation [[32](#B32-biomedicines-08-00094),[33](#B33-biomedicines-08-00094)].

One of the *CYP3A4* most commonly studied reduced-function single nucleotide polymorphism (SNP) is *20, which gives rise to a truncated protein with complete loss of function of the enzymatic activity [[34](#B34-biomedicines-08-00094)]. Similarly, *CYP3A4**22 results in a reduced mRNA expression and a low CYP3A4 enzyme activity [[17](#B17-biomedicines-08-00094),[35](#B35-biomedicines-08-00094),[36](#B36-biomedicines-08-00094)].

A previous study characterised CYP3A4 as a highly polymorphic enzyme as *CYP3A4**20 allele was present in 1.2% of the Spanish population and up to 3.8% in specific regions [[37](#B37-biomedicines-08-00094)]. This study described the *CYP3A4**20 no function variant with such a high frequency for the first time [[37](#B37-biomedicines-08-00094)].

Regarding the *CYP3A4**22 variant, it is of especial interest to find relevant associations with several drugs, such as tacrolimus [[38](#B38-biomedicines-08-00094),[39](#B39-biomedicines-08-00094),[40](#B40-biomedicines-08-00094),[41](#B41-biomedicines-08-00094),[42](#B42-biomedicines-08-00094),[43](#B43-biomedicines-08-00094)]. Briefly, it was linked to lower clearance, and therefore lower dose requirements. Moreover, *CYP3A4**22 was also associated with a greater reduction in total and LDL cholesterol levels in an observational study with simvastatin users from a Caucasian cohort [[44](#B44-biomedicines-08-00094)].

Another relevant variant which was predominantly studied in Caucasians is *CYP3A4**1B (rs2740574G), a single base change in the 5’-UTR of *CYP3A4*. At first, this variant was studied related to prostate tumors [[45](#B45-biomedicines-08-00094),[46](#B46-biomedicines-08-00094)]. It was associated with lower enzyme activity and reduced binding of nuclear proteins in vitro [[47](#B47-biomedicines-08-00094)]. However, other studies contradicted these results [[46](#B46-biomedicines-08-00094),[48](#B48-biomedicines-08-00094)]. In PK studies, *CYP3A4**1B was associated to decreased dose-adjusted trough concentrations (C0/D) and higher dose requirements of tacrolimus [[38](#B38-biomedicines-08-00094),[41](#B41-biomedicines-08-00094),[42](#B42-biomedicines-08-00094),[49](#B49-biomedicines-08-00094),[50](#B50-biomedicines-08-00094),[51](#B51-biomedicines-08-00094)], and cyclosporine [[52](#B52-biomedicines-08-00094),[53](#B53-biomedicines-08-00094),[54](#B54-biomedicines-08-00094)] in transplanted patients. It was also associated with a lower risk of dose decrease or switching therapy during simvastatin treatment [[55](#B55-biomedicines-08-00094)]. Usually when studied together with *CYP3A5**3 or *CYP3A4**22, it loses relevance against them [[39](#B39-biomedicines-08-00094),[41](#B41-biomedicines-08-00094),[50](#B50-biomedicines-08-00094)]. On the other hand, a significant linkage disequilibrium between this variant and *CYP3A5**1 was described, which could partly explain the underlying cause of the observed associations with *CYP3A4**1B [[56](#B56-biomedicines-08-00094)].

*CYP3A4**1G variant is located in intron 10 of the *CYP3A4* gene [[57](#B57-biomedicines-08-00094)] and was predominantly studied in Asian populations [[58](#B58-biomedicines-08-00094)]. In some studies, *CYP3A4**1G carriers showed an increased clearance and a decreased C0/D of tacrolimus [[59](#B59-biomedicines-08-00094),[60](#B60-biomedicines-08-00094),[61](#B61-biomedicines-08-00094),[62](#B62-biomedicines-08-00094),[63](#B63-biomedicines-08-00094)] and cyclosporine treatment [[64](#B64-biomedicines-08-00094),[65](#B65-biomedicines-08-00094)]. *CYP3A4**1G was associated with less dose requirements of fentanyl by patient-controlled analgesia (PCA) in the treatment of postsurgical pain [[66](#B66-biomedicines-08-00094),[67](#B67-biomedicines-08-00094),[68](#B68-biomedicines-08-00094),[69](#B69-biomedicines-08-00094),[70](#B70-biomedicines-08-00094)]. However, further evidence is needed from independent studies to confirm these associations.

*CYP3A4**2, *CYP3A4**12, and *CYP3A4**17 alleles have a very low frequency and their functional impact is not established clearly [[15](#B15-biomedicines-08-00094),[57](#B57-biomedicines-08-00094),[71](#B71-biomedicines-08-00094)].

Finally, the *CYP3A4**3 variant (rs4986910) is a missense polymorphism [[71](#B71-biomedicines-08-00094)] with very little clinical evidence. In a study with 2735 individuals on statin therapy, *CYP3A4**3 was associated with an increase in HDL-cholesterol levels with fluvastatin treatment [[72](#B72-biomedicines-08-00094)].

To date, eight splice variants of *CYP3A5* have been described. Three of these encode proteins with the size range between 1720 and 4473 amino acids [[73](#B73-biomedicines-08-00094)]. The presence of the SNP 6986A>G (rs776746) in intron 3 defines *CYP3A5**3 and results in a non-functional CYP3A5 protein in homozygous carriers (*CYP3A5**3/*3) [[74](#B74-biomedicines-08-00094)]. This defective variant is the most frequent among Caucasians and Asians. As a result, CYP3A5 is expressed in approximately 10–25% of individuals, depending on their ethnicity ([Table 1](#biomedicines-08-00094-t001)). When expressed, CYP3A5 can represent about 50% of the total CYP3A hepatic content, which is equal to CYP3A4 activity [[75](#B75-biomedicines-08-00094)]. On the other hand, CYP3A4 and CYP3A5 usually share substrate specificity. Therefore, a combined phenotype including both enzymes were proposed for several drugs, such as midazolam [[76](#B76-biomedicines-08-00094)], everolimus [[77](#B77-biomedicines-08-00094)], statins [[78](#B78-biomedicines-08-00094)], or tamoxifen [[79](#B79-biomedicines-08-00094)].

### Table 1.

| CYP3A Allele | Reference SNP Identifier | In Vitro Effect | PK Effect in CYP3A Metabolism | Minor Allele Frequencies |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Europeans | Latin-Americans | Africans | East Asians |  |  |  |  |
| CYP3A4*22 | rs35599367 | ↓ [35,36] | ↓ | 50/1006 (4.97%) | 18/694 (2.59%) | 1/1322 (0.08%) | 0/1008 (0%) |
| CYP3A4*1B | rs2740574 | (= [46], ↓ [47]?, ↑ [48]?) | (↑?) [38], [41,42,49,50,51,52,53,54] | 28/1006 (2.78%) | 73/694 (10.52%) | 1012/1322 (76.55%) | 4/1008 (0.4%) |
| CYP3A4*1G | rs2242480 |   | (↑?) [59], [60,61,62,63,64,65] | 82/1006 (8.15%) | 273/694 (39.34%) | 1124/1322 (85.02%) | 270/1008 (26.79%) |
| CYP3A4*3 | rs4986910 |   | (↓?) [72] | 7/1006 (0.70%) | 5/694 (0.72%) | 1/1322 (0.08%) | 0/1008 (0%) |
| CYP3A4*20 | rs67666821 | None | ↓ | 26/64600 (0.04%)~ | 22/13290 (0.17%)~ | 3/42021 (0.01%)~ | 0/3134 (0.00%)~ |
| CYP3A5*3 | rs776746 | ↓ | ↓ | 949/1006 (94.33%) | 553/694 (79.68%) | 238/1322 (18.00%) | 719/1008 (71.33%) |
| CYP3A5*6 | rs10264272 | (↓?) [74] | (↓?) [39], [40] | 3/1006 (0.30%) | 16/694 (2.31%) | 204/1322 (15.43%) | 0/1008 (0.00%) |
| CYP3A5*7 | rs41303343 | =/↓ | (↓?) [39], [40] | 0/942 (0.00%)^ | 27/1090 (2.48%)^ | 174/2014 (8.64%)^ | 0/480 (0.00%)^ |
| CYP3A5*8 | rs55817950 | ↓ |   | 0/ 113648 (0.00%) | 0/ 34590 (0.00%) | 0/ 16216 (0.00%) | 0/ 18391 (0.00%) |
| CYP3A5*9 | rs28383479 | ↓ |   | 0/113434(0.00%) | 0/34340(0.00%) | 0/16226(0.00%) | 2/18358 (0.01%) |

Table 1 Caption: Allele frequencies of the most studied variants of CYP3A4 and CYP3A5.

Other variants that result in a non-functional CYP3A5 protein are *CYP3A5**6 (rs10264272), which causes an alternative splicing of *CYP3A5* [[74](#B74-biomedicines-08-00094)] and *CYP3A5**7 (rs41303343), an insertion that results in a frameshift [[80](#B80-biomedicines-08-00094)]. Both are greatly frequent in Latin American and African populations ([Table 1](#biomedicines-08-00094-t001)). In a study with 140 Latin-American adult kidney transplant recipients under tacrolimus treatment, *CYP3A5* defective haplotypes (*3, *6, *7) showed an increased C0/D [[39](#B39-biomedicines-08-00094),[40](#B40-biomedicines-08-00094)].

Moreover, *CYP3A5**8 (rs55817950) and *CYP3A5**9 (rs28383479) are defective coding alleles that showed decreased CYP3A5 activity in in vitro models [[81](#B81-biomedicines-08-00094)]. However, to date, these SNPs have been rarely studied in vivo.

[Figure 2](#biomedicines-08-00094-f002) depicts the polymorphic context of *CYP3A4* and *CYP3A5*, with the main polymorphisms described above.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd5f/7235792/feccf9cb5bbe/biomedicines-08-00094-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7235792_biomedicines-08-00094-g002.jpg)

Polymorphic context of CYP3A4 and CYP3A5.

Tacrolimus is the only drug to which CYP3A-genotype guided recommendations are applied. In 2015, the Clinical Pharmacogenetics Implementation Consortium (CPIC) published the guideline for *CYP3A5* genotype and tacrolimus dosing [[82](#B82-biomedicines-08-00094)]. This association is well established; nevertheless, the variable frequency of *CYP3A5**1 allele among different populations ([Table 1](#biomedicines-08-00094-t001)) makes the utility of the genetic test variable.

In most of the studies that analyzed *CYP3A4**22, *CYP3A5**3 was also analyzed and in the majority of them the latter showed less associations [[83](#B83-biomedicines-08-00094),[84](#B84-biomedicines-08-00094)]. This could be due to the different contribution of each gene to the metabolism of the different substrates, as it was demonstrated in vitro [[85](#B85-biomedicines-08-00094)], or can alternatively be caused by a bias related to the low frequency of CYP3A5 expressers in some populations ([Table 1](#biomedicines-08-00094-t001)).

CYP3A4 protein liver content and activity show an important inter-individual variation ranging from 10 to >100 fold [[11](#B11-biomedicines-08-00094),[46](#B46-biomedicines-08-00094),[86](#B86-biomedicines-08-00094)]. In addition, CYP3A4 content was higher in female liver specimens compared to males [[29](#B29-biomedicines-08-00094),[87](#B87-biomedicines-08-00094)]. Furthermore, a higher clearance of various CYP3A substrates was observed in females treated with different drugs, such as nifedipine [[88](#B88-biomedicines-08-00094)], oxycodone [[89](#B89-biomedicines-08-00094)], or cyclosporine [[90](#B90-biomedicines-08-00094)]. CYP3A4 activity may also be affected by the phases of the menstrual cycle [[91](#B91-biomedicines-08-00094)].

Drugs, herbals or alimentary products can also increase CYP3A4 metabolic activity by increasing enzyme expression. The mechanism of induction involves various nuclear receptors: pregnane X receptor (PXR), constitutive androstane receptor (CAR), glucocorticoid receptor (GR), vitamin D receptor (VDR), and hepatocyte nuclear factor—4 alpha (HNF4α). These nuclear receptors bind to DNA segments present in CYP3A promoter regions, such as PXR responsive element (prPXRE), xenobiotic-responsive enhancer module (XREM), or constitutive liver enhancer module (CLEM4), while increasing transcription and expression of CYP3A4 [[6](#B6-biomedicines-08-00094),[92](#B92-biomedicines-08-00094)]. Some known CYP3A4 inducers are rifampicin, carbamazepine, phenytoin, phenobarbital, paclitaxel, cyclophosphamide, glucocorticoids, and herbs such as St. John’s Wort [[92](#B92-biomedicines-08-00094)]. CYP3A4 is also susceptible to reversible and irreversible inhibition. Some of the most used drugs that inhibit CYP3A4 are macrolide antibiotics, protease inhibitors, anti-HIV agents, azole antifungals, fluoxetine, verapamil, diltiazem, and several herbal and dietary components [[92](#B92-biomedicines-08-00094)]. Irreversible or mechanism-based inhibition is more relevant in in vivo drug interactions, specifically, nicotinamide adenine dinucleotide phosphate hydrogen (NADPH)-, time- and concentration-dependent [[93](#B93-biomedicines-08-00094)].

Evidence of the effect of some CYP3A polymorphisms on the PK of some relevant drugs is scarce and often inconclusive; therefore, its inclusion in prescription guidelines is limited. Our aim was to verify and quantify the effect of the main *CYP3A4* and *CYP3A5* polymorphisms and a merged phenotype on the pharmacokinetic parameters of a large number of CYP3A substrates.

## 2. Materials and Methods

### 2.1. Study Population and Study Design

Our study population comprised 251 healthy volunteers who participated in one of 11 bioequivalence clinical trials (10 of them performed at the Clinical Trials Unit of Hospital Universitario de La Princesa, Madrid, Spain, and 1 at the Clinical Trials Unit of Universidad Autónoma de Madrid, Madrid, Spain) with CYP3A-substrates and agreed to participate in the pharmacogenetic study. The volunteers received a single oral dose of ambrisentan 10 mg (1 trial, *n* = 23), atorvastatin 80 mg (2 trials, *n* = 50), imatinib 400 mg (1 trial, *n* = 12), aripiprazole 10 mg (1 trial, *n* = 26), fentanyl 0.3 mg (1 trial, *n* = 35), amlodipine 10 mg (1 trial, *n* = 25), donepezil 10 mg (1 trial, *n* = 23), olanzapine 5 mg (1 trial, *n* = 25), fesoterodine 8 mg (1 trial, *n* = 13) or quetiapine 25 mg (1 trial, *n* = 19). Each study was a bioequivalence clinical trial in which a test formulation was compared to a reference formulation, after a single oral dose administration under fasting conditions.

All protocols met the terms of the current Spanish Legislation on human clinical research and were approved by the Research Ethics Committee, duly authorized by the Spanish Drugs Agency and under the guidelines of Good Clinical Practice. All participants signed a written informed consent for both the clinical trial and the pharmacogenetic study. Subjects were free to withdraw from the study at any time.

The inclusion criteria for each study were as follows: non-smoking male and female volunteers, age 18–55 years, body mass index within the 18.5–30.0 range, free from any organic or psychiatric conditions, with normal vital signs, electrocardiogram, medical records and physical examination.

Healthy volunteers were asked prior to study inclusion if they had taken any medications or supplements in the past month. Those volunteers who had taken any drugs or supplements were excluded from the clinical trial. Concomitant pharmacological treatments were not administered during none of the phases of the study. At the time of admission, the night prior to the administration of the study drug, volunteers were again asked if they had taken any concomitant medication during the time they had not been admitted to the Clinical Trials Unit.

### 2.2. Pharmacokinetic Analysis

Since the determination of every drug plasma concentration was outsourced, all samples were stored at −20 ± 5 °C or −75 ± 5 °C until their shipment to the external analytical laboratory. Each quantification was accomplished by a high-performance liquid chromatography coupled to tandem mass spectrometry (HPLC-MS/MS) validated method, according to the European Medicines Agency guidelines. Pharmacokinetic parameters were calculated by means of a non-compartmental analysis with WinNonlin Professional software, Version 2.0 (Pharsight Corporation, Palo Alto, California), as described previously [[94](#B94-biomedicines-08-00094)]. The AUC and the maximum concentration (C_max_) were corrected by dose/weight. Clearance (Cl) and volume of distribution (Vd) were divided by weight.

### 2.3. Genotyping

DNA was obtained from 1 mL of peripheral blood samples using MagNA Pure LC DNA Isolation Kit in an automatic DNA extractor (MagNa Pure^®^ System, Roche Applied Science, Indianapolis, Indiana). *CYP3A4**2 (rs55785340), *CYP3A4**3 (rs4986910), *CYP3A4**12 (rs12721629), *CYP3A4**17 (rs4987161) and *CYP3A4**22 (rs35599367) and *CYP3A5**2 (rs28365083), *CYP3A5**3(rs776746), *CYP3A5**6 (rs10264272), *CYP3A5**7 (rs41303343), *CYP3A5**8 (rs55817950), and *CYP3A5**9 (rs28383479) polymorphisms were genotyped by quantitative PCR (qPCR), in a QuantStudio 12k Flex instrument (Applied Biosystems, Foster City, CA, USA). We analysed these SNPs as they were the ones included in the commercial and predesigned TaqMan™ OpenArray™ PGx Express Panel (Applied Biosystems, Foster City, CA, USA).

The *CYP3A4**20 (rs67666821) polymorphism call was assessed by KASPar SNP Genotyping System (LGC Genomics, Herts, UK) in an ABI PRISM 7900HT Sequence Detection System (Applied Biosystems, Darmstadt, Germany). All *CYP3A4**20 carriers were confirmed by Sanger sequencing in an ABI PRISM 3700 DNA Analyser capillary sequencer (Applied Biosystems, Foster City, California, USA) [[37](#B37-biomedicines-08-00094)].

The *1 allele, considered as wild-type, was assigned when the subject lacked all of the analyzed alleles.

### 2.4. Statistical Analysis

The influence of *CYP3A4* and *CYP3A5* genotypes were analysed separately. However, to assess the influence of both enzyme conjunctively, *CYP3A4* and *CYP3A5* genotypes were merged into a “CYP3A phenotype” as follows: subjects with at least one *CYP3A4* reduced activity allele (i.e., *CYP3A4* *1/*22, *1/*3, *22/*22 or *3/*22) and no CYP3A5 activity (*CYP3A5* *3/*3,*3/*6 or *3/*7) were considered PM; subjects with normal CYP3A4 activity (*CYP3A4* *1/*1) and no CYP3A5 activity (*CYP3A5* *3/*3, *3/*6 or *3/*7) were considered IM; finally, subjects with normal CYP3A4 activity (*CYP3A4**1/*1) and at least one *CYP3A5* functional allele (*CYP3A5* *1/*1 or *1/*3) were categorized as extensive metabolizers (EM), based on Sanchez Spitman et al. [[79](#B79-biomedicines-08-00094)]

Statistical analyses were accomplished with the SPSS 22.0 software (SPSS Inc., Chicago, Illinois, USA). *p* values < 0.05 were considered statistically significant. Chi-square test was used to define sex-related differences in the genotypic frequencies. T-test or ANOVA were used to evaluate differences in PK among sexes and genotypes. To compare the PK parameters across studies, they were divided by the mean of each parameter for every drug and logarithmically transformed, thus termed as normalized parameter. The power analysis to know how many patients would be necessary to be evaluated in order to reveal a difference in PK parameters was performed with the G.power 3.1.9.7 software.

## 3. Results and Discussion

From all the volunteers included, 164 (65.3%) were Caucasians, 81 (32.3%) were Latin-Americans, four (1.6%) were Blacks, and two (0.8%) were Arabs. Regarding sex, 102 (40.6%) were women, while 149 (59.4%) were men. Mean age was 27.8 years old (ranging between 18 and 55). Mean weight was 69.6 kg (ranging between 44.4 and 98.5 kg) and mean height was 1.70 m (ranging between 1.49 and 1.97 m). Men were younger (26.5 ± 6.5 years vs. 29.8 ± 9.3 years), weighted more (76.0 ± 9.1 kg vs. 60.2 ± 8.1 kg) and were taller (1.76 ± 0.06 m vs. 1.61± 0.06 m) than women, *p* < 0.001.

Mean age was 25.8 ± 6.0, 32.0 ± 9.8, 32.0 ± 9.1 and 23.0 ± 0.0 years old (*p* < 0.001) in Caucasians, Latin-Americans, Blacks, and Arabs, respectively. Mean weight was 69.0 ± 11.3, 70.1 ± 12.1, 85.7 ± 5.2, 63.6 ± 17.0 kg (*p* = 0.033) in Caucasians, Latin-Americans, Blacks, and Arabs, respectively. Finally, mean height was 1.72 ± 0.08, 1.67 ± 0.11, 1.74 ± 0.06 and 1.70 ± 0.13 m (*p* = 0.001) in Caucasians, Latin-Americans, Blacks, and Arabs, respectively.

[Table 2](#biomedicines-08-00094-t002) shows the main *CYP3A* genotypes found in our study. None of the genotyped subjects carried *CYP3A4**2, *CYP3A4**12, *CYP3A4**17, *CYP3A5**2, *CYP3A5**8, or *CYP3A5**9 alleles. Our allele frequencies (AF) did not differ from those reported in the literature: *CYP3A4**3 AF was 0.3% (Minor allele frequency, MAF: <1%) [[95](#B95-biomedicines-08-00094)], *CYP3A4**20 AF was 0.9% (MAF: <1%) [[96](#B96-biomedicines-08-00094)], and *CYP3A4**22 AF was 2.3% (Iberian MAF: 3.7%; Latin MAF: 2.6%) [[97](#B97-biomedicines-08-00094)]. Additionally, *CYP3A5**3 AF was 87.2% (Iberian MAF: 92.5%; Latin MAF: 79.7%) [[98](#B98-biomedicines-08-00094)], *CYP3A5**6 AF was 1.5% (Iberian MAF: 0.9%; Latin MAF: 2.3%) [[99](#B99-biomedicines-08-00094)] and *CYP3A5**7 AF was 0.2% (MAF: <1%) [[100](#B100-biomedicines-08-00094)]. No statistically significant difference was found in *CYP3A* genotype frequencies among sexes.

### Table 2.

| Gene | Genotype | N | % |
| --- | --- | --- | --- |
| CYP3A4 | *1/*1 | 233 | 92.8 |
| *1/*20 | 5 | 2.0 |  |
| *1/*22 | 11 | 4.4 |  |
| *1/*3 | 1 | 0.4 |  |
| *3/*22 | 1 | 0.4 |  |
| CYP3A5 | *1/*1 | 5 | 2.0 |
| *1/*3 | 45 | 17.9 |  |
| *3/*3 | 192 | 76.5 |  |
| *3/*6 | 8 | 3.2 |  |
| *3/*7 | 1 | 0.4 |  |
|   | Phenotype | N | % |
| CYP3A | PM | 17 | 6.8 |
| IM | 184 | 73.3 |  |
| EM | 50 | 19.9 |  |

Table 2 Caption: CYP3A genotype and phenotype frequencies in our study population.

Sex influenced the PK parameters since women showed higher normalized T_1/2_ (1.05 ± 0.34 vs. 0.96 ± 0.30, *p* = 0.03) and normalized Vd (1.06 ± 0.41 vs. 0.96 ± 0.32, *p* = 0.057), compared to men ([Table 3](#biomedicines-08-00094-t003)). No statistically significant difference was found between men and women in normalized AUC, C_max_, Cl and T_max_. A higher T_1/2_ in women is the opposite of what is expected according to the greater CYP3A4 content in women [[29](#B29-biomedicines-08-00094),[87](#B87-biomedicines-08-00094)]. The higher Vd could be due to the higher fat percentage in women. However, these differences are minor and possibly not sufficient to establish a dose adjustment based on sex.

### Table 3.

| Gene | Genotype/Phenotype | Pharmacokinetic Parameter |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Normalized AUC | Normalized Cmax | Normalized T1/2 | Normalized Tmax | Normalized Cl | Normalized Vd |  |  |
| CYP3A4 | *1/*1 (n = 233) | 0.99 (0.35) | 0.99 (0.42) | 0.99 (0.32) | 1.00 (0.56) | 0.99 (0.38) | 1.00 (0.37) |
| *1/*20 (n = 5) | 1.12 (0.24) | 0.96 (0.33) | 1.18 (0.24) | 0.66 (0.27) | 0.73 (0.18) | 0.90 (0.10) |  |
| *1/*22 (n = 11) | 1.06 (0.46) | 0.98 (0.38) | 0.99 (0.18) | 1.00 (0.71) | 0.96 (0.43) | 0.98 (0.43) |  |
| *1/*3 + *3/*22 (n = 2) | 1.24 (0.40) | 1.25 (0.10) | 1.07 (0.28) | 0.93 (0.03) | 0.75 (0.39) | 0.83 (0.18) |  |
| p-value | 0.523 | 0.738 | 0.566 | 0.482 | 0.338 | 0.874 |  |
| CYP3A4 | Wild-type (n = 233) | 0.99 (0.35) | 0.99 (0.42) | 0.99 (0.32) | 1.00 (0.56) | 0.99 (0.37) | 1.00 (0.37) |
| Mutated (n = 18) | 1.10 (0.39) | 1.00 (0.34) | 1.05 (0.20) | 0.90 (0.58) | 0.87 (0.37) | 0.94 (0.34) |  |
| p-value | 0.240 | 0.853 | 0.276 | 0.351 | 0.146 | 0.428 |  |
| CYP3A5 | *1/*1 (n = 5) | 0.90 (0.30) | 1.13 (0.48) | 0.88 (0.19) | 0.73 (0.26) | 1.20 (0.51) | 1.05 (0.27) |
| *1/*3 (n = 45) | 1.02 (0.32) | 1.02 (0.44) | 1.04 (0.36) | 0.90 (0.52) | 0.94 (0.38) | 0.96 (0.23) |  |
| *3/*3 + *3/*6 + *3/*7 (n = 201) | 0.99 (0.37) | 0.99 (0.40) | 0.99 (0.31) | 1.03 (0.57) | 0.98 (0.38) | 1.00 (0.39) |  |
| p-value | 0.710 | 0.626 | 0.731 | 0.257 | 0.486 | 0.833 |  |
| CYP3A | EM (n = 50) | 1.01 (0.31) | 1.03 (0.44) | 1.02 (0.35) | 0.88 (0.50) | 0.97 (0.39) | 0.97 (0.23) |
| IM (n = 183) | 0.99 (0.37) | 0.99 (0.41) | 0.98 (0.31) | 1.03 (0.57) | 0.99 (0.37) | 1.01 (0.40) |  |
| PM (n = 18) | 1.09 (0.39) | 1.00 (0.34) | 1.05 (0.21) | 0.93 (0.59) | 0.87 (0.37) | 0.94 (0.34) |  |
| p-value | 0.408 | 0.692 | 0.501 | 0.155 | 0.324 | 0.720 |  |
| Sex | Men (n = 149) | 1.00 (0.37) | 1.00 (0.43) | 0.96 (0.30) | 0.99 (0.58) | 0.99 (0.38) | 0.96 (0.32) |
| Women (n = 102) | 0.99 (0.35) | 1.00 (0.39) | 1.05 (0.34) | 1.01 (0.54) | 0.97 (0.37) | 1.06 (0.41) |  |
| p-value | 0.765 | 0.832 | 0.030 | 0.426 | 0.889 | 0.057 |  |

Table 3 Caption: Pharmacokinetic parameters (normalized by the mean of each drug) according to CYP3A genotypes and phenotype and sex in all the included drugs.

Regarding the influence of CYP3A alleles or phenotype categories on normalized PK parameters, we performed a separate analysis for each compound ([Supplementary Table S1](#app1-biomedicines-08-00094)) that showed no significant association of CYP3A polymorphisms for ambrisentan, aripiprazole, donepezil, atorvastatin and olanzapine. However, some associations were found for amlodipine (higher AUC and lower Cl in CYP3A5 PM), imatinib, and quetiapine (higher AUC in CYP3A4 mutated allele carriers). Although not significant, some tendencies were observed in CYP3A4 mutated alleles carriers for fentanyl (higher AUC and lower Cl) and fesoterodine (lower AUC and higher Cl). However, the limited number of individuals carrying mutations in each compound separately makes it difficult to reach any conclusion. Therefore, we merged the compounds for a conjunct analysis.

Regarding the influence of CYP3A genotypes on the PK parameters of the analysed compounds, we observed that neither CYP3A phenotype nor individual *CYP3A4* or *CYP3A5* polymorphisms were significantly associated with PK variability ([Table 2](#biomedicines-08-00094-t002)). This fact could be explained by the influence of other metabolizing enzymes (i.e., CYP2D6 for aripiprazole [[101](#B101-biomedicines-08-00094)]), whose polymorphisms might have a higher impact on PK parameters of some drugs.

Although it was not statistically significant, when analysing the substrates that are only metabolized by CYP3A (ambrisentan, atorvastatin, imatinib, fentanyl, amlodipine and quetiapine), some tendencies could be observed ([Table 4](#biomedicines-08-00094-t004)). Subjects with *CYP3A4* mutated alleles (*3, *20, *22) showed higher normalized AUC (*p* = 0.099) and almost significantly lower normalized Cl (*p* = 0.069), which could be a result of a diminished metabolism capacity. Moreover, the only subject carrying double mutation (*CYP3A4**3/*22) had a 53% increase in AUC and a 53% lower Cl. However, the validity of this result need to be confirmed in other studies due to the limited sample size.

### Table 4.

| Gene | Genotype/Phenotype | Pharmacokinetic Parameter |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Normalized AUC | Normalized Cmax | Normalized T1/2 | Normalized Tmax | Normalized Cl | Normalized Vd |  |  |
| CYP3A4 | *1/*1 (n = 150) | 0.98 (0.41) | 1.00 (0.49) | 1.00 (0.31) | 1.00 (0.56) | 1.00 (0.40) | 1.00 (0.41) |
| *1/*20 (n = 4) | 1.08 (0.26) | 0.91 (0.36) | 1.13 (0.25) | 0.74 (0.24) | 0.78 (0.16) | 0.89 (0.12) |  |
| *1/*22 (n = 9) | 1.14 (0.46) | 1.02 (0.39) | 1.01 (0.19) | 1.01 (0.80) | 0.85 (0.39) | 0.91 (0.45) |  |
| *3/*22 (n = 1) | 1.53 | 1.32 | 1.27 | 0.96 | 0.47 | 0.71 |  |
| p-value | 0.317 | 0.796 | 0.626 | 0.822 | 0.194 | 0.673 |  |
| CYP3A4 | Wild-type (n = 150) | 0.98 (0.41) | 1.00 (0.49) | 0.99 (0.31) | 1.00 (0.56) | 1.00 (0.40) | 1.00 (0.41) |
| Mutated (n = 14) | 1.15 (0.40) | 1.01 (0.37) | 1.06 (0.20) | 0.93 (0.64) | 0.81 (0.33) | 0.89 (0.36) |  |
| p-value | 0.099 | 0.723 | 0.274 | 0.459 | 0.069 | 0.264 |  |
| CYP3A5 | *1/*1 (n = 2) | 0.96 (0.49) | 1.24 (0.72) | 0.83 (0.19) | 0.75 (0.48) | 1.25 (0.81) | 0.99 (0.47) |
| *1/*3 (n = 23) | 1.02 (0.42) | 1.01 (0.59) | 1.03 (0.40) | 0.94 (0.63) | 0.97 (0.48) | 0.92 (0.28) |  |
| *3/*3 + *3/*6 + *3/*7 (n = 139) | 1.00 (0.41) | 0.99 (0.45) | 1.00 (0.29) | 1.01 (0.56) | 0.98 (0.38) | 1.01 (0.42) |  |
| p-value | 0.987 | 0.760 | 0.834 | 0.538 | 0.731 | 0.818 |  |
| CYP3A | EM (n = 25) | 1.01 (0.42) | 1.03 (0.59) | 1.01 (0.39) | 0.92 (0.62) | 0.99 (0.49) | 0.93 (0.29) |
| IM (n = 126) | 0.98 (0.41) | 1.00 (0.47) | 0.99 (0.30) | 1.01 (0.55) | 0.99 (0.38) | 1.02 (0.43) |  |
| PM (n = 13) | 1.14 (0.41) | 0.94 (0.28) | 1.06 (0.21) | 0.97 (0.65) | 0.82 (0.34) | 0.91 (0.37) |  |
| p-value | 0.342 | 0.988 | 0.566 | 0.560 | 0.280 | 0.526 |  |
| Sex | Men (n = 95) | 0.99 (0.43) | 0.98 (0.50) | 0.96 (0.27) | 0.96 (0.57) | 1.00 (0.41) | 0.96 (0.34) |
| Women (n = 69) | 1.01 (0.39) | 1.03 (0.44) | 1.05 (0.34) | 1.04 (0.56) | 0.96 (0.38) | 1.05 (0.48) |  |
| p-value | 0.747 | 0.278 | 0.093 | 0.209 | 0.618 | 0.304 |  |

Table 4 Caption: Pharmacokinetic parameters (normalized by the mean of each drug) according to CYP3A genotypes and phenotype and sex in pure CYP3A-substrates (ambisentran, atorvastatine, imatinib, fentanyl, amlodipine and quetiapine).

In a previous study with the CYP3A-substrate fentanyl, our group was able to demonstrate that carrying *CYP3A4**22 polymorphism results in higher AUC and lower Cl [[102](#B102-biomedicines-08-00094)]. Here, congruently, the same tendency was observed, yet the significance level was not reached as we used AUCt instead of AUCinf [[102](#B102-biomedicines-08-00094)]. Moreover, the validity of our previous results need to be confirmed in other studies with similar settings due to the small sample size. Other *CYP3A4* genotypes were not related to different exposure to fentanyl [[103](#B103-biomedicines-08-00094)]. Regarding *CYP3A5*, the presence of *3 allele was linked to a two-fold increase in fentanyl systemic exposure [[104](#B104-biomedicines-08-00094)]. However, we could not find any homozygous wild-type subject to compare with *CYP3A5**3/*3 carriers.

*CYP3A4**22 and *CYP3A5**3 were not associated with lipid reductions in 105 children and adolescents with familial hypercholesterolemia treated with atorvastatin [[105](#B105-biomedicines-08-00094)]. Moreover, *CYP3A4**22 neither affected the lipid reduction response to atorvastatin and simvastatin treatment in 416 adults [[106](#B106-biomedicines-08-00094)]. However, Rosales et al. described a better response to atorvastatin in Chilean patients who carried *CYP3A4**1B [[107](#B107-biomedicines-08-00094)]. Likewise, Peng et al. found an influence of the polymorphism *CYP3A4**1G (rs2242480) on atorvastatin treatment in patients with ischemic stroke [[108](#B108-biomedicines-08-00094)]. However, these two polymorphisms were not analysed in our study since they were not included in the commercial TaqMan™ OpenArray™ PGx Express Panel (Applied Biosystems, Foster City, CA, USA).

Indeed, no association with *CYP3A4**1G was reported in tacrolimus concentrations in paediatric patients [[109](#B109-biomedicines-08-00094)] or imatinib pharmacokinetics in Chinese patients [[110](#B110-biomedicines-08-00094)]. Another *CYP3A4* polymorphism, rs755828176, was significantly associated with dose-adjusted trough plasma concentrations of imatinib and its metabolite [[111](#B111-biomedicines-08-00094)]. Nevertheless, this frameshift variant is reported not to be present in our population [[112](#B112-biomedicines-08-00094)]. Another study in 112 patients with chronic myeloid leukaemia found no relationship between *CYP3A4* rs2740574 and *CYP3A5**3 and imatinib plasma levels and therapeutic response [[113](#B113-biomedicines-08-00094)]. Neither did another study in 82 patients treated with imatinib [[114](#B114-biomedicines-08-00094)]. Indeed, Petain et al. stated that morphologic and biological characteristics have a greater effect on imatinib PK than pharmacogenetics [[115](#B115-biomedicines-08-00094)]. However, *CYP3A5**3 was associated with higher imatinib trough plasma concentrations in Nigerian population [[116](#B116-biomedicines-08-00094)] and a significant lower risk of acquiring imatinib resistance, while *CYP3A4**18 had no statistically significant effect [[117](#B117-biomedicines-08-00094)]. In our study, the only subject with the *CYP3A4**20 allele showed higher imatinib levels (*p* = 0.046), although it is not conclusive due to the limited sample size in our study; the frequency of *CYP3A4**20 previously found in Spanish population (1.2%) was not reached [[37](#B37-biomedicines-08-00094)]. Moreover, in contrast to the study by Adeagbo et al. [[116](#B116-biomedicines-08-00094)], CYP3A5 PMs showed lower AUC compared to the only IM subject, although, as before, it is not conclusive.

According to our knowledge, there is no study evaluating the effect of polymorphisms in CYP3A on the PK of ambrisentan. However, in our study, it does not appear that *CYP3A4* or *CYP3A5* polymorphisms influence its elimination.

Regarding amlodipine, Huang et al. described an altered antihypertensive efficacy of amlodipine related to *CYP3A4**1G and *CYP3A5**3 alleles in patients with hypertension following renal transplantation [[118](#B118-biomedicines-08-00094)]. Moreover, in a study in 40 healthy volunteers, *CYP3A5**3/*3 carriers exhibited lower plasma amlodipine concentrations [[119](#B119-biomedicines-08-00094)]. In our study, although not significant, we found the lowest amlodipine plasma concentration in the only *CYP3A5* *1/*1 subject. Indeed, we found a significantly higher Cl in this subject, but our sample size is not sufficient to arise any conclusion.

Regarding quetiapine, as shown in [Supplementary Table S1](#app1-biomedicines-08-00094), we found significant higher plasma levels in carriers of a *CYP3A4* mutated allele (AUC 1107.7 ± 376.1 ng·h·Kg/mL·mg vs. 588.9 ± 272.6 ng·h·Kg/mL·mg in wild-type subjects, *p* = 0.024). A previous work from our laboratory showed that quetiapine AUC was influenced by polymorphisms in *CYP1A2* [[120](#B120-biomedicines-08-00094)]. According to our knowledge, a direct relationship between *CYP3A4* polymorphisms and quetiapine was not described before. Notwithstanding, the association we found has to be interpreted with caution, given that only two subjects were carriers of mutations. Moreover, Shilbayeh et al. described in a population PK model a 29% increased Cl of quetiapine in *CYP3A5* *1/*1 subjects compared to*1/*3 and *3/*3, but they stated that these simulations do not call for a genotype-based quetiapine dosing scheme [[121](#B121-biomedicines-08-00094)]. Indeed, polymorphisms in *CYP3A5* had no significant influence on steady-state serum concentrations of quetiapine in psychiatric patients [[122](#B122-biomedicines-08-00094)]. In our study, we found no difference in any PK parameter of quetiapine according to *CYP3A5* genotypes.

Finally, in regards to the other substrates that are not only metabolized by CYP3A, Noetzli et al. evaluated the effect of polymorphisms in *CYP2D6* and CYP3A, finding that only CYP2D6 influenced donepezil clearance [[123](#B123-biomedicines-08-00094)]. Moreover, another study in 54 patients did not find any statistically significant association between *CYP3A4* or *CYP3A5* genotypes and plasma donepezil concentrations or clinical response [[124](#B124-biomedicines-08-00094)].

Moreover, in the current work, when analysing the combined effect of *CYP3A4* and *CYP3A5* into the CYP3A phenotype, the tendencies towards higher AUC and lower Cl disappeared. It is of great importance to increase our sample size and to analyse whether the influence of *CYP3A4* polymorphisms alone is sufficient to have a clear effect on CYP3A-substrates PK parameters, since the majority of subjects lack CYP3A5 activity. With the 251 subjects included in the study the statistical power achieved was 0.81 for the ANOVA analysis of CYP3A phenotype and for an expected effect size of 0.20 for all the normalized PK parameters. The effect size observed was 0.074 for normalized AUC, 0.045 for normalized C_max_, 0.090 for normalized T_1/2_, 0.074 for normalized T_max_, 0.081 for normalized Cl, and 0.063 for normalized Vd. The sample size required for a smaller effect size of 0.09 (if clinically relevant) in an ANOVA analysis of normalized PK variables is 1.194.

Moreover, further approaches might shed light on this study field, such as the design of human recombinant microsomes with different *CYP3A4* and *CYP3A5* polymorphisms, which might allow the assessment of these polymorphisms on in vitro PK, and its comparison with the in vivo PK data. For instance, the study of recombinant CYP3A human liver microsomes showed that CYP3A5 expression influenced the metabolism of the antiviral asunaprevir, but not daclatasvir and beclabuvir [[125](#B125-biomedicines-08-00094)].

Finally, it is important to note that CYP3A4 is an enzyme that is easily induced and inhibited by a wide variety of drugs, so the effect of polymorphisms, if any, could be masked in patients with multiple medications. Since this enzyme metabolizes many of the drugs actually on the market, it is likely that several CYP3A4 substrates are used concomitantly and that some of them are also CYP3A4 inhibitors or inducers of CYP3A4.

### Study Limitations

Our study was performed after a single-dose administration of each drug to healthy volunteers, that does not allow the assessment of long-term effectiveness and safety. Our results might differ from patients under chronic treatment. Nevertheless, this study design allows the investigation of genetic polymorphisms affecting PK while avoiding the influence of other confounding factors, such as smoking or concomitant treatment. However, further research is needed to increase the statistical power of these results, especially to evaluate the influence in homozygote subjects for inactivating alleles. Finally, there is a possibility that other polymorphisms that were not included in the study could be functionally impactful, but we did not screen for them.

## 4. Conclusions

Our main conclusion is that *CYP3A* polymorphisms themselves do not have a pronounced influence on the PK parameters of the CYP3A-substrates analysed in this study. If so, the influence of *CYP3A4* polymorphisms could be detectable in a very small percentage of subjects, so the evidence so far is scarce and does not support a dose adjustment in the majority of CYP3A-metabolized drugs. Indeed, it is of importance to highlight that, although there are few subjects carrying *CYP3A4* double mutations, the effect in those might be relevant. The fact that the majority of subjects lack the CYP3A5 enzyme, might make the effect of *CYP3A4* double mutations even more important when present. However, in *CYP3A4* heterozygous subjects the consequence might be less noticeable, partly due to the high inducible potential of CYP3A4 enzyme.

## Supplementary Materials

Supplementary materials can be found at [https://www.mdpi.com/2227-9059/8/4/94/s1](https://www.mdpi.com/2227-9059/8/4/94/s1).

## Author Contributions

Conceptualization, M.S.-R., S.A. and F.A.-S.; methodology, M.N.-G., D.O. and M.R.; software, M.S.-R.; validation, M.S.-R., S.A. and F.A.-S.; formal analysis, M.N.-G, M.S., C.R.-A.; investigation, , M.S.-R., S.A., M.N.-G., D.O., M.R., P.Z., D.K., G.M., M.S., C.R.-A., A.M.B. and F.A.-S.; resources, D.O. and F.A.-S.; data curation, M.S.-R.; writing—original draft preparation, M.S.-R. and S.A.; writing—review and editing, M.S.-R., S.A. and F.A.-S.; visualization, M.S.-R. and S.A.; supervision, F.A.-S.; project administration, F.A.-S.; funding acquisition, F.A.-S., D.O. and C.R.-A. All authors have read and agreed to the published version of the manuscript.

## Funding

F.A.S. and D.O. have been consultants or investigators in clinical trials sponsored by the following pharmaceutical companies: Abbott, Alter, Chemo, Cinfa, FAES, Farmalíder, Ferrer, GlaxoSmithKline, Galenicum, Gilead, Italfarmaco, Janssen-Cilag, Kern, Normon, Novartis, Servier, Silverpharma, Teva and Zambon. M.N.G is co-financed by the European Social Fund and the Youth European Initiative, grant number PEJ-2018-TL/BMD-11080. D.K. is co-financed by the H2020 Marie Sklodowska-Curie Innovative Training Network 721236 grant.

## Conflicts of Interest

The authors declare no conflict of interest.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. König I.R., Fuchs O., Hansen G., von Mutius E., Kopp M.V. What is precision medicine? Eur. Respir. J. 2017;50 doi: 10.1183/13993003.00391-2017.  [DOI](https://doi.org/10.1183/13993003.00391-2017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29051268/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Respir.%20J.&title=What%20is%20precision%20medicine?&author=I.R.%20K%C3%B6nig&author=O.%20Fuchs&author=G.%20Hansen&author=E.%20von%20Mutius&author=M.V.%20Kopp&volume=50&publication_year=2017&pmid=29051268&doi=10.1183/13993003.00391-2017&)

2. Weinshilboum R., Wang L. Pharmacogenomics: Bench to bedside. Nat. Rev. Drug Discov. 2004;3:739–748. doi: 10.1038/nrd1497.  [DOI](https://doi.org/10.1038/nrd1497) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15340384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Drug%20Discov.&title=Pharmacogenomics:%20Bench%20to%20bedside&author=R.%20Weinshilboum&author=L.%20Wang&volume=3&publication_year=2004&pages=739-748&pmid=15340384&doi=10.1038/nrd1497&)

3. Motulsky A.G. Drug reactions, enzymes, and biochemical genetics. J. Am. Med. Assoc. 1957;165:835–837. doi: 10.1001/jama.1957.72980250010016.  [DOI](https://doi.org/10.1001/jama.1957.72980250010016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/13462859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Med.%20Assoc.&title=Drug%20reactions,%20enzymes,%20and%20biochemical%20genetics&author=A.G.%20Motulsky&volume=165&publication_year=1957&pages=835-837&pmid=13462859&doi=10.1001/jama.1957.72980250010016&)

4. Moderne Probleme der Human Genetik Ergebn. [(accessed on 24 February 2020)];inn-Google Académico. Available online: https://scholar.google.es/scholar?hl=es&as_sdt=0%2C5&q=Moderne+Probleme+der+Human+genetik.+Ergebn.+inn&btnG=  [https://scholar.google.es/scholar?hl=es&as_sdt=0%2C5&q=Moderne+Probleme+der+Human+genetik.+Ergebn.+inn&btnG=](https://scholar.google.es/scholar?hl=es&as_sdt=0%2C5&q=Moderne+Probleme+der+Human+genetik.+Ergebn.+inn&btnG=)

5. US Food and Drug Administration . Clinical Pharmacogenomics. Premarket Evaluation in Early-Phase Clinical Studies and Recommendations for Labeling. Volume 26 Silver Spring; Rockville, MD, USA: 2013.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Clinical%20Pharmacogenomics.%20Premarket%20Evaluation%20in%20Early-Phase%20Clinical%20Studies%20and%20Recommendations%20for%20Labeling&publication_year=2013&)

6. Zanger U.M., Schwab M. Cytochrome P450 enzymes in drug metabolism: Regulation of gene expression, enzyme activities, and impact of genetic variation. Pharm. Ther. 2013;138:103–141. doi: 10.1016/j.pharmthera.2012.12.007.  [DOI](https://doi.org/10.1016/j.pharmthera.2012.12.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23333322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Ther.&title=Cytochrome%20P450%20enzymes%20in%20drug%20metabolism:%20Regulation%20of%20gene%20expression,%20enzyme%20activities,%20and%20impact%20of%20genetic%20variation&author=U.M.%20Zanger&author=M.%20Schwab&volume=138&publication_year=2013&pages=103-141&pmid=23333322&doi=10.1016/j.pharmthera.2012.12.007&)

7. Feyereisen R., Nelson D.R., Coon M.J., Estabrook R.W., Nebert D.W., Fujii-kuriyama Y., Gonzalez F.J., Guengerich F.P., Gunsalus I.C., Johnson E.F. The P450 superfamily: Update on new sequences, gene mapping, and recommended nomenclature. DNA Cell Biol. 1991;10:1–14. doi: 10.1089/dna.1991.10.1.  [DOI](https://doi.org/10.1089/dna.1991.10.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1991046/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=DNA%20Cell%20Biol.&title=The%20P450%20superfamily:%20Update%20on%20new%20sequences,%20gene%20mapping,%20and%20recommended%20nomenclature&author=R.%20Feyereisen&author=D.R.%20Nelson&author=M.J.%20Coon&author=R.W.%20Estabrook&author=D.W.%20Nebert&volume=10&publication_year=1991&pages=1-14&pmid=1991046&doi=10.1089/dna.1991.10.1&)

8. Ingelman-Sundberg M., Rodriguez-Antona C. Pharmacogenetics of drug-metabolizing enzymes: Implications for a safer and more effective drug therapy. Philos. Trans. R. Soc. Lond. B Biol. Sci. 2005;360:1563–1570. doi: 10.1098/rstb.2005.1685.  [DOI](https://doi.org/10.1098/rstb.2005.1685) | [PMC free article](/articles/PMC1569528/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16096104/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B%20Biol.%20Sci.&title=Pharmacogenetics%20of%20drug-metabolizing%20enzymes:%20Implications%20for%20a%20safer%20and%20more%20effective%20drug%20therapy&author=M.%20Ingelman-Sundberg&author=C.%20Rodriguez-Antona&volume=360&publication_year=2005&pages=1563-1570&pmid=16096104&doi=10.1098/rstb.2005.1685&)

9. Rostami-Hodjegan A., Amin A.M., Spencer E.P., Lennard M.S., Tucker G.T., Flanagan R.J. Influence of dose, cigarette smoking, age, sex, and metabolic activity on plasma clozapine concentrations: A predictive model and nomograms to aid clozapine dose adjustment and to assess compliance in individual patients. J. Clin. Psychopharmacol. 2004;24:70–78. doi: 10.1097/01.jcp.0000106221.36344.4d.  [DOI](https://doi.org/10.1097/01.jcp.0000106221.36344.4d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14709950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Psychopharmacol.&title=Influence%20of%20dose,%20cigarette%20smoking,%20age,%20sex,%20and%20metabolic%20activity%20on%20plasma%20clozapine%20concentrations:%20A%20predictive%20model%20and%20nomograms%20to%20aid%20clozapine%20dose%20adjustment%20and%20to%20assess%20compliance%20in%20individual%20patients&author=A.%20Rostami-Hodjegan&author=A.M.%20Amin&author=E.P.%20Spencer&author=M.S.%20Lennard&author=G.T.%20Tucker&volume=24&publication_year=2004&pages=70-78&pmid=14709950&doi=10.1097/01.jcp.0000106221.36344.4d&)

10. Domanski T.L., Finta C., Halpert J.R., Zaphiropoulos P.G. cDNA cloning and initial characterization of CYP3A43, a novel human cytochrome P450. Mol. Pharmacol. 2001;59:386–392. doi: 10.1124/mol.59.2.386.  [DOI](https://doi.org/10.1124/mol.59.2.386) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11160876/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=cDNA%20cloning%20and%20initial%20characterization%20of%20CYP3A43,%20a%20novel%20human%20cytochrome%20P450&author=T.L.%20Domanski&author=C.%20Finta&author=J.R.%20Halpert&author=P.G.%20Zaphiropoulos&volume=59&publication_year=2001&pages=386-392&pmid=11160876&doi=10.1124/mol.59.2.386&)

11. Shimada T., Yamazaki H., Mimura M., Inui Y., Guengerich F.P. Interindividual variations in human liver cytochrome P-450 enzymes involved in the oxidation of drugs, carcinogens and toxic chemicals: Studies with liver microsomes of 30 Japanese and 30 Caucasians. J. Pharmacol. Exp. Ther. 1994;270:414–423.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8035341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=Interindividual%20variations%20in%20human%20liver%20cytochrome%20P-450%20enzymes%20involved%20in%20the%20oxidation%20of%20drugs,%20carcinogens%20and%20toxic%20chemicals:%20Studies%20with%20liver%20microsomes%20of%2030%20Japanese%20and%2030%20Caucasians&author=T.%20Shimada&author=H.%20Yamazaki&author=M.%20Mimura&author=Y.%20Inui&author=F.P.%20Guengerich&volume=270&publication_year=1994&pages=414-423&pmid=8035341&)

12. Brooks B.A., McBride O.W., Dolphin C.T., Farrall M., Scambler P.J., Gonzalez F.J., Idle J.R. The gene CYP3 encoding P450pcn1 (nifedipine oxidase) is tightly linked to the gene COL1A2 encoding collagen type 1 alpha on 7q21-q22. 1. Am. J. Hum. Genet. 1988;43:280.  [PMC free article](/articles/PMC1715388/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2901225/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=The%20gene%20CYP3%20encoding%20P450pcn1%20(nifedipine%20oxidase)%20is%20tightly%20linked%20to%20the%20gene%20COL1A2%20encoding%20collagen%20type%201%20alpha%20on%207q21-q22.%201&author=B.A.%20Brooks&author=O.W.%20McBride&author=C.T.%20Dolphin&author=M.%20Farrall&author=P.J.%20Scambler&volume=43&publication_year=1988&pages=280&pmid=2901225&)

13. Finta C., Zaphiropoulos P.G. The human cytochrome P450 3A locus. Gene evolution by capture of downstream exons. Gene. 2000;260:13–23. doi: 10.1016/S0378-1119(00)00470-4.  [DOI](https://doi.org/10.1016/S0378-1119(00)00470-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11137287/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=The%20human%20cytochrome%20P450%203A%20locus.%20Gene%20evolution%20by%20capture%20of%20downstream%20exons&author=C.%20Finta&author=P.G.%20Zaphiropoulos&volume=260&publication_year=2000&pages=13-23&pmid=11137287&doi=10.1016/S0378-1119(00)00470-4&)

14. Hashimoto H., Toide K., Kitamura R., Fujita M., Tagawa S., Itoh S., Kamataki T. Gene structure of CYP3A4, an adult-specific form of cytochrome P450 in human livers, and its transcriptional control. Eur. J. Bioch. 1993;218:585–595. doi: 10.1111/j.1432-1033.1993.tb18412.x.  [DOI](https://doi.org/10.1111/j.1432-1033.1993.tb18412.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8269949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Bioch.&title=Gene%20structure%20of%20CYP3A4,%20an%20adult-specific%20form%20of%20cytochrome%20P450%20in%20human%20livers,%20and%20its%20transcriptional%20control&author=H.%20Hashimoto&author=K.%20Toide&author=R.%20Kitamura&author=M.%20Fujita&author=S.%20Tagawa&volume=218&publication_year=1993&pages=585-595&pmid=8269949&doi=10.1111/j.1432-1033.1993.tb18412.x&)

15. Gellner K., Eiselt R., Hustert E., Arnold H., Koch I., Haberl M., Deglmann C.J., Burk O., Buntefuss D., Escher S., et al. Genomic organization of the human CYP3A locus: Identification of a new, inducible CYP3A gene. Pharm. Genom. 2001;11:111–121. doi: 10.1097/00008571-200103000-00002.  [DOI](https://doi.org/10.1097/00008571-200103000-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11266076/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Genom.&title=Genomic%20organization%20of%20the%20human%20CYP3A%20locus:%20Identification%20of%20a%20new,%20inducible%20CYP3A%20gene&author=K.%20Gellner&author=R.%20Eiselt&author=E.%20Hustert&author=H.%20Arnold&author=I.%20Koch&volume=11&publication_year=2001&pages=111-121&pmid=11266076&doi=10.1097/00008571-200103000-00002&)

16. Plant N. The human cytochrome P450 sub-family: Transcriptional regulation, inter-individual variation and interaction networks. Biochim. Biophys. Acta. 2007;1770:478–488. doi: 10.1016/j.bbagen.2006.09.024.  [DOI](https://doi.org/10.1016/j.bbagen.2006.09.024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17097810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta&title=The%20human%20cytochrome%20P450%20sub-family:%20Transcriptional%20regulation,%20inter-individual%20variation%20and%20interaction%20networks&author=N.%20Plant&volume=1770&publication_year=2007&pages=478-488&pmid=17097810&doi=10.1016/j.bbagen.2006.09.024&)

17. Elens L., Van Gelder T., Hesselink D.A., Haufroid V., Van Schaik R.H. CYP3A4* 22: Promising newly identified CYP3A4 variant allele for personalizing pharmacotherapy. Pharmacogenomics. 2013;14:47–62. doi: 10.2217/pgs.12.187.  [DOI](https://doi.org/10.2217/pgs.12.187) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23252948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CYP3A4*%2022:%20Promising%20newly%20identified%20CYP3A4%20variant%20allele%20for%20personalizing%20pharmacotherapy&author=L.%20Elens&author=T.%20Van%20Gelder&author=D.A.%20Hesselink&author=V.%20Haufroid&author=R.H.%20Van%20Schaik&volume=14&publication_year=2013&pages=47-62&pmid=23252948&doi=10.2217/pgs.12.187&)

18. Daly A.K. Significance of the minor cytochrome P450 3A isoforms. Clin. Pharm. 2006;45:13–31. doi: 10.2165/00003088-200645010-00002.  [DOI](https://doi.org/10.2165/00003088-200645010-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16430309/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharm.&title=Significance%20of%20the%20minor%20cytochrome%20P450%203A%20isoforms&author=A.K.%20Daly&volume=45&publication_year=2006&pages=13-31&pmid=16430309&doi=10.2165/00003088-200645010-00002&)

19. Shimada T., Yamazaki H., Mimura M., Wakamiya N., Ueng Y.F., Guengerich F.P., Inui Y. Characterization of microsomal cytochrome P450 enzymes involved in the oxidation of xenobiotic chemicals in human fetal liver and adult lungs. Drug Metab. Dispos. 1996;24:515–522.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8723730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Characterization%20of%20microsomal%20cytochrome%20P450%20enzymes%20involved%20in%20the%20oxidation%20of%20xenobiotic%20chemicals%20in%20human%20fetal%20liver%20and%20adult%20lungs&author=T.%20Shimada&author=H.%20Yamazaki&author=M.%20Mimura&author=N.%20Wakamiya&author=Y.F.%20Ueng&volume=24&publication_year=1996&pages=515-522&pmid=8723730&)

20. Szklarczyk D., Gable A.L., Lyon D., Junge A., Wyder S., Huerta-Cepas J., Simonovic M., Doncheva N.T., Morris J.H., Bork P., et al. STRING v11: Protein–protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets. Nucleic Acids Res. 2019;47:D607–D613. doi: 10.1093/nar/gky1131.  [DOI](https://doi.org/10.1093/nar/gky1131) | [PMC free article](/articles/PMC6323986/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30476243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=STRING%20v11:%20Protein%E2%80%93protein%20association%20networks%20with%20increased%20coverage,%20supporting%20functional%20discovery%20in%20genome-wide%20experimental%20datasets&author=D.%20Szklarczyk&author=A.L.%20Gable&author=D.%20Lyon&author=A.%20Junge&author=S.%20Wyder&volume=47&publication_year=2019&pages=D607-D613&pmid=30476243&doi=10.1093/nar/gky1131&)

21. Williams J.A., Ring B.J., Cantrell V.E., Jones D.R., Eckstein J., Ruterbories K., Hamman M.A., Hall S.D., Wrighton S.A. Comparative metabolic capabilities of CYP3A4, CYP3A5, and CYP3A7. Drug Metab. Dispos. 2002;30:883–891. doi: 10.1124/dmd.30.8.883.  [DOI](https://doi.org/10.1124/dmd.30.8.883) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12124305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Comparative%20metabolic%20capabilities%20of%20CYP3A4,%20CYP3A5,%20and%20CYP3A7&author=J.A.%20Williams&author=B.J.%20Ring&author=V.E.%20Cantrell&author=D.R.%20Jones&author=J.%20Eckstein&volume=30&publication_year=2002&pages=883-891&pmid=12124305&doi=10.1124/dmd.30.8.883&)

22. Huang W., Lin Y.S., McConn D.J., Calamia J.C., Totah R.A., Isoherranen N., Glodowski M., Thummel K.E. Evidence of significant contribution from CYP3A5 to hepatic drug metabolism. Drug Metab. Dispos. 2004;32:1434–1445. doi: 10.1124/dmd.104.001313.  [DOI](https://doi.org/10.1124/dmd.104.001313) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15383492/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Evidence%20of%20significant%20contribution%20from%20CYP3A5%20to%20hepatic%20drug%20metabolism&author=W.%20Huang&author=Y.S.%20Lin&author=D.J.%20McConn&author=J.C.%20Calamia&author=R.A.%20Totah&volume=32&publication_year=2004&pages=1434-1445&pmid=15383492&doi=10.1124/dmd.104.001313&)

23. Kivistö K.T., Bookjans G., Fromm M.F., Griese E.-U., Münzel P., KRoemer H.K. Expression of CYP3A4, CYP3A5 and CYP3A7 in human duodenal tissue. Br. J. Clin. Pharm. 1996;42:387–389. doi: 10.1046/j.1365-2125.1996.42615.x.  [DOI](https://doi.org/10.1046/j.1365-2125.1996.42615.x) | [PMC free article](/articles/PMC2042681/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8877031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharm.&title=Expression%20of%20CYP3A4,%20CYP3A5%20and%20CYP3A7%20in%20human%20duodenal%20tissue&author=K.T.%20Kivist%C3%B6&author=G.%20Bookjans&author=M.F.%20Fromm&author=E.-U.%20Griese&author=P.%20M%C3%BCnzel&volume=42&publication_year=1996&pages=387-389&pmid=8877031&doi=10.1046/j.1365-2125.1996.42615.x&)

24. Kleinbloesem C.H., van Brummelen P., Faber H., Danhof M., Vermeulen N.P., Breimer D.D. Variability in nifedipine pharmacokinetics and dynamics: A new oxidation polymorphism in man. Biochem. Pharmacol. 1984;33:3721–3724. doi: 10.1016/0006-2952(84)90165-5.  [DOI](https://doi.org/10.1016/0006-2952(84)90165-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6508829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Pharmacol.&title=Variability%20in%20nifedipine%20pharmacokinetics%20and%20dynamics:%20A%20new%20oxidation%20polymorphism%20in%20man&author=C.H.%20Kleinbloesem&author=P.%20van%20Brummelen&author=H.%20Faber&author=M.%20Danhof&author=N.P.%20Vermeulen&volume=33&publication_year=1984&pages=3721-3724&pmid=6508829&doi=10.1016/0006-2952(84)90165-5&)

25. Guengerich F.P., Martin M.V., Beaune P.H., Kremers P., Wolff T., Waxman D.J. Characterization of rat and human liver microsomal cytochrome P-450 forms involved in nifedipine oxidation, a prototype for genetic polymorphism in oxidative drug metabolism. J. Biol. Chem. 1986;261:5051–5060.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/3514607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Characterization%20of%20rat%20and%20human%20liver%20microsomal%20cytochrome%20P-450%20forms%20involved%20in%20nifedipine%20oxidation,%20a%20prototype%20for%20genetic%20polymorphism%20in%20oxidative%20drug%20metabolism&author=F.P.%20Guengerich&author=M.V.%20Martin&author=P.H.%20Beaune&author=P.%20Kremers&author=T.%20Wolff&volume=261&publication_year=1986&pages=5051-5060&pmid=3514607&)

26. Inoue K., Inazawa J., Nakagawa H., Shimada T., Yamazaki H., Guengerich F.P., Abe T. Assignment of the human cytochrome P-450 nifedipine oxidase gene (CYP3A4) to chromosome 7 at band q22. 1 by fluorescencein situ hybridization. Jpn. Hum. Genet. 1992;37:133–138. doi: 10.1007/BF01899734.  [DOI](https://doi.org/10.1007/BF01899734) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1391968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Jpn.%20Hum.%20Genet.&title=Assignment%20of%20the%20human%20cytochrome%20P-450%20nifedipine%20oxidase%20gene%20(CYP3A4)%20to%20chromosome%207%20at%20band%20q22.%201%20by%20fluorescencein%20situ%20hybridization&author=K.%20Inoue&author=J.%20Inazawa&author=H.%20Nakagawa&author=T.%20Shimada&author=H.%20Yamazaki&volume=37&publication_year=1992&pages=133-138&pmid=1391968&doi=10.1007/BF01899734&)

27. The Nucleotide Database Nucleotide–NCBI. [(accessed on 24 February 2020)]; Available online: https://www.ncbi.nlm.nih.gov/nuccore/197313719?report=graph.  [https://www.ncbi.nlm.nih.gov/nuccore/197313719?report=graph](https://www.ncbi.nlm.nih.gov/nuccore/197313719?report=graph)

28. Ensembl Genome Browser. [(accessed on 24 February 2020)]; Available online: http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000160868;r=7:99756960-99784248;t=ENST00000415003.  [http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000160868;r=7:99756960-99784248;t=ENST00000415003](http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000160868;r=7:99756960-99784248;t=ENST00000415003)

29. Achour B., Barber J., Rostami-Hodjegan A. Expression of hepatic drug-metabolizing cytochrome p450 enzymes and their intercorrelations: A meta-analysis. Drug Metab. Dispos. 2014;42:1349–1356. doi: 10.1124/dmd.114.058834.  [DOI](https://doi.org/10.1124/dmd.114.058834) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24879845/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Expression%20of%20hepatic%20drug-metabolizing%20cytochrome%20p450%20enzymes%20and%20their%20intercorrelations:%20A%20meta-analysis&author=B.%20Achour&author=J.%20Barber&author=A.%20Rostami-Hodjegan&volume=42&publication_year=2014&pages=1349-1356&pmid=24879845&doi=10.1124/dmd.114.058834&)

30. Michaels S., Wang M.Z. The Revised Human Liver Cytochrome P450 “Pie”: Absolute Protein Quantification of CYP4F and CYP3A Enzymes Using Targeted Quantitative Proteomics. Drug Metab. Dispos. 2014;42:1241–1251. doi: 10.1124/dmd.114.058040.  [DOI](https://doi.org/10.1124/dmd.114.058040) | [PMC free article](/articles/PMC4109210/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24816681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=The%20Revised%20Human%20Liver%20Cytochrome%20P450%20%E2%80%9CPie%E2%80%9D:%20Absolute%20Protein%20Quantification%20of%20CYP4F%20and%20CYP3A%20Enzymes%20Using%20Targeted%20Quantitative%20Proteomics&author=S.%20Michaels&author=M.Z.%20Wang&volume=42&publication_year=2014&pages=1241-1251&pmid=24816681&doi=10.1124/dmd.114.058040&)

31. Zhang Q.Y., Dunbar D., Ostrowska A., Zeisloft S., Yang J., Kaminsky L.S. Characterization of human small intestinal cytochromes P-450. Drug Metab. Dispos. 1999;27:804–809.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10383924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Characterization%20of%20human%20small%20intestinal%20cytochromes%20P-450&author=Q.Y.%20Zhang&author=D.%20Dunbar&author=A.%20Ostrowska&author=S.%20Zeisloft&author=J.%20Yang&volume=27&publication_year=1999&pages=804-809&pmid=10383924&)

32. Paine M.F., Shen D.D., Kunze K.L., Perkins J.D., Marsh C.L., McVicar J.P., Barr D.M., Gillies B.S., Thummel K.E. First-pass metabolism of midazolam by the human intestine. Clin. Pharm. Ther. 1996;60:14–24. doi: 10.1016/S0009-9236(96)90162-9.  [DOI](https://doi.org/10.1016/S0009-9236(96)90162-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8689807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharm.%20Ther.&title=First-pass%20metabolism%20of%20midazolam%20by%20the%20human%20intestine&author=M.F.%20Paine&author=D.D.%20Shen&author=K.L.%20Kunze&author=J.D.%20Perkins&author=C.L.%20Marsh&volume=60&publication_year=1996&pages=14-24&pmid=8689807&doi=10.1016/S0009-9236(96)90162-9&)

33. Kolars J.C., Watkins P.B., Merion R.M., Awni W.M. First-pass metabolism of cyclosporin by the gut. Lancet. 1991;338:1488–1490. doi: 10.1016/0140-6736(91)92302-I.  [DOI](https://doi.org/10.1016/0140-6736(91)92302-I) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1683920/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=First-pass%20metabolism%20of%20cyclosporin%20by%20the%20gut&author=J.C.%20Kolars&author=P.B.%20Watkins&author=R.M.%20Merion&author=W.M.%20Awni&volume=338&publication_year=1991&pages=1488-1490&pmid=1683920&doi=10.1016/0140-6736(91)92302-I&)

34. Westlind-Johnsson A., Hermann R., Huennemeyer A., Hauns B., Lahu G., Nassr N., Zech K., Ingelman-Sundberg M., von Richter O. Identification and characterization of CYP3A4*20, a novel rare CYP3A4 allele without functional activity. Clin. Pharmacol. Ther. 2006;79:339–349. doi: 10.1016/j.clpt.2005.11.015.  [DOI](https://doi.org/10.1016/j.clpt.2005.11.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16580902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Identification%20and%20characterization%20of%20CYP3A4*20,%20a%20novel%20rare%20CYP3A4%20allele%20without%20functional%20activity&author=A.%20Westlind-Johnsson&author=R.%20Hermann&author=A.%20Huennemeyer&author=B.%20Hauns&author=G.%20Lahu&volume=79&publication_year=2006&pages=339-349&pmid=16580902&doi=10.1016/j.clpt.2005.11.015&)

35. Wang D., Guo Y., Wrighton S.A., Cooke G.E., Sadee W. Intronic polymorphism in CYP3A4 affects hepatic expression and response to statin drugs. Pharm. J. 2011;11:274–286. doi: 10.1038/tpj.2010.28.  [DOI](https://doi.org/10.1038/tpj.2010.28) | [PMC free article](/articles/PMC3248744/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20386561/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20J.&title=Intronic%20polymorphism%20in%20CYP3A4%20affects%20hepatic%20expression%20and%20response%20to%20statin%20drugs&author=D.%20Wang&author=Y.%20Guo&author=S.A.%20Wrighton&author=G.E.%20Cooke&author=W.%20Sadee&volume=11&publication_year=2011&pages=274-286&pmid=20386561&doi=10.1038/tpj.2010.28&)

36. Klein K., Thomas M., Winter S., Nussler A.K., Niemi M., Schwab M., Zanger U.M. PPARA: A novel genetic determinant of CYP3A4 in vitro and in vivo. Clin. Pharm. Ther. 2012;91:1044–1052. doi: 10.1038/clpt.2011.336.  [DOI](https://doi.org/10.1038/clpt.2011.336) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22510778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharm.%20Ther.&title=PPARA:%20A%20novel%20genetic%20determinant%20of%20CYP3A4%20in%20vitro%20and%20in%20vivo&author=K.%20Klein&author=M.%20Thomas&author=S.%20Winter&author=A.K.%20Nussler&author=M.%20Niemi&volume=91&publication_year=2012&pages=1044-1052&pmid=22510778&doi=10.1038/clpt.2011.336&)

37. Apellániz-Ruiz M., Inglada-Pérez L., Naranjo M.E.G., Sánchez L., Mancikova V., Currás-Freixes M., de Cubas A.A., Comino-Méndez I., Triki S., Rebai A., et al. High frequency and founder effect of the CYP3A4*20 loss-of-function allele in the Spanish population classifies CYP3A4 as a polymorphic enzyme. Pharm. J. 2015;15:288–292. doi: 10.1038/tpj.2014.67.  [DOI](https://doi.org/10.1038/tpj.2014.67) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25348618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20J.&title=High%20frequency%20and%20founder%20effect%20of%20the%20CYP3A4*20%20loss-of-function%20allele%20in%20the%20Spanish%20population%20classifies%20CYP3A4%20as%20a%20polymorphic%20enzyme&author=M.%20Apell%C3%A1niz-Ruiz&author=L.%20Inglada-P%C3%A9rez&author=M.E.G.%20Naranjo&author=L.%20S%C3%A1nchez&author=V.%20Mancikova&volume=15&publication_year=2015&pages=288-292&pmid=25348618&doi=10.1038/tpj.2014.67&)

38. Tavira B., Coto E., Diaz-Corte C., Alvarez V., López-Larrea C., Ortega F. A search for new CYP3A4 variants as determinants of tacrolimus dose requirements in renal-transplanted patients. Pharm. Genom. 2013;23:445–448. doi: 10.1097/FPC.0b013e3283636856.  [DOI](https://doi.org/10.1097/FPC.0b013e3283636856) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23778326/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Genom.&title=A%20search%20for%20new%20CYP3A4%20variants%20as%20determinants%20of%20tacrolimus%20dose%20requirements%20in%20renal-transplanted%20patients&author=B.%20Tavira&author=E.%20Coto&author=C.%20Diaz-Corte&author=V.%20Alvarez&author=C.%20L%C3%B3pez-Larrea&volume=23&publication_year=2013&pages=445-448&pmid=23778326&doi=10.1097/FPC.0b013e3283636856&)

39. Santoro A.B., Struchiner C.J., Felipe C.R., Tedesco-Silva H., Medina-Pestana J.O., Suarez-Kurtz G. CYP3A5 genotype, but not CYP3A4*1b, CYP3A4*22, or hematocrit, predicts tacrolimus dose requirements in Brazilian renal transplant patients. Clin. Pharmacol. Ther. 2013;94:201–202. doi: 10.1038/clpt.2013.68.  [DOI](https://doi.org/10.1038/clpt.2013.68) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23588314/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=CYP3A5%20genotype,%20but%20not%20CYP3A4*1b,%20CYP3A4*22,%20or%20hematocrit,%20predicts%20tacrolimus%20dose%20requirements%20in%20Brazilian%20renal%20transplant%20patients&author=A.B.%20Santoro&author=C.J.%20Struchiner&author=C.R.%20Felipe&author=H.%20Tedesco-Silva&author=J.O.%20Medina-Pestana&volume=94&publication_year=2013&pages=201-202&pmid=23588314&doi=10.1038/clpt.2013.68&)

40. Santoro A., Felipe C.R., Tedesco-Silva H., Medina-Pestana J.O., Struchiner C.J., Ojopi E.B., Suarez-Kurtz G. Pharmacogenetics of calcineurin inhibitors in Brazilian renal transplant patients. Pharmacogenomics. 2011;12:1293–1303. doi: 10.2217/pgs.11.70.  [DOI](https://doi.org/10.2217/pgs.11.70) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21806386/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetics%20of%20calcineurin%20inhibitors%20in%20Brazilian%20renal%20transplant%20patients&author=A.%20Santoro&author=C.R.%20Felipe&author=H.%20Tedesco-Silva&author=J.O.%20Medina-Pestana&author=C.J.%20Struchiner&volume=12&publication_year=2011&pages=1293-1303&pmid=21806386&doi=10.2217/pgs.11.70&)

41. Kurzawski M., Dąbrowska J., Dziewanowski K., Domański L., Perużyńska M., Droździk M. CYP3A5 and CYP3A4, but not ABCB1 polymorphisms affect tacrolimus dose-adjusted trough concentrations in kidney transplant recipients. Pharmacogenomics. 2014;15:179–188. doi: 10.2217/pgs.13.199.  [DOI](https://doi.org/10.2217/pgs.13.199) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24444408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CYP3A5%20and%20CYP3A4,%20but%20not%20ABCB1%20polymorphisms%20affect%20tacrolimus%20dose-adjusted%20trough%20concentrations%20in%20kidney%20transplant%20recipients&author=M.%20Kurzawski&author=J.%20D%C4%85browska&author=K.%20Dziewanowski&author=L.%20Doma%C5%84ski&author=M.%20Peru%C5%BCy%C5%84ska&volume=15&publication_year=2014&pages=179-188&pmid=24444408&doi=10.2217/pgs.13.199&)

42. de Jonge H., Elens L., de Loor H., van Schaik R.H., Kuypers D.R.J. The CYP3A4*22 C>T single nucleotide polymorphism is associated with reduced midazolam and tacrolimus clearance in stable renal allograft recipients. Pharm. J. 2015;15:144–152. doi: 10.1038/tpj.2014.49.  [DOI](https://doi.org/10.1038/tpj.2014.49) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25287072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20J.&title=The%20CYP3A4*22%20C>T%20single%20nucleotide%20polymorphism%20is%20associated%20with%20reduced%20midazolam%20and%20tacrolimus%20clearance%20in%20stable%20renal%20allograft%20recipients&author=H.%20de%20Jonge&author=L.%20Elens&author=H.%20de%20Loor&author=R.H.%20van%20Schaik&author=D.R.J.%20Kuypers&volume=15&publication_year=2015&pages=144-152&pmid=25287072&doi=10.1038/tpj.2014.49&)

43. Elens L., van Schaik R.H., Panin N., de Meyer M., Wallemacq P., Lison D., Mourad M., Haufroid V. Effect of a new functional CYP3A4 polymorphism on calcineurin inhibitors’ dose requirements and trough blood levels in stable renal transplant patients. Pharmacogenomics. 2011;12:1383–1396. doi: 10.2217/pgs.11.90.  [DOI](https://doi.org/10.2217/pgs.11.90) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21902502/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Effect%20of%20a%20new%20functional%20CYP3A4%20polymorphism%20on%20calcineurin%20inhibitors%E2%80%99%20dose%20requirements%20and%20trough%20blood%20levels%20in%20stable%20renal%20transplant%20patients&author=L.%20Elens&author=R.H.%20van%20Schaik&author=N.%20Panin&author=M.%20de%20Meyer&author=P.%20Wallemacq&volume=12&publication_year=2011&pages=1383-1396&pmid=21902502&doi=10.2217/pgs.11.90&)

44. Elens L., Becker M.L., Haufroid V., Hofman A., Visser L.E., Uitterlinden A.G., Stricker B.C., van Schaik R.H.N. Novel CYP3A4 intron 6 single nucleotide polymorphism is associated with simvastatin-mediated cholesterol reduction in the Rotterdam Study. Pharm. Genom. 2011;21:861–866. doi: 10.1097/FPC.0b013e32834c6edb.  [DOI](https://doi.org/10.1097/FPC.0b013e32834c6edb) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21946898/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Genom.&title=Novel%20CYP3A4%20intron%206%20single%20nucleotide%20polymorphism%20is%20associated%20with%20simvastatin-mediated%20cholesterol%20reduction%20in%20the%20Rotterdam%20Study&author=L.%20Elens&author=M.L.%20Becker&author=V.%20Haufroid&author=A.%20Hofman&author=L.E.%20Visser&volume=21&publication_year=2011&pages=861-866&pmid=21946898&doi=10.1097/FPC.0b013e32834c6edb&)

45. Rebbeck T.R., Jaffe J.M., Walker A.H., Wein A.J., Malkowicz S.B. Modification of clinical presentation of prostate tumors by a novel genetic variant in CYP3A4. J. Natl. Cancer Inst. 1998;90:1225–1229. doi: 10.1093/jnci/90.16.1225.  [DOI](https://doi.org/10.1093/jnci/90.16.1225) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9719084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst.&title=Modification%20of%20clinical%20presentation%20of%20prostate%20tumors%20by%20a%20novel%20genetic%20variant%20in%20CYP3A4&author=T.R.%20Rebbeck&author=J.M.%20Jaffe&author=A.H.%20Walker&author=A.J.%20Wein&author=S.B.%20Malkowicz&volume=90&publication_year=1998&pages=1225-1229&pmid=9719084&doi=10.1093/jnci/90.16.1225&)

46. Westlind A., Löfberg L., Tindberg N., Andersson T.B., Ingelman-Sundberg M. Interindividual differences in hepatic expression of CYP3A4: Relationship to genetic polymorphism in the 5′-upstream regulatory region. Biochem. Biophys. Res. Commun. 1999;259:201–205. doi: 10.1006/bbrc.1999.0752.  [DOI](https://doi.org/10.1006/bbrc.1999.0752) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10334940/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&title=Interindividual%20differences%20in%20hepatic%20expression%20of%20CYP3A4:%20Relationship%20to%20genetic%20polymorphism%20in%20the%205%E2%80%B2-upstream%20regulatory%20region&author=A.%20Westlind&author=L.%20L%C3%B6fberg&author=N.%20Tindberg&author=T.B.%20Andersson&author=M.%20Ingelman-Sundberg&volume=259&publication_year=1999&pages=201-205&pmid=10334940&doi=10.1006/bbrc.1999.0752&)

47. Rodríguez-Antona C., Sayi J.G., Gustafsson L.L., Bertilsson L., Ingelman-Sundberg M. Phenotype-genotype variability in the human CYP3A locus as assessed by the probe drug quinine and analyses of variant CYP3A4 alleles. Biochem. Biophys. Res. Commun. 2005;338:299–305. doi: 10.1016/j.bbrc.2005.09.020.  [DOI](https://doi.org/10.1016/j.bbrc.2005.09.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16171783/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&title=Phenotype-genotype%20variability%20in%20the%20human%20CYP3A%20locus%20as%20assessed%20by%20the%20probe%20drug%20quinine%20and%20analyses%20of%20variant%20CYP3A4%20alleles&author=C.%20Rodr%C3%ADguez-Antona&author=J.G.%20Sayi&author=L.L.%20Gustafsson&author=L.%20Bertilsson&author=M.%20Ingelman-Sundberg&volume=338&publication_year=2005&pages=299-305&pmid=16171783&doi=10.1016/j.bbrc.2005.09.020&)

48. Amirimani B., Ning B., Deitz A.C., Weber B.L., Kadlubar F.F., Rebbeck T.R. Increased transcriptional activity of the CYP3A4*1B promoter variant. Environ. Mol. Mutagen. 2003;42:299–305. doi: 10.1002/em.10199.  [DOI](https://doi.org/10.1002/em.10199) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14673875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ.%20Mol.%20Mutagen.&title=Increased%20transcriptional%20activity%20of%20the%20CYP3A4*1B%20promoter%20variant&author=B.%20Amirimani&author=B.%20Ning&author=A.C.%20Deitz&author=B.L.%20Weber&author=F.F.%20Kadlubar&volume=42&publication_year=2003&pages=299-305&pmid=14673875&doi=10.1002/em.10199&)

49. Hesselink D.A., van Schaik R.H.N., van der Heiden I.P., van der Werf M., Gregoor P.J.H.S., Lindemans J., Weimar W., van Gelder T. Genetic polymorphisms of the CYP3A4, CYP3A5, and MDR-1 genes and pharmacokinetics of the calcineurin inhibitors cyclosporine and tacrolimus. Clin. Pharmacol. Ther. 2003;74:245–254. doi: 10.1016/S0009-9236(03)00168-1.  [DOI](https://doi.org/10.1016/S0009-9236(03)00168-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12966368/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genetic%20polymorphisms%20of%20the%20CYP3A4,%20CYP3A5,%20and%20MDR-1%20genes%20and%20pharmacokinetics%20of%20the%20calcineurin%20inhibitors%20cyclosporine%20and%20tacrolimus&author=D.A.%20Hesselink&author=R.H.N.%20van%20Schaik&author=I.P.%20van%20der%20Heiden&author=M.%20van%20der%20Werf&author=P.J.H.S.%20Gregoor&volume=74&publication_year=2003&pages=245-254&pmid=12966368&doi=10.1016/S0009-9236(03)00168-1&)

50. Tavira B., Garciá E.C., Díaz-Corte C., Ortega F., Arias M., Torres A., Díaz J.M., Selgas R., López-Larrea C., Campistol J.M. Pharmacogenetics of tacrolimus after renal transplantation: Analysis of polymorphisms in genes encoding 16 drug metabolizing enzymes. Clin. Chem. Lab. Med. 2011;49:825–833. doi: 10.1515/CCLM.2011.143.  [DOI](https://doi.org/10.1515/CCLM.2011.143) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21480817/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Chem.%20Lab.%20Med.&title=Pharmacogenetics%20of%20tacrolimus%20after%20renal%20transplantation:%20Analysis%20of%20polymorphisms%20in%20genes%20encoding%2016%20drug%20metabolizing%20enzymes&author=B.%20Tavira&author=E.C.%20Garci%C3%A1&author=C.%20D%C3%ADaz-Corte&author=F.%20Ortega&author=M.%20Arias&volume=49&publication_year=2011&pages=825-833&pmid=21480817&doi=10.1515/CCLM.2011.143&)

51. Gervasini G., Garcia M., Macias R.M., Cubero J.J., Caravaca F., Benitez J. Impact of genetic polymorphisms on tacrolimus pharmacokinetics and the clinical outcome of renal transplantation. Transpl. Int. 2012;25:471–480. doi: 10.1111/j.1432-2277.2012.01446.x.  [DOI](https://doi.org/10.1111/j.1432-2277.2012.01446.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22369694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transpl.%20Int.&title=Impact%20of%20genetic%20polymorphisms%20on%20tacrolimus%20pharmacokinetics%20and%20the%20clinical%20outcome%20of%20renal%20transplantation&author=G.%20Gervasini&author=M.%20Garcia&author=R.M.%20Macias&author=J.J.%20Cubero&author=F.%20Caravaca&volume=25&publication_year=2012&pages=471-480&pmid=22369694&doi=10.1111/j.1432-2277.2012.01446.x&)

52. Crettol S., Venetz J.-P., Fontana M., Aubert J.-D., Pascual M., Eap C.B. CYP3A7, CYP3A5, CYP3A4, and ABCB1 genetic polymorphisms, cyclosporine concentration, and dose requirement in transplant recipients. Ther. Drug Monit. 2008;30:689–699. doi: 10.1097/FTD.0b013e31818a2a60.  [DOI](https://doi.org/10.1097/FTD.0b013e31818a2a60) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18978522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther.%20Drug%20Monit.&title=CYP3A7,%20CYP3A5,%20CYP3A4,%20and%20ABCB1%20genetic%20polymorphisms,%20cyclosporine%20concentration,%20and%20dose%20requirement%20in%20transplant%20recipients&author=S.%20Crettol&author=J.-P.%20Venetz&author=M.%20Fontana&author=J.-D.%20Aubert&author=M.%20Pascual&volume=30&publication_year=2008&pages=689-699&pmid=18978522&doi=10.1097/FTD.0b013e31818a2a60&)

53. Żochowska D., Wyzgał J., Pączek L. Impact of CYP3A4*1B and CYP3A5*3 polymorphisms on the pharmacokinetics of cyclosporine and sirolimus in renal transplant recipients. Ann. Transplant. 2012;17:36–44. doi: 10.12659/aot.883456.  [DOI](https://doi.org/10.12659/aot.883456) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23018254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Transplant.&title=Impact%20of%20CYP3A4*1B%20and%20CYP3A5*3%20polymorphisms%20on%20the%20pharmacokinetics%20of%20cyclosporine%20and%20sirolimus%20in%20renal%20transplant%20recipients&author=D.%20%C5%BBochowska&author=J.%20Wyzga%C5%82&author=L.%20P%C4%85czek&volume=17&publication_year=2012&pages=36-44&pmid=23018254&doi=10.12659/aot.883456&)

54. Gervasini G., García-Pino G., Vergara E., Mota-Zamorano S., García-Cerrada M., Luna E. CYP3A genotypes of donors but not those of the patients increase the risk of acute rejection in renal transplant recipients on calcineurin inhibitors: A pilot study. Eur. J. Clin. Pharmacol. 2018;74:53–60. doi: 10.1007/s00228-017-2353-9.  [DOI](https://doi.org/10.1007/s00228-017-2353-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29043387/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=CYP3A%20genotypes%20of%20donors%20but%20not%20those%20of%20the%20patients%20increase%20the%20risk%20of%20acute%20rejection%20in%20renal%20transplant%20recipients%20on%20calcineurin%20inhibitors:%20A%20pilot%20study&author=G.%20Gervasini&author=G.%20Garc%C3%ADa-Pino&author=E.%20Vergara&author=S.%20Mota-Zamorano&author=M.%20Garc%C3%ADa-Cerrada&volume=74&publication_year=2018&pages=53-60&pmid=29043387&doi=10.1007/s00228-017-2353-9&)

55. Becker M.L., Visser L.E., van Schaik R.H.N., Hofman A., Uitterlinden A.G., Stricker B.H.C. Influence of genetic variation in CYP3A4 and ABCB1 on dose decrease or switching during simvastatin and atorvastatin therapy. Pharmacoepidemiol. Drug Saf. 2010;19:75–81. doi: 10.1002/pds.1866.  [DOI](https://doi.org/10.1002/pds.1866) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19802823/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacoepidemiol.%20Drug%20Saf.&title=Influence%20of%20genetic%20variation%20in%20CYP3A4%20and%20ABCB1%20on%20dose%20decrease%20or%20switching%20during%20simvastatin%20and%20atorvastatin%20therapy&author=M.L.%20Becker&author=L.E.%20Visser&author=R.H.N.%20van%20Schaik&author=A.%20Hofman&author=A.G.%20Uitterlinden&volume=19&publication_year=2010&pages=75-81&pmid=19802823&doi=10.1002/pds.1866&)

56. Birdwell K.A., Grady B., Choi L., Xu H., Bian A., Denny J.C., Jiang M., Vranic G., Basford M., Cowan J.D., et al. The use of a DNA biobank linked to electronic medical records to characterize pharmacogenomic predictors of tacrolimus dose requirement in kidney transplant recipients. Pharm. Genom. 2012;22:32–42. doi: 10.1097/FPC.0b013e32834e1641.  [DOI](https://doi.org/10.1097/FPC.0b013e32834e1641) | [PMC free article](/articles/PMC3237759/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22108237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Genom.&title=The%20use%20of%20a%20DNA%20biobank%20linked%20to%20electronic%20medical%20records%20to%20characterize%20pharmacogenomic%20predictors%20of%20tacrolimus%20dose%20requirement%20in%20kidney%20transplant%20recipients&author=K.A.%20Birdwell&author=B.%20Grady&author=L.%20Choi&author=H.%20Xu&author=A.%20Bian&volume=22&publication_year=2012&pages=32-42&pmid=22108237&doi=10.1097/FPC.0b013e32834e1641&)

57. Dai D., Tang J., Rose R., Hodgson E., Bienstock R.J., Mohrenweiser H.W., Goldstein J.A. Identification of variants of CYP3A4 and characterization of their abilities to metabolize testosterone and chlorpyrifos. J. Pharm. Exp. Ther. 2001;299:825–831.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11714865/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharm.%20Exp.%20Ther.&title=Identification%20of%20variants%20of%20CYP3A4%20and%20characterization%20of%20their%20abilities%20to%20metabolize%20testosterone%20and%20chlorpyrifos&author=D.%20Dai&author=J.%20Tang&author=R.%20Rose&author=E.%20Hodgson&author=R.J.%20Bienstock&volume=299&publication_year=2001&pages=825-831&pmid=11714865&)

58. Fukushima-Uesaka H., Saito Y., Watanabe H., Shiseki K., Saeki M., Nakamura T., Kurose K., Sai K., Komamura K., Ueno K., et al. Haplotypes of CYP3A4 and their close linkage with CYP3A5 haplotypes in a Japanese population. Hum. Mutat. 2004;23:100. doi: 10.1002/humu.9210.  [DOI](https://doi.org/10.1002/humu.9210) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14695543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mutat.&title=Haplotypes%20of%20CYP3A4%20and%20their%20close%20linkage%20with%20CYP3A5%20haplotypes%20in%20a%20Japanese%20population&author=H.%20Fukushima-Uesaka&author=Y.%20Saito&author=H.%20Watanabe&author=K.%20Shiseki&author=M.%20Saeki&volume=23&publication_year=2004&pages=100&pmid=14695543&doi=10.1002/humu.9210&)

59. Zuo X., Ng C.M., Barrett J.S., Luo A., Zhang B., Deng C., Xi L., Cheng K., Ming Y., Yang G., et al. Effects of CYP3A4 and CYP3A5 polymorphisms on tacrolimus pharmacokinetics in Chinese adult renal transplant recipients: A population pharmacokinetic analysis. Pharm. Genom. 2013;23:251–261. doi: 10.1097/FPC.0b013e32835fcbb6.  [DOI](https://doi.org/10.1097/FPC.0b013e32835fcbb6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23459029/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Genom.&title=Effects%20of%20CYP3A4%20and%20CYP3A5%20polymorphisms%20on%20tacrolimus%20pharmacokinetics%20in%20Chinese%20adult%20renal%20transplant%20recipients:%20A%20population%20pharmacokinetic%20analysis&author=X.%20Zuo&author=C.M.%20Ng&author=J.S.%20Barrett&author=A.%20Luo&author=B.%20Zhang&volume=23&publication_year=2013&pages=251-261&pmid=23459029&doi=10.1097/FPC.0b013e32835fcbb6&)

60. Uesugi M., Hosokawa M., Shinke H., Hashimoto E., Takahashi T., Kawai T., Matsubara K., Ogawa K., Fujimoto Y., Okamoto S., et al. Influence of cytochrome P450 (CYP) 3A4*1G polymorphism on the pharmacokinetics of tacrolimus, probability of acute cellular rejection, and mRNA expression level of CYP3A5 rather than CYP3A4 in living-donor liver transplant patients. Biol. Pharm. Bull. 2013;36:1814–1821. doi: 10.1248/bpb.b13-00509.  [DOI](https://doi.org/10.1248/bpb.b13-00509) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24189425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol.%20Pharm.%20Bull.&title=Influence%20of%20cytochrome%20P450%20(CYP)%203A4*1G%20polymorphism%20on%20the%20pharmacokinetics%20of%20tacrolimus,%20probability%20of%20acute%20cellular%20rejection,%20and%20mRNA%20expression%20level%20of%20CYP3A5%20rather%20than%20CYP3A4%20in%20living-donor%20liver%20transplant%20patients&author=M.%20Uesugi&author=M.%20Hosokawa&author=H.%20Shinke&author=E.%20Hashimoto&author=T.%20Takahashi&volume=36&publication_year=2013&pages=1814-1821&pmid=24189425&doi=10.1248/bpb.b13-00509&)

61. Li C.-J., Li L., Lin L., Jiang H.-X., Zhong Z.-Y., Li W.-M., Zhang Y.-J., Zheng P., Tan X.-H., Zhou L. Impact of the CYP3A5, CYP3A4, COMT, IL-10 and POR genetic polymorphisms on tacrolimus metabolism in Chinese renal transplant recipients. PLoS ONE. 2014;9:e86206. doi: 10.1371/journal.pone.0086206.  [DOI](https://doi.org/10.1371/journal.pone.0086206) | [PMC free article](/articles/PMC3897654/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24465960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Impact%20of%20the%20CYP3A5,%20CYP3A4,%20COMT,%20IL-10%20and%20POR%20genetic%20polymorphisms%20on%20tacrolimus%20metabolism%20in%20Chinese%20renal%20transplant%20recipients&author=C.-J.%20Li&author=L.%20Li&author=L.%20Lin&author=H.-X.%20Jiang&author=Z.-Y.%20Zhong&volume=9&publication_year=2014&pages=e86206&pmid=24465960&doi=10.1371/journal.pone.0086206&)

62. Li J., Liu S., Fu Q., Zhang Y., Wang X., Liu X., Liu L., Wang C., Huang M. Interactive effects of CYP3A4, CYP3A5, MDR1 and NR1I2 polymorphisms on tracrolimus trough concentrations in early postrenal transplant recipients. Pharmacogenomics. 2015;16:1355–1365. doi: 10.2217/pgs.15.78.  [DOI](https://doi.org/10.2217/pgs.15.78) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26228923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Interactive%20effects%20of%20CYP3A4,%20CYP3A5,%20MDR1%20and%20NR1I2%20polymorphisms%20on%20tracrolimus%20trough%20concentrations%20in%20early%20postrenal%20transplant%20recipients&author=J.%20Li&author=S.%20Liu&author=Q.%20Fu&author=Y.%20Zhang&author=X.%20Wang&volume=16&publication_year=2015&pages=1355-1365&pmid=26228923&doi=10.2217/pgs.15.78&)

63. Liu M., He H., Zhang Y., Hu Y., He F., Luo J., Luo Z., Chen X., Liu Z., Zhou H., et al. IL-3 and CTLA4 gene polymorphisms may influence the tacrolimus dose requirement in Chinese kidney transplant recipients. Acta Pharm. Sin. 2017;38:415–423. doi: 10.1038/aps.2016.153.  [DOI](https://doi.org/10.1038/aps.2016.153) | [PMC free article](/articles/PMC5342670/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28112181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Pharm.%20Sin.&title=IL-3%20and%20CTLA4%20gene%20polymorphisms%20may%20influence%20the%20tacrolimus%20dose%20requirement%20in%20Chinese%20kidney%20transplant%20recipients&author=M.%20Liu&author=H.%20He&author=Y.%20Zhang&author=Y.%20Hu&author=F.%20He&volume=38&publication_year=2017&pages=415-423&pmid=28112181&doi=10.1038/aps.2016.153&)

64. Sun B., Guo Y., Gao J., Shi W., Fan G., Li X., Qiu J., Qin Y., Liu G. Influence of CYP3A and ABCB1 polymorphisms on cyclosporine concentrations in renal transplant recipients. Pharmacogenomics. 2017;18:1503–1513. doi: 10.2217/pgs-2017-0127.  [DOI](https://doi.org/10.2217/pgs-2017-0127) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28952408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Influence%20of%20CYP3A%20and%20ABCB1%20polymorphisms%20on%20cyclosporine%20concentrations%20in%20renal%20transplant%20recipients&author=B.%20Sun&author=Y.%20Guo&author=J.%20Gao&author=W.%20Shi&author=G.%20Fan&volume=18&publication_year=2017&pages=1503-1513&pmid=28952408&doi=10.2217/pgs-2017-0127&)

65. Li T.-F., Hu L., Ma X.-L., Huang L., Liu X.-M., Luo X.-X., Feng W.-Y., Wu C.-F. Population pharmacokinetics of cyclosporine in Chinese children receiving hematopoietic stem cell transplantation. Acta Pharm. Sin. 2019;40:1603–1610. doi: 10.1038/s41401-019-0277-x.  [DOI](https://doi.org/10.1038/s41401-019-0277-x) | [PMC free article](/articles/PMC7471407/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31341257/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Pharm.%20Sin.&title=Population%20pharmacokinetics%20of%20cyclosporine%20in%20Chinese%20children%20receiving%20hematopoietic%20stem%20cell%20transplantation&author=T.-F.%20Li&author=L.%20Hu&author=X.-L.%20Ma&author=L.%20Huang&author=X.-M.%20Liu&volume=40&publication_year=2019&pages=1603-1610&pmid=31341257&doi=10.1038/s41401-019-0277-x&)

66. Zhang W., Chang Y.-Z., Kan Q.-C., Zhang L.-R., Li Z.-S., Lu H., Wang Z.-Y., Chu Q.-J., Zhang J. CYP3A4*1G genetic polymorphism influences CYP3A activity and response to fentanyl in Chinese gynecologic patients. Eur. J. Clin. Pharm. 2010;66:61–66. doi: 10.1007/s00228-009-0726-4.  [DOI](https://doi.org/10.1007/s00228-009-0726-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19784640/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharm.&title=CYP3A4*1G%20genetic%20polymorphism%20influences%20CYP3A%20activity%20and%20response%20to%20fentanyl%20in%20Chinese%20gynecologic%20patients&author=W.%20Zhang&author=Y.-Z.%20Chang&author=Q.-C.%20Kan&author=L.-R.%20Zhang&author=Z.-S.%20Li&volume=66&publication_year=2010&pages=61-66&pmid=19784640&doi=10.1007/s00228-009-0726-4&)

67. Yuan R., Zhang X., Deng Q., Wu Y., Xiang G. Impact of CYP3A4*1G polymorphism on metabolism of fentanyl in Chinese patients undergoing lower abdominal surgery. Clin. Chim. Acta. 2011;412:755–760. doi: 10.1016/j.cca.2010.12.038.  [DOI](https://doi.org/10.1016/j.cca.2010.12.038) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21223952/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Chim.%20Acta&title=Impact%20of%20CYP3A4*1G%20polymorphism%20on%20metabolism%20of%20fentanyl%20in%20Chinese%20patients%20undergoing%20lower%20abdominal%20surgery&author=R.%20Yuan&author=X.%20Zhang&author=Q.%20Deng&author=Y.%20Wu&author=G.%20Xiang&volume=412&publication_year=2011&pages=755-760&pmid=21223952&doi=10.1016/j.cca.2010.12.038&)

68. Zhang W., Yuan J.-J., Kan Q.-C., Zhang L.-R., Chang Y.-Z., Wang Z.-Y., Li Z.-S. Influence of CYP3A5*3 polymorphism and interaction between CYP3A5*3 and CYP3A4*1G polymorphisms on post-operative fentanyl analgesia in Chinese patients undergoing gynaecological surgery. Eur. J. Anaesthesiol. 2011;28:245–250. doi: 10.1097/EJA.0b013e3283438b39.  [DOI](https://doi.org/10.1097/EJA.0b013e3283438b39) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21513075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Anaesthesiol.&title=Influence%20of%20CYP3A5*3%20polymorphism%20and%20interaction%20between%20CYP3A5*3%20and%20CYP3A4*1G%20polymorphisms%20on%20post-operative%20fentanyl%20analgesia%20in%20Chinese%20patients%20undergoing%20gynaecological%20surgery&author=W.%20Zhang&author=J.-J.%20Yuan&author=Q.-C.%20Kan&author=L.-R.%20Zhang&author=Y.-Z.%20Chang&volume=28&publication_year=2011&pages=245-250&pmid=21513075&doi=10.1097/EJA.0b013e3283438b39&)

69. Dong Z.-L., Li H., Chen Q.-X., Hu Y., Wu S.-J., Tang L.-Y., Gong W.-Y., Xie G.-H., Fang X.-M. Effect of CYP3A4*1G on the fentanyl consumption for intravenous patient-controlled analgesia after total abdominal hysterectomy in Chinese Han population. J. Clin. Pharm. Ther. 2012;37:153–156. doi: 10.1111/j.1365-2710.2011.01268.x.  [DOI](https://doi.org/10.1111/j.1365-2710.2011.01268.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21535061/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharm.%20Ther.&title=Effect%20of%20CYP3A4*1G%20on%20the%20fentanyl%20consumption%20for%20intravenous%20patient-controlled%20analgesia%20after%20total%20abdominal%20hysterectomy%20in%20Chinese%20Han%20population&author=Z.-L.%20Dong&author=H.%20Li&author=Q.-X.%20Chen&author=Y.%20Hu&author=S.-J.%20Wu&volume=37&publication_year=2012&pages=153-156&pmid=21535061&doi=10.1111/j.1365-2710.2011.01268.x&)

70. Zhang J., Zhang L., Zhao X., Shen S., Luo X., Zhang Y. Association between MDR1/CYP3A4/OPRM1 gene polymorphisms and the post-caesarean fentanyl analgesic effect on Chinese women. Gene. 2018;661:78–84. doi: 10.1016/j.gene.2018.03.081.  [DOI](https://doi.org/10.1016/j.gene.2018.03.081) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29601950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Association%20between%20MDR1/CYP3A4/OPRM1%20gene%20polymorphisms%20and%20the%20post-caesarean%20fentanyl%20analgesic%20effect%20on%20Chinese%20women&author=J.%20Zhang&author=L.%20Zhang&author=X.%20Zhao&author=S.%20Shen&author=X.%20Luo&volume=661&publication_year=2018&pages=78-84&pmid=29601950&doi=10.1016/j.gene.2018.03.081&)

71. Sata F., Sapone A., Elizondo G., Stocker P., Miller V.P., Zheng W., Raunio H., Crespi C.L., Gonzalez F.J. CYP3A4 allelic variants with amino acid substitutions in exons 7 and 12: Evidence for an allelic variant with altered catalytic activity. Clin. Pharmacol. Ther. 2000;67:48–56. doi: 10.1067/mcp.2000.104391.  [DOI](https://doi.org/10.1067/mcp.2000.104391) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10668853/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=CYP3A4%20allelic%20variants%20with%20amino%20acid%20substitutions%20in%20exons%207%20and%2012:%20Evidence%20for%20an%20allelic%20variant%20with%20altered%20catalytic%20activity&author=F.%20Sata&author=A.%20Sapone&author=G.%20Elizondo&author=P.%20Stocker&author=V.P.%20Miller&volume=67&publication_year=2000&pages=48-56&pmid=10668853&doi=10.1067/mcp.2000.104391&)

72. Thompson J.F., Man M., Johnson K.J., Wood L.S., Lira M.E., Lloyd D.B., Banerjee P., Milos P.M., Myrand S.P., Paulauskis J., et al. An association study of 43 SNPs in 16 candidate genes with atorvastatin response. Pharm. J. 2005;5:352–358. doi: 10.1038/sj.tpj.6500328.  [DOI](https://doi.org/10.1038/sj.tpj.6500328) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16103896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20J.&title=An%20association%20study%20of%2043%20SNPs%20in%2016%20candidate%20genes%20with%20atorvastatin%20response&author=J.F.%20Thompson&author=M.%20Man&author=K.J.%20Johnson&author=L.S.%20Wood&author=M.E.%20Lira&volume=5&publication_year=2005&pages=352-358&pmid=16103896&doi=10.1038/sj.tpj.6500328&)

73. Ensembl Genome Browser. [(accessed on 24 February 2020)]; Available online: http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000106258;r=7:99648194-99679998.  [http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000106258;r=7:99648194-99679998](http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000106258;r=7:99648194-99679998)

74. Kuehl P., Zhang J., Lin Y., Lamba J., Assem M., Schuetz J., Watkins P.B., Daly A., Wrighton S.A., Hall S.D. Sequence diversity in CYP3A promoters and characterization of the genetic basis of polymorphic CYP3A5 expression. Nat. Genet. 2001;27:383–391. doi: 10.1038/86882.  [DOI](https://doi.org/10.1038/86882) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11279519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Sequence%20diversity%20in%20CYP3A%20promoters%20and%20characterization%20of%20the%20genetic%20basis%20of%20polymorphic%20CYP3A5%20expression&author=P.%20Kuehl&author=J.%20Zhang&author=Y.%20Lin&author=J.%20Lamba&author=M.%20Assem&volume=27&publication_year=2001&pages=383-391&pmid=11279519&doi=10.1038/86882&)

75. Lamba J.K., Lin Y.S., Schuetz E.G., Thummel K.E. Genetic contribution to variable human CYP3A-mediated metabolism. Adv. Drug Deliv. Rev. 2002;54:1271–1294. doi: 10.1016/S0169-409X(02)00066-2.  [DOI](https://doi.org/10.1016/S0169-409X(02)00066-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12406645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv.%20Drug%20Deliv.%20Rev.&title=Genetic%20contribution%20to%20variable%20human%20CYP3A-mediated%20metabolism&author=J.K.%20Lamba&author=Y.S.%20Lin&author=E.G.%20Schuetz&author=K.E.%20Thummel&volume=54&publication_year=2002&pages=1271-1294&pmid=12406645&doi=10.1016/S0169-409X(02)00066-2&)

76. Miao J., Jin Y., Marunde R.L., Kim S., Quinney S., Radovich M., Li L., Hall S.D. Association of genotypes of the CYP3A cluster with midazolam disposition in vivo. Pharm. J. 2009;9:319–326. doi: 10.1038/tpj.2009.21.  [DOI](https://doi.org/10.1038/tpj.2009.21) | [PMC free article](/articles/PMC2749890/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19506580/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20J.&title=Association%20of%20genotypes%20of%20the%20CYP3A%20cluster%20with%20midazolam%20disposition%20in%20vivo&author=J.%20Miao&author=Y.%20Jin&author=R.L.%20Marunde&author=S.%20Kim&author=S.%20Quinney&volume=9&publication_year=2009&pages=319-326&pmid=19506580&doi=10.1038/tpj.2009.21&)

77. Moes D.J.A.R., Swen J.J., den Hartigh J., van der Straaten T., van der Heide J.J.H., Sanders J.S., Bemelman F.J., de Fijter J.W., Guchelaar H.J. Effect of CYP3A4*22, CYP3A5*3, and CYP3A Combined Genotypes on Cyclosporine, Everolimus, and Tacrolimus Pharmacokinetics in Renal Transplantation. CPT Pharm. Syst. Pharmacol. 2014;3:e100. doi: 10.1038/psp.2013.78.  [DOI](https://doi.org/10.1038/psp.2013.78) | [PMC free article](/articles/PMC3944116/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24522145/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CPT%20Pharm.%20Syst.%20Pharmacol.&title=Effect%20of%20CYP3A4*22,%20CYP3A5*3,%20and%20CYP3A%20Combined%20Genotypes%20on%20Cyclosporine,%20Everolimus,%20and%20Tacrolimus%20Pharmacokinetics%20in%20Renal%20Transplantation&author=D.J.A.R.%20Moes&author=J.J.%20Swen&author=J.%20den%20Hartigh&author=T.%20van%20der%20Straaten&author=J.J.H.%20van%20der%20Heide&volume=3&publication_year=2014&pages=e100&pmid=24522145&doi=10.1038/psp.2013.78&)

78. Kitzmiller J.P., Sullivan D.M., Phelps M.A., Wang D., Sadee W. CYP3A4/5 combined genotype analysis for predicting statin dose requirement for optimal lipid control. Drug Metab. Drug Interact. 2013;28:59–63. doi: 10.1515/dmdi-2012-0031.  [DOI](https://doi.org/10.1515/dmdi-2012-0031) | [PMC free article](/articles/PMC3681953/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23314529/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Drug%20Interact.&title=CYP3A4/5%20combined%20genotype%20analysis%20for%20predicting%20statin%20dose%20requirement%20for%20optimal%20lipid%20control&author=J.P.%20Kitzmiller&author=D.M.%20Sullivan&author=M.A.%20Phelps&author=D.%20Wang&author=W.%20Sadee&volume=28&publication_year=2013&pages=59-63&pmid=23314529&doi=10.1515/dmdi-2012-0031&)

79. Sanchez Spitman A.B., Moes D.J.A.R., Gelderblom H., Dezentje V.O., Swen J.J., Guchelaar H.J. Effect of CYP3A4*22, CYP3A5*3, and CYP3A combined genotypes on tamoxifen metabolism. Eur. J. Clin. Pharmacol. 2017;73:1589–1598. doi: 10.1007/s00228-017-2323-2.  [DOI](https://doi.org/10.1007/s00228-017-2323-2) | [PMC free article](/articles/PMC5684327/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28849250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=Effect%20of%20CYP3A4*22,%20CYP3A5*3,%20and%20CYP3A%20combined%20genotypes%20on%20tamoxifen%20metabolism&author=A.B.%20Sanchez%20Spitman&author=D.J.A.R.%20Moes&author=H.%20Gelderblom&author=V.O.%20Dezentje&author=J.J.%20Swen&volume=73&publication_year=2017&pages=1589-1598&pmid=28849250&doi=10.1007/s00228-017-2323-2&)

80. Hustert E., Haberl M., Burk O., Wolbold R., He Y.Q., Klein K., Nuessler A.C., Neuhaus P., Klattig J., Eiselt R., et al. The genetic determinants of the CYP3A5 polymorphism. Pharmacogenetics. 2001;11:773–779. doi: 10.1097/00008571-200112000-00005.  [DOI](https://doi.org/10.1097/00008571-200112000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11740341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20genetic%20determinants%20of%20the%20CYP3A5%20polymorphism&author=E.%20Hustert&author=M.%20Haberl&author=O.%20Burk&author=R.%20Wolbold&author=Y.Q.%20He&volume=11&publication_year=2001&pages=773-779&pmid=11740341&doi=10.1097/00008571-200112000-00005&)

81. Lee S.-J., Usmani K.A., Chanas B., Ghanayem B., Xi T., Hodgson E., Mohrenweiser H.W., Goldstein J.A. Genetic findings and functional studies of human CYP3A5 single nucleotide polymorphisms in different ethnic groups. Pharmacogenetics. 2003;13:461–472. doi: 10.1097/00008571-200308000-00004.  [DOI](https://doi.org/10.1097/00008571-200308000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12893984/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Genetic%20findings%20and%20functional%20studies%20of%20human%20CYP3A5%20single%20nucleotide%20polymorphisms%20in%20different%20ethnic%20groups&author=S.-J.%20Lee&author=K.A.%20Usmani&author=B.%20Chanas&author=B.%20Ghanayem&author=T.%20Xi&volume=13&publication_year=2003&pages=461-472&pmid=12893984&doi=10.1097/00008571-200308000-00004&)

82. Birdwell K.A., Decker B., Barbarino J.M., Peterson J.F., Stein C.M., Sadee W., Wang D., Vinks A.A., He Y., Swen J.J. Clinical Pharmacogenetics Implementation Consortium (CPIC) guidelines for CYP3A5 genotype and tacrolimus dosing. Clin. Pharm. Ther. 2015;98:19–24. doi: 10.1002/cpt.113.  [DOI](https://doi.org/10.1002/cpt.113) | [PMC free article](/articles/PMC4481158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25801146/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharm.%20Ther.&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guidelines%20for%20CYP3A5%20genotype%20and%20tacrolimus%20dosing&author=K.A.%20Birdwell&author=B.%20Decker&author=J.M.%20Barbarino&author=J.F.%20Peterson&author=C.M.%20Stein&volume=98&publication_year=2015&pages=19-24&pmid=25801146&doi=10.1002/cpt.113&)

83. Tsamandouras N., Dickinson G., Guo Y., Hall S., Rostami-Hodjegan A., Galetin A., Aarons L. Identification of the effect of multiple polymorphisms on the pharmacokinetics of simvastatin and simvastatin acid using a population-modeling approach. Clin. Pharmacol. Ther. 2014;96:90–100. doi: 10.1038/clpt.2014.55.  [DOI](https://doi.org/10.1038/clpt.2014.55) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24598718/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Identification%20of%20the%20effect%20of%20multiple%20polymorphisms%20on%20the%20pharmacokinetics%20of%20simvastatin%20and%20simvastatin%20acid%20using%20a%20population-modeling%20approach&author=N.%20Tsamandouras&author=G.%20Dickinson&author=Y.%20Guo&author=S.%20Hall&author=A.%20Rostami-Hodjegan&volume=96&publication_year=2014&pages=90-100&pmid=24598718&doi=10.1038/clpt.2014.55&)

84. Luzum J.A., Theusch E., Taylor K.D., Wang A., Sadee W., Binkley P.F., Krauss R.M., Medina M.W., Kitzmiller J.P. Individual and Combined Associations of Genetic Variants in CYP3A4, CYP3A5, and SLCO1B1 With Simvastatin and Simvastatin Acid Plasma Concentrations. J. Cardiovasc. Pharmacol. 2015;66:80–85. doi: 10.1097/FJC.0000000000000246.  [DOI](https://doi.org/10.1097/FJC.0000000000000246) | [PMC free article](/articles/PMC4503374/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26164721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cardiovasc.%20Pharmacol.&title=Individual%20and%20Combined%20Associations%20of%20Genetic%20Variants%20in%20CYP3A4,%20CYP3A5,%20and%20SLCO1B1%20With%20Simvastatin%20and%20Simvastatin%20Acid%20Plasma%20Concentrations&author=J.A.%20Luzum&author=E.%20Theusch&author=K.D.%20Taylor&author=A.%20Wang&author=W.%20Sadee&volume=66&publication_year=2015&pages=80-85&pmid=26164721&doi=10.1097/FJC.0000000000000246&)

85. Prueksaritanont T., Gorham L.M., Ma B., Liu L., Yu X., Zhao J.J., Slaughter D.E., Arison B.H., Vyas K.P. In vitro metabolism of simvastatin in humans [SBT] identification of metabolizing enzymes and effect of the drug on hepatic P450s. Drug Metab. Dispos. 1997;25:1191–1199.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9321523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=In%20vitro%20metabolism%20of%20simvastatin%20in%20humans%20%5BSBT%5D%20identification%20of%20metabolizing%20enzymes%20and%20effect%20of%20the%20drug%20on%20hepatic%20P450s&author=T.%20Prueksaritanont&author=L.M.%20Gorham&author=B.%20Ma&author=L.%20Liu&author=X.%20Yu&volume=25&publication_year=1997&pages=1191-1199&pmid=9321523&)

86. Westlind-Johnsson A., Malmebo S., Johansson A., Otter C., Andersson T.B., Johansson I., Edwards R.J., Boobis A.R., Ingelman-Sundberg M. Comparative analysis of CYP3A expression in human liver suggests only a minor role for CYP3A5 in drug metabolism. Drug Metab. Dispos. 2003;31:755–761. doi: 10.1124/dmd.31.6.755.  [DOI](https://doi.org/10.1124/dmd.31.6.755) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12756208/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Comparative%20analysis%20of%20CYP3A%20expression%20in%20human%20liver%20suggests%20only%20a%20minor%20role%20for%20CYP3A5%20in%20drug%20metabolism&author=A.%20Westlind-Johnsson&author=S.%20Malmebo&author=A.%20Johansson&author=C.%20Otter&author=T.B.%20Andersson&volume=31&publication_year=2003&pages=755-761&pmid=12756208&doi=10.1124/dmd.31.6.755&)

87. Wolbold R., Klein K., Burk O., Nüssler A.K., Neuhaus P., Eichelbaum M., Schwab M., Zanger U.M. Sex is a major determinant of CYP3A4 expression in human liver. Hepatology. 2003;38:978–988. doi: 10.1002/hep.1840380424.  [DOI](https://doi.org/10.1002/hep.1840380424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14512885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Sex%20is%20a%20major%20determinant%20of%20CYP3A4%20expression%20in%20human%20liver&author=R.%20Wolbold&author=K.%20Klein&author=O.%20Burk&author=A.K.%20N%C3%BCssler&author=P.%20Neuhaus&volume=38&publication_year=2003&pages=978-988&pmid=14512885&doi=10.1002/hep.1840380424&)

88. Krecic-Shepard M.E., Park K., Barnas C., Slimko J., Kerwin D.R., Schwartz J.B. Race and sex influence clearance of nifedipine: Results of a population study. Clin. Pharm. Ther. 2000;68:130–142. doi: 10.1067/mcp.2000.108678.  [DOI](https://doi.org/10.1067/mcp.2000.108678) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10976544/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharm.%20Ther.&title=Race%20and%20sex%20influence%20clearance%20of%20nifedipine:%20Results%20of%20a%20population%20study&author=M.E.%20Krecic-Shepard&author=K.%20Park&author=C.%20Barnas&author=J.%20Slimko&author=D.R.%20Kerwin&volume=68&publication_year=2000&pages=130-142&pmid=10976544&doi=10.1067/mcp.2000.108678&)

89. Andreassen T.N., Klepstad P., Davies A., Bjordal K., Lundström S., Kaasa S., Dale O. Influences on the pharmacokinetics of oxycodone: A multicentre cross-sectional study in 439 adult cancer patients. Eur. J. Clin. Pharmacol. 2011;67:493–506. doi: 10.1007/s00228-010-0948-5.  [DOI](https://doi.org/10.1007/s00228-010-0948-5) | [PMC free article](/articles/PMC3076582/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21140139/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=Influences%20on%20the%20pharmacokinetics%20of%20oxycodone:%20A%20multicentre%20cross-sectional%20study%20in%20439%20adult%20cancer%20patients&author=T.N.%20Andreassen&author=P.%20Klepstad&author=A.%20Davies&author=K.%20Bjordal&author=S.%20Lundstr%C3%B6m&volume=67&publication_year=2011&pages=493-506&pmid=21140139&doi=10.1007/s00228-010-0948-5&)

90. Kahan B.D., Kramer W.G., Wideman C., Flechner S.M., Lorber M.I., Van C.B. Demographic factors affecting the pharmacokinetics of cyclosporine estimated by radioimmunoassay. Transplantation. 1986;41:459–464. doi: 10.1097/00007890-198604000-00009.  [DOI](https://doi.org/10.1097/00007890-198604000-00009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3515645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplantation&title=Demographic%20factors%20affecting%20the%20pharmacokinetics%20of%20cyclosporine%20estimated%20by%20radioimmunoassay&author=B.D.%20Kahan&author=W.G.%20Kramer&author=C.%20Wideman&author=S.M.%20Flechner&author=M.I.%20Lorber&volume=41&publication_year=1986&pages=459-464&pmid=3515645&doi=10.1097/00007890-198604000-00009&)

91. Zhu B., Liu Z.-Q., Chen G.-L., Chen X.-P., Ou-Yang D.-S., Wang L.-S., Huang S.-L., Tan Z.-R., Zhou H.-H. The distribution and gender difference of CYP3A activity in Chinese subjects. Br. J. Clin. Pharm. 2003;55:264–269. doi: 10.1046/j.1365-2125.2003.01728.x.  [DOI](https://doi.org/10.1046/j.1365-2125.2003.01728.x) | [PMC free article](/articles/PMC1884221/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12630976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharm.&title=The%20distribution%20and%20gender%20difference%20of%20CYP3A%20activity%20in%20Chinese%20subjects&author=B.%20Zhu&author=Z.-Q.%20Liu&author=G.-L.%20Chen&author=X.-P.%20Chen&author=D.-S.%20Ou-Yang&volume=55&publication_year=2003&pages=264-269&pmid=12630976&doi=10.1046/j.1365-2125.2003.01728.x&)

92. Zhou S.-F. Drugs behave as substrates, inhibitors and inducers of human cytochrome P450 3A4. Curr. Drug Metab. 2008;9:310–322. doi: 10.2174/138920008784220664.  [DOI](https://doi.org/10.2174/138920008784220664) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18473749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Drug%20Metab.&title=Drugs%20behave%20as%20substrates,%20inhibitors%20and%20inducers%20of%20human%20cytochrome%20P450%203A4&author=S.-F.%20Zhou&volume=9&publication_year=2008&pages=310-322&pmid=18473749&doi=10.2174/138920008784220664&)

93. Zhou S., Chan S.Y., Goh B.C., Chan E., Duan W., Huang M., McLeod H.L. Mechanism-Based Inhibition of Cytochrome P450 3A4 by Therapeutic Drugs. Clin. Pharmacokinet. 2005;44:279–304. doi: 10.2165/00003088-200544030-00005.  [DOI](https://doi.org/10.2165/00003088-200544030-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15762770/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet.&title=Mechanism-Based%20Inhibition%20of%20Cytochrome%20P450%203A4%20by%20Therapeutic%20Drugs&author=S.%20Zhou&author=S.Y.%20Chan&author=B.C.%20Goh&author=E.%20Chan&author=W.%20Duan&volume=44&publication_year=2005&pages=279-304&pmid=15762770&doi=10.2165/00003088-200544030-00005&)

94. Saiz-Rodríguez M., Ochoa D., Belmonte C., Román M., Vieira de Lara D., Zubiaur P., Koller D., Mejía G., Abad-Santos F. Polymorphisms in CYP1A2, CYP2C9 and ABCB1 affect agomelatine pharmacokinetics. J. Psychopharmacol. (Oxf.) 2019;33:522–531. doi: 10.1177/0269881119827959.  [DOI](https://doi.org/10.1177/0269881119827959) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30789308/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Psychopharmacol.%20(Oxf.)&title=Polymorphisms%20in%20CYP1A2,%20CYP2C9%20and%20ABCB1%20affect%20agomelatine%20pharmacokinetics&author=M.%20Saiz-Rodr%C3%ADguez&author=D.%20Ochoa&author=C.%20Belmonte&author=M.%20Rom%C3%A1n&author=D.%20Vieira%20de%20Lara&volume=33&publication_year=2019&pages=522-531&pmid=30789308&doi=10.1177/0269881119827959&)

95. Ensembl Data CYP3A4 rs4986910. [(accessed on 25 February 2020)]; Available online: https://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99760401-99761401;v=rs4986910;vdb=variation;vf=17958388.  [https://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99760401-99761401;v=rs4986910;vdb=variation;vf=17958388](https://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99760401-99761401;v=rs4986910;vdb=variation;vf=17958388)

96. Ensembl Data CYP3A4 rs67666821. [(accessed on 25 February 2020)]; Available online: http://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99757684-99758683;v=rs67666821;vdb=variation;vf=12903884.  [http://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99757684-99758683;v=rs67666821;vdb=variation;vf=12903884](http://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99757684-99758683;v=rs67666821;vdb=variation;vf=12903884)

97. Ensembl Data CYP3A4 rs35599367. [(accessed on 25 February 2020)]; Available online: https://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99768193-99769193;v=rs35599367;vdb=variation;vf=20923380.  [https://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99768193-99769193;v=rs35599367;vdb=variation;vf=20923380](https://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99768193-99769193;v=rs35599367;vdb=variation;vf=20923380)

98. Ensembl Data CYP3A5 rs776746. [(accessed on 25 February 2020)]; Available online: http://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99672416-99673416;v=rs776746;vdb=variation;vf=550116.  [http://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99672416-99673416;v=rs776746;vdb=variation;vf=550116](http://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99672416-99673416;v=rs776746;vdb=variation;vf=550116)

99. Ensembl Data CYP3A5 rs10264272. [(accessed on 25 February 2020)]; Available online: https://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99664712-99665712;v=rs10264272;vdb=variation;vf=19096090.  [https://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99664712-99665712;v=rs10264272;vdb=variation;vf=19096090](https://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99664712-99665712;v=rs10264272;vdb=variation;vf=19096090)

100. Ensembl Data CYP3A5 rs41303343. [(accessed on 25 February 2020)]; Available online: https://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99652271-99653271;v=rs41303343;vdb=variation;vf=21091103.  [https://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99652271-99653271;v=rs41303343;vdb=variation;vf=21091103](https://www.ensembl.org/Homo_sapiens/Variation/Population?db=core;r=7:99652271-99653271;v=rs41303343;vdb=variation;vf=21091103)

101. Belmonte C., Ochoa D., Román M., Saiz-Rodríguez M., Wojnicz A., Gómez-Sánchez C.I., Martín-Vílchez S., Abad-Santos F. Influence of CYP2D6, CYP3A4, CYP3A5 and ABCB1 Polymorphisms on Pharmacokinetics and Safety of Aripiprazole in Healthy Volunteers. Basic Clin. Pharmacol. Toxicol. 2018;122:596–605. doi: 10.1111/bcpt.12960.  [DOI](https://doi.org/10.1111/bcpt.12960) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29325225/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin.%20Pharmacol.%20Toxicol.&title=Influence%20of%20CYP2D6,%20CYP3A4,%20CYP3A5%20and%20ABCB1%20Polymorphisms%20on%20Pharmacokinetics%20and%20Safety%20of%20Aripiprazole%20in%20Healthy%20Volunteers&author=C.%20Belmonte&author=D.%20Ochoa&author=M.%20Rom%C3%A1n&author=M.%20Saiz-Rodr%C3%ADguez&author=A.%20Wojnicz&volume=122&publication_year=2018&pages=596-605&pmid=29325225&doi=10.1111/bcpt.12960&)

102. Saiz-Rodríguez M., Ochoa D., Herrador C., Belmonte C., Román M., Alday E., Koller D., Zubiaur P., Mejía G., Hernández-Martínez M., et al. Polymorphisms associated with fentanyl pharmacokinetics, pharmacodynamics and adverse effects. Basic Clin. Pharmacol. Toxicol. 2019;124:321–329. doi: 10.1111/bcpt.13141.  [DOI](https://doi.org/10.1111/bcpt.13141) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30281924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin.%20Pharmacol.%20Toxicol.&title=Polymorphisms%20associated%20with%20fentanyl%20pharmacokinetics,%20pharmacodynamics%20and%20adverse%20effects&author=M.%20Saiz-Rodr%C3%ADguez&author=D.%20Ochoa&author=C.%20Herrador&author=C.%20Belmonte&author=M.%20Rom%C3%A1n&volume=124&publication_year=2019&pages=321-329&pmid=30281924&doi=10.1111/bcpt.13141&)

103. Kuip E.J.M., Zandvliet M.L., Koolen S.L.W., Mathijssen R.H.J., van der Rijt C.C.D. A review of factors explaining variability in fentanyl pharmacokinetics; focus on implications for cancer patients. Br. J. Clin. Pharmacol. 2017;83:294–313. doi: 10.1111/bcp.13129.  [DOI](https://doi.org/10.1111/bcp.13129) | [PMC free article](/articles/PMC5237702/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27619152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=A%20review%20of%20factors%20explaining%20variability%20in%20fentanyl%20pharmacokinetics;%20focus%20on%20implications%20for%20cancer%20patients&author=E.J.M.%20Kuip&author=M.L.%20Zandvliet&author=S.L.W.%20Koolen&author=R.H.J.%20Mathijssen&author=C.C.D.%20van%20der%20Rijt&volume=83&publication_year=2017&pages=294-313&pmid=27619152&doi=10.1111/bcp.13129&)

104. Takashina Y., Naito T., Mino Y., Yagi T., Ohnishi K., Kawakami J. Impact of CYP3A5 and ABCB1 gene polymorphisms on fentanyl pharmacokinetics and clinical responses in cancer patients undergoing conversion to a transdermal system. Drug Metab. Pharmacokinet. 2012;27:414–421. doi: 10.2133/dmpk.dmpk-11-rg-134.  [DOI](https://doi.org/10.2133/dmpk.dmpk-11-rg-134) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22277678/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Pharmacokinet.&title=Impact%20of%20CYP3A5%20and%20ABCB1%20gene%20polymorphisms%20on%20fentanyl%20pharmacokinetics%20and%20clinical%20responses%20in%20cancer%20patients%20undergoing%20conversion%20to%20a%20transdermal%20system&author=Y.%20Takashina&author=T.%20Naito&author=Y.%20Mino&author=T.%20Yagi&author=K.%20Ohnishi&volume=27&publication_year=2012&pages=414-421&pmid=22277678&doi=10.2133/dmpk.dmpk-11-rg-134&)

105. Drogari E., Ragia G., Mollaki V., Elens L., Van Schaik R.H., Manolopoulos V.G. POR*28 SNP is associated with lipid response to atorvastatin in children and adolescents with familial hypercholesterolemia. Pharmacogenomics. 2014;15:1963–1972. doi: 10.2217/pgs.14.138.  [DOI](https://doi.org/10.2217/pgs.14.138) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25521355/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=POR*28%20SNP%20is%20associated%20with%20lipid%20response%20to%20atorvastatin%20in%20children%20and%20adolescents%20with%20familial%20hypercholesterolemia&author=E.%20Drogari&author=G.%20Ragia&author=V.%20Mollaki&author=L.%20Elens&author=R.H.%20Van%20Schaik&volume=15&publication_year=2014&pages=1963-1972&pmid=25521355&doi=10.2217/pgs.14.138&)

106. Ragia G., Kolovou V., Tavridou A., Elens L., Tselepis A.D., Elisaf M., Van Schaik R.H.N., Kolovou G., Manolopoulos V.G. No effect of CYP3A4 intron 6 C>T polymorphism (CYP3A4*22) on lipid-lowering response to statins in Greek patients with primary hypercholesterolemia. [(accessed on 25 February 2020)];Drug Metab. Personal. Ther. 2015 30 doi: 10.1515/dmdi-2014-0021. Available online: https://www.degruyter.com/view/j/dmdi.2015.30.issue-1/dmdi-2014-0021/dmdi-2014-0021.xml.  [https://www.degruyter.com/view/j/dmdi.2015.30.issue-1/dmdi-2014-0021/dmdi-2014-0021.xml](https://www.degruyter.com/view/j/dmdi.2015.30.issue-1/dmdi-2014-0021/dmdi-2014-0021.xml) | [DOI](https://doi.org/10.1515/dmdi-2014-0021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25274942/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Personal.%20Ther.&title=No%20effect%20of%20CYP3A4%20intron%206%20C>T%20polymorphism%20(CYP3A4*22)%20on%20lipid-lowering%20response%20to%20statins%20in%20Greek%20patients%20with%20primary%20hypercholesterolemia&author=G.%20Ragia&author=V.%20Kolovou&author=A.%20Tavridou&author=L.%20Elens&author=A.D.%20Tselepis&volume=30&publication_year=2015&pmid=25274942&doi=10.1515/dmdi-2014-0021&)

107. Rosales A., Alvear M., Cuevas A., Saavedra N., Zambrano T., Salazar L.A. Identification of pharmacogenetic predictors of lipid-lowering response to atorvastatin in Chilean subjects with hypercholesterolemia. Clin. Chim. Acta. 2012;413:495–501. doi: 10.1016/j.cca.2011.11.003.  [DOI](https://doi.org/10.1016/j.cca.2011.11.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22120734/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Chim.%20Acta&title=Identification%20of%20pharmacogenetic%20predictors%20of%20lipid-lowering%20response%20to%20atorvastatin%20in%20Chilean%20subjects%20with%20hypercholesterolemia&author=A.%20Rosales&author=M.%20Alvear&author=A.%20Cuevas&author=N.%20Saavedra&author=T.%20Zambrano&volume=413&publication_year=2012&pages=495-501&pmid=22120734&doi=10.1016/j.cca.2011.11.003&)

108. Peng C., Ding Y., Yi X., Shen Y., Dong Z., Cao L., Li Q., Ren H., He L., Zhou D., et al. Polymorphisms in CYP450 Genes and the Therapeutic Effect of Atorvastatin on Ischemic Stroke: A Retrospective Cohort Study in Chinese Population. Clin. Ther. 2018;40:469–477. doi: 10.1016/j.clinthera.2018.02.002.  [DOI](https://doi.org/10.1016/j.clinthera.2018.02.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29500141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Ther.&title=Polymorphisms%20in%20CYP450%20Genes%20and%20the%20Therapeutic%20Effect%20of%20Atorvastatin%20on%20Ischemic%20Stroke:%20A%20Retrospective%20Cohort%20Study%20in%20Chinese%20Population&author=C.%20Peng&author=Y.%20Ding&author=X.%20Yi&author=Y.%20Shen&author=Z.%20Dong&volume=40&publication_year=2018&pages=469-477&pmid=29500141&doi=10.1016/j.clinthera.2018.02.002&)

109. Liu H., Xu Q., Huang W., Zhao Q., Jiang Z., Kuang X., Li Z., Sun H., Qiu X. CYP3A5 and CYP3A7 genetic polymorphisms affect tacrolimus concentration in pediatric patients with nephrotic range proteinuria. Eur. J. Clin. Pharmacol. 2019;75:1533–1540. doi: 10.1007/s00228-019-02726-w.  [DOI](https://doi.org/10.1007/s00228-019-02726-w) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31401678/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=CYP3A5%20and%20CYP3A7%20genetic%20polymorphisms%20affect%20tacrolimus%20concentration%20in%20pediatric%20patients%20with%20nephrotic%20range%20proteinuria&author=H.%20Liu&author=Q.%20Xu&author=W.%20Huang&author=Q.%20Zhao&author=Z.%20Jiang&volume=75&publication_year=2019&pages=1533-1540&pmid=31401678&doi=10.1007/s00228-019-02726-w&)

110. Qiu H., Zhuang W., Wang X., Huang M., Zhou Z. Association between genetic polymorphisms and variation of imatinib pharmacokinetics in gastrointestinal stromal tumors. Zhonghua Wei Chang Wai Ke Za Zhi. 2017;20:1031–1034.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/28900995/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Zhonghua%20Wei%20Chang%20Wai%20Ke%20Za%20Zhi&title=Association%20between%20genetic%20polymorphisms%20and%20variation%20of%20imatinib%20pharmacokinetics%20in%20gastrointestinal%20stromal%20tumors&author=H.%20Qiu&author=W.%20Zhuang&author=X.%20Wang&author=M.%20Huang&author=Z.%20Zhou&volume=20&publication_year=2017&pages=1031-1034&pmid=28900995&)

111. Qian Y., Sun L.-N., Liu Y.-J., Zhang Q., Xu J.-H., Ma Z.-Q., Zhang X.-H., Xu H., Wang Y.-Q. Genetic Polymorphisms and Adverse Events on Unbound Imatinib and Its Active Metabolite Concentration in Patients With Gastrointestinal Stromal Tumors. Front. Pharmacol. 2019;10:854. doi: 10.3389/fphar.2019.00854.  [DOI](https://doi.org/10.3389/fphar.2019.00854) | [PMC free article](/articles/PMC6682687/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31417408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Genetic%20Polymorphisms%20and%20Adverse%20Events%20on%20Unbound%20Imatinib%20and%20Its%20Active%20Metabolite%20Concentration%20in%20Patients%20With%20Gastrointestinal%20Stromal%20Tumors&author=Y.%20Qian&author=L.-N.%20Sun&author=Y.-J.%20Liu&author=Q.%20Zhang&author=J.-H.%20Xu&volume=10&publication_year=2019&pages=854&pmid=31417408&doi=10.3389/fphar.2019.00854&)

112. Ensembl Data CYP3A4 rs755828176. [(accessed on 17 February 2020)]; Available online: https://www.ensembl.org/Homo_sapiens/Variation/Explore?r=6:160139351-160140352;v=rs755828176;vdb=variation;vf=213359709.  [https://www.ensembl.org/Homo_sapiens/Variation/Explore?r=6:160139351-160140352;v=rs755828176;vdb=variation;vf=213359709](https://www.ensembl.org/Homo_sapiens/Variation/Explore?r=6:160139351-160140352;v=rs755828176;vdb=variation;vf=213359709)

113. Belohlavkova P., Vrbacky F., Voglova J., Racil Z., Zackova D., Hrochova K., Malakova J., Mayer J., Zak P. The significance of enzyme and transporter polymorphisms for imatinib plasma levels and achieving an optimal response in chronic myeloid leukemia patients. Aoms. 2018;14:1416–1423. doi: 10.5114/aoms.2018.73538.  [DOI](https://doi.org/10.5114/aoms.2018.73538) | [PMC free article](/articles/PMC6209720/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30393497/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aoms&title=The%20significance%20of%20enzyme%20and%20transporter%20polymorphisms%20for%20imatinib%20plasma%20levels%20and%20achieving%20an%20optimal%20response%20in%20chronic%20myeloid%20leukemia%20patients&author=P.%20Belohlavkova&author=F.%20Vrbacky&author=J.%20Voglova&author=Z.%20Racil&author=D.%20Zackova&volume=14&publication_year=2018&pages=1416-1423&pmid=30393497&doi=10.5114/aoms.2018.73538&)

114. Seong S.J., Lim M., Sohn S.K., Moon J.H., Oh S.-J., Kim B.S., Ryoo H.M., Chung J.S., Joo Y.D., Bang S.M., et al. Influence of enzyme and transporter polymorphisms on trough imatinib concentration and clinical response in chronic myeloid leukemia patients. Ann. Oncol. 2013;24:756–760. doi: 10.1093/annonc/mds532.  [DOI](https://doi.org/10.1093/annonc/mds532) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23117072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol.&title=Influence%20of%20enzyme%20and%20transporter%20polymorphisms%20on%20trough%20imatinib%20concentration%20and%20clinical%20response%20in%20chronic%20myeloid%20leukemia%20patients&author=S.J.%20Seong&author=M.%20Lim&author=S.K.%20Sohn&author=J.H.%20Moon&author=S.-J.%20Oh&volume=24&publication_year=2013&pages=756-760&pmid=23117072&doi=10.1093/annonc/mds532&)

115. Petain A., Kattygnarath D., Azard J., Chatelut E., Delbaldo C., Geoerger B., Barrois M., Séronie-Vivien S., LeCesne A., Vassal G., et al. Population pharmacokinetics and pharmacogenetics of imatinib in children and adults. Clin. Cancer Res. 2008;14:7102–7109. doi: 10.1158/1078-0432.CCR-08-0950.  [DOI](https://doi.org/10.1158/1078-0432.CCR-08-0950) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18981009/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Population%20pharmacokinetics%20and%20pharmacogenetics%20of%20imatinib%20in%20children%20and%20adults&author=A.%20Petain&author=D.%20Kattygnarath&author=J.%20Azard&author=E.%20Chatelut&author=C.%20Delbaldo&volume=14&publication_year=2008&pages=7102-7109&pmid=18981009&doi=10.1158/1078-0432.CCR-08-0950&)

116. Adeagbo B.A., Bolaji O.O., Olugbade T.A., Durosinmi M.A., Bolarinwa R.A., Masimirembwa C. Influence of CYP3A5*3 and ABCB1 C3435T on clinical outcomes and trough plasma concentrations of imatinib in Nigerians with chronic myeloid leukaemia. J. Clin. Pharm. Ther. 2016;41:546–551. doi: 10.1111/jcpt.12424.  [DOI](https://doi.org/10.1111/jcpt.12424) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27426203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharm.%20Ther.&title=Influence%20of%20CYP3A5*3%20and%20ABCB1%20C3435T%20on%20clinical%20outcomes%20and%20trough%20plasma%20concentrations%20of%20imatinib%20in%20Nigerians%20with%20chronic%20myeloid%20leukaemia&author=B.A.%20Adeagbo&author=O.O.%20Bolaji&author=T.A.%20Olugbade&author=M.A.%20Durosinmi&author=R.A.%20Bolarinwa&volume=41&publication_year=2016&pages=546-551&pmid=27426203&doi=10.1111/jcpt.12424&)

117. Maddin N., Husin A., Gan S.H., Aziz B.A., Ankathil R. Impact of CYP3A4*18 and CYP3A5*3 Polymorphisms on Imatinib Mesylate Response Among Chronic Myeloid Leukemia Patients in Malaysia. Oncol. Ther. 2016;4:303–314. doi: 10.1007/s40487-016-0035-x.  [DOI](https://doi.org/10.1007/s40487-016-0035-x) | [PMC free article](/articles/PMC5315081/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28261657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncol.%20Ther.&title=Impact%20of%20CYP3A4*18%20and%20CYP3A5*3%20Polymorphisms%20on%20Imatinib%20Mesylate%20Response%20Among%20Chronic%20Myeloid%20Leukemia%20Patients%20in%20Malaysia&author=N.%20Maddin&author=A.%20Husin&author=S.H.%20Gan&author=B.A.%20Aziz&author=R.%20Ankathil&volume=4&publication_year=2016&pages=303-314&pmid=28261657&doi=10.1007/s40487-016-0035-x&)

118. Huang Y., Wen G., Lu Y., Wen J., Ji Y., Xing X., Li Y., Wen J., Yuan H. CYP3A4*1G and CYP3A5*3 genetic polymorphisms alter the antihypertensive efficacy of amlodipine in patients with hypertension following renal transplantation. CP. 2017;55:109–118. doi: 10.5414/CP202559.  [DOI](https://doi.org/10.5414/CP202559) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27841150/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CP&title=CYP3A4*1G%20and%20CYP3A5*3%20genetic%20polymorphisms%20alter%20the%20antihypertensive%20efficacy%20of%20amlodipine%20in%20patients%20with%20hypertension%20following%20renal%20transplantation&author=Y.%20Huang&author=G.%20Wen&author=Y.%20Lu&author=J.%20Wen&author=Y.%20Ji&volume=55&publication_year=2017&pages=109-118&pmid=27841150&doi=10.5414/CP202559&)

119. Kim K., Park P., Lee O., Choi S., Min B., Shin K., Chun B., Shin J., Park J. Effect of CYP3A53 genotype on the pharmacokinetics and pharmacodynamics of amlodipine in healthy Korean subjects. Clin. Pharm. Ther. 2006;80:646–656. doi: 10.1016/j.clpt.2006.09.009.  [DOI](https://doi.org/10.1016/j.clpt.2006.09.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17178265/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharm.%20Ther.&title=Effect%20of%20CYP3A53%20genotype%20on%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20amlodipine%20in%20healthy%20Korean%20subjects&author=K.%20Kim&author=P.%20Park&author=O.%20Lee&author=S.%20Choi&author=B.%20Min&volume=80&publication_year=2006&pages=646-656&pmid=17178265&doi=10.1016/j.clpt.2006.09.009&)

120. Cabaleiro T., López-Rodríguez R., Román M., Ochoa D., Novalbos J., Borobia A., Carcas A., Abad-Santos F. Pharmacogenetics of quetiapine in healthy volunteers: Association with pharmacokinetics, pharmacodynamics, and adverse effects. Int. Clin. Psychopharmacol. 2015;30:82–88. doi: 10.1097/YIC.0000000000000047.  [DOI](https://doi.org/10.1097/YIC.0000000000000047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25025989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20Clin.%20Psychopharmacol.&title=Pharmacogenetics%20of%20quetiapine%20in%20healthy%20volunteers:%20Association%20with%20pharmacokinetics,%20pharmacodynamics,%20and%20adverse%20effects&author=T.%20Cabaleiro&author=R.%20L%C3%B3pez-Rodr%C3%ADguez&author=M.%20Rom%C3%A1n&author=D.%20Ochoa&author=J.%20Novalbos&volume=30&publication_year=2015&pages=82-88&pmid=25025989&doi=10.1097/YIC.0000000000000047&)

121. Shilbayeh S.A.R., Sy S.K.B., Melhem M., Zmeili R., Derendorf H. Quantitation of the impact of CYP3A5 A6986G polymorphism on quetiapine pharmacokinetics by simulation of target attainment: Clinical Pharmacology in Drug Development. Clin. Pharm. Drug Dev. 2015;4:387–394. doi: 10.1002/cpdd.172.  [DOI](https://doi.org/10.1002/cpdd.172) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27137148/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharm.%20Drug%20Dev.&title=Quantitation%20of%20the%20impact%20of%20CYP3A5%20A6986G%20polymorphism%20on%20quetiapine%20pharmacokinetics%20by%20simulation%20of%20target%20attainment:%20Clinical%20Pharmacology%20in%20Drug%20Development&author=S.A.R.%20Shilbayeh&author=S.K.B.%20Sy&author=M.%20Melhem&author=R.%20Zmeili&author=H.%20Derendorf&volume=4&publication_year=2015&pages=387-394&pmid=27137148&doi=10.1002/cpdd.172&)

122. Bakken G.V., Molden E., Hermann M. Impact of Genetic Variability in CYP2D6, CYP3A5, and ABCB1 on Serum Concentrations of Quetiapine and N-desalkylquetiapine in Psychiatric Patients. Ther. Drug Monit. 2015;37:256–261. doi: 10.1097/FTD.0000000000000135.  [DOI](https://doi.org/10.1097/FTD.0000000000000135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25254417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther.%20Drug%20Monit.&title=Impact%20of%20Genetic%20Variability%20in%20CYP2D6,%20CYP3A5,%20and%20ABCB1%20on%20Serum%20Concentrations%20of%20Quetiapine%20and%20N-desalkylquetiapine%20in%20Psychiatric%20Patients&author=G.V.%20Bakken&author=E.%20Molden&author=M.%20Hermann&volume=37&publication_year=2015&pages=256-261&pmid=25254417&doi=10.1097/FTD.0000000000000135&)

123. Noetzli M., Guidi M., Ebbing K., Eyer S., Wilhelm L., Michon A., Thomazic V., Stancu I., Alnawaqil A.-M., Bula C., et al. Population pharmacokinetic approach to evaluate the effect of CYP2D6, CYP3A, ABCB1, POR and NR1I2 genotypes on donepezil clearance: Pharmacogenetic study on donepezil. Br. J. Clin. Pharmacol. 2014;78:135–144. doi: 10.1111/bcp.12325.  [DOI](https://doi.org/10.1111/bcp.12325) | [PMC free article](/articles/PMC4168388/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24433464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Population%20pharmacokinetic%20approach%20to%20evaluate%20the%20effect%20of%20CYP2D6,%20CYP3A,%20ABCB1,%20POR%20and%20NR1I2%20genotypes%20on%20donepezil%20clearance:%20Pharmacogenetic%20study%20on%20donepezil&author=M.%20Noetzli&author=M.%20Guidi&author=K.%20Ebbing&author=S.%20Eyer&author=L.%20Wilhelm&volume=78&publication_year=2014&pages=135-144&pmid=24433464&doi=10.1111/bcp.12325&)

124. Magliulo L., Dahl M.-L., Lombardi G., Fallarini S., Villa L.M., Biolcati A., Scordo M.G. Do CYP3A and ABCB1 genotypes influence the plasma concentration and clinical outcome of donepezil treatment? Eur. J. Clin. Pharmacol. 2011;67:47–54. doi: 10.1007/s00228-010-0883-5.  [DOI](https://doi.org/10.1007/s00228-010-0883-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20931330/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=Do%20CYP3A%20and%20ABCB1%20genotypes%20influence%20the%20plasma%20concentration%20and%20clinical%20outcome%20of%20donepezil%20treatment?&author=L.%20Magliulo&author=M.-L.%20Dahl&author=G.%20Lombardi&author=S.%20Fallarini&author=L.M.%20Villa&volume=67&publication_year=2011&pages=47-54&pmid=20931330&doi=10.1007/s00228-010-0883-5&)

125. Matsumoto J., San S.N., Fujiyoshi M., Kawauchi A., Chiba N., Tagai R., Sanbe R., Yanaka S., Sakaue H., Kato Y., et al. Effect of CYP3A5*3 genetic variant on the metabolism of direct-acting antivirals in vitro: A different effect on asunaprevir versus daclatasvir and beclabuvir. J. Hum. Genet. 2020;65:143–153. doi: 10.1038/s10038-019-0685-2.  [DOI](https://doi.org/10.1038/s10038-019-0685-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31645655/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hum.%20Genet.&title=Effect%20of%20CYP3A5*3%20genetic%20variant%20on%20the%20metabolism%20of%20direct-acting%20antivirals%20in%20vitro:%20A%20different%20effect%20on%20asunaprevir%20versus%20daclatasvir%20and%20beclabuvir&author=J.%20Matsumoto&author=S.N.%20San&author=M.%20Fujiyoshi&author=A.%20Kawauchi&author=N.%20Chiba&volume=65&publication_year=2020&pages=143-153&pmid=31645655&doi=10.1038/s10038-019-0685-2&)
