# Genome-wide Study Identifies Association between HLA-B∗55:01 and Self-Reported Penicillin Allergy

## Metadata
**Authors:** Kristi Krebs, Jonas Bovijn, Neil Zheng, Maarja Lepamets, Jenny C Censin, Tuuli Jürgenson, Dage Särg, Erik Abner, Triin Laisk, Yang Luo, Line Skotte, Frank Geller, Bjarke Feenstra, Wei Wang, Adam Auton, 23andMe Research Team, Soumya Raychaudhuri, Tõnu Esko, Andres Metspalu, Sven Laur, Dan M Roden, Wei-Qi Wei, Michael V Holmes, Cecilia M Lindgren, Elizabeth J Phillips, Reedik Mägi, Lili Milani, João Fadista
**Journal:** American Journal of Human Genetics
**Date:** 2020 Sep 3
**DOI:** [10.1016/j.ajhg.2020.08.008](https://doi.org/10.1016/j.ajhg.2020.08.008)
**PMID:** 32888428
**PMCID:** PMC7536643
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7536643/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7536643/pdf/main.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7536643/pdf/main.pdf)

## Abstract

Hypersensitivity reactions to drugs are often unpredictable and can be life threatening, underscoring a need for understanding their underlying mechanisms and risk factors. The extent to which germline genetic variation influences the risk of commonly reported drug allergies such as penicillin allergy remains largely unknown. We extracted data from the electronic health records of more than 600,000 participants from the UK, Estonian, and Vanderbilt University Medical Center’s BioVU biobanks to study the role of genetic variation in the occurrence of self-reported penicillin hypersensitivity reactions. We used imputed SNP to HLA typing data from these cohorts to further fine map the human leukocyte antigen (HLA) association and replicated our results in 23andMe’s research cohort involving a total of 1.12 million individuals. Genome-wide meta-analysis of penicillin allergy revealed two loci, including one located in the HLA region on chromosome 6. This signal was further fine-mapped to the HLA-B∗55:01 allele (OR 1.41 95% CI 1.33–1.49, p value 2.04 × 10−31) and confirmed by independent replication in 23andMe’s research cohort (OR 1.30 95% CI 1.25–1.34, p value 1.00 × 10−47). The lead SNP was also associated with lower lymphocyte counts and in silico follow-up suggests a potential effect on T-lymphocytes at HLA-B∗55:01. We also observed a significant hit in PTPN22 and the GWAS results correlated with the genetics of rheumatoid arthritis and psoriasis. We present robust evidence for the role of an allele of the major histocompatibility complex (MHC) I gene HLA-B in the occurrence of penicillin allergy.

Keywords: GWAS, penicillin allergy, HLA-B∗55:01, PTPN22, EHR, pharmacogenomics, UKBB, EstBB, BioVu, 23andMe

## Introduction

Adverse drug reactions (ADRs) are common in clinical practice and are associated with high morbidity and mortality. A meta-analysis of prospective studies in the US revealed the incidence of serious ADRs to be 6.7% among hospitalized patients and the cause of more than 100,000 deaths annually.[1](#bib1) In Europe, ADRs are responsible for 3.5% of all hospital admissions, with 10.1% of patients experiencing ADRs during hospitalization and 197,000 fatal cases per year.[2](#bib2)^,^[3](#bib3) In the US, the cost of a single ADR event falls between 1,439 to 13,462 USD.[4](#bib4)

ADRs are typically divided into two types of reactions. Type A reactions are more predictable and related to the pharmacological action of a drug, whereas type B reactions are idiosyncratic, less predictable, largely dose independent, and typically driven by hypersensitivity reactions involving the immune system.[5](#bib5) Although type B reactions are less frequent (<20%) than type A reactions, they tend to be more severe and more often lead to the withdrawal of a drug from the market.[6](#bib6) One of the most common causes of type B reactions are antibiotics,[5](#bib5) typically from the beta-lactam class, with the prevalence of penicillin allergy estimated to be as high as 25% in some settings.[7](#bib7)^,^[8](#bib8) Despite the relative frequency of such reactions, there are very few studies of the genetic determinants of penicillin allergy.[9](#bib9)^,^[10](#bib10) This underscores the need for a better understanding of the mechanisms and risk factors, including the role of genetic variation, that contribute to these reactions.

The increasing availability of genetic and phenotypic data in large biobanks provides an opportune means for investigating the role of genetic variation in drug-induced hypersensitivity reactions. In the present study, we sought to identify genetic risk factors underlying penicillin-induced hypersensitivity reactions by harnessing data from the Estonian Biobank (EstBB), UK Biobank (UKBB), and Vanderbilt University Medical Center’s (VUMC) DNA Biobank (BioVU), with further replication in the 23andMe research cohort.

## Subjects and Methods

### Study Subjects and Phenotype Definitions

We studied individual-level genotypic and phenotypic data of 52,000 participants from the Estonian Biobank (EstBB), 500,000 participants from UK Biobank (UKBB), and a subset of 67,323 individuals from BioVU, the VUMC biorepository linked to de-identified electronic health records with self-reported European ancestry.[11](#bib11) EstBB, UKBB, and BioVU are population- or hospital-based cohorts, providing a rich variety of phenotypic and health-related information collected for each participant. All participants have signed a consent form to allow follow-up linkage. In UKBB and EstBB we extracted information on penicillin allergy by searching the records of the participants for the Z88.0 ICD10 code indicating patient-reported allergy status to penicillin. Information on phenotypic features like age and gender were obtained from the biobank recruitment records. We also extracted likely penicillin allergies in EstBB from the recruitment questionnaires and free text fields of the electronic health records (EHRs) using a rule-based approach (see [Supplemental Subjects and Methods](#mmc1) for further details). In BioVU there were no records of Z88.0 diagnoses, so we used drug allergy labels from the allergy section of the EHRs, which includes adverse drug reactions reported by an individual or observed by the health care provider ([Supplemental Subjects and Methods](#mmc1)).

This study was approved by the Research Ethics Committee of the University of Tartu (Approval number 288/M-18) and conducted using the UK Biobank Resource under Application Number 11867.

### Genome-wide Study and Meta-analysis

The details on genotyping, quality control, and imputation are fully described elsewhere for EstBB[12](#bib12)^,^[13](#bib13) and UKBB;[14](#bib14) see [Supplemental Subjects and Methods](#mmc1) for further details. In EstBB, we conducted the penicillin GWAS on 44,348 individuals, including 1,320 case subjects with self-reported allergy to beta-lactam drugs or penicillin and 43,028 control subjects. In the UKBB, GWAS on penicillin allergy (defined using ICD-10 code Z88.0) was performed among 15,782 case subjects and 370,782 control subjects. In BioVU, GWAS on penicillin allergy (defined using drug allergy labels in the EHR) was performed among 12,294 case subjects and 38,284 control subjects. For all three cohorts, the GWAS was performed with SAIGE[15](#bib15) including related individuals and adjusting for the first ten principal components (PCs) of the genotype matrix, as well as for age or birth year, sex (see [Supplemental Subjects and Methods](#mmc1)), and in BioVU, additionally for EHR length (years). We performed meta-analysis of 19,724,685 markers (with minor allele frequency [MAF] > 0.1%) and SNP effect estimates and their standard errors were combined in a fixed effects model with the inverse variance weighted method using the METAL software.[16](#bib16) Results were visualized with the R software (3.3.2) (see [Web Resources](#app4)).

### HLA-Typing

HLA imputation of the EstBB genotype data was performed at the Broad Institute using the SNP2HLA tool.[17](#bib17) The imputation was done for genotype data generated on the Global Screening Array v1, and after quality control the four-digit HLA alleles of 22,554 individuals were used for analysis. In UKBB we used four-digit imputed HLA data released by UKBB (see [Web Resources](#app4)).[14](#bib14) The imputation process, performed using HLA^∗^IMP:02,[18](#bib18) is described fully elsewhere[14](#bib14) and in the [Supplemental Subjects and Methods](#mmc1). For the BioVU cohort, four-digit HLA-typing was imputed from SNP data with the SNP2HLA tool ([Supplemental Subjects and Methods](#mmc1)).

We performed separate additive logistic regression analysis with the called HLA alleles using R *glm* function in EstBB, UKBB, and BioVU (see [Supplemental Subjects and Methods](#mmc1) for further details). Meta-analysis of 164 HLA alleles present in all three cohorts was performed with the GWAMA software tool.[19](#bib19) A Bonferroni-corrected p value threshold of 3.05 × 10^−4^ was applied based on the number of tested alleles (0.05/164).

For detection of the strongest tagging SNP for the HLA-B^∗^55:01 allele, we calculated Pearson correlation coefficients between the HLA-B^∗^55:01 allele and all the SNPs within ±50 kb of the *HLA-B* region using the *cor* function in R (3.3.2) (see [Web Resources](#app4)).

### HLA-B∗55:01 Replication

We performed replication analysis of the HLA-B^∗^55:01 allele in 87,996 case subjects and 1,031,087 control subjects of European ancestry (close relatives removed) from the 23andMe research cohort using an additive logistic regression model (see details in the [Supplemental Subjects and Methods](#mmc1)). The self-reported phenotype of penicillin allergy was defined based on questionnaire data as a positive allergy test or allergic symptoms related to penicillin exposure (see [Supplemental Subjects and Methods](#mmc1) for further details). Meta-analysis of the HLA-B^∗^55:01 association across the four cohorts was performed with the GWAMA software tool[19](#bib19) and results were visualized with R software (3.3.2) (see [Web Resources](#app4)).

## Results

### Genome-wide Association Analysis of Penicillin Allergy

To discover genetic factors that may predispose to penicillin allergy, we conducted a genome-wide association study (GWAS) of 19.7 million single-nucleotide polymorphisms (SNPs) and insertions/deletions in UKBB, EstBB, and BioVu (MAF in all cohorts > 0.1%) among individuals with European ancestry. Case subjects were defined as participants with a Z88.0 ICD10 code (“Allergy status to penicillin”), which indicates a reported history of penicillin allergy (previously ICD9 “personal history of allergy to penicillin”). In total, we identified 15,782 individuals (4.1% of the total cohort size of 386,564) in UKBB with this diagnostic code. However, the corresponding number of case subjects in EstBB was only 7 (0.01% of the total cohort size of 51,936) and zero in BioVu, suggesting heterogeneity in the use of the Z88.0 ICD10 code in different countries. We therefore also identified participants that had reported drug allergy at recruitment in EstBB and categorized the EstBB self-reported reactions by drug class, using the Anatomical Therapeutic Chemical (ATC) Classification System code J01C^∗^ (beta-lactam antibacterials, penicillins) to match this to the respective Z88.0 ICD10 code. We also extracted 321 individuals with mentions of penicillin allergy in the free text fields of their EHR. This resulted in 1,320 (2.5%) case subjects with penicillin allergy in EstBB. We validated the approach in EstBB by evaluating the association between the number of filled (i.e., prescribed and purchased) penicillin (using the ATC code J01C^∗^) prescriptions per person and self-reported penicillin allergy. Using Poisson regression analysis, we identified a negative association among individuals with self-reported allergy in EstBB on the number of filled penicillin prescriptions (p value 2.41 × 10^−15^, estimate −0.18, i.e., 16% lower penicillin prescription count for individuals with penicillin allergy). In BioVU, we used drug allergy labels from the allergy section of the EHR to identify 12,294 case subjects (18.3% of the total cohort of genotyped individuals of 67,323), which is consistent with previous penicillin allergy reports using drug allergy labels.[20](#bib20) To characterize the proportion of severe reactions (anaphylaxis) to penicillin among our phenotype, we analyzed the self-reported reactions among 1,017 individuals in EstBB and found that around 3% (n = 31) of the participants reported anaphylaxis and 4% (n = 42) some form of breathing difficulties. In the BioVU cohort, 5% of participants (n = 673 out of 12,294 with penicillin allergy label) reported anaphylaxis. These figures indicate that our phenotype likely captures less-severe forms of penicillin hypersensitivity.

We then meta-analyzed the results of the GWASes in these three cohorts and identified two genome-wide significant (p < 5 × 10^−8^) signals for penicillin allergy. The top hit on chromosome 6 was located in the major histocompatibility complex (MHC) region (rs114892859, MAF(EstBB) = 0.7%, MAF(UKBB) = 2%, MAF(BioVU) = 2%; p value 1.29 × 10^−29^; OR 1.47 95% CI 1.38–1.57) ([Figures 1](#fig1)A and [S1](#mmc1), [Table S1](#mmc2)). We also identified a further signal for rs2476601, a missense variant in *PTPN22* on chromosome 1 (p value 2.68 × 10^−9^; OR 1.09 95% CI 1.06–1.12).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4e24/7536643/e73c39b142b2/gr1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7536643_gr1.jpg)

Manhattan Plot and HLA Locus of the Genome-wide Association Study of Penicillin Allergy The X axes indicate chromosomal positions and Y axes −log10 of the p Values. (A) Each dot represents a single-nucleotide polymorphism (SNP). The dotted line indicates the genome-wide significance (p value < 5.0 × 10−8) p value threshold. (B) SNPs are colored according to their linkage disequilibrium (LD; based on the 1000 Genomes phase3 EUR reference panel) with the lead SNP. The SNP marked with a purple diamond is the lead SNP rs114892859.

### Fine-Mapping the Penicillin Allergy-Associated HLA Locus

To further characterize the identified association with penicillin allergy, we performed a functional annotation analysis with FUMA (Functional Mapping and Annotation of Genome-Wide Association Studies).[21](#bib21) We detected an independent intronic lead SNP for the penicillin allergy meta-analysis (GWAS lead variant rs114892859, p value 1.29 × 10^−29^) in *MICA* ([Figure 1](#fig1)B). When testing the SNP for expression quantitative trait locus (eQTL) associations in blood based on data from the eQTLGen Consortium,[22](#bib22) the variant appeared to be associated with the expression levels of several nearby genes, with the most significant being *PSORS1C3* (p value 8.10 × 10^−62^) and *MICA* (p value 1.21 × 10^−52^) ([Table S2](#mmc3)). We further performed an *in silico* investigation of the lead SNP rs114892859 and its best proxy (rs144626001, the only proxy with r^2^ > 0.9 in UKBB and EstBB) in HaploReg v.4 to explore annotations and impact of the non-coding variant.[23](#bib23) rs114892859 in particular had several annotations indicative of a regulatory function, including its location in both promoter and enhancer marks in T cells and evidence of RNA polymerase II binding.[24](#bib24)^,^[25](#bib25) Interestingly, its proxy is more likely to be deleterious based on the scaled Combined Annotation Dependent Depletion (CADD) score (scaled score of 15.78 for rs144626001 (C/T) and 4.47 for rs114892859 (G/T)).[26](#bib26)^,^[27](#bib27) To assess the association of the rs114892859 variant with self-reported penicillin allergy in non-European ancestries, we used the recently developed Pan-UKB resource (see [Web Resources](#app4)) and retrieved summary statistics for individuals of Central/South Asian, African, East Asian, and Middle Eastern ([Table S3](#mmc1)) ancestries. We did not find an association with penicillin allergy in these other ancestry groups. Neither did we find any association of the rs114892859 variant with penicillin allergy (p value 0.288; OR 0.67 95% CI 0.14–1.19) in a subset of 14,416 BioVU individuals with self-reported African ancestry, including 1,894 case subjects and 9,539 control subjects. Nevertheless, these sample sizes are substantially smaller than the European-ancestry groups we studied and larger cohorts of diverse ancestries will be needed to provide more definitive insights.

Due to the high LD in the MHC region, we used imputed SNP to HLA typing data available at four-digit resolution[28](#bib28) for up to 22,554, 488,377, and 67,323 individuals from the Estonian, UK, and BioVU cohorts, respectively, to further fine-map the identified HLA association with penicillin allergy. In all cohorts a shared total of 104 alleles at four-digit level were present for all of the MHC class I genes (*HLA-A, HLA-B, HLA-C*) and 60 alleles for three of the classical MHC class II genes (*HLA-DRB1, HLA-DQA1, HLA-DQB1*). To assess the variation in the frequencies of the HLA alleles in different populations, we compared the obtained allele frequencies in EstBB and UKBB ([Table S4](#mmc4)) with the frequencies of HLA alleles in different European, Asian, and African populations reported in the HLA frequency database ([Figures S2](#mmc1) and [S3](#mmc1), [Table S5](#mmc5)).

We then used an additive logistic regression model to test for associations between different four-digit HLA alleles and penicillin allergy in UKBB, EstBB, and BioVU. The results from these three cohorts were meta-analyzed, using a Bonferroni-corrected p value threshold (0.05/164 = 3.05 × 10^−4^, where 164 is the number of meta-analyzed HLA alleles). One of the two results that surpassed the threshold had discordant effects in the tested cohorts ([Table S6](#mmc6)). The only association with the same directional effect in all three cohorts that we detected for penicillin allergy was the HLA-B^∗^55:01 allele (p value 2.04 × 10^−31^; OR 1.41 95% CI 1.33–1.49; [Table S6](#mmc6)), which is tagged (r^2^ > 0.95) by the GWAS lead variant rs114892859 ([Table S7](#mmc1)). We performed a separate meta-analysis for the HLA-B^∗^55:01 allele in all case subjects from BioVU and EstBB (p value 1.98 × 10^−8^; OR 1.32 95% CI 1.20–1.45) and compared it to a meta-analysis where severe reactions of anaphylaxis were excluded. Despite the smaller sample size, the estimates from this analysis were similar (p value 1.28 × 10^−8^; OR 1.33 95% CI 1.20–1.46), indicating that the association is not driven by more severe hypersensitivity reactions.

### Replication of the HLA-B∗55:01 Association with Penicillin Allergy

To further confirm association with penicillin allergy, we analyzed the association of the HLA-B^∗^55:01 allele with self-reported penicillin allergy among 87,996 case subjects and 1,031,087 control subjects of European ancestry from the 23andMe research cohort. We observed an association (p value 1.00 × 10^−47^; OR 1.30 95% CI 1.25–1.34; [Figure 2](#fig2)) with a similar effect size as seen for the HLA-B^∗^55:01 allele in the meta-analysis of the EstBB, UKBB, and BioVU. Meta-analysis of estimates for HLA-B^∗^55:01 from the discovery and replication cohorts demonstrated a 33% higher relative odds of penicillin allergy among carriers of the allele (p value 1.15 × 10^−77^; OR 1.33 95% CI 1.29–1.37; [Figure 2](#fig2)).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4e24/7536643/cccd1d463090/gr2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7536643_gr2.jpg)

HLA-B∗55:01 Allele Association with Self-Reported Penicillin Allergy The odds ratios (dots) and 95% confidence intervals (CI, horizontal lines) for the association of the HLA allele with penicillin allergy are presented. The plot is annotated with p values and case-control numbers. Color coding indicates the results for discovery cohorts UKBB (black), EstBB (blue), and BioVU (purple) and replication results of the HLA-B∗55:01 allele in the 23andMe research cohort (green). Results of the meta-analysis of all four cohorts is indicated with a diamond (red). Self-reported penicillin allergy is defined as ICD10 code Z88.0 (UKBB), reported drug allergy labels from the allergy section of the EHR (BioVU), reported allergy to drugs in ATC J01C∗ class (EstBB), or reported allergy to penicillin (23andMe).

### Further Associations at HLA-B∗55:01

Finally, we used the Open Targets Genetics platform’s UKBB PheWAS data[29](#bib29) to further characterize the association of the GWAS lead variant (and HLA-B^∗^55:01 allele tag-SNP) rs114892859 with other traits. We found associations with lower lymphocyte counts (p value 9.21 × 10^−14^, −0.098 cells per nanoliter, per allergy-increasing T allele) and lower white blood cell counts (p value 3.17 × 10^−9^, −0.078 cells per nanoliter, per allergy-increasing T allele). To confirm this finding, we extracted data on lymphocyte counts from the EHR data of 4,567 EstBB participants (see [Supplemental Subjects and Methods](#mmc1)) and observed the same inverse association of the HLA-B^∗^55:01 allele with lymphocyte counts (−0.148 cells per nanoliter, per T allele; p value = 0.047).

To investigate the possible functional impact of the HLA-B^∗^55:01 allele, we compared the amino acid sequence of all the *HLA-B* alleles commonly present in Estonian and UK biobank (see [Supplemental Subjects and Methods](#mmc1)). Only one allele that was represented in our populations, HLA-B^∗^56:01, shared a high sequence similarity (>99%) with HLA-B^∗^55:01, while all the other 46 alleles shared 86.5%–93% of the amino acids within the antigen-binding cleft ([Table S8](#mmc7)). Further analysis revealed that HLA-B^∗^56:01 and HLA-B^∗^55:01 differ by only two amino acids in the α2 domain: p.Glu152Val and p.Thr163Leu ([Figure S4](#mmc1)). We did not observe an association between the HLA-B^∗^56:01 allele and penicillin allergy (p value = 0.24), which might suggest that the two amino acid differences may have functional relevance in penicillin allergy. However, despite the large number of case subjects in our study, power was limited to rule out an association with HLA-B^∗^56:01, as it is only present at a frequency of 0.3% in European populations.

To get closer to the possible endophenotypes tagging the identified associations, we investigated genetic correlation of self-reported penicillin allergy with studies on autoimmune and hematological traits using LDhub ([Table S9](#mmc1)).[30](#bib30) The analysis pointed toward genetic correlation (r_g_) of 0.35 (p value 3.65 × 10^−7^) between self-reported penicillin allergy and rheumatoid arthritis (RA). This result was virtually unchanged when we excluded 1 Mb around *PTPN22* (r_g_ = 0.35, p value 6.13 × 10^−6^), a known RA risk locus. Since we detected this genetic correlation between RA and self-reported penicillin allergy, we redid the penicillin allergy association analysis for the HLA-B^∗^55:01 allele among only RA case subjects in UKBB (468 penicillin allergy case subjects and 4,065 control subjects). The effect estimate for the HLA-B^∗^55:01 allele was similar to that from the whole UKBB cohort (p value 0.032; OR 1.57 95% CI 1.04–2.38). Because LDhub did not have data for psoriasis, we further used summary statistics of the GWAS meta-analysis available for psoriasis from the PAN UKBB resource (2,868 case subjects and 417,663 control subject subjects). The genetic correlation of the GWAS meta-analysis of penicillin allergy with psoriasis was 0.44 (p value 0.002). In summary, our results suggest that the self-reported penicillin allergy phenotype could be tagging a less severe, T cell-mediated, delayed-type penicillin allergy, and that it may involve an autoimmune component.

## Discussion

In the present study, we identify associations of the HLA-B^∗^55:01 allele and a missense variant rs2476601 in *PTPN22* with self-reported penicillin allergy using data from four large cohorts: UKBB, EstBB, BioVu, and 23andMe. Hypersensitivity or allergic reactions to medications are type B adverse drug reactions that are known to be mediated by the immune system. One major driver of hypersensitivity reactions is thought to be the HLA system. HLA class I alleles are expressed on all nucleated cells. HLA is the most polymorphic region of the human genome that has played a major evolutionary role in adaptive immune responses through presentation of foreign peptides to T cell receptors that in the case of an HLA-class I restricted response leads to activation of CD8^+^ T cells.[31](#bib31) Genetic variation in the HLA region alters the shape of the peptide-binding pocket in HLA molecules and enables their binding to a vast number of different peptides—a crucial step in the adaptive immune response.[32](#bib32) However, this ability of HLA molecules to bind a wide variety of peptides may also facilitate binding of exogenous molecules such as drugs, potentially leading to off-target drug effects and immune-mediated ADRs.[33](#bib33) The precise mechanism of most HLA-drug interactions remains unknown, but it seems that T cell activation is necessary for the majority of HLA-mediated ADRs.[33](#bib33), [34](#bib34), [35](#bib35) Despite the increasing evidence for a role of the HLA system in drug-induced hypersensitivity, much is still unclear mechanistically as to how genetic variation in the HLA region predisposes to specific drug reactions.

Penicillin is the most common cause of drug allergy, with clinical manifestations ranging from relatively benign cutaneous reactions to life-threatening systemic syndromes.[7](#bib7)^,^[8](#bib8) There is a previous GWAS on the immediate type of penicillin allergy, where a borderline genome-wide significant protective association of an allele of the MHC class II gene *HLA-DRA* was detected and further replicated in a different cohort.[36](#bib36) Here we detect a robust association between self-reported penicillin allergy and an allele of the MHC class I gene *HLA-B*. The allele and its tag-SNP were also associated with lower lymphocyte counts and overlapped with T cell regulatory annotations. This raises the possibility that the variant may predispose to a T cell-mediated process that could lead to a delayed penicillin reaction through a heterologous response from an HLA-B^∗^55:01 restricted immune response that occurred earlier in life to a prevalent pathogen or an infection or disease interaction. MHC I molecules are expressed by almost all cells and present peptides to cytotoxic CD8^+^ T cells, whereas MHC II molecules are expressed by antigen-presenting cells to present peptides to CD4^+^ T helper lymphocyte.[32](#bib32)^,^[35](#bib35) There are several examples of MHC I alleles associated with drug-induced hypersensitivity mediated by CD8^+^ T cells.[35](#bib35)^,^[37](#bib37)^,^[38](#bib38) The involvement of T cells in delayed hypersensitivity reactions has been shown by isolating drug-reactive T cell clones,[39](#bib39) and cytotoxic CD8^+^ T cells have been shown to be relevant especially in allergic skin reactions.[40](#bib40), [41](#bib41), [42](#bib42) More than 20 years ago, CD8^+^ T cells reactive to penicillin were isolated from patients with delayed type of hypersensitivity to penicillin.[43](#bib43) The association with the HLA-B^∗^55:01 allele detected in our study might be a relevant factor in this established connection with CD8^+^ T cells as HLA-B07-supertype alleles that share peptide binding specificities with HLA-B^∗^55:01 have previously been associated with nevirapine-induced rash.[44](#bib44) The underlying mechanism in penicillin allergy remains a question and various models have been proposed for T cell-mediated hypersensitivity.[37](#bib37)^,^[42](#bib42) For example, the hapten model suggests that drugs may alter proteins and thereby induce an immune response[37](#bib37)^,^[45](#bib45)—penicillins have been shown to bind proteins[45](#bib45)^,^[46](#bib46) to form hapten-carrier complexes, which may in turn elicit a T cell response.[47](#bib47) Drugs may also non-covalently interact with MHC molecules and alter the repertoire of bound peptides leading to presentation of antigens to which the host has not been previously tolerized. For example, abacavir has been shown to bind non-covalently within the F pocket of the antigen binding cleft of HLA-B^∗^57:01, altering its peptide specificity and leading to a CD8^+^ T cell-mediated hypersensitivity response.[48](#bib48), [49](#bib49), [50](#bib50)

It is increasingly recognized that the involvement of HLA variation in hypersensitivity reactions goes beyond peptide specificity. Other factors, such as effects on HLA expression that influence the strength of the immune response, have also been described.[51](#bib51) The analysis of eQTLs based on the data of the eQTLGen Consortium[22](#bib22) revealed that the lead SNP rs114892859 identified in our GWAS of penicillin allergy appears to be associated with the expression of several nearby genes, including expression of both *HLA-B* and *HLA-C*, and an even stronger effect on RNA levels of *PSORS1C3* and *MICA* ([Table S2](#mmc3)). Variants in *PSORS1C3* have been associated with the risk of allopurinol-, carbamazepine-, and phenytoin-induced SJS/TEN hypersensitivity reactions[52](#bib52) and *MICA* encodes the protein MHC class I polypeptide-related sequence A[53](#bib53) which has been implicated in immune surveillance.[54](#bib54)^,^[55](#bib55) Our findings therefore support the observation that variants associated with expression of HLA genes may contribute to the development of hypersensitivity reactions.

We also detected an association with variants in *PTPN22* on chromosome 1. *PTPN22* encodes a tyrosine phosphatase involved in the regulation of immune cell signaling.[56](#bib56) The lead missense variant rs2476601 has previously been associated with several autoimmune diseases[57](#bib57) and is a risk allele for rheumatoid arthritis.[58](#bib58) Interestingly, this variant was also recently shown to be associated with drug-induced liver injury (DILI).[59](#bib59) The association with the rs2476601 variant was strongest for cases of amoxicillin- and clavulanic acid-associated liver injury (OR 1.62, p value 4.0 × 10^−6^). Case subjects in this study were clinically sourced and comprehensively phenotyped, which suggests that our self-reported penicillin allergy phenotype might also capture signal related to more severe forms of beta-lactam hypersensitivity. However, the effect of this variant on penicillin allergy in the current study is relatively small (OR 1.09) and its role in the development of allergic reactions needs further studies.

A genetic correlation analysis of the penicillin allergy GWAS results in the current study revealed overlap with rheumatoid arthritis, even when excluding the *PTPN22* region from the analysis. Furthermore, we identified a genetic correlation with psoriasis, another autoimmune disease. Both psoriasis and psoriatic arthritis also have associations with *HLA-B* alleles.[60](#bib60)^,^[61](#bib61) This indicates a possible underlying autoimmune factor in the development of the penicillin allergy phenotype investigated in our study.

Studies have suggested that penicillin allergy labels are acquired in childhood and of children labeled as penicillin allergic, 75% have acquired this label by age 3.[8](#bib8) In addition, approximately 10% of patients tested per year will lose their skin test reactivity, meaning that by adulthood >95% of allergy labels can be removed with formal testing. The main limitation of this study is the unverified nature of the phenotypes extracted from EHRs and self-reported data in the biobanks. Previous work has found that most (90%–95%) individuals labeled as having beta-lactam hypersensitivity may not actually have true hypersensitivity by adulthood when they more commonly undergo validated testing.[7](#bib7)^,^[8](#bib8)^,^[62](#bib62)^,^[63](#bib63) However, we believe that the phenotype we have studied is valid for several reasons.[7](#bib7)^,^[62](#bib62) The most commonly reported penicillin allergy is delayed-type allergy, which usually manifests as a transient benign rash that does not recur on rechallenge many years later.[7](#bib7)^,^[62](#bib62)^,^[64](#bib64) Furthermore, many individuals who were once labeled as having IgE-mediated penicillin allergy develop tolerance over time.[7](#bib7) In both of these cases, a true penicillin-induced reaction occurred initially, even if tolerance develops subsequently. Our phenotype therefore could represent individuals that experienced a reaction associated to penicillin when they were previously exposed, but who may, over time, tolerate penicillin administration. A delayed rash is the most common self-reported reaction seen in association with penicillin,[7](#bib7)^,^[62](#bib62)^,^[64](#bib64) which is frequently a T cell-mediated process. The results from our *in silico* analyses, which link HLA-B^∗^55:01 to T cell biology, would support this observation. Therefore, we posit that the association with the HLA-B^∗^55:01 allele may represent a predisposition to an immune response associated with penicillin which does not appear to be associated with a severe immediate or delayed reaction and which may wane with time.

Despite the possibility that some cases in our study may be misclassified, we detect a robust HLA association that was replicated in several independent cohorts against related phenotypes. The increased power arising from biobank-scale sample sizes therefore mitigates some of the challenges associated with EHR data. The robustness of the genetic signal across cohorts with orthogonal phenotyping methods, ranging from EHR-sourced in EstBB and BioVU to various forms of self-reported data in UKBB and 23andMe, also supports a true association. Finally, the modest effect size of the HLA-B^∗^55:01 allele (OR 1.33), particularly when compared to effect sizes of HLA alleles with established pharmacogenetic relevance,[65](#bib65), [66](#bib66), [67](#bib67) suggests that this variant has limited predictive value. However, further phenotypic refinement, including investigation of specific penicillin derivatives and specific types of drug reactions, may yield more clinically actionable insight.

In summary, we have leveraged data from four large-scale cohorts, including more than 100,000 case subjects, to provide insights into the genetic architecture of self-reported penicillin allergy and to provide robust evidence implicating the HLA-B^∗^55:01 allele in this condition. Further studies are necessary to determine the precise underlying immune processes and how these change over time.

## Consortia

The members of 23andMe research team: Michelle Agee, Stella Aslibekyan, Robert K. Bell, Katarzyna Bryc, Sarah K. Clark, Sarah L. Elson, Kipper Fletez-Brant, Pierre Fontanillas, Nicholas A. Furlotte, Pooja M. Gandhi, Karl Heilbron, Barry Hicks, David A. Hinds, Karen E. Huber, Ethan M. Jewett, Yunxuan Jiang, Aaron Kleinman, Keng-Han Lin, Nadia K. Litterman, Marie K. Luff, Jennifer C. McCreight, Matthew H. McIntyre, Kimberly F. McManus, Joanna L. Mountain, Sahar V. Mozaffari, Priyanka Nandakumar, Elizabeth S. Noblin, Carrie A.M. Northover, Jared O’Connell, Aaron A. Petrakovitz, Steven J. Pitts, G. David Poznik, J. Fah Sathirapongsasuti, Anjali J. Shastri, Janie F. Shelton, Suyash Shringarpure, Chao Tian, Joyce Y. Tung, Robert J. Tunney, Vladimir Vacic, Xin Wang, and Amir S. Zare.

## Author Contributions

K.K., L.M., and J.F. designed the study. R.M., M.L., Y.L., S.R., E.J.P., D.M.R., W.-Q.W., A.M., and T.E. supervised and generated genotype data or HLA typing data. D.S. and S.L. generated allergy data from free text. K.K., J.B., N.Z., M.L., T.J., J.C.C., T.L., J.F., W.W., and A.A. performed the data analysis. E.A. conducted amino acid sequence analysis. K.K., J.B., N.Z., M.V.H., C.M.L., R.M., L.M., J.C.C., E.J.P., W.-Q.W., and J.F. conducted data interpretation. K.K. prepared the figures and tables. K.K., J.B., L.M., and J.F. drafted the manuscript. K.K., J.B., N.Z., M.V.H., C.M.L., M.L., R.M., L.M., J.C.C., W.W., A.A., E.J.P., J.F., B.F., F.G., and L.S. reviewed and edited the manuscript. All authors contributed to critical revisions and approved the final manuscript.

The corresponding author attests that all listed authors meet authorship criteria and that no others meeting the criteria have been omitted.

## Declaration of Interests

C.M.L. has collaborated with Novo Nordisk and Bayer in research, and in accordance with a university agreement, did not accept any personal payment. W.W., A.A., and members of the 23andMe Research Team are employed by and hold stock or stock options in 23andMe, Inc. There were no other relationships or activities that could appear to have influenced the submitted work. The views expressed are those of the author(s) and not necessarily those of the NHS, the NIHR, the Department of Health, or the NIH.

## Acknowledgments

We thank all participants and staff of the Estonian, UK, and BioVU biobanks, and 23andMe for their contribution to this research. This work was carried out in part in the High Performance Computing Center of the University of Tartu.

This study has been supported by grants from the Estonian Research Council (PRG184, PRG687, IUT20-60, and IUT24-6) and the Oak Foundation. The BioVU analyses used Vanderbilt University Medical Center’s resources, the Synthetic Derivative, which are supported by institutional funding and the National Center for Advancing Translational Science grant 2UL1 TR000445-06. J.B. is supported by the Rhodes Trust, Clarendon Fund, and the Medical Sciences Doctoral Training Centre, University of Oxford. J.C.C. is funded by the Oxford Medical Research Council Doctoral Training Partnership and the Nuffield Department of Clinical Medicine, University of Oxford. C.M.L. is supported by the Li Ka Shing Foundation; WT-SSI/John Fell funds; the NIHR Biomedical Research Centre, Oxford; Widenlife; and NIH (5P50HD028138-27). M.V.H. is supported by a BHF Intermediate Clinical Research Fellowship (FS/18/23/33512) and the NIHR Biomedical Research Centre, Oxford. Computation used the Oxford Biomedical Research Computing (BMRC) facility, a joint development between the Wellcome Centre for Human Genetics and the Big Data Institute supported by Health Data Research UK and the NIHR Oxford Biomedical Research Centre. Financial support was provided by the Wellcome Trust Core Award Grant Number 203141/Z/16/Z (see full acknowledgments in the [Supplemental Information](#app3)). E.J.P. receives support from NIH (P50GM115305 [also D.M.R. and W.-Q.W.], R01HG010863, R01AI152183, R21AI139021, U01AI154659) and the National Health and Medical Research Council of Australia.

Published: September 3, 2020

## Footnotes

## Contributor Information

Lili Milani, Email: lili.milani@ut.ee.

23andMe Research Team:

[Michelle Agee](https://pubmed.ncbi.nlm.nih.gov/?term=%22Agee%20M%22%5BAuthor%5D), [Stella Aslibekyan](https://pubmed.ncbi.nlm.nih.gov/?term=%22Aslibekyan%20S%22%5BAuthor%5D), [Robert K. Bell](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bell%20RK%22%5BAuthor%5D), [Katarzyna Bryc](https://pubmed.ncbi.nlm.nih.gov/?term=%22Bryc%20K%22%5BAuthor%5D), [Sarah K. Clark](https://pubmed.ncbi.nlm.nih.gov/?term=%22Clark%20SK%22%5BAuthor%5D), [Sarah L. Elson](https://pubmed.ncbi.nlm.nih.gov/?term=%22Elson%20SL%22%5BAuthor%5D), [Kipper Fletez-Brant](https://pubmed.ncbi.nlm.nih.gov/?term=%22Fletez-Brant%20K%22%5BAuthor%5D), [Pierre Fontanillas](https://pubmed.ncbi.nlm.nih.gov/?term=%22Fontanillas%20P%22%5BAuthor%5D), [Nicholas A. Furlotte](https://pubmed.ncbi.nlm.nih.gov/?term=%22Furlotte%20NA%22%5BAuthor%5D), [Pooja M. Gandhi](https://pubmed.ncbi.nlm.nih.gov/?term=%22Gandhi%20PM%22%5BAuthor%5D), [Karl Heilbron](https://pubmed.ncbi.nlm.nih.gov/?term=%22Heilbron%20K%22%5BAuthor%5D), [Barry Hicks](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hicks%20B%22%5BAuthor%5D), [David A. Hinds](https://pubmed.ncbi.nlm.nih.gov/?term=%22Hinds%20DA%22%5BAuthor%5D), [Karen E. Huber](https://pubmed.ncbi.nlm.nih.gov/?term=%22Huber%20KE%22%5BAuthor%5D), [Ethan M. Jewett](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jewett%20EM%22%5BAuthor%5D), [Yunxuan Jiang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Y%22%5BAuthor%5D), [Aaron Kleinman](https://pubmed.ncbi.nlm.nih.gov/?term=%22Kleinman%20A%22%5BAuthor%5D), [Keng-Han Lin](https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20KH%22%5BAuthor%5D), [Nadia K. Litterman](https://pubmed.ncbi.nlm.nih.gov/?term=%22Litterman%20NK%22%5BAuthor%5D), [Marie K. Luff](https://pubmed.ncbi.nlm.nih.gov/?term=%22Luff%20MK%22%5BAuthor%5D), [Jennifer C. McCreight](https://pubmed.ncbi.nlm.nih.gov/?term=%22McCreight%20JC%22%5BAuthor%5D), [Matthew H. McIntyre](https://pubmed.ncbi.nlm.nih.gov/?term=%22McIntyre%20MH%22%5BAuthor%5D), [Kimberly F. McManus](https://pubmed.ncbi.nlm.nih.gov/?term=%22McManus%20KF%22%5BAuthor%5D), [Joanna L. Mountain](https://pubmed.ncbi.nlm.nih.gov/?term=%22Mountain%20JL%22%5BAuthor%5D), [Sahar V. Mozaffari](https://pubmed.ncbi.nlm.nih.gov/?term=%22Mozaffari%20SV%22%5BAuthor%5D), [Priyanka Nandakumar](https://pubmed.ncbi.nlm.nih.gov/?term=%22Nandakumar%20P%22%5BAuthor%5D), [Elizabeth S. Noblin](https://pubmed.ncbi.nlm.nih.gov/?term=%22Noblin%20ES%22%5BAuthor%5D), [Carrie A.M. Northover](https://pubmed.ncbi.nlm.nih.gov/?term=%22Northover%20CAM%22%5BAuthor%5D), [Jared O’Connell](https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Connell%20J%22%5BAuthor%5D), [Aaron A. Petrakovitz](https://pubmed.ncbi.nlm.nih.gov/?term=%22Petrakovitz%20AA%22%5BAuthor%5D), [Steven J. Pitts](https://pubmed.ncbi.nlm.nih.gov/?term=%22Pitts%20SJ%22%5BAuthor%5D), [G. David Poznik](https://pubmed.ncbi.nlm.nih.gov/?term=%22Poznik%20GD%22%5BAuthor%5D), [J. Fah Sathirapongsasuti](https://pubmed.ncbi.nlm.nih.gov/?term=%22Sathirapongsasuti%20JF%22%5BAuthor%5D), [Anjali J. Shastri](https://pubmed.ncbi.nlm.nih.gov/?term=%22Shastri%20AJ%22%5BAuthor%5D), [Janie F. Shelton](https://pubmed.ncbi.nlm.nih.gov/?term=%22Shelton%20JF%22%5BAuthor%5D), [Suyash Shringarpure](https://pubmed.ncbi.nlm.nih.gov/?term=%22Shringarpure%20S%22%5BAuthor%5D), [Chao Tian](https://pubmed.ncbi.nlm.nih.gov/?term=%22Tian%20C%22%5BAuthor%5D), [Joyce Y. Tung](https://pubmed.ncbi.nlm.nih.gov/?term=%22Tung%20JY%22%5BAuthor%5D), [Robert J. Tunney](https://pubmed.ncbi.nlm.nih.gov/?term=%22Tunney%20RJ%22%5BAuthor%5D), [Vladimir Vacic](https://pubmed.ncbi.nlm.nih.gov/?term=%22Vacic%20V%22%5BAuthor%5D), [Xin Wang](https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D), and [Amir S. Zare](https://pubmed.ncbi.nlm.nih.gov/?term=%22Zare%20AS%22%5BAuthor%5D)

## Web Resources

## Data and Code Availability

All code used for the analysis is described in the [Subjects and Methods](#sec2) section of the manuscript. Genotype and phenotype data are available from the Estonian Biobank ([https://genomics.ut.ee/en/biobank.ee/data-access](https://genomics.ut.ee/en/biobank.ee/data-access)) and UK Biobank ([https://www.ukbiobank.ac.uk/using-the-resource](https://www.ukbiobank.ac.uk/using-the-resource)/) upon request.

## Supplemental Data

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All code used for the analysis is described in the [Subjects and Methods](#sec2) section of the manuscript. Genotype and phenotype data are available from the Estonian Biobank ([https://genomics.ut.ee/en/biobank.ee/data-access](https://genomics.ut.ee/en/biobank.ee/data-access)) and UK Biobank ([https://www.ukbiobank.ac.uk/using-the-resource](https://www.ukbiobank.ac.uk/using-the-resource)/) upon request.

### Supplementary Materials

### Data Availability Statement

All code used for the analysis is described in the [Subjects and Methods](#sec2) section of the manuscript. Genotype and phenotype data are available from the Estonian Biobank ([https://genomics.ut.ee/en/biobank.ee/data-access](https://genomics.ut.ee/en/biobank.ee/data-access)) and UK Biobank ([https://www.ukbiobank.ac.uk/using-the-resource](https://www.ukbiobank.ac.uk/using-the-resource)/) upon request.

## References

1. Lazarou J., Pomeranz B.H., Corey P.N. Incidence of adverse drug reactions in hospitalized patients: a meta-analysis of prospective studies. JAMA. 1998;279:1200–1205. doi: 10.1001/jama.279.15.1200.  [DOI](https://doi.org/10.1001/jama.279.15.1200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9555760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Incidence%20of%20adverse%20drug%20reactions%20in%20hospitalized%20patients:%20a%20meta-analysis%20of%20prospective%20studies&author=J.%20Lazarou&author=B.H.%20Pomeranz&author=P.N.%20Corey&volume=279&publication_year=1998&pages=1200-1205&pmid=9555760&doi=10.1001/jama.279.15.1200&)

2. Santoro A., Genov G., Spooner A., Raine J., Arlett P. Promoting and Protecting Public Health: How the European Union Pharmacovigilance System Works. Drug Saf. 2017;40:855–869. doi: 10.1007/s40264-017-0572-8.  [DOI](https://doi.org/10.1007/s40264-017-0572-8) | [PMC free article](/articles/PMC5606958/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28735357/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Saf.&title=Promoting%20and%20Protecting%20Public%20Health:%20How%20the%20European%20Union%20Pharmacovigilance%20System%20Works&author=A.%20Santoro&author=G.%20Genov&author=A.%20Spooner&author=J.%20Raine&author=P.%20Arlett&volume=40&publication_year=2017&pages=855-869&pmid=28735357&doi=10.1007/s40264-017-0572-8&)

3. Bouvy J.C., De Bruin M.L., Koopmanschap M.A. Epidemiology of adverse drug reactions in Europe: a review of recent observational studies. Drug Saf. 2015;38:437–453. doi: 10.1007/s40264-015-0281-0.  [DOI](https://doi.org/10.1007/s40264-015-0281-0) | [PMC free article](/articles/PMC4412588/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25822400/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Saf.&title=Epidemiology%20of%20adverse%20drug%20reactions%20in%20Europe:%20a%20review%20of%20recent%20observational%20studies&author=J.C.%20Bouvy&author=M.L.%20De%20Bruin&author=M.A.%20Koopmanschap&volume=38&publication_year=2015&pages=437-453&pmid=25822400&doi=10.1007/s40264-015-0281-0&)

4. Alagoz O., Durham D., Kasirajan K. Cost-effectiveness of one-time genetic testing to minimize lifetime adverse drug reactions. Pharmacogenomics J. 2016;16:129–136. doi: 10.1038/tpj.2015.39.  [DOI](https://doi.org/10.1038/tpj.2015.39) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25987241/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Cost-effectiveness%20of%20one-time%20genetic%20testing%20to%20minimize%20lifetime%20adverse%20drug%20reactions&author=O.%20Alagoz&author=D.%20Durham&author=K.%20Kasirajan&volume=16&publication_year=2016&pages=129-136&pmid=25987241&doi=10.1038/tpj.2015.39&)

5. Böhm R., Cascorbi I. Pharmacogenetics and predictive testing of drug hypersensitivity reactions. Front. Pharmacol. 2016;7:396. doi: 10.3389/fphar.2016.00396.  [DOI](https://doi.org/10.3389/fphar.2016.00396) | [PMC free article](/articles/PMC5073094/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27818635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Pharmacogenetics%20and%20predictive%20testing%20of%20drug%20hypersensitivity%20reactions&author=R.%20B%C3%B6hm&author=I.%20Cascorbi&volume=7&publication_year=2016&pages=396&pmid=27818635&doi=10.3389/fphar.2016.00396&)

6. Iasella C.J., Johnson H.J., Dunn M.A. Adverse Drug Reactions: Type A (Intrinsic) or Type B (Idiosyncratic) Clin. Liver Dis. 2017;21:73–87. doi: 10.1016/j.cld.2016.08.005.  [DOI](https://doi.org/10.1016/j.cld.2016.08.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27842776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Liver%20Dis.&title=Adverse%20Drug%20Reactions:%20Type%20A%20(Intrinsic)%20or%20Type%20B%20(Idiosyncratic)&author=C.J.%20Iasella&author=H.J.%20Johnson&author=M.A.%20Dunn&volume=21&publication_year=2017&pages=73-87&pmid=27842776&doi=10.1016/j.cld.2016.08.005&)

7. Blumenthal K.G., Peter J.G., Trubiano J.A., Phillips E.J. Antibiotic allergy. Lancet. 2019;393:183–198. doi: 10.1016/S0140-6736(18)32218-9.  [DOI](https://doi.org/10.1016/S0140-6736(18)32218-9) | [PMC free article](/articles/PMC6563335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30558872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Antibiotic%20allergy&author=K.G.%20Blumenthal&author=J.G.%20Peter&author=J.A.%20Trubiano&author=E.J.%20Phillips&volume=393&publication_year=2019&pages=183-198&pmid=30558872&doi=10.1016/S0140-6736(18)32218-9&)

8. Castells M., Khan D.A., Phillips E.J. Penicillin Allergy. N. Engl. J. Med. 2019;381:2338–2351. doi: 10.1056/NEJMra1807761.  [DOI](https://doi.org/10.1056/NEJMra1807761) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31826341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=Penicillin%20Allergy&author=M.%20Castells&author=D.A.%20Khan&author=E.J.%20Phillips&volume=381&publication_year=2019&pages=2338-2351&pmid=31826341&doi=10.1056/NEJMra1807761&)

9. Mirakian R., Leech S.C., Krishna M.T., Richter A.G., Huber P.A.J., Farooque S., Khan N., Pirmohamed M., Clark A.T., Nasser S.M., Standards of Care Committee of the British Society for Allergy and Clinical Immunology Management of allergy to penicillins and other beta-lactams. Clin. Exp. Allergy. 2015;45:300–327. doi: 10.1111/cea.12468.  [DOI](https://doi.org/10.1111/cea.12468) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25623506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Exp.%20Allergy&title=Management%20of%20allergy%20to%20penicillins%20and%20other%20beta-lactams&author=R.%20Mirakian&author=S.C.%20Leech&author=M.T.%20Krishna&author=A.G.%20Richter&author=P.A.J.%20Huber&volume=45&publication_year=2015&pages=300-327&pmid=25623506&doi=10.1111/cea.12468&)

10. Drug and Therapeutics Bulletin Penicillin allergy-getting the label right. BMJ. 2017;358:j3402. doi: 10.1136/dtb.2017.3.0463.  [DOI](https://doi.org/10.1136/dtb.2017.3.0463) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28279979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Penicillin%20allergy-getting%20the%20label%20right&volume=358&publication_year=2017&pages=j3402&pmid=28279979&doi=10.1136/dtb.2017.3.0463&)

11. Roden D.M., Pulley J.M., Basford M.A., Bernard G.R., Clayton E.W., Balser J.R., Masys D.R. Development of a large-scale de-identified DNA biobank to enable personalized medicine. Clin. Pharmacol. Ther. 2008;84:362–369. doi: 10.1038/clpt.2008.89.  [DOI](https://doi.org/10.1038/clpt.2008.89) | [PMC free article](/articles/PMC3763939/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18500243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Development%20of%20a%20large-scale%20de-identified%20DNA%20biobank%20to%20enable%20personalized%20medicine&author=D.M.%20Roden&author=J.M.%20Pulley&author=M.A.%20Basford&author=G.R.%20Bernard&author=E.W.%20Clayton&volume=84&publication_year=2008&pages=362-369&pmid=18500243&doi=10.1038/clpt.2008.89&)

12. Mitt M., Kals M., Pärn K., Gabriel S.B., Lander E.S., Palotie A., Ripatti S., Morris A.P., Metspalu A., Esko T. Improved imputation accuracy of rare and low-frequency variants using population-specific high-coverage WGS-based imputation reference panel. Eur. J. Hum. Genet. 2017;25:869–876. doi: 10.1038/ejhg.2017.51.  [DOI](https://doi.org/10.1038/ejhg.2017.51) | [PMC free article](/articles/PMC5520064/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28401899/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Hum.%20Genet.&title=Improved%20imputation%20accuracy%20of%20rare%20and%20low-frequency%20variants%20using%20population-specific%20high-coverage%20WGS-based%20imputation%20reference%20panel&author=M.%20Mitt&author=M.%20Kals&author=K.%20P%C3%A4rn&author=S.B.%20Gabriel&author=E.S.%20Lander&volume=25&publication_year=2017&pages=869-876&pmid=28401899&doi=10.1038/ejhg.2017.51&)

13. Kals M., Nikopensius T., Läll K., Pärn K., Sikka T.T., Suvisaari J., Salomaa V., Ripatti S., Palotie A., Metspalu A. Advantages of genotype imputation with ethnically matched reference panel for rare variant association analyses. bioRxiv. 2019 doi: 10.1101/579201.  [DOI](https://doi.org/10.1101/579201) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=bioRxiv&title=Advantages%20of%20genotype%20imputation%20with%20ethnically%20matched%20reference%20panel%20for%20rare%20variant%20association%20analyses&author=M.%20Kals&author=T.%20Nikopensius&author=K.%20L%C3%A4ll&author=K.%20P%C3%A4rn&author=T.T.%20Sikka&publication_year=2019&doi=10.1101/579201&)

14. Bycroft C., Freeman C., Petkova D., Band G., Elliott L.T., Sharp K., Motyer A., Vukcevic D., Delaneau O., O’Connell J. The UK Biobank resource with deep phenotyping and genomic data. Nature. 2018;562:203–209. doi: 10.1038/s41586-018-0579-z.  [DOI](https://doi.org/10.1038/s41586-018-0579-z) | [PMC free article](/articles/PMC6786975/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30305743/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20UK%20Biobank%20resource%20with%20deep%20phenotyping%20and%20genomic%20data&author=C.%20Bycroft&author=C.%20Freeman&author=D.%20Petkova&author=G.%20Band&author=L.T.%20Elliott&volume=562&publication_year=2018&pages=203-209&pmid=30305743&doi=10.1038/s41586-018-0579-z&)

15. Zhou W., Nielsen J.B., Fritsche L.G., Dey R., Gabrielsen M.E., Wolford B.N., LeFaive J., VandeHaar P., Gagliano S.A., Gifford A. Efficiently controlling for case-control imbalance and sample relatedness in large-scale genetic association studies. Nat. Genet. 2018;50:1335–1341. doi: 10.1038/s41588-018-0184-y.  [DOI](https://doi.org/10.1038/s41588-018-0184-y) | [PMC free article](/articles/PMC6119127/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30104761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Efficiently%20controlling%20for%20case-control%20imbalance%20and%20sample%20relatedness%20in%20large-scale%20genetic%20association%20studies&author=W.%20Zhou&author=J.B.%20Nielsen&author=L.G.%20Fritsche&author=R.%20Dey&author=M.E.%20Gabrielsen&volume=50&publication_year=2018&pages=1335-1341&pmid=30104761&doi=10.1038/s41588-018-0184-y&)

16. Willer C.J., Li Y., Abecasis G.R. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics. 2010;26:2190–2191. doi: 10.1093/bioinformatics/btq340.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&author=C.J.%20Willer&author=Y.%20Li&author=G.R.%20Abecasis&volume=26&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

17. Jia X., Han B., Onengut-Gumuscu S., Chen W.-M., Concannon P.J., Rich S.S., Raychaudhuri S., de Bakker P.I.W. Imputing amino acid polymorphisms in human leukocyte antigens. PLoS ONE. 2013;8:e64683. doi: 10.1371/journal.pone.0064683.  [DOI](https://doi.org/10.1371/journal.pone.0064683) | [PMC free article](/articles/PMC3675122/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23762245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Imputing%20amino%20acid%20polymorphisms%20in%20human%20leukocyte%20antigens&author=X.%20Jia&author=B.%20Han&author=S.%20Onengut-Gumuscu&author=W.-M.%20Chen&author=P.J.%20Concannon&volume=8&publication_year=2013&pages=e64683&pmid=23762245&doi=10.1371/journal.pone.0064683&)

18. Dilthey A., Leslie S., Moutsianas L., Shen J., Cox C., Nelson M.R., McVean G. Multi-population classical HLA type imputation. PLoS Comput. Biol. 2013;9:e1002877. doi: 10.1371/journal.pcbi.1002877.  [DOI](https://doi.org/10.1371/journal.pcbi.1002877) | [PMC free article](/articles/PMC3572961/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23459081/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput.%20Biol.&title=Multi-population%20classical%20HLA%20type%20imputation&author=A.%20Dilthey&author=S.%20Leslie&author=L.%20Moutsianas&author=J.%20Shen&author=C.%20Cox&volume=9&publication_year=2013&pages=e1002877&pmid=23459081&doi=10.1371/journal.pcbi.1002877&)

19. Mägi R., Morris A.P. GWAMA: software for genome-wide association meta-analysis. BMC Bioinformatics. 2010;11:288. doi: 10.1186/1471-2105-11-288.  [DOI](https://doi.org/10.1186/1471-2105-11-288) | [PMC free article](/articles/PMC2893603/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20509871/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&title=GWAMA:%20software%20for%20genome-wide%20association%20meta-analysis&author=R.%20M%C3%A4gi&author=A.P.%20Morris&volume=11&publication_year=2010&pages=288&pmid=20509871&doi=10.1186/1471-2105-11-288&)

20. Zhou L., Dhopeshwarkar N., Blumenthal K.G., Goss F., Topaz M., Slight S.P., Bates D.W. Drug allergies documented in electronic health records of a large healthcare system. Allergy. 2016;71:1305–1313. doi: 10.1111/all.12881.  [DOI](https://doi.org/10.1111/all.12881) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26970431/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=Drug%20allergies%20documented%20in%20electronic%20health%20records%20of%20a%20large%20healthcare%20system&author=L.%20Zhou&author=N.%20Dhopeshwarkar&author=K.G.%20Blumenthal&author=F.%20Goss&author=M.%20Topaz&volume=71&publication_year=2016&pages=1305-1313&pmid=26970431&doi=10.1111/all.12881&)

21. Watanabe K., Taskesen E., van Bochoven A., Posthuma D. Functional mapping and annotation of genetic associations with FUMA. Nat. Commun. 2017;8:1826. doi: 10.1038/s41467-017-01261-5.  [DOI](https://doi.org/10.1038/s41467-017-01261-5) | [PMC free article](/articles/PMC5705698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29184056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Functional%20mapping%20and%20annotation%20of%20genetic%20associations%20with%20FUMA&author=K.%20Watanabe&author=E.%20Taskesen&author=A.%20van%20Bochoven&author=D.%20Posthuma&volume=8&publication_year=2017&pages=1826&pmid=29184056&doi=10.1038/s41467-017-01261-5&)

22. Võsa U., Claringbould A., Westra H.-J., Bonder M.J., Deelen P., Zeng B., Kirsten H., Saha A., Kreuzhuber R., Kasela S. Unraveling the polygenic architecture of complex traits using blood eQTL metaanalysis. bioRxiv. 2018 doi: 10.1101/447367.  [DOI](https://doi.org/10.1101/447367) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=bioRxiv&title=Unraveling%20the%20polygenic%20architecture%20of%20complex%20traits%20using%20blood%20eQTL%20metaanalysis&author=U.%20V%C3%B5sa&author=A.%20Claringbould&author=H.-J.%20Westra&author=M.J.%20Bonder&author=P.%20Deelen&publication_year=2018&doi=10.1101/447367&)

23. Ward L.D., Kellis M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Nucleic Acids Res. 2012;40:D930–D934. doi: 10.1093/nar/gkr917.  [DOI](https://doi.org/10.1093/nar/gkr917) | [PMC free article](/articles/PMC3245002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22064851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=HaploReg:%20a%20resource%20for%20exploring%20chromatin%20states,%20conservation,%20and%20regulatory%20motif%20alterations%20within%20sets%20of%20genetically%20linked%20variants&author=L.D.%20Ward&author=M.%20Kellis&volume=40&publication_year=2012&pages=D930-D934&pmid=22064851&doi=10.1093/nar/gkr917&)

24. Kundaje A., Meuleman W., Ernst J., Bilenky M., Yen A., Heravi-Moussavi A., Kheradpour P., Zhang Z., Wang J., Ziller M.J., Roadmap Epigenomics Consortium Integrative analysis of 111 reference human epigenomes. Nature. 2015;518:317–330. doi: 10.1038/nature14248.  [DOI](https://doi.org/10.1038/nature14248) | [PMC free article](/articles/PMC4530010/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25693563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Integrative%20analysis%20of%20111%20reference%20human%20epigenomes&author=A.%20Kundaje&author=W.%20Meuleman&author=J.%20Ernst&author=M.%20Bilenky&author=A.%20Yen&volume=518&publication_year=2015&pages=317-330&pmid=25693563&doi=10.1038/nature14248&)

25. Myers R.M., Stamatoyannopoulos J., Snyder M., Dunham I., Hardison R.C., Bernstein B.E., Gingeras T.R., Kent W.J., Birney E., Wold B., ENCODE Project Consortium A user’s guide to the encyclopedia of DNA elements (ENCODE) PLoS Biol. 2011;9:e1001046. doi: 10.1371/journal.pbio.1001046.  [DOI](https://doi.org/10.1371/journal.pbio.1001046) | [PMC free article](/articles/PMC3079585/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21526222/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&title=A%20user%E2%80%99s%20guide%20to%20the%20encyclopedia%20of%20DNA%20elements%20(ENCODE)&author=R.M.%20Myers&author=J.%20Stamatoyannopoulos&author=M.%20Snyder&author=I.%20Dunham&author=R.C.%20Hardison&volume=9&publication_year=2011&pages=e1001046&pmid=21526222&doi=10.1371/journal.pbio.1001046&)

26. Rentzsch P., Witten D., Cooper G.M., Shendure J., Kircher M. CADD: predicting the deleteriousness of variants throughout the human genome. Nucleic Acids Res. 2019;47(D1):D886–D894. doi: 10.1093/nar/gky1016.  [DOI](https://doi.org/10.1093/nar/gky1016) | [PMC free article](/articles/PMC6323892/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30371827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=CADD:%20predicting%20the%20deleteriousness%20of%20variants%20throughout%20the%20human%20genome&author=P.%20Rentzsch&author=D.%20Witten&author=G.M.%20Cooper&author=J.%20Shendure&author=M.%20Kircher&volume=47&issue=D1&publication_year=2019&pages=D886-D894&pmid=30371827&doi=10.1093/nar/gky1016&)

27. Kircher M., Witten D.M., Jain P., O’Roak B.J., Cooper G.M., Shendure J. A general framework for estimating the relative pathogenicity of human genetic variants. Nat. Genet. 2014;46:310–315. doi: 10.1038/ng.2892.  [DOI](https://doi.org/10.1038/ng.2892) | [PMC free article](/articles/PMC3992975/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24487276/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=A%20general%20framework%20for%20estimating%20the%20relative%20pathogenicity%20of%20human%20genetic%20variants&author=M.%20Kircher&author=D.M.%20Witten&author=P.%20Jain&author=B.J.%20O%E2%80%99Roak&author=G.M.%20Cooper&volume=46&publication_year=2014&pages=310-315&pmid=24487276&doi=10.1038/ng.2892&)

28. Marsh S.G.E., Albert E.D., Bodmer W.F., Bontrop R.E., Dupont B., Erlich H.A., Fernández-Viña M., Geraghty D.E., Holdsworth R., Hurley C.K. Nomenclature for factors of the HLA system, 2010. Tissue Antigens. 2010;75:291–455. doi: 10.1111/j.1399-0039.2010.01466.x.  [DOI](https://doi.org/10.1111/j.1399-0039.2010.01466.x) | [PMC free article](/articles/PMC2848993/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20356336/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tissue%20Antigens&title=Nomenclature%20for%20factors%20of%20the%20HLA%20system,%202010&author=S.G.E.%20Marsh&author=E.D.%20Albert&author=W.F.%20Bodmer&author=R.E.%20Bontrop&author=B.%20Dupont&volume=75&publication_year=2010&pages=291-455&pmid=20356336&doi=10.1111/j.1399-0039.2010.01466.x&)

29. Koscielny G., An P., Carvalho-Silva D., Cham J.A., Fumis L., Gasparyan R., Hasan S., Karamanis N., Maguire M., Papa E. Open Targets: a platform for therapeutic target identification and validation. Nucleic Acids Res. 2017;45(D1):D985–D994. doi: 10.1093/nar/gkw1055.  [DOI](https://doi.org/10.1093/nar/gkw1055) | [PMC free article](/articles/PMC5210543/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27899665/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Open%20Targets:%20a%20platform%20for%20therapeutic%20target%20identification%20and%20validation&author=G.%20Koscielny&author=P.%20An&author=D.%20Carvalho-Silva&author=J.A.%20Cham&author=L.%20Fumis&volume=45&issue=D1&publication_year=2017&pages=D985-D994&pmid=27899665&doi=10.1093/nar/gkw1055&)

30. Zheng J., Erzurumluoglu A.M., Elsworth B.L., Kemp J.P., Howe L., Haycock P.C., Hemani G., Tansey K., Laurin C., Pourcain B.S., Early Genetics and Lifecourse Epidemiology (EAGLE) Eczema Consortium LD Hub: a centralized database and web interface to perform LD score regression that maximizes the potential of summary level GWAS data for SNP heritability and genetic correlation analysis. Bioinformatics. 2017;33:272–279. doi: 10.1093/bioinformatics/btw613.  [DOI](https://doi.org/10.1093/bioinformatics/btw613) | [PMC free article](/articles/PMC5542030/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27663502/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LD%20Hub:%20a%20centralized%20database%20and%20web%20interface%20to%20perform%20LD%20score%20regression%20that%20maximizes%20the%20potential%20of%20summary%20level%20GWAS%20data%20for%20SNP%20heritability%20and%20genetic%20correlation%20analysis&author=J.%20Zheng&author=A.M.%20Erzurumluoglu&author=B.L.%20Elsworth&author=J.P.%20Kemp&author=L.%20Howe&volume=33&publication_year=2017&pages=272-279&pmid=27663502&doi=10.1093/bioinformatics/btw613&)

31. Williams T.M. Human leukocyte antigen gene polymorphism and the histocompatibility laboratory. J. Mol. Diagn. 2001;3:98–104. doi: 10.1016/S1525-1578(10)60658-7.  [DOI](https://doi.org/10.1016/S1525-1578(10)60658-7) | [PMC free article](/articles/PMC1906958/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11486048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Mol.%20Diagn.&title=Human%20leukocyte%20antigen%20gene%20polymorphism%20and%20the%20histocompatibility%20laboratory&author=T.M.%20Williams&volume=3&publication_year=2001&pages=98-104&pmid=11486048&doi=10.1016/S1525-1578(10)60658-7&)

32. Chaplin D.D. Overview of the immune response. J. Allergy Clin. Immunol. 2010;125(2, Suppl 2):S3–S23. doi: 10.1016/j.jaci.2009.12.980.  [DOI](https://doi.org/10.1016/j.jaci.2009.12.980) | [PMC free article](/articles/PMC2923430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20176265/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Allergy%20Clin.%20Immunol.&title=Overview%20of%20the%20immune%20response&author=D.D.%20Chaplin&volume=125&issue=2,%20Suppl%202&publication_year=2010&pages=S3-S23&pmid=20176265&doi=10.1016/j.jaci.2009.12.980&)

33. Illing P.T., Purcell A.W., McCluskey J. The role of HLA genes in pharmacogenomics: unravelling HLA associated adverse drug reactions. Immunogenetics. 2017;69:617–630. doi: 10.1007/s00251-017-1007-5.  [DOI](https://doi.org/10.1007/s00251-017-1007-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28695285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunogenetics&title=The%20role%20of%20HLA%20genes%20in%20pharmacogenomics:%20unravelling%20HLA%20associated%20adverse%20drug%20reactions&author=P.T.%20Illing&author=A.W.%20Purcell&author=J.%20McCluskey&volume=69&publication_year=2017&pages=617-630&pmid=28695285&doi=10.1007/s00251-017-1007-5&)

34. Pavlos R., Mallal S., Phillips E. HLA and pharmacogenetics of drug hypersensitivity. Pharmacogenomics. 2012;13:1285–1306. doi: 10.2217/pgs.12.108.  [DOI](https://doi.org/10.2217/pgs.12.108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22920398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=HLA%20and%20pharmacogenetics%20of%20drug%20hypersensitivity&author=R.%20Pavlos&author=S.%20Mallal&author=E.%20Phillips&volume=13&publication_year=2012&pages=1285-1306&pmid=22920398&doi=10.2217/pgs.12.108&)

35. Negrini S., Becquemont L. HLA-associated drug hypersensitivity and the prediction of adverse drug reactions. Pharmacogenomics. 2017;18:1441–1457. doi: 10.2217/pgs-2017-0090.  [DOI](https://doi.org/10.2217/pgs-2017-0090) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29017379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=HLA-associated%20drug%20hypersensitivity%20and%20the%20prediction%20of%20adverse%20drug%20reactions&author=S.%20Negrini&author=L.%20Becquemont&volume=18&publication_year=2017&pages=1441-1457&pmid=29017379&doi=10.2217/pgs-2017-0090&)

36. Guéant J.L., Romano A., Cornejo-Garcia J.A., Oussalah A., Chery C., Blanca-López N., Guéant-Rodriguez R.M., Gaeta F., Rouyer P., Josse T. HLA-DRA variants predict penicillin allergy in genome-wide fine-mapping genotyping. J. Allergy Clin. Immunol. 2015;135:253–259. doi: 10.1016/j.jaci.2014.07.047.  [DOI](https://doi.org/10.1016/j.jaci.2014.07.047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25224099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Allergy%20Clin.%20Immunol.&title=HLA-DRA%20variants%20predict%20penicillin%20allergy%20in%20genome-wide%20fine-mapping%20genotyping&author=J.L.%20Gu%C3%A9ant&author=A.%20Romano&author=J.A.%20Cornejo-Garcia&author=A.%20Oussalah&author=C.%20Chery&volume=135&publication_year=2015&pages=253-259&pmid=25224099&doi=10.1016/j.jaci.2014.07.047&)

37. Pavlos R., Mallal S., Ostrov D., Buus S., Metushi I., Peters B., Phillips E. T cell-mediated hypersensitivity reactions to drugs. Annu. Rev. Med. 2015;66:439–454. doi: 10.1146/annurev-med-050913-022745.  [DOI](https://doi.org/10.1146/annurev-med-050913-022745) | [PMC free article](/articles/PMC4295772/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25386935/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Med.&title=T%20cell-mediated%20hypersensitivity%20reactions%20to%20drugs&author=R.%20Pavlos&author=S.%20Mallal&author=D.%20Ostrov&author=S.%20Buus&author=I.%20Metushi&volume=66&publication_year=2015&pages=439-454&pmid=25386935&doi=10.1146/annurev-med-050913-022745&)

38. Sousa-Pinto B., Correia C., Gomes L., Gil-Mata S., Araújo L., Correia O., Delgado L. HLA and delayed drug-induced hypersensitivity. Int. Arch. Allergy Immunol. 2016;170:163–179. doi: 10.1159/000448217.  [DOI](https://doi.org/10.1159/000448217) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27576480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20Arch.%20Allergy%20Immunol.&title=HLA%20and%20delayed%20drug-induced%20hypersensitivity&author=B.%20Sousa-Pinto&author=C.%20Correia&author=L.%20Gomes&author=S.%20Gil-Mata&author=L.%20Ara%C3%BAjo&volume=170&publication_year=2016&pages=163-179&pmid=27576480&doi=10.1159/000448217&)

39. Yawalkar N., Egli F., Hari Y., Nievergelt H., Braathen L.R., Pichler W.J. Infiltration of cytotoxic T cells in drug-induced cutaneous eruptions. Clin. Exp. Allergy. 2000;30:847–855. doi: 10.1046/j.1365-2222.2000.00847.x.  [DOI](https://doi.org/10.1046/j.1365-2222.2000.00847.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10848903/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Exp.%20Allergy&title=Infiltration%20of%20cytotoxic%20T%C2%A0cells%20in%20drug-induced%20cutaneous%20eruptions&author=N.%20Yawalkar&author=F.%20Egli&author=Y.%20Hari&author=H.%20Nievergelt&author=L.R.%20Braathen&volume=30&publication_year=2000&pages=847-855&pmid=10848903&doi=10.1046/j.1365-2222.2000.00847.x&)

40. Kalish R.S., Askenase P.W. Molecular mechanisms of CD8+ T cell-mediated delayed hypersensitivity: implications for allergies, asthma, and autoimmunity. J. Allergy Clin. Immunol. 1999;103:192–199. doi: 10.1016/s0091-6749(99)70489-6.  [DOI](https://doi.org/10.1016/s0091-6749(99)70489-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9949307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Allergy%20Clin.%20Immunol.&title=Molecular%20mechanisms%20of%20CD8+%20T%C2%A0cell-mediated%20delayed%20hypersensitivity:%20implications%20for%20allergies,%20asthma,%20and%20autoimmunity&author=R.S.%20Kalish&author=P.W.%20Askenase&volume=103&publication_year=1999&pages=192-199&pmid=9949307&doi=10.1016/s0091-6749(99)70489-6&)

41. Romano A., Blanca M., Torres M.J., Bircher A., Aberer W., Brockow K., Pichler W.J., Demoly P., ENDA. EAACI Diagnosis of nonimmediate reactions to beta-lactam antibiotics. Allergy. 2004;59:1153–1160. doi: 10.1111/j.1398-9995.2004.00678.x.  [DOI](https://doi.org/10.1111/j.1398-9995.2004.00678.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15461594/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=Diagnosis%20of%20nonimmediate%20reactions%20to%20beta-lactam%20antibiotics&author=A.%20Romano&author=M.%20Blanca&author=M.J.%20Torres&author=A.%20Bircher&author=W.%20Aberer&volume=59&publication_year=2004&pages=1153-1160&pmid=15461594&doi=10.1111/j.1398-9995.2004.00678.x&)

42. Adam J., Pichler W.J., Yerly D. Delayed drug hypersensitivity: models of T-cell stimulation. Br. J. Clin. Pharmacol. 2011;71:701–707. doi: 10.1111/j.1365-2125.2010.03764.x.  [DOI](https://doi.org/10.1111/j.1365-2125.2010.03764.x) | [PMC free article](/articles/PMC3093075/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21480949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Delayed%20drug%20hypersensitivity:%20models%20of%20T-cell%20stimulation&author=J.%20Adam&author=W.J.%20Pichler&author=D.%20Yerly&volume=71&publication_year=2011&pages=701-707&pmid=21480949&doi=10.1111/j.1365-2125.2010.03764.x&)

43. Hertl M., Geisel J., Boecker C., Merk H.F. Selective generation of CD8+ T-cell clones from the peripheral blood of patients with cutaneous reactions to beta-lactam antibiotics. Br. J. Dermatol. 1993;128:619–626. doi: 10.1111/j.1365-2133.1993.tb00255.x.  [DOI](https://doi.org/10.1111/j.1365-2133.1993.tb00255.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8338745/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Dermatol.&title=Selective%20generation%20of%20CD8+%20T-cell%20clones%20from%20the%20peripheral%20blood%20of%20patients%20with%20cutaneous%20reactions%20to%20beta-lactam%20antibiotics&author=M.%20Hertl&author=J.%20Geisel&author=C.%20Boecker&author=H.F.%20Merk&volume=128&publication_year=1993&pages=619-626&pmid=8338745&doi=10.1111/j.1365-2133.1993.tb00255.x&)

44. Pavlos R., McKinnon E.J., Ostrov D.A., Peters B., Buus S., Koelle D., Chopra A., Schutte R., Rive C., Redwood A. Shared peptide binding of HLA Class I and II alleles associate with cutaneous nevirapine hypersensitivity and identify novel risk alleles. Sci. Rep. 2017;7:8653. doi: 10.1038/s41598-017-08876-0.  [DOI](https://doi.org/10.1038/s41598-017-08876-0) | [PMC free article](/articles/PMC5561238/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28819312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Shared%20peptide%20binding%20of%20HLA%20Class%20I%20and%20II%20alleles%20associate%20with%20cutaneous%20nevirapine%20hypersensitivity%20and%20identify%20novel%20risk%20alleles&author=R.%20Pavlos&author=E.J.%20McKinnon&author=D.A.%20Ostrov&author=B.%20Peters&author=S.%20Buus&volume=7&publication_year=2017&pages=8653&pmid=28819312&doi=10.1038/s41598-017-08876-0&)

45. Pirmohamed M., Ostrov D.A., Park B.K. New genetic findings lead the way to a better understanding of fundamental mechanisms of drug hypersensitivity. J. Allergy Clin. Immunol. 2015;136:236–244. doi: 10.1016/j.jaci.2015.06.022.  [DOI](https://doi.org/10.1016/j.jaci.2015.06.022) | [PMC free article](/articles/PMC4534769/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26254050/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Allergy%20Clin.%20Immunol.&title=New%20genetic%20findings%20lead%20the%20way%20to%20a%20better%20understanding%20of%20fundamental%20mechanisms%20of%20drug%20hypersensitivity&author=M.%20Pirmohamed&author=D.A.%20Ostrov&author=B.K.%20Park&volume=136&publication_year=2015&pages=236-244&pmid=26254050&doi=10.1016/j.jaci.2015.06.022&)

46. Meng X., Jenkins R.E., Berry N.G., Maggs J.L., Farrell J., Lane C.S., Stachulski A.V., French N.S., Naisbitt D.J., Pirmohamed M., Park B.K. Direct evidence for the formation of diastereoisomeric benzylpenicilloyl haptens from benzylpenicillin and benzylpenicillenic acid in patients. J. Pharmacol. Exp. Ther. 2011;338:841–849. doi: 10.1124/jpet.111.183871.  [DOI](https://doi.org/10.1124/jpet.111.183871) | [PMC free article](/articles/PMC3164351/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21680886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Pharmacol.%20Exp.%20Ther.&title=Direct%20evidence%20for%20the%20formation%20of%20diastereoisomeric%20benzylpenicilloyl%20haptens%20from%20benzylpenicillin%20and%20benzylpenicillenic%20acid%20in%20patients&author=X.%20Meng&author=R.E.%20Jenkins&author=N.G.%20Berry&author=J.L.%20Maggs&author=J.%20Farrell&volume=338&publication_year=2011&pages=841-849&pmid=21680886&doi=10.1124/jpet.111.183871&)

47. Weltzien H.U., Padovan E. Molecular features of penicillin allergy. J. Invest. Dermatol. 1998;110:203–206. doi: 10.1046/j.1523-1747.1998.00122.x.  [DOI](https://doi.org/10.1046/j.1523-1747.1998.00122.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9506435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Invest.%20Dermatol.&title=Molecular%20features%20of%20penicillin%20allergy&author=H.U.%20Weltzien&author=E.%20Padovan&volume=110&publication_year=1998&pages=203-206&pmid=9506435&doi=10.1046/j.1523-1747.1998.00122.x&)

48. Chessman D., Kostenko L., Lethborg T., Purcell A.W., Williamson N.A., Chen Z., Kjer-Nielsen L., Mifsud N.A., Tait B.D., Holdsworth R. Human leukocyte antigen class I-restricted activation of CD8+ T cells provides the immunogenetic basis of a systemic drug hypersensitivity. Immunity. 2008;28:822–832. doi: 10.1016/j.immuni.2008.04.020.  [DOI](https://doi.org/10.1016/j.immuni.2008.04.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18549801/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunity&title=Human%20leukocyte%20antigen%20class%20I-restricted%20activation%20of%20CD8+%20T%C2%A0cells%20provides%20the%20immunogenetic%20basis%20of%20a%20systemic%20drug%20hypersensitivity&author=D.%20Chessman&author=L.%20Kostenko&author=T.%20Lethborg&author=A.W.%20Purcell&author=N.A.%20Williamson&volume=28&publication_year=2008&pages=822-832&pmid=18549801&doi=10.1016/j.immuni.2008.04.020&)

49. Ostrov D.A., Grant B.J., Pompeu Y.A., Sidney J., Harndahl M., Southwood S., Oseroff C., Lu S., Jakoncic J., de Oliveira C.A.F. Drug hypersensitivity caused by alteration of the MHC-presented self-peptide repertoire. Proc. Natl. Acad. Sci. USA. 2012;109:9959–9964. doi: 10.1073/pnas.1207934109.  [DOI](https://doi.org/10.1073/pnas.1207934109) | [PMC free article](/articles/PMC3382472/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22645359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Drug%20hypersensitivity%20caused%20by%20alteration%20of%20the%20MHC-presented%20self-peptide%20repertoire&author=D.A.%20Ostrov&author=B.J.%20Grant&author=Y.A.%20Pompeu&author=J.%20Sidney&author=M.%20Harndahl&volume=109&publication_year=2012&pages=9959-9964&pmid=22645359&doi=10.1073/pnas.1207934109&)

50. Illing P.T., Vivian J.P., Dudek N.L., Kostenko L., Chen Z., Bharadwaj M., Miles J.J., Kjer-Nielsen L., Gras S., Williamson N.A. Immune self-reactivity triggered by drug-modified HLA-peptide repertoire. Nature. 2012;486:554–558. doi: 10.1038/nature11147.  [DOI](https://doi.org/10.1038/nature11147) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22722860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Immune%20self-reactivity%20triggered%20by%C2%A0drug-modified%20HLA-peptide%20repertoire&author=P.T.%20Illing&author=J.P.%20Vivian&author=N.L.%20Dudek&author=L.%20Kostenko&author=Z.%20Chen&volume=486&publication_year=2012&pages=554-558&pmid=22722860&doi=10.1038/nature11147&)

51. Aguiar V.R.C., César J., Delaneau O., Dermitzakis E.T., Meyer D. Expression estimation and eQTL mapping for HLA genes with a personalized pipeline. PLoS Genet. 2019;15:e1008091. doi: 10.1371/journal.pgen.1008091.  [DOI](https://doi.org/10.1371/journal.pgen.1008091) | [PMC free article](/articles/PMC6497317/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31009447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Expression%20estimation%20and%20eQTL%20mapping%20for%20HLA%20genes%20with%20a%20personalized%20pipeline&author=V.R.C.%20Aguiar&author=J.%20C%C3%A9sar&author=O.%20Delaneau&author=E.T.%20Dermitzakis&author=D.%20Meyer&volume=15&publication_year=2019&pages=e1008091&pmid=31009447&doi=10.1371/journal.pgen.1008091&)

52. Génin E., Schumacher M., Roujeau J.C., Naldi L., Liss Y., Kazma R., Sekula P., Hovnanian A., Mockenhaupt M. Genome-wide association study of Stevens-Johnson Syndrome and Toxic Epidermal Necrolysis in Europe. Orphanet J. Rare Dis. 2011;6:52. doi: 10.1186/1750-1172-6-52.  [DOI](https://doi.org/10.1186/1750-1172-6-52) | [PMC free article](/articles/PMC3173287/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21801394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Orphanet%20J.%20Rare%20Dis.&title=Genome-wide%20association%20study%20of%20Stevens-Johnson%20Syndrome%20and%20Toxic%20Epidermal%20Necrolysis%20in%20Europe&author=E.%20G%C3%A9nin&author=M.%20Schumacher&author=J.C.%20Roujeau&author=L.%20Naldi&author=Y.%20Liss&volume=6&publication_year=2011&pages=52&pmid=21801394&doi=10.1186/1750-1172-6-52&)

53. Bateman A., UniProt Consortium UniProt: a worldwide hub of protein knowledge. Nucleic Acids Res. 2019;47(D1):D506–D515. doi: 10.1093/nar/gky1049.  [DOI](https://doi.org/10.1093/nar/gky1049) | [PMC free article](/articles/PMC6323992/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30395287/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=UniProt:%20a%20worldwide%20hub%20of%20protein%20knowledge&author=A.%20Bateman&volume=47&issue=D1&publication_year=2019&pages=D506-D515&pmid=30395287&doi=10.1093/nar/gky1049&)

54. Duan Q., Li H., Gao C., Zhao H., Wu S., Wu H., Wang C., Shen Q., Yin T. High glucose promotes pancreatic cancer cells to escape from immune surveillance via AMPK-Bmi1-GATA2-MICA/B pathway. J. Exp. Clin. Cancer Res. 2019;38:192. doi: 10.1186/s13046-019-1209-9.  [DOI](https://doi.org/10.1186/s13046-019-1209-9) | [PMC free article](/articles/PMC6518784/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31088566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Exp.%20Clin.%20Cancer%20Res.&title=High%20glucose%20promotes%20pancreatic%20cancer%20cells%20to%20escape%20from%20immune%20surveillance%20via%20AMPK-Bmi1-GATA2-MICA/B%20pathway&author=Q.%20Duan&author=H.%20Li&author=C.%20Gao&author=H.%20Zhao&author=S.%20Wu&volume=38&publication_year=2019&pages=192&pmid=31088566&doi=10.1186/s13046-019-1209-9&)

55. Shafi S., Vantourout P., Wallace G., Antoun A., Vaughan R., Stanford M., Hayday A. An NKG2D-mediated human lymphoid stress surveillance response with high interindividual variation. Sci. Transl. Med. 2011;3:113ra124. doi: 10.1126/scitranslmed.3002922.  [DOI](https://doi.org/10.1126/scitranslmed.3002922) | [PMC free article](/articles/PMC3966512/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22133594/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Transl.%20Med.&title=An%20NKG2D-mediated%20human%20lymphoid%20stress%20surveillance%20response%20with%20high%20interindividual%20variation&author=S.%20Shafi&author=P.%20Vantourout&author=G.%20Wallace&author=A.%20Antoun&author=R.%20Vaughan&volume=3&publication_year=2011&pages=113ra124&pmid=22133594&doi=10.1126/scitranslmed.3002922&)

56. Stanford S.M., Rapini N., Bottini N. Regulation of TCR signalling by tyrosine phosphatases: from immune homeostasis to autoimmunity. Immunology. 2012;137:1–19. doi: 10.1111/j.1365-2567.2012.03591.x.  [DOI](https://doi.org/10.1111/j.1365-2567.2012.03591.x) | [PMC free article](/articles/PMC3449242/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22862552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunology&title=Regulation%20of%20TCR%20signalling%20by%20tyrosine%20phosphatases:%20from%20immune%20homeostasis%20to%20autoimmunity&author=S.M.%20Stanford&author=N.%20Rapini&author=N.%20Bottini&volume=137&publication_year=2012&pages=1-19&pmid=22862552&doi=10.1111/j.1365-2567.2012.03591.x&)

57. Stanford S.M., Bottini N. PTPN22: the archetypal non-HLA autoimmunity gene. Nat. Rev. Rheumatol. 2014;10:602–611. doi: 10.1038/nrrheum.2014.109.  [DOI](https://doi.org/10.1038/nrrheum.2014.109) | [PMC free article](/articles/PMC4375551/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25003765/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Rheumatol.&title=PTPN22:%20the%20archetypal%20non-HLA%20autoimmunity%20gene&author=S.M.%20Stanford&author=N.%20Bottini&volume=10&publication_year=2014&pages=602-611&pmid=25003765&doi=10.1038/nrrheum.2014.109&)

58. Begovich A.B., Carlton V.E.H., Honigberg L.A., Schrodi S.J., Chokkalingam A.P., Alexander H.C., Ardlie K.G., Huang Q., Smith A.M., Spoerke J.M. A missense single-nucleotide polymorphism in a gene encoding a protein tyrosine phosphatase (PTPN22) is associated with rheumatoid arthritis. Am. J. Hum. Genet. 2004;75:330–337. doi: 10.1086/422827.  [DOI](https://doi.org/10.1086/422827) | [PMC free article](/articles/PMC1216068/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15208781/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=A%20missense%20single-nucleotide%20polymorphism%20in%20a%20gene%20encoding%20a%20protein%20tyrosine%20phosphatase%20(PTPN22)%20is%20associated%20with%20rheumatoid%20arthritis&author=A.B.%20Begovich&author=V.E.H.%20Carlton&author=L.A.%20Honigberg&author=S.J.%20Schrodi&author=A.P.%20Chokkalingam&volume=75&publication_year=2004&pages=330-337&pmid=15208781&doi=10.1086/422827&)

59. Cirulli E.T., Nicoletti P., Abramson K., Andrade R.J., Bjornsson E.S., Chalasani N., Fontana R.J., Hallberg P., Li Y.J., Lucena M.I., Drug-Induced Liver Injury Network (DILIN) investigators. International DILI consortium (iDILIC) A Missense Variant in PTPN22 is a Risk Factor for Drug-induced Liver Injury. Gastroenterology. 2019;156:1707–1716.e2. doi: 10.1053/j.gastro.2019.01.034.  [DOI](https://doi.org/10.1053/j.gastro.2019.01.034) | [PMC free article](/articles/PMC6511989/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30664875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=A%20Missense%20Variant%20in%20PTPN22%20is%20a%20Risk%20Factor%20for%20Drug-induced%20Liver%20Injury&author=E.T.%20Cirulli&author=P.%20Nicoletti&author=K.%20Abramson&author=R.J.%20Andrade&author=E.S.%20Bjornsson&volume=156&publication_year=2019&pages=1707-1716.e2&pmid=30664875&doi=10.1053/j.gastro.2019.01.034&)

60. Chen H., Hayashi G., Lai O.Y., Dilthey A., Kuebler P.J., Wong T.V., Martin M.P., Fernandez Vina M.A., McVean G., Wabl M. Psoriasis patients are enriched for genetic variants that protect against HIV-1 disease. PLoS Genet. 2012;8:e1002514. doi: 10.1371/journal.pgen.1002514.  [DOI](https://doi.org/10.1371/journal.pgen.1002514) | [PMC free article](/articles/PMC3343879/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22577363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Psoriasis%20patients%20are%20enriched%20for%20genetic%20variants%20that%20protect%20against%20HIV-1%20disease&author=H.%20Chen&author=G.%20Hayashi&author=O.Y.%20Lai&author=A.%20Dilthey&author=P.J.%20Kuebler&volume=8&publication_year=2012&pages=e1002514&pmid=22577363&doi=10.1371/journal.pgen.1002514&)

61. Winchester R., Giles J., Jadon D., Haroon M., McHugh N., FitzGerald O. Implications of the diversity of class I HLA associations in psoriatic arthritis. Clin. Immunol. 2016;172:29–33. doi: 10.1016/j.clim.2016.07.019.  [DOI](https://doi.org/10.1016/j.clim.2016.07.019) | [PMC free article](/articles/PMC5159216/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27455857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Immunol.&title=Implications%20of%20the%20diversity%20of%20class%20I%20HLA%20associations%20in%20psoriatic%20arthritis&author=R.%20Winchester&author=J.%20Giles&author=D.%20Jadon&author=M.%20Haroon&author=N.%20McHugh&volume=172&publication_year=2016&pages=29-33&pmid=27455857&doi=10.1016/j.clim.2016.07.019&)

62. Shenoy E.S., Macy E., Rowe T., Blumenthal K.G. Evaluation and Management of Penicillin Allergy: A Review. JAMA. 2019;321:188–199. doi: 10.1001/jama.2018.19283.  [DOI](https://doi.org/10.1001/jama.2018.19283) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30644987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Evaluation%20and%20Management%20of%20Penicillin%20Allergy:%20A%20Review&author=E.S.%20Shenoy&author=E.%20Macy&author=T.%20Rowe&author=K.G.%20Blumenthal&volume=321&publication_year=2019&pages=188-199&pmid=30644987&doi=10.1001/jama.2018.19283&)

63. Jani Y.H., Williams I., Krishna M.T. Sustaining and spreading penicillin allergy delabelling: A narrative review of the challenges for service delivery and patient safety. Br. J. Clin. Pharmacol. 2020;86:548–559. doi: 10.1111/bcp.14190.  [DOI](https://doi.org/10.1111/bcp.14190) | [PMC free article](/articles/PMC7080618/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31823385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Sustaining%20and%20spreading%20penicillin%20allergy%20delabelling:%20A%20narrative%20review%20of%20the%20challenges%20for%20service%20delivery%20and%20patient%20safety&author=Y.H.%20Jani&author=I.%20Williams&author=M.T.%20Krishna&volume=86&publication_year=2020&pages=548-559&pmid=31823385&doi=10.1111/bcp.14190&)

64. Sousa-Pinto B., Fonseca J.A., Gomes E.R. Frequency of self-reported drug allergy: A systematic review and meta-analysis with meta-regression. Ann. Allergy Asthma Immunol. 2017;119:362–373.e2. doi: 10.1016/j.anai.2017.07.009.  [DOI](https://doi.org/10.1016/j.anai.2017.07.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28779998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Allergy%20Asthma%20Immunol.&title=Frequency%20of%20self-reported%20drug%20allergy:%20A%20systematic%20review%20and%20meta-analysis%20with%20meta-regression&author=B.%20Sousa-Pinto&author=J.A.%20Fonseca&author=E.R.%20Gomes&volume=119&publication_year=2017&pages=362-373.e2&pmid=28779998&doi=10.1016/j.anai.2017.07.009&)

65. Mallal S., Nolan D., Witt C., Masel G., Martin A.M., Moore C., Sayer D., Castley A., Mamotte C., Maxwell D. Association between presence of HLA-B∗5701, HLA-DR7, and HLA-DQ3 and hypersensitivity to HIV-1 reverse-transcriptase inhibitor abacavir. Lancet. 2002;359:727–732. doi: 10.1016/s0140-6736(02)07873-x.  [DOI](https://doi.org/10.1016/s0140-6736(02)07873-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11888582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Association%20between%20presence%20of%20HLA-B%E2%88%975701,%20HLA-DR7,%20and%20HLA-DQ3%20and%20hypersensitivity%20to%20HIV-1%20reverse-transcriptase%20inhibitor%20abacavir&author=S.%20Mallal&author=D.%20Nolan&author=C.%20Witt&author=G.%20Masel&author=A.M.%20Martin&volume=359&publication_year=2002&pages=727-732&pmid=11888582&doi=10.1016/s0140-6736(02)07873-x&)

66. Chen P., Lin J.-J., Lu C.-S., Ong C.-T., Hsieh P.F., Yang C.-C., Tai C.-T., Wu S.-L., Lu C.-H., Hsu Y.-C., Taiwan SJS Consortium Carbamazepine-induced toxic effects and HLA-B∗1502 screening in Taiwan. N. Engl. J. Med. 2011;364:1126–1133. doi: 10.1056/NEJMoa1009717.  [DOI](https://doi.org/10.1056/NEJMoa1009717) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428768/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=Carbamazepine-induced%20toxic%20effects%20and%20HLA-B%E2%88%971502%20screening%20in%20Taiwan&author=P.%20Chen&author=J.-J.%20Lin&author=C.-S.%20Lu&author=C.-T.%20Ong&author=P.F.%20Hsieh&volume=364&publication_year=2011&pages=1126-1133&pmid=21428768&doi=10.1056/NEJMoa1009717&)

67. McCormack M., Alfirevic A., Bourgeois S., Farrell J.J., Kasperavičiūtė D., Carrington M., Sills G.J., Marson T., Jia X., de Bakker P.I.W. HLA-A∗3101 and carbamazepine-induced hypersensitivity reactions in Europeans. N. Engl. J. Med. 2011;364:1134–1143. doi: 10.1056/NEJMoa1013297.  [DOI](https://doi.org/10.1056/NEJMoa1013297) | [PMC free article](/articles/PMC3113609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=HLA-A%E2%88%973101%20and%20carbamazepine-induced%20hypersensitivity%20reactions%20in%20Europeans&author=M.%20McCormack&author=A.%20Alfirevic&author=S.%20Bourgeois&author=J.J.%20Farrell&author=D.%20Kasperavi%C4%8Di%C5%ABt%C4%97&volume=364&publication_year=2011&pages=1134-1143&pmid=21428769&doi=10.1056/NEJMoa1013297&)
