# Functional Measurement of CYP2C9 and CYP3A4 Allelic Polymorphism on Sildenafil Metabolism

## Metadata
**Authors:** Peng-fei Tang, Xiang Zheng, Xiao-xia Hu, Cheng-cheng Yang, Zhe Chen, Jian-chang Qian, Jian-ping Cai, Guo-xin Hu
**Journal:** Drug Design, Development and Therapy
**Date:** 2020 Nov 24
**DOI:** [10.2147/DDDT.S268796](https://doi.org/10.2147/DDDT.S268796)
**PMID:** 33262574
**PMCID:** PMC7699448
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7699448/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7699448/pdf/dddt-14-5129.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7699448/pdf/dddt-14-5129.pdf)

## Abstract

**Aim:** 
We aimed to systematically examine the effects of enzymatic activity of 38 human CYP2C9 alleles and 21 human CYP3A4 alleles, including wild-type CYP2C9.1 and CYP3A4.1, which contain the 24 CYP2C9 novel alleles (*36–*60) and 6 CYP3A4 novel alleles (*28–*34) newly found in the Chinese population, on sildenafil metabolism through in vitro experiment.

**Methods:** 
The recombinant cytochrome P450 alleles protein of CYP2C9 and CYP3A4 expressed in insect baculovirus expression system were reacted with 10–500 µM sildenafil for 30 minutes at 37°C, and the reaction was terminated by cooling to −80°C immediately. Next, we used ultra-performance liquid chromatography tandem mass spectrometry (UHPLC-MS/MS) detection system to detect sildenafil and its active metabolite N-desmethyl sildenafil.

**Results:** 
The intrinsic clearance (Vmax/Km) values of most CYP2C9 variants were significantly altered when compared with the wild-type CYP2C9*1, with most of these variants exhibiting either reduced Vmax and/or increased Km values. Four alleles (CYP2C9*11, *14, *31, *49) exhibited no markedly decreased relative clearance (1-fold). The relative clearance of the remaining thirty-three variants exhibited decrease in different levels, ranging from 1.81% to 88.42%. For the CYP3A4 metabolic pathway, when compared with the wild-type CYP3A4*1, the relative clearance values of four variants (CYP3A4*3, *10, *14 and *I335T) showed significantly higher relative clearance (130.7–134.9%), while five variants (CYP3A4*2, *5, *24, *L22V and *F113I) exhibited sharply reduced relative clearance values (1.80–74.25%), and the remaining nine allelic variants showed no statistical difference. In addition, the kinetic parameters of two CYP3A4 variants (CYP3A4*17 and CYP3A4*30) could not be detected, due to the defect of the CYP3A4 gene.

**Conclusion:** 
These findings were the first evaluation of all these infrequent CYP2C9 and CYP3A4 alleles for sildenafil metabolism; when treating people who carry these CYP2C9 and CYP3A4 variants, there should be more focus on the relation of dose intensity, side effects and therapeutic efficacy when administering sildenafil. The study will provide fundamental data on effect of CYP2C9 and CYP3A4 allelic variation on sildenafil metabolism for further clinical research.

Keywords: allelic variants, CYP2C9 polymorphisms, CYP3A4 polymorphisms, enzymatic activity, sildenafil, individual treatment

### Aim

We aimed to systematically examine the effects of enzymatic activity of 38 human *CYP2C9* alleles and 21 human *CYP3A4* alleles, including wild-type *CYP2C9.1* and *CYP3A4.1*, which contain the 24 *CYP2C9* novel alleles (**36–*60)* and 6 *CYP3A4* novel alleles (**28–*34*) newly found in the Chinese population, on sildenafil metabolism through in vitro experiment.

### Methods

The recombinant cytochrome P450 alleles protein of *CYP2C9* and *CYP3A4* expressed in insect baculovirus expression system were reacted with 10–500 µM sildenafil for 30 minutes at 37°C, and the reaction was terminated by cooling to −80°C immediately. Next, we used ultra-performance liquid chromatography tandem mass spectrometry (UHPLC-MS/MS) detection system to detect sildenafil and its active metabolite N-desmethyl sildenafil.

### Results

The intrinsic clearance (Vmax/Km) values of most *CYP2C9* variants were significantly altered when compared with the wild-type *CYP2C9*1*, with most of these variants exhibiting either reduced Vmax and/or increased Km values. Four alleles (*CYP2C9*11, *14, *31, *49*) exhibited no markedly decreased relative clearance (1-fold). The relative clearance of the remaining thirty-three variants exhibited decrease in different levels, ranging from 1.81% to 88.42%. For the *CYP3A4* metabolic pathway, when compared with the wild-type *CYP3A4*1*, the relative clearance values of four variants (*CYP3A4*3, *10, *14* and **I335T*) showed significantly higher relative clearance (130.7–134.9%), while five variants (*CYP3A4*2, *5, *24, *L22V* and **F113I*) exhibited sharply reduced relative clearance values (1.80–74.25%), and the remaining nine allelic variants showed no statistical difference. In addition, the kinetic parameters of two *CYP3A4* variants (*CYP3A4*17* and *CYP3A4*30*) could not be detected, due to the defect of the *CYP3A4* gene.

### Conclusion

These findings were the first evaluation of all these infrequent *CYP2C9* and *CYP3A4* alleles for sildenafil metabolism; when treating people who carry these CYP2C9 and CYP3A4 variants, there should be more focus on the relation of dose intensity, side effects and therapeutic efficacy when administering sildenafil. The study will provide fundamental data on effect of *CYP2C9* and *CYP3A4* allelic variation on sildenafil metabolism for further clinical research.

## Introduction

Sildenafil is a novel inhibitor of phosphodiesterase type 5 enzyme (PDE5). It can increase levels of cyclic guanosine monophosphate (cGMP) by inhibiting its breakdown via PDE5. Ultimately, intracellular calcium levels decrease by these pathways, thereby smooth muscle cells will be relaxed.[1](#cit0001),[2](#cit0002) PDE5 is mainly distributed in penile corpora cavernosa, which is the basis for the success of sildenafil in the treatment of erectile dysfunction (ED).[3](#cit0003) Sildenafil was the first oral medicine to treat male erectile dysfunction approved by the FDA. Additionally, numerous research has shown that besides the penis, PDE5 is also distributed in cardiac, brain, vasculature, central nervous system (CNS) and so on, which implies that sildenafil is effective for pulmonary arterial hypertension (PAH), acute mountain sickness (AMS), Raynaud’s disease and stroke.[4–6](#cit0004)

Sildenafil is extensively metabolized to its active metabolite, namely, N-desmethyl sildenafil ([Figure 1](#f0001)). Plasma concentrations of this metabolite are approximately 40% of those seen for sildenafil.[7](#cit0007) In vitro experiments showed that the N-desmethyl sildenafil has a similar selective characteristic for PDE enzymes to sildenafil, but for PDE5, the activity of metabolite is 2.5-fold lower compared with sildenafil.[8](#cit0008) Therefore, the metabolic rate of sildenafil affects the plasma drug concentrations, thus further leading to either reduced therapeutic efficiency or adverse drug reaction. Evidence from sildenafil metabolism in rat and human liver microsomes inferred that *CYP3A4* and *CYP2C9* are the major metabolic enzymes in the biotransformation.[8](#cit0008)

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f30f/7699448/b087b5b660c7/DDDT-14-5129-g0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7699448_DDDT-14-5129-g0001.jpg)

Chemical structure of the analytes and metabolic pathway of sildenafil.

The cytochromes *P450* (*CYPs*) are the most important drug metabolic enzymes in human liver, which are capable of catalyzing the oxidative biotransformation processes of ~90% current therapeutic drugs.[9](#cit0009) *CYP2C9* belongs to the *CYP2C* subfamily and constitutes ~20% of the *CYPs* protein content in human liver microsomes, and it is responsible for clearing ~15% of clinical drugs that undergo Phase I metabolic reaction.[10](#cit0010),[11](#cit0011) *CYP2C9* plays a major role in the hydroxylation metabolism processes of acidic or neutral drug, mainly including antihypertensives losartan, anticoagulants warfarin, anticonvulsants phenytoin, hypoglycemic agents glimepiride and tolbutamide, anti-inflammatory drugs diclofenac and flurbiprofen.[12](#cit0012) Like other *CYP2C* enzymes, *CYP2C9* manifests prominent genetic polymorphisms, and it can lead to wide interindividual variations in drug metabolism.[13](#cit0013) Up to now, a total of 60 allelic variants of *CYP2C9* have been reported ([http://www.cypalleles.ki.se/cyp2c9.htm](http://www.cypalleles.ki.se/cyp2c9.htm)). The most common allelic variants, *CYP2C9*2 (Arg144Cys)* and **3 (Ile359Leu)*, have been well studied a highly impact on its enzymatic activity both in clinical studies and in vitro.[14](#cit0014),[15](#cit0015) The *CYP2C9 *2* variant has a C>T transition at position 432 of exon 3 encoding arginine, causing an exchange of cysteine at position 144 (Arg144Cys) of *CYP2C9* protein, while the analysis of the variant *CYP2C9 *3* demonstrated A>T1077 transversion in exon 7 which encodes an amino acid exchanging isoleucine to leucine at position 359 (Ile359Leu).[16](#cit0016) The *CYP2C9*2* and *CYP2C9*3* allelic variants result in substantial decrease in *CYP2C9* enzyme activity due to reduced interaction with NADPH-CYP-reductase or with *CYP2C9* substrate, respectively.[17–19](#cit0017) These loss-of-function mutations of *CYP2C9* have been demonstrated to be less active in valproate metabolism than the wild type *CYP2C9*1*.[20](#cit0020) CYP2C9*2 and CYP2C9*3 alleles thus lead to genetically determined poor metabolism of valproic acid. A separate study on individuals identified with variants *CYP2C9 *2* and *CYP2C9 *3* showed significantly reduced enzymatic activity for phenytoin metabolism, as individuals who presented these variants had a reduction of 27–54% of the phenytoin metabolism, exhibiting a higher predisposition to phenytoin toxicity at the usual dosages.[21–23](#cit0021) The prevalence of these two alleles displays inter-ethnic differences. *CYP2C9*2* is almost absent in Asian and African populations, whereas it is present in 10–19% of Europeans. The frequency of *CYP2C9*3* is 3–6% in Asians, 1–5% in Africans and 6–9% in Europeans.[9](#cit0009),[24](#cit0024),[25](#cit0025) In a recent study, Dai et al sequenced all nine exons of *CYP2C9* in 2127 healthy Chinese people and discovered 37 sorts of new mutation sites, 21 of which could lead to amino acid coding change. Twenty-one sorts of new mutation sites have been nominated as new alleles *CYP2C9*36-*56* by Human *CYP450* Allele Nomenclature Committee.[26](#cit0026) In subsequent studies, Luo et al further analyzed the *CYP2C9* nine exons in previous 2127 healthy Chinese people and found three novel mutation positions that could result in amino acid variation, which have been named *CYP2C9*58 (P337T), CYP2C9*59 (I434F)* and *CYP2C9*60 (L467P)*.[27–29](#cit0027)

*CYP3A4* is another important and abundantly expressed *CYP450* drug-metabolizing enzyme, which belongs to the human *CYP3A* subfamily and constitutes ~30% of the total *CYPs* in human liver.[30](#cit0030) It is in charge of clearing approximately half of clinical drugs that experience phase I metabolic reaction, such as anticancer drugs (tamoxifen), Hypnotics (zolpidem), antibiotics (erythrocin), Hormone Drugs (prednisolone), anticonvulsants (carbamazepine), opioids (fentanyl), calcium channel blockers (nifedipine), antihistamines (loratadine) and so forth.[31–33](#cit0031) Previous research suggested that the gene polymorphism of *CYP3A4*, leads to up to 60-fold different expression levels and enzyme activity of *CYP3A4* between the individual human livers.[34](#cit0034) Finally, the different enzyme activity and expression levels may further lead to interindividual variations of substrate metabolism, which would finally result in treatment failure or adverse medicine reactions. Therefore, the relation of individual *CYP3A4* gene polymorphisms to metabolic effects of clinical drug will be meaningful for further studies.

Up to now, Human CYP Allele Nomenclature Committee has named 53 nonsynonymous mutations and 34 variants of *CYP3A4* ([https://www.pharmvar.org/gene/CYP3A4](https://www.pharmvar.org/gene/CYP3A4)). In past research, Hu et al systematically analyzed all thirteen exons’ genetic sequence of *CYP3A4* in 1114 healthy Chinese people and found five sorts of allelic variants (*1G, 4, 5, 18B, 23*), which have been reported in other populations.[35](#cit0035) In addition, seven novel allele variants were also identified and designated *CYP3A4*28−*34*. 25 The frequency of these new allele variants was detected at only 0.04%. Whereas, based on nearly 14 hundred million Chinese population, it makes sense to study the relationship of new allele variants and drug metabolism in guiding clinical medication.

However, little attention has been paid to the effect of *CYP3A4* and *CYP2C9* allele variants on sildenafil metabolism. In the present study, we systematically assessed the enzymatic activities of 38 *CYP2C9* and 20 *CYP3A4* allele variants, including wild-type and their novel identified variants, toward the metabolism of sildenafil, to provide evidence for the individual therapy of this important vasodilator in clinical application.

## Materials and Methods

### Chemicals and Reagents

Sildenafil (purity 99.0%), its metabolite N-desmethyl sildenafil (purity 99.0%), and the Internal Standard (IS) Midazolam (over 99% purity) were purchased from J&K Scientific Ltd. (Hong Kong, China). The NADPH-Na4 was bought from Roche (Basel, Switzerland). Wild-type and recombinant human cytochrome P450 microsomes (*CYP2C9* and *CYP3A4*) and *CYP b5* expressed in insect baculovirus expression system were acquired from Beijing Hospital (Beijing, China). We got mass spectrometry analytical–grade methanol, acetonitrile and other organic solvents from Merck (Darmstadt, Germany). The ultrapure water was gained via Merck Millipore Milli-Q Advantage 10 system (Billerica, MA, USA). All other materials and reagents used were acquired from Sunflower and Technology Ltd. (Beijing, China). Spodoptera frugiperda (*Sf*)21 insect cells were purchased from Invitrogen (Carlsbad, CA, USA).

### Microsomes’ Preparation

Wild-type and recombinant human *CYP2C9* and *CYP3A4* microsomes and purified cytochrome b5 were prepared according to previously reported method.[26](#cit0026),[36–38](#cit0036) In brief, cDNAs encoding *CYP2C9* and *CYP3A4* allelic variants were amplified by overlap extension PCR using wild-type *CYP2C9* and *CYP3A4* cDNAs as template, and ligated to the previously constructed dual expression baculovirus vector pFastBac-OR so as to obtain the ultimate expression vector pFastBac-OR-*CYP2C9* and pFastBac-OR-*CYP3A4*. To ensure that no errors were introduced during PCR amplification, all of the cDNA regions were confirmed by gene sequencing. We transfected the resulting pFastBac-OR-*CYP2C9* and *CYP3A4* plasmid into DH10Bac to get the corresponding recombinant Bacmid. The recombinant Bacmid containing fused *CYPOR* and *CYP2C9* or *CYP3A4* gene was confirmed by PCR and then transfected into *Sf21* cells. Three days after transfection, *Sf21* cells were collected. Then the target proteins were released from the cells by sonication and recombinant microsomes were further collected by differential centrifugation. Finally, the microsomal preparations were quantified by measured difference spectra.

### Instrumentation

We implemented the compound analysis by liquid chromatographic system of Waters ACQUITY UPLC I-Class Plus (Waters Technologies, Milford, MA, USA) coupled to a Waters Xevo TQD Mass Spectrometer (Waters Technologies), equipped with a vacuum degasser, an automatic injector, a column heating device and a binary gradient pump. Waters Masslynx 4.1 workstation (Waters Technologies) was used for data collection and instrument control.

### UPLC-MS/MS

N-desmethyl sildenafil and Midazolam (IS) were separated by an XBridge BEH C18 column (2.1 mm*50 mm, 1.7 µm particle size, Waters Corp.) at 40°C constant column temperature. The temperature of the automatic injector was maintained at 4°C. The mobile phase of gradient elution was made up of acetonitrile (B) and 0.1% formic acid (A) at 0.3 mL/min solvent flow, and the injection sample size was 2 µL. The optimization gradient elution process was performed as follows: 35% B (0–0.4 minutes), solvent B increased linearly 35–90% (0.4–0.8 minutes), maintained at 90% B (0.8–1.5 minutes), then solvent B decreased linearly to 35% (1.5–1.8 minutes), and finally maintained at 35% B (1.8–2.5 minutes). The running time of the whole elution process was 2.5 min. After complete detection of each sample, autosampler and pipeline access road were washed (cleaning procedure: 50% methanol in water, 0.3 mL/min, 2.5 min). Under previously mentioned chromatography separating quantitative conditions, retention time of N-desmethyl sildenafil and Midazolam (IS) were 0.73 and 0.85 minutes, respectively. The N-desmethyl sildenafil was quantified by a seven point standard curve.

The multiple reaction monitoring mode and positive electrospray ionization (ESI) of XEVO TQD triple-quadrupole MS spectrometer (Waters Technologies) were used to detect target products, optimized instrument parameters were as follows: ionization source temperature 150°C, desolvation temperature 500°C, capillary voltage 3500 V, and 1000 liter/h nitrogen was applied to evaporate the solvent. The ions of target compound detected by multiple reaction monitoring were m/z 461.0→283.0, m/z 326.2→291.0 for N-desmethyl sildenafil and Midazolam (IS), respectively. In addition, the collision energy for N-desmethyl sildenafil and Midazolam (IS) were set to 30, 40 V, respectively.

### Conditions and Methods for Incubation

The reaction system contained the following ingredients: 5 pmol wild-type CYP450 (*CYP2C9*1* and *CYP3A4*1*) microsomes or 5 pmol other recombinant CYP450 (*CYP2C9* and *CYP3A4*) microsomes, 5 pmol prepared CYP b5 (0.21 μg/μL, P450/b5 = 1:1) and 4.74 µL Sildenafil in 100mM Tris–HCl buffer (pH7.4). Sildenafil standard was dissolved in methanol, and working solutions of respective concentration levels prepared by methanol diluting. The final content of sildenafil in the reaction system was determined from 10–500 µM according to pretest study. First, the prepared incubation mixture was preincubated in a 37°C shaking incubator for five minutes. Then we added the NADPH regenerating system into the incubation mixture to initiate the reaction, and the final volume of 200 µL, that this reaction was last for 30 min at 37°C. Finally, the reaction was terminated by rapidly transferring to −80°C refrigerator. All experimental conditions about the sildenafil N-demethylation activity assay were based on linearity range with reaction time, substrate and microsome protein concentration respectively, incubations were repeated three times for each substrate concentration and the data are displayed as the mean ± S.D (standard deviation) from three experiments. 40 µL Midazolam (500 ng/mL) as the internal standard was added to the enzyme reaction mixture and then 400 µL acetonitrile was added for protein precipitation. Then the incubations were put under vortex movement for 2 min, and centrifuged to discard sediment at 14,000 rpm for 10 min at 4°C. We used ultrapure water 1:1 diluting the supernatant and then 2 µL of the processed mixture was injected into the UPLC-MS/MS instrument for following quantitative analysis of Sildenafil and its metabolite.

### Statistical Analysis

Michaelis-Menten analysis and enzyme kinetic parameters were performed by program Prism 8.0 (GraphPad Software Inc., SanDiego, CA, USA). The kinetic data of each CYP mutant allele recombinant microsome on sildenafil substrates are shown as mean ± S.D of 3 parallel tests. Differences among wild type and allelic mutants were compared with one-way ANOVA . Comparison between V_max_, K_m_ and C_lint_ of each variant was done with Dunnett’s test. All experimental results were analyzed with statistical analysis software SPSS version 17.0 (SPSS Inc., Chicago, IL, USA), and a p value lower than 0.05 means statistical significance.

## Results

To acquire the accurate catalytic activities of different genotypes *CYP2C9* on sildenafil metabolism, the catalytic activities of wild-type *CYP2C9*1* and 37 allelic variants were assessed in this study. Michaelis-Menten equation plots of sildenafil for *CYP2C9* wild-type and allelic variants are shown in [Figure 2](#f0002), and the details about the kinetic parameters of each variant are summarized in [Table 1](#t0001). As shown in [Figure 2](#f0002) and [Table 1](#t0001), the resulting enzyme kinetic parameters of Km, Vmax and CLint for sildenafil of wild type *CYP2C9*1* were 42.75 µM, 0.75 pmol/min/pmol, 17.46 µL/min/µmol. Almost all of the tested variants showed altered values in Vmax or Km compared with *CYP2C9*1*, which resulted in change of the CLint (Vmax/Km) values of these variants for sildenafil metabolism. Thirty-seven allelic variants could be divided into two groups according to their relative clearance for sildenafil compared with wild type: four variants (*CYP2C9*11, *14, *31, *49*) exhibited no significant difference by statistic analysis in CL_int_ value; the remaining thirty-three alleles (*CYP2C9*2, *3, *8, *13,*16, *19, *23,*27, *29, *34, *36, *37, *38, *39, *40, *41, *42, *43, *44, *45, *46, *47, *48, *50, *51, *52, *53, *54, *55, *56, *58, *59, *60*) exhibited significant reduction of CL_int_ values (1.81–88.42%). Five of the thirty-three variants (*CYP2C9*2,*23, *27, *29, *38*) showed higher or flat CL_int_ values (71.05–88.42% relative clearance) compared with the most common allelic variant *CYP2C9*2* (78.10% relative clearance), which showed approximately 20% declined activity for sildenafil metabolism in comparison with wild type, and was considered as the slight-reduction group; sixteen variants (*CYPC29*16, *34, *37, *39, *40, *41, *44, *46, *47, *48, *50, *51, *53, *54, *55, *56*) exhibited lower CLint values (20.05–60.63% relative clearance) than *CYP2C9*2*, but higher than another common allelic variant *CYP2C9*3* (18.89% relative clearance), and then, they could be considered as the moderate-reduction group; the remaining twelve variants (*CYP2C9*3, *8, *13, *19, *36, *42, *43, *45**52, *58, *59, *60*) exhibited lower CLint values (1.82–19.17% relative clearance) than CYP2C9*3 and could be considered as the significant-reduction group.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f30f/7699448/cd1497d969dc/DDDT-14-5129-g0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7699448_DDDT-14-5129-g0002.jpg)

Michaelis–Menten curves of the enzymatic activity of the recombinant wild-type CYP2C9*1 and 37 variants toward sildenafil N-demethylation (each point represents the mean ±SD of three parallel experiments).

### Table 1.

| Variants | Vmax (pmol/min/pmol) | Km (µM) | CLint (µL/min/µmol) | Relative Clearance (% of Wild Type) |
| --- | --- | --- | --- | --- |
| CYP2C9*1 | 0.75±0.02 | 42.75±2.45 | 17.46±0.45 | 100.00 |
| CYP2C9*2 | 0.88±0.05 | 64.75±6.99 | 13.71±0.79 | 78.10 |
| CYP2C9*3 | 0.30±0.01** | 89.98±3.22** | 3.30±0.07** | 18.89 |
| CYP2C9*8 | 0.35±0.00* | 153.73±0.06** | 2.25±0.01** | 12.87 |
| CYP2C9*11 | 1.13±0.01** | 60.71±0.60 | 18.55±0.23 | 106.20 |
| CYP2C9*13 | 0.11±0.01 | 155.23±12.01* | 0.74±0.02** | 4.21 |
| CYP2C9*14 | 0.96±0.13 | 57.48±22.69 | 17.72±3.96 | 100.72 |
| CYP2C9*16 | 0.47±0.00* | 45.96±8.98 | 10.58±0.12 | 60.64 |
| CYP2C9*19 | 0.19±0.06* | 280.63±114.29 | 0.70±0.08** | 3.96 |
| CYP2C9*23 | 0.54±0.01* | 42.57±1.45 | 12.60±0.13* | 72.09 |
| CYP2C9*27 | 0.92±0.02* | 59.77±1.89* | 15.44±0.29 | 88.42 |
| CYP2C9*29 | 0.96±0.20 | 85.60±48.25 | 12.49±3.53 | 71.05 |
| CYP2C9*31 | 0.76±0.03 | 39.48±3.94 | 19.30±1.18 | 110.66 |
| CYP2C9*34 | 0.39±0.03** | 111.70±8.74* | 3.51±0.16** | 20.05 |
| CYP2C9*36 | 0.07±0.00** | 224.97±12.59* | 0.32±0.00** | 1.82 |
| CYP2C9*37 | 0.51±0.09 | 73.53±33.54 | 7.69±2.54 | 41.00 |
| CYP2C9*38 | 0.31±0.03** | 20.47±5.50 | 15.59±3.01 | 87.04 |
| CYP2C9*39 | 0.28±0.03** | 46.46±14.72 | 6.24±1.44* | 34.91 |
| CYP2C9*40 | 1.17±0.12 | 140.93±26.79 | 8.36±0.69** | 47.83 |
| CYP2C9*41 | 1.13±0.12 | 178.47±52.69 | 6.63±1.40* | 37.11 |
| CYP2C9*42 | 0.07±0.00** | 83.50±31.53 | 0.89±0.25*** | 5.00 |
| CYP2C9*43 | 0.46±0.23 | 1,044.00±540.35 | 0.45±0.03** | 2.38 |
| CYP2C9*44 | 0.82±0.16 | 142.40±68.39 | 6.37±1.93 | 34.98 |
| CYP2C9*45 | 0.18±0.02*** | 54.15±13.77 | 3.39±0.54*** | 19.17 |
| CYP2C9*46 | 0.29±0.02** | 66.54±11.44 | 4.48±0.51*** | 25.62 |
| CYP2C9*47 | 0.25±0.04** | 52.47±15.16 | 4.91±0.69** | 27.95 |
| CYP2C9*48 | 0.70±0.06 | 193.07±33.91 | 3.68±0.29*** | 20.97 |
| CYP2C9*49 | 1.11±0.01** | 63.31±2.64 | 17.51±0.53 | 100.15 |
| CYP2C9*50 | 0.28±0.00** | 70.54±3.24* | 3.94±0.11** | 22.56 |
| CYP2C9*51 | 0.20±0.00** | 51.84±2.83 | 3.89±0.15** | 22.27 |
| CYP2C9*52 | 0.19±0.01** | 62.53±6.19 | 3.00±0.16** | 17.13 |
| CYP2C9*53 | 0.29±0.02** | 47.46±1.69 | 6.09±0.18** | 34.84 |
| CYP2C9*54 | 0.22±0.01** | 60.71±5.28 | 3.61±0.22** | 20.69 |
| CYP2C9*55 | 0.26±0.00** | 49.92±3.98 | 5.18±0.32*** | 29.64 |
| CYP2C9*56 | 0.37±0.02** | 76.22±7.41 | 4.83±0.20** | 27.64 |
| CYP2C9*58 | 0.41±0.02** | 129.13±12.93 | 3.18±0.15** | 18.24 |
| CYP2C9*59 | 0.10±0.00** | 231.80±33.88 | 0.45±0.04** | 2.54 |
| CYP2C9*60 | 0.25±0.04* | 156.97±58.23 | 1.64±0.27*** | 9.20 |

Table 1 Caption: Michaelis-Menten Kinetic Parameters for Demethylation Activities of Wild-Type CYP2C9*1 and 37 Mutant CYP2C9 Alleles Toward Sildenafil

As [Table 1](#t0001) shows, fourteen allelic variants had a lower Vmax value than wild-type (*CYP2C9*16, *23, *37, *39, *45, *46, *47, *50-*56*), and the value of Km having no obvious variation (1-fold), therefor these variants showed lower CLint than wild-type. Meanwhile, the CLint values of six variants (*CYP2C9*40, *41, *43, *44, *48, *58*) were lower compared with the wild type *CYP2C9*1* due to high level of Km values and Vmax values of these variants having no significant difference (1-fold). As a result, the Vmax values of ten variants (*CYP2C9*2, *14, *29, *31, *37, *40, *41, *43, *44, *48*) showed no significant difference compared with the CYP2C9*1, whereas the Km values of seven variants (*CYP2C9*3, *8, *13, *27, *34, *36, *50*) showed a significant difference compared with the *CYP2C9*1* (P<0.05). The CLint values of twenty-six variants for sildenafil metabolism (*CYP2C9*3, *8, *13, *19, *23, *34, *36, *39, *40, *41, *42, *43, *45, *46, *47, *48, *50-*56, *58, *59, *60*) showed significant difference compared with the *CYP2C9*1* (*P<0.05, **P<0.01, ***P<0.001).

Michaelis-Menten equation plots of sildenafil for *CYP3A4* wild-type and allelic variants are shown in [Figure 3](#f0003), and the details about the kinetic parameters of each variant are displayed in [Table 2](#t0002). The N-desmethyl sildenafil was not detected in *CYP3A4*17*, which indicated that the enzyme activity was too weak to result in any sildenafil metabolism. The resulting enzyme kinetic parameters of Km, Vmax and CLint for sildenafil of wild type *CYP3A4*1* were 64.02 µM, 9.29 pmol/min/pmol, 0.15 µL/min/µmol. As shown in [Table 2](#t0002), except for the variant of CYP3A4*17, almost all of the allelic variants showed changed Vmax values compared with the wild-type, the five variants (*CYP3A4*5, *24, *L22V, *H324Q and *F113I*) exhibited decreased Vmax values (36.1–98.8%), the remaining thirteen variants displayed increased Vmax values compared to CYP3A4*1 (229.3–106.2%), *CYP3A4*14* and **F113I* are the only variants that had no statistical difference with the wild-type on Vmax values. Meanwhile, most *CYP3A4* allelic variants presented significant alterations of values in Km: eight variants (*CYP3A4*2, *5, *9, *11, *23, *24, *I427V and *F113I*) showed considerably higher Km values (1.11–20.58 fold) compared with *CYP3A4*1*, while only two variants, *CYP3A4*H32Q* and **14*, exhibited decreased Km values: 80.7% and 83.7%, and the remaining eight variants showed no statistical difference in Km value. Therefore, based on the CLint values of sildenafil metabolism, allelic variants were classified into three categories. Four variants (*CYP3A4*3, *10, *14* and **I335T*) showed significantly higher CLint values (130.7–134.9%) compared with the *CYP3A4*1*. Five variants (*CYP3A4*2, *5, *24, *L22V and *F113I*) exhibited sharply reduced CLint values (1.80–74.25%) compared with the *CYP3A4*1*, particularly, *CYP3A4*24* whose CLint values were extremely decreased to 1.80% compared with the CYP3A4*1. The remaining nine allelic variants (*CYP3A4*4, *9, *11, *15, *19, *23, *H324Q, *I427V* and **A370S*) exhibited no statistical difference in CLint values compared with the *CYP3A4*1*.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f30f/7699448/13307d596281/DDDT-14-5129-g0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7699448_DDDT-14-5129-g0003.jpg)

Michaelis–Menten curves of the enzymatic activity of the recombinant wild-type CYP3A4*1 and 22 variants toward sildenafil N-demethylation (each point represents the mean ±SD of three parallel experiments).

### Table 2.

| Variants | Vmax (pmol/min/pmol) | Km (µM) | CLint (µL/min/µmol) | Relative Clearance (% of Wild Type) |
| --- | --- | --- | --- | --- |
| CYP3A4*1 | 9.29±0.45 | 64.02±5.07 | 0.15±0.02 | 100.03 |
| CYP3A4*2 | 13.39±0.43*** | 169.77±11.26*** | 0.08±0.00** | 55.54 |
| CYP3A4*3 | 10.68±0.17** | 56.99±3.79 | 0.19±0.01* | 132.00 |
| CYP3A4*4 | 10.91±0.10** | 71.13±1.20 | 0.15±0.00 | 107.79 |
| CYP3A4*5 | 6.01±0.43** | 147.87±26.30** | 0.04±0.00** | 28.91 |
| CYP3A4*9 | 10.82±0.14** | 85.07±6.63* | 0.13±0.01 | 89.53 |
| CYP3A4*10 | 12.88±0.15*** | 67.05±1.03 | 0.19±0.00* | 134.97 |
| CYP3A4*11 | 21.31±0.74*** | 167.87±16.17*** | 0.13±0.01 | 89.56 |
| CYP3A4*14 | 9.87±0.20 | 53.13±3.69* | 0.19±0.01* | 130.70 |
| CYP3A4*15 | 10.06±0.10* | 62.29±3.60 | 0.16±0.01 | 113.64 |
| CYP3A4*17 | ND | ND | ND | ND |
| CYP3A4*19 | 10.65±0.29* | 62.22±2.04 | 0.17±0.00 | 120.34 |
| CYP3A4*23 | 11.33±0.30** | 76.10±4.29* | 0.15±0.00 | 104.76 |
| CYP3A4*24 | 3.36±0.56*** | 1,317.67±298.59** | 0.00±0.00*** | 1.80 |
| CYP3A4*28 | 5.91±0.13*** | 68.89±5.69 | 0.09±0.01** | 60.47 |
| CYP3A4*29 | 9.18±0.03 | 86.90±2.89** | 0.11±0.00* | 74.25 |
| CYP3A4*30 | ND | ND | ND | ND |
| CYP3A4*31 | 7.49±0.02** | 51.67±3.65* | 0.15±0.01 | 102.20 |
| CYP3A4*32 | 13.20±0.31*** | 69.34±2.67 | 0.19±0.00* | 133.84 |
| CYP3A4*33 | 10.45±0.33* | 60.64±3.97 | 0.17±0.01 | 121.15 |
| CYP3A4*34 | 11.78±0.40** | 77.36±5.91* | 0.15±0.01 | 107.11 |

Table 2 Caption: Michaelis-Menten Kinetic Parameters for Demethylation Activities of Wild-Type CYP3A4*1 and 22 Mutant CYP3A4 Alleles Toward Sildenafil

## Discussion

Owing to the fact that the carriers of these low-frequency *CYP450* mutations are hard to find, it is difficult to carry out metabolism evaluations in vivo; therefore, targeted drug in vitro metabolic study is a great way to identifythe metabolic enzyme activities of variation individual. The evidence so far suggests that genetic polymorphisms in *CYP450* is a main reason of individual variation in drug reactions, perhaps resulting in the problems of medications’ effectiveness and adverse reactions. So, our research is crucially important for reflecting complicated pharmacogenetics in clinical individualized medication. Sildenafil has robust efficacy in vasodilatation; patients with defective *CYP450* on long-term treatment with sildenafil may have higher risks of ADRs (adverse drug reaction), such as renal toxicity and induce or worsen heart. Precision medicine can achieve clinical individualized medication based on enzyme metabolic abilities of each *CYP450* genotype. Patients with increased *CYP450* enzymatic activity should up their doses, while patients with decreased enzymatic activity should use smaller doses or extend dosing interval.

To further investigate the effect of the *CYP2C9* allele on the metabolism of sildenafil, the recombinant insect microsomes were utilized to screen the 38 *CYP2C9* variants. Among these variants, some exhibited lower enzyme activity than wild-type *CYP2C9*1*, in agreement with previous research findings, while some led to no change in enzyme activity, a finding that was not corroborated by previous studies.

*CYP2C9*2* and *CYP2C9*3*, two widely investigated variants, are present in approximately 8–14% and 10–15% of Caucasian individuals, but are not as prevalent in Asian and black populations. The amino acid change Arg144Cys exists on the external surface of the protein *CYP2C9*2*. Using recombinant insect microsomes, Wang et al and Xia et al observed that *CYP2C9*2(R144C)* had lower CLint values for flurbiprofen and diclofenac, about 61.36% and 89.48% of wild type, respectively.[39](#cit0039),[40](#cit0040) Compared with *CYP2C9*1*, its enzyme activity was only slightly reduced. In this study, *CYP2C9*2* resulted in a small increase in Vmax (1.2 times) and Km (1.5 times) for the catalysis of sildenafil, thereby leading to a small decrease in clearance rate. This finding was consistent with previous studies. Our research also adds to previous empirical evidence by showing that, compared to wild-type *CYP2C9*1, CYP2C9*3* sildenafil Clint was significantly reduced. Blaisdell et al demonstrated that compared to wild-type *CYP2C9*1*, there was a marked decrease in enzyme activity in tolbutamide metabolism for *CYP2C9*8 (R150H)*. However, Wang et al and Pan et al found that *CYP2C9*8* was associated with lower CLint values for flurbiprofen and carvedilol.[39](#cit0039),[41](#cit0041),[42](#cit0042) In our study, we found that *CYP2C9*8* showed significant reduction in Vmax (46%) and 3.6-fold higher Km, which caused a significantly lower clearance rate for sildenafil. Apparent discrepancies of *CYP2C9*8* might be explained by the R150H substitution altering an essential interaction between the *CYPs* enzyme and the NADPH-*CYP450* reductase.

Some studies on tolbutamide and carvedilol metabolism have shown that compared with the wild-type *CYP2C9*1*, the *CYP2C9*11 (R335W)* allele is linked to a decrease in enzyme activity.[28](#cit0028),[29](#cit0029) However, in some other studies, it has been noted that *CYP2C9*11* may have little to do with the dosage requirements of warfarin, and that it shows a significant increase in the enzyme activity of bosentan metabolism.[43](#cit0043),[44](#cit0044) In this study, the catalytic activity of the *CYP2C9*11* allele on sildenafil was not significantly different from that of the wild type. First identified in a Chinese poor metabolizer of lornoxicam, *CYP2C9*13 (Leu90Pro)* is a variant that occurs among roughly 2% of the Chinese population.[45](#cit0045),[46](#cit0046) The catalytic activity of *CYP2C9*13* on diclofenac and tolbutamide decreases in vitro.[47](#cit0047) In our study, *CYP2C9*13* had lower Vmax, higher Km, and lower CLint enzyme activity than the *CYP2C9*1* allele. However, this non-synonymous mutation of *CYP2C9* was located in the non-heme binding region and is far away from the substrate binding pocket. Therefore, it is unclear why there was *CYP2C9*13*-mediated reduction in catalytic activity.

*CYP2C9*14*, *CYP2C9*16*, and *CYP2C9*19* were all first discovered in Southeast Asians,[47](#cit0047) and the CLint of tolbutamide metabolism appears to be moderately or significantly reduced in vitro.[48](#cit0048) However, in this study, there was no significant difference in the CLint value of *CYP2C9*14* for sildenafil. Compared with the wild type, only the CLint value of sildenafil for *CYP2C9*19* was significantly reduced (3.9%), which resulted from a 3.9-fold reduction in Vmax, a 6.5-fold increase in Km value, and a 25.3-fold reduction in CLint. These findings are consistent with those of previous studies.

In spite of the fact that the enzymatic function of *CYP2C9*23* was not ascertained from Veenstra et al’s study,[49](#cit0049) our study showed that allelic variation leads to a reduction in the clearance of sildenafil, while Km has no significant change. *CYP2C9*27 (Arg150Leu)* and *CYP2C9*29 (Pro279Thr)* were originally found in the Japanese population, but compared with the *CYP2C9*1*, these two alleles showed similar enzymatic activities against diclofenac.[50](#cit0050) In our study, there was no significant change in the clearance rate of sildenafil by these two alleles, which is consistent with the results of previous studies. *CYP2C9*31* was considered to be a functionally defective allelic variant,[51](#cit0051) but our research shows that there was no significant change in the clearance rate of *CYP2C9*31*. *CYP2C9*34* was first discovered in 724 Japanese subjects. Compared with wild type *CYP2C9*1*, the catalytic activity of *CYP2C9*34* showed no significant change.[52](#cit0052) Our results discovered that *CYP2C9*34* significantly reduced the CLint value of sildenafil metabolism.

*CYP2C9*36* has an M1V amino acid substitution that results from 1A>G nucleotide substitution. This alteration leads to a decrease in the expression level of *CYP2C9* protein. In this study, *CYP2C9*36* resulted in a significant decrease in Vmax of Sildenafil metabolism (9.3%) and a significant increase in Km value (5-fold), leading to a significant reduction in clearance rate. Therefore, we infer that *CYP2C9*36* will have an effect on its enzymatic activity due to its low protein expression. Our results indicated that *CYP2C9*42(R124Q)* and *CYP2C9*43(R124W)* also significantly reduced the CLint of sildenafil in vitro, a finding consistent with the study by Dai et al.[36](#cit0036) Our results demonstrated that compared with wild type *CYP2C9*1, CYP2C9*37, *39, *41, *46, *47, *48, *50, *51, *52, *53, *55* and **56* showed lower CLint values for sildenafil in vitro. Similarly, these findings were also corroborated by Dai et al.[36](#cit0036) However, when sildenafil was used as a substrate, the relative catalytic activity between these *CYP2C9* allelic variants appeared to be completely different. For example, *CYP2C9*40(F110S)* and *CYP2C9*54(S343R)* are both predicted to be rapidly metabolizing allele variants,[36](#cit0036) but our results showed that the activity of either of them was reduced. *CYP2C9*38(G96A)* and *CYP2C9*49(I222V)* showed impaired *CYP2C9* activity in the study by Dai et al.[53](#cit0053) However, we found that it had no effect on the enzyme activity of sildenafil metabolism compared with wild-type *CYP2C9*1*. The two variants *CYP2C9*44 (T130M)* and *CYP2C9*45 (R132W)* are similar new coding allelic variants, which were first discovered in Chinese populations and showed similar reduced activity in vitro.[36](#cit0036) In contrast, our results showed that the CLint value was much lower for *CYP2C9*45* than for *CYP2C9*44*.

For the newly discovered variants *CYP2C9*58 (P337T)*, *CYP2C9*59 (I434F)*, and *CYP2C9*60 (L467P)*, all showed significantly lower CLint. We believe that the different characteristics of these substrates might be the main reason for this discrepancy of enzymatic activity, because previous research has indicated that decrease in the CLint of nine various substrates could change from 3-fold for diclofenac 4-hydroxylation to 27-fold for piroxicam 5-hydroxylation in vitro for the typical allele variant *CYP2C9*3*.[36](#cit0036)

We carefully analyzed 22 *CYP3A4* variants and the wild-type *CYP3A4*1* to display reliable and complete data with regard to the effects of *CYP3A4* allele on the metabolism in vitro of sildenafil. The metabolic activity of wild-type *CYP3A4*1* was chosen as the positive contrast. Dai et al constructed the *CYP3A4 *17* and identified the enzymatic activity by metabolizing chlorpyrifos and testosterone in an Escherichia coli system, results showed that *CYP3A4*17* had an F189S substitution in exon 7 that caused decreased metabolic activity of chlorpyrifos and testosterone compared with the wild type.[54](#cit0054) Our results were consistent with past research, *CYP3A4*17* exhibited remarkably low enzymatic activity toward sildenafil that no metabolites had been detected. Similar to *CYP3A4 *17*, the novel allelic variant *CYP3A4*30* had premature termination codon mutation that produces truncated protein, resulting in the metabolites of *CYP3A4*30* not being detectable and it means that the *CYP3A4*30* had no metabolic activity on sildenafil. Therefore, people who carry these two *CYP3A4* alleles can be categorized as PMs (poor metabolizers) for sildenafil and much more attention should be paid to sildenafil-caused adverse reactions.

Hsieh et al sequenced all thirteen exons of *CYP3A4* in 102 Taiwanese Chinese people and discovered three sorts of new mutation sites including *CYP3A4*4* and *CYP3A4*5*.[55](#cit0055) By studying the ratio of free cortisol to 6beta-hydroxycortisol, it was confirmed that these mutations obviously decreased the activity of *CYP3A4*. But for the sildenafil metabolism, we discovered only *CYP3A4*5* exhibited reduced activity, the *CYP3A4*4* showed no statistical difference in CLint values compared with the wild type.

Eiselt et al indicated that compared with *CYP3A4*1*, *CYP3A4*9* and *CYP3A4*10* exhibited no obviously different catalytic activity in steroid hydroxylase, whereas the catalytic activity of *CYP3A4*11* was decreased significantly.[56](#cit0056) In our study, we found that *CYP3A4*9* and *CYP3A4*11* showed no statistical difference in CLint values, while *CYP3A4*10* exhibited slight increment in Vmax (1.38-fold), which caused a higher clearance rate for sildenafil. For apparent *CYP3A4*10* and **11* difference might be interpreted by the activity of substrate-dependent, D174H and T363M substitution potentially leading to a change in an essential interaction between NADPH and *CYP3A4* in oxidative drug metabolism.

The *CYP3A4*2 (S222P)* and *CYP3A4*3 (M445T)* variant alleles make up approximately 2.7% and 4% of Caucasians, respectively.[57](#cit0057) Miyazaki et al’s research showed reduced metabolic activity toward midazolam, testosterone and nifedipine in the Escherichia coli protein expression system.[58](#cit0058) In the present study, in agreement with previous research, *CYP3A4*2* exhibited a significantly higher Km and slightly higher Vmax, which induced lower CLint (55.54%) compared with wild-type. For *CYP3A4*3 (M445T)* carried genetic mutation of 1334T>C in exon twelve, which is located in heme binding section, and may promote the heme binding of *CYP3A4*.[57](#cit0057) Consequently, compared with *CYP3A4*1*, *CYP3A4*3* exhibited a slight increment in CL_int_ value (1.32-fold of *CYP3A4*1*) with increased Vmax and reduced Km value.

Two variations, *CYP3A4*14* and **15*, were first detected in liver donors (fifty-three Caucasians and twenty-one African-Americans).[59](#cit0059) The allelic frequency of *CYP3A*14 (L15P)* in different races was between 3% and 4.2%, the allelic frequency of *CYP3A*15 (R162Q)* was 2%, but only detected in Black populations.[59](#cit0059) Lamba et al used midazolam as substrate to measure the enzyme activities of *CYP3A4*14* and *CYP3A4*15*, and found metabolic activity of midazolam hydroxylation was not substantially altered compared with wild-type.[59](#cit0059) Our study showed that *CYP3A4*14* showed higher CLint values of sildenafil (130.7%) in vitro, whereas *CYP3A4*15* exhibited no obvious changes.

Drögemöller et al sequenced all the coding exons, flanking intronic regions of *CYP3A4* in 159 South Africans from 3 South African population groups, and detected 2 novel non-synonymous variants (*R162W and Q200H*), named *CPY3A4*23* and *CPY3A4*24*.[60](#cit0060) Our previous study also identified *CYPY3A4*23*, for the first time in Chinese populations.[35](#cit0035) Although the allelic variant function of *CYP2C9*23* was not revealed by Veenstra’s study (2005), our research showed that *CPY3A4*24* caused a massive reduction in clearance value (close to 0% of wild type) with significantly decreased Vmax (36%) and sharply increased Km (20-fold), relative to wild type, whereas *CYP2C9*23* showed no significant changes in enzymatic functions.

In the present study, *CYP3A4*32 (I335T)* exhibited a slightly increased Vmax value (1.42-fold of wild-type) and the km value was not increased significantly, which caused a higher CLint rate for sildenafil compared with wild-type in vitro. Our result showed that *CYP3A4*28* and *CYP3A4*29* exhibited various degrees of decreased relative clearance rate for sildenafil with a lower Vmax value but a higher Km value when compared with *CYP3A4*1*. The remaining three allelic variants (*CYP3A4*31, CYP3A4*33* and *CYP3A4*34*) exhibited both Vmax and Km values pointing toward the same trend, as the metabolic activities of these variants for sildenafil showed no significant change compared with *CYP3A4*1*. These results are not consistent with our previous study for amiodarone, lidocaine and Ibrutinib.[37](#cit0037),[38](#cit0038),[61](#cit0061) We thought the metabolic activities among *CYP3A4* variants were influenced by affinity differences when using different substrates.

## Conclusion

In summary, the present study provides another way to analyze drug metabolism in vitro, and functionally assessed the enzymatic activity of wild-type, 37 variants of *CYP2C9* and 20 variants of *CYP3A4* in the metabolism of sildenafil, including 30 recently detected new coding variants. For all we know, this is the first academic thesis to systematically study all these rare allele variants for sildenafil metabolism. Although gene frequency of these rare *CYP* genetic variations in the Chinese population are minimum 0.1% and maximum 1.8%, against the background of more than 1.4 billion people in China, getting accurate data for rare *CYP* alleles in the Chinese population and exploring effects of alleles on drug metabolism and activity, will be valuable and important in clinical treatment. These data provide new insights about *CYP2C9* and *CYP3A4* gene polymorphisms and their impact on enzymatic activity, offer better comprehension of pharmacogenomics that will help to understand diversity of sildenafil dose–response relationship, and may contribute to offer treatment in an effective and safe manner, and these results can be used as the theoretical basis for individualized medication in Chinese populations.

## Acknowledgments

The authors thank the members of the Beijing Institute of Geriatrics of the Ministry of Health for their advice and assistance. This work was supported by National Natural Science Foundation of China (NSFC 81973397) and supported by grants from the Jinhua Science and Technology Bureau (No. 2019jzk014). In addition, Peng-fei Tang wants to thank Xue-meng Wu, in particular, for the care and support received during the process of writing and in life.

## Disclosure

The authors report no conflicts of interest in this work.

## References

1. Eardley I, Ellis P, Boolell M, Wulff M. Onset and duration of action of sildenafil for the treatment of erectile dysfunction. Br J Clin Pharmacol. 2002;53(Suppl 1\(Suppl 1)):61s–65s. doi: 10.1046/j.0306-5251.2001.00034.x  [DOI](https://doi.org/10.1046/j.0306-5251.2001.00034.x) | [PMC free article](/articles/PMC1874251/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11879261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Onset%20and%20duration%20of%20action%20of%20sildenafil%20for%20the%20treatment%20of%20erectile%20dysfunction&author=I%20Eardley&author=P%20Ellis&author=M%20Boolell&author=M%20Wulff&volume=53&issue=Suppl%201%5C(Suppl%201)&publication_year=2002&pages=61s-65s&pmid=11879261&doi=10.1046/j.0306-5251.2001.00034.x&)

2. Francis SH, Corbin JD. Sildenafil: efficacy, safety, tolerability and mechanism of action in treating erectile dysfunction. Expert Opin Drug Metab Toxicol. 2005;1(2):283–293. doi: 10.1517/17425255.1.2.283  [DOI](https://doi.org/10.1517/17425255.1.2.283) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16922643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Metab%20Toxicol&title=Sildenafil:%20efficacy,%20safety,%20tolerability%20and%20mechanism%20of%20action%20in%20treating%20erectile%20dysfunction&author=SH%20Francis&author=JD%20Corbin&volume=1&issue=2&publication_year=2005&pages=283-293&pmid=16922643&doi=10.1517/17425255.1.2.283&)

3. Leiblum SR. After sildenafil: bridging the gap between pharmacologic treatment and satisfying sexual relationships. J Clin Psychiatry. 2002;63(Suppl 5):17–22; discussion 23–15.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11964137/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychiatry&title=After%20sildenafil:%20bridging%20the%20gap%20between%20pharmacologic%20treatment%20and%20satisfying%20sexual%20relationships&author=SR%20Leiblum&volume=63&issue=Suppl%205&publication_year=2002&pmid=11964137&)

4. Fazan R Jr, Huber DA, Silva CAA, Dias da Silva VJ, Salgado MCO, Salgado HC. Sildenafil acts on the central nervous system increasing sympathetic activity. J Appl Physiol. 2008;104(6):1683–1689. doi: 10.1152/japplphysiol.01142.2007  [DOI](https://doi.org/10.1152/japplphysiol.01142.2007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18388248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Physiol&title=Sildenafil%20acts%20on%20the%20central%20nervous%20system%20increasing%20sympathetic%20activity&author=R%20Fazan&author=DA%20Huber&author=CAA%20Silva&author=VJ%20Dias%20da%20Silva&author=MCO%20Salgado&volume=104&issue=6&publication_year=2008&pages=1683-1689&pmid=18388248&doi=10.1152/japplphysiol.01142.2007&)

5. Bocchi EA, Guimarães G, Mocelin A, Bacal F, Bellotti G, Ramires JF. Sildenafil effects on exercise, neurohormonal activation, and erectile dysfunction in congestive heart failure: a double-blind, placebo-controlled, randomized study followed by a prospective treatment for erectile dysfunction. Circulation. 2002;106(9):1097–1103. doi: 10.1161/01.CIR.0000027149.83473.B6  [DOI](https://doi.org/10.1161/01.CIR.0000027149.83473.B6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12196335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Sildenafil%20effects%20on%20exercise,%20neurohormonal%20activation,%20and%20erectile%20dysfunction%20in%20congestive%20heart%20failure:%20a%20double-blind,%20placebo-controlled,%20randomized%20study%20followed%20by%20a%20prospective%20treatment%20for%20erectile%20dysfunction&author=EA%20Bocchi&author=G%20Guimar%C3%A3es&author=A%20Mocelin&author=F%20Bacal&author=G%20Bellotti&volume=106&issue=9&publication_year=2002&pages=1097-1103&pmid=12196335&doi=10.1161/01.CIR.0000027149.83473.B6&)

6. Galiè N, Ghofrani HA, Torbicki A, et al. Sildenafil citrate therapy for pulmonary arterial hypertension. N Engl J Med. 2005;353(20):2148–2157. doi: 10.1056/NEJMoa050010  [DOI](https://doi.org/10.1056/NEJMoa050010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16291984/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Sildenafil%20citrate%20therapy%20for%20pulmonary%20arterial%20hypertension&author=N%20Gali%C3%A8&author=HA%20Ghofrani&author=A%20Torbicki&volume=353&issue=20&publication_year=2005&pages=2148-2157&pmid=16291984&doi=10.1056/NEJMoa050010&)

7. Walker DK, Ackland MJ, James GC, et al. Pharmacokinetics and metabolism of sildenafil in mouse, rat, rabbit, dog and man. Xenobiotica. 1999;29(3):297–310. doi: 10.1080/004982599238687  [DOI](https://doi.org/10.1080/004982599238687) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10219969/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Pharmacokinetics%20and%20metabolism%20of%20sildenafil%20in%20mouse,%20rat,%20rabbit,%20dog%20and%20man&author=DK%20Walker&author=MJ%20Ackland&author=GC%20James&volume=29&issue=3&publication_year=1999&pages=297-310&pmid=10219969&doi=10.1080/004982599238687&)

8. Hyland R, Roe EG, Jones BC, Smith DA. Identification of the cytochrome P450 enzymes involved in the N-demethylation of sildenafil. Br J Clin Pharmacol. 2001;51(3):239–248. doi: 10.1046/j.1365-2125.2001.00318.x  [DOI](https://doi.org/10.1046/j.1365-2125.2001.00318.x) | [PMC free article](/articles/PMC2015027/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11298070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Identification%20of%20the%20cytochrome%20P450%20enzymes%20involved%20in%20the%20N-demethylation%20of%20sildenafil&author=R%20Hyland&author=EG%20Roe&author=BC%20Jones&author=DA%20Smith&volume=51&issue=3&publication_year=2001&pages=239-248&pmid=11298070&doi=10.1046/j.1365-2125.2001.00318.x&)

9. Zanger UM, Schwab M. Cytochrome P450 enzymes in drug metabolism: regulation of gene expression, enzyme activities, and impact of genetic variation. Pharmacol Ther. 2013;138(1):103–141. doi: 10.1016/j.pharmthera.2012.12.007  [DOI](https://doi.org/10.1016/j.pharmthera.2012.12.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23333322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Ther&title=Cytochrome%20P450%20enzymes%20in%20drug%20metabolism:%20regulation%20of%20gene%20expression,%20enzyme%20activities,%20and%20impact%20of%20genetic%20variation&author=UM%20Zanger&author=M%20Schwab&volume=138&issue=1&publication_year=2013&pages=103-141&pmid=23333322&doi=10.1016/j.pharmthera.2012.12.007&)

10. Van Booven D, Marsh S, McLeod H, et al. Cytochrome P450 2C9-CYP2C9. Pharmacogenet Genomics. 2010;20(4):277–281.  [DOI](https://doi.org/10.1097/FPC.0b013e3283349e84) | [PMC free article](/articles/PMC3201766/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20150829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Cytochrome%20P450%202C9-CYP2C9&author=D%20Van%20Booven&author=S%20Marsh&author=H%20McLeod&volume=20&issue=4&publication_year=2010&pages=277-281&pmid=20150829&doi=10.1097/FPC.0b013e3283349e84&)

11. Schwarz UI. Clinical relevance of genetic polymorphisms in the human CYP2C9 gene. Eur J Clin Invest. 2003;33(Suppl 2):23–30. doi: 10.1046/j.1365-2362.33.s2.6.x  [DOI](https://doi.org/10.1046/j.1365-2362.33.s2.6.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14641553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Invest&title=Clinical%20relevance%20of%20genetic%20polymorphisms%20in%20the%20human%20CYP2C9%20gene&author=UI%20Schwarz&volume=33&issue=Suppl%202&publication_year=2003&pages=23-30&pmid=14641553&doi=10.1046/j.1365-2362.33.s2.6.x&)

12. Zhou SF, Zhou ZW, Huang M. Polymorphisms of human cytochrome P450 2C9 and the functional relevance. Toxicology. 2010;278(2):165–188. doi: 10.1016/j.tox.2009.08.013  [DOI](https://doi.org/10.1016/j.tox.2009.08.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19715737/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicology&title=Polymorphisms%20of%20human%20cytochrome%20P450%202C9%20and%20the%20functional%20relevance&author=SF%20Zhou&author=ZW%20Zhou&author=M%20Huang&volume=278&issue=2&publication_year=2010&pages=165-188&pmid=19715737&doi=10.1016/j.tox.2009.08.013&)

13. Wang B, Wang J, Huang SQ, Su HH, Zhou SF. Genetic polymorphism of the human cytochrome P450 2C9 gene and its clinical significance. Curr Drug Metab. 2009;10(7):781–834. doi: 10.2174/138920009789895480  [DOI](https://doi.org/10.2174/138920009789895480) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19925388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Metab&title=Genetic%20polymorphism%20of%20the%20human%20cytochrome%20P450%202C9%20gene%20and%20its%20clinical%20significance&author=B%20Wang&author=J%20Wang&author=SQ%20Huang&author=HH%20Su&author=SF%20Zhou&volume=10&issue=7&publication_year=2009&pages=781-834&pmid=19925388&doi=10.2174/138920009789895480&)

14. Iida I, Miyata A, Arai M, et al. Catalytic roles of CYP2C9 and its variants (CYP2C9*2 and CYP2C9*3) in lornoxicam 5ʹ-hydroxylation. Drug Metab Dispos. 2004;32(1):7–9. doi: 10.1124/dmd.32.1.7  [DOI](https://doi.org/10.1124/dmd.32.1.7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14709614/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Catalytic%20roles%20of%20CYP2C9%20and%20its%20variants%20(CYP2C9*2%20and%20CYP2C9*3)%20in%20lornoxicam%205%CA%B9-hydroxylation&author=I%20Iida&author=A%20Miyata&author=M%20Arai&volume=32&issue=1&publication_year=2004&pages=7-9&pmid=14709614&doi=10.1124/dmd.32.1.7&)

15. Lee CR. CYP2C9 genotype as a predictor of drug disposition in humans. Methods Find Exp Clin Pharmacol. 2004;26(6):463–472.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/15349140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Find%20Exp%20Clin%20Pharmacol&title=CYP2C9%20genotype%20as%20a%20predictor%20of%20drug%20disposition%20in%20humans&author=CR%20Lee&volume=26&issue=6&publication_year=2004&pages=463-472&pmid=15349140&)

16. Sullivan-Klose TH, Ghanayem BI, Bell DA, et al. The role of the CYP2C9-Leu359 allelic variant in the tolbutamide polymorphism. Pharmacogenetics. 1996;6(4):341–349. doi: 10.1097/00008571-199608000-00007  [DOI](https://doi.org/10.1097/00008571-199608000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8873220/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20role%20of%20the%20CYP2C9-Leu359%20allelic%20variant%20in%20the%20tolbutamide%20polymorphism&author=TH%20Sullivan-Klose&author=BI%20Ghanayem&author=DA%20Bell&volume=6&issue=4&publication_year=1996&pages=341-349&pmid=8873220&doi=10.1097/00008571-199608000-00007&)

17. Gotoh O. Substrate recognition sites in cytochrome P450 family 2 (CYP2) proteins inferred from comparative analyses of amino acid and coding nucleotide sequences. J Biol Chem. 1992;267(1):83–90.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1730627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Substrate%20recognition%20sites%20in%20cytochrome%20P450%20family%202%20(CYP2)%20proteins%20inferred%20from%20comparative%20analyses%20of%20amino%20acid%20and%20coding%20nucleotide%20sequences&author=O%20Gotoh&volume=267&issue=1&publication_year=1992&pages=83-90&pmid=1730627&)

18. Crespi CL, Miller VP. The R144C change in the CYP2C9*2 allele alters interaction of the cytochrome P450 with NADPH: cytochromeP450 oxidoreductase. Pharmacogenetics. 1997;7(3):203–210. doi: 10.1097/00008571-199706000-00005  [DOI](https://doi.org/10.1097/00008571-199706000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9241660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20R144C%20change%20in%20the%20CYP2C9*2%20allele%20alters%20interaction%20of%20the%20cytochrome%20P450%20with%20NADPH:%20cytochromeP450%20oxidoreductase&author=CL%20Crespi&author=VP%20Miller&volume=7&issue=3&publication_year=1997&pages=203-210&pmid=9241660&doi=10.1097/00008571-199706000-00005&)

19. Wei L, Locuson CW, Tracy TS. Polymorphic variants of CYP2C9: mechanisms involved in reduced catalytic activity. Mol Pharmacol. 2007;72(5):1280–1288. doi: 10.1124/mol.107.036178  [DOI](https://doi.org/10.1124/mol.107.036178) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17686967/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Polymorphic%20variants%20of%20CYP2C9:%20mechanisms%20involved%20in%20reduced%20catalytic%20activity&author=L%20Wei&author=CW%20Locuson&author=TS%20Tracy&volume=72&issue=5&publication_year=2007&pages=1280-1288&pmid=17686967&doi=10.1124/mol.107.036178&)

20. Ho PC, Abbott FS, Zanger UM, Chang TK. Influence of CYP2C9 genotypes on the formation of a hepatotoxic metabolite of valproic acid in human liver microsomes. Pharmacogenomics J. 2003;3(6):335–342. doi: 10.1038/sj.tpj.6500210  [DOI](https://doi.org/10.1038/sj.tpj.6500210) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14597963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Influence%20of%20CYP2C9%20genotypes%20on%20the%20formation%20of%20a%20hepatotoxic%20metabolite%20of%20valproic%20acid%20in%20human%20liver%20microsomes&author=PC%20Ho&author=FS%20Abbott&author=UM%20Zanger&author=TK%20Chang&volume=3&issue=6&publication_year=2003&pages=335-342&pmid=14597963&doi=10.1038/sj.tpj.6500210&)

21. Hung CC, Lin CJ, Chen CC, Chang CJ, Liou HH. Dosage recommendation of phenytoin for patients with epilepsy with different CYP2C9/CYP2C19 polymorphisms. Ther Drug Monit. 2004;26(5):534–540. doi: 10.1097/00007691-200410000-00012  [DOI](https://doi.org/10.1097/00007691-200410000-00012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15385837/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Dosage%20recommendation%20of%20phenytoin%20for%20patients%20with%20epilepsy%20with%20different%20CYP2C9/CYP2C19%20polymorphisms&author=CC%20Hung&author=CJ%20Lin&author=CC%20Chen&author=CJ%20Chang&author=HH%20Liou&volume=26&issue=5&publication_year=2004&pages=534-540&pmid=15385837&doi=10.1097/00007691-200410000-00012&)

22. Dagenais R, Wilby KJ, Elewa H, Ensom MHH. Impact of genetic polymorphisms on phenytoin pharmacokinetics and clinical outcomes in the Middle East and North Africa Region. Drugs R D. 2017;17(3):341–361.  [DOI](https://doi.org/10.1007/s40268-017-0195-7) | [PMC free article](/articles/PMC5629135/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28748348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drugs%20R%20D&title=Impact%20of%20genetic%20polymorphisms%20on%20phenytoin%20pharmacokinetics%20and%20clinical%20outcomes%20in%20the%20Middle%20East%20and%20North%20Africa%20Region&author=R%20Dagenais&author=KJ%20Wilby&author=H%20Elewa&author=MHH%20Ensom&volume=17&issue=3&publication_year=2017&pages=341-361&pmid=28748348&doi=10.1007/s40268-017-0195-7&)

23. Tate SK, Depondt C, Sisodiya SM, et al. Genetic predictors of the maximum doses patients receive during clinical use of the anti-epileptic drugs carbamazepine and phenytoin. Proc Natl Acad Sci U S A. 2005;102(15):5507–5512. doi: 10.1073/pnas.0407346102  [DOI](https://doi.org/10.1073/pnas.0407346102) | [PMC free article](/articles/PMC556232/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15805193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Genetic%20predictors%20of%20the%20maximum%20doses%20patients%20receive%20during%20clinical%20use%20of%20the%20anti-epileptic%20drugs%20carbamazepine%20and%20phenytoin&author=SK%20Tate&author=C%20Depondt&author=SM%20Sisodiya&volume=102&issue=15&publication_year=2005&pages=5507-5512&pmid=15805193&doi=10.1073/pnas.0407346102&)

24. Kurose K, Sugiyama E, Saito Y. Population differences in major functional polymorphisms of pharmacokinetics/pharmacodynamics-related genes in Eastern Asians and Europeans: implications in the clinical trials for novel drug development. Drug Metab Pharmacokinet. 2012;27(1):9–54. doi: 10.2133/dmpk.DMPK-11-RV-111  [DOI](https://doi.org/10.2133/dmpk.DMPK-11-RV-111) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22123129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Population%20differences%20in%20major%20functional%20polymorphisms%20of%20pharmacokinetics/pharmacodynamics-related%20genes%20in%20Eastern%20Asians%20and%20Europeans:%20implications%20in%20the%20clinical%20trials%20for%20novel%20drug%20development&author=K%20Kurose&author=E%20Sugiyama&author=Y%20Saito&volume=27&issue=1&publication_year=2012&pages=9-54&pmid=22123129&doi=10.2133/dmpk.DMPK-11-RV-111&)

25. Hirota T, Eguchi S, Ieiri I. Impact of genetic polymorphisms in CYP2C9 and CYP2C19 on the pharmacokinetics of clinically used drugs. Drug Metab Pharmacokinet. 2013;28(1):28–37. doi: 10.2133/dmpk.DMPK-12-RV-085  [DOI](https://doi.org/10.2133/dmpk.DMPK-12-RV-085) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23165865/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Impact%20of%20genetic%20polymorphisms%20in%20CYP2C9%20and%20CYP2C19%20on%20the%20pharmacokinetics%20of%20clinically%20used%20drugs&author=T%20Hirota&author=S%20Eguchi&author=I%20Ieiri&volume=28&issue=1&publication_year=2013&pages=28-37&pmid=23165865&doi=10.2133/dmpk.DMPK-12-RV-085&)

26. Dai DP, Wang YH, Wang SH, et al. In vitro functional characterization of 37 CYP2C9 allelic isoforms found in Chinese Han population. Acta Pharmacol Sin. 2013;34(11):1449–1456. doi: 10.1038/aps.2013.123  [DOI](https://doi.org/10.1038/aps.2013.123) | [PMC free article](/articles/PMC4006465/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24077631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Pharmacol%20Sin&title=In%20vitro%20functional%20characterization%20of%2037%20CYP2C9%20allelic%20isoforms%20found%20in%20Chinese%20Han%20population&author=DP%20Dai&author=YH%20Wang&author=SH%20Wang&volume=34&issue=11&publication_year=2013&pages=1449-1456&pmid=24077631&doi=10.1038/aps.2013.123&)

27. Luo SB, Li CB, Dai DP, et al. Characterization of a novel CYP2C9 mutation (1009C>A) detected in a warfarin-sensitive patient. J Pharmacol Sci. 2014;125(2):150–156. doi: 10.1254/jphs.13189FP  [DOI](https://doi.org/10.1254/jphs.13189FP) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25075423/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Sci&title=Characterization%20of%20a%20novel%20CYP2C9%20mutation%20(1009C>A)%20detected%20in%20a%20warfarin-sensitive%20patient&author=SB%20Luo&author=CB%20Li&author=DP%20Dai&volume=125&issue=2&publication_year=2014&pages=150-156&pmid=25075423&doi=10.1254/jphs.13189FP&)

28. Dai DP, Li CB, Wang SH, et al. Identification and characterization of a novel CYP2C9 allelic variant in a warfarin-sensitive patient. Pharmacogenomics. 2015;16(13):1475–1486. doi: 10.2217/pgs.15.89  [DOI](https://doi.org/10.2217/pgs.15.89) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26255664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Identification%20and%20characterization%20of%20a%20novel%20CYP2C9%20allelic%20variant%20in%20a%20warfarin-sensitive%20patient&author=DP%20Dai&author=CB%20Li&author=SH%20Wang&volume=16&issue=13&publication_year=2015&pages=1475-1486&pmid=26255664&doi=10.2217/pgs.15.89&)

29. Dai D-P, Wang S-H, Li C-B, et al. Identification and functional assessment of a new CYP2C9 allelic variant CYP2C9*59. Drug Metab Dispos. 2015;43(8):1246–1249. doi: 10.1124/dmd.115.063412  [DOI](https://doi.org/10.1124/dmd.115.063412) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25994031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Identification%20and%20functional%20assessment%20of%20a%20new%20CYP2C9%20allelic%20variant%20CYP2C9*59&author=D-P%20Dai&author=S-H%20Wang&author=C-B%20Li&volume=43&issue=8&publication_year=2015&pages=1246-1249&pmid=25994031&doi=10.1124/dmd.115.063412&)

30. Zanger UM, Turpeinen M, Klein K, Schwab M. Functional pharmacogenetics/genomics of human cytochromes P450 involved in drug biotransformation. Anal Bioanal Chem. 2008;392(6):1093–1108.  [DOI](https://doi.org/10.1007/s00216-008-2291-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18695978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal%20Bioanal%20Chem&title=Functional%20pharmacogenetics/genomics%20of%20human%20cytochromes%20P450%20involved%20in%20drug%20biotransformation&author=UM%20Zanger&author=M%20Turpeinen&author=K%20Klein&author=M%20Schwab&volume=392&issue=6&publication_year=2008&pages=1093-1108&pmid=18695978&doi=10.1007/s00216-008-2291-6&)

31. Werk AN, Cascorbi I. Functional gene variants of CYP3A4. Clin Pharmacol Ther. 2014;96(3):340–348. doi: 10.1038/clpt.2014.129  [DOI](https://doi.org/10.1038/clpt.2014.129) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24926778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Functional%20gene%20variants%20of%20CYP3A4&author=AN%20Werk&author=I%20Cascorbi&volume=96&issue=3&publication_year=2014&pages=340-348&pmid=24926778&doi=10.1038/clpt.2014.129&)

32. Li AP, Kaminski DL, Rasmussen A. Substrates of human hepatic cytochrome P450 3A4. Toxicology. 1995;104(1–3):1–8. doi: 10.1016/0300-483X(95)03155-9  [DOI](https://doi.org/10.1016/0300-483X(95)03155-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8560487/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicology&title=Substrates%20of%20human%20hepatic%20cytochrome%20P450%203A4&author=AP%20Li&author=DL%20Kaminski&author=A%20Rasmussen&volume=104&issue=1%E2%80%933&publication_year=1995&pages=1-8&pmid=8560487&doi=10.1016/0300-483X(95)03155-9&)

33. Zhou SF. Drugs behave as substrates, inhibitors and inducers of human cytochrome P450 3A4. Curr Drug Metab. 2008;9(4):310–322. doi: 10.2174/138920008784220664  [DOI](https://doi.org/10.2174/138920008784220664) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18473749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Metab&title=Drugs%20behave%20as%20substrates,%20inhibitors%20and%20inducers%20of%20human%20cytochrome%20P450%203A4&author=SF%20Zhou&volume=9&issue=4&publication_year=2008&pages=310-322&pmid=18473749&doi=10.2174/138920008784220664&)

34. Westlind A, Löfberg L, Tindberg N, Andersson TB, Ingelman-Sundberg M. Interindividual differences in hepatic expression of CYP3A4: relationship to genetic polymorphism in the 5ʹ-upstream regulatory region. Biochem Biophys Res Commun. 1999;259(1):201–205. doi: 10.1006/bbrc.1999.0752  [DOI](https://doi.org/10.1006/bbrc.1999.0752) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10334940/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&title=Interindividual%20differences%20in%20hepatic%20expression%20of%20CYP3A4:%20relationship%20to%20genetic%20polymorphism%20in%20the%205%CA%B9-upstream%20regulatory%20region&author=A%20Westlind&author=L%20L%C3%B6fberg&author=N%20Tindberg&author=TB%20Andersson&author=M%20Ingelman-Sundberg&volume=259&issue=1&publication_year=1999&pages=201-205&pmid=10334940&doi=10.1006/bbrc.1999.0752&)

35. Hu GX, Dai DP, Wang H, et al. Systematic screening for CYP3A4 genetic polymorphisms in a Han Chinese population. Pharmacogenomics. 2017;18(4):369–379. doi: 10.2217/pgs-2016-0179  [DOI](https://doi.org/10.2217/pgs-2016-0179) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28244811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Systematic%20screening%20for%20CYP3A4%20genetic%20polymorphisms%20in%20a%20Han%20Chinese%20population&author=GX%20Hu&author=DP%20Dai&author=H%20Wang&volume=18&issue=4&publication_year=2017&pages=369-379&pmid=28244811&doi=10.2217/pgs-2016-0179&)

36. Dai DP, Xu RA, Hu LM, et al. CYP2C9 polymorphism analysis in Han Chinese populations: building the largest allele frequency database. Pharmacogenomics J. 2014;14(1):85–92. doi: 10.1038/tpj.2013.2  [DOI](https://doi.org/10.1038/tpj.2013.2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23400009/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=CYP2C9%20polymorphism%20analysis%20in%20Han%20Chinese%20populations:%20building%20the%20largest%20allele%20frequency%20database&author=DP%20Dai&author=RA%20Xu&author=LM%20Hu&volume=14&issue=1&publication_year=2014&pages=85-92&pmid=23400009&doi=10.1038/tpj.2013.2&)

37. Fang P, Tang PF, Xu RA, et al. Functional assessment of CYP3A4 allelic variants on lidocaine metabolism in vitro. Drug Des Devel Ther. 2017;11:3503–3510. doi: 10.2147/DDDT.S152366  [DOI](https://doi.org/10.2147/DDDT.S152366) | [PMC free article](/articles/PMC5724423/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29263648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Des%20Devel%20Ther&title=Functional%20assessment%20of%20CYP3A4%20allelic%20variants%20on%20lidocaine%20metabolism%20in%20vitro&author=P%20Fang&author=PF%20Tang&author=RA%20Xu&volume=11&publication_year=2017&pages=3503-3510&pmid=29263648&doi=10.2147/DDDT.S152366&)

38. Yang CC, Zheng X, Liu TH, et al. Functional characterization of 21 CYP3A4 variants on amiodarone metabolism in vitro. Xenobiotica. 2019;49(1):120–126. doi: 10.1080/00498254.2017.1414971  [DOI](https://doi.org/10.1080/00498254.2017.1414971) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29394111/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Functional%20characterization%20of%2021%20CYP3A4%20variants%20on%20amiodarone%20metabolism%20in%20vitro&author=CC%20Yang&author=X%20Zheng&author=TH%20Liu&volume=49&issue=1&publication_year=2019&pages=120-126&pmid=29394111&doi=10.1080/00498254.2017.1414971&)

39. Wang L, Bao SH, Pan PP, et al. Effect of CYP2C9 genetic polymorphism on the metabolism of flurbiprofen in vitro. Drug Dev Ind Pharm. 2015;41(8):1363–1367. doi: 10.3109/03639045.2014.950274  [DOI](https://doi.org/10.3109/03639045.2014.950274) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25144335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Dev%20Ind%20Pharm&title=Effect%20of%20CYP2C9%20genetic%20polymorphism%20on%20the%20metabolism%20of%20flurbiprofen%20in%20vitro&author=L%20Wang&author=SH%20Bao&author=PP%20Pan&volume=41&issue=8&publication_year=2015&pages=1363-1367&pmid=25144335&doi=10.3109/03639045.2014.950274&)

40. Xia MM, Wang L, Pan PP, et al. The role of CYP2C9 genetic polymorphisms in the oxidative metabolism of diclofenac in vitro. Pharmazie. 2014;69(12):898–903.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/25951663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmazie&title=The%20role%20of%20CYP2C9%20genetic%20polymorphisms%20in%20the%20oxidative%20metabolism%20of%20diclofenac%20in%20vitro&author=MM%20Xia&author=L%20Wang&author=PP%20Pan&volume=69&issue=12&publication_year=2014&pages=898-903&pmid=25951663&)

41. Blaisdell J, Jorge-Nebert LF, Coulter S, et al. Discovery of new potentially defective alleles of human CYP2C9. Pharmacogenetics. 2004;14(8):527–537. doi: 10.1097/01.fpc.0000114759.08559.51  [DOI](https://doi.org/10.1097/01.fpc.0000114759.08559.51) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15284535/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Discovery%20of%20new%20potentially%20defective%20alleles%20of%20human%20CYP2C9&author=J%20Blaisdell&author=LF%20Jorge-Nebert&author=S%20Coulter&volume=14&issue=8&publication_year=2004&pages=527-537&pmid=15284535&doi=10.1097/01.fpc.0000114759.08559.51&)

42. Pan PP, Weng QH, Zhou CJ, et al. The role of CYP2C9 genetic polymorphism in carvedilol O-desmethylation in vitro. Eur J Drug Metab Pharmacokinet. 2016;41(1):79–86. doi: 10.1007/s13318-014-0245-2  [DOI](https://doi.org/10.1007/s13318-014-0245-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25476996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Drug%20Metab%20Pharmacokinet&title=The%20role%20of%20CYP2C9%20genetic%20polymorphism%20in%20carvedilol%20O-desmethylation%20in%20vitro&author=PP%20Pan&author=QH%20Weng&author=CJ%20Zhou&volume=41&issue=1&publication_year=2016&pages=79-86&pmid=25476996&doi=10.1007/s13318-014-0245-2&)

43. King BP, Khan TI, Aithal GP, Kamali F, Daly AK. Upstream and coding region CYP2C9 polymorphisms: correlation with warfarin dose and metabolism. Pharmacogenetics. 2004;14(12):813–822. doi: 10.1097/00008571-200412000-00004  [DOI](https://doi.org/10.1097/00008571-200412000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15608560/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Upstream%20and%20coding%20region%20CYP2C9%20polymorphisms:%20correlation%20with%20warfarin%20dose%20and%20metabolism&author=BP%20King&author=TI%20Khan&author=GP%20Aithal&author=F%20Kamali&author=AK%20Daly&volume=14&issue=12&publication_year=2004&pages=813-822&pmid=15608560&doi=10.1097/00008571-200412000-00004&)

44. Chen M, Zhang Y, Pan P, et al. Effects of cytochrome P450 2C9 polymorphism on bosentan metabolism. Drug Metab Dispos. 2014;42(11):1820–1825. doi: 10.1124/dmd.114.060244  [DOI](https://doi.org/10.1124/dmd.114.060244) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25142737/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Effects%20of%20cytochrome%20P450%202C9%20polymorphism%20on%20bosentan%20metabolism&author=M%20Chen&author=Y%20Zhang&author=P%20Pan&volume=42&issue=11&publication_year=2014&pages=1820-1825&pmid=25142737&doi=10.1124/dmd.114.060244&)

45. Si D, Guo Y, Zhang Y, Yang L, Zhou H, Zhong D. Identification of a novel variant CYP2C9 allele in Chinese. Pharmacogenetics. 2004;14(7):465–469. doi: 10.1097/01.fpc.0000114749.08559.e4  [DOI](https://doi.org/10.1097/01.fpc.0000114749.08559.e4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15226678/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Identification%20of%20a%20novel%20variant%20CYP2C9%20allele%20in%20Chinese&author=D%20Si&author=Y%20Guo&author=Y%20Zhang&author=L%20Yang&author=H%20Zhou&volume=14&issue=7&publication_year=2004&pages=465-469&pmid=15226678&doi=10.1097/01.fpc.0000114749.08559.e4&)

46. Guo Y, Zhang Y, Wang Y, et al. Role of CYP2C9 and its variants (CYP2C9*3 and CYP2C9*13) in the metabolism of lornoxicam in humans. Drug Metab Dispos. 2005;33(6):749–753. doi: 10.1124/dmd.105.003616  [DOI](https://doi.org/10.1124/dmd.105.003616) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15764711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Role%20of%20CYP2C9%20and%20its%20variants%20(CYP2C9*3%20and%20CYP2C9*13)%20in%20the%20metabolism%20of%20lornoxicam%20in%20humans&author=Y%20Guo&author=Y%20Zhang&author=Y%20Wang&volume=33&issue=6&publication_year=2005&pages=749-753&pmid=15764711&doi=10.1124/dmd.105.003616&)

47. Guo Y, Wang Y, Si D, Fawcett PJ, Zhong D, Zhou H. Catalytic activities of human cytochrome P450 2C9*1, 2C9*3 and 2C9*13. Xenobiotica. 2005;35(9):853–861. doi: 10.1080/00498250500256367  [DOI](https://doi.org/10.1080/00498250500256367) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16308280/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica&title=Catalytic%20activities%20of%20human%20cytochrome%20P450%202C9*1,%202C9*3%20and%202C9*13&author=Y%20Guo&author=Y%20Wang&author=D%20Si&author=PJ%20Fawcett&author=D%20Zhong&volume=35&issue=9&publication_year=2005&pages=853-861&pmid=16308280&doi=10.1080/00498250500256367&)

48. DeLozier TC, Lee SC, Coulter SJ, Goh BC, Goldstein JA. Functional characterization of novel allelic variants of CYP2C9 recently discovered in southeast Asians. J Pharmacol Exp Ther. 2005;315(3):1085–1090. doi: 10.1124/jpet.105.091181  [DOI](https://doi.org/10.1124/jpet.105.091181) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16099926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Functional%20characterization%20of%20novel%20allelic%20variants%20of%20CYP2C9%20recently%20discovered%20in%20southeast%20Asians&author=TC%20DeLozier&author=SC%20Lee&author=SJ%20Coulter&author=BC%20Goh&author=JA%20Goldstein&volume=315&issue=3&publication_year=2005&pages=1085-1090&pmid=16099926&doi=10.1124/jpet.105.091181&)

49. Veenstra DL, Blough DK, Higashi MK, et al. CYP2C9 haplotype structure in European American warfarin patients and association with clinical outcomes. Clin Pharmacol Ther. 2005;77(5):353–364. doi: 10.1016/j.clpt.2005.01.019  [DOI](https://doi.org/10.1016/j.clpt.2005.01.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15900281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CYP2C9%20haplotype%20structure%20in%20European%20American%20warfarin%20patients%20and%20association%20with%20clinical%20outcomes&author=DL%20Veenstra&author=DK%20Blough&author=MK%20Higashi&volume=77&issue=5&publication_year=2005&pages=353-364&pmid=15900281&doi=10.1016/j.clpt.2005.01.019&)

50. Maekawa K, Fukushima-Uesaka H, Tohkin M, et al. Four novel defective alleles and comprehensive haplotype analysis of CYP2C9 in Japanese. Pharmacogenet Genomics. 2006;16(7):497–514. doi: 10.1097/01.fpc.0000215069.14095.c6  [DOI](https://doi.org/10.1097/01.fpc.0000215069.14095.c6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16788382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Four%20novel%20defective%20alleles%20and%20comprehensive%20haplotype%20analysis%20of%20CYP2C9%20in%20Japanese&author=K%20Maekawa&author=H%20Fukushima-Uesaka&author=M%20Tohkin&volume=16&issue=7&publication_year=2006&pages=497-514&pmid=16788382&doi=10.1097/01.fpc.0000215069.14095.c6&)

51. Matimba A, Del-Favero J, Van Broeckhoven C, Masimirembwa C. Novel variants of major drug-metabolising enzyme genes in diverse African populations and their predicted functional effects. Hum Genomics. 2009;3(2):169–190. doi: 10.1186/1479-7364-3-2-169  [DOI](https://doi.org/10.1186/1479-7364-3-2-169) | [PMC free article](/articles/PMC3525272/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19164093/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genomics&title=Novel%20variants%20of%20major%20drug-metabolising%20enzyme%20genes%20in%20diverse%20African%20populations%20and%20their%20predicted%20functional%20effects&author=A%20Matimba&author=J%20Del-Favero&author=C%20Van%20Broeckhoven&author=C%20Masimirembwa&volume=3&issue=2&publication_year=2009&pages=169-190&pmid=19164093&doi=10.1186/1479-7364-3-2-169&)

52. Yin T, Maekawa K, Kamide K, et al. Genetic variations of CYP2C9 in 724 Japanese individuals and their impact on the antihypertensive effects of losartan. Hypertens Res. 2008;31(8):1549–1557. doi: 10.1291/hypres.31.1549  [DOI](https://doi.org/10.1291/hypres.31.1549) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18971529/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertens%20Res&title=Genetic%20variations%20of%20CYP2C9%20in%20724%20Japanese%20individuals%20and%20their%20impact%20on%20the%20antihypertensive%20effects%20of%20losartan&author=T%20Yin&author=K%20Maekawa&author=K%20Kamide&volume=31&issue=8&publication_year=2008&pages=1549-1557&pmid=18971529&doi=10.1291/hypres.31.1549&)

53. Dai DP, Wang SH, Geng PW, Hu GX, Cai JP. In vitro assessment of 36 CYP2C9 allelic isoforms found in the Chinese population on the metabolism of glimepiride. Basic Clin Pharmacol Toxicol. 2014;114(4):305–310. doi: 10.1111/bcpt.12159  [DOI](https://doi.org/10.1111/bcpt.12159) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24118918/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin%20Pharmacol%20Toxicol&title=In%20vitro%20assessment%20of%2036%20CYP2C9%20allelic%20isoforms%20found%20in%20the%20Chinese%20population%20on%20the%20metabolism%20of%20glimepiride&author=DP%20Dai&author=SH%20Wang&author=PW%20Geng&author=GX%20Hu&author=JP%20Cai&volume=114&issue=4&publication_year=2014&pages=305-310&pmid=24118918&doi=10.1111/bcpt.12159&)

54. Dai D, Tang J, Rose R, et al. Identification of variants of CYP3A4 and characterization of their abilities to metabolize testosterone and chlorpyrifos. J Pharmacol Exp Ther. 2001;299(3):825–831.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11714865/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Identification%20of%20variants%20of%20CYP3A4%20and%20characterization%20of%20their%20abilities%20to%20metabolize%20testosterone%20and%20chlorpyrifos&author=D%20Dai&author=J%20Tang&author=R%20Rose&volume=299&issue=3&publication_year=2001&pages=825-831&pmid=11714865&)

55. Hsieh KP, Lin YY, Cheng CL, et al. Novel mutations of CYP3A4 in Chinese. Drug Metab Dispos. 2001;29(3):268–273.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11181494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Novel%20mutations%20of%20CYP3A4%20in%20Chinese&author=KP%20Hsieh&author=YY%20Lin&author=CL%20Cheng&volume=29&issue=3&publication_year=2001&pages=268-273&pmid=11181494&)

56. Eiselt R, Domanski TL, Zibat A, et al. Identification and functional characterization of eight CYP3A4 protein variants. Pharmacogenetics. 2001;11(5):447–458. doi: 10.1097/00008571-200107000-00008  [DOI](https://doi.org/10.1097/00008571-200107000-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11470997/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Identification%20and%20functional%20characterization%20of%20eight%20CYP3A4%20protein%20variants&author=R%20Eiselt&author=TL%20Domanski&author=A%20Zibat&volume=11&issue=5&publication_year=2001&pages=447-458&pmid=11470997&doi=10.1097/00008571-200107000-00008&)

57. Sata F, Sapone A, Elizondo G, et al. CYP3A4 allelic variants with amino acid substitutions in exons 7 and 12: evidence for an allelic variant with altered catalytic activity. Clin Pharmacol Ther. 2000;67(1):48–56. doi: 10.1067/mcp.2000.104391  [DOI](https://doi.org/10.1067/mcp.2000.104391) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10668853/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CYP3A4%20allelic%20variants%20with%20amino%20acid%20substitutions%20in%20exons%207%20and%2012:%20evidence%20for%20an%20allelic%20variant%20with%20altered%20catalytic%20activity&author=F%20Sata&author=A%20Sapone&author=G%20Elizondo&volume=67&issue=1&publication_year=2000&pages=48-56&pmid=10668853&doi=10.1067/mcp.2000.104391&)

58. Miyazaki M, Nakamura K, Fujita Y, Guengerich FP, Horiuchi R, Yamamoto K. Defective activity of recombinant cytochromes P450 3A4.2 and 3A4.16 in oxidation of midazolam, nifedipine, and testosterone. Drug Metab Dispos. 2008;36(11):2287–2291. doi: 10.1124/dmd.108.021816  [DOI](https://doi.org/10.1124/dmd.108.021816) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18669585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Defective%20activity%20of%20recombinant%20cytochromes%20P450%203A4.2%20and%203A4.16%20in%20oxidation%20of%20midazolam,%20nifedipine,%20and%20testosterone&author=M%20Miyazaki&author=K%20Nakamura&author=Y%20Fujita&author=FP%20Guengerich&author=R%20Horiuchi&volume=36&issue=11&publication_year=2008&pages=2287-2291&pmid=18669585&doi=10.1124/dmd.108.021816&)

59. Lamba JK, Lin YS, Thummel K, et al. Common allelic variants of cytochrome P4503A4 and their prevalence in different populations. Pharmacogenetics. 2002;12(2):121–132. doi: 10.1097/00008571-200203000-00006  [DOI](https://doi.org/10.1097/00008571-200203000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11875366/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Common%20allelic%20variants%20of%20cytochrome%20P4503A4%20and%20their%20prevalence%20in%20different%20populations&author=JK%20Lamba&author=YS%20Lin&author=K%20Thummel&volume=12&issue=2&publication_year=2002&pages=121-132&pmid=11875366&doi=10.1097/00008571-200203000-00006&)

60. Drögemöller B, Plummer M, Korkie L, et al. Characterization of the genetic variation present in CYP3A4 in three South African populations. Front Genet. 2013;4:17. doi: 10.3389/fgene.2013.00017  [DOI](https://doi.org/10.3389/fgene.2013.00017) | [PMC free article](/articles/PMC3574981/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23423246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet&title=Characterization%20of%20the%20genetic%20variation%20present%20in%20CYP3A4%20in%20three%20South%20African%20populations&author=B%20Dr%C3%B6gem%C3%B6ller&author=M%20Plummer&author=L%20Korkie&volume=4&publication_year=2013&pages=17&pmid=23423246&doi=10.3389/fgene.2013.00017&)

61. Xu RA, Wen J, Tang P, et al. Functional characterization of 22 CYP3A4 protein variants to metabolize ibrutinib in vitro. Basic Clin Pharmacol Toxicol. 2018;122(4):383–387. doi: 10.1111/bcpt.12934  [DOI](https://doi.org/10.1111/bcpt.12934) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29117640/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin%20Pharmacol%20Toxicol&title=Functional%20characterization%20of%2022%20CYP3A4%20protein%20variants%20to%20metabolize%20ibrutinib%20in%20vitro&author=RA%20Xu&author=J%20Wen&author=P%20Tang&volume=122&issue=4&publication_year=2018&pages=383-387&pmid=29117640&doi=10.1111/bcpt.12934&)
