# Use of Pharmacogenetic Data to Guide Individualized Opioid Prescribing After Surgery

## Metadata
**Authors:** Raffaele Rocco, Cornelius A Thiels, Daniel S Ubl, Ann M Moyer, Elizabeth B Habermann, Stephen D Cassivi
**Journal:** Surgery
**Date:** 2019 Jul 15
**DOI:** [10.1016/j.surg.2019.04.033](https://doi.org/10.1016/j.surg.2019.04.033)
**PMID:** 31320226
**PMCID:** PMC7089776
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7089776/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7089776/pdf/nihms-1067621.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7089776/pdf/nihms-1067621.pdf)

## Abstract

**Background.:** 
In spite of the current strategies aimed at avoiding opioid over prescription by implementing institutional guidelines, the use of opioids after surgical procedures remains highly variable. It is well known that opioids are activated by the cytochrome p450 CYP2D6 enzyme in order to exert pharmacologic effect. Individual variation in CYP2D6 activity impacts drug metabolism, and genotyping can be performed to predict an individual’s ability to metabolize CYP2D6 substrates. We postulate that the pharmacogenomic identification of patients with different opioid metabolism capacity may allow for the individualization of postsurgical opioid prescription.

**Methods.:** 
This study was generated by the unison of data from two prior initiatives taking place at our Institution. In the first study, patients undergoing one of 25 elective surgical procedures were prospectively identified as part of a quality initiative and surveyed by phone 21–35 days after hospital discharge to complete a 29 question survey regarding opioid utilization and pain experience. Additional chart abstraction was conducted to obtain prescribing data and pain scores during the hospitalization. The second study was the Mayo Clinic RIGHT protocol, in which 5 pharmacogenes, including CYP2D6, were genotyped for 1,000 Mayo Clinic Biobank (MCB) participants. The goal of this study was to implement preemptive pharmacogenomics in an academic health care setting and to generate data for further pharmacogenomic research. Patients were classified by their predicted CYP2D6 activity based on their CYP2D6 genotype. (intermediate - IM; intermediate to normal - INM; and normal/extensive - NM)

**Results.:** 
Of the 2486 patients with prospective opioid utilization data, 21 had pharmacogenetic data available and were included in the analysis. These patients were classified according to their activity as opioid metabolizers, with 10 patients (48%) classified as IM, 4 patients (19%) as INM and 7 patients (33%) as NM. Compared to the INM and IM phenotypes, NM patients had highest percentages of preoperative opioid naivety and recorded pain scores throughout the surgical experience. The percentage of unused opioids for IM, INM, NM categories was 79%, 63% and 46%, respectively. Moreover, of the 14 patients declaring the highest level of satisfaction for their pain control after discharge, 60% belonged to IM, 100% to INM and 57% to the NM group.

**Conclusions.:** 
This study outlines a possible correlation between genetically controlled metabolism and opioid requirements after surgery. In this setting, an increased CYP2D6 enzymatic activity was associated to a greater opioid consumption, lesser amount of unused opioids and a lower satisfaction level from opioid prescription.

Keywords: Pharmacogenomics, pain, opioids

### Background.

In spite of the current strategies aimed at avoiding opioid over prescription by implementing institutional guidelines, the use of opioids after surgical procedures remains highly variable. It is well known that opioids are activated by the cytochrome p450 CYP2D6 enzyme in order to exert pharmacologic effect. Individual variation in CYP2D6 activity impacts drug metabolism, and genotyping can be performed to predict an individual’s ability to metabolize CYP2D6 substrates. We postulate that the pharmacogenomic identification of patients with different opioid metabolism capacity may allow for the individualization of postsurgical opioid prescription.

### Methods.

This study was generated by the unison of data from two prior initiatives taking place at our Institution. In the first study, patients undergoing one of 25 elective surgical procedures were prospectively identified as part of a quality initiative and surveyed by phone 21–35 days after hospital discharge to complete a 29 question survey regarding opioid utilization and pain experience. Additional chart abstraction was conducted to obtain prescribing data and pain scores during the hospitalization. The second study was the Mayo Clinic RIGHT protocol, in which 5 pharmacogenes, including CYP2D6, were genotyped for 1,000 Mayo Clinic Biobank (MCB) participants. The goal of this study was to implement preemptive pharmacogenomics in an academic health care setting and to generate data for further pharmacogenomic research. Patients were classified by their predicted CYP2D6 activity based on their CYP2D6 genotype. (intermediate - IM; intermediate to normal - INM; and normal/extensive - NM)

### Results.

Of the 2486 patients with prospective opioid utilization data, 21 had pharmacogenetic data available and were included in the analysis. These patients were classified according to their activity as opioid metabolizers, with 10 patients (48%) classified as IM, 4 patients (19%) as INM and 7 patients (33%) as NM. Compared to the INM and IM phenotypes, NM patients had highest percentages of preoperative opioid naivety and recorded pain scores throughout the surgical experience. The percentage of unused opioids for IM, INM, NM categories was 79%, 63% and 46%, respectively. Moreover, of the 14 patients declaring the highest level of satisfaction for their pain control after discharge, 60% belonged to IM, 100% to INM and 57% to the NM group.

### Conclusions.

This study outlines a possible correlation between genetically controlled metabolism and opioid requirements after surgery. In this setting, an increased CYP2D6 enzymatic activity was associated to a greater opioid consumption, lesser amount of unused opioids and a lower satisfaction level from opioid prescription.

## INTRODUCTION

In spite of significant efforts to reduce and standardize opioid prescribing, ^1–3^ opioids remain a cornerstone of postoperative pain management for many general surgical procedures. Various procedure-specific guidelines for postoperative prescribing exist ^1–3^; however, these broad guidelines may not meet the individual needs of patients who require significantly more or less opioids, and additional patient-centered work is needed in this area.

Pharmacogenomics is the study of the role of genetic variation in drug response phenotype. Due to genetic variation, some individuals may not respond as expected to a medication, which could lead to life-threatening adverse effects at one end of the spectrum or to a lack of therapeutic efficacy at the other end of the spectrum^4–7^. CYP2D6 belongs to the cytochrome P450 gene superfamily, which includes many important phase I drug metabolizing enzymes, and participates to the metabolism of up to 25% of clinically used drugs, including codeine. The CYP2D6 gene is highly polymorphic, leading to marked interindividual variation in enzyme activity and corresponding CYP2D6-mediated drug metabolism. An individual’s enzyme activity can be predicted based on his or her genotype. Opioids like codeine require activation by CYP2D6 before for pharmacologic efficacy^8^. An individual with higher than typical ability to metabolize CYP2D6 substrates can be described as an ultrarapid metabolizer and would be expected to metabolize codiene to morphine more rapidly than average, which may lead to adverse events, such as respiratory depression. On the other hand, individuals with reduced or markedly reduced enzyme activity can be described as intermediate or poor metabolizers, respectively, and when prescribed standard doses, may not metabolize sufficient codeine to morphine and may experience lack of therapeutic efficacy. Individuals may also have activity that spans these categories (i.e. poor to intermediate metabolizer) ([Box 1](#BX1)).The frequency of CYP2D6 variant alleles and distribution of metabolizer status vary significantly across world populations; it has been reported that CYP2D6 poor metabolizers comprise 0.4 to 5.4% of individuals across populations, approximately 0.4–11% of individuals are intermediate metabolizers, 67–90% are normal metabolizers, and 1–21% of individuals across populations are predicted to be ultrarapid metabolizers^9^.|.At standard doses, a patient who is a CYP2D6 ultrarapid metabolizer would be expected to be at greater risk of rapidly converting codeine to morphine and experiencing overdose symptoms ^8,10,13^. On the other hand, a patient with an intermediate or a poor metabolizer phenotype would be predicted to be at greater risk of experiencing lack of efficacy as they may not efficiently activate a prodrug like codeine or they may experience adverse events when treated with active drugs due to reduced deactivation of the medication. While codeine has been the subject of pharmacogenomic dosing guidelines^11^, current prescribing guidelines for other opioids would however result in each of these phenotype patients receiving the same number of opioids. ([Box 1](#BX1))

| CYTOCHROME PHENOTYPE | METABOLISM RATE | EXAMPLE |
| --- | --- | --- |
| Ultra-rapid metabolizer | Highest | A patient is expected to metabolize opioids more rapidly than average. Greater risk of overdose symptoms with prodrugs; greater risk of lack of efficacy with active drugs. |
| Normal/Extensive metabolizer (NM) |   | A patient is expected to metabolize opioids at an average rate. This rate is faster than INM, IM and Poor mtabolizer rate. |
| Intermediate to normal metabolizer (INM) |   | A patient is expected to metabolize opioids less quickly than average but at a higher rate than IM and poor metabolizers. |
| Intermediate metabolizer (IM) |   | A patient is expected to metabolize opioids less quickly than average but at a higher rate than poor metabolizers. |
| Poor metabolizer | Lowest | A patient is expected to metabolize opioids less quickly than average. Greater risk of lack of efficacy with prodrugs; greater risk of overdose symptoms with active drugs. |
We postulate that pharmacogenetic data may help facilitate an improved approach to the prescription of opioids by identifying an individualized, pre-emptive, genotype-guided therapy. This study represented an opportunity to inform the postsurgical opioid prescription practice by integrating genomic data with opioid consumption, hypothesizing that different genetically predetermined metabolizers use a different number of opioids after surgery.

| CYTOCHROME PHENOTYPE | METABOLISM RATE | EXAMPLE |
| --- | --- | --- |
| Ultra-rapid metabolizer | Highest | A patient is expected to metabolize opioids more rapidly than average. Greater risk of overdose symptoms with prodrugs; greater risk of lack of efficacy with active drugs. |
| Normal/Extensive metabolizer (NM) |   | A patient is expected to metabolize opioids at an average rate. This rate is faster than INM, IM and Poor mtabolizer rate. |
| Intermediate to normal metabolizer (INM) |   | A patient is expected to metabolize opioids less quickly than average but at a higher rate than IM and poor metabolizers. |
| Intermediate metabolizer (IM) |   | A patient is expected to metabolize opioids less quickly than average but at a higher rate than poor metabolizers. |
| Poor metabolizer | Lowest | A patient is expected to metabolize opioids less quickly than average. Greater risk of lack of efficacy with prodrugs; greater risk of overdose symptoms with active drugs. |
## METHODS

This study was generated by the unison of data from two prior initiatives taking place at our institution. The first was an initiative conducted from March 2017 to January 2018 which involved prospectively surveying 2486 patients undergoing one of 25 elective surgical procedures at three affiliated academic institutions^1^. Eligible patients were selected from hospital procedure lists using stratified simple random selection without replacement. Patients were excluded if the procedure was combined with other major operations or if the patient had a second operation before the survey. As part of a quality initiative patients were surveyed by phone 21–35 days after hospital discharge and asked to complete a 28-question survey regarding opioid utilization and pain experience. Both of the aforementioned were undertaken so as to reduce social desirability and recall bias.^1, 12^. Patients were asked to report how many opioids were remaining (un-used) for each of their opioid prescriptions as well as if they had obtained a refill. Patients were also asked to report their experience with their pain control after discharge. Complete study methodology is available in the manuscript and supplemental materials; in brief, the 2486 patients that completed the survey corresponded to a response rate of 75.2%. Of the non-responders, the majority did not answer the call while the rest (3.1% of all sampled) refused to participate. ^1^

After obtaining Institutional Review Board (IRB) approval, the patients who participated in the quality initiative were matched with patients who had submitted blood samples to the Mayo Clinic Biobank (MCB) and had additional pharmacogenetic testing done as part of the “Right Drug, Right Dose, Right Time study” ^10,14^

Participants in the RIGHT study at Mayo Clinic underwent next-generation sequencing using the PGRN-Seq capture reagent. In addition, CYP2D6 was genotyped separately using a combination of the Luminex Tag-It Mutation Detection Kit for Cytochrome P450 2D6, a laboratory-developed copy number variation assay, and/or sequencing assays ^15^. Pharmacogenomic results including genotype and phenotype for CYP2C19, CYP2C9, VKORC1, SLCO1B1, and CYP2D6 were deposited in the electronic health record. Each of the 1013 participants were selected based on enrollment in the Mayo Clinic Biobank and high risk for initiating a statin drug within 3 years.

CYP2D6 genotype data were used to predict the phenotype based on known functionality of the alleles identified for each participant as previously described.^10, 15, 16^ The Activity Score for each diplotype was calculated based on data from the Pharmacogene Variation (PharmVar) Consortium and review of functional studies available in the literature as per standard clinical practice in the Mayo Clinic Personalized Genomics Laboratory^15^. The Activity Score was used to assign a CYP2D6 phenotype for each patient, and both the Activity Score and predicted phenotype were used in the analyses. ^15,17^ The median of the Activity Score range was used for plotting.

Once the patients who had both survey data and pharmacogenetic data available were identified, additional chart abstraction was conducted to obtain patient demographic and surgical information as well as opioid prescribing data. Opioid prescriptions at discharge and consumption data were converted into morphine milligram equivalents (MME). In addition, routinely collected pain scores were obtained during the hospitalization. These were recorded using the 10-point patient-reported Numeric Pain Rating Scale (NPRS) as median (IQR) values. NPRS were collected during three different perioperative phases to generate: 1) pre-operative pain score 2) maximum pain score and 3) discharge pain score.

Based on an increasing extent of metabolizing activity, three CYP2D6 metabolizer phenotypes were identified, namely, the intermediate metabolizers (IM), the intermediate to normal metabolizers (INM), and the normal (extensive - NM) metabolizers. Opioid consumption, pain scores, and patient experience with pain control were compared across CYP2D6 metabolizer phenotypes. Genotype to phenotype translation was performed using the same process as used for clinical CYP2D6 testing in our laboratory. Briefly, an Activity Score was used where each allele was assigned a value from 0 (no activity) to 1 (normal activity), then the scores for each allele detected were added. The Activity Score was then translated into the corresponding phenotype based on pre-defined ranges depicted in [Box 1](#BX1).

These two categories were compared by univariate analysis using Wilcoxon Rank-Sum tests for continous variables while Chi-square and Fisher exact tests were used for categorical variables. Statistical analysis was performed using version 9.4 of SAS (SAS Institute Inc., Cary NC).

## RESULTS

Of the 2486 surgical patients with prospective survey data on opioid utilization, 21 also had pharmacogenomic information available and were included in analysis. In this cohort of patients the median age was 63 [IQR 59–68] and 14 (67%) were females. Patients underwent a variety of operations that ranged from breast lumpectomy to total knee replacements. ([Table 1](#T1))

### TABLE 1:

| Procedure | * MME prescribed | * MME consumed | % used of MME | Satisfaction level | Current Phenotype | Activity Score |
| --- | --- | --- | --- | --- | --- | --- |
| Breast Lumpectomy ± Sentinel Node | 80 | 56 | 70.00% | Somewhat Satisfied | Normal (Extensive) Metabolizer | 1.8–2 |
| Carotid Endarterectomy | 0 | 0 | 0.00% | Very Satisfied | Intermediate Metabolizer | 0.8–1 |
| Carpal Tunnel Release | 75 | 37.5 | 50.00% | Very Satisfied | Intermediate To Normal (Extensive) Metabolizer | 1.3–1.5 |
| Carpal Tunnel Release | 75 | 0 | 0.00% | Very Satisfied | Intermediate Metabolizer | 1 |
| Knee Arthroscopic Meniscectomy | 225 | 75 | 33.33% | Very Satisfied | Intermediate To Normal (Extensive) Metabolizer | 1.3–1.5 |
| Knee Arthroscopic Meniscectomy | 0 | 0 | 0.00% | Somewhat Satisfied | Intermediate Metabolizer | 1 |
| Lumbar Fusion | 200 | 10 | 5.00% | Very Satisfied | Intermediate Metabolizer | 1 |
| MIS Inguinal Hernia Repair | 75 | 0 | 0.00% | Very Satisfied | Intermediate To Normal (Extensive) Metabolizer | 1.6–2 |
| MIS Inguinal Hernia Repair | 150 | 0 | 0.00% | Very Satisfied | Intermediate Metabolizer | 0.8–1 |
| MIS Lung Wedge Resection | 600 | 150 | 25.00% | Very Dissatisfied | Intermediate Metabolizer | 1 |
| MIS Prostatectomy | 150 | 37.5 | 25.00% | Very Satisfied | Normal (Extensive) Metabolizer | 2 |
| Parathyroidectomy | 75 | 0 | 0.00% | Very Satisfied | Normal (Extensive) Metabolizer | 2 |
| Parathyroidectomy | 100 | 5 | 5.00% | Very Satisfied | Intermediate Metabolizer | 0.8–1 |
| Rotator Cuff Surgery | 1025 | 140 | 13.66% | Somewhat Satisfied | Intermediate Metabolizer | 1 |
| Simple Mastectomy ± Sentinel Node | 212.5 | 45 | 21.18% | Very Satisfied | Intermediate Metabolizer | 1.2 |
| Total Hip | 587.5 | 115 | 19.57% | Very Satisfied | Intermediate Metabolizer | 1 |
| Total Knee | 900 | 600 | 66.67% | Very Satisfied | Intermediate To Normal (Extensive) Metabolizer | 1.3–1.5 |
| Total Knee | 600 | 600 | 100.00% | Somewhat Dissatisfied | Normal (Extensive) Metabolizer | 2 |
| Total Knee | 475 | 0 | 0.00% | Somewhat Satisfied | Normal (Extensive) Metabolizer | 1.8–2 |
| Total Knee | 550 | 475 | 86.36% | Very Satisfied | Normal (Extensive) Metabolizer | 1.8–2 |
| Total Knee | 1000 | 977.5 | 97.75% | Very Satisfied | Normal (Extensive) Metabolizer | 1.8–2 |

Table 1 Caption: SURGICAL PROCEDURES PERFORMED CORRELATED TO MME PRESCRIBED/CONSUMED, PATIENT SATISFACTION AND CYP2D6 METABOLIZER PHENOTYPE

At discharge patients were prescribed a median of 200 MME (IQR=475; 75,550). The median amount of MME consumed after discharge was 37.5 MME (IQR 150; 0, 150). This resulted in 75 MME (IQR=143.5; 24,167.5) or 63% of the prescribed opioids that remained unused. Aforementioned MME values do not sum due to the fact that they were not normalized. Based on our findings, 10 patients (48%) were found to be in the IM group, 4 patients (19%) were in the INM group and 7 patients (33%) were in the NM group^9^. None of the participants were found to be ultrarapid or poor metabolizers. If the opioid use per each metabolizer phenotype is analyzed, the NM phenotypes showed the highest utilization (57%) compared to IM (40%) and INM (25%) without reaching statistical significance (p=0.57).

In this series, IM phenotype patients were significantly older (in the 60–79 years group range) than the other patients (p = 0.02). However, the INM phenotype patients showed a trend toward lower preoperative (75% vs 100% of IM and NM, respectively; p = 0.58) and lower maximal pain scores (50% vs 90% and 100% of IM and NM, respectively; p = 0.094), as well as an overall MME consumption more than the 75^th^ percentile. (75% vs 85.7% and 80% of IM and NM, respectively; p = 0.062). Conversely, compared to the INM and IM phenotypes, NM had highest percentages of opioid naivety and pain scores throughout the surgical experience but the difference did not reach statistical significance. Of the 21 patients, 14 declared the highest level of satisfaction for their pain control after discharge. In this subgroup, the NM phenotype demonstrated the least level of satisfaction (57% vs 60% and 100% of the IM and INM, respectively; p=0.83). The need for opioid refill from the time of discharge to time of survey was not statistically different among the different metabolizer phenotypes (p = 1.00). ([Table 2](#T2))

### TABLE 2:

|   | All (n=21) | CYP2D6 Metabolizer Status: Intermediate (n=10) | CYP2D6 Metabolizer Status: Intermediate to normal (extensive) (n=4) | CYP2D6 Metabolizer Status: Normal (extensive) (n=7) | P-value |
| --- | --- | --- | --- | --- | --- |
| Age |   |   |   |   | 0.03 |
| No. (%) used | 21 (100.0%) | 10 (100.0%) | 4 (100.0%) | 7 (100.0%) |   |
| Median († IQR) | 63 (59,68) | 68 (63,73) | 59.5 (54.5,61.5) | 58 (55,65) |   |
| Range | 49 to 79 | 60 to 79 | 50 to 63 | 49 to 76 |   |
| Age Group, no. |   |   |   |   | 0.02 |
| 80+ | 0 (0.0) | 0 (0.0) | 0 (0.0) | 0 (0.0) |   |
| 60–79 | 15 (71.4) | 10 (100.0) | 2 (50.0) | 3 (42.9) |   |
| 40–59 | 6 (28.6) | 0 (0.0) | 2 (50.0) | 4 (57.1) |   |
| 18–39 | 0 (0.0) | 0 (0.0) | 0 (0.0) | 0 (0.0) |   |
| Sex, no. |   |   |   |   | 0.70 |
| Female | 14 (66.7) | 7 (70.0) | 2 (50.0) | 5 (71.4) |   |
| Male | 7 (33.3) | 3 (30.0) | 2 (50.0) | 2 (28.6) |   |
| Pre-operative Opioid User (Self-Defined) |   |   |   |   | 1.00 |
| Opioid Naive | 17 (81.0) | 8 (80.0) | 3 (75.0) | 6 (85.7) |   |
| Opioid Tolerant | 4 (19.0) | 2 (20.0) | 1 (25.0) | 1 (14.3) |   |
| Pre-operative Pain Score |   |   |   |   | 0.58 |
| No. (%) used | 20 (95.2%) | 10 (100.0%) | 3 (75.0%) | 7 (100.0%) |   |
| Median († IQR) | 2 (0,3) | 1 (0,3) | 1 (0,4) | 2 (0,6) |   |
| Range | 0 to 8 | 0 to 5 | 0 to 4 | 0 to 8 |   |
| Max pain score |   |   |   |   | 0.09 |
| No. (%) used | 18 (85.7%) | 9 (90.0%) | 2 (50.0%) | 7 (100.0%) |   |
| Median († IQR) | 6 (3,8) | 3 (2,6) | 3 (0,6) | 6 (6,9) |   |
| Range | 0 to 10 | 0 to 9 | 0 to 6 | 5 to 10 |   |
| Prescribed Opioids at Discharge |   |   |   |   | 0.67 |
| No | 2 (9.5) | 2 (20.0) | 0 (0.0) | 0 (0.0) |   |
| Yes | 19 (90.5) | 8 (80.0) | 4 (100.0) | 7 (100.0) |   |
| Discharge Morphine Milligram Equivalents (*MME) |   |   |   |   | 0.64 |
| No. (%) used | 21 (100.0%) | 10 (100.0%) | 4 (100.0%) | 7 (100.0%) |   |
| Median († IQR) | 200 (75,550) | 175 (75,450) | 150 (75,562.5) | 475 (80,600) |   |
| Range | 0 to 1025 | 0 to 1025 | 75 to 900 | 75 to 1000 |   |
| Consumed Morphine Milligram Equivalents (*MME) |   |   |   |   | 0.46 |
| No. (%) used | 21 (100.0%) | 70 (100.0%) | 4 (100.0%) | 7 (100.0%) |   |
| Median († IQR) | 37.5 (0,150) | 7.5 (0,140) | 56.25 (18.75,337.5) | 56 (0,600) |   |
| Range | 0 to 977.5 | 0 to 450 | 0 to 600 | 0 to 977.5 |   |
| MME CONSUMED >75th percentile, no. (col %) |   |   |   |   | 0.06 |
| No | 17 (81.0) | 10 (100.0) | 3 (75.0) | 4 (57.1) |   |
| Yes | 4 (19.0) | 0 (0.0) | 1 (25.0) | 3 (42.9) |   |
| Opioid Refill From Time of Discharge to Time of Survey (Self-Defined) |   |   |   |   | 1.00 |
| No | 20 (95.2) | 9 (90.0) | 4 (100.0) | 7 (100.0) |   |
| Yes | 1 (4.8) | 1 (10.0) | 0 (0.0) | 0 (0.0) |   |
| Patient satisfaction with pain control after discharge, no. (col %) |   |   |   |   | 0.83 |
| Very satisfied | 14 (66.7) | 6 (60.0) | 4 (100.0) | 4 (57.1) |   |
| Somewhat satisfied | 4 (19.0) | 2 (20.0) | 0 (0.0) | 2 (28.6) |   |
| Neither satisfied nor dissatisfied | 1 (4.8) | 1 (10.0) | 0 (0.0) | 0 (0.0) |   |
| Somewhat dissatisfied | 1 (4.8) | 0 (0.0) | 0 (0.0) | 1 (14.3) |   |
| Very dissatisfied | 1 (4.8) | 1 (10.0) | 0 (0.0) | 0 (0.0) |   |

Table 2 Caption: PATIENT CHARACTERISTICS AND PAIN SCORES CORRELATED TO CYP2D6 METABOLIZER PHENOTYPE

Activity Score based on phenotypic categories of opioid metabolizers was plotted against amount of opioids consumed (MME) for each patient and suggested a trend towards increasing opioid consumption correlating with increasing Activity Score ([Figure 1](#F1)).

### Figure 1:

![Figure 1:](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/69ea/7089776/b26724f180fb/nihms-1067621-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7089776_nihms-1067621-f0001.jpg)

OME CONSUMED BY CYP2D6 ACTIVITY SCORE. Median of the Activity Score range used for plotting

## DISCUSSION

In recent years, many institutions have adopted procedure-specific guidelines to address the issue of opioid prescribing after discharge from surgery ^11, 18,19^. Although these guidelines represent a signficant step forward from the prior era of opioid prescribing, they are not patient specific. Our preliminary data suggests that pharmacogenetics may provide an innovative, tailored approach to allow for “patient-related predictor” of opioid use ^5^.

Opioids are among the top 30 drugs with high pharmacogenetic risk^7^; thus, there is room for development and integration of the concept of using genetic data to guide clinical management. In this setting, the genomic profile for opioid sensitivity and responsiveness could be used as a surrogate of the so-called individual “pain threshold” ^20–22^. In our study, NM patients tend to show a higher pre-operative and maximal pain score as well as an overall MME consumption of more than the 75^th^ percentile. Nevertheless, NM patients were the least satisfied with their pain control after surgery. Without the information about their CYP2D6 status, these features would probably classify NM patients as having a low pain threshold.

Over-and under-prescribing of outliers is a significant concern with the current procedure specific opioid prescribing guidelines after surgery ^22^. Identifying those patients at risk of not receiving enough opioids may prevent poor patient experiences with pain control and refills ^23^. For example, 48% of patients in our study (IM) exhibited a phenotype that suggested the opioids may be more potent, and would potentially require less opioids than other patients.

The era of standardized postsurgical prescriptions, facilitated by ERAS Protocols ^24,^ may soon be over in favor of modulated opioid prescription possibly based on a genomic-related “addiction risk score” ^22^. Moreover, the genetic identification of a specific inclination towards the development of opioid-related side effects will contribute to a personalized dosing of the drug ^23^. [Figure 1](#F1) also suggested a possible association between increasing Activity Score being associated with increasing opioid consumption. This demonstrates a possible association between genetically controlled metabolism and opioid requirements in the postoperative setting ^25^. The low numerosity of our study does not allow for definite conclusions.

Our pilot study has significant limitations. The retrospective nature as well as limited patient population limit our ability to perform a statistical analysis. That being said, the aim of this preliminary study was to determine feasibility and potential value of genomic profiling in the context of peri-operative opioid use. We did not identify any participants with ultrarapid (faster than typical) or poor (greatly reduced) CYP2D6 activity. In addition, we did not account for concurrent medication with strong CYP2D6 inhibitors that could impact the phenotype. Our study only evaluated genetic variation in CYP2D6. Variation in other genes likely contributes to interindividual variation in opioid response as well. The surgical procedures that our patients were subjected to were highly variable. Furthermore, opioid prescribing was not yet standardized during the course of this study. Our study also does not address how to resolve the disadvantages of incorporating pharmacogenetic data in clinical practice, among which (1) legal, social, and financial concerns with regard to genomic medicine by patients and their families (2) lack of support for commercial electronic medical record (EMR) systems to integrate large-scale genomic data linked to automated clinical decision support (3) delay in the initiation of therapy when traditional reactive ordering of pharmacogenetic testing at point-of-care is used. Lastly, our matching of two study cohorts has significant bias and our findings can therefore likely not be generalized to other institutions or the greater population. Despite the aforementioned significant limitations, the results of this exploratory pilot study do suggest a potential avenue for future research and possible practice change.

The results of our pilot study suggests that pharmacogenomic data has the potential of reducing over and under prescription of opioids following surgery and eventually creating personalized opioid prescribing practices. Further data and research on opioid pharmacogenomics is warranted to confirm the value of our findings and guide the transition from a “one size fits all” guideline approach to post-operative opioid prescribing to a patient centered approach.

## Acknowledgments

**Source of Funding:** Support provided by the Mayo Clinic Robert D. and Patricia E. Kern Center for the Science of Health Care Delivery (Ubl and Habermann).

## Footnotes

## Contributor Information

Raffaele Rocco, Department of Surgery, Surgical Outcomes Program, Mayo Clinic, Rochester, Minnesota.; The Robert D. and Patricia E. Kern Center for the Science of Health Care Delivery, Mayo Clinic, Rochester, Minnesota.

Cornelius A. Thiels, Department of Surgery, Surgical Outcomes Program, Mayo Clinic, Rochester, Minnesota..

Daniel S. Ubl, The Robert D. and Patricia E. Kern Center for the Science of Health Care Delivery, Mayo Clinic, Rochester, Minnesota..

Ann M. Moyer, Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, Minnesota.

Elizabeth B. Habermann, The Robert D. and Patricia E. Kern Center for the Science of Health Care Delivery, Mayo Clinic, Rochester, Minnesota..

Stephen D. Cassivi, Department of Surgery, Surgical Outcomes Program, Mayo Clinic, Rochester, Minnesota.

## References

1. Thiels CA, Anderson SS, Ubl DS, et al. Wide Variation and Overprescription of Opioids After Elective Surgery. Ann Surg. 2017;266:564–573  [DOI](https://doi.org/10.1097/SLA.0000000000002365) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28697049/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Surg&title=Wide%20Variation%20and%20Overprescription%20of%20Opioids%20After%20Elective%20Surgery&author=CA%20Thiels&author=SS%20Anderson&author=DS%20Ubl&volume=266&publication_year=2017&pages=564-573&pmid=28697049&doi=10.1097/SLA.0000000000002365&)

2. Hill MV, McMahon ML, Stucke RS, et al. Wide Variation and Excessive Dosage of Opioid Prescriptions for Common General Surgical Procedures. Ann Surg. 2017; 265:709–714.  [DOI](https://doi.org/10.1097/SLA.0000000000001993) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27631771/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Surg&title=Wide%20Variation%20and%20Excessive%20Dosage%20of%20Opioid%20Prescriptions%20for%20Common%20General%20Surgical%20Procedures&author=MV%20Hill&author=ML%20McMahon&author=RS%20Stucke&volume=265&publication_year=2017&pages=709-714&pmid=27631771&doi=10.1097/SLA.0000000000001993&)

3. Hilliard PE, Waljee J, Moser S, et al. Prevalence of Preoperative Opioid Use and Characteristics Associated With Opioid Use Among Patients Presenting for Surgery. JAMA Surg. 2018;153:929–937.  [DOI](https://doi.org/10.1001/jamasurg.2018.2102) | [PMC free article](/articles/PMC6233789/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29998303/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Surg&title=Prevalence%20of%20Preoperative%20Opioid%20Use%20and%20Characteristics%20Associated%20With%20Opioid%20Use%20Among%20Patients%20Presenting%20for%20Surgery&author=PE%20Hilliard&author=J%20Waljee&author=S%20Moser&volume=153&publication_year=2018&pages=929-937&pmid=29998303&doi=10.1001/jamasurg.2018.2102&)

4. New Ballantyne JC, Mao J. Opioid therapy for chronic pain. N Engl J Med 2003; 349:1943–1953.  [DOI](https://doi.org/10.1056/NEJMra025411) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14614170/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Opioid%20therapy%20for%20chronic%20pain&author=JC%20Ballantyne&author=J%20Mao&volume=349&publication_year=2003&pages=1943-1953&pmid=14614170&doi=10.1056/NEJMra025411&)

5. Adams SM, Crisamore KR, Empey PE. Clinical Pharmacogenomics: Applications in Nephrology. Clin J Am Soc Nephrol. 2018;13:1561–1571  [DOI](https://doi.org/10.2215/CJN.02730218) | [PMC free article](/articles/PMC6218819/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29793969/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20J%20Am%20Soc%20Nephrol&title=Clinical%20Pharmacogenomics:%20Applications%20in%20Nephrology&author=SM%20Adams&author=KR%20Crisamore&author=PE%20Empey&volume=13&publication_year=2018&pages=1561-1571&pmid=29793969&doi=10.2215/CJN.02730218&)

6. Zambrano T, Salazar LA. microRNAs and response to statins in patients with hypercholesterolemia: from basic research to precision medicine.Pharmacogenomics. 2018;19:748–751  [DOI](https://doi.org/10.2217/pgs-2018-0051) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29785870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=microRNAs%20and%20response%20to%20statins%20in%20patients%20with%20hypercholesterolemia:%20from%20basic%20research%20to%20precision%20medicine&author=T%20Zambrano&author=LA%20Salazar&volume=19&publication_year=2018&pages=748-751&pmid=29785870&doi=10.2217/pgs-2018-0051&)

7. Jiang F, Choi JY, Lee JH, et al. The influences of SLCO1B1 and ABCB1 genotypes on the pharmacokinetics of simvastatin, in relation to CYP3A4 inhibition. Pharmacogenomics. 2017;18:459–469  [DOI](https://doi.org/10.2217/pgs-2016-0199) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28350522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=The%20influences%20of%20SLCO1B1%20and%20ABCB1%20genotypes%20on%20the%20pharmacokinetics%20of%20simvastatin,%20in%20relation%20to%20CYP3A4%20inhibition&author=F%20Jiang&author=JY%20Choi&author=JH%20Lee&volume=18&publication_year=2017&pages=459-469&pmid=28350522&doi=10.2217/pgs-2016-0199&)

8. Gaedigk A et al. Genetics in Medicine volume 1.9, pages 69–76 (2017)  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics%20in%20Medicine&author=A%20Gaedigk&volume=1.9&publication_year=2017&pages=69-76&)

9. St Sauver JL, Olson JE, Roger VL, et al. CYP2D6 phenotypes are associated with adverse outcomes related to opioid medications. Pharmagenomics Pers Med. 2017;10:217–227.  [DOI](https://doi.org/10.2147/PGPM.S136341) | [PMC free article](/articles/PMC5533497/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28769582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmagenomics%20Pers%20Med&title=CYP2D6%20phenotypes%20are%20associated%20with%20adverse%20outcomes%20related%20to%20opioid%20medications&author=JL%20St%20Sauver&author=JE%20Olson&author=VL%20Roger&volume=10&publication_year=2017&pages=217-227&pmid=28769582&doi=10.2147/PGPM.S136341&)

10. Crews KR, et al. Clin Pharmacol Ther 2014. 95(4):376–382;  [DOI](https://doi.org/10.1038/clpt.2013.254) | [PMC free article](/articles/PMC3975212/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24458010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&author=KR%20Crews&volume=95&issue=4&publication_year=2014&pages=376-382&pmid=24458010&doi=10.1038/clpt.2013.254&) | [DOI](https://doi.org/10.1038/clpt.2011.34) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21412232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&author=JJ%20Swen&volume=89&issue=5&publication_year=2011&pages=662-73&pmid=21412232&doi=10.1038/clpt.2011.34&)

11. Thiels CA, Ubl DS, Yost KJ, et al. Results of a Prospective, Multicenter Initiative Aimed at Developing Opioid-prescribing Guidelines After Surgery. Ann Surg. 2018;268:457–468  [DOI](https://doi.org/10.1097/SLA.0000000000002919) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30004924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Surg&title=Results%20of%20a%20Prospective,%20Multicenter%20Initiative%20Aimed%20at%20Developing%20Opioid-prescribing%20Guidelines%20After%20Surgery&author=CA%20Thiels&author=DS%20Ubl&author=KJ%20Yost&volume=268&publication_year=2018&pages=457-468&pmid=30004924&doi=10.1097/SLA.0000000000002919&)

12. Bowling A. Mode of questionnaire administration can have serious effects on data quality. J Public Health (Oxf). 2005;27:281–291  [DOI](https://doi.org/10.1093/pubmed/fdi031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15870099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Public%20Health%20(Oxf)&title=Mode%20of%20questionnaire%20administration%20can%20have%20serious%20effects%20on%20data%20quality&author=A%20Bowling&volume=27&publication_year=2005&pages=281-291&pmid=15870099&doi=10.1093/pubmed/fdi031&)

13. Olson JE, Rohrer Vitek CR, Bell EJ, McGree ME, Jacobson DJ, St Sauver JL, Caraballo PJ, Griffin JM, Roger VL, Bielinski SJ. Participant-perceived understanding and perspectives on pharmacogenomics: the Mayo Clinic RIGHT protocol (Right Drug, Right Dose, Right Time). Genet Med. 2017. July;19(7):819–825  [DOI](https://doi.org/10.1038/gim.2016.192) | [PMC free article](/articles/PMC5498279/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28055020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=Participant-perceived%20understanding%20and%20perspectives%20on%20pharmacogenomics:%20the%20Mayo%20Clinic%20RIGHT%20protocol%20(Right%20Drug,%20Right%20Dose,%20Right%20Time)&author=JE%20Olson&author=CR%20Rohrer%20Vitek&author=EJ%20Bell&author=ME%20McGree&author=DJ%20Jacobson&volume=19&issue=7&publication_year=2017&pages=819-825&pmid=28055020&doi=10.1038/gim.2016.192&)

14. Ji Y et al. Journal of Molecular Diagnostics, Vol 18, No 3, May 2016  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Molecular%20Diagnostics&author=Y%20Ji&volume=18&issue=3&publication_year=2016&)

15. www.pharmvar.org.  [www.pharmvar.org](http://www.pharmvar.org)

16. Gaedigk A, et al. J Pers Med 2018. 8(2):15.  [DOI](https://doi.org/10.3390/jpm8020015) | [PMC free article](/articles/PMC6023391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29673183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pers%20Med&author=A%20Gaedigk&volume=8&issue=2&publication_year=2018&pages=15&pmid=29673183&doi=10.3390/jpm8020015&)

17. Adams SM, Crisamore KR, Empey PE. Clinical Pharmacogenomics: Applications in Nephrology. Clin J Am Soc Nephrol. 2018;13:1561–1571  [DOI](https://doi.org/10.2215/CJN.02730218) | [PMC free article](/articles/PMC6218819/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29793969/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20J%20Am%20Soc%20Nephrol&title=Clinical%20Pharmacogenomics:%20Applications%20in%20Nephrology&author=SM%20Adams&author=KR%20Crisamore&author=PE%20Empey&volume=13&publication_year=2018&pages=1561-1571&pmid=29793969&doi=10.2215/CJN.02730218&)

18. https://opioidprescribing.info/  [https://opioidprescribing.info/](https://opioidprescribing.info/)

19. Overton HN, Hanna MN, Bruhn WE, et al. Opioids After Surgery Workgroup. Opioid-Prescribing Guidelines for Common Surgical Procedures: An Expert Panel Consensus. J Am Coll Surg. 2018;227:411–418  [DOI](https://doi.org/10.1016/j.jamcollsurg.2018.07.659) | [PMC free article](/articles/PMC6353661/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30118896/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Surg&title=Opioids%20After%20Surgery%20Workgroup.%20Opioid-Prescribing%20Guidelines%20for%20Common%20Surgical%20Procedures:%20An%20Expert%20Panel%20Consensus&author=HN%20Overton&author=MN%20Hanna&author=WE%20Bruhn&volume=227&publication_year=2018&pages=411-418&pmid=30118896&doi=10.1016/j.jamcollsurg.2018.07.659&)

20. Zahari Z, Ismail R. Influence of Cytochrome P450, Family 2, Subfamily D,Polypeptide 6 (CYP2D6) polymorphisms on pain sensitivity and clinical response to weak opioid analgesics. Drug Metab Pharmacokinet. 2014;29(1):29–43  [DOI](https://doi.org/10.2133/dmpk.dmpk-13-rv-032) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23759977/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Influence%20of%20Cytochrome%20P450,%20Family%202,%20Subfamily%20D,Polypeptide%206%20(CYP2D6)%20polymorphisms%20on%20pain%20sensitivity%20and%20clinical%20response%20to%20weak%20opioid%20analgesics&author=Z%20Zahari&author=R%20Ismail&volume=29&issue=1&publication_year=2014&pages=29-43&pmid=23759977&doi=10.2133/dmpk.dmpk-13-rv-032&)

21. Yokoshima Y, Sumitani M, Nishizawa D, Nagashima M, Ikeda K, Kato R, Hozumi J, Abe H, Azuma K, Tsuchida R, Yamada Y; Japanese TR-Cancer Pain Research Group.Gamma-aminobutyric acid transaminase genetic polymorphism is a candidate locus for responsiveness to opioid analgesics in patients with cancer pain: An exploratory study. Neuropsychopharmacol Rep. 2018. December;38(4):175–181  [DOI](https://doi.org/10.1002/npr2.12030) | [PMC free article](/articles/PMC7292329/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30277654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacol%20Rep&title=Gamma-aminobutyric%20acid%20transaminase%20genetic%20polymorphism%20is%20a%20candidate%20locus%20for%20responsiveness%20to%20opioid%20analgesics%20in%20patients%20with%20cancer%20pain:%20An%20exploratory%20study&author=Y%20Yokoshima&author=M%20Sumitani&author=D%20Nishizawa&author=M%20Nagashima&author=K%20Ikeda&volume=38&issue=4&publication_year=2018&pages=175-181&pmid=30277654&doi=10.1002/npr2.12030&)

22. Robishaw J, Caceres J, Hennekens CH. Genomics and Precision Medicine to Combat Opioid Use Disorder. Am J Med 2018. doi.org/10.1016/j.amjmed.2018.10.014  [org/10.1016/j.amjmed.2018.10.014](http://org/10.1016/j.amjmed.2018.10.014) | [DOI](https://doi.org/10.1016/j.amjmed.2018.10.014) | [PMC free article](/articles/PMC10320768/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30940352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Med&title=Genomics%20and%20Precision%20Medicine%20to%20Combat%20Opioid%20Use%20Disorder&author=J%20Robishaw&author=J%20Caceres&author=CH%20Hennekens&publication_year=2018&pmid=30940352&doi=10.1016/j.amjmed.2018.10.014&)

23. Muriel J, Margarit C, Barrachina J, Ballester P, Flor A, Morales D, Horga JF, Fernández E, Peiró AM. Pharmacogenetics and prediction of adverse events in prescription opioid use disorder patients. Basic Clin Pharmacol Toxicol. 2018. October  [DOI](https://doi.org/10.1111/bcpt.13155) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30549211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin%20Pharmacol%20Toxicol&title=Pharmacogenetics%20and%20prediction%20of%20adverse%20events%20in%20prescription%20opioid%20use%20disorder%20patients&author=J%20Muriel&author=C%20Margarit&author=J%20Barrachina&author=P%20Ballester&author=A%20Flor&publication_year=2018&pmid=30549211&doi=10.1111/bcpt.13155&)

24. Gerrish AW, Fogel S, Lockhart ER, et al. Opioid prescribing practices during implementation of an enhanced recovery program at a tertiary care hospital. Surgery. 2018;164:674–679.  [DOI](https://doi.org/10.1016/j.surg.2018.06.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30098812/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Surgery&title=Opioid%20prescribing%20practices%20during%20implementation%20of%20an%20enhanced%20recovery%20program%20at%20a%20tertiary%20care%20hospital&author=AW%20Gerrish&author=S%20Fogel&author=ER%20Lockhart&volume=164&publication_year=2018&pages=674-679&pmid=30098812&doi=10.1016/j.surg.2018.06.034&)

25. Smith DM, Weitzel KW, Elsey AR, et al. CYP2D6-guided opioid therapy improves pain control in CYP2D6 intermediate and poor metabolizers: a pragmatic clinical trial. Genet Med. 2019. doi: 10.1038/s41436-018-0431-8  [DOI](https://doi.org/10.1038/s41436-018-0431-8) | [PMC free article](/articles/PMC6650382/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30670877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=CYP2D6-guided%20opioid%20therapy%20improves%20pain%20control%20in%20CYP2D6%20intermediate%20and%20poor%20metabolizers:%20a%20pragmatic%20clinical%20trial&author=DM%20Smith&author=KW%20Weitzel&author=AR%20Elsey&publication_year=2019&pmid=30670877&doi=10.1038/s41436-018-0431-8&)
