# Dutch Pharmacogenetics Working Group (DPWG) guideline for the gene–drug interaction of DPYD and fluoropyrimidines

## Metadata
**Authors:** Carin A T C Lunenburg, Cathelijne H van der Wouden, Marga Nijenhuis, Mandy H Crommentuijn-van Rhenen, Nienke J de Boer-Veger, Anne Marie Buunk, Elisa J F Houwink, Hans Mulder, Gerard A Rongen, Ron H N van Schaik, Jan van der Weide, Bob Wilffert, Vera H M Deneer, Jesse J Swen, Henk-Jan Guchelaar
**Journal:** European Journal of Human Genetics
**Date:** 2019 Nov 19
**DOI:** [10.1038/s41431-019-0540-0](https://doi.org/10.1038/s41431-019-0540-0)
**PMID:** 31745289
**PMCID:** PMC7080718
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7080718/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7080718/pdf/41431_2019_Article_540.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7080718/pdf/41431_2019_Article_540.pdf)

## Abstract

Despite advances in the field of pharmacogenetics (PGx), clinical acceptance has remained limited. The Dutch Pharmacogenetics Working Group (DPWG) aims to facilitate PGx implementation by developing evidence-based pharmacogenetics guidelines to optimize pharmacotherapy. This guideline describes the starting dose optimization of three anti-cancer drugs (fluoropyrimidines: 5-fluorouracil, capecitabine and tegafur) to decrease the risk of severe, potentially fatal, toxicity (such as diarrhoea, hand-foot syndrome, mucositis or myelosuppression). Dihydropyrimidine dehydrogenase (DPD, encoded by the DPYD gene) enzyme deficiency increases risk of fluoropyrimidine-induced toxicity. The DPYD-gene activity score, determined by four DPYD variants, predicts DPD activity and can be used to optimize an individual’s starting dose. The gene activity score ranges from 0 (no DPD activity) to 2 (normal DPD activity). In case it is not possible to calculate the gene activity score based on DPYD genotype, we recommend to determine the DPD activity and adjust the initial dose based on available data. For patients initiating 5-fluorouracil or capecitabine: subjects with a gene activity score of 0 are recommended to avoid systemic and cutaneous 5-fluorouracil or capecitabine; subjects with a gene activity score of 1 or 1.5 are recommended to initiate therapy with 50% the standard dose of 5-fluorouracil or capecitabine. For subjects initiating tegafur: subjects with a gene activity score of 0, 1 or 1.5 are recommended to avoid tegafur. Subjects with a gene activity score of 2 (reference) should receive a standard dose. Based on the DPWG clinical implication score, DPYD genotyping is considered “essential”, therefore directing DPYD testing prior to initiating fluoropyrimidines.

Subject terms: Prognostic markers, Chemotherapy, Haplotypes

## Introduction

The role of heritable genetic variation on drug response is referred to as pharmacogenetics (PGx). Germline mutations in pharmacogenetic loci can predict phenotypic differences in drug response and can be used to guide dose and drug selection to achieve safer and more (cost-) effective pharmacotherapy. PGx guided pharmacotherapy is one of the first clinical applications of genomics in medicine. Despite scientific and clinical advances in the field of PGx, clinical adoption has remained limited. Barriers preventing implementation have been previously reported [[1](#CR1)]. Some of these barriers have been overcome in the past years. One of these barriers was the lack of clear guidelines on how to interpret and apply PGx test results.

The Royal Dutch Pharmacists Association (KNMP) established the Dutch Pharmacogenetics Working Group (DPWG) in 2005 to overcome this barrier [[2](#CR2)]. The main objectives of the DPWG are (1) to develop PGx informed therapeutic recommendations based on systematic literature review, and (2) to assist physicians and pharmacists by integrating the recommendations into computerized systems for drug prescription, dispensing and automated medication surveillance. This manuscript thus provides both the content required for enabling local translation of assay results into the predicted phenotype (in this case the gene activity score) and for programming therapeutic recommendations into local clinical decision support systems. With the objective of implementing PGx into routine care, the DPWG has additionally developed the clinical implication score, which is given to every gene–drug interaction. The aim of this score is to direct clinicians on whether or not to order relevant PGx genotyping tests before initiating therapy. Recently, the DPWG guidelines were endorsed by the European Association of Clinical Pharmacology and Therapeutics and the European Association of Hospital Pharmacists [[3](#CR3), [4](#CR4)]. Other initiatives such as the Clinical Pharmacogenetics Implementation Consortium (CPIC) were also established to support clinical implementation [[5](#CR5), [6](#CR6)].

The DPWG is a multidisciplinary group in which (clinical) pharmacists, physicians, clinical pharmacologists, clinical chemists and epidemiologists are represented. From 2005 onwards, the DPWG has systematically executed 90 risk analyses for potential gene–drug interactions resulting in 49 guidelines providing therapeutic recommendations for one or more aberrant phenotypes [[7](#CR7)]. Available DPWG guidelines and future updates will be published in an effort to provide transparency of their development and to fulfil the public demand for their publication.

This guideline describes the starting dose optimization of three anti-cancer drugs (fluoropyrimidines: 5-fluorouracil, capecitabine and tegafur) to decrease the risk of severe, potentially fatal, toxicity; such as diarrhoea, hand-foot syndrome, mucositis or myelosuppression. Dihydropyrimidine dehydrogenase enzyme (DPD) deficiency (which is encoded by the *DPYD* gene) increases the risk of fluoropyrimidine-induced toxicity. The gene activity score is currently based on the results of four *DPYD* variants, predicts DPD enzyme activity and is used to optimize an individual’s starting dose. The gene activity score ranges from 0 (no DPD activity) to 2 (normal DPD activity). This manuscript provides an overview of the guideline development and summarizes the pharmacotherapeutic recommendations. In addition, a comparison to alternative guidelines is presented. The “gene–drug interaction” section includes background on the pharmacological mechanism of the interaction. In addition it also includes a list of the *DPYD* variants associated with toxicity and the method developed by DPWG for local translation of assay results into the gene activity score. This information may be useful for laboratories to select and design a *DPYD* genotyping assay and subsequently determine the patients’ predicted phenotype based on the genotype results. Consequently, the literature review supporting the *DPYD-*fluoropyrimidine interaction is described and the DPWG guideline is presented. A summary of all references identified by the systematic review that were subsequently used to develop this guideline, can be found in Supplementary Tables [1](#MOESM1) and [2](#MOESM1). The recommendations provided in this manuscript can be used in combination with a patients’ predicted phenotype to optimize starting dose of fluoropyrimidines, thereby decreasing the risk of severe and potentially fatal toxicity.

### Drugs: fluoropyrimidines (5-fluorouracil, capecitabine and tegafur with DPD-inhibitors)

Fluoropyrimidines are antimetabolite drugs widely used in the treatment of colorectal, breast, stomach and skin cancer. Each year, over two million patients worldwide receive treatment with fluoropyrimidines. This includes 5-FU and its oral pro-drugs capecitabine and tegafur. Up to 30% of patients experience severe toxicity (Common Terminology Criteria for Adverse Events, CTCAE, Grade ≥3), including diarrhoea, hand-foot syndrome, mucositis and myelosuppression. For ~1% of patients toxicity is fatal [[8](#CR8), [9](#CR9)]. Toxicity may occur within the first treatment cycle (early onset), supporting the importance of optimizing the starting dose of fluoropyrimidine pharmacotherapy on a personalized basis, before initiating therapy [[10](#CR10)].

Capecitabine is metabolised into 5-FU in three consecutive steps. Capecitabine is firstly metabolised to 5′-deoxy-5-fluorocytidine (5′-DFCR) by carboxylesterase, subsequently, 5′-DFCR is converted into 5′-deoxy-5-fluorouridine (5′-DFUR) by cytidine deaminase, and to 5-FU by thymidine phosphorylase. 5-FU is metabolised in tissues to 5-fluoro-2′-deoxyuridine and then to 5-fluoro-2′-deoxyuridine-5′-monophosphate, the active metabolite of the drug. The active metabolite inhibits the enzyme thymidylate synthase, resulting in inhibition of DNA synthesis and repair, inducing cell apoptosis and thus, its effect. In addition, toxic effects resulting from partial incorporation of 5-FU and its metabolites in DNA and RNA contribute to the drug’s mechanism of action [[11](#CR11)].

Tegafur is metabolised into 5-FU and into the less cytotoxic metabolites 3-hydroxytegafur, 4-hydroxytegafur and dihydrotegafur by CYP2A6. The less toxic metabolites are renally cleared. Tegafur was combined with the DPD inhibitor uracil and is now combined with the DPD inhibitor gimeracil and the orotate phoshoribosyltransferase inhibitor oteracil. Oteracil diminishes the activity of 5-FU in normal gastrointestinal mucosa. The DPD inhibitors diminish the formation of functionally inactive metabolites of 5-FU that contribute to adverse events like stomatitis and mucositis. Both uracil and gimeracil inhibit DPD activity reversibly and have a shorter elimination half-life and thus shorter period of action than tegafur. For this reason, genetic variants influencing DPD enzyme activity are clinically relevant for tegafur in combination with DPD inhibitors.

### Gene: dihydropyrimidine dehydrogenase (DPYD)

The *DPYD* gene encodes the enzyme DPD. *DPYD* is located on chromosome 1p21.3, and transcription variant 1 ([NM_000110.3](https://www.ncbi.nlm.nih.gov/nuccore/NM_000110.3)) has 26 exons, spanning ~900 kb [[12](#CR12)]. Over 160 different allele variants in *DPYD* have been identified and described in literature [[13](#CR13)]. According to the gnomAD browser [[14](#CR14)], which contains whole exome data of almost 140,000 individuals, *DPYD* contains 2190 known variants. The prevalence of individual variants is low. The effect of genetic variation on DPD enzyme activity is not fully established for the majority of variants and the size of the effect can differ between variants.

The frequency of the various *DPYD* variants and the associated phenotypes appears to vary significantly between nations and ethnic groups. For example, in the Caucasian population, ~3–5% has a partial DPD enzyme deficiency and 0.1–0.2% has a complete DPD enzyme deficiency. On the other hand, ~8% of the African American population has a partial DPD enzyme deficiency [[15](#CR15), [16](#CR16)].

### Gene–drug interaction

#### Pharmacological mechanism

A schematic overview of fluoropyrimidine metabolism is shown in Fig. [1](#Fig1). The DPD enzyme is mainly found in liver, but also intestinal mucosa, leucocytes, tumour cells and other tissues. Over 80% of 5-FU is inactivated to 5-fluoro-5,6-dihydrouracil (DHFU) by DPD. The decreased metabolic activity of DPD leads to increased intracellular concentrations of active metabolites of 5-FU [[17](#CR17)]. The increased intracellular concentration of 5-fluoro-2′-deoxyuridine-5′-monophosphate (FdUMP) increases the risk of toxicity such as diarrhoea, hand-foot syndrome, mucositis and myelosuppression. Variants in the *DPYD* gene can result in reduced or even absent DPD enzyme activity, increasing the risk of severe toxicity. For example, 73% of the patients with *DPYD**2A experienced severe toxicity when treated with a full dose, compared with 23% of *1 allele carriers (wild-type patients) who experienced toxicity [[18](#CR18)]. Many enzymes are involved in fluoropyrimidine metabolism, however, this guideline is limited to the role of the DPD enzyme in causing toxicity.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/773b/7080718/9eed57759762/41431_2019_540_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7080718_41431_2019_540_Fig1_HTML.jpg)

Schematic overview of fluoropyrimidine metabolism. In brief: tegafur, 5FU, capecitabine are metabolised into three major metabolites. FdUMP, which inhibits TS and prevents conversion of dUMP to dTMP, which is necessary for pyrimidine and DNA synthesis. FdUTP is incorporated in DNA, FdUTP is incorporated in RNA, both resulting in cell death. CES carboxylesterase, CDA cytidine deaminase, 5′dFCR 5′-deoxy-5-fluorocytidine, 5′dFUR 5′-deoxy-5-fluorouridine, 5-FU 5-fluorouracil, TP thymidine phosphorylase, DPYD gene encoding DPD, DPD dihydropyrimidine dehydrogenase, DHFU 5,6-dihydrofluorouracil, FUPA fluoro-ß-ureidopropionate, F-ß-AL Fluoro-ß-alanine, FUMP fluorouridine monophosphate, FUDP fluorouridine diphosphate, FUTP fluorouridine triphosphate, FUDR fluorodeoxyuridine, FdUMP fluorodeoxyuridine monophosphate, FdUDP fluorodeoxyuridine diphosphate, FdUTP fluorodeoxyuridine triphosphate, dUMP deoxyuridine monophosphate, dTMP deoxythymidine monophosphate, TS thymidylate synthase, TYMS gene encoding TS

Since the genetic variation in *DPYD* only partially determines DPD enzyme activity, these guidelines for dose adjustment based on the predicted phenotype are no more than a tool that can be used to achieve the desired intracellular concentration of the active metabolite, to minimize risk of toxicity. The absence of tested variants does not eliminate the risk of toxicity. Pharmacokinetic dose adjustment (guided by steady-state plasma concentrations or AUC) may also be useful to optimize the dose of 5-FU. This is, however, currently not routinely used for capecitabine and tegafur, as they are mainly converted into 5-FU within tissue.

#### DPYD variants associated with toxicity

The variants known or suspected to have an effect on DPD enzyme activity, are listed in Table [1](#Tab1). These variants are mapped by the level of evidence for which association with toxicity has been established (columns) and the variant’s effect on DPD enzyme activity (rows). Novel variants in *DPYD* will continue to be identified with the introduction of NGS techniques to clinical practice. The *DPYD* variants reported in this manuscript are reported on [www.PharmGKB.org](http://www.PharmGKB.org), which is referred to on [https://databases.lovd.nl/shared/genes/DPYD](https://databases.lovd.nl/shared/genes/DPYD). However, in order for these variants to be included in Table [1](#Tab1), sufficient evidence regarding the effect on enzyme function or the onset of toxicity must be investigated, possibly by using the *DPYD-*Varifier [[19](#CR19)] or by phenotyping patients who carry a novel variant. An update of this guideline will be published when a renewed recommendation is given following newly published articles.

| Level of evidence | Sufficient evidencea | Insufficient evidenceb |
| --- | --- | --- |
| DPD enzyme activity |  |  |
| Fully functionalc | DPYD*5 = c.1627A>G DPYD*9A = c.85T>C | DPYD*4 = c.1601G>A DPYD*6 = c.2194G>A DPYD*9B = c.[85T>C;2657G>A] DPYD*11 = c.1003 G > T c.496A>G c.1896T>C c.1129-15T>C (IVS10-15T>C) |
| Reduced functionalityd | c.2846A>T c.[1236G>A;1129–5923C>G] (hapB3)e |   |
| Fully dysfunctionalf | DPYD*2A = c.1905 + 1G>A DPYD*13 = c.1679T>G | DPYD*3 = c.1898del DPYD*7 = c.299_302del DPYD*8 = c.703C>T DPYD*10 = c.2983G>T DPYD*12 = c.[62 G > A;1156 G > T] c.1651G>A c.300 C>Ag c.1024 G>Ag c.1025 A>Gg c.1475 C>Tg c.1774C>Tg c.(2058+1_2059-1)_(2299+1_2300-1)dup c.257C>Tg |

Table Caption: Known DPYD variants stratified by level of evidence on the association with toxicity and predicted DPD enzyme activity

#### Translation of genotype to predicted phenotype

The DPWG has concluded that four variants have sufficient evidence to be implemented into clinical care: *DPYD**2A (c.1905+1G>A, IVS14+1G>A), *DPYD**13 (c.1679T>G), c.2846A>T and c.1236G>A (in linkage disequilibrium with c.1129–5923C>G). The current guideline only reports recommendations for these four variants; no recommendations are provided for other variants in *DPYD* or other genes. The results of this genotyping panel can be used to predict a patient’s phenotype, i.e. the DPD enzyme activity. This predicted DPD activity can be expressed as the *DPYD-*gene activity score, which ranges from 0 (no or virtually no DPD enzyme activity) to 2 (normal DPD enzyme activity due to homozygosity for fully functional alleles, both assigned an activity score 1). The gene activity score is a sum of the two activities of protein isoforms expressed from both alleles. The development of the gene activity score is published elsewhere [[20](#CR20)]. Although the gene activity score 0.5 can be predicted based on genotype, the predicted enzyme activity is not reliable. Therefore the gene activity score 0.5 has been replaced with predict the gene activity score correctly (PHENO). A simplified translation of genotypes to predicted phenotypes as gene activity scores, is presented in Table [2](#Tab2). The included variants are those for which substantial and sufficient evidence on the relation to severe toxicity has been established. It is a limitation to restrict to these four variants, as other variants may influence DPD activity as well. However, not all variants having a possible effect on DPD enzyme activity may have been identified yet or evidence for identified variants is insufficient. Therefore, this may result in the incorrect prediction of the DPD enzyme activity. Another limitation is that currently used genotyping methods are unable to determine the allelic location of the variants, but only the dichotomous presence or absence of the variant. This becomes a limitation when two or more different genetic variants are identified in a patient. In this case, either both genetic variants may be on the same allele, resulting in a genotype with one fully functional allele and one reduced functionality allele, or alternatively, both genetic variants may reside on different alleles, resulting in two alleles with inactive or reduced functionality. The latter is more likely to occur. The total gene activity score, however, differs between these cases. When the DPD enzyme activity cannot be predicted correctly, an additional phenotyping test is required to determine the DPD enzyme activity. The relationship between genotype result and predicted phenotype in patients carrying no variants or one or more variants leading to decreased DPD enzyme activity are shown in Supplementary Table [3](#MOESM1). The frequency of individuals carrying two or more of four variants considered in the current guideline is rare, but can be assigned a gene activity score. A complete genotype to predicted phenotype translation table can be found in Supplementary Table [4](#MOESM1), which can be used to programme the translation of genotype results into predicted phenotypes in laboratory information systems.

| Patient genotype | Gene activity score |
| --- | --- |
| Carrier of no variants associated with either reduced functionality or fully dysfunctional DPD activity (*1/*1) | Gene activity score 2 |
| Carrier of one variant associated with reduced functionality of DPD activity (*1/c.1236G>A or *1/c.2846A>T) | Gene activity score 1.5 |
| Carrier of one variant associated with fully dysfunctional DPD activity (*1/*2A or *1/*13) | Gene activity score 1 |
| Carrier of two variants associated with reduced functionality of DPD activity (for example c.1236G>A/c.2846A>T) Or Carrier of one variant associated with reduced functionality of DPD activity and one variant associated with fully dysfunctional DPD activity (combinations of c.2846A>T or c.1236G>A with *2A or *13, example given *2A/c.2846A>T) | PHENO: DPD enzyme activity cannot be predicted correctly, an additional phenotyping test is required to determine the DPD enzyme activity |
| Carrier of two variants associated with fully dysfunctional DPD activity (*2A/*2A or *13/*13 or *2A/*13) | Gene activity score 0 |

Table Caption: The translation of genotypes to predicted phenotypes, as gene activity scores

#### Additional phenotyping test when genotype is unable to predict phenotype

In contrast to the *DPYD* genotyping test, which aims to predict DPD enzyme activity, a DPD phenotyping test can be performed to measure the actual DPD enzyme activity. Possible methods to perform phenotyping are to measure the DPD enzyme activity in peripheral blood mononuclear cells (PBMCs) or to measure the uracil concentrations in plasma or urine [[21](#CR21)]. The average Caucasian DPD enzyme activity is 9.9 ± 0.95 nmol/h per mg protein [[22](#CR22)]. Less commonly performed methods include: (1) the 2-^13^C-uracil breath test [[23](#CR23)], where ^13^C0_2_ is measured, which is a product of 2-C^13^-uracil degradation by DPD and other enzymes involved in the katabolic route of pyrimidines; (2) the quantification of the uracil/dihydrouracil ratio in plasma, where endogenous substrates uracil and dihydrouracil are measured [[24](#CR24), [25](#CR25)], although recently it was shown that uracil levels were superior to the dihydrouracil/uracil ratio as a predictor of severe toxicity [[26](#CR26)]; and (3) measurement the metabolism of a single dose of uracil [[27](#CR27)]. However, all DPD phenotyping tests have their limitations. Currently, the DPD enzyme activity measurements from PBMCs are considered the best developed DPD phenotyping test in The Netherlands [[27](#CR27), [28](#CR28)].

### Supporting body of evidence

A detailed description of the methods used for literature collection, assessment and preparation of the gene-drug monograph has previously been published elsewhere [[2](#CR2), [7](#CR7)]. In brief, a systematic review of literature was performed and relevant articles were summarized by a scientist of the Royal Dutch Pharmacists Association (MN). The performed search strategy can be found in Supplementary Material [1](#MOESM1). Each article was provided with two scores: (1) quality of evidence and (2) clinical impact. The quality of evidence was scored on a five-point scale ranging from 0 (lowest—data on file) to 4 (highest—well performed controlled studies or meta-analysis) and the clinical impact of clinical effect was scored on a seven-point scale ranging from AA^#^ (positive effect) to F (highest negative effect). The criteria used to develop these scores have been published in detail previously [[2](#CR2), [7](#CR7)]. This clinical impact scale (AA^#^-F) runs parallel to the Common Terminology Criteria for Adverse Events (CTCAE); where CTCAE grade 5 severity is equal to clinical relevance score F (death) and CTCAE grade 1 severity is equal to clinical relevance score B. The clinical relevance score additionally includes the scores AA^#^, AA and A, since these do not exist in the CTCAE. These regard “positive clinical effect”, “no clinical or kinetic effect”, and “significant kinetic effect or not clinically relevant effect”, respectively. The summaries of articles, and their respective scores, reviewed to devise this guideline can be found in the Supplementary Tables [1](#MOESM1) and [2](#MOESM1). The summaries of each article and their respective scores were checked by two independent DPWG members.

For 5-FU/capecitabine, the initial literature search was performed on March 24, 2009, followed by a second and third search on July 9, 2014 and October 19, 2017. To update this guideline to the current date, an additional literature search was performed on January 30, 2019. Case reports concerning systemic 5-FU or capecitabine therapy were excluded in this literature review, due to a large number of case reports and other available publications of greater evidentiary quality. Kinetic studies from 2009 onwards were only included if the kinetic parameters were given per genotype. Clinical studies were only included if the patient numbers exceeded 500 (from 2009 onwards) or 1000 (from May 2014 onwards) and the patient numbers with partially functional activity were at least ten or if the study investigated a variant for which no studies were as yet included or if the study investigated the effect of dose adjustment. From 2009, articles investigating the effect of a group containing both polymorphisms known to increase the risk of toxicity and polymorphisms not known to increase the risk of toxicity were not included. If more than one article described data of the same patient group and the same polymorphisms, only the article with data from the largest amount of patients was included.

For tegafur, the initial literature search was performed on August 20, 2009, followed by a second, third and fourth search on October 2, 2012, July 27, 2015, and October 19, 2017. To update this guideline to current date, an additional literature search was performed on January 30, 2019.

### General conclusion of evidence

In the systematic review performed for 5-FU/capecitabine, 18 of 20 studies and all three meta-analyses found an increased risk of grade ≥3 toxicity (either overall toxicity or at least one specified type of toxicity) for patients carrying variants resulting in reduced DPD enzyme activity (ranging from gene activity score 0 to 1.5, and PHENO). This increased risk was shown separately for patients assigned *DPYD-*gene activity scores 1 and 1.5, but gene activity scores 0 and PHENO were only investigated when grouped with patients assigned other gene activity scores. However, the increased risk of toxicity for patients assigned gene activity scores 0 and PHENO can be concluded based on the confirmed association for gene activity scores 1 and 1.5, where deficiency is less, and is further supported by cases of patients assigned gene activity scores 0 and PHENO who developed severe toxicity. Only one study investigating clinical outcome concluded there was no effect of variants on risk of toxicity. Based on the systematic review, the DPWG concludes that a gene–drug interaction is present and that DPD enzyme deficiency increases risk of severe toxicity in patients using capecitabine/5-FU. The highest quality of evidence concluding a gene–drug interaction was scored 4.

In the systematic review performed for tegafur with the DPD inhibitor uracil, one case report described four patients who used standard doses and developed severe toxicity. These patients were assigned *DPYD*-gene activity scores 1 and 1.5. Toxicity (CTCAE grade 4) was similar to that reported in patients treated with 5-FU or capecitabine, both of which are given without a DPD inhibitor. There were no data available for patients assigned *DPYD-*gene activity score 0 or PHENO, however, the increased risk of toxicity among these patients can be concluded based on the confirmed association with toxicity for gene activity scores 1 and 1.5, where deficiency is less. Based on the systematic review, the DPWG concludes that there is a clinically relevant gene–drug interaction present and that DPD enzyme deficiency increases risk of severe toxicity in patients using tegafur with DPD inhibitors. The highest quality of evidence concluding a gene–drug interaction was scored 2.

### Pharmacotherapeutic recommendations

The DPWG therapeutic recommendation using a patient’s pre-therapeutic PGx test result to optimize starting dose of 5-FU/capecitabine and tegafur with DPD inhibitors is summarized in Supplementary Tables [5](#MOESM1) and [6](#MOESM1), respectively.

In brief, for patients initiating 5-fluorouracil or capecitabine: subjects with a gene activity score of 0 are recommended to avoid both systemic and cutaneous 5-fluorouracil or capecitabine, alternatively, DPD enzyme activity may be determined to adjust the systemic dose accordingly; subjects with a gene activity score of 1 or 1.5 are recommended to initiate therapy with 50% of the standard dose of 5-fluorouracil or capecitabine. Further titration of the dose is possible, guided by toxicity. If genotype results cannot PHENO, for example due to multiple identified variants, it is advised to determine the DPD enzyme activity to define an initial starting dose. For patients initiating tegafur, a gene activity score of 0, 1 or 1.5 recommends avoiding tegafur; when this is not possible, starting with a low dose and titrating dose based upon toxicity. If genotype results cannot PHENO, for example due to multiple identified variants, it is advised to determine the DPD enzyme activity to define an initial starting dose. A gene activity score of 2 (reference value) does not result in a recommendation for dose adaptation for 5-FU, capecitabine or tegafur.

Where possible, dose adjustments have been calculated based on 5-FU clearance or AUC after administration of 5-FU or capecitabine. Data were also extrapolated to tegafur with DPD inhibitor, as this compound also follows the same catabolic and anabolic routes after conversion to 5-FU after clearance of the DPD inhibitor from the body. Data on 5-FU clearance are only available for patients carrying *DPYD**1/*2A, *DPYD**1/c.2846A>T and *DPYD**2A/c.2846A>T. There are data from one patient with *DPYD**1/*13 who developed severe toxicity after 5-FU use, from one patient with c.2846A>T/c.2846A>T and from one patient with c.1236G>A/c.2846A>T.

See Supplementary Tables [7](#MOESM1) and [8](#MOESM1) for an overview of suggested pop-up texts for electronic prescribing systems for pharmacists and physicians. These can be used to programme alerts into the clinical decision support system (CDSS). The guidelines and background information will be available on PharmGKB.org.

### Implications for clinical practice

There is currently an ongoing debate regarding whether and which single drug–gene pairs should be implemented into routine care. Points of debate include the amount of evidence that is necessary supporting effectiveness of pre-emptive genotyping, the cost-effectiveness of the intervention and reimbursement of PGx testing [[29](#CR29), [30](#CR30)]. This inconclusive debate seems to have hampered implementation of drug–gene pairs which seem ready for implementation [[1](#CR1), [31](#CR31)]. In an effort to overcome this inconclusiveness and to direct clinicians on whether or not to order relevant PGx genotyping tests before initiating therapy, the DPWG has developed the clinical implication score. The pre-emptive PGx results for a certain drug–gene pair can be scored as: essential, beneficial, potentially beneficial or not required. The development of these categories and the systematic scoring criteria are discussed elsewhere [[32](#CR32)]. In brief, the implications for clinical practice are based on a list of four criteria regarding the following: the clinical effect associated with the gene–drug interaction, the level of evidence supporting the clinical effect, the effectiveness of the intervention in preventing the clinical effect (which includes the number needed to genotype) and the PGx information included in the drug-label. The scores provided for each of these criteria by the DPWG can be found in Supplementary Table [9](#MOESM1).

As a result, the DPWG has concluded the clinical implication score of *DPYD*-fluoropyrimidines to be “essential”. This score dictates that *DPYD* genotyping prior to treatment must be performed for all patients initially being prescribed therapy with 5-FU, capecitabine or tegafur with DPD inhibitors, to optimize the initial dose and to prevent potentially fatal toxicity.

### Differences between available guidelines

Other guidelines regarding the gene–drug interaction of *DPYD* and fluoropyrimidines have been developed. To the best of our knowledge, guidelines are available from CPIC [[11](#CR11), [33](#CR33)], French (French Network of Pharmacogenetics—RNPGx) [[34](#CR34)] and Italian (Associazione Italiana di Oncologia Medica—AIOM-SIF) [unpublished guidelines, *edited by the AIOM-SIF Working Group*] initiatives. We have compared the DPWG guidelines with other available guidelines published in English. This regards only the CPIC guideline, since the French and Italian guidelines are unpublished or not in English.

#### CPIC

Differences between CPIC and DPWG methodology, genotype to phenotype conversion and recommendations have previously been described in detail [[6](#CR6)]. However, both guidelines have been updated [[5](#CR5), [33](#CR33), [35](#CR35), [36](#CR36)]. The current DPWG and CPIC guidelines [[5](#CR5)] for *DPYD*/fluoropyrimidines differ regarding the therapeutic recommendations. In contrast to CPIC, DPWG distinguishes between 5-FU/capecitabine and tegafur within the therapeutic recommendations for fluoropyrimidines, where the CPIC guideline does not provide any dosing recommendations for tegafur due to the limited available evidence. DPWG also further distinguishes between systemic and cutaneous routes of administration within the 5-FU/capecitabine recommendations. The therapeutic recommendations for 5-FU/capecitabine also differ regarding the following: (1) For patients with gene activity score 0: DPWG recommends phenotyping while CPIC does not when no alternative is available. (2) For patients with gene activity score PHENO DPWG recommends phenotyping to determine starting dose or selection of an alternative whereas CPIC recommends an alternative or a strongly reduced dose with therapeutic drug monitoring. (3) DPWG recommends to phenotype all homozygous carriers of any variant, whereas CPIC recommends to adjust the dose for a homozygous carrier of c.2846A>T with more than 50%.

## Supplementary information

## Acknowledgments

### Funding

The U-PGx consortium received funding from the European Community’s Horizon 2020 Programme under grant agreement No. 668353 (U-PGx). The DPWG received funding from the Royal Dutch Pharmacists Association. CL was previously supported by an unrestricted grant from Roche Pharmaceuticals. There was no involvement in the study design, data collection, analysis or interpretation of the data.

### Disclaimer

The Pharmacogenetics Working Group of the KNMP (DPWG) formulates the optimal recommendations for each phenotype group based on the available evidence. If this optimal recommendation cannot be followed due to practical restrictions, e.g. therapeutic drug monitoring or a lower dose is not available, then the health care professional should consider the next best option.

## Data availability

All data and material are either included in the supplementary information or publicly available (i.e. the published articles, PubMed). The guidelines and background information will be available on PharmGKB.org.

## Compliance with ethical standards

### Conflict of interest

The authors declare that they have no conflict of interest.

## Footnotes

## Supplementary information

The online version of this article (10.1038/s41431-019-0540-0) contains supplementary material, which is available to authorized users.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All data and material are either included in the supplementary information or publicly available (i.e. the published articles, PubMed). The guidelines and background information will be available on PharmGKB.org.

### Supplementary Materials

### Data Availability Statement

All data and material are either included in the supplementary information or publicly available (i.e. the published articles, PubMed). The guidelines and background information will be available on PharmGKB.org.

## References

1. Swen JJ, Huizinga TW, Gelderblom H, De Vries EG, Assendelft WJ, Kirchheiner J, et al. Translating pharmacogenomics: challenges on the road to the clinic. PLoS Med. 2007;4:e209. doi: 10.1371/journal.pmed.0040209.  [DOI](https://doi.org/10.1371/journal.pmed.0040209) | [PMC free article](/articles/PMC1945038/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17696640/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Med&title=Translating%20pharmacogenomics:%20challenges%20on%20the%20road%20to%20the%20clinic&author=JJ%20Swen&author=TW%20Huizinga&author=H%20Gelderblom&author=EG%20De%20Vries&author=WJ%20Assendelft&volume=4&publication_year=2007&pages=e209&pmid=17696640&doi=10.1371/journal.pmed.0040209&)

2. Swen JJ, Wilting I, de Goede AL, Grandia L, Mulder H, Touw DJ, et al. Pharmacogenetics: from bench to byte. Clin Pharm Ther. 2008;83:781–787. doi: 10.1038/sj.clpt.6100507.  [DOI](https://doi.org/10.1038/sj.clpt.6100507) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18253145/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Ther&title=Pharmacogenetics:%20from%20bench%20to%20byte&author=JJ%20Swen&author=I%20Wilting&author=AL%20de%20Goede&author=L%20Grandia&author=H%20Mulder&volume=83&publication_year=2008&pages=781-787&pmid=18253145&doi=10.1038/sj.clpt.6100507&)

3. European Association for Clinical Pharmacology and Therapeutics (EACPT). Available from: https://www.eacpt.eu/. [Accessed November 2017].  [https://www.eacpt.eu/](https://www.eacpt.eu/)

4. European Association of Hospital Pharmacists (EAHP). Available from: https://www.eahp.eu/. [Accessed November 2017].  [https://www.eahp.eu/](https://www.eahp.eu/)

5. Amstutz U, Henricks LM, Offer SM, Barbino J, Schellens JHM, Swen JJ, et al. Clinical pharmacogenetics implementation consortium (CPIC) guideline for dihydropyrimidine dehydrogenase genotype and fluoropyrimidine dosing: 2017 update. Clin Pharm Ther. 2018;103:210–216. doi: 10.1002/cpt.911.  [DOI](https://doi.org/10.1002/cpt.911) | [PMC free article](/articles/PMC5760397/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29152729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Ther&title=Clinical%20pharmacogenetics%20implementation%20consortium%20(CPIC)%20guideline%20for%20dihydropyrimidine%20dehydrogenase%20genotype%20and%20fluoropyrimidine%20dosing:%202017%20update&author=U%20Amstutz&author=LM%20Henricks&author=SM%20Offer&author=J%20Barbino&author=JHM%20Schellens&volume=103&publication_year=2018&pages=210-216&pmid=29152729&doi=10.1002/cpt.911&)

6. Bank PC, Caudle KE, Swen JJ, Gammal RS, Whirl-Carrillo M, Klein TE, et al. Comparison of the guidelines of the clinical pharmacogenetics implementation consortium and the Dutch pharmacogenetics working group. Clin Pharmacol Ther. 2018;103:599–618.  [DOI](https://doi.org/10.1002/cpt.762) | [PMC free article](/articles/PMC5723247/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28994452/)

7. Swen JJ, Nijenhuis M, De Boer BA, Grandia L, Maitland-van der Zee AH, Mulder H, et al. Pharmacogenetics: from bench to byte-an update of guidelines. Clin Pharm Ther. 2011;89:662–673. doi: 10.1038/clpt.2011.34.  [DOI](https://doi.org/10.1038/clpt.2011.34) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21412232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Ther&title=Pharmacogenetics:%20from%20bench%20to%20byte-an%20update%20of%20guidelines&author=JJ%20Swen&author=M%20Nijenhuis&author=BA%20De%20Boer&author=L%20Grandia&author=AH%20Maitland-van%20der%20Zee&volume=89&publication_year=2011&pages=662-673&pmid=21412232&doi=10.1038/clpt.2011.34&)

8. Rosmarin D, Palles C, Pagnamenta A, Kaur K, Pita G, Martin M, et al. A candidate gene study of capecitabine-related toxicity in colorectal cancer identifies new toxicity variants at DPYD and a putative role for ENOSF1 rather than TYMS. Gut. 2015;64:111–120. doi: 10.1136/gutjnl-2013-306571.  [DOI](https://doi.org/10.1136/gutjnl-2013-306571) | [PMC free article](/articles/PMC4283622/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24647007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut&title=A%20candidate%20gene%20study%20of%20capecitabine-related%20toxicity%20in%20colorectal%20cancer%20identifies%20new%20toxicity%20variants%20at%20DPYD%20and%20a%20putative%20role%20for%20ENOSF1%20rather%20than%20TYMS&author=D%20Rosmarin&author=C%20Palles&author=A%20Pagnamenta&author=K%20Kaur&author=G%20Pita&volume=64&publication_year=2015&pages=111-120&pmid=24647007&doi=10.1136/gutjnl-2013-306571&)

9. Saltz LB, Cox JV, Blanke C, Rosen LS, Fehrenbacher L, Moore MJ, et al. Irinotecan plus fluorouracil and leucovorin for metastatic colorectal cancer. Irinotecan Study Group. N Engl J Med. 2000;343:905–914. doi: 10.1056/NEJM200009283431302.  [DOI](https://doi.org/10.1056/NEJM200009283431302) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11006366/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Irinotecan%20plus%20fluorouracil%20and%20leucovorin%20for%20metastatic%20colorectal%20cancer.%20Irinotecan%20Study%20Group&author=LB%20Saltz&author=JV%20Cox&author=C%20Blanke&author=LS%20Rosen&author=L%20Fehrenbacher&volume=343&publication_year=2000&pages=905-914&pmid=11006366&doi=10.1056/NEJM200009283431302&)

10. Froehlich TK, Amstutz U, Aebi S, Joerger M, Largiader CR. Clinical importance of risk variants in the dihydropyrimidine dehydrogenase gene for the prediction of early-onset fluoropyrimidine toxicity. Int J Cancer. 2015;136:730–739. doi: 10.1002/ijc.29025.  [DOI](https://doi.org/10.1002/ijc.29025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24923815/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Clinical%20importance%20of%20risk%20variants%20in%20the%20dihydropyrimidine%20dehydrogenase%20gene%20for%20the%20prediction%20of%20early-onset%20fluoropyrimidine%20toxicity&author=TK%20Froehlich&author=U%20Amstutz&author=S%20Aebi&author=M%20Joerger&author=CR%20Largiader&volume=136&publication_year=2015&pages=730-739&pmid=24923815&doi=10.1002/ijc.29025&)

11. Thorn CF, Marsh S, Carrillo MW, McLeod HL, Klein TE, Altman RB. PharmGKB summary: fluoropyrimidine pathways. Pharmacogenet Genomics. 2011;21:237–242.. doi: 10.1097/FPC.0b013e32833c6107.  [DOI](https://doi.org/10.1097/FPC.0b013e32833c6107) | [PMC free article](/articles/PMC3098754/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20601926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=PharmGKB%20summary:%20fluoropyrimidine%20pathways&author=CF%20Thorn&author=S%20Marsh&author=MW%20Carrillo&author=HL%20McLeod&author=TE%20Klein&volume=21&publication_year=2011&pages=237-242.&pmid=20601926&doi=10.1097/FPC.0b013e32833c6107&)

12. NCBI: National Center for Biotechnology Information. DPYD dihydropyrimidine dehydrogenase [Homo sapiens (human)]. Gene. 2017. Available from: https://www.ncbi.nlm.nih.gov/gene/1806. [Accessed October 2017].  [https://www.ncbi.nlm.nih.gov/gene/1806](https://www.ncbi.nlm.nih.gov/gene/1806)

13. Toffoli G, Giodini L, Buonadonna A, Berretta M, De Paoli A, Scalone S, et al. Clinical validity of a DPYD-based pharmacogenetic test to predict severe toxicity to fluoropyrimidines. Int J Cancer. 2015;137:2971–2980. doi: 10.1002/ijc.29654.  [DOI](https://doi.org/10.1002/ijc.29654) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26099996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Clinical%20validity%20of%20a%20DPYD-based%20pharmacogenetic%20test%20to%20predict%20severe%20toxicity%20to%20fluoropyrimidines&author=G%20Toffoli&author=L%20Giodini&author=A%20Buonadonna&author=M%20Berretta&author=A%20De%20Paoli&volume=137&publication_year=2015&pages=2971-2980&pmid=26099996&doi=10.1002/ijc.29654&)

14. Genome Aggregation Database (gnomAD). GnomAD browser (Beta) 2017. Available from: http://gnomad.broadinstitute.org/. [Accessed July 2017].  [http://gnomad.broadinstitute.org/](http://gnomad.broadinstitute.org/)

15. Ensembl: Gene: DPYD ENSG00000188641. 2017. Available from: http://www.ensembl.org/Homo_sapiens/Gene/Variation_Gene/Table?db=core;g=ENSG00000188641;r=1:97077743-97921049. [Accessed July 2017].  [http://www.ensembl.org/Homo_sapiens/Gene/Variation_Gene/Table?db=core;g=ENSG00000188641;r=1:97077743-97921049](http://www.ensembl.org/Homo_sapiens/Gene/Variation_Gene/Table?db=core;g=ENSG00000188641;r=1:97077743-97921049)

16. Elraiyah T, Jerde CR, Shrestha S, Wu R, Nie Q, Giama NH, et al. Novel deleterious dihydropyrimidine dehydrogenase variants may contribute to 5-fluorouracil sensitivity in an East African population. Clin Pharm Ther. 2017;101:382–390. doi: 10.1002/cpt.531.  [DOI](https://doi.org/10.1002/cpt.531) | [PMC free article](/articles/PMC5309195/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27727460/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Ther&title=Novel%20deleterious%20dihydropyrimidine%20dehydrogenase%20variants%20may%20contribute%20to%205-fluorouracil%20sensitivity%20in%20an%20East%20African%20population&author=T%20Elraiyah&author=CR%20Jerde&author=S%20Shrestha&author=R%20Wu&author=Q%20Nie&volume=101&publication_year=2017&pages=382-390&pmid=27727460&doi=10.1002/cpt.531&)

17. van Kuilenburg AB. Dihydropyrimidine dehydrogenase and the efficacy and toxicity of 5-fluorouracil. Eur J Cancer. 2004;40:939–950. doi: 10.1016/j.ejca.2003.12.004.  [DOI](https://doi.org/10.1016/j.ejca.2003.12.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15093568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=Dihydropyrimidine%20dehydrogenase%20and%20the%20efficacy%20and%20toxicity%20of%205-fluorouracil&author=AB%20van%20Kuilenburg&volume=40&publication_year=2004&pages=939-950&pmid=15093568&doi=10.1016/j.ejca.2003.12.004&)

18. Deenen MJ, Meulendijks D, Cats A, Sechterberger MK, Severens JL, Boot H, et al. Upfront genotyping of DPYD*2A to individualize fluoropyrimidine therapy: a safety and cost analysis. J Clin Oncol. 2016;34:227–234. doi: 10.1200/JCO.2015.63.1325.  [DOI](https://doi.org/10.1200/JCO.2015.63.1325) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26573078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Upfront%20genotyping%20of%20DPYD*2A%20to%20individualize%20fluoropyrimidine%20therapy:%20a%20safety%20and%20cost%20analysis&author=MJ%20Deenen&author=D%20Meulendijks&author=A%20Cats&author=MK%20Sechterberger&author=JL%20Severens&volume=34&publication_year=2016&pages=227-234&pmid=26573078&doi=10.1200/JCO.2015.63.1325&)

19. Shrestha S, Zhang C, Jerde CR, Nie Q, Li H, Offer SM, et al. Gene-specific variant classifier (DPYD-Varifier) to identify deleterious alleles of dihydropyrimidine dehydrogenase. Clin Pharmacol Ther. 2018;104:709–18.  [DOI](https://doi.org/10.1002/cpt.1020) | [PMC free article](/articles/PMC6043412/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29327356/)

20. Henricks LM, Lunenburg CATC, Meulendijks D, Gelderblom H, Cats A, Swen JJ, et al. Translating DPYD genotype into DPD phenotype: using the DPYD gene activity score. Pharmacogenomics. 2015;16(11):1277–86.  [DOI](https://doi.org/10.2217/pgs.15.70) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26265346/)

21. Ploylearmsaeng SA, Fuhr U, Jetter A. How may anticancer chemotherapy with fluorouracil be individualised? Clin Pharmacokinet. 2006;45:567–592. doi: 10.2165/00003088-200645060-00002.  [DOI](https://doi.org/10.2165/00003088-200645060-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16719540/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=How%20may%20anticancer%20chemotherapy%20with%20fluorouracil%20be%20individualised?&author=SA%20Ploylearmsaeng&author=U%20Fuhr&author=A%20Jetter&volume=45&publication_year=2006&pages=567-592&pmid=16719540&doi=10.2165/00003088-200645060-00002&)

22. Van Kuilenburg ABP, Van Lenthe H, Tromp A, Veltman PC, Van Gennip AH. Pitfalls in the diagnosis of patients with a partial dihydropyrimidine dehydrogenase deficiency. Clin Chem. 2000;46:9–17. doi: 10.1093/clinchem/46.1.9.  [DOI](https://doi.org/10.1093/clinchem/46.1.9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10620566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Pitfalls%20in%20the%20diagnosis%20of%20patients%20with%20a%20partial%20dihydropyrimidine%20dehydrogenase%20deficiency&author=ABP%20Van%20Kuilenburg&author=H%20Van%20Lenthe&author=A%20Tromp&author=PC%20Veltman&author=AH%20Van%20Gennip&volume=46&publication_year=2000&pages=9-17&pmid=10620566&doi=10.1093/clinchem/46.1.9&)

23. Mattison LK, Ezzeldin H, Carpenter M, Modak A, Johnson MR, Diasio RB. Rapid identification of dihydropyrimidine dehydrogenase deficiency by using a novel 2-13C-uracil breath test. Clin Cancer Res. 2004;10:2652–2658. doi: 10.1158/1078-0432.CCR-03-0374.  [DOI](https://doi.org/10.1158/1078-0432.CCR-03-0374) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15102667/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Rapid%20identification%20of%20dihydropyrimidine%20dehydrogenase%20deficiency%20by%20using%20a%20novel%202-13C-uracil%20breath%20test&author=LK%20Mattison&author=H%20Ezzeldin&author=M%20Carpenter&author=A%20Modak&author=MR%20Johnson&volume=10&publication_year=2004&pages=2652-2658&pmid=15102667&doi=10.1158/1078-0432.CCR-03-0374&)

24. Ciccolini J, Mercier C, Evrard A, Dahan L, Boyer JC, Duffaud F, et al. A rapid and inexpensive method for anticipating severe toxicity to fluorouracil and fluorouracil-based chemotherapy. Ther Drug Monit. 2006;28:678–685. doi: 10.1097/01.ftd.0000245771.82720.c7.  [DOI](https://doi.org/10.1097/01.ftd.0000245771.82720.c7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17038885/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=A%20rapid%20and%20inexpensive%20method%20for%20anticipating%20severe%20toxicity%20to%20fluorouracil%20and%20fluorouracil-based%20chemotherapy&author=J%20Ciccolini&author=C%20Mercier&author=A%20Evrard&author=L%20Dahan&author=JC%20Boyer&volume=28&publication_year=2006&pages=678-685&pmid=17038885&doi=10.1097/01.ftd.0000245771.82720.c7&)

25. Zhou ZW, Wang GQ, Wan de S, Lu ZH, Chen YB, Li S, et al. The dihydrouracil/uracil ratios in plasma and toxicities of 5-fluorouracil-based adjuvant chemotherapy in colorectal cancer patients. Chemotherapy. 2007;53:127–131. doi: 10.1159/000099984.  [DOI](https://doi.org/10.1159/000099984) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17308379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chemotherapy&title=The%20dihydrouracil/uracil%20ratios%20in%20plasma%20and%20toxicities%20of%205-fluorouracil-based%20adjuvant%20chemotherapy%20in%20colorectal%20cancer%20patients&author=ZW%20Zhou&author=GQ%20Wang&author=S%20Wan%20de&author=ZH%20Lu&author=YB%20Chen&volume=53&publication_year=2007&pages=127-131&pmid=17308379&doi=10.1159/000099984&)

26. Meulendijks D, Henricks LM, Jacobs BAW, Aliev A, Deenen MJ, De Vries N, et al. Pretreatment serum uracil concentration as a predictor of severe and fatal fluoropyrimidine-associated toxicity. Br J Cancer. 2017;116:1415–1424. doi: 10.1038/bjc.2017.94.  [DOI](https://doi.org/10.1038/bjc.2017.94) | [PMC free article](/articles/PMC5520099/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28427087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Pretreatment%20serum%20uracil%20concentration%20as%20a%20predictor%20of%20severe%20and%20fatal%20fluoropyrimidine-associated%20toxicity&author=D%20Meulendijks&author=LM%20Henricks&author=BAW%20Jacobs&author=A%20Aliev&author=MJ%20Deenen&volume=116&publication_year=2017&pages=1415-1424&pmid=28427087&doi=10.1038/bjc.2017.94&)

27. van Staveren MC, Guchelaar HJ, van Kuilenburg ABP, Gelderblom H, Maring JG. Evaluation of predictive tests for screening for dihydropyrimidine dehydrogenase deficiency. Pharmacogenomics J. 2013;13:389–395. doi: 10.1038/tpj.2013.25.  [DOI](https://doi.org/10.1038/tpj.2013.25) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23856855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Evaluation%20of%20predictive%20tests%20for%20screening%20for%20dihydropyrimidine%20dehydrogenase%20deficiency&author=MC%20van%20Staveren&author=HJ%20Guchelaar&author=ABP%20van%20Kuilenburg&author=H%20Gelderblom&author=JG%20Maring&volume=13&publication_year=2013&pages=389-395&pmid=23856855&doi=10.1038/tpj.2013.25&)

28. Meulendijks D, Cats A, Beijnen JH, Schellens JH. Improving safety of fluoropyrimidine chemotherapy by individualizing treatment based on dihydropyrimidine dehydrogenase activity—ready for clinical practice? Cancer Treat Rev. 2016;50:23–34. doi: 10.1016/j.ctrv.2016.08.002.  [DOI](https://doi.org/10.1016/j.ctrv.2016.08.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27589829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Treat%20Rev&title=Improving%20safety%20of%20fluoropyrimidine%20chemotherapy%20by%20individualizing%20treatment%20based%20on%20dihydropyrimidine%20dehydrogenase%20activity%E2%80%94ready%20for%20clinical%20practice?&author=D%20Meulendijks&author=A%20Cats&author=JH%20Beijnen&author=JH%20Schellens&volume=50&publication_year=2016&pages=23-34&pmid=27589829&doi=10.1016/j.ctrv.2016.08.002&)

29. Pirmohamed M, Hughes DA. Pharmacogenetic tests: the need for a level playing field. Nat Rev Drug Discov. 2013;12:3–4. doi: 10.1038/nrd3921.  [DOI](https://doi.org/10.1038/nrd3921) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23274456/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Drug%20Discov&title=Pharmacogenetic%20tests:%20the%20need%20for%20a%20level%20playing%20field&author=M%20Pirmohamed&author=DA%20Hughes&volume=12&publication_year=2013&pages=3-4&pmid=23274456&doi=10.1038/nrd3921&)

30. Altman RB. Pharmacogenomics: “noninferiority” is sufficient for initial implementation. Clin Pharm Ther. 2011;89:348–350. doi: 10.1038/clpt.2010.310.  [DOI](https://doi.org/10.1038/clpt.2010.310) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21326263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Ther&title=Pharmacogenomics:%20%E2%80%9Cnoninferiority%E2%80%9D%20is%20sufficient%20for%20initial%20implementation&author=RB%20Altman&volume=89&publication_year=2011&pages=348-350&pmid=21326263&doi=10.1038/clpt.2010.310&)

31. Lunenburg CATC, Henricks LM, Guchelaar HJ, Swen JJ, Deenen MJ, Schellens JHM, et al. Prospective DPYD genotyping to reduce the risk of fluoropyrimidine-induced severe toxicity: ready for prime time. Eur J Cancer. 2016;54:40–48. doi: 10.1016/j.ejca.2015.11.008.  [DOI](https://doi.org/10.1016/j.ejca.2015.11.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26716401/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=Prospective%20DPYD%20genotyping%20to%20reduce%20the%20risk%20of%20fluoropyrimidine-induced%20severe%20toxicity:%20ready%20for%20prime%20time&author=CATC%20Lunenburg&author=LM%20Henricks&author=HJ%20Guchelaar&author=JJ%20Swen&author=MJ%20Deenen&volume=54&publication_year=2016&pages=40-48&pmid=26716401&doi=10.1016/j.ejca.2015.11.008&)

32. Swen JJ, Nijenhuis M, van Rhenen M, De Boer-Veger NJ, Buunk AM, Houwink EJF, et al. Pharmacogenetic information in clinical guidelines: the European Perspective. Clin Pharmacol Ther. 2018;103:795–801.  [DOI](https://doi.org/10.1002/cpt.1049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29460273/)

33. Caudle KE, Thorn CF, Klein TE, Swen JJ, McLeod HL, Diasio RB, et al. Clinical Pharmacogenetics implementation consortium guidelines for dihydropyrimidine dehydrogenase genotype and fluoropyrimidine dosing. Clin Pharm Ther. 2013;94:640–645. doi: 10.1038/clpt.2013.172.  [DOI](https://doi.org/10.1038/clpt.2013.172) | [PMC free article](/articles/PMC3831181/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23988873/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Ther&title=Clinical%20Pharmacogenetics%20implementation%20consortium%20guidelines%20for%20dihydropyrimidine%20dehydrogenase%20genotype%20and%20fluoropyrimidine%20dosing&author=KE%20Caudle&author=CF%20Thorn&author=TE%20Klein&author=JJ%20Swen&author=HL%20McLeod&volume=94&publication_year=2013&pages=640-645&pmid=23988873&doi=10.1038/clpt.2013.172&)

34. Loriot MA, Ciccolini J, Thomas F, Barin-Le-Guellec C, Royer B, Milano G, et al. Dihydropyrimidine dehydrogenase (DPD) deficiency screening and securing of fluoropyrimidine-based chemotherapies: update and recommendations of the French GPCO-Unicancer and RNPGx networks. Bull Cancer. 2018;105:397–407. doi: 10.1016/j.bulcan.2018.02.001.  [DOI](https://doi.org/10.1016/j.bulcan.2018.02.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29486921/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bull%20Cancer&title=Dihydropyrimidine%20dehydrogenase%20(DPD)%20deficiency%20screening%20and%20securing%20of%20fluoropyrimidine-based%20chemotherapies:%20update%20and%20recommendations%20of%20the%20French%20GPCO-Unicancer%20and%20RNPGx%20networks&author=MA%20Loriot&author=J%20Ciccolini&author=F%20Thomas&author=C%20Barin-Le-Guellec&author=B%20Royer&volume=105&publication_year=2018&pages=397-407&pmid=29486921&doi=10.1016/j.bulcan.2018.02.001&)

35. CPIC: Clinical Pharmacogenetics Implementation Consortium. Annotation of CPIC guideline for capecitabine and DPYD. PharmGKB; 2018 Available from: https://www.pharmgkb.org/guidelineAnnotation/PA166109594. [Accessed November 2018].  [https://www.pharmgkb.org/guidelineAnnotation/PA166109594](https://www.pharmgkb.org/guidelineAnnotation/PA166109594)

36. KNMP: Royal Dutch Society for the Advancement of Pharmacy. Fluorouracil/Capecitabine DPD gene activity score and guidelines. KNMP, Royal Dutch Society for the Advancement of Pharmacy; 2015. Available from: https://kennisbank.knmp.nl/article/farmacogenetica/2552-4893-4894.html. [Accessed May 2017].  [https://kennisbank.knmp.nl/article/farmacogenetica/2552-4893-4894.html](https://kennisbank.knmp.nl/article/farmacogenetica/2552-4893-4894.html)
