# Genomewide Association Study of Platelet Reactivity and Cardiovascular Response in Patients Treated With Clopidogrel: A Study by the International Clopidogrel Pharmacogenomics Consortium

## Metadata
**Authors:** Shefali Setia Verma, Thomas O Bergmeijer, Li Gong, Jean‐Luc Reny, Joshua P Lewis, Braxton D Mitchell, Dimitrios Alexopoulos, Daniel Aradi, Russ B Altman, Kevin Bliden, Yuki Bradford, Gianluca Campo, Kiyuk Chang, John H Cleator, Jean‐Pierre Déry, Nadia P Dridi, Israel Fernandez‐Cadenas, Pierre Fontana, Meinrad Gawaz, Tobias Geisler, Gian Franco Gensini, Betti Giusti, Paul A Gurbel, Willibald Hochholzer, Lene Holmvang, Eun‐Young Kim, Ho‐Sook Kim, Rossella Marcucci, Joan Montaner, Joshua D Backman, Ruth E Pakyz, Dan M Roden, Elke Schaeffeler, Matthias Schwab, Jae Gook Shin, Jolanta M Siller‐Matula, Jurriën M ten Berg, Dietmar Trenk, Marco Valgimigli, John Wallace, Ming‐Shien Wen, Michiaki Kubo, Ming Ta Michael Lee, Ryan Whaley, Stefan Winter, Teri E Klein, Alan R Shuldiner, Marylyn D Ritchie, for the ICPC Investigators
**Journal:** Clinical Pharmacology and Therapeutics
**Date:** 2020 Jul 9
**DOI:** [10.1002/cpt.1911](https://doi.org/10.1002/cpt.1911)
**PMID:** 32472697
**PMCID:** PMC7689744
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7689744/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7689744/pdf/CPT-108-1067.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7689744/pdf/CPT-108-1067.pdf)

## Abstract

Antiplatelet response to clopidogrel shows wide variation, and poor response is correlated with adverse clinical outcomes. CYP2C19 loss‐of‐function alleles play an important role in this response, but account for only a small proportion of variability in response to clopidogrel. An aim of the International Clopidogrel Pharmacogenomics Consortium (ICPC) is to identify other genetic determinants of clopidogrel pharmacodynamics and clinical response. A genomewide association study (GWAS) was performed using DNA from 2,750 European ancestry individuals, using adenosine diphosphate‐induced platelet reactivity and major cardiovascular and cerebrovascular events as outcome parameters. GWAS for platelet reactivity revealed a strong signal for CYP2C19*2 (P value = 1.67e−33). After correction for CYP2C19*2 no other single‐nucleotide polymorphism reached genomewide significance. GWAS for a combined clinical end point of cardiovascular death, myocardial infarction, or stroke (5.0% event rate), or a combined end point of cardiovascular death or myocardial infarction (4.7% event rate) showed no significant results, although in coronary artery disease, percutaneous coronary intervention, and acute coronary syndrome subgroups, mutations in SCOS5P1, CDC42BPA, and CTRAC1 showed genomewide significance (lowest P values: 1.07e−09, 4.53e−08, and 2.60e−10, respectively). CYP2C19*2 is the strongest genetic determinant of on‐clopidogrel platelet reactivity. We identified three novel associations in clinical outcome subgroups, suggestive for each of these outcomes.

☑ Antiplatelet response to clopidogrel shows wide variation, and poor response is correlated with adverse clinical outcome. *CYP2C19* loss‐of‐function alleles play an important role in this response, but additional genetic variants may remain unidentified.

☑ The aim of this study was to identify novel genetic loci associated with on‐clopidogrel platelet reactivity and clinical outcome, by performing a genomewide association study of individuals of European ancestry treated with clopidogrel.

☑ A strong association was found for *CYP2C19*2* and adenosine diphosphate stimulated platelet reactivity, while no single‐nucleotide polymorphism reached genomewide significance for major adverse cardiovascular event end points. Nevertheless, we observed significant novel hits in subgroup analyses for patients with coronary artery disease, acute coronary syndrome, and who underwent percutaneous coronary intervention.

☑ Our results support a *CYP2C19* genotype guided antiplatelet approach to tailoring of antiplatelet therapy, which has shown to be of clinical relevance. Nevertheless, a risk score containing other genetic, pharmacodynamic, and/or clinical risk factors might further improve assessment of responsiveness to clopidogrel and optimization of antiplatelet therapy.

Differential response to drug therapy is a common aspect of clinical practice. The causes for interindividual heterogeneity in drug response include environmental, clinical (e.g., sex, age, disease severity, drug‐drug interactions, and adherence), as well as genetic factors. Personalized medicine based on these factors can improve patient care, in particular for drugs with a narrow therapeutic range or when insufficient drug efficacy or drug toxicity can have serious, potentially life‐threatening consequences.[1](#cpt1911-bib-0001), [2](#cpt1911-bib-0002)

The P2Y_12_ inhibiting drug clopidogrel is used in combination with the cyclooxygenase‐1 inhibitor aspirin to prevent (recurrent) atherothrombotic events in patients with acute coronary syndrome (ACS), patients undergoing percutaneous coronary intervention (PCI), and in patients with stroke. Clopidogrel is a thienopyridine pro‐drug that requires bioactivation mediated by hepatic CYP P450 enzymes to inhibit thrombogenesis by irreversibly binding the P2Y_12_ receptor on the surface of platelets.[3](#cpt1911-bib-0003) There is a wide interpatient variability in active metabolite levels and platelet reactivity, influenced by genetic and clinical variables, as well as drug‐drug interactions.[4](#cpt1911-bib-0004), [5](#cpt1911-bib-0005), [6](#cpt1911-bib-0006), [7](#cpt1911-bib-0007), [8](#cpt1911-bib-0008) Both high “on‐treatment platelet reactivity” as well as being a carrier of a *CYP2C19* loss‐of‐function allele are related to a higher risk for (recurrent) atherothrombotic events.[9](#cpt1911-bib-0009), [10](#cpt1911-bib-0010), [11](#cpt1911-bib-0011), [12](#cpt1911-bib-0012)

*CYP2C19* variants, in particular the loss‐of‐function alleles *CYP2C19*2* (rs4244285) and *CYP2C19*3* (rs4986893), have previously been identified as the predominant genetic mediators of active metabolite levels and antiplatelet effect of clopidogrel.[5](#cpt1911-bib-0005), [13](#cpt1911-bib-0013) A genomewide association study (GWAS) in a large Amish population indicated that ~ 70% of the variability in clopidogrel response may be due to genetic factors, with *CYP2C19*2* being the strongest predictor, although this variant only accounted for ~ 12% of the overall variation in platelet reactivity.[5](#cpt1911-bib-0005) Combined with clinical factors (age, body mass index (BMI), and lipid levels) ~ 32% of the variation in pharmacodynamic clopidogrel response could be explained. A study by Frelinger *et al*. conducted in 160 healthy subjects taking clopidogrel, showed that all known genetic and nongenetic factors together accounted for only 18% of the pharmacokinetic variation and 32–64% of clopidogrel pharmacodynamic variation.[14](#cpt1911-bib-0014) In two studies with patients undergoing elective PCI, about 5% of the variability in platelet reactivity could be explained by *CYP2C19* genotype, and about 11–20% when *CYP2C19* genotype was combined with clinical variables.[6](#cpt1911-bib-0006), [15](#cpt1911-bib-0015) Furthermore, clopidogrel nonresponders can be found not only among patients heterozygous or homozygous for *CYP2C19* loss‐of‐function (LOF) alleles, but also in patients without a LOF allele.[16](#cpt1911-bib-0016) These data suggest that novel genetic variants for clopidogrel response remain to be discovered. The clinical utility of *CYP2C19* genotype‐guided strategy for selection of P_2_Y_12_ inhibitors has been demonstrated in a number of recent studies.[17](#cpt1911-bib-0017), [18](#cpt1911-bib-0018) A risk score including both clinical factors and *CYP2C19* LOF alleles has also been developed to identify patients at higher risk for high platelet reactivity and adverse events (ABCD‐GENE score).[19](#cpt1911-bib-0019) However, because LOF *CYP2C19* alleles contribute to only a portion of the variability in the antiplatelet effect of clopidogrel, a strategy relying solely on the basis of well‐known *CYP2C19* LOF alleles may not be the most appropriate for a diverse patient population. A risk score encompassing multiple genetic variants, along with nongenetic factors would be more predictive and helpful in the clinical setting than a single factor alone.

The International Clopidogrel Pharmacogenomics Consortium (ICPC) aims to improve the understanding of clopidogrel pharmacogenomics by combining genetic, pharmacodynamic, and clinical outcome data of patients using clopidogrel.[20](#cpt1911-bib-0020), [21](#cpt1911-bib-0021) In this study, we present the largest GWAS performed to date on patients on clopidogrel to identify novel genetic loci associated with on‐clopidogrel platelet reactivity, major adverse cardiovascular events (MACE), and combined cardiac and cerebrovascular events (MACCE).

## METHODS

The ICPC is an international effort led by the Pharmacogenomics Research Network (PGRN) and Pharmacogenomics Knowledgebase (PharmGKB).[20](#cpt1911-bib-0020) Based on the data published on [www.clinicaltrials.gov](http://www.clinicaltrials.gov) as of June 2011, studies with at least 50 clopidogrel‐treated patients potentially containing genetic and platelet reactivity data were identified for participation. Lead investigators were invited to share DNA samples, platelet reactivity test results, patient characteristics, and cardiovascular outcomes to perform candidate gene and GWAS.[22](#cpt1911-bib-0022) To date, 17 sites from 13 countries have joined the ICPC, contributing data representing 8,829 clopidogrel‐exposed patients. Of those patients, a DNA sample was available in 5,119 patients, a DNA sample and platelet reactivity data in 4,511 patients, and in 2,844 patients a DNA sample, platelet reactivity data, and clinical outcome data were available. Platelet function was measured in patients on clopidogrel maintenance dose or after adequate loading dose, which was defined as at least 2 hours between a 600‐mg clopidogrel loading dose and platelet function testing, 6 hours after 300‐mg clopidogrel loading dose, and 5 days after start of 75‐mg maintenance dose without extra loading dose. Of these, 2,750 were of European ancestry in whom GWAS genotyping was performed. Each study in the ICPC was conducted with institutional review board approval at each respective data collection site and activities of the ICPC determined as exempt from institutional review board review by the University of Maryland under 45 Code of Federal Regulations (CFR) 46.101(b).

DNA samples were made available for genotyping at RIKEN (Japan). Genotyping was performed on the Illumina Human Omni express exome chip. Variants were called using Illumina Beadchip studio. This dataset consisted of 964,193 variants. We imputed the data to 1000 Genomes phase I reference panel using IMPUTE2. Prior to imputation, strand check and phasing were performed using SHAPEIT2. Imputations were performed following best practices guidelines, as previously published.[23](#cpt1911-bib-0023)

Standard quality control measures were conducted using PLINK (version 1.90).[24](#cpt1911-bib-0024) Sex check resulted in dropping 29 samples that were inferred as sex mismatches. We removed samples and markers that did not pass 99% missingness thresholds. Variants that deviated from Hardy–Weinberg Equilibrium (*P* value = 1 × 10^−7^) were flagged. Relatedness among samples was tested using SNPrelate; one sample from each pair of related individuals at a kinship > 0.125 were excluded from the analysis.[25](#cpt1911-bib-0025) This resulted in removal of 20 additional samples. Last, principal component analysis was performed to check for ancestry. We calculated a total of 20 principal components (PCs); PC1 and PC2 explained the most variance and, thus, were used as covariates in the analyses.

GWAS was performed for platelet reactivity and clinical outcomes. Because platelet reactivity was measured using different platelet function tests in each ICPC subcohort, measurements were standardized across these different tests using a priority system laid out by the Phenotype Subcommittee of the ICPC. **Table** [S2](#cpt1911-sup-0006) shows the unique number of patient samples assayed for each platelet function test. First, we validated the association of *CYP2C19*2* (rs4244285) and *CYP2C19*17*(rs12248560) for different platelet reactivity assays used by ICPC sites (**Table** [S1](#cpt1911-sup-0005)). Standardization measures were applied to maximize the number of patients with platelet function tests that were validated based on their association with *CYP2C19**2 and, thus, statistical power for GWAS. The prioritization was as follows: VASP assay > VerifyNow P2Y_12_ > adenosine diphosphate (ADP)‐induced LTA (higher ADP concentration > lower ADP concentration) > other tests.[20](#cpt1911-bib-0020) For each subcohort, one platelet function test was chosen based on the highest‐ranked assay measured at that site that maximized the sample size. A schematic of this is shown in **Figure** [S1](#cpt1911-sup-0001). Standardization of platelet reactivity phenotypes was performed by calculating a *Z*‐score within each study for use in analyses across studies, as previously reported.[20](#cpt1911-bib-0020), [22](#cpt1911-bib-0022) Each selected variable was then standardized with mean of 0 and SD of 1 while grouping by site and the selected variable. Standardized platelet reactivity was used as a continuous response variable in our GWAS.

For the clinical outcomes, we evaluated several different phenotypes, including: (i) MACE: a combined end point consisting cardiovascular death and myocardial infarction; (ii) MACCE: a combined end point consisting of cardiovascular death, myocardial infarction, and stroke; and (iii) individual clinical end points: stent thrombosis, all‐cause death, cardiovascular death, myocardial infarction, stroke, revascularization, major bleeding, minor bleeding, and combined major and minor bleeding. Because of the heterogeneity of the database in diagnosis and risk profile, we also conducted the MACE, MACCE, and stent thrombosis analyses in overlapping subgroups with increasing atherothrombotic risk, including only patients with coronary artery disease, only patients who underwent PCI, and only patients with ACS.

Statistical analysis was performed using PLATO and PLINK software in which linear regression was used for quantitative phenotypes (standardized ADP stimulated platelet reactivity phenotypes) and logistic regression for binary phenotypes (clinical outcome phenotypes).[26](#cpt1911-bib-0026), [27](#cpt1911-bib-0027) Variants with minor allele frequency > 0.0025 were tested. Approximately 5 million (5,009,928) genotyped and imputed variants were evaluated for association. For each analysis, Manhattan and quantile‐quantile plots were generated to visualize the results. GWAS regression models were adjusted for age, sex, and the first 2 PCs (**Text** [S1](#cpt1911-sup-0013)). In platelet reactivity analysis, we aimed to identify novel variants associated with the quantitative phenotype other than the known *CYP2C19*2* variant (rs4244285) or variants in high linkage disequilibrium (LD) with the known variant. Thus, we conducted association testing where regression models were adjusted for *CYP2C19*2* along with age, sex, and the first 2 PCs. We also performed a gene‐based association test using the tool MAGMA as implemented in the web‐based tool FUMA.[28](#cpt1911-bib-0028), [29](#cpt1911-bib-0029) FUMA uses GWAS summary statistics to identify independent significant single‐nucleotide polymorphisms (SNPs) and also independent lead SNPs where the LD *r* ^2^ for each SNP in a genomic locus is < 0.1. MAGMA utilizes summary statistics from SNP‐based tests to map all SNPs to protein coding genes and then a gene‐based test is performed to identify significance of the gene. Gene based *P* values are computed for all SNP mapping to protein coding genes. Functional annotation of SNPs is obtained by ANNOVAR in FUMA.[30](#cpt1911-bib-0030) The results from MAGMA analyses are shown in Manhattan plots simultaneously with SNP‐based Manhattan plots for each phenotype.

SPSS (version 24; IBM, Armonk, NY) was used to analyze the correlation between *CYP2C19* variants and clinical outcome, using a two‐tailed Pearson χ^2^ test and logistic regression for binary and categorical variables. To calculate the adjusted odds ratio, models were adjusted for age, sex, and study center. We did not adjust for other potential covariates, such as diabetes, smoking, etc., because of incomplete data across the ICPC sites; this would have resulted in losing patient‐participants.

## RESULTS

A total of 2,750 ICPC samples of European ancestry were available in this report. After quality control, a total of 2,592 samples were available for GWAS. **Table** [1](#cpt1911-tbl-0001) shows the baseline characteristics of the participants included in the GWAS. We identify that 96% of the samples were prescribed aspirin and 86.2% samples in this study were currently using statins. In our data, we observed 39% of populations are carriers for alternate allele for *CYP2C19*17* and 31.2% population are carriers *for CYP2C19*2* and **3* alleles.

### Table 1.

|   | n (%) or mean ± SD |
| --- | --- |
| Self‐reported race white | 2,592/2,592 (100.0) |
| Sex, male | 1,996/2,592 (77.1) |
| Age, years | 64.6 ± 11.2 |
| BMI, kg/m2 | 27.8 ± 4.6 |
| Diabetes mellitus | 636/2,571 (24.7) |
| Current smoker | 613/2,147 (28.6) |
| Hypercholesterolemia | 1,259/1,951 (64.5) |
| LVEF < 35% | 82/1,020 (8.0) |
| Aspirin use | 2,482/2,585 (96.0) |
| Statin use | 2,141/2,485 (86.2) |
| CYP2C19*2 and/or *3 allele carrier | 812/2,600 (31.2) |
| CYP2C19*17 allele carrier | 980/2,512 (39.0) |
| Coronary artery disease (indication for clopidogrel use) | 2,509/2,592 (96.8) |
| PCI performed | 2,065/2,492 (82.9) |
| Acute coronary syndrome | 1,188/2,492 (47.7) |

Table 1 Caption: Baseline characteristics for all study participants analyzed in the GWAS

### Standardized ADP platelet reactivity GWAS

For the primary platelet reactivity phenotype, in models adjusted for age, sex, and principal components, we observed that the *CYP2C18* locus (rs35835168, most significant *P* value = 3.51e−35) reached genomewide significance. Rs35835168 is in high LD with the known *CYP2C19*2* locus rs4244285 (*r* ^2^ = 0.88 and |D|’ = 1). Rs4244285 has been identified in previously published GWAS for association with response to clopidogrel therapy.[5](#cpt1911-bib-0005) No other loci in the single‐SNP analyses reached genomewide significance (**Figure** [1a](#cpt1911-fig-0001)). The top 30 associations from GWAS are reported in **Table** [S3](#cpt1911-sup-0007). The results from the MAGMA analysis are shown in **Figure** [1b](#cpt1911-fig-0001).[29](#cpt1911-bib-0029) Input SNPs were mapped to 17,964 protein coding genes in the MAGMA analyses, which identified 9 significant genes after using a multiple hypothesis correction *P* value threshold of 2.75e−06 (0.05/17,964). Most genes observed from the gene‐based analyses correspond to a genomic region on chromosome 10 (10:96098093‐96990275), which encodes a CYPP‐450 gene cluster that includes *CYP2C19*, as shown in regional plot **Figure** [1c](#cpt1911-fig-0001) (lower panel). The *SYNJ1* gene on chromosome 21 was also identified as significant from the gene‐based analyses (*P* value = 1.001e−06).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d01/7689744/d6444d0093fd/CPT-108-1067-g001.jpg)

Association results from analyses adjusted by age, sex and PCs (a) Single‐nucleotide polymorphism (SNP)‐based genomewide association study (GWAS) Manhattan plot where chromosome position is on x‐axis and ‐log10 of association P value on y‐axis (genomic inflaction factor = 1.01). (b) Gene‐based GWAS Manhattan plot performed by MAGMA highlighting top 15 genes. (c) Regional plot for chromosome 10 highlighting lead SNP rs35835168. The first panel shows SNPs in linkage disequilibrium (LD) of any significant independent lead SNPs. LD range is represented based on color (blue to red). Second and third panels shows Combined Annotation Dependent Depletion (CADD) and Regulome DB scores, respectively, for only SNPs in LD with lead SNPs.

In an attempt to identify other variants associated with on‐treatment platelet reactivity, we repeated the GWAS, adjusting for *CYP2C19*2*. **Figure** [2a,b](#cpt1911-fig-0002) displays the results from SNP‐based and gene‐based analyses. Top 30 associations from GWAS are reported in **Table** [S4](#cpt1911-sup-0008). Based on the statistical test in FUMA (explained in the Methods section), 16 genomic risk loci consisting of top 17 SNPs were identified. **Figure** [2c](#cpt1911-fig-0002) highlights lead genomic loci, the number of mapped genes for each loci, and also functional annotation of SNPs (and SNPs in LD) using ANNOVAR.[30](#cpt1911-bib-0030) With adjustment for *CYP2C19*2*, no other SNPs reached the genomewide significance threshold (lowest *P* value = 1.59e−07). We explored further suggestively significant results (*P* value < 1.0e−05) to help elucidate the genetic architecture of platelet reactivity response phenotype (**Table** [2](#cpt1911-tbl-0002)). At the *CYP2C19* locus on chromosome 10, variants in *PLCE1* remained nominally associated with on‐treatment platelet reactivity (lowest *P* value = 7.01e−06). Other top hits include association of rs151216272 mapping to *LINGO2* on chromosome 9 (*P* value = 1.59e−07), which has been associated with BMI and neurotic behavior,[31](#cpt1911-bib-0031), [32](#cpt1911-bib-0032) and rs74952072 in *GAPDHP72* on chromosome 6 (*P* value = 1.47e−06), which has been associated with blood pressure, insomnia, and blood urea nitrogen.[33](#cpt1911-bib-0033), [34](#cpt1911-bib-0034), [35](#cpt1911-bib-0035), [36](#cpt1911-bib-0036) Among the lead SNPs is a cluster of 3 variants in the *NR3C2* (rs1546044, rs35464072, and rs13118022) on chromosome 4, which have been previously associated with schizophrenia from GWAS,[37](#cpt1911-bib-0037) rs7276140 in *SYNJ1* on chromosome 21 (*P* value = 4.31e−06) that has been linked with Parkinson’s disease,[38](#cpt1911-bib-0038) and rs2473481 on chromosome 6 (*P* value = 9.74e−06), which maps to the nearest gene *RP1‐20B11.2*, has an expression quantitative trait locus mapping to *EXOC2*, and was previously found to be associated with mean corpuscular hemoglobin.[39](#cpt1911-bib-0039)

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d01/7689744/20e7b4fc0f42/CPT-108-1067-g002.jpg)

Association results from analyses adjusted by age, sex, PCs and CYP2C19*2 variant (a) Single‐nucleotide polymorphism (SNP)‐based genomewide association study (GWAS) Manhattan plot where chromosome position is on x‐axis and ‐log10 of association P value on y‐axis (genomic inflation factor = 1.01). (b) Gene‐based GWAS Manhattan plot of performed by MAGMA highlighting top 15 genes. (c) Summary of lead SNPs identified by the analyses; (d) functional annotation of lead SNPs and SNPs in linkage disequilibrium (LD) using ANNOVAR.

### Table 2.

| SNP | Chromosome | Position | MAF | Gene | P value | Beta | SD | IndSigSNPs |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs151216272 | 9 | 28118945 | 0.01 | LINGO2 | 1.60E−07 | −0.64 | 0.12 | rs151216272 |
| rs35464072 | 4 | 149326236 | 0.45 | NR3C2 | 2.75E−07 | 0.14 | 0.03 | rs35464072; |
| rs1546044; |  |  |  |  |  |  |  |  |
| rs13118022 |  |  |  |  |  |  |  |  |
| rs74952072 | 6 | 166108326 | 0.04 | GAPDHP72 | 1.47E−06 | 0.32 | 0.07 | rs74952072 |
| rs1516568 | 3 | 6949230 | 0.11 | GRM7 | 2.92E−06 | −0.2 | 0.04 | rs1516568 |
| rs57908830 | 22 | 27759178 | 0.03 | MN1 | 2.99E−06 | 0.4 | 0.08 | rs57908830 |
| rs2479921 | 13 | 70749169 | 0.18 | NA | 3.42E−06 | 0.16 | 0.03 | rs2479921 |
| rs9399096 | 6 | 134740060 | 0.07 | NA | 3.59E−06 | −0.25 | 0.05 | rs9399096 |
| rs7276140 | 21 | 34005200 | 0.44 | SYNJ1 | 4.31E−06 | −0.12 | 0.03 | rs7276140 |
| rs117956006 | 13 | 97918928 | < 0.01 | MBNL2 | 5.51E−06 | 1.19 | 0.26 | rs117956006 |
| rs1219603 | 10 | 36543314 | 0.15 | NA | 5.55E−06 | 0.18 | 0.04 | rs1219603 |
| rs61670395 | 18 | 40885561 | 0.06 | NA | 6.01E−06 | −0.25 | 0.06 | rs61670395; |
| rs113478533 |  |  |  |  |  |  |  |  |
| rs76180455 | 10 | 95994508 | 0.02 | PLCE1 | 7.01E−06 | −0.4 | 0.09 | rs76180455 |
| rs10505836 | 12 | 19288508 | 0.16 | PLEKHA5; | 7.24E−06 | 0.17 | 0.04 | rs10505836 |
| SRSF11P1 |  |  |  |  |  |  |  |  |
| rs142225302 | 14 | 97480714 | 0.02 | NA | 8.25E−06 | 0.5 | 0.11 | rs142225302 |
| rs148114323 | 5 | 162707344 | 0.01 | NA | 8.38E−06 | −0.78 | 0.18 | rs148114323 |
| rs140497518 | 14 | 97483211 | 0.02 | NA | 8.83E−06 | 0.49 | 0.11 | rs140497518 |
| rs2473481 | 6 | 532089 | 0.21 | EXOC2 | 9.74E−06 | −0.15 | 0.03 | rs2473481 |

Table 2 Caption: Lead SNPs identified by platelet reactivity response GWAS, adjusted by age, sex, PC1, PC2, project site, and CYP2C19*2 locus

### Clinical outcomes

Outcome data regarding the combined clinical end point where patients were followed for an average of 14 ± 11 months were available for 2,170 (MACE end point) and 1,447 (MACCE end point) patients, with an event rate of 4.7% and 5.0%, respectively. First, univariate and multivariate analyses were performed for the correlation between the *CYP2C19*2* allele and outcome (**Table** [3](#cpt1911-tbl-0003)). For the MACE end point, there was a nonsignificant trend toward a worse outcome for carriers of the *CYP2C19*2* allele (5.8 vs. 4.2%; adjusted odds ratio (OR) 1.31; 95% confidence interval (CI) 0.87–1.99; *P* value = 0.20). This difference became more prominent in the subgroups with patients with higher thrombotic risk, in particular in patients with ACS who underwent PCI (8.3 vs. 4.4%; adjusted OR 1.83; 95% CI 1.06–3.15; *P* value = 0.03). When the MACCE end point was analyzed, this association was not present. In addition, for the individual outcome events, including bleeding end points, no statistically significant association was found in multivariate analysis (**Table** [3](#cpt1911-tbl-0003)).

### Table 3.

| Population | End point | CYP2C19*2 carriers vs. noncarriers | Unadjusted |  | Adjusted a |  |
| --- | --- | --- | --- | --- | --- | --- |
| OR (95% CI) | P value | OR (95% CI) | P value |  |  |  |
| All patients in GWAS | MACE (n = 102/2,170) | 5.8% vs. 4.2% | 1.42 (0.94–2.14) | 0.09 | 1.31 (0.87–1.99) | 0.20 |
|   | MACCE (n = 72/1,447) | 4.4% vs. 5.2% | 0.89 (0.49–1.43) | 0.52 | 0.79 (0.46–1.37) | 0.40 |
|   | Individual end points: |   |   |   |   |   |
|   | All cause death (n = 72/2,580) | 3.3% vs. 2.5% | 1.33 (0.82–2.16) | 0.25 | 1.24 (0.76–1.07) | 0.39 |
|   | Cardiovascular death (n = 40/2,492) | 2.4% vs. 1.2% | 1.99 (1.06–3.73) | 0.028 | 1.82 (0.96–3.45) | 0.065 |
|   | Myocardial infarction (n = 83/2,254) | 3.8% vs. 3.6% | 1.06 (0.66–1.69) | 0.82 | 0.99 (0.61–1.58) | 0.95 |
|   | Stroke (n = 21/1,838) | 1.1% vs. 1.2% | 0.95 (0.37–2.45) | 0.91 | 0.88 (0.33–2.30) | 0.79 |
|   | Stent thrombosis (n = 37/2,579) | 1.2% vs. 1.5% | 0.81 (0.39–1.68) | 0.57 | 0.79 (0.38–1.66) | 0.54 |
|   | Revascularization (n = 332/2,451) | 12.4% vs. 14.1% | 0.87 (0.67–1.12) | 0.26 | 0.86 (0.66–1.13) | 0.28 |
|   | Major bleeding (n = 33/1,703) | 1.8% vs. 2.0% | 0.88 (0.41–1.91) | 0.75 | 0.85 (0.39–1.86) | 0.69 |
|   | Minor bleeding (n = 61/996) | 5.0% vs. 6.6% | 0.75 (0.41–1.36) | 0.34 | 0.76 (0.41–1.42) | 0.39 |
|   | Major + minor bleeding (n = 94/1,703) | 4.7% vs. 5.8% | 0.80 (0.50–1.28) | 0.35 | 0.79 (0.49–1.29) | 0.35 |
| CAD subgroup | MACE (n = 99/2,079) | 6.1% vs. 4.1% | 1.50 (0.99–2.27) | 0.052 | 1.39 (0.92–2.11) | 0.12 |
|   | MACCE (n = 66/1,356) | 4.5% vs. 5.0% | 0.89 (0.51–1.55) | 0.68 | 0.85 (0.48–1.50) | 0.57 |
| PCI subgroup | MACE (n = 73/1,653) | 5.9% vs. 3.7% | 1.63 (1.01–2.62) | 0.043 | 1.47 (0.90–2.39) | 0.12 |
|   | MACCE (n = 30/930) | 2.6% vs. 3.5% | 0.74 (0.31–1.75) | 0.49 | 0.73 (0.30–1.76) | 0.48 |
| ACS subgroup b | MACE (n = 58/1,017) | 8.3% vs. 4.4% | 1.97 (1.16–3.36) | 0.011 | 1.83 (1.06–3.15) | 0.030 |
|   | MACCE (n = 15/459) | 3.2% vs. 3.3% | 0.98 (0.31–3.14) | 0.98 | 1.00 (0.29–3.41) | 1.00 |

Table 3 Caption: Correlation between CYP2C19*2 allele carriers vs. noncarriers and clinical outcome

GWAS of the clinical outcome traits are shown in **Figures** [[Link]](#cpt1911-sup-0002), [[Link]](#cpt1911-sup-0003), [[Link]](#cpt1911-sup-0004) and the top 30 associations are reported in **Tables** [[Link]](#cpt1911-sup-0001), [[Link]](#cpt1911-sup-0009) **–S7**. We did not find any genomewide significant associations with either of the composite clinical outcomes or any of the individual clinical outcome variables. Among the marginally significant results in MACE was variant rs151062494 on chromosome 7 (*P* value = 4.10e−07), and for MACCE variant rs4782918 on chromosome 16 in the *WFDC1* gene (*P* value = 2.63e−06).

We also conducted GWAS for clinical outcomes among the subgroups of patients with coronary artery disease (*n* = 2,509), who underwent PCI (*n* = 2,065), and with ACS (*n* = 1,188), reasoning that there might be stronger genetic determinants of on‐treatment clinical outcomes in patients at higher risk for recurrent events. Genomewide significant results were obtained for MACE (in all subgroups) and stent thrombosis (in the subgroup of patients with coronary artery disease). These results are represented in a composite Manhattan plot shown in **Figure** [3](#cpt1911-fig-0003). All other subgroups resulted in no genomewide significant results. Among the top hits in the coronary artery disease subgroup analyses are SNPs rs151062494 and rs115346894 on chromosome 7, mapped to the nearest gene *SOCS5P1*, and chromosome 1, mapped to the nearest gene *CDC42BPA*, respectively. SNPs mapping to gene *SOCS5P1* are significant in coronary artery disease, ACS, and PCI subgroup analyses as well. Stent thrombosis, coronary artery disease, and PCI subgroup analyses revealed an association in gene *CTRAC1* (*P* value = 2.59e−10 and 7.91e−09, respectively). These results are reported in **Table** [S8](#cpt1911-sup-0012).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d01/7689744/c93ac035dcc9/CPT-108-1067-g003.jpg)

Manhattan plot representing association results from subgroup analyses where patients with coronary artery disease (CAD), acute coronary syndrome (ACS), and percutaneous coronary intervention (PCI) were considered in the analyses as shown in each row. Columns represent phenotype tested (major adverse cardiac event (MACE), major adverse combined cardiac and cerebrovascular event (MACCE), and stent thrombosis). Each Manhattan plot represent chromosome position on x‐axis and ‐log10 (P value) on y‐axis. Red line represents genomewide significance threshold (5e−08).

Finally, we reasoned that variants with suggestive associations with both on‐treatment platelet reactivity and clinical events in the same expected direction may be more likely to represent true positive signals. We highlight clinical outcome analyses for variants that showed significant or suggestive association with on‐treatment platelet reactivity (*P* value < 10e−06) for analyses adjusted with *CYP2C19*2* (**Figure** [4](#cpt1911-fig-0004)).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d01/7689744/b033a28ceb4d/CPT-108-1067-g004.jpg)

Scatter plot representing chr:pos on x‐axis and ‐log10 (P value) on y‐axis for results that are marginally significant in platelet reactivity response genomewide association study (GWAS), adjusted by age, sex, principal componenents (PCs), and CYP2C19*2 (platelet reactivity phenotype P value < 1e−05). Colored points correspond to P values from clinical outcome phenotypes major adverse cardiac event (MACE), major adverse combined cardiac and cerebrovascular event (MACCE), and stent thrombosis (ST). The orientation of triangle refers to positive (up) and negative (down) betas from regression analyses. ADP, adenosine diphosphate.

## DISCUSSION

GWAS provide an agnostic approach to identifying genetic variants that influence human traits. We hypothesized that based on previous studies looking for the genetic factors’ association with response to clopidogrel, additional genetic variants remain unidentified and that these factors may be detected with larger sample sizes. As far as the authors are aware, our current study represents the largest GWAS for clopidogrel response published to date. We found *CYP2C19*2* to have a statistically significant influence on platelet reactivity in patients using clopidogrel, as expected based on previous publications.[9](#cpt1911-bib-0009) However, no new genetic variants reached genomewide significance for on‐treatment platelet reactivity. Although there was a significant association between *CYP2C19*2* and MACE in univariate and multivariate analyses in the patients with the highest ischemic risk (after PCI for ACS), no SNP reached genomewide significance for the clinical end points in the main GWAS analyses using all clopidogrel treated patients in the dataset. These findings provide additional evidence that *CYP2C19*2* is the single major genetic determinant of clopidogrel response in European ancestry individuals.

Two previous GWAS in the Amish population and one GWAS in Asians have been performed for clopidogrel response. First, Shuldiner *et al*. performed a GWAS in healthy Amish individuals and identified *CYP2C19*2* as the only genomewide significant association with on‐treatment platelet reactivity.[5](#cpt1911-bib-0005) A second GWAS for the association with clopidogrel active metabolite levels, performed in 513 Amish individuals derived from the same study population, again showed *CYP2C19*2* to have the strongest correlation with active metabolite levels.[13](#cpt1911-bib-0013) Two more loci were found to reach genomewide significance (rs187941554 on chromosome 3p25 and rs80343429 on chromosome 17q11), of which the second SNP was also significantly associated with on‐treatment platelet reactivity.[13](#cpt1911-bib-0013) Six additional loci showed suggestive evidence of association (*P* value ≤ 1.0e−8), of which four showed a significant association with on‐treatment platelet reactivity. In our study, we did not observe genomewide significance (defined as *P* value = 5e−08) for these variants or variants in LD with them. A smaller GWAS, published by Zhong *et al*., studied clopidogrel response in 115 Chinese patients with coronary artery disease. In this study, no single SNP reached genomewide significance (*P* value < 7.11e−8) for platelet reactivity measured by VerifyNow (PRU cutoff > 208), although 125 SNPs in 25 genes showed suggestive evidence of association (*P* value < 1.0e−4).[40](#cpt1911-bib-0040) Of those 125 SNPs, 27 were also associated with clopidogrel active metabolite levels (*P* value < 0.01), of which 23 were within the *HELLS‐CYP2C18‐CYP2C19* cluster, being in strong LD with one another and with *CYP2C19*2*. Multiple regression analysis showed that a combination of *CYP2C19*2*, rs2254638 in *N6AMT1*, and rs2487032 in *ABCA1* could explain 28.2% of antiplatelet response (10.9%, 14.8%, and 2.5% per SNP, respectively), which increased to 37.7% when clinical factors (use of calcium channel blockers and sex) were added to the model. For active metabolite levels, *CYP2C19*2* (explaining 16.3% of variability), rs2254638 in *A6AMT1* (4.5%), rs12456693 in *SLC4A3* (2.7%), and age (4.8%) were significant predictors. When those SNPs were tested in a group of 299 patients undergoing PCI, with 1.5‐year follow‐up for MACE end points, a significant association was found for rs12913988 in *ATP10A* (*P* = 0.001; odds ratio (OR) for T allele 1.88; 95% CI 1.29–2.74) and a borderline significant result for rs2254638 in *N6AMT1* (*P* value = 0.065; OR for the C allele 1.43; 95% CI 0.98–2.09). *CYP2C19*2* was not associated with MACE end points in this cohort. In our GWAS analyses, the above reported genes were not found to be of genomewide significant association (*P* value < 5e−08) in the clinical outcomes’ analyses (both in all patients and in clinical subgroups of patients).

A recent article published by Lewis *et al*., presented a pharmacogenomic polygenic response score based on 31 candidate gene polymorphisms and tested in patient cohorts from the ICPC. Not all candidate gene variants presented in the above‐mentioned article overlapped with our current analysis due to unavailability of same variants on genotyping platform or not passing all quality control filters from imputed data.[22](#cpt1911-bib-0022) Seven SNPs were identified to have an association with platelet reactivity, including SNPs in *CYP2C19*, *CES1*, *CYP2B6*, and *CYP2C9*. Although none of these SNPs were associated with cardiovascular events when analyzed separately, patients with an increasing number of risk alleles showed a higher cardiovascular event rate. Patients who carried eight or more risk alleles were significantly more likely to experience a cardiovascular event (OR 1.78; 95% CI 1.14–2.76; *P* = 0.01) and cardiovascular death (OR 4.39; 95% CI 1.35–14.27; *n* = 0.01) compared with patients who carried six or less of these alleles.

Significant results identified in our study are in close proximity and high LD with *CYP2C19*, suggesting an essential role in clopidogrel metabolism. Gene‐based analyses also identified several significant genes not yet mentioned in previous studies that are close to the *CYP2C19* cluster (such as *HELLS*, *PLCE1*,* NOC3L*, *TBC1D12*, *CYP2C9*, *CYP2C8*, and *CYP2C18*). MAGMA analyses also identified *SYNJ1* as significant in this association. Mutations in *SYNJ1* are linked to Parkinson’s disease, but its association with platelet reactivity has not been previously described.[41](#cpt1911-bib-0041), [42](#cpt1911-bib-0042) Regression models adjusted for *CYP2C19*2* also demonstrated a suggestive association with *SYNJ1*, and for *NR3C2*, known to be associated with schizophrenia.[37](#cpt1911-bib-0037)

There are some limitations to this study that are worth considering. First, the sample size was insufficient to detect rare variants, even those of moderately large effect sizes. Second, there may have been difficulty in imputing specific rare variants in the GWAS data. For example, the *G143E CES1* variant (rs71647871) has an allele frequency of 0.016 and was found to be highly associated with on‐treatment platelet reactivity in the candidate gene study by Lewis *et al*., as was discussed above, but this variant was not included in our study (it was not genotyped and it did not impute with high quality).[43](#cpt1911-bib-0043) Exome and/or genome sequencing of large cohorts will be required to further understand rare variants such as this one. Third, for this GWAS, only patients from European ancestry were included. Thereby, variants that have low frequency in European ancestry populations but are present at a higher frequency in other populations (e.g., *CYP2C19*3* in Asian populations), would not have been detected in our GWAS. Fourth, the study sites used different methods to measure ADP‐induced platelet reactivity as a marker for clopidogrel efficacy. Although a large GWAS using platelet reactivity measured with a single device would have been best, this was not possible across study sites of the ICPC and, thus, we applied a standardization approach across all studies in order to maximize sample size and power for GWAS discovery. We observed the *CYP2C19*2* and **17* association as expected in assay stratified analyses as well as with the standardized phenotype. We believe that this positive control demonstrates the validity of our phenotype harmonization. However, we acknowledge that the correlation of platelet reactivity between different devices is limited and for some tests, laboratory dependent. Unfortunately, sample size varied markedly for each assay and there was insufficient power to perform GWAS for each individual test. In addition, platelet reactivity is likely influenced by timing after clopidogrel loading and dose. We believe medication compliance was not a major factor because all patients were tested shortly after clopidogrel initiation of a thrombotic event. Additionally, clinical factors, such as age, diabetes, smoking status, BMI, statins use, aspirin use, and drug‐drug interactions, in addition to factors related to the testing method, like hematocrit levels and platelet, also play a role in influencing platelet reactivity.[4](#cpt1911-bib-0004), [6](#cpt1911-bib-0006), [7](#cpt1911-bib-0007), [44](#cpt1911-bib-0044), [45](#cpt1911-bib-0045) However, due to variable missingness of data across sites, we were not able to adjust our analyses for these factors. These nongenetic factors may decrease the sensitivity of our GWAS to identify loci for platelet reactivity. Another potential limitation is that we could not evaluate whether aspirin had any effect in our study; a total of 96% patients in our cohort were taking low‐dose aspirin.

With regard to clinical outcomes, the power to identify genomewide associations with individual clinical outcomes was limited due to the small number of outcome events. Thus, our analyses for clinical outcomes is highly exploratory and hypothesis generating. In addition, our dataset contains a patient population with relatively low risk for (recurrent) events, with most patients treated after elective PCI. This might explain the findings that although *CYP2C19*2* has been linked to clinical outcomes in previous studies, we did not identify this signal in the GWAS performed here in the overall sample, but did detect nominal association of *CYP2C19*2* with clinical outcomes in the subgroups at higher ischemic risk (in particular in patients after PCI for ACS). That said, several potential novel candidates were identified among the subgroup of samples with MACE or stent thrombosis outcomes (*CDC42BPA*, *CTRAC1*, and *SOCS5P1*). These associations are based on small sample sizes, however, and will need further replication in larger, well‐powered studies.

To have an effect on everyday clinical practice, genetic determinants affecting clopidogrel efficacy must demonstrate clinical utility and be easily integrated into patient care. For the *CYP2C19*2* and **3* polymorphisms, point‐of‐care and laboratory‐based testing is available, which makes it feasible to tailor antiplatelet therapy at the bedside.[46](#cpt1911-bib-0046), [47](#cpt1911-bib-0047), [48](#cpt1911-bib-0048) The recently published randomized, open‐label, assessor‐blinded *CYP2C19* Genotype‐Guided Antiplatelet Therapy in ST‐Segment Elevation Myocardial Infarction Patients—Patient Outcome after Primary PCI (POPular Genetics) trial tested a strategy of *CYP2C19*‐guided antiplatelet therapy in 2,488 patients with ST‐segment elevation myocardial infarction undergoing primary PCI, prescribing ticagrelor or prasugrel to *CYP2C19*2* or **3* LOF allele carriers and clopidogrel to noncarriers, in comparison to a standard treatment arm in which all patients were prescribed ticagrelor or prasugrel.[17](#cpt1911-bib-0017) The study showed a significant lower event rate for bleeding events in the genotype‐guided arm, without increase in thrombotic events. The randomized Tailored Antiplatelet Therapy Following PCI (TAILOR‐PCI) trial, of which results are expected to be published soon, uses a comparable strategy (prescribing ticagrelor in patients with a *CYP2C19* LOF allele and clopidogrel in noncarriers), but in patients after PCI for stable coronary artery disease or ACS.[18](#cpt1911-bib-0018)

When additional genetic determinants of clopidogrel response are identified, one could imagine the creation of a risk score composed of several genetic variants, along with nongenetic factors that would be more predictive than single factors alone. Our GWAS, however, suggests that there are no additional common variants with an effect size as great as that of *CYP2C19*. Therefore, our results strengthen the strategy of POPular Genetics to use *CYP2C19* genotyping in clinical practice to optimize antiplatelet therapy. An example of a risk score using clinical risk factors and genetic variants is the recently published ABCD‐GENE score, which shows a good predictive value for patients with high on‐clopidogrel platelet reactivity and clinical outcome based on age, BMI, kidney failure, diabetes, and *CYP2C19* genotype.[19](#cpt1911-bib-0019)

Larger studies, studies in non‐European ancestry populations, and/or sequencing efforts to identify rare variants not tagged by GWAS are directions of potential future research.

## Funding

The ICPC research reported in this publication was supported by the National Heart, Lung, and Blood Institute (NHLBI) of the NIH Award Number U01HL105198 and NIH National Institute of General Medical Sciences grant R24GM61374. The content is solely the responsibility of the authors and does not necessarily represent the official views of the NIH. Genomewide SNP genotyping was supported by the Pharmacogenomics Research Network & CGM Global Alliance. Drs. Schwab, Schaeffeler, and Winter are supported by the Deutsche Forschungsgemeinschaft (DFG), Germany (grant number SCHW858/1‐2) and, in part, by the EU Horizon 2020 UPGx grant (668353), and the Robert Bosch Stiftung, Stuttgart, Germany. Dr. Lewis is supported by NHLBI grant R01 HL137922. This project was also supported by the Deutsche Forschungsgemeinschaft (Klinische Forschungsgruppe‐KFO‐274: “Platelets‐Molecular Mechanisms and Translational Implications”) and by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) – Project number 374031971 – TRR 240.

## Conflict of Interest

R.B.A. is a board member at Youscript.com. D.A. receives honoraria for lectures from AstraZeneca, Bayer, Boehringer Ingelheim, Pfizer, and Biotronik; receives honoraria for advisory board activities from Bayer, Boehringer Ingelheim, and Medtronic. D.A. receives lecture fees from AstraZeneca, Richter, Roche Diagnostics, and Krka. W.H. receives speaker and advisory board fees from Bayer, Daiichi Sankyo, The Medicines Company, and Brystol‐Myer Squibb. M.G. is an honorary speaker for Bayer and Astra Zeneca. T.G. receives personal fees from Astra Zeneca, Boehringer Ingelheim, Bayer, Ferrer, and Pfizer; receives grants and personal fees from Bayer Healthcare, Bristol Myers Squibb, Daiichi Sankyo, Eli Lilly, and Spartan Bioscience. J.L. reports National Institutes of Health (NIH) grant support to study the pharmacogenetics of antiplatelet therapy. M.D.R. is on the Scientific Advisory Board at Cipherome; and receives speaker fees from the American Society of Health System Pharmacists. A.R.S. is an employee at Regeneron Pharmaceuticals, Inc. and receives compensation and stock options for his employment. D.T. receives honoraria for lectures from Amgen, AstraZeneca, Bayer, Bristol‐Myers Squibb, Boehringer Ingelheim, Daiichi Sankyo, Novartis, and Pfizer; receives honoraria for advisory board activities from Bayer, Boehringer Ingelheim, and Daiichi Sankyo; has participation in clinical trials and institutional trials for Amgen, Astra Zeneca, Bayer, Daiichi Sankyo, Doasense, Esperion, Idorsia, and Novartis; and receives research funding from Deutsche Herzstiftung and PharmComp Net Baden‐Wuerttemberg. All other authors declared no competing interests for this work.

## Author Contributions

S.S.V., T.O.B., L.G., R.E.P., T.E.K., A.R.S., and M.D.R wrote the manuscript. S.S.V., T.O.B., L.G., R.E.P., T.E.K., A.R.S., M.D.R, J.‐L.R., J.D.B., J.P.L., Y.B., B.D.M., Di.A., Da.A., R.A., K.B., G.C., K.C., J.H.C., J.‐P.D., N.P.D., I.F.‐C., P.F., M.G., T.G., G.F.G., B.G., P.A.G., W.H., L.H., E.‐Y.K., H.‐S.K., M.K., M.T.M.L., R.M., J.M., D.M.R., E.S., M.S., J.G.S., J.M.S.‐M., J.M.tB., D.T., M.V., J.W., M.‐S.W., R.W., and S.W. designed the research. S.V., T.B., L.G., J.‐L.R., J.L., Y.B., T.K., A.S., and M.R. performed the research. S.S.V., T.O.B., L.G., R.E.P., T.E.K., A.R.S., M.D.R., J.‐L.R., J.D.B., J.P.L., Y.B., and B.D.M. analyzed the data.

## Supporting information

## Acknowledgments

I.F.C. thanks Dr. Christina‐Gallego‐Fabrega for compiling clinical data and performing genotyping. J.L.R. recognizes Dr. Christophe Combescure’s contributions to the analysis plan. The authors gratefully acknowledge the data contributions of Prof. Bernd Jilma, principal investigator of the PEGASUS‐PCI study.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Relling, M.V. & Evans, W.E. Pharmacogenomics in the clinic. Nature 526, 343–350 (2015).  [DOI](https://doi.org/10.1038/nature15817) | [PMC free article](/articles/PMC4711261/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26469045/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Pharmacogenomics%20in%20the%20clinic&author=M.V.%20Relling&author=W.E.%20Evans&volume=526&publication_year=2015&pages=343-350&pmid=26469045&doi=10.1038/nature15817&)

2. Weeke, P.E. Pharmacogenetics in cardiovascular medicine. Adv. Pharmacol. 83, 333–360 (2018).  [DOI](https://doi.org/10.1016/bs.apha.2018.04.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29801582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv.%20Pharmacol.&title=Pharmacogenetics%20in%20cardiovascular%20medicine&author=P.E.%20Weeke&volume=83&publication_year=2018&pages=333-360&pmid=29801582&doi=10.1016/bs.apha.2018.04.002&)

3. Kazui, M. et al Identification of the human cytochrome P450 enzymes involved in the two oxidative steps in the bioactivation of clopidogrel to its pharmacologically active metabolite. Drug Metab. Dispos. 38, 92–99 (2010).  [DOI](https://doi.org/10.1124/dmd.109.029132) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19812348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Identification%20of%20the%20human%20cytochrome%20P450%20enzymes%20involved%20in%20the%20two%20oxidative%20steps%20in%20the%20bioactivation%20of%20clopidogrel%20to%20its%20pharmacologically%20active%20metabolite&author=M.%20Kazui&volume=38&publication_year=2010&pages=92-99&pmid=19812348&doi=10.1124/dmd.109.029132&)

4. Tantry, U.S. et al Consensus and update on the definition of on‐treatment platelet reactivity to adenosine diphosphate associated with ischemia and bleeding. J. Am. Coll. Cardiol. 62, 2261–2273 (2013).  [DOI](https://doi.org/10.1016/j.jacc.2013.07.101) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24076493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=Consensus%20and%20update%20on%20the%20definition%20of%20on%E2%80%90treatment%20platelet%20reactivity%20to%20adenosine%20diphosphate%20associated%20with%20ischemia%20and%20bleeding&author=U.S.%20Tantry&volume=62&publication_year=2013&pages=2261-2273&pmid=24076493&doi=10.1016/j.jacc.2013.07.101&)

5. Shuldiner, A.R. Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel therapy. JAMA 302, 849–857 (2009).  [DOI](https://doi.org/10.1001/jama.2009.1232) | [PMC free article](/articles/PMC3641569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19706858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Association%20of%20cytochrome%20P450%202C19%20genotype%20with%20the%20antiplatelet%20effect%20and%20clinical%20efficacy%20of%20clopidogrel%20therapy&author=A.R.%20Shuldiner&volume=302&publication_year=2009&pages=849-857&pmid=19706858&doi=10.1001/jama.2009.1232&)

6. Hochholzer, W. et al Impact of cytochrome P450 2C19 loss‐of‐function polymorphism and of major demographic characteristics on residual platelet function after loading and maintenance treatment with clopidogrel in patients undergoing elective coronary stent placement. J. Am. Coll. Cardiol. 55, 2427–2434 (2010).  [DOI](https://doi.org/10.1016/j.jacc.2010.02.031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20510210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=Impact%20of%20cytochrome%20P450%202C19%20loss%E2%80%90of%E2%80%90function%20polymorphism%20and%20of%20major%20demographic%20characteristics%20on%20residual%20platelet%20function%20after%20loading%20and%20maintenance%20treatment%20with%20clopidogrel%20in%20patients%20undergoing%20elective%20coronary%20stent%20placement&author=W.%20Hochholzer&volume=55&publication_year=2010&pages=2427-2434&pmid=20510210&doi=10.1016/j.jacc.2010.02.031&)

7. Siller‐Matula, J.M. , Trenk, D. , Krahenbuhl, S. , Michelson, A.D. & Delle‐Karth, G. Clinical implications of drug‐drug interactions with P2Y12 receptor inhibitors. J. Thromb. Haemost. 12, 2–13 (2014).  [DOI](https://doi.org/10.1111/jth.12445) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24406062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thromb.%20Haemost.&title=Clinical%20implications%20of%20drug%E2%80%90drug%20interactions%20with%20P2Y12%20receptor%20inhibitors&author=J.M.%20Siller%E2%80%90Matula&author=D.%20Trenk&author=S.%20Krahenbuhl&author=A.D.%20Michelson&author=G.%20Delle%E2%80%90Karth&volume=12&publication_year=2014&pages=2-13&pmid=24406062&doi=10.1111/jth.12445&)

8. Geisler, T. et al CYP2C19 and nongenetic factors predict poor responsiveness to clopidogrel loading dose after coronary stent implantation. Pharmacogenomics 9, 1251–1259 (2008).  [DOI](https://doi.org/10.2217/14622416.9.9.1251) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18781853/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CYP2C19%20and%20nongenetic%20factors%20predict%20poor%20responsiveness%20to%20clopidogrel%20loading%20dose%20after%20coronary%20stent%20implantation&author=T.%20Geisler&volume=9&publication_year=2008&pages=1251-1259&pmid=18781853&doi=10.2217/14622416.9.9.1251&)

9. Brar, S.S. et al Impact of platelet reactivity on clinical outcomes after percutaneous coronary intervention. A collaborative meta‐analysis of individual participant data. J. Am. Coll. Cardiol. 58, 1945–1954 (2011).  [DOI](https://doi.org/10.1016/j.jacc.2011.06.059) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22032704/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=Impact%20of%20platelet%20reactivity%20on%20clinical%20outcomes%20after%20percutaneous%20coronary%20intervention.%20A%20collaborative%20meta%E2%80%90analysis%20of%20individual%20participant%20data&author=S.S.%20Brar&volume=58&publication_year=2011&pages=1945-1954&pmid=22032704&doi=10.1016/j.jacc.2011.06.059&)

10. Niu, X. et al CYP2C19 polymorphism and clinical outcomes among patients of different races treated with clopidogrel: a systematic review and meta‐analysis. J. Huazhong Univ. Sci. Technol. Med. Sci. 35, 147–156 (2015).  [DOI](https://doi.org/10.1007/s11596-015-1404-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25877345/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Huazhong%20Univ.%20Sci.%20Technol.%20Med.%20Sci.&title=CYP2C19%20polymorphism%20and%20clinical%20outcomes%20among%20patients%20of%20different%20races%20treated%20with%20clopidogrel:%20a%20systematic%20review%20and%20meta%E2%80%90analysis&author=X.%20Niu&volume=35&publication_year=2015&pages=147-156&pmid=25877345&doi=10.1007/s11596-015-1404-7&)

11. Sorich, M.J. , Rowland, A. , McKinnon, R.A. & Wiese, M.D. CYP2C19 genotype has a greater effect on adverse cardiovascular outcomes following percutaneous coronary intervention and in Asian populations treated with clopidogrel: a meta‐analysis. Circ. Cardiovasc. Genet. 7, 895–902 (2014).  [DOI](https://doi.org/10.1161/CIRCGENETICS.114.000669) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25258374/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc.%20Genet.&title=CYP2C19%20genotype%20has%20a%20greater%20effect%20on%20adverse%20cardiovascular%20outcomes%20following%20percutaneous%20coronary%20intervention%20and%20in%20Asian%20populations%20treated%20with%20clopidogrel:%20a%20meta%E2%80%90analysis&author=M.J.%20Sorich&author=A.%20Rowland&author=R.A.%20McKinnon&author=M.D.%20Wiese&volume=7&publication_year=2014&pages=895-902&pmid=25258374&doi=10.1161/CIRCGENETICS.114.000669&)

12. Pan, Y. et al Genetic polymorphisms and clopidogrel efficacy for acute ischemic stroke or transient ischemic attack: a systematic review and meta‐analysis. Circulation 135, 21–33 (2017).  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.116.024913) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27806998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Genetic%20polymorphisms%20and%20clopidogrel%20efficacy%20for%20acute%20ischemic%20stroke%20or%20transient%20ischemic%20attack:%20a%20systematic%20review%20and%20meta%E2%80%90analysis&author=Y.%20Pan&volume=135&publication_year=2017&pages=21-33&pmid=27806998&doi=10.1161/CIRCULATIONAHA.116.024913&)

13. Backman, J.D. et al Genome‐wide analysis of clopidogrel active metabolite levels identifies novel variants that influence antiplatelet response. Pharmacogenet. Genomics 27, 159–163 (2017).  [DOI](https://doi.org/10.1097/FPC.0000000000000272) | [PMC free article](/articles/PMC5346037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28207573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Genome%E2%80%90wide%20analysis%20of%20clopidogrel%20active%20metabolite%20levels%20identifies%20novel%20variants%20that%20influence%20antiplatelet%20response&author=J.D.%20Backman&volume=27&publication_year=2017&pages=159-163&pmid=28207573&doi=10.1097/FPC.0000000000000272&)

14. Frelinger, A.L. et al Clopidogrel pharmacokinetics and pharmacodynamics vary widely despite exclusion or control of polymorphisms (CYP2C19, ABCB1, PON1), noncompliance, diet, smoking, co‐medications (including proton pump inhibitors), and pre‐existent variability in platelet function. J. Am. Coll. Cardiol. 61, 872–879 (2013).  [DOI](https://doi.org/10.1016/j.jacc.2012.11.040) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23333143/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Coll.%20Cardiol.&title=Clopidogrel%20pharmacokinetics%20and%20pharmacodynamics%20vary%20widely%20despite%20exclusion%20or%20control%20of%20polymorphisms%20(CYP2C19,%20ABCB1,%20PON1),%20noncompliance,%20diet,%20smoking,%20co%E2%80%90medications%20(including%20proton%20pump%20inhibitors),%20and%20pre%E2%80%90existent%20variability%20in%20platelet%20function&author=A.L.%20Frelinger&volume=61&publication_year=2013&pages=872-879&pmid=23333143&doi=10.1016/j.jacc.2012.11.040&)

15. Bouman, H.J. et al Variability in on‐treatment platelet reactivity explained by CYP2C19*2 genotype is modest in clopidogrel pretreated patients undergoing coronary stenting. Heart 97, 1239–1244 (2011).  [DOI](https://doi.org/10.1136/hrt.2010.220509) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21628721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Heart&title=Variability%20in%20on%E2%80%90treatment%20platelet%20reactivity%20explained%20by%20CYP2C19*2%20genotype%20is%20modest%20in%20clopidogrel%20pretreated%20patients%20undergoing%20coronary%20stenting&author=H.J.%20Bouman&volume=97&publication_year=2011&pages=1239-1244&pmid=21628721&doi=10.1136/hrt.2010.220509&)

16. Nasyuhana Sani, Y. et al The CYP2C19( *)1/( *)2 Genotype does not adequately predict clopidogrel response in healthy Malaysian volunteers. Cardiol. Res. Pract. 2013, 128795 (2013).  [DOI](https://doi.org/10.1155/2013/128795) | [PMC free article](/articles/PMC3574649/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23431496/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiol.%20Res.%20Pract&title=The%20CYP2C19(%20*)1/(%20*)2%20Genotype%20does%20not%20adequately%20predict%20clopidogrel%20response%20in%20healthy%20Malaysian%20volunteers&author=Y.%20Nasyuhana%20Sani&volume=2013&publication_year=2013&pages=128795&pmid=23431496&doi=10.1155/2013/128795&)

17. Claassens, D.M.F. A genotype‐guided strategy for oral p2y12 inhibitors in primary PCI. N. Engl. J. Med. 381, 1621–1631 (2019).  [DOI](https://doi.org/10.1056/NEJMoa1907096) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31479209/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=A%20genotype%E2%80%90guided%20strategy%20for%20oral%20p2y12%20inhibitors%20in%20primary%20PCI&author=D.M.F.%20Claassens&volume=381&publication_year=2019&pages=1621-1631&pmid=31479209&doi=10.1056/NEJMoa1907096&)

18. Pereira, N.L. Clopidogrel pharmacogenetics. Circ. Cardiovasc. Interv. 12, e007811 (2019).  [DOI](https://doi.org/10.1161/CIRCINTERVENTIONS.119.007811) | [PMC free article](/articles/PMC6581205/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30998396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc.%20Interv&title=Clopidogrel%20pharmacogenetics&author=N.L.%20Pereira&volume=12&publication_year=2019&pages=e007811&pmid=30998396&doi=10.1161/CIRCINTERVENTIONS.119.007811&)

19. Angiolillio, D.J. Derivation, validation, and prognostic utility of a prediction rule for nonresponse to clopidogrel ‐ the ABCD‐GENE score. JACC Cardiovasc. Interv. 13, 606–617 (2020).  [DOI](https://doi.org/10.1016/j.jcin.2020.01.226) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32139218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JACC%20Cardiovasc.%20Interv.&title=Derivation,%20validation,%20and%20prognostic%20utility%20of%20a%20prediction%20rule%20for%20nonresponse%20to%20clopidogrel%20%E2%80%90%20the%20ABCD%E2%80%90GENE%20score&author=D.J.%20Angiolillio&volume=13&publication_year=2020&pages=606-617&pmid=32139218&doi=10.1016/j.jcin.2020.01.226&)

20. Bergmeijer, T.O. et al Genome‐wide and candidate gene approaches of clopidogrel efficacy using pharmacodynamic and clinical end points‐Rationale and design of the International Clopidogrel Pharmacogenomics Consortium (ICPC). Am. Heart J. 198, 152–159 (2018).  [DOI](https://doi.org/10.1016/j.ahj.2017.12.010) | [PMC free article](/articles/PMC5903579/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29653637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20Heart%20J.&title=Genome%E2%80%90wide%20and%20candidate%20gene%20approaches%20of%20clopidogrel%20efficacy%20using%20pharmacodynamic%20and%20clinical%20end%20points%E2%80%90Rationale%20and%20design%20of%20the%20International%20Clopidogrel%20Pharmacogenomics%20Consortium%20(ICPC)&author=T.O.%20Bergmeijer&volume=198&publication_year=2018&pages=152-159&pmid=29653637&doi=10.1016/j.ahj.2017.12.010&)

21. ICPC Investigators . ICPC – International Clopidogrel Pharmacogenomics Consortium. 2019.

22. Lewis, J.P. Pharmacogenomic polygenic response score predicts ischemic events and cardiovascular mortality in clopidogrel‐treated patients. Eur. Heart J. Cardiovasc. Pharmacother. 2019.  [DOI](https://doi.org/10.1093/ehjcvp/pvz045) | [PMC free article](/articles/PMC7363022/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31504375/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20Heart%20J.%20Cardiovasc.%20Pharmacother&title=Pharmacogenomic%20polygenic%20response%20score%20predicts%20ischemic%20events%20and%20cardiovascular%20mortality%20in%20clopidogrel%E2%80%90treated%20patients&author=J.P.%20Lewis&publication_year=2019&pmid=31504375&doi=10.1093/ehjcvp/pvz045&)

23. Verma, S.S. et al Imputation and quality control steps for combining multiple genome‐wide datasets. Front. Genet. 5, 370 (2014).  [DOI](https://doi.org/10.3389/fgene.2014.00370) | [PMC free article](/articles/PMC4263197/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25566314/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Genet.&title=Imputation%20and%20quality%20control%20steps%20for%20combining%20multiple%20genome%E2%80%90wide%20datasets&author=S.S.%20Verma&volume=5&publication_year=2014&pages=370&pmid=25566314&doi=10.3389/fgene.2014.00370&)

24. Chang, C.C. , Chow, C.C. , Tellier, L.C. , Vattikuti, S. , Purcell, S.M. & Lee, J.J. Second‐generation PLINK: rising to the challenge of larger and richer datasets. Gigascience 4, 7 (2015).  [DOI](https://doi.org/10.1186/s13742-015-0047-8) | [PMC free article](/articles/PMC4342193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25722852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gigascience&title=Second%E2%80%90generation%20PLINK:%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&author=C.C.%20Chang&author=C.C.%20Chow&author=L.C.%20Tellier&author=S.%20Vattikuti&author=S.M.%20Purcell&volume=4&publication_year=2015&pages=7&pmid=25722852&doi=10.1186/s13742-015-0047-8&)

25. Zheng, X. , Levine, D. , Shen, J. , Gogarten, S.M. , Laurie, C. & Weir, B.S. A high‐performance computing toolset for relatedness and principal component analysis of SNP data. Bioinformatics 28, 3326–3328 (2012).  [DOI](https://doi.org/10.1093/bioinformatics/bts606) | [PMC free article](/articles/PMC3519454/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23060615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=A%20high%E2%80%90performance%20computing%20toolset%20for%20relatedness%20and%20principal%20component%20analysis%20of%20SNP%20data&author=X.%20Zheng&author=D.%20Levine&author=J.%20Shen&author=S.M.%20Gogarten&author=C.%20Laurie&volume=28&publication_year=2012&pages=3326-3328&pmid=23060615&doi=10.1093/bioinformatics/bts606&)

26. Hall, M.A. et al Software provides analytic framework for investigating complexity beyond genome‐wide association studies. Nat. Commun. 8, 1167 (2017).  [DOI](https://doi.org/10.1038/s41467-017-00802-2) | [PMC free article](/articles/PMC5660079/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29079728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Software%20provides%20analytic%20framework%20for%20investigating%20complexity%20beyond%20genome%E2%80%90wide%20association%20studies&author=M.A.%20Hall&volume=8&publication_year=2017&pages=1167&pmid=29079728&doi=10.1038/s41467-017-00802-2&)

27. Slifer, S.H. PLINK: key functions for data analysis. Curr. Protoc. Hum. Genet. 97, e59 (2018).  [DOI](https://doi.org/10.1002/cphg.59) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30040203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.%20Hum.%20Genet.&title=PLINK:%20key%20functions%20for%20data%20analysis&author=S.H.%20Slifer&volume=97&publication_year=2018&pages=e59&pmid=30040203&doi=10.1002/cphg.59&)

28. Watanabe, K. , Taskesen, E. , Bochoven, A. & Posthuma, D. Functional mapping and annotation of genetic associations with FUMA. Nat. Commun. 8, 1826 (2017).  [DOI](https://doi.org/10.1038/s41467-017-01261-5) | [PMC free article](/articles/PMC5705698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29184056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Functional%20mapping%20and%20annotation%20of%20genetic%20associations%20with%20FUMA&author=K.%20Watanabe&author=E.%20Taskesen&author=A.%20Bochoven&author=D.%20Posthuma&volume=8&publication_year=2017&pages=1826&pmid=29184056&doi=10.1038/s41467-017-01261-5&)

29. Leeuw, C.A. , Mooij, J.M. , Heskes, T. & Posthuma, D. MAGMA: generalized gene‐set analysis of GWAS data. PLoS Comput. Biol. 11, e1004219 (2015).  [DOI](https://doi.org/10.1371/journal.pcbi.1004219) | [PMC free article](/articles/PMC4401657/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25885710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput.%20Biol.&title=MAGMA:%20generalized%20gene%E2%80%90set%20analysis%20of%20GWAS%20data&author=C.A.%20Leeuw&author=J.M.%20Mooij&author=T.%20Heskes&author=D.%20Posthuma&volume=11&publication_year=2015&pages=e1004219&pmid=25885710&doi=10.1371/journal.pcbi.1004219&)

30. Wang, K. , Li, M. & Hakonarson, H. ANNOVAR: functional annotation of genetic variants from high‐throughput sequencing data. Nucl. Acids Res. 38, e164 (2010).  [DOI](https://doi.org/10.1093/nar/gkq603) | [PMC free article](/articles/PMC2938201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20601685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucl.%20Acids%20Res.&title=ANNOVAR:%20functional%20annotation%20of%20genetic%20variants%20from%20high%E2%80%90throughput%20sequencing%20data&author=K.%20Wang&author=M.%20Li&author=H.%20Hakonarson&volume=38&publication_year=2010&pages=e164&pmid=20601685&doi=10.1093/nar/gkq603&)

31. Locke, A.E. et al Genetic studies of body mass index yield new insights for obesity biology. Nature 518, 197–206 (2015).  [DOI](https://doi.org/10.1038/nature14177) | [PMC free article](/articles/PMC4382211/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25673413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genetic%20studies%20of%20body%20mass%20index%20yield%20new%20insights%20for%20obesity%20biology&author=A.E.%20Locke&volume=518&publication_year=2015&pages=197-206&pmid=25673413&doi=10.1038/nature14177&)

32. Luciano, M. et al Association analysis in over 329,000 individuals identifies 116 independent variants influencing neuroticism. Nat. Genet. 50, 6–11 (2018).  [DOI](https://doi.org/10.1038/s41588-017-0013-8) | [PMC free article](/articles/PMC5985926/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29255261/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Association%20analysis%20in%20over%20329,000%20individuals%20identifies%20116%20independent%20variants%20influencing%20neuroticism&author=M.%20Luciano&volume=50&publication_year=2018&pages=6-11&pmid=29255261&doi=10.1038/s41588-017-0013-8&)

33. Simino, J. et al Gene‐age interactions in blood pressure regulation: a large‐scale investigation with the CHARGE, Global BPgen, and ICBP Consortia. Am. J. Hum. Genet. 95, 24–38 (2014).  [DOI](https://doi.org/10.1016/j.ajhg.2014.05.010) | [PMC free article](/articles/PMC4085636/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24954895/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Gene%E2%80%90age%20interactions%20in%20blood%20pressure%20regulation:%20a%20large%E2%80%90scale%20investigation%20with%20the%20CHARGE,%20Global%20BPgen,%20and%20ICBP%20Consortia&author=J.%20Simino&volume=95&publication_year=2014&pages=24-38&pmid=24954895&doi=10.1016/j.ajhg.2014.05.010&)

34. Jansen, P.R. et al Genome‐wide analysis of insomnia in 1,331,010 individuals identifies new risk loci and functional pathways. Nat. Genet. 51, 394–403 (2019).  [DOI](https://doi.org/10.1038/s41588-018-0333-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30804565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Genome%E2%80%90wide%20analysis%20of%20insomnia%20in%201,331,010%20individuals%20identifies%20new%20risk%20loci%20and%20functional%20pathways&author=P.R.%20Jansen&volume=51&publication_year=2019&pages=394-403&pmid=30804565&doi=10.1038/s41588-018-0333-3&)

35. Kanai, M. et al Genetic analysis of quantitative traits in the Japanese population links cell types to complex human diseases. Nat. Genet. 50, 390–400 (2018).  [DOI](https://doi.org/10.1038/s41588-018-0047-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29403010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Genetic%20analysis%20of%20quantitative%20traits%20in%20the%20Japanese%20population%20links%20cell%20types%20to%20complex%20human%20diseases&author=M.%20Kanai&volume=50&publication_year=2018&pages=390-400&pmid=29403010&doi=10.1038/s41588-018-0047-6&)

36. Wuttke, M. et al A catalog of genetic loci associated with kidney function from analyses of a million individuals. Nat. Genet. 51, 957–972 (2019).  [DOI](https://doi.org/10.1038/s41588-019-0407-x) | [PMC free article](/articles/PMC6698888/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31152163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=A%20catalog%20of%20genetic%20loci%20associated%20with%20kidney%20function%20from%20analyses%20of%20a%20million%20individuals&author=M.%20Wuttke&volume=51&publication_year=2019&pages=957-972&pmid=31152163&doi=10.1038/s41588-019-0407-x&)

37. Goes, F.S. et al Genome‐wide association study of schizophrenia in Ashkenazi Jews. Am. J. Med. Genet. B Neuropsychiatr. Genet. 168, 649–659 (2015).  [DOI](https://doi.org/10.1002/ajmg.b.32349) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26198764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Med.%20Genet.%20B%20Neuropsychiatr.%20Genet.&title=Genome%E2%80%90wide%20association%20study%20of%20schizophrenia%20in%20Ashkenazi%20Jews&author=F.S.%20Goes&volume=168&publication_year=2015&pages=649-659&pmid=26198764&doi=10.1002/ajmg.b.32349&)

38. Quadri, M. Mutation in the SYNJ1 gene associated with autosomal recessive, early‐onset Parkinsonism. Hum. Mutat. 34, 1208–1215 (2013).  [DOI](https://doi.org/10.1002/humu.22373) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23804577/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mutat.&title=Mutation%20in%20the%20SYNJ1%20gene%20associated%20with%20autosomal%20recessive,%20early%E2%80%90onset%20Parkinsonism&author=M.%20Quadri&volume=34&publication_year=2013&pages=1208-1215&pmid=23804577&doi=10.1002/humu.22373&)

39. Kichaev, G. et al Leveraging polygenic functional enrichment to improve GWAS power. Am. J. Hum. Genet. 104, 65–75 (2019).  [DOI](https://doi.org/10.1016/j.ajhg.2018.11.008) | [PMC free article](/articles/PMC6323418/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30595370/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Leveraging%20polygenic%20functional%20enrichment%20to%20improve%20GWAS%20power&author=G.%20Kichaev&volume=104&publication_year=2019&pages=65-75&pmid=30595370&doi=10.1016/j.ajhg.2018.11.008&)

40. Zhong, W.P. et al Genomewide association study identifies novel genetic loci that modify antiplatelet effects and pharmacokinetics of clopidogrel. Clin. Pharmacol. Ther. 101, 791–802 (2017).  [DOI](https://doi.org/10.1002/cpt.589) | [PMC free article](/articles/PMC5485718/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27981573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genomewide%20association%20study%20identifies%20novel%20genetic%20loci%20that%20modify%20antiplatelet%20effects%20and%20pharmacokinetics%20of%20clopidogrel&author=W.P.%20Zhong&volume=101&publication_year=2017&pages=791-802&pmid=27981573&doi=10.1002/cpt.589&)

41. Kirola, L. , Behari, M. , Shishir, C. & Thelma, B.K. Identification of a novel homozygous mutation Arg459Pro in SYNJ1 gene of an Indian family with autosomal recessive juvenile Parkinsonism. Parkinsonism Relat. Disord. 31, 124–128 (2016).  [DOI](https://doi.org/10.1016/j.parkreldis.2016.07.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27496670/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Parkinsonism%20Relat.%20Disord.&title=Identification%20of%20a%20novel%20homozygous%20mutation%20Arg459Pro%20in%20SYNJ1%20gene%20of%20an%20Indian%20family%20with%20autosomal%20recessive%20juvenile%20Parkinsonism&author=L.%20Kirola&author=M.%20Behari&author=C.%20Shishir&author=B.K.%20Thelma&volume=31&publication_year=2016&pages=124-128&pmid=27496670&doi=10.1016/j.parkreldis.2016.07.014&)

42. Ben Romdhan, S. , Sakka, S. , Farhat, N. , Triki, S. , Dammak, M. & Mhiri, C. A novel SYNJ1 mutation in a tunisian family with juvenile Parkinson’s disease associated with epilepsy. J. Mol. Neurosci. 66, 273–278 (2018).  [DOI](https://doi.org/10.1007/s12031-018-1167-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30187305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Neurosci.&title=A%20novel%20SYNJ1%20mutation%20in%20a%20tunisian%20family%20with%20juvenile%20Parkinson%E2%80%99s%20disease%20associated%20with%20epilepsy&author=S.%20Ben%20Romdhan&author=S.%20Sakka&author=N.%20Farhat&author=S.%20Triki&author=M.%20Dammak&volume=66&publication_year=2018&pages=273-278&pmid=30187305&doi=10.1007/s12031-018-1167-2&)

43. Lewis, J.P. et al The functional G143E variant of carboxylesterase 1 is associated with increased clopidogrel active metabolite levels and greater clopidogrel response. Pharmacogenet. Genomics 23, 1–8 (2013).  [DOI](https://doi.org/10.1097/FPC.0b013e32835aa8a2) | [PMC free article](/articles/PMC3682407/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23111421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=The%20functional%20G143E%20variant%20of%20carboxylesterase%201%20is%20associated%20with%20increased%20clopidogrel%20active%20metabolite%20levels%20and%20greater%20clopidogrel%20response&author=J.P.%20Lewis&volume=23&publication_year=2013&pages=1-8&pmid=23111421&doi=10.1097/FPC.0b013e32835aa8a2&)

44. Choi, S.Y. & Kim, M.H. Comparison of factors affecting platelet reactivity in various platelet function tests. Platelets 30, 631–636 (2019).  [DOI](https://doi.org/10.1080/09537104.2018.1499887) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30059640/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Platelets&title=Comparison%20of%20factors%20affecting%20platelet%20reactivity%20in%20various%20platelet%20function%20tests&author=S.Y.%20Choi&author=M.H.%20Kim&volume=30&publication_year=2019&pages=631-636&pmid=30059640&doi=10.1080/09537104.2018.1499887&)

45. Janssen, P.W. et al The effect of correcting VerifyNow P2Y12 assay results for hematocrit in patients undergoing percutaneous coronary interventions. J. Thromb. Haemost. 15, 618–623 (2017).  [DOI](https://doi.org/10.1111/jth.13642) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28135786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thromb.%20Haemost.&title=The%20effect%20of%20correcting%20VerifyNow%20P2Y12%20assay%20results%20for%20hematocrit%20in%20patients%20undergoing%20percutaneous%20coronary%20interventions&author=P.W.%20Janssen&volume=15&publication_year=2017&pages=618-623&pmid=28135786&doi=10.1111/jth.13642&)

46. Bergmeijer, T.O. Feasibility and implementation of CYP2C19 genotyping in patients using antiplatelet therapy. Pharmacogenomics 19, 621–628 (2018).  [DOI](https://doi.org/10.2217/pgs-2018-0013) | [PMC free article](/articles/PMC6021912/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29701129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Feasibility%20and%20implementation%20of%20CYP2C19%20genotyping%20in%20patients%20using%20antiplatelet%20therapy&author=T.O.%20Bergmeijer&volume=19&publication_year=2018&pages=621-628&pmid=29701129&doi=10.2217/pgs-2018-0013&)

47. Lee, C.R. Clinical outcomes and sustainability of using CYP2C19 genotype‐guided antiplatelet therapy after percutaneous coronary intervention. Circ. Genom. Precis. Med. 11, e002069 (2018).  [DOI](https://doi.org/10.1161/CIRCGEN.117.002069) | [PMC free article](/articles/PMC5889089/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29615454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Genom.%20Precis.%20Med.&title=Clinical%20outcomes%20and%20sustainability%20of%20using%20CYP2C19%20genotype%E2%80%90guided%20antiplatelet%20therapy%20after%20percutaneous%20coronary%20intervention&author=C.R.%20Lee&volume=11&publication_year=2018&pages=e002069&pmid=29615454&doi=10.1161/CIRCGEN.117.002069&)

48. Cavallari, L.H. Multisite investigation of outcomes with implementation of CYP2C19 genotype‐guided antiplatelet therapy after percutaneous coronary intervention. JACC Cardiovasc. Interv. 11, 181–191 (2018).  [DOI](https://doi.org/10.1016/j.jcin.2017.07.022) | [PMC free article](/articles/PMC5775044/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29102571/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JACC%20Cardiovasc.%20Interv.&title=Multisite%20investigation%20of%20outcomes%20with%20implementation%20of%20CYP2C19%20genotype%E2%80%90guided%20antiplatelet%20therapy%20after%20percutaneous%20coronary%20intervention&author=L.H.%20Cavallari&volume=11&publication_year=2018&pages=181-191&pmid=29102571&doi=10.1016/j.jcin.2017.07.022&)
