# Prediction and visualization of CYP2D6 genotype-based phenotype using clustering algorithms

## Metadata
**Authors:** Eun-Young Kim, Sang-Goo Shin, Jae-Gook Shin
**Journal:** Translational and Clinical Pharmacology
**Date:** 2017 Sep 15
**DOI:** [10.12793/tcp.2017.25.3.147](https://doi.org/10.12793/tcp.2017.25.3.147)
**PMID:** 32095466
**PMCID:** PMC7033374
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7033374/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC7033374/pdf/tcp-25-147.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC7033374/pdf/tcp-25-147.pdf)

## Abstract

This study focused on the role of cytochrome P450 2D6 (CYP2D6) genotypes to predict phenotypes in the metabolism of dextromethorphan. CYP2D6 genotypes and metabolic ratios (MRs) of dextromethorphan were determined in 201 Koreans. Unsupervised clustering algorithms, hierarchical and k-means clustering analysis, and color visualizations of CYP2D6 activity were performed on a subset of 130 subjects. A total of 23 different genotypes were identified, five of which were observed in one subject. Phenotype classifications were based on the means, medians, and standard deviations of the log MR values for each genotype. Color visualization was used to display the mean and median of each genotype as different color intensities. Cutoff values were determined using receiver operating characteristic curves from the k-means analysis, and the data were validated in the remaining subset of 71 subjects. Using the two highest silhouette values, the selected numbers of clusters were three (the best) and four. The findings from the two clustering algorithms were similar to those of other studies, classifying *5/*5 as a lowest activity group and genotypes containing duplicated alleles (i.e., CYP2D6*1/*2N) as a highest activity group. The validation of the k-means clustering results with data from the 71 subjects revealed relatively high concordance rates: 92.8% and 73.9% in three and four clusters, respectively. Additionally, color visualization allowed for rapid interpretation of results. Although the clustering approach to predict CYP2D6 phenotype from CYP2D6 genotype is not fully complete, it provides general information about the genotype to phenotype relationship, including rare genotypes with only one subject.

Keywords: CYP2D6, clustering analysis, dextromethorphan, genotype, phenotype

## Introduction

Genetic polymorphism in drug-metabolizing enzymes can cause inter-individual differences in drug response. Because of the pharmacokinetic and pharmacodynamic diversity of these drugs, the same dose can result in a variety of clinical consequences in different patients, such as adverse drug reactions or therapeutic failure.[[1](#B1),[2](#B2)] Cytochrome P450 (CYP) enzymes play an important role in the metabolism of endogenous substrates and xenobiotics.[[3](#B3),[4](#B4)] More than 50 CYP enzymes have been identified so far.[[5](#B5)]

The CYP2D6 enzyme is involved in the metabolism of more than 100 drugs, including antipsychotics, antiarrhythmics, and analgesics, which account for approximately 15–25% of all clinically used drugs.[[6](#B6),[7](#B7)] *CYP2D6* is highly polymorphic and more than 100 allelic variants have been reported.[[8](#B8)] A range of enzyme activity results from the broad spectrum of genetic variation, point mutation, defect or insertion, genetic rearrangement, and deletion or duplication of the whole gene. [[7](#B7)] These *CYP2D6* genetic polymorphisms are considered as distinct ethnic differences in drug metabolism depending on each gene variation. Enzyme activity, defined as the degree of metabolism, can be classified into four groups: ultra-rapid metabolizers (UM), extensive metabolizers (EM), intermediate metabolizers (IM), and poor metabolizers (PM).[[7](#B7),[9](#B9)] The Clinical Pharmacogenetics Implementation Consortium recommends that normal metabolizer (NM) be the standard term in place of EM.[[10](#B10)] Nortriptyline, a tricyclic antidepressant primarily metabolized by CYP2D6, requires < 10–20 mg for PMs, and up to 500 mg for UMs, to achieve the desired therapeutic effect.[[1](#B1)] This suggests that treatment outcomes are dependent on drug metabolism capacity, as determined by the genetic predisposition of *CYP2D6*. At the same dose, PM patients may suffer from toxic effects, while UM patients may not be affected. Knowledge of a patient's *CYP2D6* genotype prior to drug therapy allows prediction of enzyme activity, which can be useful for optimization of individual therapies. Many clinically important drugs metabolized by CYP2D6 have narrow therapeutic dose ranges especially in PMs, so genotype information for CYP2D6 has the potential to be beneficial to patients.

In this study, *CYP2D6* genotypes were classified based on CYP2D6 metabolic activity to predict genotype-based phenotypes in Korean subjects. The results of the study can be used for direct individualized therapy based on *CYP2D6* genotypes. The relationship between genotype and phenotype was assessed using machine learning algorithms. Use of color visualizations allowed for easier identification of phenotype based on *CYP2D6* genotype.

## Methods

### CYP2D6 genotype and phenotype data

CYP2D6 phenotyping data performed in 201 healthy Korean subjects using dextromethorphan (DM) as the CYP2D6 substrate were used. The Institutional Review Board of the Inje University Busan Paik Hospital (Busan, Republic of Korea) approved the study protocol.(06-087) The genotype and phenotype data retrieved from the database of the Pharmacogenomics Research Center (PGRC), Inje University (Busan, Republic of Korea),[[11](#B11),[12](#B12)] were used for this study. The method of phenotyping with urine dextromethorphan and its metabolite is described in the previous study.[[11](#B11)] Genotypes were identified using the multiplex single-base extension method to select 12 CYP2D6 alleles (**1*, **2*, **5*, **10*, **14*, **18*, **21*, **41*, **49*, **52*, **60*, and a duplication of *CYP2D6*).[[12](#B12)] CYP2D6 activity was measured as the log-transformed urinary metabolic ratio (MR) of DM, log10 (DM/Dextrorphan (DX)).[[11](#B11)]

### CYP2D6 genotype to phenotype classification analysis

Cluster analysis was performed to classify *CYP2D6* genotypes according to phenotype (based on the mean, median, and standard deviation (SD) of log-transformed MR) using data from a subset of 130 subjects (clustering set). Two algorithms were applied for the cluster analysis: hierarchical clustering and k-means clustering. In hierarchical clustering, ward linkage was selected as the agglomeration procedure and Euclidean distance was used to calculate similarities between individuals. The number of likely groups was determined by visual observation of a dendrogram. In k-means clustering analysis, the number of phenotype clusters was assessed by calculating the average of the silhouette values obtained by 20 different initial values. K-means clustering with squared Euclidian distance was performed with the two numbers of clusters with the highest silhouette values. The calculated silhouette value ranges from −1 to 1. A value ≥ 0 indicates that the value should be classified into the group, while a value of −1 indicates the value is not related to the group. A value close to 1 indicates a well-separated group.

### Color visualization and validation of phenotype classifications

Hierarchical and k-means clustering were plotted on a map using color to indicate phenotype (i.e., the level of drug metabolism activity) for each genotype. A dendrogram and silhouette values were also visualized in hierarchical and k-means clustering, respectively. Higher CYP2D6 activity levels were represented by the color blue, while higher logarithmic MR values (lower enzyme activity) were represented by red.

To verify the phenotype classification determined using the k-means analysis in the clustering set (130 subjects), the results were applied to the validation set (71 subjects who were recruited separately). The following procedure was used:

Cluster analysis, determination of cutoff values using ROC curves, and color visualizations were performed using MATLAB 7.1 (MathWorks Inc., Natick, MA, USA).

## Results

Subject characteristics (age: 25 ± 3.6 years) have been previously described.[[11](#B11)] There were 23 different genotypes in the clustering set (130 subjects), five of which were observed in one subject only. The most common genotypes were *CYP2D6*1/*10B*, **10B/*10B*, and **1/*1*, occurring at frequencies of 20.0%, 19.2%, and 16.2%, respectively. Other genotypes occurred at significantly lower frequencies. Sixteen genotypes were identified in the validation set of 71 subjects, the most common of which were *CYP2D6*1/*10B*, **10B/*10B*, and **1/*1*. The genotypes *CYP2D6*1/*14* and *CYP2D6*1/*21* each occurred in one subject only and were unique to the validation set.

A dendrogram was used to visualize the relationship between genotypes and phenotypes ([Fig. 1](#F1)). *CYP2D6*5/*5*, a known PM type of *CYP2D6*, had the lowest observed enzyme activity and was represented by the color brown. The genotypes with the highest enzyme activity were *CYP2D6*2N/*10B* and **1/*2N*. The remaining genotypes were divided into two groups in the dendrogram, one group with higher levels of enzyme activity and the other with lower levels of activity. The higher activity group contained **1*, **2*, and **2N* genotypes, which are known to have normal or increased enzyme activity. The lower activity group contained non-functional alleles (**5* or **21*) or alleles known to have decreased activity (**10*, **14*, **18*, and **41*).

### Figure 1. Color visualization of log-transformed urinary metabolic ratios (MR, dextromethorphan/dextrorphan) in hierarchical clustering (n=130). Each row indicates a CYP2D6 genotype. The mean and the median log MR values are colored based on intensity (cold to warm color). Genotypes indicated in blue font are based on data from a single subject. * No., number of subjects.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6493/7033374/d4288526c5d1/tcp-25-147-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7033374_tcp-25-147-g001.jpg)

Based on the hierarchical clustering results, the number of the phenotype cluster was set between 2 and 10 for the k-means analysis. The number of the phenotype clusters with the highest mean silhouette value (average = 0.79) was three and the second highest (average = 0.71) was four. Color visualizations and silhouette values are presented in [Figs. 2](#F2) and [3](#F3). The first genotype listed in each cluster was closest to the central value of the cluster, and the last genotype was furthest from the central value (and less characteristic of that cluster). Group numbers (Cluster 1, Cluster 2) were randomly assigned. *CYP2D6*5/*5*, when classified into three clusters and four clusters by the k-means method, were classified as one separated cluster (Cluster 2) and *CYP2D6*1/*2N*, **2N/*5*, and **2N/*10B* were in a cluster (Cluster 1) of four clusters. With the exception of *CYP2D6*5/*41*, one cluster (Cluster 4) contained genotypes with at least one normal or increased function gene and the other cluster (Cluster 3) did not contain these genes in the four clusters. It was similar in the three clusters. The three clusters are shown in [Fig. 2](#F2) and listed in [Table 1](#T1). The information was easier and faster to interpret using color visualization. The mean log MR value for each genotype ranged from −3.47 to 0.07, while the range for all subjects was −3.53 to 0.07.

### Figure 2. Color visualization and silhouette values of log-transformed urinary metabolic ratios (MR, dextromethorphan/dextrorphan) using the k-means cluster analysis based on three clusters (n=130). Each row indicates a CYP2D6 genotype. The mean and the median log MR values are colored based on intensity (cold to warm color). * No., number of subjects.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6493/7033374/9ab343a43c97/tcp-25-147-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7033374_tcp-25-147-g002.jpg)

### Figure 3. Color visualization and silhouette values of log-transformed urinary metabolic ratios (MR, dextromethorphan/dextrorphan) using the k-means cluster analysis based on four clusters (n=130). Each row indicates a CYP2D6 genotype. The mean and the median log MR values are colored based on intensity (cold to warm color). * No., number of subjects.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6493/7033374/2fc06d63350c/tcp-25-147-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=7033374_tcp-25-147-g003.jpg)

### Table 1. Results of the k-means cluster analysis based on three clusters of CYP2D6 genotype and phenotype data (n=130).

| Genotype | Meana | Mediana | Nob | Clusters |
| --- | --- | --- | --- | --- |
| *2/*10B | −2.72 | −2.72 | 4 | 1 |
| *1/*1 | −2.73 | −2.75 | 21 | 1 |
| *1/*5 | −2.61 | −2.48 | 3 | 1 |
| *1/*49 | −2.50 | −2.51 | 3 | 1 |
| *1/*10B | −2.47 | −2.53 | 26 | 1 |
| *1/*2 | −2.54 | −2.40 | 5 | 1 |
| *2N/*5 | −2.82 | −2.82 | 1 | 1 |
| *5/*41 | −2.46 | −2.46 | 1 | 1 |
| *2/*2 | −2.49 | −2.48 | 5 | 1 |
| *1/*41 | −2.22 | −2.28 | 3 | 1 |
| *2/*5 | −2.18 | −2.45 | 3 | 1 |
| *1/*2N | −3.08 | −3.08 | 2 | 1 |
| *2N/*10B | −3.47 | −3.47 | 2 | 1 |
| *5/*5 | 0.07 | 0.07 | 1 | 2 |
| *10B/*10B | −1.62 | −1.69 | 25 | 3 |
| *10B/*41 | −1.73 | −1.72 | 4 | 3 |
| *10B/*14 | −1.65 | −1.65 | 2 | 3 |
| *41/*41 | −1.53 | −1.53 | 1 | 3 |
| *10B/*18 | −1.42 | −1.42 | 2 | 3 |
| *5/*10B | −1.31 | −1.38 | 6 | 3 |
| *14/*41 | −1.83 | −1.83 | 1 | 3 |
| *10B/*49 | −1.97 | −2.00 | 6 | 3 |
| *10B/*21 | −1.03 | −1.00 | 3 | 3 |

Table 1 Caption: amean and median of log-transformed urinary metabolic ratios (dextromethorphan/dextrorphan) for each genotype,bnumber of subjects.

To verify the results of the clustering analysis, phenotype groups in the validation set were determined using the ROC curve cutoff value of three and four clusters from the clustering set. The cluster with the *CYC2D6*5/*5* genotype, which included only one subject with the lowest level of enzyme activity, differed significantly from the other groups in terms of the cutoff value. There were no individuals with the *CYP2D6*5/*5* genotype among the 71 subjects in the validation set, and cutoff values were determined as one for three clusters and two for four clusters. The cutoff value in the three clusters was -2.053 with 90.0% sensitivity and 92.4% specificity. The cutoff values of the four clusters were −2.9668 and −2.053. Sensitivity and specificity values were 90.5% and 80.0%, respectively, for the former cutoff and 90.0% and 91.9%, respectively, for the latter cutoff. The concordance rate was 91.5% for one cutoff of the three clusters, and 85.3% for two cutoffs of the four clusters for the clustering set. When the cutoff values were applied to the validation set, the concordance rate was 92.8% for the three clusters and 73.9% for the four clusters. A lower concordance rate was observed in both the clustering and validation data for the four clusters, and one cutoff value in the three clusters was equal to one in the four clusters. Genotypes included in two clusters (Cluster 3) were the same in the cluster with the lowest enzymatic activity, except the cluster containing *CYP2D6*5/*5* of the three and the four clusters. Concordance rates in both cases were high, but they were lower in the clusters with the highest (Cluster 1) and second highest (Cluster 4) enzyme activities in the four clusters.

## Discussion

CYP2D6 phenotype classification based on genotype was determined by categorization into four groups (UM to PM according to functionality of alleles), by choosing an antimode using a frequency histogram or a probit plot of MR. Using an antimode to separate NM (EM) and IM groups was not feasible, and its use was limited to separating the PM group from other groups.[[13](#B13),[14](#B14),[15](#B15)]

With regard to function, PM was defined as a gene in which both alleles are nonfunctional (**5/*5*), and IM (**5/*10*) as having a nonfunctional and a decreased functional allele. UM and NM (EM) were classified as carrying at least three functional genes (**1/*2N*) and having one or two functional genes (**1/*1*), respectively.[[16](#B16),[17](#B17)] To predict CYP2D6 phenotype, a combination of genetic variations were selected as markers and four data mining techniques were employed to predict IM and PM groups in Hong Kong-Chinese data.[[18](#B18)] Two pieces of genetic information, *188C>T *and deletion genes, were sufficient to predict IM and PM groups that had clinical significance. The *188C>T* mutation was not previously identified in a study of 51 Korean subjects using direct DNA sequencing,[[11](#B11)] suggesting that different ethnic groups within Asian populations are genetically diverse. Previous study with the Hong Kong-Chinese data had applied the supervised method for *CYP2D6* phenotype classification based on known genetic function without phenotype data, which excludes the influence of other genes or environmental factors. Considering the effects of specific genotypes and environmental factors on drug metabolism, it is necessary to classify these genotypes based on phenotypes. Use of genetic information to predict phenotype, as was done in the present study, provides a more accurate assessment.

Gaedigk et al. divided groups into allele scores using the activity score (AS) system, which assigns scores according to the relative functionality of each allele.[[19](#B19)] The possible distribution of DM MR in each score, from 0 to over 2, was presented. Phenotype groups were further defined as slower EM (EM-S), faster EM (EM-F), PM, IM, EM, and UM using arbitrary antimode. Ethnic differences were identified as playing an important role, and the results could not be generalized to all ethnicities. Factors such as food and other genes may also affect CYP2D6 enzyme activity. An advantage of the AS system is that the characteristics of the genotype can be regarded as the distribution of enzyme activity; however, the disadvantage is that information on the function of each gene must precede it. Recent studies have emphasized the need to standardize CYP2D6 phenotype assignment from genotype, proposing the AS system as a more suitable option than the traditional classification method (four groups from UM to PM).[[20](#B20)] The AS system has been used to predict phenotypes from *CYP2D6* genotype across major populations using data from previous research.[[21](#B21)]

This study classified CYP2D6 phenotype for each genotype using an unsupervised method without considering the known genotype classification. The phenotype features extracted from each genotype were the mean, median, and SD of DM log MR values of subjects with the same genotype. More accurate results could be obtained with a larger number of subjects representing each genotype. A single subject may yield results that are inconsistent with other studies. It was possible to classify low frequency genotypes, including a single subject, in our study. The cluster with the lowest enzyme activity consisted of nonfunctional alleles and the cluster with the highest enzyme activity contained alleles with amplified function (i.e., *CYP2D6*2N*); similar findings have been observed in previous reports.[[17](#B17),[22](#B22)] Color visualization of the results allowed for rapid and easy identification of the characteristics of each genotype. Users are not required of knowledge on applied clustering algorithm, making the tool usable for clinicians and other researchers.

K-means clustering analysis was applied to the extracted phenotype features. The genotypes of the clustering and validation sets were different, and clustering results could not be applied directly to the validation data to evaluate accuracy. The cutoff value from the ROC curve, a log MR value, was derived from the two closest phenotype activity clusters obtained from the clustering set analysis. Concordance rates were calculated to determine whether genotypes and phenotypes from each individual in the validation set were consistent with the clusters established using the cutoff value. When classified into three clusters, the concordance rate was > 90% in both the clustering and validation sets, confirming that it could be used for clustering of other genotype-phenotype data. When divided into four clusters, the concordance rate was ~85% for the clustering set and ~74% for the validation set. It is possible that fewer subjects were sorted into the cluster with the highest level of activity of four clusters than in the group with three clusters, not effectively reflecting the features of each genotype. The three clusters were better in the aspect of silhouette values and concordance rates, but it would be more useful when divided into more than three clusters when considering usefulness for personalized therapy. Two subjects in the validation set had *CYP2D6*1/*14* and *CYP2D6*1/*21*, which were not observed in the clustering set. A more complete set that includes all genotypes may improve predictions of phenotype based on genotype. This type of cluster analysis can be used widely for ethnicities other than Koreans, if ethnicity-specific factors are considered.

CYP2D6 activity is influenced by ethnicity. High frequencies of **10* in Asian populations and **17* in African populations may influence phenotype patterns.[[21](#B21),[23](#B23)] The frequency of *CYP2D6*10* in the Korean population has been reported as 45.6%.[[11](#B11)] When applying a DM log MR value of ≥ −0.5 (the PM cutoff proposed by Schmid et al.[[15](#B15)]), only **5/*5* was included in the data with a frequency of 0.8%. The findings of this study were consistent with the results of previous studies.

To predict CYP2D6 activity based on *CYP2D6* genotype, we analyzed genotype and phenotype data using two clustering algorithms and statistical analyses. The results were presented using color visualization, which allows for rapid identification of phenotype based on genotype. The clustering results and cutoff values were evaluated using untouched data, and relatively good concordance rates were observed. This technique can be applied to various drug metabolizing enzymes, such as other CYPs, for optimal personalized therapy.

## Acknowledgements

This research was supported by a grant of Korea Health Technology R&D Project through the Korea Health Industry Development Institute (KHIDI), funded by the Ministry of Health & Welfare, Republic of Korea (grant number : HI15C1537).

## Footnotes

## References

1. Ingelman-Sundberg M, Oscarson M, McLellan RA. Polymorphic human cytochrome P450 enzymes: An opportunity for individualized drug treatment. Trends Pharmacol Sci. 1999;20:342–349. doi: 10.1016/s0165-6147(99)01363-2.  [DOI](https://doi.org/10.1016/s0165-6147(99)01363-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10431214/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Pharmacol%20Sci&title=Polymorphic%20human%20cytochrome%20P450%20enzymes:%20An%20opportunity%20for%20individualized%20drug%20treatment&author=M%20Ingelman-Sundberg&author=M%20Oscarson&author=RA%20McLellan&volume=20&publication_year=1999&pages=342-349&pmid=10431214&doi=10.1016/s0165-6147(99)01363-2&)

2. Budnitz DS, Shehab N, Kegler SR, Richards CL. Medication use leading to emergency department visits for adverse drug events in older adults. Ann Intern Med. 2007;147:755–765. doi: 10.7326/0003-4819-147-11-200712040-00006.  [DOI](https://doi.org/10.7326/0003-4819-147-11-200712040-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18056659/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Medication%20use%20leading%20to%20emergency%20department%20visits%20for%20adverse%20drug%20events%20in%20older%20adults&author=DS%20Budnitz&author=N%20Shehab&author=SR%20Kegler&author=CL%20Richards&volume=147&publication_year=2007&pages=755-765&pmid=18056659&doi=10.7326/0003-4819-147-11-200712040-00006&)

3. Ma MK, Woo MH, McLeod HL. Genetic basis of drug metabolism. Am J Health Syst Pharm. 2002;59:2061–2069. doi: 10.1093/ajhp/59.21.2061.  [DOI](https://doi.org/10.1093/ajhp/59.21.2061) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12434718/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Health%20Syst%20Pharm&title=Genetic%20basis%20of%20drug%20metabolism&author=MK%20Ma&author=MH%20Woo&author=HL%20McLeod&volume=59&publication_year=2002&pages=2061-2069&pmid=12434718&doi=10.1093/ajhp/59.21.2061&)

4. Nebert DW, Wikvall K, Miller WL. Human cytochromes P450 in health and disease. Philos Trans R Soc Lond B Biol Sci. 2013;368:20120431. doi: 10.1098/rstb.2012.0431.  [DOI](https://doi.org/10.1098/rstb.2012.0431) | [PMC free article](/articles/PMC3538421/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23297354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Philos%20Trans%20R%20Soc%20Lond%20B%20Biol%20Sci&title=Human%20cytochromes%20P450%20in%20health%20and%20disease&author=DW%20Nebert&author=K%20Wikvall&author=WL%20Miller&volume=368&publication_year=2013&pages=20120431&pmid=23297354&doi=10.1098/rstb.2012.0431&)

5. Cytochrome P450 homepage. [html Accessed 18 July 2017]. http://drnelson.uthsc.edu/cytochromeP450.  [http://drnelson.uthsc.edu/cytochromeP450](http://drnelson.uthsc.edu/cytochromeP450)

6. Ingelman-Sundberg M. Genetic polymorphisms of cytochrome P450 2D6 (CYP2D6): Clinical consequences, evolutionary aspects and functional diversity. Pharmacogenomics J. 2005;5:6–13. doi: 10.1038/sj.tpj.6500285.  [DOI](https://doi.org/10.1038/sj.tpj.6500285) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15492763/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genetic%20polymorphisms%20of%20cytochrome%20P450%202D6%20(CYP2D6):%20Clinical%20consequences,%20evolutionary%20aspects%20and%20functional%20diversity&author=M%20Ingelman-Sundberg&volume=5&publication_year=2005&pages=6-13&pmid=15492763&doi=10.1038/sj.tpj.6500285&)

7. Zanger UM, Schwab M. Cytochrome P450 enzymes in drug metabolism: Regulation of gene expression, enzyme activities, and impact of genetic variation. Pharmacol Ther. 2013;138:103–141. doi: 10.1016/j.pharmthera.2012.12.007.  [DOI](https://doi.org/10.1016/j.pharmthera.2012.12.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23333322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Ther&title=Cytochrome%20P450%20enzymes%20in%20drug%20metabolism:%20Regulation%20of%20gene%20expression,%20enzyme%20activities,%20and%20impact%20of%20genetic%20variation&author=UM%20Zanger&author=M%20Schwab&volume=138&publication_year=2013&pages=103-141&pmid=23333322&doi=10.1016/j.pharmthera.2012.12.007&)

8. Sim SC, Ingelman-Sundberg M. Update on allele nomenclature for human cytochromes P450 and the human cytochrome P450 allele (CYP-allele) nomenclature database. Methods Mol Biol. 2013;987:251–259. doi: 10.1007/978-1-62703-321-3_21.  [DOI](https://doi.org/10.1007/978-1-62703-321-3_21) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23475683/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&title=Update%20on%20allele%20nomenclature%20for%20human%20cytochromes%20P450%20and%20the%20human%20cytochrome%20P450%20allele%20(CYP-allele)%20nomenclature%20database&author=SC%20Sim&author=M%20Ingelman-Sundberg&volume=987&publication_year=2013&pages=251-259&pmid=23475683&doi=10.1007/978-1-62703-321-3_21&)

9. Bernard S, Neville KA, Nguyen AT, Flockhart DA. Interethnic differences in genetic polymorphisms of CYP2D6 in the U.S. Population: Clinical implications. Oncologist. 2006;11:126–135. doi: 10.1634/theoncologist.11-2-126.  [DOI](https://doi.org/10.1634/theoncologist.11-2-126) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16476833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Interethnic%20differences%20in%20genetic%20polymorphisms%20of%20CYP2D6%20in%20the%20U.S.%20Population:%20Clinical%20implications&author=S%20Bernard&author=KA%20Neville&author=AT%20Nguyen&author=DA%20Flockhart&volume=11&publication_year=2006&pages=126-135&pmid=16476833&doi=10.1634/theoncologist.11-2-126&)

10. Caudle KE, Dunnenberger HM, Freimuth RR, Peterson JF, Burlison JD, Whirl-Carrillo M, et al. Standardizing terms for clinical pharmacogenetic test results: Consensus terms from the Clinical Pharmacogenetics Implementation Consortium (CPIC) Genet Med. 2017;19:215–223. doi: 10.1038/gim.2016.87.  [DOI](https://doi.org/10.1038/gim.2016.87) | [PMC free article](/articles/PMC5253119/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27441996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=Standardizing%20terms%20for%20clinical%20pharmacogenetic%20test%20results:%20Consensus%20terms%20from%20the%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)&author=KE%20Caudle&author=HM%20Dunnenberger&author=RR%20Freimuth&author=JF%20Peterson&author=JD%20Burlison&volume=19&publication_year=2017&pages=215-223&pmid=27441996&doi=10.1038/gim.2016.87&)

11. Lee SJ, Lee SS, Jung HJ, Kim HS, Park SJ, Yeo CW, et al. Discovery of novel functional variants and extensive evaluation of CYP2D6 genetic polymorphisms in Koreans. Drug Metab Dispos. 2009;37:1464–1470. doi: 10.1124/dmd.108.022368.  [DOI](https://doi.org/10.1124/dmd.108.022368) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19364831/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Discovery%20of%20novel%20functional%20variants%20and%20extensive%20evaluation%20of%20CYP2D6%20genetic%20polymorphisms%20in%20Koreans&author=SJ%20Lee&author=SS%20Lee&author=HJ%20Jung&author=HS%20Kim&author=SJ%20Park&volume=37&publication_year=2009&pages=1464-1470&pmid=19364831&doi=10.1124/dmd.108.022368&)

12. Kim EY, Lee SS, Jung HJ, Jung HE, Yeo CW, Shon JH, et al. Robust CYP2D6 genotype assay including copy number variation using multiplex single-base extension for Asian populations. Clin Chim Acta. 2010;411:2043–2048. doi: 10.1016/j.cca.2010.08.042.  [DOI](https://doi.org/10.1016/j.cca.2010.08.042) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20828547/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=Robust%20CYP2D6%20genotype%20assay%20including%20copy%20number%20variation%20using%20multiplex%20single-base%20extension%20for%20Asian%20populations&author=EY%20Kim&author=SS%20Lee&author=HJ%20Jung&author=HE%20Jung&author=CW%20Yeo&volume=411&publication_year=2010&pages=2043-2048&pmid=20828547&doi=10.1016/j.cca.2010.08.042&)

13. Gaedigk A, Gotschall RR, Forbes NS, Simon SD, Kearns GL, Leeder JS. Optimization of cytochrome P450 2D6 (CYP2D6) phenotype assignment using a genotyping algorithm based on allele frequency data. Pharmacogenetics. 1999;9:669–682. doi: 10.1097/01213011-199912000-00002.  [DOI](https://doi.org/10.1097/01213011-199912000-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10634130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Optimization%20of%20cytochrome%20P450%202D6%20(CYP2D6)%20phenotype%20assignment%20using%20a%20genotyping%20algorithm%20based%20on%20allele%20frequency%20data&author=A%20Gaedigk&author=RR%20Gotschall&author=NS%20Forbes&author=SD%20Simon&author=GL%20Kearns&volume=9&publication_year=1999&pages=669-682&pmid=10634130&doi=10.1097/01213011-199912000-00002&)

14. Eichelbaum M, Woolhouse NM. Inter-ethnic difference in sparteine oxidation among Ghanaians and Germans. Eur J Clin Pharmacol. 1985;28:79–83. doi: 10.1007/BF00635712.  [DOI](https://doi.org/10.1007/BF00635712) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3987789/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Inter-ethnic%20difference%20in%20sparteine%20oxidation%20among%20Ghanaians%20and%20Germans&author=M%20Eichelbaum&author=NM%20Woolhouse&volume=28&publication_year=1985&pages=79-83&pmid=3987789&doi=10.1007/BF00635712&)

15. Schmid B, Bircher J, Preisig R, Kupfer A. Polymorphic dextromethorphan metabolism: Co-segregation of oxidative O-demethylation with debrisoquin hydroxylation. Clin Pharmacol Ther. 1985;38:618–624. doi: 10.1038/clpt.1985.235.  [DOI](https://doi.org/10.1038/clpt.1985.235) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4064464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Polymorphic%20dextromethorphan%20metabolism:%20Co-segregation%20of%20oxidative%20O-demethylation%20with%20debrisoquin%20hydroxylation&author=B%20Schmid&author=J%20Bircher&author=R%20Preisig&author=A%20Kupfer&volume=38&publication_year=1985&pages=618-624&pmid=4064464&doi=10.1038/clpt.1985.235&)

16. Sachse C, Brockmöller J, Hildebrand M, Müller K, Roots I. Correctness of prediction of the CYP2D6 phenotype confirmed by genotyping 47 intermediate and poor metabolizers of debrisoquine. Pharmacogenetics. 1998;8:181–185.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10022755/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Correctness%20of%20prediction%20of%20the%20CYP2D6%20phenotype%20confirmed%20by%20genotyping%2047%20intermediate%20and%20poor%20metabolizers%20of%20debrisoquine&author=C%20Sachse&author=J%20Brockm%C3%B6ller&author=M%20Hildebrand&author=K%20M%C3%BCller&author=I%20Roots&volume=8&publication_year=1998&pages=181-185&pmid=10022755&)

17. Chou WH, Yan FX, Robbins-Weilert DK, Ryder TB, Liu WW, Perbost C, et al. Comparison of two CYP2D6 genotyping methods and assessment of genotype-phenotype relationships. Clin Chem. 2003;49:542–551. doi: 10.1373/49.4.542.  [DOI](https://doi.org/10.1373/49.4.542) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12651805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Comparison%20of%20two%20CYP2D6%20genotyping%20methods%20and%20assessment%20of%20genotype-phenotype%20relationships&author=WH%20Chou&author=FX%20Yan&author=DK%20Robbins-Weilert&author=TB%20Ryder&author=WW%20Liu&volume=49&publication_year=2003&pages=542-551&pmid=12651805&doi=10.1373/49.4.542&)

18. Sabbagh A, Darlu P. Data-mining methods as useful tools for predicting individual drug response: Application to CYP2D6 data. Hum Hered. 2006;62:119–134. doi: 10.1159/000096416.  [DOI](https://doi.org/10.1159/000096416) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17057402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Hered&title=Data-mining%20methods%20as%20useful%20tools%20for%20predicting%20individual%20drug%20response:%20Application%20to%20CYP2D6%20data&author=A%20Sabbagh&author=P%20Darlu&volume=62&publication_year=2006&pages=119-134&pmid=17057402&doi=10.1159/000096416&)

19. Gaedigk A, Simon SD, Pearce RE, Bradford LD, Kennedy MJ, Leeder JS. The CYP2D6 activity score: Translating genotype information into a qualitative measure of phenotype. Clin Pharmacol Ther. 2008;83:234–242. doi: 10.1038/sj.clpt.6100406.  [DOI](https://doi.org/10.1038/sj.clpt.6100406) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20CYP2D6%20activity%20score:%20Translating%20genotype%20information%20into%20a%20qualitative%20measure%20of%20phenotype&author=A%20Gaedigk&author=SD%20Simon&author=RE%20Pearce&author=LD%20Bradford&author=MJ%20Kennedy&volume=83&publication_year=2008&pages=234-242&pmid=17971818&doi=10.1038/sj.clpt.6100406&)

20. Hicks JK, Swen JJ, Gaedigk A. Challenges in CYP2D6 phenotype assignment from genotype data: A critical assessment and call for standardization. Curr Drug Metab. 2014;15:218–232. doi: 10.2174/1389200215666140202215316.  [DOI](https://doi.org/10.2174/1389200215666140202215316) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24524666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Drug%20Metab&title=Challenges%20in%20CYP2D6%20phenotype%20assignment%20from%20genotype%20data:%20A%20critical%20assessment%20and%20call%20for%20standardization&author=JK%20Hicks&author=JJ%20Swen&author=A%20Gaedigk&volume=15&publication_year=2014&pages=218-232&pmid=24524666&doi=10.2174/1389200215666140202215316&)

21. Gaedigk A, Sangkuhl K, Whirl-Carrillo M, Klein T, Leeder JS. Prediction of CYP2D6 phenotype from genotype across world populations. Genet Med. 2017;19:69–76. doi: 10.1038/gim.2016.80.  [DOI](https://doi.org/10.1038/gim.2016.80) | [PMC free article](/articles/PMC5292679/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27388693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=Prediction%20of%20CYP2D6%20phenotype%20from%20genotype%20across%20world%20populations&author=A%20Gaedigk&author=K%20Sangkuhl&author=M%20Whirl-Carrillo&author=T%20Klein&author=JS%20Leeder&volume=19&publication_year=2017&pages=69-76&pmid=27388693&doi=10.1038/gim.2016.80&)

22. Streetman DS, Ellis RE, Nafziger AN, Leeder JS, Gaedigk A, Gotschall R, et al. Dose dependency of dextromethorphan for cytochrome P450 2D6 (CYP2D6) phenotyping. Clin Pharmacol Ther. 1999;66:535–541. doi: 10.1016/S0009-9236(99)70018-4.  [DOI](https://doi.org/10.1016/S0009-9236(99)70018-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10579482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Dose%20dependency%20of%20dextromethorphan%20for%20cytochrome%20P450%202D6%20(CYP2D6)%20phenotyping&author=DS%20Streetman&author=RE%20Ellis&author=AN%20Nafziger&author=JS%20Leeder&author=A%20Gaedigk&volume=66&publication_year=1999&pages=535-541&pmid=10579482&doi=10.1016/S0009-9236(99)70018-4&)

23. Droll K, Bruce-Mensah K, Otton SV, Gaedigk A, Sellers EM, Tyndale RF. Comparison of three CYP2D6 probe substrates and genotype in Ghanaians, Chinese and Caucasians. Pharmacogenetics. 1998;8:325–333. doi: 10.1097/00008571-199808000-00006.  [DOI](https://doi.org/10.1097/00008571-199808000-00006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9731719/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Comparison%20of%20three%20CYP2D6%20probe%20substrates%20and%20genotype%20in%20Ghanaians,%20Chinese%20and%20Caucasians&author=K%20Droll&author=K%20Bruce-Mensah&author=SV%20Otton&author=A%20Gaedigk&author=EM%20Sellers&volume=8&publication_year=1998&pages=325-333&pmid=9731719&doi=10.1097/00008571-199808000-00006&)
