# Role of Genetic Variation in Cytochromes P450 in Breast Cancer Prognosis and Therapy Response

## Metadata
**Authors:** Viktor Hlaváč, Radka Václavíková, Veronika Brynychová, Pavel Ostašov, Renata Koževnikovová, Katerina Kopečková, David Vrána, Jiří Gatěk, Pavel Souček
**Journal:** International Journal of Molecular Sciences
**Date:** 2021 Mar 10
**DOI:** [10.3390/ijms22062826](https://doi.org/10.3390/ijms22062826)
**PMID:** 33802237
**PMCID:** PMC8001203
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8001203/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8001203/pdf/ijms-22-02826.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8001203/pdf/ijms-22-02826.pdf)

## Abstract

Breast cancer is the most frequent cancer in the female population worldwide. The role of germline genetic variability in cytochromes P450 (CYP) in breast cancer prognosis and individualized therapy awaits detailed elucidation. In the present study, we used the next-generation sequencing to assess associations of germline variants in the coding and regulatory sequences of all human CYP genes with response of the patients to the neoadjuvant cytotoxic chemotherapy and disease-free survival (n = 105). A total of 22 prioritized variants associating with a response or survival in the above evaluation phase were then analyzed by allelic discrimination in the large confirmation set (n = 802). Associations of variants in CYP1B1, CYP4F12, CYP4X1, and TBXAS1 with the response to the neoadjuvant cytotoxic chemotherapy were replicated by the confirmation phase. However, just association of variant rs17102977 in CYP4X1 passed the correction for multiple testing and can be considered clinically and statistically validated. Replicated associations for variants in CYP4X1, CYP24A1, and CYP26B1 with disease-free survival of all patients or patients stratified to subgroups according to therapy type have not passed a false discovery rate test. Although statistically not confirmed by the present study, the role of CYP genes in breast cancer prognosis should not be ruled out. In conclusion, the present study brings replicated association of variant rs17102977 in CYP4X1 with the response of patients to the neoadjuvant cytotoxic chemotherapy and warrants further research of genetic variation CYPs in breast cancer.

Keywords: breast cancer, cytochrome P450, therapy, response, survival, prognosis, next-generation sequencing

## 1. Introduction

Breast cancer (Online Mendelian Inheritance in Man, OMIM No. 114480) is the most frequent cancer in females worldwide [[1](#B1-ijms-22-02826)], with 2.1 million new cases diagnosed and more than 620,000 individuals deceased in 2018 [[2](#B2-ijms-22-02826)]. Treatment of so many patients is a serious burden for healthcare systems and calls for an individualized approach. Imbalances in absorption, distribution, metabolism, and excretion of drugs used for cancer therapy influence the drug efficacy and cause adverse drug reactions in some patients. Thus, inter-individual genetic variations in drug-metabolizing enzymes became important in the clinical setting [[3](#B3-ijms-22-02826)]. However, prognostic and predictive biomarkers for precision therapy of breast cancer, e.g., pharmacogenetic variants, are still missing [[4](#B4-ijms-22-02826)].

Cytochromes P450 (CYPs) represent a large superfamily of membrane hemoproteins classified into 18 families in humans [[5](#B5-ijms-22-02826),[6](#B6-ijms-22-02826)]. CYP monooxygenases oxidize or reduce a broad range of physiological substrates, e.g., sterols and fatty acids [[7](#B7-ijms-22-02826)] and xenobiotics, including drugs [[8](#B8-ijms-22-02826)], and together with ATP-binding transporters, represent the majority of known pharmacogenes [[4](#B4-ijms-22-02826)]. However, of the total number of 57 human CYP coding genes with putative enzymatic activities, only 15 seem to metabolize drugs to date [[9](#B9-ijms-22-02826)], and genetic polymorphisms of *CYP2C9*, *CYP2C19*, and *CYP2D6* belong to the most frequently studied in pharmacogenomics [[6](#B6-ijms-22-02826)]. Although the rest of the CYPs are not involved in drug metabolism perturbations in the homeostasis of steroid hormones, e.g., estrogen may also influence the prognosis and therapy outcome of the patients [[10](#B10-ijms-22-02826)]. Moreover, inhibitors of CYP19 aromatase are frequently used for endocrine therapy of breast cancer patients, while the role of genetic variability of the target gene in the treatment efficacy or adverse effects is yet unexploited for therapeutic decisions [[11](#B11-ijms-22-02826)].

Our pilot pharmacogenomics study followed germline alterations in 509 selected genes and their potential for prognosis and prediction of response to therapy in breast cancer patients [[12](#B12-ijms-22-02826)]. We provided a proof-of-the principle for the study design and established bioinformatics methodology for variant prioritization. Out of all genes whose coding and regulatory sequences were screened by the next-generation sequencing (NGS) approach, only a few variants were validated in the replication phase. We down-sampled the first-phase results by the synthesis of in silico predictions and statistically significant clinical associations. This strict process resulted in considerably smaller numbers of variants for replication, and some potentially useful biomarkers of prognosis or prediction of therapy outcome potentially remained unexplored.

In the present study, we used less-strict criteria for investigating the importance of germline genetic variability in coding, untranslated regions (UTR), and adjacent regions of all human members of the CYP superfamily for prognosis and response to the neoadjuvant cytotoxic therapy (NACT) of breast cancer patients. First, we correlated variants with the response of patients to NACT and disease-free survival (DFS). We then thoroughly reviewed haplotypes and gene dosage and corrected results for multiple testing. This study has not addressed the functional relevance to enable also the identification of purely correlative biomarkers. Prioritized variants underwent confirmation in a large cohort of breast cancer patients from a single population. Taken together, the present study brings a more detailed view of the relevance of germline genetic variability of CYPs for breast cancer prognosis and therapy outcome.

## 2. Results

### 2.1. Evaluation Phase

The clinical characteristics of the patients (n = 105) are in [Table S1](#app1-ijms-22-02826). A subgroup of patients received NACT (n = 68), and the response to this treatment was available. The rest of the patients received adjuvant cytotoxic therapy based on monotherapy or combinations of anthracyclines, cyclophosphamide, 5-fluorouracil, and taxanes. The mean follow-up of the patients was 70 ± 28 months.

The average coverage was 92.5 ± 32.2 with 94.6% of the captured regions covered at least 10 times. Altogether, we found 1274 variants in exonic and adjacent intronic sequences. The human CYP superfamily (57 genes) contained on average 22.4 ± 13.0 variants per gene. The lowest counts of variants were found in *CYP21A2* (one variant), *CYP26C1* (seven), *CYP3A4*, and *CYP19A1* (both eight variants). On the other hand, *CYP4V2* (58 variants), *CYP4F8* (56), and *CYP4F12* (55) were the most polymorphic genes. Of the total number of 1274 variants, 302 were in exons, 685 intronic, and 210 were in 3′ or 5′ UTRs according to the National Center for Biotechnology Information (NCBI) Reference Sequence Database (RefSeq; [https://www.ncbi.nlm.nih.gov/refseq/](https://www.ncbi.nlm.nih.gov/refseq/)) ([Table 1](#ijms-22-02826-t001)).

### Table 1.

| Function | Total | Percentage |
| --- | --- | --- |
| Intronic | 685 | 53.8 |
| Exonic (coding) | 302 | 23.7 |
| 3′UTR | 167 | 13.1 |
| 5′UTR | 43 | 3.4 |
| Upstream 1 | 45 | 3.5 |
| Downstream 1 | 19 | 1.5 |
| Intergenic | 10 | 0.8 |
| Splicing 2 | 3 | 0.2 |

Table 1 Caption: Observed alterations in cytochromes P450 divided by function according to Annovar.

On average, each patient showed 336 ± 27 variants. We found 14 loss-of-function (LOF) truncating variants that were either affecting the stop codon (stop-gain) or frameshift insertions or deletions (indels). Out of exonic variants, 178 were non-synonymous single-nucleotide variants (SNVs) and 99 synonymous SNVs ([Table 2](#ijms-22-02826-t002)). In total, 568 variants (45%) had minor allele frequency (MAF) > 0.05; the rest, 706 variants, had MAF 0.05 or below.

### Table 2.

| Classification | Total | Percentage |
| --- | --- | --- |
| Non-synonymous SNV | 178 | 58.9 |
| Synonymous SNV | 99 | 32.8 |
| Stop-gain | 6 | 2.0 |
| Frameshift deletion | 4 | 1.3 |
| Frameshift insertion | 4 | 1.3 |
| Non-frameshift deletion | 2 | 0.7 |
| Unknown | 9 | 3.0 |

Table 2 Caption: Overview of the observed exonic alterations in cytochromes P450 by coding consequence.

Altogether, 103 (8%) of the variants were novel (i.e., not found in dbSNP Build 150). Out of these, ten had MAF > 0.01 and were classified as single-nucleotide polymorphisms (SNPs) in the Czech population. The distribution of variants according to their functional classes and frequencies of novel variants in gene groups is shown in [Figure 1](#ijms-22-02826-f001).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cb1a/8001203/a82fd7a207e3/ijms-22-02826-g001a.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8001203_ijms-22-02826-g001a.jpg)

Distribution of alterations in individual cytochromes P450 (CYPs). The picture shows: (a) the frequency of genetic alterations according to their functional classes; (b) The frequency of genetic alterations according to their exonic functional classes analyzed by the National Center for Biotechnology Information Reference Sequence Database (https://www.ncbi.nlm.nih.gov/refseq/) (Access on: 29 September 2019); (c) The distribution of novel variants. Numbers of variants, normalized to the transcript length in kilobase pairs (kbp), are depicted for each gene by the overlaid line in each plot on the right axis.

Variants departing from Hardy–Weinberg equilibrium (*p* < 0.01, n = 76) were excluded from analysis. Further, we selected variants with MAF > 0.05 to achieve adequate statistical power in the confirmation phase. Besides, variants with the missing data in more than 50% of patients were excluded (n = 25). This filtration process resulted in a set of 564 variants which were further evaluated for associations with the response of patients to the NACT and DFS. We found 32 variants associated with the response to the NACT and other 32 variants with DFS. Following haplotype evaluations, 27 variants were considered tagging other selected variants (r^2^ > 0.8) and not assessed further. The gene dosage relationship was then evaluated for variants associated with DFS, and variants not fulfilling this condition were excluded (n = 13). Neither of these variants was significant in the recessive genetic model (variant allele versus common homozygote). Following these control steps, we prioritized 24 variants (23 SNVs and 1 insertion, [Table S2](#app1-ijms-22-02826)) for the confirmation phase in a larger cohort of breast cancer patients, but optimization of three variants failed during the reaction design. One variant was then included in the list (rs593421) based on haplotype tagging (r^2^ = 1) to replace the variant rs79882219, whose analysis failed. No tagging variants (r^2^ > 0.8) were available to replace the rest of failed variants.

### 2.2. Confirmation Phase

The clinical characteristics of the patients (n = 802) are shown in [Table S3](#app1-ijms-22-02826). A subgroup of patients treated with the NACT comprised 168 patients. In total, 371 patients received adjuvant cytotoxic therapy. Patients with localized disease and the generally good prognosis did not receive any further treatment (n = 83), and a portion of patients was treated only with hormonal therapy (n = 311). The mean follow-up of the patients was 76 ± 30 months.

All successfully genotyped variants (n = 22) were in Hardy–Weinberg equilibrium. Less than 1% of theoretical data points were missing due to the uncertainty in genotype calling or absent signal. [Table 3](#ijms-22-02826-t003) summarizes genotypes’ distribution of the variants in the confirmation phase. The MAFs of assessed variants in the confirmation set were comparable to MAFs observed in the evaluation set.

### Table 3.

| Gene | SNP ID 1 | Genotype Distribution 2 |  |  | Minor Allele Frequency |  |
| --- | --- | --- | --- | --- | --- | --- |
|   |   | Common Homozygotes | Heterozygotes | Rare Homozygotes | Confirmation Set | Evaluation Set |
| CYP1B1 | rs1056827 | 362 | 354 | 77 | 0.32 | 0.34 |
| CYP2S1 | rs184623 | 308 | 379 | 100 | 0.37 | 0.38 |
| CYP2W1 | rs3808348 | 538 | 237 | 23 | 0.18 | 0.20 |
| CYP2W1 | rs12701220 | 533 | 239 | 25 | 0.18 | 0.11 |
| CYP4A11 | rs3890011 | 459 | 291 | 46 | 0.24 | 0.27 |
| CYP4F2 | rs2074900 | 367 | 343 | 83 | 0.32 | 0.32 |
| CYP4F2 | rs3093198 | 398 | 325 | 73 | 0.30 | 0.29 |
| CYP4F8 | rs714772 | 506 | 258 | 35 | 0.21 | 0.25 |
| CYP4F8 | rs4646522 | 225 | 401 | 158 | 0.46 | 0.42 |
| CYP4F12 | rs593421 | 416 | 308 | 54 | 0.27 | 0.29 |
| CYP4F12 | rs593818 | 230 | 373 | 187 | 0.47 | 0.43 |
| CYP4F12 | rs2074568 | 518 | 211 | 23 | 0.17 | 0.21 |
| CYP4V2 | rs62350517 | 693 | 104 | 4 | 0.07 | 0.08 |
| CYP4X1 | rs17102977 | 653 | 125 | 8 | 0.09 | 0.10 |
| CYP24A1 | rs2259735 | 246 | 365 | 155 | 0.44 | 0.39 |
| CYP24A1 | rs2762934 | 549 | 231 | 17 | 0.17 | 0.17 |
| CYP24A1 | rs6022999 | 496 | 251 | 50 | 0.22 | 0.21 |
| CYP24A1 | rs10623012 | 294 | 382 | 105 | 0.38 | 0.32 |
| CYP26B1 | rs61138718 | 606 | 183 | 12 | 0.13 | 0.11 |
| CYP26B1 | rs62150087 | 661 | 132 | 6 | 0.09 | 0.07 |
| CYP27C1 | rs12476709 | 236 | 379 | 174 | 0.46 | 0.47 |
| TBXAS1 | rs3819733 | 590 | 195 | 14 | 0.14 | 0.15 |

Table 3 Caption: Distribution of genotypes for variants assessed in the confirmation phase.

For validation purposes, we also evaluated associations of variants with the response to the NACT and DFS of patients in the confirmation set. For SNPs with less frequent homozygous genotypes (n < 5), the recessive genetic model was used to maintain sufficient statistical power. Significant results for response are in [Table 4](#ijms-22-02826-t004) and [Figure 2](#ijms-22-02826-f002). Subsequently, we evaluated associations of variants with DFS of all patients and patients stratified according to the received therapy. Significant results for all patients with complete follow-up data (n = 744) are in [Figure 3](#ijms-22-02826-f003)a, for patients treated with cytotoxic therapy (n = 373) in [Figure 3](#ijms-22-02826-f003)b, and for patients treated only with hormonal therapy (n = 312) in [Figure 3](#ijms-22-02826-f003)c.

### Table 4.

| Gene | SNP ID | Genotype | Good Response 1 | Poor Response 1 | χ−Square | p |
| --- | --- | --- | --- | --- | --- | --- |
| CYP1B1 3 | rs1056827 | C allele | 122 | 35 | 3.96 | 0.047/0.339 2 |
| AA | 5 | 5 |  |  |  |  |
| CYP4F12 | rs593421 | TT | 63 | 22 | 8.81 | 0.012/0.130 2 |
| TC | 57 | 12 |  |  |  |  |
| CC | 4 | 6 |  |  |  |  |
| CYP4X1 | rs17102977 | AA | 111 | 27 | 12.02 | 5.30 × 10−4/0.034 2 |
| G allele | 12 | 13 |  |  |  |  |
| TBXAS1 | rs3819733 | TT | 81 | 35 | 6.76 | 0.009/0.130 2 |
| C allele | 46 | 6 |  |  |  |  |

Table 4 Caption: Cytochrome P450 variants significantly associating with the response of patients to the neoadjuvant cytotoxic therapy in the confirmation set.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cb1a/8001203/bda44b3ff750/ijms-22-02826-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8001203_ijms-22-02826-g002.jpg)

Common odds ratios of variants significantly associated with the response to neoadjuvant chemotherapy. OR = odds ratio.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cb1a/8001203/7eb79b3168f0/ijms-22-02826-g003a.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8001203_ijms-22-02826-g003a.jpg)

Kaplan–Meier plots with significant associations of cytochrome P450 variants with disease-free survival. (a) All patients, (b) subgroup of patients treated with the cytotoxic therapy, and (c) subgroup of patients treated only with the hormonal therapy. Blue line represents the common homozygous genotype, green heterozygote, and magenta rare homozygote. Violet color represents rare allele carriers (recessive model). Significance was evaluated by the log-rank test; numbers show individuals in the compared groups.

Association of SNP rs17102977 in *CYP4X1* with the response to the NACT passed the correction for multiple testing and therefore can be considered clinically and statistically validated. The rest of the associations with response to NACT did not pass the false discovery rate (FDR) test ([Table 4](#ijms-22-02826-t004)). Several significant associations with DFS were observed as well, but none of these associations remained significant after the adjustment for multiple comparisons. Interestingly, the variant rs17102977 in *CYP4X1*, associating with the response, also associated with DFS in the group of all patients without regard to therapy and in the subgroup treated only with hormonal therapy ([Figure 3](#ijms-22-02826-f003)a,c).

In a multivariate analysis adjusted to tumor size and grade, presence of regional lymph node metastasis, and estrogen receptor status, the rs17102977 in *CYP4X1* significantly associated with DFS in the unselected set of patients (*p* = 0.048; hazard ratio, HR = 1.69; 95% confidence interval, CI = 1.01–2.85). The rs62150087 in *CYP26B1* significantly associated with DFS in a group of cytotoxic therapy-treated patients (*p* = 0.016, HR = 0.54, CI = 0.33–0.89), but not in unselected patients (*p* = 0.122). The rs2762934 SNP in *CYP24A1* in a subgroup of the cytotoxic therapy-treated group was insignificant in multivariate analysis (*p* = 0.075).

To clarify the effect of intrinsic molecular subtypes on the prognosis, we stratified patients into four groups: Luminal A, Luminal B, HER2-enriched, and triple-negative breast cancer (TNBC) according to their subtypes. Subsequently, we assessed the associations with DFS for each subtype separately. Results are depicted in [Table 5](#ijms-22-02826-t005) and [Figure S1](#app1-ijms-22-02826). Variant rs62150087 in *CYP26B1* was significantly associated with DFS in the HER2-enriched group of the patients regardless of the therapy (all patients or patients who received cytotoxic therapy). Rs2762934 in *CYP24A1* was significantly associated with DFS in a subgroup of TNBC patients treated with cytotoxic therapy. Rs17102977 in *CYP4X1* was insignificant in all subtypes ([Table 5](#ijms-22-02826-t005)).

### Table 5.

| Gene | SNP ID | Genotypes | Subtypes |  |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| Luminal A | Luminal B | HER2 | TNBC |  |  |  |
| All patients (n = 744) |  |  |  |  |  |  |
| CYP26B1 | rs62150087 | CC 1 | 174 | 230 | 44 | 73 |
|   |   | G allele 1 | 36 | 42 | 12 | 12 |
|   |   | p 2 | 0.754 | 0.086 | 0.010 | 0.178 |
| CYP4X1 | rs17102977 | AA 1 | 166 | 223 | 49 | 48 |
|   |   | G allele 1 | 44 | 42 | 6 | 16 |
|   |   | p 2 | 0.245 | 0.130 | 0.150 | 0.778 |
| Patients treated with cytotoxic therapy (n = 371) |  |  |  |  |  |  |
| CYP26B1 | rs62150087 | CC 1 | 65 | 128 | 26 | 58 |
|   |   | G allele 1 | 9 | 25 | 10 | 8 |
|   |   | p 2 | 0.244 | 0.232 | 0.011 | 0.060 |
| CYP24A1 | rs2762934 | GG 1 | 50 | 91 | 27 | 45 |
|   |   | A allele 1 | 24 | 60 | 9 | 19 |
|   |   | p 2 | 0.181 | 0.172 | 0.400 | 0.001 |
| Patients treated only with hormonal therapy (n = 311) |  |  |  |  |  |  |
| CYP4X1 | rs17102977 | AA 1 | 102 | 81 | 3 | 1 |
|   |   | G allele 1 | 22 | 19 | 0 | 1 |
|   |   | p 2 | 0.123 | 0.202 | N/A | 0.317 |

Table 5 Caption: The effect of breast cancer molecular subtypes on cytochrome P450 variants significantly associating with DFS of patients in the confirmation set.

## 3. Discussion

We analyzed associations of genetic variants in all human CYP monooxygenases with chemotherapy outcome and survival of breast cancer patients. Firstly, we genotyped all coding sequences and surrounding areas using the next-generation sequencing. Variants significantly associated with response to NACT and DFS of the patients using Chi-square and log-rank tests with permutations were further assessed in a large cohort of breast cancer patients (n = 802) by competitive allele-specific PCR. Of the total number of 22 variants selected for validation, the results were confirmed, in a large cohort, for six of them.

In total, we found 1274 variants in a set of 105 breast cancer patients used for the evaluation phase. We found 14 LOF variants. Of these, six were stop-gain mutations, and eight were frameshift indels. However, the MAF of these variants was too low to maintain the statistical power for correlation with clinical data precluding their further study. Thus, we evaluated 22 common variants in a large confirmation cohort of patients and assessed their associations with DFS and response to NACT.

The substitution rs17102977 in *CYP4X1* intron is associated with both response of the patients to the NACT and DFS of hormonally treated patients. It is also prognostic in patients unselected according to the therapy. The association of rs17102977 with the response to NACT was significant after correction to multiple testing and thus can be considered validated in both datasets. Patients carrying the rare allele G were more often poorly responding to chemotherapy than the patients with the wild-type genotype AA. Intriguingly, patients bearing rare alleles had longer DFS than wild-type patients. Although the frequency of the rare allele in the European non-Finnish population in GnomAD is 0.08, we found no record for this variant in the scientific literature (PubMed). The gene *CYP4X1* encodes an orphan CYP enzyme expressed mainly in the brain, aorta, or breast [[13](#B13-ijms-22-02826),[14](#B14-ijms-22-02826)]. According to recent studies, this enzyme catalyzes epoxidation of endogenous cannabinoid anandamide and arachidonic acid [[15](#B15-ijms-22-02826),[16](#B16-ijms-22-02826)]. Its important paralog is *CYP4Z1* [[17](#B17-ijms-22-02826)]. The role of *CYP4X1* in cancer has been proposed as well. Protein expression of CYP4X1 was associated with increasing tumor grade in tissue microarrays from 170 breast cancer patients detected by immunohistochemistry [[18](#B18-ijms-22-02826)]. Recently, a lower gene expression of *CYP4X1* was associated with shorter overall survival of Chinese gastric cancer patients treated with capecitabine and oxaliplatin [[19](#B19-ijms-22-02826)]. Two SNPs in *CYP4X1* were also associated with the early onset of Creutzfeldt-Jakob disease in Italian patients [[20](#B20-ijms-22-02826)]. The endocannabinoid system is involved in various physiological processes, including inflammation, immunomodulation, or suppression of different cancers, e.g., breast cancer [[21](#B21-ijms-22-02826)]. Therefore, CYP4X1 might play a role in response to cancer chemotherapy through physiological processes. The role of rs17102977 in cancer is unknown. According to the GTEx portal ([https://www.gtexportal.org](https://www.gtexportal.org)) (Access on: 10 February 2021), the expression quantitative trait loci (eQTL) analysis showed that rs17102977 was significantly associated with *CYP4A22* gene expression in the brain, but not in the breast tissue. No significant association with *CYP4X1* gene expression was found. Therefore, further elucidation of the function of rs17102977 and mainly the whole gene locus 1p33 containing also *CYP4B1*, *CYP4A11, CYP4Z1*, and *CYP4A22* genes is needed. This locus also contains mitogen-activated protein kinase *MNK1*, frequently studied in cancer research.

The variant rs62150087 (500 bp downstream of *CYP26B1*) was associated with DFS in our sets of patients. Carriers of the rare allele G treated with cytotoxic therapy had significantly shorter DFS than wild-type patients, but the observed *p*-value of 0.002 does not guarantee a true positive association after correction for multiple testing. However, this borderline significant result is interesting. A genome-wide association study (GWAS) on the Chinese population identified *CYP26B1* as a candidate gene for esophageal squamous cell carcinoma risk. According to the authors, a variant rs138478634 in *CYP26B1* influences the risk through catabolism of an anticancer nutrient all-*trans* retinoic acid [[22](#B22-ijms-22-02826)]. Considering the ability of CYP26B1 to metabolize retinoic acid, there is support for the observed effect of rs62150087 on worse patient survival. However, the influence of this downstream variant on enzyme function or expression is not known, and no significant eQTLs were found in any of the available tissues.

Among other associations found, the variant rs1056827 in *CYP1B1* was associated with the response of the patients to NACT. Homozygotes of the variant allele A were more often poorly responding to chemotherapy than wild-type patients or heterozygotes. This SNP 355G>T changing Alanine to Serine in codon 119 is associated with susceptibility to colorectal [[23](#B23-ijms-22-02826),[24](#B24-ijms-22-02826)], breast cancer [[25](#B25-ijms-22-02826)], and primary open-angle glaucoma [[26](#B26-ijms-22-02826)]. According to ClinVar, this variant is considered benign, and thus no clear explanation of the observed association is available now. Also, the link between breast cancer risk and therapy response is unclear. CYP1B1 oxidizes estrogen as well as a wide variety of xenobiotics [[14](#B14-ijms-22-02826),[27](#B27-ijms-22-02826)]. This fact perhaps could help to explain associations with both risk and therapy outcome and focus further studies. Similarly, synonymous variant rs593421 in *CYP4F12* has no clear support for an observed association with response. *CYP4F12* was cloned originally from the intestine and liver, its recognized substrates are fatty acids [[13](#B13-ijms-22-02826),[14](#B14-ijms-22-02826)]. Also, intronic variant rs3819733 in thromboxane synthase (*TBXAS1)* was associated with response to NACT in our patients. In the literature, lower *TBXAS1* expression was associated with higher grade and poor prognosis of breast cancer patients [[28](#B28-ijms-22-02826)]. On the contrary, high expression was associated with worse overall survival of patients with diffuse low-grade glioma [[29](#B29-ijms-22-02826)]. CYP24A1 is responsible for vitamin D metabolism [[30](#B30-ijms-22-02826)]. SNP rs2762934 in an intronic region of *CYP24A1* was associated with an increased risk of breast cancer [[31](#B31-ijms-22-02826)], ischemic stroke [[32](#B32-ijms-22-02826)], and hypertension [[33](#B33-ijms-22-02826)]. We have seen an association of rs2762934 with DFS of patients treated with cytotoxic therapy, especially in the TNBC subgroup. Taken together, these associations are interesting but should be treated with caution because of their low significance level after FDR correction, as well as considering vague support in the literature.

The present study has some limitations. First, the modest size of the evaluation set may be seen as a study limitation, because it precludes assessment of the importance of very rare (MAF < 0.001) and rare (<0.01) variants. In light of the recently acknowledged contribution of rare variants to inter-individual variability in drug response [[3](#B3-ijms-22-02826)], this limitation needs attention in future studies in precision oncology. On the other hand, we consider ethnical homogeneity and robustness of clinical follow-up as notable study benefits. Moreover, we have employed the multivariate analyses adjusted to major disease characteristics and stratified patients according to intrinsic molecular subtypes to circumvent the frequently overlooked issue of non-homogeneity from the clinical point of view. Major conclusions of the study remained unchanged. Up to that, further elucidations are needed to explore additional genetic components, e.g., non-coding sequences, copy numbers and structural variations, somatic mutations, etc., of the CYP superfamily in breast cancer.

## 4. Materials and Methods

### 4.1. Patients

In the evaluation phase of the study, we included 105 breast cancer patients diagnosed in the Medicon in Prague and the Hospital Atlas in Zlin (both in the Czech Republic) in the period 2006–2013. The NACT administered to patients (n = 68) before tumor resection contained regimens composed of 5-fluorouracil, anthracyclines, cyclophosphamide (FAC or FEC), and/or taxanes. The rest of the patients underwent adjuvant postoperative treatment based on the same drug combinations. [Table S1](#app1-ijms-22-02826) illustrates the clinical data of the patients.

We used 802 breast cancer patients in the confirmation phase. The patients’ recruitment proceeded between 2001 and 2013 in Medicon and the Motol University Hospital (both in Prague) and the Hospital Atlas in Zlin. Patients received neoadjuvant/adjuvant cytotoxic therapy or hormonal therapy (clinical data in [Table S3](#app1-ijms-22-02826)).

We described the recruitment schema before [[12](#B12-ijms-22-02826),[34](#B34-ijms-22-02826)]. DFS was defined as the time elapsed between surgery and the first disease relapse, including local relapses. The response to the NACT was evaluated using the Response Evaluation Criteria In Solid Tumors (RECIST) based on imaging data retrieved from medical records [[35](#B35-ijms-22-02826)].

Procedures performed in the present study followed the 1964 Helsinki Declaration and its later amendments or comparable ethical standards. The Ethical Commission of the National Institute of Public Health in Prague approved the study protocol (approval code no. 9799-4 (issued on 30 January 2008), 15-25618A (6 August 2014), and 17-28470A (22 June 2016)). All patients were informed about the study, and those who agreed and signed informed consent of the patient further participated in the study.

### 4.2. Panel Sequencing—Evaluation Phase

Blood samples were collected during the diagnostic procedures using tubes with K3EDTA anticoagulant and genomic DNA was isolated from human peripheral blood lymphocytes by the standard phenol/chloroform extraction and ethanol precipitation.

All 57 human CYP genes were sequenced using target enrichment protocol on MiSeq (Illumina, San Diego, CA, USA) platform as described previously [[12](#B12-ijms-22-02826)]. Briefly, reads were mapped on a reference sequence hg19 using Burrows–Wheeler Alignment (BWA) mem tool [[36](#B36-ijms-22-02826)], base and indel recalibration as well as the short indels and SNVs discovery was performed using the genome analysis toolkit (GATK) [[37](#B37-ijms-22-02826)]. Variant annotation was performed with the help of Annovar [[38](#B38-ijms-22-02826)]. For details of the library preparation, target enrichment, data processing, and variant calling, see [[12](#B12-ijms-22-02826)].

### 4.3. Genotyping—Confirmation Phase

Variants in the confirmation phase were commercially genotyped by the allelic discrimination method (KASP™, LGC Genomics, Hoddesdon, UK) using primers and probes designed by the service provider. Of note, 10% of the samples were analyzed in duplicates for quality control. The genotyping concordance between duplicate samples exceeded 99%.

### 4.4. Quantitative Real-Time PCR

Quantitative real-time PCR (qPCR) and evaluation of results were performed as described previously [[34](#B34-ijms-22-02826)]. The relative standard curve was generated from five log dilutions of one non-neoplastic control tissue sample (calibrator). Amplification efficiencies (E) for each reference and target gene were calculated applying the formula E = 10^−1^/slope^−1^. *EIF2B1, MRPL19*, and *IPO8* were used as the most stable reference genes for data normalization [[34](#B34-ijms-22-02826)]. The qPCR study design adhered to the MIQE Guidelines (Minimum Information for Publication of Quantitative Real-Time PCR Experiments [[39](#B39-ijms-22-02826)].

### 4.5. Statistical Analyses

DFS was calculated for groups of patients divided by the common homozygous, heterozygous, and rare homozygous genotype in the evaluation phase. The Kaplan–Meier plot served for visual inspection of gene dosage, and the log-rank test was used for evaluation of statistical significance among survival curves of patients with given genotypes or alleles. We set the study follow-up end to 120 months (10 years), and thus all subjects with DFS exceeding 120 months were censored. The response of patients to the NACT was set to “good” in the case of complete or partial pathological remission (CR/PR) and “poor” for stable or progressive disease (SD/PD). We evaluated associations between genotypes (common homozygous, heterozygous and rare homozygous) and response using the Pearson Chi-square test. The adjusted *p*-value was calculated for each variant and each of these tests. The adjusted *p*-value for the log-rank test was based on 100 permutations of original data. A *p*-value of less than 0.05 after adjustment for multiple testing was considered statistically significant. Variants significantly associating with either DFS or the response to NACT in the evaluation phase entered the confirmation phase of the study.

In the confirmation phase, the Pearson Chi-square test and the log-rank tests were used as described above. For the evaluation of variant effects, recessive, dominant, and additive genetic models were employed. Multivariate analysis was done using the Cox proportional hazards model with tumor size and grade, lymph node metastasis, and estrogen receptor expression as covariates. Associations of variants with transcript levels were assessed by one-way ANOVA. Adjusted *p*-values were calculated using the Benjamini–Hochberg FDR test as a correction for multiple testing [[40](#B40-ijms-22-02826)]. Haplotype analysis was conducted in HaploView 4.2 (Broad Institute, Cambridge, MA, USA). Statistical analyses were performed using R (v3.5) with packages VariantAnnotation (v3.8), tidyverse (v1.2.1), survival (2.43), survminer (v0.4.3) and survMisc (v0.5.5) and the statistical program SPSS v16.0 (SPSS, Chicago, IL, USA).

The sequencing data that support the findings of this study are openly available in Sequence Read Archive (SRA, [https://www.ncbi.nlm.nih.gov/sra](https://www.ncbi.nlm.nih.gov/sra)), under accession No. PRJNA510917.

## 5. Conclusions

In conclusion, we have seen associations of selected variants in CYP genes with the response to NACT and with the DFS of the patients. Apart from rs17102977 found in an intronic region of *CYP4X1*, which was associated with response to NACT, no variants passed the correction for multiple testing. Protein expression of *CYP4X1* was already studied in breast cancer, but the role of its genetic variation is unexplored. Another intriguing association we found, though not FDR-validated, is an association of rs62150087 (*CYP26B1*) with shorter DFS. CYP26B1 metabolizes retinoic acid, a compound with potential anticancer abilities. In this regard, a recent GWAS revealed an association of *CYP26B1* with esophageal squamous cell carcinoma risk. However, no studies describing associations with survival are available to date. Taken together, genetic polymorphisms in CYP genes may play a role in the prediction of breast cancer therapy and perhaps modify the prognosis of the patients. More studies must follow to confirm these associations.

## Acknowledgments

The authors would like to thank all participating patients for their kind consent to the study and clinical personnel for outstanding support.

## Abbreviations

| CI | Confidence interval |
| --- | --- |
| CYP | Cytochrome P450 |
| DFS | Disease-free survival |
| FDR | False discovery rate |
| GWAS | Genome-wide association study |
| HR | Hazard ratio |
| LOF | Loss-of-function |
| MAF | Minor allele frequency |
| NACT | Neoadjuvant cytotoxic therapy |
| qPCR | Quantitative real-time PCR |
| SNP | Single nucleotide polymorphism |
| SNV | Single nucleotide variant |
| UTR | Untranslated region |
## Supplementary Materials

Supplementary materials can be found at [https://www.mdpi.com/1422-0067/22/6/2826/s1](https://www.mdpi.com/1422-0067/22/6/2826/s1). Figure S1: The effect of breast cancer molecular subtypes on cytochrome P450 variants significantly associating with DFS of patients in the confirmation set, Table S1: Clinical data of patient in the testing set, Table S2: Prioritized variants for the validation phase, Table S3: Clinical data of patients in the validation set.

## Author Contributions

Conceptualization, P.S., V.H., and R.V.; methodology, V.H.; software, P.O. and V.H.; validation, P.S., V.B., and V.H.; resources, R.K., K.K., D.V., and J.G.; data curation, P.O., R.K., K.K., D.V., and J.G.; writing—original draft preparation, P.S. and V.H.; writing—review and editing, all authors; visualization, P.S., and V.H.; supervision, P.S.; project administration, P.S., V.H., and D.V.; funding acquisition, P.S., V.H., and V.B. All authors have read and agreed to the published version of the manuscript.

## Funding

This research was funded by the Czech Medical Council, grant number NV19-08-00113 to P.S., the Czech Ministry of Education, Youth and Sports, INTER-EXCELLENCE LTA-USA grant number 19032 to R.V., and the Charles University Research Fund, project number Q39 to P.O.

## Institutional Review Board Statement

The study was conducted according to the guidelines of the Declaration of Helsinki, and approved by the Ethical Commission of the National Institute of Public Health in Prague (approval code no. 9799-4 (issued on 30 January 2008), 15-25618A (6 August 2014), and 17-28470A (22 June 2016)).

## Informed Consent Statement

Informed consent was obtained from all subjects involved in the study.

## Data Availability Statement

The sequencing data that support the findings of this study are openly available in Sequence Read Archive (SRA, [https://www.ncbi.nlm.nih.gov/sra](https://www.ncbi.nlm.nih.gov/sra)), under accession No. PRJNA510917 ([https://www.ncbi.nlm.nih.gov/bioproject/PRJNA510917](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA510917)).

## Conflicts of Interest

The authors declare no conflict of interest. The funders had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript, or in the decision to publish the results.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The sequencing data that support the findings of this study are openly available in Sequence Read Archive (SRA, [https://www.ncbi.nlm.nih.gov/sra](https://www.ncbi.nlm.nih.gov/sra)), under accession No. PRJNA510917 ([https://www.ncbi.nlm.nih.gov/bioproject/PRJNA510917](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA510917)).

### Supplementary Materials

### Data Availability Statement

The sequencing data that support the findings of this study are openly available in Sequence Read Archive (SRA, [https://www.ncbi.nlm.nih.gov/sra](https://www.ncbi.nlm.nih.gov/sra)), under accession No. PRJNA510917 ([https://www.ncbi.nlm.nih.gov/bioproject/PRJNA510917](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA510917)).

## References

1. Bray F., Ferlay J., Soerjomataram I., Siegel R.L., Torre L.A., Jemal A. Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. CA Cancer J. Clin. 2018;68:394–424. doi: 10.3322/caac.21492.  [DOI](https://doi.org/10.3322/caac.21492) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30207593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J.%20Clin.&title=Global%20cancer%20statistics%202018:%20GLOBOCAN%20estimates%20of%20incidence%20and%20mortality%20worldwide%20for%2036%20cancers%20in%20185%20countries&author=F.%20Bray&author=J.%20Ferlay&author=I.%20Soerjomataram&author=R.L.%20Siegel&author=L.A.%20Torre&volume=68&publication_year=2018&pages=394-424&pmid=30207593&doi=10.3322/caac.21492&)

2. Heer E., Harper A., Escandor N., Sung H., McCormack V., Fidler-Benaoudia M.M. Global burden and trends in premenopausal and postmenopausal breast cancer: A population-based study. Lancet Glob. Health. 2020;8:e1027–e1037. doi: 10.1016/S2214-109X(20)30215-1.  [DOI](https://doi.org/10.1016/S2214-109X(20)30215-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32710860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Glob.%20Health&title=Global%20burden%20and%20trends%20in%20premenopausal%20and%20postmenopausal%20breast%20cancer:%20A%20population-based%20study&author=E.%20Heer&author=A.%20Harper&author=N.%20Escandor&author=H.%20Sung&author=V.%20McCormack&volume=8&publication_year=2020&pages=e1027-e1037&pmid=32710860&doi=10.1016/S2214-109X(20)30215-1&)

3. Ingelman-Sundberg M., Mkrtchian S., Zhou Y., Lauschke V.M. Integrating rare genetic variants into pharmacogenetic drug response predictions. Hum. Genom. 2018;12:1–12. doi: 10.1186/s40246-018-0157-3.  [DOI](https://doi.org/10.1186/s40246-018-0157-3) | [PMC free article](/articles/PMC5968569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29793534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Genom.&title=Integrating%20rare%20genetic%20variants%20into%20pharmacogenetic%20drug%20response%20predictions&author=M.%20Ingelman-Sundberg&author=S.%20Mkrtchian&author=Y.%20Zhou&author=V.M.%20Lauschke&volume=12&publication_year=2018&pages=1-12&pmid=29793534&doi=10.1186/s40246-018-0157-3&)

4. Hlaváč V., Holý P., Souček P. Pharmacogenomics to predict tumor therapy response: A focus on atp-binding cassette transporters and cytochromes P450. J. Pers. Med. 2020;10:108. doi: 10.3390/jpm10030108.  [DOI](https://doi.org/10.3390/jpm10030108) | [PMC free article](/articles/PMC7565825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32872162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pers.%20Med.&title=Pharmacogenomics%20to%20predict%20tumor%20therapy%20response:%20A%20focus%20on%20atp-binding%20cassette%20transporters%20and%20cytochromes%20P450&author=V.%20Hlav%C3%A1%C4%8D&author=P.%20Hol%C3%BD&author=P.%20Sou%C4%8Dek&volume=10&publication_year=2020&pages=108&pmid=32872162&doi=10.3390/jpm10030108&)

5. Nelson D.R. The Cytochrome P450 Homepage. Hum. Genom. 2009;4:59–65. doi: 10.1186/1479-7364-4-1-59.  [DOI](https://doi.org/10.1186/1479-7364-4-1-59) | [PMC free article](/articles/PMC3500189/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19951895/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Genom.&title=The%20Cytochrome%20P450%20Homepage&author=D.R.%20Nelson&volume=4&publication_year=2009&pages=59-65&pmid=19951895&doi=10.1186/1479-7364-4-1-59&)

6. Gaedigk A., Ingelman-Sundberg M., Miller N.A., Leeder J.S., Whirl-Carrillo M., Klein T.E. The PharmVar steering committee the pharmacogene variation (PharmVar) consortium: Incorporation of the human cytochrome P450 (CYP) allele nomenclature database. Clin. Pharmacol. Ther. 2018;103:399–401. doi: 10.1002/cpt.910.  [DOI](https://doi.org/10.1002/cpt.910) | [PMC free article](/articles/PMC5836850/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29134625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20PharmVar%20steering%20committee%20the%20pharmacogene%20variation%20(PharmVar)%20consortium:%20Incorporation%20of%20the%20human%20cytochrome%20P450%20(CYP)%20allele%20nomenclature%20database&author=A.%20Gaedigk&author=M.%20Ingelman-Sundberg&author=N.A.%20Miller&author=J.S.%20Leeder&author=M.%20Whirl-Carrillo&volume=103&publication_year=2018&pages=399-401&pmid=29134625&doi=10.1002/cpt.910&)

7. Rendic S.P., Guengerich F.P. Human cytochrome P450 enzymes 5–51 as targets of drugs and natural and environmental compounds: Mechanisms, induction, and inhibition—Toxic effects and benefits. Drug Metab. Rev. 2018;50:256–342. doi: 10.1080/03602532.2018.1483401.  [DOI](https://doi.org/10.1080/03602532.2018.1483401) | [PMC free article](/articles/PMC6421578/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30717606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Rev.&title=Human%20cytochrome%20P450%20enzymes%205%E2%80%9351%20as%20targets%20of%20drugs%20and%20natural%20and%20environmental%20compounds:%20Mechanisms,%20induction,%20and%20inhibition%E2%80%94Toxic%20effects%20and%20benefits&author=S.P.%20Rendic&author=F.P.%20Guengerich&volume=50&publication_year=2018&pages=256-342&pmid=30717606&doi=10.1080/03602532.2018.1483401&)

8. Guengerich F.P. Human Cytochrome P450 Enzymes. In: Ortiz de Montellano P.R., editor. Cytochrome P450: Structure, Mechanism, and Biochemistry. Springer International Publishing; Cham, Switzerland: 2015. pp. 523–785.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Cytochrome%20P450:%20Structure,%20Mechanism,%20and%20Biochemistry&author=F.P.%20Guengerich&publication_year=2015&isbn=978-3-319-12108-6&)

9. Guengerich F.P. A history of the roles of cytochrome P450 enzymes in the toxicity of drugs. Toxicol. Res. 2021;37:1–23. doi: 10.1007/s43188-020-00056-z.  [DOI](https://doi.org/10.1007/s43188-020-00056-z) | [PMC free article](/articles/PMC7431904/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32837681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol.%20Res.&title=A%20history%20of%20the%20roles%20of%20cytochrome%20P450%20enzymes%20in%20the%20toxicity%20of%20drugs&author=F.P.%20Guengerich&volume=37&publication_year=2021&pages=1-23&pmid=32837681&doi=10.1007/s43188-020-00056-z&)

10. Samavat H., Kurzer M.S. Estrogen metabolism and breast cancer. Cancer Lett. 2015;356:231–243. doi: 10.1016/j.canlet.2014.04.018.  [DOI](https://doi.org/10.1016/j.canlet.2014.04.018) | [PMC free article](/articles/PMC4505810/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24784887/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Lett.&title=Estrogen%20metabolism%20and%20breast%20cancer&author=H.%20Samavat&author=M.S.%20Kurzer&volume=356&publication_year=2015&pages=231-243&pmid=24784887&doi=10.1016/j.canlet.2014.04.018&)

11. Artigalás O., Vanni T., Hutz M.H., Ashton-Prolla P., Schwartz I.V. Influence of CYP19A1 polymorphisms on the treatment of breast cancer with aromatase inhibitors: A systematic review and meta-analysis. BMC Med. 2015;13:139. doi: 10.1186/s12916-015-0373-9.  [DOI](https://doi.org/10.1186/s12916-015-0373-9) | [PMC free article](/articles/PMC4475294/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26067721/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med.&title=Influence%20of%20CYP19A1%20polymorphisms%20on%20the%20treatment%20of%20breast%20cancer%20with%20aromatase%20inhibitors:%20A%20systematic%20review%20and%20meta-analysis&author=O.%20Artigal%C3%A1s&author=T.%20Vanni&author=M.H.%20Hutz&author=P.%20Ashton-Prolla&author=I.V.%20Schwartz&volume=13&publication_year=2015&pages=139&pmid=26067721&doi=10.1186/s12916-015-0373-9&)

12. Hlavac V., Kovacova M., Elsnerova K., Brynychova V., Kozevnikovova R., Raus K., Kopeckova K., Mestakova S., Vrana D., Gatek J., et al. Use of germline genetic variability for prediction of chemoresistance and prognosis of breast cancer patients. Cancers. 2018;10:511. doi: 10.3390/cancers10120511.  [DOI](https://doi.org/10.3390/cancers10120511) | [PMC free article](/articles/PMC6316878/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30545124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers&title=Use%20of%20germline%20genetic%20variability%20for%20prediction%20of%20chemoresistance%20and%20prognosis%20of%20breast%20cancer%20patients&author=V.%20Hlavac&author=M.%20Kovacova&author=K.%20Elsnerova&author=V.%20Brynychova&author=R.%20Kozevnikovova&volume=10&publication_year=2018&pages=511&pmid=30545124&doi=10.3390/cancers10120511&)

13. Stark K., Guengerich F.P. Characterization of orphan human cytochromes P450. Drug Metab. Rev. 2007;39:627–637. doi: 10.1080/03602530701467708.  [DOI](https://doi.org/10.1080/03602530701467708) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17786643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Rev.&title=Characterization%20of%20orphan%20human%20cytochromes%20P450&author=K.%20Stark&author=F.P.%20Guengerich&volume=39&publication_year=2007&pages=627-637&pmid=17786643&doi=10.1080/03602530701467708&)

14. Guengerich F.P., Cheng Q. Orphans in the human cytochrome p450 superfamily: Approaches to discovering functions and relevance in pharmacology. Pharmacol. Rev. 2011;63:684–699. doi: 10.1124/pr.110.003525.  [DOI](https://doi.org/10.1124/pr.110.003525) | [PMC free article](/articles/PMC3141877/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21737533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Rev.&title=Orphans%20in%20the%20human%20cytochrome%20p450%20superfamily:%20Approaches%20to%20discovering%20functions%20and%20relevance%20in%20pharmacology&author=F.P.%20Guengerich&author=Q.%20Cheng&volume=63&publication_year=2011&pages=684-699&pmid=21737533&doi=10.1124/pr.110.003525&)

15. Stark K., Dostalek M., Guengerich F.P. Expression and purification of orphan cytochrome P450 4X1 and oxidation of anandamide. FEBS J. 2008;275:3706–3717. doi: 10.1111/j.1742-4658.2008.06518.x.  [DOI](https://doi.org/10.1111/j.1742-4658.2008.06518.x) | [PMC free article](/articles/PMC4765179/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18549450/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20J.&title=Expression%20and%20purification%20of%20orphan%20cytochrome%20P450%204X1%20and%20oxidation%20of%20anandamide&author=K.%20Stark&author=M.%20Dostalek&author=F.P.%20Guengerich&volume=275&publication_year=2008&pages=3706-3717&pmid=18549450&doi=10.1111/j.1742-4658.2008.06518.x&)

16. Kumar S. Computational identification and binding analysis of orphan human cytochrome P450 4X1 enzyme with substrates. BMC Res. Notes. 2015;8:9. doi: 10.1186/s13104-015-0976-4.  [DOI](https://doi.org/10.1186/s13104-015-0976-4) | [PMC free article](/articles/PMC4322450/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25595103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Res.%20Notes&title=Computational%20identification%20and%20binding%20analysis%20of%20orphan%20human%20cytochrome%20P450%204X1%20enzyme%20with%20substrates&author=S.%20Kumar&volume=8&publication_year=2015&pages=9&pmid=25595103&doi=10.1186/s13104-015-0976-4&)

17. GeneCards® Home Page. [(accessed on 1 August 2012)]; Available online: http://www.genecards.org/  [http://www.genecards.org/](http://www.genecards.org/)

18. Murray G., Patimalla S., Stewart K.N., Miller I.D., Heys S.D. Profiling the expression of cytochrome P450 in breast cancer. Histopathology. 2010;57:202–211. doi: 10.1111/j.1365-2559.2010.03606.x.  [DOI](https://doi.org/10.1111/j.1365-2559.2010.03606.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20716162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Histopathology&title=Profiling%20the%20expression%20of%20cytochrome%20P450%20in%20breast%20cancer&author=G.%20Murray&author=S.%20Patimalla&author=K.N.%20Stewart&author=I.D.%20Miller&author=S.D.%20Heys&volume=57&publication_year=2010&pages=202-211&pmid=20716162&doi=10.1111/j.1365-2559.2010.03606.x&)

19. Zhang Y., Yuan Z., Shen R., Jiang Y., Xu W., Gu M., Gu X. Identification of biomarkers predicting the chemotherapeutic outcomes of capecitabine and oxaliplatin in patients with gastric cancer. Oncol. Lett. 2020;20:1. doi: 10.3892/ol.2020.12153.  [DOI](https://doi.org/10.3892/ol.2020.12153) | [PMC free article](/articles/PMC7530885/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33029206/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncol.%20Lett.&title=Identification%20of%20biomarkers%20predicting%20the%20chemotherapeutic%20outcomes%20of%20capecitabine%20and%20oxaliplatin%20in%20patients%20with%20gastric%20cancer&author=Y.%20Zhang&author=Z.%20Yuan&author=R.%20Shen&author=Y.%20Jiang&author=W.%20Xu&volume=20&publication_year=2020&pages=1&pmid=33029206&doi=10.3892/ol.2020.12153&)

20. Poleggi A., Van Der Lee S., Capellari S., Puopolo M., Ladogana A., De Pascali E., Lia D., Formato A., Bartoletti-Stella A., Parchi P., et al. Age at onset of genetic (E200K) and sporadic Creutzfeldt-Jakob diseases is modulated by the CYP4X1 gene. J. Neurol. Neurosurg. Psychiatry. 2018;89:1243–1249. doi: 10.1136/jnnp-2018-318756.  [DOI](https://doi.org/10.1136/jnnp-2018-318756) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30032116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Neurol.%20Neurosurg.%20Psychiatry&title=Age%20at%20onset%20of%20genetic%20(E200K)%20and%20sporadic%20Creutzfeldt-Jakob%20diseases%20is%20modulated%20by%20the%20CYP4X1%20gene&author=A.%20Poleggi&author=S.%20Van%20Der%20Lee&author=S.%20Capellari&author=M.%20Puopolo&author=A.%20Ladogana&volume=89&publication_year=2018&pages=1243-1249&pmid=30032116&doi=10.1136/jnnp-2018-318756&)

21. Guindon J., Hohmann A.G. The endocannabinoid system and cancer: Therapeutic implication. Br. J. Pharmacol. 2011;163:1447–1463. doi: 10.1111/j.1476-5381.2011.01327.x.  [DOI](https://doi.org/10.1111/j.1476-5381.2011.01327.x) | [PMC free article](/articles/PMC3165955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21410463/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Pharmacol.&title=The%20endocannabinoid%20system%20and%20cancer:%20Therapeutic%20implication&author=J.%20Guindon&author=A.G.%20Hohmann&volume=163&publication_year=2011&pages=1447-1463&pmid=21410463&doi=10.1111/j.1476-5381.2011.01327.x&)

22. Chang J., Zhong R., Tian J., Li J., Zhai K., Ke J., Lou J., Chen W., Zhu B., Shen N., et al. Exome-wide analyses identify low-frequency variant in CYP26B1 and additional coding variants associated with esophageal squamous cell carcinoma. Nat. Genet. 2018;50:338–343. doi: 10.1038/s41588-018-0045-8.  [DOI](https://doi.org/10.1038/s41588-018-0045-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29379198/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Exome-wide%20analyses%20identify%20low-frequency%20variant%20in%20CYP26B1%20and%20additional%20coding%20variants%20associated%20with%20esophageal%20squamous%20cell%20carcinoma&author=J.%20Chang&author=R.%20Zhong&author=J.%20Tian&author=J.%20Li&author=K.%20Zhai&volume=50&publication_year=2018&pages=338-343&pmid=29379198&doi=10.1038/s41588-018-0045-8&)

23. Trubicka J., Grabowska-Kłujszo E., Suchy J., Masojć B., Serrano-Fernández P., Kurzawski G., Cybulski C., Górski B., Huzarski T., Byrski T., et al. Variant alleles of the CYP1B1 gene are associated with colorectal cancer susceptibility. BMC Cancer. 2010;10:420. doi: 10.1186/1471-2407-10-420.  [DOI](https://doi.org/10.1186/1471-2407-10-420) | [PMC free article](/articles/PMC2929240/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20701755/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Variant%20alleles%20of%20the%20CYP1B1%20gene%20are%20associated%20with%20colorectal%20cancer%20susceptibility&author=J.%20Trubicka&author=E.%20Grabowska-K%C5%82ujszo&author=J.%20Suchy&author=B.%20Masoj%C4%87&author=P.%20Serrano-Fern%C3%A1ndez&volume=10&publication_year=2010&pages=420&pmid=20701755&doi=10.1186/1471-2407-10-420&)

24. Kamiza A.B., You J.-F., Wang W.-C., Tang R., Chang C.-Y., Chien H.-T., Lai C.-H., Chiu L.-L., Lo T.-P., Hung K.-Y., et al. Polymorphisms of xenobiotic-metabolizing genes and colorectal cancer risk in patients with lynch syndrome: A retrospective cohort study in Taiwan. Environ. Mol. Mutagen. 2018;59:69–78. doi: 10.1002/em.22113.  [DOI](https://doi.org/10.1002/em.22113) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28714190/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ.%20Mol.%20Mutagen.&title=Polymorphisms%20of%20xenobiotic-metabolizing%20genes%20and%20colorectal%20cancer%20risk%20in%20patients%20with%20lynch%20syndrome:%20A%20retrospective%20cohort%20study%20in%20Taiwan&author=A.B.%20Kamiza&author=J.-F.%20You&author=W.-C.%20Wang&author=R.%20Tang&author=C.-Y.%20Chang&volume=59&publication_year=2018&pages=69-78&pmid=28714190&doi=10.1002/em.22113&)

25. Reding K.W., Weiss N.S., Chen C., Li C.I., Carlson C.S., Wilkerson H.-W., Farin F.M., Thummel K.E., Daling J.R., Malone K.E. Genetic polymorphisms in the catechol estrogen metabolism pathway and breast cancer risk. Cancer Epidemiol. Biomark. Prev. 2009;18:1461–1467. doi: 10.1158/1055-9965.EPI-08-0917.  [DOI](https://doi.org/10.1158/1055-9965.EPI-08-0917) | [PMC free article](/articles/PMC2764317/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19383894/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol.%20Biomark.%20Prev.&title=Genetic%20polymorphisms%20in%20the%20catechol%20estrogen%20metabolism%20pathway%20and%20breast%20cancer%20risk&author=K.W.%20Reding&author=N.S.%20Weiss&author=C.%20Chen&author=C.I.%20Li&author=C.S.%20Carlson&volume=18&publication_year=2009&pages=1461-1467&pmid=19383894&doi=10.1158/1055-9965.EPI-08-0917&)

26. Wang Z., Li M., Li L., Sun H., Lin X. Association of single nucleotide polymorphisms in the CYP1B1 gene with the risk of primary open-angle glaucoma: A meta-analysis. Genet. Mol. Res. 2015;14:17262–17272. doi: 10.4238/2015.December.16.26.  [DOI](https://doi.org/10.4238/2015.December.16.26) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26681220/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Mol.%20Res.&title=Association%20of%20single%20nucleotide%20polymorphisms%20in%20the%20CYP1B1%20gene%20with%20the%20risk%20of%20primary%20open-angle%20glaucoma:%20A%20meta-analysis&author=Z.%20Wang&author=M.%20Li&author=L.%20Li&author=H.%20Sun&author=X.%20Lin&volume=14&publication_year=2015&pages=17262-17272&pmid=26681220&doi=10.4238/2015.December.16.26&)

27. Shimada T., Hayes C.L., Yamazaki H., Amin S., Hecht S.S., Guengerich F.P., Sutter T.R. Activation of chemically diverse procarcinogens by human cytochrome P-450 1B1. Cancer Res. 1996;56:2979–2984.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/8674051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=Activation%20of%20chemically%20diverse%20procarcinogens%20by%20human%20cytochrome%20P-450%201B1&author=T.%20Shimada&author=C.L.%20Hayes&author=H.%20Yamazaki&author=S.%20Amin&author=S.S.%20Hecht&volume=56&publication_year=1996&pages=2979-2984&pmid=8674051&)

28. Watkins G., Douglas-Jones A., Mansel R.E., Jiang W.G. Expression of thromboxane synthase, TBXAS1 and the thromboxane A2 receptor, TBXA2R, in human breast cancer. Int. Semin. Surg. Oncol. 2005;2:23. doi: 10.1186/1477-7800-2-23.  [DOI](https://doi.org/10.1186/1477-7800-2-23) | [PMC free article](/articles/PMC1289289/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16250911/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20Semin.%20Surg.%20Oncol.&title=Expression%20of%20thromboxane%20synthase,%20TBXAS1%20and%20the%20thromboxane%20A2%20receptor,%20TBXA2R,%20in%20human%20breast%20cancer&author=G.%20Watkins&author=A.%20Douglas-Jones&author=R.E.%20Mansel&author=W.G.%20Jiang&volume=2&publication_year=2005&pages=23&pmid=16250911&doi=10.1186/1477-7800-2-23&)

29. Wang S., Jin F., Fan W., Liu F., Zou Y., Hu X., Xu H., Han P. Gene expression meta-analysis in diffuse low-grade glioma and the corresponding histological subtypes. Sci. Rep. 2017;7:11741. doi: 10.1038/s41598-017-12087-y.  [DOI](https://doi.org/10.1038/s41598-017-12087-y) | [PMC free article](/articles/PMC5603565/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28924174/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Gene%20expression%20meta-analysis%20in%20diffuse%20low-grade%20glioma%20and%20the%20corresponding%20histological%20subtypes&author=S.%20Wang&author=F.%20Jin&author=W.%20Fan&author=F.%20Liu&author=Y.%20Zou&volume=7&publication_year=2017&pages=11741&pmid=28924174&doi=10.1038/s41598-017-12087-y&)

30. Liu X., Wang G., Hong X., Wang D., Tsai H.-J., Zhang S., Arguelles L., Kumar R., Wang H., Liu R., et al. Gene-vitamin D interactions on food sensitization: A prospective birth cohort study. Allergy. 2011;66:1442–1448. doi: 10.1111/j.1398-9995.2011.02681.x.  [DOI](https://doi.org/10.1111/j.1398-9995.2011.02681.x) | [PMC free article](/articles/PMC3189275/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21819409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=Gene-vitamin%20D%20interactions%20on%20food%20sensitization:%20A%20prospective%20birth%20cohort%20study&author=X.%20Liu&author=G.%20Wang&author=X.%20Hong&author=D.%20Wang&author=H.-J.%20Tsai&volume=66&publication_year=2011&pages=1442-1448&pmid=21819409&doi=10.1111/j.1398-9995.2011.02681.x&)

31. Fuhrman B.J., Freedman D.M., Bhatti P., Doody M.M., Fu Y.P., Chang S.C., Linet M.S., Sigurdson A.J. Sunlight, Poly-morphisms of vitamin d-related genes and risk of breast cancer. Anticancer. Res. 2013;33:543–551.  [PMC free article](/articles/PMC3866631/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23393347/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer.%20Res.&title=Sunlight,%20Poly-morphisms%20of%20vitamin%20d-related%20genes%20and%20risk%20of%20breast%20cancer&author=B.J.%20Fuhrman&author=D.M.%20Freedman&author=P.%20Bhatti&author=M.M.%20Doody&author=Y.P.%20Fu&volume=33&publication_year=2013&pages=543-551&pmid=23393347&)

32. Yang W., Ma F., Wang L., He X., Zhang H., Zheng J., Wang Y., Jin T., Yuan D., He Y. The association analysis between CYP24A1 genetic polymorphisms and the risk of ischemic stroke in Chinese Han population. Brain Behav. 2019;10:e01503. doi: 10.1002/brb3.1503.  [DOI](https://doi.org/10.1002/brb3.1503) | [PMC free article](/articles/PMC7010572/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31872978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Behav.&title=The%20association%20analysis%20between%20CYP24A1%20genetic%20polymorphisms%20and%20the%20risk%20of%20ischemic%20stroke%20in%20Chinese%20Han%20population&author=W.%20Yang&author=F.%20Ma&author=L.%20Wang&author=X.%20He&author=H.%20Zhang&volume=10&publication_year=2019&pages=e01503&pmid=31872978&doi=10.1002/brb3.1503&)

33. Bao Q., Wang D., Zhang Y., Bao L., Jia H. The impact of CYP24A1 polymorphisms on hypertension susceptibility. Kidney Blood Press. Res. 2020;45:28–37. doi: 10.1159/000503925.  [DOI](https://doi.org/10.1159/000503925) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31918428/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Kidney%20Blood%20Press.%20Res.&title=The%20impact%20of%20CYP24A1%20polymorphisms%20on%20hypertension%20susceptibility&author=Q.%20Bao&author=D.%20Wang&author=Y.%20Zhang&author=L.%20Bao&author=H.%20Jia&volume=45&publication_year=2020&pages=28-37&pmid=31918428&doi=10.1159/000503925&)

34. Hlaváč V., Brynychová V., Václavíková R., Ehrlichová M., Vrána D., Pecha V., Koževnikovová R., Trnková M., Gatěk J., Kopperová D., et al. The expression profile of ATP-binding cassette transporter genes in breast carcinoma. Pharmacogenomics. 2013;14:515–529. doi: 10.2217/pgs.13.26.  [DOI](https://doi.org/10.2217/pgs.13.26) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23556449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=The%20expression%20profile%20of%20ATP-binding%20cassette%20transporter%20genes%20in%20breast%20carcinoma&author=V.%20Hlav%C3%A1%C4%8D&author=V.%20Brynychov%C3%A1&author=R.%20V%C3%A1clav%C3%ADkov%C3%A1&author=M.%20Ehrlichov%C3%A1&author=D.%20Vr%C3%A1na&volume=14&publication_year=2013&pages=515-529&pmid=23556449&doi=10.2217/pgs.13.26&)

35. Schwartz L.H., Litière S., De Vries E., Ford R., Gwyther S., Mandrekar S., Shankar L., Bogaerts J., Chen A., Dancey J., et al. RECIST 1.1—Update and clarification: From the RECIST committee. Eur. J. Cancer. 2016;62:132–137. doi: 10.1016/j.ejca.2016.03.081.  [DOI](https://doi.org/10.1016/j.ejca.2016.03.081) | [PMC free article](/articles/PMC5737828/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27189322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Cancer&title=RECIST%201.1%E2%80%94Update%20and%20clarification:%20From%20the%20RECIST%20committee&author=L.H.%20Schwartz&author=S.%20Liti%C3%A8re&author=E.%20De%20Vries&author=R.%20Ford&author=S.%20Gwyther&volume=62&publication_year=2016&pages=132-137&pmid=27189322&doi=10.1016/j.ejca.2016.03.081&)

36. Li H. Toward better understanding of artifacts in variant calling from high-coverage samples. Bioinformatics. 2014;30:2843–2851. doi: 10.1093/bioinformatics/btu356.  [DOI](https://doi.org/10.1093/bioinformatics/btu356) | [PMC free article](/articles/PMC4271055/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24974202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Toward%20better%20understanding%20of%20artifacts%20in%20variant%20calling%20from%20high-coverage%20samples&author=H.%20Li&volume=30&publication_year=2014&pages=2843-2851&pmid=24974202&doi=10.1093/bioinformatics/btu356&)

37. Van Der Auwera G.A., Carneiro M.O., Hartl C., Poplin R., Del Angel G., Levy-Moonshine A., Jordan T., Shakir K., Roazen D., Thibault J., et al. From FastQ data to high-confidence variant calls: The genome analysis toolkit best practices pipeline. Curr. Protoc. Bioinform. 2013;43:11.10.1–11.10.33. doi: 10.1002/0471250953.bi1110s43.  [DOI](https://doi.org/10.1002/0471250953.bi1110s43) | [PMC free article](/articles/PMC4243306/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25431634/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.%20Bioinform.&title=From%20FastQ%20data%20to%20high-confidence%20variant%20calls:%20The%20genome%20analysis%20toolkit%20best%20practices%20pipeline&author=G.A.%20Van%20Der%20Auwera&author=M.O.%20Carneiro&author=C.%20Hartl&author=R.%20Poplin&author=G.%20Del%20Angel&volume=43&publication_year=2013&pages=11.10.1-11.10.33&pmid=25431634&doi=10.1002/0471250953.bi1110s43&)

38. Wang K., Li M., Hakonarson H. Annovar: Functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res. 2010;38:e164. doi: 10.1093/nar/gkq603.  [DOI](https://doi.org/10.1093/nar/gkq603) | [PMC free article](/articles/PMC2938201/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20601685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Annovar:%20Functional%20annotation%20of%20genetic%20variants%20from%20high-throughput%20sequencing%20data&author=K.%20Wang&author=M.%20Li&author=H.%20Hakonarson&volume=38&publication_year=2010&pages=e164&pmid=20601685&doi=10.1093/nar/gkq603&)

39. Bustin S.A., Benes V., Garson J.A., Hellemans J., Huggett J., Kubista M., Mueller R., Nolan T., Pfaffl M.W., Shipley G.L., et al. The MIQE guidelines: Minimum information for publication of quantitative real-time PCR experiments. Clin. Chem. 2009;55:611–622. doi: 10.1373/clinchem.2008.112797.  [DOI](https://doi.org/10.1373/clinchem.2008.112797) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19246619/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Chem.&title=The%20MIQE%20guidelines:%20Minimum%20information%20for%20publication%20of%20quantitative%20real-time%20PCR%20experiments&author=S.A.%20Bustin&author=V.%20Benes&author=J.A.%20Garson&author=J.%20Hellemans&author=J.%20Huggett&volume=55&publication_year=2009&pages=611-622&pmid=19246619&doi=10.1373/clinchem.2008.112797&)

40. Benjamini Y., Hochberg Y. Controlling the false discovery rate—A practical and powerful approach to multiple testing. J. R. Stat. Soc. Ser. B Methodol. 1995;57:289–300. doi: 10.1111/j.2517-6161.1995.tb02031.x.  [DOI](https://doi.org/10.1111/j.2517-6161.1995.tb02031.x) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20R.%20Stat.%20Soc.%20Ser.%20B%20Methodol.&title=Controlling%20the%20false%20discovery%20rate%E2%80%94A%20practical%20and%20powerful%20approach%20to%20multiple%20testing&author=Y.%20Benjamini&author=Y.%20Hochberg&volume=57&publication_year=1995&pages=289-300&doi=10.1111/j.2517-6161.1995.tb02031.x&)
