# Class II HLA Variants Associate with Risk of Pegaspargase Hypersensitivity

## Metadata
**Authors:** Yiwei Liu, Wenjian Yang, Colton Smith, Cheng Cheng, Seth E Karol, Eric C Larsen, Naomi Winick, William L Carroll, Mignon L Loh, Elizabeth A Raetz, Stephen P Hunger, Stuart S Winter, Kimberly P Dunsmore, Meenakshi Devidas, Jun J Yang, William E Evans, Sima Jeha, Ching-Hon Pui, Hiroto Inaba, Mary V Relling
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2021 Apr 21
**DOI:** [10.1002/cpt.2241](https://doi.org/10.1002/cpt.2241)
**PMID:** 33768542
**PMCID:** PMC8790808
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8790808/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8790808/pdf/nihms-1765892.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8790808/pdf/nihms-1765892.pdf)

## Abstract

We conducted the first HLA allele and genome wide association study to identify loci associated with hypersensitivity reactions exclusively to the PEGylated preparation of asparaginase (pegaspargase) in racially diverse cohorts of pediatric leukemia patients: St. Jude Children’s Research Hospital’s Total XVI (TXVI, n = 598), Children’s Oncology Group AALL0232 (n = 2472) and AALL0434 (n = 1189). Germline DNA was genotyped using arrays. Genetic variants not genotyped directly were imputed. HLA alleles were imputed using SNP2HLA or inferred using BWAkit. Analyses between genetic variants and hypersensitivity were performed in each cohort first using cohort-specific covariates and then combined using meta-analyses. Nongenetic risk factors included fewer intrathecal injections (P = 2.7×10−5 in TXVI) and male sex (P = 0.025 in AALL0232). HLA alleles DQB1*02:02, DRB1*07:01, and DQA1*02:01 had the strongest associations with pegaspargase hypersensitivity (P < 5.0×10−5) in patients with primarily European ancestry (EA), with the three alleles associating in a single haplotype. The top allele HLA-DQB1*02:02 was tagged by HLA-DQB1 rs1694129 in EAs (r2 = 0.96) and less so in non-EAs. All single nucleotide polymorphisms associated with pegaspargase hypersensitivity reaching genome-wide significance in EAs were in class II HLA loci, and were partially replicated in non-EAs, as is true for other HLA associations. The rs9958628 variant, in ARHGAP28 (previously linked to immune response in children) had the strongest genetic association (P = 8.9×10−9) in non-EAs. The HLA-DQB1*02:02-DRB1*07:01-DQA1*02:01 associated with hypersensitivity reactions to pegaspargase is the same haplotype associated with reactions to non-PEGylated asparaginase, even though the antigens differ between the two preparations.

Keywords: Acute Lymphoblastic Leukemia, Asparaginase, Human Leukocyte Antigen, Hypersensitivity, Pharmacogenomics

## Introduction

Asparaginase is a crucial treatment component for pediatric acute lymphoblastic leukemia (ALL) and lymphoma. Hypersensitivity reaction is the primary adverse effect of both native and PEGylated *E. coli* asparaginase (ASNase and pegaspargase). Although IgG antibodies against both preparations of asparaginase have been detected and associated with reactions, we showed that antibodies against pegaspargase predominantly targeted polyethylene glycol (PEG), whereas antibodies against ASNase primarily targeted the asparaginase protein, indicating differences in the mechanisms of their immunogenicity.([1](#R1)) Efforts have been made to understand the mechanisms of these reactions and to develop effective preventative treatment based on genetic risk factors such as an *NFATC2* variant previously identified.([2](#R2)) Mutations have been introduced to *E. coli* asparaginase protein to make it less immunogenic.([3](#R3), [4](#R4)) Previously, we and others identified variations in several *HLA* loci that were associated with asparaginase hypersensitivity in children of predominantly European ancestry treated for ALL.([5](#R5)–[8](#R8)) Nevertheless, our previous study used combined cohorts consisting of patients treated with either ASNase or pegaspargase, while two other studies investigated hypersensitivity only to native ASNase. Because ASNase is no longer used in frontline therapy in major cooperative group trials in North America and the European Union, understanding the risk of hypersensitivity to pegaspargase in a cohort without ASNase treatment is needed. Here we present a genome-wide and HLA-focused analysis of genomic risk factors for hypersensitivity reactions exclusively to pegaspargase using the largest cohort of pediatric ALL and include results from multiple ancestries.

## Methods

### Patients and phenotype

Three cohorts of ALL patients, all using pegaspargase upfront, were included in this study. One cohort was treated with St. Jude Children’s Research Hospital’s Total XVI (TXVI) protocol (n = 598, B- and T-lineage ALL). The other two were treated with Children’s Oncology Group (COG) protocols: AALL0232 (n = 2472, B-ALL only) and AALL0434 (n = 1189, T-ALL only). Hypersensitivity reaction data were retrieved from research databases with reactions documented as part of the prospective toxicity reporting of each protocol (as of June 2018 for TXVI and December 2018 for COG protocols). Treatment details have been described.([1](#R1), [9](#R9), [10](#R10)) Reactions were graded according to Common Terminology Criteria for Adverse Events (CTCAE) v3 (allergic reaction/hypersensitivity) in TXVI and CTCAE v4 (allergic reaction/anaphylaxis) in AALL0232 and AALL0434. Grade 1 reactions were defined by transient flushing/rash, drug fever < 38C° with no intervention indicated and were not requested to be reported in these trials. Therefore, in this study, patients with pegaspargase hypersensitivity reactions of grade 2 or higher were treated as reaction positive; the rest were treated as reaction negative. Associations between patient- and treatment- related variables and pegaspargase reactions were analyzed with general logistic regression. Treatment-related variables analyzed for their association with pegaspargase hypersensitivity reactions included: number of intrathecal injections during remission induction (TXVI), number of pegaspargase doses during remission induction (TXVI, or as receiving extended induction or not in AALL0232), glucocorticoid (dexamethasone vs prednisone) used in remission induction (AALL0232), number of delayed intensifications (AALL0232), methotrexate arm (AALL0232 and AALL0434), and receiving nelarabine or not (AALL0434). Informed consent from the parents or guardians and assent from the patients, as appropriate, were obtained with oversight by the local Institutional Review Boards.

### Genotyping

Germline DNA collected after remission induction therapy was genotyped using the Illumina Exome-24 BeadChip (596 and 2056 patients on TXVI and AALL0232 respectively), the Affymetrix Genome-Wide Human SNP Array 6.0 (596 patients on TXVI and 2275 patients on AALL0232), or the Illumina Infinium Omni2.5-exome BeadChip (1026 patients on AALL0434). Illumina Infinium Omni2.5-exome array included most the content of Illumina Exome-24 BeadChip, and the single nucleotide polymorphisms (SNPs) in the HLA region that were common to both arrays were used to infer HLA alleles. Whole exome sequencing (WES) was performed only on TXVI germline DNA samples (n = 595). iAdmix ([11](#R11)) was used to infer genetic ancestry percentage for each patient using 1000genome data ([12](#R12)) as reference. Percent ancestry was used to assign patients to ancestry groups: patients having ≥ 90% European ancestry (EA, reference population: GBR/CEU/FIN/TSI/IBS); patients having ≥ 70% African ancestry (AA, reference population: YRI/LWK/GWD/MSL/ESN/ASW/ACB); patients having ≥ 10% Native American ancestry (AMR, reference population: PEL/CLM/PUR/MXL) and Native American ancestry being the greatest non-EA ancestry; patients having ≥ 90% East Asian (EAS, reference population: CHB/JPT/CDX/CHS/KHV) or South Asian (SAS, reference population: BEB/GIH/PJL/ITU/STU) ancestry (AS), and others whose ancestry was outside the above boundaries (Other). The principal component analysis (PCA) plot with 1000genome samples is in [Figure S1](#SD1), and the comparison between ancestry group and self-reported race/ethnicity group is in [Table S1](#SD1). We also estimated the percentage of Native American ancestry for TXVI patients using an independent cohort of Native American population with Affymetrix SNPCHIP available as reference,([13](#R13)) and the estimated percentage Native American ancestries were highly consistent (r^2^ > 0.99) with what was obtained using 1000genome as the reference population. Genetic variants not genotyped directly were imputed using the Michigan Imputation Server. SNPs were included in analyses with imputation r^2^ > 0.6 and Hardy-Weinberg disequilibrium P value (within EA patients) > 0.0001 across all cohorts.

### Data analysis

Genome-wide association studies (GWAS) were performed on all patients and within different ancestry groups (EAs, non-EAs, AAs and AMRs) for each treatment protocol adjusting for protocol-specific covariates and ancestry. Meta-analyses were performed combining results from the same ancestry group across protocols ([Figure S2](#SD1)). Additional trans-ancestry meta-analysis was performed to combine the results across multiple ancestry groups. All GWAS were performed using PLINK 2.00 alpha.([14](#R14)) All meta-analyses were performed using generic inverse variance method in PLINK 1.9 ([Figure S2](#SD1)).([15](#R15)) Pooled odds ratios (ORs) were calculated using random effects models. An OR > 1 indicates that the presence of the HLA allele, or the amino acid variant, or the minor allele of the SNP, was associated with increased risk of hypersensitivity. In all GWAS except for ones in EAs only, percentage ancestries were treated as continuous variable covariates.

HLA alleles were imputed for all patients using SNP2HLA v1.0.2.([16](#R16)) EA, AA, AMR, and Other patients were imputed using T1DGC as the reference panel. AS patients were imputed using the Pan-Asian reference panel.([17](#R17)) The association between SNP2HLA-imputed HLA alleles and pegaspargase hypersensitivity reactions was analyzed in EAs, AAs, AMRs, and non-EAs within each protocol using protocol-specific covariates using PLINK 2.00 alpha. For TXVI patients, HLA alleles were also inferred from WES data using BWAkit 0.7.11,([18](#R18)) and inferred HLA alleles were compared to those obtained by SNP2HLA ([Table S2](#SD1)). The BWAkit results were not combined with SNP2HLA results in any analyses. Associations between BWAkit-inferred HLA alleles and hypersensitivity on TXVI were analyzed adjusting for the number of intrathecal injections during induction;([1](#R1)) associations in non-EAs treated on TXVI also adjusted for ancestry percentage.

We used the following for the alpha level for significance for association analyses: for HLA alleles, we used 5.2×10^−4^ (0.05/97 [number of imputed HLA alleles]); for HLA amino acids, we used 4.8×10^−5^ (0.05/1041 [number of imputed HLA amino acid variants]), and for GWAS of SNPs, we used 5×10^−8^.

Locus zoom plots were made with [https://locuszoom.org/](https://locuszoom.org/).([19](#R19)) Other statistical analyses and graphics were generated using R version 3.6.

## Results

### Patient- and treatment-related risk factors for pegaspargase hypersensitivity reactions

Of the evaluable patients ([Table 1](#T1)) in TXVI, AALL0232, and AALL0434, 13.7% (82 of 598), 15.2% (375 of 2472) and 8.2% (98 of 1189) patients had reactions to pegaspargase, respectively. As we previously reported, a higher number of intrathecal injections during remission induction was the only clinical covariate associated with lower risk of reactions to pegaspargase in TXVI (*P* = 6.4×10^−5^ OR 0.73, [Table S3A](#SD1)).([1](#R1)) Male patients were at higher risk for reactions in AALL0232 compared with females (*P* = 0.025 OR 1.24, [Table S3B](#SD1)). No patient- or treatment-related variable was associated with reactions in AALL0434 ([Table S3C](#SD1)). Risk for reactions did not differ by ancestry in all three protocols ([Table S3](#SD1)).

### Table 1.

| Cohort | EA (%) | AA (%) | AMR (%) | AS (%) | Other (%) |
| --- | --- | --- | --- | --- | --- |
| TXVI | 402 (70.5) | 81 (13.6) | 63 (10.6) | 8 (1.3) | 42 (7.0) |
| AALL0232 | 1295 (56.9) | 115 (5.1) | 577 (25.4) | 67 (2.9) | 221 (9.7) |
| AALL0434 | 593 (57.8) | 114 (11.1) | 158 (15.4) | 37 (3.6) | 124 (12.1) |

Table 1 Caption: Number of evaluable patients in different ancestry groups in protocols TXVI (n = 596), AALL0232 (n = 2275), and AALL0434 (n = 1026) assigned using iAdmix inferred ancestry percentage. Numbers in parentheses represent the percentage of an ancestry group in a cohort.

### Imputed HLA alleles in EA patients

HLA alleles are known to be associated with hypersensitivity reactions to asparaginase and other drugs;([5](#R5)–[7](#R7), [20](#R20)) we successfully imputed HLA alleles in EA patients (for whom adequate reference data exist), and analyzed associations between HLA alleles, amino acid variants in HLA genes, and reactions to pegaspargase among EAs including the protocol-specific covariates (number of intrathecal injections during induction for TXVI, sex for AALL0232, none for AALL0434). In a meta-analysis combining results from the three cohorts, among 97 HLA alleles, *HLA-DQB1*02:02*, *HLA-DRB1*07:01*, *HLA-DQA1*02:01, and HLA-DRB1*04:02* had the strongest association with reactions to pegaspargase (*P*_meta_ = 3.8×10^−9^ OR 2.13, *P*_meta_ = 2.2×10^−5^ OR 1.96, *P*_meta_ = 2.7×10^−5^ OR 1.96, and *P*_meta_ = 6.3×10^−5^ OR 4.61 respectively, [Table 2](#T2), [Figure 1A](#F1)); among 1041 amino acid variants, glycine at position 135 of *HLA-DQB1*, a marker for the *HLA-DQB1*02:02* allele, had the strongest association with reactions (*P*_meta_ = 3.8×10^−9^ OR 2.13, [Table S4](#SD1)). Phasing data acquired through Beagle showed that the top three alleles represent an extended haplotype in EAs (r^2^ = 1.0 between *HLA-DRB1*07:01* and *HLA-DQA1*02:01*, r^2^ = 0.73 between *HLA-DQB1*02:02* and either *HLA-DRB1*07:01* or *HLA-DQA1*02:01*, [Table S5](#SD1)). All but one patient with *HLA-DQB1*02:02* also had the extended haplotype *HLA-DRB1*07:01-DQA1*02:01-DQB1*02:02*. In multivariable analyses, none of the alleles/haplotypes remained significant after adjusting for another allele due to the extended linkage disequilibrium (LD) (*P* > 0.07, [Table S6](#SD1)). The top DRB1 amino acids do not mark *HLA-DRB1*07:01* and are not shared by the top *HLA-DRB1* allele hits (*DRB1*07:01, *04:02, and *04:05*).

### Table 2.

| 4-digit HLA allele | Meta-analysis |  |  |  |  | TXVI |  |  | AALL0232 |  |  | AALL0434 |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| P | ORa | I2 | P.het | freq | P | ORa | freq | P | ORa | freq | P | ORa | freq |  |
| HLA-DQB1*02:02 | 3.76E-9 | 2.14 | 0.00 | 0.695 | 0.100 | 1.44E-3 | 2.38 | 0.1 | 9.29E-5 | 1.94 | 0.095 | 1.53E-3 | 2.47 | 0.111 |
| HLA-DRB1*07:01 | 2.22E-5 | 1.99 | 0.37 | 0.202 | 0.132 | 3.28E-5 | 2.91 | 0.133 | 3.27E-4 | 1.74 | 0.123 | 5.78E-2 | 1.70 | 0.149 |
| HLA-DQA1*02:01 | 2.74E-5 | 1.99 | 0.39 | 0.196 | 0.132 | 3.28E-5 | 2.91 | 0.133 | 3.93E-4 | 1.73 | 0.123 | 5.20E-2 | 1.72 | 0.148 |
| HLA-DRB1*04:02 | 6.30E-5 | 4.49 | 0.00 | 0.819 | 0.008 | 4.97E-1 | 2.22 | 0.005 | 3.02E-3 | 4.74 | 0.006 | 7.48E-3 | 5.01 | 0.013 |
| HLA-DRB1*04:05 | 5.44E-4 | 3.75 | 0.00 | 0.711 | 0.007 | 1.78E-1 | 3.39 | 0.006 | 1.60E-2 | 3.18 | 0.009 | 2.50E-2 | 7.22 | 0.004 |
| HLA-B*50:01 | 5.06E-3 | 3.13 | 0.00 | 0.636 | 0.008 | 2.86E-1 | 2.53 | 0.009 | 6.08E-3 | 4.07 | 0.007 | 7.55E-1 | 1.39 | 0.009 |
| HLA-DQA1*01:03 | 5.82E-3 | 0.54 | 0.00 | 0.852 | 0.065 | 9.16E-2 | 0.41 | 0.066 | 4.20E-2 | 0.57 | 0.066 | 3.31E-1 | 0.56 | 0.063 |
| HLA-DQB1*06:02 | 6.74E-3 | 0.66 | 0.00 | 0.708 | 0.125 | 5.07E-1 | 0.81 | 0.122 | 1.84E-2 | 0.63 | 0.124 | 1.54E-1 | 0.54 | 0.128 |
| HLA-A*02:05 | 1.18E-2 | 3.03 | 0.00 | 0.967 | 0.006 | 3.54E-1 | 3.08 | 0.006 | 2.63E-2 | 3.20 | 0.007 | 4.37E-1 | 2.34 | 0.006 |
| HLA-B*38:01 | 1.79E-2 | 1.9 | 0.00 | 0.952 | 0.020 | 3.69E-1 | 1.86 | 0.015 | 7.10E-2 | 1.82 | 0.022 | 2.01E-1 | 2.28 | 0.019 |
| HLA-DQA1*01:01 | 1.81E-2 | 0.71 | 0.02 | 0.360 | 0.147 | 6.16E-2 | 0.54 | 0.144 | 3.71E-2 | 0.69 | 0.141 | 9.71E-1 | 1.01 | 0.161 |
| HLA-DPA1*02:02 | 3.14E-2 | 1.59 | 0.00 | 0.658 | 0.033 | 9.96E-1 | 1.00 | 0.032 | 2.62E-2 | 1.75 | 0.038 | 4.69E-1 | 1.58 | 0.025 |
| HLA-DRB1*13:01 | 3.17E-2 | 0.61 | 0.00 | 0.629 | 0.059 | 1.43E-1 | 0.46 | 0.058 | 1.88E-1 | 0.70 | 0.059 | 1.97E-1 | 0.39 | 0.058 |
| HLA-DRB1*15:01 | 4.05E-2 | 0.71 | 0.14 | 0.313 | 0.132 | 9.49E-1 | 1.02 | 0.134 | 1.50E-2 | 0.63 | 0.134 | 1.57E-1 | 0.55 | 0.126 |
| HLA-DQB1*06:03 | 4.62E-2 | 0.66 | 0.00 | 0.649 | 0.064 | 6.43E-1 | 0.84 | 0.072 | 7.96E-2 | 0.62 | 0.065 | 2.13E-1 | 0.40 | 0.057 |
| HLA-C*07:02 | 4.87E-2 | 0.78 | 0.00 | 0.829 | 0.160 | 6.62E-1 | 0.89 | 0.158 | 8.06E-2 | 0.75 | 0.157 | 3.14E-1 | 0.71 | 0.166 |

Table 2 Caption: Four-digit HLA alleles significantly (P < 0.05) associated with hypersensitivity reactions to pegaspargase in EAs (n = 2168) in meta-analysis. Association analysis was adjusted for induction intrathecal therapy in TXVI, and for sex in AALL0232. I2 indicates the heterogenicity of the effect of an allele across the three cohorts. P.het indicates the statistical significance of I2.

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ee/8790808/0c65eee360c0/nihms-1765892-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8790808_nihms-1765892-f0001.jpg)

A) Association between imputed HLA alleles and hypersensitivity reactions to pegaspargase in EA patients (n = 2168): meta-analyses from individual analyses on TXVI (adjusting for number of intrathecal treatment doses during remission induction), AALL0232 (adjusting for sex), and AALL0434. P = 5.2×10−4 was used as HLA allele-wide cut-off for statistical significance based on the total number of alleles in meta-analysis result (n = 97). B) Genome-wide association between SNP genotypes and hypersensitivity reaction to pegaspargase in all patients (all ancestries, n = 3897): meta-analysis from individual analyses on TXVI (adjusting for number of intrathecal treatment doses during remission induction, and ancestry percentage), AALL0232 (adjusting for sex and ancestry percentage), and AALL0434 (adjusting for ancestry percentage). C) Genome-wide association between SNP genotypes and hypersensitivity reaction to pegaspargase in EAs (n = 2168): meta-analysis from individual analyses on TXVI (adjusting for number of intrathecal treatment doses during remission induction), AALL0232 (adjusting for sex), and AALL0434. D) Genome-wide association between SNP genotypes and hypersensitivity reactions to pegaspargase in non-EAs (n = 1607): meta-analysis from individual analyses on TXVI (adjusting for number of intrathecal treatment doses during remission induction, and ancestry percentage), AALL0232 (adjusting for sex and ancestry percentage), and AALL0434 (adjusting for ancestry percentage).

### HLA alleles in non-EA patients

WES data available for TXVI patients was used for direct assignment of HLA alleles (*HLA-DRB1*07:01*, *HLA-DQA1*02:01* and *HLA-DQB1*02:02*) in this cohort, including 194 non-EA patients. Among the three top predictive HLA alleles in EAs, none had significant associations with reactions in TXVI non-EA patients after adjusting for ancestry percentage and number of intrathecal injections (*P* = 0.42, 0.36, and 0.96 for *HLA-DRB1*07:01, -DQA1*02:01*, and *-DQB1*02:02*). Within TXVI, inferred *HLA* alleles with *P* < 0.05 among non-EAs, with corresponding P values in EAs, are shown in [Table S7](#SD1): none of the associations for HLA alleles in non-EAs were replicated in EAs.

In order to examine HLA alleles in all ancestry groups in all protocols, not just TXVI with its WES data, we imputed HLA alleles in the combined cohort of 3775 patients with Illumina chip array genotype from all three protocols ([Table S8](#SD1)). Because SNP2HLA imputation using the EA T1DGC reference panel has been shown to have > 90% accuracy for class II HLA genes in an AA population,([21](#R21)) we also imputed HLA alleles in non-EA patients (AA, AMR, and Other) using SNP2HLA, although SNP2HLA may not have performed well for estimating HLA alleles in non-EA patients ([Table S2](#SD1)). None of the imputed alleles reached the significance threshold after multiple testing adjustment, but *HLA-DRB1*07:01*, *HLA-DQA1*02:01* and *HLA-DQB1*02:02* reached nominal significance in non-EA patients (*P*_meta_ = 0.011 OR 1.94, *P*_meta_ = 0.024 OR 2.09, and *P*_meta_ = 0.041 OR 1.75 respectively, [Table S8](#SD1)).

### GWAS of SNPs in patients of all ancestries

To look for genetic variants in addition to HLA alleles associated with hypersensitivity, genome-wide analyses of Illumina typed and Michigan-imputed genotypes passing QC were performed for all patients (all ancestries) in each individual cohort adjusting for percent ancestries and cohort-specific covariates. In a meta-analysis combining results from the three cohorts, all genome-wide significant variants fell in the *HLA-DRB1* and *-DQA1* loci on chromosome 6 (*P*_meta_ < 5×10^−8^, [Table S9](#SD1), [Figure 1B](#F1)). The top hit rs28383308 (*P*_meta_ = 4.9×10^−14^ OR 1.95) locates 9173bp 5’ of the *HLA-DQA1* gene. Among EA patients rs28383308 tags *HLA-DRB1*07:01* (r^2^ = 0.85) and *HLA-DQB1*02:02* to a lesser extent (r^2^ = 0.62, [Table S10A](#SD1)). LD between rs28383308 and *HLA-DQA1*02:01* is the same as that with *HLA-DRB1*07:01* due to the complete linkage between the two alleles. Additionally, we performed GWAS adjusting for top five principal components for ancestry, and the results of top variants are consistent with the results adjusting for percent ancestries (r^2^ > 0.99, [Figure S3](#SD1)).

We also performed GWASs within ancestry groups. In a meta-analysis of EA patients in these protocols, all genome-wide significant variants fell in the *HLA-DQA1* and *-DQB1* locus on chromosome 6 (*P*_meta_ < 5×10^−8^, [Table S11](#SD1), [Figure 1C](#F1)). The top hit rs1694129 (*P*_meta_ = 1.1×10^−8^, OR 2.03) locates 1582 bp 5’ of the *HLA-DQB1* gene in a histone-binding site. rs1694129 tags *HLA-DQB1*02:02*, the top hit among imputed 4-digit *HLA* alleles in EAs (r^2^ = 0.96, [Table S10B](#SD1)). The same analysis adjusting for *HLA-DQB1*02:02* or *HLA-DRB1*07:01* did not yield any genome-wide significant variants, indicating a high level of LD between these individual SNPs and the top HLA alleles ([Figure S4](#SD1)).

Similar SNP-based GWAS and meta-analyses were conducted in combined non-EA (n = 1607), AA (n = 310), and AMR (n = 798) patients adjusting for genetic ancestry. Among non-EAs, rs9958628, located in the 5’-UTR of the Rho GTPase activating protein 28 (*ARHGAP28*) gene transcript, was the top genetic variant and reached genome-wide significance in its association with hypersensitivity (*P*_meta_ = 8.9×10^−9^, OR 3.69, [Figure 1D](#F1), [Table S12](#SD1)). This SNP was also the top non-HLA hit among all evaluable patients (*P*_meta_ = 1.2×10^−7^, OR 3.03, [Figure 1B](#F1)) and was associated with reactions in a meta-analysis in AAs from the three cohorts (*P* = 0.010 in TXVI, *P* = 0.039 in AALL0232, and *P* = 0.0015 in AALL0434, *P*_meta_ = 3.2×10^−5^, OR 2.03) but not in EAs or AMRs, plausibly due to low minor allele frequency (< 1%). The other two genome-wide significant hits among non-EAs were rs79377225 and rs11739459, located 43kb and 39kb 3’ of GC-rich promoter binding protein 1 (*GPBP1*) (*P*_meta_ = 2.5×10^−8^, OR 2.70 and *P*_meta_ = 2.7×10^−8^, OR 2.50 respectively, [Figure 1D](#F1), [Table S12](#SD1)). Some variants in the *HLA-DRA* locus, although they did not reach genome-wide significance, were also associated with reactions ([Table S12](#SD1), [Figure 1D](#F1)). No genetic variants reached genome-wide significance in meta-analyses of AAs or AMRs ([Tables S13](#SD1)&[S14](#SD1), [Figure S5](#SD1)). In addition, we performed trans-ancestry meta-analysis based on the ancestry specific analyses. rs11739459 and rs79377225 remained genome-wide significant ([Table S15](#SD1), *P*_meta_ = 8.8×10^−9^ OR 2.60 and *P*_meta_ = 1.6×10^−8^ OR 2.74 respectively).

Next, we examined if the tagging SNPs for the significant imputed HLA alleles in EA patients were also significant in non-EAs. The top SNP rs1694129 in EAs remained nominally significantly associated with pegaspargase reactions in non-EA patients, adjusting for percent ancestry (*P*_meta_ = 0.013, OR 2.01, [Table S11](#SD1)). This SNP well tags *HLA-DQB1*02:02* in EAs (r^2^ = 0.96), but only moderately tags *HLA-DRB1*07:01* (r^2^ = 0.55) and *HLA-DQB1*02:02* (r^2^ = 0.52) in non-EAs based on TXVI WES data (n = 194). The same SNP did not reach statistical significance when analyzed in AAs only (*P*_meta_ = 0.14) or AMRs only (*P*_meta_ = 0.15), likely due to the limit of smaller sample size, although the association was in the same direction as in EAs. There was some degree of heterogeneity across ancestry groups for this SNP but it did not reach statistical significance (I^2^ = 0.54, P.het = 0.11) The differences in HLA SNPs associated with pegaspargase reactions by ancestry groups are reflected in the locus zoom plots on a region on chromosome 6 covering major class II HLA genes (*HLA-DRA, -DRB1, -DQA1, -DQB1, -DPA1,* and *-DPB1*) ([Figure 2](#F2)). Top HLA hits in all patients (all ancestries) mainly overlap with those in EAs, probably because EA patients made up most of the three cohorts ([Table 1](#T1)). The top HLA hit in non-EA patients *HLA-DRA* rs9268670 did not replicate in EAs (*P*_meta_ = 0.066), although the direction of association was the same. Top HLA hits in all non-EAs overlap with those in AMRs, both in the *HLA-DRA* loci. SNPs in the HLA region seem to have the weakest contribution to risk for pegaspargase reactions in AAs. None of the SNPs in the class I HLA loci had stronger associations than the top hits in class II HLA loci.

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ee/8790808/d05c655476f7/nihms-1765892-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8790808_nihms-1765892-f0002.jpg)

Locus zoom plot showing the SNPs in the major class II HLA genes on chromosome 6. The purple diamond indicates the top class II HLA SNP in this region. Other SNPs analyzed in this region were colored by linkage disequilibrium (LD) with the top hit. LD was calculated based on populations with European ancestry.

## Discussion

Pegaspargase and native ASNase differ in their risk of causing hypersensitivity reactions, the severity of the reactions, and the major epitope targeted by anti-ASP antibodies that mediate the reactions.([1](#R1)) Two previous studies associated native ASNase hypersensitivity with the *HLA-DRB1*07:01-DQA1*02:01-DQB1*02:02* haplotype in pediatric ALL cohorts with predominantly European ancestry.([5](#R5), [6](#R6)) In this report, we analyzed associations between HLA alleles and pegaspargase hypersensitivity in the largest cohort so far of patients exclusively treated with pegaspargase as first line therapy, and our patients were ancestrally diverse. Although predominantly mediated by antibodies against PEG, not the asparaginase protein,([1](#R1)) hypersensitivity reactions to pegaspargase were strongly associated with the same HLA haplotype as in the case of ASNase. Interestingly, Gagne et al attributed the high risk of this haplotype mainly to the presence of *DQB1*02:02*, not the other two alleles.([6](#R6)) Analysis of our cohorts showed similar results ([Table 2](#T2), [Figure 1A](#F1), [2](#F2) & [3](#F3)): *HLA-DQB1*02:02* had a lower P value than the other two alleles ([Table 2](#T2), *P*_meta_ = 3.8×10^−9^ vs. 2.2×10^−5^ and 2.7×10^−5^) and was the only allele significantly associated with reactions in all three cohorts. *HLA-DRB1*07:01* only increased the risk for reactions when accompanied by *-DQB1*02:*02 (*P* = 0.45 vs *P* = 2.0×10^−7^, [Figure 3](#F3)). Patients harboring the full haplotype with all three of *HLA-DRB1*07:01*, *-DQA1*02:01* and -*DQB1*02:02* tended to be at higher risk for reactions than those who only had *-DRB1*07:01* and *-DQA1*02:01* but not *-DQB1*02:02* (21.3% vs 13.7%, *P* = 0.061, [Figure 3](#F3)). A study on type I diabetes also found that the effect of *HLA-DRB1*07:01* relied on the *DQB1* allele it shared a haplotype with.([22](#R22)) However in multivariable analyses, we could not statistically distinguish which of the three alleles or haplotype was dominant due to the extended LD. On the HLA amino acid level, the top amino acid variant associated with reactions marks *HLA-DQB1*02:02*, while none of the top DRB1 amino acid variants mark *HLA1-DRB1*07:01*, further supporting the importance of the DQB1 over the DRB1 allele. After adjusting for *HLA-DQB1*02:02*, the top amino acid was no longer significant and no additional amino acid was significantly associated with reactions adjusting for multiple testing (alpha = 0.05/1041 of AA). Moreover, the top DRB1 amino acids are not shared by the top *HLA-DRB1* allele hits (*HLA-DRB1*07:01*, **04:02*, and **04:05*), and therefore the mechanism of association is not readily explained by the coded amino acids.

### Figure 3.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e4ee/8790808/b9541919551f/nihms-1765892-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8790808_nihms-1765892-f0003.jpg)

Distribution of HLA-DRB1*07:01 and HLA-DRB1*0701-DQB1*02:02 combinations in ALL patients in our combined cohort with and without hypersensitivity reactions (EA patients only, n = 2167, one patient who was HLA-DRB1*07:01-/DQA1*02:01+/DQB1*02:02+ was excluded): Patients with the full HLA-DRB1*07:01-DQA1*02:01-DQB1*02:02 haplotype tended to have higher risk for reactions compared with ones with only the partial haplotype HLA-DRB1*07:01-DQA1*02:01 (P = 0.061). P values were generated from Chi-square test.

We are the first to interrogate genetic risk factors for asparaginase hypersensitivity reactions in non-EA patients as separate ancestry groups. Our results identified *ARHGAP28* rs9958628 as the top genetic variant associated with pegaspargase hypersensitivity in non-EAs with genome-wide significance. The same SNP was also the top non-HLA hit in all patients. Interestingly, variants in the *ARHGAP28* locus were associated with poor response to corticosteroid treatment in asthma patients, especially non-EA children, indicating the immune regulatory function of this locus.([23](#R23), [24](#R24))

Using BWAkit-inferred HLA alleles in the relatively small number of non-EAs enrolled on the TXVI protocol (who had available WES data), HLA alleles highly associated with reactions in EA patients did not replicate in non-EA patients ([Table S7](#SD1)). However, using SNP2HLA-imputed HLA alleles on all three cohorts, the imputed alleles associated with reactions in EA patients did replicate in non-EA patients ([Table S8](#SD1)), but the imputation had lower accuracy than in EA patients ([Table S2](#SD1)). When limited to SNPs (rather than alleles), the top SNP *HLA-DQB1* rs1694129 associated with pegaspargase hypersensitivity reactions in EAs also replicated in non-EAs (*P*_meta_ = 0.013 OR 2.01, [Table S11](#SD1)). Of the 82 SNPs in the *HLA* loci associated with pegaspargase hypersensitivity in EAs with meta-analysis P values ≤ 10^−6^, 27 were nominally significant (*P*_meta_ < 0.05) in non-EAs. Of the 8 SNPs in the *HLA* loci associated with pegaspargase hypersensitivity in non-EAs with meta-analysis P values ≤ 10^−6^, 3 were nominally significant (*P*_meta_ < 0.05) in EAs ([Table S12](#SD1)). Studies on the role of HLA alleles in other diseases, including hematological malignancies and infectious diseases, in different ancestries also revealed shared risk variants as well as substantial ancestry-specific HLA associations.([25](#R25)–[28](#R28)) Better understanding of the alleles and other genetic variants in the *HLA* loci in hypersensitivity for pegaspargase and other drugs and their difference by ancestry need to be further investigated in additional cohorts with HLA typing available across ancestries.

Different from the study of Kutszegi et al,([5](#R5)) we found that T-ALL patients had a higher frequency of the high risk HLA alleles than B-lineage ALL patients in our cohorts ([Figure S6](#SD1)). Although we observed fewer reactions among EA patients in the AALL0434 cohort (all with T-ALL) than other protocol cohorts (*P* = 1.1×10^−7^ vs. AALL0232, *P* = 9.4×10^−6^ vs. TXVI), we cannot conclude that T-ALL patients were at lower or higher risk for this toxicity because: 1) treatment among the three cohorts differed, especially in intrathecal therapy, radiotherapy, and the use of nelarabine; 2) risk for hypersensitivity did not differ by lineage within the TXVI cohort (*P* = 0.99 adjusting for intrathecal therapy); 3) adverse event reporting practice may have differed among protocols. The same group also identified the combination of rs28383172 and rs7775228 as a tag for the risk HLA haplotype.([29](#R29)) This is also true in EAs in our study: 98.3% (403 out of 410) of the carriers of minor alleles for both SNPs harbored the risk HLA haplotype; 99.7% (1753 out of 1758) of the rest were absent for the risk haplotype. Both SNPs were associated with reactions in EAs in our study (*P*_meta_ = 2.9×10^−4^ and 3.9×10^−5^ for rs28383172 and rs7775228, respectively).

Limitations of our study include that the results from the primary cohort were not formally replicated in any individual cohort. HLA allele inference in non-EAs is not based on strong population data. We acknowledge that some protocol-specific treatments, such as glucocorticoid used during remission induction, radiotherapy regimen, and methotrexate regimen, were not consistent across the three cohorts and could impact hypersensitivity. Information on actual therapy delivered was only available for the TXVI protocol. Genotyping methods differed among the three protocols because of the large scale and duration of the study, although potential biases were minimized by our meta-analysis approach.

In summary, the *HLA-DRB1*07:01-DQA1*02:01-DQB1*02:02* haplotype was associated with hypersensitivity to pegaspargase. The same haplotype was previously associated with reactions to non-PEGylated ASNase.([5](#R5), [6](#R6)) This study demonstrated for the first time the association between a non-HLA locus (*ARHGAP28*) and pegaspargase hypersensitivity with genome-wide significance in non-EAs. The role of this locus in immune response and how HLA loci contribute to pegaspargase hypersensitivity in non-EAs require further investigation.

## Supplementary Material

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

HLA haplotype *DRB1*07:01-DQA1*02:01-DQB1*02:02* was associated with higher risk for asparaginase hypersensitivity reactions in -European-ancestry (EA) dominant cohorts treated with native *E. coli* asparaginase (ASNase) only or both ASNase and polyethylene-glycol conjugated asparaginase (pegaspargase).

### What question did this study address?

Which HLA alleles and non-HLA genetic variants contribute to hypersensitivity reactions exclusively to pegaspargase? In an ancestrally diverse cohort, are there ancestry-related differences in these associations?

### What this study adds to our knowledge?

The same HLA haplotype harboring the *DRB1*07:01, DQA1*02:01*, and *DQB1*02:02* alleles also associated with higher risk for hypersensitivity reactions exclusively to pegaspargase in EA patients. Non-HLA genetic variants associated with reactions at genome-wide significance were found at the *ARHGAP28* locus in non-EAs, but not in EAs.

### How might this change clinical pharmacology or translational science?

In this largest cohort study of ancestrally diverse patients treated with pegaspargase, the frontline asparaginase now, we show that ancestry groups differ in their genetic predisposition to allergy. By implicating *HLA* loci as well as *ARHGAP28*, further studies can focus on the mechanisms underlying allergy to pegaspargase and possibly other drugs by ancestry.

## Funding information:

This research was supported by NIH grant(s) NIH P50 115279, GM NCI P30 CA 21765, R01 CA 142665, investigator-initiated research funding from Servier, and ALSAC. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.

Conflict of Interest:

MVR, HI and SJ and St. Jude received investigator-initiated research funding from Servier. All other authors declared no competing interests for this work.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Liu Y et al. Antibodies Predict Pegaspargase Allergic Reactions and Failure of Rechallenge. J Clin Oncol 37, 2051–61 (2019).  [DOI](https://doi.org/10.1200/JCO.18.02439) | [PMC free article](/articles/PMC6804844/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31188727/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Antibodies%20Predict%20Pegaspargase%20Allergic%20Reactions%20and%20Failure%20of%20Rechallenge&author=Y%20Liu&volume=37&publication_year=2019&pages=2051-61&pmid=31188727&doi=10.1200/JCO.18.02439&)

2. Rathod S, Ramsey M, Finkelman FD & Fernandez CA Genetic inhibition of NFATC2 attenuates asparaginase hypersensitivity in mice. Blood Adv 4, 4406–16 (2020).  [DOI](https://doi.org/10.1182/bloodadvances.2020002478) | [PMC free article](/articles/PMC7509855/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32931581/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Adv&title=Genetic%20inhibition%20of%20NFATC2%20attenuates%20asparaginase%20hypersensitivity%20in%20mice&author=S%20Rathod&author=M%20Ramsey&author=FD%20Finkelman&author=CA%20Fernandez&volume=4&publication_year=2020&pages=4406-16&pmid=32931581&doi=10.1182/bloodadvances.2020002478&)

3. Mehta RK et al. Mutations in subunit interface and B-cell epitopes improve antileukemic activities of Escherichia coli asparaginase-II: evaluation of immunogenicity in mice. J Biol Chem 289, 3555–70 (2014).  [DOI](https://doi.org/10.1074/jbc.M113.486530) | [PMC free article](/articles/PMC3916557/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24297177/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Mutations%20in%20subunit%20interface%20and%20B-cell%20epitopes%20improve%20antileukemic%20activities%20of%20Escherichia%20coli%20asparaginase-II:%20evaluation%20of%20immunogenicity%20in%20mice&author=RK%20Mehta&volume=289&publication_year=2014&pages=3555-70&pmid=24297177&doi=10.1074/jbc.M113.486530&)

4. Belen LH et al. A structural in silico analysis of the immunogenicity of l-asparaginase from Escherichia coli and Erwinia carotovora. Biologicals 59, 47–55 (2019).  [DOI](https://doi.org/10.1016/j.biologicals.2019.03.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30871932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biologicals&title=A%20structural%20in%20silico%20analysis%20of%20the%20immunogenicity%20of%20l-asparaginase%20from%20Escherichia%20coli%20and%20Erwinia%20carotovora&author=LH%20Belen&volume=59&publication_year=2019&pages=47-55&pmid=30871932&doi=10.1016/j.biologicals.2019.03.003&)

5. Kutszegi N et al. HLA-DRB1*07:01-HLA-DQA1*02:01-HLA-DQB1*02:02 haplotype is associated with a high risk of asparaginase hypersensitivity in acute lymphoblastic leukemia. Haematologica 102, 1578–86 (2017).  [DOI](https://doi.org/10.3324/haematol.2017.168211) | [PMC free article](/articles/PMC5685222/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28596278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=HLA-DRB1*07:01-HLA-DQA1*02:01-HLA-DQB1*02:02%20haplotype%20is%20associated%20with%20a%20high%20risk%20of%20asparaginase%20hypersensitivity%20in%20acute%20lymphoblastic%20leukemia&author=N%20Kutszegi&volume=102&publication_year=2017&pages=1578-86&pmid=28596278&doi=10.3324/haematol.2017.168211&)

6. Gagne V et al. HLA alleles associated with asparaginase hypersensitivity in childhood ALL: a report from the DFCI Consortium. Pharmacogenomics 21, 541–7 (2020).  [DOI](https://doi.org/10.2217/pgs-2019-0195) | [PMC free article](/articles/PMC7299187/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32372697/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=HLA%20alleles%20associated%20with%20asparaginase%20hypersensitivity%20in%20childhood%20ALL:%20a%20report%20from%20the%20DFCI%20Consortium&author=V%20Gagne&volume=21&publication_year=2020&pages=541-7&pmid=32372697&doi=10.2217/pgs-2019-0195&)

7. Fernandez CA et al. HLA-DRB1*07:01 is associated with a higher risk of asparaginase allergies. Blood 124, 1266–76 (2014).  [DOI](https://doi.org/10.1182/blood-2014-03-563742) | [PMC free article](/articles/PMC4141516/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24970932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=HLA-DRB1*07:01%20is%20associated%20with%20a%20higher%20risk%20of%20asparaginase%20allergies&author=CA%20Fernandez&volume=124&publication_year=2014&pages=1266-76&pmid=24970932&doi=10.1182/blood-2014-03-563742&)

8. Højfeldt SG et al. Genetic predisposition to PEG-asparaginase hypersensitivity in children treated according to NOPHO ALL2008. British journal of haematology 184, 405–17 (2019).  [DOI](https://doi.org/10.1111/bjh.15660) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30450575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=British%20journal%20of%20haematology&title=Genetic%20predisposition%20to%20PEG-asparaginase%20hypersensitivity%20in%20children%20treated%20according%20to%20NOPHO%20ALL2008&author=SG%20H%C3%B8jfeldt&volume=184&publication_year=2019&pages=405-17&pmid=30450575&doi=10.1111/bjh.15660&)

9. Liu Y et al. Genome-Wide Study Links PNPLA3 Variant With Elevated Hepatic Transaminase After Acute Lymphoblastic Leukemia Therapy. Clin Pharmacol Ther, (2017).  [DOI](https://doi.org/10.1002/cpt.629) | [PMC free article](/articles/PMC5511775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28090653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genome-Wide%20Study%20Links%20PNPLA3%20Variant%20With%20Elevated%20Hepatic%20Transaminase%20After%20Acute%20Lymphoblastic%20Leukemia%20Therapy&author=Y%20Liu&publication_year=2017&pmid=28090653&doi=10.1002/cpt.629&)

10. Dunsmore KP et al. Children’s Oncology Group AALL0434: A Phase III Randomized Clinical Trial Testing Nelarabine in Newly Diagnosed T-Cell Acute Lymphoblastic Leukemia. J Clin Oncol 38, 3282–93 (2020).  [DOI](https://doi.org/10.1200/JCO.20.00256) | [PMC free article](/articles/PMC7526719/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32813610/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Children%E2%80%99s%20Oncology%20Group%20AALL0434:%20A%20Phase%20III%20Randomized%20Clinical%20Trial%20Testing%20Nelarabine%20in%20Newly%20Diagnosed%20T-Cell%20Acute%20Lymphoblastic%20Leukemia&author=KP%20Dunsmore&volume=38&publication_year=2020&pages=3282-93&pmid=32813610&doi=10.1200/JCO.20.00256&)

11. Bansal V & Libiger O Fast individual ancestry inference from DNA sequence data leveraging allele frequencies for multiple populations. BMC Bioinformatics 16, 4 (2015).  [DOI](https://doi.org/10.1186/s12859-014-0418-7) | [PMC free article](/articles/PMC4301802/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25592880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&title=Fast%20individual%20ancestry%20inference%20from%20DNA%20sequence%20data%20leveraging%20allele%20frequencies%20for%20multiple%20populations&author=V%20Bansal&author=O%20Libiger&volume=16&publication_year=2015&pages=4&pmid=25592880&doi=10.1186/s12859-014-0418-7&)

12. Genomes Project C et al. A global reference for human genetic variation. Nature 526, 68–74 (2015).  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation&author=C%20Genomes%20Project&volume=526&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

13. Mao X et al. A genomewide admixture mapping panel for Hispanic/Latino populations. Am J Hum Genet 80, 1171–8 (2007).  [DOI](https://doi.org/10.1086/518564) | [PMC free article](/articles/PMC1867104/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17503334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20genomewide%20admixture%20mapping%20panel%20for%20Hispanic/Latino%20populations&author=X%20Mao&volume=80&publication_year=2007&pages=1171-8&pmid=17503334&doi=10.1086/518564&)

14. Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM & Lee JJ Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience 4, 7 (2015).  [DOI](https://doi.org/10.1186/s13742-015-0047-8) | [PMC free article](/articles/PMC4342193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25722852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gigascience&title=Second-generation%20PLINK:%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&author=CC%20Chang&author=CC%20Chow&author=LC%20Tellier&author=S%20Vattikuti&author=SM%20Purcell&volume=4&publication_year=2015&pages=7&pmid=25722852&doi=10.1186/s13742-015-0047-8&)

15. Purcell S et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet 81, 559–75 (2007).  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&volume=81&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

16. Jia X et al. Imputing amino acid polymorphisms in human leukocyte antigens. PLoS One 8, e64683 (2013).  [DOI](https://doi.org/10.1371/journal.pone.0064683) | [PMC free article](/articles/PMC3675122/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23762245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Imputing%20amino%20acid%20polymorphisms%20in%20human%20leukocyte%20antigens&author=X%20Jia&volume=8&publication_year=2013&pages=e64683&pmid=23762245&doi=10.1371/journal.pone.0064683&)

17. Pillai NE et al. Predicting HLA alleles from high-resolution SNP data in three Southeast Asian populations. Hum Mol Genet 23, 4443–51 (2014).  [DOI](https://doi.org/10.1093/hmg/ddu149) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24698974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Predicting%20HLA%20alleles%20from%20high-resolution%20SNP%20data%20in%20three%20Southeast%20Asian%20populations&author=NE%20Pillai&volume=23&publication_year=2014&pages=4443-51&pmid=24698974&doi=10.1093/hmg/ddu149&)

18. Li H Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv 1303, (2013).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=arXiv&title=Aligning%20sequence%20reads,%20clone%20sequences%20and%20assembly%20contigs%20with%20BWA-MEM&author=H%20Li&volume=1303&publication_year=2013&)

19. Pruim RJ et al. LocusZoom: regional visualization of genome-wide association scan results. Bioinformatics 26, 2336–7 (2010).  [DOI](https://doi.org/10.1093/bioinformatics/btq419) | [PMC free article](/articles/PMC2935401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20634204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LocusZoom:%20regional%20visualization%20of%20genome-wide%20association%20scan%20results&author=RJ%20Pruim&volume=26&publication_year=2010&pages=2336-7&pmid=20634204&doi=10.1093/bioinformatics/btq419&)

20. Mallal S et al. Association between presence of HLA-B*5701, HLA-DR7, and HLA-DQ3 and hypersensitivity to HIV-1 reverse-transcriptase inhibitor abacavir. Lancet 359, 727–32 (2002).  [DOI](https://doi.org/10.1016/s0140-6736(02)07873-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11888582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Association%20between%20presence%20of%20HLA-B*5701,%20HLA-DR7,%20and%20HLA-DQ3%20and%20hypersensitivity%20to%20HIV-1%20reverse-transcriptase%20inhibitor%20abacavir&author=S%20Mallal&volume=359&publication_year=2002&pages=727-32&pmid=11888582&doi=10.1016/s0140-6736(02)07873-x&)

21. Karnes JH et al. Comparison of HLA allelic imputation programs. PLoS One 12, e0172444 (2017).  [DOI](https://doi.org/10.1371/journal.pone.0172444) | [PMC free article](/articles/PMC5312875/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28207879/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Comparison%20of%20HLA%20allelic%20imputation%20programs&author=JH%20Karnes&volume=12&publication_year=2017&pages=e0172444&pmid=28207879&doi=10.1371/journal.pone.0172444&)

22. Noble JA, Johnson J, Lane JA & Valdes AM HLA Class II Genotyping of African American Type 1 Diabetic Patients Reveals Associations Unique to African Haplotypes. Diabetes 62, 3292–9 (2013).  [DOI](https://doi.org/10.2337/db13-0094) | [PMC free article](/articles/PMC3749336/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23801574/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=HLA%20Class%20II%20Genotyping%20of%20African%20American%20Type%201%20Diabetic%20Patients%20Reveals%20Associations%20Unique%20to%20African%20Haplotypes&author=JA%20Noble&author=J%20Johnson&author=JA%20Lane&author=AM%20Valdes&volume=62&publication_year=2013&pages=3292-9&pmid=23801574&doi=10.2337/db13-0094&)

23. Dahlin A et al. CMTR1 is associated with increased asthma exacerbations in patients taking inhaled corticosteroids. Immunity, inflammation and disease 3, 350–9 (2015).  [DOI](https://doi.org/10.1002/iid3.73) | [PMC free article](/articles/PMC4693729/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26734457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunity,%20inflammation%20and%20disease&title=CMTR1%20is%20associated%20with%20increased%20asthma%20exacerbations%20in%20patients%20taking%20inhaled%20corticosteroids&author=A%20Dahlin&volume=3&publication_year=2015&pages=350-9&pmid=26734457&doi=10.1002/iid3.73&)

24. Hernandez-Pacheco N et al. Genome-wide association study of inhaled corticosteroid response in admixed children with asthma. Clinical and experimental allergy : journal of the British Society for Allergy and Clinical Immunology 49, 789–98 (2019).  [DOI](https://doi.org/10.1111/cea.13354) | [PMC free article](/articles/PMC7054824/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30697902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20and%20experimental%20allergy%20:%20journal%20of%20the%20British%20Society%20for%20Allergy%20and%20Clinical%20Immunology&title=Genome-wide%20association%20study%20of%20inhaled%20corticosteroid%20response%20in%20admixed%20children%20with%20asthma&author=N%20Hernandez-Pacheco&volume=49&publication_year=2019&pages=789-98&pmid=30697902&doi=10.1111/cea.13354&)

25. Gragert L, DiPrima S, Albrecht M, Maiers M, Kalaycio M & Hill BT HLA Is a Determinant Of The Ethnic Predisposition Of Chronic Lymphocytic Leukemia. Blood 122, (2013).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=HLA%20Is%20a%20Determinant%20Of%20The%20Ethnic%20Predisposition%20Of%20Chronic%20Lymphocytic%20Leukemia&author=L%20Gragert&author=S%20DiPrima&author=M%20Albrecht&author=M%20Maiers&author=M%20Kalaycio&volume=122&publication_year=2013&)

26. Gragert L et al. HLA Haplotypes Are Associated with Multiple Myeloma Risk in the African American Multiple Myeloma Study (AAMMS). Blood 128, (2016).  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=HLA%20Haplotypes%20Are%20Associated%20with%20Multiple%20Myeloma%20Risk%20in%20the%20African%20American%20Multiple%20Myeloma%20Study%20(AAMMS)&author=L%20Gragert&volume=128&publication_year=2016&)

27. Thio CL et al. Racial differences in HLA class II associations with hepatitis C virus outcomes. Journal of Infectious Diseases 184, 16–21 (2001).  [DOI](https://doi.org/10.1086/321005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11398104/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Infectious%20Diseases&title=Racial%20differences%20in%20HLA%20class%20II%20associations%20with%20hepatitis%20C%20virus%20outcomes&author=CL%20Thio&volume=184&publication_year=2001&pages=16-21&pmid=11398104&doi=10.1086/321005&)

28. Sudenga SL et al. Dense Genotyping of Immune-Related Loci Identifies Variants Associated with Clearance of HPV among HIV-Positive Women in the HIV Epidemiology Research Study (HERS). Plos One 9, (2014).  [DOI](https://doi.org/10.1371/journal.pone.0099109) | [PMC free article](/articles/PMC4053382/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24918582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plos%20One&title=Dense%20Genotyping%20of%20Immune-Related%20Loci%20Identifies%20Variants%20Associated%20with%20Clearance%20of%20HPV%20among%20HIV-Positive%20Women%20in%20the%20HIV%20Epidemiology%20Research%20Study%20(HERS)&author=SL%20Sudenga&volume=9&publication_year=2014&pmid=24918582&doi=10.1371/journal.pone.0099109&)

29. Kutszegi N et al. Two tagging SNPs to capture HLA-DRB1*07:01-DQA1*02:01-DQB1*02:02 haplotype associated with asparaginase hypersensitivity. Br J Clin Pharmacol, (2020).  [DOI](https://doi.org/10.1111/bcp.14664) | [PMC free article](/articles/PMC8247313/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33217039/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Two%20tagging%20SNPs%20to%20capture%20HLA-DRB1*07:01-DQA1*02:01-DQB1*02:02%20haplotype%20associated%20with%20asparaginase%20hypersensitivity&author=N%20Kutszegi&publication_year=2020&pmid=33217039&doi=10.1111/bcp.14664&)
