# The Cyp2b6 Gene Polymorphism and Phenotypic Correlation of Efavirenz-Based Combination Therapy Among the Niger Delta Ethnic Population: Implications in Modern Pharmacogenomics

## Metadata
**Authors:** Samuel J Bunu, Azibanasamesa D C Owaba, Edebi N Vaikosen, Benjamin U Ebeshi
**Journal:** Pharmacogenomics and Personalized Medicine
**Date:** 2022 Jan 25
**DOI:** [10.2147/PGPM.S345038](https://doi.org/10.2147/PGPM.S345038)
**PMID:** 35115810
**PMCID:** PMC8800862
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8800862/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8800862/pdf/pgpm-15-45.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8800862/pdf/pgpm-15-45.pdf)

## Abstract

**Purpose:** 
DNA polymorphism describes the difference between individuals, groups, or ethnicities, races, etc., in terms of their DNA sequences or phenotypes as relates to drug metabolism. Using predictive genotyping of drug-metabolizing genes, we can develop individuals’ drug therapies that are less toxic and more effective. The main aim of the study was to evaluate genotype–phenotype-based correlation and incidence of genetic polymorphism of efavirenz blood levels among HIV/AIDS patients of the Niger Delta population.

**Methods:** 
A study questionnaire was designed to obtained patients’ data, blood samples were obtained, plasma was separated from the serum using a centrifuge for 5 minutes at 4000 rpm for HPLC analysis, polymerase chain reaction (PCR) and restriction fragment length polymorphism (RFLP) analysis was conducted using Bsrl endonuclease enzyme to digest the PCR amplicons. Standard efavirenz was used at 0.5, 1, 2, 4, 16 mg/L to construct a calibration curve. Data were analyzed with SPSS software using chi-square test at p-value ≤0.5 and Microsoft excel 2013, while PCR and RFLP results were obtained after 1% Agarose gel electrophoresis, respectively.

**Results:** 
Phenotypic results showed that the participants had different efavirenz plasma concentrations. Six subjects (12%) had efavirenz plasma levels below 0.10 mg/L, considered ultra-rapid metabolizers (UMs), 22 (44%) 0.10 mg/L to 0.90 mg/L, classified as extensive metabolizers (EMs), 19 (38%) had 1.0 to 3.9 mg/L and were noted as intermediate metabolizers (IM), while 3 (6%) subjects showed efavirenz plasma levels from 4.0 mg/L to 6.0 mg/L, categorized as poor metabolizers (PM). RFLP results showed more than half of the population (56%) with a homozygous wild-type gene with CYP2B6*1*1 allele, 38% were CYP2B6*1*6 (heterozygous mutant) allele and 6% had homozygous mutant gene (CYP2B6*6*6 allele). Out of the 15 male subjects among the 50 patients that participated in the study, 8% were UM, 12% EM, 14% IM while no PM was observed, on the contrary, out of the 35 females participated in the study, 4% were observed as UM, 32% EM, 24% IM, while 6% were PM.

**Conclusion:** 
There was no significant difference (p ≤ 0.05) between genotype and phenotype data for CYP2B6 polymorphism, among the HIV/AIDS patients that participated in this study. Genetic polymorphism of the CYP2B6 gene is prevalent among HIV/AIDs patients in the Niger Delta ethnic population on efavirenz-based HAART treatment, as the population having homozygous mutant gene or PM are >1% (6%).

Keywords: Antiretroviral, NNRTIs, CYP2B6, Genetic polymorphism, efavirenz, phenotype

### Purpose

DNA polymorphism describes the difference between individuals, groups, or ethnicities, races, etc., in terms of their DNA sequences or phenotypes as relates to drug metabolism. Using predictive genotyping of drug-metabolizing genes, we can develop individuals’ drug therapies that are less toxic and more effective. The main aim of the study was to evaluate genotype–phenotype-based correlation and incidence of genetic polymorphism of efavirenz blood levels among HIV/AIDS patients of the Niger Delta population.

### Methods

A study questionnaire was designed to obtained patients’ data, blood samples were obtained, plasma was separated from the serum using a centrifuge for 5 minutes at 4000 rpm for HPLC analysis, polymerase chain reaction (PCR) and restriction fragment length polymorphism (RFLP) analysis was conducted using *Bsrl* endonuclease enzyme to digest the PCR amplicons. Standard efavirenz was used at 0.5, 1, 2, 4, 16 mg/L to construct a calibration curve. Data were analyzed with SPSS software using chi-square test at p-value ≤0.5 and Microsoft excel 2013, while PCR and RFLP results were obtained after 1% Agarose gel electrophoresis, respectively.

### Results

Phenotypic results showed that the participants had different efavirenz plasma concentrations. Six subjects (12%) had efavirenz plasma levels below 0.10 mg/L, considered ultra-rapid metabolizers (UMs), 22 (44%) 0.10 mg/L to 0.90 mg/L, classified as extensive metabolizers (EMs), 19 (38%) had 1.0 to 3.9 mg/L and were noted as intermediate metabolizers (IM), while 3 (6%) subjects showed efavirenz plasma levels from 4.0 mg/L to 6.0 mg/L, categorized as poor metabolizers (PM). RFLP results showed more than half of the population (56%) with a homozygous wild-type gene with CYP2B6*1*1 allele, 38% were CYP2B6*1*6 (heterozygous mutant) allele and 6% had homozygous mutant gene (CYP2B6*6*6 allele). Out of the 15 male subjects among the 50 patients that participated in the study, 8% were UM, 12% EM, 14% IM while no PM was observed, on the contrary, out of the 35 females participated in the study, 4% were observed as UM, 32% EM, 24% IM, while 6% were PM.

### Conclusion

There was no significant difference (p ≤ 0.05) between genotype and phenotype data for CYP2B6 polymorphism, among the HIV/AIDS patients that participated in this study. Genetic polymorphism of the CYP2B6 gene is prevalent among HIV/AIDs patients in the Niger Delta ethnic population on efavirenz-based HAART treatment, as the population having homozygous mutant gene or PM are >1% (6%).

## Introduction

By maximizing medication regimens using precision medicine approaches, modern medicine offers powerful techniques for optimizing treatment.[1](#cit0001) As the name implies, precision medicine refers to prescribing individual therapeutic regimes based on one’s genetic makeup.[2](#cit0002) The use of pharmacogenetics to identify genetic biomarkers that predict individual responses to medications has been studied.[3](#cit0003) The Cytochrome P450 enzymes metabolize over 70% of clinically prescribed drugs worldwide, and pharmacogenetic studies have mainly focused on polymorphisms of genes encoding these enzymes.[4](#cit0004)

Efavirenz, (4S)-6-chloro-4-(2-cyclopropylethynyl)-4-(trifluoromethyl)-2, 4-dihydro-1H-1,3-benzoxazin-2-one), pharmacokinetics is associated with a long steady-state half-life of 40–55 hours, which is suitable for a once-daily dose.[5](#cit0005) A threefold increase in oral clearance has been observed after multiple administrations of efavirenz due to its self-metabolism-inducing ability.[6](#cit0006) Likewise, intermediate plasma efavirenz levels have been associated with therapeutic failure and can result in viral resistance, while levels at or above 4 mg/L increase the risk of neuropsychiatric adverse effects.[7](#cit0007) Studies have also reported that efavirenz concentration inside cells as well as the genotype of CYP2B6 were predictors of neuropsychological toxicity. In addition, these authors suggested that CYP2B6 genotyping may be useful as a complement to individualized strategies derived from plasma drug levels to improve safety and reduce side effects associated with efavirenz. It was observed in Nigerians that no significant differences existed between wild type and the genotype (P≤ 0.05) for CYP2B6*6 and CYP2D6.[8](#cit0008)^,^[9](#cit0009) Since there are platforms that can massively detect single-nucleotide polymorphisms (SNPs), these genetic differences are better characterized.[10](#cit0010) There is a strong impact of HIV/AIDS on society; especially when considering the issue of discrimination against immunocompromised people. Also, it has indirect and direct economic consequences.[11](#cit0011) There are guidelines of WHO for the use of Highly Active Anti-Retroviral Treatment (HAART) in sub-Saharan African countries.[12](#cit0012) Clinical correlations have been found for the variants of CYP2B6 in numerous studies. In a population pharmacokinetic analysis, significant interindividual differences in bioavailability of efavirenz were observed, as well as an inverse correlation between exposure levels and central nervous system toxicities.[13](#cit0013)

In contrast to cloning and propagation from host cells, polymerase chain reaction provides an excellent molecular technique for amplification of a specific segment of DNA.[14](#cit0014) DNA sequence variations can be used to analyze restriction fragment length polymorphism (RFLP). Gel electrophoresis is used to separate fragments of DNA following restriction digestion by enzymatic restriction endonuclease (RE). Both alleles can be identified using RFLP markers in heterozygous samples and are highly locus-specific.[15](#cit0015) In addition to PCR-RFLP, another analytical technique used was high-performance liquid chromatography (HPLC), an important analytical approach used in chemical and biochemical analysis. This method was used to determine the plasma levels of efavirenz among the study population.[16](#cit0016) Therefore, the study aimed to evaluate genotype, and phenotype-based correlation, and incidence of genetic polymorphism of efavirenz blood levels among HIV/AIDS positive patients.

## Materials and Methods

### Subjects

Fifty (50), HIV/AIDs positive not related patients consisting of males (15) and females (35), age 18 years and above, who were receiving efavirenz (600mg daily) in combination with either tenofovir and lamivudine or tenofovir with zidovudine, respectively, that met the study inclusion criteria, were randomly selected from the various ethnic groups in Niger Delta consisting of Akwa Ibom (9, 18%), Bayelsa (15, 3%), Cross River (6, 12%), Delta (7, 14%), Edo (5, 10%), Rivers (8, 16%). Details of the study procedures were explained to the potential subjects after which, they were allowed to make an independent decision whether or not to participate in the study. Qualified subjects were enrolled and signed the consent form.

### Sampling and Genomic DNA Preparation

Blood samples (5mL) were collected from all participants who had received 600 mg of efavirenz for at least 3 weeks, using venous-puncture into EDTA tubes clearly labeled with the participant’s name. 2.0 mL was used for the genotyping, while 3.0 mL for HPLC analysis. The whole blood (3.0 mL) was centrifuged for 10 minutes at 10,000 rpm to produce plasma, which was then pipetted into EDTA tubes and heated for 60 minutes at 60°C to inactivate the HIV. After collection, blood samples and plasma were frozen at −20°C until further analysis. The remaining 2.0 mL of blood was used for genotyping procedures. Quick-DNA Blood Miniprep Kits (Inqaba Biotec, West Africa) was used to prepare the DNA. The frozen blood samples were thawed at normal room temperature (20°C) and 100 µL of blood was transferred to a polypropylene tube. On a separate tube rack, four sets of tubes were arranged, with each set labeled for a particular sample. The third tube set was added with a column and the fourth set with a secure label (tubes and columns are provided). We added 650 µL of Genomic lysis buffer and vortexed it for 4–6 seconds before letting it stand at 20°C for 6–10 minutes. The mixture was transferred to a Zymo-Spin Column in a collection tube and centrifuged at 10,000 rpm for about 1 minute. The collection tube was discarded with the flow-through. The Zymo-Spin Column was transferred into a new collection tube; 200 µL of DNA Pre-Wash Buffer was added to the spin column and was centrifuged at 10,000 rpm for one minute and 500 µL of gDNA wash buffer was added to the spin column and centrifuged at 10,000 rpm for another one minute. The spin column was transferred to a clean microcentrifuge tube and 50 µL DNA elution buffer was added to the spin column. Thereafter, the tube was incubated at room temperature for 2–5 minutes and then centrifuged at top speed for 30 seconds to elute the DNA. The eluted DNA purity, yield, UV/Vis absorbance, and ratio were quantified using the NanoDrop-1000 Spectrophotometer. DNA samples were then stored at 4°C before genotyping analysis and aliquots of the samples were stored at −20°C for further analysis.

### Genotyping

DNA quantification was done using a Nanodrop-1000 spectrophotometer following the method by Bunu et al.[17](#cit0017) The genotyping of *CYP2B6* was performed using standard polymerase chain reaction-restriction fragment length polymorphism (PCR-RFLP) techniques following the method of Rotger et al[18](#cit0018) validated by Ebeshi et al.[9](#cit0009) The sequences for the reverse and forward primers for *CYP2B6*6* were 5’-GGTCTGCCCATCTATAAAC-3’ (forward primer) and 5’-CTGATTCTTCACATGTCTGCG-3’ (reverse primer) were used for the gene amplification. The specificity of the primer sequence for each gene studied was confirmed by a BLAST analysis search and comparison of genomic sequences in the National Centre for Biotechnology Information (NCBI) databases ([http://blast.ncbi.nlm.nih.gov/Blast.cgi](http://blast.ncbi.nlm.nih.gov/Blast.cgi)).[19](#cit0019) The PCR reaction was carried out in a total of 25 µL reaction mixture (Master Mix) containing, 200 µM of deoxynucleoside triphosphate mixture, 2.0 mM of MgCl_2_, 1xPCR buffer, and 1 Unit of Taq polymerase; 0.5 µM of each primer, 5 ng genomic DNA, nuclease-free water. All PCRs were performed on GeneAmp^®^ PCR System 9700 programmable thermal cycler. The gene amplification method, PCR cycles for *CYP2B6*6* consists of three major phases: including an initial denaturation of the DNA by incubating the reaction mixture at 94°C for 6 minutes, followed by 35 cycles of denaturation at 94°C for 30 seconds; primer annealing was performed at 54°C for 30 seconds; and primer extension at 72°C for 1 minute and a final extension at 72°C for 6 minutes. DNA polymerase (Taq polymerase) was used in extending the primers, adding nucleotides onto the primer in a sequential manner, using the target DNA as a template. The PCR product’s base pair (bp) was confirmed by 1% agarose gel electrophoresis and Gel Photosystem (blue light) – P1-1002.[20](#cit0020)

The restriction fragment length polymorphism (RFLP) reaction was performed in a total of 20 µL reaction mixture containing 1 µL of the restriction enzyme (B*sr*I), 5 µL of 10x NEBuffer, 8 µL of PCR amplicon, and 6 µL of nuclease-free water for 15 minutes at 65°C. The digested PCR products (amplicons) were visualized on 1% agarose gel electrophoresis. The agarose gel (1%) was prepared by weighing 1.5g of agarose into a conical flask and adding 100mL of 1xTBE gel buffer solution. The agarose was completely dissolved by heating in the microwave for 2 minutes, allowed to cool (55°C) and 1.5 µL of EZ vision dye was added to the gel and poured onto the plate. 5 µL of PCR product was loaded on the gel for analysis of amplified product, and 20 µL for digestion product analysis while 10 µL of the diluted molecular weight marker (DNA ladder) was loaded to the gel. The gel was run in the electrophoretic system at 130V, allowing migration of 2.5 to 3.0 cm, and viewed using the Gel Photosystem P1-1002.

### High-Performance Liquid Chromatography

The HPLC mobile phase consists of methanol and 10mM potassium dihydrogen phosphate at pH 4.5 (70:30) percentage compositions. The column used was HYPERSIL C_18_ ODS, 250mm by 4.6mm, and 5µm particle size at a flow rate of 1.000mL/min. The Ultraviolet-Visible detector was set at 247 nm with a column oven at 40°C temperature. From the 5 mL of the blood collected from participants, 3 mL was used for the HPLC/phenotype analysis. Plasma was separated from the whole blood using a centrifuge at 4000 rpm for 5 minutes. The total concentrations of efavirenz in plasma following protein precipitation with acetonitrile were measured using reversed-phase high-performance liquid chromatography (HPLC) with UV detection following the method of Ebeshi et al[21](#cit0021) with some modifications: introduction of an internal standard and solvent system. Standard curves were analyzed in the concentration ranging from 0.5 to 16.0 µg/mL for efavirenz. Plasma samples (1000 µL) were pipetted into new plain sample bottles followed by 10 µL of lamivudine (IS). 2mL of acetonitrile was added to precipitate the proteins. The mixture was vortex for 30 seconds and then centrifuged at 4000 rpm for 5 minutes. The supernatants were then filtered using a micro Millipore filter of 0.45µm particle size per sample. The filtrate of 20 µL was injected twice manually onto the HPLC, and the peak areas were recorded and integrated by an enhanced ChemStation software integrator.

## Results

Series of information was generated in the study including demographics, genotyping, and phenotypic data. Demographic data were analyzed with Microsoft Excel windows vista 2013 (Microsoft Corporation, USA) and GraphPad Instat software. Results are presented in Tables and Charts.

### Polymerase Chain Reaction (PCR) - Amplicons

The polymerase chain reaction (PCR) was carried for all the 50 samples obtained from study participants, and the results are presented below with codes B1–B50 ([Figures 2](#f0002)), indicating participants’ numbers.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/263f/8800862/4ae0b9c7357c/PGPM-15-45-g0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8800862_PGPM-15-45-g0002.jpg)

Polymerase chain reaction amplicons of Sample B11 – B30: All PCR gene amplicons appeared at 500bp, using a 100bp DNA molecular ladder.

### Restriction Fragment Length Polymorphism (RFLP)

The DNA amplicons (PCR products) were digested using B*rs*l endonuclease enzymes, and participants were classified into ultra-rapid, extensive, intermediate, or poor metabolizers based on the restriction fragment length polymorphism (RFLP) results ([Figures 3](#f0003)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/263f/8800862/5ce0ddac3c59/PGPM-15-45-g0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8800862_PGPM-15-45-g0003.jpg)

(B31–B40): Agarose gel electrophoresis showing the amplified CYP gene bands. Lane 1, 3, 5, 7, 9, 11, 13, 15, 17, and 19 represent the undigested CYP amplicons (Sample code B31 – B40), lane 2, 6, 8, 12, and 18 represents the digested homozygous (wild type) CYP gene, lane 4, 10, 14, 16, and 20 represent the heterozygous CYP gene (wild and mutant allele), while lane L represents the 100bp molecular ladder.

## Discussion

Participants were selected across all the major ethnic groups in the Niger Delta region (South–South) of Nigeria with Bayelsa state having the highest number (30%), while Edo state recorded the least participants (10%). This is largely due to the fact that, the facility approved for the study was within Bayelsa State; hence, more of the indigenes had ease access to HAART therapy. Efavirenz plasma concentration was determined by high-performance liquid chromatography (HPLC) and genotype frequencies were obtained by direct counting following PCR-RFLP analyses. The standard curve for HPLC was constructed with Microsoft Excel spreadsheet, Windows 2013 (Microsoft Corporation, USA). The linearity of the regression line, as well as the regression coefficient (*R*^2^), were; y = 0.1795 x + 0.6192 and 0.995, respectively, ([Table 1](#t0001) and [Figure 1](#f0001)). The total plasma concentration of efavirenz following protein precipitation was accurately measured using reverse-phase high-performance liquid chromatographic (HPLC) technique with lamivudine as an internal standard ([Table 1](#t0001)). Results from the analysis showed that the participants had efavirenz plasma concentrations from 0.00 mg/L to 6.0 mg/L. Six subjects had efavirenz plasma levels below 0.10 mg/L ([Table 2](#t0002)). This category of patients may be considered as ultra-rapid metabolizers (UMs), as very low concentration of the drug was observed in their plasma ([Table 1](#t0001)). It has been reported in several previous studies that polymorphisms in genes that normally encodes drug-metabolizing enzymes may lead to diverse enzyme activity and inter-individual variability in drug efficacy and as well as toxicity.[22](#cit0022) However, several factors other than genetic mutation could lead to this low plasma levels among these subjects including non-adherence, missed dosage, alcoholic habit, polypharmacy, and opportunistic infections.[23](#cit0023) Moreover, the rest of the participants (45) had efavirenz plasma levels ranging from 0.10mg/L to 6.0mg/L. Twenty-two (22) of the participants showed efavirenz plasma levels from 0.10m g/L to 0.90m g/L. These categories of patients were classified as extensive metabolizers (EMs), nineteen (19) had 1.0 to 3.9m g/L and were noted as intermediate metabolizers (IMs). In addition, 3 (6%) subjects showed efavirenz plasma levels from 4.0mg/L to 6.0 mg/L, which were categorized as poor metabolizers (PMs) ([Table 3](#t0003)).

### Table 1.

| Sample Code | Peak Area Efavirenz (a) (mAU) | Peak Area Efavirenz (b) (mAU) | Average Peak Area Efavirenz (mAU) | Peak Area Lamivudine (ISa) (mAU) | Peak Area Lamivudine (ISb) (mAU) | Average Peak Area IS (mAU) | Peak Area Ratio | Conc (µg/mL) | Suspected Phenotype |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| B1 | 850.16 | 860.98 | 855.57 | 1316.46 | 1348.01 | 1332.235 | 0.642 | 0.1 | EM |
| B2 | 920.34 | 1818.88 | 1369.61 | 2320.96 | 1423.4 | 1872.18 | 0.732 | 0.6 | EM |
| B3 | 1182.19 | 1182.19 | 1182.19 | 1325.45 | 2325.45 | 1825.45 | 0.648 | 0.2 | EM |
| B4 | 159.57 | 164.23 | 161.9 | 191.67 | 167.86 | 179.765 | 0.901 | 1.6 | IM |
| B5 | 1158.53 | 164.55 | 661.54 | 1117.86 | 111.08 | 614.47 | 1.077 | 2.6 | IM |
| B6 | 160.58 | 167.37 | 163.975 | 136.96 | 100.18 | 118.57 | 1.383 | 4.3 | PM |
| B7 | 2251.87 | 2242.02 | 2246.945 | 4451.14 | 2445.35 | 3448.245 | 0.652 | 0.2 | EM |
| B8 | 1942.74 | 2245.02 | 2093.88 | 3737.07 | 3147.09 | 3442.08 | 0.608 | −0.1 | UM |
| B9 | 5912 | 2196.27 | 4054.135 | 2909.52 | 2299.61 | 2604.565 | 1.557 | 5.2 | PM |
| B10 | 2501.86 | 2383.77 | 2442.815 | 4240.93 | 3423.93 | 3832.43 | 0.637 | 0.1 | EM |
| B11 | 401.55 | 4040.82 | 2221.185 | 401.55 | 4040.82 | 2221.185 | 1.000 | 2.1 | IM |
| B12 | 2178.77 | 2837.54 | 2508.155 | 3729.88 | 3771.72 | 3750.8 | 0.669 | 0.3 | EM |
| B13 | 354.5 | 346.96 | 350.73 | 777.04 | 341.69 | 559.365 | 0.627 | 0.0 | UM |
| B14 | 1341.01 | 536.11 | 938.56 | 1040.93 | 1614.96 | 1327.945 | 0.707 | 0.5 | EM |
| B15 | 1145.54 | 2181.65 | 1663.595 | 1848.28 | 2105.35 | 1976.815 | 0.842 | 1.2 | IM |
| B16 | 0 | 0 | 0 | 4445.13 | 3104.04 | 3774.585 | 0.000 | −3.5 | UM |
| B17 | 1118.22 | 1116.76 | 1117.49 | 1234.98 | 1185.42 | 1210.2 | 0.923 | 1.7 | IM |
| B18 | 1121.12 | 1112.8 | 1116.96 | 1153.53 | 2064.32 | 1608.925 | 0.694 | 0.4 | EM |
| B19 | 1633.59 | 555.99 | 1094.79 | 1130.01 | 1020.39 | 1075.2 | 1.018 | 2.2 | IM |
| B20 | 1134.24 | 134.24 | 634.24 | 1043.95 | 1017.82 | 1030.885 | 0.615 | 0.0 | UM |
| B21 | 1111.57 | 1113.61 | 1112.59 | 1047.47 | 1483.86 | 1265.665 | 0.879 | 1.5 | IM |
| B22 | 283.95 | 2105.02 | 1194.485 | 1821 | 874.4 | 1347.7 | 0.886 | 1.5 | IM |
| B23 | 742.56 | 642.14 | 692.35 | 1204.2 | 1027.16 | 1115.68 | 0.621 | 0.0 | UM |
| B24 | 1416.71 | 1518.57 | 1467.64 | 2118.27 | 2452.34 | 2285.305 | 0.642 | 0.1 | EM |
| B25 | 1110.61 | 1116.96 | 1113.785 | 1068.31 | 2068.3 | 1568.305 | 0.710 | 0.5 | EM |
| B26 | 2454.84 | 3713.69 | 3084.265 | 4653.15 | 5083.85 | 4868.5 | 0.634 | 0.1 | EM |
| B27 | 4500.91 | 5500.91 | 5000.91 | 5063.07 | 5063.07 | 5063.07 | 0.988 | 2.1 | IM |
| B28 | 1317.36 | 1412.35 | 1364.855 | 1096.08 | 2095.8 | 1595.94 | 0.855 | 1.3 | IM |
| B29 | 4142.86 | 3145.39 | 3644.125 | 3982.58 | 6704.96 | 5343.77 | 0.682 | 0.4 | EM |
| B30 | 4167.3 | 3129.5 | 3648.4 | 5743.31 | 3309.1 | 4526.205 | 0.806 | 1.0 | IM |
| B31 | 4418.29 | 1117.88 | 2768.085 | 1328.65 | 3682.03 | 2505.34 | 1.105 | 2.7 | IM |
| B32 | 230.42 | 164.86 | 197.64 | 220.32 | 176.73 | 198.525 | 0.996 | 2.1 | IM |
| B33 | 421.84 | 722.36 | 572.1 | 807.2 | 588.64 | 697.92 | 0.820 | 1.1 | IM |
| B34 | 1430.66 | 628.25 | 1029.455 | 970.24 | 928.11 | 949.175 | 1.085 | 2.6 | IM |
| B35 | 110.46 | 138.13 | 124.295 | 95.06 | 208.59 | 151.825 | 0.819 | 1.1 | IM |
| B36 | 58.13 | 410.65 | 234.39 | 408.59 | 48.99 | 228.79 | 1.024 | 2.3 | IM |
| B37 | 411.41 | 340.43 | 375.92 | 197.95 | 246.22 | 222.085 | 1.693 | 6.0 | PM |
| B38 | 221.47 | 115.08 | 168.275 | 112.88 | 330.71 | 221.795 | 0.759 | 0.8 | EM |
| B39 | 227.66 | 425.75 | 326.705 | 493.27 | 458.84 | 476.055 | 0.686 | 0.4 | EM |
| B40 | 531.56 | 638.09 | 584.825 | 918.8 | 896.79 | 907.795 | 0.644 | 0.1 | EM |
| B41 | 319.15 | 348.87 | 334.01 | 475.44 | 440.12 | 457.78 | 0.730 | 0.6 | EM |
| B42 | 545.78 | 214.06 | 379.92 | 271.06 | 975.84 | 623.45 | 0.609 | −0.1 | UM |
| B43 | 516.71 | 20.51 | 268.61 | 217.34 | 223.73 | 220.535 | 1.218 | 3.3 | IM |
| B44 | 826.68 | 437.48 | 632.08 | 830.16 | 824.18 | 827.17 | 0.764 | 0.8 | EM |
| B45 | 225.46 | 243.94 | 234.7 | 414.64 | 183.17 | 298.905 | 0.785 | 0.9 | EM |
| B46 | 918.42 | 418.19 | 668.305 | 781.55 | 686.04 | 733.795 | 0.911 | 1.6 | IM |
| B47 | 465.64 | 460.39 | 463.015 | 602.34 | 761.51 | 681.925 | 0.679 | 0.3 | EM |
| B48 | 155.53 | 1181.96 | 668.745 | 903.52 | 1052.92 | 978.22 | 0.684 | 0.4 | EM |
| B49 | 1181.96 | 1174.26 | 1178.11 | 1652.92 | 1865.01 | 1758.965 | 0.670 | 0.3 | EM |
| B50 | 1188.29 | 177.15 | 682.72 | 390.64 | 1545.7 | 968.17 | 0.705 | 0.5 | EM |

Table 1 Caption: HLPC Plasma Concentration of Efavirenz

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/263f/8800862/2b7af902de0e/PGPM-15-45-g0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8800862_PGPM-15-45-g0001.jpg)

A plot of peak area ratio versus concentration: calibration curve for efavirenz in blank plasma. Standard concentrations of efavirenz, 0.5, 1.0, 2.0, 4.0, 8.0, and 16.0 µg/mL were used to plot the calibration curve, with regression coefficient of 0.995, showing the linearity of the curve.

### Table 2.

| S/N | Sample Code | Nature | Allele/Genotype |
| --- | --- | --- | --- |
| 1. | B1 | Homozygous wild type | CYP2B6*1*1 |
| 2. | B2 | Homozygous mutant | CYP2B6*6*6 |
| 3. | B3 | Homozygous mutant | CYP2B6*6*6 |
| 4. | B4 | Heterozygous mutant | CYP2B6*1*6 |
| 5. | B5 | Heterozygous mutant | CYP2B6*1*6 |
| 6. | B6 | Heterozygous mutant | CYP2B6*1*6 |
| 7. | B7 | Homozygous wild type | CYP2B6*1*1 |
| 8. | B8 | Homozygous wild type | CYP2B6*1*1 |
| 9. | B9 | Homozygous wild type | CYP2B6*1*1 |
| 10. | B10 | Heterozygous mutant | CYP2B6*1*6 |
| 11. | B11 | Homozygous wild type | CYP2B6*1*1 |
| 12. | B12 | Heterozygous mutant | CYP2B6*1*6 |
| 13. | B13 | Heterozygous mutant | CYP2B6*1*6 |
| 14. | B14 | Heterozygous mutant | CYP2B6*1*6 |
| 15. | B15 | Homozygous mutant | CYP2B6*6*6 |
| 16. | B16 | Homozygous wild type | CYP2B6*1*1 |
| 17. | B17 | Homozygous mutant | CYP2B6*6*6 |
| 18. | B18 | Heterozygous mutant | CYP2B6*1*6 |
| 19. | B19 | Homozygous wild type | CYP2B6*1*1 |
| 20. | B20 | Homozygous wild type | CYP2B6*1*1 |
| 21. | B21 | Homozygous wild type | CYP2B6*1*1 |
| 22. | B22 | Homozygous wild type | CYP2B6*1*1 |
| 23. | B23 | Heterozygous mutant | CYP2B6*1*6 |
| 24. | B24 | Homozygous wild type | CYP2B6*1*1 |
| 25. | B25 | Homozygous wild type | CYP2B6*1*1 |
| 26. | B26 | Homozygous wild type | CYP2B6*1*1 |
| 27. | B27 | Homozygous wild type | CYP2B6*1*1 |
| 28. | B28 | Homozygous wild type | CYP2B6*1*1 |
| 29. | B29 | Heterozygous mutant | CYP2B6*1*6 |
| 30. | B30 | Homozygous mutant | CYP2B6*6*6 |
| 31. | B31 | Homozygous wild type | CYP2B6*1*1 |
| 32. | B32 | Heterozygous mutant | CYP2B6*1*6 |
| 33. | B33 | Homozygous wild type | CYP2B6*1*1 |
| 34. | B34 | Homozygous wild type | CYP2B6*1*1 |
| 35. | B35 | Heterozygous mutant | CYP2B6*1*6 |
| 36. | B36 | Homozygous wild type | CYP2B6*1*1 |
| 37. | B37 | Heterozygous mutant | CYP2B6*1*6 |
| 38. | B38 | Heterozygous mutant | CYP2B6*1*6 |
| 39. | B39 | Homozygous wild type | CYP2B6*1*1 |
| 40. | B40 | Heterozygous mutant | CYP2B6*1*6 |
| 41. | B41 | Homozygous wild type | CYP2B6*1*1 |
| 42. | B42 | Heterozygous mutant | CYP2B6*1*6 |
| 43. | B43 | Homozygous wild type | CYP2B6*1*1 |
| 44. | B44 | Homozygous wild type | CYP2B6*1*1 |
| 45. | B45 | Heterozygous mutant | CYP2B6*1*6 |
| 46. | B46 | Heterozygous mutant | CYP2B6*1*6 |
| 47. | B47 | Homozygous wild type | CYP2B6*1*1 |
| 48. | B48 | Heterozygous mutant | CYP2B6*1*6 |
| 49. | B49 | Heterozygous mutant | CYP2B6*1*6 |
| 50. | B50 | Homozygous wild type | CYP2B6*1*1 |

Table 2 Caption: Restriction Fragment Length Polymorphism Products and Suspected Alleles (516G>T SNP)

### Table 3.

| Phenotype | Male (%) | Female (%) | Total (%) | Plasma Conc (mg/L) |
| --- | --- | --- | --- | --- |
| UM | 4 (8) | 2 (4) | 6 (12) | <0.1 |
| EM | 6 (12) | 16 (32) | 22 (44) | 0.1–0.9 |
| IM | 7 (14) | 12 (24) | 19 (38) | 1.0–3.9 |
| PM | 0 (0.0) | 3 (6) | 3 (6) | 4.0–6.0 |
| N | 15 | 35 | 50 (100) |   |

Table 3 Caption: Gender and Phenotypic Distribution of Polymorphic CYP2B6*6 Allele in Niger Delta Population [n = 50]

All the extracted DNA products showed at 500bp after subjecting them to polymerase chain reaction (PCR) at 94°C, 54°C, and 72°C, for initial denaturation, primer annealing, and extension, respectively, using Taq polymerases enzyme and 100bp DNA molecular ladder as shown in [Figures 2](#f0002) and [3](#f0003). This indicates that, all the participants had the target gene to encode for the enzyme (CYP2B6) responsible for metabolizing the substrate of choice (efavirenz). The PCR products (amplicons) were digested with restriction endonuclease enzyme (*Bsr*l) and subjected to restriction fragment length polymorphism (RFLP) on agarose gel electrophoretic medium. After the digestion, the RFLP results showed more than half of the population (56%) with a homozygous wild-type gene with CYP2B6*1*1 allele. Individuals with CYP2B6*1*1 allele are believed to metabolize the drug very extensively or rapidly, therefore may not have serious adverse events as regards to HAART regimen, also 38% were CYP2B6*1*6 (heterozygous mutant) allele ([Table 2](#t0002)). These category of patients might not extensively handle the drug (intermediate), thus might have occasional adverse reactions, especially in cases of high concentration >4m g/L. Finally, 6% had homozygous mutant gene (CYP2B6*6*6 allele); these patients are prone to experience frequent adverse drug reactions due to poor metabolism of the drug ([Table 4](#t0004)). This is in line with a study conducted by García-González et al[23](#cit0023), they reported that individual drug-metabolizing activity for corresponding drug substrates varies from poor, intermediate, and to wild-type activity (extensive metabolizer), and that increased enzyme expression in the case of gene duplication (rapid metabolizer phenotype).

### Table 4.

| Males (%) | Females (%) | Total (%) | Allele | Genotype |
| --- | --- | --- | --- | --- |
| 10 (20) | 18 (36) | 28 (56) | CYP2B6*1*1 | Homozygous wild type |
| 7 (14) | 12 (24) | 19 (38) | CYP2B6*1*6 | Heterozygous mutant |
| 0 (0.0) | 3 (6) | 3 (6) | CYP2B6*6*6 | Homozygous mutant |
| 15 | 35 | 50 (100) |   |   |

Table 4 Caption: Genotypes and Alleles Obtained from Restriction Fragment Length Polymorphism Analysis

Several factors can affect the way an individual metabolizes drugs, one of such factors seen in this was gender difference. Out of the 15 male subjects among the 50 patients that participated in the study, 8% were ultra-rapid metabolizers, 12% extensive metabolizers, 14% intermediate metabolizers while no poor metabolizer was observed, following the drug concentration quantified from their plasma. On the contrary, out of the 35 females that participated in the study, from the pool of 50 patients, 4% were observed as ultra-rapid metabolizers, 32% extensive, 24% intermediate metabolizers, while 6% were poor metabolizers ([Figure 4](#f0004)). This indicates that the drug was more poorly metabolized by the female population than that of their male counterparts, comparing the number of participants (35:15). Hence, this calls for individualization of therapy based on gender differences when prescribing medication, especially genotypic individualized-based dosage prescription as regards to HAART regimen, etc. Poor metabolizers (PMs) may need down regulation of dosage strength, as the drug is likely to accumulate in the system, the intermediate metabolizers (IMs) need no adjustment in the dose but extensive (EMs) and ultra-rapid metabolizers may require the usual or higher dose, as the normal therapeutic level of the drug will be rapidly cleared off the system, this form the basis of pharmacogenomics (modern precision or individualized medicine). This was also reported by Nemaura et al[24](#cit0024), that gender, weight, and 516G>T genotypes are important determinants on whether a patient will have efavirenz concentrations within the therapeutic range or above 4 mμg/L. This will definitely enhance and enable the healthcare team to manage these patients by conducting pharmacogenetic testing and provide modern precision medicine (personalized, individualized treatment regimens based on their genetic variations), to optimize medication (antiretrovirals) regimens and thereby improving the quality of life, as well reducing unprecedented adverse drug reactions and expenditures.[3](#cit0003)^,^[25](#cit0025)

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/263f/8800862/8ef8b4a5b003/PGPM-15-45-g0004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8800862_PGPM-15-45-g0004.jpg)

Demography of study population. This figure depicts the participant’s age group and gender that were enrolled in the study. It was observed that majority of the participants were female (70%) and were between the age of 30–49 years (76%).

One of the major limitation of the study was the inability to conduct DNA sequencing in a bid to ascertain the actual strand or mutation of the amino acid sequence from the patients. However, Mehlotra et al[26](#cit0026) reported the *CYP2B6* 983T>C polymorphism in West African countries and found a high prevalence of *CYP2B6* functional polymorphism of 983T>C in Papua New Guinea, where HIV/AIDS is a significant health problem. This is very similar to the findings of the current study on the prevalence of *CYP2B6* polymorphism in the Niger Delta, South of Nigeria.

## Conclusion

There was no significant difference (p ≤ 0.05) between genotype and phenotype data for CYP2B6 polymorphism, among the HIV/AIDS patients that participated in this study. Based on genotype data, there was more (58%) homozygous wild-type gene (CYP2B6*1*6 alleles), while the other participants (38%) exhibited heterozygous mutant gene (CYP2B6*1*6, 40%) and homozygous mutant gene (CYP2B6*6*6, 6%), respectively. Following the different phenotypes obtained from the phenotypic analysis, genetic polymorphism of the *CYP2B6* gene is prevalent among HIV/AIDs patients in the Niger Delta ethnic population undergoing HAART regimen (efavirenz-based HAART treatment), as the population having homozygous mutant gene or PM are >1% (6%). Also, gender differences could affect drug metabolism as no PM was observed among the male participants, whereas 6% of the female participants were PMs.

## Acknowledgment

The authors sincerely acknowledge the staff of the Department of Pharmaceutical and Medicinal Chemistry, Faculty of Pharmacy, Niger Delta University, Wilberforce Island; Federal Medical Centre, Yenagoa, Bayelsa State, for granting ethical approval and provision of facility to conduct the study, and the Department of Medical Laboratory Sciences, Faculty of Basic Medical Sciences, College of Health Sciences, Niger Delta University.

## Ethical Issues

This research was carried out on approval by the research and ethics committee of the Federal Medical Centre Yenagoa, Bayelsa State, Nigeria. The study complies with the Declaration of Helsinki (Adopted by the 18th WMA General Assembly, Helsinki, Finland, June 1964 and amended by the 64th WMA General Assembly, Fortaleza, Brazil, October 2013), in terms of research protocol approval, obtaining of participants informed consent, precautions taken to protect the privacy of subjects and the confidentiality of their personal information, and proper education on the methods to be employed and they were allowed to make independent choices.

## Disclosure

The authors declare no conflicts of interest in this work.

## References

1. Oetjens MT, Denny JC, Ritchie MD. Assessment of a pharmacogenomics marker panel in a polypharmacy population identified from electronic medical records. Pharmacogenomics. 2013;14(7):735–744. doi: 10.2217/pgs.13.64  [DOI](https://doi.org/10.2217/pgs.13.64) | [PMC free article](/articles/PMC3725600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23651022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Assessment%20of%20a%20pharmacogenomics%20marker%20panel%20in%20a%20polypharmacy%20population%20identified%20from%20electronic%20medical%20records&author=MT%20Oetjens&author=JC%20Denny&author=MD%20Ritchie&volume=14&issue=7&publication_year=2013&pages=735-744&pmid=23651022&doi=10.2217/pgs.13.64&)

2. Leeder JS. Pharmacogenetics and pharmacogenomics. Pediatr Clin North Am. 2001;48(3):765–781. doi: 10.1016/S0031-3955(05)70338-2  [DOI](https://doi.org/10.1016/S0031-3955(05)70338-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11411304/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr%20Clin%20North%20Am&title=Pharmacogenetics%20and%20pharmacogenomics&author=JS%20Leeder&volume=48&issue=3&publication_year=2001&pages=765-781&pmid=11411304&doi=10.1016/S0031-3955(05)70338-2&)

3. Li J, Bluth MH. Pharmacogenomics of drug-metabolizing enzymes and transporters: implications for cancer therapy. Pharmgenomics Pers Med. 2011;4:11–33. doi: 10.2147/PGPM.S18861  [DOI](https://doi.org/10.2147/PGPM.S18861) | [PMC free article](/articles/PMC3513217/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23226051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenomics%20Pers%20Med&title=Pharmacogenomics%20of%20drug-metabolizing%20enzymes%20and%20transporters:%20implications%20for%20cancer%20therapy&author=J%20Li&author=MH%20Bluth&volume=4&publication_year=2011&pages=11-33&pmid=23226051&doi=10.2147/PGPM.S18861&)

4. Wijnen PA, Op den Buijsch RA, Drent M. Review article: the prevalence and clinical relevance of cytochrome P450 polymorphisms. Aliment Pharmacol Ther. 2007;26(Suppl 2):211–219. doi: 10.1111/j.1365-2036.2007.03490.x  [DOI](https://doi.org/10.1111/j.1365-2036.2007.03490.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18081664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aliment%20Pharmacol%20Ther&title=Review%20article:%20the%20prevalence%20and%20clinical%20relevance%20of%20cytochrome%20P450%20polymorphisms&author=PA%20Wijnen&author=RA%20Op%20den%20Buijsch&author=M%20Drent&volume=26&issue=Suppl%202&publication_year=2007&pages=211-219&pmid=18081664&doi=10.1111/j.1365-2036.2007.03490.x&)

5. Smith BT, editor. Concepts in Immunology and Immunotherapeutics. 4th ed. Bethesda, MD: American Society of Health-System Pharmacists; 2008:143.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Concepts%20in%20Immunology%20and%20Immunotherapeutics&author=BT%20Smith&publication_year=2008&)

6. Barret J, Joshi AS, Chai M, Ludden TM, Fiske WD, Pieniaszek HJ. Population pharmacokinetic meta-analysis with efavirenz. Int J Clin Pharmacol Ther. 2002;40:507–519. doi: 10.5414/CPP40507  [DOI](https://doi.org/10.5414/CPP40507) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12698988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Pharmacol%20Ther&title=Population%20pharmacokinetic%20meta-analysis%20with%20efavirenz&author=J%20Barret&author=AS%20Joshi&author=M%20Chai&author=TM%20Ludden&author=WD%20Fiske&volume=40&publication_year=2002&pages=507-519&pmid=12698988&doi=10.5414/CPP40507&)

7. Marzolini C, Telenti A, Descosterd LA, Grueb G, Biolazz J, Buclin T. Efavirenz plasma level can predict treatment failure and central nervous system side effects in HIV-1 infected patients. AIDS. 2001;5:71–75. doi: 10.1097/00002030-200101050-00011  [DOI](https://doi.org/10.1097/00002030-200101050-00011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11192870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AIDS&title=Efavirenz%20plasma%20level%20can%20predict%20treatment%20failure%20and%20central%20nervous%20system%20side%20effects%20in%20HIV-1%20infected%20patients&author=C%20Marzolini&author=A%20Telenti&author=LA%20Descosterd&author=G%20Grueb&author=J%20Biolazz&volume=5&publication_year=2001&pages=71-75&pmid=11192870&doi=10.1097/00002030-200101050-00011&)

8. Klein K, Lang T, Saussele T, et al. Genetic variability of CYP2B6 in populations of Africa and Asian origin: Allele frequencies, novel functional variations and possible implications for anti-HIV therapy with efavirenz. Pharmacogent Genomics. 2005;15:861–873. doi: 10.1097/01213011-200512000-00004  [DOI](https://doi.org/10.1097/01213011-200512000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16272958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogent%20Genomics&title=Genetic%20variability%20of%20CYP2B6%20in%20populations%20of%20Africa%20and%20Asian%20origin:%20Allele%20frequencies,%20novel%20functional%20variations%20and%20possible%20implications%20for%20anti-HIV%20therapy%20with%20efavirenz&author=K%20Klein&author=T%20Lang&author=T%20Saussele&volume=15&publication_year=2005&pages=861-873&pmid=16272958&doi=10.1097/01213011-200512000-00004&)

9. Ebeshi BU, Bolaji OO, Masimirembwa CM. Cytochrome P450 2D6 (CYP2D6) genotype and phenotype determination in the Nigeria population. Asian J Pharm Health Sci. 2011;1(2):47–54.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asian%20J%20Pharm%20Health%20Sci&title=Cytochrome%20P450%202D6%20(CYP2D6)%20genotype%20and%20phenotype%20determination%20in%20the%20Nigeria%20population&author=BU%20Ebeshi&author=OO%20Bolaji&author=CM%20Masimirembwa&volume=1&issue=2&publication_year=2011&pages=47-54&)

10. Padh H. Pharmacogenetics: polymorphism and genotype-phenotype correlation of drug response in Indian population. Mol Cytogenet. 2014;7:I52. doi: 10.1186/1755-8166-7-S1-I52  [DOI](https://doi.org/10.1186/1755-8166-7-S1-I52) | [PMC free article](/articles/PMC4044520/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24949101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cytogenet&title=Pharmacogenetics:%20polymorphism%20and%20genotype-phenotype%20correlation%20of%20drug%20response%20in%20Indian%20population&author=H%20Padh&volume=7&publication_year=2014&pages=I52&pmid=24949101&doi=10.1186/1755-8166-7-S1-I52&)

11. UNAIDS. Fact sheet; Latest statistics on the status of AIDS epidemic [UNAIDS]. Available from: www.unaids.org. Accessed March 16, 2018.  [www.unaids.org](http://www.unaids.org)

12. WHO. Guideline on when to start antiretroviral therapy and on pre-exposure prophylaxis for HIV. Geneva: World Health Organization; 2015. Available from: http://www.who.int/hiv/pub/guidelines/earlyrelease-arv/en. Accessed January 6, 2022.  [http://www.who.int/hiv/pub/guidelines/earlyrelease-arv/en](http://www.who.int/hiv/pub/guidelines/earlyrelease-arv/en) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26598776/) | [Google Scholar](https://scholar.google.com/scholar_lookup?WHO.%20Guideline%20on%20when%20to%20start%20antiretroviral%20therapy%20and%20on%20pre-exposure%20prophylaxis%20for%20HIV.%20Geneva:%20World%20Health%20Organization;%202015.%20Available%20from:%20http://www.who.int/hiv/pub/guidelines/earlyrelease-arv/en.%20Accessed%20January%206,%202022.)

13. Csajka C, Marzolini C, Fattinger K, Decosterd LA, Fellay J, Telenti A. Population pharmacokinetics and effects of efavirenz in patients with human immunodeficiency virus infection. Clin Pharmacol Ther. 2003;73:20–30. doi: 10.1067/mcp.2003.22  [DOI](https://doi.org/10.1067/mcp.2003.22) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12545140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Population%20pharmacokinetics%20and%20effects%20of%20efavirenz%20in%20patients%20with%20human%20immunodeficiency%20virus%20infection&author=C%20Csajka&author=C%20Marzolini&author=K%20Fattinger&author=LA%20Decosterd&author=J%20Fellay&volume=73&publication_year=2003&pages=20-30&pmid=12545140&doi=10.1067/mcp.2003.22&)

14. Saiki R, Scharf S, Faloona F, et al. Enzymatic amplification of beta-globin genomic sequences and restriction site analysis for diagnosis of sickle cell anemia. Science. 1985;230(4732):1350–1354. doi: 10.1126/science.2999980  [DOI](https://doi.org/10.1126/science.2999980) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2999980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Enzymatic%20amplification%20of%20beta-globin%20genomic%20sequences%20and%20restriction%20site%20analysis%20for%20diagnosis%20of%20sickle%20cell%20anemia&author=R%20Saiki&author=S%20Scharf&author=F%20Faloona&volume=230&issue=4732&publication_year=1985&pages=1350-1354&pmid=2999980&doi=10.1126/science.2999980&)

15. National Centre for Biotechnology Information (NCBI). Polymorphism, Restriction Fragment Length [MAJR] in PubMed. 8600 Rockville Pike, Bethesda MD, 20894 USA: U.S. National Library of Medicine; 2017.  [Google Scholar](https://scholar.google.com/scholar_lookup?National%20Centre%20for%20Biotechnology%20Information%20(NCBI).%20Polymorphism,%20Restriction%20Fragment%20Length%20%5BMAJR%5D%20in%20PubMed.%208600%20Rockville%20Pike,%20Bethesda%20MD,%2020894%20USA:%20U.S.%20National%20Library%20of%20Medicine;%202017.)

16. Clark J. High-Performance Liquid Chromatography - HPLC. Available from: https://www.chemguide.co.uk/analysis/chromatography/hplc.html. Accessed January 6, 2022. 2016.  [https://www.chemguide.co.uk/analysis/chromatography/hplc.html](https://www.chemguide.co.uk/analysis/chromatography/hplc.html)

17. Bunu SJ, Otele D, Alade T, Dodoru RT. Determination of serum DNA purity among patients undergoing antiretroviral therapy using NanoDrop-1000 spectrophotometer and polymerase chain reaction. Biomed Biotechnol Res J. 2020;4:214–219. doi: 10.4103/bbrj.bbrj_68_20  [DOI](https://doi.org/10.4103/bbrj.bbrj_68_20) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Biotechnol%20Res%20J&title=Determination%20of%20serum%20DNA%20purity%20among%20patients%20undergoing%20antiretroviral%20therapy%20using%20NanoDrop-1000%20spectrophotometer%20and%20polymerase%20chain%20reaction&author=SJ%20Bunu&author=D%20Otele&author=T%20Alade&author=RT%20Dodoru&volume=4&publication_year=2020&pages=214-219&doi=10.4103/bbrj.bbrj_68_20&)

18. Rotger M, Colombo S, Furrer H, Bleiber G, Buclin T, Lee BL. Influence of CYP2B6 polymorphism on plasma and intracellular concentrations and toxicity of efavirenz and nevirapine in HIV-infected patients. Pharmacogenet Genomics. 2005;15:1–5. doi: 10.1097/01213011-200501000-00001  [DOI](https://doi.org/10.1097/01213011-200501000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15864119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Influence%20of%20CYP2B6%20polymorphism%20on%20plasma%20and%20intracellular%20concentrations%20and%20toxicity%20of%20efavirenz%20and%20nevirapine%20in%20HIV-infected%20patients&author=M%20Rotger&author=S%20Colombo&author=H%20Furrer&author=G%20Bleiber&author=T%20Buclin&volume=15&publication_year=2005&pages=1-5&pmid=15864119&doi=10.1097/01213011-200501000-00001&)

19. National Centre for Biotechnology Information (NCBI) databases. Available from: http://blast.ncbi.nlm.nih.gov/Blast.cgi. Accessed August 25, 2021.  [http://blast.ncbi.nlm.nih.gov/Blast.cgi](http://blast.ncbi.nlm.nih.gov/Blast.cgi)

20. Aryal S. Polymerase chain reaction (PCR), principle, procedure, types, applications, and animation. Microbiology info.com; [updated July 6, 2018].  [Google Scholar](https://scholar.google.com/scholar_lookup?Aryal%20S.%20Polymerase%20chain%20reaction%20(PCR),%20principle,%20procedure,%20types,%20applications,%20and%20animation.%20Microbiology%20info.com;%20%5Bupdated%20July%206,%202018%5D.)

21. Ebeshi BU, Bolaji OO, Oluka M, Edebi VN, Soyinka JO, Guantai A. Simple reversed-phase high-performance liquid chromatographic estimation of the antiretroviral agent Efavirenz from human plasma. Br J Pharm Res. 2014;4(1):145–157. doi: 10.9734/BJPR/2014/4123  [DOI](https://doi.org/10.9734/BJPR/2014/4123) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharm%20Res&title=Simple%20reversed-phase%20high-performance%20liquid%20chromatographic%20estimation%20of%20the%20antiretroviral%20agent%20Efavirenz%20from%20human%20plasma&author=BU%20Ebeshi&author=OO%20Bolaji&author=M%20Oluka&author=VN%20Edebi&author=JO%20Soyinka&volume=4&issue=1&publication_year=2014&pages=145-157&doi=10.9734/BJPR/2014/4123&)

22. Deshpande N, Sharanya V, Murthy HV, et al. Rapid and ultra-rapid metabolizers with CYP2C19*17 polymorphism do not respond to standard therapy with proton pump inhibitors. Meta Gene. 2016;9:159–164. doi: 10.1016/j.mgene.2016.06.004  [DOI](https://doi.org/10.1016/j.mgene.2016.06.004) | [PMC free article](/articles/PMC4932617/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27419077/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Meta%20Gene&title=Rapid%20and%20ultra-rapid%20metabolizers%20with%20CYP2C19*17%20polymorphism%20do%20not%20respond%20to%20standard%20therapy%20with%20proton%20pump%20inhibitors&author=N%20Deshpande&author=V%20Sharanya&author=HV%20Murthy&volume=9&publication_year=2016&pages=159-164&pmid=27419077&doi=10.1016/j.mgene.2016.06.004&)

23. García-González X, Cabaleiro T, Herrero MJ, McLeod H, López-Fernández LA. Clinical implementation of pharmacogenetics. Drug Metab Pers Ther. 2016;31(1):9–16. doi: 10.1515/dmpt-2015-0031  [DOI](https://doi.org/10.1515/dmpt-2015-0031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26751902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pers%20Ther&title=Clinical%20implementation%20of%20pharmacogenetics&author=X%20Garc%C3%ADa-Gonz%C3%A1lez&author=T%20Cabaleiro&author=MJ%20Herrero&author=H%20McLeod&author=LA%20L%C3%B3pez-Fern%C3%A1ndez&volume=31&issue=1&publication_year=2016&pages=9-16&pmid=26751902&doi=10.1515/dmpt-2015-0031&)

24. Nemaura T, Nhachi C, Masimirembwa C. Impact of gender, weight and CYP2B6 genotype on efavirenz exposure in patients on HIV/AIDS and TB treatment: implications for individualizing therapy. Afr J Pharm Pharmacol. 2012;6(29):2188–2193.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Afr%20J%20Pharm%20Pharmacol&title=Impact%20of%20gender,%20weight%20and%20CYP2B6%20genotype%20on%20efavirenz%20exposure%20in%20patients%20on%20HIV/AIDS%20and%20TB%20treatment:%20implications%20for%20individualizing%20therapy&author=T%20Nemaura&author=C%20Nhachi&author=C%20Masimirembwa&volume=6&issue=29&publication_year=2012&pages=2188-2193&)

25. Thelma A, Samuel BJ. Genetic testing and non-chemotherapeutic management of prostate cancer. Int J Pharm Sci Drug Anal. 2021;1(1):12–17.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Pharm%20Sci%20Drug%20Anal&title=Genetic%20testing%20and%20non-chemotherapeutic%20management%20of%20prostate%20cancer&author=A%20Thelma&author=BJ%20Samuel&volume=1&issue=1&publication_year=2021&pages=12-17&)

26. Mehlotra RK, Bockarie MJ, Zimmerman PA. CYP2B6 983T>C polymorphism is prevalent in West Africa but absent in Papua New Guinea: implications for HIV/AIDS treatment. Br J Clin Pharmacol. 2007;64(3):391–395. doi: 10.1111/j.1365-2125.2007.02884.x  [DOI](https://doi.org/10.1111/j.1365-2125.2007.02884.x) | [PMC free article](/articles/PMC2000644/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17391322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=CYP2B6%20983T>C%20polymorphism%20is%20prevalent%20in%20West%20Africa%20but%20absent%20in%20Papua%20New%20Guinea:%20implications%20for%20HIV/AIDS%20treatment&author=RK%20Mehlotra&author=MJ%20Bockarie&author=PA%20Zimmerman&volume=64&issue=3&publication_year=2007&pages=391-395&pmid=17391322&doi=10.1111/j.1365-2125.2007.02884.x&)
