# Angiotensin System Polymorphisms’ in SARS-CoV-2 Positive Patients: Assessment Between Symptomatic and Asymptomatic Patients: A Pilot Study

## Metadata
**Authors:** Concetta Cafiero, Felice Rosapepe, Raffaele Palmirotta, Agnese Re, Maria Pia Ottaiano, Giulio Benincasa, Romina Perone, Elisa Varriale, Gerardo D’Amato, Andrea Cacciamani, Alessandra Micera, Salvatore Pisconti
**Journal:** Pharmacogenomics and Personalized Medicine
**Date:** 2021 May 27
**DOI:** [10.2147/PGPM.S303666](https://doi.org/10.2147/PGPM.S303666)
**PMID:** 34079337
**PMCID:** PMC8166347
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8166347/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8166347/pdf/pgpm-14-621.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8166347/pdf/pgpm-14-621.pdf)

## Abstract

**Introduction:** 
The renin–angiotensin–aldosterone system (RAAS), a metabolic cascade regulating pressure and circulating blood volume, has been considered the main system involved in the pathogenesis of severe lung injury and organs decline in COVID-19 patients. The angiotensin I-converting enzyme (ACE1), angiotensin-converting enzyme 2 (ACE2), angiotensinogen (AGT) and receptors angiotensin II receptor type 1 (AGTR1) are key factors for SARS-CoV-2 entering in the cells, sodium and water retention with an increase blood pressure, promotion of fibrotic and inflammatory phenomena resulting in a cytokine storm.

**Methods:** 
In this pilot study, the frequencies of six polymorphisms in the ACE1, ACE2, AGT and AGTR1 genes were analysed in symptomatic patients affected by COVID-19 and compared with the results obtained from asymptomatic subjects.

**Results:** 
Thus, we have identified that rs2074192 (ACE2), rs1799752 (ACE1) and rs699 (AGT) SNPs could potentially be a valuable tool for predicting the clinical outcome of SARS-CoV-2 infected patients. A genetic predisposition may be prospected for severe internal organ damages and poor prognosis in patients with COVID-19 disease, as observed in symptomatic vs asymptomatic.

**Conclusion:** 
This study provides evidence that analysis of RAAS polymorphisms could be considered the key point in understanding and predicting the SARS-CoV-2 course infection.

Keywords: ACE, AGT, AGTR1, RAAS, polymorphisms, SARS-CoV-2, asymptomatic, COVID-19

### Introduction

The renin–angiotensin–aldosterone system (RAAS), a metabolic cascade regulating pressure and circulating blood volume, has been considered the main system involved in the pathogenesis of severe lung injury and organs decline in COVID-19 patients. The angiotensin I-converting enzyme (*ACE1*), angiotensin-converting enzyme 2 (*ACE2*), angiotensinogen (AGT) and receptors angiotensin II receptor type 1 (*AGTR1*) are key factors for SARS-CoV-2 entering in the cells, sodium and water retention with an increase blood pressure, promotion of fibrotic and inflammatory phenomena resulting in a cytokine storm.

### Methods

In this pilot study, the frequencies of six polymorphisms in the *ACE1, ACE2, AGT* and *AGTR1* genes were analysed in symptomatic patients affected by COVID-19 and compared with the results obtained from asymptomatic subjects.

### Results

Thus, we have identified that rs2074192 (*ACE2*), rs1799752 (*ACE1*) and rs699 (*AGT*) SNPs could potentially be a valuable tool for predicting the clinical outcome of SARS-CoV-2 infected patients. A genetic predisposition may be prospected for severe internal organ damages and poor prognosis in patients with COVID-19 disease, as observed in symptomatic vs asymptomatic.

### Conclusion

This study provides evidence that analysis of RAAS polymorphisms could be considered the key point in understanding and predicting the SARS-CoV-2 course infection.

## Introduction

Over the last two decades, seven coronaviruses responsible for respiratory diseases in humans have been studied and deeply analyzed for their major lung and multi-organ damages (adverse myocardial condition, cardiomyopathy and kidney failures), including the SARS-CoV-2 responsible for COVID-19 pandemic.[1–3](#cit0001) The clinical spectrum of SARS-CoV-2 infection varies from the most frequent asymptomatic or mild symptomatic forms to severe progressive pneumonia and death, with specific virulence factors to achieve mortality rates around 3%.[4](#cit0004)

In some cases, after an asymptomatic period, a sudden and inexplicable worsening of clinical conditions is observed.[5](#cit0005)

According to current knowledge, the main cause of death in COVID-19 patients is a refractory acute respiratory distress syndrome (ARDS) secondary to SARS-CoV-2 pneumonia,[3](#cit0003) but the pathophysiology of COVID-19 remains partly unknown with the resulting lack of effective treatments for these patients.

Based on the observation of the high similarity of SARS-CoV-2 with SARS-CoV-1, a recent study suggests the hypothesis that also during COVID-19 many genes can be downregulated leading to immune system hyperactivation, induction of signaling pathways and a subsequent cytokine storm.[6](#cit0006) Several studies reported that SARS-CoV-2 is highly selective towards the angiotensin system.[7](#cit0007) In fact, it has been ascertained that the virus binds strictly the *ACE2*-expressing cells widely distributed in blood vessels and tissues/organs (lungs, heart, kidney and eye), a condition that explains the SARS-CoV-2 widespreading.[8](#cit0008)^,^[9](#cit0009) Furthermore, recent data obtained by bioinformatic approaches have shown that the presence of an ACE2-Neprilysin-carbonic anhydrase complex in most of vital organs and as a receptor of COVID-19 could be the basis of multi-organ damages caused by SARS-CoV-2.[10](#cit0010) This ability explains the observation of hypertension, diabetes and cardiovascular conditions as frequent comorbidities in COVID-19 disease.[11](#cit0011)

The ACE2 protein is a well-known carboxypeptidase significantly involved in RAAS, essential in regulating blood pressure and fluid homeostasis as well as electrolytes, with functional influences on many organs such as blood vessels, heart, kidneys and eyes.[12](#cit0012) The whole angiotensin system, including the angiotensin I-converting enzyme (*ACE1*), the angiotensin-converting enzyme 2 (*ACE2*), the angiotensinogen (*AGT*) and the receptors angiotensin II receptor type 1 (*AGTR1*) were identified as the key factors for SARS-CoV-2 in entering into the cells, tissues and organs.[13](#cit0013) Based on our previous study concerning the potential application of a personalized approach in the selection of therapy for COVID patients,[14](#cit0014) we decided to evaluate whether six different Single Nucleotide polymorphisms (SNPs), pharmacogenetically relevant belonging to *ACE1, ACE2, AGT* and *AGTR1* genes or their haplotype combinations, could be predictive of susceptibility to SARS-CoV-2 infection or could be related to the severity of disease. For this reason, we performed a pilot study analyzing the rs2074192 and rs2106809 (*ACE2* gene), rs1799752 (*ACE1* gene), rs4762 and rs699 (*AGT* gene) and rs5186 (*AGTR1* gene) genetic variants in a selected population with SARS-CoV-2 infection clinically divided into asymptomatic and symptomatic groups.

## Methods

### Patients

The pilot study was performed according to Institutional guidelines and approved by the Azienda Sanitaria Locale Brindisi Ethical Committee, Brindisi, Italy (Prot n. R.CE 30/21). All procedures followed the principles embodied in the Declaration of Helsinki and written informed consent was obtained from all patients voluntarily joined the study.

The study population included a total of 104 patients, 54 symptomatic (19/35 F/M, mean age 68.00 ± 12.38 yrs., range 40–90 yrs) and 50 asymptomatic (27/23 F/M, mean age 45.00 ± 11.67 yrs., range 22–62 yrs) consecutively enrolled according to inclusion criteria (test positivity and RX imaging). In the symptomatic patient group, the recovered clinical information included the severity of respiratory compromise, dividing into two groups with high and intermediate intensity of care (40% and 60%, respectively), obesity (45%), hypertension (62%), diabetes (57%) and the presence of conjunctivitis (42%).

### DNA Extraction and Genotyping

Five milliliters of peripheral blood were collected in EDTA-vacutainer tubes and DNA was extracted by using the Extra kit DNA (salting out); Dia-Chem srl., Naples, Italy, according to the manufacturers’ procedure. DNA concentration was measured by NanoDrop (Thermo Fisher Scientific, Waltham, MA, USA) and samples with A260/280 ≥1.8 were considered appropriate for analysis.[15](#cit0015)

All patients were analyzed for six genetic polymorphisms: rs2074192 (*ACE2*, c.*1860-449C>T, intron variant), rs2106809 (*ACE2*, c.*264+788T>C, intron variant), rs1799752 (*ACE1*, c.2306–117_2306-116ins-del, intron variant), rs4762 (*AGT*, c.620C>T, p.Thr207Met), rs699 (*AGT*, c.803T>C, p.Met268Thr) and rs5186 (*AGTR1*, c.*86A>C, 3 Prime UTR Variant 1166A-C) using the commercial kits Ampli ACE2 rs2074192, AMPLI ACE2 *rs2106809, AMPLI ACE2 *rs2106809, AMPLI-SET-ACE I/D, AMPLI-SET-AGT T174M, AMPLI-SET-AGT M235T and AMPLI SET AGTR1 A1166C (Dia-Chem) according to the manufacturer's procedures.

### Statistical Analysis

All analyses were performed to assess the possible associations between genotypes’ frequencies and gender, age, clinical variables and asymptomatic or symptomatic status.

Allelic frequencies (%) were estimated by gene counting and genotypes were scored. Data were analysed by using Student’s *t*-test or one-way ANOVA with Bonferroni post-test, as appropriate. Two-sided tests were used for analysis and a P-value≤0.05 was considered statistically significant. All statistical analysis was performed by using GraphPad Prism 5 software (GraphPad Software, La Jolla, CA, USA; [https://www.graphpad.com](https://www.graphpad.com)).

A comparison between our results obtained from the genotyping of our six SNPs analyzed and the allele frequency data available from the 1000 Genomes Browsers (A Deep Catalog of Human Genetic Variation, [https://www.internationalgenome.org](https://www.internationalgenome.org)) and from the Genome Aggregation Database v3.1 (GnomAD, [https://gnomad.broadinstitute.org](https://gnomad.broadinstitute.org))[16](#cit0016)^,^[17](#cit0017) was performed. In the case of rs1799752 *ACE1* I/D polymorphism, for which frequency data are not available, we used our mutational data obtained in a previous study on 825 Italian subjects.[18](#cit0018)

The frequencies of each SNPs genotype were compared with those expected for a population in Hardy–Weinberg equilibrium (HWE). For rs2074192 and rs2106809, the HWE was analyzed separately for females and males due to the localization of the *ACE* gene on X chromosome. The significance of the differences of observed alleles and genotypes, haplotype frequencies and associations between groups as well as analysis of multiple inheritance models (codominant, dominant, recessive, over dominant, and log-additive) were tested using free web-based applications SNPStats software ([http://bioinfo.iconcologia.net/snpstats/start.htm](http://bioinfo.iconcologia.net/snpstats/start.htm)) and SHEsis software ([http://analysis.bio-x.cn/myAnalysis.php](http://analysis.bio-x.cn/myAnalysis.php)).[19–21](#cit0019)

## Results

### Comparison of Allelic Frequencies with World Population Databases

Our study population included 104 Caucasian individuals, as tested positive for SARS-CoV-2 after naso-oropharyngeal swabs and subdivided in 50 asymptomatic subjects and 54 moderate-to-severe COVID-19 patients.

We first performed a comparison of the allele frequencies obtained from our subjects and the data available in the 1000 Genomes Project Phase 3, using global and European population data, and GnomAD genomes v3.1 databases by extrapolating global population ([Table 1](#t0001)). Only rs699 SNP displayed a significant different frequency distribution in our population with respect to global population from the 1000 Genome Project (P<0.00001) and GnomAD genomes (P=0.006877) but no significant difference with the European Population from the 1000 Genome Project (P=0.14).

### Table 1.

|   | Alleles (%) | 1000 Genomes Project Phase 3 Global Population | 1000 Genomes Project Phase 3 European Population | GnomAD Genomes r3.0 Global Population | Total 104 Patients % | P value* |
| --- | --- | --- | --- | --- | --- | --- |
| rs2074192 | C | 63.7 | 57 | 59.8 | 66.3 | 0.77–0.19–0.38 |
| T | 36.3 | 43 | 40.2 | 33.7 |  |  |
| rs2106809 | A | 68.4 | 75 | 80.6 | 78.4 | 0.11–0.62–0.60 |
| G | 31.6 | 25 | 19.4 | 21.6 |  |  |
| rs1799752 | I | 41.4# |  |  | 45.2 | 0.57# |
| D | 58.6# |  |  | 54.8 |  |  |
| rs4762 | C | 89.8 | 87 | 88.9 | 84.6 | 0.28–0.68–0.4 |
| T | 10.2 | 13 | 11.1 | 15.4 |  |  |
| rs699 | T | 29.5 | 59 | 42.2 | 61.6 | <0.00001–0.14–0.006877 |
| C | 70.5 | 41 | 57.8 | 38.4 |  |  |
| rs5186 | A | 88.2 | 73 | 77.3 | 77.9 | 0.06–0.41–0.86 |
| C | 11.8 | 27 | 22.7 | 22.1 |  |  |

Table 1 Caption: Comparison Between Allele Frequencies Obtained from the Genotyping of Our Study and the Allele Frequency Data Available from the 1000 Genomes and GnomAD Databases

### Genotype Association Analysis

Distributions of genotypes and allele frequencies of the analyzed polymorphism observed in our population are listed in [Table 2](#t0002). No significant differences in all subjects were found between genotypes frequencies and gender or age, and no association was identified in the group of symptomatic patients regarding severity of pneumonia, obesity, hypertension, diabetes and presence of conjunctivitis. Conversely, comparing the allele frequencies obtained between asymptomatic subjects and symptomatic patients we found a significant difference for rs2074192 (P=0.001754), rs1799752 (P>0.00001) and rs699 (P=0.0334759) variants.

### Table 2.

|   |   |   | Asymptomatic (n=50) | Symptomatic (n=54) |
| --- | --- | --- | --- | --- |
| rs2074192ACE2 (Xp22.2)c.*1860-449C>Tintron variant | Genotypes (%) | C/C | 28 (56) | 27 (50) |
| C/T | 21 (42) | 7 (13) |  |  |
| T/T | 1 (2) | 20 (37) |  |  |
| Alleles (%) | C | 77 (77) | 61 (56) |  |
| T | 23 (23) | 47 (44) |  |  |
| HW (p) females |   | 0.37 | 0.0088 |  |
| HW (p) males |   | 0.13 | <0.0001 |  |
| rs2106809ACE2 (Xp22.2)c.*264+788T>Cintron variant | Genotypes (%) | A/A | 38 (76) | 40 (74) |
| A/G | 4 (8) | 3 (6) |  |  |
| G/G | 8 (16) | 11 (20) |  |  |
| Alleles (%) | A | 80 (80) | 83 (77) |  |
| G | 20 (20) | 25 (23) |  |  |
| HW (p) females |   | 0.27 | 0.076 |  |
| HW (p) males |   | <0.0001 | <0.0001 |  |
| rs1799752ACE1 (17q23.3)c.2306-117_2306-116ins-delintron variant | Genotypes (%) | I/I | 22(44) | 7(13) |
| I/D | 21(42) | 15(28) |  |  |
| D/D | 7(14) | 32(59) |  |  |
| Alleles (%) | I | 65(65) | 29(27) |  |
| D | 35 (35) | 79(73) |  |  |
| HW (p) |   | 0.55 | 0.037 |  |
| rs4762AGT (1q42.2)c.620C>Tp.Thr207Met | Genotypes (%) | C/C | 35 (70) | 38 (70) |
| C/T | 14 (28) | 16 (30) |  |  |
| T/T | 1 (2) | 0 (0) |  |  |
| Alleles (%) | C | 84 (84) | 92 (85) |  |
| T | 16 (16) | 16 (15) |  |  |
| HW (p) |   | 1 | 0.58 |  |
| rs699AGT (1q42.2)c.803T>Cp.Met268Thr | Genotypes (%) | T/T | 19 (38) | 17 (31) |
| T/C | 31 (62) | 25 (46) |  |  |
| C/C | 0 (0) | 12 (22) |  |  |
| Alleles (%) | T | 69 (69) | 59 (55) |  |
| C | 31 (31) | 49 (45) |  |  |
| HW (p) |   | 0.0018 | 0.59 |  |
| rs5186AGTR1 (3q24)p.Met268Thr3ʹ Prime UTR Variant | Genotypes (%) | A/A | 27 (54) | 35 (65) |
| A/C | 21 (42) | 17 (31) |  |  |
| C/C | 2 (4) | 2 (4) |  |  |
| Alleles (%) | A | 75 (75) | 87 (81) |  |
| C | 25 (25) | 21 (19) |  |  |
| HW (p) |   | 0.7 | 1 |  |

Table 2 Caption: Distributions of Genotype and Allele Frequencies of SNPs rs2074192, rs2106809, rs1799752, rs4762, rs699 and rs5186 Observed in Asymptomatic and Symptomatic Patients

Genotype distribution of rs2074192 was different from those predicted by the HWE for both females (P=0.0088) and males (P<0.0001) in symptomatic group. In particular, for the male group, the T/T genotype was more frequent in codominant [odds ratio (95% CI): 15.79 (1.90–131.47), P=0.0019], dominant [odds ratio (95% CI): 5.61 (1.41–22.40), P=0.0069] and recessive [odds ratio (95% CI): 16.50 (1.99–136.49), P=0.0005] inheritance models, while for the female group, the C/T genotype shows a higher frequency in symptomatic patients in codominant [odds ratio (95% CI): 0.32 (0.08–1.21), P=0.0018] and overdominant [odds ratio (95% CI): 0.19 (0.55–0.69), P=0.0086] inheritance models.

Similarly, the genotypic frequencies in the group of symptomatic patients were different from those predicted by the HWE for rs1799752 (P=0.037). The II genotype of rs1799752 variant was associated with a lower frequency in symptomatic subjects in codominant [odds ratio (95% CI): 0.05 (0.00–0.07), P<0.0001], dominant [odds ratio (95% CI): 0.02 (0.00–0.14), P=<0.0001] recessive [odds ratio (95% CI): 0.02 (0.00–0.14), P<0.0001] and log-additive [odds ratio (95% CI): 0.08 (0.02–0.25), P<0.0001] inheritance models. Conversely, distributions of rs699 (P=0.0018) were different by Hardy–Weinberg distribution in the asymptomatic patients with a T/C genotype less frequent in this group in codominant [odds ratio (95% CI): 0.69 (0.13–3.72), P<0.0001], overdominant [odds ratio (95% CI): 0.17 (0.04–0.76), P<0.015] and log-additive [odds ratio (95% CI): 4.83 (1.36–17.16), P<0.009] inheritance models. Genotype distributions of rs4762 and rs5186 variants did not differ significantly from those predicted by the HWE and their frequencies did not significantly differ between asymptomatic and symptomatic patients ([Table 2](#t0002)). Instead, the rs2106809 variant was found to be in disequilibrium in both symptomatic and asymptomatic groups only in male patients.

### Haplotype Analysis

Haplotype analysis performed using rs2074192, rs1799752 and rs699 SNPs, demonstrated the occurrence of nine haplotypes. In particular, the haplotypes *CIT* and *TIC* were significantly higher in asymptomatic patients (P=0.00001 and P=0.048806, respectively), while haplotypes *TDT* and *TDC* were associated with symptomatic patient group (P=0.001157 and P=0.000052, respectively). The prevalence of the other haplotypes was comparable between the two groups ([Table 3](#t0003)).

### Table 3.

| Haplotypes |  |  | Frequency |  |  |  |   |   |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs2074192 | rs1799752 | rs699 | Total | Asymptomatic | Symptomatic | Cumulative | χ2 | P value |
| C | D | T | 20.35 | 18.43 | 22.65 | 20.35 | 0.563 | 0.453054 |
| C | I | T | 19.44 | 36.7 | 6.27 | 39.79 | 29.067 | < 0.00001 |
| C | D | C | 13.41 | 12.08 | 16.20 | 53.20 | 0.726 | 0.394183 |
| C | I | C | 13.15 | 9.79 | 11.36 | 66.35 | 0.133 | 0.715342 |
| T | D | T | 12.43 | 4.49 | 19.22 | 78.77 | 10.558 | 0.001157 |
| T | I | C | 9.32 | 9.13 | 2.73 | 88.10 | 3.882 | 0.048806 |
| T | D | C | 8.62 | 0 | 15.08 | 96.71 | 16.360 | 0.000052 |
| T | I | T | 3.29 | 9.37 | 6.49 | 100 | 0.598 | 0.439342 |

Table 3 Caption: Haplotype Analysis Performed on rs2074192, rs1799752 and rs699 SNPs and Their Corresponding Frequencies in Asymptomatic (N= 50) and Symptomatic (N= 54) Patients

## Discussion

We analyzed the genetic variants located on *ACE2* (rs2074192, rs2106809), *ACE1* (rs1799752), *AGT* (rs4762, rs699) and *AGTR1* (rs5186) genes in a cohort of 104 Italian patients positive for SARS-COV-2 infection, divided into 54 symptomatic COVID-19 patients and 50 asymptomatic subjects, in order to investigate the potential correlation of some polymorphisms of the RAAS pathway with the susceptibility to SARS-CoV-2 infection and the clinical outcome of disease.

COVID-19 is a severe form of lung disease (infectious pneumonia) leading to respiratory failure in acute forms. Lung lesions remain after healing. Symptomatic patients show interstitial bilateral pneumonia (multiple foci) and have 1) dyspnea “hunger for air”, 2) dry cough and 3) high fever. CT and PCR are elective diagnostic tools. In addition to nose and mouth epithelia, the ocular structures (cornea, conjunctiva and retina) were found to express high levels of *ACE2* and *TMPRSS2* (two main virus doors) and tears can spread the virus through the nasolacrimal system. Although having high levels of *ACE2* and *TMPRSS2*, some antiviral countermeasures to lower virus infection at the ocular surface have been identified.[22](#cit0022) As widely recognized, RAAS represents the main gate for SARS-CoV-2.

In this pilot study, we found that the rs2074192 (*ACE2*), rs1799752 (*ACE1*) and rs699 (*AGT*) SNPs could potentially be a valuable tool for predicting the clinical outcome of SARS-CoV-2 infected patients.

The *ACE2* gene (OMIM *300335, Cytogenetic location: Xp22.2) encodes a surface enzyme protein with extensive biological activities including the effect of antagonism on RAAS-promoting release of vasoactive peptides with a vasodilating, anti-inflammatory and organ-protective effect, suggesting a role in the regulation of cardiovascular, renal and even fertility functions.[23](#cit0023) This enzyme is expressed on cell membranes of lungs, arteries, heart, kidneys, intestines and eye (cornea, conjunctiva and retina) tissues. This enzyme catalyzes the cleavage of angiotensin I into angiotensin 1–9,[8](#cit0008) and angiotensin II into the angiotensin 1–7.[24](#cit0024) This protein is a functional receptor for the spike glycoprotein of HCoV-NL63, SARS-CoV and SARS-CoV-2 human severe acute respiratory syndrome coronaviruses, allowing glycoprotein internalization into target host cells and subsequent intracellular replication and transcription of virus.[23](#cit0023)^,^[25](#cit0025) Benetti et al identified three polymorphisms (rs41303171, rs148771870, rs4646116) potentially responsible for the destabilization of ACE2 protein, by comparing the results of a whole-exome sequencing data of 6930 Italian control subjects with the 131 COVID-19 patients and 258 healthy controls.[26](#cit0026) In addition, Benetti and coworkers observed a greater allelic variability in the control group compared with COVID-19 patient one, hypothesizing that this heterogeneity may be responsible for the wide clinical variability of COVID-19 disease.[26](#cit0026)

Cao et al, by analyzing 1700 *ACE2* variant from ChinaMAP and 1000 Genomes Project databases and comparing the allele frequency differences between different populations, identified a truncating mutation and seven hotspot variants potentially related to different susceptibility to SARS-CoV-2 infection.[27](#cit0027) Asselta et al compared *ACE2* exome and SNP-array data from an Italian cohort of 3.984 cases failing to identify any association with the severity of disease.[28](#cit0028) Novelli et al, in a whole-exome sequencing pilot study performed on 131 Italian COVID-19 patients, identified three *ACE2* gene variants (rs2285666, rs41303171 and rs140312271) of which only the rs140312271 showed a significantly different statistical frequency compared to an ethnicity-matched control group.[29](#cit0029)

In this study, we have identified for the rs2074192 SNP (located at intron 16 of *ACE2* gene) a significant higher frequency of the T allele in the symptomatic group compared to the asymptomatic ones in both females and males ([Table 2](#t0002)). The T allele of rs2074192 polymorphism is already known for its association with cardiovascular risk, retinopathy in type-2 diabetes mellitus individuals, hypertension and hypertensive left ventricular hypertrophy.[30](#cit0030)^,^[31](#cit0031) Furthermore, it should be noted that our data are in agreement with a very recent large study conducted on 1644 COVID-19 patients from the UK Biobank, showing that the T allele is correlated with more severe outcomes of SARS-COV2 infection.[32](#cit0032)

The *ACE1* gene (OMIM 106180, cytogenetic location: 17q23.3) encodes an enzyme physiologically active on blood pressure regulation and electrolyte balance by catalyzing the conversion of angiotensin I (vasodilator) into a physiologically active peptide angiotensin II (vasoconstrictor) and aldosterone-stimulating peptide that also controls blood pressure and electrolyte balance.[33](#cit0033) The ACE also inactivates bradykinin in BK (vasodilator) that could play a relevant role in COVID-19 as already described for other viral models.[34](#cit0034) Approximately 50% of variability in plasma levels of ACE depends on the rs1799752 polymorphism, located at intron 16 of the *ACE1* gene and characterized by the presence of an insertion (I) or a deletion (D) of a 287 bp Alu repeat sequence, directly related to a lower or higher serum ACE levels, respectively.[18](#cit0018) The presence of allele D is associated with a greater risk of hypercoagulability, hypertension, endothelial damage, diabetic nephropathy, diabetes mellitus and the risk of overweight/obesity, cerebral ischemia and response to Interferon-β treatment.[35](#cit0035) Moreover, patients with the D/D genotype show an increased mean pulmonary arterial pressure and vascular resistance after exercise, as compared to the remaining genotypes when treated with ACE inhibitors such as captopril.[14](#cit0014) A recent study performed comparing COVID-19 prevalence and rs1799752 allele frequency data, recovered by Johns Hopkins University and GnomAD, respectively, demonstrates that the ACE1 I/I genotype is significantly negatively correlated with susceptibility to SARS-CoV-2 infection and worse clinical outcome.[36](#cit0036) These results are in line with our data indicating that genotype I/I and allele I are significantly more frequent in the asymptomatic than symptomatic patient group ([Table 2](#t0002)).

The *AGT* gene (OMIM 106150, cytogenetic location: 1q42.2) encodes for the angiotensinogen precursor or pre-angiotensinogen, highly expressed in liver as precursor and quickly cleaved into angiotensin I by renin following a reduced blood pressure. In turn, angiotensin I, through the cleavage carried out by the ACE is converted into angiotensin II, a physiologically active form that participates in the homeostasis of electrolytes and the stability of blood pressure.[35](#cit0035) Numerous experimental evidences suggest that sequence variants of this gene are correlated with hypertension, heart failure and cardiovascular risk factors.[37](#cit0037) In particular, for rs699 SNP, the presence of the TT genotype has been associated with the development of arterial hypertension, systolic blood pressure, coronary artery disease, mean arterial pressure.[38–40](#cit0038) Patients with the TT genotype and hypertension may have an increased risk of stroke when treated with ACE inhibitors.[41](#cit0041) In our study, the rs699 SNP, as a unique case, displayed a significant increased frequency of the T allele in comparison to the global population from the 1000 Genome Project and GnomAD while it did not differ from the data extrapolated from the European population ([Table 1](#t0001)). Furthermore, we also found a lower frequency of the T/C genotype in the group of asymptomatic patients different by the Hardy–Weinberg distribution ([Table 2](#t0002)), but at the present time there are no data of this polymorphism in COVID-19 patients in order to make a comparison.

The results of this study, albeit with a limited but well-selected number of patients, suggest that some gene variants of RAAS pathway could modulate some pathological conditions associated with the heterogeneous clinical picture caused by SARS-CoV-2 infection such as disseminated intravascular coagulation and thrombosis, interstitial pneumonia, conjunctivitis and the cytokine storm.[14](#cit0014)^,^[36](#cit0036)^,^[42](#cit0042) Furthermore, data would suggest that the early genetic evaluation of subjects infected with SARS-CoV-2 can predict the ongoing/severity of disease, becoming a useful tool for selecting those patients deserving more attention to avoid complications. This personalized approach (predictive medicine) might represent a step forward in the development of strategies to counteract this complicated pandemic situation, seriously affecting worldwide.

In this context, precision medicine is an innovative approach to disease prevention and treatment based on genetic differences between individuals and the influence of environment and lifestyle. After mapping and sequencing the human genome, the scientific community has focused on the functional significance of all the differences that define the genetic characters of each patient. Some of the elements that underlie the differences in the genome are due to SNPs. The study/analysis of SNPs finds applications in the field of diagnostic (differential diagnosis), giving clinicians the ability to identify individual susceptibilities to numerous diseases and responses to drugs, including side effects and toxic reactions (pharmacogenomics).[43](#cit0043)

In this regard, in a previous study using a silicon prediction of drug effects, we have extensively evaluated the interactions between SNPs and drugs that show efficacy or toxicity in countering COVID-19, suggesting the application of personalized medicine tools during the treatment of SARS-CoV-2 infection.[12](#cit0012) Overall, the main strength of this study is the possibility of analyzing two groups of clinically selected patients for whom a specific case–control association study could be performed. In fact, as far as we know, this is one of the first studies to perform a genetic comparison between asymptomatic subjects and symptomatic patients, while numerous previous studies have compared data from databases of population of different ancestries.

Finally, further molecular – epidemiological studies are required to understand the exact mechanisms underlying the clinical variability of COVID-19 disease, even in populations from different ethnic groups, and predict the most severe clinical manifestations, to develop personalized approaches or alternative strategies.

## Acknowledgments

We thank Dr Vincenzo Varriale for unvaluable contribution in providing some reagents. AC and AM thank the Italian Ministry of Health and Fondazione Roma (Italy).

## Ethics Approval

All subjects were treated according to the Helsinki Declaration of Biomedical Ethics and provided informed consent before participating in the study.[44](#cit0044)

## Author Contributions

All authors made substantial contributions to conception and design, acquisition of data, or analysis and interpretation of data; took part in drafting the article or revising it critically for important intellectual content; agreed to submit to the current journal; gave final approval of the version to be published; and agree to be accountable for all aspects of the work.

## Disclosure

The authors reported no conflicts of interest for this work and declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## References

1. Zhu N, Zhang D, Wang W, et al. A novel coronavirus from patients with pneumonia in China, 2019. N Engl J Med. 2020;382(8):727–733. doi: 10.1056/NEJMoa2001017  [DOI](https://doi.org/10.1056/NEJMoa2001017) | [PMC free article](/articles/PMC7092803/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31978945/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=A%20novel%20coronavirus%20from%20patients%20with%20pneumonia%20in%20China,%202019&author=N%20Zhu&author=D%20Zhang&author=W%20Wang&volume=382&issue=8&publication_year=2020&pages=727-733&pmid=31978945&doi=10.1056/NEJMoa2001017&)

2. Yan R, Zhang Y, Li Y, et al. Structural basis for the recognition of SARS-CoV-2 by full-length human ACE2. Science. 2020;367(6485):1444–1448. doi: 10.1126/science.abb2762  [DOI](https://doi.org/10.1126/science.abb2762) | [PMC free article](/articles/PMC7164635/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32132184/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Structural%20basis%20for%20the%20recognition%20of%20SARS-CoV-2%20by%20full-length%20human%20ACE2&author=R%20Yan&author=Y%20Zhang&author=Y%20Li&volume=367&issue=6485&publication_year=2020&pages=1444-1448&pmid=32132184&doi=10.1126/science.abb2762&)

3. Cheng Y, Luo R, Wang K, et al. Kidney disease is associated with in-hospital death of patients with COVID-19. Kidney Int. 2020;97(5):829–838. doi: 10.1016/j.kint.2020.03.005  [DOI](https://doi.org/10.1016/j.kint.2020.03.005) | [PMC free article](/articles/PMC7110296/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32247631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Kidney%20Int&title=Kidney%20disease%20is%20associated%20with%20in-hospital%20death%20of%20patients%20with%20COVID-19&author=Y%20Cheng&author=R%20Luo&author=K%20Wang&volume=97&issue=5&publication_year=2020&pages=829-838&pmid=32247631&doi=10.1016/j.kint.2020.03.005&)

4. Kumar A, Prasoon P, Kumari C, et al. SARS-CoV-2-specific virulence factors in COVID-19. J Med Virol. 2020. doi: 10.1002/jmv.26615  [DOI](https://doi.org/10.1002/jmv.26615) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33085084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Med%20Virol&title=SARS-CoV-2-specific%20virulence%20factors%20in%20COVID-19&author=A%20Kumar&author=P%20Prasoon&author=C%20Kumari&publication_year=2020&pmid=33085084&doi=10.1002/jmv.26615&)

5. Cascella M, Rajnik M, Cuomo A, et al. Features, evaluation, and treatment of Coronavirus. In: StatPearls. Treasure Island (FL): StatPearls Publishing; 2021.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/32150360/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=StatPearls&author=M%20Cascella&author=M%20Rajnik&author=A%20Cuomo&publication_year=2021&)

6. Zolfaghari ER, Nosrati H, Eftekhari M, et al. Expansion of single cell transcriptomics data of SARS-CoV infection in human bronchial epithelial cells to COVID-19. Biol Proced Online. 2020;22:16. doi: 10.1186/s12575-020-00127-3  [DOI](https://doi.org/10.1186/s12575-020-00127-3) | [PMC free article](/articles/PMC7377208/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32754004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Proced%20Online&title=Expansion%20of%20single%20cell%20transcriptomics%20data%20of%20SARS-CoV%20infection%20in%20human%20bronchial%20epithelial%20cells%20to%20COVID-19&author=ER%20Zolfaghari&author=H%20Nosrati&author=M%20Eftekhari&volume=22&publication_year=2020&pages=16&pmid=32754004&doi=10.1186/s12575-020-00127-3&)

7. Kolberg ES, Wickstrøm K, Tonby K, et al. Serum ACE as a prognostic biomarker in COVID-19: a case series. APMIS. 2020. doi: 10.1111/apm.13108  [DOI](https://doi.org/10.1111/apm.13108) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33368687/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=APMIS&title=Serum%20ACE%20as%20a%20prognostic%20biomarker%20in%20COVID-19:%20a%20case%20series&author=ES%20Kolberg&author=K%20Wickstr%C3%B8m&author=K%20Tonby&publication_year=2020&pmid=33368687&doi=10.1111/apm.13108&)

8. Tipnis SR, Hooper NM, Hyde R, et al. A human homolog of angiotensin-converting enzyme. Cloning and functional expression as a captopril-insensitive carboxypeptidase. J Biol Chem. 2000;275(43):33238–33243. doi: 10.1074/jbc.M002615200  [DOI](https://doi.org/10.1074/jbc.M002615200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10924499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=A%20human%20homolog%20of%20angiotensin-converting%20enzyme.%20Cloning%20and%20functional%20expression%20as%20a%20captopril-insensitive%20carboxypeptidase&author=SR%20Tipnis&author=NM%20Hooper&author=R%20Hyde&volume=275&issue=43&publication_year=2000&pages=33238-33243&pmid=10924499&doi=10.1074/jbc.M002615200&)

9. Donoghue M, Hsieh F, Baronas E, et al. A novel angiotensin-converting enzyme-related carboxypeptidase (ACE2) converts angiotensin I to angiotensin 1-9. Circ Res. 2000;87(5):E1–E9. doi: 10.1161/01.res.87.5.e1  [DOI](https://doi.org/10.1161/01.res.87.5.e1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10969042/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Res&title=A%20novel%20angiotensin-converting%20enzyme-related%20carboxypeptidase%20(ACE2)%20converts%20angiotensin%20I%20to%20angiotensin%201-9&author=M%20Donoghue&author=F%20Hsieh&author=E%20Baronas&volume=87&issue=5&publication_year=2000&pages=E1-E9&pmid=10969042&doi=10.1161/01.res.87.5.e1&)

10. Zolfaghari ER, Falak R, Bahreini E. Application of system biology to explore the association of neprilysin, angiotensin-converting enzyme 2 (ACE2), and carbonic anhydrase (CA) in pathogenesis of SARS-CoV-2. Biol Proced Online. 2020;22:11. doi: 10.1186/s12575-020-00124-6  [DOI](https://doi.org/10.1186/s12575-020-00124-6) | [PMC free article](/articles/PMC7302923/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32572334/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Proced%20Online&title=Application%20of%20system%20biology%20to%20explore%20the%20association%20of%20neprilysin,%20angiotensin-converting%20enzyme%202%20(ACE2),%20and%20carbonic%20anhydrase%20(CA)%20in%20pathogenesis%20of%20SARS-CoV-2&author=ER%20Zolfaghari&author=R%20Falak&author=E%20Bahreini&volume=22&publication_year=2020&pages=11&pmid=32572334&doi=10.1186/s12575-020-00124-6&)

11. Bavishi C, Maddox TM, Messerli FH. Coronavirus disease 2019 (COVID-19) infection and renin angiotensin system blockers. JAMA Cardiol. 2020;5(7):745–747. doi: 10.1001/jamacardio.2020.1282  [DOI](https://doi.org/10.1001/jamacardio.2020.1282) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32242890/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Cardiol&title=Coronavirus%20disease%202019%20(COVID-19)%20infection%20and%20renin%20angiotensin%20system%20blockers&author=C%20Bavishi&author=TM%20Maddox&author=FH%20Messerli&volume=5&issue=7&publication_year=2020&pages=745-747&pmid=32242890&doi=10.1001/jamacardio.2020.1282&)

12. Mirabito Colafella KM, Bovée DM, Danser AHJ. The renin-angiotensin-aldosterone system and its therapeutic targets. Exp Eye Res. 2019;186:107680. doi: 10.1016/j.exer.2019.05.020  [DOI](https://doi.org/10.1016/j.exer.2019.05.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31129252/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp%20Eye%20Res&title=The%20renin-angiotensin-aldosterone%20system%20and%20its%20therapeutic%20targets&author=KM%20Mirabito%20Colafella&author=DM%20Bov%C3%A9e&author=AHJ%20Danser&volume=186&publication_year=2019&pages=107680&pmid=31129252&doi=10.1016/j.exer.2019.05.020&)

13. Banu N, Panikar SS, Leal LR, Leal AR. Protective role of ACE2 and its downregulation in SARS-CoV-2 infection leading to macrophage activation syndrome: therapeutic implications. Life Sci. 2020;256:117905. doi: 10.1016/j.lfs.2020.117905  [DOI](https://doi.org/10.1016/j.lfs.2020.117905) | [PMC free article](/articles/PMC7832382/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32504757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life%20Sci&title=Protective%20role%20of%20ACE2%20and%20its%20downregulation%20in%20SARS-CoV-2%20infection%20leading%20to%20macrophage%20activation%20syndrome:%20therapeutic%20implications&author=N%20Banu&author=SS%20Panikar&author=LR%20Leal&author=AR%20Leal&volume=256&publication_year=2020&pages=117905&pmid=32504757&doi=10.1016/j.lfs.2020.117905&)

14. Cafiero C, Re A, Micera A, et al. Pharmacogenomics and pharmacogenetics: in silico prediction of drug effects in treatments for novel Coronavirus SARS-CoV2 disease. Pharmgenomics Pers Med. 2020;13:463–484. doi: 10.2147/PGPM.S270069  [DOI](https://doi.org/10.2147/PGPM.S270069) | [PMC free article](/articles/PMC7568633/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33116761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenomics%20Pers%20Med&title=Pharmacogenomics%20and%20pharmacogenetics:%20in%20silico%20prediction%20of%20drug%20effects%20in%20treatments%20for%20novel%20Coronavirus%20SARS-CoV2%20disease&author=C%20Cafiero&author=A%20Re&author=A%20Micera&volume=13&publication_year=2020&pages=463-484&pmid=33116761&doi=10.2147/PGPM.S270069&)

15. Palmirotta R, Ludovici G, De Marchis ML, et al. Preanalytical procedures for DNA studies: the experience of the interinstitutional multidisciplinary BioBank (BioBIM). Biopreserv Biobank. 2011;9(1):35–45. doi: 10.1089/bio.2010.0027  [DOI](https://doi.org/10.1089/bio.2010.0027) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24850204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biopreserv%20Biobank&title=Preanalytical%20procedures%20for%20DNA%20studies:%20the%20experience%20of%20the%20interinstitutional%20multidisciplinary%20BioBank%20(BioBIM)&author=R%20Palmirotta&author=G%20Ludovici&author=ML%20De%20Marchis&volume=9&issue=1&publication_year=2011&pages=35-45&pmid=24850204&doi=10.1089/bio.2010.0027&)

16. 1000 Genomes Project Consortium. A global reference for human genetic variation. Nature. 2015;526(7571):68–74. doi: 10.1038/nature15393.  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation&volume=526&issue=7571&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

17. Karczewski KJ, Francioli LC, Tiao G, et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature. 2020;581(7809):434–443. doi: 10.1038/s41586-020-2308-7  [DOI](https://doi.org/10.1038/s41586-020-2308-7) | [PMC free article](/articles/PMC7334197/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32461654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141,456%20humans&author=KJ%20Karczewski&author=LC%20Francioli&author=G%20Tiao&volume=581&issue=7809&publication_year=2020&pages=434-443&pmid=32461654&doi=10.1038/s41586-020-2308-7&)

18. Palmirotta R, Barbanti P, Ludovici G, et al. Association between migraine and ACE gene (insertion/deletion) polymorphism: the BioBIM study. Pharmacogenomics. 2014;15(2):147–155. doi: 10.2217/pgs.13.186  [DOI](https://doi.org/10.2217/pgs.13.186) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24444405/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Association%20between%20migraine%20and%20ACE%20gene%20(insertion/deletion)%20polymorphism:%20the%20BioBIM%20study&author=R%20Palmirotta&author=P%20Barbanti&author=G%20Ludovici&volume=15&issue=2&publication_year=2014&pages=147-155&pmid=24444405&doi=10.2217/pgs.13.186&)

19. Solé X, Guinó E, Valls J, Iniesta R, Moreno V. SNPStats: a web tool for the analysis of association studies. Bioinformatics. 2006;22(15):1928–1929. doi: 10.1093/bioinformatics/btl268  [DOI](https://doi.org/10.1093/bioinformatics/btl268) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16720584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SNPStats:%20a%20web%20tool%20for%20the%20analysis%20of%20association%20studies&author=X%20Sol%C3%A9&author=E%20Guin%C3%B3&author=J%20Valls&author=R%20Iniesta&author=V%20Moreno&volume=22&issue=15&publication_year=2006&pages=1928-1929&pmid=16720584&doi=10.1093/bioinformatics/btl268&)

20. Shi YY, He L. SHEsis, a powerful software platform for analyses of linkage disequilibrium, haplotype construction, and genetic association at polymorphism loci. Cell Res. 2005;15(2):97–98. doi: 10.1038/sj.cr.7290272  [DOI](https://doi.org/10.1038/sj.cr.7290272) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15740637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=SHEsis,%20a%20powerful%20software%20platform%20for%20analyses%20of%20linkage%20disequilibrium,%20haplotype%20construction,%20and%20genetic%20association%20at%20polymorphism%20loci&author=YY%20Shi&author=L%20He&volume=15&issue=2&publication_year=2005&pages=97-98&pmid=15740637&doi=10.1038/sj.cr.7290272&)

21. Li Z, Zhang Z, He Z, et al. A partition-ligation-combination-subdivision EM algorithm for haplotype inference with multiallelic markers: update of the SHEsis (http://analysis.bio-x.cn). Cell Res. 2009;19(4):519–523. doi: 10.1038/cr.2009.33  [(http://analysis.bio-x.cn)](http://analysis.bio-x.cn) | [DOI](https://doi.org/10.1038/cr.2009.33) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19290020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=A%20partition-ligation-combination-subdivision%20EM%20algorithm%20for%20haplotype%20inference%20with%20multiallelic%20markers:%20update%20of%20the%20SHEsis%20(http://analysis.bio-x.cn)&author=Z%20Li&author=Z%20Zhang&author=Z%20He&volume=19&issue=4&publication_year=2009&pages=519-523&pmid=19290020&doi=10.1038/cr.2009.33&)

22. Leonardi A, Rosani U, Brun P. Ocular surface expression of SARS-CoV-2 receptors. Ocul Immunol Inflamm. 2020;28(5):735–738. doi: 10.1080/09273948.2020.1772314  [DOI](https://doi.org/10.1080/09273948.2020.1772314) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32589459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ocul%20Immunol%20Inflamm&title=Ocular%20surface%20expression%20of%20SARS-CoV-2%20receptors&author=A%20Leonardi&author=U%20Rosani&author=P%20Brun&volume=28&issue=5&publication_year=2020&pages=735-738&pmid=32589459&doi=10.1080/09273948.2020.1772314&)

23. Devaux CA, Rolain JM, Raoult D. ACE2 receptor polymorphism: susceptibility to SARS-CoV-2, hypertension, multi-organ failure, and COVID-19 disease outcome. J Microbiol Immunol Infect. 2020;53(3):425–435. doi: 10.1016/j.jmii.2020.04.015  [DOI](https://doi.org/10.1016/j.jmii.2020.04.015) | [PMC free article](/articles/PMC7201239/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32414646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Microbiol%20Immunol%20Infect&title=ACE2%20receptor%20polymorphism:%20susceptibility%20to%20SARS-CoV-2,%20hypertension,%20multi-organ%20failure,%20and%20COVID-19%20disease%20outcome&author=CA%20Devaux&author=JM%20Rolain&author=D%20Raoult&volume=53&issue=3&publication_year=2020&pages=425-435&pmid=32414646&doi=10.1016/j.jmii.2020.04.015&)

24. Santos RAS, Sampaio WO, Alzamora AC, et al. The ACE2/angiotensin-(1-7)/MAS axis of the renin-angiotensin system: focus on angiotensin-(1-7). Physiol Rev. 2018;98(1):505–553. doi: 10.1152/physrev.00023.2016  [DOI](https://doi.org/10.1152/physrev.00023.2016) | [PMC free article](/articles/PMC7203574/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29351514/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiol%20Rev&title=The%20ACE2/angiotensin-(1-7)/MAS%20axis%20of%20the%20renin-angiotensin%20system:%20focus%20on%20angiotensin-(1-7)&author=RAS%20Santos&author=WO%20Sampaio&author=AC%20Alzamora&volume=98&issue=1&publication_year=2018&pages=505-553&pmid=29351514&doi=10.1152/physrev.00023.2016&)

25. Saxena SK, Kumar S, Baxi P, et al. Chasing COVID-19 through SARS-CoV-2 spike glycoprotein. Virusdisease. 2020;31(4):1–9. doi: 10.1007/s13337-020-00642-7  [DOI](https://doi.org/10.1007/s13337-020-00642-7) | [PMC free article](/articles/PMC7718591/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33313362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Virusdisease&title=Chasing%20COVID-19%20through%20SARS-CoV-2%20spike%20glycoprotein&author=SK%20Saxena&author=S%20Kumar&author=P%20Baxi&volume=31&issue=4&publication_year=2020&pages=1-9&pmid=33313362&doi=10.1007/s13337-020-00642-7&)

26. Benetti E, Tita R, Spiga O, et al. ACE2 gene variants may underlie interindividual variability and susceptibility to COVID-19 in the Italian population. Eur J Hum Genet. 2020;28(11):1602–1614. doi: 10.1038/s41431-020-0691-z  [DOI](https://doi.org/10.1038/s41431-020-0691-z) | [PMC free article](/articles/PMC7366459/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32681121/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=ACE2%20gene%20variants%20may%20underlie%20interindividual%20variability%20and%20susceptibility%20to%20COVID-19%20in%20the%20Italian%20population&author=E%20Benetti&author=R%20Tita&author=O%20Spiga&volume=28&issue=11&publication_year=2020&pages=1602-1614&pmid=32681121&doi=10.1038/s41431-020-0691-z&)

27. Cao Y, Li L, Feng Z, et al. Comparative genetic analysis of the novel coronavirus (2019-nCoV/SARS-CoV-2) receptor ACE2 in different populations. Cell Discov. 2020;6:11. doi: 10.1038/s41421-020-0147-1  [DOI](https://doi.org/10.1038/s41421-020-0147-1) | [PMC free article](/articles/PMC7040011/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32133153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Discov&title=Comparative%20genetic%20analysis%20of%20the%20novel%20coronavirus%20(2019-nCoV/SARS-CoV-2)%20receptor%20ACE2%20in%20different%20populations&author=Y%20Cao&author=L%20Li&author=Z%20Feng&volume=6&publication_year=2020&pages=11&pmid=32133153&doi=10.1038/s41421-020-0147-1&)

28. Asselta R, Paraboschi EM, Mantovani A, Duga S. ACE2 and TMPRSS2 variants and expression as candidates to sex and country differences in COVID-19 severity in Italy. Aging (Albany NY). 2020;12(11):10087–10098. doi: 10.18632/aging.103415  [DOI](https://doi.org/10.18632/aging.103415) | [PMC free article](/articles/PMC7346072/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32501810/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aging%20(Albany%20NY)&title=ACE2%20and%20TMPRSS2%20variants%20and%20expression%20as%20candidates%20to%20sex%20and%20country%20differences%20in%20COVID-19%20severity%20in%20Italy&author=R%20Asselta&author=EM%20Paraboschi&author=A%20Mantovani&author=S%20Duga&volume=12&issue=11&publication_year=2020&pages=10087-10098&pmid=32501810&doi=10.18632/aging.103415&)

29. Novelli A, Biancolella M, Borgiani P, et al. Analysis of ACE2 genetic variants in 131 Italian SARS-CoV-2-positive patients. Hum Genomics. 2020;14(1):29. doi: 10.1186/s40246-020-00279-z  [DOI](https://doi.org/10.1186/s40246-020-00279-z) | [PMC free article](/articles/PMC7483483/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32917283/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Genomics&title=Analysis%20of%20ACE2%20genetic%20variants%20in%20131%20Italian%20SARS-CoV-2-positive%20patients&author=A%20Novelli&author=M%20Biancolella&author=P%20Borgiani&volume=14&issue=1&publication_year=2020&pages=29&pmid=32917283&doi=10.1186/s40246-020-00279-z&)

30. Liu C, Li Y, Guan T, et al. ACE2 polymorphisms associated with cardiovascular risk in Uygurs with type 2 diabetes mellitus. Cardiovasc Diabetol. 2018;17(1):127. doi: 10.1186/s12933-018-0771-3  [DOI](https://doi.org/10.1186/s12933-018-0771-3) | [PMC free article](/articles/PMC6142339/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30227878/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Diabetol&title=ACE2%20polymorphisms%20associated%20with%20cardiovascular%20risk%20in%20Uygurs%20with%20type%202%20diabetes%20mellitus&author=C%20Liu&author=Y%20Li&author=T%20Guan&volume=17&issue=1&publication_year=2018&pages=127&pmid=30227878&doi=10.1186/s12933-018-0771-3&)

31. Fan Z, Wu G, Yue M, et al. Hypertension and hypertensive left ventricular hypertrophy are associated with ACE2 genetic polymorphism. Life Sci. 2019;225:39–45. doi: 10.1016/j.lfs.2019.03.059  [DOI](https://doi.org/10.1016/j.lfs.2019.03.059) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30917908/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life%20Sci&title=Hypertension%20and%20hypertensive%20left%20ventricular%20hypertrophy%20are%20associated%20with%20ACE2%20genetic%20polymorphism&author=Z%20Fan&author=G%20Wu&author=M%20Yue&volume=225&publication_year=2019&pages=39-45&pmid=30917908&doi=10.1016/j.lfs.2019.03.059&)

32. Hamet P, Pausova Z, Attaoua R, et al. SARS-COV-2 receptor ACE2 gene is associated with hypertension and severity of COVID 19: interaction with sex, obesity and smoking. Am J Hypertens. 2021;34:hpaa223. doi: 10.1093/ajh/hpaa223  [DOI](https://doi.org/10.1093/ajh/hpaa223) | [PMC free article](/articles/PMC7799248/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33386398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hypertens&title=SARS-COV-2%20receptor%20ACE2%20gene%20is%20associated%20with%20hypertension%20and%20severity%20of%20COVID%2019:%20interaction%20with%20sex,%20obesity%20and%20smoking&author=P%20Hamet&author=Z%20Pausova&author=R%20Attaoua&volume=34&publication_year=2021&pages=hpaa223&pmid=33386398&doi=10.1093/ajh/hpaa223&)

33. Krege JH, Kim HS, Moyer JS, et al. Angiotensin-converting enzyme gene mutations, blood pressures, and cardiovascular homeostasis. Hypertension. 1997;29(1):150–157. doi: 10.1161/01.hyp.29.1.150  [DOI](https://doi.org/10.1161/01.hyp.29.1.150) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9039095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Angiotensin-converting%20enzyme%20gene%20mutations,%20blood%20pressures,%20and%20cardiovascular%20homeostasis&author=JH%20Krege&author=HS%20Kim&author=JS%20Moyer&volume=29&issue=1&publication_year=1997&pages=150-157&pmid=9039095&doi=10.1161/01.hyp.29.1.150&)

34. Meini S, Zanichelli A, Sbrojavacca R, et al. Understanding the pathophysiology of COVID-19: could the contact system be the key? Front Immunol. 2020;11:2014. doi: 10.3389/fimmu.2020.02014  [DOI](https://doi.org/10.3389/fimmu.2020.02014) | [PMC free article](/articles/PMC7432138/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32849666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&title=Understanding%20the%20pathophysiology%20of%20COVID-19:%20could%20the%20contact%20system%20be%20the%20key?&author=S%20Meini&author=A%20Zanichelli&author=R%20Sbrojavacca&volume=11&publication_year=2020&pages=2014&pmid=32849666&doi=10.3389/fimmu.2020.02014&)

35. Ghafouri-Fard S, Noroozi R, Omrani MD, et al. Angiotensin converting enzyme: a review on expression profile and its association with human disorders with special focus on SARS-CoV-2 infection. Vascul Pharmacol. 2020;130:106680. doi: 10.1016/j.vph.2020.106680  [DOI](https://doi.org/10.1016/j.vph.2020.106680) | [PMC free article](/articles/PMC7211701/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32423553/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Vascul%20Pharmacol&title=Angiotensin%20converting%20enzyme:%20a%20review%20on%20expression%20profile%20and%20its%20association%20with%20human%20disorders%20with%20special%20focus%20on%20SARS-CoV-2%20infection&author=S%20Ghafouri-Fard&author=R%20Noroozi&author=MD%20Omrani&volume=130&publication_year=2020&pages=106680&pmid=32423553&doi=10.1016/j.vph.2020.106680&)

36. Yamamoto N, Ariumi Y, Nishida N, et al. SARS-CoV-2 infections and COVID-19 mortalities strongly correlate with ACE1 I/D genotype. Gene. 2020;758:144944. doi: 10.1016/j.gene.2020.144944  [DOI](https://doi.org/10.1016/j.gene.2020.144944) | [PMC free article](/articles/PMC7833925/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32628976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=SARS-CoV-2%20infections%20and%20COVID-19%20mortalities%20strongly%20correlate%20with%20ACE1%20I/D%20genotype&author=N%20Yamamoto&author=Y%20Ariumi&author=N%20Nishida&volume=758&publication_year=2020&pages=144944&pmid=32628976&doi=10.1016/j.gene.2020.144944&)

37. Xu Y, Rong J, Zhang Z. The emerging role of angiotensinogen in cardiovascular diseases. J Cell Physiol. 2021;236(1):68–78. doi: 10.1002/jcp.29889  [DOI](https://doi.org/10.1002/jcp.29889) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32572956/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Physiol&title=The%20emerging%20role%20of%20angiotensinogen%20in%20cardiovascular%20diseases&author=Y%20Xu&author=J%20Rong&author=Z%20Zhang&volume=236&issue=1&publication_year=2021&pages=68-78&pmid=32572956&doi=10.1002/jcp.29889&)

38. Giri A, Hellwege JN, Keaton JM, et al. Trans-ethnic association study of blood pressure determinants in over 750,000 individuals. Nat Genet. 2019;51(1):51–62. doi: 10.1038/s41588-018-0303-9  [DOI](https://doi.org/10.1038/s41588-018-0303-9) | [PMC free article](/articles/PMC6365102/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30578418/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Trans-ethnic%20association%20study%20of%20blood%20pressure%20determinants%20in%20over%20750,000%20individuals&author=A%20Giri&author=JN%20Hellwege&author=JM%20Keaton&volume=51&issue=1&publication_year=2019&pages=51-62&pmid=30578418&doi=10.1038/s41588-018-0303-9&)

39. van der Harst P, Verweij N. Identification of 64 novel genetic loci provides an expanded view on the genetic architecture of coronary artery disease. Circ Res. 2018;122(3):433–443. doi: 10.1161/CIRCRESAHA.117.312086  [DOI](https://doi.org/10.1161/CIRCRESAHA.117.312086) | [PMC free article](/articles/PMC5805277/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29212778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Res&title=Identification%20of%2064%20novel%20genetic%20loci%20provides%20an%20expanded%20view%20on%20the%20genetic%20architecture%20of%20coronary%20artery%20disease&author=P%20van%20der%20Harst&author=N%20Verweij&volume=122&issue=3&publication_year=2018&pages=433-443&pmid=29212778&doi=10.1161/CIRCRESAHA.117.312086&)

40. Liu C, Kraja AT, Smith JA, et al. Meta-analysis identifies common and rare variants influencing blood pressure and overlapping with metabolic trait loci. Nat Genet. 2016;48(10):1162–1170. doi: 10.1038/ng.3660  [DOI](https://doi.org/10.1038/ng.3660) | [PMC free article](/articles/PMC5320952/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27618448/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Meta-analysis%20identifies%20common%20and%20rare%20variants%20influencing%20blood%20pressure%20and%20overlapping%20with%20metabolic%20trait%20loci&author=C%20Liu&author=AT%20Kraja&author=JA%20Smith&volume=48&issue=10&publication_year=2016&pages=1162-1170&pmid=27618448&doi=10.1038/ng.3660&)

41. Kurland L, Melhus H, Karlsson J, et al. Angiotensin converting enzyme gene polymorphism predicts blood pressure response to angiotensin II receptor type 1 antagonist treatment in hypertensive patients. J Hypertens. 2001;19(10):1783–1787. doi: 10.1097/00004872-200110000-00012  [DOI](https://doi.org/10.1097/00004872-200110000-00012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11593098/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hypertens&title=Angiotensin%20converting%20enzyme%20gene%20polymorphism%20predicts%20blood%20pressure%20response%20to%20angiotensin%20II%20receptor%20type%201%20antagonist%20treatment%20in%20hypertensive%20patients&author=L%20Kurland&author=H%20Melhus&author=J%20Karlsson&volume=19&issue=10&publication_year=2001&pages=1783-1787&pmid=11593098&doi=10.1097/00004872-200110000-00012&)

42. Ye Q, Wang B, Mao J. The pathogenesis and treatment of the `Cytokine Storm’ in COVID-19. J Infect. 2020;80(6):607–613. doi: 10.1016/j.jinf.2020.03.037  [DOI](https://doi.org/10.1016/j.jinf.2020.03.037) | [PMC free article](/articles/PMC7194613/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32283152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Infect&title=The%20pathogenesis%20and%20treatment%20of%20the%20%60Cytokine%20Storm%E2%80%99%20in%20COVID-19&author=Q%20Ye&author=B%20Wang&author=J%20Mao&volume=80&issue=6&publication_year=2020&pages=607-613&pmid=32283152&doi=10.1016/j.jinf.2020.03.037&)

43. Strianese O, Rizzo F, Ciccarelli M, et al. Precision and personalized medicine: how genomic approach improves the management of cardiovascular and neurodegenerative disease. Genes (Basel). 2020;11(7):747. doi: 10.3390/genes11070747  [DOI](https://doi.org/10.3390/genes11070747) | [PMC free article](/articles/PMC7397223/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32640513/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20(Basel)&title=Precision%20and%20personalized%20medicine:%20how%20genomic%20approach%20improves%20the%20management%20of%20cardiovascular%20and%20neurodegenerative%20disease&author=O%20Strianese&author=F%20Rizzo&author=M%20Ciccarelli&volume=11&issue=7&publication_year=2020&pages=747&pmid=32640513&doi=10.3390/genes11070747&)

44. World Medical Association Declaration of Helsinki. Ethical principles for medical research involving human subjects [document on the Internet]. Revised by the 64th WMA General Assembly, Fortlazea, Brazil, October 2013. Ferney-Voltaire: WMA; 2013. Available from: http://www.wma.net/e/policy/b3.html.  [http://www.wma.net/e/policy/b3.html](http://www.wma.net/e/policy/b3.html) | [Google Scholar](https://scholar.google.com/scholar_lookup?World%20Medical%20Association%20Declaration%20of%20Helsinki.%20Ethical%20principles%20for%20medical%20research%20involving%20human%20subjects%20%5Bdocument%20on%20the%20Internet%5D.%20Revised%20by%20the%2064th%20WMA%20General%20Assembly,%20Fortlazea,%20Brazil,%20October%202013.%20Ferney-Voltaire:%20WMA;%202013.%20Available%20from:%20http://www.wma.net/e/policy/b3.html.)
