# New genetic variants associated with major adverse cardiovascular events in patients with acute coronary syndromes and treated with clopidogrel and aspirin

## Metadata
**Authors:** Xiaomin Liu, Hanshi Xu, Huaiqian Xu, Qingshan Geng, Wai-Ho Mak, Fei Ling, Zheng Su, Fang Yang, Tao Zhang, Jiyan Chen, Huanming Yang, Jian Wang, Xiuqing Zhang, Xun Xu, Huijue Jia, Zhiwei Zhang, Xiao Liu, Shilong Zhong
**Journal:** The Pharmacogenomics Journal
**Date:** 2021 Jun 22
**DOI:** [10.1038/s41397-021-00245-5](https://doi.org/10.1038/s41397-021-00245-5)
**PMID:** 34158603
**PMCID:** PMC8602039
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8602039/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8602039/pdf/41397_2021_Article_245.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8602039/pdf/41397_2021_Article_245.pdf)

## Abstract

Although a few studies have reported the effects of several polymorphisms on major adverse cardiovascular events (MACE) in patients with acute coronary syndromes (ACS) and those undergoing percutaneous coronary intervention (PCI), these genotypes account for only a small fraction of the variation and evidence is insufficient. This study aims to identify new genetic variants associated with MACE end point during the 18-month follow-up period by a two-stage large-scale sequencing data, including high-depth whole exome sequencing of 168 patients in the discovery cohort and high-depth targeted sequencing of 1793 patients in the replication cohort. We discovered eight new genotypes and their genes associated with MACE in patients with ACS, including MYOM2 (rs17064642), WDR24 (rs11640115), NECAB1 (rs74569896), EFR3A (rs4736529), AGAP3 (rs75750968), ZDHHC3 (rs3749187), ECHS1 (rs140410716), and KRTAP10-4 (rs201441480). Notably, the expressions of MYOM2 and ECHS1 are downregulated in both animal models and patients with phenotypes related to MACE. Importantly, we developed the first superior classifier for predicting 18-month MACE and achieved high predictive performance (AUC ranged between 0.92 and 0.94 for three machine-learning methods). Our findings shed light on the pathogenesis of cardiovascular outcomes and may help the clinician to make a decision on the therapeutic intervention for ACS patients.

Subject terms: Genetic association study, Cardiovascular diseases, Outcomes research

## Introduction

As a standard treatment procedure for patients suffering from ACS and those undergoing PCI with stenting, dual-antiplatelet therapy (DAPT) with clopidogrel in addition to aspirin, significantly reduces the risk of adverse cardiac events in patients [[1](#CR1)]. However, the pharmacodynamic response to DAPT varies substantially among patients [[2](#CR2)].

Studies have reported the associations of several gene polymorphisms (*CYP2C19*2*, *CYP2C19*3, CYP2C9*2*, *PON1* Q192R, and *ABCB1* C3435T) with cardiovascular outcomes in patients with ACS and those undergoing PCI [[3](#CR3)–[8](#CR8)]. However, there are meta-analysis studies failed to support the associations of *CYP2C19* and cardiovascular events [[9](#CR9), [10](#CR10)]. In summary, these studies about the effects of single-nucleotide variants (SNPs) on clinical outcomes were inconsistent and inconclusive. Meanwhile, most of the previous studies were conducted in western populations, Asians have very different genotype distributions of *CYP2C19* and *PON1* Q192R, for example, the allele frequency of *CYP2C19*2* was much higher in the Asian population (35%) than Africans and Caucasians (15%) [[11](#CR11)], *CYP2C19*3* and *CYP2C19*17* in Chinese population was far lower than other populations, and the frequency of the *PON1* 192Q allele in Chinese patients is also relatively lower than in Caucasians [[12](#CR12)], in whom relevant studies based on large samples are scarce. Therefore, it is very necessary to identify the association of genetic polymorphisms with cardiovascular events and offer valuable information for intervention in Han Chinese population. Moreover, previous studies on cardiac adverse events after PCI mainly focused on a few specific genotypes or genes [[3](#CR3)–[10](#CR10)]. Therefore, the majority of the hereditability in affecting cardiovascular events remains unexplained, and other important genetic determinants have yet to be identified. Genome-wide studies, such as whole-exome sequencing, should be conducted to enhance the understanding of this research field.

In the present study, we combined whole-exome and targeted sequencing to investigate the genetic factors associated with the major adverse cardiovascular events (MACE) among patients receiving clopidogrel and aspirin treatments after PCI. The logistic regression and Cox proportional hazard models were used to analyze 1961 samples in two independent cohorts with detailed clinical information, respectively. This process aimed to evaluate the previously reported cardiovascular outcome-related loci and discovered novel genes/alleles associated with the effect of treatment with clopidogrel and aspirin on 18-month MACE in the Han Chinese ethnic group. In addition, we performed the gene-based analysis to further replenish the candidate gene set that maybe missing in single-variant association analysis. To date, machine-learning methods have not been substantially applied to develop effectively predictive prognostic classifiers for adverse cardiovascular events. Thus, we developed SVM classifier to predict the occurrence possibility of 18-month MACE, as well as to provide a target for therapeutic intervention of patients with ACS.

## Materials and methods

### Study population and design

In this study, we included a total of 1961 patients with acute coronary syndromes (ACS) undergoing PCI obtained through Guangdong General Hospital in China from 2009 to 2012. These patients were treated with 12 months of dual-antiplatelet therapy with clopidogrel in addition to aspirin following stent implantation. All patients had detailed baseline and follow-up information during 18-month follow-up periods. Patient information was collected based on inpatient and outpatient hospital visits, and telephone contacts with the patients or their family at 1-, 6-, 12-, and 18-months following discharge. During the 18-month follow-up period, trained staff systematically recorded medical conditions of each patient to determine the occurrence of major adverse cardiovascular events (MACE). An experienced cardiologist adjudicated the end point through review of source documents obtained from medical records. The major adverse cardiovascular events (MACE) as a composite endpoint included cardiovascular death, myocardial infarction (MI), stroke (CT or MR scan confirmed) and repeated revascularization (RR). Repeat revascularization included target vessel revascularization–PCI, nontarget vessel revascularization–PCI, and coronary artery bypass grafting (CABG).

A two-stage analysis was performed in this study. In the discovery cohort, we randomly selected 168 patients for whole-exome sequencing from the total 1021 individuals who have been diagnosed with ACS, have undergone PCI operation in Guangdong General Hospital, and have taken clopidogrel for treatment from 2009 to 2010, of which 51 had MACE end point during the 1-year follow-up period and 117 did not have any clinical events during 18-month follow-up periods. In the replication cohort, we performed targeted sequencing for all 1793 hospitalized patients with ACS and receiving PCI from 2010 to 2012. As replication, 1703 samples went into subsequent analysis after sample quality control, of which 123 had MACE end point during the 18-month follow-up period and 1580 did not have any clinical events during that period. This study has been registered at [http://www.chictr.org.cn](http://www.chictr.org.cn) on 8 March 2011 (registration number: ChiCTR-OCH-11001198).

This study was approved by both the Guangdong general hospital ethics committee and the BGI ethics committee. All protocols were conducted in compliance with the Declaration of Helsinki. All participants provided written informed consent to take part in the study.

### Sample sequencing in two stages

We sequenced the whole exome of 168 patients in the discovery cohort. Genomic DNA for each sample was used to produce each exome-captured library with the NimbleGen SeqCap EZ Exome (44MB, Roche) array. Then each captured library was independently sequenced on the Illumina Hiseq 2000 platform. Each sample was designed to get high-quality bases with coverage of >90×.

We sequenced the 6-MB targeted region of 1793 patients in the replication cohort. The targeted region consisted of three parts. The first part was the top associated SNVs with *P* < 0.05 in the discovery stage. The second part was top genes with *P* < 0.05 in gene-based test in the discovery study. Gene-based analysis was performed using the Fast Association Tests (FAST) tool [[13](#CR13)], which includes a series of gene-based methods. Four algorithms, including (i) Gene-Wide Significance test (GWiS); (ii) MinSNP-p and MinSNP-gene; (iii) Versatile Gene-Based Test for Genome-wide Association (VEGAS); (v) the Gates test (GATES), were used to test the gene-based association with MACE. The significant gene (*P* < 0.05) in at least three of the four tests was selected for the targeted region design. The third part was 49 reported genes within the pharmacokinetic and pharmadynamics pathway of clopidogrel, aspirin, statin, or beta-blockers. All the three parts were combined and merged into 6MB target regions. Then the targeted region for each sample was sequenced on Complete Genomics (CG) platform [[14](#CR14)] with high quality and coverage.

### Sample quality control

We required the samples to meet these criteria: (i) average sequencing depth ≥90× in exome-sequencing stage, (ii) average sequencing depth ≥30× in targeted sequencing stage, (iii) genotype calling rate ≥90%, (iv) not-existing population stratification by performing principal component analysis (PCA) analysis that was performed via the multidimensional scaling (MDS) procedure implemented in PLINK v 1.07 [[15](#CR15)], and (v) not be duplicates or first-degree relatives while evaluating pairwise by identity by descent (IBD). After quality control filtering, 90 samples from the targeted sequencing stage were excluded from subsequent analysis.

### Alignment, variant calling, and quality control

For Illumina exome-sequencing data, reads were mapped to human genome reference assembly (hg19, GRCh37) with SOAP2 [[16](#CR16)] and variants were detected by SOAPsnp [[17](#CR17)]. The high-quality Illumina SNVs that we defined for each individual had to meet the following condition: sequenced quality ≥Q20, sequencing depth ≥8× and ≤500×, and depth of nonreference allele ≥4×.

For CG-targeted sequencing data, sequence reads were also aligned to human reference genome hg19 and variations were detected using the CG analysis toolkit (CGATools) which is available at CG website ([http://cgatools.sourceforge.net/](http://cgatools.sourceforge.net/)).

After all initial SNV calls from Illumina and CG platforms were generated, further filtering was performed to identify high-confidence SNVs. We required SNVs from discovery and replication studies to meet two conditions: (i) genotype calling rate ≥90%, (ii) minor allele frequency (MAF) ≥ 0.01, and (iii) Hardy–Weinberg equilibrium (HWE) *P* > 1.0 × 10^−6^.

To evaluate the data quality, we compared the genotypes from the sequencing data with the genotypes called from the genotyping arrays. In the discovery study, the average genotype concordance is 98.6% by comparing 5 genotypes overlapping in the exome-sequencing data and genotyping array data in 126 samples (Table [S1](#MOESM1)). In the replication study, the whole-genome sequencing result of the YanHuang (YH) sample was obtained using CG platform. We evaluated the CG data quality by comparing the whole-genome sequencing result and genotyping results in the YH sample and the genotype concordance was 99.5%.

### Statistical analysis

Analysis was performed using PLINK (version 1.07) [[15](#CR15)] and R (version 3.2.3, [http://www.R-project.org/](http://www.R-project.org/)). The demographic and clinical characteristics were summarized using counts (percentages) for the categorical variables (e.g., sex, previous MI, and diabetes mellitus) and mean (standard deviation, SD) for the continuous variables (e.g., age, BMI). A multivariate Cox proportional hazard model [[18](#CR18)] was used to calculate the significance by comparing the baseline demographic and clinical characteristics between the groups with and without MACE in the follow-up periods of 18-months. Baseline variables and genetic variants explanation for MACE incidence during the 18-month follow-up were calculated by using a regression-based approach as implemented in the SOLAR-Eclipse version 8.1.1 software ([http://solar-eclipse-genetics.org/index.html](http://solar-eclipse-genetics.org/index.html)).

In the discovery cohort, we applied the logistic regression to calculate the P values and odds ratio (OR) of SNPs on the clinical end points by adjusting for 17 variables, including the first four principal components PC1–PC4, three demographic (sex, age, and BMI) and 10 clinical variables (e.g., diabetes mellitus, hypertension, and previous MI). In the replication cohort, we use the multivariate Cox proportional hazard regression to model the survival time and the incidence of MACE by adding SNPs and the same 17 adjustment variables as covariates to evaluate the hazard ratio (HR) and *P* value for each SNP.

The Cox model is expressed by the hazard function denoted by h(t). Briefly, the hazard function can be interpreted as the risk of MACE at time t. It can be estimated as follows:

*h*(*t*)=*h*_0_(*t*)×exp(*b*_1_*x*_1_ + *b*_2_*x*_2_ + …+*b*_p_*x*_p_)

where

We fit the multivariate Cox proportional hazards regression model with coxph() function in survival package in R. The coxph function was written as

>coxph(Surv(time, status) ~age + sex + … + genotypes, data = MACE)

Finally, we performed a meta-analysis of these two datasets, via inverse-variance weighted fixed effect meta-analysis method [[19](#CR19)] based on log hazard ratio and standard error. Before that, the Cox regression model was also implemented for the discovery cohort to get the HR for meta-analysis.

Power analysis was used to investigate if we have enough power to detect the associated SNPs in the sample size of the replication cohort. R package survSNP (version 0.23.2) was used to evaluate the power for SNPs with different HR or MAF.

We estimated a variant as ‘deleterious’ (or ‘functional’ or ‘damaging’) or ‘benign’ (or ‘nonfunctional’ or ‘neutral’) using Polyphen2 [[20](#CR20)] and combined annotation-dependent depletion (CADD) [[21](#CR21)] methods. A variant with CADD score >15 was defined as “damaging” as recommended by Itan et al. [[22](#CR22)].

### Gene expression analysis

We mined four publicly available genome-wide expression data sets from Gene Expression Omnibus (GEO) database in NCBI, including [GSE27962](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE27962) [[23](#CR23)], [GSE48060](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48060) [[24](#CR24)], [GSE7487](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE7487) [[25](#CR25)] and [GSE47495](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE47495) [[26](#CR26)]. These datasets recorded gene expression data of cardiac remodeling after myocardial infarction or myocardial infarction-induced heart failure. After downloading the four publicly available datasets, we first detected the expression data distribution for the eight genes in this study, then we applied log transformation and quantile normalization to the expression data if they were not normally distributed. To compare the gene expression difference between case groups (mainly myocardial infarction (MI) operation) and control groups (sham operation), we used unpaired Student’s *t*-test to compute the significance as GEO website recommended. Data are presented as mean ± SE (standard error). Finally, we corrected for multiple testing of the eight genes and genes having a threshold of *P* < 0.00625 = 0.05/8 were considered to be differentially expressed between two groups.

### ECHS1 plasma experiments

We randomly selected ACS patients with heart failure (HF) symptoms at NYHA stage II or less (*n* = 61), stage III or IV (*n* = 89) from an independent study cohort. *ECHS1* protein levels were measured in plasma sample using sandwich enzyme-linked immunosorbent assays (ELISA) (ECHS1 ELISA kit, Action-award Biotech co. Ltd., Guangzhou, China) and a Multiskan GO Microplate Reader (Thermo Scientific Inc., USA). The comparison of parameters and *ECHS1* protein levels between patients with HF at stage II or less (*n* = 61) and patients at stage III or IV (*n* = 89) was performed using the logistic regression analysis. Because the distribution of *ECHS1* protein levels was skewed, logarithmic transformation was performed before analysis.

### Prediction of 18-month MACE

This study used three machine-learning models, including support vector machine (SVM) method [[27](#CR27)], Light Gradient Boosting Machine (LightGBM) [[28](#CR28)] and XGBoost [[29](#CR29)], to predict the binary phenotype—whether MACE occurred in a patient during the 18-month follow-up. The prediction analysis was completely independent from the association analyses, and all the genetic and nongenetic factors were input for analysis. There are 14,253 SNPs overlapped between the discovery and replication cohorts. After LD pruning (PLINK -indep-pairwise 50 10 0.2), 7246 independent SNPs remained. These 7246 independent SNPs as well as 20 clinical factors entered into the three machine-learning models for prediction. The whole dataset was split into 80% and 20% subsets for model training and testing, using stratified folds by balancing the percentage of samples for each class in train and test. We applied a fivefold cross-validation procedure on the train dataset to select the best hyperparameters. One fold was used as the validation set and the other four folds were involved in parameter tuning and model constructing. We repeated the process until each one of the five folds was used as the validation set once. Then we constructed the final model on the train data using the best parameters and calculated the feature importance. In each model, the factors with feature importance over zero were selected for prediction. Finally, we estimated the predictive performance based on the final model on the test dataset. Feature importance was also output to help us pinpoint the important factors that contributed to 18-month MACE. Three models were utilized so that we were able to compare their prediction performances, which could potentially reduce the bias from simply assessing one model. All the modeling procedures were conducted in Python (v3.7.8) with scikit-learn package and publicly available packages.

## Results

### Characteristics of the patients

Two independent cohorts were recruited and comprised 1961 patients with ACS who underwent PCI and treated with clopidogrel and aspirin for 6–12 months in accordance with the consensus guidelines. After performing sample quality control, 168 patients were included in the discovery study and 1703 patients remained in the replication study (Table [S2](#MOESM1) and [S3](#MOESM1)). For the total 1871 patients, the average age was 63.3 (±10.7) years, 357 (19.1%) were women, 174 (9.3%) had a MACE endpoint during the 18-month follow-up period. Table [1](#Tab1) shows the association of demographic and clinical characteristics with MACE. MACE was associated with increasing age (65.84 ± 10.17 vs. 63.07 ± 10.74, *P* < 0.001), hypertension (68% vs. 56%, *P* < 0.001), and high creatinine (100.9 ± 85.96 vs. 87.03 ± 44.84, *P* < 0.001) (Table [1](#Tab1)). These combined clinical variables explained 5.80% of occurrence of the MACE, thus suggesting potentially substantial genetic contribution.

### Table 1.

| Characteristics | β (95% CI) | P value | Variants explanation |
| --- | --- | --- | --- |
| Age – yrs, mean (±s.d.) | 1.025(1.010–1.040) | <0.001*** | 0.94% |
| Sex, Men – no. (%) | 1.073(0.7404–1.554) | 0.71 |   |
| BMI –kg/m2, mean (±s.d.) | 1.016(0.9593–1.077) | 0.58 |   |
| Risk factors, n (%) |  |  |  |
| Previous MI | 1.075(0.7905–1.461) | 0.65 |   |
| Diabetes mellitus | 1.322(0.9657–1.809) | 0.08 |   |
| Hypertension | 1.689(1.227–2.325) | <0.001*** | 0.87% |
| Medications used before event |  |  |  |
| ACEI_ARB | 1.356(0.8811–2.086) | 0.17 |   |
| BBI | 1.178(0.7313–1.896) | 0.5 |   |
| CCB | 1.231(0.8747–1.733) | 0.23 |   |
| PPI | 1.269(0.9425–1.709) | 0.12 |   |
| Statins | 1.069(0.3967–2.882) | 0.89 |   |
| Blood metabolites level |  |  |  |
| HDLC, mmol/L | 0.8664(0.4562–1.646) | 0.66 |   |
| LDLC, mmol/L | 0.8714(0.7305–1.040) | 0.13 |   |
| Triglycerides, mmol/L | 0.9466(0.8074–1.110) | 0.5 |   |
| HbA1c, % total hemoglobin | 0.8917(0.7732–1.028) | 0.12 |   |
| ALT, U/L | 1.002(1–1.005) | 0.06 |   |
| AST, U/L | 0.9973(0.993–1.002) | 0.21 |   |
| CREA, umol/L | 1.003(1.001–1.005) | <0.001*** | 0.73% |
| CK, U/L | 0.9994(0.9987–1) | 0.28 |   |
| CKMB, U/L | 0.9813(0.9622–1.001) | 0.68 |   |

Table 1 Caption: Demographic and clinical characteristics contributed to MACE in ACS patients (n = 1871).

### Two-stage association study

The study design and total workflow is shown in Fig. [1](#Fig1). In the discovery study, we deep-sequenced the whole exome of the 168 ACS patients with a mean coverage of approximately 210× (Table [S4](#MOESM1), Fig. [S1](#MOESM1)). All cases with MACE and controls without MACE were ethnically and genetically well matched (Fig. [S2](#MOESM1)). A total of 127,834 SNPs passed the quality control for the single-variant association analysis. Logistic regression analysis determined 6268 SNPs associated with MACE with *P* < 0.05 adjusting for the covariates **(**Fig. [S3](#MOESM1)). Gene-based association analysis identified 408 genes associated with MACE at the *P* < 0.05 level (Table [S5](#MOESM1)).

### Fig. 1. Description of the study design.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fdca/8602039/141153005fe1/41397_2021_245_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8602039_41397_2021_245_Fig1_HTML.jpg)

First, we performed whole exome sequencing of 168 patients and 51 out of them had MACE end point. After quality filtering, a total of 127,834 variants were subjected to single variant association analysis and 6268 variants showed nominal association (P < 0.05). Gene-based association analyses identified 408 genes associated with MACE (P < 0.05). As validation, the 6MB targeted region including 6268 top SNPs and 408 top genes was further analyzed in additional 1703 patients through multivariable Cox regression analysis. A total of 177 SNPs and 82 genes were replicated in validation datasets. Finally, we performed meta-analysis of the two-stage associations and identified eight genetic variants contributed to MACE (P < 7.98 × 10−6 = 0.05/6268). Then, we performed functional analysis on the eight significant SNPs or genes; further, we developed the first superior classifier for predicting MACE.

To further replicate the identified associations in the discovery cohort and increase the statistical power of the present study, we performed a replication analysis in an independent cohort of 1703 patients using targeted sequencing. In the replication analysis, a total of 6-MB (million bases) targeted regions consisting of SNP/gene associations discovered above and previously reported, were sequenced with approximately 94× coverage for each individual (Table [S5](#MOESM1), Fig. [S1](#MOESM1)). After performing quality control on the samples and variants, 28,316 SNPs entered the subsequent analyses (Fig. [S3](#MOESM1)**)**. Then, we performed replication analysis on the 6268 SNPs and 408 genes and found 177 replicated SNPs for MACE with *P* < 0.05 both in the discovery and replication studies. Although no SNPs reached the multiple test-adjusted significance in the replication cohort, there are eight SNPs left after multiple-test correction (*P* < 7.98 × 10^−6^ = 0.05/6,268; Table [2](#Tab2)) while performing the meta-analysis of two-stage data.

### Table 2.

| Chr. | Gene | Variant ID | Stagea | Alleles | RAF | OR/HR+ (95% CI) | Pvalue | Meta Pvalue |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 8 | MYOM2 | rs17064642 | i | C/T | 0.083 | 6.48(1.83–15.01) | 2.95 × 10−4 | 1.84 × 10−7 |
| ii | C/T | 0.064 | 2.40(1.59–3.63) | 3.26 × 10−5 |   |  |  |  |
| 16 | WDR24 | rs11640115 | i | G/A | 0.712 | 3.13(1.25–3.85) | 2.13 × 10−3 | 3.21 × 10−7 |
| ii | G/A | 0.694 | 2.08(1.47–2.88) | 2.56 × 10−5 |   |  |  |  |
| 8 | NECAB1 | rs74569896 | i | G/A | 0.138 | 2.87(1.51–5.45) | 3.87 × 10−3 | 1.31 × 10−6 |
| ii | G/A | 0.154 | 1.90(1.39–2.61) | 5.91 × 10−5 |   |  |  |  |
| 8 | EFR3A | rs4736529 | i | G/C | 0.060 | 3.16(1.04–9.61) | 4.25 × 10−2 | 2.52 × 10−6 |
| ii | G/C | 0.060 | 2.41(1.45–4.27) | 1.95 × 10−5 |   |  |  |  |
| 7 | AGAP3 | rs75750968 | i | A/T | 0.036 | 3.38(1.05–10.9) | 2.90 × 10−2 | 4.85 × 10−6 |
| ii | A/T | 0.021 | 3.21(1.83–5.66) | 5.21 × 10−5 |   |  |  |  |
| 3 | ZDHHC3 | rs3749187 | i | A/G | 0.030 | 6.12(1.17–34.01) | 4.12 × 10−2 | 5.19 × 10−6 |
| ii | A/G | 0.027 | 2.99(1.26–4.14) | 3.37 × 10−5 |   |  |  |  |
| 10 | ECHS1 | rs140410716 | i | T/C | 0.018 | 12.01(1.10–104.1) | 1.49 × 10−2 | 6.88 × 10−6 |
| ii | T/C | 0.015 | 3.32(1.84–5.99) | 6.54 × 10−5 |   |  |  |  |
| 21 | KRTAP10–4 | rs201441480 | i | A/C | 0.024 | 7.25(1.44–36.56) | 3.12 × 10−3 | 7.26 × 10−6 |
| ii | A/C | 0.011 | 3.86(1.87–7.99) | 2.72 × 10−4 |   |  |  |  |

Table 2 Caption: Identifying eight genetic variants contributed to MACE.

### Identify single variants and genes associated with 18-month MACE

The most significant SNP associated with 18-month MACE was rs17064642 at *MYOM2* (*P* = 1.84 × 10^−7^; Table [2](#Tab2)). Carriers of the CC/CT genotype had higher MACE occurrence rates during the 18 months of follow-up compared with noncarriers (17.8% vs. 8.1%; HR, 2.76; 95% CI, 1.98–3.87) (Fig. [2](#Fig2)). Rs17064642 has moderate linkage disequilibrium (LD) of r^2^ = 0.57 (East Asians) with the missense SNP rs34823600 and these two SNPs are located only 16 bp apart (Table [S6](#MOESM1)). Similarly, rs34823600 was also significantly contributed to MACE (HR 2.25, *P* = 5.32 × 10^−4^). Rs17064642 and rs34823600 both coincide with enhancer markers (H3K4me1_Enh and H3K27ac_Enh) in four tissues, especially in heart cell types (right atrium, left ventricle and right ventricle), suggesting this locus may function as an enhancer in heart tissue (Fig. [S4](#MOESM1)).

### Fig. 2. Event-free survival over 18 months of follow-up in 1703 patients with ACS.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fdca/8602039/c390df75a90a/41397_2021_245_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8602039_41397_2021_245_Fig2_HTML.jpg)

Cumulative probabilities of survival without MACE according to gene polymorphisms: MYOM2 (rs17064642), WDR24 (rs11640115), NECAB1 (rs74569896), EFR3A (rs4736529), AGAP3 (rs75750968), ZDHHC3 (rs3749187), ECHS1 (rs140410716), and KRTAP10-4 (rs201441480). The red, green, and blue colors represented the genotypes containing two, one or zero copy of the risk allele, respectively.

The other seven significant associations included *WDR24* (rs11640115), *NECAB1* (rs74569896), *EFR3A* (rs4736529), *AGAP3* (rs75750968), *ZDHHC3* (rs3749187), *ECHS1* (rs140410716), and *KRTAP10-4* (rs201441480) (Table [2](#Tab2)). At *WDR24*, we identified two SNPs associated with MACE, rs11640115 (HR 2.08, *P* = 3.21 × 10^−7^) and rs763053 (HR 1.92, *P* = 8.02 × 10^−7^), which were in high LD (D′ = 0.98, r² = 0.96, Table [S6](#MOESM1)). According to public GTEx [[30](#CR30)] databases, both rs11640115 and rs763053 at *WDR24* showed significant eQTL associations with gene *WDR90* (*P* = 1.16 × 10^−6^) in the heart tissues, including heart left ventricle and heart atrial appendage (Table [S6](#MOESM1)**)**. Evidence from [GSE48060](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48060) [[24](#CR24)] dataset confirmed that *WDR90* transcription was associated with long-term recurrent events following first-time MI (0.122 ± 0.031, *P* = 4.78 × 10^−3^, Table [S7](#MOESM1) and [S8](#MOESM1)). Similarly, we also identified two SNPs at *NECAB1* associated with MACE, rs74569896 (HR 1.90, *P* = 1.31 × 10^−6^) and rs73694346 (HR 1.72, *P* = 4.86 × 10^−5^), which were in strong LD (D′ = 0.96, r² = 0.92). Rs4736529 at *EFR3A* increased the risk of occurring MACE and expression of gene *EFR3A* was significantly upregulated in the coronary artery disease (CAD) patients compared with the control group [[31](#CR31)]. However, *EFR3A* showed significantly decreased expression in the left ventricular remodeling in swine after myocardial infarction cardiac by analyzing the GEO database [GSE27962](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE27962) [[23](#CR23)] (*P* = 6.94 × 10^−4^). SNP rs75750968 locates in gene *AGAP3*, which relates to GTP binding and GTPase activator activity. Rs74569896 located in *NECAB1* binds to the protein *GATA3* in ChIP-Seq experiments (ENCODE Project Consortium, 2011) and lack of *GATA3* results in conotruncal heart anomalies in mouse [[32](#CR32)]. Rs201441480 at *KRTAP10-4*, which is a missense alteration that was predicted to be damaging by both Polyphen2 [[20](#CR20)] and CADD [[21](#CR21)] methods, significantly increased the risk of MACE (HR 3.86, *P* = 7.26 × 10^−6^). Together, we discovered eight novel genetic variants associated with MACE and achieved an average of 94% GWAS statistical power for the eight SNPs, of which, 99.7% statistical power for rs11640115 at *WDR24* (Table [S9](#MOESM1), Fig. [S5](#MOESM1)). Fig. [2](#Fig2) showed cumulative proportions of individuals without MACE over 18 months of follow-up under the eight polymorphisms.

Since we got the whole-exome and targeted region sequencing data, we further performed the gene-based analysis and confirmed the significance of the eight genes (Table [3](#Tab3)). Together, we identified eight novel genetic variants and their genes contributed to 18-month MACE.

### Table 3.

| Gene | Stage i |  | Stage ii |  |
| --- | --- | --- | --- | --- |
| Gene. Pvalue | TopSNP. Pvalue | Gene. Pvalue | TopSNP. Pvalue |  |
| MYOM2 | 3.54E-02 | 2.95E-04 | 2.75E-02 | 8.87E-05 |
| WDR24 | 7.33E-03 | 2.13E-03 | 2.64E-04 | 5.36E-05 |
| NECAB1 | 8.82E-02 | 2.77E-03 | 6.73E-03 | 1.32E-04 |
| EFR3A | 4.20E-02 | 2.32E-02 | 7.43E-03 | 3.41E-05 |
| AGAP3 | 4.16E-02 | 1.93E-02 | 2.17E-02 | 1.26E-03 |
| ZDHHC3 | 4.12E-02 | 1.25E-02 | 3.11E-04 | 3.21E-05 |
| ECHS1 | 1.30E-02 | 1.49E-02 | 1.70E-04 | 8.73E-05 |
| KRTAP10-4 | 3.23E-02 | 3.12E-03 | 8.18E-03 | 7.94E-04 |

Table 3 Caption: Gene-based results showing eight genes contributed to MACE.

### Biological implications of variants associated with 18-month MACE

To further explore the functional evidence of the eight genes that contribute to 18-month MACE, we mined several publicly available genome-wide expression data sets from the GEO database (Table [S8](#MOESM1)) [[23](#CR23)–[26](#CR26)], which recorded cardiac remodeling data after myocardial infarction (MI). We found that two of eight genes, *MYOM2* and *ECHS1*, showed abundant evidences of decreased expression in cases with major adverse cardiac events, either in previous literatures or in our data analysis. M-protein (myomesin-2) encoded by *MYOM2* or total myomesin is downregulated in cardiac hypertrophy in rats [[33](#CR33)], in acute myocardial infarction (AMI) patients [[34](#CR34)], or in chronic heart failure [[35](#CR35)]. Similarly, *ECHS1*, as an ischemic post-conditioning (PostC) modified protein, showed significantly decreased expression in the cardiac remodeling group after MI compared with the sham group, by analyzing the two different groups’ expression data from GEO database [GSE7487](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE7487) [[25](#CR25)] (7.748 ± 4.319 vs. 4.214 ± 1.668, *P* = 5.46 × 10^−6^) and [GSE47495](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE47495) [[26](#CR26)] (11.89 ± 0.11 vs. 12.13 ± 0.04, *P* = 7.28 × 10^−4^, Table [S6](#MOESM1) and [S7](#MOESM1)). Meanwhile, we compared the plasma *ECHS1* protein levels in ACS patients with HF symptoms at New York Heart Association (NYHA) stage III or IV (*n* = 89) to those with HF symptoms at NYHA stage II or less (*n* = 61). We found that advanced HF patients showed significantly decreased levels of *ECHS1* protein (215.18 ± 115.67 for stage II or less vs. 161.84 ± 76.67 for stage III or stage IV, *P* = 0.0012) (Table [S10](#MOESM1), Fig. [S6](#MOESM1)).

### Predictive effectiveness of 18-month MACE

One of the ultimate objectives for identifying factors associated with MACE is to predict whether MACE occurred during the follow-up period of 18 months for ACS patients receiving PCI. Based on all the genetic and nongenetic factors, including all 20 clinical factors and 14,253 genetic variants that overlapped in two-stage datasets (remaining 7226 SNPs after LD pruning), we constructed predictive models for 18-months MACE using three machine learning methods, including support vector machine (SVM) [[27](#CR27)], Light Gradient Boosting Machine (LightGBM) [[28](#CR28)], and XGBoost [[29](#CR29)]. These three predictive models showed similar performances, with AUCs ranging between 0.92 and 0.94 from the average of fivefold cross-validation (Fig. [3](#Fig3)). We generated the feature importance score plot from the best-performing model LightGBM and the result was listed in Table [S11](#MOESM2). Among the 7246 independent factors (20 clinical factors and 7226 LD-pruning SNPs), 195 factors were selected as predictors of 18-month MACE with importance score over zero in LightGBM model. Three clinical factors, creatinine (importance score 776), age (score 676) and hypertension (score 46), were identified as important predictors (Table [S11](#MOESM2)), which is consistent with our observational correlation findings (Table [1](#Tab1)). In addition, seven of the eight significant genes were also selected as predictors (Table [S11](#MOESM2)). We further investigated the significance of the SNPs selected by LightGBM for the prediction in our genetic association study. We found that those SNPs that effectively contributed to the prediction exhibited significantly lower *p*-value enrichment (Fig. [S7](#MOESM1)). Thus, the present study developed the first superior classifier for predicting 18-month MACE by selecting the most informative factors that independently contributed to MACE.

### Fig. 3. Receiver-operating characteristic (ROC) curves for prediction of 18-month MACE.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fdca/8602039/2a4fd7037841/41397_2021_245_Fig3_HTML.jpg)

The predictive effectiveness was evaluated by three machine learning models, including support vector machine (SVM, green), Light Gradient Boosting Machine (LightGBM, blue) and XGBoost (orange). A total of 7246 independent factors (20 clinical factors and 7226 LD-pruning SNPs) entered into each model for modeling, fitting and prediction.

## Discussion

To our knowledge, the present study is the first genome-wide and large-scale association analysis that integrates whole-exome and targeted sequencing to identify novel genetic variants associated with MACE in patients with clopidogrel and aspirin treatment after PCI. We also investigated the associations of five commonly reported SNPs [[4](#CR4), [6](#CR6)–[8](#CR8), [36](#CR36)–[38](#CR38)] with MACE. Our meta-analysis of the two-stage sequencing data confirmed that *PON1* (*P* = 0.0036) but not *CYP2C19* genetic variants contributed to cardiovascular outcomes in Han Chinese patients (Table [S12](#MOESM1)). Moreover, we identified eight novel genetic variants associated with MACE: *MYOM2* (rs17064642), *WDR24* (rs11640115), *NECAB1* (rs74569896), *EFR3A* (rs4736529), *AGAP3* (rs75750968), *ZDHHC3* (rs3749187), *ECHS1* (rs140410716), and *KRTAP10-4* (rs201441480).

To further explore the functional evidence of the eight genes that contribute to 18-month MACE, we searched all literatures referring to the eight genes and cardiovascular diseases in PubMed database, as well as mined previously reported publicly available GEO databases. Among the eight genes contributed to MACE, we find the expression levels of *MYOM2* and *ECHS1* are evidently downregulated in cases subjected to adverse cardiac events compared with normal controls. Interestingly, many clues reveal that *MYOM2* and *ECHS1* play an important role in heart function. First, *MYOM2* encodes the M-protein, which is also known as Myomesin-2. Myomesin-2 is the primary myosin M-band cross-linking protein and binds titin in a complex with obscurin/obs1. The protein is key to normal heart function, as evidenced by the associations between heart failure and low expression. Myomesin-2 showed decreased expression in multiple heart diseases or heart attack. An animal model of cardiac hypertrophy driven by the thyroid hormone (T3) in rats showed that the downregulated *MYOM2* causes significant contractile dysfunction (*P* < 0.05) [[33](#CR33)]. In addition, it has been reported that myomesin (encoded by both *MYOM1* and *MYOM*2) protein levels decrease in acute ischemia and in chronic heart failure [[35](#CR35)]. A proteomic analysis agrees with previous reports that the level of myomesin-2 in cardiac tissue is decreased in AMI patients (*n* = 10) compared with control cases (*n* = 11) [[34](#CR34)]. Second, the *ECHS1* protein (short-chain enoyl-CoA hydratase) is a multifunctional mitochondrial enzyme with several functions in β-oxidation of short- and medium-chain fatty acids, as well as in isoleukine and valine metabolism. A previous study reported that the mitochondrial protein *ECHS1* could regulate cellular ATP consumption/production and influence the defense response to myocardial ischemic stress [[39](#CR39)]. Haack et al. reported that *ECHS1* deficiency causes mitochondrial encephalopathy with cardiac involvement [[40](#CR40)]. Third, *ECHS1* showed significantly decreased expression in the cardiac remodeling group after MI compared with the sham group in two GEO datasets [GSE7487](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE7487) and [GSE47495](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE47495). We further confirmed that *ECHS1* contributed to an ischemic heart failure in ACS patients by gene expression experiment. Combining all these evidences, we concluded that *MYOM2* and *ECHS1* deficiency causes dysfunction of cardiac function, which will help us understand the occurrence mechanism of MACE.

Apart from identifying the novel genotypes and genes associated with MACE, we improved the prediction of MACE by developing a novel classifier for 18-month MACE, when compared with previous studies [[41](#CR41)–[44](#CR44)]. Previous studies were mainly based on clinical features; no comprehensive and complete genetic makers were available for the prognostic classification of MACE. In the current study, we included all candidate genetic variants of MACE to well construct predictive models. We confirmed genetic variants that significantly associated with MACE that efficiently contributed to prediction of MACE than other less-associated factors (Fig. [S7](#MOESM1)). In summary, we developed the first fine classifier that combined the clinical factors and multiple independently informative genotypes to predict 18-month MACE and achieved high accuracy (AUCs ranging between 0.92 and 0.94 from three machine-learning methods).

There are two issues that need to be addressed in this study. The first one is that the design of the discovery and the replication studies was different, the discovery stage is designed as the case–control study and the replication stage is designed as the cohort study. There was an obvious time interval between the patient cases and the controls in the discovery study, one is that the event occurred within 1-year old, the other is that the event did not occur within 1.5-years old, which can be clearly divided into case and control, and it is suitable for logical regression analysis. However, the cases and controls were defined as whether the event occurred within 1.5 years or not in the replication study; the multivariate Cox proportional hazards regression could model the survival time and the incidence of MACE simultaneously, thus, Cox proportional model is most suitable for replication study. In summary, this study is a complete and reasonable design that the discovery cohort is a screening for candidate target genes and we designed the 6-MB targeted region for further validation in the replication cohort. Although we used logistic regression analysis for the discovery cohort and performed multivariate Cox analysis for the replication cohort, some associations were still verified with each other and proved the reliability and consistency of our findings. The second issue was about the prognostic effectiveness of MACE classifier, we maybe overestimating the performance of the classifier because we have not got more new available independent samples as test dataset. Further validation and more work still need to be done in larger samples or more populations.

In conclusion, we provide here the first genome-wide, large-scale association analysis on ACS patients receiving clopidogrel and aspirin treatment after PCI. We successfully identified eight novel genes for MACE and found that these genetic variants may regulate the function of nearby genes. Especially, the expressions of *MYOM2* and *ECHS1* are downregulated in both animal models and patients with phenotypes related to MACE. Importantly, we developed the first superior classifier to predict 18-month MACE and achieved high accuracy. These findings will provide clinicians with potential biomarkers for an improved prediction of 18-month MACE and provide new insight on the therapeutics of ACS.

## Supplementary information

## Acknowledgements

We gratefully acknowledge Guangdong general hospital for sample collection and thank colleagues at BGI-Shenzhen for DNA extraction, library construction, sequencing, and discussions. The research was supported by the National Key R&D Program (No. 2017YFC0909301, 2016YFC0905003), National Natural Science Foundation of China (No. 81872934, 81673514, and 81373486), the Key-Area Research and Development Program of Guangdong Province, China (No. 2019B020229003), Science and Technology Planning Project of Guangdong Province, China (No. 2017B030314041), and Guangdong Provincial People’s Hospital Clinical Research Fund (Y012018085).

## Author contributions

S.Z. and Xiao Liu conceived of and designed the research. Xiaomin Liu, Hanshi Xu and Huaiqian Xu managed the project. S.Z., Q.G., J.C. and Z.Z. conducted sample selection and data management, H.Y., J.W., X.Z. and X.X. generated the sequence data. Xiaomin Liu, Hanshi Xu and Huaiqian Xu did statistical analyses and interpreted the data. W.M., F.L., Z.S., F.Y., T.Z. and H.J. discussed the results. Xiaomin Liu wrote the paper. Each author made critical revisions to the manuscript for important intellectual content. Each author is responsible for the content and the paper.

## Data availability

All materials and datasets generated and/or analyzed during the current study are available from the corresponding author on reasonable request.

## Competing interests

The authors declare no competing financial and nonfinancial interests.

## Footnotes

## Contributor Information

Xiao Liu, Email: liuxiao@sz.tsinghua.edu.cn.

Shilong Zhong, Email: gdph_zhongsl@gd.gov.cn.

## Supplementary information

The online version contains supplementary material available at 10.1038/s41397-021-00245-5.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All materials and datasets generated and/or analyzed during the current study are available from the corresponding author on reasonable request.

### Supplementary Materials

### Data Availability Statement

All materials and datasets generated and/or analyzed during the current study are available from the corresponding author on reasonable request.

## References

1. Zeymer U, Gitt A, Junger C, Bauer T, Heer T, Koeth O, et al. Clopidogrel in addition to aspirin reduces in-hospital major cardiac and cerebrovascular events in unselected patients with acute ST segment elevation myocardial. Thromb Haemost. 2008;99:155–60. doi: 10.1160/TH07-09-0556.  [DOI](https://doi.org/10.1160/TH07-09-0556) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18217148/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb%20Haemost&title=Clopidogrel%20in%20addition%20to%20aspirin%20reduces%20in-hospital%20major%20cardiac%20and%20cerebrovascular%20events%20in%20unselected%20patients%20with%20acute%20ST%20segment%20elevation%20myocardial&author=U%20Zeymer&author=A%20Gitt&author=C%20Junger&author=T%20Bauer&author=T%20Heer&volume=99&publication_year=2008&pages=155-60&pmid=18217148&doi=10.1160/TH07-09-0556&)

2. Gurbel PA, Bliden KP, Hiatt BL, O’Connor CM. Clopidogrel for coronary stenting: response variability, drug resistance, and the effect of pretreatment platelet reactivity. Circulation. 2003;107:2908–13. doi: 10.1161/01.CIR.0000072771.11429.83.  [DOI](https://doi.org/10.1161/01.CIR.0000072771.11429.83) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12796140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Clopidogrel%20for%20coronary%20stenting:%20response%20variability,%20drug%20resistance,%20and%20the%20effect%20of%20pretreatment%20platelet%20reactivity&author=PA%20Gurbel&author=KP%20Bliden&author=BL%20Hiatt&author=CM%20O%E2%80%99Connor&volume=107&publication_year=2003&pages=2908-13&pmid=12796140&doi=10.1161/01.CIR.0000072771.11429.83&)

3. Mega JL, Simon T, Collet JP, Anderson JL, Antman EM, Bliden K, et al. Reduced-function CYP2C19 genotype and risk of adverse clinical outcomes among patients treated with clopidogrel predominantly for PCI: a meta-analysis. JAMA. 2010;304:1821–30. doi: 10.1001/jama.2010.1543.  [DOI](https://doi.org/10.1001/jama.2010.1543) | [PMC free article](/articles/PMC3048820/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20978260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Reduced-function%20CYP2C19%20genotype%20and%20risk%20of%20adverse%20clinical%20outcomes%20among%20patients%20treated%20with%20clopidogrel%20predominantly%20for%20PCI:%20a%20meta-analysis&author=JL%20Mega&author=T%20Simon&author=JP%20Collet&author=JL%20Anderson&author=EM%20Antman&volume=304&publication_year=2010&pages=1821-30&pmid=20978260&doi=10.1001/jama.2010.1543&)

4. Shuldiner AR, O’Connell JR, Bliden KP, Gandhi A, Ryan K, Horenstein RB, et al. Association of cytochrome P450 2C19 genotype with the antiplatelet effect and clinical efficacy of clopidogrel therapy. JAMA. 2009;302:849–57. doi: 10.1001/jama.2009.1232.  [DOI](https://doi.org/10.1001/jama.2009.1232) | [PMC free article](/articles/PMC3641569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19706858/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Association%20of%20cytochrome%20P450%202C19%20genotype%20with%20the%20antiplatelet%20effect%20and%20clinical%20efficacy%20of%20clopidogrel%20therapy&author=AR%20Shuldiner&author=JR%20O%E2%80%99Connell&author=KP%20Bliden&author=A%20Gandhi&author=K%20Ryan&volume=302&publication_year=2009&pages=849-57&pmid=19706858&doi=10.1001/jama.2009.1232&)

5. Mega JL, Close SL, Wiviott SD, Shen L, Hockett RD, Brandt JT, et al. Cytochrome P-450 polymorphisms and response to clopidogrel. N. Engl J Med. 2009;360:354–62. doi: 10.1056/NEJMoa0809171.  [DOI](https://doi.org/10.1056/NEJMoa0809171) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19106084/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl%20J%20Med&title=Cytochrome%20P-450%20polymorphisms%20and%20response%20to%20clopidogrel&author=JL%20Mega&author=SL%20Close&author=SD%20Wiviott&author=L%20Shen&author=RD%20Hockett&volume=360&publication_year=2009&pages=354-62&pmid=19106084&doi=10.1056/NEJMoa0809171&)

6. Bouman HJ, Schomig E, van Werkum JW, Velder J, Hackeng CM, Hirschhauser C, et al. Paraoxonase-1 is a major determinant of clopidogrel efficacy. Nat Med. 2011;17:110–6. doi: 10.1038/nm.2281.  [DOI](https://doi.org/10.1038/nm.2281) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21170047/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=Paraoxonase-1%20is%20a%20major%20determinant%20of%20clopidogrel%20efficacy&author=HJ%20Bouman&author=E%20Schomig&author=JW%20van%20Werkum&author=J%20Velder&author=CM%20Hackeng&volume=17&publication_year=2011&pages=110-6&pmid=21170047&doi=10.1038/nm.2281&)

7. Chen Y, Huang X, Tang Y, Xie Y, Zhang Y. Both PON1 Q192R and CYP2C19*2 influence platelet response to clopidogrel and ischemic events in Chinese patients undergoing percutaneous coronary intervention. Int J Clin Exp Med. 2015;8:9266–74.  [PMC free article](/articles/PMC4538118/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26309584/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Exp%20Med&title=Both%20PON1%20Q192R%20and%20CYP2C19*2%20influence%20platelet%20response%20to%20clopidogrel%20and%20ischemic%20events%20in%20Chinese%20patients%20undergoing%20percutaneous%20coronary%20intervention&author=Y%20Chen&author=X%20Huang&author=Y%20Tang&author=Y%20Xie&author=Y%20Zhang&volume=8&publication_year=2015&pages=9266-74&pmid=26309584&)

8. Mega JL, Close SL, Wiviott SD, Shen L, Walker JR, Simon T, et al. Genetic variants in ABCB1 and CYP2C19 and cardiovascular outcomes after treatment with clopidogrel and prasugrel in the TRITON-TIMI 38 trial: a pharmacogenetic analysis. Lancet. 2010;376:1312–9. doi: 10.1016/S0140-6736(10)61273-1.  [DOI](https://doi.org/10.1016/S0140-6736(10)61273-1) | [PMC free article](/articles/PMC3036672/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20801494/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genetic%20variants%20in%20ABCB1%20and%20CYP2C19%20and%20cardiovascular%20outcomes%20after%20treatment%20with%20clopidogrel%20and%20prasugrel%20in%20the%20TRITON-TIMI%2038%20trial:%20a%20pharmacogenetic%20analysis&author=JL%20Mega&author=SL%20Close&author=SD%20Wiviott&author=L%20Shen&author=JR%20Walker&volume=376&publication_year=2010&pages=1312-9&pmid=20801494&doi=10.1016/S0140-6736(10)61273-1&)

9. Holmes MV, Perel P, Shah T, Hingorani AD, Casas JP. CYP2C19 genotype, clopidogrel metabolism, platelet function, and cardiovascular events: a systematic review and meta-analysis. JAMA. 2011;306:2704–14. doi: 10.1001/jama.2011.1880.  [DOI](https://doi.org/10.1001/jama.2011.1880) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22203539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=CYP2C19%20genotype,%20clopidogrel%20metabolism,%20platelet%20function,%20and%20cardiovascular%20events:%20a%20systematic%20review%20and%20meta-analysis&author=MV%20Holmes&author=P%20Perel&author=T%20Shah&author=AD%20Hingorani&author=JP%20Casas&volume=306&publication_year=2011&pages=2704-14&pmid=22203539&doi=10.1001/jama.2011.1880&)

10. Bauer T, Bouman HJ, van Werkum JW, Ford NF, ten Berg JM, Taubert D. Impact of CYP2C19 variant genotypes on clinical efficacy of antiplatelet treatment with clopidogrel: systematic review and meta-analysis. BMJ. 2011;343:d4588. doi: 10.1136/bmj.d4588.  [DOI](https://doi.org/10.1136/bmj.d4588) | [PMC free article](/articles/PMC3191560/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21816733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Impact%20of%20CYP2C19%20variant%20genotypes%20on%20clinical%20efficacy%20of%20antiplatelet%20treatment%20with%20clopidogrel:%20systematic%20review%20and%20meta-analysis&author=T%20Bauer&author=HJ%20Bouman&author=JW%20van%20Werkum&author=NF%20Ford&author=JM%20ten%20Berg&volume=343&publication_year=2011&pages=d4588&pmid=21816733&doi=10.1136/bmj.d4588&)

11. Scott SA, Sangkuhl K, Stein CM, Hulot JS, Mega JL, Roden DM, et al. Clinical Pharmacogenetics Implementation Consortium guidelines for CYP2C19 genotype and clopidogrel therapy: 2013 update. Clin Pharm Ther. 2013;94:317–23. doi: 10.1038/clpt.2013.105.  [DOI](https://doi.org/10.1038/clpt.2013.105) | [PMC free article](/articles/PMC3748366/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23698643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guidelines%20for%20CYP2C19%20genotype%20and%20clopidogrel%20therapy:%202013%20update&author=SA%20Scott&author=K%20Sangkuhl&author=CM%20Stein&author=JS%20Hulot&author=JL%20Mega&volume=94&publication_year=2013&pages=317-23&pmid=23698643&doi=10.1038/clpt.2013.105&)

12. Peng W, Shi X, Xu X, Lin Y. Both CYP2C19 and PON1 Q192R genotypes influence platelet response to clopidogrel by thrombelastography in patients with acute coronary syndrome. Cardiovasc Ther. 2019;2019:3470145. doi: 10.1155/2019/3470145.  [DOI](https://doi.org/10.1155/2019/3470145) | [PMC free article](/articles/PMC6739775/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31772608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Ther&title=Both%20CYP2C19%20and%20PON1%20Q192R%20genotypes%20influence%20platelet%20response%20to%20clopidogrel%20by%20thrombelastography%20in%20patients%20with%20acute%20coronary%20syndrome&author=W%20Peng&author=X%20Shi&author=X%20Xu&author=Y%20Lin&volume=2019&publication_year=2019&pages=3470145&pmid=31772608&doi=10.1155/2019/3470145&)

13. Chanda P, Huang H, Arking DE, Bader JS. Fast association tests for genes with FAST. PLoS ONE. 2013;8:e68585. doi: 10.1371/journal.pone.0068585.  [DOI](https://doi.org/10.1371/journal.pone.0068585) | [PMC free article](/articles/PMC3720833/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23935874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Fast%20association%20tests%20for%20genes%20with%20FAST&author=P%20Chanda&author=H%20Huang&author=DE%20Arking&author=JS%20Bader&volume=8&publication_year=2013&pages=e68585&pmid=23935874&doi=10.1371/journal.pone.0068585&)

14. Lam HY, Clark MJ, Chen R, Chen R, Natsoulis G, O’Huallachain M, et al. Performance comparison of whole-genome sequencing platforms. Nat Biotechnol. 2012;30:78–82. doi: 10.1038/nbt.2065.  [DOI](https://doi.org/10.1038/nbt.2065) | [PMC free article](/articles/PMC4076012/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22178993/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Performance%20comparison%20of%20whole-genome%20sequencing%20platforms&author=HY%20Lam&author=MJ%20Clark&author=R%20Chen&author=R%20Chen&author=G%20Natsoulis&volume=30&publication_year=2012&pages=78-82&pmid=22178993&doi=10.1038/nbt.2065&)

15. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–75. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

16. Li R, Yu C, Li Y, Lam TW, Yiu SM, Kristiansen K, et al. SOAP2: an improved ultrafast tool for short read alignment. Bioinformatics. 2009;25:1966–7. doi: 10.1093/bioinformatics/btp336.  [DOI](https://doi.org/10.1093/bioinformatics/btp336) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19497933/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=SOAP2:%20an%20improved%20ultrafast%20tool%20for%20short%20read%20alignment&author=R%20Li&author=C%20Yu&author=Y%20Li&author=TW%20Lam&author=SM%20Yiu&volume=25&publication_year=2009&pages=1966-7&pmid=19497933&doi=10.1093/bioinformatics/btp336&)

17. Li R, Li Y, Fang X, Yang H, Wang J, Kristiansen K, et al. SNP detection for massively parallel whole-genome resequencing. Genome Res. 2009;19:1124–32. doi: 10.1101/gr.088013.108.  [DOI](https://doi.org/10.1101/gr.088013.108) | [PMC free article](/articles/PMC2694485/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19420381/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=SNP%20detection%20for%20massively%20parallel%20whole-genome%20resequencing&author=R%20Li&author=Y%20Li&author=X%20Fang&author=H%20Yang&author=J%20Wang&volume=19&publication_year=2009&pages=1124-32&pmid=19420381&doi=10.1101/gr.088013.108&)

18. Bradburn MJ, Clark TG, Love SB, Altman DG. Survival analysis part II: multivariate data analysis-an introduction to concepts and methods. Br J Cancer. 2003;89:431–6. doi: 10.1038/sj.bjc.6601119.  [DOI](https://doi.org/10.1038/sj.bjc.6601119) | [PMC free article](/articles/PMC2394368/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12888808/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Survival%20analysis%20part%20II:%20multivariate%20data%20analysis-an%20introduction%20to%20concepts%20and%20methods&author=MJ%20Bradburn&author=TG%20Clark&author=SB%20Love&author=DG%20Altman&volume=89&publication_year=2003&pages=431-6&pmid=12888808&doi=10.1038/sj.bjc.6601119&)

19. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics. 2010;26:2190–1. doi: 10.1093/bioinformatics/btq340.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&author=CJ%20Willer&author=Y%20Li&author=GR%20Abecasis&volume=26&publication_year=2010&pages=2190-1&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

20. Adzhubei IA, Schmidt S, Peshkin L, Ramensky VE, Gerasimova A, Bork P, et al. A method and server for predicting damaging missense mutations. Nat Methods. 2010;7:248–9. doi: 10.1038/nmeth0410-248.  [DOI](https://doi.org/10.1038/nmeth0410-248) | [PMC free article](/articles/PMC2855889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20354512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=A%20method%20and%20server%20for%20predicting%20damaging%20missense%20mutations&author=IA%20Adzhubei&author=S%20Schmidt&author=L%20Peshkin&author=VE%20Ramensky&author=A%20Gerasimova&volume=7&publication_year=2010&pages=248-9&pmid=20354512&doi=10.1038/nmeth0410-248&)

21. Rentzsch P, Witten D, Cooper GM, Shendure J, Kircher M. CADD: predicting the deleteriousness of variants throughout the human genome. Nucleic Acids Res. 2019;47:D886–D894. doi: 10.1093/nar/gky1016.  [DOI](https://doi.org/10.1093/nar/gky1016) | [PMC free article](/articles/PMC6323892/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30371827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=CADD:%20predicting%20the%20deleteriousness%20of%20variants%20throughout%20the%20human%20genome&author=P%20Rentzsch&author=D%20Witten&author=GM%20Cooper&author=J%20Shendure&author=M%20Kircher&volume=47&publication_year=2019&pages=D886-D894&pmid=30371827&doi=10.1093/nar/gky1016&)

22. Itan Y, Shang L, Boisson B, Ciancanelli MJ, Markle JG, Martinez-Barricarte R, et al. The mutation significance cutoff: gene-level thresholds for variant predictions. Nat Methods. 2016;13:109–10. doi: 10.1038/nmeth.3739.  [DOI](https://doi.org/10.1038/nmeth.3739) | [PMC free article](/articles/PMC4980758/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26820543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=The%20mutation%20significance%20cutoff:%20gene-level%20thresholds%20for%20variant%20predictions&author=Y%20Itan&author=L%20Shang&author=B%20Boisson&author=MJ%20Ciancanelli&author=JG%20Markle&volume=13&publication_year=2016&pages=109-10&pmid=26820543&doi=10.1038/nmeth.3739&)

23. Kuster DW, Merkus D, Kremer A, van Ijcken WF, de Beer VJ, Verhoeven AJ, et al. Left ventricular remodeling in swine after myocardial infarction: a transcriptional genomics approach. Basic Res Cardiol. 2011;106:1269–81. doi: 10.1007/s00395-011-0229-1.  [DOI](https://doi.org/10.1007/s00395-011-0229-1) | [PMC free article](/articles/PMC3228945/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22057716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Res%20Cardiol&title=Left%20ventricular%20remodeling%20in%20swine%20after%20myocardial%20infarction:%20a%20transcriptional%20genomics%20approach&author=DW%20Kuster&author=D%20Merkus&author=A%20Kremer&author=WF%20van%20Ijcken&author=VJ%20de%20Beer&volume=106&publication_year=2011&pages=1269-81&pmid=22057716&doi=10.1007/s00395-011-0229-1&)

24. Suresh R, Li X, Chiriac A, Goel K, Terzic A, Perez-Terzic C, et al. Transcriptome from circulating cells suggests dysregulated pathways associated with long-term recurrent events following first-time myocardial infarction. J Mol Cell Cardiol. 2014;74:13–21. doi: 10.1016/j.yjmcc.2014.04.017.  [DOI](https://doi.org/10.1016/j.yjmcc.2014.04.017) | [PMC free article](/articles/PMC4115027/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24801707/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Cell%20Cardiol&title=Transcriptome%20from%20circulating%20cells%20suggests%20dysregulated%20pathways%20associated%20with%20long-term%20recurrent%20events%20following%20first-time%20myocardial%20infarction&author=R%20Suresh&author=X%20Li&author=A%20Chiriac&author=K%20Goel&author=A%20Terzic&volume=74&publication_year=2014&pages=13-21&pmid=24801707&doi=10.1016/j.yjmcc.2014.04.017&)

25. Lin RC, Weeks KL, Gao XM, Williams RB, Bernardo BC, Kiriazis H, et al. PI3K(p110 alpha) protects against myocardial infarction-induced heart failure: identification of PI3K-regulated miRNA and mRNA. Arterioscler Thromb Vasc Biol. 2010;30:724–32. doi: 10.1161/ATVBAHA.109.201988.  [DOI](https://doi.org/10.1161/ATVBAHA.109.201988) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20237330/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=PI3K(p110%20alpha)%20protects%20against%20myocardial%20infarction-induced%20heart%20failure:%20identification%20of%20PI3K-regulated%20miRNA%20and%20mRNA&author=RC%20Lin&author=KL%20Weeks&author=XM%20Gao&author=RB%20Williams&author=BC%20Bernardo&volume=30&publication_year=2010&pages=724-32&pmid=20237330&doi=10.1161/ATVBAHA.109.201988&)

26. Tulacz D, Mackiewicz U, Maczewski M, Maciejak A, Gora M, Burzynska B. Transcriptional profiling of left ventricle and peripheral blood mononuclear cells in a rat model of postinfarction heart failure. BMC Med Genomics. 2013;6:49. doi: 10.1186/1755-8794-6-49.  [DOI](https://doi.org/10.1186/1755-8794-6-49) | [PMC free article](/articles/PMC4226214/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24206753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genomics&title=Transcriptional%20profiling%20of%20left%20ventricle%20and%20peripheral%20blood%20mononuclear%20cells%20in%20a%20rat%20model%20of%20postinfarction%20heart%20failure&author=D%20Tulacz&author=U%20Mackiewicz&author=M%20Maczewski&author=A%20Maciejak&author=M%20Gora&volume=6&publication_year=2013&pages=49&pmid=24206753&doi=10.1186/1755-8794-6-49&)

27. Vapnik VN. An overview of statistical learning theory. IEEE Trans Neural Netw. 1999;10:988–99. doi: 10.1109/72.788640.  [DOI](https://doi.org/10.1109/72.788640) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18252602/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=IEEE%20Trans%20Neural%20Netw&title=An%20overview%20of%20statistical%20learning%20theory&author=VN%20Vapnik&volume=10&publication_year=1999&pages=988-99&pmid=18252602&doi=10.1109/72.788640&)

28. LightGBM: A highly efficient gradient boosting decision tree. NIPS; 2017.

29. Chen T, Guestrin C. XGBoost: A Scalable Tree Boosting System. In: Proceedings of the 22nd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining. San Francisco: Association for Computing Machinery; 2016. p 785–94.

30. Consortium GT. Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Science. 2015;348:648–60. doi: 10.1126/science.1262110.  [DOI](https://doi.org/10.1126/science.1262110) | [PMC free article](/articles/PMC4547484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25954001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Human%20genomics.%20The%20Genotype-Tissue%20Expression%20(GTEx)%20pilot%20analysis:%20multitissue%20gene%20regulation%20in%20humans&volume=348&publication_year=2015&pages=648-60&pmid=25954001&doi=10.1126/science.1262110&)

31. Sun J, Shen C, Jin X, Li X, Wu D. Mir-367 is downregulated in coronary artery disease and its overexpression exerts anti-inflammatory effect via inhibition of the NF-κB-activated inflammatory pathway. Int J Clin Exp Pathol. 2017;10:4047–57.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Exp%20Pathol&title=Mir-367%20is%20downregulated%20in%20coronary%20artery%20disease%20and%20its%20overexpression%20exerts%20anti-inflammatory%20effect%20via%20inhibition%20of%20the%20NF-%CE%BAB-activated%20inflammatory%20pathway&author=J%20Sun&author=C%20Shen&author=X%20Jin&author=X%20Li&author=D%20Wu&volume=10&publication_year=2017&pages=4047-57&)

32. Raid R, Krinka D, Bakhoff L, Abdelwahid E, Jokinen E, Karner M, et al. Lack of Gata3 results in conotruncal heart anomalies in mouse. Mechanisms Dev. 2009;126:80–89. doi: 10.1016/j.mod.2008.10.001.  [DOI](https://doi.org/10.1016/j.mod.2008.10.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18955134/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mechanisms%20Dev&title=Lack%20of%20Gata3%20results%20in%20conotruncal%20heart%20anomalies%20in%20mouse&author=R%20Raid&author=D%20Krinka&author=L%20Bakhoff&author=E%20Abdelwahid&author=E%20Jokinen&volume=126&publication_year=2009&pages=80-89&pmid=18955134&doi=10.1016/j.mod.2008.10.001&)

33. Rozanski A, Takano AP, Kato PN, Soares AG, Lellis-Santos C, Campos JC, et al. M-protein is down-regulated in cardiac hypertrophy driven by thyroid hormone in rats. Mol Endocrinol. 2013;27:2055–65. doi: 10.1210/me.2013-1018.  [DOI](https://doi.org/10.1210/me.2013-1018) | [PMC free article](/articles/PMC5426604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24176915/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Endocrinol&title=M-protein%20is%20down-regulated%20in%20cardiac%20hypertrophy%20driven%20by%20thyroid%20hormone%20in%20rats&author=A%20Rozanski&author=AP%20Takano&author=PN%20Kato&author=AG%20Soares&author=C%20Lellis-Santos&volume=27&publication_year=2013&pages=2055-65&pmid=24176915&doi=10.1210/me.2013-1018&)

34. Kakimoto Y, Ito S, Abiru H, Kotani H, Ozeki M, Tamaki K, et al. Sorbin and SH3 domain-containing protein 2 is released from infarcted heart in the very early phase: proteomic analysis of cardiac tissues from patients. J Am Heart Assoc. 2013;2:e000565–e000565. doi: 10.1161/JAHA.113.000565.  [DOI](https://doi.org/10.1161/JAHA.113.000565) | [PMC free article](/articles/PMC3886759/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24342996/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Heart%20Assoc&title=Sorbin%20and%20SH3%20domain-containing%20protein%202%20is%20released%20from%20infarcted%20heart%20in%20the%20very%20early%20phase:%20proteomic%20analysis%20of%20cardiac%20tissues%20from%20patients&author=Y%20Kakimoto&author=S%20Ito&author=H%20Abiru&author=H%20Kotani&author=M%20Ozeki&volume=2&publication_year=2013&pages=e000565-e000565&pmid=24342996&doi=10.1161/JAHA.113.000565&)

35. Hein S, Kostin S, Heling A, Maeno Y, Schaper J. The role of the cytoskeleton in heart failure. Cardiovasc Res. 2000;45:273–8. doi: 10.1016/s0008-6363(99)00268-0.  [DOI](https://doi.org/10.1016/s0008-6363(99)00268-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10728347/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Res&title=The%20role%20of%20the%20cytoskeleton%20in%20heart%20failure&author=S%20Hein&author=S%20Kostin&author=A%20Heling&author=Y%20Maeno&author=J%20Schaper&volume=45&publication_year=2000&pages=273-8&pmid=10728347&doi=10.1016/s0008-6363(99)00268-0&)

36. Brandt JT, Close SL, Iturria SJ, Payne CD, Farid NA, Ernest CS, 2nd, et al. Common polymorphisms of CYP2C19 and CYP2C9 affect the pharmacokinetic and pharmacodynamic response to clopidogrel but not prasugrel. J Thromb Haemost. 2007;5:2429–36. doi: 10.1111/j.1538-7836.2007.02775.x.  [DOI](https://doi.org/10.1111/j.1538-7836.2007.02775.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17900275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Thromb%20Haemost&title=Common%20polymorphisms%20of%20CYP2C19%20and%20CYP2C9%20affect%20the%20pharmacokinetic%20and%20pharmacodynamic%20response%20to%20clopidogrel%20but%20not%20prasugrel&author=JT%20Brandt&author=SL%20Close&author=SJ%20Iturria&author=CD%20Payne&author=NA%20Farid&volume=5&publication_year=2007&pages=2429-36&pmid=17900275&doi=10.1111/j.1538-7836.2007.02775.x&)

37. Simon T, Verstuyft C, Mary-Krause M, Quteineh L, Drouet E, Meneveau N, et al. Genetic determinants of response to clopidogrel and cardiovascular events. N. Engl J Med. 2009;360:363–75. doi: 10.1056/NEJMoa0808227.  [DOI](https://doi.org/10.1056/NEJMoa0808227) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19106083/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl%20J%20Med&title=Genetic%20determinants%20of%20response%20to%20clopidogrel%20and%20cardiovascular%20events&author=T%20Simon&author=C%20Verstuyft&author=M%20Mary-Krause&author=L%20Quteineh&author=E%20Drouet&volume=360&publication_year=2009&pages=363-75&pmid=19106083&doi=10.1056/NEJMoa0808227&)

38. Li XQ, Ma N, Li XG, Wang B, Sun SS, Gao F, et al. Association of PON1, P2Y12 and COX1 with recurrent ischemic events in patients with extracranial or intracranial stenting. PLoS ONE. 2016;11:e0148891. doi: 10.1371/journal.pone.0148891.  [DOI](https://doi.org/10.1371/journal.pone.0148891) | [PMC free article](/articles/PMC4752331/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26870959/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Association%20of%20PON1,%20P2Y12%20and%20COX1%20with%20recurrent%20ischemic%20events%20in%20patients%20with%20extracranial%20or%20intracranial%20stenting&author=XQ%20Li&author=N%20Ma&author=XG%20Li&author=B%20Wang&author=SS%20Sun&volume=11&publication_year=2016&pages=e0148891&pmid=26870959&doi=10.1371/journal.pone.0148891&)

39. Zhu SG, Xi L, Kukreja RC. Type 2 diabetic obese db/db mice are refractory to myocardial ischaemic post-conditioning in vivo: potential role for Hsp20, F1-ATPase delta and Echs1. J Cell Mol Med. 2012;16:950–8. doi: 10.1111/j.1582-4934.2011.01376.x.  [DOI](https://doi.org/10.1111/j.1582-4934.2011.01376.x) | [PMC free article](/articles/PMC3204159/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21722304/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Mol%20Med&title=Type%202%20diabetic%20obese%20db/db%20mice%20are%20refractory%20to%20myocardial%20ischaemic%20post-conditioning%20in%20vivo:%20potential%20role%20for%20Hsp20,%20F1-ATPase%20delta%20and%20Echs1&author=SG%20Zhu&author=L%20Xi&author=RC%20Kukreja&volume=16&publication_year=2012&pages=950-8&pmid=21722304&doi=10.1111/j.1582-4934.2011.01376.x&)

40. Haack TB, Jackson CB, Murayama K, Kremer LS, Schaller A, Kotzaeridou U, et al. Deficiency of ECHS1 causes mitochondrial encephalopathy with cardiac involvement. Ann Clin Transl Neurol. 2015;2:492–509. doi: 10.1002/acn3.189.  [DOI](https://doi.org/10.1002/acn3.189) | [PMC free article](/articles/PMC4435704/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26000322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Clin%20Transl%20Neurol&title=Deficiency%20of%20ECHS1%20causes%20mitochondrial%20encephalopathy%20with%20cardiac%20involvement&author=TB%20Haack&author=CB%20Jackson&author=K%20Murayama&author=LS%20Kremer&author=A%20Schaller&volume=2&publication_year=2015&pages=492-509&pmid=26000322&doi=10.1002/acn3.189&)

41. D’Ascenzo F, De Filippo O, Gallone G, Mittone G, Deriu MA, Iannaccone M, et al. Machine learning-based prediction of adverse events following an acute coronary syndrome (PRAISE): a modelling study of pooled datasets. Lancet. 2021;397:199–207. doi: 10.1016/S0140-6736(20)32519-8.  [DOI](https://doi.org/10.1016/S0140-6736(20)32519-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33453782/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Machine%20learning-based%20prediction%20of%20adverse%20events%20following%20an%20acute%20coronary%20syndrome%20(PRAISE):%20a%20modelling%20study%20of%20pooled%20datasets&author=F%20D%E2%80%99Ascenzo&author=O%20De%20Filippo&author=G%20Gallone&author=G%20Mittone&author=MA%20Deriu&volume=397&publication_year=2021&pages=199-207&pmid=33453782&doi=10.1016/S0140-6736(20)32519-8&)

42. Huang Z, Chan TM, Dong W. MACE prediction of acute coronary syndrome via boosted resampling classification using electronic medical records. J Biomed Inf. 2017;66:161–70. doi: 10.1016/j.jbi.2017.01.001.  [DOI](https://doi.org/10.1016/j.jbi.2017.01.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28065840/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biomed%20Inf&title=MACE%20prediction%20of%20acute%20coronary%20syndrome%20via%20boosted%20resampling%20classification%20using%20electronic%20medical%20records&author=Z%20Huang&author=TM%20Chan&author=W%20Dong&volume=66&publication_year=2017&pages=161-70&pmid=28065840&doi=10.1016/j.jbi.2017.01.001&)

43. Vanniyasingam T, Rodseth RN, Lurati Buse GA, Bolliger D, Burkhart CS, Cuthbertson BH, et al. Predicting the occurrence of major adverse cardiac events within 30 days of a vascular surgery: an empirical comparison of the minimum p value method and ROC curve approach using individual patient data meta-analysis. Springerplus. 2016;5:304. doi: 10.1186/s40064-016-1936-8.  [DOI](https://doi.org/10.1186/s40064-016-1936-8) | [PMC free article](/articles/PMC4783313/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27066338/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Springerplus&title=Predicting%20the%20occurrence%20of%20major%20adverse%20cardiac%20events%20within%2030%20days%20of%20a%20vascular%20surgery:%20an%20empirical%20comparison%20of%20the%20minimum%20p%20value%20method%20and%20ROC%20curve%20approach%20using%20individual%20patient%20data%20meta-analysis&author=T%20Vanniyasingam&author=RN%20Rodseth&author=GA%20Lurati%20Buse&author=D%20Bolliger&author=CS%20Burkhart&volume=5&publication_year=2016&pages=304&pmid=27066338&doi=10.1186/s40064-016-1936-8&)

44. Baluja A, Rodriguez-Manero M, Cordero A, Kreidieh B, Iglesias-Alvarez D, Garcia-Acuna JM, et al. Prediction of major adverse cardiac, cerebrovascular events in patients with diabetes after acute coronary syndrome. Diab Vasc Dis Res. 2020;17:1479164119892137. doi: 10.1177/1479164119892137.  [DOI](https://doi.org/10.1177/1479164119892137) | [PMC free article](/articles/PMC7510367/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31841030/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diab%20Vasc%20Dis%20Res&title=Prediction%20of%20major%20adverse%20cardiac,%20cerebrovascular%20events%20in%20patients%20with%20diabetes%20after%20acute%20coronary%20syndrome&author=A%20Baluja&author=M%20Rodriguez-Manero&author=A%20Cordero&author=B%20Kreidieh&author=D%20Iglesias-Alvarez&volume=17&publication_year=2020&pages=1479164119892137&pmid=31841030&doi=10.1177/1479164119892137&)
