# Genetic contributions to alcohol use disorder treatment outcomes: a genome-wide pharmacogenomics study

## Metadata
**Authors:** Joanna M Biernacka, Brandon J Coombes, Anthony Batzler, Ada Man-Choi Ho, Jennifer R Geske, Josef Frank, Colin Hodgkinson, Michelle Skime, Colin Colby, Lea Zillich, Sofia Pozsonyiova, Ming-Fen Ho, Falk Kiefer, Marcella Rietschel, Richard Weinshilboum, Stephanie S O’Malley, Karl Mann, Ray Anton, David Goldman, Victor M Karpyak
**Journal:** Neuropsychopharmacology
**Date:** 2021 Jul 23
**DOI:** [10.1038/s41386-021-01097-0](https://doi.org/10.1038/s41386-021-01097-0)
**PMID:** 34302059
**PMCID:** PMC8505452
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8505452/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8505452/pdf/41386_2021_Article_1097.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8505452/pdf/41386_2021_Article_1097.pdf)

## Abstract

Naltrexone can aid in reducing alcohol consumption, while acamprosate supports abstinence; however, not all patients with alcohol use disorder (AUD) benefit from these treatments. Here we present the first genome-wide association study of AUD treatment outcomes based on data from the COMBINE and PREDICT studies of acamprosate and naltrexone, and the Mayo Clinic CITA study of acamprosate. Primary analyses focused on treatment outcomes regardless of pharmacological intervention and were followed by drug-stratified analyses to identify treatment-specific pharmacogenomic predictors of acamprosate and naltrexone response. Treatment outcomes were defined as: (1) time until relapse to any drinking (TR) and (2) time until relapse to heavy drinking (THR; ≥ 5 drinks for men, ≥4 drinks for women in a day), during the first 3 months of treatment. Analyses were performed within each dataset, followed by meta-analysis across the studies (N = 1083 European ancestry participants). Single nucleotide polymorphisms (SNPs) in the BRE gene were associated with THR (min p = 1.6E−8) in the entire sample, while two intergenic SNPs were associated with medication-specific outcomes (naltrexone THR: rs12749274, p = 3.9E−8; acamprosate TR: rs77583603, p = 3.1E−9). The top association signal for TR (p = 7.7E−8) and second strongest signal in the THR (p = 6.1E−8) analysis of naltrexone-treated patients maps to PTPRD, a gene previously implicated in addiction phenotypes in human and animal studies. Leave-one-out polygenic risk score analyses showed significant associations with TR (p = 3.7E−4) and THR (p = 2.6E−4). This study provides the first evidence of a polygenic effect on AUD treatment response, and identifies genetic variants associated with potentially medication-specific effects on AUD treatment response.

Subject terms: Predictive markers, Behavioural genetics

## Introduction

Alcohol use disorder (AUD) is highly prevalent, and presents a significant health burden worldwide [[1](#CR1)]. Several medications have been developed for treatment of AUD, but they are underutilized and pharmacological treatment of AUD remains a major challenge [[2](#CR2)]. Recent reviews have highlighted the need for a precision medicine approach in the context of AUD treatment in order to increase the utility and safety of available medications [[3](#CR3), [4](#CR4)].

Treatment efficacy of acamprosate and naltrexone is supported by large systematic meta-analyses of randomized controlled trials [[5](#CR5), [6](#CR6)], which suggest that naltrexone helps people refrain from excessive drinking while acamprosate is effective in supporting abstinence [[6](#CR6)–[12](#CR12)]. However, a considerable proportion of AUD patients fail to benefit from these drugs, with the number needed to treat estimated to be 7–12 for acamprosate and 12–20 for naltrexone [[7](#CR7), [8](#CR8), [13](#CR13), [14](#CR14)]. Garbutt et al. reviewed moderators of naltrexone response and concluded that available data were insufficient to guide clinical treatment selection [[15](#CR15)]. Similarly, limited success was achieved in studies searching for clinical moderators of acamprosate response [[16](#CR16)]. Although secondary analyses have identified potential moderators of naltrexone and acamprosate response [[17](#CR17)], further efforts are needed to identify predictors for personalized medicine [[18](#CR18)].

Genetic variation contributes to inter-individual differences in drug response [[19](#CR19)–[22](#CR22)], and may facilitate the prediction of response to AUD treatment [[23](#CR23), [24](#CR24)]. Moreover, discovery of genetic variation and biological pathways involved in treatment response may reveal information about the mechanisms of drug action, accelerating further drug discovery efforts. Candidate gene studies identified several genetic variations that may influence AUD treatment outcomes [[25](#CR25)–[28](#CR28)]. However, the contribution of candidate gene studies to the understanding of genetic effects on AUD treatment response has been limited, highlighting the need for genome-wide pharmacogenomic studies. Yet, to date, no genome-wide association studies (GWASs) of acamprosate or naltrexone treatment response have been published.

Recent GWASs of AUD and other alcohol use related traits have identified a growing number of loci associated with these phenotypes. Various downstream analyses including gene, pathway and polygenic risk score (PRS) analyses have provided further insights into the genetic architecture of AUD and related traits, including genetic overlap with other psychiatric and behavioral traits [[29](#CR29)–[33](#CR33)]. While identifying loci contributing to AUD has been challenging due to the complexity of AUD and difficulties with compiling large samples of patients evaluated for this phenotype, GWAS of quantitative measures of problem drinking and alcohol consumption have been more successful [[29](#CR29)–[33](#CR33)]. Despite its clinical relevance, AUD treatment response has not yet been studied with the GWAS approach. Investigating the genetics of AUD treatment response is expected to not only reveal genetic variation potentially useful in treatment selection but may also contribute to our understanding of AUD risk and prognosis.

GWAS of treatment outcomes have been performed in the context of other addictions, most notably nicotine dependence. Large studies of smoking behavior phenotypes, including smoking cessation, demonstrated shared and unique genetic contributions across these phenotypes, nicotine use disorder, and other diseases [[34](#CR34)–[36](#CR36)]. Studies of nicotine clearance and nicotine metabolism biomarkers have provided further insights regarding the genetic contributions to complex smoking related phenotypes [[37](#CR37), [38](#CR38)]. However, similar to AUD, little is known about pharmacogenetic factors that contribute to response to specific smoking cessation treatments.

Here we present the first GWAS of AUD treatment response based on data from three of the largest studies of acamprosate and naltrexone completed to date, with a total sample of more than 1000 patients treated for AUD: the COMBINE [[39](#CR39)], PREDICT [[40](#CR40)], and CITA [[27](#CR27)] studies. COMBINE and PREDICT were randomized, placebo-controlled studies designed to evaluate response to acamprosate and naltrexone, while CITA was an open-label study of acamprosate designed for pharmacogenomics analyses. Using data from these three studies, we performed GWAS of AUD treatment outcomes (across treatment options) as well as pharmacogenomics GWAS of acamprosate and naltrexone response, separately. Our analyses included gene-level tests, gene-set and tissue enrichment analyses, and PRS analyses. Our results provide the first evidence of a polygenic effect on AUD treatment outcomes.

## Methods

### Samples, genotyping, quality control, and imputation

This study used a new genomic dataset derived from three previously completed studies of acamprosate and/or naltrexone treatment of AUD: the COMBINE, PREDICT, and CITA studies [[27](#CR27), [39](#CR39), [40](#CR40)]. Key characteristics of these three studies are summarized in the [Supplementary text](#MOESM1) and Supplementary Table [S1](#MOESM1). All subjects included in our analyses provided consent allowing use of their clinical data and DNA for genetic studies of AUD and response to its treatment, and this study was approved by the Mayo Clinic Institutional Review Board.

Sample genotyping, quality control, and imputation are described in the [Supplementary Methods](#MOESM1). After quality control, data from 498 COMBINE participants, 266 PREDICT participants, and 319 CITA participants of European ancestry with treatment outcome data were available for analysis. Following imputation, 5.6 million SNPs with minor allele frequency (MAF) > 0.05 in at least one of the studies were included in the meta-analyses.

### Assessment of treatment outcomes

Detailed descriptions of study procedures and assessments in the COMBINE, PREDICT, and CITA studies are presented elsewhere [[27](#CR27), [39](#CR39), [40](#CR40)]. In brief, baseline patient characteristics were collected in each study, and outcomes were assessed during 3 or more months of treatment. We compared baseline measures (demographic data and clinical information including alcohol consumption in the 30 days prior to treatment initiation) across studies using chi-square tests or one-way analysis of variance. Treatment outcomes were derived from timeline follow back (TLFB) data collected after treatment initiation. For this study, the primary treatment outcome measures were: (1) time until relapse (TR) to any drinking during the first 3 months of treatment, and (2) TR to heavy drinking (≥5 drinks for men, ≥4 drinks for women in a day) during the first 3 months of treatment. Outcomes for patients that were lost to follow-up in the first 3 months prior to relapse (or heavy relapse) were treated as censored observations in the survival analyses. Although the two outcomes we analyzed are highly correlated because for many patients first relapse was an episode of heavy drinking, we considered both outcomes, as acamprosate is believed to be effective in preventing relapse to any drinking while naltrexone has been reported to be more effective in preventing relapse to heavy drinking [[6](#CR6), [8](#CR8), [9](#CR9)].

### Genome-wide association analyses

The primary analyses included patients treated with acamprosate, naltrexone, or placebo (*N* = 1083) to identify predictors of treatment outcomes regardless of pharmacological intervention. Drug-stratified analyses were then run to identify treatment-specific (i.e., pharmacogenomic) predictors of acamprosate (*N* = 652) and naltrexone response (*N* = 301). In these drug-stratified analyses, patients were included if they were treated with the medication of interest (acamprosate or naltrexone), irrespective of other co-therapies. Thus, the GWAS of naltrexone outcomes in the COMBINE dataset included patients regardless of whether they had received the combined behavioral intervention and whether they had received naltrexone co-therapy; this means the acamprosate and naltrexone samples overlapped by a subset of COMBINE subjects that had been treated with both medications (*N* = 101). All GWAS were run in COMBINE, PREDICT and CITA datasets separately, followed by fixed-effects meta-analysis. In each dataset, allelic associations with TR and time until heavy relapse (THR) were assessed using Cox proportional hazards models. All analyses were adjusted for genetic principal components (PCs), if needed, to control for remaining population stratification in the European ancestry samples. The Cox proportional hazards analyses were run using the survival package in R (version 3.6.2), with SNP effect sizes estimated using hazard ratios (HR). Meta-analyses of GWAS summary statistics were performed using METAL [[41](#CR41)]. Methods for gene-level as well as gene-set and tissue enrichment analyses are described in the [Supplementary Text](#MOESM1).

### Polygenic risk score analyses

Leave-one-out PRS analyses were used to test for a reproducible polygenic predictor of treatment outcomes between datasets. Specifically, leave-one-out PRSs for TR and THR were generated for each target study based on the results of a discovery GWAS meta-analysis where the target cohort was left out. The PRSs were constructed using PRSice2 [[42](#CR42)] to prune (*r*^2^ > 0.1 within a 500 kb window) and restrict SNPs to a given *p* value threshold (p_t_ = 0.0001, 0.001, 0.01, 0.05, 0.1, 0.2, 1), with SNP alleles weighted by their log(HR) estimates. We then performed a principal component analysis (PCA) on the set of PRSs estimated at different *p* value thresholds and used the first PRS principal component to test for association with the outcome; this PRS-PCA strategy eliminates the multiple testing across PRSs based on different *p* value thresholds [[43](#CR43)]. The PRSs for TR and THR were tested for association with the respective treatment outcome in each left out dataset using Cox proportional hazards models, and the results from the analyses of the three datasets were meta-analyzed to assess overall PRS prediction of treatment response.

To investigate whether PRS for other AUD-related traits predict AUD treatment outcomes, we generated PRS for AUD and alcohol consumption using summary statistics from several published GWAS. For AUD, we derived the PRS based on the largest clinically assessed AUD sample [[32](#CR32)], a multi-ethnic sample with ICD diagnoses of AUD [[31](#CR31)], and a large meta-analysis of problematic alcohol use [[33](#CR33)]. For alcohol consumption, we used PRS derived from the UK Biobank GWAS [[29](#CR29)] and from the Million Veterans Program data [[31](#CR31)]. These PRSs were also constructed using PRSice2 [[42](#CR42)] (*r*^2^ > 0.1 within a 500 kb window; p_t_ = 5e−8, 1e−7, 1e−06, 1e−5, 0.0001, 0.001, 0.01, 0.05, 0.1, 0.2, 1). As with the leave-one-out analyses, we used the PRS-PCA strategy to reduce multiple testing across *p* value thresholds by computing a single PRS for each set of GWAS summary statistics and testing its association with the outcome.

The PRS analyses of the COMBINE study data were adjusted for the first four PCs to account for the heterogeneity observed in this dataset. Nagelkerke’s pseudo-*R*^2^ was calculated to estimate the variance explained in TR or THR by the PRS in each dataset, and weighted average *R*^2^ values were calculated across the three datasets using the effective N for each cohort.

## Results

Descriptive statistics for the three datasets are shown in Table [1](#Tab1). In total, 639 (59%) of the study participants relapsed to any drinking, while 564 (52%) relapsed to heavy drinking.

### Table 1.

|   | COMBINE | PREDICT | CITA |   |
| --- | --- | --- | --- | --- |
| Sample sizes |   |   |   | p value |
| Total N | 498 | 266 | 319 |   |
| Acamprosate N | 223 | 110 | 319 |   |
| Naltrexone N | 199 | 102 | – |   |
| Placebo only or no pills N | 177 | 54 | – |   |
| Age, mean(SD) | 45.4 (10.6) | 45.2 (8.5) | 43.4 (11.6) | 0.012 |
| Sex, male N(%) | 343 (68.9%) | 266 (100%) | 204 (64.0%) | <0.0001 |
| Age of onset | 30.5 (11.9) | 30.2 (10.2) | 29.6 (12.0) | 0.63 |
| Baseline alcohol consumptiona, mean (sd) |  |  |  |  |
| Days since last drinking day | 8.0 (5.4) | 22.0 (4.3) | 19.7 (8.8) | <0.0001 |
| Average number of drinks per drinking day | 12.1 (7.4) | 21.0 (12.7) | 12.0 (8.6) | <0.0001 |
| % Drinking daysb | 56.2 (22.8) | 82.0 (26.8) | 32.2 (28.2) | <0.0001 |
| % heavy drinking days | 47.2 (23.8) | 79.4 (27.9) | 28.6 (26.8) | <0.0001 |
| Treatment Outcomes: |  |  |  |  |
| Relapse, N (%) | 380 (76.3%) | 158 (59.4%) | 101 (31.7%) | <0.0001 |
| Relapse in acamprosate subset, N (%) | 167 (74.9%) | 62 (56.4%) | 101 (31.7%) | <0.0001 |
| Relapse in naltrexone subset, N (%) | 146 (73.4%) | 65 (63.7%) | – | 0.084 |
| Heavy Relapse, N (%) | 338 (67.9%) | 142 (53.4%) | 84 (26.3%) | <0.0001 |
| Heavy relapse in acamprosate subset, N (%) | 144 (64.6%) | 56 (50.9%) | 84 (26.3%) | <0.0001 |
| Heavy relapse in naltrexone subset, N (%) | 127 (63.8%) | 57 (55.9%) | – | 0.18 |

Table 1 Caption: Demographic and clinical characteristics of the COMBINE, PREDICT, and CITA participants included in the pharmacogenomics GWAS.

### Genome-wide association study

Manhattan plots for the two GWAS meta-analyses in the full cohort (all treatments) are shown in Fig. [1](#Fig1); the corresponding Quantile/Quantile plots are shown in Supplementary Fig. [S1](#MOESM1). In the meta-analysis across the three studies no individual SNPs were significantly associated with TR; the SNP with the strongest evidence for association is located in an intron of the potassium voltage-gated channel subfamily Q member 4 gene (*KCNQ4*, rs1078110, MAF = 0.30, *P* = 6.2E−7). In the full cohort analysis of THR, significant associations were observed for a set of SNPs in the brain and reproductive organ-expressed gene *BRE*, also known as BABAM2 (BRISC and BRCA1 A complex member 2); there were 14 genome-wide significant SNPs at this locus (top SNP rs56951679, MAF = 0.17, minor allele HR = 1.5, *p* = 1.6E−8). The effect estimate for the top SNP at this locus (rs56951679) suggests that carrying one additional copy of the minor allele (C) is associated with 1.5 times higher risk of relapse to heavy drinking. A regional association plot for this locus is shown in Supplementary Fig. [S2](#MOESM1).

### Fig. 1. Manhattan plots for analyses of outcomes in all subjects.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/321a/8505452/85771a16dce5/41386_2021_1097_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8505452_41386_2021_1097_Fig1_HTML.jpg)

Manhattan plots are shown for (A) time until relapse to any drinking, and (B) time until relapse to heavy drinking. In each panel, -log10(pvalue)s are shown (y-axis) for all SNPs by SNP position in the genome (x-axis).

In drug-stratified meta-analyses of outcomes in patients treated with naltrexone (Supplementary Fig. [S3](#MOESM1)), an intergenic SNP located between the long intergenic non-coding RNA RP4-710M16.2 and the *PPAP2B* gene (phospholipid phosphatase 3, also known as PLPP3) was associated with THR (rs12749274, MAF = 0.079, HR = 2.90, *p* = 3.9E−8). While not achieving genome-wide significance, the strongest evidence for association with TR and second strongest for THR was observed with a SNP in an intron of the protein tyrosine phosphatase receptor type D gene *PTPRD* (rs62533259, MAF = 0.14: TR minor allele HR = 2.2, *p* = 7.7E−8; THR minor allele HR = 2.4, *p* = 6.1E−8). *PTPRD* has been implicated in multiple addiction-related phenotypes in humans and other species [[44](#CR44)].

One SNP was significantly associated with TR during acamprosate treatment; this intergenic SNP is located between a non-coding RNA gene and the ribosomal protein L29 pseudogene (rs77583603, MAF = 0.093, *p* = 3.1E−9; Supplementary Fig. [S4](#MOESM1)). The top signal in the analyses of THR in acamprosate-treated patients was rs34797278 in a non-coding RNA (MAF = 0.079, *p* = 5.4E−8). All loci with suggestive evidence of association (*p* < 5*10^−6^) with one of the outcome measures are shown in Supplementary Tables [S1](#MOESM1) (all treatments), [S2](#MOESM1) (naltrexone), and [S3](#MOESM1) (acamprosate).

The top association findings described above were identified using meta-analyses of COMBINE, PREDICT, and CITA results. As these studies had significant differences in patient populations, we examined Forest Plots for the top loci to investigate the consistency of observed associations across these heterogeneous samples (Supplementary Fig. [S5](#MOESM1)). The results were highly consistent across samples at the genome-wide significant loci as shown in Supplementary Fig. S[5A/C/D](#MOESM1). However, the top *PTPRD* SNP association that almost reached significance in the meta-analysis of naltrexone-treated patients in COMBINE and PREDICT, was driven by a genome-wide significant association in the COMBINE sample, while showing no significant association in the PREDICT sample, although the direction of effect in PREDICT was consistent with the effect in COMBINE and the 95% confidence intervals overlapped (Supplementary Fig. S[5B](#MOESM1)).

Figure [2](#Fig2) shows scatterplots of results (*p* values) from analyses of TR vs. THR, and for analyses of acamprosate treatment outcomes vs. naltrexone treatment outcomes. As expected, TR GWAS signals were highly correlated with THR signals (panels A, B, and C of Fig. [2](#Fig2)), but results for acamprosate treatment outcomes showed little correlation with signals for naltrexone treatment outcomes (panels D and E of Fig. [2](#Fig2)). Although TR results were generally highly correlated with THR results, the top association from the THR analysis of the full cohort (the *BRE* top SNP *p* = 1.6E−8) showed much weaker evidence for association with TR (*p* = 1E−0). On the other hand, the *PTPRD* rs62533259 association in the naltrexone-treated subset was a top signal in both the TR and THR analyses (shown in panel B of Fig. [2](#Fig2)). However, the same *PTPRD* variants were not associated with TR or THR in the acamprosate-treated patient subset (panels D and E of Fig. [2](#Fig2)).

### Fig. 2. Scatterplots comparing results (p values) from GWAS of TR vs. THR and comparing results of analyses of different patient subsets.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/321a/8505452/031452070408/41386_2021_1097_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8505452_41386_2021_1097_Fig2_HTML.jpg)

Scatterplots of -log(p) for (A) analysis of TR vs. analysis of THR in all subjects, (B) analysis of TR vs. analysis of THR in naltrexone-treated subjects, (C) analysis of TR vs. analysis of THR in acamprosate-treated subjects, (D) analysis of TR in acamprosate-treated patients vs. analysis of TR in naltrexone-treated patients, and (E) analysis of THR in acamprosate-treated patients vs. analysis of THR in naltrexone-treated patients.

Results of the gene-level, gene-set, and tissue enrichment analyses are described in the [Supplementary Text](#MOESM1).

### Polygenic risk score analyses

Leave-one-out analyses provided evidence of a polygenic effect on treatment response in AUD. Specifically, for both TR and THR, PRSs derived from SNP effects observed in any two of the studies explained a statistically significant proportion of variance in the same outcome in the left-out dataset (TR: Nagelkerke’s *R*^2^ = 0.8%, *p* = 0.007 with PRS-PCA, max *R*^2^ = 1.3%, *p* = 3.7E−4 at *p*_t_ = 0.05; THR: Nagelkerke’s *R*^2^ = 1.0%; *p* = 0.002 with PRS-PCA, max *R*^2^ = 1.3%, *p* = 2.6e−4 at *p*_t_ = 0.05; Fig. [3](#Fig3)). In contrast, none of the PRSs for AUD, problematic alcohol use or alcohol consumption were associated with either treatment outcome (Supplementary Fig. [S7](#MOESM1)).

### Fig. 3. Leave-one-out PRS analyses.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/321a/8505452/f813d5b514f1/41386_2021_1097_Fig3_HTML.jpg)

Leave-one-out PRS analysis of (A) time until relapse to any drinking and (B) time until relapse to heavy drinking. In each of the three studies (CITA, COMBINE and PREDICT) PRSs were constructed based on a discovery GWAS in the remaining two samples across a range of p value thresholds (pT denoted using different colors described in the legend) to select SNPs for inclusion in the PRS. The selected SNPs (after LD pruning) were used to compute PRSs and the association of the PRSs with the outcome (TR or THR) was tested. The plots show the –log10(p values) for these association tests (on the y-axis) in each sample, as well as the meta-analysis of the leave-one-out PRS associations across the studies. The PRS association meta-analyses provided significant results for both time until relapse (p = 3.7E−04, Nagelkerke’s R2 = 1.3% at pT = 0.05) and time until heavy relapse (p = 2.6E−04, Nagelkerke’s R2 = 1.3% at pT = 0.10).

## Discussion

Using data from three studies of AUD treatment outcomes, we performed the first GWAS for AUD treatment response, including drug-stratified pharmacogenomic analyses of acamprosate and naltrexone treatment outcomes, and identified several genome-wide significant associations. We also performed PRS analyses to assess if a polygenic signal captured by this GWAS is associated with AUD treatment outcomes in independent samples. Results of these leave-one-out PRS analyses provided the first evidence of association between AUD treatment outcomes and PRSs, reflecting combined effect of variation across the genome on response to AUD treatment. While the proportion of variation in the treatment response explained by the PRSs is small, as expected for relatively small studies of complex traits, the significant association demonstrates that polygenic effects contribute to AUD treatment outcomes motivating further research into identifying contributing genetic factors.

Significant evidence of association was observed between THR and SNPs in the brain and reproductive organ-expressed protein (*BRE*) gene (aliases include *BABAM2*, *BRCC4* and *BRCC45*), which encodes BRISC and BRACA1 A complex member 2. *BRE* is ubiquitously expressed in human tissues, but most prominently in the zona glomerulosa of the adrenal cortex, where mineralocorticoids (e.g., aldosterone) are synthesized and secreted, as well as in glia and neurons [[45](#CR45)]. *BRE* expression is altered in adrenal abnormalities, suggesting possible involvement in adrenal function [[45](#CR45)]. Chronic alcohol consumption has been shown to increase blood aldosterone levels, and increased aldosterone level was correlated with higher alcohol consumption, craving and anxiety levels in AUD patients [[46](#CR46)]. Aldosterone may regulate alcohol use behaviors and anxiety via mineralocorticoid receptors expressed in limbic regions involved in regulating anxiety, stress-induced alcohol consumption, craving, and inhibitory control [[47](#CR47)]. Thus, our finding of association of *BRE* SNPs with THR may be related to regulation of stress response and alcohol craving. Furthermore, *BRE* has shown genome-wide significant associations with a range of phenotypes from different domains (e.g., metabolic, immunological, cardiovascular) [[48](#CR48)]. With respect to psychiatric traits, genetic variation in *BRE* has been associated with alcohol intake frequency and drinks per week in the general population [[34](#CR34)]. In our study, *BRE* SNPs were significantly associated with THR, but they were not significantly associated with TR despite the overall correlation of results for these two outcomes. It is possible that this gene has a differential effect on relapse and heavy relapse (e.g., if it is involved in regulation of alcohol consumption quantity or AUD severity), particularly given the prior reports of *BRE* genetic variation being associated with alcohol intake frequency and drinks per week. However, it is also worth noting that according to GTEx, *BRE* rs56951679 (and other top SNPs in our analysis) is a splicing quantitative trait locus for *FNDC4* exon 7 in human cortex, hippocampus, and nucleus accumbens. FNDC4, a secreted factor highly expressed in liver and brain, was implicated in sex-specific neurotoxic consequences of chronic alcohol withdrawal [[49](#CR49)]. The mechanisms of action through which variants in the *BRE* region may impact response to AUD treatment need to be further explored.

The one SNP with genome-wide significant evidence of association with acamprosate treatment response (rs77583603; *p* = 3.1E−9 for TR) is located between a non-coding RNA gene and the ribosomal protein L29 pseudogene and is not itself annotated as functionally significant. Analyses of naltrexone treatment outcomes identified a SNP near *PPAP2B* (a.k.a. *PLPP3, LPP3*), which encodes phospholipid phosphatase 3, that was significantly associated with THR. Changes in *PPAP2B* gene expression have been observed in nucleus accumbens and amygdala of rodents after voluntary ethanol consumption [[50](#CR50), [51](#CR51)], and *PPAP2B* has been implicated in alcoholic fatty liver in humans [[52](#CR52)]. These genes/variants identified in the medication-specific analyses have not been previously implicated in AUD or other addiction-related traits in humans, and their potential role in AUD treatment response needs further investigation.

An intronic SNP of *PTPRD*, while not quite genome-wide significant, was the strongest association signal for TR (*p* = 7.7E−8) and the second strongest association for THR (*p* = 6.1E−8) in the naltrexone analysis. This finding is notable because *PTPRD* (protein tyrosine phosphatase receptor type D) has been implicated in multiple addiction-related phenotypes in humans and in animal studies [[53](#CR53), [54](#CR54)]. Prior studies have not provided genome-wide significant evidence for association of *PTPRD* alleles with any substance use disorders, but the *PTPRD* locus showed genome-wide significant associations in a GWAS of obsessive-compulsive traits [[55](#CR55)], suggestive association in a GWAS of opioid cessation [[56](#CR56)], and it appears to have pleiotropic effects across brain phenotypes [[44](#CR44)]. Protein tyrosine phosphatases are signaling molecules that regulate a variety of cellular processes, and *PTPRD* likely plays a role in neuronal cell adhesion. Uhl and Martinez [[44](#CR44)] provided a comprehensive review of *PTPRD* genetics and neurobiology, and discussed its potential role as a pharmacological target with effects on brain phenotypes. Of note, the *PTPRD* effect we observed appears to be a pharmacogenomic effect specific to naltrexone, as our study provided no evidence of association of the same *PTPRD* variant with acamprosate treatment response. The impact of *PTPRD* on naltrexone treatment outcomes is, therefore, intriguing and warrants further investigation.

We performed GWAS of treatment outcomes in the full cohort of patients from the COMBINE, PREDICT, and CITA studies, as well as drug-specific analyses of acamprosate and naltrexone treatment response. The drug-stratified analyses aim to identify genetic variants that impact response to a given drug. The analyses of the full sample may also identify pharmacogenomic effects, but have greater power to identify genetic variants that impact risk of relapse regardless of specific treatment—these variants may include markers of severity of AUD or AUD subtypes that confer differential risk of relapse following treatment. Two genes implicated in this study, *BRE* and *PTPRD*, were previously associated with AUD related phenotypes. Known AUD risk genes, particularly *ADH1B* [[32](#CR32)], were not significant predictors of treatment response in our analyses. This may be partly due to a relatively small contribution of AUD risk genes to treatment response, which would be consistent with findings in other psychiatric disorders including major depressive disorder [[57](#CR57), [58](#CR58)]. In addition, low-frequency variants that are protective against AUD could have very low frequencies in our samples of AUD patients, reducing the power to detect their contribution to treatment response. Indeed, this may be the case with *ADH1B* variants that are protective against AUD and have low allele frequencies in populations of European ancestry. For example, the AUD-associated SNP rs1229984 in *ADH1B* has a frequency of ~5% in European populations [[59](#CR59)]. In the COMBINE dataset this SNP had a frequency of only ~2%, whereas in CITA and PREDICT its frequency was ~1% and <1%, respectively. Because of its low frequency and poor imputation quality in the CITA and PREDICT samples, this SNP was not analyzed in these datasets. However, analysis of the COMBINE data provided nominal evidence of association for rs1229984 with TR (HR = 0.32, *p* = 0.016) and THR (HR = 0.27, *p* = 0.0053); the effect estimates suggest the minor allele at rs1229984, which is protective against AUD, may have a strong protective effect against relapse during treatment, reducing the risk of relapse by about 70%. Because its MAF was below 0.05 in all of the samples, this SNP was excluded from the meta-analyses. We also found that PRSs for AUD and alcohol consumption were not associated with treatment response outcomes. These results stand in contrast to the significant prediction of the leave-one-out PRS for both TR and THR, suggesting that the genetic mechanisms involved in treatment response are different from the genetic mechanisms affecting consumption in non-AUD samples and addiction to alcohol.

This study made use of data from three prior studies of AUD treatment. These three studies have important similarities allowing for combined analysis, including use of the TLFB to assess drinking outcomes. However, the three studies also differed in relevant patient characteristics. The available PREDICT study data were only from men, but the COMBINE and CITA study samples included 31% and 36% women, respectively. Reported pre-treatment baseline alcohol consumption also differed, with PREDICT participants having the highest (and CITA participants having the lowest) alcohol consumption at baseline and participants in the COMBINE study having the shortest duration of abstinence prior to treatment initiation. The studies also differed in rates of treatment outcomes, with the CITA sample having the lowest percentage of any-drinking relapse and heavy-drinking relapse. While this heterogeneity may have limited the replicability of findings between the three datasets and reduced power of the meta-analysis, the differences between the studies also mean that identified genetic associations are likely robust genetic predictors of treatment outcomes that apply to a broad range of patients. In fact, the genome-wide significant loci in our study showed highly consistent results across the different samples. The *PTPRD* signal was driven by a genome-wide significant association in the COMBINE sample, while showing no significant association in the PREDICT sample; however, the direction of effect was the same and the confidence intervals for the SNP effect in the two samples overlapped. A more in-depth analysis of potential effect modifiers or confounders of the observed *PTPRD* association in a larger sample of patients treated with naltrexone is warranted.

In addition to sample heterogeneity that may have reduced the power of this study, other limitations need to be noted. The sample size of this study is smaller than what is typically required for well-powered GWAS of complex traits, particularly when the goal is identification of specific genetic risk variants. With a genome-wide significance threshold of 5e−8 and our sample of *N* = 1083 subjects, we had 80% power to detect SNP effects of HR = 1.80 and HR = 1.87 for TR and THR, respectively, for variants with MAF = 0.1. For more common variants with MAF = 0.4, our sample provided 80% power to detect SNP effects of 1.43 and 1.47 for the two outcomes, respectively. While genome-wide significant associations with several SNPs were identified in the full cohort and in the drug-stratified analyses, no significant gene-level associations were detected by MAGMA analysis. In the GWAS we used the standard significance threshold of 5e−8, although several different GWAS were performed (two outcomes, in the full sample as well as in drug-stratified samples). A Bonferroni correction for the two treatment outcome measures would not be appropriate as these outcomes are highly correlated, but the exploratory analyses of naltrexone and acamprosate treatment outcomes were largely independent. Thus, the observed SNP-level associations in these analyses should be interpreted cautiously and need to be replicated in independent samples. We also note that the sample size in this study was expected to be adequate to detect polygenic effects, and indeed our leave-one-out PRS results revealed a significant polygenic effect for both TR and THR.

Other limitations of this study include a lack of thorough investigation of the role of intermediate or potentially confounding factors such as demographic and clinical variables or comorbidities (e.g., smoking) as well as measures of AUD severity or alcohol consumptions. Because each of the contributing studies collected different data at baseline, harmonization of these variables and investigation of their role in the context of pharmacogenomics effects on AUD treatment response was beyond the scope of this study, but should be further investigated in the future. Similarly, the different study designs and limited sample size prohibited a formal evaluation of the specificity of findings in the drug-stratified analyses with SNP-drug interaction tests, which should be investigated in future studies. Finally, because of the limited ancestral diversity in the contributing studies, the analysis was limited to patients of European ancestry, limiting the generalizability of the findings to other populations, and reducing the power to identify effects of variants that are rare in European ancestry populations such as the *ADH1B* variant discussed above, or the *ALDH2* Glu504Lys polymorphism (rs671) that shows extraordinarily significant association to AUD, but only in populations where the locus is actually polymorphic. Studies of the genetics of AUD treatment response in samples with other ancestries are needed.

In conclusion, these first GWAS analyses of AUD treatment outcomes had limited power for discovery of specific genetic variants associated with response to AUD treatment. Nevertheless, they provided important insights, including the first demonstration of polygenic effects on AUD treatment outcomes, and identification of genetic variants potentially associated with AUD treatment response. These findings motivate further investigation of the genetic contribution to AUD outcomes and study of the biological mechanisms underlying response to medications used in the treatment of AUD.

## Supplementary information

## Acknowledgement

This work was supported by the National Institute on Alcohol Abuse and Alcoholism grants R21 AA25214, U01 AA027487, and R01 AA27486, and by the German Federal Ministry of Education and Research (BMBF) through grant SysmedSUD 01ZX01909A. Collection of data for the CITA study was supported by P20 AA017830. Dr RW is a co-founder of and stockholder in OneOme LLC, a pharmacogenomics decision-support company. Dr. SSO’M reports non-financial support from Amygdala, Astra Zeneca, and Novartis, personal fees from Emmes Corporation (DSMB member NIDA Clinical Trials Network), Alkermes, Dicerna, Opiant, and from the American Society of Clinical Psychopharmacology Alcohol Clinical Trials Initiative supported by Alkermes, Amygdala Neurosciences, Arbor Pharmaceuticals, Dicerna, Ethypharm, Indivior, Lundbeck, Mitsubishi and Otsuka, grants from National Institute on Alcohol Abuse and Addiction, and grants from National Institute on Drug Abuse outside the submitted work. In the past 3 years, Dr RA has been a consultant for Alkermes, Allergan, Dicerna, Insys, Labortorio Farmaceutico C.T., Foxo Bioscience. He also received grant funding from Labortorio Farmaceutico C.T. He is a chair and participant in the Alcohol Clinical Trials Initiative (ACTIVE) that has received support (in the past or currently) from Abbvie, Alkermes, Amygdala, Arbor, Dicerna, Ethypharm, Glaxo Smith Kline, Indivior, Janssen, Eli Lilly, Lundbeck, Mitsubishi, Otsuka, Pfizer, and Schering.

## Author contributions

JMB and VMK conceptualized and designed the study. JMB, VMK, MS, CH, FK, MR, SO, KM, RA, and DG acquired data that were used for the study. JMB, VMK, SO, RA, MR, DG, and BJC planned the analyses, and BJC, AB, AM-CH, JRG, JF, CC, LZ, and SP performed the analyses. JMB, BJC, AM-CH, and VMK drafted the paper, and all authors revised it critically for important intellectual content. All authors approved the final version of the paper to be published.

## Competing interests

The authors declare no competing interests.

## Footnotes

## Supplementary information

The online version contains supplementary material available at 10.1038/s41386-021-01097-0.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. GBD 2016 Alcohol and Drug Use Collaborators. The global burden of disease attributable to alcohol and drug use in 195 countries and territories, 1990-2016: a systematic analysis for the Global Burden of Disease Study 2016. Lancet Psychiatry. 2018;5:987–1012. doi: 10.1016/S2215-0366(18)30337-7.  [DOI](https://doi.org/10.1016/S2215-0366(18)30337-7) | [PMC free article](/articles/PMC6251968/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30392731/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Psychiatry&title=The%20global%20burden%20of%20disease%20attributable%20to%20alcohol%20and%20drug%20use%20in%20195%20countries%20and%20territories,%201990-2016:%20a%20systematic%20analysis%20for%20the%20Global%20Burden%20of%20Disease%20Study%202016&volume=5&publication_year=2018&pages=987-1012&pmid=30392731&doi=10.1016/S2215-0366(18)30337-7&)

2. Litten RZ, Falk DE, Ryan ML, Fertig JB. Discovery, Development, and Adoption of Medications to Treat Alcohol Use Disorder: goals for the Phases of Medications Development. Alcohol Clin Exp Res. 2016;40:1368–79. doi: 10.1111/acer.13093.  [DOI](https://doi.org/10.1111/acer.13093) | [PMC free article](/articles/PMC4930402/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27184259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Clin%20Exp%20Res&title=Discovery,%20Development,%20and%20Adoption%20of%20Medications%20to%20Treat%20Alcohol%20Use%20Disorder:%20goals%20for%20the%20Phases%20of%20Medications%20Development&author=RZ%20Litten&author=DE%20Falk&author=ML%20Ryan&author=JB%20Fertig&volume=40&publication_year=2016&pages=1368-79&pmid=27184259&doi=10.1111/acer.13093&)

3. Hartwell EE, Kranzler HR. Pharmacogenetics of alcohol use disorder treatments: an update. Expert Opin Drug Metab Toxicol. 2019;15:553–64. doi: 10.1080/17425255.2019.1628218.  [DOI](https://doi.org/10.1080/17425255.2019.1628218) | [PMC free article](/articles/PMC6614002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31162983/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Metab%20Toxicol&title=Pharmacogenetics%20of%20alcohol%20use%20disorder%20treatments:%20an%20update&author=EE%20Hartwell&author=HR%20Kranzler&volume=15&publication_year=2019&pages=553-64&pmid=31162983&doi=10.1080/17425255.2019.1628218&)

4. Helton SG, Lohoff FW. Pharmacogenetics of alcohol use disorders and comorbid psychiatric disorders. Psychiatry Res. 2015;230:121–9. doi: 10.1016/j.psychres.2015.09.019.  [DOI](https://doi.org/10.1016/j.psychres.2015.09.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26455758/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychiatry%20Res&title=Pharmacogenetics%20of%20alcohol%20use%20disorders%20and%20comorbid%20psychiatric%20disorders&author=SG%20Helton&author=FW%20Lohoff&volume=230&publication_year=2015&pages=121-9&pmid=26455758&doi=10.1016/j.psychres.2015.09.019&)

5. Donoghue K, Elzerbi C, Saunders R, Whittington C, Pilling S, Drummond C, et al., The efficacy of acamprosate and naltrexone in the treatment of alcohol dependence, Europe versus the Rest of the World: a meta-analysis. Addiction. 2015;110:920–30.  [DOI](https://doi.org/10.1111/add.12875) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25664494/)

6. Jonas DE, Amick HR, Feltner C, Bobashev G, Thomas K, Wines R, et al. Pharmacotherapy for adults with alcohol use disorders in outpatient settings: a systematic review and meta-analysis. JAMA: J Am Med Assoc. 2014;311:1889–900. doi: 10.1001/jama.2014.3628.  [DOI](https://doi.org/10.1001/jama.2014.3628) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24825644/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA:%20J%20Am%20Med%20Assoc&title=Pharmacotherapy%20for%20adults%20with%20alcohol%20use%20disorders%20in%20outpatient%20settings:%20a%20systematic%20review%20and%20meta-analysis&author=DE%20Jonas&author=HR%20Amick&author=C%20Feltner&author=G%20Bobashev&author=K%20Thomas&volume=311&publication_year=2014&pages=1889-900&pmid=24825644&doi=10.1001/jama.2014.3628&)

7. Bouza C, Angeles M, Muñoz A, Amate JM. Efficacy and safety of naltrexone and acamprosate in the treatment of alcohol dependence: a systematic review. Addiction. 2004;99:811–28. doi: 10.1111/j.1360-0443.2004.00763.x.  [DOI](https://doi.org/10.1111/j.1360-0443.2004.00763.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15200577/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=Efficacy%20and%20safety%20of%20naltrexone%20and%20acamprosate%20in%20the%20treatment%20of%20alcohol%20dependence:%20a%20systematic%20review&author=C%20Bouza&author=M%20Angeles&author=A%20Mu%C3%B1oz&author=JM%20Amate&volume=99&publication_year=2004&pages=811-28&pmid=15200577&doi=10.1111/j.1360-0443.2004.00763.x&)

8. Rösner S, Leucht S, Lehert P, Soyka M. Acamprosate supports abstinence, naltrexone prevents excessive drinking: evidence from a meta-analysis with unreported outcomes. J Psychopharmacol. 2008;22:11–23. doi: 10.1177/0269881107078308.  [DOI](https://doi.org/10.1177/0269881107078308) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18187529/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Psychopharmacol&title=Acamprosate%20supports%20abstinence,%20naltrexone%20prevents%20excessive%20drinking:%20evidence%20from%20a%20meta-analysis%20with%20unreported%20outcomes&author=S%20R%C3%B6sner&author=S%20Leucht&author=P%20Lehert&author=M%20Soyka&volume=22&publication_year=2008&pages=11-23&pmid=18187529&doi=10.1177/0269881107078308&)

9. Maisel NC, Blodgett JC, Wilbourne PL, Humphreys K, Finney JW. Meta-analysis of naltrexone and acamprosate for treating alcohol use disorders: when are these medications most helpful? Addiction. 2013;108:275–93. doi: 10.1111/j.1360-0443.2012.04054.x.  [DOI](https://doi.org/10.1111/j.1360-0443.2012.04054.x) | [PMC free article](/articles/PMC3970823/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23075288/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=Meta-analysis%20of%20naltrexone%20and%20acamprosate%20for%20treating%20alcohol%20use%20disorders:%20when%20are%20these%20medications%20most%20helpful?&author=NC%20Maisel&author=JC%20Blodgett&author=PL%20Wilbourne&author=K%20Humphreys&author=JW%20Finney&volume=108&publication_year=2013&pages=275-93&pmid=23075288&doi=10.1111/j.1360-0443.2012.04054.x&)

10. Dawson DA, Goldstein RB, Grant BF. Rates and correlates of relapse among individuals in remission from DSM-IV alcohol dependence: a 3-year follow-up. Alcohol Clin Exp Res. 2007;31:2036–45. doi: 10.1111/j.1530-0277.2007.00536.x.  [DOI](https://doi.org/10.1111/j.1530-0277.2007.00536.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18034696/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Clin%20Exp%20Res&title=Rates%20and%20correlates%20of%20relapse%20among%20individuals%20in%20remission%20from%20DSM-IV%20alcohol%20dependence:%20a%203-year%20follow-up&author=DA%20Dawson&author=RB%20Goldstein&author=BF%20Grant&volume=31&publication_year=2007&pages=2036-45&pmid=18034696&doi=10.1111/j.1530-0277.2007.00536.x&)

11. Witkiewitz K, Saville K, Hamreus K. Acamprosate for treatment of alcohol dependence: mechanisms, efficacy, and clinical utility. Ther Clin risk Manag. 2012;8:45–53. doi: 10.2147/TCRM.S23184.  [DOI](https://doi.org/10.2147/TCRM.S23184) | [PMC free article](/articles/PMC3277871/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22346357/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Clin%20risk%20Manag&title=Acamprosate%20for%20treatment%20of%20alcohol%20dependence:%20mechanisms,%20efficacy,%20and%20clinical%20utility&author=K%20Witkiewitz&author=K%20Saville&author=K%20Hamreus&volume=8&publication_year=2012&pages=45-53&pmid=22346357&doi=10.2147/TCRM.S23184&)

12. Cheng HY, McGuinness LA, Elbers RG, MacArthur GJ, Taylor A, McAleenan A, et al. Treatment interventions to maintain abstinence from alcohol in primary care: systematic review and network meta-analysis. BMJ. 2020;371:m3934. doi: 10.1136/bmj.m3934.  [DOI](https://doi.org/10.1136/bmj.m3934) | [PMC free article](/articles/PMC7687021/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33239318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Treatment%20interventions%20to%20maintain%20abstinence%20from%20alcohol%20in%20primary%20care:%20systematic%20review%20and%20network%20meta-analysis&author=HY%20Cheng&author=LA%20McGuinness&author=RG%20Elbers&author=GJ%20MacArthur&author=A%20Taylor&volume=371&publication_year=2020&pages=m3934&pmid=33239318&doi=10.1136/bmj.m3934&)

13. Rosner S, Hackl-Herrwerth A, Leucht S, Lehert P, Vecchi S, Soyka M, et al., Acamprosate for alcohol dependence. Cochrane Database Syst Rev. 2010:9:CD004332.  [DOI](https://doi.org/10.1002/14651858.CD004332.pub2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20824837/)

14. Heilig M, Egli M. Pharmacological treatment of alcohol dependence: target symptoms and target mechanisms. Pharmacol Ther. 2006;111:855–76. doi: 10.1016/j.pharmthera.2006.02.001.  [DOI](https://doi.org/10.1016/j.pharmthera.2006.02.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16545872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Ther&title=Pharmacological%20treatment%20of%20alcohol%20dependence:%20target%20symptoms%20and%20target%20mechanisms&author=M%20Heilig&author=M%20Egli&volume=111&publication_year=2006&pages=855-76&pmid=16545872&doi=10.1016/j.pharmthera.2006.02.001&)

15. Garbutt JC, Greenblatt AM, West SL, Morgan LC, Kampov-Polevoy A, Jordan HS, et al. Clinical and biological moderators of response to naltrexone in alcohol dependence: a systematic review of the evidence. Addiction. 2014;109:1274–84. doi: 10.1111/add.12557.  [DOI](https://doi.org/10.1111/add.12557) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24661324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=Clinical%20and%20biological%20moderators%20of%20response%20to%20naltrexone%20in%20alcohol%20dependence:%20a%20systematic%20review%20of%20the%20evidence&author=JC%20Garbutt&author=AM%20Greenblatt&author=SL%20West&author=LC%20Morgan&author=A%20Kampov-Polevoy&volume=109&publication_year=2014&pages=1274-84&pmid=24661324&doi=10.1111/add.12557&)

16. Verheul R, Lehert P, Geerlings PJ, Koeter MW, van den Brink W. Predictors of acamprosate efficacy: results from a pooled analysis of seven European trials including 1485 alcohol-dependent patients. Psychopharmacology. 2005;178:167–73. doi: 10.1007/s00213-004-1991-7.  [DOI](https://doi.org/10.1007/s00213-004-1991-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15322728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychopharmacology&title=Predictors%20of%20acamprosate%20efficacy:%20results%20from%20a%20pooled%20analysis%20of%20seven%20European%20trials%20including%201485%20alcohol-dependent%20patients&author=R%20Verheul&author=P%20Lehert&author=PJ%20Geerlings&author=MW%20Koeter&author=W%20van%20den%20Brink&volume=178&publication_year=2005&pages=167-73&pmid=15322728&doi=10.1007/s00213-004-1991-7&)

17. Gueorguieva R, Wu R, O'Connor PG, Weisner C, Fucito LM, Hoffmann S, et al. Predictors of abstinence from heavy drinking during treatment in COMBINE and external validation in PREDICT. Alcohol Clin Exp Res. 2014;38:2647–56. doi: 10.1111/acer.12541.  [DOI](https://doi.org/10.1111/acer.12541) | [PMC free article](/articles/PMC4397985/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25346505/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Clin%20Exp%20Res&title=Predictors%20of%20abstinence%20from%20heavy%20drinking%20during%20treatment%20in%20COMBINE%20and%20external%20validation%20in%20PREDICT&author=R%20Gueorguieva&author=R%20Wu&author=PG%20O'Connor&author=C%20Weisner&author=LM%20Fucito&volume=38&publication_year=2014&pages=2647-56&pmid=25346505&doi=10.1111/acer.12541&)

18. Litten RZ, Falk DE, Ryan ML, Fertig J, Leggio L. Five Priority Areas for Improving Medications Development for Alcohol Use Disorder and Promoting Their Routine Use in Clinical Practice. Alcohol Clin Exp Res. 2020;44:23–35. doi: 10.1111/acer.14233.  [DOI](https://doi.org/10.1111/acer.14233) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31803968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Clin%20Exp%20Res&title=Five%20Priority%20Areas%20for%20Improving%20Medications%20Development%20for%20Alcohol%20Use%20Disorder%20and%20Promoting%20Their%20Routine%20Use%20in%20Clinical%20Practice&author=RZ%20Litten&author=DE%20Falk&author=ML%20Ryan&author=J%20Fertig&author=L%20Leggio&volume=44&publication_year=2020&pages=23-35&pmid=31803968&doi=10.1111/acer.14233&)

19. Wilson JF, Weale ME, Smith AC, Gratrix F, Fletcher B, Thomas MG, et al. Population genetic structure of variable drug response. Nat Genet. 2001;29:265–9. doi: 10.1038/ng761.  [DOI](https://doi.org/10.1038/ng761) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11685208/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Population%20genetic%20structure%20of%20variable%20drug%20response&author=JF%20Wilson&author=ME%20Weale&author=AC%20Smith&author=F%20Gratrix&author=B%20Fletcher&volume=29&publication_year=2001&pages=265-9&pmid=11685208&doi=10.1038/ng761&)

20. Weinshilboum RM, Wang L. Pharmacogenetics and pharmacogenomics: development, science, and translation. Annu Rev Genom Hum Genet. 2006;7:223–45. doi: 10.1146/annurev.genom.6.080604.162315.  [DOI](https://doi.org/10.1146/annurev.genom.6.080604.162315) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16948615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Genom%20Hum%20Genet&title=Pharmacogenetics%20and%20pharmacogenomics:%20development,%20science,%20and%20translation&author=RM%20Weinshilboum&author=L%20Wang&volume=7&publication_year=2006&pages=223-45&pmid=16948615&doi=10.1146/annurev.genom.6.080604.162315&)

21. Wong ML, Dong C, Andreev V, Arcos-Burgos M, Licinio J. Prediction of susceptibility to major depression by a model of interactions of multiple functional genetic variants and environmental factors. Mol Psychiatry. 2012;17:624–33. doi: 10.1038/mp.2012.13.  [DOI](https://doi.org/10.1038/mp.2012.13) | [PMC free article](/articles/PMC3359641/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22449891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Prediction%20of%20susceptibility%20to%20major%20depression%20by%20a%20model%20of%20interactions%20of%20multiple%20functional%20genetic%20variants%20and%20environmental%20factors&author=ML%20Wong&author=C%20Dong&author=V%20Andreev&author=M%20Arcos-Burgos&author=J%20Licinio&volume=17&publication_year=2012&pages=624-33&pmid=22449891&doi=10.1038/mp.2012.13&)

22. van der Wouden CH, van Rhenen MH, Jama W, Ingelman-Sundberg M, Lauschke VM, Konta L, et al. Development of the PGx-Passport: a Panel of Actionable Germline Genetic Variants for Pre-Emptive Pharmacogenetic Testing. Clin Pharm Ther. 2019;106:866–73. doi: 10.1002/cpt.1489.  [DOI](https://doi.org/10.1002/cpt.1489) | [PMC free article](/articles/PMC6771671/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31038729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Ther&title=Development%20of%20the%20PGx-Passport:%20a%20Panel%20of%20Actionable%20Germline%20Genetic%20Variants%20for%20Pre-Emptive%20Pharmacogenetic%20Testing&author=CH%20van%20der%20Wouden&author=MH%20van%20Rhenen&author=W%20Jama&author=M%20Ingelman-Sundberg&author=VM%20Lauschke&volume=106&publication_year=2019&pages=866-73&pmid=31038729&doi=10.1002/cpt.1489&)

23. Kranzler HR, Edenberg HJ. Pharmacogenetics of alcohol and alcohol dependence treatment. Curr Pharm Des. 2010;16:2141–8. doi: 10.2174/138161210791516387.  [DOI](https://doi.org/10.2174/138161210791516387) | [PMC free article](/articles/PMC4142701/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20482509/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Pharm%20Des&title=Pharmacogenetics%20of%20alcohol%20and%20alcohol%20dependence%20treatment&author=HR%20Kranzler&author=HJ%20Edenberg&volume=16&publication_year=2010&pages=2141-8&pmid=20482509&doi=10.2174/138161210791516387&)

24. Litten RZ, Bradley AM, Moss HB. Alcohol biomarkers in applied settings: recent advances and future research opportunities. Alcohol Clin Exp Res. 2010;34:955–67. doi: 10.1111/j.1530-0277.2010.01170.x.  [DOI](https://doi.org/10.1111/j.1530-0277.2010.01170.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20374219/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Clin%20Exp%20Res&title=Alcohol%20biomarkers%20in%20applied%20settings:%20recent%20advances%20and%20future%20research%20opportunities&author=RZ%20Litten&author=AM%20Bradley&author=HB%20Moss&volume=34&publication_year=2010&pages=955-67&pmid=20374219&doi=10.1111/j.1530-0277.2010.01170.x&)

25. Anton RF, Oroszi G, O'Malley S, Couper D, Swift R, Pettinati H, et al. An evaluation of mu-opioid receptor (OPRM1) as a predictor of naltrexone response in the treatment of alcohol dependence: results from the Combined Pharmacotherapies and Behavioral Interventions for Alcohol Dependence (COMBINE) study. Arch Gen Psychiatry. 2008;65:135–44. doi: 10.1001/archpsyc.65.2.135.  [DOI](https://doi.org/10.1001/archpsyc.65.2.135) | [PMC free article](/articles/PMC2666924/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18250251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=An%20evaluation%20of%20mu-opioid%20receptor%20(OPRM1)%20as%20a%20predictor%20of%20naltrexone%20response%20in%20the%20treatment%20of%20alcohol%20dependence:%20results%20from%20the%20Combined%20Pharmacotherapies%20and%20Behavioral%20Interventions%20for%20Alcohol%20Dependence%20(COMBINE)%20study&author=RF%20Anton&author=G%20Oroszi&author=S%20O'Malley&author=D%20Couper&author=R%20Swift&volume=65&publication_year=2008&pages=135-44&pmid=18250251&doi=10.1001/archpsyc.65.2.135&)

26. Kiefer F, Witt SH, Frank J, Richter A, Treutlein J, Lemenager T, et al. Involvement of the atrial natriuretic peptide transcription factor GATA4 in alcohol dependence, relapse risk and treatment response to acamprosate. Pharmacogenom J. 2011;11:368–74. doi: 10.1038/tpj.2010.51.  [DOI](https://doi.org/10.1038/tpj.2010.51) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20585342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom%20J&title=Involvement%20of%20the%20atrial%20natriuretic%20peptide%20transcription%20factor%20GATA4%20in%20alcohol%20dependence,%20relapse%20risk%20and%20treatment%20response%20to%20acamprosate&author=F%20Kiefer&author=SH%20Witt&author=J%20Frank&author=A%20Richter&author=J%20Treutlein&volume=11&publication_year=2011&pages=368-74&pmid=20585342&doi=10.1038/tpj.2010.51&)

27. Karpyak VM, Biernacka JM, Geske JR, Jenkins GD, Cunningham JM, Rüegg J, et al. Genetic markers associated with abstinence length in alcohol-dependent subjects treated with acamprosate. Transl Psychiatry. 2014;4:e453. doi: 10.1038/tp.2014.103.  [DOI](https://doi.org/10.1038/tp.2014.103) | [PMC free article](/articles/PMC4350512/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25290263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transl%20Psychiatry&title=Genetic%20markers%20associated%20with%20abstinence%20length%20in%20alcohol-dependent%20subjects%20treated%20with%20acamprosate&author=VM%20Karpyak&author=JM%20Biernacka&author=JR%20Geske&author=GD%20Jenkins&author=JM%20Cunningham&volume=4&publication_year=2014&pages=e453&pmid=25290263&doi=10.1038/tp.2014.103&)

28. Spanagel R, Pendyala G, Abarca C, Zghoul T, Sanchis-Segura C, Magnone MC, et al. The clock gene Per2 influences the glutamatergic system and modulates alcohol consumption. Nat Med. 2005;11:35–42. doi: 10.1038/nm1163.  [DOI](https://doi.org/10.1038/nm1163) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15608650/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=The%20clock%20gene%20Per2%20influences%20the%20glutamatergic%20system%20and%20modulates%20alcohol%20consumption&author=R%20Spanagel&author=G%20Pendyala&author=C%20Abarca&author=T%20Zghoul&author=C%20Sanchis-Segura&volume=11&publication_year=2005&pages=35-42&pmid=15608650&doi=10.1038/nm1163&)

29. Clarke TK, Adams MJ, Davies G, Howard DM, Hall LS, Padmanabhan S, et al. Genome-wide association study of alcohol consumption and genetic overlap with other health-related traits in UK Biobank (N=112 117) Mol Psychiatry. 2017;22:1376–84. doi: 10.1038/mp.2017.153.  [DOI](https://doi.org/10.1038/mp.2017.153) | [PMC free article](/articles/PMC5622124/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28937693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Genome-wide%20association%20study%20of%20alcohol%20consumption%20and%20genetic%20overlap%20with%20other%20health-related%20traits%20in%20UK%20Biobank%20(N=112%20117)&author=TK%20Clarke&author=MJ%20Adams&author=G%20Davies&author=DM%20Howard&author=LS%20Hall&volume=22&publication_year=2017&pages=1376-84&pmid=28937693&doi=10.1038/mp.2017.153&)

30. Deak JD, Miller AP, Gizer IR. Genetics of alcohol use disorder: a review. Curr Opin Psychol. 2019;27:56–61. doi: 10.1016/j.copsyc.2018.07.012.  [DOI](https://doi.org/10.1016/j.copsyc.2018.07.012) | [PMC free article](/articles/PMC6368900/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30170251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Psychol&title=Genetics%20of%20alcohol%20use%20disorder:%20a%20review&author=JD%20Deak&author=AP%20Miller&author=IR%20Gizer&volume=27&publication_year=2019&pages=56-61&pmid=30170251&doi=10.1016/j.copsyc.2018.07.012&)

31. Kranzler HR, Zhou H, Kember RL, Vickers Smith R, Justice AC, Damrauer S, et al. Genome-wide association study of alcohol consumption and use disorder in 274,424 individuals from multiple populations. Nat Commun. 2019;10:1499. doi: 10.1038/s41467-019-09480-8.  [DOI](https://doi.org/10.1038/s41467-019-09480-8) | [PMC free article](/articles/PMC6445072/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30940813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Genome-wide%20association%20study%20of%20alcohol%20consumption%20and%20use%20disorder%20in%20274,424%20individuals%20from%20multiple%20populations&author=HR%20Kranzler&author=H%20Zhou&author=RL%20Kember&author=R%20Vickers%20Smith&author=AC%20Justice&volume=10&publication_year=2019&pages=1499&pmid=30940813&doi=10.1038/s41467-019-09480-8&)

32. Walters RK, Polimanti R, Johnson EC, McClintick JN, Adams MJ, Adkins AE, et al. Transancestral GWAS of alcohol dependence reveals common genetic underpinnings with psychiatric disorders. Nat Neurosci. 2018;21:1656–69. doi: 10.1038/s41593-018-0275-1.  [DOI](https://doi.org/10.1038/s41593-018-0275-1) | [PMC free article](/articles/PMC6430207/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30482948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Neurosci&title=Transancestral%20GWAS%20of%20alcohol%20dependence%20reveals%20common%20genetic%20underpinnings%20with%20psychiatric%20disorders&author=RK%20Walters&author=R%20Polimanti&author=EC%20Johnson&author=JN%20McClintick&author=MJ%20Adams&volume=21&publication_year=2018&pages=1656-69&pmid=30482948&doi=10.1038/s41593-018-0275-1&)

33. Zhou H, Sealock JM, Sanchez-Roige S, Clarke TK, Levey DF, Cheng Z, et al. Genome-wide meta-analysis of problematic alcohol use in 435,563 individuals yields insights into biology and relationships with other traits. Nat Neurosci. 2020;23:809–18. doi: 10.1038/s41593-020-0643-5.  [DOI](https://doi.org/10.1038/s41593-020-0643-5) | [PMC free article](/articles/PMC7485556/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32451486/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Neurosci&title=Genome-wide%20meta-analysis%20of%20problematic%20alcohol%20use%20in%20435,563%20individuals%20yields%20insights%20into%20biology%20and%20relationships%20with%20other%20traits&author=H%20Zhou&author=JM%20Sealock&author=S%20Sanchez-Roige&author=TK%20Clarke&author=DF%20Levey&volume=23&publication_year=2020&pages=809-18&pmid=32451486&doi=10.1038/s41593-020-0643-5&)

34. Liu M, Jiang Y, Wedow R, Li Y, Brazel DM, Chen F, et al. Association studies of up to 1.2 million individuals yield new insights into the genetic etiology of tobacco and alcohol use. Nat Genet. 2019;51:237–44. doi: 10.1038/s41588-018-0307-5.  [DOI](https://doi.org/10.1038/s41588-018-0307-5) | [PMC free article](/articles/PMC6358542/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30643251/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Association%20studies%20of%20up%20to%201.2%20million%20individuals%20yield%20new%20insights%20into%20the%20genetic%20etiology%20of%20tobacco%20and%20alcohol%20use&author=M%20Liu&author=Y%20Jiang&author=R%20Wedow&author=Y%20Li&author=DM%20Brazel&volume=51&publication_year=2019&pages=237-44&pmid=30643251&doi=10.1038/s41588-018-0307-5&)

35. Matoba N, Akiyama M, Ishigaki K, Kanai M, Takahashi A, Momozawa Y, et al. GWAS of smoking behaviour in 165,436 Japanese people reveals seven new loci and shared genetic architecture. Nat Hum Behav. 2019;3:471–7. doi: 10.1038/s41562-019-0557-y.  [DOI](https://doi.org/10.1038/s41562-019-0557-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31089300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Hum%20Behav&title=GWAS%20of%20smoking%20behaviour%20in%20165,436%20Japanese%20people%20reveals%20seven%20new%20loci%20and%20shared%20genetic%20architecture&author=N%20Matoba&author=M%20Akiyama&author=K%20Ishigaki&author=M%20Kanai&author=A%20Takahashi&volume=3&publication_year=2019&pages=471-7&pmid=31089300&doi=10.1038/s41562-019-0557-y&)

36. Saccone NL, Emery LS, Sofer T, Gogarten SM, Becker DM, Bottinger EP, et al. Genome-Wide Association Study of Heavy Smoking and Daily/Nondaily Smoking in the Hispanic Community Health Study/Study of Latinos (HCHS/SOL) Nicotine Tob Res. 2018;20:448–57. doi: 10.1093/ntr/ntx107.  [DOI](https://doi.org/10.1093/ntr/ntx107) | [PMC free article](/articles/PMC5896462/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28520984/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nicotine%20Tob%20Res&title=Genome-Wide%20Association%20Study%20of%20Heavy%20Smoking%20and%20Daily/Nondaily%20Smoking%20in%20the%20Hispanic%20Community%20Health%20Study/Study%20of%20Latinos%20(HCHS/SOL)&author=NL%20Saccone&author=LS%20Emery&author=T%20Sofer&author=SM%20Gogarten&author=DM%20Becker&volume=20&publication_year=2018&pages=448-57&pmid=28520984&doi=10.1093/ntr/ntx107&)

37. Buchwald J, Chenoweth MJ, Palviainen T, Zhu G, Benner C, Gordon S, et al., Genome-wide association meta-analysis of nicotine metabolism and cigarette consumption measures in smokers of European descent. Mol Psychiatry. 2020.  [DOI](https://doi.org/10.1038/s41380-020-0702-z) | [PMC free article](/articles/PMC7483250/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32157176/)

38. Chenoweth MJ, Ware JJ, Zhu A, Cole CB, Cox LS, Nollen N, et al. Genome-wide association study of a nicotine metabolism biomarker in African American smokers: impact of chromosome 19 genetic influences. Addiction. 2018;113:509–23. doi: 10.1111/add.14032.  [DOI](https://doi.org/10.1111/add.14032) | [PMC free article](/articles/PMC5807179/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28921760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=Genome-wide%20association%20study%20of%20a%20nicotine%20metabolism%20biomarker%20in%20African%20American%20smokers:%20impact%20of%20chromosome%2019%20genetic%20influences&author=MJ%20Chenoweth&author=JJ%20Ware&author=A%20Zhu&author=CB%20Cole&author=LS%20Cox&volume=113&publication_year=2018&pages=509-23&pmid=28921760&doi=10.1111/add.14032&)

39. Anton RF, O'Malley SS, Ciraulo DA, Cisler RA, Couper D, Donovan DM, et al. Combined pharmacotherapies and behavioral interventions for alcohol dependence: the COMBINE study: a randomized controlled trial. JAMA. 2006;295:2003–17. doi: 10.1001/jama.295.17.2003.  [DOI](https://doi.org/10.1001/jama.295.17.2003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16670409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Combined%20pharmacotherapies%20and%20behavioral%20interventions%20for%20alcohol%20dependence:%20the%20COMBINE%20study:%20a%20randomized%20controlled%20trial&author=RF%20Anton&author=SS%20O'Malley&author=DA%20Ciraulo&author=RA%20Cisler&author=D%20Couper&volume=295&publication_year=2006&pages=2003-17&pmid=16670409&doi=10.1001/jama.295.17.2003&)

40. Mann K, Kiefer F, Smolka M, Gann H, Wellek S, Heinz A, et al. Searching for responders to acamprosate and naltrexone in alcoholism treatment: rationale and design of the PREDICT study. Alcohol Clin Exp Res. 2009;33:674–83. doi: 10.1111/j.1530-0277.2008.00884.x.  [DOI](https://doi.org/10.1111/j.1530-0277.2008.00884.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19170666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Alcohol%20Clin%20Exp%20Res&title=Searching%20for%20responders%20to%20acamprosate%20and%20naltrexone%20in%20alcoholism%20treatment:%20rationale%20and%20design%20of%20the%20PREDICT%20study&author=K%20Mann&author=F%20Kiefer&author=M%20Smolka&author=H%20Gann&author=S%20Wellek&volume=33&publication_year=2009&pages=674-83&pmid=19170666&doi=10.1111/j.1530-0277.2008.00884.x&)

41. Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics. 2010;26:2190–1. doi: 10.1093/bioinformatics/btq340.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genomewide%20association%20scans&author=CJ%20Willer&author=Y%20Li&author=GR%20Abecasis&volume=26&publication_year=2010&pages=2190-1&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

42. Choi SW, O’Reilly PF. PRSice-2: polygenic risk score software for biobank-scale data. Gigascience. 2019;8:giz082.  [DOI](https://doi.org/10.1093/gigascience/giz082) | [PMC free article](/articles/PMC6629542/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31307061/)

43. Coombes BJ, Ploner A, Bergen SE, Biernacka JM. A principal component approach to improve association testing with polygenic risk scores. Genet Epidemiol. 2020;44:676–86. doi: 10.1002/gepi.22339.  [DOI](https://doi.org/10.1002/gepi.22339) | [PMC free article](/articles/PMC7722089/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32691445/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=A%20principal%20component%20approach%20to%20improve%20association%20testing%20with%20polygenic%20risk%20scores&author=BJ%20Coombes&author=A%20Ploner&author=SE%20Bergen&author=JM%20Biernacka&volume=44&publication_year=2020&pages=676-86&pmid=32691445&doi=10.1002/gepi.22339&)

44. Uhl GR, Martinez MJ. PTPRD: neurobiology, genetics, and initial pharmacology of a pleiotropic contributor to brain phenotypes. Ann N Y Acad Sci. 2019;1451:112–29. doi: 10.1111/nyas.14002.  [DOI](https://doi.org/10.1111/nyas.14002) | [PMC free article](/articles/PMC6629525/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30648269/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20N%20Y%20Acad%20Sci&title=PTPRD:%20neurobiology,%20genetics,%20and%20initial%20pharmacology%20of%20a%20pleiotropic%20contributor%20to%20brain%20phenotypes&author=GR%20Uhl&author=MJ%20Martinez&volume=1451&publication_year=2019&pages=112-29&pmid=30648269&doi=10.1111/nyas.14002&)

45. Miao J, Panesar NS, Chan KT, Lai FM, Xia N, Wang Y, et al. Differential expression of a stress-modulating gene, BRE, in the adrenal gland, in adrenal neoplasia, and in abnormal adrenal tissues. J Histochem Cytochem. 2001;49:491–500. doi: 10.1177/002215540104900409.  [DOI](https://doi.org/10.1177/002215540104900409) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11259452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Histochem%20Cytochem&title=Differential%20expression%20of%20a%20stress-modulating%20gene,%20BRE,%20in%20the%20adrenal%20gland,%20in%20adrenal%20neoplasia,%20and%20in%20abnormal%20adrenal%20tissues&author=J%20Miao&author=NS%20Panesar&author=KT%20Chan&author=FM%20Lai&author=N%20Xia&volume=49&publication_year=2001&pages=491-500&pmid=11259452&doi=10.1177/002215540104900409&)

46. Aoun EG, Jimenez VA, Vendruscolo LF, Walter N, Barbier E, Ferrulli A, et al. A relationship between the aldosterone-mineralocorticoid receptor pathway and alcohol drinking: preliminary translational findings across rats, monkeys and humans. Mol Psychiatry. 2018;23:1466–73. doi: 10.1038/mp.2017.97.  [DOI](https://doi.org/10.1038/mp.2017.97) | [PMC free article](/articles/PMC5668213/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28461696/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=A%20relationship%20between%20the%20aldosterone-mineralocorticoid%20receptor%20pathway%20and%20alcohol%20drinking:%20preliminary%20translational%20findings%20across%20rats,%20monkeys%20and%20humans&author=EG%20Aoun&author=VA%20Jimenez&author=LF%20Vendruscolo&author=N%20Walter&author=E%20Barbier&volume=23&publication_year=2018&pages=1466-73&pmid=28461696&doi=10.1038/mp.2017.97&)

47. Koning A, Buurstede JC, van Weert L, Meijer OC. Glucocorticoid and Mineralocorticoid Receptors in the Brain: a Transcriptional Perspective. J Endocr Soc. 2019;3:1917–30. doi: 10.1210/js.2019-00158.  [DOI](https://doi.org/10.1210/js.2019-00158) | [PMC free article](/articles/PMC6777400/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31598572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Endocr%20Soc&title=Glucocorticoid%20and%20Mineralocorticoid%20Receptors%20in%20the%20Brain:%20a%20Transcriptional%20Perspective&author=A%20Koning&author=JC%20Buurstede&author=L%20van%20Weert&author=OC%20Meijer&volume=3&publication_year=2019&pages=1917-30&pmid=31598572&doi=10.1210/js.2019-00158&)

48. Watanabe K, Stringer S, Frei O, Umićević Mirkov M, de Leeuw C, Polderman T, et al. A global overview of pleiotropy and genetic architecture in complex traits. Nat Genet. 2019;51:1339–48. doi: 10.1038/s41588-019-0481-0.  [DOI](https://doi.org/10.1038/s41588-019-0481-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31427789/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20global%20overview%20of%20pleiotropy%20and%20genetic%20architecture%20in%20complex%20traits&author=K%20Watanabe&author=S%20Stringer&author=O%20Frei&author=M%20Umi%C4%87evi%C4%87%20Mirkov&author=C%20de%20Leeuw&volume=51&publication_year=2019&pages=1339-48&pmid=31427789&doi=10.1038/s41588-019-0481-0&)

49. Hashimoto JG, Wiren KM. Neurotoxic consequences of chronic alcohol withdrawal: expression profiling reveals importance of gender over withdrawal severity. Neuropsychopharmacology. 2008;33:1084–96. doi: 10.1038/sj.npp.1301494.  [DOI](https://doi.org/10.1038/sj.npp.1301494) | [PMC free article](/articles/PMC3019135/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17593928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Neurotoxic%20consequences%20of%20chronic%20alcohol%20withdrawal:%20expression%20profiling%20reveals%20importance%20of%20gender%20over%20withdrawal%20severity&author=JG%20Hashimoto&author=KM%20Wiren&volume=33&publication_year=2008&pages=1084-96&pmid=17593928&doi=10.1038/sj.npp.1301494&)

50. Lesscher HM, Houthuijzen JM, Groot Koerkamp MJ, Holstege FC, Vanderschuren LJ. Amygdala 14-3-3ζ as a novel modulator of escalating alcohol intake in mice. PLoS ONE. 2012;7:e37999. doi: 10.1371/journal.pone.0037999.  [DOI](https://doi.org/10.1371/journal.pone.0037999) | [PMC free article](/articles/PMC3358291/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22629472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Amygdala%2014-3-3%CE%B6%20as%20a%20novel%20modulator%20of%20escalating%20alcohol%20intake%20in%20mice&author=HM%20Lesscher&author=JM%20Houthuijzen&author=MJ%20Groot%20Koerkamp&author=FC%20Holstege&author=LJ%20Vanderschuren&volume=7&publication_year=2012&pages=e37999&pmid=22629472&doi=10.1371/journal.pone.0037999&)

51. Bell RL, Kimpel MW, McClintick JN, Strother WN, Carr LG, Liang T, et al. Gene expression changes in the nucleus accumbens of alcohol-preferring rats following chronic ethanol consumption. Pharm Biochem Behav. 2009;94:131–47. doi: 10.1016/j.pbb.2009.07.019.  [DOI](https://doi.org/10.1016/j.pbb.2009.07.019) | [PMC free article](/articles/PMC2771758/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19666046/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Biochem%20Behav&title=Gene%20expression%20changes%20in%20the%20nucleus%20accumbens%20of%20alcohol-preferring%20rats%20following%20chronic%20ethanol%20consumption&author=RL%20Bell&author=MW%20Kimpel&author=JN%20McClintick&author=WN%20Strother&author=LG%20Carr&volume=94&publication_year=2009&pages=131-47&pmid=19666046&doi=10.1016/j.pbb.2009.07.019&)

52. Covarrubias MY, Khan RL, Vadigepalli R, Hoek JB, Schwaber JS. Chronic alcohol exposure alters transcription broadly in a key integrative brain nucleus for homeostasis: the nucleus tractus solitarius. Physiol Genom. 2005;24:45–58. doi: 10.1152/physiolgenomics.00184.2005.  [DOI](https://doi.org/10.1152/physiolgenomics.00184.2005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16189278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiol%20Genom&title=Chronic%20alcohol%20exposure%20alters%20transcription%20broadly%20in%20a%20key%20integrative%20brain%20nucleus%20for%20homeostasis:%20the%20nucleus%20tractus%20solitarius&author=MY%20Covarrubias&author=RL%20Khan&author=R%20Vadigepalli&author=JB%20Hoek&author=JS%20Schwaber&volume=24&publication_year=2005&pages=45-58&pmid=16189278&doi=10.1152/physiolgenomics.00184.2005&)

53. Drgonova J, Walther D, Wang KJ, Hartstein GL, Lochte B, Troncoso J, et al. Mouse Model for Protein Tyrosine Phosphatase D (PTPRD) Associations with Restless Leg Syndrome or Willis-Ekbom Disease and Addiction: reduced Expression Alters Locomotion, Sleep Behaviors and Cocaine-Conditioned Place Preference. Mol Med. 2015;21:717–25. doi: 10.2119/molmed.2015.00017.  [DOI](https://doi.org/10.2119/molmed.2015.00017) | [PMC free article](/articles/PMC4749486/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26181631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Med&title=Mouse%20Model%20for%20Protein%20Tyrosine%20Phosphatase%20D%20(PTPRD)%20Associations%20with%20Restless%20Leg%20Syndrome%20or%20Willis-Ekbom%20Disease%20and%20Addiction:%20reduced%20Expression%20Alters%20Locomotion,%20Sleep%20Behaviors%20and%20Cocaine-Conditioned%20Place%20Preference&author=J%20Drgonova&author=D%20Walther&author=KJ%20Wang&author=GL%20Hartstein&author=B%20Lochte&volume=21&publication_year=2015&pages=717-25&pmid=26181631&doi=10.2119/molmed.2015.00017&)

54. Uhl GR, Martinez MJ, Paik P, Sulima A, Bi GH, Iyer MR, et al. Cocaine reward is reduced by decreased expression of receptor-type protein tyrosine phosphatase D (PTPRD) and by a novel PTPRD antagonist. Proc Natl Acad Sci USA. 2018;115:11597–602. doi: 10.1073/pnas.1720446115.  [DOI](https://doi.org/10.1073/pnas.1720446115) | [PMC free article](/articles/PMC6233130/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30348770/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Cocaine%20reward%20is%20reduced%20by%20decreased%20expression%20of%20receptor-type%20protein%20tyrosine%20phosphatase%20D%20(PTPRD)%20and%20by%20a%20novel%20PTPRD%20antagonist&author=GR%20Uhl&author=MJ%20Martinez&author=P%20Paik&author=A%20Sulima&author=GH%20Bi&volume=115&publication_year=2018&pages=11597-602&pmid=30348770&doi=10.1073/pnas.1720446115&)

55. Burton CL, Lemire M, Xiao B, Corfield EC, Erdman L, Bralten J, et al. Genome-wide association study of pediatric obsessive-compulsive traits: shared genetic risk between traits and disorder. Transl Psychiatry. 2021;11:91. doi: 10.1038/s41398-020-01121-9.  [DOI](https://doi.org/10.1038/s41398-020-01121-9) | [PMC free article](/articles/PMC7870035/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33531474/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transl%20Psychiatry&title=Genome-wide%20association%20study%20of%20pediatric%20obsessive-compulsive%20traits:%20shared%20genetic%20risk%20between%20traits%20and%20disorder&author=CL%20Burton&author=M%20Lemire&author=B%20Xiao&author=EC%20Corfield&author=L%20Erdman&volume=11&publication_year=2021&pages=91&pmid=33531474&doi=10.1038/s41398-020-01121-9&)

56. Cox JW, Sherva RM, Lunetta KL, Johnson EC, Martin NG, Degenhardt L, et al., Genome-Wide Association Study of Opioid Cessation. J Clin Med. 2020;9:180.  [DOI](https://doi.org/10.3390/jcm9010180) | [PMC free article](/articles/PMC7019731/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31936517/)

57. García-González J, Tansey KE, Hauser J, Henigsberg N, Maier W, Mors O, et al. Pharmacogenetics of antidepressant response: a polygenic approach. Prog Neuropsychopharmacol Biol Psychiatry. 2017;75:128–34. doi: 10.1016/j.pnpbp.2017.01.011.  [DOI](https://doi.org/10.1016/j.pnpbp.2017.01.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28159590/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Neuropsychopharmacol%20Biol%20Psychiatry&title=Pharmacogenetics%20of%20antidepressant%20response:%20a%20polygenic%20approach&author=J%20Garc%C3%ADa-Gonz%C3%A1lez&author=KE%20Tansey&author=J%20Hauser&author=N%20Henigsberg&author=W%20Maier&volume=75&publication_year=2017&pages=128-34&pmid=28159590&doi=10.1016/j.pnpbp.2017.01.011&)

58. Ward J, Graham N, Strawbridge RJ, Ferguson A, Jenkins G, Chen W, et al. Polygenic risk scores for major depressive disorder and neuroticism as predictors of antidepressant response: meta-analysis of three treatment cohorts. PLoS ONE. 2018;13:e0203896. doi: 10.1371/journal.pone.0203896.  [DOI](https://doi.org/10.1371/journal.pone.0203896) | [PMC free article](/articles/PMC6150505/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30240446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Polygenic%20risk%20scores%20for%20major%20depressive%20disorder%20and%20neuroticism%20as%20predictors%20of%20antidepressant%20response:%20meta-analysis%20of%20three%20treatment%20cohorts&author=J%20Ward&author=N%20Graham&author=RJ%20Strawbridge&author=A%20Ferguson&author=G%20Jenkins&volume=13&publication_year=2018&pages=e0203896&pmid=30240446&doi=10.1371/journal.pone.0203896&)

59. Lek M, Karczewski KJ, Minikel EV, Samocha KE, Banks E, Fennell T, et al. Analysis of protein-coding genetic variation in 60,706 humans. Nature. 2016;536:285–91. doi: 10.1038/nature19057.  [DOI](https://doi.org/10.1038/nature19057) | [PMC free article](/articles/PMC5018207/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27535533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Analysis%20of%20protein-coding%20genetic%20variation%20in%2060,706%20humans&author=M%20Lek&author=KJ%20Karczewski&author=EV%20Minikel&author=KE%20Samocha&author=E%20Banks&volume=536&publication_year=2016&pages=285-91&pmid=27535533&doi=10.1038/nature19057&)
