# Pharmacogenomics of Vincristine-Induced Peripheral Neuropathy in Children with Cancer: A Systematic Review and Meta-Analysis

## Metadata
**Authors:** Aniek Uittenboogaard, Céline L G Neutel, Johannes C F Ket, Festus Njuguna, Alwin D R Huitema, Gertjan J L Kaspers, Mirjam E van de Velde
**Journal:** Cancers
**Date:** 2022 Jan 26
**DOI:** [10.3390/cancers14030612](https://doi.org/10.3390/cancers14030612)
**PMID:** 35158880
**PMCID:** PMC8833506
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8833506/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8833506/pdf/cancers-14-00612.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8833506/pdf/cancers-14-00612.pdf)

## Abstract

**Simple Summary:** 
Vincristine is a drug that is part of the treatment for many children with cancer. Its main side-effect is vincristine-induced peripheral neuropathy (VIPN), which often presents as tingling, pain, and lack of strength in the hands and feet. It is not yet possible to predict which children will suffer from VIPN. In this review, we report on all genetic variations that are associated with VIPN. We found that variations in genes related to vincristine transport, cell structure, hereditary nerve disease, and genes without a previously known connection to vincristine or VIPN are related to VIPN. Variations in genes involved in vincristine breakdown are not significantly associated with VIPN. In conclusion, genetic variations affect a child’s tendency to develop VIPN. In the future, this information might be used to predict the risk of VIPN and adapt treatment on this.

**Abstract:** 
Vincristine-induced peripheral neuropathy (VIPN) is a debilitating side-effect of vincristine. It remains a challenge to predict which patients will suffer from VIPN. Pharmacogenomics may explain an individuals’ susceptibility to side-effects. In this systematic review and meta-analysis, we describe the influence of pharmacogenomic parameters on the development of VIPN in children with cancer. PubMed, Embase and Web of Science were searched. In total, 1597 records were identified and 21 studies were included. A random-effects meta-analysis was performed for the influence of CYP3A5 expression on the development of VIPN. Single-nucleotide polymorphisms (SNPs) in transporter-, metabolism-, cytoskeleton-, and hereditary neuropathy-associated genes and SNPs in genes previously unrelated to vincristine or neuropathy were associated with VIPN. CYP3A5 expression status was not significantly associated with VIPN. The comparison and interpretation of the results of the included studies was limited due to heterogeneity in the study population, treatment protocol and assessment methods and definitions of VIPN. Independent replication is essential to validate the clinical significance of the reported associations. Future research should aim for prospective VIPN assessment in both a discovery and a replication cohort. Ultimately, the goal would be to screen patients upfront to determine optimal vincristine dosage with regards to efficacy and risk of VIPN.

Keywords: vincristine, vincristine-induced peripheral neuropathy, pediatric oncology, pharmacogenomics, meta-analysis, CYP3A5, children, CEP72, cancer

### Simple Summary

Vincristine is a drug that is part of the treatment for many children with cancer. Its main side-effect is vincristine-induced peripheral neuropathy (VIPN), which often presents as tingling, pain, and lack of strength in the hands and feet. It is not yet possible to predict which children will suffer from VIPN. In this review, we report on all genetic variations that are associated with VIPN. We found that variations in genes related to vincristine transport, cell structure, hereditary nerve disease, and genes without a previously known connection to vincristine or VIPN are related to VIPN. Variations in genes involved in vincristine breakdown are not significantly associated with VIPN. In conclusion, genetic variations affect a child’s tendency to develop VIPN. In the future, this information might be used to predict the risk of VIPN and adapt treatment on this.

### Abstract

Vincristine-induced peripheral neuropathy (VIPN) is a debilitating side-effect of vincristine. It remains a challenge to predict which patients will suffer from VIPN. Pharmacogenomics may explain an individuals’ susceptibility to side-effects. In this systematic review and meta-analysis, we describe the influence of pharmacogenomic parameters on the development of VIPN in children with cancer. PubMed, Embase and Web of Science were searched. In total, 1597 records were identified and 21 studies were included. A random-effects meta-analysis was performed for the influence of CYP3A5 expression on the development of VIPN. Single-nucleotide polymorphisms (SNPs) in transporter-, metabolism-, cytoskeleton-, and hereditary neuropathy-associated genes and SNPs in genes previously unrelated to vincristine or neuropathy were associated with VIPN. CYP3A5 expression status was not significantly associated with VIPN. The comparison and interpretation of the results of the included studies was limited due to heterogeneity in the study population, treatment protocol and assessment methods and definitions of VIPN. Independent replication is essential to validate the clinical significance of the reported associations. Future research should aim for prospective VIPN assessment in both a discovery and a replication cohort. Ultimately, the goal would be to screen patients upfront to determine optimal vincristine dosage with regards to efficacy and risk of VIPN.

## 1. Introduction

Vincristine is an important chemotherapeutic agent that is commonly used in treatment for pediatric cancers. It is approved by the United States Food and Drug Administration (FDA) for the treatment of acute lymphoblastic leukemia (ALL), Hodgkin and non-Hodgkin lymphoma, neuroblastoma, rhabdomyosarcoma, low-grade glioma and nephroblastoma. Furthermore, off-label uses include the treatment of Ewing sarcoma and medulloblastoma [[1](#B1-cancers-14-00612),[2](#B2-cancers-14-00612)]. The main side-effect of vincristine is vincristine-induced peripheral neuropathy (VIPN), which often presents as a symmetric sensory-motoric neuropathy progressing distally to proximally [[1](#B1-cancers-14-00612),[2](#B2-cancers-14-00612)]. Presenting signs include foot drop, loss of deep tendon reflexes, impaired balance, pain or tingling [[1](#B1-cancers-14-00612),[2](#B2-cancers-14-00612)]. In addition, patients can suffer from autonomic symptoms such as constipation or orthostatic hypotension. The reported prevalence of VIPN varies, depending on assessment method and study population, but it is estimated that the majority of patients receiving vincristine will experience some form of VIPN during treatment [[1](#B1-cancers-14-00612),[2](#B2-cancers-14-00612),[3](#B3-cancers-14-00612),[4](#B4-cancers-14-00612)]. Up to 30% of patients may suffer from severe VIPN, requiring dose reduction or cessation of treatment [[3](#B3-cancers-14-00612),[5](#B5-cancers-14-00612)]. Suffering from VIPN is associated with a lower health-related quality of life, both by self- and proxy assessment and consistently when using different assessment tools for VIPN [[6](#B6-cancers-14-00612)]. This effect of VIPN on health-related quality of life seems to persevere after treatment, as was shown in a recent study in ALL survivors in which over 16% suffered from long-term VIPN and experienced impact on both physical health and social functioning [[7](#B7-cancers-14-00612)].

It is recognized that different populations might have an altered risk for VIPN [[3](#B3-cancers-14-00612)]. Older age has been associated with an increased risk of VIPN, although results have been inconsistent [[8](#B8-cancers-14-00612),[9](#B9-cancers-14-00612),[10](#B10-cancers-14-00612),[11](#B11-cancers-14-00612),[12](#B12-cancers-14-00612)]. In addition, white children appear to have a higher risk of VIPN than black children [[3](#B3-cancers-14-00612),[9](#B9-cancers-14-00612),[12](#B12-cancers-14-00612),[13](#B13-cancers-14-00612),[14](#B14-cancers-14-00612),[15](#B15-cancers-14-00612)], which is corroborated by a recent study in Kenyan pediatric cancer patients in which only one out of 78 black patients developed severe VIPN and less than 5% developed clinically relevant VIPN, despite the use of sensitive assessment methods [[16](#B16-cancers-14-00612)]. Interestingly, these children are being treated at a higher vincristine dose than what is common in Western countries (2.0 mg/m^2^ as opposed to 1.5 mg/m^2^) [[1](#B1-cancers-14-00612),[16](#B16-cancers-14-00612)]. Studies assessing the relationship between VIPN and vincristine pharmacokinetics (PK) have shown inconsistent results. Some studies show a correlation between VIPN and PK parameters such as area under the curve (AUC) [[17](#B17-cancers-14-00612)], an estimate of vincristine exposure, and intercompartmental clearance [[18](#B18-cancers-14-00612)], whereas others do not confirm these findings [[19](#B19-cancers-14-00612),[20](#B20-cancers-14-00612),[21](#B21-cancers-14-00612),[22](#B22-cancers-14-00612)]. Therefore, potential risk factors for VIPN could be genetic variations in genes involved in vincristine PK, such as variations in the cytochrome (CYP) 450 family of enzymes. Vincristine is predominantly metabolized by CYP3A4 and CYP3A5, of which the latter has a higher intrinsic clearance [[23](#B23-cancers-14-00612)]. Genetic variants in both enzymes result in different metabolic activity [[23](#B23-cancers-14-00612),[24](#B24-cancers-14-00612)]. Racial populations have different distributions of wild-type and variant CYP3A4/5 alleles [[25](#B25-cancers-14-00612),[26](#B26-cancers-14-00612),[27](#B27-cancers-14-00612)]. Combined with the observation that black patients develop less VIPN, it has led to the hypothesis that faster clearance of vincristine in black children results in a lower risk of VIPN in comparison to white patients [[14](#B14-cancers-14-00612)]. Indeed, several studies have described the effect of variations in CYP3A4 and CYP3A5 on the development of VIPN [[8](#B8-cancers-14-00612),[13](#B13-cancers-14-00612),[14](#B14-cancers-14-00612),[16](#B16-cancers-14-00612),[20](#B20-cancers-14-00612),[28](#B28-cancers-14-00612),[29](#B29-cancers-14-00612),[30](#B30-cancers-14-00612),[31](#B31-cancers-14-00612),[32](#B32-cancers-14-00612)]. Differences in VIPN prevalence across populations may thus stem from variations in genetic background, which can be studied via the rapidly expanding field of pharmacogenomics.

Pharmacogenomics aims to assess the influence of genomics on an individuals’ treatment response and susceptibility to side-effects, such as VIPN [[33](#B33-cancers-14-00612),[34](#B34-cancers-14-00612)]. Often, the effect of single nucleotide polymorphisms (SNPs) is assessed [[35](#B35-cancers-14-00612),[36](#B36-cancers-14-00612)]. The frequency distribution of major and minor alleles varies across racial groups and study populations, which has been well characterized in large projects such as the 1000 Genomes Project and the genome Aggregation Database (gnomAD) [[37](#B37-cancers-14-00612),[38](#B38-cancers-14-00612)]. Pharmacogenomics aims to find those SNPs or genetic variations that are biologically relevant [[35](#B35-cancers-14-00612),[36](#B36-cancers-14-00612)]. Two main study designs have been used to assess this: candidate gene studies or population-based genome- or exome-wide association studies (GWAS or EWAS respectively) [[39](#B39-cancers-14-00612),[40](#B40-cancers-14-00612)]. Candidate gene studies determine, a priori, a set of genes, based on available literature or mechanism of action, whose influence on a certain outcome is to be assessed [[39](#B39-cancers-14-00612)]. Population-based GWAS or EWAS, on the other hand, assess the whole exome or genome (by whole exome sequencing (WES) or whole genome sequencing (WGS)) for genetic variation in relation to a certain outcome measure [[39](#B39-cancers-14-00612)]. These studies may therefore result in previously unknown genotype—phenotype associations.

Pharmacogenomics can serve as a guidance tool for precision therapy in which a priori a patients’ genetic susceptibility for side-effects or therapeutic efficacy is determined. Although this has been implemented in clinical practice for some drugs, such as thiopurine methyltransferase (TPMT), this is currently not possible for vincristine [[41](#B41-cancers-14-00612),[42](#B42-cancers-14-00612)]. Especially since there is a lack of understanding of what causes variability in VIPN across patients, pharmacogenomics can provide valuable insight into the pathogenesis of VIPN. If genes affecting vincristine PK are implicated, this may emphasize the potential of therapeutic drug monitoring. Moreover, since it is unlikely that VIPN is caused by differences in PK alone, variation in cellular sensitivity to vincristine and in neuronal pathways could be contributing factors. The implication of genes related to neuronal pathways, the cytoskeleton or cellular integrity with VIPN might then help guiding clinicians in deciding a priori if patients have a high chance of being developing (clinically relevant) VIPN and thus if patients should be monitored more closely than others, or even given an adapted vincristine dosage. In contrast, other patients might be identified who tolerate higher levels of vincristine and might thus not benefit from the generally applied dose capping at 1.5 mg/m^2^. Ultimately, the goal would be to develop a protocol for vincristine in which patients are stratified based on the presence of genetic polymorphisms and given a dosage that limits the risk of severe VIPN while maintaining the highest possible therapeutic efficacy. However, to explore this possibility, the first step is to provide a detailed overview of the effect of SNPs in all reported genes so far on VIPN. Therefore, in this systematic review, we aim to describe the influence of pharmacogenomic parameters on the development of VIPN in children with cancer. Furthermore, we performed a meta-analysis on the influence of CYP3A5 expression status on the development of VIPN.

## 2. Materials and Methods

### 2.1. Protocol and Registration

This study was performed according to the Preferred Reporting Items for Systematic Reviews and Meta-analyses guidelines (PRISMA) [[43](#B43-cancers-14-00612)]. The study protocol was registered at the PROSPERO International prospective register of systematic reviews (registration number CRD42021210437) [[44](#B44-cancers-14-00612)].

### 2.2. Eligibility Criteria

We included prospective and retrospective case-control or cohort studies assessing the relation between VIPN and pharmacogenomic parameters in five or more children with cancer. No restrictions regarding the characteristics of the children with cancer were applied. Pharmacogenomic parameters could include RNA variations such as microRNAs (miRNA) as well. To make sure we described the relationship between VIPN and pharmacogenomic parameters, we excluded descriptive reviews and studies in which no distinction could be made whether patients suffered from VIPN or neuropathy due to other causes, such as diabetes or Charcot-Marie-Tooth disease. If necessary, authors were contacted for clarification or additional data. Systematic reviews and meta-analyses were screened for additional inclusions.

### 2.3. Information Sources

PubMed, Embase, and Clarivate Analytics/Web of Science Core Collection were searched from inception up to 30 September 2021 (by AU and JCFK). Search terms were used as controlled vocabulary (e.g., MeSH) as well as free text terms for title, abstract and author keywords were amongst others: ‘pharmacogenetics’ and ‘child’ and ‘vincristine’ and ‘neuropathy’ or ‘constipation’. No limitations on language or date were applied. See [Supplementary Materials 1](#app1-cancers-14-00612) for the full search strategies in all databases.

### 2.4. Study Selection

Title and abstract screening were performed independently by two reviewers (AU and CLGN) based on pre-defined in- and exclusion criteria. Next, studies were screened full-text for eligibility by two reviewers (AU and CLGN). In case of disagreement, a third reviewer was consulted (MEvdV). Reasons for exclusion after full-text screening were documented. A meta-analysis was performed for studies assessing the relationship between CYP3A5 expression status and VIPN.

### 2.5. Risk of Bias Assessment

Risk of bias was assessed by two independent reviewers (AU and CLGN) according to a modified version of the quality assessment tool for quantitative studies by the Effective Public Health Practice Project (EPHPP), scoring each study as strong, moderate or weak in seven domains: study design, confounders, blinding, data collection methods, withdrawals and drop-outs, analysis, and selection of reported results [[45](#B45-cancers-14-00612)]. The global rating of each study was determined as follows: no weak ratings of the domains resulted in an overall strong rating, one weak rating of the domains resulted in an overall moderate rating, and two or more weak ratings of the domains resulted in an overall weak rating.

### 2.6. Data Extraction and Synthesis

For the included studies, study and baseline characteristics were extracted according to a data extraction template ([Supplementary Materials 2](#app1-cancers-14-00612)). The main outcome was the relationship between VIPN and pharmacogenomic parameters expressed as an effect size (odds ratio (OR)) or *p*-value. ORs were either reported by studies or calculated by the authors. If studies reported multiple ORs using different definitions of cases of VIPN and controls, the significant result with highest clinical relevance was shown (severe VIPN or any grade VIPN). If studies performed both univariate and multivariate analyses, both effect sizes were included in this study.

For the meta-analysis on CYP3A5 expression status and VIPN, dominant OR was calculated if raw data were available, or pre-calculated ORs provided by the authors were used. CYP3A5 expressers were defined as having at least one functional allele (*1) and CYP3A5 non-expressers were defined as having only non-functional variant alleles. If needed, authors were contacted for missing data.

### 2.7. Statistical Analysis

The meta-analysis on CYP3A5 was performed in R, version 3.6.1, using the ‘meta’ package (Rstudio Inc.) [[46](#B46-cancers-14-00612)]. A random-effects model was applied to pool odds ratios since considerable between-study heterogeneity was suspected. The Paule–Mandel procedure was used to estimate variance τ^2^. Heterogeneity was estimated using I^2^ with the following interpretations: I^2^ of 25% indicated low heterogeneity, I^2^ of 50% indicated moderate heterogeneity, and I^2^ of 75% indicated high heterogeneity. The inverse-variance approach was used with the ‘metagen’ command. If study heterogeneity I^2^ was higher than 50%, an assessment of outliers or influential cases was performed. The risk of publication bias was assessed via evaluation of a funnel plot and Egger’s testing for asymmetry. A two-sided *p*-value of <0.05 was considered statistically significant.

## 3. Results

### 3.1. Study Selection

We identified 1597 reports through database searching ([Figure 1](#cancers-14-00612-f001)). After removal of duplicates, the abstracts and titles of 1367 records were screened. Of these, 109 were selected for full-text review. One report was sought for retrieval and no full-text version was published. Since insufficient data was available in the abstract, the report was excluded. Assessment of eligibility resulted in the exclusion of 88 reports based on: consisted of a narrative review (35 reports), no data available on pharmacogenomic parameters (24 reports), no data available on VIPN (13 reports), no administration of vincristine (12 reports), and same data were used as in another report (four reports). Finally, 21 reports were included in this systematic review.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/77ed/8833506/f42c804b3cd4/cancers-14-00612-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8833506_cancers-14-00612-g001.jpg)

PRISMA flow diagram identification of studies included in the systematic review. VIPN = vincristine-induced peripheral neuropathy.

### 3.2. Study Characteristics

Sociodemographic and baseline characteristics of the included studies can be found in [Table 1](#cancers-14-00612-t001). Eighteen studies followed a candidate gene approach [[8](#B8-cancers-14-00612),[10](#B10-cancers-14-00612),[11](#B11-cancers-14-00612),[13](#B13-cancers-14-00612),[14](#B14-cancers-14-00612),[16](#B16-cancers-14-00612),[20](#B20-cancers-14-00612),[22](#B22-cancers-14-00612),[28](#B28-cancers-14-00612),[29](#B29-cancers-14-00612),[30](#B30-cancers-14-00612),[31](#B31-cancers-14-00612),[32](#B32-cancers-14-00612),[47](#B47-cancers-14-00612),[48](#B48-cancers-14-00612),[49](#B49-cancers-14-00612),[50](#B50-cancers-14-00612),[51](#B51-cancers-14-00612)], whereas three studies were population-based GWAS or EWAS [[9](#B9-cancers-14-00612),[52](#B52-cancers-14-00612),[53](#B53-cancers-14-00612)] ([Table 1](#cancers-14-00612-t001)). Four studies included a replication cohort to confirm their findings of the discovery cohort [[8](#B8-cancers-14-00612),[9](#B9-cancers-14-00612),[52](#B52-cancers-14-00612),[53](#B53-cancers-14-00612)]. In total, the number of included patients with available genotype and VIPN data ranged from 24 to 1132. The majority of included patients were diagnosed with ALL and were white ([Table 1](#cancers-14-00612-t001)). The prevalence of moderate to severe VIPN (grade 2–4) ranged from 19.5–53.2%, with the exception of the study by Skiles et al. who reported an incidence of 2.8% in black Kenyan patients [[16](#B16-cancers-14-00612)]. Studies used different definitions of cases (patients with VIPN) and controls (patients without VIPN) ([Table 2](#cancers-14-00612-t002)). Different measurement tools for VIPN were used in the different studies, most often the Common Terminology Criteria for Adverse Events (CTCAE), in which a subset of items was used to score peripheral neuropathy [[8](#B8-cancers-14-00612),[9](#B9-cancers-14-00612),[10](#B10-cancers-14-00612),[13](#B13-cancers-14-00612),[14](#B14-cancers-14-00612),[16](#B16-cancers-14-00612),[20](#B20-cancers-14-00612),[29](#B29-cancers-14-00612),[30](#B30-cancers-14-00612),[47](#B47-cancers-14-00612),[50](#B50-cancers-14-00612),[51](#B51-cancers-14-00612),[52](#B52-cancers-14-00612),[53](#B53-cancers-14-00612)], followed by the modified Balis scale [[9](#B9-cancers-14-00612),[16](#B16-cancers-14-00612),[31](#B31-cancers-14-00612),[32](#B32-cancers-14-00612)] and World Health Organization (WHO) scale [[11](#B11-cancers-14-00612),[48](#B48-cancers-14-00612),[49](#B49-cancers-14-00612)] were used. Finally, The Children’s Cancer Group (CCG) toxicity criteria, National Cancer Institute (NCI) common toxicity criteria, Total Neuropathy Score—Pediatric Vincristine (TNS-PV) and pediatric modified total neuropathy score (ped-mTNS) were all used in one study each [[16](#B16-cancers-14-00612),[22](#B22-cancers-14-00612),[28](#B28-cancers-14-00612),[53](#B53-cancers-14-00612)]. Seven studies assessed VIPN prospectively [[9](#B9-cancers-14-00612),[16](#B16-cancers-14-00612),[20](#B20-cancers-14-00612),[28](#B28-cancers-14-00612),[32](#B32-cancers-14-00612),[47](#B47-cancers-14-00612),[53](#B53-cancers-14-00612)], while the rest of the studies assessed VIPN retrospectively.

### Table 1.

| Author and Year of Publication | Study Design | Patients with Genotype + VIPN Data (n) | Patient Characteristics |  |  |  | Vincristine Dosage |  | VIPN |  | Global Rating Risk of Bias Assessment |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Disease Studied | Age | Male (%) | Race (%) | Single Dosage, (per mg/m2 and max) | Cumulative Dosage (mg) | Method Used for VIPN Assessment | Prevalence VIPN |  |  |  |  |
| Abaji et al., 2018—QcALL cohort [52] | EWAS | 237 | ALL | 82.7% <10 y/o, 17.3% ≥10 y/o. | 54.9 | All white | 1.5, max. 2.0 | Not available | NCI-CTCAE 3.0, retrospectiveGrade 3–4 peripheral neuropathy | 14.8% | Strong |
| Abaji et al., 2018—AIEOP cohort [52] | EWAS | 405 | ALL | 83.2% <10 y/o, 16.8% ≥10 y/o. | 53.1 | All white | 1.5, max. 2.0 | Not available | NCI-CTCAE 3.0, retrospectiveGrade 3–4 peripheral neuropathy | 3.2% | Strong |
| Abo–Bakr et al., 2017 [47] | Candidate gene | 97 | ALL | 79.4% ≤10 y/o, 20.6% >10 y/o | 58.8 | All white | 1.5, max. 2.0 | Not available | NCI-CTCAE 3.0, prospective Foot drop, ileus, vocal cord paralysis, ptosis | Foot drop: 4.1% | Moderate |
| Aplenc et al., 2003 [28] | Candidate gene, case–control | 533 | ALL | 70.0% ≤5 y/o, 30.0% >5 y/o | 32.5 | 5.8 black 94.2 other | 1.5, max. not available | 46.5–64.5 | CCG toxicity criteria, prospective Grade 3 or 4 peripheral neuropathy | 5.3% | Moderate |
| Ceppi et al., 2014 [8] | Candidate gene | 320 | ALL | 80.0% ≤10 y/o, 20.0% >10 y/o | 55.3 | All white | 1.5–2.0, max. 2.0 | 73.5–74.0 | NCI-CTCAE 3.0, retrospective Peripheral neuropathy | Grade 1–2: 20.0%Grade 3–4: 10.6% | Strong |
| Diouf et al., 2015—St. Jude cohort [9] | GWAS | St. Jude: 222. | ALL | 68.9% ≤10 y/o, 31.1% >10 y/o | 42.3 | 67.1 white, 19.8 black, 14.0 other | 1.5, max. 2.0 COG: 1.5– | 54.0 | NCI-CTCAE 1.0, prospective Grade 2–4 peripheral neuropathy | 28.8% | Moderate |
| Diouf et al., 2015—COG cohort [9] | GWAS | 99 | Relapsed ALL | 47.5% ≤10 y/o, 52.5% >10 y/o | 59.6 | 60.6 white, 1.0 black, 38.3 other | 1.5–2.0, max. 2.0–2.5 | 78.0–97.5 | Modified Balis scale, prospectiveGrade 2–4 peripheral neuropathy | 22.9% | Moderate |
| Egbelakin et al., 2011 [29] | Candidate gene | 107 | ALL | Not available | Not available | 92.5 white0.9 black6.5 other | 1.5, max. 2.0 | Not available | NCI-CTCAE 3.0, retrospective Peripheral and autonomic neuropathy | Grade 1–4: 98.1%Grade 3–4: 53.2% | Strong |
| Guilhaumou et al., 2011 [20] | Candidate–gene | 24 | Solid tumors | 57.7% <10 y/o, 42.3% ≥10 y/o | 57.7 | All white | 1.5, max 2.0 | Mean (SD) at time of enrolment: 7.35 (5.30) | NCI-CTCAE 3.0, prospective Pain, peripheral neuropathy, gastro–intestinal toxicity | 33.3% | Moderate |
| Gutierrez–Camino et al., 2016 [10] | Candidate gene | 142 | ALL | 88.7% ≤10 y/o, 11.3% >10 y/o | 57.0 | All white | 1.5, max 2.0 | 15.0–30.0 | NCI-CTCAE 1.0, retrospectiveGrade 2–4 peripheral neuropathy | 25.4% | Strong |
| Gutierrez–Camino et al., 2017 [48] | Candidate gene (miRNA) | 155 | ALL | Mean (SD): 5.1 (3.2) y/o | 58.9 | Mainly white | 1.5, max 2.0 | 15.0–30.0 | WHO criteria, retrospectivePeripheral neuropathy | Grade 1–2: 16.0%Grade 3–4: 10.1% | Strong |
| Kayilioğlu et al., 2017 [30] | Candidate gene, case–control | Cases: 115 (VCR), controls: 50 (no VCR) | Cases: ALL and solid tumors.Controls: no neurological disorders or symptoms | Mean (SD): ALL 7.0 (4.6), solid tumors 7.5 (5.0), controls 10.2 (4.6) | ALL and solid tumors: 61.7Controls: 62.0 | All white | 1.5, max 2.0 | Mean (SD) total: ALL 7.71 (0.89), solid tumors 6.5 (1.5) | NCI-CTCAE 3.0, retrospectiveGrade 2–5 neurotoxicity | 20.8% | Strong |
| Kishi et al., 2007 [13] | Candidate gene | 240 | ALL | 70.4% ≤10 y/o, 29.6% >10 y/o | 59.2 | 69.6 white 18.3 black 12.1 other | 1.5, max 2.0 | 54.0–97.5 | NCI-CTCAE 1.0, prospective/retrospective not available. Peripheral neuropathy and constipation | Grade 3: 12.1%Grade 4: 0.4% | Moderate |
| Li et al., 2019—POG cohort [53] | GWAS | 1069. | ALL | Not available | 52.3 | All white | 1.5, max not available | 18–23 doses of 1.5 mg/m2 | NCI-CTCAE 2.0, prospectiveGrade 3–5 peripheral neuropathy. | 4.8% | Moderate |
| Li et al., 2019—ADVANCE cohort [53] | GWAS | 63 | ALL | Mean (SD): 8.2 (4.7) y/o | 46.0 | All white | 1.5, max 2.0 | Not available | TNS–PV, prospective. Sensory symptoms, temperature and vibration sensibility, strength, tendon reflexes. | Mean + SD: 3.8 (2.6) | Moderate |
| Lopez–Lopez et al., 2016 [11] | Candidate gene | 133 | ALL | Mean (SD): 5.5 (3.4) y/o | 56.6 | Mainly white | 1.5, max 2.0 | 15.0–30.0 | WHO criteria, retrospectivePeripheral neuropathy | Grade 1–2: 18.4%Grade 3–4: 11.8% | Strong |
| Martin–Guerrero et al., 2019 [49] | Candidate gene | 133 | ALL | Mean (SD): 5.5 (3.4) y/o | 56.6 | Mainly white | 1.5, max 2.0 | 15.0–30.0 | WHO criteria, retrospectiveGrade 2–4peripheral neuropathy | 25.4% | Strong |
| McClain et al., 2018 [31] | Candidate gene | 239 | ALL | Mean (SD): 5.8 (3.9) y/o | 53.1 | All white | Not available | Mean (SD), at time of event: extensive metabolizers: 10.0 (5.7), intermediate: 13.4 (13.6), poor: 10.4 (8.9) | Modified Balis scale, retrospectiveGrade 3–4 peripheral neuropathy | Grade 3–4: 18.4% | Strong |
| Plasschaert et al., 2004 [22] | Candidate gene | 52 | ALL | 73.1% < 10 y/o, 26.9% ≥ 10 y/o | 61.5 | 98.1 white1.9 other | Once 1.5, other doses 2.0, max. 2.5 | 13.5 mg/m2 | NCI common toxicity criteriaConstipation | Grade 1–2: 55.8%, Grade 3–4 26.9% | Strong |
| Renbarger et al., 2008 [14] | Race as surrogate for genotype, case–control | Cases: 21 blackControls: 92 white | ALL | Mean (SD): black: 8.2 (4.8) y/o, white: 5.0 (3.1) y/o | Cases + controls: 50.4 | 81.4 white18.6 black | Not available | Mean (SD), Caucasians: 48.5 (14.3), AAs: 42.4 (11.6) | NCI-CTCAE 3.0, retrospectiveNeurotoxicity | Grade 1–4: 34.8% white, 4.8 black | Moderate |
| Sims et al., 2016 [32] | Candidate gene | 52 | BALL | 77.4% < 10 y/o, 22.6% ≥ 10 y/o | 62.2 | 68.5 white31.5 black | 1.5, max. 2.0 | Not available | Modified Balis scale, prospectivePeripheral neuropathy, constipation if grade 3–4 | Grade 1–4: 80.6% white, 76.5% black | Moderate |
| Skiles et al., 2018 [16] | Candidate gene | 72 | Leukemia, lymphoma, solid tumors | Mean (SD): low expressers: 6.1 (5.2), intermediate: 6.5 (4.0), high: 6.1 (4.6) | 53.8 | All black Kenyan | 2.0, max. 2.5 | 8.5 mg/m2 | NCI-CTCAE 4.0, modified Balis scale, Faces Pain Scale, Pediatric Neuropathic Pain Scale, ped–mTNS, all prospective. Peripheral neuropathy and neuropathic pain | NCI–CTCAE: grade 2–4: 2.8%. Ped–mTNS: 4.3% 5 or higher. | Moderate |
| Wright et al., 2019 [51] | Candidate gene, case–control | Cases: 167 (VIPN), controls: 57 (no VIPN) | ALL | Median (IQR): cases 4.8 (3.3–9.0), controls: 5.4 (3.3–9.0) | Cases: 60.4, controls: 40.4 | Mainly white | Not available | Median + IQR: cases: 61.4 (48.0–72.0), controls: 66.0 (51.0–74.8) | NCI-CTCAE 4.0, retrospectivePeripheral neuropathy | Grade 2–4: 167 cases | Strong |
| Zgheib et al., 2018 [50] | Candidate gene | 133 | ALL | Mean (SD): 6.7 (5.0) | 57.1 | All white | Induction and re–induction: 1.5, max. 2.0. Continuation: 2.0, max. 2.0 | Mean (SD), patients without VIPN: 66.0 (6.1), with VIPN grade 2–4: 27.9 (12.1) | NCI-CTCAE 4.0, retrospectivePeripheral neuropathy | Grade 2–4: 19.5% | Strong |

Table 1 Caption: Sociodemographic and clinical characteristics of studies included in the systematic review.

### Table 2.

| Gene | SNP | Allele, Major/Minor | Author and Year of Publication | MAF (%) | Number of Patients (n) |  | Method Effect Size | Effect Size with 95% CI (If Applicable) | Effect |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Cases of VIPN * | Controls * |  |  |  |  |  |  |  |  |
| Transport |  |  |  |  |  |  |  |  |  |
| ABCB1 | rs4728709 | C/T | Ceppi et al., 2014 [8] | TT/TC: 17.1CC: 82.9 | 63 (grade 1–2) | 214 (grade 0) | Dominant OR | 0.3 (0.1–0.9) | Protective 1 |
|   | rs10244266 | T/G | Lopez-Lopez et al., 2016 [11] | 14.3 | 46 (WHO grade 1–4) | 103 (WHO grade 0) | Dominant OR | 2.60 (1.16–5.83) | Risk 2 |
|   | rs10268314 | T/C | Lopez-Lopez et al., 2016 [11] | 14.3 | 27 (WHO grade 1–2) | 103 (WHO grade 0) | Dominant OR | 3.19 (1.23–8.25) | Risk 2 |
|   | rs10274587 | G/A | Lopez-Lopez et al., 2016 [11] | 14.6 | 27 (WHO grade 1–2) | 103 (WHO grade 0) | Dominant OR | 3.48 (1.36–8.86) | Risk 2 |
| ABCC1 | rs1967120 | T/C | Lopez-Lopez et al., 2016 [11] | 27.3 | 18 (WHO grade 3–4) | 103 (WHO grade 0) | Dominant OR | 0.29 (0.09–0.99) | Protective 2 |
|   | rs3743527 | C/T | Lopez-Lopez et al., 2016 [11] | 19.7 | 46 (WHO grade 1–4) | 103 (WHO grade 0) | Dominant OR | 0.32 (0.13–0.79) | Protective 2 |
|   | rs3784867 | C/T | Wright et al., 2019 [51] | 32.0 | 170 (grade 2–4) | 57 (grade 0) | Additive OR | 4.91 (1.99–12.10) | Risk 3 |
|   | rs11642957 | T/C | Lopez-Lopez et al., 2016 [11] | 48.1 | 46 (WHO grade 1–4) | 103 (WHO grade 0) | Dominant OR | 0.43 (0.19–0.98) | Protective 2 |
|   | rs11864374 | G/A | Lopez-Lopez et al., 2016 [11] | 24.4 | 46 (WHO grade 1–4) | 103 (WHO grade 0) | Dominant OR | 0.35 (0.15–0.79) | Protective 2 |
|   | rs12923345 | T/C | Lopez-Lopez et al., 2016 [11] | 15.4 | 46 (WHO grade 1–4) | 103 (WHO grade 0) | Dominant OR | 2.39 (1.08–5.25) | Risk 2 |
|   | rs17501331 | A/G | Lopez-Lopez et al., 2016 [11] | 13.2 | 46 (WHO grade 1–4) | 103 (WHO grade 0) | Dominant OR | 2.50 (1.10–5.68) | Risk 2 |
| ABCC2 | rs12826 | G/A | Lopez-Lopez et al., 2016 [11] | 42.6 | 46 (WHO grade 1–4) | 103 (WHO grade 0) | Dominant OR | 0.24 (0.10–0.54) | Protective |
|   | rs3740066 | G/A | Lopez-Lopez et al., 2016 [11] | 36.2 | 46 (WHO grade 1–4) | 103 (WHO grade 0) | Dominant OR | 0.23 (0.10–0.53) | Protective |
|   | rs2073337 | A/G | Lopez-Lopez et al., 2016 [11] | 45.8 | 18 (WHO grade 3–4) | 103 (WHO grade 0) | Dominant OR | 0.35 (0.10–1.24) | Protective |
|   | rs4148396 | C/T | Lopez-Lopez et al., 2016 [11] | 42.1 | 46 (WHO grade 1–4) | 103 (WHO grade 0) | Dominant OR | 0.36 (0.16–0.81) | Protective |
|   | rs11190298 | G/A | Lopez-Lopez et al., 2016 [11] | 45.0 | 46 (WHO grade 1–4) | 103 (WHO grade 0) | Recessive OR | 2.44 (1.01–5.86) | Risk |
| ABCC1/RALPB1: miR–3117 | rs12402181 | G/A | Gutierrez–Camino et al., 2017 [48] | 14.8 | 19 (WHO grade 3–4) | 128 (WHO grade 0) | Dominant OR | 0.13 (0.02–0.99) | Protective 2 |
| Vincristine metabolism |  |  |  |  |  |  |  |  |  |
| CYP3A4 | rs2740574 | A/G(*1B) | Aplenc et al., 2003 [28] | 8.6 | 28 (CCG grade 3–4) | 505 (CCG grade 0–2) | Allelic OR | 0 (0–0.75) | Protective 2 |
|   |   |   | Guilhaumou et al., 2011 [20] | 6.3 | Nr of neurotoxicity events |  | Chi–square | p = 1.00 | Not significant |
|   |   |   | Kishi et al., 2007 [13] | AA: 79.6AG + GG: 20.4 | 30 (grade 2–4) | 210 (grade 0–1) | Dominant OR | 1.37 (0.57–3.29) | Not significant |
| GSTM1 | Deletion | Non–null/null | Kishi et al., 2007 [13] | Non–null: 57.5Null: 42.5 | 30 (grade 2–4) | 210 (grade 0–1) | OR | 0.46 (0.22–0.94) | Protective2 |
| VDR | rs1544410 | G/A | Kishi et al., 2007 [13] | GG: 45.8AA and AG: 54.2 | 30 (grade 2–4) | 210 (grade 0–1) | Recessive OR | 2.22 (1.06–4.67) | Risk |
| Cytoskeleton–associated |  |  |  |  |  |  |  |  |  |
| ACTG1 | rs1135989 | G/A | Ceppi et al., 2014 [8] | 36.5 | 38 (grade 3–4) | 214 (grade 0) | Dominant OR | 2.8 (1.3–6.3) | Risk 1 |
| CAPG | rs2229668 | G/A | Ceppi et la. 2014 [8] | 12.6 | 39 (grade 3–4) | 214 (grade 0) | Dominant OR | 2.1 (1.1–3.7) | Risk 1 |
|   | rs3770102 | C/A | Ceppi et al., 2014 [8] | 41.4 | 39 (grade 3–4) | 214 (grade 0) | Dominant OR | 0.1 (0.01–0.8) | Protective 1 |
| CEP72 | rs924607 | C/T | Diouf et al., 2015—St. Jude cohort [9] | 36.7 | 64 (grade 2–4) | 158 (grade 0) | Recessive OR | 5.5 (2.5–12.2) | Risk |
|   |   |   | Diouf et al., 2015—COG cohort [9] | 36.4 | 22 (grade 2–4) | 74 (grade 0) | Recessive OR | 3.8 (1.3–11.4) | Risk |
|   |   |   | Gutierrez–Camino et al., 2016 [10] | 39.4 | 36 (WHO grade 2–4) | 106 (WHO grade 0–1) | Recessive OR | 0.7 (0.2–2.4) | Not significant |
|   |   |   | Wright et al., 2019 [51] | TT: 13.5CT and CC: 86.5 | 156 (grade 2–4) | 56 (grade 0) | Recessive OR | 3.4 (0.9–12.6) | Not significant |
|   |   |   | Zgheib et al., 2018 [50] | 36.9 | 23 (grade 2–4) | 107 (grade 0–1) | Recessive OR | 1.04 (0.32–3.43) | Not significant |
| MAPT | rs11867549 | A/G | Martin–Guerrero et al., 2019 [49] | 22.5 | 18 (WHO grade 3–4) | 103 (WHO grade 0) | Dominant OR | 0.21 (0.04–0.96) | Protective 2 |
| SYNE2 | rs2781377 | G/A | Abaji et al., 2018—QcALL cohort [52] | 7.8 | 35 (grade 3–4) | 201 (grade 0) | Additive OR | 2.5 (1.2–5.2) | Risk |
| TUBB2B: miR–202 | rs12355840 | T/C | Martin–Guerrero et al., 2019 [49] | 23.4 | 27 (WHO grade 1–2) | 103 (WHO grade 0) | Dominant OR | 2.88 (1.07–7.72) | Risk |
| Hereditary neuropathy |  |  |  |  |  |  |  |  |  |
| SLC5A7 | rs1013940 | T/C | Wright et al., 2019 [51] | 15.2 | 170 (grade 2–4) | 57 (grade 0) | Additive OR | 8.60 (1.68–44.15) | Risk 3 |
| Other (GWAS/EWAS studies) |  |  |  |  |  |  |  |  |  |
| BAHD1 | rs3803357 | C/A | Abaji et al., 2018—QcALL cohort [52] | 41.7 | 35 (grade 3–4) | 201 (grade 0) | Dominant OR | 0.35 (0.2–0.7) | Protective |
| COCH | rs1045466 | T/G | Li et al., 2020—POG cohort [53] | 38 | Maximum neuropathy score |  | Dominant HR | 0.27 (0.16–0.50) | Protective |
|   |   |   | Li et al., 2020—ADVANCE cohort [53] | 33 |   |   | Linear regression | −3.56 (−5.45;−1.67) | Protective |
| Chromosome 12/ chemerin | rs7963521 | T/C | Li et al., 2020—POG cohort [53] | 41 | Maximum neuropathy score |  | Additive HR | 2.23 (1.49–3.35) | Risk |
|   |   |   | Li et al., 2020—ADVANCE cohort [53] | 43 |   |   | Additive HR | 2.16 (0.53–3.70) | Not significant |
| ETAA1 | rs17032980 | A/G | Diouf et al., 2015—St. Jude cohort [9] | 26.6 | 64 (grade 2–4) | 158 (grade 0) | Allelic OR | 3.17 (1.95–5.17) | Risk |
|   |   |   | Diouf et al., 2015—COG cohort [9] | 19.2 | 22 (grade 2–4) | 74 (grade 0) | Allelic OR | 10.4 (2.97–36.15) | Risk |
| MRPL4 | rs10513762 | C/T | Abaji et al., 2018—QcALL cohort [52] | 7.0 | 35 (grade 3–4) | 202 (grade 0) | Dominant OR | 3.3 (1.4–7.7) | Risk |
| MTNR1B | rs12786200 | C/T | Diouf et al., 2015—St. Jude cohort [9] | 22.7 | 64 (grade 2–4) | 158 (grade 0) | Allelic OR | 0.23 (0.13–0.40) | Protective |
|   |   |   | Diouf et al., 2015—COG cohort [9] | 20.7 | 22 (grade 2–4) | 74 (grade 0) | Allelic OR | 0.24 (0.08–0.76) | Protective |
|   |   |   | Zgheib et al., 2018 [50] | 18.1 | 23 (grade 2–4) | 107 (grade 0–1) | Dominant OR | 0.59 (0.22–1.62) | Not significant |
| NDUFAF6 | rs7818688 | C/A | Diouf et al., 2015—St. Jude cohort [9] | 12.6 | 64 (grade 2–4) | 158 (grade 0) | Allelic OR | 4.26 (2.45–7.42) | Risk |
|   |   |   | Diouf et al., 2015—COG cohort [9] | 14.1 | 22 (grade 2–4) | 74 (grade 0) | Allelic OR | 4.59 (1.35–15.59) | Risk |
| TMEM215 | rs4463516 | C/G | Diouf et al., 2015—St. Jude cohort [9] | 33.6 | 64 (grade 2–4) | 158 (grade 0) | Allelic OR | 3.17 (1.95–5.17) | Risk |
|   |   |   | Diouf et al., 2015—COG cohort [9] | 24.2 | 22 (grade 2–4) | 74 (grade 0) | Allelic OR | 4.94 (1.65–14.79) | Risk |
| miRNA |  |  |  |  |  |  |  |  |  |
| miR–4481 | rs7896283 | T/C | Gutierrez–Camino et al., 2017 [48] | 37.5 | 19 (WHO grade 3–4) | 128 (WHO grade 0) | Dominant OR | 4.69 (1.43–15.43) | Risk 2 |
| miR–6076 | rs35650931 | G/C | Gutierrez–Camino et al., 2017 [48] | 8.7 | 47 (WHO grade 1–4) | 128 (WHO grade 0) | Dominant OR | 0.22 (0.05–0.97) | Protective 2 |

Table 2 Caption: Single-nucleotide polymorphisms that were significantly associated with vincristine-induced peripheral neuropathy in the pediatric oncology population.

### 3.3. Risk of Bias

An overview per domain of risk of bias can be found in [Table S1](#app1-cancers-14-00612). Twelve [[8](#B8-cancers-14-00612),[10](#B10-cancers-14-00612),[11](#B11-cancers-14-00612),[22](#B22-cancers-14-00612),[29](#B29-cancers-14-00612),[30](#B30-cancers-14-00612),[31](#B31-cancers-14-00612),[48](#B48-cancers-14-00612),[49](#B49-cancers-14-00612),[50](#B50-cancers-14-00612),[51](#B51-cancers-14-00612),[52](#B52-cancers-14-00612)] and nine studies [[9](#B9-cancers-14-00612),[13](#B13-cancers-14-00612),[14](#B14-cancers-14-00612),[16](#B16-cancers-14-00612),[20](#B20-cancers-14-00612),[28](#B28-cancers-14-00612),[32](#B32-cancers-14-00612),[47](#B47-cancers-14-00612),[53](#B53-cancers-14-00612)] scored an overall strong and moderate rating on risk of bias, respectively. No studies received an overall weak rating on risk of bias.

### 3.4. Association between Pharmacogenomic Parameters and VIPN

[Table 2](#cancers-14-00612-t002) and [Table 3](#cancers-14-00612-t003) show an overview of all SNPs found to have a statistically significant and non-significant association with VIPN, respectively. [Figure 2](#cancers-14-00612-f002) shows a schematic overview of the function of genes associated with VIPN. Sixteen SNPs in three ATP-binding cassette transporter genes (ABCB1, ABCC1, ABCC2) and one SNP in an miRNA targeting ABCC1/RalA binding protein 1 (RALPB1) were described to be significantly associated with VIPN ([Table 2](#cancers-14-00612-t002)). Ten SNPs were associated with a protective effect against VIPN, whereas seven SNPs were associated with an increased risk of VIPN. Of note, the strongest protective associations with high precision were reported for SNPs rs3740066 and rs12826 in ABCC2 (OR 0.23, 95% CI 0.10−0.53, and 0.24, 95% CI 0.10−0.54 respectively). The strongest risk association with acceptable precision was reported for rs3784867 in ABCC1 (OR 4.91, 95% CI 1.99−12.10).

### Table 3.

| Gene | SNP | Author and Year of Publication |
| --- | --- | --- |
| ABCB1 | rs1045642 | Plasschaert et al., 2004 [22], Ceppi et al., 2014 [8], Zgheib et al., 2018 [50] |
|   | rs1128503 | Ceppi et al., 2014 [8], Zgheib et al., 2018 [50] |
|   | rs2032582 | Plasschaert et al., 2004 [22], Ceppi et al., 2014 [8] |
| ABCC2 | rs717620 | Zgheib et al., 2018 [50] |
| ACTG1 | rs1139405 | Ceppi et al., 2014 [8] |
|   | rs7406609 | Ceppi et al., 2014 [8] |
| CAPG | rs6886 | Ceppi et al., 2014 [8] |
| CYP1A1 | rs4646903 | Abo-Bakr et al., 2017 1 [47] |
| GSTP1 | rs1695 | Kishi et al., 2007 [13], Abo-Bakr et al., 2017 1 [47] |
| GSTT1 | Deletion | Kishi et al., 2007 [13] |
| MAP4 | rs11268924 | Ceppi et al., 2014 [8] |
|   | rs1137524 | Ceppi et al., 2014 [8] |
|   | rs1875103 | Ceppi et al., 2014 [8] |
|   | rs11711953 | Ceppi et al., 2014 [8] |
| MDR1 | Exon 21, G > T/A | Kishi et al., 2007 [13] |
|   | Exon 26, C/T | Kishi et al., 2007 [13] |
| MTHFR | rs1801133 | Kishi et al., 2007 [13] |
|   | rs1801131 | Kishi et al., 2007 [13] |
| SLC19A1 | rs1051266 | Kishi et al., 2007 [13] |
| TPMT | Combined genotypes: 238GG, 460GG, 719AA/others | Kishi et al., 2007 [13] |
| TUBB | rs6070697 | Ceppi et al., 2014 [8] |
|   | rs10485828 | Ceppi et al., 2014 [8] |
| TYMS | Enhancer repeat: others/3AND3 | Kishi et al., 2007 [13] |
| UGT1A1 | Enhancer repeat: others/7AND7 | Kishi et al., 2007 [13] |
| VDR | rs2228570 | Kishi et al., 2007 [13] |
| XRCC1 | rs1799782 | Abo-Bakr et al., 2017 1 [47] |

Table 3 Caption: Single-nucleotide polymorphisms that were not significantly associated with vincristine-induced peripheral neuropathy in the pediatric oncology population.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/77ed/8833506/89709930a55d/cancers-14-00612-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8833506_cancers-14-00612-g002.jpg)

Schematic overview of the function of genes associated with VIPN. Red: described SNPs in this gene are associated with a higher risk of VIPN; green: described SNPs in this gene are associated with a lower risk of VIPN, brown: described SNPs in this gene are associated with both a higher and lower risk of VIPN (different per SNP). Created with BioRender.com.

In terms of metabolism-associated genes, a deletion in glutathione S-transferase mu 1 (GSTM1) and an SNP in vitamin D receptor (VDR) were implicated with a heightened and a decreased risk to VIPN, respectively ([Table 2](#cancers-14-00612-t002)) [[13](#B13-cancers-14-00612)]. Furthermore, six SNPs in cytoskeleton-associated genes or in miRNAs targeting those were associated with VIPN (microtubule associated protein tau (MAPT), targeting tubulin beta 2B class IIB (TUBB2), actin gamma 1 (ACTG1), capping actin protein gelsolin like (CAPG) and spectrin repeat containing nuclear envelope protein 2 (SYNE2)) ([Table 2](#cancers-14-00612-t002)). Of those, two SNPs were related to microtubules (MAPT and TUBB2) and associated with a protective effect and an increased risk of VIPN, respectively ([Table 2](#cancers-14-00612-t002)) [[49](#B49-cancers-14-00612)]. The four other SNPs were located in cytoskeleton-associated genes (ACTG1, CAPG, and SYNE2) and associated with a CTCAE grade 3−4 VIPN ([Table 2](#cancers-14-00612-t002)) [[8](#B8-cancers-14-00612),[52](#B52-cancers-14-00612)]. The latter passed the stringent significance threshold for multiple comparisons, but the results could not be confirmed in a replication cohort [[52](#B52-cancers-14-00612)]. The strongest protective association was noted for SNP rs3770102 in CAPG with an effect size of 0.1, although the uncertainty was high (95% CI 0.01−0.8). One SNP in a gene associated with hereditary neuropathies (solute carrier family 5 member 7 (SLC5A7)) resulted in an increased susceptibility to VIPN ([Table 2](#cancers-14-00612-t002)) [[51](#B51-cancers-14-00612)]. The reported effect size was large, but the size of the confidence interval indicated relatively high uncertainty (OR 8.60, 95% CI 1.68−44.15) Except for the SNP in SYNE2, all aforementioned SNPS were solely assessed in a discovery cohort and no replication studies were performed for any of those associations [[52](#B52-cancers-14-00612)].

Four studies assessed the influence of SNP rs924607 in CEP72 on the development of CTCAE or WHO grade 2−4 VIPN [[9](#B9-cancers-14-00612),[10](#B10-cancers-14-00612),[50](#B50-cancers-14-00612),[51](#B51-cancers-14-00612)]. In both their discovery and replication cohort, Diouf et al. described an increased risk of VIPN in patients with the risk genotype [[9](#B9-cancers-14-00612)]. The strongest association was seen in the discovery cohort (OR 5.5, 95% CI 2.5−12.2); this effect size was smaller in the replication cohort (OR 3.8, 95% CI 1.3−11.4) where the prevalence of VIPN was also slightly lower (28.8 and 22.9% respectively). Three replication studies could not confirm these findings [[10](#B10-cancers-14-00612),[50](#B50-cancers-14-00612),[51](#B51-cancers-14-00612)].

GWAS or EWAS demonstrated significant associations between VIPN and eight SNPs in genes previously not associated with neuropathy, vincristine mechanism of action or metabolism ([Table 2](#cancers-14-00612-t002)). All studies first reporting these associations made use of both a discovery and replication cohort to validate their results [[9](#B9-cancers-14-00612),[52](#B52-cancers-14-00612),[53](#B53-cancers-14-00612)]. SNPs in cochlin (COCH), Ewing’s tumor-associated antigen 1 (ETAA1), melatonin receptor 1B (MTNR1B), NADH: ubiquinone oxidoreductase complex assembly factor (NDUFAF6), and transmembrane protein 215 (TMEM215) were significantly associated with VIPN both in a discovery and replication cohort, whereas this relationship was only established in the discovery cohort for SNPs in bromo adjacent homology domain containing 1 (BAHD1), chromosome 12/chemerin, and mitochondrial ribosomal protein L4 (MRPL4). The described SNPS in BAHD1 and COCH were protective against VIPN. The strongest protective association with high precision was reported for the latter (OR 0.27, 95% CI 0.16−0.50). The SNPs in chromosome 12/chemerin, ETAA1, MRPL4, NDUFAF6, and TMEM215 were associated with an increased risk of VIPN. The SNP in ETAA1 showed a strong effect on risk of VIPN, especially in the replication cohort of Diouf et al., although the precision was relatively low (OR 10.4, 95% CI 2.97−36.15). Moreover, the SNPs in NDUFAF6 and TMEM215 also showed relatively large effect sizes with acceptable uncertainty in both a discovery and replication cohort. Finally, Diouf et al. described an SNP in melatonin receptor 1B (MTNR1B) as protective against VIPN both in a discovery and replication cohort with a large effect size and high precision (OR 0.23, 95% CI 0.13−0.40, and OR 0.24, 95% CI 0.08−0.76), but another study by Zgheib et al. could not confirm these results [[9](#B9-cancers-14-00612),[50](#B50-cancers-14-00612)]. All significant associations passed the stringent threshold for multiple comparisons.

Gutierrez-Camino et al. found two SNPs in miRNA to be associated with VIPN, of which one miRNA could be related to the axon-guidance pathway, whereas the other could not be related to any known vincristine- or neurotoxicity-related pathway ([Table 2](#cancers-14-00612-t002)) [[48](#B48-cancers-14-00612)].

Several studies assessed the influence of covariates such as cumulative vincristine dosage, treatment protocol, and patient characteristics on their results, but these covariates did not have a significant influence on the reported associations ([Table S2](#app1-cancers-14-00612)). Only the significant associations reported by Diouf et al. did not maintain their significance when corrected for genetically defined ancestry and cumulative vincristine dosage ([Table S2](#app1-cancers-14-00612)) [[9](#B9-cancers-14-00612)].

### 3.5. CYP3A4 and CYP3A5

In regard to CYP3A4, Aplenc et al. found an SNP in CYP3A4 to be protective against VIPN [[28](#B28-cancers-14-00612)], but two follow-up studies could not replicate these findings ([Table 2](#cancers-14-00612-t002)) [[13](#B13-cancers-14-00612),[20](#B20-cancers-14-00612)]. Furthermore, ten studies assessed the influence of CYP3A5 expression on the development of VIPN [[8](#B8-cancers-14-00612),[13](#B13-cancers-14-00612),[14](#B14-cancers-14-00612),[16](#B16-cancers-14-00612),[20](#B20-cancers-14-00612),[28](#B28-cancers-14-00612),[29](#B29-cancers-14-00612),[30](#B30-cancers-14-00612),[31](#B31-cancers-14-00612),[32](#B32-cancers-14-00612)]. Of those studies, nine either presented a pre-calculated OR or raw data to calculate an OR and could thus be included in the meta-analysis [[8](#B8-cancers-14-00612),[13](#B13-cancers-14-00612),[14](#B14-cancers-14-00612),[16](#B16-cancers-14-00612),[20](#B20-cancers-14-00612),[28](#B28-cancers-14-00612),[29](#B29-cancers-14-00612),[31](#B31-cancers-14-00612),[32](#B32-cancers-14-00612)]. If possible, dominant ORs were calculated based on data presented in the article or additional data provided by the authors ([Tables S3 and S4](#app1-cancers-14-00612)). As shown in [Figure 3](#cancers-14-00612-f003), there was no statistically significant pooled effect between CYP3A5 expression status and the development of VIPN (pooled OR 0.69, 95% CI 0.38−1.26, I^2^ = 50%, τ^2^ = 0.33). The study by Kayilioğlu et al., which could not be included in the meta-analysis due to the unavailability of appropriate data, did not find a significant association either ([Table S3](#app1-cancers-14-00612)). However, all studies either found no effect of CYP3A5 status or found that expression of CYP3A5 was a protective factor for VIPN; the opposite was not reported. Of note, the included studies all used different definitions for cases (patients with VIPN) and controls (patients without VIPN) ([Table S3](#app1-cancers-14-00612)), likely contributing to the moderate heterogeneity. In addition, all studies performed genotyping to determine CYP3A5 expression status, whereas Renbarger et al., the study with the strongest association, used race as a surrogate. All studies compared expressers of CYP3A5 to non-expressers, except for Ceppi et al. who calculated allelic OR ([Table S3](#app1-cancers-14-00612)). Evaluation of the funnel plot and Egger’s test for asymmetry were not indicative of obvious publication bias (*p*-value Egger’s test = 0.40) ([Figure S1](#app1-cancers-14-00612)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/77ed/8833506/48c65376638a/cancers-14-00612-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8833506_cancers-14-00612-g003.jpg)

Forest plot showing the effect of CYP3A5 expression status on VIPN, ORs describe the effect of expression of CYP3A5 in comparison to non-expression of CYP3A5 [8,13,14,16,20,28,29,31,32]. The functional allele is *1 and variant alleles are *3 (rs776746), *6 (rs10264272), and *7 (rs41303343). A dominant model was adopted: patients with at least one *1 allele are considered to be expressers of CYP3A5. Patients without *1 allele are considered to be non-expressers of CYP3A5.

## 4. Discussion

This systematic review shows that pharmacogenomic parameters have a significant influence on VIPN in children with cancer and show potential for clinical relevance. Several SNPs in genes related to vincristine metabolism, hereditary neuropathy, the cytoskeleton and microtubules have been associated with VIPN. Furthermore, population-based GWAS and EWAS identified significant interactions with SNPs in genes previously unrelated to VIPN or vincristine. Our meta-analysis showed that CYP3A5 expression status was not a significant risk factor for VIPN.

Several significant associations were found between SNPs in the ABC family of genes (ABCB1, ABCC1, ABCC2). These genes code for transmembrane proteins that mediate vincristine efflux across cell membranes; variations may thus contribute to different vincristine levels and therefore VIPN ([Figure 2](#cancers-14-00612-f002)) [[54](#B54-cancers-14-00612),[55](#B55-cancers-14-00612)]. Three candidate gene studies described associations between ABCB1, ABCC1 and ABCC2 and VIPN, all in children with ALL [[8](#B8-cancers-14-00612),[11](#B11-cancers-14-00612),[51](#B51-cancers-14-00612)]. Of note, the vast majority of the associations were reported in the same discovery cohort (Lopez-Lopez et al.) [[11](#B11-cancers-14-00612)]. Interestingly, all associations between SNPs in ABCC2 and VIPN passed a significance threshold corrected for multiple comparisons and showed the largest effect sizes, suggesting that a stronger relationship may exist between ABCC2 and VIPN than between ABCB1/ABCC1 and VIPN. Indeed, several cell line studies have shown that ABCC2 function was associated with vincristine resistance or sensitivity [[56](#B56-cancers-14-00612),[57](#B57-cancers-14-00612),[58](#B58-cancers-14-00612)]. However, none of the reported associations have been replicated in other cohorts and results should thus be interpreted with caution.

Furthermore, SNPs in cytoskeleton-associated genes were associated with VIPN ([Figure 2](#cancers-14-00612-f002)). Vincristine exerts its cytostatic effect via binding to the β-subunit of tubulins, which inhibits microtubule polymerization and consequently causes arrest of mitosis in the metaphase [[1](#B1-cancers-14-00612),[59](#B59-cancers-14-00612)]. During cell division, there is a well-known interaction between microtubules and the actin cytoskeleton; the latter contributes to mitotic spindle assembly and formation [[60](#B60-cancers-14-00612),[61](#B61-cancers-14-00612),[62](#B62-cancers-14-00612)]. It is possible that SNPs in genes that affect microtubule formation or the actin cytoskeleton affect binding of vincristine to tubulins or the effect of vincristine binding to tubulins. While this can result in an altered risk of VIPN, one could also hypothesize that this influences the effect of vincristine on mitotic spindle disintegration and thus ultimately the cytotoxic effect. Should that be the case, patients with a lower risk of VIPN might also experience less antitumor effect in comparison with patients with a higher risk of VIPN, which would argue for dose individualization in which standard dose capping is not applied to every patient. Future studies assessing the relationship between VIPN incidence and long-term treatment outcome, correcting for received cumulative vincristine dosage, may provide further insight. Of note, the studies reporting these associations concerned predominantly white patients with ALL and except for one study, the reported associations have not been assessed in a replication cohort [[8](#B8-cancers-14-00612),[49](#B49-cancers-14-00612),[52](#B52-cancers-14-00612)]. Therefore, these results regarding SNPs in microtubule- and cytoskeleton-associated genes should be interpreted with caution until independent replication is performed. An association that has been replicated in several independent studies is the association between rs924607 in CEP72 and VIPN. CEP72 encodes for a centrosomal protein that is required for adequate chromosome segregation [[63](#B63-cancers-14-00612),[64](#B64-cancers-14-00612)]. Centrosomes enable correct alignment of chromosomes during mitosis by controlling the position and orientation of the microtubule spindles at the spindle poles [[63](#B63-cancers-14-00612),[64](#B64-cancers-14-00612)]. A recent meta-analysis on the effect of this SNP in CEP72 on VIPN in children with ALL confirmed this finding across three studies in the continuation phase of treatment [[65](#B65-cancers-14-00612)]. A clinical trial enrolling newly diagnosed children with ALL and lymphomas is randomizing patients with the high risk (TT) genotype between a decreased dosage (1.0 mg/m^2^) and conventional dosage (1.5 mg/m^2^) of vincristine during the continuation phase of treatment [[66](#B66-cancers-14-00612)]. Recruitment is still ongoing. However, it is important to note that the effect of CEP72 on VIPN likely differs depending on treatment phase and genetic background, since this association was not significant in a white Spanish population in the induction phase and a white Arab population during induction and continuation phases [[10](#B10-cancers-14-00612),[50](#B50-cancers-14-00612)].

In addition, we assessed the effect of CYP3A5 expression status on VIPN in a meta-analysis and found an overall pooled effect of 0.69 (95% CI 0.38−1.26) ([Figure 2](#cancers-14-00612-f002)). Two studies reported a significant effect of CYP3A5 expression status on VIPN. Renbarger et al. found the strongest association, but it is important to note that they used race as a surrogate for CYP3A5 expression status [[14](#B14-cancers-14-00612)]. This can be debated, since white children can express a CYP3A5 as well, albeit less often than in black children (10−20% and >55%, respectively) [[67](#B67-cancers-14-00612),[68](#B68-cancers-14-00612)]. Furthermore, Kishi et al. found a significant association in a relatively large cohort (240 children) with a prevalence of 12.5% of severe VIPN [[13](#B13-cancers-14-00612)]. However, all other studies could not replicate these findings, even those with sample sizes adequately powered to detect a difference, such as the population wide GWAS or EWAS. Nonetheless, no study reported CYP3A5 expression as a risk factor for VIPN. In conclusion, this meta-analysis shows that there is no significant effect of CYP3A5 expression status on VIPN.

The comparison and interpretation of the results of the included studies is limited due to heterogeneity in the study population, treatment protocol and varying assessment methods and definitions of VIPN. Firstly, cumulative vincristine dosage varied between 6.5 and 97.5 mg across studies. Since cumulative vincristine dosage likely is an independent risk factor for VIPN, it could be of influence when establishing a relationship between a pharmacogenomic parameter and VIPN [[1](#B1-cancers-14-00612),[3](#B3-cancers-14-00612)]. However, except for Diouf et al., studies that included cumulative vincristine dosage or treatment protocol as covariates did not find an effect on outcome [[8](#B8-cancers-14-00612),[9](#B9-cancers-14-00612),[11](#B11-cancers-14-00612),[13](#B13-cancers-14-00612),[49](#B49-cancers-14-00612),[51](#B51-cancers-14-00612),[52](#B52-cancers-14-00612)]. Another source of heterogeneity was the variety of measurement approaches to report VIPN. The majority of studies used the NCI-CTCAE for peripheral neuropathy, but other studies used the modified Balis scale, WHO scale, or other methods, to quantify VIPN. The sensitivity and specificity differ across assessment methods and their results can thus not be compared one-on-one [[1](#B1-cancers-14-00612),[3](#B3-cancers-14-00612),[17](#B17-cancers-14-00612)]. Similarly, seven studies assessed VIPN prospectively, whereas the other studies assessed VIPN retrospectively. Retrospective VIPN assessment is less sensitive, especially when quantifying the presence of any grade or low grade VIPN [[1](#B1-cancers-14-00612),[3](#B3-cancers-14-00612),[17](#B17-cancers-14-00612)]. Moreover, the majority of included studies had a study population of less than 150 patients and were thus limited by a relatively small sample size. This is further reinforced by the observation that most studies assessed the relationship between several genetic variations and VIPN and thus performed multiple comparisons. Therefore, it is advised to counteract the likelihood of false positives by adjusting the significance threshold with for example Bonferroni or False Discovery Rate (FDR) correction [[69](#B69-cancers-14-00612),[70](#B70-cancers-14-00612)]. This systematic review shows that approximately half of the reported significant associations either did not pass a stringent significance threshold, or that it was not applied. However, associations that did not pass the stringent significance threshold could still be biologically relevant since (stringent) statistical significance should be interpreted within the context of the study design [[35](#B35-cancers-14-00612),[39](#B39-cancers-14-00612)]. A lack of statistical significance could be an indication that the study was not adequately powered to detect a difference [[35](#B35-cancers-14-00612),[39](#B39-cancers-14-00612)].

Independent replication is essential to validate the clinical significance of reported associations [[33](#B33-cancers-14-00612),[34](#B34-cancers-14-00612)]. This could be facilitated by the uniform and reliable assessment of VIPN while employing a sensitive assessment method such as the ped-mTNS or ped-TNV [[2](#B2-cancers-14-00612),[3](#B3-cancers-14-00612)]. A larger number of patients could be included if VIPN was consistently noted in patient charts. Subsequently, this would allow for reliable comparisons between studies. Furthermore, the growing availability of high-throughput techniques allows for genome- or exome wide analysis in an increasing number of studies. Interestingly, in this systematic review and meta-analysis, the majority of included studies followed a candidate gene approach, limiting the findings to pre-defined selection of genes. The studies that employed population-wide GWA or EWA analysis expanded our knowledge by uncovering genotype-phenotype associations that were not previously described in relation to VIPN. To actualize the potential of pharmacogenomic testing, future studies should apply sensitive and uniform measurement approaches to report VIPN while employing robust genotyping methods such as EWAS or GWAS. These data could be used by consortiums such as the Clinical Pharmacogenetics Implementation Consortium (CPIC) to create guidelines for clinicians to implement pharmacogenomic testing into clinical practice. The CPIC assesses pharmacogenomic studies by assigning levels of evidence and syntheses the results [[71](#B71-cancers-14-00612)]. Another promising approach to assess the influence of pharmacogenomics on VIPN is combining the effect of different SNPs or genetic variations in one effect size, since single SNPs most likely have limited clinical relevance. Such an approach was adopted by Abaji et al., in which a combined-effect model was established to assess the additive effect of several SNPs associated with VIPN [[52](#B52-cancers-14-00612)]. Patients were classified in risk groups according to their weighted genetic risk score; and this model could successfully predict the risk of VIPN in both their discovery and replication cohort [[52](#B52-cancers-14-00612)].

The strength of this systematic review and meta-analysis is the inclusion of all studies assessing the relationship between pharmacogenomic parameters and VIPN. No restrictions were applied regarding patient or study characteristics. Furthermore, we followed the PRISMA guidelines and two independent authors performed screening, data extraction and risk of bias assessment. A standardized risk of bias assessment was performed with the validated EPHPP tool [[45](#B45-cancers-14-00612)]. The weakness of this study is that the meta-analysis was limited to CYP3A5 expression status and that the other pharmacogenomic parameters could not be assessed in a meta-analysis.

## 5. Conclusions

From this systematic review, we can conclude that the following pharmacogenomic parameters have a significant influence on VIPN in children with cancer: SNPs in ABCB1, ABCC1, ABCC2, CYP3A4, GSTM1, VDR, ACTG1, CAPG, CEP72, MAPT, SYNE2, TUBB2B, SLC5A7, BAHD1, COCH, chromosome 12/chemerin, ETAA1, MRPL4, MTNR1B, NDUFAF6, TMEM215 and in three miRNAs. Our meta-analysis shows that CYP3A5 expression does not result in a heightened susceptibility of VIPN. To actualize the potential of pharmacogenomic testing, future research should prospectively assess VIPN with a sensitive measurement tool in both a discovery and replication cohort. Ultimately, the goal would be to develop an individualized protocol based on a patients’ genotype, taking all risk and protective genes into account, and subsequently give patients a dosage that limits the risk of VIPN while maintaining highest possible therapeutic efficacy. Dosage reductions or cessation of treatment, or for some patients even standardized dose capping, would no longer be necessary.

## Acknowledgments

The authors would like to thank Skiles et al. and Guilhaumou et al. for contributing original data to this study. Furthermore, they would like to gracefully acknowledge Sharon Remmelzwaal for advice on the statistical analysis.

## Supplementary Materials

The following are available online at [https://www.mdpi.com/article/10.3390/cancers14030612/s1](https://www.mdpi.com/article/10.3390/cancers14030612/s1), Supplementary Materials 1, Supplementary Materials 2, Figure S1: Funnel plot for evaluation of small study effects in the meta-analysis on the effect of CYP3A5 expression status on vincristine-induced peripheral neuropathy, Table S1: Risk of bias of the included studies, Table S2: Studies that assessed the effect of covariates on significant associations, Table S3: Effect of CYP3A5 single-nucleotide polymorphisms (SNPs) on vincristine-induced peripheral neuropathy (VIPN). Reference allele is *1 (extensive metabolizer), variant alleles are *3 (rs776746), *6 (rs10264272), and *7 (rs41303343). Patients with at least one *1 allele are considered to be expressers of CYP3A5. Patients without *1 allele are considered to be non-expressers of CYP3A5. Recessive OR: non-expresser (*3/*3) compared to expresser (other genotypes), Table S4: Raw data used to calculate odds ratios (OR) for the effect of CYP3A5 single-nucleotide polymorphisms (SNPs) on vincristine-induced peripheral neuropathy (VIPN). Reference allele is *1 (extensive metabolizer), variant alleles are *3 (rs776746), *6 (rs10264272), and *7 (rs41303343). Patients with at least one *1 allele are considered to be expressers of CYP3A5. Patients without *1 allele are considered to be non-expressers of CYP3A5. Dominant OR: non-expresser (*3/*3) compared to expresser (other genotypes).

## Author Contributions

Conceptualization: A.U., M.E.v.d.V., G.J.L.K. Methodology: A.U., C.L.G.N., J.C.F.K. Validation: A.U., C.L.G.N. Formal analysis: A.U., C.L.G.N. Writing—original draft preparation: A.U. Writing—review & editing: A.U., M.E.v.d.V., F.N., A.D.R.H., G.J.L.K. Visualization: A.U. Supervision: M.E.v.d.V., F.N., A.D.R.H., G.J.L.K. All authors have read and agreed to the published version of the manuscript.

## Funding

This research received no external funding.

## Data Availability Statement

The data presented in this study are available in this article and in the [Supplementary Materials](#app1-cancers-14-00612).

## Conflicts of Interest

The authors declare no conflict of interest.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data presented in this study are available in this article and in the [Supplementary Materials](#app1-cancers-14-00612).

### Supplementary Materials

### Data Availability Statement

The data presented in this study are available in this article and in the [Supplementary Materials](#app1-cancers-14-00612).

## References

1. Mora E., Smith E.M.L., Donohoe C., Hertz D.L. Vincristine-induced peripheral neuropathy in pediatric cancer patients. Am. J. Cancer Res. 2016;6:2416–2430.  [PMC free article](/articles/PMC5126263/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27904761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Cancer%20Res.&title=Vincristine-induced%20peripheral%20neuropathy%20in%20pediatric%20cancer%20patients&author=E.%20Mora&author=E.M.L.%20Smith&author=C.%20Donohoe&author=D.L.%20Hertz&volume=6&publication_year=2016&pages=2416-2430&pmid=27904761&)

2. Smith E.M.L., Kuisell C., Cho Y., Kanzawa-Lee G.A., Gilchrist L.S., Park S.B., Scott M.R., Alberti P. Characteristics and patterns of pediatric chemotherapy-induced peripheral neuropathy: A systematic review. Cancer Treat. Res. Commun. 2021;28:100420. doi: 10.1016/j.ctarc.2021.100420.  [DOI](https://doi.org/10.1016/j.ctarc.2021.100420) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34225104/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Treat.%20Res.%20Commun.&title=Characteristics%20and%20patterns%20of%20pediatric%20chemotherapy-induced%20peripheral%20neuropathy:%20A%20systematic%20review&author=E.M.L.%20Smith&author=C.%20Kuisell&author=Y.%20Cho&author=G.A.%20Kanzawa-Lee&author=L.S.%20Gilchrist&volume=28&publication_year=2021&pages=100420&pmid=34225104&doi=10.1016/j.ctarc.2021.100420&)

3. Van de Velde M.E., Kaspers G.L., Abbink F.C.H., Wilhelm A.J., Ket J.C.F., van den Berg M.H. Vincristine-induced peripheral neuropathy in children with cancer: A systematic review. Crit. Rev. Oncol. Hematol. 2017;114:114–130. doi: 10.1016/j.critrevonc.2017.04.004.  [DOI](https://doi.org/10.1016/j.critrevonc.2017.04.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28477739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit.%20Rev.%20Oncol.%20Hematol.&title=Vincristine-induced%20peripheral%20neuropathy%20in%20children%20with%20cancer:%20A%20systematic%20review&author=M.E.%20Van%20de%20Velde&author=G.L.%20Kaspers&author=F.C.H.%20Abbink&author=A.J.%20Wilhelm&author=J.C.F.%20Ket&volume=114&publication_year=2017&pages=114-130&pmid=28477739&doi=10.1016/j.critrevonc.2017.04.004&)

4. Madsen M.L., Due H., Ejskjær N., Jensen P., Madsen J., Dybkær K. Aspects of vincristine-induced neuropathy in hematologic malignancies: A systematic review. Cancer Chemother. Pharmacol. 2019;84:471–485. doi: 10.1007/s00280-019-03884-5.  [DOI](https://doi.org/10.1007/s00280-019-03884-5) | [PMC free article](/articles/PMC6682573/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31214762/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother.%20Pharmacol.&title=Aspects%20of%20vincristine-induced%20neuropathy%20in%20hematologic%20malignancies:%20A%20systematic%20review&author=M.L.%20Madsen&author=H.%20Due&author=N.%20Ejskj%C3%A6r&author=P.%20Jensen&author=J.%20Madsen&volume=84&publication_year=2019&pages=471-485&pmid=31214762&doi=10.1007/s00280-019-03884-5&)

5. Lavoie Smith E.M., Li L., Chiang C., Thomas K., Hutchinson R.J., Wells E.M., Ho R.H., Skiles J., Chakraborty A., Bridges C.M., et al. Patterns and severity of vincristine-induced peripheral neuropathy in children with acute lymphoblastic leukemia. J. Peripher. Nerv. Syst. JPNS. 2015;20:37–46. doi: 10.1111/jns.12114.  [DOI](https://doi.org/10.1111/jns.12114) | [PMC free article](/articles/PMC4610712/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25977177/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Peripher.%20Nerv.%20Syst.%20JPNS&title=Patterns%20and%20severity%20of%20vincristine-induced%20peripheral%20neuropathy%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=E.M.%20Lavoie%20Smith&author=L.%20Li&author=C.%20Chiang&author=K.%20Thomas&author=R.J.%20Hutchinson&volume=20&publication_year=2015&pages=37-46&pmid=25977177&doi=10.1111/jns.12114&)

6. Van de Velde M.E., van den Berg M.H., Kaspers G.J.L., Abbink F.C.H., Twisk J.W.R., van der Sluis I.M., van den Bos C., van den Heuvel-Eibrink M.M., Segers H., Chantrain C., et al. The association between vincristine-induced peripheral neuropathy and health-related quality of life in children with cancer. Cancer Med. 2021;10:8172–8181. doi: 10.1002/cam4.4289.  [DOI](https://doi.org/10.1002/cam4.4289) | [PMC free article](/articles/PMC8607258/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34725942/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Med.&title=The%20association%20between%20vincristine-induced%20peripheral%20neuropathy%20and%20health-related%20quality%20of%20life%20in%20children%20with%20cancer&author=M.E.%20Van%20de%20Velde&author=M.H.%20van%20den%20Berg&author=G.J.L.%20Kaspers&author=F.C.H.%20Abbink&author=J.W.R.%20Twisk&volume=10&publication_year=2021&pages=8172-8181&pmid=34725942&doi=10.1002/cam4.4289&)

7. Tay C.G., Lee V.W.M., Ong L.C., Goh K.J., Ariffin H., Fong C.Y. Vincristine-induced peripheral neuropathy in survivors of childhood acute lymphoblastic leukaemia. Pediatr. Blood Cancer. 2017;64:e26471. doi: 10.1002/pbc.26471.  [DOI](https://doi.org/10.1002/pbc.26471) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28139029/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Blood%20Cancer&title=Vincristine-induced%20peripheral%20neuropathy%20in%20survivors%20of%20childhood%20acute%20lymphoblastic%20leukaemia&author=C.G.%20Tay&author=V.W.M.%20Lee&author=L.C.%20Ong&author=K.J.%20Goh&author=H.%20Ariffin&volume=64&publication_year=2017&pages=e26471&pmid=28139029&doi=10.1002/pbc.26471&)

8. Ceppi F., Langlois-Pelletier C., Gagné V., Rousseau J., Ciolino C., De Lorenzo S., Kevin K.M., Cijov D., Sallan S.E., Silverman L.B., et al. Polymorphisms of the vincristine pathway and response to treatment in children with childhood acute lymphoblastic leukemia. Pharmacogenomics. 2014;15:1105–1116. doi: 10.2217/pgs.14.68.  [DOI](https://doi.org/10.2217/pgs.14.68) | [PMC free article](/articles/PMC4443746/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25084203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Polymorphisms%20of%20the%20vincristine%20pathway%20and%20response%20to%20treatment%20in%20children%20with%20childhood%20acute%20lymphoblastic%20leukemia&author=F.%20Ceppi&author=C.%20Langlois-Pelletier&author=V.%20Gagn%C3%A9&author=J.%20Rousseau&author=C.%20Ciolino&volume=15&publication_year=2014&pages=1105-1116&pmid=25084203&doi=10.2217/pgs.14.68&)

9. Diouf B., Crews K.R., Lew G., Pei D., Cheng C., Bao J., Zheng J.J., Yang W., Fan Y., Wheeler H.E., et al. Association of an inherited genetic variant with vincristine-related peripheral neuropathy in children with acute lymphoblastic leukemia. JAMA. 2015;313:815–823. doi: 10.1001/jama.2015.0894.  [DOI](https://doi.org/10.1001/jama.2015.0894) | [PMC free article](/articles/PMC4377066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25710658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Association%20of%20an%20inherited%20genetic%20variant%20with%20vincristine-related%20peripheral%20neuropathy%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=B.%20Diouf&author=K.R.%20Crews&author=G.%20Lew&author=D.%20Pei&author=C.%20Cheng&volume=313&publication_year=2015&pages=815-823&pmid=25710658&doi=10.1001/jama.2015.0894&)

10. Gutierrez-Camino A., Martin-Guerrero I., Lopez-Lopez E., Echebarria-Barona A., Zabalza I., Ruiz I., Guerra-Merino I., Garcia-Orad A. Lack of association of the CEP72 rs924607 TT genotype with vincristine-related peripheral neuropathy during the early phase of pediatric acute lymphoblastic leukemia treatment in a Spanish population. Pharm. Genom. 2016;26:100–102. doi: 10.1097/FPC.0000000000000191.  [DOI](https://doi.org/10.1097/FPC.0000000000000191) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26618658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Genom.&title=Lack%20of%20association%20of%20the%20CEP72%20rs924607%20TT%20genotype%20with%20vincristine-related%20peripheral%20neuropathy%20during%20the%20early%20phase%20of%20pediatric%20acute%20lymphoblastic%20leukemia%20treatment%20in%20a%20Spanish%20population&author=A.%20Gutierrez-Camino&author=I.%20Martin-Guerrero&author=E.%20Lopez-Lopez&author=A.%20Echebarria-Barona&author=I.%20Zabalza&volume=26&publication_year=2016&pages=100-102&pmid=26618658&doi=10.1097/FPC.0000000000000191&)

11. Lopez-Lopez E., Gutierrez-Camino A., Astigarraga I., Navajas A., Echebarria-Barona A., Garcia-Miguel P., Garcia de Andoin N., Lobo C., Guerra-Merino I., Martin-Guerrero I., et al. Vincristine pharmacokinetics pathway and neurotoxicity during early phases of treatment in pediatric acute lymphoblastic leukemia. Pharmacogenomics. 2016;17:731–741. doi: 10.2217/pgs-2016-0001.  [DOI](https://doi.org/10.2217/pgs-2016-0001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27180762/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Vincristine%20pharmacokinetics%20pathway%20and%20neurotoxicity%20during%20early%20phases%20of%20treatment%20in%20pediatric%20acute%20lymphoblastic%20leukemia&author=E.%20Lopez-Lopez&author=A.%20Gutierrez-Camino&author=I.%20Astigarraga&author=A.%20Navajas&author=A.%20Echebarria-Barona&volume=17&publication_year=2016&pages=731-741&pmid=27180762&doi=10.2217/pgs-2016-0001&)

12. Anghelescu D.L., Faughnan L.G., Jeha S., Relling M.V., Hinds P.S., Sandlund J.T., Cheng C., Pei D., Hankins G., Pauley J.L., et al. Neuropathic pain during treatment for childhood acute lymphoblastic leukemia. Pediatr. Blood Cancer. 2011;57:1147–1153. doi: 10.1002/pbc.23039.  [DOI](https://doi.org/10.1002/pbc.23039) | [PMC free article](/articles/PMC3136575/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21319291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Blood%20Cancer&title=Neuropathic%20pain%20during%20treatment%20for%20childhood%20acute%20lymphoblastic%20leukemia&author=D.L.%20Anghelescu&author=L.G.%20Faughnan&author=S.%20Jeha&author=M.V.%20Relling&author=P.S.%20Hinds&volume=57&publication_year=2011&pages=1147-1153&pmid=21319291&doi=10.1002/pbc.23039&)

13. Kishi S., Cheng C., French D., Pei D., Das S., Cook E.H., Hijiya N., Rizzari C., Rosner G.L., Frudakis T., et al. Ancestry and pharmacogenetics of antileukemic drug toxicity. Blood. 2007;109:4151–4157. doi: 10.1182/blood-2006-10-054528.  [DOI](https://doi.org/10.1182/blood-2006-10-054528) | [PMC free article](/articles/PMC1885506/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17264302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Ancestry%20and%20pharmacogenetics%20of%20antileukemic%20drug%20toxicity&author=S.%20Kishi&author=C.%20Cheng&author=D.%20French&author=D.%20Pei&author=S.%20Das&volume=109&publication_year=2007&pages=4151-4157&pmid=17264302&doi=10.1182/blood-2006-10-054528&)

14. Renbarger J.L., McCammack K.C., Rouse C.E., Hall S.D. Effect of race on vincristine-associated neurotoxicity in pediatric acute lymphoblastic leukemia patients. Pediatr. Blood Cancer. 2008;50:769–771. doi: 10.1002/pbc.21435.  [DOI](https://doi.org/10.1002/pbc.21435) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18085684/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Blood%20Cancer&title=Effect%20of%20race%20on%20vincristine-associated%20neurotoxicity%20in%20pediatric%20acute%20lymphoblastic%20leukemia%20patients&author=J.L.%20Renbarger&author=K.C.%20McCammack&author=C.E.%20Rouse&author=S.D.%20Hall&volume=50&publication_year=2008&pages=769-771&pmid=18085684&doi=10.1002/pbc.21435&)

15. Smitherman A.B., Faircloth C.B., Deal A., Troy M., Gold S.H. Vincristine toxicity with co-administration of fluconazole during induction therapy for pediatric acute lymphoblastic leukemia. Pediatr. Blood Cancer. 2017;64:e26525. doi: 10.1002/pbc.26525.  [DOI](https://doi.org/10.1002/pbc.26525) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28333402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Blood%20Cancer&title=Vincristine%20toxicity%20with%20co-administration%20of%20fluconazole%20during%20induction%20therapy%20for%20pediatric%20acute%20lymphoblastic%20leukemia&author=A.B.%20Smitherman&author=C.B.%20Faircloth&author=A.%20Deal&author=M.%20Troy&author=S.H.%20Gold&volume=64&publication_year=2017&pages=e26525&pmid=28333402&doi=10.1002/pbc.26525&)

16. Skiles J.L., Chiang C., Li C.H., Martin S., Smith E.L., Olbara G., Jones D.R., Vik T.A., Mostert S., Abbink F., et al. CYP3A5 genotype and its impact on vincristine pharmacokinetics and development of neuropathy in Kenyan children with cancer. Pediatr. Blood Cancer. 2018;65:e26854. doi: 10.1002/pbc.26854.  [DOI](https://doi.org/10.1002/pbc.26854) | [PMC free article](/articles/PMC5766375/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29115708/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Blood%20Cancer&title=CYP3A5%20genotype%20and%20its%20impact%20on%20vincristine%20pharmacokinetics%20and%20development%20of%20neuropathy%20in%20Kenyan%20children%20with%20cancer&author=J.L.%20Skiles&author=C.%20Chiang&author=C.H.%20Li&author=S.%20Martin&author=E.L.%20Smith&volume=65&publication_year=2018&pages=e26854&pmid=29115708&doi=10.1002/pbc.26854&)

17. Lavoie Smith E.M., Li L., Hutchinson R.J., Ho R., Burnette W.B., Wells E., Bridges C., Renbarger J. Measuring vincristine-induced peripheral neuropathy in children with acute lymphoblastic leukemia. Cancer Nurs. 2013;36:E49–E60. doi: 10.1097/NCC.0b013e318299ad23.  [DOI](https://doi.org/10.1097/NCC.0b013e318299ad23) | [PMC free article](/articles/PMC3951303/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23842524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Nurs.&title=Measuring%20vincristine-induced%20peripheral%20neuropathy%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=E.M.%20Lavoie%20Smith&author=L.%20Li&author=R.J.%20Hutchinson&author=R.%20Ho&author=W.B.%20Burnette&volume=36&publication_year=2013&pages=E49-E60&pmid=23842524&doi=10.1097/NCC.0b013e318299ad23&)

18. Van de Velde M.E., Panetta J.C., Wilhelm A.J., van den Berg M.H., van der Sluis I.M., van den Bos C., Abbink F.C.H., van den Heuvel-Eibrink M.M., Segers H., Chantrain C., et al. Population pharmacokinetics of vincristine related to infusion duration and peripheral neuropathy in pediatric oncology patients. Cancers. 2020;12:1789. doi: 10.3390/cancers12071789.  [DOI](https://doi.org/10.3390/cancers12071789) | [PMC free article](/articles/PMC7407622/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32635465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers&title=Population%20pharmacokinetics%20of%20vincristine%20related%20to%20infusion%20duration%20and%20peripheral%20neuropathy%20in%20pediatric%20oncology%20patients&author=M.E.%20Van%20de%20Velde&author=J.C.%20Panetta&author=A.J.%20Wilhelm&author=M.H.%20van%20den%20Berg&author=I.M.%20van%20der%20Sluis&volume=12&publication_year=2020&pages=1789&pmid=32635465&doi=10.3390/cancers12071789&)

19. Crom W.R., de Graaf S.S., Synold T., Uges D.R., Bloemhof H., Rivera G., Christensen M.L., Mahmoud H., Evans W.E. Pharmacokinetics of vincristine in children and adolescents with acute lymphocytic leukemia. J. Pediatr. 1994;125:642–649. doi: 10.1016/S0022-3476(94)70027-3.  [DOI](https://doi.org/10.1016/S0022-3476(94)70027-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7931891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pediatr.&title=Pharmacokinetics%20of%20vincristine%20in%20children%20and%20adolescents%20with%20acute%20lymphocytic%20leukemia&author=W.R.%20Crom&author=S.S.%20de%20Graaf&author=T.%20Synold&author=D.R.%20Uges&author=H.%20Bloemhof&volume=125&publication_year=1994&pages=642-649&pmid=7931891&doi=10.1016/S0022-3476(94)70027-3&)

20. Guilhaumou R., Simon N., Quaranta S., Verschuur A., Lacarelle B., Andre N., Solas C. Population pharmacokinetics and pharmacogenetics of vincristine in paediatric patients treated for solid tumour diseases. Cancer Chemother. Pharmacol. 2011;68:1191–1198. doi: 10.1007/s00280-010-1541-4.  [DOI](https://doi.org/10.1007/s00280-010-1541-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21404110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother.%20Pharmacol.&title=Population%20pharmacokinetics%20and%20pharmacogenetics%20of%20vincristine%20in%20paediatric%20patients%20treated%20for%20solid%20tumour%20diseases&author=R.%20Guilhaumou&author=N.%20Simon&author=S.%20Quaranta&author=A.%20Verschuur&author=B.%20Lacarelle&volume=68&publication_year=2011&pages=1191-1198&pmid=21404110&doi=10.1007/s00280-010-1541-4&)

21. Moore A.S., Norris R., Price G., Nguyen T., Ni M., George R., van Breda K., Duley J., Charles B., Pinkerton R. Vincristine pharmacodynamics and pharmacogenetics in children with cancer: A limited-sampling, population modelling approach. J. Paediatr. Child Health. 2011;47:875–882. doi: 10.1111/j.1440-1754.2011.02103.x.  [DOI](https://doi.org/10.1111/j.1440-1754.2011.02103.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21658147/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Paediatr.%20Child%20Health&title=Vincristine%20pharmacodynamics%20and%20pharmacogenetics%20in%20children%20with%20cancer:%20A%20limited-sampling,%20population%20modelling%20approach&author=A.S.%20Moore&author=R.%20Norris&author=G.%20Price&author=T.%20Nguyen&author=M.%20Ni&volume=47&publication_year=2011&pages=875-882&pmid=21658147&doi=10.1111/j.1440-1754.2011.02103.x&)

22. Plasschaert S.L., Groninger E., Boezen M., Kema I., de Vries E.G., Uges D., Veerman A.J., Kamps W.A., Vellenga E., de Graaf S.S., et al. Influence of functional polymorphisms of the MDR1 gene on vincristine pharmacokinetics in childhood acute lymphoblastic leukemia. Clin. Pharm. 2004;76:220–229. doi: 10.1016/j.clpt.2004.05.007.  [DOI](https://doi.org/10.1016/j.clpt.2004.05.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15371983/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharm.&title=Influence%20of%20functional%20polymorphisms%20of%20the%20MDR1%20gene%20on%20vincristine%20pharmacokinetics%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=S.L.%20Plasschaert&author=E.%20Groninger&author=M.%20Boezen&author=I.%20Kema&author=E.G.%20de%20Vries&volume=76&publication_year=2004&pages=220-229&pmid=15371983&doi=10.1016/j.clpt.2004.05.007&)

23. Dennison J.B., Jones D.R., Renbarger J.L., Hall S.D. Effect of CYP3A5 expression on vincristine metabolism with human liver microsomes. J. Pharmacol. Exp. Ther. 2007;321:553–563. doi: 10.1124/jpet.106.118471.  [DOI](https://doi.org/10.1124/jpet.106.118471) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17272675/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=Effect%20of%20CYP3A5%20expression%20on%20vincristine%20metabolism%20with%20human%20liver%20microsomes&author=J.B.%20Dennison&author=D.R.%20Jones&author=J.L.%20Renbarger&author=S.D.%20Hall&volume=321&publication_year=2007&pages=553-563&pmid=17272675&doi=10.1124/jpet.106.118471&)

24. Arbitrio M., Scionti F., Di Martino M.T., Pensabene L., Tassone P., Tagliaferri P. Reference Module in Biomedical Sciences. Elsevier; Amsterdam, The Netherlands: 2021. Pharmacogenetics/pharmacogenomics of drug-metabolizing enzymes and transporters.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Reference%20Module%20in%20Biomedical%20Sciences&author=M.%20Arbitrio&author=F.%20Scionti&author=M.T.%20Di%20Martino&author=L.%20Pensabene&author=P.%20Tassone&publication_year=2021&)

25. Kuehl P., Zhang J., Lin Y., Lamba J., Assem M., Schuetz J., Watkins P.B., Daly A., Wrighton S.A., Hall S.D., et al. Sequence diversity in CYP3A promoters and characterization of the genetic basis of polymorphic CYP3A5 expression. Nat. Genet. 2001;27:383–391. doi: 10.1038/86882.  [DOI](https://doi.org/10.1038/86882) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11279519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Sequence%20diversity%20in%20CYP3A%20promoters%20and%20characterization%20of%20the%20genetic%20basis%20of%20polymorphic%20CYP3A5%20expression&author=P.%20Kuehl&author=J.%20Zhang&author=Y.%20Lin&author=J.%20Lamba&author=M.%20Assem&volume=27&publication_year=2001&pages=383-391&pmid=11279519&doi=10.1038/86882&)

26. Blanco J.G., Edick M.J., Hancock M.L., Winick N.J., Dervieux T., Amylon M.D., Bash R.O., Behm F.G., Camitta B.M., Pui C.-H., et al. Genetic polymorphisms in CYP3A5, CYP3A4 and NQO1 in children who developed therapy-related myeloid malignancies. Pharm. Genom. 2002;12:605–611. doi: 10.1097/00008571-200211000-00004.  [DOI](https://doi.org/10.1097/00008571-200211000-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12439220/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Genom.&title=Genetic%20polymorphisms%20in%20CYP3A5,%20CYP3A4%20and%20NQO1%20in%20children%20who%20developed%20therapy-related%20myeloid%20malignancies&author=J.G.%20Blanco&author=M.J.%20Edick&author=M.L.%20Hancock&author=N.J.%20Winick&author=T.%20Dervieux&volume=12&publication_year=2002&pages=605-611&pmid=12439220&doi=10.1097/00008571-200211000-00004&)

27. Roy J.-N., Lajoie J., Zijenah L.S., Barama A., Poirier C., Ward B.J., Roger M. CYP3A5 genetic polymorphisms in different ethnic populations. Drug Metab. Dispos. 2005;33:884–887. doi: 10.1124/dmd.105.003822.  [DOI](https://doi.org/10.1124/dmd.105.003822) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15833928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=CYP3A5%20genetic%20polymorphisms%20in%20different%20ethnic%20populations&author=J.-N.%20Roy&author=J.%20Lajoie&author=L.S.%20Zijenah&author=A.%20Barama&author=C.%20Poirier&volume=33&publication_year=2005&pages=884-887&pmid=15833928&doi=10.1124/dmd.105.003822&)

28. Aplenc R., Glatfelter W., Han P., Rappaport E., La M., Cnaan A., Blackwood M.A., Lange B., Rebbeck T. CYP3A genotypes and treatment response in paediatric acute lymphoblastic leukaemia. Br. J. Haematol. 2003;122:240–244. doi: 10.1046/j.1365-2141.2003.04430.x.  [DOI](https://doi.org/10.1046/j.1365-2141.2003.04430.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12846892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Haematol.&title=CYP3A%20genotypes%20and%20treatment%20response%20in%20paediatric%20acute%20lymphoblastic%20leukaemia&author=R.%20Aplenc&author=W.%20Glatfelter&author=P.%20Han&author=E.%20Rappaport&author=M.%20La&volume=122&publication_year=2003&pages=240-244&pmid=12846892&doi=10.1046/j.1365-2141.2003.04430.x&)

29. Egbelakin A., Ferguson M.J., MacGill E.A., Lehmann A.S., Topletz A.R., Quinney S.K., Li L., McCammack K.C., Hall S.D., Renbarger J.L. Increased risk of vincristine neurotoxicity associated with low CYP3A5 expression genotype in children with acute lymphoblastic leukemia. Pediatr. Blood Cancer. 2011;56:361–367. doi: 10.1002/pbc.22845.  [DOI](https://doi.org/10.1002/pbc.22845) | [PMC free article](/articles/PMC3020258/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21225912/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Blood%20Cancer&title=Increased%20risk%20of%20vincristine%20neurotoxicity%20associated%20with%20low%20CYP3A5%20expression%20genotype%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=A.%20Egbelakin&author=M.J.%20Ferguson&author=E.A.%20MacGill&author=A.S.%20Lehmann&author=A.R.%20Topletz&volume=56&publication_year=2011&pages=361-367&pmid=21225912&doi=10.1002/pbc.22845&)

30. Kayilioğlu H., Kocak U., Kan Karaer D., Percin E.F., Sal E., Tekkesin F., Isik M., Oner N., Belen F.B., Yilmaz Keskin E., et al. Association of CYP3A5 expression and vincristine neurotoxicity in pediatric malignancies in Turkish population. J. Pediatr. Hematol. Oncol. 2017;39:458–462. doi: 10.1097/MPH.0000000000000910.  [DOI](https://doi.org/10.1097/MPH.0000000000000910) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28697165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pediatr.%20Hematol.%20Oncol.&title=Association%20of%20CYP3A5%20expression%20and%20vincristine%20neurotoxicity%20in%20pediatric%20malignancies%20in%20Turkish%20population&author=H.%20Kayilio%C4%9Flu&author=U.%20Kocak&author=D.%20Kan%20Karaer&author=E.F.%20Percin&author=E.%20Sal&volume=39&publication_year=2017&pages=458-462&pmid=28697165&doi=10.1097/MPH.0000000000000910&)

31. McClain C.A., Bernhardt M.B., Berger A., Bernini J.C., Marquez-Do D., Winslow R., Scheurer M.E., Schafer E.S. Pharmacogenetic association with neurotoxicity in Hispanic children with acute lymphoblastic leukaemia. Br. J. Haematol. 2018;181:684–687. doi: 10.1111/bjh.14678.  [DOI](https://doi.org/10.1111/bjh.14678) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28419449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Haematol.&title=Pharmacogenetic%20association%20with%20neurotoxicity%20in%20Hispanic%20children%20with%20acute%20lymphoblastic%20leukaemia&author=C.A.%20McClain&author=M.B.%20Bernhardt&author=A.%20Berger&author=J.C.%20Bernini&author=D.%20Marquez-Do&volume=181&publication_year=2018&pages=684-687&pmid=28419449&doi=10.1111/bjh.14678&)

32. Sims R.P. The effect of race on the CYP3A-mediated metabolism of vincristine in pediatric patients with acute lymphoblastic leukemia. J. Oncol. Pharm. Pract. 2016;22:76–81. doi: 10.1177/1078155214553143.  [DOI](https://doi.org/10.1177/1078155214553143) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25305360/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Oncol.%20Pharm.%20Pract.&title=The%20effect%20of%20race%20on%20the%20CYP3A-mediated%20metabolism%20of%20vincristine%20in%20pediatric%20patients%20with%20acute%20lymphoblastic%20leukemia&author=R.P.%20Sims&volume=22&publication_year=2016&pages=76-81&pmid=25305360&doi=10.1177/1078155214553143&)

33. Lee J.W., Aminkeng F., Bhavsar A.P., Shaw K., Carleton B.C., Hayden M.R., Ross C.J. The emerging era of pharmacogenomics: Current successes, future potential, and challenges. Clin. Genet. 2014;86:21–28. doi: 10.1111/cge.12392.  [DOI](https://doi.org/10.1111/cge.12392) | [PMC free article](/articles/PMC4233969/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24684508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Genet.&title=The%20emerging%20era%20of%20pharmacogenomics:%20Current%20successes,%20future%20potential,%20and%20challenges&author=J.W.%20Lee&author=F.%20Aminkeng&author=A.P.%20Bhavsar&author=K.%20Shaw&author=B.C.%20Carleton&volume=86&publication_year=2014&pages=21-28&pmid=24684508&doi=10.1111/cge.12392&)

34. Weinshilboum R.M., Wang L. Pharmacogenomics: Precision medicine and drug response. Mayo Clin. Proc. 2017;92:1711–1722. doi: 10.1016/j.mayocp.2017.09.001.  [DOI](https://doi.org/10.1016/j.mayocp.2017.09.001) | [PMC free article](/articles/PMC5682947/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29101939/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mayo%20Clin.%20Proc.&title=Pharmacogenomics:%20Precision%20medicine%20and%20drug%20response&author=R.M.%20Weinshilboum&author=L.%20Wang&volume=92&publication_year=2017&pages=1711-1722&pmid=29101939&doi=10.1016/j.mayocp.2017.09.001&)

35. Alwi Z.B. The Use of SNPs in pharmacogenomics studies. Malays. J. Med. Sci. 2005;12:4–12.  [PMC free article](/articles/PMC3349395/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22605952/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Malays.%20J.%20Med.%20Sci.&title=The%20Use%20of%20SNPs%20in%20pharmacogenomics%20studies&author=Z.B.%20Alwi&volume=12&publication_year=2005&pages=4-12&pmid=22605952&)

36. Wang J., Pang G.S., Chong S.S., Lee C.G. SNP web resources and their potential applications in personalized medicine. Curr. Drug Metab. 2012;13:978–990. doi: 10.2174/138920012802138552.  [DOI](https://doi.org/10.2174/138920012802138552) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22591348/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Drug%20Metab.&title=SNP%20web%20resources%20and%20their%20potential%20applications%20in%20personalized%20medicine&author=J.%20Wang&author=G.S.%20Pang&author=S.S.%20Chong&author=C.G.%20Lee&volume=13&publication_year=2012&pages=978-990&pmid=22591348&doi=10.2174/138920012802138552&)

37. Auton A., Abecasis G.R., Altshuler D.M., Durbin R.M., Abecasis G.R., Bentley D.R., Chakravarti A., Clark A.G., Donnelly P., Eichler E.E., et al. A global reference for human genetic variation. Nature. 2015;526:68–74. doi: 10.1038/nature15393.  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation&author=A.%20Auton&author=G.R.%20Abecasis&author=D.M.%20Altshuler&author=R.M.%20Durbin&author=G.R.%20Abecasis&volume=526&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

38. Karczewski K.J., Francioli L.C., Tiao G., Cummings B.B., Alföldi J., Wang Q., Collins R.L., Laricchia K.M., Ganna A., Birnbaum D.P., et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature. 2020;581:434–443. doi: 10.1038/s41586-020-2308-7.  [DOI](https://doi.org/10.1038/s41586-020-2308-7) | [PMC free article](/articles/PMC7334197/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32461654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141,456%20humans&author=K.J.%20Karczewski&author=L.C.%20Francioli&author=G.%20Tiao&author=B.B.%20Cummings&author=J.%20Alf%C3%B6ldi&volume=581&publication_year=2020&pages=434-443&pmid=32461654&doi=10.1038/s41586-020-2308-7&)

39. Clarke G.M., Anderson C.A., Pettersson F.H., Cardon L.R., Morris A.P., Zondervan K.T. Basic statistical analysis in genetic case-control studies. Nat. Protoc. 2011;6:121–133. doi: 10.1038/nprot.2010.182.  [DOI](https://doi.org/10.1038/nprot.2010.182) | [PMC free article](/articles/PMC3154648/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21293453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&title=Basic%20statistical%20analysis%20in%20genetic%20case-control%20studies&author=G.M.%20Clarke&author=C.A.%20Anderson&author=F.H.%20Pettersson&author=L.R.%20Cardon&author=A.P.%20Morris&volume=6&publication_year=2011&pages=121-133&pmid=21293453&doi=10.1038/nprot.2010.182&)

40. Zondervan K.T., Cardon L.R. Designing candidate gene and genome-wide case-control association studies. Nat. Protoc. 2007;2:2492–2501. doi: 10.1038/nprot.2007.366.  [DOI](https://doi.org/10.1038/nprot.2007.366) | [PMC free article](/articles/PMC4180089/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17947991/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&title=Designing%20candidate%20gene%20and%20genome-wide%20case-control%20association%20studies&author=K.T.%20Zondervan&author=L.R.%20Cardon&volume=2&publication_year=2007&pages=2492-2501&pmid=17947991&doi=10.1038/nprot.2007.366&)

41. Lennard L., Cartwright C.S., Wade R., Vora A. Thiopurine methyltransferase and treatment outcome in the UK acute lymphoblastic leukaemia trial ALL2003. Br. J. Haematol. 2015;170:550–558. doi: 10.1111/bjh.13469.  [DOI](https://doi.org/10.1111/bjh.13469) | [PMC free article](/articles/PMC4687427/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25940902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Haematol.&title=Thiopurine%20methyltransferase%20and%20treatment%20outcome%20in%20the%20UK%20acute%20lymphoblastic%20leukaemia%20trial%20ALL2003&author=L.%20Lennard&author=C.S.%20Cartwright&author=R.%20Wade&author=A.%20Vora&volume=170&publication_year=2015&pages=550-558&pmid=25940902&doi=10.1111/bjh.13469&)

42. Stocco G., Cheok M.H., Crews K.R., Dervieux T., French D., Pei D., Yang W., Cheng C., Pui C.H., Relling M.V., et al. Genetic polymorphism of inosine triphosphate pyrophosphatase is a determinant of mercaptopurine metabolism and toxicity during treatment for acute lymphoblastic leukemia. Clin. Pharm. 2009;85:164–172. doi: 10.1038/clpt.2008.154.  [DOI](https://doi.org/10.1038/clpt.2008.154) | [PMC free article](/articles/PMC2762405/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18685564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharm.&title=Genetic%20polymorphism%20of%20inosine%20triphosphate%20pyrophosphatase%20is%20a%20determinant%20of%20mercaptopurine%20metabolism%20and%20toxicity%20during%20treatment%20for%20acute%20lymphoblastic%20leukemia&author=G.%20Stocco&author=M.H.%20Cheok&author=K.R.%20Crews&author=T.%20Dervieux&author=D.%20French&volume=85&publication_year=2009&pages=164-172&pmid=18685564&doi=10.1038/clpt.2008.154&)

43. Liberati A., Altman D.G., Tetzlaff J., Mulrow C., Gøtzsche P.C., Ioannidis J.P.A., Clarke M., Devereaux P.J., Kleijnen J., Moher D. The PRISMA statement for reporting systematic reviews and meta-analyses of studies that evaluate healthcare interventions: Explanation and elaboration. BMJ. 2009;339:b2700. doi: 10.1136/bmj.b2700.  [DOI](https://doi.org/10.1136/bmj.b2700) | [PMC free article](/articles/PMC2714672/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19622552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=The%20PRISMA%20statement%20for%20reporting%20systematic%20reviews%20and%20meta-analyses%20of%20studies%20that%20evaluate%20healthcare%20interventions:%20Explanation%20and%20elaboration&author=A.%20Liberati&author=D.G.%20Altman&author=J.%20Tetzlaff&author=C.%20Mulrow&author=P.C.%20G%C3%B8tzsche&volume=339&publication_year=2009&pages=b2700&pmid=19622552&doi=10.1136/bmj.b2700&)

44. Uittenboogaard A., Van de Velde M.E., Kaspers G.J. Pharmacogenomics of Vincristine-Induced Peripheral Neuropathy: A Systematic Review (CRD42021210437) [(accessed on 30 September 2021)]. Available online: https://www.crd.york.ac.uk/prospero/display_record.php?RecordID=210437.  [https://www.crd.york.ac.uk/prospero/display_record.php?RecordID=210437](https://www.crd.york.ac.uk/prospero/display_record.php?RecordID=210437)

45. Armijo-Olivo S., Stiles C.R., Hagen N.A., Biondo P.D., Cummings G.G. Assessment of study quality for systematic reviews: A comparison of the cochrane collaboration risk of bias tool and the effective public health practice project quality assessment tool: Methodological research. J. Eval. Clin. Pract. 2012;18:12–18. doi: 10.1111/j.1365-2753.2010.01516.x.  [DOI](https://doi.org/10.1111/j.1365-2753.2010.01516.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20698919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Eval.%20Clin.%20Pract.&title=Assessment%20of%20study%20quality%20for%20systematic%20reviews:%20A%20comparison%20of%20the%20cochrane%20collaboration%20risk%20of%20bias%20tool%20and%20the%20effective%20public%20health%20practice%20project%20quality%20assessment%20tool:%20Methodological%20research&author=S.%20Armijo-Olivo&author=C.R.%20Stiles&author=N.A.%20Hagen&author=P.D.%20Biondo&author=G.G.%20Cummings&volume=18&publication_year=2012&pages=12-18&pmid=20698919&doi=10.1111/j.1365-2753.2010.01516.x&)

46. Harrer M., Cuijpers P., Furukawa T.A., Ebert D.D. Doing Meta-Analysis with R: A Hands-On Guide. 1st ed. Chapman & Hall/CRC Press; Boca Raton, FL, USA: London, UK: 2021.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Doing%20Meta-Analysis%20with%20R:%20A%20Hands-On%20Guide&author=M.%20Harrer&author=P.%20Cuijpers&author=T.A.%20Furukawa&author=D.D.%20Ebert&publication_year=2021&)

47. Abo-Bakr A., Mossallam G., El Azhary N., Hafez H., Badawy R. Impact of CYP1A1, GSTP1 and XRCC1 genes polymorphisms on toxicity and response to chemotherapy in childhood acute lymphoblastic leukemia. J. Egypt. Natl. Cancer Inst. 2017;29:127–133. doi: 10.1016/j.jnci.2017.07.002.  [DOI](https://doi.org/10.1016/j.jnci.2017.07.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28844589/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Egypt.%20Natl.%20Cancer%20Inst.&title=Impact%20of%20CYP1A1,%20GSTP1%20and%20XRCC1%20genes%20polymorphisms%20on%20toxicity%20and%20response%20to%20chemotherapy%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=A.%20Abo-Bakr&author=G.%20Mossallam&author=N.%20El%20Azhary&author=H.%20Hafez&author=R.%20Badawy&volume=29&publication_year=2017&pages=127-133&pmid=28844589&doi=10.1016/j.jnci.2017.07.002&)

48. Gutierrez-Camino Ã., Umerez M., Martin-Guerrero I., García de Andoin N., Santos B., Sastre A., Echebarria-Barona A., Astigarraga I., Navajas A., Garcia-Orad A. Mir-pharmacogenetics of Vincristine and peripheral neurotoxicity in childhood B-cell acute lymphoblastic leukemia. Pharm. J. 2017;18:704–712. doi: 10.1038/s41397-017-0003-3.  [DOI](https://doi.org/10.1038/s41397-017-0003-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29282364/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20J.&title=Mir-pharmacogenetics%20of%20Vincristine%20and%20peripheral%20neurotoxicity%20in%20childhood%20B-cell%20acute%20lymphoblastic%20leukemia&author=%C3%83.%20Gutierrez-Camino&author=M.%20Umerez&author=I.%20Martin-Guerrero&author=N.%20Garc%C3%ADa%20de%20Andoin&author=B.%20Santos&volume=18&publication_year=2017&pages=704-712&pmid=29282364&doi=10.1038/s41397-017-0003-3&)

49. Martin-Guerrero I., Gutierrez-Camino A., Echebarria-Barona A., Astigarraga I., Garcia de Andoin N., Navajas A., Garcia-Orad A. Variants in vincristine pharmacodynamic genes involved in neurotoxicity at induction phase in the therapy of pediatric acute lymphoblastic leukemia. Pharm. J. 2019;19:564–569. doi: 10.1038/s41397-019-0081-5.  [DOI](https://doi.org/10.1038/s41397-019-0081-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30723315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20J.&title=Variants%20in%20vincristine%20pharmacodynamic%20genes%20involved%20in%20neurotoxicity%20at%20induction%20phase%20in%20the%20therapy%20of%20pediatric%20acute%20lymphoblastic%20leukemia&author=I.%20Martin-Guerrero&author=A.%20Gutierrez-Camino&author=A.%20Echebarria-Barona&author=I.%20Astigarraga&author=N.%20Garcia%20de%20Andoin&volume=19&publication_year=2019&pages=564-569&pmid=30723315&doi=10.1038/s41397-019-0081-5&)

50. Zgheib N.K., Ghanem K.M., Tamim H., Aridi C., Shahine R., Tarek N., Saab R., Abboud M.R., El-Solh H., Muwakkit S.A. Genetic polymorphisms in candidate genes are not associated with increased vincristine-related peripheral neuropathy in Arab children treated for acute childhood leukemia: A single institution study. Pharm. Genom. 2018;28:189–195. doi: 10.1097/FPC.0000000000000345.  [DOI](https://doi.org/10.1097/FPC.0000000000000345) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30119132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Genom.&title=Genetic%20polymorphisms%20in%20candidate%20genes%20are%20not%20associated%20with%20increased%20vincristine-related%20peripheral%20neuropathy%20in%20Arab%20children%20treated%20for%20acute%20childhood%20leukemia:%20A%20single%20institution%20study&author=N.K.%20Zgheib&author=K.M.%20Ghanem&author=H.%20Tamim&author=C.%20Aridi&author=R.%20Shahine&volume=28&publication_year=2018&pages=189-195&pmid=30119132&doi=10.1097/FPC.0000000000000345&)

51. Wright G.E.B., Amstutz U., Drögemöller B.I., Shih J., Rassekh S.R., Hayden M.R., Carleton B.C., Ross C.J.D. Pharmacogenomics of vincristine-induced peripheral neuropathy implicates pharmacokinetic and inherited neuropathy genes. Clin. Pharmacol. Ther. 2019;105:402–410. doi: 10.1002/cpt.1179.  [DOI](https://doi.org/10.1002/cpt.1179) | [PMC free article](/articles/PMC6519044/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29999516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Pharmacogenomics%20of%20vincristine-induced%20peripheral%20neuropathy%20implicates%20pharmacokinetic%20and%20inherited%20neuropathy%20genes&author=G.E.B.%20Wright&author=U.%20Amstutz&author=B.I.%20Dr%C3%B6gem%C3%B6ller&author=J.%20Shih&author=S.R.%20Rassekh&volume=105&publication_year=2019&pages=402-410&pmid=29999516&doi=10.1002/cpt.1179&)

52. Abaji R., Ceppi F., Patel S., Gagné V., Xu C.J., Spinella J.F., Colombini A., Parasole R., Buldini B., Basso G., et al. Genetic risk factors for VIPN in childhood acute lymphoblastic leukemia patients identified using whole-exome sequencing. Pharmacogenomics. 2018;19:1181–1193. doi: 10.2217/pgs-2018-0093.  [DOI](https://doi.org/10.2217/pgs-2018-0093) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30191766/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genetic%20risk%20factors%20for%20VIPN%20in%20childhood%20acute%20lymphoblastic%20leukemia%20patients%20identified%20using%20whole-exome%20sequencing&author=R.%20Abaji&author=F.%20Ceppi&author=S.%20Patel&author=V.%20Gagn%C3%A9&author=C.J.%20Xu&volume=19&publication_year=2018&pages=1181-1193&pmid=30191766&doi=10.2217/pgs-2018-0093&)

53. Li L., Sajdyk T., Smith E.M.L., Chang C.W., Li C., Ho R.H., Hutchinson R., Wells E., Skiles J.L., Winick N., et al. Genetic variants associated with vincristine-induced peripheral neuropathy in two populations of children with acute lymphoblastic leukemia. Clin. Pharm. 2019;105:1421–1428. doi: 10.1002/cpt.1324.  [DOI](https://doi.org/10.1002/cpt.1324) | [PMC free article](/articles/PMC6513686/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30506673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharm.&title=Genetic%20variants%20associated%20with%20vincristine-induced%20peripheral%20neuropathy%20in%20two%20populations%20of%20children%20with%20acute%20lymphoblastic%20leukemia&author=L.%20Li&author=T.%20Sajdyk&author=E.M.L.%20Smith&author=C.W.%20Chang&author=C.%20Li&volume=105&publication_year=2019&pages=1421-1428&pmid=30506673&doi=10.1002/cpt.1324&)

54. Dean M., Rzhetsky A., Allikmets R. The human ATP-binding cassette (ABC) transporter superfamily. Genome Res. 2001;11:1156–1166. doi: 10.1101/gr.184901.  [DOI](https://doi.org/10.1101/gr.184901) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11435397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=The%20human%20ATP-binding%20cassette%20(ABC)%20transporter%20superfamily&author=M.%20Dean&author=A.%20Rzhetsky&author=R.%20Allikmets&volume=11&publication_year=2001&pages=1156-1166&pmid=11435397&doi=10.1101/gr.184901&)

55. Hodges L.M., Markova S.M., Chinn L.W., Gow J.M., Kroetz D.L., Klein T.E., Altman R.B. Very important pharmacogene summary: ABCB1 (MDR1, P-glycoprotein) Pharm. Genom. 2011;21:152–161. doi: 10.1097/FPC.0b013e3283385a1c.  [DOI](https://doi.org/10.1097/FPC.0b013e3283385a1c) | [PMC free article](/articles/PMC3098758/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20216335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm.%20Genom.&title=Very%20important%20pharmacogene%20summary:%20ABCB1%20(MDR1,%20P-glycoprotein)&author=L.M.%20Hodges&author=S.M.%20Markova&author=L.W.%20Chinn&author=J.M.%20Gow&author=D.L.%20Kroetz&volume=21&publication_year=2011&pages=152-161&pmid=20216335&doi=10.1097/FPC.0b013e3283385a1c&)

56. Cui Y., König J., Buchholz J.K., Spring H., Leier I., Keppler D. Drug resistance and ATP-dependent conjugate transport mediated by the apical multidrug resistance protein, MRP2, permanently expressed in human and canine cells. Mol. Pharmacol. 1999;55:929–937.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10220572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=Drug%20resistance%20and%20ATP-dependent%20conjugate%20transport%20mediated%20by%20the%20apical%20multidrug%20resistance%20protein,%20MRP2,%20permanently%20expressed%20in%20human%20and%20canine%20cells&author=Y.%20Cui&author=J.%20K%C3%B6nig&author=J.K.%20Buchholz&author=H.%20Spring&author=I.%20Leier&volume=55&publication_year=1999&pages=929-937&pmid=10220572&)

57. Folmer Y., Schneider M., Blum H.E., Hafkemeyer P. Reversal of drug resistance of hepatocellular carcinoma cells by adenoviral delivery of anti-ABCC2 antisense constructs. Cancer Gene Ther. 2007;14:875–884. doi: 10.1038/sj.cgt.7701082.  [DOI](https://doi.org/10.1038/sj.cgt.7701082) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17704753/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Gene%20Ther.&title=Reversal%20of%20drug%20resistance%20of%20hepatocellular%20carcinoma%20cells%20by%20adenoviral%20delivery%20of%20anti-ABCC2%20antisense%20constructs&author=Y.%20Folmer&author=M.%20Schneider&author=H.E.%20Blum&author=P.%20Hafkemeyer&volume=14&publication_year=2007&pages=875-884&pmid=17704753&doi=10.1038/sj.cgt.7701082&)

58. Liu Z., Qiu M., Tang Q.-L., Liu M., Lang N., Bi F. Establishment and biological characteristics of oxaliplatin-resistant human colon cancer cell lines. Chin. J. Cancer. 2010;29:661–667. doi: 10.5732/cjc.009.10666.  [DOI](https://doi.org/10.5732/cjc.009.10666) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20591218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chin.%20J.%20Cancer&title=Establishment%20and%20biological%20characteristics%20of%20oxaliplatin-resistant%20human%20colon%20cancer%20cell%20lines&author=Z.%20Liu&author=M.%20Qiu&author=Q.-L.%20Tang&author=M.%20Liu&author=N.%20Lang&volume=29&publication_year=2010&pages=661-667&pmid=20591218&doi=10.5732/cjc.009.10666&)

59. Below J., Das M.J. StatPearls. StatPearls Publishing LLC.; Treasure Island, FL, USA: 2021. Vincristine.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=StatPearls&author=J.%20Below&author=M.J.%20Das&publication_year=2021&)

60. Lancaster O.M., Baum B. Shaping up to divide: Coordinating actin and microtubule cytoskeletal remodelling during mitosis. Semin. Cell Dev. Biol. 2014;34:109–115. doi: 10.1016/j.semcdb.2014.02.015.  [DOI](https://doi.org/10.1016/j.semcdb.2014.02.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24607328/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin.%20Cell%20Dev.%20Biol.&title=Shaping%20up%20to%20divide:%20Coordinating%20actin%20and%20microtubule%20cytoskeletal%20remodelling%20during%20mitosis&author=O.M.%20Lancaster&author=B.%20Baum&volume=34&publication_year=2014&pages=109-115&pmid=24607328&doi=10.1016/j.semcdb.2014.02.015&)

61. Kunda P., Baum B. The actin cytoskeleton in spindle assembly and positioning. Trends Cell Biol. 2009;19:174–179. doi: 10.1016/j.tcb.2009.01.006.  [DOI](https://doi.org/10.1016/j.tcb.2009.01.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19285869/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Cell%20Biol.&title=The%20actin%20cytoskeleton%20in%20spindle%20assembly%20and%20positioning&author=P.%20Kunda&author=B.%20Baum&volume=19&publication_year=2009&pages=174-179&pmid=19285869&doi=10.1016/j.tcb.2009.01.006&)

62. Wang Y., Stear J.H., Swain A., Xu X., Bryce N.S., Carnell M., Alieva I.B., Dugina V.B., Cripe T.P., Stehn J., et al. Drug targeting the actin cytoskeleton potentiates the cytotoxicity of low dose vincristine by abrogating actin-mediated repair of spindle defects. Mol. Cancer Res. 2020;18:1074–1087. doi: 10.1158/1541-7786.MCR-19-1122.  [DOI](https://doi.org/10.1158/1541-7786.MCR-19-1122) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32269073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Cancer%20Res.&title=Drug%20targeting%20the%20actin%20cytoskeleton%20potentiates%20the%20cytotoxicity%20of%20low%20dose%20vincristine%20by%20abrogating%20actin-mediated%20repair%20of%20spindle%20defects&author=Y.%20Wang&author=J.H.%20Stear&author=A.%20Swain&author=X.%20Xu&author=N.S.%20Bryce&volume=18&publication_year=2020&pages=1074-1087&pmid=32269073&doi=10.1158/1541-7786.MCR-19-1122&)

63. Meraldi P. Centrosomes in spindle organization and chromosome segregation: A mechanistic view. Chromosome Res. 2016;24:19–34. doi: 10.1007/s10577-015-9508-2.  [DOI](https://doi.org/10.1007/s10577-015-9508-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26643311/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chromosome%20Res.&title=Centrosomes%20in%20spindle%20organization%20and%20chromosome%20segregation:%20A%20mechanistic%20view&author=P.%20Meraldi&volume=24&publication_year=2016&pages=19-34&pmid=26643311&doi=10.1007/s10577-015-9508-2&)

64. Oshimori N., Li X., Ohsugi M., Yamamoto T. Cep72 regulates the localization of key centrosomal proteins and proper bipolar spindle formation. EMBO J. 2009;28:2066–2076. doi: 10.1038/emboj.2009.161.  [DOI](https://doi.org/10.1038/emboj.2009.161) | [PMC free article](/articles/PMC2718283/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19536135/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J.&title=Cep72%20regulates%20the%20localization%20of%20key%20centrosomal%20proteins%20and%20proper%20bipolar%20spindle%20formation&author=N.%20Oshimori&author=X.%20Li&author=M.%20Ohsugi&author=T.%20Yamamoto&volume=28&publication_year=2009&pages=2066-2076&pmid=19536135&doi=10.1038/emboj.2009.161&)

65. Zečkanović A., Jazbec J., Kavčič M. Centrosomal protein72 rs924607 and vincristine-induced neuropathy in pediatric acute lymphocytic leukemia: Meta-analysis. Future Sci. OA. 2020;6:FSO582. doi: 10.2144/fsoa-2020-0044.  [DOI](https://doi.org/10.2144/fsoa-2020-0044) | [PMC free article](/articles/PMC7421539/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32802391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Future%20Sci.%20OA&title=Centrosomal%20protein72%20rs924607%20and%20vincristine-induced%20neuropathy%20in%20pediatric%20acute%20lymphocytic%20leukemia:%20Meta-analysis&author=A.%20Ze%C4%8Dkanovi%C4%87&author=J.%20Jazbec&author=M.%20Kav%C4%8Di%C4%8D&volume=6&publication_year=2020&pages=FSO582&pmid=32802391&doi=10.2144/fsoa-2020-0044&)

66. Total Therapy XVII for Newly Diagnosed Patients with Acute Lymphoblastic Leukemia and Lymphoma. ClinicalTrials.gov Identifier: NCT03117751. [(accessed on 30 September 2021)]; Available online: https://clinicaltrials.gov/ct2/show/NCT03117751.  [NCT03117751](https://clinicaltrials.gov/ct2/show/NCT03117751) | [https://clinicaltrials.gov/ct2/show/NCT03117751](https://clinicaltrials.gov/ct2/show/NCT03117751)

67. Garsa A.A., McLeod H.L., Marsh S. CYP3A4 and CYP3A5genotyping by Pyrosequencing. BMC Med Genet. 2005;6:19. doi: 10.1186/1471-2350-6-19.  [DOI](https://doi.org/10.1186/1471-2350-6-19) | [PMC free article](/articles/PMC1142317/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15882469/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med%20Genet.&title=CYP3A4%20and%20CYP3A5genotyping%20by%20Pyrosequencing&author=A.A.%20Garsa&author=H.L.%20McLeod&author=S.%20Marsh&volume=6&publication_year=2005&pages=19&pmid=15882469&doi=10.1186/1471-2350-6-19&)

68. Lamba J.K., Lin Y.S., Schuetz E.G., Thummel K.E. Genetic contribution to variable human CYP3A-mediated metabolism. Adv. Drug Deliv. Rev. 2002;54:1271–1294. doi: 10.1016/S0169-409X(02)00066-2.  [DOI](https://doi.org/10.1016/S0169-409X(02)00066-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12406645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv.%20Drug%20Deliv.%20Rev.&title=Genetic%20contribution%20to%20variable%20human%20CYP3A-mediated%20metabolism&author=J.K.%20Lamba&author=Y.S.%20Lin&author=E.G.%20Schuetz&author=K.E.%20Thummel&volume=54&publication_year=2002&pages=1271-1294&pmid=12406645&doi=10.1016/S0169-409X(02)00066-2&)

69. Chen S.-Y., Feng Z., Yi X. A general introduction to adjustment for multiple comparisons. J. Thorac. Dis. 2017;9:1725–1729. doi: 10.21037/jtd.2017.05.34.  [DOI](https://doi.org/10.21037/jtd.2017.05.34) | [PMC free article](/articles/PMC5506159/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28740688/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Dis&title=A%20general%20introduction%20to%20adjustment%20for%20multiple%20comparisons&author=S.-Y.%20Chen&author=Z.%20Feng&author=X.%20Yi&volume=9&publication_year=2017&pages=1725-1729&pmid=28740688&doi=10.21037/jtd.2017.05.34&)

70. Menyhart O., Weltz B., Győrffy B. MultipleTesting.com: A tool for life science researchers for multiple hypothesis testing correction. PLoS ONE. 2021;16:e0245824. doi: 10.1371/journal.pone.0245824.  [DOI](https://doi.org/10.1371/journal.pone.0245824) | [PMC free article](/articles/PMC8189492/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34106935/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=MultipleTesting.com:%20A%20tool%20for%20life%20science%20researchers%20for%20multiple%20hypothesis%20testing%20correction&author=O.%20Menyhart&author=B.%20Weltz&author=B.%20Gy%C5%91rffy&volume=16&publication_year=2021&pages=e0245824&pmid=34106935&doi=10.1371/journal.pone.0245824&)

71. Relling M.V., Klein T.E. CPIC: Clinical pharmacogenetics implementation consortium of the pharmacogenomics research network. Clin. Pharmacol. Ther. 2011;89:464–467. doi: 10.1038/clpt.2010.279.  [DOI](https://doi.org/10.1038/clpt.2010.279) | [PMC free article](/articles/PMC3098762/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21270786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=CPIC:%20Clinical%20pharmacogenetics%20implementation%20consortium%20of%20the%20pharmacogenomics%20research%20network&author=M.V.%20Relling&author=T.E.%20Klein&volume=89&publication_year=2011&pages=464-467&pmid=21270786&doi=10.1038/clpt.2010.279&)
