# Interaction Between SNP Genotype and Efficacy of Anastrozole and Exemestane in Early‐Stage Breast Cancer

## Metadata
**Authors:** Junmei Cairns, Krishna R Kalari, James N Ingle, Lois E Shepherd, Matthew J Ellis, Paul E Goss, Poulami Barman, Erin E Carlson, Barbara Goodnature, Matthew P Goetz, Richard M Weinshilboum, Huanyao Gao, Liewei Wang
**Journal:** Clinical Pharmacology and Therapeutics
**Date:** 2021 Jun 21
**DOI:** [10.1002/cpt.2311](https://doi.org/10.1002/cpt.2311)
**PMID:** 34048027
**PMCID:** PMC8449801
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8449801/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8449801/pdf/CPT-110-1038.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8449801/pdf/CPT-110-1038.pdf)

## Abstract

Aromatase inhibitors (AIs) are the treatment of choice for hormone receptor–positive early breast cancer in postmenopausal women. None of the third‐generation AIs are superior to the others in terms of efficacy. We attempted to identify genetic factors that could differentiate between the effectiveness of adjuvant anastrozole and exemestane by examining single‐nucleotide polymorphism (SNP)‐treatment interaction in 4,465 patients. A group of SNPs were found to be differentially associated between anastrozole and exemestane regarding outcomes. However, they showed no association with outcome in the combined analysis. We followed up common SNPs near LY75 and GPR160 that could differentiate anastrozole from exemestane efficacy. LY75 and GPR160 participate in epithelial‐to‐mesenchymal transition and growth pathways, in both cases with SNP‐dependent variation in regulation. Collectively, these studies identified SNPs that differentiate the efficacy of anastrozole and exemestane and they suggest additional genetic biomarkers for possible use in selecting an AI for a given patient.

☑ None of the third‐generation aromatase inhibitors (AIs) are superior to the others in terms of efficacy. As a result, there is currently no basis upon which to choose one AI over the others as adjuvant therapy for early‐stage breast cancer.

☑ To identify genetic factors that could differentiate between the effectiveness of adjuvant anastrozole and exemestane.

☑ We have obtained evidence that anastrozole and exemestane have differing efficacy depending on genetic background.

☑ We identified a single‐nucleotide polymorphism biomarker panel for future individualized therapy of selected patients who might benefit from anastrozole vs. exemestane.

Breast cancer is the most frequent cancer diagnosed among women, impacting more than two million women each year worldwide and causing over 600,000 deaths in 2018.[1](#cpt2311-bib-0001) The majority of those cases were in postmenopausal women with early‐stage estrogen receptor α (ERα)‐positive tumors for which adjuvant endocrine treatment with aromatase inhibitors (AIs), inhibitors of estrogen synthesis,[2](#cpt2311-bib-0002) is the first‐line treatment. Aromatase is the enzyme responsible for the conversion of testosterone to estrone (E1) and of androstenedione to estradiol (E2). Inhibitors of aromatase were developed to block the synthesis of estrogen in peripheral tissues and, thus, as antiestrogen therapy for the treatment of ERα positive breast cancer in postmenopausal women.[3](#cpt2311-bib-0003) The third‐generation AIs that are widely used in the clinic include anastrozole and letrozole, two nonsteroidal compounds, and exemestane, a steroidal aromatase inhibitor. All three of the third‐generation AIs are superior to tamoxifen in the adjuvant setting for the treatment of hormone‐dependent early breast cancer in postmenopausal women.[4](#cpt2311-bib-0004), [5](#cpt2311-bib-0005), [6](#cpt2311-bib-0006), [7](#cpt2311-bib-0007) Pharmacodynamic studies have shown that letrozole more effectively suppresses plasma estradiol levels than does anastrozole in patients with advanced breast cancer,[8](#cpt2311-bib-0008) but the Femara Versus Anastrozole Clinical Evaluation (FACE) trial in 4,136 patients comparing the efficacy of adjuvant anastrozole vs. letrozole head‐to‐head showed no statistically significant difference in disease‐free survival or overall survival between the treatment arms.[9](#cpt2311-bib-0009) Another head‐to‐head comparison of anastrozole and exemestane in the MA.27 trial in 7,576 patients also indicated that treatment with exemestane for 5 years was not better than anastrozole for 5 years.[10](#cpt2311-bib-0010) The more recent FATA‐GIM3 trial directly compared anastrozole, exemestane, and letrozole for the first time as adjuvant treatment for hormone receptor‐positive early breast cancer, and found no significant difference in disease‐free survival 5 years after the initiation of treatments among the 3 groups.[11](#cpt2311-bib-0011) As a result, there is currently no basis upon which to choose one AI over the others as adjuvant therapy for early‐stage breast cancer.

The “dogma” that the three AIs are essentially the same has been revisited based on our recent data showing that anastrozole, but not exemestane or letrozole, displays an association between the degree of estrogen suppression and outcomes in early breast cancer, and that anastrozole has a mechanism of action in addition to aromatase inhibition in that it is a ligand for ERα.[12](#cpt2311-bib-0012), [13](#cpt2311-bib-0013) These findings have potential implications for the individualization of AIs therapy.

Genomewide association studies (GWAS) have identified several genomic regions that affect risks for breast cancer recurrence in women treated with AIs.[13](#cpt2311-bib-0013), [14](#cpt2311-bib-0014), [15](#cpt2311-bib-0015) However, prior studies did not systematically test whether there might be an interaction between individual AI treatment and genetic variability. In the present study, we have investigated germline genetic variation as a predictive marker for differential efficacy of the nonsteroidal AI anastrozole and the steroidal exemestane and as a prognostic marker in the adjuvant Canadian Cancer Trials Group (CCTG) MA.27 trial[10](#cpt2311-bib-0010) in patients with hormone receptor‐positive early breast cancer. We found that a group of single‐nucleotide polymorphisms (SNPs) were differentially associated between anastrozole and exemestane regarding outcomes.

## METHODS

### Source of patients, study design, and study population

The design of the open‐label, randomized phase III trial CCTG MA.27 (ClinicalTrials.gov identifier: [NCT00066573](https://clinicaltrials.gov/ct2/show/NCT00066573)), including eligibility criteria, randomization, treatment plan, follow‐up, and monitoring and primary statistical analyses have been described elsewhere.[10](#cpt2311-bib-0010) In brief, 7,576 postmenopausal patients with hormone receptor‐positive breast cancer were randomized to anastrozole (1 mg/day) or exemestane (25 mg/day) as adjuvant therapy with planned treatment duration of 5 years. Only North American patients (6,827 of 7,576 of MA.27 accrual) were offered participation in the collection of blood specimens, and 5,221 of the North American patients contributed blood and gave consent for genetic testing. This research was approved by health regulatory authorities and the institutional review boards of participating centers.

Genotyping was performed in three cohorts as detailed in Supplementary Material.[14](#cpt2311-bib-0014), [16](#cpt2311-bib-0016), [17](#cpt2311-bib-0017) For the current analysis, we restricted the studies to the White only population that included 4,465 patients with 245 breast events: 2,226 patients on the anastrozole arm with 121 breast events, and 2,239 patients on the exemestane arm with 124 breast events. African American and Han Chinese cohorts had nine breast events in the combined exemestane and anastrozole cohorts. In addition, we did not have breast events in Han Chinese for the exemestane cohort; therefore, the covariate models with all races were unstable and did not converge. Hence, we chose to analyze only the White cohort.

### Primary end point

The primary outcome was the STEEP[18](#cpt2311-bib-0018) end point of Breast Cancer Free Interval (BCFI), defined as the time from randomization to the first local‐regional breast cancer recurrence (including ipsilateral ductal carcinoma in situ (DCIS)), distant breast cancer recurrence, contralateral breast cancer (invasive or DCIS), or death with or from breast cancer without a prior recurrence date. Follow‐up was censored at non‐breast cancer death or the longest follow‐up without recurrence.

### Statistical design

The pharmacogenomic analysis focused on identifying SNPs effects whose associations with BCFI were significantly different based on the treatment (anastrozole or exemestane). Thus, our analysis used a stratified genomewide Cox‐proportional hazards model using an SNP/treatment interaction term and was controlled for significant stratification factors (chemotherapy, nodal status, and trastuzumab), and additional clinical covariates, including treatment arm, cohort, estrogen receptor (ER)/progesterone receptor status, T stage, Eastern Cooperative Oncology Group (ECOG) performance score, and bisphosphonate use. To avoid bias that might arise from differences in genetic ancestry (i.e., population stratification), the EigenStrat software was used to determine eigenvalues for the SNP correlation matrix that statistically differed from zero based on Tracy–Widom *P* values.[19](#cpt2311-bib-0019), [20](#cpt2311-bib-0020) All the analyses were run using the R statistical computing package, PLINK,[21](#cpt2311-bib-0021) and SAS (SAS Institute, Cary, NC, USA).

### Reporter gene assay

Luciferase reporter gene constructs containing rs1877193 wild type (WT) and variant genotypes were generated by gene synthesis. Specifically, a 1,001 bp segment of the *LY75* promoter containing estrogen response element (ERE) and rs1877193 SNP WT or variant genotype was synthesized and cloned into the Kpnl and Xhol sites upstream of the luc2 gene in the pGL4‐Basic vector (Promega, Madison, WI, USA). Reporter gene constructs containing WT or variant SNP genotypes were transfected into cells using Lipofectamine 2000 (Invitrogen, Waltham, MA, USA). A pRL‐CMV vector (Promega) that expresses renilla luciferase was co‐transfected as an internal control. After AI treatment, the cells were lysed, and luciferase activity was determined using the Dual‐Luciferase Reporter assay system (Promega). Renilla luciferase activity was used to correct for transfection efficiency.

### Statistical analysis

For cell survival, proliferation, gene expression, CHIP, reporter gene assays, and quantifications, data are represented as the mean ± SEM of three independent experiments. Unless otherwise described, two‐way analysis of variance (ANOVA) was performed to test group difference. Then post hoc analysis was carried out to check if specific groups are significantly different or similar. Tukey Honest Significant Differences (HSD; R function: TukeyHSD), which is essentially a modified *t*‐test corrected for multiple comparisons, was applied in this analysis. Statistical significance level is 0.05.

## RESULTS

### Genetic variation interaction with aromatase inhibitor therapy

The Participant Flow Diagram for the MA.27 trial (**Figure ** [S1](#cpt2311-sup-0001)) shows the patients included in the SNP‐treatment interaction studies. The analysis cohort was restricted to 4,465 White patients (2,226 patients treated with anastrozole and 2,239 patients treated with exemestane, with 121 and 124 breast events in each arm). To determine the possible SNP‐treatment interaction effect using the MA.27 cohort, we performed a stratified Cox‐proportional hazard analysis utilizing stratification factors and other covariates as detailed in the Methods section. The Manhattan plot (**Figure ** [1a](#cpt2311-fig-0001)) shows the results of SNP‐treatment interaction analyses for BCFI, and the quantile‐quantile plot (**Figure ** [S2](#cpt2311-sup-0001) **A**) revealed a lambda of 1.025. Although none of these SNPs met the criteria for genomewide significance, we had used the largest sample set available to perform the GWAS, so we chose to pursue the results functionally and mechanistically. The results of those studies supported the novel insights that are described subsequently regarding genetic mechanisms for variation in AI effect.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/41a8/8518419/47748e408918/CPT-110-1038-g001.jpg)

SNP treatment interaction analysis. (a) SNP‐treatment interaction GWAS Manhattan Plot. (b) Anastrozole arm GWAS Manhattan Plot. (c) Exemestane arm GWAS Manhattan Plot. (d) SNP‐treatment interaction GWAS signals show opposite effects on BCFI in the anastrozole arm GWAS vs. the exemestane arm GWAS. Hazard ration from two GWASs. X axis: Hazard ration from the exemestane arm GWAS (BCFI_EXE_HR). Y axis: Hazard ration from the anastrozole arm GWAS (BCFI_ANA_HR). Significance indicated by size, and averaged log (p value) by color. (e) 261 SNP‐treatment interaction GWAS signals showed opposite effects on BCFI in the anastrozole arm GWAS (green) vs. the exemestane arm GWAS (blue), but not associated with BCFI in the combined cohort GWAS (red). Three sets of ordered HRs of 261 SNPs from three GWASs; the whole cohort (red), patients who received only anastrozole (green) and patients who received only exemestane (blue). (f) Venn diagram of the SNPs that were selected after performing clumping using three different criteria (see Methods). BCFI, Breast Cancer Free Interval; GWAS, genomewide association studies; HR, hazard ratio; MAF, Minor Allele Frequency; SNP, single‐nucleotide polymorphism.

**Table ** [S1](#cpt2311-sup-0002) presents the 887 significant SNP‐by‐drug interactions (*P* < 1.0E‐04) on BCFI when comparing exemestane to anastrozole. Signals with an Exemestane/Anastrozole hazard ratio (HR) of > 1 indicate that patients carrying a variant SNP genotype have reduced efficacy when treated with exemestane compared with anastrozole. On the contrary, signals with an Exemestane/Anastrozole HR of < 1, indicate that patients carrying a variant SNP genotype have a better outcome when treated with exemestane compared with anastrozole. We next investigated these 887 SNPs for their associations with anastrozole (**Figure ** [1b](#cpt2311-fig-0001), **Figure ** [S2](#cpt2311-sup-0001) **B**) and exemestane (**Figure ** [1c](#cpt2311-fig-0001), **Figure ** [S2](#cpt2311-sup-0001) **C**) efficacy (BCFI) separately based on GWAS conducted within each treatment arm in the MA.27 study. One hundred thirty‐four of 887 SNPs (**Figure ** [1d](#cpt2311-fig-0001), **Table ** [S2](#cpt2311-sup-0003)) were associated with better outcomes (longer BCFI) in exemestane‐treated patients (HR = 0.04~0.71, *P* = 2.21E‐04~0.009), but with shorter BCFI in anastrozole‐treated patients (HR = 1.42~3.43, *P* = 6.36E‐05~0.0097). One hundred twenty‐seven of 887 SNPs (**Figure ** [1d](#cpt2311-fig-0001), **Table ** [S2](#cpt2311-sup-0003)) were associated with shorter BCFI in exemestane‐treated patients (HR=1.41~2.94, *P* = 2.21E‐04~0.0099), but with longer BCFI in anastrozole‐treated patients (HR = 0.01~0.68, *P* = 1.99E‐05~0.0095). However, these two groups of SNPs (134 + 127 = 261 SNPs; **Figure ** [1e](#cpt2311-fig-0001), **Figure ** [S3](#cpt2311-sup-0001) **A**) showed no association with BCFI in the combined analysis of anastrozole and exemestane (HR = 0.90~1.56, *P* = 0.08~0.99) based on our previous published GWAS results for the MA.27 trial.[14](#cpt2311-bib-0014) **Figure ** [1e](#cpt2311-fig-0001) plots 3 sets of ordered HRs of 261 SNPs from 3 GWASs; the whole cohort (red), patients who received anastrozole (green), and patients who received exemestane (blue). In the left half of the plot, SNPs whose HRs range from 0 to 1 in exemestane cohort have an HR ranging from 1 to 3 in the anastrozole cohort implying the presence of minor alleles in these SNPs will be protective if treated with exemestane as compared to anastrozole. Similarly, in the right half of the plot, we see a set of SNPs whose HRs suggest being protective alleles for anastrozole and risk alleles for exemestane. HRs obtained from the whole cohort remained constant around one, suggesting low power to detect any SNP effect. In summary, the current analyses revealed a difference in anastrozole and exemestane efficacy in the context of the host genome and identified germline genetic variants that might be able to identify which AI might be superior in preventing breast cancer recurrence in that specific patient. Importantly, over 98.7% of the signals (876 of 887 SNPs) were lost (*P* > 0.01) in the GWAS analysis when anastrozole and exemestane were combined. Moreover, 261 SNPs were observed to be risk alleles for one AI and protective for the other (**Figure ** [1e](#cpt2311-fig-0001)).

### Genetic model for guiding aromatase inhibitor selection

We next set out to build a genetic model for guiding AI selection. Of the 261 SNPs, 105 SNPs marginally affected BCFI based on any of the clinical covariates and were removed from further analysis. After filtering the 105 SNPs, the remaining 156 SNPs underwent linkage disequilibrium (LD) –clumping analyses.[22](#cpt2311-bib-0022) In brief, “clumping” reported the SNPs that were within 500 kb of the significant SNP, were in high LD (*r* ^2^ > 0.8), and had a low *P* value. For anastrozole and exemestane cohorts, we chose a *P* value cutoff < 0. 001, and for the interaction GWAS analysis, we chose a *P* value cutoff < 9.99E‐05. From the clumping analysis for each study, we obtained a set of SNPs that were common and unique to each study, as shown in **Figure ** [1f](#cpt2311-fig-0001) and **Table ** [1](#cpt2311-tbl-0001). Coincidentally, the number of SNPs remaining after clumping was 46 for all 3 studies. As shown in the Venn diagram, we overlapped the SNP IDs from the three studies after clumping to identify a reduced list of SNPs that were significantly associated with anastrozole and with exemestane. Thirty‐two SNPs were retained after clumping in all the 3 studies, 14 of those SNPs were present in 2 or more studies (interaction and anastrozole or interaction and exemestane LD analysis), 8 SNPs were unique to exemestane only, and 6 SNPs were retained in the anastrozole only clumping analysis, as shown in **Table ** [S3](#cpt2311-sup-0001). We ultimately obtained a list of 60 unique SNPs (32 + 14 + 8 + 6 = 60) as our SNP panel. Note that these SNPs had been obtained from a pure discovery activity and had not been tested or validated in an independent set. **Figure ** [S3](#cpt2311-sup-0001) **b,c** show the HR for SNPs selected based on each of the drug‐specific association results.

### Table 1.

| GWAS used | P value cutoff | No. of SNPs from clumping |
| --- | --- | --- |
| BCFI Interaction | 9.99E‐05 | 46 |
| BCFI with anastrozole cohort | 0.001 | 46 |
| BCFI with exemestane cohort | 0.001 | 46 |

Table 1 Caption: SNPs that remained significant after clumping in each of the three GWAS

### Functional genomics of the top treatment interacting GWAS SNPs

The next series of experiments were performed to determine the function of the 261 SNPs from our SNP‐treatment interaction analysis. Expression quantitative trait loci (eQTL) are particularly useful for annotating variants associated with complex traits.[23](#cpt2311-bib-0023) We, therefore, determined whether any of the 261 SNPs were eQTL SNPs using the Genotype‐Tissue Expression (GTEx) website.[24](#cpt2311-bib-0024) Cis‐eQTL analysis identified 46 SNPs that were eQTLs with 2 genes in breast tissue, and an additional 4 genes in other tissues (**Table ** [S4](#cpt2311-sup-0001)). Collectively, the eQTL analysis identified six “eGenes” (genes exhibiting association with genotypes for at least one SNP). The SNPs associated with any given gene were in high LD with each other, with pairwise correlation *r* ^2^ = 1.

We then investigated which SNP‐eGenes were differentially associated with efficacy for anastrozole and exemestane. We took advantage of cell lines selected on the basis of each of the 6 eQTL SNP genotypes from a genomic data‐rich panel of 300 human lymphoblastoid cell lines (LCLs) that has proven to be a powerful tool for testing pharmacogenomic hypotheses.[15](#cpt2311-bib-0015), [25](#cpt2311-bib-0025), [26](#cpt2311-bib-0026), [27](#cpt2311-bib-0027), [28](#cpt2311-bib-0028), [29](#cpt2311-bib-0029) We selected four LCLs with homozygous WT genotypes for each SNP and four LCLs with the variant genotypes with detectable aromatase expression (**Figure ** [S4](#cpt2311-sup-0001) **a**) to test the impact of anastrozole and exemestane on cell growth. We ensured that the aromatase activity was inhibited by anastrozole and exemestane (**Figure ** [S4](#cpt2311-sup-0001) **b**). The eQTL SNPs for the following genes, *LY75*, *GPR160*, *PLA2R1*, *NPLOC4*, and *NT5C3A*, showed that the variant genotypes all had a differential effect on cell growth between exemestane and anastrozole (**Figure ** [2a](#cpt2311-fig-0002), **Figure ** [S4](#cpt2311-sup-0001) **c**). Among these five eQTL relationships, three were not consistent with our analysis results. The directional effect of the eQTL SNPs for *NT5C3A* on individual drug efficacy identified through SNP‐drug interaction GWAS was not consistent with experimental results (**Figure ** [S4](#cpt2311-sup-0001), **Table ** [S1](#cpt2311-sup-0002)). As for the *NPLOC4* SNPs, anastrozole, in comparison to exemestane, had greater growth inhibition on cells with variant SNP genotypes, which was consistent with the SNP‐drug interaction GWAS results (Minor Allele Frequency (MAF) = 0.037, HR: Anas vs. Exem = 0). However, the anastrozole to exemestane HR was 0 due to the low MAF, resulting in low statistical power,[30](#cpt2311-bib-0030), [31](#cpt2311-bib-0031) raising the possibility that the association might be a false positive. Indeed, regardless of the *NPLOC4* SNP genotype, cells grew faster in the presence of exemestane compared with anastrozole (**Figure ** [S4](#cpt2311-sup-0001)). The eQTL SNPs for *LY75* (lymphocyte antigen 75), *GPR160* (G Protein‐Coupled Receptor 160), and *PLA2R1* (Phospholipase A2 Receptor 1) showed a consistent effect on drug response between GWAS result and the experimental validation (**Figure ** [S4](#cpt2311-sup-0001), **Figure ** [2a](#cpt2311-fig-0002), **Table ** [S1](#cpt2311-sup-0002)), suggesting that these SNPs may differentially affect anastrozole and exemestane response through regulation of the *LY75*, *GPR160*, and *PLA2R1* genes. We sought to study the functional impact of *LY75*, *GPR160*, and *PLA2R1* genes, which are expressed in breast tissue (**Figure ** [S5](#cpt2311-sup-0001)). Knockdown of *LY75* and *GPR160*, but not *PLA2R1*, decreased breast cancer cell proliferation (**Figure ** [2b,c](#cpt2311-fig-0002), **Figure ** [S6](#cpt2311-sup-0001) **a**). We then validated the interaction effects of these three genes with AIs. *LY75* and *GPR160*, but not *PLA2R1*, knockdown cells grew more slowly in the presence of anastrozole compared to exemestane (**Figure ** [2d,e](#cpt2311-fig-0002), **Figure ** [S6](#cpt2311-sup-0001) **b**). Therefore, we chose to focus on the functional implications of the *LY75* and *GPR160* SNPs because those SNPs may represent regulatory variants.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/41a8/8518419/2b5b48af755b/CPT-110-1038-g004.jpg)

LY75 and GPR160 SNPs dependent effect on cell proliferation. (a) Cell growth of LCLs with known genotypes for variant (V) and wild type (WT) LY75 SNPs and GPR160 SNP were incubated with 100 nM anastrozole (Ana) or exemestane (Exe) for the indicated times. WT: homozygous WT, V: heterozygous and v/v: homozygous variant LCLs (n = 3, means ± SEM). (b) Cell growth of MCF‐7 and T47D cells after knocking down LY75. (c) Cell growth of MCF‐7 and T47D cells after knocking down GPR160 (n = 3, means ± SEM). **P < 0.01. Statistical test: two‐way ANOVA. (d) MCF‐7 and T47D cells were transfected with LY75 siRNA, 24 hours later cells were treated with Ana or Exe. Cell growth of MCF‐7 and T47D cells were then measured (n = 3, means ± SEM). **P < 0.01. (e) MCF‐7 and T47D cells were transfected with GPR160 siRNA, 24 hours later cells were treated with Ana or Exe. Cell growth of MCF‐7 and T47D cells were then measured (n = 3, means ± SEM). **P < 0.01. Statistical test: two‐way ANOVA. (f) The rs1877193 SNP and the rs62293499 SNP cis‐eQTLs for LY75 and GPR160, respectively in GTEX. ANOVA, analysis of variance; eQTL, expression quantitative trait loci; LCL, lymphoblastoid cell line; SNP, single‐nucleotide polymorphism.

### Associations with breast cancer free interval in treatment‐stratified analysis

The two common SNPs (MAF = 0.455), rs1877193 and rs6735923, were eQTLs for *LY75* gene expression in breast tissue (**Figure ** [2f](#cpt2311-fig-0002)), and they were also in perfect LD (*r* ^2^ = 1). These 2 SNPs mapped upstream of *LY75* on chromosome 2 (**Figure ** [3a,b](#cpt2311-fig-0003)) and had significant interaction with treatments (SNP rs1877193: HR: Exe/Ana = 0.45; SNP rs6735923: HR: Exe/Ana = 0.46; **Table ** [S5](#cpt2311-sup-0004)), indicating ~ 55% reduction in the hazard for a breast cancer event for patients carrying the variant genotype treated with exemestane compared with anastrozole. In treatment‐specific cohort GWAS analysis, the two SNPs were associated with opposite effects on risk for a breast event between the two treatment cohorts: increased risk in the anastrozole cohort (HR = 1.57, 1.58; *P* = 5.3E‐04, 6.9E‐04), and decreased risk in the exemestane cohort (HR = 0.70, 0.71; *P* = 6.0E‐03, 8.0E‐03). However, the two SNPs showed no association with breast events if the two treatment arms were combined[14](#cpt2311-bib-0014) (HR = 1.07, 1.08; *P* = 0.37, 0.40; **Table ** [S5](#cpt2311-sup-0004)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/41a8/8518419/bc57a9cdc5da/CPT-110-1038-g005.jpg)

LY75 SNP‐ and anastrozole‐dependent gene regulation. (a, b) Locus zoom plot of the chromosome 2 region surrounding the LY75 gene. (c, d) SNP‐androstenedione‐ and anastrozole (c, Ana) or exemestane (d, Exe) ‐dependent regulation of LY75 gene expression in LCLs selected based on LY75 SNP genotypes. WT: homozygous WT, V: heterozygous variant LCLs. (n = 3, means ± SEM). **P < 0.01. (e) Schematic figure showing the estrogen response elements (ERE) surrounding the rs1877193 SNP. (f) ERα ChIP assay showing SNP dependent ERα binding to the ERE region that is 200 bp upstream from the rs1877193 SNP in LCLs with different genotypes treated with the indicated drugs: androstenedione (A), anastrozole (Ana), and exemestane (Exe) (n = 3, means ± SEM). **P < 0.01. Statistical test: two‐way ANOVA. (g) MCF‐7 and T47D cells were transfected with ESR1 siRNA. Forty‐eight hours later, mRNA was harvested for qRT‐PCR (n = 3, means ± SEM). **P < 0.01. (h–j) Reporter gene assays for the LY75 rs1877193 SNP. The rs1877193 SNP was included in the LY75 promotor sequence that was subcloned upstream of the luciferase reporter gene, Luc2. The plasmids contained either WT or variant sequences for the SNP and were transfected into h LCLs, i MCF‐7, and j T47D cells. Values shown for h–j are ratios of relative light units (RLUs) compared with the internal reference after vehicle (baseline), A, Ana, or Exe treatment (n = 3, means ± SEM). **P < 0.01, ns, not significant. Statistical test: two‐way ANOVA. ANOVA, analysis of variance; LCL, lymphoblastoid cell line; PCR, polymerase chain reaction; qRT‐PCR, quantitative real‐time polymerase chain reaction; SNP, single‐nucleotide polymorphism.

The *GPR160* SNP rs62293499 mapped downstream of *GPR160* on chromosome 3p7, and the variant allele was associated with increased *GPR160* expression (**Figure ** [2f](#cpt2311-fig-0002)). The minor allele (MAF = 0.338) was associated with decreased risk for breast events when treated with exemestane compared with anastrozole (HR: Exe/Ana = 0.43; **Table ** [S5](#cpt2311-sup-0004)). This effect was also confirmed by independent GWAS analysis in each treatment arm, in that a trend for increased risk of a breast event within the anastrozole arm (HR = 1.39, *P* = 0.014), and for significantly decreased risk within the exemestane arm (HR = 0.58, *P* = 2.0E‐04) were observed (**Table ** [S5](#cpt2311-sup-0004)). Similar to the *LY75* SNPs, rs62293499 was not associated with breast events when the two treatment arms were combined (HR = 0.92, *P* = 0.39; **Table ** [S5](#cpt2311-sup-0004)).

### LY75 SNP effects differ in the presence of anastrozole vs. exemestane

We next set out to determine whether and how the *LY75* SNPs might have a differential impact on response to the two AIs through *LY75*. We selected LCLs with WT and variant genotypes for *LY75* SNPs, and then exposed them to an AI in the presence of androstenedione, a precursor of estrogen. The variant cell lines showed a reduction of *LY75* expression in response to androstenedione exposure, whereas WT LCLs did not (**Figure ** [3c,d](#cpt2311-fig-0003)). Addition of anastrozole showed a striking reversal of the *LY75* expression pattern in LCLs with variant SNP genotypes: *LY75* expression increased almost threefold with the addition of anastrozole. In contrast, WT expression did not change (**Figure ** [3c](#cpt2311-fig-0003)). Of importance, exemestane did not change the expression patterns of *LY75* in either WT or variant genotype LCLs (**Figure ** [3d](#cpt2311-fig-0003)).

### Anastrozole, but not exemestane, regulates LY75 SNP‐dependent ERα/ERE binding

To determine whether the SNP‐ and AI‐dependent *LY75* gene regulation was mediated by ERα, we first identified two EREs; 2 bp overlap with each other) located ~ 200 bp distant from the *LY75* SNP rs1877193 based on ENCODE data (**Figure ** [3e](#cpt2311-fig-0003)). We then performed ChIP assays with ERα antibody and found that the rs1877193 SNP (**Figure ** [3f](#cpt2311-fig-0003)) showed striking differential binding to the ERE that was located closest to the SNP, with greater binding for the variant SNP genotype in the presence of androstenedione, but less binding when additional anastrozole was present. However, this differential binding pattern was not detected after exemestane treatment (**Figure ** [3f](#cpt2311-fig-0003)). *LY75* expression was reduced whereas ERα/ERE binding increased in cells with the variant genotypes in response to androstenedione exposure, suggesting that ERα functions as a transcription repressor of *LY75*. Indeed, *LY75* expression increased when ESR1 was knocked down (**Figure ** [3g](#cpt2311-fig-0003)). These results were compatible with our SNP‐treatment interaction analysis results in that variant *LY75* SNP genotypes could differentiate anastrozole from exemestane in the risk for a breast event during 5 years of AI therapy for those enrolled in MA.27.

Finally, to determine whether the differences in binding shown in **Figure ** [3f](#cpt2311-fig-0003) might have functional consequences; we created reporter gene constructs that included ~ 1.1 kb of the *LY75* promoter with WT and variant genotypes for the rs1877193 SNP. These reporter gene constructs were then transfected into different cell lines to compare *LY75* transcriptional activities for WT and variant SNP genotypes in those cells. within the presence of androstenedione, cells transfected with the variant construct showed higher *LY75* transcriptional activity than did cells transfected with the WT construct. However, when the cells were incubated with the addition of 100 nM anastrozole, the transcriptional activity in cells transfected with the variant SNP construct decreased (**Figure ** [3h–j](#cpt2311-fig-0003)). We did not observe any change when we repeated the same experiments with exemestane (**Figure ** [3h–j](#cpt2311-fig-0003)).

### LY75 suppression directs breast cancer epithelial‐to‐mesenchymal transition

To identify molecular mechanisms relevant to the *LY75*—treatment interaction, we interrogated The Cancer Genome Atlas (TCGA)[32](#cpt2311-bib-0032) for genes co‐expressed with *LY75* in ER+ breast cancer sample set[33](#cpt2311-bib-0033) using the cBioPortal online platform. After filtering the co‐expression gene prediction results, 2,352 predicted genes (**Figure ** [4a](#cpt2311-fig-0004)) with Spearman’s correlations ≥ 0.3 or ≤ −0.3 were selected (**Table ** [S6](#cpt2311-sup-0001)). Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway analysis of these 2,352 genes identified PI3K‐AKT, MAPK, cell adhesion, and JAK‐STAT pathways (**Figure ** [4b](#cpt2311-fig-0004)). We evaluated the top pathways, and our results revealed that JAK/STAT signaling was dysregulated by *LY75* depletion (**Figure ** [4c](#cpt2311-fig-0004)). Activation of the JAK/STAT3 signaling via IL‐6/IL‐6R leads to increased metastasis resulting from enhanced epithelial‐to‐mesenchymal (EMT).[34](#cpt2311-bib-0034), [35](#cpt2311-bib-0035), [36](#cpt2311-bib-0036), [37](#cpt2311-bib-0037) Remarkably, LY75 suppression induced EMT as shown by the expression of specific EMT markers: a significant decrease of E‐cadherin and EPCAM; an increase of N‐cadherin and SNAIL1 (**Figure ** [4c](#cpt2311-fig-0004)). *LY75* knockdown cells also displayed decreased proliferation rates compared with the control (**Figure ** [2b](#cpt2311-fig-0002)), which was associated with decreased AKT and MAPK phosphorylation (**Figure ** [4c](#cpt2311-fig-0004)). Moreover, *LY75* suppression significantly promoted cell migration (**Figure ** [4d](#cpt2311-fig-0004)). Thus, the *in vitro* invasiveness and motility of *LY75*‐knockdown cells was inversely correlated with their proliferative potential which might be due to the acquiring of the mesenchymal phenotype.[38](#cpt2311-bib-0038) To provide potential clues to the observed genetic correlations, we examined the SNP‐treatment interaction effect on the JAK/STAT and MAPK activities in LCLs with WT and variant genotypes for the *LY75* SNPs. The variant cells showed increased expression of pSTAT3 and decreased pERK in response to androstenedione treatment. In contrast, in WT LCLs, pSTAT3, and pERK, expressions were virtually unchanged with androstenedione. Importantly, the addition of anastrozole, but not exemestane, significantly reduced the pSTAT3 and pERK levels in variant cells (**Figure ** [4e](#cpt2311-fig-0004)). The next question that we addressed was the possible mechanism by which the *GPR160* SNP might result in differential effects in response to anastrozole and exemestane treatment.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/41a8/8518419/f6bc0015e953/CPT-110-1038-g003.jpg)

LY75 suppression directs epithelial‐to‐mesenchymal transition (EMT). (a) LY75 correlated genes in 593 TCGA ER+ patients. (b) Pathway analysis of LY75 correlated gene in TCGA ER+ population. (c) Western blot analysis of the expression of different EMT (epithelial and mesenchymal) markers in the control and the LY75 knockdown breast cancer cells. (d) Cell migration of LY75 knockdown cells was compared to the control cells (Neg). Equal numbers of cells from Neg and siLY75 groups were seeded into 12‐well plates for wound healing assay. Original magnification: ×10. (e) LY75 SNP effect on pSTAT3 and pERK protein levels in LCLs with either WT or variant sequences for the SNP treated with androstenedione (A), anastrozole (Ana), and exemestane (Exe). ER, estrogen receptor; LCL, lymphoblastoid cell line; SNP, single‐nucleotide polymorphism; TCGA, The Cancer Genome Atlas; V, variant; WT, wild type. [Colour figure can be viewed at wileyonlinelibrary.com]

### Anastrozole differs from exemestane regarding GPR160 SNP‐dependent gene regulation

The *GPR160* SNP rs62293499 (**Figure ** [5a](#cpt2311-fig-0005)) variant allele was associated with longer BCFI after treatment with exemestane as compared to anastrozole (**Table ** [S5](#cpt2311-sup-0004)). We began our functional genomic studies of the *GPR160* SNP starting with the question of whether *GPR160* expression might be regulated in an SNP‐treatment‐dependent fashion. *GPR160* mRNA expression was induced by exposure to increasing concentrations of androstenedione in cells with WT genotype, but not in those with the variant SNP genotype (**Figure ** [5b](#cpt2311-fig-0005)). We observed a striking reversal of expression with anastrozole, but not exemestane, that is, increased expression in the presence of anastrozole in cells with variant *GPR160* SNP genotype, but decreased expression in cells with WT genotype (**Figure ** [5b](#cpt2311-fig-0005)). Several EREs were located within 500 bp of the *GPR160* SNP (**Figure ** [5c](#cpt2311-fig-0005)), and the rs62293499 SNP genotype was associated with striking differential binding to the EREs, with more binding for the WT SNP genotype in the presence of androstenedione, but a greater binding for the variant SNP when anastrozole was present. Exemestane, however, did not change the binding activity (**Figure ** [5d](#cpt2311-fig-0005)).

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/41a8/8518419/a94fda236820/CPT-110-1038-g002.jpg)

GPR160 SNP‐ and anastrozole‐dependent gene regulation. (a) Locus zoom plot of the chromosome 3 region surrounding the GPR160 gene. (b) SNP‐androstenedione‐ and anastrozole‐dependent regulation of GPR160 gene expression in LCLs selected based on the GPR160 SNP genotypes. WT: homozygous WT, V: homozygous variant LCLs (n = 3, means ± SEM).**P < 0.01. (c) Schematic figure showing the EREs surrounding the rs62293499 SNP. (d) ERα ChIP assay that show SNP‐dependent ERα binding to the EREs region that are upstream from the rs62293499 SNP in LCLs with different genotypes treated with the indicated drugs: androstenedione (A), anastrozole (Ana), and exemestane (Exe) (n = 3, means ± SEM). **P < 0.01. Statistical test: two‐way ANOVA. ANOVA, analysis of variance; ERE, estrogen response element; LCL, lymphoblastoid cell line; SNP, single‐nucleotide polymorphism.

We identified 223 genes that were correlated with GPR160 in TCGA ER+ breast cancer data set (**Table ** [S6](#cpt2311-sup-0001), **Figure ** [S7](#cpt2311-sup-0001) **a**), enriched in metabolic, PI3K‐AKT, and MAPK signaling pathways. GPR160 knockdown resulted in the downregulation of cell proliferation pathway, such as PI3K‐AKT (**Figure ** [S7](#cpt2311-sup-0001) **b**).

## DISCUSSION

In the present investigation, we evaluated the interaction between genetic variation and 2 AIs, exemestane and anastrozole, on BCFI in the MA.27 clinical trial in which postmenopausal women with resected early‐stage breast cancer were treated with anastrozole or exemestane adjuvant therapy (**Figure ** [1a](#cpt2311-fig-0001)). The SNPs identified in the interaction analysis were analyzed for their association with BCFI in the treatment‐specific cohort (**Figure ** [1b,c](#cpt2311-fig-0001)), as well as in the combined cohort using our pervious GWAS results.[14](#cpt2311-bib-0014) This approach allowed the identification of genetic variants that have a differential effect on BCFI between the two AIs (**Figure ** [1d](#cpt2311-fig-0001)). The SNPs that interacted with treatments were otherwise lost in the BCFI analysis when the GWAS was adjusted for the treatment arms (**Figure ** [1e](#cpt2311-fig-0001), **Table ** [S2](#cpt2311-sup-0003)). Our study has provided additional evidence of potential differences among the third‐generation AIs given differences in individual genetic background. We identified an SNP‐based model (**Table ** [S3](#cpt2311-sup-0001), **Figure ** [1f](#cpt2311-fig-0001)) as a biomarker signature panel for predicting superiority in efficacy between anastrozole and exemestane in a given patient. Our current study only focused on White patients because we only had nine patients from Asian or African decedents who have developed a breast event. In the future, it needs to extend these genetic biomarkers to other ethnicities.

Support for the position that anastrozole differs from exemestane is provided by our functional validation studies for the SNPs near the *LY75* [39](#cpt2311-bib-0039) and *GPR160* [40](#cpt2311-bib-0040) genes (Figures [3a](#cpt2311-fig-0003),[b](#cpt2311-fig-0005) and [5a](#cpt2311-fig-0005)). Of note, there had been no prior reports of a relationship of either of these genes with AI efficacy or breast cancer risk. The expression of *LY75* in cells with variant *LY75* SNP sequences was dysregulated by exposure to anastrozole, but not exemestane (**Figure ** [3c,d](#cpt2311-fig-0003)), which was compatible with the association of variant *LY75* SNP with protection from a breast cancer recurrence while subjects were being treated with exemestane, but not anastrozole. We also showed that the SNP regulated *LY75* expression through ERα (**Figure ** [3e–j](#cpt2311-fig-0003)). We report for the first time a role for the *LY75* gene in controlling breast cancer cellular phenotypes and corresponding metastatic potential (Figures [4](#cpt2311-fig-0004) and [2b](#cpt2311-fig-0002)). These findings also raise the possibility of targeting downstream EMT in combination with a specific AI to increase its sensitivity for future testing. The *GPR160* SNP mediated *GPR160* expression in response to anastrozole, but not exemestane, by regulating ERα binding to EREs (**Figure ** [5](#cpt2311-fig-0005)). *GPR160* played a role in breast cancer cell growth (**Figure ** [S7](#cpt2311-sup-0001) **b**, **Figure ** [2c](#cpt2311-fig-0002)).

Although all three AIs are interchangeable clinically based on large clinical studies, those studies did not consider patient genetic variability for recruitment and stratification. Our study raises the possibility of salvaging specific AI‐treated patients identified to be at increased risk of breast cancer recurrence with a given AI in that they could be switched to an alternative AI. Thus, the genetic markers identified in this study could help select the appropriate AI for a given patient. A limitation of our study is that no other large adjuvant trial of monotherapy with anastrozole and exemestane collected genotyping data is available. However, we have obtained evidence that anastrozole and exemestane have differing efficacy depending on genetic background and have identified an SNP biomarker panel that could potentially be used to select optimal AI for a given patient.

## FUNDING

This research was supported in part by NIH grants P50CA116201 (Mayo Clinic Breast Cancer Specialized Program of Research Excellence), U1961388 (The Pharmacogenomics Research Network), U54 GM114838, BCRF‐17‐075 (the Breast Cancer Research Foundation), U10CA77202, R01CA196648, CCS 015469 from the Canadian Cancer Society, the George M. Eisenberg Foundation for Charities, and the Nan Sawyer Breast Cancer Fund.

## CONFLICT OF INTEREST

M.J.E. reports employment with Bioclassifier, LLC; royalty income from Prosigma/Nanostring; honoraria from Nanostring, Novartis, AstraZeneca, Pfizer, Sermonix, Abbvie; patent interest from Bioclassifier, LLC, Prosigma/PAM50; P.E.G. reports research funding from Amgen. M.P.G. reports consulting roles with Lilly, Bovica, Novartis, Sermonix, Context Pharmaceuticals, and Genomic Health; R.M.W. and L.W. are co‐founders of, and stockholders in OneOme, LLC. All other authors declared no competing interests for this work.

## AUTHOR CONTRIBUTIONS

J.C., K.R.K., J.N.I., R.M.W., and L.W. wrote the manuscript. J.C., J.N.I., and L.W. designed the research. J.C., J.N.I., K.R.K., E.E.C., and P.B. performed the research. J.C., J.N.I., K.R.K., E.E.C., P.B., M.J.E., P.E.G., L.E.S., M.P.G., B.G., and H.G. analyzed the data. K.R.K., E.E.C., and P.B. contributed new reagents/analytical tools.

## Supporting information

## ACKNOWLEDGMENTS

The authors acknowledge the women who participated in the MA.27 clinical trial and provided blood samples used in these analyses.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Bray, F. , Ferlay, J. , Soerjomataram, I. , Siegel, R.L. , Torre, L.A. & Jemal, A. Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. CA Cancer J. Clin. 68, 394–424 (2018).  [DOI](https://doi.org/10.3322/caac.21492) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30207593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J.%20Clin.&title=Global%20cancer%20statistics%202018:%20GLOBOCAN%20estimates%20of%20incidence%20and%20mortality%20worldwide%20for%2036%20cancers%20in%20185%20countries&author=F.%20Bray&author=J.%20Ferlay&author=I.%20Soerjomataram&author=R.L.%20Siegel&author=L.A.%20Torre&volume=68&publication_year=2018&pages=394-424&pmid=30207593&doi=10.3322/caac.21492&)

2. Burstein, H.J. et al. Adjuvant endocrine therapy for women with hormone receptor‐positive breast cancer: American society of clinical oncology clinical practice guideline focused update. J. Clin. Oncol. 32, 2255–2269 (2014).  [DOI](https://doi.org/10.1200/JCO.2013.54.2258) | [PMC free article](/articles/PMC4876310/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24868023/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Adjuvant%20endocrine%20therapy%20for%20women%20with%20hormone%20receptor%E2%80%90positive%20breast%20cancer:%20American%20society%20of%20clinical%20oncology%20clinical%20practice%20guideline%20focused%20update&author=H.J.%20Burstein&volume=32&publication_year=2014&pages=2255-2269&pmid=24868023&doi=10.1200/JCO.2013.54.2258&)

3. Smith, I.E. & Dowsett, M. Aromatase inhibitors in breast cancer. N. Engl. J. Med. 348, 2431–2442 (2003).  [DOI](https://doi.org/10.1056/NEJMra023246) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12802030/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Aromatase%20inhibitors%20in%20breast%20cancer&author=I.E.%20Smith&author=M.%20Dowsett&volume=348&publication_year=2003&pages=2431-2442&pmid=12802030&doi=10.1056/NEJMra023246&)

4. Baum, M. et al. Anastrozole alone or in combination with tamoxifen versus tamoxifen alone for adjuvant treatment of postmenopausal women with early‐stage breast cancer: results of the ATAC (arimidex, tamoxifen alone or in combination) trial efficacy and safety update analyses. Cancer 98, 1802–1810 (2003).  [DOI](https://doi.org/10.1002/cncr.11745) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14584060/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Anastrozole%20alone%20or%20in%20combination%20with%20tamoxifen%20versus%20tamoxifen%20alone%20for%20adjuvant%20treatment%20of%20postmenopausal%20women%20with%20early%E2%80%90stage%20breast%20cancer:%20results%20of%20the%20ATAC%20(arimidex,%20tamoxifen%20alone%20or%20in%20combination)%20trial%20efficacy%20and%20safety%20update%20analyses&author=M.%20Baum&volume=98&publication_year=2003&pages=1802-1810&pmid=14584060&doi=10.1002/cncr.11745&)

5. Breast International Group (BIG) 1‐98 Collaborative group et al. A comparison of letrozole and tamoxifen in postmenopausal women with early breast cancer. N. Engl. J. Med. 353, 2747–2757 (2005).  [DOI](https://doi.org/10.1056/NEJMoa052258) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16382061/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=A%20comparison%20of%20letrozole%20and%20tamoxifen%20in%20postmenopausal%20women%20with%20early%20breast%20cancer&volume=353&publication_year=2005&pages=2747-2757&pmid=16382061&doi=10.1056/NEJMoa052258&)

6. Coombes, R.C. et al. A randomized trial of exemestane after two to three years of tamoxifen therapy in postmenopausal women with primary breast cancer. N. Engl. J. Med. 350, 1081–1092 (2004).  [DOI](https://doi.org/10.1056/NEJMoa040331) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15014181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=A%20randomized%20trial%20of%20exemestane%20after%20two%20to%20three%20years%20of%20tamoxifen%20therapy%20in%20postmenopausal%20women%20with%20primary%20breast%20cancer&author=R.C.%20Coombes&volume=350&publication_year=2004&pages=1081-1092&pmid=15014181&doi=10.1056/NEJMoa040331&)

7. Early Breast Cancer Trialists' Collaborative Group . Aromatase inhibitors versus tamoxifen in early breast cancer: patient‐level meta‐analysis of the randomised trials. Lancet 386, 1341–1352 (2015).  [DOI](https://doi.org/10.1016/S0140-6736(15)61074-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26211827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Aromatase%20inhibitors%20versus%20tamoxifen%20in%20early%20breast%20cancer:%20patient%E2%80%90level%20meta%E2%80%90analysis%20of%20the%20randomised%20trials&volume=386&publication_year=2015&pages=1341-1352&pmid=26211827&doi=10.1016/S0140-6736(15)61074-1&)

8. Geisler, J. et al. Letrozole is superior to anastrozole in suppressing breast cancer tissue and plasma estrogen levels. Clin. Cancer Res. 14, 6330–6335 (2008).  [DOI](https://doi.org/10.1158/1078-0432.CCR-07-5221) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18829517/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Letrozole%20is%20superior%20to%20anastrozole%20in%20suppressing%20breast%20cancer%20tissue%20and%20plasma%20estrogen%20levels&author=J.%20Geisler&volume=14&publication_year=2008&pages=6330-6335&pmid=18829517&doi=10.1158/1078-0432.CCR-07-5221&)

9. Smith, I. et al. Comparative efficacy and safety of adjuvant letrozole versus anastrozole in postmenopausal patients with hormone receptor‐positive, node‐positive early breast cancer: final results of the randomized phase III femara versus anastrozole clinical evaluation (FACE) trial. J. Clin. Oncol. 35, 1041–1048 (2017).  [DOI](https://doi.org/10.1200/JCO.2016.69.2871) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28113032/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Comparative%20efficacy%20and%20safety%20of%20adjuvant%20letrozole%20versus%20anastrozole%20in%20postmenopausal%20patients%20with%20hormone%20receptor%E2%80%90positive,%20node%E2%80%90positive%20early%20breast%20cancer:%20final%20results%20of%20the%20randomized%20phase%20III%20femara%20versus%20anastrozole%20clinical%20evaluation%20(FACE)%20trial&author=I.%20Smith&volume=35&publication_year=2017&pages=1041-1048&pmid=28113032&doi=10.1200/JCO.2016.69.2871&)

10. Goss, P.E. et al. Exemestane versus anastrozole in postmenopausal women with early breast cancer: NCIC CTG MA.27–a randomized controlled phase III trial. J. Clin. Oncol. 31, 1398–1404 (2013).  [DOI](https://doi.org/10.1200/JCO.2012.44.7805) | [PMC free article](/articles/PMC3612593/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23358971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Exemestane%20versus%20anastrozole%20in%20postmenopausal%20women%20with%20early%20breast%20cancer:%20NCIC%20CTG%20MA.27%E2%80%93a%20randomized%20controlled%20phase%20III%20trial&author=P.E.%20Goss&volume=31&publication_year=2013&pages=1398-1404&pmid=23358971&doi=10.1200/JCO.2012.44.7805&)

11. De Placido, S. et al. Adjuvant anastrozole versus exemestane versus letrozole, upfront or after 2 years of tamoxifen, in endocrine‐sensitive breast cancer (FATA‐GIM3): a randomised, phase 3 trial. Lancet Oncol. 19, 474–485 (2018).  [DOI](https://doi.org/10.1016/S1470-2045(18)30116-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29482983/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol.&title=Adjuvant%20anastrozole%20versus%20exemestane%20versus%20letrozole,%20upfront%20or%20after%202%20years%20of%20tamoxifen,%20in%20endocrine%E2%80%90sensitive%20breast%20cancer%20(FATA%E2%80%90GIM3):%20a%20randomised,%20phase%203%20trial&author=S.%20De%20Placido&volume=19&publication_year=2018&pages=474-485&pmid=29482983&doi=10.1016/S1470-2045(18)30116-5&)

12. Ingle, J.N. et al. Anastrozole has an association between degree of estrogen suppression and outcomes in early breast cancer and is a ligand for estrogen receptor alpha. Clin. Cancer Res. 26, 2986–2996 (2020).  [DOI](https://doi.org/10.1158/1078-0432.CCR-19-3091) | [PMC free article](/articles/PMC7299827/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32098767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Anastrozole%20has%20an%20association%20between%20degree%20of%20estrogen%20suppression%20and%20outcomes%20in%20early%20breast%20cancer%20and%20is%20a%20ligand%20for%20estrogen%20receptor%20alpha&author=J.N.%20Ingle&volume=26&publication_year=2020&pages=2986-2996&pmid=32098767&doi=10.1158/1078-0432.CCR-19-3091&)

13. Cairns, J. et al. Pharmacogenomics of aromatase inhibitors in postmenopausal breast cancer and additional mechanisms of anastrozole action. Science 5(16), e137571 (2020). 10.1172/jci.insight.137571  [DOI](https://doi.org/10.1172/jci.insight.137571) | [PMC free article](/articles/PMC7455128/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32701512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Pharmacogenomics%20of%20aromatase%20inhibitors%20in%20postmenopausal%20breast%20cancer%20and%20additional%20mechanisms%20of%20anastrozole%20action&author=J.%20Cairns&volume=5(16)&publication_year=2020&pmid=32701512&doi=10.1172/jci.insight.137571&)

14. Ingle, J.N. et al. Genetic polymorphisms in the long noncoding RNA MIR2052HG offer a pharmacogenomic basis for the response of breast cancer patients to aromatase inhibitor therapy. Cancer Res. 76, 7012–7023 (2016).  [DOI](https://doi.org/10.1158/0008-5472.CAN-16-1371) | [PMC free article](/articles/PMC5135610/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27758888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=Genetic%20polymorphisms%20in%20the%20long%20noncoding%20RNA%20MIR2052HG%20offer%20a%20pharmacogenomic%20basis%20for%20the%20response%20of%20breast%20cancer%20patients%20to%20aromatase%20inhibitor%20therapy&author=J.N.%20Ingle&volume=76&publication_year=2016&pages=7012-7023&pmid=27758888&doi=10.1158/0008-5472.CAN-16-1371&)

15. Cairns, J. et al. The lncRNA MIR2052HG regulates ERalpha levels and aromatase inhibitor resistance through LMTK3 by recruiting EGR1. Breast Cancer Res. 21, 47 (2019).  [DOI](https://doi.org/10.1186/s13058-019-1130-3) | [PMC free article](/articles/PMC6448248/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30944027/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Breast%20Cancer%20Res.&title=The%20lncRNA%20MIR2052HG%20regulates%20ERalpha%20levels%20and%20aromatase%20inhibitor%20resistance%20through%20LMTK3%20by%20recruiting%20EGR1&author=J.%20Cairns&volume=21&publication_year=2019&pages=47&pmid=30944027&doi=10.1186/s13058-019-1130-3&)

16. Ingle, J.N. et al. Genome‐wide associations and functional genomic studies of musculoskeletal adverse events in women receiving aromatase inhibitors. J. Clin. Oncol. 28, 4674–4682 (2010).  [DOI](https://doi.org/10.1200/JCO.2010.28.5064) | [PMC free article](/articles/PMC3020700/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20876420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Genome%E2%80%90wide%20associations%20and%20functional%20genomic%20studies%20of%20musculoskeletal%20adverse%20events%20in%20women%20receiving%20aromatase%20inhibitors&author=J.N.%20Ingle&volume=28&publication_year=2010&pages=4674-4682&pmid=20876420&doi=10.1200/JCO.2010.28.5064&)

17. Liu, M. et al. Aromatase inhibitor‐associated bone fractures: a case‐cohort GWAS and functional genomics. Mol. Endocrinol. 28, 1740–1751 (2014).  [DOI](https://doi.org/10.1210/me.2014-1147) | [PMC free article](/articles/PMC4179631/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25148458/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Endocrinol.&title=Aromatase%20inhibitor%E2%80%90associated%20bone%20fractures:%20a%20case%E2%80%90cohort%20GWAS%20and%20functional%20genomics&author=M.%20Liu&volume=28&publication_year=2014&pages=1740-1751&pmid=25148458&doi=10.1210/me.2014-1147&)

18. Hudis, C.A. et al. Proposal for standardized definitions for efficacy end points in adjuvant breast cancer trials: the STEEP system. J. Clin. Oncol. 25, 2127–2132 (2007).  [DOI](https://doi.org/10.1200/JCO.2006.10.3523) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17513820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Proposal%20for%20standardized%20definitions%20for%20efficacy%20end%20points%20in%20adjuvant%20breast%20cancer%20trials:%20the%20STEEP%20system&author=C.A.%20Hudis&volume=25&publication_year=2007&pages=2127-2132&pmid=17513820&doi=10.1200/JCO.2006.10.3523&)

19. Price, A.L. , Patterson, N.J. , Plenge, R.M. , Weinblatt, M.E. , Shadick, N.A. & Reich, D. Principal components analysis corrects for stratification in genome‐wide association studies. Nat. Genet. 38, 904–909 (2006).  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome%E2%80%90wide%20association%20studies&author=A.L.%20Price&author=N.J.%20Patterson&author=R.M.%20Plenge&author=M.E.%20Weinblatt&author=N.A.%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

20. Patterson, N. , Price, A.L. & Reich, D. Population structure and Eigen analysis. PLoS Genet. 2, e190 (2006).  [DOI](https://doi.org/10.1371/journal.pgen.0020190) | [PMC free article](/articles/PMC1713260/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17194218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Population%20structure%20and%20Eigen%20analysis&author=N.%20Patterson&author=A.L.%20Price&author=D.%20Reich&volume=2&publication_year=2006&pages=e190&pmid=17194218&doi=10.1371/journal.pgen.0020190&)

21. Purcell, S. et al. PLINK: a tool set for whole‐genome association and population‐based linkage analyses. Am. J. Hum. Genet. 81, 559–575 (2007).  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=PLINK:%20a%20tool%20set%20for%20whole%E2%80%90genome%20association%20and%20population%E2%80%90based%20linkage%20analyses&author=S.%20Purcell&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

22. Price, A.L. et al. Long‐range LD can confound genome scans in admixed populations. Am. J. Hum. Genet. 83, 132–135 (2008), author reply 135–139.  [DOI](https://doi.org/10.1016/j.ajhg.2008.06.005) | [PMC free article](/articles/PMC2443852/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18606306/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Long%E2%80%90range%20LD%20can%20confound%20genome%20scans%20in%20admixed%20populations&author=A.L.%20Price&volume=83&publication_year=2008&pages=132-135&pmid=18606306&doi=10.1016/j.ajhg.2008.06.005&)

23. Zhu, Z. et al. Integration of summary data from GWAS and eQTL studies predicts complex trait gene targets. Nat. Genet. 48, 481–487 (2016).  [DOI](https://doi.org/10.1038/ng.3538) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27019110/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Integration%20of%20summary%20data%20from%20GWAS%20and%20eQTL%20studies%20predicts%20complex%20trait%20gene%20targets&author=Z.%20Zhu&volume=48&publication_year=2016&pages=481-487&pmid=27019110&doi=10.1038/ng.3538&)

24. GTEx Consortium . The Genotype‐Tissue Expression (GTEx) pilot analysis. Science 348, 648–660 (2015).  [DOI](https://doi.org/10.1126/science.1262110) | [PMC free article](/articles/PMC4547484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25954001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20Genotype%E2%80%90Tissue%20Expression%20(GTEx)%20pilot%20analysis&volume=348&publication_year=2015&pages=648-660&pmid=25954001&doi=10.1126/science.1262110&)

25. Cairns, J. , Ingle, J.N. , Wickerham, L.D. , Weinshilboum, R. , Liu, M. & Wang, L. SNPs near the cysteine proteinase cathepsin O gene (CTSO) determine tamoxifen sensitivity in ERalpha‐positive breast cancer through regulation of BRCA1. PLoS Genet. 13, e1007031 (2017).  [DOI](https://doi.org/10.1371/journal.pgen.1007031) | [PMC free article](/articles/PMC5638617/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28968398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=SNPs%20near%20the%20cysteine%20proteinase%20cathepsin%20O%20gene%20(CTSO)%20determine%20tamoxifen%20sensitivity%20in%20ERalpha%E2%80%90positive%20breast%20cancer%20through%20regulation%20of%20BRCA1&author=J.%20Cairns&author=J.N.%20Ingle&author=L.D.%20Wickerham&author=R.%20Weinshilboum&author=M.%20Liu&volume=13&publication_year=2017&pages=e1007031&pmid=28968398&doi=10.1371/journal.pgen.1007031&)

26. Hanson, C. , Cairns, J. , Wang, L. & Sinha, S. Principled multi‐omic analysis reveals gene regulatory mechanisms of phenotype variation. Genome Res. 28, 1207–1216 (2018).  [DOI](https://doi.org/10.1101/gr.227066.117) | [PMC free article](/articles/PMC6071639/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29898900/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Principled%20multi%E2%80%90omic%20analysis%20reveals%20gene%20regulatory%20mechanisms%20of%20phenotype%20variation&author=C.%20Hanson&author=J.%20Cairns&author=L.%20Wang&author=S.%20Sinha&volume=28&publication_year=2018&pages=1207-1216&pmid=29898900&doi=10.1101/gr.227066.117&)

27. Cairns, J. et al. A network‐based phenotype mapping approach to identify genes that modulate drug response phenotypes. Sci. Rep. 6, 37003 (2016).  [DOI](https://doi.org/10.1038/srep37003) | [PMC free article](/articles/PMC5107984/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27841317/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=A%20network%E2%80%90based%20phenotype%20mapping%20approach%20to%20identify%20genes%20that%20modulate%20drug%20response%20phenotypes&author=J.%20Cairns&volume=6&publication_year=2016&pages=37003&pmid=27841317&doi=10.1038/srep37003&)

28. Niu, N. et al. Radiation pharmacogenomics: a genome‐wide association approach to identify radiation response biomarkers using human lymphoblastoid cell lines. Genome Res. 20, 1482–1492 (2010).  [DOI](https://doi.org/10.1101/gr.107672.110) | [PMC free article](/articles/PMC2963812/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20923822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Radiation%20pharmacogenomics:%20a%20genome%E2%80%90wide%20association%20approach%20to%20identify%20radiation%20response%20biomarkers%20using%20human%20lymphoblastoid%20cell%20lines&author=N.%20Niu&volume=20&publication_year=2010&pages=1482-1492&pmid=20923822&doi=10.1101/gr.107672.110&)

29. Ingle, J.N. et al. Selective estrogen receptor modulators and pharmacogenomic variation in ZNF423 regulation of BRCA1 expression: individualized breast cancer prevention. Cancer Discov. 3, 812–825 (2013).  [DOI](https://doi.org/10.1158/2159-8290.CD-13-0038) | [PMC free article](/articles/PMC3710533/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23764426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov.&title=Selective%20estrogen%20receptor%20modulators%20and%20pharmacogenomic%20variation%20in%20ZNF423%20regulation%20of%20BRCA1%20expression:%20individualized%20breast%20cancer%20prevention&author=J.N.%20Ingle&volume=3&publication_year=2013&pages=812-825&pmid=23764426&doi=10.1158/2159-8290.CD-13-0038&)

30. Wang, W.Y. , Barratt, B.J. , Clayton, D.G. & Todd, J.A. Genome‐wide association studies: theoretical and practical concerns. Nat. Rev. Genet. 6, 109–118 (2005).  [DOI](https://doi.org/10.1038/nrg1522) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15716907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&title=Genome%E2%80%90wide%20association%20studies:%20theoretical%20and%20practical%20concerns&author=W.Y.%20Wang&author=B.J.%20Barratt&author=D.G.%20Clayton&author=J.A.%20Todd&volume=6&publication_year=2005&pages=109-118&pmid=15716907&doi=10.1038/nrg1522&)

31. Lachance, J. Disease‐associated alleles in genome‐wide association studies are enriched for derived low frequency alleles relative to HapMap and neutral expectations. BMC Med. Genomics 3, 57 (2010).  [DOI](https://doi.org/10.1186/1755-8794-3-57) | [PMC free article](/articles/PMC3017004/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21143973/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med.%20Genomics&title=Disease%E2%80%90associated%20alleles%20in%20genome%E2%80%90wide%20association%20studies%20are%20enriched%20for%20derived%20low%20frequency%20alleles%20relative%20to%20HapMap%20and%20neutral%20expectations&author=J.%20Lachance&volume=3&publication_year=2010&pages=57&pmid=21143973&doi=10.1186/1755-8794-3-57&)

32. Cancer Genome Atlas Network . Comprehensive molecular portraits of human breast tumours. Nature 490, 61–70 (2012).  [DOI](https://doi.org/10.1038/nature11412) | [PMC free article](/articles/PMC3465532/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23000897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Comprehensive%20molecular%20portraits%20of%20human%20breast%20tumours&volume=490&publication_year=2012&pages=61-70&pmid=23000897&doi=10.1038/nature11412&)

33. Ciriello, G. et al. Comprehensive molecular portraits of invasive lobular breast cancer. Cell 163, 506–519 (2015).  [DOI](https://doi.org/10.1016/j.cell.2015.09.033) | [PMC free article](/articles/PMC4603750/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26451490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Comprehensive%20molecular%20portraits%20of%20invasive%20lobular%20breast%20cancer&author=G.%20Ciriello&volume=163&publication_year=2015&pages=506-519&pmid=26451490&doi=10.1016/j.cell.2015.09.033&)

34. Yu, H. , Pardoll, D. & Jove, R. STATs in cancer inflammation and immunity: a leading role for STAT3. Nat. Rev. Cancer 9, 798–809 (2009).  [DOI](https://doi.org/10.1038/nrc2734) | [PMC free article](/articles/PMC4856025/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19851315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&title=STATs%20in%20cancer%20inflammation%20and%20immunity:%20a%20leading%20role%20for%20STAT3&author=H.%20Yu&author=D.%20Pardoll&author=R.%20Jove&volume=9&publication_year=2009&pages=798-809&pmid=19851315&doi=10.1038/nrc2734&)

35. Sullivan, N.J. et al. Interleukin‐6 induces an epithelial‐mesenchymal transition phenotype in human breast cancer cells. Oncogene 28, 2940–2947 (2009).  [DOI](https://doi.org/10.1038/onc.2009.180) | [PMC free article](/articles/PMC5576031/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19581928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Interleukin%E2%80%906%20induces%20an%20epithelial%E2%80%90mesenchymal%20transition%20phenotype%20in%20human%20breast%20cancer%20cells&author=N.J.%20Sullivan&volume=28&publication_year=2009&pages=2940-2947&pmid=19581928&doi=10.1038/onc.2009.180&)

36. Yadav, A. , Kumar, B. , Datta, J. , Teknos, T.N. & Kumar, P. IL‐6 promotes head and neck tumor metastasis by inducing epithelial‐mesenchymal transition via the JAK‐STAT3‐SNAIL signaling pathway. Mol. Cancer Res. 9, 1658–1667 (2011).  [DOI](https://doi.org/10.1158/1541-7786.MCR-11-0271) | [PMC free article](/articles/PMC3243808/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21976712/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Cancer%20Res.&title=IL%E2%80%906%20promotes%20head%20and%20neck%20tumor%20metastasis%20by%20inducing%20epithelial%E2%80%90mesenchymal%20transition%20via%20the%20JAK%E2%80%90STAT3%E2%80%90SNAIL%20signaling%20pathway&author=A.%20Yadav&author=B.%20Kumar&author=J.%20Datta&author=T.N.%20Teknos&author=P.%20Kumar&volume=9&publication_year=2011&pages=1658-1667&pmid=21976712&doi=10.1158/1541-7786.MCR-11-0271&)

37. Gong, C. et al. Abnormally expressed JunB transactivated by IL‐6/STAT3 signaling promotes uveal melanoma aggressiveness via epithelial‐mesenchymal transition. Biosci. Rep. 38(4), BSR20180532 (2018).  [DOI](https://doi.org/10.1042/BSR20180532) | [PMC free article](/articles/PMC6028753/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29899166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biosci.%20Rep.&title=Abnormally%20expressed%20JunB%20transactivated%20by%20IL%E2%80%906/STAT3%20signaling%20promotes%20uveal%20melanoma%20aggressiveness%20via%20epithelial%E2%80%90mesenchymal%20transition&author=C.%20Gong&volume=38&issue=4&publication_year=2018&pages=BSR20180532&pmid=29899166&doi=10.1042/BSR20180532&) | [Retracted](/articles/PMC10214088/)

38. Kalluri, R. & Weinberg, R.A. The basics of epithelial‐mesenchymal transition. J. Clin. Invest. 119, 1420–1428 (2009).  [DOI](https://doi.org/10.1172/JCI39104) | [PMC free article](/articles/PMC2689101/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19487818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest.&title=The%20basics%20of%20epithelial%E2%80%90mesenchymal%20transition&author=R.%20Kalluri&author=R.A.%20Weinberg&volume=119&publication_year=2009&pages=1420-1428&pmid=19487818&doi=10.1172/JCI39104&)

39. East, L. & Isacke, C.M. The mannose receptor family. Biochim. Biophys. Acta 1572, 364–386 (2002).  [DOI](https://doi.org/10.1016/s0304-4165(02)00319-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12223280/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta&title=The%20mannose%20receptor%20family&author=L.%20East&author=C.M.%20Isacke&volume=1572&publication_year=2002&pages=364-386&pmid=12223280&doi=10.1016/s0304-4165(02)00319-7&)

40. Regard, J.B. , Sato, I.T. & Coughlin, S.R. Anatomical profiling of G protein‐coupled receptor expression. Cell 135, 561–571 (2008).  [DOI](https://doi.org/10.1016/j.cell.2008.08.040) | [PMC free article](/articles/PMC2590943/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18984166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Anatomical%20profiling%20of%20G%20protein%E2%80%90coupled%20receptor%20expression&author=J.B.%20Regard&author=I.T.%20Sato&author=S.R.%20Coughlin&volume=135&publication_year=2008&pages=561-571&pmid=18984166&doi=10.1016/j.cell.2008.08.040&)
