# Genome‐wide Association Study Identified Chromosome 8 Locus Associated with Medication‐Related Osteonecrosis of the Jaw

## Metadata
**Authors:** Guang Yang, Sonal Singh, Caitrin W McDonough, Jatinder K Lamba, Issam Hamadeh, L Shannon Holliday, Danxin Wang, Joseph Katz, Peter A Lakatos, Bernadett Balla, Janos P Kosa, Gian Andrea Pelliccioni, Douglas K Price, Sara L Van Driest, William D Figg, Taimour Langaee, Jan S Moreb, Yan Gong
**Journal:** Clinical Pharmacology and Therapeutics
**Date:** 2021 Aug 31
**DOI:** [10.1002/cpt.2397](https://doi.org/10.1002/cpt.2397)
**PMID:** 34390503
**PMCID:** PMC8630710
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8630710/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8630710/pdf/CPT-110-1558.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8630710/pdf/CPT-110-1558.pdf)

## Abstract

Medication‐related osteonecrosis of the jaw (MRONJ) is a rare but serious drug‐related adverse event. To identify pharmacogenomic markers of MRONJ associated with bisphosphonate therapy, we conducted a genomewide association study (GWAS) meta‐analysis followed by functional analysis of 5,008 individuals of European ancestry treated with bisphosphonates, which includes the largest number of MRONJ cases to date (444 cases and 4,564 controls). Discovery GWAS was performed in randomly selected 70% of the patients with cancer and replication GWAS was performed in the remaining 30% of the patients with cancer treated with intravenous bisphosphonates followed by meta‐analysis of all 3,639 patients with cancer. GWAS was also performed in 1,369 patients with osteoporosis treated with oral bisphosphonates. The lead single‐nucleotide polymorphism (SNP), rs2736308 on chromosome 8, was associated with an increased risk of MRONJ with an odds ratio (OR) of 2.71 and 95% confidence interval (CI) of 1.90–3.86 (P = 3.57*10−8) in the meta‐analysis of patients with cancer. This SNP was validated in the MRONJ GWAS in patients with osteoporosis (OR: 2.82, 95% CI: 1.55–4.09, P = 6.84*10−4). The meta‐analysis combining patients with cancer and patients with osteoporosis yielded the same lead SNP rs2736308 on chromosome 8 as the top SNP (OR: 2.74, 95% CI: 2.09–3.39, P = 9.65*10−11). This locus is associated with regulation of the BLK, CTSB, and FDFT1 genes, which had been associated with bone mineral density. FDFT1 encodes a membrane‐associated enzyme, which is implicated in the bisphosphonate pathway. This study provides insights into the potential mechanism of MRONJ.

☑ Medication‐related osteonecrosis of the jaw (MRONJ) is a drug induced adverse reaction related to antiresorptive drugs, such as bisphosphonates (BPs) and RANKL inhibitor denosumab. The mechanisms of MRONJ is unclear.

☑ We conducted an international genome‐wide association study meta‐analysis followed by functional analyses to identify pharmacogenomic markers associated with an increased risk of BP induced MRONJ.

☑ We identified one locus on chromosome 8 to be significantly associated with increased risk of MRONJ in cancer patients treated with i.v. BPs and this locus was validated in patients with osteoporosis treated with oral BPs. This locus is associated with differential mRNA expression of multiple genes important to the bone‐remodeling pathways.

☑ Our study provides insights into potential mechanisms of MRONJ. If validated, our findings could provide basis for a Precision Medicine approach to antiresorptive therapy.

## INTRODUCTION

Medication‐related osteonecrosis of the jaw (MRONJ) is a major adverse drug reaction related to antiresorptive drugs, such as bisphosphonates (BPs), denosumab, and antiangiogenic medications.[1](#cpt2397-bib-0001) The American Association of Oral and Maxillofacial Surgeons (AAOMS) defined MRONJ as exposure of jawbone (mandible, maxilla, or both; no history of radiation therapy) with slow healing for more than 8 weeks or no healing.[2](#cpt2397-bib-0002) The exposure of jawbone increases the risk of inflammation/infection and progress to a fracture. Other risk factors of MRONJ include older age, female gender, inflammation/infection, autoimmune disease, and diabetes.[3](#cpt2397-bib-0003) BPs are the primary agents for the treatment or prevention of osteoclast‐mediated bone loss in patients with multiple myeloma (MM), solid metastatic cancers, or osteoporosis.[4](#cpt2397-bib-0004), [5](#cpt2397-bib-0005) MM and solid metastatic cancers, such as breast cancer, lung cancer, and prostate cancer, increase the activation of osteoclasts’ proliferation and differentiation, which commonly induce bone complications, skeletal‐related events, and bone destruction.[6](#cpt2397-bib-0006), [7](#cpt2397-bib-0007), [8](#cpt2397-bib-0008) Osteoporosis is a common age‐related disease, which involves bone loss and increased risk of fracture.[9](#cpt2397-bib-0009) The incidence of MRONJ ranges from 0.5–12% in patients with cancer, and 0.001–0.15% in patients with osteoporosis depending on the class of medications and doses.[1](#cpt2397-bib-0001), [10](#cpt2397-bib-0010), [11](#cpt2397-bib-0011)

A previous study showed that the incidences of MRONJ vary in genetically diverse populations.[5](#cpt2397-bib-0005) We previously performed a whole‐exome sequencing analysis followed by functional studies and identified a genetic variant (rs932658) in the promoter region of the *SIRT1* gene to be associated with the development of MRONJ.[12](#cpt2397-bib-0012), [13](#cpt2397-bib-0013) Only two MRONJ genomewide association studies (GWAS) have been published to date, and both studies were conducted in patients with cancer treated with i.v. BPs.[14](#cpt2397-bib-0014), [15](#cpt2397-bib-0015) Sarasquete *et al*. published the first MRONJ GWAS, which included 87 patients with MM (22 cases and 65 controls) treated with i.v. BP (pamidronate or zoledronate).[14](#cpt2397-bib-0014) This study identified a *CYP2C8* single nucleotide polymorphism (SNP; rs1934951) to be associated with MRONJ (odds ratio (OR): 12.75, 95% confidence interval (CI): 3.7–43.5, Bonferroni corrected *P* value = 0.02). Nicoletti *et al*. published another MRONJ GWAS in 2012, which included 47 BP‐treated patients with breast cancer (30 with MRONJ) and an additional 1,726 healthy general population controls.[15](#cpt2397-bib-0015) SNP rs17024608 in the *RBMS3* gene was identified to be significantly associated with MRONJ (OR: 5.8, 95% CI: 3.0–11.0, *P* = 7.47*10^−8^). However, the major limitations of these two studies include small number of cases, lack of medication treated control participants, and lack of replication. Furthermore, multiple subsequent independent studies have failed to replicate the association of these two loci.[16](#cpt2397-bib-0016), [17](#cpt2397-bib-0017) There has been no GWAS published yet on MRONJ in patients with osteoporosis treated with oral BPs.

The objective of this study was to identify and replicate common genetic biomarkers associated with MRONJ in BP‐treated individuals through a GWAS approach followed by *in silico* functional analyses. In this GWAS with the largest number of MRONJ cases to date, discovery and replication analyses were performed in patients with cancer treated with i.v. BPs and in patients with osteoporosis treated with oral BPs, followed by meta‐analyses.

## MATERIALS AND METHODS

### Data collection

The participants included in this study were from multiple institutions, including the University of Florida in Gainesville, FL, USA; the National Cancer Institute (NCI) in Bethesda, MD, USA; Semmelweis University of Medical School and Dental School in Budapest, Hungary, and the University of Bologna, Italy. Additional data were obtained from the international Serous Adverse Event Consortium (iSAEC)[18](#cpt2397-bib-0018) and the Vanderbilt University Medical Center (VUMC) biobank (BioVU), which associates DNA and genomic data with de‐identified electronic health records data.[19](#cpt2397-bib-0019) The BioVU portion of the study was reviewed by the VUMC Institutional Review Board and determined to be non‐human subjects research (**Table ** [S1](#cpt2397-sup-0001)). All participants signed informed consent at each study site. This study was approved by the Institutional Review Board (IRB201800934) at the University of Florida.

### Case and control definition

The MRONJ cases were patients who were treated with BPs and developed osteonecrosis of the jaw (ONJ) based on the criteria for MRONJ according to the AAOMS definition as exposure of jawbone (mandible, maxilla, or both; no history of radiation therapy) with slow healing for more than 8 weeks or no healing.[2](#cpt2397-bib-0002) The non‐MRONJ controls were patients who were treated with BPs for at least 12 months but did not develop ONJ. All study sites defined MRONJ cases and control based on the same AAOMS definition criteria.

### Genotyping and imputation

The iSAEC samples were genotyped using Illumina HumanOmniExpressExome‐8 version 1.0 kit (Illumina, San Diego, CA), which contains ~ 1 million SNPs. The BioVU samples were genotyped using the Illumina Multi‐Ethnic Global Array (MEGA; Illumina; ~ 1.7 million SNPs). The other samples were genotyped on Illumina HumanOmni2.5S‐8 version 1 (Illumina; ~ 2.5 million SNPs) or Illumina Infinium Global Screening Array version 2 (Illumina; ~ 0.65 million SNPs; **Table ** [S2](#cpt2397-sup-0001)). Quality controls were performed for each of the four datasets separately. SNPs common to all GWAS panels were extracted and the four datasets were then merged. Genotype imputation was performed on the merged dataset using 1000 Genomes phase III version 5 reference panel. The detailed quality control steps for all raw data and the imputation steps were descripted in Supplementary Methods.

### Genomewide association studies analyses

Discovery GWAS was performed in randomly selected 70% of the patients with cancer and replication GWAS was performed in the remaining 30% of the patients with cancer treated with i.v. BPs followed by meta‐analysis of all 3,639 patients with cancer. GWAS was also performed in 1,369 patients with osteoporosis treated with oral BPs. Multivariable logistic regression was performed to estimate the ORs and 95% CIs of each variant for the development of ONJ (yes/no) using EPACTS ([http://genome.sph.umich.edu/wiki/EPACTS](http://genome.sph.umich.edu/wiki/EPACTS)) adjusting for age, gender, principal components PC1‐PC3, and cancer types (in GWAS of patients with cancer). The results were filtered by effect allele frequency (EAF) > 0.01. Meta‐analysis was then performed using R (sommer package)[20](#cpt2397-bib-0020) and/or Meta Analysis Helper (METAL)[21](#cpt2397-bib-0021) to estimate the combined ORs for all independent studies. The *I* ^2^ statistics and Cochran *Q* test were used to assess the heterogeneity of the studies in the meta‐analyses. The random‐effects model meta‐analysis was used if the inconsistency index *I* ^2^ > 50% and fix‐effect model meta‐analysis was used if the *I* ^2^ < 50%. SNPs with *P* < 5*10^−8^ were considered as genomewide significant, whereas SNPs with *P* < 10^−5^ were considered as suggestive.

### Gene‐based association analysis

Gene‐set association analysis was performed using Multi‐marker analysis of genomic annotation (MAGMA)[22](#cpt2397-bib-0022) through the Functional Mapping and Annotation of Genome‐Wide Association Studies (FUMA GWAS) platform[23](#cpt2397-bib-0023) based on our GWAS data. The MAGMA was to detect the multi‐marker effects of SNPs based on *P* values and linkage disequilibrium (LD) between markers using a multiple regression approach. Our input SNPs were mapped to 17,529 protein coding genes. Genomewide significance was defined at *P* = 0.05/17,529 = 2.852*10^−6^.

### Enrichment analysis

Pathway and biological enrichment analyses were performed using the FUMA platform,[23](#cpt2397-bib-0023) with independent genomewide significant SNPs identified. Positional gene mapping was used to identify genes up to 10 kb from each independent significant SNP.

### Topologically associating domains analysis

The 3D Genome Browser platform[24](#cpt2397-bib-0024) was used to identify the topologically associating domains of our target SNPs. The Hi‐C data was performed to identify the interaction between the target region and multiple genes. The human Mesenchymal stem cells (H‐MSCs) was selected for our Hi‐C analysis.

### GWAS 4D analysis

The SNPs with *P* value < 5*10^−8^ from meta‐analysis among cancer and osteoporosis cohorts were analyzed by GWAS4D[25](#cpt2397-bib-0025) ([http://mulinlab.tmu.edu.cn/gwas4d](http://mulinlab.tmu.edu.cn/gwas4d)). GWAS4D detects human regulatory variants by integrative analysis of genomewide associations, chromosome interactions, and histone modifications. The top SNPs in the significant loci and suggestive loci were entered into the GWAS4D to identify functional SNPs.

### In silico analyses

All SNPs of interest and the related gene were also analyzed using the UCSC genome browser, TWAS hub, ENCODE, Ensembl, regulomeDB and Open Target Platform to identify the functional significance of SNPs and genes.

## RESULTS

We performed a GWAS of MRONJ in a total of 5,008 individuals of European ancestry (including 444 MRONJ cases and 4,564 controls) who were treated with BPs (oral: alendronate or risedronate; or i.v.: zoledronate or pamidronate) for at least 12 months. GWAS of the patients with cancer treated with i.v. BPs and patients with osteoporosis treated with oral BPs were performed separately. To perform MRONJ GWAS in patients with cancer treated with i.v. BPs, we randomly selected 70% of the cancer patients as discovery (217 cases and 2,208 controls) and the remaining 30% as replication (109 cases and 1,105 controls) followed by meta‐analysis in patients with cancer. We also performed a GWAS in 1,369 patients with osteoporosis treated with oral BPs included 118 MRONJ cases and 1,251 controls. Another meta‐analysis was performed to identify SNPs associated with MRONJ in patients with cancer and patients with osteoporosis combined. The study flow diagram is shown in **Figure ** [1](#cpt2397-fig-0001). The characteristics of the patients are summarized in the **Table ** [S3](#cpt2397-sup-0001). The Manhattan plots and quantile‐quantile plots for each study are shown in **Figure ** [S1](#cpt2397-sup-0001).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d7a9/9291811/f1062bc1a7dc/CPT-110-1558-g006.jpg)

Flowchart of GWAS and meta‐analyses. GWAS, genomewide association studies.

### MRONJ in patients with cancer

The discovery GWAS in patients with cancer identified two loci at the suggestive level of significance (*P* < 10^−5^). The top SNP rs72817334 in an intergenic region on chromosome 2 was associated with an increased risk of MRONJ (OR: 2.27, 95% CI: 1.64–3.14, *P* = 7.33*10^−7^). The top SNP rs11948737 in an intergenic region on chromosome 5 was associated with an increased risk of MRONJ (OR: 2.76, 95% CI: 1.83–4.17, *P* = 1.51*10^–6^). However, these two SNPs were not replicated in cancer replication GWAS analysis (*P* > 0.05). The meta‐analysis of the discovery and replication cohort in patients with cancer identified one genomewide significant (*P* < 5*10^−8^) locus at chromosome 8p23.1 to be associated with MRONJ in patients with cancer treated with i.v. BPs. The minor allele C of the lead SNP rs2736308 on chromosome 8 located in the intronic region of *C8orf12* (also known as *FAM167A‐AS1*) was associated with an increased risk of MRONJ with an OR of 2.71, 95% CI of 1.90–3.86 (*P* = 3.57*10^−8^; **Table ** [1](#cpt2397-tbl-0001), **Figure ** [2a](#cpt2397-fig-0002)). According to the eQTLGen[26](#cpt2397-bib-0026) and GTEx[27](#cpt2397-bib-0027) databases, rs2736308 is an expression quantitative trait locus (eQTL) for multiple genes including B lymphocyte kinase (*BLK*), cathepsin B (*CTSB*), Family With Sequence Similarity 167 Member A (*FAM167A*), FAM167A Antisense RNA 1 (*FAM167A‐AS1*; also named *C8orf12*), Farnesyl‐Diphosphate Farnesyltransferase 1 (*FDFT1*), Myotubularin Related Protein 9 (*MTMR9*), Nei Like DNA Glycosylase 2 (*NEIL2*), Solute Carrier Family 35 Member G5 (*SLC35G5*), L‐Threonine Dehydrogenase (*TDH*), and XK Related 6 (*XKR6*; **Table ** [S4](#cpt2397-sup-0001)).[27](#cpt2397-bib-0027), [28](#cpt2397-bib-0028), [29](#cpt2397-bib-0029), [30](#cpt2397-bib-0030) We identified 26 LD blocks in this locus based on the LD information (r^2^ > 0.8) and the lead SNPs for each block are listed in **Table ** [S5](#cpt2397-sup-0001).

### Table 1.

| Chr | Position (hg19) | SNP | Ref /Alt | EA | EAF | 1000 G EUR | Meta‐stage | OR | 95% CI | P value | Annotated genes | Direction |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 8 | 11248500 | rs2736308 | C/T | C | 0.34 | 0.35 | Cancer only | 2.71 | 1.90–3.86 | 3.57*10−8 | BLK/FDFT1/SLC35G5/FAM167A/FAM167A‐AS1/NEIL2/MTMR9/CTSB/TDH/XKR6 | ** |
|   | Cancer and osteoporosis | 2.74 | 2.09–3.39 | 9.65*10−11 |  |  |  |  |  |  |  |  |
| 6 | 34104970 | rs2029462 | T/A | T | 0.44 | 0.43 | Cancer only | 0.62 | 0.48–0.79 | 1.72*10−4 | GRM4 | ** |
|   | Cancer and osteoporosis | 0.60 | 0.49–0.75 | 4.15*10−6 |  |  |  |  |  |  |  |  |
| 4 | 182447323 | rs974578 | T/C | T | 0.29 | 0.25 | Cancer only | 2.14 | 1.55–2.96 | 3.93*10−6 | TENM3 | * |
|   | Cancer and osteoporosis | 1.79 | 1.35–2.36 | 5.01*10−5 |  |  |  |  |  |  |  |  |
| 1 | 108030093 | rs599873 | T/C | C | 0.43 | 0.44 | Cancer only | 1.71 | 1.35–2.16 | 8.37*10−6 | NTNG1/VAV3 | * |
|   | Cancer and osteoporosis | 1.48 | 1.20–1.83 | 2.73*10−4 |  |  |  |  |  |  |  |  |

Table 1 Caption: Lead SNPs in loci with genomewide or suggestive level of significance

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d7a9/9291811/ae8a43d6631f/CPT-110-1558-g001.jpg)

Manhattan plots of GWAS meta‐analyses. (a) The meta‐analysis in patients with cancer identified one locus on chromosome 8 (C8orf12) to be significantly associated with MRONJ (P = 3.57*10−8). Two other loci (TENM3 on chromosome 4 and VAV3 on chromosome 1) were associated with MRONJ at suggestive level of significance (P < 10−5). (b) The meta‐analysis of patients with cancer and patients with osteoporosis validated the chromosome 8 locus to be significantly associated with MRONJ on a genomewide level (P = 9.65*10−11). One novel locus on chromosome 6 (GRM4) reached suggestive level of significance. GWAS, genomewide association study; MRONJ, medication‐related osteonecrosis of the jaw.

Two loci were associated with MRONJ in patients with cancer at suggestive level of significance (*P* < 10^−5^). The minor allele T of the chromosome 4 top SNP rs974578 at 4q34.3 in the intronic region of Teneurin Transmembrane Protein 3 (*TENM3*) was associated with an increased risk of MRONJ (OR: 2.14, 95% CI: 1.55–2.96, *P* = 3.93*10^−6^). The top SNP rs599873 on chromosome 1p13, an intergenic SNP between Netrin G1 (*NTNG1*) and Vav Guanine Nucleotide Exchange Factor 3 (*VAV3*), was associated with an increased risk of MRONJ (OR: 1.71, 95% CI: 1.35–2.16, *P* =8.37*10^−6^; **Figure ** [2a](#cpt2397-fig-0002), **Table ,** [1](#cpt2397-tbl-0001) **Table ** [S4](#cpt2397-sup-0001)).

The pharmacogenetic marker on the *SIRT1* gene identified in our previous study, rs932658, was also replicated in the meta‐analysis in patients with cancer treated with i.v. BPs, with OR of 0.75, 95% CI: 0.61‐0.92 (*P* = 0.006).

### MRONJ in patients with osteoporosis

The GWAS in the patients with osteoporosis did not yield any loci with genomewide significance, with three loci reaching suggestive level of significance **(** **Figure ** [S1](#cpt2397-sup-0001) **e,f**). The minor allele G of the chromosome 3 top SNP rs13072463 in the intergenic region was associated with an increased risk of MRONJ (OR: 2.99, 95% CI: 1.86–4.80, *P* = 6.01*10^−6^). The top SNP rs599873 on chromosome 5, an intronic region of *RAB3C* (RAB3C, Member RAS Oncogene Family), was associated with an increased risk of MRONJ (OR: 2.37, 95% CI: 1.64–3.43, *P* = 4.30*10^−6^). The top SNP rs111352217 on chromosome 7, an intergenic region SNP, was associated with an increased risk of MRONJ (OR: 7.23, 95% CI: 3.04–17.19, *P* = 7.60*10^−6^). In addition, the top SNP from the MRONJ GWAS meta‐analysis in the patients with cancer treated with i.v. BPs, rs2736308, had a *P* value of 6.84*10^−4^ in the patients with osteoporosis treated with oral BPs.

### Meta‐analysis in patients with cancer and patients with osteoporosis

The meta‐analysis combining patients with cancer and osteoporosis yielded the same lead SNP rs2736308 on chromosome 8 as the top SNP (OR: 2.74, 95% CI: 2.09–3.39, *P* = 9.65*10^−11^). In addition, one new locus was identified at suggestive level of significance on chromosome 6. The lead SNP rs2029462 on chromosome 6p23.21 in the intronic region of *GRM4* gene (Glutamate Metabotropic Receptor 4) was associated with a decreased risk of MRONJ (OR: 0.60, 95% CI: 0.49–0.75, *P* = 4.15*10^−6^). The two suggestive level loci in cancer meta‐analysis were not replicated in the patients with osteoporosis (*P* = 0.91 for rs974578 and *P* = 0.63 for rs599873; **Figure ** [2b](#cpt2397-fig-0002), **Figure ** [3](#cpt2397-fig-0003), **Table ** [1](#cpt2397-tbl-0001)). The regional plots of the top SNPs of these loci are shown in **Figure ** [S2](#cpt2397-sup-0001).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d7a9/9291811/9da647fe8e5b/CPT-110-1558-g002.jpg)

Forest plots for the top SNPs in the meta‐analysis of patients with cancer and patients with osteoporosis. We performed the multiple logistic regression adjusted for age, gender, PCs to calculate the odds ratio (OR) of each SNP. The I 2 was calculated to estimate the heterogeneity. The random effects model meta‐analysis was used if the I 2 > 50% and fix‐effect model meta‐analysis was used if the I 2 < 50%. CI, confidence interval; MAF, minor allele frequency; OR, odds ratio; SNP, single‐nucleotide polymorphism.

### Pathway and biological enrichment analyses

To identify the genes related to the top SNP, we performed gene‐based association analyses using MAGMA by the FUMA GWAS.[23](#cpt2397-bib-0023) Genes *XKR6*, *BLK*, *C8orf12*, *AF131215.5*, *MSRA*, *SLC35G5*, *FAM167A*, and *MTMR9* on chromosome 8 were identified as genomewide significant genes associated with MRONJ with *P* < 2.85*10^−6^ (**Figure ** [4](#cpt2397-fig-0004), **Table ** [S6](#cpt2397-sup-0001)). The *FAM167A*, *FDFT1*, *BLK*, *MSRA*, *NEIL2*, *SLC35G5*, and *CTSB* genes were identified as the strongest genes linked to the locus (8p23.1) by FUMA GWAS platform **(** **Table ** [S7](#cpt2397-sup-0001) **)**.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d7a9/9291811/761ff73aa58c/CPT-110-1558-g005.jpg)

Functional mapping and annotation of genomewide association studies. XKR6, BLK, C8orf12, AF131215.5, MSRA, SLC35G5, FAM167A, and MTMR9 as the genomewide significant genes associated with the significant locus identified in the GWAS meta‐analysis. The red dash line in the plot is the genomewide significance, which was defined at P = 0.05/17529 = 2.852*10−6. GWAS, genomewide association study.

The minor allele C of the lead SNP on chromosome 8 rs2736308 was associated with lower expression of *FAM167A*, *FDFT1*, and *NEIL2*, but higher expression of *BLK*, *MSRA*, *CTSB*, and *SLC35G5* based on the eqtlDirection (**Table ** [S7](#cpt2397-sup-0001)). The circos plot showed multiple chromatin interactions between the genomic risk locus (chr8:10803465–11401116) and genes *FAM167A*, *FDFT1*, *BLK*, *MSRA*, *NEIL2*, *SLC35G5*, and *CTSB* (**Figure ** [5a](#cpt2397-fig-0005)). The gene expression heatmap by GTEx V8 dataset showed that *CTSB* and *FDFT1* had a higher mRNA expression compared with other related genes **(** **Figure ** [S3](#cpt2397-sup-0001) **)**. The 6p23.21 locus was identified to have chromatin interactions with multiple genes (**Figure ** [5b](#cpt2397-fig-0005)).

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d7a9/9291811/b37ff5d46617/CPT-110-1558-g004.jpg)

FUMA circos plot for the genomewide level locus. Chromatin interaction between genomic risk locus (blue arc) and genes were showed by orange line. The green color line linked the top SNP and the eQTL genes. (a) Circos plot for chromosome 8 top SNP. (b) Circos plot for chromosome 6 top SNP. (c) Circos plot for chromosome 1 top SNP. (d) Circos plot for chromosome 4 top SNP. eQTL, expression quantitative trait locus; FUMA, Functional Mapping and Annotation; SNP, single‐nucleotide polymorphism.

The lead SNP rs599873 of chromosome 1 locus is an intergenic SNP between *NTNG1* and *VAV3* genes. No interaction between this locus and genes was identified by chromatin interactions analysis (**Figure ** [5c](#cpt2397-fig-0005)). The lead SNP rs974578 of chromosome 4 locus interacted with the *DCTD* gene (**Figure ** [5d](#cpt2397-fig-0005)).

### Topologically associated domains analysis

The 3D Genome browser[24](#cpt2397-bib-0024), [31](#cpt2397-bib-0031) predicted that there are seven topologically associating domains surrounding the chromosome 8 locus region (chr8:10803465–11401116; **Figure ** [6](#cpt2397-fig-0006)). This locus included the peak of H3k4me3 (a marker associated with the activation of transcription of nearby genes), the peak of H3K4me1 (an enhancer mark), and the peak of H3K9me3 and H3K27me3 (two marks associated with heterochromatin). The circular chromosomal conformation capture (4C) plot using 3D Genome browser and the UCSC genome browser[32](#cpt2397-bib-0032) further corroborates that there are long‐range promoter enhancer interactions between the region around the lead SNP and other genes, such as *MTMR9*, *BLK*, and *FAM167A* in H‐MSC (**Figure ** [6](#cpt2397-fig-0006)). The Hi‐C data also showed that the chromosome 6 locus (chr6:34092806–34109404) was associated with the peaks of H3k9me3, and H3k27me3 in H‐MSC. The 4C plot data using 3D Genome browser and the UCSC genome browser showed SNP rs2029462 was interacted with multiple genes, including the *GRM4* gene (**Figure ** [S4](#cpt2397-sup-0001)).

### Figure 6.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d7a9/9291811/dff43edefb18/CPT-110-1558-g003.jpg)

Summary of Hi‐C data of the top locus on chromosome 8. Seven Topologically associated domains (TADs) surrounding the chromosome 8 locus region (chr8:10803465–11401116) (a). Circular chromosomal conformation capture (4C) plot measures the interaction frequencies between rs2736308 and other loci (peak signal). The plot showed that there is a chromatin interaction event between rs2736308 and MTMR9, FAM167A, and BLK in human mesenchymal stem cell (H‐MSC). The anchoring point is the interesting SNP position (b). SNP, single‐nucleotide polymorphism.

The Hi‐C data of the chromosome 4 locus (chr4:182367808–182450209) reported a CpG island, the peak of H3M27me3, and H3K4me3 **(** **Figure ** [S4](#cpt2397-sup-0001) **c**,**d**). There was no evidence that this locus had any significant chromatin interaction with protein‐coding genes. The top SNP of chromosome 1 locus had interactions with the *NTNG1* and *VAV3* gene (**Figure ** [S4](#cpt2397-sup-0001) **e**,**f**). The 4C data further corroborates that there are long‐range promoter enhancer interactions between the region around the lead SNP and genes *NTNG1*, and *VAV3* in H‐MSC (**Figure ** [S4](#cpt2397-sup-0001)).

### GWAS4D

GWAS4D ([http://mulinlab.tmu.edu.cn/gwas4d](http://mulinlab.tmu.edu.cn/gwas4d))[25](#cpt2397-bib-0025) is an *in silico* tool that evaluates GWAS results and identifies potential regulatory variants by integrating the latest multidimensional functional genomics resources and the published algorithms by Mulinlab. The lead SNP rs34128921 of block 3 (OR: 2.33, 95% CI: 2.02–2.70, *P* = 3.60*10^−9^) on chromosome 8 has a significant Hi‐C interaction with the *BLK* gene (**Figure ** [S5](#cpt2397-sup-0001)). The GWAS4D predicted that this SNP was associated with decreasing transcription factor binding affinity (THAP1, MLLT1, and CUX1; **Figure ** [S5](#cpt2397-sup-0001), **Table ** [S8](#cpt2397-sup-0001)). SNP rs76154097 in block 15 (the lead SNP is rs77072957) on chromosome 8 also has a significant Hi‐C interaction with the *BLK* gene (**Figure ** [S5](#cpt2397-sup-0001) **c**). SNPs rs1600252 (**Figure ** [S5](#cpt2397-sup-0001) **d**) in block 8 showed a significant Hi‐C interaction with the *CTSB* gene. SNPs rs2248699 (**Figure ** [S5](#cpt2397-sup-0001) **e**) in block 20. GWAS4D analysis also identified the SNP rs11753307 to be associated with *GRM4* gene regulation (**Figure ** [S5](#cpt2397-sup-0001) **f**). The GWAS4D of chromosome 6 locus showed this locus to have Hi‐C interaction with multiple genes.

### Phenome Wide Association Studies

The phenome wide association studies (PheWAS) of the lead SNPs using the UK Biobank dataset based on the data of Neale lab ([http://www.nealelab.is](http://www.nealelab.is)) showed alternative allele T of the top SNP rs2736308 was associated with higher bone mineral density (BMD; **Figure ** [S6](#cpt2397-sup-0001) **)**.

### Transcriptome‐wide association study analysis

The transcriptome‐wide association study (TWAS)[32](#cpt2397-bib-0032) identified the *BLK* gene to be negatively associated with heel BMD in blood tissue (**Table ** [S9](#cpt2397-sup-0001)). The *FDFT1* gene was positively correlated with heel BMD in blood tissue (**Table ** [S9](#cpt2397-sup-0001)). The *CTSB* gene was negatively associated with heel BMD (**Table ** [S9](#cpt2397-sup-0001)).

### GeneMANIA pathway analysis

All the above *in silico* analyses identified *BLK*, *CTSB*, *FDFT1*, *GRM4*, *NTNG1*, and *VAV3* genes to be important genes associated with MRONJ. The GeneMANIA[33](#cpt2397-bib-0033) pathway analysis showed that the *VAV3*, *BLK*, *FDFT1*, and *CTSB* genes interact through epidermal growth factor receptor (EGFR) and *SLC25A24*. These genes are associated with small GTPases or initial immune system pathway (**Figure ** [S7](#cpt2397-sup-0001)).

## DISCUSSION

In this GWAS meta‐analysis that included the largest number of MRONJ cases to date, we identified and validated pharmacogenomic markers of MRONJ in patients with cancer treated with i.v. BPs. We also performed the first MRONJ GWAS in patients with osteoporosis treated with oral BPs. One locus on chromosome 8 was identified to be significantly associated with MRONJ at the genomewide level in patients with cancer treated with i.v. BPs and was validated in patients with osteoporosis treated with oral BPs. The minor allele C of the top SNP rs2736308 was associated with an increased risk of MRONJ with an OR of 2.74 (CI, 2.09–3.39, *P* = 9.65*10^−11^) in the combined meta‐analysis of patients with cancer and patients with osteoporosis. The *in silico* analyses and pathway analyses provided evidence that the minor allele C of rs2736308 was associated with lower expression of *FDFT1* and *CTSB* and higher expression of *BLK* gene.

The *FDFT1* gene encodes squalene synthase which is a membrane‐associated enzyme in the mevalonate pathway. This pathway generates cholesterol in cells, and side pathways produce farnesyl and geranylgeranyl groups that are attached to small GTPase, crucial regulators of membrane trafficking, and the cytoskeleton.[34](#cpt2397-bib-0034), [35](#cpt2397-bib-0035) BPs are incorporated into bone, and released as bone is resorbed by osteoclasts and are internalized into osteoclasts.[36](#cpt2397-bib-0036) The nitrogen‐containing BPs, such as zoledronate and pamidronate, inhibit farnesyl diphosphate synthase, a key enzyme in the mevalonate pathway.[37](#cpt2397-bib-0037) This prevents the synthesis of geranylgeranyl diphosphate in the downstream of the mevalonate pathway and blocks the prenylation of small GTPases. This disrupts normal osteoclast function and induces apoptosis of osteoclast cells.[12](#cpt2397-bib-0012), [38](#cpt2397-bib-0038) Lower expression of squalene synthase might decrease the competitive inhibition of the activation of small GTPases and thus modify the effects of nitrogen‐containing BPs on osteoclasts. This could alter the propensity for the development of MRONJ. Our proposed mechanism for MRONJ based on the observed association is presented in **Figure ** [S8](#cpt2397-sup-0001).

The *BLK* gene encodes a nonreceptor tyrosine‐kinase of the src family of proto‐oncogenes. This kinase stimulates the insulin synthesis and secretion in response to glucose and also increases the expression of several pancreatic beta‐cell transcription factors.[39](#cpt2397-bib-0039) *BLK* deficiency was shown to be associated with diabetes, which is a risk factor of MRONJ.[10](#cpt2397-bib-0010) The *BLK* gene was also associated with inflammation/infection responses by stimulating the secretion of metalloproteinase‐9 (MMP‐9),[40](#cpt2397-bib-0040) which was reported to be associated with rheumatoid arthritis, atherosclerosis, cancer, diabetes, obesity, and osteoporosis.[41](#cpt2397-bib-0041) AAOMS have reported the inflammation/infection as a risk factor of MRONJ.[1](#cpt2397-bib-0001)

The *CTSB* gene has also been observe to be involved in this macrophage‐mediated inflammation/infection responses. Álvaro de Mingo *et al*. reported that the *CTSB* gene regulates the inflammation responses by regulating the expression of *SIRT1*, which was identified to be associated with MRONJ.[12](#cpt2397-bib-0012), [13](#cpt2397-bib-0013), [42](#cpt2397-bib-0042)

Although the *GRM4* locus on chromosome 6 and *VAV3* locus on chromosome 1 did not reach genomewide significance, we believe these two loci merit discussion. The *GRM4* gene on chromosome 6 is expressed in bone cells, modulated osteoblast differentiation, and activity,[43](#cpt2397-bib-0043) and plays an important role in bone remodeling and homeostasis.[44](#cpt2397-bib-0044) The deficiency in GRM4 was shown to increase the risk of cancer metastasis to the bone[45](#cpt2397-bib-0045) and cancer bone metastasis was associated with higher risk of MRONJ.[1](#cpt2397-bib-0001) The *VAV3* gene encodes the Vav protein, which belongs to the family of mammalian Rho Guanine nucleotide exchange factors. The Rho Guanine nucleotide exchange factors were associated with activation of the Rho GTPases. Rho family of GTPases plays an essential role in osteoclast differentiation. VAV3 is most abundant in osteoclasts during differentiation, while the mature osteoblasts have low VAV3 expression levels. *VAV3* overexpression in osteoclasts stimulates osteoclasts differentiation and bone resorption, leading to bone loss.[46](#cpt2397-bib-0046)

Our pathway analysis result suggests that *VAV3*, *BLK*, *FDFT1*, and *CTSB* interact with each other through *EGFR* and *SLC25A24*. Karin Writzl *et al*. demonstrated that *SLC25A24* regulated mitochondrial ATP synthesis and hyperpolarization.[47](#cpt2397-bib-0047) EGFR plays an essential role in osteoblast cell proliferation, differentiation, and survial.[48](#cpt2397-bib-0048) The osteoblast‐specific deletion of *EGFR* was associated with bone defects[49](#cpt2397-bib-0049) and low expression or lack of *EGFR* induced low BMD.[49](#cpt2397-bib-0049)

The PheWAS analysis for the target SNPs across all UK Biobank phenotypes released by Neale and colleagues indicated that the alternative allele T of the lead SNP rs2736308 was associated with an increased BMD. So, the minor allele C was associated with a decreased BMD of heel bone, which is positively associated with jawbone BMD[50](#cpt2397-bib-0050) (**Figure ** [S4](#cpt2397-sup-0001)). This is consistent with the TWAS result. Therefore, it is plausible that the minor allele C of the lead SNP rs2736308 on chromosome 8 is associated with lower BMD by regulating the expression of *FDFT1*, *CTSB*, and *BLK*, which may influence BMD through interaction with EGFR.

Our study has some limitations that need to be recognized. First, due to the imbalance of case and controls across different datasets, we combined all patients with cancer and randomly selected 70% for discovery and 30% for replication. Second, different datasets were genotyped using different GWAS platforms and only 126,146 variants were common across different platforms and were included in the GWAS genotype imputation. It is not clear how this might have affected the imputation quality. Last, there were not enough patients of other ancestries, we only focused on patients of European ancestry. Therefore, our findings might not be generalizable to individuals of other ancestries.

In summary, we identified a genomewide significant locus on chromosome 8 to be associated with higher odds of MRONJ in patients with cancer treated with i.v. BPs and this locus was validated in patients with osteoporosis treated with oral BPs. This was made possible by the unprecedented size of this international study. This locus is associated with differential mRNA expression of multiple genes important to the bone‐remodeling pathways. We also identified three loci at suggestive level of significance that may merit further investigation. Our study provides insights into potential mechanisms of MRONJ. If validated, our findings could provide basis for a Precision Medicine approach to antiresorptive therapy.

## FUNDING

Y.G. is supported by University of Florida College or Pharmacy, University of Florida Health Cancer Center, University of Florida Informatics Institute, and University of Florida Clinical and Translational Science Institute, which is supported in part by the NIH National Center for Advancing Translational Sciences under award number UL1 TR001427. Some of the dataset(s) used for the analyses described were obtained from Vanderbilt University Medical Center’s BioVU which is supported by numerous sources: institutional funding, private agencies, and federal grants. These include the NIH‐funded Shared Instrumentation Grant S10RR025141; and CTSA grants UL1TR002243, UL1TR000445, and UL1RR024975. Genomic data are also supported by investigator‐led projects that include U01HG004798, R01NS032830, RC2GM092618, P50GM115305, U01HG006378, U19HL065962, R01HD074711; and additional funding sources listed at https://victr.vumc.org/biovu‐funding/.

## CONFLICT OF INTEREST

The authors declared no competing interests for this work.

## AUTHOR CONTRIBUTIONS

G.Y. wrote the manuscript. G.Y., S.S., J.L., C.M., I.H., L.S.H., D.W., T.L., J.K., P.L., B.B., J.K., S.V.D., D.P., W.F., G.P., J.M., and Y.G. performed the research. G.Y., C.M., and Y.G. analyzed the data. Y.G. and G.Y. designed the research.

## DISCLAIMER

As an Associate Editor of *Clinical Pharmacology & Therapeutics*, Dr. Sara L. Van Driest was not involved in the review of decision process for this paper.

## Supporting information

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Ruggiero, S.L. et al. American Association of Oral and Maxillofacial Surgeons position paper on medication‐related osteonecrosis of the jaw–2014 update. J. Oral Maxillofac. Surg. 72:1938–1956 (2014).  [DOI](https://doi.org/10.1016/j.joms.2014.04.031) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25234529/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Oral%20Maxillofac.%20Surg.&title=American%20Association%20of%20Oral%20and%20Maxillofacial%20Surgeons%20position%20paper%20on%20medication%E2%80%90related%20osteonecrosis%20of%20the%20jaw%E2%80%932014%20update&author=S.L.%20Ruggiero&volume=72&publication_year=2014&pages=1938-1956&pmid=25234529&doi=10.1016/j.joms.2014.04.031&)

2. Advisory Task Force on Bisphosphonate‐Related Ostenonecrosis of the Jaws, American Association of Oral and Maxillofacial Surgeons . American Association of Oral and Maxillofacial Surgeons position paper on bisphosphonate‐related osteonecrosis of the jaws. J. Oral Maxillofac. Surg. 65, 369–376 (2007).  [DOI](https://doi.org/10.1016/j.joms.2006.11.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17307580/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Oral%20Maxillofac.%20Surg.&title=American%20Association%20of%20Oral%20and%20Maxillofacial%20Surgeons%20position%20paper%20on%20bisphosphonate%E2%80%90related%20osteonecrosis%20of%20the%20jaws&volume=65&publication_year=2007&pages=369-376&pmid=17307580&doi=10.1016/j.joms.2006.11.003&)

3. Wan, J.T. , Sheeley, D.M. , Somerman, M.J. & Lee, J.S. Mitigating osteonecrosis of the jaw (ONJ) through preventive dental care and understanding of risk factors. Bone Res. 8, 14 (2020).  [DOI](https://doi.org/10.1038/s41413-020-0088-1) | [PMC free article](/articles/PMC7064532/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32195012/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bone%20Res.&title=Mitigating%20osteonecrosis%20of%20the%20jaw%20(ONJ)%20through%20preventive%20dental%20care%20and%20understanding%20of%20risk%20factors&author=J.T.%20Wan&author=D.M.%20Sheeley&author=M.J.%20Somerman&author=J.S.%20Lee&volume=8&publication_year=2020&pages=14&pmid=32195012&doi=10.1038/s41413-020-0088-1&)

4. Drake, M.T. , Clarke, B.L. & Khosla, S. Bisphosphonates: mechanism of action and role in clinical practice. Mayo Clin. Proc. 83, 1032–1045 (2008).  [DOI](https://doi.org/10.4065/83.9.1032) | [PMC free article](/articles/PMC2667901/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18775204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mayo%20Clin.%20Proc.&title=Bisphosphonates:%20mechanism%20of%20action%20and%20role%20in%20clinical%20practice&author=M.T.%20Drake&author=B.L.%20Clarke&author=S.%20Khosla&volume=83&publication_year=2008&pages=1032-1045&pmid=18775204&doi=10.4065/83.9.1032&)

5. Yang, G. et al. Pharmacogenomics of osteonecrosis of the jaw. Bone 124, 75–82 (2019).  [DOI](https://doi.org/10.1016/j.bone.2019.04.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31022475/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bone&title=Pharmacogenomics%20of%20osteonecrosis%20of%20the%20jaw&author=G.%20Yang&volume=124&publication_year=2019&pages=75-82&pmid=31022475&doi=10.1016/j.bone.2019.04.010&)

6. Sutherland, A. et al. The role of prolactin in bone metastasis and breast cancer cell‐mediated osteoclast differentiation. J. Natl. Cancer Inst. 108, djv338 (2016).  [DOI](https://doi.org/10.1093/jnci/djv338) | [PMC free article](/articles/PMC5943829/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26586670/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Cancer%20Inst.&title=The%20role%20of%20prolactin%20in%20bone%20metastasis%20and%20breast%20cancer%20cell%E2%80%90mediated%20osteoclast%20differentiation&author=A.%20Sutherland&volume=108&publication_year=2016&pages=djv338&pmid=26586670&doi=10.1093/jnci/djv338&)

7. von Moos, R. et al. Management of bone health in solid tumours: from bisphosphonates to a monoclonal antibody. Cancer Treat. Rev. 76, 57–67 (2019).  [DOI](https://doi.org/10.1016/j.ctrv.2019.05.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31136850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Treat.%20Rev.&title=Management%20of%20bone%20health%20in%20solid%20tumours:%20from%20bisphosphonates%20to%20a%20monoclonal%20antibody&author=R.%20von%20Moos&volume=76&publication_year=2019&pages=57-67&pmid=31136850&doi=10.1016/j.ctrv.2019.05.003&)

8. Kim, C. , Bhatta, S. , Cyprien, L. , Fonseca, R. & Hernandez, R.K. Incidence of skeletal‐related events among multiple myeloma patients in the United States at oncology clinics: observations from real‐world data. J. Bone Oncol. 14, 100215 (2019).  [DOI](https://doi.org/10.1016/j.jbo.2018.100215) | [PMC free article](/articles/PMC6329702/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30666288/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bone%20Oncol.&title=Incidence%20of%20skeletal%E2%80%90related%20events%20among%20multiple%20myeloma%20patients%20in%20the%20United%20States%20at%20oncology%20clinics:%20observations%20from%20real%E2%80%90world%20data&author=C.%20Kim&author=S.%20Bhatta&author=L.%20Cyprien&author=R.%20Fonseca&author=R.K.%20Hernandez&volume=14&publication_year=2019&pages=100215&pmid=30666288&doi=10.1016/j.jbo.2018.100215&)

9. Wright, N.C. et al. The recent prevalence of osteoporosis and low bone mass in the United States based on bone mineral density at the femoral neck or lumbar spine. J. Bone Miner. Res. 29, 2520–2526 (2014).  [DOI](https://doi.org/10.1002/jbmr.2269) | [PMC free article](/articles/PMC4757905/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24771492/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bone%20Miner.%20Res.&title=The%20recent%20prevalence%20of%20osteoporosis%20and%20low%20bone%20mass%20in%20the%20United%20States%20based%20on%20bone%20mineral%20density%20at%20the%20femoral%20neck%20or%20lumbar%20spine&author=N.C.%20Wright&volume=29&publication_year=2014&pages=2520-2526&pmid=24771492&doi=10.1002/jbmr.2269&)

10. Saad, F. et al. Incidence, risk factors, and outcomes of osteonecrosis of the jaw: integrated analysis from three blinded active‐controlled phase III trials in cancer patients with bone metastases. Ann. Oncol. 23, 1341–1347 (2012).  [DOI](https://doi.org/10.1093/annonc/mdr435) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21986094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Oncol.&title=Incidence,%20risk%20factors,%20and%20outcomes%20of%20osteonecrosis%20of%20the%20jaw:%20integrated%20analysis%20from%20three%20blinded%20active%E2%80%90controlled%20phase%20III%20trials%20in%20cancer%20patients%20with%20bone%20metastases&author=F.%20Saad&volume=23&publication_year=2012&pages=1341-1347&pmid=21986094&doi=10.1093/annonc/mdr435&)

11. Durie, B.G. , Katz, M. & Crowley, J. Osteonecrosis of the jaw and bisphosphonates. N. Engl. J. Med. 353, 99–102 (2005), discussion 99–102.  [DOI](https://doi.org/10.1056/NEJM200507073530120) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16000365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Osteonecrosis%20of%20the%20jaw%20and%20bisphosphonates&author=B.G.%20Durie&author=M.%20Katz&author=J.%20Crowley&volume=353&publication_year=2005&pages=99-102&pmid=16000365&doi=10.1056/NEJM200507073530120&)

12. Yang, G. et al. SIRT1/HERC4 locus associated with bisphosphonate‐induced osteonecrosis of the jaw: an exome‐wide association analysis. J. Bone Miner. Res. 33, 91–98 (2018).  [DOI](https://doi.org/10.1002/jbmr.3285) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28856724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bone%20Miner.%20Res.&title=SIRT1/HERC4%20locus%20associated%20with%20bisphosphonate%E2%80%90induced%20osteonecrosis%20of%20the%20jaw:%20an%20exome%E2%80%90wide%20association%20analysis&author=G.%20Yang&volume=33&publication_year=2018&pages=91-98&pmid=28856724&doi=10.1002/jbmr.3285&)

13. Yang, G. et al. SIRT1 Gene SNP rs932658 is associated with medication‐related osteonecrosis of the jaw. J. Bone Miner Res. 36, 347–356 (2021).  [DOI](https://doi.org/10.1002/jbmr.4185) | [PMC free article](/articles/PMC8733933/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32967053/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bone%20Miner%20Res.&title=SIRT1%20Gene%20SNP%20rs932658%20is%20associated%20with%20medication%E2%80%90related%20osteonecrosis%20of%20the%20jaw&author=G.%20Yang&volume=36&publication_year=2021&pages=347-356&pmid=32967053&doi=10.1002/jbmr.4185&)

14. Sarasquete, M.E. et al. Bisphosphonate‐related osteonecrosis of the jaw is associated with polymorphisms of the cytochrome P450 CYP2C8 in multiple myeloma: a genome‐wide single nucleotide polymorphism analysis. Blood 112, 2709–2712 (2008).  [DOI](https://doi.org/10.1182/blood-2008-04-147884) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18594024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Bisphosphonate%E2%80%90related%20osteonecrosis%20of%20the%20jaw%20is%20associated%20with%20polymorphisms%20of%20the%20cytochrome%20P450%20CYP2C8%20in%20multiple%20myeloma:%20a%20genome%E2%80%90wide%20single%20nucleotide%20polymorphism%20analysis&author=M.E.%20Sarasquete&volume=112&publication_year=2008&pages=2709-2712&pmid=18594024&doi=10.1182/blood-2008-04-147884&)

15. Nicoletti, P. et al. Genomewide pharmacogenetics of bisphosphonate‐induced osteonecrosis of the jaw: the role of RBMS3. Oncologist 17, 279–287 (2012).  [DOI](https://doi.org/10.1634/theoncologist.2011-0202) | [PMC free article](/articles/PMC3286178/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22267851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=Genomewide%20pharmacogenetics%20of%20bisphosphonate%E2%80%90induced%20osteonecrosis%20of%20the%20jaw:%20the%20role%20of%20RBMS3&author=P.%20Nicoletti&volume=17&publication_year=2012&pages=279-287&pmid=22267851&doi=10.1634/theoncologist.2011-0202&)

16. Katz, J. et al. Genetic polymorphisms and other risk factors associated with bisphosphonate induced osteonecrosis of the jaw. Int. J. Oral Maxillofac. Surg. 40, 605–611 (2011).  [DOI](https://doi.org/10.1016/j.ijom.2011.02.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21396799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Oral%20Maxillofac.%20Surg.&title=Genetic%20polymorphisms%20and%20other%20risk%20factors%20associated%20with%20bisphosphonate%20induced%20osteonecrosis%20of%20the%20jaw&author=J.%20Katz&volume=40&publication_year=2011&pages=605-611&pmid=21396799&doi=10.1016/j.ijom.2011.02.002&)

17. Such, E. et al. CYP2C8 gene polymorphism and bisphosphonate‐related osteonecrosis of the jaw in patients with multiple myeloma. Haematologica 96, 1557–1559 (2011).  [DOI](https://doi.org/10.3324/haematol.2011.042572) | [PMC free article](/articles/PMC3186320/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21685474/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=CYP2C8%20gene%20polymorphism%20and%20bisphosphonate%E2%80%90related%20osteonecrosis%20of%20the%20jaw%20in%20patients%20with%20multiple%20myeloma&author=E.%20Such&volume=96&publication_year=2011&pages=1557-1559&pmid=21685474&doi=10.3324/haematol.2011.042572&)

18. Holden, A.L. , Contreras, J.L. , John, S. & Nelson, M.R. The international serious adverse events consortium. Nat. Rev. Drug Discov. 13, 795–796 (2014).  [DOI](https://doi.org/10.1038/nrd4441) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25359365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Drug%20Discov.&title=The%20international%20serious%20adverse%20events%20consortium&author=A.L.%20Holden&author=J.L.%20Contreras&author=S.%20John&author=M.R.%20Nelson&volume=13&publication_year=2014&pages=795-796&pmid=25359365&doi=10.1038/nrd4441&)

19. Roden, D.M. et al. Development of a large‐scale de‐identified DNA biobank to enable personalized medicine. Clin. Pharmacol. Ther. 84, 362–369 (2008).  [DOI](https://doi.org/10.1038/clpt.2008.89) | [PMC free article](/articles/PMC3763939/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18500243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Development%20of%20a%20large%E2%80%90scale%20de%E2%80%90identified%20DNA%20biobank%20to%20enable%20personalized%20medicine&author=D.M.%20Roden&volume=84&publication_year=2008&pages=362-369&pmid=18500243&doi=10.1038/clpt.2008.89&)

20. Covarrubias‐Pazaran, G. Genome‐assisted prediction of quantitative traits using the R package sommer. PLoS One 11, e0156744 (2016).  [DOI](https://doi.org/10.1371/journal.pone.0156744) | [PMC free article](/articles/PMC4894563/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27271781/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Genome%E2%80%90assisted%20prediction%20of%20quantitative%20traits%20using%20the%20R%20package%20sommer&author=G.%20Covarrubias%E2%80%90Pazaran&volume=11&publication_year=2016&pages=e0156744&pmid=27271781&doi=10.1371/journal.pone.0156744&)

21. Willer, C.J. , Li, Y. & Abecasis, G.R. METAL: fast and efficient meta‐analysis of genomewide association scans. Bioinformatics 26, 2190–2191 (2010).  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta%E2%80%90analysis%20of%20genomewide%20association%20scans&author=C.J.%20Willer&author=Y.%20Li&author=G.R.%20Abecasis&volume=26&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

22. de Leeuw, C.A. , Mooij, J.M. , Heskes, T. & Posthuma, D. MAGMA: generalized gene‐set analysis of GWAS data. PLoS Comput. Biol. 11, e1004219 (2015).  [DOI](https://doi.org/10.1371/journal.pcbi.1004219) | [PMC free article](/articles/PMC4401657/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25885710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput.%20Biol.&title=MAGMA:%20generalized%20gene%E2%80%90set%20analysis%20of%20GWAS%20data&author=C.A.%20de%20Leeuw&author=J.M.%20Mooij&author=T.%20Heskes&author=D.%20Posthuma&volume=11&publication_year=2015&pages=e1004219&pmid=25885710&doi=10.1371/journal.pcbi.1004219&)

23. Watanabe, K. , Taskesen, E. , van Bochoven, A. & Posthuma, D. Functional mapping and annotation of genetic associations with FUMA. Nat. Commun. 8, 1826 (2017).  [DOI](https://doi.org/10.1038/s41467-017-01261-5) | [PMC free article](/articles/PMC5705698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29184056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Functional%20mapping%20and%20annotation%20of%20genetic%20associations%20with%20FUMA&author=K.%20Watanabe&author=E.%20Taskesen&author=A.%20van%20Bochoven&author=D.%20Posthuma&volume=8&publication_year=2017&pages=1826&pmid=29184056&doi=10.1038/s41467-017-01261-5&)

24. Wang, Y. et al. The 3D Genome Browser: a web‐based browser for visualizing 3D genome organization and long‐range chromatin interactions. Genome Biol. 19, 151 (2018).  [DOI](https://doi.org/10.1186/s13059-018-1519-9) | [PMC free article](/articles/PMC6172833/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30286773/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&title=The%203D%20Genome%20Browser:%20a%20web%E2%80%90based%20browser%20for%20visualizing%203D%20genome%20organization%20and%20long%E2%80%90range%20chromatin%20interactions&author=Y.%20Wang&volume=19&publication_year=2018&pages=151&pmid=30286773&doi=10.1186/s13059-018-1519-9&)

25. Huang, D. et al. GWAS4D: multidimensional analysis of context‐specific regulatory variant for human complex diseases and traits. Nucleic Acids Res. 46, W114–W120 (2018).  [DOI](https://doi.org/10.1093/nar/gky407) | [PMC free article](/articles/PMC6030885/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29771388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=GWAS4D:%20multidimensional%20analysis%20of%20context%E2%80%90specific%20regulatory%20variant%20for%20human%20complex%20diseases%20and%20traits&author=D.%20Huang&volume=46&publication_year=2018&pages=W114-W120&pmid=29771388&doi=10.1093/nar/gky407&)

26. Porcu, E. et al. Mendelian randomization integrating GWAS and eQTL data reveals genetic determinants of complex and clinical traits. Nat. Commun. 10, 3300 (2019).  [DOI](https://doi.org/10.1038/s41467-019-10936-0) | [PMC free article](/articles/PMC6656778/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31341166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Mendelian%20randomization%20integrating%20GWAS%20and%20eQTL%20data%20reveals%20genetic%20determinants%20of%20complex%20and%20clinical%20traits&author=E.%20Porcu&volume=10&publication_year=2019&pages=3300&pmid=31341166&doi=10.1038/s41467-019-10936-0&)

27. Consortium . Human genomics. The Genotype‐Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Science 348, 648–660 (2015).  [DOI](https://doi.org/10.1126/science.1262110) | [PMC free article](/articles/PMC4547484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25954001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Human%20genomics.%20The%20Genotype%E2%80%90Tissue%20Expression%20(GTEx)%20pilot%20analysis:%20multitissue%20gene%20regulation%20in%20humans&volume=348&publication_year=2015&pages=648-660&pmid=25954001&doi=10.1126/science.1262110&)

28. Consortium . The Genotype‐Tissue Expression (GTEx) project. Nat. Genet. 45, 580–585 (2013).  [DOI](https://doi.org/10.1038/ng.2653) | [PMC free article](/articles/PMC4010069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23715323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=The%20Genotype%E2%80%90Tissue%20Expression%20(GTEx)%20project&volume=45&publication_year=2013&pages=580-585&pmid=23715323&doi=10.1038/ng.2653&)

29. Carvalho‐Silva, D. et al. Open Targets Platform: new developments and updates two years on. Nucleic Acids Res. 47, D1056–D1065 (2019).  [DOI](https://doi.org/10.1093/nar/gky1133) | [PMC free article](/articles/PMC6324073/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30462303/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Open%20Targets%20Platform:%20new%20developments%20and%20updates%20two%20years%20on&author=D.%20Carvalho%E2%80%90Silva&volume=47&publication_year=2019&pages=D1056-D1065&pmid=30462303&doi=10.1093/nar/gky1133&)

30. Belinky, F. et al. PathCards: multi‐source consolidation of human biological pathways. Database (Oxford) 2015, bav006 (2015).  [DOI](https://doi.org/10.1093/database/bav006) | [PMC free article](/articles/PMC4343183/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25725062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Database%20(Oxford)&title=PathCards:%20multi%E2%80%90source%20consolidation%20of%20human%20biological%20pathways&author=F.%20Belinky&volume=2015&publication_year=2015&pages=bav006&pmid=25725062&doi=10.1093/database/bav006&)

31. Zhou, X. & Wang, T. Using the Wash U Epigenome Browser to examine genome‐wide sequencing data. Curr. Protoc. Bioinformatics 10, Unit 10.10 (2012).  [DOI](https://doi.org/10.1002/0471250953.bi1010s40) | [PMC free article](/articles/PMC3643794/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23255151/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.%20Bioinformatics%2010,%20Unit%2010.10&title=Using%20the%20Wash%20U%20Epigenome%20Browser%20to%20examine%20genome%E2%80%90wide%20sequencing%20data&author=X.%20Zhou&author=T.%20Wang&publication_year=2012&pmid=23255151&doi=10.1002/0471250953.bi1010s40&)

32. Goldman, M. et al. The UCSC cancer genomics browser: update 2013. Nucleic Acids Res. 41, D949–D954 (2013).  [DOI](https://doi.org/10.1093/nar/gks1008) | [PMC free article](/articles/PMC3531186/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23109555/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=The%20UCSC%20cancer%20genomics%20browser:%20update%202013&author=M.%20Goldman&volume=41&publication_year=2013&pages=D949-D954&pmid=23109555&doi=10.1093/nar/gks1008&)

33. Warde‐Farley, D. et al. The GeneMANIA prediction server: biological network integration for gene prioritization and predicting gene function. Nucleic Acids Res. 38, W214–W220 (2010).  [DOI](https://doi.org/10.1093/nar/gkq537) | [PMC free article](/articles/PMC2896186/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20576703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=The%20GeneMANIA%20prediction%20server:%20biological%20network%20integration%20for%20gene%20prioritization%20and%20predicting%20gene%20function&author=D.%20Warde%E2%80%90Farley&volume=38&publication_year=2010&pages=W214-W220&pmid=20576703&doi=10.1093/nar/gkq537&)

34. Langemeyer, L. , Fröhlich, F. & Ungermann, C. Rab GTPase function in endosome and lysosome biogenesis. Trends Cell. Biol. 28, 957–970 (2018).  [DOI](https://doi.org/10.1016/j.tcb.2018.06.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30025982/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Cell.%20Biol.&title=Rab%20GTPase%20function%20in%20endosome%20and%20lysosome%20biogenesis&author=L.%20Langemeyer&author=F.%20Fr%C3%B6hlich&author=C.%20Ungermann&volume=28&publication_year=2018&pages=957-970&pmid=30025982&doi=10.1016/j.tcb.2018.06.007&)

35. Lawson, C.D. & Ridley, A.J. Rho GTPase signaling complexes in cell migration and invasion. J. Cell Biol. 217, 447–457 (2018).  [DOI](https://doi.org/10.1083/jcb.201612069) | [PMC free article](/articles/PMC5800797/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29233866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol.&title=Rho%20GTPase%20signaling%20complexes%20in%20cell%20migration%20and%20invasion&author=C.D.%20Lawson&author=A.J.%20Ridley&volume=217&publication_year=2018&pages=447-457&pmid=29233866&doi=10.1083/jcb.201612069&)

36. Thompson, K. , Rogers, M.J. , Coxon, F.P. & Crockett, J.C. Cytosolic entry of bisphosphonate drugs requires acidification of vesicles after fluid‐phase endocytosis. Mol. Pharmacol. 69, 1624–1632 (2006).  [DOI](https://doi.org/10.1124/mol.105.020776) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16501031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=Cytosolic%20entry%20of%20bisphosphonate%20drugs%20requires%20acidification%20of%20vesicles%20after%20fluid%E2%80%90phase%20endocytosis&author=K.%20Thompson&author=M.J.%20Rogers&author=F.P.%20Coxon&author=J.C.%20Crockett&volume=69&publication_year=2006&pages=1624-1632&pmid=16501031&doi=10.1124/mol.105.020776&)

37. Luckman, S.P. et al. Nitrogen‐containing bisphosphonates inhibit the mevalonate pathway and prevent post‐translational prenylation of GTP‐binding proteins, including Ras. J. Bone Miner. Res. 13, 581–589 (1998).  [DOI](https://doi.org/10.1359/jbmr.1998.13.4.581) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9556058/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Bone%20Miner.%20Res.&title=Nitrogen%E2%80%90containing%20bisphosphonates%20inhibit%20the%20mevalonate%20pathway%20and%20prevent%20post%E2%80%90translational%20prenylation%20of%20GTP%E2%80%90binding%20proteins,%20including%20Ras&author=S.P.%20Luckman&volume=13&publication_year=1998&pages=581-589&pmid=9556058&doi=10.1359/jbmr.1998.13.4.581&)

38. Gong, L. , Altman, R.B. & Klein, T.E. Bisphosphonates pathway. Pharmacogenet. Genomics. 21, 50–53 (2011).  [DOI](https://doi.org/10.1097/FPC.0b013e328335729c) | [PMC free article](/articles/PMC3086066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20023594/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics.&title=Bisphosphonates%20pathway&author=L.%20Gong&author=R.B.%20Altman&author=T.E.%20Klein&volume=21&publication_year=2011&pages=50-53&pmid=20023594&doi=10.1097/FPC.0b013e328335729c&)

39. Borowiec, M. et al. Mutations at the BLK locus linked to maturity onset diabetes of the young and beta‐cell dysfunction. Proc. Natl. Acad. Sci. USA 106, 14460–14465 (2009).  [DOI](https://doi.org/10.1073/pnas.0906474106) | [PMC free article](/articles/PMC2732833/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19667185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Mutations%20at%20the%20BLK%20locus%20linked%20to%20maturity%20onset%20diabetes%20of%20the%20young%20and%20beta%E2%80%90cell%20dysfunction&author=M.%20Borowiec&volume=106&publication_year=2009&pages=14460-14465&pmid=19667185&doi=10.1073/pnas.0906474106&)

40. Tang, C.H. , Hsu, C.J. , Yang, W.H. & Fong, Y.C. Lipoteichoic acid enhances IL‐6 production in human synovial fibroblasts via TLR2 receptor, PKCdelta and c‐Src dependent pathways. Biochem. Pharmacol. 79, 1648–1657 (2010).  [DOI](https://doi.org/10.1016/j.bcp.2010.01.025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20109438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Pharmacol.&title=Lipoteichoic%20acid%20enhances%20IL%E2%80%906%20production%20in%20human%20synovial%20fibroblasts%20via%20TLR2%20receptor,%20PKCdelta%20and%20c%E2%80%90Src%20dependent%20pathways&author=C.H.%20Tang&author=C.J.%20Hsu&author=W.H.%20Yang&author=Y.C.%20Fong&volume=79&publication_year=2010&pages=1648-1657&pmid=20109438&doi=10.1016/j.bcp.2010.01.025&)

41. Byeon, S.E. et al. The role of Src kinase in macrophage‐mediated inflammatory responses. Mediators Inflamm. 2012, 512926 (2012).  [DOI](https://doi.org/10.1155/2012/512926) | [PMC free article](/articles/PMC3504478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23209344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mediators%20Inflamm.&title=The%20role%20of%20Src%20kinase%20in%20macrophage%E2%80%90mediated%20inflammatory%20responses&author=S.E.%20Byeon&volume=2012&publication_year=2012&pages=512926&pmid=23209344&doi=10.1155/2012/512926&)

42. de Mingo, Á. et al. Cysteine cathepsins control hepatic NF‐κB‐dependent inflammation via sirtuin‐1 regulation. Cell Death Dis. 7, e2464 (2016).  [DOI](https://doi.org/10.1038/cddis.2016.368) | [PMC free article](/articles/PMC5260902/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27831566/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Dis.&title=Cysteine%20cathepsins%20control%20hepatic%20NF%E2%80%90%CE%BAB%E2%80%90dependent%20inflammation%20via%20sirtuin%E2%80%901%20regulation&author=%C3%81.%20de%20Mingo&volume=7&publication_year=2016&pages=e2464&pmid=27831566&doi=10.1038/cddis.2016.368&)

43. Brakspear, K.S. & Mason, D.J. Glutamate signaling in bone. Front. Endocrinol. (Lausanne) 3, 97 (2012).  [DOI](https://doi.org/10.3389/fendo.2012.00097) | [PMC free article](/articles/PMC3412269/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22888325/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Endocrinol.%20(Lausanne)&title=Glutamate%20signaling%20in%20bone&author=K.S.%20Brakspear&author=D.J.%20Mason&volume=3&publication_year=2012&pages=97&pmid=22888325&doi=10.3389/fendo.2012.00097&)

44. Nedergaard, M. , Takano, T. & Hansen, A.J. Beyond the role of glutamate as a neurotransmitter. Nat. Rev. Neurosci. 3, 748–755 (2002).  [DOI](https://doi.org/10.1038/nrn916) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12209123/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Neurosci.&title=Beyond%20the%20role%20of%20glutamate%20as%20a%20neurotransmitter&author=M.%20Nedergaard&author=T.%20Takano&author=A.J.%20Hansen&volume=3&publication_year=2002&pages=748-755&pmid=12209123&doi=10.1038/nrn916&)

45. Sharma, M.K. , Seidlitz, E.P. & Singh, G. Cancer cells release glutamate via the cystine/glutamate antiporter. Biochem. Biophys. Res. Commun. 391, 91–95 (2010).  [DOI](https://doi.org/10.1016/j.bbrc.2009.10.168) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19896463/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&title=Cancer%20cells%20release%20glutamate%20via%20the%20cystine/glutamate%20antiporter&author=M.K.%20Sharma&author=E.P.%20Seidlitz&author=G.%20Singh&volume=391&publication_year=2010&pages=91-95&pmid=19896463&doi=10.1016/j.bbrc.2009.10.168&)

46. Faccio, R. et al. Vav3 regulates osteoclast function and bone mass. Nat. Med. 11, 284–290 (2005).  [DOI](https://doi.org/10.1038/nm1194) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15711558/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&title=Vav3%20regulates%20osteoclast%20function%20and%20bone%20mass&author=R.%20Faccio&volume=11&publication_year=2005&pages=284-290&pmid=15711558&doi=10.1038/nm1194&)

47. Writzl, K. et al. De novo mutations in SLC25A24 cause a disorder characterized by early aging, bone dysplasia, characteristic face, and early demise. Am. J. Hum. Genet. 101, 844–855 (2017).  [DOI](https://doi.org/10.1016/j.ajhg.2017.09.017) | [PMC free article](/articles/PMC5673633/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29100094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=De%20novo%20mutations%20in%20SLC25A24%20cause%20a%20disorder%20characterized%20by%20early%20aging,%20bone%20dysplasia,%20characteristic%20face,%20and%20early%20demise&author=K.%20Writzl&volume=101&publication_year=2017&pages=844-855&pmid=29100094&doi=10.1016/j.ajhg.2017.09.017&)

48. Qin, L. & Beier, F. EGFR signaling: friend or foe for cartilage? JBMR Plus 3, e10177 (2019).  [DOI](https://doi.org/10.1002/jbm4.10177) | [PMC free article](/articles/PMC6383702/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30828691/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JBMR%20Plus&title=EGFR%20signaling:%20friend%20or%20foe%20for%20cartilage?&author=L.%20Qin&author=F.%20Beier&volume=3&publication_year=2019&pages=e10177&pmid=30828691&doi=10.1002/jbm4.10177&)

49. Linder, M. et al. EGFR controls bone development by negatively regulating mTOR‐signaling during osteoblast differentiation. Cell Death Differ. 25, 1094–1106 (2018).  [DOI](https://doi.org/10.1038/s41418-017-0054-7) | [PMC free article](/articles/PMC5988706/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29445126/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Differ.&title=EGFR%20controls%20bone%20development%20by%20negatively%20regulating%20mTOR%E2%80%90signaling%20during%20osteoblast%20differentiation&author=M.%20Linder&volume=25&publication_year=2018&pages=1094-1106&pmid=29445126&doi=10.1038/s41418-017-0054-7&)

50. Esfahanizadeh, N. et al. Correlation between bone mineral density of jaws and skeletal sites in an Iranian population using dual X‐ray energy absorptiometry. Dent. Res. J. (Isfahan) 10, 460–466 (2013).  [PMC free article](/articles/PMC3793408/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24130580/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dent.%20Res.%20J.%20(Isfahan)&title=Correlation%20between%20bone%20mineral%20density%20of%20jaws%20and%20skeletal%20sites%20in%20an%20Iranian%20population%20using%20dual%20X%E2%80%90ray%20energy%20absorptiometry&author=N.%20Esfahanizadeh&volume=10&publication_year=2013&pages=460-466&pmid=24130580&)
