# Beta-lactam-induced immediate hypersensitivity reactions: A genome-wide association study of a deeply phenotyped cohort

## Metadata
**Authors:** Paola Nicoletti, Daniel F Carr, Sarah Barrett, Laurence McEvoy, Peter S Friedmann, Neil H Shear, Matthew R Nelson, Anca M Chiriac, Natalia Blanca-López, José A Cornejo-García, Francesco Gaeta, Alla Nakonechna, Maria J Torres, Cristiano Caruso, Rocco L Valluzzi, Aris Floratos, Yufeng Shen, Rebecca K Pavlos, Elizabeth J Phillips, Pascal Demoly, Antonino Romano, Miguel Blanca, Munir Pirmohamed
**Journal:** The Journal of Allergy and Clinical Immunology
**Date:** 2021 May
**DOI:** [10.1016/j.jaci.2020.10.004](https://doi.org/10.1016/j.jaci.2020.10.004)
**PMID:** 33058932
**PMCID:** PMC8100096
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8100096/

## Abstract

**Background:** 
β-lactam antibiotics are associated with a variety of immune-mediated or hypersensitivity reactions, including immediate (type I) reactions mediated by antigen-specific IgE.

**Objective:** 
We sought to identify genetic predisposing factors for immediate reactions to β-lactam antibiotics.

**Methods:** 
Patients with a clinical history of immediate hypersensitivity reactions to either penicillins or cephalosporins, which were immunologically confirmed, were recruited from allergy clinics. A genome-wide association study was conducted on 662 patients (the discovery cohort) with a diagnosis of immediate hypersensitivity and the main finding was replicated in a cohort of 98 Spanish cases, recruited using the same diagnostic criteria as the discovery cohort.

**Results:** 
Genome-wide association study identified rs71542416 within the Class II HLA region as the top hit (P = 2 × 10−14); this was in linkage disequilibrium with HLA-DRB1∗10:01 (odds ratio, 2.93; P = 5.4 × 10−7) and HLA-DQA1∗01:05 (odds ratio, 2.93, P = 5.4 × 10−7). Haplotype analysis identified that HLA-DRB1∗10:01 was a risk factor even without the HLA-DQA1∗01:05 allele. The association with HLA-DRB1∗10:01 was replicated in another cohort, with the meta-analysis of the discovery and replication cohorts showing that HLA-DRB1∗10:01 increased the risk of immediate hypersensitivity at a genome-wide level (odds ratio, 2.96; P = 4.1 × 10−9). No association with HLA-DRB1∗10:01 was identified in 268 patients with delayed hypersensitivity reactions to β-lactams.

**Conclusions:** 
HLA-DRB1∗10:01 predisposed to immediate hypersensitivity reactions to penicillins. Further work to identify other predisposing HLA and non-HLA loci is required.

Key words: Type I hypersensitivity, β-lactams, penicillins, cephalosporins, allergy, anaphylaxis, pharmacogenomics

Abbreviations used: BL, β-Lactam; OR, Odds ratio; QTL, Quantitative trait loci; SNP, Single nucleotide polymorphism

### Background

β-lactam antibiotics are associated with a variety of immune-mediated or hypersensitivity reactions, including immediate (type I) reactions mediated by antigen-specific IgE.

### Objective

We sought to identify genetic predisposing factors for immediate reactions to β-lactam antibiotics.

### Methods

Patients with a clinical history of immediate hypersensitivity reactions to either penicillins or cephalosporins, which were immunologically confirmed, were recruited from allergy clinics. A genome-wide association study was conducted on 662 patients (the discovery cohort) with a diagnosis of immediate hypersensitivity and the main finding was replicated in a cohort of 98 Spanish cases, recruited using the same diagnostic criteria as the discovery cohort.

### Results

Genome-wide association study identified rs71542416 within the Class II HLA region as the top hit (*P* = 2 × 10^−14^); this was in linkage disequilibrium with *HLA-DRB1∗10:01* (odds ratio, 2.93; *P* = 5.4 × 10^−7^) and *HLA-DQA1∗01:05* (odds ratio, 2.93, *P* = 5.4 × 10^−7^). Haplotype analysis identified that *HLA-DRB1∗10:01* was a risk factor even without the *HLA-DQA1∗01:05* allele. The association with *HLA-DRB1∗10:01* was replicated in another cohort, with the meta-analysis of the discovery and replication cohorts showing that *HLA-DRB1∗10:01* increased the risk of immediate hypersensitivity at a genome-wide level (odds ratio, 2.96; *P* = 4.1 × 10^−9^). No association with *HLA-DRB1∗10:01* was identified in 268 patients with delayed hypersensitivity reactions to β-lactams.

### Conclusions

*HLA-DRB1∗10:01* predisposed to immediate hypersensitivity reactions to penicillins. Further work to identify other predisposing HLA and non-HLA loci is required.

## Methods

### Cases

All subjects were recruited between 2009 and 2013 as part of International Consortium on Drug Hypersensitivity, involving 5 recruitment centers worldwide (Australia, France, Italy, Spain, and United Kingdom). The study was approved by ethics committees in all countries, and all patients gave written informed consent.

We recruited 662 patients (the discovery cohort) with a diagnosis of immediate hypersensitivity to BL antibiotics ([Table I](#tbl1)). The diagnosis of immediate hypersensitivity was made in specialist allergy clinics, as per published criteria.[12](#bib12) All patients required immunological assessment (total and specific IgE, skin testing including skin prick test and intradermal and/or oral provocation) as part of the inclusion criteria. Independent adjudication of all cases was undertaken by N.H.S. and P.S.F. For replication of any signals, we separately recruited another 98 patients with immediate hypersensitivity from a clinic in Spain, diagnosed according to the same criteria.

### Table I.

| Clinical characteristics | Immediate hypersensitivity |  | Delayed hypersensitivity cohort (n = 268) |
| --- | --- | --- | --- |
| Discovery cohort (n = 662) | Replication cohort (n = 98) |  |  |
| Female, n (%) | 416 (62) | 56 (57) | 174 (64) |
| Age (y) mean ± SD (% missing) | 42.0 ± 16 (27) | 51.4 ± 12.3 (0) | 44.5 ± 20 (73) |
| History of allergies, % (no. with available information) | 31 (658) | 9 (98) | 30.6 (268) |
| No. of ADRs, mean ± SD (no. with available information) | 1.1 ± 0.3 (659) | 1.2 ± 0.5 (98) | 1 ± 0.2 (251) |
| Autoimmune disease diagnosis, % | 9 | 6 | 7 |
| Positive skin test, % (total no. tested) | 85 (578) | 78 (67) | 93 (204) |
| Positive prick test, % (total no. tested) | 45 (142) | 37 (82) | 82 (207) |
| Positive oral provocation/rechallenge, % (total no. tested) | 76 (106) | 65 (20) | 94 (17) |
| Clinical symptoms, n (%) |  |  |  |
| Immediate hypersensitivity manifestations∗ | 662 (100) | 98 (100) | — |
| AGEP | — | — | 14 (5) |
| DRESS | — | — | 7 (3) |
| Mild reactions including maculopapular exanthem | — | — | 212 (79) |
| SJS/TEN | — | — | 36 (13) |
| Drug class, n (%) |  |  |  |
| Penicillin | 501 (75) | 98 (100) | 246 (92) |
| Cephalosporin | 162 (25) | — | 20 (7.5) |
| Other BLs | — | — | 2 (0.01) |
| Suspected causal drug |  |  |  |
| Amoxicillin | 165 (25) | 65 (66) | 77 (29) |
| Ampicillin | 36 (5) | — | 54 (20) |
| Bacampicillin | 20 (3) | — | 21 (8) |
| Cefaclor | 23 (3) | — | — |
| Cefazolin | 17 (3) | — | 4 (1.5) |
| Cefotaxime | 17 (3) | — | — |
| Ceftazidime | 18 (3) | — | 1 (0.4) |
| Ceftriaxone | 52 (8) | — | 4 (1.5) |
| Cefuroxime | 14 (3) | — | 2 (0.7) |
| Co-amoxiclav | 218 (33) | 26 (26) | 70 (26) |
| Phenoxymethylpenicillin | 24 (4) | 7 (7) | 5 (2) |
| Piperacillin | 18 (3) | — | 5 (2) |
| Other | 41 (6) | — | 25 (9) |

Table Caption: Causative drugs and clinical variables broken down across the discovery and replication cohorts

To determine specificity of any signals identified in patients with immediate hypersensitivity, we also evaluated 268 patients with delayed hypersensitivity reactions across multiple BLs. The diagnosis again was in accordance with published guidance,[12](#bib12) and all cases were adjudicated by N.H.S. and P.S.F. We also included an additional 17 BL-induced delayed hypersensitivity reaction cases analyzed in Shen et al.[13](#bib13)

### Controls

We used general population samples as study controls. This comprised 9217 European ancestry controls from multiple available sources enriching the group with Spanish, Italian, and north European origin samples because cases were mainly recruited from those countries. We used the Wellcome Trust Case Control Consortium ([http://www.wtccc.org.uk](http://www.wtccc.org.uk)); the Population Reference Sample (POPRES),[14](#bib14) PGX4000119,[13](#bib13) LAM30004,[13](#bib13) and Spanish Bladder cancer cohort (phs000346.v1)[15](#bib15) from the Database of Genotypes and Phenotypes (dbGaP); Hypergenes cohort ([http://www.hypergenes.eu/](http://www.hypergenes.eu/)); the National Spanish DNA Bank ([http://www.bancoadn.org/](http://www.bancoadn.org/)); and Toscani in Italia ([TSI] HapMap data) to obtain ancestry control data. In addition, we also recruited a group of 137 penicillin-tolerant controls from Italy.

### Genotyping

Genome-wide genotyping of DNA extracted from whole blood was performed at the Broad Institute (Boston, Mass) for 662 cases with BL-induced immediate hypersensitivity and 268 cases with delayed hypersensitivity reaction, and from 137 penicillin-tolerant controls from Italy. In 474 (354 BL-induced immediate and 120 BL-induced delayed) cases, the Illumina Infinium HumanCoreExome BeadChip (Illumina, Inc, San Diego, Calif) was used while for 439 (308 BL-induced immediate and 131 BL-induced delayed) cases, the Illumina HumanOmniExpress BeadChip was used. In this last batch, we also genotyped 137 Italian penicillin-tolerant controls. In addition, the BL-induced delayed case group also included 17 BL-delayed hypersensitivity cases previously genotyped by the Illumina 1M Duo chip, extracted from a larger Stevens-Johnson syndrome/toxic epidermal necrolysis study that included multiple drugs, as described by Shen et al.[13](#bib13) Other control cohorts were publicly available (see [Table E1](#tblE1) in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org)). For each of the genotyping cohort, standard quality control was conducted at both single marker and subject levels as previously described.[13](#bib13) This was followed by SNP and HLA imputation and amino acid analysis (see the [Methods](#appsec1) section in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org)).

### Replication cohort SNP and HLA genotyping

The top associated imputed SNPs were validated by SNP genotyping using either TaqMan, SNP genotyping assays (Thermo Fisher Scientific, Paisley, UK) or iPLEX MassArray genotyping platform (Agena Biosciences, Hamburg, Germany). High-resolution genotyping of *HLA-A, HLA-B, HLA-C, DRB1, DQA1,* and *DQB1* was performed by Histogenetics (Ossining, NY). Sequencing data files were analyzed using Histogenetics’ proprietary analysis software (Histomatcher and HistoMagic) for HLA genotype calling. Allele assignments are based on IMGT/HLA Database (release version 2.21.0, dated April 2008; [http://www.ebi.ac.uk/imgt/hla/](http://www.ebi.ac.uk/imgt/hla/)).

### Statistical analysis

The effect of population structure was assessed through principal component analysis using the smartPCA program from the EIGENSTRAT package (version 3.0; Alkes Price, Harvard T.H. Chan School, Boston, Mass).[16](#bib16) Single marker and haplotype association analyses and heterogeneity test analyses were carried out by PLINK 1.07.[17](#bib17) The statistical association of each marker, HLA alleles and SNPs, was determined in a logistic regression framework with scores for the first 7 principal components as covariates under an additive model using PLINK. We used the same statistical test for subpopulation analyses, using the 2, 7, and 10 most significant principal components as covariates in Italian, Spanish, and North European populations, respectively. We set the genome-wide traditional significance P-value threshold to 5.0 × 10^−8^ to correct for multiple testing and MHC-wide significance threshold to 2.0 × 10^−4^ to correct for total number of predicted alleles. When we obtained genome-wide significant signals, we tested for independent effects from the neighboring variants by including the most associated variants as a covariate and then testing the significance of others in the region. All detailed analyses and Manhattan plots were performed with R (version 3.0.2; R Foundation, Vienna, Austria). Regional plots were drawn by LocusZoom.[18](#bib18) Meta-analysis was performed using a fixed-effect model in the *metafor* package ([http://www.metafor-project.org/doku.php/metafor](http://www.metafor-project.org/doku.php/metafor)).

## Results

### Patient cohorts

The clinical characteristics of the patients are shown in [Table I](#tbl1). Clinical manifestations in the discovery cohort included angioedema (35%), bronchospasm (24%), and urticaria (34%), while hypotension was reported in only 4% of cases. The length of reaction in patients with immediate hypersensitivity was 2 to 11 days, while it ranged from 21 to 26 days for patients with delayed hypersensitivity reactions. Patients were included if they had positive diagnostic assessment, as highlighted in [Table I](#tbl1). Penicillins accounted for 75% of cases, with the most common culprit drug being amoxicillin accounting for 58% of cases in the discovery cohort.

### Association with immediate reactions to BLs

We first conducted a genome-wide association study on 662 patients of European descent with immediate hypersensitivity reactions and 9217 previously genotyped population controls matched for ethnicity. The total number of SNPs, which were included in the analyses after quality control, was 4,265,742. The cases clustered within 3 major groups (Italian, Spanish, and Northern European) (see [Fig E1](#figE1) in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org)) in keeping with the self-reported ethnicity.

### Fig E1.

![Fig E1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3372/8100096/89849c513c36/fx1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8100096_fx1.jpg)

Scatterplot representing the first 2 principal components (PCs) of the current study cohorts, cases and controls: BL-immediate hypersensitivity reactions (A) and BL-delayed hypersensitive reactions (B).

A genome-wide significant association was identified within the Class II HLA region, rs71542416 being the top hit (odds ratio [OR], 5.17; 95% CI, 3.40-5.17; *P* = 2 × 10^−14^) ([Table II](#tbl2), [Fig 1](#fig1), *A*, and see [Fig E2](#figE2) in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org)). The frequency of rs71542416 in our control population was comparable with that reported in publicly available sources ([Table II](#tbl2)). HLA allele imputation using HLA genotype imputation with attribute bagging, or HIBAG,[19](#bib19) showed the *HLA-DRB1∗10:01* (OR, 2.95; 95% CI, 1.99-4.36; *P* = 6.0 × 10^−8^) and *HLA-DQA1∗01:05* (OR, 2.93; 95% CI, 1.92-4.45; *P* = 5.4 × 10^−7^) alleles to be significantly associated with the immediate reactions, with consistent ORs ([Table II](#tbl2)) and were tagged by rs71542416 (*r*^2^ = 0.76). Haplotype analysis identified that *HLA-DRB1∗10:01* was a risk factor even without the *HLA-DQA1∗01:05* allele (see [Table E2](#tblE2) in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org)). *HLA-DRB1∗10:01* was seen in 3% of cases and <1% of controls. The frequency of the HLA alleles within the Italian penicillin-tolerant controls was 10 times less than in the Italian general population (0.1% vs 1%).

### Table II.

|   | Minor allele frequency |  |  | Association analysis |  | Association conditioned for HLA haplotype∗ |  | Association conditioned for rs71542416 |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Cases | Controls | Population reference cohort | OR (95% CI) | P value | OR (95% CI) | P value | OR (95% CI) | P value |  |
| HLA-DRB1∗10:01 | 0.03 | 0.008 | 0.008 | 2.95 (1.99-4.36) | 6.0 × 10−8 | — | — | 0.60 (90.19-1.85) | .37 |
| HLA-DQA1∗01:05 | 0.03 | 0.01 | 0.01 | 2.93 (1.92-4.46) | 5.4 × 10−7 | — | — | 0.79 (0.32-1.91) | .60 |
| rs71542416 | 0.03 | 0.006 | 0.008 | 5.17 (3.40-5.17) | 1.2 × 10−14 | 8.22 (2.68-25.23) | .0002 | — | — |
| rs114632839† | 0.25 | 0.367 | 0.40 | 0.77 (0.67-0.89) | .0003 | 0.69 (0.60-0.80) | 1.1 × 10−6 | 0.68 (0.59-0.79) | 6.1 × 10−7 |

Table Caption: The most significantly associated variants for immediate hypersensitivity reactions to BLs

### Fig 1.

![Fig 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3372/8100096/bf3e2f51079a/gr1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8100096_gr1.jpg)

Genomic data in patients with immediate hypersensitivity reactions. A, Manhattan plot displaying the association analysis undertaken in patients with immediate hypersensitivity reactions to BLs (n = 662). SNPs in green have a significance level <5 × 10−6 and red have a significance level <5 × 10−8. B, Forest plot showing the effect size of the association between HLA-DRB1∗10:01 and immediate reactions in the discovery and replication cohorts. For each analysis, the OR of the association is reported with 95% CI. The dimension of the squares is proportional of the number of cases.

### Fig E2.

![Fig E2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3372/8100096/9766207a8d0e/fx2.jpg)

The figure shows the QQ plot for the current analysis. The inflation factor is 1.09.

The HLA allele effect size was similar across the 3 major clusters (heterogeneity test, *P* = .11) ([Table III](#tbl3)). The positive predictions in cases were fully validated by direct HLA typing. An additional 67 cases with low-quality predictions in both the loci were also typed. Among them, we found only 1 positive carrier for *HLA-DRB1∗10:01*. All cases were also genotyped for rs71542416—this showed a concordance of 99% between typed and imputed genotypes of rs71542416. *HLA-DRB1∗10:01* co-occurred with rs71542416 in 89% of the *HLA-DRB1∗10:01*–positive patients, while 12% of all cases carried rs71542416 alone.

### Table III.

| Ethnic cluster | No. of cases∗ | Minor allele frequency |  | OR (95% CI) | P value |
| --- | --- | --- | --- | --- | --- |
| Cases | Controls |  |  |  |  |
| HLA-DRB1∗10:01 |  |  |  |  |  |
| Italians | 352 | 0.021 | 0.012 | 2.33 (1.15-4.73) | .02 |
| Spanish | 226 | 0.049 | 0.014 | 3.81 (2.27-6.42) | 4.74 × 10−7 |
| Northern Europeans | 61 | 0.025 | 0.004 | 3.93 (1.17-13.21) | .03 |
| rs71542416 |  |  |  |  |  |
| Italians | 352 | 0.02 | 0.007 | 4.33 (1.98-9.49) | .0002 |
| Spanish | 226 | 0.05 | 0.008 | 6.80 (3.89-11.87) | 1.69 × 10−11 |
| Northern Europeans | 61 | 0.02 | 0.004 | 4.42 (1.29-15.13) | .02 |

Table Caption: The association between HLA-DRB1∗10:01 and rs71542416, and BL-induced immediate hypersensitivity reactions across the different nationalities

Including rs71542416 or the HLA alleles as covariates revealed a residual protective effect of the *HLA-DRA* locus, tagged by rs114632839, an intronic gene variant, in accordance with the findings of Gueant et al[11](#bib11) ([Table II](#tbl2) and see [Figs E3](#figE3) and [E4](#figE4), *A* in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org)). Interestingly Genotype-Tissue Expression Program, or GTEx (Common Fund, National Institutes of Health, Bethesda, Md), analysis revealed that this variant was a strong expression quantitative trait loci (QTL) for *HLA-DRB5* (*P* = 5.3 × 10^−23^) and splicing QTL for the *HLA-DRB1* (*P* = 1.1 × 10^−16^), *HLA-DRB5* (*P* = 1.1 × 10^−16^) and *HLA-DRB6* (*P* = 1.1 × 10^−16^) loci with the minor alleles showing a lower intron excision ratio. Both effects were detected in whole blood and shared across other tissues ([Fig E4](#figE4), *B*).

### Fig E3.

![Fig E3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3372/8100096/147b3eceee19/fx3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8100096_fx3.jpg)

Regional plots of the MHC region for the conditioned analysis on HLA-DRB1∗10:01/HLA-DQA1∗01:05 haplotype (A) and on rs71542416, the most associated SNP (B).

### Fig E4.

![Fig E4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3372/8100096/e717f2bf60dc/fx4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8100096_fx4.jpg)

Violin plots from the GTEx website for expression QTL (A) and splicing QTL (B) effects of rs114632839 in whole blood.

A replication cohort of 98 patients with anaphylaxis induced by either amoxicillin or amoxicillin-clavulanate ([Table I](#tbl1)) was recruited separately from Spain. We identified 7 individuals who were positive for *HLA-DRB1∗10:01*, as confirmed by HLA typing. Comparison using the 11 Spanish HLA-typed cohorts reported at [allelefrequency.net](http://allelefrequency.net) provided a total of 3137 Spanish subjects (see [Fig E5](#figE5) in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org)) as ethnically matched population controls. This analysis replicated the association with an OR of 2.80 (95% CI, 1.17-6.71; Fisher exact test, *P* = .016) ([Fig 1](#fig1), *B*).

### Fig E5.

![Fig E5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3372/8100096/bf66d4b3a511/fx5.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8100096_fx5.jpg)

Spanish studies reported on allelefrequency.net used as replication control set.

Meta-analysis of the discovery and replication cohorts showed that *HLA-DRB1∗10:01* increased the risk of immediate hypersensitivity at a genome-wide level (OR, 2.96; 95% CI, 1.99-4.37; *P* = 4.1 × 10^−9^) ([Fig 1](#fig1), *B*). The sensitivity and specificity of the allele is 0.06 and 0.98, respectively, while the positive and negative predictive values are 17% and 94%, respectively.

The most significantly associated amino acid with immediate hypersensitivity reactions was glutamate at position 10 (OR, 2.72; 95% CI, 1.81-4.08; *P* = 1.4 × 10^−6^) (see [Table E3](#tblE3) in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org)). Amoxicillin, amoxicillin-clavulanic acid, and phenoxymethylpenicillin showed the highest effect size (see [Table E4](#tblE4) in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org)). Glutamate-10 co-occurred with other amino acids (arginine-30, valine-31, alanine-38, tyrosine-40, proline-231, glutamine-166) that had the same frequency in cases and controls as glutamate-10 and *HLA-DRB1∗10:01* ([Table E3](#tblE3)). However, association with these amino acids disappeared after condition for either glutamate-10 or *HLA-DRB1∗10:01* ([Table E3](#tblE3)). Interestingly, glutamate-10 co-occurred with the shared epitope RRA at positions 70, 71 and 74, previously associated with seropositive rheumatoid arthritis[20](#bib20) and specific for the *HLA-DRB1∗10:01* allele. The ERRA haplotype increased risk (OR, 2.72; *P* = 1.4 × 10^−6^) equivalent to that seen with glutamate-10 alone. None of the other risk/protective amino acid motifs for seropositive rheumatoid arthritis[20](#bib20) spanning positions 70 to 74 in the *DRB1* locus (such as QRRAA risk motif or DERAA and DRRAA protective motifs) were associated with our phenotype.

### HLA analysis in patients with delayed hypersensitivity reactions

To determine whether the association with *HLA-DRB1∗10:01* was limited to patients with immediate hypersensitivity reactions, we analyzed 268 patients with delayed hypersensitivity to a variety of BLs ([Table I](#tbl1)) using the same control set ([Fig E1](#figE1), *B*). No association was identified for *HLA-DRB1∗10:01* (n = 249; OR, 1.34; 95% CI, 0.55-3.26; *P* = .5).

### Drug-specific associations with immediate hypersensitivity

*HLA-DRB1∗10:01* was associated with penicillins as a class (OR, 3.07), but not with cephalosporins ([Table IV](#tbl4)). Among the penicillins, the strongest signals were for amoxicillin (OR, 3.48), amoxicillin clavulanic acid (OR, 2.85), and phenoxymethylpenicillin (OR, 6.66) ([Table IV](#tbl4)). When we combined amoxicillin and amoxicillin clavulanic acid cases (assuming that amoxicillin rather than clavulanic acid was the culprit), the OR was 3.1 (95% CI, 2.01-4.85; *P* = 4.0 × 10^−7^). Additional drug-specific HLA allele associations that we identified will need confirmation (see [Tables E5](#tblE5) and [E6](#tblE6) in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org)).

### Table IV.

| Drug | Ethnicity∗ | No. of cases | Case MAF | OR (95% CI) | P value |
| --- | --- | --- | --- | --- | --- |
| Cephalosporins | Caucasian | 162 | 0.019 | 2.03 (0.82-5.07) | .13 |
| Cefaclor | Caucasian | 23 | 0 | — | — |
| Cefazolin | Caucasian | 17 | 0.059 | 6.12 (1.32-28.30) | .02 |
| Cefotaxime | Italian | 17 | 0 | — | — |
| Ceftazidime | Italian | 17 | 0 | — | — |
| Ceftriaxone | Italian | 48 | 0.010 | 1.05 (0.13-8.33) | .96 |
| Cefuroxime | Caucasian | 14 | 0.050 | 2.90 (0.37-22.76) | .31 |
| Penicillins | Caucasian | 501 | 0.036 | 3.07 (2.04-4.62) | 7.42 × 10−8 |
| Amoxicillin | Caucasian | 166 | 0.042 | 3.48 (1.92-6,28) | 3.74 × 10−5 |
| Ampicillin | Italian | 29 | 0.014 | 1.98 (0.25-15.79) | .52 |
| Co-amoxiclav | Caucasian | 218 | 0.034 | 2.85 (1.60-5.10) | .0004 |
| Phenoxymethylpenicillin | Caucasian | 25 | 0.080 | 6.66 (2.14-20.79) | .001 |
| Piperacillin | Caucasian | 18 | 0.028 | 2.32 (0.29-18.78) | .43 |
| Bacampicillin | Italian | 21 | 0.024 | 2.09 (0.26-17.03) | .49 |

Table Caption: Effect size of the association of HLA-DRB1∗10:01 with immediate hypersensitivity reactions broken down by drug classes and individual drugs

In the drug-specific analysis, a genome-wide signal (rs71437970) on chromosome 13 upstream of *SLC15A1* (see [Fig E6](#figE6) in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org)) was identified for the amoxicillin cases (OR, 2.94; *P* = 3.8 × 10^−9^) ([Table E6](#tblE6)). This association was shared across the European subpopulations and with amoxicillin-clavulanate cases (see [Tables E7](#tblE7) and [E8](#tblE8) in this article’s Online Repository at [www.jacionline.org](http://www.jacionline.org)). However, we failed to replicate the association, with an allele frequency that was lower than that observed in Spanish controls (0.007 vs 0.04).

### Fig E6.

![Fig E6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3372/8100096/197c8eab5b00/fx6.jpg)

Regional plot of the chromosome 13 signal associated with amoxicillin-induced immediate hypersensitivity in the discovery cohort. The horizontal axis shows SNPs along the chromosomal region and the vertical axis shows the (−log10 transformed) observed P value. SNPs are color-coded to depict their linkage disequilibrium (LD) with the top SNP, shown in purple. Estimated recombination rates are plotted on the right vertical axis in cyan to reflect the local LD structure.

## Discussion

We have identified an association between the SNP rs71542416 and immediate hypersensitivity reactions to penicillins. The SNP does not affect gene expression in GTEx but is in linkage disequilibrium with *HLA-DRB1∗10:01* and *HLA-DQA1∗01:05.* Haplotype analysis identified that *HLA-DRB1∗10:01* was a risk factor even without the *HLA-DQA1∗01:05* allele suggesting that *HLA-DRB1∗10:01* may be the predominant driver of the association. However, 12% of cases carried rs71542416 but were negative for *HLA-DRB1∗10:01,* suggesting that the SNP may be a tag for other rare HLA alleles, which is consistent with the hypothesis of Heap et al[21](#bib21) who showed an association between *HLA-DQA1-HLA-DRB1* variants and thiopurine-induced pancreatitis.

The association with *HLA-DRB1∗10:01* and rs71542416 was most pronounced in the Spanish cohort ([Table III](#tbl3)), but given that the ORs were of similar magnitude in all populations studied, there was overlap in the CIs, and the prevalence of the SNP and HLA allele, our findings can be generalized across the European subethnicities studied ([Table III](#tbl3)). However, further studies will be needed in both European and non-European populations to determine the global relevance of this association. Additionally, the association was limited to immediate reactions and was not observed with the delayed hypersensitivity reactions, highlighting the specific nature of the association. Evaluation of drug-specificity showed associations with amoxicillin, amoxicillin-clavulanate, and phenoxymethylpenicillin. However, given the limited sample size with the other penicillins, we cannot exclude the possibility of an association with all penicillins ([Table IV](#tbl4)). Similarly, we did not find an association with cephalosporins, but this may also be because of a lower sample size.

The clear strength of our study is that all patients were deeply phenotyped: there was a clear clinical history with a temporal relationship to drug intake, and the diagnosis was confirmed immunologically by skin testing and/or oral provocation. Such deep phenotyping is important because many patients claim to be allergic to penicillin, but very few are; of those claiming to be allergic, <1 in 20 have an acute reaction to an oral challenge (the gold standard clinical test to confirm an IgE-mediated reaction).[22](#bib22)

Our data add to the increasing evidence of HLA in predisposing to different clinical phenotypes of drug hypersensitivity reactions.[23](#bib23) The most well-known of these associations is *HLA-B∗57:01* and abacavir hypersensitivity,[24](#bib24) which has been implemented into clinical practice and has resulted in a significant reduction in abacavir hypersensitivity.[25](#bib25) It is important to note that most of the HLA associations identified to date have been with delayed hypersensitivity reactions.[23](#bib23) However, more recent studies have identified HLA alleles as predisposing factors for immediate reactions. For instance, *HLA-DRB1∗07:01* is a risk factor for the development of anti-asparaginase antibodies and immediate reactions.[26](#bib26) Our data, which show that *HLA-DRB1∗10:01* predisposes to immediate hypersensitivity, are also consistent with the pathogenesis of immediate reactions where the interaction between B cells and CD4^+^/T_H_2-positive cells, through HLA Class II alleles, is central to the immunoglobulin switching that leads to the generation of specific IgE antibodies. Different HLA alleles have been associated with other types of immune-mediated reactions caused by BLs. For example, *HLA-B∗57:01* predisposes to flucloxacillin-induced cholestatic hepatitis,[27](#bib27) while liver injury caused by amoxicillin-clavulanate is associated with the Class II HLA haplotype *HLA-DRB1∗1501-DQB1∗0602.*[28](#bib28) Mechanistic studies undertaken in our laboratory have shown that drug-specific, HLA-restricted T cells can be isolated from patients with a past history of liver injury due to flucloxacillin[29](#bib29) and amoxicillin-clavulanate.[30](#bib30) It will be valuable to conduct similar studies in patients with a history of penicillin-induced immediate reactions to understand the mechanistic basis of the association with *HLA-DRB1∗10:01.*

Another potentially interesting finding in this study was the association between *SLC15A1* gene variants and amoxicillin-induced immediate reactions. SLC15A1 encodes the human peptide transporter 1, which is known to transport amoxicillin.[31](#bib31) Therefore, it is plausible that variation in the activity of human peptide transporter 1 could result in altered amoxicillin pharmacokinetics and thereby increase risk of a type I reaction. However, we were not able to replicate this finding, and further work (including functional studies) to understand whether this gene is important in predisposing to immediate reactions will be required.

What are the clinical implications of this finding? Given the rarity of penicillin-induced anaphylaxis, the low population prevalence and sensitivity of *HLA-DRB1∗10:01,* and the very wide usage of penicillins, the prospective use of this allele in screening patients before penicillin prescription would not be practical or feasible in terms of both the high numbers needed to test to prevent 1 case and patients unnecessarily excluded from therapy. However, this association of immediate penicillin hypersensitivity with *HLA-DRB1∗10:01* may provide much novel insights into the mechanisms of immediate reactions associated with penicillins, including the mechanisms of sensitization and natural loss or waning of penicillin, which is known to occur over time. Moreover, the higher negative predictive value of the allele (94%) may be of use in risk stratifying patients where penicillin cannot be excluded as an etiological agent in the setting of an immediate reaction.

Our study has limitations. First, the overall sample size is small compared with that used in complex diseases, but it is larger than that used in many pharmacogenomic studies. Our efforts to identify deeply phenotyped patients in this study was a result of an extensive international collaboration highlighting the difficulties in achieving large sample sizes in pharmacogenomic studies. Furthermore, we were unable to perform permutation testing to validate the replication *P* value for *HLA-DRB1∗10:01.* Second, because we used population controls, we could not adjust for self-reported ethnicity, but this is unlikely to have had a major impact as we accounted for this through an analysis of population stratification ([Fig E1](#figE1)). Third, matching cases and controls for age, sex, and other comorbidities was not possible because of the use of population controls, and because sex could not be determined due to the absence of X chromosome SNP data. Whether this impacts on the association with the genetic signals identified by us will require further study.

In summary, we have for the first time reported an association of *HLA-DRB1∗10:01* carriage in deeply immunologically phenotyped European ancestry individuals with penicillin-induced immediate type I hypersensitivity reactions. However, we cannot exclude the possibility of other HLA alleles or HLA haplotypes also being important in conferring susceptibility in some patients, and therefore further work in both European and non-European patients is required to identify other HLA alleles, and also whether *HLA-DRB1∗10:01* is universally important. It is also interesting to note that we also identified that rs114632839, which is a proxy for the *HLA-DRA* locus, protected against the development of immediate hypersensitivity reactions to BLs, consistent with a previous study.[11](#bib11) rs114632839 is an expression QTL and splicing QTL for several HLA loci, suggesting that predisposition to immediate hypersensitivity to penicillins is likely to be complex and mediated by a combination of susceptibility and protective HLA and non-HLA alleles. Clearly we have reported associations, and proof of causality will require a full understanding of the immunopathogenesis of initial sensitization to penicillin, and in particular, the mechanism of antigen presentation (including the relative importance of the BL ring vs the side chains) and interaction with CD4^+^ T cells that ultimately leads to IgE-switching and the generation of hapten-specific IgE antibodies.

This novel insight into the mechanisms of immediate reactions associated with penicillins may be of use in risk stratifying patients where penicillin cannot be excluded as an etiological agent.

This novel insight into the mechanisms of immediate reactions associated with penicillins may be of use in risk stratifying patients where penicillin cannot be excluded as an etiological agent.

## Acknowledgments

We offer special thanks to Arthur Holden for his help and effort in guiding this collaborative work and to the Broad genotyping facility for their contribution to the genome-wide association study genotyping. We also acknowledge the contribution of all our clinical collaborators and the study participants.

## Footnotes

## Methods

### Imputation procedure

Prephasing and imputation was performed in batches by dividing the cases and controls according to the genotyping platform. For each batch, we first prephased the genotype scaffold using SHAPEIT (version v2.r727).[E1](#bib32) Imputation was then undertaken using IMPUTE2 (version 3)[E2](#bib33) using the reference panel from the 1000 Genomes Project (release v3).[E1](#bib32) We used the “all ancestries” reference panel to improve the quality of the imputation especially of lower frequency variants.[E2](#bib33) For downstream analysis, we used best-guess genotypes retaining imputed genotypes with posterior probability >0.9. In the combined dataset, we then retained SNPs with: (1) no significant difference in missingness between cases and controls (*P* > .0001); (2) no significant deviation from Hardy-Weinberg equilibrium (*P* > .0001); (3) < 5% missing genotypes in all batches; (4) imputation information score >0.8 in all batches; and (5) MAF in the European descent haplotypes from the 1000 Genomes Project reference panel of at least 0.01. Finally, we compared SNP allele frequencies in controls between genotyping platforms. Specifically, we tested for association of genotyping platforms with SNPs in same ethnicity controls in a logistic regression framework, under an additive genetic model, with adjustment for principal components from smartPCA to account for population structure. SNPs with nominal evidence of association (*P* < .0001) were excluded. Sex chromosomes and mitochondria were not imputed.

For each batch, HLA alleles were inferred using HIBAG[E3](#bib34) using the reference predictor panels specific for the genotyping chip and Caucasian ancestry. Amino acid changes were inferred using SNP2HLA utilizing the reference data collected by the Type 1 Diabetes Genetics Consortium.[E4](#bib35)

### Table E1.

| Cohort | No. of samples | Genotyping Array |
| --- | --- | --- |
| WTCCC | 4824 | Illumina 1M BeadChip |
| phs000346.v1 (dbGaP) | 2072 | Illumina 1M BeadChip |
| Hypergenes | 901 | Illumina 1M BeadChip |
| LAM30004 collection | 60 | Illumina 1M BeadChip |
| National Spanish DNA Bank | 173 | Illumina 1M Duo/Illumina Infinium HumanCoreExome BeadChip |
|   | 222 | Illumina Infinium HumanCoreExome BeadChip |
| iSAEC Italian penicillin-tolerant controls | 137 | Illumina HumanOmniExpress BeadChip |
| PGX40001 collection | 85 | Illumina 1M Duo BeadChip |
| TSI | 88 | Illumina 1M Duo BeadChip |
| POPRES | 655 | Illumina 1M Duo BeadChip |

Table Caption: Genotyping details of the control cohorts

### Table E2.

| HLA-DRB1∗10:01 | HLA-DQA1∗01:05 | MAF |  | OR | P value |
| --- | --- | --- | --- | --- | --- |
| Cases | Controls |  |  |  |  |
| P | P | 0.03 | 0.01 | 2.93 | .000001 |
| P | A | 0.005 | 0.001 | 2.84 | .04 |

Table Caption: Effect size for the association with the HLA-DRB1∗10:01 and HLA-DQA1∗01:05 haplotypes

### Table E3.

| Marker |  | MAF |  | Association analysis |  | Association conditioned for glutamate-10 |  | Association conditioned for HLA haplotype |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Position | AA | Cases | Controls | OR (95% CI) | P value | OR (95% CI) | P value | OR (95% CI) | P value |
| 10 | E | 0.03 | 0.008 | 2.72 (1.81-4.09) | .000001 | — | — | — | — |
| 30 | R | 0.03 | 0.008 | 2.67 (1.78-4.01) | .000002 | — | — | — | — |
| 31 | V | 0.03 | 0.008 | 2.67 (1.78-4.01) | .000002 | — | — | — | — |
| 38 | A | 0.03 | 0.008 | 2.67 (1.78-4.01) | .000002 | — | — | — | — |
| 40 | Y | 0.03 | 0.008 | 2.67 (1.78-4.01) | .000002 | — | — | — | — |
| 231 | P | 0.03 | 0.008 | 2.53 (1.68-3.8) | .000008 | — | — | 0.4 (0.04-4.2) | .44 |
| 166 | Q | 0.03 | 0.008 | 2.52 (1.68-3.79) | .000009 | — | — | — | — |
| 11 | V | 0.17 | 0.15 | 1.42 (1.19-1.69) | .00008 | 1.28 (1.06-1.55) | .008 | 1.29 (1.07-1.55) | .008 |
| 37 | Y | 0.41 | 0.3 | 1.27 (1.11-1.44) | .0004 | 1.21 (1.06-1.38) | .005 | 1.21 (1.06-1.38) | .005 |

Table Caption: Most significant association results for amino acids in HLA molecules and risk of immediate hypersensitivity to penicillins

### Table E4.

| Drug | Ethnicity | No. of cases | Case MAF | OR (95% CI) | P value |
| --- | --- | --- | --- | --- | --- |
| Cephalosporins | Caucasian | 162 | 0.01 | 1.61 (0.59-4.33) | .34 |
| Cefaclor | Caucasian | 23 | — | — | — |
| Cefazolin | Caucasian | 17 | 0.06 | 6.26(1.35-29.98) | .01 |
| Cefotaxime | Italian | 17 | — | — | — |
| Ceftazidime | Italian | 17 | — | — | — |
| Ceftriaxone | Italian | 48 | — | — | — |
| Cefuroxime | Caucasian | 14 | 0.04 | 2.73 (0.34-21.69) | .34 |
| Penicillins | Caucasian | 501 | 0.03 | 2.85 (1.85-4.35) | 1.2 × 10−6 |
| Amoxicillin | Caucasian | 166 | 0.04 | 3.31 (.80-6.10) | .0001 |
| Ampicillin | Italian | 29 | 0.01 | 1.88 (0.23-6.10) | .54 |
| Co-amoxiclav | Caucasian | 218 | 0.03 | 2.44 (1.31-4.51) | .004 |
| Phenoxymethylpenicillin | Caucasian | 25 | 0.08 | 7.35 (2.37-22.95) | .0005 |
| Piperacillin | Caucasian | 18 | 0.03 | 2.17 (0.26-17.78) | .46 |
| Bacampicillin | Italian | 21 | 0.02 | 1.98 (0.24-16.10) | .52 |

Table Caption: Association between glutamate at position 10 in the HLA-DRB1∗10:01 molecule and risk of immediate hypersensitivity stratified by class of BL and individual molecules

### Table E5.

|   | MAF |  | Association analysis |  | Association conditioned for HLA haplotype |  |
| --- | --- | --- | --- | --- | --- | --- |
| Cases | Controls | OR (95% CI) | P value | OR (95% CI) | P value |  |
| HLA-DRB1∗01:03 | 0.02 | 0.007 | 5.21 (2.45-11.09) | 1.8 × 10−5 | 5.45 (2.56-11.61) | 1.1 × 10−5 |
| HLA-DQA1∗01:05 | 0.03 | 0.01 | 3.42 (1.90-6.15) | 4.0 × 10−5 | — | — |
| HLA-DPB1∗13:01 | 0.06 | 0.02 | 2.24 (1.47-3.42) | 1.8 × 10−4 | 2.28 (1.49-3.48) | 1.4 × 10−4 |
| HLA-DRB1∗10:01 | 0.03 | 0.008 | 2.85 (1.60-5.10) | 3.9 × 10−4 | — | — |

Table Caption: Effect size of the association of the most significant HLA risk alleles in the amoxicillin clavulanic acid analysis

### Table E6.

|   | No. of cases | MAF |  | OR (95% CI) | P value |
| --- | --- | --- | --- | --- | --- |
| Cases | Controls |  |  |  |  |
| Italians | 96 | 0.005 | 0.002 | 4.08 (0.39-42.02) | .2 |
| Spanish | 101 | 0.040 | 0.005 | 9.53 (4.00-22.71) | 3.5 × 10−7 |
| Northern Europeans | 17 | 0 | 0.009 | — | — |

Table Caption: Effect size of HLA-DRB1∗01:03 and amoxicillin clavulanic acid–induced immediate hypersensitivity reactions across the different ethnic clusters

### Table E7.

| Drug | Ethnicity∗ | No. of cases | Case MAF | OR (95% CI) | P value |
| --- | --- | --- | --- | --- | --- |
| Cephalosporins | Caucasian | 162 | 0.05 | 0.64 (0.77-1.81) | .77 |
| Cefaclor | Caucasian | 23 | 0.09 | 0.58 (0.34-4.90) | .34 |
| Cefazolin | Caucasian | 17 | 0.06 | 0.34 (0.63-5.81) | .63 |
| Cefotaxime | Italian | 17 | — | — | — |
| Ceftazidime | Italian | 17 | — | — | — |
| Ceftriaxone | Italian | 48 | — | — | — |
| Cefuroxime | Caucasian | 14 | 0.05 | 0.18 (0.76-10.15) | .76 |
| Penicillins | Caucasian | 501 | 0.08 | 1.78 (1.36-2.32) | 2.3 × 10−5 |
| Amoxicillin | Caucasian | 166 | 0.11 | 2.93 (2.05-4.19) | 3.8 × 10−9 |
| Ampicillin | Italian | 29 | 0.02 | 0.29 (0.04-2.07) | .22 |
| Co-amoxiclav | Caucasian | 218 | 0.07 | 1.48 (0.99-2.21) | .06 |
| Phenoxymethylpenicillin | Caucasian | 25 | 0.02 | 1.59 (0.34-7.46) | .56 |
| Piperacillin | Caucasian | 18 | 0.11 | 2.13 (0.71-6,40) | .18 |
| Bacampicillin | Italian | 21 | 0.07 | 1.33 (0.39-4.51) | .65 |

Table Caption: Effect size of rs71437970 with immediate hypersensitivity reactions broken down by drug classes and individual drugs

### Table E8.

|   | No. of cases | Minor allele frequency |  | OR (95% CI) | P value |
| --- | --- | --- | --- | --- | --- |
| Cases | Controls |  |  |  |  |
| Italians | 52 | 0.15 | 0.05 | 2.91 (1.59-5.32) | .0005 |
| Spanish | 80 | 0.09 | 0.04 | 2.55 (1.42-4.57) | .002 |
| North Europeans | 24 | 0.12 | 0.04 | 3.89 (1.54-9.86) | .004 |

Table Caption: Effect size of rs71437970 and amoxicillin-induced immediate hypersensitivity reactions across the different ethnic clusters

## References

1. Dona I., Barrionuevo E., Blanca-Lopez N., Torres M.J., Fernandez T.D., Mayorga C. Trends in hypersensitivity drug reactions: more drugs, more response patterns, more heterogeneity. J Investig Allergol Clin Immunol. 2014;24:143–153. quiz 1 p following 153.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/25011351/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Investig%20Allergol%20Clin%20Immunol&title=Trends%20in%20hypersensitivity%20drug%20reactions:%20more%20drugs,%20more%20response%20patterns,%20more%20heterogeneity&author=I.%20Dona&author=E.%20Barrionuevo&author=N.%20Blanca-Lopez&author=M.J.%20Torres&author=T.D.%20Fernandez&volume=24&publication_year=2014&pages=143-153&pmid=25011351&)

2. Castells M., Khan D.A., Phillips E.J. Penicillin allergy. N Engl J Med. 2019;381:2338–2351. doi: 10.1056/NEJMra1807761.  [DOI](https://doi.org/10.1056/NEJMra1807761) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31826341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Penicillin%20allergy&author=M.%20Castells&author=D.A.%20Khan&author=E.J.%20Phillips&volume=381&publication_year=2019&pages=2338-2351&pmid=31826341&doi=10.1056/NEJMra1807761&)

3. Pichler W.J. Delayed drug hypersensitivity reactions. Ann Intern Med. 2003;139:683–693. doi: 10.7326/0003-4819-139-8-200310210-00012.  [DOI](https://doi.org/10.7326/0003-4819-139-8-200310210-00012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14568857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Delayed%20drug%20hypersensitivity%20reactions&author=W.J.%20Pichler&volume=139&publication_year=2003&pages=683-693&pmid=14568857&doi=10.7326/0003-4819-139-8-200310210-00012&)

4. Torres M.J., Salas M., Ariza A., Fernandez T.D. Understanding the mechanisms in accelerated drug reactions. Curr Opin Allergy Clin Immunol. 2016;16:308–314. doi: 10.1097/ACI.0000000000000285.  [DOI](https://doi.org/10.1097/ACI.0000000000000285) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27285487/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Allergy%20Clin%20Immunol&title=Understanding%20the%20mechanisms%20in%20accelerated%20drug%20reactions&author=M.J.%20Torres&author=M.%20Salas&author=A.%20Ariza&author=T.D.%20Fernandez&volume=16&publication_year=2016&pages=308-314&pmid=27285487&doi=10.1097/ACI.0000000000000285&)

5. Jerschow E., Lin R.Y., Scaperotti M.M., McGinn A.P. Fatal anaphylaxis in the United States, 1999-2010: temporal patterns and demographic associations. J Allergy Clin Immunol. 2014;134:1318–1328.e7. doi: 10.1016/j.jaci.2014.08.018.  [DOI](https://doi.org/10.1016/j.jaci.2014.08.018) | [PMC free article](/articles/PMC4260987/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25280385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Fatal%20anaphylaxis%20in%20the%20United%20States,%201999-2010:%20temporal%20patterns%20and%20demographic%20associations&author=E.%20Jerschow&author=R.Y.%20Lin&author=M.M.%20Scaperotti&author=A.P.%20McGinn&volume=134&publication_year=2014&pages=1318-1328.e7&pmid=25280385&doi=10.1016/j.jaci.2014.08.018&)

6. Weiss M.E., Adkinson N.F. Immediate hypersensitivity reactions to penicillin and related antibiotics. Clin Allergy. 1988;18:515–540. doi: 10.1111/j.1365-2222.1988.tb02904.x.  [DOI](https://doi.org/10.1111/j.1365-2222.1988.tb02904.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2977302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Allergy&title=Immediate%20hypersensitivity%20reactions%20to%20penicillin%20and%20related%20antibiotics&author=M.E.%20Weiss&author=N.F.%20Adkinson&volume=18&publication_year=1988&pages=515-540&pmid=2977302&doi=10.1111/j.1365-2222.1988.tb02904.x&)

7. Bhattacharya S. The facts about penicillin allergy: a review. J Adv Pharm Technol Res. 2010;1:11–17.  [PMC free article](/articles/PMC3255391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22247826/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Adv%20Pharm%20Technol%20Res&title=The%20facts%20about%20penicillin%20allergy:%20a%20review&author=S.%20Bhattacharya&volume=1&publication_year=2010&pages=11-17&pmid=22247826&)

8. Romano A., Gaeta F., Arribas Poves M.F., Valluzzi R.L. Cross-reactivity among beta-lactams. Curr Allergy Asthma Rep. 2016;16:24. doi: 10.1007/s11882-016-0594-9.  [DOI](https://doi.org/10.1007/s11882-016-0594-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26898316/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Allergy%20Asthma%20Rep&title=Cross-reactivity%20among%20beta-lactams&author=A.%20Romano&author=F.%20Gaeta&author=M.F.%20Arribas%20Poves&author=R.L.%20Valluzzi&volume=16&publication_year=2016&pages=24&pmid=26898316&doi=10.1007/s11882-016-0594-9&)

9. Blumenthal K.G., Peter J.G., Trubiano J.A., Phillips E.J. Antibiotic allergy. Lancet. 2019;393:183–198. doi: 10.1016/S0140-6736(18)32218-9.  [DOI](https://doi.org/10.1016/S0140-6736(18)32218-9) | [PMC free article](/articles/PMC6563335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30558872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Antibiotic%20allergy&author=K.G.%20Blumenthal&author=J.G.%20Peter&author=J.A.%20Trubiano&author=E.J.%20Phillips&volume=393&publication_year=2019&pages=183-198&pmid=30558872&doi=10.1016/S0140-6736(18)32218-9&)

10. Apter A.J., Schelleman H., Walker A., Addya K., Rebbeck T. Clinical and genetic risk factors of self-reported penicillin allergy. J Allergy Clin Immunol. 2008;122:152–158. doi: 10.1016/j.jaci.2008.03.037.  [DOI](https://doi.org/10.1016/j.jaci.2008.03.037) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18538381/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Clinical%20and%20genetic%20risk%20factors%20of%20self-reported%20penicillin%20allergy&author=A.J.%20Apter&author=H.%20Schelleman&author=A.%20Walker&author=K.%20Addya&author=T.%20Rebbeck&volume=122&publication_year=2008&pages=152-158&pmid=18538381&doi=10.1016/j.jaci.2008.03.037&)

11. Gueant J.L., Romano A., Cornejo-Garcia J.A., Oussalah A., Chery C., Blanca-Lopez N. HLA-DRA variants predict penicillin allergy in genome-wide fine-mapping genotyping. J Allergy Clin Immunol. 2015;135:253–259. doi: 10.1016/j.jaci.2014.07.047.  [DOI](https://doi.org/10.1016/j.jaci.2014.07.047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25224099/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=HLA-DRA%20variants%20predict%20penicillin%20allergy%20in%20genome-wide%20fine-mapping%20genotyping&author=J.L.%20Gueant&author=A.%20Romano&author=J.A.%20Cornejo-Garcia&author=A.%20Oussalah&author=C.%20Chery&volume=135&publication_year=2015&pages=253-259&pmid=25224099&doi=10.1016/j.jaci.2014.07.047&)

12. Mirakian R., Leech S.C., Krishna M.T., Richter A.G., Huber P.A., Farooque S. Management of allergy to penicillins and other beta-lactams. Clin Exp Allergy. 2015;45:300–327. doi: 10.1111/cea.12468.  [DOI](https://doi.org/10.1111/cea.12468) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25623506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Exp%20Allergy&title=Management%20of%20allergy%20to%20penicillins%20and%20other%20beta-lactams&author=R.%20Mirakian&author=S.C.%20Leech&author=M.T.%20Krishna&author=A.G.%20Richter&author=P.A.%20Huber&volume=45&publication_year=2015&pages=300-327&pmid=25623506&doi=10.1111/cea.12468&)

13. Shen Y., Nicoletti P., Floratos A., Pirmohamed M., Molokhia M., Geppetti P. Genome-wide association study of serious blistering skin rash caused by drugs. Pharmacogenomics J. 2012;12:96–104. doi: 10.1038/tpj.2010.84.  [DOI](https://doi.org/10.1038/tpj.2010.84) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21221126/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genome-wide%20association%20study%20of%20serious%20blistering%20skin%20rash%20caused%20by%20drugs&author=Y.%20Shen&author=P.%20Nicoletti&author=A.%20Floratos&author=M.%20Pirmohamed&author=M.%20Molokhia&volume=12&publication_year=2012&pages=96-104&pmid=21221126&doi=10.1038/tpj.2010.84&)

14. Nelson M.R., Bryc K., King K.S., Indap A., Boyko A.R., Novembre J. The Population Reference Sample, POPRES: a resource for population, disease, and pharmacological genetics research. Am J Hum Genet. 2008;83:347–358. doi: 10.1016/j.ajhg.2008.08.005.  [DOI](https://doi.org/10.1016/j.ajhg.2008.08.005) | [PMC free article](/articles/PMC2556436/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18760391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=The%20Population%20Reference%20Sample,%20POPRES:%20a%20resource%20for%20population,%20disease,%20and%20pharmacological%20genetics%20research&author=M.R.%20Nelson&author=K.%20Bryc&author=K.S.%20King&author=A.%20Indap&author=A.R.%20Boyko&volume=83&publication_year=2008&pages=347-358&pmid=18760391&doi=10.1016/j.ajhg.2008.08.005&)

15. Tryka K.A., Hao L., Sturcke A., Jin Y., Wang Z.Y., Ziyabari L. NCBI's Database of Genotypes and Phenotypes: dbGaP. Nucleic Acids Res. 2014;42:D975–D979. doi: 10.1093/nar/gkt1211.  [DOI](https://doi.org/10.1093/nar/gkt1211) | [PMC free article](/articles/PMC3965052/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24297256/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=NCBI's%20Database%20of%20Genotypes%20and%20Phenotypes:%20dbGaP&author=K.A.%20Tryka&author=L.%20Hao&author=A.%20Sturcke&author=Y.%20Jin&author=Z.Y.%20Wang&volume=42&publication_year=2014&pages=D975-D979&pmid=24297256&doi=10.1093/nar/gkt1211&)

16. Price A.L., Patterson N.J., Plenge R.M., Weinblatt M.E., Shadick N.A., Reich D. Principal components analysis corrects for stratification in genome-wide association studies. Nat Genet. 2006;38:904–909. doi: 10.1038/ng1847.  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=A.L.%20Price&author=N.J.%20Patterson&author=R.M.%20Plenge&author=M.E.%20Weinblatt&author=N.A.%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

17. Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M.A., Bender D. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet. 2007;81:559–575. doi: 10.1086/519795.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S.%20Purcell&author=B.%20Neale&author=K.%20Todd-Brown&author=L.%20Thomas&author=M.A.%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

18. Pruim R.J., Welch R.P., Sanna S., Teslovich T.M., Chines P.S., Gliedt T.P. LocusZoom: regional visualization of genome-wide association scan results. Bioinformatics. 2010;26:2336–2337. doi: 10.1093/bioinformatics/btq419.  [DOI](https://doi.org/10.1093/bioinformatics/btq419) | [PMC free article](/articles/PMC2935401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20634204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LocusZoom:%20regional%20visualization%20of%20genome-wide%20association%20scan%20results&author=R.J.%20Pruim&author=R.P.%20Welch&author=S.%20Sanna&author=T.M.%20Teslovich&author=P.S.%20Chines&volume=26&publication_year=2010&pages=2336-2337&pmid=20634204&doi=10.1093/bioinformatics/btq419&)

19. Zheng X., Shen J., Cox C., Wakefield J.C., Ehm M.G., Nelson M.R. HIBAG--HLA genotype imputation with attribute bagging. Pharmacogenomics J. 2014;14:192–200. doi: 10.1038/tpj.2013.18.  [DOI](https://doi.org/10.1038/tpj.2013.18) | [PMC free article](/articles/PMC3772955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23712092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=HIBAG--HLA%20genotype%20imputation%20with%20attribute%20bagging&author=X.%20Zheng&author=J.%20Shen&author=C.%20Cox&author=J.C.%20Wakefield&author=M.G.%20Ehm&volume=14&publication_year=2014&pages=192-200&pmid=23712092&doi=10.1038/tpj.2013.18&)

20. Raychaudhuri S., Sandor C., Stahl E.A., Freudenberg J., Lee H.S., Jia X. Five amino acids in three HLA proteins explain most of the association between MHC and seropositive rheumatoid arthritis. Nat Genet. 2012;44:291–296. doi: 10.1038/ng.1076.  [DOI](https://doi.org/10.1038/ng.1076) | [PMC free article](/articles/PMC3288335/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22286218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Five%20amino%20acids%20in%20three%20HLA%20proteins%20explain%20most%20of%20the%20association%20between%20MHC%20and%20seropositive%20rheumatoid%20arthritis&author=S.%20Raychaudhuri&author=C.%20Sandor&author=E.A.%20Stahl&author=J.%20Freudenberg&author=H.S.%20Lee&volume=44&publication_year=2012&pages=291-296&pmid=22286218&doi=10.1038/ng.1076&)

21. Heap G.A., Weedon M.N., Bewshea C.M., Singh A., Chen M., Satchwell J.B. HLA-DQA1-HLA-DRB1 variants confer susceptibility to pancreatitis induced by thiopurine immunosuppressants. Nat Genet. 2014;46:1131–1134. doi: 10.1038/ng.3093.  [DOI](https://doi.org/10.1038/ng.3093) | [PMC free article](/articles/PMC6379053/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25217962/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=HLA-DQA1-HLA-DRB1%20variants%20confer%20susceptibility%20to%20pancreatitis%20induced%20by%20thiopurine%20immunosuppressants&author=G.A.%20Heap&author=M.N.%20Weedon&author=C.M.%20Bewshea&author=A.%20Singh&author=M.%20Chen&volume=46&publication_year=2014&pages=1131-1134&pmid=25217962&doi=10.1038/ng.3093&)

22. Macy E., Ngor E.W. Safely diagnosing clinically significant penicillin allergy using only penicilloyl-poly-lysine, penicillin, and oral amoxicillin. J Allergy Clin Immunol Pract. 2013;1:258–263. doi: 10.1016/j.jaip.2013.02.002.  [DOI](https://doi.org/10.1016/j.jaip.2013.02.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24565482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol%20Pract&title=Safely%20diagnosing%20clinically%20significant%20penicillin%20allergy%20using%20only%20penicilloyl-poly-lysine,%20penicillin,%20and%20oral%20amoxicillin&author=E.%20Macy&author=E.W.%20Ngor&volume=1&publication_year=2013&pages=258-263&pmid=24565482&doi=10.1016/j.jaip.2013.02.002&)

23. Pirmohamed M., Ostrov D.A., Park B.K. New genetic findings lead the way to a better understanding of fundamental mechanisms of drug hypersensitivity. J Allergy Clin Immunol. 2015;136:236–244. doi: 10.1016/j.jaci.2015.06.022.  [DOI](https://doi.org/10.1016/j.jaci.2015.06.022) | [PMC free article](/articles/PMC4534769/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26254050/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=New%20genetic%20findings%20lead%20the%20way%20to%20a%20better%20understanding%20of%20fundamental%20mechanisms%20of%20drug%20hypersensitivity&author=M.%20Pirmohamed&author=D.A.%20Ostrov&author=B.K.%20Park&volume=136&publication_year=2015&pages=236-244&pmid=26254050&doi=10.1016/j.jaci.2015.06.022&)

24. Mallal S., Nolan D., Witt C., Masel G., Martin A.M., Moore C. Association between presence of HLA-B∗5701, HLA-DR7, and HLA-DQ3 and hypersensitivity to HIV-1 reverse-transcriptase inhibitor abacavir. Lancet. 2002;359:727–732. doi: 10.1016/s0140-6736(02)07873-x.  [DOI](https://doi.org/10.1016/s0140-6736(02)07873-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11888582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Association%20between%20presence%20of%20HLA-B%E2%88%975701,%20HLA-DR7,%20and%20HLA-DQ3%20and%20hypersensitivity%20to%20HIV-1%20reverse-transcriptase%20inhibitor%20abacavir&author=S.%20Mallal&author=D.%20Nolan&author=C.%20Witt&author=G.%20Masel&author=A.M.%20Martin&volume=359&publication_year=2002&pages=727-732&pmid=11888582&doi=10.1016/s0140-6736(02)07873-x&)

25. Rauch A., Nolan D., Martin A., McKinnon E., Almeida C., Mallal S. Prospective genetic screening decreases the incidence of abacavir hypersensitivity reactions in the Western Australian HIV cohort study. Clin Infect Dis. 2006;43:99–102. doi: 10.1086/504874.  [DOI](https://doi.org/10.1086/504874) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16758424/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Prospective%20genetic%20screening%20decreases%20the%20incidence%20of%20abacavir%20hypersensitivity%20reactions%20in%20the%20Western%20Australian%20HIV%20cohort%20study&author=A.%20Rauch&author=D.%20Nolan&author=A.%20Martin&author=E.%20McKinnon&author=C.%20Almeida&volume=43&publication_year=2006&pages=99-102&pmid=16758424&doi=10.1086/504874&)

26. Fernandez C.A., Smith C., Yang W., Date M., Bashford D., Larsen E. HLA-DRB1∗07:01 is associated with a higher risk of asparaginase allergies. Blood. 2014;124:1266–1276. doi: 10.1182/blood-2014-03-563742.  [DOI](https://doi.org/10.1182/blood-2014-03-563742) | [PMC free article](/articles/PMC4141516/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24970932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=HLA-DRB1%E2%88%9707:01%20is%20associated%20with%20a%20higher%20risk%20of%20asparaginase%20allergies&author=C.A.%20Fernandez&author=C.%20Smith&author=W.%20Yang&author=M.%20Date&author=D.%20Bashford&volume=124&publication_year=2014&pages=1266-1276&pmid=24970932&doi=10.1182/blood-2014-03-563742&)

27. Daly A.K., Donaldson P.T., Bhatnagar P., Shen Y., Pe'er I., Floratos A. HLA-B∗5701 genotype is a major determinant of drug-induced liver injury due to flucloxacillin. Nat Genet. 2009;41:816–819. doi: 10.1038/ng.379.  [DOI](https://doi.org/10.1038/ng.379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19483685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=HLA-B%E2%88%975701%20genotype%20is%20a%20major%20determinant%20of%20drug-induced%20liver%20injury%20due%20to%20flucloxacillin&author=A.K.%20Daly&author=P.T.%20Donaldson&author=P.%20Bhatnagar&author=Y.%20Shen&author=I.%20Pe'er&volume=41&publication_year=2009&pages=816-819&pmid=19483685&doi=10.1038/ng.379&)

28. Lucena M.I., Molokhia M., Shen Y., Urban T.J., Aithal G.P., Andrade R.J. Susceptibility to amoxicillin-clavulanate-induced liver injury is influenced by multiple HLA class I and II alleles. Gastroenterology. 2011;141:338–347. doi: 10.1053/j.gastro.2011.04.001.  [DOI](https://doi.org/10.1053/j.gastro.2011.04.001) | [PMC free article](/articles/PMC3129430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21570397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Susceptibility%20to%20amoxicillin-clavulanate-induced%20liver%20injury%20is%20influenced%20by%20multiple%20HLA%20class%20I%20and%20II%20alleles&author=M.I.%20Lucena&author=M.%20Molokhia&author=Y.%20Shen&author=T.J.%20Urban&author=G.P.%20Aithal&volume=141&publication_year=2011&pages=338-347&pmid=21570397&doi=10.1053/j.gastro.2011.04.001&)

29. Monshi M.M., Faulkner L., Gibson A., Jenkins R.E., Farrell J., Earnshaw C.J. Human leukocyte antigen (HLA)-B∗57:01-restricted activation of drug-specific T cells provides the immunological basis for flucloxacillin-induced liver injury. Hepatology. 2013;57:727–739. doi: 10.1002/hep.26077.  [DOI](https://doi.org/10.1002/hep.26077) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22987284/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Human%20leukocyte%20antigen%20(HLA)-B%E2%88%9757:01-restricted%20activation%20of%20drug-specific%20T%20cells%20provides%20the%20immunological%20basis%20for%20flucloxacillin-induced%20liver%20injury&author=M.M.%20Monshi&author=L.%20Faulkner&author=A.%20Gibson&author=R.E.%20Jenkins&author=J.%20Farrell&volume=57&publication_year=2013&pages=727-739&pmid=22987284&doi=10.1002/hep.26077&)

30. Kim S.H., Saide K., Farrell J., Faulkner L., Tailor A., Ogese M. Characterization of amoxicillin- and clavulanic acid-specific T cells in patients with amoxicillin-clavulanate-induced liver injury. Hepatology. 2015;62:887–899. doi: 10.1002/hep.27912.  [DOI](https://doi.org/10.1002/hep.27912) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25998949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Characterization%20of%20amoxicillin-%20and%20clavulanic%20acid-specific%20T%20cells%20in%20patients%20with%20amoxicillin-clavulanate-induced%20liver%20injury&author=S.H.%20Kim&author=K.%20Saide&author=J.%20Farrell&author=L.%20Faulkner&author=A.%20Tailor&volume=62&publication_year=2015&pages=887-899&pmid=25998949&doi=10.1002/hep.27912&)

31. Li M., Anderson G.D., Phillips B.R., Kong W., Shen D.D., Wang J. Interactions of amoxicillin and cefaclor with human renal organic anion and peptide transporters. Drug Metab Dispos. 2006;34:547–555. doi: 10.1124/dmd.105.006791.  [DOI](https://doi.org/10.1124/dmd.105.006791) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16434549/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Interactions%20of%20amoxicillin%20and%20cefaclor%20with%20human%20renal%20organic%20anion%20and%20peptide%20transporters&author=M.%20Li&author=G.D.%20Anderson&author=B.R.%20Phillips&author=W.%20Kong&author=D.D.%20Shen&volume=34&publication_year=2006&pages=547-555&pmid=16434549&doi=10.1124/dmd.105.006791&)
