# Impact of Variants in the ATIC and ARID5B Genes on Therapeutic Failure with Imatinib in Patients with Chronic Myeloid Leukemia

## Metadata
**Authors:** Karla Beatriz Cardias Cereja Pantoja, Tereza Cristina de Brito Azevedo, Darlen Cardoso de Carvalho, Natasha Monte, Amanda de Nazaré Cohen Paes, Maria Clara da Costa Barros, Lui Wallacy Morikawa Souza Vinagre, Ana Rosa Sales de Freitas, Rommel Mario Rodríguez Burbano, Paulo Pimentel de Assumpção, Sidney Emanuel Batista dos Santos, Marianne Rodrigues Fernandes, Ney Pereira Carneiro dos Santos
**Journal:** Genes
**Date:** 2022 Feb 10
**DOI:** [10.3390/genes13020330](https://doi.org/10.3390/genes13020330)
**PMID:** 35205374
**PMCID:** PMC8872593
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8872593/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8872593/pdf/genes-13-00330.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8872593/pdf/genes-13-00330.pdf)

## Abstract

Chronic myeloid leukemia (CML) is a myeloproliferative neoplasm derived from the balanced reciprocal translocation of chromosomes 9 and 22 t (9q34 and 22q11), which leads to the formation of the Philadelphia chromosome and fusion of the BCR-ABL genes. The first-line treatment for CML is imatinib, a tyrosine kinase inhibitor that acts on the BCR-ABL protein. However, even though it is a target-specific drug, about 25% of patients do not respond to this treatment. The resistance mechanisms involved in this process have been investigated and studies have shown that germinal alterations can influence this mechanism. The aim of this work was to investigate 32 polymorphisms in 24 genes of carcinogenic pathway to verify the influence of these genetic variants on the response to treatment with imatinib. Our results demonstrated that individuals with the recessive GG genotype for the rs2372536 variant in the ATIC gene are approximately three times more likely to experience treatment failure with imatinib (p = 0.045, HR = 2.726, 95% CI = 0.9986–7.441), as well as individuals with the TT genotype for the rs10821936 variant in the ARID5B gene, who also have a higher risk for treatment failure with imatinib over time (p = 0.02, HR = 0.4053, IC 95% = 0.1802–0.911). In conclusion, we show that variants in the ATIC and ARIDB5 gene, never screened in previous studies, could potentially influence the therapeutic response to imatinib in patients treated for CML.

Keywords: chronic myeloid leukemia, imatinib, ATIC gene, ARIDB5 gene, pharmacogenomics

## 1. Introduction

Chronic myeloid leukemia (CML) has as its main characteristic the reciprocal and balanced translocation of chromosomes 9 and 22 t (9q34 and 22q11), which results in the Philadelphia chromosome and fusion of the *BCR-ABL* genes [[1](#B1-genes-13-00330),[2](#B2-genes-13-00330)]. The expression of this gene is constitutive with tyrosine kinase activity, and it is responsible for leukemogenesis and maintenance of carcinogenic activity in CML [[3](#B3-genes-13-00330),[4](#B4-genes-13-00330)].

Treatment of CML is performed with the tyrosine kinase inhibitor (TKI), imatinib, which acts directly on the BCR-ABL protein by decreasing its intracellular activity and, thus, controlling the carcinogenic environment [[5](#B5-genes-13-00330)]. However, even though this is a target-specific drug, the response to this treatment is variable, therefore, about 25% of patients are not responsive to imatinib [[4](#B4-genes-13-00330),[6](#B6-genes-13-00330)].

Variations in response to treatment can be influenced by several factors, including alterations in drug metabolism genes. Genetic variants can alter gene expression and, thus, modulate the interaction of the expressed protein with the drug, making the response inefficient [[7](#B7-genes-13-00330),[8](#B8-genes-13-00330),[9](#B9-genes-13-00330),[10](#B10-genes-13-00330)].

Another factor that can interfere with treatment responses is the genetic composition of a population. It is known that the responses among populations vary worldwide, due to different frequencies of genetic variants in genes involved in absorption, distribution, metabolism, and excretion (ADME) of drugs [[11](#B11-genes-13-00330),[12](#B12-genes-13-00330),[13](#B13-genes-13-00330)]. In addition, in this study, the population investigated is highly mixed, and it is important to carry out a genomic control based on genetic ancestry, therefore, that there is no population substructuring and ancestry is not a confounding factor in the analyses [[14](#B14-genes-13-00330),[15](#B15-genes-13-00330)].

Thus, this study investigated 32 polymorphisms in 24 carcinogenic pathway genes: *ABCC1*, *ABCC2*, *ABCC3*, *AMPD1*, *ARID5B*, *ATIC*, *CCND1*, *CDKN2A*, *CEBPE*, *GGH*, *IKZF1*, *ITPA*, *MTHFD1*, *MTHFR*, *MTRR*, *NALCN*, *NOS3*, *PIP4K2A*, *SHMT1*, *SLCO1B1*, *SLCO1B3*, *TLR4*, *TNFAIP3*, and *TPMT*, aiming to verify the influence of these genetic variants on the response to treatment with imatinib.

## 2. Materials and Methods

### 2.1. Ethics, Consent, and Permissions

This study was approved by the Research Ethics Committee of the participating institutions, at the Ophir Loyola Hospital under license number 1.575.920/2016 and at the Núcleo de Pesquisas em Oncologia (NPO) under protocol number 3.354.571/2019. All participants agreed to participate in the research and signed an informed consent form allowing the use of their clinical and genetic data.

### 2.2. Investigated Population

We investigated a total of 165 patients diagnosed with CML, followed for at least 1 year of treatment at Hospital Ophir Loyola, a reference hospital in the onco-hematology service in the city of Belém do Pará, in Northern Brazil. All patients started treatment with imatinib mesylate and had a detailed clinical follow-up. For the analyses, the patients were divided into two groups: patients who responded well to treatment and patients who did not respond well to treatment. The criteria used to define the hematologic and molecular response followed the National Comprehensive Cancer Network [[16](#B16-genes-13-00330)].

### 2.3. Selected Markers

For marker selection, the criteria were based on PharmGKB, NCBI, and Ensembl databases, as well as data available in literature regarding important variables for the carcinogenic pathway. The description of markers can be found in the [Supplementary Materials Table S1](#app1-genes-13-00330).

### 2.4. DNA Extraction and Quantification

Genetic material was extracted from peripheral blood collected in EDTA tubes and using an Axy PrepTM Blood Genomic DNA Miniprep kit (Axygen Biotechnology, San Francisco, CA, USA), following the manufacturer’s instructions. The DNA concentration and purity were measured using a NanoDrop 1000 spectrophotometer (Thermo Scientific NanoDrop 1000, NanoDrop Technologies, Wilmington, DE, USA).

### 2.5. Genotyping

Genotyping of the samples was performed on a QuantStudio™ 12K Flex Real-Time PCR system (Applied Biosystems, Life Technologies, Carlsbad, CA, USA) using real-time PCR technology (TaqMan OpenArray Genotyping) by allelic discrimination, following all the manufacturer’s recommendations.

### 2.6. Genetic Ancestry

An ancestry analysis was performed as described by Ramos et al. [[17](#B17-genes-13-00330)] using 61 autosomal ancestry informative markers (AIMs) in three multiplex PCR reactions, aiming to accurately estimate the individual and global interethnic mix [[14](#B14-genes-13-00330)]. Amplicons were analyzed using an ABI Prism 3130 sequencer (Thermo Fisher Scientific, Waltham, MA, USA) and Gene Mapper ID v.3.2 software (Thermo Fisher Scientific, Waltham, MA, USA). The proportions of individual genetic ancestors were estimated using the STRUCTURE v.2.3.3 software (Stanford University, Stanford, CA, USA), assuming three parental populations. This analysis was performed to control a possible population substructure, as the investigated population was highly mixed.

### 2.7. Statistical Analysis

To be included in the statistical analyses, the genotyping data needed at least 70% in coverage. The allelic and genotypic distribution is shown in [Supplementary Table S2](#app1-genes-13-00330).

The statistical analyses were run in SNPassoc library in RStudio v.3.6.1 software (Boston, MA, USA). Differences in the categorical variable (sex) were tested using Pearson’s chi square, while the quantitative variable (mean age) was evaluated using Student’s *t*-test. The ancestry indices were compared between the groups using the Mann–Whitney test. Multiple logistic regressions were used to assess possible associations between the polymorphisms and the response to treatment with imatinib, by estimating the odds ratios (ORs) and their 95% confidence intervals (CIs). The Kaplan–Meier survival analysis was used to estimate possible differences in the time of loss of response for each genotype by estimating the hazard ratio (HR). For this statistical test, we evaluated the variable “time of treatment failure” (TTF)—an event in which treatment was changed, due to absence of molecular or cytogenetic responses or intolerance to treatment. A significance level of *p* < 0.05 was considered for all the statistical analyses.

## 3. Results

The clinical epidemiological data are shown in [Table 1](#genes-13-00330-t001), showing that, among the 165 patients included in this study, 103 (63.1%) patients had an excellent response to treatment and 62 (36.9%) patients did not. When comparing the variable “age at diagnosis” between the groups (responders and non-responders) we found no significant difference (*p*-value = 0.451), the same result was found when gender distribution was analyzed (*p*-value = 0.078). Regarding the genetic ancestry in both groups, we also found no significant difference between them.

### Table 1.

| Variable | Responders | No Responders | p-Value |
| --- | --- | --- | --- |
| 103 (63.1%) | 62 (36.9%) |  |  |
| Age (years) | 48.50 ± 15.07 | 46.76 ± 15.00 | 0.451 |
| Sex (%) |   |   | 0.078 |
| Men | 64 (62.1) | 29 (46.7) |   |
| Women | 39 (37.9) | 33 (53.3) |   |
| Ancestry Mean |   |  |  |
| European | 0.468 ± 0.147 | 0.458 ± 0.147 | 0.797 |
| Amerindian | 0.299 ± 0.132 | 0.319 ± 0.133 | 0.604 |
| African | 0.233 ± 0.101 | 0.223 ± 0.101 | 0.686 |

Table 1 Caption: Clinical and epidemiological variables of the investigated patients.

Further information on clinical data reveals that 34 (20.2%) patients failed to respond during treatment (mean 47.11 months) and 12 (7.14%) patients were unresponsive from the beginning of treatment.

### 3.1. Genotype and Imatinib Response-Relative Risk Assessment (OR)

Regarding genetic variants, we found no association between responsiveness to therapy at a specific time and the investigated SNPs ([Supplementary Table S3](#app1-genes-13-00330)).

### 3.2. Time of Treatment Failure (TTF)/Risk Analysis over the Response Time (HR)

We investigated the relationship between genetic variants and the variable “time of treatment failure” (TTF) to estimate the risk of treatment failure over time. Our results indicate that the SNVs rs2372536 in the *ATIC* gene and the rs10821936 in the *ARID5B* gene were statistically significant ([Table 2](#genes-13-00330-t002)).

### Table 2.

| Genotype | HR (95% CI) | Lower | Upper | p-Value |
| --- | --- | --- | --- | --- |
| ATIC rs2372536 |   |   |   |   |
| CC/CG vs. GG 1 | 2.726 | 0.9986 | 7.441 | 0.04 |
| CC vs. CG + GG 2 | 1.484 | 0.7235 | 3.046 | 0.3 |
| ARID5B rs10821936 |   |   |   |   |
| TT vs. CT + CC 1 | 0.4053 | 0.1802 | 0.911 | 0.02 |
| CC vs. TT + CT 2 | 0.6114 | 0.2436 | 1.535 | 0.3 |
| TPMT rs1142345 |   |   |   |   |
| TT + CT vs. CC 1 | 0.711 | 0.165 | 3.063 | 0.6 |
| TT vs. CT + CC 2 | 0.9895 | 0.4341 | 2.255 | 1.00 |
| TPMT rs12201199 |   |   |   |   |
| NA 1,3 |   |   |   |   |
| AA vs. AT + TT 2 | 0.6824 | 0.2561 | 1.818 | 0.4 |
| SLC01B1 rs4149056 |   |   |   |   |
| NA 1,3 |   |   |   |   |
| TT vs. CT + CC 2 | 1.109 | 0.3502 | 2.323 | 0.8 |
| ABCC2 rs717620 |   |   |   |   |
| NA 1,3 |   |   |   |   |
| CC vs. CT + TT 2 | 1.909 | 0.1807 | 1.518 | 0.2 |
| ABCC3 rs9895420 |   |   |   |   |
| NA 1,3 |   |   |   |   |
| TT vs. AT + AA 2 | 1.738 | 0.1983 | 1.67 | 0.3 |
| GGH rs11545078 |   |   |   |   |
| NA 1,3 |   |   |   |   |
| GG vs. AG + AA 2 | 1.769 | 0.81 | 3.862 | 0.1 |
| GGH rs3758149 |   |   |   |   |
| GG + AG vs. AA 1 | 0.9296 | 0.2162 | 3.997 | 0.9 |
| GG vs. AG + AA 2 | 1.148 | 0.5384 | 2.446 | 0.7 |
| ATIC rs4673993 |   |   |   |   |
| TT + CT vs. CC 1 | 2.561 | 0.3905 | 0.9388 | 0.06 |
| TT vs. CT + CC 2 | 1.633 | 0.7888 | 3.382 | 0.2 |
| AMPD1 rs17602729 |   |   |   |   |
| NA 1,3 |   |   |   |   |
| GG vs. AG + AA 2 | 0.7357 | 0.2756 | 1.964 | 0.5 |
| CCND1 rs9344 |   |   |   |   |
| GG + AG vs. AA 1 | 1.536 | 0.5155 | 4.576 | 0.4 |
| GG vs. AG + AA 2 | 1.174 | 0.5439 | 2.532 | 0.7 |
| IKZF1 rs4132601 |   |   |   |   |
| TT + GT vs. GG 1 | 0.7569 | 0.1013 | 5.657 | 0.8 |
| TT vs. GT + GG 2 | 1.164 | 0.5258 | 2.579 | 0.7 |
| ITPA rs1127354 |   |   |   |   |
| CC vs. AC 1 | 1.957 | 0.4465 | 8.575 | 0.4 |
| MTRR rs1801394 |   |   |   |   |
| AA + AG vs. GG 1 | 0.8794 | 0.328 | 2.358 | 0.8 |
| AA vs. AG + GG 2 | 0.9615 | 0.4536 | 2.038 | 0.9 |
| MTHFD1 rs2236225 |   |   |   |   |
| GG + AG vs. AA 1 | 0.6535 | 0.2462 | 1.735 | 0.4 |
| GG vs. AG + AA 2 | 0.7404 | 0.3318 | 1.652 | 0.5 |
| NOS3 rs1799983 |   |   |   |   |
| NA 1,3 |   |   |   |   |
| GG vs. GT 2 | 1.052 | 0.2408 | 4.592 | 0.9 |
| MTHFR rs1801133 |   |   |   |   |
| GG + GA vs. AA 1 | 1.445 | 0.4198 | 4.97 | 0.6 |
| GG vs. GA + AA 2 | 0.9837 | 0.4521 | 2.141 | 1.00 |
| TLR4 rs4986790 |   |   |   |   |
| NA 1,3 |   |   |   |   |
| AA vs. AG 2 | 1.545 | 0.6471 | 0.2034 | 0.7 |
| TPMT rs1800460 |   |   |   |   |
| CC vs. CT + TT 1 | 0.6339 | 0.218 | 1.843 | 0.4 |
| NA2 |   |   |   |   |
| SLCO1B1 rs4149015 |   |   |   |   |
| GG vs. AG + AA 1 | 0.8043 | 0.2271 | 2.849 | 0.7 |
| AA vs. GG + AG 2 | 3.472 | 0.288 | 0.4051 | 0.2 |
| GGH rs1800909 |   |   |   |   |
| AA+AG vs. GG 1 | 0.8698 | 0.2995 | 2.526 | 0.8 |
| NA 2 |   |   |   |   |
| NALCN rs7992226 |   |   |   |   |
| AA+AG vs. GG 1 | 0.03471 | 0.42 | 2.552 | 0.9 |
| AA vs. AG + GG 2 | 1.199 | 0.348 | 4.132 | 0.8 |
| SHMT1 rs1979277 |   |   |   |   |
| AA+AG vs. GG 1 | 0.873 | 0.3515 | 2.168 | 0.8 |
| GG vs. AA + AG 2 | 3.361 | 0.6504 | 1.447 | 0.1 |
| SLCO1B1 rs2306283 |   |   |   |   |
| GG+AG vs. AA 1 | 1.733 | 0.6368 | 4.718 | 0.3 |
| GG vs. AG + AA 2 | 1.276 | 0.7837 | 0.4663 |   |
| CEBPE rs2239633 |   |   |   |   |
| GG+AG vs. AA 1 | 1.899 | 0.8027 | 4.493 | 0.1 |
| GG vs. AG + AA 2 | 2.229 | 0.4485 | 0.506 | 0.3 |
| TNFAIP3 rs6920220 |   |   |   |   |
| GG+AG vs. AA 1 | 0.6463 | 0.2426 | 1.722 | 0.4 |
| NA 2 |   |   |   |   |
| PIP4K2A rs7088318 |   |   |   |   |
| AA vs. AC + CC 1 | 0.5205 | 1.921 | 0.1135 | 0.4 |
| NA 2 |   |   |   |   |

Table 2 Caption: Hazard ratio analysis of the genotypes analyzed with the time of treatment failure.

According to our results, we can infer that individuals with the recessive GG genotype for the rs2372536 variant in the *ATIC* gene are approximately three times more likely to experience treatment failure with imatinib (*p* = 0.045, HR = 2.726, 95% CI 0.9986–7.441), as well as individuals with the TT genotype for the rs10821936 variant in the *ARID5B* gene, who also have a higher risk for treatment failure with imatinib over time (*p* = 0.02, HR = 0.4053, IC 95% 0.1802–0.911) ([Figure 1](#genes-13-00330-f001)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3225/8872593/5e6ff118bf85/genes-13-00330-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8872593_genes-13-00330-g001.jpg)

Kaplan–Meier curve demonstrating time of treatment failure (TTF) associated with rs10821936 of ARID5B gene and rs2372536 variant of ATIC.

## 4. Discussion

Imatinib (STI-571is a 2-phenylamino-pyrimidine compound that inhibits the autophosphorylation of the BCR-ABL protein. This happens through the binding of TKI to the ATP receptor (adenosine triphosphate), which does not allow the binding of the phosphate group of the ATP molecule, keeping the protein inactivated. In this way, the entire downstream signaling cascade is turned off and the leukemic cells stop dividing [[4](#B4-genes-13-00330),[5](#B5-genes-13-00330)].

This drug is used primarily for the treatment of CML, gastrointestinal stromal tumors (GISTs), and Philadelphia chromosome-positive acute lymphoblastic leukemia (ALL Ph+) [[18](#B18-genes-13-00330)]. Imatinib was the first TKI that showed efficiency in the treatment of CML, and it was approved by the Food and Drug Administration (FDA) of United States of America and also by Agência Nacional de Vigilância Sanitária (Anvisa of Brazil) in 2001, soon beginning to be used as a first-line treatment for CML.

In addition to inhibiting the BCR-ABL protein, imatinib also works as an inhibitor of other signaling pathways, such as those activated by the platelet-derived growth factor receptor (PDGFR), c-Kit (type III member of kinase receptors), MAPK (mitogen-activated protein kinase), and PI3K/AKT (phosphatidyl inositol 3 kinase), thus, acting in several ways to block cell division [[19](#B19-genes-13-00330)].

Resistance to imatinib therapy occurs in about 25% of patients with chronic myeloid leukemia; the mechanisms involved in this process have been investigated and studies show that genetic and epigenetic alterations can influence it [[7](#B7-genes-13-00330)].

In this study, genetic variants in genes involved in the carcinogenic pathway drugs were investigated in order to understand their influence on the response to treatment with imatinib in patients with CML. Our results demonstrated a significant association between the rs2372536 of the *ATIC* gene and rs10821936 of *ARID5B* gene treatment failure with imatinib.

### 4.1. ATIC

The *ATIC* gene (5-aminoimidazole-4-carboxamide ribonucleotide formyltransferase/IMP cyclohydrolase) it is located on chromosome 2q35 and encodes a bifunctional enzyme that catalyzes the last two steps of purine biosynthesis, generating inosine monophosphate from the aminoimidazole carboxamide ribonucleotide. Its pathways are related to AMP-activated protein kinase signaling (AMPK) and to antifolate resistance, as it is able to convert (5-amino-1-(5-phospho-β-D-ribosyl) imidazole-4-carboxamide)) into 6-mercaptopurine ribonucleotide, an inhibitor of purine biosynthesis used in the treatment of leukemias [[20](#B20-genes-13-00330)].

The de novo purine synthesis pathway is important for the disordered tumor growth process, as it is part of an anabolic pathway of cell multiplication; it is widely used in metabolic reprogramming for cell survival [[21](#B21-genes-13-00330),[22](#B22-genes-13-00330)]. However, the specific role of the *ATIC* gene in modulating cancer progression remains unknown [[23](#B23-genes-13-00330),[24](#B24-genes-13-00330),[25](#B25-genes-13-00330)].

This gene has been shown to be overexpressed in hepatocellular carcinoma (HCC) and related to a worse prognosis in this neoplasm. The authors found that ATIC activates mTOR-S6 kinase 1 signaling and consequently stimulates the proliferation and migration of oncotic cells [[25](#B25-genes-13-00330)]. Furthermore, the gene has been associated with autophagy and an increased risk of developing HCC [[26](#B26-genes-13-00330)], lung cancer [[27](#B27-genes-13-00330)], and multiple myeloma [[28](#B28-genes-13-00330)].

In addition, the *ATIC* gene is associated with the risk of lymphoma progression in cases of ATIC protein fusion with the protein of oncogene *ALK* (anaplastic lymphoma kinase) [[29](#B29-genes-13-00330),[30](#B30-genes-13-00330)]. This fusion even influences the treatment because when ALK phosphorylates ATIC in Y104, there is an increase in enzymatic activity. ALK-mediated phosphorylation of ATIC can rescue cancer cells from cell death induced by antifolate agents [[31](#B31-genes-13-00330)]. These results together suggest that *ATIC* may play an important role in carcinogenesis and cancer cell survival even under treatment [[32](#B32-genes-13-00330)].

This type of relationship of *ATIC* gene variants has also been demonstrated in other investigations with cancer therapeutic resistance such as in the treatment of breast cancer with tamoxifen [[33](#B33-genes-13-00330)], the use of pemetrexed for non-small cell lung cancer [[34](#B34-genes-13-00330)], as well as the use of methotrexate for rheumatoid arthritis [[35](#B35-genes-13-00330)], pediatric osteosarcoma [[36](#B36-genes-13-00330)], and acute lymphoblastic leukemia [[37](#B37-genes-13-00330)]. The investigated rs2372536 polymorphism is a missense mutation, responsible for the substitution of a threonine for a serine at position 116 of exon 5 of the expressed protein (c.347C > G; Thr116Ser), and it is one of the main biomarkers investigated in the response to methotrexate in rheumatoid arthritis [[38](#B38-genes-13-00330),[39](#B39-genes-13-00330),[40](#B40-genes-13-00330)].

### 4.2. ARID5B

The *ARID5B* gene is part of the AT-rich interaction domain (ARID) family of DNA-binding proteins, which are described as chromatin remodeling factors and also responsible for regulating the transcription of target genes [[41](#B41-genes-13-00330),[42](#B42-genes-13-00330)]. ARID5B forms a complex with the PHF2 protein, which has H3K9me2 histone demethylase activity. H3K9me2 is one of the main markers of silenced chromatin and, thus, there is an epigenetic regulation of gene expression [[43](#B43-genes-13-00330)].

In addition, recent findings demonstrate that ARID5B is involved in cell proliferation and acts in the growth and differentiation of progenitor B-lymphocytes. It is a co-activator that binds to the 5′-AATA(CT)-3′ sequence [[43](#B43-genes-13-00330)]. The rs10821936 is a variant located in intron 3, and variants present in this intron are the most associated with susceptibility to ALL [[44](#B44-genes-13-00330),[45](#B45-genes-13-00330),[46](#B46-genes-13-00330),[47](#B47-genes-13-00330)]. Although its role in leukemogenesis is not fully understood, SNPs in intron 3 of *ARID5B* may alter the transcription network involving normal hematopoiesis, thus, altering cell growth and differentiation [[48](#B48-genes-13-00330)].

Variants in the *ARID5B* gene have also been related to ALL regarding relapse and treatment response [[49](#B49-genes-13-00330)]. It has also been associated with risk of developing colorectal cancer [[50](#B50-genes-13-00330)], participation in breast cancer metabolism [[51](#B51-genes-13-00330)], and with a protein with unregulated function in prostate cancer [[52](#B52-genes-13-00330)]. These findings suggest that this gene plays an important role in the carcinogenic process.

Therefore, we suggest that variants of the *ATIC* and *ARID5B* genes may interfere with imatinib response in patients with CML, once, even though these genes do not interact directly with the drug, they act in the cellular environment supporting the survival of cancer cells, thus, impairing the effect of the treatment.

## 5. Conclusions

We conclude that the never-before-screened genetic variants of the genes *ATIC* (rs2372536) and *ARID5B* (rs10821936) play a role in therapeutic failure with imatinib, the gold standard treatment for CML.

## Acknowledgments

We acknowledge the Universidade Federal do Pará (UFPA); Núcleo de Pesquisas em Oncologia (NPO/UFPA); Hospital Ophir Loyola (HOL); Laboratório de Genética Humana e Médica (LGHM/UFPA); Programa de Pós Graduação em Genética e Biologia Molecular (PPGBM/UFPA); and Programa de Pós Graduação em Oncologia e Ciências Médicas (PPGOCM/UFPA).

## Supplementary Materials

The following are available online at [https://www.mdpi.com/article/10.3390/genes13020330/s1](https://www.mdpi.com/article/10.3390/genes13020330/s1), Table S1: Polymorphisms chosen after applying the criteria of selection; Table S2: Allelic and genotypic distribution and quality control of the polymorphisms; Table S3: Odds ratio and genotype distributions of the polymorphisms not statistically significant between the patients with response and without response.

## Author Contributions

Writing—original draft preparation and formal analysis, K.B.C.C.P. and T.C.d.B.A.; Methodology, D.C.d.C., N.M., L.W.M.S.V., A.R.S.d.F. and M.C.d.C.B.; Writing—review and editing, A.d.N.C.P.; Funding acquisition, P.P.d.A., R.M.R.B. and S.E.B.d.S.; Supervision and project administration, M.R.F. and N.P.C.d.S. All authors have read and agreed to the published version of the manuscript.

## Funding

This research received external funding from the Conselho Nacional de Desenvolvimento Científico e Tecnológico (CNPQ): Chamada Universal-MCTI/CNPq No. 14/2013; Universidade Federal do Pará: Edital 06/2021–PROGRAMA DE APOIO À PUBLICAÇÃO QUALIFICADA-PAPQ/ Pró-Reitoria de Pesquisa e Pós-Graduação (PROPESP).

## Institutional Review Board Statement

The study was conducted according to the guidelines of the Declaration of Helsinki and approved by the Research Ethics Committees of the Núcleo de Pesquisas em Oncologia, under protocol number 3.354.571/2019, and by the Hospital Ophir Loyola, under protocol number 1.575.920/2016.

## Informed Consent Statement

All participants agreed to participate in the research and signed an informed consent.

## Data Availability Statement

The data presently in this study are available on request from the corresponding author. The data are not publicly available due to the privacy topics contained in informed consent.

## Conflicts of Interest

The authors declare that they have no competing interests.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data presently in this study are available on request from the corresponding author. The data are not publicly available due to the privacy topics contained in informed consent.

### Supplementary Materials

### Data Availability Statement

The data presently in this study are available on request from the corresponding author. The data are not publicly available due to the privacy topics contained in informed consent.

## References

1. Perrotti D., Jamieson C., Goldman J., Skorski T. Chronic Myeloid Leukemia: Mechanisms of Blastic Transformation. J. Clin. Investig. 2010;120:2254–2264. doi: 10.1172/JCI41246.  [DOI](https://doi.org/10.1172/JCI41246) | [PMC free article](/articles/PMC2898591/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20592475/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Investig.&title=Chronic%20Myeloid%20Leukemia:%20Mechanisms%20of%20Blastic%20Transformation&author=D.%20Perrotti&author=C.%20Jamieson&author=J.%20Goldman&author=T.%20Skorski&volume=120&publication_year=2010&pages=2254-2264&pmid=20592475&doi=10.1172/JCI41246&)

2. Apperley J.F. CML and Tyrosine Kinase Inhibition: The Hope Becomes Reality. Lancet Haematol. 2015;2:e176–e177. doi: 10.1016/S2352-3026(15)00072-1.  [DOI](https://doi.org/10.1016/S2352-3026(15)00072-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26688089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Haematol.&title=CML%20and%20Tyrosine%20Kinase%20Inhibition:%20The%20Hope%20Becomes%20Reality&author=J.F.%20Apperley&volume=2&publication_year=2015&pages=e176-e177&pmid=26688089&doi=10.1016/S2352-3026(15)00072-1&)

3. Kang Z.J., Liu Y.F., Xu L.Z., Long Z.J., Huang D., Yang Y., Liu B., Feng J.X., Pan Y.J., Yan J.S., et al. The Philadelphia Chromosome in Leukemogenesis. Chin. J. Cancer. 2016;35:48. doi: 10.1186/s40880-016-0108-0.  [DOI](https://doi.org/10.1186/s40880-016-0108-0) | [PMC free article](/articles/PMC4896164/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27233483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chin.%20J.%20Cancer&title=The%20Philadelphia%20Chromosome%20in%20Leukemogenesis&author=Z.J.%20Kang&author=Y.F.%20Liu&author=L.Z.%20Xu&author=Z.J.%20Long&author=D.%20Huang&volume=35&publication_year=2016&pages=48&pmid=27233483&doi=10.1186/s40880-016-0108-0&)

4. Ankathil R., Azlan H., Dzarr A.A., Baba A.A. Pharmacogenetics and the Treatment of Chronic Myeloid Leukemia: How Relevant Clinically? An Update. Pharmacogenomics. 2018;19:475–493. doi: 10.2217/pgs-2017-0193.  [DOI](https://doi.org/10.2217/pgs-2017-0193) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29569526/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Pharmacogenetics%20and%20the%20Treatment%20of%20Chronic%20Myeloid%20Leukemia:%20How%20Relevant%20Clinically?%20An%20Update&author=R.%20Ankathil&author=H.%20Azlan&author=A.A.%20Dzarr&author=A.A.%20Baba&volume=19&publication_year=2018&pages=475-493&pmid=29569526&doi=10.2217/pgs-2017-0193&)

5. Fausel C. Targeted Chronic Myeloid Leukemia Therapy: Seeking a Cure; Proceedings of the American Journal of Health-System Pharmacy; Oxford, UK. 15 December 2007;  [DOI](https://doi.org/10.2146/ajhp070482) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18056932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20American%20Journal%20of%20Health-System%20Pharmacy&title=Targeted%20Chronic%20Myeloid%20Leukemia%20Therapy:%20Seeking%20a%20Cure&author=C.%20Fausel&volume=Volume%2064&pmid=18056932&doi=10.2146/ajhp070482&)

6. Bixby D., Talpaz M. Mechanisms of Resistance to Tyrosine Kinase Inhibitors in Chronic Myeloid Leukemia and Recent Therapeutic Strategies to Overcome Resistance. Hematol. Am. Soc. Hematol. Educ. Program. 2009:461–476. doi: 10.1182/asheducation-2009.1.461.  [DOI](https://doi.org/10.1182/asheducation-2009.1.461) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20008232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hematol.%20Am.%20Soc.%20Hematol.%20Educ.%20Program&title=Mechanisms%20of%20Resistance%20to%20Tyrosine%20Kinase%20Inhibitors%20in%20Chronic%20Myeloid%20Leukemia%20and%20Recent%20Therapeutic%20Strategies%20to%20Overcome%20Resistance&author=D.%20Bixby&author=M.%20Talpaz&publication_year=2009&pages=461-476&pmid=20008232&doi=10.1182/asheducation-2009.1.461&)

7. Milojkovic D., Apperley J.F. Mechanisms of Resistance to Imatinib and Second-Generation Tyrosine Inhibitors in Chronic Myeloid Leukemia. Clin. Cancer Res. 2009;15:7519–7527. doi: 10.1158/1078-0432.CCR-09-1068.  [DOI](https://doi.org/10.1158/1078-0432.CCR-09-1068) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20008852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Mechanisms%20of%20Resistance%20to%20Imatinib%20and%20Second-Generation%20Tyrosine%20Inhibitors%20in%20Chronic%20Myeloid%20Leukemia&author=D.%20Milojkovic&author=J.F.%20Apperley&volume=15&publication_year=2009&pages=7519-7527&pmid=20008852&doi=10.1158/1078-0432.CCR-09-1068&)

8. Shinohara Y., Takahashi N., Nishiwaki K., Hino M., Kashimura M., Wakita H., Hatano Y., Hirasawa A., Nakagawa Y., Itoh K., et al. A Multicenter Clinical Study Evaluating the Confirmed Complete Molecular Response Rate in Imatinib-Treated Patients with Chronic Phase Chronic Myeloid Leukemia by Using the International Scale of Real-Time Quantitative Polymerase Chain Reaction. Haematologica. 2013;98:1407–1413. doi: 10.3324/haematol.2013.085167.  [DOI](https://doi.org/10.3324/haematol.2013.085167) | [PMC free article](/articles/PMC3762097/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23716542/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=A%20Multicenter%20Clinical%20Study%20Evaluating%20the%20Confirmed%20Complete%20Molecular%20Response%20Rate%20in%20Imatinib-Treated%20Patients%20with%20Chronic%20Phase%20Chronic%20Myeloid%20Leukemia%20by%20Using%20the%20International%20Scale%20of%20Real-Time%20Quantitative%20Polymerase%20Chain%20Reaction&author=Y.%20Shinohara&author=N.%20Takahashi&author=K.%20Nishiwaki&author=M.%20Hino&author=M.%20Kashimura&volume=98&publication_year=2013&pages=1407-1413&pmid=23716542&doi=10.3324/haematol.2013.085167&)

9. Polillo M., Galimberti S., Baratè C., Petrini M., Danesi R., Di Paolo A. Pharmacogenetics of BCR/ABL Inhibitors in Chronic Myeloid Leukemia. Int. J. Mol. Sci. 2015;16:22811–22829. doi: 10.3390/ijms160922811.  [DOI](https://doi.org/10.3390/ijms160922811) | [PMC free article](/articles/PMC4613337/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26402671/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Pharmacogenetics%20of%20BCR/ABL%20Inhibitors%20in%20Chronic%20Myeloid%20Leukemia&author=M.%20Polillo&author=S.%20Galimberti&author=C.%20Barat%C3%A8&author=M.%20Petrini&author=R.%20Danesi&volume=16&publication_year=2015&pages=22811-22829&pmid=26402671&doi=10.3390/ijms160922811&)

10. Guo C., Xie X., Li J., Huang L., Chen S., Li X., Yi X., Wu Q., Yang G., Zhou H., et al. Pharmacogenomics Guidelines: Current Status and Future Development. Clin. Exp. Pharmacol. Physiol. 2019;46:689–693. doi: 10.1111/1440-1681.13097.  [DOI](https://doi.org/10.1111/1440-1681.13097) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31009088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Exp.%20Pharmacol.%20Physiol.&title=Pharmacogenomics%20Guidelines:%20Current%20Status%20and%20Future%20Development&author=C.%20Guo&author=X.%20Xie&author=J.%20Li&author=L.%20Huang&author=S.%20Chen&volume=46&publication_year=2019&pages=689-693&pmid=31009088&doi=10.1111/1440-1681.13097&)

11. Weinshilboum R.M., Wang L. Pharmacogenetics and Pharmacogenomics: Development, Science, and Translation. Annu. Rev. Genom. Hum. Genet. 2006;7:223–245. doi: 10.1146/annurev.genom.6.080604.162315.  [DOI](https://doi.org/10.1146/annurev.genom.6.080604.162315) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16948615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Genom.%20Hum.%20Genet.&title=Pharmacogenetics%20and%20Pharmacogenomics:%20Development,%20Science,%20and%20Translation&author=R.M.%20Weinshilboum&author=L.%20Wang&volume=7&publication_year=2006&pages=223-245&pmid=16948615&doi=10.1146/annurev.genom.6.080604.162315&)

12. Rodrigues J.C.G., Fernandes M.R., Guerreiro J.F., Silva A.L.D.C.D., Ribeiro-dos-Santos Â., Santos S., Santos N.P.C. Dos Polymorphisms of ADME-Related Genes and Their Implications for Drug Safety and Efficacy in Amazonian Amerindians. Sci. Rep. 2019;9:7201. doi: 10.1038/s41598-019-43610-y.  [DOI](https://doi.org/10.1038/s41598-019-43610-y) | [PMC free article](/articles/PMC6510895/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31076604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Dos%20Polymorphisms%20of%20ADME-Related%20Genes%20and%20Their%20Implications%20for%20Drug%20Safety%20and%20Efficacy%20in%20Amazonian%20Amerindians&author=J.C.G.%20Rodrigues&author=M.R.%20Fernandes&author=J.F.%20Guerreiro&author=A.L.D.C.D.%20Silva&author=%C3%82.%20Ribeiro-dos-Santos&volume=9&publication_year=2019&pages=7201&pmid=31076604&doi=10.1038/s41598-019-43610-y&)

13. De Carvalho D.C., Wanderley A.V., dos Santos A.M.R., Moreira F.C., de Sá R.B.A., Fernandes M.R., Modesto A.A.C., de Souza T.P., Cohen-Paes A., Leitão L.P.C., et al. Characterization of Pharmacogenetic Markers Related to Acute Lymphoblastic Leukemia Toxicity in Amazonian Native Americans Population. Sci. Rep. 2020;10:10292. doi: 10.1038/s41598-020-67312-y.  [DOI](https://doi.org/10.1038/s41598-020-67312-y) | [PMC free article](/articles/PMC7314857/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32581388/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Characterization%20of%20Pharmacogenetic%20Markers%20Related%20to%20Acute%20Lymphoblastic%20Leukemia%20Toxicity%20in%20Amazonian%20Native%20Americans%20Population&author=D.C.%20De%20Carvalho&author=A.V.%20Wanderley&author=A.M.R.%20dos%20Santos&author=F.C.%20Moreira&author=R.B.A.%20de%20S%C3%A1&volume=10&publication_year=2020&pages=10292&pmid=32581388&doi=10.1038/s41598-020-67312-y&)

14. Santos N.P.C., Ribeiro-Rodrigues E.M., Ribeiro-dos-Santos Â.K., Pereira R., Gusmão L., Amorim A., Guerreiro J.F., Zago M.A., Matte C., Hutz M.H., et al. Assessing Individual Interethnic Admixture and Population Substructure Using a 48-Insertion-Deletion (INSEL) Ancestry-Informative Marker (AIM) Panel. Hum. Mutat. 2010;31:184–190. doi: 10.1002/humu.21159.  [DOI](https://doi.org/10.1002/humu.21159) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19953531/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mutat.&title=Assessing%20Individual%20Interethnic%20Admixture%20and%20Population%20Substructure%20Using%20a%2048-Insertion-Deletion%20(INSEL)%20Ancestry-Informative%20Marker%20(AIM)%20Panel&author=N.P.C.%20Santos&author=E.M.%20Ribeiro-Rodrigues&author=%C3%82.K.%20Ribeiro-dos-Santos&author=R.%20Pereira&author=L.%20Gusm%C3%A3o&volume=31&publication_year=2010&pages=184-190&pmid=19953531&doi=10.1002/humu.21159&)

15. Suarez-Kurtz G. Population Impact of Pharmacogenetic Tests in Admixed Populations across the Americas. Pharmacogenom. J. 2021;21:216–221. doi: 10.1038/s41397-020-00200-w.  [DOI](https://doi.org/10.1038/s41397-020-00200-w) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33110249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom.%20J.&title=Population%20Impact%20of%20Pharmacogenetic%20Tests%20in%20Admixed%20Populations%20across%20the%20Americas&author=G.%20Suarez-Kurtz&volume=21&publication_year=2021&pages=216-221&pmid=33110249&doi=10.1038/s41397-020-00200-w&)

16. Shah N.P. NCCN Guidelines Updates: Discontinuing TKI Therapy in the Treatment of Chronic Myeloid Leukemia. J. Natl. Compr. Cancer Netw. 2019;17:611–613. doi: 10.6004/jnccn.2019.5013.  [DOI](https://doi.org/10.6004/jnccn.2019.5013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31117031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Natl.%20Compr.%20Cancer%20Netw.&title=NCCN%20Guidelines%20Updates:%20Discontinuing%20TKI%20Therapy%20in%20the%20Treatment%20of%20Chronic%20Myeloid%20Leukemia&author=N.P.%20Shah&volume=17&publication_year=2019&pages=611-613&pmid=31117031&doi=10.6004/jnccn.2019.5013&)

17. Ramos B.R., Mendes N.D., Tanikawa A.A., Amador M.A., dos Santos N.P., dos Santos S.E., Castelli E.C., Witkin S.S., da Silva M.G. Ancestry informative markers and selected single nucleotide polymorphisms in immunoregulatory genes on preterm labor and preterm premature rupture of membranes: A case control study. BMC Pregnancy Childbirth. 2016;16:30. doi: 10.1186/s12884-016-0823-1.  [DOI](https://doi.org/10.1186/s12884-016-0823-1) | [PMC free article](/articles/PMC4743423/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26846412/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Pregnancy%20Childbirth&title=Ancestry%20informative%20markers%20and%20selected%20single%20nucleotide%20polymorphisms%20in%20immunoregulatory%20genes%20on%20preterm%20labor%20and%20preterm%20premature%20rupture%20of%20membranes:%20A%20case%20control%20study&author=B.R.%20Ramos&author=N.D.%20Mendes&author=A.A.%20Tanikawa&author=M.A.%20Amador&author=N.P.%20dos%20Santos&volume=16&publication_year=2016&pages=30&pmid=26846412&doi=10.1186/s12884-016-0823-1&)

18. NOVARTIS Gleevec: Imatinib Mesylate Tablet. [(accessed on 25 November 2021)]; Available online: https://www.accessdata.fda.gov/scripts/cder/daf/index.cfm?event=overview.process&varApplNo=021588.  [https://www.accessdata.fda.gov/scripts/cder/daf/index.cfm?event=overview.process&varApplNo=021588](https://www.accessdata.fda.gov/scripts/cder/daf/index.cfm?event=overview.process&varApplNo=021588)

19. Yang K., Fu L.W. Mechanisms of Resistance to BCR-ABL TKIs and the Therapeutic Strategies: A Review. Crit. Rev. Oncol. Hematol. 2015;93:277–292. doi: 10.1016/j.critrevonc.2014.11.001.  [DOI](https://doi.org/10.1016/j.critrevonc.2014.11.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25500000/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit.%20Rev.%20Oncol.%20Hematol.&title=Mechanisms%20of%20Resistance%20to%20BCR-ABL%20TKIs%20and%20the%20Therapeutic%20Strategies:%20A%20Review&author=K.%20Yang&author=L.W.%20Fu&volume=93&publication_year=2015&pages=277-292&pmid=25500000&doi=10.1016/j.critrevonc.2014.11.001&)

20. Wall M., Shim J.H., Benkovic S.J. Human AICAR Transformylase: Role of the 4-Carboxamide of AICAR in Binding and Catalysis. Biochemistry. 2000;39:11303–11311. doi: 10.1021/bi0007268.  [DOI](https://doi.org/10.1021/bi0007268) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10985775/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Human%20AICAR%20Transformylase:%20Role%20of%20the%204-Carboxamide%20of%20AICAR%20in%20Binding%20and%20Catalysis&author=M.%20Wall&author=J.H.%20Shim&author=S.J.%20Benkovic&volume=39&publication_year=2000&pages=11303-11311&pmid=10985775&doi=10.1021/bi0007268&)

21. Martinez-Outschoorn U.E., Peiris-Pagés M., Pestell R.G., Sotgia F., Lisanti M.P. Erratum: Cancer Metabolism: A Therapeutic Perspective. Nat. Rev. Clin. Oncol. 2017;14:113. doi: 10.1038/nrclinonc.2017.1.  [DOI](https://doi.org/10.1038/nrclinonc.2017.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28094266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Clin.%20Oncol.&title=Erratum:%20Cancer%20Metabolism:%20A%20Therapeutic%20Perspective&author=U.E.%20Martinez-Outschoorn&author=M.%20Peiris-Pag%C3%A9s&author=R.G.%20Pestell&author=F.%20Sotgia&author=M.P.%20Lisanti&volume=14&publication_year=2017&pages=113&pmid=28094266&doi=10.1038/nrclinonc.2017.1&)

22. Yamaoka T., Kondo M., Honda S., Iwahana H., Moritani M., Ii S., Yoshimoto K., Itakura M. Amidophosphoribosyltransferase Limits the Rate of Cell Growth-Linked de Novo Purine Biosynthesis in the Presence of Constant Capacity of Salvage Purine Biosynthesis. J. Biol. Chem. 1997;272:17719–17725. doi: 10.1074/jbc.272.28.17719.  [DOI](https://doi.org/10.1074/jbc.272.28.17719) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9211923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Amidophosphoribosyltransferase%20Limits%20the%20Rate%20of%20Cell%20Growth-Linked%20de%20Novo%20Purine%20Biosynthesis%20in%20the%20Presence%20of%20Constant%20Capacity%20of%20Salvage%20Purine%20Biosynthesis&author=T.%20Yamaoka&author=M.%20Kondo&author=S.%20Honda&author=H.%20Iwahana&author=M.%20Moritani&volume=272&publication_year=1997&pages=17719-17725&pmid=9211923&doi=10.1074/jbc.272.28.17719&)

23. Chan C.Y., Zhao H., Pugh R.J., Pedley A.M., French J., Jones S.A., Zhuang X., Jinnah H., Huan T.J., Benkovic S.J. Purinosome Formation as a Function of the Cell Cycle. Proc. Natl. Acad. Sci. USA. 2015;112:1368–1373. doi: 10.1073/pnas.1423009112.  [DOI](https://doi.org/10.1073/pnas.1423009112) | [PMC free article](/articles/PMC4321313/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25605889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Purinosome%20Formation%20as%20a%20Function%20of%20the%20Cell%20Cycle&author=C.Y.%20Chan&author=H.%20Zhao&author=R.J.%20Pugh&author=A.M.%20Pedley&author=J.%20French&volume=112&publication_year=2015&pages=1368-1373&pmid=25605889&doi=10.1073/pnas.1423009112&)

24. Liu X., Chhipa R.R., Pooya S., Wortman M., Yachyshin S., Chow L.M.L., Kumar A., Zhou X., Sun Y., Quinn B., et al. Discrete Mechanisms of MTOR and Cell Cycle Regulation by AMPK Agonists Independent of AMPK. Proc. Natl. Acad. Sci. USA. 2014;111:E435–E444. doi: 10.1073/pnas.1311121111.  [DOI](https://doi.org/10.1073/pnas.1311121111) | [PMC free article](/articles/PMC3910576/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24474794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Discrete%20Mechanisms%20of%20MTOR%20and%20Cell%20Cycle%20Regulation%20by%20AMPK%20Agonists%20Independent%20of%20AMPK&author=X.%20Liu&author=R.R.%20Chhipa&author=S.%20Pooya&author=M.%20Wortman&author=S.%20Yachyshin&volume=111&publication_year=2014&pages=E435-E444&pmid=24474794&doi=10.1073/pnas.1311121111&)

25. Li M., Jin C., Xu M., Zhou L., Li D., Yin Y. Bifunctional Enzyme ATIC Promotes Propagation of Hepatocellular Carcinoma by Regulating AMPK-MTOR-S6 K1 Signaling. Cell Commun. Signal. 2017;15:52. doi: 10.1186/s12964-017-0208-8.  [DOI](https://doi.org/10.1186/s12964-017-0208-8) | [PMC free article](/articles/PMC5732395/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29246230/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Commun.%20Signal.&title=Bifunctional%20Enzyme%20ATIC%20Promotes%20Propagation%20of%20Hepatocellular%20Carcinoma%20by%20Regulating%20AMPK-MTOR-S6%20K1%20Signaling&author=M.%20Li&author=C.%20Jin&author=M.%20Xu&author=L.%20Zhou&author=D.%20Li&volume=15&publication_year=2017&pages=52&pmid=29246230&doi=10.1186/s12964-017-0208-8&)

26. Huo X., Qi J., Huang K., Bu S., Yao W., Chen Y., Nie J. Identification of an Autophagy-Related Gene Signature That Can Improve Prognosis of Hepatocellular Carcinoma Patients. BMC Cancer. 2020;20:771. doi: 10.1186/s12885-020-07277-3.  [DOI](https://doi.org/10.1186/s12885-020-07277-3) | [PMC free article](/articles/PMC7433127/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32807131/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Identification%20of%20an%20Autophagy-Related%20Gene%20Signature%20That%20Can%20Improve%20Prognosis%20of%20Hepatocellular%20Carcinoma%20Patients&author=X.%20Huo&author=J.%20Qi&author=K.%20Huang&author=S.%20Bu&author=W.%20Yao&volume=20&publication_year=2020&pages=771&pmid=32807131&doi=10.1186/s12885-020-07277-3&)

27. Zhu J., Wang M., Hu D. Development of an Autophagy-Related Gene Prognostic Signature in Lung Adenocarcinoma and Lung Squamous Cell Carcinoma. PeerJ. 2020;2020:e8288. doi: 10.7717/peerj.8288.  [DOI](https://doi.org/10.7717/peerj.8288) | [PMC free article](/articles/PMC6953332/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31938577/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PeerJ&title=Development%20of%20an%20Autophagy-Related%20Gene%20Prognostic%20Signature%20in%20Lung%20Adenocarcinoma%20and%20Lung%20Squamous%20Cell%20Carcinoma&author=J.%20Zhu&author=M.%20Wang&author=D.%20Hu&volume=2020&publication_year=2020&pages=e8288&pmid=31938577&doi=10.7717/peerj.8288&)

28. Li R., Chen G., Dang Y., He R., Liu A., Ma J., Wang C. Upregulation of ATIC in Multiple Myeloma Tissues Based on Tissue Microarray and Gene Microarrays. Int. J. Lab. Hematol. 2021;43:409–417. doi: 10.1111/ijlh.13397.  [DOI](https://doi.org/10.1111/ijlh.13397) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33226193/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Lab.%20Hematol.&title=Upregulation%20of%20ATIC%20in%20Multiple%20Myeloma%20Tissues%20Based%20on%20Tissue%20Microarray%20and%20Gene%20Microarrays&author=R.%20Li&author=G.%20Chen&author=Y.%20Dang&author=R.%20He&author=A.%20Liu&volume=43&publication_year=2021&pages=409-417&pmid=33226193&doi=10.1111/ijlh.13397&)

29. Trinei M., Lanfrancone L., Campo E., Pulford K., Mason D.Y., Pelicci P.G., Falini B. A New Variant Anaplastic Lymphoma Kinase (ALK)-Fusion Protein (ATIC-ALK) in a Case of ALK-Positive Anaplastic Large Cell Lymphoma. Cancer Res. 2000;60:793–798.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10706082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=A%20New%20Variant%20Anaplastic%20Lymphoma%20Kinase%20(ALK)-Fusion%20Protein%20(ATIC-ALK)%20in%20a%20Case%20of%20ALK-Positive%20Anaplastic%20Large%20Cell%20Lymphoma&author=M.%20Trinei&author=L.%20Lanfrancone&author=E.%20Campo&author=K.%20Pulford&author=D.Y.%20Mason&volume=60&publication_year=2000&pages=793-798&pmid=10706082&)

30. Van Der Krogt J.A., Bempt M.V., Ferreiro J.F., Mentens N., Jacobs K., Pluys U., Doms K., Geerdens E., Uyttebroeck A., Pierre P., et al. Anaplastic Lymphoma Kinase-Positive Anaplastic Large Cell Lymphoma with the Variant RNF213-, ATIC- and TPM3-ALK Fusions Is Characterized by Copy Number Gain of the Rearranged ALK Gene. Haematologica. 2017;102:1605–1616. doi: 10.3324/haematol.2016.146571.  [DOI](https://doi.org/10.3324/haematol.2016.146571) | [PMC free article](/articles/PMC5685221/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28659337/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=Anaplastic%20Lymphoma%20Kinase-Positive%20Anaplastic%20Large%20Cell%20Lymphoma%20with%20the%20Variant%20RNF213-,%20ATIC-%20and%20TPM3-ALK%20Fusions%20Is%20Characterized%20by%20Copy%20Number%20Gain%20of%20the%20Rearranged%20ALK%20Gene&author=J.A.%20Van%20Der%20Krogt&author=M.V.%20Bempt&author=J.F.%20Ferreiro&author=N.%20Mentens&author=K.%20Jacobs&volume=102&publication_year=2017&pages=1605-1616&pmid=28659337&doi=10.3324/haematol.2016.146571&)

31. Boccalatte F.E., Voena C., Riganti C., Bosia A., D’Amico L., Riera L., Cheng M., Ruggeri B., Jensen O.N., Goss V.L., et al. The Enzymatic Activity of 5-Aminoimidazole-4-Carboxamide Ribonucleotide Formyltransferase/IMP Cyclohydrolase Is Enhanced by NPM-ALK: New Insights in ALK-Mediated Pathogenesis and the Treatment of ALCL. Blood. 2009;113:2776–2790. doi: 10.1182/blood-2008-06-161018.  [DOI](https://doi.org/10.1182/blood-2008-06-161018) | [PMC free article](/articles/PMC2661863/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18845790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=The%20Enzymatic%20Activity%20of%205-Aminoimidazole-4-Carboxamide%20Ribonucleotide%20Formyltransferase/IMP%20Cyclohydrolase%20Is%20Enhanced%20by%20NPM-ALK:%20New%20Insights%20in%20ALK-Mediated%20Pathogenesis%20and%20the%20Treatment%20of%20ALCL&author=F.E.%20Boccalatte&author=C.%20Voena&author=C.%20Riganti&author=A.%20Bosia&author=L.%20D%E2%80%99Amico&volume=113&publication_year=2009&pages=2776-2790&pmid=18845790&doi=10.1182/blood-2008-06-161018&)

32. Su W.J., Lu P.Z., Wu Y., Kalpana K., Yang C.K., Lu G.D. Identification of Key Genes in Purine Metabolism as Prognostic Biomarker for Hepatocellular Carcinoma. Front. Oncol. 2021;10:3057. doi: 10.3389/fonc.2020.583053.  [DOI](https://doi.org/10.3389/fonc.2020.583053) | [PMC free article](/articles/PMC7841304/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33520699/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Oncol.&title=Identification%20of%20Key%20Genes%20in%20Purine%20Metabolism%20as%20Prognostic%20Biomarker%20for%20Hepatocellular%20Carcinoma&author=W.J.%20Su&author=P.Z.%20Lu&author=Y.%20Wu&author=K.%20Kalpana&author=C.K.%20Yang&volume=10&publication_year=2021&pages=3057&pmid=33520699&doi=10.3389/fonc.2020.583053&)

33. Zhang K., Jiang K., Hong R., Xu F., Xia W., Qin G., Lee K., Zheng Q., Lu Q., Zhai Q., et al. Identification and Characterization of Critical Genes Associated with Tamoxifen Resistance in Breast Cancer. PeerJ. 2020;8:e10468. doi: 10.7717/peerj.10468.  [DOI](https://doi.org/10.7717/peerj.10468) | [PMC free article](/articles/PMC7720728/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33335811/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PeerJ&title=Identification%20and%20Characterization%20of%20Critical%20Genes%20Associated%20with%20Tamoxifen%20Resistance%20in%20Breast%20Cancer&author=K.%20Zhang&author=K.%20Jiang&author=R.%20Hong&author=F.%20Xu&author=W.%20Xia&volume=8&publication_year=2020&pages=e10468&pmid=33335811&doi=10.7717/peerj.10468&)

34. Visser S., Koolen S., Van Donk N., Van Walree N., Van Der Leest C., Cornelissen R., Van Schaik R., Mathijssen R., Aerts J., Stricker B.H. Genetic Polymorphism in ATIC Is Associated with Effectiveness and Toxicity of Pemetrexed in Non-Small-Cell Lung Cancer. Thorax. 2021;76:1150–1153. doi: 10.1136/thoraxjnl-2020-216504.  [DOI](https://doi.org/10.1136/thoraxjnl-2020-216504) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33859051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thorax&title=Genetic%20Polymorphism%20in%20ATIC%20Is%20Associated%20with%20Effectiveness%20and%20Toxicity%20of%20Pemetrexed%20in%20Non-Small-Cell%20Lung%20Cancer&author=S.%20Visser&author=S.%20Koolen&author=N.%20Van%20Donk&author=N.%20Van%20Walree&author=C.%20Van%20Der%20Leest&volume=76&publication_year=2021&pages=1150-1153&pmid=33859051&doi=10.1136/thoraxjnl-2020-216504&)

35. Kurzawski M., Malinowski D., Szarmach N., Nowak A., Goryniak A., Pawlik A., Droździk M. ATIC Missense Variant Affects Response to Methotrexate Treatment in Rheumatoid Arthritis Patients. Pharmacogenomics. 2016;17:1971–1978. doi: 10.2217/pgs-2016-0125.  [DOI](https://doi.org/10.2217/pgs-2016-0125) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27885916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=ATIC%20Missense%20Variant%20Affects%20Response%20to%20Methotrexate%20Treatment%20in%20Rheumatoid%20Arthritis%20Patients&author=M.%20Kurzawski&author=D.%20Malinowski&author=N.%20Szarmach&author=A.%20Nowak&author=A.%20Goryniak&volume=17&publication_year=2016&pages=1971-1978&pmid=27885916&doi=10.2217/pgs-2016-0125&)

36. Park J.A., Shin H.Y. ATIC Gene Polymorphism and Histologic Response to Chemotherapy in Pediatric Osteosarcoma. J. Pediatr. Hematol. Oncol. 2017;39:e270–e274. doi: 10.1097/MPH.0000000000000802.  [DOI](https://doi.org/10.1097/MPH.0000000000000802) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28267080/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pediatr.%20Hematol.%20Oncol.&title=ATIC%20Gene%20Polymorphism%20and%20Histologic%20Response%20to%20Chemotherapy%20in%20Pediatric%20Osteosarcoma&author=J.A.%20Park&author=H.Y.%20Shin&volume=39&publication_year=2017&pages=e270-e274&pmid=28267080&doi=10.1097/MPH.0000000000000802&)

37. Adam de Beaumais T., Jacqz-Aigrain E. Intracellular Disposition of Methotrexate in Acute Lymphoblastic Leukemia in Children. Curr. Drug Metab. 2012;13:822–834. doi: 10.2174/138920012800840400.  [DOI](https://doi.org/10.2174/138920012800840400) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22571483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Drug%20Metab.&title=Intracellular%20Disposition%20of%20Methotrexate%20in%20Acute%20Lymphoblastic%20Leukemia%20in%20Children&author=T.%20Adam%20de%20Beaumais&author=E.%20Jacqz-Aigrain&volume=13&publication_year=2012&pages=822-834&pmid=22571483&doi=10.2174/138920012800840400&)

38. Hinks A., Moncrieffe H., Martin P., Ursu S., Lal S., Kassoumeri L., Weiler T., Glass D.N., Thompson S.D., Wedderburn L.R., et al. Association of the 5-Aminoimidazole-4-Carboxamide Ribonucleotide Transformylase Gene with Response to Methotrexate in Juvenile Idiopathic Arthritis. Ann. Rheum. Dis. 2011;70:1395–1400. doi: 10.1136/ard.2010.146191.  [DOI](https://doi.org/10.1136/ard.2010.146191) | [PMC free article](/articles/PMC3128324/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21515602/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Rheum.%20Dis.&title=Association%20of%20the%205-Aminoimidazole-4-Carboxamide%20Ribonucleotide%20Transformylase%20Gene%20with%20Response%20to%20Methotrexate%20in%20Juvenile%20Idiopathic%20Arthritis&author=A.%20Hinks&author=H.%20Moncrieffe&author=P.%20Martin&author=S.%20Ursu&author=S.%20Lal&volume=70&publication_year=2011&pages=1395-1400&pmid=21515602&doi=10.1136/ard.2010.146191&)

39. Lee Y.H., Bae S.C. Association of the ATIC 347 C/G Polymorphism with Responsiveness to and Toxicity of Methotrexate in Rheumatoid Arthritis: A Meta-Analysis. Rheumatol. Int. 2016;36:1591–1599. doi: 10.1007/s00296-016-3523-2.  [DOI](https://doi.org/10.1007/s00296-016-3523-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27379764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheumatol.%20Int.&title=Association%20of%20the%20ATIC%20347%20C/G%20Polymorphism%20with%20Responsiveness%20to%20and%20Toxicity%20of%20Methotrexate%20in%20Rheumatoid%20Arthritis:%20A%20Meta-Analysis&author=Y.H.%20Lee&author=S.C.%20Bae&volume=36&publication_year=2016&pages=1591-1599&pmid=27379764&doi=10.1007/s00296-016-3523-2&)

40. Eektimmerman F., Swen J.J., Madhar M.B., Allaart C.F., Guchelaar H.J. Predictive Genetic Biomarkers for the Efficacy of Methotrexate in Rheumatoid Arthritis: A Systematic Review. Pharmacogenom. J. 2020;20:159–168. doi: 10.1038/s41397-019-0098-9.  [DOI](https://doi.org/10.1038/s41397-019-0098-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31624331/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenom.%20J.&title=Predictive%20Genetic%20Biomarkers%20for%20the%20Efficacy%20of%20Methotrexate%20in%20Rheumatoid%20Arthritis:%20A%20Systematic%20Review&author=F.%20Eektimmerman&author=J.J.%20Swen&author=M.B.%20Madhar&author=C.F.%20Allaart&author=H.J.%20Guchelaar&volume=20&publication_year=2020&pages=159-168&pmid=31624331&doi=10.1038/s41397-019-0098-9&)

41. Lahoud M.H., Ristevski S., Venter D.J., Jermiin L.S., Bertoncello I., Zavarsek S., Hasthorpe S., Drago J., de Kretser D., Hertzog P.J., et al. Gene targeting of Desrt, a novel ARID class DNA-binding protein, causes growth retardation and abnormal development of reproductive organs. Genome Res. 2001;11:1327–1334. doi: 10.1101/gr.168801.  [DOI](https://doi.org/10.1101/gr.168801) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11483573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Gene%20targeting%20of%20Desrt,%20a%20novel%20ARID%20class%20DNA-binding%20protein,%20causes%20growth%20retardation%20and%20abnormal%20development%20of%20reproductive%20organs&author=M.H.%20Lahoud&author=S.%20Ristevski&author=D.J.%20Venter&author=L.S.%20Jermiin&author=I.%20Bertoncello&volume=11&publication_year=2001&pages=1327-1334&pmid=11483573&doi=10.1101/gr.168801&)

42. Lin C., Song W., Bi X., Zhao J., Huang Z., Li Z., Zhou J., Cai J., Zhao H. Recent advances in the ARID family: Focusing on roles in human cancer. OncoTargets Ther. 2014;7:315–324. doi: 10.2147/OTT.S57023.  [DOI](https://doi.org/10.2147/OTT.S57023) | [PMC free article](/articles/PMC3933769/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24570593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=OncoTargets%20Ther.&title=Recent%20advances%20in%20the%20ARID%20family:%20Focusing%20on%20roles%20in%20human%20cancer&author=C.%20Lin&author=W.%20Song&author=X.%20Bi&author=J.%20Zhao&author=Z.%20Huang&volume=7&publication_year=2014&pages=315-324&pmid=24570593&doi=10.2147/OTT.S57023&)

43. Baba A., Ohtake F., Okuno Y., Yokota K., Okada M., Imai Y., Ni M., Meyer C.A., Igarashi K., Kanno J., et al. PKA-dependent regulation of the histone lysine demethylase complex PHF2-ARID5B. Nat. Cell Biol. 2011;13:668–675. doi: 10.1038/ncb2228.  [DOI](https://doi.org/10.1038/ncb2228) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21532585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol.&title=PKA-dependent%20regulation%20of%20the%20histone%20lysine%20demethylase%20complex%20PHF2-ARID5B&author=A.%20Baba&author=F.%20Ohtake&author=Y.%20Okuno&author=K.%20Yokota&author=M.%20Okada&volume=13&publication_year=2011&pages=668-675&pmid=21532585&doi=10.1038/ncb2228&)

44. Orsi L., Rudant J., Bonaventure A., Goujon-Bellec S., Corda E., Evans T.J., Petit A., Bertrand Y., Nelken B., Robert A., et al. Genetic polymorphisms and childhood acute lymphoblastic leukemia: GWAS of the ESCALE study (SFCE) Leukemia. 2012;26:2561–2564. doi: 10.1038/leu.2012.148.  [DOI](https://doi.org/10.1038/leu.2012.148) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22660188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Genetic%20polymorphisms%20and%20childhood%20acute%20lymphoblastic%20leukemia:%20GWAS%20of%20the%20ESCALE%20study%20(SFCE)&author=L.%20Orsi&author=J.%20Rudant&author=A.%20Bonaventure&author=S.%20Goujon-Bellec&author=E.%20Corda&volume=26&publication_year=2012&pages=2561-2564&pmid=22660188&doi=10.1038/leu.2012.148&)

45. Vijayakrishnan J., Qian M., Studd J.B., Yang W., Kinnersley B., Law P.J., Broderick P., Raetz E.A., Allan J., Pui C.H., et al. Identification of four novel associations for B-cell acute lymphoblastic leukaemia risk. Nat. Commun. 2019;10:5348. doi: 10.1038/s41467-019-13069-6.  [DOI](https://doi.org/10.1038/s41467-019-13069-6) | [PMC free article](/articles/PMC6877561/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31767839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Identification%20of%20four%20novel%20associations%20for%20B-cell%20acute%20lymphoblastic%20leukaemia%20risk&author=J.%20Vijayakrishnan&author=M.%20Qian&author=J.B.%20Studd&author=W.%20Yang&author=B.%20Kinnersley&volume=10&publication_year=2019&pages=5348&pmid=31767839&doi=10.1038/s41467-019-13069-6&)

46. Yang J.L., Liu Y.N., Bi Y.Y., Wang H. ARID5B gene polymorphisms and the risk of childhood acute lymphoblastic leukemia: A meta-analysis. Int. J. Hematol. 2019;110:272–284. doi: 10.1007/s12185-019-02658-2.  [DOI](https://doi.org/10.1007/s12185-019-02658-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31111395/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Hematol.&title=ARID5B%20gene%20polymorphisms%20and%20the%20risk%20of%20childhood%20acute%20lymphoblastic%20leukemia:%20A%20meta-analysis&author=J.L.%20Yang&author=Y.N.%20Liu&author=Y.Y.%20Bi&author=H.%20Wang&volume=110&publication_year=2019&pages=272-284&pmid=31111395&doi=10.1007/s12185-019-02658-2&)

47. Tamai M., Huang M., Kagami K., Abe M., Somazu S., Shinohara T., Harama D., Watanabe A., Akahane K., Goi K., et al. Association of relapse-linked ARID5B single nucleotide polymorphisms with drug resistance in B-cell precursor acute lymphoblastic leukemia cell lines. Cancer Cell Int. 2020;20:434. doi: 10.1186/s12935-020-01524-0.  [DOI](https://doi.org/10.1186/s12935-020-01524-0) | [PMC free article](/articles/PMC7839197/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33499894/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell%20Int.&title=Association%20of%20relapse-linked%20ARID5B%20single%20nucleotide%20polymorphisms%20with%20drug%20resistance%20in%20B-cell%20precursor%20acute%20lymphoblastic%20leukemia%20cell%20lines&author=M.%20Tamai&author=M.%20Huang&author=K.%20Kagami&author=M.%20Abe&author=S.%20Somazu&volume=20&publication_year=2020&pages=434&pmid=33499894&doi=10.1186/s12935-020-01524-0&)

48. Studd J.B., Vijayakrishnan J., Yang M., Migliorini G., Paulsson K., Houlston R.S. Genetic and regulatory mechanism of susceptibility to high-hyperdiploid acute lymphoblastic leukaemia at 10p21.2. Nat. Commun. 2017;8:14616. doi: 10.1038/ncomms14616.  [DOI](https://doi.org/10.1038/ncomms14616) | [PMC free article](/articles/PMC5337971/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28256501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Genetic%20and%20regulatory%20mechanism%20of%20susceptibility%20to%20high-hyperdiploid%20acute%20lymphoblastic%20leukaemia%20at%2010p21.2&author=J.B.%20Studd&author=J.%20Vijayakrishnan&author=M.%20Yang&author=G.%20Migliorini&author=K.%20Paulsson&volume=8&publication_year=2017&pages=14616&pmid=28256501&doi=10.1038/ncomms14616&)

49. Xu H., Zhao X., Bhojwani D., E S., Goodings C., Zhang H., Seibel N.L., Yang W., Li C., Carroll W.L., et al. ARID5B Influences Antimetabolite Drug Sensitivity and Prognosis of Acute Lymphoblastic Leukemia. Clin. Cancer Res. Off. J. Am. Assoc. Cancer Res. 2020;26:256–264. doi: 10.1158/1078-0432.CCR-19-0190.  [DOI](https://doi.org/10.1158/1078-0432.CCR-19-0190) | [PMC free article](/articles/PMC7574699/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31573954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.%20Off.%20J.%20Am.%20Assoc.%20Cancer%20Res.&title=ARID5B%20Influences%20Antimetabolite%20Drug%20Sensitivity%20and%20Prognosis%20of%20Acute%20Lymphoblastic%20Leukemia&author=H.%20Xu&author=X.%20Zhao&author=D.%20Bhojwani&author=S.%20E&author=C.%20Goodings&volume=26&publication_year=2020&pages=256-264&pmid=31573954&doi=10.1158/1078-0432.CCR-19-0190&)

50. Sharma B., Angurana S., Shah R., Verma S., Bhat A., Bhat G.R., Bakshi D., Jamwal R.S., Tanwar M., Singh S., et al. Genetic association of ARID5B with the risk of colorectal cancer within Jammu and Kashmir. India. Genes Genet. Syst. 2021;96:187–191. doi: 10.1266/ggs.21-00010.  [DOI](https://doi.org/10.1266/ggs.21-00010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34803080/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=India.%20Genes%20Genet.%20Syst.&title=Genetic%20association%20of%20ARID5B%20with%20the%20risk%20of%20colorectal%20cancer%20within%20Jammu%20and%20Kashmir&author=B.%20Sharma&author=S.%20Angurana&author=R.%20Shah&author=S.%20Verma&author=A.%20Bhat&volume=96&publication_year=2021&pages=187-191&pmid=34803080&doi=10.1266/ggs.21-00010&)

51. Singh B., Kinne H.E., Milligan R.D., Washburn L.J., Olsen M., Lucci A. Important role of FTO in the survival of rare panresistant triple-negative inflammatory breast cancer cells facing a severe metabolic challenge. PLoS ONE. 2016;11:e0159072. doi: 10.1371/journal.pone.0159072.  [DOI](https://doi.org/10.1371/journal.pone.0159072) | [PMC free article](/articles/PMC4938613/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27390851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Important%20role%20of%20FTO%20in%20the%20survival%20of%20rare%20panresistant%20triple-negative%20inflammatory%20breast%20cancer%20cells%20facing%20a%20severe%20metabolic%20challenge&author=B.%20Singh&author=H.E.%20Kinne&author=R.D.%20Milligan&author=L.J.%20Washburn&author=M.%20Olsen&volume=11&publication_year=2016&pages=e0159072&pmid=27390851&doi=10.1371/journal.pone.0159072&)

52. Davalieva K., Kostovska I.M., Kiprijanovska S., Markoska K., Kubelka-Sabit K., Filipovski V., Polenakovic M. Proteomics analysis of malignant and benign prostate tissue by 2D DIGE/MS reveals new insights into proteins involved in prostate cancer. Prostate. 2015;75:1586–1600. doi: 10.1002/pros.23034.  [DOI](https://doi.org/10.1002/pros.23034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26074449/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prostate&title=Proteomics%20analysis%20of%20malignant%20and%20benign%20prostate%20tissue%20by%202D%20DIGE/MS%20reveals%20new%20insights%20into%20proteins%20involved%20in%20prostate%20cancer&author=K.%20Davalieva&author=I.M.%20Kostovska&author=S.%20Kiprijanovska&author=K.%20Markoska&author=K.%20Kubelka-Sabit&volume=75&publication_year=2015&pages=1586-1600&pmid=26074449&doi=10.1002/pros.23034&)
