# RARG variant predictive of doxorubicin-induced cardiotoxicity identifies a cardioprotective therapy

## Metadata
**Authors:** Tarek Magdy, Zhengxin Jiang, Mariam Jouni, Hananeh Fonoudi, Davi Lyra-Leite, Gwanghyun Jung, Marisol Romero-Tejeda, Hui-Hsuan Kuo, K Ashley Fetterman, Mennat Gharib, Brian T Burmeister, Mingming Zhao, Yadav Sapkota, Colin J Ross, Bruce C Carleton, Daniel Bernstein, Paul W Burridge
**Journal:** Cell stem cell
**Date:** 2021 Sep 14
**DOI:** [10.1016/j.stem.2021.08.006](https://doi.org/10.1016/j.stem.2021.08.006)
**PMID:** 34525346
**PMCID:** PMC8642268
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8642268/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8642268/pdf/nihms-1734504.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8642268/pdf/nihms-1734504.pdf)

## Abstract

Doxorubicin is an anthracycline chemotherapy agent effective in treating a wide-range of malignancies, but its use is limited by dose-dependent cardiotoxicity. A recent genome-wide association study identified a SNP (rs2229774) in retinoic acid receptor-γ (RARG) as statistically associated with increased risk of anthracycline-induced cardiotoxicity. Here, we show that hiPSC-CMs from patients with rs2229774 and who suffered doxorubicin-induced cardiotoxicity (DIC) are more sensitive to doxorubicin. We determine that the mechanism of this RARG variant effect is mediated via suppression of TOP2B expression and activation of the cardioprotective ERK pathway. We use patient-specific hiPSC-CMs as a drug discovery platform, determining that the RARG agonist CD1530 attenuates DIC and confirm that this cardioprotective effect in an established in vivo mouse model of DIC. This study provides a rationale for clinical pre-chemotherapy genetic screening for rs2229774 and a foundation for the clinical use of RARG agonist treatment to protect cancer patients from DIC.

## Introduction

Advances in pharmacogenomics have been driven by both candidate gene and genome-wide association studies (GWAS) and have resulted in the identification of numerous single nucleotide polymorphisms (SNPs) statistically associated with drug efficacy and toxicity. There is great need to develop suitable tools to fully validate these variants as studies can be underpowered, and variants discovered can be in non-coding regions, not change the amino acid sequence, or be in genes whose connection to drug efficacy or toxicity is totally unknown. Human models are required in order to establish variant function, as prior non-human models have proven inapplicable to patients ([Cheng et al., 2011](#R15)). A major advantage of human models is that they can provide in-depth and relevant mechanistic insight into how a specific genetic variant alters drug efficacy or toxicity ([Musunuru et al., 2018](#R37)), while providing a useful platform for subsequent drug discovery and facilitating rapid translation to the clinic.

The anthracycline doxorubicin (Adriamycin) is a prime candidate for this type of pharmacogenomic analysis. Doxorubicin is utilized in nearly 60% of pediatric cancer treatments ([Hudson et al., 2013](#R23); [van Dalen et al., 2014](#R48)) and 35% of breast cancer treatments, and has made a substantial contribution to the improvement in average 5-year cancer survival rate, which exceeds 80% today. However, the presence of dose-dependent cardiotoxicity was recognized almost immediately after the introduction of doxorubicin treatment began ([Lefrak et al., 1973](#R30); [Swain et al., 2003](#R47); [Von Hoff et al., 1979](#R51)). 7-10% of all childhood cancer survivors will develop heart failure up to 30 years after diagnosis ([Mulrooney et al., 2009](#R36); [van der Pal et al., 2012](#R49)), and the risk of congestive heart failure in adult survivors of childhood cancer is 12 times higher compared to their siblings ([Oeffinger et al., 2006](#R40)). Despite its cardiotoxicity, doxorubicin is still widely used, due to the lack of suitable alternatives.

The high inter-individual variability in predisposition to DIC ([Granger, 2006](#R21); [Hasan et al., 2004](#R22); [Krischer et al., 1997](#R28)) has prompted more than 40 candidate gene SNP association studies ([Blanco et al., 2012](#R7); [Wojnowski et al., 2005](#R52)), and a small number of GWASs including our recent study on 456 pediatric cancer patients treated with anthracyclines (73 cases and 383 controls). This study identified the correlation (odds ratio = 4.7, *P* = 5.9 × 10^−8^) of a non-synonymous coding variant rs2229774 (S427L, [NC_000012.11](https://www.ncbi.nlm.nih.gov/nuccore/NC_000012.11): g. 1253605545G>A) in *RARG*, encoding retinoic acid receptor-γ, with DIC in both the original and in two replication cohorts ([Aminkeng et al., 2015](#R2)). Although *RARG* has been known to play a role in cardiac looping during early development ([Iulianella and Lohnes, 2002](#R26)), the role of RARG in the mature heart is currently unknown. Variant rs2229774 occurs in approximately 15.3% of the population (global minor allele frequency of ~0.08, 1000Genomes ([Auton et al., 2015](#R5))) and is only 16 bp downstream of the sequence that encodes the ligand binding domain of RARG ([Fig. 1a](#F1)) and is predicted to be damaging to RARG function (PROVEAN/SIFT score 0.047). Although GWAS data such as these represent a potential major advance in applying a pharmacogenomic approach to cardio-oncology ([Magdy et al., 2016](#R34)), the connection between these SNP associations and clinical cardiotoxicity is far from proven ([Boyle et al., 2017](#R8)).

### Figure 1. Patient-specific hiPSC-CMs with the RARG variant recapitulate increased risk of doxorubicin-induced cardiotoxicity.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0d10/8642268/e35557565ef5/nihms-1734504-f0001.jpg)

Comparison of hiPSC-CMs from control patients and patients harboring the heterozygous (S427L) rs2229774 variant in RARG, after 24 h or 72 h of doxorubicin treatment. Derived from three Control and three S427L patient hiPSC lines. (A) Upper: RARG composed of ligand independent activation function (AF-1) domain, DNA binding domain (DBD), and ligand binding domain (LBD) that setups the ligand-dependent AF-2 domain to recruit coactivators. Arrow indicates relative location of the RARG SNP. Lower: Sanger sequencing of the Control (C/C, major allele) and S427L (C/T, minor allele) hiPSC lines. Asterisk indicates SNP locus. (B) Representative images for sarcomeric organization in hiPSC-CMs after 24 h treatment with doxorubicin at the indicated concentrations, as assessed by immunofluorescence staining for α-actinin (ACTN2) and cardiac troponin T (TNNT2). (C) Effect of doxorubicin (72 h) on hiPSC-CM viability. Control: n = 22, S427L: n = 47. (D) Effect of doxorubicin (24 h) on apoptosis measured by caspase 3 and 7 activity in hiPSC-CMs. Control: n = 32, S427L: n = 55. (E) Representative images for immunofluorescent staining of DNA damage marker γH2AX in hiPSC-CMs after 24 h treatment with doxorubicin at the indicated concentrations. (F) Quantification of γH2AX staining in hiPSC-CMs by flow cytometry. Control: n = 7, S427L: n = 12. Data are represented as mean ± s.d. (c and d) or s.e.m. (F). ; n.s., not significant; by non-parametric bootstrap test (C and D) or unpaired two-tailed Student’s t-test (F). Scale bars, 25 μm.

Our prior work has demonstrated that human induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CMs) accurately recapitulate patients’ predilection to DIC ([Burridge et al., 2016](#R10)). Briefly, hiPSC-CMs from breast cancer patients who developed DIC recapitulate that increased risk *in vitro*, with decreased cell viability, metabolic function, contraction, sarcomeric structure, calcium handling, and increased reactive oxygen species (ROS) production when exposed to doxorubicin, compared to hiPSC-CMs from patients who were treated with doxorubicin but did not experience cardiotoxicity.

Using this platform, we validate the GWAS-identified variant rs2229774 as directly causative in DIC, confirming that cells from patients harboring this variant are at higher risk of DIC. We discover that a small molecule RARG agonist significantly attenuates cardiomyocyte doxorubicin sensitivity both *in vitro* and *in vivo*, reducing murine acute cardiotoxicity by almost 50%. Further analyses reveal that RARG-associated DIC risk is mediated via two established mechanisms of cardiotoxicity, direct repression of *TOP2B* and regulation of ERK phosphorylation-mediated cardioprotection, pathways known to lead to downstream modulation of mitochondrial integrity. We have thus used patient-derived hiPSC-CMs as a platform to confirm a GWAS suspected variant, discover the mechanism underlying the effect of that SNP, and develop and test a cardioprotective strategy using RARG agonists suitable for further clinical studies. Pre-chemotherapy genetic screening for rs2229774 and/or RARG agonist treatments have the potential to protect cancer patients from DIC.

## Results

Our previous work identified the DIC-associated *RARG* variant rs2229774 in a Canadian pediatric patient cohort and validated it in both Dutch and US patient populations ([Aminkeng et al., 2015](#R2)). We returned to the Canadian cohort and specifically re-recruited these well-phenotyped, doxorubicin-exposed patients, using the original inclusion criteria ([Supplementary Table 1](#SD1)). We recruited three patients with the heterozygous rs2229774 variant and cardiotoxicity (S427L-1, -2, -3. collectively S427L) and three control patients without the variant and without cardiotoxicity (Control-1, -2, -3, collectively Control) ([Fig. 1a](#F1) and [Supplementary Table 2](#SD1)). We derived hiPSCs from peripheral blood of these individuals, and all cell lines passed standard tests for pluripotency and genomic stability ([Peterson and Loring, 2014](#R41)) ([Supplementary Figs. 1a](#SD1)–[c](#SD1)). We then differentiated these patient-derived hiPSCs into cardiomyocytes using our established protocol ([Burridge et al., 2014](#R11)), resulting in an average cardiomyocyte purity of >90% ([Supplementary Fig. 1d](#SD1)–[f](#SD1)).

### hiPSC-CMs with the rs2229774 variant recapitulate patients’ increased risk of DIC

We first sought to determine whether hiPSC-CMs with the *RARG* variant could recapitulate patient-specific increased risk of DIC *in vitro*. A concentration-dependent increase in sarcomeric disarray is a well-established effect of doxorubicin, demonstrated in both neonatal rat ventricular cardiomyocytes ([Minotti et al., 2004](#R35)) and in our previous work in hiPSC-CMs ([Burridge et al., 2016](#R10)). As assessed by immunofluorescent imaging, we observed a consistent effect of sarcomeric disarray in S427L hiPSC-CMs but not in control cells treated with 0.5 μM doxorubicin for 24 h ([Fig. 1b](#F1)). Along with cytoskeletal damage, the S427L hiPSC-CMs showed significantly reduced cell viability after a 72 h treatment with doxorubicin, as compared to the control cells at all concentrations tested (*P* < 0.0001), with half-maximal lethal doses (LD_50_) of 0.63 μM and 2.622 μM for S427L and control cells, respectively ([Fig. 1c](#F1) and [Supplementary Fig. 2a](#SD1)). After 24 h of doxorubicin treatment, apoptosis analysis demonstrated a concentration-dependent increase in cells positive for caspase 3/7 activity, with EC_50_ of 1.228 μM for S427L cells and 2.078 μM for control cells ([Fig. 1d](#F1) and [Supplementary Fig. 2b](#SD1)), suggesting that S427L hiPSC-CMs were more susceptible to programmed cell death. As oxidative stress is one of the major mechanisms underlying the cardiotoxic effects of doxorubicin, we assessed ROS production after 24 h of doxorubicin treatment. Reactive oxygen species (ROS) levels were significantly (*P* < 0.0002) higher in S427L than in the control cells, across the tested concentrations ([Supplementary Fig. 2c](#SD1)). Another mechanism for doxorubicin cardiotoxicity is induction of DNA damage, so we next assessed the level of double-stranded DNA breaks by staining for phosphorylated H2A histone family member X (γH2AX). We observed a concentration-dependent increase in DNA damage ([Fig. 1e](#F1)), and the level of DNA damage was significantly higher in S427L cells than in control cells, at all doxorubicin concentrations tested ([Fig. 1f](#F1)). Combined, these data confirm that hiPSC-CMs from patients with the rs2229774 SNP are more sensitive to DIC.

### Loss of function of RARG leads to increased DIC in hiPSC-CMs

To validate the role of RARG in regulating DIC, we generated knockout (*RARG*-KO) and overexpression (*RARG*-OE) hiPSC-CMs from an isogenic control hiPSC line using CRISPR/Cas9-mediated genome editing ([Fig. 2](#F2)). Altered *RARG* expression in these lines was confirmed at the RNA level by real-time RT-PCR, at the protein level by western blot, and by immunofluorescent imaging ([Fig. 3a](#F3)). At the basal level, both of these lines were phenotypically indistinguishable from their isogenic control and differentiated normally into cardiomyocytes ([Supplementary Fig. 1](#SD1)), indicating that *RARG* was not required for pluripotency or cardiac differentiation *in vitro*. We observed a consistent sarcomeric disarray in *RARG*-KO cells treated with 0.5 μM of doxorubicin, whereas this effect was rarely seen in *RARG*-OE cells, even at 1 μM of doxorubicin compared with *RARG*-WT cells ([Fig. 3b](#F3)). Consistently, *RARG*-KO hiPSC-CMs showed significantly reduced cell viability (*P* < 0.005), whereas *RARG*-OE cells showed significantly improved viability (*P* = 0.0064), as compared to *RARG*-WT cells at all doxorubicin concentrations tested, with LD_50_ of 1.582 μM, 2.724 μM, and 2.061 μM for *RARG*-KO, *RARG*-OE, and *RARG*-WT cells, respectively ([Fig. 3c](#F3)). Analysis of apoptosis via caspase 3/7 activity demonstrated a doxorubicin concentration-dependent increase, with EC_50_ of 0.314 μM, 1.83 μM, and 0.683 μM for *RARG*-KO, *RARG*-OE, and *RARG*-WT cells, respectively ([Fig. 3d](#F3)), suggesting that knockout of *RARG* increased doxorubicin-induced apoptosis, while overexpression attenuated it. Furthermore, we observed a doxorubicin concentration-dependent increase in ROS production ([Supplementary Fig. 3a](#SD1)), mitochondrial membrane dysregulation ([Supplementary Fig. 3b](#SD1)), and DNA damage ([Figs. 3e](#F3) and [f](#F3)), and the levels of ROS, mitochondrial dysfunction, and DNA damage were significantly higher in *RARG*-KO hiPSC-CMs than in *RARG*-WT and *RARG*-OE hiPSC-CMs. Collectively, these data confirm that RARG expression has an important role in influencing the level of DIC and suggest that loss of function of RARG underlies the effect of the rs2229774 SNP in DIC.

### Figure 2. CRISPR/Cas9–mediated knockout and overexpression of RARG gene in isogenic hiPSCs.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0d10/8642268/12f03693810e/nihms-1734504-f0002.jpg)

(A) Schematic of the CRISPR/Cas9-mediated strategy to knockout the RARG gene. Shown are the genomic loci of the three major RARG splice variants, an enlarged view of the first exon with alternative start codon (highlighted in blue), PAM and gRNA targeting sequences (underlined), cutting site of Cas9 (vertical line) and PCR primers for sanger sequencing. (B) Schematic of the CRISPR/Cas9-mediated strategy for the overexpression RARG. Shown are the genomic locus of the PPP1R12C gene, an enlarged view of the AAVS1 locus in the first intron of PPP1R12C, PAM and gRNA targeting sequence (underlined), cutting site of Cas9 (vertical line) and PCR primers for sanger sequencing of the RARG cDNA plasmid (containing most prevalent RARG splice variant 1 cDNA). (C) Table summarizing all targeted modifications in the isogenic hiPSC line. All targeted modifications were determined by Sanger sequencing. One knockout clone with confirmed indels causing frameshift near alternative start codons of all major RARG splice variants and clones with correct RARG cDNA insertion were maintained for subsequent analysis.

### Figure 3. Validation of RARG expression as correlated with doxorubicin-induced cardiotoxicity.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0d10/8642268/90cd108f63b9/nihms-1734504-f0003.jpg)

(A) Expression of RARG in hiPSC-CMs with CRISPR/Cas9-mediated RARG knockout (KO) and AAVS1-based RARG overexpression (OE) in an isogenic hiPSC line, detected by real-time PCR (top left), western blot (top right), and immunofluorescent staining (bottom). (B) Representative images for sarcomeric organization in isogenic, RARG knockout, and RARG overexpression hiPSC-CMs after 24 h treatment with doxorubicin at the indicated concentrations, as assessed by immunofluorescence staining for α-actinin (ACTN2) and cardiac troponin T (TNNT2). (C) Effect of doxorubicin (72 h) on hiPSC-CM cell viability. RARG-WT: n = 27, RARG-KO: n = 13, RARG-OE: n = 17. (D) Effect of doxorubicin (24 h) on apoptosis measured by caspase 3 and 7 activity in hiPSC-CMs. RARG-WT: n = 15, RARG-KO: n = 18, RARG-OE: n = 32. (E) Representative images for immunofluorescent staining of DNA damage marker γH2AX, after treatment of hiPSC-CMs with the indicated concentrations of doxorubicin for 24 h. (F) Quantification of γH2AX staining in hiPSC-CMs by flow cytometry. RARG-WT: n = 5, RARG-KO: n = 7, RARG-OE: n = 10. Data are represented as mean ± s.d. (C and D) or s.e.m. (A and F); n.s., not significant; by non-parametric bootstrap test (C and D) or ANOVA (A and F). Scale bars, 25 μm.

### Doxorubicin treatment results in S427L-dependent transcriptome changes in patient-derived hiPSC-CMs.

We next sought to investigate transcriptome changes of S427L and control hiPSC-CMs in response to 1 μM doxorubicin for 24 h, using an unbiased approach. Principal component analysis (PCA) of RNA-seq data in all 37,557 genes revealed a clear distinction in hiPSC-CMs before and after doxorubicin treatment ([Supplementary Fig. 4a](#SD1)). MA plot ([Supplementary Fig. 4b](#SD1)) and heat map ([Fig. 4a](#F4) and [Supplementary Fig. 4c](#SD1)) analysis shows the significant upregulation (adjusted *P* value < 0.05) of genes associated with apoptosis, DNA damage, oxidative stress, and doxorubicin resistance, and downregulation of genes associated with cardiac function, resulting from doxorubicin treatment ([Supplementary Fig. 4d](#SD1), [e](#SD1) and [Supplementary Table 3](#SD1)). Of particular note, S427L hiPSC-CMs had significantly higher baseline expression of *CAV3*, a gene that is required for doxorubicin-induced apoptosis, through activation of caspase 3 ([Volonte et al., 2008](#R50)). Pathway enrichment analysis revealed no significant difference between S427L and control hiPSC-CMs at the basal level, before doxorubicin treatment ([Supplementary Fig. 4d](#SD1)). After treatment with doxorubicin, S427L hiPSC-CMs showed significantly more activation of doxorubicin-induced apoptosis, TP53 targets, and oxidative phosphorylation than did control cells. In contrast, doxorubicin resistance and DNA repair were more significantly activated in control cells than in S427L cells ([Fig. 4b](#F4) and [Supplementary Fig. 4e](#SD1)). This finding is consistent with our observation that S427L hiPSC-CMs exhibit greater sarcomeric disarray, apoptosis, ROS production, and DNA damage than control cells, upon doxorubicin treatment. Furthermore, we observed significant upregulation of *RARG* in S427L hiPSC-CMs compared to control cells ([Fig. 4c](#F4) and [d](#F4)), suggesting a compensatory mechanism for the loss of function of *RARG* in these cells.

### Figure 4. S427L-dependent transcriptome changes after doxorubicin treatment of patient-derived hiPSC-CMs.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0d10/8642268/a0c23c9548a0/nihms-1734504-f0004.jpg)

(A and B) Differential regulation of genes and pathways in Control and S427L hiPSC lines after treatment with 1 μM doxorubicin for 24 h. Data were obtained using hiPSC-CMs from three patients per group, one cell line per patient (n = 3); 12 RNA-seq samples in total. (A) Heat map of the top 100 most differentially expressed genes. (B) Pathway enrichment analysis of most significant transcriptome changes, showing S427L-dependent differential response. (C) Real-time PCR confirming the effect of the S427L SNP on the expression of RARG and selected genes involved in the TOP2B pathway after doxorubicin treatment of hiPSC-CMs. n = 3 per group. Expression was normalized to ACTB. (D) Western blot showing S427L-dependent upregulation of RARG and TOP2B, both at the basal level and after doxorubicin treatment in hiPSC-CMs. n = 6. Figure is a composite; the original is provided in Supplementary Figure 8. (E) Representative images of JC-10 staining for assessing mitochondrial membrane potential in patient-derived hiPSC-CMs, after treatment with doxorubicin at the indicated concentrations for 24 h. Scale bar, 100 μM. (F) MSD analysis demonstrating significant upregulation of ERK phosphorylation in S427L hiPSC-CMs, compared with the Control, n = 4 per group. Data are represented as mean ± s.e.m. n.s., not significant; by unpaired two-tailed Student’s t-test.

We next sought to analyze how *RARG* variants might influence known mechanisms of DIC. In the mouse model, knockout of *Top2b* attenuates DIC ([Zhang et al, 2012](#R56)). We re-analyzed existing data and found that RARG binds a retinoic acid response element (RARE) 907 bp downstream of the *TOP2B* transcription start site ([Delacroix et al., 2010](#R17); [Lalevee et al., 2011](#R29)). Genes crucial for mitochondrial biogenesis downstream of *TOP2B*, such as *PPARGC1A* and *PPARGC1B*, encoding the transcriptional coactivators PGC-1α and PGC-1β, were also significantly downregulated in S427L hiPSC-CMs compared to control cells ([Fig. 4c](#F4)). The functional effect of these gene expression changes on mitochondrial membrane potential was validated using a lipophilic fluorochrome JC-10, where the red/green fluorescence ratio was markedly reduced in S427L hiPSC-CMs but not in control cells, at lower doxorubicin concentrations ([Fig. 4e](#F4)).

As a second potential mechanism, we were interested in the existing well-established link between ERK activation and prevention of doxorubicin-induced cardiomyocyte damage ([Fryer et al., 2001](#R19); [Izumi et al., 2006](#R27); [Simoncikova et al., 2008](#R44); [Su et al., 2006](#R46); [Xiang et al., 2009](#R53); [Yang et al., 2016](#R55)). In addition to the canonical RARE pathways, retinoic acids are also well known to function through a non-canonical pathway by directly binding protein kinase C α (PRCKA) ([Ochoa et al., 2002](#R39)), resulting in phosphorylation of extracellular regulated kinase (ERK1/2 or MAPK1/3) ([Canon et al., 2004](#R12)), which leads to activation of cAMP response element-binding protein (CREB) ([Aggarwal et al., 2006](#R1)), a transcription factor known to regulate many genes involved in cardiomyocyte survival ([Ichiki, 2006](#R25)). We observed significant increase in phosphorylated ERK (pERK) in S427L hiPSC-CMs upon doxorubicin treatment, compared to control cells ([Fig. 4f](#F4)), suggesting that the effect of the *RARG* SNP on doxorubicin-activated ERK signaling may also play a role in DIC, a finding we further investigate below.

Our data strongly suggest that RARG directly represses *TOP2B* expression and that the rs2229774 SNP disrupts this regulation, leading to increased TOP2B-mediated DNA damage and decreased mitochondrial biogenesis/function, upon doxorubicin treatment, connecting RARG to a previously established mechanism of DIC ([Zhang et al., 2012](#R56)). Of particular note, *RARG* overexpression was as potent as *TOP2B* knockout for reducing DIC in hiPSC-CMs ([Fig. 5](#F5)), suggesting that RARG agonism might be a suitable drug-based cardioprotective methodology to reduce TOP2B-related cardiotoxicity.

### Figure 5. TOP2B knockout or RARG overexpression in isogenic hiPSC-CMs protects against doxorubicin-induced cardiotoxicity.

![Figure 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0d10/8642268/2b8ead30262f/nihms-1734504-f0005.jpg)

(A) Schematic of the CRISPR/Cas9-mediated strategy to knockout the TOP2B gene. Shown is the genomic locus of TOP2B, an enlarged view of the first exon with start codon (highlighted in blue), PAM and gRNA targeting sequences (underlined), cutting site of Cas9 (vertical line) and PCR primers for sanger sequencing. (B) Table summarizing all targeted modifications in the isogenic hiPSC line. Indels were determined by Sanger sequencing. (C) Western blot showing knockout of TOP2B in the isogenic hiPSC line. Sequencing and western blot confirmed knockout clones were maintained for subsequent analysis. (D) Effect of doxorubicin (72 h) on hiPSC-CM viability. TOP2B-WT: n = 26, TOP2B-KO: n = 26, RARG-OE: n = 17. Data are represented as mean ± s.e.m.; by non-parametric bootstrap test.

### Introduction of the RARG variant rs2229774 increases hiPSC-CMs sensitivity to DIC and confirms causality

To directly confirm that rs2229774 is the causative SNP for the increased risk of DIC, we utilized CRISPR/Cas9-mediated genome editing to introduce the CT rs2229774 variant allele in to control patient-specific hiPSC lines that harbored the reference CC genotype (S427S). In total we generated four different hiPSC lines that now harbor the heterozygous genotype CT (S427L) ([Fig. 6a](#F6)), two from Control-2, and two from Control-3. A common mistake made during the attempt to correct a heterozygous variant by homology-directed repair (HDR) is to obtain clones that look like they are homozygous for the major allele by Sanger sequencing of a small PCR amplicon around the target site and conclude that the correction was achieved. However, what happens quite commonly is that a large non-homologous end joining (NHEJ) indel is induced on the minor allele chromosome, preventing PCR amplification of that allele and making a single hemizygous major allele site look like a homozygous major site. In order to avoid this, (1) we introduced the heterozygous allele (CC > CT) of rs2229774 in to hiPSCs harboring the reference allele of that variant (CC) rather than correcting the heterozygous variant allele back to homozygous reference allele; and (2) we introduced two additional synonymous variants including a protospacer adjacent motif (PAM) blocking variant that, once identified by sanger sequencing of our PCR amplicons, confirms the amplification of both alleles ([Fig. 6a](#F6)).

### Figure 6. Introduction of the RARG variant in to Control hiPSCs increase the sensitivity to doxorubicin-induced cardiotoxicity.

![Figure 6.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0d10/8642268/5c015d97ce7a/nihms-1734504-f0006.jpg)

(A) Generation of RARG rs2229774 edited clones (n = 4). SNP rs2229774 is marked by red asterisks. Additional introduced silent mutations are marked by green asterisks. (B) Effect of doxorubicin (72 h) on hiPSC-CM viability. Control (n = 25); Controledited, (n = 40). (C) Representative images of JC-10 staining for assessing mitochondrial membrane potential in hiPSC-CMs, after treatment with doxorubicin at the indicated concentrations for 24 h. Scale bars, 100 μm. (D) Representative images for immunofluorescent staining of DNA damage marker γH2AX in hiPSC-CMs after 24 h treatment with doxorubicin at the indicated concentrations. (E) Quantification of γH2AX staining in hiPSC-CMs by flow cytometry. Control (n = 4); Controledited (n = 11-13). Data are represented as mean ± s.d. (B) or s.e.m. (D). n.s., not significant; by non-parametric bootstrap test (B) or ANOVA (E)

By comparing Control hiPSC-CMs with their edited cells, Control^edited^, we found that the introduction of SNP rs2229774 significantly increased sensitivity to doxorubicin at all concentrations tested, as assessed by cell viability, *P* < 0.0001 ([Fig. 6b](#F6)), mitochondrial membrane potential ([Fig. 6c](#F6)), and DNA damage ([Fig. 6d](#F6)). These results confirm that the rs2229774 SNP alone is responsible for the increased risk of DIC.

### RARG-specific agonists attenuate the cardiotoxicity of doxorubicin without impeding anticancer efficacy

Since our data indicated that loss of function of *RARG* caused by the rs2229774 SNP led to increased doxorubicin sensitivity, we next asked whether stimulating RARG signaling in S427L hiPSC-CMs could reduce doxorubicin sensitivity. We pre/co-treated cells with three *RARG*-specific small molecule agonists (BMS961, CD437, and CD1530) or the general RAR agonist all-trans retinoic acid (ATRA) at concentrations of 0 μM, 0.3 μM, 1 μM, and 10 μM an assessed cell viability after doxorubicin treatment for 72 h over a 5-log concentration range. All of these agonists were able to improve cell viability, with CD1530 being the most effective at an optimal concentration of 1 μM, increasing LD_50_ by 4 to 7-fold for Control and S427L hiPSC-CMs, respectively ([Fig. 7a](#F7), [Supplementary Fig. 5](#SD1) and [Supplementary Table 4](#SD1)). These findings indicate that activation of RARG signaling could reduce doxorubicin cardiotoxicity in hiPSC-CMs both with and without the rs2229774 SNP. Furthermore, activation of RARG signaling by CD1530 in S427L hiPSC-CMs increased *RARG* expression, repressed *TOP2B* expression, and also upregulated genes involved in mitochondrial biogenesis ([Fig. 5b](#F5)–[c](#F5) and [Supplementary Fig. 6a](#SD1)). In order to confirm the mechanism of action of CD1530, we tested whether or not CD1530 protects *TOP2B* knockout hiPSC-CMs against DIC. We found that CD1530 did not alter the DIC sensitivity in *TOP2B* knockout cardiomyocytes. This finding demonstrates that CD1530 exerts its cardioprotective action mainly the RARG mediated attenuation of *TOP2B* ([Fig. 7d](#F7)).

### Figure 7. An RARG specific agonist protects against doxorubicin-induced cardiotoxicity.

![Figure 7.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0d10/8642268/21a0467b33cc/nihms-1734504-f0007.jpg)

(A) Effect of RARG agonist CD81530 on the viability of S427L hiPSC-CMs after 72 h of doxorubicin treatment. n = 6 replicates for each data point. (B) Western blot showing CD1530-dependent regulation of RARG and TOP2B (C) Effect of 1 μM CD 1530 on the RARG expression and TOP2B signaling in the S427L hiPSC-CMs after 3 μM doxorubicin treatment for 24 h, as assessed by real-time PCR, normalized to 18S. n = 3 replicates for each group. (D) Assessment of cell viability of TOP2B knockout hiPSC-CMs after 72 h of doxorubicin and CD1530 co-treatment. (E) MSD analysis showing significant upregulation of ERK phosphorylation in Control hiPSC-CMs with CD1530 treatment, in a dose dependent manner. Fold change is the ratio of pERK between 1 μM and 0 μM of doxorubicin treatment for 24 h. n = 4 per group. (F) Effect of CD1530 (1 μM) and ERK inhibitor U0126 (30 μM) on the viability of control hiPSC-CMs after 72 h of doxorubicin treatment (10 μM). n = 6 replicates for each data point. (G) Representative M-mode echocardiograms of mice at baseline and three weeks after treatment of doxorubicin alone or in combination with CD1530 or ATRA. Scale shown in centimeters. (H) Changes in left ventricular fractional shortening (FS) at three weeks after treatment, normalized to the baseline. (i) Relative expression of TOP2B, pERK and BCL2 in the hearts of mice at three weeks after treatment as assessed by western blot, normalized to GAPDH. (j) Assessment of cell viability of MCF7 breast cancer cell line after 72 h of doxorubicin and RARG agonists (1 μM each) co-treatment, n = 2 replicates for each data point. Data are represented as mean ± s.d. (A, D, and J) or s.e.m. (C, E, F, H, and I); n.s., not significant; by non-parametric bootstrap test (A and J), unpaired two-tailed Student’s t-test (b) or ANOVA / non-parametric Kruskal-Wallis test (D, E, G and H).

To further investigate the non-canonical retinoic acid pathway, we assessed pERK in hiPSC-CMs treated with doxorubicin, with or without pre/co-treatment with CD1530, and we found a significant increase in pERK in CD1530-treated cells ([Fig. 7e](#F7)). Furthermore, inhibition of ERK signaling using the MEK1/2 inhibitor U0126 ([Yang et al., 2016](#R55)) completely abolished the protective effect of CD1530 ([Fig. 7f](#F7)), confirming RARG agonism allows the pERK-dependent cardioprotective mechanism to function.

To investigate whether RARG agonists could attenuate DIC *in vivo*, we treated mice with doxorubicin (3 mg/kg intraperitoneal twice weekly for 3 weeks) plus oral CD1530, ATRA, or vehicle control. Doxorubicin treatment results in a steady decline in cardiac function, as assessed by fractional shortening. Critically, cardiac function was significantly higher at three weeks with CD1530 (*P* < 0.05) and was marginally improved with ATRA (*P* = 0.05), compared with vehicle treatment ([Fig. 7g](#F7) and [h](#F7)). Furthermore, we observed a significant upregulation of the pro-apoptosis protein inhibitor BCL2 and modest but non-significant downregulation of TOP2B in the murine hearts, with treatment of CD1530 but not ATRA, whereas pERK was significantly increased with both CD1530 and ATRA treatment ([Fig. 7i](#F7)). Lastly, to rule out the possibility that RARG agonists could diminish the effectiveness of doxorubicin against cancer cells, we studied four commonly used breast cancer cell lines and found that co-treatment with RARG agonists did not impede the anticancer efficacy of doxorubicin. ([Fig. 7j](#F7), [Supplementary Fig. 6b](#SD1)–[d](#SD1), and [Supplementary Table 5](#SD1)).

## Discussion

We have demonstrated that hiPSC-CMs can recapitulate patient-specific differences in the predilection to DIC and can be utilized for the direct validation of GWAS-identified genetic variants. When coupled with CRISPR/Cas9-mediated genome editing, hiPSCs serve as a powerful new platform for pharmacogenomic evaluation of both drug efficacy and toxicity. This platform can be extended to validating other GWAS-identified SNPs, both in the field of pharmacogenomics and potentially for any disease where the phenotype can be modeled in the dish. By editing hiPSC lines derived from patients with a specific disease phenotype, we can query single SNPs as the sole experimental variable, whilst maintaining other SNPs that may influence the phenotype.

The mechanisms by which doxorubicin affects cancer cells and causes cardiotoxicity are thought to be different, providing an opportunity for the development of cardioprotective agents that do not interfere with chemotherapeutic efficacy. Cancer cells rely on topoisomerase 2α (TOP2A) to introduce double-stranded DNA breaks that release DNA torsional stress generated during rapid DNA replication. Doxorubicin inhibits the decoupling of TOP2A from DNA, preventing DNA re-ligation, resulting in DNA damage ([Yang et al. 2014](#R54)). Doxorubicin also directly intercalates with DNA, causing interference with RNA transcription and DNA polymerases.

In the case of the toxicity of doxorubicin to cardiomyocytes, three major interrelated mechanisms have been proposed: (1) the production of ROS ([Berlin and Haseltine, 1981](#R6)) via redox cycling, using mitochondrial NAD(P)H oxidoreductases ([Minotti et al., 2004](#R35)), and complexing of doxorubicin with iron in mitochondria ([Ichikawa et al., 2014](#R24)), causing mitochondrial lipid peroxidation, mitochondrial permeability transition pore (MPTP) opening, and activation of programmed cell death ([Singal and Iliskovic, 1998](#R45)); (2) doxorubicin-TOP2B-DNA interaction, like TOP2A, TOP2B functions to release DNA torsional stress, although primarily during transcription ([Zhang et al., 2012](#R56)); (3) calcium dysregulation, which results in the pumping of Ca^2+^ out of the sarcoplasmic reticulum causing an increase in intracellular calcium levels, and subsequent sarcomeric and myofibrillar disarray ([Fajardo et al., 2011](#R18); [Lim et al., 2004](#R32)).

In our data, differences in the doxorubicin response of hiPSC-CMs carrying rs2229774 and control hiPSC-CMs are evident in multiple processes that have been identified as downstream determinants of doxorubicin cardiotoxicity, including reduction in cell survival, sarcomeric integrity, ROS production, and mitochondrial function along with increased apoptosis and DNA damage. We confirm that RARG directly represses TOP2B ([Fig. 4d](#F4)), whereas S427L variant leads to de-repression of TOP2B, resulting in comparatively increased TOP2B-mediated DNA damage and decreased mitochondrial biogenesis/function, upon doxorubicin treatment of hiPSC-CMs, a previously established mechanism of DIC ([Zhang et al., 2012](#R56)). Our work comparing the use of *RARG* overexpression with *TOP2B* knockout shows similar levels of DIC attenuation, although it is likely that both manipulations function via multiple downstream mechanisms, requiring substantial further research. Interestingly, retinoic acid receptors and topoisomerases are commonly co-localized within the genome, with *TOP2B* and *RARB* overlapping each other, and *TOP2A* and *RARA* within 30 kb of each other. This genome topology is conserved as far back in evolution as *Xenopus*, suggesting that the control of topoisomerases by retinoic acid receptors may be an evolutionarily ancient mechanism.

Phosphorylation of ERK2 in response to doxorubicin is a well-established phenomenon in DIC that has been connected to mitochondrial protection ([Yang et al., 2016](#R55)). Although we confirm that this process is also present in hiPSC-CMs and that cells with rs2229774 do not as readily undergo this process, without further mechanistic understanding of the relationship between pERK and mitochondrial function and integrity it is not currently possible to separate this variance from the other mechanisms simultaneously occurring.

Our demonstration that hiPSC-CMs can be used to validate the rs2229774 SNP as causative in DIC provides a powerful tool that can be used to screen other SNPs that have been associated with DIC, although most were discovered in much smaller patient populations or using candidate gene-based methodologies rather than GWAS ([Magdy et al., 2016](#R34)). Ultimately, with suitably validated variants, it will be possible to utilize a genetic test to predict the risk of DIC in patients receiving anthracyclines.

Our data also provide strong support for pre-chemotherapy screening of all patients who will receive doxorubicin for the rs2229774 variant ([Aminkeng et al., 2016](#R3)). For patients who harbor this variant, increased cardiac monitoring, reduction of doxorubicin dosing, and/or the use of the only existing approved cardioprotectant, dexrazoxane, are recommended. In addition, RARG agonist treatment (such as CD1530) has the potential to further protect patients with or without the rs2229774 variant from cardiotoxicity.

### Limitations of the study

Although our functional validation of the rs2229774 and *RARG* in DIC are clear, so far the use of pharmacogenomic data clinically has been limited ([Aminkeng et al., 2016](#R3); [Aminkeng et al., 2017](#R4)). It is likely that revised guidelines from national level organizations such as the American Society for Clinical Oncology (ASCO) and the American College of Cardiology (ACC) will be required to overcome this lack of uptake. As currently not even routine echocardiogram is required for doxorubicin-treated patients, this many prove unpalatable. The dramatic decline in the cost of genetic screening and analyses has increased the accessibility of patients to such tests. Thus, a wider clinical application of genetic screening will provide the data to increase the confidence in SNP rs2229774-DIC statistical association. In regard to CD1530 (and CD437 and BMS961), these drugs are tool compounds and not approved to be used in routine clinical practice, therefore development of an alternative or derivative compounds will be required. Further basic and clinical research on these alternatives or derivatives is clearly warranted, especially additional studies in multiple cancer types (such as patient-derived xenograph models) to prove that chemotherapy efficacy is not attenuated.

## STAR METHODS

### 

#### Lead contact

Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Paul Burridge (paul.burridge@northwestern.edu).

#### Materials availability

hiPSC lines used in this study are available from the lead contact lead contact, Paul Burridge (paul.burridge@northwestern.edu). with a completed Materials Transfer Agreement.

#### Data and code availability

RNA-seq data produced in this study is available on GEO with the accession number [GSE164637](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE164637). The software and algorithms for data analyses used in this study are published and referenced. Further details on data analysis and code customization are available from the lead contact, Paul Burridge (paul.burridge@northwestern.edu). upon request.

### EXPERIMENTAL MODEL AND SUBJECT DETAILS

#### Human induced pluripotent cell derivation.

All pluripotent and reprogramming cells were maintained at 37 °C in Heracell VIOS 160i humidified incubators (Thermo Scientific) with 5% CO_2_ and 5% O_2_. During differentiation cells were maintained at 5% CO_2_ and atmospheric O_2_. Protocols were approved by the Northwestern University and University of British Columbia Institutional Review Boards. Patients had previously been genotyped with Illumina Infinium HumanOmniExpress array (738,432 SNPs). With informed written consent, ~9 mL of peripheral blood was taken from each volunteer and shipped at 4 °C, samples were transferred to LeucoSep tubes (Greiner) filled with Histopaque-1077 (Sigma). 1 x 10^6^ isolated peripheral blood mononuclear cells (PMBC) were grown in 24-well tissue culture-treated plates (Greiner) in 2 mL of SFEM II (Stem Cell Technologies) supplemented with 10 ng/mL IL3, 50 ng/mL SCF (KITLG), 40 ng/mL IGF1 (all Peprotech), 2 U mL EPO, 1 μM dexamethasone (both Sigma) ([Chou et al., 2015](#R16)). 50% of the medium (1 mL) was changed every other day. After 12 days of growth, 6 x 10^4^ cells were transferred to a well of a 24-well plate in 500 μL of SFEM II with growth factors supplemented with CytoTune-iPS 2.0 Sendai Reprogramming Kit viral particles (Invitrogen) ([Fusaki et al., 2009](#R20)) diluted to 10% of the manufacturer’s recommendations. Cells were treated with 3.5 μL, 3.5 μL, and 2.2 μL of hKOS (0.85 × 10^8^ CIU/mL), hMYC (0.85 × 10^8^ CIU/mL), and hKLF4 (0.82 × 10^8^ CIU/mL), respectively at MOI of 5:5:3 (KOS:MYC:KLF4). 100% of the medium was changed after 24 h by centrifugation (300 × *g*, 4 min) to 2 mL fresh SFEM II with growth factors, and cells were transferred to one well of a 6-well plate (Greiner) coated with 2 mL of 1:800 growth factor reduced Matrigel (Corning) diluted in DMEM (Corning). 50% of the medium (1 mL) was then changed gently every other day. On d8 after transduction, 100% of medium was changed to E8 medium. E8 medium was made in-house as previously described ([Chen et al., 2011](#R14)) with minor modifications and consisted of DMEM/F12 (10-092-CM, Corning), 20 μg/mL *E. coli*-derived recombinant human insulin (Gibco), 64 μg/mL l-ascorbic acid 2-phosphate trisodium salt (Wako), 5 μg/mL *Oryza sativa*-derived recombinant human transferrin (InVitria/VWR), 14 ng/mL sodium selenite (Sigma), 100 ng/mL recombinant human FGF2 (154 amino acids, *E. coli*-derived, Peprotech), 2 ng/mL recombinant human TGFβ1 (112 amino acid, HEK293-derived, Peprotech), and 100 ng/mL heparin sodium salt (>180 U/mg, Sigma). Medium was changed every day. At d17 individual colonies were picked in to a Matrigel-treated 12-well plate (one colony per well). Subsequently, cells were expanded in Matrigel-coated 6-well plates by passaging using 0.5 mM EDTA (Gibco) in DPBS without Ca^2+^ or Mg^2+^ (Coming) for 6 min at RT. Specific hiPSC clones used for this study were summarized in [Supplementary Table 2](#SD1). The identities of all parental hiPSC lines were confirmed by Sanger sequencing of the rs2229774 SNP after PCR amplification of the genomic DNA with the SNP primer set ([Supplementary Table 6](#SD1)).

### METHOD DETAILS

#### Human induced pluripotent stem cell culture.

Cells were routinely maintained in E8 medium (made as above) on 1:800 diluted growth factor reduced Matrigel. E8 was supplemented with 10 μM Rho kinase inhibitor (Y27632) (LC Labs), hereby referred to as E8Y, for the first 24 h after passage. Cells were passaged at a ratio of ~1:12 to 1:15 every 4 days using 6 min of 0.5 mM EDTA in DPBS at RT, achieving 80% confluence. Cell lines were used between passages 20 and 80. All cultures (pluripotent and differentiation) were maintained with 2 mL medium per 9.6 cm^2^ of surface area or equivalent. All cultures were routinely tested for mycoplasma using a MycoAlert PLUS Kit (Lonza) and a Varioskan LUX (Thermo Scientific) plate reader.

#### Single nucleotide polymorphism karyotyping.

Genomic DNA was extracted from the cell pellets using a Quick-DNA Miniprep Plus kit (Zymo). SNP karyotyping was performed using a whole-genome Infinium CytoSNP-12 BeadChip Array (Illumina) covering 300,000 SNP using a NextSeq 500 (Illumina). Data was analyzed using BlueFuse Multi software (Illumina).

#### CRISPR/Cas9 gRNA and donor vector design.

To generate *RARG* and *TOP2B* knockout gRNA expression vectors, two gRNAs targeting alternative start codons of three major *RARG* splicing variants and one gRNA targeting the start codon of *TOP2B* were designed using an online CRISPR design tool ([http://tools.genome-engineering.org](http://tools.genome-engineering.org)) with minimal predicted off-target effect ([Ran et al., 2013](#R42)). DNA oligos (IDT) encoding each gRNA with BbsI ligation overhangs were annealed and inserted into the BbsI restriction site of a pSpCas9(BB)-2A-GFP (PX458, Addgene 48138) plasmid ([Ran et al., 2013](#R42)). The constructed gRNA expression plasmids were confirmed by Sanger sequencing (Eurofins) with the LKO1_5_primer (5’-GACTATCATATGCTTACCG-3’).

To generate *RARG* overexpression vector, an *AAVS1* gRNA expression vector ([Oceguera-Yanez et al., 2016](#R38)) (pXAT2, Addgene 80494) was used to target *AAVS1* locus in the first intron of the *PPP1R12C* gene ([Oceguera-Yanez et al., 2016](#R38)). Donor plasmid was generated by inserting *RARG* cDNA (CCSB-Broad LentiORF - RARG Clone, Dharmacon ccsbBroad304_01377) under the CAG promoter of a pAAVS1-Nst-CAG-DEST gateway cloning vector (Addgene 80489), which has a neomycin selection cassette in addition to homology arms for *AAVS1.* The constructed *RARG* donor plasmid was confirmed by Sanger sequencing with the P3 primer set ([Supplementary Table 6](#SD1)).

#### CRISPR/Cas9-mediated genome editing of hiPSC.

hiPSC were cultured in E8 medium to ~80% confluence before electroporation. Cells were harvested using TrypLE Express (Gibco) for 3 min at RT and resuspended in E8Y medium. To generate *RARG* and *TOP2B* knockout hiPSCs, 5 × 10^6^ cells from the isogenic *RARG*-WT hiPSC line were electroporated with 1 μg of Cas9 expressing plasmid (pSpCas9(BB)-2A-GFP (PX458), modified as above, ([Supplementary Fig. 3](#SD1)). Cells were plated in to one Matrigel-coated 6-well plate and maintained for 48 h in E8Y medium followed by dissociation with TrypLE and FACS sorting (FACSAria SORP, BD Biosciences) for cells expressing GFP, and subsequently plated at a low density (500 cells per 9.6 cm^2^ of surface area) in E8Y medium. Individual colonies were picked and expanded ~12 days after electroporation. Correctly targeted clones were identified by Sanger sequencing with primers outside of the targeting region ([Supplementary Table 6](#SD1)).

To generate *RARG* overexpression hiPSCs, 5 × 10^6^ cells from cells the isogenic *RARG*-WT hiPSC line were electroporated with 3 μg *AAVS1* gRNA expression vector (pXAT2, Addgene) and 1 μg *RARG* donor plasmid ([Fig. 2b](#F2)). Cells were plated in to one Matrigel-coated 6-well plate and maintained for 48 h in E8Y medium on Matrigel-coated plates followed by E8Y medium supplemented with 100 μg/mL of geneticin (Gibco) for one week of neomycin selection. Neomycin resistant colonies were subsequently picked and validated for *RARG* insertion at *AAVS1* locus by Sanger sequencing with the P3 primer set ([Supplementary Table 6](#SD1)).

For *RARG* SNP rs2229774 editing, 1 × 10^6^ cells from patient-specific control hiPSC lines were electroporated with 5 μg Cas9 expressing plasmid (pSpCas9(BB)-2A-Puro (PX459) V2.0), 3 μg gBlock that expresses a FE-modified sgRNA with an A-U flip and extended stem that cuts two bp away from the rs2229774 locus ([Chen et al., 2013](#R13)), and 3 μl of 100 μM 80 bp symmetric ssODN repair template. Electroplated cells were plated into one 6-well plate in E8Y for 48 h, cells were then selected with 0.3 μg/ml of puromycin for 48 h in E8Y medium. Surviving individual clones were picked and expanded. DNA was isolated, and rs2229774 locus was PCR amplified and Sanger sequenced. Confirmed edited clones were expanded in E8 medium on Matrigel-coated plates.

#### Cardiac differentiation.

Differentiation into cardiomyocytes was performed as previously described ([Supplementary Fig. 1d](#SD1)) ([Burridge et al., 2015](#R9); [Burridge et al., 2014](#R11)). All cell lines for each individual experiment were differentiated in parallel to further reduce experimental variability. Briefly, hiPSCs were split at a 1:15 ratio using 0.5 mM EDTA as above and grown in E8 medium for 4 days reaching ~80% confluence. At the start of differentiation (day 0), E8 medium was changed to CDM3 ([Burridge et al., 2014](#R11)), consisting of RPMI 1640 (Corning), 500 μg/ml *Oryza sativa*-derived recombinant human albumin (Oryzogen), and 213 μg/ml l-ascorbic acid 2-phosphate (Wako). For the first 24 h, CDM3 medium was supplemented with 6 μM of glycogen synthase kinase 3-β inhibitor CHIR99021 (LC Labs). On day 1, medium was changed to CDM3 and on day 2 medium was changed to CDM3 supplemented with 2 μM of Wnt inhibitor Wnt-C59 (Biorbyt). Medium was then changed on day 4 and every other day for CDM3. Contracting cells were noted from day 7. On day 10, medium was changed to CDM3-L consisting of RPMI 1640 no glucose (Corning), CDM3 supplement, and with 4 mM l-lactic acid diluted in 1 M HEPES (both Sigma-Aldrich). On day 14, cardiomyocytes were dissociated using DPBS for 20 min at 37 °C followed by 1:200 Liberase TH (Roche) in DPBS for 20 min at 37 °C, centrifuged at 300 *g* for 5 min, filtered through a 100 μm cell strainer (Fisherbrand). Live cells were counted using a LUNA-FL Dual Fluorescence cell counter (Logos Biosystems) then plated onto Matrigel-treated Nunc Lab-Tek II 8-chamber slides (50,000 cells per well), No 1.5 coverslips (100,000 cells per coverslip) in 12-well plates, 24-well plates (1 × 10^6^ cells per well), or 384-well white-sided μClear plates (25,000 cells per well) (all Greiner), in CDM3 medium supplemented with 40% FBS (Seradigm, VWR) for 48 h and changed back to CDM3 medium thereafter. Cardiomyocytes were used for analysis between day 20 and 30 days after differentiation.

#### Immunofluorescent staining.

hiPSCs were dissociated with 0.5 mM EDTA and plated onto Matrigel-coated Nunc Lab-Tek II 8-chamber slides in E8 medium for three days (E8Y for the first 24 h). Cells were fixed, permeabilized, and stained for POU5F1, SSEA4, SOX2, TRA-1-60 with the PSC 4-Marker Immunocytochemistry Kit (Invitrogen) according to manufacturer’s instructions. Cardiomyocytes were dissociated with Liberase TH (as above) and plated onto Matrigel-coated Nunc Lab-Tek II 8-chamber slides as previously described and allowed to adhere and spread for a week. Cells were fixed with 4% paraformaldehyde (Electron Microscopy Services) in DPBS for 15 min at RT, permeabilized with 1% saponin (Sigma) in DPBS for 15 min at RT, blocked with 3% bovine serum albumin (BSA, Sigma) in DPBS for 30 min at RT, and stained overnight in 3% BSA/1% saponin/DPBS at 4 °C with 1:200 polyclonal rabbit IgG anti-TNNT2 (Abeam, ab45932), 1:500 monoclonal mouse IgG_1_ anti-ACTN2 (Sigma, A7811), 1:200 monoclonal rabbit IgG RARG (Abeam, abl87159), or 1:200 monoclonal mouse IgG_1_ γH2AX (Sigma, 05-636). Cells were washed and then stained with secondary antibodies 1:1000 Alexa Fluor 488 goat anti-rabbit IgG, Alexa Fluor 594 goat anti-mouse IgG_1_, or Alexa Fluor 488 goat anti-mouse IgG_1_, Alexa Fluor 594 goat anti-rabbit IgG (all Invitrogen) in 3% BSA/1% saponin/DPBS for 1 hr. at RT in the dark. Cells were washed three times and mounted with ProLong Diamond Antifade Mountant with DAPI (Invitrogen). Slides were imaged with a Ti-E inverted fluorescent microscope (Nikon Instruments) and a Zyla sCMOS camera (Andor) using NIS-Elements 4.4 Advanced software.

#### Flow cytometry.

hiPSCs were dissociated with TrypLE Express (Gibco) for 3 min at RT and 1 × 10^6^ cells were transferred to flow cytometry tubes (Fisherbrand). For staining of surface markers, cells were incubated in 1% FBS and 0.09% sodium azide (Sigma) in DPBS using 1:20 mouse IgG_3_ SSEA4-488 (BD Biosciences, 560308) for 30 min at RT then washed. For intracellular staining, cells were fixed and permeabilized with −20 °C methanol (Fisher Bioreagents) for 10 min, washed with DPBS, and stained using 1:20 mouse IgG_1_ POU5F1-647 (BD Biosciences, 560307), and mouse IgG_1_ NANOG-647 (BD Biosciences, 561300) for 30 min at RT then washed. Isotype controls mouse IgG_3_-488 (BD Biosciences, 563636) and mouse IgG_1_-647 (BD Biosciences, 565571) were used to establish gating. For staining of intracellular markers, cardiomyocytes were dissociated with Liberase TH (as above), fixed and permeabilized with −20 °C methanol for 10 min, washed with DPBS, and stained using 1:500 mouse monoclonal IgG_1_ TNNT2-647 (BD Biosciences, 565744) or 1:500 mouse monoclonal IgG_2b_ MYH-PE (BD Biosciences, 564408) for 30 min at RT and washed again with DPBS. Isotype controls mouse IgG_1_-647 (BD Biosciences, 565571) and mouse IgG_2b_-PE (BD Biosciences, 555743) were used to establish gating. Human dermal fibroblasts showed no positive staining under these conditions. All cells were analyzed using a CytoFLEX (Beckman Coulter) with CytExpert 2.0 software.

#### Doxorubicin treatment.

Doxorubicin hydrochloride (MedChem Express) was resuspended to 10 mM in cell culture-grade water (Corning) and aliquots were stored at −20 °C. Day 30 hiPSC-CMs were treated for 24 h or 72 h with doxorubicin (0.01-100 μM) diluted in RPMI 1640 medium (no phenol red, Corning) supplemented with 500 μg/ml recombinant human serum albumin (Oryzogen). For RARG agonist treatment, day 30 hiPSC-CMs were treated with respective agonists (all from Tocris, resuspended in DMSO) for 24 h prior to doxorubicin administration and then a second dose was co-administered with doxorubicin as above.

#### 384-well plate-based cell viability, caspase 3/7 activity, and reactive oxygen species (ROS) assays.

To measure cell viability after 72 h of doxorubicin (0.01-100 μM) treatment, CellTiter-Glo 2.0 (Promega) was used per manufacturer’s instructions. Luminescence was measured using a VarioSkan Lux Multi-Mode Reader (Thermo Scientific) with an integration time of 0.25 sec. 10 μM staurosporine (MedChemExpress) was used as a positive control. After 24 h of doxorubicin (0.01-100 μM) treatment, apoptosis and ROS was measured using Caspase 3/7-Glo and ROS-Glo H_2_O_2_ (both Promega) respectively according to manufacturer’s instructions with an integration time of 1 sec. 10 μM staurosporine and 50 μM menadione (both MedChemExpress) was used as a positive control, respectively. Data were analyzed using Prism 7.0 software (GraphPad) using standard dose-response guidelines.

#### Mitochondrial membrane potential assay.

hiPSC-CMs were cultured on Matrigel-coated Nunc Lab-Tek II 8-chamber slides, treated with doxorubicin for 24 h, washed and stained with JC-10 (Enzo Life Sciences) diluted to 2 μM in RPMI 1640, for 30 min at 37 °C, then mounted with ProLong Diamond Antifade Mountant with NucBlue live cell stain (Invitrogen). 50 μM FCCP (MedChemExpress) was used as a positive control. Slides were imaged with a Ti-E inverted fluorescent microscope (Nikon Instruments) and a Zyla sCMOS camera (Andor) using NIS-Elements 4.4 Advanced software.

#### Flow cytometry-based DNA damage assay.

After 24 h of doxorubicin treatment, hiPSC-CMs were dissociated with Liberase TH as described before, processed with BD Cytofix/Cytoperm fixation/permeabilization kit per manufacturer’s instructions, and stained with 1:20 mouse IgG_1_ γH2AX-647 (BD Biosciences, 560447) at 4 °C for 30 min in the dark and washed again with DPBS. Isotype control mouse IgG_1_-647 (BD Biosciences, 565571) was used to establish gating. Cells were analyzed using a CytoFLEX (Beckman Coulter) with CytExpert 2.0 software and Prism 7.0 software (GraphPad).

#### RNA-seq gene expression analysis.

Day 30 hiPSC-CMs from 6 patients (3 Control and 3 S427L) were treated with 1 μM doxorubicin or vehicle for 24 h. We have previously determined that this is a physiologically relevant dose ([Burridge et al., 2016](#R10)). RNA was extracted using TRI Reagent and Direct-zol RNA microprep kit (both Zymo) including on-column DNase digestion to remove genomic DNA. Samples were quantified using an Agilent 2100 Bioanalyzer and passed QC. Library preparation was done using a TruSeq RNA v2 kit (Illumina) and sequencing with NextSeq 500 instrument (Illumina) by Northwestern’s NuSeq core facility, generating ~40 million single-end 75 bp reads for each sample. Reads were mapped to the GRCh38 reference human genome using Subread software ([Liao et al., 2013](#R31)). Gene expression levels and exon usage were estimated using featureCounts function in the Subread software ([Liao et al., 2013](#R31)). Differential gene expression analysis was done using DEseq2 package ([Love et al., 2014](#R33)) and R (v3.3.3). Bioinformatics script and codes for the analysis are available upon request.

#### Quantitative Real-time PCR.

RNA was isolated using TRI reagent (Zymo) and Direct-zol RNA microprep kit (Zymo) including on-column DNase digestion to remove genomic DNA. cDNA was produced from 1 μg of total RNA using Maxima H Minus cDNA Synthesis Master Mix (Thermo Scientific). All PCR reactions were performed in triplicate in a 384-well plate format using TaqMan Fast Advanced Master Mix in a QuantStudio 5 Real-Time PCR System (both Applied Biosystems) with following TaqMan Gene Expression Assays (Applied Biosystems): 18S (Hs99999901_s1), *ACTB* (Hs01060665_g1), *GAPDH* (Hs02786624_g1), *NANOG* (Hs02387400_g1), *POU5F1* (Hs00999632_g1), *SOX2* (Hs01053049_s1), *KLF4* (Hs00358836_m1), *LIN28* (Hs00702808_s1), *MYC* (Hs00153408_m1), *UTF1* (Hs00747497_g1), *DNMT3B* (Hs01003405_m1), *TERT* (Hs99999022_m1), *ZFP42* (Hs00399279_m1), *TP53* (Hs99999147_m1), *RARG* (Hs01559234_m1), *TOP2B* (Hs00172259_m1), *PPARGC1A* (Hs00173304) and *PPARGC1B* (Hs00993805_m1). Relative quantification of gene expression was calculated using 2^−ΔΔCt^ method ([Schmittgen and Livak, 2008](#R43)), normalized to the reference 18S, *ACTB*, or *GAPDH* and untreated control samples as specified in the figure legends.

#### Western blot.

We washed and harvested cells in cold PBS and collected them by centrifugation at 20,000 × *g* for 10 min. at 4 °C. Pellet was lysed in PBS containing 1% SDS and protease inhibitors (cOmplete, ROCHE) and sonicated (2 pulses of 12 sec at 125 Watt) to shear DNA. We centrifuged the samples at 20,000 × *g* for 15 min. at RT to precipitate and discard the insoluble material and the protein concentration in the supernatant was quantified using BCA assay (Thermo Scientific) following manufacturer’s instructions. 30 μg protein from each sample was subjected to SDS-PAGE in a 10% acrylamide gel for 90 min. at 150 V. and transferred to a PVDF membrane (Immobilon-P, Millipore) at 240 mA. for 90 min. Membranes were blocked in TBST containing 5% BSA for 1 hr. at RT and incubated at 4 °C overnight with the following primary antibodies: 1:1000 monoclonal rabbit IgG anti-RARG (Abeam, AM91368), 1:250 monoclonal mouse IgG_1_ anti-TOP2B (BD Bioscience, 611493), 1:5000 monoclonal mouse IgG anti-β-Tubulin (Invitrogen, MA5-18308). The membrane was washed three times for 10 min with TBST and incubated with the appropriate secondary antibody: 1:5,000 HRP-goat anti-mouse IgG or HRP-goat anti-rabbit IgG (Life Technologies, 626520 and 656120) for 1 hr. at RT. After three additional washes with TBST, membranes were developed by chemiluminiscence (SuperSignal West Pico, Thermo Scientific). Signals were captured using a CCD camera-based imager (Azure Biosystems) and quantified using Image J software. Mice heart tissues were harvested and homogenized using lysis buffer (80 mM NaCl, 0.05% Triton X, 1 mM EDTA, 20 mM HEPES, 0.5% sodium deoxycholate, 1 mM DTT, 20 mM β-glycerophosphate) with TissueLyser LT (Qiagen) and centrifuged at 10,000 rpm for 15 min at 4 °C. Proteins were quantified with Bradford (Bio-Rad) and separated by TrisGlycine SDS/PAGE (Bio-Rad). Transferred proteins on nitrocellulose membrane were probed against TOP2B (R&D Systems), phosphorylated ERK1/2 (Cell Signaling Technologies), BCL2 (Santa Cruz Biotechnology), and GAPDH (Advanced ImmunoChemical).

#### MSD analysis of phosphorylated ERK.

hiPSC-CMs were collected in lysis buffer (150 mM NaCl, 1% Triton X-100, 50 mM Tris-HCl, pH 8.0) supplemented with Phosphatase Inhibitor II (1:100, Sigma-Aldrich), Phosphatase Inhibitor Cocktail 3 (1:100, Sigma-Aldrich), 1 mM NaVO_4_ and protease inhibitors (cOmplete, ROCHE). We centrifuged the samples at 50,000 g for 15 min. at 4 °C to precipitate and discard the insoluble material and the protein concentration in the supernatant was quantified using BCA assay (Thermo Scientific) following manufacturer’s instructions. 2.5 μg protein/sample were analyzed using the MSD Phospho(Thr202/Tyr204; Thr185/Tyr187)/Total ERK1/2 Assay Whole Cell Lysate Kit (Meso Scale Discovery, K15100D-1), according to manufacturer’s recommendations. The ratio of phosphorylated ERK was normalized to Total ERK1/2 signal and analyzed using Prism 7.0 software (GraphPad).

#### Breast cancer cell lines.

Four human breast cancer cell lines were used, MCF7 (ATCC HTB-22) and Hs 578T (ATCC HTB-126) both cultured in RPMI 1640 (Corning) with 10% FBS (Opti-Gold, GenDEPOT), MDA-MB-231 (ATCC HTB-26) and MDA-MB-468 (ATCC HTB-131) both cultured in DMEM (Coming) with 10% FBS. All cells were cultured on uncoated tissue culture plates (Greiner) and passaged with TrypLE Express (Gibco).

#### Mouse model of doxorubicin-induced cardiomyopathy and drug administration.

C57BL/6J Mice were treated with doxorubicin (NovaPlus) alone as a control (*n* = 16), either co-treated with ATRA (all-trans retinoic acid, R2625 Sigma-Aldrich) or CD1530 (RARG-specific agonist, CAS107430-66-0, Tocris Bioscience) as experimental groups (*n* = 8 and *n* = 10, respectively). Mice were pretreated with CD1530 or ATRA (12 mg/kg each) for 3 days (day −3 ~ day 0) and at day 0, mice were treated with doxorubicin (3 mg/kg) intraperitoneally twice a week and CD1530 or ATRA once a day by oral gavage for 3 weeks (day 0-day 21). For the control group, we treated mice with corn oil in the same schedule as agonist administration. We recorded an echocardiogram once a week (day 0, day 7, day 14, and day 21) and terminated the experiment at day 21. Hearts were harvested for western blot analyses as mentioned above.

#### Echocardiographic evaluation.

Mice were studied at baseline and weekly during the protocol under light anesthesia with isoflurane (induction 3%, maintenance 1.5%). 2D images in the parasternal short axis were obtained with a GE Vivid 7 ultrasound system (GE Healthcare) equipped with a 13 MHz transducer. Left ventricular end-systolic (LVESD) and end-diastolic (LVEDD) dimensions were measured and left ventricular fractional shortening (FS) was calculated.

#### Statistical methods.

Data were analyzed in Excel or R and graphed in GraphPad Prism 7. Detailed statistical information is included in the corresponding figure legends. Comparisons were conducted via one-way ANOVA test, an unpaired two-tailed Student’s t-test, or bootstrap testing with significant differences defined as *P* < 0.05 (*), *P* < 0.01 (**), *P* < 0.001 (***), and *P* < 0.0001 (****). Our sample size (3 patients in each category) was based on the feasibility of handling this number of hiPSC lines. Patient exclusion criteria are outlined in [Supplementary Table 1](#SD1). No statistical methods were used to predetermine sample size. The experiments were not randomized, and the investigators were not blinded to allocation during experiments and outcome assessment.

## Supplementary Material

| REAGENT or RESOURCE | SOURCE | IDENTIFIER |
| --- | --- | --- |
| Antibodies |  |  |
| Rabbit polycolonal IgG anti-POU5F1 | Invitrogen | A24867; RRID: AB_2650999 |
| Mouse monoclonal IgG3 anti-SSEA4 | Invitrogen | A24866; RRID: AB_2651001 |
| Rat monoclonal IgG2a anti-SOX2 | Invitrogen | A24759; RRID: AB_2651000 |
| Mouse monoclonal IgM anti-TRA-1-60 | Invitrogen | A24868; RRID: AB_2651002 |
| Mouse monoclonal IgG1 anti-ACTN2 (clone EA-53) | Sigma | A7811; RRID: AB_476766 |
| Rabbit polyclonal IgG anti-TNNT2 | Abcam | ab45932; RRID: AB_956386 |
| Rabbit monoclonal IgG anti-RARG (KO validated) | Abcam | ab187159; RRID: NA |
| Mouse monoclonal IgG1 anti-γH2AX | Sigma | 05-636; RRID: AB_309864 |
| Goat anti-rabbit IgG Alexa Fluor 488 | Invitrogen | A-11008; RRID: AB_143165 |
| Goat anti-mouse IgG1 Alexa Fluor 594 | Invitrogen | A-21125; RRID: AB_2535767 |
| Goat anti-mouse IgG1 Alexa Fluor 488 | Invitrogen | A-21121; RRID: AB_143165 |
| Goat anti-rabbit IgG Alexa Fluor 594 | Invitrogen | A-11021; RRID: NA |
| Mouse monoclonal IgG3 SSEA4-488 (clone MC813-70) | BD Biosciences | 560308; RRID: AB_1645371 |
| Mouse monoclonal IgG1 POU5F1-647 (clone 40/Oct-3) | BD Biosciences | 560307; RRID: AB_1645319 |
| Mouse monoclonal IgG1 TNNT2-647 (clone 13-11) | BD Biosciences | 565744; RRID: AB_1645371 |
| Mouse monoclonal IgG2b MYH-PE (clone MF20) | BD Biosciences | 564408; RRID: AB_2739341 |
| Isotype control: mouse IgG3-488 | BD Biosciences | 563636; RRID: NA |
| Isotype control: mouse IgG1647 | BD Biosciences | 565571; RRID: NA |
| Isotype control: mouse IgG2b-PE | BD Biosciences | 555743; RRID: NA |
| Mouse monoclonal IgG1 anti-TOP2B (clone40/Topo II) | BD Biosciences | 611493; RRID: AB_398953 |
| Rabbit polyclonal IgG anti-RARG | Abcam | Ab191368; RRID: NA |
| Mouse monoclonal IgG anti-β-Tubulin | Invitrogen | MA5-18308; RRID: NA |
| Mouse monoclonal IgG2b anti-TOP2B | R&D Systems | MAB6348; RRID: AB_10889629 |
| Rabbit polyclonal anti-phosphorylated ERK1/2 | Cell Signalling Technologies | 9102; RRID: AB_330244 |
| Chemicals, Peptides, and Recombinant Proteins |  |  |
| Recombinant human IL3 | Peprotech | 200-03 |
| Recombinant human SCF | Peprotech | 300-07 |
| Recombinant human IGF1 | Peprotech | 100-11 |
| Recombinant human EPO | Calbiochem | 329871 |
| Dexamethasone | Sigma | D4902 |
| Recombinant human insulin | Gibco | A11382IJ |
| l-ascorbic acid 2-phosphate trisodium salt | Wako | 321-44823 |
| Recombinant human transferrin | In Vitria | 777TRF029 |
| Recombinant human FGF2 | Peprotech | 100-18B |
| Recombinant human TGFβ1 | Peprotech | 100-21 |
| Y27632 | LC Labs | Y-5301 |
| Recombinant human albumin | Oryzogen | OsrHSA |
| CHIR99021 | LC Labs | C-6556 |
| Wnt-C59 | Biorbyt | orb1881132 |
| l-lactic acid | Sigma | 199257 |
| Doxorubicin hydrochloride | MedChem Express | HY-15142 |
| CD1530 | Tocris Bioscience | 2554 |
| BMS 961 | Tocris Bioscience | 3410 |
| CD 437 | Tocris Bioscience | 1549 |
| Retinoic acid | Sigma | R2625 |
| Staurosporine | MedChemExpress | HY-15141 |
| Menadione | MedChemExpress | HY-B0332 |
| FCCP | MedChemExpress | HY-100410 |
| Critical Commercial Assays |  |  |
| MycoAlert PLUS Kit | Lonza | LT07-705 |
| Infinium CytoSNP-12 BeadChip Array | Illumina | WG-320-2101 |
| CellTiter-Glo 2.0 | Promega | G9242 |
| Caspase 3/7-Glo | Promega | G8091 |
| ROS-Glo H2O2 | Promega | G8820 |
| JC-10 | Enzo Life Sciences | ENZ-52305 |
| TruSeq RNA Library Prep Kit v2 | Illumina | RS-122-2001 |
| Deposited Data |  |  |
| Raw and analysed data | This paper | GEO: GSE164637 |
| Experimental Models: Cell Lines |  |  |
| Control-1 | This paper | Jiang et al. |
| Control-2 | This paper | Jiang et al. |
| Control-3 | This paper | Jiang et al. |
| S427L-1 | This paper | Jiang et al. |
| S427L-2 | This paper | Jiang et al. |
| S427L-3 | This paper | Jiang et al. |
| 19-3 RARG-OE | This paper | Jiang et al. |
| 19-3 RARG-KO | This paper | Jiang et al. |
| Control-1edited-1 | This paper | Jiang et al. |
| Control-1edited-2 | This paper | Jiang et al. |
| Control-3edited-1 | This paper | Jiang et al. |
| Control-3edited-2 | This paper | Jiang et al. |
| MCF7 (invasive ductal carcinoma) | ATCC | HTB-22; RRID: CVCL_0031 |
| Hs 578T (invasive ductal carcinoma) | ATCC | HTB-126; RRID: CVCL_0332 |
| MDA-MB-231 (breast adenocarcinoma) | ATCC | HTB-26; RRID: CVCL_0062 |
| MDA-MB-468 (breast adenocarcinoma) | ATCC | HTB-131; RRID: CVCL_0418 |
| Experimental Models: Organisms/Strains |  |  |
| C57BL/6J mice | The Jackson Laboratory | 000664 |
| Oligonucleotides |  |  |
| 18S (Hs99999901_s1) | Applied Biosystems | 4331182 |
| ACTB (Hs01060665_g1) | Applied Biosystems | 4331182 |
| GAPDH (Hs02786624_g1) | Applied Biosystems | 4331182 |
| NANOG (Hs02387400_g1) | Applied Biosystems | 4331182 |
| POU5F1 (Hs00999632_g1) | Applied Biosystems | 4331182 |
| SOX2 (Hs01053049_s1) | Applied Biosystems | 4331182 |
| KLF4 (Hs00358836_m1) | Applied Biosystems | 4331182 |
| LIN28 (Hs00702808_s1) | Applied Biosystems | 4331182 |
| MYC (Hs00153408_m1) | Applied Biosystems | 4331182 |
| UTF1 (Hs00747497_g1) | Applied Biosystems | 4331182 |
| DNMT3B (Hs01003405_m1) | Applied Biosystems | 4331182 |
| TERT (Hs99999022_m1) | Applied Biosystems | 4331182 |
| ZFP42 (Hs00399279_m1) | Applied Biosystems | 4331182 |
| TP53 (Hs99999147_m1) | Applied Biosystems | 4331182 |
| RARG (Hs01559234_m1) | Applied Biosystems | 4331182 |
| TOP2B (Hs00172259_m1) | Applied Biosystems | 4331182 |
| PPARGC1A (Hs00173304) | Applied Biosystems | 4331182 |
| PPARGC1B (Hs00993805_m1) | Applied Biosystems | 4331182 |
| Recombinant DNA |  |  |
| pSpCas9(BB)-2A-GFP (PX458) | Addgene | 48138 |
| pSpCas9(BB)-2A-Puro (PX459) V2.0 | Addgene | 62987 |
| pXAT2 (AAVS1 sgRNA expression vector) | Addgene | 80494 |
| pAAVS1-Nst-CAG-DEST (gateway cloning vector) | Addgene | 80489 |
| CCSB-Broad LentiORF - RARG Clon | Dharmacon | ccsbBroad304_01377 |
| Software and Algorithms |  |  |
| Subread | Liao et al., 2013 | http://subread.sourceforge.net/ |
| FeatureCounts | Liao et al., 2013 | http://subread.sourceforge.net/ |
| DESeq2 | Love et al., 2014 | https://bioconductor.org/packages/release/bioc/html/DESeq2.html |
| R | R Core Team, 2020 | https://www.r-project.org/ |

Table Caption: KEY RESOURCES TABLE

## Acknowledgments.

This work was supported by NUT grants K99/R00 HL121177, R01 CA220002, and R01 CA261898, American Heart Association Transformational Project Award 18TPA34230105, and the Leducq Foundation 19CVD02 (P.W.B.), Canadian Cancer Society (C.J.R., B.C.C., P.W.B.), Michael Smith Foundation for Health Research Scholar Award (C.J.R.), and NIH R33 HL123655 (D.B., B.C.C.). Patients from which hiPSC were derived were recruited by the Canadian Pharmacogenomics Network for Drug Safety (B.C.C. and C.J.R.) which has received grant funding from the Canadian Institutes for Health Research (CIHR), the CIHR Drug Safety and Effectiveness Network, Genome Canada, Genome British Columbia, and the British Columbia Provincial Health Services Authority.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

RNA-seq data produced in this study is available on GEO with the accession number [GSE164637](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE164637). The software and algorithms for data analyses used in this study are published and referenced. Further details on data analysis and code customization are available from the lead contact, Paul Burridge (paul.burridge@northwestern.edu). upon request.

### Supplementary Materials

### Data Availability Statement

RNA-seq data produced in this study is available on GEO with the accession number [GSE164637](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE164637). The software and algorithms for data analyses used in this study are published and referenced. Further details on data analysis and code customization are available from the lead contact, Paul Burridge (paul.burridge@northwestern.edu). upon request.

## References

1. Aggarwal S, Kim SW, Cheon K, Tabassam FH, Yoon JH, and Koo JS, (2006). Nonclassical action of retinoic acid on the activation of the cAMP response element-binding protein in normal human bronchial epithelial cells. Mol Biol Cell 17, 566–575.  [DOI](https://doi.org/10.1091/mbc.E05-06-0519) | [PMC free article](/articles/PMC1356569/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16280361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Cell&title=Nonclassical%20action%20of%20retinoic%20acid%20on%20the%20activation%20of%20the%20cAMP%20response%20element-binding%20protein%20in%20normal%20human%20bronchial%20epithelial%20cells&author=S%20Aggarwal&author=SW%20Kim&author=K%20Cheon&author=FH%20Tabassam&author=JH%20Yoon&volume=17&publication_year=2006&pages=566-575&pmid=16280361&doi=10.1091/mbc.E05-06-0519&)

2. Aminkeng F, Bhavsar AP, Visscher H, Rassekh SR, Li Y, Lee JW, Brunham LR, Caron HN, van Dalen EC, Kremer LC, et al. (2015). A coding variant in RARG confers susceptibility to anthracycline-induced cardiotoxicity in childhood cancer. Nat Genet 47, 1079–1084.  [DOI](https://doi.org/10.1038/ng.3374) | [PMC free article](/articles/PMC4552570/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26237429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20coding%20variant%20in%20RARG%20confers%20susceptibility%20to%20anthracycline-induced%20cardiotoxicity%20in%20childhood%20cancer&author=F%20Aminkeng&author=AP%20Bhavsar&author=H%20Visscher&author=SR%20Rassekh&author=Y%20Li&volume=47&publication_year=2015&pages=1079-1084&pmid=26237429&doi=10.1038/ng.3374&)

3. Aminkeng F, Ross CJ, Rassekh SR, Hwang S, Rieder MJ, Bhavsar AP, Smith A, Sanatani S, Gelmon KA, Bernstein D, et al. (2016). Recommendations for genetic testing to reduce the incidence of anthracycline-induced cardiotoxicity. Br J Clin Pharmacol 82, 683–695.  [DOI](https://doi.org/10.1111/bcp.13008) | [PMC free article](/articles/PMC5338111/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27197003/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Recommendations%20for%20genetic%20testing%20to%20reduce%20the%20incidence%20of%20anthracycline-induced%20cardiotoxicity&author=F%20Aminkeng&author=CJ%20Ross&author=SR%20Rassekh&author=S%20Hwang&author=MJ%20Rieder&volume=82&publication_year=2016&pages=683-695&pmid=27197003&doi=10.1111/bcp.13008&)

4. Aminkeng F, Ross CJD, Rassekh SR, Rieder MJ, Bhavsar AP, Sanatani S, Bernstein D , Hayden MR, Amstutz U, and Carleton BC, (2017). Pharmacogenomic screening for anthracycline-induced cardiotoxicity in childhood cancer. Br J Clin Pharmacol 83, 1143–1145.  [DOI](https://doi.org/10.1111/bcp.13218) | [PMC free article](/articles/PMC5401971/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28317142/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Pharmacogenomic%20screening%20for%20anthracycline-induced%20cardiotoxicity%20in%20childhood%20cancer&author=F%20Aminkeng&author=CJD%20Ross&author=SR%20Rassekh&author=MJ%20Rieder&author=AP%20Bhavsar&volume=83&publication_year=2017&pages=1143-1145&pmid=28317142&doi=10.1111/bcp.13218&)

5. Auton A, Brooks LD, Durbin RM, Garrison EP, Kang HM, Korbel JO, Marchini JL, McCarthy S, McVean GA, and Abecasis GR, (2015). A global reference for human genetic variation. Nature 526, 68–74.  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation&author=A%20Auton&author=LD%20Brooks&author=RM%20Durbin&author=EP%20Garrison&author=HM%20Kang&volume=526&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

6. Berlin V, and Haseltine WA, (1981). Reduction of adriamycin to a semiquinone-free radical by NADPH cytochrome P-450 reductase produces DNA cleavage in a reaction mediated by molecular oxygen. J Biol Chem 256, 4747–4756.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/6262301/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Reduction%20of%20adriamycin%20to%20a%20semiquinone-free%20radical%20by%20NADPH%20cytochrome%20P-450%20reductase%20produces%20DNA%20cleavage%20in%20a%20reaction%20mediated%20by%20molecular%20oxygen&author=V%20Berlin&author=WA%20Haseltine&volume=256&publication_year=1981&pages=4747-4756&pmid=6262301&)

7. Blanco JG, Sun CL, Landier W, Chen L, Esparza-Duran D, Leisenring W, Mays A, Friedman DL, Ginsberg JP, Hudson MM, et al. (2012). Anthracycline-related cardiomyopathy after childhood cancer: role of polymorphisms in carbonyl reductase genes--a report from the Children’s Oncology Group. J Clin Oncol 30, 1415–1421.  [DOI](https://doi.org/10.1200/JCO.2011.34.8987) | [PMC free article](/articles/PMC3383117/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22124095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Anthracycline-related%20cardiomyopathy%20after%20childhood%20cancer:%20role%20of%20polymorphisms%20in%20carbonyl%20reductase%20genes--a%20report%20from%20the%20Children%E2%80%99s%20Oncology%20Group&author=JG%20Blanco&author=CL%20Sun&author=W%20Landier&author=L%20Chen&author=D%20Esparza-Duran&volume=30&publication_year=2012&pages=1415-1421&pmid=22124095&doi=10.1200/JCO.2011.34.8987&)

8. Boyle EA, Li YI, and Pritchard JK, (2017). An Expanded View of Complex Traits: From Polygenic to Omnigenic. Cell 169, 1177–1186.  [DOI](https://doi.org/10.1016/j.cell.2017.05.038) | [PMC free article](/articles/PMC5536862/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28622505/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=An%20Expanded%20View%20of%20Complex%20Traits:%20From%20Polygenic%20to%20Omnigenic&author=EA%20Boyle&author=YI%20Li&author=JK%20Pritchard&volume=169&publication_year=2017&pages=1177-1186&pmid=28622505&doi=10.1016/j.cell.2017.05.038&)

9. Burridge PW, Holmstrom A, and Wu JC, (2015). Chemically Defined Culture and Cardiomyocyte Differentiation of Human Pluripotent Stem Cells. Curr Protoc Hum Genet 87, 2123 21-15.  [DOI](https://doi.org/10.1002/0471142905.hg2103s87) | [PMC free article](/articles/PMC4597313/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26439715/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Protoc%20Hum%20Genet&title=Chemically%20Defined%20Culture%20and%20Cardiomyocyte%20Differentiation%20of%20Human%20Pluripotent%20Stem%20Cells&author=PW%20Burridge&author=A%20Holmstrom&author=JC%20Wu&volume=87&publication_year=2015&pages=2123&pmid=26439715&doi=10.1002/0471142905.hg2103s87&)

10. Burridge PW, Li YF, Matsa E, Wu H, Ong SG, Sharma A, Holmstrom A, Chang AC, Coronado MJ, Ebert AD, et al. (2016). Human induced pluripotent stem cell-derived cardiomyocytes recapitulate the predilection of breast cancer patients to doxorubicin-induced cardiotoxicity. Nat Med 22, 547–556.  [DOI](https://doi.org/10.1038/nm.4087) | [PMC free article](/articles/PMC5086256/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27089514/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=Human%20induced%20pluripotent%20stem%20cell-derived%20cardiomyocytes%20recapitulate%20the%20predilection%20of%20breast%20cancer%20patients%20to%20doxorubicin-induced%20cardiotoxicity&author=PW%20Burridge&author=YF%20Li&author=E%20Matsa&author=H%20Wu&author=SG%20Ong&volume=22&publication_year=2016&pages=547-556&pmid=27089514&doi=10.1038/nm.4087&)

11. Burridge PW, Matsa E, Shukla P, Lin ZC, Churko JM, Ebert AD, Lan F, Diecke S, Huber B, Mordwinkin NM, et al. (2014). Chemically defined generation of human cardiomyocytes. Nat Methods 11, 855–860.  [DOI](https://doi.org/10.1038/nmeth.2999) | [PMC free article](/articles/PMC4169698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24930130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=Chemically%20defined%20generation%20of%20human%20cardiomyocytes&author=PW%20Burridge&author=E%20Matsa&author=P%20Shukla&author=ZC%20Lin&author=JM%20Churko&volume=11&publication_year=2014&pages=855-860&pmid=24930130&doi=10.1038/nmeth.2999&)

12. Canon E, Cosgaya JM, Scsucova S, and Aranda A, (2004). Rapid effects of retinoic acid on CREB and ERK phosphorylation in neuronal cells. Mol Biol Cell 15, 5583–5592.  [DOI](https://doi.org/10.1091/mbc.E04-05-0439) | [PMC free article](/articles/PMC532036/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15371543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Cell&title=Rapid%20effects%20of%20retinoic%20acid%20on%20CREB%20and%20ERK%20phosphorylation%20in%20neuronal%20cells&author=E%20Canon&author=JM%20Cosgaya&author=S%20Scsucova&author=A%20Aranda&volume=15&publication_year=2004&pages=5583-5592&pmid=15371543&doi=10.1091/mbc.E04-05-0439&)

13. Chen B, Gilbert LA, Cimini BA, Schnitzbauer J, Zhang W, Li GW, Park J, Blackburn EH, Weissman JS, Qi LS, and Huang B, (2013). Dynamic imaging of genomic loci in living human cells by an optimized CRISPR/Cas system. Cell 155, 1479–1491.  [DOI](https://doi.org/10.1016/j.cell.2013.12.001) | [PMC free article](/articles/PMC3918502/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24360272/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Dynamic%20imaging%20of%20genomic%20loci%20in%20living%20human%20cells%20by%20an%20optimized%20CRISPR/Cas%20system&author=B%20Chen&author=LA%20Gilbert&author=BA%20Cimini&author=J%20Schnitzbauer&author=W%20Zhang&volume=155&publication_year=2013&pages=1479-1491&pmid=24360272&doi=10.1016/j.cell.2013.12.001&)

14. Chen G, Gulbranson DR, Hou Z, Bolin JM, Ruotti V, Probasco MD, Smuga-Otto K, Howden SE, Diol NR, Propson NE, et al. (2011). Chemically defined conditions for human iPSC derivation and culture. Nat Methods 8, 424–429.  [DOI](https://doi.org/10.1038/nmeth.1593) | [PMC free article](/articles/PMC3084903/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21478862/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=Chemically%20defined%20conditions%20for%20human%20iPSC%20derivation%20and%20culture&author=G%20Chen&author=DR%20Gulbranson&author=Z%20Hou&author=JM%20Bolin&author=V%20Ruotti&volume=8&publication_year=2011&pages=424-429&pmid=21478862&doi=10.1038/nmeth.1593&)

15. Cheng H, Kari G, Dicker AP, Rodeck U, Koch WJ, and Force T, (2011). A novel preclinical strategy for identifying cardiotoxic kinase inhibitors and mechanisms of cardiotoxicity. Circ Res 109, 1401–1409.  [DOI](https://doi.org/10.1161/CIRCRESAHA.111.255695) | [PMC free article](/articles/PMC3908774/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21998323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Res&title=A%20novel%20preclinical%20strategy%20for%20identifying%20cardiotoxic%20kinase%20inhibitors%20and%20mechanisms%20of%20cardiotoxicity&author=H%20Cheng&author=G%20Kari&author=AP%20Dicker&author=U%20Rodeck&author=WJ%20Koch&volume=109&publication_year=2011&pages=1401-1409&pmid=21998323&doi=10.1161/CIRCRESAHA.111.255695&)

16. Chou BK, Gu H, Gao Y, Dowey SN, Wang Y, Shi J, Li Y, Ye Z, Cheng T, and Cheng L, (2015). A facile method to establish human induced pluripotent stem cells from adult blood cells under feeder-free and xeno-free culture conditions: a clinically compliant approach. Stem Cells Transl Med 4, 320–332.  [DOI](https://doi.org/10.5966/sctm.2014-0214) | [PMC free article](/articles/PMC4367508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25742692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stem%20Cells%20Transl%20Med&title=A%20facile%20method%20to%20establish%20human%20induced%20pluripotent%20stem%20cells%20from%20adult%20blood%20cells%20under%20feeder-free%20and%20xeno-free%20culture%20conditions:%20a%20clinically%20compliant%20approach&author=BK%20Chou&author=H%20Gu&author=Y%20Gao&author=SN%20Dowey&author=Y%20Wang&volume=4&publication_year=2015&pages=320-332&pmid=25742692&doi=10.5966/sctm.2014-0214&)

17. Delacroix L, Moutier E, Altobelli G, Legras S, Poch O, Choukrallah MA, Bertin I, Jost B, and Davidson I, (2010). Cell-specific interaction of retinoic acid receptors with target genes in mouse embryonic fibroblasts and embryonic stem cells. Mol Cell Biol 30, 231–244.  [DOI](https://doi.org/10.1128/MCB.00756-09) | [PMC free article](/articles/PMC2798310/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19884340/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Biol&title=Cell-specific%20interaction%20of%20retinoic%20acid%20receptors%20with%20target%20genes%20in%20mouse%20embryonic%20fibroblasts%20and%20embryonic%20stem%20cells&author=L%20Delacroix&author=E%20Moutier&author=G%20Altobelli&author=S%20Legras&author=O%20Poch&volume=30&publication_year=2010&pages=231-244&pmid=19884340&doi=10.1128/MCB.00756-09&)

18. Fajardo G, Zhao M, Berry G, Wong LJ, Mochly-Rosen D, and Bernstein D, (2011). beta2-adrenergic receptors mediate cardioprotection through crosstalk with mitochondrial cell death pathways. J Mol Cell Cardiol 51, 781–789.  [DOI](https://doi.org/10.1016/j.yjmcc.2011.06.019) | [PMC free article](/articles/PMC3184305/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21756913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Cell%20Cardiol&title=beta2-adrenergic%20receptors%20mediate%20cardioprotection%20through%20crosstalk%20with%20mitochondrial%20cell%20death%20pathways&author=G%20Fajardo&author=M%20Zhao&author=G%20Berry&author=LJ%20Wong&author=D%20Mochly-Rosen&volume=51&publication_year=2011&pages=781-789&pmid=21756913&doi=10.1016/j.yjmcc.2011.06.019&)

19. Fryer RM, Pratt PF, Hsu AK, and Gross GJ, (2001). Differential activation of extracellular signal regulated kinase isoforms in preconditioning and opioid-induced cardioprotection. J Pharmacol Exp Ther 296, 642–649.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11160653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Differential%20activation%20of%20extracellular%20signal%20regulated%20kinase%20isoforms%20in%20preconditioning%20and%20opioid-induced%20cardioprotection&author=RM%20Fryer&author=PF%20Pratt&author=AK%20Hsu&author=GJ%20Gross&volume=296&publication_year=2001&pages=642-649&pmid=11160653&)

20. Fusaki N, Ban H, Nishiyama A, Saeki K, and Hasegawa M, (2009). Efficient induction of transgene-free human pluripotent stem cells using a vector based on Sendai virus, an RNA virus that does not integrate into the host genome. Proc Jpn Acad Ser B Phys Biol Sci 85, 348–362.  [DOI](https://doi.org/10.2183/pjab.85.348) | [PMC free article](/articles/PMC3621571/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19838014/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Jpn%20Acad%20Ser%20B%20Phys%20Biol%20Sci&title=Efficient%20induction%20of%20transgene-free%20human%20pluripotent%20stem%20cells%20using%20a%20vector%20based%20on%20Sendai%20virus,%20an%20RNA%20virus%20that%20does%20not%20integrate%20into%20the%20host%20genome&author=N%20Fusaki&author=H%20Ban&author=A%20Nishiyama&author=K%20Saeki&author=M%20Hasegawa&volume=85&publication_year=2009&pages=348-362&pmid=19838014&doi=10.2183/pjab.85.348&)

21. Granger CB, (2006). Prediction and prevention of chemotherapy-induced cardiomyopathy: can it be done? Circulation 114, 2432–2433.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.106.666248) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17146002/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Prediction%20and%20prevention%20of%20chemotherapy-induced%20cardiomyopathy:%20can%20it%20be%20done?&author=CB%20Granger&volume=114&publication_year=2006&pages=2432-2433&pmid=17146002&doi=10.1161/CIRCULATIONAHA.106.666248&)

22. Hasan S, Dinh K, Lombardo F, and Kark J, (2004). Doxorubicin cardiotoxicity in African Americans. J Natl Med Assoc 96, 196–199.  [PMC free article](/articles/PMC2594938/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14977278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Med%20Assoc&title=Doxorubicin%20cardiotoxicity%20in%20African%20Americans&author=S%20Hasan&author=K%20Dinh&author=F%20Lombardo&author=J%20Kark&volume=96&publication_year=2004&pages=196-199&pmid=14977278&)

23. Hudson MM, Ness KK, Gurney JG, Mulrooney DA, Chemaitilly W, Krull KR, Green DM, Armstrong GT, Nottage KA, Jones KE, et al. (2013). Clinical ascertainment of health outcomes among adults treated for childhood cancer. JAMA 309, 2371–2381.  [DOI](https://doi.org/10.1001/jama.2013.6296) | [PMC free article](/articles/PMC3771083/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23757085/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Clinical%20ascertainment%20of%20health%20outcomes%20among%20adults%20treated%20for%20childhood%20cancer&author=MM%20Hudson&author=KK%20Ness&author=JG%20Gurney&author=DA%20Mulrooney&author=W%20Chemaitilly&volume=309&publication_year=2013&pages=2371-2381&pmid=23757085&doi=10.1001/jama.2013.6296&)

24. Ichikawa Y, Ghanefar M, Bayeva M, Wu R, Khechaduri A, Naga Prasad SV, Mutharasan RK, Naik TJ, and Ardehali H, (2014). Cardiotoxicity of doxorubicin is mediated through mitochondrial iron accumulation. J Clin Invest 124, 617–630.  [DOI](https://doi.org/10.1172/JCI72931) | [PMC free article](/articles/PMC3904631/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24382354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Cardiotoxicity%20of%20doxorubicin%20is%20mediated%20through%20mitochondrial%20iron%20accumulation&author=Y%20Ichikawa&author=M%20Ghanefar&author=M%20Bayeva&author=R%20Wu&author=A%20Khechaduri&volume=124&publication_year=2014&pages=617-630&pmid=24382354&doi=10.1172/JCI72931&)

25. Ichiki T, (2006). Role of cAMP response element binding protein in cardiovascular remodeling: good, bad, or both? Arterioscler Thromb Vasc Biol 26, 449–455.  [DOI](https://doi.org/10.1161/01.ATV.0000196747.79349.d1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16293792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Role%20of%20cAMP%20response%20element%20binding%20protein%20in%20cardiovascular%20remodeling:%20good,%20bad,%20or%20both?&author=T%20Ichiki&volume=26&publication_year=2006&pages=449-455&pmid=16293792&doi=10.1161/01.ATV.0000196747.79349.d1&)

26. Iulianella A, and Lohnes D, (2002). Chimeric analysis of retinoic acid receptor function during cardiac looping. Dev Biol 247, 62–75.  [DOI](https://doi.org/10.1006/dbio.2002.0685) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12074552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dev%20Biol&title=Chimeric%20analysis%20of%20retinoic%20acid%20receptor%20function%20during%20cardiac%20looping&author=A%20Iulianella&author=D%20Lohnes&volume=247&publication_year=2002&pages=62-75&pmid=12074552&doi=10.1006/dbio.2002.0685&)

27. Izumi M, Masaki M, Hiramoto Y, Sugiyama S, Kuroda T, Terai K, Hori M, Kawase I, and Hirota H, (2006). Cross-talk between bone morphogenetic protein 2 and leukemia inhibitory factor through ERK 1/2 and Smad1 in protection against doxorubicin-induced injury of cardiomyocytes. J Mol Cell Cardiol 40, 224–233.  [DOI](https://doi.org/10.1016/j.yjmcc.2005.11.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16427075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Cell%20Cardiol&title=Cross-talk%20between%20bone%20morphogenetic%20protein%202%20and%20leukemia%20inhibitory%20factor%20through%20ERK%201/2%20and%20Smad1%20in%20protection%20against%20doxorubicin-induced%20injury%20of%20cardiomyocytes&author=M%20Izumi&author=M%20Masaki&author=Y%20Hiramoto&author=S%20Sugiyama&author=T%20Kuroda&volume=40&publication_year=2006&pages=224-233&pmid=16427075&doi=10.1016/j.yjmcc.2005.11.007&)

28. Krischer JP, Epstein S, Cuthbertson DD, Goorin AM, Epstein ML, and Lipshultz SE, (1997). Clinical cardiotoxicity following anthracycline treatment for childhood cancer: the Pediatric Oncology Group experience. J Clin Oncol 15, 1544–1552.  [DOI](https://doi.org/10.1200/JCO.1997.15.4.1544) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9193351/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Clinical%20cardiotoxicity%20following%20anthracycline%20treatment%20for%20childhood%20cancer:%20the%20Pediatric%20Oncology%20Group%20experience&author=JP%20Krischer&author=S%20Epstein&author=DD%20Cuthbertson&author=AM%20Goorin&author=ML%20Epstein&volume=15&publication_year=1997&pages=1544-1552&pmid=9193351&doi=10.1200/JCO.1997.15.4.1544&)

29. Lalevee S, Anno YN, Chatagnon A, Samarut E, Poch O, Laudet V, Benoit G, Lecompte O, and Rochette-Egly C, (2011). Genome-wide in silico identification of new conserved and functional retinoic acid receptor response elements (direct repeats separated by 5 bp). J. Biol. Chem 286, 33322–33334.  [DOI](https://doi.org/10.1074/jbc.M111.263681) | [PMC free article](/articles/PMC3190930/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21803772/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem&title=Genome-wide%20in%20silico%20identification%20of%20new%20conserved%20and%20functional%20retinoic%20acid%20receptor%20response%20elements%20(direct%20repeats%20separated%20by%205%20bp)&author=S%20Lalevee&author=YN%20Anno&author=A%20Chatagnon&author=E%20Samarut&author=O%20Poch&volume=286&publication_year=2011&pages=33322-33334&pmid=21803772&doi=10.1074/jbc.M111.263681&)

30. Lefrak EA, Pitha J, Rosenheim S, and Gottlieb JA, (1973). A clinicopathologic analysis of adriamycin cardiotoxicity. Cancer 32, 302–314.  [DOI](https://doi.org/10.1002/1097-0142(197308)32:2<302::aid-cncr2820320205>3.0.co;2-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4353012/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=A%20clinicopathologic%20analysis%20of%20adriamycin%20cardiotoxicity&author=EA%20Lefrak&author=J%20Pitha&author=S%20Rosenheim&author=JA%20Gottlieb&volume=32&publication_year=1973&pages=302-314&pmid=4353012&doi=10.1002/1097-0142(197308)32:2<302::aid-cncr2820320205>3.0.co;2-2&)

31. Liao Y, Smyth GK, and Shi W, (2013). The Subread aligner: fast, accurate and scalable read mapping by seed-and-vote. Nucleic Acids Res 41, e108.  [DOI](https://doi.org/10.1093/nar/gkt214) | [PMC free article](/articles/PMC3664803/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23558742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20Subread%20aligner:%20fast,%20accurate%20and%20scalable%20read%20mapping%20by%20seed-and-vote&author=Y%20Liao&author=GK%20Smyth&author=W%20Shi&volume=41&publication_year=2013&pages=e108&pmid=23558742&doi=10.1093/nar/gkt214&)

32. Lim CC, Zuppinger C, Guo X, Kuster GM, Helmes M, Eppenberger HM, Suter TM, Liao R, and Sawyer DB, (2004). Anthracyclines induce calpain-dependent titin proteolysis and necrosis in cardiomyocytes. J Biol Chem 279, 8290–8299.  [DOI](https://doi.org/10.1074/jbc.M308033200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14676206/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Anthracyclines%20induce%20calpain-dependent%20titin%20proteolysis%20and%20necrosis%20in%20cardiomyocytes&author=CC%20Lim&author=C%20Zuppinger&author=X%20Guo&author=GM%20Kuster&author=M%20Helmes&volume=279&publication_year=2004&pages=8290-8299&pmid=14676206&doi=10.1074/jbc.M308033200&)

33. Love MI, Huber W, and Anders S, (2014). Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550.  [DOI](https://doi.org/10.1186/s13059-014-0550-8) | [PMC free article](/articles/PMC4302049/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25516281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&author=MI%20Love&author=W%20Huber&author=S%20Anders&volume=15&publication_year=2014&pages=550&pmid=25516281&doi=10.1186/s13059-014-0550-8&)

34. Magdy T, Burmeister BT, and Burridge PW, (2016). Validating the pharmacogenomics of chemotherapy-induced cardiotoxicity: What is missing? Pharmacol Ther 168, 113–125.  [DOI](https://doi.org/10.1016/j.pharmthera.2016.09.009) | [PMC free article](/articles/PMC5140687/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27609196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Ther&title=Validating%20the%20pharmacogenomics%20of%20chemotherapy-induced%20cardiotoxicity:%20What%20is%20missing?&author=T%20Magdy&author=BT%20Burmeister&author=PW%20Burridge&volume=168&publication_year=2016&pages=113-125&pmid=27609196&doi=10.1016/j.pharmthera.2016.09.009&)

35. Minotti G, Menna P, Salvatorelli E, Cairo G, and Gianni L, (2004). Anthracyclines: molecular advances and pharmacologic developments in antitumor activity and cardiotoxicity. Pharmacol Rev 56, 185–229.  [DOI](https://doi.org/10.1124/pr.56.2.6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15169927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=Anthracyclines:%20molecular%20advances%20and%20pharmacologic%20developments%20in%20antitumor%20activity%20and%20cardiotoxicity&author=G%20Minotti&author=P%20Menna&author=E%20Salvatorelli&author=G%20Cairo&author=L%20Gianni&volume=56&publication_year=2004&pages=185-229&pmid=15169927&doi=10.1124/pr.56.2.6&)

36. Mulrooney DA, Yeazel MW, Kawashima T, Mertens AC, Mitby P, Stovall M, Donaldson SS, Green DM, Sklar CA, Robison LL, and Leisenring WM, (2009). Cardiac outcomes in a cohort of adult survivors of childhood and adolescent cancer: retrospective analysis of the Childhood Cancer Survivor Study cohort. BMJ 339, b4606.  [DOI](https://doi.org/10.1136/bmj.b4606) | [PMC free article](/articles/PMC3266843/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19996459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Cardiac%20outcomes%20in%20a%20cohort%20of%20adult%20survivors%20of%20childhood%20and%20adolescent%20cancer:%20retrospective%20analysis%20of%20the%20Childhood%20Cancer%20Survivor%20Study%20cohort&author=DA%20Mulrooney&author=MW%20Yeazel&author=T%20Kawashima&author=AC%20Mertens&author=P%20Mitby&volume=339&publication_year=2009&pages=b4606&pmid=19996459&doi=10.1136/bmj.b4606&)

37. Musunuru K, Bernstein D, Cole FS, Khokha MK, Lee FS, Lin S, McDonald TV, Moskowitz IP, Quertermous T, Sankaran VG, et al. (2018). Functional Assays to Screen and Dissect Genomic Hits: Doubling Down on the National Investment in Genomic Research. Circ Genom Precis Med 11, e002178.  [DOI](https://doi.org/10.1161/CIRCGEN.118.002178) | [PMC free article](/articles/PMC5901889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29654098/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ%20Genom%20Precis%20Med&title=Functional%20Assays%20to%20Screen%20and%20Dissect%20Genomic%20Hits:%20Doubling%20Down%20on%20the%20National%20Investment%20in%20Genomic%20Research&author=K%20Musunuru&author=D%20Bernstein&author=FS%20Cole&author=MK%20Khokha&author=FS%20Lee&volume=11&publication_year=2018&pages=e002178&pmid=29654098&doi=10.1161/CIRCGEN.118.002178&)

38. Oceguera-Yanez F, Kim SI, Matsumoto T, Tan GW, Xiang L, Hatani T, Kondo T, Ikeya M, Yoshida Y, Inoue H, and Woltjen K, (2016). Engineering the AAVS1 locus for consistent and scalable transgene expression in human iPSCs and their differentiated derivatives. Methods 101, 43–55.  [DOI](https://doi.org/10.1016/j.ymeth.2015.12.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26707206/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods&title=Engineering%20the%20AAVS1%20locus%20for%20consistent%20and%20scalable%20transgene%20expression%20in%20human%20iPSCs%20and%20their%20differentiated%20derivatives&author=F%20Oceguera-Yanez&author=SI%20Kim&author=T%20Matsumoto&author=GW%20Tan&author=L%20Xiang&volume=101&publication_year=2016&pages=43-55&pmid=26707206&doi=10.1016/j.ymeth.2015.12.012&)

39. Ochoa WF, Corbalan-Garcia S, Eritja R, Rodriguez-Alfaro JA, Gomez-Fernandez JC, Fita I, and Verdaguer N, (2002). Additional binding sites for anionic phospholipids and calcium ions in the crystal structures of complexes of the C2 domain of protein kinase calpha. J Mol Biol 320, 277–291.  [DOI](https://doi.org/10.1016/S0022-2836(02)00464-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12079385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Biol&title=Additional%20binding%20sites%20for%20anionic%20phospholipids%20and%20calcium%20ions%20in%20the%20crystal%20structures%20of%20complexes%20of%20the%20C2%20domain%20of%20protein%20kinase%20calpha&author=WF%20Ochoa&author=S%20Corbalan-Garcia&author=R%20Eritja&author=JA%20Rodriguez-Alfaro&author=JC%20Gomez-Fernandez&volume=320&publication_year=2002&pages=277-291&pmid=12079385&doi=10.1016/S0022-2836(02)00464-3&)

40. Oeffinger KC, Mertens AC, Sklar CA, Kawashima T, Hudson MM, Meadows AT, Friedman DL, Marina N, Hobbie W, Kadan-Lottick NS, et al. (2006). Chronic health conditions in adult survivors of childhood cancer. N Engl J Med 355, 1572–1582.  [DOI](https://doi.org/10.1056/NEJMsa060185) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17035650/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Chronic%20health%20conditions%20in%20adult%20survivors%20of%20childhood%20cancer&author=KC%20Oeffinger&author=AC%20Mertens&author=CA%20Sklar&author=T%20Kawashima&author=MM%20Hudson&volume=355&publication_year=2006&pages=1572-1582&pmid=17035650&doi=10.1056/NEJMsa060185&)

41. Peterson SE, and Loring JF, (2014). Genomic instability in pluripotent stem cells: implications for clinical applications. J Biol Chem 289, 4578–4584.  [DOI](https://doi.org/10.1074/jbc.R113.516419) | [PMC free article](/articles/PMC3931019/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24362040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Genomic%20instability%20in%20pluripotent%20stem%20cells:%20implications%20for%20clinical%20applications&author=SE%20Peterson&author=JF%20Loring&volume=289&publication_year=2014&pages=4578-4584&pmid=24362040&doi=10.1074/jbc.R113.516419&)

42. Ran FA, Hsu PD, Wright J, Agarwala V, Scott DA, and Zhang F, (2013). Genome engineering using the CRISPR-Cas9 system. Nat Protoc 8, 2281–2308.  [DOI](https://doi.org/10.1038/nprot.2013.143) | [PMC free article](/articles/PMC3969860/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24157548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Genome%20engineering%20using%20the%20CRISPR-Cas9%20system&author=FA%20Ran&author=PD%20Hsu&author=J%20Wright&author=V%20Agarwala&author=DA%20Scott&volume=8&publication_year=2013&pages=2281-2308&pmid=24157548&doi=10.1038/nprot.2013.143&)

43. Schmittgen TD, and Livak KJ, (2008). Analyzing real-time PCR data by the comparative C(T) method. Nat Protoc 3, 1101–1108.  [DOI](https://doi.org/10.1038/nprot.2008.73) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18546601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Analyzing%20real-time%20PCR%20data%20by%20the%20comparative%20C(T)%20method&author=TD%20Schmittgen&author=KJ%20Livak&volume=3&publication_year=2008&pages=1101-1108&pmid=18546601&doi=10.1038/nprot.2008.73&)

44. Simoncikova P, Ravingerova T, and Barancik M, (2008). The effect of chronic doxorubicin treatment on mitogen-activated protein kinases and heat stress proteins in rat hearts. Physiol Res 57 Suppl 2, S97–S102.  [DOI](https://doi.org/10.33549/physiolres.931558) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18373386/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiol%20Res&title=The%20effect%20of%20chronic%20doxorubicin%20treatment%20on%20mitogen-activated%20protein%20kinases%20and%20heat%20stress%20proteins%20in%20rat%20hearts&author=P%20Simoncikova&author=T%20Ravingerova&author=M%20Barancik&volume=57&issue=Suppl%202&publication_year=2008&pages=S97-S102&pmid=18373386&doi=10.33549/physiolres.931558&)

45. Singal PK, and Iliskovic N, (1998). Doxorubicin-induced cardiomyopathy. N Engl J Med 339, 900–905.  [DOI](https://doi.org/10.1056/NEJM199809243391307) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9744975/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Doxorubicin-induced%20cardiomyopathy&author=PK%20Singal&author=N%20Iliskovic&volume=339&publication_year=1998&pages=900-905&pmid=9744975&doi=10.1056/NEJM199809243391307&)

46. Su HF, Samsamshariat A, Fu J, Shan YX, Chen YH, Piomelli D, and Wang PH, (2006). Oleylethanolamide activates Ras-Erk pathway and improves myocardial function in doxorubicin-induced heart failure. Endocrinology 147, 827–834.  [DOI](https://doi.org/10.1210/en.2005-1098) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16269455/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocrinology&title=Oleylethanolamide%20activates%20Ras-Erk%20pathway%20and%20improves%20myocardial%20function%20in%20doxorubicin-induced%20heart%20failure&author=HF%20Su&author=A%20Samsamshariat&author=J%20Fu&author=YX%20Shan&author=YH%20Chen&volume=147&publication_year=2006&pages=827-834&pmid=16269455&doi=10.1210/en.2005-1098&)

47. Swain SM, Whaley FS, and Ewer MS, (2003). Congestive heart failure in patients treated with doxorubicin: a retrospective analysis of three trials. Cancer 97, 2869–2879.  [DOI](https://doi.org/10.1002/cncr.11407) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12767102/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Congestive%20heart%20failure%20in%20patients%20treated%20with%20doxorubicin:%20a%20retrospective%20analysis%20of%20three%20trials&author=SM%20Swain&author=FS%20Whaley&author=MS%20Ewer&volume=97&publication_year=2003&pages=2869-2879&pmid=12767102&doi=10.1002/cncr.11407&)

48. van Dalen EC, Raphael MF, Caron HN, and Kremer LC, (2014). Treatment including anthracyclines versus treatment not including anthracyclines for childhood cancer. Cochrane Database Syst Rev, CD006647.  [DOI](https://doi.org/10.1002/14651858.CD006647.pub4) | [PMC free article](/articles/PMC11231984/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25188452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cochrane%20Database%20Syst%20Rev&title=Treatment%20including%20anthracyclines%20versus%20treatment%20not%20including%20anthracyclines%20for%20childhood%20cancer&author=EC%20van%20Dalen&author=MF%20Raphael&author=HN%20Caron&author=LC%20Kremer&publication_year=2014&pages=CD006647&pmid=25188452&doi=10.1002/14651858.CD006647.pub4&)

49. van der Pal HJ, van Dalen EC, van Delden E, van Dijk IW, Kok WE, Geskus RB, Sieswerda E, Oldenburger F, Koning CC, van Leeuwen FE, et al. (2012). High risk of symptomatic cardiac events in childhood cancer survivors. J Clin Oncol 30, 1429–1437.  [DOI](https://doi.org/10.1200/JCO.2010.33.4730) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22473161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=High%20risk%20of%20symptomatic%20cardiac%20events%20in%20childhood%20cancer%20survivors&author=HJ%20van%20der%20Pal&author=EC%20van%20Dalen&author=E%20van%20Delden&author=IW%20van%20Dijk&author=WE%20Kok&volume=30&publication_year=2012&pages=1429-1437&pmid=22473161&doi=10.1200/JCO.2010.33.4730&)

50. Volonte D, McTiernan CF, Drab M, Kasper M, and Galbiati F, (2008). Caveolin-1 and caveolin-3 form heterooligomeric complexes in atrial cardiac myocytes that are required for doxorubicin-induced apoptosis. Am. J. Physiol. Heart Circ. Physiol 294, H392–401.  [DOI](https://doi.org/10.1152/ajpheart.01039.2007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17982011/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Physiol.%20Heart%20Circ.%20Physiol&title=Caveolin-1%20and%20caveolin-3%20form%20heterooligomeric%20complexes%20in%20atrial%20cardiac%20myocytes%20that%20are%20required%20for%20doxorubicin-induced%20apoptosis&author=D%20Volonte&author=CF%20McTiernan&author=M%20Drab&author=M%20Kasper&author=F%20Galbiati&volume=294&publication_year=2008&pages=H392-401&pmid=17982011&doi=10.1152/ajpheart.01039.2007&)

51. Von Hoff DD, Layard MW, Basa P, Davis HL Jr., Von Hoff AL, Rozencweig M, and Muggia FM, (1979). Risk factors for doxorubicin-induced congestive heart failure. Ann Intern Med 97, 710–717.  [DOI](https://doi.org/10.7326/0003-4819-91-5-710) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/496103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Risk%20factors%20for%20doxorubicin-induced%20congestive%20heart%20failure&author=DD%20Von%20Hoff&author=MW%20Layard&author=P%20Basa&author=HL%20Davis&author=AL%20Von%20Hoff&volume=97&publication_year=1979&pages=710-717&pmid=496103&doi=10.7326/0003-4819-91-5-710&)

52. Wojnowski L, Kulle B, Schirmer M, Schluter G, Schmidt A, Rosenberger A, Vonhof S, Bickeboller H, Toliat MR, Suk EK, et al. (2005). NAD(P)H oxidase and multidrug resistance protein genetic polymorphisms are associated with doxorubicin-induced cardiotoxicity. Circulation 112, 3754–3762.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.105.576850) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16330681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=NAD(P)H%20oxidase%20and%20multidrug%20resistance%20protein%20genetic%20polymorphisms%20are%20associated%20with%20doxorubicin-induced%20cardiotoxicity&author=L%20Wojnowski&author=B%20Kulle&author=M%20Schirmer&author=G%20Schluter&author=A%20Schmidt&volume=112&publication_year=2005&pages=3754-3762&pmid=16330681&doi=10.1161/CIRCULATIONAHA.105.576850&)

53. Xiang P, Deng HY, Li K, Huang GY, Chen Y, Tu L, Ng PC, Pong NH, Zhao H, Zhang L, and Sung RY, (2009). Dexrazoxane protects against doxorubicin-induced cardiomyopathy: upregulation of Akt and Erk phosphorylation in a rat model. Cancer Chemother Pharmacol 63, 343–349.  [DOI](https://doi.org/10.1007/s00280-008-0744-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18379782/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Dexrazoxane%20protects%20against%20doxorubicin-induced%20cardiomyopathy:%20upregulation%20of%20Akt%20and%20Erk%20phosphorylation%20in%20a%20rat%20model&author=P%20Xiang&author=HY%20Deng&author=K%20Li&author=GY%20Huang&author=Y%20Chen&volume=63&publication_year=2009&pages=343-349&pmid=18379782&doi=10.1007/s00280-008-0744-4&)

54. Yang F, Teves SS, Kemp CJ, and Henikoff S, (2014). Doxorubicin, DNA torsion, and chromatin dynamics. Biochim Biophys Acta 1845, 84–89.  [DOI](https://doi.org/10.1016/j.bbcan.2013.12.002) | [PMC free article](/articles/PMC3927826/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24361676/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Doxorubicin,%20DNA%20torsion,%20and%20chromatin%20dynamics&author=F%20Yang&author=SS%20Teves&author=CJ%20Kemp&author=S%20Henikoff&volume=1845&publication_year=2014&pages=84-89&pmid=24361676&doi=10.1016/j.bbcan.2013.12.002&)

55. Yang L, Luo C, Chen C, Wang X, Shi W, and Liu J, (2016). All-trans retinoic acid protects against doxorubicin-induced cardiotoxicity by activating the ERK2 signalling pathway. Br J Pharmacol 173, 357–371.  [DOI](https://doi.org/10.1111/bph.13377) | [PMC free article](/articles/PMC5341234/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26507774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=All-trans%20retinoic%20acid%20protects%20against%20doxorubicin-induced%20cardiotoxicity%20by%20activating%20the%20ERK2%20signalling%20pathway&author=L%20Yang&author=C%20Luo&author=C%20Chen&author=X%20Wang&author=W%20Shi&volume=173&publication_year=2016&pages=357-371&pmid=26507774&doi=10.1111/bph.13377&)

56. Zhang S, Liu X, Bawa-Khalfe T, Lu LS, Lyu YL, Liu LF, and Yeh ET, (2012). Identification of the molecular basis of doxorubicin-induced cardiotoxicity. Nat Med 18, 1639–1642.  [DOI](https://doi.org/10.1038/nm.2919) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23104132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=Identification%20of%20the%20molecular%20basis%20of%20doxorubicin-induced%20cardiotoxicity&author=S%20Zhang&author=X%20Liu&author=T%20Bawa-Khalfe&author=LS%20Lu&author=YL%20Lyu&volume=18&publication_year=2012&pages=1639-1642&pmid=23104132&doi=10.1038/nm.2919&)
