# Strong Association of Angiotensin Converting Enzyme-2 Gene Insertion/Deletion Polymorphism with Susceptibility to SARS-CoV-2, Hypertension, Coronary Artery Disease and COVID-19 Disease Mortality

## Metadata
**Authors:** Mohammad Muzaffar Mir, Rashid Mir, Mushabab Ayed Abdullah Alghamdi, Badr Abdulmohsin Alsayed, Javed Iqbal Wani, Muffarah Hamid Alharthi, Abdullah M AL-Shahrani
**Journal:** Journal of Personalized Medicine
**Date:** 2021 Oct 27
**DOI:** [10.3390/jpm11111098](https://doi.org/10.3390/jpm11111098)
**PMID:** 34834450
**PMCID:** PMC8621157
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8621157/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8621157/pdf/jpm-11-01098.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8621157/pdf/jpm-11-01098.pdf)

## Abstract

Background: The ongoing outbreak of SARS-CoV-2 represents a significant challenge to international health. Several reports have highlighted the importance of ACE2 on the pathogenesis of COVID-19. The spike protein of SARS-CoV-2 efficiently binds to the angiotensin-converting enzyme 2 (ACE2) receptors and facilitates virus entry into the host cell. In the present study, we hypothesize that a functional insertion/deletion polymorphism-rs4646994 I/D and rs4240157 T > C in the ACE gene could be associated with SARS-CoV-2 infection and mortality. Methodology: This study included 117 consecutive COVID-19 patients and 150 age matched healthy controls (ACE2-rs4646994 I/D) and 100 age matched healthy controls with ACE2 rs4240157 T > C. We used Mutation specific PCR (MSP) for ACE2-rs4646994 I/D genotyping and amplification refractory mutation system (ARMS-PCR) for ACE2 rs4240157 T > C genotyping. Results: Results indicated that there were significant differences in the genotype distributions of ACE2-rs4646994 I/D polymorphisms (p < 0.030) and ACE2 rs4240157 T > C between COVID-19 patients and controls (p-values < 0.05). Higher frequency of DD genotype (48.71%) and D allele (0.67) was reported in COVID-19 patients than controls. Our results showed that the ACE2-DD genotype was strongly associated with increased COVID-19 severity (OR 2.37 (95%) CI = (1.19–4.70), RR = 1.39 (1.09–1.77), p < 0.013) and also a strong association was seen with ACE2-ID genotype with COVID-19 severity (OR 2.20 (95%) CI = (1.08–4.46), p < 0.020) in the codominant model. In allelic comparison, the D allele was strongly associated with COVID-19 severity (OR 1.58 (95% CI) (1.11–2.27), RR 1.21 (1.05–1.41) p < 0.010). A significant correlation of ACE2-I/D genotypes was reported with Age (p < 0.035), T2D (p < 0.0013), hypertension (p < 0.0031) and coronary artery disease (p < 0.0001). Our results indicated ACE2-DD genotype was strongly associated with increased COVID-19 mortality (OR 8.25 (95%) CI = (2.40 to 28.34), p < 0.008) and also ACE2-DD + DI genotype was strongly associated with increased COVID-19 mortality with OR 4.74 (95%) CI = (1.5214 to 14.7915), p < 0.007. A significant correlation was reported between COVID-19 patients and age matched controls (p < 0.0007). Higher frequency of heterozygosity TC (40%) followed by ACE2-CC genotype (24.78%) was reported among COVID-19 patients. Using multivariate analysis, ACE2–CT genotype was strong associated with SARS-CoV-2 severity with an OR 2.18 (95% CI) (1.92–3.99), p < 0.010 and also ACE2–CC genotype was linked with COVID-19 severity with an OR 2.66 (95% CI) (1.53–4.62), p < 0.005. A significant correlation of ACE2-T > C genotypes was reported with gender (p < 0.04), T2D (p < 0.035). ACE2-CC genotype was strongly associated with increased COVID-19 mortality OR 3.66 (95%) CI = (1.34 to 9.97), p < 0.011 and also ACE2-C allele was associated with COVID-19 mortality OR 2, 01 (1.1761–3.45), p < 0.010. Conclusions: It is concluded that ACE-DD genotype and D allele was strongly associated with increased COVID-19 patient severity. In addition, ACE I/D polymorphism were strongly associated with advanced age, diabetes and ischemic heart disease in COVID-19 patients whereas ACE-II genotype was a protective factor against the development of severe COVID-19. ACE2-DD genotype was strongly associated with increased COVID-19 mortality. Additionally, ACE2–CC and CT genotypes were strongly associated with COVID-19 severity. Therefore, our study might be useful for identifying the susceptible population groups for targeted interventions and for making relevant public health policy decisions.

Keywords: gene polymorphism, infection, SARS-CoV-2, COVID-19, pathogenesis, ACE1-angiotensin-converting enzyme 1, ACE2-angiotensin-converting enzyme 2, severe acute respiratory syndrome coronavirus 2, susceptibility to SARS-CoV-2

## 1. Introduction

The coronavirus disease, designated as COVID-19 and declared as pandemic by the World Health Organization is caused by severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) [[1](#B1-jpm-11-01098),[2](#B2-jpm-11-01098),[3](#B3-jpm-11-01098),[4](#B4-jpm-11-01098)]. COVID-19 has infected around 224 million people across the globe and has resulted in more than 4.86 million deaths till date ([https://www.worldometers.info/coronavirus](https://www.worldometers.info/coronavirus), accessed on 12 October 2021). While as the individuals more than 50 years of age were prone to higher severity and mortality of the disease, the common comorbidities included hypertension, obesity, and diabetes [[5](#B5-jpm-11-01098),[6](#B6-jpm-11-01098)]. COVID-19 has exhibited geographical variations in its spread and mortality and it has caused more morbidity and mortality in Europe and USA [[7](#B7-jpm-11-01098),[8](#B8-jpm-11-01098),[9](#B9-jpm-11-01098)].

SARS-CoV-2 is an enveloped RNA virus with three structural protein components which are described as M (membrane), E (envelop) and S (spike) proteins [[10](#B10-jpm-11-01098),[11](#B11-jpm-11-01098)]. The SARS-COV-2 makes its entry into the host cells through the S proteins. The two subunits of S protein, S1 and S2 are used by the virus to attach to the host cell membrane. The C-terminal domain (CTD) of S1 subunits is vital for the interaction and attachment of S proteins to the protein receptors of the host cell [[4](#B4-jpm-11-01098),[12](#B12-jpm-11-01098),[13](#B13-jpm-11-01098)].

Just like the SARS-CoV, the human angiotensin converting enzyme-2 (ACE2) has emerged as the target for the SARS-CoV-2 [[14](#B14-jpm-11-01098),[15](#B15-jpm-11-01098)]. ACE2 is expressed ubiquitously in different organs of the human body that include heart, lungs, kidneys, intestines and endothelium [[16](#B16-jpm-11-01098),[17](#B17-jpm-11-01098),[18](#B18-jpm-11-01098),[19](#B19-jpm-11-01098)]. ACE2, a type I transmembrane zinc finger glycoprotein, is a mono-carboxypeptidase (with 805 amino acid residues and MW of 100 KD) and converts angiotensin 1 to angiotensin 1-9 [[20](#B20-jpm-11-01098)]. The gene for ACE2, present on chromosome X (Xp22) comprises of 39.98 kb of genomic DNA and has 18 exons [[20](#B20-jpm-11-01098)]. Many ACE2 variants are associated with several common diseases, whose incidence depends on the balance in the renin-angiotensin-aldosterone (RAAS) pathway. Hypertension is associated with rs1514283, rs2074192, rs233575, rs4646155, rs4646176, rs2285666, rs879922, rs2106809, rs4646188, rs4240157, rs4830542, rs2158083, and rs879922 [[21](#B21-jpm-11-01098),[22](#B22-jpm-11-01098)]. The role of ACE2 is very critical since the inhibition of RAAS pathway leads to upregulation of ACE2 which alleviates ARDS and myocarditis symptoms in COVID-19 patients but at the same time increased ACE2 expression may increase the entry of virus into host cells, complication the overall clinical picture [[23](#B23-jpm-11-01098)].

Geographic distribution of ACE I (insertion in intron 16) allele was summarized by Saab et al. and its frequency increases eastwards and westwards from the Middle East [[24](#B24-jpm-11-01098)]. The genetic polymorphism of ACE2 gene is well known world over with racial and ethnic variations [[25](#B25-jpm-11-01098),[26](#B26-jpm-11-01098)] having varying influences on the altered functions of RAAS pathway [[27](#B27-jpm-11-01098)]. The polymorphism of ACE2 gene has been associated with hypertension in Chinese [[28](#B28-jpm-11-01098),[29](#B29-jpm-11-01098)], Canadian [[30](#B30-jpm-11-01098)], Indian [[31](#B31-jpm-11-01098)] and Brazilian populations [[32](#B32-jpm-11-01098)]. The ACE2 polymorphism has been associated with varying degree of disease severity and clinical outcomes of COVID-19, with absence of ACE D/D genotype conferring protection against severe lung injury [[33](#B33-jpm-11-01098)]. The African Americans with high frequency of D allele have depicted higher mortality rates in USA [[34](#B34-jpm-11-01098),[35](#B35-jpm-11-01098)]. Similarly, higher fatalities have been reported from many parts of Europe ([https://ec.europa.eu/eurostat/statistics-explained/index.php/Main_Page](https://ec.europa.eu/eurostat/statistics-explained/index.php/Main_Page), accessed on 12 October 2021). In contrast lower mortality rates have been observed in Asian patients who harbor high frequency of II genotype as compared to DD genotype [[36](#B36-jpm-11-01098)] and ([https://www.worldometers.info/coronavirus/coronavirus-death-rate](https://www.worldometers.info/coronavirus/coronavirus-death-rate), accessed on 12 October 2021). The D allele results in an increased angiotensin-2 level (as a consequence of increased ACE-1 and decreased ACE-2 levels) that leads to increased microvascular permeability resulting in pulmonary edema and other complications [[37](#B37-jpm-11-01098),[38](#B38-jpm-11-01098)].

In the current scenario, the knowledge of ACE2 gene polymorphism in different populations assumes a huge significance. However, although there have been many published reports on the ACE2 gene polymorphism from different parts of the globe, we came across only one report from Saudi Arabia suggesting a link between the ACE2-rs4646994 I/D polymorphism and the obesity [[39](#B39-jpm-11-01098)]. To the best of our knowledge, the current study on the genetic polymorphism ACE2-rs4646994 I/D and ACE2 rs4240157 C > T in the ACE2 gene is the first one from Saudi Arabia and was carried out in the COVID-19 patients with different disease severity/outcomes and healthy controls.

## 2. Methodology

### 2.1. Study Population

This population-based case-control study (prospective cohort study) was conducted on COVID-19 patients and 150 age matched healthy controls. COVID-19 laboratory confirmation was defined as a positive result on RT-qPCR (SARS-CoV-2 RT-qPCR detection kit) of nasal and oropharyngeal swab specimens. We selected COVID-19 positive cases confirmed by a real-time reverse transcription polymerase chain reaction (rRT-PCR) test.

This study was a collaborative study. Specimens from positive cases of COVID-19 were collected from different hospitals in Saudi Arabia (Bisha, Abha and Tabuk). The sample collection was conducted between September 2020 and April 2021.

### 2.2. Ethics Approvals

The ethical approvals were obtained from local RELOC committees of College of Medicine, University of Bisha (Ref. No. UBCOM/H-06-BH-087 (05/25)), University of Tabuk (Ref. No: KAEK2020/4/4) and College of Medicine, King Khalid University, Abha (Ref. No. KKU-H-06-B-091) in accordance with human subjects and complied with the principles of the Helsinki Declaration. Informed consent was obtained before collecting samples from all patients and control subjects.

### 2.3. Data Collection

All the subjects were interviewed using a structured questionnaire regarding epidemiological/demographic data, past history of CAD, T2D, and history of addiction particularly smoking, family history of any significant diseases.

#### 2.3.1. Sample Collection from COVID-19 Patients

About 3 mL of peripheral blood sample was collected in an EDTA or Lavender top tube for all COVID-19 patients. The Blood specimens were immediately stored at −20 °C to −30 °C. The COVID-19 patients who’s who oxygen saturation was less than 60 and needed mechanical ventilatory support in ICU were classified as having severe COVID-19 disease.

#### 2.3.2. Sample Collection from Healthy Controls

All healthy age matched controls specimens have been timed around routine blood draws that are part of routine workout, and hence did not require additional phlebotomy and all participants were provided written informed consent form. About 3 mL peripheral blood was collected in EDTA tubes. The Blood specimens were immediately stored at −20 °C to −30 °C.

### 2.4. Genomic DNA Extraction

Genomic DNA was extracted using DNeasy Blood K (Qiagen, Hilden, Germany) as per the manufacturer’s instructions. The extracted DNA was dissolved in nuclease-free water and stored at 4 °C until use. Quality and integrity of DNA were checked by NanoDrop™ (Thermo Scientific, Waltham, MA, USA). All DNA samples from COVID-19 and controls were screened for purity by measuring optical density (OD) at 260 nm (OD_260_) and 280 nm (OD_280_). The λ260/λ280 ratios ranged from 1.83–1.99 indicating good quality DNA.

### 2.5. Angiotensin-Converting Enzyme Genotyping

The Angiotensin-converting enzyme-2 genotyping (ACE2-rs4646994 I/D) was done by mutation specific PCR and amplification refractory mutation system (ARMS-PCR) PCR was used to detect ACE2 rs4240157 T > C genotyping. The ARMS primers were designed by using the Primer3 software as depicted in [Table 1](#jpm-11-01098-t001).

### Table 1.

| Direction | Primer Sequence | PCR Product | Annealing Temperature |
| --- | --- | --- | --- |
| ARMS-PCR primers of ACE2-rs4646994 I/D gene polymorphism |  |  |  |
| ACE-F | 5′-CTGGAGACCACTCCCATCCTTTCT-3′ | 490-bp (II) | 58 °C |
| ACE-R | 5′-GATGTGGCCATCACATTCGTCAGAT-3′. | 190-bp(DD) |   |
| ARMS-PCR primers of ACE2 rs4240157 T > C gene polymorphism |  |  |  |
| ACE2 Fo | GCTGAGTTCTCAAAATAATGCCATAGAT | 386 bp | 60 °C |
| ACE2 Ro | GCATTTCTTTCCAATCATTAAGAGTTCA |   |   |
| ACE2 FI-T | GCCTCAGAACATTACAGAATCAACCT | 244 bp |   |
| ACE2 RI-C | GAGGGTTGGTAAATAGTGTTCAGTGG | 194 bp |   |

Table 1 Caption: Primers for ACE2 gene polymorphisms.

The PCR was done in a reaction volume of 25 µL containing template DNA (50 ng), Fo—0.25 µL, R0—0.25 µL, RI—0.25 µL, RI—0.25 µL of 25 pmol of each primer and 10 µL from GoTaq^®^ Green Master Mix (cat no M7122) (Promega, Madison, WI, USA). The final volume of 25 µL was adjusted by adding nuclease free double distilled water (ddH_2_O). Finally, 2 µL of DNA was added from each patient. The thermocycling conditions used were: 95 °C for 10 min followed by 40 cycles of 95 °C for 35 s, annealing temperature ACE2-rs4646994 I/D (58 °C) and ACE2 rs4240157 C > T (60 °C) gene polymorphism for 40 s, 72 °C for 43 s followed by the final extension at 72 °C for 10 min.

PCR products were separated on 2% agarose gel stained with 2 µL of sybre safe stain (Thermo Scientific, Waltham, MA, USA) and visualized on a UV trans illuminator from Bio-Rad (Hercules, CA, USA). The PCR products were electrophoresed on a 2% agarose gel stained with ethidium bromide to visualize three patterns: I/I (490-bp fragment), D/D (190-bp fragment), and I/D (both 490- and 190-bp fragments) as depicted in [Figure 1](#jpm-11-01098-f001).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c687/8621157/2230e46c2581/jpm-11-01098-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8621157_jpm-11-01098-g001.jpg)

Detection of Angiotensin I-converting enzymes (ACE2) insertion/deletion (I/D) rs4343 gene polymorphism.

Primers FO and RO flank the exon of the ACE2 rs4240157 T > C gene, resulting in a band of 386 bp to act as a control for DNA quality and quantity. Primers FI and RO amplify a wild-type allele (T allele), generating a band of 244 bp, and primers FO and RI generate a band of 194 bp from the mutant allele (C allele) as depicted in [Figure 2](#jpm-11-01098-f002).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c687/8621157/aeee655ca7ce/jpm-11-01098-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8621157_jpm-11-01098-g002.jpg)

Two percent gel electrophoresis of angiotensin-converting enzyme 2 (ACE2) rs4240157 C/T genotyping by the amplification-refractory mutation system (ARMS) in COVID-19 cases.

### 2.6. Statistical Analysis

Deviations from Hardy–Weinberg disequilibrium (HWD) were calculated by chi-square (χ^2^) goodness-of-fit test. Group differences were compared using Student’s two-sample t-test or one-way analysis of variance (ANOVA) for continuous variables and chi-squared for categorical variables. Differences in the ACE-rs4646994 I/D, and ACE2 rs4240157 C > T allele and genotype frequencies between groups were evaluated using Chi-square test. The associations between ACE2-rs4646994 I/D, ACE2 rs4240157 C > T genotypes and risk of COVID-19 patients were estimated by computing the odds ratios (ORs), risk ratios (RRs) and risk differences (RDs) with 95% confidence intervals (CIs). Allele frequencies among cases as well as controls were evaluated by using the chi-square Hardy–Weinberg equilibrium (HWE) test. A *p*-value < 0.05 was considered significant. All statistical analyses were performed SPSS 16.0.

## 3. Results

### 3.1. Demographic Features

All demographic features of the 117 consecutive COVID-19 patients are summarized in [Table 2](#jpm-11-01098-t002). Of 117 consecutive patients, 20 (17.10%) patients were below or equal to 40 years and 97 (82.90%) were above 40 years of age, 85 (72.64%) were males and 32 (27.36%) were females. Among 117 COVID-19 patients, 47 (40.17%) had T2DM, 37 (9.40%) were having hypertension, 11 (9.40%) having CKD and 47 (40.17%) were having low oxygen saturation (<60 mm Hg) at the time of admission ([Table 2](#jpm-11-01098-t002)). 57 (48.71%) stayed more than 30 days in hospital. Regarding treatment of COVID-19 patients 79/117 (67.52%) were treated with antiviral therapy whereas 77/117 (65.81%) were treated with steroid therapy. Clinical outcome of the COVID-19 patients showed that 43/117 (36.75%) patients died and 74/117 (63.24%) were discharged and survived. At the time of diagnosis of COVID-19 patients, 45/117 (38.57%) had high levels of ALT, 104/117 (97.44%) had high levels of CRP and 48/117 (41.3%) had high levels of AST ([Table 2](#jpm-11-01098-t002)).

### Table 2.

| Clinical Feature | Variable | N = 117 | % |
| --- | --- | --- | --- |
| Age | >40 | 97 | 82.90 |
| <40 | 20 | 17.09 |  |
| Gender | Males | 85 | 72.64 |
| Females | 32 | 27.36 |  |
| T2D | Yes | 47 | 40.17 |
| No | 70 | 59.83 |  |
| CKD | Yes | 11 | 9.40 |
| No | 106 | 90.60 |  |
| Hypertension | Yes | 37 | 31.62 |
| No | 80 | 68.37 |  |
| CAD | Yes | 17 | 14.53 |
| No | 100 | 85.47 |  |
| Oxygen saturation | <60 | 47 | 40.17 |
| >80 | 70 | 59.83 |  |
| Duration in hospital | >30 | 57 | 48.71 |
| <30 | 60 | 51.29 |  |
| ALT | <36 | 72 | 61.53 |
| >36 | 45 | 38.57 |  |
| CRP | 0.8 | 13 | 2.56 |
| 0.8 high | 104 | 97.44 |  |
| AST | <40 | 69 | 58.97 |
| >40 | 48 | 41.3 |  |
| Antiviral therapy | Yes | 79 | 67.52 |
| No | 38 | 32.48 |  |
| Steroids therapy | Yes | 77 | 65.81 |
| No | 40 | 34.19 |  |
| Survival | Yes | 43 | 36.75 |
| No | 74 | 63.24 |  |

Table 2 Caption: Baseline characteristics of the COVID-19 patients. T2D—type2 diabetes; CKD—chronic kidney disease; CAD—coronary artery disease; SpO2—partial pressure of oxygen; ALT—alanine transaminase; CRP—C reactive protein; AST—aspartate transaminase.

### 3.2. Healthy Controls

Out of 150 age matched heathy controls 100 (66.67%) were males and 50 (33.33%) were females and 110 (73.33%) were above 40 years of age and 40 (26.67%) were below or equal to 40 years.

### 3.3. Statistical Comparisons between Patients and Controls (p Values) for ACE I/D Genotypes

We found that the ACE I/D genotype (ACE2-rs4646994 I/D) frequency among all the study participants is in compliance to the HWE. The genotype distributions and allele frequencies of the SNPs located in the ACE2-rs4646994 I/D showed no deviation in HWE (all *p* > 0.05) (χ^2^ = 2.24 *p* ≤ 0.13) in the COVID-19 patient group and similarly the genotype distributions and allele frequencies showed that no deviation in HWE (all *p* > 0.05) (χ^2^ = 0.52 *p* ≤ 0.47) in the controls. Thus, we chose 10% samples from normal control group randomly to review genotyping results, showing that the accuracy rate was more than 99%. In COVID-19 patients, the II, DI and DD genotype frequencies were 13.67%, 37.60% and 48.71% respectively whereas in healthy controls II, DI and DD genotype frequencies were 26.66%, 33.34%, and 40% respectively ([Table 3](#jpm-11-01098-t003)). The distribution of ACE2-rs4646994 I/D genotypes observed between COVID-19 patients and healthy controls was significant (*p* < 0.030). Moreover, the frequency of D allele (fD) was found to be significantly higher among COVID-19 patients than in HC (0.67 vs. 0.57) ([Table 3](#jpm-11-01098-t003)).

### Table 3.

| Subjects | N | II% | DI% | DD% | fD | χ2 | I | D | p-Value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Cases | 117 | 16 (13.67) | 44 (37.60) | 57 (48.71) | 2 | 6.77 | 0.33 | 0.67 | 0.03 |
| Controls | 150 | 40 (26.66) | 50 (33.34) | 60 (40) |   |   | 0.43 | 0.57 |   |

Table 3 Caption: Association of ACE2-rs4646994 I/D genotypes in COVID-19 severity.

### 3.4. Multivariate Analysis to Estimate the Association between ACE2-rs4646994 I/D Genotypes and Risk to COVID-19 Severity

A multivariate analysis based on logistic regression like odds ratio (OD) and risk ratio (RR) with 95% confidence intervals (CI) were calculated for each group to estimate the association between ACE2-rs4646994 I/D genotypes and risk to COVID-19 and the data are summarized in [Table 4](#jpm-11-01098-t004). Our results indicated that in the codominant model, the ACE2-ID genotype was strongly associated with increased COVID-19 severity with OR 2.20 (95%) CI = (1.08–4.46), RR = 1.34 (1.04–1.72), *p* < 0.020. Similarly, ACE2-DD genotype was strongly associated with increased COVID-19 severity with OR 2.37 (95%) CI = (1.19–4.70), RR = 1.39 (1.09–1.77), *p* < 0.013. There is a strong association observed between ACE2–II genotype vs. ACE2-(DI + DD) genotype in dominant inheritance model and leads to increased COVID-19 severity with OR = 2.39 (95%) CI (1.21–4.35), RR = 1.37 (1.11–1.69), *p* < 0.010 ([Table 4](#jpm-11-01098-t004)). In allelic comparison, the D allele was strongly associated with COVID-19 severity with an OR 1.58 (95% CI) (1.11–2.27), RR 1.21 (1.05–1.41) *p*-value = 0.010. No significance was observed in over dominant inheritance model. This result indicates a potential dominant effect of ACE2-DD genotype and D allele on COVID-19 severity in our study group.

### Table 4.

| Genotypes | Healthy Controls | COVID-19 Cases | Odd Ratio OR (95% CI) | Risk RatioRR (95% CI) | p-Value | Status |
| --- | --- | --- | --- | --- | --- | --- |
|   | (N = 150) | (N = 117) |   |   |   |   |
| Codominant |   |   |   |   |   |   |
| ACE2-II | 40 | 16 | 1 (ref.) | 1 (ref.) |   |   |
| ACE2-ID | 50 | 44 | 2.20 (1.08–4.46) | 1.34 (1.04–1.72) | 0.02 | S |
| ACE2-DD | 60 | 57 | 2.37 (1.19–4.70) | 1.39 (1.09–1.77) | 0.013 | S |
| Dominant |   |   |   |   |   |   |
| ACE2-II | 40 | 16 | 1 (ref.) | 1 (ref.) |   |   |
| ACE2-(DI + DD) | 110 | 101 | 2.39 (1.21–4.35) | 1.37 (1.11–1.69) | 0.010 | S |
| Recessive |   |   |   |   |   |   |
| ACE2-(II + DI) | 90 | 60 | 1 (ref.) | 1 (ref.) |   |   |
| ACE2-DD | 60 | 57 | 1.42 (0.87–2.32) | 1.17 (0.93–1.45) | 0.15 | NS |
| Allele |   |   |   |   |   |   |
| ACE2-I | 130 | 76 | 1 (ref.) | 1 (ref.) |   |   |
| ACE2-D | 170 | 158 | 1.58 (1.11–2.27) | 1.21 (1.05–1.41) | 0.010 | S |
| Over dominant |   |   |   |   |   |   |
| ACE2-II + DD | 90 | 73 | 1 (ref.) | 1 (ref.) |   |   |
| ACE2-ID | 50 | 44 | 1.08 (0.65–1.80) | 1.03 (0.82–1.31) | 0.75 | NS |

Table 4 Caption: Multivariate and ordinal regression risk factor analysis for COVID-19 severity with ACE2-rs4646994 I/D genotypes.

### 3.5. Association of ACE2-rs4646994 I/D Genotypes with Comorbid Conditions by COVID-19 Severity

The statistical comparisons (*p*-values) of ACE2-rs4646994 I/D genotypes with comorbid conditions by COVID-19 severity was done by using a multivariate analysis based on logistic regression like odds ratio (OD) and risk ratio (RR) with 95% confidence intervals (CI) ([Table 5](#jpm-11-01098-t005)). Results showed that the there was a significant correlation between the ACE2-rs4646994 I/D genotypes with respect to the age of the COVID-19 patients (*p* < 0.035), COVID-19 patients having T2D (*p* < 0.0013), COVID-19 patients having hypertension (*p* < 0.0031), COVID-19 patients having coronary artery disease (*p* < 0.0001) ([Table 5](#jpm-11-01098-t005)). Similarly, a significant correlation was reported between ACE2 rs4240157 genotypes and COVID-19 cases whose oxygen saturation was less than <60 mm Hg. (*p* < 0.0001). A significant correlation was reported for ACE2 rs4240157 genotype distribution with respect to duration of COVID-19 cases in the hospital (*p* < 0.044). Similarly, in terms of treatment, a significant correlation was reported between antiviral therapy with respect to and ACE2D/I genotype ([Table 5](#jpm-11-01098-t005)).

### Table 5.

| Clinical Feature | Variable | N = 117 | II | DI | DD | fD | χ2 | p |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Age | >40 | 97 | 10 | 40 | 47 | 2 | 6.7 | 0.035 |
|   | <40 | 20 | 6 | 04 | 10 |   |   |   |
| Gender | Males | 85 | 12 | 30 | 43 | 2 | 0.71 | 0.70 |
| Females | 32 | 04 | 14 | 14 |   |   |   |  |
| T2D | Yes | 47 | 5 | 27 | 15 | 2 | 13.3 | 0.0013 |
| No | 70 | 11 | 17 | 42 |   |   |   |  |
| CKD | Yes | 11 | 3 | 2 | 6 | 2 | 2.94 | 0.222 |
| No | 106 | 13 | 42 | 51 |   |   |   |  |
| Hypertension | Yes | 37 | 10 | 17 | 10 | 2 | 13.28 | 0.0031 |
| No | 80 | 06 | 27 | 47 |   |   |   |  |
| CAD | Yes | 17 | 10 | 02 | 05 | 2 | 34.22 | 0.0001 |
| No | 100 | 06 | 42 | 52 |   |   |   |  |
| Oxygen saturation [SpO2, mm Hg] | <60 | 47 | 06 | 05 | 36 | 2 | 27.1 | 0.0001 |
| >80 | 70 | 10 | 39 | 21 |   |   |   |  |
| Duration in hospital | >30 | 57 | 10 | 15 | 32 | 2 | 6.24 | 0.044 |
| <30 | 60 | 06 | 29 | 25 |   |   |   |  |
| ALT | <36 | 72 | 10 | 24 | 38 | 2 | 1.55 | 0.46 |
| >36 | 45 | 06 | 20 | 19 |   |   |   |  |
| CRP | <0.8 | 13 | 0 | 4 | 9 | 2 | 3.44 | 0.17 |
| >0.8 high | 104 | 16 | 40 | 48 |   |   |   |  |
| AST | <40 | 69 | 10 | 19 | 40 | 2 | 7.57 | 0.022 |
| >40 | 48 | 06 | 25 | 17 |   |   |   |  |
| Antiviral therapy | Yes | 79 | 08 | 21 | 50 | 2 | 20.1 | 0.0001 |
| No | 38 | 08 | 23 | 07 |   |   |   |  |
| Steroids therapy | Yes | 77 | 11 | 29 | 37 | 2 | 0.08 | 0.96 |
| No | 40 | 05 | 15 | 20 |   |   |   |  |
| Survival | Death occurred | 43 | 11 | 20 | 12 | 2 | 13.23 | 0.0007 |
|   | Survived | 74 | 05 | 24 | 45 |   |   |   |

Table 5 Caption: Statistical comparisons (p-values) of ACE2-rs4646994 I/D genotypes with comorbid conditions and some clinical parameters with COVID-19 disease severity. T2D—type2 diabetes; CKD—chronic kidney disease; CAD—coronary artery disease; SpO2—partial pressure of oxygen; ALT—alanine transaminase; CRP—C reactive protein; AST—aspartate transaminase.

### 3.6. Correlation of ACE2 I/D Genotypes with Age of COVID-19 Disease Severity

As can be seen in [Table 6](#jpm-11-01098-t006), our result indicated that in the codominant model, the ACE2-ID heterozygosity genotype (II vs. DI) was strongly associated with increased COVID-19 severity in subjects with age >40 with OR 6.0 (95%) CI = (1.41 to 25.38), RR = 4.12 (1.33 to 12.74), *p* < 0.014. Similarly, ACE2-DD genotype (II vs. DD + DI) was strongly associated with increased COVID-19 severity in subjects with age >40 with OR 3.72 (95%) CI = (1.17 to 11.88), RR = 2.70 (1.21–6.0), *p* < 0.026. In allelic comparison, the D allele was strongly associated with COVID-19 severity with an OR 1.58 (95% CI) (1.11–2.27), RR 1.21 (1.05–1.41) *p*-value = 0.010 ([Table 6](#jpm-11-01098-t006)).

### Table 6.

| Age | <40 | >40 | Odd Ratio (95% CI) | Risk Ratio (95% CI) | p-Value |
| --- | --- | --- | --- | --- | --- |
|   | N = 20 | N = 97 |   |   |   |
| Codominant inheritance model |  |  |  |  |  |
| II | 06 | 10 | Ref | Ref |   |
| DI | 04 | 40 | 6.0 (1.41 to 25.38) | 4.12 (1.33 to 12.74) | 0.0149 |
| DD | 10 | 47 | 2.82 (0.83 to 9.55) | 2.13 (0.91 to 4.98) | 0.09 |
| Dominant inheritance model |  |  |  |  |  |
| II | 06 | 10 | Ref | Ref |   |
| DD + DI | 14 | 87 | 3.72 (1.17 to 11.88) | 2.70 (1.21 to 6.0) | 0.026 |
| Recessive inheritance model |  |  |  |  |  |
| II + DI | 10 | 50 | Ref | Ref |   |
| DD | 10 | 47 | 0.94 (0.35 to 2.46) | 0.95 (0.42 to 2.11) | 0.89 |
| Over dominant inheritance model |  |  |  |  |  |
| II + DD | 16 | 57 | Ref | Ref |   |
| DI | 04 | 40 | 2.80 (0.87 to 9.025) | 2.41 (0.86 to 6.75) | 0.08 |

Table 6 Caption: Statistical comparisons (p-values) of ACE2-rs4646994 I/D genotypes with age of COVID-19 severity.

### 3.7. Correlation of ACE2 I/D Genotypes with Mortality of COVID-19 Patients

The results are summarized in [Table 7](#jpm-11-01098-t007) and indicate that in the codominant model, ACE2-DD genotype (II vs. DD) was strongly associated with increased COVID-19 mortality OR 8.25 (95%) CI = (2.40 to 28.34), RR = 3.26 (1.7894 to 5.9596), *p* < 0.008. Also, in dominant model, ACE2-DD + DI genotype (II vs. DD + DI) was strongly associated with increased COVID-19 mortality with OR 4.74 (95%) CI = (1.5214 to 14.7915), RR = 2.16 (1.4014 to 3.3599), *p* < 0.007. Similarly, in recessive inheritance model, ACE2-DI + II genotype (DD vs. DI + II) was strongly associated with increased COVID-19 mortality with OR 4.0 (95%) CI = (1.7769 to 9.0432), RR = 2.45 (1.4031 to 4.2927), *p* < 0.0008.

### Table 7.

|   | N = 43 | N = 74 | Odd Ratio (95% CI) | Risk Ratio (95% CI) | p-Value |
| --- | --- | --- | --- | --- | --- |
| Codominant inheritance model |  |  |   |   |   |
| II | 11 | 05 | Ref | Ref |   |
| DI | 20 | 24 | 2.64 (0.7854 to 8.8737) | 1.51 (0.9524 to 2.4019) | 0.11 |
| DD | 12 | 45 | 8.25 (2.40 to 28.34) | 3.26 (1.7894 to 5.9596) | 0.008 |
| Dominant inheritance model |  |  |   |   |   |
| II | 11 | 05 | Ref | Ref |   |
| DD + DI | 32 | 69 | 4.74 (1.5214 to 14.7915) | 2.16 (1.4014 to 3.3599) | 0.007 |
| Recessive inheritance model |  |  |   |   |   |
| II + DI | 31 | 29 | Ref | Ref |   |
| DD | 12 | 45 | 4.0 (1.7769 to 9.0432) | 2.45 (1.4031 to 4.2927) | 0.0008 |
| Overdominant inheritance model |  |  |   |   |   |
| II + DD | 23 | 50 | Ref | Ref |   |
| DI | 20 | 24 | 0.55 (0.2551 to 1.1946) | 0.69 (0.4340 to 1.1070) | 0.13 |

Table 7 Caption: Statistical comparisons (p-values) of ACE2-rs4646994 I/D genotypes with mortality of COVID-19 patients.

### 3.8. Statistical Comparisons between Cases and Controls (p-Values) for ACE2 rs4240157 C > T Genotypes

The genotype distributions and allele frequencies of the ACE2 rs4240157 T > C showed that no deviation was detected in HWE (all *p* > 0.05) (χ^2^ = 3.38 *p* ≤ 0.06) in the controls. Thus, we chose 10% samples from normal control group randomly to review genotyping results, showing that the accuracy rate were more than 99%. The frequency of ACE2 rs4240157 T > C gene variation in COVID-19 cases and controls was TT (35%), TC (40.17%) and CC (24.78%) and controls TT (59%), TC (31%) and CC (10%) respectively ([Table 8](#jpm-11-01098-t008)). The ACE2 rs4240157 T > C gene variation observed between COVID-19 patients and controls was statistically significant (*p* < 0.0007). Moreover, the frequency of C allele was found to be higher among COVID-19 patients than in HC (0.45 vs. 0.26).

### Table 8.

| Subjects | N | TT% | CT% | CC% | fD | χ2 | T | C | p-Value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Cases | 117 | 41 (35) | 47 (40.17) | 29 (24.78) | 2 | 14.34 | 0.55 | 0.45 | 0.0007 |
| Controls | 100 | 59 (59) | 31 (31) | 10 (10) |   |   | 0.74 | 0.26 |   |

Table 8 Caption: ACE2 rs4240157 T > C gene polymorphism in COVID-19 patients and controls.

### 3.9. Multivariate Analysis to Estimate the Association of ACE2-rs4646994 I/D Genotypes with COVID-19 Severity

A multivariate analysis based on logistic regression like odds ratio (OD) and risk ratio (RR) with 95% confidence intervals (CI) were calculated for each group to estimate the association of ACE2 rs4240157 T > C genotypes with COVID-19 severity ([Table 9](#jpm-11-01098-t009)). In the co-dominant inheritance model, ACE2–CT genotype was associated with COVID-19 severity with an OR 2.18 (95% CI) (1.92–3.99), RR 1.48 (1.07–2.04) *p* < 0.010 as well as ACE2–CC genotype was linked with COVID-19 severity with an OR 2.66 (95% CI) (1.53–4.62), RR 1.68 (1.25–2.26) *p* < 0.005. ([Table 9](#jpm-11-01098-t009)). In case of dominant inheritance model, ACE2-(CT + CC) genotype was associated with COVID-19 severity with an OR 2.66 (95% CI) (1.53–4.62), RR 1.68 (1.25–2.26) *p* < 0.005. In case of recessive inheritance model, ACE2–CC vs. ACE2-(TT + CT) genotype was associated with covid-19 severity with an OR 2.96 (95% CI) (1.36–6.44), RR 1.97 (1.13–3.43) *p* < 0.006. In allelic comparison, the T allele was compared with the C allele. There was a no significant association of C allele with the COVID-19 severity with OR 0.86 (0.59–1.27), RR 0.92 (0.75–1.13), *p*-value = 0.46.

### Table 9.

| Genotypes | Healthy Controls | COVID-19 Cases | OR (95% CI) | Risk Ratio (RR) | p-Value |
| --- | --- | --- | --- | --- | --- |
|   | (N = 100) | (N = 117) |   |   |   |
| Codominant |   |   |   |   |   |
| ACE2-TT | 59 | 41 | 1 (ref.) | 1 (ref.) |   |
| ACE2-CT | 31 | 47 | 2.18 (1.92–3.99) | 1.48(1.07–2.04) | 0.01 |
| ACE2-CC | 10 | 29 | 2.66 (1.53–4.62) | 1.68(1.25–2.26) | 0.005 |
| Dominant |   |   |   |   |   |
| ACE2-TT | 59 | 41 | 1 (ref.) | 1 (ref.) |   |
| ACE2-(CT + CC) | 41 | 76 | 2.66 (1.53–4.62) | 1.68(1.25–2.26) | 0.005 |
| Recessive |   |   |   |   |   |
| ACE2-(TT + CT) | 90 | 88 | 1 (ref.) | 1 (ref.) |   |
| ACE2-CC | 10 | 29 | 2.96(1.36–6.44) | 1.97 (1.13–3.43) | 0.006 |
| Allele |   |   |   |   |   |
| ACE2-T | 100 | 121 | 1 (ref.) | 1 (ref.) |   |
| ACE2-C | 100 | 105 | 0.86 (0.59–1.27) | 0.92 (0.75–1.13) | 0.46 |

Table 9 Caption: Multivariate analysis of ACE2 rs4240157 T > C gene polymorphism with COVID-19 severity.

### 3.10. Association of ACE2 rs4240157 T > C Genotypes with Gender and Comorbid Conditions by COVID-19 Severity

The statistical comparisons (*p*-values) of ACE2 rs4240157 T > C genotypes with comorbid conditions by COVID-19 severity was done by using a multivariate analysis based on logistic regression like odds ratio (OD) and risk ratio (RR) with 95% confidence intervals (CI) ([Table 10](#jpm-11-01098-t010)). Results showed that there was a significant correlation between the ACE2 rs4240157 T > C genotypes with respect to the gender of the COVID-19 patients (*p* < 0.046). A significant correlation was reported between the ACE2 rs4240157 T > C genotypes and COVID-19 severity having T2D (*p* < 0.007), hypertension (0.006) and coronary artery disease (*p* < 0.049). Similarly, a significant correlation was reported between ACE2 rs4240157 T > C genotypes and COVID-19 cases whose oxygen saturation was less than <60 mm Hg. (*p* < 0.0001). A significant correlation was reported between steroid therapy and ACE2 rs4240157 T > C genotypes in COVID-19 patients. A significant correlation was also reported between ACE2 rs4240157 T > C genotypes in COVID-19 patients with hypertension (*p* < 0.006) as shown in [Table 10](#jpm-11-01098-t010).

### Table 10.

| Clinical Feature | Variable | N | TT | CT | CC | fD | χ2 | p |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Age | <45 | 20 | 07 | 07 | 06 | 2 | 0.43 | 0.80 |
|   | >45 | 97 | 34 | 40 | 23 |   |   |   |
| Gender | Males | 85 | 26 | 33 | 26 | 2 | 6.12 | 0.046 |
|   | Females | 32 | 15 | 14 | 03 |   |   |   |
| T2D | Yes | 47 | 11 | 27 | 09 | 2 | 9.88 | 0.007 |
|   | No | 70 | 30 | 20 | 20 |   |   |   |
| CKD | Yes | 11 | 04 | 04 | 03 | 2 | 0.08 | 0.960 |
|   | No | 106 | 37 | 43 | 26 |   |   |   |
| Hypertension | Yes | 37 | 14 | 16 | 07 | 2 | 1 | 0.006 |
|   | No | 80 | 27 | 31 | 22 |   |   |   |
| CAD | Yes | 17 | 05 | 11 | 01 | 2 | 6.03 | 0.049 |
|   | No | 100 | 36 | 36 | 28 |   |   |   |
| Oxygen saturation | <60 mm Hg | 47 | 14 | 13 | 20 | 2 | 13.34 | 0.001 |
|   | >80 mm Hg | 70 | 33 | 28 | 09 |   |   |   |
| Duration in hospital | >30 days | 57 | 17 | 26 | 14 | 2 | 1.69 | 0.420 |
|   | <30 days | 60 | 24 | 21 | 15 |   |   |   |
| ALT | >36 units | 45 | 19 | 21 | 05 | 2 | 7.36 | 0.025 |
|   | <36 units | 72 | 22 | 26 | 24 |   |   |   |
| CRP | >0.8 mg | 104 | 38 | 43 | 23 | 2 | 3.61 | 0.16 |
|   | <0.8 mg | 13 | 03 | 04 | 06 |   |   |   |
| AST | >40 | 48 | 20 | 19 | 09 | 2 | 2.22 | 0.32 |
|   | <40 | 69 | 21 | 28 | 20 |   |   |   |
| Antiviral therapy | Yes | 79 | 27 | 33 | 19 | 2 | 0.26 | 0.87 |
|   | No | 38 | 14 | 14 | 10 |   |   |   |
| Steroids therapy | Yes | 77 | 31 | 23 | 23 | 2 | 10.23 | 0.006 |
|   | No | 40 | 10 | 24 | 06 |   |   |   |
| Survival | Death occurred | 43 | 14 | 10 | 19 | 2 | 15.29 | 0.0005 |
|   | Survived | 74 | 27 | 37 | 10 |   |   |   |

Table 10 Caption: Statistical comparisons (p-values) of ACE2 rs4240157 T > C genotypes with comorbid conditions and some clinical parameters with COVID-19 severity. T2D—type2 diabetes; CKD—chronic kidney disease; CAD—coronary artery disease; SpO2—partial pressure of oxygen; ALT—alanine transaminase; CRP—C reactive protein; AST—aspartate transaminase.

### 3.11. Correlation of ACE2 rs4240157 T > C Genotypes with Mortality of COVID-19 Patients

Our results indicated that in the codominant model, the ACE2-CC genotype (TT vs. CC) was strongly associated with increased COVID-19 mortality OR 3.66 (95%) CI = (1.34 to 9.97), RR = 1.90 (1.10–3.30), *p* < 0.011 ([Table 11](#jpm-11-01098-t011)). Similarly, in the recessive inheritance model, ACE2-(TT + CT) genotype was strongly associated with increased COVID-19 mortality with OR 5.0 (95%) CI = (2.0640–12.437), RR = 2.10 (1.2567 to 3.53), *p* < 0.004. In allelic comparison, the T allele was compared with the C allele. There was a significant association of C allele with the COVID-19 severity with OR 2, 01 (1.1761–3.45), RR 1.29 (1.05–1.59), *p* < 0.010 ([Table 11](#jpm-11-01098-t011)).

### Table 11.

| Codominant Model | N = 74 | N = 43 | Odd Ratio (95% CI) | Risk Ratio (95% CI) | p-Value |
| --- | --- | --- | --- | --- | --- |
| TT | 27 | 14 | Ref | Ref |   |
| CT | 37 | 10 | 0.52 (0.20 to 1.34) | 0.83 (0.64 to 1.09) | 0.17 |
| CC | 10 | 19 | 3.66 (1.34 to 9.97) | 1.90 (1.10–3.30) | 0.011 |
| Dominant model |   |   |   |   |   |
| TT | 27 | 14 | Ref | Ref |   |
| CC + CT | 47 | 29 | 1.19 (0.5378 to 2.6332) | 1.06 (0.8028 to 1.4124) | 0.66 |
| Recessive model |   |   |   |   |   |
| TT + CT | 64 | 24 | Ref | Ref |   |
| CC | 10 | 19 | 5.06 (2.0640–12.437) | 2.10 (1.2567 to 3.53) | 0.004 |
| Allele |   |   |   |   |   |
| T | 91 | 38 | Ref | Ref |   |
| C | 57 | 48 | 2.01 (1.1761–3.45) | 1.29 (1.05 to 1.59) | 0.010 |

Table 11 Caption: Statistical comparisons (p-values) of ACE2 rs4240157 T > C genotypes with mortality of COVID-19 patients.

[Table 12](#jpm-11-01098-t012) summarizes the comparison of ACE2 rs4240157 T > C genotypes with COVID-19 comorbidities. As can be seen, CT genotype is highly significant in T2D OR 3.68 (95% CI 1.495 to 9.064), and RR 1.71 (1.175 to 2.515) in codominant model. In the dominant model CC + CT genotype is also significant in T2D with OR 2.45 (95% CI 1.076 to 5.599) and RR 1.39 (1.048 to 1.844). CT genotype is also very significant with reference to age with *p*-value of 0.014, OR 3.36 (95% CI 1.268 to 8.908) and RR 2.48 (1.155 to 5.353).

### Table 12.

| Type 2 Diabetes |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- |
| Codominant Model | N = 70 | N = 47 | Odd Ratio (95% CI) | Risk Ratio (95% CI) | p-Value |
| TT | 30 | 11 | Ref | Ref |   |
| CT | 20 | 27 | 3.68 (1.495 to 9.064) | 1.71 (1.175 to 2.515) | 0.004 |
| CC | 20 | 09 | 1.22 (0.430 to 3.496) | 1.06 (0.780 to 1.441) | 0.070 |
| Dominant model |   |   |   |   |   |
| TT | 30 | 11 | Ref | Ref |   |
| CC + CT | 40 | 36 | 2.45 (1.076 to 5.599) | 1.39 (1.048 to 1.844) | 0.032 |
| Allele |   |   |   |   |   |
| T | 70 | 47 | Ref | Ref |   |
| C | 60 | 45 | 1.11 (0.654 to 1.906) | 1.04 (0.838 to 1.307) | 0.068 |
| Hypertension |  |  |  |  |  |
| Codominant model | N = 80 | N = 37 | Odd Ratio (95% CI) | Risk Ratio (95% CI) | p-value |
| TT | 27 | 14 | Ref | Ref |   |
| CT | 31 | 16 | 0.99 (0.4114 to 2.4084) | 0.99 (0.7387 to 1.3495) | 0.99 |
| CC | 22 | 07 | 0.61 (0.2110 to 1.7850) | 0.86 (0.642 to 1.173) | 0.370 |
| Dominant model |   |   |   |   |   |
| TT | 27 | 14 | Ref | Ref |   |
| CC + CT | 53 | 23 | 0.83 (0.3723 to 1.8814) | 0.94 (0.7241 to 1.2315) | 0.67 |
| Coronary artery disease |  |  |  |  |  |
| Codominant model | N = 100 | N = 17 | Odd Ratio (95% CI) | Risk Ratio (95% CI) | p-value |
| TT | 36 | 05 | Ref | Ref |   |
| CT | 36 | 11 | 2.20 (0.694 to 6.973) | 1.14 (0.943 to 1.393) | 0.180 |
| CC | 28 | 1 |   |   |   |
| Dominant model |   |   |   |   |   |
| TT | 36 | 05 | Ref | Ref |   |
| CC + CT | 64 | 12 | 0.25 (0.028 to 2.328) | 0.90 (0.796 to 1.039) | 0.22 |
| Age |  |  |  |  |  |
| Codominant model | <45N = 20 | >45N = 97 | Odd Ratio (95% CI) | Risk Ratio (95% CI) | p-value |
| TT | 20 | 34 | Ref | Ref |   |
| CT | 07 | 40 | 3.36 (1.268 to 8.908) | 2.48 (1.155 to 5.353) | 0.014 |

Table 12 Caption: Statistical comparisons (p-values) of ACE2 rs4240157 T > C genotypes in COVID-19 patients with various comorbidities.

## 4. Discussion

Coronaviruses are divided into α, β, γ, and δ genera on the basis of the target host [[1](#B1-jpm-11-01098),[2](#B2-jpm-11-01098),[3](#B3-jpm-11-01098)]. Out of these, mammals are infected by α and β-CoV whereas γ and δ-CoV genera tend to infect birds. The current COVID-19 pandemic is found to be caused by SARS-CoV-2 [[3](#B3-jpm-11-01098),[4](#B4-jpm-11-01098)]. It is a β-CoV that is enveloped, non-segmented and positive-sense RNA virus. Genome sequencing results showed that this newly discovered virus shares 96.2% identity with bat CoV RaTG13 and 79.5% identity with SARS-CoV [[10](#B10-jpm-11-01098)].

### 4.1. Role of RAAS in SARS-CoV-2 Infection

Different ACE genotypes are believed to be associated with development of acute respiratory distress syndrome and several studies have reported that the distribution frequency of ACE insertion/insertion (II) genotype might have a significant influence on COVID-19 mortality [[36](#B36-jpm-11-01098),[40](#B40-jpm-11-01098)]. It has been reported that ACE2 levels correlate with susceptibility to SARS-CoV-2 infection and men have a higher ACE2 expression in lung than women and Asian people express ACE2 higher than Caucasian and African American populations [[41](#B41-jpm-11-01098),[42](#B42-jpm-11-01098)].

### 4.2. Association of ACE2-D/I with Gender

Angiotensin converting enzyme is a metallopeptidase that converts angiotensin I (AT-I) to angiotensin II (AT-II) that acts as a vasoconstrictor in addition to its other functions. ACE also degrades bradykinin, a vasodilator substance. ACE2 is also a known SARS virus receptor [[14](#B14-jpm-11-01098)]. Both ACE1 and ACE2 are highly polymorphic. Polymorphisms present in ACE1 are just as important as those in ACE2. As it was reported that the conversion of Ang II to Ang (1–7) by ACE2 is higher in males than females [[43](#B43-jpm-11-01098)] because ACE2 gene is located on the X chromosome and men express more ACE2 than women that depends on the allelic expressions by women, therefore women may be considered of lower sensitivity against the SARS-CoV-2 infection and its lethal effects [[43](#B43-jpm-11-01098),[44](#B44-jpm-11-01098)].

Several research reports in mice have shown the protective effect of ACE2, whereby severe lung failure is associated with ACE2 downregulation. Immediately after SARS-CoV-2 infection, there is a downregulation of ACE2 [[37](#B37-jpm-11-01098),[38](#B38-jpm-11-01098),[45](#B45-jpm-11-01098)]. ACE gene on chromosome 17 consists of 26 exons and 25 introns. A functional insertion-deletion (I/D) polymorphism of 287 bp Alu repeat sequence has been reported in the intron 16 of the ACE gene [[44](#B44-jpm-11-01098),[46](#B46-jpm-11-01098)]. Jeong et al. have reported that the deletion (D) and insertion allele (I) are associated with an increased and decreased ACE level and enzyme activity, respectively [[47](#B47-jpm-11-01098)]. The ACE2 gene exhibits a high degree of genetic polymorphism [[42](#B42-jpm-11-01098)]. Itoyama et al. [[48](#B48-jpm-11-01098)] reported the correlation of ACE-D allele with the incidence of pneumonia in SARS patients and the death of subjects with acute respiratory distress syndrome. These results tempted us to investigate the role of ACE2 I/D polymorphism in susceptibility to SARS-CoV-2 infection and related mortality. Our results indicated higher overall frequencies of deletion polymorphism in COVID-19 patients (67%) than the insertion polymorphism (33%). The distribution of homozygote insertion (II) genotype is seen to be higher in the healthy population (26.66%) compared with the COVID-19 patients (13.67%). The frequency of heterozygote ID genotype is comparable between COVID-19 patients (37.60%) and control (33.34%) groups. We observed a significant difference in the distribution of homozygote deletion (DD) genotype between COVID-19 cases and controls (*p* < 0.03) ([Table 3](#jpm-11-01098-t003)).

### 4.3. Distribution of ACE2-II, DI and DD Genotypes in the World Populations

The distribution of ACE2-II, DI and DD genotypes in the SARS-CoV-2 patients in the world populations has been summarized in the [Table 13](#jpm-11-01098-t013) [[49](#B49-jpm-11-01098)]. The frequency of DD genotype in our patient population was high (48.71%) corresponding to the frequency of Brazil (49%), Finland (42.61%), Turkey (34.44%), Iran (37.50%), Spain (37.84%), Russia (39.38%), Italy (37.61%).

### Table 13.

| Country | Sample Size (N) | DD Genotype | % DD | ID Genotype | %ID | II Genotype | %II |
| --- | --- | --- | --- | --- | --- | --- | --- |
| USA | 4320 | 1331 | 30.81 | 2145 | 49.65 | 844 | 19.54 |
| UK | 10566 | 2896 | 27.41 | 5241 | 49.60 | 2429 | 22.99 |
| Canada | 299 | 85 | 28.43 | 149 | 49.83 | 65 | 21.74 |
| Turkey | 672 | 238 | 35.42 | 303 | 45.09 | 131 | 19.49 |
| Australia | 444 | 134 | 30.18 | 216 | 48.65 | 94 | 21.17 |
| Austria | 2251 | 619 | 27.50 | 1108 | 49.22 | 524 | 23.28 |
| Belgium | 3980 | 1074 | 26.98 | 2030 | 51.01 | 876 | 22.01 |
| India | 660 | 130 | 19.70 | 316 | 47.88 | 214 | 32.42 |
| China | 3764 | 1154 | 30.66 | 1228 | 32.62 | 1382 | 36.72 |
| Czech | 443 | 116 | 26.19 | 225 | 50.79 | 102 | 23.02 |
| Germany | 5047 | 1345 | 26.65 | 2581 | 51.14 | 1121 | 22.21 |
| Japan | 379 | 68 | 17.94 | 185 | 48.81 | 126 | 33.25 |
| Netherlands | 6110 | 1705 | 27.91 | 3056 | 50.02 | 1349 | 22.08 |
| South-Africa | 300 | 46 | 15.33 | 158 | 52.67 | 96 | 32.00 |
| Israel | 87 | 9 | 10.34 | 50 | 57.47 | 28 | 32.18 |
| Korea | 785 | 256 | 32.61 | 294 | 37.45 | 235 | 29.94 |
| Denmark | 126 | 41 | 32.54 | 43 | 34.13 | 42 | 33.33 |
| Italy | 3151 | 1185 | 37.61 | 1358 | 43.10 | 608 | 19.30 |
| Russia | 320 | 126 | 39.38 | 125 | 39.06 | 69 | 21.56 |
| Spain | 1665 | 630 | 37.84 | 767 | 46.07 | 268 | 16.10 |
| Iran | 200 | 75 | 37.50 | 96 | 48.00 | 29 | 14.50 |
| Turkey | 90 | 31 | 34.44 | 14 | 15.55 | 45 | 50 |
| Finland | 115 | 49 | 42.61 | 52 | 45.22 | 14 | 12.17 |
| Brazil | 104 | 51 | 49.04 | 33 | 31.73 | 20 | 19.23 |
| Saudi Arabia[current study] | 117 | 57 | 48.71 | 44 | 37.60 | 16 | 13.67 |

Table 13 Caption: Distribution of ACE2- II, DI and DD genotypes in the SARS-CoV-2 patients in the world populations.

Lee et al., reported that populations in France, Italy and Spain have shown high *D* allele frequency between 82% and 87% [[50](#B50-jpm-11-01098)]. Similarly, African Americans in the United States have been reported to have the highest *D* allele frequency (89%) compared with white Americans (69%) [[33](#B33-jpm-11-01098)]. Conversely, populations in East Asian countries such as China, Japan, Taiwan and Korea have a high frequency of the II genotype [[51](#B51-jpm-11-01098)]. Pati et al. [[52](#B52-jpm-11-01098)] reported that the low DD genotype frequency and high II genotype frequency in the ACE gene is strongly correlated with the relatively low mortality rate of COVID-19 among these populations [[52](#B52-jpm-11-01098)]. A higher mortality rate has been shown among the European populations as in the case of black ethnicity in the United States [[34](#B34-jpm-11-01098)]. In the present study, we observed a positive correlation of allele D with infection and mortality rate of COVID-19.

Recent studies have also shown that the ethnic variations of the ACE I/D genotype tend to correlate with the variations in outcomes where populations with a high D genotype frequency tend to experience higher mortality rates [[23](#B23-jpm-11-01098)]. Our results indicated that in the codominant model, the ACE2-DD genotype was strongly associated with increased COVID-19 mortality OR 8.25 (95%) CI = (2.40 to 28.34), *p* < 0.008 ([Table 7](#jpm-11-01098-t007)). Mortality of SARS-CoV-2 subjects is dependent on a wide range of complex phenomena and genetic mutations are one among them [[53](#B53-jpm-11-01098),[54](#B54-jpm-11-01098),[55](#B55-jpm-11-01098),[56](#B56-jpm-11-01098)]. The prevalence of the DD genotype is higher in patients with severe lung infections and is significantly correlated with a high death rate. Mortality related to SARS-CoV-2 infections has been associated to various comorbid conditions such as hypertension, diabetes, hyperlipidemia, coronary artery disease and renal disease [[23](#B23-jpm-11-01098),[57](#B57-jpm-11-01098),[58](#B58-jpm-11-01098),[59](#B59-jpm-11-01098),[60](#B60-jpm-11-01098),[61](#B61-jpm-11-01098),[62](#B62-jpm-11-01098),[63](#B63-jpm-11-01098),[64](#B64-jpm-11-01098)]. High producer of ACE, the DD genotype has been associated with susceptibility to hypertension. [[34](#B34-jpm-11-01098),[35](#B35-jpm-11-01098),[55](#B55-jpm-11-01098),[62](#B62-jpm-11-01098)], Type 2 diabetes [[59](#B59-jpm-11-01098),[60](#B60-jpm-11-01098)], coronary artery disease [[57](#B57-jpm-11-01098),[59](#B59-jpm-11-01098)] indicating a possible role of ACE I/D polymorphism with SARS-CoV-2 related mortality. Our results reported a significant correlation of ACE2-DD genotypes with advanced age of the COVID-19 patients (*p* < 0.035), with T2D (*p* < 0.0013), hypertension (*p* < 0.0031) and coronary artery disease (*p* < 0.0001) [[Table 5](#jpm-11-01098-t005)]. Our study revealed that age is a significant risk factor for severe disease. Results showed that the there was a significant correlation between the ACE2-rs4646994 I and D genotypes with respect to the Age of the COVID-19 patients (*p* < 0.035). Advanced age is a significant risk factor for mortality from COVID-19, the important reason being the associated increased frequency of comorbid conditions with advancing age [[61](#B61-jpm-11-01098),[63](#B63-jpm-11-01098)].

### 4.4. Role of ACE2 rs4240157 T > C Gene Polymorphism in COVID-19 Severity

Our results indicated a significant correlation of ACE2 rs4240157 T > C genotypes with the COVID-19 severity among the COVID-19 patients with T2D (*p* < 0.007), with coronary artery disease (*p* < 0.049) and hypertension (*p* < 0.006) ([Table 10](#jpm-11-01098-t010)). Recent studies identified ACE2 polymorphisms that might influence disease severity and indicated that out of 10 studied SNPs, 5 polymorphisms (rs6632680, rs4830965, rs1476524, rs4240157 and rs2048683) indicated an association with higher tissue specific expression of ACE2 resulting in hospitalization whereas rs1548474 polymorphism showed correlation with low tissue expression and lesser severity [[23](#B23-jpm-11-01098),[37](#B37-jpm-11-01098),[38](#B38-jpm-11-01098),[39](#B39-jpm-11-01098),[40](#B40-jpm-11-01098)]. Variation in circulating ACE2 levels was speculated to be controlled by genetic factors including rs2106809 polymorphism [[39](#B39-jpm-11-01098),[63](#B63-jpm-11-01098)]. Xiao et al., [[64](#B64-jpm-11-01098)] have earlier reported that a point mutation in the ACE2 gene (Leu584Ala) facilitates entry of SARS-CoV-1 into host cells. Recent studies have proven that several amino acid variants can potentially affect the interaction between the viral S1 protein and ACE2 receptors and thus the level of infection [[65](#B65-jpm-11-01098)]. Different amino acid residues expressed within the ACE2 receptor were observed to be very relevant either by promoting or preventing viral infection. A total of 13 ACE2 polymorphisms enhanced ACE2/S1 recognition, thereby facilitating SARS-CoV-2 infection while as in contrast, 18 SNPs hindered interactions between ACE2 and S1, thereby reduces the infection rate [[62](#B62-jpm-11-01098)]. Patel et al., [[59](#B59-jpm-11-01098)] reported that ACE 2 rs2074192, rs4240157 and rs4646188 variations in T2D exhibited higher risk with hypertension among persons of Australian descent. ACE2 s4240157 (*p* < 0.001) and rs4646156 (*p* < 0.037) are correlated with increased blood pressure [[64](#B64-jpm-11-01098)]. Our results reported higher frequency of heterozygosity among SARS-CoV-2 cases of ACE2-rs4240157 TC genotypes (40.17%) followed by TT genotypes (35%), and CC genotypes (24.78%) respectively [[Table 8](#jpm-11-01098-t008)].

In our study, the ACE2 rs4240157 T > C gene polymorphism observed between COVID-19 patients and controls was statistically significant (*p* < 0.0007). Wooster et al., [[55](#B55-jpm-11-01098)] reported that ACE2 rs4240157 polymorphisms is associated with COVID-19 disease severity as it might be inducing higher tissue specific expression of ACE2 resulting in the hospitalization of COVID-19 patients. Pouladi et al., [[62](#B62-jpm-11-01098)] reported the association ACE2 rs4240157 T > C gene polymorphism with hypertension and other related heart diseases. Our results indicated ACE2-CT genotype association with COVID-19 severity with an OR 2.18 (95% CI) (1.92–3.99), *p* < 0.010 and also ACE2-CC genotype was linked with COVID-19 severity with an OR 2.66 (95% CI) (1.53–4.62), *p* < 0.005 ([Table 4](#jpm-11-01098-t004)). We observed that ACE2-CC genotype was associated with increased COVID-19 mortality with OR 3.66 (95%) CI = (1.34 to 9.97), *p* < 0.011 ([Table 10](#jpm-11-01098-t010)). Similarly, in recessive inheritance model, ACE2-(TT + CT) genotype was associated with increased COVID-19 mortality with OR 5.0 (95%) CI = (2.0640–12.437), *p* < 0.004.

Despite the relatively small sample size, our results indicate that there is a close association between the ACE I/D gene polymorphism with clinical severity of COVID-19 disease. In addition, advanced age, coronary artery disease and diabetes were independent risk factors for the development of severe COVID-19 disease and mortality. The ACE-II genotype was a protective factor against the development of severe COVID-19 disease. Since this study is a first of its kind from Saudi Arabia, we believe that the relation of ACE2 DD genotype with the disease severity and clinical outcome in COVID-19 patients should be further investigated with more extensive studies. However, none of the studied SNPs are common in the general population. This finding might help in the recognition of people less and more prone to COVID-19.

## 5. Conclusions

It is concluded that ACE-DD genotype and D allele was strongly associated with increased COVID-19 disease severity. In addition, ACE I/D polymorphisms were strongly associated with advanced age, diabetes and coronary artery disease in COVID-19 patients whereas ACE-II genotype was a protective factor against the development of severe COVID-19. ACE2-DD genotype was strongly associated with increased SARS-CoV-2 mortality. Additionally, ACE2-CC and CT genotypes were strongly associated with SARS-CoV-2 severity. Therefore, our study might be useful for identifying the susceptible population groups for targeted therapeutic interventions and for making relevant public health policy decisions. The limitations of the study include a relatively smaller sample size of 117 patients and the fact that COVID-19 patients without symptoms were not included in the study.

## Acknowledgments

The authors extend their appreciation to the Deanship of Scientific Research at University of Bisha Saudi Arabia for funding this work through “COVID-19 Initiative Project’” under Grant Number (UB-COVID-02-1441). The authors highly appreciate the COVID-19 patients and healthy controls for their participation in this study. The facilities extended by universities of Bisha, Tabuk and King Khalid, Abha, KSA are gratefully acknowledged. Special thanks to Mohammad Jeelani and Rabia Farooq for their technical assistance.

## Author Contributions

Conceptualization, M.M.M., R.M., B.A.A. and M.A.A.A.; Data Curation, R.M., M.M.M., B.A.A., J.I.W.; Formal Analysis, M.M.M., R.M., M.A.A.A. and A.M.A.-S.; Investigation, M.M.M., R.M., B.A.A. and J.I.W.; Methodology, M.M.M., R.M. and M.A.A.A.; Project Administration, M.M.M. and M.A.A.A.; Resources, M.M.M., R.M., B.A.A., J.I.W., M.H.A. and M.A.A.A.; Validation, M.M.M., R.M. and M.A.A.A.; Visualization, M.M.M., B.A.A., M.A.A.A. and A.M.A.-S.; Writing—Original Draft, M.M.M. and R.M.; Writing—Review and Editing, M.M.M., R.M., B.A.A., J.I.W., M.A.A.A., M.H.A. and A.M.A.-S. All authors have read and agreed to the published version of the manuscript.

## Funding

COVID-19 initiative project from Deanship of Scientific Research at University of Bisha Saudi Arabia.

## Institutional Review Board Statement

The study was conducted according to the guidelines of the Declaration of Helsinki, and approved by the Institutional Ethics Committees of College of Medicine, University of Bisha, Bisha (Ref No. UBCOM/H-06-BH-087(05/25) Dated: 21/12/2020, University of Tabuk, Tabuk (Ref. No. UT-128-25-2020, Dated: 07/11/2020) and King Khalid University, Abha (Ref No. KKU-H-06-B-091, Dated: 12/03/2021).

## Informed Consent Statement

Informed consent was obtained from all subjects involved in the study.

## Conflicts of Interest

The authors declare no conflict of interest.

## Footnotes

## References

1. Wu F., Zhao S., Yu B., Chen Y.-M., Wang W., Song Z.-G., Hu Y., Tao Z.-W., Tian J.-H., Pei Y.-Y., et al. A new coronavirus associated with human respiratory disease in China. Nature. 2020;579:265–269. doi: 10.1038/s41586-020-2008-3.  [DOI](https://doi.org/10.1038/s41586-020-2008-3) | [PMC free article](/articles/PMC7094943/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32015508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20new%20coronavirus%20associated%20with%20human%20respiratory%20disease%20in%20China&author=F.%20Wu&author=S.%20Zhao&author=B.%20Yu&author=Y.-M.%20Chen&author=W.%20Wang&volume=579&publication_year=2020&pages=265-269&pmid=32015508&doi=10.1038/s41586-020-2008-3&)

2. Li Q., Guan X., Wu P., Wang X., Zhou L., Tong Y., Ren R., Leung K.S.M., Lau E.H.Y., Wong J.Y., et al. Early Transmission Dynamics in Wuhan, China, of Novel Coronavirus–Infected Pneumonia. N. Engl. J. Med. 2020;382:1199–1207. doi: 10.1056/NEJMoa2001316.  [DOI](https://doi.org/10.1056/NEJMoa2001316) | [PMC free article](/articles/PMC7121484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31995857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Early%20Transmission%20Dynamics%20in%20Wuhan,%20China,%20of%20Novel%20Coronavirus%E2%80%93Infected%20Pneumonia&author=Q.%20Li&author=X.%20Guan&author=P.%20Wu&author=X.%20Wang&author=L.%20Zhou&volume=382&publication_year=2020&pages=1199-1207&pmid=31995857&doi=10.1056/NEJMoa2001316&)

3. Huang C., Wang Y., Li X., Ren L., Zhao J., Hu Y., Zhang L., Fan G., Xu J., Gu X., et al. Clinical features of patients infected with 2019 novel coronavirus in Wuhan, China. Lancet. 2020;395:497–506. doi: 10.1016/S0140-6736(20)30183-5.  [DOI](https://doi.org/10.1016/S0140-6736(20)30183-5) | [PMC free article](/articles/PMC7159299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31986264/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Clinical%20features%20of%20patients%20infected%20with%202019%20novel%20coronavirus%20in%20Wuhan,%20China&author=C.%20Huang&author=Y.%20Wang&author=X.%20Li&author=L.%20Ren&author=J.%20Zhao&volume=395&publication_year=2020&pages=497-506&pmid=31986264&doi=10.1016/S0140-6736(20)30183-5&)

4. Zhou P., Yang X.-L., Wang X.-G., Hu B., Zhang L., Zhang W., Si H.-R., Zhu Y., Li B., Huang C.-L., et al. A pneumonia outbreak associated with a new coronavirus of probable bat origin. Nature. 2020;579:270–273. doi: 10.1038/s41586-020-2012-7.  [DOI](https://doi.org/10.1038/s41586-020-2012-7) | [PMC free article](/articles/PMC7095418/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32015507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20pneumonia%20outbreak%20associated%20with%20a%20new%20coronavirus%20of%20probable%20bat%20origin&author=P.%20Zhou&author=X.-L.%20Yang&author=X.-G.%20Wang&author=B.%20Hu&author=L.%20Zhang&volume=579&publication_year=2020&pages=270-273&pmid=32015507&doi=10.1038/s41586-020-2012-7&)

5. Bajgain K.T., Badal S., Bajgain B.B., Santana M.J. Prevalence of comorbidities among individuals with COVID-19: A rapid review of current literature. Am. J. Infect. Control. 2020;49:238–246. doi: 10.1016/j.ajic.2020.06.213.  [DOI](https://doi.org/10.1016/j.ajic.2020.06.213) | [PMC free article](/articles/PMC7351042/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32659414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Infect.%20Control&title=Prevalence%20of%20comorbidities%20among%20individuals%20with%20COVID-19:%20A%20rapid%20review%20of%20current%20literature&author=K.T.%20Bajgain&author=S.%20Badal&author=B.B.%20Bajgain&author=M.J.%20Santana&volume=49&publication_year=2020&pages=238-246&pmid=32659414&doi=10.1016/j.ajic.2020.06.213&)

6. Gasmi A., Peana M., Pivina L., Srinath S., Benahmed A.G., Semenova Y., Menzel A., Dadar M., Bjørklund G. Interrelations between COVID-19 and other disorders. Clin. Immunol. 2021;224:108651. doi: 10.1016/j.clim.2020.108651.  [DOI](https://doi.org/10.1016/j.clim.2020.108651) | [PMC free article](/articles/PMC7833539/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33333255/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Immunol.&title=Interrelations%20between%20COVID-19%20and%20other%20disorders&author=A.%20Gasmi&author=M.%20Peana&author=L.%20Pivina&author=S.%20Srinath&author=A.G.%20Benahmed&volume=224&publication_year=2021&pages=108651&pmid=33333255&doi=10.1016/j.clim.2020.108651&)

7. Poorolajal J. Geographical Distribution of COVID-19 Cases and Deaths Worldwide. J. Res. Heal. Sci. 2020;20:e00483. doi: 10.34172/jrhs.2020.24.  [DOI](https://doi.org/10.34172/jrhs.2020.24) | [PMC free article](/articles/PMC7585770/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33169715/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Res.%20Heal.%20Sci.&title=Geographical%20Distribution%20of%20COVID-19%20Cases%20and%20Deaths%20Worldwide&author=J.%20Poorolajal&volume=20&publication_year=2020&pages=e00483&pmid=33169715&doi=10.34172/jrhs.2020.24&)

8. CDC COVID-19 Response Team. Bialek S., Bowen V., Chow N., Curns A., Gierke R., Hall A., Hughes M., Pilishvili T., Rithey M., et al. Geographic Differences in COVID-19 Cases, Deaths, and Incidence—United States, February 12–April 7, 2020. MMWR Morb. Mortal. Wkly. Rep. 2020;69:465–471. doi: 10.15585/mmwr.mm6915e4.  [DOI](https://doi.org/10.15585/mmwr.mm6915e4) | [PMC free article](/articles/PMC7755058/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32298250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=MMWR%20Morb.%20Mortal.%20Wkly.%20Rep.&title=Geographic%20Differences%20in%20COVID-19%20Cases,%20Deaths,%20and%20Incidence%E2%80%94United%20States,%20February%2012%E2%80%93April%207,%202020&author=S.%20Bialek&author=V.%20Bowen&author=N.%20Chow&author=A.%20Curns&author=R.%20Gierke&volume=69&publication_year=2020&pages=465-471&pmid=32298250&doi=10.15585/mmwr.mm6915e4&)

9. Sorci G., Faivre B., Morand S. Explaining among-country variation in COVID-19 case fatality rate. Sci. Rep. 2020;10:1–11. doi: 10.1038/s41598-020-75848-2.  [DOI](https://doi.org/10.1038/s41598-020-75848-2) | [PMC free article](/articles/PMC7609641/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33144595/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Explaining%20among-country%20variation%20in%20COVID-19%20case%20fatality%20rate&author=G.%20Sorci&author=B.%20Faivre&author=S.%20Morand&volume=10&publication_year=2020&pages=1-11&pmid=33144595&doi=10.1038/s41598-020-75848-2&)

10. Weiss S.R., Navas-Martin S. Coronavirus Pathogenesis and the Emerging Pathogen Severe Acute Respiratory Syndrome Coronavirus. Microbiol. Mol. Biol. Rev. 2005;69:635–664. doi: 10.1128/MMBR.69.4.635-664.2005.  [DOI](https://doi.org/10.1128/MMBR.69.4.635-664.2005) | [PMC free article](/articles/PMC1306801/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16339739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Mol.%20Biol.%20Rev.&title=Coronavirus%20Pathogenesis%20and%20the%20Emerging%20Pathogen%20Severe%20Acute%20Respiratory%20Syndrome%20Coronavirus&author=S.R.%20Weiss&author=S.%20Navas-Martin&volume=69&publication_year=2005&pages=635-664&pmid=16339739&doi=10.1128/MMBR.69.4.635-664.2005&)

11. Hadi-Alijanvand H., Rouhani M. Studying the Effects of ACE2 Mutations on the Stability, Dynamics, and Dissociation Process of SARS-CoV-2 S1/hACE2 Complexes. J. Proteome Res. 2020;19:4609–4623. doi: 10.1021/acs.jproteome.0c00348.  [DOI](https://doi.org/10.1021/acs.jproteome.0c00348) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32786692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&title=Studying%20the%20Effects%20of%20ACE2%20Mutations%20on%20the%20Stability,%20Dynamics,%20and%20Dissociation%20Process%20of%20SARS-CoV-2%20S1/hACE2%20Complexes&author=H.%20Hadi-Alijanvand&author=M.%20Rouhani&volume=19&publication_year=2020&pages=4609-4623&pmid=32786692&doi=10.1021/acs.jproteome.0c00348&)

12. Li F. Structure, Function, and Evolution of Coronavirus Spike Proteins. Annu. Rev. Virol. 2016;3:237–261. doi: 10.1146/annurev-virology-110615-042301.  [DOI](https://doi.org/10.1146/annurev-virology-110615-042301) | [PMC free article](/articles/PMC5457962/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27578435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Virol.&title=Structure,%20Function,%20and%20Evolution%20of%20Coronavirus%20Spike%20Proteins&author=F.%20Li&volume=3&publication_year=2016&pages=237-261&pmid=27578435&doi=10.1146/annurev-virology-110615-042301&)

13. Wan Y., Shang J., Graham R., Baric R.S., Li F. Receptor Recognition by the Novel Coronavirus from Wuhan: An Analysis Based on Decade-Long Structural Studies of SARS Coronavirus. J. Virol. 2020;94:e00127-20. doi: 10.1128/JVI.00127-20.  [DOI](https://doi.org/10.1128/JVI.00127-20) | [PMC free article](/articles/PMC7081895/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31996437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Virol.&title=Receptor%20Recognition%20by%20the%20Novel%20Coronavirus%20from%20Wuhan:%20An%20Analysis%20Based%20on%20Decade-Long%20Structural%20Studies%20of%20SARS%20Coronavirus&author=Y.%20Wan&author=J.%20Shang&author=R.%20Graham&author=R.S.%20Baric&author=F.%20Li&volume=94&publication_year=2020&pages=e00127-20&pmid=31996437&doi=10.1128/JVI.00127-20&)

14. Li W., Moore M.J., Vasilieva N., Sui J., Wong S.K., Berne M.A., Somasundaran M., Sullivan J.L., Luzuriaga K., Greenough T.C., et al. Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature. 2003;426:450–454. doi: 10.1038/nature02145.  [DOI](https://doi.org/10.1038/nature02145) | [PMC free article](/articles/PMC7095016/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14647384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Angiotensin-converting%20enzyme%202%20is%20a%20functional%20receptor%20for%20the%20SARS%20coronavirus&author=W.%20Li&author=M.J.%20Moore&author=N.%20Vasilieva&author=J.%20Sui&author=S.K.%20Wong&volume=426&publication_year=2003&pages=450-454&pmid=14647384&doi=10.1038/nature02145&)

15. Kuba K., Imai Y., Rao S., Gao H., Guo F., Guan B., Huan Y., Yang P., Zhang Y., Deng W., et al. A crucial role of angiotensin converting enzyme 2 (ACE2) in SARS coronavirus-induced lung injury. Nat. Med. 2005;11:875–879. doi: 10.1038/nm1267.  [DOI](https://doi.org/10.1038/nm1267) | [PMC free article](/articles/PMC7095783/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16007097/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&title=A%20crucial%20role%20of%20angiotensin%20converting%20enzyme%202%20(ACE2)%20in%20SARS%20coronavirus-induced%20lung%20injury&author=K.%20Kuba&author=Y.%20Imai&author=S.%20Rao&author=H.%20Gao&author=F.%20Guo&volume=11&publication_year=2005&pages=875-879&pmid=16007097&doi=10.1038/nm1267&)

16. Hoffmann M., Kleine-Weber H., Schroeder S., Krüger N., Herrler T., Erichsen S., Schiergens T.S., Herrler G., Wu N.H., Nitsche A., et al. SARS-CoV-2 cell entry depends on ACE2 and TMPRSS2 and is blocked by a clinically proven protease inhibitor. Cell. 2020;181:271–280. doi: 10.1016/j.cell.2020.02.052.  [DOI](https://doi.org/10.1016/j.cell.2020.02.052) | [PMC free article](/articles/PMC7102627/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32142651/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=SARS-CoV-2%20cell%20entry%20depends%20on%20ACE2%20and%20TMPRSS2%20and%20is%20blocked%20by%20a%20clinically%20proven%20protease%20inhibitor&author=M.%20Hoffmann&author=H.%20Kleine-Weber&author=S.%20Schroeder&author=N.%20Kr%C3%BCger&author=T.%20Herrler&volume=181&publication_year=2020&pages=271-280&pmid=32142651&doi=10.1016/j.cell.2020.02.052&)

17. Crackower M.A., Sarao R., Oudit G.Y., Yagil C., Kozieradzki I., Scanga S.E., Oliveira-Dos-Santos A.J., Da Costa J., Zhang L., Pei Y., et al. Angiotensin-converting enzyme 2 is an essential regulator of heart function. Nat. Cell Biol. 2002;417:822–828. doi: 10.1038/nature00786.  [DOI](https://doi.org/10.1038/nature00786) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12075344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol.&title=Angiotensin-converting%20enzyme%202%20is%20an%20essential%20regulator%20of%20heart%20function&author=M.A.%20Crackower&author=R.%20Sarao&author=G.Y.%20Oudit&author=C.%20Yagil&author=I.%20Kozieradzki&volume=417&publication_year=2002&pages=822-828&pmid=12075344&doi=10.1038/nature00786&)

18. Keidar S., Kaplan M., Gamliel-Lazarovich A. ACE2 of the heart: From angiotensin I to angiotensin (1–7) Cardiovasc. Res. 2007;73:463–469. doi: 10.1016/j.cardiores.2006.09.006.  [DOI](https://doi.org/10.1016/j.cardiores.2006.09.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17049503/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc.%20Res.&title=ACE2%20of%20the%20heart:%20From%20angiotensin%20I%20to%20angiotensin%20(1%E2%80%937)&author=S.%20Keidar&author=M.%20Kaplan&author=A.%20Gamliel-Lazarovich&volume=73&publication_year=2007&pages=463-469&pmid=17049503&doi=10.1016/j.cardiores.2006.09.006&)

19. Danilczyk U., Sarao R., Remy C., Benabbas C., Stange G., Richter A., Arya S., Pospisilik J.A., Singer D., Camargo S., et al. Essential role for collectrin in renal amino acid transport. Nat. Cell Biol. 2006;444:1088–1091. doi: 10.1038/nature05475.  [DOI](https://doi.org/10.1038/nature05475) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17167413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol.&title=Essential%20role%20for%20collectrin%20in%20renal%20amino%20acid%20transport&author=U.%20Danilczyk&author=R.%20Sarao&author=C.%20Remy&author=C.%20Benabbas&author=G.%20Stange&volume=444&publication_year=2006&pages=1088-1091&pmid=17167413&doi=10.1038/nature05475&)

20. Tipnis S.R., Hooper N.M., Hyde R., Karran E., Christie G., Turner A.J. A human homolog of angiotensin-converting enzyme. Cloning and functional expression as a captopril-insensitive carboxypeptidase. J. Biol. Chem. 2000;275:33238–33243. doi: 10.1074/jbc.M002615200.  [DOI](https://doi.org/10.1074/jbc.M002615200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10924499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=A%20human%20homolog%20of%20angiotensin-converting%20enzyme.%20Cloning%20and%20functional%20expression%20as%20a%20captopril-insensitive%20carboxypeptidase&author=S.R.%20Tipnis&author=N.M.%20Hooper&author=R.%20Hyde&author=E.%20Karran&author=G.%20Christie&volume=275&publication_year=2000&pages=33238-33243&pmid=10924499&doi=10.1074/jbc.M002615200&)

21. Pan Y., Wang T., Li Y., Guan T., Lai Y., Shen Y., Zeyaweiding A., Maimaiti T., Li F., Zhao H., et al. Association of ACE2 polymorphisms with susceptibility to essential hypertension and dyslipidemia in Xinjiang, China. Lipids Heal. Dis. 2018;17:1–9. doi: 10.1186/s12944-018-0890-6.  [DOI](https://doi.org/10.1186/s12944-018-0890-6) | [PMC free article](/articles/PMC6195726/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30342552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lipids%20Heal.%20Dis.&title=Association%20of%20ACE2%20polymorphisms%20with%20susceptibility%20to%20essential%20hypertension%20and%20dyslipidemia%20in%20Xinjiang,%20China&author=Y.%20Pan&author=T.%20Wang&author=Y.%20Li&author=T.%20Guan&author=Y.%20Lai&volume=17&publication_year=2018&pages=1-9&pmid=30342552&doi=10.1186/s12944-018-0890-6&)

22. Lu N., Yang Y., Wang Y., Liu Y., Fu G., Chen D., Dai H., Fan X., Hui R., Zheng Y. ACE2 gene polymorphism and essential hypertension: An updated meta-analysis involving 11,051 subjects. Mol. Biol. Rep. 2012;39:6581–6589. doi: 10.1007/s11033-012-1487-1.  [DOI](https://doi.org/10.1007/s11033-012-1487-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22297693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Rep.&title=ACE2%20gene%20polymorphism%20and%20essential%20hypertension:%20An%20updated%20meta-analysis%20involving%2011,051%20subjects&author=N.%20Lu&author=Y.%20Yang&author=Y.%20Wang&author=Y.%20Liu&author=G.%20Fu&volume=39&publication_year=2012&pages=6581-6589&pmid=22297693&doi=10.1007/s11033-012-1487-1&)

23. Magrone T., Magrone M., Jirillo E. Focus on Receptors for Coronaviruses with Special Reference to Angiotensin-Converting Enzyme 2 as a Potential Drug Target—A Perspective. Endocr. Metab. Immune Disord. Drug Targets. 2020;20:807–811. doi: 10.2174/1871530320666200427112902.  [DOI](https://doi.org/10.2174/1871530320666200427112902) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32338224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocr.%20Metab.%20Immune%20Disord.%20Drug%20Targets&title=Focus%20on%20Receptors%20for%20Coronaviruses%20with%20Special%20Reference%20to%20Angiotensin-Converting%20Enzyme%202%20as%20a%20Potential%20Drug%20Target%E2%80%94A%20Perspective&author=T.%20Magrone&author=M.%20Magrone&author=E.%20Jirillo&volume=20&publication_year=2020&pages=807-811&pmid=32338224&doi=10.2174/1871530320666200427112902&)

24. Saab Y.B., Gard P.R., Overall A.D.J. The geographic distribution of the ACE II genotype: A novel finding. Genet. Res. 2007;89:259–267. doi: 10.1017/S0016672307009019.  [DOI](https://doi.org/10.1017/S0016672307009019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18208631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Res.&title=The%20geographic%20distribution%20of%20the%20ACE%20II%20genotype:%20A%20novel%20finding&author=Y.B.%20Saab&author=P.R.%20Gard&author=A.D.J.%20Overall&volume=89&publication_year=2007&pages=259-267&pmid=18208631&doi=10.1017/S0016672307009019&)

25. Singh H., Choudhari R., Nema V., Khan A.A. ACE2 and TMPRSS2 polymorphisms in various diseases with special reference to its impact on COVID-19 disease. Microb. Pathog. 2021;150:104621. doi: 10.1016/j.micpath.2020.104621.  [DOI](https://doi.org/10.1016/j.micpath.2020.104621) | [PMC free article](/articles/PMC7709597/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33278516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microb.%20Pathog.&title=ACE2%20and%20TMPRSS2%20polymorphisms%20in%20various%20diseases%20with%20special%20reference%20to%20its%20impact%20on%20COVID-19%20disease&author=H.%20Singh&author=R.%20Choudhari&author=V.%20Nema&author=A.A.%20Khan&volume=150&publication_year=2021&pages=104621&pmid=33278516&doi=10.1016/j.micpath.2020.104621&)

26. Devaux C.A., Rolain J.M., Raoult D. ACE2 receptor polymorphism: Susceptibility to SARS-CoV-2, hypertension, multi-organ failure, and COVID-19 disease outcome. J. Microbiol. Immunol. Infect. 2020;53:425–435. doi: 10.1016/j.jmii.2020.04.015.  [DOI](https://doi.org/10.1016/j.jmii.2020.04.015) | [PMC free article](/articles/PMC7201239/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32414646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Microbiol.%20Immunol.%20Infect.&title=ACE2%20receptor%20polymorphism:%20Susceptibility%20to%20SARS-CoV-2,%20hypertension,%20multi-organ%20failure,%20and%20COVID-19%20disease%20outcome&author=C.A.%20Devaux&author=J.M.%20Rolain&author=D.%20Raoult&volume=53&publication_year=2020&pages=425-435&pmid=32414646&doi=10.1016/j.jmii.2020.04.015&)

27. Yi L., Gu Y.H., Wang X.L., An L.Z., Xie X.D., Shao W., Ma L.Y., Fang J.R., An Y.D., Wang F., et al. Association of ACE, ACE2 and UTS2 Polymorphisms with Essential Hypertension in Han and Dongxiang Populations from North-western China. J. Int. Med Res. 2006;34:272–283. doi: 10.1177/147323000603400306.  [DOI](https://doi.org/10.1177/147323000603400306) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16866021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Int.%20Med%20Res.&title=Association%20of%20ACE,%20ACE2%20and%20UTS2%20Polymorphisms%20with%20Essential%20Hypertension%20in%20Han%20and%20Dongxiang%20Populations%20from%20North-western%20China&author=L.%20Yi&author=Y.H.%20Gu&author=X.L.%20Wang&author=L.Z.%20An&author=X.D.%20Xie&volume=34&publication_year=2006&pages=272-283&pmid=16866021&doi=10.1177/147323000603400306&)

28. Niu W., Qi Y., Hou S., Zhou W., Qiu C. Correlation of angiotensin-converting enzyme 2 gene polymorphisms with stage 2 hypertension in Han Chinese. Transl. Res. 2007;150:374–380. doi: 10.1016/j.trsl.2007.06.002.  [DOI](https://doi.org/10.1016/j.trsl.2007.06.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18022600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transl.%20Res.&title=Correlation%20of%20angiotensin-converting%20enzyme%202%20gene%20polymorphisms%20with%20stage%202%20hypertension%20in%20Han%20Chinese&author=W.%20Niu&author=Y.%20Qi&author=S.%20Hou&author=W.%20Zhou&author=C.%20Qiu&volume=150&publication_year=2007&pages=374-380&pmid=18022600&doi=10.1016/j.trsl.2007.06.002&)

29. Malard L., Kakinami L., O’Loughlin J., Roy-Gagnon M.H., Labbe A., Pilote L., Hamet P., Tremblay J., Paradis G. The association between the angioten-sin-converting enzyme-2 gene and blood pressure in a cohort study of adolescents. BMC Med. Genet. 2013;14:117. doi: 10.1186/1471-2350-14-117.  [DOI](https://doi.org/10.1186/1471-2350-14-117) | [PMC free article](/articles/PMC4228362/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24191856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med.%20Genet.&title=The%20association%20between%20the%20angioten-sin-converting%20enzyme-2%20gene%20and%20blood%20pressure%20in%20a%20cohort%20study%20of%20adolescents&author=L.%20Malard&author=L.%20Kakinami&author=J.%20O%E2%80%99Loughlin&author=M.H.%20Roy-Gagnon&author=A.%20Labbe&volume=14&publication_year=2013&pages=117&pmid=24191856&doi=10.1186/1471-2350-14-117&)

30. Patnaik M., Pati P., Swain S.N., Mohapatra M.K., Dwibedi B., Kar S.K., Ranjit M. Association of angiotensin-converting enzyme and angiotensin-converting enzyme-2 gene polymorphisms withessential hypertension in the population of Odisha, India. Ann. Hum. Biol. 2014;41:145–152. doi: 10.3109/03014460.2013.837195.  [DOI](https://doi.org/10.3109/03014460.2013.837195) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24112034/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Hum.%20Biol.&title=Association%20of%20angiotensin-converting%20enzyme%20and%20angiotensin-converting%20enzyme-2%20gene%20polymorphisms%20withessential%20hypertension%20in%20the%20population%20of%20Odisha,%20India&author=M.%20Patnaik&author=P.%20Pati&author=S.N.%20Swain&author=M.K.%20Mohapatra&author=B.%20Dwibedi&volume=41&publication_year=2014&pages=145-152&pmid=24112034&doi=10.3109/03014460.2013.837195&)

31. Pinheiro D.S., Santos R.D.S., Jardim P.C.B.V., Silva E.G., Reis A., Pedrino G.R., Ulhoa C.J. The combination of ACE I/D and ACE2 G8790A polymorphisms revels susceptibility to hypertension: A genetic association study in Brazilian patients. PLoS ONE. 2019;14:e0221248. doi: 10.1371/journal.pone.0221248.  [DOI](https://doi.org/10.1371/journal.pone.0221248) | [PMC free article](/articles/PMC6701835/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31430320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=The%20combination%20of%20ACE%20I/D%20and%20ACE2%20G8790A%20polymorphisms%20revels%20susceptibility%20to%20hypertension:%20A%20genetic%20association%20study%20in%20Brazilian%20patients&author=D.S.%20Pinheiro&author=R.D.S.%20Santos&author=P.C.B.V.%20Jardim&author=E.G.%20Silva&author=A.%20Reis&volume=14&publication_year=2019&pages=e0221248&pmid=31430320&doi=10.1371/journal.pone.0221248&)

32. Zheng H., Cao J.J. Angiotensin-Converting Enzyme Gene Polymorphism and Severe Lung Injury in Patients with Coronavirus Disease 2019. Am. J. Pathol. 2020;190:2013–2017. doi: 10.1016/j.ajpath.2020.07.009.  [DOI](https://doi.org/10.1016/j.ajpath.2020.07.009) | [PMC free article](/articles/PMC7387924/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32735889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Pathol.&title=Angiotensin-Converting%20Enzyme%20Gene%20Polymorphism%20and%20Severe%20Lung%20Injury%20in%20Patients%20with%20Coronavirus%20Disease%202019&author=H.%20Zheng&author=J.J.%20Cao&volume=190&publication_year=2020&pages=2013-2017&pmid=32735889&doi=10.1016/j.ajpath.2020.07.009&)

33. Yancy C.W. COVID-19 and African Americans. JAMA. 2020;323:1891. doi: 10.1001/jama.2020.6548.  [DOI](https://doi.org/10.1001/jama.2020.6548) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32293639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=COVID-19%20and%20African%20Americans&author=C.W.%20Yancy&volume=323&publication_year=2020&pages=1891&pmid=32293639&doi=10.1001/jama.2020.6548&)

34. Dyer O. COVID-19: Black people and other minorities are hardest hit in US. BMJ. 2020;369:m1483. doi: 10.1136/bmj.m1483.  [DOI](https://doi.org/10.1136/bmj.m1483) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32291262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=COVID-19:%20Black%20people%20and%20other%20minorities%20are%20hardest%20hit%20in%20US&author=O.%20Dyer&volume=369&publication_year=2020&pages=m1483&pmid=32291262&doi=10.1136/bmj.m1483&)

35. Sabir J.S., El Omri A., Khan I.A., Banaganapalli B., Hajrah N.H., Zrelli H., Omar A.M.S., Alharbi M.G., Alhebshi A.M., Jansen R.K., et al. ACE insertion/deletion genetic polymorphism, serum ACE levels and high dietary salt intake influence the risk of obesity development among the Saudi adult population. J. Renin-Angiotensin-Aldosterone Syst. 2019;20:1470320319870945. doi: 10.1177/1470320319870945.  [DOI](https://doi.org/10.1177/1470320319870945) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Renin-Angiotensin-Aldosterone%20Syst.&title=ACE%20insertion/deletion%20genetic%20polymorphism,%20serum%20ACE%20levels%20and%20high%20dietary%20salt%20intake%20influence%20the%20risk%20of%20obesity%20development%20among%20the%20Saudi%20adult%20population&author=J.S.%20Sabir&author=A.%20El%20Omri&author=I.A.%20Khan&author=B.%20Banaganapalli&author=N.H.%20Hajrah&volume=20&publication_year=2019&pages=1470320319870945&doi=10.1177/1470320319870945&)

36. Aung A.K., Aitken T., Teh B.M., Yu C., Ofori-Asenso R., Chin K.L., Liew D. Angiotensin converting enzyme genotypes and mortality from COVID-19: An ecological study. J. Infect. 2020;81:961–965. doi: 10.1016/j.jinf.2020.11.012.  [DOI](https://doi.org/10.1016/j.jinf.2020.11.012) | [PMC free article](/articles/PMC7666537/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33197472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Infect.&title=Angiotensin%20converting%20enzyme%20genotypes%20and%20mortality%20from%20COVID-19:%20An%20ecological%20study&author=A.K.%20Aung&author=T.%20Aitken&author=B.M.%20Teh&author=C.%20Yu&author=R.%20Ofori-Asenso&volume=81&publication_year=2020&pages=961-965&pmid=33197472&doi=10.1016/j.jinf.2020.11.012&)

37. Marshall R.P., Webb S., Bellingan G.J., Montgomery H.E., Chaudhari B., McAnulty R.J., Humphries S.E., Hill M.R., Laurent G.J. An-giotensin converting enzyme insertion/deletion polymorphism is associated with susceptibility and outcome in acute res-piratory distress syndrome. Am. J. Respir. Crit. Care Med. 2002;166:646–650. doi: 10.1164/rccm.2108086.  [DOI](https://doi.org/10.1164/rccm.2108086) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12204859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Respir.%20Crit.%20Care%20Med.&title=An-giotensin%20converting%20enzyme%20insertion/deletion%20polymorphism%20is%20associated%20with%20susceptibility%20and%20outcome%20in%20acute%20res-piratory%20distress%20syndrome&author=R.P.%20Marshall&author=S.%20Webb&author=G.J.%20Bellingan&author=H.E.%20Montgomery&author=B.%20Chaudhari&volume=166&publication_year=2002&pages=646-650&pmid=12204859&doi=10.1164/rccm.2108086&)

38. Gómez J., Albaiceta G.M., García-Clemente M., López-Larrea C., Amado-Rodríguez L., Lopez-Alonso I., Hermida T., Enriquez A.I., Herrero P., Melón S., et al. Angiotensin-converting enzymes (ACE, ACE2) gene variants and COVID-19 outcome. Gene. 2020;762:145102. doi: 10.1016/j.gene.2020.145102.  [DOI](https://doi.org/10.1016/j.gene.2020.145102) | [PMC free article](/articles/PMC7456966/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32882331/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Angiotensin-converting%20enzymes%20(ACE,%20ACE2)%20gene%20variants%20and%20COVID-19%20outcome&author=J.%20G%C3%B3mez&author=G.M.%20Albaiceta&author=M.%20Garc%C3%ADa-Clemente&author=C.%20L%C3%B3pez-Larrea&author=L.%20Amado-Rodr%C3%ADguez&volume=762&publication_year=2020&pages=145102&pmid=32882331&doi=10.1016/j.gene.2020.145102&)

39. Khamlaoui W., Mehri S., Hammami S., Elosua R., Hammami M. Association of angiotensin-converting enzyme insertion/deletion (ACE I/D) and angiotensinogen (AGT M235T) polymorphisms with the risk of obesity in a Tunisian population. J. Renin-Angiotensin-Aldosterone Syst. 2020;21:1470320320907820. doi: 10.1177/1470320320907820.  [DOI](https://doi.org/10.1177/1470320320907820) | [PMC free article](/articles/PMC7227147/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32356512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Renin-Angiotensin-Aldosterone%20Syst.&title=Association%20of%20angiotensin-converting%20enzyme%20insertion/deletion%20(ACE%20I/D)%20and%20angiotensinogen%20(AGT%20M235T)%20polymorphisms%20with%20the%20risk%20of%20obesity%20in%20a%20Tunisian%20population&author=W.%20Khamlaoui&author=S.%20Mehri&author=S.%20Hammami&author=R.%20Elosua&author=M.%20Hammami&volume=21&publication_year=2020&pages=1470320320907820&pmid=32356512&doi=10.1177/1470320320907820&)

40. Yamamoto N., Ariumi Y., Nishida N., Yamamoto R., Bauer G., Gojobori T., Shimotohno K., Mizokami M. SARS-CoV-2 infections and COVID-19 mortalities strongly correlate with ACE1 I/D genotype. Gene. 2020;758:144944. doi: 10.1016/j.gene.2020.144944.  [DOI](https://doi.org/10.1016/j.gene.2020.144944) | [PMC free article](/articles/PMC7833925/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32628976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=SARS-CoV-2%20infections%20and%20COVID-19%20mortalities%20strongly%20correlate%20with%20ACE1%20I/D%20genotype&author=N.%20Yamamoto&author=Y.%20Ariumi&author=N.%20Nishida&author=R.%20Yamamoto&author=G.%20Bauer&volume=758&publication_year=2020&pages=144944&pmid=32628976&doi=10.1016/j.gene.2020.144944&)

41. Sun P., Lu X., Xu C., Sun W., Pan B. Understanding of COVID-19 based on current evidence. J. Med. Virol. 2020;92:548–551. doi: 10.1002/jmv.25722.  [DOI](https://doi.org/10.1002/jmv.25722) | [PMC free article](/articles/PMC7228250/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32096567/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Virol.&title=Understanding%20of%20COVID-19%20based%20on%20current%20evidence&author=P.%20Sun&author=X.%20Lu&author=C.%20Xu&author=W.%20Sun&author=B.%20Pan&volume=92&publication_year=2020&pages=548-551&pmid=32096567&doi=10.1002/jmv.25722&)

42. Hatami N., Ahi S., Sadeghinikoo A., Foroughian M., Javdani F., Kalani N., Fereydoni M., Keshavarz P. Worldwide ACE (I/D) polymorphism may affect COVID-19 recovery rate: An ecological meta-regression. Endocrine. 2020;68:479–484. doi: 10.1007/s12020-020-02381-7.  [DOI](https://doi.org/10.1007/s12020-020-02381-7) | [PMC free article](/articles/PMC7294766/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32542429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocrine&title=Worldwide%20ACE%20(I/D)%20polymorphism%20may%20affect%20COVID-19%20recovery%20rate:%20An%20ecological%20meta-regression&author=N.%20Hatami&author=S.%20Ahi&author=A.%20Sadeghinikoo&author=M.%20Foroughian&author=F.%20Javdani&volume=68&publication_year=2020&pages=479-484&pmid=32542429&doi=10.1007/s12020-020-02381-7&)

43. Gwathmey T.M., Shaltout H.A., Nixon P.A., O’Shea T.M., Rose J.C., Washburn L.K. Gender differences in urinary ACE and ACE2 activities in adolescents. FASEB J. 2008;22:940. doi: 10.1096/fasebj.22.1_supplement.940.6.  [DOI](https://doi.org/10.1096/fasebj.22.1_supplement.940.6) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FASEB%20J.&title=Gender%20differences%20in%20urinary%20ACE%20and%20ACE2%20activities%20in%20adolescents&author=T.M.%20Gwathmey&author=H.A.%20Shaltout&author=P.A.%20Nixon&author=T.M.%20O%E2%80%99Shea&author=J.C.%20Rose&volume=22&publication_year=2008&pages=940&doi=10.1096/fasebj.22.1_supplement.940.6&)

44. Burrell L.M., Harrap S.B., Velkoska E., Patel S.K. The ACE2 gene: Its potential as a functional candidate for cardiovascular disease. Clin. Sci. (Lond.) 2013;124:65–76. doi: 10.1042/CS20120269.  [DOI](https://doi.org/10.1042/CS20120269) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23013041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Sci.%20(Lond.)&title=The%20ACE2%20gene:%20Its%20potential%20as%20a%20functional%20candidate%20for%20cardiovascular%20disease&author=L.M.%20Burrell&author=S.B.%20Harrap&author=E.%20Velkoska&author=S.K.%20Patel&volume=124&publication_year=2013&pages=65-76&pmid=23013041&doi=10.1042/CS20120269&)

45. Gemmati D., Bramanti B., Serino M.L., Secchiero P., Zauli G., Tisato V. COVID-19 and individual genetic susceptibility/receptivity: Role of ACE1/ACE2 genes, immunity, inflammation and coagulation. Might the double X-chromosome in females be protective against SARS-CoV-2 compared to the single X-chromosome in males? Int. J. Mol. Sci. 2020;21:3474. doi: 10.3390/ijms21103474.  [DOI](https://doi.org/10.3390/ijms21103474) | [PMC free article](/articles/PMC7278991/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32423094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=COVID-19%20and%20individual%20genetic%20susceptibility/receptivity:%20Role%20of%20ACE1/ACE2%20genes,%20immunity,%20inflammation%20and%20coagulation.%20Might%20the%20double%20X-chromosome%20in%20females%20be%20protective%20against%20SARS-CoV-2%20compared%20to%20the%20single%20X-chromosome%20in%20males?&author=D.%20Gemmati&author=B.%20Bramanti&author=M.L.%20Serino&author=P.%20Secchiero&author=G.%20Zauli&volume=21&publication_year=2020&pages=3474&pmid=32423094&doi=10.3390/ijms21103474&)

46. Villard E., Soubrier F. Molecular biology and genetics of the angiotensin-I-converting enzyme: Potential implications in cardiovascular diseases. Cardiovasc. Res. 1996;32:999–1007. doi: 10.1016/S0008-6363(96)00170-8.  [DOI](https://doi.org/10.1016/S0008-6363(96)00170-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9015402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc.%20Res.&title=Molecular%20biology%20and%20genetics%20of%20the%20angiotensin-I-converting%20enzyme:%20Potential%20implications%20in%20cardiovascular%20diseases&author=E.%20Villard&author=F.%20Soubrier&volume=32&publication_year=1996&pages=999-1007&pmid=9015402&doi=10.1016/S0008-6363(96)00170-8&)

47. Jeong K.-H., Lee T.-W., Ihm C.-G., Lee S.-H., Moon J.-Y. Polymorphisms in two genes, IL-1B and ACE, are associated with erythropoietin resistance in Korean patients on maintenance hemodialysis. Exp. Mol. Med. 2008;40:161–166. doi: 10.3858/emm.2008.40.2.161.  [DOI](https://doi.org/10.3858/emm.2008.40.2.161) | [PMC free article](/articles/PMC2679299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18446054/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp.%20Mol.%20Med.&title=Polymorphisms%20in%20two%20genes,%20IL-1B%20and%20ACE,%20are%20associated%20with%20erythropoietin%20resistance%20in%20Korean%20patients%20on%20maintenance%20hemodialysis&author=K.-H.%20Jeong&author=T.-W.%20Lee&author=C.-G.%20Ihm&author=S.-H.%20Lee&author=J.-Y.%20Moon&volume=40&publication_year=2008&pages=161-166&pmid=18446054&doi=10.3858/emm.2008.40.2.161&)

48. Itoyama S., Keicho N., Quy T., Phi N.C., Long H.T., Ha L.D., Van Ban V., Ohashi J., Hijikata M., Matsushita I., et al. ACE1 polymorphism and progression of SARS. Biochem. Biophys. Res. Commun. 2004;323:1124–1129. doi: 10.1016/j.bbrc.2004.08.208.  [DOI](https://doi.org/10.1016/j.bbrc.2004.08.208) | [PMC free article](/articles/PMC7092806/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15381116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&title=ACE1%20polymorphism%20and%20progression%20of%20SARS&author=S.%20Itoyama&author=N.%20Keicho&author=T.%20Quy&author=N.C.%20Phi&author=H.T.%20Long&volume=323&publication_year=2004&pages=1124-1129&pmid=15381116&doi=10.1016/j.bbrc.2004.08.208&)

49. Mathew J., Basheeruddin K., Prabhakar S. Differences in frequency of the deletion polymorphism of the angiotensin-converting enzyme gene in different ethnic groups. Angiology. 2001;52:375–379. doi: 10.1177/000331970105200602.  [DOI](https://doi.org/10.1177/000331970105200602) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11437027/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Angiology&title=Differences%20in%20frequency%20of%20the%20deletion%20polymorphism%20of%20the%20angiotensin-converting%20enzyme%20gene%20in%20different%20ethnic%20groups&author=J.%20Mathew&author=K.%20Basheeruddin&author=S.%20Prabhakar&volume=52&publication_year=2001&pages=375-379&pmid=11437027&doi=10.1177/000331970105200602&)

50. Lee Y.-J., Tsai J.C. ACE gene insertion/deletion polymorphism associated with 1998 World Health Organization definition of metabolic syndrome in Chinese type 2 diabetic patients. Diabetes Care. 2002;25:1002–1008. doi: 10.2337/diacare.25.6.1002.  [DOI](https://doi.org/10.2337/diacare.25.6.1002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12032106/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes%20Care&title=ACE%20gene%20insertion/deletion%20polymorphism%20associated%20with%201998%20World%20Health%20Organization%20definition%20of%20metabolic%20syndrome%20in%20Chinese%20type%202%20diabetic%20patients&author=Y.-J.%20Lee&author=J.C.%20Tsai&volume=25&publication_year=2002&pages=1002-1008&pmid=12032106&doi=10.2337/diacare.25.6.1002&)

51. Huang T., Shu Y., Cai Y.-D. Genetic differences among ethnic groups. BMC Genom. 2015;16:1093. doi: 10.1186/s12864-015-2328-0.  [DOI](https://doi.org/10.1186/s12864-015-2328-0) | [PMC free article](/articles/PMC4687076/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26690364/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Genom.&title=Genetic%20differences%20among%20ethnic%20groups&author=T.%20Huang&author=Y.%20Shu&author=Y.-D.%20Cai&volume=16&publication_year=2015&pages=1093&pmid=26690364&doi=10.1186/s12864-015-2328-0&)

52. Pati A., Mahto H., Padhi S., Panda A.K. ACE deletion allele is associated with susceptibility to SARS-CoV-2 infection and mortality rate: An epidemiological study in the Asian population. Clin. Chim. Acta. 2020;510:455–458. doi: 10.1016/j.cca.2020.08.008.  [DOI](https://doi.org/10.1016/j.cca.2020.08.008) | [PMC free article](/articles/PMC7416133/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32791137/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Chim.%20Acta&title=ACE%20deletion%20allele%20is%20associated%20with%20susceptibility%20to%20SARS-CoV-2%20infection%20and%20mortality%20rate:%20An%20epidemiological%20study%20in%20the%20Asian%20population&author=A.%20Pati&author=H.%20Mahto&author=S.%20Padhi&author=A.K.%20Panda&volume=510&publication_year=2020&pages=455-458&pmid=32791137&doi=10.1016/j.cca.2020.08.008&)

53. Sanyaolu A., Okorie C., Marinkovic A., Patidar R., Younis K., Desai P., Hosein Z., Padda I., Mangat J., Altaf M. Comorbidity and its Impact on Patients with COVID-19. SN Compr. Clin. Med. 2020:1–8. doi: 10.1007/s42399-020-00363-4.  [DOI](https://doi.org/10.1007/s42399-020-00363-4) | [PMC free article](/articles/PMC7314621/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32838147/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=SN%20Compr.%20Clin.%20Med.&title=Comorbidity%20and%20its%20Impact%20on%20Patients%20with%20COVID-19&author=A.%20Sanyaolu&author=C.%20Okorie&author=A.%20Marinkovic&author=R.%20Patidar&author=K.%20Younis&publication_year=2020&pages=1-8&pmid=32838147&doi=10.1007/s42399-020-00363-4&)

54. Krishnan R., Sekar D., Karunanithy S., Subramanium S. Association of angiotensin converting enzyme gene insertion/deletion polymorphism with essential hypertension in south Indian population. Genes Dis. 2016;3:159–163. doi: 10.1016/j.gendis.2016.03.001.  [DOI](https://doi.org/10.1016/j.gendis.2016.03.001) | [PMC free article](/articles/PMC6146176/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30258884/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Dis.&title=Association%20of%20angiotensin%20converting%20enzyme%20gene%20insertion/deletion%20polymorphism%20with%20essential%20hypertension%20in%20south%20Indian%20population&author=R.%20Krishnan&author=D.%20Sekar&author=S.%20Karunanithy&author=S.%20Subramanium&volume=3&publication_year=2016&pages=159-163&pmid=30258884&doi=10.1016/j.gendis.2016.03.001&)

55. Wooster L., Nicholson C.J., Sigurslid H.H., Cardenas C.L.L., Malhotra R. Polymorphisms in the ACE2 locus associate with severity of COVID-19 infection. medRxiv. 2020 doi: 10.1101/2020.06.18.20135152.  [DOI](https://doi.org/10.1101/2020.06.18.20135152) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=medRxiv&title=Polymorphisms%20in%20the%20ACE2%20locus%20associate%20with%20severity%20of%20COVID-19%20infection&author=L.%20Wooster&author=C.J.%20Nicholson&author=H.H.%20Sigurslid&author=C.L.L.%20Cardenas&author=R.%20Malhotra&publication_year=2020&doi=10.1101/2020.06.18.20135152&)

56. Feng Y., Niu T., Xu X., Chen C., Li Q., Qian R., Wang G., Xu X. Insertion/deletion polymorphism of the ACE gene is associated with type 2 diabetes. Diabetes. 2002;51:1986–1988. doi: 10.2337/diabetes.51.6.1986.  [DOI](https://doi.org/10.2337/diabetes.51.6.1986) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12031990/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes&title=Insertion/deletion%20polymorphism%20of%20the%20ACE%20gene%20is%20associated%20with%20type%202%20diabetes&author=Y.%20Feng&author=T.%20Niu&author=X.%20Xu&author=C.%20Chen&author=Q.%20Li&volume=51&publication_year=2002&pages=1986-1988&pmid=12031990&doi=10.2337/diabetes.51.6.1986&)

57. Amara A., Mrad M., Sayeh A., Lahideb D., Layouni S., Haggui A., Fekih-Mrissa N., Haouala H., Nsiri B. The Effect of ACE I/D Polymorphisms Alone and With Concomitant Risk Factors on Coronary Artery Disease. Clin. Appl. Thromb. 2018;24:157–163. doi: 10.1177/1076029616679505.  [DOI](https://doi.org/10.1177/1076029616679505) | [PMC free article](/articles/PMC6714628/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27895197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Appl.%20Thromb.&title=The%20Effect%20of%20ACE%20I/D%20Polymorphisms%20Alone%20and%20With%20Concomitant%20Risk%20Factors%20on%20Coronary%20Artery%20Disease&author=A.%20Amara&author=M.%20Mrad&author=A.%20Sayeh&author=D.%20Lahideb&author=S.%20Layouni&volume=24&publication_year=2018&pages=157-163&pmid=27895197&doi=10.1177/1076029616679505&)

58. Wu Z., McGoogan J.M. Characteristics of and important lessons from the Coronavirus Disease 2019 (COVID-19) outbreak in China: Summary of a report of 72,314 cases from the Chinese center for disease control and prevention. JAMA. 2020;323:1239. doi: 10.1001/jama.2020.2648.  [DOI](https://doi.org/10.1001/jama.2020.2648) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32091533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Characteristics%20of%20and%20important%20lessons%20from%20the%20Coronavirus%20Disease%202019%20(COVID-19)%20outbreak%20in%20China:%20Summary%20of%20a%20report%20of%2072,314%20cases%20from%20the%20Chinese%20center%20for%20disease%20control%20and%20prevention&author=Z.%20Wu&author=J.M.%20McGoogan&volume=323&publication_year=2020&pages=1239&pmid=32091533&doi=10.1001/jama.2020.2648&)

59. Patel S.K., Wai B., Ord M., MacIsaac R.J., Grant S., Velkoska E., Panagiotopoulos S., Jerums G., Srivastava P.M., Burrell L.M. Association of ACE2 genetic variants with blood pressure, left ventricular mass, and cardiac function in Caucasians with type 2 diabetes. Am. J. Hypertens. 2012;25:216–222. doi: 10.1038/ajh.2011.188.  [DOI](https://doi.org/10.1038/ajh.2011.188) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21993363/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hypertens.&title=Association%20of%20ACE2%20genetic%20variants%20with%20blood%20pressure,%20left%20ventricular%20mass,%20and%20cardiac%20function%20in%20Caucasians%20with%20type%202%20diabetes&author=S.K.%20Patel&author=B.%20Wai&author=M.%20Ord&author=R.J.%20MacIsaac&author=S.%20Grant&volume=25&publication_year=2012&pages=216-222&pmid=21993363&doi=10.1038/ajh.2011.188&)

60. Liu C., Li Y., Guan T., Lai Y., Shen Y., Zeyaweiding A., Zhao H., Li F., Maimaiti T. ACE2 polymorphisms associated with cardiovascular risk in Uygurs with type 2 diabetes mellitus. Cardiovasc. Diabetol. 2018;17:1–11. doi: 10.1186/s12933-018-0771-3.  [DOI](https://doi.org/10.1186/s12933-018-0771-3) | [PMC free article](/articles/PMC6142339/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30227878/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cardiovasc.%20Diabetol.&title=ACE2%20polymorphisms%20associated%20with%20cardiovascular%20risk%20in%20Uygurs%20with%20type%202%20diabetes%20mellitus&author=C.%20Liu&author=Y.%20Li&author=T.%20Guan&author=Y.%20Lai&author=Y.%20Shen&volume=17&publication_year=2018&pages=1-11&pmid=30227878&doi=10.1186/s12933-018-0771-3&)

61. Cangiano B., Fatti L.M., Danesi L., Gazzano G., Croci M., Vitale G., Bonomi M. Mortality in an Italian nursing home during COVID-19 pandemic: Correlation with gender, age, ADL, vitamin D supplementation, and limitations of the diagnostic tests. Aging (Albany NY) 2020;12:24522. doi: 10.18632/aging.202307.  [DOI](https://doi.org/10.18632/aging.202307) | [PMC free article](/articles/PMC7803543/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33353888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Aging%20(Albany%20NY)&title=Mortality%20in%20an%20Italian%20nursing%20home%20during%20COVID-19%20pandemic:%20Correlation%20with%20gender,%20age,%20ADL,%20vitamin%20D%20supplementation,%20and%20limitations%20of%20the%20diagnostic%20tests&author=B.%20Cangiano&author=L.M.%20Fatti&author=L.%20Danesi&author=G.%20Gazzano&author=M.%20Croci&volume=12&publication_year=2020&pages=24522&pmid=33353888&doi=10.18632/aging.202307&)

62. Pouladi N., Abdolahi S. Investigating the ACE2 polymorphisms in COVID-19 susceptibility: An in silico analysis. Mol. Genet. Genom. Med. 2021;9:e1672. doi: 10.1002/mgg3.1672.  [DOI](https://doi.org/10.1002/mgg3.1672) | [PMC free article](/articles/PMC8222831/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33818000/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Genet.%20Genom.%20Med.&title=Investigating%20the%20ACE2%20polymorphisms%20in%20COVID-19%20susceptibility:%20An%20in%20silico%20analysis&author=N.%20Pouladi&author=S.%20Abdolahi&volume=9&publication_year=2021&pages=e1672&pmid=33818000&doi=10.1002/mgg3.1672&)

63. Darbani B. The Expression and Polymorphism of Entry Machinery for COVID-19 in Human: Juxtaposing Population Groups, Gender, and Different Tissues. Int. J. Environ. Res. Public Heal. 2020;17:3433. doi: 10.3390/ijerph17103433.  [DOI](https://doi.org/10.3390/ijerph17103433) | [PMC free article](/articles/PMC7277542/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32423095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Environ.%20Res.%20Public%20Heal.&title=The%20Expression%20and%20Polymorphism%20of%20Entry%20Machinery%20for%20COVID-19%20in%20Human:%20Juxtaposing%20Population%20Groups,%20Gender,%20and%20Different%20Tissues&author=B.%20Darbani&volume=17&publication_year=2020&pages=3433&pmid=32423095&doi=10.3390/ijerph17103433&)

64. Xiao F., Zimpelmann J., Agaybi S., Gurley S.B., Puente L., Burns K.D. Characterization of angiotensin-converting enzyme 2 ectodomain shedding from mouse proximal tubular cells. PLoS ONE. 2014;9:e85958. doi: 10.1371/journal.pone.0085958.  [DOI](https://doi.org/10.1371/journal.pone.0085958) | [PMC free article](/articles/PMC3893316/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24454948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Characterization%20of%20angiotensin-converting%20enzyme%202%20ectodomain%20shedding%20from%20mouse%20proximal%20tubular%20cells&author=F.%20Xiao&author=J.%20Zimpelmann&author=S.%20Agaybi&author=S.B.%20Gurley&author=L.%20Puente&volume=9&publication_year=2014&pages=e85958&pmid=24454948&doi=10.1371/journal.pone.0085958&)

65. Chen J., Wang R., Wang M., Wei G.W. Mutations strengthened SARS-CoV-2 infectivity. arXiv. 2020 doi: 10.1016/j.jmb.2020.07.009.2005.14669  [DOI](https://doi.org/10.1016/j.jmb.2020.07.009) | [PMC free article](/articles/PMC7375973/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32710986/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=arXiv&title=Mutations%20strengthened%20SARS-CoV-2%20infectivity&author=J.%20Chen&author=R.%20Wang&author=M.%20Wang&author=G.W.%20Wei&publication_year=2020&pmid=32710986&doi=10.1016/j.jmb.2020.07.009&)
