# ERICH3: vesicular association and antidepressant treatment response

## Metadata
**Authors:** Duan Liu, Yongxian Zhuang, Lingxin Zhang, Huanyao Gao, Drew Neavin, Tania Carrillo-Roa, Yani Wang, Jia Yu, Sisi Qin, Daniel C Kim, Erica Liu, Thanh Thanh Le Nguyen, Joanna M Biernacka, Rima Kaddurah-Daouk, Boadie W Dunlop, W Edward Craighead, Helen S Mayberg, Elisabeth B Binder, Mark A Frye, Liewei Wang, Richard M Weinshilboum
**Journal:** Molecular Psychiatry
**Date:** 2020 Nov 23
**DOI:** [10.1038/s41380-020-00940-y](https://doi.org/10.1038/s41380-020-00940-y)
**PMID:** 33230203
**PMCID:** PMC8141066
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8141066/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8141066/pdf/41380_2020_Article_940.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8141066/pdf/41380_2020_Article_940.pdf)

## Abstract

Selective serotonin reuptake inhibitors (SSRIs) are standard of care for major depressive disorder (MDD) pharmacotherapy, but only approximately half of these patients remit on SSRI therapy. Our previous genome-wide association study identified a single-nucleotide polymorphism (SNP) signal across the glutamate-rich 3 (ERICH3) gene that was nearly genome-wide significantly associated with plasma serotonin (5-HT) concentrations, which were themselves associated with SSRI response for MDD patients enrolled in the Mayo Clinic PGRN-AMPS SSRI trial. In this study, we performed a meta-analysis which demonstrated that those SNPs were significantly associated with SSRI treatment outcomes in four independent MDD trials. However, the function of ERICH3 and molecular mechanism(s) by which it might be associated with plasma 5-HT concentrations and SSRI clinical response remained unclear. Therefore, we characterized the human ERICH3 gene functionally and identified ERICH3 mRNA transcripts and protein isoforms that are highly expressed in central nervous system cells. Coimmunoprecipitation identified a series of ERICH3 interacting proteins including clathrin heavy chain which are known to play a role in vesicular function. Immunofluorescence showed ERICH3 colocalization with 5-HT in vesicle-like structures, and ERICH3 knock-out dramatically decreased 5-HT staining in SK-N-SH cells as well as 5-HT concentrations in the culture media and cell lysates without changing the expression of 5-HT synthesizing or metabolizing enzymes. Finally, immunofluorescence also showed ERICH3 colocalization with dopamine in human iPSC-derived neurons. These results suggest that ERICH3 may play a significant role in vesicular function in serotonergic and other neuronal cell types, which might help explain its association with antidepressant treatment response.

Subject terms: Genetics, Biomarkers, Molecular biology

## Introduction

Major depressive disorder (MDD) is the most common psychiatric disease and a major cause of medical disability worldwide [[1](#CR1), [2](#CR2)]. Relative deficiency of the monoamine neurotransmitter serotonin (5-HT) appears to play a role in the pathophysiology of MDD and, as a result, drugs such as selective serotonin reuptake inhibitors (SSRIs) that enhance serotonergic neurotransmission are used to treat MDD. SSRIs are standard-of-care pharmacotherapy for MDD [[3](#CR3)], but many MDD patients fail to respond to SSRI therapy and, even in those who do respond, the response may be delayed for weeks or months [[4](#CR4)]. Several genome-wide association studies (GWAS) for SSRI treatment outcomes have been performed [[5](#CR5)–[9](#CR9)], but few of the top signals from those GWAS have been replicated or functionally validated [[10](#CR10), [11](#CR11)]. This situation may result, in part, from MDD phenotypic heterogeneity and/or the lack of biologically based phenotypes [[12](#CR12)].

In an effort to move beyond genomics alone, we previously applied a “pharmacometabolomics-informed pharmacogenomic” research strategy [[13](#CR13)–[15](#CR15)] with the goal of discovering mechanisms related to variation in SSRI response [[15](#CR15)]. That approach began with the association of plasma metabolite concentrations with SSRI response, followed by GWAS using concentrations of those metabolites as phenotypes to identify genetic loci associated with variation in metabolite concentrations—concentrations that represented biologically-based quantitative phenotypes. Of the metabolites that we assayed, plasma 5-HT concentrations at baseline as well as changes in plasma 5-HT concentrations after SSRI treatment were most significantly associated with SSRI response in the MDD patients whom we studied [[15](#CR15), [16](#CR16)]. The association of plasma 5-HT concentrations with SSRI response has now been replicated in two independent studies [[17](#CR17), [18](#CR18)].

GWAS for baseline plasma 5-HT concentrations in our MDD patients identified a single-nucleotide polymorphism (SNP) signal across the glutamate-rich 3 (*ERICH3*) gene (rs696692, *P* = 9.28E–08; *N* = 290), which did not reach the genome-wide significant threshold (*P* ≤ 5.0E–08). That same SNP signal was also observed in our GWAS for change in plasma 5-HT concentrations after SSRI treatment [[15](#CR15)]. One of the *ERICH3* SNPs (rs10580409, *P* = 1.12E–07) which was in tight linkage disequilibrium with the rs696692 “top” SNP (D’ = −0.98, *r*^2^ = 0.96 in European subjects as reported by the 1000 Genomes Project [[19](#CR19)]) was a nonsynonymous SNP (nsSNP) associated with accelerated proteasome-mediated ERICH3 protein degradation [[15](#CR15)]. In addition, we observed that changing ERICH3 expression by siRNA knock-down or cDNA over-expression in central nervous system (CNS) derived cell lines resulted in significant changes of 5-HT concentrations in the cell culture media [[15](#CR15)]. We also found that the *ERICH3* rs10580409 nsSNP which was identified in our GWAS for plasma 5-HT concentrations in MDD patients, was also directly associated with SSRI response in independent SSRI treatment response GWAS [[15](#CR15)], including those for the Sequenced Treatment Alternatives to Relieve Depression (STAR*D) [[4](#CR4)] and the International SSRI Pharmacogenomics Consortium (ISPC) [[9](#CR9)].

When we initially identified *ERICH3* with our GWAS for plasma 5-HT concentrations in MDD patients, as described in the preceding paragraph, it was named “chromosome 1 open reading frame 173 (*C1orf173*)” based on its map location in the human genome, and nothing was known about the gene or the protein that it encoded functionally. It was subsequently renamed “E-rich 3” as a result of the high content of glutamate (E) in its encoded amino acid sequence. A total of six *ERICH* genes, from *ERICH1* to *ERICH6*, have been identified in the human genome. However, with the exception of their high content of E, the proteins encoded by these genes do not display similarity in their amino acid sequences or protein domains. As a result, they may not be related functionally. None of the ERICH proteins have been well characterized and their functions are presently unknown. RNA sequencing (RNA-seq) [[20](#CR20)–[22](#CR22)] and mass-spectrometry-based proteomic data [[23](#CR23), [24](#CR24)] for different human tissues have shown that ERICH3 is most highly expressed in a variety of regions of the human brain, including the nucleus accumbens (basal ganglia) and frontal cortex ([https://www.gtexportal.org/home/gene/ERICH3](https://www.gtexportal.org/home/gene/ERICH3)). In addition to the *ERICH3* nsSNP that we found to be associated with plasma 5-HT concentrations and SSRI response in MDD patients [[15](#CR15)], another *ERICH3* SNP, rs114396406, has recently been associated with several depression-related phenotypes including risk for severe depression (*P* = 9.362E-05; OR = 0.926; *N* = 54,567), in a large cohort GWAS reported by the Global Biobank Engine using the UK Biobank dataset [[25](#CR25)] ([https://gbe.stanford.edu/](https://gbe.stanford.edu/)). Transcriptome data generated by the PsychENCODE project also showed that ERICH3 mRNA was elevated in postmortem human brain samples from subjects with schizophrenia (FDR = 0.01) [[26](#CR26)]. Taken together, these observations suggest that ERICH3 might be related to risk for psychiatric disorders such as depression and schizophrenia.

It is against this background that the series of experiments described subsequently was performed. Specifically, we set out to (1) define the *ERICH3* gene structure, (2) identify CNS cells in which ERICH3 was expressed, (3) define ERICH3 mRNA species and the proteins which those species encoded, (4) apply coimmunopreciptiation (Co-IP) to identify proteins that interact with ERICH3, many of which proved to be vesicle-related proteins, (5) to define the effect of ERICH3 knock-out (KO) on monoamine neurotransmitter vesicular function and finally, (6) to perform a meta-analysis to test association of the *ERICH3* rs11580409 SNP that alters the amino acid sequence of the encoded protein with SSRI response for MDD patients enrolled in four independent MDD SSRI trials. The results of those experiments are described in subsequent paragraphs, providing a foundation for understanding the contribution of ERICH3 to monoamine neurotransmitter function and perhaps other neurotransmitter systems in order to understand the possible contribution of this gene to the pathophysiology of neuropsychiatric disease.

## Materials and methods

### ERICH3 transcriptional analysis at the single cell level

Single-cell RNA-seq data generated using surgically-removed human cerebral cortical samples from 12 subjects [[27](#CR27), [28](#CR28)] were downloaded and combined for analysis. Those sequenced CNS cells were “clustered” by mRNA expression levels of marker genes and were visualized using previously published methods [[29](#CR29), [30](#CR30)]. RNA raw counts were mapped to the human reference genome GRCh38 (hg38) to quantify relative *ERICH3* exon and exon junction expression. RNA counts were further normalized by exon length and RNA-seq cDNA library size. Please see the Supplementary Text for details.

### ERICH3 protein characterization

Western blot analysis was used to characterize ERICH3 protein isoforms. cDNA constructs for use in overexpressing ERICH3 protein isoforms were created and transfected into HEK (human embryonic kidney) 293 T cells from which protein lysates were used as positive controls to test ERICH3 antibodies. Lysates from human primary neurons were also used for ERICH3 protein identification. The reagents used, including antibodies, plasmids, and other reagents are listed in Supplementary Table S[1](#MOESM2).

### ERICH3-interacting protein identification

Co-Immunoprecipitation (Co-IP) using ERICH3 antibody (Ab-Ex 14) was performed with SK-N-SH (human brain neuroblastoma) cells. Non-specific rabbit IgG was used as a negative control. Co-IP protein samples were separated by SDS-PAGE and bands were cut out of the gels for mass spectrometric (MS) analysis. MS results were validated by Western blot and immunofluorescence (IF) assays. See the Supplementary Text for details.

### ERICH3 relationship to 5-HT concentrations

SK-N-SH cells were edited by CRISPR/cas9 using two guide RNAs (gRNAs) targeting the *ERICH3* gene. SK-N-SH cells were then incubated with 40 µM 5-hydroxytryptophan (5-HTP), a compound that can be metabolized by DOPA decarboxylase (DDC) to form 5-HT. Concentrations of 5-HT and its primary metabolite, 5-HIAA, were assayed in the cell media and lysates by HPLC-with electrochemical detection. 5-HT in cells was also stained and visualized by IF. Protein levels of 5-HT metabolizing enzymes were quantified by Western blot analysis. Human iPSC-derived neurons were infected by lentivirus packaged with shRNAs for ERICH3 KD. Cell culture media were then collected for the detection of catecholamines and catecholamine metabolites. See the Supplementary Text for details.

### Association of the ERICH3 rs11580409 SNP with antidepressant treatment response

The possible association between the *ERICH3* rs11580409 SNP genotype and antidepressant treatment responses in the STAR*D [[4](#CR4)], ISPC [[9](#CR9)], PReDICT [[31](#CR31), [32](#CR32)], and PGRN-AMPS [[8](#CR8), [33](#CR33)] studies was determined using logistic regression after adjusting for age, gender, and race. Results of those 4 studies were combined for an inverse-variance weighted fixed-effects meta-analysis using the “rmeta” package in R. Treatment “response” was defined as a reduction of at least 50% in depression score as determined by use of the 17-item Hamilton Depression Rating Scale or the 16-item Quick Inventory of Depressive Symptomatology rating scale. A *P* value of < 0.05 was considered nominally statistically significant [[9](#CR9), [34](#CR34), [35](#CR35)]. See the Supplementary Text for details.

## Results

### ERICH3 gene structure and transcript variants

The *ERICH3* gene structure is depicted schematically in Fig. [1a](#Fig1). The Ensembl human genome assembly annotated five ERICH3 mRNA transcripts. The reference transcript (ENST00000326665) consisted of fifteen exons (Fig. [1b](#Fig1), T-1), with exon 14 encoding half of the open reading frame. Another Ensembl protein-coding ERICH3 transcript variant (ENST00000420661) consisted of seven exons (Fig. [1b](#Fig1), T-2), with the first exon (numbered as “6a” in Fig. [1b](#Fig1)) mapping to intron 6, and with other exons having sequences identical to exons 7 to 11 of the T-1 transcript. The final exon of this transcript, numbered as “12+” in Fig. [1b](#Fig1), contained an additional 304 nucleotides which included a stop codon and a 3’-untranslated region. Other variant transcripts annotated by Ensembl were predicted to encode either long non-coding RNAs (lncRNA) or nonsense mediated decay transcripts. Two “lncRNA” transcripts (Fig. [1b](#Fig1)**;** T-3 and T-4), which initiated transcription from intron 12, had initial exons numbered as 12a–12c. Although RNA-seq indicated that ERICH3 was highly expressed in a variety of brain tissues [[20](#CR20)], exactly which ERICH3 transcript(s) was/were most highly expressed in the CNS was unclear.

### Fig. 1. ERICH3 gene structure and mRNA transcripts in human brain.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4ded/8440192/f2d54a0c66ea/41380_2020_940_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8440192_41380_2020_940_Fig1_HTML.jpg)

a Depiction of ERICH3 gene structure based on its physical location on chromosome 1 (GRCh38/p13). ERICH3 exons were numbered from 1 to 15 based on its reference mRNA transcript (Transcript ID: ENST00000326665). Alternative exons that map to intron 6 and intron 12 of the reference mRNA transcript were named with the intron number followed by a letter of the alphabet. The positions of the rs696692 and rs11580409 SNPs are also labeled. b ERICH3 transcript variants (T-1 to T-4) annotated by Ensembl. Exons depicted as solid boxes represent open-reading frames. Untranslated regions have been depicted as empty (white) boxes. Translation start codons and stop sites have been labeled as “AUG” and “Stop,” respectively. c Clusters of human brain cells were assigned cell types based on the expression of cell-type-specific marker genes quantified by single-cell RNA-seq. Each dot represents a single cell. Cell types have been color-coded and the number of sequenced cells of each cell type is listed in parentheses (left). ERICH3-positive cells were color-coded based on ERICH3 expression levels, and the ERICH3-negative cells are shown as gray dots. Neurons and oligodendrocyte progenitor cells (OPC) were two cell types that included the most ERICH3 positive cells. Percentages of ERICH3-positive cells in these two cell types are presented in parentheses (right). d Heat map of ERICH3 exon (“E”) expression in each CNS cell type. Darker color represents higher mRNA level. Exon 6a was not detected in any of the CNS cells. e Heat map of ERICH3 exon junction (“J”) expression. Darker color represents higher expression. In addition to the “canonical” exon junctions, a junction of alternative exon 12c to exon 13 (“J 12c-13”) was detected at low abundance in neurons.

To determine which human brain cell type(s) expressed ERICH3, we used information from two single-cell RNA-seq (scRNA-seq) datasets generated using surgically obtained human brain tissue [[27](#CR27), [28](#CR28)]. Those CNS cells were “clustered” and cell types were assigned on the basis of mRNA expression levels for marker genes (see Supplementary Fig. [S1a](#MOESM1)) [[29](#CR29), [30](#CR30)] as depicted graphically in Fig. [1c](#Fig1) (left). ERICH3 was expressed in neurons and in oligodendrocyte progenitor cells, but predominantly in neurons, e.g., 43% of neurons were ERICH3 positive, a much higher percentage than in any other CNS cell types (Fig. [1c](#Fig1), right; Supplementary Fig. [S1b](#MOESM1)). Using the scRNA-seq data, we generated heat maps depicting RNA expression of ERICH3 exons (Fig. [1d](#Fig1)) and exon junctions (Fig. [1e](#Fig1)) in CNS cells. As shown in Fig. [1d, e](#Fig1), a majority of exon and junction reads in neurons aligned to reference ERICH3 transcripts (Fig. [1b](#Fig1), T-1). Our results were similar when we used an independent scRNA-seq study performed with human cerebral cortical samples which showed that ERICH3 was broadly expressed in inhibitory and excitatory neurons but not in non-neuronal cells [[36](#CR36)] (Supplementary Fig. [S2a](#MOESM1)). ERICH3 was also reported to be expressed in serotonergic and dopaminergic neurons as shown in a scRNA-seq study performed with human fetal midbrain samples [[37](#CR37)] (Supplementary Fig. [S2b](#MOESM1)). To further validate these observations, we next characterized ERICH3 protein isoforms that were expressed in neurons.

### ERICH3 protein isoforms

The scRNA-seq data suggested the presence of three ERICH3 mRNA transcripts in neurons (Fig. [1d, e](#Fig1)), transcripts that could potentially be translated into three protein isoforms (Fig. [2a](#Fig2), P-1 to P-3). To test that possibility, we performed Western blot analysis to detect ERICH3 protein isoforms expressed in human primary cortical neurons. To obtain control ERICH3 proteins, cDNA constructs for the three putative ERICH3 isoforms shown in Fig. [2a](#Fig2) were cloned. We then used three ERICH3 antibodies that we created (Ab-Ex12, Ab-Ex13, and Ab-Ex14) to differentiate the ERICH3 protein isoforms (see Fig. [2a](#Fig2)). Those same three ERICH3 isoforms, “tagged” with FLAG-epitopes, were also overexpressed in HEK293T cells, and the HEK293T lysates were used to validate our three ERICH3 antibodies which, as anticipated, differentiated ERICH3 protein isoforms based on Western blot results (Fig. [2b](#Fig2)). When those antibodies were applied to protein lysates from human primary cortical neurons, an ERICH3 protein band pattern that matched the molecular weight (MW) of the “canonical” ERICH3 protein (P-1) was observed in human neurons (Fig. [2c](#Fig2)). The P-1 isoform band was also detected in preparations from SK-N-SH neuroblastoma cells. In addition to P-1, a band that ran between P-1 and P-3, namely P-4, was identified in SK-N-SH cells (Fig. [2c](#Fig2)). P-4 was assumed to be encoded by exons 12 to 14, a transcript variant identified in neuroblastoma cells [[38](#CR38)].

### Fig. 2. ERICH3 protein characterization.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4ded/8440192/fb13e77c739d/41380_2020_940_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8440192_41380_2020_940_Fig2_HTML.jpg)

a Schematic depiction of ERICH3 protein isoforms. The “canonical” protein isoform encoded by the reference mRNA transcript is designated “P-1.” The UniProtKB protein ID for this isoform is “Q5RHP9-1.” The protein coding exons corresponding to each protein isoforms are shown in parentheses. Numbers of the first and last amino acids for each isoform, matching to the number of amino acids in the “canonical” protein sequence are listed above each schematic representation. “P-2” represents a putative protein isoform corresponding to the mRNA transcript variant that ends at exon 12+ in neurons. “P-3” is encoded by exon 14 which is designated “Q5RHP9-2” in UniProtKB. Antigen sequences and corresponding antibodies that target different regions of the ERICH3 protein are shown below each depiction. b Western blots which validate ERICH3 antibodies. ERICH3 protein isoforms fused with FLAG-tags were overexpressed in HEK293T cells and those protein lysates were subjected to Western blots using different ERICH3 antibodies. Anti-FLAG antibody was used to confirm that the FLAG-tagged ERICH3 isoforms had been successfully overexpressed. ERICH3 antibodies that target different regions of the amino acid sequences performed as expected in Western blot. All three antibodies detected P-1 as well as their specifically targeted isoforms. GAPDH were “blotted” as internal control. c Western blots performed using validated ERICH3 antibodies to study protein lysates from human primary neurons and SK-N-SH neuroblastoma cells, with nontagged, cDNA-overexpressed ERICH3 isoforms used as a control. “P-4” is encoded by exons 12 to 14 which corresponds to a transcript variant identified in neuroblastoma cells. Figure shown represents result from triplicate experiments.

The “canonical” ERICH3 protein (P-1) was predicted to have an MW of 168.5 kD, but a band at ~250 kD was observed by Western blot (Fig. [2b](#Fig2)). Experimentally, we found that this striking difference (>80kD) between predicted and observed MWs was unlikely to result from post-translational modification such as glycosylation or phosphorylation (see Supplementary Fig. [S3](#MOESM1)). Previous studies have reported that proteins with a high content of glutamate (E) and/or aspartate (D), amino acids with acidic side chains, can display higher apparent MW values during Western blot analysis than would be predicted [[39](#CR39)], and an equation to estimating the observed MW for E/D-rich proteins has been published [[39](#CR39)]. When that equation was used to calculate MW values for ERICH3 isoform proteins, the calculated MW values were much closer to those that we observed by Western blot analysis (Supplementary Table S[2](#MOESM2)). Therefore, it is possible that the “excessive” MW values which we observed may have resulted from the high content of glutamate (E) in ERICH3. Of note, the observed MW for the “C-terminal” ERICH3 isoform P-3 which is about half the size (788 aa) of ERICH3 isoform P-1, was very close to 165.5 kD, the predicted MW of the ERICH3 P-1 isoform (Fig. [2b](#Fig2)), thus making it difficult to distinguish these two isoforms by Western blot. However, as suggested subsequently, these two isoforms may have different functions.

### ERICH3 interaction with proteins involved in vesicular function

After characterizing the ERICH3 transcripts and proteins expressed in neurons, we next attempted to identify possible ERICH3-interacting proteins by Co-IP assays. Proteins that were “pulled-down” were separated by SDS-PAGE and visualized by silver staining (Supplementary Fig. [S4](#MOESM1)). Protein bands were then cut from the gels for MS analysis. Ranked by the Sum Intensity of peptides that were quantified by MS, the “top” 50 candidate proteins identified in ERICH3 pull-down samples are listed in Supplementary Table S[3](#MOESM2). Several of those candidates proteins are known to play a role in vesicle biogenesis and/or function, including clathrin heavy chain (CLTC), adaptor related protein complex 2 subunit alpha 2 (AP2A2), AP2 associated kinase 1 (AAK1), phosphatidylinositol-4-phosphate 3-kinase catalytic subunit type 2 alpha (PIK3C2A), and myosin heavy chain 10. Those five proteins are all known to participate in vesicle formation and function [[40](#CR40)–[44](#CR44)]. Other proteins observed during Co-IP included TMPO, LMNA, LIMCH1, TMOD1, and POF1B, proteins that play a role in the cytoskeleton and mitosis, processes in which clathrin is also known to be involved [[45](#CR45), [46](#CR46)]. Finally, the two most abundant proteins precipitated by ERICH3 antibodies were ALK and CUX1, both of which are encoded by genes that carry genetic polymorphisms that have been associated with antidepressant treatment response [[8](#CR8), [47](#CR47)]. Both ALK and CUX1 are highly expressed in neurons and are known to play roles in neuronal differentiation, morphology, and function [[48](#CR48)–[55](#CR55)].

To further validate the MS results, Western blot assays were performed using antibodies that targeted MS-identified ERICH3-interacting proteins. Because the *ERICH3* SNPs were identified by a GWAS for plasma 5-HT concentrations [[15](#CR15)] and since 5-HT, like other monoamine neurotransmitters, is stored in vesicles, we focused our subsequent studies on proteins known to be involved in vesicular trafficking and/or function (Supplementary Table S[3](#MOESM2)). Three proteins, CLTC, PIK3C2A, and AP2A2, all of which could be precipitated by ERICH3 antibody from SK-N-SH cells, were validated by Western blot analysis (Fig. [3a](#Fig3)). Immunofluorescent (IF) costaining of ERICH3 and CLTC showed colocalization of these proteins in SK-N-SH cells (Supplementary Fig. [S5](#MOESM1)). Neither protein was localized in the nucleus but rather in cytoplasm, often with colocalization to vesicle-like structures (Supplementary Fig. [S5](#MOESM1)). We also observed that ERICH3 was partially colocalized with vesicular monoamine transporter 2 (VMAT2) in SK-N-SH cells (Supplementary Fig. [S6](#MOESM1)). To determine whether similar interactions between ERICH3 and vesicle-related proteins might also exist in other cell lines, Co-IP assays using anti-mCherry (mCh) antibody were performed with HEK293T cells after overexpression of three mCh-tagged ERICH3 isoforms. All three of the vesicle-related proteins were pulled-down by anti-mCh antibody in HEK293T cells after the overexpression of mCh-tagged ERICH3 isoforms P-1 and P-2 but not isoform P-3 (Fig. [3b](#Fig3)). This result suggested that ERICH3 may interact with these three proteins through its N-terminal amino acid sequence which is encoded by exons 1-12. In addition, the mCh-tagged ERICH3 (mCh-ERICH3) made it possible to visualize ERICH3 cellular location by fluorescent microscopy which showed that all of three ERICH3 isoforms were present in the cytoplasm but not the nucleus (data not shown).

### Fig. 3. ERICH3 interacts with proteins involved in vesicular function.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4ded/8440192/1d31829821bb/41380_2020_940_Fig3_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8440192_41380_2020_940_Fig3_HTML.jpg)

a Western blot validation of proteins that were identified by mass spectrometry in Co-IP samples precipitated by ERICH3 antibody. CLTC, AP2A2, and PIK3C2A, three proteins that play a role in vesicle biogenesis, could be “pulled-down” by ERICH3 antibody but not by control IgG in SK-N-SH neuroblastoma cells. Figure shown represents results from triplicate experiments. b ERICH3 protein isoforms fused with mCherry (mCh) were overexpressed in HEK293T cells and pulled-down by anti-mCherry antibody (mCherry IP). CLTC, AP2A2, and PIK3C2A were “pulled-down” by P-1 and P-2 but not P-3, indicating that the interaction between ERICH3 and those three proteins was not taking place at amino-acid sequences that mapped to P-3, a portion of the protein encoded by exon 14. Figure shown represents results from duplicate experiments. c Coexpression of fluorescent protein-tagged ERICH3 (mCh-ERICH3) and CLTC (GFP-CLTC) in SK-N-SH cells made it possible to visualize their cellular localization and track their movements in living cells (Supplementary Video 1). Magnification is ×100. d Proposed role for ERICH3 in vesicle function. The schematic is not meant to propose a specific site for ERICH3 in the structure of the vesicle.

In an attempt to visualize the possible interaction between ERICH3 and CLTC in living cells, mCh-ERICH3 was co-overexpressed with GFP-tagged CLTC (GFP-CLTC) in SK-N-SH cells. The overexpressed mCh-ERICH3 and GFP-CLTC proteins once again showed strong colocalization (Fig. [3c](#Fig3)). A video tracking both of the fluorescent fusion proteins showed that yellow vesicle-like structures, labeled by mCh-ERICH3 and GFP-CLTC, were moving in the cytoplasm (Supplementary Video [1](#MOESM4)), a phenotype not observed in SK-N-SH cells coexpressing mCherry itself (empty vector) with GFP-CLTC (Supplementary Video [2](#MOESM5)).

Based on the results of Western blot analysis performed after subcellular fractionation of intracellular vesicles by density gradient centrifugation, we concluded that ERICH3 is unlikely to be an intrinsic vesicle membrane protein like vesicle-associated membrane protein 2 [[56](#CR56)] and synaptophysin [[57](#CR57)], but rather that it appears to be a cytosolic and vesicle-associated protein like AP2A2 (Supplementary Fig. [S7](#MOESM1)). Specifically, ERICH3 was primarily enriched in the same layer as was AP2A2, an “adaptor” protein that functions with clathrin during vesicle biogenesis [[40](#CR40)] (Fig. [3d](#Fig3)). In summary, this series of experiments suggested a possible role for ERICH3 in vesicular function (Fig. [3d](#Fig3)), a function that can plausibly be associated with neurotransmitter concentrations, as tested directly in the next series of experiments.

### ERICH3 and cellular 5-HT

To help determine how ERICH3 might influence 5-HT concentrations, both within the cell and in cell culture media, we performed a series of experiments using SK-N-SH neuroblastoma cells. 5-HT metabolism in these cells is depicted schematically in Fig. [4a](#Fig4). SK-N-SH cells do not express either TPH1 or TPH2, but they do express DDC and MAOA (see Supplementary Fig. [S8](#MOESM1)) so they can potentially synthesize 5-HT from 5-HTP and then metabolize it to form 5-HIAA. Assessment of the dynamics of 5-HTP metabolism in SK-N-SH cells by assaying 5-HT and 5-HIAA in cell culture media showed that 5-HTP was metabolized to form 5-HT and 5-HIAA as early as after 3-h of incubation (Supplementary Fig. [S9a](#MOESM1)). In cells cotransfected with Cas9 plasmid and two ERICH3 guide RNAs (gRNAs), ERICH3 protein level was decreased when compared with that in nontarget control cells (Supplementary Fig. [S9b](#MOESM1)), indicating that *ERICH3* KO occurred in a portion of those transfected cells. After 6 h of incubation for the cells and 12 h for the culture medium, significant decreases in 5-HT concentrations were observed for ERICH3 gRNA-transfected cells (Supplementary Fig. [S9c](#MOESM1)). 5-HIAA concentrations were also decreased in the cell culture media, but they increased in cell lysates from ERICH3 gRNA-transfected cells (Supplementary Fig. [S9d](#MOESM1)). The significant decreases in 5-HT concentrations in culture media for ERICH3 gRNA-transfected cells were probably not caused by 5-HT metabolism since the expression of DDC and MAOA was unchanged (Supplementary Fig. [S9b](#MOESM1)). To further confirm that the observed phenotypic changes resulted from *ERICH3* KO, two single colonies in which *ERICH3* was edited by Cas9 with gRNA2 were selected. CRISPR/Cas9 editing of *ERICH3* and the protein KO in those two single colonies (KO#1 and KO#2) were confirmed by Sanger sequencing (Fig. [4b](#Fig4)) and Western blot assay (Fig. [4c](#Fig4)), respectively. Even more dramatic decreases in 5-HT and 5-HIAA concentrations in culture media for those two *ERICH3* KO single colonies were observed (Fig. [4d](#Fig4)) than had been observed for the experiment shown in Supplementary Figure [S9](#MOESM1) in which a mixture of KO and non-KO cells were present. In both cases, there were not significant changes in DDC and MAOA protein levels (Fig. [4c](#Fig4)).

### Fig. 4. ERICH3 and monoamine neurotransmitters.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4ded/8440192/83670a038ebe/41380_2020_940_Fig4_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8440192_41380_2020_940_Fig4_HTML.jpg)

a 5-HT is synthesized from tryptophan (TRP). Specifically, TRP is metabolized by tryptophan hydroxylases (TPH1 or TPH2) to generate 5-hydroxytryptophan (5-HTP) which can be further metabolized by aromatic-L-amino-acid decarboxylase (also known as DOPA decarboxylase, DDC) to form 5-HT. 5-HT can then be transported into vesicles by vesicular monoamine transporters (VMATs) and is released from cells by exocytosis. Vesicle-free 5-HT can be metabolized by monoamine oxidases (MAOs), which are located in the outer mitochondrial membrane to generate 5-hydroxyindoleacetic acid (5-HIAA). b Sanger sequencing of the ERICH3 exon 14 region, which was targeted by ERICH3 guide RNA 2 (gRNA2) in WT SK-N-SH cells, and two single colonies edited by CRISPR/Cas9 (KO#1 and KO#2). The Cas9-cutting site is labeled by dashed line and the protospacer adjacent motif (PAM), “GGG,” is boxed. Compared to WT cells, DNA sequences on the right side of the Cas9-cutting site shifted in KO#1 and KO#2 cells, indicating that the CRISPR/Cas9 editing occurred at the “designed” site in KO#1 and KO#2 cells. c Western blots showing that ERICH3 protein was not detectable and that MAOA and DDC protein levels were not significantly changed in the KO#1 or KO#2 cells when compared with WT cells. d 5-HT (left) and 5-HIAA (right) concentrations were significantly decreased in the cell culture media of SK-N-SH KO#1 and KO#2 cells in which the ERICH3 gene had been edited by CRISPR/Cas9. Cells were incubated with 40 µM 5-HTP for 6 h since SK-N-SH cells express neither TPH1 nor TPH2. Data are mean ± s.d. (n = 3), with statistical significance determined by Dunnett’s test denoted as ***P < 0.001. e Immunofluorescent (IF) costaining of ERICH3 and 5-HT showing the colocalization of ERICH3 with 5-HT in control SK-N-SH cells. In ERICH3 KO cells, 5-HT was hardly to detect by IF under the same conditions used to study control cells (upper panel). KD of CLTC resulted in a similar effect on 5-HT as did KO of ERICH3 (lower panel). Magnification is ×150. Figure shown represents results from duplicate experiments. f Costaining of ERICH3 and dopamine (DA) in human iPSC-derived dopaminergic neurons (hDN) showing the colocalization of ERICH3 with DA in neurites. Magnification is ×60 for upper panel and 300× for lower panel. Figure shown represents results from duplicate experiments. g Epinephrine, an end product of the dopamine metabolizing pathway was decreased in cell culture media for hDN 24 h after transfection with lentivirus packaged with ERICH3 shRNAs. Neither DA nor norepinephrine could be detected after 24 h incubation. Data are mean ± s.d. (n = 3), with statistical significance determined by unpaired t-test denoted as *P < 0.05.

Since we had observed that ERICH3 interacts with proteins such as CLTC that are involved in vesicular function (Fig. [3](#Fig3)), we next determined whether the observed decreases in 5-HT concentrations might be relevant to the packaging or transport of 5-HT by vesicles. Cellular 5-HT was detected by immunofluorescent staining of SK-N-SH cells incubated with 5-HTP for 6 h. We observed strong colocalization of ERICH3 and 5-HT in control cells, but both were barely detectable after ERICH3 KO (Fig. [4e](#Fig4), upper panels). KD of CLTC had an effect on cellular 5-HT which was similar to that observed after ERICH3 KO (Fig. [4e](#Fig4), lower panels).

### ERICH3 in iPSC-derived dopaminergic neurons

Our results suggested that ERICH3 might regulate 5-HT concentrations, at least in part, through a role in vesicular function, a mechanism that could potentially apply to other neurotransmitters since scRNA-seq data raised the possibility of ERICH3 expression in a variety of neurons. To test the hypothesis that ERICH3 might play a similar role for other monoamine neurotransmitters, human iPSC-derived dopaminergic neurons (hDNs) were cultured for use in ERICH3 and dopamine (DA) costaining. Those cells were first characterized by immunostaining tyrosine hydroxylase (TH), the enzyme that catalyzes the rate-limiting step in catecholamine biosynthesis, and tubulin beta-3 chain (TUJ-1), a neuron-specific microtubular protein (Supplementary Fig. [S10](#MOESM1)). ERICH3 was localized to hDN cell cytosol and neurites. It also colocalized with DA in vesicle-like “dots” present in neurites (Fig. [4f](#Fig4)), perhaps representing dopamine-containing vesicles. Cell culture media was also collected for the assay of catecholamines and their metabolites. Those assays showed that the KD of ERICH3 in hDNs decreased the concentration in culture media of epinephrine, a catecholamine neurotransmitter that requires DA as a precursor (Fig. [4g](#Fig4)). Other catecholamines, ie dopamine and norepinephrine, could not be detected in the culture media, which might be due to the presence of several types of catecholamine synthesizing cells in the culture—including cells that synthesize epinephrine.

### ERICH3 rs11580409 SNP association with antidepressant response

When we tested the possible association of genotype for rs11580409 with drug response in independent SSRI clinical studies, the rs11580409 variant allele (C) was significantly associated with better response in two other SSRI response GWAS, with an odds ratio (OR) of 1.17 in the STAR*D study (*N* = 1578, *P* = 0.041) and 1.26 in the ISPC study (*N* = 865, *P* = 0.022) [[15](#CR15)] (Fig. [5](#Fig5)). In the present study, the association of this *ERICH3* nsSNP with antidepressant response was also replicated in MDD patients enrolled in the Predictors of Remission in Depression to Individual and Combined Treatments (PReDICT) study [[31](#CR31), [32](#CR32)], in which patients were treated with either an SSRI or a serotonin–norepinephrine reuptake inhibitor (SNRI). Once again, the rs11580409 variant allele (C) was significantly associated with better response (*N* = 151; *P* = 0.029; OR = 2.12) (Fig. [5](#Fig5)). Meta-analysis that combined samples from these four independent antidepressant studies, STAR*D, ISPC, PReDICT and PGRN-AMPS, showed an even more significant association of the rs11580409 C allele with better response with an OR = 1.21 and *a P* value of 7.5E–04 (Fig. [5](#Fig5)).

### Fig. 5. Forest plot for meta-analysis of the association of the ERICH3 rs11580409 SNP with antidepressant treatment response.

![Fig. 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4ded/8440192/e8357d7b9754/41380_2020_940_Fig5_HTML.jpg)

Data were obtained from four antidepressant treatment response clinical studies: Sequenced Treatment Alternatives to Relieve Depression (STAR*D), International SSRI Pharmacogenomics Consortium (ISPC), Predictors of Remission in Depression to Individual and Combined Treatments (PReDICT), and Pharmacogenomics Research Network Antidepressant Medication Pharmacogenomic Study (PGRN-AMPS). MDD patients in the STAR*D, ISPC and PGRN-AMPS studies were treated with an SSRI, either citalopram or escitalopram. PReDICT patients were treated with escitalopram or duloxetine, an SNRI. Response was defined as ⩾50% reduction in depression score as measured by QIDS-C16 or HAM-D, dependent on the study. For each of the studies, response at the last-available time point after antidepressant treatment was included in the analysis (STAR*D at 6 weeks, ISPC at 4 weeks, PReDICT at 12 weeks and PGRN-AMPS at 8 weeks). P values are for the association of the ERICH3 rs11580409 SNP variant allele (C) with antidepressant response; N = number of MDD patients; OR = odds ratio with the stand errors in parentheses.

## Discussion

When we initially identified *ERICH3* as a “top hit” in our original GWAS, it was an uncharacterized gene that was most highly expressed in human brain [[20](#CR20)–[22](#CR22)]. That GWAS identified an SNP signal in *ERICH3* that was associated with plasma serotonin concentrations which were themselves associated with SSRI response in MDD patients enrolled in the Mayo PGRN-AMPS SSRI trial [[15](#CR15)]. The same *ERICH3* SNP was associated with SSRI response in MDD patients in two independent large-cohort GWA studies, STAR*D [[4](#CR4)] and ISPC [[9](#CR9)]. Those observations motivated us to characterize the function of ERICH3 in order to help us understand its role in antidepressant treatment response. In the present study, by analyzing published scRNA-seq data for human brain samples [[27](#CR27), [28](#CR28)] we found that ERICH3 is predominantly expressed in neurons rather than other CNS cell types (Fig. [1](#Fig1)). The major ERICH3 protein isoform in neurons, an isoform encoded by one of several mRNA splice variants, was also identified (Fig. [2](#Fig2)). ERICH3-interacting proteins were then identified by performing Co-IP studies. Those proteins included CLTC as well as several other proteins involved in vesicular function (Fig. [3](#Fig3)). IF staining for ERICH3 showed a cytoplasmic localization as well as localization to vesicle-like structures, supporting a possible role for ERICH3 in vesicle function. We then demonstrated that KO of ERICH3 decreased 5-HT concentrations both in cell culture media and in cell lysates (Fig. [4](#Fig4), Supplementary Fig. [S9](#MOESM1)). Finally, ERICH3 was also found to colocalize with dopamine in human iPSC-derived neurons (Fig. [4](#Fig4)), and the association between *ERICH3* SNPs and antidepressant response was replicated in the PReDICT study [[31](#CR31), [32](#CR32)], in which patients were treated with either an SSRI or SNRI (Fig. [5](#Fig5)), indicating a possible role of ERICH3 in monoamine neurotransmitter function that extends beyond 5-HT. Since we also observed that ERICH3 is expressed in glutamatergic and GABAergic neurons [[36](#CR36)] (Supplementary Fig. [S2](#MOESM1)), the vesicular function of ERICH3 that we report in the present study might also apply to neurons beyond those involved in monoamine storage and function.

Monoamine neurotransmitters, including 5-HT, are stored in vesicles and released by exocytosis into the synaptic cleft when vesicles fuse with the plasma membrane [[58](#CR58)]. After exocytosis, synaptic vesicles undergo endocytosis and recycle for a new round of exocytosis [[58](#CR58)]. This process of vesicle trafficking may play a role in neuropsychiatric disorders. Clathrin-mediated vesicle formation is part of the machinery for vesicle recycling [[59](#CR59)–[61](#CR61)]. Components of the synaptic vesicle recycling machinery have been proposed previously as potential targets for the treatment of neuropsychiatric disorders [[62](#CR62)]. In the present study, we have shown that ERICH3 is required for the maintenance of 5-HT concentrations, a process which could involve its interactions with CLTC, AP2A2, and PIK3C2A, proteins known to be involved in clathrin-mediated vesicle formation [[40](#CR40)–[42](#CR42)]. These observations may explain, at least in part, the association of *ERICH3* SNPs with SSRI response that we have observed in MDD patients [[15](#CR15)]. Of potential interest is the fact that we also found that ERICH3 interacted with ALK and CUX1 (Supplementary Table S[3](#MOESM2)), both of which have been reported to have genetic polymorphisms associated with antidepressant treatment response [[8](#CR8), [47](#CR47)]. However, molecular mechanism(s) that might explain those associations remain unknown.

SSRIs target the reuptake mechanism for 5-HT by inhibiting the activity of the 5-HT transporter (SERT or SLC6A4), resulting in increased synaptic 5-HT concentrations [[63](#CR63)–[65](#CR65)]. This mechanism applies to both the CNS and peripheral tissues. For example, more than 99% of the 5-HT in blood is found in platelets [[66](#CR66), [67](#CR67)]. However, platelets do not synthesize 5-HT but rather utilize SLC6A4 in their membrane to take up 5-HT generated in the intestine by enterochromaffin cells [[68](#CR68)]. As a result, plasma 5-HT concentrations may reflect platelet 5-HT which might, in turn, reflect SLC6A4 function in the platelet membrane—thus providing a measure of the effect of SSRIs that can be accessed in the periphery. A striking decrease in plasma 5-HT concentrations has been observed in MDD patients after SSRI treatment [[15](#CR15)–[18](#CR18)]. This might explain our [[15](#CR15)] and others’ [[17](#CR17), [18](#CR18)] observations that larger decreases in plasma 5-HT concentration after SSRI treatment, which could be a reflection of greater drug sensitivity, are associated with better SSRI response in MDD patients. ERICH3 is also highly expressed in platelets [[23](#CR23), [24](#CR24)], and 5-HT in platelets is stored in dense granules, lysosome-related vesicles with biogenesis that is known to involve the clathrin-mediated endocytic pathway [[69](#CR69)].

Characterization of ERICH3 mRNA transcripts and proteins in the human CNS—as reported in this study—represents a necessary step toward understanding the possible role of ERICH3 in MDD and the response of MDD to SSRI therapy. We should point out several limitations of our studies including the fact that the *P* value for association of *ERICH3* SNPs in our original GWAS for plasma 5-HT concentrations in MDD patients did not reach the genome-wide significant threshold. Validation of this association in larger cohorts will be required. In addition, our results with regard to the role of ERICH3 in 5-HT biology were obtained using SK-N-SH neuroblastoma cells, results that will also require validation using serotonergic neurons once the technology to culture that type of neuron becomes less challenging technically. Finally, the possible functional effects of ERICH3 beyond 5-HT, as suggested by its broad expression in different types of neurons, should be addressed in the course of future studies using a variety of neurons as should the effect of CRISPR/cas9 editing of the *ERICH3* rs10580409 SNP to clarify the mechanism of the effect of this SNP on *ERICH3* function.

In summary, the present study has characterized the molecular genomics and neuronal function of *ERICH3*, a novel gene studied here primarily because of its association with SSRI antidepressant response. The results of the present experiments represent a step toward understanding the functional genomics of *ERICH3*. The results also point toward future studies of the functional basis for its role in SSRI response—a role that may involve neurotransmitter vesicular function, future studies of which might contribute to our understanding of individual variation in both MDD disease risk and antidepressant treatment response.

## Supplementary information

## Acknowledgements

This work was supported, in part, by NIH grants U19 GM61388 and RO1 GM28157 to Dr. Weinshilboum; R01MH108348, R01AG046171, U01AG061359 and RF1AG051550 to Dr. Kaddurah-Daouk; The PReDICT study was supported by NIH grants P50 MH077083, R01 MH080880, UL1 RR025008, and M01 RR0039 and by the Fuqua Family Foundations.

## Compliance with ethical standards

### Conflict of interest

RMW and LW are cofounders of and stockholders in OneOme, LLC, a pharmacogenomic clinical decision support company. RK-D is inventor on key patents in the field of metabolomics in the study of CNS diseases and holds equity in Metabolon, Inc., a biotechnology company that provides metabolic profiling capabilities. BWD has received research support from Acadia, Axsome, Intra-Cellular Therapies, Janssen, and Takeda; he has served as a consultant to Myriad Neuroscience and Aptinyx. HSM has received consulting and IP licensing fees from Abbott Labs. MAF has received research support from Assurex Health, Myriad, Mayo Foundation, and Medibio; he has served as a consultant for Actify Neurotherapies, Allergan, Intra-Cellular Therapies, Inc., Janssen, Myriad, Neuralstem Inc., Sanofi, Takeda, Teva Pharmaceuticals; he has received CME Travel/Honoraria from American Physician Institute, CME Outfitters, Global Academy for Medical Education. All other authors declared no competing interests for this work.

## Footnotes

## Supplementary information

The online version of this article (10.1038/s41380-020-00940-y) contains supplementary material, which is available to authorized users.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Martinowich K, Jimenez DV, Zarate CA, Jr., Manji HK. Rapid antidepressant effects: moving right along. Mol Psychiatry. 2013;18:856–63. doi: 10.1038/mp.2013.55.  [DOI](https://doi.org/10.1038/mp.2013.55) | [PMC free article](/articles/PMC3790255/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23689537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Rapid%20antidepressant%20effects:%20moving%20right%20along&author=K%20Martinowich&author=DV%20Jimenez&author=CA%20Zarate&author=HK%20Manji&volume=18&publication_year=2013&pages=856-63&pmid=23689537&doi=10.1038/mp.2013.55&)

2. Kessler RC, Akiskal HS, Ames M, Birnbaum H, Greenberg P, Hirschfeld RM, et al. Prevalence and effects of mood disorders on work performance in a nationally representative sample of U.S. workers. Am J Psychiatry. 2006;163:1561–8. doi: 10.1176/ajp.2006.163.9.1561.  [DOI](https://doi.org/10.1176/ajp.2006.163.9.1561) | [PMC free article](/articles/PMC1924724/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16946181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Prevalence%20and%20effects%20of%20mood%20disorders%20on%20work%20performance%20in%20a%20nationally%20representative%20sample%20of%20U.S.%20workers&author=RC%20Kessler&author=HS%20Akiskal&author=M%20Ames&author=H%20Birnbaum&author=P%20Greenberg&volume=163&publication_year=2006&pages=1561-8&pmid=16946181&doi=10.1176/ajp.2006.163.9.1561&)

3. Gelenberg AJ. A review of the current guidelines for depression treatment. J Clin Psychiatry. 2010;71:e15. doi: 10.4088/JCP.9078tx1c.  [DOI](https://doi.org/10.4088/JCP.9078tx1c) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20667285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychiatry&title=A%20review%20of%20the%20current%20guidelines%20for%20depression%20treatment&author=AJ%20Gelenberg&volume=71&publication_year=2010&pages=e15&pmid=20667285&doi=10.4088/JCP.9078tx1c&)

4. Trivedi MH, Rush AJ, Wisniewski SR, Nierenberg AA, Warden D, Ritz L, et al. Evaluation of outcomes with citalopram for depression using measurement-based care in STAR*D: implications for clinical practice. Am J Psychiatry. 2006;163:28–40. doi: 10.1176/appi.ajp.163.1.28.  [DOI](https://doi.org/10.1176/appi.ajp.163.1.28) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16390886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Evaluation%20of%20outcomes%20with%20citalopram%20for%20depression%20using%20measurement-based%20care%20in%20STAR*D:%20implications%20for%20clinical%20practice&author=MH%20Trivedi&author=AJ%20Rush&author=SR%20Wisniewski&author=AA%20Nierenberg&author=D%20Warden&volume=163&publication_year=2006&pages=28-40&pmid=16390886&doi=10.1176/appi.ajp.163.1.28&)

5. Ising M, Lucae S, Binder EB, Bettecken T, Uhr M, Ripke S, et al. A genomewide association study points to multiple loci that predict antidepressant drug treatment outcome in depression. Arch Gen Psychiatry. 2009;66:966–75. doi: 10.1001/archgenpsychiatry.2009.95.  [DOI](https://doi.org/10.1001/archgenpsychiatry.2009.95) | [PMC free article](/articles/PMC4465570/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19736353/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Gen%20Psychiatry&title=A%20genomewide%20association%20study%20points%20to%20multiple%20loci%20that%20predict%20antidepressant%20drug%20treatment%20outcome%20in%20depression&author=M%20Ising&author=S%20Lucae&author=EB%20Binder&author=T%20Bettecken&author=M%20Uhr&volume=66&publication_year=2009&pages=966-75&pmid=19736353&doi=10.1001/archgenpsychiatry.2009.95&)

6. Garriock HA, Kraft JB, Shyn SI, Peters EJ, Yokoyama JS, Jenkins GD, et al. A genomewide association study of citalopram response in major depressive disorder. Biol Psychiatry. 2010;67:133–8. doi: 10.1016/j.biopsych.2009.08.029.  [DOI](https://doi.org/10.1016/j.biopsych.2009.08.029) | [PMC free article](/articles/PMC2794921/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19846067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=A%20genomewide%20association%20study%20of%20citalopram%20response%20in%20major%20depressive%20disorder&author=HA%20Garriock&author=JB%20Kraft&author=SI%20Shyn&author=EJ%20Peters&author=JS%20Yokoyama&volume=67&publication_year=2010&pages=133-8&pmid=19846067&doi=10.1016/j.biopsych.2009.08.029&)

7. Uher R, Perroud N, Ng MY, Hauser J, Henigsberg N, Maier W, et al. Genome-wide pharmacogenetics of antidepressant response in the GENDEP project. Am J Psychiatry. 2010;167:555–64. doi: 10.1176/appi.ajp.2009.09070932.  [DOI](https://doi.org/10.1176/appi.ajp.2009.09070932) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20360315/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Genome-wide%20pharmacogenetics%20of%20antidepressant%20response%20in%20the%20GENDEP%20project&author=R%20Uher&author=N%20Perroud&author=MY%20Ng&author=J%20Hauser&author=N%20Henigsberg&volume=167&publication_year=2010&pages=555-64&pmid=20360315&doi=10.1176/appi.ajp.2009.09070932&)

8. Ji Y, Biernacka JM, Hebbring S, Chai Y, Jenkins GD, Batzler A, et al. Pharmacogenomics of selective serotonin reuptake inhibitor treatment for major depressive disorder: genome-wide associations and functional genomics. Pharmacogenomics J. 2013;13:456–63. doi: 10.1038/tpj.2012.32.  [DOI](https://doi.org/10.1038/tpj.2012.32) | [PMC free article](/articles/PMC3941038/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22907730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Pharmacogenomics%20of%20selective%20serotonin%20reuptake%20inhibitor%20treatment%20for%20major%20depressive%20disorder:%20genome-wide%20associations%20and%20functional%20genomics&author=Y%20Ji&author=JM%20Biernacka&author=S%20Hebbring&author=Y%20Chai&author=GD%20Jenkins&volume=13&publication_year=2013&pages=456-63&pmid=22907730&doi=10.1038/tpj.2012.32&)

9. Biernacka JM, Sangkuhl K, Jenkins G, Whaley RM, Barman P, Batzler A, et al. The International SSRI Pharmacogenomics Consortium (ISPC): a genome-wide association study of antidepressant treatment response. Transl Psychiatry. 2015;5:e553. doi: 10.1038/tp.2015.47.  [DOI](https://doi.org/10.1038/tp.2015.47) | [PMC free article](/articles/PMC4462610/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25897834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transl%20Psychiatry&title=The%20International%20SSRI%20Pharmacogenomics%20Consortium%20(ISPC):%20a%20genome-wide%20association%20study%20of%20antidepressant%20treatment%20response&author=JM%20Biernacka&author=K%20Sangkuhl&author=G%20Jenkins&author=RM%20Whaley&author=P%20Barman&volume=5&publication_year=2015&pages=e553&pmid=25897834&doi=10.1038/tp.2015.47&)

10. GENDEP-Investigators MARS-Investigators. STAR*D-Investigators. Common genetic variation and antidepressant efficacy in major depressive disorder: a meta-analysis of three genome-wide pharmacogenetic studies. Am J Psychiatry. 2013;170:207–17. doi: 10.1176/appi.ajp.2012.12020237.  [DOI](https://doi.org/10.1176/appi.ajp.2012.12020237) | [PMC free article](/articles/PMC10416089/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23377640/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=STAR*D-Investigators.%20Common%20genetic%20variation%20and%20antidepressant%20efficacy%20in%20major%20depressive%20disorder:%20a%20meta-analysis%20of%20three%20genome-wide%20pharmacogenetic%20studies&author=MARS-Investigators%20GENDEP-Investigators&volume=170&publication_year=2013&pages=207-17&pmid=23377640&doi=10.1176/appi.ajp.2012.12020237&)

11. Fabbri C, Tansey KE, Perlis RH, Hauser J, Henigsberg N, Maier W, et al. New insights into the pharmacogenomics of antidepressant response from the GENDEP and STAR*D studies: rare variant analysis and high-density imputation. Pharmacogenomics J. 2018;18:413–21. doi: 10.1038/tpj.2017.44.  [DOI](https://doi.org/10.1038/tpj.2017.44) | [PMC free article](/articles/PMC10204124/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29160301/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=New%20insights%20into%20the%20pharmacogenomics%20of%20antidepressant%20response%20from%20the%20GENDEP%20and%20STAR*D%20studies:%20rare%20variant%20analysis%20and%20high-density%20imputation&author=C%20Fabbri&author=KE%20Tansey&author=RH%20Perlis&author=J%20Hauser&author=N%20Henigsberg&volume=18&publication_year=2018&pages=413-21&pmid=29160301&doi=10.1038/tpj.2017.44&)

12. Krishnan V, Nestler EJ. The molecular neurobiology of depression. Nature. 2008;455:894–902. doi: 10.1038/nature07455.  [DOI](https://doi.org/10.1038/nature07455) | [PMC free article](/articles/PMC2721780/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18923511/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20molecular%20neurobiology%20of%20depression&author=V%20Krishnan&author=EJ%20Nestler&volume=455&publication_year=2008&pages=894-902&pmid=18923511&doi=10.1038/nature07455&)

13. Neavin D, Kaddurah-Daouk R, Weinshilboum R. Pharmacometabolomics informs Pharmacogenomics. Metabolomics. 2016;12:121. doi: 10.1007/s11306-016-1066-x.  [DOI](https://doi.org/10.1007/s11306-016-1066-x) | [PMC free article](/articles/PMC4976774/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27516730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Metabolomics&title=Pharmacometabolomics%20informs%20Pharmacogenomics&author=D%20Neavin&author=R%20Kaddurah-Daouk&author=R%20Weinshilboum&volume=12&publication_year=2016&pages=121&pmid=27516730&doi=10.1007/s11306-016-1066-x&)

14. Liu D, Ray B, Neavin DR, Zhang J, Athreya AP, Biernacka JM, et al. Beta-defensin 1, aryl hydrocarbon receptor and plasma kynurenine in major depressive disorder: metabolomics-informed genomics. Transl Psychiatry. 2018;8:10. doi: 10.1038/s41398-017-0056-8.  [DOI](https://doi.org/10.1038/s41398-017-0056-8) | [PMC free article](/articles/PMC5802574/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29317604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transl%20Psychiatry&title=Beta-defensin%201,%20aryl%20hydrocarbon%20receptor%20and%20plasma%20kynurenine%20in%20major%20depressive%20disorder:%20metabolomics-informed%20genomics&author=D%20Liu&author=B%20Ray&author=DR%20Neavin&author=J%20Zhang&author=AP%20Athreya&volume=8&publication_year=2018&pages=10&pmid=29317604&doi=10.1038/s41398-017-0056-8&)

15. Gupta M, Neavin D, Liu D, Biernacka J, Hall-Flavin D, Bobo WV, et al. TSPAN5, ERICH3 and selective serotonin reuptake inhibitors in major depressive disorder: pharmacometabolomics-informed pharmacogenomics. Mol Psychiatry. 2016;21:1717–25. doi: 10.1038/mp.2016.6.  [DOI](https://doi.org/10.1038/mp.2016.6) | [PMC free article](/articles/PMC5003027/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26903268/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=TSPAN5,%20ERICH3%20and%20selective%20serotonin%20reuptake%20inhibitors%20in%20major%20depressive%20disorder:%20pharmacometabolomics-informed%20pharmacogenomics&author=M%20Gupta&author=D%20Neavin&author=D%20Liu&author=J%20Biernacka&author=D%20Hall-Flavin&volume=21&publication_year=2016&pages=1717-25&pmid=26903268&doi=10.1038/mp.2016.6&)

16. Bhattacharyya S, Ahmed AT, Arnold M, Liu D, Luo C, Zhu H, et al. Metabolomic signature of exposure and response to citalopram/escitalopram in depressed outpatients. Transl Psychiatry. 2019;9:173. doi: 10.1038/s41398-019-0507-5.  [DOI](https://doi.org/10.1038/s41398-019-0507-5) | [PMC free article](/articles/PMC6609722/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31273200/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transl%20Psychiatry&title=Metabolomic%20signature%20of%20exposure%20and%20response%20to%20citalopram/escitalopram%20in%20depressed%20outpatients&author=S%20Bhattacharyya&author=AT%20Ahmed&author=M%20Arnold&author=D%20Liu&author=C%20Luo&volume=9&publication_year=2019&pages=173&pmid=31273200&doi=10.1038/s41398-019-0507-5&)

17. Holck A, Wolkowitz OM, Mellon SH, Reus VI, Nelson JC, Westrin A, et al. Plasma serotonin levels are associated with antidepressant response to SSRIs. J Affect Disord. 2019;250:65–70. doi: 10.1016/j.jad.2019.02.063.  [DOI](https://doi.org/10.1016/j.jad.2019.02.063) | [PMC free article](/articles/PMC6699768/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30831543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Affect%20Disord&title=Plasma%20serotonin%20levels%20are%20associated%20with%20antidepressant%20response%20to%20SSRIs&author=A%20Holck&author=OM%20Wolkowitz&author=SH%20Mellon&author=VI%20Reus&author=JC%20Nelson&volume=250&publication_year=2019&pages=65-70&pmid=30831543&doi=10.1016/j.jad.2019.02.063&)

18. Sun Y, Drevets W, Turecki G, Li QS. The relationship between plasma serotonin and kynurenine pathway metabolite levels and the treatment response to escitalopram and desvenlafaxine. Brain Behav Immun. 2020;87:404–12. doi: 10.1016/j.bbi.2020.01.011.  [DOI](https://doi.org/10.1016/j.bbi.2020.01.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31978524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Behav%20Immun&title=The%20relationship%20between%20plasma%20serotonin%20and%20kynurenine%20pathway%20metabolite%20levels%20and%20the%20treatment%20response%20to%20escitalopram%20and%20desvenlafaxine&author=Y%20Sun&author=W%20Drevets&author=G%20Turecki&author=QS%20Li&volume=87&publication_year=2020&pages=404-12&pmid=31978524&doi=10.1016/j.bbi.2020.01.011&)

19. Genomes Project C, Abecasis GR, Altshuler D, Auton A, Brooks LD, Durbin RM, et al. A map of human genome variation from population-scale sequencing. Nature. 2010;467:1061–73. doi: 10.1038/nature09534.  [DOI](https://doi.org/10.1038/nature09534) | [PMC free article](/articles/PMC3042601/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20981092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20map%20of%20human%20genome%20variation%20from%20population-scale%20sequencing&author=C%20Genomes%20Project&author=GR%20Abecasis&author=D%20Altshuler&author=A%20Auton&author=LD%20Brooks&volume=467&publication_year=2010&pages=1061-73&pmid=20981092&doi=10.1038/nature09534&)

20. GTExConsortium he genotype-tissue expression (GTEx) project. Nat Genet. 2013;45:580–5. doi: 10.1038/ng.2653.  [DOI](https://doi.org/10.1038/ng.2653) | [PMC free article](/articles/PMC4010069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23715323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=he%20genotype-tissue%20expression%20(GTEx)%20project&volume=45&publication_year=2013&pages=580-5&pmid=23715323&doi=10.1038/ng.2653&)

21. Fagerberg L, Hallstrom BM, Oksvold P, Kampf C, Djureinovic D, Odeberg J, et al. Analysis of the human tissue-specific expression by genome-wide integration of transcriptomics and antibody-based proteomics. Mol Cell Proteom. 2014;13:397–406. doi: 10.1074/mcp.M113.035600.  [DOI](https://doi.org/10.1074/mcp.M113.035600) | [PMC free article](/articles/PMC3916642/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24309898/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell%20Proteom&title=Analysis%20of%20the%20human%20tissue-specific%20expression%20by%20genome-wide%20integration%20of%20transcriptomics%20and%20antibody-based%20proteomics&author=L%20Fagerberg&author=BM%20Hallstrom&author=P%20Oksvold&author=C%20Kampf&author=D%20Djureinovic&volume=13&publication_year=2014&pages=397-406&pmid=24309898&doi=10.1074/mcp.M113.035600&)

22. Duff MO, Olson S, Wei X, Garrett SC, Osman A, Bolisetty M, et al. Genome-wide identification of zero nucleotide recursive splicing in drosophila. Nature. 2015;521:376–9. doi: 10.1038/nature14475.  [DOI](https://doi.org/10.1038/nature14475) | [PMC free article](/articles/PMC4529404/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25970244/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genome-wide%20identification%20of%20zero%20nucleotide%20recursive%20splicing%20in%20drosophila&author=MO%20Duff&author=S%20Olson&author=X%20Wei&author=SC%20Garrett&author=A%20Osman&volume=521&publication_year=2015&pages=376-9&pmid=25970244&doi=10.1038/nature14475&)

23. Wilhelm M, Schlegl J, Hahne H, Gholami AM, Lieberenz M, Savitski MM, et al. Mass-spectrometry-based draft of the human proteome. Nature. 2014;509:582–7. doi: 10.1038/nature13319.  [DOI](https://doi.org/10.1038/nature13319) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24870543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Mass-spectrometry-based%20draft%20of%20the%20human%20proteome&author=M%20Wilhelm&author=J%20Schlegl&author=H%20Hahne&author=AM%20Gholami&author=M%20Lieberenz&volume=509&publication_year=2014&pages=582-7&pmid=24870543&doi=10.1038/nature13319&)

24. Schmidt T, Samaras P, Frejno M, Gessulat S, Barnert M, Kienegger H, et al. ProteomicsDB. Nucleic Acids Res. 2018;46:D1271–D1281. doi: 10.1093/nar/gkx1029.  [DOI](https://doi.org/10.1093/nar/gkx1029) | [PMC free article](/articles/PMC5753189/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29106664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=ProteomicsDB&author=T%20Schmidt&author=P%20Samaras&author=M%20Frejno&author=S%20Gessulat&author=M%20Barnert&volume=46&publication_year=2018&pages=D1271-D1281&pmid=29106664&doi=10.1093/nar/gkx1029&)

25. McInnes G, Tanigawa Y, DeBoever C, Lavertu A, Olivieri JE, Aguirre M, et al. Global Biobank Engine: enabling genotype-phenotype browsing for biobank summary statistics. Bioinformatics. 2019;35:2495–7. doi: 10.1093/bioinformatics/bty999.  [DOI](https://doi.org/10.1093/bioinformatics/bty999) | [PMC free article](/articles/PMC6612820/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30520965/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Global%20Biobank%20Engine:%20enabling%20genotype-phenotype%20browsing%20for%20biobank%20summary%20statistics&author=G%20McInnes&author=Y%20Tanigawa&author=C%20DeBoever&author=A%20Lavertu&author=JE%20Olivieri&volume=35&publication_year=2019&pages=2495-7&pmid=30520965&doi=10.1093/bioinformatics/bty999&)

26. Gandal MJ, Zhang P, Hadjimichael E, Walker RL, Chen C, Liu S, et al. Transcriptome-wide isoform-level dysregulation in ASD, schizophrenia, and bipolar disorder. Science. 2018;362:eaat8127. doi: 10.1126/science.aat8127.  [DOI](https://doi.org/10.1126/science.aat8127) | [PMC free article](/articles/PMC6443102/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30545856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Transcriptome-wide%20isoform-level%20dysregulation%20in%20ASD,%20schizophrenia,%20and%20bipolar%20disorder&author=MJ%20Gandal&author=P%20Zhang&author=E%20Hadjimichael&author=RL%20Walker&author=C%20Chen&volume=362&publication_year=2018&pages=eaat8127&pmid=30545856&doi=10.1126/science.aat8127&)

27. Darmanis S, Sloan SA, Croote D, Mignardi M, Chernikova S, Samghababi P, et al. Single-cell RNA-Seq analysis of infiltrating neoplastic cells at the migrating front of human glioblastoma. Cell Rep. 2017;21:1399–410. doi: 10.1016/j.celrep.2017.10.030.  [DOI](https://doi.org/10.1016/j.celrep.2017.10.030) | [PMC free article](/articles/PMC5810554/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29091775/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&title=Single-cell%20RNA-Seq%20analysis%20of%20infiltrating%20neoplastic%20cells%20at%20the%20migrating%20front%20of%20human%20glioblastoma&author=S%20Darmanis&author=SA%20Sloan&author=D%20Croote&author=M%20Mignardi&author=S%20Chernikova&volume=21&publication_year=2017&pages=1399-410&pmid=29091775&doi=10.1016/j.celrep.2017.10.030&)

28. Darmanis S, Sloan SA, Zhang Y, Enge M, Caneda C, Shuer LM, et al. A survey of human brain transcriptome diversity at the single cell level. Proc Natl Acad Sci USA. 2015;112:7285–90. doi: 10.1073/pnas.1507125112.  [DOI](https://doi.org/10.1073/pnas.1507125112) | [PMC free article](/articles/PMC4466750/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26060301/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=A%20survey%20of%20human%20brain%20transcriptome%20diversity%20at%20the%20single%20cell%20level&author=S%20Darmanis&author=SA%20Sloan&author=Y%20Zhang&author=M%20Enge&author=C%20Caneda&volume=112&publication_year=2015&pages=7285-90&pmid=26060301&doi=10.1073/pnas.1507125112&)

29. Fan J, Salathia N, Liu R, Kaeser GE, Yung YC, Herman JL, et al. Characterizing transcriptional heterogeneity through pathway and gene set overdispersion analysis. Nat Methods. 2016;13:241–4. doi: 10.1038/nmeth.3734.  [DOI](https://doi.org/10.1038/nmeth.3734) | [PMC free article](/articles/PMC4772672/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26780092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=Characterizing%20transcriptional%20heterogeneity%20through%20pathway%20and%20gene%20set%20overdispersion%20analysis&author=J%20Fan&author=N%20Salathia&author=R%20Liu&author=GE%20Kaeser&author=YC%20Yung&volume=13&publication_year=2016&pages=241-4&pmid=26780092&doi=10.1038/nmeth.3734&)

30. Becht E, McInnes L, Healy J, Dutertre C-A, Kwok IWH, Ng LG, et al. Dimensionality reduction for visualizing single-cell data using UMAP. Nat Biotechnol. 2018;37:38. doi: 10.1038/nbt.4314.  [DOI](https://doi.org/10.1038/nbt.4314) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30531897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Dimensionality%20reduction%20for%20visualizing%20single-cell%20data%20using%20UMAP&author=E%20Becht&author=L%20McInnes&author=J%20Healy&author=C-A%20Dutertre&author=IWH%20Kwok&volume=37&publication_year=2018&pages=38&pmid=30531897&doi=10.1038/nbt.4314&)

31. Dunlop BW, Kelley ME, Aponte-Rivera V, Mletzko-Crowe T, Kinkead B, Ritchie JC, et al. Effects of patient preferences on outcomes in the predictors of remission in depression to individual and combined treatments (PReDICT) study. Am J Psychiatry. 2017;174:546–56. doi: 10.1176/appi.ajp.2016.16050517.  [DOI](https://doi.org/10.1176/appi.ajp.2016.16050517) | [PMC free article](/articles/PMC6690210/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28335624/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Effects%20of%20patient%20preferences%20on%20outcomes%20in%20the%20predictors%20of%20remission%20in%20depression%20to%20individual%20and%20combined%20treatments%20(PReDICT)%20study&author=BW%20Dunlop&author=ME%20Kelley&author=V%20Aponte-Rivera&author=T%20Mletzko-Crowe&author=B%20Kinkead&volume=174&publication_year=2017&pages=546-56&pmid=28335624&doi=10.1176/appi.ajp.2016.16050517&)

32. Dunlop BW, Binder EB, Cubells JF, Goodman MM, Kelley ME, Kinkead B, et al. Predictors of remission in depression to individual and combined treatments (PReDICT): study protocol for a randomized controlled trial. Trials. 2012;13:106. doi: 10.1186/1745-6215-13-106.  [DOI](https://doi.org/10.1186/1745-6215-13-106) | [PMC free article](/articles/PMC3539869/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22776534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trials&title=Predictors%20of%20remission%20in%20depression%20to%20individual%20and%20combined%20treatments%20(PReDICT):%20study%20protocol%20for%20a%20randomized%20controlled%20trial&author=BW%20Dunlop&author=EB%20Binder&author=JF%20Cubells&author=MM%20Goodman&author=ME%20Kelley&volume=13&publication_year=2012&pages=106&pmid=22776534&doi=10.1186/1745-6215-13-106&)

33. Mrazek DA, Biernacka JM, McAlpine DE, Benitez J, Karpyak VM, Williams MD, et al. Treatment outcomes of depression: the pharmacogenomic research network antidepressant medication pharmacogenomic study. J Clin Psychopharmacol. 2014;34:313–7. doi: 10.1097/JCP.0000000000000099.  [DOI](https://doi.org/10.1097/JCP.0000000000000099) | [PMC free article](/articles/PMC3992481/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24743713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychopharmacol&title=Treatment%20outcomes%20of%20depression:%20the%20pharmacogenomic%20research%20network%20antidepressant%20medication%20pharmacogenomic%20study&author=DA%20Mrazek&author=JM%20Biernacka&author=DE%20McAlpine&author=J%20Benitez&author=VM%20Karpyak&volume=34&publication_year=2014&pages=313-7&pmid=24743713&doi=10.1097/JCP.0000000000000099&)

34. O’Connell CP, Goldstein-Piekarski AN, Nemeroff CB, Schatzberg AF, Debattista C, Carrillo-Roa T, et al. Antidepressant outcomes predicted by genetic variation in corticotropin-releasing hormone binding protein. Am J Psychiatry. 2018;175:251–61. doi: 10.1176/appi.ajp.2017.17020172.  [DOI](https://doi.org/10.1176/appi.ajp.2017.17020172) | [PMC free article](/articles/PMC5832545/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29241359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Antidepressant%20outcomes%20predicted%20by%20genetic%20variation%20in%20corticotropin-releasing%20hormone%20binding%20protein&author=CP%20O%E2%80%99Connell&author=AN%20Goldstein-Piekarski&author=CB%20Nemeroff&author=AF%20Schatzberg&author=C%20Debattista&volume=175&publication_year=2018&pages=251-61&pmid=29241359&doi=10.1176/appi.ajp.2017.17020172&)

35. Wei BY, McCarthy M, Ren H, Carrillo-Roa T, Shekhtman T, DeModena A, et al. A functional variant in the serotonin receptor 7 gene (HTR7), rs7905446, is associated with good response to SSRIs in bipolar and unipolar depression. Mol Psychiatry. 2020;25:1312–22. doi: 10.1038/s41380-019-0397-1.  [DOI](https://doi.org/10.1038/s41380-019-0397-1) | [PMC free article](/articles/PMC6745302/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30874608/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=A%20functional%20variant%20in%20the%20serotonin%20receptor%207%20gene%20(HTR7),%20rs7905446,%20is%20associated%20with%20good%20response%20to%20SSRIs%20in%20bipolar%20and%20unipolar%20depression&author=BY%20Wei&author=M%20McCarthy&author=H%20Ren&author=T%20Carrillo-Roa&author=T%20Shekhtman&volume=25&publication_year=2020&pages=1312-22&pmid=30874608&doi=10.1038/s41380-019-0397-1&)

36. Hodge RD, Bakken TE, Miller JA, Smith KA, Barkan ER, Graybuck LT, et al. Conserved cell types with divergent features in human versus mouse cortex. Nature. 2019;573:61–68. doi: 10.1038/s41586-019-1506-7.  [DOI](https://doi.org/10.1038/s41586-019-1506-7) | [PMC free article](/articles/PMC6919571/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31435019/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Conserved%20cell%20types%20with%20divergent%20features%20in%20human%20versus%20mouse%20cortex&author=RD%20Hodge&author=TE%20Bakken&author=JA%20Miller&author=KA%20Smith&author=ER%20Barkan&volume=573&publication_year=2019&pages=61-68&pmid=31435019&doi=10.1038/s41586-019-1506-7&)

37. La Manno G, Gyllborg D, Codeluppi S, Nishimura K, Salto C, Zeisel A, et al. Molecular diversity of midbrain development in mouse, human, and stem. Cells Cell. 2016;167:566–80.e519. doi: 10.1016/j.cell.2016.09.027.  [DOI](https://doi.org/10.1016/j.cell.2016.09.027) | [PMC free article](/articles/PMC5055122/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27716510/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cells%20Cell&title=Molecular%20diversity%20of%20midbrain%20development%20in%20mouse,%20human,%20and%20stem&author=G%20La%20Manno&author=D%20Gyllborg&author=S%20Codeluppi&author=K%20Nishimura&author=C%20Salto&volume=167&publication_year=2016&pages=566-80.e519&pmid=27716510&doi=10.1016/j.cell.2016.09.027&)

38. Klijn C, Durinck S, Stawiski EW, Haverty PM, Jiang Z, Liu H, et al. A comprehensive transcriptional portrait of human cancer cell lines. Nat Biotechnol. 2015;33:306–12. doi: 10.1038/nbt.3080.  [DOI](https://doi.org/10.1038/nbt.3080) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25485619/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=A%20comprehensive%20transcriptional%20portrait%20of%20human%20cancer%20cell%20lines&author=C%20Klijn&author=S%20Durinck&author=EW%20Stawiski&author=PM%20Haverty&author=Z%20Jiang&volume=33&publication_year=2015&pages=306-12&pmid=25485619&doi=10.1038/nbt.3080&)

39. Guan Y, Zhu Q, Huang D, Zhao S, Jan LoL, Peng J. An equation to estimate the difference between theoretically predicted and SDS PAGE-displayed molecular weights for an acidic peptide. Sci Rep. 2015;5:13370. doi: 10.1038/srep13370.  [DOI](https://doi.org/10.1038/srep13370) | [PMC free article](/articles/PMC4550835/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26311515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep.&title=An%20equation%20to%20estimate%20the%20difference%20between%20theoretically%20predicted%20and%20SDS%20PAGE-displayed%20molecular%20weights%20for%20an%20acidic%20peptide&author=Y%20Guan&author=Q%20Zhu&author=D%20Huang&author=S%20Zhao&author=LoL%20Jan&volume=5&publication_year=2015&pages=13370&pmid=26311515&doi=10.1038/srep13370&)

40. Kaksonen M, Roux A. Mechanisms of clathrin-mediated endocytosis. Nat Rev Mol Cell Biol. 2018;19:313–26. doi: 10.1038/nrm.2017.132.  [DOI](https://doi.org/10.1038/nrm.2017.132) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29410531/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&title=Mechanisms%20of%20clathrin-mediated%20endocytosis&author=M%20Kaksonen&author=A%20Roux&volume=19&publication_year=2018&pages=313-26&pmid=29410531&doi=10.1038/nrm.2017.132&)

41. McMahon HT, Boucrot E. Molecular mechanism and physiological functions of clathrin-mediated endocytosis. Nat Rev Mol Cell Biol. 2011;12:517–33. doi: 10.1038/nrm3151.  [DOI](https://doi.org/10.1038/nrm3151) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21779028/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&title=Molecular%20mechanism%20and%20physiological%20functions%20of%20clathrin-mediated%20endocytosis&author=HT%20McMahon&author=E%20Boucrot&volume=12&publication_year=2011&pages=517-33&pmid=21779028&doi=10.1038/nrm3151&)

42. Gaidarov I, Smith ME, Domin J, Keen JH. The class II phosphoinositide 3-kinase C2alpha is activated by clathrin and regulates clathrin-mediated membrane trafficking. Mol Cell. 2001;7:443–9. doi: 10.1016/S1097-2765(01)00191-5.  [DOI](https://doi.org/10.1016/S1097-2765(01)00191-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11239472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell&title=The%20class%20II%20phosphoinositide%203-kinase%20C2alpha%20is%20activated%20by%20clathrin%20and%20regulates%20clathrin-mediated%20membrane%20trafficking&author=I%20Gaidarov&author=ME%20Smith&author=J%20Domin&author=JH%20Keen&volume=7&publication_year=2001&pages=443-9&pmid=11239472&doi=10.1016/S1097-2765(01)00191-5&)

43. Conner SD, Schmid SL. Identification of an adaptor-associated kinase, AAK1, as a regulator of clathrin-mediated endocytosis. J Cell Biol. 2002;156:921–9. doi: 10.1083/jcb.200108123.  [DOI](https://doi.org/10.1083/jcb.200108123) | [PMC free article](/articles/PMC2173317/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11877461/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&title=Identification%20of%20an%20adaptor-associated%20kinase,%20AAK1,%20as%20a%20regulator%20of%20clathrin-mediated%20endocytosis&author=SD%20Conner&author=SL%20Schmid&volume=156&publication_year=2002&pages=921-9&pmid=11877461&doi=10.1083/jcb.200108123&)

44. Hayashida M, Tanifuji S, Ma H, Murakami N, Mochida S. Neural activity selects myosin IIB and VI with a specific time window in distinct dynamin isoform-mediated synaptic vesicle reuse pathways. J Neurosci. 2015;35:8901–13. doi: 10.1523/JNEUROSCI.5028-14.2015.  [DOI](https://doi.org/10.1523/JNEUROSCI.5028-14.2015) | [PMC free article](/articles/PMC6605209/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26063922/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=Neural%20activity%20selects%20myosin%20IIB%20and%20VI%20with%20a%20specific%20time%20window%20in%20distinct%20dynamin%20isoform-mediated%20synaptic%20vesicle%20reuse%20pathways&author=M%20Hayashida&author=S%20Tanifuji&author=H%20Ma&author=N%20Murakami&author=S%20Mochida&volume=35&publication_year=2015&pages=8901-13&pmid=26063922&doi=10.1523/JNEUROSCI.5028-14.2015&)

45. Royle SJ, Bright NA, Lagnado L. Clathrin is required for the function of the mitotic spindle. Nature. 2005;434:1152–7. doi: 10.1038/nature03502.  [DOI](https://doi.org/10.1038/nature03502) | [PMC free article](/articles/PMC3492753/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15858577/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Clathrin%20is%20required%20for%20the%20function%20of%20the%20mitotic%20spindle&author=SJ%20Royle&author=NA%20Bright&author=L%20Lagnado&volume=434&publication_year=2005&pages=1152-7&pmid=15858577&doi=10.1038/nature03502&)

46. Foraker AB, Camus SM, Evans TM, Majeed SR, Chen CY, Taner SB, et al. Clathrin promotes centrosome integrity in early mitosis through stabilization of centrosomal ch-TOG. J Cell Biol. 2012;198:591–605. doi: 10.1083/jcb.201205116.  [DOI](https://doi.org/10.1083/jcb.201205116) | [PMC free article](/articles/PMC3514040/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22891263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&title=Clathrin%20promotes%20centrosome%20integrity%20in%20early%20mitosis%20through%20stabilization%20of%20centrosomal%20ch-TOG&author=AB%20Foraker&author=SM%20Camus&author=TM%20Evans&author=SR%20Majeed&author=CY%20Chen&volume=198&publication_year=2012&pages=591-605&pmid=22891263&doi=10.1083/jcb.201205116&)

47. Sasayama D, Hiraishi A, Tatsumi M, Kamijima K, Ikeda M, Umene-Nakano W, et al. Possible association of CUX1 gene polymorphisms with antidepressant response in major depressive disorder. Pharmacogenomics J. 2013;13:354–8. doi: 10.1038/tpj.2012.18.  [DOI](https://doi.org/10.1038/tpj.2012.18) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22584459/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Possible%20association%20of%20CUX1%20gene%20polymorphisms%20with%20antidepressant%20response%20in%20major%20depressive%20disorder&author=D%20Sasayama&author=A%20Hiraishi&author=M%20Tatsumi&author=K%20Kamijima&author=M%20Ikeda&volume=13&publication_year=2013&pages=354-8&pmid=22584459&doi=10.1038/tpj.2012.18&)

48. Chen Y, Takita J, Choi YL, Kato M, Ohira M, Sanada M, et al. Oncogenic mutations of ALK kinase in neuroblastoma. Nature. 2008;455:971–4. doi: 10.1038/nature07399.  [DOI](https://doi.org/10.1038/nature07399) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18923524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Oncogenic%20mutations%20of%20ALK%20kinase%20in%20neuroblastoma&author=Y%20Chen&author=J%20Takita&author=YL%20Choi&author=M%20Kato&author=M%20Ohira&volume=455&publication_year=2008&pages=971-4&pmid=18923524&doi=10.1038/nature07399&)

49. Gouzi JY, Moog-Lutz C, Vigny M, Brunet-de Carvalho N. Role of the subcellular localization of ALK tyrosine kinase domain in neuronal differentiation of PC12 cells. J Cell Sci. 2005;118:5811–23. doi: 10.1242/jcs.02695.  [DOI](https://doi.org/10.1242/jcs.02695) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16317043/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&title=Role%20of%20the%20subcellular%20localization%20of%20ALK%20tyrosine%20kinase%20domain%20in%20neuronal%20differentiation%20of%20PC12%20cells&author=JY%20Gouzi&author=C%20Moog-Lutz&author=M%20Vigny&author=N%20Brunet-de%20Carvalho&volume=118&publication_year=2005&pages=5811-23&pmid=16317043&doi=10.1242/jcs.02695&)

50. Motegi A, Fujimoto J, Kotani M, Sakuraba H, Yamamoto T. ALK receptor tyrosine kinase promotes cell growth and neurite outgrowth. J Cell Sci. 2004;117:3319–29. doi: 10.1242/jcs.01183.  [DOI](https://doi.org/10.1242/jcs.01183) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15226403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&title=ALK%20receptor%20tyrosine%20kinase%20promotes%20cell%20growth%20and%20neurite%20outgrowth&author=A%20Motegi&author=J%20Fujimoto&author=M%20Kotani&author=H%20Sakuraba&author=T%20Yamamoto&volume=117&publication_year=2004&pages=3319-29&pmid=15226403&doi=10.1242/jcs.01183&)

51. Zhu S, Lee JS, Guo F, Shin J, Perez-Atayde AR, Kutok JL, et al. Activated ALK collaborates with MYCN in neuroblastoma pathogenesis. Cancer Cell. 2012;21:362–73. doi: 10.1016/j.ccr.2012.02.010.  [DOI](https://doi.org/10.1016/j.ccr.2012.02.010) | [PMC free article](/articles/PMC3315700/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22439933/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&title=Activated%20ALK%20collaborates%20with%20MYCN%20in%20neuroblastoma%20pathogenesis&author=S%20Zhu&author=JS%20Lee&author=F%20Guo&author=J%20Shin&author=AR%20Perez-Atayde&volume=21&publication_year=2012&pages=362-73&pmid=22439933&doi=10.1016/j.ccr.2012.02.010&)

52. Cubelos B, Sebastian-Serrano A, Beccari L, Calcagnotto ME, Cisneros E, Kim S, et al. Cux1 and Cux2 regulate dendritic branching, spine morphology, and synapses of the upper layer neurons of the cortex. Neuron. 2010;66:523–35. doi: 10.1016/j.neuron.2010.04.038.  [DOI](https://doi.org/10.1016/j.neuron.2010.04.038) | [PMC free article](/articles/PMC2894581/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20510857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Cux1%20and%20Cux2%20regulate%20dendritic%20branching,%20spine%20morphology,%20and%20synapses%20of%20the%20upper%20layer%20neurons%20of%20the%20cortex&author=B%20Cubelos&author=A%20Sebastian-Serrano&author=L%20Beccari&author=ME%20Calcagnotto&author=E%20Cisneros&volume=66&publication_year=2010&pages=523-35&pmid=20510857&doi=10.1016/j.neuron.2010.04.038&)

53. Nieto M, Monuki ES, Tang H, Imitola J, Haubst N, Khoury SJ, et al. Expression of Cux-1 and Cux-2 in the subventricular zone and upper layers II-IV of the cerebral cortex. J Comp Neurol. 2004;479:168–80. doi: 10.1002/cne.20322.  [DOI](https://doi.org/10.1002/cne.20322) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15452856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Comp%20Neurol&title=Expression%20of%20Cux-1%20and%20Cux-2%20in%20the%20subventricular%20zone%20and%20upper%20layers%20II-IV%20of%20the%20cerebral%20cortex&author=M%20Nieto&author=ES%20Monuki&author=H%20Tang&author=J%20Imitola&author=N%20Haubst&volume=479&publication_year=2004&pages=168-80&pmid=15452856&doi=10.1002/cne.20322&)

54. Rodriguez-Tornos FM, Briz CG, Weiss LA, Sebastian-Serrano A, Ares S, Navarrete M, et al. Cux1 enables interhemispheric connections of layer II/III neurons by regulating Kv1-dependent firing. Neuron. 2016;89:494–506. doi: 10.1016/j.neuron.2015.12.020.  [DOI](https://doi.org/10.1016/j.neuron.2015.12.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26804994/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Cux1%20enables%20interhemispheric%20connections%20of%20layer%20II/III%20neurons%20by%20regulating%20Kv1-dependent%20firing&author=FM%20Rodriguez-Tornos&author=CG%20Briz&author=LA%20Weiss&author=A%20Sebastian-Serrano&author=S%20Ares&volume=89&publication_year=2016&pages=494-506&pmid=26804994&doi=10.1016/j.neuron.2015.12.020&)

55. Li N, Zhao CT, Wang Y, Yuan XB. The transcription factor Cux1 regulates dendritic morphology of cortical pyramidal neurons. PLoS One. 2010;5:e10596. doi: 10.1371/journal.pone.0010596.  [DOI](https://doi.org/10.1371/journal.pone.0010596) | [PMC free article](/articles/PMC2868054/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20485671/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=The%20transcription%20factor%20Cux1%20regulates%20dendritic%20morphology%20of%20cortical%20pyramidal%20neurons&author=N%20Li&author=CT%20Zhao&author=Y%20Wang&author=XB%20Yuan&volume=5&publication_year=2010&pages=e10596&pmid=20485671&doi=10.1371/journal.pone.0010596&)

56. Baumert M, Maycox PR, Navone F, De Camilli P, Jahn R. Synaptobrevin: an integral membrane protein of 18,000 daltons present in small synaptic vesicles of rat brain. EMBO J. 1989;8:379–84. doi: 10.1002/j.1460-2075.1989.tb03388.x.  [DOI](https://doi.org/10.1002/j.1460-2075.1989.tb03388.x) | [PMC free article](/articles/PMC400817/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2498078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J&title=Synaptobrevin:%20an%20integral%20membrane%20protein%20of%2018,000%20daltons%20present%20in%20small%20synaptic%20vesicles%20of%20rat%20brain&author=M%20Baumert&author=PR%20Maycox&author=F%20Navone&author=P%20De%20Camilli&author=R%20Jahn&volume=8&publication_year=1989&pages=379-84&pmid=2498078&doi=10.1002/j.1460-2075.1989.tb03388.x&)

57. Jahn R, Schiebler W, Ouimet C, Greengard P. A 38,000-dalton membrane protein (p38) present in synaptic vesicles. Proc Natl Acad Sci USA. 1985;82:4137–41. doi: 10.1073/pnas.82.12.4137.  [DOI](https://doi.org/10.1073/pnas.82.12.4137) | [PMC free article](/articles/PMC397950/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3923488/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=A%2038,000-dalton%20membrane%20protein%20(p38)%20present%20in%20synaptic%20vesicles&author=R%20Jahn&author=W%20Schiebler&author=C%20Ouimet&author=P%20Greengard&volume=82&publication_year=1985&pages=4137-41&pmid=3923488&doi=10.1073/pnas.82.12.4137&)

58. Sudhof TC. The synaptic vesicle cycle. Annu Rev Neurosci. 2004;27:509–47. doi: 10.1146/annurev.neuro.26.041002.131412.  [DOI](https://doi.org/10.1146/annurev.neuro.26.041002.131412) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15217342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Neurosci&title=The%20synaptic%20vesicle%20cycle&author=TC%20Sudhof&volume=27&publication_year=2004&pages=509-47&pmid=15217342&doi=10.1146/annurev.neuro.26.041002.131412&)

59. Granseth B, Odermatt B, Royle SJ, Lagnado L. Clathrin-mediated endocytosis is the dominant mechanism of vesicle retrieval at hippocampal synapses. Neuron. 2006;51:773–86. doi: 10.1016/j.neuron.2006.08.029.  [DOI](https://doi.org/10.1016/j.neuron.2006.08.029) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16982422/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Clathrin-mediated%20endocytosis%20is%20the%20dominant%20mechanism%20of%20vesicle%20retrieval%20at%20hippocampal%20synapses&author=B%20Granseth&author=B%20Odermatt&author=SJ%20Royle&author=L%20Lagnado&volume=51&publication_year=2006&pages=773-86&pmid=16982422&doi=10.1016/j.neuron.2006.08.029&)

60. Lopez-Murcia FJ, Royle SJ, Llobet A. Presynaptic clathrin levels are a limiting factor for synaptic transmission. J Neurosci. 2014;34:8618–29. doi: 10.1523/JNEUROSCI.5081-13.2014.  [DOI](https://doi.org/10.1523/JNEUROSCI.5081-13.2014) | [PMC free article](/articles/PMC6608216/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24948816/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurosci&title=Presynaptic%20clathrin%20levels%20are%20a%20limiting%20factor%20for%20synaptic%20transmission&author=FJ%20Lopez-Murcia&author=SJ%20Royle&author=A%20Llobet&volume=34&publication_year=2014&pages=8618-29&pmid=24948816&doi=10.1523/JNEUROSCI.5081-13.2014&)

61. Delvendahl I, Vyleta NP, von Gersdorff H, Hallermann S. Fast, temperature-sensitive and clathrin-independent endocytosis at central synapses. Neuron. 2016;90:492–8. doi: 10.1016/j.neuron.2016.03.013.  [DOI](https://doi.org/10.1016/j.neuron.2016.03.013) | [PMC free article](/articles/PMC5125781/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27146271/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuron&title=Fast,%20temperature-sensitive%20and%20clathrin-independent%20endocytosis%20at%20central%20synapses&author=I%20Delvendahl&author=NP%20Vyleta&author=H%20von%20Gersdorff&author=S%20Hallermann&volume=90&publication_year=2016&pages=492-8&pmid=27146271&doi=10.1016/j.neuron.2016.03.013&)

62. Li YC, Kavalali ET. Synaptic vesicle-recycling machinery components as potential therapeutic targets. Pharm Rev. 2017;69:141–60. doi: 10.1124/pr.116.013342.  [DOI](https://doi.org/10.1124/pr.116.013342) | [PMC free article](/articles/PMC5394918/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28265000/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharm%20Rev&title=Synaptic%20vesicle-recycling%20machinery%20components%20as%20potential%20therapeutic%20targets&author=YC%20Li&author=ET%20Kavalali&volume=69&publication_year=2017&pages=141-60&pmid=28265000&doi=10.1124/pr.116.013342&)

63. Feighner JP. Mechanism of action of antidepressant medications. J Clin Psychiatry. 1999;60:12–13. doi: 10.4088/JCP.v60n1204.  [DOI](https://doi.org/10.4088/JCP.v60n1204) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10086478/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Psychiatry&title=Mechanism%20of%20action%20of%20antidepressant%20medications&author=JP%20Feighner&volume=60&publication_year=1999&pages=12-13&pmid=10086478&doi=10.4088/JCP.v60n1204&)

64. Artigas F, Nutt DJ, Shelton R. Mechanism of action of antidepressants. Psychopharmacol Bull. 2002;36:123–32.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/12490828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychopharmacol%20Bull&title=Mechanism%20of%20action%20of%20antidepressants&author=F%20Artigas&author=DJ%20Nutt&author=R%20Shelton&volume=36&publication_year=2002&pages=123-32&pmid=12490828&)

65. Stahl SM. Mechanism of action of serotonin selective reuptake inhibitors. Serotonin receptors and pathways mediate therapeutic effects and side effects. J Affect Disord. 1998;51:215–35. doi: 10.1016/S0165-0327(98)00221-3.  [DOI](https://doi.org/10.1016/S0165-0327(98)00221-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10333979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Affect%20Disord&title=Mechanism%20of%20action%20of%20serotonin%20selective%20reuptake%20inhibitors.%20Serotonin%20receptors%20and%20pathways%20mediate%20therapeutic%20effects%20and%20side%20effects&author=SM%20Stahl&volume=51&publication_year=1998&pages=215-35&pmid=10333979&doi=10.1016/S0165-0327(98)00221-3&)

66. Crane JD, Palanivel R, Mottillo EP, Bujak AL, Wang H, Ford RJ, et al. Inhibiting peripheral serotonin synthesis reduces obesity and metabolic dysfunction by promoting brown adipose tissue thermogenesis. Nat Med. 2015;21:166–72. doi: 10.1038/nm.3766.  [DOI](https://doi.org/10.1038/nm.3766) | [PMC free article](/articles/PMC5647161/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25485911/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=Inhibiting%20peripheral%20serotonin%20synthesis%20reduces%20obesity%20and%20metabolic%20dysfunction%20by%20promoting%20brown%20adipose%20tissue%20thermogenesis&author=JD%20Crane&author=R%20Palanivel&author=EP%20Mottillo&author=AL%20Bujak&author=H%20Wang&volume=21&publication_year=2015&pages=166-72&pmid=25485911&doi=10.1038/nm.3766&)

67. Gehin M, Welford RWD, Garzotti M, Vercauteren M, Groenen PMA, Nayler O, et al. Assessment of peripheral serotonin synthesis using stable isotope-labeled tryptophan. Clin Pharm Ther. 2018;104:1260–7. doi: 10.1002/cpt.1087.  [DOI](https://doi.org/10.1002/cpt.1087) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29663345/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Ther&title=Assessment%20of%20peripheral%20serotonin%20synthesis%20using%20stable%20isotope-labeled%20tryptophan&author=M%20Gehin&author=RWD%20Welford&author=M%20Garzotti&author=M%20Vercauteren&author=PMA%20Groenen&volume=104&publication_year=2018&pages=1260-7&pmid=29663345&doi=10.1002/cpt.1087&)

68. Berger M, Gray JA, Roth BL. The expanded biology of serotonin. Annu Rev Med. 2009;60:355–66. doi: 10.1146/annurev.med.60.042307.110802.  [DOI](https://doi.org/10.1146/annurev.med.60.042307.110802) | [PMC free article](/articles/PMC5864293/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19630576/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Med&title=The%20expanded%20biology%20of%20serotonin&author=M%20Berger&author=JA%20Gray&author=BL%20Roth&volume=60&publication_year=2009&pages=355-66&pmid=19630576&doi=10.1146/annurev.med.60.042307.110802&)

69. Ambrosio AL, Boyle JA, Di Pietro SM. Mechanism of platelet dense granule biogenesis: study of cargo transport and function of Rab32 and Rab38 in a model system. Blood. 2012;120:4072–81. doi: 10.1182/blood-2012-04-420745.  [DOI](https://doi.org/10.1182/blood-2012-04-420745) | [PMC free article](/articles/PMC3496959/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22927249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Mechanism%20of%20platelet%20dense%20granule%20biogenesis:%20study%20of%20cargo%20transport%20and%20function%20of%20Rab32%20and%20Rab38%20in%20a%20model%20system&author=AL%20Ambrosio&author=JA%20Boyle&author=SM%20Di%20Pietro&volume=120&publication_year=2012&pages=4072-81&pmid=22927249&doi=10.1182/blood-2012-04-420745&)
