# Impact of CYP2D6 Pharmacogenomic Status on Pain Control Among Opioid‐Treated Oncology Patients

## Metadata
**Authors:** Natalie Reizine, Keith Danahey, Emily Schierer, Ping Liu, Merisa Middlestadt, Jenna Ludwig, Tien M Truong, Xander MR van Wijk, Kiang‐Teck J Yeo, Monica Malec, Mark J Ratain, Peter H O'Donnell
**Journal:** The Oncologist
**Date:** 2021 Sep 19
**DOI:** [10.1002/onco.13953](https://doi.org/10.1002/onco.13953)
**PMID:** 34423496
**PMCID:** PMC8571740
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8571740/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8571740/pdf/ONCO-26-e2042.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8571740/pdf/ONCO-26-e2042.pdf)

## Abstract

**Background:** 
Several opioids have pharmacogenomic associations impacting analgesic efficacy. However, germline pharmacogenomic testing is not routinely incorporated into supportive oncology. We hypothesized that CYP2D6 profiling would correlate with opioid prescribing and hospitalizations.

**Materials and Methods:** 
We analyzed 61,572 adult oncology patients from 2012 to 2018 for opioid exposures. CYP2D6 metabolizer phenotype (ultra‐rapid [UM], normal metabolizer [NM], intermediate [IM], or poor [PM]), the latter two of which may cause inefficacy of codeine, tramadol, and standard‐dose hydrocodone, was determined for patients genotyped for reasons unrelated to pain. The primary endpoint was number of opioid medications received during longitudinal care (IM/PMs vs. NMs). Secondary endpoint was likelihood of pain‐related hospital encounters.

**Results:** 
Most patients with cancer (n = 34,675, 56%) received multiple opioids (average 2.8 ± 1.6/patient). Hydrocodone was most commonly prescribed (62%), followed by tramadol, oxycodone, and codeine. In the CYP2D6 genotyped cohort (n = 105), IM/PMs received a similar number of opioids (3.4 ± 1.4) as NMs (3.3 ± 1.9). However, IM/PMs were significantly more likely to experience pain‐related hospital encounters compared with NMs, independent of other variables (odds ratio [OR] = 5.4; 95% confidence interval [CI], 1.2–23.6; p = .03). IM/PMs were also more likely to be treated with later‐line opioids that do not require CYP2D6 metabolism, such as morphine and hydromorphone (OR = 3.3; 95% CI, 1.1–9.8; p = .03).

**Conclusion:** 
CYP2D6 genotype may identify patients with cancer at increased risk for inadequate analgesia when treated with typical first‐line opioids like codeine, tramadol, or standard‐dose hydrocodone. Palliative care considerations are an integral part of optimal oncology care, and these findings justify prospective evaluation of preemptive genotyping as a strategy to improve oncology pain management.

**Implications for Practice:** 
Genomic variation in metabolic enzymes can predispose individuals to inefficacy when receiving opioid pain medications. Patients with intermediate and/or poor CYP2D6 metabolizer status do not adequately convert codeine, tramadol, and hydrocodone into active compounds, with resulting increased risk of inadequate analgesia. This study showed that patients with cancer frequently receive CYP2D6‐dependent opioids. However, patients with CYP2D6 intermediate and poor metabolizer status had increased numbers of pain‐related hospitalizations and more frequently required the potent non‐CYP2D6 opioids morphine and hydromorphone. This may reflect inadequate initial analgesia with the common “first‐line” CYP2D6‐metabolized opioids. Preemptive genotyping to guide opioid prescribing during cancer care may improve pain‐related patient outcomes.

Keywords: Oncology, Pain, Opioids, Pharmacogenomics, Cancer

### Background

Several opioids have pharmacogenomic associations impacting analgesic efficacy. However, germline pharmacogenomic testing is not routinely incorporated into supportive oncology. We hypothesized that *CYP2D6* profiling would correlate with opioid prescribing and hospitalizations.

### Materials and Methods

We analyzed 61,572 adult oncology patients from 2012 to 2018 for opioid exposures. CYP2D6 metabolizer phenotype (ultra‐rapid [UM], normal metabolizer [NM], intermediate [IM], or poor [PM]), the latter two of which may cause inefficacy of codeine, tramadol, and standard‐dose hydrocodone, was determined for patients genotyped for reasons unrelated to pain. The primary endpoint was number of opioid medications received during longitudinal care (IM/PMs vs. NMs). Secondary endpoint was likelihood of pain‐related hospital encounters.

### Results

Most patients with cancer (*n* = 34,675, 56%) received multiple opioids (average 2.8 ± 1.6/patient). Hydrocodone was most commonly prescribed (62%), followed by tramadol, oxycodone, and codeine. In the *CYP2D6* genotyped cohort (*n* = 105), IM/PMs received a similar number of opioids (3.4 ± 1.4) as NMs (3.3 ± 1.9). However, IM/PMs were significantly more likely to experience pain‐related hospital encounters compared with NMs, independent of other variables (odds ratio [OR] = 5.4; 95% confidence interval [CI], 1.2–23.6; *p* = .03). IM/PMs were also more likely to be treated with later‐line opioids that do not require CYP2D6 metabolism, such as morphine and hydromorphone (OR = 3.3; 95% CI, 1.1–9.8; *p* = .03).

### Conclusion

*CYP2D6* genotype may identify patients with cancer at increased risk for inadequate analgesia when treated with typical first‐line opioids like codeine, tramadol, or standard‐dose hydrocodone. Palliative care considerations are an integral part of optimal oncology care, and these findings justify prospective evaluation of preemptive genotyping as a strategy to improve oncology pain management.

### Implications for Practice

Genomic variation in metabolic enzymes can predispose individuals to inefficacy when receiving opioid pain medications. Patients with intermediate and/or poor CYP2D6 metabolizer status do not adequately convert codeine, tramadol, and hydrocodone into active compounds, with resulting increased risk of inadequate analgesia. This study showed that patients with cancer frequently receive CYP2D6‐dependent opioids. However, patients with CYP2D6 intermediate and poor metabolizer status had increased numbers of pain‐related hospitalizations and more frequently required the potent non‐CYP2D6 opioids morphine and hydromorphone. This may reflect inadequate initial analgesia with the common “first‐line” CYP2D6‐metabolized opioids. Preemptive genotyping to guide opioid prescribing during cancer care may improve pain‐related patient outcomes.

**Keywords:** Oncology, Pain, Opioids, Pharmacogenomics, Cancer

## Introduction

Pain is one of the most commonly reported symptoms in patients with cancer, and opioid medications are an integral part of treatment [[1](#onco13953-bib-0001), [2](#onco13953-bib-0002), [3](#onco13953-bib-0003), [4](#onco13953-bib-0004)]. However, patients who are prescribed opioids are at risk for adverse drug effects, inadequate pain control, or developing opioid misuse syndrome [[5](#onco13953-bib-0005)]. Oncology patients represent a particularly vulnerable population when treating pain given their high burden of comorbidities, polypharmacy, numerous surgeries and procedures, and extensive disease courses [[2](#onco13953-bib-0002), [6](#onco13953-bib-0006)]. Adequate pain control is vital for quality of life and health outcomes in patients with cancer [[2](#onco13953-bib-0002), [7](#onco13953-bib-0007)].

Patients can differ greatly in their interindividual responses to distinct opioid agents [[8](#onco13953-bib-0008), [9](#onco13953-bib-0009)]. Several opioid agents require activation by cytochrome P450 (CYP) enzymes to achieve their analgesic activities (Fig. [1](#onco13953-fig-0001)) [[8](#onco13953-bib-0008), [10](#onco13953-bib-0010)]. Notably, codeine and tramadol are metabolized by CYP450 enzymes in phase I reactions, with key metabolites of CYP2D6 enzyme conversion having enhanced μ‐receptor potency resulting in the predominant analgesic properties of these drugs [[8](#onco13953-bib-0008), [9](#onco13953-bib-0009), [10](#onco13953-bib-0010), [11](#onco13953-bib-0011), [12](#onco13953-bib-0012), [13](#onco13953-bib-0013), [14](#onco13953-bib-0014)]. Hydrocodone is also metabolized by CYP2D6 to hydromorphone, a metabolite that likely mediates the primary analgesic effect; although, the relationship of this CYP2D6‐mediated conversion to pain relief is not as established [[10](#onco13953-bib-0010), [13](#onco13953-bib-0013), [15](#onco13953-bib-0015), [16](#onco13953-bib-0016), [17](#onco13953-bib-0017), [18](#onco13953-bib-0018)]. In contrast, although CYP2D6 also contributes to the metabolism of oxycodone, and although many studies have shown a relationship between CYP2D6 genotype/phenotype and oxymorphone concentrations after administering oxycodone, most have not shown a correlation with pain outcomes, perhaps because the majority of this agent's analgesic activity is hypothesized to result from its parent form [[8](#onco13953-bib-0008), [10](#onco13953-bib-0010), [19](#onco13953-bib-0019), [20](#onco13953-bib-0020), [21](#onco13953-bib-0021), [22](#onco13953-bib-0022)]. Finally, opioids such as morphine and hydromorphone are potent parent compounds themselves, which undergo further metabolism via phase II glucuronidation, with little to no impact from CYP450 genetic or drug‐based interactions [[8](#onco13953-bib-0008), [11](#onco13953-bib-0011)].

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aef4/8571740/1895c948f0f5/ONCO-26-e2042-g004.jpg)

Clinical impacts of CYP2D6 metabolism on opioid analgesic effects. Significant contributors to opioid metabolism, including modification reactions performed by CYP450 enzymes (phase I metabolism) and/or conjugation reactions (phase II metabolism), are displayed. The opioid forms predominantly responsible for the majority of analgesic activities are highlighted in yellow. CYP2D6 plays an essential role in the prodrugs codeine and tramadol bioactivation to their active metabolites, morphine and O‐desmethyltramadol. CYP2D6 is also responsible for hydrocodone metabolism to the active metabolite hydromorphone, which has a 100‐fold higher affinity for μ‐opioid receptors than its parent compound and is postulated to contribute significantly to its analgesic effects. Although CYP2D6 may minimally contribute to the metabolism of oxycodone, the majority of its analgesic activity is hypothesized to result from its parent drug and CYP2D6 metabolism is not generally considered to have a meaningful impact on clinical outcomes for oxycodone [10]. Morphine and hydromorphone do not undergo phase I metabolism but undergo glucuronidation by the enzyme UGT2B7 in phase II metabolism. Because of the infrequent incidence of methadone use in the CYP2D6 genotyped cohort, methadone was omitted from this figure; however, CYP2D6 does play a minor role in this agent's metabolism as well. The information displayed was adopted and summarized from multiple references [8, 11].*Some phase II reactions are omitted from the diagram for simplicity.

The *CYP2D6* gene is highly polymorphic, with numerous alleles described and categorized as conferring normal, reduced, or no‐function based on the activity of the enzyme they encode [[23](#onco13953-bib-0023)]. CYP2D6 poor metabolizers (PMs‐ two nonfunctional alleles) and intermediate metabolizers (IMs‐ one nonfunctional and one reduced function allele, or one functional and one nonfunctional allele) have decreased CYP2D6 activity [[10](#onco13953-bib-0010), [23](#onco13953-bib-0023)]. Compared with normal metabolizers (NMs), IM/PMs demonstrate lower concentrations of the active metabolites of codeine, tramadol, and hydrocodone with the potential for decreased analgesic effects when treated with standard dosing regimens (such as in hydrocodone/acetaminophen formations) [[23](#onco13953-bib-0023), [24](#onco13953-bib-0024), [25](#onco13953-bib-0025), [26](#onco13953-bib-0026), [27](#onco13953-bib-0027), [28](#onco13953-bib-0028), [29](#onco13953-bib-0029)]. In contrast, ultra‐rapid metabolizers (UMs) demonstrate increased CYP2D6 activity and may be at risk for toxicity [[10](#onco13953-bib-0010)].

A recent prospective study of CYP2D6‐guided opioid prescribing in patients with chronic pain demonstrated that this approach resulted in improved patient outcomes and pain control in IM/PM patients, particularly in those treated with codeine, tramadol, and hydrocodone [[10](#onco13953-bib-0010), [30](#onco13953-bib-0030)]. As a result of these and other data, the Clinical Pharmacogenetics Implementation Consortium (CPIC) recently published updated guidelines recommending against codeine and tramadol use in CYP2D6 PMs (and UMs), and close monitoring in codeine and tramadol‐treated IMs (and in IM and PM hydrocodone‐treated subjects) with transition to an alternative non‐CYP2D6‐metabolized analgesic if inadequate response is observed [[10](#onco13953-bib-0010)]. These precision medicine approaches deserve consideration for potentially informing the selection or dosing of opioids when treating patients with cancer to maximize treatment efficacy and minimize adverse drug effects. To our knowledge, no large studies have previously studied the role of *CYP2D6* genotype on key palliative outcomes in a broad cancer population receiving different CYP2D6‐dependent opioids. Given the necessary and frequent role of opioids in the appropriate palliative care of patients with cancer, we first sought to characterize usual opioid prescribing patterns in a large oncology population in this study. We then hypothesized that germline *CYP2D6* genotypes (available for a subset of these patients) would predict risk for inadequate analgesia when using standard opioid prescribing patterns.

## Materials and Methods

### Data Collection and Analysis for the Opioid Exposed Oncology Cohort

We reviewed the chronologic medication histories of patients with cancer treated at the University of Chicago Medical Center (UCMC) between December 1, 2012, and December 31, 2018, for exposure to opioids to demonstrate the prevalence of opioid medication exposure in a large oncology population (Fig. [2](#onco13953-fig-0002)). Oncology patients were defined by International Classification of Diseases 9/10 diagnostic codes for neoplasm, with benign neoplasms excluded [[31](#onco13953-bib-0031)]. We reviewed and analyzed deidentified medical records obtained from the University of Chicago Clinical Research Data Warehouse for exposure to any of eight opioid medications (codeine, tramadol, hydrocodone, oxycodone, morphine, hydromorphone, fentanyl, and methadone), including both scheduled and as needed prescriptions and exposures. Patients were excluded if they did not have a documented opioid exposure during the study period. Relevant patient demographics were captured from the initial oncology provider encounter, including age, gender, self‐reported race and ethnicity, and tumor type.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aef4/8571740/1ac3a96760c2/ONCO-26-e2042-g001.jpg)

Opioid exposed oncology cohort and CYP2D6 genotyped cohort consort diagram. The chronologic medical histories of 61,572 oncology patients (as defined by ICD9/10 malignant neoplasm diagnostic codes) treated at the University of Chicago Medical Center from December 1, 2012, to December 31, 2018, were analyzed for exposure to opioid medications (codeine, tramadol, hydrocodone, oxycodone, morphine, hydromorphone, fentanyl, and methadone; all formulation types). Patients were excluded if they did not have a documented opioid exposure (n = 26,887). A total of 34,675 were included in the opioid exposed oncology cohort. Oncology patients who were genotyped as part of the 1200 Patients Project (Clinicaltrials.gov #NCT01280825) during a study time period of January 21, 2011 to December 31, 2018, and for whom CYP2D6 metabolizer status was available, were analyzed for opioid exposure. A total of 105 patients were included in the CYP2D6 genotyped cohort analysis.

### Data Collection and Analysis for the CYP2D6 Genotyped Cohort

Since 2011, outpatients at UCMC have been eligible to enroll in an ongoing institutional review board–approved institutional pharmacogenomics implementation study entitled The 1200 Patients Project ([ClinicalTrials.gov](http://ClinicalTrials.gov) #[NCT01280825](https://clinicaltrials.gov/ct2/show/NCT01280825)), which evaluated the feasibility of incorporating pharmacogenomic testing into routine outpatient medical care. Eligible patients included all adults without a history of liver or kidney transplant who were receiving routine outpatient care from a selected study provider representing one of eight different primary care and subspecialty clinics including cardiology, gastroenterology, nephrology, oncology, hepatology, pulmonology, and executive health. Subjects consented to preemptive genotyping across a broad panel of variants selected for their clinical pharmacogenomic role, including *CYP2D6* [[32](#onco13953-bib-0032)]. For *CYP2D6* specifically, genotyping was performed using an Invader technology from Hologic (Marlborough, MA), complemented with copy number assessment using Taqman (ThermoFisher, Waltham, MA), with 29 variants assessed including: *1, *2, *3, *4, *4N, *6, *7, *8, *9, *10, *11, *12, *15, *17, *18, *19, *20, *21, *29, *35, *26, *40, *41, *42, *43, *44, *45, *56, *59 [[33](#onco13953-bib-0033)].

For our current study, we identified all opioid‐treated oncology patients (from step one, above paragraph) who had genotype results available for post‐hoc analysis because of their past participation in The 1200 Patients Project. These patients had not been genotyped for the purposes of pain prescribing, and genotypes were not routinely available to oncology physicians at the outset of oncology care and thus were not universally available prior to opioid prescriptions with the exception of one patient, whose results were viewed by one oncologist at a single patient encounter during the observation period, but with no opioids prescribed at that visit. We therefore analyzed clinical endpoints for the purposes of this study independent of knowledge of genotype, and retrospectively correlated the *CYP2D6* genotype data with pain‐related outcomes.

### Data Abstraction

For the *CYP2D6* genotyped cohort, we defined the period of observation as starting on the date of the first oncology provider encounter and ending on the date of the last encounter type of any kind, within The 1200 Patients Project study time period of January 21, 2011, through December 31, 2018. For both the opioid exposed and *CYP2D6* genotyped cohorts, we obtained patient demographics including age, gender, self‐reported race and ethnicity, and cancer diagnosis at the time of first oncology provider encounter. On the date of first opioid exposure, we recorded additional clinical information, including the clinical cancer stage (early‐stage I–III; advanced or metastatic, stage IV), the Eastern Cooperative Oncology Group performance status (if available), the number of concomitant active medication prescriptions, and the number and type of comorbidities, as defined by diagnosis code and active problem list.

### Categorization of Opioid Treatment Efficacy and Pain‐Related Outcomes

For both the opioid exposed and *CYP2D6* cohorts, we analyzed medical histories throughout the entire study period for opioid prescriptions with start and end dates, prescription route, dose, and prescription setting (ambulatory, inpatient, postoperative, or postprocedure). For the *CYP2D6* cohort, the following variables were also obtained: (a) Date and dosage of adjunct pain medications including acetaminophen, nonsteroidal anti‐inflammatory drugs (ibuprofen, naproxen, meloxicam, celecoxib, ketorolac, indomethacin), lidocaine patches, and agents prescribed primarily for the treatment of neuropathy (amitriptyline, carbamazepine, duloxetine, gabapentin, pregabalin); (b) date and dosage of CYP2D6 strong, moderate, and weak inhibitors (supplemental online Table [1](#onco13953-tbl-0001)) [[34](#onco13953-bib-0034)]; (c) dates and type of all surgeries and procedures (i.e., biopsies, resections, reconstructions, endoscopy/colonoscopy/bronchoscopy); (d) number and type of pain‐related hospital encounters including pain‐related hospitalizations, pain clinic encounters, and pain‐related procedures (including joint and epidural injections), and Emergency Department (ED) encounters for pain (as defined by problem list/diagnostic code for the encounter), including only those occurring during the study observation period of January 21, 2011–December 31, 2018; and (e) date and frequency of palliative care consults. During data extraction from the UCMC electronic medical record, the dedicated study personnel performing data collection were blinded to patients' *CYP2D6* genotype/metabolizer status.

### Table 1.

| Characteristic | Opioid exposed oncology cohort (n = 34,675) | CYP2D6 genotyped cohort (n = 105) |
| --- | --- | --- |
| Age at first oncology provider encounter, yr |   |   |
| Mean | 65.3 | 61.1 |
| Median | 66.8 | 64 |
| Gender, female, n (%) | 17,930 (51.7) | 61 (58.1) |
| Race |   |   |
| Black, n (%) | 10,844 (31.4) | 31 (29.5) |
| White, n (%) | 21,452 (61.9) | 69 (65.7) |
| Other a , n (%) | 2,379 (6.7) | 5 (4.8) |
| Ethnicity |   |   |
| Not Hispanic or Latino, n (%) | 32,409 (93.5) | 34 (32.4) |
| Hispanic or Latino, n (%) | 1,644 (4.7) | 4 (3.8) |
| Unknown, n (%) | 622 (1.8) | 67 (63.8) |
| Cancer type |   |   |
| Breast | 3,373 (9.7) | 42 (40) |
| Central nervous system b | 333 (0.9) | 0 |
| Gastrointestinal c | 4,059 (11.7) | 33 (31.4) |
| Genitourinary d | 7,438 (21.5) | 30 (28.6) |
| Head and neck e | 1,932 (5.6) | 0 |
| Leukemia/lymphoma | 7,443 (21.5) | 0 |
| Skin/melanoma | 1,239 (3.6) | 0 |
| Thoracic | 2,972 (8.6) | 0 |
| Other f | 5,886 (16.9) | 0 |

Table 1 Caption: Demographics of the opioid exposed and CYP2D6 genotyped oncology populations

### CYP2D6 Phenotype Determination

For those with archival *CYP2D6* results, metabolizer phenotypes were assigned using genotypes and copy number results. Our primary analysis was based on this *CYP2D6* genotype‐predicted phenotype. Each allele was given an activity score, and the metabolizer phenotype was assigned based on the cumulative value. Scores of 0, 0.5, and 1 were assigned for each no activity, decreased activity, and normal activity allele, respectively. The phenotype was assigned based on the sum of the scores (0, PM; 0.5, IM; 1–2, NM; and > 2, ultra‐rapid metabolizer [UM]) using the phenotype classification schema that was in place during the study period [[35](#onco13953-bib-0035), [36](#onco13953-bib-0036)], although we acknowledge that an updated system was recently adopted that would potentially increase by a very small fraction the number of patients classified as IMs [[10](#onco13953-bib-0010), [23](#onco13953-bib-0023)].

A separate analysis was performed for patients prescribed a CYP2D6 inhibitor synchronously with opioid therapy to encompass phenoconversion potential. Specifically, we reviewed participants' medical histories for exposure to CYP2D6 strong, moderate, and weak inhibitors, as defined by the U.S. Food and Drug Administration (supplemental online Table [1](#onco13953-tbl-0001)) [[34](#onco13953-bib-0034)]. Patients who received a CYP2D6 inhibitor for ≥50% of the time that they received a CYP2D6‐metabolized opioid (defined as codeine, tramadol, hydrocodone) were phenoconverted by multiplying the activity score by an inhibitor factor (0.5 for weak and moderate inhibitors, 0 for strong inhibitors), as performed previously in other studies [[10](#onco13953-bib-0010), [30](#onco13953-bib-0030), [34](#onco13953-bib-0034), [37](#onco13953-bib-0037), [38](#onco13953-bib-0038)].

### Statistical Analysis

Because of the hypothesis that different CYP2D6 phenotypes (and the resulting interindividual variability in analgesia responses) may contribute to imprecision in opioid prescribing, the study's primary endpoint was the number of different opioid medications required for pain control throughout longitudinal care, comparing IM/PM patients with NM patients. The secondary endpoint was the composite likelihood of UCMC hospitalization, pain clinic encounter/procedure, or ED visit for a pain diagnosis comparing IM/PM with NM patients. Both analyses were repeated for phenoconversion. There was no minimum amount of follow up time required. Instead, the varied observation periods between patients were specifically accounted for in the analyses, with the number of each patient's observation years included as a clinical covariate in the Poisson regression analyses. Descriptive statistics were used to summarize the demographic and clinical variables of interest. We also examined multiple clinical variables (described below) between the NM and IM/PM groups. Two‐sample *t*‐test or Fisher's exact tests were used to assess whether these clinical factors were significantly different between the two groups.

For the primary endpoint, Poisson regression analyses were used to evaluate the relationship between the number of different opioid medications and CYP2D6 metabolizer status and other clinical variables. Clinical covariates included number of observation years, patient age, gender, race, ethnicity, number of concomitant medications, number of comorbidities, presence of osteoarthritis and/or neuropathy, cancer type and stage, performance status, exposure to adjunct pain medications, surgeries/procedures, and palliative care consults. Univariable analysis was first used on each factor, and those with a *p* value < .10 from the univariable analysis were included in the multivariable model selection process. Backward selection was used to identify significant predictors of the primary endpoint in the final multivariable Poisson model. Logistic regression models were used to analyze the secondary endpoint of pain‐related hospital encounters. A similar approach was used to determine the final multivariable logistic regression model for pain‐related hospital encounters. Although the main analyses used the genotype‐assigned metabolizer status for each patient, both the primary and secondary analyses were repeated, accounting for phenoconversion. Nominal *p* values < .05 were considered significant.

## Results

### Demographics

A total of 61,572 patients treated at UCMC during the study period were found to have an oncologic diagnosis, with the majority (56.3%, *n* = 34,675) receiving at least one opioid medication during their course of treatment. Of the 127 oncology patients for whom *CYP2D6* genotype data were available, most (*n* = 105, 82.7%) had a documented opioid exposure (Fig. [2](#onco13953-fig-0002)). Subject demographics, including age, self‐reported race/ethnicity, and cancer type, were similar between the total opioid exposed and *CYP2D6* genotyped patient cohorts (Table [1](#onco13953-tbl-0001)). Patients in the *CYP2D6* genotyped cohort were followed for an average of 6.1 ± 2.3 years.

### Opioid Prescribing Patterns Among Oncology Patients

In the opioid exposed oncology cohort (*n* = 34,675), a total of 97,986 separate opioid exposures were identified. The majority (73.7%) of patients with cancer received more than one opioid type during longitudinal care. Median follow up was 426 days, with an average of 2.8 ± 1.6 different opioid medication exposures per patient (representing a median of 2.07 different opioids/year for each patient). The most common opioid prescribed was hydrocodone (*n* = 21,753), as detailed in Figure [3A](#onco13953-fig-0003). A total of *n* = 28,530 (82.3%) received one or more of the opioids for which CYP2D6 metabolism is important for analgesic activity (codeine, tramadol, and/or hydrocodone). A second analysis (Fig. [3B](#onco13953-fig-0003)) subsequently excluded intravenous (IV) formulations of fentanyl, hydromorphone, and morphine to more closely approximate outpatient prescribing patterns. There was no significant difference in prevalence of different opioid types prescribed when analyzed by each demographic variable (age, race, ethnicity, and cancer type), with hydrocodone remaining the most commonly used agent within each specific group (data not shown). These findings demonstrate that a majority of oncology patients are exposed to opioids throughout their longitudinal care and that a high proportion of the prescribed agents are dependent on CYP2D6 metabolism for their analgesic activity.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aef4/8571740/625fce267be6/ONCO-26-e2042-g003.jpg)

(A): Opioid prescribing patterns in a population with cancer. A total of 97,986 separate opioid drug exposures were identified in 34,675 patients with an oncologic diagnosis during the study period (December 1, 2012–December 31, 2018). The most common oral opioid prescribed was hydrocodone. CYP2D6‐metabolized opioids (codeine, tramadol, hydrocodone) are highlighted in orange. (B): Oral and transdermal opioid prescribing patterns in a population with cancer. To approximate outpatient prescribing patterns, intravenous formulations of morphine, hydromorphone, and fentanyl were excluded. A total of 59,540 separate oral (and transdermal, i.e., fentanyl) opioid drug exposures were identified in 34,675 patients with an oncologic diagnosis during the study period (December 1, 2012–December 31, 2018). CYP2D6‐metabolized opioids (codeine, tramadol, hydrocodone) are highlighted in orange.

### CYP2D6 Genotype and Metabolizer Assignments

The primary analysis for the *CYP2D6* genotyped cohort was based on *CYP2D6* genotype‐predicted phenotypes. One genotyped patient was determined to have a UM phenotype (*n* = 1, 0.95%). Given that this phenotype is not correlated with the inadequate analgesic response outcomes of interest in this study (increased number of opioid exposures and composite evaluation of inadequate pain control) this patient was included for analysis within the NM phenotype group for a total of *n* = 91. Although the finding of only one UM patient was perhaps slightly lower than expected, this is likely because of the relatively large proportion of Black patients included in our cohort, a group with a known lower frequency of observed UM phenotypes compared with European‐descent populations (0.4% vs. 2.5%, respectively) [[10](#onco13953-bib-0010)].

Seven patients (6.7%) each were IM and PM, respectively, for a total of *n* = 14 in the combined IM/PM metabolizer group, which is consistent with expected allele frequencies previously published [[23](#onco13953-bib-0023), [24](#onco13953-bib-0024)]. Eight patients in the *CYP2D6* genotyped cohort were found to be taking CYP2D6 inhibitors ≥50% of the time concurrently with a prescription for codeine, tramadol, and/or hydrocodone. For the secondary phenoconversion analysis described previously, these eight patients were subsequently included in the IM/PM metabolizer group (*n* = 22).

### Clinical Factors Contribute to Opioid Use Prevalence Among CYP2D6 Genotyped Oncology Patients

Patients in both IM/PM and NM groups were exposed to a similar number of opioids throughout their longitudinal course, as demonstrated in Table [3](#onco13953-tbl-0003). Prescribing of CYP2D6‐metabolized opioids such as codeine, tramadol, and hydrocodone was very common, with the majority of patients in both IM/PM and NM groups receiving at least one of these agents (13/14 or 93% vs. 74/91 or 81%, respectively). Of note, seven of the IM/PM patients (50%) were prescribed codeine, tramadol, or hydrocodone contemporaneously with a non‐CYP2D6 metabolized opioid (morphine and/or oxycodone), which could have dampened the clinical manifestations of CYP2D6 effects in these patients.

### Table 3.

| Opioid medication | CYP2D6 genotyped cohort IM/PM | CYP2D6 genotyped cohort NM | CYP2D6 genotyped cohort ‐ IM/PM phenoconversion | CYP2D6 genotyped cohort – NM phenoconversion |
| --- | --- | --- | --- | --- |
| (n = 14) | (n = 91) | (n = 22) | (n = 83) |  |
| Codeine n (%) a , b | 8 (57) | 37 (41) | 13 (59) | 33 (40) |
| Tramadol n (%) a , b | 3 (21) | 30 (33) | 5 (23) | 28 (34) |
| Hydrocodone n (%) a , b | 8 (57) | 57 (63) | 12 (55) | 52 (63) |
| Oxycodone n (%) a , b | 4 (29) | 22 (24) | 7 (32) | 19 (23) |
| Morphine n (%) a , b | 7 (50) | 29 (32) | 11 (50) | 25 (30) |
| Hydromorphone n (%) a , b | 8 (57) | 52 (57) | 12 (55) | 47 (57) |
| Fentanyl n (%) a , b | 11 (79) | 68 (75) | 17 (77) | 62 (75) |
| Methadone n (%) a , b | 0 (0.0) | 2 (2.2) | 1 (4.5) | 1 (1.2) |
| The average number of distinct opioid exposures per patient |   |   |   |   |
| Mean ± SE c | 3.5 ± 1.4 | 3.3 ± 1.9 | 3.6 ± 1.7 | 3.2 ± 1.9 |
| Patients exposed to morphine and/or hydromorphone n (%) c | 12 (85.7) | 55 (60.4) | 17 (77.3) | 42 (50.6) |
| Odds ratio of morphine and/or hydromorphone exposure, IM/PM vs. NM, c (95% CI) | 3.9 (0.83–18.6), p = .08 |  | 3.3 (1.12–9.8), p = .03 |  |

Table 3 Caption: Opioid medication exposures in CYP2D6 genotyped (n = 105) population with phenoconversion analysis

As oncology patients frequently undergo diagnostic and therapeutic procedures during their treatment, with opioids being a mainstay of management in these settings, we crucially included the presence (or lack thereof) of surgery and/or a procedure as a clinical covariate for predicting opioid exposure (Table [2](#onco13953-tbl-0002)). The receipt of a surgery/procedure was expectedly predictive of exposure to a greater number of opioids (*p* = .003), particularly in the setting of IV opioids (85.9% of IV opioids exposures were associated with surgery/procedure) and, to a lesser extent, oral opioids (50.2% of oral opioids were prescribed in conjunction with a surgery/procedure). In addition, patients who were prescribed additional adjunct pain medications during treatment (i.e., ibuprofen, naproxen, meloxicam, celecoxib, ketorolac, indomethacin, lidocaine patches, amitriptyline, carbamazepine, duloxetine, gabapentin, pregabalin) also had more opioid exposures throughout their longitudinal course (*p* = .0008). The associations of these two clinical factors with patients' increasing exposure to opioid medications remained similar when repeated with the exclusion of IV fentanyl and also when the Poisson regression was repeated for the phenoconversion population.

### Table 2.

| Characteristic | NM | IM/PM | p value |
| --- | --- | --- | --- |
| n = 91 | n = 14 |  |  |
| Age at first encounter, years |   |   |   |
| Mean | 58.8 | 61.5 | .43 |
| Median | 55.5 | 64 |   |
| Gender, female, n (%) | 50 (54.9) | 11 (78.6) | .15 |
| Race |   |   |   |
| Black, n (%) | 24 (26.4) | 7 (50) | .23 |
| White, n (%) | 62 (68.1) | 7 (50) |   |
| Other, n (%) | 5 (5.5) | 0 |   |
| Ethnicity |   |   |   |
| Not Hispanic or Latino, n (%) | 30 (33.0) | 4 (28.6) | .87 |
| Hispanic or Latino, n (%) | 4 (4.4) | 0 |   |
| Unknown, n (%) | 57 (62.6) | 10 (71.4) |   |
| Cancer type |   |   |   |
| Breast | 34 (37.4) | 8 (57.1) | .36 |
| Gastrointestinal a | 29 (31.9) | 4 (28.6) |   |
| Genitourinary b | 28 (30.8) | 2 (14.3) |   |
| Cancer stage at 1st opioid prescription |   |   |   |
| I–III, n (%) | 49 (53.9) | 12 (85.7) | .10 |
| IV, n (%) | 38 (41.8) | 2 (14.3) |   |
| n/a c , n (%) | 4 (4.4) | 0 (0) |   |
| Diagnosis of osteoarthritis d , n (%) | 36 (39.6) | 4 (28.6) | .56 |
| Diagnosis of neuropathy d , n (%) | 28 (30.8) | 5 (35.7) | .76 |
| Number of comorbid medical conditions |   |   |   |
| Mean ± SE d | 3.58 ± 0.43 | 2.57 ± 0.60 | .18 |
| Number of total medication prescriptions |   |   |   |
| Mean ± SE d | 7.80 ± 0.46 | 6.50 ± 0.81 | .29 |
| Number of surgeries/procedures |   |   |   |
| Mean ± SE e | 3.41 ± 0.31 | 3.64 ± 0.44 | .67 |

Table 2 Caption: Characteristics of CYP2D6‐genotyped study population (n = 105)

### Correlation of CYP2D6 Genotype with Efficacy of Pain Control and Incidence of Pain‐Related Hospital Encounters

The vast majority of IM/PM patients (10/14, or 71.4%) experienced a pain‐related hospitalization, ED visit, and/or pain clinic consultation for a pain‐related procedure. This finding was in stark contrast to NM patients, who required these in only one third of cases (30/91, or 33%). Half of the IM/PM patients who required a pain‐related hospital encounter were prescribed one of the CYP2D6‐activated opioids (codeine, tramadol, hydrocodone) as an outpatient before their pain‐related event.

The majority of pain‐related encounters in both IM/PM and NM patients included pain clinic visits, pain‐related procedures (e.g., injections), and/or ED visits (75%). However, as a surrogate for capturing the potential burden of inadequate pain control on health care utilization, for patients with hospitalizations we examined hospital length of stay (arguably the most costly of the three types of included encounters). In this exploratory analysis (limited by the number of IM/PM patients), we did not find a significant difference in hospital length of stay between NM and IM/PM patients (median 3.5 days [range 1–15] vs. 1.0 days [[1](#onco13953-bib-0001), [2](#onco13953-bib-0002), [3](#onco13953-bib-0003)], *p* = .07). Also, the average time between first opioid exposure and first pain‐related hospitalization was variable and did not appear explanatory (190.2 ± 357.2 days for NMs vs. 429.5 ± 324.6 for IM/PMs).

The logistic regression model identified CYP2D6 metabolizer status, Black race, and exposure to adjunct pain medications as each being independently associated with an increased need for pain‐related hospitalizations and/or procedures (Fig. [4](#onco13953-fig-0004)). Notably, in the multivariable regression model, IM/PM patients were five times more likely than NMs to require a pain‐related procedure/hospital encounter (odds ratio [OR], 5.4; confidence interval [CI], 1.2–23.6; *p* = .03). Black race (OR, 3.1; CI, 1.2–8.4; *p* = .03) and exposure to adjunct pain medications (OR, 16.3; CI, 3.0–87.7; *p* = .001) were other independent predictors of pain‐related hospitalization/emergency visit. Given the clinical importance of considering phenoconversion (which appropriately captures the “effective” metabolizer status), these analyses were repeated in the larger group (*n* = 22) of phenoconverted IM/PM subjects. The results were importantly reproduced and accentuated for all three significant factors predictive of increased risk for pain‐related hospitalization, ED visit, or pain procedure: IM/PM vs. NM (OR, 5.5; CI, 1.6–19.2; *p* = .007); Black vs. White race (OR, 3.5; CI, 1.3–9.6; *p* = .02); and exposure to adjunct pain medications (OR, 18.7; CI, 3.4–101.4; *p* = .0007).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/aef4/8571740/6f5c90837641/ONCO-26-e2042-g002.jpg)

Forest plot demonstrating independent risk factors for pain‐related hospital encounters in CYP2D6‐genotyped cohort. Pain‐related hospital encounters during the study period (January 21, 2011–December 31, 2018) were defined as a hospitalization, emergency department visit, or pain clinic visit/procedure for a pain‐related diagnosis (as defined by problem list/diagnostic code for the encounter). The relationship between the likelihood of pain‐related hospital encounters and important demographic and clinical variables was examined using univariable logistic regression models. The corresponding model results are graphically presented. The following three factors were identified as significant predictors of pain‐related hospital encounters through the multivariable logistic regression model selection process: additional exposure to adjunct pain medications, Black race, and IM/PM CYP2D6 genotypes. LCL and UCL are defined.Abbreviations: EM, extensive metabolizer; GI, gastrointestinal; GU, genitourinary; IM, intermediate metabolizer; LCL, lower confidence limit; OR, odds ratio; PM, poor metabolizer; UCL, upper confidence limit; UM, ultra‐rapid metabolizer.

Finally, although the majority (13/14, or 93%) of IM/PM patients received a CYP2D6‐metabolized opioid such as codeine, tramadol, or hydrocodone, prescribing patterns demonstrated that these patients also more frequently received treatment with CYP2D6‐independent opioids such as morphine and/or hydromorphone. IM/PM patients were more likely than NM patients to be exposed to morphine and/or hydromorphone during their longitudinal treatment course (12/14 IM/PM patients or 85.7% vs. 60.4% NMs; OR 3.9, 95% CI, 0.8–18.6; *p* = .08). Six of the 12 (50%) of these exposures occurred during a pain‐related hospital encounter. Most of these patients (four of the six, 67%) had an active outpatient prescription for codeine, tramadol, or hydrocodone at the time of the pain‐related hospital encounter, suggesting failure of their prescribed CYP2D6‐metabolized medication. The association of increased exposure to morphine/hydromorphone in the IM/PM population was significant when accounting for phenoconversion (77.3% vs. 50.6% with OR, 3.3; CI 1.1–9.8; *p* = .03), as displayed in Table [3](#onco13953-tbl-0003).

## Discussion

A high prevalence of opioid exposures among oncology patients was found in our study, in which patients were noted to receive numerous different types of opioid medications throughout their longitudinal course of care. Hydrocodone and tramadol, both dependent on CYP2D6 activation, were the most frequently used oral opioid medications, regardless of cancer type or demographic factors (race, age). When analyzed by CYP2D6 status, IM/PM patients were more likely to have a pain‐related hospital encounter than NM patients despite active outpatient prescriptions for CYP2D6‐metabolized opioids, suggesting that IM/PMs may have experienced insufficient analgesia with the commonly used CYP2D6‐dependent opioids. In addition, an increased proportion of IM/PM patients were subsequently found to be exposed to less commonly used but more potent CYP2D6‐independent opioids such as morphine and/or hydromorphone. These data suggest that initial inadequate pain control may have precipitated a pain‐related hospital encounter in which patients were exposed to later‐line opioids not dependent on CYP2D6 activation. A pattern of prescribing tramadol, hydrocodone, or codeine‐containing medications as “first choice” pain control drugs (as recommended by the World Health Organization Cancer Pain Management Ladder [[39](#onco13953-bib-0039)]) or using opioid “trial and error” may thus be inadequate or potentially harmful in CYP2D6 IM/PM patients. Prospective studies will be necessary to confirm these important findings for cancer care.

Despite some prior investigations in small cohorts or examining single opioids, convincing evidence of the pharmacogenomic role for CYP2D6 in guiding pain prescribing in the oncology setting has not been previously established [[19](#onco13953-bib-0019), [40](#onco13953-bib-0040), [41](#onco13953-bib-0041), [42](#onco13953-bib-0042)]. Smith et al. recently demonstrated the successful implementation of *CYP2D6*‐guided opioid therapy (using both genotype and phenoconversion) in the primary care setting [[30](#onco13953-bib-0030)]. Their results showed improved pain control in the population postulated to benefit most from this strategy (IM/PMs initially prescribed a CYP2D6‐bioactivated opioid such as codeine, tramadol, or hydrocodone), with improved patient outcomes including pain control. Our findings now extend these important implications to the oncology treatment setting. Despite the relatively modest size of our genotyped cohort in this study, our genotyped cohort represents to our knowledge the largest collection of *CYP2D6*‐genotyped patients with cancer receiving multiple different CYP2D6‐dependent opioids analyzed for pain outcomes. Our findings were reproduced and strengthened when considering CYP2D6 phenoconversion, which extended the analysis to an even larger group of functional IM/PM patients. This step is likely important, given that concomitant use of CYP2D6 inhibitors and opioids is common in cancer care, highlighting the potentially important role of considering both drug‐drug interactions as well as drug‐gene interactions in perpetuating the clinical variability of opioid responses.

During cancer care, the need for ED visits/hospitalizations and pain consults (with related procedures) to address pain directly not only impacts quality of life but leads to increased health care utilization and associated costs [[43](#onco13953-bib-0043), [44](#onco13953-bib-0044)]. Despite the fact that most IM/PM patients in our study had early‐stage cancers, we found that these patients were five times more likely than NM patients to have a pain‐related hospital encounter throughout their clinical course, even when controlled for additional clinical variables. In this study, self‐reported Black race was also independently associated with an increased risk of pain‐related hospital encounters, supporting results found in prior studies [[45](#onco13953-bib-0045), [46](#onco13953-bib-0046)].

Our study had several limitations. First, our findings describe a single institution experience, and our analyses were retrospective in nature. However, we used a diverse patient population first to establish opioid prescribing patterns, and our CYP2D6‐based findings (although based on a relatively small number of IM/PM genotypic individuals) were recapitulated when considering both *CYP2D6* genotype‐conferred metabolizer status and when considering phenoconversion (representing a larger group), and were directionally consistent across two different pain‐related phenotypes (increased pain‐related encounters; increased requirement for CYP2D6‐independent opioids), thus significantly decreasing the possibility that these results were found by chance alone. Although CYP2D6 enzymatic activities and opioid pharmacokinetic parameters were not directly analyzed, the use of *CYP2D6* genotype to reliably predict phenotype is well‐established [[10](#onco13953-bib-0010), [23](#onco13953-bib-0023), [47](#onco13953-bib-0047)]. It is nonetheless acknowledged that although CPIC does include hydrocodone prescribing guidance in its latest recommendations (albeit as limited evidence and “optional”), given the strong data demonstrating the role of CYP2D6 genotype/phenotype in impacting hydrocodone‐related pain outcomes demonstrated by Smith et al. [[30](#onco13953-bib-0030)], and consistent with current ongoing large‐scale National Institutes of Health‐sponsored prospective trials that include CYP2D6‐guided adjustment for hydrocodone, tramadol, and codeine ([clinicaltrials.gov](http://clinicaltrials.gov) #[NCT04445792](https://clinicaltrials.gov/ct2/show/NCT04445792)), we included hydrocodone in our analysis but the role of hydrocodone/CYP2D6 effects deserves ongoing validation. Separately, although pain scores were not available in our analysis, our findings on the role of genotype on pain‐related clinical outcomes were on‐par with several clinical factors that sensibly correlated with pain outcomes, and *CYP2D6* genotype was independently associated even after controlling for these clinical factors in multivariable analyses. To prevent potential coding bias given the retrospective nature of this study, the two independent phenotype coders were blinded to CYP2D6 metabolizer status at the time of clinical data abstraction. Another limitation of our study is that CYP3A4 status, which impacts hydrocodone metabolism, was not included in our analysis. Other pharmacogenes, including *COMT* and the pharmacodynamic *OPRM1* were also not analyzed because consensus guidelines have not established a meaningful clinical impact on pain outcomes for these genes [[10](#onco13953-bib-0010)]. If further evidence justifies the consideration of these additional genes or other pharmacodynamic variables [[48](#onco13953-bib-0048)], future studies could consider composite models of biological factors (including CYP2D6) on opioid‐related outcomes. One such consideration, organ dysfunction, which may have effects on opioid metabolism and could affect response, was not explicitly evaluated in our study; instead, patients' comorbidities at the time of first opioid prescription were recorded and were not found to be predictive of either the primary or secondary outcomes. Cancer progression and refractory disease is also a well‐known mechanism of worsening cancer pain; whereas cancer stage was included as an independent factor and not found to be significant, future studies should explore this relationship further. Lastly, information regarding costs of pain‐related health care utilization were not available for this analysis but could be important to explore in future studies.

## Conclusion

Our findings suggest that CYP2D6 IM/PM patients may not be adequately palliated by commonly used front‐line opioid agents including codeine, tramadol, and hydrocodone, and such patients have an increased risk of experiencing hospitalization or pain‐related encounters and requiring non‐CYP2D6 activated opioids for pain control. These results provide support for the prospective consideration of preemptive *CYP2D6* genotyping to assist in personalized opioid prescribing among oncology patients.

## Author Contributions

**Conception/design:** Natalie Reizine, Peter H. O'Donnell

**Provision of study material or patients:** Keith Danahey, Emily Schierer, Merisa Middlestadt, Jenna Ludwig, Xander M.R. van Wijk, Kiang‐Teck J. Yeo, Mark Ratain, Peter H. O'Donnell

**Collection and/or assembly of data:** Natalie Reizine, Keith Danahey, Emily Schierer, Ping Liu, Merisa Middlestadt, Jenna Ludwig

**Data analysis and interpretation:** Natalie Reizine, Ping Liu, Tien M. Truong, Xander M.R. van Wijk, Kiang‐Teck J. Yeo, Peter H. O'Donnell

**Manuscript writing:** Natalie Reizine, Peter H. O'Donnell

**Final approval of manuscript:** Natalie Reizine, Keith Danahey, Emily Schierer Ping Liu, Merisa Middlestadt, Jenna Ludwig, Tien M. Truong, Xander M.R. van Wijk, Kiang‐Teck J. Yeo, Monica Malec, Mark Ratain, Peter H. ODonnell

## Disclosures

**Kiang‐Teck J. Yeo:** Truvian Sciences, Roche Diagnostics (C/A, H, SAB), Truvian Sciences (RF). The other authors indicated no financial relationships.

(C/A) Consulting/advisory relationship; (RF) Research funding; (E) Employment; (ET) Expert testimony; (H) Honoraria received; (OI) Ownership interests; (IP) Intellectual property rights/inventor/patent holder; (SAB) Scientific advisory board

## Supporting information

## Acknowledgments

This work was supported by the National Institutes for Health [R01 HG009938‐01A1 to P.H.O, 5T32GM007019‐41 through the University of Chicago Committee on Clinical Pharmacology and Pharmacogenomics to N. R.]; the University of Chicago Comprehensive Cancer Center; The William F. O'Connor Foundation; and the Benjamin McAllister Research Fellowship Award [to N. R.].

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Nahin RL. Estimates of pain prevalence and severity in adults: United States, 2012. J Pain 2015;16:769–780.  [DOI](https://doi.org/10.1016/j.jpain.2015.05.002) | [PMC free article](/articles/PMC4562413/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26028573/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pain&title=Estimates%20of%20pain%20prevalence%20and%20severity%20in%20adults:%20United%20States,%202012&author=RL%20Nahin&volume=16&publication_year=2015&pages=769-780&pmid=26028573&doi=10.1016/j.jpain.2015.05.002&)

2. Page R, Blanchard E. Opioids and cancer pain: Patients' needs and access challenges. J Oncol Pract 2019;15:229–231.  [DOI](https://doi.org/10.1200/JOP.19.00081) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31013181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Oncol%20Pract&title=Opioids%20and%20cancer%20pain:%20Patients'%20needs%20and%20access%20challenges&author=R%20Page&author=E%20Blanchard&volume=15&publication_year=2019&pages=229-231&pmid=31013181&doi=10.1200/JOP.19.00081&)

3. van den Beuken‐van Everdingen MH, Hochstenbach LM, Joosten EA et al. Update on prevalence of pain in patients with cancer: Systematic review and meta‐analysis. J Pain Symptom Manage 2016;51:1070–1090.e9.  [DOI](https://doi.org/10.1016/j.jpainsymman.2015.12.340) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27112310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pain%20Symptom%20Manage&title=Update%20on%20prevalence%20of%20pain%20in%20patients%20with%20cancer:%20Systematic%20review%20and%20meta%E2%80%90analysis&author=MH%20van%20den%20Beuken%E2%80%90van%20Everdingen&author=LM%20Hochstenbach&author=EA%20Joosten&volume=51&publication_year=2016&pages=1070-1090.e9&pmid=27112310&doi=10.1016/j.jpainsymman.2015.12.340&)

4. Paice JA. Managing pain in patients and survivors: Challenges within the United States opioid crisis. J Natl Compr Canc Netw 2019;17:595–598.  [DOI](https://doi.org/10.6004/jnccn.2019.5010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31117028/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Compr%20Canc%20Netw&title=Managing%20pain%20in%20patients%20and%20survivors:%20Challenges%20within%20the%20United%20States%20opioid%20crisis&author=JA%20Paice&volume=17&publication_year=2019&pages=595-598&pmid=31117028&doi=10.6004/jnccn.2019.5010&)

5. Benyamin R, Trescot AM, Datta S et al. Opioid complications and side effects. Pain Physician 2008;11(suppl 2):S105–120.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/18443635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain%20Physician&title=Opioid%20complications%20and%20side%20effects&author=R%20Benyamin&author=AM%20Trescot&author=S%20Datta&volume=11&issue=suppl%202&publication_year=2008&pages=S105-120&pmid=18443635&)

6. Lee JS, Hu HM, Edelman AL et al. New persistent opioid use among patients with cancer after curative‐intent surgery. J Clin Oncol 2017;35:4042–4049.  [DOI](https://doi.org/10.1200/JCO.2017.74.1363) | [PMC free article](/articles/PMC5736238/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29048972/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=New%20persistent%20opioid%20use%20among%20patients%20with%20cancer%20after%20curative%E2%80%90intent%20surgery&author=JS%20Lee&author=HM%20Hu&author=AL%20Edelman&volume=35&publication_year=2017&pages=4042-4049&pmid=29048972&doi=10.1200/JCO.2017.74.1363&)

7. Bennett M, Paice JA, Wallace M. Pain and opioids in cancer care: Benefits, risks, and alternatives. Am Soc Clin Oncol Educ Book 2017;37:705–713.  [DOI](https://doi.org/10.1200/EDBK_180469) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28561731/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20Soc%20Clin%20Oncol%20Educ%20Book&title=Pain%20and%20opioids%20in%20cancer%20care:%20Benefits,%20risks,%20and%20alternatives&author=M%20Bennett&author=JA%20Paice&author=M%20Wallace&volume=37&publication_year=2017&pages=705-713&pmid=28561731&doi=10.1200/EDBK_180469&)

8. Smith HS. Opioid metabolism. Mayo Clin Proc 2009;84:613–624.  [DOI](https://doi.org/10.1016/S0025-6196(11)60750-7) | [PMC free article](/articles/PMC2704133/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19567715/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mayo%20Clin%20Proc&title=Opioid%20metabolism&author=HS%20Smith&volume=84&publication_year=2009&pages=613-624&pmid=19567715&doi=10.1016/S0025-6196(11)60750-7&)

9. Smith DM, Weitzel KW, Cavallari LH et al. Clinical application of pharmacogenetics in pain management. Per Med 2018;15:117–126.  [DOI](https://doi.org/10.2217/pme-2017-0032) | [PMC free article](/articles/PMC6460918/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29714124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Per%20Med&title=Clinical%20application%20of%20pharmacogenetics%20in%20pain%20management&author=DM%20Smith&author=KW%20Weitzel&author=LH%20Cavallari&volume=15&publication_year=2018&pages=117-126&pmid=29714124&doi=10.2217/pme-2017-0032&)

10. Crews KR, et al. Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for CYP2D6, OPRM1, and COMT genotype and select opioid therapy. Clin Pharmacol Ther 2021. [epub ahead of print].  [DOI](https://doi.org/10.1002/cpt.2149) | [PMC free article](/articles/PMC8249478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33387367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guideline%20for%20CYP2D6,%20OPRM1,%20and%20COMT%20genotype%20and%20select%20opioid%20therapy&author=KR%20Crews&publication_year=2021&pmid=33387367&doi=10.1002/cpt.2149&)

11. Overholser BR, Foster DR. Opioid pharmacokinetic drug‐drug interactions. Am J Manag Care 2011;17(suppl 11):S276–287.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/21999760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Manag%20Care&title=Opioid%20pharmacokinetic%20drug%E2%80%90drug%20interactions&author=BR%20Overholser&author=DR%20Foster&volume=17&issue=suppl%2011&publication_year=2011&pages=S276-287&pmid=21999760&)

12. Gong L, Stamer UM, Tzvetkov MV et al. PharmGKB summary: Tramadol pathway. Pharmacogenet Genomics 2014;24:374–380.  [DOI](https://doi.org/10.1097/FPC.0000000000000057) | [PMC free article](/articles/PMC4100774/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24849324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=PharmGKB%20summary:%20Tramadol%20pathway&author=L%20Gong&author=UM%20Stamer&author=MV%20Tzvetkov&volume=24&publication_year=2014&pages=374-380&pmid=24849324&doi=10.1097/FPC.0000000000000057&)

13. DePriest AZ, Puet BL, Holt AC et al. Metabolism and disposition of prescription opioids: A review. Forensic Sci Rev 2015;27:115–145.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/26227254/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Forensic%20Sci%20Rev&title=Metabolism%20and%20disposition%20of%20prescription%20opioids:%20A%20review&author=AZ%20DePriest&author=BL%20Puet&author=AC%20Holt&volume=27&publication_year=2015&pages=115-145&pmid=26227254&)

14. Gasche Y, Daali Y, Fathi M et al. Codeine intoxication associated with ultrarapid CYP2D6 metabolism. N Engl J Med 2014;351:2827–2831.  [DOI](https://doi.org/10.1056/NEJMoa041888) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15625333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Codeine%20intoxication%20associated%20with%20ultrarapid%20CYP2D6%20metabolism&author=Y%20Gasche&author=Y%20Daali&author=M%20Fathi&volume=351&publication_year=2014&pages=2827-2831&pmid=15625333&doi=10.1056/NEJMoa041888&)

15. Otton SV, Schadel M, Cheung SW et al. CYP2D6 phenotype determines the metabolic conversion of hydrocodone to hydromorphone. Clin Pharmacol Ther 1993;54:463–472.  [DOI](https://doi.org/10.1038/clpt.1993.177) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7693389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CYP2D6%20phenotype%20determines%20the%20metabolic%20conversion%20of%20hydrocodone%20to%20hydromorphone&author=SV%20Otton&author=M%20Schadel&author=SW%20Cheung&volume=54&publication_year=1993&pages=463-472&pmid=7693389&doi=10.1038/clpt.1993.177&)

16. Stauble ME, et al. Hydrocodone in postoperative personalized pain management: Pro‐drug or drug? Clin Chim Acta 2014;429:26–29.  [DOI](https://doi.org/10.1016/j.cca.2013.11.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24269714/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=Hydrocodone%20in%20postoperative%20personalized%20pain%20management:%20Pro%E2%80%90drug%20or%20drug?&author=ME%20Stauble&volume=429&publication_year=2014&pages=26-29&pmid=24269714&doi=10.1016/j.cca.2013.11.015&)

17. Linares OA, Fudin J, Daly AL et al. Individualized hydrocodone therapy based on phenotype, pharmacogenetics, and pharmacokinetic dosing. Clin J Pain 2015;31:1026–1035.  [DOI](https://doi.org/10.1097/AJP.0000000000000214) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25621429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20J%20Pain&title=Individualized%20hydrocodone%20therapy%20based%20on%20phenotype,%20pharmacogenetics,%20and%20pharmacokinetic%20dosing&author=OA%20Linares&author=J%20Fudin&author=AL%20Daly&volume=31&publication_year=2015&pages=1026-1035&pmid=25621429&doi=10.1097/AJP.0000000000000214&)

18. Linares OA, Tod M, Daly AL et al. Feasibility and utility of the individualized hydrocodone therapy based on phenotype, pharmacogenetics, and pharmacokinetic dosing. Clin J Pain 2016;32:1106–1107.  [DOI](https://doi.org/10.1097/AJP.0000000000000382) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27824618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20J%20Pain&title=Feasibility%20and%20utility%20of%20the%20individualized%20hydrocodone%20therapy%20based%20on%20phenotype,%20pharmacogenetics,%20and%20pharmacokinetic%20dosing&author=OA%20Linares&author=M%20Tod&author=AL%20Daly&volume=32&publication_year=2016&pages=1106-1107&pmid=27824618&doi=10.1097/AJP.0000000000000382&)

19. Andreassen TN, Eftedal I, Klepstad P et al. Do CYP2D6 genotypes reflect oxycodone requirements for cancer patients treated for cancer pain? A cross‐sectional multicentre study. Eur J Clin Pharmacol 2012;68:55–64.  [DOI](https://doi.org/10.1007/s00228-011-1093-5) | [PMC free article](/articles/PMC3249195/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21735164/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=Do%20CYP2D6%20genotypes%20reflect%20oxycodone%20requirements%20for%20cancer%20patients%20treated%20for%20cancer%20pain?%20A%20cross%E2%80%90sectional%20multicentre%20study&author=TN%20Andreassen&author=I%20Eftedal&author=P%20Klepstad&volume=68&publication_year=2012&pages=55-64&pmid=21735164&doi=10.1007/s00228-011-1093-5&)

20. Zwisler ST, Enggaard TP, Noehr‐Jensen L et al. The hypoalgesic effect of oxycodone in human experimental pain models in relation to the CYP2D6 oxidation polymorphism. Basic Clin Pharmacol Toxicol 2009;104:335–344.  [DOI](https://doi.org/10.1111/j.1742-7843.2009.00378.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19281600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin%20Pharmacol%20Toxicol&title=The%20hypoalgesic%20effect%20of%20oxycodone%20in%20human%20experimental%20pain%20models%20in%20relation%20to%20the%20CYP2D6%20oxidation%20polymorphism&author=ST%20Zwisler&author=TP%20Enggaard&author=L%20Noehr%E2%80%90Jensen&volume=104&publication_year=2009&pages=335-344&pmid=19281600&doi=10.1111/j.1742-7843.2009.00378.x&)

21. Samer CF, Daali Y, Wagner M et al. Genetic polymorphisms and drug interactions modulating CYP2D6 and CYP3A activities have a major effect on oxycodone analgesic efficacy and safety. Br J Pharmacol 2010;160:919–930.  [DOI](https://doi.org/10.1111/j.1476-5381.2010.00709.x) | [PMC free article](/articles/PMC2935998/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20590588/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=Genetic%20polymorphisms%20and%20drug%20interactions%20modulating%20CYP2D6%20and%20CYP3A%20activities%20have%20a%20major%20effect%20on%20oxycodone%20analgesic%20efficacy%20and%20safety&author=CF%20Samer&author=Y%20Daali&author=M%20Wagner&volume=160&publication_year=2010&pages=919-930&pmid=20590588&doi=10.1111/j.1476-5381.2010.00709.x&)

22. Zwisler ST, Enggaard TP, Mikkelsen S et al. Impact of the CYP2D6 genotype on post‐operative intravenous oxycodone analgesia. Acta Anaesthesiol Scand 2010;54:232–240.  [DOI](https://doi.org/10.1111/j.1399-6576.2009.02104.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19719813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Anaesthesiol%20Scand&title=Impact%20of%20the%20CYP2D6%20genotype%20on%20post%E2%80%90operative%20intravenous%20oxycodone%20analgesia&author=ST%20Zwisler&author=TP%20Enggaard&author=S%20Mikkelsen&volume=54&publication_year=2010&pages=232-240&pmid=19719813&doi=10.1111/j.1399-6576.2009.02104.x&)

23. Caudle KE, et al. Standardizing CYP2D6 genotype to phenotype translation: Consensus recommendations from the Clinical Pharmacogenetics Implementation Consortium and Dutch Pharmacogenetics Working Group. Clin Transl Sci 2020;13:116–124.  [DOI](https://doi.org/10.1111/cts.12692) | [PMC free article](/articles/PMC6951851/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31647186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Standardizing%20CYP2D6%20genotype%20to%20phenotype%20translation:%20Consensus%20recommendations%20from%20the%20Clinical%20Pharmacogenetics%20Implementation%20Consortium%20and%20Dutch%20Pharmacogenetics%20Working%20Group&author=KE%20Caudle&volume=13&publication_year=2020&pages=116-124&pmid=31647186&doi=10.1111/cts.12692&)

24. Gaedigk A, Sangkuhl K, Whirl‐Carrillo M et al. Prediction of CYP2D6 phenotype from genotype across world populations. Genet Med 2017;19:69–76.  [DOI](https://doi.org/10.1038/gim.2016.80) | [PMC free article](/articles/PMC5292679/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27388693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=Prediction%20of%20CYP2D6%20phenotype%20from%20genotype%20across%20world%20populations&author=A%20Gaedigk&author=K%20Sangkuhl&author=M%20Whirl%E2%80%90Carrillo&volume=19&publication_year=2017&pages=69-76&pmid=27388693&doi=10.1038/gim.2016.80&)

25. Gaedigk A, Dinh JC, Jeong H et al. Ten years' experience with the CYP2D6 activity score: A perspective on future investigations to improve clinical predictions for precision therapeutics. J Pers Med 2018;8:15.  [DOI](https://doi.org/10.3390/jpm8020015) | [PMC free article](/articles/PMC6023391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29673183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pers%20Med&title=Ten%20years'%20experience%20with%20the%20CYP2D6%20activity%20score:%20A%20perspective%20on%20future%20investigations%20to%20improve%20clinical%20predictions%20for%20precision%20therapeutics&author=A%20Gaedigk&author=JC%20Dinh&author=H%20Jeong&volume=8&publication_year=2018&pages=15&pmid=29673183&doi=10.3390/jpm8020015&)

26. Stamer UM, Musshoff F, Kobilay M et al. Concentrations of tramadol and O‐desmethyltramadol enantiomers in different CYP2D6 genotypes. Clin Pharmacol Ther 2007;82:41–47.  [DOI](https://doi.org/10.1038/sj.clpt.6100152) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17361124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Concentrations%20of%20tramadol%20and%20O%E2%80%90desmethyltramadol%20enantiomers%20in%20different%20CYP2D6%20genotypes&author=UM%20Stamer&author=F%20Musshoff&author=M%20Kobilay&volume=82&publication_year=2007&pages=41-47&pmid=17361124&doi=10.1038/sj.clpt.6100152&)

27. St Sauver JL, Olson JE, Roger VL et al. CYP2D6 phenotypes are associated with adverse outcomes related to opioid medications. Pharmgenomics Pers Med 2017;10:217–227.  [DOI](https://doi.org/10.2147/PGPM.S136341) | [PMC free article](/articles/PMC5533497/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28769582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenomics%20Pers%20Med&title=CYP2D6%20phenotypes%20are%20associated%20with%20adverse%20outcomes%20related%20to%20opioid%20medications&author=JL%20St%20Sauver&author=JE%20Olson&author=VL%20Roger&volume=10&publication_year=2017&pages=217-227&pmid=28769582&doi=10.2147/PGPM.S136341&)

28. Fanelli A, Palazzo C, Balzani E et al. An explorative study of CYP2D6's polymorphism in a sample of chronic pain patients. Pain Med 2020;21:1010–1017.  [DOI](https://doi.org/10.1093/pm/pnz265) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31710684/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain%20Med&title=An%20explorative%20study%20of%20CYP2D6's%20polymorphism%20in%20a%20sample%20of%20chronic%20pain%20patients&author=A%20Fanelli&author=C%20Palazzo&author=E%20Balzani&volume=21&publication_year=2020&pages=1010-1017&pmid=31710684&doi=10.1093/pm/pnz265&)

29. Dagostino C, Allegri M, Napolioni V et al. CYP2D6 genotype can help to predict effectiveness and safety during opioid treatment for chronic low back pain: Results from a retrospective study in an Italian cohort. Pharmgenomics Pers Med 2018;11:179–191.  [DOI](https://doi.org/10.2147/PGPM.S181334) | [PMC free article](/articles/PMC6205525/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30425549/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenomics%20Pers%20Med&title=CYP2D6%20genotype%20can%20help%20to%20predict%20effectiveness%20and%20safety%20during%20opioid%20treatment%20for%20chronic%20low%20back%20pain:%20Results%20from%20a%20retrospective%20study%20in%20an%20Italian%20cohort&author=C%20Dagostino&author=M%20Allegri&author=V%20Napolioni&volume=11&publication_year=2018&pages=179-191&pmid=30425549&doi=10.2147/PGPM.S181334&)

30. Smith DM, Weitzel KW, Elsey AR et al. CYP2D6‐guided opioid therapy improves pain control in CYP2D6 intermediate and poor metabolizers: A pragmatic clinical trial. Genet Med 2019;21:1842–1850.  [DOI](https://doi.org/10.1038/s41436-018-0431-8) | [PMC free article](/articles/PMC6650382/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30670877/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=CYP2D6%E2%80%90guided%20opioid%20therapy%20improves%20pain%20control%20in%20CYP2D6%20intermediate%20and%20poor%20metabolizers:%20A%20pragmatic%20clinical%20trial&author=DM%20Smith&author=KW%20Weitzel&author=AR%20Elsey&volume=21&publication_year=2019&pages=1842-1850&pmid=30670877&doi=10.1038/s41436-018-0431-8&)

31. ICD‐9‐CM to ICD‐10‐CM based on FY2014 ICD‐9‐CM codes ‐ Reportable Neoplasms. Available at https://seer.cancer.gov/tools/conversion/2014/ICD9CM_to_ICD10CM_2014CF.pdf. Accessed April 1, 2020.  [https://seer.cancer.gov/tools/conversion/2014/ICD9CM_to_ICD10CM_2014CF.pdf](https://seer.cancer.gov/tools/conversion/2014/ICD9CM_to_ICD10CM_2014CF.pdf)

32. O'Donnell PH, Bush A, Spitz J et al. The 1200 Patients Project: Creating a new medical model system for clinical implementation of pharmacogenomics. Clin Pharmacol Ther 2012;92:446–449.  [DOI](https://doi.org/10.1038/clpt.2012.117) | [PMC free article](/articles/PMC4482357/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22929923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%201200%20Patients%20Project:%20Creating%20a%20new%20medical%20model%20system%20for%20clinical%20implementation%20of%20pharmacogenomics&author=PH%20O'Donnell&author=A%20Bush&author=J%20Spitz&volume=92&publication_year=2012&pages=446-449&pmid=22929923&doi=10.1038/clpt.2012.117&)

33. Leung EKY, Agolini E, Pei X et al. Validation of an extensive CYP2D6 assay panel based on Invader and TaqMan copy number assays. J Appl Lab Med 2019;1:471–482.  [DOI](https://doi.org/10.1373/jalm.2016.021923) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33379799/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Appl%20Lab%20Med&title=Validation%20of%20an%20extensive%20CYP2D6%20assay%20panel%20based%20on%20Invader%20and%20TaqMan%20copy%20number%20assays&author=EKY%20Leung&author=E%20Agolini&author=X%20Pei&volume=1&publication_year=2019&pages=471-482&pmid=33379799&doi=10.1373/jalm.2016.021923&)

34. Drug development and drug interactions : Table of substrates, inhibitors and inducers. U.S. Food and Drug Administration Web site, 2020. Available from: https://www.fda.gov/drugs/drug‐interactions‐labeling/drug‐development‐and‐drug‐interactions‐table‐substrates‐inhibitors‐and‐inducers. Accessed April 1, 2020.  [https://www.fda.gov/drugs/drug‐interactions‐labeling/drug‐development‐and‐drug‐interactions‐table‐substrates‐inhibitors‐and‐inducers](https://www.fda.gov/drugs/drug%2010interactions%2010labeling/drug%2010development%2010and%2010drug%2010interactions%2010table%2010substrates%2010inhibitors%2010and%2010inducers)

35. Gaedigk A, Simon SD, Pearce RE et al. The CYP2D6 activity score: Translating genotype information into a qualitative measure of phenotype. Clin Pharmacol Ther 2008;83:234–242.  [DOI](https://doi.org/10.1038/sj.clpt.6100406) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20CYP2D6%20activity%20score:%20Translating%20genotype%20information%20into%20a%20qualitative%20measure%20of%20phenotype&author=A%20Gaedigk&author=SD%20Simon&author=RE%20Pearce&volume=83&publication_year=2008&pages=234-242&pmid=17971818&doi=10.1038/sj.clpt.6100406&)

36. Crews KR, Gaedik A, Dunnenberger HM et al. Clinical Pharmacogenetics Implementation Consortium guidelines for cytochrome P450 2D6 genotype and codeine therapy: 2014 update. Clin Pharmacol Ther 2014;95:376–382.  [DOI](https://doi.org/10.1038/clpt.2013.254) | [PMC free article](/articles/PMC3975212/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24458010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20guidelines%20for%20cytochrome%20P450%202D6%20genotype%20and%20codeine%20therapy:%202014%20update&author=KR%20Crews&author=A%20Gaedik&author=HM%20Dunnenberger&volume=95&publication_year=2014&pages=376-382&pmid=24458010&doi=10.1038/clpt.2013.254&)

37. Borges S, Desta Z, Jin Y et al. Composite functional genetic and comedication CYP2D6 activity score in predicting tamoxifen drug exposure among breast cancer patients. J Clin Pharmacol 2010;50:450–458.  [DOI](https://doi.org/10.1177/0091270009359182) | [PMC free article](/articles/PMC3816977/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20081063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Composite%20functional%20genetic%20and%20comedication%20CYP2D6%20activity%20score%20in%20predicting%20tamoxifen%20drug%20exposure%20among%20breast%20cancer%20patients&author=S%20Borges&author=Z%20Desta&author=Y%20Jin&volume=50&publication_year=2010&pages=450-458&pmid=20081063&doi=10.1177/0091270009359182&)

38. Shah RR, Smith RL. Addressing phenoconversion: The Achilles' heel of personalized medicine. Br J Clin Pharmacol 2015;79:222–240.  [DOI](https://doi.org/10.1111/bcp.12441) | [PMC free article](/articles/PMC4309629/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24913012/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Addressing%20phenoconversion:%20The%20Achilles'%20heel%20of%20personalized%20medicine&author=RR%20Shah&author=RL%20Smith&volume=79&publication_year=2015&pages=222-240&pmid=24913012&doi=10.1111/bcp.12441&)

39. Reidenberg MM. Pain control and the world health organization analgesic ladder. JAMA 1996;275:835.  [DOI](https://doi.org/10.1001/jama.275.11.835c) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8596211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Pain%20control%20and%20the%20world%20health%20organization%20analgesic%20ladder&author=MM%20Reidenberg&volume=275&publication_year=1996&pages=835&pmid=8596211&doi=10.1001/jama.275.11.835c&)

40. Patel JN, et al. Pain management using clinical pharmacy assessments with and without pharmacogenomics in an oncology palliative medicine clinic. JCO Oncol Pract 2020;16:e166–e174.  [DOI](https://doi.org/10.1200/JOP.19.00206) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32045554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JCO%20Oncol%20Pract&title=Pain%20management%20using%20clinical%20pharmacy%20assessments%20with%20and%20without%20pharmacogenomics%20in%20an%20oncology%20palliative%20medicine%20clinic&author=JN%20Patel&volume=16&publication_year=2020&pages=e166-e174&pmid=32045554&doi=10.1200/JOP.19.00206&)

41. Klepstad P, Fladvad T, Skorpen F et al. Influence from genetic variability on opioid use for cancer pain: A European genetic association study of 2294 cancer pain patients. Pain 2011;152:1139–1145.  [DOI](https://doi.org/10.1016/j.pain.2011.01.040) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21398039/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain&title=Influence%20from%20genetic%20variability%20on%20opioid%20use%20for%20cancer%20pain:%20A%20European%20genetic%20association%20study%20of%202294%20cancer%20pain%20patients&author=P%20Klepstad&author=T%20Fladvad&author=F%20Skorpen&volume=152&publication_year=2011&pages=1139-1145&pmid=21398039&doi=10.1016/j.pain.2011.01.040&)

42. Kekic A, Seetharam M, Singh P et al. Integrating pharmacogenomics panel testing for supportive care medications in patients with solid tumors. Journal of Clinical Oncology 2020;38:e24114.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Oncology&title=Integrating%20pharmacogenomics%20panel%20testing%20for%20supportive%20care%20medications%20in%20patients%20with%20solid%20tumors&author=A%20Kekic&author=M%20Seetharam&author=P%20Singh&volume=38&publication_year=2020&pages=e24114&)

43. Whitney RL, Bell JF, Tancredi DJ et al. Unplanned hospitalization among individuals with cancer in the year after diagnosis. J Oncol Pract 2019;15:e20–e29.  [DOI](https://doi.org/10.1200/JOP.18.00254) | [PMC free article](/articles/PMC7010432/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30523749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Oncol%20Pract&title=Unplanned%20hospitalization%20among%20individuals%20with%20cancer%20in%20the%20year%20after%20diagnosis&author=RL%20Whitney&author=JF%20Bell&author=DJ%20Tancredi&volume=15&publication_year=2019&pages=e20-e29&pmid=30523749&doi=10.1200/JOP.18.00254&)

44. Whitney RL, Bell JF, Tancredi DJ et al. Hospitalization rates and predictors of rehospitalization among individuals with advanced cancer in the year after diagnosis. J Clin Oncol 2017;35:3610–3617.  [DOI](https://doi.org/10.1200/JCO.2017.72.4963) | [PMC free article](/articles/PMC5946701/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28850290/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Hospitalization%20rates%20and%20predictors%20of%20rehospitalization%20among%20individuals%20with%20advanced%20cancer%20in%20the%20year%20after%20diagnosis&author=RL%20Whitney&author=JF%20Bell&author=DJ%20Tancredi&volume=35&publication_year=2017&pages=3610-3617&pmid=28850290&doi=10.1200/JCO.2017.72.4963&)

45. Manzano JG, Luo R, Elting LS et al. Patterns and predictors of unplanned hospitalization in a population‐based cohort of elderly patients with GI cancer. J Clin Oncol 2014;32:3527–3533.  [DOI](https://doi.org/10.1200/JCO.2014.55.3131) | [PMC free article](/articles/PMC4209103/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25287830/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Patterns%20and%20predictors%20of%20unplanned%20hospitalization%20in%20a%20population%E2%80%90based%20cohort%20of%20elderly%20patients%20with%20GI%20cancer&author=JG%20Manzano&author=R%20Luo&author=LS%20Elting&volume=32&publication_year=2014&pages=3527-3533&pmid=25287830&doi=10.1200/JCO.2014.55.3131&)

46. Iqbal J, Ginsburg O, Rochon PA et al. Differences in breast cancer stage at diagnosis and cancer‐specific survival by race and ethnicity in the United States. JAMA 2015;313:165–173.  [DOI](https://doi.org/10.1001/jama.2014.17322) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25585328/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Differences%20in%20breast%20cancer%20stage%20at%20diagnosis%20and%20cancer%E2%80%90specific%20survival%20by%20race%20and%20ethnicity%20in%20the%20United%20States&author=J%20Iqbal&author=O%20Ginsburg&author=PA%20Rochon&volume=313&publication_year=2015&pages=165-173&pmid=25585328&doi=10.1001/jama.2014.17322&)

47. Fang H, Liu X, Ramírez et al. Establishment of CYP2D6 reference samples by multiple validated genotyping platforms. Pharmacogenomics J 2014;14:564–572.  [DOI](https://doi.org/10.1038/tpj.2014.27) | [PMC free article](/articles/PMC4237721/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24980783/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Establishment%20of%20CYP2D6%20reference%20samples%20by%20multiple%20validated%20genotyping%20platforms&author=H%20Fang&author=X%20Liu&author=%20Ram%C3%ADrez&volume=14&publication_year=2014&pages=564-572&pmid=24980783&doi=10.1038/tpj.2014.27&)

48. Lopes GS, Bielinski SJ, Moyer AM et al. Sex differences in associations between CYP2D6 phenotypes and response to opioid analgesics. Pharmgenomics Pers Med 2020;13:71–79.  [DOI](https://doi.org/10.2147/PGPM.S239222) | [PMC free article](/articles/PMC7081062/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32214840/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenomics%20Pers%20Med&title=Sex%20differences%20in%20associations%20between%20CYP2D6%20phenotypes%20and%20response%20to%20opioid%20analgesics&author=GS%20Lopes&author=SJ%20Bielinski&author=AM%20Moyer&volume=13&publication_year=2020&pages=71-79&pmid=32214840&doi=10.2147/PGPM.S239222&)
