# Identification of Drug Transporter Genomic Variants and Inhibitors that Protect Against Doxorubicin–Induced Cardiotoxicity

## Metadata
**Authors:** Tarek Magdy, Mariam Jouni, Hui–Hsuan Kuo, Carly J Weddle, Davi Lyra–Leite, Hananeh Fonoudi, Marisol Romero–Tejeda, Mennat Gharib, Hoor Javed, Giovanni Fajardo, Colin J D Ross, Bruce C Carleton, Daniel Bernstein, Paul W Burridge
**Journal:** Circulation
**Date:** 2021 Dec 7
**DOI:** [10.1161/CIRCULATIONAHA.121.055801](https://doi.org/10.1161/CIRCULATIONAHA.121.055801)
**PMID:** 34874743
**PMCID:** PMC8792344
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8792344/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8792344/pdf/nihms-1763787.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8792344/pdf/nihms-1763787.pdf)

## Abstract

**Background:** 
Multiple pharmacogenomic studies have identified the synonymous genomic variant rs7853758 (G>A, L461L) and the intronic variant rs885004 in SLC28A3 as statistically associated with a lower incidence of anthracycline–induced cardiotoxicity (AIC). However, the true causal variant(s), the cardioprotective mechanism of this locus, the role of SLC28A3 and other solute carrier (SLC) transporters in AIC, and the suitability of SLC transporters as targets for cardioprotective drugs has not been investigated.

**Methods:** 
Six well–phenotyped, doxorubicin–treated pediatric patients from the original association study cohort were re–recruited and human induced pluripotent stem cell–derived cardiomyocytes were generated. Patient-specific doxorubicin–induced cardiotoxicity (DIC) was then characterized using assays of cell viability, activated caspase 3/7, and doxorubicin uptake. The role of SLC28A3 in DIC was then queried using overexpression and knockout of SLC28A3 in isogenic hiPSCs using a CRISPR/Cas9. Fine–mapping of the SLC28A3 locus was then completed after SLC28A3 resequencing and an extended in silico haplotype and functional analysis. Genome editing of potential causal variant was done using cytosine base editor. SLC28A3-AS1 overexpression was done using a lentiviral plasmid-based transduction and was validated using stranded RNA-Seq after ribosomal RNA depletion. Drug screening was done using the Prestwick drug library (n = 1200) followed by in vivo validation in mice. The effect of desipramine on DOX cytotoxicity was also investigated in eight cancer cell lines.

**Results:** 
Here, using the most commonly used anthracycline, doxorubicin, we demonstrate that patient–derived cardiomyocytes recapitulate the cardioprotective effect of the SLC28A3 locus and that SLC28A3 expression influences the severity of DIC. Using Nanopore–based fine–mapping and base editing we identify a novel cardioprotective SNP rs11140490 in the SLC28A3 locus which exerts its effect by regulating an antisense long noncoding–RNA (SLC28A3-AS1) that overlaps with SLC28A3. Using high–throughput drug screening in patient–derived cardiomyocytes and whole organism validation in mice, we identify the SLC competitive inhibitor desipramine as protective against DIC.

**Conclusions:** 
This work demonstrates the power of the human induced pluripotent stem cell model to take a SNP from a statistical association through to drug discovery, providing human cell–tested data for clinical trials to attenuate DIC.

Keywords: Doxorubicin–induced cardiotoxicity, GWAS, solute carriers, human induced pluripotent stem cells, cardiomyocytes, cardioprotectant

### Background:

Multiple pharmacogenomic studies have identified the synonymous genomic variant rs7853758 (G>A, L461L) and the intronic variant rs885004 in *SLC28A3* as statistically associated with a lower incidence of anthracycline–induced cardiotoxicity (AIC). However, the true causal variant(s), the cardioprotective mechanism of this locus, the role of *SLC28A3* and other solute carrier (SLC) transporters in AIC, and the suitability of SLC transporters as targets for cardioprotective drugs has not been investigated.

### Methods:

Six well–phenotyped, doxorubicin–treated pediatric patients from the original association study cohort were re–recruited and human induced pluripotent stem cell–derived cardiomyocytes were generated. Patient-specific doxorubicin–induced cardiotoxicity (DIC) was then characterized using assays of cell viability, activated caspase 3/7, and doxorubicin uptake. The role of *SLC28A3* in DIC was then queried using overexpression and knockout of *SLC28A3* in isogenic hiPSCs using a CRISPR/Cas9. Fine–mapping of the *SLC28A3* locus was then completed after *SLC28A3* resequencing and an extended *in silico* haplotype and functional analysis. Genome editing of potential causal variant was done using cytosine base editor. *SLC28A3-AS1* overexpression was done using a lentiviral plasmid-based transduction and was validated using stranded RNA-Seq after ribosomal RNA depletion. Drug screening was done using the Prestwick drug library (*n* = 1200) followed by *in vivo* validation in mice. The effect of desipramine on DOX cytotoxicity was also investigated in eight cancer cell lines.

### Results:

Here, using the most commonly used anthracycline, doxorubicin, we demonstrate that patient–derived cardiomyocytes recapitulate the cardioprotective effect of the *SLC28A3* locus and that SLC28A3 expression influences the severity of DIC. Using Nanopore–based fine–mapping and base editing we identify a novel cardioprotective SNP rs11140490 in the *SLC28A3* locus which exerts its effect by regulating an antisense long noncoding–RNA (*SLC28A3-AS1*) that overlaps with *SLC28A3.* Using high–throughput drug screening in patient–derived cardiomyocytes and whole organism validation in mice, we identify the SLC competitive inhibitor desipramine as protective against DIC.

### Conclusions:

This work demonstrates the power of the human induced pluripotent stem cell model to take a SNP from a statistical association through to drug discovery, providing human cell–tested data for clinical trials to attenuate DIC.

## Introduction

Doxorubicin (DOX), a cytotoxic anthracycline antibiotic, is a common anti–cancer agent used to treat a wide variety of adult and childhood cancers. The cardiotoxicity of anthracyclines has been documented to be occur in 9% of treated adult patients^1^, and on average occurs in just 3.5 months after the last chemotherapy dose and 98% of patients experience cardiotoxicity within the first year^1^. Early cardiotoxicity leads to dose limitation or treatment discontinuation to the detriment of therapy. The cardiotoxicity of doxorubicin is also well–understood to be dose–dependent, with 65% and 85% of cancer patients experiencing a decline in left ventricular ejection fraction (LVEF) when treated with DOX dose of 550 and 700 mg/m^2^, respectively^2^. However, even despite attempts to severely limit cumulative dose, cardiotoxicity occurs in 14.5% of breast cancer patients receiving the most common 240 mg/m^2^ cumulative dose^3^.

Pharmacogenomic research has attempted to discover predictive DNA biomarkers for anthracycline–induced cardiotoxicity (AIC) and has so far identified about 75 AIC–associated loci^4, 5^. However, the true connection between these loci and cardiotoxicity is far from proven, as the vast majority of AIC pharmacogenomic studies lack functional validation of the identified associations. As a result, there are currently no FDA–approved genetic biomarkers being used in routine clinical practice to predict AIC^6^, and only a single on–market drug, dexrazoxane is approved to decrease the incidence of AIC.

*SLC28A3* encodes solute carrier transporter family 28 member 3 and is the most robustly replicated AIC–associated cardioprotective loci. This locus was initially discovered in a large multi–center pediatric candidate gene association study (CGAS), identifying two single nucleotide polymorphisms (SNPs), rs7853758 (G>A, L461L), which is in a coding region but synonymous, and rs885004 which is located in intron 8. Both SNPs are in high linkage disequilibrium. rs7853758 is highly associated with a lower risk of developing DIC in both discovery (*n* = 188, *P*^adj^ = 0.0071, OR = 0.29) and replication (*n* = 156, *P*^adj^ = 0.0072, OR = 0.33) cohorts^7^. Importantly, this genetic association was replicated in a third additional multicenter independent cohort of 218 patients^8^. The sensitivity and specificity (95% CI) of rs7853758 is 17.4 (7.8–31.4) and 64.6 (58.8–70.1), respectively ^9^. Despite identification of this SNP through two replication cohorts in CGAS, the mechanisms by which a synonymous variant can influence AIC is unclear. Thus, the validity of this this locus in relation to AIC, the true causal variant and the cardioprotective mechanism of this locus, the role of SLC28A3 and other solute carrier (SLC) transporters in AIC, and the suitability of SLC transporters as targets for cardioprotective drugs are critical unanswered questions.

Here, we show that patient–specific hiPSC–CMs recapitulate the cardioprotective effect of SNP rs7853758. *SLC28A3* knockdown and overexpression using CRISPR/Cas9 reduces and increases DOX uptake into cardiomyocytes respectively, altering their sensitivity to DOX, thus confirming the role of this locus in DIC. Fine–mapping of the *SLC28A3* locus uncovered that rather than the original CGAS–identified synonymous SNP (rs7853758), it is actually the linked SNP rs11140490 that is the causal cardioprotective variant within that locus. Further mechanistic studies showed that rs11140490 exerts its action by regulating a *SLC28A3*–overlapping antisense long non–coding RNA *SLC28A3-AS1*. Screening for other potential cardiac–specific SLC transporters in relation to DIC revealed that *SLC22A4* and *SLC22A17* are also implicated in DIC. Finally screening a drug library using hiPSC–CMs followed by *in vivo* validation in a mouse model of DIC discovered that the SLC competitive inhibitor, desipramine protects against DIC by without hindering DOX chemotherapy efficacy. Together these findings provide a novel genetic test for rs11140490 that can identify patients who are protected from DIC and two potential therapeutic options, either using the lncRNA *SLC28A3-AS1* or developing a derivative of desipramine to attenuate DIC.

## Methods

The data, analytic methods, and study materials are available other researchers upon reasonable request for purposes of reproducing the results. Detailed methods are provided in the [Supplementary Information](#SD1) online. RNA–seq data have been deposited in Gene Expression Omnibus ([https://www.ncbi.nlm.nih.gov/geo/](https://www.ncbi.nlm.nih.gov/geo/)) with accession code [GSE165731](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165731).

### Human induced pluripotent cell derivation and cardiac differentiation.

Protocols and consents were approved by the Northwestern University and University of British Columbia Institutional Review Boards. Six well–phenotyped, doxorubicin–treated pediatric patients from the original association study cohort were re–recruited with informed consent. Peripheral blood mononuclear cells were isolated from blood and reprogramed to hiPSCs using CytoTune–iPS 2.0 Sendai Reprogramming Kit (Invitrogen)^10^. SNP karyotyping was performed using a whole–genome Infinium HumanCytoSNP–12 BeadChip Array (Illumina). Differentiation into cardiomyocytes was performed according to previously described protocol with some modifications^11, 12^ including the use of a *TNNT2*-driven antibiotic selection cassette for cardiomyocyte purification.

### CRISPR/Cas9–mediated gene knockout and overexpression.

To generate *SLC28A3* knockout gRNA expression vectors, gRNA targeting the start codon designed with minimal predicted off–target effect^13^. Each gRNA was annealed and inserted into pSpCas9(BB)-2A-Puro (PX459) V2.0 (48138, Addgene) plasmid that expresses puromycin resistance gene for downstream antibiotic selection, in addition to Cas9. 10^6^ cells were electroporated with 5 μg PX459 plasmid and positive clones were selected 24 h post transfection using puromycin treatment for 48 h. To generate *SLC28A3* overexpressing cells, Human *SLC28A3* Sequence-Verified cDNA was first amplified and cloned under the CAG promoter of a pAAVS1–Nst–CAG–DEST gateway cloning vector (80489, Addgene). *AAVS1* gRNA expression vector^14^ (pXAT2, Addgene 80494), which expresses gRNA and Cas9, was used to target *AAVS1* locus in the first intron of the *PPP1R12C* gene^14^. Cells were then electroporated with AAVS1 targeting plasmid and SLC28A3 overexpression donor plasmid. Positive clones were selected using neomycin treatment for 14 days.

### Patient-specific doxorubicin–induced cardiotoxicity (DIC) characterization.

Patient-specific DIC was characterized by assays of cell viability using CellTiter–Glo 2.0 (Promega) and activated caspase 3/7 using Caspase 3/7–Glo (Promega) that were used per manufacturer’s instructions. Doxorubicin uptake was quantified using flow cytometry by measuring DOX intrinsic fluorescence-PE 1 and 3 h post doxorubicin treatment and normalized to baseline fluorescence. All cells were stained with NucRed Live ReadyProbes Reagent (Invitrogen) to monitor cell viability.

### SLC28A3 locus genetic fine mapping.

Fine–mapping of the *SLC28A3* locus was then completed after *SLC28A3* resequencing using MinION Nanopore sequencer and an extended *in silico* haplotype and functional analysis.

### Genome editing of potential causal variant rs11140490.

Locus-specific base-editor protein complex and the gRNA were designed using Beditor ^15^, and the designed gRNA was cloned in the gRNA expressing plasmid (73797, Addgene). Then 1 × 10^6^ cells were electroporated with 4 μg of the base editor expressing plasmid (pSI-Target-AID-NG, 119861, Addgene) and 4 μg of the gRNA expressing plasmid (lenti sgRNA (MS2) _puro, 73797, Addgene). Cells were then selected with puromycin 24 h post transfection for 48 h, clones were hand-picked, the target locus was PCR-amplified and sanger-sequenced to confirm the SNP editing in all clones.

### SLC28A3-AS1 overexpression in isogenic hiPSCs.

The *SLC28A3-AS1* cDNA was cloned into pLenti-C-Myc-DDK-IRES-Puro lentiviral vector (Origene) which was then co–transfected with packaging plasmids psPAX2 (Addgene 12260) and pMD2.G (Addgene 12259) into Lenti-X 293T cells (Takara) to generate lentivirus. Virus–containing supernatant was collected at 48– and 72–hours post–transfection. Lentivirus was concentrated 1:100 from cleared supernatant using PEG-iT (SBI). Isogeneic hiPSCs were then transduced and positive clones were selected with puromycin for seven days to generate ISO^SLC28A3-AS1^. *SLC28A3-AS1* overexpression was confirmed using stranded RNA–Seq after ribosomal RNA depletion.

### Mouse model of doxorubicin–induced cardiomyopathy and drug administration.

Procedures followed were in accordance with Stanford University’s institutional guidelines. *In vivo* validation was done using C57BL/6J 10 week old male mice were co-treated with doxorubicin (NovaPlus) and water as a control vehicle (*n* = 10), or with desipramine (Sigma) as experimental groups (*n* = 8). At day 0, mice were treated with doxorubicin (3 mg kg^−1^) intraperitoneally twice a week alone or with desipramine by Alzet pump infusion (20 mg kg^−1^ day^−1^) for 3 weeks (day 0–day 21). For the control group, we treated mice with corn oil in the same schedule as desipramine administration. We recorded an echocardiogram once a week (day 0, day 7, day 14, and day 21) and terminated the experiment at day 21.

### Statistical Analysis.

Data were analyzed in R version 4.0.3 and graphed in GraphPad Prism 6. Detailed statistical information is included in the corresponding figure legends. Data were presented as mean ± SEM. Comparisons were conducted via one way–ANOVA test, or an unpaired two–tailed Student’s t–test with significant differences defined as *P* < 0.05 (*), *P* < 0.01 (**), *P* < 0.001 (***), and *P* < 0.0001 (****). Our sample size (3 patients in each category) was based on the feasibility of handling this number of hiPSC lines. For dose response curves, log-logistic non-linear regression model was used to estimate the value of the four parameters, and t–statistic was used to test for significant difference in LD_50_ between different groups using “drc” package^16^ in R.

## Results

### Investigation of the protective role of variant rs7853758

Six well–phenotyped, doxorubicin–treated pediatric patients from the original CGAS cohort were specifically re–recruited according to the original inclusion criteria^17^ ([Supplementary Table 1](#SD1)). These included three patients who were heterozygous for the rs7853758 variant and were protected from DIC (*SLC*^var1^, *SLC*^var2^, *SLC*^var3^; collectively *SLC*^var^), and three control patients who did not carry this protective variant (they carry the reference allele) and developed DIC after the DOX therapy (*SLC*^ref1^, *SLC*^ref2^, *SLC*^ref3^; collectively *SLC*^ref^). Detailed patient data including age, sex, ethnicity, type of cancer, treatment regimen, and presence or absence or DIC are provided in [Supplementary Table 2](#SD1). hiPSC lines were established from patients’ peripheral blood using non–integrating (Sendai virus–based) reprogramming and our well–established protocols^18, 19^. These lines showed normal hiPSC morphology ([Supplementary Fig. 1a](#SD1)), expressed high levels of undifferentiated cell markers ([Supplementary Figs. 1b](#SD1)–[c](#SD1)), and were karyotypically normal ([Supplementary Fig. 2](#SD1)). The genotypes of the rs7853758 SNP were validated using Nanopore–based sequencing ([Fig. 1a](#F1)). Cardiomyocytes were generated using our established chemically defined, small molecule–based monolayer differentiation system^11, 20^ ([Supplementary Fig. 3a](#SD1)) along with a TNNT2-based antibiotic selection cassette which consistently produces cardiomyocytes which are 80–98% TNNT2^+^ ([Supplementary Figs. 3b](#SD1)–[d](#SD1)). hiPSC-CMs express *SLC28A3* throughout the cell ([Fig. 1b](#F1)).

### Figure 1. Patient-specific hiPSC–CMs recapitulate the cardioprotective effect of SLC28A3 variant rs7853758.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/76b0/8792344/2d983ec4ddcd/nihms-1763787-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8792344_nihms-1763787-f0001.jpg)

Comparison of hiPSC–CMs derived from three patients harboring the heterozygous rs7853758 variant and were protected from DIC after DOX treatment (SLCvar1, SLCvar2, SLCvar3; collectively SLCvar), to hiPSC-CMs from three control patients who did not carry this protective SNP and developed DIC upon same DOX treatment (SLCref1, SLCref2, SLCref3 SLCref3SLCref3; collectively SLCref). A, Nanopore sequencing reads at SNP rs7853758 locus confirming its genotypes in all patient–derived hiPSC lines. B, Immunofluorescent staining showing the expression and localization of SLC28A3 throughout the cell in patient–derived hiPSC–CMs. C, Effect of DOX (72 h) on cell viability in SLCvar (n = 126) and SLCref (n = 81) hiPSC–CMs measured by a CellTiter-Glo 2.0 assay. D, Effect of DOX (72 h) on apoptosis measured by activated caspase 3/7 in SLCvar (n = 20) and SLCref (n = 20) hiPSC–CMs. E, Assessment of DOX uptake via measurement of percentage of cells with DOX intrinsic fluorescence using a flow cytometry-based assay in patient–derived hiPSC–CMs (n = 8–13). F, SLC28A3 expression in SLCref (n = 3), and SLCvar (n = 3) hiPSC-CMs using western blot. n = full independent experimental replicates, Error bars, s.e.m, *P ≤ 0.05, **P ≤ 0.01, ***P ≤ 0.001, ****P ≤ 0.0001 by unpaired two-tailed Student’s t-test (E and F). For (C and D) log-logistic non-linear regression model was used to estimate the value of the four parameters, and t-statistic was used to test for significant difference in LD50 between different groups.

### Patient–specific hiPSC–CMs recapitulate the cardioprotective effect of rs7853758 against DIC.

We first sought to model the cardioprotective effect of rs7853758 variant in patient–specific hiPSC–CMs. A cell viability assay completed at 72 h post DOX treatment demonstrated that the three *SLC*^var^ lines recapitulated the protective effect of SNP (rs7853758 G>A, L461L) (LD_50_ = 2.7 μM, *P* <0.0001, *n* = 126) as compared to the *SLC*^ref^ lines (LD_50_ = 1.38 μM, *n* = 81) ([Fig. 1c](#F1) and [Supplementary Fig. 12a](#SD1)). Consistently, apoptosis as quantified by a caspase–3 and –7 activity assay completed 72 h post DOX treatment was significantly lower in *SLC*^var^ lines (IC_50_ = 1 μM, *P* = 0.001, *n* = 20) compared to the *SLC*^ref^ lines (IC_50_ = 0.43 μM, *n* = 20) ([Fig. 1d](#F1)). Since *SLC28A3* is an uptake transporter, we hypothesized that the lower level of DIC in the *SLC*^var^ lines might be due to reduced DOX uptake as a result of impaired *SLC28A3* function in these patients. To test this, we used a flow cytometry–based assay and found that intracellular DOX uptake was ~50% lower in *SLC*^var^ as compared to *SLC*^ref^ ([Fig. 1e](#F1)), which is consistent with the magnitude LD_50_ (1.38 μM vs. 2.7 μM) and IC_50_ (0.43 μM and 1 μM) changes on our *in vitro* DIC assays ([Fig. 1c](#F1)). To investigate whether this cardioprotective effect seen in *SLC*^var^ is due to altered SLC28A3 protein expression, we then quantified SLC28A3 in our patient–specific hiPSC–CMs using western blot. SLC28A3 expression in *SLC*^var^ was significantly lower than *SLC*^ref^ (*P* = 0.03) ([Fig. 1f](#F1), and [Supplementary Fig. 12b](#SD1)).

### SLC28A3 altered expression affects doxorubicin–induced cardiotoxicity in hiPSC–CMs.

To further confirm the implication of *SLC28A3* in DIC regardless of patient-specific genetic, transcriptomic, and environmental background and to isolate the effect of *SLC28A3* in relation to DIC, we next examined whether a gain or loss of function of *SLC28A3* altered DIC in an isogenic hiPSC line (ISO). The ISO line was derived from a healthy individual and its whole genome sequencing confirmed that it doesn’t harbor any other DIC–associated loci identified by the original study. *SLC28A3* overexpression (ISO–OE) and knockout (ISO–KO) lines were generated via a CRISPR/Cas9–mediated approach. Disturbance at start codon region was confirmed by Sanger sequencing ([Fig. 2a](#F2)–[b](#F2)) and altered *SLC28A3* expression in these lines was confirmed by RT-PCR and western blot showing 90% downregulation of SLC28A3 in ISO-KO ([Fig. 2c](#F2)). The effect of *SLC28A3* OE and KO on *in vitro* DIC was investigated using the above cell viability and caspase assays at 72 h post DOX treatment. The cell viability assay showed that the ISO–OE hiPSC–CMs (LD_50_ = 0.9 μM) were ~3.3–fold and 1.4–fold more sensitive to DOX as compared to ISO–KO (LD_50_ = 4 μM, *P* = 0.03) and ISO (LD_50_ = 2.5 μM, *P* < 0.0001), respectively ([Fig. 2d](#F2)). Likewise, caspase activity was ~2.6–fold higher in ISO–OE (LD_50_ = 0.14 μM) as compared to ISO (LD_50_ = 0.53 μM, *P* <0.0001) ([Fig. 2e](#F2)). We next sought to investigate the effect of *SLC28A3* KO and OE on DOX intracellular uptake. DOX uptake was significantly higher in ISO–OE as compared to ISO both at 1 h (*P* = 0.035) and 3 h (*P* < 0.0017) post DOX treatment ([Fig. 2f](#F2)). Similarly, DOX uptake was significantly lower in ISO–KO as compared to ISO both at 1 h (*P* = 0.0009) and 3 h (*P* = 0.0006) post DOX treatment ([Fig. 2f](#F2)). These findings show that *SLC28A3* is implicated in DIC regulation through affecting DOX uptake into cardiomyocytes.

### Figure 2. SLC28A3 expression affects the severity of DIC by regulating DOX uptake into cardiomyocytes.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/76b0/8792344/c39030c3df90/nihms-1763787-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8792344_nihms-1763787-f0002.jpg)

A, Validation of CRISPR/Cas9–mediated SLC28A3 knockout (KO) in an isogenic hiPSC line detected by Sanger sequencing, showing 8 bp deletion downstream of the transcription start site (TSS). PAM, protospacer adjacent motif. B, Demonstration that 91% of the cell population acquire the introduced deletion. C, Validation of KO and AAVS1-based SLC28A3 overexpression (OE) by western blot and RT-PCR. D, Effect of DOX (72 h) on viability in ISO (n = 45), ISO–OE (n = 14), and ISO–KO (n = 6) hiPSC–CMs. E, Effect of doxorubicin (72 h) on apoptosis measured by activated caspase 3/7 in ISO (n = 8), ISO–OE (n = 10), and ISO–KO (n = 6) hiPSC–CMs. F, Assessment of DOX uptake via measurement of DOX intrinsic fluorescence using flow cytometry-based assay (n = 6–9). n = full independent experimental replicates, Error bars, s.e.m, *P ≤ 0.05, **P ≤ 0.01, ***P ≤ 0.001, ****P ≤ 0.0001 by unpaired two-tailed Student’s t-test (f). For (d and e) log-logistic non–linear regression model was used to estimate the value of the four parameters, and t-statistic was used to test for significant difference in LD50 between different groups.

### Fine mapping at the SLC28A3 locus prioritizes potential causal variant.

rs7853758 is a coding synonymous SNP located in exon 14 and thus does not affect the amino acid sequence. Because of the tag–SNP and linkage disequilibrium (LD) issues associated with GWAS^21^, we expected that rs7853758 is linked (coinherited) to additional SNPs including the causal one(s). To elucidate this, we next fine–mapped the *SLC28A3* locus to identify the potential causal variant. We sequenced the *SLC28A3* gene in all six *SLC*^var^ and *SLC*^ref^ patients using a Nanopore MinION sequencer and SNPs were called using Nanopolish^22^ (see [online methods](#SD1)). For all patients, Nanopore genotypes of the original association study hit, rs7853758 were in concordance with the GWAS–chip genotypes ([Fig. 1a](#F1)). In total 133 SNPs were identified all of which have at least one variant allele in at least one patient ([Supplementary Fig. 4e](#SD1)). The vast majority of identified SNPs were intronic (*n* = 93), 25 SNPs were located in 5’–UTR, 12 SNPs were located in 3’–UTR, in addition to three coding SNPs of which two were synonymous and one non–synonymous ([Supplementary Table 6](#SD1)). We then examined which SNPs were exclusively co–inherited in cardioprotected patients and identified a cardioprotective haplotype, Hap^SLC28A3^ comprising 24 SNPs that is co–inherited only in cardioprotected patients. These SNPs are distributed as follows, eight SNPs are located in the 3'–UTR, 14 SNPs are intronic, and two are coding synonymous SNPs ([Figs. 3a](#F3)–[b](#F3) and [Supplementary Table 6](#SD1)).

### Figure 3. Fine–mapping at the SLC28A3–SLC28A3-AS1 locus identifies rs11140490 as the potential causal cardioprotective variant.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/76b0/8792344/e1249b24a9e5/nihms-1763787-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8792344_nihms-1763787-f0003.jpg)

A, Location of the HapSLC28A3 comprising 24 SNPs that are co–inherited only in SLCvar protected patients. SNP rs11140490 marked by red rectangle is located at the splice site of the first exon of an overlapping long non–coding RNA, SLC28A3-AS1 (adapted from Magdy et al.29). B, Consequence of co–inherited HapSLC28A3 SNPs (n = 24). C, Overall prioritization of candidate causal SNPs based on functional annotation analyses including, eQTL annotation, chromatin regulatory analyses, and overlapping with regulatory regions (RR) in cardiac tissues. FA, functional annotation (adopted from Magdy et al.29). D, Editing of rs11140490 (CT>TT) in two patient-specific hiPSC lines using cytosine base editor (Target-AID-NG), rs11140490 is marked by red asterisk. E, Effect of DOX (72 h) on viability in SLCvar-rs11140490 (CT) (2 lines, n = 31) and SLCvar-rs11140490(CT>TT edited) (2 lines, n = 75) hiPSC–CMs. F, Assessment of DOX uptake via measurement percentage of cells with DOX intrinsic fluorescence using flow cytometry-based assay in SLCvar-rs11140490 (CT) and SLCvar-rs11140490(CT>TT edited) (n = 4). n = full independent experimental replicates, Error bars, s.e.m, *P ≤ 0.05, **P ≤ 0.01, ***P ≤ 0.001, ****P ≤ 0.0001 by unpaired two-tailed Student’s t-test (F). For (E) log-logistic non-linear regression model was used to estimate the value of the four parameters, and t-statistic was used to test for significant difference in LD50 between different groups.

Interestingly, seven SNPs within Hap^SLC28A3^ are located within a long non–coding RNA, ensemble gene id: ENSG00000233262 that we called “*SLC28A3-AS1”* (submitted to HGNC registry) that overlaps with *SLC28A3* forming another haplotype, Hap^SLC28A3-AS1^ ([Fig. 3a](#F3), [Supplementary Figs. 5a](#SD1)–[b](#SD1)). Hap^SLC28A3-AS1^ consists of SNPs, rs11140490 (A>G), rs10868135 (T>C), rs4877831 (C>G), rs4877833 (T>C), rs7853066 (A>G), rs7853758 (G>A), and rs7030019 (A>G).

In order to validate and confirm the linkage disequilibrium pattern of the Nanopore–identified cardioprotective haplotype, Hap^SLC28A3^, we investigated the haplotype structure and allelic frequency of this cardioprotective haplotype on a wider population level in 99 individuals of the CEU (Utah Residents (CEPH) with Northern and Western European Ancestry) population, the same ethnic population of the study cohort. This analysis showed that the 24 SNPs constituting the Hap^SLC28A3^ were in high LD with an average D’ and R^2^ of 0.99 and 0.84, respectively ([Supplementary Figs. 5a](#SD1)–[b](#SD1) and [Supplementary Table 11](#SD1)). Regarding Hap^SLC28A3-AS1^, seven structures were identified, Hap–I^SLC28A3-AS1^ to Hap–VII^SLC28A3-AS1^ ([Supplementary Fig. 5c](#SD1)). In that, Hap–I^SLC28A3-AS1^ consists of the reference alleles for all seven SNPs (ATCTAGA) and is inherited in 71.7% of the examined population, whereas Hap–II^SLC28A3-AS1^ comprises the variant alleles for all seven SNPs (GCGCGAG) and is inherited in 17.7% of the examined population ([Supplementary Fig. 5c](#SD1)). This finding confirms the linkage disequilibrium pattern identified by the Nanopore pipeline in all of the study patients.

To eliminate the probability that the causal variant might be located in one of the adjacent genes to the *SLC28A3 / SLC28A3-AS1* target locus, we did an extended LD analysis to include all variants that are located with 1 MB up and down–stream of the target locus in both European (the same ethnic population of the original genotype–phenotype association study) and all ethnicity populations. These analyses did not identify any other SNPs that are linked to rs7853758 and are not included in our original haplotype analysis. This eliminates the probability that the causal cardioprotective variant is located in adjacent genes ([Supplementary Fig. 6](#SD1)).

In order to prioritize cardioprotective haplotype SNPs, we investigated the regulatory properties of all candidate SNPs (*n* = 24). Using the Encyclopedia of DNA Elements (ENCODE) and Roadmap Epigenomics^23^ data and DeepSEA^24^ algorithm, we examined the functional effect of each SNP on altering chromatin features (transcription factors, DNase hypersensitive site, and histone marks) binding sites. Among all SNPs, rs11140490 and rs4877835 had the top chromatin regulatory effect as both SNPs predicted to alter the binding site of 206 and 204 chromatin features, respectively ([Supplementary Fig. 4f](#SD1), [Supplementary Tables 7](#SD1) and [8](#SD1)). Moreover, SNP rs11140490 has the most substantial regulatory effect as it is predicted to alter the binding sites of 43 features with log_2_ fold change of ≥1, whereas rs4877835 is predicted to alter the binding sites of only 4 features with log_2_ fold change of ≥1. Unsurprisingly, the primary study significant association, rs7853758 does not show any significant chromatin regulatory effect ([Supplementary Fig. 4g](#SD1)).

Since DIC affects cardiomyocytes, we performed an additional regulatory analysis exclusively focusing on human cardiac tissue, and for that we used ensemble regulatory build that includes transcription factors, histone mark, and DNase hypersensitive regions. Six SNPs, rs11140490, rs4877835, rs4877831, rs7047898, rs885004, and rs10868137 were found to be located in at least one regulatory region in human cardiac tissue ([Supplementary Table 9](#SD1)). Finally, to investigate further regulatory consequences of these candidate SNPs, we used the Genotype–Tissue Expression (GTEx) project database ([https://www.gtexportal.org/home/](https://www.gtexportal.org/home/)) to investigate which of the identified candidate SNPs have been shown to be an expression quantitative trait loci (eQTL). All candidate SNPs except rs7858075 have been previously identified as eQTL in cultured fibroblasts, thyroid, and brain tissues ([Supplementary Table 10](#SD1)). Although the eQTLs identified by the GTEx are located in non-cardiac tissues, these associations emphasize the regulatory function of these SNPs. These findings when taken together suggest that SNP rs11140490, interestingly located at the splice site of the first exon of *SLC28A3-AS1*, is the SNP with the highest likelihood to be the causal cardioprotective SNP ([Fig. 3c](#F3)).

### Editing rs11140490 in hiPSC–CMs confirms its causality in relation to protection against DIC.

After we prioritized rs11140490 to be the top candidate causal variant within the DIC–associated *SLC28A3*–*SLC28A3-AS1* locus, we went on to confirm the causality of this variant. Using a base–editor mediated approach, we edited the SNP rs11140490 in hiPSCs from two *SLC*^var^ patients that harbor the heterozygous genotype CT back to the reference genotype TT. The cytosine base editor that we have used is composed of a catalytically inactive “dead” Cas9 (dCas9) fused to (CBE) cytidine deaminase that converts C•G base pair to a T•A base pair. Importantly, the CBE does not induce a DNA cut and thus helped us avoid the mono-allelic genomic deletions and loss-of-heterozygosity problem associated with the homology directed repair (HDR)-based genomic editing approaches^25^. Positive hiPSC clones were differentiated into cardiomyocytes and finally DIC and DOX uptake were quantified using the above assays. *SLC*^rs11140490(CT>TT)^ cardiomyocytes were more sensitive to DOX (LD_50_ = 1.37 μM, *P* = 0.005) as compared to *SLC*^rs11140490(CT)^ LD_50_ = 1.9 μM) ([Fig. 3e](#F3)). Moreover, DOX uptake was significantly higher in *SLC*^rs11140490(CT>TT)^ when compared to *SLC* ^rs11140490(CT)^ at 3 h post DOX treatment (*P* = 0.006) ([Fig. 3f](#F3)). This finding confirms that the SNP rs11140490 is the causal cardioprotective SNP affecting DIC.

### Variant rs11140490 exert its cardioprotective effect by regulating the long non–coding RNA, SLC28A3-AS1.

Next, we investigated the cardioprotective mechanism of rs11140490. This variant is located at the splice site of the first exon of the anti–sense *SLC28A3-AS1* that overlaps with *SLC28A3*. Thus, we hypothesized that SNP rs11140490 might exert its cardioprotective action via regulating the transcription of *SLC28A3-AS1*, which in turn alters the expression of doxorubicin–related genes including *SLC28A3*, and eventually alters patients’ susceptibility to DIC. To test this hypothesis, we investigated the effect of altered expression of *SLC28A3-AS1* on DIC phenotype. We overexpressed *SLC28A3-AS1* in an isogenic cell line (ISO^SLC28A3-AS1^) by transducing isogenic hiPSCs with the *SLC28A3-AS1* cDNA cloned into pLenti-*MYC*-DDK-IRES-Puro lentivirus expression vector (ORIGENE, PS100069). *SLC28A3-AS1* overexpression was confirmed by stranded RNA-Seq after ribosomal RNA depletion ([Fig. 4a](#F4)). We then assessed the effect SLC28A3-AS1 overexpression on *SLC28A3* expression in cardiomyocytes and showed that *SLC28A3* is significantly downregulated in ISO^SLC28A3-AS1^ cardiomyocytes after DOX treatment ([Fig. 4b](#F4)). We then investigated the effect of *SLC28A3-AS1* overexpression on DIC and DOX uptake as before. ISO^SLC28A3-AS1^ hiPSC–CMs were significantly more resistant to DOX (LD_50_ = 8.4 μM, *P* < 0.0001) as compared to ISO (LD_50_ = 1.8 μM) ([Fig. 4c](#F4)). Moreover, DOX uptake was significantly impaired in ISO^SLC28A3-AS1^ cardiomyocytes when compare to ISO both at 1 h (*P* = 0.001) and 3 h (*P* < 0.001) post DOX treatment ([Fig. 4d](#F4)). These results show that the regulation of *SLC28A3-AS1* is a potential mechanism by which SNP rs11140490 exerts its cardioprotective effect.

### Figure 4. The cardioprotective role of SLC28A3-AS1.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/76b0/8792344/f707007ab725/nihms-1763787-f0004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8792344_nihms-1763787-f0004.jpg)

A, Relative Expression of SLC28A3-AS1 in ISO hiPSC compared to ISO transduced by viral plasmid encoding SLC28A3-AS1 cDNA to overexpress SLC28A3-AS1 (ISOSLC28A3-AS1) assessed by RNA–Seq (n = 2). B, SLC28A3 relative expression in ISO and ISOSLC28A3-AS1 hiPSC-CMs (n = 5–11) assessed by real-time PCR. C, Effect of SLC28A3-AS1 overexpression on cell viability after DOX (72 h) treatment, ISO (n = 17), ISOSLC28A3-AS1 (n = 12). D, Effect of SLC28A3-AS1 overexpression on DOX uptake 1 h and 3 h post DOX treatment (n = 6–14). E, Relative human cardiomyocyte expression of SLC transporters (n = 12) previously identified as transporting DOX or a DOX metabolite and/or by genetic associations with DOX clinical outcomes. Red dashed line denoted for the expression cutoff for SLC transporter selection. F, Effect of knocking out DOX–relevant SLC transporters on DOX uptake into patient–derived cardiomyocytes [SLC28A3KO, SLC22A4KO, SLC22A3KO, and SLC22A17KO (n = 5–13)]. G, Effect of knocking out potential cardiac-specific SLC transporters on cell viability after DOX treatment [SLC28A3KO (n = 14), SLC22A4KO (n = 58), SLC22A3KO (n = 17), and SLC22A17KO, (n = 10), ISO (n = 128)]. n = full independent experimental replicates, Error bars, s.e.m, *P ≤ 0.05, **P ≤ 0.01, ***P ≤ 0.001, ****P ≤ 0.0001 by unpaired two-tailed Student’s t-test (B, D, and F). For (C and G) log-logistic non-linear regression model was used to estimate the value of the four parameters, and t-statistic was used to test for significant difference in LD50 between different groups.

### Implication of other SLC transporters in DIC regulation.

After we provided proof of principle for the importance of SLC transporters in DIC by showing that *SLC28A3* expression and genomic variants affect the severity of DIC, we examined other potential SLC transporters that might affect DIC. To date, there are more than 450 identified SLC transporters, 12 of which have been shown to either transport DOX or a DOX metabolite and/or their genes harbor SNPs that are significantly associated with DOX clinical outcomes^26^. We examined which of these 12 *SLC* transporters are expressed in adult heart tissue, fetal heart tissue, and hiPSC–CMs. Only three additional SLC transporters met these criteria, *SLC22A4*, *SLC22A3*, *SLC22A17* were selected for further investigation ([Fig. 4e](#F4)). For each of these transporters, we generated CRISPR/Cas9–mediated KO hiPSC (*SLC22A4*^KO^, *SLC22A3*^KO^, and *SLC22A17*^KO^) and differentiated them to cardiomyocytes ([Supplementary Fig. 7](#SD1)). We examined the effect of altered expression of each candidate transporter on DOX intracellular uptake and cell viability after DOX treatment. Expectedly, *SLC22A3*, *SLC22A4*, *SLC28A3*, and *SLC22A17* knockout resulted in a significant decrease in DOX uptake in isogenic hiPSC–CMs ([Fig. 4f](#F4)). DIC quantification revealed that *SLC22A4*^KO^ cardiomyocytes (LD_50_ = 3.8 μM, *P* < 0.0001, *n* = 58), *SLC28A3*^KO^ cardiomyocytes (LD_50_ = 3.3 μM, *P* < 0.0001, *n* = 17), and *SLC22A17*^KO^ cardiomyocytes (LD_50_ = 3 μM, *P* = 0.005, *n* = 10) were less sensitive to DOX and more protected against DIC as compared to ISO (LD_50_ = 1.9 μM, *P* < 0.0001, *n* = 128). Whereas, knocking out *SLC22A3* had a detectable yet not significant effect on DIC (LD_50_ = 2.1 μM, *P* = 0.5, *n* = 17) ([Fig. 4g](#F4)).

### High–throughput drug screening reveals the SLC inhibitor, desipramine as a novel cardioprotective drug against DIC in hiPSC–CMs and murine.

Since SLC transporters affect DIC and represent well–founded druggable targets, we went on to examine whether SLC substrates can alter DOX uptake and subsequently regulate DIC in hiPSC–CMs. To test this, we first screened 17 drugs with well-established roles in affecting SLC transporter efficacy in relation to DOX uptake ([Supplementary Table 12](#SD1)). In order eliminate potential toxicity from high doses of the SLC substrate drugs, LD_50_ was determined in our ISO cardiomyocytes for the 17 drugs to determine relevant maximum tolerable dose ([Supplementary Fig. 8](#SD1)). This cherry–picked screening revealed that desipramine was the only substrate that altered DOX uptake at both 1 h and 3 h post DOX treatment. DOX uptake was significantly lower in cells co–treated with desipramine (3 μM) plus doxorubicin (3 μM) as compared to cells treated with doxorubicin (3 μM) only, after 1 h (*P* = 0.008) and 3 h (*P* = 0.04) posttreatment ([Supplementary Figs. 9a](#SD1)–[b](#SD1)). Since desipramine significantly decreased doxorubicin transport into cardiomyocytes, we next inspected whether lower doxorubicin intracellular concentration in desipramine pre–treated cells affect the magnitude of DIC.

Next, we sought to expand our screening for novel cardioprotectants by testing the cardioprotective role of the Prestwick drug library that includes 1219 drugs. Cells were pre–treated with 3 μM of relevant drugs for 24 h, then co–treated with relevant drug (3 μM) and doxorubicin (10 μM) for 72 h after which DIC was quantified. For each plate, untreated and doxorubicin (10 μM) only treated cells were included to serve as negative and positive controls, respectively. Drug library screening reveled 34 cardioprotective molecules that attenuated cell death after DOX treatment ([Fig. 5a](#F5)). Drug repurposing is significantly faster and more cost–effective than *de novo* drug discovery approaches, and thus we focused on the top FDA–approved drugs identified from our screening. Interestingly, the top ten significant cardio–protectants included six FDA approved drugs; fluoxetine, butoconcazole, perospirone, tetracaine, propofol, and desipramine ([Fig. 5b](#F5)), of these drugs, desipramine again showed the most significant protection against DOX–induced cardiomyocyte death (*P* = 0.007, 2.5–fold) ([Fig. 5a](#F5)–[b](#F5)). To further validate these results, we then investigated the cardioprotective effect of these FDA–approved cardioprotective drugs against 10 log–doses of doxorubicin. This analysis showed that desipramine has the strongest cardioprotective effect when compared to cells treated only with DOX (LD_50_ = 10.66 μM, *P* < 0.0001), followed by propofol (LD_50_ = 4.77 μM, *P* = 0.003), then tetracaine (LD_50_ = 4.65 μM, *P* < 0.001), then butoconazole (LD_50_ = 4.4 μM, *P* = 0.01) ([Fig. 5c](#F5)–[d](#F5), [Supplementary Fig. 9c](#SD1)).

### Figure 5. Multi-modality drug screening identifies desipramine as a novel effective cardioprotectant against DIC.

![Figure 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/76b0/8792344/65bda140b635/nihms-1763787-f0005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8792344_nihms-1763787-f0005.jpg)

A, Prestwick drug library screening (n = 1219) in relation to DIC (n = 5). All drugs were used at 3 μM. Red dashed line represents cell viability 72 h post DOX (10 μM) treatment; the top ten significant cardioprotective drugs based on cell viability are labeled. B, Bar plot showing top ten significant cardioprotective (based on P value) compared to DOX alone (72 h, 10 μM) treated cells. Non–FDA–approved drugs are represented by teal bars. C, Further validation of top FDA-approved drugs (identified from the Prestwick library screening) against 10 log–doses of doxorubicin. LD50, median lethal dose. D, Effect of co-treatment of desipramine (3 μM) and doxorubicin (72 h) on hiPSC-CM viability [DOX (n = 42), DOX + DESP (n = 35)]. E, Percent change in ventricular fraction shortening (FS) normalized to baseline, after 3 weeks of doxorubicin treatment (3 mg/kg, ip, n = 10) compared co-treatment (n = 8) of desipramine (20 mg/kg/day, Alzet pump) and doxorubicin (3 mg/kg, ip) in mice. F, Ventricular fractional shortening at baseline, 1-, 2-, 3-weeks post treatment. G, Assessment of cell viability of MCF breast cancer cell line after 72 h of DOX and desipramine (DESP) cotreatment (n = 12–20). f = full independent experimental replicates, Error bars, s.e.m, *P < 0.05, **P ≤ 0.01, ***P < 0.001, ****P < 0.0001 by unpaired two-tailed Student’s t-test (A-C and E) and by ANOVA with post-hoc testing (F). For (D and G) log-logistic non-linear regression model was used to estimate the value of the four parameters, and t-statistic was used to test for significant difference in LD50 between different groups.

Based on these findings, we selected desipramine as our lead cardioprotective drug, and thus we then investigated whether or not desipramine could attenuate DIC *in vivo*. For that, we treated mice with doxorubicin (3 mg kg^−1^ intraperitoneal twice weekly for 3 weeks) plus desipramine (20 mg kg^−1^ day^−1^ infusion for 3 weeks), or water as a vehicle control. Doxorubicin treatment results in a steady decline in cardiac function, as assessed by fractional shortening. Critically, cardiac function was significantly less attenuated by doxorubicin at three weeks with desipramine (*P* < 0.05), compared with vehicle treatment ([Fig. 5e](#F5)–[f](#F5), [Supplementary Fig. 12b](#SD1)–[c](#SD1), and [Supplementary Table 13](#SD1)). To rule out the possibility that desipramine could diminish DOX chemotherapeutic effect, we studied eight cancer cell lines, representing breast, liver, colorectal, prostate, uterus, cervix, and bone cancers and found that co–treatment with desipramine did not impede the anticancer efficacy of DOX in any of these lines ([Fig. 5f](#F5), and [Supplementary Fig. 10](#SD1)).

## Discussion

The identification of reliable predictive genomic biomarkers for DIC and the discovery of efficient cardioprotectants are indispensable to enhance the clinical utility of doxorubicin in cancer treatment. The identification of gene variants predictive of altering DIC through GWAS has provided impetus for developing platforms to confirm these GWAS hits, moving them from ‘association’ to ‘confirmed mechanism’. In this work we demonstrated that the patient–specific hiPSC–CM model is ideal for studying the implication of transporter inhibitors and genetic variants in DIC. We show that patient–specific cardiomyocytes do indeed recapitulate the cardioprotective effect of the CGAS-identified *SLC28A3* locus; confirm for the first time the role of *SLC28A3* in DIC independent of patient–specific genetic background; and critically reveal that another SNP within this cardioprotective locus, rs11140490 has the highest likelihood to be causal. Furthermore, this platform allowed us to discover that the SLC competitive inhibitor, desipramine protects against DIC without hampering DOX chemotherapy efficacy.

The *SLC28A3* genetic variant rs7853758 is the most robustly replicated AIC strongly associated cardioprotective loci. We show that rs7853758 is in perfect LD with 23 other SNPs of which 22 are non–coding variants, forming the cardioprotective Hap^SLC28A3^. Pinpointing causal SNP within this locus is crucial for clinical translation because testing for the causal variant guarantees the detection of the best possible clinical correlation with AIC. Almost 93% of phenotype–associated genetic variants are non–coding^27^. Using the ENCODE project dataset, it has been shown that ~80% of GWAS–identified non–coding SNPs are not the causal SNP^28^. Using our recently developed cost–effective fine mapping pipeline^29^, we found that rs11140490, but not the CGAS–identified hit rs7853758, is the variant with the highest likelihood to be causal in DIC.

Editing only the rs11140490 cardioprotective genotype (CT) back to the reference genotype (TT) in patient–specific hiPSC–CMs renders these cardiomyocytes more susceptible to DIC confirming the causality of this novel variants. Interestingly, editing rs11140490 increase the sensitivity of hiPSC-CMs by ~28% which is modest and thus denotes that there might be more SNPs within the Hap^SLC28A3^ that interact in an additive manner with rs11140490 to predispose to protection against DIC

Going forward, we propose that a simple clinical test to detect the presence of rs11140490 can be used to predict that a patient will be less likely to experience DIC and that, with future clinical trials, it may be possible for these patients to be treated with a longer duration (higher cumulative dose) of doxorubicin to enhance the efficacy of their chemotherapy. Similarly, the rs11140490 genetic testing could be employed as a part of a polygenic cardiotoxicity risk stratification score for doxorubicin-containing chemotherapy regimens such that the protective effect of rs11140490 could balance out for a risk factor that would have otherwise prevented the administration of a relevant doxorubicin-containing chemotherapy regimen to a particular patient.

*SLC28A3* encodes a cardiac–specific uptake transporter that has no/marginal expression in several cancer cells and hence, represents a highly druggable target to screen for cardioprotective agents. A large number of drugs have been identified as being trafficked by *SLC* transporters which explains the substantial role of these transporters in both drug pharmacokinetics and pharmacodynamics and emphasizes the importance of this class of transporters in drug response disposition^30^. Our high–throughput drug screening discovered that treating patient–specific cardiomyocytes with the *SLC* inhibitor desipramine protects against DIC through decreasing the intracellular uptake of DOX into human heart. Our results suggest that a single dose of 3 μM desipramine 24 h before the administration of DOX in addition to another dose of 3 μM desipramine co–administered with DOX is sufficient for protecting against DIC.

Desipramine is a tricyclic antidepressant sold under that brand name Norpramin, that was first patented in 1962^31^. The typical adult dose of desipramine 100 mg to 200 mg/day. In more severely ill patients, dosage may be further increased gradually to 300 mg/day if necessary. The typical therapeutic concentration is 100–300 ng/ml with daily dosing. Our 3 μM *in vitro* dose would be 798 ng/ml, but importantly, our preliminary data suggest that just two doses or potentially one dose per cycle of doxorubicin would be required to attenuate DIC. We would therefore suggest that these desipramine doses are potentially within the acceptable clinical range, although further animal model work to confirming dosing strategy is required prior to clinical trial.

Desipramine is far from an ideal drug, being the most potent sodium channel blocker among its group and causing cardiotoxicity when used chronically. Desipramine-treated patients have been shown to have significantly lower rates of sinus pauses and junctional rhythm, but significantly higher rates of single or paired premature atrial contractions and runs of supraventricular tachycardia as well as lengthening of the QT interval. The solution to this is to develop a desipramine derivative without the sodium channel blockade effect, although approach eliminates the primary advantage of repurposing a drug like desipramine to attenuate DIC. Still, the advantage of using our hiPSC-CM platform for subsequent testing will be a major advantage in such an effort

The majority (56%) of candidate drugs have failed in clinical trial due to the lack of efficacy, most likely because the pre-clinical models used to test the drug does not recapitulate what happens in humans^32^ or in the specific target cell type. Additionally, studies staring with genetic correlations in genes encoding targets increases the success rate in clinical development by 2–fold^33^. Patient-derived hiPSC–CMs provides a unique platform that firstly permits a thorough validation of GWAS–identified AIC–associated loci, and secondly recapitulate alteration in DIC phenotype in a human–relevant manner. Hence the utilization of hiPSC–CMs in the development of cardioprotectants substantially improves the potential of developing novel derivatives of desipramine that have the same SLC28A3 competitive inhibition effect as desipramine without the well–known side effects common to the tricyclic antidepressant drug family.

Clearly hiPSC–CMs do not fully mimic the human whole–body model and do not recapitulate all the steps of drug pharmaco–kinetics and –dynamics. However, current hiPSC–CM generation methodologies have enhanced the robustness, purity, maturation, and scalability to a point where these cells are suitable for a wide-range of disease modelling and drug response assays^34–42^. Here we demonstrated that patient–specific cardiomyocytes recapitulate intra–individual variability in genomic–dependent DIC susceptibility. We show that hiPSC–CMs are appropriate to study drug response–associated loci especially for genes with a known mechanism of action such as transporter–encoding genes. The integration of CRISPR/Cas9–based genetic editing to our patient–specific hiPSC model proves to be a powerful tool in identify causal genetic variations in relation to a specific drug–response. The expansion of the utility of patient–specific hiPSC–CMs to study additional anthracycline–relevant loci as well as those of other anti-cancer agents will help to identify patients/population–specific chemotherapeutic–induced cardiotoxicity genetic biomarkers and new cardioprotective agents. Ultimately, the information derived from this platform may allow physicians to tailor chemotherapeutics doses based on patient genotype, bringing the promise of personalized medicine to the field of cardio–oncology.

## Supplementary Material

### What Is New?

### What Is New?

### What Is New?

### What Is New?

### What Are the Clinical Implications?

## Acknowledgments

P.W.B. and T.M. designed the research. T.M., M.J., H.–H.K., C.W., D.L–L., H.F., M.R.–T., M.G., H.J., and P.W.B. performed the experiments. G.J. and D.B. designed and the performed mouse experiments. C.J.D.R. and B.C.C determined the AIC phenotype to be used patient recruitment and provided patient samples. P.W.B. supervised the project. T.M. and P.W.B. wrote the manuscript with input from all other authors.

### Funding Sources

This work was supported by NIH grants K99/R00 HL121177, R01 CA220002, R01 CA261898, American Heart Association Transformational Project Award 18TPA34230105, and the Fondation Leducq (P.W.B.), Canadian Cancer Society (C.J.R., B.C.C., P.W.B.), Michael Smith Foundation for Health Research Scholar Award (C.J.R.), and NIH R33 HL123655 (D.B., B.C.C.). Patients from which hiPSC were derived were recruited by the Canadian Pharmacogenomics Network for Drug Safety (B.C.C. and C.J.R.) which has received grant funding from the Canadian Institutes for Health Research (CIHR), the CIHR Drug Safety and Effectiveness Network, Genome Canada, Genome British Columbia, and the British Columbia Provincial Health Services Authority.

## Non–Standard Abbreviations and Acronyms

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Cardinale D, Colombo A, Bacchiani G, Tedeschi I, Meroni CA, Veglia F, Civelli M, Lamantia G, Colombo N, Curigliano G, et al. Early detection of anthracycline cardiotoxicity and improvement with heart failure therapy. Circulation. 2015;131:1981–8.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.114.013777) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25948538/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Early%20detection%20of%20anthracycline%20cardiotoxicity%20and%20improvement%20with%20heart%20failure%20therapy&author=D%20Cardinale&author=A%20Colombo&author=G%20Bacchiani&author=I%20Tedeschi&author=CA%20Meroni&volume=131&publication_year=2015&pages=1981-8&pmid=25948538&doi=10.1161/CIRCULATIONAHA.114.013777&)

2. Swain SM, Whaley FS and Ewer MS. Congestive heart failure in patients treated with doxorubicin: A retrospective analysis of three trials. Cancer. 2003;97:2869–2879.  [DOI](https://doi.org/10.1002/cncr.11407) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12767102/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Congestive%20heart%20failure%20in%20patients%20treated%20with%20doxorubicin:%20A%20retrospective%20analysis%20of%20three%20trials&author=SM%20Swain&author=FS%20Whaley&author=MS%20Ewer&volume=97&publication_year=2003&pages=2869-2879&pmid=12767102&doi=10.1002/cncr.11407&)

3. Avila MS, Ayub-Ferreira SM, de Barros Wanderley MR Jr., das Dores Cruz F, Goncalves Brandao SM, Rigaud VOC, Higuchi-Dos-Santos MH, Hajjar LA, Kalil Filho R, Hoff PM, et al. Carvedilol for Prevention of Chemotherapy-Related Cardiotoxicity: The CECCY Trial. J Am Coll Cardiol. 2018;71:2281–2290.  [DOI](https://doi.org/10.1016/j.jacc.2018.02.049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29540327/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Coll%20Cardiol&title=Carvedilol%20for%20Prevention%20of%20Chemotherapy-Related%20Cardiotoxicity:%20The%20CECCY%20Trial&author=MS%20Avila&author=SM%20Ayub-Ferreira&author=MR%20de%20Barros%20Wanderley&author=F%20das%20Dores%20Cruz&author=SM%20Goncalves%20Brandao&volume=71&publication_year=2018&pages=2281-2290&pmid=29540327&doi=10.1016/j.jacc.2018.02.049&)

4. Magdy T, Burmeister BT and Burridge PW. Validating the pharmacogenomics of chemotherapy-induced cardiotoxicity: What is missing? Pharmacol Ther. 2016.  [DOI](https://doi.org/10.1016/j.pharmthera.2016.09.009) | [PMC free article](/articles/PMC5140687/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27609196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Ther&title=Validating%20the%20pharmacogenomics%20of%20chemotherapy-induced%20cardiotoxicity:%20What%20is%20missing?&author=T%20Magdy&author=BT%20Burmeister&author=PW%20Burridge&publication_year=2016&pmid=27609196&doi=10.1016/j.pharmthera.2016.09.009&)

5. Aminkeng F, Ross CJD, Rassekh SR, Rieder MJ, Bhavsar AP, Sanatani S, Bernstein D, Hayden MR, Amstutz U and Carleton BC. Pharmacogenomic screening for anthracycline-induced cardiotoxicity in childhood cancer. Br J Clin Pharmacol. 2017;83:1143–1145.  [DOI](https://doi.org/10.1111/bcp.13218) | [PMC free article](/articles/PMC5401971/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28317142/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Pharmacogenomic%20screening%20for%20anthracycline-induced%20cardiotoxicity%20in%20childhood%20cancer&author=F%20Aminkeng&author=CJD%20Ross&author=SR%20Rassekh&author=MJ%20Rieder&author=AP%20Bhavsar&volume=83&publication_year=2017&pages=1143-1145&pmid=28317142&doi=10.1111/bcp.13218&)

6. Tan LL and Lyon AR. Role of Biomarkers in Prediction of Cardiotoxicity During Cancer Treatment. Curr Treat Options Cardiovasc Med. 2018;20:55.  [DOI](https://doi.org/10.1007/s11936-018-0641-z) | [PMC free article](/articles/PMC6008350/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29923056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Treat%20Options%20Cardiovasc%20Med&title=Role%20of%20Biomarkers%20in%20Prediction%20of%20Cardiotoxicity%20During%20Cancer%20Treatment&author=LL%20Tan&author=AR%20Lyon&volume=20&publication_year=2018&pages=55&pmid=29923056&doi=10.1007/s11936-018-0641-z&)

7. Visscher H, Ross CJD, Rassekh SR, Barhdadi A, Dubé M-P, Al-Saloos H, Sandor GS, Caron HN, van Dalen EC, Kremer LC, et al. Pharmacogenomic prediction of anthracycline-induced cardiotoxicity in children. J Clin Oncol. 2012;30:1422–1428.  [DOI](https://doi.org/10.1200/JCO.2010.34.3467) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21900104/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Pharmacogenomic%20prediction%20of%20anthracycline-induced%20cardiotoxicity%20in%20children&author=H%20Visscher&author=CJD%20Ross&author=SR%20Rassekh&author=A%20Barhdadi&author=M-P%20Dub%C3%A9&volume=30&publication_year=2012&pages=1422-1428&pmid=21900104&doi=10.1200/JCO.2010.34.3467&)

8. Visscher H, Ross CJD, Rassekh SR, Sandor GSS, Caron HN, van Dalen EC, Kremer LC, van der Pal HJ, Rogers PC, Rieder MJ, et al. Validation of variants in SLC28A3 and UGT1A6 as genetic markers predictive of anthracycline-induced cardiotoxicity in children. Pediatr Blood Cancer. 2013;60:1375–1381.  [DOI](https://doi.org/10.1002/pbc.24505) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23441093/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr%20Blood%20Cancer&title=Validation%20of%20variants%20in%20SLC28A3%20and%20UGT1A6%20as%20genetic%20markers%20predictive%20of%20anthracycline-induced%20cardiotoxicity%20in%20children&author=H%20Visscher&author=CJD%20Ross&author=SR%20Rassekh&author=GSS%20Sandor&author=HN%20Caron&volume=60&publication_year=2013&pages=1375-1381&pmid=23441093&doi=10.1002/pbc.24505&)

9. Aminkeng F, Ross CJD, Rassekh SR, Hwang S, Rieder MJ, Bhavsar AP, Smith A, Sanatani S, Gelmon KA, Bernstein D, et al. Recommendations for genetic testing to reduce the incidence of anthracycline-induced cardiotoxicity. Br J Clin Pharmacol. 2016;82:683–695.  [DOI](https://doi.org/10.1111/bcp.13008) | [PMC free article](/articles/PMC5338111/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27197003/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Recommendations%20for%20genetic%20testing%20to%20reduce%20the%20incidence%20of%20anthracycline-induced%20cardiotoxicity&author=F%20Aminkeng&author=CJD%20Ross&author=SR%20Rassekh&author=S%20Hwang&author=MJ%20Rieder&volume=82&publication_year=2016&pages=683-695&pmid=27197003&doi=10.1111/bcp.13008&)

10. Fusaki N, Ban H, Nishiyama A, Saeki K and Hasegawa M. Efficient induction of transgene-free human pluripotent stem cells using a vector based on Sendai virus, an RNA virus that does not integrate into the host genome. Proc Jpn Acad Ser B Phys Biol Sci. 2009;85:348–62.  [DOI](https://doi.org/10.2183/pjab.85.348) | [PMC free article](/articles/PMC3621571/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19838014/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Jpn%20Acad%20Ser%20B%20Phys%20Biol%20Sci&title=Efficient%20induction%20of%20transgene-free%20human%20pluripotent%20stem%20cells%20using%20a%20vector%20based%20on%20Sendai%20virus,%20an%20RNA%20virus%20that%20does%20not%20integrate%20into%20the%20host%20genome&author=N%20Fusaki&author=H%20Ban&author=A%20Nishiyama&author=K%20Saeki&author=M%20Hasegawa&volume=85&publication_year=2009&pages=348-62&pmid=19838014&doi=10.2183/pjab.85.348&)

11. Burridge PW, Matsa E, Shukla P, Lin ZC, Churko JM, Ebert AD, Lan F, Diecke S, Huber B, Mordwinkin NM, et al. Chemically defined generation of human cardiomyocytes. Nat Methods. 2014;11:855–60.  [DOI](https://doi.org/10.1038/nmeth.2999) | [PMC free article](/articles/PMC4169698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24930130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=Chemically%20defined%20generation%20of%20human%20cardiomyocytes&author=PW%20Burridge&author=E%20Matsa&author=P%20Shukla&author=ZC%20Lin&author=JM%20Churko&volume=11&publication_year=2014&pages=855-60&pmid=24930130&doi=10.1038/nmeth.2999&)

12. Burridge PW, Holmstrom A and Wu JC. Chemically Defined Culture and Cardiomyocyte Differentiation of Human Pluripotent Stem Cells. Curr Protoc Hum Genet. 2015;87:21 3 1–15.  [DOI](https://doi.org/10.1002/0471142905.hg2103s87) | [PMC free article](/articles/PMC4597313/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26439715/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Protoc%20Hum%20Genet&title=Chemically%20Defined%20Culture%20and%20Cardiomyocyte%20Differentiation%20of%20Human%20Pluripotent%20Stem%20Cells&author=PW%20Burridge&author=A%20Holmstrom&author=JC%20Wu&volume=87&publication_year=2015&pages=21%203%201-15&pmid=26439715&doi=10.1002/0471142905.hg2103s87&)

13. Ran FA, Hsu PD, Wright J, Agarwala V, Scott DA and Zhang F. Genome engineering using the CRISPR-Cas9 system. Nat Protoc. 2013;8:2281–308.  [DOI](https://doi.org/10.1038/nprot.2013.143) | [PMC free article](/articles/PMC3969860/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24157548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Genome%20engineering%20using%20the%20CRISPR-Cas9%20system&author=FA%20Ran&author=PD%20Hsu&author=J%20Wright&author=V%20Agarwala&author=DA%20Scott&volume=8&publication_year=2013&pages=2281-308&pmid=24157548&doi=10.1038/nprot.2013.143&)

14. Oceguera-Yanez F, Kim SI, Matsumoto T, Tan GW, Xiang L, Hatani T, Kondo T, Ikeya M, Yoshida Y, Inoue H, et al. Engineering the AAVS1 locus for consistent and scalable transgene expression in human iPSCs and their differentiated derivatives. Methods. 2016;101:43–55.  [DOI](https://doi.org/10.1016/j.ymeth.2015.12.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26707206/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods&title=Engineering%20the%20AAVS1%20locus%20for%20consistent%20and%20scalable%20transgene%20expression%20in%20human%20iPSCs%20and%20their%20differentiated%20derivatives&author=F%20Oceguera-Yanez&author=SI%20Kim&author=T%20Matsumoto&author=GW%20Tan&author=L%20Xiang&volume=101&publication_year=2016&pages=43-55&pmid=26707206&doi=10.1016/j.ymeth.2015.12.012&)

15. Dandage R, Despres PC, Yachie N and Landry CR. beditor: A Computational Workflow for Designing Libraries of Guide RNAs for CRISPR-Mediated Base Editing. Genetics. 2019.  [DOI](https://doi.org/10.1534/genetics.119.302089) | [PMC free article](/articles/PMC6553823/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30936113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics&title=beditor:%20A%20Computational%20Workflow%20for%20Designing%20Libraries%20of%20Guide%20RNAs%20for%20CRISPR-Mediated%20Base%20Editing&author=R%20Dandage&author=PC%20Despres&author=N%20Yachie&author=CR%20Landry&publication_year=2019&pmid=30936113&doi=10.1534/genetics.119.302089&)

16. Ritz C, Baty F, Streibig JC and Gerhard D. Dose-Response Analysis Using R. PLoS ONE. 2015;10:e0146021.  [DOI](https://doi.org/10.1371/journal.pone.0146021) | [PMC free article](/articles/PMC4696819/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26717316/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Dose-Response%20Analysis%20Using%20R&author=C%20Ritz&author=F%20Baty&author=JC%20Streibig&author=D%20Gerhard&volume=10&publication_year=2015&pages=e0146021&pmid=26717316&doi=10.1371/journal.pone.0146021&)

17. Aminkeng F, Bhavsar AP, Visscher H, Rassekh SR, Li Y, Lee JW, Brunham LR, Caron HN, van Dalen EC, Kremer LC, et al. A coding variant in RARG confers susceptibility to anthracycline-induced cardiotoxicity in childhood cancer. Nat Genet. 2015;47:1079–84.  [DOI](https://doi.org/10.1038/ng.3374) | [PMC free article](/articles/PMC4552570/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26237429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20coding%20variant%20in%20RARG%20confers%20susceptibility%20to%20anthracycline-induced%20cardiotoxicity%20in%20childhood%20cancer&author=F%20Aminkeng&author=AP%20Bhavsar&author=H%20Visscher&author=SR%20Rassekh&author=Y%20Li&volume=47&publication_year=2015&pages=1079-84&pmid=26237429&doi=10.1038/ng.3374&)

18. Churko JM, Burridge PW and Wu JC. Generation of human iPSCs from human peripheral blood mononuclear cells using non-integrative Sendai virus in chemically defined conditions. Methods Mol Biol. 2013;1036:81–8.  [DOI](https://doi.org/10.1007/978-1-62703-511-8_7) | [PMC free article](/articles/PMC6430631/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23807788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&title=Generation%20of%20human%20iPSCs%20from%20human%20peripheral%20blood%20mononuclear%20cells%20using%20non-integrative%20Sendai%20virus%20in%20chemically%20defined%20conditions&author=JM%20Churko&author=PW%20Burridge&author=JC%20Wu&volume=1036&publication_year=2013&pages=81-8&pmid=23807788&doi=10.1007/978-1-62703-511-8_7&)

19. Diecke S, Lu J, Lee J, Termglinchan V, Kooreman NG, Burridge PW, Ebert AD, Churko JM, Sharma A, Kay MA, et al. Novel codon-optimized mini-intronic plasmid for efficient, inexpensive, and xeno-free induction of pluripotency. Sci Rep. 2015;5:8081.  [DOI](https://doi.org/10.1038/srep08081) | [PMC free article](/articles/PMC4308704/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25628230/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Novel%20codon-optimized%20mini-intronic%20plasmid%20for%20efficient,%20inexpensive,%20and%20xeno-free%20induction%20of%20pluripotency&author=S%20Diecke&author=J%20Lu&author=J%20Lee&author=V%20Termglinchan&author=NG%20Kooreman&volume=5&publication_year=2015&pages=8081&pmid=25628230&doi=10.1038/srep08081&)

20. Burridge PW, Holmstrom A and Wu JC. Chemically Defined Culture and Cardiomyocyte Differentiation of Human Pluripotent Stem Cells. Curr Protoc Hum Genet. 2015;87:21 3 1–21 3 15.  [DOI](https://doi.org/10.1002/0471142905.hg2103s87) | [PMC free article](/articles/PMC4597313/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26439715/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Protoc%20Hum%20Genet&title=Chemically%20Defined%20Culture%20and%20Cardiomyocyte%20Differentiation%20of%20Human%20Pluripotent%20Stem%20Cells&author=PW%20Burridge&author=A%20Holmstrom&author=JC%20Wu&volume=87&publication_year=2015&pages=21%203%201-21%203%2015&pmid=26439715&doi=10.1002/0471142905.hg2103s87&)

21. Magdy T and Burridge PW. The future role of pharmacogenomics in anticancer agent-induced cardiovascular toxicity. Pharmacogenomics. 2018;19:79–82.  [DOI](https://doi.org/10.2217/pgs-2017-0177) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29199515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=The%20future%20role%20of%20pharmacogenomics%20in%20anticancer%20agent-induced%20cardiovascular%20toxicity&author=T%20Magdy&author=PW%20Burridge&volume=19&publication_year=2018&pages=79-82&pmid=29199515&doi=10.2217/pgs-2017-0177&)

22. Loman NJ, Quick J and Simpson JT. A complete bacterial genome assembled de novo using only nanopore sequencing data. Nat Methods. 2015;12:733–5.  [DOI](https://doi.org/10.1038/nmeth.3444) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26076426/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=A%20complete%20bacterial%20genome%20assembled%20de%20novo%20using%20only%20nanopore%20sequencing%20data&author=NJ%20Loman&author=J%20Quick&author=JT%20Simpson&volume=12&publication_year=2015&pages=733-5&pmid=26076426&doi=10.1038/nmeth.3444&)

23. Kundaje A, Meuleman W, Ernst J, Bilenky M, Yen A, Heravi-Moussavi A, Kheradpour P, Zhang Z, Wang J, Ziller MJ, et al. Integrative analysis of 111 reference human epigenomes. Nature. 2015;518:317–30.  [DOI](https://doi.org/10.1038/nature14248) | [PMC free article](/articles/PMC4530010/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25693563/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Integrative%20analysis%20of%20111%20reference%20human%20epigenomes&author=A%20Kundaje&author=W%20Meuleman&author=J%20Ernst&author=M%20Bilenky&author=A%20Yen&volume=518&publication_year=2015&pages=317-30&pmid=25693563&doi=10.1038/nature14248&)

24. Zhou J and Troyanskaya OG. Predicting effects of noncoding variants with deep learning-based sequence model. Nat Methods. 2015;12:931–4.  [DOI](https://doi.org/10.1038/nmeth.3547) | [PMC free article](/articles/PMC4768299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26301843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&title=Predicting%20effects%20of%20noncoding%20variants%20with%20deep%20learning-based%20sequence%20model&author=J%20Zhou&author=OG%20Troyanskaya&volume=12&publication_year=2015&pages=931-4&pmid=26301843&doi=10.1038/nmeth.3547&)

25. Weisheit I, Kroeger JA, Malik R, Klimmt J, Crusius D, Dannert A, Dichgans M and Paquet D. Detection of Deleterious On-Target Effects after HDR-Mediated CRISPR Editing. Cell Rep. 2020;31:107689.  [DOI](https://doi.org/10.1016/j.celrep.2020.107689) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32460021/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&title=Detection%20of%20Deleterious%20On-Target%20Effects%20after%20HDR-Mediated%20CRISPR%20Editing&author=I%20Weisheit&author=JA%20Kroeger&author=R%20Malik&author=J%20Klimmt&author=D%20Crusius&volume=31&publication_year=2020&pages=107689&pmid=32460021&doi=10.1016/j.celrep.2020.107689&)

26. Magdy T, Burmeister BT and Burridge PW. Validating the pharmacogenomics of chemotherapy-induced cardiotoxicity: What is missing? Pharmacol Ther. 2016;168:113–125.  [DOI](https://doi.org/10.1016/j.pharmthera.2016.09.009) | [PMC free article](/articles/PMC5140687/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27609196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Ther&title=Validating%20the%20pharmacogenomics%20of%20chemotherapy-induced%20cardiotoxicity:%20What%20is%20missing?&author=T%20Magdy&author=BT%20Burmeister&author=PW%20Burridge&volume=168&publication_year=2016&pages=113-125&pmid=27609196&doi=10.1016/j.pharmthera.2016.09.009&)

27. Maurano MT, Humbert R, Rynes E, Thurman RE, Haugen E, Wang H, Reynolds AP, Sandstrom R, Qu H, Brody J, et al. Systematic localization of common disease-associated variation in regulatory DNA. Science. 2012;337:1190–5.  [DOI](https://doi.org/10.1126/science.1222794) | [PMC free article](/articles/PMC3771521/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Systematic%20localization%20of%20common%20disease-associated%20variation%20in%20regulatory%20DNA&author=MT%20Maurano&author=R%20Humbert&author=E%20Rynes&author=RE%20Thurman&author=E%20Haugen&volume=337&publication_year=2012&pages=1190-5&pmid=22955828&doi=10.1126/science.1222794&)

28. Schaub MA, Boyle AP, Kundaje A, Batzoglou S and Snyder M. Linking disease associations with regulatory information in the human genome. Genome Res. 2012;22:1748–59.  [DOI](https://doi.org/10.1101/gr.136127.111) | [PMC free article](/articles/PMC3431491/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955986/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res&title=Linking%20disease%20associations%20with%20regulatory%20information%20in%20the%20human%20genome&author=MA%20Schaub&author=AP%20Boyle&author=A%20Kundaje&author=S%20Batzoglou&author=M%20Snyder&volume=22&publication_year=2012&pages=1748-59&pmid=22955986&doi=10.1101/gr.136127.111&)

29. Magdy T, Kuo HH and Burridge PW. Precise and Cost-Effective Nanopore Sequencing for Post-GWAS Fine-Mapping and Causal Variant Identification. iScience. 2020;23:100971.  [DOI](https://doi.org/10.1016/j.isci.2020.100971) | [PMC free article](/articles/PMC7096756/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32203907/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=iScience&title=Precise%20and%20Cost-Effective%20Nanopore%20Sequencing%20for%20Post-GWAS%20Fine-Mapping%20and%20Causal%20Variant%20Identification&author=T%20Magdy&author=HH%20Kuo&author=PW%20Burridge&volume=23&publication_year=2020&pages=100971&pmid=32203907&doi=10.1016/j.isci.2020.100971&)

30. Giacomini KM, Huang SM, Tweedie DJ, Benet LZ, Brouwer KL, Chu X, Dahlin A, Evers R, Fischer V, Hillgren KM, et al. Membrane transporters in drug development. Nat Rev Drug Discov. 2010;9:215–36.  [DOI](https://doi.org/10.1038/nrd3028) | [PMC free article](/articles/PMC3326076/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20190787/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Drug%20Discov&title=Membrane%20transporters%20in%20drug%20development&author=KM%20Giacomini&author=SM%20Huang&author=DJ%20Tweedie&author=LZ%20Benet&author=KL%20Brouwer&volume=9&publication_year=2010&pages=215-36&pmid=20190787&doi=10.1038/nrd3028&)

31. Andersen J, Kristensen AS, Bang-Andersen B and Stromgaard K. Recent advances in the understanding of the interaction of antidepressant drugs with serotonin and norepinephrine transporters. Chem Commun (Camb). 2009:3677–92.  [DOI](https://doi.org/10.1039/b903035m) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19557250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem%20Commun%20(Camb)&title=Recent%20advances%20in%20the%20understanding%20of%20the%20interaction%20of%20antidepressant%20drugs%20with%20serotonin%20and%20norepinephrine%20transporters&author=J%20Andersen&author=AS%20Kristensen&author=B%20Bang-Andersen&author=K%20Stromgaard&publication_year=2009&pages=3677-92&pmid=19557250&doi=10.1039/b903035m&)

32. Arrowsmith J and Miller P. Trial watch: phase II and phase III attrition rates 2011–2012. Nat Rev Drug Discov. 2013;12:569.  [DOI](https://doi.org/10.1038/nrd4090) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23903212/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Drug%20Discov&title=Trial%20watch:%20phase%20II%20and%20phase%20III%20attrition%20rates%202011%E2%80%932012&author=J%20Arrowsmith&author=P%20Miller&volume=12&publication_year=2013&pages=569&pmid=23903212&doi=10.1038/nrd4090&)

33. Nelson MR, Tipney H, Painter JL, Shen J, Nicoletti P, Shen Y, Floratos A, Sham PC, Li MJ, Wang J, et al. The support of human genetic evidence for approved drug indications. Nat Genet. 2015;47:856–60.  [DOI](https://doi.org/10.1038/ng.3314) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26121088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20support%20of%20human%20genetic%20evidence%20for%20approved%20drug%20indications&author=MR%20Nelson&author=H%20Tipney&author=JL%20Painter&author=J%20Shen&author=P%20Nicoletti&volume=47&publication_year=2015&pages=856-60&pmid=26121088&doi=10.1038/ng.3314&)

34. Itzhaki I, Maizels L, Huber I, Zwi-Dantsis L, Caspi O, Winterstern A, Feldman O, Gepstein A, Arbel G, Hammerman H, et al. Modelling the long QT syndrome with induced pluripotent stem cells. Nature. 2011;471:225–229.  [DOI](https://doi.org/10.1038/nature09747) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21240260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Modelling%20the%20long%20QT%20syndrome%20with%20induced%20pluripotent%20stem%20cells&author=I%20Itzhaki&author=L%20Maizels&author=I%20Huber&author=L%20Zwi-Dantsis&author=O%20Caspi&volume=471&publication_year=2011&pages=225-229&pmid=21240260&doi=10.1038/nature09747&)

35. Malan D, Zhang M, Stallmeyer B, Müller J, Fleischmann BK, Schulze-Bahr E, Sasse P and Greber B. Human iPS cell model of type 3 long QT syndrome recapitulates drug-based phenotype correction. Basic Res Cardiol. 2016;111.  [DOI](https://doi.org/10.1007/s00395-016-0530-0) | [PMC free article](/articles/PMC4724360/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26803770/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Res%20Cardiol&title=Human%20iPS%20cell%20model%20of%20type%203%20long%20QT%20syndrome%20recapitulates%20drug-based%20phenotype%20correction&author=D%20Malan&author=M%20Zhang&author=B%20Stallmeyer&author=J%20M%C3%BCller&author=BK%20Fleischmann&volume=111&publication_year=2016&pmid=26803770&doi=10.1007/s00395-016-0530-0&)

36. Carvajal-Vergara X, Sevilla A, D’Souza SL, Ang Y-S, Schaniel C, Lee D-F, Yang L, Kaplan AD, Adler ED, Rozov R, et al. Patient-specific induced pluripotent stem cell derived models of LEOPARD syndrome. Nature. 2010;465:808–812.  [DOI](https://doi.org/10.1038/nature09005) | [PMC free article](/articles/PMC2885001/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20535210/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Patient-specific%20induced%20pluripotent%20stem%20cell%20derived%20models%20of%20LEOPARD%20syndrome&author=X%20Carvajal-Vergara&author=A%20Sevilla&author=SL%20D%E2%80%99Souza&author=Y-S%20Ang&author=C%20Schaniel&volume=465&publication_year=2010&pages=808-812&pmid=20535210&doi=10.1038/nature09005&)

37. Yazawa M, Hsueh B, Jia X, Pasca AM, Bernstein JA, Hallmayer J and Dolmetsch RE. Using iPS cells to investigate cardiac phenotypes in patients with Timothy Syndrome. Nature. 2011;471:230–234.  [DOI](https://doi.org/10.1038/nature09855) | [PMC free article](/articles/PMC3077925/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21307850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Using%20iPS%20cells%20to%20investigate%20cardiac%20phenotypes%20in%20patients%20with%20Timothy%20Syndrome&author=M%20Yazawa&author=B%20Hsueh&author=X%20Jia&author=AM%20Pasca&author=JA%20Bernstein&volume=471&publication_year=2011&pages=230-234&pmid=21307850&doi=10.1038/nature09855&)

38. Kim C, Wong J, Wen J, Wang S, Wang C, Spiering S, Kan NG, Forcales S, Puri PL, Leone TC, et al. Studying arrhythmogenic right ventricular dysplasia with patient-specific iPSCs. Nature. 2013;494:105–110.  [DOI](https://doi.org/10.1038/nature11799) | [PMC free article](/articles/PMC3753229/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23354045/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Studying%20arrhythmogenic%20right%20ventricular%20dysplasia%20with%20patient-specific%20iPSCs&author=C%20Kim&author=J%20Wong&author=J%20Wen&author=S%20Wang&author=C%20Wang&volume=494&publication_year=2013&pages=105-110&pmid=23354045&doi=10.1038/nature11799&)

39. Sun N, Yazawa M, Liu J, Han L, Sanchez-Freire V, Abilez OJ, Navarrete EG, Hu S, Wang L, Lee A, et al. Patient-specific induced pluripotent stem cells as a model for familial dilated cardiomyopathy. Sci Transl Med. 2012;4:130ra47.  [DOI](https://doi.org/10.1126/scitranslmed.3003552) | [PMC free article](/articles/PMC3657516/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22517884/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&title=Patient-specific%20induced%20pluripotent%20stem%20cells%20as%20a%20model%20for%20familial%20dilated%20cardiomyopathy&author=N%20Sun&author=M%20Yazawa&author=J%20Liu&author=L%20Han&author=V%20Sanchez-Freire&volume=4&publication_year=2012&pages=130ra47&pmid=22517884&doi=10.1126/scitranslmed.3003552&)

40. Wang G, McCain ML, Yang L, He A, Pasqualini FS, Agarwal A, Yuan H, Jiang D, Zhang D, Zangi L, et al. Modeling the mitochondrial cardiomyopathy of Barth syndrome with induced pluripotent stem cell and heart-on-chip technologies. Nat Med. 2014;20:616–623.  [DOI](https://doi.org/10.1038/nm.3545) | [PMC free article](/articles/PMC4172922/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24813252/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Med&title=Modeling%20the%20mitochondrial%20cardiomyopathy%20of%20Barth%20syndrome%20with%20induced%20pluripotent%20stem%20cell%20and%20heart-on-chip%20technologies&author=G%20Wang&author=ML%20McCain&author=L%20Yang&author=A%20He&author=FS%20Pasqualini&volume=20&publication_year=2014&pages=616-623&pmid=24813252&doi=10.1038/nm.3545&)

41. Magdy T and Burridge PW. Unraveling Difficult Answers: From Genotype to Phenotype in Coronary Artery Disease. Cell Stem Cell. 2019;24:203–205.  [DOI](https://doi.org/10.1016/j.stem.2019.01.008) | [PMC free article](/articles/PMC7375859/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30735646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Stem%20Cell&title=Unraveling%20Difficult%20Answers:%20From%20Genotype%20to%20Phenotype%20in%20Coronary%20Artery%20Disease&author=T%20Magdy&author=PW%20Burridge&volume=24&publication_year=2019&pages=203-205&pmid=30735646&doi=10.1016/j.stem.2019.01.008&)

42. Drawnel FM, Boccardo S, Prummer M, Delobel F, Graff A, Weber M, Gérard R, Badi L, Kam-Thong T, Bu L, et al. Disease modeling and phenotypic drug screening for diabetic cardiomyopathy using human induced pluripotent stem cells. Cell Rep. 2014;9:810–821.  [DOI](https://doi.org/10.1016/j.celrep.2014.09.055) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25437537/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&title=Disease%20modeling%20and%20phenotypic%20drug%20screening%20for%20diabetic%20cardiomyopathy%20using%20human%20induced%20pluripotent%20stem%20cells&author=FM%20Drawnel&author=S%20Boccardo&author=M%20Prummer&author=F%20Delobel&author=A%20Graff&volume=9&publication_year=2014&pages=810-821&pmid=25437537&doi=10.1016/j.celrep.2014.09.055&)

43. Chou BK, Gu H, Gao Y, Dowey SN, Wang Y, Shi J, Li Y, Ye Z, Cheng T and Cheng L. A facile method to establish human induced pluripotent stem cells from adult blood cells under feeder-free and xeno-free culture conditions: a clinically compliant approach. Stem Cells Transl Med. 2015;4:320–32.  [DOI](https://doi.org/10.5966/sctm.2014-0214) | [PMC free article](/articles/PMC4367508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25742692/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stem%20Cells%20Transl%20Med&title=A%20facile%20method%20to%20establish%20human%20induced%20pluripotent%20stem%20cells%20from%20adult%20blood%20cells%20under%20feeder-free%20and%20xeno-free%20culture%20conditions:%20a%20clinically%20compliant%20approach&author=BK%20Chou&author=H%20Gu&author=Y%20Gao&author=SN%20Dowey&author=Y%20Wang&volume=4&publication_year=2015&pages=320-32&pmid=25742692&doi=10.5966/sctm.2014-0214&)

44. Kuo HH, Gao X, DeKeyser JM, Fetterman KA, Pinheiro EA, Weddle CJ, Fonoudi H, Orman MV, Romero-Tejeda M, Jouni M, et al. Negligible-Cost and Weekend-Free Chemically Defined Human iPSC Culture. Stem Cell Reports. 2020;14:256–270.  [DOI](https://doi.org/10.1016/j.stemcr.2019.12.007) | [PMC free article](/articles/PMC7013200/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31928950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stem%20Cell%20Reports&title=Negligible-Cost%20and%20Weekend-Free%20Chemically%20Defined%20Human%20iPSC%20Culture&author=HH%20Kuo&author=X%20Gao&author=JM%20DeKeyser&author=KA%20Fetterman&author=EA%20Pinheiro&volume=14&publication_year=2020&pages=256-270&pmid=31928950&doi=10.1016/j.stemcr.2019.12.007&)

45. Kim D, Paggi JM, Park C, Bennett C and Salzberg SL. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Nat Biotechnol. 2019;37:907–915.  [DOI](https://doi.org/10.1038/s41587-019-0201-4) | [PMC free article](/articles/PMC7605509/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31375807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=Graph-based%20genome%20alignment%20and%20genotyping%20with%20HISAT2%20and%20HISAT-genotype&author=D%20Kim&author=JM%20Paggi&author=C%20Park&author=C%20Bennett&author=SL%20Salzberg&volume=37&publication_year=2019&pages=907-915&pmid=31375807&doi=10.1038/s41587-019-0201-4&)

46. Liao Y, Smyth GK and Shi W. The Subread aligner: fast, accurate and scalable read mapping by seed-and-vote. Nucleic Acids Res. 2013;41:e108.  [DOI](https://doi.org/10.1093/nar/gkt214) | [PMC free article](/articles/PMC3664803/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23558742/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20Subread%20aligner:%20fast,%20accurate%20and%20scalable%20read%20mapping%20by%20seed-and-vote&author=Y%20Liao&author=GK%20Smyth&author=W%20Shi&volume=41&publication_year=2013&pages=e108&pmid=23558742&doi=10.1093/nar/gkt214&)

47. Love MI, Huber W and Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 2014;15:550.  [DOI](https://doi.org/10.1186/s13059-014-0550-8) | [PMC free article](/articles/PMC4302049/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25516281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&author=MI%20Love&author=W%20Huber&author=S%20Anders&volume=15&publication_year=2014&pages=550&pmid=25516281&doi=10.1186/s13059-014-0550-8&)

48. Schmittgen TD and Livak KJ. Analyzing real-time PCR data by the comparative C(T) method. Nat Protoc. 2008;3:1101–8.  [DOI](https://doi.org/10.1038/nprot.2008.73) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18546601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Analyzing%20real-time%20PCR%20data%20by%20the%20comparative%20C(T)%20method&author=TD%20Schmittgen&author=KJ%20Livak&volume=3&publication_year=2008&pages=1101-8&pmid=18546601&doi=10.1038/nprot.2008.73&)

49. Wick RR, Judd LM, Gorrie CL and Holt KE. Completing bacterial genome assemblies with multiplex MinION sequencing. Microbial genomics. 2017;3:e000132.  [DOI](https://doi.org/10.1099/mgen.0.000132) | [PMC free article](/articles/PMC5695209/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29177090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microbial%20genomics&title=Completing%20bacterial%20genome%20assemblies%20with%20multiplex%20MinION%20sequencing&author=RR%20Wick&author=LM%20Judd&author=CL%20Gorrie&author=KE%20Holt&volume=3&publication_year=2017&pages=e000132&pmid=29177090&doi=10.1099/mgen.0.000132&)

50. De Coster W, D'Hert S, Schultz DT, Cruts M and Van Broeckhoven C. NanoPack: visualizing and processing long-read sequencing data. Bioinformatics. 2018;34:2666–2669.  [DOI](https://doi.org/10.1093/bioinformatics/bty149) | [PMC free article](/articles/PMC6061794/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29547981/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=NanoPack:%20visualizing%20and%20processing%20long-read%20sequencing%20data&author=W%20De%20Coster&author=S%20D'Hert&author=DT%20Schultz&author=M%20Cruts&author=C%20Van%20Broeckhoven&volume=34&publication_year=2018&pages=2666-2669&pmid=29547981&doi=10.1093/bioinformatics/bty149&)

51. Li H Minimap2: pairwise alignment for nucleotide sequences. Bioinformatics. 2018;34:3094–3100.  [DOI](https://doi.org/10.1093/bioinformatics/bty191) | [PMC free article](/articles/PMC6137996/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29750242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Minimap2:%20pairwise%20alignment%20for%20nucleotide%20sequences&author=H%20Li&volume=34&publication_year=2018&pages=3094-3100&pmid=29750242&doi=10.1093/bioinformatics/bty191&)

52. Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G and Durbin R. The Sequence Alignment/Map format and SAMtools. Bioinformatics. 2009;25:2078–9.  [DOI](https://doi.org/10.1093/bioinformatics/btp352) | [PMC free article](/articles/PMC2723002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19505943/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=The%20Sequence%20Alignment/Map%20format%20and%20SAMtools&author=H%20Li&author=B%20Handsaker&author=A%20Wysoker&author=T%20Fennell&author=J%20Ruan&volume=25&publication_year=2009&pages=2078-9&pmid=19505943&doi=10.1093/bioinformatics/btp352&)

53. Ramirez F, Ryan DP, Gruning B, Bhardwaj V, Kilpert F, Richter AS, Heyne S, Dundar F and Manke T. deepTools2: a next generation web server for deep-sequencing data analysis. Nucleic Acids Res. 2016;44:W160–5.  [DOI](https://doi.org/10.1093/nar/gkw257) | [PMC free article](/articles/PMC4987876/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27079975/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=deepTools2:%20a%20next%20generation%20web%20server%20for%20deep-sequencing%20data%20analysis&author=F%20Ramirez&author=DP%20Ryan&author=B%20Gruning&author=V%20Bhardwaj&author=F%20Kilpert&volume=44&publication_year=2016&pages=W160-5&pmid=27079975&doi=10.1093/nar/gkw257&)

54. Danecek P, Auton A, Abecasis G, Albers CA, Banks E, DePristo MA, Handsaker RE, Lunter G, Marth GT, Sherry ST, et al. The variant call format and VCFtools. Bioinformatics. 2011;27:2156–8.  [DOI](https://doi.org/10.1093/bioinformatics/btr330) | [PMC free article](/articles/PMC3137218/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21653522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=The%20variant%20call%20format%20and%20VCFtools&author=P%20Danecek&author=A%20Auton&author=G%20Abecasis&author=CA%20Albers&author=E%20Banks&volume=27&publication_year=2011&pages=2156-8&pmid=21653522&doi=10.1093/bioinformatics/btr330&)

55. Cingolani P, Patel VM, Coon M, Nguyen T, Land SJ, Ruden DM and Lu X. Using Drosophila melanogaster as a Model for Genotoxic Chemical Mutational Studies with a New Program, SnpSift. Front Genet. 2012;3:35.  [DOI](https://doi.org/10.3389/fgene.2012.00035) | [PMC free article](/articles/PMC3304048/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22435069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Genet&title=Using%20Drosophila%20melanogaster%20as%20a%20Model%20for%20Genotoxic%20Chemical%20Mutational%20Studies%20with%20a%20New%20Program,%20SnpSift&author=P%20Cingolani&author=VM%20Patel&author=M%20Coon&author=T%20Nguyen&author=SJ%20Land&volume=3&publication_year=2012&pages=35&pmid=22435069&doi=10.3389/fgene.2012.00035&)

56. Narasimhan V, Danecek P, Scally A, Xue Y, Tyler-Smith C and Durbin R. BCFtools/RoH: a hidden Markov model approach for detecting autozygosity from next-generation sequencing data. Bioinformatics. 2016;32:1749–51.  [DOI](https://doi.org/10.1093/bioinformatics/btw044) | [PMC free article](/articles/PMC4892413/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26826718/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=BCFtools/RoH:%20a%20hidden%20Markov%20model%20approach%20for%20detecting%20autozygosity%20from%20next-generation%20sequencing%20data&author=V%20Narasimhan&author=P%20Danecek&author=A%20Scally&author=Y%20Xue&author=C%20Tyler-Smith&volume=32&publication_year=2016&pages=1749-51&pmid=26826718&doi=10.1093/bioinformatics/btw044&)

57. Durinck S, Spellman PT, Birney E and Huber W. Mapping identifiers for the integration of genomic datasets with the R/Bioconductor package biomaRt. Nat Protoc. 2009;4:1184–1191.  [DOI](https://doi.org/10.1038/nprot.2009.97) | [PMC free article](/articles/PMC3159387/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19617889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Mapping%20identifiers%20for%20the%20integration%20of%20genomic%20datasets%20with%20the%20R/Bioconductor%20package%20biomaRt&author=S%20Durinck&author=PT%20Spellman&author=E%20Birney&author=W%20Huber&volume=4&publication_year=2009&pages=1184-1191&pmid=19617889&doi=10.1038/nprot.2009.97&)

58. Braun D, Kim TD, le Coutre P, Köhrle J, Hershman JM and Schweizer U. Tyrosine kinase inhibitors noncompetitively inhibit MCT8-mediated iodothyronine transport. The Journal of clinical endocrinology and metabolism. 2012;97:E100–5.  [DOI](https://doi.org/10.1210/jc.2011-1837) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22031512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20clinical%20endocrinology%20and%20metabolism&title=Tyrosine%20kinase%20inhibitors%20noncompetitively%20inhibit%20MCT8-mediated%20iodothyronine%20transport&author=D%20Braun&author=TD%20Kim&author=P%20le%20Coutre&author=J%20K%C3%B6hrle&author=JM%20Hershman&volume=97&publication_year=2012&pages=E100-5&pmid=22031512&doi=10.1210/jc.2011-1837&)

59. Damaraju VL, Weber D, Kuzma M, Cass CE and Sawyer MB. Selective Inhibition of Human Equilibrative and Concentrative Nucleoside Transporters by BCR-ABL Kinase Inhibitors: IDENTIFICATION OF KEY hENT1 AMINO ACID RESIDUES FOR INTERACTION WITH BCR-ABL KINASE INHIBITORS. J Biol Chem. 2016;291:18809–17.  [DOI](https://doi.org/10.1074/jbc.M116.741074) | [PMC free article](/articles/PMC5009255/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27432881/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Selective%20Inhibition%20of%20Human%20Equilibrative%20and%20Concentrative%20Nucleoside%20Transporters%20by%20BCR-ABL%20Kinase%20Inhibitors:%20IDENTIFICATION%20OF%20KEY%20hENT1%20AMINO%20ACID%20RESIDUES%20FOR%20INTERACTION%20WITH%20BCR-ABL%20KINASE%20INHIBITORS&author=VL%20Damaraju&author=D%20Weber&author=M%20Kuzma&author=CE%20Cass&author=MB%20Sawyer&volume=291&publication_year=2016&pages=18809-17&pmid=27432881&doi=10.1074/jbc.M116.741074&)

60. Braun D and Schweizer U. Authentic bosutinib inhibits triiodothyronine transport by monocarboxylate transporter 8. Thyroid. 2014;24:926–7.  [DOI](https://doi.org/10.1089/thy.2013.0660) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24479416/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thyroid&title=Authentic%20bosutinib%20inhibits%20triiodothyronine%20transport%20by%20monocarboxylate%20transporter%208&author=D%20Braun&author=U%20Schweizer&volume=24&publication_year=2014&pages=926-7&pmid=24479416&doi=10.1089/thy.2013.0660&)

61. Yin J, Duan H and Wang J. Impact of Substrate-Dependent Inhibition on Renal Organic Cation Transporters hOCT2 and hMATE1/2-K-Mediated Drug Transport and Intracellular Accumulation. J Pharmacol Exp Ther. 2016;359:401–410.  [DOI](https://doi.org/10.1124/jpet.116.236158) | [PMC free article](/articles/PMC5118648/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27758931/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Impact%20of%20Substrate-Dependent%20Inhibition%20on%20Renal%20Organic%20Cation%20Transporters%20hOCT2%20and%20hMATE1/2-K-Mediated%20Drug%20Transport%20and%20Intracellular%20Accumulation&author=J%20Yin&author=H%20Duan&author=J%20Wang&volume=359&publication_year=2016&pages=401-410&pmid=27758931&doi=10.1124/jpet.116.236158&)

62. Tsuda M, Terada T, Ueba M, Sato T, Masuda S, Katsura T and Inui K. Involvement of human multidrug and toxin extrusion 1 in the drug interaction between cimetidine and metformin in renal epithelial cells. J Pharmacol Exp Ther. 2009;329:185–91.  [DOI](https://doi.org/10.1124/jpet.108.147918) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19164462/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Involvement%20of%20human%20multidrug%20and%20toxin%20extrusion%201%20in%20the%20drug%20interaction%20between%20cimetidine%20and%20metformin%20in%20renal%20epithelial%20cells&author=M%20Tsuda&author=T%20Terada&author=M%20Ueba&author=T%20Sato&author=S%20Masuda&volume=329&publication_year=2009&pages=185-91&pmid=19164462&doi=10.1124/jpet.108.147918&)

63. Shitara Y, Itoh T, Sato H, Li AP and Sugiyama Y. Inhibition of transporter-mediated hepatic uptake as a mechanism for drug-drug interaction between cerivastatin and cyclosporin A. J Pharmacol Exp Ther. 2003;304:610–6.  [DOI](https://doi.org/10.1124/jpet.102.041921) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12538813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Inhibition%20of%20transporter-mediated%20hepatic%20uptake%20as%20a%20mechanism%20for%20drug-drug%20interaction%20between%20cerivastatin%20and%20cyclosporin%20A&author=Y%20Shitara&author=T%20Itoh&author=H%20Sato&author=AP%20Li&author=Y%20Sugiyama&volume=304&publication_year=2003&pages=610-6&pmid=12538813&doi=10.1124/jpet.102.041921&)

64. Shitara Y, Takeuchi K, Nagamatsu Y, Wada S, Sugiyama Y and Horie T. Long-lasting inhibitory effects of cyclosporin A, but not tacrolimus, on OATP1B1- and OATP1B3-mediated uptake. Drug Metab Pharmacokinet. 2012;27:368–78.  [DOI](https://doi.org/10.2133/dmpk.dmpk-11-rg-096) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22240838/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Long-lasting%20inhibitory%20effects%20of%20cyclosporin%20A,%20but%20not%20tacrolimus,%20on%20OATP1B1-%20and%20OATP1B3-mediated%20uptake&author=Y%20Shitara&author=K%20Takeuchi&author=Y%20Nagamatsu&author=S%20Wada&author=Y%20Sugiyama&volume=27&publication_year=2012&pages=368-78&pmid=22240838&doi=10.2133/dmpk.dmpk-11-rg-096&)

65. Craddock AL, Love MW, Daniel RW, Kirby LC, Walters HC, Wong MH and Dawson PA. Expression and transport properties of the human ileal and renal sodium-dependent bile acid transporter. The American journal of physiology. 1998;274:G157–69.  [DOI](https://doi.org/10.1152/ajpgi.1998.274.1.G157) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9458785/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20American%20journal%20of%20physiology&title=Expression%20and%20transport%20properties%20of%20the%20human%20ileal%20and%20renal%20sodium-dependent%20bile%20acid%20transporter&author=AL%20Craddock&author=MW%20Love&author=RW%20Daniel&author=LC%20Kirby&author=HC%20Walters&volume=274&publication_year=1998&pages=G157-69&pmid=9458785&doi=10.1152/ajpgi.1998.274.1.G157&)

66. Schroeder A, Eckhardt U, Stieger B, Tynes R, Schteingart CD, Hofmann AF, Meier PJ and Hagenbuch B. Substrate specificity of the rat liver Na(+)-bile salt cotransporter in Xenopus laevis oocytes and in CHO cells. The American journal of physiology. 1998;274:G370–5.  [DOI](https://doi.org/10.1152/ajpgi.1998.274.2.G370) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9486191/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20American%20journal%20of%20physiology&title=Substrate%20specificity%20of%20the%20rat%20liver%20Na(+)-bile%20salt%20cotransporter%20in%20Xenopus%20laevis%20oocytes%20and%20in%20CHO%20cells&author=A%20Schroeder&author=U%20Eckhardt&author=B%20Stieger&author=R%20Tynes&author=CD%20Schteingart&volume=274&publication_year=1998&pages=G370-5&pmid=9486191&doi=10.1152/ajpgi.1998.274.2.G370&)

67. Taguchi T, Masuo Y, Kogi T, Nakamichi N and Kato Y. Characterization of Long-Lasting Oatp Inhibition by Typical Inhibitor Cyclosporine A and In Vitro-In Vivo Discrepancy in Its Drug Interaction Potential in Rats. J Pharm Sci. 2016;105:2231–9.  [DOI](https://doi.org/10.1016/j.xphs.2016.04.025) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27290622/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharm%20Sci&title=Characterization%20of%20Long-Lasting%20Oatp%20Inhibition%20by%20Typical%20Inhibitor%20Cyclosporine%20A%20and%20In%20Vitro-In%20Vivo%20Discrepancy%20in%20Its%20Drug%20Interaction%20Potential%20in%20Rats&author=T%20Taguchi&author=Y%20Masuo&author=T%20Kogi&author=N%20Nakamichi&author=Y%20Kato&volume=105&publication_year=2016&pages=2231-9&pmid=27290622&doi=10.1016/j.xphs.2016.04.025&)

68. Pahwa S, Alam K, Crowe A, Farasyn T, Neuhoff S, Hatley O, Ding K and Yue W. Pretreatment With Rifampicin and Tyrosine Kinase Inhibitor Dasatinib Potentiates the Inhibitory Effects Toward OATP1B1- and OATP1B3-Mediated Transport. J Pharm Sci. 2017;106:2123–2135.  [DOI](https://doi.org/10.1016/j.xphs.2017.03.022) | [PMC free article](/articles/PMC5511785/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28373111/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharm%20Sci&title=Pretreatment%20With%20Rifampicin%20and%20Tyrosine%20Kinase%20Inhibitor%20Dasatinib%20Potentiates%20the%20Inhibitory%20Effects%20Toward%20OATP1B1-%20and%20OATP1B3-Mediated%20Transport&author=S%20Pahwa&author=K%20Alam&author=A%20Crowe&author=T%20Farasyn&author=S%20Neuhoff&volume=106&publication_year=2017&pages=2123-2135&pmid=28373111&doi=10.1016/j.xphs.2017.03.022&)

69. Xu Q, Wang C, Meng Q, Liu Q, Sun H, Peng J, Ma X, Kaku T and Liu K. OAT1 and OAT3: targets of drug-drug interaction between entecavir and JBP485. European journal of pharmaceutical sciences : official journal of the European Federation for Pharmaceutical Sciences. 2013;48:650–7.  [DOI](https://doi.org/10.1016/j.ejps.2012.12.024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23313623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=European%20journal%20of%20pharmaceutical%20sciences%20:%20official%20journal%20of%20the%20European%20Federation%20for%20Pharmaceutical%20Sciences&title=OAT1%20and%20OAT3:%20targets%20of%20drug-drug%20interaction%20between%20entecavir%20and%20JBP485&author=Q%20Xu&author=C%20Wang&author=Q%20Meng&author=Q%20Liu&author=H%20Sun&volume=48&publication_year=2013&pages=650-7&pmid=23313623&doi=10.1016/j.ejps.2012.12.024&)

70. Kouzuki H, Suzuki H and Sugiyama Y. Pharmacokinetic study of the hepatobiliary transport of indomethacin. Pharmaceutical research. 2000;17:432–8.  [DOI](https://doi.org/10.1023/a:1007576903935) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10870987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmaceutical%20research&title=Pharmacokinetic%20study%20of%20the%20hepatobiliary%20transport%20of%20indomethacin&author=H%20Kouzuki&author=H%20Suzuki&author=Y%20Sugiyama&volume=17&publication_year=2000&pages=432-8&pmid=10870987&doi=10.1023/a:1007576903935&)

71. Takeda M, Khamdang S, Narikawa S, Kimura H, Hosoyamada M, Cha SH, Sekine T and Endou H. Characterization of methotrexate transport and its drug interactions with human organic anion transporters. J Pharmacol Exp Ther. 2002;302:666–71.  [DOI](https://doi.org/10.1124/jpet.102.034330) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12130730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Characterization%20of%20methotrexate%20transport%20and%20its%20drug%20interactions%20with%20human%20organic%20anion%20transporters&author=M%20Takeda&author=S%20Khamdang&author=S%20Narikawa&author=H%20Kimura&author=M%20Hosoyamada&volume=302&publication_year=2002&pages=666-71&pmid=12130730&doi=10.1124/jpet.102.034330&)

72. Hu S, Mathijssen RH, de Bruijn P, Baker SD and Sparreboom A. Inhibition of OATP1B1 by tyrosine kinase inhibitors: in vitro-in vivo correlations. Br J Cancer. 2014;110:894–8.  [DOI](https://doi.org/10.1038/bjc.2013.811) | [PMC free article](/articles/PMC3929889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24398510/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Inhibition%20of%20OATP1B1%20by%20tyrosine%20kinase%20inhibitors:%20in%20vitro-in%20vivo%20correlations&author=S%20Hu&author=RH%20Mathijssen&author=P%20de%20Bruijn&author=SD%20Baker&author=A%20Sparreboom&volume=110&publication_year=2014&pages=894-8&pmid=24398510&doi=10.1038/bjc.2013.811&)

73. Taguchi T, Masuo Y, Sakai Y and Kato Y. Short-lasting inhibition of hepatic uptake transporter OATP1B1 by tyrosine kinase inhibitor pazopanib. Drug Metab Pharmacokinet. 2019;34:372–379.  [DOI](https://doi.org/10.1016/j.dmpk.2019.08.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31703927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Short-lasting%20inhibition%20of%20hepatic%20uptake%20transporter%20OATP1B1%20by%20tyrosine%20kinase%20inhibitor%20pazopanib&author=T%20Taguchi&author=Y%20Masuo&author=Y%20Sakai&author=Y%20Kato&volume=34&publication_year=2019&pages=372-379&pmid=31703927&doi=10.1016/j.dmpk.2019.08.001&)

74. Oulianova N, Falk S and Berteloot A. Two-step mechanism of phlorizin binding to the SGLT1 protein in the kidney. The Journal of membrane biology. 2001;179:223–42.  [DOI](https://doi.org/10.1007/s002320010049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11246421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20membrane%20biology&title=Two-step%20mechanism%20of%20phlorizin%20binding%20to%20the%20SGLT1%20protein%20in%20the%20kidney&author=N%20Oulianova&author=S%20Falk&author=A%20Berteloot&volume=179&publication_year=2001&pages=223-42&pmid=11246421&doi=10.1007/s002320010049&)

75. David-Silva A, Esteves JV, Morais M, Freitas HS, Zorn TM, Correa-Giannella ML and Machado UF. Dual SGLT1/SGLT2 Inhibitor Phlorizin Ameliorates Non-Alcoholic Fatty Liver Disease and Hepatic Glucose Production in Type 2 Diabetic Mice. Diabetes, metabolic syndrome and obesity : targets and therapy. 2020;13:739–751.  [DOI](https://doi.org/10.2147/DMSO.S242282) | [PMC free article](/articles/PMC7085338/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32231437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diabetes,%20metabolic%20syndrome%20and%20obesity%20:%20targets%20and%20therapy&title=Dual%20SGLT1/SGLT2%20Inhibitor%20Phlorizin%20Ameliorates%20Non-Alcoholic%20Fatty%20Liver%20Disease%20and%20Hepatic%20Glucose%20Production%20in%20Type%202%20Diabetic%20Mice&author=A%20David-Silva&author=JV%20Esteves&author=M%20Morais&author=HS%20Freitas&author=TM%20Zorn&volume=13&publication_year=2020&pages=739-751&pmid=32231437&doi=10.2147/DMSO.S242282&)

76. Urakami Y, Akazawa M, Saito H, Okuda M and Inui K. cDNA cloning, functional characterization, and tissue distribution of an alternatively spliced variant of organic cation transporter hOCT2 predominantly expressed in the human kidney. Journal of the American Society of Nephrology : JASN. 2002;13:1703–10.  [DOI](https://doi.org/10.1097/01.asn.0000019413.78751.46) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12089365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20of%20Nephrology%20:%20JASN&title=cDNA%20cloning,%20functional%20characterization,%20and%20tissue%20distribution%20of%20an%20alternatively%20spliced%20variant%20of%20organic%20cation%20transporter%20hOCT2%20predominantly%20expressed%20in%20the%20human%20kidney&author=Y%20Urakami&author=M%20Akazawa&author=H%20Saito&author=M%20Okuda&author=K%20Inui&volume=13&publication_year=2002&pages=1703-10&pmid=12089365&doi=10.1097/01.asn.0000019413.78751.46&)

77. Bednarczyk D, Ekins S, Wikel JH and Wright SH. Influence of molecular structure on substrate binding to the human organic cation transporter, hOCT1. Mol Pharmacol. 2003;63:489–98.  [DOI](https://doi.org/10.1124/mol.63.3.489) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12606755/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Influence%20of%20molecular%20structure%20on%20substrate%20binding%20to%20the%20human%20organic%20cation%20transporter,%20hOCT1&author=D%20Bednarczyk&author=S%20Ekins&author=JH%20Wikel&author=SH%20Wright&volume=63&publication_year=2003&pages=489-98&pmid=12606755&doi=10.1124/mol.63.3.489&)

78. Ohashi R, Tamai I, Yabuuchi H, Nezu JI, Oku A, Sai Y, Shimane M and Tsuji A. Na(+)-dependent carnitine transport by organic cation transporter (OCTN2): its pharmacological and toxicological relevance. J Pharmacol Exp Ther. 1999;291:778–84.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10525100/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Na(+)-dependent%20carnitine%20transport%20by%20organic%20cation%20transporter%20(OCTN2):%20its%20pharmacological%20and%20toxicological%20relevance&author=R%20Ohashi&author=I%20Tamai&author=H%20Yabuuchi&author=JI%20Nezu&author=A%20Oku&volume=291&publication_year=1999&pages=778-84&pmid=10525100&)

79. van Montfoort JE, Müller M, Groothuis GM, Meijer DK, Koepsell H and Meier PJ. Comparison of "type I" and "type II" organic cation transport by organic cation transporters and organic anion-transporting polypeptides. J Pharmacol Exp Ther. 2001;298:110–5.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11408531/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Comparison%20of%20%22type%20I%22%20and%20%22type%20II%22%20organic%20cation%20transport%20by%20organic%20cation%20transporters%20and%20organic%20anion-transporting%20polypeptides&author=JE%20van%20Montfoort&author=M%20M%C3%BCller&author=GM%20Groothuis&author=DK%20Meijer&author=H%20Koepsell&volume=298&publication_year=2001&pages=110-5&pmid=11408531&)

80. Cha SH, Sekine T, Fukushima JI, Kanai Y, Kobayashi Y, Goya T and Endou H. Identification and characterization of human organic anion transporter 3 expressing predominantly in the kidney. Mol Pharmacol. 2001;59:1277–86.  [DOI](https://doi.org/10.1124/mol.59.5.1277) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11306713/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Identification%20and%20characterization%20of%20human%20organic%20anion%20transporter%203%20expressing%20predominantly%20in%20the%20kidney&author=SH%20Cha&author=T%20Sekine&author=JI%20Fukushima&author=Y%20Kanai&author=Y%20Kobayashi&volume=59&publication_year=2001&pages=1277-86&pmid=11306713&doi=10.1124/mol.59.5.1277&)

81. Yabuuchi H, Tamai I, Nezu J, Sakamoto K, Oku A, Shimane M, Sai Y and Tsuji A. Novel membrane transporter OCTN1 mediates multispecific, bidirectional, and pH-dependent transport of organic cations. J Pharmacol Exp Ther. 1999;289:768–73.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10215651/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Novel%20membrane%20transporter%20OCTN1%20mediates%20multispecific,%20bidirectional,%20and%20pH-dependent%20transport%20of%20organic%20cations&author=H%20Yabuuchi&author=I%20Tamai&author=J%20Nezu&author=K%20Sakamoto&author=A%20Oku&volume=289&publication_year=1999&pages=768-73&pmid=10215651&)

82. Nozawa T, Tamai I, Sai Y, Nezu J and Tsuji A. Contribution of organic anion transporting polypeptide OATP-C to hepatic elimination of the opioid pentapeptide analogue [D-Ala2, D-Leu5]-enkephalin. J Pharm Pharmacol. 2003;55:1013–20.  [DOI](https://doi.org/10.1211/0022357021440) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12906759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharm%20Pharmacol&title=Contribution%20of%20organic%20anion%20transporting%20polypeptide%20OATP-C%20to%20hepatic%20elimination%20of%20the%20opioid%20pentapeptide%20analogue%20%5BD-Ala2,%20D-Leu5%5D-enkephalin&author=T%20Nozawa&author=I%20Tamai&author=Y%20Sai&author=J%20Nezu&author=A%20Tsuji&volume=55&publication_year=2003&pages=1013-20&pmid=12906759&doi=10.1211/0022357021440&)

83. Vavricka SR, Van Montfoort J, Ha HR, Meier PJ and Fattinger K. Interactions of rifamycin SV and rifampicin with organic anion uptake systems of human liver. Hepatology (Baltimore, Md). 2002;36:164–72.  [DOI](https://doi.org/10.1053/jhep.2002.34133) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12085361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology%20(Baltimore,%20Md)&title=Interactions%20of%20rifamycin%20SV%20and%20rifampicin%20with%20organic%20anion%20uptake%20systems%20of%20human%20liver&author=SR%20Vavricka&author=J%20Van%20Montfoort&author=HR%20Ha&author=PJ%20Meier&author=K%20Fattinger&volume=36&publication_year=2002&pages=164-72&pmid=12085361&doi=10.1053/jhep.2002.34133&)

84. Fattinger K, Cattori V, Hagenbuch B, Meier PJ and Stieger B. Rifamycin SV and rifampicin exhibit differential inhibition of the hepatic rat organic anion transporting polypeptides, Oatp1 and Oatp2. Hepatology (Baltimore, Md). 2000;32:82–6.  [DOI](https://doi.org/10.1053/jhep.2000.8539) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10869292/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology%20(Baltimore,%20Md)&title=Rifamycin%20SV%20and%20rifampicin%20exhibit%20differential%20inhibition%20of%20the%20hepatic%20rat%20organic%20anion%20transporting%20polypeptides,%20Oatp1%20and%20Oatp2&author=K%20Fattinger&author=V%20Cattori&author=B%20Hagenbuch&author=PJ%20Meier&author=B%20Stieger&volume=32&publication_year=2000&pages=82-6&pmid=10869292&doi=10.1053/jhep.2000.8539&)

85. Cui Y, König J, Leier I, Buchholz U and Keppler D. Hepatic uptake of bilirubin and its conjugates by the human organic anion transporter SLC21A6. J Biol Chem. 2001;276:9626–30.  [DOI](https://doi.org/10.1074/jbc.M004968200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11134001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Hepatic%20uptake%20of%20bilirubin%20and%20its%20conjugates%20by%20the%20human%20organic%20anion%20transporter%20SLC21A6&author=Y%20Cui&author=J%20K%C3%B6nig&author=I%20Leier&author=U%20Buchholz&author=D%20Keppler&volume=276&publication_year=2001&pages=9626-30&pmid=11134001&doi=10.1074/jbc.M004968200&)

86. Sekine T, Cha SH, Tsuda M, Apiwattanakul N, Nakajima N, Kanai Y and Endou H. Identification of multispecific organic anion transporter 2 expressed predominantly in the liver. FEBS letters. 1998;429:179–82.  [DOI](https://doi.org/10.1016/s0014-5793(98)00585-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9650585/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=FEBS%20letters&title=Identification%20of%20multispecific%20organic%20anion%20transporter%202%20expressed%20predominantly%20in%20the%20liver&author=T%20Sekine&author=SH%20Cha&author=M%20Tsuda&author=N%20Apiwattanakul&author=N%20Nakajima&volume=429&publication_year=1998&pages=179-82&pmid=9650585&doi=10.1016/s0014-5793(98)00585-7&)

87. Hirano M, Maeda K, Shitara Y and Sugiyama Y. Drug-drug interaction between pitavastatin and various drugs via OATP1B1. Drug Metab Dispos. 2006;34:1229–36.  [DOI](https://doi.org/10.1124/dmd.106.009290) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16595711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Drug-drug%20interaction%20between%20pitavastatin%20and%20various%20drugs%20via%20OATP1B1&author=M%20Hirano&author=K%20Maeda&author=Y%20Shitara&author=Y%20Sugiyama&volume=34&publication_year=2006&pages=1229-36&pmid=16595711&doi=10.1124/dmd.106.009290&)

88. Minematsu T and Giacomini KM. Interactions of tyrosine kinase inhibitors with organic cation transporters and multidrug and toxic compound extrusion proteins. Mol Cancer Ther. 2011;10:531–9.  [DOI](https://doi.org/10.1158/1535-7163.MCT-10-0731) | [PMC free article](/articles/PMC3063525/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21252289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&title=Interactions%20of%20tyrosine%20kinase%20inhibitors%20with%20organic%20cation%20transporters%20and%20multidrug%20and%20toxic%20compound%20extrusion%20proteins&author=T%20Minematsu&author=KM%20Giacomini&volume=10&publication_year=2011&pages=531-9&pmid=21252289&doi=10.1158/1535-7163.MCT-10-0731&)

89. Shen H, Yang Z, Zhao W, Zhang Y and Rodrigues AD. Assessment of vandetanib as an inhibitor of various human renal transporters: inhibition of multidrug and toxin extrusion as a possible mechanism leading to decreased cisplatin and creatinine clearance. Drug Metab Dispos. 2013;41:2095–103.  [DOI](https://doi.org/10.1124/dmd.113.053215) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24026623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Assessment%20of%20vandetanib%20as%20an%20inhibitor%20of%20various%20human%20renal%20transporters:%20inhibition%20of%20multidrug%20and%20toxin%20extrusion%20as%20a%20possible%20mechanism%20leading%20to%20decreased%20cisplatin%20and%20creatinine%20clearance&author=H%20Shen&author=Z%20Yang&author=W%20Zhao&author=Y%20Zhang&author=AD%20Rodrigues&volume=41&publication_year=2013&pages=2095-103&pmid=24026623&doi=10.1124/dmd.113.053215&)

90. Cho SK, Kim CO, Park ES and Chung JY. Verapamil decreases the glucose-lowering effect of metformin in healthy volunteers. Br J Clin Pharmacol. 2014;78:1426–32.  [DOI](https://doi.org/10.1111/bcp.12476) | [PMC free article](/articles/PMC4256631/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25060604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Verapamil%20decreases%20the%20glucose-lowering%20effect%20of%20metformin%20in%20healthy%20volunteers&author=SK%20Cho&author=CO%20Kim&author=ES%20Park&author=JY%20Chung&volume=78&publication_year=2014&pages=1426-32&pmid=25060604&doi=10.1111/bcp.12476&)

91. Oostendorp RL, van de Steeg E, van der Kruijssen CM, Beijnen JH, Kenworthy KE, Schinkel AH and Schellens JH. Organic anion-transporting polypeptide 1B1 mediates transport of Gimatecan and BNP1350 and can be inhibited by several classic ATP-binding cassette (ABC) B1 and/or ABCG2 inhibitors. Drug Metab Dispos. 2009;37:917–23.  [DOI](https://doi.org/10.1124/dmd.108.024901) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19139163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Organic%20anion-transporting%20polypeptide%201B1%20mediates%20transport%20of%20Gimatecan%20and%20BNP1350%20and%20can%20be%20inhibited%20by%20several%20classic%20ATP-binding%20cassette%20(ABC)%20B1%20and/or%20ABCG2%20inhibitors&author=RL%20Oostendorp&author=E%20van%20de%20Steeg&author=CM%20van%20der%20Kruijssen&author=JH%20Beijnen&author=KE%20Kenworthy&volume=37&publication_year=2009&pages=917-23&pmid=19139163&doi=10.1124/dmd.108.024901&)

92. Cvetkovic M, Leake B, Fromm MF, Wilkinson GR and Kim RB. OATP and P-glycoprotein transporters mediate the cellular uptake and excretion of fexofenadine. Drug Metab Dispos. 1999;27:866–71.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10421612/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=OATP%20and%20P-glycoprotein%20transporters%20mediate%20the%20cellular%20uptake%20and%20excretion%20of%20fexofenadine&author=M%20Cvetkovic&author=B%20Leake&author=MF%20Fromm&author=GR%20Wilkinson&author=RB%20Kim&volume=27&publication_year=1999&pages=866-71&pmid=10421612&)

93. Radchenko M, Symersky J, Nie R and Lu M. Structural basis for the blockade of MATE multidrug efflux pumps. Nat Commun. 2015;6:7995.  [DOI](https://doi.org/10.1038/ncomms8995) | [PMC free article](/articles/PMC4866600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26246409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=basis%20for%20the%20blockade%20of%20MATE%20multidrug%20efflux%20pumps&author=M%20Radchenko&author=J%20Symersky&author=R%20Nie&author=M%20Lu&volume=6&publication_year=2015&pages=7995&pmid=26246409&doi=10.1038/ncomms8995&)

94. Zhang L, Schaner ME and Giacomini KM. Functional characterization of an organic cation transporter (hOCT1) in a transiently transfected human cell line (HeLa). J Pharmacol Exp Ther. 1998;286:354–61.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9655880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Functional%20characterization%20of%20an%20organic%20cation%20transporter%20(hOCT1)%20in%20a%20transiently%20transfected%20human%20cell%20line%20(HeLa)&author=L%20Zhang&author=ME%20Schaner&author=KM%20Giacomini&volume=286&publication_year=1998&pages=354-61&pmid=9655880&)

95. Gorboulev V, Ulzheimer JC, Akhoundova A, Ulzheimer-Teuber I, Karbach U, Quester S, Baumann C, Lang F, Busch AE and Koepsell H. Cloning and characterization of two human polyspecific organic cation transporters. DNA and cell biology. 1997;16:871–81.  [DOI](https://doi.org/10.1089/dna.1997.16.871) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9260930/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=DNA%20and%20cell%20biology&title=Cloning%20and%20characterization%20of%20two%20human%20polyspecific%20organic%20cation%20transporters&author=V%20Gorboulev&author=JC%20Ulzheimer&author=A%20Akhoundova&author=I%20Ulzheimer-Teuber&author=U%20Karbach&volume=16&publication_year=1997&pages=871-81&pmid=9260930&doi=10.1089/dna.1997.16.871&)

96. Wu X, Huang W, Ganapathy ME, Wang H, Kekuda R, Conway SJ, Leibach FH and Ganapathy V. Structure, function, and regional distribution of the organic cation transporter OCT3 in the kidney. American journal of physiology Renal physiology. 2000;279:F449–58.  [DOI](https://doi.org/10.1152/ajprenal.2000.279.3.F449) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10966924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=American%20journal%20of%20physiology%20Renal%20physiology&title=Structure,%20function,%20and%20regional%20distribution%20of%20the%20organic%20cation%20transporter%20OCT3%20in%20the%20kidney&author=X%20Wu&author=W%20Huang&author=ME%20Ganapathy&author=H%20Wang&author=R%20Kekuda&volume=279&publication_year=2000&pages=F449-58&pmid=10966924&doi=10.1152/ajprenal.2000.279.3.F449&)

97. Wu X, George RL, Huang W, Wang H, Conway SJ, Leibach FH and Ganapathy V. Structural and functional characteristics and tissue distribution pattern of rat OCTN1, an organic cation transporter, cloned from placenta. Biochim Biophys Acta. 2000;1466:315–27.  [DOI](https://doi.org/10.1016/s0005-2736(00)00189-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10825452/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&title=Structural%20and%20functional%20characteristics%20and%20tissue%20distribution%20pattern%20of%20rat%20OCTN1,%20an%20organic%20cation%20transporter,%20cloned%20from%20placenta&author=X%20Wu&author=RL%20George&author=W%20Huang&author=H%20Wang&author=SJ%20Conway&volume=1466&publication_year=2000&pages=315-27&pmid=10825452&doi=10.1016/s0005-2736(00)00189-9&)

98. Wu X, Huang W, Prasad PD, Seth P, Rajan DP, Leibach FH, Chen J, Conway SJ and Ganapathy V. Functional characteristics and tissue distribution pattern of organic cation transporter 2 (OCTN2), an organic cation/carnitine transporter. J Pharmacol Exp Ther. 1999;290:1482–92.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10454528/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Functional%20characteristics%20and%20tissue%20distribution%20pattern%20of%20organic%20cation%20transporter%202%20(OCTN2),%20an%20organic%20cation/carnitine%20transporter&author=X%20Wu&author=W%20Huang&author=PD%20Prasad&author=P%20Seth&author=DP%20Rajan&volume=290&publication_year=1999&pages=1482-92&pmid=10454528&)
