# Association between angiotensinogen (AGT), angiotensin-converting enzyme (ACE) and angiotensin-II receptor 1 (AGTR1) polymorphisms and COVID-19 infection in the southeast of Iran: a preliminary case-control study

## Metadata
**Authors:** Hamid Reza Kouhpayeh, Farhad Tabasi, Mohammad Dehvari, Mohammad Naderi, Gholamreza Bahari, Tahereh Khalili, Courtney Clark, Saeid Ghavami, Mohsen Taheri
**Journal:** Translational Medicine Communications
**Date:** 2021 Nov 17
**DOI:** [10.1186/s41231-021-00106-0](https://doi.org/10.1186/s41231-021-00106-0)
**PMID:** 34805533
**PMCID:** PMC8596349
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8596349/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8596349/pdf/41231_2021_Article_106.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8596349/pdf/41231_2021_Article_106.pdf)

## Abstract

**Background:** 
The COVID-19 pandemic remains an emerging public health crisis with serious adverse effects. The disease is caused by severe acute respiratory syndrome coronavirus-2 (SARS-CoV--2) infection, targeting angiotensin-converting enzyme-2 (ACE2) receptor for cell entry. However, changes in the renin-angiotensin system (RAS) balance alter an individual’s susceptibility to COVID-19 infection. We aimed to evaluate the association between AGT rs699 C > T, ACE rs4646994 I/D, and AGTR1 rs5186 C > A variants and the risk of COVID-19 infection and the severity in a sample of the southeast Iranian population.

**Methods:** 
A total of 504 subjects, including 258 COVID-19 positives, and 246 healthy controls, were recruited. Genotyping of the ACE gene rs4646994, and AGT rs699, and AGTR1 rs5186 polymorphisms was performed by polymerase chain reaction (PCR) and PCR-restriction fragment length polymorphism (PCR-RFLP), respectively.

**Results:** 
Our results showed that the II genotype of ACE rs4646994 and the I allele decreased the risk of COVID-19 infection. Moreover, we found that the TC genotype and C allele of AGT rs699 increased the risk of COVID-19 infection. The AGTR1 rs5186 was not associated with COVID-19 infection. Also, we did not find any association between these polymorphisms and the severity of the disease. However, we found a significantly higher age and prevalence of diabetes and hypertension in patients with severe disease than a non-severe disease.

**Conclusions:** 
These findings suggest that ACE rs4646994 and AGT rs699 polymorphisms increase the risk of COVID-19 infection in a southeast Iranian population.

Keywords: COVID-19, SARS-CoV-2, Angiotensin, AGT, ACE, AGTR1, Polymorphism

### Background

The COVID-19 pandemic remains an emerging public health crisis with serious adverse effects. The disease is caused by severe acute respiratory syndrome coronavirus-2 (SARS-CoV--2) infection, targeting angiotensin-converting enzyme-2 (ACE2) receptor for cell entry. However, changes in the renin-angiotensin system (RAS) balance alter an individual’s susceptibility to COVID-19 infection. We aimed to evaluate the association between AGT rs699 C > T, ACE rs4646994 I/D, and AGTR1 rs5186 C > A variants and the risk of COVID-19 infection and the severity in a sample of the southeast Iranian population.

### Methods

A total of 504 subjects, including 258 COVID-19 positives, and 246 healthy controls, were recruited. Genotyping of the ACE gene rs4646994, and AGT rs699, and AGTR1 rs5186 polymorphisms was performed by polymerase chain reaction (PCR) and PCR-restriction fragment length polymorphism (PCR-RFLP), respectively.

### Results

Our results showed that the II genotype of ACE rs4646994 and the I allele decreased the risk of COVID-19 infection. Moreover, we found that the TC genotype and C allele of AGT rs699 increased the risk of COVID-19 infection. The AGTR1 rs5186 was not associated with COVID-19 infection. Also, we did not find any association between these polymorphisms and the severity of the disease. However, we found a significantly higher age and prevalence of diabetes and hypertension in patients with severe disease than a non-severe disease.

### Conclusions

These findings suggest that ACE rs4646994 and AGT rs699 polymorphisms increase the risk of COVID-19 infection in a southeast Iranian population.

## Introduction

Coronavirus diseases-2019 (COVID-19) is an emerging global pandemic caused by severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2). It was first identified in Wuhan, China, at the end of 2019 and has spread rapidly worldwide [[1](#CR1)]. After about two years from the beginning of the pandemic, more than 235 million cases with approximately 4.8 million deaths are reported worldwide, with turned COVID-19 as a significant global crisis in modern history ([https://www.worldometers.info/coronavirus](https://www.worldometers.info/coronavirus)). In Iran, beyond 5.5 million cases and 120,880 deaths due to COVID-19 are officially reported until the date. COVID-19 is mainly pneumonia characterized by cough, a fever, shortness of breath, fatigue, and gastrointestinal symptoms such as anorexia, nausea, vomiting, and diarrhea [[2](#CR2)]; however, all organs could be affected by the virus, including the endocrine [[3](#CR3)], cardiovascular [[4](#CR4)], renal [[5](#CR5)] and nervous systems [[6](#CR6)] regardless the presence of pneumonia. SARS-CoV-2 is transmitted from human to human by many mechanisms such as respiratory droplets, aerosols, and unprotected contact [[7](#CR7)].

SARS-CoV-2 is a highly infectious virus [[8](#CR8), [9](#CR9)] that uses angiotensin-converting enzyme 2 (ACE2) as the major receptor for viral entry in humans [[10](#CR10)–[13](#CR13)]. It has been shown that SARS-CoV-2 spike (S) glycoprotein binds via its receptor-binding domain (RDB) with a high affinity to human ACE2 [[14](#CR14)], and therefore, mediates virus internalization [[10](#CR10), [15](#CR15)]. This mechanism is quite similar to the SARS-CoV virus [[16](#CR16)], but SARS-CoV-2 has a remarkably higher affinity for ACE2 [[17](#CR17)]. Therefore, recognizing features of the angiotensin pathway in COVID-19 can elucidate how individuals have differences in symptoms, severity, complications, and mortality. Understanding this mechanism could also be beneficial for therapeutic targeting. Potentially, an imbalance in this pathway, with the centrality of ACE1/ACE2 activity, can be responsible for COVID-19 pathophysiology [[18](#CR18)–[20](#CR20)].

Angiotensin II receptor 1 (AGTR1) may drive COVID-19 pathology: AGTR1 is G-protein coupled receptor (GPCR), mediates signaling and most functions of angiotensin-II that generated by the angiotensin-I converting enzyme (ACE1) [[21](#CR21), [22](#CR22)]. The angiotensin II via AGTR1 induces inflammation, apoptosis, and organ damage in pulmonary and cardiovascular tissues [[23](#CR23)]. Previous evidence regarding SARS-CoV infection, mediated by ACE-2, which highly express in pulmonary tissue, indicates that binding SARS virus results in a decrease in ACE2 activity and expression [[17](#CR17)], leading to an increase in angiotensin-II level [[24](#CR24)]. Further, downregulated ACE2 decreases peptides converted from angiotensin-II to counteract its effects, including pro-apoptotic properties [[18](#CR18)], inflammation, and fibrosis [[19](#CR19), [20](#CR20)], all of which are the basis of COVID-19 pathobiology.

On the other hand, genetic variations of a gene potentially alter the expression and functions of an encoded product, which can be considered the basis of inter-individual differences in susceptibility to infectious diseases [[25](#CR25)]. Although the pandemic affected many populations, there are discrepancies regarding symptoms and disease severity, ranging from asymptomatic to devastating pneumonia with progressive multi-organ involvement. Part of these differences can be due to underlying conditions in at-risk populations; however, polymorphism in pathways related to the pathobiology of COVID-19, including angiotensin receptor and renin-angiotensin system (RAS), can be another contributing factor [[26](#CR26)].

Angiotensinogen is a peptide hormone encoded by the *AGT* gene mapped on chromosome 1q42.2 [[27](#CR27)]. The rs699 (M268T, previously known as M235T) is a missense polymorphism on exon 2, encodes the threonine variant, which is associated with increased angiotensin levels [[28](#CR28), [29](#CR29)]. Angiotensinogen is cleaved by renin, produce angiotensin-I, which is later converted to angiotensin-II by ACE1. ACE1 is encoded by the *ACE* gene on chromosome 17q23.3. The rs4646994 is a 287-bp insertion/deletion (indel) variant in intron 16 of the *ACE* gene, which is strongly correlated with ACE level [[30](#CR30)] and activity [[31](#CR31)]. Finally, angiotensin-II exerts its effects via AGTR1, encoded by the AGTR1 gene on chromosome 3q24. The rs5186 (A1166C) variant is located at the 3’untranlated region (3’UTR) of the *AGTR1* gene, potentially can affect mRNA stability and, therefore, AGTR1 levels [[32](#CR32)], and associated with increased risk of hypertension [[33](#CR33)].

Therefore, in addition to underlying known risk factors, host genetic predisposition may influence the risk, severity, and outcome of the disease [[34](#CR34)]. Given the current evidence regarding the potential implication of angiotensin-related signaling cascade in COVID-19 [[18](#CR18), [19](#CR19)], we investigate the association between AGT rs699 C > T, ACE rs4646994 indel, and AGTR1 rs5186 C > A variants and the risk of COVID-19 infection in a sample of the southeast Iranian population.

## Materials and methods

### Subjects

The sample population for this case-control study comprised 258 COVID-19 subjects who tested positive for SARS-CoV-2 infection using real-time reverse transcription polymerase chain reaction (RT-PCR) technique and 246 healthy control subjects with similar geographic and ethnic backgrounds. According to World Health Organization (WHO) case definition, we enrolled patients who met with the definition for confirmed COVID-19 patient [[35](#CR35)]. All patients were assessed thoroughly regarding preexisting conditions such as previous infectious diseases and underlying chronic conditions. All patients with a concurrent infectious disease were excluded. Samples were collected in Bu-Ali Sina Hospital, designated as a specialized center for infectious disease in Zahedan, from May 2020 through September 2020, in which the original SARS-CoV-2 was dominant. Further, the control group comprised individuals who tested negative to COVID-19 by RT-PCR method AND clinical diagnostic criteria, which finally ruled out the disease.

Severe and non-severe cases were defined based on WHO guidance for disease severity definition [[35](#CR35)]. Accordingly, severe cases were defined as patients with a positive RT-PCR result for COVID-19, and clinical signs comply with severe pneumonia plus one of the following conditions: SpO2 < 90% on room air or respiratory rate > 30 breath/min or sign of severe respiratory distress. The non-severe case was defined as patients with a positive RT-PCR in the absence of any criteria for severe disease. Clinical diagnosis of participants was made by two infectious diseases specialists in Bu-Ali Sina Hospital. The study protocol was approved by the Ethics Committee of Zahedan University of Medical Sciences (IR.ZAUMS.REC.1399). Informed consent was provided by all subjects or their family members.

### Genotyping

Blood samples were drawn from each participant into a tube containing EDTA, and DNA was isolated using the salting-out method. We used PCR to detect ACE rs4646994 indel polymorphism. The PCR-restriction fragment length polymorphism (PCR-RFLP) method was applied for genotyping AGT rs699 and AGTR1 rs5186 polymorphisms. Primer sequences, restriction enzymes, and length of the fragments are summarized in Table [1](#Tab1). PCR was performed in a final volume of 20 μL containing 1 μL of genomic DNA (~ 100 ng/μL), 1 μL of each primer (10 μM), and 10 μL of 2X Prime Taq Premix (Genet Bio, Korea) and 7 μL ddH2O. PCR conditions included an initial denaturing step at 95 °C for 5 min, followed by 30 cycles of 95 °C for 30 s, annealing at 68 °C for AGT rs699, 60 °C for AT1R rs5186, and 66 °C for ACE rs4646994 for 30 s and 72 °C for 30 s, and a final extension at 72 °C for 5 min. The PCR product was digested by suitable restriction enzymes (Table [1](#Tab1)). The fragments were then separated by electrophoresis in 2.5% agarose gels (Figs. [1](#Fig1), [2](#Fig2) and [3](#Fig3)). We randomly selected 10% of the samples for quality control to repeat the results, which was 100% reproducible.

### Table 1.

| Polymorphisms | Sequence (5`- > 3`) | Restriction Enzyme | Product size (bp) | Annealing temperature (°C) |
| --- | --- | --- | --- | --- |
| ACE rs4646994 | F: GCCCTGCAGGTGTCTGCAGCATGT R: GGATGGCTCTCCCCGCCTTGTCTC | – | I allele: 599 bp D allele: 312 bp | 66 |
| AGT rs699 | F: CCGTTTGTGCAGGGCCTGGCTCTCT R: CAGGGTGCTGTCCACACTGGACCCC | Tth111I | T allele: 165 C allele: 141 + 24 | 68 |
| AGTR1 rs5186 | F: AGAAGCCTGCACCATGTTTTGAG R: CCTGTTGCTCCTCTAACGATTTA | DdeI | A allele: 410 bp C allele: 292 + 118 | 60 |

Table 1 Caption: Primer sequence used for detection of ACE rs4646994, AGT rs699 and AGTR1 rs5186 gene polymorphisms

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/72e1/8596349/6bde96ff1ea4/41231_2021_106_Fig1_HTML.jpg)

Electrophoresis pattern of the PCR products for ACE rs4646994 indel polymorphism detection. M: DNA marker; Lanes 1, 6: ID; Lanes 2, 5: II; Lanes 3, 4: DD

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/72e1/8596349/9d47a07cdb42/41231_2021_106_Fig2_HTML.jpg)

Electrophoresis pattern of the PCR-RFLP products for AGT rs699 polymorphism detection. M: DNA marker; Lanes 1, 4: TC; Lanes 2, 3: TT

### Fig. 3.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/72e1/8596349/4ac708dce6bd/41231_2021_106_Fig3_HTML.jpg)

Electrophoresis pattern of the PCR-RFLP products for AGTR1 rs5186 polymorphism detection. M: DNA marker; Lanes 1, 4: AA; Lanes 2, 3: AC

### Statistical analysis

Statistical analysis was performed using the SPSS 20.0 package software (IBM Corporation, USA). The Kolmogorov-Smirnov test was applied to assess the distribution normality. Comparisons were made by χ^2^ or independent sample *t*-test (or Mann-Whitney test, as appropriate) according to the data. Logistic regression analyses were used to calculate the odds ratios (ORs) and 95% confidence intervals (CIs) under different genetic models to explore the association between genotypes, COVID-19 infection risk, and disease severity. The level of statistical significance was defined as *p* < 0.05.

## Results

The study group included 258 confirmed COVID-19 patients (144 males, 114 females) and 246 healthy subjects (132 males, 114 females). The mean (SD) age of patients and controls were 50.23 (14.82) and 49.01 (14.95), respectively, and the median (IQR) of patients and control was 51.0 (23.0) and 48.5 (24.0), respectively. There was no statistically significant difference between the groups (*p* = 0.39). The demographic and clinical characteristics of patients are summarized in Table [2](#Tab2). Also, we compared COVID-19 patients based on the disease severity. Although there were no differences regarding sex between severe and non-severe cases (*p* = 0.31), patients with severe disease had significantly higher age than non-severe patients (*p* = 0.003). Additionally, we compared patients with established hypertension and diabetes (based on their medical records and investigations/examinations during hospitalization); the frequency of hypertension and diabetes was significantly higher in patients with severe disease (*p* < 0.001 and *p* = 0.042, respectively). The characteristics of severe and non-severe cases are summarized in Table [3](#Tab3).

### Table 2.

| Characteristics |  | Patients, n (%) | Control, n (%) | p |
| --- | --- | --- | --- | --- |
| Gender | Male | 144 (55.8) | 132 (53.7) | 0.34* |
| Female | 114 (44.2) | 114 (46.3) |  |  |
| Total | 258 (100) | 246 (100) |  |  |
| Age | Mean (SD) | 50.23 (14.82) | 49.01 (14.95) | 0.39** |
| Median (IQR) | 51.0 (23.0) | 48.5 (24.0) |  |  |

Table 2 Caption: Demographic characteristics of COVID-19 patients and control individuals

### Table 3.

| Characteristics |  | Non-Severe, n(%) | Severe, n(%) | p |
| --- | --- | --- | --- | --- |
| Gender | Male | 55 (51.9) | 89 (58.6) | 0.31* |
| Female | 51 (48.1) | 63 (41.4) |  |  |
| Total | 106 (100) | 152 (100) |  |  |
| Age | Mean (SD) | 45.69 (13.87) | 52.27 (14.82) | 0.003** |
| Median (IQR) | 46.0 (23.5) | 52 (23.0) |  |  |
| Hypertension | Yes | 8 (16.7) | 65 (43.9) | < 0.001* |
| No | 40 (83.3) | 83 (56.1) |  |  |
| Total | 48 (100) | 148 (100) |  |  |
| Diabetes | Yes | 10 (20.8) | 53 (35.3) | 0.042* |
| No | 38 (79.2) | 97 (64.7) |  |  |
| Total | 48 (100) | 150 (100) |  |  |

Table 3 Caption: Demographic and clinical characteristics of severe and non-severe patients

The genotype and allele frequencies of ACE rs4646994, AGT rs699, and AGTR1 rs5186 are shown in Table [4](#Tab4). We calculated the Hardy-Winberg equilibrium (HWE) for patients and control subjects of each polymorphism. The results showed that distributions of ACE 4646994 and AGTR1 rs5186 (patients) were in HWE (*p* > 0.05), but distributions of AGTR1 rs5186 (controls) and AGT rs699 groups significantly deviated from HWE (*p* < 0.05, Table [4](#Tab4)).

### Table 4.

| Polymorphism | Group | Genotypes, n (%) |  |  | Allele, n (%) |  | HWE |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| χ2 | p |  |  |  |  |  |  |  |
|   |   | DD | DI | II | D | I |   |   |
| ACE rs4646994 | Patients | 144 (55.8) | 89 (34.5) | 25 (9.7) | 377 (73.6) | 139 (26.94) | 3.94 | 0.052 |
| Controls | 70 (28.7) | 123 (50.4) | 51 (20.9) | 263 (53.89) | 225 (46.11) | 0.05 | 0.90 |  |
|   |  | TT | TC |   | T | C |   |   |
| AGT rs699 | Patients | 20 (9.2) | 197 (90.8) | – | 237 (54.61) | 197 (45.39) | 142.9 | < 0.001 |
| Controls | 113 (46.1) | 132 (53.9) | – | 358 (73.06) | 132 (26.94) | 33.3 | < 0.001 |  |
|   |  | AA | AC | CC | A | C |   |   |
| AGTR1 rs5186 | Patients | 154 (74.4) | 49 (23.7) | 4 (1.9) | 357 (86.81) | 57 (13.19) | 0.001 | > 0.99 |
| Controls | 185 (75.8) | 59 (24.2) | 0 | 429 (87.91) | 59 (12.09) | 4.61 | 0.031 |  |

Table 4 Caption: Alleles and genotypes frequencies of ACE rs4646994, AGT rs699 and AGTR1 rs5186 polymorphisms in COVID-19 patients and control subjects

### Association between the ACE rs4646994, AGT rs699, and AGTR1 rs5186 variants and risk of COVID-19 infection

The frequency of ACE rs4646994, AGT rs699, and AGTR1 rs5186 polymorphisms in COVID-19 patients and control subjects are shown in Table [4](#Tab4). The results revealed significant differences between the ACE rs4646994 and ATG rs699 polymorphisms, but not the ATGR1 rs5186 variant between patient and control groups. Our analyses of ACE rs4646994 shows that rs4646944 is associated with lower risk of COVID-19 in heterozygous (OR = 0.35, 95%CI = 0.23–0.52, *p* < 0.001, ID vs DD), homozygous (OR = 0.23, 95%CI = 0.13–0.41, *p* < 0.001, II vs DD), dominant (OR = 0.31, 95%CI = 0.21–0.46, *p* < 0.001, ID+DD vs DD), recessive (OR = 0.40, 95%CI = 0.24–0.67, *p* < 0.001, II vs ID+II), and overdominant (OR = 0.51, 95%CI = 0.36–0.74, *p* < 0.001, ID vs DD + II) models increased the risk of COVID-19 infection. In addition, the I allele was significantly associated with decreased risk of COVID-19 infection (OR = 0.43, 95%CI = 0.33–0.56, *p* < 0.001, I vs. D). These results showed that the D allele can be considered a risk factor for COVID-19 (Table [5](#Tab5)). Genotyping of AGT rs699 revealed a remarkable difference between COVID-19 positive patients and healthy controls; TC genotype (OR = 8.43, 95% CI = 4.99–14.24, *p* < 0.001, TC vs. TT) as well as C allele (OR = 2.25, 95% CI = 1.71–2.96, *p* < 0.001, C vs. T) increased the risk of COVID-19 infection. As presented in Table [5](#Tab5), there was no significant difference in genotype and allelic distribution of AGTR1 rs5186 gene polymorphisms between COVID-19 patients and controls (*p* > 0.05). We performed the logistic regression adjusted for age and sex, which the result was the same as crude analysis (Table [5](#Tab5)).

### Table 5.

| Polymorphisms | Genetic Models | Patients, n (%) | Control, n (%) | Crude analysis |  | Adjusted analysisa |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| ACE rs4646994 | Codominant |   |   | OR (95% CI) | p | OR (95% CI) | p |
| DD | 144 (55.8) | 70 (28.7) | 1.00 | – | 1.00 | – |  |
| ID | 89 (34.5) | 123 (50.4) | 0.35 (0.23–0.52) | < 0.001 | 0.38 (0.24–0.60) | < 0.001 |  |
| II | 25 (9.7) | 51 (20.9) | 0.23 (0.13–0.41) | < 0.001 | 0.11 (0.05–0.21) | < 0.001 |  |
| Dominant |   |   |   |   |   |   |  |
| DD | 144 (55.8) | 70 (28.7) | 1.00 | – | 1.00 | – |  |
| ID + II | 114 (44.2) | 174 (71.3) | 0.31 (0.21–0.46) | < 0.001 | 0.27 (0.18–0.42) | < 0.001 |  |
| Recessive |   |   |   |   |   |   |  |
| DD + ID | 233 (90.3) | 193 (79.1) | 1.00 | – | 1.00 | – |  |
| II | 25 (9.7) | 51 (20.9) | 0.40 (0.24–0.67) | < 0.001 | 0.18 (0.09–0.34) | < 0.001 |  |
| Overdominant |   |   |   |   |   |   |  |
| DD + II | 169 (65.5) | 121 (49.6) | 1.00 | – | 1.00 | – |  |
| ID | 89 (34.5) | 123 (50.4) | 0.51 (0.36–0.74) | < 0.001 | 0.71 (0.47–1.06) | 0.058 |  |
| Alleles |   |   |   |   |   |   |  |
| D | 377 (73.6) | 263 (53.89) | 1.00 | – | 1.00 | – |  |
| I | 139 (26.94) | 225 (46.11) | 0.43 (0.33–0.56) | < 0.001 | 0.32 (0.24–0.43) | < 0.001 |  |
| ATG rs699 | Codominant |   |   |   |   |   |   |
| TT | 20 (9.2) | 113 (46.1) | 1.00 | – |   |   |  |
| TC | 197 (90.8) | 132 (53.9) | 8.43 (4.99–14.24) | < 0.001 | 18.17 (9.88–33.39) | < 0.001 |  |
| Alleles |   |   |   |   |   |   |  |
| T | 237 (54.61) | 358 (73.06) | 1.00 | – | 1.00 | – |  |
| C | 197 (45.39) | 132 (26.94) | 2. 25 (1.71–2.96) | < 0.001 | 3.61 (2.58–5.05) | < 0.001 |  |
| ATGR1 rs5186 | Codominant |   |   |   |   |   |   |
| AA | 154 (74.4) | 185 (75.8) | 1.00 | – | 1.00 | – |  |
| AC | 49 (23.7) | 59 (24.2) | 0.99 (0.64–1.54) | 0.5 | 0.83 (0.51–1.36) | 0.28 |  |
| CC | 4 (1.9) | 0 | – | – | – | – |  |
| Dominant |   |   |   |   |   |   |  |
| AA | 154 (74.4) | 185 (75.8) | 1.00 | – | 1.00 | – |  |
| AC + CC | 53 (25.6) | 59 (24.2) | 1.07 (0.70–1.65) | 0.40 | 0.90 (0.55–1.45) | 0.38 |  |
| Recessive |   |   |   |   |   |   |  |
| AA + AC | 203 (98.1) | 244 (100) | 1.00 | – | 1.00 | – |  |
| CC | 4 (1.9) | 0 | – | – | – | – |  |
| Overdominant |   |   |   |   |   |   |  |
| AA + CC | 158 (76.3) | 185 (75.8) | 1.00 | – | 1.00 | – |  |
| AC | 49 (23.7) | 59 (24.2) | 0.97 (0.62–1.50) | 0.49 | 0.82 (0.50–1.33) | 0.24 |  |
| Alleles |   |   |   |   |   |   |  |
| A | 357 (86.81) | 429 (87.91) | 1.00 | – | 1.00 | – |  |
| C | 57 (13.19) | 59 (12.09) | 1.10 (0.74–1.63) | 0.34 | 0.99 (0.64–1.52) | 0.52 |  |

Table 5 Caption: Genotypes and allele frequencies of ACE rs4646994, ATG rs699 and AGTR1 rs5186 polymorphisms in COVID-19 patients and controls

### Association between the ACE rs4646994, AGT rs699, and AGTR1 rs5186 variants and the severity of COVID-19 infection

We compared the prevalence of ACE rs4646994, AGT rs699, and AGTR1 rs5186 polymorphisms between severe and non-severe cases of COVID-19 (Table [6](#Tab6)). As presented in Table [6](#Tab6), there was no significant difference in genotype and allelic distribution of ACE rs4646994, AGT rs699, and AGTR1 rs5186 gene polymorphisms and severity of the disease. However, ACE rs4646994 showed a significant association with non-severe disease in recessive model (OR = 0.42, 95%CI = 0.18–0.99, *p* = 0.03, II vs DD + ID). Also, the C allele of AGTR1 rs5186 was a significant association with non-severe disease (OR = 0.51, 95%CI = 0.29–0.90, *p* = 0.01, C vs. A). These trends were not observed in other hereditary models (Table [6](#Tab6)). Further, we performed logistic regression adjusted for age, sex, and existence for diabetes and hypertension. The results were the same as the crude analyses regarding the association between genetic models and severity (Table [6](#Tab6)).

### Table 6.

| Polymorphisms | Genetic Models | Severe, n (%) | Non-severe, n (%) | Crude analysis |  | Adjusted analysisa |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
| OR (95% CI) | p | OR (95% CI) | p |  |  |  |  |
| ACE rs4646994 | Codominant |   |   |   |   |   |   |
| DD | 84 (55.3) | 60 (56.6) | 1.00 | – | 1.00 | – |  |
| ID | 58 (38.2) | 31 (29.2) | 1.33 (0.77–2.31) | 0.18 | 1.12 (0.57–2.21) | 0.42 |  |
| II | 10 (6.6) | 15 (14.2) | 0.47 (0.20–1.13) | 0.06 | 0.70 (0.14–3.45) | 0.49 |  |
| Dominant |   |   |   |   |   |   |  |
| DD | 84 (55.3) | 60 (56.6) | 1.00 | – | 1.00 | – |  |
| ID + II | 68 (44.7) | 46 (43.4) | 1.05 (0.64–1.74) | 0.46 | 1.06 (0.55–2.05) | 0.48 |  |
| Recessive |   |   |   |   |   |   |  |
| DD + ID | 142 (93.4) | 91 (85.8) | 1.00 | – | 1.00 | – |  |
| II | 10 (6.6) | 15 (14.2) | 0.42 (0.18–0.99) | 0.03 | 0.66 (0.13–3.19) | 0.46 |  |
| Overdominant |   |   |   |   |   |   |  |
| DD + II | 94 (61.8) | 75 (70.8) | 1.00 | – | 1.00 | – |  |
| ID | 58 (38.2) | 31 (29.2) | 1.49 (0.87–2.53) | 0.08 | 1.16 (0.59–2.25) | 0.39 |  |
| Alleles |   |   |   |   |   |   |  |
| D | 226 (74.5) | 151 (75.0) | 1.00 | – | 1.00 | – |  |
| I | 78 (25.50) | 61 (25.0) | 0.85 (0.57–1.26) | 0.24 | 0.99 (0.58–1.68) | 0.54 |  |
| ATG rs699 | Codominant |   |   |   |   |   |   |
| TT | 8 (6.3) | 12 (13.3) | 1.00 | – | 1.00 | – |  |
| TC | 119 (93.7) | 78 (86.7) | 2.28 (0.89–5.85) | 0.06 | 2.33 (0.75–7.14) | 0.11 |  |
| Alleles |   |   |   |   |   |   |  |
| T | 135 (53.15) | 102 (56.67) | 1.00 | – | 1.00 | – |  |
| C | 119 (46.85) | 78 (43.44) | 1.15 (0.78–1.69) | 0.26 | 1.16 (0.70–1.90) | 0.31 |  |
| ATGR1 rs5186 | Codominant |   |   |   |   |   |   |
| AA | 96 (80.0) | 58 (66.7) | 1.00 | – | 1.00 | – |  |
| AC | 23 (19.2) | 26 (29.9) | 0.53 (0.27–1.02) | 0.06 | 0.55 (0.24–1.25) | 0.11 |  |
| CC | 1 (0.8%) | 3 (3.5) | 0.20 (0.02–1.98) | 0.16 | 0.30 (0.01–4.97) | 0.41 |  |
| Dominant |   |   |   |   |   |   |  |
| AA | 95 (80 | 58 (66.7) | 1.00 | – | 1.00 | – |  |
| AC + CC | 24 (20.0) | 19 (35.3) | 0.77 (0.38–1.52) | 0.28 | 0.53 (0.23–1.18) | 0.09 |  |
| Recessive |   |   |   |   |   |   |  |
| AA + AC | 119 (99.2) | 84 (96.5) | 1.00 | – | 1.00 | – |  |
| CC | 1 (0.8) | 3 (3.5) | 23 (0.02–2.30) | 0.20 | 0.34 (0.02–5.69) | 0.45 |  |
| Overdominant |   |   |   |   |   |   |  |
| AA + CC | 97 (80.8) | 61 (70.1) | 1.00 | – | 1.00 | – |  |
| AC | 23 (19.2) | 26 (29.2) | 0.55 (0.29–1.06) | 0.052 | 0.56 (0.24–1.28) | 0.12 |  |
| Alleles |   |   |   |   |   |   |  |
| A | 215 (89.58) | 142 (81.61) | 1.00 | – |   |   |  |
| C | 25 (10.42) | 32 (18.39) | 0.51 (0.29–0.90) | 0.01 | 0.56 (0.27–1.14) | 0.08 |  |

Table 6 Caption: Genotypes and allele frequencies of ACE rs4646994, ATG rs699 and AGTR1 rs5186 polymorphisms in non-severe and severe COVID-19 patients

## Discussion

In the present study, we explored the association of three important variants in the angiotensin pathway with susceptibility to COVID-19 in patients from southeast Iran. Our result showed that ACE indel and ATG rs699 is associated with susceptibility to COVID-19 infection; however, we did not observe any association between these polymorphisms and the risk of severe disease, but the age and prevalence of diabetes and hypertension in patients with severe disease were remarkably higher than patients with a non-severe disease.

COVID-19 infection has a variety of symptoms and severity in different individuals. Symptoms of patients infected with SARS-CoV-2 range from entirely asymptomatic to mild constitutional symptoms, which are commonly manifesting with fever, cough, and fatigue, to severe pneumonia associated with acute respiratory distress syndrome (ARDS), systemic inflammation, and organ failure [[36](#CR36), [37](#CR37)]. The latter frequently necessitated hospitalization and intensive care [[38](#CR38)]. Certain individuals are at increased risk for developing more symptomatic and severe illnesses. Previous studies introduced older ages [[39](#CR39)] and specific medical conditions, including diabetes and hypertension, as major risk factors for more severe disease [[38](#CR38), [40](#CR40), [41](#CR41)]. This was consistent with our results that demonstrated a higher prevalence of hypertension and diabetes in severe COVID-19, as well as older age.

On the other hand, other host factors, such as genetic variations, can be responsible for the observed differences in susceptibility and disease severity. Mounting evidence indicates that the RAS imbalance, which is closely associated with ARDS [[42](#CR42)], plays a key role in COVID-19 pathophysiology [[43](#CR43)]. SARS-CoV-2 binds to ACE-II surface receptors, one of the major components of RAS for cell entrance in humans [[10](#CR10), [11](#CR11)]. Angiotensinogen converted by renin to angiotensin-I and then by ACE to angiotensin-II. Notably, angiotensin-II, via binding to AGTR1, induces a pro-inflammatory state, vasoconstriction, and subsequently fibrosis. ACE2, the known target surface receptor of SARS-CoV-2, converts angiotensin-II to angiotensin 1–7. Angiotensin 1–7 has contradictory effects to angiotensin-II, such as anti-inflammatory and vasodilatory actions [[44](#CR44)]. Binding SARS-CoV-2 to ACE2 can change the ACE/ACE2 balance, which may increase angiotensin-II levels, and further increase its harmful effects, mainly in lung tissue [[45](#CR45), [46](#CR46)]. This process can be the starting point of ARDS and subsequent cytokine overproduction and over-activation, known as the cytokine storm. Patients with underlying diabetes and hypertension show a decreased ACE2 expression state, associated with increased angiotensin-II and a pro-inflammatory condition [[47](#CR47)]. Therefore, a more severe COVID-19 may occur in these patients [[48](#CR48)].

Any alteration in expression and function of RAS elements, for instance, caused by genetic variations, can results in differences in susceptibility to COVID-19 pathogenesis. Previously reported that the C allele (threonine variant) of AGT rs699 is associated with increased plasma angiotensinogen and hypertension [[29](#CR29), [49](#CR49), [50](#CR50)]. Therefore, this variant potentially could be associated with increased susceptibility to COVID-19. Consistent with this hypothesis, we observed that TC genotype and C allele of rs699 were associated with 8.4- and 2.2-fold increase in the risk of COVID-19 compared to TT genotype and T allele, respectively. Although patients with severe disease in our study had a higher prevalence of TC genotype and C allele, we did not observe a statistically significant difference, which could be due to the relatively small sample size.

The ACE rs4646994 is a common indel polymorphism with 287-bp *Alu*-type sequence at the intron 16 position, leading to higher ACE activity and serum levels and angiotensin-II levels in D carriers [[51](#CR51), [52](#CR52)]. This elevated level could be a strong risk factor for cardiovascular and renal diseases [[53](#CR53)]. Also, a positive correlation between the D allele, ACE levels, and ARDS has been observed [[54](#CR54)]. However, its association with hypertension seems to be ethnic-dependent [[55](#CR55)]. Carriers of the D allele are reported to have a higher risk for COVID-19 infection, severity, and outcome [[26](#CR26), [56](#CR56)]. In a study on the Asian population, Pati et al. revealed a positive correlation between the D allele and SARS-CoV-2 infection [[57](#CR57)]. Verma et al. reported that the DD genotype, D allele, diabetes, and hypertension were significantly higher in severe cases of COVID-19 [[58](#CR58)]. Yamamoto et al. also showed that the II genotype has a strong negative correlation with COVID-19 prevalence and death [[59](#CR59)]. Similar to previous findings, we observed a marked decrease of COVID-19 risk in carriers of the I allele in the present study. The presence of the I allele significantly reduced the risk of COVID-19 in all hereditary models, which means that the D allele is a risk factor for COVID-19 infection. Further, we found that II genotype in recessive model, decreases the risk of severe disease. Although not statistically significant, the same trend was observed in codominant model.

The ATGR1 rs5186 variant is one of the most extensively studied polymorphisms associated with essential hypertension [[33](#CR33)]. This polymorphism alters the transcription of impairs the binding of target microRNA (i.e., miR-155), leading to increased expression of the *AGTR1* gene [[60](#CR60)]. Thus, it could be associated with a higher response to angiotensin-II. However, we could not find any significant association between rs5186 and susceptibility to COVID-19 in our studied sample population.

In conclusion, our results suggest ACE rs4646994 and AGT rs699 are associated with the risk of COVID-19 infection and may be used as predictive biomarkers, for instabce the diagnosis of patients at higher risk for developing COVID-19 infection. We found that ACE rs4646994 I allele and AGT C allele are associated with decreased and increased risk of COVID-19 infection, respectively. In addition, age, diabetes, and hypertension are risk factors for severe COVID-19 infection. Further studies with a larger sample size in different populations are required to validate our findings.

## Acknowledgments

This work was funded by a research grant (#9830) from Zahedan University of Medical sciences. We thank Mrs. Asghari for their kind help. The authors thank to the patients and healthy subjects who willingly participated in the study.

### Financial disclosures

The authors have no financial interest related to the material in the manuscript.

## Authors’ contributions

HRK – supervision on data collection, validation of data source and contents, interpretation of data. FT – statistical analysis, writing statistical components, interpretation of data, revising the manuscript. MD – acquisition, data collection and tabulation. MN – supervision on data collection, validation of data source and contents, interpretation of data. GB – methodology, formal analysis. TK – acquisition, data collection and tabulation. CC – writing the original draft. SG – supervision, formal analysis, review and editing the final draft, revising the manuscript. MT – concept and study design, supervision, methodology, formal analysis, writing original draft, review and editing final draft. The author(s) read and approved the final manuscript.

## Funding

This work was funded by a research grant (#9830) from Zahedan University of Medical sciences.

## Availability of data and materials

All data of the manuscript will be provided upon reasonable request and approval by the ethics committee.

## Declarations

### Ethics approval and consent to participate

The study protocol was approved by the Ethics Committee of Zahedan University of Medical Sciences (IR.ZAUMS.REC.1399).

### Consent for publication

All patients signed the informed consent form before participation.

### Competing interests

All authors declared that they have no conflict of interest.

## Footnotes

## Contributor Information

Saeid Ghavami, Email: saeid.ghavami@gmail.com.

Mohsen Taheri, Email: mohsen.taheri.gene@gmail.com, Email: taheri@zaums.ac.ir.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

All data of the manuscript will be provided upon reasonable request and approval by the ethics committee.

### Data Availability Statement

All data of the manuscript will be provided upon reasonable request and approval by the ethics committee.

## References

1. Bellone M, Calvisi SL. ACE polymorphisms and COVID-19-related mortality in Europe. J Mol Med (Berl) 2020;98(11):1505–1509. doi: 10.1007/s00109-020-01981-0.  [DOI](https://doi.org/10.1007/s00109-020-01981-0) | [PMC free article](/articles/PMC7491982/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32935155/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Med%20(Berl)&title=ACE%20polymorphisms%20and%20COVID-19-related%20mortality%20in%20Europe&author=M%20Bellone&author=SL%20Calvisi&volume=98&issue=11&publication_year=2020&pages=1505-1509&pmid=32935155&doi=10.1007/s00109-020-01981-0&)

2. Zhang H, et al. Clinical characteristics of coronavirus disease 2019 (COVID-19) in patients out of Wuhan from China: a case control study. BMC Infect Dis. 2021;21(1):207. doi: 10.1186/s12879-021-05897-z.  [DOI](https://doi.org/10.1186/s12879-021-05897-z) | [PMC free article](/articles/PMC7903394/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33627079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Infect%20Dis&title=Clinical%20characteristics%20of%20coronavirus%20disease%202019%20(COVID-19)%20in%20patients%20out%20of%20Wuhan%20from%20China:%20a%20case%20control%20study&author=H%20Zhang&volume=21&issue=1&publication_year=2021&pages=207&pmid=33627079&doi=10.1186/s12879-021-05897-z&)

3. Lundholm MD, et al. SARS-CoV-2 (COVID-19) and the Endocrine System. J Endocr Soc. 2020;4(11):bvaa144. doi: 10.1210/jendso/bvaa144.  [DOI](https://doi.org/10.1210/jendso/bvaa144) | [PMC free article](/articles/PMC7543511/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33145472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Endocr%20Soc&title=SARS-CoV-2%20(COVID-19)%20and%20the%20Endocrine%20System&author=MD%20Lundholm&volume=4&issue=11&publication_year=2020&pages=bvaa144&pmid=33145472&doi=10.1210/jendso/bvaa144&)

4. Azevedo RB, et al. Covid-19 and the cardiovascular system: a comprehensive review. J Hum Hypertens. 2021;35(1):4–11. doi: 10.1038/s41371-020-0387-4.  [DOI](https://doi.org/10.1038/s41371-020-0387-4) | [PMC free article](/articles/PMC7384729/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32719447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Hypertens&title=Covid-19%20and%20the%20cardiovascular%20system:%20a%20comprehensive%20review&author=RB%20Azevedo&volume=35&issue=1&publication_year=2021&pages=4-11&pmid=32719447&doi=10.1038/s41371-020-0387-4&)

5. Kaye AD, et al. COVID-19 impact on the renal system: pathophysiology and clinical outcomes. Best Pract Res Clin Anaesthesiol. 2021;35(3):449–459. doi: 10.1016/j.bpa.2021.02.004.  [DOI](https://doi.org/10.1016/j.bpa.2021.02.004) | [PMC free article](/articles/PMC7920815/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34511232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Best%20Pract%20Res%20Clin%20Anaesthesiol&title=COVID-19%20impact%20on%20the%20renal%20system:%20pathophysiology%20and%20clinical%20outcomes&author=AD%20Kaye&volume=35&issue=3&publication_year=2021&pages=449-459&pmid=34511232&doi=10.1016/j.bpa.2021.02.004&)

6. Iadecola C, Anrather J, Kamel H. Effects of COVID-19 on the Nervous System. Cell. 2020;183(1):16–27. doi: 10.1016/j.cell.2020.08.028.  [DOI](https://doi.org/10.1016/j.cell.2020.08.028) | [PMC free article](/articles/PMC7437501/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32882182/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Effects%20of%20COVID-19%20on%20the%20Nervous%20System&author=C%20Iadecola&author=J%20Anrather&author=H%20Kamel&volume=183&issue=1&publication_year=2020&pages=16-27&pmid=32882182&doi=10.1016/j.cell.2020.08.028&)

7. Jayaweera M, et al. Transmission of COVID-19 virus by droplets and aerosols: a critical review on the unresolved dichotomy. Environ Res. 2020;188:109819. doi: 10.1016/j.envres.2020.109819.  [DOI](https://doi.org/10.1016/j.envres.2020.109819) | [PMC free article](/articles/PMC7293495/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32569870/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ%20Res&title=Transmission%20of%20COVID-19%20virus%20by%20droplets%20and%20aerosols:%20a%20critical%20review%20on%20the%20unresolved%20dichotomy&author=M%20Jayaweera&volume=188&publication_year=2020&pages=109819&pmid=32569870&doi=10.1016/j.envres.2020.109819&)

8. Sanche S, et al. High contagiousness and rapid spread of severe acute respiratory syndrome coronavirus 2. Emerg Infect Dis. 2020;26(7):1470. doi: 10.3201/eid2607.200282.  [DOI](https://doi.org/10.3201/eid2607.200282) | [PMC free article](/articles/PMC7323562/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32255761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Emerg%20Infect%20Dis&title=High%20contagiousness%20and%20rapid%20spread%20of%20severe%20acute%20respiratory%20syndrome%20coronavirus%202&author=S%20Sanche&volume=26&issue=7&publication_year=2020&pages=1470&pmid=32255761&doi=10.3201/eid2607.200282&)

9. Peymani P, et al. Statins in patients with COVID-19: a retrospective cohort study in Iranian COVID-19 patients. Transl Med Commun. 2021;6(1):3. doi: 10.1186/s41231-021-00082-5.  [DOI](https://doi.org/10.1186/s41231-021-00082-5) | [PMC free article](/articles/PMC7829327/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33521322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transl%20Med%20Commun&title=Statins%20in%20patients%20with%20COVID-19:%20a%20retrospective%20cohort%20study%20in%20Iranian%20COVID-19%20patients&author=P%20Peymani&volume=6&issue=1&publication_year=2021&pages=3&pmid=33521322&doi=10.1186/s41231-021-00082-5&)

10. Parit R, Jayavel S. Association of ACE inhibitors and angiotensin type II blockers with ACE2 overexpression in COVID-19 comorbidities: a pathway-based analytical study. Eur J Pharmacol. 2021;896:173899. doi: 10.1016/j.ejphar.2021.173899.  [DOI](https://doi.org/10.1016/j.ejphar.2021.173899) | [PMC free article](/articles/PMC7839513/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33508281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&title=Association%20of%20ACE%20inhibitors%20and%20angiotensin%20type%20II%20blockers%20with%20ACE2%20overexpression%20in%20COVID-19%20comorbidities:%20a%20pathway-based%20analytical%20study&author=R%20Parit&author=S%20Jayavel&volume=896&publication_year=2021&pages=173899&pmid=33508281&doi=10.1016/j.ejphar.2021.173899&)

11. Hoffmann M, et al. SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor. Cell. 2020;181(2):271–280. doi: 10.1016/j.cell.2020.02.052.  [DOI](https://doi.org/10.1016/j.cell.2020.02.052) | [PMC free article](/articles/PMC7102627/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32142651/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=SARS-CoV-2%20Cell%20Entry%20Depends%20on%20ACE2%20and%20TMPRSS2%20and%20Is%20Blocked%20by%20a%20Clinically%20Proven%20Protease%20Inhibitor&author=M%20Hoffmann&volume=181&issue=2&publication_year=2020&pages=271-280&pmid=32142651&doi=10.1016/j.cell.2020.02.052&)

12. Siri M, et al. Autophagy, Unfolded Protein Response, and Neuropilin-1 Cross-Talk in SARS-CoV-2 Infection: What Can Be Learned from Other Coronaviruses. Int J Mol Sci. 2021;22(11):5992. doi: 10.3390/ijms22115992.  [DOI](https://doi.org/10.3390/ijms22115992) | [PMC free article](/articles/PMC8199451/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34206057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&title=Autophagy,%20Unfolded%20Protein%20Response,%20and%20Neuropilin-1%20Cross-Talk%20in%20SARS-CoV-2%20Infection:%20What%20Can%20Be%20Learned%20from%20Other%20Coronaviruses&author=M%20Siri&volume=22&issue=11&publication_year=2021&pages=5992&pmid=34206057&doi=10.3390/ijms22115992&)

13. Peng R, et al. Cell entry by SARS-CoV-2. Trends Biochem Sci. 2021;46(10):848–860. doi: 10.1016/j.tibs.2021.06.001.  [DOI](https://doi.org/10.1016/j.tibs.2021.06.001) | [PMC free article](/articles/PMC8180548/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34187722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Biochem%20Sci&title=Cell%20entry%20by%20SARS-CoV-2&author=R%20Peng&volume=46&issue=10&publication_year=2021&pages=848-860&pmid=34187722&doi=10.1016/j.tibs.2021.06.001&)

14. Lu J, Sun PD. High affinity binding of SARS-CoV-2 spike protein enhances ACE2 carboxypeptidase activity. J Biol Chem. 2020;295(52):18579–18588. doi: 10.1074/jbc.RA120.015303.  [DOI](https://doi.org/10.1074/jbc.RA120.015303) | [PMC free article](/articles/PMC7833600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33122196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=High%20affinity%20binding%20of%20SARS-CoV-2%20spike%20protein%20enhances%20ACE2%20carboxypeptidase%20activity&author=J%20Lu&author=PD%20Sun&volume=295&issue=52&publication_year=2020&pages=18579-18588&pmid=33122196&doi=10.1074/jbc.RA120.015303&)

15. Sharma P, et al. Chloroquine: autophagy inhibitor, antimalarial, bitter taste receptor agonist in fight against COVID-19, a reality check? Eur J Pharmacol. 2021;897:173928. doi: 10.1016/j.ejphar.2021.173928.  [DOI](https://doi.org/10.1016/j.ejphar.2021.173928) | [PMC free article](/articles/PMC7857018/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33545161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&title=Chloroquine:%20autophagy%20inhibitor,%20antimalarial,%20bitter%20taste%20receptor%20agonist%20in%20fight%20against%20COVID-19,%20a%20reality%20check?&author=P%20Sharma&volume=897&publication_year=2021&pages=173928&pmid=33545161&doi=10.1016/j.ejphar.2021.173928&)

16. Li W, et al. Angiotensin-converting enzyme 2 is a functional receptor for the SARS coronavirus. Nature. 2003;426(6965):450–454. doi: 10.1038/nature02145.  [DOI](https://doi.org/10.1038/nature02145) | [PMC free article](/articles/PMC7095016/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14647384/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Angiotensin-converting%20enzyme%202%20is%20a%20functional%20receptor%20for%20the%20SARS%20coronavirus&author=W%20Li&volume=426&issue=6965&publication_year=2003&pages=450-454&pmid=14647384&doi=10.1038/nature02145&)

17. Shang J, et al. Structural basis of receptor recognition by SARS-CoV-2. Nature. 2020;581(7807):221–224. doi: 10.1038/s41586-020-2179-y.  [DOI](https://doi.org/10.1038/s41586-020-2179-y) | [PMC free article](/articles/PMC7328981/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32225175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Structural%20basis%20of%20receptor%20recognition%20by%20SARS-CoV-2&author=J%20Shang&volume=581&issue=7807&publication_year=2020&pages=221-224&pmid=32225175&doi=10.1038/s41586-020-2179-y&)

18. Sriram K, Insel PA. A hypothesis for pathobiology and treatment of COVID-19: the centrality of ACE1/ACE2 imbalance. Br J Pharmacol. 2020;177(21):4825–4844. doi: 10.1111/bph.15082.  [DOI](https://doi.org/10.1111/bph.15082) | [PMC free article](/articles/PMC7572451/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32333398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pharmacol&title=A%20hypothesis%20for%20pathobiology%20and%20treatment%20of%20COVID-19:%20the%20centrality%20of%20ACE1/ACE2%20imbalance&author=K%20Sriram&author=PA%20Insel&volume=177&issue=21&publication_year=2020&pages=4825-4844&pmid=32333398&doi=10.1111/bph.15082&)

19. Sriram K, Loomba R, Insel PA. Targeting the renin− angiotensin signaling pathway in COVID-19: unanswered questions, opportunities, and challenges. Proc Natl Acad Sci. 2020;117(47):29274–29282. doi: 10.1073/pnas.2009875117.  [DOI](https://doi.org/10.1073/pnas.2009875117) | [PMC free article](/articles/PMC7703541/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33203679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci&title=Targeting%20the%20renin%E2%88%92%20angiotensin%20signaling%20pathway%20in%20COVID-19:%20unanswered%20questions,%20opportunities,%20and%20challenges&author=K%20Sriram&author=R%20Loomba&author=PA%20Insel&volume=117&issue=47&publication_year=2020&pages=29274-29282&pmid=33203679&doi=10.1073/pnas.2009875117&)

20. Drozdzal S, et al. FDA approved drugs with pharmacotherapeutic potential for SARS-CoV-2 (COVID-19) therapy. Drug Resist Updat. 2020;53:100719. doi: 10.1016/j.drup.2020.100719.  [DOI](https://doi.org/10.1016/j.drup.2020.100719) | [PMC free article](/articles/PMC7362818/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32717568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Resist%20Updat&title=FDA%20approved%20drugs%20with%20pharmacotherapeutic%20potential%20for%20SARS-CoV-2%20(COVID-19)%20therapy&author=S%20Drozdzal&volume=53&publication_year=2020&pages=100719&pmid=32717568&doi=10.1016/j.drup.2020.100719&)

21. Nabavi SF, et al. Rationale for effective prophylaxis against COVID-19 through simultaneous blockade of both Endosomal and non-Endosomal SARS-CoV-2 entry into host cell. Clin Transl Sci. 2021;14(2):431–433. doi: 10.1111/cts.12949.  [DOI](https://doi.org/10.1111/cts.12949) | [PMC free article](/articles/PMC7993271/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33406317/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Rationale%20for%20effective%20prophylaxis%20against%20COVID-19%20through%20simultaneous%20blockade%20of%20both%20Endosomal%20and%20non-Endosomal%20SARS-CoV-2%20entry%20into%20host%20cell&author=SF%20Nabavi&volume=14&issue=2&publication_year=2021&pages=431-433&pmid=33406317&doi=10.1111/cts.12949&)

22. Darbeheshti F, et al. Coronavirus: pure infectious disease or genetic predisposition. Adv Exp Med Biol. 2021;1318:91–107. doi: 10.1007/978-3-030-63761-3_6.  [DOI](https://doi.org/10.1007/978-3-030-63761-3_6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33973174/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Exp%20Med%20Biol&title=Coronavirus:%20pure%20infectious%20disease%20or%20genetic%20predisposition&author=F%20Darbeheshti&volume=1318&publication_year=2021&pages=91-107&pmid=33973174&doi=10.1007/978-3-030-63761-3_6&)

23. Forrester SJ, et al. Angiotensin II signal transduction: an update on mechanisms of physiology and pathophysiology. Physiol Rev. 2018;98(3):1627–1738. doi: 10.1152/physrev.00038.2017.  [DOI](https://doi.org/10.1152/physrev.00038.2017) | [PMC free article](/articles/PMC6335102/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29873596/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Physiol%20Rev&title=Angiotensin%20II%20signal%20transduction:%20an%20update%20on%20mechanisms%20of%20physiology%20and%20pathophysiology&author=SJ%20Forrester&volume=98&issue=3&publication_year=2018&pages=1627-1738&pmid=29873596&doi=10.1152/physrev.00038.2017&)

24. Haga S, et al. Modulation of TNF-α-converting enzyme by the spike protein of SARS-CoV and ACE2 induces TNF-α production and facilitates viral entry. Proc Natl Acad Sci. 2008;105(22):7809–7814. doi: 10.1073/pnas.0711241105.  [DOI](https://doi.org/10.1073/pnas.0711241105) | [PMC free article](/articles/PMC2409424/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18490652/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci&title=Modulation%20of%20TNF-%CE%B1-converting%20enzyme%20by%20the%20spike%20protein%20of%20SARS-CoV%20and%20ACE2%20induces%20TNF-%CE%B1%20production%20and%20facilitates%20viral%20entry&author=S%20Haga&volume=105&issue=22&publication_year=2008&pages=7809-7814&pmid=18490652&doi=10.1073/pnas.0711241105&)

25. Burgner D, Jamieson SE, Blackwell JM. Genetic susceptibility to infectious diseases: big is beautiful, but will bigger be even better? Lancet Infect Dis. 2006;6(10):653–663. doi: 10.1016/S1473-3099(06)70601-6.  [DOI](https://doi.org/10.1016/S1473-3099(06)70601-6) | [PMC free article](/articles/PMC2330096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17008174/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Infect%20Dis&title=Genetic%20susceptibility%20to%20infectious%20diseases:%20big%20is%20beautiful,%20but%20will%20bigger%20be%20even%20better?&author=D%20Burgner&author=SE%20Jamieson&author=JM%20Blackwell&volume=6&issue=10&publication_year=2006&pages=653-663&pmid=17008174&doi=10.1016/S1473-3099(06)70601-6&)

26. Delanghe JR, Speeckaert MM, De Buyzere ML. The host's angiotensin-converting enzyme polymorphism may explain epidemiological findings in COVID-19 infections. Clin Chim Acta. 2020;505:192–193. doi: 10.1016/j.cca.2020.03.031.  [DOI](https://doi.org/10.1016/j.cca.2020.03.031) | [PMC free article](/articles/PMC7102561/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32220422/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=The%20host's%20angiotensin-converting%20enzyme%20polymorphism%20may%20explain%20epidemiological%20findings%20in%20COVID-19%20infections&author=JR%20Delanghe&author=MM%20Speeckaert&author=ML%20De%20Buyzere&volume=505&publication_year=2020&pages=192-193&pmid=32220422&doi=10.1016/j.cca.2020.03.031&)

27. Corvol P, Jeunemaitre X. Molecular genetics of human hypertension: role of angiotensinogen*. Endocr Rev. 1997;18(5):662–677. doi: 10.1210/edrv.18.5.0312.  [DOI](https://doi.org/10.1210/edrv.18.5.0312) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9331547/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocr%20Rev&title=Molecular%20genetics%20of%20human%20hypertension:%20role%20of%20angiotensinogen*&author=P%20Corvol&author=X%20Jeunemaitre&volume=18&issue=5&publication_year=1997&pages=662-677&pmid=9331547&doi=10.1210/edrv.18.5.0312&)

28. Brand E, et al. Detection of putative functional angiotensinogen (AGT) gene variants controlling plasma AGT levels by combined segregation-linkage analysis. Eur J Hum Genet. 2002;10(11):715–723. doi: 10.1038/sj.ejhg.5200874.  [DOI](https://doi.org/10.1038/sj.ejhg.5200874) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12404103/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Detection%20of%20putative%20functional%20angiotensinogen%20(AGT)%20gene%20variants%20controlling%20plasma%20AGT%20levels%20by%20combined%20segregation-linkage%20analysis&author=E%20Brand&volume=10&issue=11&publication_year=2002&pages=715-723&pmid=12404103&doi=10.1038/sj.ejhg.5200874&)

29. Sethi AA, Nordestgaard BG, Tybjærg-Hansen A. Angiotensinogen gene polymorphism, plasma angiotensinogen, and risk of hypertension and ischemic heart disease: a meta-analysis. Arterioscler Thromb Vasc Biol. 2003;23(7):1269–1275. doi: 10.1161/01.ATV.0000079007.40884.5C.  [DOI](https://doi.org/10.1161/01.ATV.0000079007.40884.5C) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12805070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arterioscler%20Thromb%20Vasc%20Biol&title=Angiotensinogen%20gene%20polymorphism,%20plasma%20angiotensinogen,%20and%20risk%20of%20hypertension%20and%20ischemic%20heart%20disease:%20a%20meta-analysis&author=AA%20Sethi&author=BG%20Nordestgaard&author=A%20Tybj%C3%A6rg-Hansen&volume=23&issue=7&publication_year=2003&pages=1269-1275&pmid=12805070&doi=10.1161/01.ATV.0000079007.40884.5C&)

30. Rigat B, et al. An insertion/deletion polymorphism in the angiotensin I-converting enzyme gene accounting for half the variance of serum enzyme levels. J Clin Invest. 1990;86(4):1343–1346. doi: 10.1172/JCI114844.  [DOI](https://doi.org/10.1172/JCI114844) | [PMC free article](/articles/PMC296868/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1976655/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=An%20insertion/deletion%20polymorphism%20in%20the%20angiotensin%20I-converting%20enzyme%20gene%20accounting%20for%20half%20the%20variance%20of%20serum%20enzyme%20levels&author=B%20Rigat&volume=86&issue=4&publication_year=1990&pages=1343-1346&pmid=1976655&doi=10.1172/JCI114844&)

31. Rigat B, et al. PCR detection of the insertion/deletion polymorphism of the human angiotensin converting enzyme gene (DCP1)(dipeptidyl carboxypeptidase 1) Nucleic Acids Res. 1992;20(6):1433. doi: 10.1093/nar/20.6.1433-a.  [DOI](https://doi.org/10.1093/nar/20.6.1433-a) | [PMC free article](/articles/PMC312213/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1313972/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=PCR%20detection%20of%20the%20insertion/deletion%20polymorphism%20of%20the%20human%20angiotensin%20converting%20enzyme%20gene%20(DCP1)(dipeptidyl%20carboxypeptidase%201)&author=B%20Rigat&volume=20&issue=6&publication_year=1992&pages=1433&pmid=1313972&doi=10.1093/nar/20.6.1433-a&)

32. Abdollahi MR, et al. Quantitated transcript haplotypes (QTH) of AGTR1, reduced abundance of mRNA haplotypes containing 1166C (rs5186: a> C), and relevance to metabolic syndrome traits. Hum Mutat. 2007;28(4):365–373. doi: 10.1002/humu.20454.  [DOI](https://doi.org/10.1002/humu.20454) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17211857/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mutat&title=Quantitated%20transcript%20haplotypes%20(QTH)%20of%20AGTR1,%20reduced%20abundance%20of%20mRNA%20haplotypes%20containing%201166C%20(rs5186:%20a>%20C),%20and%20relevance%20to%20metabolic%20syndrome%20traits&author=MR%20Abdollahi&volume=28&issue=4&publication_year=2007&pages=365-373&pmid=17211857&doi=10.1002/humu.20454&)

33. Mottl AK, Shoham DA, North KE. Angiotensin II type 1 receptor polymorphisms and susceptibility to hypertension: a HuGE review. Genet Med. 2008;10(8):560–574. doi: 10.1097/GIM.0b013e3181809613.  [DOI](https://doi.org/10.1097/GIM.0b013e3181809613) | [PMC free article](/articles/PMC4993203/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18641512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=Angiotensin%20II%20type%201%20receptor%20polymorphisms%20and%20susceptibility%20to%20hypertension:%20a%20HuGE%20review&author=AK%20Mottl&author=DA%20Shoham&author=KE%20North&volume=10&issue=8&publication_year=2008&pages=560-574&pmid=18641512&doi=10.1097/GIM.0b013e3181809613&)

34. Hashemi SMA, et al. Human gene polymorphisms and their possible impact on the clinical outcome of SARS-CoV-2 infection. Arch Virol. 2021;166(8):2089–108.  [DOI](https://doi.org/10.1007/s00705-021-05070-6) | [PMC free article](/articles/PMC8088757/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33934196/)

35. Organization, W.H. COVID-19 clinical management: living guidance, 25 January 2021. Geneva: World Health Organization; 2021.

36. Hu Y, et al. Prevalence and severity of corona virus disease 2019 (COVID-19): a systematic review and meta-analysis. J Clin Virol. 2020;127:104371. doi: 10.1016/j.jcv.2020.104371.  [DOI](https://doi.org/10.1016/j.jcv.2020.104371) | [PMC free article](/articles/PMC7195434/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32315817/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Virol&title=Prevalence%20and%20severity%20of%20corona%20virus%20disease%202019%20(COVID-19):%20a%20systematic%20review%20and%20meta-analysis&author=Y%20Hu&volume=127&publication_year=2020&pages=104371&pmid=32315817&doi=10.1016/j.jcv.2020.104371&)

37. Singhavi H, et al. SARS-Cov2: a meta-analysis of symptom distribution by continent in 7310 adult COVID-19 infected patients. Virusdisease. 2021;32(3):400–9.  [DOI](https://doi.org/10.1007/s13337-021-00699-y) | [PMC free article](/articles/PMC8187893/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34124318/)

38. Vahedi A, et al. Clinical features and outcomes of ICU patients with COVID-19 infection in Tehran, Iran: a single-centered retrospective cohort study. Tanaffos. 2020;19(4):300–311.  [PMC free article](/articles/PMC8088143/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33959167/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tanaffos&title=Clinical%20features%20and%20outcomes%20of%20ICU%20patients%20with%20COVID-19%20infection%20in%20Tehran,%20Iran:%20a%20single-centered%20retrospective%20cohort%20study&author=A%20Vahedi&volume=19&issue=4&publication_year=2020&pages=300-311&pmid=33959167&)

39. Incerti D, et al. Prognostic model to identify and quantify risk factors for mortality among hospitalised patients with COVID-19 in the USA. BMJ Open. 2021;11(4):e047121. doi: 10.1136/bmjopen-2020-047121.  [DOI](https://doi.org/10.1136/bmjopen-2020-047121) | [PMC free article](/articles/PMC8029269/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33827848/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ%20Open&title=Prognostic%20model%20to%20identify%20and%20quantify%20risk%20factors%20for%20mortality%20among%20hospitalised%20patients%20with%20COVID-19%20in%20the%20USA&author=D%20Incerti&volume=11&issue=4&publication_year=2021&pages=e047121&pmid=33827848&doi=10.1136/bmjopen-2020-047121&)

40. McGurnaghan SJ, et al. Risks of and risk factors for COVID-19 disease in people with diabetes: a cohort study of the total population of Scotland. Lancet Diabetes Endocrinol. 2021;9(2):82–93. doi: 10.1016/S2213-8587(20)30405-8.  [DOI](https://doi.org/10.1016/S2213-8587(20)30405-8) | [PMC free article](/articles/PMC7832778/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33357491/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Diabetes%20Endocrinol&title=Risks%20of%20and%20risk%20factors%20for%20COVID-19%20disease%20in%20people%20with%20diabetes:%20a%20cohort%20study%20of%20the%20total%20population%20of%20Scotland&author=SJ%20McGurnaghan&volume=9&issue=2&publication_year=2021&pages=82-93&pmid=33357491&doi=10.1016/S2213-8587(20)30405-8&)

41. Gao YD, et al. Risk factors for severe and critically ill COVID-19 patients: a review. Allergy. 2021;76(2):428–455. doi: 10.1111/all.14657.  [DOI](https://doi.org/10.1111/all.14657) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33185910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Allergy&title=Risk%20factors%20for%20severe%20and%20critically%20ill%20COVID-19%20patients:%20a%20review&author=YD%20Gao&volume=76&issue=2&publication_year=2021&pages=428-455&pmid=33185910&doi=10.1111/all.14657&)

42. Vrigkou E, et al. The evolving role of the renin-angiotensin system in ARDS. Crit Care. 2017;21(1):329. doi: 10.1186/s13054-017-1917-5.  [DOI](https://doi.org/10.1186/s13054-017-1917-5) | [PMC free article](/articles/PMC5746002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29284527/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit%20Care&title=The%20evolving%20role%20of%20the%20renin-angiotensin%20system%20in%20ARDS&author=E%20Vrigkou&volume=21&issue=1&publication_year=2017&pages=329&pmid=29284527&doi=10.1186/s13054-017-1917-5&)

43. Henry BM, et al. Hyperinflammation and derangement of renin-angiotensin-aldosterone system in COVID-19: A novel hypothesis for clinically suspected hypercoagulopathy and microvascular immunothrombosis. Clin Chim Acta. 2020;507:167–173. doi: 10.1016/j.cca.2020.04.027.  [DOI](https://doi.org/10.1016/j.cca.2020.04.027) | [PMC free article](/articles/PMC7195008/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32348783/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=Hyperinflammation%20and%20derangement%20of%20renin-angiotensin-aldosterone%20system%20in%20COVID-19:%20A%20novel%20hypothesis%20for%20clinically%20suspected%20hypercoagulopathy%20and%20microvascular%20immunothrombosis&author=BM%20Henry&volume=507&publication_year=2020&pages=167-173&pmid=32348783&doi=10.1016/j.cca.2020.04.027&)

44. Bader M. ACE2, angiotensin-(1–7), and mas: the other side of the coin. Pflügers Arch. 2013;465(1):79–85. doi: 10.1007/s00424-012-1120-0.  [DOI](https://doi.org/10.1007/s00424-012-1120-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23463883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pfl%C3%BCgers%20Arch&title=ACE2,%20angiotensin-(1%E2%80%937),%20and%20mas:%20the%20other%20side%20of%20the%20coin&author=M%20Bader&volume=465&issue=1&publication_year=2013&pages=79-85&pmid=23463883&doi=10.1007/s00424-012-1120-0&)

45. Tikellis C, Thomas M. Angiotensin-converting enzyme 2 (ACE2) is a key modulator of the renin angiotensin system in health and disease. Int J Pept. 2012;2012:1–8.  [DOI](https://doi.org/10.1155/2012/256294) | [PMC free article](/articles/PMC3321295/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22536270/)

46. Liu Y, et al. Clinical and biochemical indexes from 2019-nCoV infected patients linked to viral loads and lung injury. Sci China Life Sci. 2020;63(3):364–374. doi: 10.1007/s11427-020-1643-8.  [DOI](https://doi.org/10.1007/s11427-020-1643-8) | [PMC free article](/articles/PMC7088566/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32048163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20China%20Life%20Sci&title=Clinical%20and%20biochemical%20indexes%20from%202019-nCoV%20infected%20patients%20linked%20to%20viral%20loads%20and%20lung%20injury&author=Y%20Liu&volume=63&issue=3&publication_year=2020&pages=364-374&pmid=32048163&doi=10.1007/s11427-020-1643-8&)

47. Di Raimondo D, et al. Effects of ACE-inhibitors and angiotensin receptor blockers on inflammation. Curr Pharm Des. 2012;18(28):4385–4413. doi: 10.2174/138161212802481282.  [DOI](https://doi.org/10.2174/138161212802481282) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22283779/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Pharm%20Des&title=Effects%20of%20ACE-inhibitors%20and%20angiotensin%20receptor%20blockers%20on%20inflammation&author=D%20Di%20Raimondo&volume=18&issue=28&publication_year=2012&pages=4385-4413&pmid=22283779&doi=10.2174/138161212802481282&)

48. Guan W-J, et al. Clinical characteristics of coronavirus disease 2019 in China. N Engl J Med. 2020;382(18):1708–1720. doi: 10.1056/NEJMoa2002032.  [DOI](https://doi.org/10.1056/NEJMoa2002032) | [PMC free article](/articles/PMC7092819/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32109013/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Clinical%20characteristics%20of%20coronavirus%20disease%202019%20in%20China&author=W-J%20Guan&volume=382&issue=18&publication_year=2020&pages=1708-1720&pmid=32109013&doi=10.1056/NEJMoa2002032&)

49. Takeuchi F, et al. Reevaluation of the association of seven candidate genes with blood pressure and hypertension: a replication study and meta-analysis with a larger sample size. Hypertens Res. 2012;35(8):825–831. doi: 10.1038/hr.2012.43.  [DOI](https://doi.org/10.1038/hr.2012.43) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22456346/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertens%20Res&title=Reevaluation%20of%20the%20association%20of%20seven%20candidate%20genes%20with%20blood%20pressure%20and%20hypertension:%20a%20replication%20study%20and%20meta-analysis%20with%20a%20larger%20sample%20size&author=F%20Takeuchi&volume=35&issue=8&publication_year=2012&pages=825-831&pmid=22456346&doi=10.1038/hr.2012.43&)

50. Jeunemaitre X, et al. Molecular basis of human hypertension: role of angiotensinogen. Cell. 1992;71(1):169–180. doi: 10.1016/0092-8674(92)90275-H.  [DOI](https://doi.org/10.1016/0092-8674(92)90275-H) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1394429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Molecular%20basis%20of%20human%20hypertension:%20role%20of%20angiotensinogen&author=X%20Jeunemaitre&volume=71&issue=1&publication_year=1992&pages=169-180&pmid=1394429&doi=10.1016/0092-8674(92)90275-H&)

51. Staessen JA, et al. The deletion/insertion polymorphism of the angiotensin converting enzyme gene and cardiovascular-renal risk. J Hypertens. 1997;15(12):1579–1592. doi: 10.1097/00004872-199715120-00059.  [DOI](https://doi.org/10.1097/00004872-199715120-00059) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9488209/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hypertens&title=The%20deletion/insertion%20polymorphism%20of%20the%20angiotensin%20converting%20enzyme%20gene%20and%20cardiovascular-renal%20risk&author=JA%20Staessen&volume=15&issue=12&publication_year=1997&pages=1579-1592&pmid=9488209&doi=10.1097/00004872-199715120-00059&)

52. Danser AJ, et al. ACE phenotyping as a first step toward personalized medicine for ACE inhibitors. Why does ACE genotyping not predict the therapeutic efficacy of ACE inhibition? Pharmacol Ther. 2007;113(3):607–618. doi: 10.1016/j.pharmthera.2006.12.001.  [DOI](https://doi.org/10.1016/j.pharmthera.2006.12.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17257685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Ther&title=ACE%20phenotyping%20as%20a%20first%20step%20toward%20personalized%20medicine%20for%20ACE%20inhibitors.%20Why%20does%20ACE%20genotyping%20not%20predict%20the%20therapeutic%20efficacy%20of%20ACE%20inhibition?&author=AJ%20Danser&volume=113&issue=3&publication_year=2007&pages=607-618&pmid=17257685&doi=10.1016/j.pharmthera.2006.12.001&)

53. Rudnicki M, Mayer G. Significance of genetic polymorphisms of the renin-angiotensin-aldosterone system in cardiovascular and renal disease. Pharmacogenomics. 2009;10(3):463–476. doi: 10.2217/14622416.10.3.463.  [DOI](https://doi.org/10.2217/14622416.10.3.463) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19290794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Significance%20of%20genetic%20polymorphisms%20of%20the%20renin-angiotensin-aldosterone%20system%20in%20cardiovascular%20and%20renal%20disease&author=M%20Rudnicki&author=G%20Mayer&volume=10&issue=3&publication_year=2009&pages=463-476&pmid=19290794&doi=10.2217/14622416.10.3.463&)

54. Marshall RP, et al. Angiotensin converting enzyme insertion/deletion polymorphism is associated with susceptibility and outcome in acute respiratory distress syndrome. Am J Respir Crit Care Med. 2002;166(5):646–650. doi: 10.1164/rccm.2108086.  [DOI](https://doi.org/10.1164/rccm.2108086) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12204859/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Respir%20Crit%20Care%20Med&title=Angiotensin%20converting%20enzyme%20insertion/deletion%20polymorphism%20is%20associated%20with%20susceptibility%20and%20outcome%20in%20acute%20respiratory%20distress%20syndrome&author=RP%20Marshall&volume=166&issue=5&publication_year=2002&pages=646-650&pmid=12204859&doi=10.1164/rccm.2108086&)

55. Ned RM, et al. The ACE I/D polymorphism in US adults: limited evidence of association with hypertension-related traits and sex-specific effects by race/ethnicity. Am J Hypertens. 2012;25(2):209–215. doi: 10.1038/ajh.2011.182.  [DOI](https://doi.org/10.1038/ajh.2011.182) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21993364/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hypertens&title=The%20ACE%20I/D%20polymorphism%20in%20US%20adults:%20limited%20evidence%20of%20association%20with%20hypertension-related%20traits%20and%20sex-specific%20effects%20by%20race/ethnicity&author=RM%20Ned&volume=25&issue=2&publication_year=2012&pages=209-215&pmid=21993364&doi=10.1038/ajh.2011.182&)

56. Sarangarajan R, et al. Ethnic prevalence of angiotensin-converting enzyme deletion (D) polymorphism and COVID-19 risk: rationale for use of angiotensin-converting enzyme inhibitors/angiotensin receptor blockers. J Racial Ethn Health Disparities. 2021;8(4):973–980. doi: 10.1007/s40615-020-00853-0.  [DOI](https://doi.org/10.1007/s40615-020-00853-0) | [PMC free article](/articles/PMC7478439/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32901433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Racial%20Ethn%20Health%20Disparities&title=Ethnic%20prevalence%20of%20angiotensin-converting%20enzyme%20deletion%20(D)%20polymorphism%20and%20COVID-19%20risk:%20rationale%20for%20use%20of%20angiotensin-converting%20enzyme%20inhibitors/angiotensin%20receptor%20blockers&author=R%20Sarangarajan&volume=8&issue=4&publication_year=2021&pages=973-980&pmid=32901433&doi=10.1007/s40615-020-00853-0&)

57. Pati A, et al. ACE deletion allele is associated with susceptibility to SARS-CoV-2 infection and mortality rate: an epidemiological study in the Asian population. Clin Chim Acta. 2020;510:455–458. doi: 10.1016/j.cca.2020.08.008.  [DOI](https://doi.org/10.1016/j.cca.2020.08.008) | [PMC free article](/articles/PMC7416133/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32791137/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=ACE%20deletion%20allele%20is%20associated%20with%20susceptibility%20to%20SARS-CoV-2%20infection%20and%20mortality%20rate:%20an%20epidemiological%20study%20in%20the%20Asian%20population&author=A%20Pati&volume=510&publication_year=2020&pages=455-458&pmid=32791137&doi=10.1016/j.cca.2020.08.008&)

58. Verma S, et al. Impact of I/D polymorphism of angiotensin-converting enzyme 1 (ACE1) gene on the severity of COVID-19 patients. Infect Genet Evol. 2021;91:104801. doi: 10.1016/j.meegid.2021.104801.  [DOI](https://doi.org/10.1016/j.meegid.2021.104801) | [PMC free article](/articles/PMC7929788/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33676010/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Infect%20Genet%20Evol&title=Impact%20of%20I/D%20polymorphism%20of%20angiotensin-converting%20enzyme%201%20(ACE1)%20gene%20on%20the%20severity%20of%20COVID-19%20patients&author=S%20Verma&volume=91&publication_year=2021&pages=104801&pmid=33676010&doi=10.1016/j.meegid.2021.104801&)

59. Yamamoto N, et al. SARS-CoV-2 infections and COVID-19 mortalities strongly correlate with ACE1 I/D genotype. Gene. 2020;758:144944. doi: 10.1016/j.gene.2020.144944.  [DOI](https://doi.org/10.1016/j.gene.2020.144944) | [PMC free article](/articles/PMC7833925/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32628976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=SARS-CoV-2%20infections%20and%20COVID-19%20mortalities%20strongly%20correlate%20with%20ACE1%20I/D%20genotype&author=N%20Yamamoto&volume=758&publication_year=2020&pages=144944&pmid=32628976&doi=10.1016/j.gene.2020.144944&)

60. Sethupathy P, et al. Human microRNA-155 on chromosome 21 differentially interacts with its polymorphic target in the AGTR1 3′ untranslated region: a mechanism for functional single-nucleotide polymorphisms related to phenotypes. Am J Hum Genet. 2007;81(2):405–413. doi: 10.1086/519979.  [DOI](https://doi.org/10.1086/519979) | [PMC free article](/articles/PMC1950808/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17668390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Human%20microRNA-155%20on%20chromosome%2021%20differentially%20interacts%20with%20its%20polymorphic%20target%20in%20the%20AGTR1%203%E2%80%B2%20untranslated%20region:%20a%20mechanism%20for%20functional%20single-nucleotide%20polymorphisms%20related%20to%20phenotypes&author=P%20Sethupathy&volume=81&issue=2&publication_year=2007&pages=405-413&pmid=17668390&doi=10.1086/519979&)
