# Influence of genetic variants of opioid-related genes on opioid-induced adverse effects in patients with lung cancer: A STROBE-compliant observational study

## Metadata
**Authors:** Rei Tanaka, Junya Sato, Hiroshi Ishikawa, Tetsu Sato, Michihiro Shino, Yasuhisa Ohde, Tetsumi Sato, Keita Mori, Akifumi Notsu, Sumiko Ohnami, Maki Mizuguchi, Takeshi Nagashima, Ken Yamaguchi
**Journal:** Medicine
**Date:** 2021 Nov 5
**DOI:** [10.1097/MD.0000000000027565](https://doi.org/10.1097/MD.0000000000027565)
**PMID:** 34871222
**PMCID:** PMC8568420
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8568420/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8568420/pdf/medi-100-e27565.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8568420/pdf/medi-100-e27565.pdf)

## Abstract

Despite the dramatic advancement of cancer chemotherapy and immunotherapy, the insufficient progress has been made in basic or translational research on personalization of opioid therapy. Predicting the effectiveness of opioid analgesic therapy and the risk of adverse effects prior to therapy are expected to enable safer and more appropriate opioid therapy for cancer patients. In this study, we compared the incidence of opioid-induced adverse effects between patients with different variants of the genes related to responsiveness to opioid analgesics.

Participants were 88 patients with lung cancer who provided general consent for exome sequencing and were treated with morphine or oxycodone at Shizuoka Cancer Center Hospital between April 2014 and August 2018. Incidence rates for 6 adverse effects of opioid therapy (somnolence, nausea, constipation, delirium, urinary retention, and pruritus) were determined and the influence of single nucleotide polymorphisms in coding regions of the opioid μ receptor 1 (OPRM1) (rs1799971), opioid δ receptor 1 (rs2234918), opioid κ receptor 1 (rs1051660), catechol-O-methyltransferase (COMT) (rs4680), dopamine receptor D2 (rs6275), adenosine triphosphate binding cassette B1 (rs1045642), G-protein regulated inward rectifier potassium channel 2 (rs2070995), and fatty acid amide hydrolase (rs324420) genes on those adverse effects were analyzed.

Analysis of OPRM1 gene variant status (Asn133Asp A > G) showed that G/G homozygotes were at significantly lower risk of somnolence compared with A allele carriers (0% vs 28.4%; Fisher exact test, P = .005; OR, 0; 95% CI, 0–0.6), and analysis of COMT gene variant status (Val158Met, G > A) showed that G/G homozygotes were at significantly higher risk of somnolence compared with A allele carriers (35.0% vs 10.4%; Fisher exact test, P = .008; OR, 4.5; 95% CI, 1.4–18.1). No relationship between variant status and adverse effects was found for the other genes.

These findings demonstrate that OPRM1 and COMT gene variants influence the risk of somnolence as an adverse effect of opioid analgesic therapy.

Keywords: opioid, personalized medicine, rs1799971, rs4680, single nucleotide polymorphism

## 1. Introduction

The advancement of personalized medicine has been one of the major breakthroughs in cancer therapy in the 21st century. In the area of chemotherapy, the development of the tyrosine kinase inhibitor imatinib in 2001 dramatically improved the survival rate of patients with chronic myelogenous leukemia.^[1]^ Later in 2014 came nivolumab, from a new therapeutic class of drugs called immune checkpoint inhibitors.^[2]^ To date, many targeted drugs and immunotherapy drugs have been developed and the development of new drugs is ongoing.^[3,4]^

However, despite the dramatic advancement of cancer chemotherapy and immunotherapy, 20% to 50% of all cancer patients still experience pain,^[5]^ and insufficient progress has been made in basic or translational research on personalization of opioid therapy, which is the mainstay of palliative care for cancer pain. The history of opioid therapy began with the introduction of morphine in the 19th century,^[6,7]^ and today dozens of opioids, including morphine derivatives and synthetic opioids, are used in clinical practice. However, with all of these drugs, the extent of therapy personalization is essentially limited to dose adjustment based on a patient's physical condition as assessed by hepatic function indicators such as blood aspartate aminotransferase and alanine transaminase or renal function indicators such as creatinine.^[8,9]^

This limitation can be explained by our poor understanding of the biological mechanisms and genetic factors that influence pain. In addition, research on analgesic therapy is complicated by the subjective nature of pain, including the influence of psychological factors. However, various recent studies have shown that sequence variations in different genes influence response to analgesic therapy (Fig. [1](#F1)).^[10–12]^ Studies investigating the genetic factors that are important for personalized therapy generally adopt 1 of 2 approaches: a pharmacokinetic or pharmacodynamic approach. The pharmacokinetic approach looks at genes encoding proteins related to drug absorption, distribution, metabolism, and excretion. In contrast, the pharmacodynamic approach looks directly at sequence variations in receptors directly involved in producing the pharmacological effect (eg, opioid receptors) as well as genes involved in mechanisms such as the release and reuptake of secondary hormones involved in analgesic effects, including dopamine and noradrenaline.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/8568420/55c1db9ba8e6/medi-100-e27565-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8568420_medi-100-e27565-g001.jpg)

shows opioid mechanisms of action and related genetic variants. ABCB = adenosine triphosphate binding cassette B, ADRB = adrenergic receptor B, ATF = activating transcription factor, CACN = calcium voltage-gated channel, COMT = catechol-O-methyltransferase, CREB = cyclic adenosine monophosphate response element binding protein, CYP = cytochrome P450, DRD = dopamine receptor D, FAAH = fatty acid amide hydrolase, GIRK = G-protein regulated inward rectifier potassium channel, HTR = 5-hydroxytryptamine receptor, LAMB = laminin B, MAO = monoamine oxidase, OCT = organic cation transporter, SULT = sulfotransferase, TAOK = thousand-and-one amino acid protein kinase, UGT = uridine diphosphate glucuronosyltransferase.

The body of clinical research on sequence variations influencing pharmacokinetic and pharmacological effects in opioid analgesic therapy have continued to grow in recent years, but the majority of that research focused solely on the main analgesic effect of the drugs, and few studies have investigated the incidence of adverse effects or safety. In addition, of the variants that previous research suggested as possibly influencing the clinical effects of opioids, only a few have been analyzed to determine their influence on adverse effects, and thus there may exist some influence from the variants that have not yet been studied. Therefore, to identify variants with the potential to provide clinically useful information, it is important to assess the degree to which these variants influence the clinical effects of opioids by comparative analysis of multiple variants by using whole-exome sequencing. In this study, we determined the single nucleotide polymorphisms (SNPs) that are associated with a higher risk of adverse effects by conducting an analysis of opioid-treated cancer patients with lung cancer registered in a Project HOPE^[13,14]^ that is conducted at Shizuoka Cancer Center.

There are symptoms that are caused by the cancer itself, and the incidence varies depending on the site.^[15]^ For example, the presence of gastrointestinal cancer may lead to an increased incidence of opioid-induced nausea and constipation. To eliminate confounding factors to the extent possible, it is desirable that participants have a single type of cancer, and so this study investigated patients with lung cancer.

## 2. Methods

### 2.1. Subjects

Participants were cancer patients with lung cancer who provided general consent for exome sequencing at the Shizuoka Cancer Center Hospital and later started treatment with morphine or oxycodone between April 2014 and August 2018. Data from the first 7 days of treatment were analyzed.

### 2.2. Adverse effect and patient characteristics

Incidence rates for adverse effects (somnolence, nausea, constipation, delirium, urinary retention, and pruritus) in the morphine and oxycodone groups were determined retrospectively from electronic medical records and compared. Cases counted as adverse effects were those the study researchers graded as grade 1 or higher per the Common Terminology Criteria for Adverse Events Ver. 4.0 based on the description in electronic medical records written by a physician, nurse, or pharmacist. Data on the opioids used, initial opioid dose (mg/day), and final opioid dose (mg/day), as well as sex, age, weight, performance status, renal dysfunction, hepatic dysfunction, history of alcohol consumption, history of smoking, and chemotherapy and radiation therapy during the study period were collected as patient characteristics that could influence the incidence of adverse effects. Renal dysfunction was defined as grade 1 or higher elevation of blood creatinine, and hepatic dysfunction as grade 1 or higher elevation of blood aspartate aminotransferase/alanine transaminase per Common Terminology Criteria for Adverse Events Ver. 4.0. Equivalent doses of opioids were calculated based on oral morphine. The calculated ratio was 6:3:4:3 for oral morphine to intravenous morphine to oral oxycodone to intravenous oxycodone.^[6,16]^

### 2.3. Genetic variants

In this study, we systematically detected selected SNPs using whole-exome sequencing, theses variants were visually confirmed by Integrative Genomics Viewer.^[17]^ Variants at the following 8 sites were investigated: the opioid μ receptor 1 variant rs1799971,^[18]^ the opioid δ receptor 1 variant rs2234918,^[19]^ the opioid κ receptor 1 variant rs1051660,^[20]^ the catechol-O-methyltransferase (*COMT*) variant rs4680,^[21]^ the dopamine receptor D2 variant rs6275,^[22]^ the P-glycoprotein adenosine triphosphate binding cassette B1 variant rs1045642,^[23]^ the G-protein regulated inward rectifier potassium channel 2 variant rs2070995,^[24]^ and the fatty acid amide hydrolase variant rs324420.^[25]^ Incidence of each adverse effect was compared between variants.

### 2.4. Statistical analysis

The incidence of each adverse effect was compared between variants by Fisher exact test. A comparative analysis of patient characteristics according to alleles carried and an odds ratio (OR) analysis according to allele was also performed as supplemental analyses of SNPs found to significantly influence the incidence of adverse effects. We compared patient characteristics between the variants by the Mann–Whitney *U* test for continuous variables, initial opioid dose (mg/day), final opioid doses (mg/day), age, and weight, and Fisher exact test for nominal variables. The significance level was set at 0.05. All analyses were performed by Bell Curve for Excel (Social Survey Research Information Co., Ltd., Tokyo, Japan) and R version 4.0.3 (R Foundation for Statistical Computing, Vienna, Austria).

### 2.5. Ethical considerations

This study was conducted in compliance with the Ethical Guidelines for Medical and Health Research Involving Human Subjects and the Ethical Guidelines for Human Genome/Gene Analysis Research. The retrospective data review and statistical analysis were conducted with the approval of the institutional review board of the Shizuoka Cancer Center (Approval No. 30-28-30-1-3). All participants provided written informed consent prior to this study.

## 3. Results

### 3.1. Patient characteristics

A total of 12,634 patients were treated with opioids between April 2014 and August 2018, 550 of whom (all types of cancer) provided general consent for exome sequencing. Eighty-eight of those 550 patients had lung cancer. Table [1](#T1) shows the characteristics of those patients.

### Table 1.

| Characteristic | n = 88 |
| --- | --- |
| Opioid (morphine/oxycodone) | 49/39 |
| Initial dose (mg), median (range) | 15 (2–48) |
| Final dose (mg), median (range) | 15 (2–144) |
| Sex (men/women) | 64/24 |
| Median age (range) | 73 (14–89) |
| Median weight (kg) (range) | 54.1 (29.2–77.8) |
| Performance status (≤1/≥2) | 41/47 |
| Renal dysfunction (yes/no) | 12/76 |
| Hepatic dysfunction (yes/no) | 14/74 |
| History of alcohol consumption (yes/no) | 55/33 |
| History of smoking (yes/no) | 72/16 |
| Chemotherapy (yes/no) | 7/81 |
| Radiation therapy (yes/no) | 14/74 |

Table 1 Caption: Patient characteristics.

### 3.2. Comparative analysis of each adverse effect between variants.

Analysis according to rs1799971 variant status showed that incidence of somnolence was significantly lower in GG carriers than in carriers of other alleles (0% vs 28.4%; *P* = .005; OR, 0; 95% confidence interval (CI), 0–0.6) (Table [2](#T2)). Analysis according to allele showed that G allele carriers were at lower risk of somnolence compared with A allele carriers (*P* = .044; OR, 0.5; 95% CI, 0.2–0.95).

### Table 2.

|   | rs1799971 Alt allele freq: 0.494 A>G ORRM1 |  |  |  |  |  |  |  |  | rs2234918 Alt allele freq: 0.79 C>TOPRD1 |  |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |  |  |  |  |  |  |  |  |   |  |  |  |  |  |  |  |  |  |  |
| Allele | A/A | A/G+G/G | OR | 95% CI | P | G/G | A/A+A/G | OR | 95% CI | P | C/C | C/T+T/T | OR | 95% CI | P | T/T | C/C+C/T | OR | 95% CI | P |
|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
|   | n = 22 | n = 66 |   |   |   | n = 21 | n = 67 |   |   |   | n = 5 | n = 83 |   |   |   | n = 56 | n = 32 |   |   |   |
| Somnolence | 6 (27.3%) | 13 (19.7%) | 1.5 | 0.5 to 4.7 | .55∗ | 0 (0.0%) | 19 (28.4%) | 0 | 0 to 0.6 | .005∗ | 1 (20.0%) | 18 (21.7%) | 0.9 | 0.1 to 8.6 | 1.00∗ | 10 (17.9%) | 9 (28.1%) | 0.6 | 0.2 to 1.6 | .29∗ |
| Nausea | 1 (4.5%) | 13 (19.7%) | 0.2 | 0.02 to 1.6 | .17∗ | 5 (23.8%) | 9 (13.4%) | 2.0 | 0.6 to 6.9 | .31∗ | 0 (0.0%) | 14 (16.9%) | 0 | 0 to 6.0 | 1.00∗ | 12 (21.4%) | 2 (6.3%) | 4.1 | 0.9 to 19.6 | .07∗ |
| Constipation | 10 (45.5%) | 24 (36.4%) | 1.5 | 0.6 to 3.9 | .46∗ | 9 (42.9%) | 25 (37.3%) | 1.3 | 0.5 to 3.4 | .49∗ | 1 (20.0%) | 33 (39.8%) | 0.4 | 0.04 to 3.5 | .64∗ | 23 (41.1%) | 11 (34.4%) | 1.3 | 0.5 to 3.3 | .65∗ |
| Delirium | 1 (4.5%) | 9 (13.6%) | 0.3 | 0.04 to 2.5 | .44∗ | 3 (14.2%) | 7 (10.4%) | 1.4 | 0.3 to 6.1 | .70∗ | 0 (0.0%) | 10 (12.0%) | 0 | 0 to 9.2 | 1.00∗ | 6 (10.7%) | 4 (12.5%) | 0.8 | 0.2 to 3.2 | 1.00∗ |
| Urinary retention | 0 (0.0%) | 3 (4.5%) | 0 | 0 to 7.6 | 1.00∗ | 2 (9.5%) | 1 (1.5%) | 7.0 | 0.6 to 80.8 | .14∗ | 0 (0.0%) | 3 (3.6%) | 0 | 0 to 46.4 | 1.00∗ | 2 (3.6%) | 1 (3.1%) | 1.2 | 0.1 to 13.2 | 1.00∗ |
| Pruritus | 4 (18.1%) | 8 (12.1%) | 1.6 | 0.5 to 6.0 | .49∗ | 2 (9.5%) | 10 (14.9%) | 0.6 | 0.1 to 3.00 | .72∗ | 0 (0.0%) | 12 (14.5%) | 0 | 0 to 7.3 | 1.00∗ | 8 (14.3%) | 4 (12.5%) | 1.2 | 0.3 to 4.2 | 1.00∗ |

Table 2 Caption: Correlation analysis for 8 SNPs.

Analysis according to rs4680 variant status revealed that incidence of somnolence was significantly higher in GG carriers than in carriers of other alleles (35.0% vs 10.4%; *P* = .008; OR, 4.5; 95% CI, 1.4–18.1). Analysis according to allele showed that G allele carriers were at higher risk of somnolence compared with A allele carriers (*P* = .003; OR, 4.2; 95% CI, 1.6–11.5).

No other variants were found to significantly influence the incidence of any of the 6 investigated adverse effects. However, AA carriers for rs1045642 had a higher incidence of nausea compared with carriers of other alleles (34.8% vs 16.9%; *P* = .08; OR, 2.6; 95% CI, 0.9–7.7). GG carriers for rs6275 had a higher incidence of constipation compared with carriers of other alleles (60.0% vs 35.7%, *P* = .09; OR, 2.7; 95% CI, 0.9–8.5) and CC carriers for rs2070995 had a higher incidence of constipation compared with carriers of other alleles (51.4% vs 30.2%; *P* = .07; OR, 2.5; 95% CI, 1.0–5.9).

### 3.3. Supplemental analysis of patient characteristics

Supplemental analysis comparing the opioids used, initial opioid dose (mg/day), final opioid dose (mg/day), sex, age, weight, performance status, renal dysfunction, hepatic dysfunction, history of alcohol consumption, history of smoking, and chemotherapy and radiation therapy during the study period between GG carriers and carriers of other alleles for rs1799971 showed no significant difference in any characteristic between the groups (Table [3](#T3)). Similarly, an analysis comparing these 12 patient characteristics between GG carriers and carriers of other alleles for rs4680 showed no significant difference between the groups.

### Table 3.

|   | rs1799971 ALT allele freq: 0.494 A > G OPRM1 |  |  |  |  | rs4680 ALT allele freq: 0.335 G > A COMT |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Characteristic | G/G | A/A + A/G | OR | 95% CI | P | G/G | G/A + A/A | OR | 95% CI | P |
|   | n = 21 | n = 67 |   |   |   | n = 40 | n = 48 |   |   |   |
| Opioid (morphine/oxycodone) | 12/9 | 37/30 | 1.1 | 0.4 to 2.9 | 1.00∗ | 23/17 | 26/22 | 1.1 | 0.5 to 2.8 | .83∗ |
| Initial dose (mg), median (range) | 15 (3-48) | 15 (2-30) | – | – | .25† | 15 (5-20) | 15 (2-48) | – | – | .55† |
| Final dose (mg), median (range) | 20 (3-120) | 15 (2-144) | – | – | .52† | 15 (5-36) | 15 (2-144) | – | – | .99† |
| Sex (men/women) | 18/3 | 46/21 | 2.7 | 0.7 to 10.3 | .16∗ | 26/14 | 38/10 | 0.5 | 0.2 to 1.3 | .16∗ |
| Median age (range) | 73 (55-85) | 73 (14-89) | – | – | .71† | 74 (49-89) | 70 (14-83) | – | – | .15† |
| Median weight (kg) (range) | 56.5 (37.9-77.5) | 53 (29.2-77.8) | – | – | .29† | 51.5 (29.2-77.5) | 55.4 (31.8-77.8) | – | – | .09† |
| Performance status (≤1/≥2) | 12/9 | 29/38 | 1.8 | 0.7 to 4.7 | .32∗ | 18/22 | 23/25 | 0.9 | 0.4 to 2.1 | .83∗ |
| Renal dysfunction (yes/no) | 2/19 | 10/57 | 0.6 | 0.1 to 3.0 | .72∗ | 5/35 | 7/41 | 0.8 | 0.2 to 2.9 | 1.00∗ |
| Hepatic dysfunction (yes/no) | 4/17 | 10/57 | 1.3 | 0.4 to 4.8 | .73∗ | 4/36 | 10/38 | 0.4 | 0.1 to 1.5 | .24∗ |
| History of alcohol consumption (yes/no) | 14/7 | 41/26 | 1.3 | 0.5 to 3.6 | .79∗ | 23/17 | 32/16 | 0.7 | 0.3 to 1.6 | .38∗ |
| History of smoking (yes/no) | 19/2 | 53/14 | 2.5 | 0.5 to 12.1 | .34∗ | 31/9 | 41/7 | 0.6 | 0.2 to 1.8 | .41∗ |
| Chemotherapy (yes/no) | 2/19 | 5/62 | 1.3 | 0.2 to 7.3 | .67∗ | 2/38 | 5/43 | 0.5 | 0.1 to 2.5 | .44∗ |
| Radiation therapy (yes/no) | 3/18 | 11/56 | 0.9 | 0.2 to 3.4 | 1.00∗ | 4/36 | 10/38 | 0.4 | 0.1 to 1.5 | .24∗ |

Table 3 Caption: Supplemental analysis by patient characteristics for rs1799971 and rs4680.

## 4. Discussion

The results of this study suggest that the risk of somnolence during opioid therapy is low in GG carriers for rs1799971 and high in GG carriers for rs4680. These findings should be very useful because there are currently no drugs that prevent or reduce opioid-induced somnolence. Therefore, in clinical practice, opioid therapy could be started carefully in patients who are not GG carriers for rs1799971 and in patients who are GG carriers for rs4680.

The reason that GG carriers for rs1799971 have a low incidence of somnolence might be due to the change in the base from A to G at rs1799971 causes the amino acid to convert from asparagine to aspartic acid, and receptor activity is strongest for AA but weaker for AG and GG, in that order.^[26]^ This reduction in the activity caused by the change in amino acid also influences the analgesic effect. A clinical study in patients with cancer pain showed that the opioid dose necessary to achieve analgesia was also 2.1 times higher for GG carriers than AA carriers.^[23]^ Another study that compared patients’ self-reported ratings of pain relief revealed that GG carriers experience less reduction in numerical rating scale pain ratings.^[27]^ Given that our study was retrospective, we were unable to compare self-reported ratings of pain relief due to lack of numerical rating scale data for some patients. However, our finding that GG carriers had a higher, though not significantly higher, final opioid dose than carriers of other alleles (20 mg/day vs 15 mg/day) is consistent with the findings of previous studies that GG carriers require a higher opioid dose. In addition, 1 study showed that receptor activity even decreases in AG or GG carriers who are given an inactive placebo,^[28]^ which indicates that carrier status also influences the effects of endogenous opioid peptides such as endorphins. This weakening of activity is much more pronounced for morphine (AA carriers, inhibitory concentration (IC) 50 = 47 nM; GG carriers, IC50 = 260 nM) than fentanyl (AA carriers, IC50 = 62 nM; GG carriers, IC50 = 102 nM).^[29]^ Further research into this effect is warranted for many different opioids.

A study on the involvement of rs1799971 in opioid-induced somnolence in patients with noncancer pain showed that many GG carriers experience sleep disorders.^[30]^ The article did not discuss these sleep disorders in depth, but it is likely that the GG genotype caused these insomnia symptoms. On the other hand, there are few studies that examine the influence of genetic variants on opioid-induced somnolence for cancer patients in detail. Since the consensus of important clinical difference for the incidence of somnolence is not defined, it is difficult to compare with previous studies. However, this study showed large differences for rs1799971 variant status (GG carriers, 0%; AG or AA carriers 28.4%; *P* = .005) and rs4860 variant status (GG carriers, 35.0%; AG or AA carriers 10.4%; *P* = .008) in Table [2](#T2). In addition, no significant difference was found in the comparison of background factors related to the incidence of adverse events such as performance status in Table [3](#T3). Despite the low number of patients (N = 88), it can be considered that the results have clinical significance.

In the *COMT* variant rs4860, a base change from G to A converts the encoded amino acid from valine to methionine and reduces *COMT* activity to a third or fourth of its original level.^[31,32]^ Reduced *COMT* activity leads to poor metabolism of noradrenaline and increased concentration of noradrenaline in the body, and the effects of this on the brain's arousal centers may explain the reduction in somnolence. The increased noradrenaline concentration could also have influenced the analgesic effects. A clinical study in patients with cancer pain showed that GG carriers require an approximately 1.5-fold higher intravenous morphine dose than AA carriers.^[21]^ However, we did not observe a similar trend in this study. We only reviewed data from the first week of opioid therapy in order to reduce the impact of confounding factors, but it is possible that we could have observed a difference in the required therapeutic dose if the treatment period had been longer. Long-term studies spanning the entire period from initiation of opioid therapy until death are warranted.

We had initially predicted that reduced receptor activity associated with rs1799971 would also influence the risk of adverse effects besides somnolence, including nausea or constipation. However, we found no significant differences for those symptoms. Some following limitations of this study may have influenced the result. This could be because we were unable to rule out effects of concomitant medications due to the retrospective nature of this study. In practice, many patients are treated prophylactically with laxatives such as magnesium oxide and sennoside or antiemetics such as prochlorperazine before starting opioid therapy, and these drugs might have influenced our results. The lack of a standardized approach for evaluating adverse effects between the healthcare providers who wrote the electronic medical records and the researchers in this study may also have affected results. Accordingly, a more detailed prospective study must be conducted in the future. We also obtained unexpected results for some other SNPs. We thought that rs6275 would influence the incidence of nausea and delirium because it is a dopamine receptor variant,^[22]^ and that rs324420 would influence the incidence of nausea, as has been previously shown,^[25]^ but we found no significant difference for either SNP in the present study. Although no significant difference was observed, the AA carrier for rs1045642 tended to have a higher risk of nausea. This result was similar to the previous study investigating the incidence of nausea and vomiting with morphine response in the postoperative period.^[33]^ Reasons for this could be effects of concomitant medications given to prevent adverse effects, as we discussed for rs1799971, or that we needed larger sample size to detect a significant difference in delirium because it occurs less frequently than somnolence. We had also predicted that rs2070995 would influence the incidence of pruritus because G protein active potassium channels are involved in cutaneous sensation,^[24]^ but it actually appeared to be associated with constipation rather than pruritus. Further basic research to investigate the effect of rs2070995 on gastrointestinal motility is warranted because no study to date has reported such an effect.

## 5. Conclusion

In summary, our analysis of the correlation between 6 adverse effects of opioid therapy and genetic variants at eight sites showed that rs1799971 and rs4680 may be risk factors for somnolence. No drugs to prevent or reduce opioid-induced somnolence have yet been developed, and we believe our findings could aid in further personalization of opioid therapy and better understanding of the mechanisms involved in somnolence.

## Author contributions

**Conceptualization:** Rei Tanaka.

**Data curation:** Rei Tanaka, Junya Sato, Hiroshi Ishikawa, Sumiko Ohnami, Takeshi Nagashima.

**Formal analysis:** Rei Tanaka, Keita Mori, Akifumi Notsu.

**Funding acquisition:** Rei Tanaka.

**Investigation:** Rei Tanaka, Junya Sato, Hiroshi Ishikawa, Sumiko Ohnami, Takeshi Nagashima.

**Methodology:** Junya Sato, Sumiko Ohnami.

**Project administration:** Rei Tanaka.

**Resources:** Yasuhisa Ohde, Tetsumi Sato, Ken Yamaguchi.

**Supervision:** Ken Yamaguchi.

**Validation:** Tetsu Sato, Michihiro Shino.

**Visualization:** Maki Mizuguchi.

**Writing – original draft:** Rei Tanaka.

**Writing – review & editing:** Junya Sato, Hiroshi Ishikawa, Tetsu Sato, Michihiro Shino, Yasuhisa Ohde, Tetsumi Sato, Keita Mori, Akifumi Notsu, Sumiko Ohnami, Maki Mizuguchi, Takeshi Nagashima, Ken Yamaguchi.

## Footnotes

## References

1. Lyseng-Williamson K, Jarvis B. Imatinib. Drugs 2001;61:1765–74.  [DOI](https://doi.org/10.2165/00003495-200161120-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11693465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drugs&title=Imatinib&author=K%20Lyseng-Williamson&author=B%20Jarvis&volume=61&publication_year=2001&pages=1765-74&pmid=11693465&doi=10.2165/00003495-200161120-00007&)

2. George S, Motzer RJ, Hammers HJ, et al. Safety and efficacy of nivolumab in patients with metastatic renal cell carcinoma treated beyond progression: a subgroup analysis of a randomized clinical trial. JAMA Oncol 2016;2:1179–86.  [DOI](https://doi.org/10.1001/jamaoncol.2016.0775) | [PMC free article](/articles/PMC5568541/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27243803/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Oncol&title=Safety%20and%20efficacy%20of%20nivolumab%20in%20patients%20with%20metastatic%20renal%20cell%20carcinoma%20treated%20beyond%20progression:%20a%20subgroup%20analysis%20of%20a%20randomized%20clinical%20trial&author=S%20George&author=RJ%20Motzer&author=HJ%20Hammers&volume=2&publication_year=2016&pages=1179-86&pmid=27243803&doi=10.1001/jamaoncol.2016.0775&)

3. Zhong L, Li Y, Xiong L, et al. Small molecules in targeted cancer therapy: advances, challenges, and future perspectives. Signal Transduct Target Ther 2020;6:201.  [DOI](https://doi.org/10.1038/s41392-021-00572-w) | [PMC free article](/articles/PMC8165101/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34054126/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Signal%20Transduct%20Target%20Ther&title=Small%20molecules%20in%20targeted%20cancer%20therapy:%20advances,%20challenges,%20and%20future%20perspectives&author=L%20Zhong&author=Y%20Li&author=L%20Xiong&volume=6&publication_year=2020&pages=201&pmid=34054126&doi=10.1038/s41392-021-00572-w&)

4. Lu RM, Hwang YC, Liu IJ, et al. Development of therapeutic antibodies for the treatment of diseases. J Biomed Sci 2020;27:01.  [DOI](https://doi.org/10.1186/s12929-019-0592-z) | [PMC free article](/articles/PMC6939334/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31894001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biomed%20Sci&title=Development%20of%20therapeutic%20antibodies%20for%20the%20treatment%20of%20diseases&author=RM%20Lu&author=YC%20Hwang&author=IJ%20Liu&volume=27&publication_year=2020&pages=01&pmid=31894001&doi=10.1186/s12929-019-0592-z&)

5. Fisher DJ, Villines D, Kim YO, Epstein JB, Wilkie DJ. Anxiety, depression, and pain: differences by primary cancer. Support Care Cancer 2010;18:801–10.  [DOI](https://doi.org/10.1007/s00520-009-0712-5) | [PMC free article](/articles/PMC3682075/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19685346/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Support%20Care%20Cancer&title=Anxiety,%20depression,%20and%20pain:%20differences%20by%20primary%20cancer&author=DJ%20Fisher&author=D%20Villines&author=YO%20Kim&author=JB%20Epstein&author=DJ%20Wilkie&volume=18&publication_year=2010&pages=801-10&pmid=19685346&doi=10.1007/s00520-009-0712-5&)

6. Japanese Society for Palliative Medicine. Clinical Guideline for Pharmacological Management of Cancer Pain (2020 Edition). Tokyo: Kanehara Shuppan; 2020.  [Google Scholar](https://scholar.google.com/scholar_lookup?Japanese%20Society%20for%20Palliative%20Medicine.%20Clinical%20Guideline%20for%20Pharmacological%20Management%20of%20Cancer%20Pain%20(2020%20Edition).%20Tokyo:%20Kanehara%20Shuppan;%202020.)

7. Trescot AM, Datta S, Lee M, Hansen H. Opioid pharmacology. Pain Physician 2008;11: (2 Suppl): S133–53.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/18443637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain%20Physician&title=Opioid%20pharmacology&author=AM%20Trescot&author=S%20Datta&author=M%20Lee&author=H%20Hansen&volume=11&issue=(2%20Suppl)&publication_year=2008&pages=S133-53&pmid=18443637&)

8. Dean M. Opioids in renal failure and dialysis patients. J Pain Symptom Manage 2004;28:497–504.  [DOI](https://doi.org/10.1016/j.jpainsymman.2004.02.021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15504625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pain%20Symptom%20Manage&title=Opioids%20in%20renal%20failure%20and%20dialysis%20patients&author=M%20Dean&volume=28&publication_year=2004&pages=497-504&pmid=15504625&doi=10.1016/j.jpainsymman.2004.02.021&)

9. Soleimanpour H, Safari S, Shahsavari Nia K, Sanaie S, Alavian SM. Opioid drugs in patients with liver disease: a systematic review. Hapat Mon 2016;16:e32636.  [DOI](https://doi.org/10.5812/hepatmon.32636) | [PMC free article](/articles/PMC4887963/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27257423/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hapat%20Mon&title=Opioid%20drugs%20in%20patients%20with%20liver%20disease:%20a%20systematic%20review&author=H%20Soleimanpour&author=S%20Safari&author=K%20Shahsavari%20Nia&author=S%20Sanaie&author=SM%20Alavian&volume=16&publication_year=2016&pages=e32636&pmid=27257423&doi=10.5812/hepatmon.32636&)

10. Smith MT, Muralidharan A. Pharmacogenetics of pain and analgesia. Clin Genet 2012;82:321–30.  [DOI](https://doi.org/10.1111/j.1399-0004.2012.01936.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22779698/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Genet&title=Pharmacogenetics%20of%20pain%20and%20analgesia&author=MT%20Smith&author=A%20Muralidharan&volume=82&publication_year=2012&pages=321-30&pmid=22779698&doi=10.1111/j.1399-0004.2012.01936.x&)

11. Pathan H, Williams J. Basic opioid pharmacology: an update. Br J Pain 2012;6:11–6.  [DOI](https://doi.org/10.1177/2049463712438493) | [PMC free article](/articles/PMC4590096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26516461/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Pain&title=Basic%20opioid%20pharmacology:%20an%20update&author=H%20Pathan&author=J%20Williams&volume=6&publication_year=2012&pages=11-6&pmid=26516461&doi=10.1177/2049463712438493&)

12. Pergolizzi JV, Jr, LeQuang JA, Berger GK, Raffa RB. The basic pharmacology of opioids informs the opioid discourse about misuse and abuse: a review. Pain Ther 2017;6:01–16.  [DOI](https://doi.org/10.1007/s40122-017-0068-3) | [PMC free article](/articles/PMC5447545/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28341939/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain%20Ther&title=The%20basic%20pharmacology%20of%20opioids%20informs%20the%20opioid%20discourse%20about%20misuse%20and%20abuse:%20a%20review&author=JV%20Pergolizzi&author=JA%20LeQuang&author=GK%20Berger&author=RB%20Raffa&volume=6&publication_year=2017&pages=01-16&pmid=28341939&doi=10.1007/s40122-017-0068-3&)

13. Yamaguchi K, Urakami K, Ohshima K, et al. Implementation of individualized medicine for cancer patients by multiomics-based analyses–the Project HOPE. Biomed Res 2014;35:407–12.  [DOI](https://doi.org/10.2220/biomedres.35.407) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25743347/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biomed%20Res&title=Implementation%20of%20individualized%20medicine%20for%20cancer%20patients%20by%20multiomics-based%20analyses%E2%80%93the%20Project%20HOPE&author=K%20Yamaguchi&author=K%20Urakami&author=K%20Ohshima&volume=35&publication_year=2014&pages=407-12&pmid=25743347&doi=10.2220/biomedres.35.407&)

14. Nagashima T, Yamaguchi K, Urakami K, Shimoda Y, et al. Japanese version of The Cancer Genome Atlas, JCGA, established using fresh frozen tumors obtained from 5143 cancer patients. Cancer Sci 2020;111:687–99.  [DOI](https://doi.org/10.1111/cas.14290) | [PMC free article](/articles/PMC7004528/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31863614/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&title=Japanese%20version%20of%20The%20Cancer%20Genome%20Atlas,%20JCGA,%20established%20using%20fresh%20frozen%20tumors%20obtained%20from%205143%20cancer%20patients&author=T%20Nagashima&author=K%20Yamaguchi&author=K%20Urakami&author=Y%20Shimoda&volume=111&publication_year=2020&pages=687-99&pmid=31863614&doi=10.1111/cas.14290&)

15. Vainio A, Auvinen A. Prevalence of symptoms among patients with advanced cancer: an international collaborative study. J Pain Symptom Manage 1996;12:03–10.  [DOI](https://doi.org/10.1016/0885-3924(96)00042-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8718910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pain%20Symptom%20Manage&title=Prevalence%20of%20symptoms%20among%20patients%20with%20advanced%20cancer:%20an%20international%20collaborative%20study&author=A%20Vainio&author=A%20Auvinen&volume=12&publication_year=1996&pages=03-10&pmid=8718910&doi=10.1016/0885-3924(96)00042-5&)

16. Hanks GW, Conno F, Cherny N, et al. Morphine and alternative opioids in cancer pain: the EAPC recommendations. Br J Cancer 2001;84:587–93.  [DOI](https://doi.org/10.1054/bjoc.2001.1680) | [PMC free article](/articles/PMC2363790/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11237376/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Morphine%20and%20alternative%20opioids%20in%20cancer%20pain:%20the%20EAPC%20recommendations&author=GW%20Hanks&author=F%20Conno&author=N%20Cherny&volume=84&publication_year=2001&pages=587-93&pmid=11237376&doi=10.1054/bjoc.2001.1680&)

17. Ohnami S, Nagashima T, Urakami K, et al. Whole exome sequencing detects variants of genes that mediate response to anticancer drugs. J Toxicol Sci 2017;42:137–44.  [DOI](https://doi.org/10.2131/jts.42.137) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28321040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Toxicol%20Sci&title=Whole%20exome%20sequencing%20detects%20variants%20of%20genes%20that%20mediate%20response%20to%20anticancer%20drugs&author=S%20Ohnami&author=T%20Nagashima&author=K%20Urakami&volume=42&publication_year=2017&pages=137-44&pmid=28321040&doi=10.2131/jts.42.137&)

18. Bastami S, Gupta A, Zackrisson A-L, Ahlner J, Osman A, Uppugunduri S. Influence of UGT2B7, OPRM1 and ABCB1 Gene Polymorphisms on postoperative morphine consumption. Basic Clin Pharmacol Toxicol 2014;115:423–31.  [DOI](https://doi.org/10.1111/bcpt.12248) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24703092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin%20Pharmacol%20Toxicol&title=Influence%20of%20UGT2B7,%20OPRM1%20and%20ABCB1%20Gene%20Polymorphisms%20on%20postoperative%20morphine%20consumption&author=S%20Bastami&author=A%20Gupta&author=A-L%20Zackrisson&author=J%20Ahlner&author=A%20Osman&volume=115&publication_year=2014&pages=423-31&pmid=24703092&doi=10.1111/bcpt.12248&)

19. Olesen AE, Sato H, Nielsen LM, et al. The genetic influences on oxycodone response characteristics in human experimental pain. Fundam Clin Pharmacol 2015;29:417–25.  [DOI](https://doi.org/10.1111/fcp.12128) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26042474/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fundam%20Clin%20Pharmacol&title=The%20genetic%20influences%20on%20oxycodone%20response%20characteristics%20in%20human%20experimental%20pain&author=AE%20Olesen&author=H%20Sato&author=LM%20Nielsen&volume=29&publication_year=2015&pages=417-25&pmid=26042474&doi=10.1111/fcp.12128&)

20. Ho KWD, Wallace MR, Staud R, Fillingim RB. OPRM1, OPRK1, and COMT genetic polymorphisms associated with opioid effects on experimental pain: a randomized, double-blind, placebo-controlled study. Pharmacogenomics J 2020;20:471–81.  [DOI](https://doi.org/10.1038/s41397-019-0131-z) | [PMC free article](/articles/PMC7260086/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31806881/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=OPRM1,%20OPRK1,%20and%20COMT%20genetic%20polymorphisms%20associated%20with%20opioid%20effects%20on%20experimental%20pain:%20a%20randomized,%20double-blind,%20placebo-controlled%20study&author=KWD%20Ho&author=MR%20Wallace&author=R%20Staud&author=RB%20Fillingim&volume=20&publication_year=2020&pages=471-81&pmid=31806881&doi=10.1038/s41397-019-0131-z&)

21. Lucenteforte E, Vannacci A, Crescioli G, et al. Opioid response in pediatric cancer patients and the Val158Met polymorphism of the human catechol-O-methyltransferase (COMT) gene: an Italian study on 87 cancer children and a systematic review. BMC Cancer 2019;19:113.  [DOI](https://doi.org/10.1186/s12885-019-5310-4) | [PMC free article](/articles/PMC6357360/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30704436/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Opioid%20response%20in%20pediatric%20cancer%20patients%20and%20the%20Val158Met%20polymorphism%20of%20the%20human%20catechol-O-methyltransferase%20(COMT)%20gene:%20an%20Italian%20study%20on%2087%20cancer%20children%20and%20a%20systematic%20review&author=E%20Lucenteforte&author=A%20Vannacci&author=G%20Crescioli&volume=19&publication_year=2019&pages=113&pmid=30704436&doi=10.1186/s12885-019-5310-4&)

22. Ma L, Zhang X, Xiang Q, et al. Association between dopamine receptor gene polymorphisms and effects of risperidone treatment: A systematic review and meta-analysis. Basic Clin Pharmacol Toxicol 2019;124:94–104.  [DOI](https://doi.org/10.1111/bcpt.13111) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30103286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin%20Pharmacol%20Toxicol&title=Association%20between%20dopamine%20receptor%20gene%20polymorphisms%20and%20effects%20of%20risperidone%20treatment:%20A%20systematic%20review%20and%20meta-analysis&author=L%20Ma&author=X%20Zhang&author=Q%20Xiang&volume=124&publication_year=2019&pages=94-104&pmid=30103286&doi=10.1111/bcpt.13111&)

23. Gong XD, Wang JY, Liu F, et al. Gene polymorphisms of OPRM1 A118G and ABCB1 C3435T may influence opioid requirements in Chinese patients with cancer pain. Asian Pac J Cancer Prev 2013;14:2937–43.  [DOI](https://doi.org/10.7314/apjcp.2013.14.5.2937) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23803057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asian%20Pac%20J%20Cancer%20Prev&title=Gene%20polymorphisms%20of%20OPRM1%20A118G%20and%20ABCB1%20C3435T%20may%20influence%20opioid%20requirements%20in%20Chinese%20patients%20with%20cancer%20pain&author=XD%20Gong&author=JY%20Wang&author=F%20Liu&volume=14&publication_year=2013&pages=2937-43&pmid=23803057&doi=10.7314/apjcp.2013.14.5.2937&)

24. Lötsch J, Prüss H, Veh RW, Doehring A. A KCNJ6 (Kir3.2, GIRK2) gene polymorphism modulates opioid effects on analgesia and addiction but not on pupil size. Pharmacogenet Genomics 2010;20:291–7.  [DOI](https://doi.org/10.1097/FPC.0b013e3283386bda) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20220551/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=A%20KCNJ6%20(Kir3.2,%20GIRK2)%20gene%20polymorphism%20modulates%20opioid%20effects%20on%20analgesia%20and%20addiction%20but%20not%20on%20pupil%20size&author=J%20L%C3%B6tsch&author=H%20Pr%C3%BCss&author=RW%20Veh&author=A%20Doehring&volume=20&publication_year=2010&pages=291-7&pmid=20220551&doi=10.1097/FPC.0b013e3283386bda&)

25. Sadhasivam S, Zhang X, Chidambaran V, et al. Novel associations between FAAH genetic variants and postoperative central opioid related adverse effects. Pharmacogenomics J 2015;15:436–42.  [DOI](https://doi.org/10.1038/tpj.2014.79) | [PMC free article](/articles/PMC4492912/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25558980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Novel%20associations%20between%20FAAH%20genetic%20variants%20and%20postoperative%20central%20opioid%20related%20adverse%20effects&author=S%20Sadhasivam&author=X%20Zhang&author=V%20Chidambaran&volume=15&publication_year=2015&pages=436-42&pmid=25558980&doi=10.1038/tpj.2014.79&)

26. De Gregori M, Diatchenko L, Ingelmo PM, et al. Human genetic variability contributes to postoperative morphine consumption. J Pain 2016;17:628–36.  [DOI](https://doi.org/10.1016/j.jpain.2016.02.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26902643/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pain&title=Human%20genetic%20variability%20contributes%20to%20postoperative%20morphine%20consumption&author=M%20De%20Gregori&author=L%20Diatchenko&author=PM%20Ingelmo&volume=17&publication_year=2016&pages=628-36&pmid=26902643&doi=10.1016/j.jpain.2016.02.003&)

27. Campa D, Gioia A, Tomei A, Poli P, Barale R. Association of ABCB1/MDR1 and OPRM1 gene polymorphisms with morphine pain relief. Clin Pharmacol Ther 2008;83:559–66.  [DOI](https://doi.org/10.1038/sj.clpt.6100385) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17898703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Association%20of%20ABCB1/MDR1%20and%20OPRM1%20gene%20polymorphisms%20with%20morphine%20pain%20relief&author=D%20Campa&author=A%20Gioia&author=A%20Tomei&author=P%20Poli&author=R%20Barale&volume=83&publication_year=2008&pages=559-66&pmid=17898703&doi=10.1038/sj.clpt.6100385&)

28. Peciña M, Love T, Stohler CS, Goldman D, Zubieta JK. Effects of the mu opioid receptor polymorphism (OPRM1 A118G) on pain regulation, placebo effects and associated personality trait measures. Neuropsychopharmacology 2015;40:957–65.  [DOI](https://doi.org/10.1038/npp.2014.272) | [PMC free article](/articles/PMC4330509/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25308352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Effects%20of%20the%20mu%20opioid%20receptor%20polymorphism%20(OPRM1%20A118G)%20on%20pain%20regulation,%20placebo%20effects%20and%20associated%20personality%20trait%20measures&author=M%20Peci%C3%B1a&author=T%20Love&author=CS%20Stohler&author=D%20Goldman&author=JK%20Zubieta&volume=40&publication_year=2015&pages=957-65&pmid=25308352&doi=10.1038/npp.2014.272&)

29. Mahmoud S, Thorsell A, Sommer WH, et al. Pharmacological consequence of the A118G mu opioid receptor polymorphism on morphine- and fentanyl-mediated modulation of Ca2+ channels in humanized mouse sensory neurons. Anesthesiology 2011;115:1054–62.  [DOI](https://doi.org/10.1097/ALN.0b013e318231fc11) | [PMC free article](/articles/PMC3203255/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21926562/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anesthesiology&title=Pharmacological%20consequence%20of%20the%20A118G%20mu%20opioid%20receptor%20polymorphism%20on%20morphine-%20and%20fentanyl-mediated%20modulation%20of%20Ca2+%20channels%20in%20humanized%20mouse%20sensory%20neurons&author=S%20Mahmoud&author=A%20Thorsell&author=WH%20Sommer&volume=115&publication_year=2011&pages=1054-62&pmid=21926562&doi=10.1097/ALN.0b013e318231fc11&)

30. Margarit C, Ballester P, Inda MDM, et al. OPRM1 gene interaction with sleep in chronic pain patients treated with opioids. Pain Physician 2019;22:97–107.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/30700073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain%20Physician&title=OPRM1%20gene%20interaction%20with%20sleep%20in%20chronic%20pain%20patients%20treated%20with%20opioids&author=C%20Margarit&author=P%20Ballester&author=MDM%20Inda&volume=22&publication_year=2019&pages=97-107&pmid=30700073&)

31. Lachman HM, Papolos DF, Saito T, Yu YM, Szumlanski CL, Weinshilboum RM. Human catechol-O-methyltransferase pharmacogenetics: description of a functional polymorphism and its potential application to neuropsychiatric disorders. Pharmacogenetics 1996;6:243–50.  [DOI](https://doi.org/10.1097/00008571-199606000-00007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8807664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Human%20catechol-O-methyltransferase%20pharmacogenetics:%20description%20of%20a%20functional%20polymorphism%20and%20its%20potential%20application%20to%20neuropsychiatric%20disorders&author=HM%20Lachman&author=DF%20Papolos&author=T%20Saito&author=YM%20Yu&author=CL%20Szumlanski&volume=6&publication_year=1996&pages=243-50&pmid=8807664&doi=10.1097/00008571-199606000-00007&)

32. Bastos P, Gomes T, Ribeiro L. Catechol-O-methyltransferase (COMT): an update on its role in cancer, neurological and cardiovascular diseases. Rev Physiol Biochem Pharmacol 2017;173:01–39.  [DOI](https://doi.org/10.1007/112_2017_2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28456872/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rev%20Physiol%20Biochem%20Pharmacol&title=Catechol-O-methyltransferase%20(COMT):%20an%20update%20on%20its%20role%20in%20cancer,%20neurological%20and%20cardiovascular%20diseases&author=P%20Bastos&author=T%20Gomes&author=L%20Ribeiro&volume=173&publication_year=2017&pages=01-39&pmid=28456872&doi=10.1007/112_2017_2&)

33. Coulbault L, Beaussier M, Verstuyft C, et al. Environmental and genetic factors associated with morphine response in the postoperative period. Clin Pharmacol Ther 2006;79:316–24.  [DOI](https://doi.org/10.1016/j.clpt.2006.01.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16580900/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Environmental%20and%20genetic%20factors%20associated%20with%20morphine%20response%20in%20the%20postoperative%20period&author=L%20Coulbault&author=M%20Beaussier&author=C%20Verstuyft&volume=79&publication_year=2006&pages=316-24&pmid=16580900&doi=10.1016/j.clpt.2006.01.007&)
