# Pharmacogenetic Predictors of Cannabidiol Response and Tolerability in Treatment‐Resistant Epilepsy

## Metadata
**Authors:** Brittney H Davis, T Mark Beasley, Michelle Amaral, Jerzy P Szaflarski, Tyler Gaston, Leslie Perry Grayson, David G Standaert, E Martina Bebin, Nita A Limdi, the UAB CBD Study Group (includes all the investigators involved in the UAB EAP CBD program)
**Journal:** Clinical Pharmacology and Therapeutics
**Date:** 2021 Sep 22
**DOI:** [10.1002/cpt.2408](https://doi.org/10.1002/cpt.2408)
**PMID:** 34464454
**PMCID:** PMC8530979
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8530979/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8530979/pdf/CPT-110-1368.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8530979/pdf/CPT-110-1368.pdf)

## Abstract

In patients with treatment‐resistant epilepsy (TRE), cannabidiol (CBD) produces variable improvement in seizure control. Patients in the University of Alabama at Birmingham CBD Expanded Access Program (EAP) were enrolled in the genomic study and genotyped using the Affymetrix Drug Metabolizing Enzymes and Transporters plus array. Associations between variants and CBD response (≥50% seizure reduction) and tolerability (diarrhea, sedation, and abnormal liver function) was evaluated under dominant and recessive models. Expression quantitative trait loci (eQTL) influencing potential CBD targets was evaluated in the UK Brain Expression Consortium data set (Braineac), and genetic co‐expression examined. Of 169 EAP patients, 112 (54.5% pediatric and 50.0% female) were included in the genetic analyses. Patients with AOX1 rs6729738 CC (aldehyde oxidase; odds ratio (OR) 6.69, 95% confidence interval (CI) 2.19–20.41, P = 0.001) or ABP1 rs12539 (diamine oxidase; OR 3.96, 95% CI 1.62–9.73, P = 0.002) were more likely to respond. Conversely, patients with SLC15A1 rs1339067 TT had lower odds of response (OR 0.06, 95% CI 0.01–0.56, P = 0.001). ABCC5 rs3749442 was associated with lower likelihood of response and abnormal liver function tests, and higher likelihood of sedation. The eQTL revealed that rs1339067 decreased GPR18 expression (endocannabinoid receptor) in white matter (P = 5.6 × 10−3), and rs3749442 decreased hippocampal HTR3E expression (serotonin 5‐HT3E; P = 8.5 × 10−5). Furthermore, 75% of genes associated with lower likelihood of response were co‐expressed. Pharmacogenetic variation is associated with CBD response and influences expression of CBD targets in TRE. Implicated pathways, including cholesterol metabolism and glutathione conjugation, demonstrate potential interactions between CBD and common medications (e.g., statins and acetaminophen) that may require closer monitoring. These results highlight the role of pharmacogenes in fundamental biologic processes and potential genetic underpinnings of treatment‐resistance.

☑ Cannabidiol (CBD) improves seizure control in patients with treatment‐resistant epilepsy (TRE); however, response remains highly variable. Genetic factors underlying this variability have not been evaluated.

☑ Do genetic factors influence CBD response (≥ 50% seizure reduction) and adverse effects in patients with TRE?

☑ Pharmacogenetic variation is associated with CBD response in TRE and influences expression of potential CBD targets, offering insight into potential mechanisms through which CBD exerts its therapeutic effects in TRE. Additionally, implicated pathways, such as cholesterol metabolism and glutathione conjugation, highlight the potential for interactions between CBD and common medications (e.g., statins, acetaminophen, etc.) that may require closer monitoring when co‐administered. Furthermore, genes associated with lower likelihood of response were largely co‐expressed.

☑ Understanding genetic mechanisms influencing CBD response in TRE can help identify patients who may benefit from treatment. Additionally, these results shed light on the role of pharmacogenes in fundamental biologic processes and potential genetic underpinnings of treatment‐resistance.

In the United States, approximately 3 million adults and 470,000 children have epilepsy.[1](#cpt2408-bib-0001) Despite the availability of antiseizure drugs (ASDs) with varied mechanisms of action,[2](#cpt2408-bib-0002) an estimated 25% of patients (15% children and 34% adults), have treatment‐resistant epilepsy (TRE), failing to achieve adequate seizure control despite treatment with ≥2 ASDs.[3](#cpt2408-bib-0003), [4](#cpt2408-bib-0004) Additional therapies that can improve seizure control, mitigate disability, improve outcomes, and reduce costs are urgently needed.[5](#cpt2408-bib-0005)

Cannabidiol (CBD) is a naturally occurring cannabinoid with antiseizure properties. Highly purified CBD (Epidiolex; Greenwich Biosciences Inc., Carlsbad, CA) is US Food and Drug Administration (FDA) approved for the treatment of seizures associated with Lennox‐Gastaut, Dravet syndromes, and tuberous sclerosis complex, in patients ≥ 1 year, and has recently demonstrated utility in other TREs.[6](#cpt2408-bib-0006), [7](#cpt2408-bib-0007) However, CBD response remains highly variable, and the mechanisms underlying its therapeutic effects are not fully understood. CBD dose (mg/kg/day) directly related with higher CBD plasma levels (ng/mL), is associated with better seizure control.[8](#cpt2408-bib-0008)

CBD is primarily metabolized by cytochrome P450 (CYP) CYP2C19 and CYP3A4, and the UDP‐glucuronosyltransferases (UGTs) UGT1A7, UGT1A9, and UGT2B7.[9](#cpt2408-bib-0009) However, other CYPs, including CYP1A2, CYP2C9, and CYP2D6 are capable of metabolizing CBD.[10](#cpt2408-bib-0010) Furthermore, CBD has the potential to inhibit CYP1A2, CYP2B6, CYP2C8, CYP2C9, CYP2C19, CYP2D6, UGT1A9, and UGT2B7.[9](#cpt2408-bib-0009), [11](#cpt2408-bib-0011) The most common metabolites of CBD are 7‐COOH‐CBD, the inactive metabolite, 7‐OH‐CBD, the active metabolite, and the 6‐α‐OH‐CBD and 6‐β‐OH‐CBD minor metabolites.[12](#cpt2408-bib-0012) However, CBD metabolism is complex, and over 40 phase I metabolites alone have been identified. The role of other drug metabolizing enzymes, such as those involved in glucuronide and sulfate conjugation, is still being elucidated.[13](#cpt2408-bib-0013), [14](#cpt2408-bib-0014)

It is well recognized that genetic variation in pharmacogenes contributes to variability in drug response, ranging from lack of efficacy to susceptibility to adverse drug reactions.[15](#cpt2408-bib-0015) Identification of genetic predictors of CBD response, both therapeutic and adverse, can help determine which patients could benefit from adjunct CBD treatment. Furthermore, given that many adverse drug effects occur as a result of interactions with target or off‐target tissues,[16](#cpt2408-bib-0016) the identification of shared genetic predictors of response and adverse effects can offer insight into potential mechanisms underlying CBD action. To this end, we investigate the genetic underpinnings of therapeutic and adverse response in patients with TRE treated with CBD.

## METHODS

### Study population

Patients were enrolled in a compassionate‐use open‐label CBD study for TRE with approval by the University of Alabama at Birmingham (UAB) Institutional Review Board as part of an Expanded Access Program (EAP; [www.clinicaltrials.gov](http://www.clinicaltrials.gov); [NCT02695537](https://clinicaltrials.gov/ct2/show/NCT02695537) and [NCT02700412](https://clinicaltrials.gov/ct2/show/NCT02700412)).[7](#cpt2408-bib-0007) All patients were Alabama residents with a TRE diagnosis confirmed by video electroencephalography monitoring; adequate trial/failure of ≥ 4 ASDs, including ≥ 1 trial of 2 concomitant ASDs; and an average of ≥ 4 seizures per month, averaged over 3 months. CBD was initiated at 5 mg/kg/day, administered in 2 divided doses, and titrated up in increments of 5 mg/kg/day every 2 weeks (up to 50 mg/kg/day) depending on tolerability and therapeutic response.

### Pharmacogenomic study (May 2018–March 2019)

An additional consent was obtained for the genomic study, and DNA samples collected when stable doses of CBD and concomitant ASDs were reached, with no changes for ≥ 2 weeks. Seizure frequencies/severity, concomitant ASDs, and laboratory data were documented at baseline and updated at each visit along with CBD dose and any adverse effects. All patients (or caregivers) were instructed to keep seizure diaries, which were verified during in‐person visits.

### Genotyping and quality control

Clinical Laboratory Improvement Amendment (CLIA)‐certified genotyping on the Affymetrix Drug Metabolizing Enzymes and Transporters plus array (1,931 variants/5 copy number regions across ~ 230 drug‐related genes)[17](#cpt2408-bib-0017) was performed at the Coriell Institute for Medical Research (Camden, NJ) in two batches. Nine samples were re‐genotyped due to low call rates. Of the 1,931 variants, 1,328 were removed due to location on the X‐chromosome or a minor allele frequency < 0.05. No variants were removed due to deviation from Hardy‐Weinberg equilibrium (*P* < 0.0001). Variants in linkage disequilibrium (LD) were removed (*n* = 207, *r* ^2^ = 0.5), leaving 396 variants. Due to the novel nature of the study, results including variants removed due to LD are presented in the [Supporting Information](#cpt2408-sup-0001). Additionally, for variants associated with response, HaploReg was used to identify variants in LD (*R* ^2^ = 0.8).[18](#cpt2408-bib-0018) Call rates and variant frequencies are presented in **Table ** [S1](#cpt2408-sup-0001).

### Assessment of CBD response and tolerability

As previously described, percent change in seizure frequency was determined at CBD stable dose, defined as the time/dose when each patient reached maximal seizure control, and calculated as ((seizure frequency per 28 days) – (seizure frequency at baseline))/(seizure frequency at baseline) × 100).[6](#cpt2408-bib-0006), [8](#cpt2408-bib-0008) Response was categorized as no change/increase, > 0 to 25%, ≥ 25% to < 50%, ≥ 50% to < 75%, and ≥ 75% reduction. Therapeutic response was defined as ≥ 50% reduction from baseline. Adverse effects were classified according to the Medical Dictionary for Regulatory Activities (MedDRA, version 17.1), and those that differed by response status (diarrhea, sedation, and abnormal liver function tests (LFTs)) were evaluated in the genetic analyses.

### Statistical analyses

After stratification by response status, differences in patient characteristics were assessed using analysis of variance models for continuous variables and *χ* ^2^ tests for categorical variables. Due to concomitant ASD combinations, their limited influence on response,[19](#cpt2408-bib-0019) and low interaction potential,[20](#cpt2408-bib-0020) only ASDs expected to influence pharmacokinetic properties or have additive toxicity (e.g., clobazam sedation)[21](#cpt2408-bib-0021) with CBD and used by ≥ 10% of patients were evaluated. This included clobazam, valproate, zonisamide, topiramate, and rufinamide. Eslicarbazepine (*n* = 5) and stiripentol (*n* = 0) were not included due to no/limited use. All statistical analyses were performed using PLINK (version 1.9) and SAS (version 9.4).

The relationship between clinical predictors (dose, baseline weight, and concomitant ASDs) and CBD response/adverse effects was evaluated using logistic regression (**Table ** [S2](#cpt2408-sup-0002)). Genetic predictors of CBD response/adverse effects were evaluated using logistic regression under dominant (AA vs. Ab + bb) and recessive (AA + Ab vs. bb) models. Treatment‐group (adult/pediatric), race, and sex were included as covariates in clinical and genetic analyses. A permutation approach was used to account for multiple testing (number of permutations: 1,000), because other methods (e.g., Bonferroni correction) assume independence and would likely be overly conservative.[22](#cpt2408-bib-0022) Given the exploratory nature of the study, *P* values < 0.05 after permutation were considered significant. Additionally, for each outcome, a gene‐based correction was applied, based on the number of significant genes identified, to highlight impactful associations. For variants associated with response and an adverse effect, HaploReg was used to explore the influence of the variant on regulatory motifs.[18](#cpt2408-bib-0018)

### Expression quantitative trait loci and co‐expression

For response‐associated variants, we examined expression quantitative trait loci on potential CBD targets in epilepsy (e.g., G‐protein coupled receptors (GPCRs), voltage‐gated ion channels, etc.),[23](#cpt2408-bib-0023) using the UK Brain Expression Consortium data set (Braineac)[24](#cpt2408-bib-0024) and evaluated co‐expression using the co‐expression database, COEXPRESdb (version 7.3).[25](#cpt2408-bib-0025) Promoter sequences of genes in the co‐expression network were obtained from Ensembl (GRCh37 Release 104), and homology evaluated using Basic Local Alignment Search Tool (BLAST). Sequences with an *E* value ≤ 1 × 10^−50^ were considered high quality matches. PROMO (version 3.0.2),[26](#cpt2408-bib-0026) was used to search for potential transcription factor binding sites shared among co‐expressed genes. The search was limited to sites with ≥ 95% similarity and present in all sequences.

## RESULTS

### Study population

Of the 169 patients in the open‐label study,[7](#cpt2408-bib-0007), [8](#cpt2408-bib-0008) 113 participated in the genomic study. One patient was excluded due to discordant sex, resulting in 112 patients (54.5% pediatric and 50.0% female) in the genetic analyses. Therapeutic response (≥ 50% seizure reduction) was achieved for 56.3% (63/112) patients at an average CBD dose of 26.6 2 ± 14.24 mg/kg/day. Clinical characteristics did not differ by response status (**Table ** [1](#cpt2408-tbl-0001)), however, as previously reported,[19](#cpt2408-bib-0019) responders received higher CBD doses. On average, patients had tried/failed eight ASDs and were on three concomitant ASDs. Diarrhea (*P* = 0.01) was more frequent in responders, whereas sedation (*P* = 0.05) was more common among nonresponders. The majority of patients (80.4%) experienced some degree of weight loss independent of therapeutic response.

### Table 1.

|   | Overall (N = 112) | Responders (N = 63) | Non‐Responders (N = 49) | p‐value |
| --- | --- | --- | --- | --- |
| Demographics | N (%) | N (%) | N (%) |   |
| Treatment Group |   |   |   |   |
| Adult | 51 (45.5%) | 30 (47.6%) | 21 (42.9%) | 0.62 |
| Pediatric | 61 (54.5%) | 33 (52.4%) | 28(57.1%) |   |
| Gender |   |   |   |   |
| Female | 56 (50.0%) | 34 (54.0%) | 22 (44.9%) | 0.34 |
| Male | 56 (50.0%) | 29 (46.0%) | 27 (55.1%) |   |
| Self‐reported race |   |   |   |   |
| African American | 15 (13.4%) | 10 (15.9%) | 5 (10.2%) | 0.65 |
| White | 94 (83.9%) | 52 (82.5%) | 42 (85.7%) |   |
| Other | 3 (0.03%) | 1 (0.02%) | 2 (0.04%) |   |
| Baseline Characteristics | Mean ± SD | Mean ± SD | Mean ± SD |   |
| Age | 20.96 ± 16.11 | 19.76 ± 14.72 | 22.50 ± 17.77 | 0.39 |
| Age at epilepsy onset | 5.50 ± 9.82 | 5.37 ± 7.56 | 5.67 ± 12.22 | 0.88 |
| Weight (kg) | 55.02 ± 28.88 | 55.70 ± 29.71 | 54.13 ± 28.07 | 0.77 |
| Bilirubin (mg/dl) | 0.34 ± 0.16 | 0.32 ± 0.13 | 0.37 ± 0.19 | 0.13 |
| ALT | 24.10 ± 12.06 | 24.03 ± 11.19 | 24.19 ± 13.17 | 0.95 |
| AST | 27.88 ± 14.53 | 27.41 ± 11.71 | 28.47 ± 17.54 | 0.72 |
| Seizure frequencies | 136.26 ± 442.52 | 132.62 ± 407.24 | 140.94 ± 488.45 | 0.92 |
| Median (IQR) | 23.05 (6.36, 65.4) | 29 (10, 68.7) | 13.7 (5.2, 56.7) | 0.18 |
| ASDs tried/failed | 8.32 ± 3.48 | 8.25 ± 3.35 | 8.41 ± 3.67 | 0.82 |
| Concomitant ASDs | 2.64 ± 0.99 | 2.67 ± 1.03 | 2.61 ± 0.95 | 0.77 |
| Concomitant ASDs N (%) |   |   |   |   |
| clobazam | 40 (35.7%) | 21 (33.3%) | 19 (38.8%) | 0.55 |
| valproate | 21 (18.8%) | 13 (20.6%) | 8 (16.3%) | 0.56 |
| zonisamide | 20 (17.9%) | 13 (20.6%) | 7 (14.3%) | 0.38 |
| topiramate | 18 (16.1%) | 10 (15.9%) | 8 (16.3%) | 0.95 |
| rufinamide | 12 (10.7%) | 6 (9.5%) | 6 (12.2%) | 0.64 |
| Adverse Events* | N (%) | N (%) | N (%) |   |
| Diarrhea | 66 (58.9%) | 44 (69.8%) | 22 (44.9%) | 0.01 |
| Sedation | 39 (34.8%) | 17 (27.0%) | 22 (44.9%) | 0.05 |
| Nausea/vomiting | 14 (12.5%) | 9 (14.3%) | 5 (10.2%) | 0.52 |
| Abnormal liver function tests | 12 (10.7%) | 10 (15.9%) | 2 (4.1%) | 0.05 |
| Weight Change |   |   |   |   |
| No Change or increase | 22 (19.6%) | 14 (22.2%) | 8 (16.3%) | 0.67 |
| <10% Weight loss | 58 (51.8%) | 32 (50.8%) | 26 (53.1%) |   |
| 10–20% Weight loss | 23 (20.5%) | 11 (17.5%) | 12 (24.5%) |   |
| >20% Weight loss | 9 (8.0%) | 6 (9.5%) | 3 (6.1%) |   |
| Treatment‐Related Measures | Mean ± SD | Mean ± SD | Mean ± SD | p‐value |
| CBD dose (mg/kg/day) | 23.81 ± 14.21 | 26.62 ± 14.24 | 20.20 ± 13.46 | 0.02 |
| Maintenance seizure frequency | 58.54 ± 222.27 | 19.65 ± 46.09 | 108.52 ± 327.05 | 0.07 |
| Median (IQR) | 7.76 (2.58, 32.2) | 4.50 (1.28, 13.6) | 14.73 (5.67, 51) | <0.001 |
| Percent change seizure frequency | –42.08 ± 58.76 | –79.20 ± 16.58 | 5.65 ± 59.13 | <0.001 |
| Median (IQR) | –53.9 (–85.3, –17.1) | –82 (–94, –63.3) | –14.3 (–32.4, 2.7) | <0.001 |

Table 1 Caption: Demographic and clinical characteristics between CBD responders and non‐responders

### Genetic predictors of CBD response

After accounting for treatment group, sex, race, and CBD dose, variation in *AOX1* (phase I aldehyde oxidase), *SLC15A1* (hydrogen peptide co‐transporter), and *ABP1* (*AOC1*, involved in histamine degradation)[27](#cpt2408-bib-0027) were associated with CBD response (**Figure ** [1a](#cpt2408-fig-0001) **;** **Table ** [S3](#cpt2408-sup-0003)). Patients with the *AOX1* rs6729738 CC genotype (odds ratio (OR) 6.69, 95% confidence interval (CI) 2.19–20.41, *P* = 0.001), and the *ABP1* rs12539 variant (OR 3.96, 95% CI 1.62–9.73, *P* = 0.002) were more likely to respond to CBD. Conversely, patients with the *SLC15A1* rs1339067 TT genotype had a 94% lower likelihood of response (OR 0.06, 95% CI 0.01–0.56, *P* = 0.001).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1a86/9540479/221e8b0e1026/CPT-110-1368-g003.jpg)

Manhattan plots of (a) CBD response (≥ 50% seizure reduction) and CBD‐associated (b) diarrhea (c) sedation and (d) abnormal liver function tests. For all assessments, the blue line indicates a P value threshold of 0.05. a Genetic variants associated with CBD response adjusted for treatment group, race, sex, and CBD dose. Red line indicates a P value of 0.002 (0.05/29 significant genes). b Genetic variants associated with CBD‐related diarrhea after adjustment for treatment group, race, sex, baseline weight, and clobazam. Red line indicates a P value of 0.001 (0.05/36 significant genes). c Genetic variants associated with CBD‐associated sedation adjusted for treatment group, race, sex, CBD dose, clobazam, and rufinamide. Red line indicates a P value of 0.002 (0.05/27 significant genes). d Genetic variants associated with CBD‐associated abnormal liver function tests adjusted for treatment group, race, sex, and weight. Red line indicates a P value of 0.003 (0.05/18 significant genes). CBD, cannabidiol. [Colour figure can be viewed at wileyonlinelibrary.com]

With the exception of *CYP17A1* rs6162 (involved in steroid hormone biosynthesis),[27](#cpt2408-bib-0027) variation in CYP enzymes was associated with lower likelihood of response. Patients lacking *CYP1A2* rs762551 AA, associated with higher enzyme inducibility,[28](#cpt2408-bib-0028) and those with *CYP2D6* rs28371725, had 56% (OR 0.44, 95% CI 0.19–0.98, *P* = 0.04) and 77% (OR 0.23, 95% CI 0.07–0.80, *P* = 0.02) lower odds of response, respectively. Similarly, patients with variation in the phase I flavin‐containing monooxygenases, *FMO2* rs7515157 TT (OR 0.18, 95% CI 0.04–0.82, *P* = 0.01), and *FMO4* rs2223477 G (OR 0.37, 95% CI 0.15–0.92, *P* = 0.04) were less likely to respond to CBD. Among phase I dehydrogenases, patients with homozygous variants in the alcohol dehydrogenase *ADH4* rs3762894 were less likely to respond to CBD (OR 0.10, 95% CI 0.01–0.92, *P* = 0.01); whereas patients with a dihydropyrimidine dehydrogenase (*DPYD*) rs1801265 variant had approximately threefold higher odds of response (OR 2.78, 95% CI 1.22–6.30, *P* = 0.02).

Variants in phase II genes were primarily associated with higher likelihood of response. Patients with a carbohydrate sulfotransferase, *CHST11* rs903247 C allele (OR 3.93, 95% CI 1.68–9.19, *P* = 0.004), a *UGT2B4* rs1966151 C allele (OR 3.62, 95% CI 1.52–8.62, *P* = 0.004), or the *SULT1A2* rs1059491 CC genotype (OR 16.50, 95% CI 1.96–138.80, *P* = 0.005) were more likely to respond to CBD. Alternatively, patients with variants in glutathione‐S‐transferases (GSTs), including *GSTM5* rs2479390 (OR 0.33, 95% CI 0.14–0.79, *P* = 0.01) and *GSTP1* rs1695 CC (OR 0.12, 95% CI 0.02–0.65, *P* = 0.003) had a lower likelihood of response.

Among transporters, patients with an *ABCC4* rs2274406 A allele (OR 3.08, 95% CI 1.29–7.37, *P* = 0.005) or an *ABCG1* rs914189 G allele (OR 2.64, 95% CI 1.09–6.39, *P* = 0.03) had a higher likelihood of response, whereas patients with *ABCC5* rs3749442 T had 64% lower odds of response (OR 0.36, 95% CI 0.15–0.85, *P* = 0.03). Variation in nuclear receptors was also found to be associated with CBD response. Patients with homozygous rs3814055 T variants in *NR1I2* (OR 0.35, 95% CI 0.12–0.99, *P* = 0.04), and those with a peroxisome proliferator activated receptor gamma (*PPARγ*) rs9833097 A allele (OR 0.34, 95% CI 0.12–0.92, *P* = 0.03) were less likely to respond to CBD. Variants in LD (*R* ^2^ = 0.8) with response‐associated variants are presented in **Table ** [S4](#cpt2408-sup-0004).

### CBD‐associated diarrhea

After accounting for treatment‐group, sex, race, weight, and clobazam, variation in phase I enzymes was associated with CBD‐related diarrhea (**Figure ** [1b](#cpt2408-fig-0001) **;** **Table ** [S5](#cpt2408-sup-0005)). Patients with the *CYP2A6* rs28399433 variant (OR 12.15, 95% CI 2.64–55.93, *P* = 0.001), involved in coumarin, nicotine, and caffeine metabolism,[27](#cpt2408-bib-0027) and those with the *FMO6* rs2272797 variant (OR 7.34, 95% CI 2.25–23.91, *P* = 0.001), encoding a pseudogene, were more likely to experience diarrhea.

Additionally, patients with homozygous *CYP39A1* rs7761731 A alleles, involved in neural cholesterol clearance, had ninefold higher likelihood of CBD‐associated diarrhea (OR 9.03, 95% CI 1.34–60.65, *P* = 0.02). Among phase II pathways, patients with the rs9787901 variant in carbohydrate sulfotransferase (*CHST1*), were more likely to experience diarrhea (OR 4.80; 95% CI 1.24–18.55, *P* = 0.02); whereas patients with variation in pathways related to glutathione (*GSTA3* rs512795) and glucuronide conjugation (*UGT2A1* rs11249454) were less likely to experience diarrhea (*P* values < 0.05). Patients with an *ABCB11* rs7563233 G allele or the rs496550 GG genotype, encoding the bile salt export pump, and patients with variants in *SLCO1B3* (rs3764006 and rs4149117), involved in bile acid clearance, were less likely to experience diarrhea (*P* values < 0.05). *ABCB1* rs2214102, encoding P‐glycoprotein, and *SLCO1B1* rs11045819, encoding OATP1B1, were associated with higher (OR 3.54, 95% CI 1.13–11.13, *P* = 0.03), and lower (OR 0.28, 95% CI 0.08–0.98, *P* = 0.04) likelihood of diarrhea, respectively. Among variants associated with CBD response, *ABP1* rs12539, associated with higher odds of response, was also associated with higher likelihood of treatment‐related diarrhea (OR 3.25, 95% CI 1.20–8.85, *P* = 0.02). Alternatively, *ABCC4* rs2274406, associated with higher response under a dominant model, was associated with lower likelihood of diarrhea under a recessive model (OR 0.14, 95% CI 0.03–0.63, *P* = 0.01).

### CBD‐associated sedation

After accounting for treatment group, sex, race, CBD dose, clobazam, and rufinamide (**Figure ** [1c](#cpt2408-fig-0001) **;** **Table ** [S6](#cpt2408-sup-0006)), patients with homozygous *ADH1A* rs6811453 T alleles (OR 8.98, 95% CI 2.11–38.22, *P* = 0.002) or homozygous *ABCG1* rs1541290 G alleles (OR 6.71, 95% CI 2.15–20.92, *P* = 0.002) were more likely to experience sedation. Conversely, patients with the *GSTM5* rs11807 variant (phase II GST; OR 0.10, 95% CI 0.03–0.34, *P* = 0.002), or the *SLC28A1* rs8025045 variant (nucleoside transporter; OR 0.07, 95% CI 0.01–0.35, *P* = 0.002) were less likely to experience CBD‐associated sedation. Among variants associated with CBD response, patients with the *ABCC5* rs3749442 variant, associated with lower odds of response, were more likely to experience sedation (OR 3.51, 95% CI 1.42–8.66, *P* = 0.005). Alternatively, patients with the *ABCG1* rs914189 variant, associated with higher likelihood of response, had a 71% lower likelihood of sedation (OR 0.29, 95% CI 0.10–0.82, *P* = 0.02). Additionally, patients with *CHST1* rs9787901, associated with higher odds of diarrhea, had lower odds of sedation (OR 0.17, 95% CI 0.04–0.69, *P* = 0.01).

### CBD‐associated abnormal LFTs

After adjustment for treatment group, sex, race, and weight (**Figure ** [1d](#cpt2408-fig-0001) **;** **Table ** [S7](#cpt2408-sup-0007)), patients with the *SULT1B1* rs1604741 CC genotype had higher likelihood of developing abnormal LFTs (OR 8.49, 95% CI 1.77–40.77, *P* = 0.002). Patients with a *SLC22A5* rs274558 G allele, encoding a carnitine transporter, were less likely to develop abnormal LFTs (OR 0.11, 95% CI 0.02–0.53, *P* = 0.001).

Patients with the apolipoprotein A2 (*APOA2*) rs5085 CC genotype (OR 7.00, 95% CI 1.20–41.16, *P* = 0.03) or a *QPRT* rs3862476 variant (involved in quinolinic acid degradation,[27](#cpt2408-bib-0027) OR 8.83, 95% CI 1.42–54.73, *P* = 0.02), were more likely to develop abnormal LFTs. Lower odds of abnormal LFTs were observed for patients with an *ADH1A* rs6811453 T allele (OR 0.20, 95% CI 0.04–0.93, *P* = 0.03) or a *CYP39A1* rs7761731 A allele (OR 0.17, 95% CI 0.03–0.91, *P* = 0.03), associated with higher odds of sedation and diarrhea, respectively, under recessive models. Valproic acid was not associated with abnormal LFTs (*P* = 0.56) and was not included as a covariate.

### Shared genetic predictors of CBD response and adverse effects

At the gene level, variation in pharmacogenes involved in phase I and II metabolism, drug transport, and other drug‐related processes, were associated with CBD response and adverse effects (**Figure ** [2](#cpt2408-fig-0002)). At the variant level, with the exception of *SLC22A11* rs2078267, all variants associated with CBD response and an adverse effect altered a regulatory motif (**Table ** [2](#cpt2408-tbl-0002)). *FMO4* rs2223477, associated with lower odds of response and diarrhea, altered the Farnesoid x receptor motif, involved in bile acid synthesis and transport, and the RXRA motif. RXRA forms a heterodimer with PPARα, which is required for PPARα mediated activation of CYP genes and those involved in fatty acid oxidation.[27](#cpt2408-bib-0027) *ABP1* rs12539, associated with increased odds of response and diarrhea, was found to influence the Cdx2 motif, involved in intestinal gene regulation, and Pax‐2, involved in central nervous system development and kidney cell differentiation.[27](#cpt2408-bib-0027)

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1a86/9540479/d46e2ce15e80/CPT-110-1368-g004.jpg)

Genes associated with CBD response (≥ 50% seizure reduction) and adverse effects based on permutation P values < 0.05. For each gene, the variant with the smallest P value was used for each outcome. CBD, cannabidiol. [Colour figure can be viewed at wileyonlinelibrary.com]

### Table 2.

| Gene variant | CBD‐related outcome(s) | Motif(s) altered | Function27 |
| --- | --- | --- | --- |
| FMO4 rs2223477 | Response (↓) Diarrhea (↓) | FXR (Farnesoid X Nuclear receptor) | Regulates genes involved in bile acid synthesis and transport |
|   |   | Nrf1 (Nuclear Respiratory Factor 1) | Activates expression of metabolic and nuclear genes. May also regulate neurite outgrowth |
|   |   | RXRA (Retinoid X Receptor Alpha) | Mediates effects of retinoids. Forms heterodimer with PPARα, which is required for PPARα activation of genes involved in fatty acid oxidation and cytochrome P450 genes |
| ABCC4 rs2274406 | Response (↑) Diarrhea (↓) | AP‐1 (Activator Protein 1) | Complex composed of members from Jun, Fos, ATF/cAMP‐responsive element binding and Maf families. Involved in cellular processes including inflammation, differentiation, and apoptosis |
|   |   | AP‐2 (Activating Enhancer‐Binding Protein 2) | Activates genes involved in biologic processes including proper development. Suppresses genes including C/EBP alpha |
| ABCC5 rs3749442 | Response (↓) Sedation (↑) Abnormal LFTs (↓) | Maf (MAF BZIP Transcription Factor) | Increases T‐cell apoptosis, activates G1 element of glucagon promoter, overexpression blocks anti‐oxidant response element mediated transcription |
|   |   | Rad21 (RAD21 Cohesin Complex Component) | Member of the cohesin complex, required for proper chromosome organization and post‐replication DNA repair |
| ABCG1 rs914189 | Response (↑) Sedation (↓) | AP‐1 (Activator Protein 1) | Complex composed of members from Jun, Fos, ATF/cAMP‐responsive element binding and Maf families. Involved in cellular processes including inflammation, differentiation, and apoptosis |
|   |   | SETDB1 (SET Domain Bifurcated Histone Lysine Methyltransferase 1) | Involved in the regulation of histone methylation, gene silencing and repressing transcription |
| SLC22A11 rs2078267 | Response (↑) Diarrhea (↑) | NA | NA |
| SLCO3A1 rs2190748 | Response (↓) Diarrhea (↓) | HNF4 (Hepatocyte Nuclear Factor 4) | Controls hepatic gene expression during endodermal transition to hepatic cells |
|   |   | SRF (Serum Response Factor) | Binds to serum response element and regulates cell cycle, growth and differentiation, and apoptosis. Downstream target of MAPK pathway |
| SLCO4A1 rs2236553 | Response (↑) Sedation (↓) | TAL1 (TAL BHLH Transcription Factor 1, Erythroid Differentiation Factor) | Activates or represses transcription of hematopoietic, neural, and endothelial precursors |
| ABP1 rs12539 | Response (↑) Diarrhea (↑) | Cdx2 (Caudal Type Homeobox 2) | Regulator of intestinal genes involved in cell growth and differentiation |
|   |   | HNF1 (Hepatic Nuclear Factor 1) | Regulates many liver‐specific genes and pancreatic islet cells |
|   |   | Pax‐2 (Paired Box Gene 2) | Critical role in CNS development, may also have role in kidney cell differentiation |

Table 2 Caption: Influence of variants associated with CBD response and an adverse effect on regulatory motifs

### Brain expression quantitative trait loci

Variants associated with CBD response influenced the expression of GPCRs, serotonin receptors, tumor necrosis factor receptors, and voltage‐gated potassium channels in the brain (**Table ** [3](#cpt2408-tbl-0003)). With the exception of VGKCs and *GPR180*, variants influencing the expression of potential CBD targets were primarily associated with lower likelihood of response. The rs3749442 A allele, associated with CBD response, sedation, and abnormal LFTs, was also associated with decreased hippocampal *HTR3E* (serotonin 5‐HT_3E_ receptor) expression (**Figure ** [3a](#cpt2408-fig-0003); *P* = 8.5 × 10^−5^). Additionally, rs1339067, associated with lower likelihood of CBD response under a recessive model (TT genotype), was associated with decreased *GPR18* expression (endocannabinoid receptor) in white matter (**Figure ** [3b](#cpt2408-fig-0003); *P* = 5.6 × 10^−3^), and increased *GPR183* expression (oxysterol receptor) in the temporal cortex (**Figure ** [3c](#cpt2408-fig-0003); *P* = 8.6 × 10^−3^).

### Table 3.

| Variant | CBD‐related outcome(s) | Affected gene | eQTL P value | eQTL tissue | Function 27 |
| --- | --- | --- | --- | --- | --- |
| G protein coupled receptors (GPCRs) |  |  |  |  |  |
| rs1339067 | Response (↓) | GPR18 | 5.6 × 10−3 | WHMT | Involved in inflammatory and immune responses, and the endocannabinoid system. N‐arachidonoylglycine (anandamide metabolite) and resolvin D2 (polyunsaturated fatty acid metabolite) are proposed ligands. 46 |
| GPR183 | 8.6 × 10−3 | TCTX | Oxysterol receptor expressed in lymphocytes. Involved in astrocyte migration and astrocyte‐macrophage communication. Ligands include 7‐alpha, 25‐dihydroxycholesterol. |  |  |
| rs2479390 | Response (↓) | GPR61 | 4.4 × 10−4 | CRBL | Orphan GPCR related to the biogenic amine receptor. Activates G(s)‐α/cAMP constitutively. |
| rs1695 | Response (↓) | GPR152 | 2.8 × 10−2 | WHMT | Ligands may include acetylcholine, serotonin, adrenaline, noradrenaline, dopamine, histamine, tyramine. 49 |
| rs3814055 | Response (↓) | GPR156 | 4.3 × 10−3 | PUTM | Thought to be GABAB‐related G‐protein coupled receptor, but no response to GABAB ligands. Function unknown. 50 |
| rs2274406 | Response (↑) Diarrhea (↓) | GPR180 | 2.7 × 10−2 | TCTX | May play a role in vascular remodeling. |
| 5‐Hydroxytryptamine receptors |  |  |  |  |  |
| rs3749442 | Response (↓) Abnormal LFTs (↓) Sedation (↑) | HTR3E | 8.5 × 10−5 | HIPP | Serotonin receptor subunit. May be involved in neurotransmission in myenteric neurons. |
| Tumor necrosis factor |  |  |  |  |  |
| rs28371725 | Response (↓) | TNFRSF13C | 9.0 × 10−3 | HIPP | Receptor for B‐cell activating factor (BAFF), involved in B‐cell regulation. |
| Voltage‐gated potassium channels |  |  |  |  |  |
| rs2479390 | Response (↓) | KCNC4 | 1.8 × 10−4 | MEDU | Delayed rectifier potassium channel, mediates potassium permeability of excitable membranes. |
| KCNA2 | 4.2 × 10−2 | SNIG | Delayed rectifier potassium channel, which regulates neuronal output by preventing abnormal action potential firing. |  |  |
| KCNA3 | 3.5 × 10−3 | MEDU | Delayed rectifier potassium channel, mediates potassium permeability of excitable membranes. Also involved in T‐cell response. |  |  |
| KCNA10 | 2.2 × 10−2 | HIPP | Mediates potassium permeability of excitable membranes. |  |  |
| rs12539 | Response (↑) Diarrhea (↑) | KCNH2 | 3.5 × 10−4 | FCTX | Forms subunit of voltage‐gated inward rectifying potassium channel. |
| rs2078267 | Response (↑) Diarrhea (↑) | KCNK4 | 2.8 × 10−3 | MEDU | TWIK‐related arachidonic‐acid stimulated potassium channel. Forms voltage‐insensitive outward rectifying channel regulated by fatty acids, temperature, and mechanical stimulation. |
| rs2236553 | Response (↑) Sedation (↓) | KCNQ2 | 9.3 × 10−3 | HIPP | Forms the M potassium channel upon association with KCNQ3. Plays critical role in neuronal excitability. |

Table 3 Caption: Influence of CBD response‐associated variants on brain eQTL of potential CBD targets in epilepsy

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1a86/9540479/38a54e3d47b8/CPT-110-1368-g001.jpg)

Expression of genes encoding potential CBD targets stratified by response‐associated variants. (a) HTR3E expression stratified by rs3749442. (b) GPR18 expression stratified by rs1339067. (c) GPR183 expression stratified by rs1339067. CBD, cannabidiol; eQTL, expression quantitative trait loci. [Colour figure can be viewed at wileyonlinelibrary.com]

### Genetic co‐expression, sequence homology, and epistasis

Among CBD response‐related genes (16 lower and 13 higher), 58.6% (17/29) were identified in the co‐expression network (**Figure ** [4](#cpt2408-fig-0004)). Notably, 75% (12/16) of genes associated with lower likelihood of response were co‐expressed, whereas only 38.5% (5/13) of genes associated with higher likelihood of response were co‐expressed. Pathways represented in the co‐expression network included retinoic acid and MAPK pathways. Additionally, *NOTCH2NL*, involved in the regulation of genes involved in neuronal differentiation, was co‐expressed with *GSTP1* and *SLC22A5*, both associated with lower odds of CBD response.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1a86/9540479/4040dd8e3a1e/CPT-110-1368-g002.jpg)

Co‐expression network among CBD response associated genes. Genes associated with CBD response are outlined in black. Colored circles indicate blue: phase I metabolism, green: phase II metabolism, coral: SLC family transporters, pink: ABC family transporters, yellow: other drug‐related genes, and black: other genes. CBD, cannabidiol.

Among shared promoter sequences of genes in the co‐expression network, *PDZD3* and *SLC9A3R1* shared 90.6% (*E* value 1 × 10^−135^) and 91.3% (*E* value 4.7 × 10^−75^) alignment with *DLG2*, involved in the regulation of NMDA receptors and synaptic stability.[27](#cpt2408-bib-0027) Similarly, *PDZD3* and *SLC9AR1* shared 85.4% (*E* value 1.3 × 10^−114^) and 86.6% (*E* value 5.3 × 10^−61^) alignment, respectively, with *NRG3* (neuregulin 3), an ERBB4 ligand potentially involved in oligodendrocyte survival. *ERC1*, encoding an RIM‐binding protein, involved in the regulation of neurotransmitter release,[27](#cpt2408-bib-0027) shared 91.0% alignment with *GSTP1* (*E* value 1.7 × 10^−57^), associated with lower odds of response, and 88.8% alignment with *SLC9A3R2* (*E* value 4.5 × 10^−84^). Furthermore, *SLC35F2*, whose transcripts are enriched in brain microvascular endothelial cells forming the blood brain barrier,[29](#cpt2408-bib-0029) shared 88.9% alignment with *PDZD3* (*E* value 2.2 × 10^−130^) and 91.5% alignment with *SLC9A3R2* (*E* value 2.3 × 10^−59^).

When the promoter sequences of genes in the co‐expression network were evaluated for potential shared transcription factor binding sites, CEBPβ (CCAAT/enhancer binding protein beta), GR‐β (glucocorticoid receptor beta), and STAT4 (signal transducer and activator of transcription 4) sites were present (≥ 95% similarity) in all promoter sequences of genes in the co‐expression network. Notably, *AOX1* rs6729738, the most significant variant associated with response, alters the *NR3C1* motif,[18](#cpt2408-bib-0018), [30](#cpt2408-bib-0030) encoding the glucocorticoid receptor (GR).

When epistasis was evaluated, an interaction was observed between *UGT2B4* rs1966151, associated with higher odds of response, and rs1152003 in the nuclear receptor, *PPARγ* (*P* = 1.46 × 10^−5^).

## DISCUSSION

Our study highlights the effect of genetic variation on both seizure reduction and susceptibility to adverse effects in patients with TRE treated with CBD. Concordant with previous studies, our results support complex CBD metabolism, with genetic variation across pharmacogenes implicated in response and tolerability. Additionally, our results demonstrate that genes associated with CBD response may influence the expression of potential CBD targets in epilepsy, and that genes associated with lower likelihood of therapeutic response appear to be largely co‐expressed. Pharmacogenes influence fundamental biologic processes independent of their role in drug metabolism/transport and can shed light on potential mechanisms that may contribute to pathologic processes, such as TRE. The analysis highlights the influence of pharmacogenes on interconnected pathways related to cholesterol, bile acids, steroid hormones, purine and pyrimidine metabolism, proteoglycans and neuroprotection, and free radical generation and scavenging. This complex interplay allows us to identify potential interactions with commonly used medications (e.g., statins and acetaminophen) that may influence CBD efficacy and/or tolerability.

*AOX1* and *DPYD* were associated with response. *AOX1*, involved in retinaldehyde, benzaldehyde, and vanillin metabolism, can catalyze the formation of hydrogen peroxide and superoxide.[27](#cpt2408-bib-0027) CBD has been shown to decrease reactive oxygen species,[31](#cpt2408-bib-0031) a potential explanation of its therapeutic effect in TRE. Additionally, *AOX1* and *DPYD* are associated with impaired purine and pyrimidine metabolism,[32](#cpt2408-bib-0032) and have been implicated in amyotrophic lateral sclerosis (*AOX1*),[27](#cpt2408-bib-0027) and epilepsy (DPYD). DPYD is also responsible for the metabolism of fluoropyrimidines (fluorouracil and capecitabine).[33](#cpt2408-bib-0033) Although DPYD‐mediated response may be due to underlying biologic process, and not CBD metabolism, additional monitoring may be required if fluoropyrimidines and CBD are co‐administered.

Variants in CHSTs were associated with response (*CHST11*), and diarrhea and sedation (*CHST1*). CHST11 and CHST1 sulfate chondroitin and keratan sulfate proteoglycans, respectively. Both are important components of the brain extracellular matrix, and help form perineuronal nets, which are vital to neuroprotection, neuronal development, and plasticity.[34](#cpt2408-bib-0034), [35](#cpt2408-bib-0035) Additionally, CHST1 has been shown to play a role in inflammatory responses.[36](#cpt2408-bib-0036) Remodeling of chondroitin sulfate proteoglycans has been implicated in neurologic conditions, including epilepsy and Alzheimer’s disease.[37](#cpt2408-bib-0037) Given that variation in *CHSTs* was identified across CBD‐related outcomes, carbohydrate‐dependent mechanisms may contribute to CBD response and tolerability in TRE.

Variants in GSTs were associated with lower response (*GSTM5* and *GSTP1*), abnormal LFTs (*GSTM2*), diarrhea (*GSTA3*), and sedation (*GSTM3* and *GSTM5*). Glutathione is an antioxidant and free radical scavenger, with deficiency implicated in neurodegenerative conditions, including epilepsy, Alzheimer’s disease, and multiple sclerosis.[38](#cpt2408-bib-0038) CBD has been shown to increase glutathione activity, thereby decreasing oxidative stress.[39](#cpt2408-bib-0039) Although it is not clear if glutathione depletion contributes to CBD resistance, or if variation in *GSTM5* and *GSTP1* impair CBD‐mediated increases in glutathione activity, glutathione‐dependent pathways appear to play a role in CBD response and tolerability. Saturability of the glutathione conjugation pathway highlights the need to monitor patients on co‐therapy with drugs (e.g., acetaminophen) that utilize this pathway.

Genes involved in cholesterol and bile acid‐associated pathways were identified across CBD‐related outcomes and altered bile acid signaling has been implicated in epilepsy development.[40](#cpt2408-bib-0040) Genes involved in neural cholesterol conversion to bile acids (*CYP39A1*), bile acid conjugation (*UGT2B4*) and transport (*ABCB11*, *ABCB4*, *SLCO1B1*, and *SLCO1B3*), and apolipoprotein A2 (*APOA2*), were associated with CBD response and adverse effects. Additionally, *FMO4* rs2223477, associated with lower odds of response and diarrhea, affected the Farnesoid x receptor motif, involved in the regulation of genes involved in bile acid synthesis and transport.[27](#cpt2408-bib-0027) *ABCG1* rs914189, involved in phospholipid export (sphingomyelin, cholesterol, and oxysterols)[27](#cpt2408-bib-0027) has previously been associated with high‐density lipoprotein cholesterol levels.[41](#cpt2408-bib-0041) Furthermore, *SLCO1B1* rs11045819 A, encoding OATP1B1, involved in the transport of numerous endogenous substances and statin drugs, was associated with diarrhea. This variant is associated with increased fluvastatin efficacy, as measured by low‐density lipoprotein cholesterol reduction.[42](#cpt2408-bib-0042) Given the potential CBD‐statin interaction, monitoring of LFTs in patients on CBD‐statin co‐therapy should be considered.

Steroid hormone pathways were also implicated in CBD response and tolerability. In addition to bile acid conjugation, UGT2B4 is active on catechol estrogens, or endogenous estrogen metabolites. Catechol estrogens modulate calcium influx and insulin secretion through activation of the transient receptor potential (TRP) A1 channel,[43](#cpt2408-bib-0043) and CBD has activity at TRP channels, specifically TRPV1.[23](#cpt2408-bib-0023) *CYP17A1* rs6162, associated with higher odds of response, was previously associated with higher androstenedione levels, the precursor to estrone and testosterone.[44](#cpt2408-bib-0044) *AOX1* rs6729738, associated with response, influenced the GR motif, and GR‐β was identified in the promoter sequences of all co‐expressed genes. GR‐β does not bind glucocorticoids or activate glucocorticoid responsive genes, but antagonizes the effects of GR‐α, thereby inhibiting glucocorticoid effects.[45](#cpt2408-bib-0045) Given that patients with homozygous *AOX1* rs6729738 reference C alleles had higher odds of response, GR alteration may contribute to therapeutic response to CBD. Additionally, apart from drug metabolism, CYP1A2 is involved in cholesterol and caffeine metabolism, and the conversion of estrone to catecholestrogens.[27](#cpt2408-bib-0027) Given that lower ability to induce CYP1A2 was associated with lower odds of response, CYP1A2 induction may contribute to response in TRE. However, it is not clear if this is due to CYP1A2‐mediated CBD metabolism or underlying biologic processes. CYP1A2 activity is influenced by factors including smoking, diet, caffeine intake, and drugs (proton‐pump inhibitors, oral contraceptives, antibiotics, and antidepressants),[28](#cpt2408-bib-0028) and these factors may influence CBD response in TRE. The involvement of steroid hormone‐related pathways indicates that CBD response may be dependent on sex and age. We plan to assess these influences in larger CBD‐treated TRE cohorts.

Among variants associated with CBD response, we identified variants influencing the expression of GPCRs, tumor necrosis factor receptors, serotonin receptors, VGKCs, and PPARγ. Variation in *PPARγ* was associated with lower odds of response and was involved in a significant interaction with *UGT2B4*. Furthermore, CEBP β, whose transcription factor binding site was identified in the promoter sequences of all genes in the co‐expression network, induces CEBPα and *PPARγ* upon phosphorylation. *PPARγ* is highly expressed in adipose tissue and is involved in glucose metabolism and lipid storage.[27](#cpt2408-bib-0027) Given that ~ 80% of patients experienced some degree of weight loss, *PPARγ*‐mediated mechanisms influencing *UGT2B4* may contribute to CBD response. Additionally, we found that rs1339067, associated with lower odds of response, decreased the expression of the GPR18 endocannabinoid receptor. GPR18 shares some similarities with the GPR55 cannabinoid receptor, and both bind ligands that are not active at the CB_1_R and CB_2_R cannabinoid receptors.[46](#cpt2408-bib-0046) *N*‐arachidonyl glycine (NAGly), a metabolite of the endogenous cannabinoid anandamide, serves as the ligand for GPR18; however, the role of NAGly at GPR18 is not fully understood. We also found that rs3749442, associated with response and adverse effects, was associated with decreased expression of *HTR3E*, encoding the serotonin 5‐HT_3E_ receptor. The 5‐HT_3E_ receptor requires co‐expression with 5‐HT_3A_ to form a functional receptor, and serotonin has been shown to have increased activity at 5‐HT_3A/E_.[47](#cpt2408-bib-0047) Although not much is known about the pharmacology of the 5‐HT_3A/E_ receptor,[48](#cpt2408-bib-0048) it may play a role in modulating CBD response in TRE.

Although exquisitely phenotyped, we recognize limitations, including limited sample size, numerous ASD combinations, and the inability to assess response based on seizure subtypes. Additionally, data on concomitant medications and environmental factors (e.g., smoking, caffeine intake, and diet) was not available, so these influences could not be evaluated. Whereas we did not identify variants in *CYP2C19* associated with response, rs3758581 a component on numerous *CYP2C19* star alleles was associated with lower likelihood of sedation and higher likelihood of developing abnormal LFTs. However, this variant is also in complete LD with the *CYP2C9*3* (rs1057910) decreased function allele. Although CYP2C19 is a major CBD metabolic pathway, given the number of identified metabolites,[13](#cpt2408-bib-0013) non‐CYP2C19 dependent pathways may contribute to CBD effects and require further evaluation (i.e., aldehyde oxidase and flavin‐containing monooxygenases). Furthermore, because CBD dose was titrated based on response and adverse effects, this may have negated the effects of *CYP2C19* genetic variation.

Genetic variation in pharmacogenes is associated with CBD response and the development of adverse effects in TRE. Furthermore, variation in these genes influences the expression of potential CBD targets. Our study demonstrates that pharmacogenes implicated in CBD response are also involved in fundamental biologic processes and may offer novel insight into mechanisms through which CBD exerts its therapeutic effects in TRE and potential genetic underpinnings of treatment‐resistance. Additionally, as many of these pathways are associated with other neurodegenerative diseases, CBD’s therapeutic potential for neurodegenerative diseases needs further evaluation.

## FUNDING

This work was supported by the Alabama Cannabinoid Program and in part by National Institutes of Health (NIH) grants (K24HL133373 (N.A.L.) and T32HG008961 (B.H.D.)). GW Research Ltd. provided Epidiolex for the study.

## CONFLICT OF INTEREST

In the past 2 years, E.M.B. received funding from the NIH, State of Alabama, Greenwich Biosciences; has served on consulting/advisory boards for Greenwich Biosciences, Springworks Therapeutics, Inc., Neurelis, Aquestive Therapeutics, and REGENXBIO; and editorial board member for Pediatric Neurology Journal. In the last 2 years, J.P.S. received funding from the NIH, National Science Foundation, US Department of Defense, State of Alabama, Shor Foundation for Epilepsy Research, UCB Pharma, NeuroPace, Greenwich Biosciences, Biogen, Xenon Pharmaceuticals, and Serina Therapeutics; has served on consulting/advisory boards for Greenwich Biosciences, NeuroPace, Serina Therapeutics, LivaNova, UCB Pharma, and iFovea; and has served as an editorial board member for *Epilepsy & Behavior*, *Journal of Epileptology* (Associate Editor), *Epilepsy & Behavior Reports* (Associate Editor), *Journal of Medical Science*, *Epilepsy Currents* (Contributing Editor), and *Folia Medica Copernicana*. J.P.S. is a member of the faculty of the University of Alabama at Birmingham and is supported by endowment and University funds. J.P.S. is an investigator in studies funded by Abbvie, Inc., the American Parkinson Disease Association, the Michael J. Fox Foundation for Parkinson Research, Alabama Department of Commerce, the Department of Defense, and NIH grants P50NS108675, R25NS079188, and T32NS095775. He has a clinical practice and is compensated for these activities through the University of Alabama Health Services Foundation. In addition, since January 1, 2020, he has served as a consultant for or received honoraria from Abbvie Inc., Sutter Health, the International Parkinson Disease and Movement Disorder Society, Theravance, McGraw Hill, and Sanofi‐Aventis. In the last 2 years, T.G. has received consulting fees from Greenwich Biosciences, WebMD/MedScape, and Versar, Inc. All other authors declared no competing interests for this work.

## AUTHOR CONTRIBUTIONS

All authors wrote the manuscript. B.H.D., E.M.B., and N.A.L. designed the research. B.H.D., E.M.B., and N.A.L. performed the research. B.H.D., M.B., M.A., E.M.B., and N.A.L. analyzed the data.

## Supporting information

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Zack, M.M. & Kobau, R. National and state estimates of the numbers of adults and children with active epilepsy ‐ United States, 2015. MMWR Morb. Mortal Wkly. Rep. 66, 821–825 (2017).  [DOI](https://doi.org/10.15585/mmwr.mm6631a1) | [PMC free article](/articles/PMC5687788/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28796763/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=MMWR%20Morb.%20Mortal%20Wkly.%20Rep.&title=National%20and%20state%20estimates%20of%20the%20numbers%20of%20adults%20and%20children%20with%20active%20epilepsy%20%E2%80%90%20United%20States,%202015&author=M.M.%20Zack&author=R.%20Kobau&volume=66&publication_year=2017&pages=821-825&pmid=28796763&doi=10.15585/mmwr.mm6631a1&)

2. Vossler, D.G. , Weingarten, M. , Gidal, B.E. & American Epilepsy Society Treatments Committee . Summary of antiepileptic drugs available in the United States of America: working toward a world without epilepsy. Epilepsy Curr. 18, 1–26 (2018).  [DOI](https://doi.org/10.5698/1535-7597.18.4s1.1) | [PMC free article](/articles/PMC6130739/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30233275/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Curr.&title=Summary%20of%20antiepileptic%20drugs%20available%20in%20the%20United%20States%20of%20America:%20working%20toward%20a%20world%20without%20epilepsy&author=D.G.%20Vossler&author=M.%20Weingarten&author=B.E.%20Gidal&volume=18&publication_year=2018&pages=1-26&pmid=30233275&doi=10.5698/1535-7597.18.4s1.1&)

3. Kalilani, L. , Sun, X. , Pelgrims, B. , Noack‐Rink, M. & Villanueva, V. The epidemiology of drug‐resistant epilepsy: a systematic review and meta‐analysis. Epilepsia 59, 2179–2193 (2018).  [DOI](https://doi.org/10.1111/epi.14596) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30426482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=The%20epidemiology%20of%20drug%E2%80%90resistant%20epilepsy:%20a%20systematic%20review%20and%20meta%E2%80%90analysis&author=L.%20Kalilani&author=X.%20Sun&author=B.%20Pelgrims&author=M.%20Noack%E2%80%90Rink&author=V.%20Villanueva&volume=59&publication_year=2018&pages=2179-2193&pmid=30426482&doi=10.1111/epi.14596&)

4. Kwan, P. et al. Definition of drug resistant epilepsy: consensus proposal by the ad hoc Task Force of the ILAE Commission on Therapeutic Strategies. Epilepsia 51, 1069–1077 (2010).  [DOI](https://doi.org/10.1111/j.1528-1167.2009.02397.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19889013/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Definition%20of%20drug%20resistant%20epilepsy:%20consensus%20proposal%20by%20the%20ad%20hoc%20Task%20Force%20of%20the%20ILAE%20Commission%20on%20Therapeutic%20Strategies&author=P.%20Kwan&volume=51&publication_year=2010&pages=1069-1077&pmid=19889013&doi=10.1111/j.1528-1167.2009.02397.x&)

5. Sheikh, S.R. , Thompson, N. , Frech, F. , Malhotra, M. & Jehi, L. Quantifying the burden of generalized tonic‐clonic seizures in patients with drug‐resistant epilepsy. Epilepsia 61, 1627–1637 (2020).  [DOI](https://doi.org/10.1111/epi.16603) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32658343/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Quantifying%20the%20burden%20of%20generalized%20tonic%E2%80%90clonic%20seizures%20in%20patients%20with%20drug%E2%80%90resistant%20epilepsy&author=S.R.%20Sheikh&author=N.%20Thompson&author=F.%20Frech&author=M.%20Malhotra&author=L.%20Jehi&volume=61&publication_year=2020&pages=1627-1637&pmid=32658343&doi=10.1111/epi.16603&)

6. Szaflarski, J.P. et al. Long‐term safety and treatment effects of cannabidiol in children and adults with treatment‐resistant epilepsies: Expanded access program results. Epilepsia 59, 1540–1548 (2018).  [DOI](https://doi.org/10.1111/epi.14477) | [PMC free article](/articles/PMC6175436/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29998598/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Long%E2%80%90term%20safety%20and%20treatment%20effects%20of%20cannabidiol%20in%20children%20and%20adults%20with%20treatment%E2%80%90resistant%20epilepsies:%20Expanded%20access%20program%20results&author=J.P.%20Szaflarski&volume=59&publication_year=2018&pages=1540-1548&pmid=29998598&doi=10.1111/epi.14477&)

7. Szaflarski, J.P. et al. Cannabidiol improves frequency and severity of seizures and reduces adverse events in an open‐label add‐on prospective study. Epilepsy Behav. 87, 131–136 (2018).  [DOI](https://doi.org/10.1016/j.yebeh.2018.07.020) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30100226/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Behav.&title=Cannabidiol%20improves%20frequency%20and%20severity%20of%20seizures%20and%20reduces%20adverse%20events%20in%20an%20open%E2%80%90label%20add%E2%80%90on%20prospective%20study&author=J.P.%20Szaflarski&volume=87&publication_year=2018&pages=131-136&pmid=30100226&doi=10.1016/j.yebeh.2018.07.020&)

8. Szaflarski, J.P. et al. Higher cannabidiol plasma levels are associated with better seizure response following treatment with a pharmaceutical grade cannabidiol. Epilepsy Behav. 95, 131–136 (2019).  [DOI](https://doi.org/10.1016/j.yebeh.2019.03.042) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31048098/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Behav.&title=Higher%20cannabidiol%20plasma%20levels%20are%20associated%20with%20better%20seizure%20response%20following%20treatment%20with%20a%20pharmaceutical%20grade%20cannabidiol&author=J.P.%20Szaflarski&volume=95&publication_year=2019&pages=131-136&pmid=31048098&doi=10.1016/j.yebeh.2019.03.042&)

9. Epidiolex [package insert] (Greenwich Biosciences, Inc., Carlsbad, CA, 2020).  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Epidiolex%20%5Bpackage%20insert%5D&publication_year=2020&)

10. Jiang, R. , Yamaori, S. , Takeda, S. , Yamamoto, I. & Watanabe, K. Identification of cytochrome P450 enzymes responsible for metabolism of cannabidiol by human liver microsomes. Life Sci. 89, 165–170 (2011).  [DOI](https://doi.org/10.1016/j.lfs.2011.05.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21704641/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Life%20Sci.&title=Identification%20of%20cytochrome%20P450%20enzymes%20responsible%20for%20metabolism%20of%20cannabidiol%20by%20human%20liver%20microsomes&author=R.%20Jiang&author=S.%20Yamaori&author=S.%20Takeda&author=I.%20Yamamoto&author=K.%20Watanabe&volume=89&publication_year=2011&pages=165-170&pmid=21704641&doi=10.1016/j.lfs.2011.05.018&)

11. Yamaori, S. , Okamoto, Y. , Yamamoto, I. & Watanabe, K. Cannabidiol, a major phytocannabinoid, as a potent atypical inhibitor for CYP2D6. Drug Metab. Dispos. 39, 2049–2056 (2011).  [DOI](https://doi.org/10.1124/dmd.111.041384) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21821735/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Cannabidiol,%20a%20major%20phytocannabinoid,%20as%20a%20potent%20atypical%20inhibitor%20for%20CYP2D6&author=S.%20Yamaori&author=Y.%20Okamoto&author=I.%20Yamamoto&author=K.%20Watanabe&volume=39&publication_year=2011&pages=2049-2056&pmid=21821735&doi=10.1124/dmd.111.041384&)

12. Pérez‐Acevedo, A.P. et al. Disposition of cannabidiol metabolites in serum and urine from healthy individuals treated with pharmaceutical preparations of medical cannabis. Pharmaceuticals (Basel) 13, 459 (2020).  [DOI](https://doi.org/10.3390/ph13120459) | [PMC free article](/articles/PMC7763054/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33322849/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmaceuticals%20(Basel)&title=Disposition%20of%20cannabidiol%20metabolites%20in%20serum%20and%20urine%20from%20healthy%20individuals%20treated%20with%20pharmaceutical%20preparations%20of%20medical%20cannabis&author=A.P.%20P%C3%A9rez%E2%80%90Acevedo&volume=13&publication_year=2020&pages=459&pmid=33322849&doi=10.3390/ph13120459&)

13. Ujváry, I. & Hanuš, L. Human metabolites of cannabidiol: a review on their formation, biological activity, and relevance in therapy. Cannabis Cannabinoid Res. 1, 90–101 (2016).  [DOI](https://doi.org/10.1089/can.2015.0012) | [PMC free article](/articles/PMC5576600/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28861484/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cannabis%20Cannabinoid%20Res.&title=Human%20metabolites%20of%20cannabidiol:%20a%20review%20on%20their%20formation,%20biological%20activity,%20and%20relevance%20in%20therapy&author=I.%20Ujv%C3%A1ry&author=L.%20Hanu%C5%A1&volume=1&publication_year=2016&pages=90-101&pmid=28861484&doi=10.1089/can.2015.0012&)

14. Mazur, A. et al. Characterization of human hepatic and extrahepatic UDP‐glucuronosyltransferase enzymes involved in the metabolism of classic cannabinoids. Drug Metab. Dispos. 37, 1496–1504 (2009).  [DOI](https://doi.org/10.1124/dmd.109.026898) | [PMC free article](/articles/PMC2698943/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19339377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Characterization%20of%20human%20hepatic%20and%20extrahepatic%20UDP%E2%80%90glucuronosyltransferase%20enzymes%20involved%20in%20the%20metabolism%20of%20classic%20cannabinoids&author=A.%20Mazur&volume=37&publication_year=2009&pages=1496-1504&pmid=19339377&doi=10.1124/dmd.109.026898&)

15. Roden, D.M. et al. Pharmacogenomics. Lancet 394, 521–532 (2019).  [DOI](https://doi.org/10.1016/S0140-6736(19)31276-0) | [PMC free article](/articles/PMC6707519/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31395440/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Pharmacogenomics&author=D.M.%20Roden&volume=394&publication_year=2019&pages=521-532&pmid=31395440&doi=10.1016/S0140-6736(19)31276-0&)

16. Berger, S.I. & Iyengar, R. Role of systems pharmacology in understanding drug adverse events. Wiley Interdiscip. Rev. Syst. Biol. Med. 3, 129–135 (2011).  [DOI](https://doi.org/10.1002/wsbm.114) | [PMC free article](/articles/PMC3057924/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20803507/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Wiley%20Interdiscip.%20Rev.%20Syst.%20Biol.%20Med.&title=Role%20of%20systems%20pharmacology%20in%20understanding%20drug%20adverse%20events&author=S.I.%20Berger&author=R.%20Iyengar&volume=3&publication_year=2011&pages=129-135&pmid=20803507&doi=10.1002/wsbm.114&)

17. Arbitrio, M. et al. DMET™ (Drug Metabolism Enzymes and Transporters): a pharmacogenomic platform for precision medicine. Oncotarget 7, 54028–54050 (2016).  [DOI](https://doi.org/10.18632/oncotarget.9927) | [PMC free article](/articles/PMC5288240/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27304055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=DMET%E2%84%A2%20(Drug%20Metabolism%20Enzymes%20and%20Transporters):%20a%20pharmacogenomic%20platform%20for%20precision%20medicine&author=M.%20Arbitrio&volume=7&publication_year=2016&pages=54028-54050&pmid=27304055&doi=10.18632/oncotarget.9927&)

18. Ward, L.D. & Kellis, M. HaploReg: a resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Nucleic Acids Res. 40, D930–D934 (2012).  [DOI](https://doi.org/10.1093/nar/gkr917) | [PMC free article](/articles/PMC3245002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22064851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=HaploReg:%20a%20resource%20for%20exploring%20chromatin%20states,%20conservation,%20and%20regulatory%20motif%20alterations%20within%20sets%20of%20genetically%20linked%20variants&author=L.D.%20Ward&author=M.%20Kellis&volume=40&publication_year=2012&pages=D930-D934&pmid=22064851&doi=10.1093/nar/gkr917&)

19. Gaston, T.E. et al. Drug‐drug interactions with cannabidiol (CBD) appear to have no effect on treatment response in an open‐label Expanded Access Program. Epilepsy Behav. 98, 201–206 (2019).  [DOI](https://doi.org/10.1016/j.yebeh.2019.07.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31382177/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Behav.&title=Drug%E2%80%90drug%20interactions%20with%20cannabidiol%20(CBD)%20appear%20to%20have%20no%20effect%20on%20treatment%20response%20in%20an%20open%E2%80%90label%20Expanded%20Access%20Program&author=T.E.%20Gaston&volume=98&publication_year=2019&pages=201-206&pmid=31382177&doi=10.1016/j.yebeh.2019.07.008&)

20. Patsalos, P.N. et al. Clinical implications of trials investigating drug‐drug interactions between cannabidiol and enzyme inducers or inhibitors or common antiseizure drugs. Epilepsia 61, 1854–1868 (2020).  [DOI](https://doi.org/10.1111/epi.16674) | [PMC free article](/articles/PMC7693203/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32918835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Clinical%20implications%20of%20trials%20investigating%20drug%E2%80%90drug%20interactions%20between%20cannabidiol%20and%20enzyme%20inducers%20or%20inhibitors%20or%20common%20antiseizure%20drugs&author=P.N.%20Patsalos&volume=61&publication_year=2020&pages=1854-1868&pmid=32918835&doi=10.1111/epi.16674&)

21. Gaston, T.E. et al. Interactions between cannabidiol and commonly used antiepileptic drugs. Epilepsia 58, 1586–1592 (2017).  [DOI](https://doi.org/10.1111/epi.13852) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28782097/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Interactions%20between%20cannabidiol%20and%20commonly%20used%20antiepileptic%20drugs&author=T.E.%20Gaston&volume=58&publication_year=2017&pages=1586-1592&pmid=28782097&doi=10.1111/epi.13852&)

22. Sham, P.C. & Purcell, S.M. Statistical power and significance testing in large‐scale genetic studies. Nat. Rev. Genet. 15, 335–346 (2014).  [DOI](https://doi.org/10.1038/nrg3706) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24739678/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&title=Statistical%20power%20and%20significance%20testing%20in%20large%E2%80%90scale%20genetic%20studies&author=P.C.%20Sham&author=S.M.%20Purcell&volume=15&publication_year=2014&pages=335-346&pmid=24739678&doi=10.1038/nrg3706&)

23. Gaston, T.E. & Friedman, D. Pharmacology of cannabinoids in the treatment of epilepsy. Epilepsy Behav. 70, 313–318 (2017).  [DOI](https://doi.org/10.1016/j.yebeh.2016.11.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28087250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsy%20Behav.&title=Pharmacology%20of%20cannabinoids%20in%20the%20treatment%20of%20epilepsy&author=T.E.%20Gaston&author=D.%20Friedman&volume=70&publication_year=2017&pages=313-318&pmid=28087250&doi=10.1016/j.yebeh.2016.11.016&)

24. Ramasamy, A. et al. Genetic variability in the regulation of gene expression in ten regions of the human brain. Nat. Neurosci. 17, 1418–1428 (2014).  [DOI](https://doi.org/10.1038/nn.3801) | [PMC free article](/articles/PMC4208299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25174004/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Neurosci.&title=Genetic%20variability%20in%20the%20regulation%20of%20gene%20expression%20in%20ten%20regions%20of%20the%20human%20brain&author=A.%20Ramasamy&volume=17&publication_year=2014&pages=1418-1428&pmid=25174004&doi=10.1038/nn.3801&)

25. Okamura, Y. et al. COXPRESdb in 2015: coexpression database for animal species by DNA‐microarray and RNAseq‐based expression data with multiple quality assessment systems. Nucleic Acids Res. 43, D82–D86 (2015).  [DOI](https://doi.org/10.1093/nar/gku1163) | [PMC free article](/articles/PMC4383961/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25392420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=COXPRESdb%20in%202015:%20coexpression%20database%20for%20animal%20species%20by%20DNA%E2%80%90microarray%20and%20RNAseq%E2%80%90based%20expression%20data%20with%20multiple%20quality%20assessment%20systems&author=Y.%20Okamura&volume=43&publication_year=2015&pages=D82-D86&pmid=25392420&doi=10.1093/nar/gku1163&)

26. Messeguer, X. et al. PROMO: detection of known transcription regulatory elements using species‐tailored searches. Bioinformatics 18, 333–334 (2002).  [DOI](https://doi.org/10.1093/bioinformatics/18.2.333) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11847087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=PROMO:%20detection%20of%20known%20transcription%20regulatory%20elements%20using%20species%E2%80%90tailored%20searches&author=X.%20Messeguer&volume=18&publication_year=2002&pages=333-334&pmid=11847087&doi=10.1093/bioinformatics/18.2.333&)

27. Stelzer, G. et al. The GeneCards Suite: from gene data mining to disease genome sequence analyses. Curr. Protoc. Bioinformat. 54, 1.30.1‐1.30.33 (2016).  [DOI](https://doi.org/10.1002/cpbi.5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27322403/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.%20Bioinformat.&title=The%20GeneCards%20Suite:%20from%20gene%20data%20mining%20to%20disease%20genome%20sequence%20analyses&author=G.%20Stelzer&volume=54&publication_year=2016&pages=1.30.1%E2%80%901.30.33&pmid=27322403&doi=10.1002/cpbi.5&)

28. Thorn, C.F. , Aklillu, E. , Klein, T.E. & Altman, R.B. PharmGKB summary: very important pharmacogene information for CYP1A2. Pharmacogenet. Genomics 22, 73–77 (2012).  [DOI](https://doi.org/10.1097/FPC.0b013e32834c6efd) | [PMC free article](/articles/PMC3346273/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21989077/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=PharmGKB%20summary:%20very%20important%20pharmacogene%20information%20for%20CYP1A2&author=C.F.%20Thorn&author=E.%20Aklillu&author=T.E.%20Klein&author=R.B.%20Altman&volume=22&publication_year=2012&pages=73-77&pmid=21989077&doi=10.1097/FPC.0b013e32834c6efd&)

29. Bangsow, T. et al. The epithelial membrane protein 1 is a novel tight junction protein of the blood‐brain barrier. J. Cereb. Blood Flow. Metab. 28, 1249–1260 (2008).  [DOI](https://doi.org/10.1038/jcbfm.2008.19) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18382472/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Cereb.%20Blood%20Flow.%20Metab.&title=The%20epithelial%20membrane%20protein%201%20is%20a%20novel%20tight%20junction%20protein%20of%20the%20blood%E2%80%90brain%20barrier&author=T.%20Bangsow&volume=28&publication_year=2008&pages=1249-1260&pmid=18382472&doi=10.1038/jcbfm.2008.19&)

30. Boyle, A.P. et al. Annotation of functional variation in personal genomes using RegulomeDB. Genome Res. 22, 1790–1797 (2012).  [DOI](https://doi.org/10.1101/gr.137323.112) | [PMC free article](/articles/PMC3431494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&title=Annotation%20of%20functional%20variation%20in%20personal%20genomes%20using%20RegulomeDB&author=A.P.%20Boyle&volume=22&publication_year=2012&pages=1790-1797&pmid=22955989&doi=10.1101/gr.137323.112&)

31. Silvestro, S. , Schepici, G. , Bramanti, P. & Mazzon, E. Molecular targets of cannabidiol in experimental models of neurological disease. Molecules 25, 5186 (2020).  [DOI](https://doi.org/10.3390/molecules25215186) | [PMC free article](/articles/PMC7664437/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33171772/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecules&title=Molecular%20targets%20of%20cannabidiol%20in%20experimental%20models%20of%20neurological%20disease&author=S.%20Silvestro&author=G.%20Schepici&author=P.%20Bramanti&author=E.%20Mazzon&volume=25&publication_year=2020&pages=5186&pmid=33171772&doi=10.3390/molecules25215186&)

32. Jurecka, A. Inborn errors of purine and pyrimidine metabolism. J. Inherit. Metab. Dis. 32, 247–263 (2009).  [DOI](https://doi.org/10.1007/s10545-009-1094-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19291420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Inherit.%20Metab.%20Dis.&title=Inborn%20errors%20of%20purine%20and%20pyrimidine%20metabolism&author=A.%20Jurecka&volume=32&publication_year=2009&pages=247-263&pmid=19291420&doi=10.1007/s10545-009-1094-z&)

33. Caudle, K.E. et al. Clinical pharmacogenetics implementation consortium guidelines for dihydropyrimidine dehydrogenase genotype and fluoropyrimidine dosing. Clin. Pharmacol. Ther. 94, 640–645 (2013).  [DOI](https://doi.org/10.1038/clpt.2013.172) | [PMC free article](/articles/PMC3831181/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23988873/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guidelines%20for%20dihydropyrimidine%20dehydrogenase%20genotype%20and%20fluoropyrimidine%20dosing&author=K.E.%20Caudle&volume=94&publication_year=2013&pages=640-645&pmid=23988873&doi=10.1038/clpt.2013.172&)

34. Melrose, J. Keratan sulfate (KS)‐proteoglycans and neuronal regulation in health and disease: the importance of KS‐glycodynamics and interactive capability with neuroregulatory ligands. J. Neurochem. 149, 170–194 (2019).  [DOI](https://doi.org/10.1111/jnc.14652) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30578672/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Neurochem.&title=Keratan%20sulfate%20(KS)%E2%80%90proteoglycans%20and%20neuronal%20regulation%20in%20health%20and%20disease:%20the%20importance%20of%20KS%E2%80%90glycodynamics%20and%20interactive%20capability%20with%20neuroregulatory%20ligands&author=J.%20Melrose&volume=149&publication_year=2019&pages=170-194&pmid=30578672&doi=10.1111/jnc.14652&)

35. Miller, G.M. & Hsieh‐Wilson, L.C. Sugar‐dependent modulation of neuronal development, regeneration, and plasticity by chondroitin sulfate proteoglycans. Exp. Neurol. 274, 115–125 (2015).  [DOI](https://doi.org/10.1016/j.expneurol.2015.08.015) | [PMC free article](/articles/PMC4679498/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26315937/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp.%20Neurol.&title=Sugar%E2%80%90dependent%20modulation%20of%20neuronal%20development,%20regeneration,%20and%20plasticity%20by%20chondroitin%20sulfate%20proteoglycans&author=G.M.%20Miller&author=L.C.%20Hsieh%E2%80%90Wilson&volume=274&publication_year=2015&pages=115-125&pmid=26315937&doi=10.1016/j.expneurol.2015.08.015&)

36. Li, X. et al. CHST1 and CHST2 sulfotransferase expression by vascular endothelial cells regulates shear‐resistant leukocyte rolling via L‐selectin. J. Leukoc. Biol. 69, 565–574 (2001).  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11310842/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Leukoc.%20Biol.&title=CHST1%20and%20CHST2%20sulfotransferase%20expression%20by%20vascular%20endothelial%20cells%20regulates%20shear%E2%80%90resistant%20leukocyte%20rolling%20via%20L%E2%80%90selectin&author=X.%20Li&volume=69&publication_year=2001&pages=565-574&pmid=11310842&)

37. Smith, P.D. , Coulson‐Thomas, V.J. , Foscarin, S. , Kwok, J.C. & Fawcett, J.W. "GAG‐ing with the neuron": The role of glycosaminoglycan patterning in the central nervous system. Exp. Neurol. 274, 100–114 (2015).  [DOI](https://doi.org/10.1016/j.expneurol.2015.08.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26277685/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp.%20Neurol.&title=%22GAG%E2%80%90ing%20with%20the%20neuron%22:%20The%20role%20of%20glycosaminoglycan%20patterning%20in%20the%20central%20nervous%20system&author=P.D.%20Smith&author=V.J.%20Coulson%E2%80%90Thomas&author=S.%20Foscarin&author=J.C.%20Kwok&author=J.W.%20Fawcett&volume=274&publication_year=2015&pages=100-114&pmid=26277685&doi=10.1016/j.expneurol.2015.08.004&)

38. Rae, C.D. & Williams, S.R. Glutathione in the human brain: Review of its roles and measurement by magnetic resonance spectroscopy. Anal. Biochem. 529, 127–143 (2017).  [DOI](https://doi.org/10.1016/j.ab.2016.12.022) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28034792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal.%20Biochem.&title=Glutathione%20in%20the%20human%20brain:%20Review%20of%20its%20roles%20and%20measurement%20by%20magnetic%20resonance%20spectroscopy&author=C.D.%20Rae&author=S.R.%20Williams&volume=529&publication_year=2017&pages=127-143&pmid=28034792&doi=10.1016/j.ab.2016.12.022&)

39. Atalay, S. , Jarocka‐Karpowicz, I. & Skrzydlewska, E. Antioxidative and anti‐inflammatory properties of cannabidiol. Antioxidants (Basel) 9, 21 (2019).  [DOI](https://doi.org/10.3390/antiox9010021) | [PMC free article](/articles/PMC7023045/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31881765/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Antioxidants%20(Basel)&title=Antioxidative%20and%20anti%E2%80%90inflammatory%20properties%20of%20cannabidiol&author=S.%20Atalay&author=I.%20Jarocka%E2%80%90Karpowicz&author=E.%20Skrzydlewska&volume=9&publication_year=2019&pages=21&pmid=31881765&doi=10.3390/antiox9010021&)

40. Grant, S.M. & DeMorrow, S. Bile acid signaling in neurodegenerative and neurological disorders. Int. J. Mol. Sci. 21, 5982 (2020).  [DOI](https://doi.org/10.3390/ijms21175982) | [PMC free article](/articles/PMC7503576/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32825239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Bile%20acid%20signaling%20in%20neurodegenerative%20and%20neurological%20disorders&author=S.M.%20Grant&author=S.%20DeMorrow&volume=21&publication_year=2020&pages=5982&pmid=32825239&doi=10.3390/ijms21175982&)

41. Edmondson, A.C. et al. Dense genotyping of candidate gene loci identifies variants associated with high‐density lipoprotein cholesterol. Circ. Cardiovasc. Genet. 4, 145–155 (2011).  [DOI](https://doi.org/10.1161/CIRCGENETICS.110.957563) | [PMC free article](/articles/PMC3319351/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21303902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circ.%20Cardiovasc.%20Genet.&title=Dense%20genotyping%20of%20candidate%20gene%20loci%20identifies%20variants%20associated%20with%20high%E2%80%90density%20lipoprotein%20cholesterol&author=A.C.%20Edmondson&volume=4&publication_year=2011&pages=145-155&pmid=21303902&doi=10.1161/CIRCGENETICS.110.957563&)

42. Couvert, P. et al. Association between a frequent allele of the gene encoding OATP1B1 and enhanced LDL‐lowering response to fluvastatin therapy. Pharmacogenomics 9, 1217–1227 (2008).  [DOI](https://doi.org/10.2217/14622416.9.9.1217) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18781850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Association%20between%20a%20frequent%20allele%20of%20the%20gene%20encoding%20OATP1B1%20and%20enhanced%20LDL%E2%80%90lowering%20response%20to%20fluvastatin%20therapy&author=P.%20Couvert&volume=9&publication_year=2008&pages=1217-1227&pmid=18781850&doi=10.2217/14622416.9.9.1217&)

43. Ma, W. et al. Catechol estrogens stimulate insulin secretion in pancreatic β‐cells via activation of the transient receptor potential A1 (TRPA1) channel. J. Biol. Chem. 294, 2935–2946 (2019).  [DOI](https://doi.org/10.1074/jbc.RA118.005504) | [PMC free article](/articles/PMC6393604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30587572/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=Catechol%20estrogens%20stimulate%20insulin%20secretion%20in%20pancreatic%20%CE%B2%E2%80%90cells%20via%20activation%20of%20the%20transient%20receptor%20potential%20A1%20(TRPA1)%20channel&author=W.%20Ma&volume=294&publication_year=2019&pages=2935-2946&pmid=30587572&doi=10.1074/jbc.RA118.005504&)

44. Al‐Rubae'i, S.H. , Naji, T.S. & Turki, K.M. Common variation of the CYP17 gene in Iraqi women with endometriosis disease. Genomics Data 11, 55–59 (2017).  [DOI](https://doi.org/10.1016/j.gdata.2016.11.019) | [PMC free article](/articles/PMC5144748/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27981034/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genomics%20Data&title=Common%20variation%20of%20the%20CYP17%20gene%20in%20Iraqi%20women%20with%20endometriosis%20disease&author=S.H.%20Al%E2%80%90Rubae'i&author=T.S.%20Naji&author=K.M.%20Turki&volume=11&publication_year=2017&pages=55-59&pmid=27981034&doi=10.1016/j.gdata.2016.11.019&)

45. Kino, T. , Su, Y.A. & Chrousos, G.P. Human glucocorticoid receptor isoform beta: recent understanding of its potential implications in physiology and pathophysiology. Cell. Mol. Life Sci. 66, 3435–3448 (2009).  [DOI](https://doi.org/10.1007/s00018-009-0098-z) | [PMC free article](/articles/PMC2796272/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19633971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell.%20Mol.%20Life%20Sci.&title=Human%20glucocorticoid%20receptor%20isoform%20beta:%20recent%20understanding%20of%20its%20potential%20implications%20in%20physiology%20and%20pathophysiology&author=T.%20Kino&author=Y.A.%20Su&author=G.P.%20Chrousos&volume=66&publication_year=2009&pages=3435-3448&pmid=19633971&doi=10.1007/s00018-009-0098-z&)

46. Morales, P. et al. Therapeutic exploitation of GPR18: beyond the cannabinoids? J. Med. Chem. 63, 14216–14227 (2020).  [DOI](https://doi.org/10.1021/acs.jmedchem.0c00926) | [PMC free article](/articles/PMC7949482/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32914978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&title=Therapeutic%20exploitation%20of%20GPR18:%20beyond%20the%20cannabinoids?&author=P.%20Morales&volume=63&publication_year=2020&pages=14216-14227&pmid=32914978&doi=10.1021/acs.jmedchem.0c00926&)

47. Niesler, B. et al. Characterization of the novel human serotonin receptor subunits 5‐HT3C,5‐HT3D, and 5‐HT3E. Mol. Pharmacol. 72, 8–17 (2007).  [DOI](https://doi.org/10.1124/mol.106.032144) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17392525/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=Characterization%20of%20the%20novel%20human%20serotonin%20receptor%20subunits%205%E2%80%90HT3C,5%E2%80%90HT3D,%20and%205%E2%80%90HT3E&author=B.%20Niesler&volume=72&publication_year=2007&pages=8-17&pmid=17392525&doi=10.1124/mol.106.032144&)

48. Fakhfouri, G. , Rahimian, R. , Dyhrfjeld‐Johnsen, J. , Zirak, M.R. & Beaulieu, J.M. 5‐HT3 receptor antagonists in neurologic and neuropsychiatric disorders: the iceberg still lies beneath the surface. Pharmacol. Rev. 71, 383–412 (2019).  [DOI](https://doi.org/10.1124/pr.118.015487) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31243157/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Rev.&title=5%E2%80%90HT3%20receptor%20antagonists%20in%20neurologic%20and%20neuropsychiatric%20disorders:%20the%20iceberg%20still%20lies%20beneath%20the%20surface&author=G.%20Fakhfouri&author=R.%20Rahimian&author=J.%20Dyhrfjeld%E2%80%90Johnsen&author=M.R.%20Zirak&author=J.M.%20Beaulieu&volume=71&publication_year=2019&pages=383-412&pmid=31243157&doi=10.1124/pr.118.015487&)

49. Civelli, O. et al. G protein‐coupled receptor deorphanizations. Annu. Rev. Pharmacol. Toxicol. 53, 127–146 (2013).  [DOI](https://doi.org/10.1146/annurev-pharmtox-010611-134548) | [PMC free article](/articles/PMC5828024/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23020293/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Pharmacol.%20Toxicol.&title=G%20protein%E2%80%90coupled%20receptor%20deorphanizations&author=O.%20Civelli&volume=53&publication_year=2013&pages=127-146&pmid=23020293&doi=10.1146/annurev-pharmtox-010611-134548&)

50. Calver, A.R. et al. Molecular cloning and characterisation of a novel GABAB‐related G‐protein coupled receptor. Brain Res. Mol. Brain Res. 110, 305–317 (2003).  [DOI](https://doi.org/10.1016/s0169-328x(02)00662-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12591167/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brain%20Res.%20Mol.%20Brain%20Res.&title=Molecular%20cloning%20and%20characterisation%20of%20a%20novel%20GABAB%E2%80%90related%20G%E2%80%90protein%20coupled%20receptor&author=A.R.%20Calver&volume=110&publication_year=2003&pages=305-317&pmid=12591167&doi=10.1016/s0169-328x(02)00662-9&)
