# Effects of a Common Eight Base Pairs Duplication at the Exon 7-Intron 7 Junction on Splicing, Expression, and Function of OCT1

## Metadata
**Authors:** Sarah Römer, Marleen J Meyer, Kathrin Klein, Lennart V Schneider, Johannes Matthaei, Ana Tzvetkova, Joanna Łapczuk-Romańska, Jochen Gaedcke, Marek Droździk, Jürgen Brockmöller, Anne T Nies, Mladen V Tzvetkov
**Journal:** Frontiers in Pharmacology
**Date:** 2021 May 7
**DOI:** [10.3389/fphar.2021.661480](https://doi.org/10.3389/fphar.2021.661480)
**PMID:** 34025422
**PMCID:** PMC8137991
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8137991/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8137991/pdf/fphar-12-661480.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8137991/pdf/fphar-12-661480.pdf)

## Abstract

Organic cation transporter 1 (OCT1, SLC22A1) is localized in the sinusoidal membrane of human hepatocytes and mediates hepatic uptake of weakly basic or cationic drugs and endogenous compounds. Common amino acid substitutions in OCT1 were associated with altered pharmacokinetics and efficacy of drugs like sumatriptan and fenoterol. Recently, the common splice variant rs35854239 has also been suggested to affect OCT1 function. rs35854239 represents an 8 bp duplication of the donor splice site at the exon 7-intron 7 junction. Here we quantified the extent to which this duplication affects OCT1 splicing and, as a consequence, the expression and the function of OCT1. We used pyrosequencing and deep RNA-sequencing to quantify the effect of rs35854239 on splicing after minigene expression of this variant in HepG2 and Huh7 cells and directly in human liver samples. Further, we analyzed the effects of rs35854239 on OCT1 mRNA expression in total, localization and activity of the resulting OCT1 protein, and on the pharmacokinetics of sumatriptan and fenoterol. The 8 bp duplication caused alternative splicing in 38% (deep RNA-sequencing) to 52% (pyrosequencing) of the minigene transcripts when analyzed in HepG2 and Huh7 cells. The alternatively spliced transcript encodes for a truncated protein that after transient transfection in HEK293 cells was not localized in the plasma membrane and was not able to transport the OCT1 model substrate ASP+. In human liver, however, the alternatively spliced OCT1 transcript was detectable only at very low levels (0.3% in heterozygous and 0.6% in homozygous carriers of the 8 bp duplication, deep RNA-sequencing). The 8 bp duplication was associated with a significant reduction of OCT1 expression in the human liver, but explained only 9% of the general variability in OCT1 expression and was not associated with significant changes in the pharmacokinetics of sumatriptan and fenoterol. Therefore, the rs35854239 variant only partially changes splicing, causing moderate changes in OCT1 expression and may be of only limited therapeutic relevance.

Keywords: ins/del variant, organic cation transporter 1, SLC22A1, minigene, allelic expression imbalance (AEI), fenoterol, sumatriptan, pharmacokinetics

## Introduction

OCT1 (SLC22A1) is by far the most strongly expressed transporter of organic cations in the sinusoidal membrane of the human liver ([Nies et al., 2009](#B19); [Wang et al., 2015](#B45); [Drozdzik et al., 2019](#B5)). OCT1 mediates the first step of hepatic clearance of weakly basic or positively charged drugs. Metformin, morphine, sumatriptan or fenoterol and endogenous compounds like thiamine belong to substrates transported by OCT1 ([Wang et al., 2002](#B43); [Tzvetkov et al., 2013](#B37); [Chen et al., 2014](#B4); [Matthaei et al., 2016](#B17); [Tzvetkov et al., 2018](#B38)). A loss of OCT1 function was shown to increase plasma concentrations of several drugs including sumatriptan and fenoterol ([Kerb et al., 2002](#B12); [Shu et al., 2007](#B31); [Tzvetkov et al., 2013](#B37); [Arimany-Nardi et al., 2016](#B1); [Matthaei et al., 2016](#B17); [Tzvetkov et al., 2018](#B38)). Depending on administered drugs, an increase may bear the risk of toxic side effects and may affect drug efficacy.

OCT1 is encoded by the SLC22A1 gene, which is located on the long arm of human chromosome 6 (6q26) and contains 11 exons and 10 introns ([Koehler et al., 1997](#B15); [Zhang et al., 1997](#B48)). The resulting OCT1 protein has 554 amino acids and is composed of 12 transmembrane helices (TMHs) with intracellularly localized N- and C-termini.

The SLC22A1 gene shows the highest genetic variability within the pharmacologically relevant members of the SLC22 family ([Tzvetkov et al., 2016](#B36); [Schaller and Lauschke, 2019](#B26)). Fourteen single nucleotide polymorphisms (SNPs) result in amino acid substitutions. Thereof, four common amino acid substitutions (Arg61Cys, Cys88Arg, Gly401Ser, and Gly465Arg) and a deletion of Met420 are known to confer strongly reduced or completely abolished OCT1 activity ([Kerb et al., 2002](#B12); [Shu et al., 2003](#B30); [Shu et al., 2008](#B29); [Seitz et al., 2015](#B27)). Nine percent of Europeans and White Americans are homozygous or compound heterozygous carriers of these reduce function variants ([Seitz et al., 2015](#B27)). These individuals (also referred to as poor OCT1 transporters) have significantly altered pharmacokinetics resulting in altered efficacy and toxicity of clinically relevant drugs like sumatriptan, fenoterol, tramadol and morphine ([Shu et al., 2008](#B29); [Becker et al., 2011](#B2); [Tzvetkov et al., 2011](#B40); [Tzvetkov et al., 2012](#B39); [Fukuda et al., 2013](#B8); [Tzvetkov et al., 2013](#B37); [Stamer et al., 2016](#B33)).

Non-coding variants may also affect OCT1 activity, e.g. by altering OCT1 expression. Indeed, SLC22A1 expression varies strongly between individuals ([Nies et al., 2009](#B19); [O’Brien et al., 2013](#B22)). The OCT1 mRNA levels differ up to 113-fold and protein levels up to 83-fold between individuals ([Nies et al., 2009](#B19)). However, common promoter variants did not significantly affect the SLC22A1 promoter activity or mRNA expression ([Bokelmann et al., 2018](#B3)).

Another explanation of the high variability in OCT1 expression may be related to genetic variants that cause alternative splicing. Indeed, an 8 base pairs insertion/deletion variant rs35854239 (formerly also designated as rs113569197 or rs36056065) was suggested to affect OCT1 expression and activity by altering splicing ([Tarasova et al., 2012](#B35); [Grinfeld et al., 2013](#B10); [Kim et al., 2017](#B14)). This variant is located at the junction of exon 7 and intron 7 of the SLC22A1 gene and represents an 8 bp duplication of the 5’ part of intron 7 including the splice donor site ([Figure 1A](#F1)). The newly generated donor site results in an 8 bp longer transcript with shift in the open reading frame and a premature stop-codon.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d6e/8137991/8a9688fb69e7/fphar-12-661480-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8137991_fphar-12-661480-g001.jpg)

Effects of rs35854239 on SLC22A1 exon 7 minigene splicing in Huh7 and HepG2 (A) Splicing at the exon 7-intron 7 junction. Splice donor sites within the intronic sequence are shown in red. The 8 bp insertion/deletion variant rs35854239 carries a second splice donor site that is proposed to be spliced alternatively. (B) Representation of the pSPL3b splicing vector consisting of two exons of the rabbit β-globulin gene under control of the SV40 promoter (the “minigene”). SLC22A1 exon 7 and its flanking intronic regions with or without rs35854239 (referred to as duplication or wild-type, respectively) were cloned between both exons of the rabbit β-globulin gene. Minigene constructs were transiently transfected into Huh7 and HepG2 cells, and mRNA was isolated 48 h after transfection. As positive control we used the CYP2C19*2 variant, for which alternative splicing is known, (Morais et al., 1994). (C,D) Correctly and alternatively spliced OCT1 transcripts were shown (C) and quantified using pyrosequencing (D), or deep RNA-sequencing (E). Percentages within boxes represent relative values of correctly spliced minigene transcripts. Data are shown as mean and standard errors of the mean of at least three independent experiments.

The rs35854239 variant is genetically highly linked to the coding variant Met408Val (*r* ^2^ of 0.95). In several studies, Met408Val was associated with drug efficacy. This association has been explained by decreasing cellular uptake and thus altering the systemic concentrations or concentrations at the site of action of the drug. However, multiple independent *in vitro* studies demonstrated that the Met408Val substitution does not directly affect OCT1 uptake ([Kerb et al., 2002](#B12); [Shu et al., 2003](#B30); [Shu et al., 2007](#B31); [Nies et al., 2014](#B21); [Tzvetkov et al., 2014](#B41); [Seitz et al., 2015](#B27)). Thus, the rs35854239 variant may be the true cause for the observed associations of Met408Val with clinically relevant phenotypes.

The rs35854239 variant is very common. If functional, with its minor allele frequency of 40.6% in Europeans and White Americans, the rs35854239 variant could be the most frequent variant affecting OCT1 expression and activity. However, it is not clear whether the duplicated splicing donor site always leads to alternative splicing, and to what extend the alternatively spliced transcript is functionally active.

In this study, we analyzed to what extend the SLC22A1 8 bp duplication (rs35854239) affects splicing and what are the consequences of the variable splicing on the transporter function *in vitro* and *in vivo*. To this end, first, we quantified the alternatively spliced transcripts both using the minigene assay and direct analyses of human liver samples. Second, we analyzed whether the protein resulting from the alternatively spliced transcripts is active. Finally, we analyzed whether the rs35854239 variant is associated with changes in OCT1 mRNA and protein expression in human livers and pharmacokinetics of sumatriptan and fenoterol in humans.

## Materials and Methods

### Materials

Dulbecco’s Modified Eagle Medium (DMEM), Hank’s Buffered Salt Solution (HBSS), and 4-(4-(dimethylamino)styryl)-N-methylpyridinium (ASP^+^) were obtained from Life Technologies (Darmstadt, Germany). Poly-D-lysine (1–5 kDa), 2-[4-(2-hydroxyethyl)piperazin-1-yl]ethanesulfonic acid (HEPES), bicinchoninic acid, and copper sulfate pentahydrate were obtained from Sigma-Aldrich (Taufkirchen, Germany). Dulbecco’s phosphate-buffered saline (DPBS) and additives for cell culturing were obtained from PAN-Biotech (Aidenbach, Germany). Twelve-well plates were obtained from CytoOne (Langenselbold, Germany), 6-well plates from Corning GmbH (Kaiserslautern, Germany), and tissue culture flasks from Sarstedt (Nümbrecht, Germany). All chemicals used in this study were purchased from commercial sources and had purities ≥95%.

### Cell Lines and Cell Culturing

HEK293 (Thermo Fisher Scientific, Darmstadt, Germany), Huh7, and HepG2 (ATCC, Manassas, United States) cells were cultured in DMEM supplemented with 10% fetal calf serum (FCS), 100 U/mL penicillin and 100 μg/ml streptomycin at 37°C and 5% CO_2_.

### Generation of an Alternatively Spliced OCT1 Plasmid

The 8 bp insertion in exon 7 that results from alternative splicing of rs35854239 was introduced into an OCT1 encoding pcDNA5/FRT vector by site-directed mutagenesis as described previously ([Seitz et al., 2015](#B27)). The used primer pair 1 is listed in [Supplementary table S1](#s10). The sequence was validated by capillary sequencing prior to transient transfection into HEK293 cells.

### Cellular Uptake Experiments After Transient Transfection

HEK293 cells were seeded at a density of 5 × 10^5^ cells per well in a 12-well plate precoated with poly-D-lysine. Twenty-four hours after seeding, cells were transfected with 2 µg of the alternatively spliced OCT1 vector DNA using Lipofectamine™ 2000 (Invitrogen, Darmstadt, Germany) according to the manufacturer’s instructions. Transfection efficiency was evaluated by co-transfection with 0.5 µg of the green fluorescent protein coding vector pGFP-tpz (Thermo Fisher Scientific). The next day, uptake experiments were performed at 37°C and pH 7.4 using HBSS+ (HBSS supplemented with 10 mM HEPES buffer). Cells were washed once with pre-warmed (37°C) HBSS+. Uptake was initiated by adding 20 µM ASP^+^ diluted in HBSS+ and stopped after two minutes by adding ice-cold HBSS+. Cells were washed twice with ice-cold HBSS+ and lyzed with RIPA buffer. Fluorescence of ASP^+^ in lysates was measured with an excitation of 485 nm and emission of 612 nm using the Tecan infinite M200 Microplate Reader (Tecan Group Ltd., Männedorf, Switzerland). ASP^+^ fluorescence intensities were normalized to the total protein amount in the samples as measured using the bicinchoninic acid assay ([Smith et al., 1985](#B32)).

### Generation of Minigene Constructs

Splicing of exon 7-intron 7 was analyzed in the splicing vector pSPL3b, further referred to as minigene. Exon 7-intron 7 of OCT1 for both rs35854239 genotypes was amplified with primer pair 2, listed in [Supplementary table S1](#s10). The PCR product was cloned into the pSPL3b vector after restriction of the PCR product and the vector with *Pst*I and *Eco*RV. The Met408 and Val408 were introduced by site-directed mutagenesis using primer pairs 3 and 4, respectively, listed in [Supplementary table S1](#s10). The minigene constructs were validated by capillary sequencing and then used for transient transfection into Huh7 and HepG2 cells.

### Transient Transfection of the Minigene Constructs

Huh7 and HepG2 cells were seeded in six-well plates at a density of 4 × 10^5^ and 1.7 × 10^6^ cells per well, respectively. After 24 hours, cells were transfected with 2 µg minigene vector DNA using Lipofectamine™ 2000 as described above. Transfection efficiency was evaluated by co-transfection with pGFP-tpz as described above. Forty-eight hours after transfection, cells were lysed and RNA was isolated using the RNeasy Plus Mini Kit (QIAGEN, Hilden, Germany) according to the manufacturer’s instructions.

### PCR Amplification of Spliced Exon 7 Variants

After RNA isolation from transfected cells, complementary DNA (cDNA) was synthesized using the MultiScribe™ Reverse Transcriptase Kit (Applied Biosystems). Spliced exon 7 was amplified with primer pair 5 listed in [Supplementary table S1](#s10). PCR products were separated by gel electrophoresis, bands were visualized under UV light and band intensities were quantified using the Fiji software (ImageJ version 1.52p, National Institutes of Health, Bethesda, United States).

### Analysis of rs35854239 in Human Liver Samples

Human liver samples were obtained from normal liver tissue that had to be removed for technical reasons during liver surgery or from organ donors. Patients gave their informed consent for research use of the removed liver tissue, and the procedures were approved by the ethics committee of the University Medicine Göttingen, Georg-August-Universität Göttingen (application number 26/01/17) and the ethics committee of the Pomeranian Medical University (application number KB-0012/64/12). Deep-frozen human liver samples were homogenized using the Mikro dismembrator S (B. Braun, Melsungen, Germany) at 2500 rpm for 1 min. DNA was isolated using the DNeasy Blood & Tissue kit (QIAGEN) according to the manufacturer’s instructions. For the genotyping of functionally relevant polymorphisms in the SLC22A1 gene, the single base primer extension method was used as described previously ([Seitz et al., 2015](#B27)) using primer 6 listed in [Supplementary table S1](#s10). RNA from human liver samples was isolated from homogenates using the RNeasy Plus Mini Kit and cDNA was synthesized as described above.

### Pyrosequencing

The ratio of correctly vs. alternatively spliced exon 7 in the transfected cell lines with hepatic origin and human liver samples was analyzed using pyrosequencing. Spliced exon 7 from minigene experiments and from cDNA from human liver samples was amplified using primer pair 7 (minigene) and primer pair 8 (liver samples) listed in [Supplementary table S1](#s10). Samples were prepared using PyroMark™ Binding and Annealing Buffer (QIAGEN) and the PyroMark™ Vacuum Prep Station (Biotage, Uppsala, Sweden). Pyrosequencing was carried out on the PyroMark™ Q96 ID (Pyrosequencing AB, Uppsala, Sweden) with PyroMark™ Gold Q96 reagents (QIAGEN) using the primer 9 (minigene) and primer 10 (liver samples).

### Deep RNA-Sequencing and Sequence Mapping

Next-generation DNA and RNA sequencing was performed with cDNA from minigene experiments and DNA and cDNA from human liver samples. Exon 7 and its 3′ flanking region were first amplified using the primer pairs 11 to 14 listed in [Supplementary table S1](#s10). The PCR products were purified by magnetic separation using Agencourt^®^ AMPure^®^ XP reagent (Beckman Coulter GmbH, Krefeld, Germany). Unique indices were attached to the purified amplicons by PCR using the Nextera^®^ XT Index Kit v2 (Illumina Inc., San Diego, United States). The samples were again cleaned up with Agencourt^®^ AMPure^®^ XP reagent. All samples were pooled in appropriate ratios. The pooled library was quantified using the Qubit^®^ 2.0 fluorometer and the Qubit^®^ dsDNA BR assay kit (Thermo Fisher Scientific) and diluted to DNA concentration of 2 nM. DNA was denatured and diluted according to the manufacturer’s instructions. As internal control and to increase variability within the sequencing run, 30% PhiX control (Illumina) was spiked in prior to denaturation. The sequencing run was performed using the MiSeq^®^ Reagent Kit v3 (600 cycles) and paired-end 221 reads on the Illumina MiSeq™ (Illumina). The sequencing run was analyzed using the IGV v.2.6.3 software (Broad Institute, Cambridge, United States).

The paired-end sequence reads were merged using PEAR (release 0.9.11; [Zhang et al., 2014](#B51)). The mapping to a reference sequence was performed with Bowtie2 ([Langmead et al., 2012](#B49)) version 2.3.4.1. DNA was mapped against the human genome assembly (hg19) downloaded from the UCSC Genome Browser. The cDNA from human liver samples was mapped against the “Homo sapiens solute carrier family 22 member 1 (SLC22A1), transcript variant 1, mRNA” ([NM_003057.3](https://www.ncbi.nlm.nih.gov/nuccore/NM_003057.3)) with artificially introduced duplication in it in order to better visualize the possible insertion using local mapping mode. The cDNA from minigene was mapped against the minigene itself. For both experiments with human liver samples we calculated subsequently the sequencing depth for each allelic combination of rs35854239 variant and rs628031 (Met408Val) variant with our own script.

### Immunocytochemistry

Five x 10^5^ HEK293 cells were seeded on poly-D-lysine coated cover slips and transfected as described above. One day after transfection, cells were washed twice with PBS and were fixed with 100% ethanol for 20 min at –20°C. After washing three times with PBS, cell membranes were permeabilized with PBS containing 0.4% Tween 20. Cells were washed three times with PBS and subsequently blocked for 3 hours with blocking buffer (5% FCS in PBS). OCT1 was stained using the NBP1-51684 (2C5) antibody (Novus Biologicals, Abingdon, United Kingdom). Cells were co-stained with the EP 1845Y antibody (Abcam, Cambridge, United Kingdom) against the membrane marker Na^+^/K^+^-ATPase. The primary antibodies against OCT1 and Na^+^/K^+^-ATPase were diluted in blocking buffer in a dilution of 1:400 and 1:200, respectively. Per cover slip, 50 µL antibody solution was added, cells were covered with parafilm and incubated in a humid chamber overnight. The next day, after washing three times with PBS, fluorescently-labeled secondary antibodies (Alexa Fluor^®^ 546 goat anti-rabbit IgG (H + L), polyclonal and Alexa Fluor^®^ 488 goat anti-mouse IgG (H + L), polyclonal; Thermo Fisher Scientific) were diluted 1:400 in PBS, added and incubated for 2 hours in the dark. Cells were washed three times with PBS and cover slips were mounted with Roti^®^ Mount Fluor Care DAPI (Carl Roth, Karlsruhe, Germany). The staining was analyzed using the laser scanning microscope LSM780 (Carl Zeiss Microscopy GmbH, Oberkochen, Germany). Images were processed using the Fiji software.

### Expression Data From Human Liver Samples

OCT1 mRNA and OCT1 protein expression data were extracted from a previous study describing expression of OCT1 and OCT3 in human liver samples ([Nies et al., 2009](#B19)). Analysis was performed on the subset of samples (*n* = 90) that were from individuals who were non-cholestatic and had no hepatocellular, cholangiocellular or gallbladder carcinoma ([Schaeffeler et al., 2011](#B25); [Nies et al., 2013](#B20)). The study was approved by the ethics committees of the Charité, Humboldt University (Berlin, Germany) and the University of Tübingen (Tübingen, Germany) in accordance with the principles of the Declaration of Helsinki. Written informed consent was obtained from each patient.

### Clinical Trial

The clinical trials on the effects of SLC22A1 genetic variants on fenoterol and sumatriptan pharmacokinetics have been described in details before ([Matthaei et al., 2016](#B17); [Tzvetkov et al., 2018](#B38)). The rs35854239 variant was genotyped using the available DNA from those studies and the single base primer extension method as described previously ([Seitz et al., 2015](#B27)) with the SNaPshot primers listed in [Supplementary table S1](#s10).

### Statistical Analysis

Differences in OCT1 mRNA and protein expression, or drug plasma concentration between homozygous wild-types and homozygous duplication allele carriers were performed using the Mann-Whitney-*U* test. Differences between DNA and RNA allele frequencies in allelic expression imbalance analyses were performed using the paired sample *t*-test. All analyses were performed using SPSS Statistics version 25 (SPSS INC., IBM, Chicago, IL) Statistical significance was defined as *p* < 0.05. Post-hoc power calculations of the clinical studies were performed with the G*Power software version 3.1.9.4 ([Faul et al., 2007](#B7)).

## Results

### Minigene Analyses of the Effects of SLC22A1 rs35854239 on Splicing

We used minigene assays to quantify the percentage of alternatively spliced transcripts in the 8 bp duplication allele of the rs35854239 variant. For this purpose, exon 7, including 306 bp upstream and 310 bp downstream of the flanking intronic regions, was cloned in the minigene vector pSPL3b between the exons 1 and 2 of the rabbit β-globulin gene. Next to the construct carrying the 8 bp duplication allele, the wild-type allele was also cloned and used as a control in the analyses ([Figure 1B](#F1)). Two independent minigene clones containing the duplication allele were analyzed to account for potential artifacts from the quality of the clones and the DNA preparation. The minigene constructs were transiently transfected into HepG2 and Huh7 cells, and the resulting correctly and alternatively spliced transcripts were quantified 48 hours later using three independent quantification techniques: semi-quantitative PCR, pyrosequencing and deep RNA-sequencing. In all cases, first, total RNA was reverse transcribed into cDNA. For the semi-quantitative PCR, the spliced SLC22A1 exon 7 was amplified by PCR using primers within the flanking exons of the rabbit β-globulin gene. The PCR products were separated by gel electrophoresis to enable the selective identification of the correctly and the alternatively spliced transcripts, and the band intensities were quantified ([Figure 1C](#F1)). As expected, the wild-type allele was spliced 100% correctly. However, the duplication allele was only spliced 47% correctly (range 42–51%) in transiently transfected Huh7 cells and 52% correctly (range 46–58%) in HepG2 cells (data not shown).

Next, we used pyrosequencing to quantify more precisely the ratio of the alternatively spliced transcripts. The pyrosequencing quantification method was validated by calibration series of vectors encoding the correctly or alternatively spliced minigene ([Supplementary Figure S1](#s10)). The pyrosequencing-based quantification showed that in Huh7 cells, the duplication allele was correctly spliced in 49% (range 46–52%) of all transcripts ([Figure 1D](#F1)). This was highly comparable with the previously semi-quantitatively determined ratios. In HepG2 cells, the duplication allele was correctly spliced in 57% (range 55–60%) of all transcripts.

Finally, the minigene insertion allele clone 1 spliced in Huh7 or HepG2 cells at 48 h was reanalyzed using deep massive-parallel sequencing ([Figure 1E](#F1)). The average depth of targeted RNA sequencing was 59,135 reads (range 32,902–84,691). The quantification of reads carrying the 8 bp insertion as a result of alternative splicing showed a percentage of 62% correctly spliced minigene in both cell lines. These results confirm the alternative splicing of rs35854239. More importantly, these results suggest that the 8 bp duplication causes erroneous splicing in only a part of the transcripts. Estimated by the data of all experiments maximally 52% of the transcripts are erroneously spliced. Thus, our *in vitro* data suggest that even in homozygous carriers of the duplication about the half of OCT1 transcripts will be correctly spliced.

### Effects of SLC22A1 Exon 7 Genetic Variants on rs35854239 Splicing

Within the SLC22A1 gene, exon 7 harbors the highest density of coding functionally relevant polymorphisms. Thereof, the Met408Val substitution is almost completely linked to the rs35854239 duplication ([Figure 2A](#F2)). Under native conditions, it could not be excluded that the coding variant substantially contributes to the effects of splicing. Here were took advantage of the minigene technique and addressed separately the effects of the two variants on splicing. The Val408Met substitution alone did not significantly affect splicing, neither on duplication nor on wild-type rs35854239 background ([Figure 2B](#F2)). Therefore, it could be concluded that the effects on splicing are completely caused by the rs35854239 duplication and there is no contribution of the highly linked Met408Val.

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d6e/8137991/b4d19134be86/fphar-12-661480-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8137991_fphar-12-661480-g002.jpg)

Effects of Met408Val on rs35854239 and on exon 7 splicing (A) Localization of rs35854239 at the exon 7-intron 7 junction of the SLC22A1 gene. Known variants reducing OCT1 function in exon 7 of the SLC22A1 gene are shown in red. The highly genetically linked variants that are separately analyzed here are highlighted in gray (B) The Met408Val polymorphism was introduced by site-directed mutagenesis into both minigenes: harboring the wild-type and the duplication alleles of rs35854239. After transient transfection in Huh7 cells, spliced transcripts were amplified and separated by gel electrophoresis and afterwards, band intensities of correctly and alternatively spliced transcripts were quantified. Percentages within boxes represent relative values of correctly spliced minigene. Shown are means and standard errors of the means of three independent experiments.

### Functional Characterization of the Protein Encoded by the Alternatively Spliced Transcript

The alternative splicing leads to an 8 bp longer exon 7, entailing a frame shift. This results in an altered amino acid sequence after codon 425 followed by a premature stop after seven amino acids. The resulting truncated OCT1 protein p. Asp426fs consists of the first nine TMHs only ([Figure 3A](#F3)).

### FIGURE 3.

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d6e/8137991/c8a0f69f1d52/fphar-12-661480-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8137991_fphar-12-661480-g003.jpg)

Effects of the alternatively spliced OCT1 protein p.Asp426fs on OCT1 function (A) Alternative splicing of rs35854239 leads to a premature stop after Asn431, resulting in an OCT1 protein that is truncated after transmembrane helix 9. (B) HEK293 cells transiently transfected with a vector coding for the alternatively spliced OCT1 p. Asp426fs were incubated for 2 min with 20 μM ASP+. The OCT1-mediated uptake was calculated as fold change compared to control cells (transfected with the empty vector). Transfection of wild-type OCT1 served as a positive control for functional transporter. Data are shown as mean and standard error of the mean (SEM) of three independent experiments. (C) Membrane localization was analyzed using immunofluorescence staining and confocal microscopy (magnification factor 63). The OCT1 antibody used for this purpose recognizes the intracellular loop of the protein between TMH6 and TMH7. OCT1 (green) was co-stained with Na+/K+-ATPase (red) as membrane marker. Scale bar indicates 10 µm. TMH, transmembrane helix.

To analyze whether the truncated OCT1 protein p. Asp426fs is able to function as an uptake transporter, the 8 bp insertion sequence was introduced between exon 7 and exon 8 of the OCT1 carrying pcDNA5 vector using site-directed mutagenesis. The resulting vector was transiently transfected into HEK293 cells and the uptake of the model OCT1 substrate ASP^+^ was compared to the uptake of the wild-type OCT1. Three independent clones of p.Asp426fs were analyzed. The alternatively spliced OCT1 protein showed no transport activity ([Figure 3B](#F3)). ASP^+^ uptake of all alternatively spliced OCT1 clones was at same levels as the empty pcDNA5 vector, indicating no OCT1-mediated substrate uptake.

Immunofluorescence staining revealed aberrant membrane localization of the alternatively spliced OCT1 ([Figure 3C](#F3)). This demonstrates that the truncated OCT1 protein, which results from alternative splicing of exon 7, is completely inactive and lacks correct membrane localization.

### Effects of rs35854239 on Splicing in Human Liver Samples

Minigene analyses in cell lines with hepatic origin showed that the 8 bp duplication leads to maximally 52% of alternatively spliced transcripts that encode a non-functional protein. In order to validate these results *in vivo*, we quantified the effects of the 8 bp duplication on OCT1 mRNA splicing in human liver. To this end, DNA from 24 liver samples was genotyped for rs35854239 and the correct splicing of the exon 7-intron 7 junction was quantified using pyrosequencing and deep RNA-sequencing ([Figure 4](#F4)).

### FIGURE 4.

![FIGURE 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d6e/8137991/802843deb5b8/fphar-12-661480-g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8137991_fphar-12-661480-g004.jpg)

Effects of rs35854239 on OCT1 splicing in human liver samples. Splicing of the exon7-intron7 junction of OCT1 mRNA, depending on rs35854239 genotype was quantified using (A) pyrosequencing and (B) deep RNA-sequencing. On the left, the mean percentage of correctly spliced transcripts depending on the rs35854239 genotype is shown. The percentage of (C) correctly and (D) alternatively spliced transcripts for each individual sample is shown. (D) The mean sequencing depth for alternatively spliced and total transcripts is stated below. Dup, rs35854239 duplication allele; WT, wild-type SLC22A1 allele. Dup/dup: n = 3, WT/dup: n = 15, WT/WT: n = 2.

Using pyrosequencing, we observed 99% correctly spliced SLC22A1 mRNA irrespective of the genotype of the liver donors. Alternative splicing of OCT1 in liver samples from homozygous or heterozygous 8 bp duplication allele carriers appeared with a maximum of 2.1% ([Figure 4A](#F4)). This percentage is far below the observed results in minigene experiments ([Figure 1](#F1)) but is still substantially higher than the observed 0.4% in the SLC22A1 mRNA from donors with wild-type genotype, which can only be spliced correctly.

Using deep RNA-sequencing, we detected very low levels of alternatively spliced transcripts that were, however, dependent on the rs35854239 genotype ([Figure 4B](#F4)). The average depth of sequencing was 74,326 reads per RNA sample (range 41,116–133,715). The liver samples from homozygous duplication allele carriers showed mean values of 0.58% alternatively spliced transcripts (range 0.19–1.14%). In heterozygous genotypes, alternative splicing was detected with a mean of 0.36% (range 0.08–0.83%). The samples of the homozygous wild-type allele carriers showed a mean of 0.02% alternatively spliced transcripts indicating very low levels of possible contamination with this highly sensitive method. In conclusion, both techniques demonstrated that despite the close to 50% probability of alternative splicing of the rs35854239 duplication allele estimated by the minigene assays, alternatively spliced OCT1 could only barely be detected in human liver samples. These results suggest that the alternatively spliced transcripts may be recognized and rapidly degraded under native conditions.

To verify this, we performed an allelic expression imbalance analysis in the human liver samples. We took advantage of the strong genetic linkage between the duplication allele of rs35854239 and the A-allele of the coding variant Met408Val (rs628031, 1222A>G, *r* ^2^ = 0.95; [Figure 5A](#F5)). Based on the strong linkage, in heterozygous carriers of the rs35854239 duplication and Met408Val A-allele haplotype, a degradation of alternatively spliced OCT1 could be detected as a lower abundance of the Met408 A-allele in the RNA transcripts compared to the expected 50% of the DNA reads ([Figure 5B](#F5)). We used all nine liver samples from which both DNA and RNA was available and applied deep sequencing for quantification. While the A-allele was detected in 50% of the DNA reads (range 49–53%), the abundance in RNA was significantly decreased to 42% (range 40–44%, *p* = 2.77 × 10^−7^, paired *t*-test; [Figure 5C](#F5)). This result supports the degradation of the alternatively spliced transcripts and suggests that the presence of the rs35854239 duplication will result in the reduction of OCT1 mRNA levels, and as a consequence OCT1 protein in general.

### FIGURE 5.

![FIGURE 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d6e/8137991/d13cb4423f07/fphar-12-661480-g005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8137991_fphar-12-661480-g005.jpg)

Allelic expression imbalance in heterozygous carriers of rs35854239 (A) The duplication allele of rs35854239 is highly linked to the A-allele of Met408. The abundance of both Met408Val alleles was analyzed on DNA and RNA level in heterozygous Met408Val/rs35854239 human liver samples using deep RNA-sequencing. (B) The abundance of both Met408Val alleles on RNA level was analyzed and (C) the percentage of the A-allele (Met408) on DNA and RNA levels was compared in nine human liver samples (green). Dashed lines represent expected allele balance without (dark gray) and with (light gray) non sense-mediated mRNA decay (NMD). (D) Sequencing depth is given in absolute reads and relative to the Val408 G-allele and rs35854239 wild-type haplotype.

In addition, more precise analyses of the sequencing reads suggest that the correct “canonical” splicing may be preferred under native conditions. Indeed, reads of alternatively spliced RNA carrying the A-allele of Met408Val were almost not detectable ([Figure 5D](#F5)). However, the reads of correctly spliced RNA carrying A-allele Met408Val were 71.3% of the G-allele reads (range from 65.8 to 76.7%) instead of the expected 50% from the minigene analyses. This suggests that in parallel to degradation of the alternatively spliced transcripts also a preference for correct splicing of rs35854239 duplication allele under native conditions may exist.

To address this, we analyzed total OCT1 mRNA and protein expression in 73 human liver samples. The liver samples had been characterized for their OCT1 expression before ([Nies et al., 2009](#B19)). We included only those samples lacking the Arg61Cys substitution, which is known to significantly affect OCT1 protein levels in the liver ([Supplementary table S2](#s10) and ([Nies et al., 2009](#B19)) by affecting the correct membrane localization ([Seitz et al., 2015](#B27)). The OCT1 mRNA expression was on median 47% lower in homozygous duplication than in homozygous wild-type rs35854239 allele carriers (median of 0.014 and 0.026 transcripts per beta-actin transcript, respectively; *p* = 0.007; [Figure 6A](#F6)). The OCT1 protein levels were on median 35% lower in homozygous duplication than in homozygous wild-type rs35854239 allele carriers (median of 3.91 and 6.04, respectively; *p* = 0.045; [Figure 6B](#F6)). However, although statistically significant, the effects of rs35854239 genotypes could explain only 9% of the variability of mRNA and protein expression in this sample set.

### FIGURE 6.

![FIGURE 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d6e/8137991/2cf2883b6679/fphar-12-661480-g006.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8137991_fphar-12-661480-g006.jpg)

Effect of rs35854239 on OCT1 (A) mRNA and (B) protein expression. Median OCT1 (A) mRNA expression and (B) protein expression was analyzed in 73 liver samples depending on the rs35854239 genotype. Dup, rs35854239 duplication allele; WT, wild-type OCT1 allele; a. u., arbitrary unit.

### Effects of rs35854239 Duplication on the Pharmacokinetics of Sumatriptan and Fenoterol

Finally, we analyzed to what extend the decrease of OCT1 expression in carriers of the rs35854239 duplication allele leads to changes in sumatriptan and fenoterol pharmacokinetics in humans. We took advantage of the existing studies on the effects of OCT1 genotypes on the pharmacokinetics of both drugs ([Matthaei et al., 2016](#B17); [Tzvetkov et al., 2018](#B38)) and analyzed them in the context of the rs35854239 genotype. The AUC of sumatriptan was slightly increased in homozygous rs35854239 duplication allele carriers compared to the wild-type (means of 7187 vs. 6277 min × ng/ ml, respectively, [Figure 7A](#F7)). However, this increase was not significant and was on average by 14% compared to the observed 127% increase in poor OCT1 transporters (homozygous or compound heterozygous carriers of the coding variants Arg61Cys, Gly401Ser, Gly465Arg) observed in the same study.

### FIGURE 7.

![FIGURE 7](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d6e/8137991/9b416e33311c/fphar-12-661480-g007.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8137991_fphar-12-661480-g007.jpg)

Effects of rs35854239 on the pharmacokinetics of (A) sumatriptan and (B) fenoterol. The AUC of (A) sumatriptan and (B) fenoterol is depicted depending on the rs35854239 genotype and compared to the OCT1 phenotype (poor transporters). Poor transporters comprise homozygous or compound heterozygous carriers of the OCT1 alleles *3, *4, and *5 harboring the coding variants Arg61Cys, Gly401Ser or Gly465Arg, respectively. Dup, rs35854239 duplication allele; WT, wild-type OCT1 allele; AUC, area under the plasma concentration-time curve. Boxplots show median, lower (25%) and upper (75%) quartiles.

Even more, the AUC of fenoterol was not higher in homozygous carriers of the rs35854239 duplication allele compared to the wild-type (means of 84.25 vs. 86.84 min × ng/ml, respectively; [Figure 7B](#F7)). In comparison, poor OCT1 transporters showed 1.89-fold higher AUCs for fenoterol. This data suggests that compared to the well-known loss-of-function coding variants, the 8 bp duplication shows only limited effects on drugs pharmacokinetics.

## Discussion

The eight base pairs duplication at the exon 7-intron 7 junction (rs35854239) has been previously suggested to cause erroneous splicing of OCT1 by introducing an alternative splice site in the intronic sequence of intron 7. In this study, we confirm the alternative splicing and give more precise quantitative information about the effects on OCT1 expression and activity in order to better estimate the contribution of this variant to the highly inter-individual variability in OCT1 activity.

This study built up on the previous findings of Kim et al. ([Kim et al., 2017](#B14)). We confirmed the findings of Kim et al. that the 8 bp duplication causes alternative splicing. We did this both by using minigene assays ([Figure 1](#F1)) and by detecting (a low level) of the alternatively spliced transcript in human liver samples ([Figure 4](#F4)). We also confirmed the finding of Kim et al. that the alternatively spliced transcript is not leading to a functional protein ([Figure 2](#F2)).

The major contribution of this study beyond the previously known is the precise quantification of the effects of the 8 bp duplication rs35854239. We used minigene analyses to quantify the effects on splicing ([Figure 1](#F1)) and to confirm that these effects are caused by the 8 bp duplication and not by the highly genetically linked variant Met408Val. We quantified the effects of the 8 bp duplication on total OCT1 expression in human liver both on mRNA and on protein levels ([Figure 6](#F6)) and finally we analyzed the association of the splice variants with the pharmacokinetics of drugs that are well known OCT1 substrates ([Figure 7](#F7)). This will enable us to better evaluate the contribution of the rs35854239 duplication to the high genetic and thus to the high functional variability of OCT1 in humans.

Our data suggest that the 8 bp duplication allele can cause erroneous splicing of up to 50% of the transcripts, but, probably due to mRNA decay, the number of detectable erroneously spliced transcripts in the human liver is very low. Thus, homozygous carriers of the duplication allele are characterized by decreased expression of the correctly spliced transcripts resulting in a median decrease of OCT1 protein expression by 35% in the human liver. However, the rs35854239 effects explained only 9% of the highly variable SLC22A1 mRNA expression in humans ([Figure 6](#F6)), and the 8 bp duplication was not associated with significant changes in the pharmacokinetics of known OCT1 substrates, i.e. sumatriptan and fenoterol ([Figure 7](#F7)).

The rs35854239 ins/del variant results in duplication of the originally existing donor splice site of intron 7, giving a possibility of alternative, but also keeping the possibility of correct “canonical” splicing. Here, we demonstrated that both donor splice sites are operative. Depending on the cellular system and the quantification technique used, intensive processing of both splice sites with a slight preference for the usage of the original “correct” donor splice site was suggested ([Figure 1](#F1)).

These numbers are in contrast to the almost undetectable alternatively spliced transcripts in the human liver. These discrepancies could be explained by a nonsense-mediated mRNA decay (NMD) as suggested previously by Kim et al. ([Kim et al., 2017](#B14)). Here, we provide multiple confirmations for an mRNA decay of the incorrectly spliced OCT1 transcripts. Firstly, using minigene constructs that use rabbit flanking exons, and thus are not affected by NMD, we observed a substantially higher percentage of the alternatively spliced transcripts. Secondly, we applied allelic expression imbalance analyses, taking advantage of the genetically highly linked coding variant Met408Val, and were able to demonstrate imbalance in the expression of the duplication-linked allele Met408 ([Figure 5](#F5)). Finally, we demonstrated significantly reduced expression of the correctly spliced transcripts in homozygous carriers of the duplication allele ([Figure 6](#F6)).

Even if not degraded, the alternatively spliced transcript encodes a truncated protein p.Asp426fs. This protein is missing TMHs 10 to 12, which are essential for OCT1 activity ([Shu et al., 2003](#B30); [Gorboulev et al., 2005](#B9); [Egenberger et al., 2012](#B6)) and is not localized correctly at in the plasma membrane ([Figure 3](#F3)). Therefore, only the correctly spliced transcripts can contribute to OCT1 activity.

A major hypothesis of this study was that the 8 bp duplication that affects splicing may explain a major part of OCT1 expression variability. OCT1 mRNA expression in the human liver varied more than 100-fold between individuals ([Nies et al., 2009](#B19); [O'Brien et al., 2013](#B22)). In the 73 liver samples analyzed in this study, we observed 23-fold variability in OCT1 expression. Despite a median reduction of OCT1 expression by almost 50% in homozygous duplication carriers, the genetic variant could explain less than 10% of the general variability. Taken together with the lack of strong effects of genetic variants in the SLC22A1 promoter ([Bokelmann et al., 2018](#B3)), only minor effects of *cis*-acting variants could be concluded. Systematic analyses of the genetic component in the variability of OCT1 expression are highly complicated, as multiple sampling of the same individuals or sampling within multiple members of the family are required. However, there are already some data suggesting that *trans*-acting variants or non-genetic factors may play a role. Indeed, genetic variants in transcription factors known to regulate OCT1 expression were associated with OCT1 expression ([O’Brien et al., 2013](#B22)), and disease conditions such as cholestasis may play a role ([Nies et al., 2009](#B19)). There are a number of other transcriptional factors suggested to regulate OCT1 expression, e.g. CCAAT/enhancer binding proteins, pregnane X receptor (PXR), farnesoid X receptor (FXR), and glucocorticoid receptor (GR)([Saborowski et al., 2006](#B24); [Rulcova et al., 2013](#B23); [Hyrsova et al., 2016](#B11)). It will be interesting to study whether genetic variants within them may have *cis*-effects on OCT1 expression.

Regarding potential effects of rs35854239 on pharmacokinetics, our data suggest that the impact on OCT1 expression caused by the duplication does not significantly affect the pharmacokinetics of known OCT1 substrates in humans. We were not able to detect significant differences in the AUCs of sumatriptan or fenoterol depending on the presence or absence of the rs35854239 duplication, in contrast to the clear and highly significant effects of the well-known coding OCT1 variants ([Figure 7](#F7)). Indeed, these two studies were not designed to address effects of rs35854239. However, based on the high frequency of this variant, we had a power of 80% to detect the increase in AUC of 47% and higher for sumatriptan, and of 15% or higher for fenoterol. The significant, but less prominent effects of this variant are in line with previous reports ([Kim et al., 2017](#B14)).

One explanation may be that the 50% reduction of OCT1 expression is not sufficient to affect substantially the pharmacokinetics of substrates in humans. Our previous studies on the effects of coding genetic variants on OCT1 activity clearly demonstrated that reduction of the typical OCT1 activity by more than 50% (OCT1 gene dose of less than 1 with a typical gene dose of 2) is necessary to cause measurable changes in drug pharmacokinetics ([Tzvetkov et al., 2018](#B38); Matthaei et al., 2019). This is in line with the lack of evidence for an independent association of rs35854239 with serum isobutyrylcarnitine levels in the Kim et al. study ([Kim et al., 2017](#B14)). The authors of this study also interpreted this as a consequence of the small effect size of the duplication.

Multiple studies reported an association of the Met408Val variant with pharmacokinetics or efficacy of drugs that are potential OCT1 substrates ([White et al., 2006](#B47); [Shikata et al., 2007](#B28); [Wang et al., 2008](#B44); [Kim et al., 2009](#B13); [Takahashi et al., 2010](#B34); [Tarasova et al., 2012](#B35); [Koren-Michowitz et al., 2014](#B16); [Vaidya et al., 2015](#B42)). However, those associations were difficult to interpret, as no significant effect of this coding variant on transport activity could be demonstrated ([Shu et al., 2003](#B30); [Shu et al., 2007](#B31); [White et al., 2010](#B46); [Nies et al., 2014](#B21); [Tzvetkov et al., 2014](#B41); [Seitz et al., 2015](#B27)). The rs35854239 variant is almost completely genetically linked to Met408Val (*r* ^2^ of 0.95). Our data strongly suggest the rs35854239 variant as causative variant due to its effects on splicing and thus on OCT1 expression. However, we also demonstrated that these effects are less prominent than the influences of other common coding variants of the gene, which lead to reduced or loss of OCT1 function (Arg61Cys, Cys88Arg, Gly401Ser and Gly465Arg). Therefore, the observed strong association with Met408Val or directly with rs35854239, but absence of association with the highly functional amino acid substitutions listed above are still difficult to explain.

In conclusion, using minigene analyses we were able to quantify that the common naturally occurring 8 bp duplication at the exon 7-intron 7 junction (rs35854239) causes alternative splicing in approximately 50% of the cases. The alternatively spliced transcripts are degraded under native conditions in the liver, but even if stable they are not able to encode the active protein (as demonstrated here using expression in HEK293 cells), and thus result in a significant decrease in OCT1 expression detectable both on mRNA and protein levels. However, although very common (minor allele frequency of 40.6%), the decrease in expression, taken together with the high general variability of OCT1 expression, was not sufficient to cause strong effects on drug pharmacokinetics (as demonstrated by analyzing the effects of the variant on pharmacokinetics of sumatriptan and fenoterol in healthy individuals).

## Acknowledgments

The authors would like to acknowledge the technical support of Kerstin Schmidt in the cloning process. We acknowledge support for the Article Processing Charge from the DFG (German Research Foundation, 393148499) and the Open Access Publication Fund of the University of Greifswald.

## Data Availability Statement

The original contributions presented in the study are publicly available. This data can be found here: NCBI repository, accession number: PRJNA720275 ([https://www.ncbi.nlm.nih.gov/sra/PRJNA720275](https://www.ncbi.nlm.nih.gov/sra/PRJNA720275))

## Ethics Statement

The studies involving human participants were reviewed and approved by the Ethics committee of the University Medicine Göttingen, Georg-August-Universität Göttingen (application number 26/01/17) by the ethics committees of the Charité, Humboldt University (Berlin, Germany) and the University of Tübingen (Tübingen, Germany). The patients/participants provided their written informed consent to participate in this study.

## Author Contributions

SR, KK, LS performed experiments. SR, MT, KK, AT, AN analyzed data. JM, AT, JL-R, JG, MD, AN contributed new reagents or analytic tools. SR, MT, MM, KK, AN wrote the manuscript. All authors read and edited the manuscript. MM, MT and JB participated in research design.

## Funding

This project was in part supported by an intramural grant (FOVB-2020-22) of University Medicine Greifswald to MM, the Robert Bosch Stiftung (Stuttgart, Germany) to KK and AN and the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under Germany’s Excellence Strategy —EXC 2180–390900677 to AN.

## Conflict of Interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Supplementary Material

The Supplementary Material for this article can be found online at: [https://www.frontiersin.org/articles/10.3389/fphar.2021.661480/full#supplementary-material](https://www.frontiersin.org/articles/10.3389/fphar.2021.661480/full#supplementary-material)

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are publicly available. This data can be found here: NCBI repository, accession number: PRJNA720275 ([https://www.ncbi.nlm.nih.gov/sra/PRJNA720275](https://www.ncbi.nlm.nih.gov/sra/PRJNA720275))

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are publicly available. This data can be found here: NCBI repository, accession number: PRJNA720275 ([https://www.ncbi.nlm.nih.gov/sra/PRJNA720275](https://www.ncbi.nlm.nih.gov/sra/PRJNA720275))

## References

1. Arimany-Nardi C., Minuesa G., Keller T., Erkizia I., Koepsell H., Martinez-Picado J., et al. (2016). Role of Human Organic Cation Transporter 1 (hOCT1) Polymorphisms in Lamivudine (3TC) Uptake and Drug-Drug Interactions. Front. Pharmacol. 7, 175. 10.3389/fphar.2016.00175  [DOI](https://doi.org/10.3389/fphar.2016.00175) | [PMC free article](/articles/PMC4919327/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27445813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Role%20of%20Human%20Organic%20Cation%20Transporter%201%20(hOCT1)%20Polymorphisms%20in%20Lamivudine%20(3TC)%20Uptake%20and%20Drug-Drug%20Interactions&author=C.%20Arimany-Nardi&author=G.%20Minuesa&author=T.%20Keller&author=I.%20Erkizia&author=H.%20Koepsell&volume=7&publication_year=2016&pages=175&pmid=27445813&doi=10.3389/fphar.2016.00175&)

2. Becker M. L., Visser L. E., van Schaik R. H. N., Hofman A., Uitterlinden A. G., Stricker B. H. C. (2011). OCT1 Polymorphism Is Associated with Response and Survival Time in Anti-parkinsonian Drug Users. Neurogenetics 12 (1), 79–82. 10.1007/s10048-010-0254-5  [DOI](https://doi.org/10.1007/s10048-010-0254-5) | [PMC free article](/articles/PMC3029819/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20680652/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurogenetics&title=OCT1%20Polymorphism%20Is%20Associated%20with%20Response%20and%20Survival%20Time%20in%20Anti-parkinsonian%20Drug%20Users&author=M.%20L.%20Becker&author=L.%20E.%20Visser&author=R.%20H.%20N.%20van%20Schaik&author=A.%20Hofman&author=A.%20G.%20Uitterlinden&volume=12&issue=1&publication_year=2011&pages=79-82&pmid=20680652&doi=10.1007/s10048-010-0254-5&)

3. Bokelmann K., Brockmöller J., Tzvetkov M. (2018). Impact of Promoter Polymorphisms on the Transcriptional Regulation of the Organic Cation Transporter OCT1 (SLC22A1). J. Pers Med. 8 (4), 42. 10.3390/jpm8040042  [DOI](https://doi.org/10.3390/jpm8040042) | [PMC free article](/articles/PMC6313513/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30544975/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pers%20Med.&title=Impact%20of%20Promoter%20Polymorphisms%20on%20the%20Transcriptional%20Regulation%20of%20the%20Organic%20Cation%20Transporter%20OCT1%20(SLC22A1)&author=K.%20Bokelmann&author=J.%20Brockm%C3%B6ller&author=M.%20Tzvetkov&volume=8&issue=4&publication_year=2018&pages=42&pmid=30544975&doi=10.3390/jpm8040042&)

4. Chen L., Shu Y., Liang X., Chen E. C., Yee S. W., Zur A. A., et al. (2014). OCT1 Is a High-Capacity Thiamine Transporter that Regulates Hepatic Steatosis and Is a Target of Metformin. Proc. Natl. Acad. Sci. USA 111 (27), 9983–9988. 10.1073/pnas.1314939111  [DOI](https://doi.org/10.1073/pnas.1314939111) | [PMC free article](/articles/PMC4103324/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24961373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=OCT1%20Is%20a%20High-Capacity%20Thiamine%20Transporter%20that%20Regulates%20Hepatic%20Steatosis%20and%20Is%20a%20Target%20of%20Metformin&author=L.%20Chen&author=Y.%20Shu&author=X.%20Liang&author=E.%20C.%20Chen&author=S.%20W.%20Yee&volume=111&issue=27&publication_year=2014&pages=9983-9988&pmid=24961373&doi=10.1073/pnas.1314939111&)

5. Drozdzik M., Busch D., Lapczuk J., Müller J., Ostrowski M., Kurzawski M., et al. (2019). Protein Abundance of Clinically Relevant Drug Transporters in the Human Liver and Intestine: A Comparative Analysis in Paired Tissue Specimens. Clin. Pharmacol. Ther. 105 (5), 1204–1212. 10.1002/cpt.1301  [DOI](https://doi.org/10.1002/cpt.1301) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30447067/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Protein%20Abundance%20of%20Clinically%20Relevant%20Drug%20Transporters%20in%20the%20Human%20Liver%20and%20Intestine:%20A%20Comparative%20Analysis%20in%20Paired%20Tissue%20Specimens&author=M.%20Drozdzik&author=D.%20Busch&author=J.%20Lapczuk&author=J.%20M%C3%BCller&author=M.%20Ostrowski&volume=105&issue=5&publication_year=2019&pages=1204-1212&pmid=30447067&doi=10.1002/cpt.1301&)

6. Egenberger B., Gorboulev V., Keller T., Gorbunov D., Gottlieb N., Geiger D., et al. (2012). A Substrate Binding Hinge Domain Is Critical for Transport-Related Structural Changes of Organic Cation Transporter 1*. J. Biol. Chem. 287 (37), 31561–31573. 10.1074/jbc.M112.388793  [DOI](https://doi.org/10.1074/jbc.M112.388793) | [PMC free article](/articles/PMC3438988/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22810231/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=A%20Substrate%20Binding%20Hinge%20Domain%20Is%20Critical%20for%20Transport-Related%20Structural%20Changes%20of%20Organic%20Cation%20Transporter%201*&author=B.%20Egenberger&author=V.%20Gorboulev&author=T.%20Keller&author=D.%20Gorbunov&author=N.%20Gottlieb&volume=287&issue=37&publication_year=2012&pages=31561-31573&pmid=22810231&doi=10.1074/jbc.M112.388793&)

7. Faul F., Erdfelder E., Lang A.-G., Buchner A. (2007). G*Power 3: a Flexible Statistical Power Analysis Program for the Social, Behavioral, and Biomedical Sciences. Behav. Res. Methods 39 (2), 175–191. 10.3758/BF03193146  [DOI](https://doi.org/10.3758/BF03193146) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17695343/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Behav.%20Res.%20Methods&title=G*Power%203:%20a%20Flexible%20Statistical%20Power%20Analysis%20Program%20for%20the%20Social,%20Behavioral,%20and%20Biomedical%20Sciences&author=F.%20Faul&author=E.%20Erdfelder&author=A.-G.%20Lang&author=A.%20Buchner&volume=39&issue=2&publication_year=2007&pages=175-191&pmid=17695343&doi=10.3758/BF03193146&)

8. Fukuda T., Chidambaran V., Mizuno T., Venkatasubramanian R., Ngamprasertwong P., Olbrecht V., et al. (2013). OCT1genetic Variants Influence the Pharmacokinetics of Morphine in Children. Pharmacogenomics 14 (10), 1141–1151. 10.2217/pgs.13.94  [DOI](https://doi.org/10.2217/pgs.13.94) | [PMC free article](/articles/PMC4116670/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23859569/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=OCT1genetic%20Variants%20Influence%20the%20Pharmacokinetics%20of%20Morphine%20in%20Children&author=T.%20Fukuda&author=V.%20Chidambaran&author=T.%20Mizuno&author=R.%20Venkatasubramanian&author=P.%20Ngamprasertwong&volume=14&issue=10&publication_year=2013&pages=1141-1151&pmid=23859569&doi=10.2217/pgs.13.94&)

9. Gorboulev V., Shatskaya N., Volk C., Koepsell H. (2005). Subtype-specific Affinity for Corticosterone of Rat Organic Cation Transporters rOCT1 and rOCT2 Depends on Three Amino Acids within the Substrate Binding Region. Mol. Pharmacol. 67 (5), 1612–1619. 10.1124/mol.104.008821  [DOI](https://doi.org/10.1124/mol.104.008821) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15662045/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=Subtype-specific%20Affinity%20for%20Corticosterone%20of%20Rat%20Organic%20Cation%20Transporters%20rOCT1%20and%20rOCT2%20Depends%20on%20Three%20Amino%20Acids%20within%20the%20Substrate%20Binding%20Region&author=V.%20Gorboulev&author=N.%20Shatskaya&author=C.%20Volk&author=H.%20Koepsell&volume=67&issue=5&publication_year=2005&pages=1612-1619&pmid=15662045&doi=10.1124/mol.104.008821&)

10. Grinfeld J., Gerrard G., Alikian M., Alonso-Dominguez J., Ale S., Valgañon M., et al. (2013). A Common Novel Splice Variant ofSLC22A1(OCT1)is Associated with Impaired Responses to Imatinib in Patients with Chronic Myeloid Leukaemia. Br. J. Haematol. 163 (5), 631–639. 10.1111/bjh.12591  [DOI](https://doi.org/10.1111/bjh.12591) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24117365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Haematol.&title=A%20Common%20Novel%20Splice%20Variant%20ofSLC22A1(OCT1)is%20Associated%20with%20Impaired%20Responses%20to%20Imatinib%20in%20Patients%20with%20Chronic%20Myeloid%20Leukaemia&author=J.%20Grinfeld&author=G.%20Gerrard&author=M.%20Alikian&author=J.%20Alonso-Dominguez&author=S.%20Ale&volume=163&issue=5&publication_year=2013&pages=631-639&pmid=24117365&doi=10.1111/bjh.12591&)

11. Hyrsova L., Smutny T., Carazo A., Moravcik S., Mandikova J., Trejtnar F., et al. (2016). The Pregnane X Receptor Down-Regulates Organic Cation Transporter 1 (SLC22A1) in Human Hepatocytes by Competing for (“squelching”) SRC-1 Coactivator. Br. J. Pharmacol. 173 (10), 1703–1715. 10.1111/bph.13472  [DOI](https://doi.org/10.1111/bph.13472) | [PMC free article](/articles/PMC4842917/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26920453/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Pharmacol.&title=The%20Pregnane%20X%20Receptor%20Down-Regulates%20Organic%20Cation%20Transporter%201%20(SLC22A1)%20in%20Human%20Hepatocytes%20by%20Competing%20for%20(%E2%80%9Csquelching%E2%80%9D)%20SRC-1%20Coactivator&author=L.%20Hyrsova&author=T.%20Smutny&author=A.%20Carazo&author=S.%20Moravcik&author=J.%20Mandikova&volume=173&issue=10&publication_year=2016&pages=1703-1715&pmid=26920453&doi=10.1111/bph.13472&)

12. Kerb R., Brinkmann U., Chatskaia N., Gorbunov D., Gorboulev V., Mornhinweg E., et al. (2002). Identification of Genetic Variations of the Human Organic Cation Transporter hOCT1 and Their Functional Consequences. Pharmacogenetics 12 (8), 591–595. 10.1097/00008571-200211000-00002  [DOI](https://doi.org/10.1097/00008571-200211000-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12439218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Identification%20of%20Genetic%20Variations%20of%20the%20Human%20Organic%20Cation%20Transporter%20hOCT1%20and%20Their%20Functional%20Consequences&author=R.%20Kerb&author=U.%20Brinkmann&author=N.%20Chatskaia&author=D.%20Gorbunov&author=V.%20Gorboulev&volume=12&issue=8&publication_year=2002&pages=591-595&pmid=12439218&doi=10.1097/00008571-200211000-00002&)

13. Kim D. H., Sriharsha L., Xu W., Kamel-Reid S., Liu X., Siminovitch K., et al. (2009). Clinical Relevance of a Pharmacogenetic Approach Using Multiple Candidate Genes to Predict Response and Resistance to Imatinib Therapy in Chronic Myeloid Leukemia. Clin. Cancer Res. 15 (14), 4750–4758. 10.1158/1078-0432.CCR-09-0145  [DOI](https://doi.org/10.1158/1078-0432.CCR-09-0145) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19584153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Clinical%20Relevance%20of%20a%20Pharmacogenetic%20Approach%20Using%20Multiple%20Candidate%20Genes%20to%20Predict%20Response%20and%20Resistance%20to%20Imatinib%20Therapy%20in%20Chronic%20Myeloid%20Leukemia&author=D.%20H.%20Kim&author=L.%20Sriharsha&author=W.%20Xu&author=S.%20Kamel-Reid&author=X.%20Liu&volume=15&issue=14&publication_year=2009&pages=4750-4758&pmid=19584153&doi=10.1158/1078-0432.CCR-09-0145&)

14. Kim H. I., Raffler J., Lu W., Lee J.-J., Abbey D., Saleheen D., et al. (2017). Fine Mapping and Functional Analysis Reveal a Role of SLC22A1 in Acylcarnitine Transport. Am. J. Hum. Genet. 101 (4), 489–502. 10.1016/j.ajhg.2017.08.008  [DOI](https://doi.org/10.1016/j.ajhg.2017.08.008) | [PMC free article](/articles/PMC5630162/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28942964/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Fine%20Mapping%20and%20Functional%20Analysis%20Reveal%20a%20Role%20of%20SLC22A1%20in%20Acylcarnitine%20Transport&author=H.%20I.%20Kim&author=J.%20Raffler&author=W.%20Lu&author=J.-J.%20Lee&author=D.%20Abbey&volume=101&issue=4&publication_year=2017&pages=489-502&pmid=28942964&doi=10.1016/j.ajhg.2017.08.008&)

15. Koehler M. R., Wissinger B., Gorboulev V., Koepsell H., Schmid M. (1997). The Two Human Organic Cation Transporter Genes SLC22A1 and SLC22A2 Are Located on Chromosome 6q26. Cytogenet. Cel Genet 79 (3-4), 198–200. 10.1159/000134720  [DOI](https://doi.org/10.1159/000134720) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9605850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cytogenet.%20Cel%20Genet&title=The%20Two%20Human%20Organic%20Cation%20Transporter%20Genes%20SLC22A1%20and%20SLC22A2%20Are%20Located%20on%20Chromosome%206q26&author=M.%20R.%20Koehler&author=B.%20Wissinger&author=V.%20Gorboulev&author=H.%20Koepsell&author=M.%20Schmid&volume=79&issue=3-4&publication_year=1997&pages=198-200&pmid=9605850&doi=10.1159/000134720&)

16. Koren-Michowitz M., Buzaglo Z., Ribakovsky E., Schwarz M., Pessach I., Shimoni A., et al. (2014). OCT1 Genetic Variants Are Associated with Long Term Outcomes in Imatinib Treated Chronic Myeloid Leukemia Patients. Eur. J. Haematol. 92 (4), 283–288. 10.1111/ejh.12235  [DOI](https://doi.org/10.1111/ejh.12235) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24215657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Haematol.&title=OCT1%20Genetic%20Variants%20Are%20Associated%20with%20Long%20Term%20Outcomes%20in%20Imatinib%20Treated%20Chronic%20Myeloid%20Leukemia%20Patients&author=M.%20Koren-Michowitz&author=Z.%20Buzaglo&author=E.%20Ribakovsky&author=M.%20Schwarz&author=I.%20Pessach&volume=92&issue=4&publication_year=2014&pages=283-288&pmid=24215657&doi=10.1111/ejh.12235&)

17. Langmead B., Salzberg S. L. (2012). Fast gapped-read alignment with Bowtie 2. Nature meth. 9, 357–359. 10.1038/nmeth.1923  [DOI](https://doi.org/10.1038/nmeth.1923) | [PMC free article](/articles/PMC3322381/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22388286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20meth.&title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&author=B.%20Langmead&author=S.%20L.%20Salzberg&volume=9&publication_year=2012&pages=357-359&pmid=22388286&doi=10.1038/nmeth.1923&)

18. Matthaei J., Kuron D., Faltraco F., Knoch T., Dos Santos Pereira J., Abu Abed M., et al. (2016). OCT1 Mediates Hepatic Uptake of Sumatriptan and Loss-Of-functionOCT1polymorphisms Affect Sumatriptan Pharmacokinetics. Clin. Pharmacol. Ther. 99 (6), 633–641. 10.1002/cpt.317  [DOI](https://doi.org/10.1002/cpt.317) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26659468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=OCT1%20Mediates%20Hepatic%20Uptake%20of%20Sumatriptan%20and%20Loss-Of-functionOCT1polymorphisms%20Affect%20Sumatriptan%20Pharmacokinetics&author=J.%20Matthaei&author=D.%20Kuron&author=F.%20Faltraco&author=T.%20Knoch&author=J.%20Dos%20Santos%20Pereira&volume=99&issue=6&publication_year=2016&pages=633-641&pmid=26659468&doi=10.1002/cpt.317&)

19. Matthaei J., Seitz T., Jensen O., Tann A., Prukop T., Tadjerpisheh M. V., et al. (2019). OCT1 Deficiency Affects Hepatocellular Concentrations and Pharmacokinetics of Cycloguanil, the Active Metabolite of the Antimalarial Drug Proguanil. Clin. Pharmacol. Ther. 105, 190–200. 10.1002/cpt.1128  [DOI](https://doi.org/10.1002/cpt.1128) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29882324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=OCT1%20Deficiency%20Affects%20Hepatocellular%20Concentrations%20and%20Pharmacokinetics%20of%20Cycloguanil,%20the%20Active%20Metabolite%20of%20the%20Antimalarial%20Drug%20Proguanil&author=J.%20Matthaei&author=T.%20Seitz&author=O.%20Jensen&author=A.%20Tann&author=T.%20Prukop&volume=105&publication_year=2019&pages=190-200&pmid=29882324&doi=10.1002/cpt.1128&)

20. Morais S. M. d., Wilkinson G. R., Blaisdell J., Nakamura K., Meyer U. A., Goldstein J. A. (1994). The Major Genetic Defect Responsible for the Polymorphism of S-Mephenytoin Metabolism in Humans. J. Biol. Chem. 269 (22), 15419–15422. 10.1016/S0021-9258(17)40694-6  [DOI](https://doi.org/10.1016/S0021-9258(17)40694-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8195181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=The%20Major%20Genetic%20Defect%20Responsible%20for%20the%20Polymorphism%20of%20S-Mephenytoin%20Metabolism%20in%20Humans&author=S.%20M.%20d.%20Morais&author=G.%20R.%20Wilkinson&author=J.%20Blaisdell&author=K.%20Nakamura&author=U.%20A.%20Meyer&volume=269&issue=22&publication_year=1994&pages=15419-15422&pmid=8195181&doi=10.1016/S0021-9258(17)40694-6&)

21. Nies A. T., Koepsell H., Winter S., Burk O., Klein K., Kerb R., et al. (2009). Expression of Organic Cation Transporters OCT1 (SLC22A1) and OCT3 (SLC22A3) Is Affected by Genetic Factors and Cholestasis in Human Liver. Hepatology 50 (4), 1227–1240. 10.1002/hep.23103  [DOI](https://doi.org/10.1002/hep.23103) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19591196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Expression%20of%20Organic%20Cation%20Transporters%20OCT1%20(SLC22A1)%20and%20OCT3%20(SLC22A3)%20Is%20Affected%20by%20Genetic%20Factors%20and%20Cholestasis%20in%20Human%20Liver&author=A.%20T.%20Nies&author=H.%20Koepsell&author=S.%20Winter&author=O.%20Burk&author=K.%20Klein&volume=50&issue=4&publication_year=2009&pages=1227-1240&pmid=19591196&doi=10.1002/hep.23103&)

22. Nies A. T., Niemi M., Burk O., Winter S., Zanger U. M., Stieger B., et al. (2013). Genetics Is a Major Determinant of Expression of the Human Hepatic Uptake Transporter OATP1B1, but Not of OATP1B3 and OATP2B1. Genome Med. 5 (1), 1. 10.1186/gm405  [DOI](https://doi.org/10.1186/gm405) | [PMC free article](/articles/PMC3706890/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23311897/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med.&title=Genetics%20Is%20a%20Major%20Determinant%20of%20Expression%20of%20the%20Human%20Hepatic%20Uptake%20Transporter%20OATP1B1,%20but%20Not%20of%20OATP1B3%20and%20OATP2B1&author=A.%20T.%20Nies&author=M.%20Niemi&author=O.%20Burk&author=S.%20Winter&author=U.%20M.%20Zanger&volume=5&issue=1&publication_year=2013&pages=1&pmid=23311897&doi=10.1186/gm405&)

23. Nies A. T., Schaeffeler E., van der Kuip H., Cascorbi I., Bruhn O., Kneba M., et al. (2014). Cellular Uptake of Imatinib into Leukemic Cells Is Independent of Human Organic Cation Transporter 1 (OCT1). Clin. Cancer Res. 20 (4), 985–994. 10.1158/1078-0432.CCR-13-1999  [DOI](https://doi.org/10.1158/1078-0432.CCR-13-1999) | [PMC free article](/articles/PMC3932302/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24352644/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&title=Cellular%20Uptake%20of%20Imatinib%20into%20Leukemic%20Cells%20Is%20Independent%20of%20Human%20Organic%20Cation%20Transporter%201%20(OCT1)&author=A.%20T.%20Nies&author=E.%20Schaeffeler&author=H.%20van%20der%20Kuip&author=I.%20Cascorbi&author=O.%20Bruhn&volume=20&issue=4&publication_year=2014&pages=985-994&pmid=24352644&doi=10.1158/1078-0432.CCR-13-1999&)

24. O’Brien V. P., Bokelmann K., Ramírez J., Jobst K., Ratain M. J., Brockmöller J., et al. (2013). Hepatocyte Nuclear Factor 1 Regulates the Expression of the Organic Cation Transporter 1 via Binding to an Evolutionary Conserved Region in Intron 1 of the OCT1 Gene. J. Pharmacol. Exp. Ther. 347 (1), 181–192. 10.1124/jpet.113.206359  [DOI](https://doi.org/10.1124/jpet.113.206359) | [PMC free article](/articles/PMC3781413/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23922447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=Hepatocyte%20Nuclear%20Factor%201%20Regulates%20the%20Expression%20of%20the%20Organic%20Cation%20Transporter%201%20via%20Binding%20to%20an%20Evolutionary%20Conserved%20Region%20in%20Intron%201%20of%20the%20OCT1%20Gene&author=V.%20P.%20O%E2%80%99Brien&author=K.%20Bokelmann&author=J.%20Ram%C3%ADrez&author=K.%20Jobst&author=M.%20J.%20Ratain&volume=347&issue=1&publication_year=2013&pages=181-192&pmid=23922447&doi=10.1124/jpet.113.206359&)

25. Rulcova A., Krausova L., Smutny T., Vrzal R., Dvorak Z., Jover R., et al. (2013). Glucocorticoid Receptor Regulates Organic Cation Transporter 1 (OCT1, SLC22A1) Expression via HNF4α Upregulation in Primary Human Hepatocytes. Pharmacol. Rep. 65 (5), 1322–1335. 10.1016/s1734-1140(13)71491-9  [DOI](https://doi.org/10.1016/s1734-1140(13)71491-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24399729/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Rep.&title=Glucocorticoid%20Receptor%20Regulates%20Organic%20Cation%20Transporter%201%20(OCT1,%20SLC22A1)%20Expression%20via%20HNF4%CE%B1%20Upregulation%20in%20Primary%20Human%20Hepatocytes&author=A.%20Rulcova&author=L.%20Krausova&author=T.%20Smutny&author=R.%20Vrzal&author=Z.%20Dvorak&volume=65&issue=5&publication_year=2013&pages=1322-1335&pmid=24399729&doi=10.1016/s1734-1140(13)71491-9&)

26. Saborowski M., Kullak-Ublick G. A., Eloranta J. J. (2006). The Human Organic Cation Transporter-1 Gene Is Transactivated by Hepatocyte Nuclear Factor-4α. J. Pharmacol. Exp. Ther. 317 (2), 778–785. 10.1124/jpet.105.099929  [DOI](https://doi.org/10.1124/jpet.105.099929) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16436500/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=The%20Human%20Organic%20Cation%20Transporter-1%20Gene%20Is%20Transactivated%20by%20Hepatocyte%20Nuclear%20Factor-4%CE%B1&author=M.%20Saborowski&author=G.%20A.%20Kullak-Ublick&author=J.%20J.%20Eloranta&volume=317&issue=2&publication_year=2006&pages=778-785&pmid=16436500&doi=10.1124/jpet.105.099929&)

27. Schaeffeler E., Hellerbrand C., Nies A. T., Winter S., Kruck S., Hofmann U., et al. (2011). DNA Methylation Is Associated with Downregulation of the Organic Cation Transporter OCT1 (SLC22A1) in Human Hepatocellular Carcinoma. Genome Med. 3 (12), 82. 10.1186/gm298  [DOI](https://doi.org/10.1186/gm298) | [PMC free article](/articles/PMC3334547/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22196450/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med.&title=DNA%20Methylation%20Is%20Associated%20with%20Downregulation%20of%20the%20Organic%20Cation%20Transporter%20OCT1%20(SLC22A1)%20in%20Human%20Hepatocellular%20Carcinoma&author=E.%20Schaeffeler&author=C.%20Hellerbrand&author=A.%20T.%20Nies&author=S.%20Winter&author=S.%20Kruck&volume=3&issue=12&publication_year=2011&pages=82&pmid=22196450&doi=10.1186/gm298&)

28. Schaller L., Lauschke V. M. (2019). The Genetic Landscape of the Human Solute Carrier (SLC) Transporter Superfamily. Hum. Genet. 138 (11-12), 1359–1377. 10.1007/s00439-019-02081-x  [DOI](https://doi.org/10.1007/s00439-019-02081-x) | [PMC free article](/articles/PMC6874521/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31679053/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Genet.&title=The%20Genetic%20Landscape%20of%20the%20Human%20Solute%20Carrier%20(SLC)%20Transporter%20Superfamily&author=L.%20Schaller&author=V.%20M.%20Lauschke&volume=138&issue=11-12&publication_year=2019&pages=1359-1377&pmid=31679053&doi=10.1007/s00439-019-02081-x&)

29. Seitz T., Stalmann R., Dalila N., Chen J., Pojar S., Dos Santos Pereira J. N., et al. (2015). Global Genetic Analyses Reveal Strong Inter-ethnic Variability in the Loss of Activity of the Organic Cation Transporter OCT1. Genome Med. 7 (1), 56. 10.1186/s13073-015-0172-0  [DOI](https://doi.org/10.1186/s13073-015-0172-0) | [PMC free article](/articles/PMC4495841/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26157489/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med.&title=Global%20Genetic%20Analyses%20Reveal%20Strong%20Inter-ethnic%20Variability%20in%20the%20Loss%20of%20Activity%20of%20the%20Organic%20Cation%20Transporter%20OCT1&author=T.%20Seitz&author=R.%20Stalmann&author=N.%20Dalila&author=J.%20Chen&author=S.%20Pojar&volume=7&issue=1&publication_year=2015&pages=56&pmid=26157489&doi=10.1186/s13073-015-0172-0&)

30. Shikata E., Yamamoto R., Takane H., Shigemasa C., Ikeda T., Otsubo K., et al. (2007). Human Organic Cation Transporter (OCT1 and OCT2) Gene Polymorphisms and Therapeutic Effects of Metformin. J. Hum. Genet. 52 (2), 117–122. 10.1007/s10038-006-0087-0  [DOI](https://doi.org/10.1007/s10038-006-0087-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17111267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hum.%20Genet.&title=Human%20Organic%20Cation%20Transporter%20(OCT1%20and%20OCT2)%20Gene%20Polymorphisms%20and%20Therapeutic%20Effects%20of%20Metformin&author=E.%20Shikata&author=R.%20Yamamoto&author=H.%20Takane&author=C.%20Shigemasa&author=T.%20Ikeda&volume=52&issue=2&publication_year=2007&pages=117-122&pmid=17111267&doi=10.1007/s10038-006-0087-0&)

31. Shu Y., Brown C., Castro R., Shi R., Lin E., Owen R., et al. (2008). Effect of Genetic Variation in the Organic Cation Transporter 1, OCT1, on Metformin Pharmacokinetics. Clin. Pharmacol. Ther. 83 (2), 273–280. 10.1038/sj.clpt.6100275  [DOI](https://doi.org/10.1038/sj.clpt.6100275) | [PMC free article](/articles/PMC2976713/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17609683/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Effect%20of%20Genetic%20Variation%20in%20the%20Organic%20Cation%20Transporter%201,%20OCT1,%20on%20Metformin%20Pharmacokinetics&author=Y.%20Shu&author=C.%20Brown&author=R.%20Castro&author=R.%20Shi&author=E.%20Lin&volume=83&issue=2&publication_year=2008&pages=273-280&pmid=17609683&doi=10.1038/sj.clpt.6100275&)

32. Shu Y., Leabman M. K., Feng B., Mangravite L. M., Huang C. C., Stryke D., et al. (2003). Evolutionary Conservation Predicts Function of Variants of the Human Organic Cation Transporter, OCT1. Proc. Natl. Acad. Sci. 100 (10), 5902–5907. 10.1073/pnas.0730858100  [DOI](https://doi.org/10.1073/pnas.0730858100) | [PMC free article](/articles/PMC156299/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12719534/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.&title=Evolutionary%20Conservation%20Predicts%20Function%20of%20Variants%20of%20the%20Human%20Organic%20Cation%20Transporter,%20OCT1&author=Y.%20Shu&author=M.%20K.%20Leabman&author=B.%20Feng&author=L.%20M.%20Mangravite&author=C.%20C.%20Huang&volume=100&issue=10&publication_year=2003&pages=5902-5907&pmid=12719534&doi=10.1073/pnas.0730858100&)

33. Shu Y., Sheardown S. A., Brown C., Owen R. P., Zhang S., Castro R. A., et al. (2007). Effect of Genetic Variation in the Organic Cation Transporter 1 (OCT1) on Metformin Action. J. Clin. Invest. 117 (5), 1422–1431. 10.1172/JCI30558  [DOI](https://doi.org/10.1172/JCI30558) | [PMC free article](/articles/PMC1857259/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17476361/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Invest.&title=Effect%20of%20Genetic%20Variation%20in%20the%20Organic%20Cation%20Transporter%201%20(OCT1)%20on%20Metformin%20Action&author=Y.%20Shu&author=S.%20A.%20Sheardown&author=C.%20Brown&author=R.%20P.%20Owen&author=S.%20Zhang&volume=117&issue=5&publication_year=2007&pages=1422-1431&pmid=17476361&doi=10.1172/JCI30558&)

34. Smith P. K., Krohn R. I., Hermanson G. T., Mallia A. K., Gartner F. H., Provenzano M. D., et al. (1985). Measurement of Protein Using Bicinchoninic Acid. Anal. Biochem. 150 (1), 76–85. 10.1016/0003-2697(85)90442-7  [DOI](https://doi.org/10.1016/0003-2697(85)90442-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3843705/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anal.%20Biochem.&title=Measurement%20of%20Protein%20Using%20Bicinchoninic%20Acid&author=P.%20K.%20Smith&author=R.%20I.%20Krohn&author=G.%20T.%20Hermanson&author=A.%20K.%20Mallia&author=F.%20H.%20Gartner&volume=150&issue=1&publication_year=1985&pages=76-85&pmid=3843705&doi=10.1016/0003-2697(85)90442-7&)

35. Stamer U. M., Musshoff F., Stüber F., Brockmöller J., Steffens M., Tzvetkov M. V. (2016). Loss-of-function Polymorphisms in the Organic Cation Transporter OCT1 Are Associated with Reduced Postoperative Tramadol Consumption. Pain 157 (11), 2467–2475. 10.1097/j.pain.0000000000000662  [DOI](https://doi.org/10.1097/j.pain.0000000000000662) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27541716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain&title=Loss-of-function%20Polymorphisms%20in%20the%20Organic%20Cation%20Transporter%20OCT1%20Are%20Associated%20with%20Reduced%20Postoperative%20Tramadol%20Consumption&author=U.%20M.%20Stamer&author=F.%20Musshoff&author=F.%20St%C3%BCber&author=J.%20Brockm%C3%B6ller&author=M.%20Steffens&volume=157&issue=11&publication_year=2016&pages=2467-2475&pmid=27541716&doi=10.1097/j.pain.0000000000000662&)

36. Takahashi N., Miura M., Scott S. A., Kagaya H., Kameoka Y., Tagawa H., et al. (2010). Influence of CYP3A5 and Drug Transporter Polymorphisms on Imatinib Trough Concentration and Clinical Response Among Patients with Chronic Phase Chronic Myeloid Leukemia. J. Hum. Genet. 55 (11), 731–737. 10.1038/jhg.2010.98  [DOI](https://doi.org/10.1038/jhg.2010.98) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20720558/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Hum.%20Genet.&title=Influence%20of%20CYP3A5%20and%20Drug%20Transporter%20Polymorphisms%20on%20Imatinib%20Trough%20Concentration%20and%20Clinical%20Response%20Among%20Patients%20with%20Chronic%20Phase%20Chronic%20Myeloid%20Leukemia&author=N.%20Takahashi&author=M.%20Miura&author=S.%20A.%20Scott&author=H.%20Kagaya&author=Y.%20Kameoka&volume=55&issue=11&publication_year=2010&pages=731-737&pmid=20720558&doi=10.1038/jhg.2010.98&)

37. Tarasova L., Kalnina I., Geldnere K., Bumbure A., Ritenberga R., Nikitina-Zake L., et al. (2012). Association of Genetic Variation in the Organic Cation Transporters OCT1, OCT2 and Multidrug and Toxin Extrusion 1 Transporter Protein Genes with the Gastrointestinal Side Effects and Lower BMI in Metformin-Treated Type 2 Diabetes Patients. Pharmacogenet. Genom. 22 (9), 659–666. 10.1097/FPC.0b013e3283561666  [DOI](https://doi.org/10.1097/FPC.0b013e3283561666) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22735389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genom.&title=Association%20of%20Genetic%20Variation%20in%20the%20Organic%20Cation%20Transporters%20OCT1,%20OCT2%20and%20Multidrug%20and%20Toxin%20Extrusion%201%20Transporter%20Protein%20Genes%20with%20the%20Gastrointestinal%20Side%20Effects%20and%20Lower%20BMI%20in%20Metformin-Treated%20Type%202%20Diabetes%20Patients&author=L.%20Tarasova&author=I.%20Kalnina&author=K.%20Geldnere&author=A.%20Bumbure&author=R.%20Ritenberga&volume=22&issue=9&publication_year=2012&pages=659-666&pmid=22735389&doi=10.1097/FPC.0b013e3283561666&)

38. Tzvetkov M. V., Dalila N., Faltraco F. (2016). “Genetic Variability in Organic Cation Transporters: Pathophysiological Manifestations and Consequences for Drug Pharmacokinetics and Efficacy,” in Organic Cation Transporters. Editors Ciarimboli G., Gautron S., Schlatter E. (Cham:Springer; ), 93–93.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Organic%20Cation%20Transporters&author=M.%20V.%20Tzvetkov&author=N.%20Dalila&author=F.%20Faltraco&publication_year=2016&)

39. Tzvetkov M. V., Dos Santos Pereira J. N., Meineke I., Saadatmand A. R., Stingl J. C., Brockmöller J. (2013). Morphine Is a Substrate of the Organic Cation Transporter OCT1 and Polymorphisms in OCT1 Gene Affect Morphine Pharmacokinetics after Codeine Administration. Biochem. Pharmacol. 86 (5), 666–678. 10.1016/j.bcp.2013.06.019  [DOI](https://doi.org/10.1016/j.bcp.2013.06.019) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23835420/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Pharmacol.&title=Morphine%20Is%20a%20Substrate%20of%20the%20Organic%20Cation%20Transporter%20OCT1%20and%20Polymorphisms%20in%20OCT1%20Gene%20Affect%20Morphine%20Pharmacokinetics%20after%20Codeine%20Administration&author=M.%20V.%20Tzvetkov&author=J.%20N.%20Dos%20Santos%20Pereira&author=I.%20Meineke&author=A.%20R.%20Saadatmand&author=J.%20C.%20Stingl&volume=86&issue=5&publication_year=2013&pages=666-678&pmid=23835420&doi=10.1016/j.bcp.2013.06.019&)

40. Tzvetkov M. V., Matthaei J., Pojar S., Faltraco F., Vogler S., Prukop T., et al. (2018). Increased Systemic Exposure and Stronger Cardiovascular and Metabolic Adverse Reactions to Fenoterol in Individuals with HeritableOCT1Deficiency. Clin. Pharmacol. Ther. 103 (5), 868–878. 10.1002/cpt.812  [DOI](https://doi.org/10.1002/cpt.812) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28791698/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Increased%20Systemic%20Exposure%20and%20Stronger%20Cardiovascular%20and%20Metabolic%20Adverse%20Reactions%20to%20Fenoterol%20in%20Individuals%20with%20HeritableOCT1Deficiency&author=M.%20V.%20Tzvetkov&author=J.%20Matthaei&author=S.%20Pojar&author=F.%20Faltraco&author=S.%20Vogler&volume=103&issue=5&publication_year=2018&pages=868-878&pmid=28791698&doi=10.1002/cpt.812&)

41. Tzvetkov M. V., Saadatmand A. R., Bokelmann K., Meineke I., Kaiser R., Brockmöller J. (2012). Effects of OCT1 Polymorphisms on the Cellular Uptake, Plasma Concentrations and Efficacy of the 5-HT3 Antagonists Tropisetron and Ondansetron. Pharmacogenomics J. 12 (1), 22–29. 10.1038/tpj.2010.75  [DOI](https://doi.org/10.1038/tpj.2010.75) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20921968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Effects%20of%20OCT1%20Polymorphisms%20on%20the%20Cellular%20Uptake,%20Plasma%20Concentrations%20and%20Efficacy%20of%20the%205-HT3%20Antagonists%20Tropisetron%20and%20Ondansetron&author=M.%20V.%20Tzvetkov&author=A.%20R.%20Saadatmand&author=K.%20Bokelmann&author=I.%20Meineke&author=R.%20Kaiser&volume=12&issue=1&publication_year=2012&pages=22-29&pmid=20921968&doi=10.1038/tpj.2010.75&)

42. Tzvetkov M. V., Saadatmand A. R., Lötsch J., Tegeder I., Stingl J. C., Brockmöller J. (2011). Genetically Polymorphic OCT1: Another Piece in the Puzzle of the Variable Pharmacokinetics and Pharmacodynamics of the Opioidergic Drug Tramadol. Clin. Pharmacol. Ther. 90 (1), 143–150. 10.1038/clpt.2011.56  [DOI](https://doi.org/10.1038/clpt.2011.56) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21562485/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genetically%20Polymorphic%20OCT1:%20Another%20Piece%20in%20the%20Puzzle%20of%20the%20Variable%20Pharmacokinetics%20and%20Pharmacodynamics%20of%20the%20Opioidergic%20Drug%20Tramadol&author=M.%20V.%20Tzvetkov&author=A.%20R.%20Saadatmand&author=J.%20L%C3%B6tsch&author=I.%20Tegeder&author=J.%20C.%20Stingl&volume=90&issue=1&publication_year=2011&pages=143-150&pmid=21562485&doi=10.1038/clpt.2011.56&)

43. Tzvetkov M. V., Seitz T., Bokelmann K., Mueller T., Brockmöller J., Koepsell H. (2014). Does the Haplotype Met408-Del420, Which Was Apparently Predictive for Imatinib Efficacy, Really Exist and How Strongly May it Affect OCT1 Activity? Blood 123 (9), 1427–1429. 10.1182/blood-2013-11-535864  [DOI](https://doi.org/10.1182/blood-2013-11-535864) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24578499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Does%20the%20Haplotype%20Met408-Del420,%20Which%20Was%20Apparently%20Predictive%20for%20Imatinib%20Efficacy,%20Really%20Exist%20and%20How%20Strongly%20May%20it%20Affect%20OCT1%20Activity?&author=M.%20V.%20Tzvetkov&author=T.%20Seitz&author=K.%20Bokelmann&author=T.%20Mueller&author=J.%20Brockm%C3%B6ller&volume=123&issue=9&publication_year=2014&pages=1427-1429&pmid=24578499&doi=10.1182/blood-2013-11-535864&)

44. Vaidya S., Ghosh K., Shanmukhaiah C., Vundinti B. R. (2015). Genetic Variations of hOCT1 Gene and CYP3A4/A5 Genes and Their Association with Imatinib Response in Chronic Myeloid Leukemia. Eur. J. Pharmacol. 765, 124–130. 10.1016/j.ejphar.2015.08.034  [DOI](https://doi.org/10.1016/j.ejphar.2015.08.034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26300393/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pharmacol.&title=Genetic%20Variations%20of%20hOCT1%20Gene%20and%20CYP3A4/A5%20Genes%20and%20Their%20Association%20with%20Imatinib%20Response%20in%20Chronic%20Myeloid%20Leukemia&author=S.%20Vaidya&author=K.%20Ghosh&author=C.%20Shanmukhaiah&author=B.%20R.%20Vundinti&volume=765&publication_year=2015&pages=124-130&pmid=26300393&doi=10.1016/j.ejphar.2015.08.034&)

45. Wang D.-S., Jonker J. W., Kato Y., Kusuhara H., Schinkel A. H., Sugiyama Y. (2002). Involvement of Organic Cation Transporter 1 in Hepatic and Intestinal Distribution of Metformin. J. Pharmacol. Exp. Ther. 302 (2), 510–515. 10.1124/jpet.102.034140  [DOI](https://doi.org/10.1124/jpet.102.034140) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12130709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=Involvement%20of%20Organic%20Cation%20Transporter%201%20in%20Hepatic%20and%20Intestinal%20Distribution%20of%20Metformin&author=D.-S.%20Wang&author=J.%20W.%20Jonker&author=Y.%20Kato&author=H.%20Kusuhara&author=A.%20H.%20Schinkel&volume=302&issue=2&publication_year=2002&pages=510-515&pmid=12130709&doi=10.1124/jpet.102.034140&)

46. Wang L., Giannoudis A., Lane S., Williamson P., Pirmohamed M., Clark R. (2008). Expression of the Uptake Drug Transporter hOCT1 Is an Important Clinical Determinant of the Response to Imatinib in Chronic Myeloid Leukemia. Clin. Pharmacol. Ther. 83 (2), 258–264. 10.1038/sj.clpt.6100268  [DOI](https://doi.org/10.1038/sj.clpt.6100268) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17568400/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Expression%20of%20the%20Uptake%20Drug%20Transporter%20hOCT1%20Is%20an%20Important%20Clinical%20Determinant%20of%20the%20Response%20to%20Imatinib%20in%20Chronic%20Myeloid%20Leukemia&author=L.%20Wang&author=A.%20Giannoudis&author=S.%20Lane&author=P.%20Williamson&author=M.%20Pirmohamed&volume=83&issue=2&publication_year=2008&pages=258-264&pmid=17568400&doi=10.1038/sj.clpt.6100268&)

47. Wang L., Prasad B., Salphati L., Chu X., Gupta A., Hop C. E. C. A., et al. (2015). Interspecies Variability in Expression of Hepatobiliary Transporters across Human, Dog, Monkey, and Rat as Determined by Quantitative Proteomics. Drug Metab. Dispos 43 (3), 367–374. 10.1124/dmd.114.061580  [DOI](https://doi.org/10.1124/dmd.114.061580) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25534768/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos&title=Interspecies%20Variability%20in%20Expression%20of%20Hepatobiliary%20Transporters%20across%20Human,%20Dog,%20Monkey,%20and%20Rat%20as%20Determined%20by%20Quantitative%20Proteomics&author=L.%20Wang&author=B.%20Prasad&author=L.%20Salphati&author=X.%20Chu&author=A.%20Gupta&volume=43&issue=3&publication_year=2015&pages=367-374&pmid=25534768&doi=10.1124/dmd.114.061580&)

48. White D. L., Saunders V. A., Dang P., Engler J., Hughes T. P. (2010). OCT-1 Activity Measurement Provides a Superior Imatinib Response Predictor Than Screening for Single-Nucleotide Polymorphisms of OCT-1. Leukemia 24 (11), 1962–1965. 10.1038/leu.2010.188  [DOI](https://doi.org/10.1038/leu.2010.188) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20811406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=OCT-1%20Activity%20Measurement%20Provides%20a%20Superior%20Imatinib%20Response%20Predictor%20Than%20Screening%20for%20Single-Nucleotide%20Polymorphisms%20of%20OCT-1&author=D.%20L.%20White&author=V.%20A.%20Saunders&author=P.%20Dang&author=J.%20Engler&author=T.%20P.%20Hughes&volume=24&issue=11&publication_year=2010&pages=1962-1965&pmid=20811406&doi=10.1038/leu.2010.188&)

49. White D. L., Saunders V. A., Dang P., Engler J., Zannettino A. C. W., Cambareri A. C., et al. (2006). OCT-1-mediated Influx Is a Key Determinant of the Intracellular Uptake of Imatinib but Not Nilotinib (AMN107): Reduced OCT-1 Activity Is the Cause of Low In Vitro Sensitivity to Imatinib. Blood 108 (2), 697–704. 10.1182/blood-2005-11-4687  [DOI](https://doi.org/10.1182/blood-2005-11-4687) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16597591/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=OCT-1-mediated%20Influx%20Is%20a%20Key%20Determinant%20of%20the%20Intracellular%20Uptake%20of%20Imatinib%20but%20Not%20Nilotinib%20(AMN107):%20Reduced%20OCT-1%20Activity%20Is%20the%20Cause%20of%20Low%20In%20Vitro%20Sensitivity%20to%20Imatinib&author=D.%20L.%20White&author=V.%20A.%20Saunders&author=P.%20Dang&author=J.%20Engler&author=A.%20C.%20W.%20Zannettino&volume=108&issue=2&publication_year=2006&pages=697-704&pmid=16597591&doi=10.1182/blood-2005-11-4687&)

50. Zhang L., Dresser M. J., Gray A. T., Yost S. C., Terashita S., Giacomini K. M. (1997). Cloning and Functional Expression of a Human Liver Organic Cation Transporter. Mol. Pharmacol. 51 (6), 913–921. 10.1124/mol.51.6.913  [DOI](https://doi.org/10.1124/mol.51.6.913) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9187257/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=Cloning%20and%20Functional%20Expression%20of%20a%20Human%20Liver%20Organic%20Cation%20Transporter&author=L.%20Zhang&author=M.%20J.%20Dresser&author=A.%20T.%20Gray&author=S.%20C.%20Yost&author=S.%20Terashita&volume=51&issue=6&publication_year=1997&pages=913-921&pmid=9187257&doi=10.1124/mol.51.6.913&)

51. Zhang J., Kobert K., Flouri T., Stamatakis A. (2014). PEAR: a fast and accurate Illumina Paired-End reAd mergeR. Bioinformatics 30, 614–620. 10.1093/bioinformatics/btt593  [DOI](https://doi.org/10.1093/bioinformatics/btt593) | [PMC free article](/articles/PMC3933873/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24142950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=PEAR:%20a%20fast%20and%20accurate%20Illumina%20Paired-End%20reAd%20mergeR&author=J.%20Zhang&author=K.%20Kobert&author=T.%20Flouri&author=A.%20Stamatakis&volume=30&publication_year=2014&pages=614-620&pmid=24142950&doi=10.1093/bioinformatics/btt593&)
