# Effects of NT5C2 germline variants on 6-mecaptopurine metabolism in children with acute lymphoblastic leukemia

## Metadata
**Authors:** Chuang Jiang, Wenjian Yang, Takaya Moriyama, Chengcheng Liu, Colton Smith, Wentao Yang, Maoxiang Qian, Ziping Li, Morten Tulstrup, Kjeld Schmiegelow, Kristine R Crews, Hui Zhang, Ching-Hon Pui, William Evans, Mary Relling, Smita Bhatia, Jun J YangP
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2020 Nov 24
**DOI:** [10.1002/cpt.2095](https://doi.org/10.1002/cpt.2095)
**PMID:** 33124053
**PMCID:** PMC8081740
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8081740/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8081740/pdf/nihms-1687817.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8081740/pdf/nihms-1687817.pdf)

## Abstract

6-mercaptopurine (6-MP) is widely used in the treatment of acute lymphoblastic leukemia (ALL), and its cytotoxicity is primarily mediated by thioguanine nucleotide metabolites (TGN). Recent genome-wide association study has identified germline polymorphisms (e.g., rs72846714) in the NT5C2 gene associated with 6-MP metabolism in patients with ALL. However, the full spectrum of genetic variation in NT5C2 is unclear and its impact on 6-MP drug activation has not been comprehensively examined. To this end, we performed targeted sequencing of NT5C2 in 588 children with ALL and identified 121 single nucleotide polymorphisms (SNPs) nominally associated with erythrocyte TGN during 6-MP treatment (P < 0.05). Of these, 61 variants were validated in a replication cohort of 372 children with ALL. After considering linkage disequilibrium and multivariate analysis, we confirmed two clusters of variants, represented by rs72846714 and rs58700372, that independently affected 6-MP metabolism. Functional studies showed that rs58700372 directly altered the activity of an intronic enhancer, with the variant allele linked to higher transcription activity and reduced 6-MP metabolism (lower TGN). By contrast, rs72846714 was not located in a regulatory element and instead its association signal was explained by linkage disequilibrium with a proximal functional variant rs12256506 that activated NT5C2 transcription in-cis. Our results indicated that NT5C2 germline variation significantly contributes to inter-patient variability in thiopurine drug disposition.

Keywords: NT5C2, single nucleotide polymorphisms, thiopurine metabolism, acute lymphoblastic leukemia

## Introduction

6-mercaptopurine (6-MP) is widely used as an anticancer and immunosuppressive agent [[1](#R1)–[8](#R8)]. In acute lymphoblastic leukemia (ALL), prolonged daily 6-MP exposure during maintenance therapy is one of the major components of contemporary ALL treatment regimens [[7](#R7), [9](#R9)–[13](#R13)]. 6-MP has a narrow therapeutic index and frequent dose adjustment is needed to balance between antileukemic efficacy and treatment toxicity in patients [[7](#R7), [13](#R13), [14](#R14)]. However, large inter-patient variability in 6-MP disposition poses a significant challenge to clinically determine the optimal 6-MP dosage for each individual [[15](#R15), [16](#R16)].

As a prodrug, 6-MP is converted into thioguanine nucleotides TGN (thioguanosine monophosphate [TGMP], thioguanosine diphosphate [TGDP], thioguanosine triphosphate [TGTP]) via a series of enzymatic reactions [[8](#R8), [17](#R17)]. TGTP is further reduced to deoxythioguanosine triphosphate (TdGTP), which is then incorporated into double-stranded DNA (DNA-TG) to trigger futile mismatch repair and eventually apoptosis [[7](#R7), [18](#R18)–[20](#R20)]. Levels of these intracellular 6-MP metabolites can inform drug toxicity and response and have the potential to guide treatment individualization. For example, the NOPHO ALL2008 study demonstrated that DNA-TG concentration is significantly associated with the risk of leukemia relapse [[21](#R21)]. Variation in erythrocyte TGN level during maintenance therapy has been associated with relapse-free survival in some patients with ALL [[12](#R12)]. Therefore, therapeutic drug monitoring of thiopurine metabolites may be clinically relevant for patients receiving 6-MP.

6-MP metabolism is regulated by several key enzymes (e.g., NT5C2, TPMT, and NUDT15), and genetic variants in these genes have been linked to 6-MP toxicity and/or resistance. For example, inherited deficiency of TPMT and/or NUDT15 causes excessive accumulation of 6-MP active metabolites and severe toxicity, and genotype-guided dose reduction can dramatically mitigate side effects of this drug [[22](#R22)–[27](#R27)]. By contrast, somatic *NT5C2* mutations are common in relapsed ALL and encode hyperactive nucleotidase that degrades 6-MP active metabolites thioinosine monophosphate (TIMP) and TGMP, leading to lower TGTP, decreased DNA-TG, and 6-MP resistance [[28](#R28)–[32](#R32)]. The presence of sub-clonal ALL *NT5C2* mutations was associated with high risk of treatment failure and inferior outcome after relapse [[33](#R33)]. A recent genome-wide association study (GWAS) reported germline variants at the *NT5C2* locus (e.g., rs72846714) strongly associated with erythrocyte TGN level during ALL therapy [[34](#R34)].

Although the genome-wide significant association at the *NT5C2* locus signifies the impact of inherited variation in this gene on 6-MP pharmacokinetics, the full spectrum of genetic polymorphism in *NT5C2* is unclear and its impact on drug activation has not been systematically examined. To this end, we performed targeted sequencing of *NT5C2* to evaluate the contribution of germline *NT5C2* variants to inter-individual variations in 6-MP metabolites in children with ALL.

## Materials and methods

### Patients and therapy

The discovery cohort consisted of patients prospectively enrolled onto the Children’s Oncology Group (COG) AALL03N1 study. Details of enrollment criteria and study design have been described previously [[16](#R16), [26](#R26)]. Briefly, patients were treated on COG AALL03N1 frontline protocols for newly diagnosed ALL (or per frontline protocols for small number of patients), were enrolled onto this study for a total of six months maintenance therapy that included a daily dose of oral 6-MP. The protocol planned dose of 6-MP during maintenance phase was 75 mg/m^2^ per day for the COG AALL03N1 cohort, with provisions for dose adjustment based on the degree of myelosuppression (white blood cell count) and/or occurrence of infections.

The replication cohort consisted of children with ALL treated on the St Jude Children’s Research Hospital Total Therapy XV (St. Jude Total XV) protocol, with the planned dose of 75 mg/m^2^ in the low-risk arm or 50 mg/m^2^ in the standard-or high-risk arm during the phase of continuation therapy when TGN was measured [[35](#R35)]. *TPMT* and *NUDT15* genotyping was performed as described previously [[26](#R26)], and patients with known risk variants in either one of these two genes were excluded from the main analyses (N=74 and 47 in the COG AALL03N1 and St. Jude Total XV cohorts, respectively).

This study was approved by respective institutional review boards, and informed consent was obtained from parents, guardians, or patients, as appropriate.

### 6-MP metabolites measurement

Erythrocyte TGN levels were measured monthly for each participant during the six months study period of the COG AALL03N1 study, as described previously [[16](#R16)]. We generated a summarized TGN value for each patient based on their longitudinal TGN data using a linear mixed-effect model adjusting for time point of each measurement and prescribed 6-MP dose recorded monthly prior to the TGN measurement [[26](#R26)].

For St. Jude Total XV patients, TGN was measured at the start of Reinduction I during continuation therapy, using methods described previously [[36](#R36), [37](#R37)], and total 6-MP dose two weeks prior to sampling was recorded as total mg^2^/m^2^. TGN data from patients who received fewer than 12 of the 14 doses of 6-MP prior to drug metabolite measurement (N=18) or those with non-adherence based on self-report (N=4) were excluded.

### NT5C2 targeted sequencing and SNP genotype imputation

Germline DNA was extracted from blood or bone marrow samples collected during clinical remission. Illumina dual-indexed libraries were created from genomic DNA and pooled in sets of 96 before hybridization with customized Roche NimbleGene SeqCap EZ probes (Roche, Roche NimbleGen, Madison, WI, USA) to capture the *NT5C2* genomic region. Quantitative PCR was used to define the appropriate capture product titer necessary to efficiently populate an Illumina HiSeq 2000 flow cell for paired-end 2 × 100 bp sequencing. Coverage of greater than 20 x depth was achieved across more than 80% of the targeted regions for nearly all samples. Sequence reads in the FASTQ format were mapped and aligned using the Burrows-Wheeler Aligner (BWA) and genetic variants were called using the GATK pipeline (version 3.1), as previously described [[38](#R38)–[40](#R40)], and annotated using the ANNOVA program with the annotation databases including RefSeq.

In addition to targeted sequencing, we also performed imputation for variants at the *NT5C2* locus, on the basis of SNP genotype data from Affymetrix SNP6 arrays of COG and St. Jude patients [[26](#R26)]. We employed a large reference panel of human haplotypes from the Haplotype Reference Consortium (HRC r1.1 2016) in Michigan Imputation Server with ShapeIT (v2.r790) as the phasing tool for imputation, following procedures described previously [[41](#R41)]. Imputed variants were filtered based on imputation quality, allele frequency, call rate, and deviation from Hardy-Weinberg equilibrium. Specifically, variants with one of the following features were excluded: imputation quality metric r^2^ less than 0.3 (indicating inadequate accuracy of the imputed genotype); minor allele frequency in less than 1% in either one of our two patient cohorts; Hardy-Weinberg equilibrium p value less than 5.00x10^−4^; variants located outside *NT5C2* locus (beyond 100kb of the first and last *NT5C2* exons).

### Statistical analyses

The association of *NT5C2* genotype with TGN was analyzed in patients with wildtype genotype in *TPMT* and *NUDT15* to avoid confounding effects. A linear regression model was used to test association, with 6-MP dose and genetic ancestry as covariates. R statistical software (version 3.5; [http://www.r-project.org/](http://www.r-project.org/)) was used for all analyses unless indicated otherwise. Regional association plots were created by Locus zoom ([http://locuszoom.org/](http://locuszoom.org/)). We also explored association of NT5C2 variants with TGN in patients with heterozygous genotype at known TPMT and/or NUDT15 variants.

### NT5C2 recombinant protein production and 5’-nucleotidase activity assay

C-terminal Hisx6-tagged full-length cDNA for wildtype *NT5C2* was cloned into the pET30a expression vector using NdeI and HindIII restriction sites. Point mutations, p. Thr3Ala, p.Met15Thr and p.Ile528Val, (hereafter referred to as p.T3A, p.M15T, and p.I528V) were introduced by using the QuickChange II XL Site-Directed Mutagenesis Kit (Agilent Technologies). NT5C2 recombinant protein production and purification were performed using previously described methods [[28](#R28)]. Successful purification of NT5C2 proteins was confirmed by Coomassie brilliant blue staining ([Figure S1](#SD3)).

5’-nucleotidase activity was measured by quantifying the release of phosphate from TIMP (Toronto Research Chemicals), or TGMP (Jena Bioscience), using the SensoLyte Malachite Green Assay kit (AnaSpec) in assay buffer (MgCl_2_ 2 mmol/L, NaCl 120 mmol/L, KCl 5 mmol/L, glucose 10 mmol/L, HEPES 20 mmol/L). NT5C2 protein (300 ng) was incubated with 40 mmol/L TIMP, or TGMP at 37°C; for 30 minutes. Experiments were assessed in triplicates.

### Functional annotation and characterization of non-coding variants in NT5C2

Chromatin accessibility in human hematopoietic cells were extracted from [GSE74912](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74912) [[42](#R42)]. Transcription factor ChIP-seq clusters and DNase hypersensitive site data were from ENCODE project [[43](#R43), [44](#R44)]. All peaks were aligned to GRCh37/hg19 on UCSC genome browser [[45](#R45)–[47](#R47)].

To functionally characterize potential function of non-coding variants in *NT5C2*, 356-600bp regions encompassing the SNP of interest were amplified using CloneAmp HiFi PCR Premix (Clontech), and then cloned into the pGL4.23-mini/P vector including a minimal SV40 promoter upstream of firefly luciferase. The variants were introduced by site-directed mutagenesis, by using the QuickChange Lightning Site-Directed Mutagenesis Kit (Agilent). Primers used in this experiment were listed in [Table S1](#SD1). Human erythroleukemic K562 cells were transiently co-transfected with the pGL4.23 reporter construct and pGL-TK Renilla luciferase construct using Lonza 4D-Nucleofector system (Lonza). Firefly luciferase activity was measured 24 hours post-transfection by the Dual Luciferase Assay kit (Promega). Results were normalized to Renilla luciferase activity. These experiments were performed in triplicated for three independent times.

## Results

Two cohorts of children with newly-diagnosed ALL receiving 6-MP therapy were included in this study: the COG AALL03N1 and St. Jude Total XV clinical trials as the discovery and validation set, respectively ([Table 1](#T1)). After removing patients with known risk variants in *TPMT* and/or *NUDT15* and those without TGN measurements, 588 and 372 patients in the respective cohorts were included in the subsequent genetic association analyses.

### Table 1.

|   | AALL03N1(discovery cohort) | St. Jude Total XV(Validation cohort) |
| --- | --- | --- |
| Age at diagnosis, median (range), y | 5.0 (1-19) | 5.8 (1.0-18.9) |
| Sex |   |   |
| Male (N=) | 399 (67.9%) | 209 (56.2%) |
| Female (N=) | 189 (32.1%) | 163 (43.8%) |
| Genetically determined race/ethnicity |   |   |
| European (N=) | 189 (32.2%) | 250 (67.2%) |
| African (N=) | 97 (16.5%) | 47 (12.6%) |
| Hispanic (N=) | 186 (31.6%) | 56 (15.1%) |
| Other (N=) | 116 (19.7%) | 19 (5.1%) |
| 6-MP dosage (mg/m2/day), median (range) | 65.8 (12.7-159.9) | 63.2 (17.9-119.7) |

Table 1 Caption: Patient characteristics of the COG AALL03N1 and the St. Jude Total XV cohorts, used for discovery and validation analyses, respectively.

### Targeted resequencing of NT5C2 identified three missense variants

Targeted sequencing of *NT5C2* in the COG AALL03N1 cohort identified three missense variants, p.T3A (rs10883841), p.M15T (rs751896016), and I528V (rs756913240) ([Figure 1](#F1)). The p.T3A variant was recurrent with minor allele frequency of 9.3%, and p.M15T and I528V were observed only once in this cohort ([Figure 2A](#F2)). None of the three variants were predicted to be damaging by *in silico* predication tools SIFT or PolyPhen-2. The common variant p.T3A was not associated with TGN in patients (*P* > 0.05, [Figure 2A](#F2)), and the singleton variants were not evaluated in the association analysis due to low frequency. Furthermore, we characterized enzymatic properties of all three variant NT5C2 proteins with thiopurine metabolites as substrates. NT5C2 p.I528V showed a modest increase in 5’-nucleotidase activity against TIMP but not with TGMP ([Figure 2B](#F2)), whereas the other two variants showed a similar activity compared to wildtype NT5C2 ([Figure 2B](#F2) and [2C](#F2)). Together, these results indicated that these three germline *NT5C2* missense variants did not affect erythrocyte TGN in children with ALL.

### Figure 1. Discovery of NT5C2 variants associated with 6-MP metabolism in children with ALL.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2b17/8081740/003bb1cdc695/nihms-1687817-f0001.jpg)

Targeted sequencing coupled with SNP imputation in the discovery COG AALL03N1 cohort (588 patients) identified 782 variants at the NT5C2 locus: three missense variants and 779 non-coding variants. The association of NT5C2 SNP genotype with erythrocyte TGN was tested using a linear mixed-effects model with dose intensity and genetic ancestry as covariates. This analysis was restricted to patients with wildtype genotype for TPMT and NUDT15. To identify novel TGN associated variants, we also performed multivariate analysis conditioning on index variant rs72846714 identified from previous GWAS [34]. Validation was performed in the independent St. Jude Total XV cohort (372 patients), following the same analysis strategy. #non-coding variants with minor allele frequency<1% were excluded from the association analyses due to statistical consideration. * All 22 variants are in linkage disequilibrium with pair-wise LD r2 > 0.99.

### Figure 2. Functional characterization of NT5C2 missense variants.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2b17/8081740/c7b1cb60dfae/nihms-1687817-f0002.jpg)

(A) Targeted sequencing of NT5C2 in the COG AALL03N1 cohort identified three missense variants. Chromosome location was aligned to GRCh37/hg19. SIFT and PolyPhen2 prediction tools were used to predict mutation function. MAF, minor allele frequency. (B) and (C) 5’-nucleotidase activity was determined of wildtype and variant NT5C2, with thiopurine metabolites as substrates. Purified NT5C2 proteins were incubated with TIMP or TGMP, and 5’-nucleotidase activity was measured by quantifying the release of free phosphate using the Malechite Green assay. Bars represent the mean values; the error bars represent the SD from triplicate. Ref allele, reference allele; Var allele, variant allele; #, P value of association of genotype and TGN was analyzed using a linear regression model with 6-MP dosage and genetic ancestry as covariates; *, P < 0.05 (Student t test). TGMP thioguanosine monophosphate; TIMP, thioinosine monophosphate.

### Association of non-coding variants in NT5C2 with erythrocyte TGN

Non-coding variants in *NT5C2* were determined by targeted sequencing as well as SNP imputation. A total of 779 non-coding variants in the genomic region encompassing the *NT5C2* gene and the 200Kb flanking region with a minor allele frequency of at least 1%, were identified in the COG AALL03N1 cohort and tested for association with TGN ([Figure 1](#F1)). After adjusting for 6-MP dose and genetic ancestry as covariates,121 variants were nominally associated with erythrocyte TGN during 6-MP treatment (*P* < 0.05, [Figure 1](#F1), [Figure S2](#SD3) and [Table S2](#SD2)). These variants were then tested for replication in the St. Jude Total XV cohort, and 61 of 121 variants were validated for association with TGN at the *P* < 0.05 level ([Figure 1](#F1), [Figure S2](#SD3) and [Table S2](#SD2)), including rs72846714 which was reported in the previous GWAS of 6-MP metabolites [[34](#R34)] ([Figure 3A](#F3) and [3B](#F3), [Table S2](#SD2)).

### Figure 3. Association of non-coding NT5C2 variants with erythrocyte TGN in the COG AAL03N1 and St. Jude Total XV cohorts.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2b17/8081740/6f10fa0c9807/nihms-1687817-f0003.jpg)

Effects of NT5C2 SNP genotype on TGN were shown for two non-coding variants, rs72846714 and rs58700372, selected to represent two independent association signals at this locus. (A) and (B) TGN was significantly lower for patients heterozygous or homozygous for variant allele at rs72846714 in the COG AALL03N1 cohort (A) and the St. Jude Total XV cohort (B), compared with reference allele. Genotype-TGN association was presented in a similar format in (C) and (D) for rs58700372 in the COG AALL03N1 cohort (C) and the St. Jude Total XV cohort (D). P value was estimated using linear regression model with dosage and genetic ancestry as covariates. Each box included data between 25th and 75th percentiles, with horizontal line indicating median. Whiskers indicated maximal and minimal observations within 1.5x length of box.

Of 61 validated *NT5C2* SNPs, only 22 remained significant in both discovery and validation cohorts after adjusting for the GWAS hit rs72846714 [[34](#R34)]. This cluster of 22 variants were in strong linkage disequilibrium with each other (r^2^>0.99), and represented a novel association at the *NT5C2* locus independent of the GWAS signal. For example, in the discovery cohort COG AALL03N1, there was a significant decrease in TGN metabolite levels by genotypes at the independent SNP rs58700372 from wildtype (TT, 154.8 ± 61.1) to homozygotes (CC, 108.4 ± 44.4), with the heterozygotes (TC, 136.2 ± 61.1) exhibiting an intermediate level of TGN (*P* = 6.69 × 10^−4^; [Figure 3C](#F3)). This association was replicated in the St. Jude Total XV cohort: mean levels of the metabolite were 225.5 ± 132.9, 193.3 ± 86.0, and 165.5 ± 49.3 in wildtype, heterozygous and homozygous individuals, respectively (*P* = 9.58× 10^−5^. [Figure 3D](#F3)).

Interestingly, even within patients heterozygous for known *TPMT* and/or *NUDT15* variants, *NT5C2* genotype at rs72846714 and rs58700372 appeared to further distinguish patients with differential 6-MP metabolism although this only reached statistical significance in the St. Jude Total XV cohort ([Figure S3](#SD3)). The frequency of variant allele at these two *NT5C2* SNPs also differ by race and ethnicity ([Figure S4](#SD3)). Taken together, we identified two clusters of non-coding variants that independently influenced 6-MP metabolism, represented by rs72846714 and rs58700372.

### Functional exploration of the germline variants

To gain insight into the biological basis of the association signals at these non-coding *NT5C2* variants, we firstly examined chromatin accessibility inferred from the ATAC-seq of human hematopoietic cells and histone mark ChIP-seq signals form ENCODE project [[42](#R42)–[44](#R44)].

For the previous reported GWAS hit rs72846714, we found that it was in the intronic region with minimal overlap with regulatory elements ([Figure 4A](#F4)), indicating the lack of transcriptional regulation function of this variant. However, amongst variants in strong LD with rs72846714, we identified five SNPs that showed significant association with TGN in both discovery and validation cohorts and were also in proximity of open chromatin segments in multiple hematopoiesis cells (rs12261294, rs12240508, rs7091462, rs113642369, and rs12256506) ([Figure 4A](#F4)). We postulated that one of these variants may directly regulate *NT5C2* transcription and thus causal to the GWAS signal at this locus. We performed luciferase assay to test the effects of rs72846714 and each of the five LD variants on transcription activity and found that only rs12256506 substantially influenced enhancer activity ([Figure 4B](#F4)). The substitution of the reference allele with the variant allele at this variant resulted in a 1.48-fold increase in transcription activity (*P* = 0.0049, [Figure 4B](#F4)), consistent with higher expression of *NT5C2* and thus lower TGN in erythrocytes ([Figure 3A](#F3) and [3B](#F3)). This is in line with the notion that NT5C2 degrades 6-MP active metabolites TIMP and TGMP, and thus its activity is inversely correlated with TGN.

### Figure 4. Functional characterization of NT5C2 non-coding variant rs72846714 and its LD SNPs.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2b17/8081740/7847ec072a8a/nihms-1687817-f0004.jpg)

A total of six SNPs were presented including rs72846714 and five SNPs that are in strong LD with the index variant (ALL cohorts, r2>0.6)) and also annotated to putative regulatory DNA. (A) Functional annotation of regulatory DNA at the NT5C2 locus. Genomic positions, scale and gene structure for GRCh37/hg19 were shown on the top. rs72846714 and its LD SNPs were marked in the middle panel. ATAC-seq signals of hematopoietic cells, DNase and TF ChIP clusters were also included, using publicly available datasets [42–44]. (B) Enhancer activity of cis-elements encompassing these SNPs was tested using luciferase reporter assay in K562 cells. Relative luciferase activity indicated the ratio between the value of variant allele over that from the reference allele. All experiments were performed in triplicate and repeated three times. Statistical significance was evaluated by using two-sided Student’s t test. Error bars indicate SD. ns, not significant; *, P < 0.05; **, P < 0.01.

By contrast, the novel variant rs58700372 was located within an open chromatin region based on ATAC-seq signal in erythroblast cells ([Figure 5A](#F5)). DNase accessibility and transcription factors ChIP-seq also suggested active transcription in this region ([Figure 5A](#F5)). Using luciferase assay to test its effect on transcription activity, we found that substitution of the reference allele with the variant allele resulted in a 3.78-fold increase in transcription activity ([Figure 5B](#F5)). These findings indicate that rs58700372 directly potentiates the activity of an intronic enhancer, activates *NT5C2* transcription, and thereby reduces 6-MP metabolism.

### Figure 5. Functional characterization of NT5C2 non-coding variant rs58700372.

![Figure 5.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2b17/8081740/fdb8e68d829c/nihms-1687817-f0005.jpg)

(A) Functional annotation of regulatory DNA within the genomic region encompassing rs58700372. Genomic positions, scale and gene structure for GRCh37/hg19 were shown on the top. rs58700372 was marked in the middle panel. ATAC-seq signals of hematopoietic cells, DNase and TF ChIP clusters were also included, using publicly available datasets [42–44]. (B) Luciferase assay was performed to determine enhancer activity of the DNA segment encompassing rs58700372 in K562 cells. Relative luciferase activity indicated the ratio between the value of the variant allele and that from the reference allele. This experiment was performed in triplicate and repeated three times. Statistical significance was evaluated by using two-sided Student’s t test. Error bars indicate SD; **, P < 0.01.

## Discussion

Prior pharmacogenetic studies have identified germline variants in *NUDT15* and *TPMT* associated with 6-MP sensitivity, and treatment modification based on *TPMT* and *NUDT15* genotype is a clear example of genetically guided precision medicine [[22](#R22)–[27](#R27)]. However, the majority of inter-individual variability in 6-MP pharmacokinetics remains unexplained. Recently, a genome-wide association study described a germline variant located in *NT5C2* intronic region associated with erythrocyte TGN during 6-MP therapy, pointing to potential effects of *NT5C2* germline variants on this phenotype [[34](#R34)].

NT5C2 is an ubiquitously expressed cytosolic nucleotidase involved in the maintenance of intracellular nucleotide pool homeostasis by promoting the clearance of excess purine nucleotides from cells [[48](#R48), [49](#R49)]. NT5C2 preferentially dephosphorylates the 6-hydroxypurine monophosphates inosine monophosphate (IMP), guanosine monophosphate (GMP), and xanthosine monophosphate (XMP), as well as the deoxyribose forms of IMP and GMP (dIMP and dGMP), facilitating the export of purine nucleosides. In the context of ALL therapy, NT5C2 can dephosphorylate and inactivate 6-MP metabolites TIMP and TGMP, and thus reduce the amount of TGN available for DNA incorporation and compromise cytotoxicity of this drug [[28](#R28)]. Acquisition of *NT5C2* mutations in relapsed ALL impedes the formation of DNA-TGN and is a frequent mechanism of 6-MP drug resistance [[29](#R29)–[32](#R32)].

In this study, we comprehensively evaluated *NT5C2* germline variants for their association with the 6-MP metabolite TGN during ALL therapy. We identified 61 *NT5C2* variants significantly associated with TGN after adjusting for dose and genetic ancestry in both discovery and validation cohorts. In line with previous GWAS [[34](#R34)], the variant A allele of rs72846714 was associated with lower TGN levels, indicating that this pharmacogenetic association is robust across different chemotherapy regimens and different patient populations. Importantly, we also identified a novel variant, namely rs58700372, that exhibited independent effect on 6-MP metabolite beyond that of the GWAS hit, suggesting multiple functional variants at this locus. However, neither of these two variants were associated with 6-MP dose intensity in our analyses ([Figure S5](#SD3)), plausibly because these germline *NT5C2* SNPs only modestly alter drug metabolism to a degree that is insufficient to cause clinical toxicity or drug resistance. It is also worth noting that there was a trend for *NT5C2* variants to be associated with TGN even within patients hemizygously deficient of *TPMT* and/or *NUDT15* ([Figure S3](#SD3)). This is relevant because these individuals exhibit wide variability in 6-MP tolerance and additional biomarkers are needed to precisely predict thiopurine toxicity. It remains possible that many other genetic variants influence 6-MP metabolism with relatively low penetrance, and their combined effects may warrant clinical consideration. There was also a trend that the variant alleles at rs72846714 and rs58700372 were linked to lower MeTIMP in erythrocytes in our ALL cohorts but it did not reach statistical significance ([Figure S6](#SD3)).

Unlike *TPMT* or *NUDT15*, there are no common germline coding variants in *NT5C2* that affect its activity. The majority of pharmacogenetic variants identified in this gene are non-coding. Located in intronic and intergenic regions, these *NT5C2* variants primarily affect gene function by transcription regulation. For example, the association signal at the previously reported GWAS hit rs72846714 was likely explained by a proximal variant rs12256506 that activates *NT5C2* transcription in-cis. Our analysis also identified rs58700372 for its independent association with TGN beyond that of the GWAS SNP. Our functional study indicated that this variant directly alters the activity of an intronic enhancer, with the variant allele linked to higher transcription activity and reduced 6-MP activation (lower TGN). Of note, rs58700372 is located ~50kb downstream of the coding region of *NT5C2*, and the potential mechanism by which this variant regulates *NT5C2* transcription may involve chromatin remodeling and long-distance interaction, which need to be examined in future studies. It remains unclear whether functional effects of these two *NT5C2* SNPs are restricted to erythrocytes and future studies are warranted to explore cellular models of a broader range of tissue origins.

In conclusion, we comprehensively analyzed the role of *NT5C2* germline variants in 6-MP metabolism during ALL therapy and identified functional SNPs that independently and directly regulate NT5C2 activity and 6-MP metabolism. Our results indicate that *NT5C2* germline variation contributes to inter-patient variability in thiopurine drug disposition.

## Supplementary Material

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

### What is the current knowledge on the topic?

Levels of intracellular 6-MP metabolites can inform drug toxicity and response and potentially inform treatment individualization. Recent GWAS study reported initial evidence linking *NT5C2* germline variants to 6-MP metabolism. However, the full spectrum of genetic polymorphism in *NT5C2 and* their impact on inter-patient variability in thiopurine drug disposition are largely unknown.

### What question did this study address?

This pharmacogenomics study comprehensively evaluated the contribution of genetic variations in *NT5C2* to 6-MP metabolite TGN in two ALL cohorts and identified two representative SNPs that independently affected 6-MP metabolism. Importantly, we also sought to understand the molecular mechanism by which these non-coding variants modulate NT5C2 function.

### What does this study add to our knowledge?

This is one of the first studies to describe the full spectrum of genetic variants in *NT5C2* and examined their effects on thiopurine metabolism in large cohorts of patients. We also provided functional evidence of how coding and non-coding variants modulate NT5C2 activity. Our results firmly established the importance of *NT5C2* germline variation in explaining inter-patient variability in thiopurine drug disposition.

### How might this change clinical pharmacology or translational science?

This study identified novel genetic variants significantly associated with thiopurine drug activation, therefore expanding the list of pharmacogenetic factors that may aid the optimization of pharmacotherapy using this class of drugs.

## Acknowledgments

We thank the patients and parents who participated in the clinical studies included in this report. This work was partially supported by National Institutes of Health grants GM118578, GM115279, and CA096670.

### Funding information

This work was partially supported by National Institutes of Health grants GM118578 GM115279, and CA096670.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Goldberg R and Irving PM, Toxicity and response to thiopurines in patients with inflammatory bowel disease. Expert review of gastroenterology & hepatology, 2015. 9(7): p. 891–900.  [DOI](https://doi.org/10.1586/17474124.2015.1039987) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25915575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20review%20of%20gastroenterology%20&%20hepatology&title=Toxicity%20and%20response%20to%20thiopurines%20in%20patients%20with%20inflammatory%20bowel%20disease&author=R%20Goldberg&author=PM%20Irving&volume=9&issue=7&publication_year=2015&pages=891-900&pmid=25915575&doi=10.1586/17474124.2015.1039987&)

2. Karran P and Attard N, Thiopurines in current medical practice: molecular mechanisms and contributions to therapy-related cancer. Nature Reviews Cancer, 2008. 8(1): p. 24–36.  [DOI](https://doi.org/10.1038/nrc2292) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18097462/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Cancer&title=Thiopurines%20in%20current%20medical%20practice:%20molecular%20mechanisms%20and%20contributions%20to%20therapy-related%20cancer&author=P%20Karran&author=N%20Attard&volume=8&issue=1&publication_year=2008&pages=24-36&pmid=18097462&doi=10.1038/nrc2292&)

3. Maltzman JS and Koretzky GA, Azathioprine: old drug, new actions. The Journal of clinical investigation, 2003. 111(8): p. 1122–1124.  [DOI](https://doi.org/10.1172/JCI18384) | [PMC free article](/articles/PMC152947/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12697731/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20clinical%20investigation&title=Azathioprine:%20old%20drug,%20new%20actions&author=JS%20Maltzman&author=GA%20Koretzky&volume=111&issue=8&publication_year=2003&pages=1122-1124&pmid=12697731&doi=10.1172/JCI18384&)

4. Pui C-H, et al. , Biology, risk stratification, and therapy of pediatric acute leukemias: an update. Journal of clinical oncology, 2011. 29(5): p. 551.  [DOI](https://doi.org/10.1200/JCO.2010.30.7405) | [PMC free article](/articles/PMC3071256/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21220611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20clinical%20oncology&title=Biology,%20risk%20stratification,%20and%20therapy%20of%20pediatric%20acute%20leukemias:%20an%20update&author=C-H%20Pui&volume=29&issue=5&publication_year=2011&pages=551&pmid=21220611&doi=10.1200/JCO.2010.30.7405&)

5. Reinisch W, et al. , Azathioprine versus mesalazine for prevention of postoperative clinical recurrence in patients with Crohn’s disease with endoscopic recurrence: efficacy and safety results of a randomised, double-blind, double-dummy, multicentre trial. Gut, 2010. 59(6): p. 752–759.  [DOI](https://doi.org/10.1136/gut.2009.194159) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20551460/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut&title=Azathioprine%20versus%20mesalazine%20for%20prevention%20of%20postoperative%20clinical%20recurrence%20in%20patients%20with%20Crohn%E2%80%99s%20disease%20with%20endoscopic%20recurrence:%20efficacy%20and%20safety%20results%20of%20a%20randomised,%20double-blind,%20double-dummy,%20multicentre%20trial&author=W%20Reinisch&volume=59&issue=6&publication_year=2010&pages=752-759&pmid=20551460&doi=10.1136/gut.2009.194159&)

6. Vora A, et al. Treatment reduction for children and young adults with low-risk acute lymphoblastic leukaemia defined by minimal residual disease (UKALL 2003): a randomised controlled trial. The lancet oncology, 2013. 14(3): p. 199–209.  [DOI](https://doi.org/10.1016/S1470-2045(12)70600-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23395119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20lancet%20oncology&title=Treatment%20reduction%20for%20children%20and%20young%20adults%20with%20low-risk%20acute%20lymphoblastic%20leukaemia%20defined%20by%20minimal%20residual%20disease%20(UKALL%202003):%20a%20randomised%20controlled%20trial&author=A%20Vora&volume=14&issue=3&publication_year=2013&pages=199-209&pmid=23395119&doi=10.1016/S1470-2045(12)70600-9&)

7. Koren G, et al. Systemic exposure to mercaptopurine as a prognostic factor in acute lymphocytic leukemia in children. New England Journal of Medicine, 1990. 323(1): p. 17–21.  [DOI](https://doi.org/10.1056/NEJM199007053230104) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2355954/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&title=Systemic%20exposure%20to%20mercaptopurine%20as%20a%20prognostic%20factor%20in%20acute%20lymphocytic%20leukemia%20in%20children&author=G%20Koren&volume=323&issue=1&publication_year=1990&pages=17-21&pmid=2355954&doi=10.1056/NEJM199007053230104&)

8. Elion GB, The purine path to chemotherapy. Science, 1989. 244(4900): p. 41–47.  [DOI](https://doi.org/10.1126/science.2649979) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2649979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20purine%20path%20to%20chemotherapy&author=GB%20Elion&volume=244&issue=4900&publication_year=1989&pages=41-47&pmid=2649979&doi=10.1126/science.2649979&)

9. Group CAC, Duration and intensity of maintenance chemotherapy in acute lymphoblastic leukaemia: overview of 42 trials involving 12 000 randomised children. The Lancet, 1996. 347(9018): p. 1783–1788.  [DOI](https://doi.org/10.1016/s0140-6736(96)91615-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8667921/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Lancet&title=Duration%20and%20intensity%20of%20maintenance%20chemotherapy%20in%20acute%20lymphoblastic%20leukaemia:%20overview%20of%2042%20trials%20involving%2012%20000%20randomised%20children&author=CAC%20Group&volume=347&issue=9018&publication_year=1996&pages=1783-1788&pmid=8667921&doi=10.1016/s0140-6736(96)91615-3&)

10. Lilleyman JS and Lennard L, Mercaptopurine metabolism and risk of relapse in childhood lymphoblastic leukaemia. The Lancet, 1994. 343(8907): p. 1188–1190.  [DOI](https://doi.org/10.1016/s0140-6736(94)92400-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7909868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Lancet&title=Mercaptopurine%20metabolism%20and%20risk%20of%20relapse%20in%20childhood%20lymphoblastic%20leukaemia&author=JS%20Lilleyman&author=L%20Lennard&volume=343&issue=8907&publication_year=1994&pages=1188-1190&pmid=7909868&doi=10.1016/s0140-6736(94)92400-7&)

11. Schmiegelow K, et al. , Mercaptopurine/methotrexate maintenance therapy of childhood acute lymphoblastic leukemia: clinical facts and fiction. Journal of pediatric hematology/oncology, 2014. 36(7): p. 503.  [DOI](https://doi.org/10.1097/MPH.0000000000000206) | [PMC free article](/articles/PMC4222610/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24936744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20pediatric%20hematology/oncology&title=Mercaptopurine/methotrexate%20maintenance%20therapy%20of%20childhood%20acute%20lymphoblastic%20leukemia:%20clinical%20facts%20and%20fiction&author=K%20Schmiegelow&volume=36&issue=7&publication_year=2014&pages=503&pmid=24936744&doi=10.1097/MPH.0000000000000206&)

12. Lennard L and Lilleyman JS, Variable mercaptopurine metabolism and treatment outcome in childhood lymphoblastic leukemia. Journal of Clinical Oncology, 1989. 7(12): p. 1816–1823.  [DOI](https://doi.org/10.1200/JCO.1989.7.12.1816) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2585022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Oncology&title=Variable%20mercaptopurine%20metabolism%20and%20treatment%20outcome%20in%20childhood%20lymphoblastic%20leukemia&author=L%20Lennard&author=JS%20Lilleyman&volume=7&issue=12&publication_year=1989&pages=1816-1823&pmid=2585022&doi=10.1200/JCO.1989.7.12.1816&)

13. Relling MV, et al. , Prognostic importance of 6-mercaptopurine dose intensity in acute lymphoblastic leukemia. Blood, The Journal of the American Society of Hematology, 1999. 93(9): p. 2817–2823.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10216075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood,%20The%20Journal%20of%20the%20American%20Society%20of%20Hematology&title=Prognostic%20importance%20of%206-mercaptopurine%20dose%20intensity%20in%20acute%20lymphoblastic%20leukemia&author=MV%20Relling&volume=93&issue=9&publication_year=1999&pages=2817-2823&pmid=10216075&)

14. Schmiegelow K, et al. , Maintenance therapy of childhood acute lymphoblastic leukemia revisited—Should drug doses be adjusted by white blood cell, neutrophil, or lymphocyte counts? Pediatric Blood & Cancer, 2016. 63(12): p. 2104–2111.  [DOI](https://doi.org/10.1002/pbc.26139) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27447547/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatric%20Blood%20&%20Cancer&title=Maintenance%20therapy%20of%20childhood%20acute%20lymphoblastic%20leukemia%20revisited%E2%80%94Should%20drug%20doses%20be%20adjusted%20by%20white%20blood%20cell,%20neutrophil,%20or%20lymphocyte%20counts?&author=K%20Schmiegelow&volume=63&issue=12&publication_year=2016&pages=2104-2111&pmid=27447547&doi=10.1002/pbc.26139&)

15. Bhatia S, et al. , Nonadherence to oral mercaptopurine and risk of relapse in Hispanic and non-Hispanic white children with acute lymphoblastic leukemia: a report from the children’s oncology group. Journal of Clinical Oncology, 2012. 30(17): p. 2094.  [DOI](https://doi.org/10.1200/JCO.2011.38.9924) | [PMC free article](/articles/PMC3601449/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22564992/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Oncology&title=Nonadherence%20to%20oral%20mercaptopurine%20and%20risk%20of%20relapse%20in%20Hispanic%20and%20non-Hispanic%20white%20children%20with%20acute%20lymphoblastic%20leukemia:%20a%20report%20from%20the%20children%E2%80%99s%20oncology%20group&author=S%20Bhatia&volume=30&issue=17&publication_year=2012&pages=2094&pmid=22564992&doi=10.1200/JCO.2011.38.9924&)

16. Bhatia S, et al. , Systemic exposure to thiopurines and risk of relapse in children with acute lymphoblastic leukemia: a Children’s Oncology Group Study. JAMA oncology, 2015. 1(3): p. 287–295.  [DOI](https://doi.org/10.1001/jamaoncol.2015.0245) | [PMC free article](/articles/PMC4561178/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26181173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20oncology&title=Systemic%20exposure%20to%20thiopurines%20and%20risk%20of%20relapse%20in%20children%20with%20acute%20lymphoblastic%20leukemia:%20a%20Children%E2%80%99s%20Oncology%20Group%20Study&author=S%20Bhatia&volume=1&issue=3&publication_year=2015&pages=287-295&pmid=26181173&doi=10.1001/jamaoncol.2015.0245&)

17. Elion G, The George Hitchings and Gertrude Elion lecture: the pharmacology of azathioprine. Ann NY Acad Sci, 1990. 685: p. 1239–1256.  [DOI](https://doi.org/10.1111/j.1749-6632.1993.tb35897.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8363248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20NY%20Acad%20Sci&title=The%20George%20Hitchings%20and%20Gertrude%20Elion%20lecture:%20the%20pharmacology%20of%20azathioprine&author=G%20Elion&volume=685&publication_year=1990&pages=1239-1256&pmid=8363248&doi=10.1111/j.1749-6632.1993.tb35897.x&)

18. Li G-M, The role of mismatch repair in DNA damage-induced apoptosis. Oncology research, 1999. 11(9): p. 393–400.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/10821533/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncology%20research&title=The%20role%20of%20mismatch%20repair%20in%20DNA%20damage-induced%20apoptosis&author=G-M%20Li&volume=11&issue=9&publication_year=1999&pages=393-400&pmid=10821533&)

19. Ling Y, et al. , 2′-Deoxy-6-thioguanosine 5′-triphosphate as a substrate for purified human DNA polymerases and calf thymus terminal deoxynucleotidyltransferase in vitro. Molecular pharmacology, 1991. 40(4): p. 508–514.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/1921985/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20pharmacology&title=2%E2%80%B2-Deoxy-6-thioguanosine%205%E2%80%B2-triphosphate%20as%20a%20substrate%20for%20purified%20human%20DNA%20polymerases%20and%20calf%20thymus%20terminal%20deoxynucleotidyltransferase%20in%20vitro&author=Y%20Ling&volume=40&issue=4&publication_year=1991&pages=508-514&pmid=1921985&)

20. Swann PF, et al. , Role of postreplicative DNA mismatch repair in the cytotoxic action of thioguanine. Science, 1996. 273(5278): p. 1109–1111.  [DOI](https://doi.org/10.1126/science.273.5278.1109) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8688098/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Role%20of%20postreplicative%20DNA%20mismatch%20repair%20in%20the%20cytotoxic%20action%20of%20thioguanine&author=PF%20Swann&volume=273&issue=5278&publication_year=1996&pages=1109-1111&pmid=8688098&doi=10.1126/science.273.5278.1109&)

21. Nielsen SN, et al. , DNA-thioguanine nucleotide concentration and relapse-free survival during maintenance therapy of childhood acute lymphoblastic leukaemia (NOPHO ALL2008): a prospective substudy of a phase 3 trial. The Lancet Oncology, 2017. 18(4): p. 515–524.  [DOI](https://doi.org/10.1016/S1470-2045(17)30154-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28258828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Lancet%20Oncology&title=DNA-thioguanine%20nucleotide%20concentration%20and%20relapse-free%20survival%20during%20maintenance%20therapy%20of%20childhood%20acute%20lymphoblastic%20leukaemia%20(NOPHO%20ALL2008):%20a%20prospective%20substudy%20of%20a%20phase%203%20trial&author=SN%20Nielsen&volume=18&issue=4&publication_year=2017&pages=515-524&pmid=28258828&doi=10.1016/S1470-2045(17)30154-7&)

22. Moriyama T, et al. , NUDT15 polymorphisms alter thiopurine metabolism and hematopoietic toxicity. Nature genetics, 2016. 48(4): p. 367–373.  [DOI](https://doi.org/10.1038/ng.3508) | [PMC free article](/articles/PMC5029084/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26878724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=NUDT15%20polymorphisms%20alter%20thiopurine%20metabolism%20and%20hematopoietic%20toxicity&author=T%20Moriyama&volume=48&issue=4&publication_year=2016&pages=367-373&pmid=26878724&doi=10.1038/ng.3508&)

23. Nishii R, et al. , Preclinical evaluation of NUDT15-guided thiopurine therapy and its effects on toxicity and antileukemic efficacy. Blood, 2018. 131(22): p. 2466–2474.  [DOI](https://doi.org/10.1182/blood-2017-11-815506) | [PMC free article](/articles/PMC5981167/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29572377/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Preclinical%20evaluation%20of%20NUDT15-guided%20thiopurine%20therapy%20and%20its%20effects%20on%20toxicity%20and%20antileukemic%20efficacy&author=R%20Nishii&volume=131&issue=22&publication_year=2018&pages=2466-2474&pmid=29572377&doi=10.1182/blood-2017-11-815506&)

24. Relling MV, et al. , Clinical pharmacogenetics implementation consortium guideline for thiopurine dosing based on TPMT and NUDT 15 genotypes: 2018 update. Clinical Pharmacology & Therapeutics, 2019. 105(5): p. 1095–1105.  [DOI](https://doi.org/10.1002/cpt.1304) | [PMC free article](/articles/PMC6576267/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30447069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20&%20Therapeutics&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20for%20thiopurine%20dosing%20based%20on%20TPMT%20and%20NUDT%2015%20genotypes:%202018%20update&author=MV%20Relling&volume=105&issue=5&publication_year=2019&pages=1095-1105&pmid=30447069&doi=10.1002/cpt.1304&)

25. Suiter CC, et al. , Massively parallel variant characterization identifies NUDT15 alleles associated with thiopurine toxicity. Proceedings of the National Academy of Sciences, 2020. 117(10): p. 5394–5401.  [DOI](https://doi.org/10.1073/pnas.1915680117) | [PMC free article](/articles/PMC7071893/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32094176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&title=Massively%20parallel%20variant%20characterization%20identifies%20NUDT15%20alleles%20associated%20with%20thiopurine%20toxicity&author=CC%20Suiter&volume=117&issue=10&publication_year=2020&pages=5394-5401&pmid=32094176&doi=10.1073/pnas.1915680117&)

26. Yang JJ, et al. , Inherited NUDT15 variant is a genetic determinant of mercaptopurine intolerance in children with acute lymphoblastic leukemia. Journal of clinical oncology, 2015. 33(11): p. 1235.  [DOI](https://doi.org/10.1200/JCO.2014.59.4671) | [PMC free article](/articles/PMC4375304/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25624441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20clinical%20oncology&title=Inherited%20NUDT15%20variant%20is%20a%20genetic%20determinant%20of%20mercaptopurine%20intolerance%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=JJ%20Yang&volume=33&issue=11&publication_year=2015&pages=1235&pmid=25624441&doi=10.1200/JCO.2014.59.4671&)

27. Yang S-K, et al. , A common missense variant in NUDT15 confers susceptibility to thiopurine-induced leukopenia. Nature genetics, 2014. 46(9): p. 1017.  [DOI](https://doi.org/10.1038/ng.3060) | [PMC free article](/articles/PMC4999337/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25108385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=A%20common%20missense%20variant%20in%20NUDT15%20confers%20susceptibility%20to%20thiopurine-induced%20leukopenia&author=S-K%20Yang&volume=46&issue=9&publication_year=2014&pages=1017&pmid=25108385&doi=10.1038/ng.3060&)

28. Moriyama T, et al. , Mechanisms of NT5C2-mediated thiopurine resistance in acute lymphoblastic leukemia. Molecular cancer therapeutics, 2019. 18(10): p. 1887–1895.  [DOI](https://doi.org/10.1158/1535-7163.MCT-18-1112) | [PMC free article](/articles/PMC6774896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31358663/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Molecular%20cancer%20therapeutics&title=Mechanisms%20of%20NT5C2-mediated%20thiopurine%20resistance%20in%20acute%20lymphoblastic%20leukemia&author=T%20Moriyama&volume=18&issue=10&publication_year=2019&pages=1887-1895&pmid=31358663&doi=10.1158/1535-7163.MCT-18-1112&)

29. Dieck CL, et al. , Structure and mechanisms of NT5C2 mutations driving thiopurine resistance in relapsed lymphoblastic leukemia. Cancer Cell, 2018. 34(1): p. 136–147.e6.  [DOI](https://doi.org/10.1016/j.ccell.2018.06.003) | [PMC free article](/articles/PMC6049837/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29990496/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&title=Structure%20and%20mechanisms%20of%20NT5C2%20mutations%20driving%20thiopurine%20resistance%20in%20relapsed%20lymphoblastic%20leukemia&author=CL%20Dieck&volume=34&issue=1&publication_year=2018&pages=136-147.e6&pmid=29990496&doi=10.1016/j.ccell.2018.06.003&)

30. Meyer JA, et al. , Relapse-specific mutations in NT5C2 in childhood acute lymphoblastic leukemia. Nature genetics, 2013. 45(3): p. 290.  [DOI](https://doi.org/10.1038/ng.2558) | [PMC free article](/articles/PMC3681285/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23377183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Relapse-specific%20mutations%20in%20NT5C2%20in%20childhood%20acute%20lymphoblastic%20leukemia&author=JA%20Meyer&volume=45&issue=3&publication_year=2013&pages=290&pmid=23377183&doi=10.1038/ng.2558&)

31. Tzoneva G, et al. , Clonal evolution mechanisms in NT5C2 mutant-relapsed acute lymphoblastic leukaemia. Nature, 2018. 553(7689): p. 511–514.  [DOI](https://doi.org/10.1038/nature25186) | [PMC free article](/articles/PMC5931372/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29342136/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Clonal%20evolution%20mechanisms%20in%20NT5C2%20mutant-relapsed%20acute%20lymphoblastic%20leukaemia&author=G%20Tzoneva&volume=553&issue=7689&publication_year=2018&pages=511-514&pmid=29342136&doi=10.1038/nature25186&)

32. Tzoneva G, et al. , Activating mutations in the NT5C2 nucleotidase gene drive chemotherapy resistance in relapsed ALL. Nature medicine, 2013. 19(3): p. 368–371.  [DOI](https://doi.org/10.1038/nm.3078) | [PMC free article](/articles/PMC3594483/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23377281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20medicine&title=Activating%20mutations%20in%20the%20NT5C2%20nucleotidase%20gene%20drive%20chemotherapy%20resistance%20in%20relapsed%20ALL&author=G%20Tzoneva&volume=19&issue=3&publication_year=2013&pages=368-371&pmid=23377281&doi=10.1038/nm.3078&)

33. Barz MJ, et al. , Subclonal NT5C2 mutations are associated with poor outcomes after relapse of pediatric acute lymphoblastic leukemia. Blood, The Journal of the American Society of Hematology, 2020. 135(12): p. 921–933.  [DOI](https://doi.org/10.1182/blood.2019002499) | [PMC free article](/articles/PMC7218751/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31971569/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood,%20The%20Journal%20of%20the%20American%20Society%20of%20Hematology&title=Subclonal%20NT5C2%20mutations%20are%20associated%20with%20poor%20outcomes%20after%20relapse%20of%20pediatric%20acute%20lymphoblastic%20leukemia&author=MJ%20Barz&volume=135&issue=12&publication_year=2020&pages=921-933&pmid=31971569&doi=10.1182/blood.2019002499&)

34. Tulstrup M, et al. , NT5C2 germline variants alter thiopurine metabolism and are associated with acquired NT5C2 relapse mutations in childhood acute lymphoblastic leukaemia. Leukemia, 2018. 32(12): p. 2527–2535.  [DOI](https://doi.org/10.1038/s41375-018-0245-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30201983/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=NT5C2%20germline%20variants%20alter%20thiopurine%20metabolism%20and%20are%20associated%20with%20acquired%20NT5C2%20relapse%20mutations%20in%20childhood%20acute%20lymphoblastic%20leukaemia&author=M%20Tulstrup&volume=32&issue=12&publication_year=2018&pages=2527-2535&pmid=30201983&doi=10.1038/s41375-018-0245-3&)

35. Pui C-H, et al. , Treating childhood acute lymphoblastic leukemia without cranial irradiation. New England Journal of Medicine, 2009. 360(26): p. 2730–2741.  [DOI](https://doi.org/10.1056/NEJMoa0900386) | [PMC free article](/articles/PMC2754320/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19553647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&title=Treating%20childhood%20acute%20lymphoblastic%20leukemia%20without%20cranial%20irradiation&author=C-H%20Pui&volume=360&issue=26&publication_year=2009&pages=2730-2741&pmid=19553647&doi=10.1056/NEJMoa0900386&)

36. Su Y, et al. , Assay of 6-mercaptopurine and its metabolites in patient plasma by high-performance liquid chromatography with diode-array detection. Journal of Chromatography B: Biomedical Sciences and Applications, 1999. 732(2): p. 459–468.  [DOI](https://doi.org/10.1016/s0378-4347(99)00311-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10517368/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Chromatography%20B:%20Biomedical%20Sciences%20and%20Applications&title=Assay%20of%206-mercaptopurine%20and%20its%20metabolites%20in%20patient%20plasma%20by%20high-performance%20liquid%20chromatography%20with%20diode-array%20detection&author=Y%20Su&volume=732&issue=2&publication_year=1999&pages=459-468&pmid=10517368&doi=10.1016/s0378-4347(99)00311-4&)

37. Dervieux T, et al. , HPLC determination of thiopurine nucleosides and nucleotides in vivo in lymphoblasts following mercaptopurine therapy. Clinical chemistry, 2002. 48(1): p. 61–68.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/11751539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20chemistry&title=HPLC%20determination%20of%20thiopurine%20nucleosides%20and%20nucleotides%20in%20vivo%20in%20lymphoblasts%20following%20mercaptopurine%20therapy&author=T%20Dervieux&volume=48&issue=1&publication_year=2002&pages=61-68&pmid=11751539&)

38. Churchman ML, et al. , Germline genetic IKZF1 variation and predisposition to childhood acute lymphoblastic leukemia. Cancer cell, 2018. 33(5): p. 937–948.e8.  [DOI](https://doi.org/10.1016/j.ccell.2018.03.021) | [PMC free article](/articles/PMC5953820/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29681510/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20cell&title=Germline%20genetic%20IKZF1%20variation%20and%20predisposition%20to%20childhood%20acute%20lymphoblastic%20leukemia&author=ML%20Churchman&volume=33&issue=5&publication_year=2018&pages=937-948.e8&pmid=29681510&doi=10.1016/j.ccell.2018.03.021&)

39. Moriyama T, et al. , Germline genetic variation in ETV6 and risk of childhood acute lymphoblastic leukaemia: a systematic genetic study. The Lancet Oncology, 2015. 16(16): p. 1659–1666.  [DOI](https://doi.org/10.1016/S1470-2045(15)00369-1) | [PMC free article](/articles/PMC4684709/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26522332/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Lancet%20Oncology&title=Germline%20genetic%20variation%20in%20ETV6%20and%20risk%20of%20childhood%20acute%20lymphoblastic%20leukaemia:%20a%20systematic%20genetic%20study&author=T%20Moriyama&volume=16&issue=16&publication_year=2015&pages=1659-1666&pmid=26522332&doi=10.1016/S1470-2045(15)00369-1&)

40. Qian M, et al. , TP53 germline variations influence the predisposition and prognosis of B-cell acute lymphoblastic leukemia in children. Journal of Clinical Oncology, 2018. 36(6): p. 591.  [DOI](https://doi.org/10.1200/JCO.2017.75.5215) | [PMC free article](/articles/PMC5815403/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29300620/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Oncology&title=TP53%20germline%20variations%20influence%20the%20predisposition%20and%20prognosis%20of%20B-cell%20acute%20lymphoblastic%20leukemia%20in%20children&author=M%20Qian&volume=36&issue=6&publication_year=2018&pages=591&pmid=29300620&doi=10.1200/JCO.2017.75.5215&)

41. Qian M, et al. , Genome-wide association study of susceptibility loci for T-cell acute lymphoblastic leukemia in children. JNCI: Journal of the National Cancer Institute, 2019. 111(12): p. 1350–1357.  [DOI](https://doi.org/10.1093/jnci/djz043) | [PMC free article](/articles/PMC6910193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30938820/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JNCI:%20Journal%20of%20the%20National%20Cancer%20Institute&title=Genome-wide%20association%20study%20of%20susceptibility%20loci%20for%20T-cell%20acute%20lymphoblastic%20leukemia%20in%20children&author=M%20Qian&volume=111&issue=12&publication_year=2019&pages=1350-1357&pmid=30938820&doi=10.1093/jnci/djz043&)

42. Corces MR, et al. , Lineage-specific and single-cell chromatin accessibility charts human hematopoiesis and leukemia evolution. Nature genetics, 2016. 48(10): p. 1193–1203.  [DOI](https://doi.org/10.1038/ng.3646) | [PMC free article](/articles/PMC5042844/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27526324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20genetics&title=Lineage-specific%20and%20single-cell%20chromatin%20accessibility%20charts%20human%20hematopoiesis%20and%20leukemia%20evolution&author=MR%20Corces&volume=48&issue=10&publication_year=2016&pages=1193-1203&pmid=27526324&doi=10.1038/ng.3646&)

43. Consortium EP, An integrated encyclopedia of DNA elements in the human genome. Nature, 2012. 489(7414): p. 57–74.  [DOI](https://doi.org/10.1038/nature11247) | [PMC free article](/articles/PMC3439153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955616/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=An%20integrated%20encyclopedia%20of%20DNA%20elements%20in%20the%20human%20genome&author=EP%20Consortium&volume=489&issue=7414&publication_year=2012&pages=57-74&pmid=22955616&doi=10.1038/nature11247&)

44. Davis CA, et al. , The Encyclopedia of DNA elements (ENCODE): data portal update. Nucleic acids research, 2018. 46(D1): p. D794–D801.  [DOI](https://doi.org/10.1093/nar/gkx1081) | [PMC free article](/articles/PMC5753278/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29126249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20acids%20research&title=The%20Encyclopedia%20of%20DNA%20elements%20(ENCODE):%20data%20portal%20update&author=CA%20Davis&volume=46&issue=D1&publication_year=2018&pages=D794-D801&pmid=29126249&doi=10.1093/nar/gkx1081&)

45. Karolchik D, et al. , The UCSC Table Browser data retrieval tool. Nucleic acids research, 2004. 32(suppl_1): p. D493–D496.  [DOI](https://doi.org/10.1093/nar/gkh103) | [PMC free article](/articles/PMC308837/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14681465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20acids%20research&title=The%20UCSC%20Table%20Browser%20data%20retrieval%20tool&author=D%20Karolchik&volume=32&issue=suppl_1&publication_year=2004&pages=D493-D496&pmid=14681465&doi=10.1093/nar/gkh103&)

46. Kent WJ, et al. , The human genome browser at UCSC. Genome research, 2002. 12(6): p. 996–1006.  [DOI](https://doi.org/10.1101/gr.229102) | [PMC free article](/articles/PMC186604/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12045153/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20research&title=The%20human%20genome%20browser%20at%20UCSC&author=WJ%20Kent&volume=12&issue=6&publication_year=2002&pages=996-1006&pmid=12045153&doi=10.1101/gr.229102&)

47. Rosenbloom KR, et al. , ENCODE data in the UCSC Genome Browser: year 5 update. Nucleic acids research, 2012. 41(D1): p. D56–D63.  [DOI](https://doi.org/10.1093/nar/gks1172) | [PMC free article](/articles/PMC3531152/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23193274/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20acids%20research&title=ENCODE%20data%20in%20the%20UCSC%20Genome%20Browser:%20year%205%20update&author=KR%20Rosenbloom&volume=41&issue=D1&publication_year=2012&pages=D56-D63&pmid=23193274&doi=10.1093/nar/gks1172&)

48. Gazziola C, et al. , Cytosolic high K m 5′-nucleotidase and 5′(3′)-deoxyribonucleotidase in substrate cycles involved in nucleotide metabolism. Journal of Biological Chemistry, 2001. 276(9): p. 6185–6190.  [DOI](https://doi.org/10.1074/jbc.M007623200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11083867/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&title=Cytosolic%20high%20K%20m%205%E2%80%B2-nucleotidase%20and%205%E2%80%B2(3%E2%80%B2)-deoxyribonucleotidase%20in%20substrate%20cycles%20involved%20in%20nucleotide%20metabolism&author=C%20Gazziola&volume=276&issue=9&publication_year=2001&pages=6185-6190&pmid=11083867&doi=10.1074/jbc.M007623200&)

49. Spychała J, Madrid-Marina V, and Fox I, High Km soluble 5′-nucleotidase from human placenta. Properties and allosteric regulation by IMP and ATP. Journal of Biological Chemistry, 1988. 263(35): p. 18759–18765.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2848805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&title=High%20Km%20soluble%205%E2%80%B2-nucleotidase%20from%20human%20placenta.%20Properties%20and%20allosteric%20regulation%20by%20IMP%20and%20ATP&author=J%20Spycha%C5%82a&author=V%20Madrid-Marina&author=I%20Fox&volume=263&issue=35&publication_year=1988&pages=18759-18765&pmid=2848805&)
