# Genome wide association study identifies the common variants in CYP3A4 and CYP3A5 responsible for variation in tacrolimus trough concentration in Caucasian kidney transplant recipients

## Metadata
**Authors:** William S Oetting, Baolin Wu, David P Schladt, Weihua Guan, Rory P Remmel, Roslyn B Mannon, Arthur J Matas, Ajay K Israni, DeKAF Genomics Investigators
**Journal:** The pharmacogenomics journal
**Date:** 2017 Nov 21
**DOI:** [10.1038/tpj.2017.49](https://doi.org/10.1038/tpj.2017.49)
**PMID:** 29160300
**PMCID:** PMC8513493
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8513493/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8513493/pdf/nihms-1742125.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8513493/pdf/nihms-1742125.pdf)

## Abstract

The immunosuppressant tacrolimus (TAC) is metabolized by both cytochrome P450 3A4 (CYP3A4) and CYP3A5 enzymes. It is common for European-Americans (EA), to carry two CYP3A5 loss-of-function (LoF) variants, which profoundly reduces TAC metabolism. Despite having two LoF alleles, there is still considerable variability in TAC troughs and identifying additional variants in genes outside of the CYP3A5 gene could provide insight into this variability. We analyzed TAC trough concentrations in 1,345 adult EA recipients with two CYP3A5 LoF alleles in a genome wide association study. Only CYP3A4*22 was identified and no additional variants were genome wide significant. Additional high allele frequency genetic variants with strong genetic effects associated with TAC trough variability are unlikely to be associated with TAC variation in the EA population. These data suggest that low allele frequency variants, identified by DNA sequencing, should be evaluated and may identify additional variants that contribute to TAC pharmacokinetic variability.

## INTRODUCTION

Tacrolimus (TAC) is a common immunosuppressive agent which potently inhibits T-cell function and proliferation through inhibition of calcineurin by binding to FKB12. It is primarily metabolized by the enzymes cytochrome P450 (CYP) 3A4 and CYP3A5, whereas CYP3A7 (expressed in fetal liver) does not significantly contribute to the oxidative metabolism of tacrolimus ([1](#R1), [2](#R2)). The *in vitro* TAC clearances (Km/Vmax) for TAC 13-O-demethylation (major metabolite) by the two enzymes are similar (0.48 μL/min/pmol P450 for CYP3A4 vs. 0.78 μL/min/pmol P450 for CYP3A5) ([3](#R3), [4](#R4)). We and others have shown that interpatient variation in TAC troughs is greatly affected by genetic variants in *CYP3A5* ([5](#R5)–[8](#R8)). The most common allele in the European American (EA) population is *CYP3A5*3* (rs776746), a loss-of-function (LoF) variant which creates a cryptic splice site, resulting in additional coding sequence being added to the protein product ([9](#R9)). The allele frequency for the non-functional **3* allele is approximately 0.95 in the EA population and ranges from 0.15 to 0.35 in other non-EA populations ([4](#R4), [10](#R10)). There are two additional *CYP3A5* LoF alleles which have been identified in the EA population but with significantly lower frequencies (MAF < 1%), *CYP3A5*6* (rs10264272) and *CYP3A5*7* (rs41303343) ([10](#R10), [11](#R11)). These alleles are fairly common in Africans (MAF = 0.154 and 0.118 respectively). The *CYP3A5*6* variant alters a splice site resulting in alternative splicing and the deletion of exon 7 producing an inactive enzyme and the *CYP3A5*7* variant is a frameshift mutation resulting in a truncated protein also producing a non-functional enzyme. Therefore, a complete loss of CYP3A5 function is common in EA recipients and TAC metabolism occurs primarily through the CYP3A4 enzyme.

Obtaining optimal concentrations of TAC in the blood of kidney allograft recipients is critical to reducing the risk of acute rejection (AR) and maximizing graft survival, while also reducing the risk of TAC-related toxicity. Unfortunately, there is a high degree of pharmacokinetic variability and dose requirements vary widely between individuals. This variability has significant consequences in the case of lower TAC trough concentrations in whole blood, which has been associated with increased risk for AR ([12](#R12)–[15](#R15)).

To identify additional common genetic variants beyond those in the *CYP3A5* gene, we analyzed EA kidney transplant recipients who had two *CYP3A5* LoF alleles. Analyzing variation of TAC troughs among these recipients allowed for the identification of additional variants in other genes without interference by the very large effect provided by common *CYP3A5* LoF variants. A genome-wide association study (GWAS), containing a panel of common single nucleotide polymorphisms (SNPs) designed specifically for transplantation, including candidate LoF alleles thought to be important in transplant outcomes, was analyzed to identify additional genetic variants ([16](#R16)).

## MATERIALS and METHODS

### Study Design and Population

All subjects were enrolled in the DeKAF genomics study ([14](#R14)). The study design and cohort characteristics have been previously reported ([8](#R8), [17](#R17), [18](#R18)). For this analysis, we selected only self-reported European-American kidney (EA) allograft recipients. A total of 1,446 EA kidney transplant recipients were included in this study. Subjects were 18 years and older, received TAC for maintenance immunosuppression and had TAC trough blood concentrations available in the first 6 months post-transplant. This study was registered at [www.clinicaltrials.gov](http://www.clinicaltrials.gov/) ([NCT01714440](https://clinicaltrials.gov/ct2/show/NCT01714440)). Subjects were enrolled and provided written consent at time of transplant. Informed consents were approved by the Institutional Review Boards at each of the enrolling centers.

Participants received oral immediate release TAC with mycophenolate maintenance with varying durations of steroid per transplant center standard-of-care protocols. Induction therapy was administered as per transplant center preference but mainly consisted of rabbit antithymocyte globulin (rATG), basiliximab or Campath-1H. Immunologically high risk patients were more likely to receive rATG, such as those with donor-specific antibody, prior pregnancies, or repeat transplants. Donor and recipient characteristics, such as race, serum creatinine (SCr), estimated glomerular filtration rate (GFR) and concomitant medications were obtained from the respective medical records. All TAC trough whole blood measurements were clinically measured at each site and were analyzed in a CLIA approved laboratory with >95% measured by liquid chromatography-mass spectrometry. When available, two concentrations were obtained in the first 8 weeks and two concentrations per month in months 3, 4, 5 and 6 for a maximum of 24 trough concentrations per patient. TAC doses were adjusted by the transplant center based on trough concentrations to reach institution-specific trough goals based on time post-transplant (generally 8–12 ng/mL in months 0 to 3 and 6–10 ng/mL in months 4 to 6). Dose was also adjusted for toxicity (*e.g*. nephrotoxicity attributed to TAC) by center specific preferences. A total of 25,255 TAC trough concentrations from 1,446 recipients were analyzed.

### GWAS Genotyping

Pre-transplant recipient blood was collected at the transplant centers at time of transplant and DNA was isolated at a central laboratory at the University of Minnesota. Lymphocytes were isolated by centrifugation after red blood cell lysis and the DNA isolated. Genotyping was performed on a custom exome-plus Affymetrix TxArray SNP chip ([16](#R16)). This chip contains approximately 782,000 markers including pharmacogenomic SNPs, 168,000 exonic or coding variants and over 16,000 putative LoF variants. The GWAS analysis was performed on recipients who carried two of any combination of three common *CYP3A5* LoF alleles (*CYP3A5*3*, rs776746; *CYP3A5*6*, rs10264272; or *CYP3A5*7*, rs41303343). The *CYP3A5* genotypes were taken from the SNP chip and the remaining SNPs were used for the GWAS analysis. SNPs that diverged from Hardy Weinberg equilibrium (HWE) were excluded (p < e^−3^). Additional SNPs for analysis were identified through imputation (see below).

### GWAS Genotyping Data Quality Control

Genotyping was conducted as previously described ([8](#R8)). Data quality control was carried out with PLINK software (version 1.90b1a) ([19](#R19)). Genotypes were subjected to a 95% call rate threshold. Samples with very high heterozygosity and suspected contamination were re-assayed and removed if high heterozygosity could not be resolved. Unrelated samples with pairwise identity by descent (IBD) > 0.3 were excluded from the study. Individual SNPs were excluded if they were monomorphic. The final number of variants analyzed from the SNP chip was 644,224.

### Imputation of the GWAS to 1000 Genomes

Imputation to the 1000 Genomes reference population was conducted on the GWAS SNPs and performed in two steps: (1) pre-phasing with ShapeIt2 (v2.r644) ([20](#R20)); and (2) imputation with IMPUTE2 ([21](#R21)). Measured SNPs used for imputation were restricted to those with a >99% call rate and HWE test with p < e^−3^. SNPs were removed if they had either low certainty due to imputation errors (imputation info statistic <0.9) or low minor allele frequency (MAF < 0.01). After frequency and genotyping pruning, there were 577,084 SNPs in the final set used for the imputation. The final number of genotyped and imputed SNPs used in the analysis was 1,221,308.

### Statistical Analysis

We fit a linear mixed effects model (LMM) to test for the association between SNP and the log transformed dose-normalized TAC trough concentrations. A log transformation was used to ensure that the dose-normalized TAC concentrations were normally distributed. Visual inspection showed that that dose-normalized trough concentrations initially started low, rose quickly until day 9 post-transplant and then plateaued in the early weeks post-transplant ([22](#R22), [23](#R23)). Therefore, a simple spline method was used to model the effect of time on all trough concentrations, with the change in slope occurring at day 9. The longitudinal LMM included a random intercept and random slopes for days post-transplant and adjusted for age, gender, and transplant center. To adjust for the influence of population stratification and global admixture, the first principal component from the GWAS genotypes were incorporated as covariates in the regression models. SNP genotypes were coded as an additive model. Genome-wide significance was declared with association P-value < 5e^−8^.

To determine variation of TAC troughs attributable to clinical and genetic variables, we began by creating a LMM with clinical variables alone on the cohort of EA patients with 0, 1, or 2 LoF alleles. Backward selection with a retention p-value of 0.10 was performed on the following variables (starred variables were retained): transplant center*, donor age and gender*, donor type (living or deceased)*, and recipient factors such as age*, gender, weight at baseline*, diabetes at baseline*, antibody induction* and SPK transplantation. Time-varying covariates considered for the backward selection, defined at each TAC trough observation, were: closest eGFR to TAC trough*, steroid use*, calcium channel blocker use*, ACE inhibitor use*, and antiviral drug use*. Analyses were conducted with SAS version 9.4 (SAS Institute, Cary, NC) and R software version 3.3.

## RESULTS

Characteristics of the cohort are shown in [Table 1](#T1) for both the entire EA recipient cohort and for those EA recipients with two CYP3A5 LoF alleles. There were no statistically different clinical variables between the two cohorts. After excluding recipients with less than 2 LoF alleles, there were 1,345 EA kidney transplant recipients used in the GWAS analysis: 1,342 were *CYP3A5*3/*3*, two were *CYP3A5*3/*6*, and one was *CYP3A5*3/*7*.

### Table 1.

|   | 0, 1, or 2 LoF (N=1,446) | 2 LoF (N=1,345) |
| --- | --- | --- |
| Age group (yrs), n (%) |   |   |
| 18–34 | 165 (11.4%) | 151 (11.2%) |
| 35–64 | 1048 (72.5%) | 971 (72.2%) |
| 65–84 | 233 (16.1%) | 223 (16.6%) |
| Donor age group (yrs), n (%) |   |   |
| 0–34 | 454 (31.4%) | 415 (30.9%) |
| 35–64 | 954 (66.0%) | 891 (66.3%) |
| 65–84 | 38 (2.6%) | 39 (2.9%) |
| Living donor status, n (%) | 961 (66.5%) | 884 (65.7%) |
| Female, n (%) | 538 (37.2%) | 505 (37.6%) |
| Diabetes at transplant, n (%) | 564 (39.0%) | 525 (39.1%) |
| SPK, n (%) | 120 (8.3%) | 114 (8.5%) |
| Body Mass Index, mean (SD) | 28.3 (5.5) | 28.3 (5.6) |
| Antibody Induction, n (%) |   |   |
| Combination | 42 (2.9%) | 42 (3.1%) |
| Monoclonal | 554 (38.3%) | 516 (38.4%) |
| Polyclonal | 792 (54.8%) | 730 (54.3%) |
| None | 58 (4.0%) | 57 (4.2%) |
| Median tacrolimus trough in ng/mL (IQR) in first 6 months | 8.4 (6.6–10.3) | 8.4 (6.7–10.4) |
| Median daily tacrolimus dose in mg (IQR) in first 6 months | 5.5 (4.0–8.0) | 5.0 (3.5–7.0) |
| Median dose-normalized tacrolimus trough (IQR) in ng/mL per total daily dose in mg in first 6 months | 1.52 (1.01–2.33) | 1.64 (1.13–2.47) |

Table 1 Caption: Recipient Characteristics for kidney allograft recipients with either two CYP3A5 LoF alleles for with 0, 1 or 2 CYP3A5 LoF alleles

Genome wide significant SNPs were identified after adjusting for multiple testing (p-value < 5e^−8^). The Manhattan plot revealed that the most significant SNPs were located on chromosome 7 ([Figure 1](#F1)). Genome wide significant SNPs are shown in [Table 2](#T2). The Q-Q plot is shown in [Figure 1s](#SD1) in the [supplementary data](#SD1). The most significant SNP, rs35599367 (p=2.21e^−17^), is a previously described reduced functional variant within the *CYP3A4* gene (*CYP3A4*22*) ([24](#R24)). The minor allele frequency for the *CYP3A4*22* variant in our population was 0.054. The distribution of the dose normalized TAC concentration for all three genotypes is shown in [Figure 2S](#SD1). Eleven additional SNPs within the *CYP3A* locus region on chromosome 7 were also significant. After adjusting for the top SNP rs35599367 (*CYP3A4*22*) all other SNPs were below the threshold of genome-wide statistical significance. The Manhattan plot ([Figure 3S](#SD1)) and the Q-Q plot ([Figure 4S](#SD1)) for the adjusted analysis are shown in the [supplementary data](#SD1).

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/99b9/8513493/9562ce5dc87d/nihms-1742125-f0001.jpg)

Manhattan plot for dose-normalized tacrolimus troughs in European-American recipients with 2 CYP3A5 LoF alleles.

### Table 2.

| Chr | SNP | Position | Gene* | A1 | beta | SE | P-value |
| --- | --- | --- | --- | --- | --- | --- | --- |
| 7 | rs35599367 | 99366316 | CYP3A4 | A | 0.300 | 0.035 | 2.21e−17 |
| 7 | rs62471957 | 99440105 | CYP3A43 | G | 0.294 | 0.035 | 2.64e−17 |
| 7 | rs62471929 | 99274316 | CYP3A43 | G | 0.285 | 0.034 | 7.20e−17 |
| 7 | rs78912778 | 99133763 | ZKSCAN5 | T | 0.282 | 0.034 | 1.16e−16 |
| 7 | rs62471956 | 99421085 | CYP3A43 | A | 0.307 | 0.037 | 2.18e−16 |
| 7 | rs45459197 | 99054663 | CPSF4 | T | 0.278 | 0.034 | 3.22e−16 |
| 7 | rs62474460 | 99463342 | CYP3A43 | C | 0.295 | 0.036 | 4.32e−16 |
| 7 | rs74516408 | 99060440 | ATP5J2-PTCD1 | T | 0.275 | 0.034 | 5.81e−16 |
| 7 | rs150746244 | 99068756 | ZNF789 | T | 0.276 | 0.034 | 8.61e−16 |
| 7 | rs4986910 | 99358524 | CYP3A4 | G | 0.592 | 0.085 | 2.85e−12 |
| 7 | rs7792939 | 99207876 | GS1-259H13.2 | C | 0.172 | 0.025 | 4.35e−12 |
| 7 | rs2572004 | 99500512 | TRIM4 | A | 0.452 | 0.069 | 5.26e−11 |

Table 2 Caption: Significant SNPs (p<1e−8) and effect sizes associated with dose normalized tacrolimus troughs in recipients with two CYP3A5 LoF alleles.

We then determined the contribution of clinical variants and genetic variants to TAC trough variation. Determination of the contribution of *CYP3A5* variants to the variation in TAC troughs cannot be done in individuals with 2 LoF *CYP3A5* alleles since there is no CYP3A5 activity present. For this analysis the entire EA cohort, from which our two *CYP3A5* LoF allele recipients were derived, was used. A model with just clinical covariates explained 24.1% of variance in TAC troughs ([Table 3](#T3)). A model with clinical covariates and *CYP3A5*3* (rs776746) explained 36.6% of variance. A model with clinical covariates, *CYP3A5*3*, and *CYP3A4*22* (rs35599367) explained 40.1% of the variance, leaving 59.9% of the variance in TAC troughs still unexplained. *CYP3A5*3* and *CYP3A4*22* together explained a 16% of the total variance.

### Table 3.

| Model | Variation of TAC troughs* | Variation explained by model** |
| --- | --- | --- |
| Simple time-trend model | 0.361 | - |
| Clinical variables | 0.274 | 24.1% |
| Clinical variables + rs776746 | 0.229 | 36.6% |
| Clinical variables + rs35599367 | 0.258 | 28.5% |
| Clinical variables + rs776746 and rs35599367 | 0.216 | 40.1% |

Table 3 Caption: Variation attributed to clinical variables and genetic variants in subjects with 0, 1, or 2 CYP3A5 LoFs

## DISCUSSION

In this EA cohort of 1,342 kidney allograft recipients having two CYP3A5 LoF alleles, a common variant in the *CYP3A4* gene (*CYP3A4*22*, rs35599367), with a MAF of 0.054, was identified and significantly contributed to the total variation in TAC troughs. This variant has been previously reported to be associated with variation in TAC troughs in European recipients ([24](#R24), [26](#R26)–[28](#R28)). We previously reported in a multi-race analysis of *CYP3A5* LoF carriers and non-carriers that *CYP3A4*22* was not associated with TAC troughs ([7](#R7)). However, when the analysis is limited to only EA recipients, *CYP3A4*22* was identified as significant. Given our sample size in the EA cohort, a power analysis showed 88% power to detect a genetic variant that can explain 3% variance of TAC, at significance level of 1e^−7^. If there are any additional common variants underlying TAC metabolism, they would most likely have low effect sizes.

Surprisingly, many variants have been previously reported to be associated with TAC trough variation. Damon et al., analyzing 229 kidney allograft recipients, identified variants in the ATP-binding cassette, subfamily C, member 8 (*ABCC8*) and the solute carrier family 28, member 3 (*SLC28A3*) genes, but variants in these genes were not significant in our study ([29](#R29)). Additional variants in candidate genes have been reported to be associated with TAC troughs including the ATP-binding cassette, subfamily B, member 1 (*ABCB1*) gene, the cytochrome P450 oxidoreductase (*POR*) gene, the cytochrome P450 subfamily 2C, polypeptide 19 (*CYP2C19*) gene and the peroxisome proliferator-activated receptor-alpha (*PPARA*) gene, but none of these were found to be significant in our cohort ([30](#R30)–[38](#R38)).

When taking into consideration the common variants we identified, there is still significant variance in TAC troughs that has yet to be explained in recipients with two *CYP3A5* LoF alleles. In this cohort, we can account for 40.1% of the total TAC variance. In the paper by Damon et al ([29](#R29)), SNPs explained up to 70% of inter-patient variability of TAC metabolism in their population. There are several key differences between our study and Damon et al. First, Damon et al. included up to 85 genetic variants in their predictive model (most not found significant in our population), whereas we used only those SNPs which were genome-wide significant. Second, the R^2^ (% variance explained) value is prone to overestimation in small samples and large number of predictors. Our cohort has a much larger sample size than Damon et al., and allows us to obtain more accurate estimation of the R^2^.

Our data show that there is still a significant percentage of variation that is still unexplained. Some of this variation may be attributed to clinical factors, such as age, time posttransplant, drug drug interactions or medication adherence, which we did not include in our model. An additional source of genetic variation may lie in rare functional variants (MAF < 1%). These variants could contribute to TAC trough variability but because of their low MAF they would not be identified through a GWAS analysis. There may be many rare LoF alleles in *CYP3A4* as well as other candidate genes encoding proteins that influence tacrolimus metabolism or transport. Identifying these variants could help us predict a greater percentage of the TAC trough variance for individual tacrolimus dosing, as well as pointing to additional genes potentially associated with outcomes after transplantation. Alternatively, epigenetic and environmental factors may contribute to variability in the expression of CYP3A4, since protein expression has been shown to vary between samples in liver microsomal banks ([42](#R42), [43](#R43)).

To introduce pharmacogenetics into clinical practice, we have developed and tested genetic dosing models which will move pharmacogenomics one step closer to the clinic ([39](#R39), [40](#R40), [41](#R41)). These models will need to be continually refined and improved as additional genetic and clinical factors associated with TAC disposition are identified. Identifying determinants of efficacy and toxicity of all immunosuppressants is critical to improving precision immunosuppression and ultimately allograft survival. Understanding the effects of many genes will someday improve other areas of transplantation including donor selection (*e.g.* APOL1) and rejection (*e.g.* kidney biopsy gene expression).

### DeKAF GENOMICS INVESTIGATORS

Arthur J. Matas, MD, Department of Surgery, University of Minnesota, Minneapolis, MN; J. Michael Cecka, MD, UCLA Immunogenetics Center, Los Angeles, CA; John E. Connett, PhD, Division of Biostatistics, University of Minnesota, Minneapolis, MN; Fernando G. Cosio, MD, Division of Nephrology, Mayo Clinic, Rochester, MN; Robert S. Gaston, MD, Division of Nephrology, University of Alabama, Birmingham, AL; Rosalyn B. Mannon, MD, Division of Nephrology, University of Alabama, Birmingham, AL; Sita Gourishankar,MD, Division of Nephrology and Immunology, University of Alberta, Edmonton, Alberta, Canada; Joseph P. Grande, MD, PhD, Mayo Clinic College of Medicine, Rochester, MN; Lawrence G. Hunsicker, MD, Nephrology Division, Iowa City, IA; Bertram L. Kasiske, MD, Division of Nephrology, Hennepin County Medical Center, Minneapolis, MN; and David N. Rush, MD, Health Sciences Center, Winnipeg MB, Canada.

## Supplementary Material

## ACKNOWLEDGMENTS

The authors wish to thank the research subjects for their participation in this study. We acknowledge the dedication and hard work of our coordinators at each of the DeKAF Genomics clinical sites: University of Alberta, Nicoleta Bobocea, Tina Wong, Adrian Geambasu and Alyssa Sader; University of Manitoba, Myrna Ross and Kathy Peters; University of Minnesota, Mandi DeGrote, Monica Meyers and Danielle Berglund; Hennepin County Medical Center, Lisa Berndt; Mayo Clinic, Tom DeLeeuw; University of Iowa, Wendy Wallace and Tammy Lowe; University of Alabama, Jacquelin Vaughn, Valencia Stephens and Tena Hilario. We also acknowledge the dedicated work of our research scientists Marcia Brott and Amutha Muthusamy. This study was supported in part by NIH/NIAID grants 5U19-AI070119 and 5U01-AI058013.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Iwasaki K Metabolism of tacrolimus (FK506) and recent topics in clinical pharmacokinetics. Drug Metab Pharmacokinet 2007; 22: 328–335.  [DOI](https://doi.org/10.2133/dmpk.22.328) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17965516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Pharmacokinet&title=Metabolism%20of%20tacrolimus%20(FK506)%20and%20recent%20topics%20in%20clinical%20pharmacokinetics&author=K%20Iwasaki&volume=22&publication_year=2007&pages=328-335&pmid=17965516&doi=10.2133/dmpk.22.328&)

2. Daly AK. Significance of the minor cytochrome P450 3A isoforms. Clin Pharmacokinet. 2006; 45: 13–31.  [DOI](https://doi.org/10.2165/00003088-200645010-00002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16430309/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Significance%20of%20the%20minor%20cytochrome%20P450%203A%20isoforms&author=AK%20Daly&volume=45&publication_year=2006&pages=13-31&pmid=16430309&doi=10.2165/00003088-200645010-00002&)

3. Kamden LK, Streit F, Zanger UM, Oellerich M, Armstrong VW, Wojnowski L. Contribution of CYP3A5 to the in vitro hepatic clearance of tacrolimus. Clin Chem 2005; 51: 1374–1381.  [DOI](https://doi.org/10.1373/clinchem.2005.050047) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15951320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=Contribution%20of%20CYP3A5%20to%20the%20in%20vitro%20hepatic%20clearance%20of%20tacrolimus&author=LK%20Kamden&author=F%20Streit&author=UM%20Zanger&author=M%20Oellerich&author=VW%20Armstrong&volume=51&publication_year=2005&pages=1374-1381&pmid=15951320&doi=10.1373/clinchem.2005.050047&)

4. Dai Y, Hebert MF, Isoherrannen N, Davis CL. Marsh C, Shen DD, et al. Effect of CYP3A5 polymorphism on tacrolimus metabolic clearance in vitro. Drug Metabolism and Disposition. 2006; 34: 836–847.  [DOI](https://doi.org/10.1124/dmd.105.008680) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16501005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metabolism%20and%20Disposition&title=Effect%20of%20CYP3A5%20polymorphism%20on%20tacrolimus%20metabolic%20clearance%20in%20vitro&author=Y%20Dai&author=MF%20Hebert&author=N%20Isoherrannen&author=CL%20Davis&volume=34&publication_year=2006&pages=836-847&pmid=16501005&doi=10.1124/dmd.105.008680&)

5. Li JL, Liu S, Fu Q, Zhang Y, Wang XD, Liu XM, et al. Interactive effects of CYP3A4, CYP3A5, MDR1 and NR1I2 polymorphisms on tracrolimus trough concentrations in early postrenal transplant recipients. Pharmacogenomics. 2015; 16: 1355–1365.  [DOI](https://doi.org/10.2217/pgs.15.78) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26228923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Interactive%20effects%20of%20CYP3A4,%20CYP3A5,%20MDR1%20and%20NR1I2%20polymorphisms%20on%20tracrolimus%20trough%20concentrations%20in%20early%20postrenal%20transplant%20recipients&author=JL%20Li&author=S%20Liu&author=Q%20Fu&author=Y%20Zhang&author=XD%20Wang&volume=16&publication_year=2015&pages=1355-1365&pmid=26228923&doi=10.2217/pgs.15.78&)

6. Zhang JJ, Liu SB, Xue L, Ding XL, Zhang H, Miao LY. The genetic polymorphisms of POR*28 and CYP3A5*3 significantly influence the pharmacokinetics of tacrolimus in Chinese renal transplant recipients. Int J Clin Pharmacol Ther. 2015; 53: 728–736.  [DOI](https://doi.org/10.5414/CP202152) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26227094/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Clin%20Pharmacol%20Ther&title=The%20genetic%20polymorphisms%20of%20POR*28%20and%20CYP3A5*3%20significantly%20influence%20the%20pharmacokinetics%20of%20tacrolimus%20in%20Chinese%20renal%20transplant%20recipients&author=JJ%20Zhang&author=SB%20Liu&author=L%20Xue&author=XL%20Ding&author=H%20Zhang&volume=53&publication_year=2015&pages=728-736&pmid=26227094&doi=10.5414/CP202152&)

7. Pulk RA, Schladt DS, Oetting WS, Guan W, Israni AK, Matas AJ, et al. Multigene predictors of tacrolimus exposure in kidney transplant recipients. Pharmacogenomics. 2015; 16: 841–854.  [DOI](https://doi.org/10.2217/pgs.15.42) | [PMC free article](/articles/PMC4527535/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26067485/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Multigene%20predictors%20of%20tacrolimus%20exposure%20in%20kidney%20transplant%20recipients&author=RA%20Pulk&author=DS%20Schladt&author=WS%20Oetting&author=W%20Guan&author=AK%20Israni&volume=16&publication_year=2015&pages=841-854&pmid=26067485&doi=10.2217/pgs.15.42&)

8. Oetting WS, Schladt DP, Guan W, Miller MB, Remmel RP, Dorr C, et al. Genomewide association study of tacrolimus concentrations in African American kidney transplant recipients identifies multiple CYP3A5 alleles. Am J Transplant. 2016; 16: 574–582.  [DOI](https://doi.org/10.1111/ajt.13495) | [PMC free article](/articles/PMC4733408/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26485092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Transplant&title=Genomewide%20association%20study%20of%20tacrolimus%20concentrations%20in%20African%20American%20kidney%20transplant%20recipients%20identifies%20multiple%20CYP3A5%20alleles&author=WS%20Oetting&author=DP%20Schladt&author=W%20Guan&author=MB%20Miller&author=RP%20Remmel&volume=16&publication_year=2016&pages=574-582&pmid=26485092&doi=10.1111/ajt.13495&)

9. Schuetz EG, Relling MV, Kishi S, Yang W, Das S, Chen P, et al. PharmGKB update: II. CYP3A5, cytochrome P450, family 3, subfamily A, polypeptide 5. Pharmacol Rev. 2004; 56: 159.  [DOI](https://doi.org/10.1124/pr.56.2.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15169924/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Rev&title=PharmGKB%20update:%20II.%20CYP3A5,%20cytochrome%20P450,%20family%203,%20subfamily%20A,%20polypeptide%205&author=EG%20Schuetz&author=MV%20Relling&author=S%20Kishi&author=W%20Yang&author=S%20Das&volume=56&publication_year=2004&pages=159&pmid=15169924&doi=10.1124/pr.56.2.1&)

10. Kuehl P, Zhang J, Lin Y, Lamba J, Assem M, Schuetz J, et al. Sequence diversity in CYP3A promoters and characterization of the genetic basis of polymorphic CYP3A5 expression. Nature Genet. 2001; 27: 383–391.  [DOI](https://doi.org/10.1038/86882) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11279519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature%20Genet&title=Sequence%20diversity%20in%20CYP3A%20promoters%20and%20characterization%20of%20the%20genetic%20basis%20of%20polymorphic%20CYP3A5%20expression&author=P%20Kuehl&author=J%20Zhang&author=Y%20Lin&author=J%20Lamba&author=M%20Assem&volume=27&publication_year=2001&pages=383-391&pmid=11279519&doi=10.1038/86882&)

11. Hustert E, Haberl M, Burk O, Wolbold R, He YQ, Klein K, et al. The genetic determinants of the CYP3A5 polymorphism. Pharmacogenetics. 2001; 11: 773–779.  [DOI](https://doi.org/10.1097/00008571-200112000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11740341/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=The%20genetic%20determinants%20of%20the%20CYP3A5%20polymorphism&author=E%20Hustert&author=M%20Haberl&author=O%20Burk&author=R%20Wolbold&author=YQ%20He&volume=11&publication_year=2001&pages=773-779&pmid=11740341&doi=10.1097/00008571-200112000-00005&)

12. Neylan JF. Effect of race and immunosuppression in renal transplantation: three-year survival results from a US multicenter, randomized trial. FK506 Kidney Transplant Study Group. Transplant Proc. 1998; 30: 1355–1358.  [DOI](https://doi.org/10.1016/s0041-1345(98)00274-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9636551/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplant%20Proc&title=Effect%20of%20race%20and%20immunosuppression%20in%20renal%20transplantation:%20three-year%20survival%20results%20from%20a%20US%20multicenter,%20randomized%20trial.%20FK506%20Kidney%20Transplant%20Study%20Group&author=JF%20Neylan&volume=30&publication_year=1998&pages=1355-1358&pmid=9636551&doi=10.1016/s0041-1345(98)00274-7&)

13. Jacobson PA, Oetting WS, Brearley AM, Leduc R, Guan W, Schladt D, et al. Novel polymorphisms associated with tacrolimus trough concentrations: results from a multicenter kidney transplant consortium. Transplantation, 2011; 91: 300–308.  [DOI](https://doi.org/10.1097/TP.0b013e318200e991) | [PMC free article](/articles/PMC3579501/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21206424/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplantation&title=Novel%20polymorphisms%20associated%20with%20tacrolimus%20trough%20concentrations:%20results%20from%20a%20multicenter%20kidney%20transplant%20consortium&author=PA%20Jacobson&author=WS%20Oetting&author=AM%20Brearley&author=R%20Leduc&author=W%20Guan&volume=91&publication_year=2011&pages=300-308&pmid=21206424&doi=10.1097/TP.0b013e318200e991&)

14. Israni AK, Riad SM, Leduc R, Oetting WS, Guan W, Schladt D, et al. Tacrolimus trough levels after month 3 as a predictor of acute rejection following kidney transplantation: a lesson learned from DeKAF Genomics. Transplant Int. 2013; 26: 982–989.  [DOI](https://doi.org/10.1111/tri.12155) | [PMC free article](/articles/PMC3787982/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23879408/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplant%20Int&title=Tacrolimus%20trough%20levels%20after%20month%203%20as%20a%20predictor%20of%20acute%20rejection%20following%20kidney%20transplantation:%20a%20lesson%20learned%20from%20DeKAF%20Genomics&author=AK%20Israni&author=SM%20Riad&author=R%20Leduc&author=WS%20Oetting&author=W%20Guan&volume=26&publication_year=2013&pages=982-989&pmid=23879408&doi=10.1111/tri.12155&)

15. Arreola-Guerra JM, Serrano M, Morales-Buenrostro LE, Vilatobá M, Alberú J. Tacrolimus trough levels as a risk factor for acute rejection in renal transplant patients. Ann. Transplant. 2016; 21: 105–114.  [DOI](https://doi.org/10.12659/aot.895104) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26879833/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Transplant&title=Tacrolimus%20trough%20levels%20as%20a%20risk%20factor%20for%20acute%20rejection%20in%20renal%20transplant%20patients&author=JM%20Arreola-Guerra&author=M%20Serrano&author=LE%20Morales-Buenrostro&author=M%20Vilatob%C3%A1&author=J%20Alber%C3%BA&volume=21&publication_year=2016&pages=105-114&pmid=26879833&doi=10.12659/aot.895104&)

16. Li YR, van Setten J, Verma SS, Lu Y, Holmes MV, Gao H, et al. Concept and design of a genome-wide association genotyping array tailored for transplantation-specific studies. Genome Med. 2015; 7: 90.  [DOI](https://doi.org/10.1186/s13073-015-0211-x) | [PMC free article](/articles/PMC4589899/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26423053/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med&title=Concept%20and%20design%20of%20a%20genome-wide%20association%20genotyping%20array%20tailored%20for%20transplantation-specific%20studies&author=YR%20Li&author=J%20van%20Setten&author=SS%20Verma&author=Y%20Lu&author=MV%20Holmes&volume=7&publication_year=2015&pages=90&pmid=26423053&doi=10.1186/s13073-015-0211-x&)

17. Israni A, Leduc R, Holmes J, Jacobson PA, Lamba V, Guan W, et al. Single-nucleotide polymorphisms, acute rejection, and severity of tubulitis in kidney transplantation, accounting for center-to-center variation. Transplantation. 2010; 90: 1401–1408.  [DOI](https://doi.org/10.1097/TP.0b013e3182000085) | [PMC free article](/articles/PMC3594840/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21085059/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplantation&title=Single-nucleotide%20polymorphisms,%20acute%20rejection,%20and%20severity%20of%20tubulitis%20in%20kidney%20transplantation,%20accounting%20for%20center-to-center%20variation&author=A%20Israni&author=R%20Leduc&author=J%20Holmes&author=PA%20Jacobson&author=V%20Lamba&volume=90&publication_year=2010&pages=1401-1408&pmid=21085059&doi=10.1097/TP.0b013e3182000085&)

18. Jacobson PA, Oetting WS, Brearley AM, Leduc R, Guan W, Schladt D, Matas AJ, et al. Novel polymorphisms associated with tacrolimus trough concentrations: results from a multicenter kidney transplant consortium. Transplantation. 2011; 91: 300–308.  [DOI](https://doi.org/10.1097/TP.0b013e318200e991) | [PMC free article](/articles/PMC3579501/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21206424/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplantation&title=Novel%20polymorphisms%20associated%20with%20tacrolimus%20trough%20concentrations:%20results%20from%20a%20multicenter%20kidney%20transplant%20consortium&author=PA%20Jacobson&author=WS%20Oetting&author=AM%20Brearley&author=R%20Leduc&author=W%20Guan&volume=91&publication_year=2011&pages=300-308&pmid=21206424&doi=10.1097/TP.0b013e318200e991&)

19. Purcell S, Neale B, Todd-Brown K, Thomas L, Ferreira MA, Bender D, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet, 2007; 81: 559–575.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&author=L%20Thomas&author=MA%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

20. Delaneau O, Howie B, Cox AJ, Zagury JF, Marchini J. Haplotype estimation using sequencing reads. Am J Hum Genet. 2013; 93: 687–96.  [DOI](https://doi.org/10.1016/j.ajhg.2013.09.002) | [PMC free article](/articles/PMC3791270/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24094745/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Haplotype%20estimation%20using%20sequencing%20reads&author=O%20Delaneau&author=B%20Howie&author=AJ%20Cox&author=JF%20Zagury&author=J%20Marchini&volume=93&publication_year=2013&pages=687-96&pmid=24094745&doi=10.1016/j.ajhg.2013.09.002&)

21. Howie B, Marchini J, Stephens M. Genotype imputation with thousands of genomes. G3 (Bethesda). 2011; 1: 457–70.  [DOI](https://doi.org/10.1534/g3.111.001198) | [PMC free article](/articles/PMC3276165/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22384356/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=G3%20(Bethesda)&title=Genotype%20imputation%20with%20thousands%20of%20genomes&author=B%20Howie&author=J%20Marchini&author=M%20Stephens&volume=1&publication_year=2011&pages=457-70&pmid=22384356&doi=10.1534/g3.111.001198&)

22. Jacobson PA, Schladt D, Oetting WS, et al. Genetic determinants of mycophenolate-related anemia and leukopenia after transplantation. Transplantation. 2011; 91: 309–316.  [DOI](https://doi.org/10.1097/TP.0b013e318200e971) | [PMC free article](/articles/PMC3711617/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21107304/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplantation&title=Genetic%20determinants%20of%20mycophenolate-related%20anemia%20and%20leukopenia%20after%20transplantation&author=PA%20Jacobson&author=D%20Schladt&author=WS%20Oetting&volume=91&publication_year=2011&pages=309-316&pmid=21107304&doi=10.1097/TP.0b013e318200e971&)

23. Jacobson PA, Schladt D, Oetting WS, Leduc R, Guan W, Matas AJ, et al. Lower calcineurin inhibitor doses in older compared to younger kidney transplant recipients yield similar troughs. Am J Transplant, 2012; 12: 3326–3336.  [DOI](https://doi.org/10.1111/j.1600-6143.2012.04232.x) | [PMC free article](/articles/PMC3513646/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22947444/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Transplant&title=Lower%20calcineurin%20inhibitor%20doses%20in%20older%20compared%20to%20younger%20kidney%20transplant%20recipients%20yield%20similar%20troughs&author=PA%20Jacobson&author=D%20Schladt&author=WS%20Oetting&author=R%20Leduc&author=W%20Guan&volume=12&publication_year=2012&pages=3326-3336&pmid=22947444&doi=10.1111/j.1600-6143.2012.04232.x&)

24. Elens L, Bouamar R, Hesselink DA, Haufroid V, van der Heiden IP, van Gelder T, et al. A new functional CYP3A4 intron 6 polymorphism significantly affects tacrolimus pharmacokinetics in kidney transplant recipients. Clin Chem. 2011; 57: 1574–1583.  [DOI](https://doi.org/10.1373/clinchem.2011.165613) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21903774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&title=A%20new%20functional%20CYP3A4%20intron%206%20polymorphism%20significantly%20affects%20tacrolimus%20pharmacokinetics%20in%20kidney%20transplant%20recipients&author=L%20Elens&author=R%20Bouamar&author=DA%20Hesselink&author=V%20Haufroid&author=IP%20van%20der%20Heiden&volume=57&publication_year=2011&pages=1574-1583&pmid=21903774&doi=10.1373/clinchem.2011.165613&)

25. Feng Y, Cheng D, Zhang C, Li Y, Zhang Z, Wang J, et al. Association of PDE4B Polymorphisms with Susceptibility to Schizophrenia: A Meta-Analysis of Case-Control Studies. PLoS One. 2016; 11: e0147092.  [DOI](https://doi.org/10.1371/journal.pone.0147092) | [PMC free article](/articles/PMC4710508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26756575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Association%20of%20PDE4B%20Polymorphisms%20with%20Susceptibility%20to%20Schizophrenia:%20A%20Meta-Analysis%20of%20Case-Control%20Studies&author=Y%20Feng&author=D%20Cheng&author=C%20Zhang&author=Y%20Li&author=Z%20Zhang&volume=11&publication_year=2016&pages=e0147092&pmid=26756575&doi=10.1371/journal.pone.0147092&)

26. Hesselink DA, Bouamar R, Elens L, van Schaik RH, van Gelder T. The role of pharmacogenetics in the disposition of and response to tacrolimus in solid organ transplantation. Clin Pharmacokinet. 2014; 53: 123–139.  [DOI](https://doi.org/10.1007/s40262-013-0120-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24249597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=The%20role%20of%20pharmacogenetics%20in%20the%20disposition%20of%20and%20response%20to%20tacrolimus%20in%20solid%20organ%20transplantation&author=DA%20Hesselink&author=R%20Bouamar&author=L%20Elens&author=RH%20van%20Schaik&author=T%20van%20Gelder&volume=53&publication_year=2014&pages=123-139&pmid=24249597&doi=10.1007/s40262-013-0120-3&)

27. de Jonge H, Elens L, de Loor H, van Schaik RH, Kuypers DR. The CYP3A4*22 C>T single nucleotide polymorphism is associated with reduced midazolam and tacrolimus clearance in stable renal allograft recipients. Pharmacogenomics J. 2015; 15: 144–152.  [DOI](https://doi.org/10.1038/tpj.2014.49) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25287072/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=The%20CYP3A4*22%20C>T%20single%20nucleotide%20polymorphism%20is%20associated%20with%20reduced%20midazolam%20and%20tacrolimus%20clearance%20in%20stable%20renal%20allograft%20recipients&author=H%20de%20Jonge&author=L%20Elens&author=H%20de%20Loor&author=RH%20van%20Schaik&author=DR%20Kuypers&volume=15&publication_year=2015&pages=144-152&pmid=25287072&doi=10.1038/tpj.2014.49&)

28. Pallet N, Jannot AS, El Bahri M, Etienne I, Buchler M, De Ligny BH, et al. Kidney transplant recipients carrying the CYP3A4*22 allelic variant have reduced tacrolimus clearance and often reach supratherapeutic tacrolimus concentrations. Am J Transplant. 2015; 15: 800–805.  [DOI](https://doi.org/10.1111/ajt.13059) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25588704/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Transplant&title=Kidney%20transplant%20recipients%20carrying%20the%20CYP3A4*22%20allelic%20variant%20have%20reduced%20tacrolimus%20clearance%20and%20often%20reach%20supratherapeutic%20tacrolimus%20concentrations&author=N%20Pallet&author=AS%20Jannot&author=M%20El%20Bahri&author=I%20Etienne&author=M%20Buchler&volume=15&publication_year=2015&pages=800-805&pmid=25588704&doi=10.1111/ajt.13059&)

29. Damon C, Luck M, Toullec L, Etienne I, Buchler M, Hurault de Ligny B, et al. Predictive Modeling of Tacrolimus Dose Requirement Based on High-Throughput Genetic Screening. Am J Transplant. 2017; 17: 1008–1019.  [DOI](https://doi.org/10.1111/ajt.14040) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27597269/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Transplant&title=Predictive%20Modeling%20of%20Tacrolimus%20Dose%20Requirement%20Based%20on%20High-Throughput%20Genetic%20Screening&author=C%20Damon&author=M%20Luck&author=L%20Toullec&author=I%20Etienne&author=M%20Buchler&volume=17&publication_year=2017&pages=1008-1019&pmid=27597269&doi=10.1111/ajt.14040&)

30. Naito T, Mino Y, Aoki Y, Hirano K, Shimoyama K, Ogawa N, et al. ABCB1 genetic variant and its associated tacrolimus pharmacokinetics affect renal function in patients with rheumatoid arthritis. Clin Chim Acta. 2015; 445: 79–84.  [DOI](https://doi.org/10.1016/j.cca.2015.03.021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25817604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Chim%20Acta&title=ABCB1%20genetic%20variant%20and%20its%20associated%20tacrolimus%20pharmacokinetics%20affect%20renal%20function%20in%20patients%20with%20rheumatoid%20arthritis&author=T%20Naito&author=Y%20Mino&author=Y%20Aoki&author=K%20Hirano&author=K%20Shimoyama&volume=445&publication_year=2015&pages=79-84&pmid=25817604&doi=10.1016/j.cca.2015.03.021&)

31. Knops N, van den Heuvel LP, Masereeuw R, Bongaers I, De Loor H, Levtchenko E, et al. The functional implications of common genetic variation in CYP3A5 and ABCB1 in human proximal tubule cells. Mol Pharm. 2015; 12: 758–768.  [DOI](https://doi.org/10.1021/mp500590s) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25590378/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharm&title=The%20functional%20implications%20of%20common%20genetic%20variation%20in%20CYP3A5%20and%20ABCB1%20in%20human%20proximal%20tubule%20cells&author=N%20Knops&author=LP%20van%20den%20Heuvel&author=R%20Masereeuw&author=I%20Bongaers&author=H%20De%20Loor&volume=12&publication_year=2015&pages=758-768&pmid=25590378&doi=10.1021/mp500590s&)

32. Lesche D, Sigurdardottir V, Setoud R, Oberhänsli M, Carrel T, Fiedler GM, et al. CYP3A5*3 and POR*28 genetic variants influence the required dose of tacrolimus in heart transplant recipients. Ther Drug Monit. 2014; 36: 710–715.  [DOI](https://doi.org/10.1097/FTD.0000000000000080) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24739669/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=CYP3A5*3%20and%20POR*28%20genetic%20variants%20influence%20the%20required%20dose%20of%20tacrolimus%20in%20heart%20transplant%20recipients&author=D%20Lesche&author=V%20Sigurdardottir&author=R%20Setoud&author=M%20Oberh%C3%A4nsli&author=T%20Carrel&volume=36&publication_year=2014&pages=710-715&pmid=24739669&doi=10.1097/FTD.0000000000000080&)

33. Elens L, Hesselink DA, Bouamar R, Budde K, de Fijter JW, De Meyer M, et al. Impact of POR*28 on the pharmacokinetics of tacrolimus and cyclosporine A in renal transplant patients. Ther Drug Monit. 2014; 36: 71–79.  [DOI](https://doi.org/10.1097/FTD.0b013e31829da6dd) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24061445/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Impact%20of%20POR*28%20on%20the%20pharmacokinetics%20of%20tacrolimus%20and%20cyclosporine%20A%20in%20renal%20transplant%20patients&author=L%20Elens&author=DA%20Hesselink&author=R%20Bouamar&author=K%20Budde&author=JW%20de%20Fijter&volume=36&publication_year=2014&pages=71-79&pmid=24061445&doi=10.1097/FTD.0b013e31829da6dd&)

34. Jannot AS, Vuillemin X, Etienne I, Buchler M, Hurault de Ligny B, Choukroun G, et al. A Lack of Significant Effect of POR*28 Allelic Variant on Tacrolimus Exposure in Kidney Transplant Recipients. Ther Drug Monit. 2016; 38: 223–229.  [DOI](https://doi.org/10.1097/FTD.0000000000000267) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26829596/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=A%20Lack%20of%20Significant%20Effect%20of%20POR*28%20Allelic%20Variant%20on%20Tacrolimus%20Exposure%20in%20Kidney%20Transplant%20Recipients&author=AS%20Jannot&author=X%20Vuillemin&author=I%20Etienne&author=M%20Buchler&author=B%20Hurault%20de%20Ligny&volume=38&publication_year=2016&pages=223-229&pmid=26829596&doi=10.1097/FTD.0000000000000267&)

35. Khaled SK, Palmer JM, Herzog J, Stiller T, Tsai NC, Senitzer D, et al. Influence of Absorption, Distribution, Metabolism, and Excretion Genomic Variants on Tacrolimus/Sirolimus Blood Levels and Graft-versus-Host Disease after Allogeneic Hematopoietic Cell Transplantation. Biol Blood Marrow Transplant. 2016; 22: 268–276.  [DOI](https://doi.org/10.1016/j.bbmt.2015.08.027) | [PMC free article](/articles/PMC4716887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26325438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Blood%20Marrow%20Transplant&title=Influence%20of%20Absorption,%20Distribution,%20Metabolism,%20and%20Excretion%20Genomic%20Variants%20on%20Tacrolimus/Sirolimus%20Blood%20Levels%20and%20Graft-versus-Host%20Disease%20after%20Allogeneic%20Hematopoietic%20Cell%20Transplantation&author=SK%20Khaled&author=JM%20Palmer&author=J%20Herzog&author=T%20Stiller&author=NC%20Tsai&volume=22&publication_year=2016&pages=268-276&pmid=26325438&doi=10.1016/j.bbmt.2015.08.027&)

36. Bosó V, Herrero MJ, Bea S, Galiana M, Marrero P, Marqués MR, et al. Increased hospital stay and allograft dysfunction in renal transplant recipients with Cyp2c19 AA variant in SNP rs4244285. Drug Metab Dispos. 2013; 41: 480–487.  [DOI](https://doi.org/10.1124/dmd.112.047977) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23175667/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Increased%20hospital%20stay%20and%20allograft%20dysfunction%20in%20renal%20transplant%20recipients%20with%20Cyp2c19%20AA%20variant%20in%20SNP%20rs4244285&author=V%20Bos%C3%B3&author=MJ%20Herrero&author=S%20Bea&author=M%20Galiana&author=P%20Marrero&volume=41&publication_year=2013&pages=480-487&pmid=23175667&doi=10.1124/dmd.112.047977&)

37. Lunde I, Bremer S, Midtvedt K, Mohebi B, Dahl M, Bergan S, et al. The influence of CYP3A, PPARA, and POR genetic variants on the pharmacokinetics of tacrolimus and cyclosporine in renal transplant recipients. Eur J Clin Pharmacol. 2014; 70: 685–693.  [DOI](https://doi.org/10.1007/s00228-014-1656-3) | [PMC free article](/articles/PMC4025175/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24658827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Clin%20Pharmacol&title=The%20influence%20of%20CYP3A,%20PPARA,%20and%20POR%20genetic%20variants%20on%20the%20pharmacokinetics%20of%20tacrolimus%20and%20cyclosporine%20in%20renal%20transplant%20recipients&author=I%20Lunde&author=S%20Bremer&author=K%20Midtvedt&author=B%20Mohebi&author=M%20Dahl&volume=70&publication_year=2014&pages=685-693&pmid=24658827&doi=10.1007/s00228-014-1656-3&)

38. Bruckmueller H, Werk AN, Renders L, Feldkamp T, Tepel M, Borst C, et al. Which Genetic Determinants Should be Considered for Tacrolimus Dose Optimization in Kidney Transplantation? A Combined Analysis of Genes Affecting the CYP3A Locus. Ther Drug Monit. 2015; 37: 288–295.  [DOI](https://doi.org/10.1097/FTD.0000000000000142) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25271728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Which%20Genetic%20Determinants%20Should%20be%20Considered%20for%20Tacrolimus%20Dose%20Optimization%20in%20Kidney%20Transplantation?%20A%20Combined%20Analysis%20of%20Genes%20Affecting%20the%20CYP3A%20Locus&author=H%20Bruckmueller&author=AN%20Werk&author=L%20Renders&author=T%20Feldkamp&author=M%20Tepel&volume=37&publication_year=2015&pages=288-295&pmid=25271728&doi=10.1097/FTD.0000000000000142&)

39. Passey C, Birnbaum AK, Brundage RC, Oetting WS, Israni AK, Jacobson PA. Dosing equation for tacrolimus using genetic variants and clinical factors. Br J Clin Pharmacol. 2011; 72: 948–957.  [DOI](https://doi.org/10.1111/j.1365-2125.2011.04039.x) | [PMC free article](/articles/PMC3244642/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21671989/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Dosing%20equation%20for%20tacrolimus%20using%20genetic%20variants%20and%20clinical%20factors&author=C%20Passey&author=AK%20Birnbaum&author=RC%20Brundage&author=WS%20Oetting&author=AK%20Israni&volume=72&publication_year=2011&pages=948-957&pmid=21671989&doi=10.1111/j.1365-2125.2011.04039.x&)

40. Passey C, Birnbaum AK, Brundage RC, Schladt DP, Oetting WS, Leduc RE, et al. Validation of tacrolimus equation to predict troughs using genetic and clinical factors. Pharmacogenomics. 2012; 13: 1141–1147.  [DOI](https://doi.org/10.2217/pgs.12.98) | [PMC free article](/articles/PMC3579500/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22909204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Validation%20of%20tacrolimus%20equation%20to%20predict%20troughs%20using%20genetic%20and%20clinical%20factors&author=C%20Passey&author=AK%20Birnbaum&author=RC%20Brundage&author=DP%20Schladt&author=WS%20Oetting&volume=13&publication_year=2012&pages=1141-1147&pmid=22909204&doi=10.2217/pgs.12.98&)

41. Sanghavi K, Brundage RC, Miller MB, Schladt DP, Israni AK, Guan W, et al. Genotype-guided tacrolimus dosing in African-American kidney transplant recipients. Pharmacogenomics J. 2017; 17: 61–68.  [DOI](https://doi.org/10.1038/tpj.2015.87) | [PMC free article](/articles/PMC4909584/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26667830/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genotype-guided%20tacrolimus%20dosing%20in%20African-American%20kidney%20transplant%20recipients&author=K%20Sanghavi&author=RC%20Brundage&author=MB%20Miller&author=DP%20Schladt&author=AK%20Israni&volume=17&publication_year=2017&pages=61-68&pmid=26667830&doi=10.1038/tpj.2015.87&)

42. Lin YS, Dowling AL, Quigley SD, Farin FM, Zhang J, Lamba J, Schuetz EG, Thummel KE. Co-regulation of CYP3A4 and CYP3A5 and contribution to hepatic and intestinal midazolam metabolism. Mol Pharmacol. 2002; 62:162–72.  [DOI](https://doi.org/10.1124/mol.62.1.162) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12065767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&title=Co-regulation%20of%20CYP3A4%20and%20CYP3A5%20and%20contribution%20to%20hepatic%20and%20intestinal%20midazolam%20metabolism&author=YS%20Lin&author=AL%20Dowling&author=SD%20Quigley&author=FM%20Farin&author=J%20Zhang&volume=62&publication_year=2002&pages=162-72&pmid=12065767&doi=10.1124/mol.62.1.162&)

43. Tracy TS, Chaudhry AS, Prasad B, Thummel KE, Schuetz EG, Zhong XB, Tien YC, Jeong H, Pan X, Shireman LM, Tay-Sontheimer J, Lin YS. Interindividual Variability in Cytochrome P450-Mediated Drug Metabolism. Drug Metab Dispos. 2016; 44:343–51.  [DOI](https://doi.org/10.1124/dmd.115.067900) | [PMC free article](/articles/PMC4767386/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26681736/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Interindividual%20Variability%20in%20Cytochrome%20P450-Mediated%20Drug%20Metabolism&author=TS%20Tracy&author=AS%20Chaudhry&author=B%20Prasad&author=KE%20Thummel&author=EG%20Schuetz&volume=44&publication_year=2016&pages=343-51&pmid=26681736&doi=10.1124/dmd.115.067900&)
