# The impact of the CYP2D6 “enhancer” single nucleotide polymorphism on CYP2D6 activity

## Metadata
**Authors:** Jean C Dinh, Erin C Boone, Vincent S Staggs, Robin E Pearce, Wendy Y Wang, Roger Gaedigk, J Steven Leeder, Andrea Gaedigk
**Journal:** Clinical pharmacology and therapeutics
**Date:** 2021 Nov 30
**DOI:** [10.1002/cpt.2469](https://doi.org/10.1002/cpt.2469)
**PMID:** 34716917
**PMCID:** PMC8825689
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8825689/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8825689/pdf/nihms-1752417.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8825689/pdf/nihms-1752417.pdf)

## Abstract

rs5758550 has been associated with enhanced transcription and suggested to be a useful marker of CYP2D6 activity. As there are limited and inconsistent data regarding the utility of this distant “enhancer” single nucleotide polymorphism (SNP), our goal was to further assess the impact of rs5758550 on CYP2D6 activity toward two probe substrates, atomoxetine (ATX) and dextromethorphan (DM), using in vivo urinary metabolite (DM; n=188) and pharmacokinetic (ATX; n=70) and in vitro metabolite formation (ATX and DM; n=166) data. All subjects and tissues were extensively genotyped, the “enhancer” SNP phased with established CYP2D6 haplotypes either computationally or experimentally, and the impact on CYP2D6 activity investigated using several linear models of varying complexity to determine the proportion of variability in CYP2D6 activity captured by each model. For all datasets and models, the “enhancer” SNP had no or only a modest impact on CYP2D6 activity prediction. An increased effect, when present, was more pronounced for ATX than DM suggesting potential substate-dependency. In addition, CYP2D6*2 alleles with the “enhancer” SNP were associated with modestly higher metabolite formation rates in vitro, but not in vivo; no effect was detected for CYP2D6*1 alleles with “enhancer” SNP. In summary, it remains inconclusive whether the small effects detected in this investigation are indeed caused by the “enhancer” SNP or are rather due to its incomplete linkage with other variants within the gene. Taken together, there does not appear to be sufficient evidence to warrant the “enhancer” SNP be included in clinical CYP2D6 pharmacogenetic testing.

Keywords: CYP2D6, dextromethorphan, atomoxetine, phenotype prediction, pharmacogenetic biomarkers

## Introduction

The highly polymorphic Cytochrome P450 2D6 (CYP2D6) enzyme contributes to the metabolism of over 21% of clinically used medications ([1](#R1)). For individuals who are on the extreme ends of the CYP2D6 activity distribution within a population, *a priori* knowledge of CYP2D6 phenotype classification (poor, PM; intermediate, IM; normal, NM and ultrarapid (UM) metabolizers) can reduce risk of inefficacy and adverse events from drugs that are cleared and/or activated by CYP2D6. The importance of the *CYP2D6* pharmacogene is further underscored by the number of drugs for which there are Clinical Pharmacogenetics Implementation Consortium (CPIC) guidelines ([2](#R2)–[6](#R6)) and the number of drugs listed by the Pharmacogenomics KnowledgeBase (PharmGKB) with drug label annotations for CYP2D6 ([7](#R7)).

Despite the available knowledge regarding genetic variation, a large portion of variability in CYP2D6 activity remains unexplained ([8](#R8)–[12](#R12)). Our research group recently reported that among individuals with the same *CYP2D6* genotype, enzymatic activity, as measured by a urinary dextromethorphan/dextrorphan (DM/DX) ratio, can vary up to four-fold ([13](#R13)). We also found that CYP2D6 protein content among samples with the same *CYP2D6* genotype can vary up to ten-fold in human liver microsomes (HLMs) ([13](#R13)), an observation that has been replicated by others ([8](#R8), [9](#R9), [12](#R12)). These findings suggest that there may be uncharacterized genetic variants, collectively referred here as single nucleotide polymorphisms (SNPs), impacting CYP2D6 expression levels and/or protein content, and thus, enzymatic capacity.

A SNP located approximately 116 kb downstream of the *CYP2D6* gene has been associated with increased transcription levels *in vitro* and thus has been referred to in the literature as the *CYP2D6* “enhancer” SNP (G>A, rs5758550) ([14](#R14), [15](#R15)). To be consistent with the literature, we refer to the ‘G’ allele as the “enhancer” SNP as ‘G’ was associated with increased transcription. rs5758550 was first identified by Wang et al. in their efforts to reconcile observations that *CYP2D6*1* and **2* alleles had comparable activity despite the latter having 2851C>T (rs16947). The authors’ observation that rs16947 causes altered splicing was in contrast to the prevailing belief at the time that the amino acid change caused by rs16947 (p.R296C) was functionally inconsequential ([15](#R15)). Subsequent investigations found that the “enhancer” SNP was in high linkage disequilibrium (LD) with rs16947 and was associated with a two-fold increase in *CYP2D6* transcription levels that appeared to compensate for the two-fold decrease in expression caused by the splicing defect ([14](#R14)). In contrast, 2989G>A (rs28371725) found in *CYP2D6*41* was not linked with the “enhancer” SNP, and Wang et al. argued that rs16947 ‘T’ and rs28371725 ‘A’ contribute to the decreased function status of this haplotype. These findings led Wang et al. to conclude that the “enhancer” SNP may account for unexplained variability in CYP2D6 activity ([14](#R14)) and propose a modified CYP2D6 activity score (AS) system including the “enhancer” SNP ([16](#R16)). However, the impact of the “enhancer” SNP could not be confirmed *in vitro* ([12](#R12)) or *in vivo* ([17](#R17), [18](#R18)) or remains inconclusive ([17](#R17), [19](#R19)). In fact, Zanger et al. elegantly demonstrated that variability in activity can rather be explained by the presence and combination of three SNPs (rs16947, rs28371725 and rs1135840) that define and differentiate the *CYP2D6*2* and **41* haplotypes rather than the “enhancer” SNP ([18](#R18)).

Further complicating matters is the fact that the “enhancer” SNP it is not in complete LD with rs16947 as it is, for example, found on 96% of *CYP2D6*2* alleles of European individuals, but only 71% of individuals with African ancestry ([20](#R20)). Additionally, the “enhancer” SNP also occurs on several other *CYP2D6* haplotypes, some of which have decreased activity or are nonfunctional ([20](#R20)). Hence, for subjects who are heterozygous for the “enhancer” SNP, and heterozygous for a *CYP2D6* star allele (e.g., *CYP2D6*1/*2),* it may be critical to know on which allele it is located to determine its contribution to variable CYP2D6 activity.

We exploited four datasets consisting of both *in vivo* and *in vitro* data, taking advantage of two CYP2D6 probe substrates, dextromethorphan (DM) and atomoxetine (ATX) ([Figure 1a](#F1), [b](#F1)) to evaluate the impact of the “enhancer” SNP after computationally or experimentally phasing it with *CYP2D6* star allele haplotypes.

### Figure 1. Overview of metabolic pathways and impact of the “enhancer” SNPs on activity for CYP2D6*1 and *2 alleles with and without the “enhancer” SNP.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b7c5/8825689/59ff79e0a976/nihms-1752417-f0001.jpg)

Panels (a) and (b) provide an overview of the Cytochrome P450 enzymes contributing to DM and ATX metabolism. Panels c-f compare the impact of the “enhancer” SNP on CYP2D6*1 and CYP2D6*2 alleles with (+) and without (−) the “enhancer” SNP, number of alleles are as indicated. The following differences in CYP2D6 activity proxies are depicted: urinary DM/DX ratio (c); 4-OH-ATX formation rate in HLMs (d); DX formation rate in HLMs (e); ATX terminal elimination rate (f); and ATX apparent oral clearance (g).

## Methods

### In vivo phenotyping using DM/DX urinary metabolic ratio.

A total of 188 children from 116 families aged 7–15 years were recruited into a longitudinal study assessing CYP2D6 activity during adolescence. On the study day, a pre-dose urine sample was collected, DM (0.3 mg/kg) administered orally, and all urine collected for 4 hours post-dose. The molar ratio of DM/DX was used as a phenotypic marker of CYP2D6 activity (for details see [Supplemental Materials](#SD2)).

### In Vitro DM and ATX HLM incubations.

Liver tissue samples were obtained from Liver Tissue Cell Distribution System (Minneapolis, MN, USA; Pittsburgh, PA, USA), the University of Maryland Brain and Tissue Bank for Developmental Disorders, and XenoTech LLC (Kansas City, KS). Human liver microsomes (HLMs) were prepared as described elsewhere ([15](#R15)). HLMs were incubated with DM (80 μM; n=23 adult, n=133 pediatric) or ATX (10 μM; n=30 adult; n=117 pediatric). To quantitate CYP2D6 enzymatic activity, formation rate of the CYP2D6-mediated metabolites, dextrorphan (DX) and 4-hydroxyatomoxetine (4-OH-ATX), were determined in individual incubations as detailed in [Supplemental Materials](#SD2). Formation rate of other ATX metabolites, N-desmethyl-atomoxetine (NDM-ATX) and 2-methyl-hydroxy atomoxetine (2-OH-ATX) were also determined.

### In vivo phenotyping using ATX pharmacokinetic (PK) data.

Seventy unique ATX PK profiles were collated to determine the impact of the “enhancer” SNP on ATX apparent oral clearance (CL_app_) and terminal elimination rate constant (k_e_). Data are from children and adolescents who participated in one of three studies approved by the Children’s Mercy Hospital Institutional Review Board: a single-dose *CYP2D6*-genotype stratified ATX PK study ([16](#R16)) (Study 1, n=23); a population-PK model-informed single-dose ATX PK study (Study 2, n=23); and a population-PK model-informed single-dose and steady-state ATX PK study (Study 3, n=24). Additional information see [Supplemental Materials](#SD2). CL_app_ and k_e_ were chosen as *in vivo* phenotypes for this analysis.

### CYP2D6 Genotyping.

*CYP2D6* genotyping was carried out as previously described ([10](#R10), [21](#R21)–[26](#R26)) and/or with a custom OpenArray run on QuantStudio 12K Flex Real-Time PCR System version 1.2.2 (Thermo Fisher Scientific, Waltham, MA). Copy number variation was assayed by long-range PCR, a quantitative gene copy number assay ([26](#R26)) and/or digital droplet PCR using a QX200 Droplet Digital PCR System (Bio-Rad, Hercules, CA) ([27](#R27)). Regardless of the method/platform used all subjects and tissue samples were genotyped for the following *CYP2D6* alleles: **2*, **3, *4, *6, *7, *9, *10, *11 *17, *29, *35, *40, *41, *42, *45, *59* and copy number variation (*CYP2D6*5* gene deletion, duplications, other rearrangements including *CYP2D6*13 CYP2D7–2D6* hybrids, *CYP2D6*36+*10* and **68+*4* tandems) and the “enhancer” SNP. Allele definitions are according to the Pharmacogene Variation Consortium (PharmVar) ([28](#R28)).

### CYP2D6 “Enhancer” SNP Phasing.

Liver tissue samples and PK study subjects were subjected to DropPhase2D6, a method to experimentally determine on which star allele the “enhancer” SNP is located ([20](#R20)). Assay details are provided in [Table S1](#SD2)). For the *in vivo* study assessing urinary DM/DX ratio in participants, the phase of the “enhancer” SNP was computationally inferred using the PHASE algorithm (v.2.1.1) ([29](#R29)). Samples with DM/DX ratios were computationally phased with the “enhancer” SNP as part of larger population samples as described ([20](#R20)).

### Allele-Specific Sequencing.

The allele of interest was amplified in its entirety using long-range PCR as described previously ([27](#R27), [30](#R30)). Resulting fragments were subsequently subjected to bidirectional Sanger sequencing.

### Statistical analyses.

The impact of the “enhancer” SNP on CYP2D6 activity was assessed by fitting several linear models and computing the proportion of variability (R^2^) in CYP2D6 activity accounted for in each model. The explanatory (i.e., independent) variables assessed were AS, *CYP2D6* genotype and “enhancer” genotype (A/A, A/G and G/G, with and without phasing with star alleles). The proxies of CYP2D6 activity, (i.e., outcome variables) were as follows: DX formation rates from a panel of HLMs; 4OH-ATX, NDM, and 2-OH-ATX formation rates from a panel of HLMs; DM/DX urinary metabolic ratio in healthy children; and ATX k_e_ and ATX CL_app_ in children with attention deficit/hyperactivity disorder (ADHD). All outcomes were log-transformed to reduce skewness.

Four models were fit for each outcome. Model 1 evaluated variability in outcome as a function of AS (assigned as recommended by CPIC ([24](#R24))). Model 2 included only *CYP2D6* genotype, captured in a set of indicator variables, one per star allele, to provide a baseline estimate of variability associated with *CYP2D6* genotype alone. In Model 3, “enhancer” SNP genotype (A/A, A/G or G/G), coded using two indicator variables, was added to Model 2 to estimate variability accounted for by this SNP beyond *CYP2D6* diplotype alone. By treating the “enhancer” SNP genotype as categorical rather than using the count of “enhancer” copies ((0, 1, 2) as a quantitative variable, we allowed for a non-additive effect of being homozygous vs. heterozygous for this SNP. Model 4 included *CYP2D6* genotype, “enhancer” SNP genotype, and additional variables reflecting phasing. For example, rather than a single binary variable for *CYP2D6*1* coded “Yes” if the allele was present and “No” if not, Model 4 included one binary variable for *CYP2D6*1* without the “enhancer” SNP (“Yes” if present, “No” if not) and a second binary variable for *CYP2D6*1 with* the “enhancer” SNP. This approach allowed “enhancer” SNP effects to vary for different alleles. Phasing was only considered for a given allele if there were at least three samples with and three samples without the “enhancer” SNP.

In addition to regression analyses, visual plots and non-parametric Wilcoxon exact two-sample tests were used for each outcome to compare samples with and without the “enhancer” SNP for two sets of samples: those having a *CYP2D6*1* allele, and those having a *CYP2D6*2* allele. For urinary DM/DX ratio, further analyses compared samples with and without the “enhancer” SNP present for five *CYP2D6* genotypes with a sufficient number of cases (*CYP2D6*1/*1, *1/*2, *1/*29, *1/*35,* and **1/*41*). Welch *t*-tests, which allow for unequal variance across groups, were used for these comparisons.

## Results

Four unique datasets assessing variable CYP2D6 activity were used to quantify the impact of the “enhancer” SNP. [Table 1](#T1) provides a summary of the demographic information from all four studies. Genotypes identified in each study and allele frequencies are provided in [Tables S2](#SD2) and [S3](#SD1). For all samples heterozygous for the “enhancer” SNP it was determined, either experimentally or computationally, on which *CYP2D6* haplotype (or star allele) the SNP was located.

### Table 1.

|   | DM HLM Incubations(Total N = 156)In vitro |  |  | ATX HLM Incubations(Total N = 147)In vitro |  |  | DM/DX Phenotyping(Total N=188)In vivo |  |  | ATX Pharmacokinetics(Total N = 70)In vivo |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | “enhancer” SNP Genotype(Experimentally phased) |  |  | “enhancer” SNP Genotype(Experimentally phased) |  |  | “enhancer” SNP Genotype(Computationally phased) |  |  | “enhancer” SNP Genotype(Experimentally phased) |  |  |
| “enhancer” SNP Genotype | A/A | A/G | G/G | A/A | A/G | G/G | A/A | A/G | G/G | A/A | A/G | G/G |
| Number of Subjects Per Study, divided by “enhancer” SNP Genotype | 86 | 60 | 10 | 84 | 53 | 10 | 88 | 80 | 20 | 34 | 33 | 3 |
| Age - Mean (Std dev) | 12.7 (15.8) | 17.1 (20.2) | 13.3 (14.7) | 14.6 (17.2) | 19.0 (21.7) | 19.1 (19.6) | 11.2 (2.8) | 11.1 (2.5) | 10.9 (1.9) | 12.9 (3.13) | 12.8 (3.44) | 14.8 (3.73) |
| Gender (%) a |   |   |   |   |   |   |   |   |   |   |   |   |
| Female | 31.4% | 40.0% | 70.0% | 29.8% | 43.4% | 70.0% | 39.8% | 42.9% | 30.0% | 17.6% | 18.2% | 33.3% |
| Male | 67.4% | 60.0% | 30.0% | 69.0% | 56.6% | 30.0% | 60.2% | 57.1% | 70.0% | 82.4% | 81.8% | 66.7% |
| Unknown | 1.20% | 0% | 0% | 1.20% | 0% | 0% | 0% | 0% | 0% | 0% | 0% | 0% |
| Ethnicity (%) a |   |   |   |   |   |   |   |   |   |   |   |   |
| African American | 22.1% | 30.0% | 40.0% | 21.4% | 26.4% | 40.0% | 27.3% | 53.8% | 65.0% | 8.82% | 48.5% | 66.7% |
| Caucasian | 46.5% | 48.3% | 30.0% | 45.2% | 50.9% | 30.0% | 62.5% | 31.2% | 30.0% | 73.5% | 36.3% | 33.3% |
| Hispanic | 3.50% | 3.33% | 0% | 4.80% | 3.80% | 0% | 1.10% | 1.20% | 0% | 0% | 0% | 0% |
| Native American | 0% | 0% | 10% | 0% | 0% | 10.0% | 0% | 0% | 0% | 0% | 0% | 0% |
| Pacific Islander | 1.20% | 0% | 0% | 1.20% | 0% | 0% | 0% | 0% | 0% | 5.88% | 0% | 0% |
| Mixed Ethnicity | 0% | 0% | 0% | 0% | 0% | 0% | 9.10% | 13.8% | 5.00% | 11.8% | 15.2% | 0% |
| Unknown | 26.7% | 18.3% | 20.0% | 27.4% | 18.9% | 20.0% | 0% | 0% | 0% | 0% | 0% | 0% |
| CYP2D6 Phenotype (%) a |   |   |   |   |   |   |   |   |   |   |   |   |
| PM (AS=0) | 12.8% | 1.67% | 0% | 11.9% | 1.90% | 0% | 11.4% | 0% | 0% | 11.8% | 0% | 0% |
| IM (AS 0<x<1.25) | 45.3% | 26.7% | 10.0% | 42.9% | 26.4% | 10.0% | 31.8% | 32.5% | 15.0% | 55.9% | 54.6% | 0% |
| NM (AS 1.25≤x≤2.25) | 41.9% | 66.7% | 90.0% | 45.2% | 66.0% | 90.0% | 55.7% | 65.0% | 70.0% | 32.3% | 39.4% | 33.3% |
| UM (AS >2.25) | 0% | 3.33% | 0% | 0% | 5.70% | 0% | 1.10% | 2.5% | 15.0% | 0% | 6.00% | 66.7% |
| IM or NM | 0% | 1.67% | 0% | 0% | 0% | 0% | 0% | 0% | 0% | 0% | 0% | 0% |

Table 1 Caption: Summary of demographic information for the subjects used in this analysis by study and “enhancer” SNP genotype.

A previous analysis of the participants in the DM/DX urinary ratio study suggested that there was a difference in CYP2D6 activity in individuals with rs16947 and the “enhancer” SNP, compared to those having only rs16947 ([15](#R15)). For this current analysis, data were re-analyzed using different models which included AS, *CYP2D6* genotype, and computationally inferred “enhancer” SNP allele linkage ([Table 2](#T2)). The “enhancer” SNP, however, did not explain variable DM/DX urinary ratio on its own (data not shown) or after controlling for *CYP2D6* genotype (Model 3). AS accounted for 18% of the variability whereas *CYP2D6* genotype accounted for 19% of variability in log_10_(DM/DX) (Model 2); variability accounted for remained at 19% after the addition of “enhancer” SNP genotype information (Model 3). When phasing was added in Model 4, variability accounted for increased to 26%. As shown in [Figure 1c](#F1), there was little evidence of a meaningful impact of the “enhancer” SNP for samples with a *CYP2D6*1* allele. In contrast, samples with a *CYP2D6*2* ([Figure 1c](#F1)) and the “enhancer” SNP had log_10_(DM/DX) values that were lower by 1.1 standard deviations (95% CI 0.3–1.9, p=0.012) on average, suggesting an increase in activity. Means for samples with and without the “enhancer” SNP by *CYP2D6* genotype are shown in [Table 3](#T3). In this analysis, the “enhancer” SNP was associated with mean values of log_10_(DM/DX) 0.6 SDs lower (i.e., increased CYP2D6 activity) for the *CYP2D6***1/*1* genotype (p=0.044); however, there were only three samples with the “enhancer” SNP. Other apparent differences could not be estimated with sufficient precision to draw definitive conclusions.

### Table 2.

| Outcome | Model | Explanatory variable(s) | R2 | R2adj | Enhancer genotype |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |   |   |   |   | A/G vs. A/AB (95% CI)1 | p | G/G vs. A/AB (95% CI)1 | p |
| Urinary DM/DX metabolic ratio | 1 | Activity score | 0.18 | 0.17 |   |   |   |   |
|   | 2 | genotype | 0.19 | 0.12 |   |   |   |   |
|   | 3 | genotype, Enhancer SNP genotype | 0.19 | 0.12 | −0.3 (−0.8, 0.2) | 0.289 | −0.3 (−1.1, 0.4) | 0.332 |
|   | 4 | genotype, Enhancer SNP genotype, Phasing | 0.26 | 0.16 | 0.4 (−0.6, 1.4) | 0.395 | 0.9 (−0.7, 2.6) | 0.268 |
| DX formation in HLMs | 1 | Activity score | 0.16 | 0.16 |   |   |   |   |
|   | 2 | genotype | 0.21 | 0.14 |   |   |   |   |
|   | 3 | genotype, Enhancer genotype | 0.21 | 0.13 | 0.0 (−0.5, 0.5) | 0.986 | 0.2 (−0.6, 1.1) | 0.596 |
|   | 4 | genotype, Enhancer genotype, Phasing | 0.22 | 0.13 | 0.0 (−0.6, 0.6) | 0.970 | 0.2 (−0.7, 1.4) | 0.737 |
| 4-OH-ATX formation in HLMs | 1 | Activity score | 0.37 | 0.36 |   |   |   |   |
|   | 2 | genotype | 0.40 | 0.34 |   |   |   |   |
|   | 3 | genotype, Enhancer genotype | 0.41 | 0.34 | 0.3 (−0.2, 0.9) | 0.195 | 0.6 (−0.3, 1.5) | 0.213 |
|   | 4 | genotype, Enhancer genotype, Phasing | 0.44 | 0.37 | 0.1 (−0.6, 0.8) | 0.851 | 0.1 (−1.2, 1.3) | 0.905 |
| ATX Single-dose PK (ke) | 1 | Activity score | 0.42 | 0.41 |   |   |   |   |
|   | 2 | genotype | 0.52 | 0.46 |   |   |   |   |
|   | 3 | genotype, Enhancer genotype | 0.55 | 0.48 | 0.3 (−0.3, 0.8) | 0.356 | 1.2 (0.0, 2.3) | 0.045 |
|   | 4 | genotype, Enhancer genotype, Phasing | 0.56 | 0.47 | 0.5 (−0.3, 1.3) | 0.182 | 1.5 (0.1, 2.8) | 0.033 |
| ATX Single-dose PK (CLapp) | 1 | Activity score | 0.52 | 0.51 |   |   |   |   |
|   | 2 | genotype | 0.51 | 0.45 |   |   |   |   |
|   | 3 | genotype, Enhancer genotype | 0.59 | 0.52 | 0.0 (−0.5, 0.5) | 0.988 | 1.6 (0.5, 2.7) | 0.004 |
|   | 4 | genotype, Enhancer genotype, Phasing | 0.60 | 0.51 | −0.2 (−1.0, 0.5) | 0.526 | 1.4 (0.1, 2.6) | 0.038 |

Table 2 Caption: Coefficient of determination (R²) for linear regression models evaluated.

### Table 3.

|   | No “enhancer” SNP (A/A) |  | With “enhancer” SNP (A/G + G/G) |  |   |   |
| --- | --- | --- | --- | --- | --- | --- |
| CYP2D6 Genotype | N | log10(DM/DX)Mean (SD) | N | log10(DM/DX)Mean(SD) | Standardized mean difference (95% CI) | p |
| *1/*1 | 20 | −2.61 (0.53) | 3 | −2.89 (0.09) | 0.55 (0.02, 1.09) | 0.044 |
| *1/*2 | 4 | −2.36 (0.37) | 20 | −2.60 (0.27) | 0.82 (−1.04, 2.69 | 0.281 |
| *1/*29 | 5 | −2.21 (0.40) | 3 | −2.34 (0.15) | 0.41 (−1.15, 1.96) | 0.539 |
| *1/*35 | 2 | −2.87 (0.50) | 2 | −1.87 (1.23) | −1.03 (−8.16, 6.09) | 0.447 |
| *1/*41 | 9 | −2.24 (0.27) | 3 | −2.41 (0.77) | 0.41 (−4.05, 4.87) | 0.748 |

Table 3 Caption: Comparison of log(DM/DX) means by CYP2D6 genotype and presence of the “enhancer” SNP

Regression analysis conducted on the HLM data sets suggests that ATX is a more sensitive CYP2D6 probe substrate than DM, with 37% of the variability in 4-OH-ATX formation rate being explained by CYP2D6 AS, compared to 16% for DX-formation rate ([Table 2](#T2), Model 1). CYP2D6 AS was not associated with variability in 2-OH-ATX or NDM-ATX formation (R^2^=0.00 for both), respectively, which is expected given that their formation is primarily mediated by other enzymes ([Figure 1a](#F1), [b](#F1)). *CYP2D6* genotype alone accounted for 40% and 21% of the variability in 4-OH-ATX and DX formation, respectively (Model 2). The addition of the categorical value for “enhancer” SNP genotype, which allowed for non-additive effects for samples that were homozygous vs. heterozygous for the “enhancer” SNP, slightly increased variability accounted for in DX formation. This finding suggests that inclusion of the “enhancer” SNP, without “enhancer” SNP location information (i.e., not knowing on which star allele the “enhancer” SNP is located), only explains an additional 1% of the variability observed in 4-OH-ATX and DX formation rates ([Table 2](#T2)). Adding the “enhancer” SNP location information explained an additional 3% of variability for 4-OH-ATX formation rate and 1% for DX formation rate ([Table 2](#T2)). In allele-specific comparisons, there was little evidence of impact of the “enhancer” SNP on 4-OH-ATX and DX metabolite formation among samples with a *CYP2D6*1* allele but some evidence of an effect among samples with a *CYP2D6*2* allele. These analyses were, however, limited by the small number of CYP2D6**1* with and *CYP2D6*2* without the enhancer SNP ([Table 4](#T4), [Figure 1d](#F1), [e](#F1)).

### Table 4.

|   |   | *1 −enh SNP2 | *1 +enh SNP3 |   | *2 −enh SNP2 | *2 +enh SNP3 |   |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   | Outcome1 | Median | Median | p-value | Median | Median | p-value |
| In vitro HLMs | 2-OH-ATX | 1.74 | 1.78 | 0.779 | 1.57 | 1.84 | 0.106 |
| 4-OH-ATX | 2.32 | 2.08 | 0.553 | 1.72 | 2.57 | 0.055 |  |
| NDM-ATX | 1.75 | 1.73 | 0.532 | 1.65 | 1.74 | 0.028 |  |
| DX | 2.15 | 2.02 | 0.388 | 1.75 | 2.40 | 0.178 |  |
| In vivo PK | ATX ke | 0.28 | 0.25 | 0.507 | 0.25 | 0.28 | 0.617 |
| ATX CLapp | 4.38 | 4.39 | 0.839 | 4.36 | 4.40 | 0.617 |  |

Table 4 Caption: Comparisons of DM and ATX metabolite formation rates in HLMs and of ATX PK parameters for CYP2D6*1 and *2 alleles with and without the “enhancer” SNP.

Finally, n=70 single-dose PK profiles were analyzed to assess the impact of the “enhancer” SNP on ATX kinetic parameters k_e_ and CL_app_ ([Table 2](#T2)). AS accounted for 42% and 52% of the variability in ATX terminal elimination rate and apparent oral clearance, respectively (Model 1). Both PK parameters were modeled as a function of eight *CYP2D6* genotypes, which were coded as categorical variables (0, haplotype not present; 1, haplotype present). Considering only *CYP2D6* genotype, 52% and 51% of the variability in k_e_ and CL_app_, respectively, were accounted for (Model 2). In Model 3, which included “enhancer” SNP genotype, 55% (k_e_) and 59% (CL_app_) of the variability were accounted for suggesting a modest “enhancer” SNP effect. Noteworthy, strong additive effects for G/G vs. A/A ‘enhancer’ SNP genotypes were noted on the PK parameters. Lastly, Model 4 included variables reflecting phasing for the two alleles observed in sufficient numbers both with and without the “enhancer” SNP allowing for the “enhancer” effect to be different for these two alleles ([Table 2](#T2)). This final model explained 56% of the variability in terminal elimination rate and 60% of the variability in apparent oral clearance ([Table 2](#T2)). Of note, all *CYP2D6*17* alleles had the “enhancer” SNP, and thus its impact on this allele could not be estimated. In comparisons of ke and CL_app_ values having *CYP2D6*1* and **2* alleles with and without the “enhancer” SNP there was little evidence of difference; it is noted though that in each comparison there was one group with only 3 or 4 samples ([Table 4](#T4), [Figure 1f](#F1), [g](#F1)).

Additionally, we resequenced four *CYP2D6*1* alleles with and seven *CYP2D6*2* alleles without the “enhancer” SNP. None had additional SNPs that could tentatively explain altered activity. All four *CYP2D6*1* were **1.006* suballeles while **2.011* (n=2), **2.015* (n=2), **2.019* (n=1), **2.025* (n=1) and **2.026* (n=1) were identified among the *CYP2D6*2* alleles.

## Discussion

Consistent with other *in vivo* reports ([17](#R17), [18](#R18)) our data did not find convincing evidence that the “enhancer” SNP is associated with a change in activity that would support revising values for AS calculation or CYP2D6 phenotype classification.

For the DM/DX urinary metabolic ratio data set, the inclusion of phasing information for the “enhancer” SNP accounted for 7% of variability while the previous analysis of the data by Wang et al. using unphased “enhancer” SNP data determined that the “enhancer” SNP accounted for ~5% of the variability in CYP2D6 activity ([15](#R15)). Although a small fraction of variability was explained by the “enhancer” SNP in this data set, it remains unknown whether it would improve the prediction of a person’s metabolic capacity given considerable intra-individual variability ([31](#R31)), variation among individuals with the same genotype ([10](#R10), [11](#R11)), and also the contribution of urinary pH on DM/DX variability (unpublished data and ([32](#R32)) which was not accounted for in the present study.

For the *in vitro* HLM datasets, AS and genotype were associated respectively with 16% and 21% of the variability in DX-formation, which is slightly lower than the 23% reported for AS by Ning et al. ([12](#R12)). The low R^2^ value may reflect that DM is also metabolized by CYP3A4. Including presence of the “enhancer” SNP in the model did not increase variability accounted for, and inclusion of the “enhancer” SNP phasing information only explained an additional 1% of variability in DX-formation. DM may not be a sensitive enough substrate to evaluate the impact of the “enhancer” SNP, given that other enzymes may also contribute to DM metabolism. Indeed, with the more sensitive CYP2D6 probe substrate ATX, 37% and 40% of variability in 4-OH-ATX formation was accounted for with activity score and *CYP2D6* genotype, respectively. Presence of the “enhancer” SNP increased variability accounted for in 4-OH-ATX formation to 41%, and addition of its phasing information resulted in a further increase to 44% of variance accounted for. Thus, the “enhancer” SNP was associated with 4% of the variability in CYP2D6 activity beyond that accounted for by *CYP2D6* genotype. However, the overall impact of the “enhancer” SNP was modest in these *in vitro* datasets.

The final of the four datasets was also the smallest with 70 subjects. This analysis included single-dose ATX PK data from three clinical studies, where ATX k_e_ and CL_app_ were used as proxies of CYP2D6 activity. Of the activity proxies evaluated, ATX PK parameters appeared to be the most sensitive measures, with AS and genotype being associated with 40–50% of the variability observed for k_e_ and CL_app_. “Enhancer” SNP presence, without additional phasing information, accounted for a further 3% and 8% increase in k_e_ and CL_app_ variability, respectively. The addition of phased “enhancer” SNP information accounted for an additional 1% of variability. Given that the fraction of ATX that is metabolized by CYP2D6 is ~80–90% ([33](#R33), [34](#R34)), making ATX a superior probe to measure CYP2D6 activity, we estimate the upper limit of “enhancer” SNP contribution to variable CYP2D6 activity does not exceed 5–10%. For substrates that are less dependent on CYP2D6 for oxidative metabolism, the variability explained by the “enhancer” SNP is anticipated to be lower.

For each dataset, we also evaluated the impact of the “enhancer” SNP on specific *CYP2D6* genotypes. In analyses using DM/DX urinary metabolic ratios there was some evidence of higher activity associated with presence of the “enhancer” SNP for the *CYP2D6*1/*1* genotype ([Table 3](#T3)), but estimates of the “enhancer” SNP’s effect for the other four genotypes were too imprecise to rule out an effect in either direction. In analyses involving the ATX *in vitro* activity and *in vivo* PK proxies of CYP2D6 activity ([Table 4](#T4)), there was some evidence for higher metabolite formation rates in HLMs in the presence of the “enhancer” SNP on the *CYP2D6*2* allele, but no such evidence was found for the *in vivo* PK parameters or for the presence of the “enhancer” SNP on the *CYP2D6*1* allele. Although the small number of samples having a *CYP2D6*1* allele with the “enhancer” SNP and *CYP2D6*2* allele without the “enhancer” SNP pose a limitation on our ability to detect potentially small effects of the “enhancer” SNP, we would have expected to see an effect for the *CYP2D6*1* allele as this haplotype does not have any other variants (amino acid changes) that may mask or offset any impact the “enhancer” SNP may have on expression levels.

While our cumulative data could be interpreted as being supportive of the “enhancer” SNP having a small impact on CYP2D6 activity, this may also be explained by recently published new findings ([14](#R14))regarding the combinatorial effect of 2851C>T (p.S486T) and 4181G>C (p.S486T) ([18](#R18)) showing that reduced activity caused by 2851C>T can be compensated by 4181G>C and explain why *CYP2D6*2* has activity that is similar to that of *CYP2D6*1*. For *CYP2D6*41*, Zanger et al. demonstrated that 4181G>C cannot fully compensate for the 2989G>A splice defect. It remains to be seen though how much the balancing act between 2851C>T and 4181G>C is impacted by other variants. Also, Wang et al. suggested that 2851C>T may contribute to alternative splicing ([15](#R15)). While this has not been independently been confirmed, this possibility cannot be dismissed. Considering that the “enhancer” SNP is in incomplete linkage with 2851C>T and 4181G>C, and that linkage varies among populations, associations found with the “enhancer” SNP may rather reflect functional consequences of other variants and/or their combinations.

Our findings can also be viewed as being consistent with those reported by others. Investigating 627 patients with breast cancer treated with tamoxifen, Sanchez-Spitman et al. reported that the inclusion of the “enhancer” SNP did not account for variability in trough plasma concentrations of tamoxifen or the CYP2D6-mediated tamoxifen metabolite, endoxifen, beyond what was accounted for by *CYP2D6* genotype ([17](#R17)). Furthermore, all patients with a *CYP2D6*2* allele and sub-therapeutic concentrations of endoxifen had the “enhancer” SNP, the opposite of what would be expected in the presence of an allele associated with increased CYP2D6 activity and increased endoxifen concentrations. One particular caveat of this study is that tamoxifen is also metabolized by other polymorphic enzymes, including CYP2C9, CYP2C19, and CYP2B6, which could mask a modest effect of the “enhancer” SNP. A study on 115 human liver tissue samples could also not find an impact of the “enhancer” SNP on variability of CYP2D6 activity ([12](#R12)). Zanger et al. using *in vivo* sparteine metabolic ratios (n=207) and data from 150 liver samples ([18](#R18)) state that they ‘did not find a consistent influence of the enhancer genotype on the metabolic ratio and that the enhancer genotype also failed to show an impact on CYP2D6 protein content and enzyme activity phenotypes’. In contrast, Thomas et al. reported that ‘taking the enhancer SNP into account when assigning AS with the *CYP2D6*2* allele led to more variability being explained in CL_0_ compared with conventional AS assignments’ in their racially mixed study population ([19](#R19)). However, this finding may also be explained, at least in part, by having ‘downgraded’ the AS of diplotypes containing *CYP2D6*2* and a **41* or **9* allele; for the latter there is mounting evidence supporting that a lower value for AS calculation may more precisely reflect their activity and will therefore likely improve phenotype prediction ([35](#R35)–[37](#R37)).

One limitation of all studies including our own is the relatively small number of samples with genotypes of interest, e.g., *CYP2D6*1* or **2* alleles with and without the “enhancer” SNP along with other uncontrolled/unknown factors. Given the highly polymorphic nature of the *CYP2D6* gene locus, substrate specificity, new insights into combinatorial effect between variants, and often multiple contributing pathways of metabolism along potentially hundreds of possible diplotypes in any given population, it may be difficult, if not impossible, to quantify a relatively small effect with precision unless substantially larger datasets are available.

In summary, taking all findings, our own and those by other investigators regarding the “enhancer” SNP under consideration, we come to the conclusion that there is not enough evidence to support the inclusion of the “enhancer” SNP into clinical testing and/or revising the activity score system as suggested by Ray et al. ([16](#R16)). It remains to be seen, whether more sophisticated models taking patient demographics such as age, weight/obesity, ethnicity, other contributing pathways of metabolism, more accurate allele function assignments and sequence rather than genotype-based genetic testing, among others into account, would benefit by including the “enhancer” SNP.

## Supplementary Material

### What is the current knowledge on this topic?

### What is the current knowledge on this topic?

### What is the current knowledge on this topic?

### What is the current knowledge on this topic?

*CYP2D6* genetic variation contributes to variability in CYP2D6 enzymatic activity across individuals and populations. However, wide ranges of activity in individuals with the same *CYP2D6* genotype remain poorly understood. There are controversial findings regarding the impact of a distant “enhancer” SNP (rs5758550) on CYP2D6 activity.

### What question did this study address?

To determine the clinical utility of the “enhancer” SNP as a biomarker, we assessed the contribution of this SNP to variable CYP2D6 activity, using four datasets that both contained extensive genotyping of the *CYP2D6* locus and CYP2D6 activity quantification.

### What does this study add to our knowledge?

This investigation found that associations of the “enhancer” SNP with increased activity are modest and may more likely be observed with substrates that are primarily cleared by CYP2D6 or may be caused by other mechanisms such as combinatorial effects of variants that are in incomplete linkage within the “enhancer “ SNP.

### How might this change clinical pharmacology or translational science?

While the findings of this investigation do not support the inclusion of the “enhancer” SNP into clinical *CYP2D6* pharmacogenetic testing.

## Funding:

This study was supported by grant P50 HD090258 from the *Eunice Kennedy Shriver* National Institute of Child Health and Human Development (NICHD). Collection of the *in vivo* dextromethorphan phenotype data included in the study was supported by grant R01 HD058556, also from NICHD.

The Liver Tissue Cell Distribution System (Minneapolis, MN, USA; Pittsburgh, PA, USA) is supported by NIH Contracts #N01-DK-7–0004/HHSN267200700004C, #HHSN276201200017C).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Saravanakumar A, Sadighi A, Ryu R & Akhlaghi F Physicochemical Properties, Biotransformation, and Transport Pathways of Established and Newly Approved Medications: A Systematic Review of the Top 200 Most Prescribed Drugs vs. the FDA-Approved Drugs Between 2005 and 2016. Clin Pharmacokinet 58, 1281–94 (2019).  [DOI](https://doi.org/10.1007/s40262-019-00750-8) | [PMC free article](/articles/PMC6773482/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30972694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacokinet&title=Physicochemical%20Properties,%20Biotransformation,%20and%20Transport%20Pathways%20of%20Established%20and%20Newly%20Approved%20Medications:%20A%20Systematic%20Review%20of%20the%20Top%20200%20Most%20Prescribed%20Drugs%20vs.%20the%20FDA-Approved%20Drugs%20Between%202005%20and%202016.&author=A%20Saravanakumar&author=A%20Sadighi&author=R%20Ryu&author=F%20Akhlaghi&volume=58&publication_year=2019&pages=1281-94&pmid=30972694&doi=10.1007/s40262-019-00750-8&)

2. Bell GC et al. Clinical Pharmacogenetics Implementation Consortium (CPIC) guideline for CYP2D6 genotype and use of ondansetron and tropisetron. Clin Pharmacol Ther 102, 213–8 (2017).  [DOI](https://doi.org/10.1002/cpt.598) | [PMC free article](/articles/PMC5479760/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28002639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20guideline%20for%20CYP2D6%20genotype%20and%20use%20of%20ondansetron%20and%20tropisetron.&author=GC%20Bell&volume=102&publication_year=2017&pages=213-8&pmid=28002639&doi=10.1002/cpt.598&)

3. Crews KR et al. Clinical Pharmacogenetics Implementation Consortium Guideline for CYP2D6, OPRM1, and COMT Genotypes and Select Opioid Therapy. Clin Pharmacol Ther 110, 888–96 (2021).  [DOI](https://doi.org/10.1002/cpt.2149) | [PMC free article](/articles/PMC8249478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33387367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20Guideline%20for%20CYP2D6,%20OPRM1,%20and%20COMT%20Genotypes%20and%20Select%20Opioid%20Therapy.&author=KR%20Crews&volume=110&publication_year=2021&pages=888-96&pmid=33387367&doi=10.1002/cpt.2149&)

4. Goetz MP et al. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for CYP2D6 and Tamoxifen Therapy. Clin Pharmacol Ther 103, 770–7 (2018).  [DOI](https://doi.org/10.1002/cpt.1007) | [PMC free article](/articles/PMC5931215/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29385237/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20Guideline%20for%20CYP2D6%20and%20Tamoxifen%20Therapy.&author=MP%20Goetz&volume=103&publication_year=2018&pages=770-7&pmid=29385237&doi=10.1002/cpt.1007&)

5. Hicks JK et al. Clinical Pharmacogenetics Implementation Consortium (CPIC) Guideline for CYP2D6 and CYP2C19 Genotypes and Dosing of Selective Serotonin Reuptake Inhibitors. Clin Pharmacol Ther 98, 127–34 (2015).  [DOI](https://doi.org/10.1002/cpt.147) | [PMC free article](/articles/PMC4512908/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25974703/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20(CPIC)%20Guideline%20for%20CYP2D6%20and%20CYP2C19%20Genotypes%20and%20Dosing%20of%20Selective%20Serotonin%20Reuptake%20Inhibitors.&author=JK%20Hicks&volume=98&publication_year=2015&pages=127-34&pmid=25974703&doi=10.1002/cpt.147&)

6. Hicks JK et al. Clinical pharmacogenetics implementation consortium guideline (CPIC) for CYP2D6 and CYP2C19 genotypes and dosing of tricyclic antidepressants: 2016 update. Clin Pharmacol Ther 102, 37–44 (2017).  [DOI](https://doi.org/10.1002/cpt.597) | [PMC free article](/articles/PMC5478479/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27997040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20pharmacogenetics%20implementation%20consortium%20guideline%20(CPIC)%20for%20CYP2D6%20and%20CYP2C19%20genotypes%20and%20dosing%20of%20tricyclic%20antidepressants:%202016%20update.&author=JK%20Hicks&volume=102&publication_year=2017&pages=37-44&pmid=27997040&doi=10.1002/cpt.597&)

7. PharmGKB CYP2D6 Drug Label Annotations <https://www.pharmgkb.org/gene/PA128/labelAnnotation>.  [https://www.pharmgkb.org/gene/PA128/labelAnnotation](https://www.pharmgkb.org/gene/PA128/labelAnnotation)

8. Dalton R et al. Interrogation of CYP2D6 Structural Variant Alleles Improves the Correlation Between CYP2D6 Genotype and CYP2D6-Mediated Metabolic Activity. Clin Transl Sci 13, 147–56 (2020).  [DOI](https://doi.org/10.1111/cts.12695) | [PMC free article](/articles/PMC6951848/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31536170/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Sci&title=Interrogation%20of%20CYP2D6%20Structural%20Variant%20Alleles%20Improves%20the%20Correlation%20Between%20CYP2D6%20Genotype%20and%20CYP2D6-Mediated%20Metabolic%20Activity.&author=R%20Dalton&volume=13&publication_year=2020&pages=147-56&pmid=31536170&doi=10.1111/cts.12695&)

9. Fang Y et al. Intraindividual Variation and Correlation of Cytochrome P450 Activities in Human Liver Microsomes. Mol Pharm 15, 5312–8 (2018).  [DOI](https://doi.org/10.1021/acs.molpharmaceut.8b00787) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30346185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Pharm&title=Intraindividual%20Variation%20and%20Correlation%20of%20Cytochrome%20P450%20Activities%20in%20Human%20Liver%20Microsomes.&author=Y%20Fang&volume=15&publication_year=2018&pages=5312-8&pmid=30346185&doi=10.1021/acs.molpharmaceut.8b00787&)

10. Gaedigk A, Simon SD, Pearce RE, Bradford LD, Kennedy MJ & Leeder JS The CYP2D6 activity score: translating genotype information into a qualitative measure of phenotype. Clin Pharmacol Ther 83, 234–42 (2008).  [DOI](https://doi.org/10.1038/sj.clpt.6100406) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20CYP2D6%20activity%20score:%20translating%20genotype%20information%20into%20a%20qualitative%20measure%20of%20phenotype.&author=A%20Gaedigk&author=SD%20Simon&author=RE%20Pearce&author=LD%20Bradford&author=MJ%20Kennedy&volume=83&publication_year=2008&pages=234-42&pmid=17971818&doi=10.1038/sj.clpt.6100406&)

11. Montane Jaime LK, Lalla A, Steimer W & Gaedigk A Characterization of the CYP2D6 gene locus and metabolic activity in Indo- and Afro-Trinidadians: discovery of novel allelic variants. Pharmacogenomics 14, 261–76 (2013).  [DOI](https://doi.org/10.2217/pgs.12.207) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23394389/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Characterization%20of%20the%20CYP2D6%20gene%20locus%20and%20metabolic%20activity%20in%20Indo-%20and%20Afro-Trinidadians:%20discovery%20of%20novel%20allelic%20variants.&author=LK%20Montane%20Jaime&author=A%20Lalla&author=W%20Steimer&author=A%20Gaedigk&volume=14&publication_year=2013&pages=261-76&pmid=23394389&doi=10.2217/pgs.12.207&)

12. Ning M, Duarte JD, Rubin LH & Jeong H CYP2D6 Protein Level Is the Major Contributor to Interindividual Variability in CYP2D6-Mediated Drug Metabolism in Healthy Human Liver Tissue. Clin Pharmacol Ther 104, 974–82 (2018).  [DOI](https://doi.org/10.1002/cpt.1032) | [PMC free article](/articles/PMC6053340/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29349771/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=CYP2D6%20Protein%20Level%20Is%20the%20Major%20Contributor%20to%20Interindividual%20Variability%20in%20CYP2D6-Mediated%20Drug%20Metabolism%20in%20Healthy%20Human%20Liver%20Tissue.&author=M%20Ning&author=JD%20Duarte&author=LH%20Rubin&author=H%20Jeong&volume=104&publication_year=2018&pages=974-82&pmid=29349771&doi=10.1002/cpt.1032&)

13. Gaedigk A, Dinh JC, Jeong H, Prasad B & Leeder JS Ten Years’ Experience with the CYP2D6 Activity Score: A Perspective on Future Investigations to Improve Clinical Predictions for Precision Therapeutics. J Pers Med 8, (2018).  [DOI](https://doi.org/10.3390/jpm8020015) | [PMC free article](/articles/PMC6023391/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29673183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pers%20Med&title=Ten%20Years%E2%80%99%20Experience%20with%20the%20CYP2D6%20Activity%20Score:%20A%20Perspective%20on%20Future%20Investigations%20to%20Improve%20Clinical%20Predictions%20for%20Precision%20Therapeutics.&author=A%20Gaedigk&author=JC%20Dinh&author=H%20Jeong&author=B%20Prasad&author=JS%20Leeder&volume=8&publication_year=2018&pmid=29673183&doi=10.3390/jpm8020015&)

14. Wang D, Papp AC & Sun X Functional characterization of CYP2D6 enhancer polymorphisms. Hum Mol Genet 24, 1556–62 (2015).  [DOI](https://doi.org/10.1093/hmg/ddu566) | [PMC free article](/articles/PMC4381757/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25381333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Functional%20characterization%20of%20CYP2D6%20enhancer%20polymorphisms.&author=D%20Wang&author=AC%20Papp&author=X%20Sun&volume=24&publication_year=2015&pages=1556-62&pmid=25381333&doi=10.1093/hmg/ddu566&)

15. Wang D, Poi MJ, Sun X, Gaedigk A, Leeder JS & Sadee W Common CYP2D6 polymorphisms affecting alternative splicing and transcription: long-range haplotypes with two regulatory variants modulate CYP2D6 activity. Hum Mol Genet 23, 268–78 (2014).  [DOI](https://doi.org/10.1093/hmg/ddt417) | [PMC free article](/articles/PMC3857955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23985325/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Common%20CYP2D6%20polymorphisms%20affecting%20alternative%20splicing%20and%20transcription:%20long-range%20haplotypes%20with%20two%20regulatory%20variants%20modulate%20CYP2D6%20activity.&author=D%20Wang&author=MJ%20Poi&author=X%20Sun&author=A%20Gaedigk&author=JS%20Leeder&volume=23&publication_year=2014&pages=268-78&pmid=23985325&doi=10.1093/hmg/ddt417&)

16. Ray B, Ozcagli E, Sadee W & Wang D CYP2D6 haplotypes with enhancer single-nucleotide polymorphism rs5758550 and rs16947 (*2 allele): implications for CYP2D6 genotyping panels. Pharmacogenet Genomics 29, 39–47 (2019).  [DOI](https://doi.org/10.1097/FPC.0000000000000363) | [PMC free article](/articles/PMC6441960/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30520769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=CYP2D6%20haplotypes%20with%20enhancer%20single-nucleotide%20polymorphism%20rs5758550%20and%20rs16947%20(*2%20allele):%20implications%20for%20CYP2D6%20genotyping%20panels.&author=B%20Ray&author=E%20Ozcagli&author=W%20Sadee&author=D%20Wang&volume=29&publication_year=2019&pages=39-47&pmid=30520769&doi=10.1097/FPC.0000000000000363&)

17. Sanchez-Spitman AB, Moes DA, Gelderblom H, Dezentje VO, Swen JJ & Guchelaar HJ The effect of rs5758550 on CYP2D6*2 phenotype and formation of endoxifen in breast cancer patients using tamoxifen. Pharmacogenomics 18, 1125–32 (2017).  [DOI](https://doi.org/10.2217/pgs-2017-0080) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28745555/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=The%20effect%20of%20rs5758550%20on%20CYP2D6*2%20phenotype%20and%20formation%20of%20endoxifen%20in%20breast%20cancer%20patients%20using%20tamoxifen.&author=AB%20Sanchez-Spitman&author=DA%20Moes&author=H%20Gelderblom&author=VO%20Dezentje&author=JJ%20Swen&volume=18&publication_year=2017&pages=1125-32&pmid=28745555&doi=10.2217/pgs-2017-0080&)

18. Zanger UM, Momoi K, Hofmann U, Schwab M & Klein K Tri-Allelic Haplotypes Determine and Differentiate Functionally Normal Allele CYP2D6*2 and Impaired Allele CYP2D6*41. Clin Pharmacol Ther 109, 1256–64 (2021).  [DOI](https://doi.org/10.1002/cpt.2078) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33043448/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Tri-Allelic%20Haplotypes%20Determine%20and%20Differentiate%20Functionally%20Normal%20Allele%20CYP2D6*2%20and%20Impaired%20Allele%20CYP2D6*41.&author=UM%20Zanger&author=K%20Momoi&author=U%20Hofmann&author=M%20Schwab&author=K%20Klein&volume=109&publication_year=2021&pages=1256-64&pmid=33043448&doi=10.1002/cpt.2078&)

19. Thomas CD et al. Examination of Metoprolol Pharmacokinetics and Pharmacodynamics Across CYP2D6 Genotype-Derived Activity Scores. CPT Pharmacometrics Syst Pharmacol 9, 678–85 (2020).  [DOI](https://doi.org/10.1002/psp4.12563) | [PMC free article](/articles/PMC7762806/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33067866/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CPT%20Pharmacometrics%20Syst%20Pharmacol&title=Examination%20of%20Metoprolol%20Pharmacokinetics%20and%20Pharmacodynamics%20Across%20CYP2D6%20Genotype-Derived%20Activity%20Scores.&author=CD%20Thomas&volume=9&publication_year=2020&pages=678-85&pmid=33067866&doi=10.1002/psp4.12563&)

20. Boone EC et al. Long-Distance Phasing of a Tentative “Enhancer” Single-Nucleotide Polymorphism With CYP2D6 Star Allele Definitions. Frontiers in pharmacology 11, 486 (2020).  [DOI](https://doi.org/10.3389/fphar.2020.00486) | [PMC free article](/articles/PMC7226225/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32457600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20pharmacology&title=Long-Distance%20Phasing%20of%20a%20Tentative%20%E2%80%9CEnhancer%E2%80%9D%20Single-Nucleotide%20Polymorphism%20With%20CYP2D6%20Star%20Allele%20Definitions.&author=EC%20Boone&volume=11&publication_year=2020&pages=486&pmid=32457600&doi=10.3389/fphar.2020.00486&)

21. Gaedigk A et al. Identification and characterization of novel sequence variations in the cytochrome P4502D6 (CYP2D6) gene in African Americans. The Pharmacogenomics Journal 5, 173–82 (2005).  [DOI](https://doi.org/10.1038/sj.tpj.6500305) | [PMC free article](/articles/PMC1440720/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15768052/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Pharmacogenomics%20Journal&title=Identification%20and%20characterization%20of%20novel%20sequence%20variations%20in%20the%20cytochrome%20P4502D6%20(CYP2D6)%20gene%20in%20African%20Americans.&author=A%20Gaedigk&volume=5&publication_year=2005&pages=173-82&pmid=15768052&doi=10.1038/sj.tpj.6500305&)

22. Gaedigk A, Bradford LD, Alander SW & Leeder JS CYP2D6*36 gene arrangements within the cyp2d6 locus: association of CYP2D6*36 with poor metabolizer status. Drug Metab Dispos 34, 563–9 (2006).  [DOI](https://doi.org/10.1124/dmd.105.008292) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16415111/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=CYP2D6*36%20gene%20arrangements%20within%20the%20cyp2d6%20locus:%20association%20of%20CYP2D6*36%20with%20poor%20metabolizer%20status.&author=A%20Gaedigk&author=LD%20Bradford&author=SW%20Alander&author=JS%20Leeder&volume=34&publication_year=2006&pages=563-9&pmid=16415111&doi=10.1124/dmd.105.008292&)

23. Gaedigk A, Fuhr U, Johnson C, Berard LA, Bradford D & Leeder JS CYP2D7–2D6 hybrid tandems: identification of novel CYP2D6 duplication arrangements and implications for phenotype prediction. Pharmacogenomics 11, 43–53 (2010).  [DOI](https://doi.org/10.2217/pgs.09.133) | [PMC free article](/articles/PMC6556886/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20017671/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CYP2D7%E2%80%932D6%20hybrid%20tandems:%20identification%20of%20novel%20CYP2D6%20duplication%20arrangements%20and%20implications%20for%20phenotype%20prediction.&author=A%20Gaedigk&author=U%20Fuhr&author=C%20Johnson&author=LA%20Berard&author=D%20Bradford&volume=11&publication_year=2010&pages=43-53&pmid=20017671&doi=10.2217/pgs.09.133&)

24. Gaedigk A et al. Identification of Novel CYP2D7–2D6 Hybrids: Non-Functional and Functional Variants. Frontiers in Pharmacology 1, 121 (2010).  [DOI](https://doi.org/10.3389/fphar.2010.00121) | [PMC free article](/articles/PMC3153001/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21833166/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Pharmacology&title=Identification%20of%20Novel%20CYP2D7%E2%80%932D6%20Hybrids:%20Non-Functional%20and%20Functional%20Variants.&author=A%20Gaedigk&volume=1&publication_year=2010&pages=121&pmid=21833166&doi=10.3389/fphar.2010.00121&)

25. Gaedigk A et al. Cytochrome P4502D6 (CYP2D6) gene locus heterogeneity: characterization of gene duplication events. Clin Pharmacol Ther 81, 242–51 (2007).  [DOI](https://doi.org/10.1038/sj.clpt.6100033) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17259947/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Cytochrome%20P4502D6%20(CYP2D6)%20gene%20locus%20heterogeneity:%20characterization%20of%20gene%20duplication%20events.&author=A%20Gaedigk&volume=81&publication_year=2007&pages=242-51&pmid=17259947&doi=10.1038/sj.clpt.6100033&)

26. Gaedigk A, Twist GP & Leeder JS CYP2D6, SULT1A1 and UGT2B17 copy number variation: quantitative detection by multiplex PCR. Pharmacogenomics 13, 91–111 (2012).  [DOI](https://doi.org/10.2217/pgs.11.135) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22111604/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CYP2D6,%20SULT1A1%20and%20UGT2B17%20copy%20number%20variation:%20quantitative%20detection%20by%20multiplex%20PCR.&author=A%20Gaedigk&author=GP%20Twist&author=JS%20Leeder&volume=13&publication_year=2012&pages=91-111&pmid=22111604&doi=10.2217/pgs.11.135&)

27. Gaedigk A et al. Characterization of Reference Materials for Genetic Testing of CYP2D6 Alleles: A GeT-RM Collaborative Project. J Mol Diagn 21, 1034–52 (2019).  [DOI](https://doi.org/10.1016/j.jmoldx.2019.06.007) | [PMC free article](/articles/PMC6854474/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31401124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Diagn&title=Characterization%20of%20Reference%20Materials%20for%20Genetic%20Testing%20of%20CYP2D6%20Alleles:%20A%20GeT-RM%20Collaborative%20Project.&author=A%20Gaedigk&volume=21&publication_year=2019&pages=1034-52&pmid=31401124&doi=10.1016/j.jmoldx.2019.06.007&)

28. PharmVar CYP2D6 gene page <https://www.pharmvar.org/gene/CYP2D6>  [https://www.pharmvar.org/gene/CYP2D6](https://www.pharmvar.org/gene/CYP2D6)

29. Stephens M, Smith NJ & Donnelly P A new statistical method for haplotype reconstruction from population data. Am J Hum Genet 68, 978–89 (2001).  [DOI](https://doi.org/10.1086/319501) | [PMC free article](/articles/PMC1275651/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11254454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=A%20new%20statistical%20method%20for%20haplotype%20reconstruction%20from%20population%20data.&author=M%20Stephens&author=NJ%20Smith&author=P%20Donnelly&volume=68&publication_year=2001&pages=978-89&pmid=11254454&doi=10.1086/319501&)

30. Gaedigk A, Riffel AK & Leeder JS CYP2D6 Haplotype Determination Using Long Range Allele-Specific Amplification: Resolution of a Complex Genotype and a Discordant Genotype Involving the CYP2D6*59 Allele. J Mol Diagn 17, 740–8 (2015).  [DOI](https://doi.org/10.1016/j.jmoldx.2015.06.007) | [PMC free article](/articles/PMC4630174/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26335396/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Diagn&title=CYP2D6%20Haplotype%20Determination%20Using%20Long%20Range%20Allele-Specific%20Amplification:%20Resolution%20of%20a%20Complex%20Genotype%20and%20a%20Discordant%20Genotype%20Involving%20the%20CYP2D6*59%20Allele.&author=A%20Gaedigk&author=AK%20Riffel&author=JS%20Leeder&volume=17&publication_year=2015&pages=740-8&pmid=26335396&doi=10.1016/j.jmoldx.2015.06.007&)

31. Chladek J, Zimova G, Martinkova J & Tuma I Intra-individual variability and influence of urine collection period on dextromethorphan metabolic ratios in healthy subjects. Fundam Clin Pharmacol 13, 508–15 (1999).  [DOI](https://doi.org/10.1111/j.1472-8206.1999.tb00011.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10456294/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Fundam%20Clin%20Pharmacol&title=Intra-individual%20variability%20and%20influence%20of%20urine%20collection%20period%20on%20dextromethorphan%20metabolic%20ratios%20in%20healthy%20subjects.&author=J%20Chladek&author=G%20Zimova&author=J%20Martinkova&author=I%20Tuma&volume=13&publication_year=1999&pages=508-15&pmid=10456294&doi=10.1111/j.1472-8206.1999.tb00011.x&)

32. Ozdemir M, Crewe KH, Tucker GT & Rostami-Hodjegan A Assessment of in vivo CYP2D6 activity: differential sensitivity of commonly used probes to urine pH. J Clin Pharmacol 44, 1398–404 (2004).  [DOI](https://doi.org/10.1177/0091270004269582) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15545311/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Assessment%20of%20in%20vivo%20CYP2D6%20activity:%20differential%20sensitivity%20of%20commonly%20used%20probes%20to%20urine%20pH.&author=M%20Ozdemir&author=KH%20Crewe&author=GT%20Tucker&author=A%20Rostami-Hodjegan&volume=44&publication_year=2004&pages=1398-404&pmid=15545311&doi=10.1177/0091270004269582&)

33. Huang W, Nakano M, Sager J, Ragueneau-Majlessi I & Isoherranen N Physiologically Based Pharmacokinetic Model of the CYP2D6 Probe Atomoxetine: Extrapolation to Special Populations and Drug-Drug Interactions. Drug Metab Dispos 45, 1156–65 (2017).  [DOI](https://doi.org/10.1124/dmd.117.076455) | [PMC free article](/articles/PMC5637815/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28860113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Physiologically%20Based%20Pharmacokinetic%20Model%20of%20the%20CYP2D6%20Probe%20Atomoxetine:%20Extrapolation%20to%20Special%20Populations%20and%20Drug-Drug%20Interactions.&author=W%20Huang&author=M%20Nakano&author=J%20Sager&author=I%20Ragueneau-Majlessi&author=N%20Isoherranen&volume=45&publication_year=2017&pages=1156-65&pmid=28860113&doi=10.1124/dmd.117.076455&)

34. Center for Drug Evaluation and Research. B4L-LC-HFMJ, B4Z-LC-LYAE, and B4Z-LC-LYAM (2002).

35. Jukic MM, Smith RL, Molden E & Ingelman-Sundberg M Evaluation of the CYP2D6 Haplotype Activity Scores Based on Metabolic Ratios of 4,700 Patients Treated With Three Different CYP2D6 Substrates. Clin Pharmacol Ther 110, 750–8 (2021).  [DOI](https://doi.org/10.1002/cpt.2246) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33792048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Evaluation%20of%20the%20CYP2D6%20Haplotype%20Activity%20Scores%20Based%20on%20Metabolic%20Ratios%20of%204,700%20Patients%20Treated%20With%20Three%20Different%20CYP2D6%20Substrates.&author=MM%20Jukic&author=RL%20Smith&author=E%20Molden&author=M%20Ingelman-Sundberg&volume=110&publication_year=2021&pages=750-8&pmid=33792048&doi=10.1002/cpt.2246&)

36. Frederiksen T, Areberg J, Schmidt E, Stage TB & Brosen K Cytochrome P450 2D6 genotype-phenotype characterization through population pharmacokinetic modeling of tedatioxetine. CPT Pharmacometrics Syst Pharmacol 10, 983–93 (2021).  [DOI](https://doi.org/10.1002/psp4.12635) | [PMC free article](/articles/PMC8452298/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33932135/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CPT%20Pharmacometrics%20Syst%20Pharmacol&title=Cytochrome%20P450%202D6%20genotype-phenotype%20characterization%20through%20population%20pharmacokinetic%20modeling%20of%20tedatioxetine.&author=T%20Frederiksen&author=J%20Areberg&author=E%20Schmidt&author=TB%20Stage&author=K%20Brosen&volume=10&publication_year=2021&pages=983-93&pmid=33932135&doi=10.1002/psp4.12635&)

37. Frederiksen T, Areberg J, Schmidt E, Bjerregaard Stage T & Brosen K Quantification of In Vivo Metabolic Activity of CYP2D6 Genotypes and Alleles Through Population Pharmacokinetic Analysis of Vortioxetine. Clin Pharmacol Ther 109, 150–9 (2021).  [DOI](https://doi.org/10.1002/cpt.1972) | [PMC free article](/articles/PMC7818194/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32599653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Quantification%20of%20In%20Vivo%20Metabolic%20Activity%20of%20CYP2D6%20Genotypes%20and%20Alleles%20Through%20Population%20Pharmacokinetic%20Analysis%20of%20Vortioxetine.&author=T%20Frederiksen&author=J%20Areberg&author=E%20Schmidt&author=T%20Bjerregaard%20Stage&author=K%20Brosen&volume=109&publication_year=2021&pages=150-9&pmid=32599653&doi=10.1002/cpt.1972&)
