# The effect of CHRNA3 rs1051730 C>T and ABCB1 rs3842 A>G polymorphisms on non-small cell lung cancer and nicotine dependence in Iranian population

## Metadata
**Authors:** Parisa Veiskarami, Massoud Houshmand, Sharareh Seifi, Nafiseh Ansarinejad, Farshid Fardad, Bahareh Abbasi
**Journal:** Heliyon
**Date:** 2021 Aug 24
**DOI:** [10.1016/j.heliyon.2021.e07867](https://doi.org/10.1016/j.heliyon.2021.e07867)
**PMID:** 34522797
**PMCID:** PMC8426517
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8426517/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8426517/pdf/main.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8426517/pdf/main.pdf)

## Abstract

**Aims:** 
Lung cancer is still the leading cause of cancer mortality in all over the world. Nicotine and its derivatives are the most well-known carcinogens that participate in both etiology and progression of lung cancer. The objective of the current study was to investigate whether single nucleotide polymorphisms (SNPs) rs1051730C > T in CHRNA3 and rs3842A > G in ABCB1, two genes contributing in the mechanism of disposition and metabolism of nicotine and its derivatives, could modify the risk of developing lung cancer, as well as nicotine dependence in Iranian.

**Main methods:** 
The genotyping analysis for these two SNPs was conducted in a case-control study of 108 lung cancer cases and 120 healthy controls using ARMS-PCR and Tetra-primer ARMS-PCR techniques. The correlation between studied SNPs and lung cancer was assessed by the regression analysis.

**Key findings:** 
We observed a significant association between lung cancer and rs1051730C > T by using four genetic models: allele (OR:1.83; 95% CI:1.24–2.6; p = 0.002), dominant (OR: 2.19; 95% CI:1.27–3.78; p = 0.005), recessive (OR: 2.25; 95% CI: 1.02–4.95; p = 0.043) and additive (TT vs CC: OR:3.25; 95% CI:1.38–7.60; p = 0.007, CT vs CC: OR:1.96; 95% CI:1.10–3.48; p = 0.021). Furthermore, a significant association between this variant and nicotine dependence (OR: 2.27; 95% CI: 1.52–3.39; p = 0.00005) was reported. However, no association was found for rs3842A > G.

**Significance:** 
The results suggested that the CHRNA3 rs1051730C > T via a smoking-dependent manner could modify susceptibility to lung cancer among Iranian population.

Keywords: Lung cancer, Nicotine dependence, Polymorphism, CHRNA3 rs1051730 C>T, ABCB1 rs3842 A>G

### Aims

Lung cancer is still the leading cause of cancer mortality in all over the world. Nicotine and its derivatives are the most well-known carcinogens that participate in both etiology and progression of lung cancer. The objective of the current study was to investigate whether single nucleotide polymorphisms (SNPs) rs1051730C > T in CHRNA3 and rs3842A > G in ABCB1, two genes contributing in the mechanism of disposition and metabolism of nicotine and its derivatives, could modify the risk of developing lung cancer, as well as nicotine dependence in Iranian.

### Main methods

The genotyping analysis for these two SNPs was conducted in a case-control study of 108 lung cancer cases and 120 healthy controls using ARMS-PCR and Tetra-primer ARMS-PCR techniques. The correlation between studied SNPs and lung cancer was assessed by the regression analysis.

### Key findings

We observed a significant association between lung cancer and rs1051730C > T by using four genetic models: allele (OR:1.83; 95% CI:1.24–2.6; p = 0.002), dominant (OR: 2.19; 95% CI:1.27–3.78; p = 0.005), recessive (OR: 2.25; 95% CI: 1.02–4.95; p = 0.043) and additive (TT vs CC: OR:3.25; 95% CI:1.38–7.60; p = 0.007, CT vs CC: OR:1.96; 95% CI:1.10–3.48; p = 0.021). Furthermore, a significant association between this variant and nicotine dependence (OR: 2.27; 95% CI: 1.52–3.39; p = 0.00005) was reported. However, no association was found for rs3842A > G.

### Significance

The results suggested that the CHRNA3 rs1051730C > T via a smoking-dependent manner could modify susceptibility to lung cancer among Iranian population.

## 1. Introduction

According to the global statistics, lung cancer (LC) is the leading cancer incidence and mortality with 2.1 million new cases and 1.8 million cause of cancer death throughout the world [[1](#bib1)]. LC is the third most prevalent cancer and second cause of cancer death among Iranian (Age-standardized incidence rates (ASR) per 100.000 for LC is12.5 in males and 5.5 in females. Age-standardized mortality rates (ASR) per 100.000 for LC is 9.1 in males and 8.3 in females) [[2](#bib2)]. Non-small cell lung cancer (NSCLC) is the most common type of LC accounts for approximately 80% of all cases and is classified into three subtypes: adenocarcinoma (ADC), squamous cell carcinoma (SCC), and large cell carcinoma (LCC) [[3](#bib3)].

LC is frequently mentioned as a disease attributed to environmental factors more than the other multiple risk factors of this malignancy [[4](#bib4)]. Tobacco smoking as an environmental factor is the predominant cause of LC [[5](#bib5)]. Approximately, more than 60 carcinogens have been identified in tobacco smoke. Nicotine and nicotine derivatives such as 4-(methylnitrosamino)-1-(3-pyridyl)-1-butanone (NNK) are the most important tobacco smoke carcinogens, which could promote proliferation, metastasis and inhibit apoptosis of LC cells [[6](#bib6)]. The majority of studies proved that over 80% of LC occurrence is associated with direct and indirect tobacco consumption. Nonetheless, only 20% of which develop LC during their life, that indicates inter-individual difference in the rate of genetic variation, may be associated with lung carcinogenesis. Single nucleotide polymorphisms (SNPs) are the most common and stable markers of human genetic variation, and may be related to the risk of a variety of cancers including LC [[7](#bib7)]. Many SNP sites in association with LC were obtained by the genome–wide association studies (GWAS) [[8](#bib8)].

Several GWAS and case-control studies have identified the chromosome 15q25.1 locus as a susceptibility region for smoking behavior, nicotine dependence (ND) and LC in some populations [[9](#bib9)]. This locus contains a cluster of three nicotinic acetylcholine receptors (nAChRs) including CHRNA3, CHRNA5, and CHRNB4 [[10](#bib10)]. SNP, rs1051730C > T which is located in the exon5 of CHRNA3 (Cholinergic Receptor Nicotinic Alpha 3 Subunit) and causes a synonymous nucleotide substitution, is shown to be significantly associated with increased risk of ND and LC [[11](#bib11)].

In addition, there are many other SNPs that are associated with LC, but their effect has not been studied as much as the SNPs on chromosome 15q52.1 region such as rs3842A > G located in 3′ untranslated region (UTR) of ABCB1 (ATP Binding Cassette Subfamily B Member 1) [[12](#bib12)].

Considering the association of these two SNPs with LC in a different population, we assume this association is likely to exist the same in Iranian. Hence, in the present case-control study, we investigated the allele frequency of CHRNA3 rs1051730C > T and ABCB1 rs3842A > G and assessed their effect on the susceptibility to NSCLC risk in Iranian population.

## 2. Materials and methods

### 2.1. Subjects

In this study, a total of 108 NSCLC cases and 120 healthy controls of genetically unrelated Iranian were enrolled from Masih Daneshvari Hospital, Tehran, Iran, between December 2018 and December 2019. Patients had histopathologically confirmed NSCLC and exclusion criteria included previous cancer history. Control participants were frequency-matched to the cases according to age, gender and residential area, and exclusion criteria included a personal and familial history of cancer and lung-related diseases to avoid probable interference from overlapping genes. Written informed consent was received from each participant and Iran National Institute of Genetic Engineering and Biotechnology Ethics Committee approved this study (IR.NIGEB.EC.1398.6.24 C). All the subjects were given a short questionnaire by personal interview including age, sex, smoking behavior and ethnicity.

### 2.2. Genotyping

An approximately 5-mL peripheral blood sample was collected from each participant in vacuum tubes with EDTA. Genomic DNA was extracted using a blood DNA extraction kit (Sinacolon, Iran) according to the manufacturer's instruction. SNP genotyping was conducted by Amplification Refractory Mutation System-Polymerase Chain Reaction (ARMS-PCR) and Tetra-primer ARMS-PCR techniques for ABCB1 rs3842A > G and CHRNA3 rs1051730C > T respectively. The primers were designed by the Oligo Analyzer software ([Table 1](#tbl1)). PCR was performed in a total volume of 15μL, containing 1μL DNA, 0.5μL of each primer, 4.5μL deionized water and 7.5μL Tag DNA Polymerase 2x Master Mix RED. The reaction program was set in a thermal cycler (Eppendorf, Germany) to an initial denaturation of 5 min at 95 °C followed by 30 cycles of 30 s at 95 °C, 30 s at 64 °C for Tetra-primer ARMS-PCR and 61 °C for ARMS-PCR, 30 s at 72 °C, and then a final extension of 5 min at 72 °C. The amplified products were separated by electrophoresis using 2% agarose gel. Ten percent of samples were randomly selected to perform repeated assays for each SNP, and the results were 100% concordant. We also randomly selected 10% of the samples for direct sequencing to confirm the results, and the results were also 100% concordant.

### Table 1.

| SNP | Primer | Primer sequence | TM (°C) |
| --- | --- | --- | --- |
| rs1051730C > T | Outer Forward (FO) | AATCGACGTGACCTACTTCCCGTTT | 64.2 |
| Outer Reverse (RO) | AGTGATCACCAGGAGAAACACCGTC | 64.4 |  |
| Inner Forward (FI) | ATCATCAAAGCCCCAGGCGAC | 63.1 |  |
| Inner Reverse (RI) | GCAGCAGTTGTACTTGATGTCGTGTGTA | 65.5 |  |
| rs3842A > G | Outer forward (FO) | TGCAGACTTAATAGTGGTGTTTCA | 60.1 |
| Outer Reverse (RO) | AAGATATATTCACAGGCAGTTTGG | 60.1 |  |
| Normal Inner Forward (FN) | ACATCATCAAGTGGAGAGAAATCA | 60.1 |  |
| Mutant Inner Forward (FM) | ACATCATCAAGTGGAGAGAAATCG | 61.8 |  |

Table 1 Caption: Sequence of primers used for CHRNA3 rs1051730C > T and ABCB1 rs3842A > G genotyping.

### 2.3. Statistical analysis

Differences in demographic characteristics and selected variables among LC patients and healthy control subjects were evaluated using Student's t-test for continuous quantitative variables or Pearson's chi-square test for nominal variables. Pearson's chi-square test was also used to assess the genotype and allele frequency and check whether genetic frequencies are accorded with Hardy-Weinberg equilibrium (HWE) or not. Odds ratio (OR) with 95% confidence interval (CI) was used to assess the association strength. The OR was calculated for estimating the outcome of four genetic models, allele, dominant and recessive and additive by using binary logistic regression and adjusted for age, sex, and smoking status. A P-value less than 0.05 were considered significant. All of the statistical analyses were performed with Statistical Analysis System software (SPSS 24.0 Inc., Chicago, IL).

## 3. Results

### 3.1. Subject characteristics

The distribution according to age, gender and smoking status among all individuals were analyzed ([Table 2](#tbl2)). No significant differences in the distribution of age and sex were found among cancer patients and control subjects (P > 0.05), suggesting that subject matching based on these variables was adequate. However, compared with the control population, the cases were more likely to be smokers (63% in cases vs 47% in controls; OR: 1.87, 95% CI: 1.10–3.19, P < 0.05).

### Table 2.

| Variable | Case (n = 108) | Control (n = 120) | P value |
| --- | --- | --- | --- |
| Age |  |  | 0.96 |
| Mean ± SD | 59.40 ± 7.62 | 58.60 ± 10.49 |   |
| Sex n (%) |  |  | 0.96 |
| Female | 33 (30.5) | 37 (30.8) |   |
| Male | 75 (69.4) | 83 (69.1) |   |
| Smoking status n (%) |  |  | 0.02 |
| Yes | 68 (62.9) | 57 (47.5) |   |
| No | 40 (37.1) | 63 (52.5) |   |
| Histologic type n (%) |  |  |  |
| Adenocarcinoma (AD) | 94 (87.1) |   |   |
| Squamous cell carcinoma (SCC) | 14 (12.9) |   |   |

Table 2 Caption: General characteristics of cases and controls.

### 3.2. Tetra-primer ARMS-PCR

Tetra-primer ARMS-PCR products of three samples including, Heterozygote CT, Homozygote CC, and Homozygote TT, are shown in [Figure 1](#fig1). The 394bp, control band defined by the FO-RO primers, amplify in all samples. The 249bp, C allele-specific band defined by the FI-RO primers, amplify just in Homozygote CC samples. The 193bp, T allele-specific band which defined by the FI-RO primers, amplify just in Homozygote TT samples. Heterozygote CT samples have all of these three bands.

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2c2d/8426517/5d68fa648220/gr1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8426517_gr1.jpg)

Genotyping of CHRNA3 rs1051730C > T by Tetra-primer ARMS-PCR. 1: Negative control, 2: Heterozygote CT, 3: Homozygote CC, 4: Homozygote TT, 5: 100bp DNA Ladder.

### 3.3. ARMS-PCR

ARMS-PCR products of three samples that include, Heterozygote AG, Homozygote AA, and Homozygote GG, are shown in [Figure 2](#fig2). The 546bp, control band defined by the FO-RO primers, amplify in all samples. The 250bp, A allele-specific band defined by the FN-RO primers, amplify just in Homozygote AA samples. The 250bp, G allele-specific band defined by the FM-RO primers, amplify just in Homozygote GG samples. Heterozygote AG samples have all of these three bands.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2c2d/8426517/7fc36592e03e/gr2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8426517_gr2.jpg)

Genotyping of ABCB1 rs3842A > G by ARMS-PCR. 1: Negative control, 2: Heterozygote AG, 3: Homozygote AA, 4: Homozygote GG, 5: 100bp DNA Ladder.

### 3.4. Genotype and allele frequency distribution of SNPs

The genotype and allele frequency of rs1051730C > T and rs3842A > G are summarized in [Table 3](#tbl3). The observed genotype frequencies of SNPs were in HWE, both in controls and cases (p > 0.05).

### Table 3.

| SNP | Case n (%) | Control n (%) |
| --- | --- | --- |
| rs1051730C > T |  |  |
| C/C | 33 (30.6%) | 59 (49.2%) |
| C/T | 55 (50.9%) | 50 (41.7%) |
| T/T | 20 (18.5%) | 11 (9.2%) |
| C allele | 121 (56.0%) | 168 (70.0%) |
| T allele | 95 (44%) | 72 (30.0%) |
| rs3842A > G |  |  |
| A/A | 65 (60.2%) | 77 (64.2%) |
| A/G | 37 (34.3%) | 38 (31.7%) |
| G/G | 6 (5.6%) | 5 (4.2%) |
| A allele | 167 (77.3%) | 192 (80%) |
| G allele | 49 (22.7%) | 48 (20%) |

Table 3 Caption: Genotype and allele frequency of CHRNA3 rs1051730C > T and ABCB1 rs3842A > G among cases and controls.

### 3.5. Association of SNPs with lung cancer and nicotine dependence

A significant association between CHRNA3 rs1051730C > T and increased risk of LC were detected under four genetic models involving, dominant, recessive, additive and allele. However, no significant association between ABCB1 rs3842A > G and LC were detected under none of these genetic models ([Table 4](#tbl4)). Also we divided the population (combined patients and controls) into two groups, smokers (>10 cigarettes per day, recently cut down from 20) and never-smokers (never more than 100 cigarettes during their lifetime), then obtained the frequency of C (34.4%, 47.6%) and T (65.6%, 52.4%) alleles of rs1051730 and A (80.0%, 77.2%) and G (20.0%, 22.8%) alleles of rs3842 between this two groups respectively. Furthermore, we found that there is a significant association between rs1051730 and increased risk of ND under three genetic models, dominant, recessive, additive (TT vs CC) and allelic. However, no significant association between ABCB1 rs3842A > G and ND were detected under none of these genetic models ([Table 5](#tbl5)).

### Table 4.

|   | OR Crude (95%CI) | P-value | OR Adjusted (95%CI) | P-value |
| --- | --- | --- | --- | --- |
| rs1051730C > T |  |  |  |  |
| Dominant (TT + CT vs CC) | 2.19 (1.27–3.78) | 0.005 | 2.06 (1.19–3.58) | 0.010 |
| Recessive (TT vs CT + CC) | 2.25 (1.02–4.95) | 0.043 | 1.74 (0.75–4.03) | 0.195 |
| Co-dominant (CT vs CC + TT) | 1.45 (0.86–2.45) | 0.162 | 1.59 (0.93–2.72) | 0.089 |
| Additive (TT vs CC) | 3.25 (1.38–7.60) | 0.007 | 2.78 (1.06–7.23) | 0.036 |
| (CT vs CC) | 1.96 (1.10–3.48) | 0.021 | 1.96 (1.10–3.50) | 0.021 |
| Allelic (T vs C) | 1.83 (1.24–2.69) | 0.002 | 1.64 (1.10–2.43) | 0.014 |
| rs3842A > G |  |  |  |  |
| Dominant (GG + AG vs AA) | 1.18 (0.69–2.02) | 0.536 | 1.25 (0.72–2.16) | 0.419 |
| Recessive (GG vs AG + AA) | 1.35 (0.40–4.56) | 0.626 | 1.28 (0.37–4.39) | 0.689 |
| Co-dominant (AG vs AA + GG) | 1.25 (0.64–1.95) | 0.677 | 1.20 (0.68–2.11) | 0.514 |
| Additive (GG vs AA) | 1.42 (0.41–4.87) | 0.576 | 1.37 (0.39–4.80) | 0.617 |
| (AG vs AA) | 1.53 (0.65–2.02) | 0.618 | 1.24 (0.70–2.21) | 0.449 |
| Allelic (G vs A) | 1.17 (0.74–1.83) | 0.484 | 0.82 (0.52–1.29) | 0.396 |

Table 4 Caption: Association of CHRNA3 rs1051730C > T and ABCB1 rs3842A > G with the risk of LC in Iranian.

### Table 5.

|   | OR Crude (95%CI) | P-value |
| --- | --- | --- |
| rs1051730C > T |  |  |
| Dominant (TT + CT vs CC) | 1.73 (1.01–2.95) | 0.044 |
| Recessive (TT vs CT + CC) | 32.21 (4.30–240.84) | 0.001 |
| Co-dominant (CT vs CC + TT) | 0.67 (0.39–1.13) | 0.137 |
| Additive (TT vs CC) | 34.18 (4.47–261.34) | 0.001 |
| (CT vs CC) | 1.11 (0.63–1.95) | 0.696 |
| Allelic (T vs C) | 2.27 (1.52–3.39) | 0.00005 |
| rs3842A > G |  |  |
| Dominant (GG + AG vs AA) | 0.67 (0.39–1.16) | 0.158 |
| Recessive (GG vs AG + AA) | 1.46 (0.41–5.16) | 0.549 |
| Co-dominant (AG vs AA + GG) | 0.61 (0.35–1.06) | 0.084 |
| Additive (GG vs AA) | 1.24 (0.34–4.44) | 0.737 |
| (AG vs AA) | 0.62 (0.35–1.09) | 0.098 |
| Allelic (G vs A) | 0.84 (0.53–1.32) | 0.464 |

Table 5 Caption: Association of CHRNA3 rs1051730C > T and ABCB1 rs3842A > G with the risk of nicotine dependence in Iranian.

## 4. Discussion

Since the first GWAS study [[13](#bib13)] which identified CHRNA3 rs1051730C > T as a polymorphism is related to ND and LC in European ancestry population, extensive subsequent studies in different populations reported the same results [[9](#bib9), [14](#bib14), [15](#bib15)].

A comprehensive meta-analysis study demonstrated that rs1051730-T allele might be risk-conferring factor for the development of LC in European ancestry (OR = 1.31; 95% CI = 1.26–1.37; P < 10^−5^, OR = 1.33; 95% CI = 1.21–1.43; P < 10^−5^, OR = 1.35; 95% CI = 1.22–1.47; P < 10^−5^ under allele, dominant and recessive model respectively) and in African-Americans (OR = 1.75; 95% CI = 1.38–2.21; P < 10^−5^, OR = 1.87; 95% CI = 1.45–2.33; P < 10^−4^, OR = 2.00; 95% CI = 1.51–2.66; P < 10^−4^ under allele, dominant and recessive model respectively) [[16](#bib16)]. Also, a preceding meta-analysis study was conducted with similar results [[17](#bib17)]. However, most studies in Chinese population show no significant association [[7](#bib7), [18](#bib18), [19](#bib19), [20](#bib20)] and just a few studies have reported a weak association (OR = 2.18; 95% CI = 1.02–4.67; P = 0.045, under allele model) [[21](#bib21)]. Also, In Japanese population, only a weak association was observed (OR = 2.57, 95% CI = 1.03–6.87, P = 0.049 under dominant model) [[22](#bib22)] and rs1051730 T allele confer susceptibility to LC just in a small subgroup of Japanese [[23](#bib23)]^.^ In present study as mentioned in results, we demonstrated that rs1051730 C > T is candidates of risk factor for ND and LC in Iranian.

The CHRNA3 is a gene which encode a subunit of nAChRs. The nAChRs which bind to nicotine and nicotine-derived nitrosamines such as NNK, are expressed on lung epithelial cells and neurons in the brain region. The binding in lung cell modulates downstream signal transduction through nAChRs, leading to cell proliferation and neoplastic transformation, while the binding in brain cell can potentially control smoking behavior and ND [[21](#bib21)]. Therefore, we claim that this SNP is related to LC probably indirectly through smoking carcinogens. Although, there is debate whether SNPs in the CHRNA3 have a direct genetic effect for susceptibility to LC.

To sum up, our results are consistent with those from the European and African-Americans and inconsistent whit East-Asian populations. There are some points concerning such different results between different populations, which above all, it may be on account of disequilibrium of rs1051730 T allele in different populations. The worldwide frequency of the rs1051730 T allele, varies extensively between different races, with a prevalence of ~35 % among European ancestry, ~16 % among African-Americans, ~3 % among Chinese, and ~2% among Japanese. Moreover, in this study, we found a frequency of ~30% in Iranian. It is conceivable that the rs1051730 T allele frequency in the East-Asian population is too low to confirm the effects seen in other populations [[16](#bib16)].

The other SNP, rs3842A > G is in ABCB1, which encode the P-glycoprotein (P-gp) expressed in a range of tissues, including lung. P-gp play an important role in absorption, distribution, metabolism and excretion of a multitude of structurally and functionally unrelated substrates such as nicotine and its derivatives especially NNK, in addition to its original role in multidrug resistance in chemotherapeutics. By playing this important role, P-gp contribute significantly in maintaining cellular homeostasis and protect tissues against xenobiotic and many other carcinogen compounds. Hence, it may influence susceptibility, occurrence, development and histopathological features of LC [[24](#bib24), [25](#bib25), [26](#bib26), [27](#bib27)]. However, in the present study, we observed no association between rs3842A > G and LC, as well as ND in Iranian.

To the best of the author's knowledge, a study by Wang et al., in the Chinese population is the only case-control study that focused on rs3842A > G which demonstrated that it might contribute to the etiology of LC because of its strong association with increased risk of LC manifested in women (OR, 2.57; 95% CI, 1.36–4.85; P = 0.003), particularly for the histological type of adenocarcinoma (OR, 1.42; 95% CI, 1.03–1.99; P = 0.034), and for individuals aged over 60 (OR, 1.50; 95% CI, 1.05–2.14; P = 0.025). Also, study indicates this SNP as a non-coding polymorphism in the 3′ UTR of ABCB1 may be associated with LC through a possible miRNA-mediated mechanism [[12](#bib12)], because rs3842A > G lies at a conserved region, very close to the potential targeting stem for has-miR-374, which also is expressed in lung tissue.

So, our results are entirely in contradiction to the impact of this SNP in Chinese population. The contradiction between our results and those from Chinese in both rs3842A > G and rs1051730C > T, gives strength to the idea of that marker SNPs in Chinese are not reliable candidates to be examined in Iranian. Probably because of the disequilibrium of allele frequency between Chinese and Iranian or because of different environmental factors which effect each of these populations.

There are some limitations in our study that may have influenced our obtained results. First, the sample size and statistical power are relatively small. Hence, more extensive studies are required in the future to confirm our findings. Second, we focused only on two polymorphisms, rs1051730C > T and rs3842A > G and did not consider other gene candidates or polymorphisms specifically those that may be in linked with the studied polymorphisms. Further combination analysis might provide higher power to characterize high-risk populations. Third, except smoking, we did not consider the other important environmental factors. The unconsidered mixed factors may cover the role of these polymorphisms. Thus, even if the variation has a causal effect on LC, it may take a long time to be observed. Fourth, Precise data regarding nicotine equivalents through measuring urinary biomarkers, were not available; therefore direct associations of risk alleles with tobacco consumption were not confirmed.

## 5. Conclusion

Despite these potential limitations, the present study suggests that ARMS-PCR and Tetra-primer ARMS-PCR, the used techniques in this study are considered to be easy, reliable, fast and economical techniques for genotyping. In addition, they demonstrate the blood-based r1051730C > T increase the risk of LC in an Iranian population, probably indirectly through smoking behavior. Hence, this variant besides smoking status can be considered as markers which allow the monitoring of vulnerable populations for the purpose of LC screening. Future studies with large samples illustrate whether the association rs1051730C > T with ND and LC risk in Iranian population to be causal or by chance. Also, it could deepen our understanding of genetic polymorphism attributing to the progression of malignancy and pave the way for future clinical applications, as well as creating more efficient anti-smoking programs, that are capable of raising the awareness of the population to reduce long-term risks.

## Declarations

### Author contribution statement

Bahareh Abbasi, Massoud Houshmand: Conceived and designed the experiments.

Parisa Veiskarami: Conceived and designed the experiments; Performed the experiments; Analyzed and interpreted the data; Wrote the paper.

Sharareh Seifi, Nafiseh Ansarinejad, Farshid Fardad: Contributed reagents, materials, analysis tools or data.

### Funding statement

This work was supported by the Iran National Institute of Genetic Engineering and Biotechnology [F. NO. 622].

### Data availability statement

Data will be made available on request.

### Declaration of interests statement

The authors declare no conflict of interest.

### Additional information

No additional information is available for this paper.

## Acknowledgements

The authors thank all the participants whose contributions made this project possible.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

Data will be made available on request.

### Data Availability Statement

Data will be made available on request.

## References

1. Bray F., Ferlay J., Soerjomataram I., Siegel R.L., Torre L.A., Jemal A. Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. CA Cancer J. Clin. 2018;68(6):394–424. doi: 10.3322/caac.21492.  [DOI](https://doi.org/10.3322/caac.21492) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30207593/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J.%20Clin.&title=Global%20cancer%20statistics%202018:%20GLOBOCAN%20estimates%20of%20incidence%20and%20mortality%20worldwide%20for%2036%20cancers%20in%20185%20countries&author=F.%20Bray&author=J.%20Ferlay&author=I.%20Soerjomataram&author=R.L.%20Siegel&author=L.A.%20Torre&volume=68&issue=6&publication_year=2018&pages=394-424&pmid=30207593&doi=10.3322/caac.21492&)

2. Zendehdel K. Cancer statistics in I.R. Iran in 2018. Basic Clin. Canc. Res. 2019;11(1):1–4.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin.%20Canc.%20Res.&title=Cancer%20statistics%20in%20I.R.%20Iran%20in%202018&author=K.%20Zendehdel&volume=11&issue=1&publication_year=2019&pages=1-4&)

3. Moreira A.L., Mino-Kenudson M. Update on histologic classification of non-small cell lung cancer. Diagn. Histopathol. 2014;20(10):385–391.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Diagn.%20Histopathol.&title=Update%20on%20histologic%20classification%20of%20non-small%20cell%20lung%20cancer&author=A.L.%20Moreira&author=M.%20Mino-Kenudson&volume=20&issue=10&publication_year=2014&pages=385-391&)

4. Trovo de Marqui A., Bonfim V., Torreglosa-Ruiz M. Genetic polymorphisms in the nicotinic receptors and lung cancer: an overview. Brazil. J. Cancerol. 2011;57(3):421–428. http://www.inca.gov.br/rbc/n_57/v03/pdf_en/16_revisao_literatura_polimorfismos_geneticos_receptores_nicotinicos_cancer_pulmao_visao_geral.pdf  [http://www.inca.gov.br/rbc/n_57/v03/pdf_en/16_revisao_literatura_polimorfismos_geneticos_receptores_nicotinicos_cancer_pulmao_visao_geral.pdf](http://www.inca.gov.br/rbc/n_57/v03/pdf_en/16_revisao_literatura_polimorfismos_geneticos_receptores_nicotinicos_cancer_pulmao_visao_geral.pdf) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brazil.%20J.%20Cancerol.&title=Genetic%20polymorphisms%20in%20the%20nicotinic%20receptors%20and%20lung%20cancer:%20an%20overview&author=A.%20Trovo%20de%20Marqui&author=V.%20Bonfim&author=M.%20Torreglosa-Ruiz&volume=57&issue=3&publication_year=2011&pages=421-428&)

5. Sun H., Pan Y., He B. Different effects of the three polymorphisms on 15q25.1 onlung cancer risk: evidence from published literatures. J. Cancer Res. Therapeut. 2016;12(1):12–19. doi: 10.4103/0973-1482.151863.  [DOI](https://doi.org/10.4103/0973-1482.151863) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27072204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Cancer%20Res.%20Therapeut.&title=Different%20effects%20of%20the%20three%20polymorphisms%20on%2015q25.1%20onlung%20cancer%20risk:%20evidence%20from%20published%20literatures&author=H.%20Sun&author=Y.%20Pan&author=B.%20He&volume=12&issue=1&publication_year=2016&pages=12-19&pmid=27072204&doi=10.4103/0973-1482.151863&)

6. Wen L., Jiang K., Yuan W., Cui W., Li M.D. Contribution of variants in CHRNA5/A3/B4 gene cluster on chromosome 15 to tobacco smoking: from genetic association to mechanism. Mol. Neurobiol. 2014;53(1):472–484. doi: 10.1007/s12035-014-8997-x.  [DOI](https://doi.org/10.1007/s12035-014-8997-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25471942/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Neurobiol.&title=Contribution%20of%20variants%20in%20CHRNA5/A3/B4%20gene%20cluster%20on%20chromosome%2015%20to%20tobacco%20smoking:%20from%20genetic%20association%20to%20mechanism&author=L.%20Wen&author=K.%20Jiang&author=W.%20Yuan&author=W.%20Cui&author=M.D.%20Li&volume=53&issue=1&publication_year=2014&pages=472-484&pmid=25471942&doi=10.1007/s12035-014-8997-x&)

7. He P., Yang X.-X., He X.-Q. CHRNA3 polymorphism modifies lung adenocarcinoma risk in the Chinese han population. Int. J. Mol. Sci. 2014;15(4):5446–5457. doi: 10.3390/ijms15045446.  [DOI](https://doi.org/10.3390/ijms15045446) | [PMC free article](/articles/PMC4013574/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24686516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=CHRNA3%20polymorphism%20modifies%20lung%20adenocarcinoma%20risk%20in%20the%20Chinese%20han%20population&author=P.%20He&author=X.-X.%20Yang&author=X.-Q.%20He&volume=15&issue=4&publication_year=2014&pages=5446-5457&pmid=24686516&doi=10.3390/ijms15045446&)

8. Han Z., Jiang Q., Zhang T. Analyzing large-scale samples confirms the association between the rs1051730 polymorphism and lung cancer susceptibility. Sci. Rep. 2015;5(October):1–9. doi: 10.1038/srep15642.  [DOI](https://doi.org/10.1038/srep15642) | [PMC free article](/articles/PMC4623668/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26508385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Analyzing%20large-scale%20samples%20confirms%20the%20association%20between%20the%20rs1051730%20polymorphism%20and%20lung%20cancer%20susceptibility&author=Z.%20Han&author=Q.%20Jiang&author=T.%20Zhang&volume=5&issue=October&publication_year=2015&pages=1-9&pmid=26508385&doi=10.1038/srep15642&)

9. Ji X., Bossé Y., Landi M.T. Identification of susceptibility pathways for the role of chromosome 15q25.1 in modifying lung cancer risk. Nat. Commun. 2018;9(1):3221. doi: 10.1038/s41467-018-05074-y.  [DOI](https://doi.org/10.1038/s41467-018-05074-y) | [PMC free article](/articles/PMC6089967/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30104567/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Identification%20of%20susceptibility%20pathways%20for%20the%20role%20of%20chromosome%2015q25.1%20in%20modifying%20lung%20cancer%20risk&author=X.%20Ji&author=Y.%20Boss%C3%A9&author=M.T.%20Landi&volume=9&issue=1&publication_year=2018&pages=3221&pmid=30104567&doi=10.1038/s41467-018-05074-y&)

10. Kita-Milczarska K., Sieminska A., Jassem E. Association between CHRNA3 and CHRNA5 nicotine receptor subunit gene variants and nicotine dependence in an isolated populationof Kashubians in Poland. Med. Sci. Monit. 2016;22:1442–1450. doi: 10.12659/MSM.895907.  [DOI](https://doi.org/10.12659/MSM.895907) | [PMC free article](/articles/PMC4915318/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27127891/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med.%20Sci.%20Monit.&title=Association%20between%20CHRNA3%20and%20CHRNA5%20nicotine%20receptor%20subunit%20gene%20variants%20and%20nicotine%20dependence%20in%20an%20isolated%20populationof%20Kashubians%20in%20Poland&author=K.%20Kita-Milczarska&author=A.%20Sieminska&author=E.%20Jassem&volume=22&publication_year=2016&pages=1442-1450&pmid=27127891&doi=10.12659/MSM.895907&)

11. Kupiainen H., Kuokkanen M., Kontto J. CHRNA5/CHRNA3 locus associates with increased mortality among smokers. J. Chronic Obstr. Pulm. Dis. 2016;13(4):464–470. doi: 10.3109/15412555.2015.1049260.  [DOI](https://doi.org/10.3109/15412555.2015.1049260) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26751916/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Chronic%20Obstr.%20Pulm.%20Dis.&title=CHRNA5/CHRNA3%20locus%20associates%20with%20increased%20mortality%20among%20smokers&author=H.%20Kupiainen&author=M.%20Kuokkanen&author=J.%20Kontto&volume=13&issue=4&publication_year=2016&pages=464-470&pmid=26751916&doi=10.3109/15412555.2015.1049260&)

12. Wang H., Jin G., Wang H. Genetic susceptibility of lung cancer associated with common variants in the 3′ untranslated regions of the adenosine triphosphate-binding cassette B1 (ABCB1) and ABCC1 candidate transporter genes for carcinogen export. Cancer. 2009;115(3):595–607. doi: 10.1002/cncr.24042.  [DOI](https://doi.org/10.1002/cncr.24042) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19107762/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Genetic%20susceptibility%20of%20lung%20cancer%20associated%20with%20common%20variants%20in%20the%203%E2%80%B2%20untranslated%20regions%20of%20the%20adenosine%20triphosphate-binding%20cassette%20B1%20(ABCB1)%20and%20ABCC1%20candidate%20transporter%20genes%20for%20carcinogen%20export&author=H.%20Wang&author=G.%20Jin&author=H.%20Wang&volume=115&issue=3&publication_year=2009&pages=595-607&pmid=19107762&doi=10.1002/cncr.24042&)

13. Thorgeirsson T.E., Geller F., Sulem P. A variant associated with nicotine dependence, lung cancer and peripheral arterial disease. Nature. 2008;452(7187):638–642. doi: 10.1038/nature06846.  [DOI](https://doi.org/10.1038/nature06846) | [PMC free article](/articles/PMC4539558/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18385739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%C2%A0variant%20associated%20with%20nicotine%20dependence,%20lung%20cancer%20and%20peripheral%20arterial%20disease&author=T.E.%20Thorgeirsson&author=F.%20Geller&author=P.%20Sulem&volume=452&issue=7187&publication_year=2008&pages=638-642&pmid=18385739&doi=10.1038/nature06846&)

14. Stadler Z.K., Thom P., Robson M.E. Genome-wide association studies of cancer. J. Clin. Oncol. 2010;28(27):4255–4267. doi: 10.1200/JCO.2009.25.7816.  [DOI](https://doi.org/10.1200/JCO.2009.25.7816) | [PMC free article](/articles/PMC2953976/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20585100/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Clin.%20Oncol.&title=Genome-wide%20association%20studies%20of%20cancer&author=Z.K.%20Stadler&author=P.%20Thom&author=M.E.%20Robson&volume=28&issue=27&publication_year=2010&pages=4255-4267&pmid=20585100&doi=10.1200/JCO.2009.25.7816&)

15. Timofeeva M.N., Hung R.J., Rafnar T. Influence of common genetic variation on lung cancer risk: meta-analysis of 14 900 cases and 29 485 controls. Hum. Mol. Genet. 2012;21(22):4980–4995. doi: 10.1093/hmg/dds334.  [DOI](https://doi.org/10.1093/hmg/dds334) | [PMC free article](/articles/PMC3607485/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22899653/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Influence%20of%20common%20genetic%20variation%20on%20lung%20cancer%20risk:%20meta-analysis%20of%2014%20900%20cases%20and%2029%20485%20controls&author=M.N.%20Timofeeva&author=R.J.%20Hung&author=T.%20Rafnar&volume=21&issue=22&publication_year=2012&pages=4980-4995&pmid=22899653&doi=10.1093/hmg/dds334&)

16. Hu B., Huang Y., Yu R., Mao H., Guan C., Zhao J. Quantitative assessment of the influence of common variations (rs8034191 and rs1051730) at 15q25 and lung cancer risk. Tumor Biol. 2014;35(3):2777–2785. doi: 10.1007/s13277-013-1369-8.  [DOI](https://doi.org/10.1007/s13277-013-1369-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24254305/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumor%20Biol.&title=Quantitative%20assessment%20of%20the%20influence%20of%20common%20variations%20(rs8034191%20and%20rs1051730)%20at%2015q25%20and%20lung%20cancer%20risk&author=B.%20Hu&author=Y.%20Huang&author=R.%20Yu&author=H.%20Mao&author=C.%20Guan&volume=35&issue=3&publication_year=2014&pages=2777-2785&pmid=24254305&doi=10.1007/s13277-013-1369-8&)

17. Gu M., Dong X., Zhang X. Strong association between two polymorphisms on 15q25.1 and lung cancer risk: a meta-analysis. Gong Y., editor. PloS One. 2012;7(6) doi: 10.1371/journal.pone.0037970.  [DOI](https://doi.org/10.1371/journal.pone.0037970) | [PMC free article](/articles/PMC3368941/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22701590/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=Strong%20association%20between%20two%20polymorphisms%20on%2015q25.1%20and%20lung%20cancer%20risk:%20a%20meta-analysis&author=M.%20Gu&author=X.%20Dong&author=X.%20Zhang&volume=7&issue=6&publication_year=2012&pmid=22701590&doi=10.1371/journal.pone.0037970&)

18. Wu C., Hu Z., Yu D. Genetic variants on chromosome 15q25 associated with lung cancer risk in Chinese populations. Cancer Res. 2009;69(12):5065–5072. doi: 10.1158/0008-5472.CAN-09-0081.  [DOI](https://doi.org/10.1158/0008-5472.CAN-09-0081) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19491260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=Genetic%20variants%20on%20chromosome%2015q25%20associated%20with%20lung%20cancer%20risk%20in%20Chinese%20populations&author=C.%20Wu&author=Z.%20Hu&author=D.%20Yu&volume=69&issue=12&publication_year=2009&pages=5065-5072&pmid=19491260&doi=10.1158/0008-5472.CAN-09-0081&)

19. Yang L., Qiu F., Lu X. Functional polymorphisms of CHRNA3 predict risks of chronic obstructive pulmonary disease and lung cancer in Chinese. Reddy R., editor. PloS One. 2012;7(10) doi: 10.1371/journal.pone.0046071.  [DOI](https://doi.org/10.1371/journal.pone.0046071) | [PMC free article](/articles/PMC3463594/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23056235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=Functional%20polymorphisms%20of%20CHRNA3%20predict%20risks%20of%20chronic%20obstructive%20pulmonary%20disease%20and%20lung%20cancer%20in%20Chinese&author=L.%20Yang&author=F.%20Qiu&author=X.%20Lu&volume=7&issue=10&publication_year=2012&pmid=23056235&doi=10.1371/journal.pone.0046071&)

20. Li H., Yang L., Zhao X. Prediction of lung cancer risk in a Chinese population using a multifactorial genetic model. BMC Med. Genet. 2012;13(1):1. doi: 10.1186/1471-2350-13-118.  [DOI](https://doi.org/10.1186/1471-2350-13-118) | [PMC free article](/articles/PMC3573944/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23228068/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med.%20Genet.&title=Prediction%20of%20lung%20cancer%20risk%20in%20a%20Chinese%20population%20using%20a%20multifactorial%20genetic%20model&author=H.%20Li&author=L.%20Yang&author=X.%20Zhao&volume=13&issue=1&publication_year=2012&pages=1&pmid=23228068&doi=10.1186/1471-2350-13-118&)

21. Ren J., Jin M., He W. Association between CHRNA3 rs1051730 genotype and lung cancer risk in Chinese Han population: a case-control study. J. Huazhong Univ. Sci. Technol. 2013;33(6):897–901. doi: 10.1007/s11596-013-1218-4.  [DOI](https://doi.org/10.1007/s11596-013-1218-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24337855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Huazhong%20Univ.%20Sci.%20Technol.&title=Association%20between%20CHRNA3%20rs1051730%20genotype%20and%20lung%20cancer%20risk%20in%20Chinese%20Han%20population:%20a%20case-control%20study&author=J.%20Ren&author=M.%20Jin&author=W.%20He&volume=33&issue=6&publication_year=2013&pages=897-901&pmid=24337855&doi=10.1007/s11596-013-1218-4&)

22. Kohno T., Kunitoh H., Mimaki S. Contribution of the TP53, OGG1, CHRNA3, and HLA-DQA1 genes to the risk for lung squamous cell carcinoma. J. Thorac. Oncol. 2011;6(4):813–817. doi: 10.1097/JTO.0b013e3181ee80ef.  [DOI](https://doi.org/10.1097/JTO.0b013e3181ee80ef) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21623257/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Thorac.%20Oncol.&title=Contribution%20of%20the%20TP53,%20OGG1,%20CHRNA3,%20and%20HLA-DQA1%20genes%20to%20the%20risk%20for%20lung%20squamous%20cell%20carcinoma&author=T.%20Kohno&author=H.%20Kunitoh&author=S.%20Mimaki&volume=6&issue=4&publication_year=2011&pages=813-817&pmid=21623257&doi=10.1097/JTO.0b013e3181ee80ef&)

23. Shiraishi K., Kohno T., Kunitoh H. Contribution of nicotine acetylcholine receptor polymorphisms to lung cancer risk in a smoking-independent manner in the Japanese. Carcinogenesis. 2009;30(1):65–70. doi: 10.1093/carcin/bgn257.  [DOI](https://doi.org/10.1093/carcin/bgn257) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19005185/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Contribution%20of%20nicotine%20acetylcholine%20receptor%20polymorphisms%20to%20lung%20cancer%20risk%20in%20a%20smoking-independent%20manner%20in%20the%20Japanese&author=K.%20Shiraishi&author=T.%20Kohno&author=H.%20Kunitoh&volume=30&issue=1&publication_year=2009&pages=65-70&pmid=19005185&doi=10.1093/carcin/bgn257&)

24. Scheffer G.L., Pijnenborg A.C.L.M., Smit E.F. Multidrug resistance related molecules in human and murine lung. J. Clin. Pathol. 2002;55(5):332–339. doi: 10.1136/jcp.55.5.332.  [DOI](https://doi.org/10.1136/jcp.55.5.332) | [PMC free article](/articles/PMC1769658/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11986335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Clin.%20Pathol.&title=Multidrug%20resistance%20related%20molecules%20in%20human%20and%20murine%20lung&author=G.L.%20Scheffer&author=A.C.L.M.%20Pijnenborg&author=E.F.%20Smit&volume=55&issue=5&publication_year=2002&pages=332-339&pmid=11986335&doi=10.1136/jcp.55.5.332&)

25. van der Deen M., de Vries E.G.E., Timens W., Scheper R.J., Timmer-Bosscha H., Postma D.S. ATP-binding cassette (ABC) transporters in normal and pathological lung. Respir. Res. 2005;6(1):59. doi: 10.1186/1465-9921-6-59.  [DOI](https://doi.org/10.1186/1465-9921-6-59) | [PMC free article](/articles/PMC1200430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15967026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Respir.%20Res.&title=ATP-binding%20cassette%20(ABC)%20transporters%20in%20normal%20and%20pathological%20lung&author=M.%20van%20der%20Deen&author=E.G.E.%20de%20Vries&author=W.%20Timens&author=R.J.%20Scheper&author=H.%20Timmer-Bosscha&volume=6&issue=1&publication_year=2005&pages=59&pmid=15967026&doi=10.1186/1465-9921-6-59&)

26. Hodges L.M., Markova S.M., Chinn L.W. Very important pharmacogene summary. Pharmacogenetics Genom. 2011;21(3):152–161. doi: 10.1097/FPC.0b013e3283385a1c.  [DOI](https://doi.org/10.1097/FPC.0b013e3283385a1c) | [PMC free article](/articles/PMC3098758/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20216335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics%20Genom.&title=Very%20important%20pharmacogene%20summary&author=L.M.%20Hodges&author=S.M.%20Markova&author=L.W.%20Chinn&volume=21&issue=3&publication_year=2011&pages=152-161&pmid=20216335&doi=10.1097/FPC.0b013e3283385a1c&)

27. Shou W., Wang D., Zhang K. Gene-wide characterization of common quantitative trait loci for ABCB1 mRNA expression in normal liver tissues in the Chinese population. Huang Q., editor. PloS One. 2012;7(9) doi: 10.1371/journal.pone.0046295.  [DOI](https://doi.org/10.1371/journal.pone.0046295) | [PMC free article](/articles/PMC3458811/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23050008/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=Gene-wide%20characterization%20of%20common%20quantitative%20trait%20loci%20for%20ABCB1%20mRNA%20expression%20in%20normal%20liver%20tissues%20in%20the%20Chinese%20population&author=W.%20Shou&author=D.%20Wang&author=K.%20Zhang&volume=7&issue=9&publication_year=2012&pmid=23050008&doi=10.1371/journal.pone.0046295&)
