# The functional role of inherited CDKN2A variants in childhood acute lymphoblastic leukemia

## Metadata
**Authors:** Chunjie Li, Xinying Zhao, Yingyi He, Ziping Li, Jiabi Qian, Li Zhang, Qian Ye, Fei Qiu, Peng Lian, Maoxiang Qian, Hui Zhang
**Journal:** Pharmacogenetics and Genomics
**Date:** 2021 Aug 6
**DOI:** [10.1097/FPC.0000000000000451](https://doi.org/10.1097/FPC.0000000000000451)
**PMID:** 34369425
**PMCID:** PMC8694244
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8694244/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8694244/pdf/pgen-32-43.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8694244/pdf/pgen-32-43.pdf)

## Abstract

Supplemental Digital Content is available in the text.

Keywords: acute lymphoblastic leukemia, CDK4/6 inhibitors, CDKN2A, inherited variants, predispose

### Methods

We genotyped the *CDKN2A* coding region in 308 childhood ALL cases enrolled in CCCG-ALL-2015 clinical trials by Sanger Sequencing. Cell growth assay, cell cycle assay, MTT-based cell toxicity assay, and western blot were performed to assess the *CDKN2A* coding variants on ALL predisposition.

### Results

We identified 10 novel exonic germline variants, including 6 missense mutations (p.A21V, p.G45A and p.V115L of p16^INK4A^; p.T31R, p.R90G, and p.R129L of p14^ARF^) and 1 nonsense mutation and 1 heterozygous termination codon mutation in exon 2 (p16^INK4A^ p.S129X). Functional studies indicate that five novel variants resulted in reduced tumor suppressor activity of p16^INK4A^, and increased the susceptibility to the leukemic transformation of hematopoietic progenitor cells. Compared to other variants, p.H142R contributes higher sensitivity to CDK4/6 inhibitors.

### Conclusion

These findings provide direct insight into the influence of inherited genetic variants at the *CDKN2A* coding region on the development of ALL and the precise clinical application of CDK4/6 inhibitors.

## Introduction

Acute lymphoblastic leukemia (ALL) is the most common childhood cancer [[1](#R1),[2](#R2)]. Studies in the past decade have shown that inherited genetic variants (germline) are strongly associated with the predisposition to ALL in children. In particular, genome-wide association studies (GWAS) have identified susceptibility loci for B-cell ALL (B-ALL) in several genes, including *ARID5B*, *IKZF1*, *CEBPE*, *PIP4K2A*-*BMI1*, *GATA3*, *CDKN2A*/*2B*, *LHPP*, *ELK3*, *BAK1*, *IGF2BP1*, *USP7*, *IKZF3*, *ERG*, *TP63*, and *SP4* [[3](#R3)–[5](#R5)]. Most of these variants are intronic and may not be directly functional; however, more recently, a coding variant in *CDKN2A/2B* has been reported to account for influencing susceptibility to ALL in children [[6](#R6)].

*CDKN2A*/2B locus on 9p21 encodes for p16^INK4A^/p14^ARF^ and p15^INK4B^, respectively. Both p16^INK4A^ and p15^INK4B^ specifically inhibit cyclin/Cyclin-Dependent Kinase 4/6 (CDK4/6) complexes that block cell division during the G1/S phase of the cell cycle, whereas p14^ARF^ prevents degradation of p53 by interacting with the MDM2 protein [[7](#R7)–[9](#R9)]. Recently, the Sherr group has reported N-terminally truncated smArf, a distinct polypeptide encoded by *Arf* mRNA, localizes to mitochondria and triggers autophagy and mitophagy [[10](#R10)]. Thus, *CDKN2A*/*2B* is an important regulator of cell growth regulation and apoptosis, and loss of cell proliferation control and regulation of the cell cycle are known to be critical to cancer development [[11](#R11)–[13](#R13)]. Recent studies have reported the frequency of *CDKN2A*/*2B* deletion to be more than 30% in pediatric B-ALL, with relevance to ALL susceptibility and poor prognosis [[7](#R7),[8](#R8),[13](#R13)]. Sherborne *et al.* have demonstrated that common variation at 9p21.3 (rs3731217, intron 1 of *CDKN2A*) influences ALL risk (odds ratio = 0.71, *P* = 3.01 × 10^−11^), irrespective of cell lineage [[14](#R14)]. Xu *et al.* have functionally identified that *CDKN2A* single nucleotide polymorphism (SNP) rs3731249 (p.A148T, coding sequence of p16^INK4A^) significantly accelerates Ba/F3 cells leukemic transformation by BCR-ABL1, indicating that the reduced tumor suppressor function of p16^INK4A^ p.A148T variant [[6](#R6)]. However, questions remain whether other coding variants within *CDKN2A* gene might also contribute to ALL leukemogenesis.

In the present study, we genotyped *CDKN2A* exons in our CCCG-ALL-2015 study cohort to screen other exonic *CDKN2A* SNPs associated with ALL pathogenesis, experimentally explored the effects of exonic *CDKN2A* SNPs on leukemic transformation and their response to CDK4/6 inhibitors (palbociclib, ribociclib, and abemaciclib), and characterized the underlying mechanisms.

## Results

### Distinctive genomic regions at CDKN2A/2B loci associated with cancer and noncancer diseases/traits

To systemically explore the role of *CDKN2A*/*2B* loci in disease susceptibility, we retrieved all the GWAS hits from NHGRI-EBI Catalog and other ALL hits from Yang’s and Houlston’s groups [[6](#R6),[14](#R14)–[17](#R17)]. Totally 510 disease/traits associated SNPs records were collected (Supplementary Table 1, Supplemental digital content 1, [http://links.lww.com/FPC/B407](http://links.lww.com/FPC/B407)). Then, we performed regional association analysis and plotted the SNPs *P* value against ±1.6 mb flanked *CDKN2A/2B* loci by using Locus zoom ([http://locuszoom.org/](http://locuszoom.org/)) [[18](#R18)]. Among 27 diseases/traits analyzed, distinctive cancer and noncancer susceptibility SNPs pattern were observed (Fig. [1](#F1)). The cancer-associated (leukemia, melanoma, breast cancer, glioma, and basal cell carcinoma) SNPs were found to locate in proximal upstream of the *CDKN2A*/*2B* promoter region (chr9:22090000-22140000), while noncancer-associated SNPs were located distal upstream of the *CDKN2A*/*2B* promoter region (chr9:22030000-22080000), suggesting that different genomic regions in *CDKN2A/2B-AS1* loci responsible for distinctive diseases susceptibility. In regard to ALL, the susceptibility SNPs were resided in the cancer-associated genomic region, with rs3731217 top-ranked association in the *CDKN2A* intronic region, indicating the important role of *CDKN2A* in ALL development.

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4dc/8694244/0e5f198c9c9b/pgen-32-43-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8694244_pgen-32-43-g001.jpg)

GWAS Catalog associations for CDKN2A/2B loci plotted across Ch9:21.8–22.2 MB. Association results [−log10(P value)] for ALL susceptibility loci (a), susceptibility loci of other cancers, and susceptibility loci of nonmalignant conditions (b) are depicted with regards to the physical location of SNPs. (c) Lists of SNPs plotted in (b). SNP, single nucleotide polymorphism.

### Genotyping of CDKN2A in Chinese children with acute lymphoblastic leukemia

To further probe the role of *CDKN2A* exonic variations in childhood ALL susceptibility, we next sequenced the coding region of the *CDKN2A* in germline DNA from 308 childhood B-ALL cases enrolled onto CCCG-ALL-2015 clinical trials (ChiCTR-IPR-14005706) in Guangzhou Women and Children’s Medical Center (GWCMC, Table [1](#T1) and Fig. [2](#F2)a). We did not observe germline insertions or deletions at the *CDKN2A* locus, but identified ten novel germline exonic variants, including six missense mutations (p.A21V, p.G45A, and p.V115L of p16^INK4A^ coding sequence; p.T31R, p.R90G, and p.R129L in p14^ARF^ coding sequence) and one heterozygous termination codon mutation in exon 2 (p.S129X), resulting in the production of truncated p16^INK4A^ (Fig. [2](#F2)b and c and Supplementary Figure 1, Supplemental digital content 2, [http://links.lww.com/FPC/B408](http://links.lww.com/FPC/B408)). The frequency of ALL patients in our Han Chinese cohort harboring the *CDKN2A* germline mutation was 3.6% (11/308), and we did not identify p16^INK4A^ p.A148T variant, which occurred frequently in European descents (12.9%, 311/2407) [[6](#R6)]. Integrating the variants reported by Xu *et al.* [[6](#R6)], we noticed that most p16^INK4A^ variants are located in C-terminus, followed by ANK domains (Fig. [2](#F2)). Up to date, only the leukemic transformation potential of p16^INK4A^ p.A148T has been experimentally validated, while the function of other twelve exonic p16^INK4A^ variants has not been demonstrated by any model.

### Table 1.

| Characteristics | Group | No. patients (%) |
| --- | --- | --- |
| Age (years) | ≥1, <10 | 291 (94.5) |
|   | <1, ≥10 | 17 (5.5) |
| Gender | Female | 120 (39.0) |
|   | Male | 188 (61.0) |
| FAB subtype | L1 | 66 (21.4) |
|   | L2 | 179 (58.1) |
|   | L3 | 62 (20.1) |
|   | Not determined | 1 (0.3) |
| WBC (×109/L) | <50 | 257 (83.4) |
|   | ≥50 | 51 (16.6) |
| Liver | Not detected | 115 (37.3) |
|   | <2 cm | 27 (8.8) |
|   | ≥2, <5 cm | 139 (45.1) |
|   | ≥5 cm | 27 (8.8) |
| Spleen | Not detected | 172 (55.8) |
|   | <2 cm | 28 (9.1) |
|   | ≥2, <5 cm | 84 (27.3) |
|   | ≥5 cm | 24 (7.8) |
| NCI Risk | Standard | 246 (79.9) |
|   | High | 62 (20.1) |
| Risk stratified by CCCG-ALL-2015 | Low | 160 (51.9) |
|   | Intermediate | 146 (47.4) |
|   | High | 2 (0.6) |
| MRD19 | <0.01% | 107 (34.7) |
|   | ≥0.01% | 177 (57.5) |
| MRD46 | <0.01% | 256 (83.1) |
|   | ≥0.01% | 52 (16.9) |
| Relapse | No | 289 (93.8) |
|   | Yes | 19 (6.2) |

Table 1 Caption: Characteristics of enrolled patients from CCCG-ALL-2015 cohort

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4dc/8694244/c461de278f8e/pgen-32-43-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8694244_pgen-32-43-g002.jpg)

Germline coding variants of CDKN2A in children with ALL. (a) Flowchart of CDKN2A genotyping. CDKN2A Exon variants were identified by Sanger sequencing in 308 ALL cases enrolled onto CCCG-ALL-2015 in Guangzhou Women and Children’s Medical Center (GWCMC). (b) and (c), Exonic variants are classified as silent, missense or nonsense, and are mapped to two distinct open reading frames at this locus: p16INK4A (b) and p14ARF (c) for ALL cases from GWCMC (upper) and cases previously reported (lower) [6]. Functional domains are indicated by color based on Pfam annotation. Each circle represents a unique individual carrying the indicated variant (heterozygous or homozygous), except for variants recurring in more than two individuals for which the number in the circle indicates the exact frequency of the observed variant.

### Leukemic transformation potential of inherited p16INK4A variations

Building upon the analysis above, we next experimentally evaluated the effects of these exonic p16^INK4A^ variations on ALL leukemogenesis. To comprehensively evaluate the effects of these variants on leukemic transformation function, we used a mouse pro-B cell, Ba/F3 cells as our study model due to its inherently defective p16^Ink4a^, which significantly enhances the development of ALL induced by Bcr-Abl1 oncogenic fusion. Thus, we compared the effect of wildtype versus p16^INK4A^ variants (p.A21V, p.A36V, p.G45A, p.A57V, p.R99Q, p.V115L, p.D125H, p.A127S, p.S129X, p.R138G, p.H142R, p.A143T, and p.A148T) on BCR–ABL1-induced leukemic transformation *in vitro* (Supplementary Figure 2, Supplemental digital content 2, [http://links.lww.com/FPC/B408](http://links.lww.com/FPC/B408)). As shown in Fig. [3](#F3)a, ectopic expression of wild-type p16^INK4A^ significantly suppressed leukemic transformation induced by BCR-ABL1, while p.A148T significantly accelerated the leukemic transformation, consistent with the previous report [[6](#R6)]. Except for p.A148T variant, another four p16^INK4A^ missense variants (the leukemic transformation potential: p.H142R < p.A148T = p.A127S < p.R99Q < p.A36V) were capable of potentiating Ba/F3 cells IL3-independent growth by BCR-ABL1, suggesting that the likely reduced tumor suppressor function (Fig. [3](#F3)a). The p16^INK4A^ p.S129X resulted in a premature truncation and almost completely disrupted the function of the gene (Fig. [3](#F3)a).

### Fig. 3.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4dc/8694244/aa864f0e89ce/pgen-32-43-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8694244_pgen-32-43-g003.jpg)

Functional characterization of p16INK4A coding variants. (a) Cytokine-independent growth of Ba/F3 cells co-expressing wildtype, variant p16INK4A, or empty vector and leukemia oncogenic BCR–ABL1 fusion gene. Cell proliferation in the absence of cytokine IL3 was measured daily as an indicator of leukemic transformation. T-test was used to compare the cell numbers of the indicated cells with Ba/F3 cells expressing wildtype p16INK4A. (b) Cell cycle analysis of transformed Ba/f3 cells, expressing indicated variant p16INK4A or empty vector. Ba/F3 cells were fixed with 66% Ethanol and then stained with propidium iodine and RNase, followed by Flow cytometric and Flowjo analysis. (c) Cytotoxicity of Palbociclib towards Ba/F3 cells. Ba/F3 cells expressing the indicated vectors were treated with increasing concentrations of Palbociclib for 48 h before assessing viability using a CCK8 assay. (d) Cytotoxicity of Palbociclib towards Nalm6 cells. Nalm6 cells expressing empty vector, p16INK4A, or p16INK4A p.H142R, were treated with 2 μM Palbociclib for 72 h and then analyzed by CCK8 assay. All experiments were repeated twice in triplicate, and data represent the mean of three replicates ± SEM. Asterisks represent statistical significance (*P ≤ 0.05; **P ≤ 0.01; ***P ≤ 0.001). WT, wild type; EV, empty vector.

The p16^INK4A^ is a critical cyclin-dependent kinase inhibitor and cell cycle entry regulator. To test the impact of these functional p16^INK4A^ variants, we applied propidium iodide staining to evaluate the cell cycle distribution of BCR-ABL1-transformed Ba/F3 cells with different p16^INK4A^ variants expression. Though we could not compare our results with the wild type p16^INK4A^ due to its incapability of leukemic transformation, we observed an increase of S phase (25.9% ± 5.2%) in Ba/F3 cells with the p.H142R variant p16^INK4A^ as compared to 20.0% ± 4.9% S phase in Ba/F3 cells with p.A148T variant (*P* = 0.006), in consistent with the great transforming potential of p.H142R over p.A148T. The S phase percentage in Ba/F3 cells with p16^INK4A^ p.A127S was also higher than that in Ba/F3 cells with p16^INK4A^ p.A148T (*P* = 0.002). In the meanwhile, the S phase percentage in Ba/F3 cells ectopically overexpressed p16^INK4A^ p.R99Q and p16^INK4A^ p.A36V was lower than p16^INK4A^ p.A148T (Fig. [3](#F3)b), which was in line with their leukemic transformation potentials.

CDK4/6 inhibitors, which block the transition from the G1 to S phase of the cell cycle by interfering with Rb phosphorylation and E2F release, have shown potent antitumor activity and manageable toxicity in ALL patients *in vitro* [[19](#R19)–[21](#R21)]. Interestingly, a few clinical trials involving CDK4/6 inhibitors in relapsed/refractory ALL have been in progress (Supplementary Table 2, Supplemental digital content 1, [http://links.lww.com/FPC/B407](http://links.lww.com/FPC/B407)). The findings above prompted us to ask whether BCR-ABL1-induced leukemic Ba/F3 cells with p16^INK4A^ variants responded to CDK4/6 inhibitors. To test the cytotoxic effects of CDK4/6 inhibition, BCR-ABL1-transformed Ba/F3 cells were treated with increased concentrations of palbociclib. As shown in Fig. [3](#F3)c, BCR-ABL1-transformed Ba/F3 cells underwent apoptosis upon Palbociclib treatment in a dose-dependent fashion. As compared to Ba/F3 cells with p16^INK4A^ p.A148T, co-transduction of p16^INK4A^ p.H142R significantly potentiated response to palbociclib, in consistence with the leukemic transformation and cell cycle distribution results. Similar effects were also observed in another two CDK4/6 inhibitors, abemaciclib and ribociclib (Supplementary Figure 3, Supplemental digital content 2, [http://links.lww.com/FPC/B408](http://links.lww.com/FPC/B408)). To further confirm p16^INK4A^ mutation in human B-ALL cells, we established Nalm6 cells ectopically overexpressing p16^INK4A^ variants by lentiviral transfection and tested the effect of mutations on the response to CDK4/6 inhibitors. In accordance with the results for Ba/F3 cells, p16^INK4A^ p.H142R were more sensitive than p16^INK4A^ wild-type (Fig. [3](#F3)d). Taken together, these results point to the exonic p16^INK4A^ variants may potentiate ALL development and drug response to CDK4/6 inhibitors due to reduced tumor suppressor function.

### Structural basis of dysfunction of inherited p16INK4A variants

To address the impact of the variant protein, we performed immunoblotting assays to test the effects of p16^INK4A^ variants on CDK4/6-RB-E2F2 signaling pathway. Unexpectedly, we could not identify too many effects of the p.H142R and p.A148T variants on this signaling pathway as evidenced by unaltered phosphorylation level of CDK4 and Rb protein. (Supplementary Figure 4, Supplemental digital content 2, [http://links.lww.com/FPC/B408](http://links.lww.com/FPC/B408)). p16^INK4A^ is the prototype of a family of CDK inhibitors, specific for CDK4/6. Most observations suggest that p16^INK4A^ binds next to the ATP-binding site of CDK4/6, opposite where the activating cyclin subunit binds [[22](#R22)–[24](#R24)]. p16^INK4A^ prevents cyclin binding indirectly by causing structural changes that propagate to the cyclin-binding site [[24](#R24)]. p16^INK4A^ consists of four ankyrin repeats, consisting of ~30 amino acids. These ankyrin repeats stack to give an extended concave surface, binding to the N lobe of CKD4/6 [[24](#R24)]. Tumor-derived missense mutations in p16^INK4A^ affect its structural integrity, as has been demonstrated by studies of its stability and aggregation state [[25](#R25),[26](#R26)]. To elucidate the structural basis of the effects of the inherited p16^INK4A^ variants, structures were generated by homology modeling methods. p.A36V, p.R99Q, and p.A127S, which locate at ANK_1, ANK_3, and ANK_4, respectively, affect the binding of p16^INK4A^ with CDK4 (Fig. [4](#F4)). To this end, we could not reveal the causes of p.H142R and p.H148T with this model, since both variants are located to the C terminal, suggesting that some other hidden mechanisms existed.

### Fig. 4.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d4dc/8694244/bab5625dcd85/pgen-32-43-g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=8694244_pgen-32-43-g004.jpg)

The p16INK4A/CDK4 complex model and the superimposition with the crystal structure of CDK4/CyclinD. (a) p16INK4A and CDK4 from homology modeling were shown in yellow and red, respectively. Cyclin D in cyan and CDK4 in pink was from the crystal structure (PDB ID: 6P8G). The two complexes were superimposed according to the CDK4 proteins. Ankyrin domains in p16INK4A were numbered from 1 to 4 with starting from N terminus. p16INK4A variants, resulting in leukemic transformation were shown as Van der Waals spheres. p.A36V, p.R99Q, and p.A127S, are in ANK_1, ANK_3, and ANK_4 domains, respectively, and p.H142R and p.A148T in C-terminus of p16INK4A. (b) p16INK4A and CDK4 from homology modeling were shown in yellow and red, respectively. p18 in brown, CDK6 in gray, and Cyclin in blue were from crystal structure (PDB ID: 1G3N). The two complexes were superimposed according to the alignment of CDK4 and CDK6 proteins. Ankyrin domains in p18 were numbered from 1 to 5 with starting from N terminus. p.H142R and p.A148T are in C-terminus of p16INK4A, which equivalent to the Ank_5 domain of p18.

## Discussion

*CDKN2A*/*2B* is a well-established tumor suppressor gene, encoding p16^INK4A^, p15^INK5B^, and p14^ARF^. Loss of *CDKN2A/2B* function caused by genomic deletion, hypermethylation, and mutations are multi-dimensionally associated with cancer development, for example, cancer susceptibility carcinogenesis, prognosis, and treatment response [[27](#R27)–[29](#R29)]. In regard to cancer susceptibility, inherited *CDKN2A*/*2B* variants confer susceptibility not only to ALL, but also to glioma, basal cell carcinoma, and melanoma [[30](#R30)–[33](#R33)]. Though over 90% of diseases/traits-associated *CDKN2A*/*2B* variants are located in noncoding (intronic and/or intergenic) regions, a few studies have revealed that *CDKN2A* exonic variants also confer susceptibility to childhood ALL [[5](#R5),[6](#R6),[34](#R34)]. In this study, we report that the genomic region of *CDKN2A/2B* is responsible for cancer susceptibility, and systemically examine the role of inherited *CDKN2A* exonic variants in ALL development and their potentials.

Zhang *et al.* have identified a repressive element proximal to the ARF promoter, which is responsible for mediating p16^INK4A^ expression, suggesting that certain crucial genomic regions for *CDKN2A*/*2B* transcription [[35](#R35)]. To better decipher the pattern of diseases/traits-associated *CDKN2A*/*2B* SNPs, we utilized online tools to assess whether specific genomic regions affect different diseases/traits and found that the genomic region (chr9: 22090000-22140000) in *CDKN2A*/*2B* loci might be responsible for cancer susceptibility, while the distal region was more enriched for noncancer diseases/traits (e.g. coronary heart disease, diabetes, myocardial infarction, etc.) (Fig. [1](#F1)), suggesting a distinct functional role of *CDKN2A/2B* loci. It is perhaps that there is variability in the long-distance chromatin interactions and co-regulatory elements among these two genomic regions.

The p16^INK4A^ is a cyclin-dependent kinase inhibitor and regulates cell cycle entry via the Rb-E2F signaling [[36](#R36)]. p16^INK4A^ is suppressed during normal hematopoiesis, while it is activated by different oncogenic stimuli, for example, BCR-ABL1 fusion and PDGFRB-fusion [[37](#R37)–[39](#R39)]. Once activation, the impact of p16^INK4A^ on CDK4/6 would lead to cell cycle arrest at the G1 phase (senescence) in order to eliminate oncogene-stressed cells. Bi- or monoallelic *CDKN2A/2B* deletions were found in 64% of BCR–ABL1 positive ALL cases and in 32–72% of ALL cases without the *BCR-ABL1* translocation [[37](#R37)]. All these points to the role of defective p16^INK4A^, p14^ARF^, and p15^INK4B^ in leukemogenesis. Through *CDKN2A*/*2B* targeted sequencing, Xu *et al.* has identified that the inherited p16^INK4A^ p.A148T variant (rs3731249) is also strongly associated with ALL [[6](#R6)]. Functional assays have demonstrated that the p16^INK4A^ p.A148T variant is preferentially retained in B-ALL leukemic cells compared to its wild-type [[5](#R5),[6](#R6),[34](#R34)], suggesting the role of exonic SNPs in B-ALL risk. Except for rs3731249 (p16^INK4A^ p.A148T), we here systemically examined the impact of other p16^INK4A^ variants on ALL leukemogenesis and identified another five variants (p.A36V, p.R99Q, p.A127S, p.S129X, and p.H142R) with leukemogenic potential (Fig. [3](#F3)a). All these five variants could induce cell cycle entry from G1 to S phase, with most prominently in p.H142R variant. Leukemic transformed Ba/F3 cells with p.H142R were well responded to CDK4/6 inhibitors (palbociclib, ribociclib, and abemaciclib) (Fig. [3](#F3)c, and Supplementary Figure 3, Supplemental digital content 2, [http://links.lww.com/FPC/B408](http://links.lww.com/FPC/B408)), suggesting clinical translation potentials. Meanwhile, these data also indicated that patients with these variants may be susceptible to experience CDK4/6 inhibitor adverse drug reactions. The structural analysis revealed that p.A36V, p.R99Q, and p.A127S in ANK domains weakened the binding of p16^INK4A^ with CDK4, but could not explain the effect of p.H142R and p.A148T, since both of these two variants located in the C-terminus, which is not involved in the interaction (Fig. [4](#F4)).

### Conclusion

The systemic analysis of diseases/traits-associated *CDKN2A/2B* SNPs revealed distinctive genomic hotspots responsible for cancers or noncancer diseases. In the meanwhile, we systemically evaluated the impacts of inherited p16^INK4A^ exonic variants and identified four missense (p.A36V, p.R99Q, p.A127S, and p.H142R) and one nonsense (p.S129X) variants conferring ALL susceptibility. Among these five variants, p16^INK4A^ p.H142R demonstrated the greatest potential of ALL leukemogenesis and CDK4/6 inhibitor clinical translation. Our findings pinpoint the function of *CDKN2A/2B* loci in ALL leukemogenesis and targeted therapy.

## Materials and methods

### Patients

The patients were prospectively enrolled in the CCCG-2015-ALL clinical trial, which was approved by the institutional review board of the Guangzhou Women and Children Medical Center (GWCMC) (2018022205). Details of the enrollment criteria and study design have been described previously [[40](#R40)]. All the investigated pediatric ALL patients were treated in the GWCMC. This study was approved by the Institutional Ethics Committee of the GWCMC (IRB No.2018022205, 2017102307, 2015020936, and 2019-04700), registered at the Chinese Clinical Trial Registry (ChiCTR-IPR-14005706), and conducted in accordance with the Declaration of Helsinki. Informed consent was obtained from the patients or their legal guardians.

### Cell culture

HEK293T cells (ATCC, Rockefeller, Maryland, USA) were maintained in DMEM supplemented with 10% fetal bovine serum (FBS), and penicillin/streptomycin. Ba/F3 cells (gift from Dr Omar Abdel-Wahab at the Memorial Sloan Kettering Cancer Center, New York, USA) were maintained in RPMI1640 supplemented with 10% FBS, 10 ng/mL IL3, and penicillin/streptomycin. Nalm6 cells (ATCC) were maintained in RPMI1640 supplemented with 10% FBS, and penicillin/streptomycin.

### Virus transduction

*BCR-ABL1* p185 cDNA was amplified from MSCV–BCR–ABL1–Luc2 construct, a gift from Dr. Charles Sherr at St Jude Children’s Research Hospital, and cloned into Lenti-MCS-Blast lentiviral empty vector [[41](#R41)]. Full-length *p16*^INK4A^ was amplified and cloned into the cL20c-IRES-GFP lentiviral vector, and mutations were generated using the Q5 Site-Directed Mutagenesis Kit (New England Biolabs, Ipswich, Massachusetts, USA). Lentiviral supernatants were produced by transient transfection of HEK-293T cells using calcium phosphate. For transduction, Ba/F3 cells were co-transduced with lentiviral supernatants expressing *BCR-ABL1* and *p16*^INK4A^, followed by Blasticidin selection and fluorescence-activated cell sorting.

### Cytokine-independent growth assay in Ba/F3 cells

Cells were expanded after being washed three times and then plated at a density of 5 × 10^5^ cells/mL in non–IL3-containing medium. Viable cell counts were obtained using Trypan blue staining on TC20 Automated Cell Counter (Bio-Rad, Shanghai, China).

### Cytotoxicity assay

Cells were seeded in 96-well plates at 25 000 cells per 100 μL per well with either vehicle or increasing concentrations (0.0002, 0.002, 0.02, 0.2, 2, 20, and 200 µM) of drugs for 48–72 h. Cell viability was assessed by adding CCK8 (Cell Counting Kit-8, Dojindo, Kumamoto, Japan) reagent according to the manufacturer’s instructions. Procedures to determine the effects of certain conditions on cell proliferation were performed in three independent experiments.

### CDKN2A genotyping

Germline genomic DNA was extracted from peripheral blood samples obtained during clinical remission for children with ALL. *CDKN2A* exons were genotyped in the samples by Sanger Sequencing using primers listed in Supplementary Table 4, Supplemental digital content 1, [http://links.lww.com/FPC/B407](http://links.lww.com/FPC/B407).

### Western blotting

Preparation of cellular protein lysates was performed by using the Cell signaling Lysis buffer (9803; Cell Signaling Technology, Shanghai, China) according to the manufacturer’s extraction protocol. Protein quantitation was done using Pierce BCA Protein Assay Kit (ThermoFisher Scientific, Shanghai, China). A total of 30 μg of protein was denatured in Leammli buffer at 95°C for 5 min and western immunoblotting was performed using the Bio-Rad system 4–15% Precast Progein Gels. The transfer was performed using the Trans-Blot turbo system (Bio-Rad) onto polyvinylidene fluoride membranes. The immunoblotting was performed with the primary antibody and secondary anti-rabbit antibodies mentioned in Supplementary Table 5, Supplemental digital content 1, [http://links.lww.com/FPC/B407](http://links.lww.com/FPC/B407). Images were acquired by using the Bio-Rad imaging chemidoc MP system. ImageJ software (National Institutes of Health, Bethesda, Maryland, USA) was used to performed densitometry analyses of western blots. Results of each band were normalized to the beta-actin/Glyceraldehyde-3-Phosphate Dehydrogenase levels in the same blot.

### Cell cycle analysis

Cells were harvested and fixed in 70% ethanol for 30 min at 4°C and washed with phosphate-buffered saline supplemented with 1% FBS. The cells were treated with 100 µg/mL RNase A (Sigma Aldrich, Shanghai, China) for 1 h at 37°C in a humidified 5% CO_2_ incubator and stained using 50 µg/mL propidium iodide (Sigma Aldrich) for 30 min at room temperature. The DNA content of the cells was analyzed using FACSCalibur (BD Biosciences, Franklin Lakes, New Jersey, USA) and FlowJo v. 10 software (FlowJo, Ashland, Oregon, USA).

### Statistical analysis

All statistical analyses were performed using GraphPad Prism and/or R (version 3.2.5, [https://www.R-project.org](https://www.R-project.org)); all tests were two-sided. *P* < 0.05 was considered to be statistically significant, *P* < 0.05, *; *P* < 0.01, **, *P* < 0.001, ***, and *P* < 0.0001, ****. Regional association plots were created by Locus zoom ([http://locuszoom.org/](http://locuszoom.org/)) [[18](#R18)].

## Acknowledgements

This work was supported by research grants from St. Baldrick’s Foundation International Scholar (581580), Natural Science Foundation of Guangdong Province (2015A030313460), and Guangzhou Women and Children’s Medical Center Internal Program (IP-2018-001, 5001-1600006, and 5001-1600008). M.Q. is supported by the Program for Professor of Special Appointment (Eastern Scholar) at Shanghai Institutions of Higher Learning and the National Natural Science Foundation of China (81973997). C.L. is supported by the National Natural Science Foundation of China (32000392). The authors thank the patients in this study, and they kindly give us a chance to work on this interesting research. We would also like to thank Xujie Zhao for English language editing.

## Conflicts of interest

There are no conflicts of interest.

## Supplementary Material

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Hunger SP, Mullighan CG. Acute lymphoblastic leukemia in children. N Engl J Med 2015; 373:1541–1552.  [DOI](https://doi.org/10.1056/NEJMra1400972) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26465987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Acute%20lymphoblastic%20leukemia%20in%20children.&author=SP%20Hunger&author=CG%20Mullighan&volume=373&publication_year=2015&pages=1541-1552&pmid=26465987&doi=10.1056/NEJMra1400972&)

2. Terwilliger T, Abdul-Hay M. Acute lymphoblastic leukemia: a comprehensive review and 2017 update. Blood Cancer J 2017; 7:e577.  [DOI](https://doi.org/10.1038/bcj.2017.53) | [PMC free article](/articles/PMC5520400/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28665419/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Cancer%20J&title=Acute%20lymphoblastic%20leukemia:%20a%20comprehensive%20review%20and%202017%20update.&author=T%20Terwilliger&author=M%20Abdul-Hay&volume=7&publication_year=2017&pages=e577&pmid=28665419&doi=10.1038/bcj.2017.53&)

3. Bloom M, Maciaszek JL, Clark ME, Pui CH, Nichols KE. Recent advances in genetic predisposition to pediatric acute lymphoblastic leukemia. Expert Rev Hematol 2020; 13:55–70.  [DOI](https://doi.org/10.1080/17474086.2020.1685866) | [PMC free article](/articles/PMC10576863/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31657974/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Rev%20Hematol&title=Recent%20advances%20in%20genetic%20predisposition%20to%20pediatric%20acute%20lymphoblastic%20leukemia.&author=M%20Bloom&author=JL%20Maciaszek&author=ME%20Clark&author=CH%20Pui&author=KE%20Nichols&volume=13&publication_year=2020&pages=55-70&pmid=31657974&doi=10.1080/17474086.2020.1685866&)

4. Moriyama T, Relling MV, Yang JJ. Inherited genetic variation in childhood acute lymphoblastic leukemia. Blood 2015; 125:3988–3995.  [DOI](https://doi.org/10.1182/blood-2014-12-580001) | [PMC free article](/articles/PMC4481591/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25999454/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Inherited%20genetic%20variation%20in%20childhood%20acute%20lymphoblastic%20leukemia.&author=T%20Moriyama&author=MV%20Relling&author=JJ%20Yang&volume=125&publication_year=2015&pages=3988-3995&pmid=25999454&doi=10.1182/blood-2014-12-580001&)

5. Vijayakrishnan J, Kumar R, Henrion MY, Moorman AV, Rachakonda PS, Hosen I, et al. A genome-wide association study identifies risk loci for childhood acute lymphoblastic leukemia at 10q26.13 and 12q23.1. Leukemia 2017; 31:573–579.  [DOI](https://doi.org/10.1038/leu.2016.271) | [PMC free article](/articles/PMC5336191/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27694927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=A%20genome-wide%20association%20study%20identifies%20risk%20loci%20for%20childhood%20acute%20lymphoblastic%20leukemia%20at%2010q26.13%20and%2012q23.1.&author=J%20Vijayakrishnan&author=R%20Kumar&author=MY%20Henrion&author=AV%20Moorman&author=PS%20Rachakonda&volume=31&publication_year=2017&pages=573-579&pmid=27694927&doi=10.1038/leu.2016.271&)

6. Xu H, Zhang H, Yang W, Yadav R, Morrison AC, Qian M, et al. Inherited coding variants at the CDKN2A locus influence susceptibility to acute lymphoblastic leukaemia in children. Nat Commun 2015; 6:7553.  [DOI](https://doi.org/10.1038/ncomms8553) | [PMC free article](/articles/PMC4544058/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26104880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Inherited%20coding%20variants%20at%20the%20CDKN2A%20locus%20influence%20susceptibility%20to%20acute%20lymphoblastic%20leukaemia%20in%20children.&author=H%20Xu&author=H%20Zhang&author=W%20Yang&author=R%20Yadav&author=AC%20Morrison&volume=6&publication_year=2015&pages=7553&pmid=26104880&doi=10.1038/ncomms8553&)

7. Bertin R, Acquaviva C, Mirebeau D, Guidal-Giroux C, Vilmer E, Cavé H. CDKN2A, CDKN2B, and MTAP gene dosage permits precise characterization of mono- and bi-allelic 9p21 deletions in childhood acute lymphoblastic leukemia. Genes Chromosomes Cancer 2003; 37:44–57.  [DOI](https://doi.org/10.1002/gcc.10188) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12661005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Chromosomes%20Cancer&title=CDKN2A,%20CDKN2B,%20and%20MTAP%20gene%20dosage%20permits%20precise%20characterization%20of%20mono-%20and%20bi-allelic%209p21%20deletions%20in%20childhood%20acute%20lymphoblastic%20leukemia.&author=R%20Bertin&author=C%20Acquaviva&author=D%20Mirebeau&author=C%20Guidal-Giroux&author=E%20Vilmer&volume=37&publication_year=2003&pages=44-57&pmid=12661005&doi=10.1002/gcc.10188&)

8. Iacobucci I, Ferrari A, Lonetti A, Papayannidis C, Paoloni F, Trino S, et al. CDKN2A/B alterations impair prognosis in adult BCR-ABL1-positive acute lymphoblastic leukemia patients. Clin Cancer Res 2011; 17:7413–7423.  [DOI](https://doi.org/10.1158/1078-0432.CCR-11-1227) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22134481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=CDKN2A/B%20alterations%20impair%20prognosis%20in%20adult%20BCR-ABL1-positive%20acute%20lymphoblastic%20leukemia%20patients.&author=I%20Iacobucci&author=A%20Ferrari&author=A%20Lonetti&author=C%20Papayannidis&author=F%20Paoloni&volume=17&publication_year=2011&pages=7413-7423&pmid=22134481&doi=10.1158/1078-0432.CCR-11-1227&)

9. Kettunen E, Savukoski S, Salmenkivi K, Böhling T, Vanhala E, Kuosma E, et al. CDKN2A copy number and p16 expression in malignant pleural mesothelioma in relation to asbestos exposure. BMC Cancer 2019; 19:507.  [DOI](https://doi.org/10.1186/s12885-019-5652-y) | [PMC free article](/articles/PMC6537412/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31138176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=CDKN2A%20copy%20number%20and%20p16%20expression%20in%20malignant%20pleural%20mesothelioma%20in%20relation%20to%20asbestos%20exposure.&author=E%20Kettunen&author=S%20Savukoski&author=K%20Salmenkivi&author=T%20B%C3%B6hling&author=E%20Vanhala&volume=19&publication_year=2019&pages=507&pmid=31138176&doi=10.1186/s12885-019-5652-y&)

10. van Oosterwijk JG, Li C, Yang X, Opferman JT, Sherr CJ. Small mitochondrial Arf (smArf) protein corrects p53-independent developmental defects of Arf tumor suppressor-deficient mice. Proc Natl Acad Sci USA 2017; 114:7420–7425.  [DOI](https://doi.org/10.1073/pnas.1707292114) | [PMC free article](/articles/PMC5514764/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28652370/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Small%20mitochondrial%20Arf%20(smArf)%20protein%20corrects%20p53-independent%20developmental%20defects%20of%20Arf%20tumor%20suppressor-deficient%20mice.&author=JG%20van%20Oosterwijk&author=C%20Li&author=X%20Yang&author=JT%20Opferman&author=CJ%20Sherr&volume=114&publication_year=2017&pages=7420-7425&pmid=28652370&doi=10.1073/pnas.1707292114&)

11. Hayward NK, Wilmott JS, Waddell N, Johansson PA, Field MA, Nones K, et al. Whole-genome landscapes of major melanoma subtypes. Nature 2017; 545:175–180.  [DOI](https://doi.org/10.1038/nature22071) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28467829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Whole-genome%20landscapes%20of%20major%20melanoma%20subtypes.&author=NK%20Hayward&author=JS%20Wilmott&author=N%20Waddell&author=PA%20Johansson&author=MA%20Field&volume=545&publication_year=2017&pages=175-180&pmid=28467829&doi=10.1038/nature22071&)

12. Monzon J, Liu L, Brill H, Goldstein AM, Tucker MA, From L, et al. CDKN2A mutations in multiple primary melanomas. N Engl J Med 1998; 338:879–887.  [DOI](https://doi.org/10.1056/NEJM199803263381305) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9516223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=CDKN2A%20mutations%20in%20multiple%20primary%20melanomas.&author=J%20Monzon&author=L%20Liu&author=H%20Brill&author=AM%20Goldstein&author=MA%20Tucker&volume=338&publication_year=1998&pages=879-887&pmid=9516223&doi=10.1056/NEJM199803263381305&)

13. Kathiravan M, Singh M, Bhatia P, Trehan A, Varma N, Sachdeva MS, et al. Deletion of CDKN2A/B is associated with inferior relapse free survival in pediatric B cell acute lymphoblastic leukemia. Leuk Lymphoma 2019; 60:433–441.  [DOI](https://doi.org/10.1080/10428194.2018.1482542) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29966470/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Lymphoma&title=Deletion%20of%20CDKN2A/B%20is%20associated%20with%20inferior%20relapse%20free%20survival%20in%20pediatric%20B%20cell%20acute%20lymphoblastic%20leukemia.&author=M%20Kathiravan&author=M%20Singh&author=P%20Bhatia&author=A%20Trehan&author=N%20Varma&volume=60&publication_year=2019&pages=433-441&pmid=29966470&doi=10.1080/10428194.2018.1482542&)

14. Sherborne AL, Hosking FJ, Prasad RB, Kumar R, Koehler R, Vijayakrishnan J, et al. Variation in CDKN2A at 9p21.3 influences childhood acute lymphoblastic leukemia risk. Nat Genet 2010; 42:492–494.  [DOI](https://doi.org/10.1038/ng.585) | [PMC free article](/articles/PMC3434228/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20453839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Variation%20in%20CDKN2A%20at%209p21.3%20influences%20childhood%20acute%20lymphoblastic%20leukemia%20risk.&author=AL%20Sherborne&author=FJ%20Hosking&author=RB%20Prasad&author=R%20Kumar&author=R%20Koehler&volume=42&publication_year=2010&pages=492-494&pmid=20453839&doi=10.1038/ng.585&)

15. Xu H, Yang W, Perez-Andreu V, Devidas M, Fan Y, Cheng C, et al. Novel susceptibility variants at 10p12.31-12.2 for childhood acute lymphoblastic leukemia in ethnically diverse populations. J Natl Cancer Inst 2013; 105:733–742.  [DOI](https://doi.org/10.1093/jnci/djt042) | [PMC free article](/articles/PMC3691938/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23512250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Novel%20susceptibility%20variants%20at%2010p12.31-12.2%20for%20childhood%20acute%20lymphoblastic%20leukemia%20in%20ethnically%20diverse%20populations.&author=H%20Xu&author=W%20Yang&author=V%20Perez-Andreu&author=M%20Devidas&author=Y%20Fan&volume=105&publication_year=2013&pages=733-742&pmid=23512250&doi=10.1093/jnci/djt042&)

16. Hungate EA, Vora SR, Gamazon ER, Moriyama T, Best T, Hulur I, et al. A variant at 9p21.3 functionally implicates CDKN2B in paediatric B-cell precursor acute lymphoblastic leukaemia aetiology. Nat Commun 2016; 7:10635.  [DOI](https://doi.org/10.1038/ncomms10635) | [PMC free article](/articles/PMC4754340/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26868379/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=A%20variant%20at%209p21.3%20functionally%20implicates%20CDKN2B%20in%20paediatric%20B-cell%20precursor%20acute%20lymphoblastic%20leukaemia%20aetiology.&author=EA%20Hungate&author=SR%20Vora&author=ER%20Gamazon&author=T%20Moriyama&author=T%20Best&volume=7&publication_year=2016&pages=10635&pmid=26868379&doi=10.1038/ncomms10635&)

17. Buniello A, MacArthur JAL, Cerezo M, Harris LW, Hayhurst J, Malangone C, et al. The NHGRI-EBI GWAS Catalog of published genome-wide association studies, targeted arrays and summary statistics 2019. Nucleic Acids Res 2019; 47:D1005–D1012.  [DOI](https://doi.org/10.1093/nar/gky1120) | [PMC free article](/articles/PMC6323933/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30445434/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20NHGRI-EBI%20GWAS%20Catalog%20of%20published%20genome-wide%20association%20studies,%20targeted%20arrays%20and%20summary%20statistics%202019.&author=A%20Buniello&author=JAL%20MacArthur&author=M%20Cerezo&author=LW%20Harris&author=J%20Hayhurst&volume=47&publication_year=2019&pages=D1005-D1012&pmid=30445434&doi=10.1093/nar/gky1120&)

18. Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, et al. LocusZoom: regional visualization of genome-wide association scan results. Bioinformatics 2010; 26:2336–2337.  [DOI](https://doi.org/10.1093/bioinformatics/btq419) | [PMC free article](/articles/PMC2935401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20634204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LocusZoom:%20regional%20visualization%20of%20genome-wide%20association%20scan%20results.&author=RJ%20Pruim&author=RP%20Welch&author=S%20Sanna&author=TM%20Teslovich&author=PS%20Chines&volume=26&publication_year=2010&pages=2336-2337&pmid=20634204&doi=10.1093/bioinformatics/btq419&)

19. Pikman Y, Alexe G, Roti G, Conway AS, Furman A, Lee ES, et al. Synergistic drug combinations with a CDK4/6 inhibitor in T-cell acute lymphoblastic leukemia. Clin Cancer Res 2017; 23:1012–1024.  [DOI](https://doi.org/10.1158/1078-0432.CCR-15-2869) | [PMC free article](/articles/PMC5432118/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28151717/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Synergistic%20drug%20combinations%20with%20a%20CDK4/6%20inhibitor%20in%20T-cell%20acute%20lymphoblastic%20leukemia.&author=Y%20Pikman&author=G%20Alexe&author=G%20Roti&author=AS%20Conway&author=A%20Furman&volume=23&publication_year=2017&pages=1012-1024&pmid=28151717&doi=10.1158/1078-0432.CCR-15-2869&)

20. Park JH, Park H, Kim KH, Kim JS, Choi IS, Roh EY, et al. Acute lymphoblastic leukemia in a patient treated with letrozole and palbociclib. J Breast Cancer 2020; 23:100–106.  [DOI](https://doi.org/10.4048/jbc.2020.23.e1) | [PMC free article](/articles/PMC7043942/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32140274/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Breast%20Cancer&title=Acute%20lymphoblastic%20leukemia%20in%20a%20patient%20treated%20with%20letrozole%20and%20palbociclib.&author=JH%20Park&author=H%20Park&author=KH%20Kim&author=JS%20Kim&author=IS%20Choi&volume=23&publication_year=2020&pages=100-106&pmid=32140274&doi=10.4048/jbc.2020.23.e1&)

21. Bortolozzi R, Mattiuzzo E, Trentin L, Accordi B, Basso G, Viola G. Ribociclib, a Cdk4/Cdk6 kinase inhibitor, enhances glucocorticoid sensitivity in B-acute lymphoblastic leukemia (B-All). Biochem Pharmacol 2018; 153:230–241.  [DOI](https://doi.org/10.1016/j.bcp.2018.01.050) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29408328/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Ribociclib,%20a%20Cdk4/Cdk6%20kinase%20inhibitor,%20enhances%20glucocorticoid%20sensitivity%20in%20B-acute%20lymphoblastic%20leukemia%20(B-All).&author=R%20Bortolozzi&author=E%20Mattiuzzo&author=L%20Trentin&author=B%20Accordi&author=G%20Basso&volume=153&publication_year=2018&pages=230-241&pmid=29408328&doi=10.1016/j.bcp.2018.01.050&)

22. Guiley KZ, Stevenson JW, Lou K, Barkovich KJ, Kumarasamy V, Wijeratne TU, et al. p27 allosterically activates cyclin-dependent kinase 4 and antagonizes palbociclib inhibition. Science 2019; 366:eaaw2106.  [DOI](https://doi.org/10.1126/science.aaw2106) | [PMC free article](/articles/PMC7592119/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31831640/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=p27%20allosterically%20activates%20cyclin-dependent%20kinase%204%20and%20antagonizes%20palbociclib%20inhibition.&author=KZ%20Guiley&author=JW%20Stevenson&author=K%20Lou&author=KJ%20Barkovich&author=V%20Kumarasamy&volume=366&publication_year=2019&pages=eaaw2106&pmid=31831640&doi=10.1126/science.aaw2106&)

23. Russo AA, Tong L, Lee JO, Jeffrey PD, Pavletich NP. Pavletich. Structural basis for inhibition of the cyclin-dependent kinase Cdk6 by the tumour suppressor p16INKa. Nature 1998; 395:237–243.  [DOI](https://doi.org/10.1038/26155) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9751050/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Pavletich.%20Structural%20basis%20for%20inhibition%20of%20the%20cyclin-dependent%20kinase%20Cdk6%20by%20the%20tumour%20suppressor%20p16INKa.&author=AA%20Russo&author=L%20Tong&author=JO%20Lee&author=PD%20Jeffrey&author=NP%20Pavletich&volume=395&publication_year=1998&pages=237-243&pmid=9751050&doi=10.1038/26155&)

24. Day PJ, Cleasby A, Tickle IJ, O’Reilly M, Coyle JE, Holding FP, et al. Crystal structure of human CDK4 in complex with a D-type cyclin. Proc Natl Acad Sci USA 2009; 106:4166–4170.  [DOI](https://doi.org/10.1073/pnas.0809645106) | [PMC free article](/articles/PMC2657441/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19237565/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Crystal%20structure%20of%20human%20CDK4%20in%20complex%20with%20a%20D-type%20cyclin.&author=PJ%20Day&author=A%20Cleasby&author=IJ%20Tickle&author=M%20O%E2%80%99Reilly&author=JE%20Coyle&volume=106&publication_year=2009&pages=4166-4170&pmid=19237565&doi=10.1073/pnas.0809645106&)

25. Williams RT, Barnhill LM, Kuo HH, Lin WD, Batova A, Yu AL, Diccianni MB. Chimeras of p14ARF and p16: functional hybrids with the ability to arrest growth. PLoS One 2014; 9:e88219.  [DOI](https://doi.org/10.1371/journal.pone.0088219) | [PMC free article](/articles/PMC3914946/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24505435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Chimeras%20of%20p14ARF%20and%20p16:%20functional%20hybrids%20with%20the%20ability%20to%20arrest%20growth.&author=RT%20Williams&author=LM%20Barnhill&author=HH%20Kuo&author=WD%20Lin&author=A%20Batova&volume=9&publication_year=2014&pages=e88219&pmid=24505435&doi=10.1371/journal.pone.0088219&)

26. Chen YW, Chu HC, Lin Z-S, Shiah WJ, Chou CP, Klimstra DS, Lewis BC. p16 Stimulates CDC42-dependent migration of hepatocellular carcinoma cells. PLoS One 2013; 8:e69389.  [DOI](https://doi.org/10.1371/journal.pone.0069389) | [PMC free article](/articles/PMC3722281/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23894465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=p16%20Stimulates%20CDC42-dependent%20migration%20of%20hepatocellular%20carcinoma%20cells.&author=YW%20Chen&author=HC%20Chu&author=Z-S%20Lin&author=WJ%20Shiah&author=CP%20Chou&volume=8&publication_year=2013&pages=e69389&pmid=23894465&doi=10.1371/journal.pone.0069389&)

27. Gutierrez-Camino A, Martin-Guerrero I, Garcia de Andoin N, Sastre A, Carbone Bañeres A, Astigarraga I, et al. Confirmation of involvement of new variants at CDKN2A/B in pediatric acute lymphoblastic leukemia susceptibility in the Spanish population. PLoS One 2017; 12:e0177421.  [DOI](https://doi.org/10.1371/journal.pone.0177421) | [PMC free article](/articles/PMC5421813/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28481918/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Confirmation%20of%20involvement%20of%20new%20variants%20at%20CDKN2A/B%20in%20pediatric%20acute%20lymphoblastic%20leukemia%20susceptibility%20in%20the%20Spanish%20population.&author=A%20Gutierrez-Camino&author=I%20Martin-Guerrero&author=N%20Garcia%20de%20Andoin&author=A%20Sastre&author=A%20Carbone%20Ba%C3%B1eres&volume=12&publication_year=2017&pages=e0177421&pmid=28481918&doi=10.1371/journal.pone.0177421&)

28. Schuster K, Venkateswaran N, Rabellino A, Girard L, Peña-Llopis S, Scaglioni PP. Nullifying the CDKN2AB locus promotes mutant K-ras lung tumorigenesis. Mol Cancer Res 2014; 12:912–923.  [DOI](https://doi.org/10.1158/1541-7786.MCR-13-0620-T) | [PMC free article](/articles/PMC4058359/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24618618/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Res&title=Nullifying%20the%20CDKN2AB%20locus%20promotes%20mutant%20K-ras%20lung%20tumorigenesis.&author=K%20Schuster&author=N%20Venkateswaran&author=A%20Rabellino&author=L%20Girard&author=S%20Pe%C3%B1a-Llopis&volume=12&publication_year=2014&pages=912-923&pmid=24618618&doi=10.1158/1541-7786.MCR-13-0620-T&)

29. Sievers P, Hielscher T, Schrimpf D, Stichel D, Reuss DE, Berghoff AS, et al. CDKN2A/B homozygous deletion is associated with early recurrence in meningiomas. Acta Neuropathol 2020; 140:409–413.  [DOI](https://doi.org/10.1007/s00401-020-02188-w) | [PMC free article](/articles/PMC7423850/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32642869/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Neuropathol&title=CDKN2A/B%20homozygous%20deletion%20is%20associated%20with%20early%20recurrence%20in%20meningiomas.&author=P%20Sievers&author=T%20Hielscher&author=D%20Schrimpf&author=D%20Stichel&author=DE%20Reuss&volume=140&publication_year=2020&pages=409-413&pmid=32642869&doi=10.1007/s00401-020-02188-w&)

30. Chan AK, Han SJ, Choy W, Beleford D, Aghi MK, Berger MS, et al. Familial melanoma-astrocytoma syndrome: synchronous diffuse astrocytoma and pleomorphic xanthoastrocytoma in a patient with germline CDKN2A/B deletion and a significant family history. Clin Neuropathol 2017; 36:213–221.  [DOI](https://doi.org/10.5414/NP301022) | [PMC free article](/articles/PMC5628627/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28699883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Neuropathol&title=Familial%20melanoma-astrocytoma%20syndrome:%20synchronous%20diffuse%20astrocytoma%20and%20pleomorphic%20xanthoastrocytoma%20in%20a%20patient%20with%20germline%20CDKN2A/B%20deletion%20and%20a%20significant%20family%20history.&author=AK%20Chan&author=SJ%20Han&author=W%20Choy&author=D%20Beleford&author=MK%20Aghi&volume=36&publication_year=2017&pages=213-221&pmid=28699883&doi=10.5414/NP301022&)

31. Qi X, Wan Y, Zhan Q, Yang S, Wang Y, Cai X. Effect of CDKN2A/B rs4977756 polymorphism on glioma risk: a meta-analysis of 16 studies including 24077 participants. Mamm Genome 2016; 27:1–7.  [DOI](https://doi.org/10.1007/s00335-015-9612-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26577493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mamm%20Genome&title=Effect%20of%20CDKN2A/B%20rs4977756%20polymorphism%20on%20glioma%20risk:%20a%20meta-analysis%20of%2016%20studies%20including%2024077%20participants.&author=X%20Qi&author=Y%20Wan&author=Q%20Zhan&author=S%20Yang&author=Y%20Wang&volume=27&publication_year=2016&pages=1-7&pmid=26577493&doi=10.1007/s00335-015-9612-9&)

32. Reinhardt A, Stichel D, Schrimpf D, Sahm F, Korshunov A, Reuss DE, et al. Anaplastic astrocytoma with piloid features, a novel molecular class of IDH wildtype glioma with recurrent MAPK pathway, CDKN2A/B and ATRX alterations. Acta Neuropathol 2018; 136:273–291.  [DOI](https://doi.org/10.1007/s00401-018-1837-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29564591/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Neuropathol&title=Anaplastic%20astrocytoma%20with%20piloid%20features,%20a%20novel%20molecular%20class%20of%20IDH%20wildtype%20glioma%20with%20recurrent%20MAPK%20pathway,%20CDKN2A/B%20and%20ATRX%20alterations.&author=A%20Reinhardt&author=D%20Stichel&author=D%20Schrimpf&author=F%20Sahm&author=A%20Korshunov&volume=136&publication_year=2018&pages=273-291&pmid=29564591&doi=10.1007/s00401-018-1837-8&)

33. Eshkoor SA, Ismail P, Rahman SA, Oshkour SA. p16 gene expression in basal cell carcinoma. Arch Med Res 2008; 39:668–673.  [DOI](https://doi.org/10.1016/j.arcmed.2008.06.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18760195/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Med%20Res&title=p16%20gene%20expression%20in%20basal%20cell%20carcinoma.&author=SA%20Eshkoor&author=P%20Ismail&author=SA%20Rahman&author=SA%20Oshkour&volume=39&publication_year=2008&pages=668-673&pmid=18760195&doi=10.1016/j.arcmed.2008.06.003&)

34. Walsh KM, de Smith AJ, Hansen HM, Smirnov IV, Gonseth S, Endicott AA, et al. A Heritable missense polymorphism in CDKN2A confers strong risk of childhood acute lymphoblastic leukemia and is preferentially selected during clonal evolution. Cancer Res 2015; 75:4884–4894.  [DOI](https://doi.org/10.1158/0008-5472.CAN-15-1105) | [PMC free article](/articles/PMC4651745/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26527286/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=A%20Heritable%20missense%20polymorphism%20in%20CDKN2A%20confers%20strong%20risk%20of%20childhood%20acute%20lymphoblastic%20leukemia%20and%20is%20preferentially%20selected%20during%20clonal%20evolution.&author=KM%20Walsh&author=AJ%20de%20Smith&author=HM%20Hansen&author=IV%20Smirnov&author=S%20Gonseth&volume=75&publication_year=2015&pages=4884-4894&pmid=26527286&doi=10.1158/0008-5472.CAN-15-1105&)

35. Zhang Y, Hyle J, Wright S, Shao Y, Zhao X, Zhang H, et al. A cis-element within the ARF locus mediates repression of p16INK4A expression via long-range chromatin interactions. Proc Natl Acad Sci U S A 2019; 116:26644–26652.  [DOI](https://doi.org/10.1073/pnas.1909720116) | [PMC free article](/articles/PMC6936709/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31818950/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=A%20cis-element%20within%20the%20ARF%20locus%20mediates%20repression%20of%20p16INK4A%20expression%20via%20long-range%20chromatin%20interactions.&author=Y%20Zhang&author=J%20Hyle&author=S%20Wright&author=Y%20Shao&author=X%20Zhao&volume=116&publication_year=2019&pages=26644-26652&pmid=31818950&doi=10.1073/pnas.1909720116&)

36. Krug U, Ganser A, Koeffler HP. Tumor suppressor genes in normal and malignant hematopoiesis. Oncogene 2002; 21:3475–3495.  [DOI](https://doi.org/10.1038/sj.onc.1205322) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12032783/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncogene&title=Tumor%20suppressor%20genes%20in%20normal%20and%20malignant%20hematopoiesis.&author=U%20Krug&author=A%20Ganser&author=HP%20Koeffler&volume=21&publication_year=2002&pages=3475-3495&pmid=12032783&doi=10.1038/sj.onc.1205322&)

37. Williams RT, Sherr CJ. The INK4-ARF (CDKN2A/B) locus in hematopoiesis and BCR-ABL-induced leukemias. Cold Spring Harb Symp Quant Biol 2008; 73:461–467.  [DOI](https://doi.org/10.1101/sqb.2008.73.039) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19028987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb%20Symp%20Quant%20Biol&title=The%20INK4-ARF%20(CDKN2A/B)%20locus%20in%20hematopoiesis%20and%20BCR-ABL-induced%20leukemias.&author=RT%20Williams&author=CJ%20Sherr&volume=73&publication_year=2008&pages=461-467&pmid=19028987&doi=10.1101/sqb.2008.73.039&)

38. Liu JY, Souroullas GP, Diekman BO, Krishnamurthy J, Hall BM, Sorrentino JA, et al. Cells exhibiting strong p16 INK4a promoter activation in vivo display features of senescence. Proc Natl Acad Sci USA 2019; 116:2603–2611.  [DOI](https://doi.org/10.1073/pnas.1818313116) | [PMC free article](/articles/PMC6377452/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30683717/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Cells%20exhibiting%20strong%20p16%20INK4a%20promoter%20activation%20in%20vivo%20display%20features%20of%20senescence.&author=JY%20Liu&author=GP%20Souroullas&author=BO%20Diekman&author=J%20Krishnamurthy&author=BM%20Hall&volume=116&publication_year=2019&pages=2603-2611&pmid=30683717&doi=10.1073/pnas.1818313116&)

39. Chien WW, Catallo R, Chebel A, Baranger L, Thomas X, Béné MC, et al. The p16(INK4A)/pRb pathway and telomerase activity define a subgroup of Ph+ adult acute lymphoblastic leukemia associated with inferior outcome. Leuk Res 2015; 39:453–461.  [DOI](https://doi.org/10.1016/j.leukres.2015.01.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25675863/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Res&title=The%20p16(INK4A)/pRb%20pathway%20and%20telomerase%20activity%20define%20a%20subgroup%20of%20Ph+%20adult%20acute%20lymphoblastic%20leukemia%20associated%20with%20inferior%20outcome.&author=WW%20Chien&author=R%20Catallo&author=A%20Chebel&author=L%20Baranger&author=X%20Thomas&volume=39&publication_year=2015&pages=453-461&pmid=25675863&doi=10.1016/j.leukres.2015.01.008&)

40. Shen S, Chen X, Cai J, Yu J, Gao J, Hu S, et al. Effect of dasatinib vs imatinib in the treatment of pediatric Philadelphia chromosome-positive acute lymphoblastic leukemia: a randomized clinical trial. JAMA Oncol 2020; 6:358–366.  [DOI](https://doi.org/10.1001/jamaoncol.2019.5868) | [PMC free article](/articles/PMC6990720/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31944221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Oncol&title=Effect%20of%20dasatinib%20vs%20imatinib%20in%20the%20treatment%20of%20pediatric%20Philadelphia%20chromosome-positive%20acute%20lymphoblastic%20leukemia:%20a%20randomized%20clinical%20trial.&author=S%20Shen&author=X%20Chen&author=J%20Cai&author=J%20Yu&author=J%20Gao&volume=6&publication_year=2020&pages=358-366&pmid=31944221&doi=10.1001/jamaoncol.2019.5868&)

41. Williams RT, Roussel MF, Sherr CJ. Arf gene loss enhances oncogenicity and limits imatinib response in mouse models of Bcr-Abl-induced acute lymphoblastic leukemia. Proc Natl Acad Sci USA 2006; 103:6688–6693.  [DOI](https://doi.org/10.1073/pnas.0602030103) | [PMC free article](/articles/PMC1440588/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16618932/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&title=Arf%20gene%20loss%20enhances%20oncogenicity%20and%20limits%20imatinib%20response%20in%20mouse%20models%20of%20Bcr-Abl-induced%20acute%20lymphoblastic%20leukemia.&author=RT%20Williams&author=MF%20Roussel&author=CJ%20Sherr&volume=103&publication_year=2006&pages=6688-6693&pmid=16618932&doi=10.1073/pnas.0602030103&)
