# Comprehensive characterization of pharmacogenetic variants in TPMT and NUDT15 in children with acute lymphoblastic leukemia

## Metadata
**Authors:** Takaya Moriyama, Wenjian Yang, Colton Smith, Ching-Hon Pui, William E Evans, Mary V Relling, Smita Bhatia, Jun J Yang
**Journal:** Pharmacogenetics and genomics
**Date:** 2022 Feb 1
**DOI:** [10.1097/FPC.0000000000000453](https://doi.org/10.1097/FPC.0000000000000453)
**PMID:** 34412101
**PMCID:** PMC8702453
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8702453/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC8702453/pdf/nihms-1726174.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC8702453/pdf/nihms-1726174.pdf)

## Abstract

Thiopurines (e.g., 6-mercaptopurine [6MP]) are essential for the cure of acute lymphoblastic leukemia (ALL) but can cause dose-limiting hematopoietic toxicity. Germline variants in drug-metabolizing enzyme genes TPMT and NUDT15 have been linked to the risk of thiopurine toxicity. However, the full spectrum of genetic polymorphism in these genes and their impact on the pharmacological effects of thiopurines remain unclear. Herein, we comprehensively sequenced the TPMT and NUDT15 genes in 685 children with ALL from the Children’s Oncology Group AALL03N1 trial and evaluated their association with 6MP dose intensity. We identified 6 and 5 coding variants in TPMT and NUDT15 respectively, confirming the association at known pharmacogenetic variants. Importantly, we discovered a novel gain-of-function non-coding variants in TPMT associated with increased 6MP tolerance (rs12199316), with independent validation in 380 patients from the St. Jude Total Therapy XV protocol. Located adjacent to a regulatory DNA element, this inter-genic variant was strongly associated TPMT transcription, with the variant allele linked to higher expression (P = 2.6 × 10−9). For NUDT15, one non-coding common variant, rs73189762, was identified as potentially related to 6MP intolerance. Collectively, we described pharmacogenetic variants in TPMT and NUDT15 associated with thiopurine sensitivity, providing further insights for implementing pharmacogenetics-based thiopurine individualization.

## Introduction

Thiopurines (e.g., 6-mercaptopurine [6MP] and azathioprine [AZA]) are widely used as chemotherapeutic agents and immunomodulators in various clinical settings, such as acute lymphoblastic leukemia (ALL)^1–4^ and inflammatory bowel disease (IBD)^5,6^. Although highly efficacious, this class of drugs also has narrow therapeutic indices mainly due to frequent hematopoietic cytotoxicity^6–9^. Therefore, dose adjustment by monitoring white blood cell counts is regularly required to avoid excessive bone marrow suppression. Previous studies by us and others reported that the dose-limiting hematopoietic toxicity of thiopurines is explained in part by inherited polymorphisms in genes involved in thiopurine metabolism (i.e., *TPMT*^10–14^ and *NUDT15*^15–19^). Coding variants in *TPMT* and *NUDT15* can result in the loss of enzymatic activities and are associated with increased intracellular accumulation of thiopurine active metabolites^17,20–23^. Because *TPMT* and *NUDT15* are responsible for thiopurine inactivation, patients carrying loss-of-function variants in these genes are predisposed to elevated levels of active metabolites and thus increased sensitivity to thiopurines, necessitating dose reduction to mitigate hematopoietic toxicity^24^.

However, the majority of inter-patient variability in thiopurine toxicity remains unexplained^16^, and our understanding of the pharmacogenetic basis of thiopurine sensitivity is incomplete for a number of reasons^10,15–17,24^. For example, previous reports focused on the coding regions in the *TPMT* and *NUDT15* genes, and the contribution of non-coding variants to thiopurine sensitivity is largely unclear. Similarly, prior studies often examined only known risk variants at these loci, and thus would have missed rare variants that can only be identified by re-sequencing. Therefore, systematically characterizing *TPMT* and *NUDT15* polymorphisms associated with thiopurine sensitivity should further improve the accuracy of genetics-guided thiopurine dosing individualization.

In this study, we performed a comprehensive sequencing study of *TPMT* and *NUDT15* in a total of 1,065 children with ALL from the Children’s Oncology Group [COG] AALL03N1 trial (N = 685) and the St. Jude Total Therapy XV protocol (N = 380), and investigated both coding and non-coding variants for their contributions to thiopurine sensitivity and explored their functional effects.

## Methods

### Patients and thiopurine therapy

This study was approved by the respective institutional review boards, and informed consent was obtained from parents, guardians, and/or patients, as appropriate. The discovery cohort for novel variants of thiopurine sensitivity in *TPMT* comprised 685 children with ALL on the Children’s Oncology Group (COG) study AALL03N1 ([https://clinicaltrails.gov/](https://clinicaltrails.gov/), identifier [NCT00268528](https://clinicaltrials.gov/ct2/show/NCT00268528))^25,26^. The planned daily 6MP dosage was 75 mg/m^2^, and 6MP dosage was clinically adjusted based on the extent of bone marrow suppression and occurrence of infections. Each patient’s longitudinal 6MP dose intensity (%) during maintenance therapy was estimated as the total cumulative prescribed 6MP dose over intended cumulative protocol dose. The validation cohort consisted of 380 patients from the St. Jude Total Therapy XV trial^27^. The protocol maintenance 6MP dosage in this cohort was 75 mg/m^2^, and each patient’s 6MP dose intensity was defined in the same manner as used for the discovery cohort. Both the AALL03N1 and St. Jude Total XV cohorts have been described previously^16,28^.

### TPMT and NUDT15 targeted sequencing

To conduct *TPMT* and *NUDT15* targeted sequencing in the discovery cohort, germline DNA was extracted from peripheral blood or bone marrow obtained during clinical remission and was subjected to the generation of Illumina dual-indexed libraries targeting *TPMT* and *NUDT15*. All exons and the 3′-UTR of the genes were summarily selected for capture, and putative regulatory regions were selected as follows. Cis-acting expression quantitative trait loci (eQTLs) were extracted from the GTEx Portal (version 6, *P* < 10^−7^), and 500 base pairs surrounding the top five most significant loci were targeted. In addition, variants in linkage disequilibrium [LD] with the eQTLs (plus or minus 1 MB) were identified by using the European reference set from the 1,000 Genomes Project^29^. The LD blocks plus 10,000 base pairs upstream of the gene’s transcriptional start site were intersected with the ENCODE data (enhancer, transcriptional factor binding, and DNase I hypersensitive sites), and those regions that overlapped in all three respects in multiple cell lines were chosen. In the case of *TPMT*, we targeted 3,269 base pairs of exonic and 3′-UTR sequences and 8,129 base pairs of regulatory sequences ([Figure 1A](#F1)). Likewise, in the case of *NUDT15*, 2,484 base pairs of exonic and 3′-UTR sequences and 7,349 base pairs of regulatory sequences were targeted ([Figure 1B](#F1)). To capture these regions, the libraries were pooled in sets of 96 and then hybridized with customized Roche NimbleGene SeqCap EZ probes (Roche, Roche NimbleGen, Madison, WI, USA). Quantitative PCR was performed to define the appropriate capture product amplification to efficiently populate an Illumina HiSeq 2000 flowcell for paired-end 2×101 bp sequencing. Sequencing reads in FASTQ format were mapped and aligned by using the Burrows-Wheeler Aligner, and genetic variations were called by using GATK pipeline, as previously described^30,31^. Only those genotypes passing GATK quality control and exhibiting call rates greater than 95% were included in further association analyses. Across samples, a depth coverage of greater than 20X was achieved for more than 90% of the targeted regions. Single nucleotide polymorphism (SNP) genotypes in the validation cohort were retrieved or imputed from the previously published dataset^16^.

### Figure 1. TPMT and NUDT15 targeted sequencing in 685 children with ALL. Captured region is shown for TPMT (A) or NUDT15 (B).

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f314/8702453/467960425821/nihms-1726174-f0001.jpg)

All exons and the 3’-UTR of the genes were selected for capture, and putative regulatory regions were selected on the basis of enhancer, transcriptional factor binding, and DNAase I hypersensitive sites from multiple cell lines. Abbreviations: Chr, chromosome; DHS, DNase I hypersensitive site.

### Discovery of novel TPMT and NUDT15 variants associated with 6MP sensitivity and eQTL analysis

Single variant analysis was conducted using the linear regression model to evaluate the association between each variant and 6MP dose intensity. Genotype was coded as 0, 1, or 2 to indicate the number of copies of the variant allele at each SNP. We also estimated European, African, Native American, and East Asian genetic ancestry for each patient using STRUCTURE software (version 2.2.3; [https://web.stanford.edu/group/pritchardlab/structure.html](https://web.stanford.edu/group/pritchardlab/structure.html)), and included the genetic ancestry as covariates in all the association tests to control for population stratification^16,32^. To rule out confounding effects of the known no-function variants in *TPMT* (i.e., rs1800462, rs1800460, and rs1142345) and *NUDT15* (i.e., rs116855232), the association tests were performed after excluding cases with any of these variants^24,33^. Variants that reached significance in the discovery cohort (*P* < 0.05) were also tested for the association with 6MP dose intensity in the validation cohort, following the same statistical procedure. To investigate if the difference in population compositions between the two cohorts would affect the power in validating the findings, we performed simulation analysis. Based on the discovery AALL03N1 cohort, we randomly generated 1000 synthetic cohorts based on bootstrap using the same sample size of the validation St. Jude Total Therapy XV cohort, but with either the population composition as in AALL03N1 or in St. Jude Total Therapy XV. For each synthetic cohort, we performed genotype-phenotype association analysis adjusting for ancestry. And finally, we compared the powers as the number of times the p-value less than 0.05 among the simulated cohorts. The rvtest software package was used to determine any association with 6MP tolerance ([https://github.com/zhanxw/rvtests](https://github.com/zhanxw/rvtests))^34^, and R (version 4.0.3) was used for all other analyses. To determine whether novel SNPs in *TPMT* and *NUDT15* are eQTLs, we queried the GTEx project database in whole blood^35^.

## Results

### Discovery of novel TPMT variants associated with 6MP sensitivity

Sequencing a total region of 11,398 bp at the *TPMT* locus ([Figure 1A](#F1)), we identified 208 variants in the discovery cohort (COG AALL03N1, N = 685). After removing two synonymous variants in *TPMT*, we classified the remaining 206 variants on the basis of the variant’s position in the *TPMT* gene (i.e., coding or non-coding) and minor allele frequencies in this cohort (common or rare for MAF >= 1.0% or < 1.0%, respectively), as summarized in [Figure 2](#F2).

### Figure 2. Discovery of novel TPMT variants by targeted sequencing in 685 children with ALL.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f314/8702453/2b1067fa1a0d/nihms-1726174-f0002.jpg)

By sequencing TPMT-related regions using the Illumine HiSeq platform following capture-based enrichment of the target site in the 03N1 cohort (N= 685), a total of 208 TPMT-related variants (including two synonymous) were identified. Of the 200 non-coding variants, two common variants (rs10949483 and rs12199316) remained significant after removing cases with known no-function variants in TPMT (i.e., rs1800462, rs1800460, and rs1142345) and NUDT15 (i.e., rs116855232). Out of the 6 coding variants, two missenses (p.Q179H and p.R215H) and one nonsense (p.R226X) with uncertainty to TPMT function were identified (reference # 24). P values were estimated in the linear regression model with the adjustment by races. *2, *3A, *3B, *3C, *8, and *24 represent haplotypes of TPMT. Abbreviation: 6MP, 6-mercaptopurine.

When evaluating coding variants, we identified a total of 6 SNPs: three with known damaging effects on TPMT activity (described as “no function”: rs1800460 [p.A154T], rs1800462 [p.A80P] and rs1142345 [p.Y240C]) and three with unknown effects (rs6921269 [p.Q179H], rs56161402 [p.R215H], and p.R226X)^24^. For the variants with unknown effects on 6MP sensitivity, we next sought to explore their contribution to 6MP tolerance. rs6921269 and rs56161402 were both recurrent (N = 6 and 7, respectively), and the respective 6MP dose intensity of carriers of these variants was 55.2 ± 24.7 mg/m^2^ or 65.9 ± 18.9 mg/m^2^, neither of which significantly differed from that of patients with normal *TPMT* and *NUDT15* (63.6 ± 15.7 mg/m^2^, *P* = 0.19 and 0.77, respectively). Although we could not perform a statistical test for p.R226X because it was a singleton in this cohort, the patient with this variant tolerated 6MP dosage at 73.1 mg/m^2^, higher than that of patients with wildtype *TPMT* and *NUDT15*. Collectively, we concluded that the three coding variants are unlikely to have strong effects on 6MP sensitivity.

When evaluating non-coding variants, we focused on common variants given our sample size and statistical power. After excluding cases with known pharmacogenetic variants in *TPMT* and *NUDT15*, two SNPs (rs10949483 and rs12199316) remained significant (*P* = 0.0036 and 0.0029, respectively); and were thus considered as novel for association with 6MP sensitivity. Because these two variants were in high LD (R^2^ = 0.92 in European population) and rs10949483 was no longer significant in a multivariable model including these two SNPs, we posited that rs12199316 is more likely to be the functional variant driving the association signal at this locus.

In the discovery cohort, the number of the variant alleles at rs12199316 was correlated with an increase in 6MP tolerance: tolerated dosage of 61.5 [4.5–93.0] mg/m^2^, 65.4 [2.3–159.9] mg/m^2^, and 66.5 [29.9–108.9] mg/m^2^ in patients with CC, CG, or GG genotype, respectively ([Figure 3A](#F3)). The same trend of association with 6MP tolerance was confirmed in the validation cohort (*P* = 0.043. [Figures 3B](#F3)). This association was also confirmed when we tested in all patients with known pharmacogenetic variants in *TPMT* and *NUDT15* added as co-variables, and the results are summarized in [Supplemental Figure 1](#SD1).

### Figure 3. Association of the non-coding variant (rs12199316) in TPMT with 6MP intensity in AALL03N1 and St. Jude Total Therapy XV.

![Figure 3.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f314/8702453/52a704d052ae/nihms-1726174-f0003.jpg)

6MP dosage was adjusted during maintenance therapy to avoid bone marrow suppression. After removing cases with known no-function common variants in TPMT (i.e., rs1800462, rs1800460, and rs1142345) and NUDT15 (i.e., rs116855232), the association of rs12199316 with 6MP intensity was tested in the discovery cohort (AALL03N1. A) and the validation cohort (St. Jude Total Therapy XV. B). Each box includes data between 25th and 75th percentiles, with horizontal line indicating median. P value was calculated by the linear regression model with genetic ancestry as covariates. Abbreviation: 6MP, 6-mercaptopurine.

### Functional exploration of non-coding variants in TPMT

The novel 6MP tolerance-associated variant is located in a transcription regulatory element approximately 6.0 kb upstream of *TPMT* ([Figure 4A](#F4)). Surveying the publicly available GTEx database revealed that the novel SNP in *TPMT* (rs12199316) is eQTL for *TPMT* and is associated with elevated *TPMT* transcription in whole blood cells (*P* = 2.6 × 10^−9^, [Figures 4B](#F4)). These results suggest that patients carrying the non-coding variant have higher 6MP tolerance because the SNP results in increased *TPMT* expression, in contrast to loss-of-function missense variants reported previously^24^.

### Figure 4. Non-coding variant in TPMT is located in a regulatory DNA element and is eQTL for TPMT.

![Figure 4.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f314/8702453/4a183daf6821/nihms-1726174-f0004.jpg)

(A) UCSC genome browser (https://genome.ucsc.edu/cgi-bin/hgGateway) tracks rs12199316 locus on chromosome 6. It is located adjacent to a regulatory DNA element, approximately 6kb upstream of TPMT. (B) eQTL analysis from GTEx in whole blood showed the association of TPMT expression with rs12199316. The minor allele was associated with increased expression in TPMT, in line with higher tolerance in ALL cases carrying the variant. Abbreviations: CLP, common lymphoid progenitor; CMP, common myeloid progenitor; HSC, hematopoietic stem cell; LMPP, lymphoid-primed multipotent progenitor cell; MEP, megakaryocyte-erythrocyte progenitor; MMP, multipotent progenitor.

### Evaluation of NUDT15 variants associated with 6MP sensitivity

Following the same strategy with *TPMT*, we next sought to identify novel *NUDT15* variants associated with 6MP sensitivity. Sequencing a 9,833 bp region at the *NUDT15* locus in the discovery cohort ([Figure 1B](#F1)) revealed 230 non-coding variants and 5 coding variants ([Supplemental Figure 2](#SD1)).

We observed five variants: four missense variants and one in-frame variant ([Supplemental Figure 2](#SD1)), all of which were reported in previous studies by us and others^17,24,33^. Among the 230 non-coding variants, only rs73189762 was significantly associated with 6MP dose intensity in the discovery cohort after removing patients with known no-function common variants in *TPMT* and *NUDT15* (*P* = 0.01). The 6MP dose intensity was lower in patients having the variant allele at this SNP in an additive manner ([Supplemental Figure 3A](#SD1)). However, this finding was not confirmed in the validation cohort, presumably due to the small number of cases in the validation cohort ([Supplemental Figure 3B](#SD1)).

Finally, we performed a multivariate analysis including all nonsynonymous coding variants in *TPMT* and *NUDT15* (N = 6 and 5 variants, respectively) plus two novel non-coding risk variants in the discovery cohort to explore independent predictors of 6MP tolerance ([Supplemental Table 2](#SD1)). In this analysis, non-coding variants in *TPMT* (rs12199316) and *NUDT15* (rs73189762) remained significant, as well as coding variants p.Y240C in *TPMT* and p.R139C in *NUDT15*.

## Discussion

The pharmacogenes *TPMT* and *NUDT15* are well-recognized as major determinants of 6MP toxicity^10,15,16^, and preemptive genotyping and dosing individualization is performed to mitigate thiopurine toxicity during ALL treatment^24^. In this study, we comprehensively sequenced the *TPMT* and *NUDT15* loci, including substantial flanking regions, and systematically described variants associated with 6MP sensitivity in large cohorts of children with ALL. We identified variants that increase 6MP toxicity as well as those related to increased 6MP tolerance, by directly affecting TPMT and NUDT15 protein function or indirectly influencing gene transcription.

The enzyme TPMT converts 6MP primarily to inactive methylated thiopurine species; therefore its activity and expression are inversely correlated with thiopurine sensitivity^22,24^. In fact, patients with inherited coding variants in *TPMT* cannot tolerate full doses of thiopurine due to decreased TPMT activity of the variant proteins that exhibit instability^10,36,37^. By contrast, the novel non-coding pharmacogenetic variant in *TPMT* described herein (rs12199316) is associated with increased tolerance to 6MP, because of increased expression of *TPMT* as suggested by the eQTL analysis, in line with its known TPMT function in thiopurine metabolism. It should be noted that the rs10949483 variant is also located in the coding region of the adjacent *NHLRC1* gene and can result in the substitution of leucine for proline at position 111 of NHLRC1 protein. While it is formally possible that this amino acid change in *NHLRC1* may contribute to the association of this variant with altered 6MP sensitivity, there is no evidence to suggest any involvement of this E3 ubiquitin ligase in thiopurine cytotoxicity.

Similarly, we also identified a non-coding variant in *NUDT15* associated with 6MP sensitivity (rs73189762) in the discovery cohort. This association was not replicated in the validation cohort, partly because of our limited sample size and the relatively low frequency of this variant. While rs73189762 is located within a DNA regulatory element 3’ to *NUDT15* ([Supplemental Figure 4](#SD1)), it is not an eQTL in the GTex dataset (*P* = 0.69 in whole blood). These results point to the need for further mechanistic studies and association analysis in larger patient cohorts to definitely determine the impact of this variant on 6MP tolerance. We also noted that the impact of non-coding variants on 6MP sensitivity was substantially smaller than known coding variants in *TPMT* and *NUDT15*. For example, the allelic effect size of rs12199316 in the discovery cohort was 2.7 ± 0.97 mg/m^2,^, ~5-fold lower than that of coding variant rs1142345 in *TPMT* (14.7 ± 2.8 mg/m^2^). This weaker association with 6MP sensitivity by non-coding variants than coding variants suggests that impacts of protein instability are more robust on thiopurine metabolism than those of gene expression level. Nonetheless, our findings highlight the potential contribution of non-coding variants to drug metabolism and further refinement of thiopurine dosing individualization. To improve dosing individualization with these new non-coding variants, it is also critical to integrate ethnic information, because their allele frequencies are different by ancestries ([Supplemental Table 3](#SD1))

Although five missense *TPMT* variants identified in this study were reported previously^28,38–44^, functional status of the p.Q179H and p.R215H alleles was unclear^24^. In our discovery cohort, we found that heterozygous cases carrying either of these variants tolerated levels of 6MP comparable to those in *TPMT* WT patients, consistent with recent results from large mutagenesis scan of *TPMT* variants^45^. Taken together, we conclude that p.Q179H and p.R215H are likely of normal TPMT activity, and 6MP may be started at a standard dosage level for these carriers during ALL therapy. Similarly, the missense p.Q6E variant in *NUDT15* is likely to be WT-like on the basis of the 6MP tolerance in our cohort and also based on result of our massively parallel functional characterization for *NUDT15* variants^33^. Because few patients in our study had these variants, these assessments require further validation in additional clinical cohorts.

Given the difference in racial compositions between AALL03N1 and St. Jude Total Therapy XV cohorts ([Supplemental Table 1](#SD1)), we performed simulation analysis to evaluate the impact of this on the power in variant detections and validations. To this end, we created a simulated validation cohort with the same level and ancestry diversity as our discovery AALL03N1 cohort. The statistical power to detect the effect of the *TPMT* SNP rs10949482 was 53.8% in the St. Jude cohort, compared to that in the hypothetical and more racially diverse validation cohort (81%). By contrast, the power for detecting *NUDT15* SNP rs73189762 was actually higher in the St. Jude validation set (64%) than in the simulated cohort (32%). This is likely due to the *NUDT15* variant is more frequent in whites and St. Jude Total Therapy XV had a higher percentage of whites than the AALL03N1 cohort. Taken together, we conclude that we had reasonable statistical power to evaluate both *TPMT* and *NUDT15* variants despite the differences in race/ethnicity between the discovery and validation cohorts.

In conclusion, we systematically discovered novel variants in *TPMT* and *NUDT15* that are associated with 6MP sensitivity, indicating that comprehensive characterization of *TPMT* and *NUDT15* variants will improve pharmacogenetics-based thiopurine dose individualization.

## Supplementary Material

## Acknowledgements

We thank the patients and parents who participated in these clinical studies. This work was supported by the National Institutes of Health grants GM118578 (JJY), GM115279 (JJY, WEE, and MVR), and CA096670 (SB), the American Lebanese Syrian Associated Charities of St. Jude Children’s Research Hospital, the V Foundation for Cancer Research, and the Alex’s Lemonade Stand Foundation (T. Moriyama). The Genotype-Tissue Expression (GTEx) Project was supported by the Common Fund of the Office of the Director of the National Institutes of Health, and by NCI, NHGRI, NHLBI, NIDA, NIMH, and NINDS. The data used for the analyses described in this manuscript were obtained from the GTEx Portal on 12/02/2020.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Vora A et al. Treatment reduction for children and young adults with low-risk acute lymphoblastic leukaemia defined by minimal residual disease (UKALL 2003): a randomised controlled trial. Lancet Oncol 14, 199–209 (2013).  [DOI](https://doi.org/10.1016/S1470-2045(12)70600-9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23395119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Treatment%20reduction%20for%20children%20and%20young%20adults%20with%20low-risk%20acute%20lymphoblastic%20leukaemia%20defined%20by%20minimal%20residual%20disease%20(UKALL%202003):%20a%20randomised%20controlled%20trial.&author=A%20Vora&volume=14&publication_year=2013&pages=199-209&pmid=23395119&doi=10.1016/S1470-2045(12)70600-9&)

2. Pui CH, Carroll WL, Meshinchi S & Arceci RJ Biology, risk stratification, and therapy of pediatric acute leukemias: an update. J Clin Oncol 29, 551–65 (2011).  [DOI](https://doi.org/10.1200/JCO.2010.30.7405) | [PMC free article](/articles/PMC3071256/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21220611/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Biology,%20risk%20stratification,%20and%20therapy%20of%20pediatric%20acute%20leukemias:%20an%20update.&author=CH%20Pui&author=WL%20Carroll&author=S%20Meshinchi&author=RJ%20Arceci&volume=29&publication_year=2011&pages=551-65&pmid=21220611&doi=10.1200/JCO.2010.30.7405&)

3. Maltzman JS & Koretzky GA Azathioprine: old drug, new actions. J Clin Invest 111, 1122–4 (2003).  [DOI](https://doi.org/10.1172/JCI18384) | [PMC free article](/articles/PMC152947/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12697731/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=Azathioprine:%20old%20drug,%20new%20actions.&author=JS%20Maltzman&author=GA%20Koretzky&volume=111&publication_year=2003&pages=1122-4&pmid=12697731&doi=10.1172/JCI18384&)

4. Karran P & Attard N Thiopurines in current medical practice: molecular mechanisms and contributions to therapy-related cancer. Nat Rev Cancer 8, 24–36 (2008).  [DOI](https://doi.org/10.1038/nrc2292) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18097462/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&title=Thiopurines%20in%20current%20medical%20practice:%20molecular%20mechanisms%20and%20contributions%20to%20therapy-related%20cancer.&author=P%20Karran&author=N%20Attard&volume=8&publication_year=2008&pages=24-36&pmid=18097462&doi=10.1038/nrc2292&)

5. de Boer NK, van Bodegraven AA, Jharap B, de Graaf P & Mulder CJ Drug Insight: pharmacology and toxicity of thiopurine therapy in patients with IBD. Nat Clin Pract Gastroenterol Hepatol 4, 686–94 (2007).  [DOI](https://doi.org/10.1038/ncpgasthep1000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18043678/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Clin%20Pract%20Gastroenterol%20Hepatol&title=Drug%20Insight:%20pharmacology%20and%20toxicity%20of%20thiopurine%20therapy%20in%20patients%20with%20IBD.&author=NK%20de%20Boer&author=AA%20van%20Bodegraven&author=B%20Jharap&author=P%20de%20Graaf&author=CJ%20Mulder&volume=4&publication_year=2007&pages=686-94&pmid=18043678&doi=10.1038/ncpgasthep1000&)

6. Goldberg R & Irving PM Toxicity and response to thiopurines in patients with inflammatory bowel disease. Expert Rev Gastroenterol Hepatol 9, 891–900 (2015).  [DOI](https://doi.org/10.1586/17474124.2015.1039987) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25915575/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Rev%20Gastroenterol%20Hepatol&title=Toxicity%20and%20response%20to%20thiopurines%20in%20patients%20with%20inflammatory%20bowel%20disease.&author=R%20Goldberg&author=PM%20Irving&volume=9&publication_year=2015&pages=891-900&pmid=25915575&doi=10.1586/17474124.2015.1039987&)

7. Connell WR, Kamm MA, Ritchie JK & Lennard-Jones JE Bone marrow toxicity caused by azathioprine in inflammatory bowel disease: 27 years of experience. Gut 34, 1081–5 (1993).  [DOI](https://doi.org/10.1136/gut.34.8.1081) | [PMC free article](/articles/PMC1374358/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8174958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut&title=Bone%20marrow%20toxicity%20caused%20by%20azathioprine%20in%20inflammatory%20bowel%20disease:%2027%20years%20of%20experience.&author=WR%20Connell&author=MA%20Kamm&author=JK%20Ritchie&author=JE%20Lennard-Jones&volume=34&publication_year=1993&pages=1081-5&pmid=8174958&doi=10.1136/gut.34.8.1081&)

8. Schmiegelow K, Nielsen SN, Frandsen TL & Nersting J Mercaptopurine/Methotrexate maintenance therapy of childhood acute lymphoblastic leukemia: clinical facts and fiction. J Pediatr Hematol Oncol 36, 503–17 (2014).  [DOI](https://doi.org/10.1097/MPH.0000000000000206) | [PMC free article](/articles/PMC4222610/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24936744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pediatr%20Hematol%20Oncol&title=Mercaptopurine/Methotrexate%20maintenance%20therapy%20of%20childhood%20acute%20lymphoblastic%20leukemia:%20clinical%20facts%20and%20fiction.&author=K%20Schmiegelow&author=SN%20Nielsen&author=TL%20Frandsen&author=J%20Nersting&volume=36&publication_year=2014&pages=503-17&pmid=24936744&doi=10.1097/MPH.0000000000000206&)

9. Lennard L, Cartwright CS, Wade R & Vora A Thiopurine dose intensity and treatment outcome in childhood lymphoblastic leukaemia: the influence of thiopurine methyltransferase pharmacogenetics. Br J Haematol 169, 228–40 (2015).  [DOI](https://doi.org/10.1111/bjh.13240) | [PMC free article](/articles/PMC4737107/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25441457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Haematol&title=Thiopurine%20dose%20intensity%20and%20treatment%20outcome%20in%20childhood%20lymphoblastic%20leukaemia:%20the%20influence%20of%20thiopurine%20methyltransferase%20pharmacogenetics.&author=L%20Lennard&author=CS%20Cartwright&author=R%20Wade&author=A%20Vora&volume=169&publication_year=2015&pages=228-40&pmid=25441457&doi=10.1111/bjh.13240&)

10. Relling MV et al. Mercaptopurine therapy intolerance and heterozygosity at the thiopurine S-methyltransferase gene locus. J Natl Cancer Inst 91, 2001–8 (1999).  [DOI](https://doi.org/10.1093/jnci/91.23.2001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10580024/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Mercaptopurine%20therapy%20intolerance%20and%20heterozygosity%20at%20the%20thiopurine%20S-methyltransferase%20gene%20locus.&author=MV%20Relling&volume=91&publication_year=1999&pages=2001-8&pmid=10580024&doi=10.1093/jnci/91.23.2001&)

11. Lennard L, Lilleyman JS, Van Loon J & Weinshilboum RM Genetic variation in response to 6-mercaptopurine for childhood acute lymphoblastic leukaemia. Lancet 336, 225–9 (1990).  [DOI](https://doi.org/10.1016/0140-6736(90)91745-v) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1973780/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genetic%20variation%20in%20response%20to%206-mercaptopurine%20for%20childhood%20acute%20lymphoblastic%20leukaemia.&author=L%20Lennard&author=JS%20Lilleyman&author=J%20Van%20Loon&author=RM%20Weinshilboum&volume=336&publication_year=1990&pages=225-9&pmid=1973780&doi=10.1016/0140-6736(90)91745-v&)

12. Gardiner SJ, Gearry RB, Begg EJ, Zhang M & Barclay ML Thiopurine dose in intermediate and normal metabolizers of thiopurine methyltransferase may differ three-fold. Clin Gastroenterol Hepatol 6, 654–60; quiz 604 (2008).  [DOI](https://doi.org/10.1016/j.cgh.2008.02.032) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18467186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Gastroenterol%20Hepatol&title=Thiopurine%20dose%20in%20intermediate%20and%20normal%20metabolizers%20of%20thiopurine%20methyltransferase%20may%20differ%20three-fold.&author=SJ%20Gardiner&author=RB%20Gearry&author=EJ%20Begg&author=M%20Zhang&author=ML%20Barclay&volume=6&publication_year=2008&pages=654-60&pmid=18467186&doi=10.1016/j.cgh.2008.02.032&)

13. Schwab M et al. Azathioprine therapy and adverse drug reactions in patients with inflammatory bowel disease: impact of thiopurine S-methyltransferase polymorphism. Pharmacogenetics 12, 429–36 (2002).  [DOI](https://doi.org/10.1097/00008571-200208000-00003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12172211/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Azathioprine%20therapy%20and%20adverse%20drug%20reactions%20in%20patients%20with%20inflammatory%20bowel%20disease:%20impact%20of%20thiopurine%20S-methyltransferase%20polymorphism.&author=M%20Schwab&volume=12&publication_year=2002&pages=429-36&pmid=12172211&doi=10.1097/00008571-200208000-00003&)

14. Relling MV, Pui CH, Cheng C & Evans WE Thiopurine methyltransferase in acute lymphoblastic leukemia. Blood 107, 843–4 (2006).  [DOI](https://doi.org/10.1182/blood-2005-08-3379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16401827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Thiopurine%20methyltransferase%20in%20acute%20lymphoblastic%20leukemia.&author=MV%20Relling&author=CH%20Pui&author=C%20Cheng&author=WE%20Evans&volume=107&publication_year=2006&pages=843-4&pmid=16401827&doi=10.1182/blood-2005-08-3379&)

15. Yang SK et al. A common missense variant in NUDT15 confers susceptibility to thiopurine-induced leukopenia. Nat Genet 46, 1017–20 (2014).  [DOI](https://doi.org/10.1038/ng.3060) | [PMC free article](/articles/PMC4999337/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25108385/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20common%20missense%20variant%20in%20NUDT15%20confers%20susceptibility%20to%20thiopurine-induced%20leukopenia.&author=SK%20Yang&volume=46&publication_year=2014&pages=1017-20&pmid=25108385&doi=10.1038/ng.3060&)

16. Yang JJ et al. Inherited NUDT15 variant is a genetic determinant of mercaptopurine intolerance in children with acute lymphoblastic leukemia. J Clin Oncol 33, 1235–42 (2015).  [DOI](https://doi.org/10.1200/JCO.2014.59.4671) | [PMC free article](/articles/PMC4375304/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25624441/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Inherited%20NUDT15%20variant%20is%20a%20genetic%20determinant%20of%20mercaptopurine%20intolerance%20in%20children%20with%20acute%20lymphoblastic%20leukemia.&author=JJ%20Yang&volume=33&publication_year=2015&pages=1235-42&pmid=25624441&doi=10.1200/JCO.2014.59.4671&)

17. Moriyama T et al. NUDT15 polymorphisms alter thiopurine metabolism and hematopoietic toxicity. Nat Genet 48, 367–73 (2016).  [DOI](https://doi.org/10.1038/ng.3508) | [PMC free article](/articles/PMC5029084/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26878724/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=NUDT15%20polymorphisms%20alter%20thiopurine%20metabolism%20and%20hematopoietic%20toxicity.&author=T%20Moriyama&volume=48&publication_year=2016&pages=367-73&pmid=26878724&doi=10.1038/ng.3508&)

18. Moriyama T et al. The effects of inherited NUDT15 polymorphisms on thiopurine active metabolites in Japanese children with acute lymphoblastic leukemia. Pharmacogenet Genomics 27, 236–239 (2017).  [DOI](https://doi.org/10.1097/FPC.0000000000000282) | [PMC free article](/articles/PMC5510236/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28445187/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=The%20effects%20of%20inherited%20NUDT15%20polymorphisms%20on%20thiopurine%20active%20metabolites%20in%20Japanese%20children%20with%20acute%20lymphoblastic%20leukemia.&author=T%20Moriyama&volume=27&publication_year=2017&pages=236-239&pmid=28445187&doi=10.1097/FPC.0000000000000282&)

19. Walker GJ et al. Association of Genetic Variants in NUDT15 With Thiopurine-Induced Myelosuppression in Patients With Inflammatory Bowel Disease. JAMA 321, 773–785 (2019).  [DOI](https://doi.org/10.1001/jama.2019.0709) | [PMC free article](/articles/PMC6439872/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30806694/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Association%20of%20Genetic%20Variants%20in%20NUDT15%20With%20Thiopurine-Induced%20Myelosuppression%20in%20Patients%20With%20Inflammatory%20Bowel%20Disease.&author=GJ%20Walker&volume=321&publication_year=2019&pages=773-785&pmid=30806694&doi=10.1001/jama.2019.0709&)

20. Valerie NC et al. NUDT15 Hydrolyzes 6-Thio-DeoxyGTP to Mediate the Anticancer Efficacy of 6-Thioguanine. Cancer Res 76, 5501–11 (2016).  [DOI](https://doi.org/10.1158/0008-5472.CAN-16-0584) | [PMC free article](/articles/PMC6847052/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27530327/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=NUDT15%20Hydrolyzes%206-Thio-DeoxyGTP%20to%20Mediate%20the%20Anticancer%20Efficacy%20of%206-Thioguanine.&author=NC%20Valerie&volume=76&publication_year=2016&pages=5501-11&pmid=27530327&doi=10.1158/0008-5472.CAN-16-0584&)

21. Carter M et al. Crystal structure, biochemical and cellular activities demonstrate separate functions of MTH1 and MTH2. Nat Commun 6, 7871 (2015).  [DOI](https://doi.org/10.1038/ncomms8871) | [PMC free article](/articles/PMC4532830/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26238318/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Crystal%20structure,%20biochemical%20and%20cellular%20activities%20demonstrate%20separate%20functions%20of%20MTH1%20and%20MTH2.&author=M%20Carter&volume=6&publication_year=2015&pages=7871&pmid=26238318&doi=10.1038/ncomms8871&)

22. Stocco G et al. Genetic polymorphism of inosine triphosphate pyrophosphatase is a determinant of mercaptopurine metabolism and toxicity during treatment for acute lymphoblastic leukemia. Clin Pharmacol Ther 85, 164–72 (2009).  [DOI](https://doi.org/10.1038/clpt.2008.154) | [PMC free article](/articles/PMC2762405/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18685564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genetic%20polymorphism%20of%20inosine%20triphosphate%20pyrophosphatase%20is%20a%20determinant%20of%20mercaptopurine%20metabolism%20and%20toxicity%20during%20treatment%20for%20acute%20lymphoblastic%20leukemia.&author=G%20Stocco&volume=85&publication_year=2009&pages=164-72&pmid=18685564&doi=10.1038/clpt.2008.154&)

23. Hartford C et al. Differential effects of targeted disruption of thiopurine methyltransferase on mercaptopurine and thioguanine pharmacodynamics. Cancer Res 67, 4965–72 (2007).  [DOI](https://doi.org/10.1158/0008-5472.CAN-06-3508) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17510427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Differential%20effects%20of%20targeted%20disruption%20of%20thiopurine%20methyltransferase%20on%20mercaptopurine%20and%20thioguanine%20pharmacodynamics.&author=C%20Hartford&volume=67&publication_year=2007&pages=4965-72&pmid=17510427&doi=10.1158/0008-5472.CAN-06-3508&)

24. Relling MV et al. Clinical Pharmacogenetics Implementation Consortium Guideline for Thiopurine Dosing Based on TPMT and NUDT15 Genotypes: 2018 Update. Clin Pharmacol Ther 105, 1095–1105 (2019).  [DOI](https://doi.org/10.1002/cpt.1304) | [PMC free article](/articles/PMC6576267/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30447069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20Implementation%20Consortium%20Guideline%20for%20Thiopurine%20Dosing%20Based%20on%20TPMT%20and%20NUDT15%20Genotypes:%202018%20Update.&author=MV%20Relling&volume=105&publication_year=2019&pages=1095-1105&pmid=30447069&doi=10.1002/cpt.1304&)

25. Bhatia S et al. 6MP adherence in a multiracial cohort of children with acute lymphoblastic leukemia: a Children’s Oncology Group study. Blood 124, 2345–53 (2014).  [DOI](https://doi.org/10.1182/blood-2014-01-552166) | [PMC free article](/articles/PMC4192748/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24829202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=6MP%20adherence%20in%20a%20multiracial%20cohort%20of%20children%20with%20acute%20lymphoblastic%20leukemia:%20a%20Children%E2%80%99s%20Oncology%20Group%20study.&author=S%20Bhatia&volume=124&publication_year=2014&pages=2345-53&pmid=24829202&doi=10.1182/blood-2014-01-552166&)

26. Bhatia S et al. Nonadherence to oral mercaptopurine and risk of relapse in Hispanic and non-Hispanic white children with acute lymphoblastic leukemia: a report from the children’s oncology group. J Clin Oncol 30, 2094–101 (2012).  [DOI](https://doi.org/10.1200/JCO.2011.38.9924) | [PMC free article](/articles/PMC3601449/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22564992/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Nonadherence%20to%20oral%20mercaptopurine%20and%20risk%20of%20relapse%20in%20Hispanic%20and%20non-Hispanic%20white%20children%20with%20acute%20lymphoblastic%20leukemia:%20a%20report%20from%20the%20children%E2%80%99s%20oncology%20group.&author=S%20Bhatia&volume=30&publication_year=2012&pages=2094-101&pmid=22564992&doi=10.1200/JCO.2011.38.9924&)

27. Pui CH et al. Treating childhood acute lymphoblastic leukemia without cranial irradiation. N Engl J Med 360, 2730–41 (2009).  [DOI](https://doi.org/10.1056/NEJMoa0900386) | [PMC free article](/articles/PMC2754320/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19553647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Treating%20childhood%20acute%20lymphoblastic%20leukemia%20without%20cranial%20irradiation.&author=CH%20Pui&volume=360&publication_year=2009&pages=2730-41&pmid=19553647&doi=10.1056/NEJMoa0900386&)

28. Liu C et al. Genomewide Approach Validates Thiopurine Methyltransferase Activity Is a Monogenic Pharmacogenomic Trait. Clin Pharmacol Ther 101, 373–381 (2017).  [DOI](https://doi.org/10.1002/cpt.463) | [PMC free article](/articles/PMC5309133/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27564568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genomewide%20Approach%20Validates%20Thiopurine%20Methyltransferase%20Activity%20Is%20a%20Monogenic%20Pharmacogenomic%20Trait.&author=C%20Liu&volume=101&publication_year=2017&pages=373-381&pmid=27564568&doi=10.1002/cpt.463&)

29. Genomes Project C et al. A global reference for human genetic variation. Nature 526, 68–74 (2015).  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation.&author=C%20Genomes%20Project&volume=526&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

30. Xu H et al. Inherited coding variants at the CDKN2A locus influence susceptibility to acute lymphoblastic leukaemia in children. Nat Commun 6, 7553 (2015).  [DOI](https://doi.org/10.1038/ncomms8553) | [PMC free article](/articles/PMC4544058/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26104880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Inherited%20coding%20variants%20at%20the%20CDKN2A%20locus%20influence%20susceptibility%20to%20acute%20lymphoblastic%20leukaemia%20in%20children.&author=H%20Xu&volume=6&publication_year=2015&pages=7553&pmid=26104880&doi=10.1038/ncomms8553&)

31. DePristo MA et al. A framework for variation discovery and genotyping using next-generation DNA sequencing data. Nat Genet 43, 491–8 (2011).  [DOI](https://doi.org/10.1038/ng.806) | [PMC free article](/articles/PMC3083463/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21478889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data.&author=MA%20DePristo&volume=43&publication_year=2011&pages=491-8&pmid=21478889&doi=10.1038/ng.806&)

32. Falush D, Stephens M & Pritchard JK Inference of population structure using multilocus genotype data: dominant markers and null alleles. Mol Ecol Notes 7, 574–578 (2007).  [DOI](https://doi.org/10.1111/j.1471-8286.2007.01758.x) | [PMC free article](/articles/PMC1974779/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18784791/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Ecol%20Notes&title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data:%20dominant%20markers%20and%20null%20alleles.&author=D%20Falush&author=M%20Stephens&author=JK%20Pritchard&volume=7&publication_year=2007&pages=574-578&pmid=18784791&doi=10.1111/j.1471-8286.2007.01758.x&)

33. Suiter CC et al. Massively parallel variant characterization identifies NUDT15 alleles associated with thiopurine toxicity. Proc Natl Acad Sci U S A 117, 5394–5401 (2020).  [DOI](https://doi.org/10.1073/pnas.1915680117) | [PMC free article](/articles/PMC7071893/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32094176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Massively%20parallel%20variant%20characterization%20identifies%20NUDT15%20alleles%20associated%20with%20thiopurine%20toxicity.&author=CC%20Suiter&volume=117&publication_year=2020&pages=5394-5401&pmid=32094176&doi=10.1073/pnas.1915680117&)

34. Zhan X, Hu Y, Li B, Abecasis GR & Liu DJ RVTESTS: an efficient and comprehensive tool for rare variant association analysis using sequence data. Bioinformatics 32, 1423–6 (2016).  [DOI](https://doi.org/10.1093/bioinformatics/btw079) | [PMC free article](/articles/PMC4848408/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27153000/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=RVTESTS:%20an%20efficient%20and%20comprehensive%20tool%20for%20rare%20variant%20association%20analysis%20using%20sequence%20data.&author=X%20Zhan&author=Y%20Hu&author=B%20Li&author=GR%20Abecasis&author=DJ%20Liu&volume=32&publication_year=2016&pages=1423-6&pmid=27153000&doi=10.1093/bioinformatics/btw079&)

35. Consortium GT The Genotype-Tissue Expression (GTEx) project. Nat Genet 45, 580–5 (2013).  [DOI](https://doi.org/10.1038/ng.2653) | [PMC free article](/articles/PMC4010069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23715323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20Genotype-Tissue%20Expression%20(GTEx)%20project.&author=GT%20Consortium&volume=45&publication_year=2013&pages=580-5&pmid=23715323&doi=10.1038/ng.2653&)

36. Evans WE Pharmacogenetics of thiopurine S-methyltransferase and thiopurine therapy. Ther Drug Monit 26, 186–91 (2004).  [DOI](https://doi.org/10.1097/00007691-200404000-00018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15228163/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther%20Drug%20Monit&title=Pharmacogenetics%20of%20thiopurine%20S-methyltransferase%20and%20thiopurine%20therapy.&author=WE%20Evans&volume=26&publication_year=2004&pages=186-91&pmid=15228163&doi=10.1097/00007691-200404000-00018&)

37. Evans WE et al. Preponderance of thiopurine S-methyltransferase deficiency and heterozygosity among patients intolerant to mercaptopurine or azathioprine. J Clin Oncol 19, 2293–301 (2001).  [DOI](https://doi.org/10.1200/JCO.2001.19.8.2293) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11304783/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Preponderance%20of%20thiopurine%20S-methyltransferase%20deficiency%20and%20heterozygosity%20among%20patients%20intolerant%20to%20mercaptopurine%20or%20azathioprine.&author=WE%20Evans&volume=19&publication_year=2001&pages=2293-301&pmid=11304783&doi=10.1200/JCO.2001.19.8.2293&)

38. Krynetski EY et al. A single point mutation leading to loss of catalytic activity in human thiopurine S-methyltransferase. Proc Natl Acad Sci U S A 92, 949–53 (1995).  [DOI](https://doi.org/10.1073/pnas.92.4.949) | [PMC free article](/articles/PMC42614/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7862671/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=A%20single%20point%20mutation%20leading%20to%20loss%20of%20catalytic%20activity%20in%20human%20thiopurine%20S-methyltransferase.&author=EY%20Krynetski&volume=92&publication_year=1995&pages=949-53&pmid=7862671&doi=10.1073/pnas.92.4.949&)

39. Tai HL et al. Thiopurine S-methyltransferase deficiency: two nucleotide transitions define the most prevalent mutant allele associated with loss of catalytic activity in Caucasians. Am J Hum Genet 58, 694–702 (1996).  [PMC free article](/articles/PMC1914689/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8644731/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Thiopurine%20S-methyltransferase%20deficiency:%20two%20nucleotide%20transitions%20define%20the%20most%20prevalent%20mutant%20allele%20associated%20with%20loss%20of%20catalytic%20activity%20in%20Caucasians.&author=HL%20Tai&volume=58&publication_year=1996&pages=694-702&pmid=8644731&)

40. Szumlanski C et al. Thiopurine methyltransferase pharmacogenetics: human gene cloning and characterization of a common polymorphism. DNA Cell Biol 15, 17–30 (1996).  [DOI](https://doi.org/10.1089/dna.1996.15.17) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8561894/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=DNA%20Cell%20Biol&title=Thiopurine%20methyltransferase%20pharmacogenetics:%20human%20gene%20cloning%20and%20characterization%20of%20a%20common%20polymorphism.&author=C%20Szumlanski&volume=15&publication_year=1996&pages=17-30&pmid=8561894&doi=10.1089/dna.1996.15.17&)

41. Ujiie S, Sasaki T, Mizugaki M, Ishikawa M & Hiratsuka M Functional characterization of 23 allelic variants of thiopurine S-methyltransferase gene (TPMT*2 - *24). Pharmacogenet Genomics 18, 887–93 (2008).  [DOI](https://doi.org/10.1097/FPC.0b013e3283097328) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18708949/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Functional%20characterization%20of%2023%20allelic%20variants%20of%20thiopurine%20S-methyltransferase%20gene%20(TPMT*2%20-%20*24).&author=S%20Ujiie&author=T%20Sasaki&author=M%20Mizugaki&author=M%20Ishikawa&author=M%20Hiratsuka&volume=18&publication_year=2008&pages=887-93&pmid=18708949&doi=10.1097/FPC.0b013e3283097328&)

42. Salavaggione OE, Wang L, Wiepert M, Yee VC & Weinshilboum RM Thiopurine S-methyltransferase pharmacogenetics: variant allele functional and comparative genomics. Pharmacogenet Genomics 15, 801–15 (2005).  [DOI](https://doi.org/10.1097/01.fpc.0000174788.69991.6b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16220112/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Thiopurine%20S-methyltransferase%20pharmacogenetics:%20variant%20allele%20functional%20and%20comparative%20genomics.&author=OE%20Salavaggione&author=L%20Wang&author=M%20Wiepert&author=VC%20Yee&author=RM%20Weinshilboum&volume=15&publication_year=2005&pages=801-15&pmid=16220112&doi=10.1097/01.fpc.0000174788.69991.6b&)

43. Hon YY et al. Polymorphism of the thiopurine S-methyltransferase gene in African-Americans. Hum Mol Genet 8, 371–6 (1999).  [DOI](https://doi.org/10.1093/hmg/8.2.371) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9931346/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Polymorphism%20of%20the%20thiopurine%20S-methyltransferase%20gene%20in%20African-Americans.&author=YY%20Hon&volume=8&publication_year=1999&pages=371-6&pmid=9931346&doi=10.1093/hmg/8.2.371&)

44. Garat A et al. Characterisation of novel defective thiopurine S-methyltransferase allelic variants. Biochem Pharmacol 76, 404–15 (2008).  [DOI](https://doi.org/10.1016/j.bcp.2008.05.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18602085/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&title=Characterisation%20of%20novel%20defective%20thiopurine%20S-methyltransferase%20allelic%20variants.&author=A%20Garat&volume=76&publication_year=2008&pages=404-15&pmid=18602085&doi=10.1016/j.bcp.2008.05.009&)

45. Matreyek KA et al. Multiplex assessment of protein variant abundance by massively parallel sequencing. Nat Genet 50, 874–882 (2018).  [DOI](https://doi.org/10.1038/s41588-018-0122-z) | [PMC free article](/articles/PMC5980760/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29785012/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Multiplex%20assessment%20of%20protein%20variant%20abundance%20by%20massively%20parallel%20sequencing.&author=KA%20Matreyek&volume=50&publication_year=2018&pages=874-882&pmid=29785012&doi=10.1038/s41588-018-0122-z&)
