# Contribution of common and rare genetic variants in CEP72 on vincristine‐induced peripheral neuropathy in brain tumour patients

## Metadata
**Authors:** Marije J Klumpers, Annouk C A M Brand, Marina Hakobjan, Giovanna Gattuso, Elisabetta Schiavello, Monica Terenziani, Maura Massimino, Corrie E M Gidding, Henk‐Jan Guchelaar, D Maroeska W M te Loo, Marieke J H Coenen
**Journal:** British Journal of Clinical Pharmacology
**Date:** 2022 Mar 24
**DOI:** [10.1111/bcp.15267](https://doi.org/10.1111/bcp.15267)
**PMID:** 35150001
**PMCID:** PMC9313903
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9313903/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9313903/pdf/BCP-88-3463.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9313903/pdf/BCP-88-3463.pdf)

## Abstract

**Aims:** 
Studies implicated a role for a genetic variant in CEP72 in vincristine‐induced peripheral neuropathy. This study aims to evaluate this association in a cohort of brain tumour patients, to perform a cross‐disease meta‐analysis and explore the protein‐coding region of CEP72.

**Methods:** 
In total, 104 vincristine‐treated brain tumour patients were genotyped for CEP72 rs924607, and sequenced for the protein‐coding region. Data regarding patient and treatment characteristics, and peripheral neuropathy, were collected. Logistic regression and meta‐analysis were performed for rs924607 replication. A weighted burden analysis was applied to evaluate impact of overall genetic variation in CEP72.

**Results:** 
Analysis of 24 cases and 80 controls did not show a significant association between CEP72 rs924607 and neuropathy (odds ratio, OR [95% confidence interval, CI] 2.076 [0.359–11.989], P = .414). When combined with 8 cohorts (1095 cancer patients), a significant increase in risk for neuropathy was found for patients with a TT genotype (OR [95% CI] 2.15 [1.35–3.43], P = .001). Additionally, a missense variant (rs12522955) was significantly associated (OR [95% CI] 2.3 [1.2–4.4], P = .041) and patients with severe neuropathy carried more impactful variants in CEP72 coding regions (P = .039).

**Conclusion:** 
The association of CEP72 rs924607 in vincristine‐induced neuropathy was not confirmed in a cohort of brain tumour patients, but did contribute to its suggested effect when combined in a cross‐disease meta‐analysis. The importance of other genetic variations in CEP72 on vincristine‐induced neuropathy was demonstrated. This study contributes to evidence of the importance of genetic variants in CEP72 in development of vincristine‐induced toxicity, and provides guidance for future prospective studies.

Keywords: brain tumours, CEP72, neuropathy, pharmacogenetics, vincristine

### Aims

Studies implicated a role for a genetic variant in *CEP72* in vincristine‐induced peripheral neuropathy. This study aims to evaluate this association in a cohort of brain tumour patients, to perform a cross‐disease meta‐analysis and explore the protein‐coding region of *CEP72*.

### Methods

In total, 104 vincristine‐treated brain tumour patients were genotyped for *CEP72* rs924607, and sequenced for the protein‐coding region. Data regarding patient and treatment characteristics, and peripheral neuropathy, were collected. Logistic regression and meta‐analysis were performed for rs924607 replication. A weighted burden analysis was applied to evaluate impact of overall genetic variation in *CEP72*.

### Results

Analysis of 24 cases and 80 controls did not show a significant association between *CEP72* rs924607 and neuropathy (odds ratio, OR [95% confidence interval, CI] 2.076 [0.359–11.989], *P* = .414). When combined with 8 cohorts (1095 cancer patients), a significant increase in risk for neuropathy was found for patients with a TT genotype (OR [95% CI] 2.15 [1.35–3.43], *P* = .001). Additionally, a missense variant (rs12522955) was significantly associated (OR [95% CI] 2.3 [1.2–4.4], *P* = .041) and patients with severe neuropathy carried more impactful variants in *CEP72* coding regions (*P* = .039).

### Conclusion

The association of *CEP72* rs924607 in vincristine‐induced neuropathy was not confirmed in a cohort of brain tumour patients, but did contribute to its suggested effect when combined in a cross‐disease meta‐analysis. The importance of other genetic variations in *CEP72* on vincristine‐induced neuropathy was demonstrated. This study contributes to evidence of the importance of genetic variants in *CEP72* in development of vincristine‐induced toxicity, and provides guidance for future prospective studies.

## What is already known about this subject

## What is already known about this subject

## What this study adds

## 1. INTRODUCTION

[Vincristine](https://www.guidetopharmacology.org/GRAC/LigandDisplayForward?ligandId=6785) is a widely used chemotherapeutic agent for the treatment of both haematological and solid malignancies.[1](#bcp15267-bib-0001) Vincristine‐induced peripheral neuropathy (VIPN) is the main side effect, which is characterized by motor dysfunction, typically combined with abnormal sensations and/or neuropathic pain, which often leads to disruption of the curative treatment and a reduced quality of life of patients.[2](#bcp15267-bib-0002), [3](#bcp15267-bib-0003), [4](#bcp15267-bib-0004) Several factors are known to influence the incidence and severity of VIPN, including vincristine dosage regimen, age, malnutrition and interactions with concomitantly used drugs.[5](#bcp15267-bib-0005), [6](#bcp15267-bib-0006), [7](#bcp15267-bib-0007), [8](#bcp15267-bib-0008) Unfortunately, previous studies investigating the exact relationship between these clinical factors and VIPN showed inconclusive results and little progress has been made to predict VIPN occurrence or to develop effective preventive approaches.[3](#bcp15267-bib-0003), [9](#bcp15267-bib-0009)

To investigate how genetic variation contributes to this interpatient variability in VIPN, genetic association studies have been performed in the past. In 2015, a genome‐wide association study (GWAS) reported that variant rs924607 in the promoter region of the *Centrosomal Protein 72* (*CEP72*) gene was statistically significantly associated with both risk and severity of VIPN in paediatric patients with acute lymphoblastic leukaemia (ALL).[7](#bcp15267-bib-0007) It was furthermore shown that the risk allele (T) creates a binding site for a transcription factor repressor, which leads to lower *CEP72* mRNA levels, and that reduced mRNA levels increased sensitivity to vincristine in iPSC‐derived neurons.[7](#bcp15267-bib-0007) Additional studies confirmed the association between *CEP72* rs924607 and VIPN,[10](#bcp15267-bib-0010), [11](#bcp15267-bib-0011), [12](#bcp15267-bib-0012) while others failed to detect a significant association.[13](#bcp15267-bib-0013), [14](#bcp15267-bib-0014), [15](#bcp15267-bib-0015), [16](#bcp15267-bib-0016) All these studies were performed in haematological malignancies, predominantly ALL. Vincristine is also commonly used in treatment regimens of other cancers, such as paediatric brain tumours. However, the association between VIPN and genetic variants is largely unknown in patients with other vincristine‐treated malignancies, and one should be cautious with direct translation of findings due to differences in treatment regimens.[17](#bcp15267-bib-0017) Understanding VIPN is particularly important in brain tumour patients, as these patients are already at risk for motor problems (ataxia and other gait disorders) and sensory impairment (optical and auditory) due to tumour localization and side effects of other facets of their treatment, resulting in additional impact of VIPN.

The present study investigated the association between *CEP72* rs924607 and VIPN in a cohort of brain tumour patients, and performed a meta‐analysis including existing studies.[7](#bcp15267-bib-0007), [10](#bcp15267-bib-0010), [11](#bcp15267-bib-0011), [12](#bcp15267-bib-0012), [13](#bcp15267-bib-0013), [14](#bcp15267-bib-0014), [15](#bcp15267-bib-0015), [16](#bcp15267-bib-0016) To date, studies have only focused on this specific variant, despite the fact that, within a gene, often multiple different variants have the potential to affect protein function and structure. This is clearly depicted by existing pharmacogenetic guidelines, where often a multitude of variants within a gene are actionable.[18](#bcp15267-bib-0018), [19](#bcp15267-bib-0019) The majority of these actionable variants are located in a gene's coding region. By sequencing the coding region of *CEP72* and analysing common and rare variants, this study assessed if other genetic variants with potential impact could predispose patients to increase risk of VIPN. The overall aim of this study was to investigate the impact of previously identified variant in *CEP72* on VIPN in a brain tumour cohort, and identify novel variants within this gene with potential clinical impact.

## 2. METHODS

### 2.1. Patients and treatment

The study cohort consisted of 104 medulloblastoma and low‐grade glioma patients who were treated between 2000 and 2016 at the Radboud university medical center in Nijmegen, the Netherlands, or at the Fondazione IRCCS Istituto Nazionale Tumori in Milan, Italy. Patients were included if their primary treatment regimen contained vincristine. Exclusion criteria were: no reliable data on neuropathy available; diabetes mellitus at baseline; and/neuropathy at baseline. Also, patients with (central) neuropathic disorders prior to the start of chemotherapeutic treatment, caused by tumour localization, surgery and/or cranial radiotherapy, were excluded from this study as assessment of VIPN could not be performed in a reliable way in these patients. Patients' clinical information concerning patient and treatment characteristics, and information for assessment of neuropathy, were retrieved from the hospital records. The current study was approved by the institutional review board of the Radboud university medical center (Commissie Mensgebonden Onderzoek Regio Arnhem Nijmegen), and the inclusion of patients from Fondazione IRCCS Istituto Nazionale Tumori in Milan, Italy was approved by its own institutional ethics committee. Written informed consent was obtained from the patients and/or their parents/legal guardians.

### 2.2. Assessment of neuropathy

Assessment and grading of VIPN was done retrospectively by the use of medical records. Notes of treating paediatric oncologists were screened for mention of neuropathic pain, numbness, paraesthesia, altered fine motor skills and/or limb weakness. VIPN was considered if the onset of the symptom(s) developed after the first administration of vincristine, and there was a high likelihood of being vincristine‐induced as being stated in the note of the paediatric oncologist (with fitting physical examination), and/or as shown by a consequent vincristine dose reduction. The occurrence of VIPN was retrospectively assessed by a clinical pharmacologist blinded for the genetic data. Peripheral motor neuropathy, peripheral sensory neuropathy and neuropathic pain (neuralgia) were individually graded according to the NCI Common Terminology Criteria for Adverse Events (CTCAE) version 4.0 ([Supplementary S1](#bcp15267-supitem-0001)). A patients' overall VIPN grade was determined by their highest (worst) grade for either of the three classifications. Case–control designation was in line with previous studies, meaning that patients with grade 0 and 1 overall VIPN were included in the control group and those with grade 2 or higher were considered to be cases.

### 2.3. Replication of CEP72 rs924607

Germline DNA was extracted from saliva (collected using GeneFiX DNA Saliva Collector GFX‐02, Isohelix, UK) and isolated with ChemagicStar (Hamilton Robotics, Reno, NV, USA), using Chemagic STAR DNA Saliva 4 k Kit, according to the manufacturer protocol. Details of the genotyping procedure of *CEP72* rs924607 is provided in [Supplementary S2](#bcp15267-supitem-0002).

Power calculations were performed using Quanto (version 1.2.4, Los Angeles, CA, USA). Other statistical analyses were conducted using SPSS Statistics (version 25.0, IBM Corp.). For all analyses, a 2‐sided *P*‐value of <.05 was considered to be statistically significant. Potential associations between clinical characteristics and the occurrence of VIPN were analysed using Pearson's χ^2^, Fisher's exact, independent samples *t* or Mann–Whitney *U*, depending on the type of data and the Gaussian distribution. To test the association between VIPN and *CEP72* rs924607, multivariate logistic regression was performed with associated clinical variables included as a covariate in the model. A recessive genetic model was used (testing CC/CT genotypes *vs*. TT genotype), as this is the model reported by the discovery study to initially identify this variant.

### 2.4. Meta‐analysis

A meta‐analysis was performed including the results on the association of *CEP72* rs924607 and VIPN from this study, alongside results from earlier publications investigating this association. To identify eligible studies, a search in MEDLINE was performed in November 2021, using the keywords ‘CEP72’ and ‘vincristine’. The studies yielded by this search were screened for inclusion based on full text, and their reference lists were searched for additional eligible articles. Studies were included in the meta‐analysis if they investigated and reported on the association of *CEP72* rs924607 and VIPN (graded according to CTCAE) in patients with cancer. Additionally, authors of studies investigating this association, but using different clinical endpoints than CTCAE to define VIPN, were contacted to evaluate the possibility re‐grading to facilitate inclusion in the meta‐analysis. Reported sample sizes, odds ratios (OR) and 95% confidence intervals (95% CI) of multivariate logistic regression analyses (CC/CT genotype *vs*. TT genotype) of included studies were used as input. Meta‐analysis was performed in Review Manager version 5.3.5.[20](#bcp15267-bib-0020) The choice for a fixed or random effects model was depending on the amount of heterogeneity (I^2^), which translates into the proportion of total variation contributed by variation between included studies.

### 2.5. Analysis of CEP72 protein‐coding region

The *CEP72* protein‐coding region and their flanking intronic regions were sequenced in all patients. Forward and reverse primers ([Supplementary S3](#bcp15267-supitem-0003)) were designed for each of the 12 exons (Sigma‐Aldrich, Saint Louis, MO, USA) with the use of Primer3Plus,[21](#bcp15267-bib-0021) and were optimized using gradient polymerase chain reaction. After amplification of the DNA samples, the products were purified using Exonuclease I (ExoI) and FastAP Thermosensitive Alkaline Phosphatase (Thermo Fisher Scientific, Waltham, MA, USA). Exons were sequenced with one of the primers using Sanger sequencing on the Big Dye Terminator version 3, according to the manufacturer's protocol (Applied Biosystems, Foster City, CA, USA). Sequence visualization and variant calling was performed using in Vector NTI Advance version 11.0 (Invitrogen Corp., Carlsbad, CA, USA). Samples with ambiguous sequencing results were re‐sequenced for that exon. All variant calling steps were performed twice by 2 members of the research team. For each patient, every identified variant was coded as reference homozygous [0], heterozygous [1] or variant homozygous [2], representing the number of variant alleles, or missing in case of repeated ambiguous sequencing results. For each variant, the Hardy–Weinberg equilibrium was tested for deviation using a χ^2^ test, and variants were excluded if *P* < .05. Linkage disequilibrium between variants was evaluated using LDlink.[22](#bcp15267-bib-0022)

All variants were annotated using variant effect predictor (VEP),[23](#bcp15267-bib-0023) and exonic and splice‐side variants were selected. Based on variant allele frequencies in our cohort, common variants (allele frequency >1%) were selected to investigate if they show a significant association with VIPN. To include more phenotypic information in these analyses, it was upfront decided to investigate VIPN as an ordinal variable (i.e. different CTCAE grades instead of a case–control designation). Analyses of common coding variants was done by performing ordinal logistic regression, including clinical variables included as covariates if needed. Correction for multiple testing was not applied as these were secondary hypothesis generating analyses.

To evaluate the potential impact of all coding region variants in *CEP72* on VIPN (including rare variants), a weighted burden analysis was performed. As described by Curtis,[24](#bcp15267-bib-0024), [25](#bcp15267-bib-0025) instead of taking the sum of all variants in a gene and comparing this total number of variants between different phenotypes, a weighted burden test takes the VEP annotation and the *in silico* prediction scores of each variant into account to allow them to be weighted more highly based on their predicted impact on protein structure and function. To perform this analysis, each coding variant was scored for predicted effect on protein function using SIFT and PolyPhen‐2.[26](#bcp15267-bib-0026), [27](#bcp15267-bib-0027) These scores, and the VEP annotation of each variant, were used to calculate a weighted *CEP72* genetic score for each patient. For example, a weight of 10 was assigned to missense variants, and a weight of 5 to synonymous variants (assigned weight for each VEP annotation is provided in [Supplementary S4](#bcp15267-supitem-0004)). An additional weight of 10 was added if Polyphen‐2 predicted the variant to be *possibly damaging* or *probably damaging*, or if the SIFT prediction was *deleterious*. The total score composed the weight for each variant. No additional weight was added to variants with lower variant allele frequencies compared to more common variants (as is usually done with these analyses), since our phenotype of interest is drug‐induced, which diminishes the argument that common variants are less likely to contribute to the phenotype than rare variants. Following the method of Curtis,[24](#bcp15267-bib-0024) a genetic score (*g*) was calculated for each patient (*j*) by multiplying the number of variant alleles (*I*, being a value of 0, 1 or 2) for each variant (*i*) with its corresponding weight (*W*), and then calculating the sum of the weighted allele counts of all coding variants:

| gj=∑i=1nWiIij |
| --- |
The genetic scores were compared between different grades of VIPN using a parametric (one‐way ANOVA) or nonparametric (Kruskal–Wallis) test, depending on data normality and homogeneity of variances between groups. Patients with one or more missing genotypes for one or more variant were excluded from this analysis.

## 3. RESULTS

### 3.1. Patient characteristics

A total of 24 cases (i.e. overall VIPN grade ≥ 2) and 80 controls (i.e. overall VIPN grade 0 and 1) were included in this study. Demographics and treatment characteristics of this cohort are shown in Table [1](#bcp15267-tbl-0001). The majority of VIPN cases were defined as grade 2 (*n =* 21), followed by grade 3 (*n =* 3), and none of the cases experienced grade 4 VIPN. Treatment protocol and age at diagnosis were statistically significantly associated with the occurrence of VIPN, therefore included in further regression models. According to all treatment protocols, vincristine dose per course was 1.5 mg/m^2^ with a maximum of 2.0 mg ([Supplementary S5](#bcp15267-supitem-0005)). However, 77.3% of cases had at least 1 vincristine dose reduction or course cancellation during treatment as a result of vincristine‐induced toxicities, including VIPN but also other common side effects such as obstipation and stomach ache. Similar modifications happened for only 36.8% of controls. The median cumulative vincristine dose was 25.9 mg/m^2^, being 25.7 mg/m^2^ for cases and 26.5 mg/m^2^ for controls.

### TABLE 1.

|   | All patients n = 104 | VIPN grade ≥ 2 n = 24 | VIPN grade 0–1 n = 80 | P d |
| --- | --- | --- | --- | --- |
| Diagnosis, n (%) |   |   |   | .733 |
| Medulloblastoma | 90 (86.5) | 20 (83.3) | 70 (87.5) |   |
| Low‐grade glioma | 14 (13.5) | 4 (16.7) | 10 (12.5) |   |
| Treatment center, n (%) |   |   |   | .129 |
| Radboud university medical center, Nijmegen (NL) | 30 (28.8) | 10 (41.7) | 20 (25.0) |   |
| Fondazione IRCCS Istituto Nazionale Tumori, Milan (IT) | 74 (71.2) | 14 (58.3) | 60 (75.0) |   |
| Age at diagnosis (y), mean (range) | 11.0 (0.5–47.0) | 17.4 (3.4–47.0) | 9.1 (0.5–28.0) | .004 |
| Sex, n of males (%) | 55 (52.9) | 13 (54.2) | 42 (52.5) | >.999 |
| Self‐reported ethnicity, n of Caucasians (%) | 102 (98.1) | 24 (100) | 78 (97.5) | >.999 |
| Treatment protocol a |   |   |   | .043 |
| Vincristine cumulative dose (mg/m2), median (range) | 25.9 (1.4–126.0) | 25.7 (1.5–66.0) | 26.5 (1.4–126.0) | .589 |
| Vincristine number of cycles, mean (range) | 22.4 (2.0–76.0) | 26.6 (8.0–44.0) | 21.2 (2.0–76.0) | .089 |
| Regimen including cisplatin, n (%) | 44 (42.3%) | 15 (62.5%) | 29 (43.8%) | .037 |
| Cisplatin cumulative dose (mg/m2), mean (range) | 350.0 (120.0–560.0) | 404.12 (180.0–560.0) | 330.2 (120.0–560.0) | .098 |
| Use of antifungal azoles, b n (%) | 21 (20.2) | 4 (17.4) | 17 (21.8) | .776 |
| Use of strong CYP3A4 inhibitors, c n (%) | 0 (0.0) | 0 (0.0) | 0 (0.0) | ‐ |

Table 1 Caption: Demographics of 104 brain tumour patients

### 3.2. CEP72 rs924607 replication

Genotyping of *CEP72* rs924607 was successful for all 104 patients, with a minor allele (T) frequency of 0.41, which is in line with European reference populations.[28](#bcp15267-bib-0028) Thirty‐four patients (32.7%) were homozygous for the major allele (CC), 54 patients (51.9%) were heterozygous (CT), and 16 (15.4%) were homozygous for the minor allele (TT). The frequencies of these genotypes are in Hardy–Weinberg equilibrium (*P* = .47). For this cohort, a power calculation based on the strongest association found in the first study by Diouf *et al*. showed a power of 0.74 for an odds ratio of 4.1 and allele frequency of 0.36.[7](#bcp15267-bib-0007) Although the TT genotype frequency was slightly higher in cases than controls as shown in Figure [1](#bcp15267-fig-0001) (16.7 *vs*. 15%, respectively), no statistically significant association was observed in multivariate logistic regression analysis (CC/CT genotype *vs*. TT genotype; OR 2.076 [95% CI 0.359–11.989], *P* = .414). Additional explorative analyses under the assumption of an additive or dominant genetic model also did not show a statistically significant association.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/26cc/9313903/05ba0dda2e72/BCP-88-3463-g001.jpg)

Percentage of patients who developed vincristine‐induced peripheral neuropathy (VIPN; grade 2 or higher, dark blue bars), and who did not develop VIPN (grade 0, light blue bars), with on the x‐axis CEP72 rs924607 genotyping under the assumption of a recessive genetic model (CC/CT vs. TT). The number within de bar indicates the number of patients. No statistically significant difference was observed in the frequency of the TT genotype between VIPN cases and controls. Multivariate logistic regression analysis (with treatment protocol and age at diagnosis included as covariates) did not show a statistically significant association (CC/CT genotype vs. TT genotype; OR 2.076 [95% CI 0.359–11.989] P = .414)

### 3.3. Meta‐analysis

Literature search yielded a total of eighteen studies, of which six met the inclusion criteria ([Supplementary S6](#bcp15267-supitem-0006)).[7](#bcp15267-bib-0007), [10](#bcp15267-bib-0010), [11](#bcp15267-bib-0011), [13](#bcp15267-bib-0013), [14](#bcp15267-bib-0014), [15](#bcp15267-bib-0015) One additional study was eligible, except for using a different endpoint than CTCAE graded peripheral neuropathy.[12](#bcp15267-bib-0012) Contact with the authors resulted in sharing of genotyping data and raw clinical data, including patient‐reported symptoms of peripheral neuropathy, and accompanying physical examination. Grading of these clinical data was performed independently by two members of this study's group, and dissimilarities were discussed afterwards until consensus was reached. Patients with insufficient or ambiguous clinical data were excluded. Including our own study (and two separate cohorts by Diouf *et al*.[7](#bcp15267-bib-0007)), this resulted in a total of nine cohorts in the meta‐analysis, and a total of 399 cases and 696 controls. Table [2](#bcp15267-tbl-0002) shows the characteristics of included studies. All used a case–control designation of CTCAE grade 0–1 *vs*. grade 2–4, except Wright *et al*.[10](#bcp15267-bib-0010) (*refined* CTCAE grade 0 *vs*. grade 2–4). Also, 1 of Diouf *et al*.’s cohorts (COG AALL0433) was graded according to modified CTCAE scale (*Balis*).[7](#bcp15267-bib-0007) Heterogeneity between studies was moderately large (I^2^ = 36%), but not significant (*P* = .13), resulting in the choice for a random effects model. Meta‐analysis of the included nine cohorts showed a statistically significant association between the *CEP72* rs924607 and VIPN (Figure [2](#bcp15267-fig-0002), OR 2.15 [95% CI 1.35–3.43], *P* = .001). Notably, one additional study, a GWAS by Li *et al*. investigating vincristine‐induced neuropathy in >1,000 patients, could not be included in this meta‐analysis due to lack of reported summary statistics on *CEP72* rs924607.[16](#bcp15267-bib-0016) It was stated in this study that this variant was not associated to VIPN, but the authors did not elaborate. To evaluate the possible impact of this study on meta‐analysis outcome, the known cohort size, population allele frequency of this variant and used *P‐*value cut off were used to simulate possible scenarios of outcome ([Supplementary S7](#bcp15267-supitem-0007)). These included a scenario of no effect (OR of 1), an effect in the same direction (OR>1) and in the opposite direction (OR<1). Based on these hypothetical calculations, one could conclude that as long as the direction of effect of the association in the study by Li *et al*.[16](#bcp15267-bib-0016) is the same as in the meta‐analysis, or no effect (OR of 1 with a 95% CI of 0.75–1.33 or larger), there remains a statistically significant association between the *CEP72* variant and VIPN in overall meta‐analysis.

### TABLE 2.

| Study (y)ref | Cases/total | Design a | Disease | Cohort | Sex | Ethnicities | VINC dose per course | Neuropathy assessment | Association between VIPN and CEP72 rs924607 c | Reported MAF of CEP72 rs924607 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| n | % male | According to protocol (mg/m2) | Used scale and case–control design |  |  |  |  |  |  |  |
| Diouf et al. (2015) 7 : St. Jude Total XIIIB | 64/222 | P | ALL | Children | 57.7 | EUR/AFR/Asian/HIS/other | 1.5 | CTCAE v1.0 0–1 vs. 2–4 | Yes | 0.367 |
| Diouf et al. (2015) 7 : COG AALL0433 | 22/99 | P | ALL | Children | 59.6 | EUR/AFR/Asian/HIS/other | 1.5 or 2.0 (depending on randomization) | Balis 0–1 vs. 2–4 | Yes | 0.364 |
| Gutierrez‐Camino et al. (2016) 13 | 36/142 | R | ALL | Children | 57.0 | EUR | 1.5 | CTCAE v1.0 0–1 vs. 2–4 | No | 0.394 |
| Stock et al. (2017) 11 | 48/96 | R/P | ALL | Adults | 54.2 | HIS/non‐HIS | 1.5 | CTCAE v4.0 0–1 vs. 2–4 | Yes | 0.440 f |
| Zgheib et al. (2018) 15 | 23/130 | R | ALL | Children | 57.1 | ARAB | 1.5 and 2.0 (depending on treatment phase) | CTCAE v4.0 0–1 vs. 2–4 | No | 0.369 |
| Wright et al. (2019) 10 | 167/224 | R | ALL | Children | 60.4/ 40.4 b | EUR/AFR/Asian | Not stated | CTCAE v4.0 (refined) 0 vs. 2–4 | Yes d | 0.377 |
| Sawaki et al. (2020) 14 | 9/56 | R | MBCL | Adults | 59.0 | JAP | Not stated | CTCAE v3.0/v4.0 0–1 vs. 2–4 | No | 0.429 |
| Kavčič et al. (2020) 12 | 9/25 | P/R | ALL /HL/NHL/RMS/EW/WT | Children/adolescents | 50.0 | EUR | Not stated | CTCAE v4.0 0–1 vs. 2–4 (graded by current study's group) | No e | 0.512 |
| Current study | 24/104 | R | MB/LGG | Children/adults | 52.9 | EUR | 1.5 | CTCAE v4.0 0–1 vs. 2–4 | No | 0.410 |

Table 2 Caption: Characteristics of cohorts investigating the association between CEP72 rs924607 and vincristine‐induced peripheral neuropathy, included in the meta‐analysis

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/26cc/9313903/86c01153539f/BCP-88-3463-g002.jpg)

Meta‐analysis (random effects model) of nine cohorts investigating the association between CEP72 rs924607 and risk for development of vincristine‐induced peripheral neuropathy (VIPN), resulting in a statistically significant effect of TT genotype on risk of development of VIPN

### 3.4. Sequencing of CEP72 coding regions

Sanger sequencing of the *CEP72* resulted in identification of a total of 46 variants, of which 31 were intronic variants (including 6 indels in intronic regions) and 2 3′‐untranslated region variants, as these were present in the flanking regions of the sequenced exons. These variants were not further investigated in this study. The remaining 13 variants consisted of 11 missense variants, 1 synonymous variant and 1 splice region variant (Table [3](#bcp15267-tbl-0003)), and were all known in dbSNP.[28](#bcp15267-bib-0028) Sequencing was successful for 95.9% of all variants, with the number of missing genotypes differing per variant. None of the identified variants showed deviation from the Hardy–Weinberg equilibrium, and all were not in substantial in linkage disequilibrium (highest *R* ^2^ = 0.079).

### TABLE 3.

| Variant | Exon | Position a | Ref | Alt | Consequence | VEP annotation | SIFT prediction | PolyPhen‐2 prediction | MAF b | Variant weight c |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| rs138365408 | 2 | 619 210 | G | A | p.Arg63His | missense_variant | Deleterious | Possibly damaging | 0.005 | 20 |
| rs62001006 | 4 | 624 587 | C | T | c.405C > T(p.=) | splice_region_variant | ‐ | ‐ | 0.010 | 5 |
| rs773198799 | 5 | 633 958 | C | G | p.Ala196Gly | missense_variant | Deleterious | Probably damaging | 0.005 | 20 |
| rs869955 | 6 | 635 508 | C | T | p.Pro238Leu | missense_variant | Deleterious | Probably damaging | 0.005 | 20 |
| rs140416835 | 7 | 637 668 | A | G | p.Lys314Arg | missense_variant | Deleterious | Possibly damaging | 0.005 | 20 |
| rs62000999 | 7 | 637 637 | A | G | p.Met304Val | missense_variant | Tolerated | Benign | 0.021 | 10 |
| rs150376362 | 7 | 637 673 | G | A | p.Asp316Asn | missense_variant | Tolerated | Benign | 0.005 | 10 |
| rs1296780032 | 7 | 637 688 | A | G | p.Met321Val | missense_variant | Tolerated | Benign | 0.005 | 10 |
| rs12522955 | 8 | 639 231 | C | A | p.Pro412Thr | missense_variant | Deleterious | Probably damaging | 0.188 | 20 |
| rs868649 | 9 | 640 705 | A | G | p.Thr509Ala | missense_variant | Tolerated | Benign | 0.193 | 10 |
| rs138955347 | 9 | 640 669 | C | G | p.Arg497Gly | missense_variant | Tolerated | Benign | 0.005 | 10 |
| rs62001010 | 9 | 640 704 | C | T | p.His508His | synonymous_variant | ‐ | ‐ | 0.005 | 5 |
| rs62000998 | 12 | 653 145 | C | G | p.Ser607Arg | missense_variant | Deleterious | Possibly damaging | 0.031 | 20 |

Table 3 Caption: Characteristics of identified variants in coding regions of CEP72

### 3.5. Common variants and weighted burden analysis

VIPN was treated as ordinal variable in the following analyses. Therefore, clinical variables were again tested for association ([Supplementary S8](#bcp15267-supitem-0008)). An increase in age in groups with more severe VIPN was observed, therefore this was included in the statistical model. Vincristine‐related variables (being dose per course and number of courses) were not included in further analyses, as differences between groups can be both the cause (dose‐dependent effect of vincristine) and the consequence (reducing vincristine doses and omitting courses due to toxicity) of VIPN. Most importantly, no major clinical differences were observed between high and lower grades regarding concomitant medication (e.g. use of strongly interacting azoles) and vincristine cumulative doses.

Out of the 13 identified coding variants in *CEP72, 2* variants (rs12522955 and rs868649) were common with variant allele frequencies of 0.188 and 0.193, respectively (Table [3](#bcp15267-tbl-0003)). *In silico* tools (SIFT and Polyphen‐2) predicted rs12522955 to be *Deleterious* and *Probably damaging*, and rs868649 to be *Tolerated* and *Benign*.[26](#bcp15267-bib-0026), [27](#bcp15267-bib-0027) Multivariate ordinal regression analysis (including age) with an additive model resulted in a statistically significant association between rs12522955 and VIPN (CC *vs*. CA *vs*. AA; OR 2.3, 95% CI 1.2–4.4, *P* = .014), showing a higher variant allele (A) frequency in higher (more severe) grades of VIPN (Table [4](#bcp15267-tbl-0004)). An even stronger effect was found when applying a dominant model (CC *vs*. CA/AA; OR 3.0, 95% CI 1.4–6.7, *P* = .006), but none when making use of a recessive genetic model (CC/CA *vs*. AA). For rs868649, a variant predicted to have no impact on protein function and structure, no statistically significant associations were identified by any of the genetic models. Univariate ordinal regression analyses investigating these 2 variants showed similar results ([Supplementary S9](#bcp15267-supitem-0009)). Two other identified variants in *CEP72* (rs62000999 and rs62000998) had allele frequencies just above 0.01 (classifying them as *common*), but absolute allele counts for these variants were too low to perform meaningful statistical testing.

### TABLE 4.

| Variant | Grade 0 |  | Grade 1 |  | Grade 2 |  | Grade 3 |  | Genetic model | Multivariate ordinal regression a |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| n | % b | n | % b | n | % b | n | % b | OR | 95% CI | p |  |  |
| rs12522955 |  |  |  |  |  |  |  |  |  |  |  |  |
| CC | 34 | 51.5% | 20 | 30.3% | 12 | 18.2% | 0 | 0.0% | Additive | 2.3 | 1.2–4.4 | .014 |
| CA | 8 | 26.7% | 12 | 40.0% | 8 | 26.7% | 2 | 6.7% | Dominant | 3.0 | 1.4–6.7 | .006 |
| AA | 1 | 20.0% | 3 | 60.0% | 0 | 0.0% | 1 | 20.0% | Recessive | 1.8 | 0.3–9.6 | .484 |
| rs868649 |  |  |  |  |  |  |  |  |  |  |  |  |
| AA | 26 | 40.0% | 23 | 35.4% | 13 | 20.0% | 3 | 4.6% | Additive | 0.9 | 0.4–2.0 | .846 |
| AG | 16 | 48.5% | 11 | 33.3% | 6 | 18.2% | 0 | 0.0% | Dominant | 0.9 | 0.4–1.9 | .726 |
| GG | 1 | 50.0% | 0 | 0.0% | 1 | 50.0% | 0 | 0.0% | Recessive | 2.5 | 0.1–52.0 | .562 |

Table 4 Caption: Analysis of common variants in CEP72 coding regions, comparing genotype frequencies between different grades of vincristine‐induced peripheral neuropathy

For the weighted burden analysis, samples were included if they did not have missing genotypes for any of the identified coding variants, resulting in 96 out of 104 (92.3%) samples. Each variant was assigned a certain weight according to its VEP annotation and its predicted impact on protein function and structure. The total assigned weight per variant is shown in Table [3](#bcp15267-tbl-0003). A weighted genotype score (sum of all weighted allele counts) was calculated for each patient. Genetic scores ranged from 0 to 60 and, when comparing the different VIPN groups, a statistically significant increase in the mean genetic scores with increase of VIPN severity was observed (*P* = .039, Figure [3](#bcp15267-fig-0003)).

### FIGURE 3.

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/26cc/9313903/42fd6f348856/BCP-88-3463-g003.jpg)

Weighted genotype analysis of CEP72 and vincristine‐induced peripheral neuropathy (VIPN). The top of the bar chart represents the mean weighted genotype score in each group and the whiskers show the corresponding standard deviation. A statistically significant difference (P = .039) was observed between the different groups

## 4. DISCUSSION AND CONCLUSIONS

*CEP72* rs924607 has been implicated to play a predisposing role to VIPN, but remained ambiguous in patients from non‐North American descent, and has not been investigated in solid malignancies. This study did not find a statistically significant association between *CEP72* rs924607 and VIPN in a European brain tumour cohort. When combining this result in a cross‐disease meta‐analysis with 8 other cohorts (1098 patients), an overall risk effect of the *CEP72* rs924607 TT genotype on development of VIPN was found. Furthermore, this study looks for the first time beyond this specific variant and performed analysis of other genetic variants in *CEP72*. A missense variant (rs12522955, p.Pro412Thr) in *CEP72* was found to be significantly associated with higher grades of VIPN. Finally, when combining all common and rare variants in the *CEP72* protein coding region, patients with more severe VIPN have an increasingly higher burden of genetic variants compared to patients with no or mild VIPN.

The inability to detect a statistically significant association between *CEP72* rs924607 and VIPN in this study's cohort could have multiple causes. This is the first study to investigate this association in a brain tumour cohort. Although the vincristine dose per course is similar compared to previously studied haematological malignancies, treatment regimens are in essence different between diseases concerning vincristine course schemes and other (chemotherapeutic) treatment aspects.[1](#bcp15267-bib-0001) Moreover, in treatment of paediatric brain tumours, platinum agents are used in the vast majority of patients, which are also potentially neurotoxic (especially cisplatin).[29](#bcp15267-bib-0029) Although neuropathy was specifically graded with a high likelihood of being vincristine‐induced, and there was no significant difference in cisplatin usage between cases and controls, there is a chance that platinum agents have also impacted the neurons and therefore make them more prone for development of VIPN.[30](#bcp15267-bib-0030) Finally, due to investigating a rare disease, this study had limited power (0.74) to detect the association, despite the fact that all eligible brain tumour patients from our centres were included. Since the direction of effect of the association was similar to the previous studies, the hypothesis remains that this genetic variant might have an effect on VIPN in this population.

The meta‐analysis summarizes the overall effect of *CEP72* rs924607 on VIPN in patients with cancer, thereby and provides a clearer picture of its impact across cohorts with differences in disease, treatment phases, age and ethnicities. The result shows a statistically significant 2.2‐fold increase in VIPN risk in patients with a TT genotype compared to the CC/CT genotype, when combining 9 cohorts. Most of the cohorts (6 out of 9) indeed showed this risk effect (OR > 1), but only 3 were able to detect a statistically significant association on their own. This suggests that sample size could be an important factor in this, but other factors are also likely to play a role. Differences in studied cohorts (regarding patient and treatment characteristics) could influence the development and severity of neurotoxicity, and the impact of *CEP72* rs924607. Increase in age is often associated to increase in VIPN risk,[6](#bcp15267-bib-0006), [31](#bcp15267-bib-0031) as was observed in this study's cohort. Even though many studies included age in their statistical model, it could contribute to differences in findings of paediatric *vs*. adult cohorts. Population ethnicities play an important role in pharmacogenetic studies. The 3 studies that found a statistically significant association between VIPN and *CEP72* rs924607 were all performed in a (North‐)American population.[7](#bcp15267-bib-0007), [10](#bcp15267-bib-0010), [11](#bcp15267-bib-0011) However when looking at population genetics of *CEP72* rs924607, T‐allele frequencies are even higher in the European population (0.409) compared to American population (0.317), and therefore not likely to be fully explanatory for the observed differences in findings.[32](#bcp15267-bib-0032) One relatively large published GWAS could not be included due to lack of reported summary statistics on the association of the *CEP72* variant and VIPN, but simulated calculations did not show high impact of these results on overall meta‐analysis outcome.[16](#bcp15267-bib-0016) Despite the factors of heterogeneity between included studies, the results of the meta‐analyses clearly suggest this variant to be a promising biomarker to be investigated in prospective studies. To our knowledge, there is 1 ongoing prospective study ([ClinicalTrials.gov](http://ClinicalTrials.gov)[NCT03117751](https://clinicaltrials.gov/ct2/show/NCT03117751)).

This study is the first to investigate *CEP72* coding region variants and their impact in VIPN. Since Diouf *et al*. already showed that reduced *CEP72* mRNA levels increased sensitivity to vincristine in iPSC‐derived neurons,[7](#bcp15267-bib-0007) one could hypothesize that other genetic variants with transcriptional consequences could have similar impact on VIPN as the previously described risk variant. A missense variant with predicted damaging effect on protein function was identified to be statistically significantly associated with VIPN, showing a risk effect of the variant (A) allele compared to the reference (C) allele. These results were similar when treating VIPN as a binary outcome (grade 0–1 *vs*. grade 2–4), and independent from previously identified variant rs924607. Other pharmacogenetic studies have already shown the importance of investigating all functional variants in a gene to establish the overall impact on protein function and consequently, drug efficacy and/or toxicity. For example, for thiopurine drugs, multiple variants in *TPMT* and *NUDT15* have been identified in their relation to hematological toxicities, resulting in an extensive list of actionable genetic variants in clinical guidelines.[33](#bcp15267-bib-0033) CEP72 is hypothesized to be involved in vincristine pharmacodynamics, being on the level of the neuron, making establishment of the consequence of a genetic variant on the eventual phenotype less straightforward compared to proteins in pharmacokinetic pathways such asTPMT and NUDT15. However, this does not take away from the importance of identifying all relevant variants in this gene. Since the majority of identified coding region variants in *CEP72* were rare (11 out of 13), a weighted burden analyses was performed, resulting in a significant increase in number of impactful variants with increase of VIPN severity. This result suggests a combined impact of coding region variants, and/or solidary impact of 1 or more rare variants on CEP72 protein function. Since rs924607 is located in the promotor region of *CEP72*, this variant was not included in initial coding region analysis. Since it has been shown that this variant has functional impact on mRNA expression,[7](#bcp15267-bib-0007) an additional analysis was performed including this variant (with a weight of 20), but this does not have significant impact on the result. This study did not include sequencing and analyses of variants located outside coding regions. Although noncoding regions are known to contain genetic variants that could potentially influence gene expression, prediction and interpretation of functional impact of these variants is currently still too uncertain to include in a systematic analysis.

The main strength of this study is the investigated study population, being paediatric brain tumour patients. Although brain tumours are rare in children when looking at absolute numbers, they are the most common malignancy of childhood after ALL. Moreover, VIPN has additional impact in this population for the high incidence of motor and sensory impairment due to tumour localization and other treatment aspects. Cross‐disease investigation of *CEP72* genetic variants and their impact on VIPN was sparse, and is of great importance to pave the way towards biomarker implementation. A drawback of this study is phenotyping in a retrospective manner, which carries the risk of misclassification. Overall, worse VIPN during and after treatment, as was done in this study, is a commonly used endpoint, especially for retrospective studies, as it provides valid and clinically relevant information about which patients eventually develop VIPN. However, VIPN is known to be a dose‐dependent toxicity, so development of VIPN is not only a question of a patient's sensitivity, but also treatment duration and cumulative dose at moment of toxicity.[1](#bcp15267-bib-0001) Therefore, it is advised that future prospective studies should consider fixed timepoints for VIPN assessment and focus on cumulative dose at moment of toxicity to provide deeper phenotyping.

This study represents the first pharmacogenetic analysis of VIPN in a cohort exclusively consisting of brain tumours patients. Based on promising results of *CEP72* coding region analyses, it is advised that future studies focus on all genetic variants with potential protein impact in relation to VIPN, including replication of the newly identified variant rs12522955, as well as assessing the potential combined impact of common and rare variants in *CEP72*. In conclusion, this study provided insight in the role of *CEP72* in VIPN in brain tumour patients, and guidance for prospective studies investigating this gene and variants within as potential biomarkers for development and severity of VIPN.

## COMPETING INTERESTS

The authors declare no conflict of interest.

## CONTRIBUTORS

Conceptualization, M.K., M.L. and M.C.; methodology, M.K., M.L. and M.C.; software, M.K. and M.H.; formal analysis M.K. and A.B.; investigation, M.K., A.B. and M.H.; resources, M.L. and M.C.; data curation, M.K., G.G., E.S. and M.T.; writing—original draft preparation, M.K. and A.B..; writing, review and editing, G. G, E.S., M.T., M.M., C.G., H.G., M.L. and M.C.; visualization, M.K. and A.B.; supervision, M.M., H.G., M.L. and M.C..; project administration, M.L. and M.C.; funding acquisition, C.G., M.L. and M.C. All authors have read and agreed to the published version of the manuscript.

## PATIENT CONSENT

Written informed consent was obtained from the patients and/or their parents/legal guardians.

## Supporting information

## ACKNOWLEDGEMENTS

We would like to acknowledge all patients and parents who participated in this study. We acknowledge the contribution of Johanne M. Groothuismink (Department of Human Genetics, Radboud university medical center, Nijmegen, The Netherlands) for her guidance and assistance with genotyping and sequencing.

This work was funded by the Princess Máxima Center Foundation.

Klumpers MJ, Brand ACAM, Hakobjan M, et al. Contribution of common and rare genetic variants in *CEP72* on vincristine‐induced peripheral neuropathy in brain tumour patients. Br J Clin Pharmacol. 2022;88(7):3463-3473. doi: 10.1111/bcp.15267

## DATA AVAILABILITY STATEMENT

Data will be made available upon request. Data have been made publicly accessible via DOI [10.17605/OSF.IO/UKERD](http://10.17605/OSF.IO/UKERD).

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Data will be made available upon request. Data have been made publicly accessible via DOI [10.17605/OSF.IO/UKERD](http://10.17605/OSF.IO/UKERD).

### Supplementary Materials

### Data Availability Statement

Data will be made available upon request. Data have been made publicly accessible via DOI [10.17605/OSF.IO/UKERD](http://10.17605/OSF.IO/UKERD).

## References

1. Gidding CE, Kellie SJ, Kamps WA, de Graaf SS. Vincristine revisited. Crit Rev Oncol Hematol. 1999;29(3):267‐287. doi: 10.1016/S1040-8428(98)00023-7  [DOI](https://doi.org/10.1016/S1040-8428(98)00023-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10226730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit%20Rev%20Oncol%20Hematol&title=Vincristine%20revisited&author=CE%20Gidding&author=SJ%20Kellie&author=WA%20Kamps&author=SS%20de%20Graaf&volume=29&issue=3&publication_year=1999&pages=267-287&pmid=10226730&doi=10.1016/S1040-8428(98)00023-7&)

2. Gomber S, Dewan P, Chhonker D. Vincristine induced neurotoxicity in cancer patients. Indian J Pediatr. 2010;77(1):97‐100. doi: 10.1007/s12098-009-0254-3  [DOI](https://doi.org/10.1007/s12098-009-0254-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19936661/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indian%20J%20Pediatr&title=Vincristine%20induced%20neurotoxicity%20in%20cancer%20patients&author=S%20Gomber&author=P%20Dewan&author=D%20Chhonker&volume=77&issue=1&publication_year=2010&pages=97-100&pmid=19936661&doi=10.1007/s12098-009-0254-3&)

3. van de Velde ME, Kaspers GL, Abbink FCH, Wilhelm AJ, Ket JCF, van den Berg MH. Vincristine‐induced peripheral neuropathy in children with cancer: a systematic review. Crit Rev Oncol Hematol. 2017;114:114‐130. doi: 10.1016/j.critrevonc.2017.04.004  [DOI](https://doi.org/10.1016/j.critrevonc.2017.04.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28477739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit%20Rev%20Oncol%20Hematol&title=Vincristine%E2%80%90induced%20peripheral%20neuropathy%20in%20children%20with%20cancer:%20a%20systematic%20review&author=ME%20van%20de%20Velde&author=GL%20Kaspers&author=FCH%20Abbink&author=AJ%20Wilhelm&author=JCF%20Ket&volume=114&publication_year=2017&pages=114-130&pmid=28477739&doi=10.1016/j.critrevonc.2017.04.004&)

4. Bradley WG, Lassman LP, Pearce GW, Walton JN. The neuromyopathy of vincristine in man. Clinical, electrophysiological and pathological studies. J Neurol Sci. 1970;10(2):107‐131. doi: 10.1016/0022-510X(70)90013-4  [DOI](https://doi.org/10.1016/0022-510X(70)90013-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/4314181/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurol%20Sci&title=The%20neuromyopathy%20of%20vincristine%20in%20man.%20Clinical,%20electrophysiological%20and%20pathological%20studies&author=WG%20Bradley&author=LP%20Lassman&author=GW%20Pearce&author=JN%20Walton&volume=10&issue=2&publication_year=1970&pages=107-131&pmid=4314181&doi=10.1016/0022-510X(70)90013-4&)

5. Böhme A, Ganser A, Hoelzer D. Aggravation of vincristine‐induced neurotoxicity by itraconazole in the treatment of adult ALL. Ann Hematol. 1995;71(6):311‐312. doi: 10.1007/BF01697985  [DOI](https://doi.org/10.1007/BF01697985) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8534764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Hematol&title=Aggravation%20of%20vincristine%E2%80%90induced%20neurotoxicity%20by%20itraconazole%20in%20the%20treatment%20of%20adult%20ALL&author=A%20B%C3%B6hme&author=A%20Ganser&author=D%20Hoelzer&volume=71&issue=6&publication_year=1995&pages=311-312&pmid=8534764&doi=10.1007/BF01697985&)

6. Okada N, Hanafusa T, Sakurada T, et al. Risk factors for early‐onset peripheral neuropathy caused by vincristine in patients with a first administration of R‐CHOP or R‐CHOP‐like chemotherapy. J Clin Med Res. 2014;6(4):252‐260. doi: 10.14740/jocmr1856w  [DOI](https://doi.org/10.14740/jocmr1856w) | [PMC free article](/articles/PMC4039096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24883150/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Med%20Res&title=Risk%20factors%20for%20early%E2%80%90onset%20peripheral%20neuropathy%20caused%20by%20vincristine%20in%20patients%20with%20a%20first%20administration%20of%20R%E2%80%90CHOP%20or%20R%E2%80%90CHOP%E2%80%90like%20chemotherapy&author=N%20Okada&author=T%20Hanafusa&author=T%20Sakurada&volume=6&issue=4&publication_year=2014&pages=252-260&pmid=24883150&doi=10.14740/jocmr1856w&)

7. Diouf B, Crews KR, Lew G, et al. Association of an inherited genetic variant with vincristine‐related peripheral neuropathy in children with acute lymphoblastic leukemia. Jama. 2015;313(8):815‐823. doi: 10.1001/jama.2015.0894  [DOI](https://doi.org/10.1001/jama.2015.0894) | [PMC free article](/articles/PMC4377066/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25710658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Jama&title=Association%20of%20an%20inherited%20genetic%20variant%20with%20vincristine%E2%80%90related%20peripheral%20neuropathy%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=B%20Diouf&author=KR%20Crews&author=G%20Lew&volume=313&issue=8&publication_year=2015&pages=815-823&pmid=25710658&doi=10.1001/jama.2015.0894&)

8. Legha SS. Vincristine neurotoxicity. Pathophysiology and management. Med Toxicol. 1986;1(6):421‐427. doi: 10.1007/BF03259853  [DOI](https://doi.org/10.1007/BF03259853) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3540519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Toxicol&title=Vincristine%20neurotoxicity.%20Pathophysiology%20and%20management&author=SS%20Legha&volume=1&issue=6&publication_year=1986&pages=421-427&pmid=3540519&doi=10.1007/BF03259853&)

9. Mora E, Smith EM, Donohoe C, Hertz DL. Vincristine‐induced peripheral neuropathy in pediatric cancer patients. Am J Cancer Res. 2016;6(11):2416‐2430.  [PMC free article](/articles/PMC5126263/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27904761/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cancer%20Res&title=Vincristine%E2%80%90induced%20peripheral%20neuropathy%20in%20pediatric%20cancer%20patients&author=E%20Mora&author=EM%20Smith&author=C%20Donohoe&author=DL%20Hertz&volume=6&issue=11&publication_year=2016&pages=2416-2430&pmid=27904761&)

10. Wright GEB, Amstutz U, Drögemöller BI, et al. Pharmacogenomics of vincristine‐induced peripheral neuropathy implicates pharmacokinetic and inherited neuropathy genes. Clin Pharmacol Ther. 2019;105(2):402‐410. doi: 10.1002/cpt.1179  [DOI](https://doi.org/10.1002/cpt.1179) | [PMC free article](/articles/PMC6519044/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29999516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacogenomics%20of%20vincristine%E2%80%90induced%20peripheral%20neuropathy%20implicates%20pharmacokinetic%20and%20inherited%20neuropathy%20genes&author=GEB%20Wright&author=U%20Amstutz&author=BI%20Dr%C3%B6gem%C3%B6ller&volume=105&issue=2&publication_year=2019&pages=402-410&pmid=29999516&doi=10.1002/cpt.1179&)

11. Stock W, Diouf B, Crews KR, et al. An inherited genetic variant in CEP72 promoter predisposes to vincristine‐induced peripheral neuropathy in adults with acute lymphoblastic Leukemia. Clin Pharmacol Ther. 2017;101(3):391‐395. doi: 10.1002/cpt.506  [DOI](https://doi.org/10.1002/cpt.506) | [PMC free article](/articles/PMC5320866/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27618250/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=An%20inherited%20genetic%20variant%20in%20CEP72%20promoter%20predisposes%20to%20vincristine%E2%80%90induced%20peripheral%20neuropathy%20in%20adults%20with%20acute%20lymphoblastic%20Leukemia&author=W%20Stock&author=B%20Diouf&author=KR%20Crews&volume=101&issue=3&publication_year=2017&pages=391-395&pmid=27618250&doi=10.1002/cpt.506&)

12. Kavcic M, Zečkanović A, Jazbec J, Debeljak M. Association of CEP72 rs924607 TT genotype with vincristine‐induced peripheral neuropathy measured by motor nerve conduction studies. Klin Padiatr. 2020;232(06):331‐333. doi: 10.1055/a-1213-5995  [DOI](https://doi.org/10.1055/a-1213-5995) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32877958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Klin%20Padiatr&title=Association%20of%20CEP72%20rs924607%20TT%20genotype%20with%20vincristine%E2%80%90induced%20peripheral%20neuropathy%20measured%20by%20motor%20nerve%20conduction%20studies&author=M%20Kavcic&author=A%20Ze%C4%8Dkanovi%C4%87&author=J%20Jazbec&author=M%20Debeljak&volume=232&issue=06&publication_year=2020&pages=331-333&pmid=32877958&doi=10.1055/a-1213-5995&)

13. Gutierrez‐Camino A, Martin‐Guerrero I, Lopez‐Lopez E, et al. Lack of association of the CEP72 rs924607 TT genotype with vincristine‐related peripheral neuropathy during the early phase of pediatric acute lymphoblastic leukemia treatment in a Spanish population. Pharmacogenet Genomics. 2016;26(2):100‐102. doi: 10.1097/FPC.0000000000000191  [DOI](https://doi.org/10.1097/FPC.0000000000000191) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26618658/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Lack%20of%20association%20of%20the%20CEP72%20rs924607%20TT%20genotype%20with%20vincristine%E2%80%90related%20peripheral%20neuropathy%20during%20the%20early%20phase%20of%20pediatric%20acute%20lymphoblastic%20leukemia%20treatment%20in%20a%20Spanish%20population&author=A%20Gutierrez%E2%80%90Camino&author=I%20Martin%E2%80%90Guerrero&author=E%20Lopez%E2%80%90Lopez&volume=26&issue=2&publication_year=2016&pages=100-102&pmid=26618658&doi=10.1097/FPC.0000000000000191&)

14. Sawaki A, Miyazaki K, Yamaguchi M, et al. Genetic polymorphisms and vincristine‐induced peripheral neuropathy in patients treated with rituximab, cyclophosphamide, doxorubicin, vincristine, and prednisone therapy. Int J Hematol. 2020;111(5):686‐691. doi: 10.1007/s12185-020-02832-x  [DOI](https://doi.org/10.1007/s12185-020-02832-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31993939/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Hematol&title=Genetic%20polymorphisms%20and%20vincristine%E2%80%90induced%20peripheral%20neuropathy%20in%20patients%20treated%20with%20rituximab,%20cyclophosphamide,%20doxorubicin,%20vincristine,%20and%20prednisone%20therapy&author=A%20Sawaki&author=K%20Miyazaki&author=M%20Yamaguchi&volume=111&issue=5&publication_year=2020&pages=686-691&pmid=31993939&doi=10.1007/s12185-020-02832-x&)

15. Zgheib NK, Ghanem KM, Tamim H, et al. Genetic polymorphisms in candidate genes are not associated with increased vincristine‐related peripheral neuropathy in Arab children treated for acute childhood leukemia: a single institution study. Pharmacogenet Genomics. 2018;28(8):189‐195. doi: 10.1097/FPC.0000000000000345  [DOI](https://doi.org/10.1097/FPC.0000000000000345) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30119132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=Genetic%20polymorphisms%20in%20candidate%20genes%20are%20not%20associated%20with%20increased%20vincristine%E2%80%90related%20peripheral%20neuropathy%20in%20Arab%20children%20treated%20for%20acute%20childhood%20leukemia:%20a%20single%20institution%20study&author=NK%20Zgheib&author=KM%20Ghanem&author=H%20Tamim&volume=28&issue=8&publication_year=2018&pages=189-195&pmid=30119132&doi=10.1097/FPC.0000000000000345&)

16. Li L, Sajdyk T, Smith EML, et al. Genetic variants associated with vincristine‐induced peripheral neuropathy in two populations of children with acute lymphoblastic Leukemia. Clin Pharmacol Ther. 2019;105(6):1421‐1428. doi: 10.1002/cpt.1324  [DOI](https://doi.org/10.1002/cpt.1324) | [PMC free article](/articles/PMC6513686/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30506673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Genetic%20variants%20associated%20with%20vincristine%E2%80%90induced%20peripheral%20neuropathy%20in%20two%20populations%20of%20children%20with%20acute%20lymphoblastic%20Leukemia&author=L%20Li&author=T%20Sajdyk&author=EML%20Smith&volume=105&issue=6&publication_year=2019&pages=1421-1428&pmid=30506673&doi=10.1002/cpt.1324&)

17. Klumpers MJ, Coenen MJH, Gidding CEM, te Loo DMWM. The role of germline variants in chemotherapy outcome in brain tumors: a systematic review of pharmacogenetic studies. Pharmacogenomics. 2017;18(5):501‐513. doi: 10.2217/pgs-2016-0189  [DOI](https://doi.org/10.2217/pgs-2016-0189) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28346057/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=The%20role%20of%20germline%20variants%20in%20chemotherapy%20outcome%20in%20brain%20tumors:%20a%20systematic%20review%20of%20pharmacogenetic%20studies&author=MJ%20Klumpers&author=MJH%20Coenen&author=CEM%20Gidding&author=DMWM%20te%20Loo&volume=18&issue=5&publication_year=2017&pages=501-513&pmid=28346057&doi=10.2217/pgs-2016-0189&)

18. Relling MV, Klein TE, Gammal RS, Whirl‐Carrillo M, Hoffman JM, Caudle KE. The clinical Pharmacogenetics implementation consortium: 10 years later. Clin Pharmacol Ther. 2020;107(1):171‐175. doi: 10.1002/cpt.1651  [DOI](https://doi.org/10.1002/cpt.1651) | [PMC free article](/articles/PMC6925644/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31562822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=The%20clinical%20Pharmacogenetics%20implementation%20consortium:%2010%20years%20later&author=MV%20Relling&author=TE%20Klein&author=RS%20Gammal&author=M%20Whirl%E2%80%90Carrillo&author=JM%20Hoffman&volume=107&issue=1&publication_year=2020&pages=171-175&pmid=31562822&doi=10.1002/cpt.1651&)

19. Whirl‐Carrillo M, McDonagh EM, Hebert JM, et al. Pharmacogenomics knowledge for personalized medicine. Clin Pharmacol Ther. 2012;92(4):414‐417. doi: 10.1038/clpt.2012.96  [DOI](https://doi.org/10.1038/clpt.2012.96) | [PMC free article](/articles/PMC3660037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22992668/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacogenomics%20knowledge%20for%20personalized%20medicine&author=M%20Whirl%E2%80%90Carrillo&author=EM%20McDonagh&author=JM%20Hebert&volume=92&issue=4&publication_year=2012&pages=414-417&pmid=22992668&doi=10.1038/clpt.2012.96&)

20. Review Manager (RevMan) [Computer Program] . The Nordic Cochrane Centre. Copenhagen: The Cochrane Collaboration; 2014.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20Nordic%20Cochrane%20Centre&publication_year=2014&)

21. Untergasser A, Cutcutache I, Koressaar T, et al. Primer3‐‐new capabilities and interfaces. Nucleic Acids Res. 2012;40(15):e115. doi: 10.1093/nar/gks596  [DOI](https://doi.org/10.1093/nar/gks596) | [PMC free article](/articles/PMC3424584/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22730293/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Primer3%E2%80%90%E2%80%90new%20capabilities%20and%20interfaces&author=A%20Untergasser&author=I%20Cutcutache&author=T%20Koressaar&volume=40&issue=15&publication_year=2012&pages=e115&pmid=22730293&doi=10.1093/nar/gks596&)

22. Machiela MJ, Chanock SJ. LDlink: a web‐based application for exploring population‐specific haplotype structure and linking correlated alleles of possible functional variants. Bioinformatics. 2015;31(21):3555‐3557. doi: 10.1093/bioinformatics/btv402  [DOI](https://doi.org/10.1093/bioinformatics/btv402) | [PMC free article](/articles/PMC4626747/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26139635/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LDlink:%20a%20web%E2%80%90based%20application%20for%20exploring%20population%E2%80%90specific%20haplotype%20structure%20and%20linking%20correlated%20alleles%20of%20possible%20functional%20variants&author=MJ%20Machiela&author=SJ%20Chanock&volume=31&issue=21&publication_year=2015&pages=3555-3557&pmid=26139635&doi=10.1093/bioinformatics/btv402&)

23. McLaren W, Gil L, Hunt SE, et al. The Ensembl variant effect predictor. Genome Biol. 2016;17(1):122. doi: 10.1186/s13059-016-0974-4  [DOI](https://doi.org/10.1186/s13059-016-0974-4) | [PMC free article](/articles/PMC4893825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27268795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=The%20Ensembl%20variant%20effect%20predictor&author=W%20McLaren&author=L%20Gil&author=SE%20Hunt&volume=17&issue=1&publication_year=2016&pages=122&pmid=27268795&doi=10.1186/s13059-016-0974-4&)

24. Curtis D. A rapid method for combined analysis of common and rare variants at the level of a region, gene, or pathway. Adv Appl Bioinforma Chem. 2012;5:1‐9. doi: 10.2147/AABC.S33049  [DOI](https://doi.org/10.2147/AABC.S33049) | [PMC free article](/articles/PMC3413013/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22888262/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Appl%20Bioinforma%20Chem&title=A%20rapid%20method%20for%20combined%20analysis%20of%20common%20and%20rare%20variants%20at%20the%20level%20of%20a%20region,%20gene,%20or%20pathway&author=D%20Curtis&volume=5&publication_year=2012&pages=1-9&pmid=22888262&doi=10.2147/AABC.S33049&)

25. Curtis D. A weighted burden test using logistic regression for integrated analysis of sequence variants, copy number variants and polygenic risk score. Eur J Hum Genet. 2019;27(1):114‐124. doi: 10.1038/s41431-018-0272-6  [DOI](https://doi.org/10.1038/s41431-018-0272-6) | [PMC free article](/articles/PMC6303265/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30258123/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=A%20weighted%20burden%20test%20using%20logistic%20regression%20for%20integrated%20analysis%20of%20sequence%20variants,%20copy%20number%20variants%20and%20polygenic%20risk%20score&author=D%20Curtis&volume=27&issue=1&publication_year=2019&pages=114-124&pmid=30258123&doi=10.1038/s41431-018-0272-6&)

26. Kumar P, Henikoff S, Ng PC. Predicting the effects of coding non‐synonymous variants on protein function using the SIFT algorithm. Nat Protoc. 2009;4(7):1073‐1081. doi: 10.1038/nprot.2009.86  [DOI](https://doi.org/10.1038/nprot.2009.86) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19561590/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Predicting%20the%20effects%20of%20coding%20non%E2%80%90synonymous%20variants%20on%20protein%20function%20using%20the%20SIFT%20algorithm&author=P%20Kumar&author=S%20Henikoff&author=PC%20Ng&volume=4&issue=7&publication_year=2009&pages=1073-1081&pmid=19561590&doi=10.1038/nprot.2009.86&)

27. Adzhubei I, Jordan DM, Sunyaev SR. Predicting functional effect of human missense mutations using PolyPhen‐2. Curr Protoc Hum Genet. 2013;Chapter 7(1):Unit7 20. doi: 10.1002/0471142905.hg0720s76  [DOI](https://doi.org/10.1002/0471142905.hg0720s76) | [PMC free article](/articles/PMC4480630/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23315928/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Protoc%20Hum%20Genet&title=Predicting%20functional%20effect%20of%20human%20missense%20mutations%20using%20PolyPhen%E2%80%902&author=I%20Adzhubei&author=DM%20Jordan&author=SR%20Sunyaev&volume=Chapter%207&issue=1&publication_year=2013&pages=Unit7%2020&pmid=23315928&doi=10.1002/0471142905.hg0720s76&)

28. Sherry ST, Ward MH, Kholodov M, et al. dbSNP: the NCBI database of genetic variation. Nucleic Acids Res. 2001;29(1):308‐311. doi: 10.1093/nar/29.1.308  [DOI](https://doi.org/10.1093/nar/29.1.308) | [PMC free article](/articles/PMC29783/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11125122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=dbSNP:%20the%20NCBI%20database%20of%20genetic%20variation&author=ST%20Sherry&author=MH%20Ward&author=M%20Kholodov&volume=29&issue=1&publication_year=2001&pages=308-311&pmid=11125122&doi=10.1093/nar/29.1.308&)

29. Hartmann JT, Lipp HP. Toxicity of platinum compounds. Expert Opin Pharmacother. 2003;4(6):889‐901. doi: 10.1517/14656566.4.6.889  [DOI](https://doi.org/10.1517/14656566.4.6.889) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12783586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Pharmacother&title=Toxicity%20of%20platinum%20compounds&author=JT%20Hartmann&author=HP%20Lipp&volume=4&issue=6&publication_year=2003&pages=889-901&pmid=12783586&doi=10.1517/14656566.4.6.889&)

30. Zajaczkowska R, Kocot‐Kępska M, Leppert W, Wrzosek A, Mika J, Wordliczek J. Mechanisms of chemotherapy‐induced peripheral neuropathy. Int J Mol Sci. 2019;20(6):1451. doi: 10.3390/ijms20061451  [DOI](https://doi.org/10.3390/ijms20061451) | [PMC free article](/articles/PMC6471666/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30909387/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&title=Mechanisms%20of%20chemotherapy%E2%80%90induced%20peripheral%20neuropathy&author=R%20Zajaczkowska&author=M%20Kocot%E2%80%90K%C4%99pska&author=W%20Leppert&author=A%20Wrzosek&author=J%20Mika&volume=20&issue=6&publication_year=2019&pages=1451&pmid=30909387&doi=10.3390/ijms20061451&)

31. Lavoie Smith EM, Li L, Chiang CW, et al. Patterns and severity of vincristine‐induced peripheral neuropathy in children with acute lymphoblastic leukemia. J Peripher Nerv Syst. 2015;20(1):37‐46. doi: 10.1111/jns.12114  [DOI](https://doi.org/10.1111/jns.12114) | [PMC free article](/articles/PMC4610712/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25977177/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Peripher%20Nerv%20Syst&title=Patterns%20and%20severity%20of%20vincristine%E2%80%90induced%20peripheral%20neuropathy%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=EM%20Lavoie%20Smith&author=L%20Li&author=CW%20Chiang&volume=20&issue=1&publication_year=2015&pages=37-46&pmid=25977177&doi=10.1111/jns.12114&)

32. The 1000 Genomes Project Consortium . A global reference for human genetic variation. Nature. 2015;526(7571):68‐74. doi: 10.1038/nature15393  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation&volume=526&issue=7571&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

33. Relling MV, Schwab M, Whirl‐Carrillo M, et al. Clinical Pharmacogenetics implementation consortium guideline for Thiopurine dosing based on TPMT and NUDT15 genotypes: 2018 update. Clin Pharmacol Ther. 2019;105(5):1095‐1105. doi: 10.1002/cpt.1304  [DOI](https://doi.org/10.1002/cpt.1304) | [PMC free article](/articles/PMC6576267/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30447069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20implementation%20consortium%20guideline%20for%20Thiopurine%20dosing%20based%20on%20TPMT%20and%20NUDT15%20genotypes:%202018%20update&author=MV%20Relling&author=M%20Schwab&author=M%20Whirl%E2%80%90Carrillo&volume=105&issue=5&publication_year=2019&pages=1095-1105&pmid=30447069&doi=10.1002/cpt.1304&)
