# Identification and single-base gene-editing functional validation of a cis-EPO variant as a genetic predictor for EPO-increasing therapies

## Metadata
**Authors:** Charli E Harlow, Josan Gandawijaya, Rosemary A Bamford, Emily-Rose Martin, Andrew R Wood, Peter J van der Most, Toshiko Tanaka, Hampton L Leonard, Amy S Etheridge, Federico Innocenti, Robin N Beaumont, Jessica Tyrrell, Mike A Nalls, Eleanor M Simonsick, Pranav S Garimella, Eric J Shiroma, Niek Verweij, Peter van der Meer, Ron T Gansevoort, Harold Snieder, Paul J Gallins, Dereje D Jima, Fred Wright, Yi-hui Zhou, Luigi Ferrucci, Stefania Bandinelli, Dena G Hernandez, Pim van der Harst, Vickas V Patel, Dawn M Waterworth, Audrey Y Chu, Asami Oguro-Ando, Timothy M Frayling
**Journal:** American Journal of Human Genetics
**Date:** 2022 Sep 1
**DOI:** [10.1016/j.ajhg.2022.08.004](https://doi.org/10.1016/j.ajhg.2022.08.004)
**PMID:** 36055212
**PMCID:** PMC9502050
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9502050/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9502050/pdf/main.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9502050/pdf/main.pdf)

## Abstract

Hypoxia-inducible factor prolyl hydroxylase inhibitors (HIF-PHIs) are currently under clinical development for treating anemia in chronic kidney disease (CKD), but it is important to monitor their cardiovascular safety. Genetic variants can be used as predictors to help inform the potential risk of adverse effects associated with drug treatments. We therefore aimed to use human genetics to help assess the risk of adverse cardiovascular events associated with therapeutically altered EPO levels to help inform clinical trials studying the safety of HIF-PHIs. By performing a genome-wide association meta-analysis of EPO (n = 6,127), we identified a cis-EPO variant (rs1617640) lying in the EPO promoter region. We validated this variant as most likely causal in controlling EPO levels by using genetic and functional approaches, including single-base gene editing. Using this variant as a partial predictor for therapeutic modulation of EPO and large genome-wide association data in Mendelian randomization tests, we found no evidence (at p < 0.05) that genetically predicted long-term rises in endogenous EPO, equivalent to a 2.2-unit increase, increased risk of coronary artery disease (CAD, OR [95% CI] = 1.01 [0.93, 1.07]), myocardial infarction (MI, OR [95% CI] = 0.99 [0.87, 1.15]), or stroke (OR [95% CI] = 0.97 [0.87, 1.07]). We could exclude increased odds of 1.15 for cardiovascular disease for a 2.2-unit EPO increase. A combination of genetic and functional studies provides a powerful approach to investigate the potential therapeutic profile of EPO-increasing therapies for treating anemia in CKD.

Keywords: anaemia in chronic kidney disease, mendelian randomization, drug-target mendelian randomization, gene editing, functional genomics, CRISPR-Cas9, functional validation, statistical genetics, population studies, genome-wide association study, cardiovascular disease risk

**Keywords:** anaemia in chronic kidney disease, mendelian randomization, drug-target mendelian randomization, gene editing, functional genomics, CRISPR-Cas9, functional validation, statistical genetics, population studies, genome-wide association study, cardiovascular disease risk

## Introduction

Anemia, one of the primary complications of chronic kidney disease (CKD), affects one out of every seven individuals with CKD.[1](#bib1)^, ^[2](#bib2)^, ^[3](#bib3) Anemia is associated with faster progression of CKD and increased risk of adverse events, particularly heart disease or stroke, two of the major causes of death associated with CKD.[4](#bib4)^,^[5](#bib5) Current therapies used to treat anemia in CKD include blood transfusions, intravenous iron therapies, or parenteral injections of recombinant erythropoietin (rhEPO). These treatments have safety and compliance concerns, including risk of infection, adverse gastro-intestinal effects, and increased risk of stroke, myocardial infarction (MI), venous thromboembolism, and heart failure.[4](#bib4)^,^[6](#bib6)^, ^[7](#bib7)^, ^[8](#bib8)^, ^[9](#bib9)^, ^[10](#bib10)^, ^[11](#bib11)^, ^[12](#bib12) These safety concerns have led to the development of hypoxia-inducible factor (HIF) prolyl hydroxylase inhibitors (PHIs) as a novel class of treatment for anemia in CKD. PHIs inhibit the prolyl hydroxylase enzymes (PHD1-3) allowing the level of HIF-1 to rise and bind to the hypoxia response element that, among other activities, increases the transcription of *EPO* and in turn endogenous EPO levels.[13](#bib13)^,^[14](#bib14) EPO stimulates the bone marrow to increase the production of red blood cells and circulating hemoglobin (Hgb) levels.[12](#bib12) The recent completion of phase III clinical trials assessing cardiovascular safety has indicated non-inferiority of PHIs compared to rhEPO and shown that PHIs can increase and maintain Hgb levels with small increases in circulating EPO levels compared to exogenous EPO administration.[4](#bib4)^,^[15](#bib15)^, ^[16](#bib16)^, ^[17](#bib17)^, ^[18](#bib18)^, ^[19](#bib19)^, ^[20](#bib20)^, ^[21](#bib21)^, ^[22](#bib22)^, ^[23](#bib23) PHIs have already received approval for clinical use in Japan, supporting ongoing development elsewhere.[11](#bib11)^,^[15](#bib15)^,^[18](#bib18)^,^[24](#bib24)^, ^[25](#bib25)^, ^[26](#bib26)

To further assess the potential side-effects of EPO-based treatments, we used a genetic approach. Several studies have shown that genetic data can provide supporting evidence of an association between the drug target and intended therapeutic indication. Genetic associations can also help identify potential unintended effects and inform potential drug safety profiles.[27](#bib27)^, ^[28](#bib28)^, ^[29](#bib29)^, ^[30](#bib30)^, ^[31](#bib31)^, ^[32](#bib32)^, ^[33](#bib33) Genetic variants lying within or nearby the gene encoding the drug target are more likely to have functional impact on the protein product than those further away in the genome.[34](#bib34) These variants can be used in Mendelian randomization (MR) tests as partial proxies for pharmacological action to help inform the effects of long-term modulation of drug targets on disease outcomes.[35](#bib35)^, ^[36](#bib36)^, ^[37](#bib37) The properties of inherited genetic variation mean the associations are far less likely to be confounded or biased, compared to observational studies. Functional studies can provide further evidence that a genetic variant is a valid proxy for a drug target, for example, by confirming that the variant controls the expression of the expected gene. The development of CRISPR-Cas9 gene editing has made functional validation more straightforward by establishing whole-gene knockouts and modifying single-nucleotide polymorphisms (SNPs).[38](#bib38)^, ^[39](#bib39)^, ^[40](#bib40)

This study aimed to use human genetics to examine the long-term effect of genetically predicted therapeutic modulation of endogenous EPO levels. First, we identified a genetic variant lying *in cis* with *EPO* that is associated with endogenous circulating EPO levels. Second, we functionally validated that variant by using CRISPR-Cas9 gene-editing techniques. Third, we used this variant in drug-target MR tests as a genetic predictor for therapeutically altered EPO levels to help inform the long-term effects of elevated endogenous EPO levels on risk of cardiovascular disease (CVD) or clinical markers (blood pressure and resting heart rate) predisposing to CVD risk factors (e.g., hypertension).

## Material and methods

The steps performed to identify and functionally validate a genetic variant associated with circulating EPO are outlined in [Figure S1](#mmc1) and, briefly, are as follows.

### Identification of genetic variants associated with circulating EPO levels

To identify genetic variants associated with circulating EPO levels, we performed a genome-wide association study (GWAS) meta-analysis of 6,127 individuals of European and African descent from four independent cohorts: InCHIANTI (n = 1,210), PREVEND (n = 2,954), BLSA (n = 458), and HealthABC (n = 1,505) ([Table S1](#mmc1)). Details of phenotype generation and inclusion criteria are described in the [supplemental material and methods](#mmc1), and an analysis plan is outlined in [Figure S2](#mmc1). Each study was approved by an institutional review board as described in the [supplemental material and methods](#mmc1). We inverse normalized EPO levels to account for a skewed distribution and performed GWAS in GEMMA[41](#bib41) by using an additive linear mixed model adjusting for age, sex, and any study-specific covariates alongside a genomic relationship matrix to account for all types of relatedness. We combined study-specific estimates and performed inverse variance-weighted fixed-effects meta-analysis on ∼25.1 million imputed SNPs in 6,127 unrelated individuals of European and African descent by using METAL[42](#bib42) with the following filters: minor allele count (MAC) > 3, effect allele frequency (EAF) > 1, EAF < 0, Info ≥ 0.3. After performing meta-analysis, we excluded SNPs with a minor allele frequency (MAF) < 0.01 and performed a multi-SNP-based step-wise conditional and joint association analysis by using GCTA-COJO[43](#bib43)^,^[44](#bib44) to select SNPs independently associated with EPO (p < 5 × 10^−8^).

### Identification and validation of cis-EPO genetic variant

To identify a genetic variant most likely to directly impact on the protein product for use as a genetic predictor for therapeutically altered endogenous EPO levels, we analyzed the GWAS data around *EPO* specifically to identify any *cis-*acting genetic variants and selected variants previously identified. We converted the genetic effect estimate (in SDs) of the genetic variant to original units (IU/L) by using the standard deviation from the InCHIANTI study ([Table S2](#mmc1)).

### Expression quantitative trait loci (eQTLs) analysis

Having identified a *cis-EPO* variant (rs1617640) associated (p = 9.32 × 10^−4^) with circulating EPO protein levels, we tested its *cis*-effects (+/− 500 kb) with gene expression in a meta-analysis of hepatic gene expression from 861 livers from European individuals in three datasets[45](#bib45) ([Table S3](#mmc1)) and 236 kidneys from 134 individuals in one renal gene expression dataset[46](#bib46) ([Table S4](#mmc1)). Additional study details can be found in [supplemental material and methods](#mmc1). We selected the liver and kidney, as *EPO* is highly expressed in both tissues.

### Colocalization analysis

We performed colocalization analysis to assess the likelihood that the liver *EPO* eQTL was the same signal as the circulating protein level association. We obtained summary data for hepatic *cis*-eQTLs associated with *EPO* expression (false discovery rate [FDR] < 0.1) 500 kb on either side of rs1617640 and extracted the summary statistics for these SNPs from our circulating EPO meta-analysis. We performed approximate Bayes factor colocalization analyses by using the R coloc package.[47](#bib47)^,^[48](#bib48) We obtained overall estimates of the posterior probability that both our EPO meta-analysis and the liver eQTL share the same causal variant.

### Establishment of whole-EPO knockout

With the use of CRISPR-Cas9 gene-editing technology, human embryonic kidney (HEK)-293 cells were subjected to gene editing to generate isogenic *EPO* knockout (*EPO*^−/−^) cell lines. In brief, paired guide-RNAs (gRNAs), one targeting exon 2 (Ensembl: ENST00000252723.3) (5′-AGAGGTACCTCTCCAGGACT**CGG**-3′) and one targeting exon 4 (5′-CATGTGGATAAAGCCGTCAG**TGG**-3′), were separately cloned into the CRISPR-Cas9 expression vector containing a green fluorescent protein (GFP) reporter (pSpCas9(BB)-2A-GFP (PX458), Addgene: #48138) and an mCherry fluorescent protein reporter (pU6-(BbsI) CBh-Cas9-T2A-mCherry, Addgene: #64324), respectively. 1 × 10^6^ HEK-293 cells were seeded in 10 cm plates and 24 h later were co-transfected with 6 μg of each CRISPR-Cas9/gRNA vector using Lipofectamine LTX reagent (Thermo Fisher Scientific, Massachusetts, USA). Successfully transfected GFP and red fluorescent protein-positive cells were manually isolated under the EVOS FLoid Imaging system (Thermo Fisher Scientific, Massachusetts, USA). Single cells were clonally expanded for around 2 weeks and media containing 10% FBS was replaced every 48 h. Single cells were screened via polymerase chain reaction (PCR) assay using primers on either side of the two gRNAs (forward: 5′-TCTAGAATGTCCTGCCTGGC-3′, reverse: 5′-GGCCCTGTGACATCCTTAGA-3′). Sanger sequencing was used to confirm successful disruption to *EPO*. All wild-type (WT) HEK-293 cells were treated the same throughout the experiments but treated with empty CRISPR-Cas9 plasmids (i.e., containing no gRNAs).

### EPO overexpression

For use as a positive control*,* wild-type HEK-293 cells were transfected with an *EPO* overexpression (hEPO) construct (Addgene: #139057)[49](#bib49) using Lipofectamine LTX reagent (Thermo Fisher Scientific, Massachusetts, USA).

### RNA extraction and qRT-PCR

Total RNA was isolated and purified from *EPO*^−/−^ and WT HEK-293 cell lines using the Direct-zol RNA Miniprep kit following manufacturers protocol (Cambridge Biosciences, Cambridge, UK). 500 ng of RNA was converted to cDNA using PrimeScript RT reagent kit (Takara Bio Europe SAS, Saint-Germain-en-Laye, France). Quantitative reverse transcription (qRT)-PCR was performed using Hot FIREPol EvaGreen qPCR Master Mix with ROX (Solis BioDyne, Teaduspargi, Estonia) with the QuantStudio 6 Flex qPCR machine (Thermo Fisher Scientific, Massachusetts, USA) on at least three biological replicates. Primer sequences are listed in [Table S5](#mmc1). Samples with Ct values > 2 SD from the mean were removed. Gene expression levels were standardized against the reference gene *GAPDH* messenger RNA (mRNA) levels using the 2^−ΔΔCt^ method.[50](#bib50) Expression of alternative housekeeping genes (*UBC* and *Pol2ra*) was also assessed. RefFinder[51](#bib51) was used to determine the most stable gene or combination of genes for use as an endogenous control ([Table S6](#mmc1)). Differences in gene expression levels between WT and *EPO*^−/−^ cell lines were investigated for statistical significance by a paired t test carried out in RStudio version 3.6.1.[52](#bib52)

### RNA sequencing (RNA-seq)

After RNA extraction, RNA concentration and quality were assessed using the Qubit 4 Fluorometer (Thermo Fisher Scientific, Massachusetts, USA) and the Agilent 2200 TapeStation System, respectively (Agilent Technologies, California, USA). 1,000 ng of RNA from samples with an RNA integrity (RIN) score > 8 were prepared for RNA-seq.[53](#bib53) Library preparation and sequencing were performed by the Exeter Sequencing Service, resulting in 75 bp paired-end sequencing (additional details in [supplemental material and methods](#mmc1)). Quality control checks were undertaken on the raw reads using MultiQC.[54](#bib54) Adapter sequences (defined by Illumina), nucleotides with poor quality from the 3′ end (Phred < 25), and reads shorter than 25 bp were removed using CutAdapt version 1.13[55](#bib55) ([Figure S3](#mmc1)). Reads were aligned to the *Homo sapiens* GRCh38/hg38 reference genome using STAR version 2.7.1[56](#bib56) ([Figure S4](#mmc1)) and gene quantification was performed using the *featureCounts* subread package[57](#bib57) based on Ensembl GRCh38/hg38 annotation release version 2.0.0 ([Figure S5](#mmc1)). All other analysis was performed in RStudio unless otherwise stated.[52](#bib52) Transcripts whose mean count across all samples were <10 were removed. Counts were normalized and transformed using the “*rlogTransformation*” function implemented in *DeSeq2*.^58^ Principal-component analysis (PCA) was performed using the R “*prcomp*” function to check for similarity between samples. To identify genotype-specific gene expression changes, we performed differential gene expression analysis by using *DeSeq2*.[58](#bib58) p values were calculated using the Wald test, and a Benjamini-Hochberg correction was applied to account for multiple testing. Statistically significant differentially expressed genes (DEGs) were determined by an adjusted p value ≤ 0.05. We determined strong differential expression when genes were regulated by 2-fold. As we performed RNA-seq on two *EPO*^−/−^ cell lines (KOA and KOB) to obtain the most accurate list of DEGs most likely to be differentially expressed due to the effect of *EPO*^−/−^, differential expression analysis was performed comparing WT to each knockout cell line (WT versus KOA and WT versus KOB). DEGs (p ≤ 0.05) overlapping in both comparisons were identified using the R “*venn.diagram*” function. Gene set enrichment analysis was performed on this list of overlapping DEGs using Enrichr.[59](#bib59)^, ^[60](#bib60)^, ^[61](#bib61) Top DEGs were subsequently subjected to qRT-PCR to validate differential expression.

### Immunoblot analysis

*EPO*^−/−^, WT controls, and HEK-293 cells transfected with hEPO overexpression vector[49](#bib49) were subjected to immunoblot analysis, as described previously,[62](#bib62) using a monoclonal mouse anti-EPO antibody (1:1,000; MAB2871, R&D systems, Abingdon, UK)[63](#bib63) and a mouse anti-GAPDH antibody (1:1,000; sc-47724, Santa Cruz Biotechnology, Texas, USA). Goat anti-mouse IgG (H + L) Cross-Absorbed Alexa Fluor 680 (1:5,000; A21057, Invitrogen, Massachusetts, USA), was used as the secondary antibody. Membranes were visualized on the LI-COR Odyssey CLx system (LI-COR Biotechnologies, Nebraska, USA). Images were converted to grayscale using Image Studio Lite version 5.2.5 (LI-COR Biotechnologies, Nebraska, USA).

### Knockin of the rs1617640 C allele via CRISPR-Cas9 gene editing with the piggyBac transposon system

Using CRISPR-Cas9 and the *piggyBac* system, HEK-293 cells were subjected to single-base gene editing to generate isogenic cell-lines with either the A/A or A/C genotype for rs16167640 (the A allele being associated with higher circulating EPO levels) using a previously described protocol.[64](#bib64) In brief, a single gRNA (5′-GGAATCTCACTCCTCTGGCTCAGGG-3′) was cloned into the GFP CRISPR-Cas9 expression vector. In addition, 5′ and 3′ homology arms were designed ([supplemental material and methods](#mmc1)), one containing the desired base edit, and cloned into the *piggyBac* multivector (SGK:005, MV-PGK-Puro-TK, Hera BioLabs, Kentucky, USA) either side of the *piggyBac* transposon (containing puromycin/TK selection cassettes) using the BsiW1 and Nsi1 cut-sites ([supplemental material and methods](#mmc1)). Co-transfection was performed as described above. After 48 h, cells were cultured in a 10 cm dish under puromycin (1 μg/mL) (Sigma-Aldrich, Missouri, USA)[65](#bib65)^,^[66](#bib66) for 14 days, replacing media every 2–3 days, to select for cells containing the *piggyBac* transposon. Following selection, single puromycin-resistant cells were isolated and clonally expanded. Cells were screened via PCR and Sanger sequencing to confirm successful insertion of the *piggyBac* transposon and successful editing of rs16167640 from the parental A/A genotype to C/C genotype (primer sequences listed in [Table S7](#mmc1)). Isogenic HEK-293 cells containing the *piggyBac* transposon were subsequently transfected with 2.5 μg of *piggyBac* transposase in a six-well plate using lipofectamine LTX (Thermo Fisher Scientific, Massachusetts, USA) to remove the *piggyBac* selection cassette from the genome. 48 h after transfection, cells were subject to 200 nM Fialuridine (FIAU) (Sigma-Aldrich, Missouri, USA) selection for 7 days, replacing media every 2–3 days, to select for cells no longer carrying the *piggyBac* transposon cassette. Following selection, single FIAU-resistant cells were isolated and clonally expanded. Cells were screened via PCR ([Table S7](#mmc1)) and Sanger sequencing to confirm complete removal of the *piggyBac* transposon and successful editing of rs1617640. Successfully edited isogenic HEK-293 cells were subjected to qRT-PCR of the *EPO* gene to measure *EPO* gene expression levels and investigate the effect of the *cis*-*EPO* SNP on dysregulated Notch signaling genes identified through transcriptomic profiling of the *EPO*^−/−^ (*HEY2*, *DTX3L*, *PARP9*).

### Using the cis-EPO variant to examine the therapeutic profile and cardiovascular risk of genetically proxied therapeutic modulation of endogenous EPO levels

#### Drug-target two-sample Mendelian randomization

To investigate the association between higher endogenous EPO levels and risk of CVD, we performed a two-sample MR by using the *cis*-*EPO* variant as the genetic instrument. We obtained genotype-exposure association statistics from our EPO meta-analysis (n = 6,127). Primary outcomes were CAD, MI, or stroke using GWAS data consisting of 60,801, 42,561, and 40,585 cases, respectively[64](#bib64)^,^[67](#bib67) ([Table S8](#mmc1)). We also performed a GWAS using UKB on CAD, MI, or stroke in 37,741, 105,90, and 9,092 cases, respectively ([supplemental materials and methods](#mmc1), [Table S8](#mmc1)). Where we had genotype-outcome association data from both UKB and publicly available GWASs, we performed an inverse variance-weighted, fixed-effects meta-analysis by using metan[68](#bib68) to estimate the overall genotype-outcome association effect estimate. As only one genetic variant was used as an instrument, we calculated an overall causal estimate between the exposure and outcome by using the Wald ratio.[69](#bib69) We also performed MR to investigate the effect of higher circulating EPO levels on levels of clinical markers (systolic blood pressure [SBP], diastolic blood pressure [DBP], and resting heart rate) predisposing to CVD risk factors[70](#bib70)^,^[71](#bib71) ([supplemental materials and methods](#mmc1), [Table S8](#mmc1)).

#### Comparing clinical trial effects and genetic association to estimate the genetically predicted impact of therapeutically altered endogenous EPO levels on cardiovascular risk

To scale the genetic effect estimates to a more representative, therapeutically relevant effect, we obtained the effects of a PHI in patients on dialysis from a phase II fixed dose randomized control trial (RCT).[72](#bib72) The RCT provided an estimate of the effect of a fixed dose of HIF-PHIs on EPO levels during the first 4 weeks (median “maximum” change in EPO levels from baseline at week 4 [27.1]/SD at baseline [61] = 0.44 SD).[72](#bib72) The scaling factor was calculated by dividing the PHI-induced effect (0.44 SD) by the effect of the *cis-EPO* SNP on EPO levels (0.063 SD). We used this value (7.05) to scale the genetically instrumented effect estimates and 95% confidence intervals of the *cis-EPO* SNP on CVD or clinical markers for CVD risk factors to the effect of PHIs on endogenous EPO levels.

### Phenome-wide association study (PheWAS)

To further investigate the therapeutic profile of modulated EPO levels, we tested the association of rs1617640 near *EPO* with 869 traits in up to 451,099 UKB individuals of European ancestry ([supplemental materials and methods](#mmc1)). Genotype-phenotype associations were generated using BOLT-LMM[73](#bib73) and traits were selected as previously described in Frayling et al.[74](#bib74) For continuous traits, we performed inverse normalization prior to regression analysis to account for skewed distributions.

## Results

### Identification of three genomic loci associated with EPO levels at genome-wide significance

To identify human genetic variants associated with circulating EPO levels and using these variants as genetic predictors of therapeutically elevated endogenous EPO levels, we performed a GWAS meta-analysis of circulating EPO. We used 6,127 individuals of European and African American descent. After conditional analysis, we identified three genomic loci containing three independent signals associated with circulating EPO (p < 5 × 10^−8^) ([Table S1](#mmc1), [Figure S6](#mmc1)). The most strongly associated genetic variant, rs4895441 (6q23, *HBS1L-MYB* locus), had been previously identified as associated with circulating EPO levels in a GWAS of 2,691 individuals.[75](#bib75) This variant has stronger primary effects on other erythrocyte phenotypes in previously published GWASs and a PheWAS in UK Biobank (UKB) European individuals ([Table S9](#mmc1)). The remaining two independent genomic loci (rs855791 and rs112631630) represent novel associations with circulating EPO levels. However, rs855791, located in the *TMPRSS6* locus, has primary, stronger effects on several other erythrocyte phenotypes compared to the effect on circulating EPO in a PheWAS in UKB European individuals ([Table S9](#mmc1)) and has been previously associated with other erythrocyte phenotypes and iron homeostasis biomarkers in GWASs.[76](#bib76)^,^[77](#bib77) The variant (rs112631630) located in the *NRAP* locus is only present in one study (African Americans), and we were unable to test in additional datasets. Therefore, these variants were not deemed suitable, specific instruments used in subsequent MR analysis to genetically predict therapeutic modulation of endogenous EPO levels.

### Identification of cis-SNP lying in the promoter region of EPO for use as a genetic predictor for the therapeutic alteration of endogenous EPO levels

The conditionally independent lead variants identified by the GWAS meta-analysis were not sufficiently specific instruments to act as genetic predictors for higher endogenous EPO levels. We therefore looked for *cis* effects at the *EPO* locus. Previous associations have been reported between variants near *EPO* and circulating EPO levels.[78](#bib78)^,^[79](#bib79) A *cis-*SNP, rs1617640, lying in the *EPO* promoter region, 1,125 bp upstream of the transcription start site, was associated with EPO levels in our study and in the previous study.[78](#bib78) On the basis of our meta-analysis, each copy of the A-allele at rs1617640 was associated with 0.063 standard deviations (SD), equivalent to 0.32 IU/L, higher endogenous EPO levels *(*p *=* 9.32 × 10^−4^) ([Table 1](#tbl1)). The effect of the *cis-EPO* variant is consistent with that previously reported in affected individuals with diabetic retinopathy or hepatitis C.[78](#bib78)^,^[79](#bib79)

### Table 1.

| Analysis | RSID | Chromosome | Position | A1 | A2 | Freq A1 | β | SE | p value | Sample size |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| EPO meta-analysis | rs1617640 | 7 | 100,317,298 | A | C | 0.62 | 0.063 | 0.02 | 9.32E−4 | 6,127 |

Table 1 Caption: Summary statistics for association between the cis-EPO genetic variant (rs1617640) and circulating EPO levels or hepatic EPO gene expression

### The cis-EPO SNP is associated with altered EPO expression and nearby TFR2 expression in the liver

To provide additional insight into the rs1617640-EPO association and further evaluate its utility as an instrument, we tested the association of the *cis-EPO* SNP with gene expression in the kidney and the liver as *EPO* is highly expressed in these two tissues.[80](#bib80) In the liver, we found that the C allele at rs1617640 was associated with higher *EPO* expression (β = 0.22 [0.14, 0.3], p = 6.86 × 10^−8^) and also *TFR2* expression (β = 0.23 [0.17, 0.29], p = 1.56 × 10^−13^), a gene that lies upstream of the *EPO* gene and is involved in iron metabolism ([Table S3](#mmc1)).[76](#bib76)^,^[81](#bib81) No effect of rs1617640 on renal expression of *EPO* (β = 0.16 [−2.46, 2.78], p > 0.05) or *TFR2* (β = 1.33 [−0.59, 3.26], p > 0.05) was found ([Table S4](#mmc1)). We, therefore, proceeded with hepatic results only. Colocalization analysis provided evidence that the variant associated with circulating EPO levels in the meta-analysis and hepatic *EPO* mRNA expression has a 71% posterior probability of being the casual variant ([Figure 1](#fig1)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b36d/9502050/6a98a9b04c8b/gr1.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9502050_gr1.jpg)

Colocalization analysis of circulating endogenous EPO levels and liver EPO gene expression shows that the same genetic variant (cis-EPO variant) is likely to be the causal variant (posterior probability = 71%) We tested for colocalization within a 500 kb flanking window of the cis-EPO genetic variant (rs1617640—labeled and represented as a black diamond on the plot). The top half of the Miami plot (red dots) represents −log10(p) for the circulating EPO levels obtained from the EPO meta-analysis whilst the bottom half of the Miami plot (blue dots) represents log10(p) for EPO gene expression in the liver obtained from a hepatic eQTL meta-analysis. The dashed lines represent genome-wide significance levels of p = 5 × 10−8.

### Functional validation of rs1617640 in influencing EPO expression

To further validate the rs1617640 variant, we first sought a better understanding of the downstream causal genes and signaling cascades of EPO. We generated two *EPO* knockout (*EPO*^−/−^*)* cell lines by using CRISPR-Cas9 gene editing with a paired gRNA approach ([Figures 2](#fig2)A and 2B, [Figure S7](#mmc1)). By performing RNA sequencing (RNA-seq) analysis, we found large transcriptional differences between *EPO*^−/−^ and wild-type (WT) cell lines ([Figures 2](#fig2)C and [S8](#mmc1)). To obtain a list of differentially expressed genes (DEGs) specific to *EPO* knockdown and not due to potential differences in cellular conditions, we performed differential gene expression analysis by comparing controls to the two *EPO*^−/−^ cell lines, KOA and KOB, ([Figures S9](#mmc1)A and S9B) and then combined lists of DEGs to obtain a final list of 3,722 overlapping DEGs (p ≤ 0.05) ([Figures 2](#fig2)D and 2E, [Table S10](#mmc2)). 3,501 of the overlapping DEGs showed consistent directions of differential expression ([Figure S9](#mmc1)C, r^2^ = 0.90, p < 2.2 × 10^−16^) and were considered for downstream analysis. 314 of the 3,501 had a log_2_ fold-change ≥ |2|. We validated the differential expression of four DEGs (p ≤ 0.05, log_2_ fold-change ≥ |2|) with the highest expression in HEK-293 cells (based on the Human Cell Atlas[82](#bib82)) by using qRT-PCR ([Figure 2](#fig2)F). Gene Ontology (GO) analysis of the 3,501 DEGs suggested enrichment for multiple biological processes involved in DNA repair, mRNA processing, ATPase activity, notch signaling, apoptosis, fatty acid oxidation, and cellular respiration ([Table S11](#mmc3)). Notch signaling and related mitogenic pathways featured prominently in these GO analyses ([Figure 2](#fig2)G), so we selected seven genes from these pathways and validated differential expression with qRT-PCR ([Figure 2](#fig2)H).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b36d/9502050/d06d22be4de6/gr2.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9502050_gr2.jpg)

Establishment of whole EPO gene knockout in HEK-293 cells using CRISPR-Cas9 gene-editing technology with paired gRNAs and whole transcriptomic profiling of knockouts to identify differentially expressed genes (A) Hypothetical schematic of the construction of the EPO−/−. Successful targeting of the paired gRNAs would result in the introduction of two double-stranded breaks (DSBs) and the removal of the intervening region from the genomic DNA (∼645 bp) after DNA repair via non-homologous end joining (NHEJ). The location of the paired gRNAs designed to target the EPO gene (ENST00000252723.3) are labeled and highlighted by the red dotted lines. P1 and P2 represent location of primers used for genotyping potential knockouts. (B) Immunoblotting analysis. Immunoblot analysis, probing for EPO and GAPDH as a control, revealed reduced EPO protein expression levels in both EPO−/− cell lines (KOA and KOB) compared to WT controls and cells transfected with an EPO over-expression construct (hEPO). (C) PCA plot illustrating the transcriptomic profiles of the EPO−/− cell lines compared to WT HEK-293 controls. The plot shows the results obtained from RNA-seq analysis of four WT samples, four KOA samples, and four KOB samples. (D) The number of differentially expressed genes (DEGs) (p ≤ 0.05) obtained by performing differential gene expression analysis comparing WT to KOA and WT to KOB. (E) Heatmap illustrating the expression profile of the 3,722 overlapping DEGs. The red blocks represent down-regulated genes, and the blue bocks represent up-regulated genes; the color scale of the heatmap represents the DEG expression level. (F) qRT-PCR of four of the top DEGs identified by differential gene expression analysis to validate our RNA-seq findings. The graph shows the relative change in mRNA expression levels (+/− SEM) between genotypes. (G) The top ten KEGG Pathways identified through GEO analysis of the 3,501 overlapping with consistent direction of effects. Analysis was performed with Enrichr online tool. The color scale represents the log10 p value. (H) qRT-PCR of genes involved in the Notch signaling pathway identified by differential gene expression analysis to determine whether EPO−/− results in altered Notch signaling activity. hEPO represents WT HEK-293 cells transfected with an over-expression EPO construct. The graph shows the relative change in mRNA expression levels (+/− SEM) between genotypes. Columns and error bars in (F) and (H) represent mean and SEM values. Paired t test was performed. ns, non-significant; ∗p ≤ 0.05, ∗∗p ≤ 0.01, ∗∗∗p ≤ 0.001, ∗∗∗∗p ≤ 0.0001.

To functionally validate the *cis*-*EPO* SNP as the most likely causal variant in controlling *EPO* expression levels and therefore the most valid instrument for use to genetically predict the associated risk of CVD, we used CRISPR-Cas9 and the *piggyBac* transposon system to generate two isogenic cell lines, one homozygous for the A allele (A/A) and one heterozygous for the A allele (A/C) at rs1617640 ([Figures 3](#fig3)A–3D, [S10](#mmc1), and [S11](#mmc1)). We performed qRT-PCR in the A/A and A/C isogenic cell lines. We showed that homozygous cells for the A allele of rs1617640 had higher *EPO* mRNA expression levels than heterozygotes for the A allele, confirming that rs1617640 has an allele-specific effect on *EPO* gene expression levels ([Figure 3](#fig3)E). These results are consistent with our genetic findings that the A allele is associated with higher circulating EPO levels. We also performed qRT-PCR on three Notch-signaling genes (*HEY2*, *DTX3L*, *PARP9*), which showed differential expression in the *EPO*^−/−^ knockouts compared to wild-type, to see whether specific alteration of the *cis-EPO* variant also resulted in dysregulated Notch signaling. We found that A allele homozygotes of rs1617640 had a down-regulated expression of these Notch-signaling genes compared to heterozygotes ([Figure 3](#fig3)F). Negative control experiments with two genes not differentially expressed in *EPO*^−/−^ confirmed that the *cis*-SNP editing was specific to the EPO pathways ([Figure 3](#fig3)F). As the eQTL analysis showed that the *cis-EPO* SNP was associated with hepatic *TFR2* expression, we also performed qRT-PCR to further investigate the potential pleiotropic effect of the *cis-*SNP. We found no difference in *TFR2* mRNA expression in A allele heterozygotes compared to A allele homozygotes or *EPO*^−/−^ knockouts ([Figure S12](#mmc1)).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b36d/9502050/bf5b697f13fb/gr3.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9502050_gr3.jpg)

Establishment of heterozygous knockin at rs1617640 (A/C) in HEK-293 cells using CRISPR-Cas9 gene-editing technology alongside piggyBac transposon system (A) Schematic of the generation of a SNP knockin model of rs1617640 using CRISPR-Cas9 gene editing alongside the piggyBac transposon system. The location of rs1617640 (A/A) and the site at which the gRNA is designed to introduce a DSB within the wild-type sequence of HEK-293 cells is shown in the top panel. Two homology arms either side of the closest TTAA site are designed with one (5′ arm) containing the desired gene edit at rs1617640 (C/C). The two homology arms are cloned into the targeting construct either side of the piggyBac transposon carrying the selectable markers, puroΔtk. Upon introduction of a DSB at the site targeted by the gRNA, DNA repair via homologous recombination due to the presence of the homologous arms is initiated and the the piggyBac transposon becomes integrated into the genomic DNA at the TTAA site. After selection with puromycin, clones with mutation-corrected lines were identified and transiently transfected with piggyBac transposase plasmid, followed by FIAU treatment to eliminate piggyBac-containing clones. Mutation-corrected heterozygous clones for rs1617640 (A/C genotype) were isolated with no marks remaining within the genomic DNA. (B) Sanger sequencing results of the sequence at each stage of the CRISPR-Cas9 and piggyBac transposon technique. The top panel represents the wild-type sequence at rs1617640 and the wild-type sequences up- and downstream of the TTAA site prior to gene editing. The middle panel represents the mutated sequence at rs1617640 and the insertion of the piggyBac transposon sequence at the TTAA site following homologous recombination after successful introduction of the DSB. The bottom panel represents the seamless excision of the piggyBac transposon from the genomic DNA. The yellow highlighted region represents the gRNA sequence. (C) PCR gel electrophoresis confirming the insertion of the piggyBac transposon in clones 61 and 67. (D) PCR gel electrophoresis confirming successful removal of the piggyBac transposon from the genome following treatment with the transposase in clone 7-3. (E) qRT-PCR of EPO to validate rs1617640 as causal in controlling EPO mRNA expression levels. The graph shows the relative change in mRNA expression levels (+/− SEM) between genotypes (A/A and A/C). (F) qRT-PCR of genes involved in the Notch signaling pathway (HEY2, DTX3L, and PARP9) and two control genes (PPIA and POLR2A). EPO was repeated again as a positive control for altered expression for comparison. The graph shows the relative change in mRNA expression levels (+/− SEM) between genotypes. Columns and error bars in (E) and (F) represent mean and SEM values. Paired t test was performed. ns, non-significant; ∗p ≤ 0.05, ∗∗p ≤ 0.01, ∗∗∗p ≤ 0.001, ∗∗∗∗p ≤ 0.0001.

### Genetically predicted long-term higher endogenous EPO levels are not associated with increased cardiovascular risk

We used the *cis-EPO* SNP as an instrument in two-sample Mendelian randomization (MR) as a genetic predictor for therapeutically altered EPO levels on risk of CVD or levels of clinical markers predisposing to CVD risk factors. The three main CVDs of interest were coronary artery disease (CAD; 98,542 cases, 442,396 controls), myocardial infarction (MI; 53,151 cases, 564,013 controls), and stroke (49,677 cases, 752,534 controls) because of availability of large-scale genetic association data. The clinical markers of interest were systolic blood pressure (SBP; n = 678,320), diastolic blood pressure (DBP; n = 677,567), and resting heart rate (n = 514,695). Using these very large sample sizes, we found no evidence (at p < 0.05) of a genetic association between 1 SD higher endogenous EPO levels (equivalent to 5.1 IU/L) and increased odds of CAD (odds ratio [OR] [95% CI] = 1.03 [0.85, 1.25], p = 0.72), stroke (OR [95% CI] = 0.92 [0.70, 1.21], p = 0.55), or MI (OR [95% CI] = 0.98 [0.75, 1.29], p = 0.89) ([Table S12](#mmc1)). We found evidence of a genetic association between 1 SD, equivalent to 5.1 IU/L, higher endogenous EPO levels, and lower resting heart rate (effect estimate [95% CI] = −0.996 [−1.74, −0.25], p = 0.01) and lower DBP (effect estimate [95% CI] = −0.98 [−1.67, −0.29], p = 0.006) but not with SBP (effect estimate [95% CI] = 0.53 [−0.65, 1.71], p = 0.38) ([Table S12](#mmc1)).

### Comparison of the genetic associations with the effects observed in clinical trials

In an RCT, individuals receiving a PHI (daprodustat) had EPO levels 0.44 SD (27.1/61), equivalent to 2.2 IU/L, higher than baseline EPO levels.[72](#bib72) Given that the per-allele effect of rs1617640 on endogenous EPO was 0.063 SD, we rescaled our genetic association by multiplying by 7.05 (0.44/0.063) to obtain a clinically relevant estimate of the likely impact of genetically predicted therapeutic rises in endogenous EPO on cardiovascular risk. Using this scaling factor allowed us to quantify the upper and lower bounds on the genetically predicted effects of long-term endogenous EPO rises on CVD equivalent to therapeutically, physiologically relevant effects ([Figure 4](#fig4), [Table S13](#mmc1)). By rescaling the genetic associations to the PHI-induced effects of genetically predicted therapeutic rises in endogenous EPO levels, the odds of disease were 1.01 (95% CI = 0.93, 1.07) for CAD, 0.99 (95% CI = 0.87, 1.15) for MI, and 0.97 (95% CI = 0.87, 1.07) for stroke ([Figure 4](#fig4), [Table S13](#mmc1)). On the basis of the upper 95% confidence intervals, we could exclude a 1.07, 1.15, and 1.07 increased odds of CAD, MI, or stroke, respectively ([Figure 4](#fig4), [Table S13](#mmc1)). For the clinical markers predisposing to CVD, we did not observe an association between higher genetically predicted therapeutic rises in endogenous EPO levels and SBP (β [95% CI] = 0.21 [−0.28, 0.78]), DBP (β [95% CI] = −0.42 [−0.71, −0.14]), or resting heart rate (β [95% CI] = −0.42 [−0.78, −0.14]). On the basis of the upper confidence intervals, we could exclude 0.78 mmHg increased SBP levels and any increase in DBP or resting heart rate ([Figure 4](#fig4), [Table S13](#mmc1)).

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/b36d/9502050/475c75ffd8fa/gr4.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9502050_gr4.jpg)

Genetically predicted therapeutic rises in endogenous EPO levels are not associated with an increased risk of CVD or clinical markers for CVD risk factors We rescaled the genetic estimates obtained through two-sample MR by using the cis-EPO SNP to genetically estimate the effect of therapeutic rises in endogenous EPO levels to the PHI-induced effect reported in a phase II RCT. The black point on the plot represents the odds ratio and the bars represent the 95% confidence intervals.For all other figures, elements have been defined in the corresponding figure legends. (A) On the basis of the upper confidence interval, we were able to exclude increased odds of 1.07, 1.15, and 1.07 for CAD, MI, or stroke, respectively with 2.2 IU/L genetically mediated higher endogenous EPO levels. (B) On the basis of the upper confidence interval, we were also able to exclude levels higher than 0.21 mmHg for SBP and no increase in DBP or resting heart rate with genetically mediated therapeutic higher endogenous EPO levels.

### The cis-EPO SNP is associated with several relevant erythrocyte phenotypes and no unintended effects or diseases

To further determine the specificity of the *cis-EPO* SNP as a genetic instrument for endogenous, physiological EPO levels and identify any potential additional, unintended effects that may be associated with long-term rises in genetically predicted endogenous EPO levels, we tested the association between the *cis*-*EPO* SNP and 869 traits in up to 451,099 unrelated European UKB individuals. We found the *cis*-*EPO* SNP was associated with 18 relevant erythrocyte traits (p < 5 × 10^−8^) with effects between 0.01 and 0.06 SD ([Figure S13](#mmc1), [Table S14](#mmc1)). We also found evidence for an association between the *EPO*-increasing A allele of rs1617640 and decreased fibrosis-4 score (β = −0.01, p = 4.7 × 10^−17^) and non-alcoholic fatty acid liver disease (NAFLD) fibrosis score (β = −0.02, p = 2.20 × 10^−25^) ([Figure S13](#mmc1), [Table S14](#mmc1)). However, these associations were not clinically significant (equivalent to a 0.06 and 0.07 change in fibrosis-4 or NAFLD for 1 IU/L increase in EPO levels). These associations are most likely driven by the strong association with higher platelet counts (β = 0.02, p = 4.7 × 10^−39^). We did not find evidence for an association between genetically predicted higher endogenous EPO levels and other unintended effects or diseases.

## Discussion

We have shown how a combination of genetic analyses and functional validation studies provides a powerful approach to assess the therapeutic profile and effects of long-term genetically mediated alterations in drug target levels. Several lines of evidence indicate the *cis-EPO* variant is an excellent proxy to naturally mimic the pharmaceutical effects of raising endogenous EPO levels and test effects on cardiovascular risk. First*,* we found the A allele of rs1617640 increases EPO levels in a meta-analysis of circulating EPO levels ([Table 1](#tbl1)). Second, we found the *cis-EPO* SNP is associated with hepatic *EPO* gene expression and, from colocalization analysis, is likely to represent the same causal variant ([Table 1](#tbl1), [Figure 1](#fig1)). Third, by establishing a heterozygous knockin cell model using CRISPR-Cas9 single-base gene editing, we provided evidence that the *cis-EPO* variant has an allele-specific effect on *EPO* expression levels with homozygotes of the A allele having higher *EPO* expression compared to heterozygotes for the A allele ([Figure 3](#fig3)). Our findings help clarify previous studies that found rs1617640 to have an allele-specific effect on *EPO* expression levels. Some had reported the A allele to be associated with higher EPO concentrations,[78](#bib78)^,^[79](#bib79) while others reported conflicting evidence with the C allele being associated with higher EPO concentrations and promoter activity.[83](#bib83)^,^[84](#bib84) We found an association between the variant and hepatic *EPO* gene expression; however, the C allele was found to be associated with increased hepatic *EPO* expression while the A allele was found to increase endogenous EPO levels ([Table 1](#tbl1)). We did not find an association between the variant and renal gene expression despite the CRISPR-Cas9 gene functional work in an embryonic kidney cell line. These differences could be due to lack of power attributable to the small sample size (n = 236 kidneys) compared to the liver dataset. The rs1617640 SNP has different effects depending upon cell type, physiological condition, state, and timing alongside the complex compensatory feedback mechanisms involved in EPO signaling.[85](#bib85) When testing the *in vitro* effects of *EPO* gene knockdown, we indicate a role for EPO in Notch signaling. The Notch-signaling pathway is known to play a role in cell-cycle signaling, cell-fate specification, and metabolic processes and therefore may represent the broad mitogenic effects of EPO.[86](#bib86)^,^[87](#bib87) We also found the *cis-EPO* SNP to alter gene expression levels of Notch-signaling genes that were identified through whole-transcriptomic profiling of *EPO*^−/−^ cell lines, indicating that rs1617640 is important in controlling *EPO* expression levels and in downstream implicated pathways involved in cell-cycle activity, DNA repair, and metabolic processes ([Figures 2](#fig2) and [3](#fig3)).

Having provided genetic and functional evidence that a *cis*-*EPO* variant alters *EPO* gene expression and circulating protein levels, we used this variant as a genetic predictor for long-term therapeutic modulation of EPO levels to show that genetically predicted higher endogenous EPO levels (equivalent to 5.1 IU/L) are not associated with increased cardiovascular risk or elevated values of clinical markers (SBP, DBP, or resting heart rate) predisposing to CVD risk factors (e.g., hypertension). To obtain a more representative effect of therapeutically altered endogenous EPO levels, we rescaled these genetic effects to the PHI-induced effects on endogenous EPO levels from a fixed dose phase II trial[72](#bib72) and did not observe an association between genetically predicted therapeutic rises in endogenous EPO levels and risk of CVD ([Figure 4](#fig4)). Using the upper bound of the confidence interval, we were able to statistically exclude increased odds of 1.07, 1.15, and 1.07 for CAD, MI, or stroke, respectively, and 0.78 mmHg increased levels of SBP ([Figure 4](#fig4)). We were able to exclude adverse effects on DBP and heart rate ([Figure 4](#fig4)). Our results are consistent with the hypothesis that PHIs are not likely to increase CVD risk or CVD risk factors in treating anemia of CKD when increasing circulating EPO levels within the physiological range. Scaling genetic estimates to the effect of a fixed dose after 4 weeks of treatment may not be the most clinically relevant because PHIs require titration to a hemoglobin target, but this was the best estimate available, as changes in EPO levels were only measured in the one fixed dose phase II trial.[72](#bib72) Because all PHIs work through the same mechanism, these genetically predicted effects of therapeutically altering endogenous EPO levels are likely applicable to all PHI compounds. Any slight differences in effects on EPO levels and CVD risk between HIF-PHI compounds would most likely be related to independent biochemical properties of the compounds, variations in dosages, and the effects of PHIs on transcription of other hypoxic response genes, which were not investigated in this study.

There are some limitations to our study. As with any study using human genetics as predictors, our results cannot rule out effects but instead can provide upper bounds on their probability.[88](#bib88) First, here we performed the genetic analyses in a general, “healthy” population as opposed to a diseased cohort in whom the treatment is used. Diseased populations may respond differently to that estimated by the genetic association as a result of having variable baseline EPO levels or additional underlying conditions.[89](#bib89)^,^[90](#bib90) Despite rescaling the genetic effect to the PHI-induced effect to try and overcome this, we still assume linearity, which may not be the case for the therapy. For example, MR estimates could change depending on baseline levels, and therefore inferences about the likely effect at the individual level need careful consideration, particularly when doses are titrated.[90](#bib90) As larger studies become available, particularly in disease cohorts, our ability to detect associations and perform stratified analyses will increase. We will become more confident about the conclusions drawn from these types of investigations.[90](#bib90)^,^[91](#bib91) Second, common genetic variants differ from clinical trials in that they represent subtle, life-long perturbations, whereas clinical trials test more acute larger changes.[92](#bib92)^,^[93](#bib93) In addition, therapeutic interventions may result in localized effects at a particular time, titration to a particular level resulting in different individual-level biomarker increases, or may only be efficacious in a certain physiological state, which is difficult to accurately predict using genetics.[89](#bib89)^,^[93](#bib93) Third, as with any MR investigation, pleiotropy could lead to biased estimates. As we only used a single genetic instrument, we were unable to fully test for horizontal pleiotropy using established statistical methods and it remains possible that our MR findings are confounded by the variant’s association with *TFR2*, as well as *EPO.* The presence of the variant in the *EPO* promoter, and the alteration of *EPO* expression, but not *TFR2* expression, in our single-base editing experiments suggests *EPO* is the main target. The single-base editing enabled separation of the primary effect of the *cis-*SNP on *EPO* from a likely secondary effect on *TFR2* ([Figure S12](#mmc1)) eliminating some concerns of biased estimates due to horizontal pleiotropy.

In summary, this study indicates that genetically predicted long-term rises in endogenous EPO levels do not increase cardiovascular risk, with upper limits of 1.07, 1.15, and 1.07 for CAD, MI, and stroke, respectively, given a clinically relevant 2.2 unit rise in endogenous EPO levels. These estimates were established using extremely large case-control studies. Our functional evidence using CRISPR-Cas9 and the *piggyBac* system to change the allele at rs1617640 validated the *cis-EPO* SNP as a partial proxy for therapeutically altered endogenous EPO. We have shown how genetic analyses combined with functional validation studies represent a powerful approach to identify relevant genetic markers that can investigate the long-term effect of therapeutic action.

### Data and code availability

The code generated during this study are available at [https://github.com/CharliHarlow/EPO_metaanalysis_rnaseq_MR_phewas](https://github.com/CharliHarlow/EPO_metaanalysis_rnaseq_MR_phewas). The accession number for the summary-level EPO meta-analysis data generated in this paper is Zenodo: 6811853 ([https://zenodo.org/record/6811853](https://zenodo.org/record/6811853)). The accession number for the raw sequencing read data generated in this paperr is Zenodo: 6811704 ([https://zenodo.org/record/6811704#.Ys1Jzy2ZPOQ](https://zenodo.org/record/6811704#.Ys1Jzy2ZPOQ)). The published article includes the rest of the data generated or analyzed during this study.

## Acknowledgments

The authors acknowledge all participants and investigators involved in the different studies. The HealthABC study was supported by National Institute on Aging (NIA) contracts (N01-AG-6-2101; N01-AG-6-2103; N01-AG-6-2106; NIA grant R01-AG0280502, and NINR grant R01-NR012459) and was funded in part by the Intramural Research Program of the NIA. The InCHIANTI study baseline (1998–2000) was supported as a “targeted project” (ICS110.1/RF97.71) by the Italian Ministry of Health and in part by the NIA (contracts: 263 MD 9164 and 263 MD 821336). The research utilized data from the UK Biobank (UKB) resource carried out under application number 9072. UKB protocols were approved by the National Research Ethics Service Committee. The authors thank the Exeter Sequencing service in carrying out the RNA-sequencing. The equipment utilized is funded by the Wellcome Trust Institutional Strategic Support Fund (WT097835MF), Wellcome Trust Multi User Equipment Award (WT101650MA), and BBSRC LOLA award (BB/K003240/1). The authors acknowledge the use of the University of Exeter High-Performance Computing (HPC) facility in carrying out this work funded by an MRC Clinical Research Infrastructure award (MRC grant: MR/M008924/1). T.M.F. is supported by MRC awards MR/WO14548/1 and MR/T002239/1. J.T is funded by an Academy of Medical Sciences grant. pLV-EF1alpha-EPO-218-PDGFR was a gift from Tao Liu. The authors thank Akshay Bhinge for providing the *piggyBac* plasmid and transposase plasmid, Dario Pacitti for his help with the RNA-seq analysis, and Natsuki Takamura and Yuriko Iizuka for help in the laboratory. For the purpose of open access, the author has applied a CC BY public copyright license to any Author Accepted Manuscript version arising from this submission.

### Declaration of interests

C.E.H. was awarded an MRC iCASE studentship (MR/P016065/1), which was cofunded by GSK and the MRC for the duration of the study. GSK are undertaking clinical development into a novel PHI and have given permission to publish this work alongside an internal review of the manuscript. A.Y.C. is an employee of GSK, is a shareholder of GSK stock, and was involved in the study design, interpretation of the data, and writing of the paper. V.V.P. was an employee of GSK at the time of the study conception and design. He was involved in interpreting the data and writing of the paper and is a shareholder of GSK and Roche Holding AG stocks. D.M.W. was an employee of GSK at the time of the study conception and design and was involved in the interpretation of the data and writing of the paper. H.L.L. and M.A.N. received support from a consulting contract between Data Tecnica International and the National Institute on Aging (NIA), National Institutes of Health (NIH). They were involved in data analysis and review of the manuscript. N.V. is a full-time employee of Regeneron Pharmaceutical Inc. and receives stock options and restricted stock units as compensation. D.D.J. received support from the Center of Human Health and the Environment P30ESO25128 grant. F.I. is an AbbVie employee and receives stocks from AbbVie, which has not been involved in this work at any level.

Published: September 1, 2022

## Footnotes

## Contributor Information

Asami Oguro-Ando, Email: a.oguro-ando@exeter.ac.uk.

Timothy M. Frayling, Email: t.m.frayling@exeter.ac.uk.

## Web resources

AddGene, [https://www.addgene.org](https://www.addgene.org)

Benchling, [https://www.benchling.com](https://www.benchling.com)

BioRender, [https://biorender.com](https://biorender.com)

BLAST, [https://www.ensembl.org/Multi/Tools/Blast](https://www.ensembl.org/Multi/Tools/Blast)

Ensembl, [https://www.ensembl.org/index.html](https://www.ensembl.org/index.html)

Enrichr, [https://maayanlab.cloud/Enrichr/](https://maayanlab.cloud/Enrichr/)

GEMMA, [https://www.xzlab.org/software/GEMMAmanual.pdf](https://www.xzlab.org/software/GEMMAmanual.pdf)

Genewiz, [https://www.genewiz.com/en-GB/](https://www.genewiz.com/en-GB/)

GTEx, [https://gtexportal.org/](https://gtexportal.org/)

Human Cell Atlas, [https://www.humancellatlas.org](https://www.humancellatlas.org)

METAL, [https://genome.sph.umich.edu/wiki/METAL_Quick_Start#Meta-Analysis_Results](https://genome.sph.umich.edu/wiki/METAL_Quick_Start#Meta-Analysis_Results)

RefFinder, [https://www.heartcure.com.au/reffinder/?type=reference](https://www.heartcure.com.au/reffinder/?type=reference)

RStudio, [https://www.rstudio.com](https://www.rstudio.com)

Stata, [https://www.stata.com](https://www.stata.com)

## Supplemental information

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The code generated during this study are available at [https://github.com/CharliHarlow/EPO_metaanalysis_rnaseq_MR_phewas](https://github.com/CharliHarlow/EPO_metaanalysis_rnaseq_MR_phewas). The accession number for the summary-level EPO meta-analysis data generated in this paper is Zenodo: 6811853 ([https://zenodo.org/record/6811853](https://zenodo.org/record/6811853)). The accession number for the raw sequencing read data generated in this paperr is Zenodo: 6811704 ([https://zenodo.org/record/6811704#.Ys1Jzy2ZPOQ](https://zenodo.org/record/6811704#.Ys1Jzy2ZPOQ)). The published article includes the rest of the data generated or analyzed during this study.

### Supplementary Materials

### Data Availability Statement

The code generated during this study are available at [https://github.com/CharliHarlow/EPO_metaanalysis_rnaseq_MR_phewas](https://github.com/CharliHarlow/EPO_metaanalysis_rnaseq_MR_phewas). The accession number for the summary-level EPO meta-analysis data generated in this paper is Zenodo: 6811853 ([https://zenodo.org/record/6811853](https://zenodo.org/record/6811853)). The accession number for the raw sequencing read data generated in this paperr is Zenodo: 6811704 ([https://zenodo.org/record/6811704#.Ys1Jzy2ZPOQ](https://zenodo.org/record/6811704#.Ys1Jzy2ZPOQ)). The published article includes the rest of the data generated or analyzed during this study.

## References

1. Stauffer M.E., Fan T. Prevalence of anemia in chronic kidney disease in the United States. PLoS One. 2014;9:e84943. doi: 10.1371/journal.pone.0084943.  [DOI](https://doi.org/10.1371/journal.pone.0084943) | [PMC free article](/articles/PMC3879360/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24392162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Prevalence%20of%20anemia%20in%20chronic%20kidney%20disease%20in%20the%20United%20States&author=M.E.%20Stauffer&author=T.%20Fan&volume=9&publication_year=2014&pages=e84943&pmid=24392162&doi=10.1371/journal.pone.0084943&)

2. Hill N.R., Fatoba S.T., Oke J.L., Hirst J.A., O'Callaghan C.A., Lasserson D.S., Hobbs F.D.R. Global Prevalence of Chronic Kidney Disease – A Systematic Review and Meta-Analysis. PLoS One. 2016;11:e0158765. doi: 10.1371/journal.pone.0158765.  [DOI](https://doi.org/10.1371/journal.pone.0158765) | [PMC free article](/articles/PMC4934905/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27383068/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Global%20Prevalence%20of%20Chronic%20Kidney%20Disease%20%E2%80%93%20A%20Systematic%20Review%20and%20Meta-Analysis&author=N.R.%20Hill&author=S.T.%20Fatoba&author=J.L.%20Oke&author=J.A.%20Hirst&author=C.A.%20O'Callaghan&volume=11&publication_year=2016&pages=e0158765&pmid=27383068&doi=10.1371/journal.pone.0158765&)

3. St Peter W.L., Guo H., Kabadi S., Gilbertson D.T., Peng Y., Pendergraft T., Li S. Prevalence, treatment patterns, and healthcare resource utilization in Medicare and commercially insured non-dialysis-dependent chronic kidney disease patients with and without anemia in the United States. BMC Nephrol. 2018;19:67. doi: 10.1186/s12882-018-0861-1.  [DOI](https://doi.org/10.1186/s12882-018-0861-1) | [PMC free article](/articles/PMC5856223/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29544446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Nephrol.&title=Prevalence,%20treatment%20patterns,%20and%20healthcare%20resource%20utilization%20in%20Medicare%20and%20commercially%20insured%20non-dialysis-dependent%20chronic%20kidney%20disease%20patients%20with%20and%20without%20anemia%20in%20the%20United%20States&author=W.L.%20St%20Peter&author=H.%20Guo&author=S.%20Kabadi&author=D.T.%20Gilbertson&author=Y.%20Peng&volume=19&publication_year=2018&pages=67&pmid=29544446&doi=10.1186/s12882-018-0861-1&)

4. Zheng Q., Wang Y., Yang H., Sun L., Fu X., Wei R., Liu Y.N., Liu W.J. Efficacy and Safety of Daprodustat for Anemia Therapy in Chronic Kidney Disease Patients: A Systematic Review and Meta-Analysis. Front. Pharmacol. 2020;11:573645. doi: 10.3389/fphar.2020.573645.  [DOI](https://doi.org/10.3389/fphar.2020.573645) | [PMC free article](/articles/PMC7883598/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33597868/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Efficacy%20and%20Safety%20of%20Daprodustat%20for%20Anemia%20Therapy%20in%20Chronic%20Kidney%20Disease%20Patients:%20A%20Systematic%20Review%20and%20Meta-Analysis&author=Q.%20Zheng&author=Y.%20Wang&author=H.%20Yang&author=L.%20Sun&author=X.%20Fu&volume=11&publication_year=2020&pages=573645&pmid=33597868&doi=10.3389/fphar.2020.573645&)

5. Cases A., Egocheaga M.I., Tranche S., Pallarés V., Ojeda R., Górriz J.L., Portolés J.M. Anemia of chronic kidney disease: Protocol of study, management and referral to Nephrology. Nefrologia. 2018;38:8–12. doi: 10.1016/j.nefro.2017.09.004.  [DOI](https://doi.org/10.1016/j.nefro.2017.09.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29128260/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nefrologia&title=Anemia%20of%20chronic%20kidney%20disease:%20Protocol%20of%20study,%20management%20and%20referral%20to%20Nephrology&author=A.%20Cases&author=M.I.%20Egocheaga&author=S.%20Tranche&author=V.%20Pallar%C3%A9s&author=R.%20Ojeda&volume=38&publication_year=2018&pages=8-12&pmid=29128260&doi=10.1016/j.nefro.2017.09.004&)

6. Fishbane S., Spinowitz B. Update on Anemia in ESRD and Earlier Stages of CKD: Core Curriculum 2018. Am. J. Kidney Dis. 2018;71:423–435. doi: 10.1053/j.ajkd.2017.09.026.  [DOI](https://doi.org/10.1053/j.ajkd.2017.09.026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29336855/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Kidney%20Dis.&title=Update%20on%20Anemia%20in%20ESRD%20and%20Earlier%20Stages%20of%20CKD:%20Core%20Curriculum%202018&author=S.%20Fishbane&author=B.%20Spinowitz&volume=71&publication_year=2018&pages=423-435&pmid=29336855&doi=10.1053/j.ajkd.2017.09.026&)

7. Jelkmann W. Physiology and pharmacology of erythropoietin. Transfus. Med. Hemother. 2013;40:302–309. doi: 10.1159/000356193.  [DOI](https://doi.org/10.1159/000356193) | [PMC free article](/articles/PMC3822280/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24273483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transfus.%20Med.%20Hemother.&title=Physiology%20and%20pharmacology%20of%20erythropoietin&author=W.%20Jelkmann&volume=40&publication_year=2013&pages=302-309&pmid=24273483&doi=10.1159/000356193&)

8. Pfeffer M.A., Burdmann E.A., Chen C.Y., Cooper M.E., de Zeeuw D., Eckardt K.U., Feyzi J.M., Ivanovich P., Kewalramani R., Levey A.S., et al. A Trial of Darbepoetin Alfa in Type 2 Diabetes and chronic kidney disease. N. Engl. J. Med. 2009;361:2019–2032. doi: 10.1056/NEJMoa0907845.  [DOI](https://doi.org/10.1056/NEJMoa0907845) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19880844/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=A%20Trial%20of%20Darbepoetin%20Alfa%20in%20Type%202%20Diabetes%20and%20chronic%20kidney%20disease&author=M.A.%20Pfeffer&author=E.A.%20Burdmann&author=C.Y.%20Chen&author=M.E.%20Cooper&author=D.%20de%20Zeeuw&volume=361&publication_year=2009&pages=2019-2032&pmid=19880844&doi=10.1056/NEJMoa0907845&)

9. Babitt J.L., Lin H.Y. Mechanisms of anemia in CKD. J. Am. Soc. Nephrol. 2012;23:1631–1634. doi: 10.1681/ASN.2011111078.  [DOI](https://doi.org/10.1681/ASN.2011111078) | [PMC free article](/articles/PMC3458456/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22935483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Am.%20Soc.%20Nephrol.&title=Mechanisms%20of%20anemia%20in%20CKD&author=J.L.%20Babitt&author=H.Y.%20Lin&volume=23&publication_year=2012&pages=1631-1634&pmid=22935483&doi=10.1681/ASN.2011111078&)

10. Bonomini M., Del Vecchio L., Sirolli V., Locatelli F. New treatment approaches for the anemia of CKD. Am. J. Kidney Dis. 2016;67:133–142. doi: 10.1053/j.ajkd.2015.06.030.  [DOI](https://doi.org/10.1053/j.ajkd.2015.06.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26372086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Kidney%20Dis.&title=New%20treatment%20approaches%20for%20the%20anemia%20of%20CKD&author=M.%20Bonomini&author=L.%20Del%20Vecchio&author=V.%20Sirolli&author=F.%20Locatelli&volume=67&publication_year=2016&pages=133-142&pmid=26372086&doi=10.1053/j.ajkd.2015.06.030&)

11. Parfrey P. Hypoxia-Inducible Factor Prolyl Hydroxylase Inhibitors for Anemia in CKD. N. Engl. J. Med. 2021;385:2390–2391. doi: 10.1056/NEJMe2117100.  [DOI](https://doi.org/10.1056/NEJMe2117100) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34739195/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=Hypoxia-Inducible%20Factor%20Prolyl%20Hydroxylase%20Inhibitors%20for%20Anemia%20in%20CKD&author=P.%20Parfrey&volume=385&publication_year=2021&pages=2390-2391&pmid=34739195&doi=10.1056/NEJMe2117100&)

12. Kaplan J.M., Sharma N., Dikdan S. Hypoxia-inducible factor and its role in the management of anemia in chronic kidney disease. Int. J. Mol. Sci. 2018;19:E389. doi: 10.3390/ijms19020389.  [DOI](https://doi.org/10.3390/ijms19020389) | [PMC free article](/articles/PMC5855611/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29382128/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Hypoxia-inducible%20factor%20and%20its%20role%20in%20the%20management%20of%20anemia%20in%20chronic%20kidney%20disease&author=J.M.%20Kaplan&author=N.%20Sharma&author=S.%20Dikdan&volume=19&publication_year=2018&pages=E389&pmid=29382128&doi=10.3390/ijms19020389&)

13. Haase V.H. Regulation of erythropoiesis by hypoxia-inducible factors. Blood Rev. 2013;27:41–53. doi: 10.1016/j.blre.2012.12.003.  [DOI](https://doi.org/10.1016/j.blre.2012.12.003) | [PMC free article](/articles/PMC3731139/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23291219/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood%20Rev.&title=Regulation%20of%20erythropoiesis%20by%20hypoxia-inducible%20factors&author=V.H.%20Haase&volume=27&publication_year=2013&pages=41-53&pmid=23291219&doi=10.1016/j.blre.2012.12.003&)

14. Sugahara M., Tanaka T., Nangaku M. Prolyl hydroxylase domain inhibitors as a novel therapeutic approach against anemia in chronic kidney disease. Kidney Int. 2017;92:306–312. doi: 10.1016/j.kint.2017.02.035.  [DOI](https://doi.org/10.1016/j.kint.2017.02.035) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28651951/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Kidney%20Int.&title=Prolyl%20hydroxylase%20domain%20inhibitors%20as%20a%20novel%20therapeutic%20approach%20against%20anemia%20in%20chronic%20kidney%20disease&author=M.%20Sugahara&author=T.%20Tanaka&author=M.%20Nangaku&volume=92&publication_year=2017&pages=306-312&pmid=28651951&doi=10.1016/j.kint.2017.02.035&)

15. Singh A.K., Carroll K., McMurray J.J.V., Solomon S., Jha V., Johansen K.L., Lopes R.D., Macdougall I.C., Obrador G.T., Waikar S.S., et al. Daprodustat for the treatment of anemia in patients not undergoing dialysis. N. Engl. J. Med. 2021;385:2313–2324. doi: 10.1056/NEJMoa2113380.  [DOI](https://doi.org/10.1056/NEJMoa2113380) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34739196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=Daprodustat%20for%20the%20treatment%20of%20anemia%20in%20patients%20not%20undergoing%20dialysis&author=A.K.%20Singh&author=K.%20Carroll&author=J.J.V.%20McMurray&author=S.%20Solomon&author=V.%20Jha&volume=385&publication_year=2021&pages=2313-2324&pmid=34739196&doi=10.1056/NEJMoa2113380&)

16. Singh A.K., Carroll K., Perkovic V., Solomon S., Jha V., Johansen K.L., Lopes R.D., Macdougall I.C., Obrador G.T., Waikar S.S., et al. Daprodustat for the treatment of anemia in pateints undergoing dialysis. N. Engl. J. Med. 2021;385:2325–2335. doi: 10.1056/NEJMoa2113379.  [DOI](https://doi.org/10.1056/NEJMoa2113379) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34739194/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=Daprodustat%20for%20the%20treatment%20of%20anemia%20in%20pateints%20undergoing%20dialysis&author=A.K.%20Singh&author=K.%20Carroll&author=V.%20Perkovic&author=S.%20Solomon&author=V.%20Jha&volume=385&publication_year=2021&pages=2325-2335&pmid=34739194&doi=10.1056/NEJMoa2113379&)

17. Eckardt K.-U., Agarwal R., Aswad A., Awad A., Block G.A., Bacci M.R., Farag Y.M.K., Fishbane S., Hubert H., Jardine A., et al. Safety and efficacy of vadadustat for anemia in patients undergoing dialysis. N. Engl. J. Med. 2021;384:1601–1612. doi: 10.1056/NEJMoa2025956.  [DOI](https://doi.org/10.1056/NEJMoa2025956) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33913638/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=Safety%20and%20efficacy%20of%20vadadustat%20for%20anemia%20in%20patients%20undergoing%20dialysis&author=K.-U.%20Eckardt&author=R.%20Agarwal&author=A.%20Aswad&author=A.%20Awad&author=G.A.%20Block&volume=384&publication_year=2021&pages=1601-1612&pmid=33913638&doi=10.1056/NEJMoa2025956&)

18. Chertow G.M., Pergola P.E., Farag Y.M.K., Agarwal R., Arnold S., Bako G., Block G.A., Burke S., Castillo F.P., Jardine A.G., et al. Vadadustat in patients with anemia and non–dialysis-dependent CKD. N. Engl. J. Med. 2021;384:1589–1600. doi: 10.1056/NEJMoa2035938.  [DOI](https://doi.org/10.1056/NEJMoa2035938) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33913637/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=Vadadustat%20in%20patients%20with%20anemia%20and%20non%E2%80%93dialysis-dependent%20CKD&author=G.M.%20Chertow&author=P.E.%20Pergola&author=Y.M.K.%20Farag&author=R.%20Agarwal&author=S.%20Arnold&volume=384&publication_year=2021&pages=1589-1600&pmid=33913637&doi=10.1056/NEJMoa2035938&)

19. Chen N., Hao C., Peng X., Lin H., Yin A., Hao L., Tao Y., Liang X., Liu Z., Xing C., et al. Roxadustat for anemia in patients with kidney disease not receiving dialysis. N. Engl. J. Med. 2019;381:1001–1010. doi: 10.1056/NEJMoa1813599.  [DOI](https://doi.org/10.1056/NEJMoa1813599) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31340089/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=Roxadustat%20for%20anemia%20in%20patients%20with%20kidney%20disease%20not%20receiving%20dialysis&author=N.%20Chen&author=C.%20Hao&author=X.%20Peng&author=H.%20Lin&author=A.%20Yin&volume=381&publication_year=2019&pages=1001-1010&pmid=31340089&doi=10.1056/NEJMoa1813599&)

20. Fishbane S., El-Shahawy M.A., Pecoits-Filho R., Van B.P., Houser M.T., Frison L., Little D.J., Guzman N.J., Pergola P.E. Roxadustat for treating anemia in patients with CKD not on dialysis: results from a randomized phase 3 study. J. Am. Soc. Nephrol. 2021;32:737–755. doi: 10.1681/ASN.2020081150.  [DOI](https://doi.org/10.1681/ASN.2020081150) | [PMC free article](/articles/PMC7920165/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33568383/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Am.%20Soc.%20Nephrol.&title=Roxadustat%20for%20treating%20anemia%20in%20patients%20with%20CKD%20not%20on%20dialysis:%20results%20from%20a%20randomized%20phase%203%20study&author=S.%20Fishbane&author=M.A.%20El-Shahawy&author=R.%20Pecoits-Filho&author=B.P.%20Van&author=M.T.%20Houser&volume=32&publication_year=2021&pages=737-755&pmid=33568383&doi=10.1681/ASN.2020081150&)

21. Akizawa T., Iwasaki M., Yamaguchi Y., Majikawa Y., Reusch M. Phase 3, Randomized, Double-Blind, Active-Comparator (Darbepoetin Alfa) Study of Oral Roxadustat in CKD Patients with Anemia on Hemodialysis in Japan. J. Am. Soc. Nephrol. 2020;31:1628–1639. doi: 10.1681/ASN.2019060623.  [DOI](https://doi.org/10.1681/ASN.2019060623) | [PMC free article](/articles/PMC7350993/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32493693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Am.%20Soc.%20Nephrol.&title=Phase%203,%20Randomized,%20Double-Blind,%20Active-Comparator%20(Darbepoetin%20Alfa)%20Study%20of%20Oral%20Roxadustat%20in%20CKD%20Patients%20with%20Anemia%20on%20Hemodialysis%20in%20Japan&author=T.%20Akizawa&author=M.%20Iwasaki&author=Y.%20Yamaguchi&author=Y.%20Majikawa&author=M.%20Reusch&volume=31&publication_year=2020&pages=1628-1639&pmid=32493693&doi=10.1681/ASN.2019060623&)

22. Akizawa T., Iwasaki M., Otsuka T., Yamaguchi Y., Reusch M. Phase 3 study of roxadustat to treat anemia in non–dialysis-dependant CKD. Kidney Int. Rep. 2021;6:1810–1828. doi: 10.1016/j.ekir.2021.04.003.  [DOI](https://doi.org/10.1016/j.ekir.2021.04.003) | [PMC free article](/articles/PMC8258605/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34307976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Kidney%20Int.%20Rep.&title=Phase%203%20study%20of%20roxadustat%20to%20treat%20anemia%20in%20non%E2%80%93dialysis-dependant%20CKD&author=T.%20Akizawa&author=M.%20Iwasaki&author=T.%20Otsuka&author=Y.%20Yamaguchi&author=M.%20Reusch&volume=6&publication_year=2021&pages=1810-1828&pmid=34307976&doi=10.1016/j.ekir.2021.04.003&)

23. Chen N., Hao C., Liu B.C., Lin H., Wang C., Xing C., Liang X., Jiang G., Liu Z., Li X., et al. Roxadustat treatment for anemia in patients undergoing long-term dialysis. N. Engl. J. Med. 2019;381:1011–1022. doi: 10.1056/NEJMoa1901713.  [DOI](https://doi.org/10.1056/NEJMoa1901713) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31340116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%C2%A0Engl.%20J.%20Med.&title=Roxadustat%20treatment%20for%20anemia%20in%20patients%20undergoing%20long-term%20dialysis&author=N.%20Chen&author=C.%20Hao&author=B.C.%20Liu&author=H.%20Lin&author=C.%20Wang&volume=381&publication_year=2019&pages=1011-1022&pmid=31340116&doi=10.1056/NEJMoa1901713&)

24. Akizawa T., Nangaku M., Yonekawa T., Okuda N., Kawamatsu S., Onoue T., Endo Y., Hara K., Cobitz A.R. Efficacy and safety of daprodustat compared with darbepoetin alfa in japanese hemodialysis patients with anemia: a randomized, double-blind, phase 3 trial. Clin. J. Am. Soc. Nephrol. 2020;15:1155–1165. doi: 10.2215/CJN.16011219.  [DOI](https://doi.org/10.2215/CJN.16011219) | [PMC free article](/articles/PMC7409739/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32723804/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20J.%20Am.%20Soc.%20Nephrol.&title=Efficacy%20and%20safety%20of%20daprodustat%20compared%20with%20darbepoetin%20alfa%20in%20japanese%20hemodialysis%20patients%20with%20anemia:%20a%20randomized,%20double-blind,%20phase%203%20trial&author=T.%20Akizawa&author=M.%20Nangaku&author=T.%20Yonekawa&author=N.%20Okuda&author=S.%20Kawamatsu&volume=15&publication_year=2020&pages=1155-1165&pmid=32723804&doi=10.2215/CJN.16011219&)

25. Kanai H., Nangaku M., Nagai R., Okuda N., Kurata K., Nagakubo T., Endo Y., Cobitz A. Efficacy and safety of daprodustat in Japanese peritoneal dialysis patients. Ther. Apher. Dial. 2021;25:979–987. doi: 10.1111/1744-9987.13686.  [DOI](https://doi.org/10.1111/1744-9987.13686) | [PMC free article](/articles/PMC9290017/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33964183/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther.%20Apher.%20Dial.&title=Efficacy%20and%20safety%20of%20daprodustat%20in%20Japanese%20peritoneal%20dialysis%20patients&author=H.%20Kanai&author=M.%20Nangaku&author=R.%20Nagai&author=N.%20Okuda&author=K.%20Kurata&volume=25&publication_year=2021&pages=979-987&pmid=33964183&doi=10.1111/1744-9987.13686&)

26. Dhillon S. First approval. Drugs. 2020;80:1491–1497. doi: 10.1007/s40265-020-01384-y.  [DOI](https://doi.org/10.1007/s40265-020-01384-y) | [PMC free article](/articles/PMC7471535/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32880805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drugs&title=First%20approval&author=S.%20Dhillon&volume=80&publication_year=2020&pages=1491-1497&pmid=32880805&doi=10.1007/s40265-020-01384-y&)

27. Gill D., Georgakis M.K., Koskeridis F., Jiang L., Feng Q., Wei W.Q., Theodoratou E., Elliott P., Denny J.C., Malik R., et al. Use of genetic variants related to antihypertensive drugs to inform on efficacy and side effects. Circulation. 2019;140:270–279. doi: 10.1161/CIRCULATIONAHA.118.038814.  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.118.038814) | [PMC free article](/articles/PMC6687408/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31234639/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Use%20of%20genetic%20variants%20related%20to%20antihypertensive%20drugs%20to%20inform%20on%20efficacy%20and%20side%20effects&author=D.%20Gill&author=M.K.%20Georgakis&author=F.%20Koskeridis&author=L.%20Jiang&author=Q.%20Feng&volume=140&publication_year=2019&pages=270-279&pmid=31234639&doi=10.1161/CIRCULATIONAHA.118.038814&)

28. Nelson M.R., Tipney H., Painter J.L., Shen J., Nicoletti P., Shen Y., Floratos A., Sham P.C., Li M.J., Wang J., et al. The support of human genetic evidence for approved drug indications. Nat. Genet. 2015;47:856–860. doi: 10.1038/ng.3314.  [DOI](https://doi.org/10.1038/ng.3314) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26121088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=The%20support%20of%20human%20genetic%20evidence%20for%20approved%20drug%20indications&author=M.R.%20Nelson&author=H.%20Tipney&author=J.L.%20Painter&author=J.%20Shen&author=P.%20Nicoletti&volume=47&publication_year=2015&pages=856-860&pmid=26121088&doi=10.1038/ng.3314&)

29. Nguyen P.A., Born D.A., Deaton A.M., Nioi P., Ward L.D. Phenotypes associated with genes encoding drug targets are predictive of clinical trial side effects. Nat. Commun. 2019;10:2132. doi: 10.1038/s41467-019-10237-6.  [DOI](https://doi.org/10.1038/s41467-019-10237-6) | [PMC free article](/articles/PMC6506545/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31068582/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Phenotypes%20associated%20with%20genes%20encoding%20drug%20targets%20are%20predictive%20of%20clinical%20trial%20side%20effects&author=P.A.%20Nguyen&author=D.A.%20Born&author=A.M.%20Deaton&author=P.%20Nioi&author=L.D.%20Ward&volume=10&publication_year=2019&pages=2132&pmid=31068582&doi=10.1038/s41467-019-10237-6&)

30. Plenge R.M., Scolnick E.M., Altshuler D. Validating therapeutic targets through human genetics. Nat. Rev. Drug Discov. 2013;12:581–594. doi: 10.1038/nrd4051.  [DOI](https://doi.org/10.1038/nrd4051) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23868113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Drug%20Discov.&title=Validating%20therapeutic%20targets%20through%20human%20genetics&author=R.M.%20Plenge&author=E.M.%20Scolnick&author=D.%20Altshuler&volume=12&publication_year=2013&pages=581-594&pmid=23868113&doi=10.1038/nrd4051&)

31. Lotta L.A., Sharp S.J., Burgess S., Perry J.R.B., Stewart I.D., Willems S.M., Luan J., Ardanaz E., Arriola L., Balkau B., et al. Association between low-density lipoprotein cholesterol-lowering genetic variants and risk of type 2 diabetes: a meta-analysis. JAMA. 2016;316:1383–1391. doi: 10.1001/jama.2016.14568.  [DOI](https://doi.org/10.1001/jama.2016.14568) | [PMC free article](/articles/PMC5386134/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27701660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA&title=Association%20between%20low-density%20lipoprotein%20cholesterol-lowering%20genetic%20variants%20and%20risk%20of%20type%202%20diabetes:%20a%20meta-analysis&author=L.A.%20Lotta&author=S.J.%20Sharp&author=S.%20Burgess&author=J.R.B.%20Perry&author=I.D.%20Stewart&volume=316&publication_year=2016&pages=1383-1391&pmid=27701660&doi=10.1001/jama.2016.14568&)

32. Scott R.A., Freitag D.F., Li L., Chu A.Y., Surendran P., Young R., Grarup N., Stancáková A., Chen Y., Varga T.V., et al. A genomic approach to therapeutic target validation identifies a glucose-lowering GLP1R variant protective for coronary heart disease. Sci. Transl. Med. 2016;8:341ra76. doi: 10.1126/scitranslmed.aad3744.  [DOI](https://doi.org/10.1126/scitranslmed.aad3744) | [PMC free article](/articles/PMC5219001/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27252175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Transl.%20Med.&title=A%20genomic%20approach%20to%20therapeutic%20target%20validation%20identifies%20a%20glucose-lowering%20GLP1R%20variant%20protective%20for%20coronary%20heart%20disease&author=R.A.%20Scott&author=D.F.%20Freitag&author=L.%20Li&author=A.Y.%20Chu&author=P.%20Surendran&volume=8&publication_year=2016&pages=341ra76&pmid=27252175&doi=10.1126/scitranslmed.aad3744&)

33. Swerdlow D.I., Preiss D., Kuchenbaecker K.B., Holmes M.V., Engmann J.E.L., Shah T., Sofat R., Stender S., Johnson P.C.D., Scott R.A., et al. HMG-coenzyme A reductase inhibition, type 2 diabetes, and bodyweight: evidence from genetic analysis and randomised trials. Lancet (London, England) 2015;385:351–361. doi: 10.1016/S0140-6736(14)61183-1.  [DOI](https://doi.org/10.1016/S0140-6736(14)61183-1) | [PMC free article](/articles/PMC4322187/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25262344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20(London,%20England)&title=HMG-coenzyme%20A%20reductase%20inhibition,%20type%202%20diabetes,%20and%20bodyweight:%20evidence%20from%20genetic%20analysis%20and%20randomised%20trials&author=D.I.%20Swerdlow&author=D.%20Preiss&author=K.B.%20Kuchenbaecker&author=M.V.%20Holmes&author=J.E.L.%20Engmann&volume=385&publication_year=2015&pages=351-361&pmid=25262344&doi=10.1016/S0140-6736(14)61183-1&)

34. Melzer D., Perry J.R.B., Hernandez D., Corsi A.M., Stevens K., Rafferty I., Lauretani F., Murray A., Gibbs J.R., Paolisso G., et al. A Genome-Wide Association Study Identifies Protein Quantitative Trait Loci (pQTLs) PLoS Genet. 2008;4:e1000072. doi: 10.1371/journal.pgen.1000072.  [DOI](https://doi.org/10.1371/journal.pgen.1000072) | [PMC free article](/articles/PMC2362067/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18464913/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=A%20Genome-Wide%20Association%20Study%20Identifies%20Protein%20Quantitative%20Trait%20Loci%20(pQTLs)&author=D.%20Melzer&author=J.R.B.%20Perry&author=D.%20Hernandez&author=A.M.%20Corsi&author=K.%20Stevens&volume=4&publication_year=2008&pages=e1000072&pmid=18464913&doi=10.1371/journal.pgen.1000072&)

35. Walker V.M., Davey Smith G., Davies N.M., Martin R.M. Mendelian randomization: a novel approach for the prediction of adverse drug events and drug repurposing opportunities. Int. J. Epidemiol. 2017;46:2078–2089. doi: 10.1093/ije/dyx207.  [DOI](https://doi.org/10.1093/ije/dyx207) | [PMC free article](/articles/PMC5837479/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29040597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Epidemiol.&title=Mendelian%20randomization:%20a%20novel%20approach%20for%20the%20prediction%20of%20adverse%20drug%20events%20and%20drug%20repurposing%20opportunities&author=V.M.%20Walker&author=G.%20Davey%20Smith&author=N.M.%20Davies&author=R.M.%20Martin&volume=46&publication_year=2017&pages=2078-2089&pmid=29040597&doi=10.1093/ije/dyx207&)

36. Swerdlow D.I., Kuchenbaecker K.B., Shah S., Sofat R., Holmes M.V., White J., Mindell J.S., Kivimaki M., Brunner E.J., Whittaker J.C., et al. Selecting instruments for Mendelian randomization in the wake of genome-wide association studies. Int. J. Epidemiol. 2016;45:1600–1616. doi: 10.1093/ije/dyw088.  [DOI](https://doi.org/10.1093/ije/dyw088) | [PMC free article](/articles/PMC5100611/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27342221/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Epidemiol.&title=Selecting%20instruments%20for%20Mendelian%20randomization%20in%20the%20wake%20of%20genome-wide%20association%20studies&author=D.I.%20Swerdlow&author=K.B.%20Kuchenbaecker&author=S.%20Shah&author=R.%20Sofat&author=M.V.%20Holmes&volume=45&publication_year=2016&pages=1600-1616&pmid=27342221&doi=10.1093/ije/dyw088&)

37. Davey Smith G., Hemani G. Mendelian randomization: genetic anchors for causal inference in epidemiological studies. Hum. Mol. Genet. 2014;23:R89–R98. doi: 10.1093/hmg/ddu328.  [DOI](https://doi.org/10.1093/hmg/ddu328) | [PMC free article](/articles/PMC4170722/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25064373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&title=Mendelian%20randomization:%20genetic%20anchors%20for%20causal%20inference%20in%20epidemiological%20studies&author=G.%20Davey%20Smith&author=G.%20Hemani&volume=23&publication_year=2014&pages=R89-R98&pmid=25064373&doi=10.1093/hmg/ddu328&)

38. Lichou F., Trynka G. Functional studies of GWAS variants are gaining momentum. Nat. Commun. 2020;11:6283. doi: 10.1038/s41467-020-20188-y.  [DOI](https://doi.org/10.1038/s41467-020-20188-y) | [PMC free article](/articles/PMC7722852/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33293539/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Functional%20studies%20of%20GWAS%20variants%20are%20gaining%20momentum&author=F.%20Lichou&author=G.%20Trynka&volume=11&publication_year=2020&pages=6283&pmid=33293539&doi=10.1038/s41467-020-20188-y&)

39. Wang H., La Russa M., Qi L.S. CRISPR/Cas9 in Genome Editing and Beyond. Annu. Rev. Biochem. 2016;85:227–264. doi: 10.1146/annurev-biochem-060815-014607.  [DOI](https://doi.org/10.1146/annurev-biochem-060815-014607) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27145843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Biochem.&title=CRISPR/Cas9%20in%20Genome%20Editing%20and%20Beyond&author=H.%20Wang&author=M.%20La%20Russa&author=L.S.%20Qi&volume=85&publication_year=2016&pages=227-264&pmid=27145843&doi=10.1146/annurev-biochem-060815-014607&)

40. Yang L., Guell M., Byrne S., Yang J.L., De Los Angeles A., Mali P., Aach J., Kim-Kiselak C., Briggs A.W., Rios X., et al. Optimization of scarless human stem cell genome editing. Nucleic Acids Res. 2013;41:9049–9061. doi: 10.1093/nar/gkt555.  [DOI](https://doi.org/10.1093/nar/gkt555) | [PMC free article](/articles/PMC3799423/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23907390/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Optimization%20of%20scarless%20human%20stem%20cell%20genome%20editing&author=L.%20Yang&author=M.%20Guell&author=S.%20Byrne&author=J.L.%20Yang&author=A.%20De%20Los%20Angeles&volume=41&publication_year=2013&pages=9049-9061&pmid=23907390&doi=10.1093/nar/gkt555&)

41. Zhou X., Stephens M. Genome-wide efficient mixed-model analysis for association studies. Nat. Genet. 2012;44:821–824. doi: 10.1038/ng.2310.  [DOI](https://doi.org/10.1038/ng.2310) | [PMC free article](/articles/PMC3386377/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22706312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Genome-wide%20efficient%20mixed-model%20analysis%20for%20association%20studies&author=X.%20Zhou&author=M.%20Stephens&volume=44&publication_year=2012&pages=821-824&pmid=22706312&doi=10.1038/ng.2310&)

42. Willer C.J., Li Y., Abecasis G.R. METAL: fast and efficient meta-analysis of genome-wide association scans. Bioinformatics. 2010;26:2190–2191. doi: 10.1093/bioinformatics/btq340.  [DOI](https://doi.org/10.1093/bioinformatics/btq340) | [PMC free article](/articles/PMC2922887/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20616382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=METAL:%20fast%20and%20efficient%20meta-analysis%20of%20genome-wide%20association%20scans&author=C.J.%20Willer&author=Y.%20Li&author=G.R.%20Abecasis&volume=26&publication_year=2010&pages=2190-2191&pmid=20616382&doi=10.1093/bioinformatics/btq340&)

43. Yang J., Lee S.H., Goddard M.E., Visscher P.M. GCTA: a tool for genome-wide complex trait analysis. Am. J. Hum. Genet. 2011;88:76–82. doi: 10.1016/j.ajhg.2010.11.011.  [DOI](https://doi.org/10.1016/j.ajhg.2010.11.011) | [PMC free article](/articles/PMC3014363/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21167468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=GCTA:%20a%20tool%20for%20genome-wide%20complex%20trait%20analysis&author=J.%20Yang&author=S.H.%20Lee&author=M.E.%20Goddard&author=P.M.%20Visscher&volume=88&publication_year=2011&pages=76-82&pmid=21167468&doi=10.1016/j.ajhg.2010.11.011&)

44. Yang J., Ferreira T., Morris A.P., Medland S.E., Genetic Investigation of ANthropometric Traits GIANT Consortium. DIAbetes Genetics Replication And Meta-analysis DIAGRAM Consortium. Madden P.A.F., Heath A.C., Martin N.G., Montgomery G.W., et al. Conditional and joint multiple-SNP analysis of GWAS summary statistics identifies additional variants influencing complex traits. Nat. Genet. 2012;44:369–375. doi: 10.1038/ng.2213.  [DOI](https://doi.org/10.1038/ng.2213) | [PMC free article](/articles/PMC3593158/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22426310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Conditional%20and%20joint%20multiple-SNP%20analysis%20of%20GWAS%20summary%20statistics%20identifies%20additional%20variants%20influencing%20complex%20traits&author=J.%20Yang&author=T.%20Ferreira&author=A.P.%20Morris&author=S.E.%20Medland&author=P.A.F.%20Madden&volume=44&publication_year=2012&pages=369-375&pmid=22426310&doi=10.1038/ng.2213&)

45. Etheridge A.S., Gallins P.J., Jima D., Broadaway K.A., Ratain M.J., Schuetz E., Schadt E., Schroder A., Molony C., Zhou Y., et al. A new liver expression quantitative trait locus map from 1, 183 individuals provides evidence for novel expression quantitative trait loci of drug response, metabolic, and sex-biased phenotypes. Clin. Pharmacol. Ther. 2020;107:1383–1393. doi: 10.1002/cpt.1751.  [DOI](https://doi.org/10.1002/cpt.1751) | [PMC free article](/articles/PMC7816646/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31868224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=A%20new%20liver%20expression%20quantitative%20trait%20locus%20map%20from%201,%20183%20individuals%20provides%20evidence%20for%20novel%20expression%20quantitative%20trait%20loci%20of%20drug%20response,%20metabolic,%20and%20sex-biased%20phenotypes&author=A.S.%20Etheridge&author=P.J.%20Gallins&author=D.%20Jima&author=K.A.%20Broadaway&author=M.J.%20Ratain&volume=107&publication_year=2020&pages=1383-1393&pmid=31868224&doi=10.1002/cpt.1751&)

46. Damman J., Bloks V.W., Daha M.R., van der Most P.J., Sanjabi B., van der Vlies P., Snieder H., Ploeg R.J., Krikke C., Leuvenink H.G.D., Seelen M.A. Hypoxia and complement-and-coagulation pathways in the deceased organ donor as the major target for intervention to improve renal allograft outcome. Transplantation. 2015;99:1293–1300. doi: 10.1097/TP.0000000000000500.  [DOI](https://doi.org/10.1097/TP.0000000000000500) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25427168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transplantation&title=Hypoxia%20and%20complement-and-coagulation%20pathways%20in%20the%20deceased%20organ%20donor%20as%20the%20major%20target%20for%20intervention%20to%20improve%20renal%20allograft%20outcome&author=J.%20Damman&author=V.W.%20Bloks&author=M.R.%20Daha&author=P.J.%20van%20der%20Most&author=B.%20Sanjabi&volume=99&publication_year=2015&pages=1293-1300&pmid=25427168&doi=10.1097/TP.0000000000000500&)

47. Giambartolomei C., Vukcevic D., Schadt E.E., Franke L., Hingorani A.D., Wallace C., Plagnol V. Bayesian Test for Colocalisation between Pairs of Genetic Association Studies Using Summary Statistics. PLoS Genet. 2014;10:e1004383. doi: 10.1371/journal.pgen.1004383.  [DOI](https://doi.org/10.1371/journal.pgen.1004383) | [PMC free article](/articles/PMC4022491/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24830394/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Bayesian%20Test%20for%20Colocalisation%20between%20Pairs%20of%20Genetic%20Association%20Studies%20Using%20Summary%20Statistics&author=C.%20Giambartolomei&author=D.%20Vukcevic&author=E.E.%20Schadt&author=L.%20Franke&author=A.D.%20Hingorani&volume=10&publication_year=2014&pages=e1004383&pmid=24830394&doi=10.1371/journal.pgen.1004383&)

48. Wallace C. Eliciting priors and relaxing the single causal variant assumption in colocalisation analyses. PLoS Genet. 2020;16:e1008720. doi: 10.1371/journal.pgen.1008720.  [DOI](https://doi.org/10.1371/journal.pgen.1008720) | [PMC free article](/articles/PMC7192519/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32310995/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=Eliciting%20priors%20and%20relaxing%20the%20single%20causal%20variant%20assumption%20in%20colocalisation%20analyses&author=C.%20Wallace&volume=16&publication_year=2020&pages=e1008720&pmid=32310995&doi=10.1371/journal.pgen.1008720&)

49. Liu T., Jia P., Ma H., Reed S.A., Luo X., Larman H.B., Schultz P.G. Construction and screening of a lentiviral secretome library. Cell Chem. Biol. 2017;24:767–771.e3. doi: 10.1016/j.chembiol.2017.05.017.  [DOI](https://doi.org/10.1016/j.chembiol.2017.05.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28602759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Chem.%20Biol.&title=Construction%20and%20screening%20of%20a%20lentiviral%20secretome%20library&author=T.%20Liu&author=P.%20Jia&author=H.%20Ma&author=S.A.%20Reed&author=X.%20Luo&volume=24&publication_year=2017&pages=767-771.e3&pmid=28602759&doi=10.1016/j.chembiol.2017.05.017&)

50. Livak K.J., Schmittgen T.D. Analysis of Relative Gene Expression Data Using Real-Time Quantitative PCR and the 2−ΔΔCT Method. Methods. 2001;25:402–408. doi: 10.1006/meth.2001.1262.  [DOI](https://doi.org/10.1006/meth.2001.1262) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11846609/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Methods&title=Analysis%20of%20Relative%20Gene%20Expression%20Data%20Using%20Real-Time%20Quantitative%20PCR%20and%20the%202%E2%88%92%CE%94%CE%94CT%20Method&author=K.J.%20Livak&author=T.D.%20Schmittgen&volume=25&publication_year=2001&pages=402-408&pmid=11846609&doi=10.1006/meth.2001.1262&)

51. Xie F., Xiao P., Chen D., Xu L., Zhang B. miRDeepFinder: a miRNA analysis tool for deep sequencing of plant small RNAs. Plant Mol. Biol. 2012;80:75–84. doi: 10.1007/s11103-012-9885-2.  [DOI](https://doi.org/10.1007/s11103-012-9885-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22290409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Plant%20Mol.%20Biol.&title=miRDeepFinder:%20a%20miRNA%20analysis%20tool%20for%20deep%20sequencing%20of%20plant%20small%20RNAs&author=F.%20Xie&author=P.%20Xiao&author=D.%20Chen&author=L.%20Xu&author=B.%20Zhang&volume=80&publication_year=2012&pages=75-84&pmid=22290409&doi=10.1007/s11103-012-9885-2&)

52. RStudio Team . RStudio; PBC, Boston, MA: 2018. RStudio: Integrated Development for R.https://www.rstudio.com/  [https://www.rstudio.com/](https://www.rstudio.com/) | [Google Scholar](https://scholar.google.com/scholar_lookup?RStudio%20Team%20.%20RStudio;%20PBC,%20Boston,%20MA:%202018.%20RStudio:%20Integrated%20Development%20for%20R.https://www.rstudio.com/)

53. Schroeder A., Mueller O., Stocker S., Salowsky R., Leiber M., Gassmann M., Lightfoot S., Menzel W., Granzow M., Ragg T. The RIN: an RNA integrity number for assigning integrity values to RNA measurements. BMC Mol. Biol. 2006;7:3. doi: 10.1186/1471-2199-7-3.  [DOI](https://doi.org/10.1186/1471-2199-7-3) | [PMC free article](/articles/PMC1413964/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16448564/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Mol.%20Biol.&title=The%20RIN:%20an%20RNA%20integrity%20number%20for%20assigning%20integrity%20values%20to%20RNA%20measurements&author=A.%20Schroeder&author=O.%20Mueller&author=S.%20Stocker&author=R.%20Salowsky&author=M.%20Leiber&volume=7&publication_year=2006&pages=3&pmid=16448564&doi=10.1186/1471-2199-7-3&)

54. Ewels P., Magnusson M., Lundin S., Käller M. MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics. 2016;32:3047–3048. doi: 10.1093/bioinformatics/btw354.  [DOI](https://doi.org/10.1093/bioinformatics/btw354) | [PMC free article](/articles/PMC5039924/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27312411/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=MultiQC:%20summarize%20analysis%20results%20for%20multiple%20tools%20and%20samples%20in%20a%20single%20report&author=P.%20Ewels&author=M.%20Magnusson&author=S.%20Lundin&author=M.%20K%C3%A4ller&volume=32&publication_year=2016&pages=3047-3048&pmid=27312411&doi=10.1093/bioinformatics/btw354&)

55. Martin M. Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet. J. 2011;17:10. doi: 10.14806/ej.17.1.200. Next Gener. Seq. Data Anal.  [DOI](https://doi.org/10.14806/ej.17.1.200) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBnet.%20J.&title=Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&author=M.%20Martin&volume=17&publication_year=2011&pages=10&doi=10.14806/ej.17.1.200&)

56. Dobin A., Davis C.A., Schlesinger F., Drenkow J., Zaleski C., Jha S., Batut P., Chaisson M., Gingeras T.R. STAR: ultrafast universal RNA-seq aligner. Bioinformatics. 2013;29:15–21. doi: 10.1093/bioinformatics/bts635.  [DOI](https://doi.org/10.1093/bioinformatics/bts635) | [PMC free article](/articles/PMC3530905/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23104886/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=STAR:%20ultrafast%20universal%20RNA-seq%20aligner&author=A.%20Dobin&author=C.A.%20Davis&author=F.%20Schlesinger&author=J.%20Drenkow&author=C.%20Zaleski&volume=29&publication_year=2013&pages=15-21&pmid=23104886&doi=10.1093/bioinformatics/bts635&)

57. Liao Y., Smyth G.K., Shi W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Bioinformatics. 2014;30:923–930. doi: 10.1093/bioinformatics/btt656.  [DOI](https://doi.org/10.1093/bioinformatics/btt656) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24227677/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=featureCounts:%20an%20efficient%20general%20purpose%20program%20for%20assigning%20sequence%20reads%20to%20genomic%20features&author=Y.%20Liao&author=G.K.%20Smyth&author=W.%20Shi&volume=30&publication_year=2014&pages=923-930&pmid=24227677&doi=10.1093/bioinformatics/btt656&)

58. Love M.I., Huber W., Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 2014;15:550. doi: 10.1186/s13059-014-0550-8.  [DOI](https://doi.org/10.1186/s13059-014-0550-8) | [PMC free article](/articles/PMC4302049/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25516281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&author=M.I.%20Love&author=W.%20Huber&author=S.%20Anders&volume=15&publication_year=2014&pages=550&pmid=25516281&doi=10.1186/s13059-014-0550-8&)

59. Chen E.Y., Tan C.M., Kou Y., Duan Q., Wang Z., Meirelles G.V., Clark N.R., Ma'ayan A. Enrichr: interactive and collaborative HTML5 gene list enrichment analysis tool. BMC Bioinf. 2013;14:128. doi: 10.1186/1471-2105-14-128.  [DOI](https://doi.org/10.1186/1471-2105-14-128) | [PMC free article](/articles/PMC3637064/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23586463/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinf.&title=Enrichr:%20interactive%20and%20collaborative%20HTML5%20gene%20list%20enrichment%20analysis%20tool&author=E.Y.%20Chen&author=C.M.%20Tan&author=Y.%20Kou&author=Q.%20Duan&author=Z.%20Wang&volume=14&publication_year=2013&pages=128&pmid=23586463&doi=10.1186/1471-2105-14-128&)

60. Kuleshov M.V., Jones M.R., Rouillard A.D., Fernandez N.F., Duan Q., Wang Z., Koplev S., Jenkins S.L., Jagodnik K.M., Lachmann A., et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. Nucleic Acids Res. 2016;44:W90–W97. doi: 10.1093/nar/gkw377.  [DOI](https://doi.org/10.1093/nar/gkw377) | [PMC free article](/articles/PMC4987924/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27141961/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Enrichr:%20a%20comprehensive%20gene%20set%20enrichment%20analysis%20web%20server%202016%20update&author=M.V.%20Kuleshov&author=M.R.%20Jones&author=A.D.%20Rouillard&author=N.F.%20Fernandez&author=Q.%20Duan&volume=44&publication_year=2016&pages=W90-W97&pmid=27141961&doi=10.1093/nar/gkw377&)

61. Xie Z., Bailey A., Kuleshov M.V., Clarke D.J.B., Evangelista J.E., Jenkins S.L., Lachmann A., Wojciechowicz M.L., Kropiwnicki E., Jagodnik K.M., et al. Gene set knowledge discovery with enrichr. Curr. Protoc. 2021;1:e90. doi: 10.1002/cpz1.90.  [DOI](https://doi.org/10.1002/cpz1.90) | [PMC free article](/articles/PMC8152575/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33780170/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.&title=Gene%20set%20knowledge%20discovery%20with%20enrichr&author=Z.%20Xie&author=A.%20Bailey&author=M.V.%20Kuleshov&author=D.J.B.%20Clarke&author=J.E.%20Evangelista&volume=1&publication_year=2021&pages=e90&pmid=33780170&doi=10.1002/cpz1.90&)

62. Oguro-Ando A., Rosensweig C., Herman E., Nishimura Y., Werling D., Bill B.R., Berg J.M., Gao F., Coppola G., Abrahams B.S., Geschwind D.H. Increased CYFIP1 dosage alters cellular and dendritic morphology and dysregulates mTOR. Mol. Psychiatry. 2015;20:1069–1078. doi: 10.1038/mp.2014.124.  [DOI](https://doi.org/10.1038/mp.2014.124) | [PMC free article](/articles/PMC4409498/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25311365/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Psychiatry&title=Increased%20CYFIP1%20dosage%20alters%20cellular%20and%20dendritic%20morphology%20and%20dysregulates%20mTOR&author=A.%20Oguro-Ando&author=C.%20Rosensweig&author=E.%20Herman&author=Y.%20Nishimura&author=D.%20Werling&volume=20&publication_year=2015&pages=1069-1078&pmid=25311365&doi=10.1038/mp.2014.124&)

63. Chin C.L., Goh J.B., Srinivasan H., Liu K.I., Gowher A., Shanmugam R., Lim H.L., Choo M., Tang W.Q., Tan A.H.M., et al. A human expression system based on HEK293 for the stable production of recombinant erythropoietin. Sci. Rep. 2019;9:16768. doi: 10.1038/s41598-019-53391-z.  [DOI](https://doi.org/10.1038/s41598-019-53391-z) | [PMC free article](/articles/PMC6856173/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31727983/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=A%20human%20expression%20system%20based%20on%20HEK293%20for%20the%20stable%20production%20of%20recombinant%20erythropoietin&author=C.L.%20Chin&author=J.B.%20Goh&author=H.%20Srinivasan&author=K.I.%20Liu&author=A.%20Gowher&volume=9&publication_year=2019&pages=16768&pmid=31727983&doi=10.1038/s41598-019-53391-z&)

64. Nikpay M., Goel A., Won H.H., Hall L.M., Willenborg C., Kanoni S., Saleheen D., Kyriakou T., Nelson C.P., Hopewell J.C., et al. A comprehensive 1, 000 Genomes-based genome-wide association meta-analysis of coronary artery disease. Nat. Genet. 2015;47:1121–1130. doi: 10.1038/ng.3396.  [DOI](https://doi.org/10.1038/ng.3396) | [PMC free article](/articles/PMC4589895/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26343387/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=A%20comprehensive%201,%20000%20Genomes-based%20genome-wide%20association%20meta-analysis%20of%20coronary%20artery%20disease&author=M.%20Nikpay&author=A.%20Goel&author=H.H.%20Won&author=L.M.%20Hall&author=C.%20Willenborg&volume=47&publication_year=2015&pages=1121-1130&pmid=26343387&doi=10.1038/ng.3396&)

65. Iwaki T., Umemura K. A single plasmid transfection that offers a significant advantage associated with puromycin selection, fluorescence-assisted cell sorting, and doxycycline-inducible protein expression in mammalian cells. Cytotechnology. 2011;63:337–343. doi: 10.1007/s10616-011-9357-6.  [DOI](https://doi.org/10.1007/s10616-011-9357-6) | [PMC free article](/articles/PMC3140840/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21553115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cytotechnology&title=A%20single%20plasmid%20transfection%20that%20offers%20a%20significant%20advantage%20associated%20with%20puromycin%20selection,%20fluorescence-assisted%20cell%20sorting,%20and%20doxycycline-inducible%20protein%20expression%20in%20mammalian%20cells&author=T.%20Iwaki&author=K.%20Umemura&volume=63&publication_year=2011&pages=337-343&pmid=21553115&doi=10.1007/s10616-011-9357-6&)

66. Yusa K. Seamless genome editing in human pluripotent stem cells using custom endonuclease–based gene targeting and the piggyBac transposon. Nat. Protoc. 2013;8:2061–2078. doi: 10.1038/nprot.2013.126.  [DOI](https://doi.org/10.1038/nprot.2013.126) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24071911/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&title=Seamless%20genome%20editing%20in%20human%20pluripotent%20stem%20cells%20using%20custom%20endonuclease%E2%80%93based%20gene%20targeting%20and%20the%20piggyBac%20transposon&author=K.%20Yusa&volume=8&publication_year=2013&pages=2061-2078&pmid=24071911&doi=10.1038/nprot.2013.126&)

67. Malik R., Chauhan G., Traylor M., Sargurupremraj M., Okada Y., Mishra A., Rutten-Jacobs L., Giese A.K., van der Laan S.W., Gretarsdottir S., et al. Multiancestry genome-wide association study of 520, 000 subjects identifies 32 loci associated with stroke and stroke subtypes. Nat. Genet. 2018;50:524–537. doi: 10.1038/s41588-018-0058-3.  [DOI](https://doi.org/10.1038/s41588-018-0058-3) | [PMC free article](/articles/PMC5968830/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29531354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Multiancestry%20genome-wide%20association%20study%20of%20520,%20000%20subjects%20identifies%2032%20loci%20associated%20with%20stroke%20and%20stroke%20subtypes&author=R.%20Malik&author=G.%20Chauhan&author=M.%20Traylor&author=M.%20Sargurupremraj&author=Y.%20Okada&volume=50&publication_year=2018&pages=524-537&pmid=29531354&doi=10.1038/s41588-018-0058-3&)

68. Harris R.J., Deeks J.J., Altman D.G., Bradburn M.J., Harbord R.M., Sterne J.A.C. metan: fixed- and random-effects meta-analysis. STATA J. 2008;8:3–28.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=STATA%20J.&title=metan:%20fixed-%20and%20random-effects%20meta-analysis&author=R.J.%20Harris&author=J.J.%20Deeks&author=D.G.%20Altman&author=M.J.%20Bradburn&author=R.M.%20Harbord&volume=8&publication_year=2008&pages=3-28&)

69. Burgess S., Small D.S., Thompson S.G. A review of instrumental variable estimators for Mendelian randomization. Stat. Methods Med. Res. 2017;26:2333–2355. doi: 10.1177/0962280215597579.  [DOI](https://doi.org/10.1177/0962280215597579) | [PMC free article](/articles/PMC5642006/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26282889/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stat.%20Methods%20Med.%20Res.&title=A%20review%20of%20instrumental%20variable%20estimators%20for%20Mendelian%20randomization&author=S.%20Burgess&author=D.S.%20Small&author=S.G.%20Thompson&volume=26&publication_year=2017&pages=2333-2355&pmid=26282889&doi=10.1177/0962280215597579&)

70. Wain L.V., Vaez A., Jansen R., Joehanes R., Van Der Most P.J., Erzurumluoglu A.M., O’Reilly P.F., Cabrera C.P., Warren H.R., Rose L.M., Verwoert G.C. Novel Blood Pressure Locus and Gene Discovery Using Genome-Wide Association Study and Expression Data Sets From Blood and the Kidney. Hypertension. 2017;70:e4–e19. doi: 10.1161/hypertensionaha.117.09438.  [DOI](https://doi.org/10.1161/hypertensionaha.117.09438) | [PMC free article](/articles/PMC5783787/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28739976/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hypertension&title=Novel%20Blood%20Pressure%20Locus%20and%20Gene%20Discovery%20Using%20Genome-Wide%20Association%20Study%20and%20Expression%20Data%20Sets%20From%20Blood%20and%20the%20Kidney&author=L.V.%20Wain&author=A.%20Vaez&author=R.%20Jansen&author=R.%20Joehanes&author=P.J.%20Van%20Der%20Most&volume=70&publication_year=2017&pages=e4-e19&pmid=28739976&doi=10.1161/hypertensionaha.117.09438&)

71. den Hoed M., Eijgelsheim M., Esko T., Brundel B.J.J.M., Peal D.S., Evans D.M., Nolte I.M., Segrè A.V., Holm H., Handsaker R.E., et al. Identification of heart rate-associated loci and their effects on cardiac conduction and rhythm disorders. Nat. Genet. 2013;45:621–631. doi: 10.1038/ng.2610.  [DOI](https://doi.org/10.1038/ng.2610) | [PMC free article](/articles/PMC3696959/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23583979/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Identification%20of%20heart%20rate-associated%20loci%20and%20their%20effects%20on%20cardiac%20conduction%20and%20rhythm%20disorders&author=M.%20den%20Hoed&author=M.%20Eijgelsheim&author=T.%20Esko&author=B.J.J.M.%20Brundel&author=D.S.%20Peal&volume=45&publication_year=2013&pages=621-631&pmid=23583979&doi=10.1038/ng.2610&)

72. Meadowcroft A.M., Cizman B., Holdstock L., Biswas N., Johnson B.M., Jones D., Nossuli A.K., Lepore J.J., Aarup M., Cobitz A.R. Daprodustat for anemia: A 24-week, open-label, randomized controlled trial in participants on hemodialysis. Clin. Kidney J. 2019;12:139–148. doi: 10.1093/ckj/sfy014.  [DOI](https://doi.org/10.1093/ckj/sfy014) | [PMC free article](/articles/PMC6366140/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30746141/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Kidney%20J.&title=Daprodustat%20for%20anemia:%20A%2024-week,%20open-label,%20randomized%20controlled%20trial%20in%20participants%20on%20hemodialysis&author=A.M.%20Meadowcroft&author=B.%20Cizman&author=L.%20Holdstock&author=N.%20Biswas&author=B.M.%20Johnson&volume=12&publication_year=2019&pages=139-148&pmid=30746141&doi=10.1093/ckj/sfy014&)

73. Loh P.-R., Tucker G., Bulik-Sullivan B.K., Vilhjálmsson B.J., Finucane H.K., Salem R.M., Chasman D.I., Ridker P.M., Neale B.M., Berger B., et al. Efficient Bayesian mixed-model analysis increases association power in large cohorts. Nat. Genet. 2015;47:284–290. doi: 10.1038/ng.3190.  [DOI](https://doi.org/10.1038/ng.3190) | [PMC free article](/articles/PMC4342297/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25642633/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Efficient%20Bayesian%20mixed-model%20analysis%20increases%20association%20power%20in%20large%20cohorts&author=P.-R.%20Loh&author=G.%20Tucker&author=B.K.%20Bulik-Sullivan&author=B.J.%20Vilhj%C3%A1lmsson&author=H.K.%20Finucane&volume=47&publication_year=2015&pages=284-290&pmid=25642633&doi=10.1038/ng.3190&)

74. Frayling T.M., Beaumont R.N., Jones S.E., Yaghootkar H., Tuke M.A., Ruth K.S., Casanova F., West B., Locke J., Sharp S., et al. A common allele in FGF21 associated with sugar intake is associated with body shape, lower total body-fat percentage, and higher blood pressure. Cell Rep. 2018;23:327–336. doi: 10.1016/j.celrep.2018.03.070.  [DOI](https://doi.org/10.1016/j.celrep.2018.03.070) | [PMC free article](/articles/PMC5912948/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29641994/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&title=A%20common%20allele%20in%20FGF21%20associated%20with%20sugar%20intake%20is%20associated%20with%20body%20shape,%20lower%20total%20body-fat%20percentage,%20and%20higher%20blood%20pressure&author=T.M.%20Frayling&author=R.N.%20Beaumont&author=S.E.%20Jones&author=H.%20Yaghootkar&author=M.A.%20Tuke&volume=23&publication_year=2018&pages=327-336&pmid=29641994&doi=10.1016/j.celrep.2018.03.070&)

75. Grote Beverborg N., Verweij N., Klip I.T., van der Wal H.H., Voors A.A., van Veldhuisen D.J., Gansevoort R.T., Bakker S.J.L., van der Harst P., van der Meer P. Erythropoietin in the general population: reference ranges and clinical, biochemical and genetic correlates. PLoS One. 2015;10:e0125215. doi: 10.1371/journal.pone.0125215.  [DOI](https://doi.org/10.1371/journal.pone.0125215) | [PMC free article](/articles/PMC4411129/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25915923/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Erythropoietin%20in%20the%20general%20population:%20reference%20ranges%20and%20clinical,%20biochemical%20and%20genetic%20correlates&author=N.%20Grote%20Beverborg&author=N.%20Verweij&author=I.T.%20Klip&author=H.H.%20van%20der%20Wal&author=A.A.%20Voors&volume=10&publication_year=2015&pages=e0125215&pmid=25915923&doi=10.1371/journal.pone.0125215&)

76. Benyamin B., Esko T., Ried J.S., Radhakrishnan A., Vermeulen S.H., Traglia M., Gögele M., Anderson D., Broer L., Podmore C., et al. Novel loci affecting iron homeostasis and their effects in individuals at risk for hemochromatosis. Nat. Commun. 2014;5:4926. doi: 10.1038/ncomms5926.  [DOI](https://doi.org/10.1038/ncomms5926) | [PMC free article](/articles/PMC4215164/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25352340/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Novel%20loci%20affecting%20iron%20homeostasis%20and%20their%20effects%20in%20individuals%20at%20risk%20for%20hemochromatosis&author=B.%20Benyamin&author=T.%20Esko&author=J.S.%20Ried&author=A.%20Radhakrishnan&author=S.H.%20Vermeulen&volume=5&publication_year=2014&pages=4926&pmid=25352340&doi=10.1038/ncomms5926&)

77. Chen M.-H., Raffield L.M., Mousas A., Sakaue S., Huffman J.E., Moscati A., Trivedi B., Jiang T., Akbari P., Vuckovic D., et al. Trans-ethnic and ancestry-specific blood-cell genetics in 746, 667 individuals from 5 global populations. Cell. 2020;182:1198–1213.e14. doi: 10.1016/j.cell.2020.06.045.  [DOI](https://doi.org/10.1016/j.cell.2020.06.045) | [PMC free article](/articles/PMC7480402/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32888493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Trans-ethnic%20and%20ancestry-specific%20blood-cell%20genetics%20in%20746,%20667%20individuals%20from%205%20global%20populations&author=M.-H.%20Chen&author=L.M.%20Raffield&author=A.%20Mousas&author=S.%20Sakaue&author=J.E.%20Huffman&volume=182&publication_year=2020&pages=1198-1213.e14&pmid=32888493&doi=10.1016/j.cell.2020.06.045&)

78. Tong Z., Yang Z., Patel S., Chen H., Gibbs D., Yang X., Hau V.S., Kaminoh Y., Harmon J., Pearson E., et al. Promoter polymorphism of the erythropoietin gene in severe diabetic eye and kidney complications. Proc. Natl. Acad. Sci. USA. 2008;105:6998–7003. doi: 10.1073/pnas.0800454105.  [DOI](https://doi.org/10.1073/pnas.0800454105) | [PMC free article](/articles/PMC2383970/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18458324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&title=Promoter%20polymorphism%20of%20the%20erythropoietin%20gene%20in%20severe%20diabetic%20eye%20and%20kidney%20complications&author=Z.%20Tong&author=Z.%20Yang&author=S.%20Patel&author=H.%20Chen&author=D.%20Gibbs&volume=105&publication_year=2008&pages=6998-7003&pmid=18458324&doi=10.1073/pnas.0800454105&)

79. Amanzada A., Goralczyk A.D., Reinhardt L., Moriconi F., Cameron S., Mihm S. Erythropoietin rs1617640 G allele associates with an attenuated rise of serum erythropoietin and a marked decline of hemoglobin in hepatitis C patients undergoing antiviral therapy. BMC Infect. Dis. 2014;14:503. doi: 10.1186/1471-2334-14-503.  [DOI](https://doi.org/10.1186/1471-2334-14-503) | [PMC free article](/articles/PMC4175618/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25227310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Infect.%20Dis.&title=Erythropoietin%20rs1617640%C2%A0G%20allele%20associates%20with%20an%20attenuated%20rise%20of%20serum%20erythropoietin%20and%20a%20marked%20decline%20of%20hemoglobin%20in%20hepatitis%20C%20patients%20undergoing%20antiviral%20therapy&author=A.%20Amanzada&author=A.D.%20Goralczyk&author=L.%20Reinhardt&author=F.%20Moriconi&author=S.%20Cameron&volume=14&publication_year=2014&pages=503&pmid=25227310&doi=10.1186/1471-2334-14-503&)

80. Franklin B.H. Erythropoietin. Cold Spring Harb. Perspect. Med. 2013;3:a011619. doi: 10.1101/cshperspect.a011619.  [DOI](https://doi.org/10.1101/cshperspect.a011619) | [PMC free article](/articles/PMC3579209/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23457296/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb.%20Perspect.%20Med.&title=Erythropoietin&author=B.H.%20Franklin&volume=3&publication_year=2013&pages=a011619&pmid=23457296&doi=10.1101/cshperspect.a011619&)

81. Forejtnikovà H., Vieillevoye M., Zermati Y., Lambert M., Pellegrino R.M., Guihard S., Gaudry M., Camaschella C., Lacombe C., Roetto A., et al. Transferrin receptor 2 is a component of the erythropoietin receptor complex and is required for efficient erythropoiesis. Blood. 2010;116:5357–5367. doi: 10.1182/blood-2010-04-281360.  [DOI](https://doi.org/10.1182/blood-2010-04-281360) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20826723/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Transferrin%20receptor%202%20is%20a%20component%20of%20the%20erythropoietin%20receptor%20complex%20and%20is%20required%20for%20efficient%20erythropoiesis&author=H.%20Forejtnikov%C3%A0&author=M.%20Vieillevoye&author=Y.%20Zermati&author=M.%20Lambert&author=R.M.%20Pellegrino&volume=116&publication_year=2010&pages=5357-5367&pmid=20826723&doi=10.1182/blood-2010-04-281360&)

82. Regev A., Teichmann S.A., Lander E.S., Amit I., Benoist C., Birney E., Bodenmiller B., Campbell P., Carninci P., Clatworthy M., et al. The Human Cell Atlas. Elife. 2017;6:e27041. doi: 10.7554/eLife.27041.  [DOI](https://doi.org/10.7554/eLife.27041) | [PMC free article](/articles/PMC5762154/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29206104/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Elife&title=The%20Human%20Cell%20Atlas&author=A.%20Regev&author=S.A.%20Teichmann&author=E.S.%20Lander&author=I.%20Amit&author=C.%20Benoist&volume=6&publication_year=2017&pages=e27041&pmid=29206104&doi=10.7554/eLife.27041&)

83. Fan Y., Fu Y.-Y., Chen Z., Hu Y.-Y., Shen J. Gene-gene interaction of erythropoietin gene polymorphisms and diabetic retinopathy in Chinese Han. Exp. Biol. Med. 2016;241:1524–1530. doi: 10.1177/1535370216645210.  [DOI](https://doi.org/10.1177/1535370216645210) | [PMC free article](/articles/PMC4994906/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27190272/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Exp.%20Biol.%20Med.&title=Gene-gene%20interaction%20of%20erythropoietin%20gene%20polymorphisms%20and%20diabetic%20retinopathy%20in%20Chinese%20Han&author=Y.%20Fan&author=Y.-Y.%20Fu&author=Z.%20Chen&author=Y.-Y.%20Hu&author=J.%20Shen&volume=241&publication_year=2016&pages=1524-1530&pmid=27190272&doi=10.1177/1535370216645210&)

84. Kästner A., Grube S., El-Kordi A., Stepniak B., Friedrichs H., Sargin D., Schwitulla J., Begemann M., Giegling I., Miskowiak K.W., et al. Common variants of the genes encoding erythropoietin and its receptor modulate cognitive performance in schizophrenia. Mol. Med. 2012;18:1029–1040. doi: 10.2119/molmed.2012.00190.  [DOI](https://doi.org/10.2119/molmed.2012.00190) | [PMC free article](/articles/PMC3459483/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22669473/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Med.&title=Common%20variants%20of%20the%20genes%20encoding%20erythropoietin%20and%20its%20receptor%20modulate%20cognitive%20performance%20in%20schizophrenia&author=A.%20K%C3%A4stner&author=S.%20Grube&author=A.%20El-Kordi&author=B.%20Stepniak&author=H.%20Friedrichs&volume=18&publication_year=2012&pages=1029-1040&pmid=22669473&doi=10.2119/molmed.2012.00190&)

85. Renner W., Kaiser M., Khuen S., Trummer O., Mangge H., Langsenlehner T. The Erythropoetin rs1617640 Gene Polymorphism Associates with Hemoglobin Levels, Hematocrit and Red Blood Cell Count in Patients with Peripheral Arterial Disease. Genes. 2020;11:1305. doi: 10.3390/genes11111305.  [DOI](https://doi.org/10.3390/genes11111305) | [PMC free article](/articles/PMC7694227/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33158076/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes&title=The%20Erythropoetin%20rs1617640%20Gene%20Polymorphism%20Associates%20with%20Hemoglobin%20Levels,%20Hematocrit%20and%20Red%20Blood%20Cell%20Count%20in%20Patients%20with%20Peripheral%20Arterial%20Disease&author=W.%20Renner&author=M.%20Kaiser&author=S.%20Khuen&author=O.%20Trummer&author=H.%20Mangge&volume=11&publication_year=2020&pages=1305&pmid=33158076&doi=10.3390/genes11111305&)

86. Duarte S., Woll P.S., Buza-Vidas N., Chin D.W.L., Boukarabila H., Luís T.C., Stenson L., Bouriez-Jones T., Ferry H., Mead A.J., et al. Canonical Notch signaling is dispensable for adult steady-state and stress myelo-erythropoiesis. Blood. 2018;131:1712–1719. doi: 10.1182/blood-2017-06-788505.  [DOI](https://doi.org/10.1182/blood-2017-06-788505) | [PMC free article](/articles/PMC5909886/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29339402/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=Canonical%20Notch%20signaling%20is%20dispensable%20for%20adult%20steady-state%20and%20stress%20myelo-erythropoiesis&author=S.%20Duarte&author=P.S.%20Woll&author=N.%20Buza-Vidas&author=D.W.L.%20Chin&author=H.%20Boukarabila&volume=131&publication_year=2018&pages=1712-1719&pmid=29339402&doi=10.1182/blood-2017-06-788505&)

87. Huang C., Yang D., Ye G.W., Powell C.A., Guo P. Vascular Notch Signaling in Stress Hematopoiesis. Front. Cell Dev. Biol. 2021;8:606448. doi: 10.3389/fcell.2020.606448.  [DOI](https://doi.org/10.3389/fcell.2020.606448) | [PMC free article](/articles/PMC7873850/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33585446/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Cell%20Dev.%20Biol.&title=Vascular%20Notch%20Signaling%20in%20Stress%20Hematopoiesis&author=C.%20Huang&author=D.%20Yang&author=G.W.%20Ye&author=C.A.%20Powell&author=P.%20Guo&volume=8&publication_year=2021&pages=606448&pmid=33585446&doi=10.3389/fcell.2020.606448&)

88. Page P. Beyond statistical significance: clinical interpretation of rehabilitation research literature. Int. J. Sports Phys. Ther. 2014;9:726–736.  [PMC free article](/articles/PMC4197528/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25328834/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Sports%20Phys.%20Ther.&title=Beyond%20statistical%20significance:%20clinical%20interpretation%20of%20rehabilitation%20research%20literature&author=P.%20Page&volume=9&publication_year=2014&pages=726-736&pmid=25328834&)

89. Mokry L.E., Ahmad O., Forgetta V., Thanassoulis G., Richards J.B. Mendelian randomisation applied to drug development in cardiovascular disease: a review. J. Med. Genet. 2015;52:71–79. doi: 10.1136/jmedgenet-2014-102438.  [DOI](https://doi.org/10.1136/jmedgenet-2014-102438) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25515070/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%C2%A0Med.%20Genet.&title=Mendelian%20randomisation%20applied%20to%20drug%20development%20in%20cardiovascular%20disease:%20a%20review&author=L.E.%20Mokry&author=O.%20Ahmad&author=V.%20Forgetta&author=G.%20Thanassoulis&author=J.B.%20Richards&volume=52&publication_year=2015&pages=71-79&pmid=25515070&doi=10.1136/jmedgenet-2014-102438&)

90. Sofianopoulou E., Kaptoge S.K., Afzal S., Jiang T., Gill D., Gundersen T.E., Bolton T.R., Allara E., Arnold M.G., Mason A.M., Chung R. Estimating dose-response relationships for vitamin D with coronary heart disease, stroke, and all-cause mortality: observational and Mendelian randomisation analyses. Lancet Diabetes Endocrinol. 2021;9:837–846. doi: 10.1016/S2213-8587(21)00263-1.  [DOI](https://doi.org/10.1016/S2213-8587(21)00263-1) | [PMC free article](/articles/PMC8600124/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34717822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Diabetes%20Endocrinol.&title=Estimating%20dose-response%20relationships%20for%20vitamin%20D%20with%20coronary%20heart%20disease,%20stroke,%20and%20all-cause%20mortality:%20observational%20and%20Mendelian%20randomisation%20analyses&author=E.%20Sofianopoulou&author=S.K.%20Kaptoge&author=S.%20Afzal&author=T.%20Jiang&author=D.%20Gill&volume=9&publication_year=2021&pages=837-846&pmid=34717822&doi=10.1016/S2213-8587(21)00263-1&) | [Retracted](/articles/PMC10834373/)

91. Visscher P.M., Wray N.R., Zhang Q., Sklar P., McCarthy M.I., Brown M.A., Yang J. 10 Years of GWAS Discovery: Biology, Function, and Translation. Am. J. Hum. Genet. 2017;101:5–22. doi: 10.1016/j.ajhg.2017.06.005.  [DOI](https://doi.org/10.1016/j.ajhg.2017.06.005) | [PMC free article](/articles/PMC5501872/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28686856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=10%20Years%20of%20GWAS%20Discovery:%20Biology,%20Function,%20and%20Translation&author=P.M.%20Visscher&author=N.R.%20Wray&author=Q.%20Zhang&author=P.%20Sklar&author=M.I.%20McCarthy&volume=101&publication_year=2017&pages=5-22&pmid=28686856&doi=10.1016/j.ajhg.2017.06.005&)

92. Pulley J.M., Shirey-Rice J.K., Lavieri R.R., Jerome R.N., Zaleski N.M., Aronoff D.M., Bastarache L., Niu X., Holroyd K.J., Roden D.M., et al. Accelerating Precision Drug Development and Drug Repurposing by Leveraging Human Genetics. Assay Drug Dev. Technol. 2017;15:113–119. doi: 10.1089/adt.2016.772.  [DOI](https://doi.org/10.1089/adt.2016.772) | [PMC free article](/articles/PMC5399743/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28379727/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Assay%20Drug%20Dev.%20Technol.&title=Accelerating%20Precision%20Drug%20Development%20and%20Drug%20Repurposing%20by%20Leveraging%20Human%20Genetics&author=J.M.%20Pulley&author=J.K.%20Shirey-Rice&author=R.R.%20Lavieri&author=R.N.%20Jerome&author=N.M.%20Zaleski&volume=15&publication_year=2017&pages=113-119&pmid=28379727&doi=10.1089/adt.2016.772&)

93. Burgess S., Butterworth A., Malarstig A., Thompson S.G. Use of Mendelian randomisation to assess potential benefit of clinical intervention. BMJ. 2012;345:e7325. doi: 10.1136/bmj.e7325.  [DOI](https://doi.org/10.1136/bmj.e7325) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23131671/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Use%20of%20Mendelian%20randomisation%20to%20assess%20potential%20benefit%20of%20clinical%20intervention&author=S.%20Burgess&author=A.%20Butterworth&author=A.%20Malarstig&author=S.G.%20Thompson&volume=345&publication_year=2012&pages=e7325&pmid=23131671&doi=10.1136/bmj.e7325&)
