# The Polymorphic Nuclear Factor NFIB Regulates Hepatic CYP2D6 Expression and Influences Risperidone Metabolism in Psychiatric Patients

## Metadata
**Authors:** Hasan Çağın Lenk, Katharina Klöditz, Inger Johansson, Robert Løvsletten Smith, Marin M Jukić, Espen Molden, Magnus Ingelman‐Sundberg
**Journal:** Clinical Pharmacology and Therapeutics
**Date:** 2022 Mar 20
**DOI:** [10.1002/cpt.2571](https://doi.org/10.1002/cpt.2571)
**PMID:** 35253216
**PMCID:** PMC9314634
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9314634/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9314634/pdf/CPT-111-1165.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9314634/pdf/CPT-111-1165.pdf)

## Abstract

The genetic background for interindividual variability of the polymorphic CYP2D6 enzyme activity remains incompletely understood and the role of NFIB genetic polymorphism for this variability was evaluated in this translational study. We investigated the effect of NFIB expression in vitro using 3D liver spheroids, Huh7 cells, and the influence of the NFIB polymorphism on metabolism of risperidone in patients in vivo. We found that NFIB regulates several important pharmacogenes, including CYP2D6. NFIB inhibited CYP2D6 gene expression in Huh7 cells and NFIB expression in livers was predominantly nuclear and reduced at the mRNA and protein level in carriers of the NFIB rs28379954 T>C allele. Based on 604 risperidone treated patients genotyped for CYP2D6 and NFIB, we found that the rate of risperidone hydroxylation was elevated in NFIB rs28379954 T>C carriers among CYP2D6 normal metabolizers, resulting in a similar rate of drug metabolism to what is observed in CYP2D6 ultrarapid metabolizers, with no such effect observed in CYP2D6 poor metabolizers lacking functional enzyme. The results indicate that NFIB constitutes a novel nuclear factor in the regulation of cytochrome P450 genes, and that its polymorphism is a predictor for the rate of CYP2D6 dependent drug metabolism in vivo.

☑ There is a high unexplained interindividual variability in CYP2D6 activity within subjects carrying the fully functional *CYP2D6*1* and *CYP2D6*2* alleles.

☑ We investigated whether the polymorphic *NFIB* gene expression influenced CYP2D6 activity and expression using human liver specimens and a patient cohort (*n* = 604) geno‐ and phenotyped for CYP2D6.

☑ We found that CYP2D6 dependent hydroxylation of risperidone was highly elevated in CYP2D6 normal metabolizers carrying the mutated *NFIB* C variant. The *NFIB* C allele had no effect on risperidone hydroxylation in poor metabolizers supporting a regulatory of role of NFIB on CYP2D6 metabolism. NFIB expression was shown to inhibit *CYP2D6* reporter gene expression in Huh7 cells and hepatocytes from mutated NFIB donors expressed substantially less NFIB mRNA and protein in the nuclei. Furthermore, we found that the NFIB polymorphism influences CYP2D6 expression implicating that the polymorphic locus remote from the *CYP2D6* gene has to be considered for pre‐emptive genotyping.

☑ The data indicate that the NFIB rs28379954 T>C polymorphism is an important determinant for the rate of CYP2D6 dependent drug metabolism. By implementing this polymorphism into algorithms for genetic contribution to CYP2D6 dependent drug metabolism *in vivo*, the precision in predicting individual CYP2D6 phenotypes will likely be improved.

Cytochrome P450 2D6 (CYP2D6) is a polymorphic enzyme involved in the metabolism of ~ 25% of all clinically used drugs,[1](#cpt2571-bib-0001) including many antipsychotics.[2](#cpt2571-bib-0002) The CYP2D6 metabolic capacity *in vivo* is predominantly genetically determined. Patients can be classified into CYP2D6 ultrarapid metabolizers (UMs), normal metabolizers (NMs), intermediate metabolizers (IMs), and poor metabolizers (PMs).[3](#cpt2571-bib-0003) The CYP2D6 IM group is heterogeneous and can be divided into two subcategories, based on having relatively higher (CYP2D6 IM+) and lower (CYP2D6 IM‐) enzyme activity.[4](#cpt2571-bib-0004)

Knowledge of the *CYP2D6* genotype can be utilized for prediction of drug metabolism *in vivo*.[5](#cpt2571-bib-0005) However, the interindividual variability in CYP2D6 activity within specific CYP2D6 metabolizer categories, based on the current knowledge of *CYP2D6* gene variants, is unexpectedly high.[6](#cpt2571-bib-0006) Thus, even the most refined predictive algorithms based on to‐date validated CYP2D6 genetic biomarkers can only explain 40–70% of interindividual variability in CYP2D6 activity observed *in vivo*.[3](#cpt2571-bib-0003), [7](#cpt2571-bib-0007) This indicates the existence of genetic variations that have not yet been considered to influence CYP2D6 expression and/or activity; in fact, such variations are not necessarily constrained to the *CYP2D* locus and may reside outside of it, within certain regulatory elements.

Nuclear factor 1B (NFIB) is involved in embryonic development of many organ systems[8](#cpt2571-bib-0008), [9](#cpt2571-bib-0009) and in the control of tumor growth.[10](#cpt2571-bib-0010) In clozapine‐treated patients, we found that the heterozygous *NFIB* rs28379954 T>C (allele NFIB TC) carriers exhibit significantly reduced plasma levels of clozapine, compared with subjects homozygous for the *NFIB* T variant (*NFIB* TT).[11](#cpt2571-bib-0011) However, the mechanism by which NFIB and *NFIB* genetic polymorphisms may influence the clozapine exposure is unknown. Clozapine is mainly metabolized by CYP1A2, CYP3A4, FMO3, and UGT1A4.[12](#cpt2571-bib-0012) We investigated whether the *NFIB* T>C polymorphism might influence the expression of hepatic absorption, distribution, metabolism, and excretion (ADME) genes, in particular CYP2D6. For the latter purpose, we performed (1) an *in vivo* study on a naturalistic cohort of 604 patients carrying different *NFIB* genotypes to examine the effect of *NFIB* polymorphism on risperidone hydroxylation, which is catalyzed by CYP2D6; and (2) an *in vitro* study on 3D hepatocyte spheroids, hepatoma cells using *CYP2D6* gene reporters and flash frozen liver pieces to examine the effect of the *NFIB* polymorphism on the expression of CYP2D6.

## METHODS

### Study population

The effect of the *NFIB* polymorphism on CYP2D6 metabolism was investigated in a patient cohort treated with the CYP2D6 substrate risperidone. Subject data was retrospectively collected from the routine therapeutic drug monitoring (TDM) database at the Center for Psychopharmacology, Diakonhjemmet Hospital, Oslo, Norway. Inclusion of the subjects and their measurements are determined further by the following criteria: (1) oral treatment with risperidone; (2) history of at least one detectable risperidone (and 9‐hydroxyrisperidone) serum concentration measurement between January 1, 2005, and Aug 13. 2020; (3) CYP2D6 genotype available from historic analyses; and (4) blood samples available in biobank for analysis of *NFIB rs28379954 T>C* polymorphism genotyping. Exclusion criteria for previously included measurements were: (1) comedication with CYP2D6 enzyme inhibitors bupropion, fluoxetine, and paroxetine; or (2) potent antiepileptic enzyme inducers; (3) unavailable information on prescribed dose, time difference between last dosage, and blood sampling for serum concentration analysis or blood sampling withdrawn outside the period of 10–30 hours; and (4) age above 65 years at the time of blood sampling. Information on risperidone prescription and blood sampling for TDM, comedications, sex, and age were obtained from the requisition forms.

All patients were Norwegian inhabitants, confirmed by a unique social security number, and most were assumed to be of Scandinavian ancestry based on the ethnic composition of the general Norwegian population.

### Serum concentration analysis of risperidone and 9‐hydroxyrisperidone

As previously described,[13](#cpt2571-bib-0013) the analytical assays were validated and certified for routine clinical TDM. During the whole study period, the assays were periodically adjusted, however, all modifications were cross‐validated and therefore considered fully equivalent. Concisely, the most recent methods were performed with initial sample purification by protein precipitation; the supernatant injection into an ultrahigh‐performance liquid chromatograph with tandem mass spectrometric detection. All the calibration curves were linear (*R* ^2^ > 0.99) in validated ranges: risperidone, 1–200 nmol/L; 9‐hydroxyrisperidone, 2.5‐300 nmol/L. Imprecision and inaccuracy parameters of the assays were lower than 5%.

### Genotyping

#### CYP2D6

Validated and certified TaqMan based real‐time polymerase chain reaction assays and copy number analyses developed for routine pharmacogenetic testing at the Center for Psychopharmacology were used for analysis of *CYP2D6* variant alleles. The genotyping panel for *CYP2D6* included analyses for gene variants and copy numbers encompassing the lack‐of‐function variants (*Nonf*) *CYP2D6*3* (rs35742686), *CYP2D6*4* (rs3892097), *CYP2D6*5* (whole gene deletion), *CYP2D6*6* (rs5030655), the decreased function (*Decr*) variants *CYP2D6*9* (rs5030656), *CYP2D6*10* (rs1065852), *CYP2D6*41* (rs28371725), and duplicated functional *CYP2D6* alleles. If none of the *CYP2D6* variant alleles included in the pharmacogenetic panel were detected, the allele was considered as normal *CYP2D6* allele (*Norm*; i.e. *CYP2D6*1* or *CYP2D6*2)*.

#### NFIB rs28379954 T>C analysis

Genotyping of the *NFIB rs28379954 T>C* variant was carried out using the same platform as for CYP2D6 genotyping. TaqMan single‐nucleotide polymorphism genotyping assays and TaqMan genotyping polymerase chain reaction (PCR) master mix (Thermo Fisher Scientific, Waltham, MA, USA) were used for *rs28379954 T>C* (C_59359617_10) genotyping. The PCR assays were run on QuantStudio12K Flex Real‐Time PCR System (Applied Biosystem Life Technologies, Thermo Fischer Scientific) or a 7500 Fast Real‐Time PCR system (Applied Biosystems).

### Gene expression analysis in liver spheroids

Cryopreserved human hepatocytes were obtained from Bioreclamation IVT (BioIVT, Westbury, NY, USA), KaLy‐Cell (Plobsheim, France), and Lonza (Basel, Switzerland), and 3D human hepatocyte spheroids prepared as described.[14](#cpt2571-bib-0014) The donors used were: SYF, GID, HJK (BioIVT), HUM201221 (Lonza), and S1506T (KaLy Cell). Total RNA was isolated from 48 pooled human hepatic spheroids using the QIAzol Lysis Reagent (Qiagen, USA) and the RNA concentration was measured at a Qubit 4 fluorometer (Thermo Fisher Scientific). Following the RNA extraction, cDNA synthesis was performed using the SuperScript III reverse transcriptase (Thermo Fisher Scientific) and gene expression was analyzed by reverse transcription polymerase chain reaction (RT‐qPCR) using a TaqMan Universal mix and TaqMan probes (obtained from Thermo Fisher; **Table ** [S1](#cpt2571-sup-0001)) on a 7500 Fast Real‐Time PCR system (Applied Biosystems). Data analysis was performed using the 2^ΔΔCt^ method and TATA‐binding protein (TBP) as the housekeeping gene.

### NFIB gene variant and protein analyses in human livers

Total RNA was extracted from deeply frozen liver samples in our liver biobank,[15](#cpt2571-bib-0015) as described above. Primers were designed to target individual exons of the *NFIB* variants and ordered from Thermo Fisher Scientific (primer sequences are listed in **Table ** [S2](#cpt2571-sup-0001)). The PCR was performed using the Phusion Green Hot Start II High‐Fidelity PCR Master Mix where the annealing temperature was set to 60°C and the elongation temperature to 72°C for 35 PCR cycles. The samples were loaded to 2% agarose gels in TAE buffer containing GelRed Nucleic Acid gel‐stain (Biotium, Fremont, CA) and separated by electrophoresis. The amplified fragments were then detected using the ChemiDoc Imaging Systems (Bio‐Rad, Hercules, CA, USA).

For quantification of the total NFIB mRNA expression—including all different *NFIB* variants—we used a TaqMan assay (Hs00232149_m1; Thermo Fisher) on a 7500 Fast Real‐Time PCR system (Applied Biosystems) using the conditions as recommended by the manufacturer. This TaqMan assay detects all the *NFIB* variants. TBP was used as a housekeeping gene.

NFIB protein expression analysis was performed using fluorescent immunohistochemistry and the NFIB antibody (#HPA003956; Sigma Aldrich) at 1:1,000 dilution. Liver sections of 8‐µm thickness were obtained from by NX70 cryostat (Thermo Fisher Scientific) and transferred to histological slides. The samples were fixated by incubation for 20 minutes at RT with 4% PFA followed by three washes with phosphate‐buffered saline (PBS). Permeabilization was obtained by incubating the samples for 2 minutes in 0.1% sodium citrate and 0.1% Triton. The samples were then blocked for 2 hours in PBTA (5% bovine serum albumin, 0.25% Triton X‐100, and 0.1% sodium azide in PBS) and stained with antibodies according to standard protocols. Cover slips were placed on the glass slides using mounting medium with DAPI (ProLong Gold antifade reagent with DAPI; Thermo Fisher Scientific) and the individual liver sections were imaged at a Zeiss LSM880 confocal microscope at a 20× and 40× magnification. At least six images were obtained from each liver donor. Analysis was performed with ImageJ using a macro developed for this purpose, which is available upon request.

### Western Blot analysis

Huh7 cells were harvested and lysed in radioimmunoprecipitation assay buffer containing Roche complete protease inhibitor cocktail (Sigma‐Aldrich). Protein concentration was determined using DC protein Assay (Bio‐Rad) and 0.62–25 µg of protein was loaded on 10% sodium dodecyl sulfate‐polyacrylamide gel electrophoresis gel (Bio‐Rad) and blotted to an Amersham Protran Supported nitrocellulose membrane. NFIB protein was detected with NF1B2 antibody ([EPR14122](https://www.ncbi.nlm.nih.gov/protein/EPR14122)) 1:1000 from Abcam (Cambridge, UK) and goat antirabbit‐HRP (DAKO, Agilent, CA) 1:5000 and visualized using SuperSignal West Pico PLUS Chemiluminescent Substrate (Thermo Fisher Scientific) and ChemiDoc MP Imaging System (Bio‐Rad Laboratories, Hercules, CA).

### Analyses of expression of CYP2D6 reporter constructs

#### Huh7 cell culture

The human hepatoma Huh7 cell line (Human Science Research Resources Bank, Tokyo, Japan) was cultured in Dulbecco’s modified Eagle’s medium, 4.5 g glucose/L, supplemented with 10% fetal bovine serum, 100 IU/mL and 100 μg/mL streptomycin (Invitrogen, Carlsbad, CA) at 37°C in 5% carbon dioxide.

#### Plasmid constructs

Reporter constructs containing three different lengths of the *CYP2D6* 5′‐flanking regulatory regions were generated by PCR using Platinum SuperFi II DNA polymerase (Invitrogen, Waltham, MA). Sequences of primer used for cloning of the CYP2D6 5′‐flanking regions are listed in **Table ** [S3](#cpt2571-sup-0001). Genomic DNA from subjects homozygous for *CYP2D6*1* was used as template. Different length of the 5′‐flanking regions (i.e., −1915/+88 to −413/+88), were inserted between the *Nhe*I*/Hind*III sites of the pGL4.10 basic vector (Promega, Madison, WI). The region −172/+88, containing the HNF4α binding site, were inserted between the *Kpn*I/*Xho*I sites. The DNA sequences were verified.

#### Transient transfections

Huh7 cells were seeded on 48‐well plates and used when confluent for at least 3 days. The cells were transfected using Lipofectamine 3000 Transfection Reagent (Invitrogen) and 0.25 μg reporter plasmid, 0.6 ng pRL‐CMV control plasmid and 0.25 μg pCMV3‐NFIB expression plasmid (from Sino Biological, Beijing, China) or corresponding empty plasmid (mock) according to the manufacturer’s recommendations. Luciferase activities were determined 65–70 hours after transfection using the Dual Luciferase Reporter Assay (Promega) and a SpectraMax iD3 Microplate Reader (Molecular Devices, San Jose, CA, USA).

### Outcomes and statistics

The primary outcomes of the study were two‐fold: (1) to investigate the potential effect of *NFIB* rs28379954 T>C polymorphism on CYP2D6 metabolism *in vivo* using risperidone as a biomarker; and (2) in light of a detected effect in (1), to evaluate the functional effects of the NFIB mediated *CYP2D6* regulation in CYP2D6 reporter systems and of the *NFIB* polymorphism in flash frozen liver pieces.

The metabolite‐to‐parent ratio (MPR) of risperidone has previously been validated as an excellent CYP2D6 phenotypic marker,[16](#cpt2571-bib-0016) which is calculated by dividing the 9‐hydroxyrisperidone serum concentration (nmol/L) with the risperidone serum concentration (nmol/L). When evaluating the effect of the *NFIB rs28379954 T>C* polymorphism in relation to CYP2D6‐metabolizer phenotype, CYP2D6 genotype/phenotype translations were performed as previously published[3](#cpt2571-bib-0003); in particular, *CYP2D6Nonf/Nonf* subjects were PMs, CYP *CYP2D6Decr/Nonf* and *CYP2D6Decr/Decr* were IMs, *CYP2D6Norm/Nonf*, *CYP2D6Norm/Decr* were IM+, *CYP2D6Norm/Norm* were NMs, and *CYP2D6Norm/Norm/Norm* were UMs.

Linear mixed model analysis with random intercepts was performed for estimation of risperidone metabolism and exposure to account for multiple measurements per subject with the following variables: (1) MPR (or CD) as dependent variable; (2) *NFIB* genotype, age, sex, and time difference between last dose and blood sampling as fixed effects; and (3) unique anonymized patient number as random effect. Because of the skewed nature of MPR and CD variables, they were natural log transformed prior to linear mixed model analysis. Estimated marginal means and 95% confidence interval (CI) ranges are back‐transformed for representation of the results.

Statistical analyses of the patient data were performed using R version 4.1.1. Statistical analyses of the hepatic NFIB protein expression data (**Figure ** [4](#cpt2571-fig-0004)) were performed using GraphPad Prism 5 software applying two‐tailed paired *t*‐test and Bonferroni correction. For all analyses, a *P* value < 0.05 was defined as statistically significant. Statistical analysis of the qPCR data was performed using GraphPad Prism 5 software applying two‐tailed paired *t*‐test and Bonferroni correction. Statistical analysis of the immunohistochemical (IHC) data was performed using GraphPad Prism 5 software applying Student’s *t*‐test or Mann–Whitney test. Violin plots showing the mean NFIB intensity of individual nuclei were generated using the GraphPad Prism 9 software.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/20a9/9314634/b1c341662f7a/CPT-111-1165-g003.jpg)

(a) NFIB staining in liver sections from donors of the NFIB TT (n = 10), TC (n = 12) and CC (n = 1) genotypes. Representative images from all donors investigates as well as the results obtained from the antibody control staining excluding the primary antibody are shown. NFIB is visualized in red and nuclei are co‐stained with DAPI. (b) Quantification of the amount of NFIB in the human livers of the NFIB TT (wt), NFIB TC (heterozygous) and NFIB CC (homozygous) livers as visualized in violin plots. Nonparametric t‐test/Mann–Whitney test was performed for statistical analysis. ***, P < 0.0001.

## RESULTS

### Investigation of NFIB dependent regulation of hepatic ADME gene expression

Our previous results showed a significant association between the *NFIB* rs28379954 T>C polymorphism and the clozapine exposure in psychiatric patients; however, the cause of this phenomenon has not been determined.[11](#cpt2571-bib-0011) Initially, we evaluated whether and to which extent NFIB could influence the expression of 21 different hepatic genes by using 3D hepatocyte spheroids, having an *in vivo* like phenotype,[14](#cpt2571-bib-0014) prepared from five independent liver donors. As evident from the **Figure ** [S1](#cpt2571-sup-0001), *NFIB* silencing highly decreased the expression of *NFIB* and *GSTP1*. In addition, the expression of the *CYP1A2*, *CYP2D6*, *CYP2C8*, and *CYP2B6* genes were also reduced to a substantial extent by the *NFIB* silencing, whereas less prominent, but still significant decrease of gene expression was observed for the phase II enzyme genes *UGT1A1*, *UGT1A3*, and *UGT1A4*. By contrast, the expression of *CYP2C9*, *CYP2C19*, *CYP3A4*, *UGT2B10*, *CRYZ*, *ABCB1*, *ABCC1*, *ABCG2*, *SLC22A1*, *SLC22A3*, and *FMO3* was not significantly affected by the *NFIB* silencing. *NMT2* was the only gene analyzed that exhibited significantly increased mRNA expression upon silencing of *NFIB* in this system. By contrast, NFIB overexpression in Huh7 cells caused inhibited expression of CYP2D6 reporter constructs (see below). Our data hitherto indicates a more general role of NFIB in the control of specific hepatic ADME gene expression and we hypothesized that NFIB could also influence CYP2D6 expression *in vivo*.

### CYP2D6‐mediated risperidone metabolism is increased in NFIB rs28379954 T>C variant carriers

We analyzed the CYP2D6‐catalyzed 9‐hydroxyrisperidone‐to‐risperidone metabolic transformation (metabolite‐to‐parent ratio) between subjects with the *NFIB* TT and TC genotypes derived from the routine TDM database based on a psychiatric patient population of 604 subjects. The application of the inclusion and exclusion criteria is schematically represented in **Figure ** [1](#cpt2571-fig-0001), whereas the population characteristics are presented in **Table ** [S4](#cpt2571-sup-0001).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/20a9/9314634/39be66fd8ae1/CPT-111-1165-g001.jpg)

Overall scheme for the selection of patients in the clinical study. Out of 841 patients, 605 were found to be eligible for analyses.

The frequency of the *NFIB* C (minor allele) variant was 5.2% (**Table ** [S5](#cpt2571-sup-0001)). The comparisons of 9‐OH‐risperidone‐to‐risperidone ratios revealed that among CYP2D6 NM, carriers of the *NFIB* TC genotype exhibited significantly higher CYP2D6 metabolic capacity than subjects of the *NFIB* TT genotype (*P* = 0.007; **Figure ** [2](#cpt2571-fig-0002); **Table ** [1](#cpt2571-tbl-0001)). A trend toward increased MPR was observed in carriers of the *NFIB* C allele in the CYP2D6 IM+ cohort subset (*P* = 0.06; **Figure ** [2](#cpt2571-fig-0002)), whereas no significant difference was observed in the CYP2D6 IM‐ and CYP2D6 PM groups (**Figure ** [2](#cpt2571-fig-0002)). Importantly, the quantitative effects of *NFIB* genotype on risperidone metabolism increased by increasing baseline CYP2D6 phenotype with a 2.14‐fold higher metabolic ratio in patients of the *NFIB* TC genotype as compared with those of the *NFIB* TT genotype among CYP2D6 NMs (**Table ** [1](#cpt2571-tbl-0001)). The number of CYP2D6 UM individuals was insufficient for a meaningful comparison. The results nevertheless suggest a substantial effect of the *NFIB* polymorphism on the rate of CYP2D6 metabolism in carriers of functional *CYP2D6* alleles, but no influence of the polymorphism among CYP2D6 PMs lacking functional CYP2D6 enzyme, indicating that the effect of *NFIB* C on risperidone metabolism is linked specifically by increased CYP2D6 enzyme expression. An overview of the clinical importance of the *NFIB* and *CYP2D6* polymorphisms in different part of the world is provided in **Table ** [2](#cpt2571-tbl-0002).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/20a9/9314634/307b090ab091/CPT-111-1165-g002.jpg)

Influence of the NFIB polymorphism on the metabolite‐to‐parent ratio (MPR) of 9‐hydroxyrisperidone to risperidone in patients of different CYP2D6 and NFIB genotypes. Each point represents median metabolic ratios per patient from several measurements. Central points represent estimated marginal means and tips of the bars represent 95% confidence interval (CI) from linear mixed model analysis. The P values for pairwise comparisons and number of patients are added for each subgroup. Linear mixed model analysis was used for pairwise comparisons of estimated means in natural logarithm transformed variables and adjusted for age, sex, and time difference of last dose and blood sampling. Estimated marginal means are converted back to linear scale for presentation of results. As seen the NFIB C allele is highly linked to increased risperidone hydroxylation in carriers of functional CYP2D6 enzyme, whereas no effect is seen in subjects lacking the CYP2D6 enzyme (CYP2D6 PM). IM, intermediate metabolizer; NM, normal metabolizer; PM, poor metabolizer; UM, ultrarapid metabolizer.

### Table 1.

| CYP2D6 subgroup | NFIB rs28379954 genotype | Subjects, n (measurements) | MPR (95% CI) | Fold change (95% CI)a | P valueb | CD (95% CI) | Fold change (95% CI)a | P valueb |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| UM | Overall | 17 (29) | 26.35 (8.48, 81.84) | – | – | 0.56 (0.15, 2.18) | – | – |
| TC | 3 (4) | 33.11 (5.61, 195.32) | 1.58 (0.30, 8.23) | 0.56 | 0.54 (0.07, 4.53) | 0.93 (0.13, 6.71) | 0.94 |  |
| TT | 14 (25) | 20.96 (8.66, 50.74) | – | – | 0.58 (0.20, 1.67) | – | – |  |
| NM | TC | 19 (76) | 21.21 (12.49, 36.03) | 2.14 (1.23, 3.70) | 0.007 | 0.70 (0.40, 1.21) | 0.51 (0.29, 0.90) | 0.02 |
| TT | 243 (680) | 9.93 (8.55, 11.54) | – | – | 1.37 (1.17, 1.60) | – | – |  |
| IM+ | CC/TC | 30 (117) | 6.63 (4.37, 10.05) | 1.53 (0.98, 2.40) | 0.06 | 1.95 (1.32, 2.89) | 0.70 (0.46, 1.07) | 0.10 |
| TT | 214 (592) | 4.32 (3.68, 5.07) | – | – | 2.79 (2.41, 3.24) | – | – |  |
| IM‐ | CC/TC | 4 (14) | 1.81 (0.79, 4.14) | 1.42 (0.59, 3.38) | 0.42 | 7.23 (2.92, 17.89) | 0.98 (0.38, 2.54) | 0.96 |
| TT | 37 (136) | 1.28 (0.98, 1.68) | – | – | 7.40 (5.50, 9.97) | – | – |  |
| PM | TC | 5 (25) | 0.25 (0.17, 0.36) | 0.75 (0.50, 1.12) | 0.16 | 19.65 (11.48, 33.66) | 1.09 (0.61, 1.95) | 0.77 |
| TT | 35 (110) | 0.33 (0.28, 0.38) | – | – | 18.03 (14.66, 22.18) | – | – |  |

Table 1 Caption: Estimated MPR of risperidone and 9‐hydroxyrisperidone and CD of risperidone in NFIB rs28379954 allele carriers within CYP2D6 subgroups

### Table 2.

|   | NFIB rs28379954 T>C |  |  |  | CYP2D6 allele frequencies (%) Zhou et al. (2017) |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 1000 G project |  | gnomADg |  |  |  |  |  |  |  |  |  |  |  |
| Allele freq (%) | No of subjects | Allele freq (%) | No of subjects | *1 | *2 | *1/2xn | *3 | *4 | *5 | *10 | *17 | *41 |  |
| Africans | 0.2 | 661 | 1 | 20,941 | 9.3 | 26.7 | 9.3 | 0.3 | 11.9 | 4 | 3.2 | 19.7 | 3 |
| Americans | 1.2 | 349 | 2.6 | 6,788 | 40.2 | 32.7 | 1 | 0.3 | 15.7 | 3 | 0 | 0.7 | 3.5 |
| East Asian | 0 | 504 | 0.032 | 1,565 | 13.6 | 14 | 2 | 0 | 0.4 | 6.5 | 58.7 | 0 | 3 |
| European | 5.7 | 503 | 5.1 | 32,165 | 33.1 | 34.3 | 2.3 | 4.1 | 15.5 | 3 | 0.2 | 0 | 3 |
| South Asian | 3.3 | 489 | 4.3 | 1,512 | 25.8 | 36.2 | 1.5 | 0.1 | 11.6 | 2 | 6.5 | 0.1 | 13 |

Table 2 Caption: Global allele frequencies (%) of the NFIB C allele and different CYP2D6 alleles primarily affecting the rate of CYP2D6 mediated drug metabolism

### NFIB modulates CYP2D6 expression by interfering with regulatory elements located between ‐172 and ‐1409 bp upstream in the CYP2D6 gene

To investigate whether NFIB can interfere with regulatory elements upstream in the *CYP2D6* gene, we measured the effect of *NFIB* on *CYP2D6* reporter gene expression by using luciferase reporter constructs. Accordingly, an NFIB cDNA expression vector was transfected into confluent Huh7 cells and the effect of NFIB expression on the rate of expression of 7 different luciferase reporter constructs encompassing different parts of the *CYP2D6* 5′‐upstream sequences between ‐172 bp and ‐1915 bp upstream. We found that heterologous expression of the NFIB caused profound inhibition of the constructs containing sequences longer than 500 bp. No inhibition was seen using the 500 bp construct and a much less inhibition was present using the ‐172 bp reporter (**Figure ** [3](#cpt2571-fig-0003)). This indicates that the NFIB protein interacts with nuclear proteins binding to the *CYP2D6* regulatory elements between ‐413 bp and ‐534 bp 5′‐upstream.

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/20a9/9314634/c3b686866ef6/CPT-111-1165-g004.jpg)

(a) Analyses of NFIB expression in Huh7 cells (triplicates; unfilled square indicates control cells). NFIB protein was only detected following heterologous expression using an NFIB expression plasmid. The amount of protein applied were: Huh7, 25 µg, Huh7‐NFIB 0.625 µg, and NFIB control HEK293‐NFIB. (b) CYP2D6 gene constructs used. (c) Effect of heterologous NFIB expression on the expression of seven different CYP2D6 5′‐upstream reporter constructs. A high inhibitory effect was observed using a 0.62 kb upstream construct but not in the 0.5 kb construct indicating an NFIB responsive element between 500 bp and 620 bp upstream of transcription start site. The data shown are based on 3–8 separate experiments.

### Expression of NFIB protein and mRNA is reduced in livers of NFIB TC donors

In order to evaluate the functional effect of the *NFIB* polymorphism, we determined NFIB expression in livers from donors of the different *NFIB* genotypes at the mRNA and protein level. RNA was extracted from 9 *NFIB* TT livers, 11 *NFIB* TC livers, and one *NFIB* CC liver. Different primer pairs were used to amplify mRNA fragments encoding different exons specific for different variants of the *NFIB* transcripts as shown in **Table ** [S2](#cpt2571-sup-0001). As shown in **Figure ** [S2](#cpt2571-sup-0001) and **Table ** [S2](#cpt2571-sup-0001), we did not find any significant differences in the different spliced mRNA variants between the *NFIB* genotypes but a significantly reduced total NFIB mRNA expression in *NFIB* TC and CC livers as compared with *NFIB* TT was revealed by TaqMan quantification. Investigation of the NFIB protein expression in liver sections from donors of the different *NFIB* genotypes by fluorescent IHC, revealed that the NFIB protein expression was predominantly located within the nuclei of the hepatocytes and that livers having the *NFIB* TT genotype had a substantially higher nuclear NFIB staining compared with the livers with *NFIB* TC and CC genotypes (**Figure ** [4](#cpt2571-fig-0004)). In addition, the proportion of nuclei with high NFIB expression was significantly lower in *NFIB* TC and CC carriers, as compared with those having a *NFIB* TT genotype (**Figure ** [S3](#cpt2571-sup-0001)). In summary, the data indicate that the NFIB mRNA and protein levels are reduced in *NFIB* TC and CC livers.

### Analyses of linkage equilibrium of the rs28379954 T>C mutation

The basis behind the effect seen in *NFIB* T>C carriers remains unknown. We investigated to which extent rs28379954 T>C genetic variant might be in linkage disequilibrium to variants in neighboring genes of potential importance. Indeed Palleja *et al*.[17](#cpt2571-bib-0017) suggests that the LD block, which includes rs28379954, extends from 14038343 to 14343279 on chr9 (hg19; i.e., it encompasses the whole *NFIB* gene with some flanks). However, these data are based on D′, not on r2. Thus, if one considers the *r* ^2^ metric instead and HapMap data for European population, then only 5 SNPs in this LD block show *r* ^2^ values above 0.2, and none of them has *r* ^2^ above 0.5 using HaploReg, SNiPA, and LDlink. Among the recent genetic variants described ([https://www.ensembl.org/](https://www.ensembl.org/)), three were found in *NFIB* introns, namely rs12156582 *r* ^2^ = 0.7; rs10481501; and *r* ^2^ = 0.56 and rs10961406 *r* ^2^ = 0.56, but they are all very rare and cannot be in linkage disequilibrium with the *NFIB* C haplotype, indicating the absence of an obvious linkage disequilibrium to other functionally active genetic variants.

## DISCUSSION

In the present investigation, we show data indicating that (1) the *NFIB* genotype is a determinant of CYP2D6 metabolism in risperidone‐treated patients; (2) that NFIB acts as a nuclear factor inhibiting CYP2D6 expression; and (3) that the *NFIB* C allele associates to lower NFIB protein levels in hepatocyte nuclei, implicating a reduced inhibition of CYP2D6 expression among such carriers. Incorporation of the *NFIB* polymorphism into algorithms predictive of CYP2D6 metabolism may contribute to a better personalization of treatment with CYP2D6 substrates, such as antipsychotics, antidepressants, antiarrhythmics, lipophilic β‐adrenoceptor blockers, opioids, and tamoxifen (cf. **Table ** [2](#cpt2571-tbl-0002)).

The allele frequency of the *NFIB* C variant, which is associated with higher CYP2D6 metabolic capacity, varies between different populations and is highest in Europeans (~ 6%; [https://www.ensembl.org/](https://www.ensembl.org/)). The frequency of *CYP2D6* active gene duplication in Northern Europe is ~ 3%[18](#cpt2571-bib-0018) and the subjects carrying this duplication are considered to constitute the CYP2D6 UM phenotype. However, there are many rapid metabolizers based on phenotype which do not carry this duplication. Interestingly, as demonstrated by the retrospective cohort study herein, the subpopulation having *NFIB* T>C and CYP2D6 NM has ultrarapid risperidone metabolism similar in magnitude to the conventional CYP2D6 UM phenotypes carrying duplicated active *CYP2D6* genes (**Figure ** [2](#cpt2571-fig-0002)). Therefore, the *NFIB* C allele holds a potential to explain a substantial proportion of the subjects which are homozygous carriers of CYP2D6 NM alleles but exhibit ultrarapid drug metabolism instead of the expected normal CYP2D6 metabolizer phenotype. Although these results still require further validation with different CYP2D6 substrates and/or independent cohorts, this observation argues in favor for that CYP2D6 NM patients having the *NFIB* TC genotype should be considered UMs. Therefore, in case that this hypothesis will be confirmed, the relevance of the regulation of CYP2D6 by NFIB and routine rs28379954 T>C genotyping will become very apparent in clinical medicine.

In addition, the results provide mechanistic insight into the regulation of CYP2D6 metabolic capacity by NFIB measured by risperidone MPR. NFIB was able to suppress *CYP2D6* reporter gene expression in Huh7 cells as revealed by expression of reporter constructs encompassing −1,915 bp to −413 bp of the 5′ upstream CYP2D6 sequences. Thus, NFIB appears to inhibit CYP2D6 expression by clustering with proteins interfering with an element −413 bp to −534 bp upstream in the *CYP2D6* gene. However, the effects of *NFIB* silencing during more chronic conditions of 17 days as used in the 3D liver spheroid model, are not compatible with the 2‐day data obtained in the Huh7 cells. This might be explained by the long‐term effects of NFIB in the spheroids influencing the expression of many other nuclear proteins involved in the regulation of CYP2D6 and other hepatic genes. Indeed, NFI proteins have been implicated in both activation and repression of gene promotors utilizing different types of mechanisms, depending on cellular systems, tissues, etc.[19](#cpt2571-bib-0019) The exact mechanisms behind and nuclear proteins involved in addition to NFIB in the control of CYP2D6 expression under *in vivo* conditions, thus remain to be identified.

The occurrence of *NFIB* C allele correlated to a substantial decrease in the level of NFIB mRNA expression, as well as a significant reduction of nuclear NFIB protein expression and, in particular, the number of hepatocyte nuclei which express high level of NFIB (**Figures ** [4](#cpt2571-fig-0004), [S3](#cpt2571-sup-0001)). Hypothetically, there is therefore a lower potential for the suppression of hepatic CYP2D6 expression in the livers carrying the *NFIB* C variant, which is a possible explanation of the increase in CYP2D6 metabolic capacity *in vivo*, observed in *NFIB* T>C carriers compared to subjects lacking this variant (*NFIB* TT). In the CYP2D6 PM patients lacking active CYP2D6 enzyme, the *NFIB* C allele did not affect the risperidone metabolic ratio, indeed supporting that the NFIB effect on risperidone hydroxylation is mediated by interfering with the CYP2D6 expression and subsequently with CYP2D6‐mediated metabolism specifically. The plausible effect of the *NFIB* C allele is thus to decrease the extent of NFIB mediated suppression of hepatic CYP2D6 expression. The mechanism is still unknown and probably involves NFIB mediated regulation of regulatory proteins. Their identities remain to be established and the absence of a true linkage equilibrium of rs28379954 to other genetic variants provides no information in this respect. In conclusion, our data indicate that the polymorphic NFIB constitutes a novel nuclear factor in the regulation of CYP2D6, which can be implemented into algorithms for genetic contribution to the rate of CYP2D6‐mediated drug metabolism *in vivo* and predictions of individual dose requirements for many medications, including those used in psychiatry. Any role of the *NFIB* polymorphism in the control of, for example, *CYP1A2*, *CYP2B6*, and *CYP2C8* gene expression as here indicated (**Figure ** [S1](#cpt2571-sup-0001)), requires further investigations.

### Limitations

The current study was based on patient data collected in a real‐life setting. This implies certain potential limitations associated with, for example, treatment adherence, access to all comedications, imprecise information by the physicians requesting risperidone TDM in terms of prescribed dose and sampling, etc. However, by including a large patient population and due to randomness of mentioned confounding factors, these limitations are largely bypassed. Nonetheless, it is important that additional studies on other CYP2D6 substrates are performed to investigate if NFIB’s effect on CYP2D6 metabolism for the purpose of biomarker validation. In addition, the mechanism of action by which NFIB interferes with the regulation of CYP2D6 expression needs to be elucidated in more detail. The opposite effects of NFIB more acute action on CYP2D6 expression in Huh7 cells as compared with the more long‐term conditions in 3D liver spheroids in particular utilizing overexpressed NFIB and CYP2D6 gene reporter constructs, requires further investigation.

## FUNDING

This work was funded by the South‐Eastern Norway Regional Health Authority (grant number 2020019), Diakonhjemmet Hospital found for research, innovation, and professional development, the Swedish Cancer Society (grant agreement 17 0599), the European Research Council (ERC)–Advanced Grant (AdG) project HEPASPHER (grant agreement 742020), The Swedish Research Council (grant 2021‐02732) and the European Union’s Horizon 2020 research and innovation program (grant agreement 668353/U‐PGx).

## CONFLICT OF INTEREST

Magnus Ingelman‐Sundberg is a co‐founder and co‐owner of HepaPredict AB. All other authors declared no competing interests for this work.

## AUTHOR CONTRIBUTIONS

H.C.L., K.K., and M.I.‐S. wrote the manuscript. E.M., K.K., I.J., and M.I.‐S. designed the research. H.C.L. and K.K. performed the research. H.C.L., K.K., I.J., R.L.S., M.M.J., E.M., and M.I.‐S. analyzed the data.

## ETHICAL APPROVAL

The clinical studies were carried out only after approval by the appropriate Ethical Committees in Oslo, Norway, and Stockholm, Sweden. Written consents were from human participants to collect and process their data. The data are presented at an aggregated level without possibility to identify individuals. Consent has been acquired from the human participants to process/share data. The use of anonymized patients’ data for the purpose of this study was approved by the Regional Committee of South‐Eastern Norway for Medical and Health Research Ethics and the Hospital’s Investigational Review Board and Privacy Officer.

For preparation of 3D human liver spheroids, primary human liver cells were obtained from commercially available sources and required no ethical approval by Karolinska Institutet. Ethical approval for the donors received from KaLy Cell was obtained. Copies of documentation from Lonza and BioIVT regarding written consent by donors was obtained from the respective companies.

## Supporting information

## ACKNOWLEDGMENTS

The authors acknowledge the excellent technical assistance from Åsa Nordling, Eida Tewolde, and Jennifer Nilsson at Karolinska Institutet and the bioinformatic support from Maxim Ivanov, University of Copenhagen, Denmark. We very much appreciate the important support from the Biomedicum Imaging Core facility (BIC) for help with the microscopy and image analysis and quantifications.

## DATA AVAILABILITY STATEMENT

All raw and processed data are available upon request.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All raw and processed data are available upon request.

### Supplementary Materials

### Data Availability Statement

All raw and processed data are available upon request.

## References

1. Ingelman‐Sundberg, M. Genetic polymorphisms of cytochrome P450 2D6 (CYP2D6): clinical consequences, evolutionary aspects and functional diversity. Pharmacogenomics J. 5, 6–13 (2005).  [DOI](https://doi.org/10.1038/sj.tpj.6500285) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15492763/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Genetic%20polymorphisms%20of%20cytochrome%20P450%202D6%20(CYP2D6):%20clinical%20consequences,%20evolutionary%20aspects%20and%20functional%20diversity&author=M.%20Ingelman%E2%80%90Sundberg&volume=5&publication_year=2005&pages=6-13&pmid=15492763&doi=10.1038/sj.tpj.6500285&)

2. Milosavljevic, F. et al. Association of CYP2C19 and CYP2D6 poor and intermediate metabolizer status with antidepressant and antipsychotic exposure: a systematic review and meta‐analysis. JAMA Psychiatry 78, 270–280 (2021).  [DOI](https://doi.org/10.1001/jamapsychiatry.2020.3643) | [PMC free article](/articles/PMC7702196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33237321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Psychiatry&title=Association%20of%20CYP2C19%20and%20CYP2D6%20poor%20and%20intermediate%20metabolizer%20status%20with%20antidepressant%20and%20antipsychotic%20exposure:%20a%20systematic%20review%20and%20meta%E2%80%90analysis&author=F.%20Milosavljevic&volume=78&publication_year=2021&pages=270-280&pmid=33237321&doi=10.1001/jamapsychiatry.2020.3643&)

3. Jukić, M.M. , Smith, R.L. , Molden, E. & Ingelman‐Sundberg, M. Evaluation of the CYP2D6 haplotype activity scores based on metabolic ratios of 4,700 patients treated with three different CYP2D6 substrates. Clin. Pharmacol. Ther. 110, 750–758 (2021).  [DOI](https://doi.org/10.1002/cpt.2246) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33792048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Evaluation%20of%20the%20CYP2D6%20haplotype%20activity%20scores%20based%20on%20metabolic%20ratios%20of%204,700%20patients%20treated%20with%20three%20different%20CYP2D6%20substrates&author=M.M.%20Juki%C4%87&author=R.L.%20Smith&author=E.%20Molden&author=M.%20Ingelman%E2%80%90Sundberg&volume=110&publication_year=2021&pages=750-758&pmid=33792048&doi=10.1002/cpt.2246&)

4. Molden, E. & Jukić, M.M. CYP2D6 reduced function variants and genotype/phenotype translations of CYP2D6 intermediate metabolizers: implications for personalized drug dosing in psychiatry. Front. Pharmacol. 12, 650750 (2021).  [DOI](https://doi.org/10.3389/fphar.2021.650750) | [PMC free article](/articles/PMC8100508/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33967790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=CYP2D6%20reduced%20function%20variants%20and%20genotype/phenotype%20translations%20of%20CYP2D6%20intermediate%20metabolizers:%20implications%20for%20personalized%20drug%20dosing%20in%20psychiatry&author=E.%20Molden&author=M.M.%20Juki%C4%87&volume=12&publication_year=2021&pmid=33967790&doi=10.3389/fphar.2021.650750&)

5. Gaedigk, A. , Sangkuhl, K. , Whirl‐Carrillo, M. , Klein, T. & Leeder, J.S. Prediction of CYP2D6 phenotype from genotype across world populations. Genetics Med. 19, 69–76 (2017).  [DOI](https://doi.org/10.1038/gim.2016.80) | [PMC free article](/articles/PMC5292679/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27388693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genetics%20Med.&title=Prediction%20of%20CYP2D6%20phenotype%20from%20genotype%20across%20world%20populations&author=A.%20Gaedigk&author=K.%20Sangkuhl&author=M.%20Whirl%E2%80%90Carrillo&author=T.%20Klein&author=J.S.%20Leeder&volume=19&publication_year=2017&pages=69-76&pmid=27388693&doi=10.1038/gim.2016.80&)

6. Jukic, M.M. , Smith, R.L. , Haslemo, T. , Molden, E. & Ingelman‐Sundberg, M. Effect of CYP2D6 genotype on exposure and efficacy of risperidone and aripiprazole: a retrospective, cohort study. Lancet Psychiatry 6, 418–426 (2019).  [DOI](https://doi.org/10.1016/S2215-0366(19)30088-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31000417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Psychiatry&title=Effect%20of%20CYP2D6%20genotype%20on%20exposure%20and%20efficacy%20of%20risperidone%20and%20aripiprazole:%20a%20retrospective,%20cohort%20study&author=M.M.%20Jukic&author=R.L.%20Smith&author=T.%20Haslemo&author=E.%20Molden&author=M.%20Ingelman%E2%80%90Sundberg&volume=6&publication_year=2019&pages=418-426&pmid=31000417&doi=10.1016/S2215-0366(19)30088-4&)

7. van der Lee, M et al. Toward predicting CYP2D6‐mediated variable drug response from CYP2D6 gene sequencing data. Sci. Transl. Med. 21, eabf3637 (2021).  [DOI](https://doi.org/10.1126/scitranslmed.abf3637) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34290055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Transl.%20Med.&title=Toward%20predicting%20CYP2D6%E2%80%90mediated%20variable%20drug%20response%20from%20CYP2D6%20gene%20sequencing%20data&author=M%20van%20der%20Lee&volume=21&publication_year=2021&pages=eabf3637&pmid=34290055&doi=10.1126/scitranslmed.abf3637&)

8. Chaudhry, A.Z. , Lyons, G.E. & Gronostajski, R.M. Expression patterns of the four nuclear factor I genes during mouse embryogenesis indicate a potential role in development. Dev. Dynamics 208, 313–325 (1997).  [DOI](https://doi.org/10.1002/(SICI)1097-0177(199703)208:3<313::AID-AJA3>3.0.CO;2-L) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9056636/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dev.%20Dynamics&title=Expression%20patterns%20of%20the%20four%20nuclear%20factor%20I%20genes%20during%20mouse%20embryogenesis%20indicate%20a%20potential%20role%20in%20development&author=A.Z.%20Chaudhry&author=G.E.%20Lyons&author=R.M.%20Gronostajski&volume=208&publication_year=1997&pages=313-325&pmid=9056636&doi=10.1002/(SICI)1097-0177(199703)208:3<313::AID-AJA3>3.0.CO;2-L&)

9. Steele‐Perkins, G. et al. The transcription factor gene Nfib is essential for both lung maturation and brain development. Mol. Cell Biol. 25, 685–698 (2005).  [DOI](https://doi.org/10.1128/MCB.25.2.685-698.2005) | [PMC free article](/articles/PMC543431/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15632069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell%20Biol.&title=The%20transcription%20factor%20gene%20Nfib%20is%20essential%20for%20both%20lung%20maturation%20and%20brain%20development&author=G.%20Steele%E2%80%90Perkins&volume=25&publication_year=2005&pages=685-698&pmid=15632069&doi=10.1128/MCB.25.2.685-698.2005&)

10. Yang, H. , Wu, Z. , Liu, X. , Chen, M. , Zhang, X. & Jiang, Y. NFIB promotes the progression of gastric cancer by upregulating circMAP7D1 to stabilize HER2 mRNA. Mol. Med. Rep. 23, 269 (2021).  [DOI](https://doi.org/10.3892/mmr.2021.11908) | [PMC free article](/articles/PMC7893781/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33576439/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Med.%20Rep.&title=NFIB%20promotes%20the%20progression%20of%20gastric%20cancer%20by%20upregulating%20circMAP7D1%20to%20stabilize%20HER2%20mRNA&author=H.%20Yang&author=Z.%20Wu&author=X.%20Liu&author=M.%20Chen&author=X.%20Zhang&volume=23&publication_year=2021&pages=269&pmid=33576439&doi=10.3892/mmr.2021.11908&)

11. Smith, R.L. et al. Identification of a novel polymorphism associated with reduced clozapine concentration in schizophrenia patients‐a genome‐wide association study adjusting for smoking habits. Transl. Psychiatry 10, 198 (2020).  [DOI](https://doi.org/10.1038/s41398-020-00888-1) | [PMC free article](/articles/PMC7303159/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32555152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Transl.%20Psychiatry&title=Identification%20of%20a%20novel%20polymorphism%20associated%20with%20reduced%20clozapine%20concentration%20in%20schizophrenia%20patients%E2%80%90a%20genome%E2%80%90wide%20association%20study%20adjusting%20for%20smoking%20habits&author=R.L.%20Smith&volume=10&publication_year=2020&pages=198&pmid=32555152&doi=10.1038/s41398-020-00888-1&)

12. Molden, E. Therapeutic drug monitoring of clozapine in adults with schizophrenia: a review of challenges and strategies. Expert Opin. Drug Metab. Toxicol. 17, 1211–1221 (2021).  [DOI](https://doi.org/10.1080/17425255.2021.1974400) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34461790/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin.%20Drug%20Metab.%20Toxicol.&title=Therapeutic%20drug%20monitoring%20of%20clozapine%20in%20adults%20with%20schizophrenia:%20a%20review%20of%20challenges%20and%20strategies&author=E.%20Molden&volume=17&publication_year=2021&pages=1211-1221&pmid=34461790&doi=10.1080/17425255.2021.1974400&)

13. Hendset, M. , Molden, E. , Knape, M. & Hermann, M. Serum concentrations of risperidone and aripiprazole in subgroups encoding CYP2D6 intermediate metabolizer phenotype. Ther. Drug Monit. 36, 80–85 (2014).  [DOI](https://doi.org/10.1097/FTD.0000000000000018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24232129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther.%20Drug%20Monit.&title=Serum%20concentrations%20of%20risperidone%20and%20aripiprazole%20in%20subgroups%20encoding%20CYP2D6%20intermediate%20metabolizer%20phenotype&author=M.%20Hendset&author=E.%20Molden&author=M.%20Knape&author=M.%20Hermann&volume=36&publication_year=2014&pages=80-85&pmid=24232129&doi=10.1097/FTD.0000000000000018&)

14. Bell, C.C. et al. Characterization of primary human hepatocyte spheroids as a model system for drug‐induced liver injury, liver function and disease. Sci. Rep. 6, 25187 (2016).  [DOI](https://doi.org/10.1038/srep25187) | [PMC free article](/articles/PMC4855186/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27143246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&title=Characterization%20of%20primary%20human%20hepatocyte%20spheroids%20as%20a%20model%20system%20for%20drug%E2%80%90induced%20liver%20injury,%20liver%20function%20and%20disease&author=C.C.%20Bell&volume=6&publication_year=2016&pages=25187&pmid=27143246&doi=10.1038/srep25187&)

15. Westlind, A. , Löfberg, L. , Tindberg, N. , Andersson, T.B. & Ingelman‐Sundberg, M. Interindividual differences in hepatic expression of CYP3A4: relationship to genetic polymorphism in the 5'‐upstream regulatory region. Biochem. Biophys. Res. Commun. 259, 201–205 (1999).  [DOI](https://doi.org/10.1006/bbrc.1999.0752) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10334940/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&title=Interindividual%20differences%20in%20hepatic%20expression%20of%20CYP3A4:%20relationship%20to%20genetic%20polymorphism%20in%20the%205'%E2%80%90upstream%20regulatory%20region&author=A.%20Westlind&author=L.%20L%C3%B6fberg&author=N.%20Tindberg&author=T.B.%20Andersson&author=M.%20Ingelman%E2%80%90Sundberg&volume=259&publication_year=1999&pages=201-205&pmid=10334940&doi=10.1006/bbrc.1999.0752&)

16. Mannheimer, B. , Haslemo, T. , Lindh, J.D. , Eliasson, E. & Molden, E. Risperidone and venlafaxine metabolic ratios strongly predict a CYP2D6 poor metabolizing genotype. Ther. Drug Monit. 38, 127–134 (2016).  [DOI](https://doi.org/10.1097/FTD.0000000000000251) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26418700/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ther.%20Drug%20Monit.&title=Risperidone%20and%20venlafaxine%20metabolic%20ratios%20strongly%20predict%20a%20CYP2D6%20poor%20metabolizing%20genotype&author=B.%20Mannheimer&author=T.%20Haslemo&author=J.D.%20Lindh&author=E.%20Eliasson&author=E.%20Molden&volume=38&publication_year=2016&pages=127-134&pmid=26418700&doi=10.1097/FTD.0000000000000251&)

17. Palleja, A. , Horn, H. , Eliasson, S. & Jensen, L.J. DistiLD database: diseases and traits in linkage disequilibrium blocks. Nucleic Acids Res. 40(D1), D1036–D1040 (2012).  [DOI](https://doi.org/10.1093/nar/gkr899) | [PMC free article](/articles/PMC3245128/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22058129/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=DistiLD%20database:%20diseases%20and%20traits%20in%20linkage%20disequilibrium%20blocks&author=A.%20Palleja&author=H.%20Horn&author=S.%20Eliasson&author=L.J.%20Jensen&volume=40&issue=D1&publication_year=2012&pages=D1036-D1040&pmid=22058129&doi=10.1093/nar/gkr899&)

18. Zhou, Y. , Ingelman‐Sundberg, M. & Lauschke, V. Worldwide distribution of cytochrome P450 alleles: a meta‐analysis of population‐scale sequencing projects. Clin. Pharm. Ther. 102, 688–700 (2017).  [DOI](https://doi.org/10.1002/cpt.690) | [PMC free article](/articles/PMC5600063/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28378927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharm.%20Ther.&title=Worldwide%20distribution%20of%20cytochrome%20P450%20alleles:%20a%20meta%E2%80%90analysis%20of%20population%E2%80%90scale%20sequencing%20projects&author=Y.%20Zhou&author=M.%20Ingelman%E2%80%90Sundberg&author=V.%20Lauschke&volume=102&publication_year=2017&pages=688-700&pmid=28378927&doi=10.1002/cpt.690&)

19. Gronostajski, R.M. Roles of NFI/CTF gene family in transcription and development. Gene 249, 31–45 (2000).  [DOI](https://doi.org/10.1016/s0378-1119(00)00140-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10831836/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gene&title=Roles%20of%20NFI/CTF%20gene%20family%20in%20transcription%20and%20development&author=R.M.%20Gronostajski&volume=249&publication_year=2000&pages=31-45&pmid=10831836&doi=10.1016/s0378-1119(00)00140-2&)
