# Analysis of Genetic and Clinical Factors Associated with Buprenorphine Response

## Metadata
**Authors:** Richard C Crist, Rachel Vickers-Smith, Rachel L Kember, Christopher T Rentsch, Heng Xu, E Jennifer Edelman, Emily E Hartwell, Kyle M Kampman, Henry R Kranzler
**Journal:** Drug and alcohol dependence
**Date:** 2021 Aug 28
**DOI:** [10.1016/j.drugalcdep.2021.109013](https://doi.org/10.1016/j.drugalcdep.2021.109013)
**PMID:** 34488071
**PMCID:** PMC9328121
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9328121/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9328121/pdf/nihms-1825216.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9328121/pdf/nihms-1825216.pdf)

## Abstract

**Background:** 
Buprenorphine, approved for treating opioid use disorder (OUD), is not equally efficacious for all patients. Candidate gene studies have shown limited success in identifying genetic moderators of buprenorphine treatment response.

**Methods:** 
We studied 1,616 European-ancestry individuals enrolled in the Million Veteran Program, of whom 1,609 had an ICD-9/10 code consistent with OUD, a 180-day buprenorphine treatment exposure, and genome-wide genotype data. We conducted a genome-wide association study (GWAS) of buprenorphine treatment response [defined as having no opioid-positive urine drug screens (UDS) following the first prescription]. We also examined correlates of buprenorphine treatment response in multivariable analyses.

**Results:** 
Although no variants reached genome-wide significance, 6 loci were nominally significant (p<1 × 10−5), four of which were located near previously characterized genes: rs756770 (ADAMTSL2), rs11782370 (SLC25A37), rs7205113 (CRISPLD2), and rs13169373 (LINC01947). A higher maximum daily buprenorphine dosage (aOR=0.98; 95%CI: 0.97, 0.995), greater number of UDS (aOR=0.97; 95%CI: 0.96, 0.99), and history of hepatitis C (HCV) infection (aOR=0.71; 95%CI: 0.57, 0.88) were associated with a reduced odds of buprenorphine response. Older age (aOR: 1.01; 95%CI: 1.000, 1.02) was associated with increased odds of buprenorphine response.

**Conclusions:** 
This study had limited statistical power to detect genetic variants associated with a complex human phenotype like buprenorphine treatment response. Meta-analysis of multiple data sets is needed to ensure adequate statistical power for a GWAS of buprenorphine treatment response. The most robust phenotypic predictor of buprenorphine treatment response was intravenous drug use, a proxy for which was HCV infection.

Keywords: buprenorphine, Genome-wide Association Study, genetics, Treatment Response, Opioid use disorder, Treatment Predictors

### Background:

Buprenorphine, approved for treating opioid use disorder (OUD), is not equally efficacious for all patients. Candidate gene studies have shown limited success in identifying genetic moderators of buprenorphine treatment response.

### Methods:

We studied 1,616 European-ancestry individuals enrolled in the Million Veteran Program, of whom 1,609 had an ICD-9/10 code consistent with OUD, a 180-day buprenorphine treatment exposure, and genome-wide genotype data. We conducted a genome-wide association study (GWAS) of buprenorphine treatment response [defined as having no opioid-positive urine drug screens (UDS) following the first prescription]. We also examined correlates of buprenorphine treatment response in multivariable analyses.

### Results:

Although no variants reached genome-wide significance, 6 loci were nominally significant (p<1 × 10^−5^), four of which were located near previously characterized genes: rs756770 (*ADAMTSL2*), rs11782370 (*SLC25A37*), rs7205113 (*CRISPLD2*), and rs13169373 (*LINC01947)*. A higher maximum daily buprenorphine dosage (aOR=0.98; 95%CI: 0.97, 0.995), greater number of UDS (aOR=0.97; 95%CI: 0.96, 0.99), and history of hepatitis C (HCV) infection (aOR=0.71; 95%CI: 0.57, 0.88) were associated with a reduced odds of buprenorphine response. Older age (aOR: 1.01; 95%CI: 1.000, 1.02) was associated with increased odds of buprenorphine response.

### Conclusions:

This study had limited statistical power to detect genetic variants associated with a complex human phenotype like buprenorphine treatment response. Meta-analysis of multiple data sets is needed to ensure adequate statistical power for a GWAS of buprenorphine treatment response. The most robust phenotypic predictor of buprenorphine treatment response was intravenous drug use, a proxy for which was HCV infection.

**Keywords:** buprenorphine, Genome-wide Association Study, genetics, Treatment Response, Opioid use disorder, Treatment Predictors

## 1. Introduction

Buprenorphine is one of three FDA-approved medications for treating opioid use disorder (OUD). Although treatment with buprenorphine is associated with substantial benefits, including decreased opioid overdose risk [[1](#R1)], a significant proportion of individuals do not achieve long-term abstinence and drop out of care [[2](#R2)]. Knowledge of the factors that predict return to opioid use would enable clinicians to identify at-risk patients prospectively and modify treatment (e.g., by prescribing a higher dosage) to increase the likelihood of successful treatment.

Demographic factors, including age, sex, race, and ethnicity, have been associated with OUD treatment outcomes [[3](#R3)–[6](#R6)]. Unemployment and injection drug use, possible proxy measures for the presence of comorbid conditions or disease severity, have also been associated with a return to opioid use or dropping out of treatment [[7](#R7)–[9](#R9)]. Electronic health records (EHRs) provide a wealth of patient data that could be used to identify factors that influence treatment outcomes. For example, mood disorder diagnoses are common among individuals with OUD [[10](#R10)] and have been associated with buprenorphine adherence rates [[11](#R11)–[13](#R13)]. EHRs also capture comorbid substance use disorders directly or through laboratory test results [e.g., urine drug screens (UDS)] or related diagnosis codes (e.g., alcohol-related cirrhosis). An important clinical problem among patients in treatment for OUD is the high rate of multi-substance use [[14](#R14)] and among patients receiving buprenorphine the use of substances other than opioids increases the likelihood that they will return to opioid use [[15](#R15), [16](#R16)].

Genetic variation among patients can result in differences in medication effectiveness through pharmacokinetic and/or pharmacodynamic mechanisms. Despite growing support for a pharmacogenetic approach across a variety of medications [[17](#R17)], the literature on genetic moderators of OUD treatment effectiveness is limited [[18](#R18)]. Response to methadone among patients of European ancestry has been associated with single nucleotide variation in *DRD2, ARRB2, ALDH5A1, MYOCD*, and *GRM6*, as well as haplotypes in *BDNF* and *OPRM1* [[19](#R19)–[24](#R24)]. Buprenorphine effectiveness in patients has also been linked to a variable number of tandem repeats polymorphism [[25](#R25)] in *SLC6A3* and, in multiple studies, to *OPRD1* polymorphisms. Two intronic *OPRD1* single nucleotide polymorphisms (SNPs) predicted sublingual buprenorphine treatment response in European-American (EA) women [[26](#R26)]. A different intronic *OPRD1* SNP (rs678849) predicted buprenorphine treatment response in African-Americans (AA) [[27](#R27)], a finding that was replicated in an independent cohort [[28](#R28)]. In a third study, in which patients were treated with extended-release buprenorphine [[29](#R29)], rs678849 predicted efficacy in EAs, but not AAs. In summary, while these pharmacogenetic studies provide suggestive evidence that genetic variation could moderate OUD treatment response, they are based on candidate gene approaches in cohorts of at most a few hundred patients, limiting their statistical power and the generalizability of the findings.

Here we present the first genome-wide association study (GWAS) of buprenorphine response in 1,609 EA patients treated for OUD and enrolled in the Million Veteran Program (MVP). Despite the modest sample size for GWAS, there is evidence that statistical power is enhanced in the study of treatment-relevant variants relative to disease-related ones [[30](#R30)]. Further, a GWAS of usual methadone dose in a sample smaller than the one here yielded a genome-wide significant finding [[31](#R31)]. We also characterized the phenotype, by examining the associations between measures of treatment outcome and EHR variables previously linked to OUD or OUD treatment effectiveness, including demographic measures and comorbid diagnoses.

## 2. Materials and Methods

### 2.1. Participants

MVP is a large biobank created and maintained by the US Department of Veteran Affairs (VA) [[32](#R32)], which at the time of the study included EHR data on over 700,000 Veterans, a majority of whom also had linked genetic data. The main MVP study and the analyses described here were approved by the Central VA Institutional Review Board. All patients provided written informed consent to participate in MVP.

We included 1,616 EA Veterans enrolled in MVP and who had 1) an ICD-9/10 code consistent with OUD between August 2003 and November 2018; and 2) at least 60 consecutive days of sublingual buprenorphine/naloxone treatment based on EHR prescription data, including, but not limited to, prescriptions for the brand name medications Suboxone, Zubsolv, and Bunavail or for generic buprenorphine/naloxone. The buprenorphine treatment window was defined as the time period from the start of the initial prescription to a) the first two-week period in which no buprenorphine was prescribed or b) 180 days, whichever was shorter. For patients who had multiple distinct periods of buprenorphine treatment, only the first treatment period was included in the analysis.

Treatment response was measured by UDS for full opioid agonists, including methadone, within the treatment window. An opioid-positive UDS was defined as one that tested positive for one or more opioid agonists. If a confirmatory test of an opioid-positive UDS returned negative results, the UDS was considered to be a false positive and was coded as opioid-negative for the purposes of treatment response. Treatment responders were defined as having no opioid-positive UDS. This definition of treatment response was chosen over other possible outcome measurements, including the percentage of UDS positive for opioid agonists and treatment duration, because the distributions of these other definitions were skewed and thus did not lend themselves to GWAS ([Supplemental Figure 1](#SD1)).

### 2.2. Phenotype Characterization

Potential correlates of buprenorphine treatment response were selected *a priori*. We used age at the time of the first buprenorphine prescription. Data on buprenorphine maximum daily dosage and UDS were obtained for the first 180 days of treatment. Comorbid psychiatric disorders and infectious diseases were considered present if there was 1 inpatient or 2 outpatient ICD-9/10 codes for them in the EHR ([Supplemental Tables 2](#SD1) and [3](#SD2)).

### 2.3. Statistical Analysis

Descriptive statistics included means with standard deviations and medians with interquartile ranges for continuous variables and frequencies with percentages for categorical variables. Bivariate associations between potential correlates and buprenorphine response were assessed through simple logistic regressions. Variables with a p-value ≤ 0.1 in bivariate analysis were included in a multivariable logistic regression model. A C-statistic was used to assess goodness of fit of the multivariable model. Analyses were performed using SAS 9.2 (Cary, NC) and an alpha of 0.05 was used to denote statistical significance.

### 2.4. Genotyping and imputation

Genotyping was performed using a custom Affymetrix Axiom biobank array containing 723,305 SNPs. The array is enriched for markers of AA and Hispanic ancestry and common diseases of interest to the VA population [[32](#R32)].

Samples with >2.5% missing genotype calls or high heterozygosity and SNPs with high missingness or deviation from the expected allele frequency were removed [[33](#R33)]. It is now standard practice to impute SNP genotype data, allowing prediction of ungenotyped variants to increase the number of variants available for association testing [[34](#R34)]. Genotypes were pre-phased using EAGLE v2 and imputed via Minimac4 software using the 1000 Genomes Project phase 3 v5 reference panel [[35](#R35)]. The top 30 principal components (PCs) were computed using FlashPCA on a dataset that included all MVP participants and an additional 2,504 individuals from 1000 Genomes Phase 3. Genetically inferred ancestry, derived from the PCs, and self-reported race/ancestry were unified to assign individuals to ancestral groups (HARE) [[36](#R36)]. Within-ancestry PCs were then computed for MVP individuals within each ancestral group for use as covariates.

### 2.5. Genome-wide association analysis

Following imputation, 318,725 EA individuals in MVP were identified based on the HARE definition. Population-specific imputation quality (INFO) score was calculated, and SNPs with imputation quality <0.3 were excluded. A total of 1,609 individuals had complete genomic and phenotypic information for analysis. We used imputed SNPs that passed quality control (minor allele frequency > 1%, genotype call rate >0.95, and Hardy-Weinberg Equilibrium p < 1e-6) to test the association of buprenorphine response using logistic regression. Covariates included age, sex, and the first 10 PCs. Gene based and functional analyses of GWAS results were performed using MAGMA and FUMA, respectively ([https://fuma.ctglab.nl/](https://fuma.ctglab.nl/)) [[37](#R37), [38](#R38)].

### 2.6. Data Availability

Summary statistics from the GWAS are available through dbGaP at accession no. phs001672.v3.p1.

## 3. Results

### 3.1. Genome-wide association study of buprenorphine response

As shown in [Figure 1](#F1), the GWAS yielded six loci that were nominally significantly associated with buprenorphine treatment response (p<1 × 10^−5^), defined as continuous abstinence during buprenorphine treatment ([Table 1](#T1)). Three of the variants were located near previously characterized protein-coding genes: rs756770 (*ADAMTSL2*), rs11782370 (*SLC25A37*), and rs7205113 (*CRISPLD2*). A fourth variant (rs13169373) was located within the long intergenic non-coding RNA *LINC01947*. Two additional intergenic loci were also identified: rs62368105 (chr5:43970054) and rs6973474 (chr7:96804). Regional association plots for all nominally significant loci are provided in [Supplemental Figure 2](#SD1). No variants were genome-wide significant (p<5 × 10^−8^). Analysis with MAGMA did not identify any genes significantly associated with buprenorphine treatment response after correction for multiple testing at a False Discovery Rate (FDR) of 0.05 (data not shown). Results for genes previously associated with OUD treatment response are provided in [Supplemental Table 1](#SD1). Functional analysis of the genes associated with the nominally significant variants identified in the GWAS was performed with FUMA (GENE2FUNC). No significant enrichment was observed (data not shown).

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/11e5/9328121/9bc2c1a326af/nihms-1825216-f0001.jpg)

Manhattan plot of results from a genome-wide association study of buprenorphine response in European-ancestry individuals treated for opioid use disorder (n = 1,609). All peaks containing lead SNPS with p <1 × 10−5 and mapping to previously characterized genes are labeled with the corresponding gene symbol.

### Table 1.

| SNP ID | Chromosome:Position | Effect Allele | Nearest Gene | Odds Ratio | Standard Error | P-Value |
| --- | --- | --- | --- | --- | --- | --- |
| rs62368105 | 5:43970054 | G | NNT/FGF10 | 1.9437 | 0.147647 | 6.76×10-06 |
| rs13169373 | 5:166345088 | T | LINC01947 | 1.4364 | 0.0818052 | 9.54×10-06 |
| rs6973474 | 7:96804 | T | FAM20C | 1.4043 | 0.0762082 | 8.36×10-06 |
| rs11782370 | 8:23370018 | T | SLC25A37 | 1.4843 | 0.0878757 | 6.97×10-06 |
| rs756770 | 9:136398858 | A | ADAMTSL2 | 1.8599 | 0.138925 | 7.93×10-06 |
| rs7205113 | 16:84847823 | T | CRISPLD2 | 1.6486 | 0.112005 | 8.05×10-06 |

Table 1 Caption: Nominally significant lead genetic variants associated with buprenorphine treatment response

### 3.2. Phenotypic characterization of buprenorphine response

Based on unadjusted associations, age, sex, and diagnoses for HIV or comorbid psychiatric or substance use disorders were not associated with buprenorphine response ([Table 2](#T2); all p>0.05). In the multivariable analysis, a higher maximum daily dosage of buprenorphine (aOR=0.98; 95% CI: 0.97, 0.995) and a greater number of UDS (aOR=0.98; 95% CI: 0.96, 0.99) were associated with significantly reduced odds of buprenorphine treatment response. Veterans with a history of hepatitis C (HCV) had 29% reduced odds of buprenorphine response compared to those without HCV (aOR=0.71; 95% CI: 0.57, 0.88). Older age was significantly associated with higher odds of buprenorphine treatment response (aOR=1.01; 95% CI: 1.000, 1.02).

### Table 2.

|   | % (n) | Treatment response (n=988) | Treatment non-response (n=628) | Unadjusted OR (95% CI) | P | Adjusted OR (95% OR) | P |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Male | 92% (1485) | 92% (908) | 92% (577) | 1.00 (0.70, 1.45) | 0.99 | -- | -- |
| Age - mean (SD)Median (IQR) | 45 (13.57)47 (32, 57) | 46 (13.54)48 (33, 57) | 45 (13.61)46 (32, 56) | 1.01 (1.00, 1.02) | 0.06 | 1.01 (1.000, 1.02) | 0.04 |
| % Opioid-positive UDS – mean (SD)Median (IQR) | 10% (18.29%)0% (0%, 12.50%) | -- | 25% (21.87%)17% (10%, 33%) | -- | -- | -- | -- |
| # of UDS – mean (SD)Median (IQR) | 11 (7.63)9 (5, 14) | 10 (7.50)8 (4, 14) | 12 (7.71)10 (6, 15) | 0.97 (0.96, 0.98) | <0.01 | 0.98 (0.96, 0.99) | <0.01 |
| Max daily dosage of buprenorphine (mg) – mean (SD)Median (IQR) | 17 (7.29)16 (12, 24) | 16 (7.46)16 (10, 20) | 17 (6.96)16 (12, 24) | 0.98 (0.96, 0.99) | <0.01 | 0.98 (0.97, 0.995) | 0.01 |
| Chronic pain | 96% (1554) | 96% (952) | 96% (602) | 1.14 (0.68, 1.91) | 0.61 | -- | -- |
| HCV | 34% (556) | 32% (313) | 39% (243) | 0.74 (0.60, 0.91) | <0.01 | 0.71 (0.57, 0.88) | <0.01 |
| HIV | 1% (17) | 1% (9) | 1% (8) | 0.71 (0.27, 1.86) | 0.49 | -- | -- |
| Anxiety | 55% (894) | 57% (559) | 53% (335) | 1.14 (0.93, 1.39) | 0.20 | -- | -- |
| Depression | 63% (1014) | 62% (612) | 64% (402) | 0.92 (0.74, 1.13) | 0.40 | -- | -- |
| PTSD | 61% (983) | 61% (605) | 60% (378) | 1.05 (0.85, 1.28) | 0.67 | -- | -- |
| Other, non- SUD psychiatric diagnosis | 88% (1429) | 89% (875) | 88% (554) | 1.03 (0.76, 1.41) | 0.83 | -- | -- |
| Alcohol use disorder | 65% (1058) | 67% (657) | 64% (401) | 1.12 (0.91, 1.39) | 0.28 | -- | -- |
| Tobacco use disorder | 82% (1322) | 82% (812) | 81% (510) | 1.07 (0.83, 1.38) | 0.62 | -- | -- |
| Cocaine use disorder | 38% (618) | 37% (365) | 40% (253) | 0.87 (0.71, 1.07) | 0.18 | -- | -- |
| Cannabis use disorder | 32% (511) | 31% (310) | 32% (201) | 0.97 (0.78, 1.20) | 0.79 | -- | -- |
| Stimulant use disorder | 23% (364) | 23% (227) | 22% (137) | 1.07 (0.84, 1.36) | 0.59 | -- | -- |
| Sedative use disorder | 25% (409) | 26% (252) | 25% (157) | 1.03 (0.82, 1.29) | 0.82 | -- | -- |
| Hallucinogen use disorder | 1% (23) | 2% (16) | 1% (7) | 1.46 (0.60, 3.57) | 0.41 | -- | -- |
| Other SUD diagnosis | 58% (932) | 57% (565) | 58% (367) | 0.95 (0.78, 1.16) | 0.62 | -- | -- |
| Any SUD (excluding OUD) | 93% (1505) | 94% (924) | 93% (581) | 1.17 (0.79, 1.73) | 0.43 | -- | -- |

Table 2 Caption: Characteristics of the sample and associations with buprenorphine response (n=1,616).

## 4. Discussion

Abstinence during buprenorphine treatment was nominally associated with six loci, including variants near *SLC25A37*, *ADAMTSL2*, *CRISPLD2*, and *LINC01947*. *SLC25A37* encodes an iron transporter localized to the mitochondrial membrane and is upregulated in the nucleus accumbens of individuals addicted to heroin or cocaine [[39](#R39), [40](#R40)]. The lead variant from this locus (rs11782370) is an expression quantitative trait locus (eQTL) for *SLC25A37* in multiple tissues, as well as for *ENTPD4*, a gene that is ~55 kb upstream of the SNP [[41](#R41)]. Polymorphisms in the *SLC25A37* have also been linked to major depressive disorder [[42](#R42), [43](#R43)], which may be relevant to buprenorphine treatment [[44](#R44)], though depression was not significantly associated with treatment response in the current sample. *ADAMTSL2* encodes a secreted glycoprotein and in the UK Biobank data set the gene was associated with heaviness of smoking, a common comorbid disorder in OUD patients [[45](#R45)]. Despite the lead variant in this locus (rs756770) being intronic, it is not associated with *ADAMTSL2* expression [[41](#R41)]. However, it is identified as an eQTL for the upstream gene *REXO4* in whole blood [[41](#R41)]. Finally, *CRISPLD2* encodes a secreted anti-inflammatory protein that has been implicated in obesity and weight loss and *LINC01947* is a long intergenic non-coding RNA with limited functional data [[46](#R46), [47](#R47)].

GWAS of psychiatric disorders generally require samples in the tens of thousands to identify variants of genome-wide significance [[48](#R48)–[50](#R50)]. Thus, the sample in this GWAS of buprenorphine treatment response provided limited statistical power to detect variants associated with a complex human phenotype like treatment response. However, studying treatment-relevant variants may provide greater statistical power for a given sample size than disease-related ones [[30](#R30)] and some pharmacogenomic studies have yielded genome-wide significant findings with smaller samples [[51](#R51), [52](#R52)], including a study of usual methadone maintenance dose [[31](#R31)], although the traits studied may be less polygenic than OUD treatment outcome. Meta-analyses similar to those conducted by the Psychiatric Genomics Consortium or the Consortium on Lithium Genetics are likely required to ensure the number of OUD patients treated with buprenorphine yields adequate statistical power for a GWAS of treatment response [[49](#R49), [51](#R51)]. Together with continued recruitment and the release of additional genome-wide genotype data by the Million Veteran Program, we are aggregating samples from other studies of buprenorphine-treated patients to increase the available statistical power of meta-analyses. It may also be possible to use natural language processing to augment UDS data to exploit the growing availability of EHR data for use in GWAS of buprenorphine response.

We observed that maximum buprenorphine daily dosage, number of UDS, and HCV seropositivity were all associated with opioid use during treatment. A greater maximum daily buprenorphine dose and number of UDS are likely attributable to providers’ response to patients who continued to use opioids and may lack value as predictors, serving instead as proxy measures of relapse. HCV status is strongly associated with injection drug use [[53](#R53), [54](#R54)] and is a valuable alternative metric, because HCV seropositivity is well captured in the EHR and direct information on injection drug use is not. Intravenous drug administration has been linked to the use of opioids and other substances during OUD treatment [[8](#R8), [55](#R55), [56](#R56)]. People who inject drugs have high rates of physical and psychiatric comorbidities and endorse a greater need for substance use treatment [[57](#R57)]. Given the association between heroin use and intravenous drug administration, this finding may also reflect differences between people who use heroin and those who use prescription opioids [[58](#R58)]. People who use heroin generally have worse buprenorphine treatment outcomes, including spending less time in treatment and having higher rates of opioid-positive UDS and of HCV infection than individuals who use only prescription opioids [[59](#R59)].

The VA is a large, nationwide medical system and standards for OUD treatment (e.g., inclusion criteria for buprenorphine treatment, dosage selection, frequency of UDS collection in buprenorphine-treated patients, methods of UDS testing) vary among VA facilities. This heterogeneity limited our ability to select potentially more informative phenotypes and identify genetic variants contributing to treatment response. The lack of specific information on drug use, including the route of administration and the type of opioids used (e.g., prescription vs. illicit) limited our ability to characterize the study sample phenotypically. Assessment of this information would be useful in future work. Factors associated with buprenorphine response in the VA may not be generalizable to other healthcare systems, although future studies can evaluate the reliability of this phenotype in other settings. Additionally, it was a predominantly male sample and the analysis was limited to EA patients because of the much smaller number of buprenorphine-treated patients of other ancestries. Whereas our ability to generalize our findings to other population groups is limited, future studies should aim to expand the number of women and non-European ancestry individuals. Further, greater uniformity in the VA approach to monitoring drug use among buprenorphine-treated patients would reduce variability in phenotyping for pharmacogenetic studies and also likely improve patient care.

In conclusion, we present the first GWAS of buprenorphine treatment response, together with the phenotype characterization of treatment outcomes. Our genetic findings include variants in several addiction-related genes that may be associated with buprenorphine treatment response, though they did not meet genome-wide statistical significance. We also found that HCV infection was correlated with buprenorphine non-response, supporting previously observed associations between measures of injection drug use and OUD treatment outcomes. We hope that this study provides an impetus for the collection of diverse cohorts of OUD patients being treated with buprenorphine, so as to permit the conduct of a multi-ancestry meta-GWAS of treatment response. Such a collaborative approach is likely to be the only way to ensure adequate statistical power to identify variants contributing to buprenorphine treatment response and to ensure the generalizability of the findings across population groups and both sexes.

## Supplementary Material

## Acknowledgements

This research is based on data from the Million Veteran Program (MVP), Office of Research and Development, Veterans Health Administration, and was supported by award #I01 CX001734-01 (MVP012); the Veterans Integrated Service Network 4 Mental Illness Research, Education and Clinical Center; and P30 DA046345 from the National Institute on Drug Abuse. This publication does not represent the views of the Department of Veteran Affairs or the United States Government.

### Declaration of Competing Interest

Dr. Kranzler is a member of an advisory board for Dicerna Pharmaceuticals; a consultant to Sophrosyne Pharmaceuticals; a member of the American Society of Clinical Psychopharmacology’s Alcohol Clinical Trials Initiative, which was supported in the last three years by AbbVie, Alkermes, Dicerna, Ethypharm, Indivior, Lilly, Lundbeck, Otsuka, Pfizer, Arbor, and Amygdala Neurosciences; and is named as an inventor on PCT patent application #15/878,640 entitled: “Genotype-guided dosing of opioid agonists,” filed January 24, 2018. Dr. Kampman has done work or served on advisory boards for Indivior Pharmaceuticals, World Meds, Alkermes, and Braeburn Pharmaceuticals.

### Disclosures:

Dr. Kranzler is a member of an advisory board for Dicerna Pharmaceuticals; a consultant to Sophrosyne Pharmaceuticals; a member of the American Society of Clinical Psychopharmacology’s Alcohol Clinical Trials Initiative, which was supported in the last three years by AbbVie, Alkermes, Dicerna, Ethypharm, Indivior, Lilly, Lundbeck, Otsuka, Pfizer, Arbor, and Amygdala Neurosciences; and is named as an inventor on PCT patent application #15/878,640 entitled: “Genotype-guided dosing of opioid agonists,” filed January 24, 2018. Dr. Kampman has consulted or served on advisory boards for Indivior Pharmaceuticals, World Meds, Alkermes, and Braeburn Pharmaceuticals.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Summary statistics from the GWAS are available through dbGaP at accession no. phs001672.v3.p1.

### Supplementary Materials

### Data Availability Statement

Summary statistics from the GWAS are available through dbGaP at accession no. phs001672.v3.p1.

## References

1. Larochelle MR, et al. , Medication for Opioid Use Disorder After Nonfatal Opioid Overdose and Association With Mortality: A Cohort Study. Ann Intern Med, 2018. 169(3): p. 137–145.  [DOI](https://doi.org/10.7326/M17-3107) | [PMC free article](/articles/PMC6387681/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29913516/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=Medication%20for%20Opioid%20Use%20Disorder%20After%20Nonfatal%20Opioid%20Overdose%20and%20Association%20With%20Mortality:%20A%20Cohort%20Study&author=MR%20Larochelle&volume=169&issue=3&publication_year=2018&pages=137-145&pmid=29913516&doi=10.7326/M17-3107&)

2. Mattick RP, et al. , Buprenorphine maintenance versus placebo or methadone maintenance for opioid dependence. Cochrane Database Syst Rev, 2014(2): p. CD002207.  [DOI](https://doi.org/10.1002/14651858.CD002207.pub4) | [PMC free article](/articles/PMC10617756/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24500948/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cochrane%20Database%20Syst%20Rev&title=Buprenorphine%20maintenance%20versus%20placebo%20or%20methadone%20maintenance%20for%20opioid%20dependence&author=RP%20Mattick&issue=2&publication_year=2014&pages=CD002207&pmid=24500948&doi=10.1002/14651858.CD002207.pub4&)

3. Subramaniam GA, et al. , Predictors of abstinence: National Institute of Drug Abuse multisite buprenorphine/naloxone treatment trial in opioid-dependent youth. J Am Acad Child Adolesc Psychiatry, 2011. 50(11): p. 1120–8.  [DOI](https://doi.org/10.1016/j.jaac.2011.07.010) | [PMC free article](/articles/PMC3786351/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22024000/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Acad%20Child%20Adolesc%20Psychiatry&title=Predictors%20of%20abstinence:%20National%20Institute%20of%20Drug%20Abuse%20multisite%20buprenorphine/naloxone%20treatment%20trial%20in%20opioid-dependent%20youth&author=GA%20Subramaniam&volume=50&issue=11&publication_year=2011&pages=1120-8&pmid=22024000&doi=10.1016/j.jaac.2011.07.010&)

4. Schuman-Olivier Z, et al. , Emerging adult age status predicts poor buprenorphine treatment retention. J Subst Abuse Treat, 2014. 47(3): p. 202–12.  [DOI](https://doi.org/10.1016/j.jsat.2014.04.006) | [PMC free article](/articles/PMC4180514/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24953168/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Subst%20Abuse%20Treat&title=Emerging%20adult%20age%20status%20predicts%20poor%20buprenorphine%20treatment%20retention&author=Z%20Schuman-Olivier&volume=47&issue=3&publication_year=2014&pages=202-12&pmid=24953168&doi=10.1016/j.jsat.2014.04.006&)

5. Huhn AS, Berry MS, and Dunn KE, Review: Sex-Based Differences in Treatment Outcomes for Persons With Opioid Use Disorder. Am J Addict, 2019. 28(4): p. 246–261.  [DOI](https://doi.org/10.1111/ajad.12921) | [PMC free article](/articles/PMC6591072/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31131505/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Addict&title=Review:%20Sex-Based%20Differences%20in%20Treatment%20Outcomes%20for%20Persons%20With%20Opioid%20Use%20Disorder&author=AS%20Huhn&author=MS%20Berry&author=KE%20Dunn&volume=28&issue=4&publication_year=2019&pages=246-261&pmid=31131505&doi=10.1111/ajad.12921&)

6. Hser YI, et al. , Treatment retention among patients randomized to buprenorphine/naloxone compared to methadone in a multi-site trial. Addiction, 2014. 109(1): p. 79–87.  [DOI](https://doi.org/10.1111/add.12333) | [PMC free article](/articles/PMC3947022/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23961726/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addiction&title=Treatment%20retention%20among%20patients%20randomized%20to%20buprenorphine/naloxone%20compared%20to%20methadone%20in%20a%20multi-site%20trial&author=YI%20Hser&volume=109&issue=1&publication_year=2014&pages=79-87&pmid=23961726&doi=10.1111/add.12333&)

7. Stein MD, Cioe P, and Friedmann PD, Buprenorphine retention in primary care. J Gen Intern Med, 2005. 20(11): p. 1038–41.  [DOI](https://doi.org/10.1111/j.1525-1497.2005.0228.x) | [PMC free article](/articles/PMC1490248/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16307630/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gen%20Intern%20Med&title=Buprenorphine%20retention%20in%20primary%20care&author=MD%20Stein&author=P%20Cioe&author=PD%20Friedmann&volume=20&issue=11&publication_year=2005&pages=1038-41&pmid=16307630&doi=10.1111/j.1525-1497.2005.0228.x&)

8. Potter JS, et al. , Buprenorphine/naloxone and methadone maintenance treatment outcomes for opioid analgesic, heroin, and combined users: findings from starting treatment with agonist replacement therapies (START). J Stud Alcohol Drugs, 2013. 74(4): p. 605–13.  [DOI](https://doi.org/10.15288/jsad.2013.74.605) | [PMC free article](/articles/PMC3711351/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23739025/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Stud%20Alcohol%20Drugs&title=Buprenorphine/naloxone%20and%20methadone%20maintenance%20treatment%20outcomes%20for%20opioid%20analgesic,%20heroin,%20and%20combined%20users:%20findings%20from%20starting%20treatment%20with%20agonist%20replacement%20therapies%20(START)&author=JS%20Potter&volume=74&issue=4&publication_year=2013&pages=605-13&pmid=23739025&doi=10.15288/jsad.2013.74.605&)

9. Dreifuss JA, et al. , Patient characteristics associated with buprenorphine/naloxone treatment outcome for prescription opioid dependence: Results from a multisite study. Drug Alcohol Depend, 2013. 131(1–2): p. 112–8.  [DOI](https://doi.org/10.1016/j.drugalcdep.2012.12.010) | [PMC free article](/articles/PMC3638044/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23333292/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depend&title=Patient%20characteristics%20associated%20with%20buprenorphine/naloxone%20treatment%20outcome%20for%20prescription%20opioid%20dependence:%20Results%20from%20a%20multisite%20study&author=JA%20Dreifuss&volume=131&issue=1%E2%80%932&publication_year=2013&pages=112-8&pmid=23333292&doi=10.1016/j.drugalcdep.2012.12.010&)

10. Savant JD, et al. , Prevalence of mood and substance use disorders among patients seeking primary care office-based buprenorphine/naloxone treatment. Drug Alcohol Depend, 2013. 127(1–3): p. 243–7.  [DOI](https://doi.org/10.1016/j.drugalcdep.2012.06.020) | [PMC free article](/articles/PMC3525769/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22771144/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depend&title=Prevalence%20of%20mood%20and%20substance%20use%20disorders%20among%20patients%20seeking%20primary%20care%20office-based%20buprenorphine/naloxone%20treatment&author=JD%20Savant&volume=127&issue=1%E2%80%933&publication_year=2013&pages=243-7&pmid=22771144&doi=10.1016/j.drugalcdep.2012.06.020&)

11. Gerra G, et al. , Buprenorphine versus methadone for opioid dependence: predictor variables for treatment outcome. Drug Alcohol Depend, 2004. 75(1): p. 37–45.  [DOI](https://doi.org/10.1016/j.drugalcdep.2003.11.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15225887/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depend&title=Buprenorphine%20versus%20methadone%20for%20opioid%20dependence:%20predictor%20variables%20for%20treatment%20outcome&author=G%20Gerra&volume=75&issue=1&publication_year=2004&pages=37-45&pmid=15225887&doi=10.1016/j.drugalcdep.2003.11.017&)

12. Litz M and Leslie D, The impact of mental health comorbidities on adherence to buprenorphine: A claims based analysis. Am J Addict, 2017. 26(8): p. 859–863.  [DOI](https://doi.org/10.1111/ajad.12644) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29143483/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Addict&title=The%20impact%20of%20mental%20health%20comorbidities%20on%20adherence%20to%20buprenorphine:%20A%20claims%20based%20analysis&author=M%20Litz&author=D%20Leslie&volume=26&issue=8&publication_year=2017&pages=859-863&pmid=29143483&doi=10.1111/ajad.12644&)

13. Peckham AD, et al. , Depression history as a predictor of outcomes during buprenorphine-naloxone treatment of prescription opioid use disorder. Drug Alcohol Depend, 2020. 213: p. 108122.  [DOI](https://doi.org/10.1016/j.drugalcdep.2020.108122) | [PMC free article](/articles/PMC7736247/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32563846/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depend&title=Depression%20history%20as%20a%20predictor%20of%20outcomes%20during%20buprenorphine-naloxone%20treatment%20of%20prescription%20opioid%20use%20disorder&author=AD%20Peckham&volume=213&publication_year=2020&pages=108122&pmid=32563846&doi=10.1016/j.drugalcdep.2020.108122&)

14. Jones CM and McCance-Katz EF, Co-occurring substance use and mental disorders among adults with opioid use disorder. Drug Alcohol Depend, 2019. 197: p. 78–82.  [DOI](https://doi.org/10.1016/j.drugalcdep.2018.12.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30784952/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Alcohol%20Depend&title=Co-occurring%20substance%20use%20and%20mental%20disorders%20among%20adults%20with%20opioid%20use%20disorder&author=CM%20Jones&author=EF%20McCance-Katz&volume=197&publication_year=2019&pages=78-82&pmid=30784952&doi=10.1016/j.drugalcdep.2018.12.030&)

15. Ferri M, et al. , Predictive factors for relapse in patients on buprenorphine maintenance. Am J Addict, 2014. 23(1): p. 62–7.  [DOI](https://doi.org/10.1111/j.1521-0391.2013.12074.x) | [PMC free article](/articles/PMC3922612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24313243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Addict&title=Predictive%20factors%20for%20relapse%20in%20patients%20on%20buprenorphine%20maintenance&author=M%20Ferri&volume=23&issue=1&publication_year=2014&pages=62-7&pmid=24313243&doi=10.1111/j.1521-0391.2013.12074.x&)

16. Sullivan LE, et al. , The association between cocaine use and treatment outcomes in patients receiving office-based buprenorphine/naloxone for the treatment of opioid dependence. Am J Addict, 2010. 19(1): p. 53–8.  [DOI](https://doi.org/10.1111/j.1521-0391.2009.00003.x) | [PMC free article](/articles/PMC3107713/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20132122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Addict&title=The%20association%20between%20cocaine%20use%20and%20treatment%20outcomes%20in%20patients%20receiving%20office-based%20buprenorphine/naloxone%20for%20the%20treatment%20of%20opioid%20dependence&author=LE%20Sullivan&volume=19&issue=1&publication_year=2010&pages=53-8&pmid=20132122&doi=10.1111/j.1521-0391.2009.00003.x&)

17. Hull LE, Lynch KG, and Oslin DW, VA Primary Care and Mental Health Providers’ Comfort with Genetic Testing: Survey Results from the PRIME Care Study. J Gen Intern Med, 2019. 34(6): p. 799–801.  [DOI](https://doi.org/10.1007/s11606-018-4776-0) | [PMC free article](/articles/PMC6544768/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30604117/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gen%20Intern%20Med&title=VA%20Primary%20Care%20and%20Mental%20Health%20Providers%E2%80%99%20Comfort%20with%20Genetic%20Testing:%20Survey%20Results%20from%20the%20PRIME%20Care%20Study&author=LE%20Hull&author=KG%20Lynch&author=DW%20Oslin&volume=34&issue=6&publication_year=2019&pages=799-801&pmid=30604117&doi=10.1007/s11606-018-4776-0&)

18. Meaden CW, et al. , A review of the existing literature on buprenorphine pharmacogenomics. Pharmacogenomics J, 2020.  [DOI](https://doi.org/10.1038/s41397-020-00198-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33154520/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=A%20review%20of%20the%20existing%20literature%20on%20buprenorphine%20pharmacogenomics&author=CW%20Meaden&publication_year=2020&pmid=33154520&doi=10.1038/s41397-020-00198-1&)

19. de Cid R, et al. , BDNF variability in opioid addicts and response to methadone treatment: preliminary findings. Genes Brain Behav, 2008. 7(5): p. 515–22.  [DOI](https://doi.org/10.1111/j.1601-183X.2007.00386.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18182069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genes%20Brain%20Behav&title=BDNF%20variability%20in%20opioid%20addicts%20and%20response%20to%20methadone%20treatment:%20preliminary%20findings&author=R%20de%20Cid&volume=7&issue=5&publication_year=2008&pages=515-22&pmid=18182069&doi=10.1111/j.1601-183X.2007.00386.x&)

20. Fonseca F, et al. , ALDH5A1 variability in opioid dependent patients could influence response to methadone treatment. Eur Neuropsychopharmacol, 2014. 24(3): p. 420–4.  [DOI](https://doi.org/10.1016/j.euroneuro.2013.10.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24230997/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20Neuropsychopharmacol&title=ALDH5A1%20variability%20in%20opioid%20dependent%20patients%20could%20influence%20response%20to%20methadone%20treatment&author=F%20Fonseca&volume=24&issue=3&publication_year=2014&pages=420-4&pmid=24230997&doi=10.1016/j.euroneuro.2013.10.003&)

21. Fonseca F, et al. , Response to methadone maintenance treatment is associated with the MYOCD and GRM6 genes. Mol Diagn Ther, 2010. 14(3): p. 171–8.  [DOI](https://doi.org/10.1007/BF03256370) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20560679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Diagn%20Ther&title=Response%20to%20methadone%20maintenance%20treatment%20is%20associated%20with%20the%20MYOCD%20and%20GRM6%20genes&author=F%20Fonseca&volume=14&issue=3&publication_year=2010&pages=171-8&pmid=20560679&doi=10.1007/BF03256370&)

22. Crettol S, et al. , Association of dopamine and opioid receptor genetic polymorphisms with response to methadone maintenance treatment. Prog Neuropsychopharmacol Biol Psychiatry, 2008. 32(7): p. 1722–7.  [DOI](https://doi.org/10.1016/j.pnpbp.2008.07.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18687376/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Neuropsychopharmacol%20Biol%20Psychiatry&title=Association%20of%20dopamine%20and%20opioid%20receptor%20genetic%20polymorphisms%20with%20response%20to%20methadone%20maintenance%20treatment&author=S%20Crettol&volume=32&issue=7&publication_year=2008&pages=1722-7&pmid=18687376&doi=10.1016/j.pnpbp.2008.07.009&)

23. Oneda B, et al. , beta-Arrestin2 influences the response to methadone in opioid-dependent patients. Pharmacogenomics J, 2011. 11(4): p. 258–66.  [DOI](https://doi.org/10.1038/tpj.2010.37) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20514076/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=beta-Arrestin2%20influences%20the%20response%20to%20methadone%20in%20opioid-dependent%20patients&author=B%20Oneda&volume=11&issue=4&publication_year=2011&pages=258-66&pmid=20514076&doi=10.1038/tpj.2010.37&)

24. Crist RC, et al. , A polymorphism in the OPRM1 3’-untranslated region is associated with methadone efficacy in treating opioid dependence. Pharmacogenomics J, 2016.  [DOI](https://doi.org/10.1038/tpj.2016.89) | [PMC free article](/articles/PMC5468510/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27958381/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=A%20polymorphism%20in%20the%20OPRM1%203%E2%80%99-untranslated%20region%20is%20associated%20with%20methadone%20efficacy%20in%20treating%20opioid%20dependence&author=RC%20Crist&publication_year=2016&pmid=27958381&doi=10.1038/tpj.2016.89&)

25. Gerra G, et al. , Association between gene variants and response to buprenorphine maintenance treatment. Psychiatry Res, 2014. 215(1): p. 202–7.  [DOI](https://doi.org/10.1016/j.psychres.2013.11.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24274990/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Psychiatry%20Res&title=Association%20between%20gene%20variants%20and%20response%20to%20buprenorphine%20maintenance%20treatment&author=G%20Gerra&volume=215&issue=1&publication_year=2014&pages=202-7&pmid=24274990&doi=10.1016/j.psychres.2013.11.001&)

26. Clarke TK, et al. , Genetic variation in OPRD1 and the response to treatment for opioid dependence with buprenorphine in European-American females. Pharmacogenomics J, 2014. 14(3): p. 303–8.  [DOI](https://doi.org/10.1038/tpj.2013.30) | [PMC free article](/articles/PMC3988270/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24126707/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Genetic%20variation%20in%20OPRD1%20and%20the%20response%20to%20treatment%20for%20opioid%20dependence%20with%20buprenorphine%20in%20European-American%20females&author=TK%20Clarke&volume=14&issue=3&publication_year=2014&pages=303-8&pmid=24126707&doi=10.1038/tpj.2013.30&)

27. Crist RC, et al. , An intronic variant in OPRD1 predicts treatment outcome for opioid dependence in African-Americans. Neuropsychopharmacology, 2013. 38(10): p. 2003–10.  [DOI](https://doi.org/10.1038/npp.2013.99) | [PMC free article](/articles/PMC3746708/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23612435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=An%20intronic%20variant%20in%20OPRD1%20predicts%20treatment%20outcome%20for%20opioid%20dependence%20in%20African-Americans&author=RC%20Crist&volume=38&issue=10&publication_year=2013&pages=2003-10&pmid=23612435&doi=10.1038/npp.2013.99&)

28. Crist RC, et al. , Replication of the pharmacogenetic effect of rs678849 on buprenorphine efficacy in African-Americans with opioid use disorder. Pharmacogenomics J, 2019. 19(3): p. 260–268.  [DOI](https://doi.org/10.1038/s41397-018-0065-x) | [PMC free article](/articles/PMC6486881/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30368523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Replication%20of%20the%20pharmacogenetic%20effect%20of%20rs678849%20on%20buprenorphine%20efficacy%20in%20African-Americans%20with%20opioid%20use%20disorder&author=RC%20Crist&volume=19&issue=3&publication_year=2019&pages=260-268&pmid=30368523&doi=10.1038/s41397-018-0065-x&)

29. Kranzler HR, et al. , A Delta-Opioid Receptor Gene Polymorphism Moderates the Therapeutic Response to Extended-Release Buprenorphine in Opioid Use Disorder. Int J Neuropsychopharmacol, 2020.  [DOI](https://doi.org/10.1093/ijnp/pyaa069) | [PMC free article](/articles/PMC7883889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32920647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Neuropsychopharmacol&title=A%20Delta-Opioid%20Receptor%20Gene%20Polymorphism%20Moderates%20the%20Therapeutic%20Response%20to%20Extended-Release%20Buprenorphine%20in%20Opioid%20Use%20Disorder&author=HR%20Kranzler&publication_year=2020&pmid=32920647&doi=10.1093/ijnp/pyaa069&)

30. Maranville JC and Cox NJ, Pharmacogenomic variants have larger effect sizes than genetic variants associated with other dichotomous complex traits. Pharmacogenomics J, 2016. 16(4): p. 388–92.  [DOI](https://doi.org/10.1038/tpj.2015.47) | [PMC free article](/articles/PMC4704992/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26149738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Pharmacogenomic%20variants%20have%20larger%20effect%20sizes%20than%20genetic%20variants%20associated%20with%20other%20dichotomous%20complex%20traits&author=JC%20Maranville&author=NJ%20Cox&volume=16&issue=4&publication_year=2016&pages=388-92&pmid=26149738&doi=10.1038/tpj.2015.47&)

31. Smith AH, et al. , Genome-wide association study of therapeutic opioid dosing identifies a novel locus upstream of OPRM1. Mol Psychiatry, 2017. 22(3): p. 346–352.  [DOI](https://doi.org/10.1038/mp.2016.257) | [PMC free article](/articles/PMC5407902/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28115739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Psychiatry&title=Genome-wide%20association%20study%20of%20therapeutic%20opioid%20dosing%20identifies%20a%20novel%20locus%20upstream%20of%20OPRM1&author=AH%20Smith&volume=22&issue=3&publication_year=2017&pages=346-352&pmid=28115739&doi=10.1038/mp.2016.257&)

32. Gaziano JM, et al. , Million Veteran Program: A mega-biobank to study genetic influences on health and disease. J Clin Epidemiol, 2016. 70: p. 214–23.  [DOI](https://doi.org/10.1016/j.jclinepi.2015.09.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26441289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Epidemiol&title=Million%20Veteran%20Program:%20A%20mega-biobank%20to%20study%20genetic%20influences%20on%20health%20and%20disease&author=JM%20Gaziano&volume=70&publication_year=2016&pages=214-23&pmid=26441289&doi=10.1016/j.jclinepi.2015.09.016&)

33. Hunter-Zinck H, et al. , Measuring genetic variation in the multi-ethnic Million Veteran Program (MVP). bioRxiv, 2020: p. 2020.01.06.896613.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=bioRxiv&title=Measuring%20genetic%20variation%20in%20the%20multi-ethnic%20Million%20Veteran%20Program%20(MVP)&author=H%20Hunter-Zinck&publication_year=2020&)

34. Marchini J and Howie B, Genotype imputation for genome-wide association studies. Nat Rev Genet, 2010. 11(7): p. 499–511.  [DOI](https://doi.org/10.1038/nrg2796) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20517342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Genotype%20imputation%20for%20genome-wide%20association%20studies&author=J%20Marchini&author=B%20Howie&volume=11&issue=7&publication_year=2010&pages=499-511&pmid=20517342&doi=10.1038/nrg2796&)

35. Auton A, et al. , A global reference for human genetic variation. Nature, 2015. 526(7571): p. 68–74.  [DOI](https://doi.org/10.1038/nature15393) | [PMC free article](/articles/PMC4750478/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26432245/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20global%20reference%20for%20human%20genetic%20variation&author=A%20Auton&volume=526&issue=7571&publication_year=2015&pages=68-74&pmid=26432245&doi=10.1038/nature15393&)

36. Fang H, et al. , Harmonizing Genetic Ancestry and Self-identified Race/Ethnicity in Genome-wide Association Studies. Am J Hum Genet, 2019. 105(4): p. 763–772.  [DOI](https://doi.org/10.1016/j.ajhg.2019.08.012) | [PMC free article](/articles/PMC6817526/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31564439/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Harmonizing%20Genetic%20Ancestry%20and%20Self-identified%20Race/Ethnicity%20in%20Genome-wide%20Association%20Studies&author=H%20Fang&volume=105&issue=4&publication_year=2019&pages=763-772&pmid=31564439&doi=10.1016/j.ajhg.2019.08.012&)

37. de Leeuw CA, et al. , MAGMA: generalized gene-set analysis of GWAS data. PLoS Comput Biol, 2015. 11(4): p. e1004219.  [DOI](https://doi.org/10.1371/journal.pcbi.1004219) | [PMC free article](/articles/PMC4401657/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25885710/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&title=MAGMA:%20generalized%20gene-set%20analysis%20of%20GWAS%20data&author=CA%20de%20Leeuw&volume=11&issue=4&publication_year=2015&pages=e1004219&pmid=25885710&doi=10.1371/journal.pcbi.1004219&)

38. Watanabe K, et al. , Functional mapping and annotation of genetic associations with FUMA. Nat Commun, 2017. 8(1): p. 1826.  [DOI](https://doi.org/10.1038/s41467-017-01261-5) | [PMC free article](/articles/PMC5705698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29184056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Functional%20mapping%20and%20annotation%20of%20genetic%20associations%20with%20FUMA&author=K%20Watanabe&volume=8&issue=1&publication_year=2017&pages=1826&pmid=29184056&doi=10.1038/s41467-017-01261-5&)

39. Albertson DN, et al. , Gene expression profile of the nucleus accumbens of human cocaine abusers: evidence for dysregulation of myelin. J Neurochem, 2004. 88(5): p. 1211–9.  [DOI](https://doi.org/10.1046/j.1471-4159.2003.02247.x) | [PMC free article](/articles/PMC2215309/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15009677/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Neurochem&title=Gene%20expression%20profile%20of%20the%20nucleus%20accumbens%20of%20human%20cocaine%20abusers:%20evidence%20for%20dysregulation%20of%20myelin&author=DN%20Albertson&volume=88&issue=5&publication_year=2004&pages=1211-9&pmid=15009677&doi=10.1046/j.1471-4159.2003.02247.x&)

40. Albertson DN, et al. , Distinctive profiles of gene expression in the human nucleus accumbens associated with cocaine and heroin abuse. Neuropsychopharmacology, 2006. 31(10): p. 2304–12.  [DOI](https://doi.org/10.1038/sj.npp.1301089) | [PMC free article](/articles/PMC2239258/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16710320/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropsychopharmacology&title=Distinctive%20profiles%20of%20gene%20expression%20in%20the%20human%20nucleus%20accumbens%20associated%20with%20cocaine%20and%20heroin%20abuse&author=DN%20Albertson&volume=31&issue=10&publication_year=2006&pages=2304-12&pmid=16710320&doi=10.1038/sj.npp.1301089&)

41. The GTEx Consortium, Human genomics. The Genotype-Tissue Expression (GTEx) pilot analysis: multitissue gene regulation in humans. Science, 2015. 348(6235): p. 648–60.  [DOI](https://doi.org/10.1126/science.1262110) | [PMC free article](/articles/PMC4547484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25954001/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Human%20genomics.%20The%20Genotype-Tissue%20Expression%20(GTEx)%20pilot%20analysis:%20multitissue%20gene%20regulation%20in%20humans&volume=348&issue=6235&publication_year=2015&pages=648-60&pmid=25954001&doi=10.1126/science.1262110&)

42. Huo YX, et al. , Identification of SLC25A37 as a major depressive disorder risk gene. J Psychiatr Res, 2016. 83: p. 168–175.  [DOI](https://doi.org/10.1016/j.jpsychires.2016.09.011) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27643475/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Psychiatr%20Res&title=Identification%20of%20SLC25A37%20as%20a%20major%20depressive%20disorder%20risk%20gene&author=YX%20Huo&volume=83&publication_year=2016&pages=168-175&pmid=27643475&doi=10.1016/j.jpsychires.2016.09.011&)

43. Peterson RE, et al. , Molecular Genetic Analysis Subdivided by Adversity Exposure Suggests Etiologic Heterogeneity in Major Depression. Am J Psychiatry, 2018. 175(6): p. 545–554.  [DOI](https://doi.org/10.1176/appi.ajp.2017.17060621) | [PMC free article](/articles/PMC5988935/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29495898/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Psychiatry&title=Molecular%20Genetic%20Analysis%20Subdivided%20by%20Adversity%20Exposure%20Suggests%20Etiologic%20Heterogeneity%20in%20Major%20Depression&author=RE%20Peterson&volume=175&issue=6&publication_year=2018&pages=545-554&pmid=29495898&doi=10.1176/appi.ajp.2017.17060621&)

44. Ghabrash MF, et al. , Depression and Outcomes of Methadone and Buprenorphine Treatment Among People with Opioid Use Disorders: A Literature Review. J Dual Diagn, 2020. 16(2): p. 191–207.  [DOI](https://doi.org/10.1080/15504263.2020.1726549) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32089124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Dual%20Diagn&title=Depression%20and%20Outcomes%20of%20Methadone%20and%20Buprenorphine%20Treatment%20Among%20People%20with%20Opioid%20Use%20Disorders:%20A%20Literature%20Review&author=MF%20Ghabrash&volume=16&issue=2&publication_year=2020&pages=191-207&pmid=32089124&doi=10.1080/15504263.2020.1726549&)

45. Quach BC, et al. , Expanding the genetic architecture of nicotine dependence and its shared genetics with multiple traits. Nat Commun, 2020. 11(1): p. 5562.  [DOI](https://doi.org/10.1038/s41467-020-19265-z) | [PMC free article](/articles/PMC7642344/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33144568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Expanding%20the%20genetic%20architecture%20of%20nicotine%20dependence%20and%20its%20shared%20genetics%20with%20multiple%20traits&author=BC%20Quach&volume=11&issue=1&publication_year=2020&pages=5562&pmid=33144568&doi=10.1038/s41467-020-19265-z&)

46. Jackson RM, et al. , Weight Loss Results in Increased Expression of Anti-Inflammatory Protein CRISPLD2 in Mouse Adipose Tissue. Obesity (Silver Spring), 2019. 27(12): p. 2025–2036.  [DOI](https://doi.org/10.1002/oby.22652) | [PMC free article](/articles/PMC6873817/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31746554/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Obesity%20(Silver%20Spring)&title=Weight%20Loss%20Results%20in%20Increased%20Expression%20of%20Anti-Inflammatory%20Protein%20CRISPLD2%20in%20Mouse%20Adipose%20Tissue&author=RM%20Jackson&volume=27&issue=12&publication_year=2019&pages=2025-2036&pmid=31746554&doi=10.1002/oby.22652&)

47. Zhu Z, et al. , Shared genetic and experimental links between obesity-related traits and asthma subtypes in UK Biobank. J Allergy Clin Immunol, 2020. 145(2): p. 537–549.  [DOI](https://doi.org/10.1016/j.jaci.2019.09.035) | [PMC free article](/articles/PMC7010560/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31669095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Shared%20genetic%20and%20experimental%20links%20between%20obesity-related%20traits%20and%20asthma%20subtypes%20in%20UK%20Biobank&author=Z%20Zhu&volume=145&issue=2&publication_year=2020&pages=537-549&pmid=31669095&doi=10.1016/j.jaci.2019.09.035&)

48. Zhou H, et al. , Association of OPRM1 Functional Coding Variant With Opioid Use Disorder: A Genome-Wide Association Study. JAMA Psychiatry, 2020.  [DOI](https://doi.org/10.1001/jamapsychiatry.2020.1206) | [PMC free article](/articles/PMC7270886/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32492095/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Psychiatry&title=Association%20of%20OPRM1%20Functional%20Coding%20Variant%20With%20Opioid%20Use%20Disorder:%20A%20Genome-Wide%20Association%20Study&author=H%20Zhou&publication_year=2020&pmid=32492095&doi=10.1001/jamapsychiatry.2020.1206&)

49. Ripke S, Biological insights from 108 schizophrenia-associated genetic loci. Nature, 2014. 511(7510): p. 421–7.  [DOI](https://doi.org/10.1038/nature13595) | [PMC free article](/articles/PMC4112379/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25056061/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Biological%20insights%20from%20108%20schizophrenia-associated%20genetic%20loci&author=S%20Ripke&volume=511&issue=7510&publication_year=2014&pages=421-7&pmid=25056061&doi=10.1038/nature13595&)

50. Kranzler HR, et al. , Genome-wide association study of alcohol consumption and use disorder in 274,424 individuals from multiple populations. Nat Commun, 2019. 10(1): p. 1499.  [DOI](https://doi.org/10.1038/s41467-019-09480-8) | [PMC free article](/articles/PMC6445072/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30940813/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Genome-wide%20association%20study%20of%20alcohol%20consumption%20and%20use%20disorder%20in%20274,424%20individuals%20from%20multiple%20populations&author=HR%20Kranzler&volume=10&issue=1&publication_year=2019&pages=1499&pmid=30940813&doi=10.1038/s41467-019-09480-8&)

51. Hou L, et al. , Genetic variants associated with response to lithium treatment in bipolar disorder: a genome-wide association study. Lancet, 2016. 387(10023): p. 1085–1093.  [DOI](https://doi.org/10.1016/S0140-6736(16)00143-4) | [PMC free article](/articles/PMC4814312/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26806518/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genetic%20variants%20associated%20with%20response%20to%20lithium%20treatment%20in%20bipolar%20disorder:%20a%20genome-wide%20association%20study&author=L%20Hou&volume=387&issue=10023&publication_year=2016&pages=1085-1093&pmid=26806518&doi=10.1016/S0140-6736(16)00143-4&)

52. Allen JD and Bishop JR, A systematic review of genome-wide association studies of antipsychotic response. Pharmacogenomics, 2019. 20(4): p. 291–306.  [DOI](https://doi.org/10.2217/pgs-2018-0163) | [PMC free article](/articles/PMC6563266/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30883267/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=A%20systematic%20review%20of%20genome-wide%20association%20studies%20of%20antipsychotic%20response&author=JD%20Allen&author=JR%20Bishop&volume=20&issue=4&publication_year=2019&pages=291-306&pmid=30883267&doi=10.2217/pgs-2018-0163&)

53. Armstrong GL, et al. , The prevalence of hepatitis C virus infection in the United States, 1999 through 2002. Ann Intern Med, 2006. 144(10): p. 705–14.  [DOI](https://doi.org/10.7326/0003-4819-144-10-200605160-00004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16702586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Intern%20Med&title=The%20prevalence%20of%20hepatitis%20C%20virus%20infection%20in%20the%20United%20States,%201999%20through%202002&author=GL%20Armstrong&volume=144&issue=10&publication_year=2006&pages=705-14&pmid=16702586&doi=10.7326/0003-4819-144-10-200605160-00004&)

54. May MT, et al. , Injection Drug Use and Hepatitis C as Risk Factors for Mortality in HIV-Infected Individuals: The Antiretroviral Therapy Cohort Collaboration. J Acquir Immune Defic Syndr, 2015. 69(3): p. 348–54.  [DOI](https://doi.org/10.1097/QAI.0000000000000603) | [PMC free article](/articles/PMC4506784/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25848927/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Acquir%20Immune%20Defic%20Syndr&title=Injection%20Drug%20Use%20and%20Hepatitis%20C%20as%20Risk%20Factors%20for%20Mortality%20in%20HIV-Infected%20Individuals:%20The%20Antiretroviral%20Therapy%20Cohort%20Collaboration&author=MT%20May&volume=69&issue=3&publication_year=2015&pages=348-54&pmid=25848927&doi=10.1097/QAI.0000000000000603&)

55. Ledgerwood DM, et al. , Injection opioid use as a predictor of treatment outcomes among methadone-maintained opioid-dependent patients. Addict Behav, 2019. 90: p. 191–195.  [DOI](https://doi.org/10.1016/j.addbeh.2018.10.046) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30412910/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addict%20Behav&title=Injection%20opioid%20use%20as%20a%20predictor%20of%20treatment%20outcomes%20among%20methadone-maintained%20opioid-dependent%20patients&author=DM%20Ledgerwood&volume=90&publication_year=2019&pages=191-195&pmid=30412910&doi=10.1016/j.addbeh.2018.10.046&)

56. Cox J, et al. , Predictors of methadone program non-retention for opioid analgesic dependent patients. J Subst Abuse Treat, 2013. 44(1): p. 52–60.  [DOI](https://doi.org/10.1016/j.jsat.2012.03.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22538172/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Subst%20Abuse%20Treat&title=Predictors%20of%20methadone%20program%20non-retention%20for%20opioid%20analgesic%20dependent%20patients&author=J%20Cox&volume=44&issue=1&publication_year=2013&pages=52-60&pmid=22538172&doi=10.1016/j.jsat.2012.03.002&)

57. Novak SP and Kral AH, Comparing injection and non-injection routes of administration for heroin, methamphetamine, and cocaine users in the United States. J Addict Dis, 2011. 30(3): p. 248–57.  [DOI](https://doi.org/10.1080/10550887.2011.581989) | [PMC free article](/articles/PMC3225003/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21745047/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Addict%20Dis&title=Comparing%20injection%20and%20non-injection%20routes%20of%20administration%20for%20heroin,%20methamphetamine,%20and%20cocaine%20users%20in%20the%20United%20States&author=SP%20Novak&author=AH%20Kral&volume=30&issue=3&publication_year=2011&pages=248-57&pmid=21745047&doi=10.1080/10550887.2011.581989&)

58. Jones CM, Trends and key correlates of prescription opioid injection misuse in the United States. Addict Behav, 2018. 78: p. 145–152.  [DOI](https://doi.org/10.1016/j.addbeh.2017.10.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29175290/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Addict%20Behav&title=Trends%20and%20key%20correlates%20of%20prescription%20opioid%20injection%20misuse%20in%20the%20United%20States&author=CM%20Jones&volume=78&publication_year=2018&pages=145-152&pmid=29175290&doi=10.1016/j.addbeh.2017.10.018&)

59. Moore BA, et al. , Primary care office-based buprenorphine treatment: comparison of heroin and prescription opioid dependent patients. J Gen Intern Med, 2007. 22(4): p. 527–30.  [DOI](https://doi.org/10.1007/s11606-007-0129-0) | [PMC free article](/articles/PMC1829433/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17372805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gen%20Intern%20Med&title=Primary%20care%20office-based%20buprenorphine%20treatment:%20comparison%20of%20heroin%20and%20prescription%20opioid%20dependent%20patients&author=BA%20Moore&volume=22&issue=4&publication_year=2007&pages=527-30&pmid=17372805&doi=10.1007/s11606-007-0129-0&)
