# IL7 genetic variation and toxicity to immune checkpoint blockade in patients with melanoma

## Metadata
**Authors:** Chelsea A Taylor, Robert A Watson, Orion Tong, Weiyu Ye, Isar Nassiri, James J Gilchrist, Alba Verge de los Aires, Piyush Kumar Sharma, Surya Koturan, Rosalin A Cooper, Victoria K Woodcock, Elsita Jungkurth, Brian Shine, Nicholas Coupe, Miranda J Payne, David N Church, Vivek Naranbhai, Stefan Groha, Paul Emery, Kulveer Mankia, Matthew L Freedman, Toni K Choueiri, Mark R Middleton, Alexander Gusev, Benjamin P Fairfax
**Journal:** Nature Medicine
**Date:** 2022 Dec 16
**DOI:** [10.1038/s41591-022-02095-5](https://doi.org/10.1038/s41591-022-02095-5)
**PMID:** 36526722
**PMCID:** PMC9800275
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9800275/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9800275/pdf/41591_2022_Article_2095.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9800275/pdf/41591_2022_Article_2095.pdf)

## Abstract

Treatment with immune checkpoint blockade (ICB) frequently triggers immune-related adverse events (irAEs), causing considerable morbidity. In 214 patients receiving ICB for melanoma, we observed increased severe irAE risk in minor allele carriers of rs16906115, intronic to IL7. We found that rs16906115 forms a B cell-specific expression quantitative trait locus (eQTL) to IL7 in patients. Patients carrying the risk allele demonstrate increased pre-treatment B cell IL7 expression, which independently associates with irAE risk, divergent immunoglobulin expression and more B cell receptor mutations. Consistent with the role of IL-7 in T cell development, risk allele carriers have distinct ICB-induced CD8+ T cell subset responses, skewing of T cell clonality and greater proportional repertoire occupancy by large clones. Finally, analysis of TCGA data suggests that risk allele carriers independently have improved melanoma survival. These observations highlight key roles for B cells and IL-7 in both ICB response and toxicity and clinical outcomes in melanoma.

Subject terms: Genetics research, Melanoma

## Main

Immune checkpoint blockade (ICB), consisting of monoclonal antibodies targeted against the proteins PD-1 and CTLA-4, can elicit durable remission of metastatic melanoma in a subset of individuals^1–3^ but is associated with autoimmune complications (immune-related adverse events (irAEs)), for which risk factors are poorly characterized^4–6^. Both oncological outcomes and occurrence of irAEs after ICB treatment vary markedly between patients^1,7^. We postulate that irAE development can be influenced by cancer-independent factors, including germline genetics. A parallel genome-wide association study (GWAS) identified carriage of the minor allele of rs16906115, a non-coding polymorphism at *IL7*, to be associated with increased risk of ICB-associated irAEs. In the present study, we replicated this clinical association in a cohort of 214 patients receiving ICB for melanoma, finding that carriers of the minor allele of rs16906115 (referred to as the risk allele) have significantly increased risk of toxicities requiring corticosteroid treatment. RNA sequencing (RNA-seq) performed on isolated immune cell types from patients pre-ICB and post-ICB treatment (*n* = 194), and healthy controls (*n* = 170), demonstrates markedly increased *IL7* expression in patient-derived B cells before treatment, with this effect being significantly greater in risk allele carriers. Notably, independent of genotype, we found that increased B cell *IL7* expression pre-treatment is associated with risk of irAEs, indicative of causality. By performing single-cell (sc) RNA-seq of B cells from patients and controls, we delineated B cell subsets expressing *IL7* and note that risk allele carriage is associated with increased B cell maturation. Finally, reflecting the importance of IL-7 in T cell maturation, we found genotype-specific T cell receptor (TCR) repertoire skewing with increased proportional repertoire occupancy by hyper-expanded CD8^+^ T cell clones in response to ICB treatment. Separate analysis of The Cancer Genome Atlas (TCGA) data demonstrates that the risk allele is associated with disease-specific and overall survival in melanoma, indicating that this pathway plays a role in the natural history of melanoma progression. In summary, we replicated and mechanistically dissected the effects of the leading identified risk locus for ICB-induced irAEs, implicating a key role for B cells and IL-7 in mediation of immunological responses and toxicity to these therapies.

## Results

### rs16906115 is a risk allele for irAE development post-ICB

We sought to replicate the reported association between rs16906115 and irAE development in a dataset of 214 prospectively recruited and genotyped patients who had received ICB for melanoma (Extended Data Fig. [1a](#Fig6)), in which rs16906115 had a minor allele frequency (MAF) of 7.4%. We found that carriage of the minor risk (A) allele (one or two copies) was associated with an increased likelihood of developing severe (grade 3 or higher) irAEs requiring corticosteroids before the fifth cycle (C5) of treatment across all patients, with an odds ratio (OR) of 2.24 (95% confidence interval (CI): 1.03–5.09, *P* = 0.046) (Fig. [1a](#Fig1)). In the primary irAE GWAS dataset, 90% of patients were treated with single-agent anti-PD-1 (sICB) or anti-PDL1, as opposed to combination anti-CTLA-4/anti-PD-1 (cICB). Given that sICB recipients have only ~20% the risk of cICB recipients for developing irAEs^1,3,7^, the sensitivity to detect genetic effects is likely greater with this more moderate treatment. In keeping with this, we found that sICB (anti-PD-1) recipients carrying the risk allele had a 6.0 OR (CI: 1.5–23.0, *P* = 0.0084) of developing irAEs requiring steroids before C5, as well as increased risk of requiring steroids at any timepoint on treatment (OR 4.4, 95% CI: 1.2–15.1, *P* = 0.019) (Fig. [1a](#Fig1)). The single-nucleotide polymorphism (SNP) rs16906115 is intronic to *IL7*, encoding the cytokine IL-7, a key lymphopoietic cytokine with pleiotropic effects across lymphocyte subsets^8–10^. We, therefore, explored the relationship between rs16906115 status and ICB-induced changes in lymphocyte count, as measured at routine hospital blood tests. Comparison of pre-treatment lymphocyte count with the first count measured at least 21 days after treatment initiation (referred to as lymphocyte stability (LS)) revealed that, although cICB treatment did not significantly affect LS, sICB treatment elicited a fall in LS (Extended Data Fig. [1b](#Fig6)), which was consistent across recipients of nivolumab and pembrolizumab (Extended Data Fig. [1c](#Fig6)). In particular, non-carriers of the risk allele had significantly reduced LS post-sICB (median pre-treatment value 1.82 × 10^9^ cells per liter, interquartile range (IQR) 1.26–2.33; post-treatment value 1.52 × 10^9^ cells per liter, IQR 1.07–2.11) (*P* = 0.00044; Fig. [1b](#Fig1)), a reduction not observed in risk allele carriers (median pre-treatment value 1.35 × 10^9^ cells per liter, IQR 1.11–2.72; post-treatment 1.53 × 10^9^ cells per liter, IQR 1.34–1.90) (*P* = 0.68; Fig. [1b](#Fig1)). Correspondingly, rs16906115 status was associated with the magnitude of sICB-induced change in lymphocyte count (non-carriers: −0.15 × 10^9^ cells per liter; IQR −0.38 to 0.02 versus carriers: −0.025 × 10^9^ cells per liter; IQR −0.11 to 0.36) (*P* = 0.023; Fig. [1c](#Fig1)). Notably, individuals with a fall in LS >20% had reduced progression-free and overall survival (Extended Data Fig. [1d,e](#Fig6)), reinforcing the clinical importance of LS and demonstrating an association between rs16906115 and post-treatment lymphopoiesis, in keeping with the known properties of IL-7.

### Extended Data Fig. 1. Extended Clinical Data.

![Extended Data Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e00/9800275/34e2c5edbddf/41591_2022_2095_Fig6_ESM.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9800275_41591_2022_2095_Fig6_ESM.jpg)

(a) Details of patient cohort, tests are as specified and were two-sided (b) Comparative response of lymphocyte count to cICB and sICB; two-sided Wilcoxon signed rank test, n = 134 patients (74 cICB, 60 sICB). c) Comparison of fall in lymphocyte count in recipients of nivolumab versus those of pembrolizumab; two-sided T tests (n = 74 ipi/nivo, n = 9 nivo, n = 51 pembrolizumab). (d) Kaplan Meier analysis of progression-free survival in immunotherapy naïve metastatic melanoma patients within the cohort according to LS (unstable defined as post-treatment lymphocyte count ≤80% pre-treatment value); log-rank test (n = 111). (e) As per (d), but curves represent overall survival across cohort according to LS (n = 111). For boxplots in Extended Data Fig. 1b, c central line reflects the median, the box reflects the 25%–75% quartiles, upper whisker extends to largest value no further than 1.5x interquartile range, lower whisker extends from 25% quartile to the smallest value no further than 1.5x interquartile range.

### Fig. 1. Association of rs16906115 with ICB-associated irAEs and flux in lymphocytes.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e00/9800275/346a32649410/41591_2022_2095_Fig1_HTML.jpg)

a, OR for developing irAEs requiring steroids before C5 of treatment, or ever, according to allele carriage of rs16906115 and treatment type, and all ICB. Whiskers represent 95% CI; two-sided Fisher exact test, n = 214 patients (98 sICB and 116 cICB). b, Pre-treatment versus first post-treatment lymphocyte count for patients receiving sICB, split by allele; two-sided Wilcoxon signed-rank test, n = 60 patients. c, Difference between pre-treatment and post-sICB treatment lymphocyte count by allele; one-sided Student’s t-test, n = 60 patients.

### rs16906115 regulates B cell IL7 expression in patients with melanoma

Plasma IL-7 concentration forms a rate-limiting determinant of T cell expansion, being regulated, in part, by peripheral lymphocyte consumption, including activated T cells^11^. In keeping with the high levels of peripheral T cell mitosis in melanoma^12–14^, plasma IL-7 levels were undetectable by ELISAs in patient samples. We, therefore, explored *IL7* expression across immune subsets isolated from patients and healthy individuals. Although circulating immune cells are not generally recognized to produce IL-7 (refs. ^11,15–17^), we found that peripheral blood isolated CD19^+^ B cells robustly express *IL7* at levels greatly in excess of other cell subsets. Notably, *IL7* expression in pre-treatment patient-derived B cells was significantly induced compared to that in B cells from healthy controls (94 controls and 92 patients; mean log_2_ expression 8.30 FKPM versus 8.87 FKPM, *P* = 5.9 × 10^−6^), with this effect not apparent across other subsets and independent of age (Fig. [2a](#Fig2) and Extended Data Fig. [2a](#Fig7)). This upregulation persisted in early post-ICB treatment samples, with neither ICB type affecting B cell *IL7* expression at day 21 (Extended Data Fig. [2b](#Fig7)). Crucially, we found that patient risk allele carriers at rs16906115 demonstrated increased B cell *IL7* induction versus non-carriers both pre-treatment and over the course of treatment, but genetic effects were not observed across other cell types in patients (Fig. [2b](#Fig2) and Extended Data Fig. [2c](#Fig7)). Conversely, genetic effects were not observed in B cells or other cell types in healthy individuals (Extended Data Fig. [2d](#Fig7)), indicating that rs16906115 forms a B cell *cis* expression quantitative trait locus (eQTL) in the context of melanoma-associated immune activity. We did not identify genotype-associated B cell *IL7* splice variation but, instead, observed upregulation of the main *IL7* transcripts, indicating activity through modulation of total *IL7* gene expression (Extended Data Fig. [2e](#Fig7)). Finally, using flow cytometry, we were able to consistently detect an IL-7^+^ population of B cells in peripheral blood mononuclear cells (PBMCs), with no IL-7^+^ cells detected in other major lineages (Extended Data Fig. [2f,g](#Fig7)). To further assess evidence for B cell *IL7* directly mediating the risk effect of rs16906115 on irAE development, we tested association using random effects mixed linear models^18^. Consistent with B cell *IL7* mediating the genotypic association, we found that, when controlling for ICB type and rs16906115 status, pre-treatment B cell *IL7* was independently associated with irAE risk (*P* = 0.0092), whereas, when controlling for B cell *IL7* and ICB type, no genotypic effect was apparent (Fig. [2c](#Fig2)). This remained the case when restricting analysis to individuals carrying the reference allele (*P* = 0.0087), underlying the importance of B cell *IL7* to irAE development and providing mechanistic insight into rs16906155 functionality.

### Fig. 2. B cell IL7 expression associates with rs16906115 allele and independently with irAE risk.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e00/9800275/c3e2a2222e4d/41591_2022_2095_Fig2_HTML.jpg)

a, Expression of IL7 measured using RNA-seq according to status (patient samples are untreated; healthy donor (HD)) split by cell type; P value from two-sided ANOVA, controlling for age (n refers to sample size for each cell type by HD or patient). b, Expression of IL7 by allelic carriage and patient status; patient samples represent untreated (C1, n = 91), after the second cycle of treatment (C2, n = 68), the earliest available later sample (late, n = 39) or HDs (n = 92); one-sided Wilcoxon rank-sum test. For box plots in a,b, the central line reflects the median; the box corresponds to 25–75% quartiles; the upper whisker extends to the largest value no farther than 1.5× IQR; and the lower whisker extends from the 25% quartile to the smallest value no farther than 1.5× IQR. c, Results from linear mixed model random effect analysis; center point shows the estimate for the variable in association with irAE development when controlling for the other two variables; ANOVA using the Satterthwaite method; error bars mark the 95% CI; n = 91 patients.

### Extended Data Fig. 2. IL7 Expression Across Cell Types.

![Extended Data Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e00/9800275/dfe294df2dc0/41591_2022_2095_Fig7_ESM.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9800275_41591_2022_2095_Fig7_ESM.jpg)

(a) Association between donor age and IL7 expression across cell types, stratified by status (HD – healthy donor); P value from two-sided ANOVA, correcting for status (CD14: n = 42 HD, 216 patient samples; CD19: n = 90 HD, 159 patient samples; CD4 naïve: n = 8 HD, 50 patient samples; CD8: n = 68 HD, 326 patient samples; NK: n = 31 HD, 120 patient samples). (b) B cell expression of IL7 in patient samples according to cycle and treatment type, no significant difference was noted with type or between baseline untreated and immediately prior to the second cycle of treatment (C2); two-sided T test, n = 113 cICB, n = 46 sICB. (c) B cell expression of IL7 split by treatment type according to rs16906115 carriage; one-sided Wilcoxon rank-sum test, C1: n = 64 cICB patients, n = 27 sICB patients; C2: n = 49 cICB patients, n = 19 sICB patients; late: n = 24 cICB patients, n = 15 sICB patients. (d) Expression of IL7 in patient and healthy donor (HD) samples dichotomized by rs16906115 status, faceted by cell type, across all samples in cohort, P value refers to result from two-sided T test between absence and presence of risk allele (one or two copies). (e) Expression of IL7 transcripts annotated using StringTie43 according to allele status across all B cell samples showing a pan transcript effect of rs16906115 carriage; two-sided Wilcoxon rank-sum test. (f) Representative flow plot showing PBMCs gated on expression of CD19 with forward scatter on the X-axis versus IL-7 positive events on the y-axis. Primary = primary biotinylated anti-IL-7, secondary = Streptavidin PE, FMO = fluorescence minus one. (g) Analysis of samples demonstrated consistent detection of IL-7 positive B cell population in PBMCs from patients (n = 2) and controls (n = 2), with IL-7 positive fractions not detected in cells gated for CD3 (T cells), CD56+, CD3- (NK cells) or CD14 (monocytes). For boxplots in Extended Data Fig. 2b–e, g; the central line reflects the median, the box 25%–75% quartiles, upper whisker extends to largest value no further than 1.5x interquartile range, lower whisker extends from 25% quartile to the smallest value no further than 1.5x interquartile range.

### B cell IL7 expression is associated with maturation

Maturation of B cells is influenced by IL-7 (refs. ^9,10^), and so we explored the relationship between B cell development and *IL7* expression. Analysis of B cells from healthy donors demonstrated correlation of *IL7* expression with 11,305 transcripts (adjusted *P* < 0.05), including the key B cell transcription factors *IKZF3* (ref. ^19^) and *POU2AF1* (refs. ^20,21^) (Fig. [3a](#Fig3) and Supplementary Table [2](#MOESM3)), with remarkably similar results observed in untreated patients (Supplementary Fig. [1c](#MOESM1)). Pathway analysis of B cell *IL7-*associated genes using Gene Ontology Biological Process (GOBP) datasets^22^ demonstrated enrichment in pathways associated with B cell maturation, including ‘immunoglobulin production’ and ‘positive regulation of B cell activation’, whereas anti-correlated genes were associated with TLR4 signaling, neutrophil degranulation and antigen presentation (Fig. [3b](#Fig3), Extended Data Fig. [3a](#Fig8) and Supplementary Table [3](#MOESM3)). Analysis of patient-derived B cells revealed that *IL7* expression was anti-correlated with expression of the naive immunoglobulin *IGHD* but was associated with the percentage of immunoglobulin chains carrying secondary divergent mutations, as observed in somatic hypermutation (Extended Data Fig. [3b](#Fig8)). Notably, in keeping with *IL7* expression driving these effects, we noted genotypic association of rs16906115 status, with risk allele carriers demonstrating an increased percentage of mutations in *IGH*, *IGK* and *IGL* chains in pre-treated patients (Fig. [3c](#Fig3)). Similarly, risk allele carriers showed reduced numbers of *IGHD-*expressing B cell clones pre-treatment (Fig. [3d](#Fig3)), indicating a pre-existent state of B cell immunoglobulin class switching and reduced B cell naivety in these individuals. To further explore *IL7* expression and genotypic association of rs16906115 across B cell subsets, we generated scRNA-seq data from B cells (*n* = 15,755 cells) from pre-treatment patients with metastatic melanoma (*n* = 24) and healthy controls (*n* = 5). We identified four major clusters of cells consisting of naive B cells, unswitched memory (USM) cells, switched memory (SM) cells and antibody secreting cells (ASCs) (Fig. [3e](#Fig3) and Extended Data Fig. [3c,d](#Fig8)). Expression of *IL7* was highest in the USM and SM clusters, intermediate in the naive cells and lowest in ASCs (Extended Data Fig. [3e,f](#Fig8)) and was significantly higher in patient samples compared to healthy controls across all subsets except ASCs (Fig. [3f](#Fig3)). Compared to those from healthy donors, patient naive B cells demonstrated reduced expression of *IGHD* and concomitant upregulation of *IGHM* (Extended Data Fig. [3g](#Fig8)). Although only four of the patient samples carried the risk allele, we again observed increased *IL7* expression in rs16906115 risk allele carriers across all B cells (Fig. [3g](#Fig3)), although, within subsets, there was no difference in median expression (Extended Data Fig. [3h](#Fig8)). Instead, we found that carriage of the risk allele was associated with proportional changes in B cell subset frequency, with the eQTL effect being driven by significantly increased counts of high *IL7*-expressing USM cells and a trend to fewer naive B cells (Fig. [3h](#Fig3)).

### Fig. 3. B cell IL7 expression is associated with pleiotropic effects.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e00/9800275/5cea73ca6352/41591_2022_2095_Fig3_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9800275_41591_2022_2095_Fig3_HTML.jpg)

a, Volcano plot demonstrating genes correlated with expression of IL7 from RNA-seq data identified using DESEQ2 from 94 healthy donor (HD) samples, two-tailed Wald test. b, GOBP pathway analysis of genes positively correlated with B cell IL7 (FDR < 0.05); x axis demonstrates fold change above background; y axis demonstrates z-score; all points above dotted line FDR < 0.01; one-tailed hypergeometric test. c, Percentage B cell receptor chains mapped from pre-immunotherapy B cell samples with germline divergent mutations dichotomized by carriage of rs16906115 risk allele, faceted by gene; two-sided Wilcoxon rank-sum test, n = 91. d, Percentage of all B cell clones detected expressing unswitched immunoglobulin IGHD in pre-treatment patient samples dichotomized by carriage of rs16906115 risk allele; two-sided Wilcoxon rank-sum test, n = 91. e, UMAP of scRNA-seq B cells from n = 24 pre-treatment patients and n = 5 healthy donors (15,755 cells total), demonstrating four main populations: naive, USM, SM and ASC. f, Relative IL7 expression across each B cell subset dichotomized according to HD or untreated patient (Mel); two-sided Wilcoxon rank-sum test, n = 24 pre-treatment patients, n = 5 HDs. g, Expression of IL7 across all untreated patient B cells in scRNA-seq data dichotomized according to rs16906115 status; two-sided Wilcoxon rank-sum test, n = 24 pre-treatment patients. h, Frequency of B cell subsets in untreated patients from scRNA-seq data dichotomized according to rs16906115 status; two-sided Wilcoxon rank-sum test, n = 24 pre-treatment patients. For box plots in c,d,f–h, the central line reflects the median value; the box corresponds to 25–75% quartiles; the upper whisker extends to the largest value no farther than 1.5× IQR; and the lower whisker extends from the 25% quartile to the smallest value no farther than 1.5× IQR.

### Extended Data Fig. 3. B cell correlates of IL7 expression.

![Extended Data Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e00/9800275/2aa88180f87b/41591_2022_2095_Fig8_ESM.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9800275_41591_2022_2095_Fig8_ESM.jpg)

(a) GOBP pathway analysis of genes anti-correlated with B cell IL7 (FDR < 0.05), x -axis demonstrates fold-change above background, y-axis z score, all points above dotted line FDR < 0.01, one-tailed hypergeometric test. (b) Percentage chains detected with mutation from germline (chain denoted in label) or clones containing IGHD (y axis) versus relative expression IL7 for that sample (x axis), P value refers to Spearman rank-test result, dotted lines reflect fitted linear model, shaded area 95% Confidence Interval. (c) Heatmap of key B cell subset specific genes from scRNA-seq data. (d) GOBP analysis of genes differentially expressed between B cell subsets, top 5 pathways per subset listed, one-tailed hypergeometric test. (e) UMAP from main Fig. 3e, depicting relative IL7 expression per cell. (f) Relative expression of IL7 (imputed) using scRNA-seq dataset across all B cells separated by subset (n = 28 samples). (g) Immunoglobulin chain usage in Naïve B cell subset according to patient status (HD - healthy donor n = 5, mel – metastatic melanoma n = 23); two-sided Wilcoxon rank-sum test. (h) Median expression of IL7 per B cell subset from scRNA-seq data dichotomised by rs16906115 demonstrating genotype acts through altering proportions of subsets as opposed to relative expression per cell (n = 23 patients); two-sided Wilcoxon rank-sum test. For boxplots in Extended Data Fig. 3f–h the central line reflects the median value, the box corresponds to 25%–75% quartiles, the upper whisker extends to largest value no further than 1.5x interquartile range, the lower whisker extends from 25% quartile to the smallest value no further than 1.5x interquartile range.

### rs16906115 and B cell IL7 associate with T cell responses

Given that T cells are central to the ICB anti-cancer response, are implicated in irAE development and are exquisitely sensitive to IL-7 (refs. ^10,12,23,24^), we explored the association between T cell ICB responses and genotype. Using flow cytometry of paired pre-treatment and post-treatment PBMC samples, carried out blinded to treatment and genotype (*n* = 54 paired samples obtained pre-treatment and immediately before the second cycle (C2), median 21 days), we observed genotypic divergence in treatment-induced subset changes. Risk allele carriers showed increases in the proportion of terminally differentiated effector memory T cells re-expressing CD45RA (TEMRA) T cells and a reciprocal fall in naive T cell proportion after ICB (Fig. [4a](#Fig4)). In keeping with the known increased sensitivity of CD8^+^ T cells to IL7 (refs. ^25,26^), we observed the genotype-specific fall in naive T cells to be most significant in the CD8^+^ subset (Fig. [4a](#Fig4); median fall homozygous reference allele = −0.31%; IQR −2.80% to 2.27% versus median fall in risk allele carriers = −11.2%; IQR −14.35% to −7.45%; *P* = 0.00017). Furthermore, we explored the relationship between LS and cell subsets in these individuals, finding that LS was correlated with an increase in all T cells and effector memory cells post-ICB and a fall in naive T cells (Extended Data Fig. [4a](#Fig9)). We proceeded to explore the effect of rs16906115 on CD8^+^ T cell gene expression across recipients of ICB. Given the stronger association of genotype with irAEs in sICB recipients and the markedly larger impact of cICB on CD8^+^ T cell transcriptomics^12^, which may saturate any genotypic effects, we restricted analysis to samples from sICB recipients. We performed differential expression analysis of CD8^+^ cell expression profiles obtained from pre-treatment and post-treatment samples from patients (194 samples from 92 patients: 86 pre-treatment and 108 post-treatment), controlling for confounding variables and dichotomizing by carriage of risk allele at rs16906115. We found that genotype was associated with 570 differentially expressed genes (DEGs) (adjusted *P* < 0.05; Fig. [4b](#Fig4) and Supplementary Table [4](#MOESM3)), with 96% of transcripts displaying the same direction of effect of genotype across each of the three timepoints (days 0, 21 and 63), showing consistent genotypic effects over time. Supporting the flow cytometry observations, we noted genotypic associations with genes associated with divergent stages of T cell differentiation, including *PRDM1*, implicated in terminal differentiation of CD8^+^ T cells^27^. This gene was upregulated in risk allele carriers, whereas these individuals simultaneously had reduced expression of *SH3BGRL2*, which was found to strongly correlate with the naive marker CD27, most significantly in patient samples (*P*_interaction.status_ < 0.001) (Fig. [4c](#Fig4) and Extended Data Fig. [4b](#Fig9)). Gene pathway analysis showed that induced genes were enriched across ten pathways (adjusted *P* < 0.05), broadly encompassing anabolic processes, including ‘regulation of phosphatidylinositol 3-kinase signaling’, ‘activation of MAPK activity’ and ‘transcription from RNA polymerase II promoter’ (Extended Data Fig. [4c](#Fig9)). Pathway enrichment for risk-allele-associated repressed genes was more marked, however, incorporating 40 pathways predominantly composed of cell division processes, including ‘mitotic nuclear envelope disassembly’ and ‘mitotic cytokinesis’ (Fig. [4d](#Fig4) and Supplementary Table [5](#MOESM3)). To assess the evidence for B cell *IL7* expression mediating the genotypic association of rs16906115 with CD8^+^ T cell expression, we correlated CD8^+^ T cell gene expression with B cell *IL7* expression from the same blood sample (*n* = 189 sICB samples). This identified 935 significant DEGs (adjusted *P* < 0.05; Supplementary Table [6](#MOESM3)) associated with B cell *IL7*. Pathway analysis of anti-correlated genes demonstrated a striking enrichment of those involved in cell division (Fig. [4e](#Fig4) and Supplementary Table [7](#MOESM3)), overlapping highly with pathways repressed by carriage of rs16906115 risk allele (Fig. [4f](#Fig4)) and, thus, providing further support for mediation of rs16906115 genotypic effects via B cell *IL7* expression. In keeping with this, when controlling for rs16906115 status, B cell *IL7-*repressed pathways remained highly associated with cell division (Extended Data Fig. [4d](#Fig9) and Supplementary Table [8](#MOESM3)). Finally, we performed the same analysis in non-carriers of the risk allele. This approach reduced the size of the cohort and the range of B cell *IL7* expression, resulting in fewer DEGs; however, at a false discovery rate (FDR) threshold of 0.2, the pathways repressed strongly reflected those in the full dataset (Extended Data Fig. [4e](#Fig9) and Supplementary Table [9](#MOESM3)). As a further corroboration, we examined allelic associations with previously described mitotic signature scores^23^, exploring the change in signature upon treatment across each allele. Consistent with our pathway analysis results, we note significantly lower induction of the mitotic signature in risk allele carriers (Fig. [4g](#Fig4)).

### Fig. 4. Risk allele carriage is associated with T-cell-induced ICB responses.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e00/9800275/6575c0f6e27d/41591_2022_2095_Fig4_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9800275_41591_2022_2095_Fig4_HTML.jpg)

a, Differences in subset proportion (depicted in facet name) between baseline untreated (C1) and immediately before the second cycle of treatment (C2) as determined by flow cytometry results assessing change in CD8+ T cell subset (depicted in facet name) with ICB treatment according to rs16906115 status; two-sided Wilcoxon rank-sum test, n = 54 patients. b, Volcano plot depicting results for DESEQ2 differential gene expression analysis of CD8+ T cell RNA-seq data, dichotomized by rs16906115 status; n = 194 samples across three cycles (untreated, C2 and C4) of sICB treatment, two-tailed Wald test. c, Example box plots of genes regulated by allele in sICB recipients, faceted by timepoint; two-sided t-test of normalized expression values, n = 86 patients at baseline, n = 69 patients at day 21. d, GOBP pathway analysis of genes suppressed in carriers of risk allele (as depicted in b); one-tailed hypergeometric test. e, GOBP pathway analysis of genes anti-correlated in patient CD8+ T cells with increasing B cell IL7 expression from the same blood samples; one-tailed hypergeometric test. f, Comparative analysis of z-scores (x axis: B cell IL7 effect, y axis: rs16906115 effect) from pathway analysis in d and e. g, Change in CD8+ T cell mitotic signature score post-sICB according to rs16906115 status; two-sided t-test, n = 65 patients. For box plots in a,c,g, the central line reflects the median value; the box corresponds to the 25–75% quartiles; the upper whisker extends to the largest value no farther than 1.5× IQR; and the lower whisker extends from the 25% quartile to the smallest value no farther than 1.5× IQR.

### Extended Data Fig. 4. Extended T cell data.

![Extended Data Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e00/9800275/9bb1a49c4ec5/41591_2022_2095_Fig9_ESM.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9800275_41591_2022_2095_Fig9_ESM.jpg)

(a) Correlation of fold change in lymphocyte count measured on routine hospital bloods (as per Lymphocyte Stability) versus % change in cell subset as measured using flow-cytometry at the same timepoint. From left: right; % all T cells in PBMC, % T Effector Memory, % Naïve T cells; P value from Spearman rank test, dotted lines reflect fitted linear model, shaded area the 95% Confidence Interval, n = 40. (b) Correlation of CD8+ T cell expression of SH3BGRL2 versus the naïve marker CD27 in healthy donors (left facet) and patient samples (right facet); Spearman rank test, shaded area reflects 95% Confidence Interval of fitted linear model, P interaction with status <0.001, n = 67 controls, n = 326 patient C1/C2 samples. (c) GOBP pathway analysis of genes upregulated in CD8+ T cells in carriers of risk allele (as depicted in Fig. 4b), one-tailed hypergeometric test. (d) GOBP pathway analysis of genes anti-correlated in patient CD8+ T cells with increasing B cell IL7 expression from same blood samples, having corrected for rs16906115 status, one-tailed hypergeometric test. e) GOBP pathway analysis of genes anti-correlated in patient CD8+ T cells with increasing B cell IL7 expression in non-carriers of risk allele only, one-tailed hypergeometric test.

### rs16906115 associates with post-ICB CD8+ T cell clonality

IL-7 is crucial for the survival and proliferation of mature T cell clones^28^, and we previously observed that clonal CD8^+^ T cell proportions, both pre-treatment and post-treatment, are prognostic for long-term clinical outcome^12,23^. To assess whether rs16906115 impacts overall clonal features, we analyzed CD8^+^ T cell clone sizes, based on unique TCR CDR3 nucleotide sequences, across the cohort (*n* = 146 CD8^+^ T cell samples taken immediately before C2, of which 133 had paired baseline samples), blinded to expression phenotype. We first examined the relationship between LS and clone size, and we found that patients with stable LS (increase in count or fall <20%) had significantly more large clones (defined as those comprising >0.5% of the TCR repertoire) both pre-treatment and post-treatment (Fig. [5a](#Fig5)), in keeping with both parameters being positively associated with beneficial response to ICB. We subsequently explored the effect of rs16906115 genotype on the Gini index of the TRB chains, a measure of clonal unevenness^29^, across pre-treatment and post-treatment samples. Notably, we found a significant interaction between ICB treatment and genotype in the change in Gini in response to treatment, with risk allele carriers showing increased clonal inequality post-ICB (Fig. [5b](#Fig5)), resultant in skewed clonality after ICB treatment according to allele (Fig. [5c](#Fig5)). Finally, we explored the relationship between clonal composition post-treatment according to genotype, and we found that risk allele carriers post-sICB had a repertoire composed of proportionally fewer small clones (<0.05% repertoire in size) and, instead, demonstrated significantly greater repertoire occupancy by large clones (>0.5% repertoire in size) after both cICB and sICB (Fig. [5d](#Fig5)).

### Fig. 5. Risk allele is associated with CD8 clonality and independently associates with survival in TCGA data.

![Fig. 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e00/9800275/2c0fc7c2ca38/41591_2022_2095_Fig5_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9800275_41591_2022_2095_Fig5_HTML.jpg)

a, Association between count of large CD8+ T cell clones (defined as those >0.5% repertoire in size using TRB) and LS for same individual pre-ICB (C1, left facet) and immediately before the second cycle of treatment (C2, right facet); two-sided t-test, n = 110 patients (C1), n = 91 patients (C2). b, Gini index (measured on TRB chain) immediately before C2 samples from CD8+ T cells across all ICB treatments with available measurement, stratified by carriage at rs16906115; two-sided t-test, n = 62 sICB patients, n = 76 cICB patients, n = 138 both. c, Change in Gini index between pre-treatment (C1) and post-treatment (C2) by ICB type; two-sided t-test, n = 133. d, Proportion of post-treatment clones per repertoire at C2 defined by TRB CDR3 sequence and V and J gene usage at different size thresholds (left to right facets) and by ICB type (cICB upper, sICB lower), dichotomized by rs16906115 carriage; two-way t-test, n = 77 patients cICB, n = 69 patients sICB. For box plots in a–d, the central line reflects the median; the box corresponds to 25–75% quartiles; the upper whisker extends to the largest value no farther than 1.5× IQR; and the lower whisker extends from the 25% quartile to the smallest value no farther than 1.5× IQR. e, Carriage of rs16906115 minor allele is associated with disease-specific survival in TCGA melanoma dataset; log-rank test. f, Carriage of rs16906115 minor allele is associated with overall survival in TCGA melanoma dataset; log-rank test.

### rs16906115 is associated with melanoma survival in TCGA

Given that rs16906115 is associated with baseline pre-treatment differences in patient B cell *IL7* expression and B cell maturation, we sought to explore whether this SNP had a relationship with the natural history of melanoma. We examined the frequency of this allele in the TCGA melanoma data series, and we found that it did not differ to that in our cohort (8.3% versus 7.5%, *P* = 0.72, chi-squared test). We then examined the clinical outcome data from this cohort^30^, restricting analysis to those whose ethnicity is recorded as ‘white’ and dichotomizing outcomes according to allele carriage (*n* = 435). Strikingly, we found that carriage of the risk allele was associated with increased progression-free survival (*P* = 0.016, log-rank test), disease-specific survival (*P* = 0.015) and overall survival (*P* = 0.014) (Extended Data Fig. [5a](#Fig10) and Fig. [5e,f](#Fig5)). This provides further independent evidence as to the importance of this allele in melanoma, inferring that, in addition to predisposing patients to ICB toxicity, B cell *IL7* plays a role in the natural history of melanoma.

### Extended Data Fig. 5. TCGA survival data by rs16906115 carriage.

![Extended Data Fig. 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e00/9800275/a4b2790066d7/41591_2022_2095_Fig10_ESM.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9800275_41591_2022_2095_Fig10_ESM.jpg)

(a) Progression-free survival according to rs16906115 allele status across All Stages in TCGA melanoma dataset. (b) As per Extended Data Fig. 5a, but Stages IIB-IV. (c) Carriage of rs16906115 minor allele is associated with disease-specific survival in Stages IIB-IV in TCGA melanoma dataset. (d) Overall Survival according to rs16906115 allele status across Stages IIB-IV Stages in TCGA melanoma dataset. All P values correspond to log-rank test.

## Discussion

Allelic variation at rs16906115 is a significant risk factor for development of irAEs, most notably in recipients of sICB who have a greatly reduced risk of severe irAEs. Our data replicate the co-reported GWAS by Groha et al. and provide novel insights into mechanisms of response to ICB and irAE risk. The allele rs16906115 is intronic to *IL7*, and, unexpectedly, we found robust expression of *IL7* in B cells, which is markedly induced in patients with melanoma. Notably, patient carriers of the rs16906115 risk allele demonstrate significantly greater induction of B-cell-specific *IL7*, a context-specific effect not observed in those without melanoma or in other immune subsets^31–33^. Using scRNA-seq, we replicated both patient-specific induction and genotypic association, providing further clarity regarding *IL7* expression across B cell subsets and demonstrating altered frequency of these according to rs16906115 status. Notably, supporting the effect of rs16906115 being mediated via B cell *IL7*, we demonstrate that B cell *IL7* is more significantly associated with irAE risk than rs16906115 genotype, with genotypic effect being completely resolved by controlling for B cell *IL7*. Although we did not observe rs16906115 effects on other circulating immune cells, given that stromal and epithelial cells are key sources of IL-7 (refs. ^11,15^) and considering the important role of tertiary lymphoid structures in immune responses after ICB^34,35^, we postulate that other B cell-containing lymphoid tissues may show similar context-specific *IL7* expression linked to this allele. We found that rs16906115 is associated with increased LS post-sICB and that LS anti-correlates with induction of naive T cells. Instead, carriage of the risk allele shifts the CD8^+^ T cell response to ICB from a hyper-proliferative state with expansion of naive subsets to one with reduced overall mitosis and maintenance of differentiated effector subsets, in keeping with known properties of IL-7 and previous observations regarding T cell mitosis and plasma IL-7 (refs. ^14,28,36^). This is also seen in terms of CD8^+^ T cell gene pathways differentially expressed in the presence of the risk allele and the regulation of a signature gene set that acts as a proxy for ICB-induced mitosis. Crucially, again in keeping with the effects of rs16906115 on CD8^+^ T cells being mediated via B cell *IL7*, we note that the same pathways are repressed by increasing expression of B cell *IL7* when correcting for rs16906115, with this association being present in individuals homozygous for the reference allele. Finally, analysis of CD8^+^ TCR, agnostic to gene expression profiles, corroborates the mechanistic consequences of rs16906115 allelic variation in melanoma, with patients who carry the risk allele displaying distinct clonal responses to treatment, consisting of clonal skewing characterized by fewer small clones but increased repertoire occupancy by expanded large clones. Although IL-7 has been described as an important mediator of naive T cell homeostasis in the resting state, it also has a vital role in T cell activation and memory formation^10,25,26,37–39^. Consequently, these observations suggest the importance of the latter in the context of malignancy-related chronic antigen stimulation and ICB therapy. Although the risk-allele-associated increases in large CD8^+^ T cell clone populations would be anticipated to provide a more cytotoxic response to both tumor and other tissues, we cannot discount a direct B cell-mediated role for increased irAE risk associated with this allele, supported by increased class switching and immunoglobulin chain mutations in carriers pre-treatment. The large effect size of this polymorphism reflects the central role of IL-7 in both B and T cell lymphopoiesis. Future studies are required to ascertain whether induction of peripheral B cell *IL7* is observed in other malignancies, potentially forming a biomarker for immune sensitivity. The finding of improved disease-specific and overall survival in patients with melanoma carrying the risk allele in the TCGA dataset indicates that IL-7 plays a role in the natural history of melanoma and further highlights the potential therapeutic importance of this pathway. This study highlights the power of agnostic genetic analyses to provide insights into human immunity of high relevance to disease and delineates a key role for IL-7 in response to ICB, revitalizing previous proposals for incorporating this molecule as a potential adjunct to immunotherapy strategies^36,40,41^.

## Methods

### Participants

Adult patients referred to receive ICB as standard-of-care therapy for the treatment of metastatic melanoma or in the adjuvant setting at Oxford University Hospitals NHS Foundation Trust were prospectively and consecutively approached for recruitment. All patients provided written informed consent to donate samples to the Oxford Radcliffe Biobank (Oxford Centre for Histopathology Research ethical approval reference 19/SC/0173, project nos. 16/A019, 18/A064 and 19/A114) and grant access to their routine clinical data; there was no compensation for this. Patients were recruited between 23 November 2015 and 30 September 2021 and received either cICB (ipilimumab 3 mg kg^−1^ plus nivolumab 1 mg kg^−1^ three times a week for ≤4 treatment cycles, followed by maintenance nivolumab) or sICB consisting of nivolumab 480 mg monthly, pembrolizumab 2 mg kg^−1^ three times a week or pembrolizumab 4 mg kg^−1^ six times a week. Patient characteristics are given in Extended Data Fig. [1a](#Fig6). Healthy donor participants were recruited via the Oxford Biobank ([www.oxfordbiobank.org.uk](http://www.oxfordbiobank.org.uk); ethical approval reference 06/Q1605/55), with written informed consent. All donors were of European ancestry; 104 were female, and 66 were male, aged between 21 years and 66 years (median, 46.5 years; IQR 18 years).

### Clinical outcomes

Patient demographic and clinical characteristics were collected from the electronic patient record (EPR). Sex was determined on the self-reported assignment extracted from the EPR. irAEs were reported according to the National Cancer Instituteʼs Common Terminology Criteria for Adverse Events version 4.03. Steroids were defined as a prescription being issued for oral or intravenous corticosteroids to treat irAE(s) according to clinical indication. ‘Prior to cycle 5’ was defined as an irAE occurring before the patient had received a cumulative five doses of ICB and is a metric that has been used previously^7^. Response data were obtained from the EPR, with progression being defined either clinically or using radiological assessment according to irRECIST.1.1, performed approximately 12 weeks and 24 weeks after initiation of treatment. All patients who had received a minimum of one cycle of ICB were incorporated into the analysis irrespective of clinical outcome.

### Sample collection

In total, 30–50 ml of blood was collected into EDTA tubes (BD Vacutainer system) collected immediately before administration of ICB. PBMCs and plasma were immediately obtained from whole blood by density centrifugation (Ficoll Paque). All cell subset isolation for RNA-seq was carried out by magnetic separation (Miltenyi Biotec) using CD8, CD19 and CD14 positive selection for CD8^+^ T cells, B cells and monocytes and negative selection for natural killer (NK) cells and CD4^+^ naive cells according to the manufacturer’s instructions, with all steps performed either at 4 °C or on ice. Laboratory blood counts were obtained from the patients’ EPR with pre-treatment samples being defined as those taken closest to the first cycle of ICB, but no more than 30 days beforehand, and post-treatment samples being defined as the first sample taken at least 21 days after ICB, but no later than 49 days. Lymphocyte counts were generated using Sysmex XN series analyzers (Sysmex Corporation) in an NHS clinical diagnostic laboratory.

### RNA and DNA extraction

After selection, cells were pelleted and resuspended in 350 μl of RLT Plus buffer with 1% beta-mercaptoethanol or DTT. Samples were snap-frozen at −80 °C for batched RNA extraction. Homogenization of the sample was carried out using a QIAshredder (Qiagen), followed by RNA and gDNA extraction using the AllPrep DNA/RNA/miRNA Kit (Qiagen). RNA and gDNA were eluted into 34 μl or 54 μl of RNase-free water, respectively, and concentration was quantified by Qubit and stored at −80 °C until further analysis.

### Genotyping

Genotyping was performed on the Illumina Global Screening Array 24 version 3 (Illumina) and was available from 214 patient participants ≥18 years of age, with sufficient follow-up, who had been diagnosed with metastatic melanoma (*n* = 203) or started on adjuvant pembrolizumab or nivolumab after recent (<12 weeks) resection of metastatic lymph node or other site disease (*n* = 11) who had received ≥1 cycles of ICB as a standard-of-care therapy. Genotypes were aligned to b37, filtered for call frequency >0.9, cluster separation >0.3, SNP missingness <3%, MAF >1% and Hardy–Weinberg equilibrium (HWE) *P* > 10^−20^, before imputation using the Michigan Imputation Server version 1.2.4 with Haplotype Reference Consortium version 1 as a reference panel, using Eagle2 and Minimac4 for phasing and imputation. After imputation, SNPs were filtered for MAF >4%, HWE *P* > 10^−10^ and *r*^2^ > 0.7 and lifted-over to b38.

### Bulk RNA-seq and analysis

Bulk RNA-seq was performed on CD19^+^ B cell and CD8^+^ T cell samples derived from healthy individuals and patients receiving single or combination ICB. RNA was thawed on ice before mRNA isolation using NEBNext Poly(A) mRNA Magnetic Isolation Module Kits. Up to 600 ng of RNA was then used to generate dsDNA libraries using NEBNext Ultra II Directional RNA Library Prep Kits for Illumina (cohort sizes given in the text) as previously described^12^. Samples were then sequenced on either an Illumina HiSeq 4000 (75-bp paired-end) or a NovaSeq 6000 (150-bp paired-end) at the Oxford Genomics Centre. Differential gene expression and pathway analysis were performed using DESeq2 (ref. ^42^) version 1.30.1 controlling for batch effects, RNA concentration, age and, where multiple cycles of treatment were incorporated, cycle of treatment. No difference was noted between adjuvant patients and metastatic patients in terms of B cell *IL7* expression at baseline (*P* > 0.2) either pre-treatment or post-treatment. Transcript assembly was performed using StringTie^43^ using Ensembl GRCh37 as the reference ([ftp://ftp.ensembl.org/pub/release-75//gtf/Homo_sapiens.GRCh37.75.gtf.gz](https://ftp.ensembl.org/pub/release-75//gtf/Homo_sapiens.GRCh37.75.gtf.gz)) for each sample individually, before merging of assemblies and quantification of isoform expression. Comparative analysis per allele was based on read counts using a Wilcoxon signed-rank test. Mitotic signature score was calculated using DESeq2-normalized expression data, taking the geometric mean of the top 50 genes most associated with the mitotic subset in previous CD8^+^ T cell single-cell sequencing data^23^.

### Adaptive receptor analysis

B cell receptor (BCR) and TCR analysis was performed using the MiXCR^44^ version 3.0.13 package with settings as previously described^12,23^. Clonal metrics were calculated from TCR and BCR data by taking the minimum number of reads/chains recorded across samples and re-sampling other samples to the same depth, bootstrapping 1,000 times. BCR usage was generated for IGHC genes. Usage was calculated as either the percentage of IGH clones or reads expressing each individual IGHC gene segment. Mutation rate was inferred in BCR samples, representative of junctional diversity and somatic hypermutation. Information regarding base substitutions, insertions and deletions was extracted from MiXCR output files from ‘allVAlignments’, ‘allDAlignments’ and ‘allJAlignments’ columns. The number of mutations per BCR clone was calculated as the number of substitutions, insertions and deletions combined. Average mutation percentage was then calculated per BCR sample as the total number of mutations detected per clone divided by the total BCR length. Analysis was repeated by chain.

### scRNA-seq

5′ scRNA and VDJ sequencing (10x Genomics) was performed on PBMCs (kit version 2) according to the manufacturer’s instructions. In brief, cryopreserved PBMCs were thawed and purified using the Dead Cell Removal Kit. Cells from 5–8 donors were pooled, and an estimated 50,000–60,000 cells were loaded per channel. Gene expression and VDJ libraries were 150-bp paired-end sequenced on a NovaSeq 6000 (Illumina) with a targeted read depth of 25,000 reads per cell. Reads were mapped using cellranger version 6.0.1, with the union of cells called by cellranger and EmptyDroplets^45^ used downstream. Donor identification was performed using cellsnp-lite and vireoSNP^46^, and ambient RNA contamination was removed with SoupX^47^. Cells identified as genotypic doublets, and those with nFeatures <300 or mitochondrial reads >20%, were excluded. The remaining PBMCs were separately clustered and projected into two-dimensional (2D) uniform manifold approximation and projection (UMAP) space in Seurat version 4.1 (ref. ^48^), using Harmony^49^ for pool harmonization. After quality control (including removal of doublets and exclusion of TCR or dual-BCR carrying cells), clusters identified as B cells based on marker expression (that is, *CD79A* and *MS4A1*) and subset inference using SingleR^50^ were retained for further analysis. Subset clusters were annotated based on a combination of gene expression, SingleR inferences and VDJ usage, and *IL7* expression was imputed using MAGIC^51^.

### Flow cytometry

Patient PBMCs frozen in 90% FCS + 10% DMSO were thawed at 37 °C and washed once in HBSS before staining of 1 × 10^6^ cells with either LIVE/DEAD Fixable Near-IR (Invitrogen) or Zombie Green (BioLegend). Cells were washed and subsequently stained with antibodies against CD3, CD4, CD8α, CD27 and CD45RA in 5% FBS + HBSS. Alternatively, cells were stained with antibodies targeted against CD19, CD3, CD56 and CD14 along with biotinylated anti-IL-7 before staining with streptavidin-PE. Fluorescence minus one (FMO) controls along with primary biotinylated anti-IL-7 and secondary streptavidin-PE alone controls were included to determine IL-7 gating per individual. After washing, cells were fixed for 10 minutes with 2% paraformaldehyde (Sigma-Aldrich), washed and resuspended in 5% FBS + HBSS before acquisition on a BD LSRII. Data were analyzed in FlowJo (Tree Star, version 10.7.1) and R (version 4.0.5). Antibody details are shown in Supplementary Table [9](#MOESM3), and gating was as previously performed^12^. All staining steps were performed for 30 minutes and at 4 °C unless otherwise stated.

### General statistical analysis

Statistical comparisons were performed as indicated in each figure legend and conducted in R (version 4.0.5), with graphics using ggplot2 version 3.3.5. Paired tests were used for paired samples, usually using Wilcoxon signed-rank test as indicated. For unpaired samples, the Wilcoxon rank-sum test was primarily used as per the figure legends. ORs were generated using a generalized linear model (family = ‘binomial’). Random effect mixed linear models were run in lme4 version 1.1-26 using lmerTest version 3.1-3 to explore association between B cell *IL7* and irAE development, controlling for rs16906115 and ICB type. Survival analysis was performed using survival version 3.2-11. Pathway analysis was performed using XGR version 1.1.8. Adjusted *P* values are reported using Benjamini–Hochberg correction for multiple comparisons.

### Reporting summary

Further information on research design is available in the [Nature Portfolio Reporting Summary](#MOESM2) linked to this article.

## Online content

Any methods, additional references, Nature Portfolio reporting summaries, source data, extended data, supplementary information, acknowledgements, peer review information; details of author contributions and competing interests; and statements of data and code availability are available at 10.1038/s41591-022-02095-5.

## Supplementary information

## Acknowledgements

We are very grateful to all patients who contributed samples and participated in the study. We thank all the staff of the Day Treatment Unit, Oxford Cancer Centre and the Brodey Centre at the Horton General Hospital. We are grateful to all the staff of the Oxford University Hospitals NHS Foundation Trust haematology and biochemistry laboratory and thank T. James for his facilitation, as well as the staff of the Oxford Radcliffe Biobank and Churchill Hospital Sample Handling Lab. This study was funded by a Wellcome Intermediate Clinical Fellowship to B.P.F. (no. 201488/Z/16/Z), additionally supporting A.V.A. and I.N. R.A.W. is funded by a Wellcome Trust Doctoral Training Fellowship (no. BST00070). O.T. is supported by the Clarendon Fund, St. Edmund Hall and an Oxford Australia Scholarship. W.Y. is an NIHR academic clinical fellow and is supported by a CRUK predoctoral fellowship (reference RCCTI\100019). R.C. was supported by a CRUK Clinical Research Training Fellowship (S_3578). C.A.T. was funded by the Engineering & Physical Sciences Research Council and the Balliol Jowett Society (no. D4T00070). D.C. is funded by a Cancer Research UK Advanced Clinician Scientist Fellowship (C26642/A27963). J.J.G. is supported by an NIHR Clinical Lectureship. M.R.M. and B.P.F. are supported by the NIHR Oxford Biomedical Research Centre. K.M. and P.E. are supported by the NIHR Leeds Biomedical Research Centre. The views expressed are those of the authors and not necessarily those of the NHS, the NIHR or the Department of Health.

### Extended data

### Extended data

## Author contributions

B.P.F., V.N. and A.G. conceived the study. N.C., M.J.P., M.R.M., B.P.F., V.W. and R.A.W. recruited patient participants. B.P.F., S.K., C.A.T., R.A.W., K.M. and P.E. recruited healthy volunteer participants. C.A.T., R.A.W., O.T., R.C., W.Y., A.V.A., P.K.S., S.K., R.C. and E.J. collected and processed blood samples. C.A.T., O.T., A.V.A. and W.Y. performed genotyping. C.A.T. and A.V.A. generated RNA-seq libraries. R.A.W., W.Y. and B.P.F. collected clinical follow-up data. B.S. collected laboratory blood results. C.A.T. performed flow cytometry experiments. D.C. assisted with TCGA analysis. O.T., W.Y. and A.V.A. performed single-cell sequencing. C.A.T., R.A.W., O.T., W.Y., J.J.G. and B.P.F. performed data analysis. B.P.F. drafted the manuscript, with support from R.A.W. V.N., M.R.M., T.C. M.F. and A.G. contributed to discussions throughout the work. B.P.F. oversaw the project. All authors read, edited and approved the final manuscript.

## Peer review

### Peer review information

*Nature Medicine* thanks Zlatko Trajanoski, Leng Han and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary handling editor: Anna Maria Ranzoni, in collaboration with the *Nature Medicine* team.

## Data availability

Cell-specific expression data are made freely available via EGA ([EGAC00001001482](https://ega-archive.org/dacs/EGAC00001001482)) upon completion of a data access agreement that can be obtained by contacting the corresponding author.

## Code availability

All scripts used to create figures, including the minimal dataset required to recreate figures, consisting of gene expression count data matrices, flow cytometry counts, genotyping at rs16906115 and scRNA-seq data of B cells in the form of a Seurat object, are available via the Fairfax Group bitbucket ([https://bitbucket.org/Fairfaxlab/](https://bitbucket.org/Fairfaxlab/)).

## Competing interests

C.A.T., R.A.W., O.T., W.Y., R.C., A.V.A., P.K.S., I.N., J.J.G. and A.G. declare no competing interests. M.R.M. reports grants from Roche, grants from AstraZeneca, grants and personal fees from GSK, personal fees and other from Novartis, other from Millenium, personal fees and other from Immunocore, personal fees and other from BMS, personal fees and other from Eisai, other from Pfizer, personal fees, non-financial support and other from Merck/MSD, personal fees and other from Rigontec (acquired by MSD), other from Regeneron, personal fees and other from BiolineRx, personal fees and other from Array Biopharma (now Pfizer), non-financial support and other from Replimune, personal fees from Kineta and personal fees from Silicon Therapeutics, all outside the submitted work. D.C. reports funding from Haliodx and consultancy for Merck. K.S. reports grants from Gilead, Eli Lilly and BMS and personal fees from UCB, AZD, Abbvie and Eli Lilly. P.E. performed consultancy for Abbvie, AstraZeneca, BMS, Boehringer Ingelheim, Galapagos, Gilead, Janssen, MSD, Eli Lilly, Novartis, Pfizer, Roche and Samsung and was involved in clinical trials for Abbvie, BMS, Eli Lilly, Novartis, Pfizer, Roche and Samsung. T.K.C. reports research/advisory boards/consultancy/honorarium (institutional and personal, paid and unpaid) from AstraZeneca, Aveo, Bayer, BMS, Eisai, EMD Serono, Exelixis, GSK, IQVA, Ipsen, Kanaph, Eli Lilly, Merck, Nikang, Novartis, Pfizer, Roche, Sanofi/Aventis, Takeda and Tempest. Travel, accommodations, expenses, medical writing in relation to consulting, advisory roles or honoraria. Stock options: Pionyr and Tempest. Other: up-to-date royalties, CME-related events (for example, OncLIve, PVI and MJH Life Sciences) honoraria. NCI GU Steering Committee. Patents filed, royalties or other intellectual properties (no income as of current date): related to biomarkers of immune checkpoint blockers and ctDNA. No speaker’s bureau. B.P.F. received conference support from BMS and performed consultancy for UCB.

## Footnotes

### Extended data

### Extended data

is available for this paper at 10.1038/s41591-022-02095-5.

### Supplementary information

The online version contains supplementary material available at 10.1038/s41591-022-02095-5.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

Cell-specific expression data are made freely available via EGA ([EGAC00001001482](https://ega-archive.org/dacs/EGAC00001001482)) upon completion of a data access agreement that can be obtained by contacting the corresponding author.

All scripts used to create figures, including the minimal dataset required to recreate figures, consisting of gene expression count data matrices, flow cytometry counts, genotyping at rs16906115 and scRNA-seq data of B cells in the form of a Seurat object, are available via the Fairfax Group bitbucket ([https://bitbucket.org/Fairfaxlab/](https://bitbucket.org/Fairfaxlab/)).

### Supplementary Materials

### Data Availability Statement

Cell-specific expression data are made freely available via EGA ([EGAC00001001482](https://ega-archive.org/dacs/EGAC00001001482)) upon completion of a data access agreement that can be obtained by contacting the corresponding author.

All scripts used to create figures, including the minimal dataset required to recreate figures, consisting of gene expression count data matrices, flow cytometry counts, genotyping at rs16906115 and scRNA-seq data of B cells in the form of a Seurat object, are available via the Fairfax Group bitbucket ([https://bitbucket.org/Fairfaxlab/](https://bitbucket.org/Fairfaxlab/)).

## References

1. Larkin J, et al. Five-year survival with combined nivolumab and ipilimumab in advanced melanoma. N. Engl. J. Med. 2019;381:1535–1546. doi: 10.1056/NEJMoa1910836.  [DOI](https://doi.org/10.1056/NEJMoa1910836) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31562797/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Five-year%20survival%20with%20combined%20nivolumab%20and%20ipilimumab%20in%20advanced%20melanoma&author=J%20Larkin&volume=381&publication_year=2019&pages=1535-1546&pmid=31562797&doi=10.1056/NEJMoa1910836&)

2. Weber J, et al. Adjuvant nivolumab versus ipilimumab in resected stage III or IV melanoma. N. Engl. J. Med. 2017;377:1824–1835. doi: 10.1056/NEJMoa1709030.  [DOI](https://doi.org/10.1056/NEJMoa1709030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28891423/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Adjuvant%20nivolumab%20versus%20ipilimumab%20in%20resected%20stage%20III%20or%20IV%20melanoma&author=J%20Weber&volume=377&publication_year=2017&pages=1824-1835&pmid=28891423&doi=10.1056/NEJMoa1709030&)

3. Robert C, et al. Pembrolizumab versus ipilimumab in advanced melanoma (KEYNOTE-006): post-hoc 5-year results from an open-label, multicentre, randomised, controlled, phase 3 study. Lancet Oncol. 2019;20:1239–1251. doi: 10.1016/S1470-2045(19)30388-2.  [DOI](https://doi.org/10.1016/S1470-2045(19)30388-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31345627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol.&title=Pembrolizumab%20versus%20ipilimumab%20in%20advanced%20melanoma%20(KEYNOTE-006):%20post-hoc%205-year%20results%20from%20an%20open-label,%20multicentre,%20randomised,%20controlled,%20phase%203%20study&author=C%20Robert&volume=20&publication_year=2019&pages=1239-1251&pmid=31345627&doi=10.1016/S1470-2045(19)30388-2&)

4. Michot JM, et al. Immune-related adverse events with immune checkpoint blockade: a comprehensive review. Eur. J. Cancer. 2016;54:139–148. doi: 10.1016/j.ejca.2015.11.016.  [DOI](https://doi.org/10.1016/j.ejca.2015.11.016) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26765102/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Cancer&title=Immune-related%20adverse%20events%20with%20immune%20checkpoint%20blockade:%20a%20comprehensive%20review&author=JM%20Michot&volume=54&publication_year=2016&pages=139-148&pmid=26765102&doi=10.1016/j.ejca.2015.11.016&)

5. Schneider BJ, et al. Management of immune-related adverse events in patients treated with immune checkpoint inhibitor therapy: ASCO guideline update. J. Clin. Oncol. 2021;39:4073–4126. doi: 10.1200/JCO.21.01440.  [DOI](https://doi.org/10.1200/JCO.21.01440) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34724392/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&title=Management%20of%20immune-related%20adverse%20events%20in%20patients%20treated%20with%20immune%20checkpoint%20inhibitor%20therapy:%20ASCO%20guideline%20update&author=BJ%20Schneider&volume=39&publication_year=2021&pages=4073-4126&pmid=34724392&doi=10.1200/JCO.21.01440&)

6. Ramos-Casals M, et al. Immune-related adverse events of checkpoint inhibitors. Nat. Rev. Dis. Prim. 2020;6:38. doi: 10.1038/s41572-020-0160-6.  [DOI](https://doi.org/10.1038/s41572-020-0160-6) | [PMC free article](/articles/PMC9728094/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32382051/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Dis.%20Prim.&title=Immune-related%20adverse%20events%20of%20checkpoint%20inhibitors&author=M%20Ramos-Casals&volume=6&publication_year=2020&pages=38&pmid=32382051&doi=10.1038/s41572-020-0160-6&)

7. Ye W, et al. Checkpoint-blocker-induced autoimmunity is associated with favourable outcome in metastatic melanoma and distinct T-cell expression profiles. Br. J. Cancer. 2021;124:1661–1669. doi: 10.1038/s41416-021-01310-3.  [DOI](https://doi.org/10.1038/s41416-021-01310-3) | [PMC free article](/articles/PMC8110747/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33723392/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Cancer&title=Checkpoint-blocker-induced%20autoimmunity%20is%20associated%20with%20favourable%20outcome%20in%20metastatic%20melanoma%20and%20distinct%20T-cell%20expression%20profiles&author=W%20Ye&volume=124&publication_year=2021&pages=1661-1669&pmid=33723392&doi=10.1038/s41416-021-01310-3&)

8. Widmer MB, et al. Lymphopoiesis and IL‐7. Int. J. Cell Cloning. 1990;8:168–172. doi: 10.1002/stem.5530080715.  [DOI](https://doi.org/10.1002/stem.5530080715) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2182735/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Cell%20Cloning&title=Lymphopoiesis%20and%20IL%E2%80%907&author=MB%20Widmer&volume=8&publication_year=1990&pages=168-172&pmid=2182735&doi=10.1002/stem.5530080715&)

9. Clark MR, Mandal M, Ochiai K, Singh H. Orchestrating B cell lymphopoiesis through interplay of IL-7 receptor and pre-B cell receptor signalling. Nat. Rev. Immunol. 2014;14:69–80. doi: 10.1038/nri3570.  [DOI](https://doi.org/10.1038/nri3570) | [PMC free article](/articles/PMC4276135/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24378843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Immunol.&title=Orchestrating%20B%20cell%20lymphopoiesis%20through%20interplay%20of%20IL-7%20receptor%20and%20pre-B%20cell%20receptor%20signalling&author=MR%20Clark&author=M%20Mandal&author=K%20Ochiai&author=H%20Singh&volume=14&publication_year=2014&pages=69-80&pmid=24378843&doi=10.1038/nri3570&)

10. Fry TJ, Mackall CL. The many faces of IL-7: from lymphopoiesis to peripheral T cell maintenance. J. Immunol. 2005;174:6571–6576. doi: 10.4049/jimmunol.174.11.6571.  [DOI](https://doi.org/10.4049/jimmunol.174.11.6571) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15905493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.&title=The%20many%20faces%20of%20IL-7:%20from%20lymphopoiesis%20to%20peripheral%20T%20cell%20maintenance&author=TJ%20Fry&author=CL%20Mackall&volume=174&publication_year=2005&pages=6571-6576&pmid=15905493&doi=10.4049/jimmunol.174.11.6571&)

11. Martin CE, et al. Interleukin-7 availability is maintained by a hematopoietic cytokine sink comprising innate lymphoid cells and T cells. Immunity. 2017;47:171–182. doi: 10.1016/j.immuni.2017.07.005.  [DOI](https://doi.org/10.1016/j.immuni.2017.07.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28723549/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunity&title=Interleukin-7%20availability%20is%20maintained%20by%20a%20hematopoietic%20cytokine%20sink%20comprising%20innate%20lymphoid%20cells%20and%20T%20cells&author=CE%20Martin&volume=47&publication_year=2017&pages=171-182&pmid=28723549&doi=10.1016/j.immuni.2017.07.005&)

12. Fairfax BP, et al. Peripheral CD8+ T cell characteristics associated with durable responses to immune checkpoint blockade in patients with metastatic melanoma. Nat. Med. 2020;26:193–199. doi: 10.1038/s41591-019-0734-6.  [DOI](https://doi.org/10.1038/s41591-019-0734-6) | [PMC free article](/articles/PMC7611047/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32042196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&title=Peripheral%20CD8+%20T%20cell%20characteristics%20associated%20with%20durable%20responses%20to%20immune%20checkpoint%20blockade%20in%20patients%20with%20metastatic%20melanoma&author=BP%20Fairfax&volume=26&publication_year=2020&pages=193-199&pmid=32042196&doi=10.1038/s41591-019-0734-6&)

13. Huang AC, et al. T-cell invigoration to tumour burden ratio associated with anti-PD-1 response. Nature. 2017;545:60–65. doi: 10.1038/nature22079.  [DOI](https://doi.org/10.1038/nature22079) | [PMC free article](/articles/PMC5554367/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28397821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=T-cell%20invigoration%20to%20tumour%20burden%20ratio%20associated%20with%20anti-PD-1%20response&author=AC%20Huang&volume=545&publication_year=2017&pages=60-65&pmid=28397821&doi=10.1038/nature22079&)

14. Shive CL, et al. Inflammation perturbs the IL-7 axis, promoting senescence and exhaustion that broadly characterize immune failure in treated HIV infection. J. Acquir. Immune Defic. Syndr. 2016;71:483–492. doi: 10.1097/QAI.0000000000000913.  [DOI](https://doi.org/10.1097/QAI.0000000000000913) | [PMC free article](/articles/PMC4788576/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26627102/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Acquir.%20Immune%20Defic.%20Syndr.&title=Inflammation%20perturbs%20the%20IL-7%20axis,%20promoting%20senescence%20and%20exhaustion%20that%20broadly%20characterize%20immune%20failure%20in%20treated%20HIV%20infection&author=CL%20Shive&volume=71&publication_year=2016&pages=483-492&pmid=26627102&doi=10.1097/QAI.0000000000000913&)

15. Miller CN, et al. IL-7 production in murine lymphatic endothelial cells and induction in the setting of peripherallymphopenia. Int. Immunol. 2013;25:471–483. doi: 10.1093/intimm/dxt012.  [DOI](https://doi.org/10.1093/intimm/dxt012) | [PMC free article](/articles/PMC3720065/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23657000/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20Immunol.&title=IL-7%20production%20in%20murine%20lymphatic%20endothelial%20cells%20and%20induction%20in%20the%20setting%20of%20peripherallymphopenia&author=CN%20Miller&volume=25&publication_year=2013&pages=471-483&pmid=23657000&doi=10.1093/intimm/dxt012&)

16. Tissue expression of IL7. Summary. The Human Protein Atlas. https://www.proteinatlas.org/ENSG00000104432-IL7/tissue  [https://www.proteinatlas.org/ENSG00000104432-IL7/tissue](https://www.proteinatlas.org/ENSG00000104432-IL7/tissue)

17. Uhlén M, et al. Proteomics. Tissue-based map of the human proteome. Science. 2015;347:1260419. doi: 10.1126/science.1260419.  [DOI](https://doi.org/10.1126/science.1260419) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25613900/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Proteomics.%20Tissue-based%20map%20of%20the%20human%20proteome&author=M%20Uhl%C3%A9n&volume=347&publication_year=2015&pages=1260419&pmid=25613900&doi=10.1126/science.1260419&)

18. Bates D, Mächler M, Bolker BM, Walker SC. Fitting linear mixed-effects models using lme4. J. Stat. Softw. 2015;67:1–48.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Stat.%20Softw.&title=Fitting%20linear%20mixed-effects%20models%20using%20lme4&author=D%20Bates&author=M%20M%C3%A4chler&author=BM%20Bolker&author=SC%20Walker&volume=67&publication_year=2015&pages=1-48&)

19. Morgan B, et al. Aiolos, a lymphoid restricted transcription factor that interacts with Ikaros to regulate lymphocyte differentiatisn. EMBO J. 1997;16:2004–2013. doi: 10.1093/emboj/16.8.2004.  [DOI](https://doi.org/10.1093/emboj/16.8.2004) | [PMC free article](/articles/PMC1169803/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9155026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EMBO%20J.&title=Aiolos,%20a%20lymphoid%20restricted%20transcription%20factor%20that%20interacts%20with%20Ikaros%20to%20regulate%20lymphocyte%20differentiatisn&author=B%20Morgan&volume=16&publication_year=1997&pages=2004-2013&pmid=9155026&doi=10.1093/emboj/16.8.2004&)

20. Hodson DJ, et al. Regulation of normal B-cell differentiation and malignant B-cell survival by OCT2. Proc. Natl Acad. Sci. USA. 2016;113:E2039–E2046. doi: 10.1073/pnas.1600557113.  [DOI](https://doi.org/10.1073/pnas.1600557113) | [PMC free article](/articles/PMC4833274/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26993806/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=Regulation%20of%20normal%20B-cell%20differentiation%20and%20malignant%20B-cell%20survival%20by%20OCT2&author=DJ%20Hodson&volume=113&publication_year=2016&pages=E2039-E2046&pmid=26993806&doi=10.1073/pnas.1600557113&)

21. Luo Y, Fujii H, Gerster T, Roeder RG. A novel B cell-derived coactivator potentiates the activation of immunoglobulin promoters by octamer-binding transcription factors. Cell. 1992;71:231–241. doi: 10.1016/0092-8674(92)90352-d.  [DOI](https://doi.org/10.1016/0092-8674(92)90352-d) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1423591/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=A%20novel%20B%20cell-derived%20coactivator%20potentiates%20the%20activation%20of%20immunoglobulin%20promoters%20by%20octamer-binding%20transcription%20factors&author=Y%20Luo&author=H%20Fujii&author=T%20Gerster&author=RG%20Roeder&volume=71&publication_year=1992&pages=231-241&pmid=1423591&doi=10.1016/0092-8674(92)90352-d&)

22. Fang H, Knezevic B, Burnham KL, Knight JC. XGR software for enhanced interpretation of genomic summary data, illustrated by application to immunological traits. Genome Med. 2016;8:129. doi: 10.1186/s13073-016-0384-y.  [DOI](https://doi.org/10.1186/s13073-016-0384-y) | [PMC free article](/articles/PMC5154134/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27964755/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med.&title=XGR%20software%20for%20enhanced%20interpretation%20of%20genomic%20summary%20data,%20illustrated%20by%20application%20to%20immunological%20traits&author=H%20Fang&author=B%20Knezevic&author=KL%20Burnham&author=JC%20Knight&volume=8&publication_year=2016&pages=129&pmid=27964755&doi=10.1186/s13073-016-0384-y&)

23. Watson RA, et al. Immune checkpoint blockade sensitivity and progression-free survival associates with baseline CD8+ T cell clone size and cytotoxicity. Sci. Immunol. 2021;6:eabj8825. doi: 10.1126/sciimmunol.abj8825.  [DOI](https://doi.org/10.1126/sciimmunol.abj8825) | [PMC free article](/articles/PMC7612602/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34597125/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci.%20Immunol.&title=Immune%20checkpoint%20blockade%20sensitivity%20and%20progression-free%20survival%20associates%20with%20baseline%20CD8+%20T%20cell%20clone%20size%20and%20cytotoxicity&author=RA%20Watson&volume=6&publication_year=2021&pages=eabj8825&pmid=34597125&doi=10.1126/sciimmunol.abj8825&)

24. Subudhi SK, et al. Clonal expansion of CD8 T cells in the systemic circulation precedes development of ipilimumab-induced toxicities. Proc. Natl Acad. Sci. USA. 2016;113:11919–11924. doi: 10.1073/pnas.1611421113.  [DOI](https://doi.org/10.1073/pnas.1611421113) | [PMC free article](/articles/PMC5081579/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27698113/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&title=Clonal%20expansion%20of%20CD8%20T%20cells%20in%20the%20systemic%20circulation%20precedes%20development%20of%20ipilimumab-induced%20toxicities&author=SK%20Subudhi&volume=113&publication_year=2016&pages=11919-11924&pmid=27698113&doi=10.1073/pnas.1611421113&)

25. Gagnon J, et al. Increased antigen responsiveness of naive CD8 T cells exposed to IL‐7 and IL‐21 is associated with decreased CD5 expression. Immunol. Cell Biol. 2010;88:451–460. doi: 10.1038/icb.2009.109.  [DOI](https://doi.org/10.1038/icb.2009.109) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20065997/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunol.%20Cell%20Biol.&title=Increased%20antigen%20responsiveness%20of%20naive%20CD8%20T%20cells%20exposed%20to%20IL%E2%80%907%20and%20IL%E2%80%9021%20is%20associated%20with%20decreased%20CD5%20expression&author=J%20Gagnon&volume=88&publication_year=2010&pages=451-460&pmid=20065997&doi=10.1038/icb.2009.109&)

26. Schluns KS, Kieper WC, Jameson SC, Lefrançois L. Interleukin-7 mediates the homeostasis of naïve and memory CD8 T cells in vivo. Nat. Immunol. 2000;1:426–432. doi: 10.1038/80868.  [DOI](https://doi.org/10.1038/80868) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11062503/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Immunol.&title=Interleukin-7%20mediates%20the%20homeostasis%20of%20na%C3%AFve%20and%20memory%20CD8%20T%20cells%20in%20vivo&author=KS%20Schluns&author=WC%20Kieper&author=SC%20Jameson&author=L%20Lefran%C3%A7ois&volume=1&publication_year=2000&pages=426-432&pmid=11062503&doi=10.1038/80868&)

27. Rutishauser RL, et al. Transcriptional repressor Blimp-1 promotes CD8+ T cell terminal differentiation and represses the acquisition of central memory T cell properties. Immunity. 2009;31:296–308. doi: 10.1016/j.immuni.2009.05.014.  [DOI](https://doi.org/10.1016/j.immuni.2009.05.014) | [PMC free article](/articles/PMC2783637/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19664941/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Immunity&title=Transcriptional%20repressor%20Blimp-1%20promotes%20CD8+%20T%20cell%20terminal%20differentiation%20and%20represses%20the%20acquisition%20of%20central%20memory%20T%20cell%20properties&author=RL%20Rutishauser&volume=31&publication_year=2009&pages=296-308&pmid=19664941&doi=10.1016/j.immuni.2009.05.014&)

28. Carrette F, Surh CD. IL-7 signaling and CD127 receptor regulation in the control of T cell homeostasis. Semin. Immunol. 2012;24:209–217. doi: 10.1016/j.smim.2012.04.010.  [DOI](https://doi.org/10.1016/j.smim.2012.04.010) | [PMC free article](/articles/PMC3367861/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22551764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Semin.%20Immunol.&title=IL-7%20signaling%20and%20CD127%20receptor%20regulation%20in%20the%20control%20of%20T%20cell%20homeostasis&author=F%20Carrette&author=CD%20Surh&volume=24&publication_year=2012&pages=209-217&pmid=22551764&doi=10.1016/j.smim.2012.04.010&)

29. Chiffelle J, et al. T-cell repertoire analysis and metrics of diversity and clonality. Curr. Opin. Biotechnol. 2020;65:284–295. doi: 10.1016/j.copbio.2020.07.010.  [DOI](https://doi.org/10.1016/j.copbio.2020.07.010) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32889231/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Biotechnol.&title=T-cell%20repertoire%20analysis%20and%20metrics%20of%20diversity%20and%20clonality&author=J%20Chiffelle&volume=65&publication_year=2020&pages=284-295&pmid=32889231&doi=10.1016/j.copbio.2020.07.010&)

30. Liu J, et al. An integrated TCGA pan-cancer clinical data resource to drive high-quality survival outcome analytics. Cell. 2018;173:400–416. doi: 10.1016/j.cell.2018.02.052.  [DOI](https://doi.org/10.1016/j.cell.2018.02.052) | [PMC free article](/articles/PMC6066282/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29625055/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=An%20integrated%20TCGA%20pan-cancer%20clinical%20data%20resource%20to%20drive%20high-quality%20survival%20outcome%20analytics&author=J%20Liu&volume=173&publication_year=2018&pages=400-416&pmid=29625055&doi=10.1016/j.cell.2018.02.052&)

31. Fairfax BP, et al. Innate immune activity conditions the effect of regulatory variants upon monocyte gene expression. Science. 2014;343:1246949. doi: 10.1126/science.1246949.  [DOI](https://doi.org/10.1126/science.1246949) | [PMC free article](/articles/PMC4064786/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24604202/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Innate%20immune%20activity%20conditions%20the%20effect%20of%20regulatory%20variants%20upon%20monocyte%20gene%20expression&author=BP%20Fairfax&volume=343&publication_year=2014&pages=1246949&pmid=24604202&doi=10.1126/science.1246949&)

32. Kim S, et al. Characterizing the genetic basis of innate immune response in TLR4-activated human monocytes. Nat. Commun. 2014;5:5236. doi: 10.1038/ncomms6236.  [DOI](https://doi.org/10.1038/ncomms6236) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25327457/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=Characterizing%20the%20genetic%20basis%20of%20innate%20immune%20response%20in%20TLR4-activated%20human%20monocytes&author=S%20Kim&volume=5&publication_year=2014&pages=5236&pmid=25327457&doi=10.1038/ncomms6236&)

33. Lee MN, et al. Common genetic variants modulate pathogen-sensing responses in human dendritic cells. Science. 2014;343:1246980. doi: 10.1126/science.1246980.  [DOI](https://doi.org/10.1126/science.1246980) | [PMC free article](/articles/PMC4124741/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24604203/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=Common%20genetic%20variants%20modulate%20pathogen-sensing%20responses%20in%20human%20dendritic%20cells&author=MN%20Lee&volume=343&publication_year=2014&pages=1246980&pmid=24604203&doi=10.1126/science.1246980&)

34. Helmink BA, et al. B cells and tertiary lymphoid structures promote immunotherapy response. Nature. 2020;577:549–555. doi: 10.1038/s41586-019-1922-8.  [DOI](https://doi.org/10.1038/s41586-019-1922-8) | [PMC free article](/articles/PMC8762581/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31942075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=B%20cells%20and%20tertiary%20lymphoid%20structures%20promote%20immunotherapy%20response&author=BA%20Helmink&volume=577&publication_year=2020&pages=549-555&pmid=31942075&doi=10.1038/s41586-019-1922-8&)

35. Cabrita R, et al. Tertiary lymphoid structures improve immunotherapy and survival in melanoma. Nature. 2020;577:561–565. doi: 10.1038/s41586-019-1914-8.  [DOI](https://doi.org/10.1038/s41586-019-1914-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31942071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Tertiary%20lymphoid%20structures%20improve%20immunotherapy%20and%20survival%20in%20melanoma&author=R%20Cabrita&volume=577&publication_year=2020&pages=561-565&pmid=31942071&doi=10.1038/s41586-019-1914-8&)

36. Lynch DH, Miller RE. Interleukin 7 promotes long-term in vitro growth of antitumor cytotoxic T lymphocytes with immunotherapeutic efficacy in vivo. J. Exp. Med. 1994;179:31–42. doi: 10.1084/jem.179.1.31.  [DOI](https://doi.org/10.1084/jem.179.1.31) | [PMC free article](/articles/PMC2191325/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8270874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Med.&title=Interleukin%207%20promotes%20long-term%20in%20vitro%20growth%20of%20antitumor%20cytotoxic%20T%20lymphocytes%20with%20immunotherapeutic%20efficacy%20in%20vivo&author=DH%20Lynch&author=RE%20Miller&volume=179&publication_year=1994&pages=31-42&pmid=8270874&doi=10.1084/jem.179.1.31&)

37. Gagnon J, et al. IL-6, in synergy with IL-7 or IL-15, stimulates TCR-independent proliferation and functional differentiation of CD8+ T lymphocytes. J. Immunol. 2008;180:7958–7968. doi: 10.4049/jimmunol.180.12.7958.  [DOI](https://doi.org/10.4049/jimmunol.180.12.7958) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18523259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.&title=IL-6,%20in%20synergy%20with%20IL-7%20or%20IL-15,%20stimulates%20TCR-independent%20proliferation%20and%20functional%20differentiation%20of%20CD8+%20T%20lymphocytes&author=J%20Gagnon&volume=180&publication_year=2008&pages=7958-7968&pmid=18523259&doi=10.4049/jimmunol.180.12.7958&)

38. Valbon SF, Condotta SA, Richer MJ. Regulation of effector and memory CD8+ T cell function by inflammatory cytokines. Cytokine. 2016;82:16–23. doi: 10.1016/j.cyto.2015.11.013.  [DOI](https://doi.org/10.1016/j.cyto.2015.11.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26688544/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cytokine&title=Regulation%20of%20effector%20and%20memory%20CD8+%20T%20cell%20function%20by%20inflammatory%20cytokines&author=SF%20Valbon&author=SA%20Condotta&author=MJ%20Richer&volume=82&publication_year=2016&pages=16-23&pmid=26688544&doi=10.1016/j.cyto.2015.11.013&)

39. Rathmell JC, Farkash EA, Gao W, Thompson CB. IL-7 enhances the survival and maintains the size of naive T cells. J. Immunol. 2001;167:6869–6876. doi: 10.4049/jimmunol.167.12.6869.  [DOI](https://doi.org/10.4049/jimmunol.167.12.6869) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11739504/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.&title=IL-7%20enhances%20the%20survival%20and%20maintains%20the%20size%20of%20naive%20T%20cells&author=JC%20Rathmell&author=EA%20Farkash&author=W%20Gao&author=CB%20Thompson&volume=167&publication_year=2001&pages=6869-6876&pmid=11739504&doi=10.4049/jimmunol.167.12.6869&)

40. Pellegrini M, et al. Adjuvant IL-7 antagonizes multiple cellular and molecular inhibitory networks to enhance immunotherapies. Nat. Med. 2009;15:528–536. doi: 10.1038/nm.1953.  [DOI](https://doi.org/10.1038/nm.1953) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19396174/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&title=Adjuvant%20IL-7%20antagonizes%20multiple%20cellular%20and%20molecular%20inhibitory%20networks%20to%20enhance%20immunotherapies&author=M%20Pellegrini&volume=15&publication_year=2009&pages=528-536&pmid=19396174&doi=10.1038/nm.1953&)

41. ElKassar N, Gress RE. An overview of IL-7 biology and its use in immunotherapy. J. Immunotoxicol. 2010;7:1–7. doi: 10.3109/15476910903453296.  [DOI](https://doi.org/10.3109/15476910903453296) | [PMC free article](/articles/PMC2826542/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20017587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Immunotoxicol.&title=An%20overview%20of%20IL-7%20biology%20and%20its%20use%20in%20immunotherapy&author=N%20ElKassar&author=RE%20Gress&volume=7&publication_year=2010&pages=1-7&pmid=20017587&doi=10.3109/15476910903453296&)

42. Love, M. I., Anders, S. & Huber, W. Differential analysis of count data—the DESeq2 package. https://bioc.ism.ac.jp/packages/3.4/bioc/vignettes/DESeq2/inst/doc/DESeq2.pdf (2016).  [https://bioc.ism.ac.jp/packages/3.4/bioc/vignettes/DESeq2/inst/doc/DESeq2.pdf](https://bioc.ism.ac.jp/packages/3.4/bioc/vignettes/DESeq2/inst/doc/DESeq2.pdf)

43. Pertea M, et al. StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. Nat. Biotechnol. 2015;33:290–295. doi: 10.1038/nbt.3122.  [DOI](https://doi.org/10.1038/nbt.3122) | [PMC free article](/articles/PMC4643835/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25690850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&title=StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads&author=M%20Pertea&volume=33&publication_year=2015&pages=290-295&pmid=25690850&doi=10.1038/nbt.3122&)

44. Bolotin DA, et al. MiXCR: software for comprehensive adaptive immunity profiling. Nat. Methods. 2015;12:380–381. doi: 10.1038/nmeth.3364.  [DOI](https://doi.org/10.1038/nmeth.3364) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25924071/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&title=MiXCR:%20software%20for%20comprehensive%20adaptive%20immunity%20profiling&author=DA%20Bolotin&volume=12&publication_year=2015&pages=380-381&pmid=25924071&doi=10.1038/nmeth.3364&)

45. Lun ATL, et al. EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data. Genome Biol. 2019;20:63. doi: 10.1186/s13059-019-1662-y.  [DOI](https://doi.org/10.1186/s13059-019-1662-y) | [PMC free article](/articles/PMC6431044/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30902100/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&title=EmptyDrops:%20distinguishing%20cells%20from%20empty%20droplets%20in%20droplet-based%20single-cell%20RNA%20sequencing%20data&author=ATL%20Lun&volume=20&publication_year=2019&pages=63&pmid=30902100&doi=10.1186/s13059-019-1662-y&)

46. Huang Y, McCarthy DJ, Stegle O. Vireo: Bayesian demultiplexing of pooled single-cell RNA-seq data without genotype reference. Genome Biol. 2019;20:273. doi: 10.1186/s13059-019-1865-2.  [DOI](https://doi.org/10.1186/s13059-019-1865-2) | [PMC free article](/articles/PMC6909514/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31836005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&title=Vireo:%20Bayesian%20demultiplexing%20of%20pooled%20single-cell%20RNA-seq%20data%20without%20genotype%20reference&author=Y%20Huang&author=DJ%20McCarthy&author=O%20Stegle&volume=20&publication_year=2019&pages=273&pmid=31836005&doi=10.1186/s13059-019-1865-2&)

47. Young MD, Behjati S. SoupX removes ambient RNA contamination from droplet-based single-cell RNA sequencing data. Gigascience. 2020;9:giaa151. doi: 10.1093/gigascience/giaa151.  [DOI](https://doi.org/10.1093/gigascience/giaa151) | [PMC free article](/articles/PMC7763177/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33367645/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gigascience&title=SoupX%20removes%20ambient%20RNA%20contamination%20from%20droplet-based%20single-cell%20RNA%20sequencing%20data&author=MD%20Young&author=S%20Behjati&volume=9&publication_year=2020&pages=giaa151&pmid=33367645&doi=10.1093/gigascience/giaa151&)

48. Hao Y, et al. Integrated analysis of multimodal single-cell data. Cell. 2021;184:3573–3587. doi: 10.1016/j.cell.2021.04.048.  [DOI](https://doi.org/10.1016/j.cell.2021.04.048) | [PMC free article](/articles/PMC8238499/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34062119/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Integrated%20analysis%20of%20multimodal%20single-cell%20data&author=Y%20Hao&volume=184&publication_year=2021&pages=3573-3587&pmid=34062119&doi=10.1016/j.cell.2021.04.048&)

49. Korsunsky I, et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Nat. Methods. 2019;16:1289–1296. doi: 10.1038/s41592-019-0619-0.  [DOI](https://doi.org/10.1038/s41592-019-0619-0) | [PMC free article](/articles/PMC6884693/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31740819/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&title=Fast,%20sensitive%20and%20accurate%20integration%20of%20single-cell%20data%20with%20Harmony&author=I%20Korsunsky&volume=16&publication_year=2019&pages=1289-1296&pmid=31740819&doi=10.1038/s41592-019-0619-0&)

50. Aran D, et al. Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage. Nat. Immunol. 2019;20:163–172. doi: 10.1038/s41590-018-0276-y.  [DOI](https://doi.org/10.1038/s41590-018-0276-y) | [PMC free article](/articles/PMC6340744/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30643263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Immunol.&title=Reference-based%20analysis%20of%20lung%20single-cell%20sequencing%20reveals%20a%20transitional%20profibrotic%20macrophage&author=D%20Aran&volume=20&publication_year=2019&pages=163-172&pmid=30643263&doi=10.1038/s41590-018-0276-y&)

51. van Dijk D, et al. Recovering gene interactions from single-cell data using data diffusion. Cell. 2018;174:716–729. doi: 10.1016/j.cell.2018.05.061.  [DOI](https://doi.org/10.1016/j.cell.2018.05.061) | [PMC free article](/articles/PMC6771278/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29961576/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Recovering%20gene%20interactions%20from%20single-cell%20data%20using%20data%20diffusion&author=D%20van%20Dijk&volume=174&publication_year=2018&pages=716-729&pmid=29961576&doi=10.1016/j.cell.2018.05.061&)
