# Roles of the m6A methyltransferases METTL3, METTL14, and WTAP in pulmonary tuberculosis

## Metadata
**Authors:** Tian-Ping Zhang, Rui Li, Li-Jun Wang, Qian Huang, Hong-Miao Li
**Journal:** Frontiers in Immunology
**Date:** 2022 Dec 7
**DOI:** [10.3389/fimmu.2022.992628](https://doi.org/10.3389/fimmu.2022.992628)
**PMID:** 36569923
**PMCID:** PMC9768477
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9768477/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9768477/pdf/fimmu-13-992628.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9768477/pdf/fimmu-13-992628.pdf)

## Abstract

**Objective:** 
The aim of the current study was to investigate the contributing role of gene variation and transcription levels among the m6A methyltransferases METTL3, METTL14, and WTAP in pulmonary tuberculosis (PTB).

**Methods:** 
A case-control study including 461 PTB patients and 467 normal controls was designed for genotyping. Three SNPs in METTL3 (rs1061027, rs1139130, rs1061026), three SNPs in METTL14 (rs62328061, rs4834698, rs1064034), and two SNPs in WTAP (rs1853259, rs11752345) were genotyped via the SNPscan™ technique. METTL3, METTL14, and WTAP transcription levels were determined in 78 PTB patients and 86 controls via quantitative real-time reverse-transcription PCR.

**Results:** 
Frequencies of the METTL14 rs62328061 GG genotype, WTAP rs11752345 CT genotype, and T allele were significantly increased in PTB patients compared to controls. An increased risk of rs62328061 was detected in a recessive model, and a decreased risk of rs11752345 was detected in a dominant model in the PTB group. METTL3 gene variation was not associated with PTB risk. The METTL3 rs1139130 GG genotype was significantly increased with drug resistance, and the G allele was significantly decreased with drug-induced liver injury in PTB patients. A reduced frequency of the METTL14 rs62328061 G allele was associated with leukopenia, a reduced frequency of the WTAP rs11752345 T allele was associated with sputum smear positivity, and a higher frequency of the METTL14 rs4834698 TC genotype was evident in PTB patients with hypoproteinemia. Compared to controls, METTL3, METTL14, and WTAP transcription levels in PTB patients were significantly decreased, and the level of WTAP was increased in PTB patients with drug resistance. METTL3 level was negatively associated with erythrocyte sedimentation rate and aspartate aminotransferase, and METTL14 level was negatively correlated with alanine aminotransferase and aspartate aminotransferase.

**Conclusion:** 
METTL14 rs62328061 and WTAP rs11752345 variants were associated with the genetic background of PTB, and METTL3, METTL14, and WTAP levels were abnormally decreased, suggesting that these m6A methyltransferases may play important roles in PTB.

Keywords: m6A methyltransferase, single nucleotide polymorphisms, pulmonary tuberculosis, METTL3, METTL14, WTAP

### Objective

The aim of the current study was to investigate the contributing role of gene variation and transcription levels among the m6A methyltransferases METTL3, METTL14, and WTAP in pulmonary tuberculosis (PTB).

### Methods

A case-control study including 461 PTB patients and 467 normal controls was designed for genotyping. Three SNPs in *METTL3* (rs1061027, rs1139130, rs1061026), three SNPs in *METTL14* (rs62328061, rs4834698, rs1064034), and two SNPs in *WTAP* (rs1853259, rs11752345) were genotyped *via* the SNPscan™ technique. *METTL3*, *METTL14*, and *WTAP* transcription levels were determined in 78 PTB patients and 86 controls *via* quantitative real-time reverse-transcription PCR.

### Results

Frequencies of the *METTL14* rs62328061 GG genotype, *WTAP* rs11752345 CT genotype, and T allele were significantly increased in PTB patients compared to controls. An increased risk of rs62328061 was detected in a recessive model, and a decreased risk of rs11752345 was detected in a dominant model in the PTB group. *METTL3* gene variation was not associated with PTB risk. The *METTL3* rs1139130 GG genotype was significantly increased with drug resistance, and the G allele was significantly decreased with drug-induced liver injury in PTB patients. A reduced frequency of the *METTL14* rs62328061 G allele was associated with leukopenia, a reduced frequency of the *WTAP* rs11752345 T allele was associated with sputum smear positivity, and a higher frequency of the *METTL14* rs4834698 TC genotype was evident in PTB patients with hypoproteinemia. Compared to controls, *METTL3*, *METTL14*, and *WTAP* transcription levels in PTB patients were significantly decreased, and the level of WTAP was increased in PTB patients with drug resistance. METTL3 level was negatively associated with erythrocyte sedimentation rate and aspartate aminotransferase, and METTL14 level was negatively correlated with alanine aminotransferase and aspartate aminotransferase.

### Conclusion

*METTL14* rs62328061 and *WTAP* rs11752345 variants were associated with the genetic background of PTB, and METTL3, METTL14, and WTAP levels were abnormally decreased, suggesting that these m6A methyltransferases may play important roles in PTB.

## Introduction

Tuberculosis (TB) is a common infectious disease caused by *Mycobacterium tuberculosis* (MTB), and it remains one of the most serious public health problems worldwide ([1](#B1)). A World Health Organization update reported that there was an estimated 9.9 million new incident TB patients globally in 2021 ([2](#B2)). It is well known that people infected with MTB can develop several possible outcomes, such as MTB clearance, primary TB, latent TB infection, and active TB. Studies indicate that approximately 10% of individuals infected with MTB will eventually progress to active TB ([3](#B3), [4](#B4)). The occurrence and development of TB is mainly influenced by complex interactions between the MTB strain, the external environment, and genetic factors ([5](#B5), [6](#B6)). Host genetics are the essential factor determining disease susceptibility and outcomes after MTB infection in many case-control studies, animal model studies, and twin and family studies ([7](#B7), [8](#B8)). Continuing studies on the roles of genetic variants in TB susceptibility would contribute to the development of advantageous approaches to TB prevention, diagnosis, and treatment, and many genetic variations have been shown to be associated with susceptibility to TB ([9](#B9), [10](#B10)).

N6-methyladenosine (m6A) is formed *via* methylation of the sixth N atom on the adenine base, which is reportedly the most abundant post-transcriptional modification of RNA in eukaryotes, particularly messenger RNA (mRNA) ([11](#B11)). m6A regulates post-transcriptional mRNA levels in a dynamic and reversible manner ([12](#B12)). m6A modification is regulated by several key regulators, including RNA methyltransferases (METTL3, METTL14, and WTAP), demethylases, and m6A-binding proteins ([13](#B13)). m6A modification is involved in many important biological processes, and it is closely related to the occurrence and development of many diseases.

Epigenetic modification, including DNA methylation and noncoding RNAs, reportedly contributes to pulmonary TB (PTB) progression ([14](#B14), [15](#B15)). In recent years m6A methylation was also found in the *MTB* genome ([16](#B16)). Therefore, m6A methylation may be involved in the pathogenesis of PTB. Strong evidence indicates that abnormal expression of m6A key regulators may result in abnormal RNA m6A modification, leading to a variety of diseases ([17](#B17), [18](#B18)). In addition, some functional single nucleotide polymorphisms (SNPs) in specific regions of m6A key regulator genes may influence m6A methylation, thereby affecting disease development. For example, several SNPs in *METTL3* and *METTL14* genes have been associated with neuroblastoma, Wilms’ tumor, acute lymphoblastic leukemia, and autoimmune thyroid disease ([19](#B19)–[22](#B22)). In a previous study, genetic variations in m6A demethylases FTO could affect susceptibility to PTB ([23](#B23)), but few studies have investigated associations between variation in key m6A modification regulatory genes and PTB risk. The current epidemiological study was conducted to identify novel m6A methyltransferases SNPs associated with susceptibility to PTB. The study assessed associations between three m6A methyltransferase genes (*METTL3*, *METTL14*, and *WTAP*) SNPs, as well as their transcription levels, and PTB risk.

## Materials and methods

### Study subjects

In this case-control study, 461 PTB patients and 467 normal controls were consecutively enrolled, and associations between m6A methyltransferase gene polymorphisms and PTB susceptibility were analyzed. Then, 78 PTB patients and 86 normal controls were enrolled to detect m6A methyltransferase transcription levels. All PTB patients were recruited from the Department of Tuberculosis at Anhui Chest Hospital, whereas normal controls were selected from a health center in the same area. PTB patients were diagnosed by specialists based on indicative clinical symptoms, chest radiography, sputum and/or bronchoalveolar lavage fluid MTB culture, microscopy for acid fast bacilli, and effects of anti-TB treatment. The exclusion criteria included HIV positivity, hepatitis, malignancy, and immunodeficiency. Individuals without a history of TB, malignant tumor, HIV or other infectious diseases were included in the study as normal controls. The control group was required to be asymptomatic with negative sputum smears and cultures, and normal chest radiographs. All subjects were of Chinese Han ethnicity.

The study was approved by the Medical Ethics Committee of Anhui Medical University (approval number 20200250). After obtaining informed consent, peripheral blood samples and relevant information were collected from each participant with the help of professional physicians. The data collected included basic demographic characteristics and some clinical data such as fever, drug resistance, drug-induced liver injury (DILI), pulmonary infection, leukopenia, sputum smears, total bilirubin, aspartate aminotransferase (AST), alanine aminotransferase (ALT), and erythrocyte sedimentation rate (ESR).

### SNP selection

The three m6A methyltransferases METTL3, METTL14, and WTAP were included in the analyses, and the functional SNPs in their corresponding genes were chosen based on previous studies ([24](#B24)). Existing studies investigating associations between *METTL3*, *METTL14*, and *WTAP* gene polymorphisms and human diseases were systematically reviewed, and the SNPs related to human diseases were identified. Ensembl Genome Browser 85 and CHBS_1000g software were then used to obtain genotype data on these genes derived from Han Chinese people living in Beijing, and the tag SNPs of these genes were selected *via* the pairwise option of HaploView 4.0 software (Cambridge, MA, USA). The tag SNP selection criteria were: (1) a minor allele frequency ≥ 5% in CHB_1000g, and an *r* ^2^ threshold > 0.8; (2) the common SNPs located within the chromosome locus transcribed into *METTL3*, *METTL14*, or *WTAP* and their flanking 2000 bp regions were captured; (3) potentially functional SNPs located within the 5’ untranslated region, 3’ untranslated region, and exon were preferentially selected. Lastly, three SNPs in the *METTL3* gene (rs1061027, rs1139130, rs1061026), three SNPs in the *METTL14* gene (rs62328061, rs4834698, rs1064034), and two SNPs in the *WTAP* gene (rs1853259, rs11752345) were selected.

### DNA extraction and genotyping

Samples of approximately 5 mL of peripheral early morning fasting blood were collected from all subjects, and genomic DNA was extracted *via* a Flexi Gene-DNA Kit (Qiagen, Valencia, CA) for genotyping. Genotyping of all DNA samples for the selected SNPs was conducted *via* the Three SNPs in METTL3 (rs1061027, rs1139130, rs1061026), three SNPs in METTL14 (rs62328061, rs4834698, rs1064034), and two SNPs in WTAP (rs1853259, rs11752345) were genotyped *via* the SNPscan™ technique, with technical support from the Center for Genetic and Genomic Analysis, Genesky Biotechnologies Inc. (Shanghai). Only participants with 100% genotyping success for the selected SNPs were included in the final analysis.

### Quantitative real-time reverse-transcription PCR

Peripheral blood mononuclear cells (PBMCs) were isolated from 3 mL anticoagulant peripheral blood, and total RNA was extracted using TRIzol Reagent (Invitrogen, Carlsbad, CA, USA). A NanoDrop 2000 spectrophotometer (Thermo Scientific, USA) was used to determine total RNA concentrations. Total RNA was then reverse transcribed into cDNA by the PrimeScript™ RT Reagent Kit (Takara Bio Inc., Japan). In this experiment a 20-μL reverse transcription reaction system with a maximum of 1 μg total RNA was used, and the amount of total RNA needed in the reaction system was determined based on the RNA concentration.

METTL3, METTL14, and WTAP mRNA levels in PBMCs were detected *via* quantitative real-time reverse transcription (qRT) PCR with SYBR Green (SYBR Premix Ex Taq II, Takara Bio Inc., Japan). qRT-PCRs were conducted using a QuantStudio 12K Flex Real-Time PCR System (Applied Biosystems, Foster City, CA, USA), and the cycle conditions of reactions were 95°C for 1 min, followed by 42 cycles at 95°C for 10 sec, 60°C for 30 sec, and 72°C for 1 min. Relative transcription levels of METTL3, METTL14, and WTAP were calculated *via* comparisons with the housekeeping gene (internal control) β-actin in the same sample, and the 2^-△△Ct^ method was used to express levels ([25](#B25)).

### Statistical analysis

Whether the genotypes distribution of all SNPs in normal controls were in Hardy-Weinberg equilibrium was assessed *via* the Chi-square test. Associations between m6A methyltransferase gene polymorphisms and PTB susceptibility were estimated *via* odds ratios (*OR*) and 95% confidence intervals (*CI*) with logistic regression analyses. Two genetic models (dominant and recessive) were used to analyze associations between these SNPs and PTB risk, and haplotype analysis was conducted *via* SHEsis software ([26](#B26)). Transcription levels of *METTL3*, *METTL14*, and *WTAP* are expressed as median and quartile intervals, and the Mann-Whitney *U* test and the Kruskal-Wallis *H* test were respectively used to evaluate differences in these m6A methyltransferase levels between two groups and among three groups. Correlations between these m6A methyltransferase levels and experimental indexes in PTB patients were analyzed *via* the Spearman rank correlation coefficient test. All statistical analyses were conducted with SPSS 23.0 (SPSS Inc., IL, USA), and two-sided *p* values of < 0.05 were deemed to indicate statistical significance.

## Results

### Associations between m6A methyltransferase gene polymorphisms and PTB susceptibility

The PTB group genotyped included 194 females and 267 males, with a mean age of 45.56 ± 17.75 years. The normal control group genotyped included 265 females and 202 males, with a mean age of 43.38 ± 13.86 years. The genotype distributions of all SNPs in normal controls were in Hardy-Weinberg equilibrium, and the allele and genotype frequencies of these SNPs are shown in [Table 1](#T1).

### Table 1.

| SNP | Analyze model |  | PTB patients | Controls | P value | OR (95% CI) |
| --- | --- | --- | --- | --- | --- | --- |
| METTL3 |  |  |  |  |  |  |
| rs1139130 | Genotype | GG | 52 (11.28) | 65 (13.92) | 0.327 | 1.065 (0.808,1.403) |
|   |   | AG | 217 (47.07) | 207 (44.33) | 0.656 | 0.813 (0.536,1.231) |
|   |   | AA | 192 (41.65) | 195 (41.76) | Reference |  |
|   | Allele | G | 321 (34.82) | 337 (36.08) | 0.599 | 1.020 (0.953,1.091) |
|   |   | A | 601 (65.18) | 597 (63.92) | Reference |  |
|   | Dominant model | AA | 192 (41.65) | 195 (41.76) | 0.974 | 0.997 (0.857,1.161) |
|   |   | AG+GG | 269 (58.35) | 272 (58.24) | Reference |  |
|   | Recessive model | GG | 52 (11.28) | 65 (13.92) | 0.226 | 1.031 (0.981,1.082) |
|   |   | AG+AA | 409 (88.72) | 402 (86.08) | Reference |  |
| rs1061026 | Genotype | GG | 4 (0.87) | 3 (0.64) | 0.723 | 1.313 (0.292,5.905) |
|   |   | TG | 68 (14.75) | 81 (17.34) | 0.289 | 0.829 (0.581,1.175) |
|   |   | TT | 389 (84.38) | 383 (82.01) | Reference |  |
|   | Allele | G | 76 (8.24) | 87 (9.31) | 0.415 | 1.012 (0.984,1.041) |
|   |   | T | 846 (91.76) | 847 (90.69) | Reference |  |
|   | Dominant model | TT | 389 (84.38) | 383 (82.01) | 0.335 | 0.868 (0.651,1.157) |
|   |   | TG+GG | 72 (15.62) | 84 (17.99) | Reference |  |
|   | Recessive model | GG | 4 (0.87) | 3 (0.64) | 0.692 | 1.351 (0.304,6.002) |
|   |   | TG+TT | 457 (99.13) | 464 (99.36) | Reference |  |
| rs1061027 | Genotype | AA | 15 (3.25) | 21 (4.50) | 0.328 | 0.712 (0.360,1.407) |
|   |   | CA | 142 (30.80) | 143 (30.62) | 0.943 | 0.990 (0.747,1.311) |
|   |   | CC | 304 (65.94) | 303 (64.88) | Reference |  |
|   | Allele | C | 172 (18.66) | 185 (19.81) | 0.529 | 1.014 (0.970,1.060) |
|   |   | A | 750 (81.34) | 749 (80.19) | Reference |  |
|   | Dominant model | CC | 304 (65.94) | 303 (64.88) | 0.734 | 1.016 (0.926,1.116) |
|   |   | CA+AA | 157 (34.06) | 164 (35.12) | Reference |  |
|   | Recessive model | AA | 15 (3.25) | 21 (4.50) | 0.327 | 0.724 (0.378,1.386) |
|   |   | CA+CC | 446 (96.75) | 446 (95.50) | Reference |  |
| METTL14 |  |  |  |  |  |  |
| rs1064034 | Genotype | AA | 41 (8.89) | 37 (7.92) | 0.858 | 1.045 (0.645,1.692) |
|   |   | AT | 192 (41.65) | 215 (46.04) | 0.211 | 0.842 (0.643,1.102) |
|   |   | TT | 228 (49.46) | 215 (46.04) | Reference |  |
|   | Allele | A | 274 (29.72) | 289 (30.94) | 0.566 | 1.018 (0.958,1.081) |
|   |   | T | 648 (70.28) | 645 (69.06) | Reference |  |
|   | Dominant model | TT | 228 (49.46) | 215 (46.04) | 0.297 | 0.937 (0.828,1.059) |
|   |   | AT+AA | 233 (50.54) | 252 (53.96) | Reference |  |
|   | Recessive model | AA | 41 (8.89) | 37 (7.92) | 0.594 | 1.123 (0.734,1.718) |
|   |   | AT+TT | 420 (91.11) | 430 (92.08) | Reference |  |
| rs62328061 | Genotype | GG | 18 (3.90) | 7 (1.50) | 0.043 | 2.501 (1.031,6.069) |
|   |   | AG | 114 (24.73) | 140 (29.98) | 0.117 | 0.792 (0.592,1.060) |
|   |   | AA | 329 (71.37) | 320 (68.52) | Reference |  |
|   | Allele | G | 150 (16.27) | 154 (16.49) | 0.898 | 1.003 (0.963,1.044) |
|   |   | A | 772 (83.73) | 780 (83.51) | Reference |  |
|   | Dominant model | AA | 329 (71.37) | 320 (68.52) | 0.345 | 0.910 (0.747,1.107) |
|   |   | AG+GG | 132 (28.63) | 147 (31.48) | Reference |  |
|   | Recessive model | GG | 18 (3.90) | 7 (1.50) | 0.024 | 2.605 (1.098,6.177) |
|   |   | AG+AA | 443 (96.10) | 460 (98.50) | Reference |  |
| rs4834698 | Genotype | CC | 108 (23.43) | 97 (20.77) | 0.786 | 1.053 (0.727,1.524) |
|   |   | TC | 225 (48.81) | 249 (53.32) | 0.314 | 0.854 (0.628,1.161) |
|   |   | TT | 128 (27.77) | 121 (25.91) | Reference |  |
|   | Allele | C | 441 (47.83) | 443 (47.43) | 0.863 | 1.008 (0.917,1.109) |
|   |   | T | 481 (52.17) | 491 (52.57) | Reference |  |
|   | Dominant model | TT | 128 (27.77) | 121 (25.91) | 0.524 | 0.975 (0.902,1.054) |
|   |   | TC +CC | 333 (72.23) | 346 (74.09) | Reference |  |
|   | Recessive model | CC | 108 (23.43) | 97 (20.77) | 0.329 | 1.128 (0.885,1.437) |
|   |   | TC +TT | 353 (76.57) | 370 (79.23) | Reference |  |
| WTAP |  |  |  |  |  |  |
| rs11752345 | Genotype | TT | 1 (0.22) | 4 (0.86) | 0.246 | 0.273 (0.030,2.449) |
|   |   | CT | 83 (18.00) | 52 (11.13) | 0.004 | 1.740 (1.120,2.529) |
|   |   | CC | 377 (81.78) | 411 (88.01) | Reference |  |
|   | Allele | T | 85 (9.22) | 60 (6.42) | 0.025 | 0.970 (0.945,0.996) |
|   |   | C | 837 (90.78) | 874 (93.58) | Reference |  |
|   | Dominant model | CC | 377 (81.78) | 411 (88.01) | 0.008 | 1.520 (1.112,2.077) |
|   |   | CT+TT | 84 (18.22) | 56 (11.99) | Reference |  |
|   | Recessive model | TT | 1 (0.22) | 4 (0.86) | 0.183 | 0.253 (0.028,2.257) |
|   |   | CT+CC | 460 (99.78) | 463 (99.14) | Reference |  |
| rs1853259 | Genotype | GG | 63 (13.67) | 74 (15.85) | 0.497 | 0.872 (0.586,1.297) |
|   |   | AG | 229 (49.67) | 220 (47.11) | 0.658 | 1.066 (0.804,1.412) |
|   |   | AA | 169 (36.66) | 173 (37.04) | Reference |  |
|   | Allele | G | 355 (38.5) | 368 (39.4) | 0.692 | 0.977 (0.799,1.161) |
|   |   | A | 567 (61.5) | 566 (60.6) | Reference |  |
|   | Dominant model | AA | 169 (36.66) | 173 (37.04) | 0.903 | 1.006 (0.912,1.110) |
|   |   | AG+GG | 292 (63.34) | 294 (62.96) | Reference |  |
|   | Recessive model | GG | 292 (13.67) | 294 (15.85) | 0.858 | 0.989 (0.875,1.118) |
|   |   | AG+GG | 398 (86.33) | 393 (84.15) | Reference |  |

Table 1 Caption: Association between METTL3, METTL14, and WTAP genes polymorphism and PTB risk.

In the *METTL14* gene, the frequency of the rs62328061 GG genotype was significantly higher in PTB patients than in normal controls, and the genotype was associated with PTB susceptibility in the recessive model (GG versus AA *p* = 0.043; GG versus AG+AA *p* = 0.024). There was no significant association between the rs1064034 variant and PTB risk. In comparisons of *WTAP* rs11752345 variant genotype and allele frequencies between the PTB group and control group, the CT genotype and T allele frequencies were significantly increased in PTB patients (CT versus CC *p* = 0.004; T versus C *p* = 0.025). There was a decreased risk of the rs11752345 variant in the dominant model (CC versus CT+TT *p* = 0.008). PTB risk was not significantly associated with the *WTAP* rs1853259 variant, or with the *METTL3* rs1139130, rs1061026, or rs1061027 variants (all *p* > 0.05).

A case-only analysis was performed to analyze associations between *METTL3*, *METTL14*, and *WTAP* gene variants and common clinical features of PTB ([Table 2](#T2)). *METTL3* gene rs1139130 GG genotype frequency was significantly positively associated with drug resistance (*p* = 0.017), whereas rs1139130 G allele frequency was significantly negatively associated with DILI (*p* = 0.041) in PTB patients. The *METTL14* gene G allele frequency of the rs62328061 variant was significantly associated with a decreased risk of leukopenia (*p* = 0.016), and a higher frequency of the rs4834698 TC genotype was observed in PTB patients with hypoproteinemia (*p* = 0.042). A decreased frequency of the *WTAP* rs11752345 T allele was significantly associated with sputum smear positivity (*p* = 0.041). There were no significant associations between other SNPs and the clinical features of PTB.

### Table 2.

| SNP | Allele | Clinical features | Group | Genotype n (%) |  |  | P value | Allele n (%) |  | P value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| (M/m) | MM | Mm | mm | M | m |  |  |  |  |  |
| METTL3 |  |  |  |  |  |  |  |  |  |  |
| rs1139130 | A/G | fever | + | 37 (51.39) | 27 (37.5) | 8 (11.11) | 0.165 | 101 (70.14) | 43 (29.86) | 0.174 |
|   |   |   | – | 155 (39.85) | 190 (48.84) | 44 (11.31) |   | 500 (64.27) | 278 (35.73) |   |
|   |   | drug resistance | + | 32 (40.51) | 30 (37.97) | 17 (21.52) | 0.017 | 94 (59.49) | 64 (40.51) | 0.114 |
|   |   |   | – | 195 (42.39) | 217 (47.17) | 48 (10.43) |   | 607 (65.98) | 313 (34.02) |   |
|   |   | DILI | + | 39 (51.32) | 32 (42.11) | 5 (6.58) | 0.119 | 110 (72.37) | 42 (27.63) | 0.041 |
|   |   |   | – | 188 (40.6) | 215 (46.44) | 60 (12.96) |   | 591 (63.82) | 335 (36.18) |   |
|   |   | pulmonary infection | + | 45 (48.39) | 37 (39.78) | 11 (11.83) | 0.377 | 127 (68.28) | 59 (31.72) | 0.307 |
|   |   |   | – | 182 (40.81) | 210 (47.09) | 54 (12.11) |   | 574 (64.35) | 318 (35.65) |   |
|   |   | hypoproteinemia | + | 30 (53.57) | 20 (35.71) | 6 (10.71) | 0.179 | 80 (71.43) | 32 (28.57) | 0.133 |
|   |   |   | – | 197 (40.79) | 227 (47) | 59 (12.22) |   | 621 (64.29) | 345 (35.71) |   |
|   |   | leukopenia | + | 15 (41.67) | 18 (50.00) | 3 (8.33) | 0.744 | 48 (66.67) | 24 (33.33) | 0.763 |
|   |   |   | – | 212 (42.15) | 229 (45.53) | 62 (12.33) |   | 653 (64.91) | 353 (35.09) |   |
|   |   | sputum smear-positive | + | 60 (39.22) | 72 (47.06) | 21 (13.73) | 0.69 | 192 (62.75) | 114 (37.25) | 0.390 |
|   |   |   | – | 149 (42.94) | 157 (45.24) | 41 (11.82) |   | 455 (65.56) | 239 (34.44) |   |
| rs1061026 | T/G | fever | + | 61 (84.72) | 10 (13.89) | 1 (1.39) | 0.856 | 132 (91.67) | 12 (8.33) | 0.966 |
|   |   |   | – | 328 (84.32) | 58 (14.91) | 3 (0.77) |   | 714 (91.77) | 64 (8.23) |   |
|   |   | drug resistance | + | 67 (84.81) | 11 (13.92) | 1 (1.27) | 0.786 | 145 (91.77) | 13 (8.23) | 0.917 |
|   |   |   | – | 385 (83.7) | 72 (15.65) | 3 (0.65) |   | 842 (91.52) | 78 (8.48) |   |
|   |   | DILI | + | 69 (90.79) | 7 (9.21) | 0 (0) | 0.186 | 145 (95.39) | 7 (4.61) | 0.066 |
|   |   |   | – | 383 (82.72) | 76 (16.41) | 4 (0.86) |   | 842 (90.93) | 84 (9.07) |   |
|   |   | pulmonary infection | + | 83 (89.25) | 10 (10.75) | 0 (0) | 0.247 | 176 (94.62) | 10 (5.38) | 0.098 |
|   |   |   | – | 369 (82.74) | 73 (16.37) | 4 (0.90) |   | 811 (90.92) | 81 (9.08) |   |
|   |   | hypoproteinemia | + | 46 (82.14) | 10 (17.86) | 0 (0) | 0.694 | 102 (91.07) | 10 (8.93) | 0.845 |
|   |   |   | – | 406 (84.06) | 73 (15.11) | 4 (0.83) |   | 885 (91.61) | 81 (8.39) |   |
|   |   | leukopenia | + | 32 (88.89) | 4 (11.11) | 0 (0) | 0.649 | 68 (94.44) | 4 (5.56) | 0.362 |
|   |   |   | – | 420 (83.50) | 79 (15.71) | 4 (0.80) |   | 919 (91.35) | 87 (8.65) |   |
|   |   | sputum smear-positive | + | 131 (85.62) | 21 (13.73) | 1 (0.65) | 0.743 | 283 (92.48) | 23 (7.52) | 0.504 |
|   |   |   | – | 288 (83.00) | 57 (16.43) | 2 (0.58) |   | 633 (91.21) | 61 (8.79) |   |
| rs1061027 | C/A | fever | + | 46 (63.89) | 23 (31.94) | 3 (4.17) | 0.857 | 115 (79.86) | 29 (20.14) | 0.619 |
|   |   |   | – | 258 (66.32) | 119 (30.59) | 12 (3.08) |   | 635 (81.62) | 143 (18.38) |   |
|   |   | drug resistance | + | 48 (60.76) | 25 (31.65) | 6 (7.59) | 0.066 | 121 (76.58) | 37 (23.42) | 0.095 |
|   |   |   | – | 308 (66.96) | 140 (30.43) | 12 (2.61) |   | 756 (82.17) | 164 (17.83) |   |
|   |   | DILI | + | 47 (61.84) | 27 (35.53) | 2 (2.63) | 0.586 | 121 (79.61) | 31 (20.39) | 0.550 |
|   |   |   | – | 309 (66.74) | 138 (29.81) | 16 (3.46) |   | 756 (81.64) | 170 (18.36) |   |
|   |   | pulmonary infection | + | 61 (65.59) | 27 (29.03) | 5 (5.38) | 0.474 | 149 (80.11) | 37 (19.89) | 0.631 |
|   |   |   | – | 295 (66.14) | 138 (30.94) | 13 (2.91) |   | 728 (81.61) | 164 (18.39) |   |
|   |   | hypoproteinemia | + | 41 (73.21) | 14 (25.00) | 1 (1.79) | 0.454 | 96 (85.71) | 16 (14.29) | 0.211 |
|   |   |   | – | 315 (65.22) | 151 (31.26) | 17 (3.52) |   | 781 (80.85) | 185 (19.15) |   |
|   |   | leukopenia | + | 22 (61.11) | 13 (36.11) | 1 (2.78) | 0.756 | 57 (79.17) | 15 (20.83) | 0.622 |
|   |   |   | – | 334 (66.4) | 152 (30.22) | 17 (3.38) |   | 820 (81.51) | 186 (18.49) |   |
|   |   | sputum smear-positive | + | 95 (62.09) | 52 (33.99) | 6 (3.92) | 0.548 | 242 (79.08) | 64 (20.92) | 0.305 |
|   |   |   | – | 233 (67.15) | 102 (29.39) | 12 (3.46) |   | 568 (81.84) | 126 (18.16) |   |
| METTL14 |  |  |  |  |  |  |  |  |  |  |
| rs1064034 | T/A | fever | + | 37 (51.39) | 33 (45.83) | 2 (2.78) | 0.135 | 107 (74.31) | 37 (25.69) | 0.250 |
|   |   |   | – | 191 (49.10) | 159 (40.87) | 39 (10.03) |   | 541 (69.54) | 237 (30.46) |   |
|   |   | drug resistance | + | 46 (58.23) | 26 (32.91) | 7 (8.86) | 0.295 | 118 (74.68) | 40 (25.32) | 0.301 |
|   |   |   | – | 228 (49.57) | 194 (42.17) | 38 (8.26) |   | 650 (70.65) | 270 (29.35) |   |
|   |   | DILI | + | 40 (52.63) | 31 (40.79) | 5 (6.58) | 0.824 | 111 (73.03) | 41 (26.97) | 0.600 |
|   |   |   | – | 234 (50.54) | 189 (40.82) | 40 (8.64) |   | 657 (70.95) | 269 (29.05) |   |
|   |   | pulmonary infection | + | 45 (48.39) | 42 (45.16) | 6 (6.45) | 0.566 | 132 (70.97) | 54 (29.030) | 0.927 |
|   |   |   | – | 229 (51.35) | 178 (39.91) | 39 (8.74) |   | 636 (71.30) | 256 (28.70) |   |
|   |   | hypoproteinemia | + | 32 (57.14) | 21 (37.50) | 3 (5.36) | 0.515 | 95 (71.97) | 37 (28.03) | 0.764 |
|   |   |   | – | 242 (50.10) | 199 (41.20) | 42 (8.70) |   | 683 (70.70) | 283 (29.30) |   |
|   |   | leukopenia | + | 23 (63.89) | 12 (33.33) | 1 (2.78) | 0.196 | 58 (80.56) | 14 (19.44) | 0.071 |
|   |   |   | – | 251 (49.90) | 208 (41.35) | 44 (8.75) |   | 710 (70.58) | 296 (29.42) |   |
|   |   | sputum smear-positive | + | 82 (53.59) | 55 (35.95) | 16 (10.46) | 0.301 | 219 (71.57) | 87 (28.43) | 0.828 |
|   |   |   | – | 172 (49.57) | 148 (42.65) | 27 (7.78) |   | 492 (70.89) | 202 (29.11) |   |
| rs62328061 | A/G | fever | + | 51 (70.83) | 20 (27.78) | 1 (1.39) | 0.426 | 122 (84.72) | 22 (15.28) | 0.726 |
|   |   |   | – | 278 (71.47) | 94 (24.16) | 17 (4.37) |   | 650 (83.55) | 128 (16.45) |   |
|   |   | drug resistance | + | 58 (73.42) | 19 (24.05) | 2 (2.53) | 0.906 | 135 (85.44) | 23 (14.56) | 0.725 |
|   |   |   | – | 332 (72.17) | 112 (24.35) | 16 (3.48) |   | 776 (84.35) | 144 (15.65) |   |
|   |   | DILI | + | 55 (72.37) | 19 (25.00) | 2 (2.63) | 0.928 | 129 (84.87) | 23 (15.13) | 0.895 |
|   |   |   | – | 335 (72.35) | 112 (24.19) | 16 (3.46) |   | 782 (84.45) | 144 (15.55) |   |
|   |   | pulmonary infection | + | 68 (73.12) | 23 (24.73) | 2 (2.15) | 0.781 | 159 (85.48) | 27 (14.52) | 0.686 |
|   |   |   | – | 322 (72.2) | 108 (24.22) | 16 (3.59) |   | 752 (84.03) | 140 (15.7) |   |
|   |   | hypoproteinemia | + | 38 (67.86) | 16 (28.57) | 2 (3.57) | 0.721 | 92 (82.14) | 20 (17.86) | 0.465 |
|   |   |   | – | 352 (72.88) | 115 (23.81) | 16 (3.31) |   | 819 (84.78) | 147 (15.22) |   |
|   |   | leukopenia | + | 32 (88.89) | 4 (11.11) | 0 (0) | 0.064 | 68 (94.44) | 4 (5.56) | 0.016 |
|   |   |   | – | 358 (71.17) | 127 (25.25) | 18 (3.58) |   | 843 (83.80) | 163 (16.20) |   |
|   |   | sputum smear-positive | + | 109 (71.24) | 37 (24.18) | 7 (4.58) | 0.731 | 255 (83.33) | 51 (16.67) | 0.537 |
|   |   |   | – | 253 (72.91) | 83 (23.92) | 11 (3.17) |   | 589 (84.87) | 105 (15.13) |   |
| rs4834698 | T/C | fever | + | 15 (20.83) | 40 (55.56) | 17 (23.61) | 0.321 | 70 (48.61) | 74 (51.39) | 0.352 |
|   |   |   | – | 113 (29.05) | 185 (47.56) | 91 (23.39) |   | 411 (52.83) | 367 (47.17) |   |
|   |   | drug resistance | + | 19 (25.68) | 35 (47.30) | 20 (27.03) | 0.716 | 73 (49.32) | 75 (50.68) | 0.450 |
|   |   |   | – | 109 (28.17) | 190 (49.10) | 88 (22.74) |   | 408 (52.71) | 366 (47.29) |   |
|   |   | DILI | + | 20 (29.41) | 36 (52.94) | 12 (17.65) | 0.474 | 76 (55.88) | 60 (44.12) | 0.348 |
|   |   |   | – | 108 (27.48) | 189 (48.09) | 96 (24.43) |   | 405 (51.53) | 381 (48.47) |   |
|   |   | pulmonary infection | + | 18 (21.95) | 43 (52.44) | 21 (25.61) | 0.430 | 79 (48.17) | 85 (51.83) | 0.258 |
|   |   |   | – | 110 (29.02) | 182 (48.02) | 87 (22.96) |   | 402 (53.03) | 356 (46.97) |   |
|   |   | hypoproteinemia | + | 6 (15.00) | 27 (67.50) | 7 (17.50) | 0.042 | 39 (48.75) | 41 (51.25) | 0.522 |
|   |   |   | – | 122 (28.98) | 198 (47.03) | 101 (23.99) |   | 442 (52.49) | 400 (47.51) |   |
|   |   | leukopenia | + | 6 (20.00) | 13 (43.33) | 11 (36.67) | 0.193 | 25 (41.67) | 35 (58.33) | 0.092 |
|   |   |   | – | 122 (28.31) | 212 (49.19) | 97 (22.51) |   | 456 (52.90) | 406 (47.10) |   |
|   |   | sputum smear-positive | + | 34 (26.98) | 65 (51.59) | 27 (21.43) | 0.933 | 133 (52.78) | 119 (47.22) | 0.948 |
|   |   |   | – | 82 (27.70) | 147 (49.66) | 67 (22.64) |   | 311 (52.53) | 281 (47.47) |   |
| WTAP |  |  |  |  |  |  |  |  |  |  |
| rs11752345 | C/T | fever | + | 55 (76.39) | 17 (23.61) | 0 (0) | 0.372 | 127 (88.19) | 17 (11.81) | 0.243 |
|   |   |   | – | 322 (82.78) | 66 (16.97) | 1 (0.26) |   | 710 (91.26) | 68 (8.74) |   |
|   |   | drug resistance | + | 65 (82.28) | 14 (17.72) | 0 (0) | 0.832 | 144 (91.14) | 14 (8.86) | 0.983 |
|   |   |   | – | 380 (82.61) | 78 (16.96) | 2 (0.43) |   | 838 (91.09) | 82 (8.91) |   |
|   |   | DILI | + | 65 (85.53) | 11 (14.47) | 0 (0) | 0.680 | 141 (92.76) | 11 (7.24) | 0.436 |
|   |   |   | – | 380 (82.07) | 81 (17.49) | 2 (0.43) |   | 841 (90.82) | 85 (9.18) |   |
|   |   | pulmonary infection | + | 75 (80.65) | 18 (19.35) | 0 (0) | 0.667 | 168 (90.32) | 18 (9.68) | 0.684 |
|   |   |   | – | 370 (82.96) | 74 (16.59) | 2 (0.45) |   | 814 (91.26) | 78 (8.74) |   |
|   |   | hypoproteinemia | + | 50 (89.29) | 6 (10.71) | 0 (0) | 0.358 | 106 (94.64) | 6 (5.36) | 0.164 |
|   |   |   | – | 395 (81.78) | 86 (17.81) | 2 (0.41) |   | 876 (90.68) | 90 (9.32) |   |
|   |   | leukopenia | + | 28 (77.78) | 8 (22.22) | 0 (0) | 0.654 | 64 (88.89) | 8 (11.11) | 0.496 |
|   |   |   | – | 417 (82.90) | 84 (16.70) | 2 (0.40) |   | 918 (91.25) | 88 (8.75) |   |
|   |   | sputum smear-positive | + | 134 (87.58) | 19 (12.42) | 0 (0) | 0.101 | 287 (93.79) | 19 (6.21) | 0.041 |
|   |   |   | – | 277 (79.83) | 69 (19.88) | 1 (0.29) |   | 623 (89.77) | 71 (10.23) |   |
| rs1853259 | A/G | fever | + | 24 (33.33) | 37 (51.39) | 11 (15.28) | 0.790 | 85 (59.03) | 59 (40.97) | 0.507 |
|   |   |   | – | 145 (37.28) | 192 (49.36) | 52 (13.37) |   | 482 (61.95) | 296 (38.05) |   |
|   |   | drug resistance | + | 26 (35.14) | 38 (51.35) | 10 (13.51) | 0.948 | 90 (60.81) | 58 (39.19) | 0.852 |
|   |   |   | – | 143 (36.95) | 191 (49.35) | 53 (13.70) |   | 477 (61.63) | 297 (38.37) |   |
|   |   | DILI | + | 24 (35.29) | 36 (52.94) | 8 (11.76) | 0.809 | 84 (61.76) | 52 (38.24) | 0.945 |
|   |   |   | – | 145 (36.9) | 193 (49.11) | 55 (13.99) |   | 483 (61.45) | 303 (38.55) |   |
|   |   | pulmonary infection | + | 29 (35.37) | 40 (48.78) | 13 (15.85) | 0.814 | 98 (59.76) | 66 (40.24) | 0.613 |
|   |   |   | – | 140 (36.94) | 189 (49.87) | 50 (13.19) |   | 469 (61.87) | 289 (38.13) |   |
|   |   | hypoproteinemia | + | 15 (37.50) | 21 (52.50) | 4 (10.00) | 0.775 | 51 (63.75) | 29 (36.25) | 0.665 |
|   |   |   | – | 154 (36.58) | 208 (49.41) | 59 (14.01) |   | 516 (61.28) | 326 (38.72) |   |
|   |   | leukopenia | + | 11 (36.67) | 12 (40.00) | 7 (23.33) | 0.247 | 34 (56.67) | 26 (43.33) | 0.427 |
|   |   |   | – | 158 (36.66) | 217 (50.35) | 56 (12.99) |   | 533 (61.83) | 329 (38.17) |   |
|   |   | sputum smear-positive | + | 45 (35.71) | 67 (53.17) | 14 (11.11) | 0.233 | 157 (62.30) | 95 (37.70) | 0.753 |
|   |   |   | – | 114 (38.51) | 134 (45.27) | 48 (16.22) |   | 362 (61.15) | 230 (38.85) |   |

Table 2 Caption: Associations between METTL3, METTL14, and WTAP genes polymorphisms and clinical features of PTB patients.

### Haplotype analysis

The main haplotypes of *METTL3*, *METTL14*, and *WTAP* genes were detected *via* SHEsis software, and the frequency distributions of these haplotypes, including five for *METTL3* (AGC, ATC, GGC, GTA, GTC), four for *METTL14* (ATA, ATG, TCA, TTA), and three for *WTAP* (CA, CG, TA) are shown in [Table 3](#T3). Compared with normal controls, the frequency of the *METTL3* AGC haplotype was significantly lower in PTB patients (*p* = 0.046), and the frequency of the *WTAP* TA haplotype was significantly higher (*p* = 0.035).

### Table 3.

| Haplotype | PTB [n (%)] | Controls [n (%)] | P value | OR (95% CI) |
| --- | --- | --- | --- | --- |
| METTL3 rs1139130-rs1061026-rs1061027 |  |  |  |  |
| AGC | 27.94 (3.0) | 45.14 (4.8) | 0.046 | 0.615 (0.380,0.995) |
| ATC | 573.06 (62.2) | 551.86 (59.1) | 0.177 | 1.137 (0.944,1.370) |
| GGC | 47.91 (5.2) | 41.62 (4.5) | 0.457 | 1.175 (0.768,1.799) |
| GTA | 171.85 (18.6) | 184.76 (19.8) | 0.531 | 0.929 (0.737,1.170) |
| GTC | 101.09 (11.0) | 110.38 (11.8) | 0.562 | 0.919 (0.690,1.224) |
| METTL14 rs1064034-rs4834698-rs62328061 |  |  |  |  |
| ATA | 124.00 (13.4) | 136.67 (14.6) | 0.476 | 0.909 (0.699,1.181) |
| ATG | 150.00 (16.3) | 152.82 (16.4) | 0.947 | 0.992 (0.775,1.182) |
| TCA | 441.00 (47.8) | 442.96 (47.4) | 0.882 | 1.014 (0.845,1.217) |
| TTA | 207.00 (22.5) | 200.86 (21.5) | 0.633 | 1.055 (0.847,1.314) |
| WTAP rs11752345- rs1853259 |  |  |  |  |
| CA | 483.83 (52.5) | 506.05 (54.2) | 0.489 | 0.938 (0.781,1.125) |
| CG | 353.17 (38.3) | 367.95 (39.4) | 0.454 | 0.958 (0.758,1.155) |
| TA | 83.17 (9.0) | 59.95 (6.4) | 0.035 | 1.449 (1.026,2.046) |

Table 3 Caption: Haplotype analysis of METTL3, METTL14, and WTAP genes in PTB patients and controls.

### m6A methyltransferase transcription levels in the PTB group and the control group

The transcription levels of *METTL3*, *METTL14*, and *WTAP* in PBMCs were significantly lower in PTB patients than in normal controls (all *p* < 0.001) **(** [Figure 1](#f1) **)**. Associations between METTL3, METTL14, and WTAP mRNA levels and multiple clinical features were investigated in PTB patients. WTAP transcription level was significantly greater in PTB patients with drug resistance than in PTB patients without drug resistance (*p* = 0.030) ([Table 4](#T4)). *METTL3* transcription level was negatively associated with ESR (*p* = 0.035) and AST (*p* = 0.022) in PTB patients, and *METTL14* transcription level was negatively associated with ALT (*p* = 0.049) and AST (*p* = 0.004) ([Table 5](#T5)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bcee/9768477/39165ea6b974/fimmu-13-992628-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9768477_fimmu-13-992628-g001.jpg)

The transcription levels of METTL3, METTL14 and WTAP in PTB patients and controls.

### Table 4.

| Group | +/- | N | METTL3 level | P value | METTL14 level | P value | WTAP level | P value |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| fever | + | 15 | 0.361 (0.256,0.835) | 0.644 | 0.400 (0.221,0.819) | 0.490 | 0.158 (0.091,0.258) | 0.226 |
|   | – | 63 | 0.491 (0.307,0.682) |   | 0.432 (0.292,0.936) |   | 0.225 (0.125,0.344) |   |
| drug resistance | + | 5 | 0.639 (0.241,1.344) | 0.333 | 0.936 (0.203,2.613) | 0.364 | 0.384 (0.232,0.945) | 0.030 |
|   | – | 73 | 0.454 (0.307,0.674) |   | 0.430 (0.280,0.831) |   | 0.192 (0.102,0.309) |   |
| DILI | + | 8 | 0.371 (0.267,0.717) | 0.521 | 0.648 (0.326,0.902) | 0.489 | 0.237 (0.040,0.385) | 0.895 |
|   | – | 70 | 0.481 (0.307,0.686) |   | 0.428 (0.271,0.865) |   | 0.199 (0.120,0.336) |   |
| pulmonary infection | + | 11 | 0.342 (0.307,0.639) | 0.651 | 0.363 (0.264,0.921) | 0.937 | 0.170 (0.077,0.242) | 0.220 |
|   | – | 67 | 0.491 (0.306,0.700) |   | 0.432 (0.287,0.845) |   | 0.209 (0.125,0.374) |   |
| hypoproteinemia | + | 16 | 0.333 (0.271,0.609) | 0.173 | 0.422 (0.255,1.014) | 0.853 | 0.179 (0.048,0.278) | 0.282 |
|   | – | 62 | 0.495 (0.312,0.747) |   | 0.431 (0.293,0.827) |   | 0.217 (0.048,0.278) |   |
| leukopenia | + | 6 | 0.420 (0.150,0.847) | 0.666 | 0.474 (0.166,1.251) | 0.680 | 0.308 (0.141,0.565) | 0.277 |
|   | – | 72 | 0.462 (0.309,0.678) |   | 0.431 (0.288,0.844) |   | 0.199 (0.103,0.313) |   |
| sputum smear-positive | + | 27 | 0.437 (0.307,0.656) | 0.781 | 0.515 (.264,0.937) | 0.498 | 0.170 (0.097,0.284) | 0.498 |
|   | – | 51 | 0.495 (0.306,0.760) |   | 0.430 (0.287,0.842) |   | 0.209 (0.149,0.369) |   |

Table 4 Caption: Association between METTL3, METTL14, and WTAP transcription levels and several clinical features in PTB patients.

### Table 5.

| Clinical parameters | METTL3 level |  | METTL14 level |  | WTAP level |  |
| --- | --- | --- | --- | --- | --- | --- |
| rs | P value | rs | P value | rs | P value |  |
| ESR | -0.243 | 0.035 | -0.054 | 0.642 | -0.034 | 0.772 |
| TBIL | 0.131 | 0.260 | 0.057 | 0.626 | -0.018 | 0.878 |
| ALT | -0.079 | 0.499 | -0.227 | 0.049 | -0.083 | 0.476 |
| AST | -0.264 | 0.022 | -0.331 | 0.004 | -0.113 | 0.336 |

Table 5 Caption: The correlation between METTL3, METTL14, and WTAP transcription levels and ESR, TBIL, ALT, AST of PTB patients.

### Associations between m6A methyltransferase gene polymorphisms and their transcription levels in PTB patients

The Associations between genes variation in m6A methyltransferase METTL3, METTL14, and WTAP and their transcription levels were assessed in 64 PTB patients, and no significant differences were detected (all *p* > 0.05) ([Table 6](#T6)).

### Table 6.

| METTL3 SNP | Genotype | number | METTL3 level | P value |
| --- | --- | --- | --- | --- |
| rs1139130 | AA | 32 | 0.554 (0.279,0.678) | 0.325 |
|   | AG | 27 | 0.409 (0.306,0.529) |   |
|   | GG | 5 | 0.559 (0.431,0.760) |   |
| rs1061026 | TT | 55 | 0.470 (0.312,0.665) | 0.412 |
|   | TG | 9 | 0.495 (0.165,0.642) |   |
|   | GG | 0 | – |   |
| rs1061027 | CC | 42 | 0.474 (0.307,0.669) | 0.981 |
|   | CA | 20 | 0.490 (0.303,0.684) |   |
|   | AA | 2 | 0.436 (0.314,0.559) |   |
| METTL14 SNP | Genotype | number | METTL14 level | P value |
| rs1064034 | TT | 32 | 0.423 (0.261,1.137) | 0.917 |
|   | AT | 26 | 0.422 (0.249,0.826) |   |
|   | AA | 6 | 0.404 (0.253,1.879) |   |
| rs62328061 | AA | 47 | 0.364 (0.255,0.819) | 0.153 |
|   | AG | 14 | 0.660 (0.359,1.162) |   |
|   | GG | 3 | 0.489 (0.319,1.625) |   |
| rs4834698 | TT | 16 | 0.449 (0.330, 0.811) | 0.690 |
|   | TC | 37 | 0.400 (0.256, 0.831) |   |
|   | CC | 11 | 0.343 (0.111,1.397) |   |
| WTAP SNP | Genotype | number | WTAP level | P value |
| rs11752345 | CC | 52 | 0.207 (0.103,0.329) | 0.618 |
|   | CT | 12 | 0.178 (0.082,0.315) |   |
|   | TT | 0 | – |   |
| rs1853259 | AA | 27 | 0.225 (0.068,0.344) | 0.635 |
|   | AG | 29 | 0.185 (0.099,0.328) |   |
|   | GG | 8 | 0.246 (0.165,0.307) |   |

Table 6 Caption: Association between METTL3, METTL14, and WTAP genes polymorphisms with their transcription levels in PTB patients.

## Discussion

Many novel genetic variants associated with susceptibility to PTB have been identified *via* genome-wide association studies and candidate gene studies, but unearthing the full range of PTB susceptibility variations remains a challenge. Given the important role of m6A modification in the development of multiple diseases, potential associations between genetic variation of m6A-modified genes and disease susceptibility have also attracted increased attention in recent years ([27](#B27), [28](#B28)). The classic m6A methyltransferase complex mainly consists of METTL3, METTL14, and WTAP, which mediate m6A methylation of mRNA. METTL3 is a vital methyltransferase as an S-adenosylmethionine-binding subunit, whereas METTL14 is an RNA-binding scaffold for substrate recognition. WTAP interacts with METTL3 and METTL14, then localizes them into nuclear speckles ([29](#B29), [30](#B30)). Notably the SNPs located in these m6A methyltransferases are reportedly associated with disease susceptibility *via* m6A methylation and related biological processes ([19](#B19), [20](#B20), [31](#B31)). To the best of our knowledge, the current study is the first to investigate associations between polymorphic variants of the m6A methyltransferase genes *METTL3*, *METTL14*, and *WTAP* and PTB susceptibility. The study provides strong evidence that the *METTL14* gene variant rs62328061 and the *WTAP* gene variant rs11752345 are associated with the risk of PTB, and that METTL3, METTL14, and WTAP levels are decreased in PTB patients.

To date, studies on genetic variation of *METTL3* and disease susceptibility have mainly focused on cancers. Lin et al. ([32](#B32)) suggested that the combination of rs1139130, rs1263801, rs1061026, and rs1061027 variants in the *METTL3* gene could reduce the risk of Wilms’ tumor in children. Another study reported that these four SNPs were also related to higher susceptibility to neuroblastoma ([19](#B19)). In contrast with these findings, rs1061027, rs1139130, and rs1061026 variants were not significantly associated with PTB susceptibility in the present study. Compared with single SNPs, association studies based on haplotypes of multiple markers can significantly enhance the mapping and characterization of disease susceptibility genes ([33](#B33)). In this study, we assessed whether the main haplotypes consisting of the aforementioned three polymorphisms were associated with PTB, and the AGC haplotype was significantly associated with a lower risk of PTB. These results suggest that there may be weak associations between *METTL3* gene variation of and PTB susceptibility, and these variants may interact with each other to modify the PTB risk. This should be confirmed in future studies with larger sample sizes. In addition, rs1139130 polymorphism was also significantly associated with drug resistance and DILI in PTB patients. This further proved that *METTL3* gene variation was involved in the pathogenesis of PTB, and indicated that this SNP could be used to predict adverse reactions in patients, thus contributing to the formulation of more appropriate treatment measures.

In previous studies, *METTL14* gene rs62328061 polymorphism was significantly associated with reduced susceptibility to neuroblastoma ([34](#B34)), and rs1064034, rs298982 polymorphisms were significantly associated with reduced susceptibility to Wilms’ tumor ([21](#B21)). In the current study, the rs62328061 GG genotype was associated with an increased risk of PTB, but there was no significant association between the rs1064034 variant and PTB susceptibility. These results provided key indications of an association between *METTL14* gene variation and PTB susceptibility. In PTB patients, a decreased frequency of the rs62328061 G allele was associated with leukopenia, and an increased frequency of the rs4834698 TC genotype was associated with hypoproteinemia. This confirmed the important role of *METTL14* gene variation in the development of PTB, and further functional studies are needed to investigate the molecular mechanisms involved in the effects of rs62328061 on PTB risk. The present study also investigated associations between *WTAP* gene polymorphisms and the risk of PTB, and the *WTAP* rs11752345 CT genotype and T allele frequencies were significantly increased in PTB patients. Moreover, the rs11752345 T allele was closely related to sputum smear in PTB patients. These results suggested that the CT genotype/T allele of rs11752345 may increase the risk of PTB, and the T allele may be useful for distinguishing different phenotypes in PTB patients.

Increasing studies indicate that transcription levels of m6A methyltransferases are closely related to the pathogenesis and progression of many diseases. Dysregulation of METTL3 is considered an important factor affecting the progression of various malignant tumors such as endometrial cancer ([35](#B35)) and bladder cancer ([36](#B36)). Another study detected lower METTL14 expression in colorectal cancer ([37](#B37)). Functional m6A methyltransferase SNPs may affect m6A methyltransferase expression and thus modify disease susceptibility. Hence, we designed a case-control study to determine METTL3, METTL14, and WTAP transcription levels in PTB patients to investigate the possible mechanism of SNP-mediated PTB susceptibility. Transcription levels of *METTL3*, *METTL14*, and *WTAP* were significantly decreased in PTB patients compared to controls. This suggested that METTL3, METTL14, and WTAP may be involved in PTB occurrence, and their levels may be used as auxiliary indicators for PTB diagnosis. This must be verified in the future by well-designed studies. WTAP level was also significantly associated with drug resistance, ALT, and AST, and METTL3 level was negatively associated with ESR and AST in PTB patients. These findings contribute to improving our understanding of the role of m6A methyltransferases in PTB development. Lastly, we analyzed potential associations between m6A methyltransferase gene variants and their transcription levels in PTB patients. No significant associations were detected, possibly due to the small sample size.

The major strengths of this study include its novelty and rational design, but the study also had several limitations. It was focused on the analysis of genetic factors in PTB, but it did not assess the effects of environmental factors on PTB, or interactions between environmental factors and genetic variation. Second, associations between gene variation and the prognosis of PTB were not analyzed. Lastly, the study lacked a functional investigation into the effects of m6A methyltransferase on PTB pathogenesis.

In summary, the current study indicates that polymorphisms of *METTL14* rs62328061 and *WTAP* rs11752345 may contribute to increased PTB susceptibility, and several SNPs in *METTL3*, *METTL14*, and *WTAP* genes were associated with the clinical features including drug resistance, DILI, and leukopenia in PTB patients. The study demonstrated the important roles of alterations in METTL3, METTL14, and WTAP in PTB, and these m6A methyltransferases in the development of PTB. In order to further validate these findings and determine the underlying biological mechanisms involving these m6A methyltransferases in PTB, more studies incorporating larger sample sizes and different ethnic populations are warranted.

## Data availability statement

The data presented in the study are deposited in the dbSNP (1063446) . Further inquiries can be directed to the corresponding authors.

## Ethics statement

This study was approved by the Ethical Committee of Anhui Medical University. The patients/participants provided their written informed consent to participate in this study.

## Author contributions

T-PZ and H-ML designed the study. H-ML conducted the experiment. RL performed the statistical analyses. L-JW and QH participated in the collection of samples. H-ML and T-PZ drafted the manuscript. All author contributed to manuscript revision. All the authors approved the final submitted version.

## Funding

This work was supported by grants from the National Natural Science Foundation of China (82003515).

## Conflict of interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Publisher’s note

All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

The data presented in the study are deposited in the dbSNP (1063446) . Further inquiries can be directed to the corresponding authors.

### Data Availability Statement

The data presented in the study are deposited in the dbSNP (1063446) . Further inquiries can be directed to the corresponding authors.

## References

1. Kaufmann SHE. New vaccines against tuberculosis. Bundesgesundheitsbla (2020) 63:56–64. doi: 10.1007/s00103-019-03065-y  [DOI](https://doi.org/10.1007/s00103-019-03065-y) | [PMC free article](/articles/PMC7222064/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31784764/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bundesgesundheitsbla&title=New%20vaccines%20against%20tuberculosis&author=SHE%20Kaufmann&volume=63&publication_year=2020&pages=56-64&pmid=31784764&doi=10.1007/s00103-019-03065-y&)

2. World Health Organization . Global tuberculosis report (2021). Available at: https://www.who.int/tb/publications/global_report/en/.  [https://www.who.int/tb/publications/global_report/en/](https://www.who.int/tb/publications/global_report/en/)

3. Ji G, Zhang M, Liu Q, Wu S, Wang Y, Chen G, et al. Functional polymorphism in the NFE2L2 gene associated with tuberculosis susceptibility. Front Immunol (2021) 12:660384. doi: 10.3389/fimmu.2021.660384  [DOI](https://doi.org/10.3389/fimmu.2021.660384) | [PMC free article](/articles/PMC8181729/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34108963/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&title=Functional%20polymorphism%20in%20the%20NFE2L2%20gene%20associated%20with%20tuberculosis%20susceptibility&author=G%20Ji&author=M%20Zhang&author=Q%20Liu&author=S%20Wu&author=Y%20Wang&volume=12&publication_year=2021&pages=660384&pmid=34108963&doi=10.3389/fimmu.2021.660384&)

4. Lawn SD, Zumla AI. Tuberculosis. Lancet (2011) 378:57–72. doi: 10.1016/S0140-6736(10)62173-3  [DOI](https://doi.org/10.1016/S0140-6736(10)62173-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21420161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Tuberculosis&author=SD%20Lawn&author=AI%20Zumla&volume=378&publication_year=2011&pages=57-72&pmid=21420161&doi=10.1016/S0140-6736(10)62173-3&)

5. Caws M, Thwaites G, Dunstan S, Hawn TR, Lan NTN, Thuong NTT, et al. The influence of host and bacterial genotype on the development of disseminated disease with mycobacterium tuberculosis. PloS Pathog (2008) 4:e1000034. doi: 10.1371/journal.ppat.1000034  [DOI](https://doi.org/10.1371/journal.ppat.1000034) | [PMC free article](/articles/PMC2268004/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18369480/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20Pathog&title=The%20influence%20of%20host%20and%20bacterial%20genotype%20on%20the%20development%20of%20disseminated%20disease%20with%20mycobacterium%20tuberculosis&author=M%20Caws&author=G%20Thwaites&author=S%20Dunstan&author=TR%20Hawn&author=NTN%20Lan&volume=4&publication_year=2008&pages=e1000034&pmid=18369480&doi=10.1371/journal.ppat.1000034&)

6. Gagneux S. Host-pathogen coevolution in human tuberculosis. Philos Trans R Soc Lond B Biol Sci (2012) 367:850–9. doi: 10.1098/rstb.2011.0316  [DOI](https://doi.org/10.1098/rstb.2011.0316) | [PMC free article](/articles/PMC3267123/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22312052/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Philos%20Trans%20R%20Soc%20Lond%20B%20Biol%20Sci&title=Host-pathogen%20coevolution%20in%20human%20tuberculosis&author=S%20Gagneux&volume=367&publication_year=2012&pmid=22312052&doi=10.1098/rstb.2011.0316&)

7. Orlova M, Schurr E. Human genomics of mycobacterium tuberculosis infection and disease. Curr Genet Med Rep (2017) 5:125–31. doi: 10.1007/s40142-017-0124-7  [DOI](https://doi.org/10.1007/s40142-017-0124-7) | [PMC free article](/articles/PMC5703416/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29201558/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Genet%20Med%20Rep&title=Human%20genomics%20of%20mycobacterium%20tuberculosis%20infection%20and%20disease&author=M%20Orlova&author=E%20Schurr&volume=5&publication_year=2017&pmid=29201558&doi=10.1007/s40142-017-0124-7&)

8. Zhang TP, Chen SS, Zhang GY, Shi SJ, Wei L, Li HM. Association of vitamin d pathway genes polymorphisms with pulmonary tuberculosis susceptibility in a Chinese population. Gnes Nutr (2021) 16:6. doi: 10.1186/s12263-021-00687-3  [DOI](https://doi.org/10.1186/s12263-021-00687-3) | [PMC free article](/articles/PMC8061222/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33882819/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gnes%20Nutr&title=Association%20of%20vitamin%20d%20pathway%20genes%20polymorphisms%20with%20pulmonary%20tuberculosis%20susceptibility%20in%20a%20Chinese%20population&author=TP%20Zhang&author=SS%20Chen&author=GY%20Zhang&author=SJ%20Shi&author=L%20Wei&volume=16&publication_year=2021&pages=6&pmid=33882819&doi=10.1186/s12263-021-00687-3&)

9. Harishankar M, Selvaraj P, Bethunaickan R. Influence of genetic polymorphism towards pulmonary tuberculosis susceptibility. Front Med (Lausanne) (2018) 5:213. doi: 10.3389/fmed.2018.00213  [DOI](https://doi.org/10.3389/fmed.2018.00213) | [PMC free article](/articles/PMC6106802/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30167433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Med%20(Lausanne)&title=Influence%20of%20genetic%20polymorphism%20towards%20pulmonary%20tuberculosis%20susceptibility&author=M%20Harishankar&author=P%20Selvaraj&author=R%20Bethunaickan&volume=5&publication_year=2018&pages=213&pmid=30167433&doi=10.3389/fmed.2018.00213&)

10. Ortega E, Hernández-Bazán S, Sánchez-Hernández B, Licona-Limón I, Fuentes-Dominguez J. TLR4 single nucleotide polymorphisms in affect susceptibility to tuberculosis in Mexican population from the state of veracruz. J Immunol Res (2020) 2020:2965697. doi: 10.1155/2020/2965697  [DOI](https://doi.org/10.1155/2020/2965697) | [PMC free article](/articles/PMC7204096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32411792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Immunol%20Res&title=TLR4%20single%20nucleotide%20polymorphisms%20in%20affect%20susceptibility%20to%20tuberculosis%20in%20Mexican%20population%20from%20the%20state%20of%20veracruz&author=E%20Ortega&author=S%20Hern%C3%A1ndez-Baz%C3%A1n&author=B%20S%C3%A1nchez-Hern%C3%A1ndez&author=I%20Licona-Lim%C3%B3n&author=J%20Fuentes-Dominguez&volume=2020&publication_year=2020&pages=2965697&pmid=32411792&doi=10.1155/2020/2965697&)

11. Cao G, Li HB, Yin Z, Flavell RA. Recent advances in dynamic m6A RNA modification. Open Biol (2016) 6:160003. doi: 10.1098/rsob.160003  [DOI](https://doi.org/10.1098/rsob.160003) | [PMC free article](/articles/PMC4852458/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27249342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Open%20Biol&title=Recent%20advances%20in%20dynamic%20m6A%20RNA%20modification&author=G%20Cao&author=HB%20Li&author=Z%20Yin&author=RA%20Flavell&volume=6&publication_year=2016&pages=160003&pmid=27249342&doi=10.1098/rsob.160003&)

12. Deng X, Su R, Weng H, Huang H, Li Z, Chen J. RNA N(6)-methyladenosine modification in cancers: Current status and perspectives. Cell Res (2018) 28:507–17. doi: 10.1038/s41422-018-0034-6  [DOI](https://doi.org/10.1038/s41422-018-0034-6) | [PMC free article](/articles/PMC5951805/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29686311/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=RNA%20N(6)-methyladenosine%20modification%20in%20cancers:%20Current%20status%20and%20perspectives&author=X%20Deng&author=R%20Su&author=H%20Weng&author=H%20Huang&author=Z%20Li&volume=28&publication_year=2018&pmid=29686311&doi=10.1038/s41422-018-0034-6&)

13. He L, Li H, Wu A, Peng Y, Shu G, Yin G. Functions of N6-methyladenosine and its role in cancer. Mol Cancer (2019) 18:176. doi: 10.1186/s12943-019-1109-9  [DOI](https://doi.org/10.1186/s12943-019-1109-9) | [PMC free article](/articles/PMC6892141/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31801551/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer&title=Functions%20of%20N6-methyladenosine%20and%20its%20role%20in%20cancer&author=L%20He&author=H%20Li&author=A%20Wu&author=Y%20Peng&author=G%20Shu&volume=18&publication_year=2019&pages=176&pmid=31801551&doi=10.1186/s12943-019-1109-9&)

14. Wang M, Kong W, He B, Li Z, Song H, Shi P, et al. Vitamin d and the promoter methylation of its metabolic pathway genes in association with the risk and prognosis of tuberculosis. Clin Epigenet (2018) 10:118. doi: 10.1186/s13148-018-0552-6  [DOI](https://doi.org/10.1186/s13148-018-0552-6) | [PMC free article](/articles/PMC6136159/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30208925/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Epigenet&title=Vitamin%20d%20and%20the%20promoter%20methylation%20of%20its%20metabolic%20pathway%20genes%20in%20association%20with%20the%20risk%20and%20prognosis%20of%20tuberculosis&author=M%20Wang&author=W%20Kong&author=B%20He&author=Z%20Li&author=H%20Song&volume=10&publication_year=2018&pages=118&pmid=30208925&doi=10.1186/s13148-018-0552-6&)

15. Shell SS, Prestwich EG, Baek SH, Shah RR, Sassetti CM, Dedon PC, et al. DNA Methylation impacts gene expression and ensures hypoxic survival of mycobacterium tuberculosis. PloS Pathog (2013) 9:e1003419. doi: 10.1371/journal.ppat.1003419  [DOI](https://doi.org/10.1371/journal.ppat.1003419) | [PMC free article](/articles/PMC3701705/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23853579/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20Pathog&title=DNA%20Methylation%20impacts%20gene%20expression%20and%20ensures%20hypoxic%20survival%20of%20mycobacterium%20tuberculosis&author=SS%20Shell&author=EG%20Prestwich&author=SH%20Baek&author=RR%20Shah&author=CM%20Sassetti&volume=9&publication_year=2013&pages=e1003419&pmid=23853579&doi=10.1371/journal.ppat.1003419&)

16. Gong Z, Wang G, Zeng J, Stojkoska A, Huang H, Xie J. Differential DNA methylomes of clinical MDR, XDR and XXDR mycobacterium tuberculosis isolates revealed by using single-molecule real-time sequencing. J Drug Target (2021) 29:69–77. doi: 10.1080/1061186X.2020.1797049  [DOI](https://doi.org/10.1080/1061186X.2020.1797049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32672115/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Drug%20Target&title=Differential%20DNA%20methylomes%20of%20clinical%20MDR,%20XDR%20and%20XXDR%20mycobacterium%20tuberculosis%20isolates%20revealed%20by%20using%20single-molecule%20real-time%20sequencing&author=Z%20Gong&author=G%20Wang&author=J%20Zeng&author=A%20Stojkoska&author=H%20Huang&volume=29&publication_year=2021&pages=69-77&pmid=32672115&doi=10.1080/1061186X.2020.1797049&)

17. Li T, Hu PS, Zuo Z, Lin JF, Li X, Wu QN, et al. METTL3 facilitates tumor progression via an m6A-IGF2BP2-dependent mechanism in colorectal carcinoma. Mol Cancer (2019) 18:112. doi: 10.1186/s12943-019-1038-7  [DOI](https://doi.org/10.1186/s12943-019-1038-7) | [PMC free article](/articles/PMC6589893/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31230592/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer&title=METTL3%20facilitates%20tumor%20progression%20via%20an%20m6A-IGF2BP2-dependent%20mechanism%20in%20colorectal%20carcinoma&author=T%20Li&author=PS%20Hu&author=Z%20Zuo&author=JF%20Lin&author=X%20Li&volume=18&publication_year=2019&pages=112&pmid=31230592&doi=10.1186/s12943-019-1038-7&)

18. Orouji E, Peitsch WK, Orouji A, Houben R, Utikal J. Oncogenic role of an epigenetic reader of m6A RNA modification: YTHDF1 in merkel cell carcinoma. Cancers (Basel) (2020) 12:202. doi: 10.3390/cancers12010202  [DOI](https://doi.org/10.3390/cancers12010202) | [PMC free article](/articles/PMC7016651/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31947544/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&title=Oncogenic%20role%20of%20an%20epigenetic%20reader%20of%20m6A%20RNA%20modification:%20YTHDF1%20in%20merkel%20cell%20carcinoma&author=E%20Orouji&author=WK%20Peitsch&author=A%20Orouji&author=R%20Houben&author=J%20Utikal&volume=12&publication_year=2020&pages=202&pmid=31947544&doi=10.3390/cancers12010202&)

19. Bian J, Zhuo Z, Zhu J, Yang Z, Jiao Z, Li Y, et al. Association between METTL3 gene polymorphisms and neuroblastoma susceptibility: A nine-centre case-control study. J Cell Mol Med (2020) 24:9280–6. doi: 10.1111/jcmm.15576  [DOI](https://doi.org/10.1111/jcmm.15576) | [PMC free article](/articles/PMC7417682/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32615646/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Mol%20Med&title=Association%20between%20METTL3%20gene%20polymorphisms%20and%20neuroblastoma%20susceptibility:%20A%20nine-centre%20case-control%20study&author=J%20Bian&author=Z%20Zhuo&author=J%20Zhu&author=Z%20Yang&author=Z%20Jiao&volume=24&publication_year=2020&pmid=32615646&doi=10.1111/jcmm.15576&)

20. Song RH, Liu XR, Gao CQ, Du P, Zhang JA. METTL3 gene polymorphisms contribute to susceptibility to autoimmune thyroid disease. Endocrine (2021) 72:495–504. doi: 10.1007/s12020-020-02503-1  [DOI](https://doi.org/10.1007/s12020-020-02503-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33025559/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Endocrine&title=METTL3%20gene%20polymorphisms%20contribute%20to%20susceptibility%20to%20autoimmune%20thyroid%20disease&author=RH%20Song&author=XR%20Liu&author=CQ%20Gao&author=P%20Du&author=JA%20Zhang&volume=72&publication_year=2021&pages=495-504&pmid=33025559&doi=10.1007/s12020-020-02503-1&)

21. Zhuo Z, Hua RX, Zhang H, Lin H, Fu W, Zhu J, et al. METTL14 gene polymorphisms decrease wilms tumor susceptibility in Chinese children. BMC Cancer (2021) 21(1):1294. doi: 10.1186/s12885-021-09019-5  [DOI](https://doi.org/10.1186/s12885-021-09019-5) | [PMC free article](/articles/PMC8643011/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34863142/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=METTL14%20gene%20polymorphisms%20decrease%20wilms%20tumor%20susceptibility%20in%20Chinese%20children&author=Z%20Zhuo&author=RX%20Hua&author=H%20Zhang&author=H%20Lin&author=W%20Fu&volume=21&issue=1&publication_year=2021&pages=1294&pmid=34863142&doi=10.1186/s12885-021-09019-5&)

22. Luo A, Yang L, Li M, Cai M, Huang A, Liu X, et al. Genetic variants in METTL14 are associated with the risk of acute lymphoblastic leukemia in southern Chinese children: A five-center case-control study. Cancer Manag Res (2021) 13:9189–200. doi: 10.2147/CMAR.S335925  [DOI](https://doi.org/10.2147/CMAR.S335925) | [PMC free article](/articles/PMC8684373/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34934362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Manag%20Res&title=Genetic%20variants%20in%20METTL14%20are%20associated%20with%20the%20risk%20of%20acute%20lymphoblastic%20leukemia%20in%20southern%20Chinese%20children:%20A%20five-center%20case-control%20study&author=A%20Luo&author=L%20Yang&author=M%20Li&author=M%20Cai&author=A%20Huang&volume=13&publication_year=2021&pmid=34934362&doi=10.2147/CMAR.S335925&)

23. Feng Y, Wang F, Pan H, Qiu S, Lü J, Wu L, et al. Obesity-associated gene FTO rs9939609 polymorphism in relation to the risk of tuberculosis. BMC Infect Dis (2014) 14:592. doi: 10.1186/s12879-014-0592-2  [DOI](https://doi.org/10.1186/s12879-014-0592-2) | [PMC free article](/articles/PMC4226896/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25377722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Infect%20Dis&title=Obesity-associated%20gene%20FTO%20rs9939609%20polymorphism%20in%20relation%20to%20the%20risk%20of%20tuberculosis&author=Y%20Feng&author=F%20Wang&author=H%20Pan&author=S%20Qiu&author=J%20L%C3%BC&volume=14&publication_year=2014&pages=592&pmid=25377722&doi=10.1186/s12879-014-0592-2&)

24. Li HM, Wang LJ, Tang F, Pan HF, Zhang TP. Association of leptin and leptin receptor genes variants and pulmonary tuberculosis susceptibility, clinical manifestations in a Chinese population. Microb Pathog (2022) 165:105499. doi: 10.1016/j.micpath.2022.105499  [DOI](https://doi.org/10.1016/j.micpath.2022.105499) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35325792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Microb%20Pathog&title=Association%20of%20leptin%20and%20leptin%20receptor%20genes%20variants%20and%20pulmonary%20tuberculosis%20susceptibility,%20clinical%20manifestations%20in%20a%20Chinese%20population&author=HM%20Li&author=LJ%20Wang&author=F%20Tang&author=HF%20Pan&author=TP%20Zhang&volume=165&publication_year=2022&pages=105499&pmid=35325792&doi=10.1016/j.micpath.2022.105499&)

25. Schmittgen TD, Livak KJ. Analyzing real-time PCR data by the comparative C(T) method. Nat Protoc (2008) 3:1101–8.27. doi: 10.1038/nprot.2008.73  [DOI](https://doi.org/10.1038/nprot.2008.73) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18546601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Protoc&title=Analyzing%20real-time%20PCR%20data%20by%20the%20comparative%20C(T)%20method&author=TD%20Schmittgen&author=KJ%20Livak&volume=3&publication_year=2008&pages=1101-8.27&pmid=18546601&doi=10.1038/nprot.2008.73&)

26. Li Z, Zhang Z, He Z, Tang W, Li T, Zeng Z, et al. A partition-ligation-combination-subdivision EM algorithm for haplotype inference with multiallelic markers: update of the SHEsis. Cell Res (2009) 19:519–23. doi: 10.1038/cr.2009.33  [DOI](https://doi.org/10.1038/cr.2009.33) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19290020/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=A%20partition-ligation-combination-subdivision%20EM%20algorithm%20for%20haplotype%20inference%20with%20multiallelic%20markers:%20update%20of%20the%20SHEsis&author=Z%20Li&author=Z%20Zhang&author=Z%20He&author=W%20Tang&author=T%20Li&volume=19&publication_year=2009&pmid=19290020&doi=10.1038/cr.2009.33&)

27. Ruan X, Tian M, Kang N, Ma W, Zeng Y, Zhuang G, et al. Genome-wide identification of m6A-associated functional SNPs as potential functional variants for thyroid cancer. Am J Cancer Res (2021) 11:5402–14.  [PMC free article](/articles/PMC8640822/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34873468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Cancer%20Res&title=Genome-wide%20identification%20of%20m6A-associated%20functional%20SNPs%20as%20potential%20functional%20variants%20for%20thyroid%20cancer&author=X%20Ruan&author=M%20Tian&author=N%20Kang&author=W%20Ma&author=Y%20Zeng&volume=11&publication_year=2021&pmid=34873468&)

28. Chen M, Lin W, Yi J, Zhao Z. Exploring the epigenetic regulatory role of m6A-associated SNPs in type 2 diabetes pathogenesis. Pharmgenom Pers Med (2021) 14:1369–78. doi: 10.2147/PGPM.S334346  [DOI](https://doi.org/10.2147/PGPM.S334346) | [PMC free article](/articles/PMC8558037/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34737607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenom%20Pers%20Med&title=Exploring%20the%20epigenetic%20regulatory%20role%20of%20m6A-associated%20SNPs%20in%20type%202%20diabetes%20pathogenesis&author=M%20Chen&author=W%20Lin&author=J%20Yi&author=Z%20Zhao&volume=14&publication_year=2021&pmid=34737607&doi=10.2147/PGPM.S334346&)

29. Meyer KD, Jaffrey SR. Rethinking m6A readers, writers, and erasers. Annu Rev Cell Dev Biol (2017) 33:319–42. doi: 10.1146/annurev-cellbio-100616-060758  [DOI](https://doi.org/10.1146/annurev-cellbio-100616-060758) | [PMC free article](/articles/PMC5963928/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28759256/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Cell%20Dev%20Biol&title=Rethinking%20m6A%20readers,%20writers,%20and%20erasers&author=KD%20Meyer&author=SR%20Jaffrey&volume=33&publication_year=2017&pmid=28759256&doi=10.1146/annurev-cellbio-100616-060758&)

30. Ping XL, Sun BF, Wang L, Xiao W, Yang X, Wang WJ, et al. Mammalian WTAP is a regulatory subunit of the RNA N6-methyladenosine methyltransferase. Cell Res (2014) 24:177–89. doi: 10.1038/cr.2014.3  [DOI](https://doi.org/10.1038/cr.2014.3) | [PMC free article](/articles/PMC3915904/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24407421/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Res&title=Mammalian%20WTAP%20is%20a%20regulatory%20subunit%20of%20the%20RNA%20N6-methyladenosine%20methyltransferase&author=XL%20Ping&author=BF%20Sun&author=L%20Wang&author=W%20Xiao&author=X%20Yang&volume=24&publication_year=2014&pmid=24407421&doi=10.1038/cr.2014.3&)

31. Zhuo ZJ, Hua RX, Chen Z, Zhu J, Wang M, Yang Z, et al. WTAP gene variants confer hepatoblastoma susceptibility: A seven-center case-control study. Mol Ther Oncol (2020) 18:118–25. doi: 10.1016/j.omto.2020.06.007  [DOI](https://doi.org/10.1016/j.omto.2020.06.007) | [PMC free article](/articles/PMC7338985/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32671187/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Ther%20Oncol&title=WTAP%20gene%20variants%20confer%20hepatoblastoma%20susceptibility:%20A%20seven-center%20case-control%20study&author=ZJ%20Zhuo&author=RX%20Hua&author=Z%20Chen&author=J%20Zhu&author=M%20Wang&volume=18&publication_year=2020&pmid=32671187&doi=10.1016/j.omto.2020.06.007&)

32. Lin A, Zhou M, Hua RX, Zhang J, Zhou H, Li S, et al. METTL3 polymorphisms and wilms tumor susceptibility in Chinese children: A five-center case control study. J Gene Med (2020) 22:e3255. doi: 10.1002/jgm.3255  [DOI](https://doi.org/10.1002/jgm.3255) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32716082/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Gene%20Med&title=METTL3%20polymorphisms%20and%20wilms%20tumor%20susceptibility%20in%20Chinese%20children:%20A%20five-center%20case%20control%20study&author=A%20Lin&author=M%20Zhou&author=RX%20Hua&author=J%20Zhang&author=H%20Zhou&volume=22&publication_year=2020&pages=e3255&pmid=32716082&doi=10.1002/jgm.3255&)

33. Manolio TA, Brooks LD, Collins FS. A HapMap harvest of insights into the genetics of common disease. J Clin Invest (2008) 118:1590–605. doi: 10.1172/JCI34772  [DOI](https://doi.org/10.1172/JCI34772) | [PMC free article](/articles/PMC2336881/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18451988/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=A%20HapMap%20harvest%20of%20insights%20into%20the%20genetics%20of%20common%20disease&author=TA%20Manolio&author=LD%20Brooks&author=FS%20Collins&volume=118&publication_year=2008&pmid=18451988&doi=10.1172/JCI34772&)

34. Zhuo Z, Lu H, Zhu J, Hua RX, Li Y, Yang Z, et al. METTL14 gene polymorphisms confer neuroblastoma susceptibility: An eight-center case-control study. Mol Ther Nucleic Acids (2020) 22:17–26. doi: 10.1016/j.omtn.2020.08.009  [DOI](https://doi.org/10.1016/j.omtn.2020.08.009) | [PMC free article](/articles/PMC7484523/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32891980/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Ther%20Nucleic%20Acids&title=METTL14%20gene%20polymorphisms%20confer%20neuroblastoma%20susceptibility:%20An%20eight-center%20case-control%20study&author=Z%20Zhuo&author=H%20Lu&author=J%20Zhu&author=RX%20Hua&author=Y%20Li&volume=22&publication_year=2020&pages=17-26&pmid=32891980&doi=10.1016/j.omtn.2020.08.009&)

35. Liu J, Eckert MA, Harada BT, Liu SM, Lu Z, Yu K, et al. M(6)A mRNA methylation regulates AKT activity to promote the proliferation and tumorigenicity of endometrial cancer. Nat Cell Biol (2018) 20:1074–83. doi: 10.1038/s41556-018-0174-4  [DOI](https://doi.org/10.1038/s41556-018-0174-4) | [PMC free article](/articles/PMC6245953/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30154548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Cell%20Biol&title=M(6)A%20mRNA%20methylation%20regulates%20AKT%20activity%20to%20promote%20the%20proliferation%20and%20tumorigenicity%20of%20endometrial%20cancer&author=J%20Liu&author=MA%20Eckert&author=BT%20Harada&author=SM%20Liu&author=Z%20Lu&volume=20&publication_year=2018&pmid=30154548&doi=10.1038/s41556-018-0174-4&)

36. Jin H, Ying X, Que B, Wang X, Chao Y, Zhang H, et al. N6-methyladenosine modification of ITGA6 mRNA promotes the development and progression of bladder cancer. EBioMedicine (2019) 47:195–207. doi: 10.1016/j.ebiom.2019.07.068  [DOI](https://doi.org/10.1016/j.ebiom.2019.07.068) | [PMC free article](/articles/PMC6796523/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31409574/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=EBioMedicine&title=N6-methyladenosine%20modification%20of%20ITGA6%20mRNA%20promotes%20the%20development%20and%20progression%20of%20bladder%20cancer&author=H%20Jin&author=X%20Ying&author=B%20Que&author=X%20Wang&author=Y%20Chao&volume=47&publication_year=2019&pages=195-207&pmid=31409574&doi=10.1016/j.ebiom.2019.07.068&)

37. Chen X, Xu M, Xu X, Zeng K, Liu X, Sun L, et al. METTL14 suppresses CRC progression via regulating N6-methyladenosine-dependent primary miR-375 processing. Mol Ther (2020) 28:599–612. doi: 10.1016/j.ymthe.2019.11.016  [DOI](https://doi.org/10.1016/j.ymthe.2019.11.016) | [PMC free article](/articles/PMC7001002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31839484/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Ther&title=METTL14%20suppresses%20CRC%20progression%20via%20regulating%20N6-methyladenosine-dependent%20primary%20miR-375%20processing&author=X%20Chen&author=M%20Xu&author=X%20Xu&author=K%20Zeng&author=X%20Liu&volume=28&publication_year=2020&pages=599-612&pmid=31839484&doi=10.1016/j.ymthe.2019.11.016&) | [Retracted](/articles/PMC9263224/)
