# Influence of cytochrome P450 and glutathione S transferase polymorphisms on response to nilotinib therapy among chronic myeloidleukemia patients from Pakistan

## Metadata
**Authors:** Samina Naz Mukry, Aneeta Shahni, Uzma Zaidi, Tahir Sultan Shamsi
**Journal:** BMC Cancer
**Date:** 2022 May 8
**DOI:** [10.1186/s12885-022-09605-1](https://doi.org/10.1186/s12885-022-09605-1)
**PMID:** 35527244
**PMCID:** PMC9080200
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9080200/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9080200/pdf/12885_2022_Article_9605.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9080200/pdf/12885_2022_Article_9605.pdf)

## Abstract

**Background:** 
Cytochrome P450 (CYP) and glutathione S transferases (GSTs) are important biotransforming enzymes responsible for detoxification of anticancer drugs and carcinogens. Polymorphisms in these enzymes may greatly influence the susceptibility to CML and overall efficacy of tyrosine kinase inhibitors. This study was aimed to estimate the possible influence of the polymorphisms of GSTs and CYP in the occurrence of CML as well as in predicting therapeutic outcome of nilotinib therapy in Pakistani CML patients.

**Methods:** 
The polymorphic variability in CYP 1A1*2C, GSTP1 (A3131G), GSTT1 and GSTM1 was assessed either by RFLP or multiplex PCR. The BCR ABL1 transcripts were quantified by qPCR to monitor response to nilotinib.

**Results:** 
The CYP1A1*2C heterozygous and GSTP1 homozygous polymorphisms seemed to be a contributing factor in developing CML. Altogether, there were 12 non-responders, 66 responders and 21 partial responders. The most frequent genotype was null GSTM1 in responders followed by CYP 1A1 and GSTP1 -wild type (p = < 0.05). Whereas, homozygous GSTP1 and GSTT1 null genotype is significantly higher only among nilotinib non-responders.

**Conclusion:** 
Hence, it can be concluded that wild type CYP1A1, GSTP1 and null GSTM1 may be frequently linked to favorable outcome in patients treated with nilotinib as depicted by sustained deep molecular response in most CML patients.

**Supplementary Information:** 
The online version contains supplementary material available at 10.1186/s12885-022-09605-1.

Keywords: Chronic myeloid leukemia, Nilotinib, Treatment, Drug metabolizing enzymes, Response

### Background

Cytochrome P450 (CYP) and glutathione S transferases (GSTs) are important biotransforming enzymes responsible for detoxification of anticancer drugs and carcinogens. Polymorphisms in these enzymes may greatly influence the susceptibility to CML and overall efficacy of tyrosine kinase inhibitors. This study was aimed to estimate the possible influence of the polymorphisms of GSTs and CYP in the occurrence of CML as well as in predicting therapeutic outcome of nilotinib therapy in Pakistani CML patients.

### Methods

The polymorphic variability in CYP 1A1*2C, GSTP1 (A3131G), GSTT1 and GSTM1 was assessed either by RFLP or multiplex PCR. The BCR ABL1 transcripts were quantified by qPCR to monitor response to nilotinib.

### Results

The CYP1A1*2C heterozygous and GSTP1 homozygous polymorphisms seemed to be a contributing factor in developing CML. Altogether, there were 12 non-responders, 66 responders and 21 partial responders. The most frequent genotype was null GSTM1 in responders followed by CYP 1A1 and GSTP1 -wild type (*p* = < 0.05). Whereas, homozygous GSTP1 and GSTT1 null genotype is significantly higher only among nilotinib non-responders.

### Conclusion

Hence, it can be concluded that wild type CYP1A1, GSTP1 and null GSTM1 may be frequently linked to favorable outcome in patients treated with nilotinib as depicted by sustained deep molecular response in most CML patients.

### Supplementary Information

The online version contains supplementary material available at 10.1186/s12885-022-09605-1.

## Background

Chronic myeloid leukemia (CML) is a hematological disorder of myeloid progenitor cells. It generally presents with leukocytosis along with accumulation of myelocytes and neutrophils due to uncontrolled over production. The etiology of CML is complex and has been associated with excessive exposure to radiation. The diagnosis of CML is based on the presence or absence of an abnormally translocated chromosome called Philadelphia chromosome(Ph). As a result of this translocation oncogene ABL1 from chromosome 22 moves to the BCR gene on chromosome 9 and BCR gene moves to ABL1position, leading to the formation of BCR/ABL1 kinase. The resultant defective tyrosine kinase stimulates the uncontrolled proliferation of cells resulting in reduced apoptosis causing genomic instability [[1](#CR1), [2](#CR2)]. As a treatment option, targeted inhibitor of tyrosine kinase (TKI) such as imatinib mesylate (Glivec) was used for the treatment of CML. Initially it provided sustained molecular remission in CML patients but up to 33% of patients developed resistance and/or loss of response due to mutations and pharmacokinetic variability. Nilotinib (Tasigna) is a 2^nd^ generation TKI which overcomes the resistance and loss of response issue and achieves good tolerability and response against Ph positive CML [[3](#CR3), [4](#CR4)]. It is an aminophenylpyrimidine derivative with higher selectivity for BCR ABL1 kinase and is approved by the European Medicines Agency (EMA) and the Food and Drug Administration, USA (FDA) as first line treatment for CML due to quicker, deeper and sustained cytogenetic and molecular responses. The associated hepatic and pancreatic toxicities can be monitored through simple tests and are often manageable [[5](#CR5)]. While treating leukemia the prime objective is to control or reduce abnormal cell proliferation through targeted TKI therapies. When the adverse events associated with TKIs are beyond the acceptable limits change in TKI is suggested. Unfortunately, the available TKIs to treat CML in Pakistan are limited to imatinib and nilotinib. A high frequency of TKI domain mutation has been reported from local population where switch to nilotinib or ponatinib is suggested as second line [[6](#CR6)]. Due to the poor healthcare infrastructure and the unaffordable prolonged cancer treatment nilotinib is being prescribed as first line treatment for chronic and accelerated phase CML in Pakistan with strict monitoring and management of adverse events. Local studies targeting the long term outcome of first line nilotinib treatment and its safety as well as efficacy are required to support such practice.

Further, the risk for adverse drug reactions cannot be overlooked while using second generation TKIs as first line treatment of CML in adults. The interest of many researchers has developed to find out any association of drug resistance and lack of response to TKI in the presence of germline polymorphisms in drug-metabolizing enzymes (DMEs) genes [[3](#CR3)] and significant associations of favorable TKI therapeutic outcome (particularly of Imatinib) and polymorphic defects in GST and CYP has been documented [[7](#CR7), [8](#CR8)]. These polymorphisms may regulate drug uptake, metabolic activation and elimination. The DMEs help in xenobiotics deactivation and drug biotransformation. Polymorphisms in these DMEs genes may lead to a loss, reduction or increased activity of these enzymes. Hence, any defect in these genes may result in treatment failures, adverse effects and intoxication [[9](#CR9)]. There are three phases of drug metabolism in the human body involving distinct drugs detoxifying enzymes of DME Phase I enzymes such as CYP along with phase II DME help to convert a lipid soluble, non-polar xenobiotic into a polar hydrophilic non- toxic metabolite, which can be readily removed by phase III transporter enzymes [[10](#CR10)]. One of the important phase II DME is GSTs which conjugates xenobiotics to water soluble compound such as reduced glutathione (GSH), UDP-glucuronic acid, glycine. Beside conjugation, reduction of hydrogen peroxide resulting in the generation of oxidized glutathione also takes place by the action of GST. GSTs are super family of isozymes which are further divided into eight classes. These are mu (M), theta (T), pi (P), alpha (A), sigma (S), kappa (K), zeta (Z), and omega (O) [[11](#CR11)].

The variability in prevalence of DMEs polymorphisms among different population has been widely reported. These differences in metabolic capability due to polymorphic DMEs may impact the drug metabolism and eventually the treatment outcome.The pharmacogenetic screening for DMEs polymorphism may help either in defining personalized dosage of TKI or offering alternate management to non-responsive patients.Due to the dearth of extensive data on the frequencies of polymorphic variations in GSTs and CYP genes in CML patients from Pakistan, this case–control study was conducted with an aim to establish their frequencies to estimate the possible link of these polymorphisms with increased susceptibility to CML and secondly to determine the influence of GSTs and CYP polymorphism in predicting TKI treatment response in CML patients.

## Methods

### Study design

This case control study was conducted at National Institute of Blood Diseases and Bone Marrow Transplantation (NIBD) Karachi, between 2013 to 2019. Informed consent were taken from all patients and healthy subjects after the approval by ethical review committee of NIBD (NIBD/RD/155–37-2013 and conformed to the tenets of the Declaration of Helsinki.

### Study population

A total of 99 patients of CML and 169 age matched healthy controls were enrolled in this study. Patient’s selection was based on the presence of clinical signs and symptoms (abdominal discomfort, fatigue, weight loss and anemia) and Philadelphia chromosome or BCR-ABL gene fusion.

### Treatment and response definitions

In order to study the individual impact of polymorphic defects in DMEs on nilotinib treatment patients receiving nilotinib 300 mg two times/day before 1 h of having meals as first line were enrolled. ELN recommendations 2013 was followed for treatment response [[12](#CR12)]. Complete hematological response was defined at the three months of treatment as normal blood count with no immature granulocytes, basophilia or presence of blast along with non-palpable spleen. Major molecular response was defined as transcript ratio of BCR-ABL/ABL less than 0.1%. Deep Molecular response (minor and minimal) was defined as transcript ratio of BCR-ABL/ABL less than 0.01% and 0.0032% on IS respectively. Loss of response at any time after achieving molecular response and failure to achieve molecular response after 12 months were defined as treatment failure. Sokal risk score was also determined. It was defined as a prognostic index for CML patients which predicts response to treatment and survival at diagnosis. The patients were categorized as low, intermediate and high risk having Sokal score < 0.8, 0.8–1.2 and > 1.2 respectively. Responders were those who had achieved deep and major molecular response. Partial responder patients were those who have achieved CHR at 3 months where as Non-responders were those who failed to achieve hematological and molecular response at the given time points. The median follow up time for treatment response assessment was 47 months.

### Molecular analysis for DME polymorphisms

Fresh whole blood samples were collected in EDTA tubes. A peripheral blood smear was prepared for microscopic assessment and cell counting was performed by automated hematolyzerSysmex XN1000 (Kobe, Japan). Genomic DNA was isolated from whole blood using the QIAmp DNA Kit from Qiagen (Qiagen Cat #51,306, USA). The genetic polymorphism analysis for the GSTM1 and GSTT1 genes was based on multiplex PCR approach using previously designed primers by Rocha et al. [[13](#CR13)]. The homozygous deletion of GSTT1 and GSTM1 or the null allele results in no expression of these enzymes as confirmed by absence of amplified product by PCR.β-globinwas used as an internal control for each sample. The expected amplicon size was 480 bp in GSTT1 positive individuals and 215 bp in GSTM1. GSTP1A313G and CYP1A1*2C genotype was analyzed by RFLP-PCR as described by Voso et al. [[14](#CR14)] and Razmkhak et al. [[15](#CR15)].

### Statistical analysis

The frequency of polymorphic DMEs was compared between patient and healthy group by chi square test. The risk rate was also determined along with 95% confidence interval using statistical package SPSS version 22. A *p* value less than 0.05 was considered as statistically significant. Sokal score was determined by Sokal calculator. The OS was calculated by Kaplan–Meier method.

## Results

### Clinical features of CML patients

There were 63 males and 36 females altogether. Furthermore, 97 and 2 were diagnosed with CML in chronic and accelerated phase respectively (Table [1](#Tab1)). Out of 99, 66 were responders (male: 36 and female: 30). Six patients expired due to transformation to blast crisis (Table [1](#Tab1)). Among the functional biochemical variables only ALP was significantly raised in CML group as compared to healthy subjects (Table [2](#Tab2)).

### Table 1.

| Clinical Parameters | Number = 99 |
| --- | --- |
| Sokal relative risk score |  |
| < 0.8 | 66 |
| 0.8–1.2 | 27 |
| > 1.2 | 6 |
| Phase at diagnosis |  |
| Chronic | 97 |
| Accelerated | 2 |
| Blast Crisis | 0 |
| Treatment Outcome |  |
| Responders | 66 |
| Partial Responders | 21 |
| Non-Responders | 12 |
| Hematological Response |  |
| Complete | 66 |
| Partial | 30 |
| No | 3 |
| Molecular Response |  |
| Major MolecularResponse | 21 |
| Deep MolecularResponse | 45 |
| No Molecular Response | 33 |
| Progression |  |
| Yes | 10 |
| No | 89 |
| Status |  |
| Alive | 93 |
| Dead | 6 |

Table 1 Caption: Clinical features of CML patients.

### Table 2.

| Biochemistry Parameter | CMLMedian(interquartile range) | ControlMedian(interquartile range) | P value |
| --- | --- | --- | --- |
| Creatinine | 1(1.3) | 0.7(0.3) | 0.073 |
| Urea | 17(8.3) | 20(9.8) | 0.001 |
| Electrolytes |  |  |  |
| Sodium | 140(8.0) | 137(5) | 0.190 |
| Potassium | 4(1.6) | 3.6(0.8) | 0.028 |
| Chloride | 101(5) | 102(5) | 0.028 |
| Bicarbonates | 25(6.0) | 24(3) | 0.853 |
| Liver function tests |  |  |  |
| vTotal Bilirubin | 1(0.5) | 0.7(0.3) | 0.001 |
| Direct Bilirubin | 0.1(0.6) | 0.4(0.2) | 0.823 |
| vALP | 328(367) | 195(75) | 0.001 |
| SGPT | 22(14.3) | 22(47) | 0.001 |

Table 2 Caption: Biochemical parameter of CML patients

### Distribution of frequency of CYP1A1*2C, GSTP1A313G, and GSTM1/GSTT1 genotypes

The frequency of CYP1A1*2C, GSTP1A313G, and GSTM1/GSTT1 genotypes in a cohort of 99 CML patients and 169 controls was recorded. The heterozygous genotype of CYP 1A1*2C was more frequent in CML patients than in controls with an OR of 2.6 (*p* = 0.006; Table [3](#Tab3)). Similarly, GSTP1 Ile/Val and Val/Val mutant genotype expression was significantly higher in CML patients 20(15%) and 56(38%) compared to control groups 21(12%) and 6(4%) respectively. The GSTT1 and GSTM1 seemed to have no association with occurrence of CML either alone or in combination with each other (Table [3](#Tab3)). Further, the impact of combination of multiple polymorphic defects in different DME genes on occurrence of CML was also explored (Supplementary Table [1](#MOESM1)). Interestingly, seven different combinations listed in Table [3](#Tab3) turned out to be significantly linked to CML with a risk rate of 4.1–11.51.

### Table 3.

| Genotype | CML (n = 99) | Control (n = 169) | P value | OR (95% CI) |
| --- | --- | --- | --- | --- |
| CYP1A1 genotype |  |  |  |  |
| AA | 63 (64) | 137(81) |   |   |
| AG | 15 (15) | 15(9) | 0.006 | 2.686(1.326–5.440) |
| GG | 21 (21) | 17(10) | 0.666 | 1.235(0.473–3.225) |
| GSTP1 genotype |  |  |  |  |
| Ile/Ile | 51 (52) | 142(84) |   |   |
| Ile/Val | 36 (36) | 6(4) | 0.242 | 1.591(0.731–3.464) |
| Val/Val | 12 (12) | 21(12) | 0.001 | 0.095(0.031–0.291) |
| GSTM1/GSTT1 |  |  |  |  |
| Normal | 36 (36.4) | 76(46) |   |   |
| M deletion | 36 (36.4) | 55(33) | 0.747 | 0.844(0.303–2.357) |
| T deletion | 21 (21.2) | 23(13) | 0.351 | 0.611(0.217–1.722) |
| Double deletion | 6 (6) | 15(8) | 0.147 | 0.438(0.143–1.338) |
| Combined genotypea |  |  |  |  |
| AA + Ile/Val | 27 (27) | 7 (4.2) | 0.001 | 8.679(3.613–20.849) |
| AG + Ile/Val | 6 (6) | 0 | 0.023 | - |
| AG + M Deletion | 3 (3) | 0 | 0.024 | - |
| GG + T Deletion | 6 (6) | 2 (1.2) | 0.042 | 5.387(1.066–27.229) |
| Ile/Val + Normal | 12 (12) | 2 (1.2) | 0.001 | 11.517(2.521–52.618) |
| Ile/Val + M deletion | 12 (12) | 5 (3) | 0.006 | 4.524(1.544–13.258) |
| Ile/Val + T deletion | 21(12) | 4 (2.4) | 0.001 | 4.106(3.687–33.452) |

Table 3 Caption: Distribution of CYP1A1*2C, GSTP1A313G, and GSTM1/GSTT1 between CML patients and control

### Association of gene polymorphisms and hematological/molecular response

Patients were segregated as per their Sokal score at diagnosis and it was observed that the wild type GSTP1 was significantly associated with low and intermediate risk group patients (Fig. [1](#Fig1)). However, the double null deletion (absence of both GSTM1 and GSTT1) had significant association with high risk Sokal relative risk score (Fig. [1](#Fig1)). Patients harboring AA wild type of CYP1A1 genotype had higher rate of complete hematological response (42.85%) and deep molecular response (73.33%, Table [4](#Tab4)). Similarly, complete hematological response was observed mostly in those patients who carried both GSTM1 and GSTT1 genes. Partial hematological response was noticed to be higher in patients with T deletion (50%) than those who have wild type genes (GSTM1/GSTT1, Table [3](#Tab3)). Furthermore, the null GSTM1 was significantly associated with major molecular response (Table [4](#Tab4)). Failure to achieve molecular response was also influenced by different gene combinations (supplementary Table [2](#MOESM1)). It was interesting to note that Val/Val was significantly high in non-responders. A significant association was noted between GSTP1 heterozygous (Ile/Val) genotype and TFS (*p* = 0.005) whereas wild type CYP1A1 and GSTP1 (Ile/ Ile) was frequent in event free survivors (Table [5](#Tab5); *p* = 0.05).

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bad0/9080200/8c6bfcf47e56/12885_2022_9605_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9080200_12885_2022_9605_Fig1_HTML.jpg)

Frequency distribution of DME genotype in different Sokal risk groups

### Table 4.

| Responses |  | Genotype |  |  |  |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| CYP1A1 |  |  |  | GSTP1 |  |  |  | GSTM1/GSTT1 combined genotype |  |  |  |  |  |  |
| AA (a) | AG (b) | GG (c) | P-value (a vs b vs c) | Ile/Ile (a) | Ile/Val (b) | Val/Val (c) | P-value (a vs b vs c) | WT (a) | M−(b) | T−(c) | M−/T−(d) | P-value (a vs b vs c vs d) |  |  |
| HR(n = 33) | CHR (n = 21) | 9 | 9 | 3 | 0.029 | 9 | 12 | 0 | 0.001 | 15 | 6 | 0 | 0 | 0.001 |
| PHR(n = 12) | 9 | 0 | 3 | 0.1 | 0 | 3 | 9 | 0.001 | 3 | 3 | 6 | 0 | 0.001 |  |
| MR (n = 66) | MMR (n = 21) | 12 | 6 | 3 | 0.139 | 15 | 6 | 0 | 0.045 | 3 | 9 | 6 | 3 | 0.045 |
| DMR (n = 42) | 33 | 0 | 12 | 0.001 | 27 | 15 | 3 | 0.184 | 15 | 18 | 9 | 3 | 0.893 |  |
| NMR (n = 33) |   | 18 | 9 | 6 | 0.06 | 9 | 15 | 9 | 0.001 | 18 | 9 | 6 | 0 | 0.032 |

Table 4 Caption: Association of Hematological and Molecular response with respect to genes polymorphism in CML patients

### Table 5.

| Treatment outcome | Genotype |  |  |  |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| CYP1A1 |  |  |  | GSTP1 |  |  |  | GSTM1/GSTT1 combined genotype |  |  |  |  |  |
| AA (a) | AG (b) | GG (c) | P-value (a vs b vs c) | Ile/Ile (a) | Ile/Val (b) | Val/Val (c) | P-value (a vs b vs c) | WT(a) | M − (b) | T –(c) | M −/T−(d) | P-value (a vs b vs c vs d) |  |
| Responders (n = 66) | 45 | 6 | 15 | 0.05 | 42 | 21 | 3 | 0.001 | 18 | 27 | 15 | 6 | 0.032 |
| Partial Responders (n = 21) | 9 | 9 | 3 | 0.001 | 9 | 12 | 0 | 0.034 | 15 | 6 | 0 | 0 | 0.001 |
| Non-Responders (n = 12) | 9 | 0 | 3 | 0.295 | 0 | 3 | 9 | 0.001 | 3 | 3 | 6 | 0 | 0.068 |

Table 5 Caption: Impact of different genotypes on over all treatment outcomes

### Impact of DME genotypes on over all treatment outcome

The overall outcome of treatment was also determined at mean follow up of 51.33 months. There were 66 responders, 21 partial responders and 12 non-responders to nilotinib treatment (Table [5](#Tab5)). It was interesting to note that the wild type CYP1A1, GSTP1 and GSTM1 deletion was significantly frequent in responders. The partial responders carried heterozygous mutant genotypes of CYP1A1, GSTP1 and wild type of GSTM1/GSTT1 whereas homozygous GSTP1genotype was significantly linked to treatment failure. The GSTT1 deletion was also frequent in failure group but it could not reach the statistical significance. Out of 99 patients 6 patients died despite treatment and irrespective of their Sokal risk score. All these patients were non-responders male with an average age of 41 years. The relationship between DMEs polymorphisms and overall survival of nilotinib treated patients was also studied by log rank test. The GSTM1 and GSTT1 polymorphisms did not affect the overall survival. Whereas, the CYP1A1 AA genotype is associated with better survival than the GG homozygous genotype (log rank test *P* = 0.05). Furthermore, GSTP1A313G heterozygous mutant genotype tends to influence better survival (log rank test *P* value: 0.029, Fig. [2](#Fig2)).

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bad0/9080200/b097f6b95888/12885_2022_9605_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9080200_12885_2022_9605_Fig2_HTML.jpg)

Kaplan–Meier survival curves for the overall survival (Days) after TKI treatment until death as an endpoint is represented on the basis of DME genotype. a CYP1A1 genotype (log rank test P = 0.05). b GSTP1A313G (log rank test P value: 0.029)

## Discussion

The genetic alterations such as mutations and SNPs in many regulatory genes including TP53, KRAS, DDR2, KLK3 etc. are the leading cause of increased susceptibility to cancer. The variation in these regulatory genes influence regulation of vital cellular processes such as cell cycle, cell differentiation/ proliferation, DNA synthesis/repair, apoptosis, breakdown or synthesis of exogenous and endogenous substances and immune response.There are several different chemotherapeutic agents used to treat cancer by destroying rapidly growing cancer cells at almost every step of cell cycle. The DMEs play key role in the activation and/or detoxification of these drugs via biotransformation [[16](#CR16), [17](#CR17)]. The presence of polymorphic defects in DME’s has been confirmed as a contributing factor in occurrence and pathogenesis of leukemia and other cancers from different ethnicities. To the best of our knowledge, this is the first case control study encompassing influence of polymorphic defects in DMEs on the CML susceptibility and nilotinib treatment outcome from Pakistan. Previous reports on absence of association of polymorphic defects of DMEs and occurrence of CML are abundant [[18](#CR18)–[20](#CR20)]. Only few researchers so far have documented association of DMEs with risk of CML [[21](#CR21)]. One possible reason for this variance in observations might be variable degree of either exposure to harmful environmental factors or difference in frequency of polymorphic defects in DMEs among different populations causing accumulation of these. The germline gene polymorphism CYP1A1*2C in first phase DME results in gain of function due to overproduction of enzyme. Presence of excessive CYP1A1 results in decreased bioavailability of therapeutics and increased accumulation of harmful drug metabolic byproducts leading to more drug adverse events [[22](#CR22)]. In the current study, healthy individuals with the heterozygous genotype (AG) of the first phase DME CYP 1A1 were 2.6 times more susceptible to CML. Similarly, Hadeil et al., also reported a high prevalence of AG in 200 CML patients (OR: 18.38) [[23](#CR23)]. The homozygous deletion of GSTM1 and GSTT1 genes (null allele) results in no expression of these enzymes. About 30–60% individuals have GSTT1 null allele [[24](#CR24)]. Inconsistent results for presence of GSTT1 and GSTM1 null genotype and occurrence of CML were reported in the past from different populations. Hishida et al., documented a little impact of GSTT1 null allele with CML whereas among Argentineans both GSTM1 and GSTT1 null genotype were not a contributing factor in increased susceptibility to CML in healthy individuals [[25](#CR25)]. Similarly, homozygous deletion of both these genes seemed to have no influence on CML pathogenesis during the present study (Table [2](#Tab2)). On the contrary a protective effect of GSTT1 was reported by Bhat et al. from Kashmiris [[26](#CR26)]. Converse to the results of Rostami et al. the GSTP1 heterozygous genotype (Ile/Val) was frequent in healthy individuals and did not confer any risk of CML alone [[27](#CR27)]. But when in combination with other DME genotypes it seemed to contribute more in pathogenesis of CML with relatively high CML chance (Table [2](#Tab2)).

In case of CML the abnormal kinase activity of BCR ABL1 enzyme is controlled by treatment with TKIs. Imatinib was the first TKI successfully used to treat CML. Despite promising results; an increased rate of resistance (in ~ 30% patients) due to inconsistent pharmacokinetics, genetic variability in TKI binding sites and associated toxicities nilotinib was introduced as a second line TKI with a separate binding site. As per current ELAN recommendation nilotinib can be used as first line drug. Serious side effects of most TKIs have been linked to metabolic activation of these TKIs by cytochrome P450 mediated biotransformation. The nilotinib is considered safer than imatinib and is greatly metabolized in liver by the action of CYP3A. The extra hepatic metabolism of nilotinib if any is poorly understood. Presence of wild type -Cyp1A1, -GSTP1 and GSTM1 null genotype seemed to be significantly linked to a more sustained DMR whereas patients harboring GSTP1 Ile/Val and wild type GSTM1/GSTT1 failed to achieve any molecular response (*p* < 0.05;Table [3](#Tab3)).

Only few studies have shown the impact of DMEs polymorphisms on treatment outcome in CML. Altogether, 6 patients failed to respond to nilotinib and expired. All these patients had defects in DMEs genes and also harbored T315I kinase domain mutation already associated with nilotinib non-responsiveness in CML patients [[6](#CR6)]. Hence it can be assumed that the sudden death in these patients may also be associated with insufficient bio-detoxification due to defective DMEs.

In Malaysians the polymorphic GSTP1 genotypes and null GSTT1 was linked with imatinib resistance similarly these genotypes were frequent in partial and non-responder to nilotinib during the current study [[28](#CR28)]. The frequency of GSTM1 null genotype was significantly linked to good response in nilotinib treated patients (Table [5](#Tab5)). Moreover, the frequency was also high in patient’s achieving MMR (42.85%) and DMR (40%) but it could not reach the statistical significance. While studying the copy number variation analysis (CNV) of cytochromes and GSTs to predict efficacy of TKI in CML the researchers proposed that GSTM1 may not be a specific marker to prognose treatment outcome but GSTT1 may be associated with CML [[3](#CR3)]. Hence, null GSTM1 seemed to be frequent in nilotinib responders with possible influence on drug response..This is contrary to previous study where the GSTM1 null allele was associated with treatment failure in imatinib treated CML patients [[29](#CR29)]. The overproduction or absence of active enzyme may have direct or indirect influence on drug response due to increased or diminished drug metabolism and may be used drug dosage designing.

## Conclusion

This study concluded that heterozygous mutant of CYP1A1 and homozygous mutant of GSTP1 gene might be a contributing factor in CML pathogenesis. This study also shows that polymorphic variation in CYP1A1, GSTP1 and GSTM1 null genotype might influence the molecular response and treatment outcome in CML patients. Further studies at larger scale should be done to evaluate the impact of polymorphic DME genes on treatment related toxicity (if any) and relapse of disease.

## Supplementary Information

## Acknowledgements

The authors would like to thanks patients for their co-operation and Dr Tasneem Farzana for her guidance. All staffs of NIBD who was directly or indirectly involved in this project.

## Abbreviations

## Authors’ contributions

S.N.M: Planned, designed and supervised the study, helped with the optimization of assays, drafted and finalized the manuscript. A.S: Collected samples, performed hematological analysis, PCR, RFLP, recorded and analyzed the data, performed statistical analysis of data and prepared initial draft of manuscript under supervision; U.Z: Provided clinical samples and critically reviewed the manuscript. T.S.S: Provided patients samples along with expert opinion to revise the initial draft and reviewed the final manuscript. All authors have read and approved the manuscript.

## Funding

Not applicable.

## Availability of data and materials

The data produced during the present study will be available on request from the corresponding author.

## Declarations

### Ethics approval and consent to participate

Written informed consent was obtained from each patient following guidelines of world medical association declaration of Helsinki. The study was approved by the institutional review board of the National Institute of Blood disease and Bone Marrow Transplantation (NIBD), Karachi, Pakistan.

### Consent for publication

Not applicable.

### Competing interests

None to declare.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data produced during the present study will be available on request from the corresponding author.

### Supplementary Materials

### Data Availability Statement

The data produced during the present study will be available on request from the corresponding author.

## References

1. Höglund M, Sandin F, Simonsson B, Stenke L. Epidemiology of Chronic Myeloid Leukaemia. Chronic Myeloid Leukemia: Springer; 2021. pp. 37–48.  [DOI](https://doi.org/10.1007/s00277-015-2314-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25814090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Epidemiology%20of%20Chronic%20Myeloid%20Leukaemia&author=M%20H%C3%B6glund&author=F%20Sandin&author=B%20Simonsson&author=L%20Stenke&publication_year=2021&)

2. Perrotti D, Jamieson C, Goldman J, Skorski T. Chronic myeloid leukemia: mechanisms of blastic transformation. J Clinic Investig. 2010;120(7):2254–2264. doi: 10.1172/JCI41246.  [DOI](https://doi.org/10.1172/JCI41246) | [PMC free article](/articles/PMC2898591/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20592475/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clinic%20Investig&title=Chronic%20myeloid%20leukemia:%20mechanisms%20of%20blastic%20transformation&author=D%20Perrotti&author=C%20Jamieson&author=J%20Goldman&author=T%20Skorski&volume=120&issue=7&publication_year=2010&pages=2254-2264&pmid=20592475&doi=10.1172/JCI41246&)

3. Lavrov AV, Ustaeva OA, Adilgereeva EP, Smirnikhina SA, Chelysheva EY, Shukhov OA, et al. Copy number variation analysis in cytochromes and glutathione S-transferases may predict efficacy of tyrosine kinase inhibitors in chronic myeloid leukemia. PLoS ONE. 2017;12(9):e0182901. doi: 10.1371/journal.pone.0182901.  [DOI](https://doi.org/10.1371/journal.pone.0182901) | [PMC free article](/articles/PMC5597128/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28902850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Copy%20number%20variation%20analysis%20in%20cytochromes%20and%20glutathione%20S-transferases%20may%20predict%20efficacy%20of%20tyrosine%20kinase%20inhibitors%20in%20chronic%20myeloid%20leukemia&author=AV%20Lavrov&author=OA%20Ustaeva&author=EP%20Adilgereeva&author=SA%20Smirnikhina&author=EY%20Chelysheva&volume=12&issue=9&publication_year=2017&pages=e0182901&pmid=28902850&doi=10.1371/journal.pone.0182901&)

4. Blay J-Y, von Mehren M, editors. Nilotinib: a novel, selective tyrosine kinase inhibitor. Semin Oncol; 2011: Elsevier.  [DOI](https://doi.org/10.1053/j.seminoncol.2011.01.016) | [PMC free article](/articles/PMC4004101/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21419934/)

5. Palandri F, Castagnetti F, Soverini S, Poerio A, Gugliotta G, Luatti S, et al. Pancreatic enzyme elevation in chronic myeloid leukemia patients treated with nilotinib after imatinib failure. Haematologica. 2009;94(12):1758–1761. doi: 10.3324/haematol.2009.010496.  [DOI](https://doi.org/10.3324/haematol.2009.010496) | [PMC free article](/articles/PMC2791949/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19608673/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Haematologica&title=Pancreatic%20enzyme%20elevation%20in%20chronic%20myeloid%20leukemia%20patients%20treated%20with%20nilotinib%20after%20imatinib%20failure&author=F%20Palandri&author=F%20Castagnetti&author=S%20Soverini&author=A%20Poerio&author=G%20Gugliotta&volume=94&issue=12&publication_year=2009&pages=1758-1761&pmid=19608673&doi=10.3324/haematol.2009.010496&)

6. Zaidi U, Kaleem B, Borhany M, Maqsood S, Fatima N, Sufaida G, et al. Early and sustained deep molecular response achieved with nilotinib in high Sokal risk chronic myeloid leukemia patients. Cancer Manag Res. 2019;11:1493. doi: 10.2147/CMAR.S181911.  [DOI](https://doi.org/10.2147/CMAR.S181911) | [PMC free article](/articles/PMC6388993/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30863159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Manag%20Res&title=Early%20and%20sustained%20deep%20molecular%20response%20achieved%20with%20nilotinib%20in%20high%20Sokal%20risk%20chronic%20myeloid%20leukemia%20patients&author=U%20Zaidi&author=B%20Kaleem&author=M%20Borhany&author=S%20Maqsood&author=N%20Fatima&volume=11&publication_year=2019&pages=1493&pmid=30863159&doi=10.2147/CMAR.S181911&)

7. Seong S, Lim M, Sohn S, Moon J, Oh S-J, Kim BS, et al. Influence of enzyme and transporter polymorphisms on trough imatinib concentration and clinical response in chronic myeloid leukemia patients. Ann Oncol. 2013;24(3):756–60.  [DOI](https://doi.org/10.1093/annonc/mds532) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23117072/)

8. Maddin N, Husin A, Gan SH, Aziz BA, Ankathil R. Impact of CYP3A4* 18 and CYP3A5* 3 polymorphisms on imatinib mesylate response among chronic myeloid leukemia patients in Malaysia. Oncol Therp. 2016;4(2):303–314. doi: 10.1007/s40487-016-0035-x.  [DOI](https://doi.org/10.1007/s40487-016-0035-x) | [PMC free article](/articles/PMC5315081/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28261657/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncol%20Therp&title=Impact%20of%20CYP3A4*%2018%20and%20CYP3A5*%203%20polymorphisms%20on%20imatinib%20mesylate%20response%20among%20chronic%20myeloid%20leukemia%20patients%20in%20Malaysia&author=N%20Maddin&author=A%20Husin&author=SH%20Gan&author=BA%20Aziz&author=R%20Ankathil&volume=4&issue=2&publication_year=2016&pages=303-314&pmid=28261657&doi=10.1007/s40487-016-0035-x&)

9. Ota T, Kamada Y, Hayashida M, Iwao-Koizumi K, Murata S, Kinoshita K. Combination analysis in genetic polymorphisms of drug-metabolizing enzymes CYP1A2, CYP2C9, CYP2C19, CYP2D6 and CYP3A5 in the Japanese population. Int J Med Sci. 2015;12(1):78. doi: 10.7150/ijms.10263.  [DOI](https://doi.org/10.7150/ijms.10263) | [PMC free article](/articles/PMC4278879/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25552922/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Med%20Sci&title=Combination%20analysis%20in%20genetic%20polymorphisms%20of%20drug-metabolizing%20enzymes%20CYP1A2,%20CYP2C9,%20CYP2C19,%20CYP2D6%20and%20CYP3A5%20in%20the%20Japanese%20population&author=T%20Ota&author=Y%20Kamada&author=M%20Hayashida&author=K%20Iwao-Koizumi&author=S%20Murata&volume=12&issue=1&publication_year=2015&pages=78&pmid=25552922&doi=10.7150/ijms.10263&)

10. Vettriselvi V, Vijayalakshmi K, Paul SF, Venkatachalam P. Genetic variation of GSTM1, GSTT1 and GSTP1 genes in a South Indian population. Asian Pac J Cancer Prev. 2006;7(2):325.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16839232/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asian%20Pac%20J%20Cancer%20Prev&title=Genetic%20variation%20of%20GSTM1,%20GSTT1%20and%20GSTP1%20genes%20in%20a%20South%20Indian%20population&author=V%20Vettriselvi&author=K%20Vijayalakshmi&author=SF%20Paul&author=P%20Venkatachalam&volume=7&issue=2&publication_year=2006&pages=325&pmid=16839232&)

11. Hayes JD, Flanagan JU, Jowsey IR. Glutathione transferases. Annu Rev Pharmacol Toxicol. 2005;45:51–88. doi: 10.1146/annurev.pharmtox.45.120403.095857.  [DOI](https://doi.org/10.1146/annurev.pharmtox.45.120403.095857) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15822171/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=Glutathione%20transferases&author=JD%20Hayes&author=JU%20Flanagan&author=IR%20Jowsey&volume=45&publication_year=2005&pages=51-88&pmid=15822171&doi=10.1146/annurev.pharmtox.45.120403.095857&)

12. Baccarani M, Deininger MW, Rosti G, Hochhaus A, Soverini S, Apperley JF, et al. European LeukemiaNet recommendations for the management of chronic myeloid leukemia: 2013. Blood. 2013;122(6):872–884. doi: 10.1182/blood-2013-05-501569.  [DOI](https://doi.org/10.1182/blood-2013-05-501569) | [PMC free article](/articles/PMC4915804/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23803709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood&title=European%20LeukemiaNet%20recommendations%20for%20the%20management%20of%20chronic%20myeloid%20leukemia:%202013&author=M%20Baccarani&author=MW%20Deininger&author=G%20Rosti&author=A%20Hochhaus&author=S%20Soverini&volume=122&issue=6&publication_year=2013&pages=872-884&pmid=23803709&doi=10.1182/blood-2013-05-501569&)

13. Rocha V, Porcher R, Fernandes J, Filion A, Bittencourt H, Silva W, et al. Association of drug metabolism gene polymorphisms with toxicities, graft-versus-host disease and survival after HLA-identical sibling hematopoietic stem cell transplantation for patients with leukemia. Leukemia. 2009;23(3):545–556. doi: 10.1038/leu.2008.323.  [DOI](https://doi.org/10.1038/leu.2008.323) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19005482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia&title=Association%20of%20drug%20metabolism%20gene%20polymorphisms%20with%20toxicities,%20graft-versus-host%20disease%20and%20survival%20after%20HLA-identical%20sibling%20hematopoietic%20stem%20cell%20transplantation%20for%20patients%20with%20leukemia&author=V%20Rocha&author=R%20Porcher&author=J%20Fernandes&author=A%20Filion&author=H%20Bittencourt&volume=23&issue=3&publication_year=2009&pages=545-556&pmid=19005482&doi=10.1038/leu.2008.323&)

14. Voso MT, Fabiani E, D'Alo F, Guidi F, Di Ruscio A, Sica S, et al. Increased risk of acute myeloid leukaemia due to polymorphisms in detoxification and DNA repair enzymes. Ann Oncol. 2007;18(9):1523–1528. doi: 10.1093/annonc/mdm191.  [DOI](https://doi.org/10.1093/annonc/mdm191) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17761709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Increased%20risk%20of%20acute%20myeloid%20leukaemia%20due%20to%20polymorphisms%20in%20detoxification%20and%20DNA%20repair%20enzymes&author=MT%20Voso&author=E%20Fabiani&author=F%20D'Alo&author=F%20Guidi&author=A%20Di%20Ruscio&volume=18&issue=9&publication_year=2007&pages=1523-1528&pmid=17761709&doi=10.1093/annonc/mdm191&)

15. Razmkhah F, Pazhakh V, Zaker F, Atashrazm F, Sheikhi M. Frequency of CYP1A1* 2C polymorphism in patients with leukemia in the Iranian population. Lab Med. 2011;42(4):220–223. doi: 10.1309/LM337JWOSVNEHPUI.  [DOI](https://doi.org/10.1309/LM337JWOSVNEHPUI) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lab%20Med&title=Frequency%20of%20CYP1A1*%202C%20polymorphism%20in%20patients%20with%20leukemia%20in%20the%20Iranian%20population&author=F%20Razmkhah&author=V%20Pazhakh&author=F%20Zaker&author=F%20Atashrazm&author=M%20Sheikhi&volume=42&issue=4&publication_year=2011&pages=220-223&doi=10.1309/LM337JWOSVNEHPUI&)

16. Fathi Z, Syn NL, Zhou J-G, Roudi R. Molecular epidemiology of lung cancer in Iran: implications for drug development and cancer prevention. J Hum Genet. 2018;63(7):783–794. doi: 10.1038/s10038-018-0450-y.  [DOI](https://doi.org/10.1038/s10038-018-0450-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29666465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hum%20Genet&title=Molecular%20epidemiology%20of%20lung%20cancer%20in%20Iran:%20implications%20for%20drug%20development%20and%20cancer%20prevention&author=Z%20Fathi&author=NL%20Syn&author=J-G%20Zhou&author=R%20Roudi&volume=63&issue=7&publication_year=2018&pages=783-794&pmid=29666465&doi=10.1038/s10038-018-0450-y&)

17. Sadeghi H, Nazemalhosseini-Mojarad E, Yassaee VR, Savabkar S, Ghasemian M, Aghdaei HA, et al. Could CYP24A1 promoter methylation status affect the gene expression in the colorectal cancer patients? Meta Gene. 2020;24:100656. doi: 10.1016/j.mgene.2020.100656.  [DOI](https://doi.org/10.1016/j.mgene.2020.100656) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Meta%20Gene&title=Could%20CYP24A1%20promoter%20methylation%20status%20affect%20the%20gene%20expression%20in%20the%20colorectal%20cancer%20patients?&author=H%20Sadeghi&author=E%20Nazemalhosseini-Mojarad&author=VR%20Yassaee&author=S%20Savabkar&author=M%20Ghasemian&volume=24&publication_year=2020&pages=100656&doi=10.1016/j.mgene.2020.100656&)

18. Ovsepian V, EIu V, Sherstneva E. Cytochrome P4501A1, glutathione S-transferase M1 and T1 gene polymorphisms in chronic myeloid leukemia. Genetika. 2010;46(10):1360–2.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/21254556/)

19. Mahmoud S, Labib DA, Khalifa RH, Khalil REA, Marie MA. CYP1A1, GSTM1 and GSTT1 genetic polymorphism in Egyptian chronic myeloid leukemia patients. J immunol Res. 2010;3(1):12–21. doi: 10.3923/rji.2010.12.21.  [DOI](https://doi.org/10.3923/rji.2010.12.21) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20immunol%20Res&title=CYP1A1,%20GSTM1%20and%20GSTT1%20genetic%20polymorphism%20in%20Egyptian%20chronic%20myeloid%20leukemia%20patients&author=S%20Mahmoud&author=DA%20Labib&author=RH%20Khalifa&author=REA%20Khalil&author=MA%20Marie&volume=3&issue=1&publication_year=2010&pages=12-21&doi=10.3923/rji.2010.12.21&)

20. Chen H-c, Hu W-x, Liu Q-x, Li W-k, Chen F-z, Rao Z-z, et al. Genetic polymorphisms of metabolic enzymes CYP1A1, CYP2D6, GSTM1 and GSTT1 and leukemia susceptibility. Euro J cancer Prev. 2008;17(3):251–258. doi: 10.1097/CEJ.0b013e3282b72093.  [DOI](https://doi.org/10.1097/CEJ.0b013e3282b72093) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18414197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Euro%20J%20cancer%20Prev&title=Genetic%20polymorphisms%20of%20metabolic%20enzymes%20CYP1A1,%20CYP2D6,%20GSTM1%20and%20GSTT1%20and%20leukemia%20susceptibility&author=H-c%20Chen&author=W-x%20Hu&author=Q-x%20Liu&author=W-k%20Li&author=F-z%20Chen&volume=17&issue=3&publication_year=2008&pages=251-258&pmid=18414197&doi=10.1097/CEJ.0b013e3282b72093&)

21. Al-Achkar W, Azeiz G, Moassass F, Wafa A. Influence of CYP1A1, GST polymorphisms and susceptibility risk of chronic myeloid leukemia in Syrian population. Med Oncol. 2014;31(5):889. doi: 10.1007/s12032-014-0889-4.  [DOI](https://doi.org/10.1007/s12032-014-0889-4) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24671854/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Oncol&title=Influence%20of%20CYP1A1,%20GST%20polymorphisms%20and%20susceptibility%20risk%20of%20chronic%20myeloid%20leukemia%20in%20Syrian%20population&author=W%20Al-Achkar&author=G%20Azeiz&author=F%20Moassass&author=A%20Wafa&volume=31&issue=5&publication_year=2014&pages=889&pmid=24671854&doi=10.1007/s12032-014-0889-4&)

22. Lakkireddy S, Aula S, Swamy A, Kapley A, Digumarti RR, Jamil K. Association of the common CYP1A1* 2C variant (Ile462Val polymorphism) with chronic myeloid leukemia (CML) in patients undergoing imatinib therapy. Cell J (Yakhteh) 2015;17(3):510. doi: 10.22074/cellj.2015.11.  [DOI](https://doi.org/10.22074/cellj.2015.11) | [PMC free article](/articles/PMC4601872/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26464823/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20J%20(Yakhteh)&title=Association%20of%20the%20common%20CYP1A1*%202C%20variant%20(Ile462Val%20polymorphism)%20with%20chronic%20myeloid%20leukemia%20(CML)%20in%20patients%20undergoing%20imatinib%20therapy&author=S%20Lakkireddy&author=S%20Aula&author=A%20Swamy&author=A%20Kapley&author=RR%20Digumarti&volume=17&issue=3&publication_year=2015&pages=510&pmid=26464823&doi=10.22074/cellj.2015.11&)

23. Idris HM, Khalil HB, Mills J, Elderdery AY. CYP1A1 and CYP2D6 polymorphisms and susceptibility to chronic myelocytic leukaemia. Curr Cancer Drug Targets. 2020;20(9):675–680. doi: 10.2174/1570163817666200518081356.  [DOI](https://doi.org/10.2174/1570163817666200518081356) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32418524/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Cancer%20Drug%20Targets&title=CYP1A1%20and%20CYP2D6%20polymorphisms%20and%20susceptibility%20to%20chronic%20myelocytic%20leukaemia&author=HM%20Idris&author=HB%20Khalil&author=J%20Mills&author=AY%20Elderdery&volume=20&issue=9&publication_year=2020&pages=675-680&pmid=32418524&doi=10.2174/1570163817666200518081356&)

24. Pemble S, Schroeder KR, Spencer SR, Meyer DJ, Hallier E, Bolt HM, et al. Human glutathione S-transferase theta (GSTT1): cDNA cloning and the characterization of a genetic polymorphism. Biochem. 1994;300(1):271–276. doi: 10.1042/bj3000271.  [DOI](https://doi.org/10.1042/bj3000271) | [PMC free article](/articles/PMC1138152/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8198545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem&title=Human%20glutathione%20S-transferase%20theta%20(GSTT1):%20cDNA%20cloning%20and%20the%20characterization%20of%20a%20genetic%20polymorphism&author=S%20Pemble&author=KR%20Schroeder&author=SR%20Spencer&author=DJ%20Meyer&author=E%20Hallier&volume=300&issue=1&publication_year=1994&pages=271-276&pmid=8198545&doi=10.1042/bj3000271&)

25. Hishida A, Terakura S, Emi N, Yamamoto K, Murata M, Nishio K, et al. GSTT1 and GSTM1 deletions, NQO1 C609T polymorphism and risk of chronic myelogenous leukemia in Japanese. Asian Pac J Cancer Prev. 2005;6(3):251.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/16235982/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asian%20Pac%20J%20Cancer%20Prev&title=GSTT1%20and%20GSTM1%20deletions,%20NQO1%20C609T%20polymorphism%20and%20risk%20of%20chronic%20myelogenous%20leukemia%20in%20Japanese&author=A%20Hishida&author=S%20Terakura&author=N%20Emi&author=K%20Yamamoto&author=M%20Murata&volume=6&issue=3&publication_year=2005&pages=251&pmid=16235982&)

26. Bhat G, Bhat A, Wani A, Sadiq N, Jeelani S, Kaur R, et al. Polymorphic variation in glutathione-S-transferase genes and risk of chronic myeloid leukaemia in the Kashmiri population. Asian Pac J Cancer Prev. 2012;13(1):69–73. doi: 10.7314/APJCP.2012.13.1.069.  [DOI](https://doi.org/10.7314/APJCP.2012.13.1.069) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22502716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asian%20Pac%20J%20Cancer%20Prev&title=Polymorphic%20variation%20in%20glutathione-S-transferase%20genes%20and%20risk%20of%20chronic%20myeloid%20leukaemia%20in%20the%20Kashmiri%20population&author=G%20Bhat&author=A%20Bhat&author=A%20Wani&author=N%20Sadiq&author=S%20Jeelani&volume=13&issue=1&publication_year=2012&pages=69-73&pmid=22502716&doi=10.7314/APJCP.2012.13.1.069&)

27. Rostami G, Assad D, Ghadyani F, Hamid M, Karami A, Jalaeikhoo H, et al. Influence of glutathione S-transferases (GSTM1, GSTT1, and GSTP1) genetic polymorphisms and smoking on susceptibility risk of chronic myeloid leukemia and treatment response. Mol Genetics Genomic Med. 2019;7(7):e00717. doi: 10.1002/mgg3.717.  [DOI](https://doi.org/10.1002/mgg3.717) | [PMC free article](/articles/PMC6625153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31111691/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Genetics%20Genomic%20Med&title=Influence%20of%20glutathione%20S-transferases%20(GSTM1,%20GSTT1,%20and%20GSTP1)%20genetic%20polymorphisms%20and%20smoking%20on%20susceptibility%20risk%20of%20chronic%20myeloid%20leukemia%20and%20treatment%20response&author=G%20Rostami&author=D%20Assad&author=F%20Ghadyani&author=M%20Hamid&author=A%20Karami&volume=7&issue=7&publication_year=2019&pages=e00717&pmid=31111691&doi=10.1002/mgg3.717&)

28. Makhtar SM, Husin A, Baba AA, Ankathil R. Association of GSTM1, GSTT1 and GSTP1 Ile105Val polymorphisms with clinical response to imatinib mesylate treatment among Malaysian chronic myeloid leukaemia patients. J Genetics. 2017;96(4):633–639. doi: 10.1007/s12041-017-0819-2.  [DOI](https://doi.org/10.1007/s12041-017-0819-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28947711/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Genetics&title=Association%20of%20GSTM1,%20GSTT1%20and%20GSTP1%20Ile105Val%20polymorphisms%20with%20clinical%20response%20to%20imatinib%20mesylate%20treatment%20among%20Malaysian%20chronic%20myeloid%20leukaemia%20patients&author=SM%20Makhtar&author=A%20Husin&author=AA%20Baba&author=R%20Ankathil&volume=96&issue=4&publication_year=2017&pages=633-639&pmid=28947711&doi=10.1007/s12041-017-0819-2&)

29. Davies A, Giannoudis A, Zhang J, Austin G, Wang L, Holyoake T, et al. Dual glutathione S transferase θ1 and μ1 gene deletions determine imatinib failure in chronic myeloid leukemia. Clin Pharma Therap. 2014;96(6):694–703. doi: 10.1038/clpt.2014.176.  [DOI](https://doi.org/10.1038/clpt.2014.176) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25188725/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharma%20Therap&title=Dual%20glutathione%20S%20transferase%20%CE%B81%20and%20%CE%BC1%20gene%20deletions%20determine%20imatinib%20failure%20in%20chronic%20myeloid%20leukemia&author=A%20Davies&author=A%20Giannoudis&author=J%20Zhang&author=G%20Austin&author=L%20Wang&volume=96&issue=6&publication_year=2014&pages=694-703&pmid=25188725&doi=10.1038/clpt.2014.176&)
