# HLA-B*07:02 and HLA-C*07:02 are associated with trimethoprim-sulfamethoxazole respiratory failure

## Metadata
**Authors:** Jennifer L Goldman, Jenna O Miller, Neil Miller, Robert Eveleigh, Andrew Gibson, Elizabeth J Phillips, Tomi Pastinen
**Journal:** The pharmacogenomics journal
**Date:** 2022 Feb 16
**DOI:** [10.1038/s41397-022-00266-8](https://doi.org/10.1038/s41397-022-00266-8)
**PMID:** 35169303
**PMCID:** PMC9125581
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9125581/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9125581/pdf/nihms-1804604.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9125581/pdf/nihms-1804604.pdf)

## Abstract

We have identified an underrecognized severe adverse drug reaction (ADR) of trimethoprim-sulfamethoxazole (TMP-SMX) associated respiratory failure in previously healthy children and young adults. We investigated potential genetic risk factors associated with TMP-SMX induced respiratory failure in a cohort of seven patients. We explored whole genome sequence among seven patients representing nearly half of all reported cases worldwide and 63 unrelated control individuals in two stages: (1) human leukocyte antigen (HLA) locus variation as several other ADRs have been associated HLA genetic variants and (2) coding variation to catalog and explore potential rare variants contributing to this devastating reaction. All cases were either heterozygous (carriers) or homozygous for the common HLA-B*07:02–HLA-C*07:02 haplotype. Despite the small sample size, this observation is statistically significant both in conservative comparison to maximum reported population frequencies (binomial P = 0.00017 for HLA-B and P = 0.00028 for HLA-C) and to our control population assessed by same HLA genotyping approach (binomial P = 0.000001 for HLA-B and P = 0.000018 for HLA-C). No gene elsewhere in the genome harnessed shared rare case enriched coding variation. Our results suggests that HLA-B*07:02 and HLA-C*07:02 are necessary for a patient to develop respiratory failure due to TMP-SMX.

## INTRODUCTION

Trimethoprim-sulfamethoxazole (TMP-SMX) is an antibiotic commonly used to treat and prevent infections. Although highly effective, TMP-SMX is associated with a wide variety of adverse drug reactions (ADR), some which are severe and life threatening. TMP-SMX associated ADRs vary and can affect many organ systems. For example, TMP-SMX can cause severe skin and mucosa damage including Stevens-Johnson syndrome/toxic epidermal necrolysis [[1](#R1), [2](#R2)]. TMP-SMX is a primary cause of drug-induced liver injury (DILI) resulting in several patterns of cholestatic or hepatocellular damage [[3](#R3)]. TMP-SMX induces blood dyscrasias resulting in bone marrow suppression and cytopenias [[4](#R4)]. Hypersensitivity syndrome is a multisystem ADR presenting with a constellation of symptoms including rash, fever, lymphadenopathy, internal organ involvement, eosinophilia and other blood cell line abnormalities [[5](#R5)].

Pharmacogenomic risk factors associated with TMP-SMX have been identified with some success and are specific to ADR phenotype. *HLA-B*14:01* has been associated with TMP-SMX DILI in European Americans with 5/51 (10%) of cases with *HLA-B*14:01* and an allele frequency (AF) 5.5-fold higher than in controls (AF = 0.049 vs. 0.009, Fisher *P* = 0.002) [[6](#R6)]. *HLA-B*35:01* has been identified as a potential genetic risk factor for TMP-SMX DILI in African Americans with 5/10 (50%) of cases carrying the allele and a 2.8-fold higher AF in cases than controls (AF: 0.25 vs. 0.087; *P* = 0.026) [[6](#R6)]. In TMP-SMX severe cutaneous reactions, *HLA-B*13:01* has been identified as an important genetic factor in Asians with AF of 53% in the patients with severe cutaneous drug reactions as compared to only 11% in the controls [[7](#R7)].

Recently, we identified an underrecognized severe ADR of respiratory failure associated with TMP-SMX [[8](#R8)]. The extreme clinical presentation involves previously healthy children and young adults with respiratory failure requiring prolonged extracorporeal membrane oxygenation (ECMO) support. The respiratory failure was associated with TMP-SMX and no other cause was identified. Furthermore, a novel lung pathological finding termed diffuse alveolar injury with delayed epithelialization was identified in these cases [[9](#R9)]. Given the striking similarity and severity of these TMP-SMX associated respiratory failure cases, we set out to investigate pharmacogenomic associations with TMP-SMX induced pulmonary failure in a well-defined cohort of patients.

## MATERIALS AND METHODS

### Subjects

Cases of TMP-SMX associated respiratory failure were identified following national news coverage of the index case of an adolescent patient requiring ECMO support [[8](#R8)]. Additional cases were referred by families and physicians across the U.S. Medical records were extensively reviewed, and inclusion criteria were: (1) no previous history of pulmonary disease, (2) ≥7 days exposure to TMP-SMX with documentation of prescribing date and dose, (3) medical records available for review by the authors, and (4) no alternative explanation for acute respiratory failure revealed by medical evaluation and testing. The Naranjo causality assessment tool for ADR was completed on review of each case and all cases scored as probable for implicating TMP-SMX based on the temporal sequence of respiratory failure following drug exposure, lack of alternative explanation despite extensive medical evaluation, and previous reports of TMP-SMX pulmonary toxicity [[10](#R10)]. Available lung biopsies were also evaluated for evidence of diffuse alveolar injury with delayed epithelialization.

To date, 14 cases have been identified that met inclusion criteria. Overall mortality was 36%. Of the patients who survived, seven were available to consent (or parental consent if <18 years) into the Repository for Adverse Drug Reactions at Children’s Mercy Kansas City (IRB# 17110669). One patient did subsequently die from the complication of the ADR following enrollment and was included in this study. Genomic analysis was performed under IRB approved study #00000504.

A set of 63 unaffected, randomly chosen controls were included from the Genomic Answers for Kids study which is a health system wide genomic repository to find genetic causes of childhood disease and includes whole genome sequencing (WGS) for all family members. Similar to case samples based on genetic ancestry analyses, these controls represent the general U.S. population allowing us to compare not only reported allele frequencies, but also to allele frequencies in our WGS based HLA pipeline [[11](#R11)]. See [Supplementary Data Fig. S1](#SD1). HLA profiles of cases were also compared to >75,000 patients in Vanderbilt’s BioVu DNA biobank [[12](#R12)].

### Genetic analyses methods

We performed WGS on seven ADR patient samples. We opted for WGS as it allows analyses of both classical MHC analyses for HLA-region variation but also builds resources for this rare disease going forward [[13](#R13)]. HLA is a natural target for 1st pass analyses due to its known role in delayed immune-mediated ADR many of which are T-cell mediated [[14](#R14)]. The WGS data were aligned to human reference genome GRCh37 (hg19) using DRAGEN v3.3.7 and variants were called in joint genotyping mode. For classical MHC allele definitions, we utilized additional approaches, which entailed realignment of chr6 and unaligned read fractions by Athlates13 along with genotype calling for three MHC Class I loci (HLA-A, HLA-B, and HLA-C) and two MHC Class II loci (HLA-DQB1 and HLA-DRB1). Four samples were also genotyped by high resolution HLA typing on the Illumina platform through the Immunogenomics microbial genetics and single cell technologies core, Vanderbilt University Medical Center (VUMC) using the reference sequences from the IMGT/HLA sequence database (release v3380), the IIID HLA analysis suite (release v3.12) and the IIID Allele Caller (release v3.1). Collectively this yielded 18/20 matching allele calls (90%) for 2-digit alleles and 17/20 matching calls for 6-digit alleles (85%) across the 5 loci. Given the prevalence of *HLA-B*07:02* and *HLA-C*07:02* in the cases we also visualized the tendency for carriage of known HLA class I alleles in the 7.1 ancestral haplotype in cases versus Genomic Answers for Kids controls and controls in the VuMC BioVu ([Fig. 1](#F1)).

### Fig. 1. Overrepresentation of HLA-B*07:02 and HLA-C*07:02 in TMP-SMX associated respiratory failure cases compared to 63 controls and 75,902 population controls of European ancestry.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/54d3/9125581/f1985e484caa/nihms-1804604-f0001.jpg)

Visualization of the 7.1 ancestral haplotype alleles HLA-A*03:01, HLA-B*07:02, HLA-C*07:02 and HLA-DRB1*15:02 shown as the proportion of the entire representative population show a similar pattern for TMP-SMX versus population controls however the HLA-B*07:02 and HLA-C*07:02 block is only held together across the cases. Study controls (A), Population controls (B), Study cases (C).

## RESULTS

### TMP-SMX cases and controls

The seven TMP-SMX respiratory failure cases occurred from 1997 to 2018 in the U.S. TMP-SMX was implicated and deemed the probable cause of symptoms by Naranjo scale in all cases [[10](#R10)]. [Table 1](#T1) describes the TMP-SMX associated pulmonary failure cases. The median case age was 16 (range 15–34) years, the majority female (57%), and majority White (57%). Notably the cohort does include a mixed race and Indian patient. The median TMP-SMX latency prior to clinical presentation was 21 days. All cases required tracheostomy, and all but 1 case was placed on ECMO support. The median hospital length of stay was 83 days. Adequate lung tissue samples were available from three respiratory failure cases, and all demonstrated evidence of diffuse alveolar injury with delayed epithelialization.

### Table 1.

| Characteristic | Value N = 7 |
| --- | --- |
| Median age (range)—year | 16 (15–34) |
| Female sex—no. (%) | 4 (57) |
| Self-reported race—no. (%) |   |
| White | 5 (72) |
| Mixed race | 1 (14) |
| Asian | 1 (14) |
| TMP-SMX exposure |   |
| Median days (range) | 21 (9–24) |
| Indications for TMP-SMX—no. (%) |   |
| Acne | 4 (57) |
| Skin and soft tissue infection | 1 (14) |
| Sinus infection | 1 (14) |
| Urinary tract infection | 1 (14) |
| Median days hospital length of stay (range) | 83 (53–459) |
| Median days advanced airway (range) | 135 (47–455) |
| Tracheostomy—no. (%) | 7 (100) |
| ECMO—no. (%) | 6 (86) |
| Median ECMO days (range) | 67 (31–193) |
| Lung transplant—no. (%) | 1 (14) |
| Death—no. (%) | 1 (14) |

Table 1 Caption: Clinical characteristics of patients with trimethoprim-sulfamethoxazole associated respiratory failure.

### HLA results

Most strikingly, all cases were carriers of both *HLA-B*07:02* and *HLA-C*07:02* (100% carrier rate) [Table 2](#T2), where the maximum (in self-reported Caucasian ancestry) carrier rate in U.S. populations has been reported at 29% and 31%, respectively [[15](#R15)]. Our control population representing unaffected parents of children suspected of diverse set of rare disease (not ascertained for any specific organ system disease), which shows better matched genetic ancestry distribution as our patient sample, (see [Supplementary Data Fig. S1](#SD1), Eigenstrat), and the carrier rate for both was lower, at 14% for *HLA-B*07:02* and 21% for *HLA-C*07:02* (see [Supplementary Data Table S1](#SD1)).

### Table 2.

| Case or Control | HLA-A Allele 1/2 | HLA-B Allele 1/2 | HLA-C Allele 1/2 | HLA-DRB1 Allele 1/2 |
| --- | --- | --- | --- | --- |
| Case | 02:01/33:03 | 07:02/53:01 | 04:01/07:02 | 15:03/15:03 |
| Case | 02:01/02:01 | 07:02/51:01 | 07:02/14:02 | 14:01/15:01 |
| Case | 02:01/31:01 | 07:02/15:01 | 03:03/07:02 | 04:01/13:01 |
| Case | 03:01/23:01 | 07:02/07:02 | 07:01/07:02 | 07:01/15:01 |
| Case | 11:01/11:01 | 07:02/40:06 | 15:02/07:02 | 04:03/14:04 |
| Case | 02:01/03:01 | 07:02/07:02 | 07:02/07:137 | 01:01/07:01 |
| Case | 31:01/03:36 | 07:02/40:01 | 07:02/03:04 | 15:01/04:04 |
| Control | 01:01/31:01 | 08:01/35:03 | 07:01/04:01 | 01:01/03:01 |
| Control | 03:01/25:01 | 18:01/44:02 | 12:03/05:01 | 11:03/15:01 |
| Control | 01:01/11:01 | 44:02/55:01 | 07:04/03:03 | 01:01/14:01 |
| Control | 02:01/31:01 | 27:05/40:01 | 02:02/03:04 | 04:01/4:04 |
| Control | 01:01/03:01 | 35:01/37:01 | 04:01/06:02 | 01:01/11:01 |
| Control | 68:01/68:01 | 27:05/44:02 | 02:02/05:01 | 04:01/13:01 |
| Control | 24:86/01:01 | 13:02/44:02 | 07:04/06:02 | 07:01/13:01 |
| Control | 02:01/02:01 | 13:02/15:01 | 04:01/06:02 | 07:01/15:01 |
| Control | 01:01/29:02 | 08:01/44:03 | 07:01/16:01 | 03:01/07:01 |
| Control | 01:01/03:01 | 08:01/57:01 | 06:02/07:01 | 04:07/13:01 |
| Control | 01:01/01:01 | 55:01/44:05 | 02:02/03:03 | 04:04/16:01 |
| Control | 11:01/32:01 | 55:01/55:01 | 03:03/03:03 | 14:01/15:01 |
| Control | 24:02/29:02 | 08:01/44:03 | 07:01/16:01 | 07:01/11:01 |
| Control | 02:01/32:01 | 40:01/44:02 | 05:01/03:04 | 04:01/04:04 |
| Control | 23:01/29:02 | 15:03/13:02 | 02:10/06:02 | 07:01/08:04 |
| Control | 01:01/24:02 | 35:43/57:01 | 07:01/01:02 | 04:07/13:01 |
| Control | 31:01/24:02 | 35:12/35:17 | 04:01/04:01 | 04:11/08:02 |
| Control | 02:01/03:01 | 08:01/27:05 | 07:01/07:02 | 01:01/01:01 |
| Control | 30:01/03:01 | 07:02/13:02 | 06:02/07:02 | 07:01/13:01 |
| Control | 01:01/03:01 | 07:02/39:06 | 07:02/07:161 | 04:01/08:01 |
| Control | 01:01/31:01 | 08:01/40:01 | 07:01/03:04 | 04:04/15:01 |
| Control | 02:01/26:01 | 07:02/14:01 | 07:02/08:02 | 08:01/07:01 |
| Control | 30:01/34:02 | 53:01/45:01 | 04:01/06:02 | 11:01/15:03 |
| Control | 74:01/24:02 | 45:01/52:01 | 16:01/16:01 | 3:02/13:03 |
| Control | 01:01/68:02 | 08:01/14:02 | 07:01/8:02 | 11:03/13:03 |
| Control | 02:01/23:01 | 15:01/35:08 | 04:01/03:04 | 04:01/04:03 |
| Control | 24:02/01:01 | 15:01/15:01 | 03:03/12:02 | 11:03/15:02 |
| Control | 26:01/02:01 | 40:01/45:01 | 06:02/03:04 | 01:01/07:01 |
| Control | 01:01/02:01 | 07:02/44:02 | 07:02/05:01 | 04:04/15:01 |
| Control | 24:02/02:01 | 27:05/44:02 | 2:02/05:01 | 04:01/13:01 |
| Control | 01:01/02:01 | 08:01/44:02 | 05:01/07:01 | 04:01/07:01 |
| Control | 01:01/24:02 | 08:01/44:03 | 07:01/16:01 | 03:01/07:01 |
| Control | 03:01/24:02 | 07:02/39:06 | 07:02/07:02 | 01:01/15:01 |
| Control | 01:01/29:02 | 08:01/44:02 | 05:01/07:01 | 03:01/11:01 |
| Control | 24:02/02:01 | 18:01/07:02 | 07:01/07:02 | 15:01/12:01 |
| Control | 01:01/31:01 | 08:01/51:01 | 07:01/15:02 | 03:01/08:01 |
| Control | 02:01/01:01 | 44:02/57:01 | 06:02/05:01 | 4:01/07:01 |
| Control | 02:01/03:01 | 15:01/35:01 | 04:01/03:04 | 01:01/11:01 |
| Control | 01:01/11:01 | 15:01/39:01 | 07:02/03:03 | 15:01/13:01 |
| Control | 26:01/11:01 | 51:01/44:02 | 05:01/15:02 | 04:01/04:05 |
| Control | 01:01/03:01 | 08:01/35:03 | 07:01/04:01 | 04:04/07:01 |
| Control | 03:01/23:01 | 14:02/49:01 | 8:02/07:01 | 04:05/11:01 |
| Control | 30:02/01:01 | 35:01/35:42 | 04:01/04:01 | 01:03/13:02 |
| Control | 02:01/24:02 | 35:02/44:03 | 04:01/16:01 | 07:01/11:04 |
| Control | 11:01/68:01 | 44:02/44:03 | 16:01/07:04 | 07:01/11:01 |
| Control | 03:01/03:01 | 51:01/51:01 | 07:02/05:01 | 15:01/13:01 |
| Control | 02:01/02:01 | 14:01/14:01 | 07:01/08:02 | 03:01/07:01 |
| Control | NAa | 40:02/40:02 | 02:02/08:02 | 04:04/16:01 |
| Control | 11:01/02:01 | 44:02/44:02 | 05:01/05:01 | 04:01/12:01 |
| Control | 02:01/23:01 | 07:02/44:03 | 04:01/07:02 | 4:01/07:01 |
| Control | 01:01/02:01 | 08:01/41:02 | 07:01/03:04 | 03:01/11:01 |
| Control | 01:01/11:01 | 18:01/57:01 | NAa | 07:01/15:01 |
| Control | 02:01/02:01 | 57:01/18:01 | 07:01/06:02 | 07:01/11:04 |
| Control | 03:01/32:01 | 35:01/44:02 | 05:01/04:33 | 13:01/11:46 |
| Control | NAa | 18:01/51:01 | 12:03/12:03 | 15:01/13:01 |
| Control | 01:01/02:01 | 07:02/55:01 | NAa | 13:01/13:01 |
| Control | 02:01/02:01 | 35:03/27:05 | 04:01/04:01 | 08:01/13:01 |
| Control | 29:02/02:207 | 44:02/44:03 | 16:01/8:02 | 13:01/07:01 |
| Control | 24:02/24:02 | 39:01/39:01 | 12:03/12:03 | 16:01/16:01 |
| Control | 01:01/03:01 | 08:01/07:02 | 07:01/07:02 | 15:01/03:01 |
| Control | 29:02/01:01 | 08:01/44:03 | 07:01/16:01 | 03:01/07:01 |
| Control | 33:01/01:01 | 08:01/14:02 | 07:01/8:02 | 07:01/04:01 |
| Control | 11:01/03:01 | 44:03/44:03 | 2:02/04:42 | 01:01/11:01 |

Table 2 Caption: HLA alleles in TMP-SMX associated respiratory failure cases (7) and controls (63).

Consequently, despite the small sample size this observation is statistically significant both in conservative comparison to maximum reported population frequencies (binomial *P* = 0.00017 for HLA-B and *P* = 0.00028 for HLA-C) or to our control population assessed by the same HLA genotyping approach (binomial *P* = 0.000001 for HLA-B and *P* = 0.000018 for HLA-C) ([Table 2](#T2)). We note the effect should be considered haplotypic, since only three potential instances of common haplotype break between *HLA-B*07:02* and *HLA-C*07:02* were observed: one control sample showing genotype (B*07:02;B*39:06/C*07:02; C*07:02) and two case samples where Athlates called genotypes (B*07:02;B*07:02/C*07:02; C*07:137 and B*07:02;B*07:02/C*07:01; C*07:02).

As a second approach allowing for systematic testing for observed genotypes and standard multiple testing correction, we performed allelic association for HLA alleles with control population frequency of over 5% or at least two patient allele observations. The allelic associations were performed as pseudomarker tests, where each common allele count was compared to all other alleles in the locus between cases and controls, with multiple testing corrected by Bonferroni (total 42 alleles tested). Similar to carrier frequency, results show *HLA-B*07:02* and *HLA-C*07:02* as significantly associated alleles (*P*_nominal_ = 1.6 × 10^−6^, *P*_bonferroni_ = 6.8 × 10^−5^ for B*07:02:01G and *P*_nominal_ = 8.6 × 10^−4^, *P*_bonferroni_ = 0.036 for C*07:02:01G). All results and independently reported earlier allele/carrier frequencies are shown in [Supplementary Data Table S1](#SD1). Haplotype visualization analysis demonstrated overrepresentation of *HLA-B*07:02* and *HLA-C *07:02* in cases as compared to controls and the VUMC BioVu population ([Fig. 1](#F1)).

We also carried out a limited genome-wide search for potential rare coding variation elsewhere in the genome with the rationale that in the limited sample size we could only rule out recurrently mutated genes seen solely in cases. These genome-wide searches did not point out other candidate genes (data available upon request), but with the increased acknowledgment of this rare ADR, we anticipate that the genomic resource may yield other contributory factors in the future.

## DISCUSSION

Cases of TMP-SMX associated severe respiratory failure, requiring prolonged ECMO support and resulting in high mortality, highlight an underrecognized life-threatening reaction to a commonly used antibiotic [[8](#R8), [9](#R9)]. As is the case for most ADRs, a clinical test does not currently exist to identify potentially risk factors associated with severe TMP-SMX respiratory failure and the pathophysiology of the reaction is not understood. Our current findings are the first to identify *HLA-B*07:02* and *HLA-C*07:02* alleles are associated with TMP-SMX severe respiratory failure with 100% carrier rate, where reported maximum carrier rate in U.S. population has been reported at 29% and 31%, respectively. *HLA-B*07:02* and *HLA-C*07:02* are linked genes that commonly occur as part of the 7.1 ancestral haplotype, the second most common haplotype represented in European populations. Carriage of *HLA-B*07:02* and *HLA*07:02* occurs in up to one third of those from European populations and was seen in 19.5% of the Vanderbilt BioVu cohort. The full 7.1 ancestral haplotype includes HLA-A*03:01/HLA-B*07:02/HLA-C*07:02/DRB1*15:01/DQB1*06:02/DQA1*01:01. However, in the TMP-SMX cases which show predominantly, but not solely European ancestry the strongest signal is from *HLA-B*07:02* and *HLA-C*07:02*, however the small sample precludes far reaching conclusions on HLA-B and -C vs. more extended haplotypic association. These findings corroborate the causality between TMP-SMX exposure and respiratory failure in these rare cases and provide some insight to possible mechanism.

HLA alleles have been associated with drug hypersensitivity reactions and HLA genotyping is recommended prior to drug initiation for several drugs to avoid ADRs [[16](#R16)]. For example, prior to initiating abacavir, patients are screened for *HLA-B*57:01* as abacavir is contraindicated in *HLA-B*57:01* allele carriers due to the high risk of a drug hypersensitivity reaction [[17](#R17)]. Similarly the *HLA-B*15:02* allele is associated with an increased risk of severe cutaneous adverse reactions caused by carbamazepine in patients of Asian ancestry and pre-emptive testing is recommended [[18](#R18)].

Specific HLA genotypes have previously been identified as risk factors for different TMP-SMX ADR phenotypes, including for DILI and severe cutaneous reactions [[6](#R6), [7](#R7)]. The HLA associations reported with TMP-SMX ADRs often follow racial or ethnic patterns and are not observed in all patients who develop the ADR [[6](#R6), [7](#R7)]. However, the variant alleles observed in our multi-racial cohort with respiratory failure were present in all cases.

Further investigation is needed to yield clues into the pathogenesis of this severe ADR phenotype. The allelic variants identified in these cases likely is one of potentially many risk factors. HLA class I molecules present peptides that are derived from self or foreign antigens to CD8+ T cells. The representation of *HLA-B*07:02* and *HLA-C*07:02* suggests that they are necessary but not sufficient for antigen-specific CD8+ T-cell immune responses in the setting of TMP-SMX lung injury. However, the lung pathology in these cases shows diffuse alveolar damage with macrophages lining denuded alveolar walls without an obvious T-cell infiltrate. Therefore, it maybe that at the time individuals present clinically, the more acute inflammatory stage has been missed.

Several proposed mechanisms of how a small molecule can induce an immune-mediated ADR exist including the covalent binding of drug to a peptide, the noncovalent binding interaction between a drug-peptide and HLA region, or the drug binds and alters the HLA peptide binding specificity [[19](#R19)]. In silico modeling has demonstrated that both SMX, 4-nitro SMX and TMP can directly bind to *HLA-B*31:01* at unique sites and SMX and *HLA-B*38:02.* Functional interactions between *HLA-B*14:01* antigen binding clefts and reactive SMX metabolites and SMX binding to *HLA-B*35:01* have been suggested [[6](#R6)]. The specific relationship between *HLA-B*07:02* and *HLA-C*07:02* and TMP-SMX has not been evaluated and it is currently unclear if SMX, TMP, or the combination of both agents are contributing to the reaction. It is quite possible that the drug needs to undergo metabolism or covalent modification prior to presentation to the immune system. It is also possible that *HLA-B*07:02/HLA-C*07:02* restricted lung damage could occur as part of a heterologous response whereby a pathogen specific memory T-cell response occurs earlier in life that is also restricted by *HLA-B*07:02* and/or *HLA-C*07:02* that is cross-reactive with the antigen induced by TMP-SMX and peptide much later. Examples of such pathogens are the Herpesvirus family that are chronic persistent DNA viruses to which we have ubiquitous exposure. For instance, both *HLA-B*07:02* and *HLA-C*07:02* are known to be immunodominant alleles for human cytomegalovirus.

A limitation of the current study is the small sample size. Replication of these findings will be important. The cases included in this cohort all had a severe and similar ADR phenotype. It is unclear how these findings would apply to cases with milder reactions. Given the overall rarity of respiratory failure associated with TMP-SMX, and the relatively common allelic frequencies of *HLA-B*07:02* and *HLA-C*07:02* in the population, genotype screening prior to prescribing of TMP-SMX is likely not feasible as a preventive measure. However, identification of allelic variants in possible clinical cases may be instrumental in understanding the immunopathogenesis of this unusual TMP-SMX lung pathology to better guide earlier diagnosis, intervention, and treatment.

## Supplementary Material

## ACKNOWLEDGEMENTS

We would like to thank the patients and families who contributed to this work.

### FUNDING

The research reported in this publication was supported by the National Institute of General Medical Sciences of the National Institutes of Health under award number R01GM129783 (JLG). Imputed HLA control data for [Fig. 1](#F1) was provided under the scope of P50GM115305 (EJP).

### COMPETING INTERESTS

EJP reports grants from National Institutes of Health (P50GM115305, R01HG010863, R01AI152183, R21AI139021, U01AI154659) and from the National Health and Medical Research Council of Australia. She receives Royalties from Uptodate and consulting fees from Janssen, Vertex, Biocryst and Regeneron. She is co-director of IIID Pty Ltd that holds a patent for *HLA-B*57:01* testing for abacavir hypersensitivity, and has a patent pending for Detection of Human Leukocyte Antigen-A*32:01 in connection with Diagnosing Drug Reaction with Eosinophilia and Systemic Symptoms without any financial remuneration and not directly related to the submitted work.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Frey N, Bircher A, Bodmer M, Jick SS, Meier CR, Spoendlin J. Antibiotic drug use and the risk of Stevens-Johnson syndrome and toxic epidermal necrolysis: a population-based case-control study. J Invest Dermatol. 2018;138:1207–9.  [DOI](https://doi.org/10.1016/j.jid.2017.12.015) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29273314/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Invest%20Dermatol&title=Antibiotic%20drug%20use%20and%20the%20risk%20of%20Stevens-Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis:%20a%20population-based%20case-control%20study&author=N%20Frey&author=A%20Bircher&author=M%20Bodmer&author=SS%20Jick&author=CR%20Meier&volume=138&publication_year=2018&pages=1207-9&pmid=29273314&doi=10.1016/j.jid.2017.12.015&)

2. Lin YF, Yang CH, Sindy H, Lin JY, Rosaline Hui CY, Tsai YC, et al. Severe cutaneous adverse reactions related to systemic antibiotics. Clin Infect Dis. 2014;58:1377–85.  [DOI](https://doi.org/10.1093/cid/ciu126) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24599767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Infect%20Dis&title=Severe%20cutaneous%20adverse%20reactions%20related%20to%20systemic%20antibiotics&author=YF%20Lin&author=CH%20Yang&author=H%20Sindy&author=JY%20Lin&author=CY%20Rosaline%20Hui&volume=58&publication_year=2014&pages=1377-85&pmid=24599767&doi=10.1093/cid/ciu126&)

3. Bjornsson E, Olsson R. Outcome and prognostic markers in severe drug-induced liver disease. Hepatology. 2005;42:481–9.  [DOI](https://doi.org/10.1002/hep.20800) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16025496/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Outcome%20and%20prognostic%20markers%20in%20severe%20drug-induced%20liver%20disease&author=E%20Bjornsson&author=R%20Olsson&volume=42&publication_year=2005&pages=481-9&pmid=16025496&doi=10.1002/hep.20800&)

4. Keisu M, Wiholm BE, Palmblad J. Trimethoprim-sulphamethoxazole-associated blood dyscrasias. Ten years’ experience of the Swedish spontaneous reporting system. J Intern Med. 1990;228:353–60.  [DOI](https://doi.org/10.1111/j.1365-2796.1990.tb00245.x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/2266345/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Intern%20Med&title=Trimethoprim-sulphamethoxazole-associated%20blood%20dyscrasias.%20Ten%20years%E2%80%99%20experience%20of%20the%20Swedish%20spontaneous%20reporting%20system&author=M%20Keisu&author=BE%20Wiholm&author=J%20Palmblad&volume=228&publication_year=1990&pages=353-60&pmid=2266345&doi=10.1111/j.1365-2796.1990.tb00245.x&)

5. Pirmohamed M, Friedmann PS, Molokhia M, Loke YK, Smith C, Phillips E, et al. Phenotype standardization for immune-mediated drug-induced skin injury. Clin Pharm Ther. 2011;89:896–901.  [DOI](https://doi.org/10.1038/clpt.2011.79) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21562486/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Ther&title=Phenotype%20standardization%20for%20immune-mediated%20drug-induced%20skin%20injury&author=M%20Pirmohamed&author=PS%20Friedmann&author=M%20Molokhia&author=YK%20Loke&author=C%20Smith&volume=89&publication_year=2011&pages=896-901&pmid=21562486&doi=10.1038/clpt.2011.79&)

6. Li YJ, Phillips E, Dellinger A, Nicoletti P, Schutte R, Li D, et al. HLA-B*14:01 and HLA-B*35:01 are associated with trimethoprim-sulfamethoxazole induced liver injury. Hepatology. 2021;73:268–81.  [DOI](https://doi.org/10.1002/hep.31258) | [PMC free article](/articles/PMC7544638/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32270503/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=HLA-B*14:01%20and%20HLA-B*35:01%20are%20associated%20with%20trimethoprim-sulfamethoxazole%20induced%20liver%20injury&author=YJ%20Li&author=E%20Phillips&author=A%20Dellinger&author=P%20Nicoletti&author=R%20Schutte&volume=73&publication_year=2021&pages=268-81&pmid=32270503&doi=10.1002/hep.31258&)

7. Wang CW, Tassaneeyakul W, Chen CB, Chen WT, Teng YC, Huang CY, et al. Whole genome sequencing identifies genetic variants associated with co-trimoxazole hypersensitivity in Asians. J Allergy Clin Immunol. 2021;147:1402–12.  [DOI](https://doi.org/10.1016/j.jaci.2020.08.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32791162/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=Whole%20genome%20sequencing%20identifies%20genetic%20variants%20associated%20with%20co-trimoxazole%20hypersensitivity%20in%20Asians&author=CW%20Wang&author=W%20Tassaneeyakul&author=CB%20Chen&author=WT%20Chen&author=YC%20Teng&volume=147&publication_year=2021&pages=1402-12&pmid=32791162&doi=10.1016/j.jaci.2020.08.003&)

8. Miller JO, Taylor J, Goldman JL. Severe acute respiratory failure in healthy adolescents exposed to trimethoprim-sulfamethoxazole. Pediatrics. 2019;143:e20183242.  [DOI](https://doi.org/10.1542/peds.2018-3242) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31142578/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatrics&title=Severe%20acute%20respiratory%20failure%20in%20healthy%20adolescents%20exposed%20to%20trimethoprim-sulfamethoxazole&author=JO%20Miller&author=J%20Taylor&author=JL%20Goldman&volume=143&publication_year=2019&pages=e20183242&pmid=31142578&doi=10.1542/peds.2018-3242&)

9. Miller JO, Shih AR, Mino-Kenudson M, Taylor MS, Goldman JL. Trimethoprim-sulfamethoxazole-associated fulminant respiratory failure in children and young adults. Am J respiratory Crit care Med. 2021;203:918–21.  [DOI](https://doi.org/10.1164/rccm.202009-3421LE) | [PMC free article](/articles/PMC8017575/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33513317/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20respiratory%20Crit%20care%20Med&title=Trimethoprim-sulfamethoxazole-associated%20fulminant%20respiratory%20failure%20in%20children%20and%20young%20adults&author=JO%20Miller&author=AR%20Shih&author=M%20Mino-Kenudson&author=MS%20Taylor&author=JL%20Goldman&volume=203&publication_year=2021&pages=918-21&pmid=33513317&doi=10.1164/rccm.202009-3421LE&)

10. Naranjo CA, Busto U, Sellers EM, Sandor P, Ruiz I, Roberts EA, et al. A method for estimating the probability of adverse drug reactions. Clin Pharm Ther. 1981;30:239–45.  [DOI](https://doi.org/10.1038/clpt.1981.154) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7249508/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Ther&title=A%20method%20for%20estimating%20the%20probability%20of%20adverse%20drug%20reactions&author=CA%20Naranjo&author=U%20Busto&author=EM%20Sellers&author=P%20Sandor&author=I%20Ruiz&volume=30&publication_year=1981&pages=239-45&pmid=7249508&doi=10.1038/clpt.1981.154&)

11. Gonzalez-Galarza FF, McCabe A, Santos E, Jones J, Takeshita L, Ortega-Rivera ND, et al. Allele frequency net database (AFND) 2020 update: gold-standard data classification, open access genotype data and new query tools. Nucleic Acids Res. 2020;48:D783–8.  [DOI](https://doi.org/10.1093/nar/gkz1029) | [PMC free article](/articles/PMC7145554/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31722398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Allele%20frequency%20net%20database%20(AFND)%202020%20update:%20gold-standard%20data%20classification,%20open%20access%20genotype%20data%20and%20new%20query%20tools&author=FF%20Gonzalez-Galarza&author=A%20McCabe&author=E%20Santos&author=J%20Jones&author=L%20Takeshita&volume=48&publication_year=2020&pages=D783-8&pmid=31722398&doi=10.1093/nar/gkz1029&)

12. Roden DM, Pulley JM, Basford MA, Bernard GR, Clayton EW, Balser JR, et al. Development of a large-scale de-identified DNA biobank to enable personalized medicine. Clin Pharm Ther. 2008;84:362–9.  [DOI](https://doi.org/10.1038/clpt.2008.89) | [PMC free article](/articles/PMC3763939/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18500243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharm%20Ther&title=Development%20of%20a%20large-scale%20de-identified%20DNA%20biobank%20to%20enable%20personalized%20medicine&author=DM%20Roden&author=JM%20Pulley&author=MA%20Basford&author=GR%20Bernard&author=EW%20Clayton&volume=84&publication_year=2008&pages=362-9&pmid=18500243&doi=10.1038/clpt.2008.89&)

13. Larjo A, Eveleigh R, Kilpelainen E, Kwan T, Pastinen T, Koskela S, et al. Accuracy of programs for the determination of human leukocyte antigen alleles from next-generation sequencing data. Front Immunol. 2017;8:1815.  [DOI](https://doi.org/10.3389/fimmu.2017.01815) | [PMC free article](/articles/PMC5733459/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29326702/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&title=Accuracy%20of%20programs%20for%20the%20determination%20of%20human%20leukocyte%20antigen%20alleles%20from%20next-generation%20sequencing%20data&author=A%20Larjo&author=R%20Eveleigh&author=E%20Kilpelainen&author=T%20Kwan&author=T%20Pastinen&volume=8&publication_year=2017&pages=1815&pmid=29326702&doi=10.3389/fimmu.2017.01815&)

14. Daly AK. Pharmacogenomics of adverse drug reactions. Genome Med. 2013;5:5.  [DOI](https://doi.org/10.1186/gm409) | [PMC free article](/articles/PMC3707028/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23360680/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med&title=Pharmacogenomics%20of%20adverse%20drug%20reactions&author=AK%20Daly&volume=5&publication_year=2013&pages=5&pmid=23360680&doi=10.1186/gm409&)

15. Gonzalez-Galarza FF, McCabe A, Melo Dos Santos EJ, Jones AR, Middleton D. A snapshot of human leukocyte antigen (HLA) diversity using data from the Allele Frequency Net Database. Hum Immunol. 2020;82:496–504.  [DOI](https://doi.org/10.1016/j.humimm.2020.10.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33755549/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Immunol&title=A%20snapshot%20of%20human%20leukocyte%20antigen%20(HLA)%20diversity%20using%20data%20from%20the%20Allele%20Frequency%20Net%20Database&author=FF%20Gonzalez-Galarza&author=A%20McCabe&author=EJ%20Melo%20Dos%20Santos&author=AR%20Jones&author=D%20Middleton&volume=82&publication_year=2020&pages=496-504&pmid=33755549&doi=10.1016/j.humimm.2020.10.004&)

16. Alfirevic A, Gonzalez-Galarza F, Bell C, Martinsson K, Platt V, Bretland G, et al. In silico analysis of HLA associations with drug-induced liver injury: use of a HLA-genotyped DNA archive from healthy volunteers. Genome Med. 2012;4:51.  [DOI](https://doi.org/10.1186/gm350) | [PMC free article](/articles/PMC3698530/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22732016/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Med&title=In%20silico%20analysis%20of%20HLA%20associations%20with%20drug-induced%20liver%20injury:%20use%20of%20a%20HLA-genotyped%20DNA%20archive%20from%20healthy%20volunteers&author=A%20Alfirevic&author=F%20Gonzalez-Galarza&author=C%20Bell&author=K%20Martinsson&author=V%20Platt&volume=4&publication_year=2012&pages=51&pmid=22732016&doi=10.1186/gm350&)

17. Abacavir sulfate drug insert. 2021. https://www.accessdata.fda.gov/drugsatfda_docs/label/2012/020977s024,020978s028lbl.pdf.  [https://www.accessdata.fda.gov/drugsatfda_docs/label/2012/020977s024,020978s028lbl.pdf](https://www.accessdata.fda.gov/drugsatfda_docs/label/2012/020977s024,020978s028lbl.pdf)

18. Carbamazepine drug insert. 2021. https://www.accessdata.fda.gov/drugsatfda_docs/label/2009/016608s101,018281s048lbl.pdf.  [https://www.accessdata.fda.gov/drugsatfda_docs/label/2009/016608s101,018281s048lbl.pdf](https://www.accessdata.fda.gov/drugsatfda_docs/label/2009/016608s101,018281s048lbl.pdf)

19. Karnes JH, Miller MA, White KD, Konvinse KC, Pavlos RK, Redwood AJ, et al. Applications of Immunopharmacogenomics: predicting, preventing, and understanding immune-mediated adverse drug reactions. Annu Rev Pharm Toxicol. 2019;59:463–86.  [DOI](https://doi.org/10.1146/annurev-pharmtox-010818-021818) | [PMC free article](/articles/PMC6409210/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30134124/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharm%20Toxicol&title=Applications%20of%20Immunopharmacogenomics:%20predicting,%20preventing,%20and%20understanding%20immune-mediated%20adverse%20drug%20reactions&author=JH%20Karnes&author=MA%20Miller&author=KD%20White&author=KC%20Konvinse&author=RK%20Pavlos&volume=59&publication_year=2019&pages=463-86&pmid=30134124&doi=10.1146/annurev-pharmtox-010818-021818&)
