# Identification of reduced ERAP2 expression and a novel HLA allele as components of a risk score for susceptibility to liver injury due to amoxicillin-clavulanate

## Metadata
**Authors:** Paola Nicoletti, Andrew Dellinger, Yi Ju Li, Huiman X Barnhart, Naga Chalasani, Robert J Fontana, Joseph A Odin, Jose Serrano, Andrew Stolz, Amy S Etheridge, Federico Innocenti, Olivier Govaere, Jane I Grove, Camilla Stephens, Guruprasad P Aithal, Raul J Andrade, Einar S Bjornsson, Ann K Daly, M Isabel Lucena, Paul B Watkins
**Journal:** Gastroenterology
**Date:** 2022 Dec 7
**DOI:** [10.1053/j.gastro.2022.11.036](https://doi.org/10.1053/j.gastro.2022.11.036)
**PMID:** 36496055
**PMCID:** PMC9974860
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9974860/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9974860/pdf/nihms-1855942.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9974860/pdf/nihms-1855942.pdf)

## Abstract

**BACKGROUND:** 
Drug-induced liver injury (DILI) due to amoxicillin–clavulanate (AC) has been associated with HLA-A*02:01, HLA-DRB1*15:01 and rs2476601, a missense variant in PTPN22. The aim of this study was to identify novel risk factors for AC-DILI and to construct a genetic risk score (GRS).

**METHODS:** 
Transcriptome-wide association (TWAS) and genome-wide association (GWAS) analyses were performed on 444 AC-DILI cases and 10,397 population-based controls of European descent. Associations were confirmed in a validation cohort (n=133 cases and 17,836 population-based controls). Discovery and validation AC-DILI cases were also compared to 1358 and 403 non-AC-DILI cases.

**RESULTS:** 
TWAS revealed a significant association of AC-DILI risk with reduced liver expression of ERAP2 (P = 3.7×10−7), coding for an aminopeptidase involved in antigen presentation. The lead eQTL SNP, rs1363907 (G), was associated with AC-DILI risk in the discovery (OR[95%CI] = 1.68 [1.23–1.66] P = 1.7×10−7) and validation cohorts (OR[95%CI] = 1.2 [1.04–2.05] P = 0.03), following a recessive model. We also identified HLA-B*15:18 as novel AC-DILI risk factor in both discovery (OR[95%CI] = 4.19 [2.09–8.36)] P = 4.9×10−5) and validation cohorts (OR[95%CI] = 7.78 [2.75–21.99] P = 0.0001). GRS, incorporating rs1363907, rs2476601, HLA-B*15:18, HLA-A*02:01, HLA-DRB1*15:01, was highly predictive of AC-DILI risk when cases were analyzed against both general population and non-AC-DILI control cohorts. GRS was the most significant predictor in a regression model containing known AC-DILI risk clinical characteristics and significantly improved the predictive model.

**CONCLUSIONS:** 
We identified novel associations of AC-DILI risk with ERAP2 low expression and with HLA-B*15:18. GRS based on the five risk variants may assist AC-DILI causality assessment and risk management.

Keywords: Amoxicillin-clavulanate, DILI, ERAP2, HLA-B*15:18, GWAS

### BACKGROUND:

Drug-induced liver injury (DILI) due to amoxicillin–clavulanate (AC) has been associated with *HLA-A*02:01*, *HLA-DRB1*15:01* and rs2476601, a missense variant in *PTPN22*. The aim of this study was to identify novel risk factors for AC-DILI and to construct a genetic risk score (GRS).

### METHODS:

Transcriptome-wide association (TWAS) and genome-wide association (GWAS) analyses were performed on 444 AC-DILI cases and 10,397 population-based controls of European descent. Associations were confirmed in a validation cohort (n=133 cases and 17,836 population-based controls). Discovery and validation AC-DILI cases were also compared to 1358 and 403 non-AC-DILI cases.

### RESULTS:

TWAS revealed a significant association of AC-DILI risk with reduced liver expression of *ERAP2* (P = 3.7×10^−7^), coding for an aminopeptidase involved in antigen presentation. The lead eQTL SNP, rs1363907 (G), was associated with AC-DILI risk in the discovery (OR[95%CI] = 1.68 [1.23–1.66] P = 1.7×10^−7^) and validation cohorts (OR[95%CI] = 1.2 [1.04–2.05] P = 0.03), following a recessive model. We also identified *HLA-B*15:18* as novel AC-DILI risk factor in both discovery (OR[95%CI] = 4.19 [2.09–8.36)] P = 4.9×10^−5^) and validation cohorts (OR[95%CI] = 7.78 [2.75–21.99] P = 0.0001). GRS, incorporating rs1363907, rs2476601*, HLA-B*15:18, HLA-A*02:01*, *HLA-DRB1*15:01*, was highly predictive of AC-DILI risk when cases were analyzed against both general population and non-AC-DILI control cohorts. GRS was the most significant predictor in a regression model containing known AC-DILI risk clinical characteristics and significantly improved the predictive model.

### CONCLUSIONS:

We identified novel associations of AC-DILI risk with *ERAP2* low expression and with *HLA-B*15:18*. GRS based on the five risk variants may assist AC-DILI causality assessment and risk management.

## Introduction

Idiosyncratic drug-induced liver injury (DILI) is a rare adverse drug reaction that is an important cause of morbidity and acute liver failure^1, 2^. Diagnosing DILI and identifying which drug is responsible in the patient receiving multiple medications is often challenging^3^. Amoxicillin-clavulanate (AC) is a very common cause of idiosyncratic DILI^1, 4^. Many patients with AC-DILI present 1 to 4 weeks after discontinuation of AC making it difficult to recognize AC as a causative drug^5^. While most cases resolve with discontinuation of AC treatment, full recovery may require many months and liver failures have been reported^4^. Furthermore, invasive medical testing including liver biopsies and endoscopic retrograde cholangiopancreatography (ERCP) are frequently undertaken in older individuals presenting with jaundice and weight loss that is eventually attributed to AC-DILI^4^.

Prior GWAS studies identified *HLA-A*02:01* and *HLA-DRB1*15:01* as genetic risk factors for AC-DILI among individuals of European descent^6–8^. A subsequent GWAS study identified a variant (rs2476601) of *PTPN22* to be a risk factor for all cause DILI, with AC-DILI as a major contributor to this association^8^. *PTPN22* is believed to be involved in regulating T-cell receptor signaling and the variant associated with DILI has also been associated with risk for certain autoimmune diseases^9^, further supporting the involvement of the immune system in AC-DILI.

The aim of the current study was to identify novel genetic risk variants for AC-DILI and to develop a sensitive and specific multi-locus genetic risk score that could help in causality assessment and ultimately risk management. We applied both transcriptome-wide association (TWAS) and genome-wide association (GWAS) analysis approaches on the largest AC-DILI cohort published up to date. We provide additional support for the role of adaptive immunity in AC-DILI by discovering and validating novel associations between AC-DILI risk and the expression of *ERAP2*, a gene involved in antigen processing^10^, as well as with a new HLA class I allele. The predictive risk associated with the combination of the five identified risk factors was also evaluated, and specificity assessed by comparing to DILI risk due to other drugs.

## Materials and Methods

### Study cohorts

#### European AC-DILI Discovery cohort

444 AC-DILI cases collected by DILIN, the Spanish DILI network, DILIGEN, Eudragene, and iDILIC consortia ([Table S1](#SD1)) were compared to 10,397 European population control samples as included in Cirulli et al.^8^ As AC-DILI has 45 in 100,000 prevalence in general population^11^, we used a large set of general population controls.

#### European AC-DILI Validation cohort

133 AC-DILI cases ([Table S2](#SD1)) were newly recruited for this study by DILIN, iDILIC, Prospective European DILI cohort study (ProEuro-DILI) and the Spanish DILI network. The eligibility criteria and causality assessment were performed as previously described.^12, 13, 8, 14^ All participants provided written informed consent, and each study was approved by the appropriate national or institutional ethical review boards. Validation AC-DILI cases were compared to 17,836 European population controls from NCBI dbGaP cohorts, as inferred by principal component analysis (PCA) ([Figure S1](#SD1)). Details on the dbGaP cohorts are reported in [Tables S2](#SD1) and their age and sex information in [Table S3](#SD1).

#### European non-AC-DILI discovery and validation cohorts

We gathered two large independent European cohorts of non-AC-DILI individuals to use as controls to compare respectively to the discovery and validation AC-DILI cases. 1,358 non-AC-DILI individuals from Cirulli et al.^8^ served as controls in the discovery cohort and 403 newly genotyped non-AC-DILI individuals recruited by DILIN served as controls in the validation cohort. Clinical information is reported in [Table S3](#SD1).

#### Hispanic and African American cohorts

13 AC-DILI cases of African American (AA) and 7 of Hispanic descent were recruited from DILIN. Based on PCA cases were compared with 2919 Hispanic and 5816 AA ancestry-matched controls from PAGE BioME (phs000925.v1.p1).

### Genome-wide genotyping.

Genotype data was available for all controls, AC-DILI European discovery cases, 1358 non-AC DILI cases, 10 Hispanic and AA DILIN AC-DILI cases and 9 iDILIC validation cases. ^6, 8, 14^ The remaining 124 European validation cases, 8 Hispanic and AA DILIN cases and 403 European non-AC-DILI cases were genotyped using the MEGAX Illumina platform at Vanderbilt University ([Table S2](#SD1)).

### Transcriptome-wide analysis (TWAS).

PrediXcan^15^ was used to predict liver gene expression in cases and controls based on GTEX V8 liver gene expression dataset using the default model and parameters. Association testing was performed for each gene using logistic regression correcting for population stratification in R. Based on number genes expressed in the liver GTEX V8, transcriptome-wide statistical significance threshold was estimated as 1.1×10^−5^, using Bonferroni correction.

### Genetic analysis.

Quality control (QC) checks on the genotype data were performed as described.^8, 14^ Genetic ancestry was inferred by principal components (PCs) computed from EIGENSTRAT as described in Li et al^16^. SNP imputation was performed using Michigan Imputation Server^17^ (see [Supplementary Materials](#SD1)). Four-digit *HLA* alleles were inferred using HIBAG^18^ using genotyped data in the MHC region. For both *HLA* alleles and SNPs, association analyses were performed using logistic regression under the additive genetic model with covariate adjustment for PCs (ten PCs in the discovery and four PCs in the validation cohort) in PLINK^19^. When indicated, age and sex were included in the model as covariates. Due to the limited size, we applied Fisher’s Exact test for association analysis between AA and Hispanic cases and controls. We set the GWAS and MHC significance *p*-value threshold to 5.0×10^−8^ and 4.0×10^−4^ (0.05/115 total predicted HLA alleles) to correct for multiple testing. We used conditional analysis to determine which variants are independent within the susceptibility locus by including the most associated variant/s as covariate/s. As published,^8^ to assess the joint effect of risk variants discovered to date, multi-marker logistic regression analysis was conducted with adjustment of PCs, where the joint negative carriers were used as the reference group. *ERAP1* haplotypes were reconstructed and tested in Plink using the set of variants and the nomenclature reported by Ombrello and collegues^20^. Meta-analysis was performed using a fixed-effect model in the *metafor* R package. Manhattan plots were performed with R (Version 3.0.2). Regional plots were drawn by LocusZoom^21^. We investigated the relevance of the significant genetic associations with AC-DILI clinical phenotypes (see [Supplementary Materials](#SD1)).

Pairwise SNP-SNP interaction association among the top associated signals was assessed using logistic regression to model the significant risk markers (expressed as dichotomic variables based on carriage status), and their interaction term with adjustment of PCs and Bonferroni correction was applied to correct for multiple comparisons, as proposed^22^. SNP interaction analysis, multivariable, multi-marker logistic regression models including the relevant genetic risk factors and Likelihood-ratio test were carried out using STATA15.

### Genetic Risk Score (GRS).

We used the five variants associated with AC-DILI risk to construct a Genetic Risk Score (GRS). We used the discovery dataset to estimate the variant weights, which were the beta-coefficients from the multivariable logistic regression model on these five variants with adjustment of population stratification. The GRS was then formulated as the sum of the product of the estimated beta-coefficient and the number of risk alleles at each locus as shown in the equation below:

| GRS=(0.77×allele dosage of HLA−A*0201)+(1.16×allele dosage of HLA−DRB1*15:01)+(1.37×allele dosage of HLA−B*15:18)+(0.47×ERAP2 rs1363907 GG genotype+(0.48×allele dosage of PTPN22 rs2476601 A) |
| --- |
Examples of how to calculate the GRS are reported in [Table S4](#SD1). The GRS values were divided into ten equal quantiles and the AC-DILI relative risk of each quantile was calculated to estimate the absolute risk of developing DILI after a treatment course with AC, assuming the overall incidence is 1:2,350 AC treated patients.^11^ We tested the GRS associations with AC-DILI in the validation dataset against both population controls and both non-AC-DILI cohorts. The Youden’s index^23^ was used to determine the optimal diagnostic threshold. The predictive power of the model was evaluated by area under receiver operator characteristic (ROC) curve (AUC) using STATA15

### HLA genotyping.

Genotypes of the most significant HLA alleles were confirmed using high resolution genotyping on the carriers by Histogenetics (Ossining, New York) for the iDILIC samples^14^, and at Vanderbilt University using Illumina MiSeq for the DILIN cases^16^.

## Results

### Genome-wide association study identifies a novel HLA risk allele for AC-DILI

GWAS was performed on 7,747,658 well-imputed or genotyped SNPs comparing 444 European ancestry cases and 10,397 matched population controls ([Figure S1A](#SD1)). Clinical characteristics of the AC-DILI cases are reported in [Table 1](#T1). Only variants in the MHC region were found to have genome-wide significant p-values, led by rs9268927 (A, OR = 3.13 95%CI [2.68–3.65] P=6.3×10^−47^ which is a proxy of *HLA-DRB1*15:01* (r^2^ = 0.96) and rs2734966 (G, OR = 2.11 95%CI [1.83–2.42] P = 4.2×10^−26^ which is a proxy of *HLA-A*02:01* (r^2^ = 0.96). ([Figure S2](#SD1) and [Table S5](#SD1)). Both are previously reported risk factors for AC-DILI^6^. Subsequent genome-wide analysis conditional on the genotypes of *HLA-DRB1*15:01* and *HLA-A*02:01* did not reveal additional genome-wide significant marker associations ([Figure S3](#SD1)). For imputed HLA genotypes, *HLA-DRB1*15:01* and *HLA-A*02:01* together with their associated known haplotype alleles *HLA-DQB1*06:02* and *HLA-B*07:02,* respectively were the most significant HLA alleles as expected. Conditional analysis on *HLA-DRB1*15:01* and *HLA-A*02:01* identified a novel and independent association with *HLA-B*15:18* which was MHC-significant (OR = 4.19, 95%CI [2.09–8.36)], P = 4.9×10^−5^, [Table S5](#SD1)). Among the AC-DILI cases, ten cases (2%) carried this rare *HLA-B* allele when just three were expected based on reported European population frequency. All nine *HLA-B*15:18* carriers, with DNA available for confirmatory HLA typing had 100% concordance between the predicted and typed *HLA-B*15:18* allele. Finally, the previously reported association^8^ between DILI from multiple drugs and *PTPN22* rs2476601 missense variant showed a similar independent association trend (OR = 1.43 95%CI [1.23–1.65] P = 2.2 ×10^−6^).

### Table 1:

| CHARACTERISTICS | European discovery | European validation | African American | Hispanic |
| --- | --- | --- | --- | --- |
|   |  |  |  |  |
| Clinical information |   |   |   |   |
|   |  |  |  |  |
| Number of patients | 444 | 133 | 13 | 7 |
| Mean age, years | 59 | 64 | 60 | 51 |
| Female, % | 44 | 46 | 69 | 14 |
| Mean alanine aminotransferase (SD), U/L | 485 (618) | 465 (425) | 705 (804) | 337 (217) |
| Mean alkaline phosphatase (SD), U/L | 445 (330) | 459 (395) | 593 (365) | 350 (151) |
| Mean latency (SD), days | 22 (19.7) | 27 (19.3) | 34 (23.6) | 28 (18.6) |
| Injury Phenotype |   |   |   |   |
|   |  |  |  |  |
| Cholestatic –R value < 2.0 (%) | 162 (38%) | 44 (33%) | 7 (54%) | 2 (29%) |
| Hepatocellular - R value > 5.0 (%) | 101 (22%) | 40 (30%) | 3 (23%) | 0 (0%) |
| Mixed - R value between 2 and 5 (%) | 158 (35%) | 46 (35%) | 3 (23%) | 5 (71%) |
| Not available (%) | 23 (5%) | 3 (2%) | 0 (0%) | 0 (0%) |

Table 1 Caption: Clinical characteristics of subjects in the four AC-DILI case cohorts

We also analyzed the most associated GWAS markers reported in [Table 2](#T2) in an independent validation cohort of 133 AC-DILI cases that we compared with a new cohort of 17,836 European population-based controls ([Figure S1B](#SD1)). This analysis confirmed the role of *HLA-B*15:18* as an AC-DILI risk factor (OR = 5.24[1.85–14.81] P= 0.002, [Table 2](#T2)) with 2% of the AC-DILI validation cases carrying this allele. All carriers were confirmed by direct HLA typing. The association was still significant when correcting for age and sex (OR = 7.31 [2.24–23.83] P = 0.0009) or when correcting for *HLA-DRB1*15:01* and *HLA-A*02:01* (OR = 7.78 [2.75–21.99] P = 0.0001). The association with rs2476601 (PTPN22) was not confirmed in this smaller cohort ([Table 2](#T2)).

### Table 2.

|   |   | Discovery cohort (444 vs 10397) |  |  |  | Validation cohort (133 cases 17836) |  |  |  | Meta-analysis |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |  |  |  |  |
| Variant of interest | GENE | OR (95%CI) | P | AF CA | AF CO | OR (95%CI) | P | AF CA | AF CO | OR (95%CI) | P |
|   |  |  |  |  |  |  |  |  |  |  |  |
| HLA alleles |   |   |   |   |   |   |   |   |   |   |   |
|   |  |  |  |  |  |  |  |  |  |  |  |
| HLA-DRB1*15:01 | HLA | 3.14 (3.66–2.69) | 1.4×10−47 | 0.30 | 0.12 | 2.46 (1.87–3.22) | 1.1×10−10 | 0.27 | 0.13 | 2.96 (2.59–3.39) | 3.8×10−56 |
| HLA-A*02:01 | HLA | 2.11 (2.42–1.84) | 4.5×10−26 | 0.44 | 0.28 | 1.97 (1.54–2.52) | 7.1×10−8 | 0.42 | 0.27 | 2.08 (1.84–2.34) | 2.2×10−32 |
| HLA-B*15:18 | HLA | 3.1 (6.08–1.58) | 0.001 | 0.01 | 0.004 | 5.24 (1.85–4.81) | 0.002 | 0.02 | 0.002 | 3.61 (2.05–6.37) | 0.000009 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| rs1363907 | ERAP2 | 1.68 (1.23–1.66) | 1.7×10−7 | 0.67 | 0.58 | 1.2 (1.04–2.05) | 0.03 | 0.66 | 0.6 | 1.38 (1.22–1.57) | 5.0×10−7 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| rs2476601 | PTPN22 | 1.62 (1.32–1.98) | 4.0×10−6 | 0.13 | 0.08 | 1.01 (0.66–1.54) | 0.95 | 0.09 | 0.09 | 1.4 (1.18–1.67) | 0.0001 |
|   |  |  |  |  |  |  |  |  |  |  |  |
| ERAP1 haplotypes |   |   |   |   |   |   |   |   |   |   |   |
|   |  |  |  |  |  |  |  |  |  |  |  |
| Haplotype 2 (Hap2) | ERAP1 | 1.41 (1.20– 1.71) | 4.9×10−5 | 0.19 | 0.14 | 1.42 (1.02–1.94) | 0.01 | 0.19 | 0.14 | 1.41 (1.23–1.63) | 2.0×10−6 |
| Haplotype 8 (Hap8) | ERAP1 | 0.78 (0.62–0.88) | 0.005 | 0.19 | 0.24 | 0.95 (0.70–1.28) | 0.63 | 0.22 | 0.23 | 0.84 (0.74–0.97) | 0.014 |
| Haplotype 10 (Hap10) | ERAP1 | 0.87 (0.73–1.05) | 0.01 | 0.18 | 0.20 | 0.88 (0.62–1.21) | 0.22 | 0.18 | 0.2 | 0.87 (0.79–0.96) | 0.004 |

Table 2 Caption: Summary statistics for the single marker analysis of the most relevant associated variants

### ERAP2 expression is associated with AC-DILI risk

We next imputed genome-wide gene expression profiles of each individual in our discovery cohort using the PrediXscan algorithm,^15^ with the liver specific eQTL genetic profile GTEX V8 as the reference panel. We then performed a Transcriptome Wide Association Study (TWAS) comparing the predicted gene expression between cases and controls. TWAS analysis revealed that several genes within the MHC region were differentially expressed, which mirrored the long LD block associated with the known HLA risk alleles ([Figure S4](#SD1)), as demonstrated by the abolition of the PrediXcan MHC signals upon conditioning on *HLA-DRB1*15:01* and *HLA-A*02:01* ([Figure 1](#F1)). Outside the MHC region, only the predicted expression of one gene, *ERAP2,* was significantly different in controls compared with cases (P = 3.7×10^−7^, [Figure S4](#SD1)). This significant difference in predicted *ERAP2* gene expression was independent of the effect of the HLA risk alleles as tested in the conditional analysis (P_conditional_ = 4.5 ×10^−6^, [Figure 1](#F1)).

### Figure 1.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/454a/9974860/f5b13dba94db/nihms-1855942-f0002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9974860_nihms-1855942-f0002.jpg)

Manhattan plot displaying difference between discovery AC-DILI cases and controls in predicted liver mRNA expression (TWAS) conditioned on the presence of known HLA risk alleles. Only ERAP2 mRNA showed a significant difference (p <1.1×10−5).

We identified rs1363907 (G) as the single SNP with the strongest association with AC-DILI in GWAS analysis in the *ERAP2* region, driving the association with *ERAP2* expression. rs1363907 (G) was associated with increased risk of AC-DILI ([Table 2](#T2)). rs1363907 (G) is part of a long LD region encompassing *ERAP2* and near LNPEP genes. The long LD region includes both intronic and exonic variants ([Figure S5](#SD1)). The genomic region in *ERAP2* has two major haplotypes called Hap A and Hap B. Evidence suggests that Hap B codes for a truncated ERAP2 mRNA that is rapidly degraded mainly although not uniquely due to a splicing variant (rs2248374, G) located in the ERAP2 LD region^24^. rs1363907 is in linkage with rs2248374 and other suggested causal variants responsible for the mRNA truncation (r^2^ => 0.80, [Table S6](#SD1)). Therefore, rs1363907 (G) was considered a proxy for Hap B.

We confirmed the association of rs1363907 (G) and ERAP2 expression in an independent liver eQTL dataset derived from 1,183 liver samples^25^. Using this large eQTL liver-specific dataset, we identified that only individuals carrying two copies of the rs1363907 (G) risk allele showed a consistently lower liver expression of *ERAP2* mRNA compared to the other genotype groups ([Figure S5](#SD1)), suggesting a recessive model ([Figure S6](#SD1)). Consistent with this observation, the increased risk of AC-DILI was associated only with homozygous status of rs1363907 (G) in the multivariable analysis ([Table S7](#SD1)). The recessive model was also a better model when all the SNPs in the LD region were considered ([Figure S7](#SD1)). rs1363907 (G) was not correlated with the expression of any HLA loci in GTEX V8 dataset ([Figure S8](#SD1)). The association between homozygous status of rs1363907 (G) and risk of AC-DILI was confirmed in the European validation cohort (OR = 1.43, 95%CI [1.01–2.03] P = 0.04, [Table 3](#T3)) with an overall significance of 4.3×10^−7^ in the meta-analysis. In this cohort, the rs1363907 (G) association was still significant when corrected by sex and age (OR = 1.40, 95%CI [1.04–1.88] P = 0.03). Moreover, allele frequency rs1363907 was similar between males and females in the GnomAD database (not shown).

### Table 3.

|   |   | Discovery cohort |  | Validation cohort |  | Meta-analysis |  |
| --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |
| Marker | Genotype | OR (95%CI) | P | OR (95%CI) | P | OR (95%CI) | P |
|   |  |  |  |  |  |  |  |
| HLA-A*02:01 | HZ | 2.23 (1.78–2.79) | 1.6×10−12 | 2.55 (1.73–3.77) | 2.6×10−6 | 2.31 (1.9–2.8) | 2.5×10−17 |
|   | HH | 4.59 (3.42–6.16) | 2.3×10−24 | 3.62 (2.09–6.26) | 4.3×10−6 | 4.35 (3.36–5.64) | 7.2×10−29 |
|   |  |  |  |  |  |  |  |
| HLA-DRB1*15:01 | HZ | 3.26 (2.66–4.04) | 6.8×10−29 | 2.48 (1.7–3.61) | 2.2×10−6 | 3.06 (2.55–3.67) | 1.9×10−33 |
|   | HH | 9.83 (6.70–14.44) | 1.8×10−31 | 6.76 (3.57–12.78) | 4.3 ×10−9 | 8.9 (6.40–12.40) | 8.5×10−39 |
|   |  |  |  |  |  |  |  |
| HLA-B*15:18 | HZ | 3.95 (1.97–7.89) | 1.1×10−4 | 7.71 (2.73–21.81) | 1.2 ×10−4 | 4.86 (2.72–8.66) | 8.7×10−8 |
|   |  |  |  |  |  |  |  |
| rs1363907 (G) ERAP2 | HH | 1.60 (1.31–1.96) | 3.3×10−6 | 1.43 (1.01–2.03) | 0.04 | 1.56 (1.31–1.85) | 4.3×10−7 |
|   |  |  |  |  |  |  |  |
| rs2476601(A) PTPN22 | HZ | 1.53 (1.20–1.95) | 5.6×10−4 | 0.92 (0.57–1.49) | 0.73 | 1.38 (1.11–1.71) | 0.003 |
|   | HH | 3.56 (1.68–7.46) | 7.8×10−4 | 1.69 (0.41–7.00) | 0.47 | 3.04 (1.57–5.86) | 0.0009 |

Table 3 Caption: Multivariable model for AC-DILI risk incorporating HLA-A*02:01, HLA-DRB1*15:01, HLA-B*15:18, ERAP2 rs1363907 and PTPN22 rs2476601 in the European population

The relationship between patients homozygous for rs1363907 (G) and AC-DILI was evaluated further focusing upon clinical phenotypes. The homozygote status of rs1363907 (GG) was not different among the three DILI clinical presentation phenotypes (e.g. hepatocellular, cholestatic or mixed) as well as other clinical characteristics (see [Supplemental Materials](#SD1)) but was associated with a longer time to DILI onset after starting AC treatment (latency) than those carrying one or no rs1363907(G) allele (25 vs 21 days, P = 0.03). A similar trend was observed in the independent European validation cohort (26 vs 24 days).

From our previously published DILI cohort,^8^ we examined 13 DILI drug groups with at least 20 European cases to determine whether rs1363907 (G) was an AC-specific risk factor. Each drug group was compared to the same set of controls. A significantly higher frequency of the rs1363907 (G) risk allele was observed only in DILI cases attributed to amoxicillin (n = 20), but not in DILI cases attributed to other causal drugs, including flucloxacillin (n = 195, [Figure S9](#SD1)). Risk of amoxicillin DILI was associated more strongly with *HLA-B*58:01* than with *HLA-A*02:01* (AF = 0.35) or with *HLA-DRB1*15:01*, as was reported previously in an amoxicillin DILI cohort^26^.

### Evaluation of ERAP1 association with AC-DILI

*ERAP1* encodes an enzyme with a similar function to *ERAP2. ERAP1* has 10 common haplotypes which were identified in cases and controls (see [Methods](#S6)). In the discovery cohort, *ERAP1* Hap2 and Hap8 were significantly associated with AC-DILI risk and Hap10 was marginally associated. Hap2 was a risk factor for AC-DILI (OR = 1.41; P = 4.9×10^−5^) whereas Hap10 and Hap8 were protective ([Table 2](#T2)). Similar allele frequencies and effect sizes for those haplotypes were observed in the validation cohort with Hap2 approaching statistical significance ([Table 2](#T2)). Due to the strong linkage disequilibrium between the two genes, conditioning on rs1363907 (*ERAP2*) reduced the *ERAP1* associations with AC-DILI, and only Hap2 still showed a marginally significant residual effect (OR = 1.26; P = 0.01; [Table S8](#SD1)).

### Predictive value for AC-DILI of the five markers and their combinations in a multiple regression analysis

The predictive value of three validated HLA risk alleles, rs2476601 (A) (*PTPN22*) and homozygous rs1363907 (G) (*ERAP2*) and their combinations were further evaluated. A multivariable logistic regression analysis established that each of the five markers was independently associated with AC-DILI risk and, except for rs1363907 and *HLA-B*15:18*, each marker showed a co-dominant effect ([Table 3](#T3)). The same model was applied to the validation cohort and supported the independent and codominant effects of the markers, including a trend for the PTPN22 variant ([Table 3](#T3)). We also applied a multi-marker logistic regression model to estimate the effect size of combinations of risk alleles compared to the baseline joint-negative group. Among most significant combinations ([Table S9](#SD1)), co-occurrence of one allele of *HLA-A*02:01* and *HLA-DRB1*15:01* appears in 7% of the cases and confers a 3.77-fold increase in odds of developing AC-DILI. The presence of both HLA risk alleles and the *PTPN22* variant doubled the odds ratio to about 6-fold and a comparable increase in risk (about 8-fold) was associated with the presence of homozygous *ERAP2*. Heterozygous status for all four of the risk markers was associated with a 17-fold increase in AC-DILI risk. The risk increases further in cases homozygous for at least one of the risk factors.

The predictive value of the multi-marker logistic regression model was assessed by ROC curve using first the discovery cases. The predictive model had comparable AUC values when these AC-DILI cases were compared to discovery population controls (AUC = 0.76, [Table S10](#SD1)) and to 1358 European non-AC-DILI cases (AUC = 0.76, [Table S10](#SD1)), confirming the drug-specificity of the model. Then, the same model was tested using the 133 validation AC-DILI cases compared to validation population controls, and to 403 European non-AC-DILI cases confirming that the model was predictive and specific for DILI (AUC = 0.79 and AUC = 0.70, [Table S10](#SD1)). The model accounted for about 13% of the AC-DILI susceptibility (R^2^ of the model, [Table S10](#SD1)).

### Risk allele interaction analysis

To assess possible interaction effects among the five risk factors, a systematic pairwise snp-snp interaction analysis^22^ was carried out ([Table S11](#SD1)). A more than additive effect between *HLA-DRB1*15:01* and *HLA-A*02:01* was the only interaction (OR = 2.0; P = 0.0002) statistically significant after multiple testing correction. However, both the rs2476601 (*PTPN22*) and homozygous rs1363907 (*ERAP2, HH*) showed consistent trends for more than additive risk effects when paired with any of the HLA risk alleles, with OR = 2.0 P = 0.03 for rs2476601 and OR = 1.7; P = 0.04 for rs1363907. Notably, homozygous rs1363907 (*ERAP2* Hap B) demonstrated a stronger trend for more than additive risk effect in association with *HLA-A*02:01* (OR = 1.62; P = 0.03) than with *HLA-DRB1*15:01* (OR = 1.18; P = 0.40). There was no interaction effect detected between the *PTPN22* variant and *ERAP2* Hap B. In the validation cohort, the interaction analysis supported similar trends for these interactions ([Table S11](#SD1)).

### Genetic Risk Score and AC-DILI risk

In order explore optimal use of the identified genotypes to prospectively assess AC-DILI risk in patients, we created a Genetic Rick Score (GRS). The AC-DILI GRS was estimated for each individual based on the weighted sum of the coefficients from a multivariable analysis of the five validated AC-DILI risk markers (see [Methods](#S6)). GRS ranged from 0 to 4.81 in the discovery and validation cohorts ([Figure S10](#SD1)). We explored the predictive value of GRS for AC-DILI risk in the validation cohort where the strong association was confirmed (OR = 2.2, 95%CI [1.88–2.69] P = 9.5×10^−19^). To quantify the risk for individuals carrying an increasing burden of risk alleles, we divided discovery and validation subjects into 10 quantiles based on the GRS score ranges. We estimated the relative risk of each quantile of developing DILI as a result of being treated with AC assuming that 1:2,350 patients treated with AC develop DILI^11^. While only about 1:5,000 patients with GRS in the lowest quantiles are predicted to experience DILI when treated with AC, the incidence increases to greater than 1:500 patients in the highest GRS quantiles ([Table 4](#T4)).

### Table 4.

|   |   | Discovery cohort (444 vs 10397) |  |  |  | Validation cohort (133 cases 17836) |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Q | GRS-range | N cases (%) | N controls (%) | RR | Estimated cases in general population | N cases (%) | N controls (%) | RR | Estimated cases in general population |
| 1 | 0–0.49 | 71 (16%) | 3918 (38%) | 0.43 | 1:5407 | 22 (17%) | 6740 (38%) | 0.44 | 1:5362 |
| 2 | 0.5–0.99 | 43 (10%) | 1979 (19%) | 0.52 | 1:4525 | 19 (14%) | 3189 (18%) | 0.80 | 1:2945 |
| 3 | 1.0–1.499 | 67 (15%) | 1925 (18%) | 0.82 | 1:2861 | 24 (18%) | 3199 (18%) | 1.00 | 1:2343 |
| 4 | 1.5–1.99 | 77 (17%) | 1442 (14%) | 1.24 | 1:1898 | 23 (17%) | 2555 (14%) | 1.20 | 1:1955 |
| 5 | 2.0–2.499 | 82 (19%) | 715 (7%) | 2.51 | 1:935 | 20 (15%) | 1364 (8%) | 1.95 | 1:1207 |
| 6 | 2.5–2.99 | 39 (9%) | 193 (2%) | 4.10 | 1:572 | 9 (7%) | 395 (2%) | 3.00 | 1:783 |
| 7 | 3.0–3.499 | 38 (9%) | 119 (1%) | 5.91 | 1:397 | 10 (8%) | 219 (1%) | 5.88 | 1:399 |
| 8 | 3.5–3.99 | 21 (5%) | 46 (0.4%) | 7.65 | 1:307 | 2 (2%) | 99 (0.6%) | 2.67 | 1:881 |
| 9 | 4.0–4.499 | 5 (1%) | 11 (0.1%) | 7.63 | 1:307 | 4 (3%) | 16 (0.09%) | 26.94 | 1:87 |
| 10 | 4.5–4.81 | 1 (0.2%) | 0 | 24.42 | 1:96 | 0 | 4 (0.02%) |   |   |

Table 4 Caption: Relative risk and estimate of incidence of AC DILI predicted from GRS in the discovery and validation cohorts.

### Drug-specificity of AC-DILI Genetic Risk Score compared to DILI due to other drugs

Finally, we explored the predictive value of GRS for AC-DILI risk when AC-DILI cases were compared with two independent cohorts of DILI due to other drugs (non-AC DILI). We found that the GRS was strongly associated with AC-DILI when our 444 AC-DILI discovery cases were compared to 1558 non-AC-DILI subjects (OR = 2.68, 95%CI [2.34–3.07] P = 5.7×10^−46^). A significant association was also found when we compared the 133 AC-DILI validation cases to a second independent set of 403 non-AC-DILI subjects (OR = 2.06, 95%CI [1.65–2.56] P=9.7×10^−11^). The predictive model had comparable AUC values in both the AC-DILI case/non-AC-DILI control cohorts (AUC = 0.76 and 0.70, respectively, [Table S10](#SD1)). Finally, we determined the GRS in the 13 largest European DILI drug groups in the Cirulli et al^8^ cohort. We found that after multiple testing correction, the score was not significantly correlated with the risk of DILI due to any of the 13 specific causal agents ([Table S12](#SD1)). Moreover, GRS distinguished high versus low causality cases among cases where AC was the lead implicated drug, and this was not the case for DILI attributed to other drugs ([Figure S11](#SD1)).

### AC-DILI Genetic Risk Score in the causality assessment compared to DILI due to other drugs

We next examined the predictive value of the GRS in addition to clinical risk factors associated with AC-DILI.^4^ For this purpose, we compared the 444 discovery AC-DILI cases to 1358 non-AC-DILI DILI cases (called non-AC-DILI discovery cohort) and the 133 replication cases against 403 newly genotyped non-AC-DILI cases (called non-AC-DILI validation cohort). As clinical risk factors for AC-DILI we considered patient age, being male, latency (time [days] from onset of AC treatment to recognition of DILI), and injury type (cholestatic or mixed vs hepatocellular). In the multivariable logistic model of the AC-DILI cases/non-AC-DILI discovery cohort, GRS and age, latency, sex and injury type and GRS were significantly associated with AC-DILI ([Table 5](#T5)). In the AC-DILI cases/non-AC-DILI validation cohort only age, latency and GRS were significant ([Table 5](#T5)). Among all the risk factors, the GRS was the most significant risk predictor in both multivariable models ([Table 5](#T5)). In both AC-DILI case/non-AC-DILI discovery and validation cohorts, the multivariable logistic model including clinical variables and GRS (AUC using the discovery cohort = 0.84 and AUC using the validation cohort = 0.85, [Table S10](#SD1) and [Figure S12](#SD1)) was significantly more predictive than the model including the only clinical variables (AUC = 0.76 and AUC = 0.80) when tested by Likelihood-ratio test (P = 3.1×10^−44^ and P = 1.1×10^−10^ respectively).

### Table 5: Summary statistics of age, sex, GRS, latency, and injury type in the multivariable logistic model developed to aid in the diagnosis of AC-DILI.

|   | Discovery (444 AC-DILI vs 1358 non-AC-DILI) |  | Validation (133 AC-DILI vs 403 non-AC-DILI) |  |
| --- | --- | --- | --- | --- |
| Factor | OR (95%CI) | P | OR (95%CI) | P |
| Age | 1.01 (1.00–1.02) | 3.7×10−5 | 1.05 (1.03–1.07) | 1.7×10−7 |
| Sex (M) | 1.97 (1.50–2.59) | 9.5×10−7 | 1.07 (0.62–1.84) | 0.82 |
| GRS | 2.90 (2.48–3.40) | 1.6×10−40 | 2.39 (1.80–3.17) | 1.7 ×10−9 |
| Latency | 0.39 (0.30–0.51) | 7.3×10−12 | 0.48 (0.39–0.63) | 1.1 ×10−7 |
| Injury type |   |   |   |   |
| cholestatic | 2.95 (208–4.18) | 1.1×10−9 | 1.19 (0.59–2.38) | 0.62 |
| mixed | 3.28 (2.32–4.64) | 1.4×10−11 | 1.53 (0.79–2.98) | 0.20 |

Table 5 Caption: The model was developed comparing the discovery European AC-DILI cohort to DILI due to drugs other than AC, and then applied to the validation AC-DILI cases using the same comparator non-AC-DILI cohort.

### Clinical Application of the GRS

To show the applicability of GRS in clinical practice, we estimated the optimal diagnostic cut off for the GRS by the Youden index commonly used for setting thresholds for diagnostic utility on medical tests, at 1.58 (95%CI [1.21–1.95]). GRS Youden index had similar specificity (SP) and sensitivity (SN) in the discovery and validation cohorts when cases were compared to population controls (SP = 79% and SN = 56% for discovery and SP = 77% and SN = 47% for the validation cohorts) and when AC-DILI cases were compared to non-AC-DILI cohorts (SP = 76% and SN = 56% for discovery and SP = 77% and SN = 47% for the validation cohorts), confirming the score specificity for AC-DILI. In fact, 56% of the AC cases had GRS > 1.58 compared to 21% and 23% of the controls and non-AC-DILI cases, respectively ([Table S12](#SD1)). Moreover, GRS was determined in the 20 cases in the DILIN registry in which co-exposure to AC was noted, but AC was not considered to be the cause of the DILI (and therefore not included in our AC DILI cohorts). The GRS was > 1.58 in 5 of these 20 cases (25%) ([Table S13](#SD1)). Each of these cases underwent repeat adjudication by hepatologists experienced in DILI (PBW, RJF, JAO, JS and AS). In the case with the highest GRS, knowledge of the GRS resulted in unanimous consensus that the causal drug was AC and not azithromycin (see footnote in [Table S13](#SD1)).

### The five AC-DILI risk markers in African American and Hispanic populations

We evaluated the association of the five markers in AA and HSP populations, which are reported in [Table S14](#SD1). Although with limited power, we found that *HLA-DRB1*15:01* risk allele had a higher frequency in cases across the three major ethnicities compared to population-based controls. Compared to population controls, the 13 AA cases had higher frequencies of *HLA-A*02:01* (AF 0.12 vs 0.19), ERAP2 rs1363907 (AF 0.65 vs 0.77) and *HLA-DRB1:15:03* (AF 0.23 vs 0.12) which has a similar peptide binding profile to *HLA-DRB1:15:01.*^27^ The 7 Hispanic cases only revealed a trend for carrying the *PTPN22* risk allele compared to controls. All these differences were not statistically significant due to the low number of cases. For non-European individuals, predictions for the *ERAP1* haplotypes could not be estimated because of the missing data of several relevant proxy SNPs.

### Genetic evaluation of European AC-DILI cases not carrying the five risk factors

Since no clinical differences were detected between the AC-DILI cases not carrying any the five risk AC-DILI markers (non-carriers) and the rest of the AC-DILI cases, we evaluated if the negative carriers would be enriched in additional novel HLA risk alleles. We compared the 43 AC-DILI non-carriers to the control population. The cases were enriched in *HLA-B*18:01* (AF 0.14 vs AF 0.05) and *HLA-B*35:01* (AF 0.13 vs AF 0.05, [Table S15](#SD1)). The trend association with *HLA-B*18:01* was also observed in the validation cohort with *HLA-B*18:01* frequency higher among the 18 non-carriers than controls (AF 0.08 vs AF 0.04).

## Discussion

In the current study, TWAS and GWAS analyses were performed on the largest existing cohort of European AC-DILI cases resulting in the identification of novel independent genetic associations of AC-DILI with rs1363907 in *ERAP2*, a gene known to be involved in antigen processing^10^, and with an additional HLA allele, *HLA-B*15:18*. These associations were confirmed in a validation cohort of 133 additional European AC-DILI cases. We demonstrated that the predicted risk associated with the combination of the five genetic risk loci identified in this and previous studies^6, 8^ was specific for AC-DILI and could be helpful in the diagnosis of AC-DILI.

The *ERAP2* association was identified using the TWAS approach where the predicted liver expression of *ERAP2* was significantly lower in the AC-DILI cases than controls. *ERAP2* has two major haplotypes (Hap A and Hap B) and our lead SNP rs1363907 appears to tag Hap B. Hap B undergoes differential splicing of exon 10, leading to generation of a premature stop codon resulting in a truncated *ERAP2* mRNA that is rapidly degraded^24^. This rapid degradation presumably accounts for the reduction of total *ERAP2 mRNA* we detected as a risk factor in the PrediXscan approach. Interestingly, for 1,183 livers that have undergone eQTL mapping^25^, the expression of *ERAP2* mRNA was significantly lower only in individuals carrying two copies of the Hap B, supporting a recessive model. Consistent with this observation, the increased risk of AC-DILI was associated only with homozygous carriage of Hap B. In the heterozygous state, compensatory mechanisms may exist to maintain *ERAP2* expression in the liver. From a clinical perspective, Hap B homozygous state seemed to be associated only with a small increase in latency to onset of DILI and not with other clinical characteristics of the liver injury.

ERAP2 is present in the endoplasmic reticulum of all cells where it functions to trim peptides generated by the proteasome so that they will fit into the peptide binding pocket of newly synthesized Class I HLA molecules prior to their transit to the cell surface^10^ ([Figure 2](#F2)). It has been reported that ERAP2 may over-trim some peptides such that they are unable to bind HLA molecules, supporting a role for *ERAP2* in controlling which peptides have antigenic potential^10^. *ERAP2* haplotype B has been associated with risk of several autoimmune diseases, including birdshot chorioretinopathy, ankylosing spondylitis, and some forms of psoriasis, all of which have strong risk associations with specific Class I HLA alleles^10^. However, unlike the case with AC-DILI, the risk of these diseases is generally lower in those carrying Hap B rather than higher^10^. In the case of birdshot chorioretinopathy, it has been recently shown that ERAP2 generates a profile of peptides that specifically bind to the class I allele associated with risk of this disease (HLAA29)^28^. The lower risk of this disease in those carrying ERAP2 Hap B may therefore result from reduced production and presentation of the culprit neoantigens by HLA-A29. The association of Hap B with increased AC-DILI risk may suggest that the culprit neoantigens are preferentially created by the other endoplasmic reticulum enzyme that trim peptides, ERAP1 ([Figure 2](#F2)).

### Figure 2.

![Figure 2.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/454a/9974860/abcf2f5653bb/nihms-1855942-f0003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9974860_nihms-1855942-f0003.jpg)

Schematic representation of the role of ERAP1 and ERAP2 in the presentation of antigens. ERAP1 and ERAP2 function to shorten peptides generated by the proteosome so that they can fit in the peptide binding domain of Class 1 HLA molecules as they are synthesized in the endoplasmic reticulum and before they are trafficked to the cell surface. ERAP1 and ERAP2 generate different profiles of peptides. Treatment with AC presumably alters some peptides produced by the proteosome. Reduced function of ERAP2 as a result of carriage of two HAP B alleles presumably shifts peptide production to ERAP1. AC altered peptides (neoantigens) presumably created by ERAP1 may stimulate an adaptive immune attack on the liver if presented on the liver cell surface by the risk associated Class 1 HLA molecules. Carriage of two Class I HLA risk alleles presumably doubles the potential load of neoantigens presented on the cell surface, increasing the risk of an adaptive immune response (Figure modified from 36). TCR = T-cell receptor.

Among the drugs most frequently associated with DILI, only AC and amoxicillin DILI were significantly associated with *ERAP2* Hap B. Amoxicillin DILI risk was not associated with our GRS and the clinical presentation is not identical to that of AC-DILI^4^. Although pharmacy records were not checked in most cases, it seems unlikely that many cases were erroneously attributed to amoxicillin and actually due to AC. Clavulanate has long been considered the main component causing AC-DILI (LiverTox [https://www.ncbi.nlm.nih.gov/books/NBK548517/](https://www.ncbi.nlm.nih.gov/books/NBK548517/)). However, the recent observation that amoxicillin-modified peptides bind *HLA-DRB1*15:01* and/or *DQB1*06:02* and activate T cells obtained from an AC-DILI patients^29^ may support a role of amoxicillin in at least some of the AC-DILI cases.

Various haplotypes of *ERAP1* have also been associated with certain autoimmune diseases^30^. We found that increased AC-DILI risk was associated *ERAP1* Hap 2 which produces an enzyme with relatively high activity, while the lower activity products of *ERAP1* Hap 8 and 10 showed a protective effect. These observations further support the notion that the culprit neoantigens sparking AC-DILI are likely produced by ERAP1. However, when conditioned on carriage of ERAP2 Hap B, only the risk association with *ERAP1* Hap 2 remained marginally significant, reflecting the long LD block encompassing the two genes. Additional studies will be needed to identify independent risk associations with ERAP1 haplotypes.

We identified a gene-dose effect *HLA-A*02:01* and *DRB1*15:01* for AC-DILI risk, and our data support this also for the *PTPN22* missense variant. In addition, we confirmed the more than additive effect of carrying both the *HLA-A*02:01* and *DRB1*15:01* risk alleles^6^, doubling the sum of risks associated by each individual allele. We also found a trend for a more than additive increase in DILI risk when *ERAP2* Hap B co-occurred with *HLA-A*02:01* but not with *HLA-DRB1*15:01*. Although not significant after multiple comparison correction, these observations are consistent with the known role of ERAP enzymes to provide suitable peptides for binding to Class 1 but not Class 2 molecules^10^.

Finally, we developed a GRS based on the five identified risk alleles. We included the PTPN22 variant although the risk association was not observed in our validation cohort. This is because the association was GWAS significant in a prior study of all cause DILI, and because a trend towards a gene-dose effect was observed in the validation cohort. We assume that the association would have been validated in a larger cohort. To our knowledge, specific genes involved in absorption, distribution, metabolism or excretion (ADME) of either amoxicillin or clavulonic acid have not been identified, and no pharmacogenetic variants located in or near any major ADME genes were genome-wide significant.

Our observations support the future value of the GRS for identifying AC as the culprit drug in DILI cases, and potentially a role in risk management. Among Europeans, the GRS was found to be predictive of AC-DILI risk when tested against population controls and was highly specific for AC-DILI when tested against non-AC-DILI cohorts. We estimated that a GRS value above 1.58 would be a minimum value to support the diagnosis of AC-DILI, although up to half true AC DILI cases will have GRS values below this cutoff (~50% sensitivity). The value to health care professionals in making an AC-DILI diagnosis increases the higher the GRS value (increasing specificity although reducing sensitivity). Because the risk of developing DILI during treatment in an average Caucasian has been estimated to be about 1:2,350^11^, we were able to use the GRS to estimate an individual risk to be both lower than this average (>1:5,000) or much higher (approaching 1:100). To apply the GRS to causality assessment, we examined cases in the DILIN registry where AC was a concomitant treatment but not felt to be the cause of the DILI. Five of these cases had GRS above the 1.58 threshold, and re-adjudication of the patient with the highest GRS (corresponding to at risk of 1:600) resulted in confident attribution of the DILI event to AC in one of these cases.

AC-DILI has been shown to be more common in elderly men, and to characteristically present as a mixed or cholestatic injury several weeks after initiating AC treatment.^4^ Accordingly, we found that a regression model including age, sex, latency and injury type did differentiate AC-DILI from non-AC-DILI in two independent large case/non-AC-DILI control cohorts. Nonetheless, addition of the GRS to this model significantly improved its sensitivity and specificity in differentiating AC-DILI from DILI due to other drugs.

It is of note that our multi-marker model based on the five risk alleles predicted about 13% of the total risk for AC-DILI, which is about one third of the total DILI risk that has been previously attributed to common genetic variants (approximately 40%^31^). This is a large proportion of total genetic risk relative to most diseases^32^. This finding suggests that DILI, as is the case for other severe adverse drug events, might have an oligogenic architecture with few highly penetrant genetic risk factors. Under this hypothesis, identification of a limited number of additional risk alleles might lead to substantial improvements in the GRS performance, increasing the sensitivity of the GRS score.

Unfortunately, our GRS is unlikely to be useful in AA and Hispanic populations, which appear to have a lower incidence of AC-DILI compared to Europeans^33^. Even though *HLA-DRB1*15:01* is less common in both minority populations compared to Europeans (AF = 0.12), both AA and Hispanic AC-DILI cases showed double the frequency of *HLA-DRB1*15:01* in AC-DILI cases *vs* controls (AF 0.04 vs 0.02 and 0.14 vs 0.06), supporting this HLA Class II allele as a general risk factor for AC-DILI. Interestingly, *HLA-DRB1*15:03* showed an even stronger enrichment in the AA AC-DILI cases (0.23 in cases vs 0.13 in controls). In AA, *HLA-DRB1*15:03* is the most common *HLA-DRB1*15* allele and has a similar peptide binding affinity as *HLA-DRB1*15:01.*^27^ Interestingly, *HLA-DRB1*15:03* has an established role in the susceptibility of autoimmune diseases^27^ and drug-induced DRESS^34^. AA cases were also enriched in *HLA-DRB1*15:02,* a known risk factor for AC-DILI among people of Indian or Pakistani origin^35^. Due to limited number of cases, no conclusion could be drawn for rs2476601and rs1363907 associations. Larger association studies are needed to confirm whether differences in HLA profiles accounts for differences in AC-DILI susceptibility.

In summary, a common variant in *ERAP2,* a proxy of Hap B, was identified as a novel risk factor for AC-DILI, implicating for the first-time antigen processing as a locus of DILI risk. A rare class 1 HLA risk allele, *HLA-B*15:18* was also identified as a risk factor for AC-DILI. When combined with three previously established genetic risk factors, the resulting five-locus GRS was highly specific for AC-DILI, supporting a future role for genetic testing in DILI causality assessment and potentially risk management.

## Supplementary Material

### BACKGROUND AND CONTEXT:

### BACKGROUND AND CONTEXT:

### BACKGROUND AND CONTEXT:

### BACKGROUND AND CONTEXT:

The diagnosis of amoxicillin-clavulanate drug-induced liver injury (AC-DILI) can be challenging.

### NEW FINDINGS:

A variant in a gene involved in antigen processing, ERAP2, and a novel Class 1 HLA allele were identified as risk factors and incorporated into a polygenic risk score highly specific for AC-DILI.

### LIMITATIONS:

The risk score accounts for ~13% of the overall AC-DILI risk but out-performed clinical risk factors.

### IMPACT:

The polygenic risk score should assist in AC-DILI causality assessment and provides an example of how this approach could improve causality assessment and ultimately risk management of DILI due to other drugs.

## Acknowledgments

We acknowledge the contributors to sample collection via DILIN, iDILIC, the Spanish DILI registry, EUDRAGENE, and DILIGEN and ProEuroDILI. We thank Jay Hoofnagle and Ludmila Pawlikowska for feedback that helped to improve the manuscript. We thank Daniele Cusi (Hypergenes), Patrik K. Magnusson (Swedish Twin Registry) and Javier Martin (Spanish DNA bank) for provision of control data and Arthur Holden (iSAEC) for establishing collaborations with their institutions. We also thank NCBI/dbGaP at [http://www.ncbi.nlm.nih.gov/gap](http://www.ncbi.nlm.nih.gov/gap), who approved our application to obtain additional control datasets. In particular, eMERGE GWAS datasets were from dbGaP accession number *phs000360.v3.p1*. The eMERGE network is a consortium of five participating sites (Group Health Seattle, Marshfield Clinic, Mayo Clinic, Northwestern University, and Vanderbilt University) funded by the National Human Genome Research Institute (NHGRI). Funding supports for the cohort and genotyping of eMERGE samples are as follows: (1) Group Health Seattle: U01AG006781 and U01HG004438 with genotyping performed at Johns Hopkins University; (2) Mayo Clinic: UOIHG004599 and U01HG004424 with genotyping performed at The Broad Institute; (3) Marshfield Clinic: U01HG004608 and U01HG004438 with genotyping performed at Johns Hopkins University; (4) Northwestern University: U01HG04603 and U01HG004424 with the genotyping performed at The Broad Institute; (5) Vanderbilt University: U01HG004603 and U01HG004424 with the genotyping performed at The Broad Institute. The PAGE dataset was from the Charles Bronfman Institute for Personalized Medicine (IPM) BioMe BioBank at the Icahn School of Medicine at Mount Sinai (New York). Phenotype data collection was supported by The Andrea and Charles Bronfman Philanthropies. Funding support for genotyping, which was performed at The Center for Inherited Disease Research (CIDR), was provided by the NIH (U01HG007417). The GWAS data were from dbGaP accession number *phs000925.v1.p1*. The Ashkenazi GWAS data were from dbGaP accession number *phs000448.v1.p1*. The dataset was provided by Dr. Todd Lencz and Ariel Darvasi, Ph.D. Support for collection and analysis was provided by RC2MH089964, R01 MH084098, the North Shore - LIJ Health System Foundation, and the Hebrew University Genetic Resource. The kidney GWAS data were from dbGaP accession number *phs001271.v1.p1*. The dataset was funded by grant NCI U01CA155309 to G Scelo. The KORA GWAS data were from dbGaP accession number *phs000303.v1.pl*. The dataset was obtained from the NEI Refractive Error Collaboration (NEIREC) Database. Funding support for NEIREC was provided by the National Eye Institute. We thank NEIREC participants and the NEIREC Research Group for their valuable contribution to this research.

### Grant support

The DILIN ([https://dilin.dcri.duke.edu/](https://dilin.dcri.duke.edu/)) is supported by the National Institute of Diabetes and Digestive and Kidney Diseases of the National Institutes of Health (NIH) as a Cooperative Agreement (U01s) under grants: U01-DK065176 (Duke), U01-DK065201 (UNC), U01-DK065184 (Michigan), U01-DK065211 (Indiana), U01DK065193 (UConn), U01-DK065238 (UCSF/CPMC), U01-DK083023 (UTSW), U01-DK083027 (TJH/UPenn), U01-DK082992 (Mayo), U01-DK083020 (USC), U01-DK100928 (Icahn). Additional funding is provided by CTSA grants UL1 RR025761 (Indiana), UL1 RR025747 (UNC), and UL1 UL1-RR024986 (UMich). The University of Nottingham sample collection was funded by the National Institute for Health Research (NIHR) Nottingham Digestive Diseases Biomedical Research Unit at the Nottingham University Hospitals NHS Trust and University of Nottingham. The views expressed are those of the author(s) and not necessarily those of the NHS, the NIHR or the Department of Health. GPA is the gastrointestinal and liver disorder theme lead for the NIHR Nottingham BRC (Reference no: BRC-1215-20003). The Spanish DILI Registry is partly funded by the Spanish Medicine Agency, Fondo Europeo de Desarrollo Regional - FEDER (FIS PI16_01748, PI15_01440). CIBERehd is funded by Instituto de Salud Carlos III. Pro-Euro DILI Network is also supported by the COST (European Cooperation in Science and Technology; [www.cost.eu](http://www.cost.eu)) Action “CA17112”.

## Abbreviations:

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Chalasani N, Bonkovsky HL, Fontana R, et al. Features and Outcomes of 899 Patients With Drug-Induced Liver Injury: The DILIN Prospective Study. Gastroenterology 2015;148:1340–52 e7.  [DOI](https://doi.org/10.1053/j.gastro.2015.03.006) | [PMC free article](/articles/PMC4446235/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25754159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Features%20and%20Outcomes%20of%20899%20Patients%20With%20Drug-Induced%20Liver%20Injury:%20The%20DILIN%20Prospective%20Study&author=N%20Chalasani&author=HL%20Bonkovsky&author=R%20Fontana&volume=148&publication_year=2015&pages=1340-52%20e7&pmid=25754159&doi=10.1053/j.gastro.2015.03.006&)

2. Reuben A, Koch DG, Lee WM. Drug-induced acute liver failure: results of a U.S. multicenter, prospective study. Hepatology 2010;52:2065–76.  [DOI](https://doi.org/10.1002/hep.23937) | [PMC free article](/articles/PMC3992250/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20949552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Drug-induced%20acute%20liver%20failure:%20results%20of%20a%20U.S.%20multicenter,%20prospective%20study&author=A%20Reuben&author=DG%20Koch&author=WM%20Lee&volume=52&publication_year=2010&pages=2065-76&pmid=20949552&doi=10.1002/hep.23937&)

3. Watkins PB. Idiosyncratic drug-induced liver injury in patients: Detection, severity assessment, and regulatory implications. Adv Pharmacol 2019;85:165–193.  [DOI](https://doi.org/10.1016/bs.apha.2019.02.004) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31307586/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv%20Pharmacol&title=Idiosyncratic%20drug-induced%20liver%20injury%20in%20patients:%20Detection,%20severity%20assessment,%20and%20regulatory%20implications&author=PB%20Watkins&volume=85&publication_year=2019&pages=165-193&pmid=31307586&doi=10.1016/bs.apha.2019.02.004&)

4. deLemos AS, Ghabril M, Rockey DC, et al. Amoxicillin-Clavulanate-Induced Liver Injury. Dig Dis Sci 2016;61:2406–16.  [DOI](https://doi.org/10.1007/s10620-016-4121-6) | [PMC free article](/articles/PMC4945382/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27003146/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Dig%20Dis%20Sci&title=Amoxicillin-Clavulanate-Induced%20Liver%20Injury&author=AS%20deLemos&author=M%20Ghabril&author=DC%20Rockey&volume=61&publication_year=2016&pages=2406-16&pmid=27003146&doi=10.1007/s10620-016-4121-6&)

5. Lucena MI, Andrade RJ, Fernandez MC, et al. Determinants of the clinical expression of amoxicillin-clavulanate hepatotoxicity: a prospective series from Spain. Hepatology 2006;44:850–6.  [DOI](https://doi.org/10.1002/hep.21324) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17006920/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Determinants%20of%20the%20clinical%20expression%20of%20amoxicillin-clavulanate%20hepatotoxicity:%20a%20prospective%20series%20from%20Spain&author=MI%20Lucena&author=RJ%20Andrade&author=MC%20Fernandez&volume=44&publication_year=2006&pages=850-6&pmid=17006920&doi=10.1002/hep.21324&)

6. Lucena MI, Molokhia M, Shen Y, et al. Susceptibility to amoxicillin-clavulanate-induced liver injury is influenced by multiple HLA class I and II alleles. Gastroenterology 2011;141:338–47.  [DOI](https://doi.org/10.1053/j.gastro.2011.04.001) | [PMC free article](/articles/PMC3129430/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21570397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Susceptibility%20to%20amoxicillin-clavulanate-induced%20liver%20injury%20is%20influenced%20by%20multiple%20HLA%20class%20I%20and%20II%20alleles&author=MI%20Lucena&author=M%20Molokhia&author=Y%20Shen&volume=141&publication_year=2011&pages=338-47&pmid=21570397&doi=10.1053/j.gastro.2011.04.001&)

7. Stephens C, Lopez-Nevot MA, Ruiz-Cabello F, et al. HLA alleles influence the clinical signature of amoxicillin-clavulanate hepatotoxicity. PLoS One 2013;8:e68111.  [DOI](https://doi.org/10.1371/journal.pone.0068111) | [PMC free article](/articles/PMC3706603/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23874514/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=HLA%20alleles%20influence%20the%20clinical%20signature%20of%20amoxicillin-clavulanate%20hepatotoxicity&author=C%20Stephens&author=MA%20Lopez-Nevot&author=F%20Ruiz-Cabello&volume=8&publication_year=2013&pages=e68111&pmid=23874514&doi=10.1371/journal.pone.0068111&)

8. Cirulli ET, Nicoletti P, Abramson K, et al. A Missense Variant in PTPN22 is a Risk Factor for Drug-induced Liver Injury. Gastroenterology 2019;156:1707–1716.e2.  [DOI](https://doi.org/10.1053/j.gastro.2019.01.034) | [PMC free article](/articles/PMC6511989/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30664875/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=A%20Missense%20Variant%20in%20PTPN22%20is%20a%20Risk%20Factor%20for%20Drug-induced%20Liver%20Injury&author=ET%20Cirulli&author=P%20Nicoletti&author=K%20Abramson&volume=156&publication_year=2019&pages=1707-1716.e2&pmid=30664875&doi=10.1053/j.gastro.2019.01.034&)

9. Stanford SM, Bottini N. PTPN22: the archetypal non-HLA autoimmunity gene. Nat Rev Rheumatol 2014;10:602–11.  [DOI](https://doi.org/10.1038/nrrheum.2014.109) | [PMC free article](/articles/PMC4375551/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25003765/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Rheumatol&title=PTPN22:%20the%20archetypal%20non-HLA%20autoimmunity%20gene&author=SM%20Stanford&author=N%20Bottini&volume=10&publication_year=2014&pages=602-11&pmid=25003765&doi=10.1038/nrrheum.2014.109&)

10. de Castro JAL, Stratikos E. Intracellular antigen processing by ERAP2: Molecular mechanism and roles in health and disease. Hum Immunol 2019;80:310–317.  [DOI](https://doi.org/10.1016/j.humimm.2018.11.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30414458/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Immunol&title=Intracellular%20antigen%20processing%20by%20ERAP2:%20Molecular%20mechanism%20and%20roles%20in%20health%20and%20disease&author=JAL%20de%20Castro&author=E%20Stratikos&volume=80&publication_year=2019&pages=310-317&pmid=30414458&doi=10.1016/j.humimm.2018.11.001&)

11. Bjornsson ES, Bergmann OM, Bjornsson HK, et al. Incidence, presentation, and outcomes in patients with drug-induced liver injury in the general population of Iceland. Gastroenterology 2013;144:1419–25, 1425 e1–3; quiz e19–20.  [DOI](https://doi.org/10.1053/j.gastro.2013.02.006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23419359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Incidence,%20presentation,%20and%20outcomes%20in%20patients%20with%20drug-induced%20liver%20injury%20in%20the%20general%20population%20of%20Iceland&author=ES%20Bjornsson&author=OM%20Bergmann&author=HK%20Bjornsson&volume=144&publication_year=2013&pages=1419-25&pmid=23419359&doi=10.1053/j.gastro.2013.02.006&)

12. Fontana RJ, Watkins PB, Bonkovsky HL, et al. Drug-Induced Liver Injury Network (DILIN) prospective study: rationale, design and conduct. Drug Saf 2009;32:55–68.  [DOI](https://doi.org/10.2165/00002018-200932010-00005) | [PMC free article](/articles/PMC3637941/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19132805/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Saf&title=Drug-Induced%20Liver%20Injury%20Network%20(DILIN)%20prospective%20study:%20rationale,%20design%20and%20conduct&author=RJ%20Fontana&author=PB%20Watkins&author=HL%20Bonkovsky&volume=32&publication_year=2009&pages=55-68&pmid=19132805&doi=10.2165/00002018-200932010-00005&)

13. Aithal GP, Watkins PB, Andrade RJ, et al. Case Definition and Phenotype Standardization in Drug-Induced Liver Injury. Clinical Pharmacology and Therapeutics 2011;89:806–815.  [DOI](https://doi.org/10.1038/clpt.2011.58) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21544079/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinical%20Pharmacology%20and%20Therapeutics&title=Case%20Definition%20and%20Phenotype%20Standardization%20in%20Drug-Induced%20Liver%20Injury&author=GP%20Aithal&author=PB%20Watkins&author=RJ%20Andrade&volume=89&publication_year=2011&pages=806-815&pmid=21544079&doi=10.1038/clpt.2011.58&)

14. Nicoletti P, Aithal GP, Bjornsson ES, et al. Association of Liver Injury From Specific Drugs, or Groups of Drugs, With Polymorphisms in HLA and Other Genes in a Genome-Wide Association Study. Gastroenterology 2017;152:1078–1089.  [DOI](https://doi.org/10.1053/j.gastro.2016.12.016) | [PMC free article](/articles/PMC5367948/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28043905/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastroenterology&title=Association%20of%20Liver%20Injury%20From%20Specific%20Drugs,%20or%20Groups%20of%20Drugs,%20With%20Polymorphisms%20in%20HLA%20and%20Other%20Genes%20in%20a%20Genome-Wide%20Association%20Study&author=P%20Nicoletti&author=GP%20Aithal&author=ES%20Bjornsson&volume=152&publication_year=2017&pages=1078-1089&pmid=28043905&doi=10.1053/j.gastro.2016.12.016&)

15. Gamazon ER, Wheeler HE, Shah KP, et al. A gene-based association method for mapping traits using reference transcriptome data. Nat Genet 2015;47:1091–8.  [DOI](https://doi.org/10.1038/ng.3367) | [PMC free article](/articles/PMC4552594/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26258848/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=A%20gene-based%20association%20method%20for%20mapping%20traits%20using%20reference%20transcriptome%20data&author=ER%20Gamazon&author=HE%20Wheeler&author=KP%20Shah&volume=47&publication_year=2015&pages=1091-8&pmid=26258848&doi=10.1038/ng.3367&)

16. Li YJ, Phillips EJ, Dellinger A, et al. Human Leukocyte Antigen B*14:01 and B*35:01 Are Associated With Trimethoprim-Sulfamethoxazole Induced Liver Injury. Hepatology 2021;73:268–281.  [DOI](https://doi.org/10.1002/hep.31258) | [PMC free article](/articles/PMC7544638/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32270503/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=Human%20Leukocyte%20Antigen%20B*14:01%20and%20B*35:01%20Are%20Associated%20With%20Trimethoprim-Sulfamethoxazole%20Induced%20Liver%20Injury&author=YJ%20Li&author=EJ%20Phillips&author=A%20Dellinger&volume=73&publication_year=2021&pages=268-281&pmid=32270503&doi=10.1002/hep.31258&)

17. Das S, Forer L, Schonherr S, et al. Next-generation genotype imputation service and methods. Nat Genet 2016;48:1284–1287.  [DOI](https://doi.org/10.1038/ng.3656) | [PMC free article](/articles/PMC5157836/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27571263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=Next-generation%20genotype%20imputation%20service%20and%20methods&author=S%20Das&author=L%20Forer&author=S%20Schonherr&volume=48&publication_year=2016&pages=1284-1287&pmid=27571263&doi=10.1038/ng.3656&)

18. Zheng X, Shen J, Cox C, et al. HIBAG--HLA genotype imputation with attribute bagging. Pharmacogenomics Journal 2014;14:192–200.  [DOI](https://doi.org/10.1038/tpj.2013.18) | [PMC free article](/articles/PMC3772955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23712092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20Journal&title=HIBAG--HLA%20genotype%20imputation%20with%20attribute%20bagging&author=X%20Zheng&author=J%20Shen&author=C%20Cox&volume=14&publication_year=2014&pages=192-200&pmid=23712092&doi=10.1038/tpj.2013.18&)

19. Purcell S, Neale B, Todd-Brown K, et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am J Hum Genet 2007;81:559–75.  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=PLINK:%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S%20Purcell&author=B%20Neale&author=K%20Todd-Brown&volume=81&publication_year=2007&pages=559-75&pmid=17701901&doi=10.1086/519795&)

20. Ombrello MJ, Kastner DL, Remmers EF. Endoplasmic reticulum-associated aminopeptidase 1 and rheumatic disease: genetics. Curr Opin Rheumatol 2015;27:349–56.  [DOI](https://doi.org/10.1097/BOR.0000000000000189) | [PMC free article](/articles/PMC4565054/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26002026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Rheumatol&title=Endoplasmic%20reticulum-associated%20aminopeptidase%201%20and%20rheumatic%20disease:%20genetics&author=MJ%20Ombrello&author=DL%20Kastner&author=EF%20Remmers&volume=27&publication_year=2015&pages=349-56&pmid=26002026&doi=10.1097/BOR.0000000000000189&)

21. Pruim RJ, Welch RP, Sanna S, et al. LocusZoom: regional visualization of genome-wide association scan results. Bioinformatics 2010;26:2336–7.  [DOI](https://doi.org/10.1093/bioinformatics/btq419) | [PMC free article](/articles/PMC2935401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20634204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LocusZoom:%20regional%20visualization%20of%20genome-wide%20association%20scan%20results&author=RJ%20Pruim&author=RP%20Welch&author=S%20Sanna&volume=26&publication_year=2010&pages=2336-7&pmid=20634204&doi=10.1093/bioinformatics/btq419&)

22. Cordell HJ. Detecting gene-gene interactions that underlie human diseases. Nat Rev Genet 2009;10:392–404.  [DOI](https://doi.org/10.1038/nrg2579) | [PMC free article](/articles/PMC2872761/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19434077/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=Detecting%20gene-gene%20interactions%20that%20underlie%20human%20diseases&author=HJ%20Cordell&volume=10&publication_year=2009&pages=392-404&pmid=19434077&doi=10.1038/nrg2579&)

23. Youden WJ. Index for rating diagnostic tests. Cancer 1950;3:32–5.  [DOI](https://doi.org/10.1002/1097-0142(1950)3:1<32::aid-cncr2820030106>3.0.co;2-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15405679/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Index%20for%20rating%20diagnostic%20tests&author=WJ%20Youden&volume=3&publication_year=1950&pages=32-5&pmid=15405679&doi=10.1002/1097-0142(1950)3:1<32::aid-cncr2820030106>3.0.co;2-3&)

24. Andres AM, Dennis MY, Kretzschmar WW, et al. Balancing selection maintains a form of ERAP2 that undergoes nonsense-mediated decay and affects antigen presentation. PLoS Genet 2010;6:e1001157.  [DOI](https://doi.org/10.1371/journal.pgen.1001157) | [PMC free article](/articles/PMC2954825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20976248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=Balancing%20selection%20maintains%20a%20form%20of%20ERAP2%20that%20undergoes%20nonsense-mediated%20decay%20and%20affects%20antigen%20presentation&author=AM%20Andres&author=MY%20Dennis&author=WW%20Kretzschmar&volume=6&publication_year=2010&pages=e1001157&pmid=20976248&doi=10.1371/journal.pgen.1001157&)

25. Etheridge AS, Gallins PJ, Jima D, et al. A New Liver Expression Quantitative Trait Locus Map From 1,183 Individuals Provides Evidence for Novel Expression Quantitative Trait Loci of Drug Response, Metabolic, and Sex-Biased Phenotypes. Clin Pharmacol Ther 2020;107:1383–1393.  [DOI](https://doi.org/10.1002/cpt.1751) | [PMC free article](/articles/PMC7816646/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31868224/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=A%20New%20Liver%20Expression%20Quantitative%20Trait%20Locus%20Map%20From%201,183%20Individuals%20Provides%20Evidence%20for%20Novel%20Expression%20Quantitative%20Trait%20Loci%20of%20Drug%20Response,%20Metabolic,%20and%20Sex-Biased%20Phenotypes&author=AS%20Etheridge&author=PJ%20Gallins&author=D%20Jima&volume=107&publication_year=2020&pages=1383-1393&pmid=31868224&doi=10.1002/cpt.1751&)

26. Nicoletti P, Aithal GP, Chamberlain TC, et al. Drug-Induced Liver Injury due to Flucloxacillin: Relevance of Multiple Human Leukocyte Antigen Alleles. Clin Pharmacol Ther 2019;106:245–253.  [DOI](https://doi.org/10.1002/cpt.1375) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30661239/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Drug-Induced%20Liver%20Injury%20due%20to%20Flucloxacillin:%20Relevance%20of%20Multiple%20Human%20Leukocyte%20Antigen%20Alleles&author=P%20Nicoletti&author=GP%20Aithal&author=TC%20Chamberlain&volume=106&publication_year=2019&pages=245-253&pmid=30661239&doi=10.1002/cpt.1375&)

27. Zumelzu C, Le Roux-Villet C, Loiseau P, et al. Black patients of African descent and HLA-DRB1*15:03 frequency overrepresented in epidermolysis bullosa acquisita. J Invest Dermatol 2011;131:2386–93.  [DOI](https://doi.org/10.1038/jid.2011.231) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21833018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Invest%20Dermatol&title=Black%20patients%20of%20African%20descent%20and%20HLA-DRB1*15:03%20frequency%20overrepresented%20in%20epidermolysis%20bullosa%20acquisita&author=C%20Zumelzu&author=C%20Le%20Roux-Villet&author=P%20Loiseau&volume=131&publication_year=2011&pages=2386-93&pmid=21833018&doi=10.1038/jid.2011.231&)

28. Kuiper JJW, Setten JV, Devall M, et al. Functionally distinct ERAP1 and ERAP2 are a hallmark of HLA-A29-(Birdshot) Uveitis. Hum Mol Genet 2018;27:4333–4343.  [DOI](https://doi.org/10.1093/hmg/ddy319) | [PMC free article](/articles/PMC6276832/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30215709/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Functionally%20distinct%20ERAP1%20and%20ERAP2%20are%20a%20hallmark%20of%20HLA-A29-(Birdshot)%20Uveitis&author=JJW%20Kuiper&author=JV%20Setten&author=M%20Devall&volume=27&publication_year=2018&pages=4333-4343&pmid=30215709&doi=10.1093/hmg/ddy319&)

29. Tailor A, Meng X, Adair K, et al. HLA DRB1*15:01-DQB1*06:02-Restricted Human CD4+ T Cells Are Selectively Activated With Amoxicillin-Peptide Adducts. Toxicol Sci 2020;178:115–126.  [DOI](https://doi.org/10.1093/toxsci/kfaa128) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32777075/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicol%20Sci&title=HLA%20DRB1*15:01-DQB1*06:02-Restricted%20Human%20CD4+%20T%20Cells%20Are%20Selectively%20Activated%20With%20Amoxicillin-Peptide%20Adducts&author=A%20Tailor&author=X%20Meng&author=K%20Adair&volume=178&publication_year=2020&pages=115-126&pmid=32777075&doi=10.1093/toxsci/kfaa128&)

30. Alvarez-Navarro C, Lopez de Castro JA. ERAP1 structure, function and pathogenetic role in ankylosing spondylitis and other MHC-associated diseases. Mol Immunol 2014;57:12–21.  [DOI](https://doi.org/10.1016/j.molimm.2013.06.012) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23916068/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Immunol&title=ERAP1%20structure,%20function%20and%20pathogenetic%20role%20in%20ankylosing%20spondylitis%20and%20other%20MHC-associated%20diseases&author=C%20Alvarez-Navarro&author=JA%20Lopez%20de%20Castro&volume=57&publication_year=2014&pages=12-21&pmid=23916068&doi=10.1016/j.molimm.2013.06.012&)

31. Overby CL, Hripcsak G, Shen Y. Estimating heritability of drug-induced liver injury from common variants and implications for future study designs. Sci Rep 2014;4:5762.  [DOI](https://doi.org/10.1038/srep05762) | [PMC free article](/articles/PMC4104390/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25042059/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Estimating%20heritability%20of%20drug-induced%20liver%20injury%20from%20common%20variants%20and%20implications%20for%20future%20study%20designs&author=CL%20Overby&author=G%20Hripcsak&author=Y%20Shen&volume=4&publication_year=2014&pages=5762&pmid=25042059&doi=10.1038/srep05762&)

32. O’Connor LJ. The distribution of common-variant effect sizes. Nat Genet 2021;53:1243–1249.  [DOI](https://doi.org/10.1038/s41588-021-00901-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34326547/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20distribution%20of%20common-variant%20effect%20sizes&author=LJ%20O%E2%80%99Connor&volume=53&publication_year=2021&pages=1243-1249&pmid=34326547&doi=10.1038/s41588-021-00901-3&)

33. Chalasani N, Reddy KRK, Fontana RJ, et al. Idiosyncratic Drug Induced Liver Injury in African-Americans Is Associated With Greater Morbidity and Mortality Compared to Caucasians. Am J Gastroenterol 2017;112:1382–1388.  [DOI](https://doi.org/10.1038/ajg.2017.215) | [PMC free article](/articles/PMC5667647/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28762375/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Gastroenterol&title=Idiosyncratic%20Drug%20Induced%20Liver%20Injury%20in%20African-Americans%20Is%20Associated%20With%20Greater%20Morbidity%20and%20Mortality%20Compared%20to%20Caucasians&author=N%20Chalasani&author=KRK%20Reddy&author=RJ%20Fontana&volume=112&publication_year=2017&pages=1382-1388&pmid=28762375&doi=10.1038/ajg.2017.215&)

34. Hollenbach JA, Ombrello MJ, Tremoulet AH, et al. IL-1 and IL-6 inhibitor hypersensitivity link to common HLA-DRB1*15 alleles. medRxiv 2021:2020.08.10.20172338.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=medRxiv&title=IL-1%20and%20IL-6%20inhibitor%20hypersensitivity%20link%20to%20common%20HLA-DRB1*15%20alleles&author=JA%20Hollenbach&author=MJ%20Ombrello&author=AH%20Tremoulet&publication_year=2021&)

35. Kaliyaperumal K, Grove JI, Delahay RM, et al. Pharmacogenomics of drug-induced liver injury (DILI): Molecular biology to clinical applications. J Hepatol 2018;69:948–957.  [DOI](https://doi.org/10.1016/j.jhep.2018.05.013) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29792895/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Pharmacogenomics%20of%20drug-induced%20liver%20injury%20(DILI):%20Molecular%20biology%20to%20clinical%20applications&author=K%20Kaliyaperumal&author=JI%20Grove&author=RM%20Delahay&volume=69&publication_year=2018&pages=948-957&pmid=29792895&doi=10.1016/j.jhep.2018.05.013&)

36. Robinson PC, Brown MA. The genetics of ankylosing spondylitis and axial spondyloarthritis. Rheum Dis Clin North Am 2012;38:539–53.  [DOI](https://doi.org/10.1016/j.rdc.2012.08.018) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23083754/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Rheum%20Dis%20Clin%20North%20Am&title=The%20genetics%20of%20ankylosing%20spondylitis%20and%20axial%20spondyloarthritis&author=PC%20Robinson&author=MA%20Brown&volume=38&publication_year=2012&pages=539-53&pmid=23083754&doi=10.1016/j.rdc.2012.08.018&)
