# A genome-wide association study of plasma concentrations of warfarin enantiomers and metabolites in sub-Saharan black-African patients

## Metadata
**Authors:** Innocent G Asiimwe, Marc Blockman, Karen Cohen, Clint Cupido, Claire Hutchinson, Barry Jacobson, Mohammed Lamorde, Jennie Morgan, Johannes P Mouton, Doreen Nakagaayi, Emmy Okello, Elise Schapkaitz, Christine Sekaggya-Wiltshire, Jerome R Semakula, Catriona Waitt, Eunice J Zhang, Andrea L Jorgensen, Munir Pirmohamed
**Journal:** Frontiers in Pharmacology
**Date:** 2022 Sep 23
**DOI:** [10.3389/fphar.2022.967082](https://doi.org/10.3389/fphar.2022.967082)
**PMID:** 36210801
**PMCID:** PMC9537548
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9537548/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9537548/pdf/fphar-13-967082.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9537548/pdf/fphar-13-967082.pdf)

## Abstract

Diversity in pharmacogenomic studies is poor, especially in relation to the inclusion of black African patients. Lack of funding and difficulties in recruitment, together with the requirement for large sample sizes because of the extensive genetic diversity in Africa, are amongst the factors which have hampered pharmacogenomic studies in Africa. Warfarin is widely used in sub-Saharan Africa, but as in other populations, dosing is highly variable due to genetic and non-genetic factors. In order to identify genetic factors determining warfarin response variability, we have conducted a genome-wide association study (GWAS) of plasma concentrations of warfarin enantiomers/metabolites in sub-Saharan black-Africans. This overcomes the issue of non-adherence and may have greater sensitivity at genome-wide level, to identify pharmacokinetic gene variants than focusing on mean weekly dose, the usual end-point used in previous studies. Participants recruited at 12 outpatient sites in Uganda and South Africa on stable warfarin dose were genotyped using the Illumina Infinium H3Africa Consortium Array v2. Imputation was conducted using the 1,000 Genomes Project phase III reference panel. Warfarin/metabolite plasma concentrations were determined by high-performance liquid chromatography with tandem mass spectrometry. Multivariable linear regression was undertaken, with adjustment made for five non-genetic covariates and ten principal components of genetic ancestry. After quality control procedures, 548 participants and 17,268,054 SNPs were retained. CYP2C9*8, CYP2C9*9, CYP2C9*11, and the CYP2C cluster SNP rs12777823 passed the Bonferroni-adjusted replication significance threshold (p < 3.21E-04) for warfarin/metabolite ratios. In an exploratory GWAS analysis, 373 unique SNPs in 13 genes, including CYP2C9*8, passed the Bonferroni-adjusted genome-wide significance threshold (p < 3.846E-9), with 325 (87%, all located on chromosome 10) SNPs being associated with the S-warfarin/R-warfarin outcome (top SNP rs11188082, CYP2C19 intron variant, p = 1.55E-17). Approximately 69% of these SNPs were in linkage disequilibrium (r 2 > 0.8) with CYP2C9*8 (n = 216) and rs12777823 (n = 8). Using a pharmacokinetic approach, we have shown that variants other than CYP2C9*2 and CYP2C9*3 are more important in sub-Saharan black-Africans, mainly due to the allele frequencies. In exploratory work, we conducted the first warfarin pharmacokinetics-related GWAS in sub-Saharan Africans and identified novel SNPs that will require external replication and functional characterization before they can be considered for inclusion in warfarin dosing algorithms.

Keywords: black-African, genome-wide association study, personalized medicine, pharmacokinetics, warfarin

## 1 Introduction

There is increasing acceptance of the need to increase diversity in genomic studies including in pharmacogenomics ([Sirugo et al., 2019](#B56); [Asiimwe and Pirmohamed, 2022](#B5); [Fatumo et al., 2022](#B22)). This is especially true in relation to the inclusion of black African patients, who have not been well-represented in previous studies ([Sirugo et al., 2019](#B56); [Asiimwe and Pirmohamed, 2022](#B5); [Fatumo et al., 2022](#B22)). Pharmacogenomic studies in sub-Saharan Africa have been hampered for several reasons: lack of critical mass of researchers in Africa interested in pharmacogenomics; lack of suitable funding; difficulties in recruiting patients because of the lack of infrastructure, which is especially important in pharmacogenomics, where there is often a need for deep phenotyping; and the huge genetic diversity in the African population which therefore requires large sample sizes ([Tishkoff et al., 2009](#B61); [Teo et al., 2010](#B59)).

Due to low cost, warfarin remains the oral anticoagulant of choice in sub-Saharan Africa ([Semakula et al., 2021](#B54)). However, it has several dosing challenges including a narrow therapeutic window, and wide inter-patient dose-requirement variability due to genetic and non-genetic factors. These have contributed to the low anticoagulation quality in sub-Saharan Africa ([Mouton et al., 2021](#B42)), with warfarin being the leading cause of preventable adverse drug reaction-related hospitalizations in South Africa ([Mouton et al., 2016](#B43)).

To improve warfarin dosing, many algorithms that incorporate genetic and non-genetic factors are in use ([Asiimwe et al., 2020b](#B7)). Among the most commonly included genetic factors are the single nucleotide polymorphisms (SNPs) in the gene *CYP2C9* (cytochrome P450, family 2, subfamily C, polypeptide 9) that encodes the most important warfarin metabolizing enzyme ([Asiimwe et al., 2020b](#B7)). Due to minor allele frequencies, these SNPs (specifically rs1799853, *CYP2C9*2* and rs1057910, *CYP2C9*3*) are more applicable to white and Asian populations as compared to black or other minority populations, in which other variants are believed to be more important ([Cavallari and Perera, 2012](#B13)). The United States Association for Molecular Pathology and College of American Pathologists in their joint recommendation for clinical *CYP2C9* genotyping allele selection include *CYP2C9*2*, *CYP2C9*3*, rs28371686 (*CYP2C9*5*), rs9332131 (*CYP2C9*6*), rs7900194 (*CYP2C9*8*), and rs28371685 (*CYP2C9*11*) as tier 1 variant alleles that should as a mimimum be included in clinical pharmacogenomic genotyping assays ([Pratt et al., 2019](#B48)). However, evidence for these alleles is mostly from studies conducted outside of sub-Saharan Africa ([Asiimwe et al., 2019](#B8)) and therefore more sub-Saharan African studies are required before these recommendations can be adopted in the region, and to fully characterize region-specific variants.

To identify and/or confirm genetic variants that are important in influencing warfarin dosing in patients of black-African ancestry, studies have utilised candidate gene and genome-wide approaches using stable warfarin dose as the outcome variable ([Asiimwe et al., 2019](#B8)). Alternatively, other studies (in other populations) have also used plasma concentrations of warfarin enantiomers and/or their metabolites to identify/confirm genetic variants of importance in warfarin pharmacokinetics, and consequently warfarin dosing ([Scordo et al., 2002](#B53); [Muszkat et al., 2007](#B44)). This is important for several reasons: 1) it is the concentration of a drug/metabolite in the circulation or in tissue, rather than the dose per se, that determines its pharmacologic effect; 2) adherence patterns significantly impact study power ([Mallayasamy et al., 2018](#B38)), which means very large sample sizes will be required in sub-Saharan African studies with dose as an outcome, since more than half of patients in this population consider themselves non-adherent to warfarin dosing ([Mouton et al., 2021](#B42)); and 3) the genetic diversity of African populations ([Tishkoff et al., 2009](#B61)) further increases sample size requirements. Since recruiting participants from resource-limited settings can be very challenging ([Teo et al., 2010](#B59)), a pharmacokinetic approach that directly measures drug or metabolite concentrations, and circumvents adherence issues, has some advantages as it is likely to be the most powerful and sensitive in identifying genetic variants.

Warfarin is administered as a racemic mixture of the R (+) and S (−) stereoisomers [with the S-stereoisomer being 3–5 times more potent than its counterpart: half-maximal inhibitory concentration, IC50, for vitamin K epoxide reductase complex subunit 1 (VKORC1) being 288 and 25 nM for the R- and S-stereoisomers, respectively] ([Choonara et al., 1986](#B15); [Pirmohamed et al., 2015](#B47)). Following oral administration, warfarin sodium is rapidly absorbed (79%–100% bioavailability) with considerable inter- and intra-individual variation in absorption rates. It is 99% protein-bound (and thus approximately 1% of the unbound drug is responsible for its pharmacologic effects) and has a relatively small apparent volume of distribution of 0.14 L/kg ([Bristol-Myers Squibb, 2018](#B11); [U.S. National Library of Medicine, 2018](#B64)). Warfarin has a half-life of between 20 and 60 h (R-warfarin, 37–89 h; S-warfarin, 21–43 h) and is eliminated almost entirely by metabolism. Both stereoisomers undergo cytochrome P450 (CYP450)-mediated metabolism (key enzymes and metabolites shown in [Supplementary Figure S1](#s11)) with the major metabolites and enzymes differing between the two isomers (7- and 6-hydroxywarfarin from S-warfarin by CYP2C9 versus 10-hydroxywarfarin from R-warfarin by CYP3A4) ([Shaik et al., 2016](#B55); [Bristol-Myers Squibb, 2018](#B11); [U.S. National Library of Medicine, 2018](#B64)). Since *CYP2C9* is the main metabolizing enzyme for warfarin, its clearance mainly depends on *CYP2C9* genotype and in Whites, S-warfarin clearance has been estimated to be 0.065 ml/min/kg for *CYP2C9*1*1* genotype, 0.041 ml/min/kg for **1*2/*1*3* genotypes, and 0.020 mg/min/kg for **2*2/*2*3/*3*3* genotypes ([Bristol-Myers Squibb, 2018](#B11); [U.S. National Library of Medicine, 2018](#B64)).

The clearance of R-warfarin is approximately half of that of S-warfarin ([Bristol-Myers Squibb, 2018](#B11); [U.S. National Library of Medicine, 2018](#B64)), resulting in an (S)-/(R)-warfarin ratio of approximately 0.5. A change in the activity of any of the metabolizing enzymes is likely to change this ratio and can be used as a basis of discovering new genetic variants involved in the pharmacokinetics of warfarin. In the present study, we have conducted the first genome-wide association study (GWAS) of plasma concentrations of warfarin enantiomers and metabolites in sub-Saharan black-African participants on stable warfarin dose in order to replicate previous findings and identify novel genetic variants of importance in any of the investigated warfarin pharmacokinetics-related outcomes, and consequently warfarin dosing in these populations.

## 2 Materials and methods

This study adheres to the STrengthening the Reporting Of Pharmacogenetic Studies (STROPS) guideline ([Chaplin et al., 2020](#B14)) ([Supplementary Table S1](#s11)).

### 2.1 Study design, setting and participants

This study included warfarin-treated adult (≥18 years) participants of self-reported black-African ethnicity who were recruited as part of an observational study by the National Institute of Health Research (NIHR) Global Health Research Group on WARfarin anticoagulation in PATients in Sub-Saharan Africa (War-PATH, [http://warpath.info/](http://warpath.info/); [ClinicalTrials.gov](http://ClinicalTrials.gov) Identifier: [NCT03512080](https://clinicaltrials.gov/ct2/show/NCT03512080)) as previously reported ([Asiimwe et al., 2020a](#B6)). Participants were recruited from 12 outpatient clinics and hospital departments in Uganda and South Africa between June 2018 and March 2020 ([Asiimwe et al., 2020a](#B6); [Asiimwe et al., 2021](#B4)). The studies involving human participants were reviewed and approved by institutional review boards of the University of Liverpool (UK; ref: 2934), University of Cape Town (South Africa; ref: 672/2017), and Joint Clinical Research Centre (Uganda; ref: JC3017). Work in Uganda was also approved by the Uganda National Council for Science and Technology (ref: HS164ES). The patients/participants provided their written informed consent to participate in this study.

All included participants had achieved stable warfarin dose in the year preceeding recruitment, defined as the unchanged dose for two consecutive clinic visits, with the international normalized ratio (INR) being in the therapeutic range (2.0–3.0 for those with venous thromboembolism or atrial fibrillation and 2.5–3.5 for those with valvular heart disease) at both visits. Patients who were unwilling to take part, pregnant women or patients with any other contraindications based on clinician judgement were excluded as previously detailed ([Asiimwe et al., 2020a](#B6)). Participants whose blood samples were not available at the time of DNA analysis were also excluded.

### 2.2 Variables

The study end points were: the individual and combined concentrations of the warfarin enantiomers (S-warfarin, R-warfarin, RS-warfarin), the main metabolites (S-6OH-warfarin, R-6OH-warfarin, RS-6OH-warfarin, S-7OH-warfarin, RS-10-hydroxywarfarin) and analyte ratios (S-warfarin/R-warfarin, S-6OH-warfarin/S-warfarin, R-6OH-warfarin/R-warfarin, S-7OH-warfarin/S-warfarin, RS-10hydroxywarfarin/RS-warfarin). R-4OH-warfarin and R-7OH-warfarin were not included as they were quantified in fewer than 100 participants, while enantiomers for 10-hydroxy-warfarin were not determined separately (see the “*Data sources/measurement*” section).

To replicate previous reports ([Asiimwe et al., 2019](#B8); [Pratt et al., 2019](#B48)), we first considered the SNPs that are known to be important in warfarin pharmacokinetics (*CYP2C9*6*, *CYP2C9*8*, *CYP2C9*11*, and rs12777823) as exposures. Although the rs2256871 (*CYP2C9*9*) missense variant (histidine to arginine change at position 251) has minimal effect on enzyme function ([Pratt et al., 2019](#B48)), slightly decreased activity towards S-warfarin compared with the wild-type has been reported for the protein encoded by this variant ([Niinuma et al., 2014](#B45)) and so it was also considered an additional exposure. We could not include the SNPs rs1799853 (*CYP2C9*2*), rs1057910 (*CYP2C9*3*) and rs28371686 (*CYP2C9*5*) in this analysis as they failed the imputation/r-squared and minor allele frequency (MAF) thresholds during genotype quality control (details under “*Data sources/measurement*” and “*Results*” sections). The widely known *VKORC1* (warfarin’s molecular target) and *CYP4F2* (a vitamin K oxidase) SNPs (rs7294*, VKORC1 3730G>A*; rs2359612, *VKORC1 2255C>T*; rs8050894, *VKORC1 1542G>C*; rs9934438, *VKORC1 1173C>T*; rs2884737, *VKORC1 497T>G*; rs9923231, *VKORC1 −1639G>A*; rs2108622, *CYP4F2*3*) were also included as “negative” controls since we don’t expect them to affect warfarin pharmacokinetics. All other genotyped and imputed SNPs (details under “*Data sources/measurement*”) that passed genotype quality control were considered as exposures in an exploratory GWAS analysis.

Five non-genetic covariates (age, sex, weight, simvastatin/amiodarone status, and efavirenz status) and ten principal components of genetic ancestry were considered as additional predictor variables during analysis. Four of the above non-genetic variables (age, sex, weight, and simvastatin/amiodarone status) were previously selected based on expert guidance and literature review during the development of the War-PATH clinical dose-initiation algorithm ([Asiimwe et al., 2020a](#B6)). Three other previously selected non-genetic variables [country of recruitment, target INR range, and human immunodeficiency virus (HIV) status] were not considered for the following reasons. First, country of recruitment was included as a proxy for underlying population substructure; with GWAS data available in this analysis, the ten principal components of genetic ancestry ([Price et al., 2006](#B49)) were preferable as they are more accurate. Second, a higher target INR range implies a stronger pharmacodynamic effect is required, which does not support the adjustment of this variable in pharmacokinetic analysis. Lastly, HIV infection affects warfarin response through pharmacodynamics (e.g., leading to a hypercoaguable state) or pharmacokinetics (interactions with antiretroviral drugs) ([Asiimwe et al., 2020a](#B6)). In this pharmacokinetic study, exploring efavirez status [efavirenz is predicted to affect warfarin’s concentration; moderate severity ([Joint Formulary Committee, 2019](#B29))] as a covariate was therefore more suitable than HIV status. Other antiretrovirals can affect warfarin concentrations ([Liedtke and Rathbun, 2009](#B35)), however, they were not considered as covariates as they were not taken or were taken by very few patients.

### 2.3 Data sources/measurement

#### 2.3.1 Warfarin and metabolite concentrations

After K3-EDTA plasma samples were acquired during patient enrolment, a chiral high-performance liquid chromatography-tandem mass spectrometry (HPLC-MS/MS) assay was developed and validated to quantify warfarin and its five major metabolites (4′-, 6-, 7-, 8-, and 10-OH warfarin). The assay was optimized to detect R-warfarin, S-warfarin, R-4-OH-warfarin, S-4-OH-warfarin, R-6-OH-warfarin, S-6-OH-warfarin, R-7-OH-warfarin, S-7-OH-warfarin, R-8-OH-warfarin, S-8-OH-warfarin, and racemic 10-OH warfarin ([Supplementary Figure S2](#s11)). Full validation criteria were met for calibration curve performance, carryover, selectivity, dilution integrity, matrix effect, extraction recovery, accuracy, precision, plasma stability, and autosampler stability. Results of the bioanalytical method validation are summarised in [Supplementary Table S2](#s11). The within run and between run accuracy of the lower limit of quantification (LLOQ), low, medium and high quality controls for all analytes were within the accepted range, precision coefficient of variation (CV) was 1.3%–12.7%. After 4-fold dilution with blank plasma, the accuracy and precision CV of all analytes was 94.7%–107.0% and 0.7%–5.0% respectively; as such patient samples higher than the upper limit of quantification (ULOQ) can be diluted up to four times. All analytes were stable in plasma on the benchtop (9 h, room temperature), in the fridge (24 h, 4–8°C), after five freeze-thaw cycles (storage at approximately −80°C), and in the autosampler (30 h, set at 4°C). Other details (chromatographic solvents and buffers, sample preparation and data acquisition methods) are shown in [Supplementary Text S1](#s11) and [Supplementary Table S3](#s11).

Raw chromatographic data were processed using the Analyst® 1.6.2 and MultiQuant™ version 3.0 (AB Sciex, UK) softwares. Accuracy ranges were set as ±20% for the lowest concentration (LLOQ) and ±15% for all other concentrations in the calibration curve, including the top calibration standard (ULOQ). All analytical runs had a minimum of seven calibration standards, with over 88% of the calibration standards fulfilling the above-mentioned accuracy range. A minimum of three quality control concentration levels (low, medium, high) were included per run with all calibration standards and quality controls run in duplicate. The dynamic range for the parent compounds was 200–8,000 ng/ml. For 10-OH-warfarin, the dynamic range was 10 times lower, 20–800 ng/ml. All other metabolites had LLOQ and ULOQ of 25 ng/ml and 1,000 ng/ml respectively. Samples with values below the LLOQ or above the ULOQ were excluded from analysis.

#### 2.3.2 Clinical data

Data pertaining to age, sex, weight, simvastatin/amiodarone and efavirenz prescription were captured using War-PATH study case report forms during enrolment as detailed in our previous report ([Asiimwe et al., 2020a](#B6)).

#### 2.3.3 DNA extraction and genotyping

During enrolment, 5 ml whole blood was drawn in K3-EDTA vacutainers from each participant. To inactivate any viable *Mycobacterium tuberculosis* and/or human immunodeficiency virus present in the blood samples, chemical and heat inactivation was carried out using 1.5 volumes of 2X lysis buffer (PerkinElmer chemagen Technologie GmbH, UK) at 58°C for 45 min in a biosafety level 3 laboratory prior to DNA extraction (University of Liverpool, United Kingdom). DNA was subsequently extracted from the inactivated blood samples using the chemagic Magnetic Separation Module (MSM) I instrument (PerkinElmer chemagen Technologie GmbH, UK) according to the manufacturer’s guidelines in a biosafety level 2 laboratory (University of Liverpool, United Kingdom). Extracted DNA samples that showed good quality with A260/A280 ratios between 1.8 and 2.0 were normalised (50 ng/µl) and shipped to the Cambridge Genomics Services, a genotyping service provider at the University of Cambridge (Cambridge, United Kingdom). A minimum of 2 μg DNA was used for genome-wide genotyping on the Illumina Infinium H3Africa Consortium Array v2 containing 2,271,503 SNPs according to the manufacturer’s instructions (Illumina, San Diego, CA, United States). Genotyping personnel were not aware of the outcome status of included patients. To ensure quality assurance, at least five duplicates were included in each 96-well plate.

#### 2.3.4 Genotyping quality control and imputation

Prior to analyses of association, both per patient and per SNP quality control criteria were applied to the genotype data. Patients were excluded from analysis if they failed to meet the following criteria:

On the other hand, SNPs were excluded if:

Unless otherwise stated, all QC analysis (details of commands used in [Supplementary Text S2](#s11)) were conducted using PLINK v1.9 ([Purcell et al., 2007](#B51)). After the above QC steps, the accuracy relative to the 1,000 Genomes phase III reference panel ([Genomes Project et al., 2010](#B25)) was checked to minimize imputation errors, following the steps available on the Michigan imputation server ([https://imputationserver.sph.umich.edu/index.html#!](https://imputationserver.sph.umich.edu/index.html)) ([Das et al., 2016](#B17)). Genotype imputation was conducted using the same server, with pre-phasing and imputation being conducted using SHAPEIT v2 ([Delaneau et al., 2011](#B18)) and IMPUTE2 ([Howie et al., 2009](#B28)) software, respectively. Finally, post-imputation QC involved filtering out SNPs with low imputation accuracies (those with an r-squared ([Browning and Browning, 2009](#B12)) < 0.3 excluded) and low MAF (those with MAF <0.01 excluded).

### 2.4 Study size calculation

No formal sample size calculations were conducted. However, all eligible participants were included in the analysis to maximize the replication/discovery sample sizes, with a minimum sample size of 100 participants (default internal limit of SNPTEST Version 2 ([Marchini and Howie, 2010](#B39)) which was used for analysis).

### 2.5 Statistical methods

#### 2.5.1 Outcome transformation

We logarithmically transformed the outcomes to achieve a normal distribution as well as to obtain a proportional/multiplicative scale that is easy to interpret ([Keene, 1995](#B31); [Vittinghoff et al., 2012](#B66)).

#### 2.5.2 Handling quantitative predictors

Quantitative predictor variables were neither transformed nor categorized.

#### 2.5.3 Missing data

Missing genotype data was imputed using the Michigan imputation server ([Das et al., 2016](#B17)) as described above, using IMPUTE2 ([Howie et al., 2009](#B28)). For consistency with previous work ([Asiimwe et al., 2020a](#B6)), missing weight information (*n* = 11 cases, 2%) was imputed using single imputation (predictive mean matching) within the Multivariate Imputation by Chained Equations (MICE) R package ([van Buuren and Groothuis-Oudshoorn, 2011](#B65)) [based on non-genetic covariates (including age, weight, sex, target INR, HIV status, simvastatin/amiodarone status), additional genetic covariates (ten principal components of genetic ancestry) and outcome (weekly stable warfarin dose)]. We included weekly stable warfarin dose in the imputation model since including outcomes such as stable warfarin dose is preferred over their non-inclusion ([Moons et al., 2006](#B41)).

#### 2.5.4 Analysis

Following genotype QC, a multivariable linear regression model was fitted with each SNP in turn as a predictor variable assuming an additive mode of inheritance, and each of the above mentioned study endpoints (warfarin concentrations, metabolite concentrations and analyte ratios) as the outcomes. We adjusted for ten principal components of genetic ancestry as well as age, sex, weight, and intake of simvastatin, amiodarone and/or efavirenz. For the replication analysis of known variants, we used a Bonferroni-adjusted replication significance threshold (0.05 divided by 12 SNPs divided by 13 outcomes = *p* < 3.21 × 10^−4^).

We used two statistical significance thresholds for the exploratory GWAS analysis: a Bonferroni multiple testing-corrected genome-wide statistical significance threshold of *p* < 5 × 10^−8^/13 outcomes = *p* < 3.846 × 10^−9^ (to reduce false positives), as well as a nominal significance threshold of *p* < 1 × 10^−5^. We conducted two sensitivity analyses: a complete case analysis in which the 11 cases missing weight were excluded from analysis (to check the accuracy of the single-imputation approach for weight), and a model-based analysis (to check if multi-marker models would detect signals that single marker analysis fails to detect in complex traits) ([Frommlet et al., 2012](#B24); [Dolejsi et al., 2014](#B20); [Hofer et al., 2017](#B27)). Model-based linear regression analysis was implemented using the software package MOSGWA (Model Selection for Genom-Wide Associations, [http://mosgwa.sourceforge.net](http://mosgwa.sourceforge.net/)), which is based on a modification of the Bayesian Information Criterion ([Dolejsi et al., 2014](#B20)), using default parameters. Other analyses were undertaken using SNPTEST Version 2 ([Marchini and Howie, 2010](#B39)) using threshold genotypes (default calling threshold of 0.9) and the results graphically presented using Manhattan and Quantile-Quantile plots [qqman R package ([Turner, 2018](#B63))]. The Single Nucleotide Polymorphism Database (dbSNP, [https://www.ncbi.nlm.nih.gov/snp/](https://www.ncbi.nlm.nih.gov/snp/)) ([Smigielski et al., 2000](#B57); [Kitts and Sherry, 2011](#B33)) was used to obtain the gene and location/functional consequences of SNPs passing the nominal significance threshold, while the Genotype-Tissue-Expression (GTEx) analysis release V8 ([https://www.gtexportal.org/home/](https://www.gtexportal.org/home/)) ([Consortium, 2013](#B16)) was used to obtain expression quantitative trait loci (eQTL) and splicing quantitative trait loci (sQTL) for SNPs passing the Bonferroni-adjusted genome-wide statistical significance threshold. Regions of potential genomic interest were further investigated using regional locus plots (LocusZoom v0.4.8, [http://locuszoom.org/](http://locuszoom.org/)) ([Pruim et al., 2010](#B50)) while Haploview (v4.2) ([Barrett et al., 2005](#B10)) was used for linkage disequilibrium analysis (*r* ^2^ threshold = 0.8). Key statistical analysis codes/commands are available in [Supplementary Text S2](#s11).

## 3 Results

### 3.1 Participants

Out of 638 eligible black-African participants recruited from Uganda and South Africa between June 2018 and March 2020, 561 samples had been shipped to the University of Liverpool by the time of analysis. During per-sample/per-individual quality control (QC), one individual was excluded due to discordant clinical/X-chromosome-derived sex, two individuals due to having genotype call rates <95%, one due to being an ethnic outlier and nine due to having an identity-by-descent coefficient greater than 0.1875 ([Figure 1](#F1); [Supplementary Figure S3](#s11)) leaving a total of 548 individuals, whose characteristics are shown in [Table 1](#T1). Only the participants with warfarin/metabolite concentrations within the dynamic range were included in the corresponding analyses, with the sample sizes ranging from 233 (RS-6OH-warfarin endpoint) to 531 (S-warfarin endpoint) as shown in [Figure 1](#F1) and [Table 1](#T1). [Supplementary Figure S4](#s11) shows the correlations between the study end-points while [Supplementary Figure S5](#s11) shows quantile–quantile plots before and after applying a logarithmic transformation to the study end-points. A comparison of participants who were included versus those excluded from a particular analysis is shown in [Supplementary Table S4](#s11).

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a306/9537548/35ad84b2e0d8/fphar-13-967082-g001.jpg)

Flow chart for included participants. aOnly patients who were within the dynamic range included in analysis. IBD = identity-by-descent.

### TABLE 1.

| Variables | Frequency (%) or median (IQR) |
| --- | --- |
| Country of recruitment |  |
| South Africa | 287 (52.37%) |
| Uganda | 261 (47.63%) |
| Age (years) | 45.62 (34.78–57.00) |
| Sex |  |
| Female | 390 (71.17%) |
| Male | 158 (28.83%) |
| Weight (kg, n = 537) | 72.00 (60.00–87.00) |
| INR Target range |  |
| 2.0–3.0 | 357 (65.15%) |
| 2.5–3.5 | 191 (34.85%) |
| HIV status |  |
| Negative | 428 (78.10%) |
| Positive | 97 (17.70%) |
| Unknown | 23 (4.20%) |
| Efavirenz |  |
| Yes | 65 (11.86%) |
| No | 483 (88.14%) |
| Simvastatin/Amiodarone |  |
| Yes | 51 (9.31%) |
| No | 497 (90.69%) |
| Weekly stable dose (mg) | 35.00 (30.00–50.00) |
| Analyte concentrations (ng/ml) |  |
| S-warfarin (n = 531) | 1811.17 (1,327.28–2,367.56) |
| R-warfarin (n = 530) | 3,160.27 (2,304.51–4,199.45) |
| RS-warfarin (n = 524) | 5,073.16 (3,825.40–6,316.03) |
| S-6OH-warfarin (n = 260) | 37.03 (30.46–49.62) |
| R-6OH-warfarin (n = 377) | 60.21 (40.37–101.77) |
| RS-6OH-warfarin (n = 233) | 118.62 (85.93–175.74) |
| S-7OH-warfarin (n = 514) | 274.23 (160.70–444.31) |
| RS-10-hydroxywarfarin (n = 492) | 64.55 (42.79–106.12) |
| Analyte concentration ratios |  |
| S-warfarin/R-warfarin (n = 524) | 0.56 (0.43–0.73) |
| S-6OH-warfarin/S-warfarin (n = 258) | 0.02 (0.01–0.03) |
| R-6OH-warfarin/R-warfarin (n = 369) | 0.02 (0.01–0.03) |
| S-7OH-warfarin/S-warfarin (n = 510) | 0.15 (0.09–0.26) |
| RS-10hydroxywarfarin/RS-warfarin (n = 481) | 0.01 (0.01–0.02) |

Table 1 Caption: Clinical/demographic characteristics of the participants who passed quality control procedures (N = 548).

### 3.2 Single nucleotide polymorphisms

A total of 2,271,503 SNPs were genotyped using the Illumina Infinium H3Africa Consortium Array v2. Per-SNP QC excluded 344,443 SNPs based on genotype success rate <95% (*n* = 41,460), Hardy-Weinberg Equilibrium *p* value < 0.000001 (*n* = 4,215), and minor allele frequency <0.01 (*n* = 298,768) resulting in a total of 1,927,060 SNPs remaining. After more SNPs were added during the imputation process (and after applying post-imputation QC steps), a total of 17,268,054 SNPs were included in the final analysis.

### 3.3 Single nucleotide polymorphisms known to influence warfarin pharmacokinetics

[Table 2](#T2) shows the three outcomes (S-warfarin/R-warfarin, S-6OH-warfarin/S-warfarin, S-7OH-warfarin/S-warfarin) for which we were able to replicate previous reports. Specifically, *CYP2C9*8* (S-warfarin/R-warfarin *p* = 9.53 × 10^−12^, S-7OH-warfarin/S-warfarin *p* = 6.68 × 10^−5^), *CYP2C9*9* (S-warfarin/R-warfarin *p* = 8.54 × 10^−8^, S-6OH-warfarin/S-warfarin *p* = 1.70 × 10^−4^), and *CYP2C9*11* (S-warfarin/R-warfarin *p* = 9.00 × 10^−9^, S-7OH-warfarin/S-warfarin *p* = 1.00 × 10^−5^) were shown to significantly influence warfarin pharmacokinetics after adjustment for multiple testing (Bonferroni-adjusted replication significance threshold *p* < 3.21 × 10^−4^). The *CYP2C* cluster SNP rs12777823 was also replicated based on the outcomes S-warfarin/R-warfarin (*p* = 3.61 × 10^−8^) and S-7OH-warfarin/S-warfarin (*p* = 5.07 × 10^−7^). As expected, none of the negative controls (using the known *VKORC1* and *CYP4F2* SNPs) were statistically significant. The other ten outcomes that did not have any statistically significant results are shown in [Supplementary Table S5](#s11).

### TABLE 2.

| # | rsID (Reference/alternative alleles) | Common name | S-warfarin/R-warfarin (N = 524) |  |  | S-6OH-warfarin/S-warfarin (N = 258) |  |  | S-7OH-warfarin/S-warfarin (N = 510) |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| MAF | Beta b (SE) | p-value | MAF | Beta b (SE) | p-value | MAF | Beta b (SE) | p-value |  |  |  |
| 1 | rs12777823 (G/A) | NA | 0.279 | 0.330 (0.059) | 3.61E-08 | 0.248 | −0.257 (0.103) | 1.31E-02 | 0.278 | −0.314 (0.062) | 5.07E-07 |
| 2 | rs9332131 (GA/G) | CYP2C9*6 | 0.010 | −0.320 (0.270) | 2.37E-01 | 0.012 | 0.223 (0.408) | 5.85E-01 | 0.010 | −0.160 (0.290) | 5.80E-01 |
| 3 | rs7900194 (G/A) | CYP2C9*8 | 0.071 | 0.698 (0.100) | 9.53E-12 | 0.054 | −0.267 (0.191) | 1.62E-01 | 0.072 | −0.427 (0.106) | 6.68E-05 |
| 4 | rs2256871 (A/G) | CYP2C9*9 | 0.168 | 0.372 (0.068) | 8.54E-08 | 0.159 | −0.454 (0.119) | 1.70E-04 | 0.172 | −0.205 (0.072) | 4.49E-03 |
| 5 | rs28371685 (C/T) | CYP2C9*11 | 0.022 | 1.063 (0.182) | 9.00E-09 | 0.010 | −1.436 (0.429) | 9.47E-04 | 0.022 | −0.861 (0.193) | 1.00E-05 |
| 6 | rs7294 (G/A) | VKORC1 3730G>A | 0.483 | 0.028 (0.058) | 6.32E-01 | 0.440 | −0.111 (0.090) | 2.20E-01 | 0.484 | −0.055 (0.060) | 3.61E-01 |
| 7 | rs2359612 (C/T) | VKORC1 2255C>T | 0.222 | −0.075 (0.066) | 2.59E-01 | 0.238 | −0.136 (0.100) | 1.77E-01 | 0.223 | −0.047 (0.068) | 4.89E-01 |
| 8 | rs8050894 (G/C) | VKORC1 1542G>C | 0.204 | −0.148 (0.067) | 2.88E-02 | 0.194 | 0.304 (0.112) | 6.87E-03 | 0.202 | 0.078 (0.071) | 2.74E-01 |
| 9 | rs9934438 (C/T) | VKORC1 1173C>T | 0.053 | −0.104 (0.125) | 4.05E-01 | 0.048 | 0.656 (0.203) | 1.39E-03 | 0.052 | 0.215 (0.131) | 1.01E-01 |
| 10 | rs2884737 (T/G) | VKORC1 497T>G | 0.019 | −0.307 (0.200) | 1.26E-01 | 0.019 | 0.527 (0.312) | 9.21E-02 | 0.018 | 0.321 (0.215) | 1.35E-01 |
| 11 | rs9923231 (G/A) | VKORC1 -1639G>A | 0.052 | −0.105 (0.126) | 4.04E-01 | 0.047 | 0.679 (0.207) | 1.19E-03 | 0.051 | 0.218 (0.132) | 9.92E-02 |
| 12 | rs2108622 (C/T) | CYP4F2*3 | 0.063 | 0.022 (0.116) | 8.50E-01 | 0.058 | −0.393 (0.191) | 4.10E-02 | 0.066 | −0.030 (0.119) | 8.02E-01 |

Table 2 Caption: P-values for widely-known SNPs a .

### 3.4 Exploratory genome-wide association analysis

[Figure 2](#F2) shows the Manhattan plots for the individual and combined concentrations of the warfarin enantiomers (S-warfarin, R-warfarin, RS-warfarin) as well as the key metabolites (S-6OH-warfarin, R-6OH-warfarin, RS-6OH-warfarin, S-7OH-warfarin, RS-10-hydroxywarfarin) while [Figure 3](#F3) shows the Manhattan plots for the analyte ratios (S-warfarin/R-warfarin, S-6OH-warfarin/S-warfarin, R-6OH-warfarin/R-warfarin, S-7OH-warfarin/S-warfarin, RS-10hydroxywarfarin/RS-warfarin). The corresponding quantile-quantile (QQ) plots showing limited evidence of genomic inflation are shown in [Supplementary Figure S6](#s11).

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a306/9537548/43fe7a562a73/fphar-13-967082-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9537548_fphar-13-967082-g002.jpg)

Manhattan plots of warfarin enantiomers and metabolites. Genome-wide association analyses were carried out using natural logarithm transformed analyte concentrations, adjusted for age, sex, weight, simvastatin/amiodarone and efavirenz statuses, and ten principal components by frequentist association testing assuming an additive model of inheritance.

### FIGURE 3.

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a306/9537548/8ff2a1a33092/fphar-13-967082-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9537548_fphar-13-967082-g003.jpg)

Manhattan plots of analyte (warfarin enantiomers, metabolites) ratios. Genome-wide association analyses were carried out using natural logarithm transformed analyte concentration ratios, adjusted for age, sex, weight, simvastatin/amiodarone status and efavirenz statuses, and ten principal components by frequentist association testing assuming an additive model of inheritance. The top SNPs already known to significantly influence warfarin pharmacokinetics are annotated.

[Supplementary Tables S6–S18](#s11) show the SNPs with *p*-values lower than the nominal significance threshold (*p* < 1 × 10^−5^, 4,970 SNPs for all outcomes, with 4,524 SNPs being unique as some SNPs appeared in more than one outcome). Of these, 373 unique SNPs in 13 genes were genome-wide significant hits (*p* < 3.846 × 10^−9^, one SNP appeared in two outcomes), as detailed below:

### FIGURE 4.

![FIGURE 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a306/9537548/a548b1ada14c/fphar-13-967082-g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9537548_fphar-13-967082-g004.jpg)

Regional LocusZoom plot of the established CYP2C9 SNP rs7900194 (CYP2C9*8). The linkage disequilibrium (LD) pattern is based on the 1,000 genomes African populations (Genomes Project et al., 2010). In this study, most of the SNPs shown above to have an r 2 between 0.6 and 0.8 (orange circles) were in LD with rs7900194 (r 2 > 0.8).

Seventy-nine (21.2%) of the 373 unique genome-wide significant SNPs had statistically significant expression quantitative trait loci (eQTLs, [Supplementary Table S20](#s11)), including eight SNPs (rs7085563, rs12570829, rs34582766, rs12775423, rs12782132, rs7085420, rs1926711, rs35835168) that are significantly associated with the hepatic expression of *CYP2C19* (*p*-values ranging from 3.40 × 10^−6^ to 4.50 × 10^−5^). On the other hand, 75 (20.1%) of the 373 unique genome-wide significant SNPs had statistically significant splicing quantitative trait loci (sQTLs, [Supplementary Table S21](#s11)), including 30, 8, 8, and 4 SNPs significantly associated with the hepatic splicing of *CYP2C8*, *CYP2C18*, *CYP2C19*, and *CYP2C9* mRNAs, respectively (all *p*-values < 4.70 × 10^−7^).

Consistent with a slightly smaller sample size, most SNPs had less precise estimates (larger standard errors) in the complete-case analysis that was conducted in order to determine the robustness of the single imputation approach used to infer missing weight for 11 cases. This resulted in slightly less genome-wide significant SNPs (*n* = 362 unique SNPs, [Supplementary Table S22](#s11)). Lastly, in the model-based analysis, nine SNPs that had all been detected by single-marker analysis ([Supplementary Tables S9, S11, S14, S17](#s11)) were included in the final models for four of the outcomes, namely: S-6OH-warfarin (4 SNPs), RS-6OH-warfarin (1 SNP), S-warfarin/R-warfarin (3 SNPs) and S-7OH-warfarin/S-warfarin (1 SNP) as shown in [Supplementary Table S23](#s11).

## 4 Discussion

In this study, we have undertaken a GWAS using plasma concentrations and ratios of warfarin and its metabolites in a sub-Saharan black African population to identify genetic determinants of warfarin pharmacokinetics, and thereby dose and response. Candidate gene studies and GWAS of plasma concentrations have been undertaken previously for a number of other drugs including acetaminophen, atorvastatin, bisoprolol, antidepressants and antipsychotics ([Turner et al., 2020](#B62); [Jukic et al., 2021](#B30); [Milosavljevic et al., 2021](#B40); [Fontana et al., 2022](#B23); [Thareja et al., 2022](#B60)). This has many advantages, as outlined in the introduction, but has disadvantages as well, including the fact that the approach increases the complexity and cost of studies, there is a need to set up validated drug/metabolite concentration analytical techniques and because of the rarity of such studies, it is more difficult to identify independent cohorts for replication.

Our report is the first warfarin pharmacokinetics-related genome-wide association study (GWAS) in sub-Saharan African individuals. We have undertaken a two-step analysis: a replication analysis, focusing on *CYP2C9* variants and the *CYP2C* gene cluster allele rs12777823 which are well known determinants of warfarin dose requirements, and an exploratory GWAS analysis. *CYP2C9* is the most important gene involved in the metabolism of S-warfarin, the more potent enantiomer. Reduced/null function variants (such as *2, **3*, **5*, **6*, **8*, and **11*) ([Rettie et al., 1994](#B52); [Haining et al., 1996](#B26); [Dickmann et al., 2001](#B19); [Kidd et al., 2001](#B32); [Allabi et al., 2004](#B1); [Tai et al., 2005](#B58); [Liu et al., 2012](#B36); [Niinuma et al., 2014](#B45); [Wanounou et al., 2022](#B67)) lead to reduced S-warfarin metabolism, which increases the S-warfarin/R-warfarin ratio (positive beta coefficients, [Table 2](#T2)), as observed for *CYP2C9*8* (*p* = 9.53 × 10^−12^) and *CYP2C9*11* (*p* = 9.00 × 10^−9^). Consistent with the above results, the S-7OH-warfarin/S-warfarin ratio for these two variants decreased (negative beta coefficients ([Table 2](#T2)) are consistent with decreased metabolite and increased substrate amounts) with both results passing the Bonferroni-adjusted replication significance threshold (*p* < 3.21 × 10^−4^). The *CYP2C9*2* variant was neither genotyped nor successfully imputed (*R* ^2^ = 22.3%) and so could not be included in analysis. Based on the 1,000 genomes African MAF of 0.8% for this variant ([Genomes Project et al., 2010](#B25)), it is likely that it would still have been excluded from analysis had it been genotyped/successfully imputed due to failing post imputation quality control checks (MAF <1%). Indeed, the latter was the reason for excluding *CYP2C9*3* (MAF in the 548 patients passing quality control procedures = 0.9%) and *CYP2C9*5* (MAF = 0.7%) variants. The very low MAF (1%) for *CYP2C9*6* could explain its non-significant associations (e.g., *p* = 0.237 for the S-warfarin/R-warfarin outcome). As previously reported ([Niinuma et al., 2014](#B45)), *CYP2C9*9* had decreased ezyme activity towards S-warfarin with *p*-values being significant for S-warfarin/R-warfarin (*p* = 8.54 × 10^−8^, beta = 0.372) and S-6OH-warfarin/S-warfarin (*p* = 1.70 × 10^−4^, beta = −0.454). These results require further study as the histidine to arginine change at position 251 that defines this variant should have minimal effect on enzyme function ([Pratt et al., 2019](#B48)). Lastly, the *CYP2C* gene cluster variant rs12777823 (MAF 27.9%) was also replicated for S-warfarin/R-warfarin (*p* = 3.61 × 10^−8^) and S-7OH-warfarin/S-warfarin (*p* = 5.07 × 10^−7^). Although this variant has been previously associated with warfarin clearance in blacks ([Perera et al., 2013](#B46)), it is thought that this is because it is in linkage disequilibrium (LD) with an unknown causal variant as this effect has not been observed in other populations where it is common in the population (European MAF = 15.1%, East Asian MAF = 31.4%) ([Genomes Project et al., 2010](#B25)).

In the exploratory GWAS, 373 unique SNPs in 13 genes passed the Bonferroni-adjusted genome-wide significance threshold (*p* < 3.846 × 10^−9^), with most (*n* = 325, 87%) SNPs being associated with the S-warfarin/R-warfarin outcome. The functional relevance of more than a third (69%) of these SNPs could be due to LD (*r* ^2^ = 0.8) with the widely-known variants *CYP2C9*8* (*n* = 216) and rs12777823 (*n* = 8). Two other biologically plausible loci were the *CYP2C9* 3′ untranslated region (UTR) SNP rs9332241 (*p*-value with S-warfarin/R-warfarin outcome = 5.38 × 10^−10^) and the *CYP2C18* stop-gained SNP rs41291550 (*p* = 3.70 × 10^−9^). The 3′ UTR SNP rs9332241 could play a role in gene silencing either by translational repression or by mRNA degradation through microRNA regulation. The stop-gained SNP rs41291550 was not associated with any literature in the National Library of Medicine’s National Centre for Biotechnological Information (NCBI) SNP database and its clinical significance is still unknown. It is mentioned in a cancer-related publication ([Li et al., 2019](#B34)), however, it is listed as non-cancer promoting and it is not linked to any disease. Additionally, the undetectable hepatic expression of CYP2C18 at protein level may mean this SNP plays little role in the metabolism of warfarin ([Lofgren et al., 2008](#B37); [Esteban et al., 2020](#B21)).

Seventy-nine (21%) SNPs had statistically significant expression quantitative trait loci (eQTLs) with eight SNPs potentially regulating *CYP2C19* hepatic expression. However, and compared to *CYP2C9* (metabolises the more potent warfarin enationer), *CYP2C19* is less important in warfarin’s metabolism as it metabolises the less potent counterpart (R-warfarin) into minor metabolites (R-6-OH-warfarin and R-8-OH-warfarin, [Supplementary Figure S1](#s11)). Lastly, 75 (20%) SNPs had statistically significant splicing quantitative trait loci (sQTLs), including some significantly associated with the hepatic splicing of *CYP2C8* (*n* = 30), *CYP2C18* (*n* = 8), *CYP2C19* (*n* = 8) and *CYP2C9* (*n* = 4) mRNAs. Alternative Splicing of these cytochrome P450 mRNAs can result in protein isoforms with altered function including lack of enzymatic activity ([Ariyoshi et al., 2007](#B3); [Annalora et al., 2017](#B2)). Since the liver is the primary metabolism site for warfarin, these hepatic eQTLs and sQTLs are biologically plausible loci. Further study (external replication and functional characterization) of these SNPs is required for full understanding.

Our study has limitations. For SNPs with low MAFs, our sample size was not large enough (despite having recruited over 500 patients) to replicate some previously reported associations such as with *CYP2C9*3, CYP2C9*5* and *CYP2C9*6*. As explained in the introduction, recruiting participants from areas where pharmacogenomic-related clinical research is in its early infancy is challenging ([Teo et al., 2010](#B59)) and this is complicated by the extreme genetic diversity of African populations ([Tishkoff et al., 2009](#B61)) which necessitates very large sample sizes. Given the paucity of pharmacogenomic evidence in sub-Saharan Africa, studies like ours are important since they help stimulate further studies through the setting up of clinical research infrastructure and changing researcher/clinician/patient perspectives of pharmacogenomic research. Additionally, results from studies such as ours can be combined with future studies through meta-analysis. Due to high correlation between some of the study end-points ([Supplementary Figure S4](#s11)), we may have used a very strict genome-wide significant threshold (*p* < 3.846 × 10^−9^) by applying a Bonferroni adjustment. However, we have also reported all nominally-significant (*p* < 1 × 10^−5^) SNPs, and these include all the signals that our strict threshold may have missed. We did not measure the time between the last dose and blood sampling. However, warfarin is usually given in the evenings and INR readings are usually taken in the morning for the majority of patients. We also included patients on stable warfarin dose, which means that they had reached steady state and together with the long half-life of warfarin (between 20 and 60 h) meant that the concentrations of the parent compounds (and metabolites) were more or less consistent. Additionally, fluctuating concentrations were less relevant to the ratio outcomes, including the S-warfarin/R-warfarin outcome, which produced the most significant hits. Another limitation is that we only included patients from Uganda and South Africa which means our results may not be generalizable to the rest of sub-Saharan Africa given the very high within-population genetic diversity of African populations ([Tishkoff et al., 2009](#B61)). Our cohort also excluded patients who had not yet achieved a stable warfarin dose, and children, which limits generalizability to these population categories. Finally, we do not have a replication cohort for the GWAS hits outside the *CYP2C* gene cluster, and thus our findings are exploratory in nature.

In conclusion, we have provided further evidence to confirm the role of *CYP2C9* in warfarin dosing and shown that variants other than *CYP2C9*2* and *CYP2C9**3 are more important in sub-Saharan black-Africans. In exploratory work, we have conducted the first warfarin pharmacokinetics-related GWAS in sub-Saharan African populations from Uganda and South Africa, and identified novel SNPs that will require external replication and functional characterization before they can be clinically-implemented. We have used a pharmacokinetic approach as it is more powerful compared to the commonly-used stable dose approach. A stable dose GWAS is nevertheless planned and will have a larger sample size to take into account the genetic diversity in the region, and will include patients of mixed-ancestry.

## Acknowledgments

We thank staff of Uganda Heart Institute, Mulago National Referral Hospital, Uganda Heart Institute, St Francis Hospital Nsambya, Infectious Diseases Institute Adult HIV clinic, Charlotte Maxeke Johannesburg Academic Hospital, Groote Schuur Hospital, Gugulethu Community Health Centre, Khayelitsha Site B Community Health Centre, Mfuleni Community Health Centre, Michael Mapongwana Community Health Centre, Nolungile Community Health Centre, and Victoria Hospital for their support during data collection. IA thanks the University of Liverpool for studentship funding support.

## Data availability statement

The derived meta-analysis summary statistics generated in this study have been deposited in the GWAS catalogue under accession code GCP000391 ([https://www.ebi.ac.uk/gwas/](https://www.ebi.ac.uk/gwas/)). The accession numbers for each of the study endpoints are found in [Supplementary Table S24](#s11).

## Ethics statement

The studies involving human participants were reviewed and approved by Institutional review boards of the University of Liverpool (United Kingdom; ref: 2934), University of Cape Town (South Africa; ref: 672/2017), and Joint Clinical Research Centre (Uganda; ref: JC3017). Work in Uganda was also approved by the Uganda National Council for Science and Technology (ref: HS164ES). The patients/participants provided their written informed consent to participate in this study.

## Author contributions

IA, EZ, AJ, and MP wrote the manuscript. IA, MB, KC, CH, ML, JM, CS-W, JS, CW, EZ, AJ, and MP designed the research. IA, MB, KC, CC, CH, BJ, ML, JM, JPM, DN, EO, ES, CS-W, JS, CW, EZ, AJ, and MP performed the research. IA analysed the data. MB, KC, CC, BJ, ML, JM, JPM, DN, EO, ES, CS-W, JS, and CW revised the manuscript and contributed data.

## Funding

This research was funded by the National Institute for Health Research (NIHR) (ref: 16/137/101) using UK aid from the UK Government to support global health research. The views expressed in this publication are those of the author(s) and not necessarily those of the NIHR or the UK government.

## Conflict of interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Publisher’s note

All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.

## Supplementary material

The Supplementary Material for this article can be found online at: [https://www.frontiersin.org/articles/10.3389/fphar.2022.967082/full#supplementary-material](https://www.frontiersin.org/articles/10.3389/fphar.2022.967082/full#supplementary-material)

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The derived meta-analysis summary statistics generated in this study have been deposited in the GWAS catalogue under accession code GCP000391 ([https://www.ebi.ac.uk/gwas/](https://www.ebi.ac.uk/gwas/)). The accession numbers for each of the study endpoints are found in [Supplementary Table S24](#s11).

### Supplementary Materials

### Data Availability Statement

The derived meta-analysis summary statistics generated in this study have been deposited in the GWAS catalogue under accession code GCP000391 ([https://www.ebi.ac.uk/gwas/](https://www.ebi.ac.uk/gwas/)). The accession numbers for each of the study endpoints are found in [Supplementary Table S24](#s11).

## References

1. Allabi A. C., Gala J. L., Horsmans Y., Babaoglu M. O., Bozkurt A., Heusterspreute M., et al. (2004). Functional impact of CYP2C95, CYP2C96, CYP2C98, and CYP2C911 in vivo among black Africans. Clin. Pharmacol. Ther. 76, 113–118. 10.1016/j.clpt.2004.04.001  [DOI](https://doi.org/10.1016/j.clpt.2004.04.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15289788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Functional%20impact%20of%20CYP2C95,%20CYP2C96,%20CYP2C98,%20and%20CYP2C911%20in%20vivo%20among%20black%20Africans&author=A.%20C.%20Allabi&author=J.%20L.%20Gala&author=Y.%20Horsmans&author=M.%20O.%20Babaoglu&author=A.%20Bozkurt&volume=76&publication_year=2004&pages=113-118&pmid=15289788&doi=10.1016/j.clpt.2004.04.001&)

2. Annalora A. J., Marcus C. B., Iversen P. L. (2017). Alternative splicing in the cytochrome P450 superfamily expands protein diversity to augment gene function and redirect human drug metabolism. Drug Metab. Dispos. 45, 375–389. 10.1124/dmd.116.073254  [DOI](https://doi.org/10.1124/dmd.116.073254) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28188297/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Alternative%20splicing%20in%20the%20cytochrome%20P450%20superfamily%20expands%20protein%20diversity%20to%20augment%20gene%20function%20and%20redirect%20human%20drug%20metabolism&author=A.%20J.%20Annalora&author=C.%20B.%20Marcus&author=P.%20L.%20Iversen&volume=45&publication_year=2017&pages=375-389&pmid=28188297&doi=10.1124/dmd.116.073254&)

3. Ariyoshi N., Shimizu Y., Kobayashi Y., Nakamura H., Nakasa H., Nakazawa K., et al. (2007). Identification and partial characterization of a novel CYP2C9 splicing variant encoding a protein lacking eight amino acid residues. Drug Metab. Pharmacokinet. 22, 187–194. 10.2133/dmpk.22.187  [DOI](https://doi.org/10.2133/dmpk.22.187) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17603219/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Pharmacokinet.&title=Identification%20and%20partial%20characterization%20of%20a%20novel%20CYP2C9%20splicing%20variant%20encoding%20a%20protein%20lacking%20eight%20amino%20acid%20residues&author=N.%20Ariyoshi&author=Y.%20Shimizu&author=Y.%20Kobayashi&author=H.%20Nakamura&author=H.%20Nakasa&volume=22&publication_year=2007&pages=187-194&pmid=17603219&doi=10.2133/dmpk.22.187&)

4. Asiimwe I. G., Blockman M., Cohen K., Cupido C., Hutchinson C., Jacobson B., et al. (2021). Stable warfarin dose prediction in sub-saharan african patients: A machine-learning approach and external validation of a clinical dose-initiation algorithm. CPT. Pharmacometrics Syst. Pharmacol. 11, 20–29. 10.1002/psp4.12740  [DOI](https://doi.org/10.1002/psp4.12740) | [PMC free article](/articles/PMC8752108/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34889080/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CPT.%20Pharmacometrics%20Syst.%20Pharmacol.&title=Stable%20warfarin%20dose%20prediction%20in%20sub-saharan%20african%20patients:%20A%20machine-learning%20approach%20and%20external%20validation%20of%20a%20clinical%20dose-initiation%20algorithm&author=I.%20G.%20Asiimwe&author=M.%20Blockman&author=K.%20Cohen&author=C.%20Cupido&author=C.%20Hutchinson&volume=11&publication_year=2021&pages=20-29&pmid=34889080&doi=10.1002/psp4.12740&)

5. Asiimwe I. G., Pirmohamed M. (2022). Ethnic diversity and warfarin pharmacogenomics. Front. Pharmacol. 13, 866058. 10.3389/fphar.2022.866058  [DOI](https://doi.org/10.3389/fphar.2022.866058) | [PMC free article](/articles/PMC9014219/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35444556/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=Ethnic%20diversity%20and%20warfarin%20pharmacogenomics&author=I.%20G.%20Asiimwe&author=M.%20Pirmohamed&volume=13&publication_year=2022&pages=866058&pmid=35444556&doi=10.3389/fphar.2022.866058&)

6. Asiimwe I. G., Waitt C., Sekaggya-Wiltshire C., Hutchinson C., Okello E., Zhang E. J., et al. (2020a). Developing and validating a clinical warfarin dose-initiation model for black-african patients in South Africa and Uganda. Clin. Pharmacol. Ther. 109, 1564–1574. 10.1002/cpt.2128  [DOI](https://doi.org/10.1002/cpt.2128) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33280090/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Developing%20and%20validating%20a%20clinical%20warfarin%20dose-initiation%20model%20for%20black-african%20patients%20in%20South%20Africa%20and%20Uganda&author=I.%20G.%20Asiimwe&author=C.%20Waitt&author=C.%20Sekaggya-Wiltshire&author=C.%20Hutchinson&author=E.%20Okello&volume=109&publication_year=2020a&pages=1564-1574&pmid=33280090&doi=10.1002/cpt.2128&)

7. Asiimwe I. G., Zhang E. J., Osanlou R., Jorgensen A. L., Pirmohamed M. (2020b). Warfarin dosing algorithms: A systematic review. Br. J. Clin. Pharmacol. 87, 1717–1729. 10.1111/bcp.14608  [DOI](https://doi.org/10.1111/bcp.14608) | [PMC free article](/articles/PMC8056736/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33080066/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Warfarin%20dosing%20algorithms:%20A%20systematic%20review&author=I.%20G.%20Asiimwe&author=E.%20J.%20Zhang&author=R.%20Osanlou&author=A.%20L.%20Jorgensen&author=M.%20Pirmohamed&volume=87&publication_year=2020b&pages=1717-1729&pmid=33080066&doi=10.1111/bcp.14608&)

8. Asiimwe I. G., Zhang E. J., Osanlou R., Krause A., Dillon C., Suarez-Kurtz G., et al. (2019). Genetic factors influencing warfarin dose in black-african patients: A systematic review and meta-analysis. Clin. Pharmacol. Ther. 107, 1420–1433. 10.1002/cpt.1755  [DOI](https://doi.org/10.1002/cpt.1755) | [PMC free article](/articles/PMC7217737/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31869433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Genetic%20factors%20influencing%20warfarin%20dose%20in%20black-african%20patients:%20A%20systematic%20review%20and%20meta-analysis&author=I.%20G.%20Asiimwe&author=E.%20J.%20Zhang&author=R.%20Osanlou&author=A.%20Krause&author=C.%20Dillon&volume=107&publication_year=2019&pages=1420-1433&pmid=31869433&doi=10.1002/cpt.1755&)

9. Barrett J. C., Fry B., Maller J., Daly M. J. (2005). Haploview: Analysis and visualization of LD and haplotype maps. Bioinformatics 21, 263–265. 10.1093/bioinformatics/bth457  [DOI](https://doi.org/10.1093/bioinformatics/bth457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15297300/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Haploview:%20Analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&author=J.%20C.%20Barrett&author=B.%20Fry&author=J.%20Maller&author=M.%20J.%20Daly&volume=21&publication_year=2005&pages=263-265&pmid=15297300&doi=10.1093/bioinformatics/bth457&)

10. Bristol-Myers Squibb (2018). Coumadin (warfarin sodium tablets) [product monograph]. Available at: https://pdf.hres.ca/dpd_pm/00047189.PDF (Accessed September 4, 2018).  [https://pdf.hres.ca/dpd_pm/00047189.PDF](https://pdf.hres.ca/dpd_pm/00047189.PDF)

11. Browning B. L., Browning S. R. (2009). A unified approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals. Am. J. Hum. Genet. 84, 210–223. 10.1016/j.ajhg.2009.01.005  [DOI](https://doi.org/10.1016/j.ajhg.2009.01.005) | [PMC free article](/articles/PMC2668004/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19200528/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=A%20unified%20approach%20to%20genotype%20imputation%20and%20haplotype-phase%20inference%20for%20large%20data%20sets%20of%20trios%20and%20unrelated%20individuals&author=B.%20L.%20Browning&author=S.%20R.%20Browning&volume=84&publication_year=2009&pages=210-223&pmid=19200528&doi=10.1016/j.ajhg.2009.01.005&)

12. Cavallari L. H., Perera M. A. (2012). The future of warfarin pharmacogenetics in under-represented minority groups. Future Cardiol. 8, 563–576. 10.2217/fca.12.31  [DOI](https://doi.org/10.2217/fca.12.31) | [PMC free article](/articles/PMC3463230/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22871196/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Future%20Cardiol.&title=The%20future%20of%20warfarin%20pharmacogenetics%20in%20under-represented%20minority%20groups&author=L.%20H.%20Cavallari&author=M.%20A.%20Perera&volume=8&publication_year=2012&pages=563-576&pmid=22871196&doi=10.2217/fca.12.31&)

13. Chaplin M., Kirkham J. J., Dwan K., Sloan D. J., Davies G., Jorgensen A. L. (2020). STrengthening the reporting of pharmacogenetic studies: Development of the STROPS guideline. PLoS Med. 17, e1003344. 10.1371/journal.pmed.1003344  [DOI](https://doi.org/10.1371/journal.pmed.1003344) | [PMC free article](/articles/PMC7505422/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32956352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Med.&title=STrengthening%20the%20reporting%20of%20pharmacogenetic%20studies:%20Development%20of%20the%20STROPS%20guideline&author=M.%20Chaplin&author=J.%20J.%20Kirkham&author=K.%20Dwan&author=D.%20J.%20Sloan&author=G.%20Davies&volume=17&publication_year=2020&pages=e1003344&pmid=32956352&doi=10.1371/journal.pmed.1003344&)

14. Choonara I. A., Haynes B. P., Cholerton S., Breckenridge A. M., Park B. K. (1986). Enantiomers of warfarin and vitamin K1 metabolism. Br. J. Clin. Pharmacol. 22, 729–732. 10.1111/j.1365-2125.1986.tb02966.x  [DOI](https://doi.org/10.1111/j.1365-2125.1986.tb02966.x) | [PMC free article](/articles/PMC1401216/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/3567019/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Enantiomers%20of%20warfarin%20and%20vitamin%20K1%20metabolism&author=I.%20A.%20Choonara&author=B.%20P.%20Haynes&author=S.%20Cholerton&author=A.%20M.%20Breckenridge&author=B.%20K.%20Park&volume=22&publication_year=1986&pages=729-732&pmid=3567019&doi=10.1111/j.1365-2125.1986.tb02966.x&)

15. Consortium G. T. (2013). The genotype-tissue expression (GTEx) project. Nat. Genet. 45, 580–585. 10.1038/ng.2653  [DOI](https://doi.org/10.1038/ng.2653) | [PMC free article](/articles/PMC4010069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23715323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=The%20genotype-tissue%20expression%20(GTEx)%20project&author=G.%20T.%20Consortium&volume=45&publication_year=2013&pages=580-585&pmid=23715323&doi=10.1038/ng.2653&)

16. Das S., Forer L., Schonherr S., Sidore C., Locke A. E., Kwong A., et al. (2016). Next-generation genotype imputation service and methods. Nat. Genet. 48, 1284–1287. 10.1038/ng.3656  [DOI](https://doi.org/10.1038/ng.3656) | [PMC free article](/articles/PMC5157836/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27571263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Next-generation%20genotype%20imputation%20service%20and%20methods&author=S.%20Das&author=L.%20Forer&author=S.%20Schonherr&author=C.%20Sidore&author=A.%20E.%20Locke&volume=48&publication_year=2016&pages=1284-1287&pmid=27571263&doi=10.1038/ng.3656&)

17. Delaneau O., Marchini J., Zagury J. F. (2011). A linear complexity phasing method for thousands of genomes. Nat. Methods 9, 179–181. 10.1038/nmeth.1785  [DOI](https://doi.org/10.1038/nmeth.1785) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22138821/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&title=A%20linear%20complexity%20phasing%20method%20for%20thousands%20of%20genomes&author=O.%20Delaneau&author=J.%20Marchini&author=J.%20F.%20Zagury&volume=9&publication_year=2011&pages=179-181&pmid=22138821&doi=10.1038/nmeth.1785&)

18. Dickmann L. J., Rettie A. E., Kneller M. B., Kim R. B., Wood A. J., Stein C. M., et al. (2001). Identification and functional characterization of a new CYP2C9 variant (CYP2C9*5) expressed among African Americans. Mol. Pharmacol. 60, 382–387. 10.1124/mol.60.2.382  [DOI](https://doi.org/10.1124/mol.60.2.382) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11455026/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=Identification%20and%20functional%20characterization%20of%20a%20new%20CYP2C9%20variant%20(CYP2C9*5)%20expressed%20among%20African%20Americans&author=L.%20J.%20Dickmann&author=A.%20E.%20Rettie&author=M.%20B.%20Kneller&author=R.%20B.%20Kim&author=A.%20J.%20Wood&volume=60&publication_year=2001&pages=382-387&pmid=11455026&doi=10.1124/mol.60.2.382&)

19. Dolejsi E., Bodenstorfer B., Frommlet F. (2014). Analyzing genome-wide association studies with an FDR controlling modification of the Bayesian Information Criterion. PLoS One 9, e103322. 10.1371/journal.pone.0103322  [DOI](https://doi.org/10.1371/journal.pone.0103322) | [PMC free article](/articles/PMC4111553/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25061809/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Analyzing%20genome-wide%20association%20studies%20with%20an%20FDR%20controlling%20modification%20of%20the%20Bayesian%20Information%20Criterion&author=E.%20Dolejsi&author=B.%20Bodenstorfer&author=F.%20Frommlet&volume=9&publication_year=2014&pages=e103322&pmid=25061809&doi=10.1371/journal.pone.0103322&)

20. Esteban A. L. M., Adriaan Van Der G., Pauline L., Marije Van Der G., Jingyuan F., Morris S., et al. (2020). Lack of association between genetic variants at ACE2 and TMPRSS2 genes involved in SARS-CoV-2 infection and human quantitative phenotypes. Front. Genet. 11, 613. 10.3389/fgene.2020.00613  [DOI](https://doi.org/10.3389/fgene.2020.00613) | [PMC free article](/articles/PMC7295011/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32582302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Genet.&title=Lack%20of%20association%20between%20genetic%20variants%20at%20ACE2%20and%20TMPRSS2%20genes%20involved%20in%20SARS-CoV-2%20infection%20and%20human%20quantitative%20phenotypes&author=A.%20L.%20M.%20Esteban&author=G.%20Adriaan%20Van%20Der&author=L.%20Pauline&author=G.%20Marije%20Van%20Der&author=F.%20Jingyuan&volume=11&publication_year=2020&pages=613&pmid=32582302&doi=10.3389/fgene.2020.00613&)

21. Fatumo S., Chikowore T., Choudhury A., Ayub M., Martin A. R., Kuchenbaecker K. (2022). A roadmap to increase diversity in genomic studies. Nat. Med. 28, 243–250. 10.1038/s41591-021-01672-4  [DOI](https://doi.org/10.1038/s41591-021-01672-4) | [PMC free article](/articles/PMC7614889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35145307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&title=A%20roadmap%20to%20increase%20diversity%20in%20genomic%20studies&author=S.%20Fatumo&author=T.%20Chikowore&author=A.%20Choudhury&author=M.%20Ayub&author=A.%20R.%20Martin&volume=28&publication_year=2022&pages=243-250&pmid=35145307&doi=10.1038/s41591-021-01672-4&)

22. Fontana V., Turner R. M., Francis B., Yin P., Putz B., Hiltunen T. P., et al. (2022). Chromosomal region 11p14.1 is associated with pharmacokinetics and pharmacodynamics of bisoprolol. Pharmgenomics. Pers. Med. 15, 249–260. 10.2147/PGPM.S352719  [DOI](https://doi.org/10.2147/PGPM.S352719) | [PMC free article](/articles/PMC8958266/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35356681/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenomics.%20Pers.%20Med.&title=Chromosomal%20region%2011p14.1%20is%20associated%20with%20pharmacokinetics%20and%20pharmacodynamics%20of%20bisoprolol&author=V.%20Fontana&author=R.%20M.%20Turner&author=B.%20Francis&author=P.%20Yin&author=B.%20Putz&volume=15&publication_year=2022&pages=249-260&pmid=35356681&doi=10.2147/PGPM.S352719&)

23. Frommlet F., Ruhaltinger F., Twaróg P., Bogdan M. (2012). Modified versions of Bayesian Information Criterion for genome-wide association studies. Comput. Statistics Data Analysis 56, 1038–1051. 10.1016/j.csda.2011.05.005  [DOI](https://doi.org/10.1016/j.csda.2011.05.005) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Comput.%20Statistics%20Data%20Analysis&title=Modified%20versions%20of%20Bayesian%20Information%20Criterion%20for%20genome-wide%20association%20studies&author=F.%20Frommlet&author=F.%20Ruhaltinger&author=P.%20Twar%C3%B3g&author=M.%20Bogdan&volume=56&publication_year=2012&pages=1038-1051&doi=10.1016/j.csda.2011.05.005&)

24. Genomes Project C., Abecasis G. R., Altshuler D., Auton A., Brooks L. D., Durbin R. M., et al. (2010). A map of human genome variation from population-scale sequencing. Nature 467, 1061–1073. 10.1038/nature09534  [DOI](https://doi.org/10.1038/nature09534) | [PMC free article](/articles/PMC3042601/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20981092/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=A%20map%20of%20human%20genome%20variation%20from%20population-scale%20sequencing&author=C.%20Genomes%20Project&author=G.%20R.%20Abecasis&author=D.%20Altshuler&author=A.%20Auton&author=L.%20D.%20Brooks&volume=467&publication_year=2010&pages=1061-1073&pmid=20981092&doi=10.1038/nature09534&)

25. Haining R. L., Hunter A. P., Veronese M. E., Trager W. F., Rettie A. E. (1996). Allelic variants of human cytochrome P450 2C9: Baculovirus-mediated expression, purification, structural characterization, substrate stereoselectivity, and prochiral selectivity of the wild-type and I359L mutant forms. Arch. Biochem. Biophys. 333, 447–458. 10.1006/abbi.1996.0414  [DOI](https://doi.org/10.1006/abbi.1996.0414) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8809086/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch.%20Biochem.%20Biophys.&title=Allelic%20variants%20of%20human%20cytochrome%20P450%202C9:%20Baculovirus-mediated%20expression,%20purification,%20structural%20characterization,%20substrate%20stereoselectivity,%20and%20prochiral%20selectivity%20of%20the%20wild-type%20and%20I359L%20mutant%20forms&author=R.%20L.%20Haining&author=A.%20P.%20Hunter&author=M.%20E.%20Veronese&author=W.%20F.%20Trager&author=A.%20E.%20Rettie&volume=333&publication_year=1996&pages=447-458&pmid=8809086&doi=10.1006/abbi.1996.0414&)

26. Hofer P., Hagmann M., Brezina S., Dolejsi E., Mach K., Leeb G., et al. (2017). Bayesian and frequentist analysis of an Austrian genome-wide association study of colorectal cancer and advanced adenomas. Oncotarget 8, 98623–98634. 10.18632/oncotarget.21697  [DOI](https://doi.org/10.18632/oncotarget.21697) | [PMC free article](/articles/PMC5716755/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29228715/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncotarget&title=Bayesian%20and%20frequentist%20analysis%20of%20an%20Austrian%20genome-wide%20association%20study%20of%20colorectal%20cancer%20and%20advanced%20adenomas&author=P.%20Hofer&author=M.%20Hagmann&author=S.%20Brezina&author=E.%20Dolejsi&author=K.%20Mach&volume=8&publication_year=2017&pages=98623-98634&pmid=29228715&doi=10.18632/oncotarget.21697&)

27. Howie B. N., Donnelly P., Marchini J. (2009). A flexible and accurate genotype imputation method for the next generation of genome-wide association studies. PLoS Genet. 5, e1000529. 10.1371/journal.pgen.1000529  [DOI](https://doi.org/10.1371/journal.pgen.1000529) | [PMC free article](/articles/PMC2689936/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19543373/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=A%20flexible%20and%20accurate%20genotype%20imputation%20method%20for%20the%20next%20generation%20of%20genome-wide%20association%20studies&author=B.%20N.%20Howie&author=P.%20Donnelly&author=J.%20Marchini&volume=5&publication_year=2009&pages=e1000529&pmid=19543373&doi=10.1371/journal.pgen.1000529&)

28. Joint Formulary Committee (2019). British national formulary 78 september 2019 – March 2020. London: BMJ Group and Pharmaceutical Press.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=British%20national%20formulary%2078%20september%202019%20%E2%80%93%20March%202020&publication_year=2019&)

29. Jukic M. M., Smith R. L., Molden E., Ingelman-Sundberg M. (2021). Evaluation of the CYP2D6 haplotype Activity scores based on metabolic ratios of 4, 700 patients treated with three different CYP2D6 substrates. Clin. Pharmacol. Ther. 110, 750–758. 10.1002/cpt.2246  [DOI](https://doi.org/10.1002/cpt.2246) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33792048/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Evaluation%20of%20the%20CYP2D6%20haplotype%20Activity%20scores%20based%20on%20metabolic%20ratios%20of%204,%20700%20patients%20treated%20with%20three%20different%20CYP2D6%20substrates&author=M.%20M.%20Jukic&author=R.%20L.%20Smith&author=E.%20Molden&author=M.%20Ingelman-Sundberg&volume=110&publication_year=2021&pages=750-758&pmid=33792048&doi=10.1002/cpt.2246&)

30. Keene O. N. (1995). The log transformation is special. Stat. Med. 14, 811–819. 10.1002/sim.4780140810  [DOI](https://doi.org/10.1002/sim.4780140810) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7644861/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stat.%20Med.&title=The%20log%20transformation%20is%20special&author=O.%20N.%20Keene&volume=14&publication_year=1995&pages=811-819&pmid=7644861&doi=10.1002/sim.4780140810&)

31. Kidd R. S., Curry T. B., Gallagher S., Edeki T., Blaisdell J., Goldstein J. A. (2001). Identification of a null allele of CYP2C9 in an African-American exhibiting toxicity to phenytoin. Pharmacogenetics 11, 803–808. 10.1097/00008571-200112000-00008  [DOI](https://doi.org/10.1097/00008571-200112000-00008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11740344/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Identification%20of%20a%20null%20allele%20of%20CYP2C9%20in%20an%20African-American%20exhibiting%20toxicity%20to%20phenytoin&author=R.%20S.%20Kidd&author=T.%20B.%20Curry&author=S.%20Gallagher&author=T.%20Edeki&author=J.%20Blaisdell&volume=11&publication_year=2001&pages=803-808&pmid=11740344&doi=10.1097/00008571-200112000-00008&)

32. Kitts A., Sherry S. (2011). “The single nucleotide polymorphism database (dbSNP) of nucleotide sequence variation,” in The NCBI handbook [internet]. Editor Mcentyre J O. J., Bethesda (MD): National Center for Biotechnology Information.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20NCBI%20handbook%20%5Binternet%5D&author=A.%20Kitts&author=S.%20Sherry&publication_year=2011&)

33. Li M., Chen W., Sun X., Wang Z., Zou X., Wei H., et al. (2019). Metastatic colorectal cancer and severe hypocalcemia following irinotecan administration in a patient with X-linked agammaglobulinemia: A case report. BMC Med. Genet. 20, 157. 10.1186/s12881-019-0880-1  [DOI](https://doi.org/10.1186/s12881-019-0880-1) | [PMC free article](/articles/PMC6739925/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31510946/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med.%20Genet.&title=Metastatic%20colorectal%20cancer%20and%20severe%20hypocalcemia%20following%20irinotecan%20administration%20in%20a%20patient%20with%20X-linked%20agammaglobulinemia:%20A%20case%20report&author=M.%20Li&author=W.%20Chen&author=X.%20Sun&author=Z.%20Wang&author=X.%20Zou&volume=20&publication_year=2019&pages=157&pmid=31510946&doi=10.1186/s12881-019-0880-1&)

34. Liedtke M. D., Rathbun R. C. (2009). Warfarin-antiretroviral interactions. Ann. Pharmacother. 43, 322–328. 10.1345/aph.1L497  [DOI](https://doi.org/10.1345/aph.1L497) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19196837/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann.%20Pharmacother.&title=Warfarin-antiretroviral%20interactions&author=M.%20D.%20Liedtke&author=R.%20C.%20Rathbun&volume=43&publication_year=2009&pages=322-328&pmid=19196837&doi=10.1345/aph.1L497&)

35. Liu Y., Jeong H., Takahashi H., Drozda K., Patel S. R., Shapiro N. L., et al. (2012). Decreased warfarin clearance associated with the CYP2C9 R150H (*8) polymorphism. Clin. Pharmacol. Ther. 91, 660–665. 10.1038/clpt.2011.269  [DOI](https://doi.org/10.1038/clpt.2011.269) | [PMC free article](/articles/PMC3879795/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22378156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Decreased%20warfarin%20clearance%20associated%20with%20the%20CYP2C9%20R150H%20(*8)%20polymorphism&author=Y.%20Liu&author=H.%20Jeong&author=H.%20Takahashi&author=K.%20Drozda&author=S.%20R.%20Patel&volume=91&publication_year=2012&pages=660-665&pmid=22378156&doi=10.1038/clpt.2011.269&)

36. Lofgren S., Baldwin R. M., Hiratsuka M., Lindqvist A., Carlberg A., Sim S. C., et al. (2008). Generation of mice transgenic for human CYP2C18 and CYP2C19: Characterization of the sexually dimorphic gene and enzyme expression. Drug Metab. Dispos. 36, 955–962. 10.1124/dmd.107.019349  [DOI](https://doi.org/10.1124/dmd.107.019349) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18276835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Generation%20of%20mice%20transgenic%20for%20human%20CYP2C18%20and%20CYP2C19:%20Characterization%20of%20the%20sexually%20dimorphic%20gene%20and%20enzyme%20expression&author=S.%20Lofgren&author=R.%20M.%20Baldwin&author=M.%20Hiratsuka&author=A.%20Lindqvist&author=A.%20Carlberg&volume=36&publication_year=2008&pages=955-962&pmid=18276835&doi=10.1124/dmd.107.019349&)

37. Mallayasamy S., Chaturvedula A., Blaschke T., Fossler M. J. (2018). A systematic evaluation of effect of adherence patterns on the sample size and power of a clinical study. CPT. Pharmacometrics Syst. Pharmacol. 7, 818–828. 10.1002/psp4.12361  [DOI](https://doi.org/10.1002/psp4.12361) | [PMC free article](/articles/PMC6310871/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30291680/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CPT.%20Pharmacometrics%20Syst.%20Pharmacol.&title=A%20systematic%20evaluation%20of%20effect%20of%20adherence%20patterns%20on%20the%20sample%20size%20and%20power%20of%20a%20clinical%20study&author=S.%20Mallayasamy&author=A.%20Chaturvedula&author=T.%20Blaschke&author=M.%20J.%20Fossler&volume=7&publication_year=2018&pages=818-828&pmid=30291680&doi=10.1002/psp4.12361&)

38. Marchini J., Howie B. (2010). Genotype imputation for genome-wide association studies. Nat. Rev. Genet. 11, 499–511. 10.1038/nrg2796  [DOI](https://doi.org/10.1038/nrg2796) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20517342/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&title=Genotype%20imputation%20for%20genome-wide%20association%20studies&author=J.%20Marchini&author=B.%20Howie&volume=11&publication_year=2010&pages=499-511&pmid=20517342&doi=10.1038/nrg2796&)

39. Milosavljevic F., Bukvic N., Pavlovic Z., Miljevic C., Pesic V., Molden E., et al. (2021). Association of CYP2C19 and CYP2D6 poor and intermediate metabolizer status with antidepressant and antipsychotic exposure: A systematic review and meta-analysis. JAMA Psychiatry 78, 270–280. 10.1001/jamapsychiatry.2020.3643  [DOI](https://doi.org/10.1001/jamapsychiatry.2020.3643) | [PMC free article](/articles/PMC7702196/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33237321/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Psychiatry&title=Association%20of%20CYP2C19%20and%20CYP2D6%20poor%20and%20intermediate%20metabolizer%20status%20with%20antidepressant%20and%20antipsychotic%20exposure:%20A%20systematic%20review%20and%20meta-analysis&author=F.%20Milosavljevic&author=N.%20Bukvic&author=Z.%20Pavlovic&author=C.%20Miljevic&author=V.%20Pesic&volume=78&publication_year=2021&pages=270-280&pmid=33237321&doi=10.1001/jamapsychiatry.2020.3643&)

40. Moons K. G., Donders R. A., Stijnen T., Harrell F. E., Jr. (2006). Using the outcome for imputation of missing predictor values was preferred. J. Clin. Epidemiol. 59, 1092–1101. 10.1016/j.jclinepi.2006.01.009  [DOI](https://doi.org/10.1016/j.jclinepi.2006.01.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16980150/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Epidemiol.&title=Using%20the%20outcome%20for%20imputation%20of%20missing%20predictor%20values%20was%20preferred&author=K.%20G.%20Moons&author=R.%20A.%20Donders&author=T.%20Stijnen&author=F.%20E.%20Harrell&volume=59&publication_year=2006&pages=1092-1101&pmid=16980150&doi=10.1016/j.jclinepi.2006.01.009&)

41. Mouton J. P., Blockman M., Sekaggya-Wiltshire C., Semakula J., Waitt C., Pirmohamed M., et al. (2021). Improving anticoagulation in sub-Saharan Africa: What are the challenges and how can we overcome them? Br. J. Clin. Pharmacol. 87, 3056–3068. 10.1111/bcp.14768  [DOI](https://doi.org/10.1111/bcp.14768) | [PMC free article](/articles/PMC8359270/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33586223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Improving%20anticoagulation%20in%20sub-Saharan%20Africa:%20What%20are%20the%20challenges%20and%20how%20can%20we%20overcome%20them?&author=J.%20P.%20Mouton&author=M.%20Blockman&author=C.%20Sekaggya-Wiltshire&author=J.%20Semakula&author=C.%20Waitt&volume=87&publication_year=2021&pages=3056-3068&pmid=33586223&doi=10.1111/bcp.14768&)

42. Mouton J. P., Njuguna C., Kramer N., Stewart A., Mehta U., Blockman M., et al. (2016). Adverse drug reactions causing admission to medical wards: A cross-sectional survey at 4 hospitals in South Africa. Med. Baltim. 95, e3437. 10.1097/MD.0000000000003437  [DOI](https://doi.org/10.1097/MD.0000000000003437) | [PMC free article](/articles/PMC4902486/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27175644/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med.%20Baltim.&title=Adverse%20drug%20reactions%20causing%20admission%20to%20medical%20wards:%20A%20cross-sectional%20survey%20at%204%20hospitals%20in%20South%20Africa&author=J.%20P.%20Mouton&author=C.%20Njuguna&author=N.%20Kramer&author=A.%20Stewart&author=U.%20Mehta&volume=95&publication_year=2016&pages=e3437&pmid=27175644&doi=10.1097/MD.0000000000003437&)

43. Muszkat M., Blotnik S., Elami A., Krasilnikov I., Caraco Y. (2007). Warfarin metabolism and anticoagulant effect: A prospective, observational study of the impact of CYP2C9 genetic polymorphism in the presence of drug-disease and drug-drug interactions. Clin. Ther. 29, 427–437. 10.1016/s0149-2918(07)80081-6  [DOI](https://doi.org/10.1016/s0149-2918(07)80081-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17577464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Ther.&title=Warfarin%20metabolism%20and%20anticoagulant%20effect:%20A%20prospective,%20observational%20study%20of%20the%20impact%20of%20CYP2C9%20genetic%20polymorphism%20in%20the%20presence%20of%20drug-disease%20and%20drug-drug%20interactions&author=M.%20Muszkat&author=S.%20Blotnik&author=A.%20Elami&author=I.%20Krasilnikov&author=Y.%20Caraco&volume=29&publication_year=2007&pages=427-437&pmid=17577464&doi=10.1016/s0149-2918(07)80081-6&)

44. Niinuma Y., Saito T., Takahashi M., Tsukada C., Ito M., Hirasawa N., et al. (2014). Functional characterization of 32 CYP2C9 allelic variants. Pharmacogenomics J. 14, 107–114. 10.1038/tpj.2013.22  [DOI](https://doi.org/10.1038/tpj.2013.22) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23752738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Functional%20characterization%20of%2032%20CYP2C9%20allelic%20variants&author=Y.%20Niinuma&author=T.%20Saito&author=M.%20Takahashi&author=C.%20Tsukada&author=M.%20Ito&volume=14&publication_year=2014&pages=107-114&pmid=23752738&doi=10.1038/tpj.2013.22&)

45. Perera M. A., Cavallari L. H., Limdi N. A., Gamazon E. R., Konkashbaev A., Daneshjou R., et al. (2013). Genetic variants associated with warfarin dose in african-American individuals: A genome-wide association study. Lancet 382, 790–796. 10.1016/S0140-6736(13)60681-9  [DOI](https://doi.org/10.1016/S0140-6736(13)60681-9) | [PMC free article](/articles/PMC3759580/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23755828/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Genetic%20variants%20associated%20with%20warfarin%20dose%20in%20african-American%20individuals:%20A%20genome-wide%20association%20study&author=M.%20A.%20Perera&author=L.%20H.%20Cavallari&author=N.%20A.%20Limdi&author=E.%20R.%20Gamazon&author=A.%20Konkashbaev&volume=382&publication_year=2013&pages=790-796&pmid=23755828&doi=10.1016/S0140-6736(13)60681-9&)

46. Pirmohamed M., Kamali F., Daly A. K., Wadelius M. (2015). Oral anticoagulation: A critique of recent advances and controversies. Trends Pharmacol. Sci. 36, 153–163. 10.1016/j.tips.2015.01.003  [DOI](https://doi.org/10.1016/j.tips.2015.01.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25698605/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Pharmacol.%20Sci.&title=Oral%20anticoagulation:%20A%20critique%20of%20recent%20advances%20and%20controversies&author=M.%20Pirmohamed&author=F.%20Kamali&author=A.%20K.%20Daly&author=M.%20Wadelius&volume=36&publication_year=2015&pages=153-163&pmid=25698605&doi=10.1016/j.tips.2015.01.003&)

47. Pratt V. M., Cavallari L. H., Del Tredici A. L., Hachad H., Ji Y., Moyer A. M., et al. (2019). Recommendations for clinical CYP2C9 genotyping allele selection: A joint recommendation of the association for molecular Pathology and College of American Pathologists. J. Mol. Diagn. 21, 746–755. 10.1016/j.jmoldx.2019.04.003  [DOI](https://doi.org/10.1016/j.jmoldx.2019.04.003) | [PMC free article](/articles/PMC7057225/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31075510/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Diagn.&title=Recommendations%20for%20clinical%20CYP2C9%20genotyping%20allele%20selection:%20A%20joint%20recommendation%20of%20the%20association%20for%20molecular%20Pathology%20and%20College%20of%20American%20Pathologists&author=V.%20M.%20Pratt&author=L.%20H.%20Cavallari&author=A.%20L.%20Del%20Tredici&author=H.%20Hachad&author=Y.%20Ji&volume=21&publication_year=2019&pages=746-755&pmid=31075510&doi=10.1016/j.jmoldx.2019.04.003&)

48. Price A. L., Patterson N. J., Plenge R. M., Weinblatt M. E., Shadick N. A., Reich D. (2006). Principal components analysis corrects for stratification in genome-wide association studies. Nat. Genet. 38, 904–909. 10.1038/ng1847  [DOI](https://doi.org/10.1038/ng1847) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16862161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Principal%20components%20analysis%20corrects%20for%20stratification%20in%20genome-wide%20association%20studies&author=A.%20L.%20Price&author=N.%20J.%20Patterson&author=R.%20M.%20Plenge&author=M.%20E.%20Weinblatt&author=N.%20A.%20Shadick&volume=38&publication_year=2006&pages=904-909&pmid=16862161&doi=10.1038/ng1847&)

49. Pruim R. J., Welch R. P., Sanna S., Teslovich T. M., Chines P. S., Gliedt T. P., et al. (2010). LocusZoom: Regional visualization of genome-wide association scan results. Bioinformatics 26, 2336–2337. 10.1093/bioinformatics/btq419  [DOI](https://doi.org/10.1093/bioinformatics/btq419) | [PMC free article](/articles/PMC2935401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20634204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LocusZoom:%20Regional%20visualization%20of%20genome-wide%20association%20scan%20results&author=R.%20J.%20Pruim&author=R.%20P.%20Welch&author=S.%20Sanna&author=T.%20M.%20Teslovich&author=P.%20S.%20Chines&volume=26&publication_year=2010&pages=2336-2337&pmid=20634204&doi=10.1093/bioinformatics/btq419&)

50. Purcell S., Neale B., Todd-Brown K., Thomas L., Ferreira M. A., Bender D., et al. (2007). Plink: A tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet. 81, 559–575. 10.1086/519795  [DOI](https://doi.org/10.1086/519795) | [PMC free article](/articles/PMC1950838/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17701901/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Plink:%20A%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&author=S.%20Purcell&author=B.%20Neale&author=K.%20Todd-Brown&author=L.%20Thomas&author=M.%20A.%20Ferreira&volume=81&publication_year=2007&pages=559-575&pmid=17701901&doi=10.1086/519795&)

51. Rettie A. E., Wienkers L. C., Gonzalez F. J., Trager W. F., Korzekwa K. R. (1994). Impaired (S)-warfarin metabolism catalysed by the R144C allelic variant of CYP2C9. Pharmacogenetics 4, 39–42. 10.1097/00008571-199402000-00005  [DOI](https://doi.org/10.1097/00008571-199402000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8004131/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Impaired%20(S)-warfarin%20metabolism%20catalysed%20by%20the%20R144C%20allelic%20variant%20of%20CYP2C9&author=A.%20E.%20Rettie&author=L.%20C.%20Wienkers&author=F.%20J.%20Gonzalez&author=W.%20F.%20Trager&author=K.%20R.%20Korzekwa&volume=4&publication_year=1994&pages=39-42&pmid=8004131&doi=10.1097/00008571-199402000-00005&)

52. Scordo M. G., Pengo V., Spina E., Dahl M. L., Gusella M., Padrini R. (2002). Influence of CYP2C9 and CYP2C19 genetic polymorphisms on warfarin maintenance dose and metabolic clearance. Clin. Pharmacol. Ther. 72, 702–710. 10.1067/mcp.2002.129321  [DOI](https://doi.org/10.1067/mcp.2002.129321) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12496751/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Influence%20of%20CYP2C9%20and%20CYP2C19%20genetic%20polymorphisms%20on%20warfarin%20maintenance%20dose%20and%20metabolic%20clearance&author=M.%20G.%20Scordo&author=V.%20Pengo&author=E.%20Spina&author=M.%20L.%20Dahl&author=M.%20Gusella&volume=72&publication_year=2002&pages=702-710&pmid=12496751&doi=10.1067/mcp.2002.129321&)

53. Semakula J. R., Kisa G., Mouton J. P., Cohen K., Blockman M., Pirmohamed M., et al. (2021). Anticoagulation in sub-saharan Africa: Are direct oral anticoagulants the answer? A review of lessons learnt from warfarin. Br. J. Clin. Pharmacol. 87, 3699–3705. 10.1111/bcp.14796  [DOI](https://doi.org/10.1111/bcp.14796) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33624331/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Anticoagulation%20in%20sub-saharan%20Africa:%20Are%20direct%20oral%20anticoagulants%20the%20answer?%20A%20review%20of%20lessons%20learnt%20from%20warfarin&author=J.%20R.%20Semakula&author=G.%20Kisa&author=J.%20P.%20Mouton&author=K.%20Cohen&author=M.%20Blockman&volume=87&publication_year=2021&pages=3699-3705&pmid=33624331&doi=10.1111/bcp.14796&)

54. Shaik A. N., Grater R., Lulla M., Williams D. A., Gan L. L., Bohnert T., et al. (2016). Comparison of enzyme kinetics of warfarin analyzed by LC-MS/MS QTrap and differential mobility spectrometry. J. Chromatogr. B Anal. Technol. Biomed. Life Sci. 1008, 164–173. 10.1016/j.jchromb.2015.11.036  [DOI](https://doi.org/10.1016/j.jchromb.2015.11.036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26655108/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Chromatogr.%20B%20Anal.%20Technol.%20Biomed.%20Life%20Sci.&title=Comparison%20of%20enzyme%20kinetics%20of%20warfarin%20analyzed%20by%20LC-MS/MS%20QTrap%20and%20differential%20mobility%20spectrometry&author=A.%20N.%20Shaik&author=R.%20Grater&author=M.%20Lulla&author=D.%20A.%20Williams&author=L.%20L.%20Gan&volume=1008&publication_year=2016&pages=164-173&pmid=26655108&doi=10.1016/j.jchromb.2015.11.036&)

55. Sirugo G., Williams S. M., Tishkoff S. A. (2019). The missing diversity in human genetic studies. Cell 177, 26–31. 10.1016/j.cell.2019.02.048  [DOI](https://doi.org/10.1016/j.cell.2019.02.048) | [PMC free article](/articles/PMC7380073/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30901543/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=The%20missing%20diversity%20in%20human%20genetic%20studies&author=G.%20Sirugo&author=S.%20M.%20Williams&author=S.%20A.%20Tishkoff&volume=177&publication_year=2019&pages=26-31&pmid=30901543&doi=10.1016/j.cell.2019.02.048&)

56. Smigielski E. M., Sirotkin K., Ward M., Sherry S. T. (2000). dbSNP: a database of single nucleotide polymorphisms. Nucleic Acids Res. 28, 352–355. 10.1093/nar/28.1.352  [DOI](https://doi.org/10.1093/nar/28.1.352) | [PMC free article](/articles/PMC102496/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10592272/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=dbSNP:%20a%20database%20of%20single%20nucleotide%20polymorphisms&author=E.%20M.%20Smigielski&author=K.%20Sirotkin&author=M.%20Ward&author=S.%20T.%20Sherry&volume=28&publication_year=2000&pages=352-355&pmid=10592272&doi=10.1093/nar/28.1.352&)

57. Tai G., Farin F., Rieder M. J., Dreisbach A. W., Veenstra D. L., Verlinde C. L., et al. (2005). In-vitro and in-vivo effects of the CYP2C9*11 polymorphism on warfarin metabolism and dose. Pharmacogenet. Genomics 15, 475–481. 10.1097/01.fpc.0000162005.80857.98  [DOI](https://doi.org/10.1097/01.fpc.0000162005.80857.98) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15970795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=In-vitro%20and%20in-vivo%20effects%20of%20the%20CYP2C9*11%20polymorphism%20on%20warfarin%20metabolism%20and%20dose&author=G.%20Tai&author=F.%20Farin&author=M.%20J.%20Rieder&author=A.%20W.%20Dreisbach&author=D.%20L.%20Veenstra&volume=15&publication_year=2005&pages=475-481&pmid=15970795&doi=10.1097/01.fpc.0000162005.80857.98&)

58. Teo Y. Y., Small K. S., Kwiatkowski D. P. (2010). Methodological challenges of genome-wide association analysis in Africa. Nat. Rev. Genet. 11, 149–160. 10.1038/nrg2731  [DOI](https://doi.org/10.1038/nrg2731) | [PMC free article](/articles/PMC3769612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20084087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&title=Methodological%20challenges%20of%20genome-wide%20association%20analysis%20in%20Africa&author=Y.%20Y.%20Teo&author=K.%20S.%20Small&author=D.%20P.%20Kwiatkowski&volume=11&publication_year=2010&pages=149-160&pmid=20084087&doi=10.1038/nrg2731&)

59. Thareja G., Evans A. M., Wood S. D., Stephan N., Zaghlool S., Halama A., et al. The Qatar genome program research Consortium (2022). Ratios of acetaminophen metabolites identify new loci of pharmacogenetic relevance in a genome-wide association study. Metabolites 12 (6), 496. 10.3390/metabo12060496  [DOI](https://doi.org/10.3390/metabo12060496) | [PMC free article](/articles/PMC9228664/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35736429/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Metabolites&title=Ratios%20of%20acetaminophen%20metabolites%20identify%20new%20loci%20of%20pharmacogenetic%20relevance%20in%20a%20genome-wide%20association%20study&author=G.%20Thareja&author=A.%20M.%20Evans&author=S.%20D.%20Wood&author=N.%20Stephan&author=S.%20Zaghlool&volume=12&issue=6&publication_year=2022&pages=496&pmid=35736429&doi=10.3390/metabo12060496&)

60. Tishkoff S. A., Reed F. A., Friedlaender F. R., Ehret C., Ranciaro A., Froment A., et al. (2009). The genetic structure and history of Africans and African Americans. Science 324, 1035–1044. 10.1126/science.1172257  [DOI](https://doi.org/10.1126/science.1172257) | [PMC free article](/articles/PMC2947357/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19407144/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=The%20genetic%20structure%20and%20history%20of%20Africans%20and%20African%20Americans&author=S.%20A.%20Tishkoff&author=F.%20A.%20Reed&author=F.%20R.%20Friedlaender&author=C.%20Ehret&author=A.%20Ranciaro&volume=324&publication_year=2009&pages=1035-1044&pmid=19407144&doi=10.1126/science.1172257&)

61. Turner R. M., Fontana V., Zhang J. E., Carr D., Yin P., Fitzgerald R., et al. (2020). A genome-wide association study of circulating levels of atorvastatin and its major metabolites. Clin. Pharmacol. Ther. 108, 287–297. 10.1002/cpt.1820  [DOI](https://doi.org/10.1002/cpt.1820) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32128760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=A%20genome-wide%20association%20study%20of%20circulating%20levels%20of%20atorvastatin%20and%20its%20major%20metabolites&author=R.%20M.%20Turner&author=V.%20Fontana&author=J.%20E.%20Zhang&author=D.%20Carr&author=P.%20Yin&volume=108&publication_year=2020&pages=287-297&pmid=32128760&doi=10.1002/cpt.1820&)

62. Turner S. (2018). qqman: an R package for visualizing GWAS results using Q-Q and manhattan plots. J. Open Source Softw. 3, 731. 10.21105/joss.00731  [DOI](https://doi.org/10.21105/joss.00731) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Open%20Source%20Softw.&title=qqman:%20an%20R%20package%20for%20visualizing%20GWAS%20results%20using%20Q-Q%20and%20manhattan%20plots&author=S.%20Turner&volume=3&publication_year=2018&pages=731&doi=10.21105/joss.00731&)

63. U.S. National Library of Medicine (2018). PubChem. Warfarin. Open chemistry database. Maryland, United States: Online: National Centre for Biotechnology Information.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=PubChem.%20Warfarin.%20Open%20chemistry%20database&publication_year=2018&)

64. Van Buuren S., Groothuis-Oudshoorn K. (2011). mice: Multivariate imputation by chained Equations in R. J. Stat. Softw. 45, 1–67. 10.18637/jss.v045.i03  [DOI](https://doi.org/10.18637/jss.v045.i03) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Stat.%20Softw.&title=mice:%20Multivariate%20imputation%20by%20chained%20Equations%20in%20R&author=S.%20Van%20Buuren&author=K.%20Groothuis-Oudshoorn&volume=45&publication_year=2011&pages=1-67&doi=10.18637/jss.v045.i03&)

65. Vittinghoff E., Glidden D., Shiboski S., Mcculloch C. (2012). Regression methods in biostatistics. New York: Springer.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=Regression%20methods%20in%20biostatistics&author=E.%20Vittinghoff&author=D.%20Glidden&author=S.%20Shiboski&author=C.%20Mcculloch&publication_year=2012&)

66. Wanounou M., Shaul C., Abu Ghosh Z., Alamia S., Caraco Y. (2022). The impact of CYP2C9*11 genetic polymorphism on the pharmacokinetics of phenytoin and (S)-Warfarin. Clin. Pharmacol. Ther. 112 (1), 156–163. 10.1002/cpt.2613  [DOI](https://doi.org/10.1002/cpt.2613) | [PMC free article](/articles/PMC9322346/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35426132/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20impact%20of%20CYP2C9*11%20genetic%20polymorphism%20on%20the%20pharmacokinetics%20of%20phenytoin%20and%20(S)-Warfarin&author=M.%20Wanounou&author=C.%20Shaul&author=Z.%20Abu%20Ghosh&author=S.%20Alamia&author=Y.%20Caraco&volume=112&issue=1&publication_year=2022&pages=156-163&pmid=35426132&doi=10.1002/cpt.2613&)
