# Potential role of regulatory DNA variants in modifying the risk of severe cutaneous reactions induced by aromatic anti‐seizure medications

## Metadata
**Authors:** Kerry A Mullan, Alison Anderson, Yi‐Wu Shi, Jia‐Hong Ding, Ching‐Ching Ng, Zhibin Chen, Larry Baum, Stacey Cherny, Slave Petrovski, Pak C Sham, Kheng‐Seang Lim, Wei‐Ping Liao, Patrick Kwan
**Journal:** Epilepsia
**Date:** 2022 Feb 16
**DOI:** [10.1111/epi.17182](https://doi.org/10.1111/epi.17182)
**PMID:** 35170024
**PMCID:** PMC9541367
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9541367/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9541367/pdf/EPI-63-936.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9541367/pdf/EPI-63-936.pdf)

## Abstract

**Objective:** 
Stevens–Johnson syndrome (SJS) and toxic epidermal necrolysis (TEN) are severe cutaneous adverse drug reactions. Antiseizure medications (ASMs) with aromatic ring structure, including carbamazepine, are among the most common culprits. Screening for human leukocyte antigen (HLA) allele HLA‐B*15:02 is recommended prior to initiating treatment with carbamazepine in Asians, but this allele has low positive predictive value.

**Methods:** 
We performed whole genome sequencing and analyzed 6 199 696 common variants among 113 aromatic ASM‐induced SJS/TEN cases and 84 tolerant controls of Han Chinese ethnicity.

**Results:** 
In the primary analysis, nine variants reached genome‐wide significance (p < 5e‐08), one in the carbamazepine subanalysis (85 cases vs. 77 controls) and a further eight identified in HLA‐B*15:02‐negative subanalysis (35 cases and 53 controls). Interaction analysis between each novel variant from the primary analysis found that five increased risk irrespective of HLA‐B*15:02 status or zygosity. HLA‐B*15:02‐positive individuals were found to have reduced risk if they also carried a chromosome 12 variant, chr12.9426934 (heterozygotes: relative risk = .71, p = .001; homozygotes: relative risk = .23, p < .001). All significant variants lie within intronic or intergenic regions with poorly understood functional consequence. In silico functional analysis of suggestive variants (p < 5e‐6) identified through the primary and subanalyses (stratified by HLA‐B*15:02 status and drug exposure) suggests that genetic variation within regulatory DNA may contribute to risk indirectly by disrupting the regulation of pathology‐related genes. The genes implicated were specific either to the primary analysis (CD9), HLA‐B*15:02 carriers (DOCK10), noncarriers (ABCA1), carbamazepine exposure (HLA‐E), or phenytoin exposure (CD24).

**Significance:** 
We identified variants that could explain why some carriers of HLA‐B*15:02 tolerate treatment, and why some noncarriers develop ASM‐induced SJS/TEN. Additionally, this analysis suggests that the mixing of HLA‐B*15:02 carrier status in previous studies might have masked variants contributing to susceptibility, and that inheritance of risk for ASM‐induced SJS/TEN is complex, likely involving multiple risk variants.

Keywords: antiseizure medications, cutaneous adverse drug reactions, genomics, Han Chinese, Stevens–Johnson syndrome

### Objective

Stevens–Johnson syndrome (SJS) and toxic epidermal necrolysis (TEN) are severe cutaneous adverse drug reactions. Antiseizure medications (ASMs) with aromatic ring structure, including carbamazepine, are among the most common culprits. Screening for human leukocyte antigen (*HLA*) allele *HLA*‐*B*15*:*02* is recommended prior to initiating treatment with carbamazepine in Asians, but this allele has low positive predictive value.

### Methods

We performed whole genome sequencing and analyzed 6 199 696 common variants among 113 aromatic ASM‐induced SJS/TEN cases and 84 tolerant controls of Han Chinese ethnicity.

### Results

In the primary analysis, nine variants reached genome‐wide significance (*p* < 5e‐08), one in the carbamazepine subanalysis (85 cases vs. 77 controls) and a further eight identified in *HLA*‐*B*15*:*02*‐negative subanalysis (35 cases and 53 controls). Interaction analysis between each novel variant from the primary analysis found that five increased risk irrespective of *HLA*‐*B*15*:*02* status or zygosity. *HLA*‐*B*15*:*02*‐positive individuals were found to have reduced risk if they also carried a chromosome 12 variant, chr12.9426934 (heterozygotes: relative risk = .71, *p* = .001; homozygotes: relative risk = .23, *p* < .001). All significant variants lie within intronic or intergenic regions with poorly understood functional consequence. In silico functional analysis of suggestive variants (*p* < 5e‐6) identified through the primary and subanalyses (stratified by *HLA*‐*B*15*:*02* status and drug exposure) suggests that genetic variation within regulatory DNA may contribute to risk indirectly by disrupting the regulation of pathology‐related genes. The genes implicated were specific either to the primary analysis (*CD9*), *HLA*‐*B*15*:*02* carriers (*DOCK10*), noncarriers (*ABCA1*), carbamazepine exposure (*HLA*‐*E*), or phenytoin exposure (*CD24*).

### Significance

We identified variants that could explain why some carriers of *HLA*‐*B*15*:*02* tolerate treatment, and why some noncarriers develop ASM‐induced SJS/TEN. Additionally, this analysis suggests that the mixing of *HLA*‐*B*15*:*02* carrier status in previous studies might have masked variants contributing to susceptibility, and that inheritance of risk for ASM‐induced SJS/TEN is complex, likely involving multiple risk variants.

**Keywords:** antiseizure medications, cutaneous adverse drug reactions, genomics, Han Chinese, Stevens–Johnson syndrome

## 1. INTRODUCTION

Epilepsy affects 7.6 per 1000 people worldwide.[1](#epi17182-bib-0001) The mainstay of treatment is antiseizure medications (ASMs). Although effective in most patients,[2](#epi17182-bib-0002) ASMs are associated with a range of adverse effects.[3](#epi17182-bib-0003) Idiosyncratic reactions, most commonly cutaneous reactions, are among the most problematic adverse effects, because they are largely dose‐independent.[4](#epi17182-bib-0004) Up to 10% of new users of ASMs develop some form of cutaneous adverse drug reactions (cADRs), typically between 1 and 4 weeks after commencement of ASMs, particularly for drugs with an aromatic ring structure.[5](#epi17182-bib-0005) Although the majority of cADRs are mild self‐limiting maculopapular exanthema, a minority of individuals develop more severe reactions, including Stevens–Johnson syndrome (SJS) and toxic epidermal necrolysis (TEN). These reactions are characterized by blistering exanthema of macules and targetlike lesions accompanied by, to different extents, cutaneous and mucosal detachment.[5](#epi17182-bib-0005) SJS/TEN has a mortality rate of 15%,[6](#epi17182-bib-0006) and survivors have long‐term disabilities including vision loss and scarring. There is a pressing clinical need to identify high‐risk patients and to understand the pathophysiology of these reactions.

Asians have increased incidence of SJS/TEN after commencing aromatic ASMs (e.g., 1 in 400 in Han Chinese)[7](#epi17182-bib-0007) compared to Caucasians (1 in 10 000).[8](#epi17182-bib-0008), [9](#epi17182-bib-0009) In Asians, several common genetic variants of the human leukocyte antigen (*HLA*) have been identified to be associated with ASM‐induced SJS/TEN.[10](#epi17182-bib-0010) The most reported is the *HLA*‐*B*15*:*02* allele, which is strongly predictive of SJS/TEN induced by carbamazepine across broad Asian populations, and for which pretreatment screening has been recommended by the US Food and Drug Administration (FDA) since 2007. Due to this recommendation, fewer *HLA*‐*B*15*:*02*‐positive patients have received carbamazepine, reducing incidence of SJS/TEN arising from this drug. However, due to changes in prescribing habits of physicians, there has been an increased incidence of phenytoin‐SJS/TEN.[11](#epi17182-bib-0011) Other risk alleles for carbamazepine‐induced SJS/TEN reported in Han Chinese include *HLA*‐*A*24*:*02* [12](#epi17182-bib-0012) and possibly *HLA*‐*A*31*:*01* (also reported in European Caucasians).[13](#epi17182-bib-0013) However, these HLA alleles are neither necessary nor specific and have low positive predictive values (<10%).[10](#epi17182-bib-0010) In Asians, *HLA*‐*B*15*:*02* has also been suggested to be associated with SJS/TEN induced by phenytoin[14](#epi17182-bib-0014) and oxcarbazepine,[15](#epi17182-bib-0015) although the genetic predisposition to SJS/TEN induced by ASMs other than carbamazepine is largely unknown.

We aimed to identify additional common risk variants for SJS/TEN induced by aromatic ASMs. We compared the frequencies of common variants in patients who developed ASM‐SJS/TEN (cases) and matched drug‐tolerant and population controls, followed by subanalysis of patients based on *HLA*‐*B*15*:*02* carrier status and drug‐specific exposure. We modeled interaction between each significant variant identified and *HLA*‐*B*15*:*02* status to determine whether carrying the variant modified the likelihood of developing SJS/TEN. Lastly, we investigated the potential functional impact of significant and suggestive variants.

## 2. MATERIALS AND METHODS

### 2.1. Cases and controls

Han Chinese patients with SJS/TEN induced by aromatic ASMs (carbamazepine, oxcarbazepine, lamotrigine, and phenytoin) were recruited as cases from southern China and Malaysia. As described in our previous studies,[12](#epi17182-bib-0012), [14](#epi17182-bib-0014) the diagnosis of SJS/TEN was based on the criteria by Roujeau and Stern[16](#epi17182-bib-0016) defined by skin detachment in two or more mucosal sites, and was confirmed by dermatologists. Patients who had been exposed to the matched ASMs for at least 3 months without any form of cADR were recruited as drug‐tolerant controls. Patients who had tolerated more than one ASM acted as drug‐matched controls for multiple cases. To identify potential variants that modify the effect of *HLA*‐*B*15*:*02* on the risk of carbamazepine‐induced SJS/TEN, we preferentially recruited carbamazepine‐tolerant controls who were known to carry this allele. All cases and controls were of Han Chinese descent. A venous blood sample was collected from each patient for DNA extraction. Due to the 2007 FDA recommendation, few patients who harbor *HLA*‐*B*15*:*02* have been prescribed carbamazepine or other aromatic ASMs. For this reason, recruitment of a replication cohort was not feasible.

The study was approved by the following institution and ethics committees: Joint Chinese University of Hong Kong‐New Territories East Cluster Clinical Research Ethics Committee (no. 2004.068), Ethics Committee of the Second Affiliated Hospital of Guangzhou Medical University (no. 18‐8071‐BO), University of Malaya Medical Center Medical Ethics Committee (no. 950.49), and the Ministry of Health Medical Research and Ethic Committee (no. nmrr‐13‐1157‐16170). All methods were performed in accordance with the relevant guidelines and regulations. Patients provided written informed consent and were subsequently deidentified.

### 2.2. Whole genome sequencing and bioinformatics pipeline

Whole genome sequencing was performed on the extracted DNA with an average of 30× coverage in batches using either the Illumina HiSeq X Ten platform (Kinghorn Centre for Clinical Genomics, Garvan Institute of Medical Research, Darlinghurst, NSW, Australia) or the BGI‐500 platform (BGI). The Picard/Genome Analysis Toolkit (GATK) data processing pipeline[17](#epi17182-bib-0017) was used to align raw sequence reads to the human reference genome (build GRCh38/hg38) and to call variants (details provided in [Supplementary Methods](#epi17182-sup-0007)).

### 2.3. Quality control

Variants were restricted to autosomal regions and filtered to remove likely false positives according to the following criteria: (1) GATK variant quality score recalibration truth tranche scores < 99.95, (2) not in Hardy–Weinberg equilibrium, and (3) missing rate of >2%. Within each sequencing batch, alleles with a frequency of <3% were removed to mitigate batch effects, resulting in a common variant call set with minor allele frequency of >2.5% (Figure [S1](#epi17182-sup-0001)). Individuals identified as outliers by multidimensional scaling were removed together with those with a missing rate of >2% or a heterozygosity rate deviating by >3 SD from the mean (Table [S1](#epi17182-sup-0007)). Identity by descent testing found no cryptically related individuals, and principal component analysis using 2548 individuals from the 1000 Genomes resource[18](#epi17182-bib-0018) confirmed East Asian descent for all cases and controls (Figure [S2](#epi17182-sup-0002)). Anonymized data will be shared by request from any qualified investigator under compliance with institutional policy.

### 2.4. Variant annotation

Variants were annotated using Ensembl's Variant Effect Predictor tool.[19](#epi17182-bib-0019) The Ensembl database was also used to determine whether significant or suggestive variants fell within regulatory features—regions of the DNA that regulate gene expression. These features include promoter regions, enhancers, and transcription factor binding sites. Their location in the DNA is based on evidence from multiple sources including the NIH Roadmap Epigenomics Mapping Consortium,[20](#epi17182-bib-0020) the Encyclopedia of DNA Elements (ENCODE)[21](#epi17182-bib-0021) Consortium, and the BLUEPRINT analysis portal.[22](#epi17182-bib-0022) Regulatory features are generated using genome‐wide assays including chromatin assays (DNase‐seq), histone modification assays (CHiP‐seq) and transcription factor binding assays. Expression quantitative trait loci (eQTL) and splicing quantitative trait loci (sQTL) data were obtained from the Genotype‐Tissue Expression (GTEx) portal,[23](#epi17182-bib-0023) and information on regulatory elements was sourced from the Ensembl website (Release 103, February 2021, [http://asia.ensembl.org/index.html](http://asia.ensembl.org/index.html)).

### 2.5. Statistical analysis

PLINK version 1.9[24](#epi17182-bib-0024) was used to conduct association analyses, using the allelic model, and to calculate linkage disequilibrium (LD) between variants.

In the primary analysis, we compared allele frequencies between all ASM‐SJS/TEN cases and ASM‐tolerant controls. We performed subanalyses by stratifying cases and controls based on their *HLA*‐*B*15*:*02* carrier status, and by comparing cases and tolerant controls exposed only to carbamazepine, or phenytoin. We did not perform subanalysis for oxcarbazepine or lamotrigine due to limited sample sizes. We used the gnomAD database, viewed in March 2021, and Fisher exact test to compare significant variant allele frequencies to frequencies observed in East Asian population controls. The interaction between genome‐wide significant variants and *HLA*‐*B*15*:*02* status was investigated using generalized linear model with Poisson distribution, log‐link, and robust variance in Stata version 16 (StataCorp).

The standard genome‐wide *p*‐value cutoff for statistical significance of 5e‐08 was applied to identify variants that modified risk. To interrogate variant enrichment in subthreshold regions (suggestive variants) observed in the quantile–quantile plots, three criteria were applied: (1) a *p*‐value between 5e‐06 and 5e‐08, (2) five or more variants in LD within a region that harbors a regulatory feature, and (3) evidence from GTEx data that a variant alters gene expression. The three most significant findings for each variant are reported in the [supplementary tables](#epi17182-sup-0006).

For the genome‐wide significant ASM variants, we used Fisher exact test to compare their frequencies between cases and population controls obtained from the gnomAD (version 3) East Asian genomes. Applying Bonferroni correction for multiple testing, *p* < .01 was considered statistically significant. Figures were generated using R Studio packages ggplot2[25](#epi17182-bib-0025) and qqman,[26](#epi17182-bib-0026) and locusZoom version 1.4.[27](#epi17182-bib-0027)

## 3. RESULTS

### 3.1. Patient characteristics

A total of 116 aromatic ASM‐SJS/TEN cases and 85 aromatic ASM‐tolerant controls were recruited and underwent whole genome sequencing. Data from four individuals (three cases and one control) failed quality control ([Supplementary Methods](#epi17182-sup-0007)), leaving 113 cases and 84 controls available for analysis (Table [1](#epi17182-tbl-0001)). In the majority of cases, SJS/TEN was induced by carbamazepine (85/113, 75.2%), followed by lamotrigine (14/113, 12.4%) and phenytoin (11/113, 9.7%). The power of the study was calculated using the online GCTA‐GREML Power Calculator ([https://shiny.cnsgenomics.com/gctaPower/](https://shiny.cnsgenomics.com/gctaPower/)), which can estimate power relative to heritability based on our numbers of cases and controls, disease prevalence in the general population (1 in 1 000 000), and estimated type I error (.05). Due to the rarity of SJS/TEN in the population, lower samples sizes are needed to detect differences based on the parameters of the calculator. For this sample size, ~80% power can detect variants with heritability of ~75%. Therefore, this dataset had the capacity to detect variants despite its small sample size (Figure [S3](#epi17182-sup-0003)).

### TABLE 1.

| Characteristic | Cases, n = 113 | Tolerant, n = 84 |
| --- | --- | --- |
| Female sex | 56 | 42 |
| Drugs |  |  |
| Carbamazepine | 85 | 77 |
| Lamotrigine | 14 | 15 |
| Phenytoin | 11 | 40 |
| Oxcarbazepine | 3 | 4 |
| HLA‐B*15:02 |  |  |
| Carrier | 78 | 31 |
| Noncarrier | 35 | 53 |

Table 1 Caption: Characteristics of cases and controls

Through retrospective screening, we identified 69% (78/113) of cases as *HLA*‐*B*15*:*02*+ and 40% (31/84) of controls to be *HLA*‐*B*15*:*02*+. The Allele Frequency Net Database identified 9.38% (712/7595) of people from the Hong Kong Chinese Bone Marrow Registry ([http://www.allelefrequencies.net/](http://www.allelefrequencies.net/)) as being *HLA*‐*B*15*:*02* carriers. We identified that both ASM cases (Fisher exact odds ratio [OR] = 21.5, *p* = 1.27e‐51) and ASM‐tolerant controls (OR = 5.65, *p* = 1.27e‐11) significantly differed from the population controls in *HLA*‐*B*15*:*02* carrier frequency. This confirmed our biased selection of *HLA*‐*B*15*:*02*+ ASM‐tolerant controls. As far as the authors are aware, this is the largest known cohort of *HLA*‐*B*15*:*02*+ ASM‐tolerant controls.

### 3.2. ASM‐SJS/TEN cases versus all ASM‐tolerant subjects

After quality control, a total of 6 198 332 common variants were available for analyses. The genomic control value (λ_GC_) for comparison between all ASM‐induced SJS/TEN and all ASM‐tolerant controls was 1.014, confirming minimal inflation (Figure [1A](#epi17182-fig-0001).1). Nine variants reached genome‐wide significance, with three being intergenic (chr3:73919920 [OR = 5.2, *p* = 3.6e‐08], chr8:21314026 [OR = 3.2, *p* = 3.3e‐08], chr12:131623246 [OR = 5.7, *p* = 2.0e‐09]) and the remainder falling within introns (chr4:820728 [OR = .3, *p* = 2.5e‐08], chr8:98263823 [OR = 4.0, *p* = 6.2e‐09], chr9:63832271 [OR = 6.5, *p* = 2.8e‐08], chr10:38643144 [OR = 5.4, *p* = 8.8e‐08], chr12:9426934 [OR = .3, *p* = 1.2e‐08]) of various genes (Figure [1A](#epi17182-fig-0001).2, Table [2](#epi17182-tbl-0002)). For two of these variants (chr12:9426934 and chr4:820728), the minor allele was underrepresented in ASM‐SJS/TEN. These variants were not in high LD with other significant or suggestive variants (Figure [S4](#epi17182-sup-0004)).

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3778/9541367/a42b376d86d9/EPI-63-936-g001.jpg)

Quantile–quantile and Manhattan plots of the common variant association analysis testing 6 198 332 variants per group with at least one genome‐wide significant variant. (A) All ASMs (λ = 1.015). (B) HLA‐B*15:02‐negative cohort (λ = 1.017). (C) Carbamazepine cohort (λ = 1.014). (1) Quantile–quantile plots. Red line: expected = observed. (2) Manhattan plots. Red solid horizontal line = 5e‐08 (genome‐wide significance); Blue dotted horizontal line = 1e‐05 (suggestive variants). Lambda (λ) was calculated using the PLINK association function

### TABLE 2.

| Variant location | HGVS name | Variant reference cluster ID | Gene ID | A1 | A2 | Controls, homozygous A1/heterozygous/homozygous A2 (MAF) | Cases, homozygous A1/heterozygous/homozygous A2 (MAF) | OR (95% CI) | p |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| All ASMs, 113 cases vs. 84 controls |  |  |  |  |  |  |  |  |  |
| chr12:131623246 | NC_000012.12:g.131623246T>C | rs4471527 | Intergenic | T | C | 0/14/70 (.08) | 5/67/41 (.34) | 5.7 (3.1–10.5) | 2.04e‐09 |
| chr8:98263823 | NC_000008.11:g.98263831del | rs199755581 | NIPAL2 (intronic) | CA | C | 3/22/59 (.17) | 36/26/48 (.45) | 4 (2.5–6.5) | 6.23e‐09 |
| chr10:38643144 | NC_000010.11:g.38643144A>G | rs1297852527 | SLC9B1P3 (intronic) | G | A | 0/14/70 (.08) | 0/74/39 (.33) | 5.4 (2.9–9.9) | 8.75e‐09 |
| NC_000010.10:g.38936275A>G |  |  |  |  |  |  |  |  |  |
| NG_025429.2:g.46033T>C |  |  |  |  |  |  |  |  |  |
| chr12:9426934 | NC_000012.12:g.9426934G>C | rs77491650 | DDX12P (intronic) | C | G | 14/58/12 (.51) | 2/49/62 (.23) | .3 (.2–.4) | 1.21e‐08 |
| chr4:820728 | NC_000004.12:g.820783_820819ins | rs59567505 and rs143960439 | CPLX1 (intronic) | G | GCACCCCTCACCAGCCTCACGTGAACCCCCAAGGTGGA | 24/35/25 (.49) | 14/22/77 (.22) | .3 (.2–.5) | 1.49e‐08 |
| chr9:63832271 | NC_000009.12:g.63832271G>T | rs77542827 | FRG1JP (intronic) | T | G | 0/9/75 (.05) | 0/61/52 (.27) | 6.5 (3.1–13.6) | 2.76e‐08 |
| chr8:21314026 | NC_000008.11:g.21314045_21314046insAGCTGGGAGTCAGTGAGAAAGAACAACACTGGGATCCAGTCCGG | rs778096762 | Intergenic | C | CACACTGGGATCCAGTCCGGAGCTGGGAGTCAGTGAGAAAGAACA | 14/23/47 (.3) | 50/31/31 (.58) | 3.2 (2.1–4.9) | 3.29e‐08 |
| chr3:73919920 | NC_000003.12:g.73919922_73919923del | rs374138762 | Intergenic | T | TTA | 0/13/71 (.08) | 0/69/44 (.31) | 5.2 (2.8–9.9) | 3.56e‐08 |
| chr10:38643136 | NC_000010.11:g.38643136C>T | rs879656274 | SLC9B1P3 (intronic) | T | C | 0/15/69 (.09) | 0/73/40 (.32) | 4.9 (2.7–8.9) | 3.61e‐08 |
| Carbamazepine, 85 cases vs. 77 controls |  |  |  |  |  |  |  |  |  |
| chr12:131623246 | NC_000012.12:g.131623246T>C | rs4471527 | Intergenic | T | C | 0/13/64 (.08) | 3/53/29 (.35) | 5.8 (3.0–11.0) | 1.36e‐08 |
| HLA‐B*15:02‐negative, 35 cases vs. 53 controls |  |  |  |  |  |  |  |  |  |
| chr6:149444135 | NC_000006.12:g.149444146_149444267del | rs1562468327 | Intergenic | TCAGCCAGTGTGTCAGTCAGCCAGTGTTAGTCAGCCAGTGTGTCAGTCAGCCAGTGTCAGCCACCCAGTGTCAGTCAGCCAGTGTGTCAGCCACTGTCAGCCAATGTCAGCCAGTGTGTCAGC | T | 2/7/44 (.10) | 13/11/11 (.53) | 9.7 (4.4–21.1) | 5.90e‐10 |
| chr8:98263823 | NC_000008.11:g.98263831 CA>C a | rs199755581 | NIPAL2 (intronic) | CA | C | 0/11/42 (.10) | 14/8/12 (.53) | 9.7 (4.4–21.3) | 6.84e‐10 |
| chr21:9790175 |   | Not reported in dbSNP | LINC01667 (intronic) | CCTCTCTCCAGGCTCACACATTGAAGAGAA | C | 0/17/36 (.16) | 16/9/10 (.59) | 7.4 (3.7–15) | 4.22e‐09 |
| chr9:63832271 | NC_000009.12:g.63832271G>T | rs77542827 | FRG1JP (intronic) | T | G | 0/3/50 (.03) | 0/24/11 (.34) | 17.9 (5.1–62.5) | 1.45e‐08 |
| chr17:26783109 | NC_000017.11:g.26783109G>A | rs1286845082 | Intergenic | A | G | 0/2/51 (.02) | 0/22/13 (.31) | 23.8 (5.4–105.5) | 2.28e‐08 |
| chr17:26783113 | NC_000017.11:g.26783113T>C | rs1597607761 | Intergenic | C | T | 0/1/52 (.01) | 0/20/15 (.29) | 42 (5.5–321.8) | 3.13e‐08 |
| chr3:183730061 | NC_000003.12:g.183730061T>G | rs1391213386 | YEATS2 (intronic) | G | T | 1/3/49 (.05) | 5/16/14 (.37) | 11.9 (4.3–33.1) | 3.26e‐08 |
| chr4:39690137 | NC_000004.12:g.39690141_39690142del | rs1211926109 | Intergenic | AAT | A | 16/18/19 (.47) | 0/5/29 (.07) | .1 (.03–.2) | 3.55e‐08 |

Table 2 Caption: Variants with genome‐wide significance

### 3.3. HLA‐B*15:02 stratification analysis

Among individuals who did not carry the *HLA*‐*B*15*:*02* allele (35 cases and 53 controls), eight variants reached genome‐wide significance, including two, chr8:98263823 (OR = 9.7, *p* = 6.8e‐10) and chr9:63832271 (OR = 17.9, *p* = 1.5e‐08), that were identified in the primary analysis (Table [2](#epi17182-tbl-0002)). No variants reached genome‐wide significance among *HLA*‐*B*15*:*02*‐positive individuals.

### 3.4. Drug‐specific subanalyses irrespective of HLA‐B*15:02 status

Subanalysis of the carbamazepine‐induced SJS/TEN cases (*n* = 85) and tolerant controls (*n* = 77) identified a single genome‐wide significant variant, chr12:131623246 (OR = 5.8, *p* = 1.4e‐08), that had been identified in the primary analysis (Figure [1B](#epi17182-fig-0001), Table [2](#epi17182-tbl-0002)). No genome‐wide significant variants were identified in the subanalysis specific to phenytoin (11 phenytoin‐SJS/TEN cases and 40 phenytoin‐tolerant controls). This was likely due to the small sample sizes, as indicated in the quantile–quantile plot (Figure [S5](#epi17182-sup-0005)).

### 3.5. Significant variants differ from population controls

Comparison of significant variant allele frequencies in cases in our primary and subanalyses with those in East Asian population controls, as reported in the gnomAD genome aggregation database, found all to be significantly different (Table [S2](#epi17182-sup-0007)).

### 3.6. Interaction analysis of significant variants and HLA‐B*15:02 carrier status

The interaction analysis showed that carrying one of the significant variants identified in the primary analysis changed the likelihood of having developed SJS/TEN (Table [3](#epi17182-tbl-0003), Table [S3](#epi17182-sup-0007)). *HLA*‐*B*15*:*02* carriers who were homozygous for chr12.9426934 had .23 times the risk of those who carry only the wild‐type allele, whereas heterozygotes had .71 times the risk. For six variants, risk was increased irrespective of *HLA*‐*B*15*:*02* status or zygosity. In those carrying *HLA*‐*B*15*:*02*, the relative risk ranged from 1.39 (heterozygous carriers of chr9:63832271) to 1.66 (heterozygous carriers of chr10:38643144). In individuals negative for *HLA*‐*B*15*:*02*, the relative risk ranged from 3.90 (heterozygous carriers of chr3:73919920) to 4.93 (heterozygous carriers of chr9:63832271). Presence of the chr8:98263823 variant, which reached genome‐wide significance in both the primary and *HLA*‐*B*15*:*02*‐negative subanalyses, increased risk for *HLA*‐*B*15*:*02*‐heterozygous carriers and for noncarriers of *HLA*‐*B*15*:*02*, irrespective of zygosity.

### TABLE 3.

| Variant | Neg PP, % | PP Neg + A1, % | RR a (p), Neg vs. Neg + A1 | Pos PP, % | Pos PP + A1, % | RR a (p), Pos vs. Pos + A1 |
| --- | --- | --- | --- | --- | --- | --- |
| chr3:73919920 | 20.3 | het = 79.3 | het: 3.90 (<.001) | 57.1 | het = 86.8 | het: 1.52 (<.001) |
| chr4:820728 | 68.4 | het = 20.7 hom = 14.3 | het: .30 (<.001) hom: .21 (<.001) | 79.7 | het = 57.1 hom = 64.7 | het: .72 (.034) hom: .81 (.237) |
| chr8:21314026 | 18.4 | het = 39.3 hom = 77.3 | het: 2.13 (.062) hom: 4.19 (<.001) | 60.0 | het = 76.9 hom = 78.6 | het: 1.28 (.136) hom: 1.31 (.064) |
| chr8:98263823 | 22.2 | het = 42.1 hom = 100 | het: 1.89 (.) hom: 4.50 (.) | 67.9 | het = 62.1 hom = 88.0 | het: .91 (.) hom: 1.30 (.) |
| chr9:63832271 | 18.0 | het = 88.9 | het: 4.93 (<.001) | 62.1 | het = 86.0 | het: 1.39 (.003) |
| chr10:38643136 | 17.0 | het = 74.3 | het: 4.37 (<.001) | 55.4 | het = 88.7 | het: 1.60 (<.001) |
| chr10:38643144 | 15.4 | het = 75.0 | het: 4.87 (<.001) | 54.4 | het = 90.4 | het: 1.66 (<.001) |
| chr12:9426934 | 75.7 | het = 25.0 hom = 9.1 | het: .33 (<.001) hom: .12 (<.001) | 88.9 | het = 62.7 hom = 20.0 | het: .71 (.001) hom: .23 (<.001) |
| chr12:131623246 | 21.1 | het = 71.4 hom = 100 | het: 3.39 (<.001) hom: 4.75 (<.001) | 53.7 | het = 88.7 hom = 100 | het: 1.65 (<.001) hom: 1.86 (<.001) |

Table 3 Caption: ASM genome‐wide significant regulator variants alter probability and RR of ASM‐related Stevens–Johnson syndrome/toxic epidermal necrolysis

### 3.7. Predicted regulatory function of noncoding significant and suggestive variants

Among the 15 unique genome‐wide significant variants of the various analyses, only one had in silico predicted functional consequence. The chr4:820728 variant falls within an intronic region of the complexin 1 (*CPLX1*) gene. eQTL data associate the variant with expression of solute carrier family 26 member 1 (*SLC26A1*) in multiple tissues and cell types, including skin and cultured fibroblasts. sQTL data show potential to modify the splicing of two further genes in skin, alpha‐L‐iduronidase (*IDUA*) and transmembrane protein 175 (*TMEM175*; Table [S4](#epi17182-sup-0007)).

We then further explored the potential functional consequence of suggestive variants. In the primary analysis, the Manhattan plot identified a cluster of eight suggestive variants on chromosome 12 (Figure [1A](#epi17182-fig-0001).2). These variants fall within a 2.6‐kb region 94.6 kb downstream of the chr12:9426934 variant and 125 Mb downstream of chr12:131623246. They lie between the pleckstrin homology and RhoGEF domain containing G6 (*PLEKHG6*) gene and the CD9 molecule (*CD9*) gene (Figure [2A](#epi17182-fig-0002)). This region is enriched for chromatin marks that are often found near regulatory elements, and six of the variants fall within promoter flanking regions at this locus. One also overlaps a region where the CCCTC‐binding factor (CTCF) binds to DNA. CTCF is a highly conserved transcription factor that can function as an activator, repressor, or insulator (blocking communication between enhancers and promoters). Tissue‐ and cell type‐specific eQTL data indicate that seven of the eight variants alter the expression levels of both genes in cultured fibroblast cells and testis tissue (Table [S5](#epi17182-sup-0007)).

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3778/9541367/caf758c05018/EPI-63-936-g002.jpg)

Distinct clusters of suggestive variants. (A) Antiseizure medication cohort. (B) HLA‐B*15:02‐negative cohort. (C) HLA‐B*15:02‐positive cohort. (D) Carbamazepine‐exposed cohort. (E, F) Phenytoin‐exposed cohort. The reference single nucleotide polymorphism (SNP) is depicted as a pink diamond. The color coding for all other SNPs indicates linkage disequilibrium relative to the top SNP, as follows: red, r 2 ≥ .8; orange, .6 ≤ r 2 < .8; green, .4 ≤ r 2 < .6; blue, .2 ≤ r 2 < .4; light blue, r 2 < .2. Gray represents unknown n r 2 from the 1000 Genomes database. The bottom row represents the gene annotations.

In addition to the eight genome‐wide significant variants identified in the subanalysis of non‐*HLA*‐*B*15*:*02* carriers, there were five suggestive variants that fall within an intronic region of the ATP binding cassette subfamily A member 1 (*ABCA1*) gene on chromosome 9 (Figure [2B](#epi17182-fig-0002)). These five were predicted to modify expression of an RNA gene: *ENSG00000226334.1* (Table [S6](#epi17182-sup-0007)).

In the analysis of *HLA*‐*B*15*:*02* carriers, 17 suggestive variants were found to cluster on chromosome 2. These variants fall within an intronic region of dedicator of cytokinesis 10 (*DOCK10*; Figure [2C](#epi17182-fig-0002)), and seven are predicted to alter the splicing ratio of *DOCK10*. In addition, 14 lie within a promoter flanking region of microRNA gene *ENSR00000638436* (*MIR4439*), and four of these also overlap a CTCF binding site (Table [S7](#epi17182-sup-0007)).

In the subanalysis of individuals exposed to carbamazepine, seven suggestive variants overlap the *HLA*‐*C* and *HLA*‐*B* gene region on chromosome 6 (Figure [2D](#epi17182-fig-0002)). Five of these variants are predicted to modify expression of HLA‐related molecules, including *HLA‐C*, MHC class I polypeptide‐related sequence B (*MICB*), and an HLA complex group 4 (*HCG4*) pseudogene (*ENSG00000271581.1*). All five variants were also predicted to alter the splicing of *HLA*‐*E* (Table [S8](#epi17182-sup-0007)).

In the analysis of phenytoin‐treated individuals, we identified suggestive variants on chromosome 6 (*n* = 36) and chromosome 11 (*n* = 16; Table [S9](#epi17182-sup-0007)). Variants on chromosome 6 are intergenic and nearby the *CD24* gene. Nine of these variants overlap regulatory DNA regions and are predicted to alter *CD24* expression in multiple tissues including the skin (Figure [2E](#epi17182-fig-0002), Table [S9](#epi17182-sup-0007)). The chromosome 11 variants lie within an intronic region of the RNA gene *AP003066*.*1*, and were predicted to modify expression of this RNA gene in regulatory T cells (Figure [2F](#epi17182-fig-0002), Table [S9](#epi17182-sup-0007)).

## 4. DISCUSSION

Our study interrogated >6 million common variants, considerably more than previous genome‐wide association studies of ASM‐induced SJS/TEN, which have typically included .5–1.2 million variants.[13](#epi17182-bib-0013), [28](#epi17182-bib-0028) We also have one of the largest cohorts to interrogate the whole genome of ASM‐SJS/TEN cases and matched drug‐tolerant controls. The previous papers by McCormack et al.[13](#epi17182-bib-0013) included 22 hypersensitivity syndrome subjects (12 with SJS/TEN) and 2691 healthy controls for their association. Unlike previous studies, we purposely recruited a large proportion of *HLA*‐*B*15*:*02* carriers who tolerated treatment. This large number of tolerant individuals with the *HLA*‐*B*15*:*02* allele enabled secondary analyses stratified by *HLA*‐*B*15*:*02* carrier status. We were able to perform treatment‐specific analysis for carbamazepine and phenytoin only, as there were too few cases for other drugs, and could not stratify analysis based on *HLA*‐*B*15*:*02* carrier status by drug. The analyses identified novel significant and suggestive variants located in non‐protein‐coding regions of the DNA, some of which have evidence to support functional effect on cell type‐specific gene expression.

Despite our relatively small sample size compared to other genome‐wide association studies, the primary analysis identified nine significant variants. This seemingly large number of variants is not unusual for detecting pharmacogenomic responses that are not as tightly controlled by natural selection, unlike disease‐related variants, and aligns with our power calculation. This has been observed in the previous genome‐wide association studies in this area with a large number of variants despite having a small sample size.[13](#epi17182-bib-0013), [28](#epi17182-bib-0028) Based on the lambda value of 1.015, we concluded that the variants identified were not being driven by population stratification, as it was less than the commonly applied 1.05 threshold.[29](#epi17182-bib-0029) However, there may be cryptic population structure that is unresolvable with the current methodology. Interestingly, in *HLA*‐*B*15*:*02* carriers, two of these variants reduced the relative risk of developing SJS/TEN and might explain why some of these carriers do not go on to develop SJS/TEN following treatment with a high‐risk drug. These findings also suggest that carrying specific combinations of significant variants can modify predisposition to a severe drug reaction.

Our study specifically analyzed risk in a cohort of *HLA*‐*B*15*:*02*‐negative individuals. Although rarer, as the negative predictive value approaches 100% without *HLA*‐*B*15*:*02*,[10](#epi17182-bib-0010) some of these individuals are nevertheless at risk of SJS/TEN. This analysis identified eight variants that can potentially explain this risk, including one, chr8:98263823, that also reached genome‐wide significance in the primary analysis. Interaction modeling showed that risk for *HLA*‐*B*15*:*02*‐negative individuals was increased, irrespective of zygosity, if they carried this variant, or one of six other genome‐wide significant variants identified in the primary analyses.

Evidence for functional analysis was found for only one of the 15 unique significant variants identified: the chr4:820728 variant that falls within an intronic region of *CPLX1*. This variant contributes to expression variation of multiple nearby genes, including *SLC26A1* and *IDUA*, in several tissues including skin. It is feasible that a variant impacts on the function of distal genes rather than the gene it falls within; for instance, the modulation of *IRX* family gene expression by intronic variants within the "obesity" *FTO* gene has been well described.[30](#epi17182-bib-0030)

Although these variants have relatively high OR for a whole genome sequencing study, they may only marginally alter the absolute risk of SJS/TEN due its overall rare occurrence in the population. Translating the OR of any association into predictive value needs to be interpreted in the context of the pretest probability of the condition in the test population. Although the negative predictive value of *HLA*‐*B*15*:*02* is very high for carbamazepine‐induced SJS/TEN, this has not been demonstrated for SJS/TEN induced by other aromatic ASMs, and approximately one third of our cases did not carry the *HLA*‐*B*15*:*02* allele. The positive predictive value of *HLA*‐*B*15*:*02* for carbamazepine‐induced SJS/TEN is estimated to be <10%. Therefore, additional risk variants identified in this population may still be clinically meaningful in refining risk stratification, and this dataset highlights why segregating *HLA*‐*B*15*:*02* carrier status is important to further understanding the complexities of SJS/TEN inheritance.

The quartile–quartile plot indicated enrichment of variants in subthreshold regions, which suggests a polygenic inheritance pattern. This pattern has been observed in other complex inherited traits such as schizophrenia and human height.[31](#epi17182-bib-0031) In our study, the quartile–quartile plot variant enrichment indicated that a cluster of variants impacts the same genomic region, but need not be the same in all individuals. The polygenic architecture does not yield specific candidates for clinical screening; however, it provides insights into candidate DNA regions with biological importance for risk of ASM‐induced SJS/TEN.

The primary analysis identified an area of enrichment for suggestive variants on chromosome 12 between the *CD9* and *PLEKHG6* genes. eQTL and sQTL data show that the variants potentially impact the regulation of *CD9* and *PLEKHG6* expression in skin tissue. Notably, CD9 is an exosomal marker, and CD9‐expressing exosomes identified in the serum of SJS/TEN patients have recently been published.[32](#epi17182-bib-0032) The authors attribute keratinocyte cell death to ferroptosis, which is a type of iron‐dependent regulated cell death, whereby the deleterious factors are delivered in exosomes internalized by keratinocytes. Our data suggest that *CD9* may be an important gene for SJS/TEN caused by aromatic ASMs.

The suggestive variants identified in the *HLA*‐*B*15*:*02*‐positive subanalysis might affect splicing of the *DOCK10* gene. The encoded protein is a member of the DOCK family of Rho GTPase activators and may have a role in activating T cells[33](#epi17182-bib-0033) and the development of CD8+ T cells.[34](#epi17182-bib-0034) *DOCK10* is also differentially downregulated in contact dermatitis when presented with sensitizers.[35](#epi17182-bib-0035) Our data suggests that genetic variation may modify risk of developing ASM‐induced SJS/TEN via perturbation of DOCK10 activity in *HLA*‐*B*15*:*02*‐positive individuals.

Suggestive variants identified in the carbamazepine subanalysis were predicted to impact the function of HLA molecules. In particular, five variants have the potential to modify the splicing ratio of HLA‐E, a nonclassical MHC class I molecule. Peptides can be presented on HLA‐E either via the CD94:NKG2 receptor[36](#epi17182-bib-0036) or the T‐cell receptor.[37](#epi17182-bib-0037) Keratinocytes from affected skin in patients with SJS/TEN and other exanthemas have been shown to express HLA‐E and be sensitized to killing by cytotoxic T lymphocytes expressing the CD94/NKG2C receptor.[38](#epi17182-bib-0038) Although *HLA*‐*B*15*:*02* remains a robust marker of carbamazepine‐induced SJS/TEN reactions, these findings suggest that genetic variants within regulatory DNA regions of chromosome 6 may indirectly contribute to risk of pathogenesis by modifying the function of additional HLA molecules.

The subanalysis of individuals exposed to phenytoin identified suggestive variants with potential to alter the expression of *CD24* in skin tissue. Interestingly, *CD24* has been found to be significantly differentially expressed with dithranol treatments in keratinocytes from psoriasis patients.[39](#epi17182-bib-0039) *CD24* can be expressed by many immune cells, including T cells, where it acts as a costimulatory molecule.[40](#epi17182-bib-0040) B cells expressing CD24+ from patients with blistering autoimmune skin disease (bullous pemphigoid) fail to repress CD4+ T‐cell inflammatory proliferation.[41](#epi17182-bib-0041) If mice CD4+ T cells expressed CD24, they were not protected from concanavalin A‐induced liver injury.[42](#epi17182-bib-0042) Therefore, genetic variation that modifies the function of *CD24* in either T cells, B cells, or keratinocytes may contribute to susceptibility to phenytoin‐induced SJS/TEN.

Information on the potential functional consequence of the non‐protein‐coding variants identified is limited. Here, we have made inferences about possible consequence based on the most reliable data available. Functional studies that could include CRISPR/Cas9, ChIP‐seq experiments, or functional genomic approaches (i.e., differential expression studies utilizing either transcriptomics or proteomics) are needed to validate our conjectures.

The functional consequences associated with suggestive variants support the conjecture that non‐protein‐coding variants modify susceptibility to ASM‐induced SJS/TEN. It should also be noted that current databases report on potential eQTL effects at baseline conditions only, and the functional consequences of variants that become apparent under certain dynamic conditions such as drug exposure, which have been termed “response eQTL,”[43](#epi17182-bib-0043) remain elusive. The genes implicated in this study warrant further functional interrogation to better understand the molecular mechanisms underlying ASM‐induced SJS/TEN.

In conclusion, our analysis highlights that mixing *HLA*‐*B*15*:*02* carrier status in previous studies might have masked variants and genes contributing to susceptibility. Our interaction analyses suggest that presence of additional variants may reduce risk conferred by *HLA*‐*B*15*:*02*, explaining why some carriers tolerate ASMs. These findings, coupled with variant enrichment observed in our quantile–quantile plots, suggest that inheritance of risk in SJS/TEN may involve multiple markers of small effect size, as has been observed in other complex inherited traits such as human height.[44](#epi17182-bib-0044) In this scenario, the combination of risk variants may be heterogeneous across those susceptible, but may disrupt pathways common to the pathology.

## CONFLICT OF INTEREST

Z.C. is supported by an Early Career Fellowship from the National Health and Medical Research Council of Australia (GNT1156444), and he/his institution has received consultancy fees and/or research grants from Arvelle Therapeutics and UCB Pharma. None of the other authors has any conflict of interest to disclose.

## AUTHORS CONTRIBUTIONS

K.A.M. processed, analyzed, and interpreted the whole genome sequence data and was a major contributor in writing the manuscript. A.A. also contributed to whole genome sequence data processing, analysis, and interpretation, as well as contributing to the writing process. Z.C. contributed to the analysis and interpretation of the Stata results. P.K., Y.‐W.S., C.‐C.N., L.B., S.C., S.P., P.C.S., K.‐S.L., and W.‐P.L. aided with the acquisition of samples, project design, and draft revisions. P.K. conceived and supervised the study. J.‐H.D. contributed to the whole genome sequence data processing and initial analysis. All authors read and approved the final manuscript.

## Supporting information

## ACKNOWLEDGMENTS

This research was supported by the Hong Kong Health and Medical Research Fund (01120086) and Australian National Health and Medical Research Council (GNT1103979; both to P.K), and the Science and Technology Project of Guangzhou Fund (201904020028 to W.‐P.L., 201904010275 to Y.‐W.S.). K.A.M. is supported by an Australian Government Research Training Program. P.K. is supported by a Medical Research Future Fund Practitioner Fellowship (MRF1136427). K.A.M. acknowledges her other PhD supervisors Dr Nicole A. Mifsud, Dr Patricia T. Illing, and Prof Anthony W. Purcell for their input and support. Open access publishing facilitated by Monash University, as part of the Wiley ‐ Monash University agreement via the Council of Australian University Librarians.

Mullan KA, Anderson A, Shi Y‐W, Ding J‐H, Ng C‐C, Chen Z, et al. Potential role of regulatory DNA variants in modifying the risk of severe cutaneous reactions induced by aromatic anti‐seizure medications. Epilepsia. 2022;63:936–949. 10.1111/epi.17182

## Contributor Information

Wei‐Ping Liao, Email: wpliao@163.net.

Patrick Kwan, Email: patrick.kwan@monash.edu.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Beghi E, Giussani G. Aging and the epidemiology of epilepsy. Neuroepidemiology. 2018;51:216–23. doi: 10.1159/000493484.  [DOI](https://doi.org/10.1159/000493484) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30253417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuroepidemiology&title=Aging%20and%20the%20epidemiology%20of%20epilepsy&author=E%20Beghi&author=G%20Giussani&volume=51&publication_year=2018&pages=216-23&pmid=30253417&doi=10.1159/000493484&)

2. Chen Z, Brodie MJ, Liew D, Kwan P. Treatment outcomes in patients with newly diagnosed epilepsy treated with established and new antiepileptic drugs: a 30‐year longitudinal cohort study. JAMA Neurol. 2018;75:279–86. doi: 10.1001/jamaneurol.2017.3949.  [DOI](https://doi.org/10.1001/jamaneurol.2017.3949) | [PMC free article](/articles/PMC5885858/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29279892/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=JAMA%20Neurol&title=Treatment%20outcomes%20in%20patients%20with%20newly%20diagnosed%20epilepsy%20treated%20with%20established%20and%20new%20antiepileptic%20drugs:%20a%2030%E2%80%90year%20longitudinal%20cohort%20study&author=Z%20Chen&author=MJ%20Brodie&author=D%20Liew&author=P%20Kwan&volume=75&publication_year=2018&pages=279-86&pmid=29279892&doi=10.1001/jamaneurol.2017.3949&)

3. Sharma S, Kwan P. The safety of treating newly diagnosed epilepsy. Expert Opin Drug Saf. 2019;18:273–83. doi: 10.1080/14740338.2019.1602607.  [DOI](https://doi.org/10.1080/14740338.2019.1602607) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30943798/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin%20Drug%20Saf&title=The%20safety%20of%20treating%20newly%20diagnosed%20epilepsy&author=S%20Sharma&author=P%20Kwan&volume=18&publication_year=2019&pages=273-83&pmid=30943798&doi=10.1080/14740338.2019.1602607&)

4. Borrelli EP, Lee EY, Descoteaux AM, Kogut SJ, Caffrey AR. Stevens‐Johnson syndrome and toxic epidermal necrolysis with antiepileptic drugs: an analysis of the US Food and Drug Administration Adverse Event Reporting System. Epilepsia. 2018;59:2318–24. doi: 10.1111/epi.14591.  [DOI](https://doi.org/10.1111/epi.14591) | [PMC free article](/articles/PMC6420776/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30395352/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=Stevens%E2%80%90Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis%20with%20antiepileptic%20drugs:%20an%20analysis%20of%20the%20US%20Food%20and%20Drug%20Administration%20Adverse%20Event%20Reporting%20System&author=EP%20Borrelli&author=EY%20Lee&author=AM%20Descoteaux&author=SJ%20Kogut&author=AR%20Caffrey&volume=59&publication_year=2018&pages=2318-24&pmid=30395352&doi=10.1111/epi.14591&)

5. Mustafa SS, Ostrov D, Yerly D. Severe cutaneous adverse drug reactions: presentation, risk factors, and management. Curr Allergy Asthma Rep. 2018;18:26.  [DOI](https://doi.org/10.1007/s11882-018-0778-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29574562/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Allergy%20Asthma%20Rep&title=Severe%20cutaneous%20adverse%20drug%20reactions:%20presentation,%20risk%20factors,%20and%20management&author=SS%20Mustafa&author=D%20Ostrov&author=D%20Yerly&volume=18&publication_year=2018&pages=26&pmid=29574562&doi=10.1007/s11882-018-0778-6&)

6. Micheletti RG, Chiesa‐Fuxench Z, Noe MH, Stephen S, Aleshin M, Agarwal A, et al. Stevens‐Johnson syndrome/toxic epidermal necrolysis: a multicenter retrospective study of 377 adult patients from the United States. J Invest Dermatol. 2018;138:2315–21. doi: 10.1016/j.jid.2018.04.027.  [DOI](https://doi.org/10.1016/j.jid.2018.04.027) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29758282/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Invest%20Dermatol&title=Stevens%E2%80%90Johnson%20syndrome/toxic%20epidermal%20necrolysis:%20a%20multicenter%20retrospective%20study%20of%20377%20adult%20patients%20from%20the%20United%20States&author=RG%20Micheletti&volume=138&publication_year=2018&pages=2315-21&pmid=29758282&doi=10.1016/j.jid.2018.04.027&)

7. Hung SI, Chung WH, Chen YT. HLA‐B genotyping to detect carbamazepine‐induced Stevens‐Johnson syndrome: implications for personalizing medicine. Per Med. 2005;2:225–37. doi: 10.2217/17410541.2.3.225.  [DOI](https://doi.org/10.2217/17410541.2.3.225) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29793265/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Per%20Med&title=HLA%E2%80%90B%20genotyping%20to%20detect%20carbamazepine%E2%80%90induced%20Stevens%E2%80%90Johnson%20syndrome:%20implications%20for%20personalizing%20medicine&author=SI%20Hung&author=WH%20Chung&author=YT%20Chen&volume=2&publication_year=2005&pages=225-37&pmid=29793265&doi=10.2217/17410541.2.3.225&)

8. De A, Rajagopalan M, Sarda A, Das S, Biswas P. Drug reaction with eosinophilia and systemic symptoms: an update and review of recent literature. Indian J Dermatol. 2018;63:30–40. doi: 10.4103/ijd.IJD_582_17.  [DOI](https://doi.org/10.4103/ijd.IJD_582_17) | [PMC free article](/articles/PMC5838752/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29527023/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Indian%20J%20Dermatol&title=Drug%20reaction%20with%20eosinophilia%20and%20systemic%20symptoms:%20an%20update%20and%20review%20of%20recent%20literature&author=A%20De&author=M%20Rajagopalan&author=A%20Sarda&author=S%20Das&author=P%20Biswas&volume=63&publication_year=2018&pages=30-40&pmid=29527023&doi=10.4103/ijd.IJD_582_17&)

9. Mockenhaupt M, Messenheimer J, Tennis P, Schlingmann J. Risk of Stevens‐Johnson syndrome and toxic epidermal necrolysis in new users of antiepileptics. Neurology. 2005;64:1134–8. doi: 10.1212/01.WNL.0000156354.20227.F0.  [DOI](https://doi.org/10.1212/01.WNL.0000156354.20227.F0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15824335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurology&title=Risk%20of%20Stevens%E2%80%90Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis%20in%20new%20users%20of%20antiepileptics&author=M%20Mockenhaupt&author=J%20Messenheimer&author=P%20Tennis&author=J%20Schlingmann&volume=64&publication_year=2005&pages=1134-8&pmid=15824335&doi=10.1212/01.WNL.0000156354.20227.F0&)

10. Mullan KA, Anderson A, Illing PT, Kwan P, Purcell AW, Mifsud NA. HLA‐associated antiepileptic drug‐induced cutaneous adverse reactions. HLA. 2019;93:417–35. doi: 10.1111/tan.13530.  [DOI](https://doi.org/10.1111/tan.13530) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30895730/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=HLA&title=HLA%E2%80%90associated%20antiepileptic%20drug%E2%80%90induced%20cutaneous%20adverse%20reactions&author=KA%20Mullan&volume=93&publication_year=2019&pages=417-35&pmid=30895730&doi=10.1111/tan.13530&)

11. Chen Z, Liew D, Kwan P. Effects of a HLA‐B*15:02 screening policy on antiepileptic drug use and severe skin reactions. Neurology. 2014;83:2077–84. doi: 10.1212/wnl.0000000000001034.  [DOI](https://doi.org/10.1212/wnl.0000000000001034) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25355835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurology&title=Effects%20of%20a%20HLA%E2%80%90B*15:02%20screening%20policy%20on%20antiepileptic%20drug%20use%20and%20severe%20skin%20reactions&author=Z%20Chen&author=D%20Liew&author=P%20Kwan&volume=83&publication_year=2014&pages=2077-84&pmid=25355835&doi=10.1212/wnl.0000000000001034&)

12. Shi YW, Min FL, Zhou D, Qin B, Wang J, Hu FY, et al. HLA‐A*24:02 as a common risk factor for antiepileptic drug‐induced cutaneous adverse reactions. Neurology. 2017;88:2183–91. doi: 10.1212/WNL.0000000000004008.  [DOI](https://doi.org/10.1212/WNL.0000000000004008) | [PMC free article](/articles/PMC5467955/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28476759/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurology&title=HLA%E2%80%90A*24:02%20as%20a%20common%20risk%20factor%20for%20antiepileptic%20drug%E2%80%90induced%20cutaneous%20adverse%20reactions&author=YW%20Shi&volume=88&publication_year=2017&pages=2183-91&pmid=28476759&doi=10.1212/WNL.0000000000004008&)

13. McCormack M, Alfirevic A, Bourgeois S, Farrell JJ, Kasperavičiūtė D, Carrington M, et al. HLA‐A*3101 and carbamazepine‐induced hypersensitivity reactions in Europeans. N Engl J Med. 2011;364:1134–43. doi: 10.1056/NEJMoa1013297.  [DOI](https://doi.org/10.1056/NEJMoa1013297) | [PMC free article](/articles/PMC3113609/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21428769/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=HLA%E2%80%90A*3101%20and%20carbamazepine%E2%80%90induced%20hypersensitivity%20reactions%20in%20Europeans&author=M%20McCormack&volume=364&publication_year=2011&pages=1134-43&pmid=21428769&doi=10.1056/NEJMoa1013297&)

14. Cheung YK, Cheng SH, Chan EJ, Lo SV, Ng MH, Kwan P. HLA‐B alleles associated with severe cutaneous reactions to antiepileptic drugs in Han Chinese. Epilepsia. 2013;54:1307–14. doi: 10.1111/epi.12217.  [DOI](https://doi.org/10.1111/epi.12217) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23692434/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Epilepsia&title=HLA%E2%80%90B%20alleles%20associated%20with%20severe%20cutaneous%20reactions%20to%20antiepileptic%20drugs%20in%20Han%20Chinese&author=YK%20Cheung&volume=54&publication_year=2013&pages=1307-14&pmid=23692434&doi=10.1111/epi.12217&)

15. Chen CB, Hsiao YH, Wu T, Hsih MS, Tassaneeyakul W, Jorns TP, et al. Risk and association of HLA with oxcarbazepine‐induced cutaneous adverse reactions in Asians. Neurology. 2017;88:78–86. doi: 10.1212/WNL.0000000000003453.  [DOI](https://doi.org/10.1212/WNL.0000000000003453) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27913699/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neurology&title=Risk%20and%20association%20of%20HLA%20with%20oxcarbazepine%E2%80%90induced%20cutaneous%20adverse%20reactions%20in%20Asians&author=CB%20Chen&volume=88&publication_year=2017&pages=78-86&pmid=27913699&doi=10.1212/WNL.0000000000003453&)

16. Roujeau JC, Stern RS. Severe adverse cutaneous reactions to drugs. N Engl J Med. 1994;331:1272–85. doi: 10.1056/NEJM199411103311906.  [DOI](https://doi.org/10.1056/NEJM199411103311906) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7794310/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Severe%20adverse%20cutaneous%20reactions%20to%20drugs&author=JC%20Roujeau&author=RS%20Stern&volume=331&publication_year=1994&pages=1272-85&pmid=7794310&doi=10.1056/NEJM199411103311906&)

17. Auwera GA, Carneiro MO, Hartl C, Poplin R, del Angel G, Levy‐Moonshine A, et al. From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline. Curr Protoc Bioinformatics. 2013;43:11.10.11–33. doi: 10.1002/0471250953.bi1110s43.  [DOI](https://doi.org/10.1002/0471250953.bi1110s43) | [PMC free article](/articles/PMC4243306/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25431634/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Protoc%20Bioinformatics&title=From%20FastQ%20data%20to%20high%20confidence%20variant%20calls:%20the%20Genome%20Analysis%20Toolkit%20best%20practices%20pipeline&author=GA%20Auwera&author=MO%20Carneiro&author=C%20Hartl&author=R%20Poplin&author=G%20del%20Angel&volume=43&publication_year=2013&pages=11.10.11-33&pmid=25431634&doi=10.1002/0471250953.bi1110s43&)

18. Siva N. 1000 Genomes project. Nat Biotechnol. 2008;26:256. doi: 10.1038/nbt0308-256b.  [DOI](https://doi.org/10.1038/nbt0308-256b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18327223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=1000%20Genomes%20project&author=N%20Siva&volume=26&publication_year=2008&pages=256&pmid=18327223&doi=10.1038/nbt0308-256b&)

19. McLaren W, Gil L, Hunt SE, Riat HS, Ritchie GR, Thormann A, et al. The Ensembl variant effect predictor. Genome Biol. 2016;17:122. doi: 10.1186/s13059-016-0974-4.  [DOI](https://doi.org/10.1186/s13059-016-0974-4) | [PMC free article](/articles/PMC4893825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27268795/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&title=The%20Ensembl%20variant%20effect%20predictor&author=W%20McLaren&volume=17&publication_year=2016&pages=122&pmid=27268795&doi=10.1186/s13059-016-0974-4&)

20. Bernstein BE, Stamatoyannopoulos JA, Costello JF, Ren B, Milosavljevic A, Meissner A, et al. The NIH roadmap epigenomics mapping consortium. Nat Biotechnol. 2010;28:1045–8. doi: 10.1038/nbt1010-1045.  [DOI](https://doi.org/10.1038/nbt1010-1045) | [PMC free article](/articles/PMC3607281/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20944595/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&title=The%20NIH%20roadmap%20epigenomics%20mapping%20consortium&author=BE%20Bernstein&volume=28&publication_year=2010&pages=1045-8&pmid=20944595&doi=10.1038/nbt1010-1045&)

21. Ecker JR, Bickmore WA, Barroso I, Pritchard JK, Gilad Y, Segal E. Genomics: ENCODE explained. Nature. 2012;489:52–5. doi: 10.1038/489052a.  [DOI](https://doi.org/10.1038/489052a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22955614/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Genomics:%20ENCODE%20explained&author=JR%20Ecker&volume=489&publication_year=2012&pages=52-5&pmid=22955614&doi=10.1038/489052a&)

22. Fernandez JM, de la Torre V, Richardson D, Royo R, Puiggròs M, Moncunill V, et al. The BLUEPRINT data analysis portal. Cell Syst. 2016;3:491– 5.e5. doi: 10.1016/j.cels.2016.10.021.  [DOI](https://doi.org/10.1016/j.cels.2016.10.021) | [PMC free article](/articles/PMC5919098/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27863955/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Syst&title=The%20BLUEPRINT%20data%20analysis%20portal&author=JM%20Fernandez&volume=3&publication_year=2016&pages=491%E2%80%93-5.&pmid=27863955&doi=10.1016/j.cels.2016.10.021&)

23. GTEx Consortium. The genotype‐tissue expression (GTEx) project. Nat Genet. 2013;45:580–5.  [DOI](https://doi.org/10.1038/ng.2653) | [PMC free article](/articles/PMC4010069/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23715323/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20genotype%E2%80%90tissue%20expression%20(GTEx)%20project&author=%20GTEx%20Consortium&volume=45&publication_year=2013&pages=580%E2%80%935&pmid=23715323&doi=10.1038/ng.2653&)

24. Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ. Second‐generation PLINK: rising to the challenge of larger and richer datasets. Gigascience. 2015;4:7. doi: 10.1186/s13742-015-0047-8.  [DOI](https://doi.org/10.1186/s13742-015-0047-8) | [PMC free article](/articles/PMC4342193/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25722852/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gigascience&title=Second%E2%80%90generation%20PLINK:%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&author=CC%20Chang&volume=4&publication_year=2015&pages=7&pmid=25722852&doi=10.1186/s13742-015-0047-8&)

25. Villanueva RAM, Chen ZJ. ggplot2: elegant graphics for data analysis. New York, NY: Taylor & Francis; 2019.  [Google Scholar](https://scholar.google.com/scholar_lookup?title=ggplot2:%20elegant%20graphics%20for%20data%20analysis&author=RAM%20Villanueva&author=ZJ%20Chen&publication_year=2019&)

26. Turner SD. qqman: an R package for visualizing GWAS results using QQ and Manhattan plots. bioRxiv. 2014;005165.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=bioRxiv&title=qqman:%20an%20R%20package%20for%20visualizing%20GWAS%20results%20using%20QQ%20and%20Manhattan%20plots&author=SD%20Turner&publication_year=2014&)

27. Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, et al. LocusZoom: regional visualization of genome‐wide association scan results. Bioinformatics. 2010;26:2336–7.  [DOI](https://doi.org/10.1093/bioinformatics/btq419) | [PMC free article](/articles/PMC2935401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20634204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LocusZoom:%20regional%20visualization%20of%20genome%E2%80%90wide%20association%20scan%20results&author=RJ%20Pruim&volume=26&publication_year=2010&pages=2336-7&pmid=20634204&doi=10.1093/bioinformatics/btq419&)

28. Ozeki T, Mushiroda T, Yowang A, Takahashi A, Kubo M, Shirakata Y, et al. Genome‐wide association study identifies HLA‐A*3101 allele as a genetic risk factor for carbamazepine‐induced cutaneous adverse drug reactions in Japanese population. Hum Mol Genet. 2011;20:1034–41. doi: 10.1093/hmg/ddq537.  [DOI](https://doi.org/10.1093/hmg/ddq537) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21149285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Mol%20Genet&title=Genome%E2%80%90wide%20association%20study%20identifies%20HLA%E2%80%90A*3101%20allele%20as%20a%20genetic%20risk%20factor%20for%20carbamazepine%E2%80%90induced%20cutaneous%20adverse%20drug%20reactions%20in%20Japanese%20population&author=T%20Ozeki&volume=20&publication_year=2011&pages=1034-41&pmid=21149285&doi=10.1093/hmg/ddq537&)

29. Price AL, Zaitlen NA, Reich D, Patterson N. New approaches to population stratification in genome‐wide association studies. Nat Rev Genet. 2010;11:459–63. doi: 10.1038/nrg2813.  [DOI](https://doi.org/10.1038/nrg2813) | [PMC free article](/articles/PMC2975875/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20548291/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&title=New%20approaches%20to%20population%20stratification%20in%20genome%E2%80%90wide%20association%20studies&author=AL%20Price&author=NA%20Zaitlen&author=D%20Reich&author=N%20Patterson&volume=11&publication_year=2010&pages=459-63&pmid=20548291&doi=10.1038/nrg2813&)

30. Smemo S, Tena JJ, Kim KH, Gamazon ER, Sakabe NJ, Gómez‐Marín C, et al. Obesity‐associated variants within FTO form long‐range functional connections with IRX3. Nature. 2014;507:371–5. doi: 10.1038/nature13138.  [DOI](https://doi.org/10.1038/nature13138) | [PMC free article](/articles/PMC4113484/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24646999/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Obesity%E2%80%90associated%20variants%20within%20FTO%20form%20long%E2%80%90range%20functional%20connections%20with%20IRX3&author=S%20Smemo&volume=507&publication_year=2014&pages=371-5&pmid=24646999&doi=10.1038/nature13138&)

31. Schork AJ, Thompson WK, Pham P, Torkamani A, Roddey JC, Sullivan PF, et al. All SNPs are not created equal: genome‐wide association studies reveal a consistent pattern of enrichment among functionally annotated SNPs. PLoS Genet. 2013;9:e1003449. doi: 10.1371/journal.pgen.1003449.  [DOI](https://doi.org/10.1371/journal.pgen.1003449) | [PMC free article](/articles/PMC3636284/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23637621/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet&title=All%20SNPs%20are%20not%20created%20equal:%20genome%E2%80%90wide%20association%20studies%20reveal%20a%20consistent%20pattern%20of%20enrichment%20among%20functionally%20annotated%20SNPs&author=AJ%20Schork&volume=9&publication_year=2013&pages=e1003449&pmid=23637621&doi=10.1371/journal.pgen.1003449&)

32. Zhang C, Zhu Z, Gao J, Yang L, Dang E, Fang H, et al. Plasma exosomal miR‐375‐3p regulates mitochondria‐dependent keratinocyte apoptosis by targeting XIAP in severe drug‐induced skin reactions. Sci Transl Med. 2020;12:eaaw6142. doi: 10.1126/scitranslmed.aaw6142.  [DOI](https://doi.org/10.1126/scitranslmed.aaw6142) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33328332/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&title=Plasma%20exosomal%20miR%E2%80%90375%E2%80%903p%20regulates%20mitochondria%E2%80%90dependent%20keratinocyte%20apoptosis%20by%20targeting%20XIAP%20in%20severe%20drug%E2%80%90induced%20skin%20reactions&author=C%20Zhang&volume=12&publication_year=2020&pmid=33328332&doi=10.1126/scitranslmed.aaw6142&)

33. Alcaraz‐García MJ, Ruiz‐Lafuente N, Sebastián‐Ruiz S, Majado MJ, González‐García C, Bernardo MV, et al. Human and mouse DOCK10 splicing isoforms with alternative first coding exon usage are differentially expressed in T and B lymphocytes. Hum Immunol. 2011;72:531–7. doi: 10.1016/j.humimm.2011.03.024.  [DOI](https://doi.org/10.1016/j.humimm.2011.03.024) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21514340/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hum%20Immunol&title=Human%20and%20mouse%20DOCK10%20splicing%20isoforms%20with%20alternative%20first%20coding%20exon%20usage%20are%20differentially%20expressed%20in%20T%20and%20B%20lymphocytes&author=MJ%20Alcaraz%E2%80%90Garc%C3%ADa&volume=72&publication_year=2011&pages=531-7&pmid=21514340&doi=10.1016/j.humimm.2011.03.024&)

34. Chen Y, Chen Y, Yin W, Han H, Miller H, Li J, et al. The regulation of DOCK family proteins on T and B cells. J Leukoc Biol. 2021;109:383–94. doi: 10.1002/JLB.1MR0520-221RR.  [DOI](https://doi.org/10.1002/JLB.1MR0520-221RR) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32542827/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Leukoc%20Biol&title=The%20regulation%20of%20DOCK%20family%20proteins%20on%20T%20and%20B%20cells&author=Y%20Chen&volume=109&publication_year=2021&pages=383-94&pmid=32542827&doi=10.1002/JLB.1MR0520-221RR&)

35. Ott H, Wiederholt T, Bergström MA, Heise R, Skazik C, Czaja K, et al. High‐resolution transcriptional profiling of chemical‐stimulated dendritic cells identifies immunogenic contact allergens, but not prohaptens. Skin Pharmacol Physiol. 2010;23:213–24. doi: 10.1159/000313897.  [DOI](https://doi.org/10.1159/000313897) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20431333/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Skin%20Pharmacol%20Physiol&title=High%E2%80%90resolution%20transcriptional%20profiling%20of%20chemical%E2%80%90stimulated%20dendritic%20cells%20identifies%20immunogenic%20contact%20allergens,%20but%20not%20prohaptens&author=H%20Ott&volume=23&publication_year=2010&pages=213-24&pmid=20431333&doi=10.1159/000313897&)

36. Iwaszko M, Bogunia‐Kubik K. Clinical significance of the HLA‐E and CD94/NKG2 interaction. Arch Immunol Ther Exp (Warsz). 2011;59:353–67. doi: 10.1007/s00005-011-0137-y.  [DOI](https://doi.org/10.1007/s00005-011-0137-y) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21800130/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Arch%20Immunol%20Ther%20Exp%20(Warsz)&title=Clinical%20significance%20of%20the%20HLA%E2%80%90E%20and%20CD94/NKG2%20interaction&author=M%20Iwaszko&author=K%20Bogunia%E2%80%90Kubik&volume=59&publication_year=2011&pages=353-67&pmid=21800130&doi=10.1007/s00005-011-0137-y&)

37. Lampen MH, Hassan C, Sluijter M, Geluk A, Dijkman K, Tjon JM, et al. Alternative peptide repertoire of HLA‐E reveals a binding motif that is strikingly similar to HLA‐A2. Mol Immunol. 2013;53:126–31.  [DOI](https://doi.org/10.1016/j.molimm.2012.07.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22898188/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Immunol&title=Alternative%20peptide%20repertoire%20of%20HLA%E2%80%90E%20reveals%20a%20binding%20motif%20that%20is%20strikingly%20similar%20to%20HLA%E2%80%90A2&author=MH%20Lampen&volume=53&publication_year=2013&pages=126-31&pmid=22898188&doi=10.1016/j.molimm.2012.07.009&)

38. Morel E, Escamochero S, Cabañas R, Díaz R, Fiandor A, Bellón T. CD94/NKG2C is a killer effector molecule in patients with Stevens‐Johnson syndrome and toxic epidermal necrolysis. J Allergy Clin Immunol. 2010;125:703–10, 710.e1–8.  [DOI](https://doi.org/10.1016/j.jaci.2009.10.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20132973/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Allergy%20Clin%20Immunol&title=CD94/NKG2C%20is%20a%20killer%20effector%20molecule%20in%20patients%20with%20Stevens%E2%80%90Johnson%20syndrome%20and%20toxic%20epidermal%20necrolysis&author=E%20Morel&volume=125&publication_year=2010&pages=703-10,%20710.e1%E2%80%938&pmid=20132973&doi=10.1016/j.jaci.2009.10.030&)

39. Benezeder T, Painsi C, Patra V, Dey S, Holcmann M, Lange‐Asschenfeldt B, et al. Dithranol targets keratinocytes, their crosstalk with neutrophils and inhibits the IL‐36 inflammatory loop in psoriasis. eLife. 2020;9:e56991. doi: 10.7554/eLife.56991.  [DOI](https://doi.org/10.7554/eLife.56991) | [PMC free article](/articles/PMC7266641/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32484435/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=eLife&title=Dithranol%20targets%20keratinocytes,%20their%20crosstalk%20with%20neutrophils%20and%20inhibits%20the%20IL%E2%80%9036%20inflammatory%20loop%20in%20psoriasis&author=T%20Benezeder&volume=9&publication_year=2020&pages=e56991&pmid=32484435&doi=10.7554/eLife.56991&)

40. Altevogt P, Sammar M, Hüser L, Kristiansen G. Novel insights into the function of CD24: a driving force in cancer. Int J Cancer. 2020;148:546–59. doi: 10.1002/ijc.33249.  [DOI](https://doi.org/10.1002/ijc.33249) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32790899/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Novel%20insights%20into%20the%20function%20of%20CD24:%20a%20driving%20force%20in%20cancer&author=P%20Altevogt&author=M%20Sammar&author=L%20H%C3%BCser&author=G%20Kristiansen&volume=148&publication_year=2020&pages=546-59&pmid=32790899&doi=10.1002/ijc.33249&)

41. Liu Z, Dang E, Li B, Qiao H, Jin L, Zhang J, et al. Dysfunction of CD19+CD24hiCD27+ B regulatory cells in patients with bullous pemphigoid. Sci Rep. 2018;8:703. doi: 10.1038/s41598-018-19226-z.  [DOI](https://doi.org/10.1038/s41598-018-19226-z) | [PMC free article](/articles/PMC5768798/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29335495/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&title=Dysfunction%20of%20CD19+CD24hiCD27+%20B%20regulatory%20cells%20in%20patients%20with%20bullous%20pemphigoid&author=Z%20Liu&volume=8&publication_year=2018&pages=703&pmid=29335495&doi=10.1038/s41598-018-19226-z&)

42. Zheng C, Yin S, Yang Y, Yu Y, Xie X. CD24 aggravates acute liver injury in autoimmune hepatitis by promoting IFN‐gamma production by CD4(+) T cells. Cell Mol Immunol. 2018;15:260–71. doi: 10.1038/cmi.2016.57.  [DOI](https://doi.org/10.1038/cmi.2016.57) | [PMC free article](/articles/PMC5843612/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28065940/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Mol%20Immunol&title=CD24%20aggravates%20acute%20liver%20injury%20in%20autoimmune%20hepatitis%20by%20promoting%20IFN%E2%80%90gamma%20production%20by%20CD4(+)%20T%20cells&author=C%20Zheng&author=S%20Yin&author=Y%20Yang&author=Y%20Yu&author=X%20Xie&volume=15&publication_year=2018&pages=260-71&pmid=28065940&doi=10.1038/cmi.2016.57&)

43. Kim‐Hellmuth S, Bechheim M, Pütz B, Mohammadi P, Nédélec Y, Giangreco N, et al. Genetic regulatory effects modified by immune activation contribute to autoimmune disease associations. Nat Commun. 2017;8:266. doi: 10.1038/s41467-017-00366-1.  [DOI](https://doi.org/10.1038/s41467-017-00366-1) | [PMC free article](/articles/PMC5559603/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28814792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Genetic%20regulatory%20effects%20modified%20by%20immune%20activation%20contribute%20to%20autoimmune%20disease%20associations&author=S%20Kim%E2%80%90Hellmuth&volume=8&publication_year=2017&pmid=28814792&doi=10.1038/s41467-017-00366-1&)

44. Yang J, Weedon MN, Purcell S, Lettre G, Estrada K, Willer CJ, et al. Genomic inflation factors under polygenic inheritance. Eur J Hum Genet. 2011;19:807–12. doi: 10.1038/ejhg.2011.39.  [DOI](https://doi.org/10.1038/ejhg.2011.39) | [PMC free article](/articles/PMC3137506/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21407268/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Hum%20Genet&title=Genomic%20inflation%20factors%20under%20polygenic%20inheritance&author=J%20Yang&volume=19&publication_year=2011&pages=807-12&pmid=21407268&doi=10.1038/ejhg.2011.39&)
