# A two-stage genome-wide association study identifies novel germline genetic variations in CACNA2D3 associated with radiotherapy response in nasopharyngeal carcinoma

## Metadata
**Authors:** Lu-Lu Yu, Bi-Wen Hu, Han-Xue Huang, Bing Yu, Qi Xiao, Qiao-Li Lv, Chen-Hui Luo, Cheng-Xian Guo, Jin-Gao Li, Xiao-Xue Xie, Ji-Ye Yin
**Journal:** Journal of Translational Medicine
**Date:** 2023 Jan 9
**DOI:** [10.1186/s12967-022-03819-4](https://doi.org/10.1186/s12967-022-03819-4)
**PMID:** 36624463
**PMCID:** PMC9830790
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9830790/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9830790/pdf/12967_2022_Article_3819.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9830790/pdf/12967_2022_Article_3819.pdf)

## Abstract

**Background:** 
Radiotherapy (RT) is the standard treatment for nasopharyngeal carcinoma (NPC). However, due to individual differences in radiosensitivity, biomarkers are needed to tailored radiotherapy to cancer patients. However, comprehensive genome-wide radiogenomic studies on them are still lacking. The aim of this study was to identify genetic variants associated with radiotherapy response in patients with NPC.

**Methods:** 
This was a large‑scale genome-wide association analysis (GWAS) including a total of 981 patients. 319 individuals in the discovery stage were genotyped for 688,783 SNPs using whole genome-wide screening microarray. Significant loci were further genotyped using MassARRAY system and TaqMan SNP assays in the validation stages of 847 patients. This study used logistic regression analysis and multiple bioinformatics tools such as PLINK, LocusZoom, LDBlockShow, GTEx, Pancan-meQTL and FUMA to examine genetic variants associated with radiotherapy efficacy in NPC.

**Results:** 
After genome-wide level analysis, 19 SNPs entered the validation stage (P < 1 × 10− 6), and rs11130424 ultimately showed statistical significance among these SNPs. The efficacy was better in minor allele carriers of rs11130424 than in major allele carriers. Further stratified analysis showed that the association existed in patients in the EBV-positive, smoking, and late-stage (III and IV) subgroups and in patients who underwent both concurrent chemoradiotherapy and induction/adjuvant chemotherapy.

**Conclusion:** 
Our study showed that rs11130424 in the CACNA2D3 gene was associated with sensitivity to radiotherapy in NPC patients.

Trial registration number: Effect of genetic polymorphism on nasopharyngeal carcinoma chemoradiotherapy reaction, ChiCTR-OPC-14005257, Registered 18 September 2014, http://www.chictr.org.cn/showproj.aspx?proj=9546.

**Supplementary Information:** 
The online version contains supplementary material available at 10.1186/s12967-022-03819-4.

Keywords: Genome-wide association study, Nasopharyngeal carcinoma, Radiotherapy, Efficacy, CACNA2D3

### Background

Radiotherapy (RT) is the standard treatment for nasopharyngeal carcinoma (NPC). However, due to individual differences in radiosensitivity, biomarkers are needed to tailored radiotherapy to cancer patients. However, comprehensive genome-wide radiogenomic studies on them are still lacking. The aim of this study was to identify genetic variants associated with radiotherapy response in patients with NPC.

### Methods

This was a large‑scale genome-wide association analysis (GWAS) including a total of 981 patients. 319 individuals in the discovery stage were genotyped for 688,783 SNPs using whole genome-wide screening microarray. Significant loci were further genotyped using MassARRAY system and TaqMan SNP assays in the validation stages of 847 patients. This study used logistic regression analysis and multiple bioinformatics tools such as PLINK, LocusZoom, LDBlockShow, GTEx, Pancan-meQTL and FUMA to examine genetic variants associated with radiotherapy efficacy in NPC.

### Results

After genome-wide level analysis, 19 SNPs entered the validation stage (P < 1 × 10^− 6^), and rs11130424 ultimately showed statistical significance among these SNPs. The efficacy was better in minor allele carriers of rs11130424 than in major allele carriers. Further stratified analysis showed that the association existed in patients in the EBV-positive, smoking, and late-stage (III and IV) subgroups and in patients who underwent both concurrent chemoradiotherapy and induction/adjuvant chemotherapy.

### Conclusion

Our study showed that rs11130424 in the CACNA2D3 gene was associated with sensitivity to radiotherapy in NPC patients.

*Trial registration number:* Effect of genetic polymorphism on nasopharyngeal carcinoma chemoradiotherapy reaction, ChiCTR-OPC-14005257, Registered 18 September 2014, [http://www.chictr.org.cn/showproj.aspx?proj=9546](http://www.chictr.org.cn/showproj.aspx?proj=9546).

### Supplementary Information

The online version contains supplementary material available at 10.1186/s12967-022-03819-4.

## Introduction

Nasopharyngeal carcinoma is a malignant tumor arising from the endothelium of the nasopharynx [[1](#CR1)] Its incidence is highly geographically specific, with a high prevalence in countries of China, India and Vietnam. China accounted for approximately half of all new cases of NPC worldwide in 2018 [[2](#CR2)].

The main treatment for NPC is radiotherapy [[3](#CR3)] The current radiation treatment strategy for NPC is to use simple radiotherapy for early-stage NPC and concurrent chemoradiotherapy for intermediate and advanced stages [[4](#CR4)] Although radiotherapy is the major treatment for NPC, there is some individual difference in the outcome of patients receiving radiotherapy, and the exact mechanism has not yet been fully understood. There is currently a problem of over- or under-treatment in clinical care as there is no good way to accurately identify patients who may benefit from radiotherapy prior to treatment. In the treatment of NPC, people seek to improve the efficacy of radiotherapy and increase patient survival.

With regard to the development of biomarkers related to the efficacy of radiotherapy, the concept of radiogenomics has been proposed [[5](#CR5)] Radiogenomics is based on the theory that the efficacy of radiotherapy is not only related to the treatment scheme, radiation dose and irradiation location, but also to the genetic characteristics of the patients. Candidate gene association and genome-wide association analysis are the main approaches in radiogenomics studies. Most current studies about biomarkers of tumor radiotherapy efficacy are based on candidate gene association studies. For example, X-ray repair cross complementing 1 (XRCC1) 399Gln was found to be significantly associated with radiotherapy resistance in patients with oesophageal carcinoma [[6](#CR6)]; Epidermal growth factor receptor (EGFR) R497K can be used as a predictor of short-term efficacy and late recurrence or metastasis of radiotherapy in cervical cancer [[7](#CR7)] In addition, rs3135001 is a useful biomarker for predicting for the sensitivity of head and neck squamous cell carcinoma to radiotherapy [[8](#CR8)] However, there are few GWAS on NPC radiotherapy sensitivity, but it is equally important to explore the genetic variants associated with NPC radiotherapy sensitivity.

In this study, we conducted a two-stage GWAS study in 1166 NPC patients to comprehensively examine the entire genome at a higher resolution than candidate gene association studies. We selected multiple NPC susceptibility-associated loci and validated these findings in two additional cohort of samples. Our findings clearly indicate that the CACNA2D3 gene plays a role in the mechanism of radiotherapy sensitivity in NPC.

## Method

### Study population and data collection

All patients in this study were recruited from Jiangxi Provincial Cancer Hospital and Hunan Provincial Cancer Hospital from 2014 to 2022 respectively, who were diagnosed with NPC by pathological histological examination, while having no distant metastases and no other concurrent tumors. They are mainly treated by Intensity Modulated Radiation Therapy (IMRT), which allows for more precise application of different doses of radiation to different target areas. The radiation dose depends on the total tumor volume (GTV), the clinical target volume (CTV) and the planned target volume (PTV). A single dose of radiation therapy is usually about 2 Gy, once a day, 5 days a week. Primary lesions and positive cervical lymph nodes received 66–77 Gy and 54–60 Gy, respectively, and patients were treated in approximately 30–33 sessions. The patients have not received surgery, targeted therapy, immunotherapy or other anti-tumor treatment prior to radiotherapy. Cisplatin was used as the base regimen for adjuvant chemotherapy, neoadjuvant chemotherapy or concurrent radiotherapy. Exclusion criteria are: (1) breastfeeding pregnant patients; (2) having serious complications.

Clinicians used the WHO Response Evaluation Criteria in Solid Tumors (RECIST) to assess the efficacy of each patient after radiotherapy based on the patient’s MRI medical records. In this study, post-radiotherapy efficacy was assessed for the primary lesion and positive lymph nodes. The results of radiotherapy efficacy assessment were classified into four categories: complete remission (CR), partial remission (PR), stable disease (SD) and progressive disease (PD). For association analysis, all patients were further divided into two groups: the group with good outcome as CR, and the group with poor outcome as non-CR (including PR, SD and PD).

### Ethics statement

The study protocol was approved by the Ethics Committee of Jiangxi Cancer Hospital and Hunan Cancer Hospital. Clinical information of all patients from both hospitals were collected with the same protocol and were not any overlap of patients. The ethical aspects of this study are in line with the Declaration of Helsinki. All subjects were provided with written informed consents. We applied this study for clinical admission in the Chinese Clinical Trial Register (registration number: ChiCTR-OPC-14005257).

### Genotype detection, quality control and imputation

For discovery stage samples, genotyping was performed using the Illumina Infinium Global Screening Array-24 v1.0 (GSA) microarray, and the data generated by the Illumina microarray platform was visualized and analyzed according to GenomeStudio software. For quality control, the raw data were first selected for sample and SNP selection using PLINK (version 1.9) software. SNP quality control (QC) conditions: (1) SNP call rate ≥ 95%; (2) HWE ≥ 1 × 10^− 4^; (3) MAF ≥ 0.01; (4) localization on autosomes. Sample QC conditions: (1) Sample call rate ≥ 95%; (2) Correcting for Cryptic relatedness. Cryptic relatedness refers to unknown more recent relatedness including family relationships such as grandparent-grandchild and full sibling pairs; (3) The heterozygosity rate was less than three standard deviations from the mean of all participants; (4) Genotype-estimated sex was discordant with biological sex. Using 1000 Genomes stage 3 v4 as the reference panel, IMPUTE2 was used for genotype imputation. SNPs with INFO below 0.4 and MAF<0.05 were excluded. Finally, 4,112,760 unique SNPs were included in the GWAS analysis. For samples in the validation stage 1, genotyping was performed using the Sequenom MassARRAY system, and genotyping data were obtained using TYPER 4.0 software. Exclusion criteria at this stage included (1) Sample call rate ≥ 95%; (2) SNP detection rate ≥ 95%; (3) MAF ≥ 0.01; (4) HWE ≥ 0.05; (5) Sample relatedness check; (6) The heterozygosity rate was less than three standard deviations from the mean of all participants; and (7) Genotype-estimated sex was discordant with biological sex. For samples in the validation stage 2, genotyping was conducted using the TaqMan SNP assays (Applied Biosystems, Foster City, CA) with a Roche LightCycler 480 Real-Time PCR System (Roche, Basle, Switzerland) for rs11130424. The amplification reaction was carried out in 10 µL of the PCR mixture containing 5 µL of LightCycler® 480 Probes Master (Roche, Switzerland), 0.5 µM of forward primer rs11130424-F: 5′- ATCATGGACGGTCCTTGCT-3′ and 0.5 µM of the reverse primer rs11130424-R: 5′- GGACATGTCAATTTAAAGTTGCTAT − 3′, 0.4 µM of each allele-specific probes (rs11130424_G: 5′-FAM- AAACTCTGTTACTCATAGAA -MGB-3′ and rs11130424_T:5′-VIC- AACTCTGTTATTCATAGAAT -MGB-3′) and 10 ng of DNA. PCR was carried out using a LightCycler® 96 System (Roche, Switzerland) under optimized conditions (preliminary denaturation – 94 ℃–3 min; 40 cycles: 94 ℃–5 s, 60 ℃–30 s). Fluorescence measurements were carried out at the annealing step of PCR in FAM and VIC channels. The results of genotyping were analyzed using the LightCycler® 96 thermal cycler software.

### Genome-wide association analysis

Principal component analysis (PCA) was conducted on all patients in the discovery stage using the SNPRelate procedure. The distribution of the samples was plotted using the R package “ggplot2”. Samples that deviated significantly from the overall were removed and then reconducted for PCA. Clinical factors that have a significantly effect on the phenotype were considered as covariates. T-tests were used for continuous variables and logistic tests for categorical variables, and clinical factors that differed markedly in subgroups were selected as covariates for correction. The significantly associated clinical factors in multivariate regression were adjusted in genome-wide association analyses. For two phenotypes of primary lesion and positive lymph node efficacy, genome-wide association analyses were performed under additive genetic effects assumption, using a logistic regression model adjusting sex, smoking and drinking as covariates. Quantitative-Quantitative plots and Manhattan plots were plotted via the R package “qqman”. A genomic inflation factor λ was calculated for population stratification using PLINK (version 1.9). The deviation of the observed versus the expected distribution of the P values was represented by the inflation factor λ. The closer λ was to 1, the less stratified the population was considered to be.

### Bioinformatics tools

Regional plot was generated using LocusZoom ([http://locuszoom.sph.umich.edu/](http://locuszoom.sph.umich.edu/)). The LD and haplotype block estimations analyses were performed using PLINK and LDBlockShow (1.35 version) [[9](#CR9)]. Expression quantitative trait loci (eQTL) and DNA methylation quantitative trait loci (meQTL) were analyzed using Genotype-Tissue Expression (GTEx) project ([https://gtexportal.org/](https://gtexportal.org/)) and Pancan-meQTL database ([http://bioinfo.life.hust.edu.cn/Pancan-meQTL/](http://bioinfo.life.hust.edu.cn/Pancan-meQTL/)) respectively [[10](#CR10)]. Functional annotation of GWAS results and chromatin interaction visualization were generated using the online tool functional mapping and annotation (FUMA) v1.3.4c ([https://fuma.ctglab.nl/](https://fuma.ctglab.nl/)) [[11](#CR11), [12](#CR12)]. Transcription regulation was predicted based on the data of ENCODE database ([https://www.encodeproject.org/](https://www.encodeproject.org/)) [[13](#CR13)]. 1000Genomes project ([https://www.ncbi.nlm.nih.gov/variation/tools/1000genomes/](https://www.ncbi.nlm.nih.gov/variation/tools/1000genomes/)) and the gnomAD database ([http://gnomad-sg.org/](http://gnomad-sg.org/)) were used to query the racial frequencies of loci in each ethnicity, respectively.

### Statistical analysis

Pearson’s chi-square test or Fisher’s exact test was used to assess the relationship between clinical characteristics and radiotherapy response. The influences of genetic variants on the radiotherapy sensitivity were examined using multivariable logistic regression by calculating odds ratios (ORs) and their 95% confidence intervals (CIs). In this study, all P values were two-sided, and P < 0.05 was considered statistically significant. All these analyses were performed by PLINK (1.9 version), SPSS (20.0 version), and R v0.4.0.3 ([www.r-project.org](http://www.r-project.org)).

## Result

### Clinical characteristics

As shown in Fig. [1](#Fig1), this was a two-stage GWAS study. The study population initially consisted of 1645 patients with NPC and 479 were excluded due to failure of sample quality control. Ultimately, 1166 patients were included in the two-stage analysis. Of these, 319 patients from the Jiangxi Provincial Cancer Hospital were recruited into the discovery phase cohort. 662 patients from Hunan Cancer Hospital and 185 patients from Jiangxi Cancer Hospital were used to validate the study results. Detailed clinical characteristics are shown in Table [1](#Tab1). Here we analyzed 2 indicators of primary lesions and positive lymph node efficacy after radiotherapy. The characteristics of NPC patients involved in these association analyses are summarized in Additional file [1](#MOESM1): Tables S1 and S2. For each phenotype, the association of clinical characteristics of age, gender, BMI, smoking status, EBV infection, clinical stage, and treatment scheme between different subgroups was analyzed. All significant clinical factors were considered as covariates and adjusted in subsequent GWAS analysis.

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3b50/9830790/8d9696aa25c7/12967_2022_3819_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9830790_12967_2022_3819_Fig1_HTML.jpg)

Flow diagram of genome-wide association analysis study design

### Table 1.

| Characteristics | Discovery stage | Validation stage 1 | Validation stage 2 |
| --- | --- | --- | --- |
| Number of patients | 319 | 662 | 185 |
| Age (Mean ± SD) | 50.75 ± 11.96 | 47.45 ± 10.49 | 51.26 ± 12.13 |
| Gender |   |   |   |
| Male | 215 | 488 | 131 |
| Female | 104 | 173 | 54 |
| BMI (Mean ± SD) | 22.61 ± 3.40 | 23.18 ± 3.26 | 23.53 ± 3.27 |
| Smoking status |   |   |   |
| Smoker | 151 | 320 | 74 |
| Non–smoker | 168 | 334 | 17 |
| NR | 0 | 7 | 94 |
| Drinking status |   |   |   |
| Drinker | 72 | 227 | 35 |
| Non–drinker | 247 | 427 | 21 |
| NR | 0 | 7 | 129 |
| Clinical stage |   |   |   |
| I | 7 | 41 | 1 |
| II | 26 | 55 | 12 |
| III | 120 | 302 | 93 |
| IV | 165 | 258 | 79 |
| NR | 1 | 5 | 0 |
| EBV |   |   |   |
| Positive | 216 | 436 | 90 |
| Negative | 103 | 219 | 95 |
| NR | 0 | 6 | 0 |
| Treatment |   |   |   |
| RT alone | 60 | 12 | 20 |
| RT + IC/AC | 4 | 39 | 19 |
| CCRT | 100 | 23 | 23 |
| CCRT + IC/AC | 154 | 551 | 123 |
| RT + other treatment* | 1 | 36 | 0 |

Table 1 Caption: Characteristics of all patients

### Data pre-processing

In the discovery stage, peripheral blood DNA samples from 319 patients were subjected to genome-wide microarray analysis and 688,783 autosomal SNPs were included in the association analysis; in the validation stage, peripheral blood DNA samples from 847 patients were subjected to mass array microarray and TaqMan SNP assays analysis of candidate loci. For each sensitivity phenotype, associations between clinical characteristics such as age, sex, Body Mass Index (BMI), smoking status, EBV infection, clinical stage and treatment regimen were analyzed between different subgroups. All clinical factors that would have an impact were considered as covariates and were adjusted for in subsequent association analyses.

### GWAS analysis

Q-Q plots showed that the expected P-value distributions for all phenotypes were only slightly deviated in the tail region (Additional file [1](#MOESM1): Fig. S1). The genomic inflation factor λ for the efficacy of primary lesions and positive lymph nodes after radiotherapy were 1.013 and 1.011 respectively, which were close to 1. In addition, the PCA results showed that there were no deviations from the sample for each efficacy (Additional file [1](#MOESM1): Fig. S2). These results suggest that the patient population is ethnically homogeneous and that the observed differences in association analysis are not caused by the internal structure of the population. The Manhattan plot in (Fig. [2](#Fig2)a, b) shows the results for both efficacy results of the discovery stage. We further selected significant variants with P < 1 × 10^− 6^ for genotyping in the validation stage samples. A total of 19 loci entered the validation stage. For the SNP locus rs6414584 screened in the discovery stage, the mass array genotyping was not performed again in the validation stage as it did not pass the conditions of quality control, and the P value and OR value in the validation stage were not given in the table. 18 loci were finally subjected to association analysis in the validation stage. For the efficacy of primary lesions, no SNPs were successfully replicated as being associated with the specific efficacy in the validation cohort (Additional file [1](#MOESM1): Table S2). For the efficacy of positive lymph nodes, rs11130424 was successfully validated by showing statistical significance (Additional file [1](#MOESM1): Table S3). We further confirmed the effect of rs11130424 on the sensitivity of radiotherapy for NPC with another validation cohort, and statistically significant difference still existed for rs11130424. After combination of the two validation stages, rs11130424 showed a highly statistically significant difference (Table [2](#Tab2)). We suggest that rs11130424 may have a correlation with the efficacy of positive lymph nodes after radiotherapy. Thus, it was conducted further analysis.

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3b50/9830790/24ab10a2c063/12967_2022_3819_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9830790_12967_2022_3819_Fig2_HTML.jpg)

Genome-wide association study results in discovery cohort. Manhattan plot showed the genome-wide association results for primary lesion (A) and positive lymph node efficacy (B). Association P values are expressed as −log10(p). P values were from multivariate logistic regression analysis

### Table 2.

| Characteristics | rs11130424 |  |
| --- | --- | --- |
| P value | OR (95%CI) |  |
| Discovery stage | 8.71×10-6 | 2.53 (1.68-3.81) |
| Validation stage 1 | 0.046 | 1.26 (1.00-1.59) |
| Validation stage 2 | 1.33×10-4 | 2.33 (1.51-3.62) |
| Combined stage | 1.38×10-5 | 1.70 (1.34-2.15) |

Table 2 Caption: The final validated SNP associated with positive lymph node efficacy in nasopharyngeal carcinoma patients after radiotherapy on CACNA2D3 gene

### Stratification analysis

As previously reported, clinical factors may also influence the sensitivity of radiotherapy. Therefore, we conducted a stratified analysis of rs11130424. Patients were grouped differently according to smoking status, EBV concentration, clinicopathological grade and treatment regimen. Correlations between the rs11130424 locus and these clinical factors were calculated. Clinical characteristics showed some significant influence on the genetic correlation of rs11130424. As shown in Table [3](#Tab3), for smoking status, EBV infection, clinical stage and treatment scheme, the association existed in subgroups of EBV positive, smoking, late stage (III and IV) and patients received both concurrent chemoradiotherapy and induction/adjuvant chemotherapy. This result suggests that clinical factors may influence the results of radiotherapy sensitivity analysis. For example, more than half of patients who were homozygous mutant with late stage (III and IV) were effectively managed after receiving radiotherapy, and the same trend was observed in the subgroup of patients who smoked and received both chemoradiotherapy and induction/adjuvant chemotherapy (Fig. [3](#Fig3)).

### Table 3.

| Characteristics | rs11130424 |  |
| --- | --- | --- |
| P value | OR (95% CI) |  |
| Smoking status |   |   |
| Nonsmoker | 0.107 | 1.234 (0.955–1.595) |
| Smoker | 0.0103 | 1.018 (0.788–1.316) |
| EBV |   |   |
| Positive | 3.175 × 10− 4 | 2.107 (1.401–3.170) |
| Negative | 0.987 | 0.998 (0.822–1.213) |
| Clinical stage |   |   |
| I + II | 0.228 | 1.368 (0.945–2.975) |
| III + IV | 0.0126 | 1.837 (1.046–3.223) |
| Treatment scheme |   |   |
| CCRT | 0.551 | 1.064 (0.868–1.305) |
| CCRT + IC/AC | 3.403 × 10− 3 | 2.214 (1.296–3.783) |

Table 3 Caption: Stratified analysis of the association between the efficacy of positive lymph nodes and rs11130424

### Fig. 3.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3b50/9830790/81b5e6275fec/12967_2022_3819_Fig3_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9830790_12967_2022_3819_Fig3_HTML.jpg)

Patients distribution in different rs11130424 genotypes according to the radiotherapy response in total and stratified subgroups

Based on these results, we concluded that rs11130424 correlates with radiotherapy response in patients of EBV positive, smoking, late stage (III and IV) and receiving both concurrent chemoradiotherapy and induction/adjuvant chemotherapy.

### Gene mapping associated with radiotherapy efficacy

To learn the functional consequence of rs11130424, we performed a fine-mapping and functional annotation. As indicated in Fig. [4](#Fig4)a, rs11130424 were located in the chromosome 3p21.1. We first performed a linkage disequilibrium analysis using LDBlockShow. Analysis of a block spanning 49.41 kb showed that rs11130424 was not in linkage with any proximal SNPs (Fig. [4](#Fig4)b). We further analyzed the eQTL and meQTL profiles of this locus in GTEX and Pancan-meQTL databases. Although rs11130424 was located in the intron region of calcium voltage-gated channel auxiliary subunit alpha2delta 3(CACNA2D3), eQTL analysis did not show that it had an effect with the expression of CACNA2D3 (Additional file [1](#MOESM1): Fig. S3). Moreover, the methylation level of the locus differed significantly among genotypes in endometrial carcinoma (P = 4.87 × 10^− 6^, Fig. [4](#Fig4)c). Whether the corresponding methylation site affects the expression of gene needs to be further explored. With the development of three-dimensional genomics, chromatin loops can spatially bring enhancers and promoters closer together, allowing enhancers to bind to promoters and play a role in regulating genes. 3D chromatin interaction indicated that they interacted with coiled-coil domain containing 66 (CCDC66) (Fig. [4](#Fig4)d).

### Fig. 4.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3b50/9830790/7cd8254bcdbd/12967_2022_3819_Fig4_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9830790_12967_2022_3819_Fig4_HTML.jpg)

Gene mapping and annotation analysis of rs11130424. A The regional plot for rs11130424. –log10 P-values are shown along the left y-axis, and the right y-axis corresponds to recombination rate, plotted as a blue line. The x-axis indicates chromosomal position. The variants are colored according to their correlation (r2) with the lead variant (see legend). B Linkage disequilibrium (LD) analysis spanning the physical position from 54.554 to 54.604 Mb of chromosome 3. The color key indicates the level of LD (r2) between variants in 1000 Genomes Project East Asian populations. C The association between different genotypes at rs11130424 and methylation levels at the significant CpG probe cg04394267. The level of methylation at the CpG probe is shown as the β-value. The box plots show the distribution of the methylation levels in each genotype category with error bars representing the 25% and 75% quantiles. D 3D chromatin interaction and eQTL analysis of rs11130424. Circos plot showing genes on chromosome 3 interacted with rs11130424 by 3D chromatin interaction. The most outer layer showed the –log10(P values) of chromosome 3 in the GWAS analysis. Only SNPs with P < 0.05 are displayed. Links colored orange are chromatin interactions

### Ethnic frequency analysis

Considering that it was in Chinese NPC patients that the rs11130424 locus was identified, we explored its ethnic differences by comparing MAF in five major populations. As shown in Additional file [1](#MOESM1): Fig. S4 in the Supplement, the locus showed significant ethnic differences with a high frequency in East Asian populations, further indicating the potential and value of the rs11130424 locus as a biomarker in Chinese populations.

## Discussion

In our study, we explored the genetic variants associated with radiotherapy response using GWAS in a total of 1166 NPC patients. We identified a novel germline mutation of rs11130424 in the CACNA2D3 gene. The G allele carriers were more resistant to radiotherapy. Further stratified analysis showed that there was also a correlation existed in subgroups of EBV positive, smoking, late stage (III and IV) and patients received both concurrent chemoradiotherapy and induction/adjuvant chemotherapy. The efficacy was better in minor allele carriers of rs11130424 than major allele. Rs11130424 is located on the intron region of the CACNA2D3 gene, and the mapped genes were CACNA2D3 and CCDC66.

Radiotherapy is the main treatment for NPC, but there is some individual difference in the efficacy of treatment for patients receiving radiotherapy, the exact mechanism of which is not fully explained. For nasopharyngeal carcinoma, little is known about the genetic variants associated with radiotherapy sensitivity. Yang Z et al. showed an association between variants in autophagy-associated genes (ATG) and radiotherapy efficacy in nasopharyngeal carcinoma. ATG10 rs10514231, rs1864183 and rs4703533 were significantly associated with poorer radiotherapy efficacy at the primary site and positive lymph nodes, while ATG16L2 rs10898880 was positively associated with radiotherapy efficacy [[14](#CR14)]. A number of genetic variants have been reported to be associated with radiotherapy response for cancer, including XRCC1, X-ray repair cross complementing 2 (XRCC2), xeroderma pigmentosum group D (XPD), and EGFR [[15](#CR15), [16](#CR16)]. However, these studies have mainly been conducted by candidate gene association strategies, focusing on DNA repair genes. GWAS provides a powerful strategy to discover novel genetic variants. Therefore, we conducted a GWAS study to identify genetic variants associated with radiotherapy sensitivity in NPC patients at a genome-wide level. After a validation stage of association analysis, we eventually found that rs11130424 on chromosome 3p21.1 was associated with response to radiotherapy. This locus is a novel finding and has not been previously reported in nasopharyngeal carcinoma or other cancers.

To date, genome-wide association analysis studies of radiotherapy in head and neck cancers, including nasopharyngeal carcinoma, have focused more on radiotherapy toxicity, and they have revealed a number of genetic variants associated with multiple toxicities, including acute xerostomia [[17](#CR17)], radiotherapy-induced mucositis [[18](#CR18)], and radiotherapy-induced brain injury. [[19](#CR19)] Previously, we also performed a GWAS study of multiple acute radiotherapy toxicities in 1084 patients with nasopharyngeal carcinoma. Rs6711678, rs4848597, rs4848598 and rs2091255 on chromosome 2q14.2 and rs5845477 are novel risk loci for skin toxicity and dysphagia [[20](#CR20)]. In this study, we provided the first GWAS study for radiotherapy response in NPC patients. We found that rs11130424 on chromosome 3p21.1 was correlated with radiotherapy sensitivity. Gene mapping and annotation indicated its implicated gene was CACNA2D3 and CCDC66.

CACNA2D3 encodes a subunit of the voltage-dependent calcium channel complex that regulates intracellular Ca2 + influx. It was revealed that CACNA2D3 promoted radiation-induced apoptosis by regulating intracellular calcium ion concentration; and affected the expression of several genes, correlating with the activity of multiple signaling pathways in oesophageal cancer (Fig. [5](#Fig5)); in particular, CACNA2D3 inhibited the phosphorylation of PI3K and AKT [[21](#CR21)]. Studies have demonstrated that activation of PI3K signaling pathways is closely associated with the development of NPC [[22](#CR22)]. As the PI3K/AKT signaling pathway is highly activated in NPC and is closely associated with tumor radiotherapy sensitivity, CACNA2D3 is involved in regulating PI3K, and we speculate that this may be the mechanism by which variants in the CACNA2D3 gene are associated with the occurrence of radiotherapy resistance in patients. In addition, it has been reported that increased levels of Ca2 + in the cytoplasm activate the MAPK cascade [[23](#CR23)], and Ras, Raf, MEK and ERK proteins are key factors in this pathway. Studies have shown that CACNA2D3/CA2 + induces phosphorylation of p38 MAPK in endometrial cancer, which in turn induces apoptosis in tumor cells [[24](#CR24)]. CACNA2D3 may influence radiotherapy resistance by regulating the PI3K/AKT and Ras/MARK pathways (Fig. [5](#Fig5)).

### Fig. 5.

![Fig. 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3b50/9830790/af222d71668b/12967_2022_3819_Fig5_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9830790_12967_2022_3819_Fig5_HTML.jpg)

The potential molecular mechanisms of correlation between rs11130424 and efficacy after radiotherapy. Rs11130424 implicated expression of CACNA2D3, which regulates PI3K/AKT and Ras/MARK pathways. In addition, 3D chromatin interaction indicated that they interacted with CCDC66, which regulate tumor proliferation, metastasis, angiogenesis and apoptosis. PI3K phosphatidylinositol 3-kinase; MEK: kinse-ERK kinase, ERK extracellular signal-regulated kinase, MARK mitogen-activated protein kinase, caspase 8 apoptosis-related cysteine peptidase 8, caspase 3 apoptosis-related cysteine peptidase 3

CCDC66 is a member of the coiled-coil structural domain (CCDC) proteins, which act primarily as oncogenes to regulate tumor proliferation, metastasis, angiogenesis and apoptosis. Although there is no evidence that CCDC66 is directly involved in resistance to radiotherapy, another protein in the family, CCDC86, promotes the proliferation and invasive migration of NPC cells by regulating the activation of the PI3K/AKT pathway by EGFR. EGFR is a key upstream regulator of the PI3K/AKT pathway, and EGFR phosphorylation can indirectly activate PI3K [[25](#CR25)]. It has been implicated that the EGFR-PI3K/AKT pathway induces NPC cell senescence and suppresses the tumor stem cell phenotype [[26](#CR26), [27](#CR27)]. We hypothesized that CCDC66 may have a similar regulatory mechanism in nasopharyngeal carcinoma radiotherapy resistance (Fig. [5](#Fig5)).

In addition, the gene expression profile and clinical characteristics of [GSE48503](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE48503) were obtained from the GEO database ([https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi)) for download. CACNA2D3 and CCDC66 genes were also found among the differentially expressed gene profiles of radiation-resistant and radiation-sensitive NPC cells, which also reaffirmed the reliability of our results (Additional file [1](#MOESM1): Fig. S5).

The present study has several limitations. Firstly, the sample size of our study did not have sufficient power to conclusively demonstrate the association between rs11130424 and radiotherapy efficacy, and also reduced the possibility of identifying other genetic variants. Another limitation is that the exact mechanism by which methylation at the rs11130424 locus affects the phenotype is unclear, as it is located in the intron region of CACNA2D3 and the role of methylation in the gene has not been fully established. And in vitro functional studies are needed to validate it. Finally, more newly identified genetic variants can be combined with clinical factors to establish a comprehensive mathematical model for predicting individual efficacy in NPC in the future, thus providing a reference for individualized clinical treatment.

## Supplementary Information

## Acknowledgements

We thank all patients who participated in the study and the participating clinic staff for their contribution to data collection.

## Abbreviations

## Author contributions

JYY, C-XG and X-XX designed and guided the experiments. L-LY collected the samples and conducted the experiments. B-WH, H-XH, BY performed the data analyses. The manuscript was written by L-LY, and revised by J-YY. All authors read and approved the final manuscript.

## Funding

This work was supported by the National Natural Science Foundation of China (82073943, 81773823 and 81974511), the Hunan Cancer Hospital Climb Plan (YF2020011), the Scientific and Technological Project of Changsha (kq2004147), Science Foundation of Hunan Province (NO. 2022JJ30363), the Wisdom Accumulation and Talent Cultivation Project of the Third Xiangya Hospital of Central South University (YX202110), and The Open Fund for Scientific Research of NHC Key Laboratory of Personalized Diagnosis and Treatment of Nasopharyngeal Carcinoma (2021NPCK01).

## Availability of data and materials

The datasets and code that support the findings of this study are available from the corresponding author on reasonable request.

## Declarations

### Ethics approval and consent to participate

The study protocol was approved by the Ethics Committee of Jiangxi Province Tumor Hospital and Hunan Cancer Hospital. All subjects were provided with written informed consents.

### Consent for publication

Not applicable.

### Competing interests

The authors declare that they have no competing interests.

## Footnotes

## Contributor Information

Cheng-Xian Guo, Email: gchxyy@163.com.

Jin-Gao Li, Email: lijingao@hotmail.com.

Xiao-Xue Xie, Email: tongwandou58@126.com.

Ji-Ye Yin, Email: yinjiye@csu.edu.cn.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The datasets and code that support the findings of this study are available from the corresponding author on reasonable request.

### Supplementary Materials

### Data Availability Statement

The datasets and code that support the findings of this study are available from the corresponding author on reasonable request.

## References

1. Wong KCW, Hui EP, Lo KW, Lam WKJ, Johnson D, Li L, et al. Nasopharyngeal carcinoma: an evolving paradigm. Nat Rev Clin Oncol. 2021;18(11):679–95. doi: 10.1038/s41571-021-00524-x.  [DOI](https://doi.org/10.1038/s41571-021-00524-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34194007/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Clin%20Oncol&title=Nasopharyngeal%20carcinoma:%20an%20evolving%20paradigm&author=KCW%20Wong&author=EP%20Hui&author=KW%20Lo&author=WKJ%20Lam&author=D%20Johnson&volume=18&issue=11&publication_year=2021&pages=679-95&pmid=34194007&doi=10.1038/s41571-021-00524-x&)

2. Chen YP, Chan ATC, Le QT, Blanchard P, Sun Y, Ma J. Nasopharyngeal carcinoma. Lancet. 2019;394(10192):64–80. doi: 10.1016/S0140-6736(19)30956-0.  [DOI](https://doi.org/10.1016/S0140-6736(19)30956-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31178151/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Nasopharyngeal%20carcinoma&author=YP%20Chen&author=ATC%20Chan&author=QT%20Le&author=P%20Blanchard&author=Y%20Sun&volume=394&issue=10192&publication_year=2019&pages=64-80&pmid=31178151&doi=10.1016/S0140-6736(19)30956-0&)

3. Guigay J, Temam S, Bourhis J, Pignon JP, Armand JP. Nasopharyngeal carcinoma and therapeutic management: the place of chemotherapy. Ann Oncol. 2006;17(Suppl 10):304–7. doi: 10.1093/annonc/mdl278.  [DOI](https://doi.org/10.1093/annonc/mdl278) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17018743/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Nasopharyngeal%20carcinoma%20and%20therapeutic%20management:%20the%20place%20of%20chemotherapy&author=J%20Guigay&author=S%20Temam&author=J%20Bourhis&author=JP%20Pignon&author=JP%20Armand&volume=17&issue=Suppl%2010&publication_year=2006&pages=304-7&pmid=17018743&doi=10.1093/annonc/mdl278&)

4. Lee AW, Ma BB, Ng WT, Chan AT. Management of nasopharyngeal carcinoma: current practice and future perspective. J Clin Oncol. 2015;33(29):3356–64. doi: 10.1200/JCO.2015.60.9347.  [DOI](https://doi.org/10.1200/JCO.2015.60.9347) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26351355/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Management%20of%20nasopharyngeal%20carcinoma:%20current%20practice%20and%20future%20perspective&author=AW%20Lee&author=BB%20Ma&author=WT%20Ng&author=AT%20Chan&volume=33&issue=29&publication_year=2015&pages=3356-64&pmid=26351355&doi=10.1200/JCO.2015.60.9347&)

5. Kerns SL, Ostrer H, Rosenstein BS. Radiogenomics: using genetics to identify cancer patients at risk for development of adverse effects following radiotherapy. Cancer Discov. 2014;4(2):155–65. doi: 10.1158/2159-8290.CD-13-0197.  [DOI](https://doi.org/10.1158/2159-8290.CD-13-0197) | [PMC free article](/articles/PMC3946319/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24441285/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&title=Radiogenomics:%20using%20genetics%20to%20identify%20cancer%20patients%20at%20risk%20for%20development%20of%20adverse%20effects%20following%20radiotherapy&author=SL%20Kerns&author=H%20Ostrer&author=BS%20Rosenstein&volume=4&issue=2&publication_year=2014&pages=155-65&pmid=24441285&doi=10.1158/2159-8290.CD-13-0197&)

6. Yoon HH, Catalano PJ, Murphy KM, Skaar TC, Philips S, Powell M, et al. Genetic variation in DNA-repair pathways and response to radiochemotherapy in esophageal adenocarcinoma: a retrospective cohort study of the Eastern Cooperative Oncology Group. BMC Cancer. 2011;11:176. doi: 10.1186/1471-2407-11-176.  [DOI](https://doi.org/10.1186/1471-2407-11-176) | [PMC free article](/articles/PMC3118194/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21586140/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer.&title=Genetic%20variation%20in%20DNA-repair%20pathways%20and%20response%20to%20radiochemotherapy%20in%20esophageal%20adenocarcinoma:%20a%20retrospective%20cohort%20study%20of%20the%20Eastern%20Cooperative%20Oncology%20Group&author=HH%20Yoon&author=PJ%20Catalano&author=KM%20Murphy&author=TC%20Skaar&author=S%20Philips&volume=11&publication_year=2011&pages=176&pmid=21586140&doi=10.1186/1471-2407-11-176&)

7. Jin G, Fan XM, Li KX, Niu SH, Zhang QY. The association between epidermal growth factor receptor single nucleotide polymorphisms and Radiochemotherapy Response in Cervical Cancer. Pathol Oncol Res. 2020;26(2):1255–61. doi: 10.1007/s12253-019-00690-5.  [DOI](https://doi.org/10.1007/s12253-019-00690-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31254173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pathol%20Oncol%20Res&title=The%20association%20between%20epidermal%20growth%20factor%20receptor%20single%20nucleotide%20polymorphisms%20and%20Radiochemotherapy%20Response%20in%20Cervical%20Cancer&author=G%20Jin&author=XM%20Fan&author=KX%20Li&author=SH%20Niu&author=QY%20Zhang&volume=26&issue=2&publication_year=2020&pages=1255-61&pmid=31254173&doi=10.1007/s12253-019-00690-5&)

8. Li Q, Liang Y, Liu Z, Yu C. Associations of GWAS-Identified risk loci with progression, efficacy and toxicity of Radiotherapy of Head and Neck squamous cell carcinoma treated with Radiotherapy. Pharmgenomics Pers Med. 2021;14:1205–10. doi: 10.2147/PGPM.S325349.  [DOI](https://doi.org/10.2147/PGPM.S325349) | [PMC free article](/articles/PMC8464356/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34584443/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenomics%20Pers%20Med&title=Associations%20of%20GWAS-Identified%20risk%20loci%20with%20progression,%20efficacy%20and%20toxicity%20of%20Radiotherapy%20of%20Head%20and%20Neck%20squamous%20cell%20carcinoma%20treated%20with%20Radiotherapy&author=Q%20Li&author=Y%20Liang&author=Z%20Liu&author=C%20Yu&volume=14&publication_year=2021&pages=1205-10&pmid=34584443&doi=10.2147/PGPM.S325349&)

9. Dong SS, He WM, Ji JJ, Zhang C, Guo Y, Yang TL. LDBlockShow: a fast and convenient tool for visualizing linkage disequilibrium and haplotype blocks based on variant call format files. Brief Bioinform. 2021;22:4. doi: 10.1093/bib/bbaa227.  [DOI](https://doi.org/10.1093/bib/bbaa227) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33126247/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Brief%20Bioinform.&title=LDBlockShow:%20a%20fast%20and%20convenient%20tool%20for%20visualizing%20linkage%20disequilibrium%20and%20haplotype%20blocks%20based%20on%20variant%20call%20format%20files&author=SS%20Dong&author=WM%20He&author=JJ%20Ji&author=C%20Zhang&author=Y%20Guo&volume=22&publication_year=2021&pages=4&pmid=33126247&doi=10.1093/bib/bbaa227&)

10. Gong J, Wan H, Mei S, Ruan H, Zhang Z, Liu C, et al. Pancan-meQTL: a database to systematically evaluate the effects of genetic variants on methylation in human cancer. Nucleic Acids Res. 2019;47(D1):D1066-d1072. doi: 10.1093/nar/gky814.  [DOI](https://doi.org/10.1093/nar/gky814) | [PMC free article](/articles/PMC6323988/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30203047/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Pancan-meQTL:%20a%20database%20to%20systematically%20evaluate%20the%20effects%20of%20genetic%20variants%20on%20methylation%20in%20human%20cancer&author=J%20Gong&author=H%20Wan&author=S%20Mei&author=H%20Ruan&author=Z%20Zhang&volume=47&issue=D1&publication_year=2019&pages=D1066-d1072&pmid=30203047&doi=10.1093/nar/gky814&)

11. Pruim RJ, Welch RP, Sanna S, Teslovich TM, Chines PS, Gliedt TP, et al. LocusZoom: regional visualization of genome-wide association scan results. Bioinformatics. 2010;26(18):2336–7. doi: 10.1093/bioinformatics/btq419.  [DOI](https://doi.org/10.1093/bioinformatics/btq419) | [PMC free article](/articles/PMC2935401/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20634204/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=LocusZoom:%20regional%20visualization%20of%20genome-wide%20association%20scan%20results&author=RJ%20Pruim&author=RP%20Welch&author=S%20Sanna&author=TM%20Teslovich&author=PS%20Chines&volume=26&issue=18&publication_year=2010&pages=2336-7&pmid=20634204&doi=10.1093/bioinformatics/btq419&)

12. Watanabe K, Taskesen E, van Bochoven A, Posthuma D. Functional mapping and annotation of genetic associations with FUMA. Nat Commun. 2017;8(1):1826. doi: 10.1038/s41467-017-01261-5.  [DOI](https://doi.org/10.1038/s41467-017-01261-5) | [PMC free article](/articles/PMC5705698/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29184056/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Functional%20mapping%20and%20annotation%20of%20genetic%20associations%20with%20FUMA&author=K%20Watanabe&author=E%20Taskesen&author=A%20van%20Bochoven&author=D%20Posthuma&volume=8&issue=1&publication_year=2017&pages=1826&pmid=29184056&doi=10.1038/s41467-017-01261-5&)

13. Davis CA, Hitz BC, Sloan CA, Chan ET, Davidson JM, Gabdank I, et al. The encyclopedia of DNA elements (ENCODE): data portal update. Nucleic Acids Res. 2018;46(D1):D794-d801. doi: 10.1093/nar/gkx1081.  [DOI](https://doi.org/10.1093/nar/gkx1081) | [PMC free article](/articles/PMC5753278/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29126249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=The%20encyclopedia%20of%20DNA%20elements%20(ENCODE):%20data%20portal%20update&author=CA%20Davis&author=BC%20Hitz&author=CA%20Sloan&author=ET%20Chan&author=JM%20Davidson&volume=46&issue=D1&publication_year=2018&pages=D794-d801&pmid=29126249&doi=10.1093/nar/gkx1081&)

14. Yang Z, Liu Z. Potentially functional variants of autophagy-related genes are associated with the efficacy and toxicity of radiotherapy in patients with nasopharyngeal carcinoma. Mol Genet Genomic Med. 2019;7(12):e1030. doi: 10.1002/mgg3.1030.  [DOI](https://doi.org/10.1002/mgg3.1030) | [PMC free article](/articles/PMC6900379/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31692259/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Genet%20Genomic%20Med&title=Potentially%20functional%20variants%20of%20autophagy-related%20genes%20are%20associated%20with%20the%20efficacy%20and%20toxicity%20of%20radiotherapy%20in%20patients%20with%20nasopharyngeal%20carcinoma&author=Z%20Yang&author=Z%20Liu&volume=7&issue=12&publication_year=2019&pages=e1030&pmid=31692259&doi=10.1002/mgg3.1030&)

15. Yang Z, Liu Z. Potential functional variants in DNA repair genes are associated with efficacy and toxicity of radiotherapy in patients with non-small-cell lung cancer. J Oncol. 2020;2020:3132786. doi: 10.1155/2020/3132786.  [DOI](https://doi.org/10.1155/2020/3132786) | [PMC free article](/articles/PMC7333048/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32684929/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Oncol.&title=Potential%20functional%20variants%20in%20DNA%20repair%20genes%20are%20associated%20with%20efficacy%20and%20toxicity%20of%20radiotherapy%20in%20patients%20with%20non-small-cell%20lung%20cancer&author=Z%20Yang&author=Z%20Liu&volume=2020&publication_year=2020&pages=3132786&pmid=32684929&doi=10.1155/2020/3132786&)

16. Butkiewicz D, Krześniak M, Gdowicz-Kłosok A, Giglok M, Marszałek-Zeńczak M, Suwiński R. Polymorphisms in EGFR gene predict clinical outcome in unresectable non-small cell lung cancer treated with radiotherapy and platinum-based chemoradiotherapy. Int J Mol Sci. 2021;22:11. doi: 10.3390/ijms22115605.  [DOI](https://doi.org/10.3390/ijms22115605) | [PMC free article](/articles/PMC8197839/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34070597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci.&title=Polymorphisms%20in%20EGFR%20gene%20predict%20clinical%20outcome%20in%20unresectable%20non-small%20cell%20lung%20cancer%20treated%20with%20radiotherapy%20and%20platinum-based%20chemoradiotherapy&author=D%20Butkiewicz&author=M%20Krze%C5%9Bniak&author=A%20Gdowicz-K%C5%82osok&author=M%20Giglok&author=M%20Marsza%C5%82ek-Ze%C5%84czak&volume=22&publication_year=2021&pages=11&pmid=34070597&doi=10.3390/ijms22115605&)

17. Naderi E, Crijns APG, Steenbakkers R, van den Hoek JGM, Boezen HM, Alizadeh BZ, et al. A two-stage genome-wide association study of radiation-induced acute toxicity in head and neck cancer. J Transl Med. 2021;19(1):481. doi: 10.1186/s12967-021-03145-1.  [DOI](https://doi.org/10.1186/s12967-021-03145-1) | [PMC free article](/articles/PMC8626989/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34838041/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Transl%20Med&title=A%20two-stage%20genome-wide%20association%20study%20of%20radiation-induced%20acute%20toxicity%20in%20head%20and%20neck%20cancer&author=E%20Naderi&author=APG%20Crijns&author=R%20Steenbakkers&author=JGM%20van%20den%20Hoek&author=HM%20Boezen&volume=19&issue=1&publication_year=2021&pages=481&pmid=34838041&doi=10.1186/s12967-021-03145-1&)

18. Schack LMH, Naderi E, Fachal L, Dorling L, Luccarini C, Dunning AM, et al. A genome-wide association study of radiotherapy induced toxicity in head and neck cancer patients identifies a susceptibility locus associated with mucositis. Br J Cancer. 2022;126(7):1082–90. doi: 10.1038/s41416-021-01670-w.  [DOI](https://doi.org/10.1038/s41416-021-01670-w) | [PMC free article](/articles/PMC8980077/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35039627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=A%20genome-wide%20association%20study%20of%20radiotherapy%20induced%20toxicity%20in%20head%20and%20neck%20cancer%20patients%20identifies%20a%20susceptibility%20locus%20associated%20with%20mucositis&author=LMH%20Schack&author=E%20Naderi&author=L%20Fachal&author=L%20Dorling&author=C%20Luccarini&volume=126&issue=7&publication_year=2022&pages=1082-90&pmid=35039627&doi=10.1038/s41416-021-01670-w&)

19. Wang TM, Shen GP, Chen MY, Zhang JB, Sun Y, He J, et al. Genome-wide association study of susceptibility loci for radiation-induced brain injury. J Natl Cancer Inst. 2019;111(6):620–8. doi: 10.1093/jnci/djy150.  [DOI](https://doi.org/10.1093/jnci/djy150) | [PMC free article](/articles/PMC6579742/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30299488/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=Genome-wide%20association%20study%20of%20susceptibility%20loci%20for%20radiation-induced%20brain%20injury&author=TM%20Wang&author=GP%20Shen&author=MY%20Chen&author=JB%20Zhang&author=Y%20Sun&volume=111&issue=6&publication_year=2019&pages=620-8&pmid=30299488&doi=10.1093/jnci/djy150&)

20. Wang Y, Xiao F, Zhao Y, Mao CX, Yu LL, Wang LY, et al. A two-stage genome-wide association study to identify novel genetic loci associated with acute radiotherapy toxicity in nasopharyngeal carcinoma. Mol Cancer. 2022;21(1):169. doi: 10.1186/s12943-022-01631-8.  [DOI](https://doi.org/10.1186/s12943-022-01631-8) | [PMC free article](/articles/PMC9400233/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35999636/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer&title=A%20two-stage%20genome-wide%20association%20study%20to%20identify%20novel%20genetic%20loci%20associated%20with%20acute%20radiotherapy%20toxicity%20in%20nasopharyngeal%20carcinoma&author=Y%20Wang&author=F%20Xiao&author=Y%20Zhao&author=CX%20Mao&author=LL%20Yu&volume=21&issue=1&publication_year=2022&pages=169&pmid=35999636&doi=10.1186/s12943-022-01631-8&)

21. Nie C, Qin X, Li X, Tian B, Zhao Y, Jin Y, et al. CACNA2D3 enhances the chemosensitivity of esophageal squamous cell carcinoma to cisplatin via inducing Ca(2+)-mediated apoptosis and suppressing PI3K/Akt pathways. Front Oncol. 2019;9:185. doi: 10.3389/fonc.2019.00185.  [DOI](https://doi.org/10.3389/fonc.2019.00185) | [PMC free article](/articles/PMC6454090/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31001468/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Oncol.&title=CACNA2D3%20enhances%20the%20chemosensitivity%20of%20esophageal%20squamous%20cell%20carcinoma%20to%20cisplatin%20via%20inducing%20Ca(2+)-mediated%20apoptosis%20and%20suppressing%20PI3K/Akt%20pathways&author=C%20Nie&author=X%20Qin&author=X%20Li&author=B%20Tian&author=Y%20Zhao&volume=9&publication_year=2019&pages=185&pmid=31001468&doi=10.3389/fonc.2019.00185&)

22. Lin DC, Meng X, Hazawa M, Nagata Y, Varela AM, Xu L, et al. The genomic landscape of nasopharyngeal carcinoma. Nat Genet. 2014;46(8):866–71. doi: 10.1038/ng.3006.  [DOI](https://doi.org/10.1038/ng.3006) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24952746/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&title=The%20genomic%20landscape%20of%20nasopharyngeal%20carcinoma&author=DC%20Lin&author=X%20Meng&author=M%20Hazawa&author=Y%20Nagata&author=AM%20Varela&volume=46&issue=8&publication_year=2014&pages=866-71&pmid=24952746&doi=10.1038/ng.3006&)

23. Chen KC, Kao PH, Lin SR, Chang LS. Upregulation of Fas and FasL in Taiwan cobra phospholipase A2-treated human neuroblastoma SK-N-SH cells through ROS- and Ca2+-mediated p38 MAPK activation. J Cell Biochem. 2009;106(1):93–102. doi: 10.1002/jcb.21979.  [DOI](https://doi.org/10.1002/jcb.21979) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19009558/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biochem&title=Upregulation%20of%20Fas%20and%20FasL%20in%20Taiwan%20cobra%20phospholipase%20A2-treated%20human%20neuroblastoma%20SK-N-SH%20cells%20through%20ROS-%20and%20Ca2+-mediated%20p38%20MAPK%20activation&author=KC%20Chen&author=PH%20Kao&author=SR%20Lin&author=LS%20Chang&volume=106&issue=1&publication_year=2009&pages=93-102&pmid=19009558&doi=10.1002/jcb.21979&)

24. Kong X, Li M, Shao K, Yang Y, Wang Q, Cai M. Progesterone induces cell apoptosis via the CACNA2D3/Ca2+/p38 MAPK pathway in endometrial cancer. Oncol Rep. 2020;43(1):121–32. doi: 10.3892/or.2019.7396.  [DOI](https://doi.org/10.3892/or.2019.7396) | [PMC free article](/articles/PMC6908942/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31746409/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncol%20Rep&title=Progesterone%20induces%20cell%20apoptosis%20via%20the%20CACNA2D3/Ca2+/p38%20MAPK%20pathway%20in%20endometrial%20cancer&author=X%20Kong&author=M%20Li&author=K%20Shao&author=Y%20Yang&author=Q%20Wang&volume=43&issue=1&publication_year=2020&pages=121-32&pmid=31746409&doi=10.3892/or.2019.7396&) | [Retracted](/articles/PMC11267494/)

25. Martelli AM, Evangelisti C, Chiarini F, Grimaldi C, McCubrey JA. The emerging role of the phosphatidylinositol 3-kinase/ akt/mammalian target of rapamycin signaling network in cancer stem cell biology. Cancers (Basel) 2010;2(3):1576–96. doi: 10.3390/cancers2031576.  [DOI](https://doi.org/10.3390/cancers2031576) | [PMC free article](/articles/PMC3837323/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24281174/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&title=The%20emerging%20role%20of%20the%20phosphatidylinositol%203-kinase/%20akt/mammalian%20target%20of%20rapamycin%20signaling%20network%20in%20cancer%20stem%20cell%20biology&author=AM%20Martelli&author=C%20Evangelisti&author=F%20Chiarini&author=C%20Grimaldi&author=JA%20McCubrey&volume=2&issue=3&publication_year=2010&pages=1576-96&pmid=24281174&doi=10.3390/cancers2031576&)

26. Chan KC, Ting CM, Chan PS, Lo MC, Lo KW, Curry JE, et al. A novel Hsp90 inhibitor AT13387 induces senescence in EBV-positive nasopharyngeal carcinoma cells and suppresses tumor formation. Mol Cancer. 2013;12(1):128. doi: 10.1186/1476-4598-12-128.  [DOI](https://doi.org/10.1186/1476-4598-12-128) | [PMC free article](/articles/PMC3834878/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24156782/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer&title=A%20novel%20Hsp90%20inhibitor%20AT13387%20induces%20senescence%20in%20EBV-positive%20nasopharyngeal%20carcinoma%20cells%20and%20suppresses%20tumor%20formation&author=KC%20Chan&author=CM%20Ting&author=PS%20Chan&author=MC%20Lo&author=KW%20Lo&volume=12&issue=1&publication_year=2013&pages=128&pmid=24156782&doi=10.1186/1476-4598-12-128&)

27. Ma L, Zhang G, Miao XB, Deng XB, Wu Y, Liu Y, et al. Cancer stem-like cell properties are regulated by EGFR/AKT/β-catenin signaling and preferentially inhibited by gefitinib in nasopharyngeal carcinoma. Febs j. 2013;280(9):2027–41. doi: 10.1111/febs.12226.  [DOI](https://doi.org/10.1111/febs.12226) | [PMC free article](/articles/PMC3831031/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23461856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Febs%20j&title=Cancer%20stem-like%20cell%20properties%20are%20regulated%20by%20EGFR/AKT/%CE%B2-catenin%20signaling%20and%20preferentially%20inhibited%20by%20gefitinib%20in%20nasopharyngeal%20carcinoma&author=L%20Ma&author=G%20Zhang&author=XB%20Miao&author=XB%20Deng&author=Y%20Wu&volume=280&issue=9&publication_year=2013&pages=2027-41&pmid=23461856&doi=10.1111/febs.12226&)
