# Associations between CES1 variants and dosing and adverse effects in children taking methylphenidate

## Metadata
**Authors:** Jacob T Brown, Nancy Beery, Allise Taran, Tyler Stevens, Christine Henzler, Jonathan Badalamenti, Ron Regal, Catherine A McCarty
**Journal:** Frontiers in Pediatrics
**Date:** 2023 Jan 18
**DOI:** [10.3389/fped.2022.958622](https://doi.org/10.3389/fped.2022.958622)
**PMID:** 36741090
**PMCID:** PMC9890192
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9890192/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9890192/pdf/fped-10-958622.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9890192/pdf/fped-10-958622.pdf)

## Abstract

**Background:** 
Methylphenidate is the most prescribed stimulant to treat attention deficit-hyperactivity disorder (ADHD). Despite its widespread usage, a fair proportion of children are classified as non-responders to the medication. Variability in response and occurrence of adverse events with methylphenidate use may be due to several factors, including drug-drug interactions as well as pharmacogenetic differences resulting in pharmacokinetic and/or pharmacodynamic variances within the general population. The objective of this study was to analyze the effect of carboxylesterase 1 (CES1) variants on the frequency of adverse effects and dosing requirements of methylphenidate in children with ADHD.

**Methods:** 
This was a retrospective cohort study of children and adolescents who met the inclusion criteria and had a routine visit during the enrollment period were invited to participate. Inclusion criteria included: ADHD diagnosis by a healthcare provider, between 6 and 16 years of age at the time of permission/assent, had not previously been prescribed methylphenidate, and treatment with any methylphenidate formulation for at least three consecutive months. Three months of records were reviewed in order to assess changes in dose and frequency of discontinuing methylphenidate. Participants’ ADHD symptoms, medication response, adverse effects, select vitals, and dose were extracted from the electronic health record. Saliva samples were collected by trained study coordinators. Haplotypes were assigned based on copy number in different portions of the CES1 gene. Due to limited numbers, diplotypes (combinations of two haplotypes) were grouped for analysis as CES1A1/CES1A1, CES1A1/CES1A1c and CES1A1c/CES1A1c.

**Results:** 
A total of 99 participants (n = 30 female; n = 69 male) had both clinical data and CES1 sequencing data, with an average age of 7.7 years old (range 3–15 years). The final weight-based dose in all individuals was 0.79 mg/kg/day. The most common adverse effects reported were decreased appetite (n = 47), weight loss (n = 24), and sleep problems (n = 19). The mean final weight-based dose by haplotype was 0.92 mg/kg for CES1A2/CES1A2, 0.81 mg/kg for CES1A2/CES1P1, and 0.78 mg/kg for CES1P1/CES1P1. After correction for multiple hypothesis testing, only one SNV, rs114119971, was significantly associated with weight-based dosing in two individuals. The individuals with the rs114119971 SNV had a significantly lower weight-based dose (0.42 mg/kg) as compared to those without (0.88 mg/kg; p < 0.001).

**Discussion:** 
Variation in CES1 activity may impact dose requirements in children who are prescribed methylphenidate, as well as other CES1 substrates. Although intriguing, this study is limited by the retrospective nature and relatively small sample size.

Keywords: pediatrics, ADHD, methylphenidate, CES1, pharmacogenetics

### Background

Methylphenidate is the most prescribed stimulant to treat attention deficit-hyperactivity disorder (ADHD). Despite its widespread usage, a fair proportion of children are classified as non-responders to the medication. Variability in response and occurrence of adverse events with methylphenidate use may be due to several factors, including drug-drug interactions as well as pharmacogenetic differences resulting in pharmacokinetic and/or pharmacodynamic variances within the general population. The objective of this study was to analyze the effect of carboxylesterase 1 (*CES1*) variants on the frequency of adverse effects and dosing requirements of methylphenidate in children with ADHD.

### Methods

This was a retrospective cohort study of children and adolescents who met the inclusion criteria and had a routine visit during the enrollment period were invited to participate. Inclusion criteria included: ADHD diagnosis by a healthcare provider, between 6 and 16 years of age at the time of permission/assent, had not previously been prescribed methylphenidate, and treatment with any methylphenidate formulation for at least three consecutive months. Three months of records were reviewed in order to assess changes in dose and frequency of discontinuing methylphenidate. Participants’ ADHD symptoms, medication response, adverse effects, select vitals, and dose were extracted from the electronic health record. Saliva samples were collected by trained study coordinators. Haplotypes were assigned based on copy number in different portions of the CES1 gene. Due to limited numbers, diplotypes (combinations of two haplotypes) were grouped for analysis as CES1A1/CES1A1, CES1A1/CES1A1c and CES1A1c/CES1A1c.

### Results

A total of 99 participants (*n* = 30 female; *n* = 69 male) had both clinical data and *CES1* sequencing data, with an average age of 7.7 years old (range 3–15 years). The final weight-based dose in all individuals was 0.79 mg/kg/day. The most common adverse effects reported were decreased appetite (*n* = 47), weight loss (*n* = 24), and sleep problems (*n* = 19). The mean final weight-based dose by haplotype was 0.92 mg/kg for CES1A2/CES1A2, 0.81 mg/kg for CES1A2/CES1P1, and 0.78 mg/kg for CES1P1/CES1P1. After correction for multiple hypothesis testing, only one SNV, rs114119971, was significantly associated with weight-based dosing in two individuals. The individuals with the rs114119971 SNV had a significantly lower weight-based dose (0.42 mg/kg) as compared to those without (0.88 mg/kg; *p* < 0.001).

### Discussion

Variation in CES1 activity may impact dose requirements in children who are prescribed methylphenidate, as well as other CES1 substrates. Although intriguing, this study is limited by the retrospective nature and relatively small sample size.

## Introduction

Attention deficit hyperactivity disorder (ADHD) prevalence worldwide varies considerably, and is estimated to be around 5%, ranging from 2%–7% ([1](#B1)). Among children 6–17 years of age in the United States, ADHD is the most prevalent neurodevelopmental disorder, with 9.5% of all U.S. children between 6 and 17 years of age having received a diagnosis of ADHD at any point in their lifetime ([2](#B2)). The overall prevalence in children 3–17 years of age has increased by 33% from 1997–1999 to 2006–2008 ([3](#B3)). Additionally, a previously estimated cost of illness of ADHD estimated the annual individual costs of ADHD to be between $12,005 and $17,458, and the total annual societal cost between $36 and $52.4 billion ([4](#B4)). ADHD currently has a prevalence estimated to be around 5% ([5](#B5)).

Methylphenidate is first-line treatment in children and adults, is available in several formulations, and is the most prescribed stimulant to treat attention deficit-hyperactivity disorder (ADHD), as well as the most dispensed medication to adolescents 12–17 years with approximately 10 million prescriptions per year from 2002 to 2010 in children 0–17 years of age ([6](#B6), [7](#B7)). Despite its widespread usage, a fair proportion of children are classified as non-responders to the medication, while roughly half remain on the medication after one year ([8](#B8)). This variation in response may be due in part to individual pharmacokinetic differences in metabolism and/or pharmacodynamic differences in receptors and transporters. Methylphenidate is a stimulant that works by inhibiting the reuptake of norepinephrine and dopamine into presynaptic neurons, thus increasing these neurotransmitters in the brain. Due to methylphenidate's mechanism of action, research has focused on several genes looking at individual response to treatment, including the dopamine transporter, dopamine receptor, and norepinephrine transporter. Studies comparing genetic variation within these genes have led to mixed results thus far and have resulted in minimal clinical impact ([9](#B9)–[11](#B11)).

Methylphenidate is metabolized to the inactive metabolite ritalinic acid by the enzyme carboxylesterase 1 (CES1) ([12](#B12)). Numerous single nucleotide variants (SNVs) and copy number variants (CNVs) have been identified within the *CES1* gene, with some resulting in altered enzyme activity, and either increased or decreased levels of methylphenidate plasma concentrations at standard doses ([13](#B13), [14](#B14)). These differences in overall exposure may lead to differences in both clinical response and adverse effects.

Few clinical studies have been completed describing the relationship between CES1 and methylphenidate with adverse event and clinical response rates. Zhu et al. demonstrated marked pharmacokinetic differences of methylphenidate in an individual with two variants in the *CES1* gene. One variant described a nonconservative amino acid substitution of glycine to glutamic acid (G143E; rs71647871), while the second was a frameshift mutation of *CES1*. This individual experienced a markedly increased area under the curve (AUC), maximum concentration, and half-life as compared to 19 other participants, as well as greater hemodynamic increase. While the frameshift mutation is considered rare (<1%), the glycine to glutamic acid was found in 3.7%, 4.3%, and 2.0% of white, black, and Hispanic populations ([13](#B13)). Furthermore, the G143E variant has also been shown to impair the bioactivation of oseltamivir, which is also a substrate for CES1 ([15](#B15)).

Variability in response and occurrence of adverse events with methylphenidate use may be due to pharmacogenetic differences resulting in pharmacokinetic and/or pharmacodynamic variances within the general population. These differences may be the result of varying levels of drug exposure among individuals at comparable doses, which may directly impact the occurrence of adverse effects and clinical response rates. Improving treatment by selecting the most appropriate dose for an individual may lead to better utilization of ADHD medications, resulting in less treatment failure and better symptomatic control. The objective of this study was to analyze the effect of *CES1* variants on the frequency and severity of adverse effects and dosing requirements of methylphenidate in children with ADHD.

## Materials and methods

### Study design

This was a retrospective cohort study examining variation in the *CES1* gene and clinical characteristics in children and adolescents with ADHD prescribed any formulation of methylphenidate. Children and adolescents who met the inclusion criteria and had a routine visit during the enrollment period were invited to participate through mailed invitation letters and a follow-up phone call from study staff. Inclusion criteria included: ADHD diagnosis by a healthcare provider included in their medical record, between 6 and 16 years of age at the time of permission/assent, no previous trial of methylphenidate, and treatment with methylphenidate for at least three consecutive months without concomitant use of another drug to treat ADHD or MAOIs. Exclusion criteria included those with hypersensitivity to methylphenidate, marked agitation, anxiety, or tension, motor tics, diagnosis of Tourette syndrome, or any cardiac abnormality. Informed parental permission, child assent, and saliva collection were obtained at the participant's regularly scheduled appointment by trained study coordinators in cooperation with their health care provider and nursing staff. This study was reviewed and approved by the Essentia Health Institutional Review Board.

### Clinical assessment

The primary outcomes of interest included known methylphenidate adverse effects and daily dose. Participants' ADHD symptoms, medication response, adverse effects, and select vitals (height, weight, blood pressure, and heart rate) were assessed using data extracted from the electronic health record through analytics and manual abstraction conducted by research staff.

Study data were collected and managed using REDCap electronic data capture tools hosted at Essentia Health. REDCap (Research Electronic Data Capture) is a secure, web-based application designed to support data capture for research studies, providing: (1) an intuitive interface for validated data entry; (2) audit trails for tracking data manipulation and export procedures; (3) automated export procedures for seamless data downloads to common statistical packages; and (4) procedures for importing data from external sources ([16](#B16), [17](#B17)).

Participants and their accompanying parent or guardian were asked at the time of saliva collection to self-report race and ethnicity based on standardized data collection tools (phenxtoolkit.org) as well as the names and dates of birth of the participant's biological parents and grandparents (if available) to determine and account for relatedness of study participants at the time of analysis.

### Sample collection and CES1 sequencing

Saliva samples were collected by trained study coordinators using Oragene® DISCOVER (OGR-500 and OGR-575) saliva collection kits, and were stored in a laboratory setting until DNA extraction and gene sequencing were conducted. Each sample was extracted following the automated QIAamp DNA Mini Kit (QIAGEN, Hilden, Germany) protocol for the QIAcube (QIAGEN, Hilden, Germany). Purified DNA extracts were set to elute in 2 increments of 100 μl of AE buffer (200 μl total elution volume). DNA quantification was performed in triplicate using the Qubit™ dsDNA HS Assay Kit (Thermo-Fisher, Waltham, MA) for the 2.0 Qubit™ fluorometric quantification system according to manufacturer's instructions.

Amplicon design and approximate primer placement were performed using Geneious Primer v. 2019.2.3 against the repeat-masked CES1 locus that included 20 kbp up- and downstream (build hg38 in UCSC Genome Browser). Concurrent GC% and conserved SNP tracks were plotted in Geneious alongside the reference to facilitate visual inspection of candidate regions for primer placement such that (1) resulting amplicons had a final size of ∼6 kbp +/− 15%, and (2) amplicons overlapped by at least 200 bp. Genomic coordinates for these candidate ranges were used to iteratively design primers for each amplicon using NCBI PrimerBLAST to have a final length of 20–26 nt and calculated Tm between 58 and 62 °C. Where possible 3′ GC clamps were included and homopolymers limited to 5 nt. Oligonucleotide primers were synthesized by Integrated DNA Technologies (Coralville, IA) with standard desalting and included a single phosphorothioate linkage before the terminal 3′ base to reduce off-target amplification from primer editing ([18](#B18)).

Amplicons were generated using Q5 HiFi HotStart mastermix (New England Biolabs, Ipswich, MA) in a 20-µl PCR containing 10 µl 2× mastermix, 1 µl of forward primer (10 µM), 1 µl of reverse primer (10 µM), 7 µl nuclease-free water, and 1 µl (10 ng/µl) of extracted gDNA. Amplification was performed according to the following cycling conditions (all except region (4): 98 °C for 30 s, followed by 30 cycles of 98 °C for 10 s, 64 °C for 15 s, and 72 °C for 3 min, and a final extension at 72 °C for 5 min. Region 4 used an extension time of 3 min 30 s. PCR products were purified using 0.5× (v/v) Ampure XP beads (Beckman-Coulter), quantified using Picogreen, and normalized to 10 ng/µl. Normalized amplicons from the same individual were pooled at equal volume (2 µl each) and concentrated to ∼5 µl by evaporation before undergoing PacBio library creation (SMRT Bell Express Template Kit 2.0, Pacific Biosciences, Menlo Park, CA) with sample barcodes added *via* ligation. Once barcoded, all libraries were pooled together and sequenced on 1 PacBio 8M SMRT cell (3 pM loading concentration) with 2-h pre-extension.

Circular consensus reads (CCS) were generated for all samples using PacBio's ccs v.6.0.0, the default minimum of 3 full passes (three full reads through the amplicon), a minimum target length of 4,000 and a maximum target length of 8,000. Reads were aligned to the human genome (GrCh38) using pbmm2 v.1.4.0, PacBio's wrapper for minimap2 ([19](#B19)). Bedtools v2.29.2 ([20](#B20)) was used to measure coverage for both CES1 and CES1P1 (to check for off-target, non-specific amplification.) For each amplicon, 30× coverage was required for a sample to be included in the final analysis. For most amplicons, only four or fewer samples had insufficient depth to be retained in the final analysis, however amplicon 4 failed to amplify for 26 samples. There was virtually no amplification of CES1P1 (as designed). The highest coverage in any sample at any point in CES1P1 was 14× (while the same sample had an average coverage of CES1P1 of only 0.08× and a minimum of 2,654× coverage for CES1).

Variants were called from the aligned bams using freebayes v.1.3.4 ([21](#B21)) using the following parameters: (1) Requiring a minimum of 5 reads and 10% of reads supporting an alternate call (-C 5, -F 0.10), (2) Restricting calls to just the amplified regions, (3) “Output all alleles which pass input filters regardless of genotyping outcome or model.” (–pooled-continuous) (–haplotype-length 0), (4) Haplotype length set to 0 to emit simple SNP and indel calls (rather than complex haplotype calls) (–haplotype-length 0). Variant calls were filtered with bcftools v.1.6 ([22](#B22)) to remove calls with a quality score < 20. SNPs were phased using whatshap v.1.0 ([23](#B23)).

In a parallel analysis to the alignment and variant calling, PacBio's (long amplicon analysis tool v.2.4.2 (LAA) [[https://github.com/PacificBiosciences/pblaa](https://github.com/PacificBiosciences/pblaa)]) was used to collapse reads into phased haplotypes, representing distinct allele sequences for each amplicon within each sample. The relative number of reads supporting each haplotype from this analysis was used as additional confirmation of the copy number calling. Haplotypes were assigned based on copy number in different portions of the *CES1* gene, using custom R code. Due to limited numbers diplotypes (i.e., combinations of two haplotypes) were grouped for analysis as CES1A1/CES1A1, CES1A1/CES1A1c and CES1A1c/CES1A1c.

The relationship between individual SNVs and clinical correlates were assessed for a list of SNVs determined from the literature. Linkage disequilibrium was calculated for all SNVs, and to account for the most closely linked SNVs, for every pair of SNVs with an *R*^2^ > 0.95 the SNV present in the highest number of samples was chosen for further analysis. (If two SNVs were in the same number of samples, the first one was chosen.) This dropped the number of key SNVs analyzed against clinical correlates to 19. ([Supplementary File](#s10)) SNVs were treated as simply present or absent, and were tested against binary clinical traits (e.g., side effects) with Fisher's exact test, and against continuous traits {e.g., log2[dose by weight (mg/kg)]} with a *t*-test. The Benjamini-Hochberg correction was used to correct for multiple hypothesis tests across all tests (all SNVs and all clinical correlates) ([Figure 1](#F1)).

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2f5b/9890192/4539e2b83ebc/fped-10-958622-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9890192_fped-10-958622-g001.jpg)

Notable CES1 SNPs.

## Results

A total of 99 participants (*n* = 30 female; *n* = 69 male) had both clinical data and *CES1* sequencing data ([Figure 2](#F2)), with an average age of 7.7 years old (range 3–15 years) at their initial visit. Of note, methylphenidate is approved for children 6 years and older, but is sometimes used in younger children, and our data included children as young as three years of age.

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2f5b/9890192/d2adf532ee5c/fped-10-958622-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9890192_fped-10-958622-g002.jpg)

Study flowchart.

Age was significantly associated with both the final absolute dose (*p* = 0.018) and weight-based dose (mg/kg; *p* = 0.005), though the *r*^2^ values for both were relatively low at 0.056 ([Figure 3](#F3)) and 0.078, respectively. The final average absolute dose in all participants was 24.2 mg/day (22.4 mg/day in females vs. 25.0 mg/day in males), while the final weight-based dose in all individuals was 0.79 mg/kg/day (0.74 mg/kg/day in females vs. 0.81 mg/kg/day in males). In total, 65 participants had any change in dose, 61 had a dose increase, and 15 had a dose decrease. Adverse effects were reported for 69 (female = 20, male = 49) of the individuals. The most common adverse effects reported were decreased appetite (*n* = 47; 14 females, 33 males), other (*n* = 27), weight loss (*n* = 24; 10 females, 14 males), and sleep problems (*n* = 19; 4 females, 15 males). The mean final weight-based dose by haplotype is described in [Table 1](#T1).

### Figure 3.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2f5b/9890192/ce15ab06cd57/fped-10-958622-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9890192_fped-10-958622-g003.jpg)

Comparing age and final methylphenidate dose across CES1 haplotypes.

### Table 1.

| Haplotype | All | Males | Females |
| --- | --- | --- | --- |
| CES1A1/CES1A1/CES1A2/CES1A2 (n = 3) | 0.92 | 0.92 | – |
| CES1A1/CES1A1/CES1A2/CES1P1 (n = 16) | 0.88 | 0.99 | 0.70 |
| CES1A1/CES1A1/CES1P1/CES1P1 (n = 45) | 0.75 | 0.77 | 0.69 |
| CES1A1/CES1A1c/CES1A2/CES1P1 (n = 8) | 0.66 | 0.79 | 0.44 |
| CES1A1/CES1A1c/CES1P1/CES1P1 (n = 22) | 0.90 | 0.88 | 0.94 |
| CES1A1c/CES1A1c/CES1A2/CES1P1 (n = 2) | 0.81 | 0.81 | – |
| CES1A1c/CES1A1c/CES1P1/CES1P1 (n = 3) | 0.38 | 0.38 | – |

Table 1 Caption: Methylphenidate dose by CES1 haplotype (mg/kg).

The CES1 locus and 12 kb upstream were sequenced with overlapping ∼6 kb amplicons and PacBio Sequel sequencing. Nine tiled, overlapping primer sets were designed to cover all CES1 and 12 kb upstream, but primer sets 8 and 9, targeting the 3′ end of the gene, were not successful and not included in the final project. Additionally, amplicon 4 performed poorly and a subset of samples are either missing amplicon 4 or have low coverage. Amplicon 4 covers exon 2 and 3. As a result, the final sequencing data cover 12 kb upstream of CES1 to partway through intron 11 of the 14-exon CES1 gene for 99 individuals, and 19 individuals are missing data for exons 2 and 3 (amplicon 4). Most samples (77 of the 99 final samples) had no amplicons with less than the threshold 30× coverage required for further analysis, and most amplicons had vastly more than 30× coverage (median coverage for all exons was >2000×). Of the 22 samples where an amplicon was below 30× coverage, in 20 samples only one amplicon was <30×, and in the other two samples, it was two amplicons. Given the good coverage across amplicons, haplotypes could reasonably be inferred for 99 samples that also had clinical correlates ([Supplementary File](#s10)).

Before correction for multiple hypothesis testing, nine SNVs showed significant associations, three with weight-based doses, five with weight loss, and one with both weight loss and dose increase. The three SNVs associated with weight-based doses [determined by t-test of log2(last dose in mg/kg) for children with and without the SNV] were rs114119971 (*p* = 4.3e–13), ([Figure 4](#F4)) rs4122238 (0.77 with the SNV mg/kg vs. 1.1 mg/kg without the SNV; *p* = 0.0095), ([Figure 5](#F5)) and rs74019272 (0.28 mg/kg with the SNV vs. 0.81 mg/kg; *p* = 0.023). ([Figure 6](#F6)) The six SNVs associated with weight loss were rs32171764 (*p* = 0.003), rs2244614 (*p* = 0.0039), rs1968753 (*p* = 0.0065), rs2302719 (*p* = 0.014), rs2244613 (*p* = 0.032), and rs2002577 (*p* = 0.039). SNV rs3217164 was also associated with dose increase (*p* = 0.032). Correction for multiple hypothesis testing assumes independence of tests. While the SNVs with the highest linkage disequilibrium were filtered from the dataset so only one of each pair of highly-linked SNVs was analyzed further, there was still considerable linkage disequilibrium among the remaining SNVs. As such, the remaining SNVs tested may not be truly independent, and the Benjamini-Hochberg correction for multiple hypothesis testing may be too stringent. It is worth considering the tests that were still significant after multiple hypothesis correction as the strongest result, but those that were only significant prior to the correction may also be worth consideration for further research.

### Figure 4.

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2f5b/9890192/bf377eeabe1c/fped-10-958622-g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9890192_fped-10-958622-g004.jpg)

Comparing dose (mg/kg) with rs114119971.

### Figure 5.

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2f5b/9890192/017453c1538e/fped-10-958622-g005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9890192_fped-10-958622-g005.jpg)

Comparing dose (mg/kg) with rs4122238.

### Figure 6.

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2f5b/9890192/f01d69658a52/fped-10-958622-g006.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9890192_fped-10-958622-g006.jpg)

Comparing dose (mg/kg) with rs74019272.

After correction for multiple hypothesis testing, only one SNV, the previously mentioned rs114119971, was significantly associated with weight-based dosing, and included only two individuals with the SNV. The two individuals with the rs114119971 SNV had a significantly lower weight-based dose [as determined by a t-test of log2(final dose in mg/kg) in children with and without the SNV, final dose with rs114119971 = 0.42 mg/kg] as compared to those without the SNV (final dose = 0.88 mg/kg; *p* = 4.3e–13, adjusted *p* = 1.3e–10). One individual was identified as having the rs71647871 SNV, with a final absolute dose of 27 mg and weight-based of 0.37 mg/kg.

## Discussion

Due to the many variables involved, clinical implementation of pharmacogenetic testing in children with ADHD has proven challenging. Although this study was unable to discern any associations between clinical response to methylphenidate and *CES1* genetic variation, we did identify several *CES1* SNVs associated with weight-based dosing and weight loss; however, after correction for multiple hypothesis testing only a single SNV remained statistically significant. While sparsely studied, associations between *CES1* and methylphenidate dose, response, and adverse effects have been previously reported.

Previously, pharmacokinetic studies have described a handful of variants significantly associated with methylphenidate exposure. The previously noted Zhu et al. article described two functional *CES1* variants in a single adult resulting in decreased enzyme activity as shown by increased exposure to methylphenidate ([13](#B13)). Additionally, Stage et al. showed a similar increased exposure to methylphenidate of the rs71647871 SNV in a group of Danish adults ([14](#B14)). Similar effects for the rs71647871 variant were shown when examined as the metabolic ratio of ritalinic acid to methylphenidate ([24](#B24)). While the pharmacokinetic impact of this variant is well defined, only a single individual with this variant was included in this study.

Nemoda et al. described children classified as responders to methylphenidate with the glycine to glutamic variant allele require significantly smaller dosing requirements as compared to responders without the variant allele (0.41 vs. 0.57 mg/kg) ([25](#B25)). However, these results are limited as they were based upon five individuals with the variant allele. We were unable to compare our own results to these, as only one individual completing all study procedures also had this variant. Of the variants with significant associations with weight-based dosing, the rs114119971 SNP is a missense variant, while the rs4122238 and rs74019272 are both intron variants.

The rs114119971 SNP identified as being significantly associated with weight-based dosing after multiple hypothesis correction is a missense variant with a low allele frequency (0.2%–0.9%). No other studies were identified describing this variant with methylphenidate (or any other CES1 substrate) dosing or clinical response, and this SNP was found in only two participants in the study described herein. Of note, these two individuals required significantly lower doses as compared to those without the rs114119971 variant. Further study examining the pharmacokinetic effect of this variant on methylphenidate and other CES1 substrates may be warranted.

Recent clinical studies have compared side effect profile with genetic variation in the *CES1* gene in children taking methylphenidate for ADHD. Specifically, Johnson et al. identified two *CES1* SNV markers (rs2244613 and rs2002577) in linkage disequilibrium with two SNVs in the norepinephrine transporter gene associated with sadness as a side effect ([26](#B26)). While our study did not find an association between these SNPs and sadness, they were found to be associated with weight loss prior to multiple hypothesis correction. The rs2244613 variant has also been associated with low trough concentrations of dabigatran, which is also a CES1 substrate requiring bioactivation, with an allele frequency of approximately 13.2%–40.3% ([27](#B27)). Additionally, Bruxel et al. described a different *CES1* variant (rs3815583) where carriers had a significant odds ratio of 3.5 for appetite reduction worsening as compared to those who lacked the variant ([28](#B28)), although we did not find a similar association.

Of the five SNPs associated with weight loss, all were intron variants with high allele frequencies. The rs2244614 variant has an allele frequency of 18%–50% and was not shown to impact the pharmacokinetics or toxicity of capecitabine in colorectal cancer patients ([29](#B29)). An additional study examining the pharmacodynamic effects of CES1 variation, including rs2244614, in patients treated with angiotensin converting enzyme (ACE) inhibitors with congestive heart failure did not show significant associations ([30](#B30)). The rs1968753 variant has an allele frequency of 19%–45% and was previously shown to possibly be a candidate for risk prediction of antituberculosis drug-induced hepatotoxicity ([31](#B31)). The rs2244613 variant is relatively well described in CES1 substrates other than methylphenidate and has an allele frequency of 15%–40%. The rs2244613 variant has been associated with lower dabigatran trough concentrations ([27](#B27)), reduced risk of diarrhea with irinotecan treatment ([32](#B32)), and lower peak and trough enalaprilat concentrations ([33](#B33)). The rs2302719 and rs2002577 SNVs are both intron variants with relatively high allele frequencies, though no studies were identified describing these.

While this study focused on the impact of variation in CES1 on methylphenidate dosing and adverse effects, other non-genetic factors also contribute to CES1 activity. These include natural products capable of inhibiting CES1 activity ([34](#B34)), alcohol ([35](#B35)), and therapeutic agents which may result in drug-drug interactions ([36](#B36)). Additionally, variation in several other genes have been described as it relates to methylphenidate adverse events and clinical response ([37](#B37), [38](#B38)).

Although studies describing the pharmacokinetic and pharmacodynamic effects of CES1 on medications such as methylphenidate, ACE-inhibitors, and dabigatran are increasing, CES1 is rarely included in commercially available pharmacogenetic testing panels. Based on registered laboratories through the National Library of Medicine Genetic Testing Registry, only six laboratories include *CES1* genetic testing, five of which are located within the United States. Variants such as rs71647871 that have been shown to increase exposure to CES1 substrates could be clinically useful in identifying individuals who may respond better to a lower dose or an alternative medication.

This study is limited by the retrospective nature of the clinical and adverse effect data, and is dependent on medical records. It is also limited by the inclusion of all methylphenidate formulations considered as the total daily dose of methylphenidate. Additionally, given the considerable variation of *CES1* the sample size is relatively small to show clinically significant differences in dosing or adverse effects.

Contributions of this study include improving researchers' and clinicians' understanding of how genetic variation in the *CES1* gene primarily responsible for the metabolism of methylphenidate impacts adverse events and dose requirements in children with ADHD.

## Acknowledgments

The study team would like to acknowledge Austin Land who programmed data pulls from the electronic health record, Nathan Tesch who performed data collection, and Hillary Henzler-Buckingham who also performed data collection.

## Funding Statement

This work was supported in part by a grant from the Essentia Health Foundation (A1512).

## Data availability statement

The original contributions presented in the study are publicly available. This data can be found here: [https://www.ncbi.nlm.nih.gov/bioproject/PRJNA919062](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA919062).

## Ethics statement

Written informed consent to participate in this study was provided by the participants and their legal guardian.

## Author contributions

JTB, NB, AT, RR, and CAM designed the study. AT and TS performed the data abstraction. CH and JB performed the gene sequencing and data analysis. All authors contributed to the article and approved the submitted version.

## Conflict of interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Publisher's note

All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.

## Supplementary material

The Supplementary Material for this article can be found online at: [https://www.frontiersin.org/articles/10.3389/fped.2022.958622/full#supplementary-material](https://www.frontiersin.org/articles/10.3389/fped.2022.958622/full#supplementary-material).

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are publicly available. This data can be found here: [https://www.ncbi.nlm.nih.gov/bioproject/PRJNA919062](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA919062).

### Supplementary Materials

### Data Availability Statement

The original contributions presented in the study are publicly available. This data can be found here: [https://www.ncbi.nlm.nih.gov/bioproject/PRJNA919062](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA919062).

## References

1. Sayal K, Prasad V, Daley D, Ford T, Coghill D. ADHD In children and young people: prevalence, care pathways, and service provision. Lancet Psychiatry. (2018) 5(2):175–86. 10.1016/S2215-0366(17)30167-0  [DOI](https://doi.org/10.1016/S2215-0366(17)30167-0) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29033005/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Psychiatry&title=ADHD%20In%20children%20and%20young%20people:%20prevalence,%20care%20pathways,%20and%20service%20provision&author=K%20Sayal&author=V%20Prasad&author=D%20Daley&author=T%20Ford&author=D%20Coghill&volume=5&issue=2&publication_year=2018&pages=175-86&pmid=29033005&doi=10.1016/S2215-0366(17)30167-0&)

2. Pastor PN, Reuben CA. Diagnosed attention deficit hyperactivity disorder and learning disability: united States, 2004–2006. Vital Health Stat. (2008) 10(237):1–14.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/18998276/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Vital%20Health%20Stat&title=Diagnosed%20attention%20deficit%20hyperactivity%20disorder%20and%20learning%20disability:%20united%20States,%202004%E2%80%932006&author=PN%20Pastor&author=CA%20Reuben&volume=10&issue=237&publication_year=2008&pages=1-14&pmid=18998276&)

3. Boyle CA, Boulet S, Schieve LA, Cohen RA, Blumberg SJ, Yeargin-Allsopp M, et al. Trends in the prevalence of developmental disabilities in US children, 1997–2008. Pediatrics. (2011) 127(6):1034–42. 10.1542/peds.2010-2989  [DOI](https://doi.org/10.1542/peds.2010-2989) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21606152/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatrics&title=Trends%20in%20the%20prevalence%20of%20developmental%20disabilities%20in%20US%20children,%201997%E2%80%932008&author=CA%20Boyle&author=S%20Boulet&author=LA%20Schieve&author=RA%20Cohen&author=SJ%20Blumberg&volume=127&issue=6&publication_year=2011&pages=1034-42&pmid=21606152&doi=10.1542/peds.2010-2989&)

4. Pelham WE, Foster EM, Robb JA. The economic impact of attention-deficit/hyperactivity disorder in children and adolescents. Ambul Pediatr Off J Ambul Pediatr Assoc. (2007) 7(1 Suppl):121–31. 10.1016/j.ambp.2006.08.002  [DOI](https://doi.org/10.1016/j.ambp.2006.08.002) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17261491/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ambul%20Pediatr%20Off%20J%20Ambul%20Pediatr%20Assoc&title=The%20economic%20impact%20of%20attention-deficit/hyperactivity%20disorder%20in%20children%20and%20adolescents&author=WE%20Pelham&author=EM%20Foster&author=JA%20Robb&volume=7&issue=1%20Suppl&publication_year=2007&pages=121-31&pmid=17261491&doi=10.1016/j.ambp.2006.08.002&)

5. Polanczyk G, Rohde LA. Epidemiology of attention-deficit/hyperactivity disorder across the lifespan. Curr Opin Psychiatry. (2007) 20(4):386–92. 10.1097/YCO.0b013e3281568d7a  [DOI](https://doi.org/10.1097/YCO.0b013e3281568d7a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17551354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Psychiatry&title=Epidemiology%20of%20attention-deficit/hyperactivity%20disorder%20across%20the%20lifespan&author=G%20Polanczyk&author=LA%20Rohde&volume=20&issue=4&publication_year=2007&pages=386-92&pmid=17551354&doi=10.1097/YCO.0b013e3281568d7a&)

6. Chai G, Governale L, McMahon AW, Trinidad JP, Staffa J, Murphy D. Trends of outpatient prescription drug utilization in US children, 2002-2010. Pediatrics. (2012) 130(1):23–31. 10.1542/peds.2011-2879  [DOI](https://doi.org/10.1542/peds.2011-2879) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22711728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatrics&title=Trends%20of%20outpatient%20prescription%20drug%20utilization%20in%20US%20children,%202002-2010&author=G%20Chai&author=L%20Governale&author=AW%20McMahon&author=JP%20Trinidad&author=J%20Staffa&volume=130&issue=1&publication_year=2012&pages=23-31&pmid=22711728&doi=10.1542/peds.2011-2879&)

7. Bolea-Alamañac B, Nutt DJ, Adamou M, Asherson P, Bazire S, Coghill D, et al. Evidence-based guidelines for the pharmacological management of attention deficit hyperactivity disorder: update on recommendations from the British association for psychopharmacology. J Psychopharmacol Oxf Engl. (2014) 28(3):179–203. 10.1177/0269881113519509  [DOI](https://doi.org/10.1177/0269881113519509) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24526134/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Psychopharmacol%20Oxf%20Engl&title=Evidence-based%20guidelines%20for%20the%20pharmacological%20management%20of%20attention%20deficit%20hyperactivity%20disorder:%20update%20on%20recommendations%20from%20the%20British%20association%20for%20psychopharmacology&author=B%20Bolea-Alama%C3%B1ac&author=DJ%20Nutt&author=M%20Adamou&author=P%20Asherson&author=S%20Bazire&volume=28&issue=3&publication_year=2014&pages=179-203&pmid=24526134&doi=10.1177/0269881113519509&)

8. McGough JJ. Attention-deficit/hyperactivity disorder pharmacogenomics. Biol Psychiatry. (2005) 57(11):1367–73. 10.1016/j.biopsych.2004.10.021  [DOI](https://doi.org/10.1016/j.biopsych.2004.10.021) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15950009/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biol%20Psychiatry&title=Attention-deficit/hyperactivity%20disorder%20pharmacogenomics&author=JJ%20McGough&volume=57&issue=11&publication_year=2005&pages=1367-73&pmid=15950009&doi=10.1016/j.biopsych.2004.10.021&)

9. McGough JJ, McCracken JT, Loo SK, Manganiello M, Leung MC, Tietjens JR, et al. A candidate gene analysis of methylphenidate response in attention-deficit/hyperactivity disorder. J Am Acad Child Adolesc Psychiatry. (2009) 48(12):1155–64. 10.1097/CHI.0b013e3181bc72e3  [DOI](https://doi.org/10.1097/CHI.0b013e3181bc72e3) | [PMC free article](/articles/PMC2888980/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19858760/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Am%20Acad%20Child%20Adolesc%20Psychiatry&title=A%20candidate%20gene%20analysis%20of%20methylphenidate%20response%20in%20attention-deficit/hyperactivity%20disorder&author=JJ%20McGough&author=JT%20McCracken&author=SK%20Loo&author=M%20Manganiello&author=MC%20Leung&volume=48&issue=12&publication_year=2009&pages=1155-64&pmid=19858760&doi=10.1097/CHI.0b013e3181bc72e3&)

10. Kambeitz J, Romanos M, Ettinger U. Meta-analysis of the association between dopamine transporter genotype and response to methylphenidate treatment in ADHD. Pharmacogenomics J. (2014) 14(1):77–84. 10.1038/tpj.2013.9  [DOI](https://doi.org/10.1038/tpj.2013.9) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23588108/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Meta-analysis%20of%20the%20association%20between%20dopamine%20transporter%20genotype%20and%20response%20to%20methylphenidate%20treatment%20in%20ADHD&author=J%20Kambeitz&author=M%20Romanos&author=U%20Ettinger&volume=14&issue=1&publication_year=2014&pages=77-84&pmid=23588108&doi=10.1038/tpj.2013.9&)

11. Stevens T, Sangkuhl K, Brown JT, Altman RB, Klein TE. PharmGKB summary: methylphenidate pathway, pharmacokinetics/pharmacodynamics. Pharmacogenet Genomics. (2019) 29(6):136–54. 10.1097/FPC.0000000000000376  [DOI](https://doi.org/10.1097/FPC.0000000000000376) | [PMC free article](/articles/PMC6581573/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30950912/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=PharmGKB%20summary:%20methylphenidate%20pathway,%20pharmacokinetics/pharmacodynamics&author=T%20Stevens&author=K%20Sangkuhl&author=JT%20Brown&author=RB%20Altman&author=TE%20Klein&volume=29&issue=6&publication_year=2019&pages=136-54&pmid=30950912&doi=10.1097/FPC.0000000000000376&)

12. Sun Z, Murry DJ, Sanghani SP, Davis WI, Kedishvili NY, Zou Q, et al. Methylphenidate is stereoselectively hydrolyzed by human carboxylesterase CES1A1. J Pharmacol Exp Ther. (2004) 310(2):469–76. 10.1124/jpet.104.067116  [DOI](https://doi.org/10.1124/jpet.104.067116) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15082749/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pharmacol%20Exp%20Ther&title=Methylphenidate%20is%20stereoselectively%20hydrolyzed%20by%20human%20carboxylesterase%20CES1A1&author=Z%20Sun&author=DJ%20Murry&author=SP%20Sanghani&author=WI%20Davis&author=NY%20Kedishvili&volume=310&issue=2&publication_year=2004&pages=469-76&pmid=15082749&doi=10.1124/jpet.104.067116&)

13. Zhu HJ, Patrick KS, Yuan HJ, Wang JS, Donovan JL, DeVane CL, et al. Two CES1 gene mutations lead to dysfunctional carboxylesterase 1 activity in man: clinical significance and molecular basis. Am J Hum Genet. (2008) 82(6):1241–8. 10.1016/j.ajhg.2008.04.015  [DOI](https://doi.org/10.1016/j.ajhg.2008.04.015) | [PMC free article](/articles/PMC2427248/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18485328/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am%20J%20Hum%20Genet&title=Two%20CES1%20gene%20mutations%20lead%20to%20dysfunctional%20carboxylesterase%201%20activity%20in%20man:%20clinical%20significance%20and%20molecular%20basis&author=HJ%20Zhu&author=KS%20Patrick&author=HJ%20Yuan&author=JS%20Wang&author=JL%20Donovan&volume=82&issue=6&publication_year=2008&pages=1241-8&pmid=18485328&doi=10.1016/j.ajhg.2008.04.015&)

14. Stage C, Jürgens G, Guski LS, Thomsen R, Bjerre D, Ferrero-Miliani L, et al. The impact of CES1 genotypes on the pharmacokinetics of methylphenidate in healthy danish subjects. Br J Clin Pharmacol. (2017) 83(7):1506–14. 10.1111/bcp.13237  [DOI](https://doi.org/10.1111/bcp.13237) | [PMC free article](/articles/PMC5465325/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28087982/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=The%20impact%20of%20CES1%20genotypes%20on%20the%20pharmacokinetics%20of%20methylphenidate%20in%20healthy%20danish%20subjects&author=C%20Stage&author=G%20J%C3%BCrgens&author=LS%20Guski&author=R%20Thomsen&author=D%20Bjerre&volume=83&issue=7&publication_year=2017&pages=1506-14&pmid=28087982&doi=10.1111/bcp.13237&)

15. Tarkiainen EK, Backman JT, Neuvonen M, Neuvonen PJ, Schwab M, Niemi M. Carboxylesterase 1 polymorphism impairs oseltamivir bioactivation in humans. Clin Pharmacol Ther. (2012) 92(1):68–71. 10.1038/clpt.2012.13  [DOI](https://doi.org/10.1038/clpt.2012.13) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22588607/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Carboxylesterase%201%20polymorphism%20impairs%20oseltamivir%20bioactivation%20in%20humans&author=EK%20Tarkiainen&author=JT%20Backman&author=M%20Neuvonen&author=PJ%20Neuvonen&author=M%20Schwab&volume=92&issue=1&publication_year=2012&pages=68-71&pmid=22588607&doi=10.1038/clpt.2012.13&)

16. Harris PA, Taylor R, Thielke R, Payne J, Gonzalez N, Conde JG. Research electronic data capture (REDCap)–a metadata-driven methodology and workflow process for providing translational research informatics support. J Biomed Inform. (2009) 42(2):377–81. 10.1016/j.jbi.2008.08.010  [DOI](https://doi.org/10.1016/j.jbi.2008.08.010) | [PMC free article](/articles/PMC2700030/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18929686/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biomed%20Inform&title=Research%20electronic%20data%20capture%20(REDCap)%E2%80%93a%20metadata-driven%20methodology%20and%20workflow%20process%20for%20providing%20translational%20research%20informatics%20support&author=PA%20Harris&author=R%20Taylor&author=R%20Thielke&author=J%20Payne&author=N%20Gonzalez&volume=42&issue=2&publication_year=2009&pages=377-81&pmid=18929686&doi=10.1016/j.jbi.2008.08.010&)

17. Harris PA, Taylor R, Minor BL, Elliott V, Fernandez M, O’Neal L, et al. The REDCap consortium: building an international community of software platform partners. J Biomed Inform. (2019) 95:103208. 10.1016/j.jbi.2019.103208  [DOI](https://doi.org/10.1016/j.jbi.2019.103208) | [PMC free article](/articles/PMC7254481/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31078660/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biomed%20Inform&title=The%20REDCap%20consortium:%20building%20an%20international%20community%20of%20software%20platform%20partners&author=PA%20Harris&author=R%20Taylor&author=BL%20Minor&author=V%20Elliott&author=M%20Fernandez&volume=95&publication_year=2019&pages=103208&pmid=31078660&doi=10.1016/j.jbi.2019.103208&)

18. Gohl D, Vangay P, Garbe J, MacLean A, Hauge A, Becker A, et al. Systematic improvement of amplicon marker gene methods for increased accuracy in microbiome studies. Nat Biotechnol. (2016) 34:942–9. 10.1038/nbt.3601  [DOI](https://doi.org/10.1038/nbt.3601) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27454739/) | [Google Scholar](https://scholar.google.com/scholar_lookup?Gohl%20D,%20Vangay%20P,%20Garbe%20J,%20MacLean%20A,%20Hauge%20A,%20Becker%20A,%20et%20al.%20Systematic%20improvement%20of%20amplicon%20marker%20gene%20methods%20for%20increased%20accuracy%20in%20microbiome%20studies.%20Nat%20Biotechnol.%20(2016)%2034:942%E2%80%939.%2010.1038/nbt.3601)

19. Li H. Minimap2: pairwise alignment for nucleotide sequences. Bioinforma Oxf Engl. (2018) 34(18):3094–100. 10.1093/bioinformatics/bty191  [DOI](https://doi.org/10.1093/bioinformatics/bty191) | [PMC free article](/articles/PMC6137996/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29750242/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinforma%20Oxf%20Engl&title=Minimap2:%20pairwise%20alignment%20for%20nucleotide%20sequences&author=H%20Li&volume=34&issue=18&publication_year=2018&pages=3094-100&pmid=29750242&doi=10.1093/bioinformatics/bty191&)

20. Quinlan AR, Hall IM. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinforma Oxf Engl. (2010) 26(6):841–2. 10.1093/bioinformatics/btq033  [DOI](https://doi.org/10.1093/bioinformatics/btq033) | [PMC free article](/articles/PMC2832824/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20110278/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinforma%20Oxf%20Engl&title=BEDTools:%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&author=AR%20Quinlan&author=IM%20Hall&volume=26&issue=6&publication_year=2010&pages=841-2&pmid=20110278&doi=10.1093/bioinformatics/btq033&)

21. Garrison E, Marth G. Haplotype-based variant detection from short-read sequencing. arXiv; 2012 [cited 2022 May 31]. Available at: http://arxiv.org/abs/1207.3907. 10.48550/arXiv.1207.3907  [http://arxiv.org/abs/1207.3907](http://arxiv.org/abs/1207.3907) | [DOI](https://doi.org/10.48550/arXiv.1207.3907)

22. Danecek P, Bonfield JK, Liddle J, Marshall J, Ohan V, Pollard MO, et al. Twelve years of SAMtools and BCFtools. GigaScience. (2021) 10(2):giab008. 10.1093/gigascience/giab008  [DOI](https://doi.org/10.1093/gigascience/giab008) | [PMC free article](/articles/PMC7931819/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33590861/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=GigaScience&title=Twelve%20years%20of%20SAMtools%20and%20BCFtools&author=P%20Danecek&author=JK%20Bonfield&author=J%20Liddle&author=J%20Marshall&author=V%20Ohan&volume=10&issue=2&publication_year=2021&pages=giab008&pmid=33590861&doi=10.1093/gigascience/giab008&)

23. Martin M, Patterson M, Garg S, Fischer S O, Pisanti N, Klau GW, et al. Whatshap: fast and accurate read-based phasing. Bioinformatics. (2016). [cited 2022 May 31]. Available at: http://biorxiv.org/lookup/doi/10.1101/085050  [http://biorxiv.org/lookup/doi/10.1101/085050](http://biorxiv.org/lookup/doi/10.1101/085050) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Whatshap:%20fast%20and%20accurate%20read-based%20phasing&author=M%20Martin&author=M%20Patterson&author=S%20Garg&author=O%20Fischer%20S&author=N%20Pisanti&publication_year=2016&)

24. Stage C, Dalhoff K, Rasmussen HB, Schow Guski L, Thomsen R, Bjerre D, et al. The impact of human CES1 genetic variation on enzyme activity assessed by ritalinic acid/methylphenidate ratios. Basic Clin Pharmacol Toxicol. (2019) 125(1):54–61. 10.1111/bcpt.13212  [DOI](https://doi.org/10.1111/bcpt.13212) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30801959/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Basic%20Clin%20Pharmacol%20Toxicol&title=The%20impact%20of%20human%20CES1%20genetic%20variation%20on%20enzyme%20activity%20assessed%20by%20ritalinic%20acid/methylphenidate%20ratios&author=C%20Stage&author=K%20Dalhoff&author=HB%20Rasmussen&author=L%20Schow%20Guski&author=R%20Thomsen&volume=125&issue=1&publication_year=2019&pages=54-61&pmid=30801959&doi=10.1111/bcpt.13212&)

25. Nemoda Z, Angyal N, Tarnok Z, Gadoros J, Sasvari-Szekely M. Carboxylesterase 1 gene polymorphism and methylphenidate response in ADHD. Neuropharmacology. (2009) 57(7–8):731–3. 10.1016/j.neuropharm.2009.08.014  [DOI](https://doi.org/10.1016/j.neuropharm.2009.08.014) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19733552/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Neuropharmacology&title=Carboxylesterase%201%20gene%20polymorphism%20and%20methylphenidate%20response%20in%20ADHD&author=Z%20Nemoda&author=N%20Angyal&author=Z%20Tarnok&author=J%20Gadoros&author=M%20Sasvari-Szekely&volume=57&issue=7%E2%80%938&publication_year=2009&pages=731-3&pmid=19733552&doi=10.1016/j.neuropharm.2009.08.014&)

26. Johnson KA, Barry E, Lambert D, Fitzgerald M, McNicholas F, Kirley A, et al. Methylphenidate side effect profile is influenced by genetic variation in the attention-deficit/hyperactivity disorder-associated CES1 gene. J Child Adolesc Psychopharmacol. (2013) 23(10):655–64. 10.1089/cap.2013.0032  [DOI](https://doi.org/10.1089/cap.2013.0032) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24350812/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Child%20Adolesc%20Psychopharmacol&title=Methylphenidate%20side%20effect%20profile%20is%20influenced%20by%20genetic%20variation%20in%20the%20attention-deficit/hyperactivity%20disorder-associated%20CES1%20gene&author=KA%20Johnson&author=E%20Barry&author=D%20Lambert&author=M%20Fitzgerald&author=F%20McNicholas&volume=23&issue=10&publication_year=2013&pages=655-64&pmid=24350812&doi=10.1089/cap.2013.0032&)

27. Paré G, Eriksson N, Lehr T, Connolly S, Eikelboom J, Ezekowitz MD, et al. Genetic determinants of dabigatran plasma levels and their relation to bleeding. Circulation. (2013) 127(13):1404–12. 10.1161/CIRCULATIONAHA.112.001233  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.112.001233) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23467860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Genetic%20determinants%20of%20dabigatran%20plasma%20levels%20and%20their%20relation%20to%20bleeding&author=G%20Par%C3%A9&author=N%20Eriksson&author=T%20Lehr&author=S%20Connolly&author=J%20Eikelboom&volume=127&issue=13&publication_year=2013&pages=1404-12&pmid=23467860&doi=10.1161/CIRCULATIONAHA.112.001233&)

28. Bruxel EM, Salatino-Oliveira A, Genro JP, Zeni CP, Polanczyk GV, Chazan R, et al. Association of a carboxylesterase 1 polymorphism with appetite reduction in children and adolescents with attention-deficit/hyperactivity disorder treated with methylphenidate. Pharmacogenomics J. (2013) 13(5):476–80. 10.1038/tpj.2012.25  [DOI](https://doi.org/10.1038/tpj.2012.25) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22688218/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Association%20of%20a%20carboxylesterase%201%20polymorphism%20with%20appetite%20reduction%20in%20children%20and%20adolescents%20with%20attention-deficit/hyperactivity%20disorder%20treated%20with%20methylphenidate&author=EM%20Bruxel&author=A%20Salatino-Oliveira&author=JP%20Genro&author=CP%20Zeni&author=GV%20Polanczyk&volume=13&issue=5&publication_year=2013&pages=476-80&pmid=22688218&doi=10.1038/tpj.2012.25&)

29. Matsumoto N, Kubota Y, Ishida H, Sekido M, Ohkuma R, Ishiguro T, et al. Variants of carboxylesterase 1 have no impact on capecitabine pharmacokinetics and toxicity in capecitabine plus oxaliplatin treated-colorectal cancer patients. Cancer Chemother Pharmacol. (2020) 85(6):1119–28. 10.1007/s00280-020-04087-z  [DOI](https://doi.org/10.1007/s00280-020-04087-z) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32458030/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemother%20Pharmacol&title=Variants%20of%20carboxylesterase%201%20have%20no%20impact%20on%20capecitabine%20pharmacokinetics%20and%20toxicity%20in%20capecitabine%20plus%20oxaliplatin%20treated-colorectal%20cancer%20patients&author=N%20Matsumoto&author=Y%20Kubota&author=H%20Ishida&author=M%20Sekido&author=R%20Ohkuma&volume=85&issue=6&publication_year=2020&pages=1119-28&pmid=32458030&doi=10.1007/s00280-020-04087-z&)

30. Nelveg-Kristensen KE, Bie P, Ferrero L, Bjerre D, Bruun NE, Egfjord M, et al. Pharmacodynamic impact of carboxylesterase 1 gene variants in patients with congestive heart failure treated with angiotensin-converting enzyme inhibitors. PloS One. (2016) 11(9):e0163341. 10.1371/journal.pone.0163341  [DOI](https://doi.org/10.1371/journal.pone.0163341) | [PMC free article](/articles/PMC5035013/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27662362/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PloS%20One&title=Pharmacodynamic%20impact%20of%20carboxylesterase%201%20gene%20variants%20in%20patients%20with%20congestive%20heart%20failure%20treated%20with%20angiotensin-converting%20enzyme%20inhibitors&author=KE%20Nelveg-Kristensen&author=P%20Bie&author=L%20Ferrero&author=D%20Bjerre&author=NE%20Bruun&volume=11&issue=9&publication_year=2016&pages=e0163341&pmid=27662362&doi=10.1371/journal.pone.0163341&)

31. Xue-qiong W, Dong-lin Z, Jun-xian Z, Zhong Y, Xi Y, Hui-ru A, et al. The relationship between carboxylesterase 1 gene polymorphisms and susceptibility to antituberculosis drug-induced hepatotoxicity. Zhonghua Nei Ke Za Zhi. (2012) 51(7):524–30.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/22943824/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Zhonghua%20Nei%20Ke%20Za%20Zhi&title=The%20relationship%20between%20carboxylesterase%201%20gene%20polymorphisms%20and%20susceptibility%20to%20antituberculosis%20drug-induced%20hepatotoxicity&author=W%20Xue-qiong&author=Z%20Dong-lin&author=Z%20Jun-xian&author=Y%20Zhong&author=Y%20Xi&volume=51&issue=7&publication_year=2012&pages=524-30&pmid=22943824&)

32. Teft WA, Welch S, Lenehan J, Parfitt J, Choi YH, Winquist E, et al. OATP1B1 And tumour OATP1B3 modulate exposure, toxicity, and survival after irinotecan-based chemotherapy. Br J Cancer. (2015) 112(5):857–65. 10.1038/bjc.2015.5  [DOI](https://doi.org/10.1038/bjc.2015.5) | [PMC free article](/articles/PMC4453959/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25611302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=OATP1B1%20And%20tumour%20OATP1B3%20modulate%20exposure,%20toxicity,%20and%20survival%20after%20irinotecan-based%20chemotherapy&author=WA%20Teft&author=S%20Welch&author=J%20Lenehan&author=J%20Parfitt&author=YH%20Choi&volume=112&issue=5&publication_year=2015&pages=857-65&pmid=25611302&doi=10.1038/bjc.2015.5&)

33. Ikonnikova A, Rodina T, Dmitriev A, Melnikov E, Kazakov R, Nasedkina T. The influence of the CES1 genotype on the pharmacokinetics of enalapril in patients with arterial hypertension. J Pers Med. (2022) 12(4):580. 10.3390/jpm12040580  [DOI](https://doi.org/10.3390/jpm12040580) | [PMC free article](/articles/PMC9028383/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35455696/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pers%20Med&title=The%20influence%20of%20the%20CES1%20genotype%20on%20the%20pharmacokinetics%20of%20enalapril%20in%20patients%20with%20arterial%20hypertension&author=A%20Ikonnikova&author=T%20Rodina&author=A%20Dmitriev&author=E%20Melnikov&author=R%20Kazakov&volume=12&issue=4&publication_year=2022&pages=580&pmid=35455696&doi=10.3390/jpm12040580&)

34. Qian Y, Markowitz JS. Natural products as modulators of CES1 activity. Drug Metab Dispos Biol Fate Chem. (2020) 48(10):993–1007. 10.1124/dmd.120.000065  [DOI](https://doi.org/10.1124/dmd.120.000065) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32591414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos%20Biol%20Fate%20Chem&title=Natural%20products%20as%20modulators%20of%20CES1%20activity&author=Y%20Qian&author=JS%20Markowitz&volume=48&issue=10&publication_year=2020&pages=993-1007&pmid=32591414&doi=10.1124/dmd.120.000065&)

35. Her L, Zhu HJ. Carboxylesterase 1 and precision pharmacotherapy: pharmacogenetics and nongenetic regulators. Drug Metab Dispos Biol Fate Chem. (2020) 48(3):230–44. 10.1124/dmd.119.089680  [DOI](https://doi.org/10.1124/dmd.119.089680) | [PMC free article](/articles/PMC7031766/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31871135/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos%20Biol%20Fate%20Chem&title=Carboxylesterase%201%20and%20precision%20pharmacotherapy:%20pharmacogenetics%20and%20nongenetic%20regulators&author=L%20Her&author=HJ%20Zhu&volume=48&issue=3&publication_year=2020&pages=230-44&pmid=31871135&doi=10.1124/dmd.119.089680&)

36. Zhu HJ, Appel DI, Peterson YK, Wang Z, Markowitz JS. Identification of selected therapeutic agents as inhibitors of carboxylesterase 1: potential sources of metabolic drug interactions. Toxicology. (2010) 270(2–3):59–65. 10.1016/j.tox.2010.01.009  [DOI](https://doi.org/10.1016/j.tox.2010.01.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20097249/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Toxicology&title=Identification%20of%20selected%20therapeutic%20agents%20as%20inhibitors%20of%20carboxylesterase%201:%20potential%20sources%20of%20metabolic%20drug%20interactions&author=HJ%20Zhu&author=DI%20Appel&author=YK%20Peterson&author=Z%20Wang&author=JS%20Markowitz&volume=270&issue=2%E2%80%933&publication_year=2010&pages=59-65&pmid=20097249&doi=10.1016/j.tox.2010.01.009&)

37. Elsayed NA, Yamamoto KM, Froehlich TE. Genetic influence on efficacy of pharmacotherapy for pediatric attention-deficit/hyperactivity disorder: overview and current Status of research. CNS Drugs. (2020) 34(4):389–414. 10.1007/s40263-020-00702-y  [DOI](https://doi.org/10.1007/s40263-020-00702-y) | [PMC free article](/articles/PMC8083895/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32133580/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CNS%20Drugs&title=Genetic%20influence%20on%20efficacy%20of%20pharmacotherapy%20for%20pediatric%20attention-deficit/hyperactivity%20disorder:%20overview%20and%20current%20Status%20of%20research&author=NA%20Elsayed&author=KM%20Yamamoto&author=TE%20Froehlich&volume=34&issue=4&publication_year=2020&pages=389-414&pmid=32133580&doi=10.1007/s40263-020-00702-y&)

38. Joensen B, Meyer M, Aagaard L. Specific genes associated with adverse events of methylphenidate use in the pediatric population: a systematic literature review. J Res Pharm Pract. (2017) 6(2):65–72. 10.4103/jrpp.JRPP_16_161  [DOI](https://doi.org/10.4103/jrpp.JRPP_16_161) | [PMC free article](/articles/PMC5463551/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28616427/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Res%20Pharm%20Pract&title=Specific%20genes%20associated%20with%20adverse%20events%20of%20methylphenidate%20use%20in%20the%20pediatric%20population:%20a%20systematic%20literature%20review&author=B%20Joensen&author=M%20Meyer&author=L%20Aagaard&volume=6&issue=2&publication_year=2017&pages=65-72&pmid=28616427&doi=10.4103/jrpp.JRPP_16_161&)
