# Identification and drug metabolic characterization of four new CYP2C9 variants CYP2C9*72-*75 in the Chinese Han population

## Metadata
**Authors:** Fang-Ling Zhao, Qing Zhang, Shuang-Hu Wang, Yun Hong, Shan Zhou, Quan Zhou, Pei-Wu Geng, Qing-Feng Luo, Jie-Fu Yang, Hao Chen, Jian-Ping Cai, Da-Peng Dai
**Journal:** Frontiers in Pharmacology
**Date:** 2022 Dec 13
**DOI:** [10.3389/fphar.2022.1007268](https://doi.org/10.3389/fphar.2022.1007268)
**PMID:** 36582532
**PMCID:** PMC9792615
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9792615/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9792615/pdf/fphar-13-1007268.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9792615/pdf/fphar-13-1007268.pdf)

## Abstract

Cytochrome 2C9 (CYP2C9), one of the most important drug metabolic enzymes in the human hepatic P450 superfamily, is required for the metabolism of 15% of clinical drugs. Similar to other CYP2C family members, CYP2C9 gene has a high genetic polymorphism which can cause significant racial and inter-individual differences in drug metabolic activity. To better understand the genetic distribution pattern of CYP2C9 in the Chinese Han population, 931 individuals were recruited and used for the genotyping in this study. As a result, seven synonymous and 14 non-synonymous variations were identified, of which 4 missense variants were designated as new alleles CYP2C9*72, *73, *74 and *75, resulting in the amino acid substitutions of A149V, R150C, Q214H and N418T, respectively. When expressed in insect cell microsomes, all four variants exhibited comparable protein expression levels to that of the wild-type CYP2C9 enzyme. However, drug metabolic activity analysis revealed that these variants exhibited significantly decreased catalytic activities toward three CYP2C9 specific probe drugs, as compared with that of the wild-type enzyme. These data indicate that the amino acid substitution in newly designated variants can cause reduced function of the enzyme and its clinical significance still needs further investigation in the future.

Keywords: CYP2C9, drug metabolism, allelic variant, baculovirus, microsome

## 1 Introduction

Drug administration is one of the main modalities for disease treatment in clinical trials, and the drug will experience the journey of absorption, distribution, metabolism, and excretion (ADME) in the human body. Of these processes, metabolism plays an important role in influencing drug effect, due to its contribution to the main elimination pathway of most drugs. Drug metabolism refers to the process of drug chemical structural modification *in vivo*, that directly determines the drug metabolic activity and pharmacological effects. Primarily occurring in the liver, drug metabolism is dependent on multiple hepatic microsomal enzymes, ([Almazroo et al., 2017](#B3); [Zhao et al., 2021](#B44)), and cytochrome P450 mixed function oxidase (abbreviated as CYP450 or CYP) is the most important enzyme member because it is responsible for catalyzing 80% of clinically used drugs. ([Guengerich, 2006](#B17); [Subramanian et al., 2012](#B36)) CYP450 enzymes, simultaneously known as monooxygenases, are a group of structurally and functionally related isozymes encoded by hemoglobin superfamily CYP genes. ([Miners and Birkett, 1998](#B27)) The most striking features of these CYP genes are their marked polymorphisms in different ethnic groups and geographical regions, and these genetic polymorphisms can cause a high variation in the metabolic activity of the P450 enzyme among different individuals. ([Waring, 2020](#B39)) To date, several public websites or organizations have been focused on the pharmacogenetics (PGx) related to drug metabolism, of which CPIC ([Relling et al., 2020](#B32)) and PharmVar ([Sangkuhl et al., 2021](#B34)) are the most important pharmacogenetic groups. Previous studies have reported that the high polymorphic status of the CYP gene is one of the crucial factors that make a great contribution to the individual differences in drug response. ([Nebert and Russell, 2002](#B28); [Fujikura et al., 2015](#B15)) Therefore, investigating polymorphism in CYP genotypes will greatly reduce the occurrence of adverse drug reactions and help clinicians and pharmacists precisely apply diverse drugs. ([Manikandan and Nagini, 2018](#B25); [Qian et al., 2021](#B31))

It has been reported that the human genome includes at least 57 CYP genes, that can be divided into 18 families and 43 subfamilies based on the homologous degree of protein amino acid sequences. ([Zanger and Schwab, 2013](#B42)) In particular, the majority of these drugs are actually metabolized by five isoforms, CYP2C9, CYP1A2, CYP2C19, CYP2D6 and CYP3A4. ([Flanagan et al., 2003](#B14)) Among these five important pharmacogenetic enzymes, CYP2C9 has received extensive attention because of its marked genetic polymorphism, high expression, and large portion in drug metabolism. The *CYP2C9* gene is located on chromosome 10q24.2 with a total length of approximately 55 kb, including nine exons and eight introns. The CYP2C9 enzyme metabolizes up to 15% of clinical drugs and accounts for 20% of total cytochrome P450 protein in human liver microsomes. ([Zhou et al., 2009](#B45)), ([Tornio and Backman, 2018](#B37)) To date, more than 80 *CYP2C9* allelic variants have been identified and illustrated in the website of the Pharmacogene Variation (PharmVar) Consortium ([https://www.pharmvar.org/gene/CYP2C9](https://www.pharmvar.org/gene/CYP2C9)), in which most of them are SNPs that are located in the coding region ([Gaedigk et al., 2021](#B16); [Sangkuhl et al., 2021](#B34)). Similar to other CYP members, *CYP2C9* gene exhibits large ethnic and individual differences in genotype and allele frequency. For example, *CYP2C9*2* is the most prevalent defective allele in European with a frequency of approximately 12.7%, but it is rarely detected in the East Asian individuals (0.21%). Therefore, identifying polymorphisms across ethnic groups is crucial to understand the difference in clinical response to drugs in these populations. ([DeLozier et al., 2005](#B13); [Nizamuddin et al., 2021](#B30)) Recently, we performed a polymorphic screening of *CYP2C9* gene in 2,127 Chinese Han individuals and found 35 allelic variants, and 21 of them were reported for the first time. ([Dai et al., 2014](#B12)) After that, we also identified another four defective alleles in warfarin sensitive patients. ([Luo et al., 2014](#B21); [Dai et al., 2015a](#B9); [Dai et al., 2015b](#B10); [Chen et al., 2020](#B7)) These studies indicate that many other undetected *CYP2C9* allelic variants may still exist in the Chinese Han population. In this study, we performed another set of genetic screenings of the *CYP2C9* gene in Chinese individuals and detected four allelic variants, *CYP2C9*72*, **73*, **74* and **75*.

## 2 Materials and methods

### 2.1 Chemicals and materials

Materials and kits were purchased from the following sources: TIANamp Blood DNA Midi Kit (TIANGEN, Beijing, China); PrimeSTAR HS DNA polymerase and restriction enzymes (Takara Bio, Inc.; Otsu, Shiga, Japan); Spodoptera frugiperda (Sf)21 insect cells, Sf-900TM III SFM insect culture medium and Bac to-Bac Baculovirus Expression System (Invitrogen, Carlsbad, CA, United States); Mouse monoclonal anti-OR antibody (Santa Cruz Biotechnology, Dallas, Texas, United States); Rabbit polyclonal anti-CYP2C9 antibody (Abcam, Cambridge, United Kingdom); Super Signal West Pico Trial Kit (Thermo Scientific, Rockford, IL, United States); 4-Hydroxytolbutamide, 4-hydroxydiclofenac, and telmisartan (Toronto Research Chemicals, Inc.; Toronto, Ontario, Canada); Diclofenac and chlorpropamide (Tokyo Chemical Industry Co., Ltd.; Tokyo, Japan); Tolbutamide, losartan and E-3174 (Sigma-Aldrich, St. Louis, MO, United States); NADPH-regenerating system (Promega, Madison, WI, United States); High-pressure liquid chromatography-grade solvents (Fisher Scientific Co.; Fair Lawn, NJ, United States). All other chemicals and reagents were of analytical grade or the highest commercially available quality.

### 2.2 DNA extraction and genotyping

Subjects in this study (*n* = 931, 61.5% male, mean age 69) were recruited from the Physical Examination Center of Beijing Hospital and written informed consent forms were obtained from all healthy Chinese Han participants before blood sample collection. Individuals with acute or chronic diseases were exclusive and subjects with excessive drinking and smoking history were also excluded. This study was approved by the Institutional Ethics Committee of Beijing Hospital institutions. According to previously reported methods ([Luo et al., 2014](#B21)), genomic DNA was extracted from white blood cells using the TIANamp Blood DNA Midi Kit, and was subsequently diluted to approximately 25 ng/μL as the template for PCR amplification of the promoter or all nine exons of *CYP2C9* genes. Briefly, the experiment was carried out in a total volume of 30 μL containing 100 ng genomic DNA, 15 μL 2×Rapid Taq Master Mix (Vazyme Biotech Co. Ltd.; Nanjing, China), and 0.5 μM of each primer ([Table 1](#T1)). The reaction condition included an initial denaturation at 95°C for 3 min, followed by 35 cycles of 95°C for 20 s, 55°C–61°C for 15 s, and 72°C for 1–3 min, and a final extension at 72°C for 5 min. After separation by agarose gel electrophoresis, the PCR products were then sent to BioMed Biotech Company for sequencing on ABI 3730XL using the sequencing primers listed in [Table 1](#T1). The acquired sequences were then aligned and compared with the reference sequence in the NCBI database with the Seqman module of DNAstar software (Version 7.1). Detected mutated sites were then compared with deposited sequences of *CYP2C9* alleles on the Nomenclature Committee website ([https://www.pharmvar.org/gene/CYP2C9](https://www.pharmvar.org/gene/CYP2C9)) for genotyping. For exons containing mutated sites not included in the PharmVar website, bidirectional sequencing was then performed for DNA sequence verification. Additionally, 2,232 bp upstream of start codon ATG was also amplified with primer pairs (5′-ACT​GAG​GCA​TTG​TGA​TTG​TGA​T-3′ and 5′- GCA​AGC​CAC​TGA​AGG​AGC​AT-3′) and sequenced with three primers (5′-AAG​GGA​AAC​AGC​ACC​AG-3′, 5′-GAG​CCT​TGA​AGA​TTC​AGT​A-3′, and 5′-GAC​TTT​GAC​CCA​CTG​ATA​CA-3′) to verify whether some other mutated sites could be detected. All the sequencing files were uploaded to PharmVar for evaluation in the new allele application process.

### TABLE 1.

| PCR primers |  |  |  |  |
| --- | --- | --- | --- | --- |
| Region | Forward primer (5′-3′) | Reverse primer (5′-3′) | Amplicon (bp) | Annealing temperature (°C) |
| Promoter + Exon1 | AGA​AGC​CCT​AGT​TTC​TCA​AAC​CCT​T | TCT​ACT​CAC​AAA​ATA​CAT​GGT​TTC​A | 1,419 | 55 |
| Exon2+3 | GCA​TCA​GTG​TTT​GAA​TAA​GCG​GA | CCC​GCT​TCA​CAT​GAG​CTA​AC | 1,297 | 55 |
| Exon4+5 | CCA​GCT​AGG​TTG​TAA​TGG​TCA​ACT | TCA​CAA​GCA​GTC​ACA​TAA​CTA​AGC | 1738 | 55 |
| Exon6 | TGG​GCA​AGT​TGG​TCT​ACA​GC | ACA​TGC​AAT​CCC​AGG​CCA​AT | 938 | 61 |
| Exon7 | TGT​GCC​ATT​TTT​CTC​CTT​TTC​CAT​C | TCC​TAA​ACA​ATA​TGA​AGA​AGG​CCA​G | 1,688 | 61 |
| Exon8 | GATTGCAGGGCACTTTA | AGGAGGAGTTCTTGGGT | 639 | 55 |
| Exon9 | ACA​CTG​AAC​AGT​TAT​TGC​ATA​TTC​T | TGT​CCA​TTC​CAC​CCT​TTG​ACT | 907 | 55 |
| Sequencing primers |  |  |  |  |
| Region | Sequencing primer (5′-3′) |   |   |   |
| Promoter | AGA​AGC​CCT​AGT​TTC​TCA​AAC​CCT​T |  |  |  |
| Exon1 | AGG​CTC​CAA​CCA​AGT​ACA​GTG​AAA |  |  |  |
| Exon2+3 | TAT​TTG​AAG​CCT​GTG​TGG​CTG​AA |  |  |  |
| Exon4 | TATGAGCACGCTTTAGGG |  |  |  |
| Exon5 | TGA​TTA​TCA​TCT​GGT​TAG​AAT​TGA​T |  |  |  |
| Exon6 | AAT​CAC​CAT​TAG​TTT​GAA​ACA​GAT​TAC​AGC |  |  |  |
| Exon7 | CCT​AAG​AGT​AGC​CAA​ACC​AAT |  |  |  |
| Exon8 | GATTGCAGGGCACTTTA |  |  |  |
| Exon9 | TCT​GTC​CTT​ATC​ATT​TTG​AGA​ACC​AGC​AT |  |  |  |

Table 1 Caption: Optimized primers for amplification or sequencing of human CYP2C9 gene.

### 2.3 Construction of dual expression vector and expression of recombinant CYP2C9 proteins in insect cells

Using previously reported methods, ([Dai et al., 2013](#B11); [Luo et al., 2014](#B21)), cDNA of the typical defective variant *CYP2C9*3* was constructed by the overlap extension PCR amplification method and used as the defective control in the functional analysis experiment. In detail, four pairs of sit-directed mutagenesis primers (5′-AGA​GGA​AG**t**CCG​CTG​CC-3’ (sense) and 5′-GGC​AGC​GG**a**CTT​CCT​CT-3’ (antisense) for **72*; 5′-AGG​AAG​CC**t**GCT​GCC​TT-3’ (sense) and 5′-AAG​GCA​GC**a**GGC​TTC​CT-3’ (antisense) for **73*; 5′-CCT​GGA​TCC​A**t**ATC​TGC​AAT​A-3’ (sense) and 5′-TAT​TGC​AGA​T**a**TGG​ATC​CAG​G-3’ (antisense) for **74*; 5′-GAA​GGT​GGC​A**c**TTT​TAA​GAA​A-3’ (sense) and 5′-TTT​CTT​AAA​AgT​GCC​ACC​TTC-3’ (antisense) for **75*. Mutated sites are illustrated as lowercase bold letters) were paired with full-length amplification primers (Forward: 5′-GCC​T*GAA​TTC*ATG​GAT​TCT​CTT​GTG​GT-3′, introducing one EcoR I site; Reverse: 5′-GAA​C*GTC​GAC*TCA​GAC​AGG​AAT​GAA​GCA-3′, introducing one Sal I site) to obtain the cDNAs of four newly detected variants. Subsequently, purified PCR products were digested with EcoRI and SalI at 37°C for 2 h. After purification, amplicons were ligated to the EcoRI/SalI double-digested pFastBac dual-OR vector, to obtain the recombinant plasmid pFastBac dual-OR-2C9. Using the Bac-to-Bac Baculovirus Expression System, these newly constructed pFastBac dual-OR-2C9 vectors were then packaged into baculoviruses for the expression of OR and 2C9 enzymes simultaneously in insect cell microsomes. Expressed CYP2C9 and OR proteins were then verified and quantified according to the methods we previously reported. ([Dai et al., 2015b](#B10); [Chen et al., 2020](#B7)) Briefly, 0.1–0.2 pmol microsomes were separated on SDS-PAGE gels and detected using rabbit polyclonal anti-CYP2C9 antibody (AbD Serotec, Oxford, United Kingdom) or mouse monoclonal anti-OR antibody (Santa Cruz Biotechnology, Dallas, Texas, United States).

### 2.4 Enzyme kinetics analysis

To characterize the enzyme kinetic features of newly designated CYP2C9 variants, three representative CYP2C9 probe drugs, tolbutamide, diclofenac and losartan, were included in the drug metabolic activity analysis according to the methods we previously reported. ([Wang et al., 2014](#B38); [Dai et al., 2015b](#B10)) In brief, the incubation mixture consisted of 10–20 pmol of cytochrome b5, 5–10 pmol recombinant CYP2C9 insect microsomes, 100 mM K_3_PO_4_ (pH 7.4) for tolbutamide or 100 mM Tris-HCl (pH 7.4) for diclofenac and losartan, and a series of gradient solutions of drugs (10–1,000 mM for tolbutamide, 1–100 mM for diclofenac and 0.5–25 mM for losartan). Following preincubation at 37°C for 5 min, an NADPH regeneration system (1.3 mmol/LNADP+, 3.3 mmol/L glucose-6-phosphate, 3.3 mmol/L magnesium chloride and 0.4 U/ml glucose-6-phosphate dehydrogenase) was added to start the reaction with a final reaction volume of 200 μL. The incubation proceeded at 37°C with gentle shaking for 60 min for tolbutamide or 30 min for diclofenac and losartan, and finally terminated by the addition of 200 μL acetonitrile containing 50 ng/ml diazepam as an internal standard. Samples were subsequently centrifuged at 12,000 × *g* for 5 min at 4°C and the supernatant was transferred to autosampler plastic vials for injection and detection on ACQUITY I-Class UPLC and Waters XEVO TQD MS (Milford, MA, United States). All samples were analyzed in triplicate and the whole experimental operation except for incubation was carried out on crushed ice.

The enzyme kinetic parameters Km and Vmax were calculated by GraphPad Prism (version seven; GraphPad Software, Inc. San Diego, CA, United States) with the Michaelis-Menten model and non-linear regression analysis parameters. The intrinsic clearance value was obtained by the following formula: Clint = Vmax/Km. All values are presented as the mean ± S.D. (standard deviations). IBM SPSS Statistics software (version 23.0, SPSS Inc. Chicago, IL, United States) was used for the enzymatic activity comparison between wild-type and CYP2C9 variants by one-way analysis of variance. *p* values less than 0.05 were regarded as statistically significant.

### 2.5 Homology modeling of newly designated variants

The three-dimension crystal structure of the wild-type protein CYP2C9.1 (in complex with multiple losartan, PDB ID: 5XXI) was used as a template for homology modeling on the SWISS-MODEL online website ([https://swissmodel. expasy.org/](https://swissmodel.expasy.org/)) to predict the crystal structures of CYP2C9.72 and CYP2C9.74. PyMOL software (Version 2.7, Schrodinger, LLC) was then used for the visualization and alignment of crystal structures of CYP2C9.1 with predicted models.

## 3 Result

### 3.1 CYP2C9 genotyping in the Chinese han population

To fit the standard for new CYP variant submission on the Pharmacogene Variation Consortium (PharmVar), we redesigned and optimized the primers for PCR amplification and sequencing ([Table 1](#T1); [Figure 1A](#F1)). As illustrated in [Figure 1B](#F1), all the target regions of *CYP2C9* could be successfully amplified and sequenced with high quality. After sequencing 931 healthy Chinese Han individuals, a total of seven synonymous polymorphic sites and 10 known *CYP2C9* alleles were detected in this study ([Table 2](#T2); [Table 3](#T3)). In addition, four non-synonymous SNVs (single nucleotide variants) were also identified and designated as *CYP2C9*72*, *CYP2C9*73*, *CYP2C9*74* and *CYP2C9*75* by the Pharmacogene Variation Consortium (PharmVar, [https://www.pharmvar.org/gene/CYP2C9](https://www.pharmvar.org/gene/CYP2C9)). These newly designated allelic variants of *CYP2C9* are located in exon 3 (for **72* and **73*), exon 4 (for **74*) or exon 8 (for **75*) of the gene and carriers with these alleles are all heterozygotes. As shown in [Figure 2](#F2), *CYP2C9*72* contains a SNV of 446C>T and is estimated to make an amino acid substitution of alanine by valine at position 149 of the CYP2C9 protein (A149V). *CYP2C9*73* has another type of SNV 448C>T that can lead to the replacement of arginine with cysteine at position 150 of the protein (R150C). *CYP2C9*73* processes the nucleotide alternation from G to T at position 642 of cDNA, resulting in the amino acid substitution of glutamine by histidine at position 214 (Q214H). *CYP2C9*75* has an A to C variation at position 1,253 of cDNA that contributes to the replacement of asparagine by threonine at position 418 of CYP2C9 protein (N418T). No other mutated sites could be detected within the promoter region and splicing sites of the *CYP2C9* gene in carriers.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5920/9792615/be4cae30912e/fphar-13-1007268-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9792615_fphar-13-1007268-g001.jpg)

Amplification of promoter region and exons of human CYP2C9 gene with newly designed PCR primers. (A): Diagram of newly designed primers and genomic structure of human CYP2C9 gene. Shaded boxes represent the exons and lines indicate the promoter region and introns. Arrowheads indicate the primers that used for the PCR amplification. (B): Gel picture of PCR amplicons of CYP2C9 promoter and exons. M: DNA molecular standard; 1: promoter region; 2: exon1; 3: exon2+exon3; 4: exon4+exon5; 5: exon6; 6: exon7; 7: exon8; 8: exon9.

### TABLE 2.

| gDNA position | Region | Effect on protein | Allele | rsID | n | Allelic frequency (%) | SIFT a | Polyphen-2 b |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 108C>G | Exon1 | L36L | / | rs1311013151 | 1 | 0.05 |   |   |
| 3235G>A | Exon2 | V76V | / | rs17847036 | 8 | 0.43 |   |   |
| 3276T>C | Exon2 | L90P | *13 | rs72558187 | 1 | 0.05 | 0.092 | 0.9 |
| 3549G>A | Exon3 | R124Q | *42 | rs12414460 | 1 | 0.05 | 0.006 | 1 |
| 3573G>A | Exon3 | R132Q | *33 | rs200183364 | 1 | 0.05 | 0.003 | 0.973 |
| 3608C>T | Exon3 | R144C | *2 | rs1799853 | 3 | 0.16 | 0.028 | 1 |
| 3623G>A | Exon3 | A149T | *46 | rs754487195 | 1 | 0.05 | 0.034 | 0.944 |
| 3624C>T | Exon3 | A149V | *72 | rs1289704600 | 1 | 0.05 | 0.032 | 0.996 |
| 3626C>T | Exon3 | R150C | *73 | rs17847037 | 1 | 0.05 | 0.079 | 0.084 |
| 3627G>T | Exon3 | R150L | *27 | rs7900194 | 1 | 0.05 | 0.318 | 0.003 |
| 8757G>T | Exon4 | Q214H | *74 | / | 1 | 0.05 | 0 | 0.999 |
| 9216T>C | Exon4 | L201L | / | rs1429669733 | 2 | 0.11 |   |   |
| 10464G>A | Exon5 | P227P | / | rs772651628 | 1 | 0.05 |   |   |
| 10491C>T | Exon5 | N236N | / | rs765176937 | 2 | 0.11 |   |   |
| 33437C>A | Exon6 | P279T | *29 | rs182132442 | 3 | 0.16 | 0.158 | 0 |
| 42614A>C | Exon7 | I359L | *3 | rs1057910 | 98 | 5.26 | 0.013 | 0.002 |
| 42676T>C | Exon7 | Y379Y | / | rs141283168 | 24 | 1.29 |   |   |
| 47454A>C | Exon8 | N418T | *75 | rs1254213342 | 1 | 0.05 | 0.227 | 0.088 |
| 50173A>T | Exon9 | I434F | *59 | / | 1 | 0.05 | 0.005 | 0.969 |
| 50273T>C | Exon9 | L467P | *60 | rs767284820 | 2 | 0.11 | 0 | 0.994 |
| 50298A>T | Exon9 | G475G | / | rs1057911 | 8 | 0.43 |   |   |

Table 2 Caption: Synonymous and missense mutations of CYP2C9 in the Chinese Han population.

### TABLE 3.

| Genotype | n | Frequency (%) |
| --- | --- | --- |
| *1/*1 | 818 | 87.86 |
| *1/*2 | 3 | 0.32 |
| *1/*3 | 92 | 9.88 |
| *3/*3 | 3 | 0.32 |
| *1/*13 | 1 | 0.11 |
| *1/*27 | 1 | 0.11 |
| *1/*29 | 3 | 0.32 |
| *1/*33 | 1 | 0.11 |
| *1/*42 | 1 | 0.11 |
| *1/*46 | 1 | 0.11 |
| *1/*59 | 1 | 0.11 |
| *1/*60 | 2 | 0.21 |
| *1/*72 | 1 | 0.11 |
| *1/*73 | 1 | 0.11 |
| *1/*74 | 1 | 0.11 |
| *1/*75 | 1 | 0.11 |

Table 3 Caption: Genotype frequencies of CYP2C9 allelic variants in Han Chinese populations population.

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5920/9792615/f6a442402ce8/fphar-13-1007268-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9792615_fphar-13-1007268-g002.jpg)

Sequencing results of the carriers with newly designated alleles CYP2C9*72, CYP2C9*73, CYP2C9*74 and CYP2C9*75. The red arrow indicates mutated sites in allelic variants CYP2C9*72-*75. Rectangular boxes show the cDNA mutations and the amino acid substitutions at codon 149, 150, 214 and 418, respectively.

### 3.2 Expression and functional characterization of the CYP2C9 variants

To better investigate the impacts of these newly found CYP2C9 variants on protein expression and drug metabolic activity, the baculovirus based insect cell expression system was used for CYP2C9 expression and microsome preparation. ([Dai et al., 2015a](#B9); [Liu et al., 2021](#B20)) Immunoblot results revealed that both OR and 2C9 enzymes were successfully co-expressed in insect microsomes and all newly detected variants had similar protein expression levels to that of the wild-type CYP2C9 enzyme ([Figure 3](#F3)). Typically, tolbutamide (TOL), diclofenac (DIC) and losartan (LOS) are three probe drugs for CYP2C9 because only CYP2C9 mediates their metabolism in human liver microsomes ([Waring, 2020](#B39)). In this study, the drug metabolic activity of newly designated variants was investigated with these drugs as substrates. As a result, CYP2C9.72 (Relative Clearance of TOL, LOS, DIC is only 17.90%, 5.54% and 24.73% of wild-type CYP2C9.1, respectively, with *p* < 0.05) and CYP2C9.74 (Relative Clearance of TOL, LOS, DIC is 24.55%, 5.28%, and 55.5% of CYP2C9.1, respectively, with *p* < 0.05) exhibited comparable or even lower catalytic activities to that of the typical defective variant CYP2C9.3, and the other two variants CYP2C9.73 (Relative Clearance of TOL, LOS, DIC is 60.50%,15.53% and 48.88% of CYP2C9.1, respectively, with *p* < 0.05) and CYP2C9.75 (Relative Clearance of TOL, LOS, DIC is 64.85%, 23.64% and 61.89% of CYP2C9.1, respectively, with *p* < 0.05) also showed significantly decreased activities relative to that of wild-type ([Figure 4](#F4) and [Table 4](#T4)). These data indicated that the amino acid substitutions in variants A149V and Q214H had greater impacts on enzyme’s drug metabolic activity than the amino acid replacements in variants R150C and N418T.

### FIGURE 3.

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5920/9792615/e6f1bb5c5f06/fphar-13-1007268-g003.jpg)

Western blotting analysis of expressed wild-type and five CYP2C9 variants in insect cell microsomes. CYP2C9.1, CYP2C9.3, CYP2C9.72, CYP2C9.73, CYP2C9.74 or CYP2C9.75 were highly co-expressed with OR in insect cell microsomes.

### FIGURE 4.

![FIGURE 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5920/9792615/a6a511258fef/fphar-13-1007268-g004.jpg)

Michaelis–Menten curves for the hydroxylation of three CYP2C9 probe substrates tolbutamide (A), losartan (B) and diclofenac (C) by the recombinant CYP2C9 variants. Each point represents the mean ± standard deviation of at least three independent experiments.

### TABLE 4.

| Substrate | Allelic variants | Vmax (pmol/min/pmol P450) | Km (μM) | Clearance (Vmax/Km) | Relative clearance (/CYP2C9.1) |
| --- | --- | --- | --- | --- | --- |
| Tolbutamide | CYP2C9.1 | 1.30 ± 0.065 | 143.60 ± 15.027 | 0.0091 ± 0.00049 | 100.00% |
| CYP2C9.3 | 0.40 ± 0.038* | 501.43 ± 65.61* | 0.00081 ± 0.000051* | 8.93%* |  |
| CYP2C9.72 | 0.27 ± 0.011* | 168.70 ± 22.13 | 0.0016 ± 0.00015* | 17.90%* |  |
| CYP2C9.73 | 0.79 ± 0.010* | 144.30 ± 14.65 | 0.0055 ± 0.00051* | 60.50% |  |
| CYP2C9.74 | 0.73 ± 0.064* | 334.13 ± 57.95 | 0.0022 ± 0.0018* | 24.55%* |  |
| CYP2C9.75 | 0.71 ± 0.020* | 122.50 ± 20.42 | 0.0059 ± 0.00076* | 64.85% |  |
| Losartan | CYP2C9.1 | 0.37 ± 0.026 | 2.16 ± 0.39 | 0.17 ± 0.020 | 100.00% |
| CYP2C9.3 | 0.058 ± 0.0024* | 4.42 ± 0.39* | 0.013 ± 0.00064* | 7.61%* |  |
| CYP2C9.72 | 0.038 ± 0.0012* | 3.94 ± 0.38* | 0.0010 ± 0.00066* | 5.54%* |  |
| CYP2C9.73 | 0.094 ± 0.0034* | 3.55 ± 0.50 | 0.027 ± 0.0038* | 15.53%* |  |
| CYP2C9.74 | 0.14 ± 0.046* | 15.53 ± 7.00 | 0.0091 ± 0.0010* | 5.28%* |  |
| CYP2C9.75 | 0.14 ± 0.0046* | 3.35 ± 0.035 | 0.041 ± 0.0013* | 23.64%* |  |
| Diclofenac | CYP2C9.1 | 10.55 ± 0.051 | 4.43 ± 0.21 | 2.38 ± 0.10 | 100.00% |
| CYP2C9.3 | 9.69 ± 0.14* | 11.65 ± 0.57* | 0.83 ± 0.029* | 34.95%* |  |
| CYP2C9.72 | 1.94 ± 0.028* | 3.29 ± 0.11* | 0.59 ± 0.014* | 24.73%* |  |
| CYP2C9.73 | 5.40 ± 0.60* | 5.59 ± 1.70 | 1.00 ± 0.17* | 48.88%* |  |
| CYP2C9.74 | 6.86 ± 0.11* | 5.19 ± 0.29 | 1.32 ± 0.055* | 55.5%* |  |
| CYP2C9.75 | 4.91 ± 0.26* | 3.16 ± 0.42 | 1.57 ± 0.13* | 61.89% |  |

Table 4 Caption: Enzyme kinetic values of recombinant wild-type and CYP2C9 variants.

### 3.3 Homology modeling of variants

To explore the mechanism for the decreased drug metabolic activity of newly designated variants, homology modeling was performed to predict the impacts of amino acid substitutions on the interactions between the substrate and critical residues of the CYP2C9 enzyme. Relative to that of the wild-type enzyme, all variants exhibited more than 75% reduction in the clearance rate for losartan ([Figure 4](#F4); [Table 4](#T4)). Thus, the crystal structure of losartan-combined wild-type CYP2C9 (PDB ID 5XXI) was selected as the template for homology modeling in this study. CYP2C9.72 and CYP2C9.74 were chosen as homology modeling objects to compare 3D structures with the wild-type CYP2C9, due to their extremely lower metabolic activities relative to that of the other two variants. Focusing on the structure and amino acids within 5 Å around the mutated sites, alanine at position 149 of wild-type CYP2C9 could form three hydrogen bonds with surrounding residues Q146, V145 and V153 ([Figure 5A](#F5)). However, the hydrogen bond between residues Q146 and V149 in CYP2C9.72 was lost ([Figure 5B](#F5)), which might impact the structural stability of the D helix. ([Cojocaru et al., 2007](#B8)) In addition, amino acid substitution in residue 149 also leads to the missing or reduced numbers of hydrogen bonds between losartan and surrounding residues Thr364, Asn218, and Gln214 ([Figures 5C,D](#F5)). We speculated that missing of these hydrogen bonds in variants CYP2C9.72 might affect the stability of CYP2C9 protein and weaken the interactions between the ligand losartan and surrounding residues, which in turn leads to the decrease in the metabolic activity of CYP2C9 enzyme. It has been reported that Glutamine at position 214 is one of the key amino acids of CYP2C9 enzyme because it helps to form the F-G loop and participates in the composition of the active site channel ([Williams et al., 2003](#B41)) Our homology modeling results showed that the amino acid replacement of glutamine by histidine at position 214 results in the missing of hydrogen bond between His214 and Leu208, and causes the blocking of losartan into the active pocket of CYP2C9 enzyme ([Figures 5E, F](#F5)). We suspect that the blockage of losartan entrance might responsible for the significantly decreased metabolic activity of variant CYP2C9.74.

### FIGURE 5.

![FIGURE 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5920/9792615/28a1a60c3945/fphar-13-1007268-g005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9792615_fphar-13-1007268-g005.jpg)

Magnified view of the crystal structure of amino acids within 5 Å around the mutated site of CYP2C9 variants and display of residues interacting with losartan. The hydrogen bonds are illustrated with green dashed lines. Replaced residues of Ala149 in CYP2C9.72, Gln214 in CYP2C9.74 and their interacted residues are shown in (B,F). Corresponding structures of CYP2C9.1 are also presented in (A,E) to illustrate the main difference between wild-type enzyme and newly detected variants. Crystal structures of losartan-interacting amino acid residues in wild-type CYP2C9.1 and variant CYP2C9.72 are displayed in (C,D), respectively. The ligand losartan is shown with yellow color and the hydrogen bonds are shown with green dashed lines.

## 4 Discussion

After being expressed in insect microsomes, all newly detected allelic variants in this study exhibited similar protein expression levels to wild-type but showed significantly reduced drug metabolic activities toward three probe drugs, although the extent of decline in metabolic activity for each variant varied for different substrates ([Figure 3](#F3); [Figure 4](#F4)). Interestingly, we previously reported another allele *CYP2C9*46* that can produce a variant with an amino acid substitution of alanine with threonine at position 149 (A149T). Subsequent *in vitro* functional evaluation revealed that variant CYP2C9.46 was also a poor metabolism mutation type but showed a significantly lower protein expression level than the wild-type. ([Dai et al., 2014](#B12)) In addition, two allelic variants, CYP2C9*8 (R150H) and CYP2C9*27 (R150L), were also reported to have amino acid substitutions at position arginine 150 of the CYP2C9 protein ([Rettie et al., 1994](#B33); [Blaisdell et al., 2004](#B6); [Maekawa et al., 2006](#B23)) Studies on their drug metabolic activities showed that compared to the wild-type CYP2C9.1, variant CYP2C9.8 exhibited a higher clearance rate for tolbutamide *in vitro* but showed little difference in losartan metabolism *in vivo*. ([Allabi et al., 2004](#B2); [Blaisdell et al., 2004](#B6)) For variant CYP2C9.27, it exhibited kinetic parameters against diclofenac similar to those of the wild-type enzyme when expressed in COS-1 cells. ([Maekawa et al., 2006](#B23)) Additionally, another type of amino acid substitution at position 214 (Q214L for *CYP2C9*28*) has been previously reported in the Japanese population. Similar to our data for Q214H, functional analysis results showed that this allelic variant of CYP2C9 exhibited two-fold higher Km values and three-fold lower Vmax values than wild-type and carriers with *CYP2C9*28* could be regarded as Poor metabolizer. ([Maekawa et al., 2006](#B23); [Maekawa et al., 2009](#B24)) These data indicated that amino acid substitutions at different positions had different impacts on the catalytic activities toward different CYP2C9 probe substrates.

To better understand the mechanism behind the reduction in catalytic activity of new CYP2C9 variants toward losartan, homology modeling of CYP2C9.72 and CYP2C9.74 was performed through the SWISSMODEL website. Crystal structure investigation revealed that Ala149 is far from the active center of heme and located in the D-helix center. However, the amino acid substitution of Ala by Val at position 149 can lead to a decrease in hydrogen bonds between mutated sites and surrounding amino acids ([Figure 5B](#F5)), which may result in the instability of the D-helix center. ([Williams et al., 2003](#B41)) In addition, the homology modeling results in this study also indicated that the amino acid substitution of A149V could weaken the interactions between losartan and the surrounding amino acid residues within 5 Å Q214, N218 and T364 ([Figure 5D](#F5)). Previous studies have shown that residues Q214 and N218 are involved in the formation of F0 and G0 in the F-G ring, which is the most important structure on top of the active CYP channel ([Williams et al., 2003](#B41); [Wester et al., 2004](#B40)). When combined with losartan, the polar residues Q214 and N218 in CYP2C9.1 can interact with the imidazole ring of losartan by hydrogen bonds, with the hydroxyl portion of the Q214-linked imidazole ring acting a potential activation site for losartan metabolism. ([Maekawa et al., 2017](#B22)) In addition, the tetrazole of losartan could contact the polar side chain of the T364 residue in the wild-type enzyme to form eight hydrogen bonds ([Figure 5C](#F5)). These interactions suggest that residues Q214, N218 and T364 play vital roles in the metabolism of losartan. Previous reports have proven that amino acid changes in Q214 could have a great impact on the enzyme’s drug metabolic activity. For example, variant CYP2C9.28 (containing Q214L amino acid substitution) exhibited an 87% and 73% reduction in the metabolism of losartan and diclofenac, respectively, compared with that of the wild-type enzyme. ([Maekawa et al., 2006](#B23); [Maekawa et al., 2009](#B24)) Our homology modeling results indicated that the amino acid replacement of Gln214 with histidine in variant CYP2C9.74 may cause the missing of interactions between losartan ligand and enzyme ([Figure 5F](#F5)). We believed that the interaction lost between losartan and enzyme in variant CYP2C9.74 and the hydrogen bonds lost between losartan and key residues Q214, N218 or T364 in CYP2C9.72 are likely to be one of the main reasons for their decreased drug metabolic activity.

In addition to four newly designated allelic variants, *CYP2C9*72-*74*, ten previously reported alleles **2*, **3*, **13*, **27, *29*, **33*, **42, *46, *59* and **60* were also detected in this study ([Table 2](#T2)). Similar to our previously reported results ([Dai et al., 2014](#B12)), *CYP2C9*3* was the most prevalent *CYP2C9* allelic variant in the Chinese Han population with an allele frequency of 5.26% and a genotype frequency of 9.88% for **1/*3* ([Table 3](#T3)). Alleles **13*, **42*, **46*, **59* and **60* are all CYP2C9 allelic variants that were first reported in the Han Chinese population. Functional evaluation revealed that these five variants exhibited significantly decreased catalytic activities toward probe drugs *in vitro* ([Dai et al., 2013](#B11); [Dai et al., 2014](#B12); [Dai et al., 2015b](#B10)). *CYP2C9*27*, **29* and **33* are rare alleles that were first detected in the Japanese population, in which **29* and **33* showed significantly reduced metabolic activity against tolbutamide while **27* exhibited similar activity to that of wild-type enzyme. ([Maekawa et al., 2006](#B23)) These data and our current results indicate that many rare *CYP2C9* allelic variants exist in East Asian populations, represented by Chinese and Japanese individuals, and most of them can lead to a reduction in the drug metabolic activity of enzymes.

In this study, only one individual was discovered to carry *CYP2C9*72*, **73*, **74* or **75* indicating that they all belong to the rare *CYP2C9* alleles in the Chinese Han population, and genetic screening in a large sample size is still needed to further elucidate their real allele frequencies in the Chinese population. With the development of sequencing techniques, many rare *CYP2C9* variants have been identified by large-scale genome sequencing projects, such as the 100,000 Genomes Project ([Marx, 2015](#B26)) and the RIGHT protocol study ([Bielinski et al., 2014](#B5)). In the population database gnomAD (version 2.1.1) ([Karczewski et al., 2020](#B18)), 409 missense variants have been deposited and 217 of them are singletons ([http://gnomad-sg.org/](http://gnomad-sg.org/)). A149V (**72*), R150C (**73*) and Q214H (**74*) are all included in the gnomAD database, in which A149V is only detected in African/African-American with a frequency of 0.0000106; R150C can be found in South Asian, European, African/African-American and other populations with a frequency from 0.00005 to 0.000392; Q214H is detected in only European people with a frequency of 0.0000088. These data indicated that these *CYP2C9* allelic variants are rarely detected in East Asian population, although we first reported their appearance in the Chinese Han population. To date, the vast majority of detected variants of *CYP2C9* are still unannotated. To address this challenge, the deep mutational scanning (MDS) method was developed recently for the evaluation of the enzyme activity of missense variants on a high throughput scale ([Kinney and McCandlish, 2019](#B19)). Zhang et al. developed a “landing pad” DMS system to determine the protein expression level of 109 missense variants of CYP2C9 in HEK 293T cells ([Zhang et al., 2020](#B43)). In that study, R150C was included and exhibited a similar protein expression level to that of the wild-type, which is in agreement with our current study. However, their system cannot analyze the drug metabolic activity for each variant. Recently, Amorosi et al. constructed a CYP2C9 library with more than 6,000 missense variants in both yeast and 293T cells by saturation mutagenesis and developed a multiplexed sequencing-based method, click-seq, for the multiplexed assay of CYP2C9 enzymatic activity ([Amorosi et al., 2021](#B4)). The advantage of this system is that it can evaluate both protein expression capacity and metabolic activity for thousands of constructed variants. In their yeast-cell-based system, both R150C and N418T were classified as “decreased variant”, while the other two variants A149V and Q214H were classified as “possibly non-sense-like” and “wt-like”, respectively. However, their system could not provide the detailed kinetic characterization for some specific drug substrates. In the current study, we reported four CYP2C9 allelic variants A149V, R150C, Q214H and N418T in the Chinese Han population and presented their kinetic characteristics for three CYP2C9 probe drugs tolbutamide, diclofenac and losartan. Our data indicated that all variants exhibited significantly reduced drug metabolic activity than the wild-type. SIFT (Sorting Intolerant From Tolerant) and PolyPhen-2 are two widely used tools for the computational predictions of the impact of protein sequence variants. ([Ng and Henikoff, 2003](#B29); [Adzhubei et al., 2010](#B1)) Consistent to our data, *CYP2C9*72* and **74* are predicted as Deleterious or Probably Damaging by SIFT or PolyPhen-2, respectively. However, *CYP2C9*73* and **75* are predicted to have no impacts on enzyme’s activity by both SIFT and PolyPhen-2 tools ([Table 2](#T2)), indicating that computational prediction may cause misinterpretation for some specific CYP variants. Recently, reported a well-validated workflow integrating the next-generation sequencing, *in silico* analysis and *in vitro* validation assays together, in order to identify the rare and novel pharmacogenomic (PGx) variants and evaluate their functional effects on enzyme’s activity. [Siamoglou et al. (2022)](#B35) However, that system is only focused on CYP2C19 and CYP2D6 and it still needs the incorporation of *in vitro* assay data into program training process to enrich the prediction potential and improve its accuracy. Expanding of this type of computational prediction to CYP2C9 and coupling it with the drug metabolic activity evaluation data will benefit the proper interpretation of rare CYP2C9 variants and assist its application for personalized medicine in clinical practice.

In conclusion, *CYP2C9*3* is the most common *CYP2C9* allelic variant in the Chinese Han population, although other rare variants are also detected. Additionally, four allelic variants (*CYP2C9*72*, **73*, **74* and **75*) were discovered and functional evaluation results indicated that these variants exhibited reduced metabolic activity toward CYP2C9 probe drugs. This study enriches the knowledge of the pharmacogenomics of *CYP2C9* and provides important theoretical guidance for the establishment of individualized dosing guidelines for the Chinese Han population, although further clinical investigation is still needed in the future.

## Acknowledgments

The authors thank to the members in Beijing Hospital and Lishui People’s Hospital for their constructive advice and full supports. Great thanks are also paid to Yu-Ying Qi for her great assistance in data analysis and language editing.

## Data availability statement

The datasets presented in this study can be found in online repositories. The names of the repository/repositories and accession number(s) can be found below: [https://www.ncbi.nlm.nih.gov/snp/](https://www.ncbi.nlm.nih.gov/snp/), rs1289704600, [https://www.ncbi.nlm.nih.gov/snp/](https://www.ncbi.nlm.nih.gov/snp/), rs17847037.

## Ethics statement

The studies involving human participants were reviewed and approved by Institutional Ethics Committee of Beijing Hospital institutions. The patients/participants provided their written informed consent to participate in this study.

## Author contributions

Primary research design, writing-review and funding acquisition by D-PD, HC, and J-PC; Modeling, data interpretation, analysis/theory development, methodology, figure and manuscript authorship were primarily performed by F-LZ, QZ, S-HW, YH, SZ, QZ, P-WG, Q-FL, and J-FY. All the authors have read and agreed to the published version of the manuscript.

## Funding

This work was supported by grants from the National Key R&D Program of China (2020YFC2008301 and 2021YFC2009205), the National High Level Hospital Clinical Research Funding (BJ-2022-121) and the CAMS Innovation Fund for Medical Sciences (2021-I2M-1-050).

## Conflict of interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Publisher’s note

All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

The datasets presented in this study can be found in online repositories. The names of the repository/repositories and accession number(s) can be found below: [https://www.ncbi.nlm.nih.gov/snp/](https://www.ncbi.nlm.nih.gov/snp/), rs1289704600, [https://www.ncbi.nlm.nih.gov/snp/](https://www.ncbi.nlm.nih.gov/snp/), rs17847037.

### Data Availability Statement

The datasets presented in this study can be found in online repositories. The names of the repository/repositories and accession number(s) can be found below: [https://www.ncbi.nlm.nih.gov/snp/](https://www.ncbi.nlm.nih.gov/snp/), rs1289704600, [https://www.ncbi.nlm.nih.gov/snp/](https://www.ncbi.nlm.nih.gov/snp/), rs17847037.

## References

1. Adzhubei I. A., Schmidt S., Peshkin L., Ramensky V. E., Gerasimova A., Bork P., et al. (2010). A method and server for predicting damaging missense mutations. Nat. Methods 7 (4), 248–249. 10.1038/nmeth0410-248  [DOI](https://doi.org/10.1038/nmeth0410-248) | [PMC free article](/articles/PMC2855889/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20354512/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&title=A%20method%20and%20server%20for%20predicting%20damaging%20missense%20mutations&author=I.%20A.%20Adzhubei&author=S.%20Schmidt&author=L.%20Peshkin&author=V.%20E.%20Ramensky&author=A.%20Gerasimova&volume=7&issue=4&publication_year=2010&pages=248-249&pmid=20354512&doi=10.1038/nmeth0410-248&)

2. Allabi A. C., Gala J. L., Horsmans Y., Babaoglu M. O., Bozkurt A., Heusterspreute M., et al. (2004). Functional impact of CYP2C95, CYP2C96, CYP2C98, and CYP2C911 in vivo among black Africans. Clin. Pharmacol. Ther. 76 (2), 113–118. 10.1016/j.clpt.2004.04.001  [DOI](https://doi.org/10.1016/j.clpt.2004.04.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15289788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Functional%20impact%20of%20CYP2C95,%20CYP2C96,%20CYP2C98,%20and%20CYP2C911%20in%20vivo%20among%20black%20Africans&author=A.%20C.%20Allabi&author=J.%20L.%20Gala&author=Y.%20Horsmans&author=M.%20O.%20Babaoglu&author=A.%20Bozkurt&volume=76&issue=2&publication_year=2004&pages=113-118&pmid=15289788&doi=10.1016/j.clpt.2004.04.001&)

3. Almazroo O. A., Miah M. K., Venkataramanan R. (2017). Drug metabolism in the liver. Clin. Liver Dis. 21 (1), 1–20. 10.1016/j.cld.2016.08.001  [DOI](https://doi.org/10.1016/j.cld.2016.08.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27842765/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Liver%20Dis.&title=Drug%20metabolism%20in%20the%20liver&author=O.%20A.%20Almazroo&author=M.%20K.%20Miah&author=R.%20Venkataramanan&volume=21&issue=1&publication_year=2017&pages=1-20&pmid=27842765&doi=10.1016/j.cld.2016.08.001&)

4. Amorosi C. J., Chiasson M. A., McDonald M. G., Wong L. H., Sitko K. A., Boyle G., et al. (2021). Massively parallel characterization of CYP2C9 variant enzyme activity and abundance. Am. J. Hum. Genet. 108 (9), 1735–1751. 10.1016/j.ajhg.2021.07.001  [DOI](https://doi.org/10.1016/j.ajhg.2021.07.001) | [PMC free article](/articles/PMC8456167/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34314704/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Hum.%20Genet.&title=Massively%20parallel%20characterization%20of%20CYP2C9%20variant%20enzyme%20activity%20and%20abundance&author=C.%20J.%20Amorosi&author=M.%20A.%20Chiasson&author=M.%20G.%20McDonald&author=L.%20H.%20Wong&author=K.%20A.%20Sitko&volume=108&issue=9&publication_year=2021&pages=1735-1751&pmid=34314704&doi=10.1016/j.ajhg.2021.07.001&)

5. Bielinski S. J., Olson J. E., Pathak J., Weinshilboum R. M., Wang L., Lyke K. J., et al. (2014). Preemptive genotyping for personalized medicine: Design of the right drug, right dose, right time-using genomic data to individualize treatment protocol. Mayo Clin. Proc. 89 (1), 25–33. 10.1016/j.mayocp.2013.10.021  [DOI](https://doi.org/10.1016/j.mayocp.2013.10.021) | [PMC free article](/articles/PMC3932754/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24388019/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mayo%20Clin.%20Proc.&title=Preemptive%20genotyping%20for%20personalized%20medicine:%20Design%20of%20the%20right%20drug,%20right%20dose,%20right%20time-using%20genomic%20data%20to%20individualize%20treatment%20protocol&author=S.%20J.%20Bielinski&author=J.%20E.%20Olson&author=J.%20Pathak&author=R.%20M.%20Weinshilboum&author=L.%20Wang&volume=89&issue=1&publication_year=2014&pages=25-33&pmid=24388019&doi=10.1016/j.mayocp.2013.10.021&)

6. Blaisdell J., Jorge-Nebert L. F., Coulter S., Ferguson S. S., Lee S. J., Chanas B., et al. (2004). Discovery of new potentially defective alleles of human CYP2C9. Pharmacogenetics 14 (8), 527–537. 10.1097/01.fpc.0000114759.08559.51  [DOI](https://doi.org/10.1097/01.fpc.0000114759.08559.51) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15284535/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Discovery%20of%20new%20potentially%20defective%20alleles%20of%20human%20CYP2C9&author=J.%20Blaisdell&author=L.%20F.%20Jorge-Nebert&author=S.%20Coulter&author=S.%20S.%20Ferguson&author=S.%20J.%20Lee&volume=14&issue=8&publication_year=2004&pages=527-537&pmid=15284535&doi=10.1097/01.fpc.0000114759.08559.51&)

7. Chen H., Dai D. P., Zhou S., Liu J., Wang S. H., Wu H. L., et al. (2020). An identification and functional evaluation of a novel CYP2C9 variant CYP2C9*62. Chem. Biol. Interact. 327, 109168. 10.1016/j.cbi.2020.109168  [DOI](https://doi.org/10.1016/j.cbi.2020.109168) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32531309/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.%20Interact.&title=An%20identification%20and%20functional%20evaluation%20of%20a%20novel%20CYP2C9%20variant%20CYP2C9*62&author=H.%20Chen&author=D.%20P.%20Dai&author=S.%20Zhou&author=J.%20Liu&author=S.%20H.%20Wang&volume=327&publication_year=2020&pages=109168&pmid=32531309&doi=10.1016/j.cbi.2020.109168&)

8. Cojocaru V., Winn P. J., Wade R. C. (2007). The ins and outs of cytochrome P450s. Biochim. Biophys. Acta 1770 (3), 390–401. 10.1016/j.bbagen.2006.07.005  [DOI](https://doi.org/10.1016/j.bbagen.2006.07.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16920266/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta&title=The%20ins%20and%20outs%20of%20cytochrome%20P450s&author=V.%20Cojocaru&author=P.%20J.%20Winn&author=R.%20C.%20Wade&volume=1770&issue=3&publication_year=2007&pages=390-401&pmid=16920266&doi=10.1016/j.bbagen.2006.07.005&)

9. Dai D. P., Li C. B., Wang S. H., Cai J., Geng P. W., Zhou Y. F., et al. (2015a). Identification and characterization of a novel CYP2C9 allelic variant in a warfarin-sensitive patient. Pharmacogenomics 16 (13), 1475–1486. 10.2217/pgs.15.89  [DOI](https://doi.org/10.2217/pgs.15.89) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26255664/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Identification%20and%20characterization%20of%20a%20novel%20CYP2C9%20allelic%20variant%20in%20a%20warfarin-sensitive%20patient&author=D.%20P.%20Dai&author=C.%20B.%20Li&author=S.%20H.%20Wang&author=J.%20Cai&author=P.%20W.%20Geng&volume=16&issue=13&publication_year=2015a&pages=1475-1486&pmid=26255664&doi=10.2217/pgs.15.89&)

10. Dai D. P., Wang S. H., Li C. B., Geng P. W., Cai J., Wang H., et al. (2015b). Identification and functional assessment of a new CYP2C9 allelic variant CYP2C9*59. Drug Metab. Dispos. 43 (8), 1246–1249. 10.1124/dmd.115.063412  [DOI](https://doi.org/10.1124/dmd.115.063412) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25994031/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Identification%20and%20functional%20assessment%20of%20a%20new%20CYP2C9%20allelic%20variant%20CYP2C9*59&author=D.%20P.%20Dai&author=S.%20H.%20Wang&author=C.%20B.%20Li&author=P.%20W.%20Geng&author=J.%20Cai&volume=43&issue=8&publication_year=2015b&pages=1246-1249&pmid=25994031&doi=10.1124/dmd.115.063412&)

11. Dai D. P., Wang Y. H., Wang S. H., Geng P. W., Hu L. M., Hu G. X., et al. (2013). In vitro functional characterization of 37 CYP2C9 allelic isoforms found in Chinese Han population. Acta Pharmacol. Sin. 34 (11), 1449–1456. 10.1038/aps.2013.123  [DOI](https://doi.org/10.1038/aps.2013.123) | [PMC free article](/articles/PMC4006465/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24077631/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Acta%20Pharmacol.%20Sin.&title=In%20vitro%20functional%20characterization%20of%2037%20CYP2C9%20allelic%20isoforms%20found%20in%20Chinese%20Han%20population&author=D.%20P.%20Dai&author=Y.%20H.%20Wang&author=S.%20H.%20Wang&author=P.%20W.%20Geng&author=L.%20M.%20Hu&volume=34&issue=11&publication_year=2013&pages=1449-1456&pmid=24077631&doi=10.1038/aps.2013.123&)

12. Dai D. P., Xu R. A., Hu L. M., Wang S. H., Geng P. W., Yang J. F., et al. (2014). CYP2C9 polymorphism analysis in han Chinese populations: Building the largest allele frequency database. Pharmacogenomics J. 14 (1), 85–92. 10.1038/tpj.2013.2  [DOI](https://doi.org/10.1038/tpj.2013.2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23400009/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=CYP2C9%20polymorphism%20analysis%20in%20han%20Chinese%20populations:%20Building%20the%20largest%20allele%20frequency%20database&author=D.%20P.%20Dai&author=R.%20A.%20Xu&author=L.%20M.%20Hu&author=S.%20H.%20Wang&author=P.%20W.%20Geng&volume=14&issue=1&publication_year=2014&pages=85-92&pmid=23400009&doi=10.1038/tpj.2013.2&)

13. DeLozier T. C., Lee S. C., Coulter S. J., Goh B. C., Goldstein J. A. (2005). Functional characterization of novel allelic variants of CYP2C9 recently discovered in southeast Asians. J. Pharmacol. Exp. Ther. 315 (3), 1085–1090. 10.1124/jpet.105.091181  [DOI](https://doi.org/10.1124/jpet.105.091181) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16099926/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Exp.%20Ther.&title=Functional%20characterization%20of%20novel%20allelic%20variants%20of%20CYP2C9%20recently%20discovered%20in%20southeast%20Asians&author=T.%20C.%20DeLozier&author=S.%20C.%20Lee&author=S.%20J.%20Coulter&author=B.%20C.%20Goh&author=J.%20A.%20Goldstein&volume=315&issue=3&publication_year=2005&pages=1085-1090&pmid=16099926&doi=10.1124/jpet.105.091181&)

14. Flanagan J. U., McLaughlin L. A., Paine M. J., Sutcliffe M. J., Roberts G. C., Wolf C. R. (2003). Role of conserved Asp293 of cytochrome P450 2C9 in substrate recognition and catalytic activity. Biochem. J. 370 (3), 921–926. 10.1042/BJ20021841  [DOI](https://doi.org/10.1042/BJ20021841) | [PMC free article](/articles/PMC1223234/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12482324/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&title=Role%20of%20conserved%20Asp293%20of%20cytochrome%20P450%202C9%20in%20substrate%20recognition%20and%20catalytic%20activity&author=J.%20U.%20Flanagan&author=L.%20A.%20McLaughlin&author=M.%20J.%20Paine&author=M.%20J.%20Sutcliffe&author=G.%20C.%20Roberts&volume=370&issue=3&publication_year=2003&pages=921-926&pmid=12482324&doi=10.1042/BJ20021841&)

15. Fujikura K., Ingelman-Sundberg M., Lauschke V. M. (2015). Genetic variation in the human cytochrome P450 supergene family. Pharmacogenet. Genomics 25 (12), 584–594. 10.1097/FPC.0000000000000172  [DOI](https://doi.org/10.1097/FPC.0000000000000172) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26340336/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Genetic%20variation%20in%20the%20human%20cytochrome%20P450%20supergene%20family&author=K.%20Fujikura&author=M.%20Ingelman-Sundberg&author=V.%20M.%20Lauschke&volume=25&issue=12&publication_year=2015&pages=584-594&pmid=26340336&doi=10.1097/FPC.0000000000000172&)

16. Gaedigk A., Casey S. T., Whirl-Carrillo M., Miller N. A., Klein T. E. (2021). Pharmacogene variation Consortium: A global resource and repository for Pharmacogene variation. Clin. Pharmacol. Ther. 110 (3), 542–545. 10.1002/cpt.2321  [DOI](https://doi.org/10.1002/cpt.2321) | [PMC free article](/articles/PMC8725060/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34091888/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Pharmacogene%20variation%20Consortium:%20A%20global%20resource%20and%20repository%20for%20Pharmacogene%20variation&author=A.%20Gaedigk&author=S.%20T.%20Casey&author=M.%20Whirl-Carrillo&author=N.%20A.%20Miller&author=T.%20E.%20Klein&volume=110&issue=3&publication_year=2021&pages=542-545&pmid=34091888&doi=10.1002/cpt.2321&)

17. Guengerich F. P. (2006). Cytochrome P450s and other enzymes in drug metabolism and toxicity. AAPS J. 8 (1), E101–E111. 10.1208/aapsj080112  [DOI](https://doi.org/10.1208/aapsj080112) | [PMC free article](/articles/PMC2751428/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16584116/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AAPS%20J.&title=Cytochrome%20P450s%20and%20other%20enzymes%20in%20drug%20metabolism%20and%20toxicity&author=F.%20P.%20Guengerich&volume=8&issue=1&publication_year=2006&pages=E101-E111&pmid=16584116&doi=10.1208/aapsj080112&)

18. Karczewski K. J., Francioli L. C., Tiao G., Cummings B. B., Alfoldi J., Wang Q., et al. (2020). The mutational constraint spectrum quantified from variation in 141, 456 humans. Nature 581 (7809), 434–443. 10.1038/s41586-020-2308-7  [DOI](https://doi.org/10.1038/s41586-020-2308-7) | [PMC free article](/articles/PMC7334197/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32461654/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141,%20456%20humans&author=K.%20J.%20Karczewski&author=L.%20C.%20Francioli&author=G.%20Tiao&author=B.%20B.%20Cummings&author=J.%20Alfoldi&volume=581&issue=7809&publication_year=2020&pages=434-443&pmid=32461654&doi=10.1038/s41586-020-2308-7&)

19. Kinney J. B., McCandlish D. M. (2019). Massively parallel assays and quantitative sequence-function relationships. Annu. Rev. Genomics Hum. Genet. 20, 99–127. 10.1146/annurev-genom-083118-014845  [DOI](https://doi.org/10.1146/annurev-genom-083118-014845) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31091417/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Genomics%20Hum.%20Genet.&title=Massively%20parallel%20assays%20and%20quantitative%20sequence-function%20relationships&author=J.%20B.%20Kinney&author=D.%20M.%20McCandlish&volume=20&publication_year=2019&pages=99-127&pmid=31091417&doi=10.1146/annurev-genom-083118-014845&)

20. Liu J., Chen H., Wang S. H., Zhou Q., Geng P. W., Zhou Y. F., et al. (2021). Functional characterization of the defective CYP2C9 variant CYP2C9*18. Pharmacol. Res. Perspect. 9 (1), e00718. 10.1002/prp2.718  [DOI](https://doi.org/10.1002/prp2.718) | [PMC free article](/articles/PMC7842875/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33508175/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Res.%20Perspect.&title=Functional%20characterization%20of%20the%20defective%20CYP2C9%20variant%20CYP2C9*18&author=J.%20Liu&author=H.%20Chen&author=S.%20H.%20Wang&author=Q.%20Zhou&author=P.%20W.%20Geng&volume=9&issue=1&publication_year=2021&pages=e00718&pmid=33508175&doi=10.1002/prp2.718&)

21. Luo S. B., Li C. B., Dai D. P., Wang S. H., Wang Z. H., Geng P. W., et al. (2014). Characterization of a novel CYP2C9 mutation (1009C > A) detected in a warfarin-sensitive patient. J. Pharmacol. Sci. 125 (2), 150–156. 10.1254/jphs.13189FP  [DOI](https://doi.org/10.1254/jphs.13189FP) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25075423/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pharmacol.%20Sci.&title=Characterization%20of%20a%20novel%20CYP2C9%20mutation%20(1009C%20>%20A)%20detected%20in%20a%20warfarin-sensitive%20patient&author=S.%20B.%20Luo&author=C.%20B.%20Li&author=D.%20P.%20Dai&author=S.%20H.%20Wang&author=Z.%20H.%20Wang&volume=125&issue=2&publication_year=2014&pages=150-156&pmid=25075423&doi=10.1254/jphs.13189FP&)

22. Maekawa K., Adachi M., Matsuzawa Y., Zhang Q., Kuroki R., Saito Y., et al. (2017). Structural basis of single-nucleotide polymorphisms in cytochrome P450 2C9. Biochemistry 56 (41), 5476–5480. 10.1021/acs.biochem.7b00795  [DOI](https://doi.org/10.1021/acs.biochem.7b00795) | [PMC free article](/articles/PMC5761073/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28972767/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Biochemistry&title=Structural%20basis%20of%20single-nucleotide%20polymorphisms%20in%20cytochrome%20P450%202C9&author=K.%20Maekawa&author=M.%20Adachi&author=Y.%20Matsuzawa&author=Q.%20Zhang&author=R.%20Kuroki&volume=56&issue=41&publication_year=2017&pages=5476-5480&pmid=28972767&doi=10.1021/acs.biochem.7b00795&)

23. Maekawa K., Fukushima-Uesaka H., Tohkin M., Hasegawa R., Kajio H., Kuzuya N., et al. (2006). Four novel defective alleles and comprehensive haplotype analysis of CYP2C9 in Japanese. Pharmacogenet. Genomics 16 (7), 497–514. 10.1097/01.fpc.0000215069.14095.c6  [DOI](https://doi.org/10.1097/01.fpc.0000215069.14095.c6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16788382/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Four%20novel%20defective%20alleles%20and%20comprehensive%20haplotype%20analysis%20of%20CYP2C9%20in%20Japanese&author=K.%20Maekawa&author=H.%20Fukushima-Uesaka&author=M.%20Tohkin&author=R.%20Hasegawa&author=H.%20Kajio&volume=16&issue=7&publication_year=2006&pages=497-514&pmid=16788382&doi=10.1097/01.fpc.0000215069.14095.c6&)

24. Maekawa K., Harakawa N., Sugiyama E., Tohkin M., Kim S.-R., Kaniwa N., et al. (2009). Substrate-dependent functional alterations of seven CYP2C9 variants found in Japanese subjects. Drug Metab. Dispos. 37 (9), 1895–1903. 10.1124/dmd.109.027003  [DOI](https://doi.org/10.1124/dmd.109.027003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19541829/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Substrate-dependent%20functional%20alterations%20of%20seven%20CYP2C9%20variants%20found%20in%20Japanese%20subjects&author=K.%20Maekawa&author=N.%20Harakawa&author=E.%20Sugiyama&author=M.%20Tohkin&author=S.-R.%20Kim&volume=37&issue=9&publication_year=2009&pages=1895-1903&pmid=19541829&doi=10.1124/dmd.109.027003&)

25. Manikandan P., Nagini S. (2018). Cytochrome P450 structure, function and clinical significance: A review. Curr. Drug Targets 19 (1), 38–54. 10.2174/1389450118666170125144557  [DOI](https://doi.org/10.2174/1389450118666170125144557) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28124606/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Drug%20Targets&title=Cytochrome%20P450%20structure,%20function%20and%20clinical%20significance:%20A%20review&author=P.%20Manikandan&author=S.%20Nagini&volume=19&issue=1&publication_year=2018&pages=38-54&pmid=28124606&doi=10.2174/1389450118666170125144557&)

26. Marx V. (2015). The DNA of a nation. Nature 524 (7566), 503–505. 10.1038/524503a  [DOI](https://doi.org/10.1038/524503a) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26310768/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=The%20DNA%20of%20a%20nation&author=V.%20Marx&volume=524&issue=7566&publication_year=2015&pages=503-505&pmid=26310768&doi=10.1038/524503a&)

27. Miners J. O., Birkett D. J. (1998). Cytochrome P4502C9: An enzyme of major importance in human drug metabolism. Br. J. Clin. Pharmacol. 45 (6), 525–538. 10.1046/j.1365-2125.1998.00721.x  [DOI](https://doi.org/10.1046/j.1365-2125.1998.00721.x) | [PMC free article](/articles/PMC1873650/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9663807/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Cytochrome%20P4502C9:%20An%20enzyme%20of%20major%20importance%20in%20human%20drug%20metabolism&author=J.%20O.%20Miners&author=D.%20J.%20Birkett&volume=45&issue=6&publication_year=1998&pages=525-538&pmid=9663807&doi=10.1046/j.1365-2125.1998.00721.x&)

28. Nebert D. W., Russell D. W. (2002). Clinical importance of the cytochromes P450. Lancet 360 (9340), 1155–1162. 10.1016/S0140-6736(02)11203-7  [DOI](https://doi.org/10.1016/S0140-6736(02)11203-7) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12387968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Clinical%20importance%20of%20the%20cytochromes%20P450&author=D.%20W.%20Nebert&author=D.%20W.%20Russell&volume=360&issue=9340&publication_year=2002&pages=1155-1162&pmid=12387968&doi=10.1016/S0140-6736(02)11203-7&)

29. Ng P. C., Henikoff S. (2003). Sift: Predicting amino acid changes that affect protein function. Nucleic Acids Res. 31 (13), 3812–3814. 10.1093/nar/gkg509  [DOI](https://doi.org/10.1093/nar/gkg509) | [PMC free article](/articles/PMC168916/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12824425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=Sift:%20Predicting%20amino%20acid%20changes%20that%20affect%20protein%20function&author=P.%20C.%20Ng&author=S.%20Henikoff&volume=31&issue=13&publication_year=2003&pages=3812-3814&pmid=12824425&doi=10.1093/nar/gkg509&)

30. Nizamuddin S., Dubey S., Singh S., Sharma S., Machha P., Thangaraj K. (2021). CYP2C9 variations and their pharmacogenetic implications among diverse South Asian populations. Pharmgenomics. Pers. Med. 14, 135–147. 10.2147/Pgpm.S272015  [DOI](https://doi.org/10.2147/Pgpm.S272015) | [PMC free article](/articles/PMC7850565/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33536773/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmgenomics.%20Pers.%20Med.&title=CYP2C9%20variations%20and%20their%20pharmacogenetic%20implications%20among%20diverse%20South%20Asian%20populations&author=S.%20Nizamuddin&author=S.%20Dubey&author=S.%20Singh&author=S.%20Sharma&author=P.%20Machha&volume=14&publication_year=2021&pages=135-147&pmid=33536773&doi=10.2147/Pgpm.S272015&)

31. Qian J. C., Cai J. P., Hu G. X. (2021). Han Chinese specific cytochrome P450 polymorphisms and their impact on the metabolism of anti-hypertensive drugs with adrenoreceptor blocking properties. Expert Opin. Drug Metab. Toxicol. 17 (6), 707–716. 10.1080/17425255.2021.1921147  [DOI](https://doi.org/10.1080/17425255.2021.1921147) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33910442/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Opin.%20Drug%20Metab.%20Toxicol.&title=Han%20Chinese%20specific%20cytochrome%20P450%20polymorphisms%20and%20their%20impact%20on%20the%20metabolism%20of%20anti-hypertensive%20drugs%20with%20adrenoreceptor%20blocking%20properties&author=J.%20C.%20Qian&author=J.%20P.%20Cai&author=G.%20X.%20Hu&volume=17&issue=6&publication_year=2021&pages=707-716&pmid=33910442&doi=10.1080/17425255.2021.1921147&)

32. Relling M. V., Klein T. E., Gammal R. S., Whirl-Carrillo M., Hoffman J. M., Caudle K. E. (2020). The clinical pharmacogenetics implementation Consortium: 10 Years later. Clin. Pharmacol. Ther. 107 (1), 171–175. 10.1002/cpt.1651  [DOI](https://doi.org/10.1002/cpt.1651) | [PMC free article](/articles/PMC6925644/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31562822/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20clinical%20pharmacogenetics%20implementation%20Consortium:%2010%20Years%20later&author=M.%20V.%20Relling&author=T.%20E.%20Klein&author=R.%20S.%20Gammal&author=M.%20Whirl-Carrillo&author=J.%20M.%20Hoffman&volume=107&issue=1&publication_year=2020&pages=171-175&pmid=31562822&doi=10.1002/cpt.1651&)

33. Rettie A. E., Wienkers L. C., Gonzalez F. J., Trager W. F., Korzekwa K. R. (1994). Impaired (S)-warfarin metabolism catalysed by the R144C allelic variant of CYP2C9. Pharmacogenetics 4 (1), 39–42. 10.1097/00008571-199402000-00005  [DOI](https://doi.org/10.1097/00008571-199402000-00005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/8004131/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&title=Impaired%20(S)-warfarin%20metabolism%20catalysed%20by%20the%20R144C%20allelic%20variant%20of%20CYP2C9&author=A.%20E.%20Rettie&author=L.%20C.%20Wienkers&author=F.%20J.%20Gonzalez&author=W.%20F.%20Trager&author=K.%20R.%20Korzekwa&volume=4&issue=1&publication_year=1994&pages=39-42&pmid=8004131&doi=10.1097/00008571-199402000-00005&)

34. Sangkuhl K., Claudio-Campos K., Cavallari L. H., Agundez J. A. G., Whirl-Carrillo M., Duconge J., et al. (2021). PharmVar GeneFocus: CYP2C9. Clin. Pharmacol. Ther. 110 (3), 662–676. 10.1002/cpt.2333  [DOI](https://doi.org/10.1002/cpt.2333) | [PMC free article](/articles/PMC8607432/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34109627/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=PharmVar%20GeneFocus:%20CYP2C9&author=K.%20Sangkuhl&author=K.%20Claudio-Campos&author=L.%20H.%20Cavallari&author=J.%20A.%20G.%20Agundez&author=M.%20Whirl-Carrillo&volume=110&issue=3&publication_year=2021&pages=662-676&pmid=34109627&doi=10.1002/cpt.2333&)

35. Siamoglou S., Koromina M., Hishinuma E., Yamazaki S., Tsermpini E. E., Kordou Z., et al. (2022). Identification and functional validation of novel pharmacogenomic variants using a next-generation sequencing-based approach for clinical pharmacogenomics. Pharmacol. Res. 176, 106087. 10.1016/j.phrs.2022.106087  [DOI](https://doi.org/10.1016/j.phrs.2022.106087) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35033648/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Res.&title=Identification%20and%20functional%20validation%20of%20novel%20pharmacogenomic%20variants%20using%20a%20next-generation%20sequencing-based%20approach%20for%20clinical%20pharmacogenomics&author=S.%20Siamoglou&author=M.%20Koromina&author=E.%20Hishinuma&author=S.%20Yamazaki&author=E.%20E.%20Tsermpini&volume=176&publication_year=2022&pages=106087&pmid=35033648&doi=10.1016/j.phrs.2022.106087&)

36. Subramanian M., Agrawal V., Sandee D., Tam H. K., Miller W. L., Tracy T. S. (2012). Effect of P450 oxidoreductase variants on the metabolism of model substrates mediated by CYP2C9.1, CYP2C9.2, and CYP2C9.3. Pharmacogenet. Genomics 22 (8), 590–597. 10.1097/FPC.0b013e3283544062  [DOI](https://doi.org/10.1097/FPC.0b013e3283544062) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22547083/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Effect%20of%20P450%20oxidoreductase%20variants%20on%20the%20metabolism%20of%20model%20substrates%20mediated%20by%20CYP2C9.1,%20CYP2C9.2,%20and%20CYP2C9.3&author=M.%20Subramanian&author=V.%20Agrawal&author=D.%20Sandee&author=H.%20K.%20Tam&author=W.%20L.%20Miller&volume=22&issue=8&publication_year=2012&pages=590-597&pmid=22547083&doi=10.1097/FPC.0b013e3283544062&)

37. Tornio A., Backman J. T. (2018). Cytochrome P450 in pharmacogenetics: An update. Adv. Pharmacol. 83, 3–32. 10.1016/bs.apha.2018.04.007  [DOI](https://doi.org/10.1016/bs.apha.2018.04.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29801580/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Adv.%20Pharmacol.&title=Cytochrome%20P450%20in%20pharmacogenetics:%20An%20update&author=A.%20Tornio&author=J.%20T.%20Backman&volume=83&publication_year=2018&pages=3-32&pmid=29801580&doi=10.1016/bs.apha.2018.04.007&)

38. Wang Y. H., Pan P. P., Dai D. P., Wang S. H., Geng P. W., Cai J. P., et al. (2014). Effect of 36 CYP2C9 variants found in the Chinese population on losartan metabolism in vitro . Xenobiotica. 44 (3), 270–275. 10.3109/00498254.2013.820007  [DOI](https://doi.org/10.3109/00498254.2013.820007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23844998/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica.&title=Effect%20of%2036%20CYP2C9%20variants%20found%20in%20the%20Chinese%20population%20on%20losartan%20metabolism%20in%20vitro&author=Y.%20H.%20Wang&author=P.%20P.%20Pan&author=D.%20P.%20Dai&author=S.%20H.%20Wang&author=P.%20W.%20Geng&volume=44&issue=3&publication_year=2014&pages=270-275&pmid=23844998&doi=10.3109/00498254.2013.820007&)

39. Waring R. H. (2020). Cytochrome P450: Genotype to phenotype. Xenobiotica. 50 (1), 9–18. 10.1080/00498254.2019.1648911  [DOI](https://doi.org/10.1080/00498254.2019.1648911) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31411087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Xenobiotica.&title=Cytochrome%20P450:%20Genotype%20to%20phenotype&author=R.%20H.%20Waring&volume=50&issue=1&publication_year=2020&pages=9-18&pmid=31411087&doi=10.1080/00498254.2019.1648911&)

40. Wester M. R., Yano J. K., Schoch G. A., Yang C., Griffin K. J., Stout C. D., et al. (2004). The structure of human cytochrome P450 2C9 complexed with flurbiprofen at 2.0-A resolution. J. Biol. Chem. 279 (34), 35630–35637. 10.1074/jbc.M405427200  [DOI](https://doi.org/10.1074/jbc.M405427200) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15181000/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&title=The%20structure%20of%20human%20cytochrome%20P450%202C9%20complexed%20with%20flurbiprofen%20at%202.0-A%20resolution&author=M.%20R.%20Wester&author=J.%20K.%20Yano&author=G.%20A.%20Schoch&author=C.%20Yang&author=K.%20J.%20Griffin&volume=279&issue=34&publication_year=2004&pages=35630-35637&pmid=15181000&doi=10.1074/jbc.M405427200&)

41. Williams P. A., Cosme J., Ward A., Angove H. C., Matak Vinkovic D., Jhoti H. (2003). Crystal structure of human cytochrome P450 2C9 with bound warfarin. Nature 424 (6947), 464–468. 10.1038/nature01862  [DOI](https://doi.org/10.1038/nature01862) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12861225/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Crystal%20structure%20of%20human%20cytochrome%20P450%202C9%20with%20bound%20warfarin&author=P.%20A.%20Williams&author=J.%20Cosme&author=A.%20Ward&author=H.%20C.%20Angove&author=D.%20Matak%20Vinkovic&volume=424&issue=6947&publication_year=2003&pages=464-468&pmid=12861225&doi=10.1038/nature01862&)

42. Zanger U. M., Schwab M. (2013). Cytochrome P450 enzymes in drug metabolism: Regulation of gene expression, enzyme activities, and impact of genetic variation. Pharmacol. Ther. 138 (1), 103–141. 10.1016/j.pharmthera.2012.12.007  [DOI](https://doi.org/10.1016/j.pharmthera.2012.12.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23333322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Ther.&title=Cytochrome%20P450%20enzymes%20in%20drug%20metabolism:%20Regulation%20of%20gene%20expression,%20enzyme%20activities,%20and%20impact%20of%20genetic%20variation&author=U.%20M.%20Zanger&author=M.%20Schwab&volume=138&issue=1&publication_year=2013&pages=103-141&pmid=23333322&doi=10.1016/j.pharmthera.2012.12.007&)

43. Zhang L., Sarangi V., Moon I., Yu J., Liu D., Devarajan S., et al. (2020). CYP2C9 and CYP2C19: Deep mutational scanning and functional characterization of genomic missense variants. Clin. Transl. Sci. 13 (4), 727–742. 10.1111/cts.12758  [DOI](https://doi.org/10.1111/cts.12758) | [PMC free article](/articles/PMC7359949/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32004414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Transl.%20Sci.&title=CYP2C9%20and%20CYP2C19:%20Deep%20mutational%20scanning%20and%20functional%20characterization%20of%20genomic%20missense%20variants&author=L.%20Zhang&author=V.%20Sarangi&author=I.%20Moon&author=J.%20Yu&author=D.%20Liu&volume=13&issue=4&publication_year=2020&pages=727-742&pmid=32004414&doi=10.1111/cts.12758&)

44. Zhao M. Z., Ma J. S., Li M., Zhang Y. T., Jiang B. X., Zhao X. L., et al. (2021). Cytochrome P450 enzymes and drug metabolism in humans. Int. J. Mol. Sci. 22 (23), 12808. 10.3390/ijms222312808  [DOI](https://doi.org/10.3390/ijms222312808) | [PMC free article](/articles/PMC8657965/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34884615/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Cytochrome%20P450%20enzymes%20and%20drug%20metabolism%20in%20humans&author=M.%20Z.%20Zhao&author=J.%20S.%20Ma&author=M.%20Li&author=Y.%20T.%20Zhang&author=B.%20X.%20Jiang&volume=22&issue=23&publication_year=2021&pages=12808&pmid=34884615&doi=10.3390/ijms222312808&)

45. Zhou S. F., Liu J. P., Chowbay B. (2009). Polymorphism of human cytochrome P450 enzymes and its clinical impact. Drug Metab. Rev. 41 (2), 89–295. 10.1080/03602530902843483  [DOI](https://doi.org/10.1080/03602530902843483) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19514967/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Rev.&title=Polymorphism%20of%20human%20cytochrome%20P450%20enzymes%20and%20its%20clinical%20impact&author=S.%20F.%20Zhou&author=J.%20P.%20Liu&author=B.%20Chowbay&volume=41&issue=2&publication_year=2009&pages=89-295&pmid=19514967&doi=10.1080/03602530902843483&)
