# Pharmacogenomic Profiling of Cisplatin-Resistant and -Sensitive Human Osteosarcoma Cell Lines by Multimodal Targeted Next Generation Sequencing

## Metadata
**Authors:** Claudia Maria Hattinger, Chiara Casotti, Maria Pia Patrizio, Silvia Luppi, Leonardo Fantoni, Katia Scotlandi, Toni Ibrahim, Massimo Serra
**Journal:** International Journal of Molecular Sciences
**Date:** 2022 Oct 4
**DOI:** [10.3390/ijms231911787](https://doi.org/10.3390/ijms231911787)
**PMID:** 36233089
**PMCID:** PMC9570120
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9570120/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9570120/pdf/ijms-23-11787.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9570120/pdf/ijms-23-11787.pdf)

## Abstract

Cisplatin (CDDP) is a drug for high-grade osteosarcoma (HGOS) treatment. Several germline pharmacogenetic studies have revealed associations between single nucleotide polymorphisms (SNPs) and CDDP-based therapy response or CDDP-related toxicity in patients with HGOS. Whether these variants could play a biological role in HGOS cells has not been studied so far. The aim of this study was to explore 28 SNPs of 14 genes in 6 CDDP-resistant and 12 drug-sensitive human HGOS cell lines. An innovative multimodal targeted next generation sequencing (mmNGS) approach with custom primers designed for the most commonly reported SNPs of genes belonging to DNA repair, CDDP transport or detoxification, or associated with CDPP-related toxicity was applied. The mmNGS approach was validated by TaqMan genotyping assays and emerged to be an innovative, reliable tool to detect genetic polymorphisms at both the DNA and RNA level. Allele changes in three SNPs (ERCC2 rs13181 and rs1799793, ERCC1 rs11615) were identified on both DNA and RNA derived libraries in association with CDDP resistance. A change of the GSTP1 rs1695 polymorphism from AA to AG genotype was observed in the RNA of all six CDDP-resistant variants. These SNPs emerged to be causally associated with CDDP resistance in HGOS cells.

Keywords: osteosarcoma, next generation sequencing, cisplatin resistance, single nucleotide polymorphism, pharmacogenomics

## 1. Introduction

High-grade osteosarcoma (HGOS), the most common malignant tumor of bone, is treated by surgery and systemic neo-adjuvant multidrug chemotherapy [[1](#B1-ijms-23-11787),[2](#B2-ijms-23-11787)]. Cisplatin (CDDP), together with high-dose methotrexate and doxorubicin, is invariably included in standard chemotherapy for this tumor [[1](#B1-ijms-23-11787),[2](#B2-ijms-23-11787)].

Pharmacogenetic studies have revealed several single nucleotide polymorphisms (SNPs) of genes belonging to either DNA repair, drug transport, folate metabolism, and detoxification pathways to be associated with therapy-related parameters in HGOS, as survival and drug response, or development of drug-associated toxicity [[3](#B3-ijms-23-11787)]. The general goal of these studies was the identification of genomic variations associated with drug response or adverse toxicities, which may provide useful information to improve treatment efficacy and simultaneously reduce the risk of chemotherapy-related toxicities [[4](#B4-ijms-23-11787)].

In the last decade, pharmacogenomic approaches have been increasingly applied to the study of HGOS providing a series of interesting insights related to genetic polymorphisms, which may be causally related to drug resistance or susceptibility to develop treatment-related adverse toxicities [[5](#B5-ijms-23-11787),[6](#B6-ijms-23-11787)]. However, all these indications must be further confirmed because the polymorphic gene status was revealed almost only in patients’ normal (germline) cells at the DNA level without providing information on how these changes were maintained at the RNA level, and influenced RNA and protein expression in tumor cells. The aim of this study was to explore the genotype status of 28 SNPs in 14 genes related to processes involved in DNA repair, CDDP transport and detoxification, or involved in CDDP-related toxicity in a panel of 6 CDDP-resistant and 12 drug-sensitive human HGOS cell lines ([Table A1](#ijms-23-11787-t0A1)). In particular, we focused our study on both pharmacogenetic (germline) and pharmacogenomic (tumor-associated, somatic) markers, which had been indicated to influence treatment response and susceptibility to CDDP-related ototoxicity in HGOS patients, thus appearing as promising candidates for a translation to clinical practice. This selection was performed by taking into consideration the body of evidence reported so far, which has also been recently reviewed [[3](#B3-ijms-23-11787)].

This analysis was performed by using an innovative multimodal targeted next generation sequencing (mmNGS) approach that allowed for the contemporary study of the selected SNPs on both DNA- and RNA-derived libraries. Data obtained by mmNGS on DNA-derived libraries were validated by TaqMan genotyping. RNA expression level of the 14 genes in CDDP-resistant variants compared to their parental cell lines was also determined. Heatmap analysis was performed, including all CDDP-resistant and drug-sensitive cell lines.

## 2. Results

### 2.1. Validation of Custom Multimodal NGS Panel

Data obtained for the 28 SNPs on DNA-derived libraries by the custom mmNGS approach were validated by TaqMan genotyping in 24/28 SNPs. [Table 1](#ijms-23-11787-t001) shows for each cell line the genotype status of all 28 SNPs, which were identified to be either heterozygous or homozygous by sequencing compared to the reference sequence ([Table A2](#ijms-23-11787-t0A2)).

### Table 1.

| Polymorphism | U-2OS |  | Saos-2 |  | IOR/OS9 |  | IOR/OS10 |  | IOR/OS14 |  | IOR/OS15 |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   | TaqM | NGS | TaqM | NGS | TaqM | NGS | TaqM | NGS | TaqM | NGS | TaqM | NGS |
| ABCB1 rs1045642 |   |   | GA | GA | GA | GA | GA | GA | GG | GG | GG | GG |
| ABCB1 rs2032582 | CA | CA | CA | CA | CA | CA | CA | CA | CC | CC | CC | CC |
| ABCB1 rs1128503 | GG | GG | GA | GA | GA | GA | GA | GA | GG | GG | GG | GG |
| ABCC2 rs717620 |   |   |   |   |   |   | CT | CT |   |   |   |   |
| ABCC2 rs2273697 |   |   |   |   |   |   | GA | GA | AA | AA |   |   |
| ABCC2 rs3740066 | CT | CT | TT | TT |   |   | TT | TT |   |   |   |   |
| ABCC2 rs17222723 |   |   |   |   |   |   |   |   |   |   |   |   |
| ABCC3 rs4793665 |   |   |   |   | / | CT | / | TT |   |   | / | TC |
| ABCC3 rs1051640 |   |   |   |   |   |   |   |   |   |   |   |   |
| ACYP2 rs1872328 |   |   |   |   |   |   |   |   |   |   |   |   |
| ATM rs664677 | TT | TT | CT | CT |   |   | TT | TT | TT | TT | TT | TT |
| ATM C11orf65 rs664143 | GG | GG | GA | GA |   |   | GG | GG | GG | GG | GG | GG |
| ATR rs2229032 |   |   |   |   |   |   |   |   | TT | TT |   |   |
| ATR rs2227928 |   |   | GA | GA |   |   | GG | GG | GG | GG | GG/GA | GA |
| COMT rs4646316 | TT | TT | TT | TT |   |   | CT | CT |   |   |   |   |
| COMT rs9332377 |   |   |   |   |   |   |   |   |   |   |   |   |
| ERCC1 rs11615 |   |   | GA | GA | GA | GA | GA | GA | GG | GA |   |   |
| ERCC1 rs3212986 |   |   |   |   | / | AC |   |   | / | AC |   |   |
| ERCC2 rs13181 | GT | GT | GT | GT | GT | GT | GT | GT | GT | GT | GG | GG |
| ERCC2 rs1799793 | CT | CT | TT | TT | CT | CT |   |   | CC | TT |   |   |
| GSTP1 rs1695 | AG | AG | AG | AG |   |   | AG | AG | AG | AG |   |   |
| TP53 rs1042522 | CC | CC |   |   |   |   |   |   | CC | CC |   |   |
| TP53 rs1642785 |   |   |   |   |   |   |   |   | CC | CC |   |   |
| TPMT rs12201199 |   |   |   |   |   |   |   |   |   |   |   |   |
| TPMT rs1142345 |   |   |   |   |   |   |   |   |   |   |   |   |
| TPMT rs1800460 |   |   |   |   |   |   |   |   |   |   |   |   |
| XPA rs1800975 | CT | CC | CT | CT | CC | CC |   |   |   |   | CC | CC |
| XRCC3;KLC1 rs861539 | AG | AG | AA | AA | AA | AA | GG | AG | AA | AA |   |   |
| Polymorphism | IOR/OS18 |  | IOR/OS20 |  | IOR/MOS |  | IOR/SARG |  | HOS |  | MG-63 |  |
|   | TaqM | NGS | TaqM | NGS | TaqM | NGS | TaqM | NGS | TaqM | NGS | TaqM | NGS |
| ABCB1 rs1045642 | GA | GA |   |   | GG | GG | GG | GA | GG | GG |   |   |
| ABCB1 rs2032582 | CA | CA |   |   | CC | CC | CC | CC | CC | CC |   |   |
| ABCB1 rs1128503 | GA | GA |   |   | GG | GG | GG | GG | GG | GG |   |   |
| ABCC2 rs717620 |   |   |   |   | CT | CT |   |   |   |   | CT | CT |
| ABCC2 rs2273697 |   |   | AA | AA |   |   | GA | GA |   |   |   |   |
| ABCC2 rs3740066 |   |   |   |   | CT | CT | CT | CT |   |   | CT | CT |
| ABCC2 rs17222723 |   |   |   |   |   |   |   |   |   |   |   |   |
| ABCC3 rs4793665 | / | TC | / | TC | / | CT | / | CT | / | TT | / | CT |
| ABCC3 rs1051640 | / | GA |   |   | / | GA |   |   |   |   |   |   |
| ACYP2 rs1872328 |   |   |   |   |   |   |   |   |   |   |   |   |
| ATM rs664677 | CT | CT | TT | TT | CT | CT | TT | TT | TT | TT | TT | TT |
| ATM;C11orf65 rs664143 | GA | GA | GG | GG | GA | GA | GG | GG | GG | GG | GG | GG |
| ATR rs2229032 |   |   |   |   | CT | CT | CT | CT | TT | TT |   |   |
| ATR rs2227928 | GG | GG | GG | GG | GG | GG | GG | GG | GG | GG |   |   |
| COMT rs4646316 | CT | CT |   |   |   |   |   |   |   |   |   |   |
| COMT rs9332377 |   |   |   |   | / | TT |   |   |   |   |   |   |
| ERCC1 rs11615 |   |   | GG | GG |   |   | GG | GG | GG | GG | GA | GA |
| ERCC1 rs3212986 |   |   |   |   |   |   | / | AA | / | AC | / | AC |
| ERCC2 rs13181 |   |   |   |   | GT | GT |   |   |   |   | GG | GG |
| ERCC2 rs1799793 |   |   | TT | TT |   |   | CC | TT |   |   | TT | TT |
| GSTP1 rs1695 | AG | AG |   |   |   |   |   |   |   |   |   |   |
| TP53 rs1042522 | CC | CC | CC | CC | CC | CC | CC | CC | CC | CC |   |   |
| TP53 rs1642785 |   |   |   |   |   |   |   |   |   |   |   |   |
| TPMT rs12201199 |   |   |   |   |   |   |   |   |   |   |   |   |
| TPMT rs1142345 |   |   |   |   |   |   |   |   |   |   |   |   |
| TPMT rs1800460 |   |   |   |   |   |   |   |   |   |   |   |   |
| XPA rs1800975 | CC | CC | CC | CC |   |   | CT | CC |   |   |   |   |
| XRCC3;KLC1 rs861539 |   |   | AA | AA | AG | AG | AG | AG |   |   |   |   |
| Polymorphism | U-2OS/ CDDP300 |  | U-2OS/ CDDP1μg |  | U-2OS/ CDDP4μg |  | Saos-2/ CDDP300 |  | Saos-2/ CDDP1μg |  | Saos-2/ CDDP6μg |  |
|   | TaqM | NGS | TaqM | NGS | TaqM | NGS | TaqM | NGS | TaqM | NGS | TaqM | NGS |
| ABCB1 rs1045642 |   |   |   |   |   |   | GA | GA | GA | GA | GA | GA |
| ABCB1 rs2032582 | CA | CA | CA | CA | CA | CA | CA | CA | CA | CA | CA | CA |
| ABCB1 rs1128503 | GG | GG | GG | GG | GG | GG | GA | GA | GA | GA | GA | GA |
| ABCC2 rs717620 |   |   |   |   |   |   |   |   |   |   |   |   |
| ABCC2 rs2273697 |   |   |   |   |   |   |   |   |   |   |   |   |
| ABCC2 rs3740066 | CT | CT | CT | CT | CT | CT | TT | TT | TT | TT | TT | TT |
| ABCC2 rs17222723 |   |   |   |   |   |   |   |   |   |   |   |   |
| ABCC3 rs4793665 |   |   | / | TT | / | TT |   |   |   |   |   |   |
| ABCC3 rs1051640 |   |   |   |   |   |   |   |   |   |   |   |   |
| ACYP2 rs1872328 |   |   |   |   |   |   |   |   |   |   |   |   |
| ATM rs664677 | TT | TT | TT | TT | TT | TT | CT | CT | CT | CT | CT | CT |
| ATM;C11orf65 rs664143 | GG | GG | GG | GG | GG | GG | GA | GA | GA | GA | GA | GA |
| ATR rs2229032 |   |   |   |   |   |   |   |   |   |   |   |   |
| ATR rs2227928 |   |   |   |   |   |   | GA | GA | GA | GA | GA | GA |
| COMT rs4646316 | TT | TT | TT | TT | TT | TT | TT | TT | TT | TT | TT | TT |
| COMT rs9332377 |   |   |   |   |   |   |   |   |   |   |   |   |
| ERCC1 rs11615 |   |   |   |   |   |   | GA | GA | GA | GA | GG | GG |
| ERCC1 rs3212986 |   |   |   |   |   |   |   |   |   |   |   |   |
| ERCC2 rs13181 | GT | GT | TT | GT |   |   | GT | GT | GT | GT | GG | GG |
| ERCC2 rs1799793 | CT | CT | CT | CT |   |   | TT | TT | TT | TT | TT | TT |
| GSTP1 rs1695 | AG | AG | AG | AG | AG | AG | AG | AG | AG | AG | AG | AG |
| TP53 rs1042522 | CC | CC | CC | CC | CC | CC |   |   |   |   |   |   |
| TP53 rs1642785 |   |   |   |   |   |   |   |   |   |   |   |   |
| TPMT rs12201199 |   |   |   |   |   |   |   |   |   |   |   |   |
| TPMT rs1142345 |   |   |   |   |   |   |   |   |   |   |   |   |
| TPMT rs1800460 |   |   |   |   |   |   |   |   |   |   |   |   |
| XPA rs1800975 | CC | CC | CC | CC | CC | CC | CT | CT | CT | CT | CT | CT |
| XRCC3;KLC1 rs861539 | AG | AG | AG | AG | GG | AG | AA | AA | AA | AA | AA | AA |

Table 1 Caption: Variants of 28 polymorphisms identified in 18 human osteosarcoma cell lines by multimodal targeted next generation sequencing and TaqMan genotyping analysis.

Variants with an allele frequency greater than 3% were considered reliable. Those SNPs that were homozygous wild-type for the reference allele were not reported in [Table 1](#ijms-23-11787-t001).

By comparing the data obtained from sequencing and genotyping, we found that for 11/18 (61%) cell lines data obtained by both techniques matched in 100% of the SNPs, whereas in 4/18 (22%) cell lines the match ranged from 90 to 93%, and was below 90% in 3/18 (17%) cell lines. The fact that 39% of the cell lines did not show a complete match could be explained by the presence of different subpopulations within the same cell line.

Interestingly, for five SNPs, *ABCC2* rs17222723, *ACYP2* rs1872328, *TPMT* rs12201199, rs1142345, and rs1800460 the homozygous wild-type genotype was identified in all 18 cell lines.

### 2.2. DNA SNP Evaluation in Relation to Level of CDDP Resistance

#### 2.2.1. Comparison between U-2OS CDDP-Resistant Variants to Parental U-2OS Cell Line

The comparison of polymorphisms identified in the group of U-2OS CDDP-resistant variants in comparison with their parental cells, identified two polymorphisms of the *ERCC2* gene, (rs13181 and rs1799793), which exhibited a genotype change in relation to the acquisition of CDDP resistance ([Table 2](#ijms-23-11787-t002)).

| Cell Line | Identified Variants |  |
| --- | --- | --- |
| ERCC2 rs13181 | ERCC2 rs1799793 |  |
| U-2OS | GT | CT |
| U-2OS/CDDP300 | GT | CT |
| U-2OS/CDDP1μg | GT (a)/TT (b) | CT |
| U-2OS/CDDP4μg | TT | CC |
|   | Amino acid change |  |
| U-2OS | Lys725Gln | Asp286Asn |
| U-2OS/CDDP300 | Lys725Gln | Asp286Asn |
| U-2OS/CDDP1μg | Lys725Gln | Asp286Asn |
| U-2OS/CDDP4μg | / | / |

Table Caption: DNA single nucleotide polymorphisms with genotype and amino acid changes in U-2OS cisplatin (CDDP)-resistant variants compared to their parental cell line. Where not differently indicated, genotypes were identified by both multimodal targeted next generation sequencing (mmNGS) and TaqMan genotyping.

In the CDDP-sensitive, parental cell line and in the two variants with the lower level of CDDP resistance (U-2OS/CDDP300 and U-2OS/CDDP1µg), the genotype of *ERCC2* rs13181 was heterozygous variant (GT), while in the variant with the highest resistance level (U-2OS/CDDP4µg) the genotype of the polymorphism shifted to homozygous wild-type (TT).

[Figure 1](#ijms-23-11787-f001) shows the graphical representation of the mmNGS data obtained by the DNA variant calling identifier tool of the CLC Genomics Workbench (GWB) analysis.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ec8/9570120/0c81daa3d34a/ijms-23-11787-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9570120_ijms-23-11787-g001.jpg)

ERCC2 rs13181 genotypes identified by multimodal targeted next generation sequencing (mmNGS) in U-2OS cell line and U-2OS/CDDP4μg variant. Mismatched nucleotides were evidenced by the CLC Genomics Workbench (GWB) with a background color according to the Rasmol color scheme. The red square box indicates the polymorphism alleles.

The data obtained for these SNPs by TaqMan genotyping and mmNGS were concordant for all cell lines except for *ERCC2* rs13181 in U-2OS/CDDP1µg variant for which mmNGS reported GT but TaqMan genotyping a TT genotype. This apparent discordance may be due to the different sensitivity of the techniques and the presence of subpopulations with TT and GT genotypes. However, these data indicate that in these cells the transition toward a TT genotype is associated with development of CDDP resistance.

For *ERCC2* rs1799793, the sensitive cell line and the two U-2OS/CDDP300 and U-2OS/CDDP1µg resistant cell lines showed a heterozygous variant genotype CT, which became homozygous (CC) in the variant with the highest resistance level ([Table 2](#ijms-23-11787-t002) and [Figure 2](#ijms-23-11787-f002)).

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ec8/9570120/f86311ac555b/ijms-23-11787-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9570120_ijms-23-11787-g002.jpg)

ERCC2 rs1799793 genotypes identified by multimodal targeted next generation sequencing (mmNGS) in U-2OS cell line and U-2OS/CDDP4μg variant. Mismatched nucleotides were evidenced by the CLC Genomics Workbench (GWB) with a background color according to the Rasmol color scheme. The red square box indicates the polymorphism alleles.

As shown in [Table 2](#ijms-23-11787-t002) and [Figure 3](#ijms-23-11787-f003), both SNPs of *ERCC2* were non-synonymous and caused amino acid changes. The SNP rs13181 caused the substitution of Lys by Gln and the rs1799793 the substitution of Asp with Asn.

![Figure 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ec8/9570120/b3735f4cbdbc/ijms-23-11787-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9570120_ijms-23-11787-g003.jpg)

Amino acid changes identified by multimodal targeted next generation sequencing (mmNGS) in the U-2OS cell line for the ERCC2 rs13181 (a) and ERCC2 rs1799793 (b). Amino acids are colored by the CLC Genomics Workbench (GWB) according to the Rasmol color scheme.

#### 2.2.2. Comparison between Saos-2 CDDP-Resistant Variants to Parental Saos-2 Cell Line

The comparison of DNA variant calling data between Saos-2 CDDP-resistant variants and their parental Saos-2 CDDP-sensitive cell line identified genotype changes of *ERCC2* rs13181 and *ERCC1* rs11615 ([Table 3](#ijms-23-11787-t003)). These genotype changes were confirmed by both TaqMan genotyping and mmNGS.

| Cell Line | Identified Variants |  |
| --- | --- | --- |
| ERCC2 rs13181 | ERCC1 rs11615 |  |
| Saos-2 | GT | GA |
| Saos-2/CDDP300 | GT | GA |
| Saos-2/CDDP1μg | GT | GA |
| Saos-2/CDDP6μg | GG | GG |
|   | Amino Acid Change |  |
| Saos-2 | Lys725Gln | Synonymous |
| Saos-2/CDDP300 | Lys725Gln | Synonymous |
| Saos-2/CDDP1μg | Lys725Gln | Synonymous |
| Saos-2/CDDP6μg | Lys725Gln | Synonymous |

Table Caption: DNA single nucleotide polymorphisms with genotype and amino acid changes in Saos-2 cisplatin (CDDP)-resistant variants compared to their parental cell line, which were identified by both multimodal targeted next generation sequencing (mmNGS) and TaqMan genotyping.

For *ERCC2* rs13181, the genotype of the detected polymorphism was heterozygous variant GT in the sensitive and the two Saos-2 resistant variants with lower resistance levels, while in the Saos-2/CDDP6µg variant the genotype changed to homozygous variant GG ([Figure 4](#ijms-23-11787-f004)).

![Figure 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ec8/9570120/caaaeee9c53a/ijms-23-11787-g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9570120_ijms-23-11787-g004.jpg)

ERCC2 rs13181 genotypes identified by multimodal targeted next generation sequencing (mmNGS) in Saos-2 cell line and Saos-2/CDDP6μg variant. Mismatched nucleotides are evidenced by the CLC Genomics Workbench (GWB) with a background color according to the Rasmol color scheme. The red square box indicates the polymorphism alleles.

The same situation occurred for *ERCC1* rs11615, which was heterozygous variant GA in the sensitive cell line and the two resistant variants with lower resistance levels, whereas homozygous variant GG in Saos-2/CDDP6µg ([Figure 5](#ijms-23-11787-f005)).

![Figure 5](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ec8/9570120/ab4ab67b47ab/ijms-23-11787-g005.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9570120_ijms-23-11787-g005.jpg)

ERCC1 rs11615 genotypes identified by multimodal targeted next generation sequencing (mmNGS) in Saos-2 cell line and Saos-2/CDDP6μg variant. Mismatched nucleotides are evidenced by the CLC Genomics Workbench (GWB) with a background color according to the Rasmol color scheme. The red square box indicates the polymorphism alleles.

Different to the *ERCC2* rs13181 variant, which caused an amino acid change from Lys to Gln, no amino acid changes were revealed by the CLC GWB analysis for the synonymous *ERCC1* rs11615 variant ([Figure 6](#ijms-23-11787-f006)).

![Figure 6](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ec8/9570120/ad30079597a3/ijms-23-11787-g006.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9570120_ijms-23-11787-g006.jpg)

Amino acids indicated by multimodal targeted next generation sequencing (mmNGS) in the Saos-2/CDDP1μg variant for the non-synonymous ERCC2 rs13181 (a) and the synonymous ERCC2 rs11615 (b). Amino acids are colored by the CLC Genomics Workbench (GWB) according to the Rasmol color scheme.

### 2.3. RNA SNP Evaluation in Relation to Level of CDDP Resistance

#### 2.3.1. Comparison between U-2OS Cell Line and U-2OS CDDP-Resistant Variants

All genotype variations identified at the DNA level and described above were also identified on the RNA level, indicating that these changes had been selected and maintained during development of CDDP resistance. Differently, the *GSTP1* rs1695 SNP changed in the RNA-derived libraries of U-2OS cell line and U-2OS/CDDP1µg variant compared to the DNA-derived libraries ([Table 4](#ijms-23-11787-t004)). The genotype of the *GSTP1* rs1695 detected on DNA remained AG in the sensitive and in the three resistant cell lines. At the RNA level, the genotype of *GSTP1* rs1695 was homozygous wild-type AA in U-2OS and heterozygous variant AG in U-2OS/CDDP300 and U-2OS/CDDP4µg variants, while in the U-2OS/CDDPP1µg variant, a multi nucleotide variant (MNV) GAT, was detected.

| Cell Line | Identified Variants |  |
| --- | --- | --- |
| GSTP1 rs1695 |  |  |
| DNA | RNA |  |
| U-2OS | AG | AA |
| U-2OS/CDDP300 | AG | AG |
| U-2OS/CDDP1μg | AG | GAT |
| U-2OS/CDDP4μg | AG | AG |
|   | Amino acid change |  |
| U-2OS | / |  |
| U-2OS/CDDP300 | Ile105Val |  |
| U-2OS/CDDP1μg | Ile105Val |  |
| U-2OS/CDDP4μg | Ile105Val |  |

Table Caption: RNA single nucleotide polymorphism (SNP) of GSTP1 gene detected in U-2OS cisplatin (CDDP)-resistant variants compared to their parental cell line.

Interestingly, the amino acid change Ile105Val caused by the *GSTP1* rs1695 variant allele was identified by the CLC GWB in all three CDDP-resistant U-2OS variants ([Table 4](#ijms-23-11787-t004)).

#### 2.3.2. Comparison between Saos-2 Cell Line and Saos-2 CDDP-Resistant Variants

All genotype changes identified at the DNA level were also identified on RNA except for the *GSTP1* rs1695 SNP. As with U-2OS cell lines, the rs1695 genotype was AG at DNA level whereas homozygous AA at RNA level in the Saos-2 parental cell line ([Table 5](#ijms-23-11787-t005)). No difference was found at DNA and RNA level for all CDDP-resistant variants ([Table 5](#ijms-23-11787-t005)).

| Cell Line | Identified Variants |  |
| --- | --- | --- |
| GSTP1 rs1695 |  |  |
| DNA | RNA |  |
| Saos-2 | AG | AA |
| Saos-2/CDDP300 | AG | AG |
| Saos-2/CDDP1μg | AG | AG |
| Saos-2/CDDP6μg | AG | AG |
|   | Amino acid change |  |
| Saos-2 | / |  |
| Saos-2/CDDP300 | Ile105Val |  |
| Saos-2/CDDP1μg | Ile105Val |  |
| Saos-2/CDDP6μg | Ile105Val |  |

Table Caption: RNA single nucleotide polymorphism (SNP) of GSTP1 gene detected in Saos-2 cisplatin (CDDP)-resistant variants compared to their parental cell line.

Accordingly, the amino acid change Ile105Val caused by the *GSTP1* rs1695 variant allele was identified by the CLC GWB in all three CDDP-resistant Saos-2 variants with the AG genotype in the RNA-derived libraries ([Table 5](#ijms-23-11787-t005)).

### 2.4. RNA Expression Analysis

Targeted RNAseq was performed for the 14 genes related to either CDDP drug response or toxicity reported after CDDP therapy. The fold-change of transcripts per million (TPM), which estimates the fold-change in RNA expression, for each CDDP-resistant variant compared to its drug-sensitive parental cell line is graphically shown in [Figure 7](#ijms-23-11787-f007).

### Figure 7.

![Figure 7](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ec8/9570120/28c7bf331ae4/ijms-23-11787-g007.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9570120_ijms-23-11787-g007.jpg)

Fold-changes of RNA expression in U-2OS cisplatin (CDDP)-resistant variants (a) and Saos-2 CDDP-resistant variants (b) compared to their parental cell lines. Thick lines indicate thresholds for overexpression (2-fold) and under-expression (0.5-fold). * indicates those genes that were significantly differentially expressed in the groups of U-2OS and Saos-2 CDDP-resistant variants compared to their respective parental cell line.

In U2OS-derived CDDP-resistant variants, six genes, *ABCC2, ABCC3, ACYP2, COMT, ERCC2*, and *XRCC3* emerged to be increased more than 2-fold compared to the parental U-2OS cell line, whereas four genes, *ATM, ATR, TP53*, and *XPA* were downregulated in CDDP-resistant variants. Considering all three CDDP-resistant variants together, the differential gene expression tool of the CLC GWB identified the downregulation of *ATM, ATR*, and *TP53* as significant with a Bonferroni corrected *p*-value < 0.05.

In Saos-2-derived CDDP-resistant variants, six genes were increased more than 2-fold: *ABCB1, ABCC2* and *XRCC3* in all three variants, whereas *ACYP2, COMT* and *ERCC2* only in Saos-2/CDDP300, the variant with the lowest resistance level. CDDP-resistant variants also presented downregulation of *ATM, ATR, GSTP1, TPMT,* and *XPA* genes. Evaluating all three CDDP-resistant variants together, a significant difference after Bonferroni correction with a *p*-value < 0.05 was identified for upregulation of *ABCB1* and downregulation of *TPMT* and *XPA*.

Similarities between CDDP-resistant and CDDP-sensitive cell lines were assessed by using the heatmap tool of the CLC GWB, including all 14 genes ([Figure 8](#ijms-23-11787-f008)). Two main clusters were revealed. One consisted of two clusters formed by all six CDDP-resistant variants clearly separated from their two parental cell lines.

### Figure 8.

![Figure 8](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ec8/9570120/8b278fa21141/ijms-23-11787-g008.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9570120_ijms-23-11787-g008.jpg)

Heatmap plus hierarchical clustering analysis of 12 drug-sensitive and 6 cisplatin (CDDP)-resistant human osteosarcoma cell lines generated from RNAseq data using the CLC Genomics Workbench (GWB) tool.

The 10 drug-sensitive cell lines formed the second main cluster, which was mostly separated from that of CDDP-resistant variants.

As also shown in [Figure 8](#ijms-23-11787-f008), the group of 14 genes resulted to be divided in 6 clusters, with genes belonging to the same family mostly grouped together.

## 3. Discussion

In this study, a custom mmNGS approach has been used to study 28 SNPs of 14 genes, contemporarily on the DNA and RNA level, in 6 CDDP-resistant and 12 CDDP-sensitive human HGOS cell lines. To our knowledge, this innovative approach has not been used so far for pharmacogenomic studies.

The successful validation of the DNA variant calling by TaqMan genotyping confirmed that this approach is an appropriate method to study even rare SNPs. Compared to genotyping by single TaqMan assays, the custom mmNGS approach is faster and also offers the possibility to identify additional SNPs mapping to the target region. Moreover, small targeted panels allow the pooling of higher sample numbers compared to whole-genome NGS. Another advantage of the mmNGS approach is the low amount of starting material that is required for library preparation, which facilitates the application of this method to tumor tissue samples. In addition, the simultaneous analysis of SNPs on the DNA and RNA level, as well as the possibility to estimate the level of RNA expression associated with the polymorphic gene status allow for a direct correlation between the genotype status with the biological function of each SNP.

The frequencies of variant alleles per cell line ranged from 9 (found in IOR/OS15 and MG-63) to 22 (detected in IOR/SARG), confirming the heterogeneity and high genetic instability of HGOS.

Particular genotype distributions were found for 9 SNPs. Five SNPs that had been reported in association with CDDP-related ototoxicity, *ACYP2* rs1872328 [[7](#B7-ijms-23-11787)], *ABCC2* rs17222723 [[8](#B8-ijms-23-11787)], *TPMT* rs12201199, rs1142345, and rs1800460 [[9](#B9-ijms-23-11787),[10](#B10-ijms-23-11787),[11](#B11-ijms-23-11787)], were present only in the wild-type status in all cell lines. Two SNPs, *ABCC2* rs717620 and *GSTP1* rs1695, were found as homozygous wild-type or heterozygous but not as homozygous variant. These findings suggest that the variant allele of these seven SNPs could be of biological disadvantage in HGOS tumor cells. Differently, the two SNPs of *TP53*, rs1042522 and rs1642785, were identified either in a homozygous wild-type or variant but not heterozygous status.

The most relevant SNPs that emerged in this study to be associated with the development of CDDP resistance were *GSTP1* rs1695, *ERCC2* rs13181, *ERCC2* rs1799793, and *ERCC1* rs11615.

The *GSTP1* rs1695 was the only SNP for which the genotype changes found in the RNA-derived libraries differed from those revealed in DNA-derived libraries. Interestingly, in all five drug-sensitive cell lines with the heterozygous genotype in the DNA (U-2OS, Saos-2, IOR/10, IOR/14, IOR/18), the genotype status in the RNA was homozygous wild-type. The presence of the variant also in the RNA of all six CDDP-resistant cell lines, with the consequent amino acid change Ile105Val, strongly suggests that the AG genotype is associated with reduced CDDP response. These findings further support the previously demonstrated relevance of GSTP1 enhanced enzymatic activity in these CDDP-resistant HGOS cell lines [[12](#B12-ijms-23-11787)]. The pharmacogenomic findings emerged from the present study thus indicate that the increase in GSTP1 activity observed in CDDP-resistant variants is correlated with the transition to the AG genotype of the rs1695 polymorphism and the consequent Ile105Val amino acid change.

This observation is concordant with the data reported in almost all germline studies. A significant association between AG+GG genotypes and poor histological response as well as decreased event-free and overall survival was observed in five studies [[8](#B8-ijms-23-11787),[13](#B13-ijms-23-11787),[14](#B14-ijms-23-11787),[15](#B15-ijms-23-11787),[16](#B16-ijms-23-11787)] whereas one study reported the GG genotype to be associated with good response [[17](#B17-ijms-23-11787)].

Interestingly, *GSTP1* rs1695 was excluded from further analyses in the study by Goricar and co-workers because the genotype frequencies of rs1695 were not in Hardy Weinberg Equilibrium [[18](#B18-ijms-23-11787)]. Since their study was performed on paraffin embedded HGOS tumor tissue samples and not on DNA extracted from lymphocytes, as almost all pharmacogenetic analyses, their observation is concordant with our data obtained on HGOS cell lines.

Genotype changes in relation to CDDP resistance were found for the two non-synonymous SNPs *ERCC2* rs1799793 and rs13181 and the synonymous *ERCC1* rs11615 at the DNA and RNA level. All three SNPs have been reported to be associated with survival and toxicity, but the data are quite discordant [[17](#B17-ijms-23-11787),[18](#B18-ijms-23-11787),[19](#B19-ijms-23-11787),[20](#B20-ijms-23-11787),[21](#B21-ijms-23-11787),[22](#B22-ijms-23-11787),[23](#B23-ijms-23-11787),[24](#B24-ijms-23-11787),[25](#B25-ijms-23-11787),[26](#B26-ijms-23-11787),[27](#B27-ijms-23-11787),[28](#B28-ijms-23-11787),[29](#B29-ijms-23-11787)].

However, the *ERCC2* rs1799793 GG genotype was reported in association with poor event-free survival compared to the GA +AA genotypes and *ERCC2* rs13181 AA with poor response to chemotherapy compared to AC+CC [[18](#B18-ijms-23-11787)]. Our data obtained in CDDP-resistance cell lines confirm the relevance of these two SNPs and suggest that they could serve as biomarkers.

In one germline study performed on 130 patients with osteosarcoma treated with neoadjuvant cisplatin-based therapy in combination with doxorubicin, methotrexate, and ifosfamide the *ERCC2* rs13181 and *ERCC2* rs1799793 SNPs were associated with survival [[28](#B28-ijms-23-11787)]. The authors suggested that the amino acid change that occurred as a result of the mutation reduced the ability of the enzyme ERCC2 to repair DNA thus resulting in greater efficacy of cisplatin. In our study this finding seems to be confirmed by the fact that the most resistant cell line returned to the wild-type genotype, restoring the repair capacity of the enzyme with the consequent increased resistance to the chemotherapeutic agent.

The functional consequences of *ERCC2* rs1799793 and rs13181 on the protein structure and stability have recently been elucidated [[30](#B30-ijms-23-11787),[31](#B31-ijms-23-11787)]. Molecular dynamics simulation of the native ERCC2 protein and the variant protein with the substitution of Asp by Asn revealed that rs1799793 resulted in a destabilized, less active protein compared to the native.

In addition, the *ERCC2* rs13181 variant caused the loss of C-terminal alpha-helix and beta-sheet [[30](#B30-ijms-23-11787)]. Although these secondary structures were lost, the overall folding was not disrupted, suggesting that this polymorphic variation has a less relevant impact on protein function.

For *ERCC1* rs11615, which changed to the homozygous wild-type genotype status in the Saos-2 CDDP-resistant variant with the highest level of resistance, two germline studies reported similar evidence being better survival associated with the TT compared to the CC genotype [[24](#B24-ijms-23-11787),[25](#B25-ijms-23-11787)]. However, five studies reported the opposite evidence for overall survival [[19](#B19-ijms-23-11787),[20](#B20-ijms-23-11787),[21](#B21-ijms-23-11787),[22](#B22-ijms-23-11787),[23](#B23-ijms-23-11787)].

Differential gene expression analysis identified dysregulations in CDDP-resistant variants compared to their parental cell lines suggesting that development of CDDP resistance influences not only genes of the NER pathway, which is known to be mainly responsible for the removal of CDDP-associated DNA adducts, but also genes belonging to other DNA repair mechanisms, such as *ATM* and *ATR*. It has been shown that cancer cells that are deficient in one DNA repair pathway can activate other functional repair pathways, which underlines the importance to study not only one of them for treatment optimization [[32](#B32-ijms-23-11787)].

The biological consequence of the significant downregulation of *ATM* and *ATR* observed in Saos-2/CDDP-resistant variants is a relevant finding that needs to be further explored, since inhibitors against ATR have already been used in clinical trials in other cancers [[33](#B33-ijms-23-11787)]. On the other hand, also the upregulation of *ACYP2* and *COMT*, although not significant, warrants attention because SNPs of these two genes had been described to be associated with ototoxicity after CDDP treatment [[7](#B7-ijms-23-11787),[9](#B9-ijms-23-11787),[10](#B10-ijms-23-11787),[11](#B11-ijms-23-11787)].

In conclusion, the mmNGS approach emerged to be an innovative, reliable tool to detect genetic polymorphisms at both DNA and RNA level, allowing for the identification of genetic changes causally related to CDDP resistance in HGOS cells. Once further validated in tumor samples series, these SNPs could be useful to identify patients with reduced sensitivity to CDDP-based therapy and/or increased susceptibility to CDDP-related adverse toxicities.

## 4. Materials and Methods

### 4.1. Cell Lines

The study was performed on a panel of 12 drug-sensitive human HGOS cell lines: U-2OS, Saos-2, MG-63, and HOS (purchased from the American Type Culture Collection ATCC, Rockville, MD, USA) and IOR/OS9, IOR/OS10, IOR/OS14, IOR/OS15, IOR/OS18, IOR/OS20, IOR/MOS, IOR/SARG, which were established from tumor specimens at the Laboratory of Experimental Oncology of the Orthopaedic Rizzoli Institute [[34](#B34-ijms-23-11787)].

The panel of 6 CDDP-resistant variants derived either from U-2OS (U-2OS/CDDP300, U-2OS/CDDP1μg, U-2OS/CDDP4μg) or Saos-2 (Saos-2/CDDP300, Saos-2/CDDP1μg, Saos-2/CDDP6μg) CDDP-sensitive cell lines, as previously reported [[12](#B12-ijms-23-11787)]. Resistant variants were established by exposing parental cells to step-by-step increases of CDDP concentrations. The in vitro continuous drug exposure resulted in the establishment of variants resistant to 300 ng/mL CDDP (U-2OS/CDDP300 and Saos-2/CDDP300), 1 µg/mL (U-2OS/CDDP1µg and Saos-2/CDDP1µg), 4 µg/mL (U-2OS/CDDP4µg), or 6 µg/mL CDDP (Saos-2/CDDP6µg). Establishment of an adequate in vitro growth at each new CDDP concentration required approximately 10–12 weeks (corresponding to 8–10 in vitro passages), and variants were considered as definitely stabilized when reaching the 20th in vitro passage.

CDDP sensitivity of each cell line was expressed as IC50 (drug concentration resulting in 50% inhibition of cell growth after 96 h of in vitro treatment). The fold-increase in CDDP resistance of each variant was determined by comparing its IC50 value with that of its corresponding parental cell line and, as previously described, ranged from 4.0- to 62.5-fold for U-2OS variants and from 7.4- to 112.1-fold for Saos-2 variants [[12](#B12-ijms-23-11787)].

All cell lines were cultured in Iscove’s modified Dulbecco’s medium (IMDM) added with 10% fetal bovine serum (Biowhittaker Europe, Cambrex-Verviers, Belgium) and maintained in a humified atmosphere with 5% CO_2_ at 37 °C. Drug resistant variants were continuously cultured in the presence of the CDDP concentrations used for their selection. Cell pellets were prepared according to standard procedures when cells were confluent, snap-frozen, and stored at −80 °C.

DNA fingerprint analyses were performed for all cell lines using 17 polymorphic short tandem repeat sequences confirming their identity.

### 4.2. Extraction of Nucleic Acids

DNA and RNA were simultaneously isolated and purified from the same pellet obtained from each cell line by using the AllPrep DNA/RNA mini kit (Qiagen, Hilden, Germany) according to the manufacturers’ instructions. During this process, the DNA and RNA were isolated from the entire sample by passing the lysate first to the AllPrep DNA spin column to isolate high molecular weight total genomic DNA and through the AllPrep RNA spin column to isolate total RNA.

A DNA and RNA quality check was performed for all samples by spectrophotometry (NP-80, Implen, Munich, Germany). All RNA samples were run on a 2100 Bioanalyzer system (Agilent, Santa Clara, CA, USA) using the RNA 6000 kit (Agilent, Santa Clara, CA, USA).

### 4.3. Custom Multi-Modal Targeted Next Generation Sequencing (mmNGS)

Library preparation was performed according to the QIAseq Multimodal Panel handbook v06/2020 (Qiagen, Hilden, Germany) for small panels. The primers for the libraries derived from DNA were designed for 28 SNPs of 14 genes related to DNA repair, CDDP transport and detoxification, and TP53 ([Table A2](#ijms-23-11787-t0A2)). For the libraries prepared from RNA, primers were designed for the SNPs mapping to exons of the 14 genes, thus allowing RNA variant calling. The specific design of the RNA panel enabled also RNA expression analysis of these 14 genes (technical service Qiagen, Hilden, Germany). This approach uses integrated unique molecular indices (UMIs) which improves the specificity of variant detection.

DNA- and RNA-derived libraries were prepared for all 18 cell lines. Prior to library preparation, the nucleic acid concentrations were determined fluorometrically by Qubit high-sensitivity assays on a Qubit reader version 4.0 (Thermo Fisher Scientific by Life Technologies Italia, Monza, Italy).

For library preparation, the input amount was 40 ng of DNA and 100 ng of RNA. All libraries were run on a 2100 Bioanalyzer system (Agilent, Santa Clara, CA, USA) using the High Sensitivity DNA kit (Agilent, Santa Clara, CA, USA) to check the profile of the samples. The fragment lengths of all libraries ranged between 400 and 600 base pairs, as expected according to the protocol.

In order to provide an accurate quantification of the amplifiable libraries, the QIAseq Library Quant Assay kit (Qiagen, Hilden, Germany) was performed on a real-time PCR system (7900HT Fast Real-time PCR system; (Thermo Fisher Scientific by Life Technologies Italia, Monza, Italy) for all of them.

For sequencing, libraries were diluted to 1.2 pM, pooled together and analyzed by paired-end sequencing on a NextSeq 500 instrument (Illumina Inc., San Diego, CA, USA) using a mid-output reagent kit v2.5 (300 cycles) with a custom sequencing primer provided with the library preparation kit.

### 4.4. mmNGS Data Analysis by CLC Genomics Workbench

All bioinformatic analyses were performed using the CLC GWB software (Qiagen Bioinformatics, Aarhus, Denmark) v22.04. FastQ files were downloaded from the BaseSpace cloud (Illumina Inc., San Diego, CA, USA) and imported in the CLC GWB (Qiagen Bioinformatics, Aarhus, Denmark).

For the detection of DNA variants and gene expression, the FastQ files were analyzed using the Biomedical Genomics Analysis plugin running the Qiaseq Multimodal Analysis workflow.

The DNA and RNA reads were aligned to the human genome hg38 reference sequence and filtered using a coverage of 100× and a variant allele frequency (VAF) higher than 3%.

For RNA variant calling a custom workflow was provided by the Qiagen bioinformatics support. This workflow worked with UMIs, mapped the reads on the human genome hg38 and filtered with specific parameters for rare RNA variant calling.

For differential gene expression analysis between the groups of drug-resistant variants and their respective parental cell line, the differential expression tools of the CLC GWB were used and changes with a Bonferroni corrected *p*-value < 0.05 were considered significant.

For hierarchical clustering analysis the tool for creating heatmaps of the CLC GWB was used with Euclidean distance and complete linkage.

### 4.5. SNP Genotyping by Real-Time PCR

In total, 24 of the 28 selected polymorphisms were validated by real-time genotyping PCR ([Table A2](#ijms-23-11787-t0A2)). TaqMan SNP genotyping assays (Thermo Fisher Scientific by Life Technologies Italia, Monza, Italy) or drug metabolizing enzymes (DMEs) assays, which had functionally been tested, were used to validate the performance of the mmNGS approach.

The genotyping experiments were performed according to standard protocols using 10 ng DNA as input material using the VIIA 7 DX realtime PCR system (Thermo Fisher Scientific by Life Technologies Italia, Monza, Italy) and the results were analyzed with the TaqMan Genotyper software (Thermo Fisher Scientific by Life Technologies Italia, Monza, Italy), which generated allelic discrimination cluster plots to determine the genotype of each SNP.

## Acknowledgments

Research activities have been conducted by using resources and equipment of the Laboratory of Experimental Oncology, where C.C. and L.F. are also executing their ongoing Ph.D. research projects. We thank the technical support provided by the R&D and bioinformatic specialists of Qiagen.

## Abbreviations

| ABCB1 | ATP Binding Cassette Subfamily B Member 1 |
| --- | --- |
| ABCC2 | ATP Binding Cassette Subfamily C Member 2 |
| ABCC3 | ATP Binding Cassette Subfamily C Member 3 |
| ACYP2 | Acylphosphatase 2 |
| ATM | ATM Serine/Threonine Kinase |
| ATR | ATR Serine/Threonine Kinase |
| CDDP | Cisplatin |
| CLC GWB | CLC Genomics Workbench |
| COMT | Cathechol-O-Methyltransferase |
| ERCC1 | Excision Repair Cross-Complementing 1, Endonuclease Non-Catalytic Subunit |
| ERCC2 | Excision Repair Cross-Complementing 2, TFIIH Core Complex Helicase Subunit |
| GSTP1 | Glutathione S-Transferase Pi 1 |
| HGOS | High-Grade Osteosarcoma |
| IMDM | Iscove’s modified Dulbecco’s medium |
| mmNGS | Multimodal Targeted Next Generation Sequencing |
| MNV | Multi Nucleotide Variant |
| NER | Nucleotide Excision Repair |
| SNP | Single Nucleotide Polymorphism |
| TP53 | Tumor Protein P53 |
| TPM | Transcripts per Million |
| TPMT | Thiopurine S-Methyltransferase |
| UMI | Unique Molecular Indices |
| VAF | Variant Allele Frequency |
| XPA | XPA, DNA Damage Recognition and Repair Factor |
| XRCC3 | X-Ray Repair Cross Complementing 3 |
## Appendix A

### Table A1.

| Gene Name | Abbreviation | Function |
| --- | --- | --- |
| ATP Binding Cassette Subfamily B Member 1 | ABCB1 | CDDP transport |
| ATP Binding Cassette Subfamily C Member 2 | ABCC2 | CDDP transport |
| ATP Binding Cassette Subfamily C Member 3 | ABCC3 | CDDP-related ototoxicity |
| Acylphosphatase 2 | ACYP2 | CDDP-related ototoxicity |
| ATM Serine/Threonine Kinase | ATM | DNA repair |
| ATR Serine/Threonine Kinase | ATR | DNA repair |
| Cathecol-O-Methyltransferase | COMT | CDDP-related ototoxicity |
| Excision Repair Cross-Complementing 1 | ERCC1 | DNA repair |
| Excision Repair Cross-Complementing 2 | ERCC2 | DNA repair |
| Glutathione S-Transferase P1 | GSTP1 | CDDP detoxification |
| Tumor Protein 53 | TP53 | Genomic stability |
| Thiopurine S-methyltransferase | TPMT | CDDP-related ototoxicity |
| Xeroderma pigmentosum group A | XPA | DNA repair |
| X-Ray Repair Cross Complementing 3 | XRCC3 | DNA repair |

Table Caption: List of genes and their main function selected for this study.

### Table A2.

| Gene_Reference Number | Assay ID | Assay Type | Reference Allele |
| --- | --- | --- | --- |
| ABCB1_rs1045642 | C___7586657_20 | DME | A |
| ABCB1_rs2032582 | C_11711720C_30 + C_11711720D_40 | DME | A |
| ABCB1_rs1128503 | C___7586662_10 | DME | A |
| ABCC2_rs717620 | C___2814642_10 | DME | T |
| ABCC2_rs2273697 | C__22272980_20 | DME | C |
| ABCC2_rs3740066 | C__11214910_20 | DME | G |
| ABCC2_rs17222723 | C__25591743_30 | DME | T |
| ABCC3_rs4793665 | Not done | Not done | C |
| ABCC3_rs1051640 | Not done | Not done | A |
| ACYP2_rs1872328 | C__11643398_10 | Funct. tested | G |
| ATM_rs664677 | C___2283171_1_ | Validated | C |
| ATM_rs664143 | C___1039783_10 | Validated | A |
| ATR_rs2229032 | C__26021082_10 | Funct. tested | C |
| ATR_rs2227928 | C____157700_10 | Validated | A |
| COMT_rs4646316 | C__29193982_10 | Funct. tested | C |
| COMT_rs9332377 | Not done | Not done | C |
| ERCC1_rs11615 | C___2532959_1_ | Validated | A |
| ERCC1_rs3212986 | Not done | Not done | C |
| ERCC2_rs13181 | C___3145033_10 | Validated | T |
| ERCC2_rs1799793 | C___3145050_10 | Funct. tested | C |
| GSTP1_rs1695 | C___3237198_20 | DME | A |
| TP53_rs1042522 | C___2403545_10 | Funct. tested | G |
| TP53_rs1642785 | C___2880090_10 | Funct. tested | G |
| TPMT_rs12201199 | C__31923406_10 | Funct. tested | A |
| TPMT_rs1142345 | C_____19567_20 | DME | T |
| TPMT_rs1800460 | C__30634116_20 | DME | C |
| XPA_rs1800975 | C____482935_1_ | Validated | T |
| XRCC3_rs861539 | C___8901525_10 | Funct. tested | G |

Table Caption: Assays used for TaqMan genotyping and reference allele of multimodal targeted next generation sequencing (mmNGS) for variant interpretation.

## Author Contributions

Conceptualization, C.M.H. and M.S.; methodology, C.M.H., C.C., M.P.P., S.L. and L.F.; software for data analysis, C.M.H. and C.C.; validation, C.M.H., C.C., M.P.P., S.L. and L.F.; formal analysis, C.M.H., C.C. and M.P.P.; writing—review and editing, all Authors; visualization, C.M.H. and C.C.; supervision, M.S., K.S. and T.I.; resources and facilities availability, K.S.; project administration, M.S.; funding acquisition, M.S. All authors have read and agreed to the published version of the manuscript.

## Institutional Review Board Statement

Not applicable since this study does not involve humans or animals.

## Informed Consent Statement

Not applicable since this study does not involve humans.

## Data Availability Statement

The data generated and analyzed in this study contained within the article or supplementary material will be made available by the corresponding author to qualified researchers, upon justified request.

## Conflicts of Interest

The authors declare no conflict of interest. The funders had no role in the design of the study; in the collection, analyses, or interpretation of data; in the writing of the manuscript; or in the decision to publish the results.

## Funding Statement

This research and the APC were funded by the Foundation AIRC for Cancer Research (project AIRC IG2018-ID.21487-P.I. Serra Massimo).

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

The data generated and analyzed in this study contained within the article or supplementary material will be made available by the corresponding author to qualified researchers, upon justified request.

### Data Availability Statement

The data generated and analyzed in this study contained within the article or supplementary material will be made available by the corresponding author to qualified researchers, upon justified request.

## References

1. Harrison D.J., Geller D.S., Gill J.D., Lewis V.O., Gorlick R. Current and future therapeutic approaches for osteosarcoma. Expert Rev. Anticancer Ther. 2018;18:39–50. doi: 10.1080/14737140.2018.1413939.  [DOI](https://doi.org/10.1080/14737140.2018.1413939) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29210294/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Expert%20Rev.%20Anticancer%20Ther.&title=Current%20and%20future%20therapeutic%20approaches%20for%20osteosarcoma&author=D.J.%20Harrison&author=D.S.%20Geller&author=J.D.%20Gill&author=V.O.%20Lewis&author=R.%20Gorlick&volume=18&publication_year=2018&pages=39-50&pmid=29210294&doi=10.1080/14737140.2018.1413939&)

2. Lilienthal I., Herold N. Targeting Molecular Mechanisms Underlying Treatment Efficacy and Resistance in Osteosarcoma: A Review of Current and Future Strategies. Int. J. Mol. Sci. 2020;21:6885. doi: 10.3390/ijms21186885.  [DOI](https://doi.org/10.3390/ijms21186885) | [PMC free article](/articles/PMC7555161/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32961800/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Targeting%20Molecular%20Mechanisms%20Underlying%20Treatment%20Efficacy%20and%20Resistance%20in%20Osteosarcoma:%20A%20Review%20of%20Current%20and%20Future%20Strategies&author=I.%20Lilienthal&author=N.%20Herold&volume=21&publication_year=2020&pages=6885&pmid=32961800&doi=10.3390/ijms21186885&)

3. Hattinger C.M., Patrizio M.P., Luppi S., Serra M. Pharmacogenomics and Pharmacogenetics in Osteosarcoma: Translational Studies and Clinical Impact. Int. J. Mol. Sci. 2020;21:4659. doi: 10.3390/ijms21134659.  [DOI](https://doi.org/10.3390/ijms21134659) | [PMC free article](/articles/PMC7369799/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32629971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Pharmacogenomics%20and%20Pharmacogenetics%20in%20Osteosarcoma:%20Translational%20Studies%20and%20Clinical%20Impact&author=C.M.%20Hattinger&author=M.P.%20Patrizio&author=S.%20Luppi&author=M.%20Serra&volume=21&publication_year=2020&pages=4659&pmid=32629971&doi=10.3390/ijms21134659&)

4. Roden D.M., McLeod H.L., Relling M.V., Williams M.S., Mensah G.A., Peterson J.F., Van Driest S.L. Pharmacogenomics. Lancet. 2019;394:521–532. doi: 10.1016/S0140-6736(19)31276-0.  [DOI](https://doi.org/10.1016/S0140-6736(19)31276-0) | [PMC free article](/articles/PMC6707519/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31395440/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Pharmacogenomics&author=D.M.%20Roden&author=H.L.%20McLeod&author=M.V.%20Relling&author=M.S.%20Williams&author=G.A.%20Mensah&volume=394&publication_year=2019&pages=521-532&pmid=31395440&doi=10.1016/S0140-6736(19)31276-0&)

5. Kager L., Tamamyan G., Bielack S. Novel insights and therapeutic interventions for pediatric osteosarcoma. Future Oncol. 2017;13:357–368. doi: 10.2217/fon-2016-0261.  [DOI](https://doi.org/10.2217/fon-2016-0261) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27651036/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Future%20Oncol.&title=Novel%20insights%20and%20therapeutic%20interventions%20for%20pediatric%20osteosarcoma&author=L.%20Kager&author=G.%20Tamamyan&author=S.%20Bielack&volume=13&publication_year=2017&pages=357-368&pmid=27651036&doi=10.2217/fon-2016-0261&)

6. Serra M., Hattinger C.M. The pharmacogenomics of osteosarcoma. Pharmacogenomics J. 2017;17:11–20. doi: 10.1038/tpj.2016.45.  [DOI](https://doi.org/10.1038/tpj.2016.45) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27241064/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=The%20pharmacogenomics%20of%20osteosarcoma&author=M.%20Serra&author=C.M.%20Hattinger&volume=17&publication_year=2017&pages=11-20&pmid=27241064&doi=10.1038/tpj.2016.45&)

7. Vos H.I., Guchelaar H.J., Gelderblom H., de Bont E.S., Kremer L.C., Naber A.M., Hakobjan M.H., van der Graaf W.T., Coenen M.J., te Loo D.M. Replication of a genetic variant in ACYP2 associated with cisplatin-induced hearing loss in patients with osteosarcoma. Pharmacogenet. Genomics. 2016;26:243–247. doi: 10.1097/FPC.0000000000000212.  [DOI](https://doi.org/10.1097/FPC.0000000000000212) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26928270/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Replication%20of%20a%20genetic%20variant%20in%20ACYP2%20associated%20with%20cisplatin-induced%20hearing%20loss%20in%20patients%20with%20osteosarcoma&author=H.I.%20Vos&author=H.J.%20Guchelaar&author=H.%20Gelderblom&author=E.S.%20de%20Bont&author=L.C.%20Kremer&volume=26&publication_year=2016&pages=243-247&pmid=26928270&doi=10.1097/FPC.0000000000000212&)

8. Windsor R.E., Strauss S.J., Kallis C., Wood N.E., Whelan J.S. Germline genetic polymorphisms may influence chemotherapy response and disease outcome in osteosarcoma: A pilot study. Cancer. 2012;118:1856–1867. doi: 10.1002/cncr.26472.  [DOI](https://doi.org/10.1002/cncr.26472) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21887680/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer&title=Germline%20genetic%20polymorphisms%20may%20influence%20chemotherapy%20response%20and%20disease%20outcome%20in%20osteosarcoma:%20A%20pilot%20study&author=R.E.%20Windsor&author=S.J.%20Strauss&author=C.%20Kallis&author=N.E.%20Wood&author=J.S.%20Whelan&volume=118&publication_year=2012&pages=1856-1867&pmid=21887680&doi=10.1002/cncr.26472&)

9. Ross C.J., Katzov-Eckert H., Dube M.P., Brooks B., Rassekh S.R., Barhdadi A., Feroz-Zada Y., Visscher H., Brown A.M., Rieder M.J., et al. Genetic variants in TPMT and COMT are associated with hearing loss in children receiving cisplatin chemotherapy. Nat. Genet. 2009;41:1345–1349. doi: 10.1038/ng.478.  [DOI](https://doi.org/10.1038/ng.478) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19898482/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Genetic%20variants%20in%20TPMT%20and%20COMT%20are%20associated%20with%20hearing%20loss%20in%20children%20receiving%20cisplatin%20chemotherapy&author=C.J.%20Ross&author=H.%20Katzov-Eckert&author=M.P.%20Dube&author=B.%20Brooks&author=S.R.%20Rassekh&volume=41&publication_year=2009&pages=1345-1349&pmid=19898482&doi=10.1038/ng.478&)

10. Pussegoda K., Ross C.J., Visscher H., Yazdanpanah M., Brooks B., Rassekh S.R., Zada Y.F., Dube M.P., Carleton B.C., Hayden M.R., et al. Replication of TPMT and ABCC3 genetic variants highly associated with cisplatin-induced hearing loss in children. Clin. Pharmacol. Ther. 2013;94:243–251. doi: 10.1038/clpt.2013.80.  [DOI](https://doi.org/10.1038/clpt.2013.80) | [PMC free article](/articles/PMC4006820/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23588304/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Replication%20of%20TPMT%20and%20ABCC3%20genetic%20variants%20highly%20associated%20with%20cisplatin-induced%20hearing%20loss%20in%20children&author=K.%20Pussegoda&author=C.J.%20Ross&author=H.%20Visscher&author=M.%20Yazdanpanah&author=B.%20Brooks&volume=94&publication_year=2013&pages=243-251&pmid=23588304&doi=10.1038/clpt.2013.80&)

11. Hagleitner M.M., Coenen M.J., Patino-Garcia A., de Bont E.S., Gonzalez-Neira A., Vos H.I., van Leeuwen F.N., Gelderblom H., Hoogerbrugge P.M., Guchelaar H.J., et al. Influence of genetic variants in TPMT and COMT associated with cisplatin induced hearing loss in patients with cancer: Two new cohorts and a meta-analysis reveal significant heterogeneity between cohorts. PLoS ONE. 2014;9:e115869. doi: 10.1371/journal.pone.0115869.  [DOI](https://doi.org/10.1371/journal.pone.0115869) | [PMC free article](/articles/PMC4281251/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25551397/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&title=Influence%20of%20genetic%20variants%20in%20TPMT%20and%20COMT%20associated%20with%20cisplatin%20induced%20hearing%20loss%20in%20patients%20with%20cancer:%20Two%20new%20cohorts%20and%20a%20meta-analysis%20reveal%20significant%20heterogeneity%20between%20cohorts&author=M.M.%20Hagleitner&author=M.J.%20Coenen&author=A.%20Patino-Garcia&author=E.S.%20de%20Bont&author=A.%20Gonzalez-Neira&volume=9&publication_year=2014&pages=e115869&pmid=25551397&doi=10.1371/journal.pone.0115869&)

12. Pasello M., Michelacci F., Scionti I., Hattinger C.M., Zuntini M., Caccuri A.M., Scotlandi K., Picci P., Serra M. Overcoming glutathione S-transferase P1-related cisplatin resistance in osteosarcoma. Cancer Res. 2008;68:6661–6668. doi: 10.1158/0008-5472.CAN-07-5840.  [DOI](https://doi.org/10.1158/0008-5472.CAN-07-5840) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18701490/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&title=Overcoming%20glutathione%20S-transferase%20P1-related%20cisplatin%20resistance%20in%20osteosarcoma&author=M.%20Pasello&author=F.%20Michelacci&author=I.%20Scionti&author=C.M.%20Hattinger&author=M.%20Zuntini&volume=68&publication_year=2008&pages=6661-6668&pmid=18701490&doi=10.1158/0008-5472.CAN-07-5840&)

13. Li J.Z., Tian Z.Q., Jiang S.N., Feng T. Effect of variation of ABCB1 and GSTP1 on osteosarcoma survival after chemotherapy. Genet. Mol. Res. 2014;13:3186–3192. doi: 10.4238/2014.April.25.3.  [DOI](https://doi.org/10.4238/2014.April.25.3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24841650/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Mol.%20Res.&title=Effect%20of%20variation%20of%20ABCB1%20and%20GSTP1%20on%20osteosarcoma%20survival%20after%20chemotherapy&author=J.Z.%20Li&author=Z.Q.%20Tian&author=S.N.%20Jiang&author=T.%20Feng&volume=13&publication_year=2014&pages=3186-3192&pmid=24841650&doi=10.4238/2014.April.25.3&)

14. Liu S., Yi Z., Ling M., Shi J., Qiu Y., Yang S. Predictive potential of ABCB1, ABCC3, and GSTP1 gene polymorphisms on osteosarcoma survival after chemotherapy. Tumour Biol. 2014;35:9897–9904. doi: 10.1007/s13277-014-1917-x.  [DOI](https://doi.org/10.1007/s13277-014-1917-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24996541/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumour%20Biol.&title=Predictive%20potential%20of%20ABCB1,%20ABCC3,%20and%20GSTP1%20gene%20polymorphisms%20on%20osteosarcoma%20survival%20after%20chemotherapy&author=S.%20Liu&author=Z.%20Yi&author=M.%20Ling&author=J.%20Shi&author=Y.%20Qiu&volume=35&publication_year=2014&pages=9897-9904&pmid=24996541&doi=10.1007/s13277-014-1917-x&)

15. Teng J.W., Yang Z.M., Li J., Xu B. Predictive role of Glutathione S-transferases (GSTs) on the prognosis of osteosarcoma patients treated with chemotherapy. Pak. J. Med. Sci. 2013;29:1182–1186. doi: 10.12669/pjms.295.3870.  [DOI](https://doi.org/10.12669/pjms.295.3870) | [PMC free article](/articles/PMC3858940/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24353716/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pak.%20J.%20Med.%20Sci.&title=Predictive%20role%20of%20Glutathione%20S-transferases%20(GSTs)%20on%20the%20prognosis%20of%20osteosarcoma%20patients%20treated%20with%20chemotherapy&author=J.W.%20Teng&author=Z.M.%20Yang&author=J.%20Li&author=B.%20Xu&volume=29&publication_year=2013&pages=1182-1186&pmid=24353716&doi=10.12669/pjms.295.3870&)

16. Zhang S.L., Mao N.F., Sun J.Y., Shi Z.C., Wang B., Sun Y.J. Predictive potential of glutathione S-transferase polymorphisms for prognosis of osteosarcoma patients on chemotherapy. Asian Pac. J. Cancer Prev. 2012;13:2705–2709. doi: 10.7314/APJCP.2012.13.6.2705.  [DOI](https://doi.org/10.7314/APJCP.2012.13.6.2705) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22938445/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asian%20Pac.%20J.%20Cancer%20Prev.&title=Predictive%20potential%20of%20glutathione%20S-transferase%20polymorphisms%20for%20prognosis%20of%20osteosarcoma%20patients%20on%20chemotherapy&author=S.L.%20Zhang&author=N.F.%20Mao&author=J.Y.%20Sun&author=Z.C.%20Shi&author=B.%20Wang&volume=13&publication_year=2012&pages=2705-2709&pmid=22938445&doi=10.7314/APJCP.2012.13.6.2705&)

17. Yang L.M., Li X.H., Bao C.F. Glutathione S-transferase P1 and DNA polymorphisms influence response to chemotherapy and prognosis of bone tumors. Asian Pac. J. Cancer Prev. 2012;13:5883–5886. doi: 10.7314/APJCP.2012.13.11.5883.  [DOI](https://doi.org/10.7314/APJCP.2012.13.11.5883) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23317281/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asian%20Pac.%20J.%20Cancer%20Prev.&title=Glutathione%20S-transferase%20P1%20and%20DNA%20polymorphisms%20influence%20response%20to%20chemotherapy%20and%20prognosis%20of%20bone%20tumors&author=L.M.%20Yang&author=X.H.%20Li&author=C.F.%20Bao&volume=13&publication_year=2012&pages=5883-5886&pmid=23317281&doi=10.7314/APJCP.2012.13.11.5883&)

18. Goricar K., Kovac V., Jazbec J., Zakotnik B., Lamovec J., Dolzan V. Genetic variability of DNA repair mechanisms and glutathione-S-transferase genes influences treatment outcome in osteosarcoma. Cancer Epidemiol. 2015;39:182–188. doi: 10.1016/j.canep.2014.12.009.  [DOI](https://doi.org/10.1016/j.canep.2014.12.009) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25592234/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol.&title=Genetic%20variability%20of%20DNA%20repair%20mechanisms%20and%20glutathione-S-transferase%20genes%20influences%20treatment%20outcome%20in%20osteosarcoma&author=K.%20Goricar&author=V.%20Kovac&author=J.%20Jazbec&author=B.%20Zakotnik&author=J.%20Lamovec&volume=39&publication_year=2015&pages=182-188&pmid=25592234&doi=10.1016/j.canep.2014.12.009&)

19. Cao Z.H., Yin H.P., Jiang N., Yu B. Association between ERCC1 and ERCC2 gene polymorphisms and chemotherapy response and overall survival in osteosarcoma. Genet. Mol. Res. 2015;14:10145–10151. doi: 10.4238/2015.August.21.21.  [DOI](https://doi.org/10.4238/2015.August.21.21) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26345951/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Mol.%20Res.&title=Association%20between%20ERCC1%20and%20ERCC2%20gene%20polymorphisms%20and%20chemotherapy%20response%20and%20overall%20survival%20in%20osteosarcoma&author=Z.H.%20Cao&author=H.P.%20Yin&author=N.%20Jiang&author=B.%20Yu&volume=14&publication_year=2015&pages=10145-10151&pmid=26345951&doi=10.4238/2015.August.21.21&)

20. Ji W.P., He N.B. Investigation on the DNA repaired gene polymorphisms and response to chemotherapy and overall survival of osteosarcoma. Int. J. Clin. Exp. Pathol. 2015;8:894–899.  [PMC free article](/articles/PMC4348835/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25755792/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Clin.%20Exp.%20Pathol.&title=Investigation%20on%20the%20DNA%20repaired%20gene%20polymorphisms%20and%20response%20to%20chemotherapy%20and%20overall%20survival%20of%20osteosarcoma&author=W.P.%20Ji&author=N.B.%20He&volume=8&publication_year=2015&pages=894-899&pmid=25755792&)

21. Sun Y., Wu Y., Li W., Kong Z., Zou X. Genetic polymorphisms in nucleotide excision repair pathway influences response to chemotherapy and overall survival in osteosarcoma. Int. J. Clin. Exp. Pathol. 2015;8:7905–7912.  [PMC free article](/articles/PMC4555683/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26339355/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Clin.%20Exp.%20Pathol.&title=Genetic%20polymorphisms%20in%20nucleotide%20excision%20repair%20pathway%20influences%20response%20to%20chemotherapy%20and%20overall%20survival%20in%20osteosarcoma&author=Y.%20Sun&author=Y.%20Wu&author=W.%20Li&author=Z.%20Kong&author=X.%20Zou&volume=8&publication_year=2015&pages=7905-7912&pmid=26339355&)

22. Zhang H., Ge J., Hong H., Bi L., Sun Z. Genetic polymorphisms in ERCC1 and ERCC2 genes are associated with response to chemotherapy in osteosarcoma patients among Chinese population: A meta-analysis. World J. Surg. Oncol. 2017;15:75. doi: 10.1186/s12957-017-1142-3.  [DOI](https://doi.org/10.1186/s12957-017-1142-3) | [PMC free article](/articles/PMC5383995/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28388903/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=World%20J.%20Surg.%20Oncol.&title=Genetic%20polymorphisms%20in%20ERCC1%20and%20ERCC2%20genes%20are%20associated%20with%20response%20to%20chemotherapy%20in%20osteosarcoma%20patients%20among%20Chinese%20population:%20A%20meta-analysis&author=H.%20Zhang&author=J.%20Ge&author=H.%20Hong&author=L.%20Bi&author=Z.%20Sun&volume=15&publication_year=2017&pages=75&pmid=28388903&doi=10.1186/s12957-017-1142-3&)

23. Zhang Q., Lv L.Y., Li B.J., Zhang J., Wei F. Investigation of ERCC1 and ERCC2 gene polymorphisms and response to chemotherapy and overall survival in osteosarcoma. Genet. Mol. Res. 2015;14:11235–11241. doi: 10.4238/2015.September.22.17.  [DOI](https://doi.org/10.4238/2015.September.22.17) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26400354/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Mol.%20Res.&title=Investigation%20of%20ERCC1%20and%20ERCC2%20gene%20polymorphisms%20and%20response%20to%20chemotherapy%20and%20overall%20survival%20in%20osteosarcoma&author=Q.%20Zhang&author=L.Y.%20Lv&author=B.J.%20Li&author=J.%20Zhang&author=F.%20Wei&volume=14&publication_year=2015&pages=11235-11241&pmid=26400354&doi=10.4238/2015.September.22.17&)

24. Hao T., Feng W., Zhang J., Sun Y.J., Wang G. Association of four ERCC1 and ERCC2 SNPs with survival of bone tumour patients. Asian Pac. J. Cancer Prev. 2012;13:3821–3824. doi: 10.7314/APJCP.2012.13.8.3821.  [DOI](https://doi.org/10.7314/APJCP.2012.13.8.3821) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23098477/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Asian%20Pac.%20J.%20Cancer%20Prev.&title=Association%20of%20four%20ERCC1%20and%20ERCC2%20SNPs%20with%20survival%20of%20bone%20tumour%20patients&author=T.%20Hao&author=W.%20Feng&author=J.%20Zhang&author=Y.J.%20Sun&author=G.%20Wang&volume=13&publication_year=2012&pages=3821-3824&pmid=23098477&doi=10.7314/APJCP.2012.13.8.3821&)

25. Wang M.J., Zhu Y., Guo X.J., Tian Z.Z. Genetic variability of genes involved in DNA repair influence treatment outcome in osteosarcoma. Genet. Mol. Res. 2015;14:11652–11657. doi: 10.4238/2015.September.28.17.  [DOI](https://doi.org/10.4238/2015.September.28.17) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26436406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Mol.%20Res.&title=Genetic%20variability%20of%20genes%20involved%20in%20DNA%20repair%20influence%20treatment%20outcome%20in%20osteosarcoma&author=M.J.%20Wang&author=Y.%20Zhu&author=X.J.%20Guo&author=Z.Z.%20Tian&volume=14&publication_year=2015&pages=11652-11657&pmid=26436406&doi=10.4238/2015.September.28.17&)

26. Obiedat H., Alrabadi N., Sultan E., Al Shatti M., Zihlif M. The effect of ERCC1 and ERCC2 gene polymorphysims on response to cisplatin based therapy in osteosarcoma patients. BMC Med. Genet. 2018;19:112. doi: 10.1186/s12881-018-0627-4.  [DOI](https://doi.org/10.1186/s12881-018-0627-4) | [PMC free article](/articles/PMC6035436/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29980176/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Med.%20Genet.&title=The%20effect%20of%20ERCC1%20and%20ERCC2%20gene%20polymorphysims%20on%20response%20to%20cisplatin%20based%20therapy%20in%20osteosarcoma%20patients&author=H.%20Obiedat&author=N.%20Alrabadi&author=E.%20Sultan&author=M.%20Al%20Shatti&author=M.%20Zihlif&volume=19&publication_year=2018&pages=112&pmid=29980176&doi=10.1186/s12881-018-0627-4&)

27. Caronia D., Patino-Garcia A., Milne R.L., Zalacain-Diez M., Pita G., Alonso M.R., Moreno L.T., Sierrasesumaga-Ariznabarreta L., Benitez J., Gonzalez-Neira A. Common variations in ERCC2 are associated with response to cisplatin chemotherapy and clinical outcome in osteosarcoma patients. Pharmacogenomics J. 2009;9:347–353. doi: 10.1038/tpj.2009.19.  [DOI](https://doi.org/10.1038/tpj.2009.19) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19434073/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Common%20variations%20in%20ERCC2%20are%20associated%20with%20response%20to%20cisplatin%20chemotherapy%20and%20clinical%20outcome%20in%20osteosarcoma%20patients&author=D.%20Caronia&author=A.%20Patino-Garcia&author=R.L.%20Milne&author=M.%20Zalacain-Diez&author=G.%20Pita&volume=9&publication_year=2009&pages=347-353&pmid=19434073&doi=10.1038/tpj.2009.19&)

28. Biason P., Hattinger C.M., Innocenti F., Talamini R., Alberghini M., Scotlandi K., Zanusso C., Serra M., Toffoli G. Nucleotide excision repair gene variants and association with survival in osteosarcoma patients treated with neoadjuvant chemotherapy. Pharmacogenomics J. 2012;12:476–483. doi: 10.1038/tpj.2011.33.  [DOI](https://doi.org/10.1038/tpj.2011.33) | [PMC free article](/articles/PMC3935514/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21826087/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J.&title=Nucleotide%20excision%20repair%20gene%20variants%20and%20association%20with%20survival%20in%20osteosarcoma%20patients%20treated%20with%20neoadjuvant%20chemotherapy&author=P.%20Biason&author=C.M.%20Hattinger&author=F.%20Innocenti&author=R.%20Talamini&author=M.%20Alberghini&volume=12&publication_year=2012&pages=476-483&pmid=21826087&doi=10.1038/tpj.2011.33&)

29. Li J., Liu S., Wang W., Zhang K., Liu Z., Zhang C., Chen S., Wu S. ERCC polymorphisms and prognosis of patients with osteosarcoma. Tumour Biol. 2014;35:10129–10136. doi: 10.1007/s13277-014-2322-1.  [DOI](https://doi.org/10.1007/s13277-014-2322-1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25023406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumour%20Biol.&title=ERCC%20polymorphisms%20and%20prognosis%20of%20patients%20with%20osteosarcoma&author=J.%20Li&author=S.%20Liu&author=W.%20Wang&author=K.%20Zhang&author=Z.%20Liu&volume=35&publication_year=2014&pages=10129-10136&pmid=25023406&doi=10.1007/s13277-014-2322-1&)

30. Pasqui A., Boddi A., Campanacci D.A., Scoccianti G., Bernini A., Grasso D., Gambale E., Scolari F., Palchetti I., Palomba A., et al. Alteration of the Nucleotide Excision Repair (NER) Pathway in Soft Tissue Sarcoma. Int. J. Mol. Sci. 2022;23:8360. doi: 10.3390/ijms23158360.  [DOI](https://doi.org/10.3390/ijms23158360) | [PMC free article](/articles/PMC9369086/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35955506/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Alteration%20of%20the%20Nucleotide%20Excision%20Repair%20(NER)%20Pathway%20in%20Soft%20Tissue%20Sarcoma&author=A.%20Pasqui&author=A.%20Boddi&author=D.A.%20Campanacci&author=G.%20Scoccianti&author=A.%20Bernini&volume=23&publication_year=2022&pages=8360&pmid=35955506&doi=10.3390/ijms23158360&)

31. Peissert S., Sauer F., Grabarczyk D.B., Braun C., Sander G., Poterszman A., Egly J.M., Kuper J., Kisker C. In TFIIH the Arch domain of XPD is mechanistically essential for transcription and DNA repair. Nat. Commun. 2020;11:1667. doi: 10.1038/s41467-020-15241-9.  [DOI](https://doi.org/10.1038/s41467-020-15241-9) | [PMC free article](/articles/PMC7125077/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32245994/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&title=In%20TFIIH%20the%20Arch%20domain%20of%20XPD%20is%20mechanistically%20essential%20for%20transcription%20and%20DNA%20repair&author=S.%20Peissert&author=F.%20Sauer&author=D.B.%20Grabarczyk&author=C.%20Braun&author=G.%20Sander&volume=11&publication_year=2020&pages=1667&pmid=32245994&doi=10.1038/s41467-020-15241-9&)

32. Kiss R.C., Xia F., Acklin S. Targeting DNA Damage Response and Repair to Enhance Therapeutic Index in Cisplatin-Based Cancer Treatment. Int. J. Mol. Sci. 2021;22:8199. doi: 10.3390/ijms22158199.  [DOI](https://doi.org/10.3390/ijms22158199) | [PMC free article](/articles/PMC8347825/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34360968/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&title=Targeting%20DNA%20Damage%20Response%20and%20Repair%20to%20Enhance%20Therapeutic%20Index%20in%20Cisplatin-Based%20Cancer%20Treatment&author=R.C.%20Kiss&author=F.%20Xia&author=S.%20Acklin&volume=22&publication_year=2021&pages=8199&pmid=34360968&doi=10.3390/ijms22158199&)

33. Rocha C.R.R., Silva M.M., Quinet A., Cabral-Neto J.B., Menck C.F.M. DNA repair pathways and cisplatin resistance: An intimate relationship. Clinics. 2018;73:e478s. doi: 10.6061/clinics/2018/e478s.  [DOI](https://doi.org/10.6061/clinics/2018/e478s) | [PMC free article](/articles/PMC6113849/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30208165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clinics&title=DNA%20repair%20pathways%20and%20cisplatin%20resistance:%20An%20intimate%20relationship&author=C.R.R.%20Rocha&author=M.M.%20Silva&author=A.%20Quinet&author=J.B.%20Cabral-Neto&author=C.F.M.%20Menck&volume=73&publication_year=2018&pages=e478s&pmid=30208165&doi=10.6061/clinics/2018/e478s&)

34. Benini S., Baldini N., Manara M.C., Chano T., Serra M., Rizzi S., Lollini P.L., Picci P., Scotlandi K. Redundancy of autocrine loops in human osteosarcoma cells. Int. J. Cancer. 1999;80:581–588. doi: 10.1002/(SICI)1097-0215(19990209)80:4<581::AID-IJC16>3.0.CO;2-O.  [DOI](https://doi.org/10.1002/(SICI)1097-0215(19990209)80:4&#x0003c;581::AID-IJC16&#x0003e;3.0.CO;2-O) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9935160/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Cancer&title=Redundancy%20of%20autocrine%20loops%20in%20human%20osteosarcoma%20cells&author=S.%20Benini&author=N.%20Baldini&author=M.C.%20Manara&author=T.%20Chano&author=M.%20Serra&volume=80&publication_year=1999&pages=581-588&pmid=9935160&doi=10.1002/(SICI)1097-0215(19990209)80:4&#x0003c;581::AID-IJC16&#x0003e;3.0.CO;2-O&)
