# Chemotherapy-related hyperbilirubinemia in pediatric acute lymphoblastic leukemia: a genome-wide association study from the AIEOP-BFM ALL study group

## Metadata
**Authors:** Stefanie V Junk, Elke Schaeffeler, Martin Zimmermann, Anja Möricke, Rita Beier, Peter Schütte, Birthe Fedders, Julia Alten, Laura Hinze, Norman Klein, Andreas Kulozik, Martina U Muckenthaler, Rolf Koehler, Arndt Borkhardt, Jayaram Vijayakrishnan, David Ellinghaus, Michael Forster, Andre Franke, Astrid Wintering, Christian P Kratz, Martin Schrappe, Matthias Schwab, Richard S Houlston, Gunnar Cario, Martin Stanulla
**Journal:** Journal of Experimental & Clinical Cancer Research : CR
**Date:** 2023 Jan 13
**DOI:** [10.1186/s13046-022-02585-x](https://doi.org/10.1186/s13046-022-02585-x)
**PMID:** 36639636
**PMCID:** PMC9838013
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9838013/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9838013/pdf/13046_2022_Article_2585.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9838013/pdf/13046_2022_Article_2585.pdf)

## Abstract

**Background:** 
Characterization of clinical phenotypes in context with tumor and host genomic information can aid in the development of more effective and less toxic risk-adapted and targeted treatment strategies. To analyze the impact of therapy-related hyperbilirubinemia on treatment outcome and to identify contributing genetic risk factors of this well-recognized adverse effect we evaluated serum bilirubin levels in 1547 pediatric patients with acute lymphoblastic leukemia (ALL) and conducted a genome-wide association study (GWAS).

**Patients and methods:** 
Patients were treated in multicenter trial AIEOP-BFM ALL 2000 for pediatric ALL. Bilirubin toxicity was graded 0 to 4 according to the Common Toxicity Criteria (CTC) of the National Cancer Institute. In the GWAS discovery cohort, including 650 of the 1547 individuals, genotype frequencies of 745,895 single nucleotide variants were compared between 435 patients with hyperbilirubinemia (CTC grades 1-4) during induction/consolidation treatment and 215 patients without it (grade 0). Replication analyses included 224 patients from the same trial.

**Results:** 
Compared to patients with no (grade 0) or moderate hyperbilirubinemia (grades 1-2) during induction/consolidation, patients with grades 3-4 had a poorer 5-year event free survival (76.6 ± 3% versus 87.7 ± 1% for grades 1-2, P = 0.003; 85.2 ± 2% for grade 0, P < 0.001) and a higher cumulative incidence of relapse (15.6 ± 3% versus 9.0 ± 1% for grades 1-2, P = 0.08; 11.1 ± 1% for grade 0, P = 0.007). GWAS identified a strong association of the rs6744284 variant T allele in the UGT1A gene cluster with risk of hyperbilirubinemia (allelic odds ratio (OR) = 2.1, P = 7 × 10− 8). TT-homozygotes had a 6.5-fold increased risk of hyperbilirubinemia (grades 1-4; 95% confidence interval (CI) = 2.9-14.6, P = 7 × 10− 6) and a 16.4-fold higher risk of grade 3-4 hyperbilirubinemia (95% CI 6.1-43.8, P = 2 × 10− 8). Replication analyses confirmed these associations with joint analysis yielding genome-wide significance (allelic OR = 2.1, P = 6 × 10− 11; 95% CI 1.7-2.7). Moreover, rs6744284 genotypes were strongly linked to the Gilbert’s syndrome-associated UGT1A1*28/*37 allele (r2 = 0.70), providing functional support for study findings. Of clinical importance, the rs6744284 TT genotype counterbalanced the adverse prognostic impact of high hyperbilirubinemia on therapy outcome.

**Conclusions:** 
Chemotherapy-related hyperbilirubinemia is a prognostic factor for treatment outcome in pediatric ALL and genetic variation in UGT1A aids in predicting the clinical impact of hyperbilirubinemia.

**Trial registration:** 
http://www.clinicaltrials.gov; #NCT00430118.

**Supplementary Information:** 
The online version contains supplementary material available at 10.1186/s13046-022-02585-x.

Keywords: Childhood acute lymphoblastic leukemia, Treatment-related toxicity, Hepatotoxicity, Hyperbilirubinemia, UGT1A

### Background

Characterization of clinical phenotypes in context with tumor and host genomic information can aid in the development of more effective and less toxic risk-adapted and targeted treatment strategies. To analyze the impact of therapy-related hyperbilirubinemia on treatment outcome and to identify contributing genetic risk factors of this well-recognized adverse effect we evaluated serum bilirubin levels in 1547 pediatric patients with acute lymphoblastic leukemia (ALL) and conducted a genome-wide association study (GWAS).

### Patients and methods

Patients were treated in multicenter trial AIEOP-BFM ALL 2000 for pediatric ALL. Bilirubin toxicity was graded 0 to 4 according to the Common Toxicity Criteria (CTC) of the National Cancer Institute. In the GWAS discovery cohort, including 650 of the 1547 individuals, genotype frequencies of 745,895 single nucleotide variants were compared between 435 patients with hyperbilirubinemia (CTC grades 1-4) during induction/consolidation treatment and 215 patients without it (grade 0). Replication analyses included 224 patients from the same trial.

### Results

Compared to patients with no (grade 0) or moderate hyperbilirubinemia (grades 1-2) during induction/consolidation, patients with grades 3-4 had a poorer 5-year event free survival (76.6 ± 3% versus 87.7 ± 1% for grades 1-2, *P* = 0.003; 85.2 ± 2% for grade 0, *P* < 0.001) and a higher cumulative incidence of relapse (15.6 ± 3% versus 9.0 ± 1% for grades 1-2, *P* = 0.08; 11.1 ± 1% for grade 0, *P* = 0.007). GWAS identified a strong association of the rs6744284 variant T allele in the *UGT1A* gene cluster with risk of hyperbilirubinemia (allelic odds ratio (OR) = 2.1, *P* = 7 × 10^− 8^). TT-homozygotes had a 6.5-fold increased risk of hyperbilirubinemia (grades 1-4; 95% confidence interval (CI) = 2.9-14.6, *P* = 7 × 10^− 6^) and a 16.4-fold higher risk of grade 3-4 hyperbilirubinemia (95% CI 6.1-43.8, *P* = 2 × 10^− 8^). Replication analyses confirmed these associations with joint analysis yielding genome-wide significance (allelic OR = 2.1, *P* = 6 × 10^− 11^; 95% CI 1.7-2.7). Moreover, rs6744284 genotypes were strongly linked to the Gilbert’s syndrome-associated *UGT1A1**28/*37 allele (r^2^ = 0.70), providing functional support for study findings. Of clinical importance, the rs6744284 TT genotype counterbalanced the adverse prognostic impact of high hyperbilirubinemia on therapy outcome.

### Conclusions

Chemotherapy-related hyperbilirubinemia is a prognostic factor for treatment outcome in pediatric ALL and genetic variation in *UGT1A* aids in predicting the clinical impact of hyperbilirubinemia.

### Trial registration

[http://www.clinicaltrials.gov](https://www.clinicaltrials.gov/); #[NCT00430118](https://clinicaltrials.gov/ct2/show/NCT00430118).

### Supplementary Information

The online version contains supplementary material available at 10.1186/s13046-022-02585-x.

## Background

The systematic evaluation of tumor and host genomic information can help to identify new predictive markers and cancer vulnerabilities and lead to novel effective and less toxic risk-adapted and targeted treatment strategies [[1](#CR1)]. During the last five decades, treatment of pediatric patients with acute lymphoblastic leukemia (ALL) has significantly improved and is one of the success stories in clinical oncology [[2](#CR2)]. However, although most children diagnosed with ALL can be cured by contemporary clinical protocols [[2](#CR2), [3](#CR3)], benefits in survival come with acute and long-term adverse effects. This may complicate administration of therapy or impact on health and quality of life during and after completion of treatment [[2](#CR2), [3](#CR3)]. Thus, there is continuing medical need to improve risk assessment and tailored treatment for children and adolescents with ALL.

Hepatotoxicity, mainly captured in clinical trials by assessment of hyperbilirubinemia and transaminasemia, is a well-known complication frequently occurring in the early phases of ALL treatment [[4](#CR4), [5](#CR5)]. Surprisingly, detailed reports describing incidence and impact of hepatotoxicity on the overall outcome of pediatric ALL are lacking. A recent study including 1872 pediatric ALL patients treated according to the ALL IC-BFM 2002 protocol reported 934 grade 3 or above events of hepatotoxicity according to the National Cancer Institute (NCI) Common Toxicity Criteria (CTC; for details see [Suppl. Methods](#MOESM1) section) [[6](#CR6)] in 527 individuals (28%) during the entire therapy [[5](#CR5)]. A previous single institution study reported that event-free survival (EFS) was rarely influenced by hyperbilirubinemia, whereas treatment modifications including delays were common [[4](#CR4)]. Overall, 17% of patients in the latter study had bilirubin levels equal or above CTC grade 3 at least once in the course of therapy [[4](#CR4)]. Another study determined such high levels in 10, 2 and 15% of patients during induction, consolidation and maintenance phases, respectively [[7](#CR7)].

High total serum bilirubin levels – with or without transaminasemia – may indicate liver dysfunction upon exposure to various antileukemic drugs including L-asparaginase or antimetabolites [[8](#CR8)–[10](#CR10)]. However, in this context the biological basis of hyperbilirubinemia is poorly understood. In the last decade, genome-wide association studies (GWAS) enabled a refined risk assessment of many diseases – including pediatric ALL – by identifying risk-associated genetic variants [[11](#CR11)–[13](#CR13)].

In the present study, we evaluated the clinical impact of hyperbilirubinemia in a large cohort of pediatric ALL patients and applied a GWAS approach to identify genetic variants influencing chemotherapy-related hyperbilirubinemia.

## Methods

### Study individuals

Patients included were 1 to 18 years of age at diagnosis of ALL and enrolled in the German part of the European AIEOP-BFM ALL 2000 multicenter clinical trial for frontline treatment of pediatric ALL from August 1999 to November 2005 [[14](#CR14)–[16](#CR16)] (for treatment details see Suppl. Table [1](#MOESM1)). Primary patient selection criteria were the availability of bilirubin toxicity gradings for induction or consolidation – protocols IA/IB – and, for the assembly of our GWAS discovery cohort, the availability of genome-wide germline genotyping information (Suppl. Fig. [1](#MOESM1)) obtained during a previous project [[13](#CR13)]. When available, we also assessed bilirubin gradings related to subsequent treatment phases (extracompartment therapy, high-risk block treatment, and re-intensification protocols II and III). No information was available for maintenance treatment. For further details including statistical analyses, see [Suppl. Methods](#MOESM1) section.

### Toxicity definitions

As part of the routine safety management, toxicity was assessed for all treatment elements except for interim maintenance and maintenance phases. Considering 17.1 μmol/L as the upper normal limit (UNL), total bilirubin serum levels were graded according to the CTC of the NCI, version 2 [[6](#CR6)] (for details see [Suppl. Methods](#MOESM1) section).

### Genotyping

In the discovery cohort, DNA obtained from bone marrow samples in morphological remission was genotyped on Human Omni1-Quad v1 arrays (Illumina, San Diego, CA, USA) as previously described [[13](#CR13)]. Genotype information on rs6744284 for replication purposes was derived from a preceding analysis using Affymetrix Genome-wide Human SNP arrays 5.0 (Affymetrix, South San Francisco, CA, USA) [[17](#CR17)].

Out of the discovery cohort, we genotyped 544 patients with additional DNA available for *UGT1A1**28/*37 variations using a pre-developed KASP assay (LGC Biosearch Technologies, Hoddesdon, UK).

### Genome-wide association study

GWAS conduction, including data pruning, association testing with plink 1.9 ([www.cog-genomics.org/plink/1.9/](http://www.cog-genomics.org/plink/1.9/)), evaluation and plotting, was realized with an in-house developed R (3.6.0) script using RStudio (1.0.143). A strict quality control was performed prior to association testing. For details on data pruning, association testing, genotype imputation and further statistical analyses see [Suppl. Methods](#MOESM1) section.

## Results

### Clinical characterization of hyperbilirubinemia

Within the 1547 patients of the AIEOP ALL-BFM 2000 study population with available toxicity information, 540 (34.9%) had normal and 1007 (65.1%) had increased bilirubin levels during induction/consolidation (protocols IA/IB) (Table [1](#Tab1)). This included 825 (53.3%) patients with moderate hyperbilirubinemia (grades 1-2; 707 during induction, 575 during consolidation, 355 in both phases) and 182 (11.8%) with high hyperbilirubinemia (grades 3-4; 158 reports during induction, 50 during consolidation, 26 in both phases). Comparing patients without to patients with hyperbilirubinemia (grade 0 vs. grades 1-4), we noticed a larger proportion of older patients (*P* < 0.001) and more T cell ALL patients (*P* = 0.022) among those affected. The group of B cell ALL patients exhibiting hyperbilirubinemia contained fewer hyperdiploid patients (*P* = 0.002). However, no differences with regard to other genetic subgroups were observed (*ETV6-RUNX1, BCR-ABL1* and *KMT2A-AFF1*) (Table [1](#Tab1)). Considering the entire course of therapy, 245 patients (16.0%) had high hyperbilirubinemia in at least one treatment element. We determined a median time to protocol day 78 (after completion of induction and consolidation) of 89 ± 11 days (range 64-185 days), analyzing 1453 of 1547 individuals of our study population with available information. Compared to patients with moderate or no hyperbilirubinemia, patients with hyperbilirubinemia grades 3-4 experienced more therapy delays, requiring 91 days (range 64-174 days) to complete induction/consolidation vs. 88 days (range 70-154 days) for grades 1-2 and 89 days (range 70-185 days) for grade 0 (*P* = 0.002). No alterations of therapy in response to hyperbilirubinemia were noted.

### Table 1.

|   |   | Patients without hyperbilirubinemia |  | Patients with hyperbilirubinemia |  | Pa |
| --- | --- | --- | --- | --- | --- | --- |
|   |   | (n = 540) | (%) | (n = 1007) | (%) |  |
| Sex | Male | 300 | (56%) | 561 | (56%) |   |
| Female | 240 | (44%) | 446 | (44%) | 0.954 |  |
| Age at diagnosis of ALL [y] | < 6 | 372 | (69%) | 483 | (48%) |   |
| ≥6 < 10 | 93 | (17%) | 205 | (20%) |   |  |
| ≥10 | 75 | (14%) | 319 | (32%) | < 0.001 |  |
| Immunophenotype | B cell ALL | 461 | (85%) | 833 | (83%) |   |
| T cell ALL | 55 | (10%) | 146 | (14%) | 0.022 |  |
| Other/not characterizedb | 24 | (4%) | 28 | (3%) |   |  |
| White blood cell count at diagnosis of ALL [/μL] | < 10,000 | 261 | (48%) | 488 | (48%) |   |
| ≥10,000 < 50,000 | 187 | (35%) | 323 | (32%) |   |  |
| ≥50,000 < 100,000 | 52 | (10%) | 95 | (9%) |   |  |
| ≥100,000 | 40 | (7%) | 100 | (10%) | 0.364 |  |
| Unknown | 0 | (0%) | 1 | (0%) |   |  |
| CNS positivityc | No | 508 | (94%) | 932 | (93%) |   |
| Yes | 13 | (2%) | 33 | (3%) | 0.326 |  |
| Unknown | 19 | (4%) | 42 | (4%) |   |  |
| Hyperdiploidyd | No | 303 | (56%) | 627 | (62%) |   |
| Yes | 105 | (19%) | 138 | (14%) | 0.002 |  |
| Unknown | 132 | (24%) | 242 | (24%) |   |  |
| ETV6-RUNX1 rearrangement | Negative | 380 | (70%) | 688 | (68%) |   |
| Positive | 120 | (22%) | 231 | (23%) | 0.636 |  |
| Unknown | 40 | (7%) | 88 | (9%) |   |  |
| BCR-ABL1 rearrangement | Positive | 8 | (1%) | 20 | (2%) |   |
| Negative | 500 | (93%) | 941 | (93%) | 0.500 |  |
| Unknown | 32 | (6%) | 46 | (5%) |   |  |
| KMT2A-AFF1 rearrangement | Positive | 2 | (0%) | 4 | (0%) |   |
| Negative | 471 | (87%) | 890 | (88%) | 0.948 |  |
| Unknown | 67 | (12%) | 113 | (11%) |   |  |
| Prednisone responsee | Good | 492 | (91%) | 898 | (89%) |   |
| Poor | 40 | (7%) | 96 | (10%) | 0.162 |  |
| Unknown | 8 | (1%) | 13 | (1%) |   |  |
| MRD risk groupf | Standard | 223 | (41%) | 428 | (43%) |   |
| Intermediate | 251 | (46%) | 432 | (43%) |   |  |
| High | 32 | (6%) | 72 | (7%) | 0.393 |  |
| Unknown | 34 | (6%) | 75 | (7%) |   |  |
| Final risk groupg | Standard | 167 | (31%) | 318 | (32%) |   |
| Intermediate | 303 | (56%) | 532 | (53%) |   |  |
| High | 69 | (13%) | 156 | (15%) | 0.282 |  |
| Other/Unknown | 1 | (0%) | 1 | (0%) |   |  |
| Maximum transaminase levels during protocols IA/IBh | CTC grade 0 | 71 | (13%) | 25 | (2%) |   |
| CTC grades 1-2 | 249 | (46%) | 429 | (43%) |   |  |
| CTC grades 3-4 | 212 | (39%) | 553 | (55%) | < 0.001 |  |
| Unknown | 8 | (1%) | 0 | (0%) |   |  |
| Maximum bilirubin levels during protocol IAi | CTC grade 0 | 499 | (92%) | 121 | (12%) |   |
| CTC grades 1-2 | 0 | (0%) | 707 | (70%) |   |  |
| CTC grades 3-4 | 0 | (0%) | 158 | (16%) | < 0.001 |  |
| Unknown | 41 | (8%) | 21 | (2%) |   |  |
| Maximum bilirubin levels during protocol IBi | CTC grade 0 | 501 | (93%) | 342 | (34%) |   |
| CTC grades 1-2 | 0 | (0%) | 575 | (57%) |   |  |
| CTC grades 3-4 | 0 | (0%) | 50 | (5%) | < 0.001 |  |
| Unknown | 39 | (7%) | 40 | (4%) |   |  |
| Maximum bilirubin levels during protocol IA/IBi | CTC grade 0 | 540 | (100%) | 0 | (0%) |   |
| CTC grades 1-2 | 0 | (0%) | 825 | (82%) |   |  |
| CTC grades 3-4 | 0 | (0%) | 182 | (18%) | < 0.001 |  |
| Maximum bilirubin levels during the entire course of therapyj | CTC grade 0 | 412 | (76%) | 0 | (0%) |   |
| CTC grades 1-2 | 123 | (23%) | 767 | (76%) |   |  |
| CTC grades 3-4 | 5 | (1%) | 240 | (24%) | < 0.001 |  |

Table 1 Caption: Characteristics of 1547 patients according to serum bilirubin levels during induction/consolidation therapy for acute lymphoblastic leukemia

In outcome analyses, patients with high hyperbilirubinemia (grades 3-4) during induction/consolidation fared significantly worse compared to patients with moderate or no hyperbilirubinemia: 5-year EFS 76.7 ± 3% vs. 87.7 ± 1% (*P* < 0.0001), and vs. 85.2 ± 2% (*P* = 0.0031), respectively (Fig. [1](#Fig1)A). The corresponding 5-year cumulative incidences of relapse (CIR) were 15.6 ± 3% for high, 9.0 ± 1% for moderate hyperbilirubinemia, and 11.1 ± 1% for patients without hyperbilirubinemia (Fig. [1](#Fig1)B).

### Fig. 1.

![Fig. 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cb58/9838013/c742220a7296/13046_2022_2585_Fig1_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9838013_13046_2022_2585_Fig1_HTML.jpg)

A Probability of 5 year event free survival (EFS) [%] and (B) corresponding cumulative incidence of relapse (CIR) according to the maximum total bilirubin toxicity during induction/consolidation – protocols IA/IB –in the in the AIEOP ALL-BFM 2000 study population with available bilirubin information (n = 1547 patients). Bilirubin toxicity grading was according to the Common Toxicity Criteria (CTC) of the National Cancer Institute, version 2; standard error (SE) and the number of included individuals are indicated for each category

In our study cohort, 1443 (93%) patients were observed with elevated hepatic transaminase activity levels (grades 1-4) during induction/consolidation (Table [1](#Tab1)), 765 of which demonstrated grades 3-4. Transaminase levels were positively correlated with hyperbilirubinemia and, in particular, patients with high hyperbilirubinemia were at risk for concurrent high transaminase levels (grades 3-4) compared to the remaining patients (70% vs. 40%, odds ratio (OR) = 21.3, 95% confidence interval (CI) = 5.1-88.2, *P* = 2.53 × 10^− 5^) (Suppl. Table [2](#MOESM1)).

Interestingly, 5-year EFS and CIR did not differ between patients with moderate (grades 1-2), high (grades 3-4) or absent transaminasemia during induction/consolidation (Suppl. Fig. [2](#MOESM1)A and B). The 5-year EFS of patients with concurrent high hyperbilirubinemia and high transaminasemia was 76.2 ± 4% and 78.1 ± 6% in patients with high hyperbilirubinemia accompanied by moderate or no transaminasemia (grades 0-2, *P* = 0.63; Suppl. Fig. [2](#MOESM1)C). The corresponding CIR were 17.5 ± 3% and 11.0 ± 4% (*P* = 0.13; Suppl. Fig. [2](#MOESM1)D).

Multivariate analyses including established prognostic factors in AIEOP-BFM trials identified high hyperbilirubinemia as an independent predictor of outcome, while severe transaminasemia (CTC grade 4) did not demonstrate an impact here (Table [2](#Tab2)).

### Table 2.

| Variable | Event |  | Relapse |  |
| --- | --- | --- | --- | --- |
| Hazard Ratio (95% CIa) | P(X2) | Hazard Ratio (95% CIa) | P(X2) |  |
| Bilirubin CTC grades 3-4b | 1.67 (1.21-2.30) | 0.002 | 1.49 (1.00-2.21) | 0.049 |
| ALT/AST CTC grade 4c | 0.88 (0.56-1.38) | 0.575 | 1.02 (0.61-1.69) | 0.955 |
| MRD standard riskd | 0.49 (0.34-0.69) | < 0.001 | 0.51 (0.34-0.77) | 0.001 |
| MRD high riskd | 4.07 (2.87-5.78) | < 0.001 | 4.17 (2.72-6.38) | < 0.001 |
| Slow early responsee | 3.32 (2.19-5.03) | < 0.001 | 4.21 (2.66-6.66) | < 0.001 |
| Poor prednisone responsef | 1.12 (0.77-1.64) | 0.556 | 0.91 (0.57-1.45) | 0.686 |
| Initial WBC count ≥100,000g | 1.39 (0.96-2.02) | 0.078 | 1.50 (0.96-2.32) | 0.074 |

Table 2 Caption: Estimated hazard ratiosa from the multivariable Cox proportional model on event-free survival and hazard of relapse in patients of the study cohort

### Characteristics of the GWAS discovery cohort

The finally pruned GWAS discovery cohort included 650 of 1547 patients with available genome-wide genotyping information (Suppl. Table [3](#MOESM1)). Both the distribution of hyperbilirubinemia and the clinical characteristics were comparable to those of the entire study population (Table [1](#Tab1) and Suppl. Tables [3](#MOESM1) and [4](#MOESM1)).

Of the 650 patients in this discovery cohort, 215 (33%) patients had normal and 435 (67%) had increased bilirubin levels during induction/consolidation: 367 (56.5%) patients demonstrated moderate hyperbilirubinemia (grades 1-2; 313 during induction, 248 during consolidation, 159 in both phases) and 68 (10.5%) had high hyperbilirubinemia (grades 3-4; 59 reports during induction, 21 during consolidation, 12 in both phases). Pre-treatment hyperbilirubinemia at diagnosis was rare, but more frequent among patients developing chemotherapy-related hyperbilirubinemia during induction/consolidation compared to those not (7.5% (19/252) vs. 1.7% (2/116), *P =* 0.025). Considering the entire course of therapy, 91 patients (14.0%) had high hyperbilirubinemia in at least one treatment element.

### Genome-wide association study

When comparing the 435 patients with hyperbilirubinemia (grades 1-4) during induction/consolidation to the 215 patients with normal bilirubin levels, the five loci most associated with this phenotype were the *UGT1A* gene cluster, *MARK2P5*, *SULF2*, *MIR924HG* and *USH2A* (Suppl. Table [5](#MOESM1)). The strongest associations were observed for variants residing in the *UGT1A* (*UDP glucuronosyltransferase family 1 member A)* locus at 2q37. The only variant reaching near genome-wide significance (OR = 2.1, 95% CI = 1.6-2.7, *P* = 7.3 × 10^− 8^), rs6744284 was also the index SNV of a 189 kb region of high linkage disequilibrium (LD). Information on the complex *UGT1A* cluster with its overlapping genes and results of imputation are presented in supplementary material (Suppl. Fig. [3](#MOESM1), Suppl. Tables [6](#MOESM1) and [7](#MOESM1)).

To examine whether inclusion of age and immunophenotype would influence allelic association, we compared results from crude and adjusted logistic regression analyses. We did not detect any differences in conferred risk for the variant rs6744284 T allele with reference to the wild-type C allele (unadjusted allelic OR = 2.1, 95% CI = 1.6-2.7, *P* = 1.8 × 10^− 7^; adjusted OR = 2.1, 95% CI = 1.6-2.8, *P* = 1.2 × 10^− 7^) (Suppl. Table [5](#MOESM1)).

The genotypic association of rs6744284 with frequency and risk of hyperbilirubinemia during induction/consolidation increased stepwise (Table [3](#Tab3)). Compared to wild-type patients (CC; 58% with hyperbilirubinemia grades 1-4), heterozygotes (TC; 71% with hyperbilirubinemia grades 1-4) demonstrated a 1.7-fold higher risk of hyperbilirubinemia, while homozygosity for the T allele (TT; 90% with hyperbilirubinemia grades 1-4) conferred an OR of 6.5 (95% CI = 2.9-14.6, *P* = 7.0 × 10^− 6^) (Fig. [2](#Fig2)A and Table [3](#Tab3)). Inclusion of age and immunophenotype as covariates or as stratifying variables did not significantly alter these results (Suppl. Tables [7](#MOESM1), [8](#MOESM1) and [9](#MOESM1)). Notably, TT-homozygotes were at particular risk of developing high hyperbilirubinemia (grades 3-4, OR with reference to CC genotype 16.4, 95% CI = 6.1-43.8, *P* = 2 × 10^− 8^).

### Table 3.

| Cohorta | Treatment element | Study individualsbn (%) |  |  | Frequency of cases and controls per rs6744284 genotypecn (%) |  |  |  |  |  | Risk in heterozygotes and homozygotesd |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| TT |  | TC |  | CC |  |  |  |  |  |  |  |  |  |  |
| Controls | Cases | Total | Controls | Cases | Controls | Cases | Controls | Cases | ORTC(95% CI) | P | ORTT (95% CI) | P |  |  |
| 1 | Protocol IA/IB | 215(33%) | 435(67%) | 650 | 7(10%) | 63(90%) | 81(29%) | 195(71%) | 127(42%) | 177(58%) | 1.73(1.22-2.44) | 0.002 | 6.46(2.86-14.57) | < 0.001 |
| 1 | Protocol IA | 254(40%) | 372(60%) | 626 | 12(18%) | 54(82%) | 95(36%) | 171(64%) | 147(50%) | 147(50%) | 1.80(1.28-2.53) | < 0.001 | 4.50(2.31-8.76) | < 0.001 |
| 1 | Protocol IB | 352(67%) | 269(43%) | 621 | 17(25%) | 50(75%) | 131(51%) | 127(49%) | 204(69%) | 92(31%) | 2.15(1.52-3.04) | < 0.001 | 6.52(3.57-11.92) | < 0.001 |
| 1 | Protocol M | 302(61%) | 190(39%) | 492 | 17(34%) | 33(66%) | 126(61%) | 82(39%) | 159(68%) | 75(32%) | 1.38(0.93-2.04) | 0.106 | 4.12(2.16-7.85) | < 0.001 |
| 1 | Protocol II/III | 349(76%) | 110(24%) | 459 | 16 (37%) | 27(63%) | 139(75%) | 47(25%) | 194(84%) | 36(16%) | 1.82(1.12-2.96) | 0.015 | 9.09(4.46-18.56) | < 0.001 |
| 1 | HR blocks | 44(41%) | 63 (59%) | 107 | 1 (8%) | 12(92%) | 15(34%) | 29(66%) | 28(56%) | 22(44%) | 2.46(1.07-5.68) | 0.035 | 15.27(1.84-126.61) | 0.012 |
| 2 | Protocol IA/IB | 79(35%) | 145 (65%) | 224 | 3 (13%) | 20(87%) | 27(28%) | 71(72%) | 49(48%) | 54(52%) | 2.39(1.33-4.30) | 0.004 | 6.05(1.69-21.62) | 0.006 |
| 1 + 2e | Protocol IA/IB | 294(34%) | 580(66%) | 874 | 10(3%) | 108(37%) | 176(60%) | 83(14%) | 266(46%) | 231(40%) | 1.88(1.39-2.53) | < 0.001 | 6.32(3.19-12.54) | < 0.001 |

Table 3 Caption: Association between rs6744284 genotype and risk of hyperbilirubinemia during different treatment elements

### Fig. 2.

![Fig. 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cb58/9838013/9b0713149d7a/13046_2022_2585_Fig2_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9838013_13046_2022_2585_Fig2_HTML.jpg)

Frequency of rs6744284 genotype by bilirubin toxicity grading in induction/consolidation (protocols IA/IB) treatment according to the Common Toxicity Criteria of the National Cancer Institute, version 2 (CTC) (A), and by UGT1A1*28/*37 genotype (B). The number of patients (n) for each rs6744284 genotype is given above the columns. A analysis based on 650 patients of the discovery cohort; (B) based on a subset of 544 patients subsequently genotyped for UGT1A1*28/*37 depending on availability of DNA

### Independent replication analysis

We performed independent replication analyses in a cohort of 224 *ETV6-RUNX1*-rearranged pediatric ALL patients (Suppl. Table [10](#MOESM1)), results of which supported our initial GWAS findings. The allelic OR for hyperbilirubinemia (grades 1-4) during induction/consolidation conferred by the variant rs6744284 T allele versus the wild-type C allele was 2.3 (95% CI = 1.5-3.7, *P* = 2.4 × 10^− 4^). Genotypic OR in comparison to wild-type patients (CC) were 2.4 (95% CI = 1.3-4.3, *P* = 3.8 × 10^− 3^) for heterozygotes (TC) and 6.1 (95% CI = 1.7-21.6, *P* = 5.6 × 10^− 3^) for homozygous variant patients (TT) (Table [3](#Tab3)). Similar to our findings in the GWAS discovery cohort, patients possessing the rs6744284 TT genotype were at particular risk of high hyperbilirubinemia (OR with reference to CC genotype 13.6, 95% CI = 2.6-71.8, *P* = 0.002).

Association testing in the combined discovery and replication cohorts resulted in genome-wide significance. Compared to the rs6744284 wild-type allele, presence of the T allele was associated with an OR of 2.1 (CI = 1.7-2.7) for hyperbilirubinemia (grades 1-4) during induction/consolidation at a significance level of *P* = 5.7 × 10^− 11^.

### UGT1A rs6744284 genotype in subsequent treatment elements

Similar to initial findings, we observed that the rs6744284 TT genotype was also strongly associated with hyperbilirubinemia during extracompartment therapy (Protocol M, OR = 4.1, 95% CI = 2.2-7.9; *P* < 0.001), re-intensification (Protocols II and III, OR = 9.1, 95% CI = 4.5-18.6, *P* < 0.001) and high-risk (HR) block elements (OR = 15.3, 95% CI = 1.8-126.6; *P* = 0.012) (Table [3](#Tab3), Suppl. Fig. [4](#MOESM1)). Thus, the effect of rs6744284 on risk of hyperbilirubinemia was not limited to early chemotherapy, but was generalizable to all intensive treatment phases for pediatric ALL.

### UGT1A rs6744284 and Gilbert’s syndrome-associated variants

The UGT1A enzyme family is crucial for bilirubin glucuronidation and related impairing genetic alterations form the mechanistic basis of the Gilbert’s syndrome (GS) [[18](#CR18)–[20](#CR20)]. Therefore, we genotyped the GS-related functional genetic variations *UGT1A1*28 and *37* [[21](#CR21), [22](#CR22)] in 544 (84%) patients of our discovery cohort with available remission DNA (Suppl. Table [11](#MOESM1)). Comparable to rs6744284, we observed a strong association with hyperbilirubinemia: the allelic OR for *28/*37 vs. wild-type (*1) was 1.9 (95% CI = 1.4-2.5, *P =* 5.0 × 10^− 6^). Genotype-based analyses demonstrated a stepwise increase of frequency and risk of hyperbilirubinemia for the variant alleles. Out of 544 patients 62 (11%) were homozygous for either *UGT1A1**28/*28 or *37/*37 – this cannot be differentiated by our assay – and had the highest rate and risk of hyperbilirubinemia (89% compared to 58% for *1/*1; OR in comparison to *1/*1 5.8; 95% CI = 2.5-13.3; *P* = 3.3 × 10^− 5^) (Suppl. Table [12](#MOESM1)). Homozygous variant patients were at particular risk of developing high hyperbilirubinemia (grades 3-4, OR = 12.4, 95% CI = 4.4-34.8, *P* = 1.9 × 10^− 6^). The strong interrelationship of rs6744284 with *UGT1A1*28/*37* is depicted in Fig. [2](#Fig2)B. Extended haplotype analyses including eight additional GS-related variants further documented a strong association with rs6744284 (see Suppl. Table [13](#MOESM1) and related additional information). Of note, none of the GS-related variants showed a stronger association with hyperbilirubinemia than rs6744284.

### Hyperbilirubinemia, transaminase levels, rs6744284 genotype, treatment delay and outcome in the GWAS discovery cohort (n = 650)

Similar to the patients of the entire study population (*n* = 1547), patients in our discovery cohort with high hyperbilirubinemia during induction/consolidation tended to take 2 days longer to complete consolidation (*P* = 0.072) (for details see [Suppl. Information](#MOESM1), page 25). Of interest, we did not observe significant differences between rs6744284 genotypes: 88 days for TT vs. 89 days for CT and 90 days for CC (*P* = 0.122).

Consistent with the results obtained for the entire study cohort, outcome analyses of the discovery cohort showed that high hyperbilirubinemia during induction/consolidation was associated with a poor 5-year EFS of 71.8 ± 5%, compared to 87.4 ± 2% and 81.7 ± 3% in patients with moderate and without hyperbilirubinemia, respectively (Fig. [3](#Fig3)A). The corresponding 5-year CIR were 19.3 ± 5% for high hyperbilirubinemia, 10.7 ± 2% for moderate, and 13.2 ± 2% for no hyperbilirubinemia (Fig. [4](#Fig4)A). Although rs6744284 was strongly associated with high hyperbilirubinemia and the proportion of patients with TT genotype among those with high hyperbilirubinemia was 28% (19/68), there were no differences between rs6744284 genotypes related to EFS or CIR (Figs. [3](#Fig3)B and [4](#Fig4)B) in the discovery cohort. However, within high hyperbilirubinemic patients those carrying the TT genotype had a better EFS (84.2 ± 8% vs 66.9 ± 7%, *P* = 0.110) and a lower CIR (5.3 ± 5% vs 24.9 ± 6%, *P* = 0.039) at 5 years compared to the remaining genotypes (TC, CC) (Figs. [3](#Fig3)D and [4](#Fig4)D).

### Fig. 3.

![Fig. 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cb58/9838013/ff203090169d/13046_2022_2585_Fig3_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9838013_13046_2022_2585_Fig3_HTML.jpg)

Event-free survival (EFS) at 5 years in ALL patients from the discovery cohort according to (A) maximum total bilirubin toxicity grade during induction/consolidation (protocols IA/IB); (B) rs6744284 genotype (CC, TC, TT); and (C) homozygosity for the UGT1A1*28 or *37 allele. D illustrates the EFS by rs6744284 genotype (CC/TC and TT) restricted to patients with high bilirubinemia (n = 68; grades 3 and 4). Bilirubin toxicity grading was according to the Common Toxicity Criteria (CTC) of the National Cancer Institute, version 2; standard error (SE) and the number of included individuals are indicated for each category

### Fig. 4.

![Fig. 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/cb58/9838013/8c6596c37a31/13046_2022_2585_Fig4_HTML.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9838013_13046_2022_2585_Fig4_HTML.jpg)

Cumulative incidence of relapse (CIR) at 5 years in ALL patients from the discovery cohort according to (A) maximum total bilirubin toxicity grade during induction/consolidation (protocols IA/IB); (B) rs6744284 genotype (CC, TC, TT); and (C) homozygosity for the UGT1A1*28 or *37 allele. D shows the effect of the rs6744284 genotype (CC/TC and TT) on CIR in the group of patients with high bilirubin levels (n = 68; grades 3 and 4). Bilirubin toxicity grading was according to the Common Toxicity Criteria (CTC) of the National Cancer Institute, version 2; standard error (SE) and the number of included individuals are indicated for each category

In the GWAS discovery cohort 604 (93%) patients were observed with elevated hepatic transaminase levels (grades 1-4) during induction/consolidation (Suppl. Table [3](#MOESM1)), 315 of which demonstrated high grades 3-4. Similar to the results obtained for the 1443 patients in the entire study cohort with available information, transaminase levels were positively associated with hyperbilirubinemia. Especially patients with high hyperbilirubinemia were at increased risk for concurrent high transaminase levels compared to the remaining patients (75% vs. 41%, OR = 16.8, 95% CI = 2.2-127.1, *P* = 0.006) (Suppl. Table [14](#MOESM1)). Of importance, transaminasemia was not associated with rs6744284 genotype (*P* = 0.74). Five-year EFS and CIR did not differ between patients with no (grade 0), moderate (grades 1-2) or high (grades 3-4) transaminasemia during induction/consolidation (Suppl. Fig. [5](#MOESM1)A and B).

Patients with high hyperbilirubinemia and concurrent high transaminasemia tended to have a higher 5-year EFS of 74.3 ± 6% compared to 64.7 ± 12% in patients with high hyperbilirubinemia accompanied by moderate or no transaminasemia (grades 0-2, *P* = 0.480; Suppl. Fig. [5](#MOESM1)C). Corresponding CIR were 21.8 ± 6% and 11.8 ± 8% (*P* = 0.260, Suppl. Fig. [5](#MOESM1)D).

Multivariate analyses including established prognostic factors in AIEOP-BFM trials identified high hyperbilirubinemia as an independent predictor of outcome, while rs6744284 TT genotype demonstrated only a tentative protective effect in these analyses (Table [4](#Tab4)). However, in multivariate analysis restricted to patients with prognostically unfavorable high hyperbilirubinemia in induction/consolidation, the rs6744284 TT genotype was associated with a statistically significant 14-fold lower relapse-risk compared to rs6744284 wild-type or heterozygous variant patients (CC or TC) (Suppl. Table [15](#MOESM1)).

### Table 4.

| Variable | Eventb |  | Relapse |  |
| --- | --- | --- | --- | --- |
| Hazard Ratio (95% CI) | P(X2) | Hazard Ratio (95% CI) | P(X2) |  |
| rs6744284 TTc | 0.56 (0.27-1.13) | 0.103 | 0.40 (0.09-1.84) | 0.240 |
| Bilirubin CTC grades 3-4d | 2.67 (1.59-4.48) | < 0.001 | 4.61 (1.71-12.42) | 0.003 |
| ALT/AST CTC grade 4e | 1.17 (0.60-2.27) | 0.643 | 0.43 (0.06-3.29) | 0.417 |
| MRD standard-riskf | 0.94 (0.57-1.54) | 0.798 | 1.10 (0.37-3.24) | 0.870 |
| MRD high-riskf | 4.52 (2.73-7.48) | < 0.001 | 6.52 (2.41-17.66) | < 0.001 |
| Slow early responseg | 2.60 (1.35-5.00) | 0.004 | 2.21 (0.48-10.17) | 0.311 |
| Poor prednisone responseh | 1.38 (0.82-2.32) | 0.223 | 2.09 (0.77-5.64) | 0.148 |
| Initial WBC count ≥100,000/μLi | 1.48 (0.92-2.39) | 0.109 | 1.18 (0.43-3.22) | 0.750 |

Table 4 Caption: Estimated hazard ratiosa from the multivariable Cox proportional model on event-free survival and hazard of relapse in patients of the discovery cohort

## Discussion

Routine hepatotoxicity monitoring in clinical trials for pediatric ALL is typically performed by grading of elevated bilirubin and liver transaminase levels as absent, mild, moderate, severe or life-threatening/fatal according to the NCI CTC criteria. Although the evaluation of laboratory values for hepatotoxicity is common practice, it can be debated how precise such measurements reflect liver dysfunction. To enhance the phenotypic characterization in context with abnormal laboratory values, alternative classifications integrate additional clinical information (e.g., coagulopathy, impairment of liver function-dependent organs) [[23](#CR23)]. Similarly, genetic biomarkers hold the potential to aid strategies directed at improved evaluation of hepatotoxicity. Nonetheless, only a few are currently used in clinical routine to guide a genotype-adapted dosing of specific chemotherapeutic agents and thereby reduce adverse reactions (e.g., *TPMT* with thiopurines [[24](#CR24)]*, UGT1A1* with irinotecan [[25](#CR25), [26](#CR26)]). In the present study, we applied an unbiased genome-wide approach and identified genetic variation in the *UGT1A* gene cluster as a major contributor to hyperbilirubinemia associated with chemotherapy for pediatric ALL.

Genetic variation in *UGT1A* is well-established to affect enzymatic glucuronidation activity and to modulate the metabolism of endogenous metabolites as well as multiple xenobiotics [[27](#CR27)–[35](#CR35)]. Out of nine functional isoforms, only *UGT1A1* is relevant for bilirubin glucuronidation [[31](#CR31), [36](#CR36)]. Various functional *UGT1A1* variants result in partial or complete reduction of enzymatic activity and determine the phenotype of heritable diseases of bilirubin metabolism [[31](#CR31), [36](#CR36), [37](#CR37)]. The most common genetic cause for reduced bilirubin conjugation is an insertion polymorphism in the TATA box of *UGT1A1* – the (TA)7 variant allele *UGT1A1**28; homozygosity for this allele confers a reduced transcriptional activity of 18 to 33% [[22](#CR22), [38](#CR38)], corresponding to the residual glucuronidation activity of ~ 30% determined in patients with Gilbert’s syndrome (GS) [[39](#CR39), [40](#CR40)]. While other associated risk alleles have been described, e.g. *UGT1A1**6 (rs4148323) in Asian populations [[41](#CR41)], *UGT1A1**28 is by far the most common cause for the Gilbert’s syndrome in Caucasians and African Americans [[22](#CR22), [37](#CR37)]. Our lead SNV, rs6744284, was closely correlated to all GS-related variants assessable in our investigational setting – including *UGT1A1*28* – and was the best predictor of hyperbilirubinemia in our patients. These findings are concordant with reports on multi-SNV haplotypes of *UGT1A* involved in impaired glucuronidation associated with GS [[20](#CR20), [42](#CR42)] and support a potential diagnostic role for the simple assessment of rs6744284.

With relevance to cancer treatment, *UGT1A1**28 or *UGT1A1**6 homozygotes are low metabolizers of irinotecan and have an increased risk of severe neutropenia, requiring preventive dose adjustments [[25](#CR25), [43](#CR43)]. Likewise, *UGT1A*28* homozygotes are recommended to receive reduced doses of belinostat [[44](#CR44), [45](#CR45)]. The *UGT1A1* genotype also influences the pharmacokinetics of other glucuronidation-dependent drugs – including several ones used in ALL treatment (e.g., methotrexate, etoposide, and cyclophosphamide) [[28](#CR28), [32](#CR32), [33](#CR33), [35](#CR35), [46](#CR46), [47](#CR47)]. For example, low UGT1A1 activity was associated with higher plasma methotrexate and bilirubin levels, suggestive of competitive interactions between the three [[7](#CR7), [48](#CR48)]. Similar to our study, these former investigations in the field of pediatric ALL showed that patients with GS were prone to hyperbilirubinemia throughout all treatment phases [[7](#CR7), [48](#CR48)].

One of the previous studies demonstrated that despite higher bilirubin levels, ALL patients with GS did not experience significant treatment modifications, including delays, or worse therapy outcomes [[48](#CR48)], which is in line with our findings. These observations may explain why universal screening for GS in all patients diagnosed with ALL is not routinely performed so far. Although it was recommended previously to screen at least ALL patients with hyperbilirubinemia for GS [[48](#CR48)], common standard recommendations agreed on between international ALL trial consortia do not exist, and may be promoted through the Ponte di Legno initiative [[23](#CR23)].

In our study, high hyperbilirubinemia was an independent prognostic factor negatively affecting EFS and CIR of patients treated on a modern risk-adapted BFM protocol. To our knowledge, this is the first report from a large pediatric ALL cohort receiving relatively homogenous therapy demonstrating an effect of hyperbilirubinemia on long-term treatment outcome. Importantly, the negative prognostic impact was not determined in patients who demonstrated high hyperbilirubinemia and were homozygous for the variant T allele of our lead SNV rs6744284. This observation could have direct clinical implications by helping to differentiate hyperbilirubinemia conferring a negative prognostic impact from hyperbilirubinemia of a less severe clinical phenotype. Moreover, our findings imply that patients with phenotypically relevant genetic variation in *UGT1A/GS* can be spared from experiencing treatment modifications, as it was suggested previously [[48](#CR48), [49](#CR49)]. Besides replication in other independent clinically and genetically comprehensively characterized cohorts, functional studies are particularly required to augment our knowledge of hyperbilirubinemia as a treatment-related toxicity in pediatric ALL and to elucidate involved pathomechanisms.

Despite interesting perspectives, there are several limitations associated with our study: 1) We were only able to study total serum bilirubin levels. However, a separate analysis of unconjugated and conjugated bilirubin will be important for an improved understanding of therapy-related hyperbilirubinemia and its prognostic value. 2) No data on additional intake (e.g., vitamin B complex, ursodiol) and/or pharmacokinetics of glucuronidation dependent drugs (e.g., methotrexate), potentially confounding our observations, were available to us. 3) Selection bias for inclusion in our GWAS and/or replication cohorts is immanent to a clinical investigation depending on availability of reported hepatotoxicity and biological material. 4) The clinical importance of our findings could be enhanced by systematic collection of information on the potential long-term burden of hepatotoxicity. 5) Finally, the generalizability of our findings to other therapy protocols for pediatric ALL is limited due to differences in medication and timing between them. All of these issues need to be addressed in future investigations and will help to resolve the limitations of our current observations.

## Conclusions

High hyperbilirubinemia acted as an independent prognostic factor of therapy outcome in pediatric ALL patients treated on the AIEOP-BFM ALL 2000 protocol. Further, the rs6744284 genotype reliably predicted hyperbilirubinemia throughout all intensive treatment phases. Thus, both assessment of hyperbilirubinemia and *UGT1A* genotyping will be useful for complementing toxicity risk profiling and optimizing risk-adapted therapeutic strategies for pediatric ALL.

## Supplementary Information

## Acknowledgements

We are indebted to all patients, parents, nurses, technicians, data managers and doctors who participated in or contributed to AIEOP-BFM ALL 2000.

## Abbreviations

## Authors’ contributions

Drs Junk and Stanulla had full access to all of the data in the study and take responsibility for the integrity of the data and the accuracy of the data analysis. Study concept and design: Junk, Stanulla, Möricke, Cario, Schrappe. Acquisition of data: Stanulla, Houlston, Schaeffeler, Möricke, Ellinghaus, Forster, Franke, Cario, Schrappe. Analysis and interpretation of data: Junk, Zimmermann, Ellinghaus, Forster, Schaeffeler, Schwab, Stanulla. Drafting of the manuscript: Junk, Stanulla. Administrative, technical, or material support: Stanulla, Zimmermann, Beier, Schütte, Möricke, Fedders, Borkhardt, Koehler, Kulozik, Muckenthaler, Vijayakrishnan, Alten, Wintering, Klein, Hinze, Kratz, Cario, Schrappe, Schwab. Study supervision: Stanulla, Cario, Schrappe. Accountable for all aspects of the work: Junk, Stanulla. Final approval of manuscript: All authors.

## Funding

Open Access funding enabled and organized by Projekt DEAL. TRANSCALL2, ERA-NET TRANSCAN/European Commission under the 7th Framework Programme (FP7), Madeleine-Schickedanz-Kinderkrebsstiftung, Deutsche Krebshilfe, Verein für krebskranke Kinder Hannover e.V., Deutsche José Carreras Leukämie-Stiftung, Wilhelm Sander-Stiftung, Titus Clinician Scientist Program of the Else Kröner-Fresenius-Stiftung. M. Schwab and E. Schaeffeler are funded by the Robert Bosch Stiftung Stuttgart, the German Cancer Consortium (DKTK), the German Cancer Research Center (DKFZ) Partner Site Tübingen, and the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under Germany’s Excellence Strategy – EXC 2180 – 390900677.

## Availability of data and materials

All data generated or analysed during this study are included in this published article and its supplementary information files.

## Declarations

### Ethics approval and consent to participate

The study was approved by the Institutional Review Board of Hannover Medical School, Hannover, Germany, and informed consent was obtained from patients and/or their guardians.

### Competing interests

None of the authors has a conflict of interest regarding the present study. The data presented here have not been previously presented and have not previously been submitted for publication in any other journal.

## Footnotes

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

All data generated or analysed during this study are included in this published article and its supplementary information files.

### Supplementary Materials

### Data Availability Statement

All data generated or analysed during this study are included in this published article and its supplementary information files.

## References

1. Di Martino MT, Meschini S, Scotlandi K, Riganti C, De Smaele E, Zazzeroni F, et al. From single gene analysis to single cell profiling: a new era for precision medicine. J Exp Clin Cancer Res. 2020;39(1):48. doi: 10.1186/s13046-020-01549-3.  [DOI](https://doi.org/10.1186/s13046-020-01549-3) | [PMC free article](/articles/PMC7059661/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32138788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Clin%20Cancer%20Res&title=From%20single%20gene%20analysis%20to%20single%20cell%20profiling:%20a%20new%20era%20for%20precision%20medicine&author=MT%20Di%20Martino&author=S%20Meschini&author=K%20Scotlandi&author=C%20Riganti&author=E%20De%20Smaele&volume=39&issue=1&publication_year=2020&pages=48&pmid=32138788&doi=10.1186/s13046-020-01549-3&)

2. Pui CH, Yang JJ, Hunger SP, Pieters R, Schrappe M, Biondi A, et al. Childhood acute lymphoblastic leukemia: Progress through collaboration. J Clin Oncol. 2015;33(27):2938–2948. doi: 10.1200/JCO.2014.59.1636.  [DOI](https://doi.org/10.1200/JCO.2014.59.1636) | [PMC free article](/articles/PMC4567699/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26304874/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Childhood%20acute%20lymphoblastic%20leukemia:%20Progress%20through%20collaboration&author=CH%20Pui&author=JJ%20Yang&author=SP%20Hunger&author=R%20Pieters&author=M%20Schrappe&volume=33&issue=27&publication_year=2015&pages=2938-2948&pmid=26304874&doi=10.1200/JCO.2014.59.1636&)

3. Möricke A, Zimmermann M, Reiter A, Henze G, Schrauder A, Gadner H, et al. Long-term results of five consecutive trials in childhood acute lymphoblastic leukemia performed by the ALL-BFM study group from 1981 to 2000. Leukemia. 2010;24(2):265–284. doi: 10.1038/leu.2009.257.  [DOI](https://doi.org/10.1038/leu.2009.257) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20010625/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia.&title=Long-term%20results%20of%20five%20consecutive%20trials%20in%20childhood%20acute%20lymphoblastic%20leukemia%20performed%20by%20the%20ALL-BFM%20study%20group%20from%201981%20to%202000&author=A%20M%C3%B6ricke&author=M%20Zimmermann&author=A%20Reiter&author=G%20Henze&author=A%20Schrauder&volume=24&issue=2&publication_year=2010&pages=265-284&pmid=20010625&doi=10.1038/leu.2009.257&)

4. Denton CC, Rawlins YA, Oberley MJ, Bhojwani D, Orgel E. Predictors of hepatotoxicity and pancreatitis in children and adolescents with acute lymphoblastic leukemia treated according to contemporary regimens. Pediatr Blood Cancer. 2018;65(3):e26891.  [DOI](https://doi.org/10.1002/pbc.26891) | [PMC free article](/articles/PMC7522002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29218844/)

5. Zawitkowska J, Lejman M, Zaucha-Prazmo A, Drabko K, Plonowski M, Bulsa J, et al. Grade 3 and 4 toxicity profiles during therapy of childhood acute lymphoblastic leukemia. In Vivo. 2019;33(4):1333–1339. doi: 10.21873/invivo.11608.  [DOI](https://doi.org/10.21873/invivo.11608) | [PMC free article](/articles/PMC6689363/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31280227/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=In%20Vivo&title=Grade%203%20and%204%20toxicity%20profiles%20during%20therapy%20of%20childhood%20acute%20lymphoblastic%20leukemia&author=J%20Zawitkowska&author=M%20Lejman&author=A%20Zaucha-Prazmo&author=K%20Drabko&author=M%20Plonowski&volume=33&issue=4&publication_year=2019&pages=1333-1339&pmid=31280227&doi=10.21873/invivo.11608&)

6. National Cancer Institute Common Toxicity Criteria (NCI-CTC), version 2.0, DCTD, NCI, NIH, DHHS; 1998. Available from: https://ctep.cancer.gov/protocolDevelopment/electronic_applications/ctc.htm.  [https://ctep.cancer.gov/protocolDevelopment/electronic_applications/ctc.htm](https://ctep.cancer.gov/protocolDevelopment/electronic_applications/ctc.htm)

7. Kishi S, Cheng C, French D, Pei D, Das S, Cook EH, et al. Ancestry and pharmacogenetics of antileukemic drug toxicity. Blood. 2007;109(10):4151–4157. doi: 10.1182/blood-2006-10-054528.  [DOI](https://doi.org/10.1182/blood-2006-10-054528) | [PMC free article](/articles/PMC1885506/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17264302/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&title=Ancestry%20and%20pharmacogenetics%20of%20antileukemic%20drug%20toxicity&author=S%20Kishi&author=C%20Cheng&author=D%20French&author=D%20Pei&author=S%20Das&volume=109&issue=10&publication_year=2007&pages=4151-4157&pmid=17264302&doi=10.1182/blood-2006-10-054528&)

8. Raetz EA, Salzer WL. Tolerability and efficacy of L-asparaginase therapy in pediatric patients with acute lymphoblastic leukemia. J Pediatr Hematol Oncol. 2010;32(7):554–563. doi: 10.1097/MPH.0b013e3181e6f003.  [DOI](https://doi.org/10.1097/MPH.0b013e3181e6f003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20724951/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pediatr%20Hematol%20Oncol&title=Tolerability%20and%20efficacy%20of%20L-asparaginase%20therapy%20in%20pediatric%20patients%20with%20acute%20lymphoblastic%20leukemia&author=EA%20Raetz&author=WL%20Salzer&volume=32&issue=7&publication_year=2010&pages=554-563&pmid=20724951&doi=10.1097/MPH.0b013e3181e6f003&)

9. Schmiegelow K, Nielsen SN, Frandsen TL, Nersting J. Mercaptopurine/methotrexate maintenance therapy of childhood acute lymphoblastic leukemia: clinical facts and fiction. J Pediatr Hematol Oncol. 2014;36(7):503–517. doi: 10.1097/MPH.0000000000000206.  [DOI](https://doi.org/10.1097/MPH.0000000000000206) | [PMC free article](/articles/PMC4222610/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24936744/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Pediatr%20Hematol%20Oncol&title=Mercaptopurine/methotrexate%20maintenance%20therapy%20of%20childhood%20acute%20lymphoblastic%20leukemia:%20clinical%20facts%20and%20fiction&author=K%20Schmiegelow&author=SN%20Nielsen&author=TL%20Frandsen&author=J%20Nersting&volume=36&issue=7&publication_year=2014&pages=503-517&pmid=24936744&doi=10.1097/MPH.0000000000000206&)

10. Schmiegelow K, Muller K, Mogensen SS, Mogensen PR, Wolthers BO, Stoltze UK, et al. Non-infectious chemotherapy-associated acute toxicities during childhood acute lymphoblastic leukemia therapy. F1000Res. 2017;6:444. doi: 10.12688/f1000research.10768.1.  [DOI](https://doi.org/10.12688/f1000research.10768.1) | [PMC free article](/articles/PMC5389408/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28413626/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=F1000Res.&title=Non-infectious%20chemotherapy-associated%20acute%20toxicities%20during%20childhood%20acute%20lymphoblastic%20leukemia%20therapy&author=K%20Schmiegelow&author=K%20Muller&author=SS%20Mogensen&author=PR%20Mogensen&author=BO%20Wolthers&volume=6&publication_year=2017&pages=444&pmid=28413626&doi=10.12688/f1000research.10768.1&)

11. Bartram T, Burkhardt B, Wossmann W, Seidemann K, Zimmermann M, Cario G, et al. Childhood acute lymphoblastic leukemia-associated risk-loci IKZF1, ARID5B and CEBPE and risk of pediatric non-Hodgkin lymphoma: a report from the Berlin-Frankfurt-Munster study group. Leuk Lymphoma. 2015;56(3):814–816. doi: 10.3109/10428194.2014.933479.  [DOI](https://doi.org/10.3109/10428194.2014.933479) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25005032/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leuk%20Lymphoma&title=Childhood%20acute%20lymphoblastic%20leukemia-associated%20risk-loci%20IKZF1,%20ARID5B%20and%20CEBPE%20and%20risk%20of%20pediatric%20non-Hodgkin%20lymphoma:%20a%20report%20from%20the%20Berlin-Frankfurt-Munster%20study%20group&author=T%20Bartram&author=B%20Burkhardt&author=W%20Wossmann&author=K%20Seidemann&author=M%20Zimmermann&volume=56&issue=3&publication_year=2015&pages=814-816&pmid=25005032&doi=10.3109/10428194.2014.933479&)

12. Vijayakrishnan J, Qian M, Studd JB, Yang W, Kinnersley B, Law PJ, et al. Identification of four novel associations for B-cell acute lymphoblastic leukaemia risk. Nat Commun. 2019;10(1):5348. doi: 10.1038/s41467-019-13069-6.  [DOI](https://doi.org/10.1038/s41467-019-13069-6) | [PMC free article](/articles/PMC6877561/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31767839/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&title=Identification%20of%20four%20novel%20associations%20for%20B-cell%20acute%20lymphoblastic%20leukaemia%20risk&author=J%20Vijayakrishnan&author=M%20Qian&author=JB%20Studd&author=W%20Yang&author=B%20Kinnersley&volume=10&issue=1&publication_year=2019&pages=5348&pmid=31767839&doi=10.1038/s41467-019-13069-6&)

13. Migliorini G, Fiege B, Hosking FJ, Ma Y, Kumar R, Sherborne AL, et al. Variation at 10p12.2 and 10p14 influences risk of childhood B-cell acute lymphoblastic leukemia and phenotype. Blood. 2013;122(19):3298–3307. doi: 10.1182/blood-2013-03-491316.  [DOI](https://doi.org/10.1182/blood-2013-03-491316) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23996088/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&title=Variation%20at%2010p12.2%20and%2010p14%20influences%20risk%20of%20childhood%20B-cell%20acute%20lymphoblastic%20leukemia%20and%20phenotype&author=G%20Migliorini&author=B%20Fiege&author=FJ%20Hosking&author=Y%20Ma&author=R%20Kumar&volume=122&issue=19&publication_year=2013&pages=3298-3307&pmid=23996088&doi=10.1182/blood-2013-03-491316&)

14. Conter V, Bartram CR, Valsecchi MG, Schrauder A, Panzer-Grumayer R, Möricke A, et al. Molecular response to treatment redefines all prognostic factors in children and adolescents with B-cell precursor acute lymphoblastic leukemia: results in 3184 patients of the AIEOP-BFM ALL 2000 study. Blood. 2010;115(16):3206–3214. doi: 10.1182/blood-2009-10-248146.  [DOI](https://doi.org/10.1182/blood-2009-10-248146) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20154213/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&title=Molecular%20response%20to%20treatment%20redefines%20all%20prognostic%20factors%20in%20children%20and%20adolescents%20with%20B-cell%20precursor%20acute%20lymphoblastic%20leukemia:%20results%20in%203184%20patients%20of%20the%20AIEOP-BFM%20ALL%202000%20study&author=V%20Conter&author=CR%20Bartram&author=MG%20Valsecchi&author=A%20Schrauder&author=R%20Panzer-Grumayer&volume=115&issue=16&publication_year=2010&pages=3206-3214&pmid=20154213&doi=10.1182/blood-2009-10-248146&)

15. Schrappe M, Valsecchi MG, Bartram CR, Schrauder A, Panzer-Grumayer R, Moricke A, et al. Late MRD response determines relapse risk overall and in subsets of childhood T-cell ALL: results of the AIEOP-BFM-ALL 2000 study. Blood. 2011;118(8):2077–2084. doi: 10.1182/blood-2011-03-338707.  [DOI](https://doi.org/10.1182/blood-2011-03-338707) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21719599/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&title=Late%20MRD%20response%20determines%20relapse%20risk%20overall%20and%20in%20subsets%20of%20childhood%20T-cell%20ALL:%20results%20of%20the%20AIEOP-BFM-ALL%202000%20study&author=M%20Schrappe&author=MG%20Valsecchi&author=CR%20Bartram&author=A%20Schrauder&author=R%20Panzer-Grumayer&volume=118&issue=8&publication_year=2011&pages=2077-2084&pmid=21719599&doi=10.1182/blood-2011-03-338707&)

16. Moricke A, Zimmermann M, Valsecchi MG, Stanulla M, Biondi A, Mann G, et al. Dexamethasone vs prednisone in induction treatment of pediatric ALL: results of the randomized trial AIEOP-BFM ALL 2000. Blood. 2016;127(17):2101–2112. doi: 10.1182/blood-2015-09-670729.  [DOI](https://doi.org/10.1182/blood-2015-09-670729) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26888258/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&title=Dexamethasone%20vs%20prednisone%20in%20induction%20treatment%20of%20pediatric%20ALL:%20results%20of%20the%20randomized%20trial%20AIEOP-BFM%20ALL%202000&author=A%20Moricke&author=M%20Zimmermann&author=MG%20Valsecchi&author=M%20Stanulla&author=A%20Biondi&volume=127&issue=17&publication_year=2016&pages=2101-2112&pmid=26888258&doi=10.1182/blood-2015-09-670729&)

17. Ellinghaus E, Stanulla M, Richter G, Ellinghaus D, te Kronnie G, Cario G, et al. Identification of germline susceptibility loci in ETV6-RUNX1-rearranged childhood acute lymphoblastic leukemia. Leukemia. 2012;26(5):902–909. doi: 10.1038/leu.2011.302.  [DOI](https://doi.org/10.1038/leu.2011.302) | [PMC free article](/articles/PMC3356560/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22076464/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Leukemia.&title=Identification%20of%20germline%20susceptibility%20loci%20in%20ETV6-RUNX1-rearranged%20childhood%20acute%20lymphoblastic%20leukemia&author=E%20Ellinghaus&author=M%20Stanulla&author=G%20Richter&author=D%20Ellinghaus&author=G%20te%20Kronnie&volume=26&issue=5&publication_year=2012&pages=902-909&pmid=22076464&doi=10.1038/leu.2011.302&)

18. Strassburg CP. Hyperbilirubinemia syndromes (Gilbert-Meulengracht, Crigler-Najjar, Dubin-Johnson, and rotor syndrome) Best Pract Res Clin Gastroenterol. 2010;24(5):555–571. doi: 10.1016/j.bpg.2010.07.007.  [DOI](https://doi.org/10.1016/j.bpg.2010.07.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20955959/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Best%20Pract%20Res%20Clin%20Gastroenterol&title=Hyperbilirubinemia%20syndromes%20(Gilbert-Meulengracht,%20Crigler-Najjar,%20Dubin-Johnson,%20and%20rotor%20syndrome)&author=CP%20Strassburg&volume=24&issue=5&publication_year=2010&pages=555-571&pmid=20955959&doi=10.1016/j.bpg.2010.07.007&)

19. Strassburg CP. Gilbert-Meulengracht's syndrome and pharmacogenetics: is jaundice just the tip of the iceberg? Drug Metab Rev. 2010;42(1):168–181. doi: 10.3109/03602530903209429.  [DOI](https://doi.org/10.3109/03602530903209429) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20070246/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Rev&title=Gilbert-Meulengracht's%20syndrome%20and%20pharmacogenetics:%20is%20jaundice%20just%20the%20tip%20of%20the%20iceberg?&author=CP%20Strassburg&volume=42&issue=1&publication_year=2010&pages=168-181&pmid=20070246&doi=10.3109/03602530903209429&)

20. Ehmer U, Kalthoff S, Fakundiny B, Pabst B, Freiberg N, Naumann R, et al. Gilbert syndrome redefined: a complex genetic haplotype influences the regulation of glucuronidation. Hepatology. 2012;55(6):1912–1921. doi: 10.1002/hep.25561.  [DOI](https://doi.org/10.1002/hep.25561) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22213127/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology.&title=Gilbert%20syndrome%20redefined:%20a%20complex%20genetic%20haplotype%20influences%20the%20regulation%20of%20glucuronidation&author=U%20Ehmer&author=S%20Kalthoff&author=B%20Fakundiny&author=B%20Pabst&author=N%20Freiberg&volume=55&issue=6&publication_year=2012&pages=1912-1921&pmid=22213127&doi=10.1002/hep.25561&)

21. Strassburg CP. Pharmacogenetics of Gilbert's syndrome. Pharmacogenomics. 2008;9(6):703–715. doi: 10.2217/14622416.9.6.703.  [DOI](https://doi.org/10.2217/14622416.9.6.703) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18518849/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics.&title=Pharmacogenetics%20of%20Gilbert's%20syndrome&author=CP%20Strassburg&volume=9&issue=6&publication_year=2008&pages=703-715&pmid=18518849&doi=10.2217/14622416.9.6.703&)

22. Beutler E, Gelbart T, Demina A. Racial variability in the UDP-glucuronosyltransferase 1 (UGT1A1) promoter: a balanced polymorphism for regulation of bilirubin metabolism? Proc Natl Acad Sci U S A. 1998;95(14):8170–8174. doi: 10.1073/pnas.95.14.8170.  [DOI](https://doi.org/10.1073/pnas.95.14.8170) | [PMC free article](/articles/PMC20948/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9653159/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=Racial%20variability%20in%20the%20UDP-glucuronosyltransferase%201%20(UGT1A1)%20promoter:%20a%20balanced%20polymorphism%20for%20regulation%20of%20bilirubin%20metabolism?&author=E%20Beutler&author=T%20Gelbart&author=A%20Demina&volume=95&issue=14&publication_year=1998&pages=8170-8174&pmid=9653159&doi=10.1073/pnas.95.14.8170&)

23. Schmiegelow K, Attarbaschi A, Barzilai S, Escherich G, Frandsen TL, Halsey C, et al. Consensus definitions of 14 severe acute toxic effects for childhood lymphoblastic leukaemia treatment: a Delphi consensus. Lancet Oncol. 2016;17(6):e231–e2e9. doi: 10.1016/S1470-2045(16)30035-3.  [DOI](https://doi.org/10.1016/S1470-2045(16)30035-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27299279/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet%20Oncol&title=Consensus%20definitions%20of%2014%20severe%20acute%20toxic%20effects%20for%20childhood%20lymphoblastic%20leukaemia%20treatment:%20a%20Delphi%20consensus&author=K%20Schmiegelow&author=A%20Attarbaschi&author=S%20Barzilai&author=G%20Escherich&author=TL%20Frandsen&volume=17&issue=6&publication_year=2016&pages=e231-e2e9&pmid=27299279&doi=10.1016/S1470-2045(16)30035-3&)

24. Relling MV, Schwab M, Whirl-Carrillo M, Suarez-Kurtz G, Pui CH, Stein CM, et al. Clinical Pharmacogenetics implementation consortium guideline for Thiopurine dosing based on TPMT and NUDT15 genotypes: 2018 update. Clin Pharmacol Ther. 2019;105(5):1095–1105. doi: 10.1002/cpt.1304.  [DOI](https://doi.org/10.1002/cpt.1304) | [PMC free article](/articles/PMC6576267/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30447069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Clinical%20Pharmacogenetics%20implementation%20consortium%20guideline%20for%20Thiopurine%20dosing%20based%20on%20TPMT%20and%20NUDT15%20genotypes:%202018%20update&author=MV%20Relling&author=M%20Schwab&author=M%20Whirl-Carrillo&author=G%20Suarez-Kurtz&author=CH%20Pui&volume=105&issue=5&publication_year=2019&pages=1095-1105&pmid=30447069&doi=10.1002/cpt.1304&)

25. Nelson RS, Seligson ND, Bottiglieri S, Carballido E, Cueto AD, Imanirad I, et al. UGT1A1 guided Ccancer therapy: review of the evidence and considerations for clinical implementation. Cancers. 2021;13(7):1566.  [DOI](https://doi.org/10.3390/cancers13071566) | [PMC free article](/articles/PMC8036652/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33805415/)

26. Recommendations from the EGAPP Working Group Can UGT1A1 genotyping reduce morbidity and mortality in patients with metastatic colorectal cancer treated with irinotecan? Genet Med. 2009;11(1):15–20. doi: 10.1097/GIM.0b013e31818efd9d.  [DOI](https://doi.org/10.1097/GIM.0b013e31818efd9d) | [PMC free article](/articles/PMC2743610/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19125128/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Med&title=Can%20UGT1A1%20genotyping%20reduce%20morbidity%20and%20mortality%20in%20patients%20with%20metastatic%20colorectal%20cancer%20treated%20with%20irinotecan?&volume=11&issue=1&publication_year=2009&pages=15-20&pmid=19125128&doi=10.1097/GIM.0b013e31818efd9d&)

27. Mackenzie PI, Owens IS, Burchell B, Bock KW, Bairoch A, Belanger A, et al. The UDP glycosyltransferase gene superfamily: recommended nomenclature update based on evolutionary divergence. Pharmacogenet Genomics. 1997;7(4):255–269. doi: 10.1097/00008571-199708000-00001.  [DOI](https://doi.org/10.1097/00008571-199708000-00001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9295054/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=The%20UDP%20glycosyltransferase%20gene%20superfamily:%20recommended%20nomenclature%20update%20based%20on%20evolutionary%20divergence&author=PI%20Mackenzie&author=IS%20Owens&author=B%20Burchell&author=KW%20Bock&author=A%20Bairoch&volume=7&issue=4&publication_year=1997&pages=255-269&pmid=9295054&doi=10.1097/00008571-199708000-00001&)

28. Zahreddine HA, Culjkovic-Kraljacic B, Gasiorek J, Duchaine J, Borden KLB. GLI1-inducible Glucuronidation targets a broad Spectrum of drugs. ACS Chem Biol. 2019;14(3):348–355. doi: 10.1021/acschembio.8b01118.  [DOI](https://doi.org/10.1021/acschembio.8b01118) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30763062/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=ACS%20Chem%20Biol&title=GLI1-inducible%20Glucuronidation%20targets%20a%20broad%20Spectrum%20of%20drugs&author=HA%20Zahreddine&author=B%20Culjkovic-Kraljacic&author=J%20Gasiorek&author=J%20Duchaine&author=KLB%20Borden&volume=14&issue=3&publication_year=2019&pages=348-355&pmid=30763062&doi=10.1021/acschembio.8b01118&)

29. Gessner T, Vaughan LA, Beehler BC, Bartels CJ, Baker RM. Elevated pentose cycle and glucuronyltransferase in daunorubicin-resistant P388 cells. Cancer Res. 1990;50(13):3921–3927.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/2112982/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=Elevated%20pentose%20cycle%20and%20glucuronyltransferase%20in%20daunorubicin-resistant%20P388%20cells&author=T%20Gessner&author=LA%20Vaughan&author=BC%20Beehler&author=CJ%20Bartels&author=RM%20Baker&volume=50&issue=13&publication_year=1990&pages=3921-3927&pmid=2112982&)

30. Weenen H, van Maanen JM, de Planque MM, McVie JG, Pinedo HM. Metabolism of 4′-modified analogs of doxorubicin. Unique glucuronidation pathway for 4′-epidoxorubicin. Eur J Cancer Clin Oncol. 1984;20(7):919–926. doi: 10.1016/0277-5379(84)90165-2.  [DOI](https://doi.org/10.1016/0277-5379(84)90165-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/6589165/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer%20Clin%20Oncol&title=Metabolism%20of%204%E2%80%B2-modified%20analogs%20of%20doxorubicin.%20Unique%20glucuronidation%20pathway%20for%204%E2%80%B2-epidoxorubicin&author=H%20Weenen&author=JM%20van%20Maanen&author=MM%20de%20Planque&author=JG%20McVie&author=HM%20Pinedo&volume=20&issue=7&publication_year=1984&pages=919-926&pmid=6589165&doi=10.1016/0277-5379(84)90165-2&)

31. Tukey RH, Strassburg CP. Human UDP-glucuronosyltransferases: metabolism, expression, and disease. Annu Rev Pharmacol Toxicol. 2000;40:581–616. doi: 10.1146/annurev.pharmtox.40.1.581.  [DOI](https://doi.org/10.1146/annurev.pharmtox.40.1.581) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10836148/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Pharmacol%20Toxicol&title=Human%20UDP-glucuronosyltransferases:%20metabolism,%20expression,%20and%20disease&author=RH%20Tukey&author=CP%20Strassburg&volume=40&publication_year=2000&pages=581-616&pmid=10836148&doi=10.1146/annurev.pharmtox.40.1.581&)

32. Watanabe Y, Nakajima M, Ohashi N, Kume T, Yokoi T. Glucuronidation of etoposide in human liver microsomes is specifically catalyzed by UDP-glucuronosyltransferase 1A1. Drug Metab Dispos. 2003;31(5):589–595. doi: 10.1124/dmd.31.5.589.  [DOI](https://doi.org/10.1124/dmd.31.5.589) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12695347/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab%20Dispos&title=Glucuronidation%20of%20etoposide%20in%20human%20liver%20microsomes%20is%20specifically%20catalyzed%20by%20UDP-glucuronosyltransferase%201A1&author=Y%20Watanabe&author=M%20Nakajima&author=N%20Ohashi&author=T%20Kume&author=T%20Yokoi&volume=31&issue=5&publication_year=2003&pages=589-595&pmid=12695347&doi=10.1124/dmd.31.5.589&)

33. Guillemette C, Lévesque É, Rouleau M. Pharmacogenomics of human uridine diphospho-glucuronosyltransferases and clinical implications. Clin Pharmacol Ther. 2014;96(3):324–339. doi: 10.1038/clpt.2014.126.  [DOI](https://doi.org/10.1038/clpt.2014.126) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24922307/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&title=Pharmacogenomics%20of%20human%20uridine%20diphospho-glucuronosyltransferases%20and%20clinical%20implications&author=C%20Guillemette&author=%C3%89%20L%C3%A9vesque&author=M%20Rouleau&volume=96&issue=3&publication_year=2014&pages=324-339&pmid=24922307&doi=10.1038/clpt.2014.126&)

34. Ha VH, Jupp J, Tsang RY. Oncology drug dosing in Gilbert syndrome associated with UGT1A1: a summary of the literature. Pharmacotherapy. 2017;37(8):956–972. doi: 10.1002/phar.1946.  [DOI](https://doi.org/10.1002/phar.1946) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28494109/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy.&title=Oncology%20drug%20dosing%20in%20Gilbert%20syndrome%20associated%20with%20UGT1A1:%20a%20summary%20of%20the%20literature&author=VH%20Ha&author=J%20Jupp&author=RY%20Tsang&volume=37&issue=8&publication_year=2017&pages=956-972&pmid=28494109&doi=10.1002/phar.1946&)

35. Allain EP, Rouleau M, Levesque E, Guillemette C. Emerging roles for UDP-glucuronosyltransferases in drug resistance and cancer progression. Br J Cancer. 2020;122(9):1277–1287. doi: 10.1038/s41416-019-0722-0.  [DOI](https://doi.org/10.1038/s41416-019-0722-0) | [PMC free article](/articles/PMC7188667/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32047295/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Emerging%20roles%20for%20UDP-glucuronosyltransferases%20in%20drug%20resistance%20and%20cancer%20progression&author=EP%20Allain&author=M%20Rouleau&author=E%20Levesque&author=C%20Guillemette&volume=122&issue=9&publication_year=2020&pages=1277-1287&pmid=32047295&doi=10.1038/s41416-019-0722-0&)

36. Ritter JK, Crawford JM, Owens IS. Cloning of two human liver bilirubin UDP-glucuronosyltransferase cDNAs with expression in COS-1 cells. J Biol Chem. 1991;266(2):1043–1047. doi: 10.1016/S0021-9258(17)35280-8.  [DOI](https://doi.org/10.1016/S0021-9258(17)35280-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/1898728/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Cloning%20of%20two%20human%20liver%20bilirubin%20UDP-glucuronosyltransferase%20cDNAs%20with%20expression%20in%20COS-1%20cells&author=JK%20Ritter&author=JM%20Crawford&author=IS%20Owens&volume=266&issue=2&publication_year=1991&pages=1043-1047&pmid=1898728&doi=10.1016/S0021-9258(17)35280-8&)

37. Bosma PJ. Inherited disorders of bilirubin metabolism. J Hepatol. 2003;38(1):107–117. doi: 10.1016/S0168-8278(02)00359-8.  [DOI](https://doi.org/10.1016/S0168-8278(02)00359-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12480568/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Inherited%20disorders%20of%20bilirubin%20metabolism&author=PJ%20Bosma&volume=38&issue=1&publication_year=2003&pages=107-117&pmid=12480568&doi=10.1016/S0168-8278(02)00359-8&)

38. Bosma PJ, Chowdhury JR, Bakker C, Gantla S, de Boer A, Oostra BA, et al. The genetic basis of the reduced expression of bilirubin UDP-glucuronosyltransferase 1 in Gilbert's syndrome. N Engl J Med. 1995;333(18):1171–1175. doi: 10.1056/NEJM199511023331802.  [DOI](https://doi.org/10.1056/NEJM199511023331802) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7565971/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=The%20genetic%20basis%20of%20the%20reduced%20expression%20of%20bilirubin%20UDP-glucuronosyltransferase%201%20in%20Gilbert's%20syndrome&author=PJ%20Bosma&author=JR%20Chowdhury&author=C%20Bakker&author=S%20Gantla&author=A%20de%20Boer&volume=333&issue=18&publication_year=1995&pages=1171-1175&pmid=7565971&doi=10.1056/NEJM199511023331802&)

39. Black M, Billing BH. Hepatic bilirubin udp-glucuronyl transferase activity in liver disease and gilbert's syndrome. N Engl J Med. 1969;280(23):1266–1271. doi: 10.1056/NEJM196906052802303.  [DOI](https://doi.org/10.1056/NEJM196906052802303) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/5770050/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Hepatic%20bilirubin%20udp-glucuronyl%20transferase%20activity%20in%20liver%20disease%20and%20gilbert's%20syndrome&author=M%20Black&author=BH%20Billing&volume=280&issue=23&publication_year=1969&pages=1266-1271&pmid=5770050&doi=10.1056/NEJM196906052802303&)

40. Jansen PL, Bosma PJ, Chowdhury JR. Molecular biology of bilirubin metabolism. Prog Liver Dis. 1995;13:125–150.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/9224500/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Prog%20Liver%20Dis&title=Molecular%20biology%20of%20bilirubin%20metabolism&author=PL%20Jansen&author=PJ%20Bosma&author=JR%20Chowdhury&volume=13&publication_year=1995&pages=125-150&pmid=9224500&)

41. Kim JY, Cheong HS, Park BL, Kim LH, Namgoong S, Kim JO, et al. Comprehensive variant screening of the UGT gene family. Yonsei Med J. 2014;55(1):232–239. doi: 10.3349/ymj.2014.55.1.232.  [DOI](https://doi.org/10.3349/ymj.2014.55.1.232) | [PMC free article](/articles/PMC3874916/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24339312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Yonsei%20Med%20J&title=Comprehensive%20variant%20screening%20of%20the%20UGT%20gene%20family&author=JY%20Kim&author=HS%20Cheong&author=BL%20Park&author=LH%20Kim&author=S%20Namgoong&volume=55&issue=1&publication_year=2014&pages=232-239&pmid=24339312&doi=10.3349/ymj.2014.55.1.232&)

42. Lankisch TO, Behrens G, Ehmer U, Mobius U, Rockstroh J, Wehmeier M, et al. Gilbert's syndrome and hyperbilirubinemia in protease inhibitor therapy--an extended haplotype of genetic variants increases risk in indinavir treatment. J Hepatol. 2009;50(5):1010–1018. doi: 10.1016/j.jhep.2008.12.030.  [DOI](https://doi.org/10.1016/j.jhep.2008.12.030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19303655/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Hepatol&title=Gilbert's%20syndrome%20and%20hyperbilirubinemia%20in%20protease%20inhibitor%20therapy--an%20extended%20haplotype%20of%20genetic%20variants%20increases%20risk%20in%20indinavir%20treatment&author=TO%20Lankisch&author=G%20Behrens&author=U%20Ehmer&author=U%20Mobius&author=J%20Rockstroh&volume=50&issue=5&publication_year=2009&pages=1010-1018&pmid=19303655&doi=10.1016/j.jhep.2008.12.030&)

43. Hoskins JM, Goldberg RM, Qu P, Ibrahim JG, McLeod HL. UGT1A1*28 genotype and irinotecan-induced neutropenia: dose matters. J Natl Cancer Inst. 2007;99(17):1290–1295. doi: 10.1093/jnci/djm115.  [DOI](https://doi.org/10.1093/jnci/djm115) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17728214/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=UGT1A1*28%20genotype%20and%20irinotecan-induced%20neutropenia:%20dose%20matters&author=JM%20Hoskins&author=RM%20Goldberg&author=P%20Qu&author=JG%20Ibrahim&author=HL%20McLeod&volume=99&issue=17&publication_year=2007&pages=1290-1295&pmid=17728214&doi=10.1093/jnci/djm115&)

44. Goey AK, Sissung TM, Peer CJ, Trepel JB, Lee MJ, Tomita Y, et al. Effects of UGT1A1 genotype on the pharmacokinetics, pharmacodynamics, and toxicities of belinostat administered by 48-hour continuous infusion in patients with cancer. J Clin Pharmacol. 2016;56(4):461–473. doi: 10.1002/jcph.625.  [DOI](https://doi.org/10.1002/jcph.625) | [PMC free article](/articles/PMC6361127/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26313268/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=Effects%20of%20UGT1A1%20genotype%20on%20the%20pharmacokinetics,%20pharmacodynamics,%20and%20toxicities%20of%20belinostat%20administered%20by%2048-hour%20continuous%20infusion%20in%20patients%20with%20cancer&author=AK%20Goey&author=TM%20Sissung&author=CJ%20Peer&author=JB%20Trepel&author=MJ%20Lee&volume=56&issue=4&publication_year=2016&pages=461-473&pmid=26313268&doi=10.1002/jcph.625&)

45. Peer CJ, Goey AKL, Sissung TM, Erlich S, Lee M-J, Tomita Y, et al. UGT1A1 genotype-dependent dose adjustment of belinostat in patients with advanced cancers using population pharmacokinetic modeling and simulation. J Clin Pharmacol. 2016;56(4):450–460. doi: 10.1002/jcph.627.  [DOI](https://doi.org/10.1002/jcph.627) | [PMC free article](/articles/PMC6357964/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26637161/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pharmacol&title=UGT1A1%20genotype-dependent%20dose%20adjustment%20of%20belinostat%20in%20patients%20with%20advanced%20cancers%20using%20population%20pharmacokinetic%20modeling%20and%20simulation&author=CJ%20Peer&author=AKL%20Goey&author=TM%20Sissung&author=S%20Erlich&author=M-J%20Lee&volume=56&issue=4&publication_year=2016&pages=450-460&pmid=26637161&doi=10.1002/jcph.627&)

46. Zahreddine HA, Culjkovic-Kraljacic B, Assouline S, Gendron P, Romeo AA, Morris SJ, et al. The sonic hedgehog factor GLI1 imparts drug resistance through inducible glucuronidation. Nature. 2014;511(7507):90–93. doi: 10.1038/nature13283.  [DOI](https://doi.org/10.1038/nature13283) | [PMC free article](/articles/PMC4138053/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24870236/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature.&title=The%20sonic%20hedgehog%20factor%20GLI1%20imparts%20drug%20resistance%20through%20inducible%20glucuronidation&author=HA%20Zahreddine&author=B%20Culjkovic-Kraljacic&author=S%20Assouline&author=P%20Gendron&author=AA%20Romeo&volume=511&issue=7507&publication_year=2014&pages=90-93&pmid=24870236&doi=10.1038/nature13283&)

47. Kishi S, Yang W, Boureau B, Morand S, Das S, Chen P, et al. Effects of prednisone and genetic polymorphisms on etoposide disposition in children with acute lymphoblastic leukemia. Blood. 2004;103(1):67–72. doi: 10.1182/blood-2003-06-2105.  [DOI](https://doi.org/10.1182/blood-2003-06-2105) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12969965/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Blood.&title=Effects%20of%20prednisone%20and%20genetic%20polymorphisms%20on%20etoposide%20disposition%20in%20children%20with%20acute%20lymphoblastic%20leukemia&author=S%20Kishi&author=W%20Yang&author=B%20Boureau&author=S%20Morand&author=S%20Das&volume=103&issue=1&publication_year=2004&pages=67-72&pmid=12969965&doi=10.1182/blood-2003-06-2105&)

48. Berrueco R, Alonso-Saladrigues A, Martorell-Sampol L, Catala-Temprano A, Ruiz-Llobet A, Toll T, et al. Outcome and toxicities associated to chemotherapy in children with acute lymphoblastic leukemia and Gilbert syndrome. Usefulness of UGT1A1 mutational screening. Pediatr Blood Cancer. 2015;62(7):1195–1201. doi: 10.1002/pbc.25457.  [DOI](https://doi.org/10.1002/pbc.25457) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25855413/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr%20Blood%20Cancer&title=Outcome%20and%20toxicities%20associated%20to%20chemotherapy%20in%20children%20with%20acute%20lymphoblastic%20leukemia%20and%20Gilbert%20syndrome.%20Usefulness%20of%20UGT1A1%20mutational%20screening&author=R%20Berrueco&author=A%20Alonso-Saladrigues&author=L%20Martorell-Sampol&author=A%20Catala-Temprano&author=A%20Ruiz-Llobet&volume=62&issue=7&publication_year=2015&pages=1195-1201&pmid=25855413&doi=10.1002/pbc.25457&)

49. Nomura A, Maruo Y, Taga T, Takeuchi Y. Contribution of UGT1A1 variations to chemotherapy-induced unconjugated hyperbilirubinemia in pediatric leukemia patients. Pediatr Res. 2016;80(2):252–257. doi: 10.1038/pr.2016.75.  [DOI](https://doi.org/10.1038/pr.2016.75) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27057738/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pediatr%20Res&title=Contribution%20of%20UGT1A1%20variations%20to%20chemotherapy-induced%20unconjugated%20hyperbilirubinemia%20in%20pediatric%20leukemia%20patients&author=A%20Nomura&author=Y%20Maruo&author=T%20Taga&author=Y%20Takeuchi&volume=80&issue=2&publication_year=2016&pages=252-257&pmid=27057738&doi=10.1038/pr.2016.75&)
