# Exonuclease 1 genetic variant is associated with clinical outcomes of pemetrexed chemotherapy in lung adenocarcinoma

## Metadata
**Authors:** Mi Jeong Hong, Ji Eun Park, Shin Yup Lee, Jang Hyuck Lee, Jin Eun Choi, Hyo-Gyoung Kang, Sook Kyung Do, Ji Yun Jeong, Kyung Min Shin, Won Ki Lee, Yangki Seok, Sun Ha Choi, Yong Hoon Lee, Hyewon Seo, Seung Soo Yoo, Jaehee Lee, Seung Ick Cha, Chang Ho Kim, Jae Yong Park
**Journal:** Journal of Cancer
**Date:** 2022 Dec 7
**DOI:** [10.7150/jca.78498](https://doi.org/10.7150/jca.78498)
**PMID:** 36606188
**PMCID:** PMC9809306
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9809306/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9809306/pdf/jcav13p3701.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9809306/pdf/jcav13p3701.pdf)

## Abstract

Pemetrexed is an anti-folate agent which is one of the most frequently used chemotherapy agents for non-squamous non-small cell lung cancer (NSCLC) patients. However, clinical response to pemetrexed chemotherapy and survival outcome of patients varies significantly. We evaluated whether the genetic variants in miRNA target sites may affect the treatment outcome of pemetrexed chemotherapy in lung adenocarcinoma patients. One hundred SNPs in miRNA binding regions in cancer-related genes were obtained from the crosslinking, ligation, and sequencing of hybrids (CLASH) and CancerGenes database, and the associations with the response to pemetrexed chemotherapy and survival outcomes were investigated in 314 lung adenocarcinoma patients. Two polymorphisms, EXO1 rs1047840G>A and CAMKK2 rs1653586G>T, were significantly associated with worse chemotherapy response (adjusted odds ratio [aOR] = 0.41, 95% CI = 0.24-0.68, P = 0.001, under dominant model; and aOR = 0.33, 95% CI = 0.16-0.67, P = 0.002, under dominant model, respectively) and worse OS (adjusted hazard ratio [aHR] = 1.34, 95% CI = 1.01-1.77, P = 0.04, under dominant model; and aHR = 1.50, 95% CI = 1.06-2.13, P = 0.02, under dominant model, respectively) in multivariate analyses. Significantly increased luciferase activity was noted in EXO1 rs1047840 A allele compared to G allele. In conclusion, two SNPs in miRNA binding sites, especially EXO1 rs1047840G>A, were associated with the chemotherapy response and survival outcome in lung adenocarcinoma patients treated with pemetrexed.

Keywords: lung adenocarcinoma, miRNA target sites, genetic variants, chemotherapy, response, survival

## Introduction

A huge effort has been made to improve the prognosis of lung cancer patients over the past decades, and the treatment of lung cancer has progressed remarkably by recent innovations of targeted therapy and immunotherapy [1](#B1)^-^[3](#B3). However, cytotoxic chemotherapy alone or in combination with other cancer therapies remains as an important treatment modality for non-small cell lung cancer (NSCLC) patients. Pemetrexed is one of the most frequently used chemotherapeutic agents for non-squamous NSCLC patients. Recently, immune checkpoint blockade combined with pemetrexed and platinum doublet chemotherapy has been approved as a standard first-line treatment in patients without driver mutations, which significantly increased overall survival (OS) and progression-free survival (PFS) compared to previous standard of care chemotherapy in non-squamous NSCLC patients [4](#B4).

Pemetrexed is an anti-folate agent that inhibits cell replication and growth through interrupting synthesis of DNA and RNA by targeting three enzymes involved in purine and pyrimidine synthesis: thymidylate synthase (TS), dihydrofolate reductase (DHFR), and glycinamide ribonucleotide formyltransferase (GARFT) [5](#B5). It is widely accepted that the efficacy of pemetrexed is worse in squamous NSCLC and small-cell lung cancer than non-squamous NSCLC [6](#B6), but therapeutic outcomes of pemetrexed varies significantly even in patients with adenocarcinoma. Despite much effort to discover and validate molecular biomarkers to predict efficacy of pemetrexed, no viable biomarkers have yet been clearly established in clinical practice [7](#B7)^-^[9](#B9).

MicroRNA (miRNA) is a small (containing about 22 nucleotides) single-stranded non-coding RNA molecule that functions as endogenous negative regulators by binding to the 3'-untranslated region (UTR) of target mRNA causing degradation or translational inhibition of the target [10](#B10)^, ^[11](#B11). Comprehensive miRNA profiling demonstrated that miRNAs are involved in vital biological processes such as cell division and death, cellular metabolism, intracellular signaling, and immunity [12](#B12)^-^[15](#B15). In addition, miRNAs are critically involved in the development and progression of diverse human cancers [16](#B16)^, ^[17](#B17). Computational approach for miRNA target prediction have focused on canonical (seed-based) target sites, demonstrating that single nucleotide polymorphisms (SNPs) in miRNA binding sites could be used as cancer biomarkers for predicting cancer risk, therapeutic response and prognosis of patients [18](#B18). However, recent advances in transcriptome-wide mapping of miRNA target sites by crosslinking, ligation, and sequencing of hybrid (CLASH), elucidated that considerable portion of miRNA-mRNA interactions are mediated not only through canonical seed sites, but also through non-canonical sites [19](#B19). Based on the association between miRNA dysregulation and human malignancies, we hypothesized that SNPs in miRNA target sites may alter miRNA-mRNA binding and consequently gene expression, thus affecting the treatment response and survival outcomes of patients treated with pemetrexed. To test this hypothesis, we selected SNPs in miRNA target sites using CLASH data and analyzed the association with the clinical outcome of pemetrexed treatment in lung adenocarcinoma patients.

## Materials and Methods

### Study populations

A total of 314 lung adenocarcinoma patients with available genomic DNA samples who underwent pemetrexed based chemotherapy at Kyungpook National University Hospital (KNUH) in Daegu, Korea, between March 2007 and July 2015, were enrolled. We excluded patients who had any active malignancies ≤ 3 years before the diagnosis of lung cancer, except resected non-melanoma skin cancer and *in situ* cancers including carcinoma *in situ* of the cervix or breast. Patients who were diagnosed with stage III/IV or had recurred disease after curative surgery were treated with either pemetrexed plus cisplatin for up to 4 to 6 cycles as first line chemotherapy regimen with/without pemetrexed maintenance therapy, or pemetrexed alone as second or further line chemotherapy. Because pemetrexed maintenance therapy was approved in 2009 in Korea, patients who were treated with pemetrexed plus cisplatin before 2009 did not underwent pemetrexed maintenance therapy even if their disease had not progressed after four cycles of first-line pemetrexed/cisplatin chemotherapy [20](#B20). Chemotherapy was discontinued upon disease progression, major toxicities, or according to patient's or physician's decision. Response to chemotherapy was assessed according to the Response Evaluation Criteria in Solid Tumors [21](#B21) and the best overall response for each patient was reported. Patients with complete response or partial response were defined as responders, and patients with stable disease or progressive disease were defined as nonresponders. Survival outcome was assessed with overall survival (OS), the time between the first date of chemotherapy and date of death or last follow-up. Genomic DNA samples were provided by the National Biobank of Korea, KNUH, which is supported by the Ministry of Health, Welfare and Family Affairs. Written informed consent was obtained from all patients and this study was approved by the Institutional Review Board of the KNUH.

### SNP selection and genotyping

PolymiRTS database 3.0 ([http://compbio.uthsc.edu/miRSNP](http://compbio.uthsc.edu/miRSNP)) [22](#B22) was used to find potentially functional variants in miRNA binding region, and 24,027 SNPs were extracted from the CLASH data which has been integrated in PolymiRTS database. Out of these, 1,574 SNPs in genes involved in human cancer were selected using the CancerGenes database ([http://cbio.mskcc.org/cancergenes](http://cbio.mskcc.org/cancergenes)) [23](#B23). Lastly, 100 SNPs were collected after excluding those with minor allele frequency < 0.05 in the HapMap-JPT and those in strong linkage disequilibrium (LD, *r*^2^ ≥ 0.8). Genotyping was performed using the iPLEX^®^ Assay and MassARRAY^®^ System (Agena Bioscience, San Diego, CA, USA).

### Cloning of the luciferase reporter gene and dual luciferase assay

Luciferase reporter assay was conducted to determine whether *EXO1* rs1047840G>A and *CAMKK2* rs1653586G>T modulate the binding of miRNA and thus affects the expression level of target genes. Luciferase reporter plasmids was constructed using the psiCHECK^TM^-2 vector (Promega, Madison, WI, USA). *EXO1* coding sequence containing rs1047840G or rs1047840A and *CAMKK2* 3'UTR sequence containing rs1653586G or rs1653586T were synthesized using PCR from human genomic DNA and cloned into the psiCHECK^TM^-2 vector. In the PolymiRTS database, we found that miR-30e and miR-185 were experimentally demonstrated to interact with the target sites on mRNAs of EXO1 and CAMKK2, respectively. Each construct was then co-transfected with miRNA (psiCHECK^TM^-2-*EXO1* with miR-30e, and psiCHECK^TM^-2-*CAMKK2* with miR-185) into H1299 cells following the manufacturer's instructions. After 24 hours of incubation, *Renilla* luciferase activities were measured using the firefly luciferase activities as a normalization control.

### Statistical analysis

Deviations from Hardy-Weinberg equilibrium was measured by a goodness-of-fit χ^2^ test. Genotypes for SNPs were analyzed as three-group categorical variable, and analyzed under dominant, recessive, and codominant models. The association between clinical variables or genotypes and response to chemotherapy was tested by odds ratio (OR) and 95% confidence interval (CI) using unconditional logistic regression. Kaplan-Meier method was used to estimate survival analysis, and log-rank test was used to compare the difference in OS according to different clinical variables or genotypes. Hazard ratio (HR) and 95% CI were estimated using multivariate Cox proportional hazards model. Variables with *P* value < 0.05 were considered statistically significant. The statistical data were obtained using Statistical Analysis System for Windows, version 9.4 (SAS Institute, Cary, NC, USA).

## Results

Associations between clinical parameters and the response to chemotherapy and survival outcomes are presented in Table [1](#T1). Overall response rate was 41.7% and median survival time was 24.1 months (95% CI = 21.7-25.9 months). Univariate analyses showed that age > 65, no benefit from tyrosine kinase inhibitor (TKI) treatment which include wild-type EGFR or ALK or no response to EGFR- or ALK-TKIs, pemetrexed maintenance therapy, and pemetrexed/cisplatin regimen were associated with better response. Regarding the OS, female sex, never-smoking status, postsurgical recurrence versus stage III/IV, ECOG performance status 0, benefit from TKI, pemetrexed maintenance therapy, and pemetrexed alone were significantly associated with the better OS. The conflicting results may be partly explained by first-line pemetrexed/cisplatin use in patients with wild-type EGFR or ALK and second-line pemetrexed monotherapy in patients with upfront EGFR- or ALK-TKI treatment, because the pemetrexed/cisplatin provides higher response rate than pemetrexed monotherapy but EGFR- or ALK- TKI is associated with better survival than chemotherapy alone.

### Table 1.

|   | No. of | Response to chemotherapy |  |  |  | Overall survival |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| respondersa | nonrespondersa | OR (95% CI) | P | MST | 95% CI | Log-Rank | HR (95% CI) | P |  |  |
| Variables | cases | (CR+PR) | (SD+PD) |   |   | (months) |   | P |   |   |
| Overall | 314 | 131 (41.7)a | 183 (58.3) |   |   | 24.1 | 21.7-25.9 |   |   |   |
| Age (years) |   |   |   |   |   |   |   |   |   |   |
| ≤ 65 | 177 | 65 (36.7) | 112 (63.3) | 1.00 |   | 25.8 | 23.6-31.1 |   | 1.00 |   |
| >65 | 137 | 66 (48.2) | 71 (51.8) | 1.60 (1.02-2.52) | 0.04 | 21.1 | 17.2-25.1 | 0.08 | 1.26(0.97-1.63) | 0.08 |
| Sex |   |   |   |   |   |   |   |   |   |   |
| Male | 151 | 64 (42.4) | 87 (57.6) | 1.00 |   | 16.6 | 14.6-20.9 |   | 1.00 |   |
| Female | 163 | 67 (41.1) | 96 (58.9) | 0.95 (0.61-1.49) | 0.82 | 28.5 | 25.1-34.3 | <0.0001 | 0.60(0.46-0.78) | 0.0001 |
| Smoking status |   |   |   |   |   |   |   |   |   |   |
| Never | 165 | 70 (42.4) | 95 (57.6) | 1.00 |   | 28.6 | 24.7-54.4 |   | 1.00 |   |
| Ever | 149 | 61 (40.9) | 88 (59.1) | 0.94 (0.60-1.47) | 0.79 | 16.4 | 14.5-22.5 | 0.0002 | 1.64(1.27-2.12) | 0.0002 |
| Stage |   |   |   |   |   |   |   |   |   |   |
| III/IV | 244 | 102 (41.8) | 142 (58.2) | 1.00 |   | 22.9 | 17.9-24.6 |   | 1.00 |   |
| Postsurgical recurrence | 70 | 29 (41.4) | 41 (58.6) | 0.99 (0.57-1.69) | 0.96 | 34.9 | 25.9-44.6 | 0.001 | 0.59(0.43-0.82) | 0.001 |
| PS ECOG |   |   |   |   |   |   |   |   |   |   |
| 0 | 102 | 45 (44.1) | 57 (55.9) | 1.00 |   | 29.5 | 24.6-35.4 |   | 1.00 |   |
| 1-2 | 212 | 86 (40.6) | 126 (59.4) | 0.87 (0.54-1.39) | 0.55 | 21.7 | 17.4-24.6 | 0.01 | 1.46(1.1-1.94) | 0.01 |
| TKI benefit |   |   |   |   |   |   |   |   |   |   |
| No | 199 | 95 (47.7) | 104 (52.3) | 1.00 |   | 15.2 | 14.0-17.2 |   | 1.00 |   |
| Yes | 115 | 36 (31.3) | 79 (68.7) | 0.50 (0.31-0.81) | 0.01 | 39.4 | 34.4-44.3 | <0.0001 | 0.31(0.24-0.42) | <0.0001 |
| Maintenanceb |   |   |   |   |   |   |   |   |   |   |
| No | 113 | 52 (46.0) | 61 (54.0) | 1.00 |   | 14.3 | 10.8-17.1 |   | 1.00 |   |
| Yes | 67 | 45 (67.2) | 22 (32.8) | 2.40 (1.28-4.50) | 0.01 | 27.2 | 16.6- | <0.0001 | 0.41(0.27-0.62) | <0.0001 |
| Regimen |   |   |   |   |   |   |   |   |   |   |
| Pem/Cis | 180 | 97 (53.9) | 83 (46.1) | 1.00 |   | 16.6 | 14.6-20.9 |   | 1.00 |   |
| Pem alone | 134 | 34 (25.4) | 100 (74.6) | 0.29 (0.18-0.47) | <0.0001 | 28.8 | 25.4-35.4 | 0.0001 | 0.60(0.46-0.78) | 0.0001 |

Table 1 Caption: Univariate analysis for response to chemotherapy and overall survival by clinical variables

Of the 100 SNPs genotyped, 74 SNPs were further analyzed ([Supplementary Table 1](#SM0)) after excluding 3 SNPs with genotyping failure and 23 SNPs with deviations from Hardy-Weinberg equilibrium (*P* < 0.05) or low call rates (< 95%). Among these, 7 SNPs were significantly associated with the response to pemetrexed (Table [2](#T2)) while 16 SNPs with OS (Table [3](#T3)). Two polymorphisms (*EXO1* rs1047840G>A and *CAMKK2* rs1653586G>T) were significantly associated with both chemotherapy response and survival outcome. The *EXO1* rs1047840G>A and *CAMKK2* rs1653586G>T were significantly associated with worse chemotherapy response (adjusted odds ratio [aOR] = 0.41, 95% CI = 0.24-0.68, *P* = 0.001, under a dominant model; and aOR = 0.33, 95% CI = 0.16-0.67, *P* = 0.002, under a dominant model, respectively) and worse OS (adjusted hazard ratio [aHR] = 1.34, 95% CI = 1.01-1.77, *P* = 0.04, under a dominant model; and aHR = 1.50, 95% CI = 1.06-2.13, *P* = 0.02, under a dominant model, respectively) in multivariate analyses (Table [4](#T4) and Figure [1](#F1)).

### Table 2.

| ID No.a | Target Gene | miRNA | Alleles | CR (%) | MAF | HWE-p | Pb for response |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| dominant | recessive | codominant |  |  |  |  |  |  |  |
| rs1047840 | EXO1 | hsa-miR-30e | GA | 97.77 | 0.22 | 0.17 | 0.001 | 0.38 | 0.002 |
| rs1653586 | CAMKK2 | hsa-miR-185 | GT | 99.68 | 0.09 | 0.08 | 0.002 | - | 0.002 |
| rs2295865 | SUPT16H | hsa-miR-186 | CA | 98.09 | 0.11 | 0.26 | 0.05 | 0.99 | 0.03 |
| rs11541557 | ARF1 | hsa-miR-92a | GT | 98.09 | 0.10 | 0.06 | 0.03 | - | 0.03 |
| rs6698826 | RAB3B | hsa-miR-27b | CA | 95.54 | 0.14 | 0.72 | 0.01 | 0.85 | 0.01 |
| rs3762158 | SUPT16H | has-miR-484 | GC | 93.95 | 0.12 | 0.27 | 0.05 | 0.99 | 0.04 |
| rs4705 | PDGFRL | hsa-miR-25 | CT | 98.73 | 0.47 | 0.41 | 0.87 | 0.04 | 0.27 |

Table 2 Caption: Summary of 7 SNPs and response to chemotherapy

### Table 3.

| ID No.a | Target Gene | miRNA | Alleles | CR (%) | MAF | HWE-p | Pb for overall survival |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| dominant | recessive | codominant |  |  |  |  |  |  |  |
| rs1047840 | EXO1 | hsa-miR-30e | GA | 97.77 | 0.22 | 0.17 | 0.04 | 0.62 | 0.07 |
| rs1653586 | CAMKK2 | hsa-miR-185 | GT | 99.68 | 0.09 | 0.08 | 0.02 |   | 0.02 |
| rs17445840 | CDH2 | hsa-miR-615-3p | CT | 98.73 | 0.04 | 0.52 | 0.04 | 0.19 | 0.03 |
| rs6934058 | CDC5L | hsa-miR-505 | TC | 97.77 | 0.41 | 0.46 | 0.04 | 0.26 | 0.05 |
| rs1056471 | HADHB | hsa-miR-99a | GC | 98.73 | 0.16 | 0.22 | 0.002 | 0.95 | 0.01 |
| rs296888 | HNRNPK | hsa-miR-615-3p | CT | 99.36 | 0.28 | 0.71 | 0.01 | 0.30 | 0.01 |
| rs3212986 | CD3EAP | hsa-miR-92a | GT | 98.09 | 0.27 | 0.24 | 0.02 | 0.45 | 0.03 |
| rs12449580 | AIPL1 | has-miR-3615 | CG | 98.41 | 0.39 | 0.29 | 0.04 | 0.05 | 0.01 |
| rs1318648 | ESPL1 | hsa-miR-149 | TG | 96.50 | 0.30 | 0.84 | 0.04 | 0.01 | 0.01 |
| rs2076345 | TCEB3 | hsa-miR-320b | CT | 93.63 | 0.25 | 0.17 | 0.04 | 0.76 | 0.10 |
| rs2228128 | POLR2A | has-miR-744 | TC | 93.63 | 0.06 | 0.28 | 0.05 | 0.97 | 0.02 |
| rs3217933 | CCND2 | hsa-miR-17 | TC | 99.68 | 0.09 | 0.68 | 0.59 | 0.004 | 0.39 |
| rs7654 | TPM3 | hsa-miR-615-3p | CA | 99.68 | 0.08 | 0.15 | 0.62 | 0.01 | 0.35 |
| rs2306409 | GTPBP4 | hsa-miR-16 | TC | 98.41 | 0.39 | 0.84 | 0.86 | 0.004 | 0.15 |
| rs7091596 | PARD3 | hsa-miR-93* | AT | 98.09 | 0.24 | 0.63 | 0.09 | 0.02 | 0.03 |
| rs2261988 | UHRF1 | has-miR-615-3p | CA | 98.41 | 0.15 | 0.16 | 0.09 | 0.06 | 0.04 |

Table 3 Caption: Summary of 16 SNPs and overall survival

### Table 4.

| Polymorphism/Genotype | Target Gene | miRNA | No. of cases (%)a | Response |  |  |  | Overall Survival |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Responders (%)b | Non-responders (%)b | OR (95% CI)c | P c | HR (95% CI)d | P d |  |  |  |  |
| rs1047840f | EXO1 | hsa-miR-30e |   |   |   |   |   |   |   |
| GG | (cds-non) |   | 191 (62.2) | 94 (49.2) | 97 (50.8) | 1.00 |   | 1.00 |   |
| GA |   |   | 97 (31.6) | 28 (28.9) | 69 (71.1) | 0.40 (0.23-0.69) | 0.001 | 1.35 (1.01-1.81) | 0.05 |
| AA |   |   | 19 (6.2) | 6 (31.6) | 13 (68.4) | 0.47 (0.16-1.37) | 0.17 | 1.28 (0.75-2.21) | 0.37 |
| Dominant |   |   |   |   |   | 0.41 (0.24-0.68) | 0.001 | 1.34 (1.01-1.77) | 0.04 |
| Recessive |   |   |   |   |   | 0.63 (0.22-1.79) | 0.38 | 1.14 (0.67-1.94) | 0.62 |
| Ptrend e |   |   |   |   |   | 0.52 (0.34-0.79) | 0.002 | 1.22 (0.98-1.51) | 0.07 |
| rs1653586f | CAMKK2 | hsa-miR-185 |   |   |   |   |   |   |   |
| GG | (UTR-3) |   | 257 (82.1) | 118 (45.9) | 139 (54.1) | 1.00 |   | 1.00 |   |
| GT |   |   | 56 (17.9) | 12 (21.4) | 44 (78.6) | 0.33 (0.16-0.67) | 0.002 | 1.50 (1.06-2.13) | 0.02 |
| TT |   |   | 0 (0.0) | 0 (0.0) | 0 (0.0) | - | - | - | - |
| Dominant |   |   |   |   |   | 0.33 (0.16-0.67) | 0.002 | 1.50 (1.06-2.13) | 0.02 |
| Recessive |   |   |   |   |   | - | - | - | - |
| Ptrend e |   |   |   |   |   | 0.33 (0.16-0.67) | 0.002 | 1.50 (1.06-2.13) | 0.02 |

Table 4 Caption: Associations between EXO1 rs1047840 and CAMKK2 rs1653586 and clinical outcomes

### Figure 1.

![Figure 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a7bb/9809306/4bf8cdd8e54b/jcav13p3701g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9809306_jcav13p3701g001.jpg)

Overall survival curves according to (A) EXO1 rs1047840G>A (B) CAMKK2 rs1653586G>T genotypes. P values by multivariate Cox proportional hazard model.

Next, to investigate whether the *EXO1* rs1047840G>A and *CAMKK2* rs1653586G>T modulates the binding of miRNAs and consequently influence the expression level of *EXO1* and *CAMKK2*, psiCHECK^TM^-2-*EXO1* containing rs1047840G>A and psiCHECK^TM^-2-*CAMKK2* containing rs1653586G>T were generated and co-transfected into H1299 cells with miR-30e and miR-185, respectively. The CLASH data showed that the binding between *EXO1* and miR-30e was non-canonical (Figure [2](#F2)A) and luciferase assay showed that the *Renilla* luciferase activity was significantly increased in *EXO1* rs1047840A allele compared with rs1047840G allele (*P* = 0.03, Figure [2](#F2)B). These results suggest that rs1047840G>A altered the binding of miR-30e and consequently increased the expression of *EXO1*. However, there was no difference in the luciferase activity between *CAMKK2* rs1653586 G and T alleles (data not shown).

### Figure 2.

![Figure 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a7bb/9809306/c156015657c3/jcav13p3701g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9809306_jcav13p3701g002.jpg)

Functional analysis of EXO1 rs1047840G>A by dual luciferase reporter assay. (A) Schematic representation of reporter plasmids containing EXO1 rs1047840G>A, and the complementarity between miR-30e and EXO1 coding sequence targeted. (B) Renilla luciferase assay for the effect of miRNA binding on rs1047840G>A using H1299 cells. Renilla luciferase activity was normalized to firefly luciferase activity and data are presented relative to the Mock control. Each bar represents mean ± SE. P values by Student's t-test. R-luc, Renilla luciferase; F-luc, firefly luciferase; HSV-TK, Herpes simplex virus thymidine kinase.

## Discussion

In this study, we investigated whether the treatment outcomes of pemetrexed chemotherapy in lung adenocarcinoma patients are different according to the genotypes of genetic polymorphisms in miRNA target sites. The study revealed that two SNPs in miRNA binding sites, *EXO1* rs1047840G>A and *CAMKK2* rs1653586G>T, could predict the response to pemetrexed chemotherapy and survival outcome. The luciferase assay suggested that rs1047840G>A alters the binding of miRNA and consequently increases the expression level of *EXO1*. These findings suggest that the two SNPs, especially *EXO1* rs1047840G>A, could be used as potential biomarkers to predict therapeutic outcomes in lung adenocarcinoma patients treated with pemetrexed chemotherapy, which may help to establish an optimal personalized treatment strategy.

Human exonuclease 1 (EXO1) is a member of the Rad2/XPG family of nucleases which exhibits 5'→3' exonuclease, 5' flap endonuclease, and 5'→3' RNase H activity [24](#B24)^, ^[25](#B25). This enzyme mainly contributes to the regulation of cell cycle checkpoints and post-replicative DNA repair pathways, such as mismatch repair (MMR), translesion DNA synthesis (TLS), nucleotide excision repair (NER), and double-strand break repair (DSBR) [26](#B26)^-^[30](#B30). Dysfunction of *EXO1* could impair DNA repair processes which can cause replication stress followed by genomic instability and development of cancer [31](#B31). *EXO1* genetic variants have been associated with the risk and prognosis of different types of cancers, including lung cancer [32](#B32)^-^[37](#B37). In addition, *EXO1* overexpression and its relation to the resistance to chemotherapy or radiotherapy and poor prognosis was reported in multiple types of cancers [38](#B38)^-^[41](#B41). Pemetrexed interferes with folate metabolism leading to ineffective DNA synthesis and tumor cell growth failure [5](#B5). Inhibition of TS by pemetrexed induces uracil misincorporation into DNA resulting in cell death, and base excision repair (BER) system is responsible for the removal of misincorporated uracil. Therefore, the upregulation of BER is associated with pemetrexed resistance and is found in small cell and squamous cell carcinoma which are known as the pemetrexed-resistant histological subtypes of lung cancer [42](#B42). Likewise, given its role in DNA damage response and cell cycle checkpoint activation, the upregulation of *EXO1* may lead to the resistance to pemetrexed chemotherapy. In this study, *EXO1* rs1047840G>A was associated with worse clinical outcomes in lung adenocarcinoma patients after pemetrexed chemotherapy and linked to increased expression of *EXO1*. Based on our results, it is postulated that the rs1047840-induced change in miRNA binding efficiency may increase the expression level of *EXO1*, resulting in worse response to pemetrexed chemotherapy and survival outcomes. Although many previous studies have investigated *EXO1* polymorphism, especially rs1047840, in lung cancer, most of those studies evaluated the relationship between *EXO1* rs1047840 and the risk of lung cancer (33, 35, 44). Regarding the chemotherapy outcomes, one study by R.Li *et al*. showed the association between *EXO1* rs9350 and survival of patients receiving platinum-based chemotherapy in NSCLC (37). To the best of our knowledge, this is the first study to suggest a predictive role of *EXO1* rs1047840 in the therapeutic response to pemetrexed chemotherapy and also survival in lung adenocarcinoma patients. In addition, by the SNP selection using a database for miRNA binding site variants, we could suggest that the rs1047840-induced change in miRNA binding efficiency may be a mechanism of the association between *EXO1* rs1047840 and lung cancer. Alternatively, the *EXO1* rs1047840G>A is a non-synonymous SNP (E589K) with an amino acid change, which may affect EXO1 protein functions. Further studies are required to understand the role of EXO1 in the mechanism of pemetrexed resistance. On the other hand, potential oncogenic roles of EXO1 have also been suggested, which may be another possible mechanism of the association between *EXO1* rs1047840G>A and worse clinical outcomes after pemetrexed chemotherapy. Expression of degradation-resistant EXO1 could result in DNA hyper-resection in homologous recombination, which severely compromised DSBR and lead to chromosomal instability [43](#B43). A bioinformatic analysis study showed that EXO1 was identified as one of the hub genes which were markedly associated with poor prognosis in patients with lung cancer [44](#B44). Recently, Zhou *et al.* showed that *EXO1* was overexpressed in lung adenocarcinoma tissues and the high expression of *EXO1* was associated with poor prognosis [45](#B45). By bioinformatic analyses using public database, they revealed the correlation between increased *EXO1* expression and decreased tumor infiltrating B cells and CD4+ T cells, suggesting that EXO1 may induce an immune-suppressive tumor microenvironment in lung adenocarcinoma. Given that immune checkpoint blockade combined with pemetrexed-platinum doublet chemotherapy has been approved as the standard first-line treatment for lung adenocarcinoma, it will be worth to further evaluate *EXO1* rs1047840G>A as a biomarker to predict clinical outcomes of combined immunotherapy and pemetrexed chemotherapy.

In this study, *CAMKK2* rs1653586G>T was significantly related with the worse response to pemetrexed and survival outcome in lung adenocarcinoma patients. Calmodulin (CaM) is an intracellular calcium receptor which controls the downstream calcium/calmodulin kinase (CaMK) cascade that has been associated with various metabolic diseases [46](#B46). Calcium/calmodulin-dependent kinase kinase 2 (CAMKK2) belongs to the subfamily of calcium/calmodulin-dependent protein kinase and is involved in the maintenance of energy balance, adiposity, glucose homeostasis, inflammation, and cancer [47](#B47). *CAMKK2* is generally upregulated in several cancers, including prostate cancer, and high expression of *CAMKK2* was correlated with poor overall survival in hepatocellular carcinoma and glioma [48](#B48)^-^[50](#B50). It was reported that knockdown of *CAMKK2* potentiated the effect of carboplatin in ovarian cancer by modulating Akt pathway [51](#B51). However, the role of *CAMKK2* in the pathogenesis and prognosis of lung cancer has yet to be clarified. Further study is required to understand the molecular mechanism of the association between the genetic variants and clinical outcomes.

There are some limitations in this study. First, because pemetrexed maintenance regimen was not approved in the early part of the enrollment period, maintenance therapy was not prescribed for some patients who are considered eligible for the maintenance therapy by the current treatment guidelines. Although maintenance therapy was adjusted for in the multivariate analysis, the prognostic differences between patients with and without pemetrexed maintenance therapy could not be ignored. Second, as a single center retrospective study, the treatment other than pemetrexed in the course of treatment including targeted agents could not be strictly controlled in this study, which should be considered in the interpretation of the survival analysis. Third, the study cohort is relatively small. Although stratified analyses may show the clinical significance of *EXO1* rs1047840 and *CAMKK2* rs1653586 in different subgroups of patients, i.e. stage III/IV and postsurgical recurrence groups, this study is underpowered for stratification analyses or propensity score matching. Therefore, a well-designed and properly powered study is warranted to validate our findings.

In conclusion, two SNPs in miRNA binding sites, especially *EXO1* rs1047840G>A, were associated with the treatment response to pemetrexed chemotherapy and survival in lung adenocarcinoma patients. Evaluation of genetic variants in miRNA binding sites may be useful for refining therapeutic decisions in the treatment of lung adenocarcinoma by helping to identify subgroups of patients who will benefit from pemetrexed chemotherapy. To verify *EXO1* rs1047840G>A and *CAMKK2* rs1653586G>T as biomarkers for predicting clinical outcomes, the results of this study need to be further tested in a larger population with diverse ethnicity.

## Supplementary Material

## Acknowledgments

### Funding Statement

This research was supported by Kyungpook National University Research Fund, 2021.

### Ethical Approval Statement

The biospecimens for this study were provided by National Biobank of Korea-Kyungpook National University Hospital (KNUH), which is supported by the Ministry of Health, Welfare and affairs. All materials derived from the National Biobank of Korea-KNUH were obtained (with informed consent) under institutional review board (IRB)-approved protocols.

### Data Availability Statement

The datasets and materials in this study are available on reasonable request from corresponding authors or first author.

### Author Contributions

Conceived and designed the study: S.Y.L. and J.Y.P. Acquired clinical data: J.E.P., S.Y.L., Y.S., S.H.C., Y.H.L., H.S., J.Y.J., K.M.S., S.S.Y., J.L., S.I.C., C.H.K., and J.Y.P. Performed experiments: M.J.H., J.H.L., J.E.C., H.-G.K., and S.K.D. Analyzed and interpreted data: M.J.H., J.E.P., J.H.L., W.K.L., S.Y.L., and J.Y.P. Wrote main manuscript: M.J.H., J.E.P., and S.Y.L. Supervised the study: S.Y.L. and J.Y.P. All authors reviewed the manuscript.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The datasets and materials in this study are available on reasonable request from corresponding authors or first author.

### Supplementary Materials

### Data Availability Statement

The datasets and materials in this study are available on reasonable request from corresponding authors or first author.

## References

1. Thai AA, Solomon BJ, Sequist LV. et al. Lung cancer. Lancet. 2021;398:535–554. doi: 10.1016/S0140-6736(21)00312-3.  [DOI](https://doi.org/10.1016/S0140-6736(21)00312-3) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34273294/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lancet&title=Lung%20cancer&author=AA%20Thai&author=BJ%20Solomon&author=LV%20Sequist&volume=398&publication_year=2021&pages=535-554&pmid=34273294&doi=10.1016/S0140-6736(21)00312-3&)

2. Yuan M, Huang LL, Chen JH. et al. The emerging treatment landscape of targeted therapy in non-small-cell lung cancer. Signal Transduct Target Ther. 2019;4:61. doi: 10.1038/s41392-019-0099-9.  [DOI](https://doi.org/10.1038/s41392-019-0099-9) | [PMC free article](/articles/PMC6914774/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31871778/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Signal%20Transduct%20Target%20Ther&title=The%20emerging%20treatment%20landscape%20of%20targeted%20therapy%20in%20non-small-cell%20lung%20cancer&author=M%20Yuan&author=LL%20Huang&author=JH%20Chen&volume=4&publication_year=2019&pages=61&pmid=31871778&doi=10.1038/s41392-019-0099-9&)

3. Mamdani H, Matosevic S, Khalid AB. et al. Immunotherapy in Lung Cancer: Current Landscape and Future Directions. Front Immunol. 2022;13:823618. doi: 10.3389/fimmu.2022.823618.  [DOI](https://doi.org/10.3389/fimmu.2022.823618) | [PMC free article](/articles/PMC8864096/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35222404/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&title=Immunotherapy%20in%20Lung%20Cancer:%20Current%20Landscape%20and%20Future%20Directions&author=H%20Mamdani&author=S%20Matosevic&author=AB%20Khalid&volume=13&publication_year=2022&pages=823618&pmid=35222404&doi=10.3389/fimmu.2022.823618&)

4. Gandhi L, Rodriguez-Abreu D, Gadgeel S. et al. Pembrolizumab plus Chemotherapy in Metastatic Non-Small-Cell Lung Cancer. N Engl J Med. 2018;378:2078–2092. doi: 10.1056/NEJMoa1801005.  [DOI](https://doi.org/10.1056/NEJMoa1801005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29658856/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Med&title=Pembrolizumab%20plus%20Chemotherapy%20in%20Metastatic%20Non-Small-Cell%20Lung%20Cancer&author=L%20Gandhi&author=D%20Rodriguez-Abreu&author=S%20Gadgeel&volume=378&publication_year=2018&pages=2078-2092&pmid=29658856&doi=10.1056/NEJMoa1801005&)

5. Adjei AA. Pharmacology and mechanism of action of pemetrexed. Clin Lung Cancer. 2004;5(Suppl 2):S51–S55. doi: 10.3816/clc.2004.s.003.  [DOI](https://doi.org/10.3816/clc.2004.s.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15117425/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Lung%20Cancer&title=Pharmacology%20and%20mechanism%20of%20action%20of%20pemetrexed&author=AA%20Adjei&volume=5&issue=Suppl%202&publication_year=2004&pages=S51-S55&pmid=15117425&doi=10.3816/clc.2004.s.003&)

6. Scagliotti G, Hanna N, Fossella F. et al. The differential efficacy of pemetrexed according to NSCLC histology: a review of two Phase III studies. Oncologist. 2009;14:253–263. doi: 10.1634/theoncologist.2008-0232.  [DOI](https://doi.org/10.1634/theoncologist.2008-0232) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19221167/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncologist&title=The%20differential%20efficacy%20of%20pemetrexed%20according%20to%20NSCLC%20histology:%20a%20review%20of%20two%20Phase%20III%20studies&author=G%20Scagliotti&author=N%20Hanna&author=F%20Fossella&volume=14&publication_year=2009&pages=253-263&pmid=19221167&doi=10.1634/theoncologist.2008-0232&)

7. Takezawa K, Okamoto I, Okamoto W. et al. Thymidylate synthase as a determinant of pemetrexed sensitivity in non-small cell lung cancer. Br J Cancer. 2011;104:1594–1601. doi: 10.1038/bjc.2011.129.  [DOI](https://doi.org/10.1038/bjc.2011.129) | [PMC free article](/articles/PMC3101907/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21487406/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&title=Thymidylate%20synthase%20as%20a%20determinant%20of%20pemetrexed%20sensitivity%20in%20non-small%20cell%20lung%20cancer&author=K%20Takezawa&author=I%20Okamoto&author=W%20Okamoto&volume=104&publication_year=2011&pages=1594-1601&pmid=21487406&doi=10.1038/bjc.2011.129&)

8. Krawczyk P, Kucharczyk T, Kowalski DM. et al. Polymorphisms in TS, MTHFR and ERCC1 genes as predictive markers in first-line platinum and pemetrexed therapy in NSCLC patients. J Cancer Res Clin Oncol. 2014;140:2047–2057. doi: 10.1007/s00432-014-1756-6.  [DOI](https://doi.org/10.1007/s00432-014-1756-6) | [PMC free article](/articles/PMC4228108/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25028118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Cancer%20Res%20Clin%20Oncol&title=Polymorphisms%20in%20TS,%20MTHFR%20and%20ERCC1%20genes%20as%20predictive%20markers%20in%20first-line%20platinum%20and%20pemetrexed%20therapy%20in%20NSCLC%20patients&author=P%20Krawczyk&author=T%20Kucharczyk&author=DM%20Kowalski&volume=140&publication_year=2014&pages=2047-2057&pmid=25028118&doi=10.1007/s00432-014-1756-6&)

9. Fennell DA, Myrand SP, Nguyen TS. et al. Association between gene expression profiles and clinical outcome of pemetrexed-based treatment in patients with advanced non-squamous non-small cell lung cancer: exploratory results from a phase II study. PLoS One. 2014;9:e107455. doi: 10.1371/journal.pone.0107455.  [DOI](https://doi.org/10.1371/journal.pone.0107455) | [PMC free article](/articles/PMC4175467/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25250715/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Association%20between%20gene%20expression%20profiles%20and%20clinical%20outcome%20of%20pemetrexed-based%20treatment%20in%20patients%20with%20advanced%20non-squamous%20non-small%20cell%20lung%20cancer:%20exploratory%20results%20from%20a%20phase%20II%20study&author=DA%20Fennell&author=SP%20Myrand&author=TS%20Nguyen&volume=9&publication_year=2014&pages=e107455&pmid=25250715&doi=10.1371/journal.pone.0107455&)

10. Bartel DP. Metazoan MicroRNAs. Cell. 2018;173:20–51. doi: 10.1016/j.cell.2018.03.006.  [DOI](https://doi.org/10.1016/j.cell.2018.03.006) | [PMC free article](/articles/PMC6091663/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29570994/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Metazoan%20MicroRNAs&author=DP%20Bartel&volume=173&publication_year=2018&pages=20-51&pmid=29570994&doi=10.1016/j.cell.2018.03.006&)

11. Bartel DP. MicroRNAs: genomics, biogenesis, mechanism, and function. Cell. 2004;116:281–297. doi: 10.1016/s0092-8674(04)00045-5.  [DOI](https://doi.org/10.1016/s0092-8674(04)00045-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14744438/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=MicroRNAs:%20genomics,%20biogenesis,%20mechanism,%20and%20function&author=DP%20Bartel&volume=116&publication_year=2004&pages=281-297&pmid=14744438&doi=10.1016/s0092-8674(04)00045-5&)

12. Ng R, Song G, Roll GR. et al. A microRNA-21 surge facilitates rapid cyclin D1 translation and cell cycle progression in mouse liver regeneration. J Clin Invest. 2012;122:1097–1108. doi: 10.1172/JCI46039.  [DOI](https://doi.org/10.1172/JCI46039) | [PMC free article](/articles/PMC3287214/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22326957/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&title=A%20microRNA-21%20surge%20facilitates%20rapid%20cyclin%20D1%20translation%20and%20cell%20cycle%20progression%20in%20mouse%20liver%20regeneration&author=R%20Ng&author=G%20Song&author=GR%20Roll&volume=122&publication_year=2012&pages=1097-1108&pmid=22326957&doi=10.1172/JCI46039&)

13. Rayner KJ, Esau CC, Hussain FN. et al. Inhibition of miR-33a/b in non-human primates raises plasma HDL and lowers VLDL triglycerides. Nature. 2011;478:404–407. doi: 10.1038/nature10486.  [DOI](https://doi.org/10.1038/nature10486) | [PMC free article](/articles/PMC3235584/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22012398/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nature&title=Inhibition%20of%20miR-33a/b%20in%20non-human%20primates%20raises%20plasma%20HDL%20and%20lowers%20VLDL%20triglycerides&author=KJ%20Rayner&author=CC%20Esau&author=FN%20Hussain&volume=478&publication_year=2011&pages=404-407&pmid=22012398&doi=10.1038/nature10486&)

14. Zhang P, Bill K, Liu J. et al. MiR-155 is a liposarcoma oncogene that targets casein kinase-1alpha and enhances beta-catenin signaling. Cancer Res. 2012;72:1751–1762. doi: 10.1158/0008-5472.CAN-11-3027.  [DOI](https://doi.org/10.1158/0008-5472.CAN-11-3027) | [PMC free article](/articles/PMC3319789/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22350414/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=MiR-155%20is%20a%20liposarcoma%20oncogene%20that%20targets%20casein%20kinase-1alpha%20and%20enhances%20beta-catenin%20signaling&author=P%20Zhang&author=K%20Bill&author=J%20Liu&volume=72&publication_year=2012&pages=1751-1762&pmid=22350414&doi=10.1158/0008-5472.CAN-11-3027&)

15. Taganov KD, Boldin MP, Chang KJ. et al. NF-kappaB-dependent induction of microRNA miR-146, an inhibitor targeted to signaling proteins of innate immune responses. Proc Natl Acad Sci U S A. 2006;103:12481–12486. doi: 10.1073/pnas.0605298103.  [DOI](https://doi.org/10.1073/pnas.0605298103) | [PMC free article](/articles/PMC1567904/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16885212/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&title=NF-kappaB-dependent%20induction%20of%20microRNA%20miR-146,%20an%20inhibitor%20targeted%20to%20signaling%20proteins%20of%20innate%20immune%20responses&author=KD%20Taganov&author=MP%20Boldin&author=KJ%20Chang&volume=103&publication_year=2006&pages=12481-12486&pmid=16885212&doi=10.1073/pnas.0605298103&)

16. Calin GA, Croce CM. MicroRNA-cancer connection: the beginning of a new tale. Cancer Res. 2006;66:7390–7394. doi: 10.1158/0008-5472.CAN-06-0800.  [DOI](https://doi.org/10.1158/0008-5472.CAN-06-0800) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16885332/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&title=MicroRNA-cancer%20connection:%20the%20beginning%20of%20a%20new%20tale&author=GA%20Calin&author=CM%20Croce&volume=66&publication_year=2006&pages=7390-7394&pmid=16885332&doi=10.1158/0008-5472.CAN-06-0800&)

17. Iqbal MA, Arora S, Prakasam G. et al. MicroRNA in lung cancer: role, mechanisms, pathways and therapeutic relevance. Mol Aspects Med. 2019;70:3–20. doi: 10.1016/j.mam.2018.07.003.  [DOI](https://doi.org/10.1016/j.mam.2018.07.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30102929/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Aspects%20Med&title=MicroRNA%20in%20lung%20cancer:%20role,%20mechanisms,%20pathways%20and%20therapeutic%20relevance&author=MA%20Iqbal&author=S%20Arora&author=G%20Prakasam&volume=70&publication_year=2019&pages=3-20&pmid=30102929&doi=10.1016/j.mam.2018.07.003&)

18. Preskill C, Weidhaas JB. SNPs in microRNA binding sites as prognostic and predictive cancer biomarkers. Critical Reviews™ in Oncogenesis. 2013;18:327–40. doi: 10.1615/critrevoncog.2013007254.  [DOI](https://doi.org/10.1615/critrevoncog.2013007254) | [PMC free article](/articles/PMC4028830/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23614619/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Critical%20Reviews%E2%84%A2%20in%20Oncogenesis&title=SNPs%20in%20microRNA%20binding%20sites%20as%20prognostic%20and%20predictive%20cancer%20biomarkers&author=C%20Preskill&author=JB%20Weidhaas&volume=18&publication_year=2013&pages=327-40&pmid=23614619&doi=10.1615/critrevoncog.2013007254&)

19. Helwak A, Kudla G, Dudnakova T. et al. Mapping the human miRNA interactome by CLASH reveals frequent noncanonical binding. Cell. 2013;153:654–665. doi: 10.1016/j.cell.2013.03.043.  [DOI](https://doi.org/10.1016/j.cell.2013.03.043) | [PMC free article](/articles/PMC3650559/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23622248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell&title=Mapping%20the%20human%20miRNA%20interactome%20by%20CLASH%20reveals%20frequent%20noncanonical%20binding&author=A%20Helwak&author=G%20Kudla&author=T%20Dudnakova&volume=153&publication_year=2013&pages=654-665&pmid=23622248&doi=10.1016/j.cell.2013.03.043&)

20. Ciuleanu T, Brodowicz T, Zielinski C. et al. Maintenance pemetrexed plus best supportive care versus placebo plus best supportive care for non-small-cell lung cancer: a randomised, double-blind, phase 3 study. The Lancet. 2009;374:1432–1440. doi: 10.1016/S0140-6736(09)61497-5.  [DOI](https://doi.org/10.1016/S0140-6736(09)61497-5) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19767093/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20Lancet&title=Maintenance%20pemetrexed%20plus%20best%20supportive%20care%20versus%20placebo%20plus%20best%20supportive%20care%20for%20non-small-cell%20lung%20cancer:%20a%20randomised,%20double-blind,%20phase%203%20study&author=T%20Ciuleanu&author=T%20Brodowicz&author=C%20Zielinski&volume=374&publication_year=2009&pages=1432-1440&pmid=19767093&doi=10.1016/S0140-6736(09)61497-5&)

21. Eisenhauer EA, Therasse P, Bogaerts J. et al. New response evaluation criteria in solid tumours: revised RECIST guideline (version 1.1) Eur J Cancer. 2009;45:228–247. doi: 10.1016/j.ejca.2008.10.026.  [DOI](https://doi.org/10.1016/j.ejca.2008.10.026) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19097774/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=New%20response%20evaluation%20criteria%20in%20solid%20tumours:%20revised%20RECIST%20guideline%20(version%201.1)&author=EA%20Eisenhauer&author=P%20Therasse&author=J%20Bogaerts&volume=45&publication_year=2009&pages=228-247&pmid=19097774&doi=10.1016/j.ejca.2008.10.026&)

22. Bhattacharya A, Ziebarth JD, Cui Y. PolymiRTS Database 3.0: linking polymorphisms in microRNAs and their target sites with human diseases and biological pathways. Nucleic Acids Res. 2014;42(Database issue):D86–91. doi: 10.1093/nar/gkt1028.  [DOI](https://doi.org/10.1093/nar/gkt1028) | [PMC free article](/articles/PMC3965097/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24163105/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=PolymiRTS%20Database%203.0:%20linking%20polymorphisms%20in%20microRNAs%20and%20their%20target%20sites%20with%20human%20diseases%20and%20biological%20pathways&author=A%20Bhattacharya&author=JD%20Ziebarth&author=Y%20Cui&volume=42&issue=Database%20issue&publication_year=2014&pages=D86-91&pmid=24163105&doi=10.1093/nar/gkt1028&)

23. Higgins ME, Claremont M, Major JE. et al. CancerGenes: a gene selection resource for cancer genome projects. Nucleic Acids Res. 2007;35(Database issue):D721–726. doi: 10.1093/nar/gkl811.  [DOI](https://doi.org/10.1093/nar/gkl811) | [PMC free article](/articles/PMC1781153/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17088289/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=CancerGenes:%20a%20gene%20selection%20resource%20for%20cancer%20genome%20projects&author=ME%20Higgins&author=M%20Claremont&author=JE%20Major&volume=35&issue=Database%20issue&publication_year=2007&pages=D721-726&pmid=17088289&doi=10.1093/nar/gkl811&)

24. Wilson DM, Coleman MA, Adamson AW. et al. Hex1: a new human Rad2 nuclease family member with homology to yeast exonuclease 1. Nucleic acids research. 1998;26:3762–3768. doi: 10.1093/nar/26.16.3762.  [DOI](https://doi.org/10.1093/nar/26.16.3762) | [PMC free article](/articles/PMC147753/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9685493/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20acids%20research&title=Hex1:%20a%20new%20human%20Rad2%20nuclease%20family%20member%20with%20homology%20to%20yeast%20exonuclease%201&author=DM%20Wilson&author=MA%20Coleman&author=AW%20Adamson&volume=26&publication_year=1998&pages=3762-3768&pmid=9685493&doi=10.1093/nar/26.16.3762&)

25. Qiu J, Qian Y, Chen V. et al. Human exonuclease 1 functionally complements its yeast homologues in DNA recombination, RNA primer removal, and mutation avoidance. J Biol Chem. 1999;274:17893–17900. doi: 10.1074/jbc.274.25.17893.  [DOI](https://doi.org/10.1074/jbc.274.25.17893) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/10364235/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Human%20exonuclease%201%20functionally%20complements%20its%20yeast%20homologues%20in%20DNA%20recombination,%20RNA%20primer%20removal,%20and%20mutation%20avoidance&author=J%20Qiu&author=Y%20Qian&author=V%20Chen&volume=274&publication_year=1999&pages=17893-17900&pmid=10364235&doi=10.1074/jbc.274.25.17893&)

26. Szankasi P, Smith GR. A role for exonuclease I from S. pombe in mutation avoidance and mismatch correction. Science. 1995;267:1166–1169. doi: 10.1126/science.7855597.  [DOI](https://doi.org/10.1126/science.7855597) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/7855597/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Science&title=A%20role%20for%20exonuclease%20I%20from%20S.%20pombe%20in%20mutation%20avoidance%20and%20mismatch%20correction&author=P%20Szankasi&author=GR%20Smith&volume=267&publication_year=1995&pages=1166-1169&pmid=7855597&doi=10.1126/science.7855597&)

27. Sertic S, Mollica A, Campus I. et al. Coordinated Activity of Y Family TLS Polymerases and EXO1 Protects Non-S Phase Cells from UV-Induced Cytotoxic Lesions. Mol Cell. 2018;70:34–47. doi: 10.1016/j.molcel.2018.02.017. e34.  [DOI](https://doi.org/10.1016/j.molcel.2018.02.017) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29551515/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cell&title=Coordinated%20Activity%20of%20Y%20Family%20TLS%20Polymerases%20and%20EXO1%20Protects%20Non-S%20Phase%20Cells%20from%20UV-Induced%20Cytotoxic%20Lesions&author=S%20Sertic&author=A%20Mollica&author=I%20Campus&volume=70&publication_year=2018&pages=34-47&pmid=29551515&doi=10.1016/j.molcel.2018.02.017&)

28. Qiu J, Guan M-X, Bailis AM. et al. Saccharomyces cerevisiae exonuclease-1 plays a role in UV resistance that is distinct from nucleotide excision repair. Nucleic acids research. 1998;26:3077–3083. doi: 10.1093/nar/26.13.3077.  [DOI](https://doi.org/10.1093/nar/26.13.3077) | [PMC free article](/articles/PMC147686/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/9628902/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20acids%20research&title=Saccharomyces%20cerevisiae%20exonuclease-1%20plays%20a%20role%20in%20UV%20resistance%20that%20is%20distinct%20from%20nucleotide%20excision%20repair&author=J%20Qiu&author=M-X%20Guan&author=AM%20Bailis&volume=26&publication_year=1998&pages=3077-3083&pmid=9628902&doi=10.1093/nar/26.13.3077&)

29. Bolderson E, Tomimatsu N, Richard DJ. et al. Phosphorylation of Exo1 modulates homologous recombination repair of DNA double-strand breaks. Nucleic Acids Res. 2010;38:1821–1831. doi: 10.1093/nar/gkp1164.  [DOI](https://doi.org/10.1093/nar/gkp1164) | [PMC free article](/articles/PMC2847229/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20019063/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&title=Phosphorylation%20of%20Exo1%20modulates%20homologous%20recombination%20repair%20of%20DNA%20double-strand%20breaks&author=E%20Bolderson&author=N%20Tomimatsu&author=DJ%20Richard&volume=38&publication_year=2010&pages=1821-1831&pmid=20019063&doi=10.1093/nar/gkp1164&)

30. Keijzers G, Liu D, Rasmussen LJ. Exonuclease 1 and its versatile roles in DNA repair. Crit Rev Biochem Mol Biol. 2016;51:440–451. doi: 10.1080/10409238.2016.1215407.  [DOI](https://doi.org/10.1080/10409238.2016.1215407) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27494243/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Crit%20Rev%20Biochem%20Mol%20Biol&title=Exonuclease%201%20and%20its%20versatile%20roles%20in%20DNA%20repair&author=G%20Keijzers&author=D%20Liu&author=LJ%20Rasmussen&volume=51&publication_year=2016&pages=440-451&pmid=27494243&doi=10.1080/10409238.2016.1215407&)

31. Keijzers G, Bakula D, Petr MA. et al. Human Exonuclease 1 (EXO1) Regulatory Functions in DNA Replication with Putative Roles in Cancer. Int J Mol Sci. 2018;20:74. doi: 10.3390/ijms20010074.  [DOI](https://doi.org/10.3390/ijms20010074) | [PMC free article](/articles/PMC6337416/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30585186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Mol%20Sci&title=Human%20Exonuclease%201%20(EXO1)%20Regulatory%20Functions%20in%20DNA%20Replication%20with%20Putative%20Roles%20in%20Cancer&author=G%20Keijzers&author=D%20Bakula&author=MA%20Petr&volume=20&publication_year=2018&pages=74&pmid=30585186&doi=10.3390/ijms20010074&)

32. Wang H-C, Chiu C-F, Tsai R-Y. et al. Association of genetic polymorphisms of EXO1 gene with risk of breast cancer in Taiwan. Anticancer research. 2009;29:3897–3901.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/19846925/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20research&title=Association%20of%20genetic%20polymorphisms%20of%20EXO1%20gene%20with%20risk%20of%20breast%20cancer%20in%20Taiwan&author=H-C%20Wang&author=C-F%20Chiu&author=R-Y%20Tsai&volume=29&publication_year=2009&pages=3897-3901&pmid=19846925&)

33. Hsu N-Y, Wang H-C, Wang C-H. et al. Lung cancer susceptibility and genetic polymorphisms of Exo1 gene in Taiwan. Anticancer research. 2009;29:725–730.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/19331228/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20research&title=Lung%20cancer%20susceptibility%20and%20genetic%20polymorphisms%20of%20Exo1%20gene%20in%20Taiwan&author=N-Y%20Hsu&author=H-C%20Wang&author=C-H%20Wang&volume=29&publication_year=2009&pages=725-730&pmid=19331228&)

34. Bayram S, Akkiz H, Bekar A. et al. The significance of Exonuclease 1 K589E polymorphism on hepatocellular carcinoma susceptibility in the Turkish population: a case-control study. Mol Biol Rep. 2012;39:5943–5951. doi: 10.1007/s11033-011-1406-x.  [DOI](https://doi.org/10.1007/s11033-011-1406-x) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22205538/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Rep&title=The%20significance%20of%20Exonuclease%201%20K589E%20polymorphism%20on%20hepatocellular%20carcinoma%20susceptibility%20in%20the%20Turkish%20population:%20a%20case-control%20study&author=S%20Bayram&author=H%20Akkiz&author=A%20Bekar&volume=39&publication_year=2012&pages=5943-5951&pmid=22205538&doi=10.1007/s11033-011-1406-x&)

35. Jin G, Wang H, Hu Z. et al. Potentially functional polymorphisms of EXO1 and risk of lung cancer in a Chinese population: A case-control analysis. Lung Cancer. 2008;60:340–346. doi: 10.1016/j.lungcan.2007.11.003.  [DOI](https://doi.org/10.1016/j.lungcan.2007.11.003) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18079015/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&title=Potentially%20functional%20polymorphisms%20of%20EXO1%20and%20risk%20of%20lung%20cancer%20in%20a%20Chinese%20population:%20A%20case-control%20analysis&author=G%20Jin&author=H%20Wang&author=Z%20Hu&volume=60&publication_year=2008&pages=340-346&pmid=18079015&doi=10.1016/j.lungcan.2007.11.003&)

36. Nogueira GA, Lourenco GJ, Oliveira CB. et al. Association between genetic polymorphisms in DNA mismatch repair-related genes with risk and prognosis of head and neck squamous cell carcinoma. Int J Cancer. 2015;137:810–818. doi: 10.1002/ijc.29435.  [DOI](https://doi.org/10.1002/ijc.29435) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25598504/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&title=Association%20between%20genetic%20polymorphisms%20in%20DNA%20mismatch%20repair-related%20genes%20with%20risk%20and%20prognosis%20of%20head%20and%20neck%20squamous%20cell%20carcinoma&author=GA%20Nogueira&author=GJ%20Lourenco&author=CB%20Oliveira&volume=137&publication_year=2015&pages=810-818&pmid=25598504&doi=10.1002/ijc.29435&)

37. Li R, Gu J, Heymach JV. et al. Hypoxia pathway genetic variants predict survival of non-small-cell lung cancer patients receiving platinum-based chemotherapy. Carcinogenesis. 2017;38:419–424. doi: 10.1093/carcin/bgx014.  [DOI](https://doi.org/10.1093/carcin/bgx014) | [PMC free article](/articles/PMC5963494/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28186269/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&title=Hypoxia%20pathway%20genetic%20variants%20predict%20survival%20of%20non-small-cell%20lung%20cancer%20patients%20receiving%20platinum-based%20chemotherapy&author=R%20Li&author=J%20Gu&author=JV%20Heymach&volume=38&publication_year=2017&pages=419-424&pmid=28186269&doi=10.1093/carcin/bgx014&)

38. Dai Y, Tang Z, Yang Z. et al. EXO1 overexpression is associated with poor prognosis of hepatocellular carcinoma patients. Cell Cycle. 2018;17:2386–2397. doi: 10.1080/15384101.2018.1534511.  [DOI](https://doi.org/10.1080/15384101.2018.1534511) | [PMC free article](/articles/PMC6237436/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30328366/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cell%20Cycle&title=EXO1%20overexpression%20is%20associated%20with%20poor%20prognosis%20of%20hepatocellular%20carcinoma%20patients&author=Y%20Dai&author=Z%20Tang&author=Z%20Yang&volume=17&publication_year=2018&pages=2386-2397&pmid=30328366&doi=10.1080/15384101.2018.1534511&)

39. de Sousa JF, Torrieri R, Serafim RB. et al. Expression signatures of DNA repair genes correlate with survival prognosis of astrocytoma patients. Tumour Biol. 2017;39:1010428317694552. doi: 10.1177/1010428317694552.  [DOI](https://doi.org/10.1177/1010428317694552) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28378638/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Tumour%20Biol&title=Expression%20signatures%20of%20DNA%20repair%20genes%20correlate%20with%20survival%20prognosis%20of%20astrocytoma%20patients&author=JF%20de%20Sousa&author=R%20Torrieri&author=RB%20Serafim&volume=39&publication_year=2017&pages=1010428317694552&pmid=28378638&doi=10.1177/1010428317694552&)

40. He D, Li T, Sheng M. et al. Exonuclease 1 (Exo1) Participates in Mammalian Non-Homologous End Joining and Contributes to Drug Resistance in Ovarian Cancer. Med Sci Monit. 2020;26:e918751. doi: 10.12659/MSM.918751.  [DOI](https://doi.org/10.12659/MSM.918751) | [PMC free article](/articles/PMC7092659/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/32167078/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Med%20Sci%20Monit&title=Exonuclease%201%20(Exo1)%20Participates%20in%20Mammalian%20Non-Homologous%20End%20Joining%20and%20Contributes%20to%20Drug%20Resistance%20in%20Ovarian%20Cancer&author=D%20He&author=T%20Li&author=M%20Sheng&volume=26&publication_year=2020&pages=e918751&pmid=32167078&doi=10.12659/MSM.918751&)

41. Zhou J, Wang Y, Wang Y. et al. FOXM1 modulates cisplatin sensitivity by regulating EXO1 in ovarian cancer. PLoS One. 2014;9:e96989. doi: 10.1371/journal.pone.0096989.  [DOI](https://doi.org/10.1371/journal.pone.0096989) | [PMC free article](/articles/PMC4019642/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24824601/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=FOXM1%20modulates%20cisplatin%20sensitivity%20by%20regulating%20EXO1%20in%20ovarian%20cancer&author=J%20Zhou&author=Y%20Wang&author=Y%20Wang&volume=9&publication_year=2014&pages=e96989&pmid=24824601&doi=10.1371/journal.pone.0096989&)

42. Weeks LD, Fu P, Gerson SL. Uracil-DNA glycosylase expression determines human lung cancer cell sensitivity to pemetrexed. Mol Cancer Ther. 2013;12:2248–2260. doi: 10.1158/1535-7163.MCT-13-0172.  [DOI](https://doi.org/10.1158/1535-7163.MCT-13-0172) | [PMC free article](/articles/PMC4201938/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23873851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&title=Uracil-DNA%20glycosylase%20expression%20determines%20human%20lung%20cancer%20cell%20sensitivity%20to%20pemetrexed&author=LD%20Weeks&author=P%20Fu&author=SL%20Gerson&volume=12&publication_year=2013&pages=2248-2260&pmid=23873851&doi=10.1158/1535-7163.MCT-13-0172&)

43. Tomimatsu N, Mukherjee B, Harris JL. et al. DNA-damage-induced degradation of EXO1 exonuclease limits DNA end resection to ensure accurate DNA repair. J Biol Chem. 2017;292:10779–10790. doi: 10.1074/jbc.M116.772475.  [DOI](https://doi.org/10.1074/jbc.M116.772475) | [PMC free article](/articles/PMC5491765/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28515316/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=DNA-damage-induced%20degradation%20of%20EXO1%20exonuclease%20limits%20DNA%20end%20resection%20to%20ensure%20accurate%20DNA%20repair&author=N%20Tomimatsu&author=B%20Mukherjee&author=JL%20Harris&volume=292&publication_year=2017&pages=10779-10790&pmid=28515316&doi=10.1074/jbc.M116.772475&)

44. Li Z, Sang M, Tian Z. et al. Identification of key biomarkers and potential molecular mechanisms in lung cancer by bioinformatics analysis. Oncol Lett. 2019;18:4429–4440. doi: 10.3892/ol.2019.10796.  [DOI](https://doi.org/10.3892/ol.2019.10796) | [PMC free article](/articles/PMC6781723/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31611952/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncol%20Lett&title=Identification%20of%20key%20biomarkers%20and%20potential%20molecular%20mechanisms%20in%20lung%20cancer%20by%20bioinformatics%20analysis&author=Z%20Li&author=M%20Sang&author=Z%20Tian&volume=18&publication_year=2019&pages=4429-4440&pmid=31611952&doi=10.3892/ol.2019.10796&)

45. Zhou CS, Feng MT, Chen X. et al. Exonuclease 1 (EXO1) is a Potential Prognostic Biomarker and Correlates with Immune Infiltrates in Lung Adenocarcinoma. Onco Targets Ther. 2021;14:1033–1048. doi: 10.2147/OTT.S286274.  [DOI](https://doi.org/10.2147/OTT.S286274) | [PMC free article](/articles/PMC7894803/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/33623391/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Onco%20Targets%20Ther&title=Exonuclease%201%20(EXO1)%20is%20a%20Potential%20Prognostic%20Biomarker%20and%20Correlates%20with%20Immune%20Infiltrates%20in%20Lung%20Adenocarcinoma&author=CS%20Zhou&author=MT%20Feng&author=X%20Chen&volume=14&publication_year=2021&pages=1033-1048&pmid=33623391&doi=10.2147/OTT.S286274&)

46. Marcelo KL, Means AR, York B. The Ca(2+)/Calmodulin/CaMKK2 Axis: Nature's Metabolic CaMshaft. Trends Endocrinol Metab. 2016;27:706–718. doi: 10.1016/j.tem.2016.06.001.  [DOI](https://doi.org/10.1016/j.tem.2016.06.001) | [PMC free article](/articles/PMC5035586/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27449752/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Trends%20Endocrinol%20Metab&title=The%20Ca(2+)/Calmodulin/CaMKK2%20Axis:%20Nature's%20Metabolic%20CaMshaft&author=KL%20Marcelo&author=AR%20Means&author=B%20York&volume=27&publication_year=2016&pages=706-718&pmid=27449752&doi=10.1016/j.tem.2016.06.001&)

47. Racioppi L, Means AR. Calcium/calmodulin-dependent protein kinase kinase 2: roles in signaling and pathophysiology. J Biol Chem. 2012;287:31658–31665. doi: 10.1074/jbc.R112.356485.  [DOI](https://doi.org/10.1074/jbc.R112.356485) | [PMC free article](/articles/PMC3442500/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22778263/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Calcium/calmodulin-dependent%20protein%20kinase%20kinase%202:%20roles%20in%20signaling%20and%20pathophysiology&author=L%20Racioppi&author=AR%20Means&volume=287&publication_year=2012&pages=31658-31665&pmid=22778263&doi=10.1074/jbc.R112.356485&)

48. Massie CE, Lynch A, Ramos-Montoya A. et al. The androgen receptor fuels prostate cancer by regulating central metabolism and biosynthesis. The EMBO journal. 2011;30:2719–2733. doi: 10.1038/emboj.2011.158.  [DOI](https://doi.org/10.1038/emboj.2011.158) | [PMC free article](/articles/PMC3155295/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21602788/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=The%20EMBO%20journal&title=The%20androgen%20receptor%20fuels%20prostate%20cancer%20by%20regulating%20central%20metabolism%20and%20biosynthesis&author=CE%20Massie&author=A%20Lynch&author=A%20Ramos-Montoya&volume=30&publication_year=2011&pages=2719-2733&pmid=21602788&doi=10.1038/emboj.2011.158&)

49. Lin F, Marcelo KL, Rajapakshe K. et al. The camKK2/camKIV relay is an essential regulator of hepatic cancer. Hepatology. 2015;62:505–520. doi: 10.1002/hep.27832.  [DOI](https://doi.org/10.1002/hep.27832) | [PMC free article](/articles/PMC4515151/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25847065/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Hepatology&title=The%20camKK2/camKIV%20relay%20is%20an%20essential%20regulator%20of%20hepatic%20cancer&author=F%20Lin&author=KL%20Marcelo&author=K%20Rajapakshe&volume=62&publication_year=2015&pages=505-520&pmid=25847065&doi=10.1002/hep.27832&)

50. Liu DM, Wang HJ, Han B. et al. CAMKK2, Regulated by Promoter Methylation, is a Prognostic Marker in Diffuse Gliomas. CNS Neurosci Ther. 2016;22:518–524. doi: 10.1111/cns.12531.  [DOI](https://doi.org/10.1111/cns.12531) | [PMC free article](/articles/PMC6492904/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27012733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CNS%20Neurosci%20Ther&title=CAMKK2,%20Regulated%20by%20Promoter%20Methylation,%20is%20a%20Prognostic%20Marker%20in%20Diffuse%20Gliomas&author=DM%20Liu&author=HJ%20Wang&author=B%20Han&volume=22&publication_year=2016&pages=518-524&pmid=27012733&doi=10.1111/cns.12531&)

51. Gocher AM, Azabdaftari G, Euscher LM. et al. Akt activation by Ca(2+)/calmodulin-dependent protein kinase kinase 2 (CaMKK2) in ovarian cancer cells. J Biol Chem. 2017;292:14188–14204. doi: 10.1074/jbc.M117.778464.  [DOI](https://doi.org/10.1074/jbc.M117.778464) | [PMC free article](/articles/PMC5572912/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28634229/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&title=Akt%20activation%20by%20Ca(2+)/calmodulin-dependent%20protein%20kinase%20kinase%202%20(CaMKK2)%20in%20ovarian%20cancer%20cells&author=AM%20Gocher&author=G%20Azabdaftari&author=LM%20Euscher&volume=292&publication_year=2017&pages=14188-14204&pmid=28634229&doi=10.1074/jbc.M117.778464&)
