# Germline Polymorphisms as Biomarkers of Tumor Response in Colorectal Cancer Patients Treated with Anti-EGFR Monoclonal Antibodies: A Systematic Review and Meta-Analysis

## Metadata
**Authors:** Eric K Morgen, Heinz-Josef Lenz, Derek J Jonker, Dongsheng Tu, Gerard Milano, Francesco Graziano, John Zalcberg, Christos S Karapetis, Alexander Dobrovic, Chris J O’Callaghan, Geoffrey Liu
**Journal:** The pharmacogenomics journal
**Date:** 2016 Nov 29
**DOI:** [10.1038/tpj.2016.56](https://doi.org/10.1038/tpj.2016.56)
**PMID:** 27897268
**PMCID:** PMC9536193
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9536193/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9536193/pdf/nihms-796212.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9536193/pdf/nihms-796212.pdf)

## Abstract

Studies of germline polymorphisms as predictors of tumor response to anti-EGFR monocloncal antibody agents in metastatic colorectal cancer have reported inconsistent results. We performed a systematic review of studies from 1990 to September, 2015, followed by random-effects meta-analyses for polymorphisms examined in at least three studies. Of 87 studies, 40 passed criteria for systematic review and 23 for meta-analysis. The polymorphisms suitable for meta-analysis were: CCND1 (rs17852153), COX2 (rs20417), EGF (rs4444903), EGFR (rs712829, rs11543848, 3’UTR CA repeat), FCGR2A (rs1801274), FCGR3A (rs396991), IL8 (rs4073), KRAS (rs61764370), and VEGFA (rs3025039). Meta-analysis yielded nominal significance (at alpha=0.05) for rs4444903 and rs11543848, but showed no significant results after multiple testing correction; this was unchanged by sensitivity analyses to address subgroups, funnel-plot asymmetries, and study quality. This highlights a tendency for lack of replication in the face of initial positive results, and possibly the unsuitability of relying on tumor response as a surrogate marker in this setting.

Keywords: colorectal cancer, cetuximab, panitumumab, EGFR, polymorphism, meta-analysis

## 1. INTRODUCTION

Colorectal cancer (CRC) is a leading cause of cancer death, with a large fraction of patients developing advanced or metastatic disease.^1^ For most of these patients, systemic control of disease is paramount, and is now achievable via targeted therapies that directly inhibit molecular drivers of tumor proliferation. By far the most commonly used of these therapies are monoclonal antibodies to the epidermal growth factor receptor (EGFR), which include cetuximab and panitumumab. These drugs not only help to achieve systemic control in metastatic disease after other agents have failed, but also have a much-improved side-effect profile compared to traditional therapies such as irinotecan, oxaliplatin, and fluoropyrimidines.^2^

Because the majority of CRC patients show no response to anti-EGFR monoclonal antibody therapies, considerable efforts are underway to identify up-front the patients who will respond, so that the rest can be spared the time, expense and side effects of an ineffective treatment. One advance in this arena has been the recognition that tumor *KRAS* mutations are strongly associated with non-response to anti-EGFR drugs,^3^ since which *KRAS* testing has become routine. However, even after such testing, more than half of patients still show no response to anti-EGFR drugs^4,5^ indicating a pressing need for additional research. This ongoing effort has led to discovery and adoption of *NRAS/HRAS* testing in many jurisdictions,^6,7^ and has identified some well-studied candidate mutations in genes such as *BRAF*, *PIK3CA*, and *PTEN* whose effectiveness as predictive markers remains uncertain.^5,8,9^

Genetic alterations with potential as predictive biomarkers in this scenario may affect either tumor (somatic) or patient (germline) DNA. Alterations of somatic tumor DNA (i.e. “tumor gene mutations”) directly affect tumor cells, and can thus alter tumor characteristics such as growth rate, invasiveness, metastatic potential, and vulnerability to particular drugs. In contrast, alterations in germline DNA (i.e. “genetic polymorphisms”) directly affect patient cells and can thus influence patient factors such as drug bioavailability, kinetics, and metabolism, as well as host immune interactions and local tissue responses.

While many germline polymorphisms have been proposed as biomarkers in anti-EGFR monoclonal antibody treated CRC, studies have often yielded inconsistent results. This may be due to the lack of an underlying true association, heterogeneity of study population, or low power in smaller studies. For these reasons, we undertook a systematic review and meta-analysis to evaluate the association of these polymorphisms as putative clinical biomarkers of response to cetuximab/panitumumab therapy. We focused on response as an outcome because: (1) response was much more widely and uniformly reported among studies in this area, and (2) the majority of studies did not have control arms of patients not treated with cetuximab/panitumumab, rendering time-to-event outcomes incapable of distinguishing between prognostic and predictive associations.

## 2. MATERIALS AND METHODS

### Systematic review methodology

Ovid MEDLINE was searched using a date-range of January, 1990 to September, 2015. The search string used was: “(exp Polymorphism, Genetic/ or polymorphism*.mp.) and (exp Colorectal Neoplasms/ or (colon or rectal or colorectal).mp.) and (neoplas* or tumor* or tumour* or cancer or carcinoma or adenocarcinoma).mp. and (cetuximab or panitumumab).mp.” Results were limited to English language publications and studies in human subjects. Methods used to identify additional papers included checking reference lists, communicating with researchers in the field, and using web search engines such as Google Scholar. The resulting studies were manually reviewed, with access of the full text articles as necessary to determine eligibility for systematic review. The inclusion criterion for the systematic review was that included studies had to evaluate one or more germline polymorphism as predictors of tumor response in colorectal cancer patients who were treated with an anti-epidermal growth factor receptor (anti-EGFR) monoclonal antibody (i.e. cetuximab or panitumumab). Exclusion criteria for the systematic review were as follows: review articles, case reports, studies with fewer than 25 patients, duplicate citations, repeat analyses of data published elsewhere, and presentations of subsets of data published elsewhere.

For eligible publications, the following data were abstracted: study authors, year of publication, characteristics of the patient population (country of recruitment or ethnicity, line of therapy, therapy during the study, total number of patients, intent of therapy, the KRAS status of included patients), the number and identity of tested polymorphisms, the reported outcomes, and whether sufficient data was reported to allow meta-analysis for each outcome. The study methodology was categorized as either prospective or retrospective (depending on method of cohort recruitment), with notation of whether the cohort was ascertained within a Phase II or randomized clinical trial. For studies with incomplete reporting of detailed results, attempts were made to contact study authors to obtain complete information. Authors were asked to provide either summary statistics or raw data from which summary statistics (e.g. counts) could be calculated.

### Meta-Analysis Methodology

Polymorphisms identified in the systematic review were evaluated for their suitability for meta-analysis. Only polymorphisms with published results from at least three separate studies were included. Studies were thus included in the meta-analysis if they (1) reported data on an appropriate polymorphism and (2) reported data in sufficient detail to allow meta-analysis (i.e. counts of responders and non-responders with each genotype).

The study quality was evaluated using a schema based on the “Strengthening the Reporting of Genetic Association studies” (STREGA) framework^10^. Two independent reviewers read the full text of each article (and accessed supplementary published materials where needed), and assigned the study a score between 1 and 3 in each of 27 categories (see [Appendix 1](#T4)); results were totaled and averaged to produce a summary score for each article. Concordance was compared between the two reviewers, and then their scores were averaged to produce the final article score. The article scores were then plotted as a histogram, and equal-sized bins chosen to divide the articles into four categories (adequate, good, very good, excellent), as none fell into the poor category by qualitative analysis. A sensitivity analysis was then performed by evaluating each meta-analysis result, where possible, using only articles in the very good and excellent categories. Because response was the primary outcome for meta-analysis, included studies were also reviewed to determine the specific definition of response used.

For each polymorphism, alleles associated with improved outcomes in prior studies were recorded. These alleles were evaluated via meta-analysis using R (v2.15.1)^11^ and the metafor package (v1.7–0)^12^ under the assumptions of additive, dominant, and recessive genetic inheritance models. A random effects model was used, with evaluation of heterogeneity using the Higgins’ I^2^ summary statistic.^13^ The model with the lowest p-value for each polymorphism was retained, provided that the direction of association remained as previously proposed. In a few cases where certain studies had reported results only for particular genetic models, these models were preferred for meta-analysis to allow inclusion of the broadest range of data. Correction for multiple testing was performed using the Benjamini-Hochberg procedure to limit the false discovery rate to less than 5%. The potential for publication bias was evaluated using funnel plots.

For each polymorphism, we performed multiple sensitivity analyses that excluded: (1) studies showing up as unbalanced points in funnel plots; (2) KRAS unselected studies (i.e. including only studies with KRAS wild-type patients); (3) KRAS wild-type studies (i.e. including only studies with patients unselected by KRAS status).; (4) studies with only poor or adequate quality ratings; and (5) studies showing deviations from Hardy-Weinberg equilibrium (relevant to the *FCGR* polymorphisms only). A few sensitivity analyses did increase the strength of association for polymorphisms with borderline statistical significance, for instance to p=0.004 for *EGF* 61 in one sensitivity analysis; nonetheless, none retained significance at q < 0.05 after correcting for even the baseline level of multiple comparisons in this meta-analysis, let alone the additional tests involved in the sensitivity analysis.

## 3. RESULTS

### Systematic review results

The systematic review yielded 87 studies, of which 47 failed to meet inclusion or exclusion criteria ([Figure 1](#F1)). Of the 40 remaining articles retained for systematic review, 23 were included in the subsequent meta-analysis. Of the 17 excluded studies, seven had insufficient data on covariate or response variables, nine involved polymorphisms that were assessed in fewer than three separate studies, and one^14^ was excluded for reporting only disease control rate in patients with *KRAS* mutant (rather than *KRAS* wild-type) tumors. Of those included in the meta-analysis, one study consisted of new data, published for the first time as part of this meta-analysis, on two of the polymorphisms (*FCGR2A* and *FCGR3A*), from analysis of samples from CO.17, a Phase III clinical trial of cetuximab versus best supportive care.^2^ These 23 studies were included for meta-analysis, and their characteristics are shown in [Table 1](#T1).

### Figure 1. Systematic review methodology and results.

![Figure 1.](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c22a/9536193/acd649edea6c/nihms-796212-f0001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9536193_nihms-796212-f0001.jpg)

### Table 1. Summary of studies eligible for meta analysis, grouped by study methodology.

| Author | Year | Region | Study Methodology | N | Intent of Therapy | Other Chemotherapy | KRAS Status | Reported Outcomes |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |  |
| Saridaki 22 | 2014 | Belgium, France, USA | mixed | 559 | mixed | mixed | unselected | response, PFS, OS |
|   |  |  |  |  |  |  |  |  |
| Loupakis 23 | 2014 | California | prospective cohort | 113 | mixed | irinotecan | wild-type | response, PFS, OS |
| Garm Spindler 24 | 2009 | Denmark | prospective cohort | 71 | salvage | irinotecan | unselected | response, PFS, OS |
| Graziano 25 | 2008 | Italy | prospective cohort | 110 | salvage | irinotecan | unselected | response, PFS, OS |
| Inoue 26 | 2014 | Japan | prospective cohort | 57 | salvage | mixed | wild-type | response, PFS, OS |
|   |  |  |  |  |  |  |  |  |
| Sclafani 27 | 2014 | response, PFS, OS | prospective cohort within Phase II trial | 105 | adjuvant | capecitabine, oxaliplatin | wild-type | response, PFS, OS |
| Etienne-Grimaldi 28 | 2012 | France | prospective cohort within Phase II trial | 52 | first-line | irinotecan, folinic acid, UFT | unselected | response, OS |
| Lurje 29 | 2008 | California | prospective cohort within Phase II trial | 130 | salvage | none | unselected | response, PFS, OS |
| Zhang 30 | 2010 | California | prospective cohort within Phase II trial | 65 | salvage | mixed | unselected | response, TTP, OS |
| Zhang 31 | 2011 | California | prospective cohort within Phase II trial | 111 | salvage | none | unselected | response, PFS, OS |
| Kjersem 32 | 2012 | Norway | prospective cohort within RCT | 180 | first-line | oxalplatin, folinic acid, UFT | unselected | response, PFS, OS |
| Kjersem 19 | 2014 | Norway | prospective cohort within RCT | 504 | first-line | oxalplatin, folinic acid, UFT | unselected | response, PFS, OS |
|   |  |  |  |  |  |  |  |  |
| Jonker 2 | 2015 | Canada & Australia | prospective cohort within RCT | 138 | salvage | none | unselected | response, OS |
|   |  |  |  |  |  |  |  |  |
| Hsieh 33 | 2012 | Taiwan | retrospective cohort | 118 | first-line | oxalplatin, folinic acid, UFT | wild-type | response, PFS, OS |
| Negri 34 | 2014 | Italy | retrospective cohort | 86 | mixed | mixed | unselected | response, TTP, OS |
| Calemma 35 | 2012 | Italy | retrospective cohort | 50 | mixedC/P | mixed | wild-type | response, PFS, OS |
| Hu-Lieskovan 36 | 2011 | Europe | retrospective cohort | 130 | neoadjuvantLA | mixed | unselected | pathologic response |
| Bibeau 37 | 2009 | France | retrospective cohort | 69 | salvage | irinotecan | unselected | response, PFS, OS |
| Dahan 38 | 2011 | France | retrospective cohort | 58 | salvage | irinotecan | unselected | response, TTP, DSS |
| Park 39 | 2012 | Korean | retrospective cohort | 118 | salvage | mixed | unselected | response, PFS, OS |
| Paez 40 | 2010 | Caucasian | retrospective cohort | 104 | salvageC/P | mixed | unselected | response, PFS |
| Sebio 41 | 2013 | Spain | retrospective cohort | 100 | salvageC/P | mixed | wild-type | response |
| Geva 18 | 2015 | Europe (multicenter) | retrospective cohort | 740 | salvage | mixed | wild-type | response, PFS, OS, DCR |

Table 1 Caption: N = number participants; UFT = Tegafur/uracil; PFS = progression free survival; OS = overall survival; TTP = time to progression; DSS = disease free survival; LA = patients had locally advanced disease; C/P = patients received either cetuximab and panitumumab

Most studies (57%) used a prospective cohort design, with the remainder used a retrospective cohort design. Most studies (56%) gave anti-EGFR agents as a salvage therapy, while a few (22%) gave it as first-line or neoadjuvant therapy, and the remainder were mixed. KRAS status was unknown in most studies (70%) and limited to wild-type in the remainder. All studies reported response, and most also reported one or more survival outcomes. The geographic location of studies and their ethnic composition were diverse. While most studies investigated survival outcomes, incomplete reporting of these outcomes was common and was a major limitation to meta-analysis. For example, many studies reported full results (i.e. at least an effect size and measure of precision) only for selected polymorphisms, often those that were the most statistically significant. Included studies were subjected to quality review by two independent reviewers. These reviewers demonstrated good agreement with a Spearman correlation coefficient of 0.90, while the mean quality score was 1.96 (range 1.56 – 2.29), and thus the category cut-offs were chosen as: adequate (<1.8), good (1.8-<2.0), very good (2.0-<2.2), and excellent (≥2.2). The resulting quality ratings are also shown in [Table 2](#T2), along with a summary of which studies investigated which of the polymorphisms that were studied via meta-analysis. The definitions of tumor response used are also shown in [Table 2](#T2); most were variations of the RECIST criteria.^15^

### Table 2. Matrix illustrating, for each study, which polymorphisms it investigated, its quality rating, and its stated definition of response.

| Study Reference | FCGR2A 131 R>H | FCGR3A 158 F>V | EGF 61 A>G | EGFR 497 R>K | KRAS Let-7 T>G | EGFR 3’UTR (CA)n S>L | EGFR −216 G>T | CCND1 870 A>G | VEGF 936 C>T | COX2 −765 G>C | IL-8 −251 T>A | Study Quality | Definition of Response |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Saridaki, 2014 |   |   |   |   | X |   |   |   |   |   |   | Good | Objective Resp. Rate |
| Loupakis, 2014 |   |   |   |   |   | X |   |   |   |   |   | Very Good | RECIST 1.0 |
| Garm Spindler, 2009 |   |   | X |   |   |   |   |   |   |   |   | Very Good | RECIST 1.0 |
| Graziano, 2008 |   |   | X | X |   | X | X |   |   |   |   | Good | RECIST 1.0 |
| Inoue, 2014 | X | X |   |   |   | X |   |   |   |   |   | Very Good | RECIST |
| Sclafani, 2014 | X | X |   |   |   |   |   |   |   |   |   | Very Good | RECIST |
| Etienne-Grimaldi, 2012 | X | X | X |   |   | X | X |   |   |   |   | Good | RECIST |
| Lurje, 2008 | X | X | X | X |   | X |   | X | X | X | X | Excellent | WHO criteria, modified |
| Zhang, 2010 | X | X | X | X |   | X |   | X | X | X | X | Adequate | RECIST |
| Zhang, 2011 |   |   |   |   | X |   |   |   |   |   |   | Adequate | WHO criteria, modified |
| Kjersem, 2012 |   |   |   |   | X |   |   |   |   |   |   | Very Good | RECIST |
| Kjersem, 2014 | X | X |   |   |   |   |   |   |   |   |   | Good | RECIST |
| Dobrovic, 2015 | X | X |   |   |   |   |   |   |   |   |   | Excluded* | RECIST 1.0 |
| Hsieh, 2012 |   |   |   | X |   |   |   |   |   |   |   | Good | RECIST |
| Negri, 2014 | X | X |   |   |   |   |   |   |   |   |   | Good | RECIST 1.1 |
| Calemma, 2012 | X | X |   |   |   |   |   |   |   |   |   | Good | RECIST |
| Hu-Lieskovan, 2011 | X | X | X | X | X | X |   | X | X | X | X | Good | Dworak grade |
| Bibeau, 2009 | X | X |   |   |   |   |   |   |   |   |   | Good | RECIST 1.0 |
| Dahan, 2011 | X | X | X | X |   |   | X | X |   |   |   | Good | RECIST, modified |
| Park, 2012 | X | X |   |   |   |   |   |   |   |   |   | Very Good | RECIST 1.1 |
| Paez, 2010 | X | X |   |   |   |   |   |   |   |   |   | Good | RECIST 1.0 |
| Sebio, 2013 |   |   |   |   | X |   |   |   |   |   |   | Good | RECIST 1.1 |
| Geva, 2015 | X | X |   |   |   |   |   |   |   |   |   | Very Good | RECIST 1.0 or WHO |

Table 2 Caption: *Jonker, 2007 was excluded because no appropriate polymorphism-related manuscript was available, and because both quality reviewers were involved with the study.

### Meta-analysis of tumor response

Eleven polymorphisms were suitable for meta-analysis (see [Table 2](#T2)). Although some polymorphisms were within genes clearly related to known mechanisms of drug activity (*EGF, EGFR, FCGR, KRAS* polymorphisms), others were relatively tangential (*CCND1, COX2, IL8, VEGF* polymorphisms). Most had some data reporting a putative functional consequence of carrying different polymorphic alleles. Among the relevant studies, both prospective and retrospective cohort study designs were well represented, and several were Phase II/III trials. The median number of patients per analysis was 110 (range 50 to 740).

For each polymorphism, we calculated a pooled effect with associated standard error, and Higgins’ I^2^ (see [Table 3](#T3)), and constructed funnel plots (not shown). Two polymorphisms showed pooled relative risks of response that differed significantly from 1.0 with alpha=0.05: the *EGF* A61G (rs4444903) and *EGFR* R497K (rs11543848) polymorphisms. However, neither result retained statistical significance after correction for multiple testing. Among the polymorphisms not associated with outcome were the *FCGR2A* H131R (rs1801274) and *FCGR3A* F158V (rs396991), which have garnered much interest and involved the largest number of studies among the polymorphisms reviewed.

### Table 3. Meta-analysis results for polymorphisms with at least three studies.

| Polymorphism | OMIM # | RS # | Test Allele (Model) | RR [95% CI] | N | Higgins’ I2 | p-val | q-val |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|   |  |  |  |  |  |  |  |  |
| CCND1 870 A>G | 168461 | rs17852153 | A (Recessive) | 1.14 [0.64, 2.04] | 5 | 34.8% | 0.652 | 0.530 |
| COX2 −765 G>C | 600262 | rs20417 | C (Recessive) | 2.67 [0.69, 10.36] | 3 | 55.0% | 0.155 | 0.427 |
| EGF 61 A>G | 131530 | rs4444903 | G (Recessive) | 1.81 [1.08, 3.02] | 6 | 48.4% | 0.023 | 0.257 |
| EGFR −216 G>T | 131550 | rs712829 | T (Recessive) | 1.27 [0.78, 2.06] | 3 | 0.0% | 0.331 | 0.460 |
| EGFR 497 R>K | 131550 | rs11543848 | K (Recessive) | 1.52 [1.01, 2.31] | 6 | 0.0% | 0.047 | 0.460 |
| EGFR 3’UTR (CA)n S>L | 131550 | N/A | S (Recessive) | 1.23 [0.81, 1.85] | 7 | 45.4% | 0.334 | 0.259 |
| FCGR2A 131 R>H | 146790 | rs1801274 | R (Dominant)* | 1.09 [0.94, 1.27] | 15 | 0.0% | 0.251 | 0.275 |
| FCGR3A 158 F>V | 146740 | rs396991 | V (Recessive) | 1.03 [0.85, 1.24 ] | 15 | 0.0% | 0.781 | 0.781 |
| IL8 −251 T>A | 146930 | rs4073 | T (Recessive) | 1.29 [0.78, 2.13] | 3 | 0.0% | 0.324 | 0.530 |
| KRAS Let-7 T>G | 190070 | rs61764370 | G (Dominant) | 1.36 [0.70, 2.65] | 5 | 83.2% | 0.370 | 0.460 |
| VEGFA 936 C>T | 192240 | rs3025039 | T (Dominant) | 1.34 [0.81, 2.21] | 3 | 0.0% | 0.710 | 0.460 |

Table 3 Caption: Each polymorphism’s common name is listed along with the corresponding OMIM gene number and dbSNP polymorphism number. For each polymorphism, the results of meta-analysis are presented for the genetic model resulting in the lowest p-value. Data presented for each analysis include the pooled relative risk with corresponding confidence interval, number of studies contributing data to the analysis, Higgin’s I2, p-value, and false discovery rate q-value),

## 4. DISCUSSION

The present systematic review and meta-analysis of 23 eligible studies was able to evaluate pooled effects for 11 polymorphisms on tumor response in colorectal cancer patients treated with anti-EGFR monoclonal antibody therapies. Two polymorphisms demonstrated nominal statistical significance, but these associations were not robust to correction for multiple testing.

The published literature studying the association of germline polymorphisms with clinical outcome in anti-EGFR treated colorectal cancer patients presents certain challenges. Most prominently, published studies have historically been quite small, fewer than 150 patients. This has likely been due to logistic difficulties in assembling large patient cohorts from individual centers, the greater cost in dealing with large cohorts, and the fact that in the past, the number of centers offering anti-EGFR drugs and the number of patients receiving such agents were both small. This combination of multiple groups studying small samples increases the risk that any discovered statistically significant association is in fact a false positive. This is a central problem in all biomedical research, and is due in large part to factors beyond the control of individual researchers.^16^

One of the main tools available to address this challenge is meta-analysis, which allows evidence to be pooled across studies and can increase the precision of estimates as well as the statistical power to discover true associations, while also showing individual more extreme results in the context of all similar studies.^17^ In this context, it is expected that many associations showing statistical significance in particular studies will fail to find support after meta-analysis, and that this generally indicates the absence of a relevant underlying association. Indeed, this same trend has been observed in recent large, carefully-conducted studies on certain polymorphisms that failed to replicate significant associations observed in smaller prior studies.^18,19^ However, negative meta-analysis results must be taken with important caveats including study and patient heterogeneity, low minor allele frequencies (limiting meta-analysis study power), low response rates to anti-EGFR agents, and inappropriate use of response rate as a surrogate for survival in this context, trade-offs in statistical modeling, and poor reporting of outcomes in the published literature. We address each of these issues below.

### Study Heterogeneity

An important limitation in this study was the heterogeneity of included study designs. While all studies enrolled patients with colorectal cancer (usually metastatic) and evaluated the association of polymorphisms with response to anti-EGFR therapy, there was room for substantial variation. Patients varied in whether they had prior surgery or chemotherapy, how many prior lines of chemotherapy might have been used, and which regimens had been previously tried. Studies also varied in whether patients were taking anti-EGFR drugs as monotherapy or along with other drugs, and also in the choice of any such drugs. The criteria employed for evaluating tumor response varied (see [Table 2](#T2)). The *KRAS* status of study patients (and indeed, whether *KRAS* testing was performed at all) also varied, with many studies of all patients regardless of KRAS status, and several others with only *KRAS* wild-type tumors. Finally, the country of recruitment and thus ethnicity distribution of patients varied considerably, with potential masking of significant associations that may be present only within particular ethnic groups. These differences represent a limitation that is difficult to fully resolve when the literature typically includes only three to six studies per polymorphism, which is too small a number for meta-regression.

However, as an alternate approach to address these sources of heterogeneity, along with the possibility that they may have led to false negative results, we performed multiple sensitivity analyses (see results section), some of which produced lower nominal p-values, but none of which were robust to correction for multiple testing, rendering them unlikely to be of great promise. Of course, it is difficult to entirely exclude the possibility a relevant, statistically significant result may be discoverable for one or more polymorphisms within some alternate subgroup of studies. However, an exhaustive search for such associations would also undoubtedly reveal many more false positives than true associations.

### Statistical Modeling Tradeoffs

We chose the more conservative random effects model for meta-analysis rather than a fixed effects model. While this may reduce the study’s ability to identify true associations, the random effects model is most suited to situations with heterogeneity between study designs,^20^ which is certainly abundant in this case.

We corrected for multiple comparisons relating to two different factors: (1) the multiple polymorphisms being tested and (2) the multiple genetic models being tested per polymorphism. However, different genetic models do not represent independent statistical tests, since there is a strong correlation between the results of recessive, dominant, and recessive models for a given allele. Thus typical methods of correction, which assume independence of statistical tests, run the risk of being too conservative. We addressed this concern in two ways. First, we chose to limit the false discovery rate to q < 0.05 using the well-known Benjamini-Hochberg procedure. Second, realizing that even this procedure could yield overly stringent results for the stated false discovery rate in the setting of dependent tests, we performed a sensitivity analysis at a more liberal rate of q < 0.2, which still failed to show any significant results.

Finally, it could be argued that our approach to meta-analysis, which included separate tests for each of three genetic models, is less likely to find true associations due to the necessary correction for multiple testing. This argument would posit that had we instead chosen a single genetic model for each polymorphism based on prior publications, a significant association might have been found. To address this concern, we simulated perfect foreknowledge by picking the lowest p-value model for each polymorphism and correcting for multiple testing using only those 11 models. The conclusions were unchanged, with no model retaining significance at the chosen cutoff of q < 0.05. Even extending this approach of perfect foreknowledge further to allow selection of the lowest p-value model for each polymorphism including all sensitivity analyses, conclusions remained unchanged with no model retaining significance.

### Limitations of Tumor Response as an Outcome

An important limitation of analyses of tumor response in cetuximab-treated colorectal carcinoma is the low rates of tumor response in the published literature. In CO.17, the study which originally demonstrated cetuximab efficacy in metastatic CRC, the response rate in the cetuximab treatment group was only 8%. This introduces a challenge with statistical power – even assuming that a beneficial effect exists for certain polymorphisms, this low rate would make it difficult to demonstrate. This issue is beginning to be addressed by new larger studies, as well as by the present meta-analysis.

However, an even more fundamental point relates to the hypothesized action of anti-EGFR agents in CRC. The low observed response rate may be a sign that the beneficial effects of cetuximab on survival (which is ultimately the most important outcome) are largely independent of tumor response. For example, if the mechanism was primarily inhibition of tumor growth (or anti-proliferation, given that EGFR acts on cellular growth) rather than inducing tumor shrinkage, then there may be a true benefit in the absence of any significant tumor response. It would thus be ideal to also perform a meta-analysis of patient survival in this clinical area. Unfortunately, our ability to do this was limited by inconsistent reporting of outcomes among reviewed studies, which is discussed below.

### Inconsistent Outcome Reporting in Published Studies

Many reviewed studies reported survival outcomes, such as progression-free survival and overall survival, in addition to tumor response. Survival outcomes are of prime importance in evaluating oncology therapeutics, including anti-EGFR therapies, and would ideally be included in this study. Indeed, tumor response is often viewed as a surrogate for survival, which is the outcome of ultimate importance (at least for drug effectiveness; for efficacy, in contrast, tumor response may sometimes be the preferred outcome). Unfortunately, it was not possible to perform a meta-analysis of survival in the present study due to incomplete and/or irregular reporting of survival outcomes. Some studies did not investigate survival outcomes; many others did, but reported effect sizes and precisions for only selected analyses. Even when survival data was reported, its format was inconsistent, variably couched as hazard ratios, Kaplan-Meier plots, p-values for log-rank tests, or median survivals by patient group. Such problems are routine in the published survival literature, and represent a significant challenge to meta-analysis generally.^21^ This was particularly true as we also attempted to contact individual studies to request primary source data and/or re-analyzed data according to a single standard, which met with only marginal success. In the present study, meta-analysis was technically possible for selected polymorphisms (i.e. more than two studies reporting adequate survival outcomes for a given polymorphism); however, given the large proportion of relevant published results that would have been excluded due to inadequate reporting, the potential for biased and misleading results would be extreme. Consequently, the present study is necessarily limited to the more proximal outcome of tumor response, and a meta-analysis of survival outcomes must await more uniform reporting, more widespread sharing of unpublished data, or methodological advances that allow the incorporation of studies with incomplete data.

### Conclusions

The present study represents the first systematic review and meta-analysis of germline polymorphisms as biomarkers of tumor response in CRC patients treated with anti-EGFR monoclonal antibody therapy. The resulting pooled analysis, which was possible for 11 of the reviewed polymorphisms, revealed no statistically significant associations after correction for multiple testing. Given the substantial heterogeneity in methodology among included studies, the relatively small numbers of analyzable studies for each polymorphism, and the inability to systematically analyze survival outcomes, this result cannot definitively exclude the possibility of a significant association for one of the included polymorphisms. Nonetheless, these findings were robust to multiple sensitivity analyses, and also parallel an observed trend in recent large, well-conducted studies in the area that have failed to replicate significant associations observed in smaller prior studies.^18,19^

Equally important in this study is how the results serve to highlight important issues for future research in this area. The present results argue for the use of potentially more fruitful approaches through the planning of larger studies, potentially within consortia to leverage the resources of multiple centers, and where possible in adopting an unbiased, genome-wide approach to biomarker discovery that will better facilitate data-sharing, patient-level meta-analysis, and validation of polymorphisms proposed by other groups. Finally, the current literature is quite variable in the reporting of survival outcomes, and it is crucial that future studies publish uniform data regarding all major clinical outcomes for all studied polymorphisms (at least as [supplementary data](#SD1)), in order to minimize publication bias and facilitate aggregation of study results, which will be indispensable to future progress in this area.

## Supplementary Material

### 9. APPENDIX 1. – Criteria used for quality review of included articles.

| Category | Criteria Summary |
| --- | --- |
| title and abstract | design in title or abstract; informative/balanced summary in abstract |
| intro - rationale | explain scientific background and rationale |
| intro - objectives | specific objectives; first report vs replication |
| meth - setting | setting, location, dates |
| meth - participants | eligibility criteria, sources, selection methods |
| meth - variables | outcomes, covariates, variants (standard nomenclature), ethnic confounding |
| meth - measurement | lab methods, source/storage of DNA, genotyping method, allele calling algorithm, error rates, call rates, laboratory identified |
| meth - size | how was study size arrived at |
| meth - Q vars | explain how quantitative variables were handled in analysis (choice of groupings) |
| Stats | describe all statistical methods, including confounding; software, version, options |
|   | any methods to examine subgroups and interactions |
|   | how was missing data addressed |
|   | loss to follow-up (cohort), matching (case/control) |
|   | describe any sensitivity analyses |
|   | state if HWE was considered and how |
|   | describe any methods to address multiple comparisons or risk of false positives |
|   | describe any methods used to address subject relatedness |
| res - participants | numbers of individuals at each stage of study, reasons for non-participation; number successfully genotyped? |
| res - descriptive | participant characteristics, information on exposures and potential confounders; number of participants with missing data for variables of interest; follow-up time |
| res - outcome | outcomes by genotype over time (cohort); summary of outcomes by genotype (case-control) |
| res - main | unadjusted and (IA) adjusted estimates (which covariates?), precisions; report category boundaries if discretized; results of multiple comparisons adjustments |
| res - other | summarize results from all variants analyzed; (IA) how can more detailed results be accessed? |
| disc - results | summarize key results with relation to objectives |
| disc - limits | study limitations discussed (bias, imprecision, direction/magnitude of bias) |
| desc - interp | give cautious interpretation considering limitations, multiple testing, other studies |
| desc - general | discuss generalizability (external validity) |
| other - funds | give sources of funding and role of funders in present study and (IA) for original studies |
## Footnotes

## Contributor Information

Eric K Morgen, Dalla Lana School of Public Health, University of Toronto; Department of Pathobiology and Laboratory Medicine, University of Toronto, Toronto, Ontario, Canada.

Heinz-Josef Lenz, USC/Norris Comprehensive Cancer Center, Los Angeles, California..

Derek J Jonker, The Ottawa Hospital Research Institute, University of Ottawa, Ottawa, Ontario, Canada.

Dongsheng Tu, Canadian Cancer Trials Group, Queen’s University, Kingston, Canada.

Gerard Milano, Laboratoire d’Oncopharmacologie EA 3836, Centre Antoine Lacassagne, Nice, France..

Francesco Graziano, Division of Medical Oncology, Azienda “Ospedali Riuniti Marche Nord”, Pesaro, Italy..

John Zalcberg, Head, Cancer Research Program, School of Public Health and Preventive Medicine Faculty of Medicine, Monash University, Melbourne. Australia.

Christos S Karapetis, Flinders University and Flinders Medical Centre, Adelaide, Australia.

Alexander Dobrovic, Translational Genomics and Epigenomics Laboratory; Olivia Newton-John Cancer Research Institute, Heidelberg, Victoria, Australia; School of Cancer Medicine, La Trobe University, Bundoora, Victoria, Australia; Department of Pathology, University of Melbourne, Parkville, Victoria, Australia..

Chris J O’Callaghan, Canadian Cancer Trials Group, Queen’s University, Kingston, Ontario, Canada.

Geoffrey Liu, Dalla Lana School of Public Health, University of Toronto; Departments of Medicine and Medical Biophysics, University of Toronto, Toronto, Ontario, Canada.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Supplementary Materials

## References

1. Donadon M, Ribero D, Morris-Stiff G, Abdalla EK, Vauthey J-N. New paradigm in the management of liver-only metastases from colorectal cancer. Gastrointest Cancer Res 2007; 1: 20–27.  [PMC free article](/articles/PMC2632517/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19262699/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gastrointest%20Cancer%20Res&title=New%20paradigm%20in%20the%20management%20of%20liver-only%20metastases%20from%20colorectal%20cancer&author=M%20Donadon&author=D%20Ribero&author=G%20Morris-Stiff&author=EK%20Abdalla&author=J-N%20Vauthey&volume=1&publication_year=2007&pages=20-27&pmid=19262699&)

2. Jonker DJ, O’Callaghan CJ, Karapetis CS, Zalcberg JR, Tu D, Au H-J et al. Cetuximab for the treatment of colorectal cancer. N Engl J Medicine 2007; 357: 2040–2048.  [DOI](https://doi.org/10.1056/NEJMoa071834) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18003960/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N%20Engl%20J%20Medicine&title=Cetuximab%20for%20the%20treatment%20of%20colorectal%20cancer&author=DJ%20Jonker&author=CJ%20O%E2%80%99Callaghan&author=CS%20Karapetis&author=JR%20Zalcberg&author=D%20Tu&volume=357&publication_year=2007&pages=2040-2048&pmid=18003960&doi=10.1056/NEJMoa071834&)

3. Bokemeyer C, Bondarenko I, Hartmann JT, de Braud F, Schuch G, Zubel A et al. Efficacy according to biomarker status of cetuximab plus FOLFOX-4 as first-line treatment for metastatic colorectal cancer: the OPUS study. Ann Oncol 2011; 22: 1535–1546.  [DOI](https://doi.org/10.1093/annonc/mdq632) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21228335/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=Efficacy%20according%20to%20biomarker%20status%20of%20cetuximab%20plus%20FOLFOX-4%20as%20first-line%20treatment%20for%20metastatic%20colorectal%20cancer:%20the%20OPUS%20study&author=C%20Bokemeyer&author=I%20Bondarenko&author=JT%20Hartmann&author=F%20de%20Braud&author=G%20Schuch&volume=22&publication_year=2011&pages=1535-1546&pmid=21228335&doi=10.1093/annonc/mdq632&)

4. Soeda H, Shimodaira H, Watanabe M, Suzuki T, Gamoh M, Mori T et al. Clinical usefulness of KRAS, BRAF, and PIK3CA mutations as predictive markers of cetuximab efficacy in irinotecan- and oxaliplatin-refractory Japanese patients with metastatic colorectal cancer. Int J Clin Oncol 2013; 18: 670–677.  [DOI](https://doi.org/10.1007/s10147-012-0422-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22638623/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Clinical%20usefulness%20of%20KRAS,%20BRAF,%20and%20PIK3CA%20mutations%20as%20predictive%20markers%20of%20cetuximab%20efficacy%20in%20irinotecan-%20and%20oxaliplatin-refractory%20Japanese%20patients%20with%20metastatic%20colorectal%20cancer.%20Int&author=H%20Soeda&author=H%20Shimodaira&author=M%20Watanabe&author=T%20Suzuki&author=M%20Gamoh&volume=18&publication_year=2013&pages=670-677&pmid=22638623&doi=10.1007/s10147-012-0422-8&)

5. Prognostic Custodio A. and predictive biomarkers for epidermal growth factor receptor-targeted therapy in colorectal cancer: beyond KRAS mutations. Crit Rev Oncol/Hematol 2013; 85: 45–81.  [DOI](https://doi.org/10.1016/j.critrevonc.2012.05.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22647972/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=KRAS%20mutations.%20Crit%20Rev%20Oncol/Hematol&title=predictive%20biomarkers%20for%20epidermal%20growth%20factor%20receptor-targeted%20therapy%20in%20colorectal%20cancer:%20beyond&author=Custodio%20A.%20Prognostic&volume=85&publication_year=2013&pages=45-81&pmid=22647972&doi=10.1016/j.critrevonc.2012.05.001&)

6. Taniguchi H, Yamazaki K, Yoshino T, Muro K, Yatabe Y, Watanabe T et al. Japanese Society of Medical Oncology Clinical Guidelines: RAS (KRAS/NRAS) mutation testing in colorectal cancer patients. Cancer Sci 2015; 106: 324–327.  [DOI](https://doi.org/10.1111/cas.12595) | [PMC free article](/articles/PMC4376442/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25800101/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&title=Japanese%20Society%20of%20Medical%20Oncology%20Clinical%20Guidelines:%20RAS%20(KRAS/NRAS)%20mutation%20testing%20in%20colorectal%20cancer%20patients&author=H%20Taniguchi&author=K%20Yamazaki&author=T%20Yoshino&author=K%20Muro&author=Y%20Yatabe&volume=106&publication_year=2015&pages=324-327&pmid=25800101&doi=10.1111/cas.12595&)

7. Wong NACS, Gonzalez D, Salto-Tellez M, Butler R, Diaz-Cano SJ, Ilyas M et al. RAS testing of colorectal carcinoma—a guidance document from the Association of Clinical Pathologists Molecular Pathology and Diagnostics Group. J Clin Pathol 2014; 67: 751–757.  [DOI](https://doi.org/10.1136/jclinpath-2014-202467) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24996433/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Pathol&title=RAS%20testing%20of%20colorectal%20carcinoma%E2%80%94a%20guidance%20document%20from%20the%20Association%20of%20Clinical%20Pathologists%20Molecular%20Pathology%20and%20Diagnostics%20Group&author=NACS%20Wong&author=D%20Gonzalez&author=M%20Salto-Tellez&author=R%20Butler&author=SJ%20Diaz-Cano&volume=67&publication_year=2014&pages=751-757&pmid=24996433&doi=10.1136/jclinpath-2014-202467&)

8. Tol J, Dijkstra JR, Klomp M, Teerenstra S, Dommerholt M, Vink-Börger ME et al. Markers for EGFR pathway activation as predictor of outcome in metastatic colorectal cancer patients treated with or without cetuximab. Eur J Cancer 2010; 46: 1997–2009.  [DOI](https://doi.org/10.1016/j.ejca.2010.03.036) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20413299/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=Markers%20for%20EGFR%20pathway%20activation%20as%20predictor%20of%20outcome%20in%20metastatic%20colorectal%20cancer%20patients%20treated%20with%20or%20without%20cetuximab&author=J%20Tol&author=JR%20Dijkstra&author=M%20Klomp&author=S%20Teerenstra&author=M%20Dommerholt&volume=46&publication_year=2010&pages=1997-2009&pmid=20413299&doi=10.1016/j.ejca.2010.03.036&)

9. Bokemeyer C, Van Cutsem E, Rougier P, Ciardiello F, Heeger S, Schlichting M et al. Addition of cetuximab to chemotherapy as first-line treatment for KRAS wild-type metastatic colorectal cancer: pooled analysis of the CRYSTAL and OPUS randomised clinical trials. Eur J Cancer 2012; 48: 1466–1475.  [DOI](https://doi.org/10.1016/j.ejca.2012.02.057) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22446022/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=Addition%20of%20cetuximab%20to%20chemotherapy%20as%20first-line%20treatment%20for%20KRAS%20wild-type%20metastatic%20colorectal%20cancer:%20pooled%20analysis%20of%20the%20CRYSTAL%20and%20OPUS%20randomised%20clinical%20trials&author=C%20Bokemeyer&author=E%20Van%20Cutsem&author=P%20Rougier&author=F%20Ciardiello&author=S%20Heeger&volume=48&publication_year=2012&pages=1466-1475&pmid=22446022&doi=10.1016/j.ejca.2012.02.057&)

10. Little J, Higgins JPT, Ioannidis JPA, Moher D, Gagnon F, von Elm E et al. STrengthening the REporting of Genetic Association Studies (STREGA)--an extension of the STROBE statement. Genet Epidemiol 2009; 33: 581–598.  [DOI](https://doi.org/10.1002/gepi.20410) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19278015/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet%20Epidemiol&title=STrengthening%20the%20REporting%20of%20Genetic%20Association%20Studies%20(STREGA)--an%20extension%20of%20the%20STROBE%20statement&author=J%20Little&author=JPT%20Higgins&author=JPA%20Ioannidis&author=D%20Moher&author=F%20Gagnon&volume=33&publication_year=2009&pages=581-598&pmid=19278015&doi=10.1002/gepi.20410&)

11. R Core Team. The R Project for Statistical Computing. R Foundation for Statistical Computing: Vienna, Austria, 2012. Available at: http://www.r-project.org/.  [http://www.r-project.org/](http://www.r-project.org/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20R%20Project%20for%20Statistical%20Computing&publication_year=2012&)

12. Viechtbauer W. Conducting meta-analyses in R with the metafor Package. J Stat Softw 2010; 36: 1–48.  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Stat%20Softw&title=Conducting%20meta-analyses%20in%20R%20with%20the%20metafor%20Package&author=W%20Viechtbauer&volume=36&publication_year=2010&pages=1-48&)

13. Higgins JPT, Thompson SG. Quantifying heterogeneity in a meta-analysis. Stat Med 2002; 21: 1539–1558.  [DOI](https://doi.org/10.1002/sim.1186) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/12111919/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Stat%20Med&title=Quantifying%20heterogeneity%20in%20a%20meta-analysis&author=JPT%20Higgins&author=SG%20Thompson&volume=21&publication_year=2002&pages=1539-1558&pmid=12111919&doi=10.1002/sim.1186&)

14. Rodríguez J, Zarate R, Bandres E, Boni V, Hernández A, Sola JJ et al. Fc γ receptor polymorphisms as predictive markers of Cetuximab efficacy in epidermal growth factor receptor downstream-mutated metastatic colorectal cancer. Eur J Cancer 2012; 48: 1774–1780.  [DOI](https://doi.org/10.1016/j.ejca.2012.01.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22305465/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer&title=Fc%20%CE%B3%20receptor%20polymorphisms%20as%20predictive%20markers%20of%20Cetuximab%20efficacy%20in%20epidermal%20growth%20factor%20receptor%20downstream-mutated%20metastatic%20colorectal%20cancer&author=J%20Rodr%C3%ADguez&author=R%20Zarate&author=E%20Bandres&author=V%20Boni&author=A%20Hern%C3%A1ndez&volume=48&publication_year=2012&pages=1774-1780&pmid=22305465&doi=10.1016/j.ejca.2012.01.007&)

15. Therasse P, Arbuck SG, Eisenhauer EA, Wanders J, Kaplan RS, Rubinstein L et al. New guidelines to evaluate the response to treatment in solid tumors. J Natl Cancer Inst 2000; 92: 205–216. Polymorphism meta-analysis for anti-EGFR monoclonal antibody response EK Morgen et al 540 The Pharmacogenomics Journal (2017), 535 – 542 © 2017 Macmillan Publishers Limited, part of Springer Nature.10655437  [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&title=New%20guidelines%20to%20evaluate%20the%20response%20to%20treatment%20in%20solid%20tumors&author=P%20Therasse&author=SG%20Arbuck&author=EA%20Eisenhauer&author=J%20Wanders&author=RS%20Kaplan&volume=92&publication_year=2000&pages=205-216&)

16. Ioannidis JPA. Why most published research findings are false. PLoS Med 2005; 2: e124.  [DOI](https://doi.org/10.1371/journal.pmed.0020124) | [PMC free article](/articles/PMC1182327/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16060722/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Med&title=Why%20most%20published%20research%20findings%20are%20false&author=JPA%20Ioannidis&volume=2&publication_year=2005&pages=e124&pmid=16060722&doi=10.1371/journal.pmed.0020124&)

17. Garg AX, Hackam D, Tonelli M. Systematic review and meta-analysis: when one study is just not enough. Clin J Am Soc Nephrol 2008; 3: 253–260.  [DOI](https://doi.org/10.2215/CJN.01430307) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18178786/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20J%20Am%20Soc%20Nephrol&title=Systematic%20review%20and%20meta-analysis:%20when%20one%20study%20is%20just%20not%20enough&author=AX%20Garg&author=D%20Hackam&author=M%20Tonelli&volume=3&publication_year=2008&pages=253-260&pmid=18178786&doi=10.2215/CJN.01430307&)

18. Geva R, Vecchione L, Kalogeras KT, Jensen BV, Lenz H-J, Yoshino T et al. FCGR polymorphisms and cetuximab efficacy in chemorefractory metastatic colorectal cancer: an international consortium study. Gut 2015; 64: 921–928.  [DOI](https://doi.org/10.1136/gutjnl-2014-307234) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25011934/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Gut&title=FCGR%20polymorphisms%20and%20cetuximab%20efficacy%20in%20chemorefractory%20metastatic%20colorectal%20cancer:%20an%20international%20consortium%20study&author=R%20Geva&author=L%20Vecchione&author=KT%20Kalogeras&author=BV%20Jensen&author=H-J%20Lenz&volume=64&publication_year=2015&pages=921-928&pmid=25011934&doi=10.1136/gutjnl-2014-307234&)

19. Kjersem JB, Skovlund E, Ikdahl T, Guren T, Kersten C, Dalsgaard AM et al. FCGR2A and FCGR3A polymorphisms and clinical outcome in metastatic colorectal cancer patients treated with first-line 5-fluorouracil/folinic acid and oxaliplatin +/− cetuximab. BMC Cancer 2014; 14: 340.  [DOI](https://doi.org/10.1186/1471-2407-14-340) | [PMC free article](/articles/PMC4045863/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24884501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=FCGR2A%20and%20FCGR3A%20polymorphisms%20and%20clinical%20outcome%20in%20metastatic%20colorectal%20cancer%20patients%20treated%20with%20first-line%205-fluorouracil/folinic%20acid%20and%20oxaliplatin%20+/%E2%88%92%20cetuximab&author=JB%20Kjersem&author=E%20Skovlund&author=T%20Ikdahl&author=T%20Guren&author=C%20Kersten&volume=14&publication_year=2014&pages=340&pmid=24884501&doi=10.1186/1471-2407-14-340&)

20. Riley RD, Higgins JPT, Deeks JJ. Interpretation of random effects meta-analyses. BMJ 2011; 342: d549.  [DOI](https://doi.org/10.1136/bmj.d549) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21310794/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMJ&title=Interpretation%20of%20random%20effects%20meta-analyses&author=RD%20Riley&author=JPT%20Higgins&author=JJ%20Deeks&volume=342&publication_year=2011&pages=d549&pmid=21310794&doi=10.1136/bmj.d549&)

21. Michiels S, Piedbois P, Burdett S, Syz N, Stewart L, Pignon J-P. Meta-analysis when only the median survival times are known: a comparison with individual patient data results. Int J Technol Assess Health Care 2005; 21: 119–125.  [DOI](https://doi.org/10.1017/s0266462305050154) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15736523/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int%20J%20Technol%20Assess%20Health%20Care&title=Meta-analysis%20when%20only%20the%20median%20survival%20times%20are%20known:%20a%20comparison%20with%20individual%20patient%20data%20results&author=S%20Michiels&author=P%20Piedbois&author=S%20Burdett&author=N%20Syz&author=L%20Stewart&volume=21&publication_year=2005&pages=119-125&pmid=15736523&doi=10.1017/s0266462305050154&)

22. Saridaki Z, Weidhaas JB, Lenz H-J, Laurent-Puig P, Jacobs B, De Schutter J et al. A let-7 microRNA-binding site polymorphism in KRAS predicts improved outcome in patients with metastatic colorectal cancer treated with salvage cetuximab/ panitumumab monotherapy. Clin Cancer Res 2014; 20: 4499–4510.  [DOI](https://doi.org/10.1158/1078-0432.CCR-14-0348) | [PMC free article](/articles/PMC4155520/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25183481/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=A%20let-7%20microRNA-binding%20site%20polymorphism%20in%20KRAS%20predicts%20improved%20outcome%20in%20patients%20with%20metastatic%20colorectal%20cancer%20treated%20with%20salvage%20cetuximab/%20panitumumab%20monotherapy&author=Z%20Saridaki&author=JB%20Weidhaas&author=H-J%20Lenz&author=P%20Laurent-Puig&author=B%20Jacobs&volume=20&publication_year=2014&pages=4499-4510&pmid=25183481&doi=10.1158/1078-0432.CCR-14-0348&)

23. Loupakis F, Antoniotti C, Cremolini C, Zhang W, Yang D, Wakatsuki T et al. Prospective study of EGFR intron 1 (CA)n repeats variants as predictors of benefit from cetuximab and irinotecan in chemo-refractory metastatic colorectal cancer (mCRC) patients. Pharmacogenomics J 2014; 14: 322–327.  [DOI](https://doi.org/10.1038/tpj.2014.1) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24513691/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Prospective%20study%20of%20EGFR%20intron%201%20(CA)n%20repeats%20variants%20as%20predictors%20of%20benefit%20from%20cetuximab%20and%20irinotecan%20in%20chemo-refractory%20metastatic%20colorectal%20cancer%20(mCRC)%20patients&author=F%20Loupakis&author=C%20Antoniotti&author=C%20Cremolini&author=W%20Zhang&author=D%20Yang&volume=14&publication_year=2014&pages=322-327&pmid=24513691&doi=10.1038/tpj.2014.1&)

24. Garm Spindler K- L, Pallisgaard N, Rasmussen AA, Lindebjerg J, Andersen RF, Crüger D et al. The importance of KRAS mutations and EGF61A4G polymorphism to the effect of cetuximab and irinotecan in metastatic colorectal cancer. Ann Oncol 2009; 20: 879–884.  [DOI](https://doi.org/10.1093/annonc/mdn712) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19179548/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=The%20importance%20of%20KRAS%20mutations%20and%20EGF61A4G%20polymorphism%20to%20the%20effect%20of%20cetuximab%20and%20irinotecan%20in%20metastatic%20colorectal%20cancer&author=L%20Garm%20Spindler%20K-&author=N%20Pallisgaard&author=AA%20Rasmussen&author=J%20Lindebjerg&author=RF%20Andersen&volume=20&publication_year=2009&pages=879-884&pmid=19179548&doi=10.1093/annonc/mdn712&)

25. Graziano F, Ruzzo A, Loupakis F, Canestrari E, Santini D, Catalano V et al. Pharmacogenetic profiling for cetuximab plus irinotecan therapy in patients with refractory advanced colorectal cancer. J Clin Oncol 2008; 26: 1427–1434.  [DOI](https://doi.org/10.1200/JCO.2007.12.4602) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18349392/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Pharmacogenetic%20profiling%20for%20cetuximab%20plus%20irinotecan%20therapy%20in%20patients%20with%20refractory%20advanced%20colorectal%20cancer&author=F%20Graziano&author=A%20Ruzzo&author=F%20Loupakis&author=E%20Canestrari&author=D%20Santini&volume=26&publication_year=2008&pages=1427-1434&pmid=18349392&doi=10.1200/JCO.2007.12.4602&)

26. Inoue Y, Hazama S, Iwamoto S, Miyake Y, Matsuda C, Tsunedomi R et al. FcγR and EGFR Polymorphisms as Predictive Markers of Cetuximab Efficacy in Metastatic Colorectal Cancer. Mol Diagn Ther 2014; 18: 541–8.  [DOI](https://doi.org/10.1007/s40291-014-0103-6) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24828248/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol%20Diagn%20Ther&title=Fc%CE%B3R%20and%20EGFR%20Polymorphisms%20as%20Predictive%20Markers%20of%20Cetuximab%20Efficacy%20in%20Metastatic%20Colorectal%20Cancer&author=Y%20Inoue&author=S%20Hazama&author=S%20Iwamoto&author=Y%20Miyake&author=C%20Matsuda&volume=18&publication_year=2014&pages=541-8&pmid=24828248&doi=10.1007/s40291-014-0103-6&)

27. Sclafani F, Gonzalez de Castro D, Cunningham D, Hulkki Wilson S, Peckitt C, Capdevila J et al. FcγRIIa and FcγRIIIa polymorphisms and cetuximab benefit in the microscopic disease. Clin Cancer Res 2014; 20: 4511–4519.  [DOI](https://doi.org/10.1158/1078-0432.CCR-14-0674) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24987061/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Fc%CE%B3RIIa%20and%20Fc%CE%B3RIIIa%20polymorphisms%20and%20cetuximab%20benefit%20in%20the%20microscopic%20disease&author=F%20Sclafani&author=D%20Gonzalez%20de%20Castro&author=D%20Cunningham&author=S%20Hulkki%20Wilson&author=C%20Peckitt&volume=20&publication_year=2014&pages=4511-4519&pmid=24987061&doi=10.1158/1078-0432.CCR-14-0674&)

28. Etienne-Grimaldi M-C, Bennouna J, Formento J-L, Douillard J-Y, Francoual M, Hennebelle I et al. Multifactorial pharmacogenetic analysis in colorectal cancer patients receiving 5-fluorouracil-based therapy together with cetuximabirinotecan. Br J Clin Pharmacol 2012; 73: 776–785.  [DOI](https://doi.org/10.1111/j.1365-2125.2011.04141.x) | [PMC free article](/articles/PMC3403205/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22486600/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br%20J%20Clin%20Pharmacol&title=Multifactorial%20pharmacogenetic%20analysis%20in%20colorectal%20cancer%20patients%20receiving%205-fluorouracil-based%20therapy%20together%20with%20cetuximabirinotecan&author=M-C%20Etienne-Grimaldi&author=J%20Bennouna&author=J-L%20Formento&author=J-Y%20Douillard&author=M%20Francoual&volume=73&publication_year=2012&pages=776-785&pmid=22486600&doi=10.1111/j.1365-2125.2011.04141.x&)

29. Lurje G, Nagashima F, Zhang W, Yang D, Chang HM, Gordon MA et al. Polymorphisms in cyclooxygenase-2 and epidermal growth factor receptor are associated with progression-free survival independent of K-ras in metastatic colorectal cancer patients treated with single-agent cetuximab. Clin Cancer Res 2008; 14: 7884–7895.  [DOI](https://doi.org/10.1158/1078-0432.CCR-07-5165) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19047118/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=Polymorphisms%20in%20cyclooxygenase-2%20and%20epidermal%20growth%20factor%20receptor%20are%20associated%20with%20progression-free%20survival%20independent%20of%20K-ras%20in%20metastatic%20colorectal%20cancer%20patients%20treated%20with%20single-agent%20cetuximab&author=G%20Lurje&author=F%20Nagashima&author=W%20Zhang&author=D%20Yang&author=HM%20Chang&volume=14&publication_year=2008&pages=7884-7895&pmid=19047118&doi=10.1158/1078-0432.CCR-07-5165&)

30. Zhang W, Azuma M, Lurje G, Gordon MA, Yang D, Pohl A et al. Molecular predictors of combination targeted therapies (cetuximab, bevacizumab) in irinotecanrefractory colorectal cancer (BOND-2 study). Anticancer Res 2010; 30: 4209–4217.  [PubMed](https://pubmed.ncbi.nlm.nih.gov/21036743/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Anticancer%20Res&title=Molecular%20predictors%20of%20combination%20targeted%20therapies%20(cetuximab,%20bevacizumab)%20in%20irinotecanrefractory%20colorectal%20cancer%20(BOND-2%20study)&author=W%20Zhang&author=M%20Azuma&author=G%20Lurje&author=MA%20Gordon&author=D%20Yang&volume=30&publication_year=2010&pages=4209-4217&pmid=21036743&)

31. Zhang W, Labonte MJ, Lenz H-J. KRAS let-7 LCS6 SNP predicts cetuximab efficacy in KRASwt metastatic colorectal cancer patients: Does treatment combination partner matter? Ann Oncol 2011; 22: 484–485.  [DOI](https://doi.org/10.1093/annonc/mdq704) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21278223/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&title=KRAS%20let-7%20LCS6%20SNP%20predicts%20cetuximab%20efficacy%20in%20KRASwt%20metastatic%20colorectal%20cancer%20patients:%20Does%20treatment%20combination%20partner%20matter?&author=W%20Zhang&author=MJ%20Labonte&author=H-J%20Lenz&volume=22&publication_year=2011&pages=484-485&pmid=21278223&doi=10.1093/annonc/mdq704&)

32. Kjersem JB, Ikdahl T, Guren T, Skovlund E, Sorbye H, Hamfjord J et al. Let-7 miRNAbinding site polymorphism in the KRAS 3’UTR; colorectal cancer screening population prevalence and influence on clinical outcome in patients with metastatic colorectal cancer treated with 5-fluorouracil and oxaliplatin +/− cetuximab. BMC Cancer 2012; 12: 534.  [DOI](https://doi.org/10.1186/1471-2407-12-534) | [PMC free article](/articles/PMC3526507/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23167843/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Let-7%20miRNAbinding%20site%20polymorphism%20in%20the%20KRAS%203%E2%80%99UTR;%20colorectal%20cancer%20screening%20population%20prevalence%20and%20influence%20on%20clinical%20outcome%20in%20patients%20with%20metastatic%20colorectal%20cancer%20treated%20with%205-fluorouracil%20and%20oxaliplatin%20+/%E2%88%92%20cetuximab&author=JB%20Kjersem&author=T%20Ikdahl&author=T%20Guren&author=E%20Skovlund&author=H%20Sorbye&volume=12&publication_year=2012&pages=534&pmid=23167843&doi=10.1186/1471-2407-12-534&)

33. Hsieh Y-Y, Tzeng C-H, Chen M-H, Chen P-M, Wang W-S. Epidermal growth factor receptor R521K polymorphism shows favorable outcomes in KRAS wild-type colorectal cancer patients treated with cetuximab-based chemotherapy. Cancer Sci 2012; 103: 791–796.  [DOI](https://doi.org/10.1111/j.1349-7006.2012.02225.x) | [PMC free article](/articles/PMC7659261/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22321154/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&title=Epidermal%20growth%20factor%20receptor%20R521K%20polymorphism%20shows%20favorable%20outcomes%20in%20KRAS%20wild-type%20colorectal%20cancer%20patients%20treated%20with%20cetuximab-based%20chemotherapy&author=Y-Y%20Hsieh&author=C-H%20Tzeng&author=M-H%20Chen&author=P-M%20Chen&author=W-S%20Wang&volume=103&publication_year=2012&pages=791-796&pmid=22321154&doi=10.1111/j.1349-7006.2012.02225.x&)

34. Negri FV, Musolino A, Naldi N, Bortesi B, Missale G, Laccabue D et al. Role of immunoglobulin G fragment C receptor polymorphism-mediated antibodydependant cellular cytotoxicity in colorectal cancer treated with cetuximab therapy. Pharmacogenomics J 2014; 14: 14–19.  [DOI](https://doi.org/10.1038/tpj.2012.54) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23296156/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics%20J&title=Role%20of%20immunoglobulin%20G%20fragment%20C%20receptor%20polymorphism-mediated%20antibodydependant%20cellular%20cytotoxicity%20in%20colorectal%20cancer%20treated%20with%20cetuximab%20therapy&author=FV%20Negri&author=A%20Musolino&author=N%20Naldi&author=B%20Bortesi&author=G%20Missale&volume=14&publication_year=2014&pages=14-19&pmid=23296156&doi=10.1038/tpj.2012.54&)

35. Calemma R, Ottaiano A, Trotta AM, Nasti G, Romano C, Napolitano M et al. Fc γ receptor IIIa polymorphisms in advanced colorectal cancer patients correlated with response to anti-EGFR antibodies and clinical outcome. J Transl Med 2012; 10: 232.  [DOI](https://doi.org/10.1186/1479-5876-10-232) | [PMC free article](/articles/PMC3551834/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23171437/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Transl%20Med&title=Fc%20%CE%B3%20receptor%20IIIa%20polymorphisms%20in%20advanced%20colorectal%20cancer%20patients%20correlated%20with%20response%20to%20anti-EGFR%20antibodies%20and%20clinical%20outcome&author=R%20Calemma&author=A%20Ottaiano&author=AM%20Trotta&author=G%20Nasti&author=C%20Romano&volume=10&publication_year=2012&pages=232&pmid=23171437&doi=10.1186/1479-5876-10-232&)

36. Hu-Lieskovan S, Vallbohmer D, Zhang W, Yang D, Pohl A, Labonte MJ et al. EGF61 polymorphism predicts complete pathologic response to cetuximab-based chemoradiation independent of KRAS status in locally advanced rectal cancer patients. Clin Cancer Res 2011; 17: 5161–5169.  [DOI](https://doi.org/10.1158/1078-0432.CCR-10-2666) | [PMC free article](/articles/PMC3149732/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21673069/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&title=EGF61%20polymorphism%20predicts%20complete%20pathologic%20response%20to%20cetuximab-based%20chemoradiation%20independent%20of%20KRAS%20status%20in%20locally%20advanced%20rectal%20cancer%20patients&author=S%20Hu-Lieskovan&author=D%20Vallbohmer&author=W%20Zhang&author=D%20Yang&author=A%20Pohl&volume=17&publication_year=2011&pages=5161-5169&pmid=21673069&doi=10.1158/1078-0432.CCR-10-2666&)

37. Bibeau F, Lopez-Crapez E, Di Fiore F, Thezenas S, Ychou M, Blanchard F et al. Impact of Fc{γ}RIIa-Fc{γ}RIIIa polymorphisms and KRAS mutations on the clinical outcome of patients with metastatic colorectal cancer treated with cetuximab plus irinotecan. J Clin Oncol 2009; 27: 1122–1129.  [DOI](https://doi.org/10.1200/JCO.2008.18.0463) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19164213/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Oncol&title=Impact%20of%20Fc%7B%CE%B3%7DRIIa-Fc%7B%CE%B3%7DRIIIa%20polymorphisms%20and%20KRAS%20mutations%20on%20the%20clinical%20outcome%20of%20patients%20with%20metastatic%20colorectal%20cancer%20treated%20with%20cetuximab%20plus%20irinotecan&author=F%20Bibeau&author=E%20Lopez-Crapez&author=F%20Di%20Fiore&author=S%20Thezenas&author=M%20Ychou&volume=27&publication_year=2009&pages=1122-1129&pmid=19164213&doi=10.1200/JCO.2008.18.0463&)

38. Dahan L, Norguet E, Etienne-Grimaldi M-C, Formento J-L, Gasmi M, Nanni I et al. Pharmacogenetic profiling and cetuximab outcome in patients with advanced colorectal cancer. BMC Cancer 2011; 11: 496.  [DOI](https://doi.org/10.1186/1471-2407-11-496) | [PMC free article](/articles/PMC3235081/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22117530/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=BMC%20Cancer&title=Pharmacogenetic%20profiling%20and%20cetuximab%20outcome%20in%20patients%20with%20advanced%20colorectal%20cancer&author=L%20Dahan&author=E%20Norguet&author=M-C%20Etienne-Grimaldi&author=J-L%20Formento&author=M%20Gasmi&volume=11&publication_year=2011&pages=496&pmid=22117530&doi=10.1186/1471-2407-11-496&)

39. Park SJ, Hong YS, Lee J-L, Ryu M-H, Chang HM, Kim K et al. Genetic polymorphisms of FcγRIIa and FcγRIIIa are not predictive of clinical outcomes after cetuximab plus irinotecan chemotherapy in patients with metastatic colorectal cancer. Oncology 2012; 82: 83–89.  [DOI](https://doi.org/10.1159/000335959) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22327884/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Oncology&title=Genetic%20polymorphisms%20of%20Fc%CE%B3RIIa%20and%20Fc%CE%B3RIIIa%20are%20not%20predictive%20of%20clinical%20outcomes%20after%20cetuximab%20plus%20irinotecan%20chemotherapy%20in%20patients%20with%20metastatic%20colorectal%20cancer&author=SJ%20Park&author=YS%20Hong&author=J-L%20Lee&author=M-H%20Ryu&author=HM%20Chang&volume=82&publication_year=2012&pages=83-89&pmid=22327884&doi=10.1159/000335959&)

40. Paez D, Paré L, Espinosa I, Salazar J, del Rio E, Barnadas A et al. Immunoglobulin G fragment C receptor polymorphisms and KRAS mutations: are they useful biomarkers of clinical outcome in advanced colorectal cancer treated with anti-EGFRbased therapy? Cancer Sci 2010; 101: 2048–2053.  [DOI](https://doi.org/10.1111/j.1349-7006.2010.01621.x) | [PMC free article](/articles/PMC11158139/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20550522/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&title=Immunoglobulin%20G%20fragment%20C%20receptor%20polymorphisms%20and%20KRAS%20mutations:%20are%20they%20useful%20biomarkers%20of%20clinical%20outcome%20in%20advanced%20colorectal%20cancer%20treated%20with%20anti-EGFRbased%20therapy?&author=D%20Paez&author=L%20Par%C3%A9&author=I%20Espinosa&author=J%20Salazar&author=E%20del%20Rio&volume=101&publication_year=2010&pages=2048-2053&pmid=20550522&doi=10.1111/j.1349-7006.2010.01621.x&)

41. Sebio A, Paré L, Páez D, Salazar J, González A, Sala N et al. The LCS6 polymorphism in the binding site of let-7 microRNA to the KRAS 3’-untranslated region: its role in the efficacy of anti-EGFR-based therapy in metastatic colorectal cancer patients. Pharmacogenet Genomics 2013; 23: 142–147.  [DOI](https://doi.org/10.1097/FPC.0b013e32835d9b0b) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23324806/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet%20Genomics&title=The%20LCS6%20polymorphism%20in%20the%20binding%20site%20of%20let-7%20microRNA%20to%20the%20KRAS%203%E2%80%99-untranslated%20region:%20its%20role%20in%20the%20efficacy%20of%20anti-EGFR-based%20therapy%20in%20metastatic%20colorectal%20cancer%20patients&author=A%20Sebio&author=L%20Par%C3%A9&author=D%20P%C3%A1ez&author=J%20Salazar&author=A%20Gonz%C3%A1lez&volume=23&publication_year=2013&pages=142-147&pmid=23324806&doi=10.1097/FPC.0b013e32835d9b0b&)
