# Genetic determinants of apixaban plasma levels and their relationship to bleeding and thromboembolic events

## Metadata
**Authors:** Sofia Attelind, Pär Hallberg, Mia Wadelius, Anna-Karin Hamberg, Agneta Siegbahn, Christopher B Granger, Renato D Lopes, John H Alexander, Lars Wallentin, Niclas Eriksson
**Journal:** Frontiers in Genetics
**Date:** 2022 Sep 14
**DOI:** [10.3389/fgene.2022.982955](https://doi.org/10.3389/fgene.2022.982955)
**PMID:** 36186466
**PMCID:** PMC9515473
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9515473/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9515473/pdf/fgene-13-982955.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9515473/pdf/fgene-13-982955.pdf)

## Abstract

Apixaban is a direct oral anticoagulant, a factor Xa inhibitor, used for the prevention of ischemic stroke in patients with atrial fibrillation. Despite using recommended dosing a few patients might still experience bleeding or lack of efficacy that might be related to inappropriate drug exposure. We conducted a genome-wide association study using data from 1,325 participants in the pivotal phase three trial of apixaban with the aim to identify genetic factors affecting the pharmacokinetics of apixaban. A candidate gene analysis was also performed for pre-specified variants in ABCB1, ABCG2, CYP3A4, CYP3A5, and SULT1A1, with a subsequent analysis of all available polymorphisms within the candidate genes. Significant findings were further evaluated to assess a potential association with clinical outcome such as bleeding or thromboembolic events. No variant was consistently associated with an altered apixaban exposure on a genome-wide level. The candidate gene analyses showed a statistically significant association with a well-known variant in the drug transporter gene ABCG2 (c.421G > T, rs2231142). Patients carrying this variant had a higher exposure to apixaban [area under the curve (AUC), beta = 151 (95% CI 59–243), p = 0.001]. On average, heterozygotes displayed a 5% increase of AUC and homozygotes a 17% increase of AUC, compared with homozygotes for the wild-type allele. Bleeding or thromboembolic events were not significantly associated with ABCG2 rs2231142. This large genome-wide study demonstrates that genetic variation in the drug transporter gene ABCG2 is associated with the pharmacokinetics of apixaban. However, the influence of this finding on drug exposure was small, and further studies are needed to better understand whether it is of relevance for ischemic and bleeding events.

Keywords: factor Xa inhibitors, apixaban, atrial fibrillation, genome-wide association study, pharmacokinetics, pharmacogenetics, drug-related side effects and adverse reactions

## 1 Introduction

Since their introduction into clinical routine, direct oral anticoagulants (DOACs) that inhibit thrombin (dabigatran) or factor Xa (rivaroxaban, apixaban, and edoxaban) have largely replaced warfarin as the first-hand choice to protect against ischemic stroke in patients with atrial fibrillation. Although safer than warfarin, the risk of major bleeding during treatment with a DOAC is estimated to 2%–3% per year ([Connolly et al., 2009](#B3); [Granger et al., 2011](#B8); [Patel et al., 2011](#B20); [Giugliano et al., 2013](#B7)); the risk of intracranial haemorrhage is estimated to about 0.1%–0.5% per year, and the risk of major gastrointestinal bleeding to 1%–1.5%. Lack of efficacy to anticoagulant therapy also has potentially severe consequences. In clinical trials of atrial fibrillation, DOAC treatment was associated with a yearly incidence of ischemic stroke of about 1%–1.5% and of systemic embolism of about 0.05%–0.15% ([Connolly et al., 2009](#B3); [Granger et al., 2011](#B8); [Patel et al., 2011](#B20); [Giugliano et al., 2013](#B7)).

Among factors known to affect the risk of complications to DOAC treatment are those that influence exposure to the drug, such as dose, renal function, and concomitant treatment with interacting drugs. Genetic variation influencing drug levels may also affect this risk. Using data from the RE-LY trial, a genome-wide association study (GWAS) showed that genetic variation in *CES1*, which encodes the liver enzyme carboxylesterase 1 that activates the prodrug dabigatran etexilate, is associated with lower concentrations of dabigatran, and lower risk of bleeding ([Pare et al., 2013](#B19)).

Apixaban is eliminated in several ways of which renal excretion amounts to around 26% ([Raghavan et al., 2009](#B22)). Metabolic pathways include O-demethylation and hydroxylation mainly through CYP3A4 ([Asic et al., 2018](#B1)). SULT1A1 plays a major role for the formation of the first inactive metabolite O-demethyl apixaban sulphate ([Nagar et al., 2006](#B17); [Raghavan et al., 2009](#B22); [Wang et al., 2009](#B30); [Asic et al., 2018](#B1); [Kanuri and Kreutz, 2019](#B12); [Pharmacogenomics and Pharmacogenetics, 2022](#B21)). Apixaban is also a substrate of the efflux transporters BCRP encoded by *ABCG2* and P-gp encoded by *ABCB1* ([Eliquis, 2022](#B6)). According to two small candidate gene studies, single nucleotide polymorphisms (SNPs) in *CYP3A5* and *ABCG2* were associated with higher concentration of apixaban ([Ueshima et al., 2017](#B28); [Ueshima et al., 2018](#B29)). In a candidate gene study on 358 patients, *ABCG2* genotype contributed to interpatient apixaban variability beyond known clinical factors, while genotyped variants of *ABCB1*, *CYP3A4*, and *CYP3A5* had no effect ([Gulilat et al., 2020](#B9)). However, an intronic *ABCB1* variant has been associated with lower apixaban peak concentrations ([Dimatteo et al., 2016](#B4)) and with a lower risk of bleeding events in apixaban users ([Lahteenmaki et al., 2021](#B13)) in small studies. In another candidate gene study on 164 patients, known polymorphisms of *CYP3A4* and *ABCB1* did not affect the exposure to apixaban measured as the area under the curve (AUC) ([Lenoir et al., 2022](#B14)).

To our knowledge, no GWAS on the PK of apixaban has been published ([GWAS Catalog, 2022](#B10)).

## 2 Objective

In this study, we used genome-wide data from a phase three trial of apixaban with the aim to identify genetic factors potentially affecting the pharmacokinetics of apixaban. Significant findings were further evaluated to assess the association with clinical outcome such as bleeding or thromboembolic events.

## 3 Materials and methods

We analysed data from patients who participated in a large phase three clinical trial for apixaban (ARISTOTLE). The ARISTOTLE trial was completed in 2011 and included in the marketing authorisation application for ELIQUIS (apixaban) ([Granger et al., 2011](#B8)). Patients in this trial (*n* = 18,201) were adult individuals with atrial fibrillation randomised to treatment with apixaban or warfarin and followed for a median of 1.8 years. Data from whole-genome scans and clinical outcome (major bleeding, clinically relevant non-major bleeding, and stroke or systemic embolism) were available for 2,800 apixaban-treated patients, of which 1,325 also had PK data ([Table 1](#T1) and [Supplementary Table S1](#s13)).

### TABLE 1.

|   | N | a/b/c | Mean ± SD |
| --- | --- | --- | --- |
| Age | 1,325 | 63/70/76 | 69 ± 9 |
| Patient sex: Female | 1,325 | 33% (438) |   |
| Height (cm) | 1,322 | 163.5/171.0/178.0 | 170.8 ± 10.4 |
| Weight (kg) | 1,323 | 74/86/99 | 88 ± 20 |
| BMI (kg/m2) | 1,322 | 26.1/29.3/33.1 | 30.0 ± 5.8 |
| Smoker | 1,325 | 9% (120) |   |
| Race | 1,325 |   |   |
| White |   | 94% (1243) |   |
| Black/African American |   | 1% (8) |   |
| Asian |   | 5% (71) |   |
| Other |   | 0% (3) |   |
| Region | 1,325 |   |   |
| Asia/Pacific |   | 7% (96) |   |
| Europe |   | 53% (705) |   |
| Latin America |   | 5% (65) |   |
| North America |   | 35% (459) |   |
| Serum creatinine at baseline (mg/dl) | 1,325 | 0.88/1.02/1.20 | 1.07 ± 0.28 |
| Calculated CrCl at baseline (ml/min) | 1,323 | 59.2/75.9/99.2 | 82.2 ± 32.9 |
| GFR CKD-EPI (ml/min/1.73 m2) | 1,325 | 55.8/68.5/81.6 | 68.2 ± 17.3 |
| CHADS-2 score | 1,325 |   |   |
| 0 |   | 1% (7) |   |
| 1 |   | 34% (447) |   |
| 2 |   | 36% (476) |   |
| 3 |   | 19% (250) |   |
| 4 |   | 8% (108) |   |
| 5 |   | 2% (33) |   |
| 6 |   | 0% (4) |   |
| Prior stroke/TIA/SEE | 1,325 | 18% (244) |   |
| Prior stroke | 1,325 | 10% (134) |   |
| Diabetes | 1,325 | 26% (340) |   |
| Hypertension (treated) | 1,325 | 88% (1164) |   |
| Myocardial infarction | 1,325 | 14% (186) |   |
| Heart failure | 1,325 | 28% (367) |   |
| CHF within 3 months or LVEF ≤40% | 1,325 | 32% (428) |   |
| History of clinically relevant or spontaneous bleeding | 1,325 | 19% (252) |   |
| ACEi/ARB | 1,325 | 72% (960) |   |
| Amiodarone | 1,325 | 8% (108) |   |
| Beta blocker | 1,325 | 66% (878) |   |
| Aspirin | 1,325 | 28% (366) |   |
| Clopidogrel | 1,325 | 1% (13) |   |
| Digoxin | 1,325 | 34% (444) |   |
| Calcium channel blocker | 1,325 | 32% (423) |   |
| Statin | 1,325 | 45% (596) |   |
| Nonsteroidal anti-inflammatory agent | 1,325 | 11% (143) |   |
| Dose: 5 mg (twice daily) | 1,325 | 96% (1266) |   |
| PK-parameters |   |   |   |
| AUCss | 1,325 | 2832.9/3501.5/4220.3 (494.6, 8600.9) |   |
| Cmax,ss | 1,325 | 128.9/163.0/204.7 (31.3, 477.9) |   |
| Cmin,ss | 1,325 | 74.7/102.5/137.5 (4.00, 376.8) |   |
| Clinical outcomes a |   |   |   |
| Major bleeding | 2,799 | 4% (105) |   |
| Major or CRNM bleeding | 2,799 | 8% (214) |   |
| Haemorrhagic stroke | 2,800 | 0.4% (11) |   |
| Ischemic stroke | 2,800 | 1% (32) |   |

Table 1 Caption: Characteristics for apixaban-treated patients with data from both whole-genome scans and pharmacokinetic evaluations.

The phase three clinical trial (ARISTOTLE study ID: [NCT00412984](https://clinicaltrials.gov/ct2/show/NCT00412984)) was approved by the appropriate ethics committees at all study sites. All patients provided written informed consent before enrolment.

Genetic associations with the PK of apixaban were investigated by utilising data on the PK parameters area under the plasma concentration time curve at steady state (AUC_ss_), the maximum concentration at steady state (C_max,ss_) as well as the trough concentration at steady state (C_min,ss_) ([Cirincione et al., 2018](#B2)). Any significant findings were tested for association with bleeding or thromboembolic events by analysing the primary and secondary outcomes as specified in the ARISTOTLE trial ([Granger et al., 2011](#B8)).

We performed an association study covering the entire genome and pre-specified variants in candidate genes that encode transport proteins and metabolic enzymes involved in the PK of apixaban. These candidate variants have previously been suggested to influence blood concentrations of apixaban or outcome of the anticoagulant treatment ([Table 2](#T2)). We further did an analysis of all available polymorphisms within these candidate genes and included a region covering ±10 kb upstream and downstream of each gene ([Supplementary Table 2](#s13)).

### TABLE 2.

| Gene and variant | SNP | Chromosome | Position, base pair | Minor allele | Major allele | MAF | Imputation quality | Genotyped 1 or imputed 0 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| ABCB1 c.3435 (Ueshima et al., 2018; Lahteenmaki et al., 2021) | rs1045642 | 7 | 87138645 | G | A | 0.4672 | 1.00 | 1 |
| ABCB1 c.2677 (Ueshima et al., 2018; Lahteenmaki et al., 2021) | rs2032582A/C | 7 | 87160618 | A | C | 0.4535 | 0.98 | 0 |
| ABCB1 c.2677 (Ueshima et al., 2018; Lahteenmaki et al., 2021) | rs2032582T/A | 7 | 87160618 | T | A | 0.0385 | 0.92 | 0 |
| ABCB1 c.1236 (Ueshima et al., 2018; Lahteenmaki et al., 2021) | rs1128503 | 7 | 87179601 | A | G | 0.4604 | 1.00 | 1 |
| ABCG2 c.421 (Ueshima et al., 2017; Ueshima et al., 2018) | rs2231142 | 4 | 89052323 | T | G | 0.1177 | 1.00 | 1 |
| ABCG2 c.34 (Gulilat et al., 2020) | rs2231137 | 4 | 89061114 | T | C | 0.0679 | 0.98 | 0 |
| CYP3A5 *3 (Ueshima et al., 2017; Ueshima et al., 2018) | rs776746 | 7 | 99270539 | T | C | 0.0936 | 1.00 | 1 |
| CYP3A4 *22 (Lenoir et al., 2022) | rs35599367 | 7 | 99366316 | A | G | 0.0366 | 1.00 | 1 |
| CYP3A4 *1B (Lenoir et al., 2022) | rs2740574 | 7 | 99382096 | C | T | 0.0468 | 0.98 | 0 |
| SULT1A1 *2 (Kanuri and Kreutz, 2019) | rs1042028 | 16 | 28617514 | T | C | 0.3192 | 0.93 | 0 |

Table 2 Caption: List of candidate single nucleotide polymorphisms.

### 3.1 Genome-wide array data and analyses

DNA was extracted from whole blood using NucleoSpin^®^ 96 Blood kit (from MACHEREY-NAGEL). Genotyping was performed on the Illumina Global Screening Array 24v2 at the SNP&SEQ Platform of the Science for Life Laboratory at Uppsala University. Genotype calls were generated using Illumina GenomeStudio 2.0.3.

Quality control (QC) of the raw genotyped data included exclusion of patient sex mismatches, related patients with PLINK pi_hat >0.2, variants with more than 2% missing information, variants with Hardy-Weinberg Equilibrium *p*-value < 5 × 10^−8^, and variants with Minor Allele Frequency (MAF) < 0.1%. The pre-imputation number of variants was 541,772 and the number of patients was 5,553 individuals of which 2,800 were treated with apixaban and included in this study. Data on clinical outcome was available for all 2,800 individuals and PK data was available for 1,325 individuals.

Pre-imputation QC was performed versus the Haplotype Reference Consortium (HRC) panel using McCarty Group Tools QC Script (v 4.2.9) ([Scattershot, 2021](#B24)). Imputation was performed with Positional Burrows-Wheeler transform (PBWT) ([Durbin, 2014](#B5)) using the HRC r1 reference panel ([McCarthy et al., 2016](#B16)) with the Sanger imputation server. The program used for phasing was Eagle v2.3.3 ([Loh et al., 2016](#B15)).

Post-imputation, variants with MAF < 0.1% were excluded and the data was filtered on imputation quality above 0.7 and converted to hard calls using PLINK v1.9. The total number of SNPs post-imputation and QC was 16.7 million. In our GWAS analysis, we reported variants with MAF >1% (7.9 million).

To account for possible population stratification, principal component analysis (PCA) was performed using PLINK v1.9. The first six genetic principal components were included in all analyses. Comparison between the ARISTOTLE genetic population and HapMap was performed by performing PCA on the merged ARISTOTLE and HapMap release 23 material ([Supplementary Figure 1](#s13)). For the GWAS analyses, separate sensitivity analyses were performed in the largest cluster based on K-means clustering of genetic PCA (Statistical methods, [Supplementary Figure 2](#s13)).

### 3.2 Statistical methods

The method presented by [Takeuchi et al. (2009)](#B27) was used to estimate the power of the GWAS analysis for a marker explaining specific amounts of variance (R^2^). For the clinical analyses, we estimated the power using the GPC function in the R-package *Genetics Design* for the outcomes major bleeding and major/clinically relevant non-major bleeding ([Supplementary Figures 3–5](#s13)).

All analyses including genetic variants assumed an additive genetic model, i.e., SNPs were coded 0, 1, and 2 according to the number of minor alleles. GWAS, candidate SNP, and gene analyses were performed using linear regression as implemented in PLINK v1.9. The analyses were adjusted for age, patient sex, calculated creatinine clearance (CrCl) at baseline (ml/min), weight (kg), dose of apixaban, concomitant treatment with amiodarone or calcium channel blockers and the first six genetic principal components. These variables were selected as they are known to affect the PK of apixaban ([Cirincione et al., 2018](#B2)). Chromosome X was analysed assuming no chromosome X inactivation as implemented in PLINK. Possible inflation in *p*-values was measured by genomic inflation factor lambda and by visually inspecting QQ plots.

In the PK analyses, two missing values on weight were imputed and in the clinical outcome analyses, five missing values of weight and four of calculated CrCl at baseline were imputed. The imputation was performed using the R-package mice with the default settings.

The sensitivity population was defined by performing K-means clustering on the first two genetic principal components. The number of clusters was determined by the elbow method, i.e., the number of clusters was set at the point where there was little change in within-cluster variance per increase of one cluster. Our optimal number of clusters was 4 and the largest cluster was used as the sensitivity population.

The coefficient of determination (R^2^) was used to describe the amount of variation explained by a regression model (including multiple and univariate models). The partial R^2^, here defined as the increase in R^2^ of a model when a variable is added last, was used to quantify the impact of a single variable on a multiple linear regression model.

The significance level for the candidate SNP and gene analyses was set to the Bonferroni adjusted limit of 0.05/number of tests. The level of GWAS significance was set to the conventional threshold of *p* < 5 × 10^−8^ ([Sham and Purcell, 2014](#B25)).

Time to event was analysed using Cox proportional hazard regression analyses adjusted by the same covariates as in the GWAS. The threshold for significance in the Cox regression analyses was set at the Bonferroni cut-off according to 0.05/number of tests. The 95% confidence intervals for the incidence rates were estimated using a gamma distribution.

Analyses other than candidate SNP, gene, and GWAS were performed using R version 3.6.0 (R Foundation for Statistical Computing, Vienna, Austria).

### 3.3 Clinical outcomes

#### 3.3.1 Pharmacokinetic data

Measures of apixaban exposure used in the analyses included AUC_ss_, C_max,ss_, and C_min,ss_ and were received from the ARISTOTLE study team ([Granger et al., 2011](#B8)). Individual exposures were derived from empirical Bayes estimates of individual PK parameters using sparse data (single sample at month two in the ARISTOTLE study) and a previously developed population PK model described by [Cirincione et al. (2018)](#B2).

#### 3.3.2 Bleeding and thromboembolic events

Bleeding and thromboembolic events were defined according to the criteria specified in the ARISTOTLE trial ([Granger et al., 2011](#B8)).

Major bleeding as defined by the International Society on Thrombosis and Haemostasis (ISTH) were acute or sub-acute clinically overt bleeding accompanied by a decrease in haemoglobin level of ≥2 g/dl, and/or a transfusion of ≥2 U of packed red blood cells, and/or bleeding that was fatal or occurred in the following critical sites: intracranial, intra-spinal, intra-ocular, pericardial, intra-articular, intra-muscular with compartment syndrome, or retroperitoneal ([Granger et al., 2011](#B8)).

Clinically relevant non-major bleeding (CRNM) was defined as acute or sub-acute clinically overt bleeding that did not satisfy the criteria for major bleeding and led to hospital admission for bleeding, physician-guided medical or surgical treatment for bleeding, or a change in antithrombotic therapy (including study drug) for bleeding. All acute clinically overt bleeding events not meeting criteria for major bleeding or clinically relevant non-major bleeding were classified as minor bleeding ([Granger et al., 2011](#B8)).

Stroke was defined as a non-traumatic focal neurologic deficit lasting ≥24 h, including retinal ischemic events (embolism or thrombosis). Strokes were classified as ischemic, ischemic with haemorrhagic transformation, haemorrhagic, or stroke of uncertain type. Systemic embolism required a clinical history consistent with an acute loss of blood flow to a peripheral artery supported by evidence of embolism from surgical specimens, autopsy, angiography, vascular imaging, or other objective testing ([Granger et al., 2011](#B8)).

## 4 Results

Baseline characteristics of the PK population are presented in [Table 1](#T1). The median age of 70 years and patient sex distribution with 33% females were similar to the main ARISTOTLE trial population. The self-reported ethnicity was predominantly white (94%), and patients were mainly included from Europe (53%), and North America (35%). A total of 1,266 patients received the standard dose of 5 mg twice daily, and 59 the reduced dose of 2.5 mg twice daily. The reduced dose is recommended in patients fulfilling at least two of the following criteria: age ≥ 80 years, body weight ≤ 60 kg, and serum creatinine ≥ 1.5 mg/dl (133 µmol/L) ([Eliquis, 2022](#B6)).

Linear regression results for the covariates used in the genetic models versus the apixaban exposure variable AUC_ss_ are presented in [Table 3](#T3). The overall model R^2^ was 0.45. The three variables with highest partial R^2^ were dose of apixaban followed by calculated CrCl at baseline (ml/min) and patient sex.

### TABLE 3.

| Variable | Comparison | Beta | CI | P | R2 | Variable R2 | Partial R2 |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Age | By increase of one SD: 9.14 | 128.012 | (69.627–186.396) | 1.86 × 10−5 | 0.458 | 0.106 | 0.008 |
| Patient sex | Female vs. Male | 631.157 | (535.03–727.284) | 8.78 × 10−36 | 0.458 | 0.113 | 0.068 |
| Calculated CrCl at baseline (ml/min) | By increase of one SD: 32.8 | −559.071 | (−636.089–482.054) | 7.82 × 10−43 | 0.458 | 0.235 | 0.083 |
| Weight (kg) | By increase of one SD: 20.36 | 75.281 | (9.557–141.006) | 2.49 × 10−2 | 0.458 | 0.112 | 0.002 |
| Dose | 5 vs. 2.5 | 2008.575 | (1786.599–2230.552) | 2.88 × 10−63 | 0.458 | 0.037 | 0.13 |
| Amiodarone | Yes vs. No | 433.036 | (274.07–592.002) | 1.10 × 10−7 | 0.458 | 0.009 | 0.012 |
| Calcium channel blocker | Yes vs. No | 276.504 | (184.124–368.885) | 5.63 × 10−9 | 0.458 | 0.012 | 0.014 |

Table 3 Caption: Area under the curve at steady-state (AUCss)—Linear regression results for covariates used in the analyses. N = 1,323 a .

### 4.1 Genome-wide association study analyses

With 1,325 observations we had the power to detect a variant explaining approximately 1.5% of the variance (R^2^) in the PK outcomes ([Supplementary Figure 3](#s13)).

Results of the GWAS analyses of AUC_ss_, C_max,ss_, and C_min,ss_ in 1,325 patients are shown in the Manhattan plots in [Figure 1](#F1) (AUC_ss_) and in [Supplementary Figures 6, 7](#s13) and [Supplementary Tables 3–5](#s13). No inflation in *p*-values was observed with lambda values of 1.006, 1.006, and 1.007 for the analyses of AUC_ss_, C_max,ss_, and C_min,ss_, respectively.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2ba3/9515473/b6005efc6742/fgene-13-982955-g001.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9515473_fgene-13-982955-g001.jpg)

Manhattan plot for the GWAS analyses of apixaban AUCss in 1,325 patients adjusted for the covariates listed in Table 3. The grey line denotes the genome-wide significance level p < 5 × 10−8.

The Manhattan plot for C_min,ss_ ([Supplementary Figure 7](#s13)) indicated a signal in an intergenic region on chromosome X. This rare SNP (rs147256925), which is not in linkage disequilibrium (LD, r^2^ ≥ 0.2) with any nearby SNP ([Ward and Kellis, 2012](#B31)) and has a MAF just above the threshold for QC, is most likely an artefact. Its closest neighbouring gene, gastrin releasing peptide receptor (*GRPR*) is located about 100k bases away. Gastrin-Releasing Peptide (GRP) regulates functions of the gastrointestinal and central nervous system, such as release of gastrointestinal hormones, smooth muscle contraction, and epithelial cell proliferation. The SNP rs147256925 has no evidence of a regulatory function ([National Center for Biotechnology Information NIH National, 2022](#B18)). Except for this SNP, no single variant reached genome-wide significance.

There were, however, several nominally significant associations with *p*-values <1 × 10^−6^. For the outcome AUC_ss_ these included rs111844911 on chromosome 7, rs183109587 and rs184794076 on chromosome X, and rs59884489 on chromosome 16. For the outcome C_min,ss_ they included rs78896694 on chromosome 7, rs143178045 on chromosome 8, and rs12107681 on chromosome 3, and for the outcome C_max,ss_, rs56293342 on chromosome 2. A description of these SNPs and their closest genes can be found in the [Supplementary Material](#s14). These SNPs and genes were not connected to an obvious or plausible biological mechanism impacting the PK of apixaban and have, to our knowledge, not previously been associated with altered drug PK. It is likely that they are spurious results.

Sensitivity GWAS analyses in 1,174 patients of mainly white ethnicity according to the PCA gave the same results as the main analyses ([Supplementary Figures 8–10](#s13)).

### 4.2 Candidate gene analyses of pre-specified variants

The candidate gene analyses of the pre-specified variants in *ABCB1*, *ABCG2*, *CYP3A4*, *CYP3A5*, and *SULT1A1* ([Table 2](#T2) and [Supplementary Table 2](#s13)) in relation to AUC_ss_, C_max,ss_, and C_min,ss_ are shown in [Table 4](#T4). The number of candidate SNPs was 11, hence the Bonferroni threshold for significance was *p* < 0.0045. The SNP rs2231142 in *ABCG2* on chromosome 4 was statistically significantly associated with all three PK outcomes. Descriptive statistics of the PK parameters by the SNP rs2231142 are shown in [Figure 2](#F2). Linear regression results for the SNP rs2231142 are shown in [Figure 3](#F3) and [Supplementary Table 6](#s13). The effect was small and explained about 1% of the total variance. On average, heterozygotes displayed a 5% increase of AUC_ss_ and homozygotes 17%, compared with homozygotes for the wild-type allele. We did not find any association between other candidate SNPs and the PK outcomes.

### TABLE 4.

| Gene | SNP | Alleles minor/major | Variable | Beta | Confidence interval lower 95 | Confidence interval upper 95 | p |
| --- | --- | --- | --- | --- | --- | --- | --- |
| ABCB1 | rs1045642 | G/A | AUCss | 31.66 | −28.71 | 92.03 | 0.3042 |
| Cmax,ss | 2.33 | −1.417 | 6.077 | 0.2231 |  |  |  |
| Cmin,ss | 1.714 | −1.665 | 5.092 | 0.3203 |  |  |  |
| ABCB1 | rs2032582A/C | A/C | AUCss | −7.011 | −65.64 | 51.61 | 0.8147 |
| Cmax,ss | −0.8717 | −4.511 | 2.767 | 0.6388 |  |  |  |
| Cmin,ss | −0.9038 | −4.184 | 2.377 | 0.5893 |  |  |  |
| ABCB1 | rs2032582T/A | T/A | AUCss | −61.480 | −217.90 | 94.97 | 0.4413 |
| Cmax,ss | −3.2330 | −12.950 | 6.481 | 0.5143 |  |  |  |
| Cmin,ss | −3.7170 | −12.470 | 5.038 | 0.4055 |  |  |  |
| ABCB1 | rs4148738 | C/T | AUCss | −1.429 | −60.31 | 57.45 | 0.9621 |
| Cmax,ss | −0.8235 | −4.478 | 2.831 | 0.6588 |  |  |  |
| Cmin,ss | −0.7336 | −4.028 | 2.561 | 0.6626 |  |  |  |
| ABCB1 | rs1128503 | A/G | AUCss | −1.962 | −61.42 | 57.50 | 0.9484 |
| Cmax,ss | −0.8854 | −4.576 | 2.806 | 0.6383 |  |  |  |
| Cmin,ss | −1.031 | −4.358 | 2.296 | 0.5436 |  |  |  |
| ABCG2 | rs2231142 | T/G | AUCss | 151 | 59.15 | 242.8 | 0.0013* |
| Cmax,ss | 9.633 | 3.934 | 15.33 | 0.0009* |  |  |  |
| Cmin,ss | 9.695 | 4.562 | 14.83 | 0.0002* |  |  |  |
| ABCG2 | rs2231137 | T/C | AUCss | −68.3 | −187.3 | 50.73 | 0.2610 |
| Cmax,ss | −4.973 | −12.36 | 2.414 | 0.1873 |  |  |  |
| Cmin,ss | −4.652 | −11.31 | 2.007 | 0.1712 |  |  |  |
| CYP3A5 | rs776746 | T/C | AUCss | −68.72 | −175.1 | 37.64 | 0.2056 |
| Cmax,ss | −5.425 | −12.03 | 1.175 | 0.1074 |  |  |  |
| Cmin,ss | −4.493 | −10.44 | 1.457 | 0.1391 |  |  |  |
| CYP3A4 | rs35599367 | A/G | AUCss | 42.74 | −113.6 | 199.1 | 0.5921 |
| Cmax,ss | 0.2975 | −9.408 | 10.00 | 0.9521 |  |  |  |
| Cmin,ss | −0.6454 | −9.394 | 8.103 | 0.8851 |  |  |  |
| CYP3A4 | rs2740574 | C/T | AUCss | −61.18 | −205.3 | 82.94 | 0.4055 |
| Cmax,ss | −5.929 | −14.87 | 3.014 | 0.1940 |  |  |  |
| Cmin,ss | −4.216 | −12.28 | 3.848 | 0.3057 |  |  |  |
| SULT1A1 | rs1042028 | T/C | AUCss | 17.830 | −46.52 | 82.17 | 0.5872 |
| Cmax,ss | 0.7968 | −3.198 | 4.792 | 0.6959 |  |  |  |
| Cmin,ss | 1.375 | −2.225 | 4.976 | 0.4542 |  |  |  |

Table 4 Caption: Association between candidate single nucleotide polymorphisms and AUCss, Cmax,ss, and Cmin,ss. N = 1,325.

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2ba3/9515473/da6564cf6421/fgene-13-982955-g002.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9515473_fgene-13-982955-g002.jpg)

Box-plot of the PK parameters by the SNP rs2231142. AUCss, area under the curve at steady state; Cmax,ss, maximum serum concentration at steady state; Cmin,ss, serum trough concentration at steady state; n, number of genotype carriers; the box represents the lower and upper quartile whereas the thick middle line denotes the median. The whiskers extend to the maximum and minimum value within 1.5 times the inter quartile range and values outside of these limits are presented as circles.

### FIGURE 3.

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2ba3/9515473/b39f78edcf54/fgene-13-982955-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9515473_fgene-13-982955-g003.jpg)

Forest plot comparing the effect on area under the curve (AUC) of rs2231142 with that of age, gender, creatinine clearance (CrCL), weight, dose, co-medication with amiodarone or calcium blockers. The beta coefficient is the degree of change in the outcome variable for every unit of change in the predictor variable. Note that these results are not adjusted by genetic principal components 1 to 6. The Bonferroni threshold for significance was p < 0.0045.

### 4.3 Candidate gene analyses of all available variants

The total number of SNPs available for analysis in a region covering ±10 kb upstream and downstream of each gene *ABCB1*, *ABCG2*, *CYP3A4*, *CYP3A5*, and *SULT1A1* after QC was 866, and the threshold for significance was therefore set to 5.77 × 10^−5^. No variant was statistically significantly associated with AUC_ss_, C_max,ss_, or C_min,ss_ ([Supplementary Table 7](#s13) shows the top ten SNPs per PK parameter and [Supplementary Tables 8–10](#s13) the top 60 variants).

### 4.4 Bleeding and thromboembolic events

With 2,799 observations, we had the power to detect a variant with a hazard ratio slightly below 2 for the outcome major bleeding and around 1.5 for the outcome major/clinically relevant non-major bleeding ([Supplementary Figures 4, 5](#s13)).

Clinical outcomes are shown in [Table 1](#T1) and [Supplementary Table 1](#s13). One candidate variant (rs2231142) passed the Bonferroni threshold for significance and was moved forward to analyses versus clinical events.

Cox proportional hazard regression analyses results for the candidate SNP rs2231142 versus bleeding and thromboembolic events are shown in [Figure 4](#F4), and incidence rates per genotype are presented in [Supplementary Table 11](#s13). No statistically significant associations were found.

### FIGURE 4.

![FIGURE 4](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2ba3/9515473/64a37e1ffa58/fgene-13-982955-g004.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9515473_fgene-13-982955-g004.jpg)

Forest plot for the Cox proportional hazard ratio for carrying rs2231142 versus bleeding and thromboembolic events. Ninety five percent confidence intervals (CI) for incidence rates were estimated using a gamma distribution. ISTH, International Society on Thrombosis and Haemostasis; CRNM, clinically relevant non-major bleeding.

## 5 Discussion

In this genome-wide study, no genetic variant was consistently associated with the PK of apixaban on a genome-wide level. In a pre-specified candidate SNP analysis, rs2231142 in the drug transporter gene *ABCG2* ([Heyes et al., 2018](#B11)) was significantly associated with the PK parameters AUC_ss_, C_max,ss_, and C_min,ss_. This variant led to a higher exposure to apixaban and could explain about 1% of the variance [AUC_ss_, beta = 151 (95% CI 59–243), *p* = 0.001]. Compared with homozygotes for the wild-type allele, the AUC_ss_ increased on average 5% in heterozygotes and 17% in homozygotes. This replicates the finding in previous studies by [Ueshima et al. (2017](#B28), [2018)](#B11), although the magnitude of the effect was lower in our study. For comparison, patient sex explains notably more of the variance in AUCss [beta = 627 (95% CI 532–723), *p* = 6.32 × 10^−36^]. We did not find support for a clinical impact of this SNP in terms of association with risk of bleeding or thromboembolic events.

Previous small studies ([Dimatteo et al., 2016](#B4); [Lahteenmaki et al., 2021](#B13)) have shown that the minor allele of rs4148738, an expression quantitative trait loci (eQTL) for the efflux transporter gene *ABCB1* ([Shou et al., 2012](#B26)), was associated with lower apixaban peak concentrations ([Dimatteo et al., 2016](#B4)) and a lower risk of bleeding events in apixaban users ([Lahteenmaki et al., 2021](#B13)). However, these results could not be confirmed in our larger study. Neither did we replicate previous findings of an association between *CYP3A5* and the PK of apixaban ([Ueshima et al., 2017](#B28); [Ueshima et al., 2018](#B29)). These discrepancies could probably be explained by our more robust estimates of apixaban exposure, larger sample size, and an adequately powered study.

A limitation of this study is that most participants were of white European ancestry, and thus the results cannot be generalised to all ancestries. Another limitation is the sparseness of the PK data, which may cause shrinkage in the individual parameter estimates (empirical Bayes estimates, EBEs) and thereby an underestimation of the true variability in apixaban exposure. However, we estimated the shrinkage to be less than 15% in this patient cohort, suggesting that this has not impacted the results ([Savic and Karlsson, 2009](#B23)). A further limitation is the small number of clinical events, especially for the stroke categories, which leads to low power to detect a genetic effect.

## 6 Conclusion

This study demonstrates that genetic variation in the drug transporter gene *ABCG2* affects the pharmacokinetics of apixaban. Patients carrying the *ABCG2* variant rs2231142 were exposed to higher concentrations of apixaban [AUC, beta = 151 (95% CI 59–243), *p* = 0.001]. The effect of rs2231142 did not appear to translate into an increased risk of bleeding or thromboembolic events, although the number of bleeding events were low, and the results should be interpreted with caution. Further studies are needed to better understand the occurrence of ischemic and bleeding events during the use of apixaban.

## Data availability statement

The data analyzed in this study is subject to the following licenses/restrictions: The datasets generated during and/or analysed during the current study are not publicly available due to the European General Data Protection Regulation (GDPR), which requires us to protect the identity of participants, but datasets are partly available from the corresponding author on reasonable request. Requests to access these datasets should be directed to Mia.Wadelius@medsci.uu.se; Niclas.Eriksson@ucr.uu.se.

## Ethics statement

The phase three clinical trial, ARISTOTLE study ID: [NCT00412984](https://clinicaltrials.gov/ct2/show/NCT00412984), was approved by the appropriate ethics committees at all study sites. All patients provided their written informed consent to participate in this study before enrolment.

## Author contributions

All authors contributed to the conceptualisation of this work. SA, A-KH, MW, PH, and NE designed the study. NE performed the data analysis and SA, A-KH, MW, PH, and NE interpreted the results. SA drafted the first draft of the manuscript. All authors read, revised, edited, and approved the final version of the manuscript.

## Funding

The phase three clinical trial for apixaban (ARISTOTLE study ID: [NCT00412984](https://clinicaltrials.gov/ct2/show/NCT00412984)) was funded by Bristol-Myers Squibb and Pfizer. The current study received grant support from the Swedish Foundation for Strategic Research (grant RB13‐0197), The Swedish Brain Foundation (FO 2020-0234), Swedish Heart-Lung Foundation (20200777), Swedish Stroke Association, Selander’s Foundation, and Clinical Research Support at Uppsala University Hospital (ALF).

## Conflict of interest

AS reports institutional research grants from AstraZeneca, Boehringer Ingelheim, Bristol-Myers Squibb/Pfizer, GlaxoSmithKline, and Roche Diagnostics. CG reports grants and consulting fees from Boehringer Ingelheim, Bristol-Myers Squibb, Pfizer, Janssen Pharmaceutica Products, L.P., Novartis Pharmaceutical Company; grants from AKROS, AstraZeneca, Daiichi Sankyo, Duke Clinical Research Institute, GlaxoSmithKline, the United States Food and Drug Administration, Medtronic Foundation; consulting fees from Abbvie, Abiomed, Anthos Therapeutic LLC, Alnylam Pharm., Bayer Corporation US, Boston Scientific Corporation, Cardionomic, CeleCor Therapeutics, Correvio, Espero BioPharma, HengRui USA, Medscape LLC, Medtronic Inc., Merck, National Institutes of Health, NovoNordisk, Philips, REATA, Roche Diagnostics; equity in Tenac.io. RL reports institutional research grants and consulting fees from Bristol Myers Squibb, GlaxoSmithKline, Medtronic, Pfizer, and Sanofi; consulting fees from Amgen, Bayer, Boehringer Ingelheim, Daiichi Sankyo, Merck, and Portola. JA reports institutional research grants from Artivion/CryoLife, Bayer, Bristol-Myers Squibb, CSL Behring, Ferring, Humacyte, and XaTek and consulting or honoraria payments from Akros, Artivion/CryoLife, AtriCure, Bristol-Myers Squibb, Ferring, Janssen, Pfizer, Portola, and Quantum Genomics. LW reports institutional research grants from AstraZeneca, Boehringer Ingelheim, Bristol-Myers Squibb/Pfizer, GlaxoSmithKline, Roche Diagnostics, Merck & Co. NE reports institutional research grant from Bristol-Myers Squibb/Pfizer. SA is employed at the Swedish Medical Products Agency, SE-751 03 Uppsala, Sweden.

The remaining authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Publisher’s note

All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.

## Author disclaimer

The views expressed in this paper are the personal views of the authors and not necessarily the views of the Government agency.

## Supplementary material

The Supplementary Material for this article can be found online at: [https://www.frontiersin.org/articles/10.3389/fgene.2022.982955/full#supplementary-material](https://www.frontiersin.org/articles/10.3389/fgene.2022.982955/full#supplementary-material)

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Supplementary Materials

### Data Availability Statement

The data analyzed in this study is subject to the following licenses/restrictions: The datasets generated during and/or analysed during the current study are not publicly available due to the European General Data Protection Regulation (GDPR), which requires us to protect the identity of participants, but datasets are partly available from the corresponding author on reasonable request. Requests to access these datasets should be directed to Mia.Wadelius@medsci.uu.se; Niclas.Eriksson@ucr.uu.se.

### Supplementary Materials

### Data Availability Statement

The data analyzed in this study is subject to the following licenses/restrictions: The datasets generated during and/or analysed during the current study are not publicly available due to the European General Data Protection Regulation (GDPR), which requires us to protect the identity of participants, but datasets are partly available from the corresponding author on reasonable request. Requests to access these datasets should be directed to Mia.Wadelius@medsci.uu.se; Niclas.Eriksson@ucr.uu.se.

## References

1. Asic A., Marjanovic D., Mirat J., Primorac D. (2018). Pharmacogenetics of novel oral anticoagulants: A review of identified gene variants & future perspectives. Per. Med. 15 (3), 209–221. 10.2217/pme-2017-0092  [DOI](https://doi.org/10.2217/pme-2017-0092) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29767545/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Per.%20Med.&title=Pharmacogenetics%20of%20novel%20oral%20anticoagulants:%20A%20review%20of%20identified%20gene%20variants%20&%20future%20perspectives&author=A.%20Asic&author=D.%20Marjanovic&author=J.%20Mirat&author=D.%20Primorac&volume=15&issue=3&publication_year=2018&pages=209-221&pmid=29767545&doi=10.2217/pme-2017-0092&)

2. Cirincione B., Kowalski K., Nielsen J., Roy A., Thanneer N., Byon W., et al. (2018). Population pharmacokinetics of apixaban in subjects with nonvalvular atrial fibrillation. CPT. Pharmacometrics Syst. Pharmacol. 7 (11), 728–738. 10.1002/psp4.12347  [DOI](https://doi.org/10.1002/psp4.12347) | [PMC free article](/articles/PMC6263664/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30259707/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=CPT.%20Pharmacometrics%20Syst.%20Pharmacol.&title=Population%20pharmacokinetics%20of%20apixaban%20in%20subjects%20with%20nonvalvular%20atrial%20fibrillation&author=B.%20Cirincione&author=K.%20Kowalski&author=J.%20Nielsen&author=A.%20Roy&author=N.%20Thanneer&volume=7&issue=11&publication_year=2018&pages=728-738&pmid=30259707&doi=10.1002/psp4.12347&)

3. Connolly S. J., Ezekowitz M. D., Yusuf S., Eikelboom J., Oldgren J., Parekh A., et al. (2009). Dabigatran versus warfarin in patients with atrial fibrillation. N. Engl. J. Med. 361 (12), 1139–1151. 10.1056/NEJMoa0905561  [DOI](https://doi.org/10.1056/NEJMoa0905561) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19717844/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Dabigatran%20versus%20warfarin%20in%20patients%20with%20atrial%20fibrillation&author=S.%20J.%20Connolly&author=M.%20D.%20Ezekowitz&author=S.%20Yusuf&author=J.%20Eikelboom&author=J.%20Oldgren&volume=361&issue=12&publication_year=2009&pages=1139-1151&pmid=19717844&doi=10.1056/NEJMoa0905561&)

4. Dimatteo C., D'Andrea G., Vecchione G., Paoletti O., Tiscia G. L., Santacroce R., et al. (2016). ABCB1 SNP rs4148738 modulation of apixaban interindividual variability. Thromb. Res. 145, 24–26. 10.1016/j.thromres.2016.07.005  [DOI](https://doi.org/10.1016/j.thromres.2016.07.005) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27434880/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Thromb.%20Res.&title=ABCB1%20SNP%20rs4148738%20modulation%20of%20apixaban%20interindividual%20variability&author=C.%20Dimatteo&author=G.%20D'Andrea&author=G.%20Vecchione&author=O.%20Paoletti&author=G.%20L.%20Tiscia&volume=145&publication_year=2016&pages=24-26&pmid=27434880&doi=10.1016/j.thromres.2016.07.005&)

5. Durbin R. (2014). Efficient haplotype matching and storage using the positional Burrows-Wheeler transform (PBWT). Bioinformatics 30 (9), 1266–1272. 10.1093/bioinformatics/btu014  [DOI](https://doi.org/10.1093/bioinformatics/btu014) | [PMC free article](/articles/PMC3998136/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24413527/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Bioinformatics&title=Efficient%20haplotype%20matching%20and%20storage%20using%20the%20positional%20Burrows-Wheeler%20transform%20(PBWT)&author=R.%20Durbin&volume=30&issue=9&publication_year=2014&pages=1266-1272&pmid=24413527&doi=10.1093/bioinformatics/btu014&)

6. Eliquis (2022). Eliquis European public assessment report. Available at: https://www.ema.europa.eu/en/medicines/human/EPAR/eliquis (Accessed 05 30 2022).  [https://www.ema.europa.eu/en/medicines/human/EPAR/eliquis](https://www.ema.europa.eu/en/medicines/human/EPAR/eliquis) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Eliquis%20European%20public%20assessment%20report&publication_year=2022&)

7. Giugliano R. P., Ruff C. T., Braunwald E., Murphy S. A., Wiviott S. D., Halperin J. L., et al. (2013). Edoxaban versus warfarin in patients with atrial fibrillation. N. Engl. J. Med. 369 (22), 2093–2104. 10.1056/NEJMoa1310907  [DOI](https://doi.org/10.1056/NEJMoa1310907) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24251359/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Edoxaban%20versus%20warfarin%20in%20patients%20with%20atrial%20fibrillation&author=R.%20P.%20Giugliano&author=C.%20T.%20Ruff&author=E.%20Braunwald&author=S.%20A.%20Murphy&author=S.%20D.%20Wiviott&volume=369&issue=22&publication_year=2013&pages=2093-2104&pmid=24251359&doi=10.1056/NEJMoa1310907&)

8. Granger C. B., Alexander J. H., McMurray J. J., Lopes R. D., Hylek E. M., Hanna M., et al. (2011). Apixaban versus warfarin in patients with atrial fibrillation. N. Engl. J. Med. 365 (11), 981–992. 10.1056/NEJMoa1107039  [DOI](https://doi.org/10.1056/NEJMoa1107039) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21870978/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Apixaban%20versus%20warfarin%20in%20patients%20with%20atrial%20fibrillation&author=C.%20B.%20Granger&author=J.%20H.%20Alexander&author=J.%20J.%20McMurray&author=R.%20D.%20Lopes&author=E.%20M.%20Hylek&volume=365&issue=11&publication_year=2011&pages=981-992&pmid=21870978&doi=10.1056/NEJMoa1107039&)

9. Gulilat M., Keller D., Linton B., Pananos A. D., Lizotte D., Dresser G. K., et al. (2020). Drug interactions and pharmacogenetic factors contribute to variation in apixaban concentration in atrial fibrillation patients in routine care. J. Thromb. Thrombolysis 49 (2), 294–303. 10.1007/s11239-019-01962-2  [DOI](https://doi.org/10.1007/s11239-019-01962-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31564018/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Thromb.%20Thrombolysis&title=Drug%20interactions%20and%20pharmacogenetic%20factors%20contribute%20to%20variation%20in%20apixaban%20concentration%20in%20atrial%20fibrillation%20patients%20in%20routine%20care&author=M.%20Gulilat&author=D.%20Keller&author=B.%20Linton&author=A.%20D.%20Pananos&author=D.%20Lizotte&volume=49&issue=2&publication_year=2020&pages=294-303&pmid=31564018&doi=10.1007/s11239-019-01962-2&)

10. GWAS Catalog (2022). The NHGRI-EBI Catalog of human genome-wide association studies. Available at: https://www.ebi.ac.uk/gwas/ (Accessed 05 30 2022).  [https://www.ebi.ac.uk/gwas/](https://www.ebi.ac.uk/gwas/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20NHGRI-EBI%20Catalog%20of%20human%20genome-wide%20association%20studies&publication_year=2022&)

11. Heyes N., Kapoor P., Kerr I. D. (2018). Polymorphisms of the multidrug pump ABCG2: A systematic review of their effect on protein expression, function, and drug pharmacokinetics. Drug Metab. Dispos. 46 (12), 1886–1899. 10.1124/dmd.118.083030  [DOI](https://doi.org/10.1124/dmd.118.083030) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30266733/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Polymorphisms%20of%20the%20multidrug%20pump%20ABCG2:%20A%20systematic%20review%20of%20their%20effect%20on%20protein%20expression,%20function,%20and%20drug%20pharmacokinetics&author=N.%20Heyes&author=P.%20Kapoor&author=I.%20D.%20Kerr&volume=46&issue=12&publication_year=2018&pages=1886-1899&pmid=30266733&doi=10.1124/dmd.118.083030&)

12. Kanuri S. H., Kreutz R. P. (2019). Pharmacogenomics of novel direct oral anticoagulants: Newly identified genes and genetic variants. J. Pers. Med. 9 (1), 7. 10.3390/jpm9010007  [DOI](https://doi.org/10.3390/jpm9010007) | [PMC free article](/articles/PMC6463033/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/30658513/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pers.%20Med.&title=Pharmacogenomics%20of%20novel%20direct%20oral%20anticoagulants:%20Newly%20identified%20genes%20and%20genetic%20variants&author=S.%20H.%20Kanuri&author=R.%20P.%20Kreutz&volume=9&issue=1&publication_year=2019&pages=7&pmid=30658513&doi=10.3390/jpm9010007&)

13. Lahteenmaki J., Vuorinen A. L., Pajula J., Harno K., Lehto M., Niemi M., et al. (2021). Pharmacogenetics of bleeding and thromboembolic events in direct oral anticoagulant users. Clin. Pharmacol. Ther. 110 (3), 768–776. 10.1002/cpt.2316  [DOI](https://doi.org/10.1002/cpt.2316) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/34043814/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Pharmacogenetics%20of%20bleeding%20and%20thromboembolic%20events%20in%20direct%20oral%20anticoagulant%20users&author=J.%20Lahteenmaki&author=A.%20L.%20Vuorinen&author=J.%20Pajula&author=K.%20Harno&author=M.%20Lehto&volume=110&issue=3&publication_year=2021&pages=768-776&pmid=34043814&doi=10.1002/cpt.2316&)

14. Lenoir C., Terrier J., Gloor Y., Gosselin P., Daali Y., Combescure C., et al. (2022). Impact of the genotype and phenotype of CYP3A and P-gp on the apixaban and rivaroxaban exposure in a real-world setting. J. Pers. Med. 12 (4), 526. 10.3390/jpm12040526  [DOI](https://doi.org/10.3390/jpm12040526) | [PMC free article](/articles/PMC9028714/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35455642/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Pers.%20Med.&title=Impact%20of%20the%20genotype%20and%20phenotype%20of%20CYP3A%20and%20P-gp%20on%20the%20apixaban%20and%20rivaroxaban%20exposure%20in%20a%20real-world%20setting&author=C.%20Lenoir&author=J.%20Terrier&author=Y.%20Gloor&author=P.%20Gosselin&author=Y.%20Daali&volume=12&issue=4&publication_year=2022&pages=526&pmid=35455642&doi=10.3390/jpm12040526&)

15. Loh P. R., Danecek P., Palamara P. F., Fuchsberger C., Reshef Y. A., Finucane K. H., et al. (2016). Reference-based phasing using the haplotype reference Consortium panel. Nat. Genet. 48 (11), 1443–1448. 10.1038/ng.3679  [DOI](https://doi.org/10.1038/ng.3679) | [PMC free article](/articles/PMC5096458/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27694958/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=Reference-based%20phasing%20using%20the%20haplotype%20reference%20Consortium%20panel&author=P.%20R.%20Loh&author=P.%20Danecek&author=P.%20F.%20Palamara&author=C.%20Fuchsberger&author=Y.%20A.%20Reshef&volume=48&issue=11&publication_year=2016&pages=1443-1448&pmid=27694958&doi=10.1038/ng.3679&)

16. McCarthy S., Das S., Kretzschmar W., Delaneau O., Wood A. R., Teumer A., et al. (2016). A reference panel of 64, 976 haplotypes for genotype imputation. Nat. Genet. 48 (10), 1279–1283. 10.1038/ng.3643  [DOI](https://doi.org/10.1038/ng.3643) | [PMC free article](/articles/PMC5388176/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27548312/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&title=A%20reference%20panel%20of%2064,%20976%20haplotypes%20for%20genotype%20imputation&author=S.%20McCarthy&author=S.%20Das&author=W.%20Kretzschmar&author=O.%20Delaneau&author=A.%20R.%20Wood&volume=48&issue=10&publication_year=2016&pages=1279-1283&pmid=27548312&doi=10.1038/ng.3643&)

17. Nagar S., Walther S., Blanchard R. L. (2006). Sulfotransferase (SULT) 1A1 polymorphic variants *1, *2, and *3 are associated with altered enzymatic activity, cellular phenotype, and protein degradation. Mol. Pharmacol. 69 (6), 2084–2092. 10.1124/mol.105.019240  [DOI](https://doi.org/10.1124/mol.105.019240) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/16517757/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Mol.%20Pharmacol.&title=Sulfotransferase%20(SULT)%201A1%20polymorphic%20variants%20*1,%20*2,%20and%20*3%20are%20associated%20with%20altered%20enzymatic%20activity,%20cellular%20phenotype,%20and%20protein%20degradation&author=S.%20Nagar&author=S.%20Walther&author=R.%20L.%20Blanchard&volume=69&issue=6&publication_year=2006&pages=2084-2092&pmid=16517757&doi=10.1124/mol.105.019240&)

18. National Center for Biotechnology Information NIH National (2022). Library of medicine 340578. Availabale at: https://www.ncbi.nlm.nih.gov/gene/340578 (Accessed 07 18, 2022).  [https://www.ncbi.nlm.nih.gov/gene/340578](https://www.ncbi.nlm.nih.gov/gene/340578) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=Library%20of%20medicine%20340578&publication_year=2022&)

19. Pare G., Eriksson N., Lehr T., Connolly S., Eikelboom J., Ezekowitz M. D., et al. (2013). Genetic determinants of dabigatran plasma levels and their relation to bleeding. Circulation 127 (13), 1404–1412. 10.1161/CIRCULATIONAHA.112.001233  [DOI](https://doi.org/10.1161/CIRCULATIONAHA.112.001233) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23467860/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Circulation&title=Genetic%20determinants%20of%20dabigatran%20plasma%20levels%20and%20their%20relation%20to%20bleeding&author=G.%20Pare&author=N.%20Eriksson&author=T.%20Lehr&author=S.%20Connolly&author=J.%20Eikelboom&volume=127&issue=13&publication_year=2013&pages=1404-1412&pmid=23467860&doi=10.1161/CIRCULATIONAHA.112.001233&)

20. Patel M. R., Mahaffey K. W., Garg J., Pan G., Singer D. E., Hacke W., et al. (2011). Rivaroxaban versus warfarin in nonvalvular atrial fibrillation. N. Engl. J. Med. 365 (10), 883–891. 10.1056/NEJMoa1009638  [DOI](https://doi.org/10.1056/NEJMoa1009638) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/21830957/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&title=Rivaroxaban%20versus%20warfarin%20in%20nonvalvular%20atrial%20fibrillation&author=M.%20R.%20Patel&author=K.%20W.%20Mahaffey&author=J.%20Garg&author=G.%20Pan&author=D.%20E.%20Singer&volume=365&issue=10&publication_year=2011&pages=883-891&pmid=21830957&doi=10.1056/NEJMoa1009638&)

21. Pharmacogenomics and Pharmacogenetics (2022). The Pharmacogenomics knowledgebase. Available at: https://www.pharmgkb.org/.2022-05-30 (Accessed 05 30, 2022).  [https://www.pharmgkb.org/.2022-05-30](https://www.pharmgkb.org/.2022-05-30) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=The%20Pharmacogenomics%20knowledgebase&publication_year=2022&)

22. Raghavan N., Frost C. E., Yu Z., He K., Zhang H., Humphreys W. G., et al. (2009). Apixaban metabolism and pharmacokinetics after oral administration to humans. Drug Metab. Dispos. 37 (1), 74–81. 10.1124/dmd.108.023143  [DOI](https://doi.org/10.1124/dmd.108.023143) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18832478/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Apixaban%20metabolism%20and%20pharmacokinetics%20after%20oral%20administration%20to%20humans&author=N.%20Raghavan&author=C.%20E.%20Frost&author=Z.%20Yu&author=K.%20He&author=H.%20Zhang&volume=37&issue=1&publication_year=2009&pages=74-81&pmid=18832478&doi=10.1124/dmd.108.023143&)

23. Savic R. M., Karlsson M. O. (2009). Importance of shrinkage in empirical bayes estimates for diagnostics: Problems and solutions. AAPS J. 11 (3), 558–569. 10.1208/s12248-009-9133-0  [DOI](https://doi.org/10.1208/s12248-009-9133-0) | [PMC free article](/articles/PMC2758126/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19649712/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=AAPS%20J.&title=Importance%20of%20shrinkage%20in%20empirical%20bayes%20estimates%20for%20diagnostics:%20Problems%20and%20solutions&author=R.%20M.%20Savic&author=M.%20O.%20Karlsson&volume=11&issue=3&publication_year=2009&pages=558-569&pmid=19649712&doi=10.1208/s12248-009-9133-0&)

24. Scattershot (2021). McCarty Group Tools QC Script. Available at: https://www.well.ox.ac.uk/∼wrayner/tools/ (Accessed 10 27, 2021).  [https://www.well.ox.ac.uk/∼wrayner/tools/](https://www.well.ox.ac.uk/%7Ewrayner/tools/) | [Google Scholar](https://scholar.google.com/scholar_lookup?title=McCarty%20Group%20Tools%20QC%20Script&publication_year=2021&)

25. Sham P. C., Purcell S. M. (2014). Statistical power and significance testing in large-scale genetic studies. Nat. Rev. Genet. 15 (5), 335–346. 10.1038/nrg3706  [DOI](https://doi.org/10.1038/nrg3706) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24739678/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&title=Statistical%20power%20and%20significance%20testing%20in%20large-scale%20genetic%20studies&author=P.%20C.%20Sham&author=S.%20M.%20Purcell&volume=15&issue=5&publication_year=2014&pages=335-346&pmid=24739678&doi=10.1038/nrg3706&)

26. Shou W., Wang D., Zhang K., Wang B., Wang Z., Shi J., et al. (2012). Gene-wide characterization of common quantitative trait loci for ABCB1 mRNA expression in normal liver tissues in the Chinese population. PLoS One 7 (9), e46295. 10.1371/journal.pone.0046295  [DOI](https://doi.org/10.1371/journal.pone.0046295) | [PMC free article](/articles/PMC3458811/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23050008/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20One&title=Gene-wide%20characterization%20of%20common%20quantitative%20trait%20loci%20for%20ABCB1%20mRNA%20expression%20in%20normal%20liver%20tissues%20in%20the%20Chinese%20population&author=W.%20Shou&author=D.%20Wang&author=K.%20Zhang&author=B.%20Wang&author=Z.%20Wang&volume=7&issue=9&publication_year=2012&pages=e46295&pmid=23050008&doi=10.1371/journal.pone.0046295&)

27. Takeuchi F., McGinnis R., Bourgeois S., Barnes C., Eriksson N., Soranzo N., et al. (2009). A genome-wide association study confirms VKORC1, CYP2C9, and CYP4F2 as principal genetic determinants of warfarin dose. PLoS Genet. 5 (3), e1000433. 10.1371/journal.pgen.1000433  [DOI](https://doi.org/10.1371/journal.pgen.1000433) | [PMC free article](/articles/PMC2652833/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19300499/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&title=A%20genome-wide%20association%20study%20confirms%20VKORC1,%20CYP2C9,%20and%20CYP4F2%20as%20principal%20genetic%20determinants%20of%20warfarin%20dose&author=F.%20Takeuchi&author=R.%20McGinnis&author=S.%20Bourgeois&author=C.%20Barnes&author=N.%20Eriksson&volume=5&issue=3&publication_year=2009&pages=e1000433&pmid=19300499&doi=10.1371/journal.pgen.1000433&)

28. Ueshima S., Hira D., Fujii R., Kimura Y., Tomitsuka C., Yamane T., et al. (2017). Impact of ABCB1, ABCG2, and CYP3A5 polymorphisms on plasma trough concentrations of apixaban in Japanese patients with atrial fibrillation. Pharmacogenet. Genomics 27 (9), 329–336. 10.1097/FPC.0000000000000294  [DOI](https://doi.org/10.1097/FPC.0000000000000294) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28678049/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&title=Impact%20of%20ABCB1,%20ABCG2,%20and%20CYP3A5%20polymorphisms%20on%20plasma%20trough%20concentrations%20of%20apixaban%20in%20Japanese%20patients%20with%20atrial%20fibrillation&author=S.%20Ueshima&author=D.%20Hira&author=R.%20Fujii&author=Y.%20Kimura&author=C.%20Tomitsuka&volume=27&issue=9&publication_year=2017&pages=329-336&pmid=28678049&doi=10.1097/FPC.0000000000000294&)

29. Ueshima S., Hira D., Kimura Y., Fujii R., Tomitsuka C., Yamane T., et al. (2018). Population pharmacokinetics and pharmacogenomics of apixaban in Japanese adult patients with atrial fibrillation. Br. J. Clin. Pharmacol. 84 (6), 1301–1312. 10.1111/bcp.13561  [DOI](https://doi.org/10.1111/bcp.13561) | [PMC free article](/articles/PMC5980466/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/29457840/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Clin.%20Pharmacol.&title=Population%20pharmacokinetics%20and%20pharmacogenomics%20of%20apixaban%20in%20Japanese%20adult%20patients%20with%20atrial%20fibrillation&author=S.%20Ueshima&author=D.%20Hira&author=Y.%20Kimura&author=R.%20Fujii&author=C.%20Tomitsuka&volume=84&issue=6&publication_year=2018&pages=1301-1312&pmid=29457840&doi=10.1111/bcp.13561&)

30. Wang L., Raghavan N., He K., Luettgen J. M., Humphreys W. G., Knabb R. M., et al. (2009). Sulfation of o-demethyl apixaban: Enzyme identification and species comparison. Drug Metab. Dispos. 37 (4), 802–808. 10.1124/dmd.108.025593  [DOI](https://doi.org/10.1124/dmd.108.025593) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19131519/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Dispos.&title=Sulfation%20of%20o-demethyl%20apixaban:%20Enzyme%20identification%20and%20species%20comparison&author=L.%20Wang&author=N.%20Raghavan&author=K.%20He&author=J.%20M.%20Luettgen&author=W.%20G.%20Humphreys&volume=37&issue=4&publication_year=2009&pages=802-808&pmid=19131519&doi=10.1124/dmd.108.025593&)

31. Ward L. D., Kellis M. (2012). HaploReg: A resource for exploring chromatin states, conservation, and regulatory motif alterations within sets of genetically linked variants. Nucleic Acids Res. 40, D930–D934. 10.1093/nar/gkr917  [DOI](https://doi.org/10.1093/nar/gkr917) | [PMC free article](/articles/PMC3245002/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22064851/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&title=HaploReg:%20A%20resource%20for%20exploring%20chromatin%20states,%20conservation,%20and%20regulatory%20motif%20alterations%20within%20sets%20of%20genetically%20linked%20variants&author=L.%20D.%20Ward&author=M.%20Kellis&volume=40&publication_year=2012&pages=D930-D934&pmid=22064851&doi=10.1093/nar/gkr917&)
