# The impact of CYP2D6*41 on CYP2D6 enzyme activity using phenotyping methods in urine, plasma, and saliva

## Metadata
**Authors:** Ye Jin, Shuquan Zhang, Pei Hu, Xin Zheng, Xiaoduo Guan, Rui Chen, Shuyang Zhang
**Journal:** Frontiers in Pharmacology
**Date:** 2022 Aug 30
**DOI:** [10.3389/fphar.2022.940510](https://doi.org/10.3389/fphar.2022.940510)
**PMID:** 36110554
**PMCID:** PMC9468644
**URL:** https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9468644/
**PDF:** [https://pmc.ncbi.nlm.nih.gov/articles/PMC9468644/pdf/fphar-13-940510.pdf](https://pmc.ncbi.nlm.nih.gov/articles/PMC9468644/pdf/fphar-13-940510.pdf)

## Abstract

Aims: The CYP2D6*41 variant is the second or third frequent reduced function allele in Chinese with a frequency of around 3–4%, while it is the major reduced function allele in Indians, Saudi Arabians and Caucasians with frequencies of around 10–20%. The present study was designed to explore the impact of CYP2D6*41 on the metabolic activity of CYP2D6 using phenotyping methods in urine, plasma, and saliva.

Methods: We used dextromethorphan as the probe drug to analyze the phenotypes of 87 subjects with CYP2D6*1/*1 (n = 22), CYP2D6*1/*2 (n = 33), CYP2D6*2/*2 (n = 4), CYP2D6*1/*41 (n = 5), CYP2D6*2/*41 (n = 3), CYP2D6*10/*41 (n = 16), and CYP2D6*5/*41 (n = 4) for CYP2D6. The ratio of parent drug to metabolite in 3 h saliva, 3 h plasma, and in 0–3 h urine was considered the metabolic ratio (MR).

Results: The CYP2D6*41 allele had substantial impact on the metabolic activity of CYP2D6 regardless of the urinary, plasma, or salivary phenotyping method used. In subjects with CYP2D6*1(or *2)/*1(or *2), *1 (or *2)/*41, *10/*41 and *5/*41 (all p < 0.001), the salivary, plasma, or urinary MR value increased. The MRs in saliva, plasma, and urine displayed high correlations.

Conclusion: The activity score system or the consensus activity score system, instead of the traditional phenotype classification, could predict the CYP2D6 enzyme activity more accurately. CYP2D6*41 had similar or more impact on the CYP2D6 enzyme activity as compared with CYP2D6*10. Assigning *41 a score of 0.5 and assigning *10 a score of 0.25 according to the consensus AS system should be reconsidered.

Keywords: CYP2D6*41, phenotyping method, metabolic ratio, polymorphism, genotype

## Introduction

Cytochrome P450 2D6 (*CYP2D6*) is directly involved in the metabolism of approximately 25% of currently approved medications, including antidepressants, antipsychotics, analgesics and antitussives, beta adrenergic blocking agents, antiarrhythmic, antiemetics, etc. ([Zhou, 2009a](#B27); [Zhou, 2009b](#B28); [Murphy and McMahon, 2013](#B20); [Zanger and Schwab, 2013](#B26)). *CYP2D6* polymorphisms have been extensively investigated in terms of their effects on enzymatic activity. Differences in *CYP2D6* activity are derived from over 100 variants (The Pharmacogene Variation Consortium; [https://www.pharmvar.org/gene/CYP2D6](https://www.pharmvar.org/gene/CYP2D6)) ([Zanger et al., 2004](#B25); [Teh and Bertilsson, 2012](#B24)). Among these are fully functional alleles, reduced function alleles, nonfunctional alleles and gene copy duplicates, that range in activity from ultra-rapid metabolism to no metabolism ([Gaedigk, 2013](#B11); [Kevin Hicks et al., 2016](#B16); [Gaedigk et al., 2017](#B12)).

Previous studies have revealed significant ethnic and geographic differences in the frequencies of *CYP2D6* alleles ([Bradford, 2002](#B3)). *CYP2D6*10* is predominant in Asian populations, with the frequency ranging from 30 to 50% ([Qian et al., 2013](#B22)). *CYP2D6*4* distinguishes Caucasians from other populations with a frequency of 12–21% ([Qian et al., 2013](#B22)). The frequency of *CYP2D6*41* was reported to be 3.05% in Chinese ([Qian et al., 2013](#B22)), while it was identified as the major reduced function allele in Indian population with a frequency of 12.3% ([Manoharan et al., 2019](#B19)). High prevalence of *CYP2D6*41* was also reported in Saudi Arabians and Caucasians with frequencies of 18.4% ([Al-Dosari et al., 2013](#B2)) and 11.1% ([Dalton et al., 2020](#B9)), respectively.

In clinical settings, *CYP2D6* phenotypes are usually classified into metabolizer groups, like the following: ultra-rapid (UM), normal (NM), intermediate (IM), and poor (PM) ([Zanger et al., 2004](#B25)). Probe drugs are used to analyze the metabolic activity of *CYP2D6*, with the most-used substrate being dextromethorphan (DM) ([Chen et al., 2017](#B7)). The metabolic ratio (MR) of DM to its metabolite dextrorphan (DX) in 8-h urine after dosing is widely used to differentiate NMs from PMs ([Chladek et al., 2000](#B8); [O’mathúna et al., 2008](#B21); [Lötsch et al., 2009](#B18); [Ito et al., 2010](#B15)). Collecting urine at 8 h intervals is not feasible in clinical settings. As such, alternative techniques are used for fast and efficient phenotyping. Saliva and plasma samples can also be used to analyze the MR value. Chladek et al. reported a close correlation between the MRs measured using urine (0–4 h post-dose) and plasma (3 h post-dose) [16]. Additionally, MR from single-point plasma collected 1–30 h post-dose displayed a positive correlation with MR from area under the curve (AUC) ([Chen et al., 2016a](#B5); [Chen et al., 2016b](#B6)). In addition, the MR from 0–3 h urine, 3 h single-point plasma, and 3 h single-point saliva were reported to have good correlations and could all be used as alternative methods to determine the phenotype of *CYP2D6* ([Chen et al., 2017](#B7)).

Clinical Pharmacogenetics Implementation Consortium (CPIC) and Dutch Pharmacogenetics Working Group (DPWG) recently reported some discrepancies in their guidelines, primarily relevant to how certain *CYP2D6* genotypes were translated into phenotypes ([Caudle et al., 2019](#B4)). Gaedigk et al. introduced the activity score (AS) system for *CYP2D6* in 2008 ([Gaedigk et al., 2008](#B13)). Based on this system, a consensus definition method was also developed for more accurate “activity scores” ([Caudle et al., 2019](#B4)). In both systems, a value is assigned to each allele that designates its function. The total value of individual allele values is considered the final AS. It is well known that the *CYP2D6*41* variant is a reduced function allele that decreases enzyme activity and thus increases the *CYP2D6* MR value. In the both systems, *CYP2D6*41* variant is assigned a value of 0.5. Quantitative analysis on the how *CYP2D6*41* affects different combinations of alleles has not been reported in relatively large populations. In the present study, *CYP2D6*1* or **2* was defined as a standard full functional allele and *CYP2D6*5* was defined as a nonfunctional allele. In addition, *CYP2D6*10*, with the very high frequency in Chinese, was defined as a reduced function allele. *CYP2D6*1*, **2*, **5*, and **10* were all used as references or control alleles to investigate how *CYP2D6*41* affects different combinations of alleles. In the present study, salivary, plasma, and urinary phenotyping methods were used to analyze the activities of the *CYP2D6* enzyme in a healthy Chinese population. The results were then compared.

## Materials and methods

### Study subjects

The clinical study was carried out in accordance with the Guidelines for Good Clinical Practice and the Declaration of Helsinki. The clinical protocol was reviewed and approved by the Ethical Committee of Peking Union Medical College Hospital, Beijing, China. Each of the participants signed an informed consent form prior to enrollment. Four hundred and twenty-one subjects were enrolled in the study. After a detailed physical examination including routine urinalysis, hematology, biochemistry, and 12-lead electrocardiography, the subjects were declared healthy. Subjects were declared ineligible if they had the following: a history of hematologic, gastrointestinal, renal, or hepatic abnormalities, a human immunodeficiency virus, syphilis, hepatitis C or B, any chronic or acute disease, or were allergic to dextromethorphan. Consuming grapefruit juice, caffeinated beverages, or alcohol was not allowed in the 24 h before DM administration, nor until all samples were collected. Subjects were also instructed to refrain from ingesting herbal remedies or medication for a minimum of 1 week before the study, and to refrain from smoking for a minimum of 3 days before the study.

### CYP2D6 genotyping

We analyzed the DNA sequence, by genotype, of each subject in the study for *CYP2D6*1*, **2*, **3*, **4*, **6*, **7*, **10*, **14*, **18*, **21*, **28*, **33*, **34*, **35*, **36*, **39*, **41*, **43*, **49*, **51*, **52*, **54*, **60*, **63*, **65*, **69*, **71*, and **75* using a previously reported method ([Qian et al., 2013](#B22); [Chen et al., 2017](#B7)).

### CYP2D6 phenotyping with DM

We used DX as the *CYP2D6*-specific metabolite and DM as the probe to analyze the phenotypes of 87 subjects with *CYP2D6*1*/**1* (*n* = 22), *CYP2D6*1*/**2* (*n* = 33), *CYP2D6*2*/**2* (*n* = 4), *CYP2D6*1*/**41* (*n* = 5), *CYP2D6*2*/**41* (*n* = 3), *CYP2D6*10*/**41* (*n* = 16), and *CYP2D6*5*/**41* (*n* = 4). We provided each subject with 15 mg of DM (Tylenol Cold Tablet containing DM, Johnson & Johnson Investment Ltd., Shanghai, China), along with 300 ml of water. Samples of saliva and venous blood were collected 3 h after the drug was administered, while samples of urine were collected at intervals from 0 to 3 h after the drug was administered. We analyzed DM concentrations and unconjugated DX in all of the samples using a validated and sensitive high-performance liquid chromatography tandem mass spectrometry (HPLC-MS/MS) assay, according to methods previously used [24, 25]. For DX and DM in the saliva, plasma, and urine samples, the lower limit of quantification was 0.05 ng/ml [15]. In order to measure the *CYP2D6* activity in the three types, a metabolic ratio of the concentration of DM over DX (MR_DM/DX_) was used.

### Statistical analysis

Data used were mean values ± SD. An analysis of variance (ANOVA) test was used to analyze the MR values between different groups of genotypes, based on the salivary, plasma, and urinary phenotyping methods. Alleles *CYP2D6*2* and **1* were considered standard and fully functional, *CYP2D6*5* was considered a nonfunctional allele, *CYP2D6*10* was defined as a reduced function allele. All *CYP2D6*1*, **2*, **5*, and **10* were considered control alleles, allowing us to analyze the effects of the *CYP2D6*41* allele on the metabolic activity of *CYP2D6*. In the original activity score system ([Gaedigk et al., 2008](#B13)), sore of 0 was assigned to *CYP2D6*5*; score of 0.5 was assigned to *CYP2D6*10* and **41*; score of one was assigned to *CYP2D6*1* and **2*. In the consensus activity score system ([Caudle et al., 2019](#B4)), sore of 0 was assigned to *CYP2D6*5*; score of 0.25 was assigned to *CYP2D6*10*; score of 0.5 was assigned to *CYP2D6*41*; score of one was assigned to *CYP2D6*1* and **2*. The score of a genotype was the sum of the values assigned to each allele in both of the systems. A *p* value less than 0.05 was considered to be statistically significant. The ANOVA was performed with SPSS (version 19.0, SPSS™).

## Results

### CYP2D6 genotypes and demographic characteristics

The gene frequency of *CYP2D6*41* was 3.56% among the 421 subjects. Twenty-two subjects had *CYP2D6*1*/**1*, 33 subjects had *CYP2D6*1*/**2*, 4 subjects had *CYP2D6*2*/**2*, 5 subjects had *CYP2D6*1*/**41*, 3 subjects had *CYP2D6*2*/**41*, 16 subjects had *CYP2D6*10*/**41*, and 4 subjects had *CYP2D6*5*/**41*. The *CYP2D6*41* allele was also observed in 2 gene duplications which were not reported in this study. We also tested additional *CYP2D6* alleles to assess the particulars of *CYP2D6*1*, **2*, **5*, **10* and **41*.

Subjects with *CYP2D6*1*/**1*, **1*/**2*, and **2*/**2* were classified as the wild type. All 87 subjects’ ages, weights, body mass index (BMI) and genders can be found in [Table 1](#T1).

### TABLE 1.

| Characteristic | All (n = 87) | Wild Type (CYP2D6*1/*1, *1/*2, and *2/*2) (as = 2, CAS = 2) (n = 59) | CYP2D6*1/*41 and *2/*41 (as = 1.5, CAS = 1.5) (n = 8) | CYP2D6*10/*41 (as = 1,CAS = 0.75) (n = 16) | CYP2D6*5/*41 (as = 0.5, CAS = 0.5) (n = 4) |
| --- | --- | --- | --- | --- | --- |
| Age (years) | 30.4 ± 8.3 | 30.9 ± 8.2 | 26.6 ± 4.6 | 29.4 ± 8.2 | 34.0 ± 14.7 |
| Weight (kg) | 66.9 ± 8.5 | 67.3 ± 8.8 | 67.3 ± 6.1 | 66.3 ± 9.0 | 63.6 ± 6.9 |
| BMI (kg/m2) | 23.7 ± 2.4 | 24.0 ± 2.5 | 23.5 ± 2.4 | 22.8 ± 2.2 | 22.9 ± 1.6 |
| Gender | Male = 63 | Male = 40 | Male = 7 | Male = 13 | Male = 3 |
|   | Female = 24 | Female = 19 | Female = 1 | Female = 3 | Female = 1 |

Table 1 Caption: Demographic characteristics of the study population of healthy Chinese volunteers (mean ± SD).

### CYP2D6 phenotypes

The mean MR in the 87 subjects was 4.69 ± 30.8, based on the MR values from the 3 h saliva samples. MR ranged from 0.102 to 288 and had a 2800-fold inter-individual range. The mean MR in the 87 subjects was 1.66 ± 10.5, based on the MR values from the 3 h plasma samples. MR ranged from 0.0266 to 97.9 and had a 3600-fold inter-individual range. The mean urinary MR in the 87 subjects was 0.276 ± 1.41, based on the MR values from the 0–3 h urine samples. MR ranged from 0.00603 to 13.1, and had a 2,100-fold inter-individual range.

The *CYP2D6*41* allele significantly affected the metabolic activity of *CYP2D6* across all methods used. The mean (±SD) MRs for different Activity Score (AS) or Consensus activity score (CAS) (referring to the consensus CYP2D6 genotype to phenotype translation method in ([Caudle et al., 2019](#B4))) groups were presented by the three phenotyping methods, respectively ([Table 2](#T2); [Figure 1](#F1)).

### TABLE 2.

|   | Group 1 (wild type: CYP2D6*1/*1, *1/*2, and *2/*2) (as = 2, CAS = 2) (n = 59) | Group 2 (CYP2D6*1/*41 and *2/*41) (as = 1.5, CAS = 1.5) (n = 8) | Group 3 (CYP2D6*10/*41) (as = 1, CAS = 0.75) (n = 16) | Group 4 (CYP2D6*5/*41) (as = 0.5, CAS = 0.5) (n = 4) | p value (group 1 vs. Group 2) | p value (group 2 vs. Group 3) | p value (group 3 vs. Group 4) | p value (all groups) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Urinary MR | 0.0496 ± 0.0364 | 0.0942 ± 0.136 | 0.304 ± 0.258 | 3.87 ± 6.17 | 0.040 | 0.044 | 0.021 | <0.001 |
| Plasma MR | 0.191 ± 0.256 | 0.494 ± 0.541 | 1.21 ± 0.432 | 27.4 ± 47.0 | 0.009 | 0.002 | 0.025 | <0.001 |
| Salivary MR | 0.461 ± 0.305 | 0.971 ± 0.547 | 3.40 ± 1.56 | 79.7 ± 138 | <0.001 | <0.001 | 0.027 | <0.001 |

Table 2 Caption: The MRs for different Activity Score (AS) or Consensus Activity Score (CAS) groups based on the three phenotyping methods.

### FIGURE 1.

![FIGURE 1](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0063/9468644/3ba84492d761/fphar-13-940510-g001.jpg)

The box plots of metabolic ratio (MR) in the wild type (CYP2D6*1/*1, *1/*2, *2/*2) and the three CYP2D6*41 allele relevant genotypes in 87 healthy Chinese subjects. Allele *W represents CYP2D6*1 or *2. The y-axis is logarithmic and the ratio is based on the MRDM/DX. (A) urinary MR; (B) plasma MR; (C) salivary MR. Box plot explanation: upper horizontal line of box, 75th percentile; lower horizontal line of box, 25th percentile; horizontal bar within box, median; upper horizontal bar outside box, 95th percentile; lower horizontal bar outside box, 5th percentile. Circles represent outliers.

The salivary, plasma, and urinary MRs exhibited a statistically significant increase in subjects with *CYP2D6*1*(or **2*)/**1*(or **2*), **1* (or **2*)/**41*, **10*/**41*, and **5*/**41* (all *p* values < 0.05) ([Table 2](#T2)).

### Correlations among urinary, plasma, and salivary MR

We observed a statistically significant correlation between plasma MR and urinary MR. The Spearman’s correlation coefficient assessing the statistical dependence between the two variables was 0.634 (*p* < 0.001) ([Figure 2A](#F2)). A statistically significant correlation was also observed between salivary MR and urinary MR with a Spearman’s correlation coefficient of 0.690 (*p* < 0.001) ([Figure 2B](#F2)). There was also a statistically significant relationship between salivary MR and plasma MR, while the Spearman’s correlation coefficient was 0.842 (*p* < 0.001) ([Figure 2C](#F2)).

### FIGURE 2.

![FIGURE 2](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0063/9468644/fdced5723072/fphar-13-940510-g002.jpg)

Correlations among urinary, plasma and salivary MRs in the wild type (CYP2D6*1/*1, *1/*2, *2/*2) and the three CYP2D6*41 allele relevant genotypes, respectively, in double logarithmic coordinates. Allele *W represents CYP2D6*1 or *2. (A) correlation between urinary MR and plasma MR (Spearman’s correlation coefficient = 0.634 with p < 0.001); (B) correlation between urinary MR and salivary MR (Spearman’s correlation coefficient = 0.690 with p < 0.001); (C) correlation between plasma MR and salivary MR (Spearman’s correlation coefficient = 0.842 with p < 0.001).

## Discussion

The *CYP2D6*41* variant is the second or third frequent reduced function allele in Chinese (3–4%) and the CYP2D6*∗*10 variant is the most frequent allele (42.6%) ([Qian et al., 2013](#B22)). As a well-known reduced function allele besides *CYP2D6*10*, *CYP2D6*41* impacts the enzyme activity of *CYP2D6* to a great extent. Significantly, relatively high frequency of both *CYP2D6*10* and **41* alleles in Chinese population as well as the large number of subjects included in this study enable the comparison of *CYP2D6*41* metabolic ratio to *CYP2D6*10* in the same population. Results of this study add to the evidence of decreasing *CYP2D6*41* activity score.

Translating *CYP2D6* genotype to metabolizer phenotype is a topic of great interest in the scientific community and is the focus of several CPIC and DPWG guidelines. According to the previous CPIC guideline, the *CYP2D6* AS is translated into a phenotype using the following classification system: individuals with an AS of 0 are PMs, those with a score of 0.5 are IMs, those with a score of 1.0–2.0 are NMs, and those with a score > 2 are UMs ([Kevin Hicks et al., 2016](#B16)). According to the previous DPWG guideline, the *CYP2D6* AS is then translated into a phenotype using another classification system: individuals with an AS of 0 are PMs, those with a score of 0.5–1.0 are IMs, those with a score of 1.5–2.5 are NMs, and those with a score > 2.5 are UMs ([Caudle et al., 2019](#B4)). In the Consensus Activity Score (CAS) system, the translation method has been modified. Individuals with an AS of 0 are PMs, those with a score of 0.25–1.0 are IMs, those with a score of 1.25–2.25 are NMs, and those with a score > 2.25 are UMs ([Caudle et al., 2019](#B4)). No matter what system is used for the genotype to phenotype translation, assigning a score to an allele is of great importance and a challenge, especially for reduced function alleles. The present study was conducted to investigate the impact of the *CYP2D6*41* allele on *CYP2D6* metabolic activity. To exclude confounding factors, we defined *CYP2D6*1* and **2* as the full functional allele and *CYP2D6*5* as the nonfunctional allele. *CYP2D6*10*, as a typical reduced function allele, was also involved for comparison. All alleles were used as controls to investigate the impact of *CYP2D6*41* in different allelic combinations.

Statistically significant increases were observed in the salivary, plasma, and urinary MR values in subjects with *CYP2D6*1*(or **2*)/**1*(or **2*), **1* (or **2*)/**41*, **10*/**41*, and **5*/**41* (all *p* values < 0.05). This result means that as compared with the wild type, one *CYP2D6*41* allele combined with one full function allele will significantly reduce the enzyme activity. Likewise, one *CYP2D6*41* allele combined with one reduced function allele and one *CYP2D6*41* allele combined with one nonfunctional allele will reduce the enzyme activity further and further with statistical significance.

According to the DPWG guideline or consensus activity score system, *CYP2D6*10*/**41* and **5*/**41* are both classified as IMs. However, the MR values of the two groups were significantly different. In addition, in the both systems, group 1 (*CYP2D6*1*/**1*, **1*/**2*, and **2*/**2*) and group 2 (*CYP2D6*1*/**41* and **2*/**41*) are both classified as NMs, while the MR values of the two groups were significantly different. According to the CPIC guideline, group 1, 2, and 3 are all classified as NMs, while the MR values in these three groups could be more than ten times different with statistical significance. In the current guidelines or translation systems of *CYP2D6*, the definition of NMs or IMs seems to be not reasonably accurate or predictable. The activity score system or the consensus activity score system, instead of the traditional phenotype classification, could predict the *CYP2D6* enzyme activity more accurately.

Assigning a score to an allele is important and a challenge for *CYP2D6*41*. In a previous study with the similar study design ([Chen et al., 2017](#B7)), the urinary, plasma, or salivary MRs in subjects with *CYP2D6*1*/**10* were reported to be 0.0915, 0.351, and 1.05, respectively. The urinary, plasma, or salivary MRs in subjects with *CYP2D6*10*/**10* were reported to be 0.420, 2.03, and 5.09, respectively. The urinary, plasma, or salivary MRs in subjects with *CYP2D6*5*/**10* were reported to be 1.96, 5.63, and 19.8, respectively (data were presented in the previous study). The comparison between subjects with *CYP2D6*1*/**41* and those with *CYP2D6*1*/**10* showed the MRs in the two groups were quite close. The MRs between subjects with *CYP2D6*10*/**41* and *CYP2D6*10*/**10* were different but within two-fold. The MRs of subjects with *CYP2D6*5*/**41* were generally larger than the MRs of those with *CYP2D6*5*/**10* (with no significance) ([Table 3](#T3); [Figure 3](#F3)). According to the results based on the limited sample size, *CYP2D6*41* had similar impact on the *CYP2D6* enzyme activity as compared with *CYP2D6*10*, or *CYP2D6*41* reduced the enzyme activity more than *CYP2D6*10*. In no matter which case, assigning **41* a score of 0.5 and assigning **10* a score of 0.25 according to the consensus AS system could be reconsidered. However, when comparing enzymatic activity of different *CYP2D6* alleles and evaluating their activity score, an indispensable consideration is the substrate specificity. Changing study substrate might come to opposite conclusions ([Steimer et al., 2004](#B23); [Hicks et al., 2014](#B14)). Further clinical studies would be needed to provide more information.

### TABLE 3.

|   | CYP2D6*1/*X |  | CYP2D6*10/*X |  | CYP2D6*5/*X |  |
| --- | --- | --- | --- | --- | --- | --- |
| CYP2D6*1/*41 (n = 5) | CYP2D6*1/*10 (n = 93) | CYP2D6*10/*41 (n = 16) | CYP2D6*10/*10 (n = 85) | CYP2D6*5/*41 (n = 4) | CYP2D6*5/*10 (n = 35) |  |
| Plasma MR | 0.586 ± 0.688 | 0.351 ± 0.180 | 1.21 ± 0.432 | 2.02 ± 3.57 | 27.4 ± 47.0 | 5.63 ± 15.7 |
| p value | 0.488 a | 0.365 | 0.424 a |   |   |   |

Table 3 Caption: The plasma MR comparisons between CYP2D6*1/*41 vs. *1/*10, *10/*41 vs. *10/*10, and *5/*41 vs. *5/*10.

### FIGURE 3.

![FIGURE 3](https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0063/9468644/52f91e43765c/fphar-13-940510-g003.jpg)

[View larger image](https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&p=PMC3&id=9468644_fphar-13-940510-g003.jpg)

The box plot of comparison of plasma MRs for CYP2D6*1/*41 vs. *1/*10, CYP2D6*10/*41 vs. *10/*10, and CYP2D6*5/*41 vs. *5/*10 genotypes in the subjects. The legend *X/*41 represents genotype *1/*41, *10/*41 or *5/*41. The legend *X/*10 represents genotype *1/*10, *10/*10 or *5/*10.

The correlation between the MRs from 0–3 h urine, 3 h single-point plasma, and 3 h single-point saliva were reported to be good in subjects with wild type and **10* relevant genotypes of *CYP2D6* in the previous study ([Chen et al., 2017](#B7)). In the present study, the correlation between the MRs from the three kinds of samples were demonstrated to be good in subjects with wild type and **41* relevant genotypes of *CYP2D6*. These two studies found that salivary, plasma, and urinary MRs displayed a high correlation, and that different phenotyping techniques can be used in clinical settings. This result would be useful for clinical practice. For a long time, urine sample collection for 8 h after dosing is the standard process for phenotyping patients, while it was quite a demanding process in clinical practice, especially for outpatients who had to deliver the urinary samples at a later time. Single-point plasma and salivary phenotyping methods would save much time and be more convenient for clinical practice. Other previous studies also indicated that MR from single-point plasma from 1 to 30 h after a single dose of DM could predict the MR from AUC well and could be used as the *CYP2D6* phenotyping method for NMs, IMs, and PMs ([Chen et al., 2016a](#B5); [Chen et al., 2016b](#B6)). The result was consistent with data previously reported from small samples ([Chladek et al., 2000](#B8); [Frank et al., 2007](#B10)).

While their absolute values might be different, the DM/total-DX ratio and the DM/free-DX ratio can both be used to phenotype *CYP2D6*. As same as in the previous study ([Chen et al., 2017](#B7)), only unconjugated DX was observed in saliva, while the DM/free-DX ratio was in each of this study’s three sample types. As such, the accepted anti-mode of DM/total-DX of 0.3 between NMs and PMs did not apply to this study. We observed significant overlaps of MR values between similar genotypes and it was always the challenge when predicting phenotypes from *CYP2D6* genotypes. It was difficult to explain some outliers observed in the present study, and therefore there was inevitable uncertainty as to whether an individual with a certain allele would express the predicted phenotype ([Leeder and Gaedigk, 2014](#B17)). Particularly, RNA level or protein level is a closer indicator to phenotype than genotype. Recent study suggests liquid biopsy provides a practical way to quantify cytochrome P450 expression level which correlates well to metabolic activity. This promising technique may contribute to precision dosing therapy ([Achour et al., 2022](#B1)).

## Conclusion

In subjects with *CYP2D6*1*(or **2*)/**1*(or **2*), **1* (or **2*)/**41*, **10*/**41*, and **5*/**41*, the salivary, plasma, or urinary MR values increased successively (all *p* < 0.001). The activity score system or the consensus activity score system, instead of the traditional phenotype classification, could predict the *CYP2D6* enzyme activity more accurately. *CYP2D6*41* had similar or more impact on the *CYP2D6* enzyme activity as compared with *CYP2D6*10*. Assigning **41* a score of 0.5 and assigning **10* a score of 0.25 according to the consensus AS system should be reconsidered. Alternative techniques for phenotyping are saliva and single-point plasma, which provide significant clinical convenience.

## Acknowledgments

We acknowledge all volunteers and working staff who have contributed to the study.

## Data availability statement

The datasets presented in this article are not readily available due to privacy and ethical restrictions. Requests to access the datasets should be directed to the corresponding authors.

## Ethics statement

The studies involving human participants were reviewed and approved by Ethical Committee of Peking Union Medical College Hospital, Beijing, China. The patients/participants provided their written informed consent to participate in this study.

## Author contributions

RC performed and designed the study. RC, YJ and ShuquanZ performed statistical analysis and wrote the manuscript. XZ, XG analyzed the samples. PH designed the study. ShuyangZ and PH contributed to conception of the study.

## Funding

The work was supported by National High Level Hospital Clinical Research Funding (2022-PUMCH-A-060), Clinical Research Fund from WU JIEPING Medical Foundation (320.6750.19090-12) and the Drug Development and Application Fund from Chinese Pharmacological Society (2018DL001). It was also supported by a grant from the Beijing Key Laboratory (Z151100001615061).

## Conflict of interest

The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

## Publisher’s note

All claims expressed in this article are solely those of the authors and do not necessarily represent those of their affiliated organizations, or those of the publisher, the editors and the reviewers. Any product that may be evaluated in this article, or claim that may be made by its manufacturer, is not guaranteed or endorsed by the publisher.

## Associated Data

*This section collects any data citations, data availability statements, or supplementary materials included in this article.*

### Data Availability Statement

The datasets presented in this article are not readily available due to privacy and ethical restrictions. Requests to access the datasets should be directed to the corresponding authors.

### Data Availability Statement

The datasets presented in this article are not readily available due to privacy and ethical restrictions. Requests to access the datasets should be directed to the corresponding authors.

## References

1. Achour B., Gosselin P., Terrier J., Gloor Y., Al-Majdoub Z. M., Polasek T. M., et al. (2022). Liquid biopsy for patient characterization in cardiovascular disease: Verification against markers of cytochrome P450 and P-glycoprotein activities. Clin. Pharmacol. Ther. 111 (6), 1268–1277. 10.1002/cpt.2576  [DOI](https://doi.org/10.1002/cpt.2576) | [PMC free article](/articles/PMC9313840/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/35262906/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Liquid%20biopsy%20for%20patient%20characterization%20in%20cardiovascular%20disease:%20Verification%20against%20markers%20of%20cytochrome%20P450%20and%20P-glycoprotein%20activities&author=B.%20Achour&author=P.%20Gosselin&author=J.%20Terrier&author=Y.%20Gloor&author=Z.%20M.%20Al-Majdoub&volume=111&issue=6&publication_year=2022&pages=1268-1277&pmid=35262906&doi=10.1002/cpt.2576&)

2. Al-Dosari M. S., Al-Jenoobi F. I., Alkharfy K. M., Alghamdi A. M., Bagulb K. M., Parvez M. K., et al. (2013). High prevalence of CYP2D6*41 (G2988A) allele in Saudi Arabians. Environ. Toxicol. Pharmacol. 36 (3), 1063–1067. 10.1016/j.etap.2013.09.008  [DOI](https://doi.org/10.1016/j.etap.2013.09.008) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24121619/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Environ.%20Toxicol.%20Pharmacol.&title=High%20prevalence%20of%20CYP2D6*41%20(G2988A)%20allele%20in%20Saudi%20Arabians&author=M.%20S.%20Al-Dosari&author=F.%20I.%20Al-Jenoobi&author=K.%20M.%20Alkharfy&author=A.%20M.%20Alghamdi&author=K.%20M.%20Bagulb&volume=36&issue=3&publication_year=2013&pages=1063-1067&pmid=24121619&doi=10.1016/j.etap.2013.09.008&)

3. Bradford L. D. (2002). CYP2D6 allele frequency in European Caucasians, Asians, Africans and their descendants. Pharmacogenomics 3 (2), 229–243. 10.1517/14622416.3.2.229  [DOI](https://doi.org/10.1517/14622416.3.2.229) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11972444/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CYP2D6%20allele%20frequency%20in%20European%20Caucasians,%20Asians,%20Africans%20and%20their%20descendants&author=L.%20D.%20Bradford&volume=3&issue=2&publication_year=2002&pages=229-243&pmid=11972444&doi=10.1517/14622416.3.2.229&)

4. Caudle K. E., Sangkuhl K., Whirl-Carrillo M., Swen J. J., Haidar C. E., Klein T. E., et al. (2019). Standardizing CYP2D6 genotype to phenotype translation: Consensus recommendations from the clinical Pharmacogenetics implementation Consortium and Dutch Pharmacogenetics working group. Clin. Transl. Sci. 13, 116–124. 10.1111/cts.12692  [DOI](https://doi.org/10.1111/cts.12692) | [PMC free article](/articles/PMC6951851/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31647186/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Transl.%20Sci.&title=Standardizing%20CYP2D6%20genotype%20to%20phenotype%20translation:%20Consensus%20recommendations%20from%20the%20clinical%20Pharmacogenetics%20implementation%20Consortium%20and%20Dutch%20Pharmacogenetics%20working%20group&author=K.%20E.%20Caudle&author=K.%20Sangkuhl&author=M.%20Whirl-Carrillo&author=J.%20J.%20Swen&author=C.%20E.%20Haidar&volume=13&publication_year=2019&pages=116-124&pmid=31647186&doi=10.1111/cts.12692&)

5. Chen R., Rostami-Hodjegan A., Wang H., Berk D., Shi J., Hu P. (2016a). Application of a physiologically based pharmacokinetic model for the evaluation of single-point plasma phenotyping method of CYP2D6. Eur. J. Pharm. Sci. 92, 131–136. 10.1016/j.ejps.2016.07.001  [DOI](https://doi.org/10.1016/j.ejps.2016.07.001) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27412587/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pharm.%20Sci.&title=Application%20of%20a%20physiologically%20based%20pharmacokinetic%20model%20for%20the%20evaluation%20of%20single-point%20plasma%20phenotyping%20method%20of%20CYP2D6&author=R.%20Chen&author=A.%20Rostami-Hodjegan&author=H.%20Wang&author=D.%20Berk&author=J.%20Shi&volume=92&publication_year=2016a&pages=131-136&pmid=27412587&doi=10.1016/j.ejps.2016.07.001&)

6. Chen R., Wang H., Shi J., Hu P. (2016b). Alternative methods for CYP2D6 phenotyping: Comparison of dextromethorphan metabolic ratios from AUC, single point plasma, and urine. Int. J. Clin. Pharmacol. Ther. 54 (5), 330–336. 10.5414/CP202387  [DOI](https://doi.org/10.5414/CP202387) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/26902503/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Clin.%20Pharmacol.%20Ther.&title=Alternative%20methods%20for%20CYP2D6%20phenotyping:%20Comparison%20of%20dextromethorphan%20metabolic%20ratios%20from%20AUC,%20single%20point%20plasma,%20and%20urine&author=R.%20Chen&author=H.%20Wang&author=J.%20Shi&author=P.%20Hu&volume=54&issue=5&publication_year=2016b&pages=330-336&pmid=26902503&doi=10.5414/CP202387&)

7. Chen R., Zheng X., Hu P. (2017). CYP2D6 phenotyping using urine, plasma, and saliva metabolic ratios to assess the impact of CYP2D6( *)10 on interindividual variation in a Chinese population. Front. Pharmacol. 8, 239. 10.3389/fphar.2017.00239  [DOI](https://doi.org/10.3389/fphar.2017.00239) | [PMC free article](/articles/PMC5411458/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/28512430/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol.&title=CYP2D6%20phenotyping%20using%20urine,%20plasma,%20and%20saliva%20metabolic%20ratios%20to%20assess%20the%20impact%20of%20CYP2D6(%20*)10%20on%20interindividual%20variation%20in%20a%20Chinese%20population&author=R.%20Chen&author=X.%20Zheng&author=P.%20Hu&volume=8&publication_year=2017&pages=239&pmid=28512430&doi=10.3389/fphar.2017.00239&)

8. Chladek J., Zimova G., Beranek M., Martinkova J. (2000). In-vivo indices of CYP2D6 activity: Comparison of dextromethorphan metabolic ratios in 4-h urine and 3-h plasma. Eur. J. Clin. Pharmacol. 56 (9-10), 651–657. 10.1007/s002280000218  [DOI](https://doi.org/10.1007/s002280000218) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/11214771/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=In-vivo%20indices%20of%20CYP2D6%20activity:%20Comparison%20of%20dextromethorphan%20metabolic%20ratios%20in%204-h%20urine%20and%203-h%20plasma&author=J.%20Chladek&author=G.%20Zimova&author=M.%20Beranek&author=J.%20Martinkova&volume=56&issue=9-10&publication_year=2000&pages=651-657&pmid=11214771&doi=10.1007/s002280000218&)

9. Dalton R., Lee S. B., Claw K. G., Prasad B., Phillips B. R., Shen D. D., et al. (2020). Interrogation of CYP2D6 structural variant alleles improves the correlation between CYP2D6 genotype and CYP2D6-mediated metabolic activity. Clin. Transl. Sci. 13 (1), 147–156. 10.1111/cts.12695  [DOI](https://doi.org/10.1111/cts.12695) | [PMC free article](/articles/PMC6951848/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31536170/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Transl.%20Sci.&title=Interrogation%20of%20CYP2D6%20structural%20variant%20alleles%20improves%20the%20correlation%20between%20CYP2D6%20genotype%20and%20CYP2D6-mediated%20metabolic%20activity&author=R.%20Dalton&author=S.%20B.%20Lee&author=K.%20G.%20Claw&author=B.%20Prasad&author=B.%20R.%20Phillips&volume=13&issue=1&publication_year=2020&pages=147-156&pmid=31536170&doi=10.1111/cts.12695&)

10. Frank D., Jaehde U., Fuhr U. (2007). Evaluation of probe drugs and pharmacokinetic metrics for CYP2D6 phenotyping. Eur. J. Clin. Pharmacol. 63 (4), 321–333. 10.1007/s00228-006-0250-8  [DOI](https://doi.org/10.1007/s00228-006-0250-8) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17273835/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&title=Evaluation%20of%20probe%20drugs%20and%20pharmacokinetic%20metrics%20for%20CYP2D6%20phenotyping&author=D.%20Frank&author=U.%20Jaehde&author=U.%20Fuhr&volume=63&issue=4&publication_year=2007&pages=321-333&pmid=17273835&doi=10.1007/s00228-006-0250-8&)

11. Gaedigk A. (2013). Complexities of CYP2D6 gene analysis and interpretation. Int. Rev. Psychiatry 25 (5), 534–553. 10.3109/09540261.2013.825581  [DOI](https://doi.org/10.3109/09540261.2013.825581) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24151800/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Int.%20Rev.%20Psychiatry&title=Complexities%20of%20CYP2D6%20gene%20analysis%20and%20interpretation&author=A.%20Gaedigk&volume=25&issue=5&publication_year=2013&pages=534-553&pmid=24151800&doi=10.3109/09540261.2013.825581&)

12. Gaedigk A., Sangkuhl K., Whirl-Carrillo M., Klein T., Leeder J. S. (2017). Prediction of CYP2D6 phenotype from genotype across world populations. Genet. Med. 19 (1), 69–76. 10.1038/gim.2016.80  [DOI](https://doi.org/10.1038/gim.2016.80) | [PMC free article](/articles/PMC5292679/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27388693/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Genet.%20Med.&title=Prediction%20of%20CYP2D6%20phenotype%20from%20genotype%20across%20world%20populations&author=A.%20Gaedigk&author=K.%20Sangkuhl&author=M.%20Whirl-Carrillo&author=T.%20Klein&author=J.%20S.%20Leeder&volume=19&issue=1&publication_year=2017&pages=69-76&pmid=27388693&doi=10.1038/gim.2016.80&)

13. Gaedigk A., Simon S., Pearce R., Bradford L., Kennedy M., Leeder J. (2008). The CYP2D6 activity score: Translating genotype information into a qualitative measure of phenotype. Clin. Pharmacol. Ther. 83 (2), 234–242. 10.1038/sj.clpt.6100406  [DOI](https://doi.org/10.1038/sj.clpt.6100406) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/17971818/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=The%20CYP2D6%20activity%20score:%20Translating%20genotype%20information%20into%20a%20qualitative%20measure%20of%20phenotype&author=A.%20Gaedigk&author=S.%20Simon&author=R.%20Pearce&author=L.%20Bradford&author=M.%20Kennedy&volume=83&issue=2&publication_year=2008&pages=234-242&pmid=17971818&doi=10.1038/sj.clpt.6100406&)

14. Hicks J. K., Swen J. J., Gaedigk A. (2014). Challenges in CYP2D6 phenotype assignment from genotype data: A critical assessment and call for standardization. Curr. Drug Metab. 15 (2), 218–232. 10.2174/1389200215666140202215316  [DOI](https://doi.org/10.2174/1389200215666140202215316) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24524666/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Curr.%20Drug%20Metab.&title=Challenges%20in%20CYP2D6%20phenotype%20assignment%20from%20genotype%20data:%20A%20critical%20assessment%20and%20call%20for%20standardization&author=J.%20K.%20Hicks&author=J.%20J.%20Swen&author=A.%20Gaedigk&volume=15&issue=2&publication_year=2014&pages=218-232&pmid=24524666&doi=10.2174/1389200215666140202215316&)

15. Ito T., Kato M., Chiba K., Okazaki O., Sugiyama Y. (2010). Estimation of the interindividual variability of cytochrome 2D6 activity from urinary metabolic ratios in the literature. Drug Metab. Pharmacokinet. 25 (3), 243–253. 10.2133/dmpk.25.243  [DOI](https://doi.org/10.2133/dmpk.25.243) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/20610883/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Pharmacokinet.&title=Estimation%20of%20the%20interindividual%20variability%20of%20cytochrome%202D6%20activity%20from%20urinary%20metabolic%20ratios%20in%20the%20literature&author=T.%20Ito&author=M.%20Kato&author=K.%20Chiba&author=O.%20Okazaki&author=Y.%20Sugiyama&volume=25&issue=3&publication_year=2010&pages=243-253&pmid=20610883&doi=10.2133/dmpk.25.243&)

16. Kevin Hicks J., Sangkuhl K., Swen J. J., Ellingrod V. L., Müller D. J., Shimoda K., et al. (2016). Clinical Pharmacogenetics implementation Consortium guideline (CPIC®) for CYP2D6 and CYP2C19 genotypes and dosing of tricyclic antidepressants: 2016 update. Clin. Pharmacol. Ther. 102 (1), 37–44. 10.1002/cpt.597  [DOI](https://doi.org/10.1002/cpt.597) | [PMC free article](/articles/PMC5478479/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/27997040/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacol.%20Ther.&title=Clinical%20Pharmacogenetics%20implementation%20Consortium%20guideline%20(CPIC%C2%AE)%20for%20CYP2D6%20and%20CYP2C19%20genotypes%20and%20dosing%20of%20tricyclic%20antidepressants:%202016%20update&author=J.%20Kevin%20Hicks&author=K.%20Sangkuhl&author=J.%20J.%20Swen&author=V.%20L.%20Ellingrod&author=D.%20J.%20M%C3%BCller&volume=102&issue=1&publication_year=2016&pages=37-44&pmid=27997040&doi=10.1002/cpt.597&)

17. Leeder J. S., Gaedigk A. (2014). CYP2D6 and pharmacogenomics: Where does future research need to focus? Part 2: Clinical aspects. Pharmacogenomics 15 (8), 1055–1058. 10.2217/pgs.14.27  [DOI](https://doi.org/10.2217/pgs.14.27) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/25084197/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=CYP2D6%20and%20pharmacogenomics:%20Where%20does%20future%20research%20need%20to%20focus?%20Part%202:%20Clinical%20aspects&author=J.%20S.%20Leeder&author=A.%20Gaedigk&volume=15&issue=8&publication_year=2014&pages=1055-1058&pmid=25084197&doi=10.2217/pgs.14.27&)

18. Lötsch J., Rohrbacher M., Schmidt H., Doehring A., Brockmöller J., Geisslinger G. (2009). Can extremely low or high morphine formation from codeine be predicted prior to therapy initiation? Pain. 144 (1), 119–124. 10.1016/j.pain.2009.03.023  [DOI](https://doi.org/10.1016/j.pain.2009.03.023) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19395173/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pain.&title=Can%20extremely%20low%20or%20high%20morphine%20formation%20from%20codeine%20be%20predicted%20prior%20to%20therapy%20initiation?&author=J.%20L%C3%B6tsch&author=M.%20Rohrbacher&author=H.%20Schmidt&author=A.%20Doehring&author=J.%20Brockm%C3%B6ller&volume=144&issue=1&publication_year=2009&pages=119-124&pmid=19395173&doi=10.1016/j.pain.2009.03.023&)

19. Manoharan A., Shewade D. G., Ravindranath P. A., Rajkumar R. P., Ramprasad V. L., Adithan S., et al. (2019). Resequencing CYP2D6 gene in Indian population: CYP2D6*41 identified as the major reduced function allele. Pharmacogenomics 20 (10), 719–729. 10.2217/pgs-2019-0049  [DOI](https://doi.org/10.2217/pgs-2019-0049) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/31368850/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Resequencing%20CYP2D6%20gene%20in%20Indian%20population:%20CYP2D6*41%20identified%20as%20the%20major%20reduced%20function%20allele&author=A.%20Manoharan&author=D.%20G.%20Shewade&author=P.%20A.%20Ravindranath&author=R.%20P.%20Rajkumar&author=V.%20L.%20Ramprasad&volume=20&issue=10&publication_year=2019&pages=719-729&pmid=31368850&doi=10.2217/pgs-2019-0049&)

20. Murphy E., McMahon F. J. (2013). Pharmacogenetics of antidepressants, mood stabilizers, and antipsychotics in diverse human populations. Discov. Med. 16 (87), 113–122.  [PMC free article](/articles/PMC6011657/) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23998447/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Discov.%20Med.&title=Pharmacogenetics%20of%20antidepressants,%20mood%20stabilizers,%20and%20antipsychotics%20in%20diverse%20human%20populations&author=E.%20Murphy&author=F.%20J.%20McMahon&volume=16&issue=87&publication_year=2013&pages=113-122&pmid=23998447&)

21. O'mathúna B., Farré M., Rostami-Hodjegan A., Yang J., Cuyàs E., Torrens M., et al. (2008). The consequences of 3, 4-methylenedioxymethamphetamine induced CYP2D6 inhibition in humans. J. Clin. Psychopharmacol. 28 (5), 523–529. 10.1097/JCP.0b013e318184ff6e  [DOI](https://doi.org/10.1097/JCP.0b013e318184ff6e) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/18794647/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Psychopharmacol.&title=The%20consequences%20of%203,%204-methylenedioxymethamphetamine%20induced%20CYP2D6%20inhibition%20in%20humans&author=B.%20O'math%C3%BAna&author=M.%20Farr%C3%A9&author=A.%20Rostami-Hodjegan&author=J.%20Yang&author=E.%20Cuy%C3%A0s&volume=28&issue=5&publication_year=2008&pages=523-529&pmid=18794647&doi=10.1097/JCP.0b013e318184ff6e&)

22. Qian J. C., Xu X. M., Hu G. X., Dai D. P., Xu R. A., Hu L. M., et al. (2013). Genetic variations of human CYP2D6 in the Chinese Han population. Pharmacogenomics 14 (14), 1731–1743. 10.2217/pgs.13.160  [DOI](https://doi.org/10.2217/pgs.13.160) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/24192122/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&title=Genetic%20variations%20of%20human%20CYP2D6%20in%20the%20Chinese%20Han%20population&author=J.%20C.%20Qian&author=X.%20M.%20Xu&author=G.%20X.%20Hu&author=D.%20P.%20Dai&author=R.%20A.%20Xu&volume=14&issue=14&publication_year=2013&pages=1731-1743&pmid=24192122&doi=10.2217/pgs.13.160&)

23. Steimer W., Zöpf K., von Amelunxen S., Pfeiffer H., Bachofer J., Popp J., et al. (2004). Allele-specific change of concentration and functional gene dose for the prediction of steady-state serum concentrations of amitriptyline and nortriptyline in CYP2C19 and CYP2D6 extensive and intermediate metabolizers. Clin. Chem. 50 (9), 1623–1633. 10.1373/clinchem.2003.030825  [DOI](https://doi.org/10.1373/clinchem.2003.030825) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/15205367/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Chem.&title=Allele-specific%20change%20of%20concentration%20and%20functional%20gene%20dose%20for%20the%20prediction%20of%20steady-state%20serum%20concentrations%20of%20amitriptyline%20and%20nortriptyline%20in%20CYP2C19%20and%20CYP2D6%20extensive%20and%20intermediate%20metabolizers&author=W.%20Steimer&author=K.%20Z%C3%B6pf&author=S.%20von%20Amelunxen&author=H.%20Pfeiffer&author=J.%20Bachofer&volume=50&issue=9&publication_year=2004&pages=1623-1633&pmid=15205367&doi=10.1373/clinchem.2003.030825&)

24. Teh L. K., Bertilsson L. (2012). Pharmacogenomics of CYP2D6: Molecular genetics, interethnic differences and clinical importance. Drug Metab. Pharmacokinet. 27 (1), 55–67. 10.2133/dmpk.dmpk-11-rv-121  [DOI](https://doi.org/10.2133/dmpk.dmpk-11-rv-121) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/22185816/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Pharmacokinet.&title=Pharmacogenomics%20of%20CYP2D6:%20Molecular%20genetics,%20interethnic%20differences%20and%20clinical%20importance&author=L.%20K.%20Teh&author=L.%20Bertilsson&volume=27&issue=1&publication_year=2012&pages=55-67&pmid=22185816&doi=10.2133/dmpk.dmpk-11-rv-121&)

25. Zanger U. M., Raimundo S., Eichelbaum M. (2004). Cytochrome P450 2D6: Overview and update on pharmacology, genetics, biochemistry. Naunyn. Schmiedeb. Arch. Pharmacol. 369 (1), 23–37. 10.1007/s00210-003-0832-2  [DOI](https://doi.org/10.1007/s00210-003-0832-2) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/14618296/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Naunyn.%20Schmiedeb.%20Arch.%20Pharmacol.&title=Cytochrome%20P450%202D6:%20Overview%20and%20update%20on%20pharmacology,%20genetics,%20biochemistry&author=U.%20M.%20Zanger&author=S.%20Raimundo&author=M.%20Eichelbaum&volume=369&issue=1&publication_year=2004&pages=23-37&pmid=14618296&doi=10.1007/s00210-003-0832-2&)

26. Zanger U. M., Schwab M. (2013). Cytochrome P450 enzymes in drug metabolism: Regulation of gene expression, enzyme activities, and impact of genetic variation. Pharmacol. Ther. 138 (1), 103–141. 10.1016/j.pharmthera.2012.12.007  [DOI](https://doi.org/10.1016/j.pharmthera.2012.12.007) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/23333322/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Ther.&title=Cytochrome%20P450%20enzymes%20in%20drug%20metabolism:%20Regulation%20of%20gene%20expression,%20enzyme%20activities,%20and%20impact%20of%20genetic%20variation&author=U.%20M.%20Zanger&author=M.%20Schwab&volume=138&issue=1&publication_year=2013&pages=103-141&pmid=23333322&doi=10.1016/j.pharmthera.2012.12.007&)

27. Zhou S. F. (2009a). Polymorphism of human cytochrome P450 2D6 and its clinical significance: Part I. Clin. Pharmacokinet. 48 (11), 689–723. 10.2165/11318030-000000000-00000  [DOI](https://doi.org/10.2165/11318030-000000000-00000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19817501/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet.&title=Polymorphism%20of%20human%20cytochrome%20P450%202D6%20and%20its%20clinical%20significance:%20Part%20I&author=S.%20F.%20Zhou&volume=48&issue=11&publication_year=2009a&pages=689-723&pmid=19817501&doi=10.2165/11318030-000000000-00000&)

28. Zhou S. F. (2009b). Polymorphism of human cytochrome P450 2D6 and its clinical significance: Part II. Clin. Pharmacokinet. 48 (12), 761–804. 10.2165/11318070-000000000-00000  [DOI](https://doi.org/10.2165/11318070-000000000-00000) | [PubMed](https://pubmed.ncbi.nlm.nih.gov/19902987/) | [Google Scholar](https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet.&title=Polymorphism%20of%20human%20cytochrome%20P450%202D6%20and%20its%20clinical%20significance:%20Part%20II&author=S.%20F.%20Zhou&volume=48&issue=12&publication_year=2009b&pages=761-804&pmid=19902987&doi=10.2165/11318070-000000000-00000&)
